<article>
<section>
<h1 id="header">A sterol-defined system for quantitative studies of sterol metabolism in <i>C. elegans</i>
</h1>
<p><time datetime="2021-08-13">Published: August 13, 2021</time></p>
<p>Benjamin Trabelcy,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Yoram Gerchman,<sup><a href="#aff1">1</a></sup> and Amir Sapir<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Faculty of Natural Sciences, University of Haifa, Haifa 3498838 Israel</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:benytrabelsy@gmail.com">benytrabelsy@gmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:amir-s@sci.haifa.ac.il">amir-s@sci.haifa.ac.il</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100710">10.1016/j.xpro.2021.100710</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes the culturing of the nematode <i>Caenorhabditis elegans (C. elegans)</i> in a sterol-defined experimental system and the subsequent quantitative analysis of <i>C. elegans</i> sterols through gas chromatography-mass spectrometry. Although studied primarily in mammals, sterols are essential biomolecules for most eukaryotes. <i>C. elegans</i> cannot synthesize sterols and thus relies on the uptake of dietary sterols. Therefore, <i>C. elegans</i> is a powerful system to study metabolism in sterol-defined conditions that are described in our protocol.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib16">Shamsuzzama et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Principles for setting up sterol-defined experimental systems in <i>C. elegans</i>
</li>
<li>A protocol for medium-throughput analysis of sterol metabolism by GC-MS</li>
<li>Strategies for quantitatively determining the levels of sterols in <i>C. elegans</i>
</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The unique biology of <i>C. elegans</i> makes this nematode a powerful experimental system for the study of sterol metabolism (<a href="#bib16">Shamsuzzama et al., 2020</a>, <a href="#bib2">Capell-Hattam and Brown, 2020</a>, <a href="#bib22">Zhou et al., 2020</a>). <i>C. elegans</i> cannot synthesize sterols and thus relies on the uptake of dietary sterols, which are precursors to the synthesis of hormones that coordinate critical stages of <i>C. elegans</i> development. Moreover, <i>C. elegans</i> can be cultured in a sterol-defined environment in which the only source of sterols is dietary sterols of interest supplemented by the researcher.</p>
<ol type="a">
<li>The principles of establishing sterol-defined cultures.</li>
</ol>
<p>Before beginning the experiment, the researcher should acquire the mindset of working in a sterol-controlled environment. A typical laboratory setting is usually contaminated with many types of sterols that can affect the experiment—especially because <i>C. elegans</i> is a sterol auxotroph that can grow on a very small amount of dietary sterols (<a href="#bib3">Chitwood, 1999</a>, <a href="#bib10">Merris et al., 2003</a>). The researcher should avoid the contamination of solutions and lab equipment (labware) by sterols, including traces of sterols from previous experiments. Therefore, all labware to be used should be new and sterile, and gloves should be used throughout the experiment, including when handling <i>C. elegans</i> and its feed bacteria. Any solution or material that is not chemically defined (e.g., yeast extract or Luria Broth medium) should be avoided. Moreover, because most eukaryotes have a significant amount of sterols in their cells (<a href="#bib5">Desmond and Gribaldo, 2009</a>) contamination of the cultures by eukaryotic organisms should be avoided.</p>
<ol start="2" type="a">
<li>Adaptation of standard <i>C. elegans</i> protocols for sterol-defined studies.</li>
</ol>
<p><i>C. elegans</i> has its minimal dietary requirements met by a defined growth medium supplemented by sterols (<a href="#bib9">Merris et al., 2004</a>) and bacterial food. Most bacteria, <i>Escherichia coli</i> (<i>E. coli</i>) included, do not synthesize nor require sterols for living (<a href="#bib20">Wei et al., 2016</a>); thus, the bacterial food of <i>C. elegans</i> can be sterol-free. A sterol-defined <i>C. elegans</i> culture, however, requires several adaptations of the standard <i>C. elegans</i> culture protocol (<a href="#bib19">Stiernagle, 2006</a>). These include: <b>i)</b> Working in a sterol-free environment, as described above; <b>ii)</b> Growing <i>C. elegans</i> in liquid culture rather than on agar plates to avoid the potential contamination of the culture by algal sterols from the ager—a product of red algae. An additional advantage of liquid cultures is the large number of developmentally synchronized animals that can be harvested from such cultures (<a href="#bib6">Heshof et al., 2019</a>), allowing for the analysis of metabolites that are found in small quantities, like sterols (<a href="#bib9">Merris et al., 2004</a>, <a href="#bib21">Yilmaz et al., 2020</a>); <b>iii)</b> Growing feed bacteria in a defined (minimal) media, such as Minimum Essential Medium (MEM). MEM does not contain any sterol, in contrast to the commonly used Luria Broth (LB) medium, which includes yeast extract and thus is a potential source of contamination by the yeast sterol ergosterol.</p>
<p>The workflow described in this protocol (<a href="#fig1">Figure 1</a>) takes several days and requires the preparation of several solutions (see <a href="#materials-and-equipment">materials and equipment</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic workflow of sterol studies of <i>C. elegans</i>
</div>
<p>Cultures of synchronized larvae grown in sterol-defined conditions are harvested, dried in a lyophilizer, and subjected to sterol extraction by hexane fractionation. Next, the organic phase of the extraction is dried using nitrogen and derivatized by N,O-Bis (trimethylsilyl) trifluoroacetamide (BSTFA). Finally, the samples are injected into a GC-MS instrument to identify and quantify sterols. Scale bar represents 500 micro micrometers.</p>
</figcaption>
</figure>
<h3 id="sec1.1">Preparing of sterol-free <i>E. coli</i> bacteria as nematode food</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Sterol-free <i>E. coli</i> bacteria are prepared and used as the exclusive food of the nematodes throughout the experiment. Establishing a sterol-free <i>E. coli</i> culture is critical for achieving the end goal of culturing the nematodes with a single sterol of interest, without contamination from other environmental sterols.</p>
<ol>
<li>Growing a culture of <i>E. coli</i> feed bacteria.
<div class="note">
<span class="note-title">Note:</span> All the bacterial and <i>C. elegans</i> culturing steps should be performed under sterile conditions.</div>
<ol type="a">
<li>Using a sterile inoculation needle, take a slur of <i>E. coli</i> NA22 bacteria from a −80°C glycerol stock and streak it on a 90 mm LB-agar plate. Grow for 16 h at 37°C. The plate can be stored for a maximum of one week at 4°C before use.</li>
<li>In sterile conditions, using an inoculation needle, transfer 2 colonies from the plate to a 2-liter Erlenmeyer that contains a 500 mL MEM medium (see <a href="#materials-and-equipment">materials and equipment</a> for preparation details). This will result in about 25 mL of concentrated nematode food. More Erlenmeyer flasks can be grown as needed.
<div class="critical">
<span class="critical-title">Critical:</span> Sterols are abundant in the environment. Use gloves, new glassware, and plasticware through the establishment and the harvest of the culture.</div>
<div class="note">
<span class="note-title">Note:</span> Although the bacteria are streaked on an agar plate traces of sterols from the plate are not affecting the culture of <i>C. elegans</i> because they are diluted in the large volume of the MEM medium and diluted again in the <i>C. elegans</i> culture.</div>
</li>
<li>Grow the cultures for 20 h in a preheated 37°C shaker at 220 rounds per minute (henceforth, rpm).</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Harvesting the bacterial culture.
<ol type="a">
<li>Pre-weigh centrifuge vessels that can hold more than 500 mL of liquid and mark the weight of each vessel.</li>
<li>Transfer the bacterial cultures to the centrifuge vessels.</li>
<li>Spin at 5,000 <i>g</i> for 30 minutes at 4°C.</li>
<li>Carefully aspirate the soup without disturbing the bacterial pellet.</li>
<li>Weigh the vessels again and calculate the weight of the pellet.</li>
<li>Resuspend the pellets in S-basal medium (see <a href="#materials-and-equipment">materials and equipment</a> for preparation details). One gram of wet bacterial pellet should be resuspended in 4 mL of S-basal medium.
<div class="note">
<span class="note-title">Note:</span> It is easier to resuspend the pellet in 5 mL S-basal medium using a 5 mL pipette tip and, only once the pellet is completely dissolved, to add the rest of the S-basal medium to the centrifuge vessel.</div>
</li>
<li>Transfer the suspension to a 50 mL sterile plastic tube and keep it at 4°C.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The concentrated bacteria can be used as nematode food for up to a month.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Preparing synchronized, developmentally arrested <i>C. elegans</i> larvae</h3>
<div class="timing">
<span class="timing-title">Timing:</span> four days</div>
<p>This step aims to facilitate the establishment of a large population of developmentally arrested <i>C. elegans</i> larvae. This population is used to establish a synchronic culture of <i>C. elegans</i> that can be harvested at a specific point during the life cycle of <i>C. elegans</i> and analyzed by a GC-MS instrument in order to study stage-specific changes in sterol metabolism.</p>
<ol start="3">
<li>Growing <i>C. elegans</i> culture for bleaching.
<ol type="a">
<li>Establishing the culture: In a 250 mL Erlenmeyer set a culture of approximately 50,000 synchronized <i>C. elegans</i> larvae at the stage of L1 in 50 mL of S-complete medium (see <a href="#materials-and-equipment">materials and equipment</a> for preparation details). This culture should be supplemented with 1 mL of concentrated nematode food and 50 μL of the stock solution (13 μM) of the sterol of interest. Grow for 3 days at 20°C while agitating the cultures at 175 rpm.
<div class="note">
<span class="note-title">Note:</span> Hermaphrodite <i>C. elegans</i> is used through the protocol. In some nematode species, <i>C. elegans</i> included, liquid culture conditions may hinder male-female reproduction. Thus, it is advisable to use <i>C. elegans</i> hermaphrodites for experiments that require liquid cultures of worms (henceforth, “worm” means C. elegans).</div>
</li>
<li>Harvesting: Once most of the animals reach the adult stage and have eggs in their uterus, transfer the culture to a 50 mL sterile plastic tube and place the tube in an ice bucket for approximately 30 minutes until most of the adults sink to the bottom (nematodes at other stages will mostly float).</li>
<li>Washing: Carefully aspirate the upper phase without disturbing the pellet of adult nematodes. Add 40 mL of S-basal buffer to the pellet, place in an ice bucket for 30 minutes, and aspirate the upper phase. Repeat washing until the upper phase becomes clear (usually 4 washes).</li>
<li>Transferring to microcentrifuge tubes: Use a diamond knife to break the end of a glass Pasteur Pipette and, using this pipette, transfer the pellet of adults to microcentrifuge tubes. Each tube should contain about 100 μL of the compact worm pellet.
<div class="note">
<span class="note-title">Note:</span> In aquatic solutions, <i>C. elegans</i> nematodes adheres to plastic; thus, it is good practice to use a glass Pasteur Pipette rather than a plastic pipette for an optimal yield of the harvested cultures.</div>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Bleaching to obtain a synchronized population of <i>C. elegans</i> arrested at the larval stage one (henceforth, L1 larvae).
<ol type="a">
<li>To each microcentrifuge tube from step 3d (containing adult <i>C. elegans</i>) add 1 mL of the bleach solution (see <a href="#materials-and-equipment">materials and equipment</a> for preparation details), place the tubes in a vortex, and agitate for 8 minutes at maximal speed. Check for the complete disappearance of adult nematodes in the tubes by examining a drop of 10 μm from the tube under a dissecting scope. In case adult nematodes are still intact continue with the bleach until the completion of 10 minutes.
<div class="critical">
<span class="critical-title">Critical:</span> Household bleach is a hazardous chemical and should be used according to the manufacturer's safety guidelines. Household bleach should be aliquoted and stored at 4°C in the dark. The bleaching solution must be freshly prepared just before use.</div>
</li>
<li>Centrifuge the tubes at 1,500 <i>g</i> for 2 minutes, carefully aspirate the soup without disturbing the pellet—a pyramid of white-colored <i>C. elegans</i> eggs.</li>
<li>Add 1 mL of S-basal medium to the tube to break and wash the pellet, centrifuge the tubes at 1,500 <i>g</i> for 2 minutes, and aspire the liquid carefully. Repeat this wash two more times.</li>
<li>After the third wash, add 1 mL of S-basal medium to the tube and transfer the tube content (S-basal medium with eggs post-bleach) to a clean 30 mL glass test tube.</li>
<li>Leave the tubes on a shaker set to 175 rpm agitation for 24 h at 20°C. After this incubation period, all embryos should hatch and develop to the L1 larval stage and arrest due to lack of food and sterols, resulting in a synchronized population of animals.
<div class="critical">
<span class="critical-title">Critical:</span> Bleaching and incubation in S-basal medium should result in a culture of synchronized L1-arrested larvae without any carcasses or debris from the mothers. Such carcasses or debris can be a food source for the arrested L1s, resulting in their development and, consequently, the impaired synchronization of the culture. Moreover, efficient bleaching according to the guidelines above should result in a culture of live L1 larvae with no dead embryos and larvae. Before establishing the sterol-defined cultures, the level of embryonic and larval lethality should be monitored under a dissecting scope, and cultures with dead embryos and larvae should not be used.</div>
<div class="note">
<span class="note-title">Note:</span> L1 larvae in S-basal medium can survive up to 3 weeks, but it is advisable to continue with the protocol within 72 h post-bleach.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Preparing of dietary-sterol stocks</h3>
<p>The preparation of dietary-sterol stocks aims to minimize experimental noise by using the same sterol stock in all experiments.</p>
<div class="note">
<span class="note-title">Note:</span> For some sterols, it takes 24 h to completely dissolve in ethanol. Using stronger organic solvents like Dimethyl sulfoxide can harm <i>C. elegans</i> (<a href="#bib18">Solis and Petrascheck, 2011</a>) and therefore should be avoided. Thus, sterol stocks should be prepared at least 24 h before setting the cultures. All sterols used (see <a href="#key-resources-table">key resources table</a>) should be dissolved in analytical-grade ethanol. The solubility of different sterols varies between 2.6 and 13 mM (see table about the solubility of sterols).</div>
<ol start="5">
<li>Preparing the stock sterols
<ol type="a">
<li>Dissolve the desired amount of sterol in ethanol.</li>
<li>Leave at 4°C for 24 h until the powder completely disappears.</li>
<li>Aliquots the sterols in microcentrifuge tubes and store them at −20°C for up to 3 months.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Feed bacteria <i>Escherichia coli:</i> strain NA22</td>
<td>Caenorhabditis Genetics Center</td>
<td>NA22 (RRID: WBStrain 00041948)</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>7-Dehydrocholesterol</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat# sc-214398</td>
</tr>
<tr>
<td>Coprostanol</td>
<td>Sigma-Aldrich</td>
<td>Cat# C7578</td>
</tr>
<tr>
<td>Cholesterol</td>
<td>Sigma-Aldrich</td>
<td>Cat# 26732</td>
</tr>
<tr>
<td>Desmosterol</td>
<td>Cayman Chemical</td>
<td>Cat# 14943</td>
</tr>
<tr>
<td>Ergosterol</td>
<td>Sigma-Aldrich</td>
<td>Cat# 45480</td>
</tr>
<tr>
<td>Stigmasterol</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat# sc-281156</td>
</tr>
<tr>
<td>β-Sitosterol</td>
<td>Sigma-Aldrich</td>
<td>Cat# S9889</td>
</tr>
<tr>
<td>Chloroform Amylene-free</td>
<td>Carlo Erba Reagents</td>
<td>Cat# 438613</td>
</tr>
<tr>
<td>Ethanol</td>
<td>BioLabs</td>
<td>Cat# 05250501</td>
</tr>
<tr>
<td>Methanol</td>
<td>BioLabs</td>
<td>Cat# 001368052100</td>
</tr>
<tr>
<td>Hexane</td>
<td>BioLabs</td>
<td>Cat# 08290601</td>
</tr>
<tr>
<td>N,O-Bis (trimethylsilyl) trifluoroacetamide (BSTFA) with 1% trimethylchlorosilane (TMCS)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 155195</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>Caenorhabditis elegans</i> (<i>C. elegans</i>): strain N2</td>
<td>Caenorhabditis Genetics Center</td>
<td>N2 (RRID: WBStrain00000001)</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Enhanced ChemStation E.02.02.1431 used to operate the GC-MS</td>
<td>N/A</td>
<td><a href="https://www.agilent.com/en/products/software-informatics/massspec-workstations/gc-msd-chemstation-software">https://www.agilent.com/en/products/software-informatics/massspec-workstations/gc-msd-chemstation-software</a></td>
</tr>
<tr>
<td>NIST MS Search 2.0 (Database NIST11) — used to identify compounds</td>
<td>N/A</td>
<td><a href="https://chemdata.nist.gov/mass-spc/ms-search/">https://chemdata.nist.gov/mass-spc/ms-search/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Gas chromatograph (GC)</td>
<td>Agilent</td>
<td>Model 7890A</td>
</tr>
<tr>
<td>Rxi-5Sil capillary column</td>
<td>Restek</td>
<td>Cat# 13623</td>
</tr>
<tr>
<td>Single-quad mass spectrometer (MS)</td>
<td>Agilent</td>
<td>Model 5975C</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>M9 salts (X5)</caption>
<thead>
<tr>
<th>Ingredients</th>
<th>Final concentration (mM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub> •7H2O</td>
<td>450.83</td>
<td>64 grams (g)</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>110.22</td>
<td>15 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>42.77</td>
<td>2.5 g</td>
</tr>
<tr>
<td>NH<sub>4</sub>Cl</td>
<td>93.47</td>
<td>5.0 g</td>
</tr>
<tr>
<td>Double distilled water (henceforth, ddH<sub>2</sub>O)</td>
<td>not applicable (henceforth, n/a)</td>
<td>Up to 1 liter</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 liter</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>Minimum Essential Medium (MEM)</caption>
<thead>
<tr>
<th>Ingredients</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>M9 salts (x5)</td>
<td>n/a</td>
<td>200 mL</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> (1M)</td>
<td>2 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> (1M)</td>
<td>100 μM</td>
<td>100 μL</td>
</tr>
<tr>
<td>20% dextrose in ddH<sub>2</sub>O (1.1M)</td>
<td>22.22 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>to 1 liter</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 liter</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>S-basal medium</caption>
<thead>
<tr>
<th>Ingredients</th>
<th>Final concentration (mM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>100</td>
<td>5.85 g</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>5.74</td>
<td>1 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>44.09</td>
<td>6 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>to 1 liter</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 liter</b></td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>Trace metals solution</caption>
<thead>
<tr>
<th>Ingredients</th>
<th>Final concentration (mM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Disodium EDTA</td>
<td>5</td>
<td>1.86 g</td>
</tr>
<tr>
<td>FeSO<sub>4</sub> •7H<sub>2</sub>O</td>
<td>2.5</td>
<td>0.69 g</td>
</tr>
<tr>
<td>MnCl<sub>2</sub> •4H<sub>2</sub>O</td>
<td>1</td>
<td>0.20 g</td>
</tr>
<tr>
<td>ZnSO<sub>4</sub> •7H<sub>2</sub>O</td>
<td>1</td>
<td>0.29 g</td>
</tr>
<tr>
<td>CuSO<sub>4</sub> •5H<sub>2</sub>O</td>
<td>0.1</td>
<td>0.025 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>0</td>
<td>n/a</td>
<td>to 1 liter</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 liter</b></td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>S-complete medium</caption>
<thead>
<tr>
<th>Ingredients</th>
<th>Final concentration (mM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub> (1M)</td>
<td>3</td>
<td>3 mL</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> (1M)</td>
<td>3</td>
<td>3 mL</td>
</tr>
<tr>
<td>Trace metals</td>
<td>n/a</td>
<td>10 mL</td>
</tr>
<tr>
<td>Potassium citrate pH 6 (1M)</td>
<td>10</td>
<td>10 mL</td>
</tr>
<tr>
<td>S-basal</td>
<td>n/a</td>
<td>974 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 liter</b></td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>Bleach solution</caption>
<thead>
<tr>
<th>Ingredients</th>
<th>Final concentration</th>
<th>Amount (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Household bleach</td>
<td>60 mg/10 mL</td>
<td>2</td>
</tr>
<tr>
<td>KOH (5N)</td>
<td>0.5 N</td>
<td>1</td>
</tr>
<tr>
<td>ddH<sub>2</sub>0</td>
<td>n/a</td>
<td>7</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10</b></td>
</tr>
</tbody>
</table>
<table id="undtbl7">
<caption>Solubility of sterols</caption>
<thead>
<tr>
<th>Sterol</th>
<th>Source</th>
<th>Cat#</th>
<th>Maximum solubility in ethanol (mM)</th>
<th>Final concentration in cultures (μM)</th>
</tr>
</thead>
<tbody>
<tr>
<td>7-dehydrocholesterol</td>
<td>Santa Cruz Biotechnology</td>
<td>sc-214398</td>
<td>13</td>
<td>13</td>
</tr>
<tr>
<td>Coprostanol</td>
<td>Sigma-Aldrich</td>
<td>C7578</td>
<td>13</td>
<td>13</td>
</tr>
<tr>
<td>Cholesterol</td>
<td>Sigma-Aldrich</td>
<td>26732</td>
<td>13<sup>a</sup>
</td>
<td>13</td>
</tr>
<tr>
<td>Desmosterol</td>
<td>Cayman Chemical</td>
<td>14943</td>
<td>13</td>
<td>13</td>
</tr>
<tr>
<td>Ergosterol</td>
<td>Sigma-Aldrich</td>
<td>45480</td>
<td>2.6</td>
<td>13</td>
</tr>
<tr>
<td>Stigmasterol</td>
<td>Santa Cruz Biotechnology</td>
<td>sc-281156</td>
<td>2.6</td>
<td>13</td>
</tr>
<tr>
<td>β-sitosterol</td>
<td>Sigma-Aldrich</td>
<td>S9889</td>
<td>2.6</td>
<td>13</td>
</tr>
</tbody>
</table>
<p class="table-legend"><sup>a</sup> Cholesterol forms white precipitates in ethanol. After 24 h at 4°C, the cholesterol is completely dissolved.</p>
<p>Preparation of solutions:</p>
<ul>
<li>M9 slates (X5): mix all ingredients, autoclave, store at ∼20°C indefinitely until required for use.</li>
<li>MEM: mix all ingredients, filter through a 0.22 μm filter, store at 4°C indefinitely until required for use.</li>
<li>S-basal medium: mix all ingredients, autoclave, store at ∼20°C indefinitely until required for use.</li>
<li>Trace metals solution: mix all ingredients, filter through a 0.22 μm filter, store at ∼20°C in the dark indefinitely until required for use.</li>
<li>S-complete medium: mix all ingredients, store at 4°C indefinitely until required for use.</li>
<li>Bleach solution: prepare fresh every time by mixing all ingredients.</li>
</ul>
<ul>
<li>Alternative: Many laboratories use NaOH in the bleach solution instead of KOH. In our experience, the use of KOH reduces the level of embryonic lethality during bleaching compared to NaOH, but we have not determined this quantitatively.</li>
<li>Alternative: Some laboratories use bleach from Sigma-Aldrich (e.g., Cat# 425044), but we found that local household bleach works well; however, because of the variability of household bleach solutions, the effectiveness of each new batch should be tested.</li>
<li>Alternative: In many protocols, the bleaching is carried out in 15 mL tubes, but we found that bleaching in microcentrifuge tubes is easier and has the same effectiveness.</li>
<li>Alternative: Many laboratories use M9 medium instead of S-basal for the incubation of post-bleached embryos. We prefer incubation in S-basal because, in the subsequent experiment, the worms are cultured in an S-complete medium that is based on S-basal.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Establishing sterol-defined <i>C. elegans</i> cultures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days—when done at 20°C</div>
<p>This step aims to grow, for one generation (three days), a synchronized population of <i>C. elegans</i> in liquid culture supplemented with the sterol of interest or a vehicle (ethanol) control. The cultured <i>C. elegans</i> is harvested for downstream GC-MS analysis. A 100 mL of culture per condition should be prepared. Each such 100 mL culture gives approximately 330 mg of worm pellet that is sufficient for several GC-MS analyses.</p>
<ol>
<li>Establishing the culture.
<ol type="a">
<li>To a 250 mL Erlenmeyer, add 100 mL S-complete medium, 2 mL from the NA22 bacterial food, the sterol of interest at 13 μM culture concentration, and 100,000 arrested, synchronized, and bacteria-free L1 larvae.
<div class="critical">
<span class="critical-title">Critical:</span> The synchronized L1-arrested larvae should not be used earlier than 16 h post-bleach or later than 72 h of incubation in S-basal at 20°C. Incubation shorter than 16 h may result in the miscalculation of the number of worms in the culture (see below), whereas culturing the L1 worms for more than 72 h may affect their survival, health, and the level of synchronization.</div>
</li>
<li>Spit the 100 mL nematode culture into two 250 mL Erlenmeyer vessels (i.e., 50 mL into each vessel), cover the Erlenmeyer's opening with aluminum foil, and incubate on a shaker agitating at 175 rpm for about 3 days at 20°C.
<div class="critical">
<span class="critical-title">Critical:</span> The concentration of the worms in the culture should be about one worm per μl. More than three worms per μl may cause the worms in the culture to develop into the dauer diapause stage (<a href="#bib1">Antebi, 2013</a>), whereas cultures that are too diluted will result in an insufficient amount of worm pellet for the GC-MS analysis.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Harvesting, cleaning, and storing the cultured worms.
<ol type="a">
<li>After 48 h, check the developmental stage of the worms in order to harvest the culture when most worms are young adults with a few eggs. This point in time usually occurs around 72 h after the establishment of the culture but it can vary. Ideally, monitor the culture microscopically every 6 h.</li>
<li>Transfer the culture to two new 50 mL sterile plastic tubes and place them in an ice bucket for 30 minutes until a pellet is visible—this pellet contains mostly adult worms.</li>
<li>Carefully aspirate the liquid phase without disturbing the pellet. Add 40 mL of S- basal buffer, leave on ice for 30 minutes, and aspirate the liquid. Repeat until the upper phase becomes clear (usually 4 washes).
<div class="note">
<span class="note-title">Note:</span> The washing steps aim to clean the adult worms from bacteria and enrich the culture for adult <i>C. elegans</i>, which sink more easily than eggs or young larvae.</div>
</li>
<li>Add another 40mL of S-basal medium, seal the tube with a parafilm laboratory tape, place the tubes horizontally on a shaker and agitate at 175 rpm for 2 h at 20°C.
<div class="note">
<span class="note-title">Note:</span> The two-hour incubation after the washing steps aims to clean as much as possible the alimentary canal of the worms from bacteria.</div>
</li>
<li>Place the tubes in an ice bucket for 30 minutes, carefully aspirate the liquid without disturbing the pellet, add a further 40 mL of S-basal solution, and place the tubes in an ice bucket again as above.</li>
<li>Repeat step 2e two more times.</li>
<li>Transfer the worm pellet to pre-weighed 2 mL microcentrifuge tubes that have been pre-cooled in an ice bucket. Use a diamond knife to break the end of a Pasteur Pipette and, using this pipette, transfer the pellet of adults to the tubes. Each tube should contain about 500 μL of loose worm pellet.
<div class="critical">
<span class="critical-title">Critical:</span> The use of 2 mL microcentrifuge tubes is important for the extraction procedure (see below). Do not use the standard 1.7 mL tubes.</div>
<div class="note">
<span class="note-title">Note:</span> The 2 mL microcentrifuge tubes will be used for sterol extraction. In chemical analysis protocols it is commonly advised to use glassware rather than plasticware in order to avoid chemical contaminants. Nevertheless, to analyze a large number of small biological samples, the use of glass tubes might be challenging. To control for possible contaminants, it is highly advisable to use the same type of plasticware for blanks, standards, and samples. Moreover, the specific plasticware used should be tested for contaminants for each analyte and in case of contamination extraction should be conducted using glassware.</div>
</li>
<li>Spin the tubes at maximal speed and immediately aspirate the liquid. Repeat centrifugation and aspiration until the tube contains only the compact worm pellet (about 300 μL in volume).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be stored at −80°C indefinitely until required for use.</div>
</li>
<li>Uncap the tubes with the worm pellets and place them in a lyophilizer, weigh the pellets every hour until there is no more weight loss due to evaporation. In our Heto FD3 freeze dryer, this takes 2 h, but time may vary with vacuum strength.</li>
<li>Weigh the tubes with the dry worm pellets and store them at −80°C until required for use. The dry weight of the pellet will be used to normalize the samples in the quantification step (<a href="#fig6">Figure 6</a>).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Lyophilized Samples can be stored indefinitely at −80°C until required for use.</div>
<div class="note">
<span class="note-title">Note:</span> To analyze worms that feed on a particular dietary sterol, worms should be cultured in the presence of this sterol for at least two generations in order for the mothers to transfer only the sterol of interest to the offspring. We usually grow cultures for ten generations on a specific sterol before the experiment. However, because <i>C. elegans</i> cannot grow to adulthood for more than one generation without sterols (<a href="#bib9">Merris et al., 2004</a>, <a href="#bib8">Matyash et al., 2004</a>), the culture of vehicle (ethanol) control worms can only be grown for one generation.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Sucrose floatation is an alternative method to clean <i>C. elegans</i> cultures from bacteria, carcasses, and debris (<a href="#bib15">Russell et al., 2020</a>). However, sucrose floatation typically results in the loss of many adult worms. Moreover, the high osmolarity of sucrose may affect downstream procedures like sterol extraction. Thus, it is advisable to use gentler washing procedures as described here, rather than sucrose floatation.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Lipid extraction and derivatization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 h</div>
<p>In this step, lipids are extracted from the dry pellets of <i>C. elegans</i> and the extracts undergo derivatization to enable the detection of sterols by the GC-MS instrument.</p>
<ol start="3">
<li>Saponifying the samples
<ol type="a">
<li>Subject the pellets to three cycles of −80°C freezing and room-temperature thawing to break the cuticle of the worms.</li>
<li>To each sample, add 500 μL of analytical-grade ethanol containing 4 μg of the internal standard; in our protocol, the internal standard is coprostanol (<a href="#bib9">Merris et al., 2004</a>).
<div class="critical">
<span class="critical-title">Critical:</span> It is critical to use internal standards to assess the efficacy of the extraction, derivatization, and detection steps. Not only is an internal standard essential for the comparison of the levels of sterols between samples, but it is also required for quantitative studies of sterols in the same sample (<a href="#bib11">Mitsche et al., 2015</a>). Before using a specific internal standard, perform one GC-MS analysis of the standard to find the retention time of the internal standard peak. Next, a worm sample should be analyzed to confirm that the internal standard is absent in worms. Finally, a <i>C. elegans</i> sample with the added internal standard should be analyzed to ensure that the peak of the internal standard does not overlap with any of the peaks of sterols of interest (for example, cholesterol in <a href="#fig2">Figure 2</a>).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative GC-MS chromatographs of worm samples with and without an internal standard</div>
<p>(A) <i>C. elegans</i> samples without any internal standard.</p>
<p>(B) <i>C. elegans</i> samples with the coprostanol internal standard. No endogenous coprostanol is detected (arrow in A) and the retention time is different between the internal standard and <i>C. elegans</i> sterols as shown in panel B (see <a href="#fig3">Figure 3</a> for more sterols).</p>
</figcaption>
</figure>
</li>
<li>Add 50 μL of freshly made 10N aqueous NaOH to each 2 mL tube. Secure the cap of each tube with a microcentrifuge tube holder and incubate the tubes at 70°C for 1 h. Next, move the tubes to a 43°C water bath for three minutes.
<div class="critical">
<span class="critical-title">Critical:</span> Securing the tube caps is critical to protect the samples from a spillover caused by ethanol that evaporates at 70°C. Alternatively use screw-cap tubes.</div>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Extracting lipids from the sample
<ol type="a">
<li>After the tubes are cooled to ∼20°C, open carefully to avoid spillover, and add 250 μL of ddH<sub>2</sub>O to each tube, mix and add 500 μL of hexane.</li>
<li>Cap and seal the tubes with a parafilm laboratory film, vortex for one minute, and spin at 4,000 <i>g</i> for five minutes.</li>
<li>Using a tip, carefully collect the upper (organic) phase of each tube, about 500 μL in volume, and transfer to a new 2 mL microcentrifuge tube. Repeat steps 4a–c two more times, combining the organic phases.</li>
<li>Add 500 μL of water to each tube that contains the organic phase, and repeat fractionation as in step 2b. This step washes the hexane fraction from carryovers.</li>
<li>Carefully aspirate the hexane upper phase and transfer to a new gas chromatography glass screw-thread vial (e.g., 9 mm wide vials).
<div class="critical">
<span class="critical-title">Critical:</span> Be sure not to take any of the lower aqueous phases because water is a strong inhibitor of the derivatization reaction.</div>
</li>
<li>Dry the hexane with N<sub>2</sub> gas flow in a fume hood. Typically, sterols are dried out as white sediments on the walls just above the bottom of the vial.
<div class="critical">
<span class="critical-title">Critical:</span> Be sure that no water in any form (drops, vapor, etc.) is left in the tube because the derivatization process is inhibited by water.</div>
</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Derivatization of the lipids
<ol type="a">
<li>Add 50 μL of BSTFA to each sample and cap the vials with PTFE/silicon caps.
<div class="critical">
<span class="critical-title">Critical:</span> The BSTFA should be fresh for the optimal derivatization of substrates.</div>
</li>
<li>Incubate the vials in an 80°C dry block for 30 minutes.</li>
<li>Add 150 μL of hexane to each vial, mix to dissolve sterols that are on the vials' walls, and cap the vials. Preferably, samples should be analyzed immediately in the GC-MS instrument.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> It is advisable not to pause during the extraction and derivatization steps. If necessary, samples after derivatization can be stored at −20°C for up to 3 days before the GC-MS analysis (longer storage may result in sample degradation or solvate evaporation).</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">GC-MS analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h per sterol</div>
<p>The GC-MS analysis aims to identify specific sterols in samples of interest and to quantify sterol levels within and between samples.</p>
<ol start="6">
<li>Building a GC-MS program for the separation of the different sterols
<ol type="a">
<li>Using a mix of sterol standards, build a GC-MS program for the separation and characterization of sterols (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative GC-MS chromatographs of a mix of extracted standard sterols</div>
<p>For GC-MS analysis we choose standards for the primary plant, fungal, and animal sterols as well as some intermediates of sterol metabolism in <i>C. elegans</i> and an internal standard (Coprostanol). 1. Coprostanol; 2. Cholesterol; 3. Desmosterol; 4. 7-dehydrocholesterol; 5. Ergosterol; 6. Stigmasterol; 7. β-sitosterol.</p>
</figcaption>
</figure>
</li>
<li>GC details (these may vary by the type of the GC-MS instrument and column used): we used a gas chromatograph (Agilent Technologies, 7890A), equipped with a Rxi-5Sil MS capillary column (Restek, 13623). The GC was coupled to a single-quad mass spectrometer detector (Agilent technologies, 5975C).</li>
<li>Program details: set the oven temperature to 70°C, raising to 250°C at a rate of 20°C per minute, and then from 250°C to 320°C at 10°C per minute, and then hold at 320°C for 10 minutes. Set injector temperature to 280°C and let the carrier gas (helium) flow at 1 mL/minute<sup>−1</sup>. Use a splitless injection with a purge time of 1 minute to inject a 2 μL volume of each sample by an auto-sampler.
<div class="critical">
<span class="critical-title">Critical:</span> The injector temperature should be higher than 250°C for the sterols to evaporate.</div>
</li>
<li>Set the mass spectrometer transfer line to 320°C, the ion source temperature to 230°C, and the single-quad mass spectrometer detector temperature to 150°C in a scan mode, from 40 to 550 m/z.
<div class="note">
<span class="note-title">Note:</span> If peak fronting or tailing appears, samples can be injected in split mode, although this could result in a lower sensitivity of detection and the loss of the detection of minor peaks, such as the 7-dehydrocholesterol in <a href="#fig2">Figure 2</a>.</div>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Identifying and quantifying different sterols.
<ol type="a">
<li>Determine the retention time (the time where the molecule leaves the column and reaches the detector) of each sterol. Comparison to the retention time of known standards can reveal the identity of the sterols of interest (<a href="#fig3">Figure 3</a>).</li>
<li>Identification by retention times, however, should be further validated through fragmentation patterns. MS analysis involves the breakages of the molecules that reach the detector to smaller ion fragments. Dependent on the original structure of the molecule, this fragmentation pattern can be used to distinguish between different molecules with the same retention time (for example 7-dehydrocholesterol and desmosterol (<a href="#fig3">Figure 3</a> and step 7f)). Validate sterol identity by comparing the fragmentation patterns of the sterols of interest obtained by scan mode screening from 40 to 550 m/z (step 6d), with published reference spectra.</li>
<li>Build a calibration curve for every sterol of interest by dissolving its purified commercial standard in ethanol in a designated quantity. In our GC-MS instrument, we use a range between 7.81 and 125 μg/mL but the exact range is instrument-dependent. Calculate the peak area for every concentration, build a calibration curve (<a href="#fig4">Figure 4</a>), and calculate the calibration equation for each sterol (<a href="#tbl1">Table 1</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Calibration curves of different standard sterols</div>
<p>Sterols were weighed and dissolved in ethanol and diluted to the designated quantities. Each calibration curve is based on three independent experiments. Error bars represent standard deviation (Std.) that for some points are too small to be visualized as bars in the graphs.</p>
</figcaption>
</figure>
<table id="tbl1">
<caption>Table 1. Calibration equations for the different sterols</caption>
<thead>
<tr>
<th> </th>
<th>Sterol</th>
<th>Equation</th>
<th>R<sup>2</sup>
</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td>Cholesterol</td>
<td>Y=5E+07X-1E+06</td>
<td>0.987</td>
</tr>
<tr>
<td>B</td>
<td>Desmosterol</td>
<td>Y=7E+07X-1E+06</td>
<td>0.982</td>
</tr>
<tr>
<td>C</td>
<td>7-dehydrocholesterol</td>
<td>Y=2E+07X-471776</td>
<td>0.969</td>
</tr>
<tr>
<td>D</td>
<td>Ergosterol</td>
<td>Y=2E+07X-725537</td>
<td>0.978</td>
</tr>
<tr>
<td>E</td>
<td>Stigmasterol</td>
<td>Y=7E+07X-2E+06</td>
<td>0.985</td>
</tr>
<tr>
<td>F</td>
<td>β-sitosterol</td>
<td>Y=5E+07X-1E+06</td>
<td>0.985</td>
</tr>
</tbody>
</table>
<p class="table-legend">The calibration equations were obtained from the calibration curves (<a href="#fig4">Figure 4</a>) of three independent GC-MS analyses.</p>
<div class="note">
<span class="note-title">Note:</span> As expected, the GC-MS analysis shows that the supplemented dietary sterol is the predominant peak in each sample but peaks of endogenous sterols are also detected (<a href="#fig6">Figure 6</a>).</div>
</li>
<li>For every extraction procedure, calculate by GC-MS analysis the level of the internal standard before and after extraction to estimate the extraction efficiency.</li>
<li>Quantify the level of each sterol of interest by placing the value of its peak area in the line equation obtained from the calibration curve (<a href="#fig4">Figure 4</a> and <a href="#tbl1">Table 1</a>). Correct the value according to the extraction efficiency of the internal standard and divide the value by the dry weight of the sample (see <a href="#tbl2">Table 2</a> and <a href="#fig6">Figure 6</a>).
<table id="tbl2">
<caption>Table 2. An example of the calculation of the quantities of extracted sterol standards according to the peak areas obtained by the chromatographs in <a href="#fig3">Figure 3</a>
</caption>
<thead>
<tr>
<th> </th>
<th>TIC peak area</th>
<th>Extraction efficiency (%)</th>
<th>Quantity (μg)</th>
<th>Corrected quantity (μg)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Coprostanol (Internal Standard)</td>
<td>4549529</td>
<td>0.79</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>Cholesterol</td>
<td>7323398</td>
<td> </td>
<td>0.12</td>
<td>0.15</td>
</tr>
<tr>
<td>Desmosterol</td>
<td>5582508<sup><a href="#tblfn1">a</a></sup>
</td>
<td> </td>
<td>0.94</td>
<td>1.19</td>
</tr>
<tr>
<td>7-dehydrocholesterol</td>
<td>2427466<sup><a href="#tblfn1">a</a></sup>
</td>
<td> </td>
<td>0.14</td>
<td>0.18</td>
</tr>
<tr>
<td>Ergosterol</td>
<td>1918794</td>
<td> </td>
<td>0.13</td>
<td>0.17</td>
</tr>
<tr>
<td>Stigmasterol</td>
<td>6304675</td>
<td> </td>
<td>0.12</td>
<td>0.15</td>
</tr>
<tr>
<td>β-sitosterol</td>
<td>3686069</td>
<td> </td>
<td>0.09</td>
<td>0.12</td>
</tr>
</tbody>
</table>
<p class="table-legend">The calculation of quantities is based on the peak area of each sterol and its calibration curve equation (<a href="#tbl1">Table 1</a>), divided by the extraction efficiency of the internal standard.</p>
<p id="tblfn1" class="table-footnote"><sup>a</sup>Areas of desmosterol and 7-dehydrocholesterol were estimated as described below.</p>
<div class="critical">
<span class="critical-title">Critical:</span> If the peak area of a given sterol is outside the range of the calibration curve sterol concentration cannot be calculated accurately. In this case, dilute or concentrate the sample to fit the range of the calibration curve.</div>
<div class="note">
<span class="note-title">Note:</span> In some studies, normalization is based on the endogenous level of a single or group of biomolecules (e.g., total phospholipids). However, because changes in sterols metabolism can affect the level of many different metabolites including phospholipids it is better to normalize to dry weight.</div>
</li>
<li>The peaks of desmosterol and 7-dehydrocholesterol overlap (<a href="#fig3">Figure 3</a>) preventing their distinction by retention time. However, the analysis of pure standards of the two sterols in scan mode revealed two ions of M/Z ratio 372.7 and 373.3, detected in desmosterol but not in 7-dehydrocholesterol (<a href="#fig5">Figure 5</a>). A constant ratio of 0.0042 between the accumulative area of the desmosterol-specific peaks and total ions peak (<a href="#tbl3">Table 3</a>) was determined. This ratio allows the estimation of the desmosterol fraction of the shared peak. For example, in <a href="#fig3">Figure 3</a>, the total-ion peak area is 8,009,974. The area of the same peak when only the 372.7 and 373.3 ions are accounted for is 23,612. Using the 0.0042 ratio, the desmosterol-specific peak area is calculated to be 5,582,508 (23,612/0.0042). The subtraction of the peak area of desmosterol from the area of the total peak reveals the peak area of 7-dehydrocholesterol (8,009,974 - 5,582,508 = 2,427,466), resulting in 1.19 μg and 0.18 μg respectively (<a href="#tbl2">Table 2</a>). We used the National Institute of Standards and Technology NIST11 library (<a href="#tbl4">Table 4</a>) as a source for fragmentation patterns of standard sterols.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Fragmentation pattern and the identification of specific MS ions for desmosterol</div>
<p>(A) The fragmentation pattern of 7-dehydrocholesterol.</p>
<p>(B) Analysis of the fragmentation pattern of desmosterol reveals two ions, 372.7 and 373.3 that are not detected in 7-dehydrocholesterol (inserts in A and B). We used these specific ions to calculate the levels of the two sterols (step 7f). Specific ions were selected only if they were separated from main ions mass in more than 0.05 units. For example, the specific ions around 366 in 7-dehydrocholesterol (insert in A) cannot be used to distinguish it from desmosterol because they are too close to the mass of the 366 ion.</p>
</figcaption>
</figure>
<table id="tbl3">
<caption>Table 3. Calculation of desmosterol and 7-dehydrocholetserol levels</caption>
<thead>
<tr>
<th>Biological replica</th>
<th>Desmosterol- specific ion 372.7 area</th>
<th>Desmosterol- specific ion 373.3 area</th>
<th>Ions 372.7+ 373.3 area</th>
<th colspan="2">Desmosteroltotal-ions area</th>
<th>Specific-ions area/total-ions area</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>23,999</td>
<td>11,329</td>
<td>35,328</td>
<td colspan="2">7,396,564</td>
<td>0.0048</td>
</tr>
<tr>
<td>2</td>
<td>12,401</td>
<td>6,578</td>
<td>18,979</td>
<td colspan="2">4,576,532</td>
<td>0.0041</td>
</tr>
<tr>
<td>3</td>
<td>35,262</td>
<td>12,745</td>
<td>48,007</td>
<td colspan="2">1,3150,659</td>
<td>0.0037</td>
</tr>
<tr>
<td>4</td>
<td>43,308</td>
<td>20,818</td>
<td>64,126</td>
<td colspan="2">14,759,511</td>
<td>0.0043</td>
</tr>
<tr>
<td> </td>
<td> </td>
<td> </td>
<td colspan="2"> </td>
<td>Avg.</td>
<td>0.0042</td>
</tr>
<tr>
<td> </td>
<td> </td>
<td> </td>
<td colspan="2"> </td>
<td>Std.</td>
<td>0.00046</td>
</tr>
</tbody>
</table>
<p class="table-legend">The ratios of peak areas were obtained by subtracting the desmosterol-peak area from the total-peak area. This ratio enables the calculation of the amount of both desmosterol and 7-dehydrocholesterol in the samples.</p>
<table id="tbl4">
<caption>Table 4. Characterization of the sterols identified and quantified using the GC-MS analysis</caption>
<thead>
<tr>
<th> </th>
<th>Trivial name</th>
<th>Systematic name</th>
<th>MF; MW (g/mol)</th>
<th>Sterol structure</th>
<th>Sterol ester structure (after derivatization)</th>
<th>Similarity to database NIST11 (%)</th>
<th>Retention time (min)</th>
<th>Major ions (<i>m/z</i>) resulted from derivatized sterols by order of prevalence (high to low)</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>Coprostanol</td>
<td>5β-cholestan-3β-ol</td>
<td>C<sub>27</sub>H<sub>48</sub>O; 388.7</td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem1.jpg" alt="Chem1.jpg"></td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem2.jpg" alt="Chem2.jpg"></td>
<td>44.5</td>
<td>19.60</td>
<td>370, 55,75,215,43,371,57,55,73,95</td>
</tr>
<tr>
<td>2</td>
<td>Cholesterol</td>
<td>Cholest-5-en-3β-ol</td>
<td>C<sub>27</sub>H<sub>46</sub>O;386.7</td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem3.jpg" alt="Chem3.jpg"></td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem4.jpg" alt="Chem4.jpg"></td>
<td>80.2</td>
<td>20.60</td>
<td>129,329,73,43,368,75,57,95,458,41</td>
</tr>
<tr>
<td>3</td>
<td>Desmosterol</td>
<td>Cholesta-5,24-dien-3β-ol</td>
<td>C<sub>27</sub>H<sub>44</sub>O;384.6</td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem5.jpg" alt="Chem5.jpg"></td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem6.jpg" alt="Chem6.jpg"></td>
<td>92.3</td>
<td>21.08</td>
<td>69,129,73,75,41,55,343,81,95,119</td>
</tr>
<tr>
<td>4</td>
<td>7-dehydro-cholesterol</td>
<td>Cholesta-5,7-dien-3β-ol</td>
<td>C<sub>27</sub>H<sub>44</sub>O;384.6</td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem7.jpg" alt="Chem7.jpg"></td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem8.jpg" alt="Chem8.jpg"></td>
<td>88.1</td>
<td>21.08</td>
<td>351,325,73,352,43,366,456,143,326,75</td>
</tr>
<tr>
<td>5</td>
<td>Ergosterol</td>
<td>24-methylcholesta-5,7,22E-trien-3β-ol</td>
<td>C<sub>28</sub>H<sub>44</sub>O;396.6</td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem9.jpg" alt="Chem9.jpg"></td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem10.jpg" alt="Chem10.jpg"></td>
<td>95.7</td>
<td>21.50</td>
<td>363,43,73,69,55,337,75,41,157,81</td>
</tr>
<tr>
<td>6</td>
<td>Stigmasterol</td>
<td>Stigma-5,22-dien-3β-ol</td>
<td>C<sub>29</sub>H<sub>48</sub>O;412.7</td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem11.jpg" alt="Chem11.jpg"></td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem12.jpg" alt="Chem12.jpg"></td>
<td>90.3</td>
<td>22.05</td>
<td>83,129,55,73,69,75,81,43,484,133</td>
</tr>
<tr>
<td>7</td>
<td>β-sitosterol</td>
<td>5-stigmasten-3β-ol</td>
<td>C<sub>29</sub>H<sub>50</sub>O;414.7</td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem13.jpg" alt="Chem13.jpg"></td>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Chem14.jpg" alt="Chem14.jpg"></td>
<td>96.2</td>
<td>22.70</td>
<td>129,357,396,43,73,57,381,486,75,95</td>
</tr>
</tbody>
</table>
<p class="table-legend">Table columns are sterol names; molecular formula (MF) and molecular weight (MW; g/mol); molecular structures before and after derivatization; percentages of similarity to the NIST11 database; retention times; and major ions detected in the GC-MS analysis (m/z).</p>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The expected outcome of the protocol is the testing of the ability of <i>C. elegans</i> to utilize different sterols and the detection, identification, and quantification of <i>C. elegans</i> sterols. By growing <i>C. elegans</i> in sterol-defined cultures, the supplementation of the cultures with sterols of interest, and GC-MS analysis the conversion of dietary sterols to cholesterol can be determined <i>in vivo</i> (e.g., <a href="#fig6">Figure 6</a>). Moreover, this method opens an avenue into the study of sterol metabolism in the different physiological and developmental states of <i>C. elegans.</i> For example, this method facilitates the determination of the composition and level of different sterols in the dauer larval stage of <i>C. elegans</i> and other larval stages. Moreover, our protocol lays the foundation for the development of protocols for genetic and biochemical studies of sterol metabolism in other nematodes by GC-MS analysis.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/917-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Quantification of sterols from <i>C. elegans</i> cultures</div>
<p>Synchronized cultures that were grown with cholesterol, ergosterol, or stigmasterol were analyzed. Before the fractionation step, an internal standard was added to each sample. The different sterols, based on their peaks, were identified by the retention time and the patterns of their fragmentation were compared to the NIST11 database in order to validate sterol identity. The absolute quantity of each sterol was calculated by placing peak areas into the calibration curves equation (<a href="#tbl1">Table 1</a>) and the resulting value was then multiplied by the extraction efficacy. The “∗” sign indicates the concentration of analytes that did not fit the calibration curve range. In these cases, samples of analytes were diluted or concentrated until they fit into the calibration curve and the results were multiplied by the dilution/concentration factor. “DW” is the dry weight of worm pellets. Desmosterol and b-sitosterol that appear as standards in <a href="#fig3">Figure 3</a> are not detected because b-sitosterol is not part of the dietary sterols of the cultures and the levels of desmosterol in <i>C. elegans</i> are below the detection level of our instrument (<a href="#bib16">Shamsuzzama et al., 2020</a>).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Endpoint analysis without kinetics: GC-MS analysis is designed to determine the composition and levels of sterols at a specific endpoint of an experiment. In theory, it is possible to collect samples at various points in time to study the kinetics of the metabolism of sterols (<a href="#bib4">Dabrowski et al., 2020</a>). However, this approach does not support the direct analysis of metabolic flux, which can be studied by labeling sterols and their precursors by heavy atoms (<a href="#bib11">Mitsche et al., 2015</a>; <a href="#bib11">Mitsche et al., 2015</a>) which is beyond the scope of this protocol.</p>
<p>Efficiency of extraction: The efficiency of extraction can vary between samples and even with regards to different sterols in the same sample (<a href="#bib12">Ms et al., 2018</a>). Thus, the use of at least one internal standard and a mix of standards of the sterols of interest is critical for relative quantitative analysis of changes in sterol levels.</p>
<p>Derivatization: For GC-MS analysis, sterols must be derivatized to increase their volatility (<a href="#bib13">Prabhu et al., 2017</a>). Differences in the levels of derivatization of various sterols in the same sample or between samples can affect data quantification, interpretation, and reproducibility. The use of Liquid Chromatography-Mass Spectrometry (LC-MS) enables the analysis of sterols from biological samples without derivatization (<a href="#bib17">Skubic et al., 2020</a>, <a href="#bib14">Qiu et al., 2021</a>) however, the analysis of sterols by LC-MS has other limitations (<a href="#bib7">Jenner and Brown, 2017</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Contamination of the sterol-defined cultures (step 1 in the <a href="#step-by-step-method-details">step-by-step method details</a>) by environmental sterols. To verify that no background sterols contaminating the experiment a sterol-free culture of <i>C. elegans</i> must be established in every experiment. Sterol-deprived phenotypes such as the developmental arrest of the F2 progeny larvae in the control culture indicate the lack of sterols in the culture whereas the development of these larvae into adults indicates sterol contamination.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>All the components of the experimental system must be replaced. Special care should be taken not to use any vessel, medium, or material that may contain traces of sterols. A preliminary experiment to validate that the culture is sterol-free is a requisite step before conducting any experiment.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Unsynchronized cultures (step 2 in the <a href="#step-by-step-method-details">step-by-step method details</a>). The synchronization of cultures is important for the analysis of stage-specific sterol metabolism. One primary reason for unsynchronized cultures is inefficient bleaching, leaving carcasses or debris that can be a food source for the L1s.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Optimize the bleaching reaction to completely dissolve the bodies of the parental <i>C. elegans</i> nematodes.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Within each sample, variation in the levels of fractionation, derivatization, and detection may affect the absolute and relative quantification of the different sterols (steps 3–7 in the <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>To circumvent the above caveat, a mixture of known amounts of commercial pure standards (above 95% of purity) should be fractionated, derivatized, and analyzed the same way as the biological samples. The derivatization of standards is used to unveil the efficacy of fractionation and derivatization, as well as the level of detection of each sterol by GC-MS.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Between samples, differences in the levels of fractionation, derivatization, and detection of the same sterol (steps 3–7 in the <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>An internal standard should be added to each sample. Quantitative analysis of the area of peaks of the different sterols should be normalized to the peak area of the internal standard to normalize variations between experiments, similar to the analysis shown in <a href="#fig6">Figure 6</a>.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Lack of detection of sterol peaks in the GC-MS analysis (step 7 in the <a href="#step-by-step-method-details">step-by-step method details</a>). Because the GC-MS analysis is the endpoint of the experiment, a lack of signal can stem from the metabolization of the specific sterol, as well as from problems in extraction, derivatization, or GC-MS operation/detection. Thus, pinpointing the problem to a specific intermediate step is critical.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Different controls can pinpoint the step in which the problem arose:</p>
<ul>
<li>The use of internal sterol standards. Adding internal sterol standards, like coprostanol, to the samples not only normalizes the detection but also can reveal whether the extraction, derivatization or GC-MS operation/detection took place in a way that enables the detection of sterols in the samples.</li>
<li>The use of a mix of sterol standards as a positive control for the extraction, derivatization, and detection steps.</li>
</ul>
<p>If the internal standard and/or the mix of sterol standards are not properly detected/quantified, each of the steps of the procedure should be tested separately:</p>
<ul>
<li>Test GC-MS settings and detection by running known sterols standards that did not undergo fractionation.</li>
<li>Test derivatization by derivatizing commercial sterol standards and running a GC-MS analysis along with standards that do not require derivatization for detection (i.e., non-sterol standards that have a lower boiling point and higher thermal stability).</li>
<li>Test for fractionation efficiency. In case the above tests for GC-MS detection and derivatization efficacy did not identify the problem, the efficiency of fractionation can be tested by determining the levels of the sterol standards before and after fractionation.</li>
</ul>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Antebi, A. (2013). Steroid regulation of C. elegans diapause, developmental timing, and longevity. Curr. Top Dev. Biol. <i>105</i>, 181-212. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref1">View at publisher</a></p>
<p id="bib2">Capell-Hattam, I.M. and Brown, A.J. (2020). Sterol evolution: cholesterol synthesis in animals is less a required trait than an acquired taste. Curr. Biol. <i>30</i>, R886-R888. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref2">View at publisher</a></p>
<p id="bib3">Chitwood, D.J. (1999). Biochemistry and function of nematode steroids. Crit. Rev. Biochem. Mol. Biol. <i>34</i>, 273-284. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref3">View at publisher</a></p>
<p id="bib4">Dabrowski, R., Ripa, R., Latza, C., Annibal, A., and Antebi, A. (2020). Optimization of mass spectrometry settings for steroidomic analysis in young and old Killifish. Anal. Bioanal. Chem. <i>412</i>, 4089-4099. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref4">View at publisher</a></p>
<p id="bib5">Desmond, E. and Gribaldo, S. (2009). Phylogenomics of sterol synthesis: insights into the origin, evolution, and diversity of a key eukaryotic feature. Genome Biol. Evol. <i>1</i>, 364-381. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref5">View at publisher</a></p>
<p id="bib6">Heshof, R., Visscher, B., Promel, S., and Hughes, S. (2019). Large-scale cultivation of caenorhabditis elegans in a bioreactor using a labor-friendly fed-batch approach. Biotechniques <i>67</i>, 33-39. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref6">View at publisher</a></p>
<p id="bib7">Jenner, A.M. and Brown, S.H.J. (2017). Sterol analysis by quantitative mass spectrometry. Methods Mol. Biol. <i>1583</i>, 221-239. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref7">View at publisher</a></p>
<p id="bib8">Matyash, V., Entchev, E.V., Mende, F., Wilsch-Brauninger, M., Thiele, C., Schmidt, A.W., Knolker, H.J., Ward, S., and Kurzchalia, T.V. (2004). Sterol-derived hormone(S) controls entry into diapause in caenorhabditis elegans by consecutive activation of Daf-12 And Daf-16. PLoS Biol. <i>2</i>, 1561-1571. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref8">View at publisher</a></p>
<p id="bib9">Merris, M., Kraeft, J., Tint, G.S., and Lenard, J. (2004). Long-term effects of sterol depletion in C. elegans: Sterol content of synchronized wild-type and mutant populations. J. Lipid Res. <i>45</i>, 2044-2051. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref9">View at publisher</a></p>
<p id="bib10">Merris, M., Wadsworth, W.G., Khamrai, U., Bittman, R., Chitwood, D.J., and Lenard, J. (2003). Sterol effects and sites of sterol accumulation in caenorhabditis elegans: developmental requirement for 4alpha-methyl sterols. J. Lipid Res. <i>44</i>, 172-181. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref10">View at publisher</a></p>
<p id="bib11">Mitsche, M.A., Mcdonald, J.G., Hobbs, H.H., and Cohen, J.C. (2015). Flux analysis of cholesterol biosynthesis in vivo reveals multiple tissue and cell-type specific pathways. Elife <i>4</i>, E07999. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref11">View at publisher</a></p>
<p id="bib12">Ms, U., Ferdosh, S., Haque Akanda, M.J., Ghafoor, K., Rukshana, A.H., Ali, M.E., Kamaruzzaman, B.Y., Fauzi, M.B.,, Hadijah, H., Shaarani, S., and Islam Sarker, M.Z. (2018). Techniques for the extraction of phytosterols and their benefits in human health: a review. Sep. Sci. Technol. <i>53</i>, 2206-2223. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref12">View at publisher</a></p>
<p id="bib13">Prabhu, A.V., Luu, W., and Brown, A.J. (2017). Measuring activity of cholesterol synthesis enzymes using gas chromatography/mass spectrometry. Methods Mol. Biol. <i>1583</i>, 211-219. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref13">View at publisher</a></p>
<p id="bib14">Qiu, J., Li, T., and Zhu, Z.J. (2021). Multi-dimensional characterization and identification of sterols in untargeted Lc-Ms analysis using all ion fragmentation technology. Anal. Chim. Acta <i>1142</i>, 108-117. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref14">View at publisher</a></p>
<p id="bib15">Russell, J.C., Postupna, N., Golubeva, A., Keene, C.D., and Kaeberlein, M. (2020). Purification and analysis of caenorhabditis elegans extracellular vesicles. J. Vis. Exp. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref15">View at publisher</a></p>
<p id="bib16">Shamsuzzama, Lebedev, R., Trabelcy, B., Langier Goncalves, I., Gerchman, Y., and Sapir, A. (2020). Metabolic reconfiguration in C. elegans suggests a pathway for widespread sterol auxotrophy in the animal kingdom. Curr. Biol. <i>30</i>, 3031-3038 E7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref16">View at publisher</a></p>
<p id="bib17">Skubic, C., Vovk, I., Rozman, D., and Krizman, M. (2020). Simplified Lc-Ms method for analysis of sterols in biological samples. Molecules <i>25</i>, 4116. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref17">View at publisher</a></p>
<p id="bib18">Solis, G.M. and Petrascheck, M. (2011). Measuring Caenorhabditis elegans life span in 96 well microtiter plates. J. Vis. Exp. 2496. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref18">View at publisher</a></p>
<p id="bib19">Stiernagle, T. (2006). Maintenance of C. elegans (Wormbook). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref19">View at publisher</a></p>
<p id="bib20">Wei, J.H., Yin, X., and Welander, P.V. (2016). Sterol synthesis in diverse bacteria. Front. Microbiol. <i>7</i>, 990. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref20">View at publisher</a></p>
<p id="bib21">Yilmaz, L.S., Li, X., Nanda, S., Fox, B., Schroeder, F., and Walhout, A.J. (2020). Modeling tissue-relevant caenorhabditis elegans metabolism at network, pathway, reaction, and metabolite levels. Mol. Syst. Biol. <i>16</i>, E9649. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref21">View at publisher</a></p>
<p id="bib22">Zhou, W., Fisher, P.M., Vanderloop, B.H., Shen, Y., Shi, H., Maldonado, A.J., Leaver, D.J., and Nes, W.D. (2020). A nematode sterol C4alpha-methyltransferase catalyzes a new methylation reaction responsible for sterol diversity. J. Lipid Res. <i>61</i>, 192-204. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00417-2/sref22">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Amir Sapir (<a href="mailto:amir-s@sci.univ.ac.il">amir-s@sci.univ.ac.il</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code<i>.</i></p>
<h3>Acknowledgments</h3>
<p>Some strains were provided by the Caenorhabditis Genetics Center, which is funded by the <a href="https://doi.org/10.13039/100000002">National Institute of Health</a> Office of Research Infrastructure Programs (P40OD010440). This work was supported by an <a href="https://doi.org/10.13039/501100003977">Israel Science Foundation</a> grant (1747/16) to A.S.</p>
<h3>Author contributions</h3>
<p>Conceptualization, B.T., Y.G., and A.S.; methodology, B.T., Y.G., and A.S.; investigation, B.T. and A.S.; writing – original draft, B.T. and A.S.; writing – review &amp; editing, B.T., Y.G., and A.S.; funding acquisition, A.S.; resources, Y.G. and A.S.; supervision, Y.G. and A.S.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>