<article>
<section>
<h1 id="header">Rapid generation of murine haploid-induced trophoblast stem cells <i>via</i> a Tet-on system</h1>
<p><time datetime="2021-11-12">Published: November 12, 2021</time></p>
<p>Mei Xu,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Wenhao Zhang,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Mengyang Geng,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Yiding Zhao,<sup><a href="#aff1">1</a></sup> Shengyi Sun,<sup><a href="#aff1">1</a></sup> Qian Gao,<sup><a href="#aff1">1</a></sup> Yan Liu,<sup><a href="#aff3">3</a>,<a href="#cor1">*</a></sup> and Ling Shuai<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>State Key Laboratory of Medicinal Chemical Biology, College of Pharmacy, Tianjin Central Hospital of Gynecology Obstetrics/Tianjin Key Laboratory of Human Development and Reproductive Regulation, Nankai University, Tianjin 300350, China</p>
<p id="aff2"><sup>2</sup>National Clinical Research Center for Obstetrics and Gynecology, Peking University Third Hospital, Beijing 100191, China</p>
<p id="aff3"><sup>3</sup>Department of Obstetrics, Tianjin First Central Hospital, Nankai University, Tianjin 300192, China</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:30819007@nankai.edu.cn">30819007@nankai.edu.cn</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:lshuai@nankai.edu.cn">lshuai@nankai.edu.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100881">10.1016/j.xpro.2021.100881</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Haploid trophoblast stem cells (TSCs) are advanced in studying placental development for their placental precursor and homozygous features. Here, we describe how to generate haploid-induced TSCs (haiTSCs) from haploid embryonic stem cells with a Tet-on system. Our haiTSCs can maintain haploidy long-term and can produce genome-wide mutants combined with transposons. It is promising in high-throughput genetic screening of trophoblast-specific modulators.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref10">Peng et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for inducing a <i>Cdx2</i>-OE Tet-on system into haploid ESCs</li>
<li>Protocol for conversion and purification of haploid induced TSCs from haploid ESCs</li>
<li>Protocol for construction of genome-wide mutated homozygous TSCs by <i>piggyBac</i> transposon</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 10 days</div>
<h3 id="sec1.1">Preparation of mitotically inactivated mouse embryonic fibroblasts (MEFs)</h3>
<p>The methods in this section were modified according to a previous protocol (<a href="#bib18">Zhao et al., 2010</a>).</p>
<ol>
<li>Mate CD1 mice, and sacrifice the pregnant female mice on embryonic day 12.5 (E12.5).</li>
<li>Dissect out the E12.5 embryos into sterile PBS, and mince the remaining tissue after removing the brain, internal organs, limps and tail of each embryo.</li>
<li>Dissociate the minced tissue with 0.05% (vol/vol) trypsin-EDTA in an incubator for 15 min at 37°C.</li>
<li>Stop the trypsin action with fresh MEF medium, centrifuge the cells in a 50-mL tube at 300 × g for 5 min at 20°C–25°C, subsequently discard the supernatant, and resuspend the cells in fresh MEF medium.</li>
<li>Plate the cells (per embryo) on a 100-mm TC dish, and culture them at 5% CO<sub>2</sub> and 37°C with medium changed every day.</li>
<li>Passage the cells with 0.05% (vol/vol) trypsin-EDTA when the cells reach 90–95% confluency.</li>
<li>Inactivate the fully grown MEFs at passage 2 or 3 with 10 μg/mL mitomycin-C for 2 h at 37°C to produce feeder cells.</li>
<li>Trypsinize the feeder cells with 0.05% (vol/vol) trypsin-EDTA, repeat step 4, and freeze the feeder cells in liquid nitrogen. Approximately 2 × 10<sup>6</sup> feeder cells can be stored per freezing vial in liquid nitrogen for up to 1 year.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Carry out all dissection steps in a tissue culture hood under sterile conditions.</div>
<h3 id="sec1.2">Derivation and identification of <i>p53</i>-KO haESCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 weeks</div>
<p>Mouse haESCs are prone to revert back to a diploid state in daily culture or during differentiation, thus requiring periodic and complicated fluorescence-activated cell sorting (FACS) for haploid enrichment (<a href="#bib13">Sun et al., 2020</a>). Many protocols, including adding inhibitors (<a href="#bib5">He et al., 2017</a>) or manipulating gene expression (<a href="#bib4">He et al., 2018</a>; <a href="#bib9">Olbrich et al., 2017</a>), have been implemented to reduce diploidization. Disruption of <i>p53</i> can stabilize the haploidy of haESCs and is beneficial for generating haploid lines of other cell types (<a href="#bib3">Gao et al., 2021</a>; <a href="#bib10">Peng et al., 2019</a>). Therefore, we choose <i>p53</i>-KO haESCs in this protocol for their advantage of haploidy maintenance.</p>
<ol start="9">
<li>Activate <i>p53</i>-KO oocytes with activation medium (the steps of superovulation and the recipe of activation medium refer to our previous protocol (<a href="#bib12">Shuai et al., 2014</a>)), and further culture the activated haploid embryos in KSOM at 5% CO<sub>2</sub> and 37°C until they reach the morula stage.</li>
<li>Plate one haploid morula per well of a 24-well TC plate coated with feeder cells in ESC medium at 5% CO<sub>2</sub> and 37°C for 5–6 days to form outgrowths (defined as passage 0 (P0)).</li>
<li>Wash the well-expanded outgrowths with PBS, and incubate them with 0.05% (vol/vol) trypsin-EDTA for 5 min in an incubator.</li>
<li>Pipet the outgrowths into small clumps, and plate them in fresh ESC medium on feeder cells (defined as P1).</li>
<li>Culture and passage the haESCs until the cell number (∼10<sup>6</sup>) is sufficient for cell sorting (10–14 days). The details for cell sorting are described in the “<a href="#sec3.5">expanding and sorting for haploid cells</a>” section.</li>
<li>Extract genomic DNA from <i>p53</i>-KO haESCs, WT-haESCs, <i>p53</i>−/− mice and <i>p53</i>+/- mice with the E.Z.N.A.® Tissue DNA Kit according to the manufacturer’s instructions (<a href="https://www.omegabiotek.com/product/e-z-n-a-tissue-dna-kit/?v=400b9db48e62&amp;cn-reloaded=1">https://www.omegabiotek.com/product/e-z-n-a-tissue-dna-kit/?v=400b9db48e62&amp;cn-reloaded=1</a>), and dissolve the DNA in 50 μL of nuclease-free water per sample.</li>
<li>Set up a 20 μL PCR system (see Table below) in a sterile 200 μL PCR tube. Primer pairs for <i>p53</i>-KO identification are shown (<a href="#fig1">Figure 1</a>A and “<a href="#key-resources-table">key resources table</a>”).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Characterization of <i>p53</i>-KO haESCs</div>
<p>(A) The strategy of <i>p53</i> deletion is provided by the manufacturer (<a href="https://www.biocytogen.com.cn/models/disease/B-p53-KO-mice.html">https://www.biocytogen.com.cn/models/disease/B-p53-KO-mice.html</a>). A PGK promoter and a <i>Neo</i><sup><i>r</i></sup> cassette are inserted into the <i>p53</i> gene (between exon 2 and exon 7). PCR primers were designed for identifying the <i>p53</i>-KO genotype. F/R1, for WT genotype; F/R2, for <i>p53</i>-KO genotype.</p>
<p>(B) <i>p53</i>-KO haESC genotyping verification by PCR using primer pairs F/R1 and F/R2, with WT-haESCs, <i>p53</i>+/- tissue and <i>p53</i>−/− tissue as controls.</p>
<p>(C) DNA content analysis of haESCs after 14 days without sorting. The percentage of the 1n peak in <i>p53</i>-KO haESCs decreased from 25.8% to 22.8%, whereas that in WT-haESCs decreased from 25.3% to 7.6%.</p>
<p>(D) Summary of cell information of <i>p53</i>-KO haESCs and WT-haESCs in a 14-day interval after sorting for haploids.</p>
</figcaption>
</figure>
</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th colspan="3">PCR system</th>
</tr>
<tr>
<th>Reaction component</th>
<th>Volume (μl)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>7</td>
<td>-</td>
</tr>
<tr>
<td>2x Taq Plus Master Mix</td>
<td>10</td>
<td>1 ×</td>
</tr>
<tr>
<td>10 μM forward primer for <i>p53</i>-KO identification</td>
<td>0.5</td>
<td>0.25 μM</td>
</tr>
<tr>
<td>10 μM reverse primer for <i>p53</i>-KO identification</td>
<td>0.5</td>
<td>0.25 μM</td>
</tr>
<tr>
<td>100–200 ng/μL</td>
<td>2.0</td>
<td>10–20 ng/μL</td>
</tr>
</tbody>
</table>
<ol start="16">
<li>Run the PCR cycler parameters according to the table below. The genotype of <i>p53-KO</i> haESCs is the same as that of <i>p53</i>−/− mice.</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>94°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">25–35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>62°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<ol start="17">
<li>Load 20 μL of PCR products onto a 1% (wt/vol) agarose gel in 1 × TAE for electrophoresis to check the PCR products (the total time of PCR experiment is 1 day), and confirm the genotypes. The genotype of <i>p53-KO</i> haESCs is the same as that of <i>p53</i>−/− mice.</li>
</ol>
<h3 id="sec1.3">Preparation of the conditioned medium</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="18">
<li>Thaw several vials of feeder cells (∼5 × 10<sup>7</sup> totally) quickly in a 37°C water bath.</li>
<li>Resuspend the cells with MEF medium in 15-mL tubes, and centrifuge at 200 × g for 3 min.</li>
<li>Discard the supernatant, and resuspend the cell pellets with TSC basic medium. Seed 2 × 10<sup>6</sup> cells per 100-mm TC-dish with 12–15 mL TSC basic medium.</li>
<li>Culture the cells for 3 days in an incubator without changing the medium.</li>
<li>Collect the medium in 50-mL tubes, and centrifuge at 2000 × g at 4°C for 20 min to remove debris.</li>
<li>Collect the supernatants, filter them with a 0.45-μm filter, and store them at –20°C in 35 mL aliquots for up to 6 months.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> You should balance the samples to ensure the safety.</div>
<div class="critical">
<span class="critical-title">Critical:</span> You can collect another batch of conditioned medium with the same feeder cells after 3 more days, but the first collection provides the best results. Thaw each aliquot as needed and store up to 1 month at 4°C, and do not refreeze.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-EOMES (1:500)</td>
<td>Abcam</td>
<td>Cat#ab23345; RRID: AB_778267</td>
</tr>
<tr>
<td>Anti-CDX2 (1:500)</td>
<td>Abcam</td>
<td>Cat#ab76541; RRID: AB_1523334</td>
</tr>
<tr>
<td>Anti-TPBPA (1 μg/mL)</td>
<td>Abcam</td>
<td>Cat#ab104401; RRID: AB_10901888</td>
</tr>
<tr>
<td>Anti-TFAP2C (1:100)</td>
<td>Santa</td>
<td>Cat#SC12762; RRID: AB_667770</td>
</tr>
<tr>
<td>Anti-CDCP1(10 μg/mL)</td>
<td>R&amp;D Systems</td>
<td>Cat#AF4515; RRID: AB_2078800s</td>
</tr>
<tr>
<td>Alexa Fluor 488 Donkey Anti-Goat (1:1000)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A-11055; RRID: AB_2534102</td>
</tr>
<tr>
<td>Cy3 Goat Anti-Rabbit IgG (H+L) (1:200)</td>
<td>Abclonal</td>
<td>Cat#AS007; RRID: AB_2769089</td>
</tr>
<tr>
<td>FITC Goat Anti-Mouse IgG (H+L) (1:200)</td>
<td>Abclonal</td>
<td>Cat#AS001; RRID: AB_2769475</td>
</tr>
<tr>
<td>FITC Goat Anti-Rabbit IgG (H+L) (1:200)</td>
<td>Abclonal</td>
<td>Cat#AS011; RRID: AB_2769476</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM/F-12</td>
<td>Thermo Fisher Scientific</td>
<td>11320033</td>
</tr>
<tr>
<td>DMEM</td>
<td>Thermo Fisher Scientific</td>
<td>12800017</td>
</tr>
<tr>
<td>Knockout serum replacement</td>
<td>Thermo Fisher Scientific</td>
<td>A3181502</td>
</tr>
<tr>
<td>RPMI 1640 Medium</td>
<td>Thermo Fisher Scientific</td>
<td>61870036</td>
</tr>
<tr>
<td>GlutaMAX™ Supplement</td>
<td>Thermo Fisher Scientific</td>
<td>35050061</td>
</tr>
<tr>
<td>MEM Non-Essential Amino Acids Solution (100X) (NEAA)</td>
<td>Thermo Fisher Scientific</td>
<td>11140050</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>15140122</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.05%), phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>25300062</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.25%), phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>25300072</td>
</tr>
<tr>
<td>Hoechst 33342</td>
<td>Thermo Fisher Scientific</td>
<td>H3570</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Thermo Fisher Scientific</td>
<td>21985023</td>
</tr>
<tr>
<td>Transfection kit</td>
<td>Thermo Fisher Scientific</td>
<td>MPK10096</td>
</tr>
<tr>
<td>Phosphate buffered saline (PBS)</td>
<td>Sigma-Aldrich</td>
<td>D8537</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Sigma-Aldrich</td>
<td>D2650</td>
</tr>
<tr>
<td>Bovine Serum Albumin (BSA)</td>
<td>Sigma-Aldrich</td>
<td>A1933</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>Sigma-Aldrich</td>
<td>P4562</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>T8787</td>
</tr>
<tr>
<td>Gelatin</td>
<td>Sigma-Aldrich</td>
<td>V900863</td>
</tr>
<tr>
<td>Paraformaldehyde (PFA)</td>
<td>Sigma-Aldrich</td>
<td>158127</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Biological Industries</td>
<td>04-002-1A</td>
</tr>
<tr>
<td>PD0325901</td>
<td>MCE</td>
<td>HY-10254</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>MCE</td>
<td>HY-10182</td>
</tr>
<tr>
<td>Doxycycline</td>
<td>MCE</td>
<td>HY-N0565B</td>
</tr>
<tr>
<td>Y27632</td>
<td>MCE</td>
<td>HY-10071</td>
</tr>
<tr>
<td>Mitomycin-C</td>
<td>MCE</td>
<td>HY-13316</td>
</tr>
<tr>
<td>Recombinant Human FGF-4</td>
<td>Peprotech</td>
<td>100-31</td>
</tr>
<tr>
<td>E.Z.N.A.® Tissue DNA Kit</td>
<td>Omega</td>
<td>D3396</td>
</tr>
<tr>
<td>KSOM medium</td>
<td>Millipore</td>
<td>MR-020P-5F</td>
</tr>
<tr>
<td>Agarose Gel</td>
<td>Biowest</td>
<td>111860</td>
</tr>
<tr>
<td>TAE Buffer (50×)</td>
<td>Solarbio</td>
<td>T1060</td>
</tr>
<tr>
<td>Fibronectin</td>
<td>Millipore</td>
<td>ECM001</td>
</tr>
<tr>
<td>Hygromycin B</td>
<td>Millipore</td>
<td>400050</td>
</tr>
<tr>
<td>LIF Protein, Mouse, Recombinant</td>
<td>Millipore</td>
<td>ESG1107</td>
</tr>
<tr>
<td>Heparin</td>
<td>Millipore</td>
<td>375095</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>
<i>p53</i>-KO haESCs</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>WT-haESCs</td>
<td>(<a href="#bib2">Gao et al., 2018</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: CD1</td>
<td>Vital River Laboratories</td>
<td>N/A</td>
</tr>
<tr>
<td>Mouse: <i>p53</i>-KO: 129Sv/Jae strain</td>
<td>Biocytogen</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Primers for Inverse PCR and qPCR, see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Mmc1.extension">Table S1</a>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: <i>p53</i>-KO identification Forward: AGTTCTGCCACGTGGTTGGT</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: <i>p53</i>-KO identification Reverse1: GTCTCCTGGCTCAGAGGGAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: <i>p53</i>-KO identification Reverse2: CAGAGGCCACTTGTGTAGCG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: <i>Cdx2</i>-OE identification T2A+Puro-F: GGTGACGTGGAGGAGAATCC</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: <i>Cdx2</i>-OE identification T2A+Puro-R: CTCGTAGAAGGGGAGGTTGC</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: <i>Cdx2</i>-OE identification rtTA-F: GGCGAAGAACTCCAGCATGAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: <i>Cdx2</i>-OE identification rtTA-R: CAGCGTCTCCGACCTGATG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>FUW-M2 rtTA vector</td>
<td>This paper</td>
<td>Addgene Plasmid #20342</td>
</tr>
<tr>
<td>2C::tdTomato reporter vector</td>
<td>This paper</td>
<td>Addgene Plasmid #40281</td>
</tr>
<tr>
<td>PB dual promoter vector</td>
<td>This paper</td>
<td>SBI Plasmid #PB513B</td>
</tr>
<tr>
<td>PBase vector</td>
<td>This paper</td>
<td>SBI Plasmid # PB210PA-1</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Costar tissue culture plates, 6-well,</td>
<td>Corning</td>
<td>3516</td>
</tr>
<tr>
<td>Costar tissue culture plates, 24-well,</td>
<td>Corning</td>
<td>3524</td>
</tr>
<tr>
<td>100-mm tissue culture dish</td>
<td>Corning</td>
<td>430167</td>
</tr>
<tr>
<td>0.22-μm sterile syringe filter</td>
<td>Millipore</td>
<td>SLGP033RK</td>
</tr>
<tr>
<td>0.45-μm sterile syringe filter</td>
<td>Millipore</td>
<td>SLHV033RS</td>
</tr>
<tr>
<td>1.5-mL Eppendorf tube</td>
<td>Axygen</td>
<td>MCT-150-C</td>
</tr>
<tr>
<td>Round Cover Slips</td>
<td>Electron Microscopy Sciences</td>
<td>72196-12</td>
</tr>
<tr>
<td>200-μL PCR tube</td>
<td>NEST</td>
<td>401001</td>
</tr>
<tr>
<td>Sterile 15-mL tube</td>
<td>Corning</td>
<td>430790</td>
</tr>
<tr>
<td>Sterile 50-mL tube</td>
<td>Corning</td>
<td>430828</td>
</tr>
<tr>
<td>5-mL BD tube</td>
<td>BD</td>
<td>352063</td>
</tr>
<tr>
<td>40-μm cell strainer</td>
<td>BD</td>
<td>352340</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<ul>
<li>
<b>“2i” solution stock.</b> The “2i” stock solution described here has been previously published (<a href="#bib17">Ying et al., 2008</a>). Prepare a stock (100 mM) solution of PD0325901 by diluting 10 mg of PD0325901 in 207 μL of DMSO; prepare a stock (50 mM) solution of CHIR99021 by diluting 50 mg of CHIR99021 in 2.15 mL of DMSO. Prepare aliquots in sterile 1.5-mL Eppendorf tubes, and store them at −80°C for up to 6 months. Do not refreeze after thawing.</li>
<li>
<b>FGF-4 solution stock.</b> Prepare a 4,000× stock (0.1 mg/mL) solution by diluting FGF-4 in 1 × PBS containing 0.1% (wt/vol) BSA (the details including source and catalog number are listed in the “<a href="#key-resources-table">key resources table</a>” above). Prepare aliquots in sterile 1.5-mL Eppendorf tubes, and store them at −80°C for up to 8 weeks. Do not refreeze after thawing.</li>
<li>
<b>Heparin solution stock.</b> Prepare a 2,0000 × stock (20 mg/mL) solution (wt/vol) of heparin in 1 × PBS. Prepare aliquots in sterile 1.5-mL Eppendorf tubes, and store them at −80°C for up to 6 months. Do not refreeze after thawing.</li>
<li>
<b>Sodium pyruvate solution stock.</b> Prepare a 100 × stock (100 mM) solution by diluting 100 mg sodium pyruvate in 9 mL of 1 × PBS. Prepare aliquots in sterile 1.5-mL Eppendorf tubes, and store them at −80°C for up to 6 months. Do not refreeze after thawing.</li>
<li>
<b>Gelatin solution.</b> Prepare a 0.1% (wt/vol) gelatin solution by dissolving 1 g of gelatin in 100 mL of water at 100°C. Store at 4°C for up to 1 month.</li>
<li>
<b>MEF medium.</b> Prepare MEF medium (<a href="#bib18">Zhao et al., 2010</a>) refer to the following table. Store at 4°C and use it within 1 month.</li>
</ul>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>89 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Penicillin–streptomycin</td>
<td>1 ×</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<ul>
<li>
<b>ESC medium.</b> Prepare ESC medium (<a href="#bib2">Gao et al., 2018</a>) refer to the following table. Store at 4°C, and use it within 1 week.</li>
</ul>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM/F12</td>
<td>n/a</td>
<td>81 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>7%</td>
<td>7 mL</td>
</tr>
<tr>
<td>Knockout serum replacement</td>
<td>10%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Glutamax</td>
<td>1 ×</td>
<td>1 mL</td>
</tr>
<tr>
<td>2-mercaptoethanol</td>
<td>0.1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Penicillin–streptomycin</td>
<td>1 ×</td>
<td>1 mL</td>
</tr>
<tr>
<td>PD0325901</td>
<td>1 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>3 μM</td>
<td>6 μL</td>
</tr>
<tr>
<td>LIF</td>
<td>1000 U/mL LIF</td>
<td>10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<ul>
<li>
<b>TSC basic medium.</b> Prepare a medium refer to the following table. Store at 4°C, and use it within 1 month.</li>
</ul>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI 1640</td>
<td>n/a</td>
<td>77 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>20%</td>
<td>20 mL</td>
</tr>
<tr>
<td>Glutamax</td>
<td>1 ×</td>
<td>1 mL</td>
</tr>
<tr>
<td>2-mercaptoethanol</td>
<td>0.1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>Penicillin–streptomycin</td>
<td>1 ×</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<ul>
<li>
<b>TSC medium.</b> Prepare a medium refer to the following table. Store at 4°C, and use it within 1 week.</li>
</ul>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TSC basic medium</td>
<td>30%</td>
<td>30 mL</td>
</tr>
<tr>
<td>Conditioned medium</td>
<td>70%</td>
<td>70 mL</td>
</tr>
<tr>
<td>FGF-4</td>
<td>25 ng/mL</td>
<td>25 μL</td>
</tr>
<tr>
<td>Heparin</td>
<td>1 μg/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<ul>
<li>
<b>4% (vol/vol) PFA solution.</b> Dissolve 2 g of PFA powder in 50 mL of PBS at 65°C to prepare 4% (vol/vol) PFA. Adjust the pH to 7.2 and store at -20°C.</li>
<li>
<b>Blocking buffer for antibody incubations.</b> Prepare a solution of PBS with 2% (wt/vol) BSA. Store at 4°C, and use within 1 week.</li>
<li>
<b>Mouse CDCP1 antibody solution.</b> Reconstitute the antibody stock (0.2 mg/mL) in sterile PBS. Prepare aliquots in sterile 1.5-mL Eppendorf tubes, and store them at −20°C for up to 6 months. Do not refreeze after thawing.</li>
<li>
<b><i>Cdx2</i> overexpression vector.</b> The <i>Cdx2</i> overexpression vector is designed to express the <i>Cdx2</i>-coding region sequence (CDS) driven by a TRE-mini CMV promoter. A self-cleaving peptide T2A sequence is inserted between the CDS and puromycin resistance gene (<i>Puro</i><sup><i>r</i></sup>). The sequence of <i>Cdx2</i>-CDS is amplified from the cDNA of mouse TSCs cells by PCR (primers, F: GGATCCATGTACGTGAGCTACCTTCTG; R: ACGCGTCTGGGTGACAGTGGAGTT). <i>Cdx2</i>-CDS is ligated into a PB-TRE-<i>Puro</i><sup><i>r</i></sup> vector at MluI and BamHI sites (<a href="#fig2">Figure 2</a>A). More details on the cloning are described in our previous report (<a href="#bib10">Peng et al., 2019</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Derivation of <i>p53</i>-KO haESCs carrying the <i>Cdx2</i> overexpression Tet-on system</div>
<p>(A) Schematic overview of the plasmids to deliver the <i>Cdx2</i> Tet-on system.</p>
<p>(B) Genotyping of <i>p53</i>-KO haESCs carrying the <i>Cdx2</i> overexpression Tet-on system, with plasmids and WT-haESCs as controls.</p>
<p>(C) The expression levels of <i>Cdx2</i> in haESCs treated with different concentrations of Dox (1 μg/mL, 2 μg/mL, 4 μg/mL and 8 μg/mL) on day 3. t test, ∗∗∗p &lt;0.001. Data are represented as the mean ± SEM. The expression levels of <i>Cdx2</i> increased significantly in the Dox-supplemented groups compared with the nonsupplemented group. However, there were no obvious differences between the Dox-supplemented groups.</p>
<p>(D) Assessment of the viability of haESCs carrying a <i>Cdx2</i> overexpression Tet-on system after treatment with different concentrations of Dox in the medium for 3 days. Scale bar, 100 μm. The percentage of dead cells in the cultures increased when the concentration of Dox increased.</p>
</figcaption>
</figure>
</li>
<li>
<b>PB-rtTA-Hygr vector.</b> For the PB-rtTA-<i>Hyg</i><sup><i>r</i></sup> vector, the reverse tetracycline transitional activator (rtTA), T2A and hygromycin resistance gene (<i>Hyg</i><sup><i>r</i></sup>) are linked to a PB vector driven by a Ubc promoter. The Ubcp-rtTA sequence is amplified from an FUW-M2 rtTA vector (Addgene, cat. no. 20342 (<a href="#bib6">Hockemeyer et al., 2008</a>)) (primers, F: CTCCAGATCTGGCCTCCGC; R: GTTAGCAGACTTCCTCTGCCCTCCCCGGGGAGCATGTCAAG), and <i>Hyg</i><sup><i>r</i></sup> is amplified from a 2C::tdTomato reporter vector (Addgene, cat. no. 40281 (<a href="#bib7">Macfarlan et al., 2012</a>)) (primers, F: GAGGGCAGAGGAAGTCTGCTAACATGCGGTGACGTCGAGGAGAATCCTGGCCCAATGAAAAAGCCTGAACTCACCGC; R: ACCGGTTAACGGCGAAGAACTCCAGCATGAG). Ubcp-rtTA and <i>Hyg</i><sup><i>r</i></sup> are linked by overlapping PCR. The whole cassette is ligated to a PB dual promoter vector (SBI, cat. no. PB513B) at NotI and HpaI sites (<a href="#fig2">Figure 2</a>A).</li>
<li>
<b>PB-SA gene-trapping vector.</b> For the PB-SA gene-trapping vector, the oligo containing the SA sequence is annealed to form sticky end fragments comprising SpeI and NotI sites. This fragment is ligated to a PB dual promoter vector (SBI, cat. no. PB513B) at NotI and SpeI sites.</li>
<li>
<b>PBase vector.</b> PBase vector (SBI, cat. no. PB210PA-1) is purchased from a local agent without modifications, which enabled insertion of the PB-SA gene-trapping vector into the genome.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">
<i>P53</i>-KO haESCs daily culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol>
<li>Seed each well of a 6-well TC-plate with ∼1.0 × 10<sup>5</sup> feeder cells in MEF medium for 12 h.</li>
<li>Thaw a vial of frozen <i>p53</i>-KO haESCs as quickly as possible, and resuspend them in ESC medium.</li>
<li>Replace the MEF medium with ESC medium each well, and seed the <i>p53</i>-KO haESCs. Typically, ∼5.0 × 10<sup>5</sup> haESCs can be plated per well of a 6-well TC plate.</li>
<li>Culture the cells at 5% CO<sub>2</sub> and 37°C, and change the medium every day. Healthy haESC colonies appear as domed and bright and present with clear edges.</li>
<li>Passage the haESCs every other day or when the cells reach 90% confluency. To passage the cells, incubate them with 0.05% (vol/vol) trypsin-EDTA for 3 min at 37°C, add two volumes of MEF medium to stop trypsinizing, and mix the cell suspension gently. Centrifuge at 200 × g for 3 min, discard the supernatant, resuspend the cell pellet, and plate them on a fresh feeder-coated well.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The cell lines used in your research should be regularly checked for mycoplasma contamination.</div>
<div class="critical">
<span class="critical-title">Critical:</span> <i>p53</i>-KO haESCs also need regular sorting for haploid enrichment; thus, the interval period of sorting should not exceed 3 weeks.</div>
<h3 id="sec3.2">Construction of a <i>Cdx2</i> overexpression Tet-on system in <i>p53</i>-KO haESCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 days</div>
<p><i>Cdx2</i> plays important roles in trophectoderm specification, and its overexpression can induce trophectoderm fate in cell cultures of mouse ESCs (<a href="#bib8">Niwa et al., 2005</a>). The doxycycline (Dox)-inducible Tet-on system is a widely used system to conditionally regulate the expression of desired genes (<a href="#bib15">Woltjen et al., 2009</a>).</p>
<ol start="6">
<li>Prepare plasmids mixture in R buffer. For example, ∼2 × 10<sup>6</sup> cells are mixed with 3 μg of <i>Cdx2</i> overexpression vector, 3 μg of PB-rtTA-<i>Hyg</i><sup><i>r</i></sup> plasmid and 1 μg of PBase plasmid in 100 μL of R buffer.</li>
<li>Wash the cells with PBS, dissociate them using 0.05% (vol/vol) trypsin-EDTA for 3 min at 37°C, and halt the reaction by adding two volumes of MEF medium.</li>
<li>Transfer the cell suspensions into 15-mL tubes, and count viable cells with a cell counter.</li>
<li>Centrifuge the cells at 200 × g for 3 min, and discard the supernatant. Resuspend the cells with the mixture comprising the plasmids and R buffer.</li>
<li>Perform transfection with an electroporator with the transfection kit using the following parameters: 1400 V; 10 ms; 3 pulses.</li>
<li>Plate the electroporated cells back onto dishes precoated with fibronectin, and add ESC medium. Each well of a 6-well TC plate is coated with 1 mL fibronectin (final, 16.7 mg/mL) at 37°C for 12 h.</li>
<li>Add hygromycin (final concentration: 150 μg/mL) into the medium 2–3 days after electroporation for selection. Replace the medium every day, and culture with hygromycin for 7 days.</li>
<li>Passage the surviving cells according to step 5, and seed the cells in feeder-cell-coated dishes with ESC medium, these haESCs should be referred to as <i>Cdx2</i> overexpression (<i>Cdx2</i>-OE) Tet-on haESCs.</li>
<li>Genotype for <i>Cdx2</i>-OE Tet-on haESCs as indicted in “Before you begin”, steps 14–17 (<a href="#fig2">Figure 2</a>B). Primer pairs for <i>Cdx2</i>-OE Tet-on identification are shown in the “<a href="#key-resources-table">key resources table</a>”.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Nonelectroporated cells must be prepared and treated with hygromycin as a negative control. Hygromycin must be removed from the cultures in advance once all the control cells are dead.</div>
<div class="critical">
<span class="critical-title">Critical:</span> You should sort for haploid cells after electroporation and selection for hygromycin because the cells undergo severe cell death and self-diploidization during these procedures. A high proportion of haploid cells in the initial <i>Cdx2</i>-OE Tet-on haESC population is essential for the successful establishment of derivation of haiTSCs.</div>
<h3 id="sec3.3">Induction of TSCs from <i>Cdx2</i>-OE Tet-on haESCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days</div>
<ol start="15">
<li>Add 1 mL of 0.1% (wt/vol) gelatin per well to a 6-well TC plate, and incubate at 37°C for 12 h.</li>
<li>Select newly haploidy enriched <i>Cdx2</i>-OE Tet-on haESCs for induction, dissociate the cells with 0.25% (vol/vol) trypsin-EDTA at 37°C for 3 min to ensure the generation of single cells, and halt the reaction with MEF medium.</li>
<li>Centrifuge the cells in 15-mL tubes at 200 × g for 3 min, and discard the supernatant.</li>
<li>Resuspend the cells and plate them in gelatin-coated wells with TSC medium containing 1 μg/mL Dox. Typically, ∼2.0 × 10<sup>5</sup> cells are seeded per well of a 6-well TC plate.</li>
<li>Culture the cells at 5% CO<sub>2</sub> and 37°C, and change the medium every day. Approximately 3–4 days later, a few typical TSC colonies begin to appear in the cell cultures instead of domed ESC colonies (<a href="#fig3">Figure 3</a>B).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Conversion of haESCs into haiTSCs <i>in vitro</i>
</div>
<p>(A) Schematic overview of the derivation of haiTSCs from haESCs via the <i>Cdx2</i> Tet-on system <i>in vitro</i>.</p>
<p>(B) Representative phase contrast images showing the chronological morphology change from day 0 (left) to the 1st sorting (middle) and the 2nd sorting for CDCP1 positivity (right). Scale bar, 100 μm.</p>
<p>(C) The percentage of CDCP1-positive cells in the 1st sorting (33.5%) and the 2nd sorting (82.6%) for CDCP1-positive cells, with haESCs as a negative control.</p>
<p>(D) Schematic showing the sorting of haploid cells from cultured haiTSCs. R1 is a gate for live cells; R2 is a gate to eliminate debris and doublets or clumps; and the 1n peak represents haploid cells from the cell cultures.</p>
</figcaption>
</figure>
</li>
<li>Passage the confluent cell cultures according to steps 16–18. The suggested splitting ratio is 1:2.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> An appropriate concentration of Dox is important because a high concentration of Dox is toxic and affects cell viability. A concentration of 1 μg/ml is suggested according to our experiments (<a href="#fig2">Figures 2</a>C and 2D, <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Mmc1.pdf">Table S1</a>).</div>
<h3 id="sec3.4">Sorting for CDCP1-positive cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>CDCP1 is a specific marker of mouse TSCs, the antibody of which is extensively used for TSC enrichment (<a href="#bib11">Rugg-Gunn et al., 2012</a>).</p>
<ol start="21">
<li>Dissociate the cell cultures to a single-cell suspension according to steps 16 to 17, and incubate the cells with anti-CDCP1 primary antibody (10 μg/mL final concentration; diluted in blocking buffer) at 4°C for 30 min. Two hundred microliters of anti-CDCP1 solution are usually suggested for ∼5 × 10<sup>6</sup> cells.</li>
<li>Centrifuge the cells at 300 × g for 3 min, discard the supernatant, wash each sample with 3 mL of PBS, and centrifuge again.</li>
<li>Discard the supernatant, and add 1 mL of diluted Alexa Fluor 488 secondary antibody. Gently mix, and incubate at 4°C for 30 min.</li>
<li>Wash each sample with 3 mL of PBS according to step 22. Discard the supernatant, and resuspend cells in 0.5 mL of TSC medium.</li>
<li>Filter cells of each sample through a 40-μm cell strainer into a new 5-mL BD tube to remove cell clumps, and prepare a control sample in parallel.</li>
<li>Add 1 mL of fresh TSC medium in a new 5-mL BD tube as a harvesting tube, and run the samples on an EQ cell sorter.</li>
<li>Transfer the sorted cells from the harvesting tubes into 15-mL tubes. Centrifuge at 300 × g for 3 min, and resuspend the cells in TSC medium.</li>
<li>Plate the cells in gelatin-coated 24-well TC plates with TSC medium. Usually, ∼1 × 10<sup>5</sup> cells are seeded per well of a 24-well TC plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To accurately gate CDCP1-positive cells, wild-type (WT) ESCs were used in this procedure as a negative control (<a href="#fig3">Figure 3</a>C).</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is normal to lose cells during the centrifugation process. To ensure that a sufficient number of cells are sorted, the initial cell counts should be increased to ∼1 × 10<sup>7</sup>.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If the harvested cell amounts are low, the harvested cells should be directly plated onto gelatin-coated wells of a 24-well TC-plate to avoid losing cells.</div>
<h3 id="sec3.5">Expanding and sorting for haploid cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days</div>
<ol start="29">
<li>Expand the CDCP1-positive cells in TSC medium for 5–7 days to reach cell numbers sufficient for haploid cell sorting, and passage the cells with a splitting ratio of 1:4 to 1:6 when necessary during this period.</li>
<li>Prepare Hoechst 33342 staining buffer (working solution: 5 μg/mL) diluted in DMEM/F12 medium.</li>
<li>Dissociate the cell cultures to single-cell suspensions according to steps 16 to 17.</li>
<li>Discard the supernatant, add the staining buffer, and mix gently. Usually, 3 mL staining buffer is added per one million cells.</li>
<li>Incubate the mixture in a 37°C water bath for 30 min in the dark with gentle shaking every 5 min.</li>
<li>Centrifuge at 200 × g for 3 min, and discard the supernatant. Resuspend the cells in 0.5 mL of TSC medium.</li>
<li>Prepare the cell samples according to steps 25 to 26, and set the sorting gate with a diploid control (<a href="#fig3">Figure 3</a>D) before sorting the haploid cells.</li>
<li>Plate the cells in gelatin-coated 24-well TC plates with TSC medium. Usually, ∼1 × 10<sup>5</sup> cells are seeded per well of a 24-well TC plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To set a 1n peak (haploid cells in G0/G1 phase) accurately, a diploid control ESC line should be stained in parallel.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Staining buffer should be preheated in a 37°C water bath and protected from light.</div>
<h3 id="sec3.6">Derivation of haiTSCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days</div>
<ol start="37">
<li>Expand the haploid-enriched cells for 5–7 days, and sort CDCP1-positive cells again according to steps 21 to 28. Usually, the percentage of CDCP1-positive cells can reach more than 80% with two sorting steps.</li>
<li>Expand the sorted cells in TSC medium without Dox; these resulting cells can be defined as haiTSCs.</li>
<li>Expand, freeze or perform other experiments with the haiTSCs as needed.</li>
</ol>
<h3 id="sec3.7">Immunostaining and differentiation of haiTSCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8 days</div>
<ol start="40">
<li>Place round cover slips into the wells of 24-well TC plates, and coat the wells with 0.1% (wt/vol) gelatin for 12 h.</li>
<li>Seed the haiTSCs onto the gelatin-coated round cover slips, and culture them with TSC medium or differentiation medium (TSC basic medium without F4H) at 37°C and 5% CO<sub>2</sub>.</li>
<li>Culture haiTSCs for 2 days for regular cell immunostaining and 6 days for differentiated haiTSC immunostaining (<a href="#bib14">Tanaka et al., 1998</a>).</li>
<li>Wash the samples with PBS 3 times, and fix within 4% PFA for 30 min at 20°C–25°C .</li>
<li>Wash the samples with PBS 3 times, and permeabilize the cells with 0.5% (vol/vol) Triton X-100 diluted in blocking buffer for 1 h at 20°C–25°C.</li>
<li>Incubate the samples with primary antibody (diluted in blocking buffer at the required concentration) at 4°C for 12 h.</li>
<li>Wash the samples with PBS 3 times, and incubate them with secondary antibodies (diluted in PBS) for 1 h at 20°C–25°C.</li>
<li>Wash the samples with PBS 3 times, and incubate them with Hoechst 33342 (1:200 diluted in PBS) for 10 min at 20°C–25°C.</li>
<li>Wash the samples with PBS 3 times, and use a confocal microscope to view and capture images (<a href="#fig4">Figures 4</a>A and 4B).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Characteristics of the haiTSCs and their derivatives</div>
<p>(A) Immunofluorescence staining of the TSC markers CDX2 (green) and EOMES (red) in haiTSCs. DNA is stained with Hoechst 33342. Scale bar, 50 μm.</p>
<p>(B) Immunofluorescence staining of the trophoblast lineage-specific markers TFAP2C and TPBPA in differentiated cells from haiTSCs <i>in vitro</i>. DNA is stained with Hoechst 33342. Scale bar, 50 mm.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.8">Genome-wide trapping of haiTSCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="49">
<li>Prepare ∼10<sup>7</sup> haiTSCs with 30 μg of PB-SA gene trapping vector and 10 μg of PBase plasmid mixed in 400 μL R buffer.</li>
<li>Electroporate the mixture in 5 mL of E2 buffer 4 times with the following parameters: 1400 V, 10 ms, 3 pulses.</li>
<li>Plate the transfected cells on gelatin coated 6 wells of a 6-well plate with TSC medium supplemented with 10 μM Y27632 (withdraw Y27632 one day later), and culture them in an incubator for 3–4 days (<a href="#fig5">Figure 5</a>B).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Genome-wide mutations in haiTSCs</div>
<p>(A) Schematic overview of gene trapping in haiTSCs. Trapping vectors carrying a GFP gene are used for the selection of mutant cells with GFP.</p>
<p>(B) Bright-field and FITC-channel images of haiTSCs after transfection. Scale bar, 100 μm.</p>
<p>(C) FACS enrichment of GFP-positive cells after transfection. Untransfected haiTSCs are used as a negative control.</p>
<p>(D) Inverse PCR validation of the insertion sites in PB-haiTSCs. The smear bands represent multiple different insertions.</p>
</figcaption>
</figure>
</li>
<li>Perform cell sorting of transfected cells on a cell sorter to enrich GFP-positive cells with nontransfected cells as a negative control (<a href="#fig5">Figure 5</a>C).</li>
<li>Plate ∼5 × 10<sup>5</sup> GFP-positive cells per well of a 6-well plate on gelatin with TSC medium, and culture for another 3–4 days.</li>
<li>Confirm the presence of the gene trapping vector in the genome in haiTSCs by PCR (<a href="#fig5">Figure 5</a>D and <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Mmc1.pdf">Table S1</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If the cell viability of electroporated cells is suboptimal or poor, you can prolong the culture days to recover from electroporation.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol for deriving haiTSCs is very detailed, which guarantees reliable and repeatable derivation of haiTSCs (<a href="#fig3">Figure 3</a>A). According to this protocol, haiTSCs can maintain haploidy without sorting for 1 month, which can be beneficial for genome-wide genetic screening (<a href="#fig5">Figure 5</a>A). The typical TSC identities of haiTSCs can maintain stable during Dox withdrawal (<a href="#fig4">Figure 4</a>A) and thus possess the potential to further differentiate into placental lineages (<a href="#fig4">Figure 4</a>B). Related key protocols allow derivation of <i>p53</i>-KO haESCs, immunofluorescent cells, and haiTSCs capable of undergoing genome-wide assays.</p>
<p>To ensure successful derivation of haiTSCs, we describe some key steps here. It is important to ensure that the generated haESCs are indeed the <i>p53</i>-KO genotype. The 281-bp band amplified with the primer pair of F/R1 represents the WT genotype, whereas the 441-bp band from the F/R2 represents the <i>p53</i>-KO genotype. Therefore, the <i>p53</i>-KO haESCs should only show the 441-bp band but not the 281-bp band (<a href="#fig1">Figure 1</a>B). To explain why we chose <i>p53</i>-KO haESCs to derive haiTSCs, we compared the haploidy maintenance ability of <i>p53</i>-KO haESCs and WT haESCs without sorting for 14 days. The total proportions of haploid cells were calculated according to a previously reported formula (total haploid cells=1n% of haESCs+(2n% of haESCs-x%), x%=(2n% of WT-ESCs)/[1-(2n% of WT-ESCs)] × 4n% of haESCs)) (<a href="#bib16">Xu et al., 2017</a>). There were more haploid cells harvested from <i>p53</i>-KO haESCs in the interval period than from WT-ESCs (<a href="#fig1">Figures 1</a>C and 1D).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Spontaneous diploidization of mouse haESCs during daily culture is an inevitable trend, although the <i>p53</i>-KO strategy slows diploidization and prolongs the interval period between cell sorting sessions. Herein, FACS for maintaining the haploid cells every 3 weeks is still necessary. As Hoechst 33342 is toxic, thus affecting the cell viability of the sorted cells, the sorting efficiency is critical for subsequent experiments (<a href="#bib1">Cui et al., 2020</a>). Another concern is that there are too many steps for sorting with CDCP1 antibody, which is risky and can reduce the sorting efficiency. Therefore, Hoechst 33342 sorting and CDCP1 enrichment should not be performed simultaneously. The heterogeneity of haiTSCs from differentiated cell cultures leads to the purity of CDCP1+ cells never reaching 100%, but our report indicated that the differentiation potential of haiTSCs into TE lineages makes them an ideal model for placental genetic screens (<a href="#bib10">Peng et al., 2019</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>There were few surviving cells from electroporation (see steps 6–11).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Cell viability is affected by electroporation. We recommend adding 10 μM Y27632 to the culture medium after plating for 24 h. Y27632 should not be added for more than one day because it might affect the cell status in long-term culture.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The cell cultures died dramatically after passaging during the conversion of haESCs to haiTSCs (see steps 15–20).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Severe cell death in this step is mainly due to two reasons: 1) Cell death is a normal phenomenon during cell fate conversion. When the confluence of cells reached 90%, the cells needed to be passaged to maintain optimal growth, and a high confluence of plated cells is necessary for passaging. 2) The reason for the poor cell status may be that the cells could be contaminated with mycoplasma. You should test whether your cell cultures are contaminated with mycoplasma. Mycoplasma cannot be eliminated and does irreversible damage to the cells. You should restart the induction with another mycoplasma-free sample.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Many cells are lost after centrifugation in the steps involving CDCP1 antibody staining (see steps 21–28).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The blocking buffer to dilute the primary antibody includes 2% BSA, which is sticky and made the majority of cells adhere to the inner sidewalls of 15-mL centrifuge tubes. To reduce cell loss, 15-mL tubes should be precoated before use, and the inner sidewalls should be flushed with pipettes several times, and the centrifugation time should be prolonged appropriately.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>There were few surviving cells after FACS for haploid sorting (see steps 29–36).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Hoechst 33342 is toxic. High concentrations of Hoechst 33342 guarantee the CV of the data and make it more accurate, but it brings more cell death to the samples. The suitable concentration should adjust to your samples and model of cell sorter. To reduce damage to cells caused by Hoechst 33342, the concentration of Hoechst 33342 (1 μg/mL - 3 μg/mL) is reduced or the staining time is reduced (20 min–30 min).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>There were few sorted haploid cells because of the low proportion of haploids in the process of deriving haiTSCs (see step 37).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Haploid cells undergo severe diploidization during cell fate conversion. Therefore, the initial proportion of haploid cells should be high. To avoid missing the best time for sorting, FACS should be performed for haploids as long as the cell amount satisfies the sorting requirement. Shortening the time interval between haploid sorting is also beneficial for the maintenance of haploidy.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cui, T., Li, Z., Zhou, Q., and Li, W. (2020). Current advances in haploid stem cells. Protein Cell <i>11</i>, 23-33. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref1">View at publisher</a></p>
<p id="bib2">Gao, Q., Zhang, W., Ma, L., Li, X., Wang, H., Li, Y., Freimann, R., Yu, Y., Shuai, L., and Wutz, A. (2018). Derivation of haploid neural stem cell lines by selection for a Pax6-GFP reporter. Stem Cell Dev. <i>27</i>, 479-487. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref2">View at publisher</a></p>
<p id="bib3">Gao, Q., Zhang, W., Zhao, Y., Tian, Y., Wang, Y., Zhang, J., Geng, M., Xu, M., Yao, C., Wang, H., et al. (2021). High-throughput screening in postimplantation haploid epiblast stem cells reveals Hs3st3b1 as a modulator for reprogramming. Stem Cell Transl. Med. <i>10</i>, 743-755. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref3">View at publisher</a></p>
<p id="bib4">He, W., Zhang, X., Zhang, Y., Zheng, W., Xiong, Z., Hu, X., Wang, M., Zhang, L., Zhao, K., Qiao, Z., et al. (2018). Reduced self-diploidization and improved survival of semi-cloned mice produced from androgenetic haploid embryonic stem cells through overexpression of Dnmt3b. Stem Cell Reports <i>10</i>, 477-493. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref4">View at publisher</a></p>
<p id="bib5">He, Z.Q., Xia, B.L., Wang, Y.K., Li, J., Feng, G.H., Zhang, L.L., Li, Y.H., Wan, H.F., Li, T.D., Xu, K., et al. (2017). Generation of mouse haploid somatic cells by small molecules for genome-wide genetic screening. Cell Rep. <i>20</i>, 2227-2237. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref5">View at publisher</a></p>
<p id="bib6">Hockemeyer, D., Soldner, F., Cook, E.G., Gao, Q., Mitalipova, M., and Jaenisch, R. (2008). A drug-inducible system for direct reprogramming of human somatic cells to pluripotency. Cell Stem Cell <i>3</i>, 346-353. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref6">View at publisher</a></p>
<p id="bib7">Macfarlan, T.S., Gifford, W.D., Driscoll, S., Lettieri, K., Rowe, H.M., Bonanomi, D., Firth, A., Singer, O., Trono, D., and Pfaff, S.L. (2012). Embryonic stem cell potency fluctuates with endogenous retrovirus activity. Nature <i>487</i>, 57-63. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref7">View at publisher</a></p>
<p id="bib8">Niwa, H., Toyooka, Y., Shimosato, D., Strumpf, D., Takahashi, K., Yagi, R., and Rossant, J. (2005). Interaction between Oct3/4 and Cdx2 determines trophectoderm differentiation. Cell <i>123</i>, 917-929. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref8">View at publisher</a></p>
<p id="bib9">Olbrich, T., Mayor-Ruiz, C., Vega-Sendino, M., Gomez, C., Ortega, S., Ruiz, S., and Fernandez-Capetillo, O. (2017). A p53-dependent response limits the viability of mammalian haploid cells. Proc. Natl. Acad. Sci. U S A <i>114</i>, 9367-9372. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref9">View at publisher</a></p>
<p id="bib10">Peng, K., Li, X., Wu, C., Wang, Y., Yu, J., Zhang, J., Gao, Q., Zhang, W., Zhang, Q., Fan, Y., et al. (2019). Derivation of haploid trophoblast stem cells via conversion in vitro. iScience <i>11</i>, 508-518. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref10">View at publisher</a></p>
<p id="bib11">Rugg-Gunn, P.J., Cox, B.J., Lanner, F., Sharma, P., Ignatchenko, V., McDonald, A.C., Garner, J., Gramolini, A.O., Rossant, J., and Kislinger, T. (2012). Cell-surface proteomics identifies lineage-specific markers of embryo-derived stem cells. Dev. Cell <i>22</i>, 887-901. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref11">View at publisher</a></p>
<p id="bib12">Shuai, L., Li, W., Wan, H., Zhao, X.Y., Wang, L., and Zhou, Q. (2014). Generation of Mammalian offspring by haploid embryonic stem cells microinjection. Curr. Protoc. Stem Cell Biol. <i>31</i>, 1A 6 1-15. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref12">View at publisher</a></p>
<p id="bib13">Sun, S., Zhao, Y., and Shuai, L. (2020). The milestone of genetic screening: mammalian haploid cells. Comput. Struct. Biotechnol. J. <i>18</i>, 2471-2479. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref13">View at publisher</a></p>
<p id="bib14">Tanaka, S., Kunath, T., Hadjantonakis, A.K., Nagy, A., and Rossant, J. (1998). Promotion of trophoblast stem cell proliferation by FGF4. Science <i>282</i>, 2072-2075. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref14">View at publisher</a></p>
<p id="bib15">Woltjen, K., Michael, I.P., Mohseni, P., Desai, R., Mileikovsky, M., Hamalainen, R., Cowling, R., Wang, W., Liu, P., Gertsenstein, M., et al. (2009). piggyBac transposition reprograms fibroblasts to induced pluripotent stem cells. Nature <i>458</i>, 766-770. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref15">View at publisher</a></p>
<p id="bib16">Xu, H., Yue, C., Zhang, T., Li, Y., Guo, A., Liao, J., Pei, G., Li, J., and Jing, N. (2017). Derivation of haploid neurons from mouse androgenetic haploid embryonic stem cells. Neurosci. Bull. <i>33</i>, 361-364. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref16">View at publisher</a></p>
<p id="bib17">Ying, Q.L., Wray, J., Nichols, J., Batlle-Morera, L., Doble, B., Woodgett, J., Cohen, P., and Smith, A. (2008). The ground state of embryonic stem cell self-renewal. Nature <i>453</i>, 519-523. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref17">View at publisher</a></p>
<p id="bib18">Zhao, X.Y., Lv, Z., Li, W., Zeng, F., and Zhou, Q. (2010). Production of mice using iPS cells and tetraploid complementation. Nat. Protoc. <i>5</i>, 963-971. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00587-6/sref18">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Ling Shuai, PhD, <a href="mailto:lshuai@nankai.edu.cn">lshuai@nankai.edu.cn</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze [data sets/code].</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1154-Mmc1.pdf">Document S1. Table S1</a></p>
<h3>Acknowledgments</h3>
<p>This work was supported by the <a href="https://doi.org/10.13039/501100012166">National Key Research and Development Program of China</a> (2019YFA0109901 and 2018YFC1004101 to L.S.), the “Fundamental Research Funds for the <a href="https://doi.org/10.13039/501100012429">Central Universities</a>”, <a href="https://doi.org/10.13039/501100004945">Nankai University</a> (63213079 to L.S.), the <a href="https://doi.org/10.13039/501100001809">National Natural Science Foundation of China</a> (32022025 and 31872841 to L.S.), Strategic Collaborative Research Program of the Ferring Institute of Reproductive Medicine (FIRMD181102 to L.S.), the Open Fund of Tianjin Central Hospital of Gynecology Obstetrics/Tianjin Key Laboratory of Human Development and Reproductive Regulation (2019XHY08 to L.S.), Funding from National Clinical Research Center for Obstetrics and Gynecology (Peking University Third Hospital) (BYSYSZKF2021007 to L.S.), and the Chunfeng Science &amp; Technology Fund of Tianjin First Central Hospital (FCHCF201804 to Y.L.).</p>
<h3>Author contributions</h3>
<p>M.X., W.Z., and M.G. performed most of the experiments. M.X., Y.Z., S.S., and Q.G. wrote the manuscript draft. Y.L. and L.S. supervised this project, provided funding, and reviewed the final manuscript. All authors discussed about the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>