<article>
<section>
<h1 id="header">Isolation of myeloid cells from mouse brain tumors for single-cell RNA-seq analysis</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Mai T. Dang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">7</a>,<a href="#cor2">**</a></sup> Fernanda Mafra,<sup><a href="#aff3">3</a></sup> and Malay Haldar<sup><a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#fn1">7</a>,<a href="#fn2">8</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Division of Neurology, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</p>
<p id="aff2"><sup>2</sup>Division of Pediatric and Developmental Neurology, Washington University School of Medicine, St. Louis, MO 63110, USA</p>
<p id="aff3"><sup>3</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA</p>
<p id="aff4"><sup>4</sup>Abramson Family Cancer Research Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p>
<p id="aff5"><sup>5</sup>Department of Pathology and Laboratory Medicine, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p>
<p id="aff6"><sup>6</sup>Institute for Immunology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p>
<p id="fn1"><sup>7</sup>Technical contact</p>
<p id="fn2"><sup>8</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:mhaldar@pennmedicine.upenn.edu">mhaldar@pennmedicine.upenn.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:mai@wustl.edu">mai@wustl.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100957">10.1016/j.xpro.2021.100957</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Current single-cell RNA sequencing (scRNA-seq) protocols are limited by the number of cells that can be simultaneously sequenced, restricting the ability to resolve heterogeneity of rare cell types. We describe here a protocol for rapid isolation of myeloid cells from tumor-harboring mouse cerebellum without cell sorting to minimize cell damage for scRNA-seq. This protocol includes the procedures for further enrichment of myeloid cells using CD11b+ magnetic beads, followed by the generation of scRNA library and sequencing analysis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref2">Dang et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Isolation of myeloid cells from tumor-bearing mouse cerebellum</li>
<li>Single-cell RNA sequencing (scRNA-seq) of murine cerebellar myeloid cells</li>
<li>Identification of the major myeloid subsets using scRNA-seq data</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1183-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Mouse models to study brain tumor immune microenvironment include genetically engineered autochthonous models, orthotopic injection of syngeneic cell lines, and injection of viral constructs expressing oncogenes. Please refer to the following publications for a detailed discussion of various mouse models for brain tumors such as medulloblastoma and high-grade gliomas: <a href="#bib11">Simeonova and Huillard (2014),</a> <a href="#bib4">Hetze et al. (2021)</a>, and <a href="#bib6">Li and Langhans (2021)</a>. The time required to generate tumors will depend on the specific model. The protocol described here uses the sonic hedgehog subgroup of medulloblastoma (SHH-MB) that spontaneously develops in <i>Ptch1</i><sup><i>+/−</i></sup> <i>p53</i><sup><i>−/−</i></sup> mice. These mice develop tumor with complete penetrance by 8-weeks of age (<a href="#bib12">Wetmore et al., 2001</a>).</p>
<p>The desired number of myeloid cells to sequence will depend on many factors including the density and complexity of myeloid cells in the tumor as well as the desired sequencing depth per cell. Therefore, it is important to perform flow cytometry or other orthogonal analyses to estimate density and complexity of tumor-associated myeloid cells prior to planning scRNA-Seq. In general, we aim to have 50 to 100 cells of each predicted subset of interest. For more thorough discussions, including a rationale for this quantity, please refer to following publications: <a href="#bib10">Shapiro et al. (2013)</a>, <a href="#bib8">Nguyen et al. (2018)</a>, and <a href="#bib13">Zhang et al. (2020)</a>.</p>
<p>To enrich samples for tumor-infiltrating myeloid cells, it is ideal to start with tumor tissue with minimal surrounding normal brain tissue. This is relatively easy for tumors that have features discernible under a dissection microscope or if tumor cells express unique surface markers or reporter genes (e.g., fluorescent proteins). Tumor cells in <i>Ptch1</i><sup><i>+/−</i></sup> <i>p53</i><sup><i>−/−</i></sup> mice described here do not express such unique markers or reporter, which is also true for most murine models and human tumors. Hence, the protocol below describes isolation of myeloid cells from brain tissue containing tumor cells, which is applicable to a wide variety of brain tumors.</p>
<div class="note">
<span class="note-title">Note:</span> Institutional permission should be obtained for the animal studies. Animal work presented here was conducted according to the National Institutes of Health guidelines and approved by the Institutional Animal Care and Use Committee at the University of Pennsylvania.</div>
<div class="note">
<span class="note-title">Note:</span> We used <i>Ptch1</i><sup><i>+/−</i></sup> <i>p53</i><sup><i>−/−</i></sup> mice at 8 weeks of age male mice before they developed symptoms of SHH-MB. All <i>Ptch1</i><sup><i>+/−</i></sup> <i>p53</i><sup><i>−/−</i></sup> mice are predicted to have tumors at this age, nevertheless we MR imaged mice at 8 weeks of age to confirm the presence of tumor and isolated cells from tumor-harboring cerebellum. We recommend using brain tissue that contains the largest proportion of tumor volume possible before animals reach predetermined symptomatic endpoints. This will allow for the least amount of homeostatic microglia from brain tissue surrounding the tumor to be present in the sample. We provide recommendations on how to identify this population in the final scRNA-Seq data analysis section.</div>
<div class="note">
<span class="note-title">Note:</span> We ascertained that the SHH-MB model contains significant myeloid infiltrates by flow cytometry and immunohistochemistry prior to scRNA-Seq (see Figures 2B, 5A, and S5E in <a href="#bib2">Dang et al., 2021</a>).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Percoll</td>
<td>Sigma-Aldrich</td>
<td>17-0891-02</td>
</tr>
<tr>
<td>Dulbecco’s 1×PBS</td>
<td>MilliporeSigma</td>
<td>D8537</td>
</tr>
<tr>
<td>Dulbecco’s 10×PBS</td>
<td>MilliporeSigma</td>
<td>59331C</td>
</tr>
<tr>
<td>RPMI1640 Medium</td>
<td>Fisher Scientific</td>
<td>MT10-040-CM</td>
</tr>
<tr>
<td>Trypan Blue Solution, 0.4%</td>
<td>Thermo Fisher Scientific</td>
<td>15250061</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>Millipore Sigma</td>
<td>A9647</td>
</tr>
<tr>
<td>EDTA</td>
<td>Thermo Fisher Scientific</td>
<td>AMB9261</td>
</tr>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>BV510 Cd45 anti-mouse antibody (1:200)</td>
<td>BioLegend</td>
<td>103138;RRID:AB_2563061</td>
</tr>
<tr>
<td>PE-Cy7 Cd11b anti-mouse antibody (1:200)</td>
<td>Fisher Scientific</td>
<td>25-0112-82;RRID:AB_469588</td>
</tr>
<tr>
<td>PerCp/Cy5.5 Ly6G anti-mouse antibody (1:200)</td>
<td>BioLegend</td>
<td>127615;RRID:AB_1877272</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>CD11b+ magnetic MicroBeads</td>
<td>Miltenyi Biotec Inc</td>
<td>130-092-636</td>
</tr>
<tr>
<td>CD45+ magnetic MicroBeads</td>
<td>Miltenyi Biotec Inc</td>
<td>130-052-301</td>
</tr>
<tr>
<td>LS columns</td>
<td>Miltenyi Biotec Inc</td>
<td>130-042-401</td>
</tr>
<tr>
<td>Chromium Next GEM Single Cell 3ʹ Kit v3.1</td>
<td>10x Genomics</td>
<td>PN-1000268</td>
</tr>
<tr>
<td>Chromium Next GEM Chip G Single Cell Kit</td>
<td>10x Genomics</td>
<td>PN-1000120</td>
</tr>
<tr>
<td>Dual Index Kit TT Set A, 96 rxns</td>
<td>10x Genomics</td>
<td>PN-1000215</td>
</tr>
<tr>
<td>Chromium i7 Sample Index Kit</td>
<td>10x Genomics</td>
<td>PN-120262</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>scRNA-seq of Cd11b+ myeloid cells from Ptch1+/-p53−/− mice</td>
<td><a href="#bib2">Dang et al. (2021)</a></td>
<td>GEO: GSE166691</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: Ptch<sup>+/-</sup>
</td>
<td>The Jackson Laboratory</td>
<td>030494</td>
</tr>
<tr>
<td>Mouse: Tp53<sup>−/−</sup>
</td>
<td>The Jackson Laboratory</td>
<td>002101</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Seurat package (v.3.0)</td>
<td><a href="#bib9">Satija et al. (2015)</a></td>
<td><a href="https://satijalab.org/seurat/">https://satijalab.org/seurat/</a></td>
</tr>
<tr>
<td>SingleR</td>
<td><a href="#bib1">Aran et al. (2019)</a></td>
<td><a href="https://bioconductor.org/packages/release/bioc/html/SingleR.html">https://bioconductor.org/packages/release/bioc/html/SingleR.html</a></td>
</tr>
<tr>
<td>R</td>
<td>R Core Team</td>
<td><a href="http://www.R-project.org">http://www.R-project.org</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Spinal needle 20G 6”</td>
<td>Medline</td>
<td>BD405211</td>
</tr>
<tr>
<td>70uM cell strainer</td>
<td>CELLTREAT</td>
<td>229483</td>
</tr>
<tr>
<td>Cole-Parmer homogenizer 10 mL</td>
<td>Cole-Parmer</td>
<td>UX-44468-03</td>
</tr>
<tr>
<td>5 mL syringe</td>
<td>Fisher Scientific</td>
<td>309646</td>
</tr>
<tr>
<td>MidiMACS separator</td>
<td>Miltenyi Biotec Inc</td>
<td>130-042-302</td>
</tr>
<tr>
<td>MACS Multistand</td>
<td>Miltenyi Biotec Inc</td>
<td>130-042-303</td>
</tr>
<tr>
<td>Chromium Single Cell Controller</td>
<td>10x Genomics</td>
<td>CG00050</td>
</tr>
<tr>
<td>C1000 Touch thermal cycler</td>
<td>Bio-Rad Laboratories</td>
<td>1851197</td>
</tr>
<tr>
<td>Tape Station High Sensitivity D5000 Reagents (used for cDNA QC)</td>
<td>Agilent Technologies</td>
<td>5067-5593</td>
</tr>
<tr>
<td>Tape Station High Sensitivity D5000 ScreenTape (used for cDNA QC)</td>
<td>Agilent Technologies</td>
<td>5067-5592</td>
</tr>
<tr>
<td>Tape Station High Sensitivity D1000 Reagents (used for Library QC)</td>
<td>Agilent Technologies</td>
<td>5067-5585</td>
</tr>
<tr>
<td>Tape Station High Sensitivity D1000 ScreenTape (used for Library QC)</td>
<td>Agilent Technologies</td>
<td>5067-5584</td>
</tr>
<tr>
<td>KAPA Library Quantification Kit for Illumina Platforms</td>
<td>Roche Holding AG</td>
<td>07960336001 - KK4873</td>
</tr>
<tr>
<td>NovaSeq 6000 sequencer</td>
<td>Illumina</td>
<td>20012850</td>
</tr>
<tr>
<td>NovaSeq 6000 SP Reagent Kit v1.5 (100 cycles) flow cell</td>
<td>Illumina</td>
<td>20028401</td>
</tr>
<tr>
<td>PhiX Control v3</td>
<td>Illumina</td>
<td>FC-110-3001</td>
</tr>
<tr>
<td>SPRI Select 60 mL</td>
<td>Beckman Coulter</td>
<td>B23318</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Stock Isotonic Percoll (SIP100), volume per sample (7.5 mL/sample [6.85 mL + 10%])</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Percoll</td>
<td>–</td>
<td>6.75 mL</td>
</tr>
<tr>
<td>Dulbecco’s 10× PBS</td>
<td>1×</td>
<td>0.75 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">[Prepare day of experiment and keep at room temperature (20°C–24°C) ]</p>
<table id="undtbl2">
<caption>70% Percoll 70% (SIP70), volume per sample (5.5 mL/sample [5 mL + 10% extra)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>SIP100</td>
<td>70%</td>
<td>3.85 mL</td>
</tr>
<tr>
<td>Dulbecco’s 1×PBS</td>
<td>-</td>
<td>1.65 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">[Prepare day of experiment and keep at room temperature]</p>
<table id="undtbl3">
<caption>MACS buffer (500 mL total)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× Dulbecco’s Phosphate-Buffered Saline (sterile PBS)</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>Bovine Serum Albumin (BSA)</td>
<td>0.5%</td>
<td>2.5 g</td>
</tr>
<tr>
<td>0.5 M EDTA (pH 8.0)</td>
<td>2 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>Distilled water (dH<sub>2</sub>O)</td>
<td>-</td>
<td>Fill to 500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">[Filter through a polystyrene membrane under sterile conditions; store at 4°C for up to 2 weeks.]</p>
<div class="critical">
<span class="critical-title">Critical:</span> Keep MACS buffer on ice and use it cold during experiment.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>This protocol has five main sections (see below). Sections 1 and 2 are specific to this protocol while sections 3–5 are generic and depend on user preferences.</p>
<p>Section 1: Isolation of myeloid cells from tumor-bearing brain tissue with Percoll.</p>
<p>Section 2: Further enrichment of myeloid cells using CD11b+ magnetic beads.</p>
<p>Section 3: Generation of Single Cell RNA library using the 10× chromium platform.</p>
<p>Section 4: Sequencing.</p>
<p>Section 5: Analyses of sequencing data.</p>
<h3 id="sec3.1">Section 1: Isolation of myeloid cells from tumor-bearing brain tissue with Percoll</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Myeloid cells are isolated from brain tumors using a fixed gradient Percoll separation.</p>
<h4 id="sec3.1.1">Homogenize brain tumor</h4>
<p>(Perform at room temperature)</p>
<ol>
<li>Resect brain tumor (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1183-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Resection of the mouse brain cerebellum with tumor</div>
<p>(A) Peeling the skin away from the skull.</p>
<p>(B) One arm of the scissors is inserted into the foramen magna where the brainstem runs through.</p>
<p>(C) Cutting skull along the sagittal suture.</p>
<p>(D) Removal of skull.</p>
<p>(E) Removal of the brain starting at the frontal lobe.</p>
<p>(F) Disconnecting nerve fibers from the brainstem.</p>
<p>(G) On the left is a normal brain. On the right is a brain with tumor. This mouse was symptomatic and had hydrocephalus resulting in a larger brain volume above the tentorium as well. Folia of the cerebellum are no longer discernible.</p>
<p>(H) Removal of the cerebellum away from the brainstem (white).</p>
<p>(I) Disconnecting the nerve fibers from the brainstem to the cerebellum.</p>
<p>(J) Cerebellum with tumor is fully removed.</p>
</figcaption>
</figure>
<ol type="a">
<li>After euthanizing the mouse, remove the head of the mouse using a pair of scissors.</li>
<li>Cut away the skin (<a href="#fig1">Figure 1</a>A) and excess neck muscle until the brain stem can be seen (<a href="#fig1">Figure 1</a>B).</li>
<li>Starting from the brain stem, cut upward along the sagittal suture applying as little pressure as possible to the brain itself (<a href="#fig1">Figure 1</a>C).</li>
<li>Peel each hemisphere of the skull away from the brain to the side (<a href="#fig1">Figure 1</a>D).</li>
<li>With a spatula or foreceps, remove the brain from the base of the skull and cut long nerve fibers to detach the brain (<a href="#fig1">Figures 1</a>E and 1F).</li>
<li>For removal of the cerebellum, gently push the cerebellum with tumor away from the brainstem and detach the nerve fibers (<a href="#fig1">Figures 1</a>H and 1I). We used the entire cerebellum containing bilateral medulloblastomas.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For tumors that are highly vascularized and intravascular immune cells are predicted to be high, perfuse mouse with 10 mL ice-cold PBS through the left ventricle prior to resection. For a detailed protocol on perfusing rodent, refer to <a href="#bib3">Gage et al. (2012)</a>. Alternatively, immune cells from peripheral blood can be identified during the analysis of the single-cell data using published data sets obtained from cells of peripheral blood (<a href="#bib2">Dang et al., 2021</a>).</div>
<ol start="2">
<li>Add 5 mL RPMI solution and tumor into a 10 mL glass homogenizer and homogenize tissue.</li>
<li>Place a 70 μm cell filter on a 50 mL conical tube and pipet onto it the homogenate to filter out cell clumps. Transfer the flow through into a 15 mL conical tube.</li>
</ol>
<h4 id="sec3.1.2">Percoll gradient-based isolation of myeloid cells</h4>
<p>(Perform at room temperature at the bench. Sterile conditions are not needed.)</p>
<ol start="4">
<li>To cell suspension, add RPMI solution to make a total of 7 mL. Then add 3 mL SIP to cell suspension and mix by pipetting to create 10 mL cell suspension in 30% Percoll.</li>
<li>Take up 5 mL of 70% Percoll solution in a 5 mL syringe attached to a spinal needle. Insert the needle into the cell suspension until the bevel reaches the bottom of the conical tube. Gently and slowly, eject 70% Percoll layer below the 30% Percoll cell suspension layer.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To minimize turbulence to make a sharp, demarcating line between the two layers, make sure that no air bubbles are in the needle or syringe. Withdraw the needle with a slow, smooth motion while keeping the needle parallel to the side of the tube.</div>
<div class="note">
<span class="note-title">Note:</span> If spinal needles are not available, perform the layering in the reverse order. First, pipette 5 mL of 70% Percoll solution into a 15 mL tube and slowly pipette onto it the 10 mL of cell suspension in 30% Percoll with a P1000 pipet. This should be performed gently to ensure a sharp, demarcating line between the two layers.</div>
<div class="note">
<span class="note-title">Note:</span> If two layers are not clearly seen, recheck original concentration of 30% Percoll cell suspension and 70% Percoll to make sure the correct percentage of Percoll was made for each.</div>
<ol start="6">
<li>Carefully and gently, carry the tubes to a centrifuge set at room temperature. Set the centrifuge acceleration to maximal speed and deceleration to 0 to eliminate turbulence which causes unwanted mixing of the two layers disrupting the gradient. Spin the gradient mixture at 500 × <i>g</i> for 30 min. This spin will take 35–40 min because of the slow deceleration.
<div class="note">
<span class="note-title">Note:</span> Percoll separation is optimized when the solution is at room temperature and the spinning is done in a centrifuge set at room temperature.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Set the centrifuge acceleration to maximal speed and deceleration to 0 to eliminate turbulence during deceleration which can cause unwanted mixing of the two layers.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If you wish to isolate a broader collection of immune cells including lymphocytes, use a 40/80 Percoll gradient, a spin speed of 325 × <i>g</i> for 30 minutes, and CD45 microbeads in section 2. Use the following steps.</div>
<ol type="a">
<li>Prepare the following stock solution:
<ol type="i">
<li>Stock Isotonic Percoll SIP 100% (SIP100), volume per sample (9.3/sample [8.4 mL+ ∼10% extra])
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Percoll</td>
<td>-</td>
<td>8.37 mL</td>
</tr>
<tr>
<td>10× PBS</td>
<td>1×</td>
<td>0.93 mL</td>
</tr>
</tbody>
</table>
</li>
<li>80% Percoll, volume per sample (5.5 mL/sample [5 mL+10% extra])
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>SIP</td>
<td>80%</td>
<td>4.4 mL</td>
</tr>
<tr>
<td>1× PBS</td>
<td>-</td>
<td>1.1 mL</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>Homogenize tissue as described above. Bring cell suspension volume up to 6 mL and add 4 mL of SIP to make 10 mL of 40% Percoll cell suspension. Layer 80% Percoll beneath the 40% Percoll layer using method described above. Spin at 325 × <i>g</i> for 30 min.</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Gently remove the tube from the centrifuge. A white, thin layer of myelin will be seen at the top of the tube. Additionally, a cloudy interface layer of several millimeters thick containing myeloid cells will be seen at the interface between the two Percoll layers.</li>
<li>Aspirate and discard the top white layer that contains myelin fragments.</li>
<li>Using a spinal needle attached to a 5 mL syringe, insert the needle into the gradient until the tip hits the interface layer and gently aspirate into the syringe the interface layer (∼2–3 mL depending on the number of cells that determines the thickness of the layer) and gently transfer to a fresh 15 mL conical tube.</li>
<li>To wash cells and remove Percoll, add ice-cold 12 mL MACS buffer. Mix with gentle pipetting. Reset centrifuge deceleration back to maximum and temperature to 4°C. Spin cell suspension 300 × <i>g</i> for 7 min.</li>
<li>Discard supernatant and resuspend cell pellet in 1 mL of cold MACs buffer and transfer to 1.5 mL microcentrifuge tube.</li>
<li>Spin at 300 × <i>g</i> speed for 1 min.</li>
<li>Discard supernatant and resuspend cell pellet in 90 μL of MACs buffer (per 10<sup>7</sup> total cells).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The number of cells in this sample is dependent on the density of myeloid cells in tumor and size of tissue that was homogenized. Quantification can be performed with a cell counter or hemocytometer. Flow cytometry with common myeloid markers can be performed to confirm the quality of myeloid enrichment. Percoll purified cells in mouse medulloblastoma that occupy the majority of the cerebellum is generally less than 10<sup>7</sup> in quantity.</div>
<h3 id="sec3.2">Section 2: Further enrichment of myeloid cells using CD11b+ magnetic beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<div class="note">
<span class="note-title">Note:</span> The detailed protocol is described in the manufacturer’s protocol (<a href="https://www.miltenyibiotec.com/US-en/products/cd11b-microbeads-human-and-mouse.html#gref">https://www.miltenyibiotec.com/US-en/products/cd11b-microbeads-human-and-mouse.html#gref</a>; click “Data sheet”). Below are the major steps and conditions used in <a href="#bib2">Dang et al. (2021)</a>.</div>
<ol start="14">
<li>Incubate cell suspension with CD11b MicroBeads<sup>TM</sup> by adding 10 μL of microbeads to every 90 μL of cell suspension and incubate for 15 min on ice.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> CD45 MicroBeads<sup>TM</sup> can be used with cells collected from a 40/80 Percoll gradient to capture lymphocytes and myeloid cells.</div>
<ol start="15">
<li>To wash cells, resuspend in 1 mL of MACs buffer and spin at 300 × <i>g</i> for 10 min.</li>
<li>Discard supernatant and resuspend in 500 μL of cold MACs buffer and mix with gentle pipetting.</li>
<li>While cells are being prepared in step 14, set up an LS column in the MidiMACS separator mounted to a MACS Multistand (following manufacturer’s recommendation in protocol listed above) over a 15 mL conical tube to collect flow through and wash with 3 mL of cold MACS buffer.</li>
<li>Apply the cell suspension onto the column and wait for the solution to run through the column.</li>
<li>Then wash the column 3× with 3 mL cold MACS buffer.</li>
<li>Remove the column from the separator and place it on a new 15 mL conical tube.</li>
<li>Pipette 5 mL of cold MACS buffer onto the column and immediately flush out magnetically labeled cells by applying and pushing the plunger supplied with the column.</li>
<li>Spin at 300 × <i>g</i> for 5 min.</li>
<li>Discard supernatant. Since it can be difficult to remove all supernatant from a 15 mL conical tube without disrupting a small cell pellet, remove as much supernatant as possible. Then add 1 mL cold MACS buffer. Mix with gentle pipetting and transfer cell suspension to a 1.5 mL microcentrifuge tube. Spin at 500 × <i>g</i> for 1 min to pellet the cell. Remove as much supernatant as possible. Save the supernatant in case not all cells are pelleted.</li>
<li>Resuspend cells in desired volume and buffer appropriate for the scRNA-sequencing platform. For 10× Genomics, we recommend using 1×PBS (calcium and magnesium free) containing 0.04% weight/volume BSA (400 μg/mL). BSA is added to minimize cell losses and aggregation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the desired volume is uncertain, we recommend resuspending cells in about 50–100 μL of buffer, which will allow for enough volume for cell counting and a cell concentration that may be approximately appropriate for loading for cDNA preparation. Adjustment to volume (either more concentrated or dilute) can be made after the cell count.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Buffer should not contain excessive amounts of EDTA (&gt;0.1 mM) or magnesium (&gt;3 mM) because those components will inhibit the reverse transcription reaction.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To minimize physical damage to cells it is important to pipette gently and slowly during cell resuspension steps. Use wide-bore pipette tips to minimize cell damage.</div>
<ol start="25">
<li>Using trypan blue and a hemocytometer, count the number of viable cells to determine quantity of sample to be used for library preparation. For detailed instructions on counting cells, refer to the following online protocol by <a href="#bib7">Moore (2012)</a> (<a href="https://www.protocols.io/view/counting-cells-with-hemocytometer-nxsdfne">https://www.protocols.io/view/counting-cells-with-hemocytometer-nxsdfne</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cell suspensions should also be checked visually for debris or cell aggregates because these can clog microfluidic channels. If observed, additional pipetting or filtering may be required to obtain optimal performance. Pipetting should be done gently and carefully to avoid lysing or damaging cells. Alternatively, filter cell suspension with one of the following cell strainers: Flowmi™ Cell Strainer (40 micron, Fisher Scientific 14-100-150) or MACS SmartStrainer (30 micron, Miltenyi 130-098-458)</div>
<div class="note">
<span class="note-title">Note:</span> If cell concentration is much lower than expected, check the concentration of saved supernatants and re-pellet the supernatant at a higher centrifugation speed. Cells may not have been efficiently pelleted at the initial speed or were lost in the cell strainer.</div>
<ol start="26">
<li>Dilute or concentrate the solution to obtain cell suspension concentration between 700 and 1200 cells/μL for optimal chip loading.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To the determine the optimal suspension volume to achieve target cell recovery, refer to the Cell Suspension Volume Calculator Table on page 30 of the Chromium Next GEM Automated Single Cell 3′ Reagent Kits v3.1 User Guide (CG000286) (<a href="https://assets.ctfassets.net/an68im79xiti/5qIua7LwDKxoNEvHJnYqcB/c778731aa23ca5e51e1aa5d1c8941828/CG000286_ChromiumSingleCell3-_v3.1_Automation_UG_RevB.pdf">https://assets.ctfassets.net/an68im79xiti/5qIua7LwDKxoNEvHJnYqcB/c778731aa23ca5e51e1aa5d1c8941828/CG000286_ChromiumSingleCell3-_v3.1_Automation_UG_RevB.pdf</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Once the target cell concentration is obtained, place the cells on ice immediately. Cell suspension should be loaded onto the chip for library preparation as soon as possible, ideally within 30 min.</div>
<h3 id="sec3.3">Section 3: Generation of Single Cell RNA library using the 10× chromium platform</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼20 h</div>
<div class="note">
<span class="note-title">Note:</span> The experiments in Dang et al. used Chromium Single Cell 3′ v2 Chemistry, and libraries sequenced on the Illumina HiSeq 2500. The v2 chemistry kits are no longer commercially available from the manufacturer. Therefore, the methodology described below is for the 3′ kit. More detailed instructions are provided on the manufacturer’s website (<a href="https://support.10xgenomics.com/single-cell-gene-expression/library-prep/doc/user-guide-chromium-single-cell-3-reagent-kits-user-guide-v31-chemistry-dual-index">https://support.10xgenomics.com/single-cell-gene-expression/library-prep/doc/user-guide-chromium-single-cell-3-reagent-kits-user-guide-v31-chemistry-dual-index</a>). Here we describe the major steps.</div>
<div class="note">
<span class="note-title">Note:</span> Next GEM (Gel Bead-In EMulsions) is a technology that is built on a new chip architecture, the Next GEM Single Cell 3′ Reagent Kits v3.1 and Next GEM Chip G Single Cell Kits allow for a high number of detected genes, minimizes the number of barcodes that can be interpreted to come from more than one cell, 65% capture rate, and is compatible with both whole cells and nuclei. The kit contains Poly(dT) primer that enables the production of barcoded, full-length cDNA from poly-adenylated mDNA. This kit also allows add-on features such as cell surface protein, CRISPR screening, and cell multiplexing.</div>
<ol start="27">
<li>Prepare a Chromium Next GEM Chip G in a Chromium Next GEM Secondary Holder and dispense 50% glycerol into all unused wells.</li>
<li>Prepare the barcode master mix using RT Reagent B, Template Switch Oligo, Reducing Agent B, and RT Enzyme C. Add the cell suspension to the master mix (with/without water depending on the stock concentration) and mix by pipetting.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not add nuclease-free water directly to single-cell suspension. Add nuclease-free water to the Master Mix if needed.</div>
<ol start="29">
<li>Load the 70 μL final mixture into row 1 of the chip. Load 50 μL of vortexed gel beads into row 2 and 45 μL of partitioning oil in row 3.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Wait 30 s after adding the gel beads into the chip.</div>
<ol start="30">
<li>Place a gasket over the chip and attach it to the Secondary Holder. Load the chip into a Chromium Single Cell Controller instrument for GEM (Gel Bead-In EMulsions) generation. This process takes about 18 min.</li>
<li>Perform GEM-RT step in a thermal cycler. We used a C1000 Touch thermal cycler (BIO-RAD). The thermal cycler protocol is the following: 45 min at 53°C, 5 min at 85°C, and hold at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> cDNA generated at this stage can be stored at −20°C for up to a week.</div>
<ol start="32">
<li>Perform post-GEM clean up
<ol type="a">
<li>Add Recovery Agent and invert to mix. Biphasic mixture will separate with the partitioning oil in pink on the bottle.
<div class="note">
<span class="note-title">Note:</span> GEM generation occurs in narrow microfluidic channels so occasionally a clog may occur. Clogs can be noticed when less than 100 μL of gel emulsion with beads is recovered from the well and presence of heterogenous GEM emulsion (<a href="#fig2">Figure 2</a>). Clogs are generally caused by a few reasons: (1) contamination of sample with debris and presence of cell clumps which should be removed with strainers/filters before loading; (2) overloading the chip with excess cells; (3) improper handling of gel beads which form the clog; to minimize the risk of gel bead clogging, completely thaw and vortex the gel beads before processing, handle the beads in a sterile environment, and store the chips in a clean area free of dust. If a clog occurs, ideally the experiment should be repeated, but if the starting material is no longer available, it is possible to proceed with the protocol and generate cDNA, but the number of recovered cells will be lower.</div>
</li>
<li>Collect the clear aqueous phase on top
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1183-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Tips 3 and 8 are not homogeneous and have an excess of partitioning oil, indicating a clog</div>
<p>The other tips are successfully generated GEM emulsions.</p>
</figcaption>
</figure>
</li>
<li>Add a master mix containing Dynabead MyOne SILANE beads, Reducing Agent B, Cleanup Buffer and water.</li>
<li>Incubate for 10 min with pipet mixing at 5 min.</li>
<li>Place tubes on 10×Magnetic Separator (High position) until the solution clears.</li>
<li>Remove the supernatant and wash the beads with ethanol.</li>
<li>Elute with elution buffer off magnet</li>
<li>Put back on the magnet to pull down beads.</li>
<li>Transfer 35 μL of the sample to a new tube.</li>
</ol>
</li>
</ol>
<ol start="33">
<li>To amplify cDNA add Amp Mix and cDNA Primers to the sample and place in the thermal cycler using the following protocol.</li>
</ol>
<table id="undtbl6">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>1 min</td>
<td rowspan="3">25–35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>63°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Indefinitely</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The number of cycles depends on the desired targeted cell recovery. The goal is to generate sufficient mass for library construction.</div>
<ol start="34">
<li>Clean the cDNA
<ol type="a">
<li>Add SPRIselect Reagent containing magnet beads to make a final 0.6× solution of the reagent.</li>
<li>Incubate for 5 min and place on the magnet until the solution clears.</li>
<li>Remove the supernatant.</li>
<li>Wash beads with ethanol x2.</li>
<li>Place on the magnet and remove residual ethanol.</li>
<li>Elute cDNA with elution buffer.</li>
</ol>
</li>
<li>Perform quality control with a 1:10 diluted sample on the D1000 High Sensitivity kit, an automated electrophoresis platform. After cDNA amplification and SPRI cleanup, the size of the cDNA fragments should be between 400 to 9000 bases. A high quality library preparation should yield an abundance of cDNA within that size range (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1183-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Examples of high and poor quality control traces of cDNA (D1000 High Sensitivity kit, TapeStation<sup>TM</sup>, Agilent)</div>
<p>The bad quality sample yielded significantly less cDNA.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Agilent Bioanalyzer, Agilent TapeStation, and LabChip are the recommended methods for accurate quantification. The total yield will vary depending on the number of PCR cycles used during cDNA amplification, the number of cells targeted, and the RNA content per cell. It is also possible to use Qubit Fluorometer and Qubit dsDNA HS Assay Kit as an alternative QC method; in this case, it is necessary to multiply the cDNA concentration reported via the Qubit Fluorometer by the elution volume (40 μL) to obtain the total cDNA yield in nanograms.</div>
<div class="note">
<span class="note-title">Note:</span> Total cDNA yield is highly dependent on the cell type, cell preparation, and the number of cells loaded onto the chip. Low cDNA recovery may be result of low cell viability, RT inhibitory contaminants, sample clog, loss of cDNA during clean-up steps, or low total mRNA content per cell. If yield is low, add an extra 1–2 PCR cycles in the cDNA amplification step.</div>
<ol start="36">
<li>Proceed to construction of dual index library by starting with 10 μL of cDNA sample and performing fragmentation, end-repair, and A-tailing in a joint reaction.</li>
<li>Purify sample using a dual-sided 0.6×/0.8× bead:sample ratio with SPRIselect Reagent.</li>
<li>Perform adapter ligation by mixing each sample with a master mix consisting of ligation buffer, DNA ligase, and Adapter Oligo.</li>
<li>Perform post-ligation cleanup with 0.8× SPRIselect Reagent purification.</li>
<li>Perform sample indexing PCR using Amp Mix and Dual Index TT Set A followed by a final size-selection consisted of a dual-sided 0.6×/0.8× bead:sample ratio.</li>
<li>After library construction, run the final library either on a Bioanalyzer or TapeStation High Sensitivity kits at 1:10 dilution. The typical size distribution for final libraries is between 300 and 1000 bp. The trace should be similar to the high-grade sample in <a href="#fig4">Figure 4</a> below.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1183-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Examples of high and poor quality control traces of library sample obtained from D1000 High Sensitivity kit (TapeStation<sup>TM</sup>, Agilent)</div>
<p>In the low grade sample there is an extra 252 bp peak from contaminated cDNA. One additional SPRI cleaning cycle using a 1× ratio removed the contaminant.</p>
</figcaption>
</figure>
</li>
<li>Determine the average fragment size from the trace, this will be used as the insert size for library quantification.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Additional peaks may be present on the electropherogram trace. If there are peaks around 200 bp, this may indicate carryover adaptor/primer dimers in the final library and may affect sequencing results. If this occurs, perform an additional 1.0× SPRIselect cleanup prior to sequencing to remove adaptor dimers. Perform this step only if the total library yield is sufficient because an extra clean up may lose approximately 40% of the library material. If the total library yield is insufficient, we recommend performing a new library preparation from left-over amplified cDNA.</div>
<ol start="43">
<li>To determine the library concentration, perform qPCR using KAPA Library Quantification Kit for Illumina Platforms by following the manufacturer’s recommendations (<a href="https://www.n-genetics.com/products/1104/1024/17225.pdf">https://www.n-genetics.com/products/1104/1024/17225.pdf</a>) for qPCR-based quantification. Determine the concentration based on insert size derived from the Bioanalyzer/TapeStation trace in the previous step.</li>
<li>Dilute the libraries to 1.75 nM and pool in equal proportions.</li>
</ol>
<h3 id="sec3.4">Section 4: Sequencing</h3>
<div class="note">
<span class="note-title">Note:</span> Sequencing time depends on the flow cell type. The time below is based on the SP 100 cycles V1.5 flow cell on the NovaSeq 6000 Illumina sequencer. This flow cell can generate about ∼650–800 million reads and can fit up to four 3′GEX libraries for a targeted capture quantity of 10,000 per sample.</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 24 h</div>
<ol start="45">
<li>Sequence the library. We performed sequencing on the NovaSeq 6000 with the run parameters recommended by the 10× user guide: Read 1–28 cycles, i7 index – 10 cycles, i5 index – 10 cycles, Read 2–90 cycles.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> By default, 1% of PhiX was added to the sequencing run for quality control monitoring. 10× Genomics recommends sequencing a minimum of 20,000 reads/cell.</div>
<h3 id="sec3.5">Section 5: Analysis of sequencing data</h3>
<ol start="46">
<li>Using a 10× Chromium Cell Ranger with instructions from the tutorial available online (<a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorials">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorials</a>), generate three files: barcodes.tsv, genes.tsv, and matrix.mtx.</li>
<li>Use Seurat (<a href="#bib9">Satija et al., 2015</a>) (that runs in R or R Studio) or equivalent tool to analyze the data. Tutorials are available for Seurat at <a href="https://satijalab.org/seurat/articles/get_started.html">https://satijalab.org/seurat/articles/get_started.html</a>. Data integration to eliminate batch effect should be considered, especially when combining samples ran at different times.</li>
<li>Annotate immune cells using key markers for specific cell types or use SingleR (<a href="#bib1">Aran et al., 2019</a>) algorithm that will allow you to import data from the Immgen database as a library reference.</li>
<li>To identify homeostatic microglia that are present in normal brain tissue, import published data (<a href="#bib5">Li et al., 2019</a>; <a href="#bib2">Dang et al., 2021</a>) as a library reference. Alternatively, samples of homeostatic microglia (from a normal brain) and peripheral blood (from mouse without tumor) can be sequenced and batched together with tumor-associated macrophages. Please see <a href="#bib2">Dang et al. (2021)</a> for detailed methods on identification of tumor infiltrating monocytes and homeostatic microglia during analysis.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The application of this protocol significantly enriched myeloid cells from tumor-bearing cerebellum, which was dominated by microglia, macrophages and monocytes (<a href="#fig5">Figure 5</a>). The minimal use of surface markers (CD11b only) and the avoidance of enzymatic tissue digestion step in this protocol minimizes the potential variations in the enriched cell population between samples or experiments due to differences in cell-surface epitope density. We also corroborated our findings from scRNA-seq with flow cytometry to rule out potential selection bias in our protocol (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1183-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Comparison of scRNA-seq of myeloid cells isolated by the protocol described above (with 30/70 Percoll separation) with scRNA-seq of all brain leukocytes isolated using CD45 beads (with 40/80 Percoll separation)</div>
<p>The UMAP projection validates myeloid enrichment (left) with our method. Data analysis was performed with data integration and annotation using SingleR with Immgen database as the reference library.</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1183-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Flow cytometry of dissociated irradiated tumor show presence of CD11b+ cells contain a mixture of CD45-low cells (microglia) and CD45-high cells (macrophage/monocytes) and Ly6G+ neutrophils.</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>A limitation of this protocol is that the myeloid cells isolated are not strictly from the tumor microenvironment. We have tried to ensure tumor-association of these cells in our model system by (1) confirming the presence of tumor by MRI prior to euthanizing the mice and (2) identifying the tumor by gross morphology and dissecting out the tumor. Nonetheless, the findings from scRNA-seq of cells isolated in this manner can be validated within the tumor microenvironment by subsequent histology as we showed in <a href="#bib2">Dang et al. (2021)</a>. Emerging technologies in spatial genomics may eventually allow us to circumvent these generic issues of sequencing cells in suspension.</p>
<p>The limitations associated with the 10× chromium platform and library chemistry as well as Illumina sequencing have been extensively covered in the literature. However, the first two sections of this protocol are independent and can be applied to other single cell and/or sequencing platforms.</p>
<p>The computational analyses of the scRNA-seq data described here and in Dang et al. has its limitations and were selected based on the goals of the study. Nonetheless, there are wide varieties of analytical methods available depending on the objectives and expertise of the user.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>After homogenizing brain tumor tissue, cell clumps are visible (step 1).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Add DNAse to HBSS buffer before homogenizing.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>No Percoll gradient after centrifugation (step 7).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Check to make sure deceleration was set to 0.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Significantly lower myeloid cell staining on flow cytometry after Percoll gradient purification (step 13).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Make sure all myelin has been removed as it can interfere with antibody binding.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The flow of cell suspension through LS column is too slow or stops entirely (step 18).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>The column is likely clogged which can be due to excessive cells or clumping of cells in the sample. Retrieve sample that has not entered the column, dilute it with more MACs buffer, and pipet gently to separate cells. Apply sample to a newly prepped column.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Cell viability is much lower than expected after isolation and only debris is seen (step 25).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Use alternate buffer listed below with up to 10% FBS or up to 2% BSA to maintain cell health with little to no adverse downstream effects:</p>
<p>Dulbecco’s Phosphate-Buffered Saline (DPBS)</p>
<p>Hank’s Balanced Salt Solution (HBSS)</p>
<p>If cell viability cannot be maintained in one of these buffers, it is also possible to wash and resuspend (steps 23 and 24) in medias listed below:</p>
<p>Eagle’s Minimum Essential Medium (EMEM) + 10% FBS</p>
<p>Dulbecco’s Modified Eagle Medium (DMEM) + 10% FBS</p>
<p>Iscove’s Modified Eagle Medium (IMEM) + 10% FBS</p>
<p>Roswell Park Memorial Institute (RPMI) + 10% FBS</p>
<p>Ham’s F12 + 10% FBS</p>
<p>1:1 DMEM/F12 +10% FBS</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aran, D., Looney, A.P., Liu, L., Wu, E., Fong, V., Hsu, A., Chak, S., Naikawadi, R.P., Wolters, P.J., Abate, A.R., et al. (2019). Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. Nat. Immunol. <i>20</i>, 163-172. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref1">View at publisher</a></p>
<p id="bib2">Dang, M.T., Gonzalez, M.V., Gaonkar, K.S., Rathi, K.S., Young, P., Arif, S., Zhai, L., Alam, Z., Devalaraj, S., To, T.K.J., et al. (2021). Macrophages in SHH subgroup medulloblastoma display dynamic heterogeneity that varies with treatment modality. Cell Rep. <i>34</i>, 108917. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref2">View at publisher</a></p>
<p id="bib3">Gage, G.J., Kipke, D.R., and Shain, W. (2012). Whole animal perfusion fixation for rodent. J. Vis. Exp. <i>65</i>, 3564. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref3">View at publisher</a></p>
<p id="bib4">Hetze, S., Sure, U., Schedlowski, M., Hadamitzky, M., and Barthel, L. (2021). Rodent models to analyze the glioma microenvironment. ASN Neuro <i>13</i>. <a class="external-link" href="https://doi.org/10.1177/17590914211005074">https://doi.org/10.1177/17590914211005074</a>.</p>
<p id="bib5">Li, Q., Cheng, Z., Zhou, L., Darmanis, S., Neff, N.F., Okamoto, J., Gulati, G., Bennett, M.L., Sun, L.O., Clarke, L.E., et al. (2019). Developmental heterogeneity of microglia and brain myeloid cells revealed by deep single-cell RNA sequencing. Neuron <i>101</i>, 207-223. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref5">View at publisher</a></p>
<p id="bib6">Li, Z. and Langhans, S.A. (2021). In vivo and ex vivo pediatric brain tumor models: an overview. Front. Oncol. <i>11</i>, 620831. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref6">View at publisher</a></p>
<p id="bib7">Moore, T.C. (2012). Counting cell with hemocytometer. Protocols.io. <a class="external-link" href="https://www.protocols.io/view/counting-cells-with-hemocytometer-nxsdfne">https://www.protocols.io/view/counting-cells-with-hemocytometer-nxsdfne</a></p>
<p id="bib8">Nguyen, A., Khoo, W.W., Moran, I., Croucher, P.I., and Phan, T.G. (2018). Single cell RNA sequencing of rare immune cell populations. Front. Immunol. <i>9</i>, 1553. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref8">View at publisher</a></p>
<p id="bib9">Satija, R., Farrell, J.A., Gennert, D., Schier, A.F., and Regev, A. (2015). Spatial reconstruction of single- cell gene expression data. Nat. Biotechnol. <i>33</i>, 495-502. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref9">View at publisher</a></p>
<p id="bib10">Shapiro, E., Biezuner, T., and Linnarsson, S. (2013). Single-cell sequencing-based technologies will revolutionize whole-organism science. Nat. Rev. Genet. <i>14</i>, 618-630. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref10">View at publisher</a></p>
<p id="bib11">Simeonova, I. and Huillard, E. (2014). In vivo models of brain tumors: roles of genetically engineered mouse models in understanding tumor biology and use in preclinical studies. Cell Mol. Life Sci. <i>71</i>, 4007-4026. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref11">View at publisher</a></p>
<p id="bib12">Wetmore, C., Eberhart, D.E., and Curran, T. (2001). Loss of p53 but not ARF accelerates medulloblastoma in mice heterozygous for patched. Cancer Res. <i>61</i>, 513-516. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref12">View at publisher</a></p>
<p id="bib13">Zhang, M.J., Ntranos, V., and Tse, D. (2020). Determining sequencing depth in a single-cell RNA-seq experiment. Nat. Commun. <i>11</i>, 774. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00663-8/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Malay Haldar (<a href="mailto:mhaldar@pennmedicine.upenn.edu">mhaldar@pennmedicine.upenn.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The accession number for the scRNA-seq data reported in this paper is GEO: GSE166691.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> NCI R37CA234027 (M.H.), <a href="https://doi.org/10.13039/100000861">Burroughs Wellcome Fund</a> CAMS award (M.H.), <a href="https://doi.org/10.13039/100000884">Cancer Research Institute</a> CRI3881 (M.H.), T32NS007413 Training Program in Neurodevelopmental Disabilities (M.T.D.), 1K12NS098482 Child Neurologist Career Development Training Program (M.T.D.), Wylie’s Day Foundation (M.T.D.), <a href="https://doi.org/10.13039/100003156">Musella Foundation</a> (M.T.D.), and Gray Matters Foundation (M.T.D.). We thank Dr. Ashwani Kesarwani for help with the pictures. Graphical abstract created with BioRender.</p>
<h3>Author contributions</h3>
<p>Conception and Design, M.T.D. and M.H.; Methodology, M.T.D., F.M., and M.H.; Data Acquisition, M.T.D.; Analysis and Data Interpretation, M.T.D., F.M., and M.H.; Writing – Review &amp; Editing, M.T.D., F.M., and M.H.; Supervision, M.T.D. and M.H.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>