<article>
<section>
<h1 id="header">Isolation of human and murine hematopoietic stem cells for DNA damage and DNA repair assays</h1>
<p><time datetime="2021-10-01">Published: October 1, 2021</time></p>
<p>Alfredo Rodríguez,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Jessica Filiatrault,<sup><a href="#aff1">1</a></sup> Patricia Flores-Guzmán,<sup><a href="#aff3">3</a></sup> Héctor Mayani,<sup><a href="#aff3">3</a></sup> Kalindi Parmar,<sup><a href="#aff1">1</a></sup> and Alan D. D’Andrea<sup><a href="#aff1">1</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Radiation Oncology and Center for DNA Damage and Repair, Dana Farber Cancer Institute, Harvard Medical School, Boston, MA 02215, USA</p>
<p id="aff2"><sup>2</sup>Departamento de Medicina Genómica y Toxicología Ambiental, Instituto de Investigaciones Biomédicas, Universidad Nacional Autónoma de México, Ciudad Universitaria, Apartado Postal 70228, México 04510, México</p>
<p id="aff3"><sup>3</sup>Unidad de Investigación Médica en Enfermedades Oncológicas, Hospital de Oncología, Centro Médico Nacional Siglo XXI, Instituto Mexicano del Seguro Social, Mexico City 06720, Mexico</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:alfredo.rodriguez@iibiomedicas.unam.mx">alfredo.rodriguez@iibiomedicas.unam.mx</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:alan_dandrea@dfci.harvard.edu">alan_dandrea@dfci.harvard.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100846">10.1016/j.xpro.2021.100846</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Hematopoietic stem and progenitor cells (HSPCs) reside in the bone marrow and supply blood cells. Efficient methods for isolation of HSPCs are required. Here, we present protocols for the isolation of human and murine HSPCs using manual and FACS-assisted techniques. Isolated HSPCs can be used for downstream applications, including colony forming unit assays and DNA damage and repair assays.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref7">Rodríguez et al. (2021a)</a> and (<a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref8">2021b</a>).</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detailed protocol for isolating human and mouse hematopoietic stem cells</li>
<li>Procedures for plating and quantification of hematopoietic colony forming unit assay</li>
<li>Protocol for assessing DNA damage using the comet assay in hematopoietic stem cells</li>
<li>Protocol for assessing DNA damage using immunofluorescence in hematopoietic stem cells</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1042-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Hematopoietic stem and progenitor cells (HSPC) are multipotent cells that reside within the bone marrow niche. HSPCs are responsible for the production of blood cells during the lifespan of an individual; however, they are limited in number and remain in a quiescent state. To achieve the blood cell requirements of an organism, the HSPCs must enter the cell cycle and produce progenitors that will complete multiple rounds of division, as well as produce the precursors that will mature into functional cells. However, it is critical that the HSPC pool is maintained to avoid exhaustion (<a href="#bib10">Seita and Weissmann, 2010</a>).</p>
<p>Multiple stimuli can modify the behavior and size of the HSPC pool. For example, infection and inflammatory stimuli can push HSPCs out of quiescence and exhaust the HSPC pool (<a href="#bib2">Lin et al., 2014</a>; <a href="#bib13">Walter et al., 2015</a>; <a href="#bib3">Matatall et al., 2016</a>). HSPCs are also highly sensitive to DNA damaging agents such as those used for chemotherapy (<a href="#bib12">Tang et al., 2019</a>). HSPCs are highly fragile in inherited bone marrow failure syndromes: genetic disorders in which patients develop childhood-onset aplastic anemia due to attrition of the HSPC pool. Patients with inherited bone marrow failure syndromes must be monitored for the emergence of hematopoietic clones that can transform into myelodysplastic syndrome and acute myeloid leukemia. The most common inherited bone marrow failure syndromes are Fanconi anemia, dyskeratosis congenita, Blackfan-Diamond anemia and Schwachman-Diamond syndrome (<a href="#bib11">Shimano et al., 2021</a>). Mouse models are widely used for the study of bone marrow biology, and some genetic models exist for the study of, for example, Fanconi anemia, including <i>Fanca</i><sup>−/−</sup>, <i>Fancg</i><sup>−/−</sup> and <i>Fancd2</i><sup>−/−</sup> mice. Fanconi anemia mouse models allow for the study of its DNA repair defect and how the HSPCs respond to DNA-damaging stimuli and chemical agents (<a href="#bib5">Parmar et al., 2009</a>).</p>
<p>In this work, we describe methods for the efficient manual and FACS-assisted isolation of HSPCs from human and mouse samples using surface markers, which we have successfully used for the study of HSPCs from patients with Fanconi anemia and Fanconi anemia mouse models, which are known to have a reduced HSPCs number. Manual enrichment allows for the rapid isolation of the population of interest; however, we also present a more detailed FACS purification protocol using the LSK and SLAM markers in mouse, if a pure population is sought (<a href="#bib1">Kiel et al., 2005</a>; <a href="#bib14">Yilmaz et al., 2006</a>). We also present standardized methods for performing downstream DNA damage and repair assays with the isolated cells. The comet assay allows for the detection of DNA damage in cells using an electrophoretic field (<a href="#bib4">Olive and Banáth, 2006</a>), whereas detection of γH2AX foci by immunofluorescence allows quantifying DNA damage, usually double strand breaks (<a href="#bib15">Zhang et al., 2016</a>). We present optimized protocols to efficiently perform these assays with small cell amounts as input.</p>
<p>For mouse studies, 2–6-month-old C57BL/6 <i>Fancd2</i><sup><i>−/−</i></sup>and WT mice were used (<a href="#bib5">Parmar et al., 2009</a>). Housing and handling of all mice, as well as experimental procedures were approved by the Animal Care and Use Committee of the Dana Farber Cancer Institute and performed according to its guidelines. Mice were euthanized by CO<sub>2</sub> asphyxiation at the experimental time point. For human samples, written informed consent was received from participants. Use of patients’ samples was approved by the ethics and research committees from Dana Farber Cancer Institute and Instituto Mexicano del Seguro Social. Healthy bone marrow was purchased from Lonza.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Biotin mouse lineage panel, diluted 1:40∗</td>
<td>BD Biosciences</td>
<td>Cat# 559971RRID: AB_10053179</td>
</tr>
<tr>
<td>PE streptavidin; diluted 1:100</td>
<td>BD Biosciences</td>
<td>Cat# 554061RRID: 554061</td>
</tr>
<tr>
<td>PE-Cy7 rat anti-mouse Ly-6A/E (Sca1) (clone D7); diluted 1:40∗</td>
<td>BD Biosciences</td>
<td>Cat# 558162; RRID:AB_647253</td>
</tr>
<tr>
<td>APC anti-mouse CD117 (c-kit) (clone ACK2); diluted 1:40∗</td>
<td>BioLegend</td>
<td>Cat# 135108; RRID:AB_2028407</td>
</tr>
<tr>
<td>Pacific Blue anti-mouse CD150 (clone TC15-12F12.2); diluted 1:40∗</td>
<td>BioLegend</td>
<td>Cat# 115924; RRID:AB_2270307</td>
</tr>
<tr>
<td>APC-Cy7 anti-mouse CD48 (clone HM48-1); diluted 1:40∗</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 47-0481-82; RRID:AB_2573962</td>
</tr>
<tr>
<td>Rabbit anti-human/mouse phospho-histone H2A.X (Ser139) (20E3); diluted 1:1000</td>
<td>Cell Signaling Technology</td>
<td>Cat# 9718S; RRID:AB_2118009</td>
</tr>
<tr>
<td>Donkey anti-Rabbit IgG, Alexa Fluor 488; diluted 1:500</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A21206;RRID:AB_2535792</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human whole bone marrow and blood from healthy donors</td>
<td>Lonza</td>
<td>Cat# 1M-105</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>FLT3-ligand, human</td>
<td>PeproTech</td>
<td>Cat# 300-19</td>
</tr>
<tr>
<td>IL-6, human</td>
<td>PeproTech</td>
<td>Cat# 200-06</td>
</tr>
<tr>
<td>SCF, human</td>
<td>PeproTech</td>
<td>Cat# 300-07</td>
</tr>
<tr>
<td>TPO, human</td>
<td>PeproTech</td>
<td>Cat# 300-18</td>
</tr>
<tr>
<td>SCF, mouse∗</td>
<td>PeproTech</td>
<td>Cat# 250-03-10UG</td>
</tr>
<tr>
<td>TPO, mouse∗</td>
<td>PeproTech</td>
<td>Cat# 315-14-10UG</td>
</tr>
<tr>
<td>Mitomycin C</td>
<td>Sigma-Aldrich</td>
<td>Cat# M0503CAS:50-07-7</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat# T8787CAS:9002-93-1</td>
</tr>
<tr>
<td>Sodium hydroxide (NaOH) pellets</td>
<td>VWR Life Science</td>
<td>Cat#0583-500GCAS: 1310-73-2</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Sigma-Aldrich</td>
<td>Cat# A9647CAS: 9048-46-8</td>
</tr>
<tr>
<td>Ammonium chloride solution</td>
<td>STEMCELL Technologies</td>
<td>Cat# 07800CAS: 12125-02-9</td>
</tr>
<tr>
<td>EDTA (0.5 M) pH 8</td>
<td>Boston BioProducts</td>
<td>Cat# BM150CAS: 6381-92-6</td>
</tr>
<tr>
<td>Methanol</td>
<td>Fisher Scientific</td>
<td>Cat# A412-4CAS: 67-56-1</td>
</tr>
<tr>
<td>70% Ethanol</td>
<td>Fisher Scientific</td>
<td>Cat# BP82011CAS: 64-17-5</td>
</tr>
<tr>
<td>16% Paraformaldehyde (formaldehyde) aqueous solution</td>
<td>Electron Microscopy Sciences</td>
<td>Cat# 15710CAS: 50-00-0</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>Fancd2</i><sup><i>-/-</i></sup> and WT mice. C57BL/6 background, 2–6-month-old. Any gender.</td>
<td><a href="#bib5">Parmar et al., 2009</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo, version 10.5.3</td>
<td>N/A</td>
<td><a href="https://www.flowjo.com/">https://www.flowjo.com/</a></td>
</tr>
<tr>
<td>OpenComet plugin</td>
<td>N/A</td>
<td><a href="http://www.cometbio.org/">http://www.cometbio.org/</a></td>
</tr>
<tr>
<td>ImageJ</td>
<td>N/A</td>
<td><a href="http://fiji.sc">http://fiji.sc</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>EasySep Kit, human</td>
<td>STEMCELL Technologies</td>
<td>Cat# 19356</td>
</tr>
<tr>
<td>EasySep Buffer</td>
<td>STEMCELL Technologies</td>
<td>Cat# 20144</td>
</tr>
<tr>
<td>EasySep Magnet</td>
<td>STEMCELL Technologies</td>
<td>Cat# 18000</td>
</tr>
<tr>
<td>MidiMACS Separator</td>
<td>Miltenyi</td>
<td>Cat# 130-042-302</td>
</tr>
<tr>
<td>MACS MultiStand</td>
<td>Miltenyi</td>
<td>Cat# 130-042-303</td>
</tr>
<tr>
<td>CD34 MicroBead Kit, human</td>
<td>Miltenyi</td>
<td>Cat# 130-046-702</td>
</tr>
<tr>
<td>FcR Blocking Reagent, human</td>
<td>Miltenyi</td>
<td>Cat# 130-059-901</td>
</tr>
<tr>
<td>Lineage Cell Depletion Kit, mouse∗</td>
<td>Miltenyi</td>
<td>Cat# 130-090-858</td>
</tr>
<tr>
<td>LS Column Miltenyi</td>
<td>Miltenyi</td>
<td>Cat# 130-042-401</td>
</tr>
<tr>
<td>MS Column Miltenyi</td>
<td>Miltenyi</td>
<td>Cat# 130-042-201</td>
</tr>
<tr>
<td>StemSpan SFEM II</td>
<td>STEMCELL Technologies</td>
<td>Cat# 09655</td>
</tr>
<tr>
<td>MethoCult H4434 Classic</td>
<td>STEMCELL Technologies</td>
<td>Cat# 04434</td>
</tr>
<tr>
<td>Methocult GF M3434 methylcellulose∗</td>
<td>STEMCELL Technologies</td>
<td>Cat# 03444</td>
</tr>
<tr>
<td>PBS pH 7.4</td>
<td>Gibco</td>
<td>Cat# 100-10-023</td>
</tr>
<tr>
<td>SYBR-Green dye</td>
<td>Invitrogen</td>
<td>Cat# S7567</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Sigma-Aldrich</td>
<td>Cat# F2442</td>
</tr>
<tr>
<td>Hanks balanced salt solution (HBSS)</td>
<td>Lonza</td>
<td>Cat# 10-547F</td>
</tr>
<tr>
<td>HEPES Buffer</td>
<td>Fisher Scientific</td>
<td>Cat# BP299-100</td>
</tr>
<tr>
<td>Penicillin-streptomycin (PenStrep)</td>
<td>Gibco</td>
<td>Cat# 15140-122</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Gibco</td>
<td>Cat# 25030081</td>
</tr>
<tr>
<td>70 μm Cell strainer</td>
<td>Fisher Scientific</td>
<td>Cat# 087712</td>
</tr>
<tr>
<td>10 mL Eccentric tip syringe</td>
<td>BD</td>
<td>Cat# 305482</td>
</tr>
<tr>
<td>18G 1 Inch needle</td>
<td>BD</td>
<td>Cat# 305195</td>
</tr>
<tr>
<td>500 mL Bottle Top Vacuum Filter, 0.45 μm</td>
<td>Corning</td>
<td>Cat# 430514</td>
</tr>
<tr>
<td>CometAssay Single Cell Gel Electrophoresis Assay Kit</td>
<td>Trevigen</td>
<td>Cat# 4250-050-K</td>
</tr>
<tr>
<td>Normal goat serum (10%)</td>
<td>Life Technologies</td>
<td>Cat# 50062Z</td>
</tr>
<tr>
<td>SuperFrost Plus Slides</td>
<td>Fisher Scientific</td>
<td>Cat# 12-550-15</td>
</tr>
<tr>
<td>ProLong Gold Antifade Mountant with DAPI</td>
<td>Life Technologies</td>
<td>Cat# P36941</td>
</tr>
<tr>
<td>Double Cytofunnel. Disposable sample chambers with white filter cards</td>
<td>Thermo Scientific</td>
<td>Cat# 5991039</td>
</tr>
<tr>
<td>Round-Bottom Polystyrene Test Tubes with Cell Strainer Snap Cap, 5mL</td>
<td>Fisher Scientific</td>
<td>Cat# 08-771-23</td>
</tr>
<tr>
<td>SmartDish</td>
<td>STEMCELL Technologies</td>
<td>Cat# 27370</td>
</tr>
<tr>
<td>Non-tissue culture treated plate, 12 well flat bottom</td>
<td>Falcon</td>
<td>Cat# 351143</td>
</tr>
<tr>
<td>STEMvision</td>
<td>STEMCELL Technologies</td>
<td>Cat# 22000</td>
</tr>
<tr>
<td>Cytospin 3</td>
<td>Shandon Scientific</td>
<td>Cat# 74000102</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗ Indicates reagents exclusive for mouse experiments</p>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>HBSS++</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sterile filtered FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>HEPES (1 M)</td>
<td>25 mM</td>
<td>12.5 mL</td>
</tr>
<tr>
<td>PenStrep (10,000 U/mL)</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>HBSS</td>
<td>1<b>×</b>
</td>
<td>432.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare under sterile conditions, store at 4°C; maximum storage time 1 month.</p>
<table id="undtbl2">
<caption>Column buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EDTA (0.5 M)</td>
<td>2 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>0.5%</td>
<td>2.5 g</td>
</tr>
<tr>
<td>PBS</td>
<td>1<b>×</b>
</td>
<td>500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>502 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Stir on a magnetic stir plate until completely dissolved and then filter through a 0.45 μm filter. Keep sterile.<br> Store at 4°C; maximum storage 1 month.</p>
<p>Cell collection media (for human HSPCs)</p>
<p>Prepare stock solutions of SCF, Flt3-ligand, TPO and IL-6 in StemSpan SFEM II media. Store aliquots at −20°C.</p>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>StemSpan SFEM II</td>
<td>1<b>×</b>
</td>
<td>848 μL</td>
</tr>
<tr>
<td>SCF (2 μg/mL)</td>
<td>100 ng/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>Flt3-ligand (2 μg/mL)</td>
<td>100 ng/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>TPO (2 μg/mL)</td>
<td>100 ng/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>IL-6 (10 μg/mL)</td>
<td>20 ng/mL</td>
<td>2 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh each time before use.</p>
<table id="undtbl4">
<caption>Cell collection media (for mouse HSPCs)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>StemSpan SFEM II</td>
<td>1<b>×</b>
</td>
<td>890 μL</td>
</tr>
<tr>
<td>SCF (2 μg/mL)</td>
<td>100 ng/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>TPO (2 μg/mL)</td>
<td>100 ng/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>1%</td>
<td>10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh each time before use.</p>
<table id="undtbl5">
<caption>Immunofluorescence blocking buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10% Triton X-100 diluted in PBS; filtered</td>
<td>0.3%</td>
<td>240 μL</td>
</tr>
<tr>
<td>10% Normal Goat Serum</td>
<td>10%</td>
<td>8 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>8.240 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh each time before use.</p>
<table id="undtbl6">
<caption>IF buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1% BSA diluted in PBS; filtered</td>
<td>1%</td>
<td>9 mL</td>
</tr>
<tr>
<td>10% Triton X-100 diluted in PBS; filtered</td>
<td>0.3%</td>
<td>270 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>9.270 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for a maximum of 48 h.</p>
<table id="undtbl7">
<caption>Alkaline Unwinding Solution, pH &gt;13</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaOH Pellets</td>
<td>200 mM</td>
<td>0.4 g</td>
</tr>
<tr>
<td>200 mM EDTA</td>
<td>1 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>1<b>×</b>
</td>
<td>49.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh each time before use. Allow the solution to cool down on the benchtop.</p>
<div class="critical">
<span class="critical-title">Critical:</span> This solution will warm up during preparation. Handle with care. Wear heat-resistant gloves and eye protection while handling.</div>
<table id="undtbl8">
<caption>Alkaline Electrophoresis Solution, pH &gt;13</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaOH Pellets</td>
<td>200 mM</td>
<td>8 g</td>
</tr>
<tr>
<td>500 mM EDTA, pH 8</td>
<td>1 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>Z</td>
<td>1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh each time. Cool to 4°C before use.</p>
<div class="critical">
<span class="critical-title">Critical:</span> This solution will warm up during preparation. Handle with care. Wear heat-resistant gloves and eye protection during while handling.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Here, we describe protocols for the isolation of human and murine HSPCs. Multiple assays can be started simultaneously with the isolated HSPCs. We present protocols for downstream applications using the isolated HSPCs, primarily focused on assays for CFU capacity and assessment of DNA damage.</p>
<h3 id="sec3.1">Isolation of human HSPCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 h</div>
<p>This protocol is optimized for the efficient isolation of Lin<sup>-</sup> cells from as little as 2 mL of bone marrow. The Lin<sup>-</sup> population is enriched in CD34<sup>+</sup> cells. Using this protocol, we were able to efficiently isolate Lin<sup>-</sup> cells from patients with Fanconi anemia, a bone marrow failure syndrome in which HSPCs are scarce. Using this protocol, we typically obtain 6.0<b>×</b>10<sup>5</sup> - 1.0<b>×</b>10<sup>6</sup> Lin<sup>-</sup> cells per mL of healthy bone marrow, or 1.0<b>×</b>10<sup>5</sup> - 2.0<b>×</b>10<sup>5</sup> Lin<sup>-</sup> cells per mL of Fanconi anemia bone marrow (<a href="#fig1">Figure 1</a>A).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1042-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Strategies for enrichment of HSPC</div>
<p>(A) Human Lin<sup>-</sup> cells can be enriched using a magnetic stand. <i>Left.</i> Cells are incubated with the progenitor cell enrichment cocktail followed by incubation with magnetic particles and incubation in the magnetic stand. <i>Right.</i> Lin<sup>+</sup> cells bound by the antibodies of the enrichment cocktail and by the magnetic particles are attracted to the tube wall by the magnetic field. Lin<sup>-</sup> cells remain in suspension and can be recovered by simply pouring off the liquid into a new tube. The Lin<sup>+</sup> cells remain in the original tube.</p>
<p>(B) Mouse and human HSPCs can be enriched using columns for magnetic cell separation. Cells are incubated with cell enrichment cocktail followed by incubation with magnetic beads. <i>Left.</i> Enrichment of mouse Lin<sup>-</sup> cells is performed with the LS Miltenyi columns; enrichment of human CD34<sup>+</sup> cells is performed with the MS Miltenyi columns. The column is mounted into the MidiMACS separator on the MACS MultiStand and prepared by adding column buffer. <i>Center.</i> The cell suspension is added to the top of the column. <i>Right.</i> The cell suspension is left to flow through the column via gravity flow. Cells bound by the magnetic particles remain bound to the column in the magnetic field, whereas unbound cells are recovered into a clean tube.</p>
<p>(C) Mouse LT-HSCs can be purified from the enriched population of Lin<sup>-</sup> cells using FACS and a 5-flurochrome strategy based on the SLAM markers. <i>Left.</i> Detection of Lin<sup>-</sup> cells. <i>Center.</i> LSK cells within the Lin- population are recognized by being c-Kit+ Sca+. <i>Right.</i> LT-HSCs within the LSK population are recognized by being CD150+ CD48-</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> All steps should be performed under sterile conditions.</div>
<ol>
<li>Plasma removal:
<ol type="a">
<li>Centrifuge the whole bone marrow sample in a 15 mL conical tube at 2000 <b>×</b> <i>g</i> 4°C for 20 min. After centrifugation, remove the plasma with a pipette without disturbing the cells.</li>
<li>Plasma can be aliquoted and stored long-term at −80°C if cytokine analysis of the bone marrow is needed.</li>
</ol>
</li>
<li>Red blood cell lysis and isolation of mononuclear cells (MNCs):
<ol type="a">
<li>Take 2 mL of bone marrow (after plasma removal) in a 50 mL tube and add 8 mL of ammonium chloride solution. Mix well with a pipette and then incubate for 10–15 min on ice.
<ol type="i">
<li>Generally, the volume of ammonium chloride required is approximately 4<b>×</b> the volume of the bone marrow.</li>
</ol>
</li>
<li>Wash by directly adding 40 mL of 1<b>×</b> PBS and centrifuge at 500 <b>×</b> <i>g</i> for 5 min.</li>
<li>Aspirate supernatant. Repeat the above steps until the cell pellet appears white or with very few red cells (usually 2 more times) by adding 4 mL of ammonium chloride directly to the cells and incubating for 5–10 min on ice. Wash every incubation of ammonium chloride by directly adding 40 mL of 1<b>×</b> PBS and centrifuge at 500 <b>×</b> <i>g</i> for 5 min.</li>
<li>Aspirate the supernatant.</li>
</ol>
</li>
<li>Enrichment of Lin<sup>-</sup> cells (we typically obtain 6.0<b>×</b>10<sup>5</sup>–1.0<b>×</b>10<sup>6</sup> Lin<sup>-</sup> cells per mL of healthy bone marrow or 1.0<b>×</b>10<sup>5</sup>–2.0<b>×</b>10<sup>5</sup> Lin<sup>-</sup> cells per mL of Fanconi anemia bone marrow):
<ol type="a">
<li>Resuspend the MNC pellet in 1 mL EasySep Buffer and transfer to a 5 mL FACS tube.</li>
<li>Add 50 μL of human progenitor cell enrichment cocktail (included in the EasySep kit) for every 1 mL of cell suspension. Mix with a pipette and incubate for 15 min at 18°C–21°C.</li>
<li>Add 50 μL of magnetic particles (included in the EasySep kit) for every 1 mL of cell suspension. Mix with a pipette and incubate for 15 min at 18°C–21°C (no rotation/rocking is needed).</li>
<li>Add 2.5 mL EasySep Buffer and place the tube in the EasySep Magnet. Incubate for 15 min at 18°C–21°C (no rotation/rocking is needed).</li>
<li>Keep the tube in the magnet and pour out the supernatant into a new tube (the supernatant contains the Lin<sup>-</sup> cells) (<a href="#fig1">Figure 1</a>A).</li>
</ol>
</li>
<li>The cells that do not bind to the magnetic beads are Lin<sup>-</sup> cells; these cells are ready for further analyses.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This a negative selection protocol; therefore Lin<sup>-</sup> cells will remain in the supernatant, untouched by the magnetic beads. The unwanted fraction of mature cells will bind to the magnetic beads.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Instead of the Lin<sup>-</sup> enrichment kit, a CD34<sup>+</sup> MicroBead kit with FcR Blocking reagent can be used.</div>
<h3 id="sec3.2">Isolation of mouse HSPCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 h</div>
<p>This protocol is optimized for the efficient isolation of HSPCs from wild type mice and mice with deficiency of Fanconi anemia genes. This protocol is also useful for the isolation of HSPCs after stress conditions that induce bone marrow failure in Fanconi anemia mice. We first describe a manual enrichment of Lin<sup>-</sup> cells using a column system, followed by staining of the Lin<sup>-</sup> fraction for FACS-mediated purification of LSK and LT-HSCs using the SLAM markers (<a href="#bib1">Kiel et al., 2005</a>; <a href="#bib14">Yilmaz et al., 2006</a>) (<a href="#fig1">Figure 1</a>B).</p>
<div class="note">
<span class="note-title">Note:</span> All steps should be performed under sterile conditions.</div>
<div class="note">
<span class="note-title">Note:</span> This protocol is applied to bone marrow pooled from 3 mice.</div>
<ol start="5">
<li>Bone marrow harvest:
<ol type="a">
<li>Mice euthanasia:
<ol type="i">
<li>Mice (8–10 weeks old) are euthanized in a CO<sub>2</sub> chamber using 2 liters per minute (Lpm) CO<sub>2</sub> for 5 min (according to the guidelines set by the Animal Care and Use Committee at the Dana Farber Cancer Institute).</li>
</ol>
</li>
<li>Bone marrow flushing from mouse hind limbs:
<ol type="i">
<li>After euthanasia, place the mouse in a ventral position and spray with 70% ethanol for sterilization.</li>
<li>Using a scalpel, remove the muscles and tissue around the legs.</li>
<li>Collect the femur and tibia by making a cut in the pelvic-hip joint.
<div class="note">
<span class="note-title">Note:</span> The muscular tissue surrounding the femur and tibia can be sticky. After the initial removal of the muscles with a scalpel, paper napkins can be used to help with the manual removal of the remaining muscles.</div>
</li>
<li>Cut the femur and tibia at both ends.</li>
<li>Using a 27-gauge needle, fill a 10 mL syringe with HBSS++ flushing media.</li>
<li>Insert the needle with HBSS++ flushing media into the hollow opening of the bone and flush the bone marrow into a 50 mL Falcon tube.</li>
<li>Continue to flush until the bone looks transparent/white.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Enrichment of mouse Lin<sup>-</sup> cells:
<div class="note">
<span class="note-title">Note:</span> The enrichment is performed using the Lineage cell depletion kit from Miltenyi Biotec.</div>
<ol type="a">
<li>Make a single cell suspension by pipetting up and down with a 10 mL pipette. Next, mix the cells using a BD 10 mL eccentric tip syringe and BD 18G 1 inch needle. Avoid foaming by releasing the cell suspension on the side of the tube.</li>
<li>Filter the cell suspension through a 70 μm cell strainer into a clean 50 mL conical tube.</li>
<li>Centrifuge at 500 <b>×</b> <i>g</i> for 5 min, then aspirate the supernatant and resuspend the cells in 600 μL of column buffer. Mix with a pipette.</li>
<li>Add 150 μL of the Lineage biotin antibody cocktail (from the Lineage cell depletion kit). Incubate 15 min on ice. Mix with a pipette.</li>
<li>Add 450 μL of column buffer, then add 300 μL of the anti-biotin microbeads (from the Lineage cell depletion kit). Mix with a pipette and incubate 15 min on ice.</li>
<li>Wash by adding 10 mL of column buffer, then centrifuge at 500 <b>×</b> <i>g</i> for 5 min. Aspirate the supernatant.</li>
<li>While the cells are in the centrifuge, set up the LS column and MidiMACS separator on the MACS MultiStand. Place a 15 mL conical tube underneath the column. Add 1 mL of column buffer to the LS column, and allow it to flow through and empty into the 15 mL conical tube by gravity flow (<a href="#fig1">Figure 1</a>B).</li>
<li>Resuspend the cell pellet in 1 mL column buffer and add it to the LS column.</li>
<li>Wash the column twice with 1 mL of column buffer.
<ol type="i">
<li>The Lin<sup>-</sup> cells are now in the 15 mL conical tube (the Lin<sup>+</sup> cells remain attached to the column) and can be used for further analyses. From pooling the bone marrow of three adult wild type mice, we obtain on average 3<b>×</b>10<sup>6</sup> Lin<sup>-</sup> cells, whereas from three adult Fanconi anemia mice we obtain 2<b>×</b>10<sup>6</sup> Lin<sup>-</sup> cells.</li>
<li>To elute Lin<sup>+</sup> cells: add 1 mL column buffer to the column, then use the plunger to expel the cells attached to the beads into a new 15 mL conical tube.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Isolation of LSK cells and LT-HSC using FACS (<a href="#fig1">Figure 1</a>C):
<div class="note">
<span class="note-title">Note:</span> The Lin<sup>-</sup> cells obtained in the above protocol are ready for downstream applications. Use the following FACS staining protocol if purification of LSK cells or LT-HSC is desired.</div>
<div class="note">
<span class="note-title">Note:</span> The following protocol purifies LT-HSCs using the SLAM markers (<a href="#bib1">Kiel et al., 2005</a>; <a href="#bib14">Yilmaz et al., 2006</a>). If using other markers to isolate LT-HSCs, different antibodies should be used.</div>
<ol type="a">
<li>Resuspend the enriched Lin<sup>-</sup> cells (from the previous step) in 1 mL of HBSS++ in a 15 mL conical tube and count them. From a single wild type mouse, we typically expect 1<b>×</b>10<sup>6</sup> Lin<sup>-</sup> cells.
<div class="critical">
<span class="critical-title">Critical:</span> Single-color controls for FACS compensation are critical in this protocol.</div>
<div class="note">
<span class="note-title">Note:</span> The following single-color controls are used in this protocol:</div>
Unstained (autofluorescence)<br> PE single-color control<br> PE Cy7 single-color control<br> APC single-color control<br> Pacific Blue single-color control APC-Cy7 single-color control
<div class="note">
<span class="note-title">Note:</span> In this protocol, we prepare single-color controls using the cells obtained from the previous Lin- enrichment.</div>
</li>
<li>Take 1.5<b>×</b>10<sup>5</sup>–2<b>×</b>10<sup>5</sup> Lin<sup>-</sup> cells for every single-color control and resuspend them in 100 μL of HBSS++. Keep your experimental samples in 1 mL of HBSS++.
<div class="note">
<span class="note-title">Note:</span> <a href="#tbl1">Table 1</a> demonstrates how single-color controls are generated and how experimental samples are stained for FACS. See also text.</div>
</li>
<li>Add 2.5 μL of each biotin-labeled lineage antibody to the PE single-color control: Biotin Hamster anti-Mouse CD3e, Biotin Rat anti-Mouse CD11b, Biotin Rat anti-Mouse CD45R, Biotin Rat anti-Mouse Ly-6G and Ly-6C, Biotin Rat anti-Mouse TER-119/Erythroid cells (all from the Biotin Mouse Lineage Panel). See <a href="#tbl1">Table 1</a>.</li>
<li>Add 25 μL of each biotin-labeled lineage antibody to each experimental sample: Biotin Hamster anti-Mouse CD3e, Biotin Rat anti-Mouse CD11b, Biotin Rat anti-Mouse CD45R, Biotin Rat anti-Mouse Ly-6G and Ly-6C, Biotin Rat anti-Mouse TER-119/Erythroid cells (all from the Biotin Mouse Lineage Panel). See <a href="#tbl1">Table 1</a>.</li>
<li>Add 2.5 μL of fluorochrome-conjugated antibody to the corresponding single-color control: PE-Cy7-Sca1 (Clone D7), APC-c-kit (Clone ACK2), Pacific Blue-CD150 (Clone TC15-12F12.2) and APC-Cy7-CD48 (Clone HM48-1). See <a href="#tbl1">Table 1</a>.</li>
<li>Add 25 μL of fluorochrome-conjugated antibody to each experimental sample: PE-Cy7-Sca1 (Clone D7), APC-c-kit (Clone ACK2), Pacific Blue-CD150 (Clone TC15-12F12.2) and APC-Cy7-CD48 (Clone HM48-1). See <a href="#tbl1">Table 1</a>.
<div class="critical">
<span class="critical-title">Critical:</span> To ensure correct preparation of compensation controls, verify that no more than one fluorochrome is added per single-color control tube.</div>
<div class="note">
<span class="note-title">Note:</span> If LSK cells are the desired cells to be isolated, use only PE-Cy7-Sca (Clone D7) and APC-c-kit (Clone ACK2).</div>
</li>
<li>Incubate on ice for 30 min. Add 10 mL of HBSS++ to both the single-color controls and the experimental samples. Centrifuge at 500 <b>×</b> <i>g</i> for 5 min.</li>
<li>Aspirate the supernatant and resuspend the single-color controls in 100 μL HBSS++ and the experimental samples in 1 mL HBSS++.</li>
<li>Add 1 μL of the PE-streptavidin antibody (BD Biosciences, cat# 554061) to the PE single-color control and 7 μL to the experimental samples.
<div class="critical">
<span class="critical-title">Critical:</span> Do not add the PE-secondary antibody to the remaining single-color controls nor to the unstained control (see chart above).</div>
</li>
<li>Incubate on ice for 20 min. Add 10 mL of HBSS++ to the single-color controls and experimental samples. Centrifuge at 500 <b>×</b> <i>g</i> for 5 min.</li>
<li>Aspirate the supernatant. Resuspend the experimental samples in 500 μL HBSS++, and the single-color controls in 200 μL HBSS++.</li>
<li>Filter the samples using a 5 mL round bottom tube with cell strainer cap (mesh size 35 μm) to avoid large cellular clumps.</li>
<li>Place the samples on ice and proceed to FACS sorting.</li>
<li>Collect the samples in 500 μL of collection media. The sorted samples are ready for downstream applications.</li>
</ol>
</li>
</ol>
<table id="tbl1">
<caption>Table 1. Generation of single-color controls for sorting of mouse LT-HSC</caption>
<thead>
<tr>
<th>Tube</th>
<th>HBSS++</th>
<th>Biotin-labeled lineage antibodies</th>
<th>PE-Cy7 (Sca1)</th>
<th>APC (c-kit)</th>
<th>Pacific blue (CD150)</th>
<th>APC-Cy7 (CD48)</th>
<th>PE-streptavidin</th>
</tr>
</thead>
<tbody>
<tr>
<td>Unstained (autofluorescence)</td>
<td>100 μL</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>PEsingle-color control</td>
<td>100 μL</td>
<td>2.5 μL of each</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>1 μL</td>
</tr>
<tr>
<td>PE-Cy7single-color control</td>
<td>100 μL</td>
<td>-</td>
<td>2.5 μL</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>APCsingle-color control</td>
<td>100 μL</td>
<td>-</td>
<td>-</td>
<td>2.5 μL</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>Pacific Bluesingle-color control</td>
<td>100 μL</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>2.5 μL</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>APC-Cy7single-color control</td>
<td>100 μL</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>2.5 μL</td>
<td>-</td>
</tr>
<tr>
<td>Experimental sample <i>1</i>
</td>
<td>1000 μL</td>
<td>25 μL of each</td>
<td>25 μL</td>
<td>25 μL</td>
<td>25 μL</td>
<td>25 μL</td>
<td>7 μL</td>
</tr>
<tr>
<td>Experimental sample <i>2</i>
</td>
<td>1000 μL</td>
<td>25 μL of each</td>
<td>25 μL</td>
<td>25 μL</td>
<td>25 μL</td>
<td>25 μL</td>
<td>7 μL</td>
</tr>
<tr>
<td>Experimental sample <b><i>n</i></b>
</td>
<td>1000 μL</td>
<td>25 μL of each</td>
<td>25 μL</td>
<td>25 μL</td>
<td>25 μL</td>
<td>25 μL</td>
<td>7 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> A representative FACS strategy is shown in Figure 1C.</div>
<h3 id="sec3.3">Colony forming unit (CFU) assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days for mouse CFU, 14 days for human CFU</div>
<p>In this protocol we present the optimized conditions for culturing enriched or purified HSPCs and assessing their CFU capacity <i>in vitro</i>. In order to determine the number of bone marrow or mobilized peripheral blood cells with the capacity to form hematopoietic colonies <i>in vitro</i>, an aliquot of the cell suspension (unselected, enriched, or purified cells) with a known number of viable cells, must be cultured on a nutritive substrate (in this case methylcellulose, a semisolid culture medium) supplemented with specific recombinant cytokines. The semisolid methylcellulose allows for the development of hematopoietic colonies of different lineages, including erythroid, granulocytic, monocytic, granulocytic-monocytic, and mixed myeloid-erythroid colonies.</p>
<div class="note">
<span class="note-title">Note:</span> The procedure must be performed under sterile conditions under a class II laboratory cabinet at 18°C–21°C.</div>
<div class="note">
<span class="note-title">Note:</span> The same protocol can also be used for CFU quantification from umbilical cord blood samples.</div>
<div class="note">
<span class="note-title">Note:</span> Thaw the MethoCult H4434 Classic (for human samples) or MethoCult GF M3434 (for mouse samples) and bring up to 37°C in a water bath.</div>
<ol start="8">
<li>Cell concentration:
<ol type="a">
<li>For Lin<sup>-</sup> cells, adjust the cell concentration to 10,000 viable cells per 100 μL in StemSpan SFEM II culture medium or HBSS++.</li>
<li>For CD34<sup>+</sup> cells or LSK cells, adjust the cell concentration to 2,000 viable cells per 100 μL in StemSpan SFEM II culture medium or HBSS++.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#sec6.1">troubleshooting 1</a>.</div>
<ol start="9">
<li>Cell culture:
<ol type="a">
<li>For every 3 mL of MethoCult H4434 Classic or MethoCult GF M3434 add: 30 μL of PenStrep, 100 μL of the cell suspension adjusted to the cell concentration indicated above, depending on the purity of the cells. Finally, if applicable, add any experimental drug. Immediately vortex the cell suspension for 5–10 s.
<div class="note">
<span class="note-title">Note:</span> For this step we selected the ready-to-use aliquots of 3 mL of MethoCult H4434 Classic<b>.</b>
</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Both MethoCult H4434 Classic and MethoCult GF M3434 are also available in 100 mL bottles. If this is preferred to the ready-to-use version, 3 mL aliquots should be made in advance (and stored in 15 mL conical tubes). Before aliquoting, shake the bottle to homogenize, and allow any bubbles to disappear. Store the aliquots at -20°C. Thaw each aliquot at 37°C in a water bath right before use.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Samples from patients with bone marrow failure syndromes such as Fanconi anemia, Diamond-Blackfan anemia or dyskeratosis congenita have very small amounts of HSPCs. If assessing CFU capacity in samples from these patients, consider using 3–5 times the number of input cells.</div>
</li>
<li>Let the culture cell suspension stand for 5 min or until the bubbles that have formed disappear.</li>
<li>If the assay will be analyzed using the automated STEMvision machine, slowly distribute 1 mL of methylcellulose per well in a 6-well SmartDish, avoiding the formation of bubbles. Repeat this step with the remaining 2 mL of methylcellulose.
<div class="note">
<span class="note-title">Note:</span> Each condition is plated in triplicate; a single tube containing 3 mL of methylcellulose is evenly distributed into 3 wells. Two conditions (each in triplicate) can be cultured in a 6-well plate.</div>
<div class="note">
<span class="note-title">Note:</span> The spaces between the wells of the plate must be filled with sterile water and the plate placed in a humid chamber.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> cells can be plated in 35 mm petri dishes (labeled on the sides) or in flat bottom non-tissue culture treated 12-well plates. Once again, plate 1 mL of the culture cell suspension in triplicate.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Never use tissue culture treated plates, as they will promote the attachment of cells to the plate bottom. This will prevent 3D colony formation and complicate the assessment and quantification of colonies.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To avoid evaporation of the methylcellulose, place the SmartDish in a humid chamber. The humid chamber must be prepared with sterile water in a sterile container. Sterility is warranted as the container will be placed inside the incubator for long periods of time (up to 14 days).</div>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Incubate the cells at 37°C for 14 days (in the case of human CFU assay), or for 7 days (in the case of mouse CFU assay). To ensure the humid chamber does not dry out, more sterile water can be added if necessary.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#sec6.3">troubleshooting 2</a>.</div>
<ol start="11">
<li>After 7 or 14 days, assess the size and shape of the colonies according to the criteria presented in the <a href="#expected-outcomes">expected outcomes</a> section.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#sec6.5">troubleshooting 3</a>.</div>
<p>The outcome of the human CFU assay plated in a SmartDish is shown in <a href="#fig2">Figure 2</a>A.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1042-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Types of hematopoietic colonies recognized in the CFU assay</div>
<p>(A) Hematopoietic cells from human bone marrow were cultured in methylcellulose semisolid medium supplemented with recombinant hematopoietic cytokines and cultured for 14 days at 37°C and 5% CO<sub>2</sub>. Image shown corresponds to a CFU assay as seen in the STEMvision system. Left panel: negative image. Right panel: standard image.</p>
<p>(B) Representative hematopoietic colonies from human bone marrow scored at day 14 of culture (10<b>×</b>).</p>
<p>(C) Morphological appearance of a CFU-GEMM. Left panel: 10<b>×</b>. Right panel: 100<b>×</b>. Scale bar represents the size of the colonies in mm.</p>
</figcaption>
</figure>
<h3 id="sec3.4">γH2AX foci detection in HSPCs by immunofluorescence</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>The formation of γH2AX foci is a well-known strategy for the detection of DNA damage and DNA repair signaling after a DNA damage insult. γH2AX foci formation is considered a surrogate for the formation of double strand breaks. Assessment of γH2AX foci formation by immunofluorescence is usually difficult to perform in suspension cells and even more difficult in rare populations such as HSPCs. Here we present an optimized protocol for performing γH2AX immunofluorescence in small amounts of sorted HSPCs. Quantification of γH2AX foci is used to assess the amount of DNA damage induced in the DNA molecule by a certain treatment, as well as for the evaluation of DNA repair kinetics when timepoints are performed. The quantification of γH2AX foci can be performed in freshly isolated HSPCs from mouse or human donors or after exposing the HSPCs in culture to agents that induce DNA damage, such as irradiation, cisplatin or mitomycin C. Use your desired cells of study: Lin<sup>-</sup> cells, CD34<sup>+</sup> cells, LSK cells or LT-HSCs.</p>
<ol start="12">
<li>Spin down the cells onto SuperFrost Plus slides:
<ol type="a">
<li>Set up the cytofunnel assembly.</li>
<li>Pipette 1.5<b>×</b>10<sup>5</sup> - 2<b>×</b>10<sup>5</sup> cells resuspended in 50 μL of PBS into each funnel, then place the assembly in the cytocentrifuge.</li>
<li>Run at 500 <b>×</b> <i>g</i> for 5 min.</li>
<li>Remove the funnel assembly from the cytocentrifuge and disassemble. Draw a circle on the back side of the microscope slide to indicate where the cells have been spun onto the slide.</li>
<li>Use a hydrophobic pen to draw a border around the cells.</li>
<li>Allow slides to dry completely (approximately 2 min).</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Immunofluorescence:
<ol type="a">
<li>Fixing &amp; permeabilization:
<ol type="i">
<li>Prepare 4% paraformaldehyde (PFA) by diluting the stock solution in PBS. Add ∼750 μL of 4% PFA to each sample, then incubate 15 min at 18°C–21°C.</li>
<li>Tap off PFA from the slides, then wash 2<b>×</b> 5 min with PBS.</li>
<li>Add enough ice cold methanol to cover each sample (∼200 μL), then incubate for 2 min at 18°C–21°C.</li>
<li>Wash 2<b>×</b> 5 min with PBS.</li>
</ol>
</li>
<li>Blocking:
<ol type="i">
<li>Prepare fresh blocking buffer filtered through a 0.45 μm filter.</li>
<li>Add 500 μL blocking buffer to each sample.</li>
<li>Incubate for 1 h at 18°C–21°C.</li>
</ol>
</li>
<li>Primary antibody:
<ol type="i">
<li>Prepare fresh IF buffer.</li>
<li>Dilute anti-phospho-Histone H2A.X antibody 1:1000 in IF buffer. Add ∼100 μL to each sample.</li>
<li>Incubate at 4°C in a humid chamber for 12–18 h.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this moment, the slides can stay overnight.</div>
</li>
<li>After the overnight incubation at 4°C, incubate the slides at 37°C for 1 h.</li>
<li>Wash 3<b>×</b> 5 min with PBS.</li>
</ol>
</li>
<li>Secondary antibody:
<ol type="i">
<li>Dilute secondary antibody (donkey anti-rabbit Alexa Fluor 488) 1:500 in IF buffer. Add ∼100 μL to each sample.
<div class="critical">
<span class="critical-title">Critical:</span> Follow the rest of the steps in the dark.</div>
</li>
<li>Incubate at 37°C for 1 h in a humid chamber</li>
<li>Wash 2<b>×</b> 5 min with PBS.</li>
</ol>
</li>
<li>Mounting:
<ol type="i">
<li>Dry the slide such that all excess PBS is cleared off.</li>
<li>Add a drop of ProLong Gold Antifade Mountant with DAPI over the cells.</li>
<li>Place glass coverslip on top, ensuring that there are no bubbles, and that the mounting solution covers the entire coverslip.</li>
<li>Seal the edges of the coverslips with clear nail polish.</li>
<li>Let dry at 18°C–21°C, ∼15 min.</li>
<li>Image cells right away, or store at 4°C until ready to be imaged.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Imaging: image the cells using FITC filter at 40<b>×</b> or 63<b>×</b> magnification.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#sec6.7">troubleshooting 4</a> and <a href="#sec6.9">5</a>.</div>
<h3 id="sec3.5">Alkaline comet assay using HSPCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>The comet assay is used for assessing the breakage induced in the DNA molecule following a specific treatment. The alkaline version of the comet assay is used to detect single and double-stranded breaks in the DNA molecule. This assay can be performed in freshly isolated HSPCs from mouse or human donors, or after exposing the HSPCs in culture to agents that induce DNA breaks, such as irradiation, cisplatin or mitomycin C. The comet assay has the advantage of requiring only a small number of cells, and results can be obtained using as few as 1000 cells. Use your desired cells of study: Lin<sup>-</sup> cells, CD34<sup>+</sup> cells, LSK cells or LT-HSCs.</p>
<div class="note">
<span class="note-title">Note:</span> Use the cells isolated from the previous protocols.</div>
<div class="note">
<span class="note-title">Note:</span> Work under dim light to avoid DNA breakage external to the treatment.</div>
<div class="note">
<span class="note-title">Note:</span> Sterility is not required.</div>
<div class="note">
<span class="note-title">Note:</span> Pre-chill the Lysis Solution to 4°C before beginning.</div>
<ol start="15">
<li>Melt LMAgarose (included in the comet assay kit) in a beaker of boiling water for 5 min with the cap loosened. Then, place bottle in a 37°C water bath for at least 20 min to cool.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure the agarose does not solidify. Maintain agarose at 37°C after melting.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure the slides are lying flat during the entire procedure.</div>
<ol start="16">
<li>Combine 1000 cells with 50 μL of LMAgarose (at 37°C) and immediately pipette this cell suspension onto a CometSlide (included in the comet assay kit). Avoid the formation of bubbles.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#sec6.11">troubleshooting 6</a> and <a href="#sec6.13">7</a>.</div>
<ol start="17">
<li>Place slides flat at 4°C in the dark for 10 min. A clear ring will appear at the edge of the CometSlide area.</li>
<li>Immerse slides in Lysis Solution (that has been pre-chilled to 4°C) and incubate for 12–18 h at 4°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The process can be continued the next day.</div>
<ol start="19">
<li>Drain excess buffer from slides and immerse in freshly prepared Alkaline Unwinding Solution. Incubate the slides in Alkaline Unwinding Solution for 1 h at 4°C in the dark.</li>
<li>Pour the Alkaline Electrophoresis Solution into an electrophoresis chamber, carefully immerse the slides, and align them equidistant from the chamber electrodes (See <a href="#fig3">Figure 3</a>B).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1042-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Immunofluorescence for γH2AX foci and the comet assay can be used for DNA damage assessment in HSPCs</div>
<p>(A) Representative pictures of an immunofluorescence for detection of γH2AX foci in sorted mouse LT-HSCs treated <i>in vitro</i> with 5 Gy of IR. Green arrowhead points a cell negative for γH2AX foci, pink arrowhead points a cell with multiple γH2AX foci. Scale bar represents 20 μm.</p>
<p>(B) Picture demonstrating the correct alignment of the comet assay slides with respect to the black cathode of the electrophoretic chamber.</p>
<p>(C) Representative pictures of a comet assay in sorted mouse LT-HSCs untreated (<i>left</i>) or treated <i>in vitro</i> with 5 Gy of IR (<i>right</i>). Scale bar represents 20 μm.</p>
</figcaption>
</figure>
</li>
<li>Set the power supply to 21 V and perform electrophoresis for 30 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#sec6.15">troubleshooting 8</a>.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure the slides are placed in the appropriate direction, with the slide label adjacent to the black cathode of the electrophoresis chamber.</div>
<ol start="22">
<li>Drain excess electrophoresis solution, then gently wash slides 2<b>×</b> 5 min in deionized water, then 1<b>×</b> 5 min in 70% ethanol.</li>
<li>Dry samples at 37°C for 10–15 min. Store at 18°C–21°C until ready to image.</li>
<li>Add 100 μL of the diluted SYBR Green solution onto each circle of dried agarose and stain 30 min at 18°C–21°C. Tap the slide gently to remove excess SYBR Green solution and rinse in water. Allow the slides to dry completely at 37°C.</li>
<li>View the slides using an epifluorescence microscope. SYBR Green excitation/emission spectrum is 496 nm/522 nm.</li>
<li>Data can be analyzed with ImageJ software using the OpenComet plugin.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p><b>CFU assay</b></p>
<p>CFU assays are analyzed after 14–16 days of incubation using an inverted light microscope, under the 5<b>×</b> objective. The culture dish is placed on a graduated plate to facilitate the exploration of the entire culture area. The types of colonies can be distinguished based on: color (red vs white colonies), cell and colony size (large vs small cells; large vs small colonies), number of lineages present in each colony (1, 2, 3 or 4 lineages), colony shape (compact vs diffuse colonies; one or multiple clusters, in the case of erythroid colonies) and type of colony edge. A colony must contain a minimum of 50 cells (<a href="#fig2">Figure 2</a>B). The reading begins at one of the edges of the culture dish, following a horizontal or vertical line, until the total cultivation area is verified. Small colonies may also grow on the edges of the culture dish.</p>
<p><b>Erythroid colonies</b> have a characteristic red color due to the presence of hemoglobin-containing cells; however, it is possible to find erythroid colonies in which the red color is barely present or in which the color is black. In the latter case, the identification of the colony is achieved by turning off the microscope light; red cells have a natural fluorescence, and they will appear reddish. For suspected erythroid colonies not showing color or fluorescence, determine lineage by considering the size of the cells (erythroid cells are smaller than granulocytes or macrophages) and the shape of the colony (compact clusters of cells overlapping). Erythroid colonies come in two types: small and large Burst-Forming Unit-Erythroid (BFU-E). Large BFU-E are colonies consisting of several compact clusters, located in different spatial planes, but close to each other, since they originated from a single progenitor. Small BFU-E, on the other hand, are single colonies, in a single plane, of relatively small size (<a href="#fig2">Figure 2</a>B, <i>upper panel</i>).</p>
<p><b>Myeloid colonies</b> (formed of white cells) can be of three types: CFU-Granulocytes-Monocytes (CFU-GM), CFU-Granulocytes (CFU-G), and CFU-Monocytes (CFU-M). The three colony types may be present with similar cell numbers, so they can be identified based on cell size and colony shape. CFU-G are radial colonies, which can present as a loose or extremely tight grouping; in the latter case, it can be so narrow that colonies may be mistaken for a BFU-E. However, turning off the microscope light helps to distinguish the type of colony (CFU-G will appear white). CFU-M are composed of large cells compared to those present in a granulocyte or erythroid colony. The cells appear opaque, and the colony may have an irregular distribution, spread over the substrate. Finally, CFU-GM are colonies containing both granulocytes and monocytes at different proportions (<a href="#fig2">Figure 2</a>B, <i>lower panel</i>). CFU-GEMM (mixed colonies of granulocytes, erythrocytes, monocytes, and megakaryocytes) contain all the lineages described above. They can also include megakaryocytic cells, although the presence of such a lineage is dependent on the addition of specific cytokines to the substrate to achieve the differentiation and identification of these cells. CFU-GEMM can be medium, large, or very large colonies, where the different lineages are intertwined on the same plane. It is important to point out that a mixed colony always contains red cells together with some of the white cell lineages, not necessarily the strict presence of all the myeloid lineages (<a href="#fig2">Figure 2</a>C).</p>
<p>Reading the content of hematopoietic progenitors in the automated STEMvision system allows for the morphology, shape, and size of hematopoietic colonies to be reviewed at different magnifications. However, the software only discriminates between red, white, and mixed colonies, not between the different myeloid lineages. It is also important that each reading is verified by the reader, since it is possible that the software will not detect very small colonies or those on the edges of the culture dish, as well as those that may be mistaken for substrate debris. In general, the reading in the automated system presents an undervaluation with respect to the manual reading of the same assay.</p>
<p>CFU numbers are usually reported as per 100,000 cells, but they can also be presented as a percentage of cultured cells. To report the content of CFU per 100,000 cells, the following equation is considered:</p>
<p class="math-ml"><math> <mrow> <mfrac> <mrow> <mrow> <mo>(</mo> <mn>100,000</mn> <mo>)</mo> </mrow> <mrow> <mo>(</mo> <mrow> <mi>C</mi> <mi>F</mi> <mi>U</mi> <mspace></mspace> <mi>r</mi> <mi>e</mi> <mi>a</mi> <mi>d</mi> </mrow> <mo>)</mo> </mrow> </mrow> <mrow> <mi>n</mi> <mi>u</mi> <mi>m</mi> <mi>b</mi> <mi>e</mi> <mi>r</mi> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mi>s</mi> <mspace></mspace> <mi>s</mi> <mi>e</mi> <mi>e</mi> <mi>d</mi> <mi>e</mi> <mi>d</mi> <mspace></mspace> <mi>i</mi> <mi>n</mi> <mspace></mspace> <mi>t</mi> <mi>h</mi> <mi>e</mi> <mspace></mspace> <mi>a</mi> <mi>s</mi> <mi>s</mi> <mi>a</mi> <mi>y</mi> </mrow> </mfrac> </mrow> </math></p>
<p>To report the content of CFU as a percentage of cultured cells, the following equation is considered:</p>
<p class="math-ml"><math> <mrow> <mfrac> <mrow> <mrow> <mo>(</mo> <mn>100</mn> <mo>)</mo> </mrow> <mrow> <mo>(</mo> <mrow> <mi>C</mi> <mi>F</mi> <mi>U</mi> <mspace></mspace> <mi>r</mi> <mi>e</mi> <mi>a</mi> <mi>d</mi> </mrow> <mo>)</mo> </mrow> </mrow> <mrow> <mi>n</mi> <mi>u</mi> <mi>m</mi> <mi>b</mi> <mi>e</mi> <mi>r</mi> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mi>s</mi> <mspace></mspace> <mi>s</mi> <mi>e</mi> <mi>e</mi> <mi>d</mi> <mi>e</mi> <mi>d</mi> <mspace></mspace> <mi>i</mi> <mi>n</mi> <mspace></mspace> <mi>t</mi> <mi>h</mi> <mi>e</mi> <mspace></mspace> <mi>a</mi> <mi>s</mi> <mi>s</mi> <mi>a</mi> <mi>y</mi> </mrow> </mfrac> </mrow> </math></p>
<p>Depending on the study objectives, it is possible to report the total CFU content or the content of independent lineages: CFU-Myeloid, CFU-Erythroid, CFU-Mixed. When analyzing the CFU content in a purified or enriched hematopoietic (CD34<sup>+</sup>) cell population during a particular treatment in liquid culture, it is important to consider the effect of such a treatment on cell proliferation. Colony assays are normally carried out with a sample of the total cell population, so the report of CFU content per 100,000 cells can be modified if there are fewer or more cells. Thus, the increase or decrease (fold-change) with respect to the CFU content at the beginning of the treatment can be presented.</p>
<p><b>γH2AX assay</b></p>
<p>γH2AX (or phospho-H2AX) is a marker of DNA double-stranded breaks, as well as genomic instability. Therefore, it can be used to evaluate DNA repair following treatment with DNA-damaging agents such as mitomycin C, cisplatin, or irradiation. Positive staining can also be used to assess DNA repair kinetics when timepoints are performed. Cells exposed to mitomycin C, cisplatin, or irradiation will contain more DNA damage and thus produce more foci compared to untreated cells. Likewise, cells without DNA damage should contain very little or no positive staining. γH2AX foci are visualized as bright dots within the nucleus (<a href="#fig3">Figure 3</a>A). To quantify these foci, the number of positive cells or the number of foci per nucleus can be counted.</p>
<p><b>Comet assay</b></p>
<p>The comet assay enables the detection of broken DNA. First, the cells are subjected to DNA damage, then the cytoplasm is lysed and the nuclei are subjected to an electrophoretic field causing the migration of broken DNA away from the cathode of the electrophoretic chamber. Cells are then stained with SYBR Green dye which binds to DNA and emits a green signal. Migration of the DNA molecule will form a tail, thus indicating the presence of DNA breaks. In cells without DNA damage, the fluorescence is confined to the nucleus, and the nuclei will appear as perfect circles. The undamaged DNA is supercoiled, and thus does not migrate very far out of in the electric current (<a href="#fig3">Figure 3</a>C). A good positive control for the comet assay is cells exposed to IR (5 Gy) allowed to recover for 2–4 h after IR. Common descriptors of DNA damage for alkaline comet assays are “percent of DNA in the tail”, “comet length” and “tail moment”. Percent DNA in the tail is a normalized measure of the percent of total cell DNA found in the tail. Tail moment is a damage measure combining the amount of DNA in the tail with distance of migration. The OpenComet plugin from Image J can be used for quantification of the percent of DNA in the tail, comet length and tail moment. At least 50 cells should be scored per sample.</p>
<h3 id="sec4.1">Usefulness of assessing DNA damage in HSPCs</h3>
<p>The continuous generation of blood from HSPCs poses a challenge on this cell compartment, and efficient DNA repair systems are needed for mitigating the damage that arises from replication stress or endogenous sources of DNA damage. A prime example of how DNA damage can affect HSPCs is Fanconi anemia, an inherited bone marrow failure syndrome with a defective FA/BRCA pathway. This pathway detects and repairs DNA interstrand crosslinks. Aplastic anemia appears at a young age in patients with Fanconi anemia since their HSPCs are remarkably sensitive to DNA damage and succumb to apoptosis (<a href="#bib9">Rodríguez and D’Andrea, 2017</a>).</p>
<p>Although the isolation of HSPCs is challenging given their rareness, their study is critical, not only for FA but also to study how DNA damaging agents affect hematopoiesis, as well as for the discovery of novel diseases or mechanisms of DNA damage. Also, the effectiveness of novel treatments aimed to improve DNA repair and aplastic anemia (<a href="#bib15">Zhang et al., 2016</a>), or even gene therapy strategies currently underway in Fanconi anemia (<a href="#bib6">Río et al., 2019</a>), can be assessed using DNA damage assays in HSPCs if they are efficiently isolated from either mouse or human.</p>
<p>Although the usage of cell lines is advantageous over primary tissues and the vast majority of research in the DNA repair field is performed in cell lines, validation of results is also critical in primary tissues or <i>in vivo</i> models if translational outcomes are sought. We consider that the thorough step-by-step protocol that we present here will reduce some methodological burden for researchers interested on exploring DNA repair in the HSPC compartment.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p><b>CFU assay.</b> The CFU assay does not detect all hematopoietic progenitors present in a particular cell population, but only those capable of forming colonies in semisolid culture. Thus, this <i>in vitro</i> assay should not be taken as a means of quantifying all hematopoietic progenitor cells. Indeed, it has been shown that only a fraction of CD34<sup>+</sup> progenitors are capable of forming colonies <i>in vitro</i>. The CFU assay is used to determine the number and phenotype of progenitor cells capable of forming hematopoietic colonies at a specific time. However, it does not detect progenitor cells that are not able to display their colony-forming potential due to particular intrinsic (e.g., cell cycle phase) or extrinsic (e.g., type, number, or concentration of cytokines present in the culture medium) factors. CFU assays allow for the detection of multipotent, myeloid, erythroid, and, under specific culture conditions, megakaryocytic progenitor cells. However, they are not suitable for the detection of lymphoid progenitors.</p>
<p><b>γH2AX assay.</b> The outcome of immunofluorescence staining depends on several factors. In order to minimize background while maintaining a strong signal, the quality and dilution of the antibody is critical. In some cases, resolving signal-to-noise ratio-related problems can require several test dilutions of the primary and secondary antibodies to determine the optimal concentrations. To prevent bleaching (decreased fluorescence signal due to light exposure), the protocol should be done in the dark after the addition of the secondary antibody, and an antifade reagent should be used in the mounting medium. It is essential that the samples are protected from light when not being imaged, and the slides should be exposed to the minimum amount of light from the microscope necessary to image the sample.</p>
<p><b>Comet assay.</b> The comet assay does not provide an absolute quantification of the amount of DNA damage in a cell; therefore, it should ideally be accompanied by other assays for the detection of DNA damage. Samples must be prepared under dim lighting conditions, otherwise the DNA breaks induced by light exposure could interfere with data interpretation. Samples should be imaged quickly after staining with SYBR Green, otherwise the fluorescence signal can be lost.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Oversaturation of CFU cultures (major step 8).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Maintaining an optimal number of cells in each assay is important to avoid reading errors. A culture with too many colonies prevents their correct identification and may lead to over- or undervaluation of the real content of progenitors in a sample.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>CFU cultures become dried out (major step 10).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>It is important to monitor the amount of water present in the humid chamber and to add more if, during the incubation period, water levels decrease too much. If at the time of reading the plates, slight dryness is observed, but colonies are well-defined, a small amount of sterile water can be added to the surfaces of the culture without shaking, so a wet layer is formed that facilitates the reading and recognition of hematopoietic colonies.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Low detection of CFU colonies (major step 11).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The concentration of the cells to be cultured must be optimal, as well as the dilution in the liquid medium in which they are found. A highly diluted sample, when placed on the substrate, would result not only in a reduced number of cells and very few colonies. It is also important to keep the methylcellulose aliquots frozen and thaw them just prior to use. It has been shown that when complete methylcellulose medium aliquots are kept in the fridge for several weeks, cytokines may lose some potency.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>γH2AX immunofluorescence has decreased or no signal (major step 14).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Ensure that the samples are not exposed to light for extended periods of time after the fluorescent secondary antibody is added. Samples should also be imaged as soon as possible after mounting; avoid extended storage periods for best results.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>High background signal in γH2AX immunofluorescence (major step 14).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Attempt longer or additional washes to ensure fixative, permeabilization buffer, and/or secondary antibody have been sufficiently removed. If blocking is inadequate, use a normal serum from the same species as the secondary antibody.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Comet tails overlap (major step 16).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Reduce the number of cells. Always quantify the number of cells before starting the protocol.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Majority of cells in the untreated control of the comet assay have long comet tails (major step 16).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>Keep cells on ice and prepare cell samples immediately before combining with melted agarose. Work under dim light.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>The positive control of the comet assay has no tails (major step 21).</p>
<h3 id="sec6.16">Potential solution</h3>
<p>Increase the time of electrophoresis. Ensure the reagents are fresh.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Kiel, M.J., Yilmaz, O.H., Iwashita, T., Yilmaz, O.H., Terhorst, C., and Morrison, S.J. (2005). SLAM family receptors distinguish hematopoietic stem and pro- genitor cells and reveal endothelial niches for stem cells. Cell <i>121</i>, 1109-1121. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref1">View at publisher</a></p>
<p id="bib2">Lin, F.C., Karwan, M., Saleh, B., Hodge, D.L., Chan, T., Boelte, K.C., Keller, J.R., and Young, H.A. (2014). IFN-g causes aplastic anemia by altering he- matopoietic stem/progenitor cell composition and disrupting lineage differen- tiation. Blood <i>124</i>, 3699-3708. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref2">View at publisher</a></p>
<p id="bib3">Matatall, K.A., Jeong, M., Chen, S., Sun, D., Chen, F., Mo, Q., Kimmel, M., and King, K.Y. (2016). Chronic infection depletes hematopoietic stem cells through stress-induced terminal differentiation. Cell Rep. <i>17</i>, 2584-2595. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref3">View at publisher</a></p>
<p id="bib4">Olive, P.L. and Banáth, J.P. (2006). The comet assay: a method to measure DNA damage in individual cells. Nat. Protoc. <i>1</i>, 23-29. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref4">View at publisher</a></p>
<p id="bib5">Parmar, K., D’Andrea, A., and Niedernhofer, L.J. (2009). Mouse models of Fanconi anemia. Mutat. Res. <i>668</i>, 133-140. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref5">View at publisher</a></p>
<p id="bib6">Río, P., Navarro, S., Wang, W., Sánchez-Domínguez, R., Pujol, R.M., Segovia, J.C., Bogliolo, M., Merino, E., Wu, N., Salgado, et al. (2019). Successful engraftment of gene-corrected hematopoietic stem cells in non-conditioned patients with Fanconi anemia. Nat. Med. <i>25</i>, 1396-1401. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref6">View at publisher</a></p>
<p id="bib7">Rodríguez, A., Zhang, K., Färkkilä, A., Filiatrault, J., Yang, C., Velázquez, M., Furutani, E., Goldman, D.C., García de Teresa, B., Garza-Mayén, G., et al. (2021a). MYC promotes bone marrow stem cell dysfunction in Fanconi anemia. Cell Stem Cell <i>28</i>, 33-47. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref7">View at publisher</a></p>
<p id="bib8">Rodríguez, A., Yang, C., Furutani, E., García de Teresa, B., Velázquez, M., Filiatrault, J., Sambel, L.A., Phan, T., Flores-Guzm, P., Sánchez, S., et al. (2021b). Inhibition of TGFβ1 and TGFβ3 promotes hematopoiesis in Fanconi anemia. Exp. Hematol. <i>93</i>, 70-84. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref8">View at publisher</a></p>
<p id="bib9">Rodríguez, A. and D’Andrea, A. (2017). Fanconi anemia pathway. Curr. Biol. <i>27</i>, R986-R988. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref9">View at publisher</a></p>
<p id="bib10">Seita, J. and Weissman, I.L. (2010). Hematopoietic stem cell: self-renewal versus differentiation. Wiley Interdiscip. Rev. Syst. Biol. Med. <i>2</i>, 640-653. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref10">View at publisher</a></p>
<p id="bib11">Shimano, K.A., Narla, A., Rose, M.J., Gloude, N.J., Allen, S.W., Bergstrom, K., Broglie, L., Carella, B.A., Castillo, P., de Jong, J.L., and Dror, Y. (2021). Diagnostic work-up for severe aplastic anemia in children: consensus of the north American pediatric aplastic anemia consortium. Am. J. Hematol. <a class="external-link" href="https://doi.org/10.1002/ajh.26310">https://doi.org/10.1002/ajh.26310</a></p>
<p id="bib12">Tang, J., Zhu, N., Rao, S., and Carlson, K.S. (2019). Stem cell damage after chemotherapy-can we do better?. Best Pract. Res. Clin. Haematol. <i>32</i>, 31-39. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref12">View at publisher</a></p>
<p id="bib13">Walter, D., Lier, A., Geiselhart, A., Thalheimer, F.B., Huntscha, S., Sobotta, M.C., Moehrle, B., Brocks, D., Bayindir, I., Kaschutnig, P., et al. (2015). Exit from dormancy provokes DNA-damage-induced attrition in haematopoietic stem cells. Nature <i>520</i>, 549-552. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref13">View at publisher</a></p>
<p id="bib14">Yilmaz, O.H., Kiel, M.J., and Morrison, S.J. (2006). SLAM family markers are conserved among hematopoietic stem cells from old and reconstituted mice and markedly increase their purity. Blood <i>107</i>, 924-930. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref14">View at publisher</a></p>
<p id="bib15">Zhang, H., Kozono, D.E., O’Connor, K.W., Vidal-Cardenas, S., Rousseau, A., Hamilton, A., Moreau, L., Gaudiano, E.F., Greenberger, J., Bagby, G., et al. (2016). TGF-β inhibition rescues hematopoietic stem cell defects and bone marrow failure in Fanconi anemia. Cell Stem Cell <i>18</i>, 668-681. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00552-9/sref15">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Alan D’Andrea (<a href="mailto:alan_dandrea@dfci.harvard.edu">alan_dandrea@dfci.harvard.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate new data or code.</p>
<h3>Acknowledgments</h3>
<p>We thank the patients with Fanconi anemia and their families and also their physicians; we also thank the FACS facility at Dana Farber Cancer Institute. This research was supported by grants from the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (R01HL052725 and P01HL048546), the <a href="https://doi.org/10.13039/100005189">Leukemia and Lymphoma Society</a> (6237-13), and the <a href="https://doi.org/10.13039/100002086">Fanconi Anemia Research Fund</a> (to A.D.D.).</p>
<h3>Author contributions</h3>
<p>A.R., P.F.-G., H.M., K.P., and A.D.D. developed protocols and designed experiments. A.R. performed experiments with the help of J.F. and P.F.-G. H.M. and A.D. provided reagents and analyzed data. A.D.D. supervised the experiments. A.R., J.F., P.F.-G., H.M., K.P., and A.D.D. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>