<article>
<section>
<h1 id="header">An optimized protocol for the generation of HBV viral antigen-specific T lymphocytes from pluripotent stem cells</h1>
<p><time datetime="2021-01-15">Published: January 15, 2021</time></p>
<p>Mohammad Haque,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Xiaofang Xiong,<sup><a href="#aff1">1</a></sup> Fengyang Lei,<sup><a href="#aff2">2</a></sup> Jugal Kishore Das,<sup><a href="#aff1">1</a></sup> and Jianxun Song<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Microbial Pathogenesis and Immunology, Texas A&amp;M University Health Science Center, MREB II, Room 3344, 8447 Riverside Pkwy, Bryan, TX 77807, USA</p>
<p id="aff2"><sup>2</sup>Department of Ophthalmology, Harvard Medical School, Boston, MA 02215, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:jus35@tamu.edu">jus35@tamu.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100264">10.1016/j.xpro.2020.100264</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>In T cell-based cancer immunotherapy, tumor antigen (Ag)-specific CD8<sup>+</sup> cytotoxic T lymphocytes (CTLs) can specifically target tumor Ags on malignant cells. This promising approach drove us to adopt this strategy of T cell transfer (ACT)-based immunotherapy for chronic viral infections. Here, we describe the generation of hepatitis B virus (HBV) Ag-specific CTLs from induced pluripotent stem cells (iPSCs), i.e., iPSC-CTLs. Ag-specific iPSC-CTLs can target HBV Ag<sup>+</sup> cells and infiltrate into the liver to suppress HBV replication in a murine model.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib9">Haque et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Generation of functional viral Ag-specific CTLs from iPSCs, i.e., iPSC-CTLs</li>
<li>Viral Ag-specific iPSC-CTLs suppress HBV replication in a murine model</li>
<li>Adoptive transfer of viral Ag-specific iPSC-CTLs induces persistent α-HBV immunity</li>
<li>Adoptive transfer of viral Ag-specific iPSC-CTLs produces antiviral cytokines</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/367-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 0–4 days</div>
<ol>
<li>Generation of the retroviral vector and transduction.
<ol type="a">
<li>Construct MSCV-IRES-DsRED (MIDR) vector from MSCV-IRES-GFP vector by replacing the GFP gene with the DsRED gene (<a href="#bib6">Lei et al., 2011</a>).</li>
<li>Generation of HBV construct.
<ol type="i">
<li>The HBV 1.2 full length DNA was sub-cloned from the plasmid pHBV-48, containing whole HBV genome to a vector pAAV.</li>
<li>A BamHI/EcoRI digested fragment (1.8 kb) and an EcoRI/BglII-digested fragment (2.0 kb) of pHBV-48 were cloned into the BglII site of the AAV vector. The resulting pAAV/HBV1.2 contains the HBV fragments spanning nucleotides 1400-3182.1-1987 flanked by inverted terminal repeats of AAV.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare all buffers freshly.</div>
<ol start="2">
<li>Retroviral transduction and cell sorting.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare media, reagents, and cell line before the experiments.</div>
<ol start="3">
<li>Platinum-E (Plat-E) packaging cell line is used to generate a pseudo virus for following retroviral transduction.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare all reagents for plasmid preparation.</div>
<ol start="4">
<li>Cells are prepared for cell sorting.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All cell incubations with Gelatin are performed at 37˚C and 230 rpm.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>CD3 (17A2)</td>
<td>BioLegend</td>
<td>CAT#100236</td>
</tr>
<tr>
<td>CD8 (63-6.7)</td>
<td>BioLegend</td>
<td>CAT#100732</td>
</tr>
<tr>
<td>CD4 (GK1.5)</td>
<td>BioLegend</td>
<td>CAT#100408</td>
</tr>
<tr>
<td>APC IFN-γ (XMG1.2)</td>
<td>BioLegend</td>
<td>CAT#505810</td>
</tr>
<tr>
<td>TNF-α (MP6-XT22)</td>
<td>BioLegend</td>
<td>CAT#506306</td>
</tr>
<tr>
<td>CD44 (IM7)</td>
<td>BioLegend</td>
<td>CAT#103015</td>
</tr>
<tr>
<td>CD69 (H1.2F3)</td>
<td>BioLegend</td>
<td>CAT#104514</td>
</tr>
<tr>
<td>CD62L (Mel-14)</td>
<td>BioLegend</td>
<td>CAT#104427</td>
</tr>
<tr>
<td>Anti-human TCRα/β</td>
<td>BioLegend</td>
<td>CAT#351711</td>
</tr>
<tr>
<td>HRP anti-mouse secondary antibody</td>
<td>Invitrogen</td>
<td>CAT#A27025</td>
</tr>
<tr>
<td>Anti-HBs antibody</td>
<td>Thermo Fisher</td>
<td>CAT#MA5-13059</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Mouse iPS-MEF-Ng-20D-17</td>
<td>Riken Cell Bank</td>
<td>N/A</td>
</tr>
<tr>
<td>OP9-DL1/DL4</td>
<td>ATCC</td>
<td>CRL 2749</td>
</tr>
<tr>
<td>SNL 76/7</td>
<td>ATCC</td>
<td>SCRC-1049</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>HLA-A2.1 transgenic mice</td>
<td>The Pasteur Institute</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>HBV construct pAAV/HBV1.2</td>
<td>Dr. Pei-Jer Chen (National Taiwan University)</td>
<td>N/A</td>
</tr>
<tr>
<td>Murine hybrid TCR genes Vα34 and Vβ28</td>
<td>Dr. Adam j. Gehring (Toronto General Hospital Research Institute (TGHRI))</td>
<td>N/A</td>
</tr>
<tr>
<td>OVA 2<sub>57-</sub>2<sub>64</sub> specific TCR genes</td>
<td>Dr. Dario A.A. Vignali (University of Pittsburgh)</td>
<td>N/A</td>
</tr>
<tr>
<td>BamH1/EcoR1</td>
<td>New England Biolabs</td>
<td>CAT# R0136T/R0101T</td>
</tr>
<tr>
<td>EcoRI/BglII</td>
<td>New England Biolabs</td>
<td>CAT# R0101T/R0144M</td>
</tr>
<tr>
<td colspan="3">Buffer, media, and reagents</td>
</tr>
<tr>
<td>α-MEM</td>
<td>Invitrogen</td>
<td>CAT# A10490-01</td>
</tr>
<tr>
<td>ACK Lysis buffer</td>
<td>Lonza</td>
<td>CAT# 10-548E</td>
</tr>
<tr>
<td>Brefeldin A</td>
<td>Sigma</td>
<td>CAT# B7651</td>
</tr>
<tr>
<td>DMEM</td>
<td>Thermo Fisher</td>
<td>CAT# 11965092</td>
</tr>
<tr>
<td>FBS</td>
<td>Hyclone</td>
<td>Product # SH3007.01</td>
</tr>
<tr>
<td>Gelatin</td>
<td>Millipore Sigma</td>
<td>CAT# G9391</td>
</tr>
<tr>
<td>Gene Jammer</td>
<td>Agilent</td>
<td>CAT# 204130</td>
</tr>
<tr>
<td>HLA-A201HBs183-91-PE pentamer</td>
<td>Proimmune</td>
<td>CAT# F027-4A-27</td>
</tr>
<tr>
<td>mFlt-3L</td>
<td>Peprotech</td>
<td>CAT# 250-31L</td>
</tr>
<tr>
<td>mIL-7</td>
<td>Peprotech</td>
<td>CAT# 217-17</td>
</tr>
<tr>
<td>Nuclease S7</td>
<td>Roche</td>
<td>10107921001</td>
</tr>
<tr>
<td>Paraformaldehyde</td>
<td>Millipore Sigma</td>
<td>SKU P6148-500G</td>
</tr>
<tr>
<td>Polybrene</td>
<td>Millipore Sigma</td>
<td>SKU 107689</td>
</tr>
<tr>
<td>Permeabilization buffer</td>
<td>BioLegend</td>
<td>CAT# 421002</td>
</tr>
<tr>
<td>Prolong Gold Antifade Mount with DAPI</td>
<td>Invitrogen</td>
<td>CAT# P36931</td>
</tr>
<tr>
<td>QIAamp MinElute Virus Spin Kit</td>
<td>QIAGEN</td>
<td>CAT# 57704</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Prism9</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Cell culture (days 0–5)</h3>
<ol>
<li>Murine iPSCs were maintained in DMEM culture medium supplemented with 15% fetal calf serum (FCS), 0.1 mmol/L nonessential amino acids, 1 mmol/L L-glutamine and 0.1 mmol/L β-mercaptoethanol. Monolayers of OP9-DL1/DL4 cells were cultured in α-MEM medium supplemented with 20% FCS and 2.2 g/L sodium bicarbonate.</li>
<li>The iPSCs were washed once in OP9-DL1/DL4 medium before plating onto sub-confluent OP9-DL1/DL4 monolayers for T lineage differentiation in the presence of murine recombinant Flt-3 ligand (mrFlt3L; 5 ng/mL) and murine IL-7 (mIL-7;1 ng/mL).</li>
</ol>
<h3 id="sec2.2">Generation of HBV viral Ag-specific iPSC-CTLs (days 0–28)</h3>
<ol start="3">
<li>Mouse iPSCs (iPS-MEF-Ng-20D-17, GFP<sup>+</sup>) (<a href="#bib8">Yu et al., 2009</a>) were transduced with the MIDR retroviral construct encoding a human-mouse hybrid HBV TCR gene (HBs183-191-specific—s183; TCR Vα34 and Vβ28) (<a href="#fig1">Figure 1</a>A), or OVA257–264 TCR (MiDR-OVA TCR) (<a href="#bib3">Gehring et al., 2011</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/367-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. How to generate HBV viral Ag-specific iPSC-CTLs</div>
<p>Transduce mouse iPSCs with the following retroviral constructs: HBs<sub>183-91</sub> TCR (MiDR-HBV s183 TCR) or OVA<sub>257–264</sub> TCR (MiDR-OVA TCR), and co-culture the transduced iPSCs with OP9-DL1/DL4 stromal cells for T lineage differentiation.</p>
<p>(A) Present the schematic representation of the retrovirus constructs expressing HBV s183 TCR. Ψ, packaging signal; 2A, picornavirus self-cleaving 2A sequence; LTR, long terminal repeats.</p>
<p>(B) Visualize the HBV TCR-transduced iPSCs by a fluorescence microscope.</p>
<p>(C) Transduce GFP<sup>+</sup> iPSCs (left and middle, no transduction) with the retroviral construct MiDR or MiDR with HBV s183 TCR, and analyze and sort the GFP<sup>+</sup> DsRed<sup>+</sup> iPSCs by flow cytometry (right) and by a high-speed cell sorter.</p>
<p>(D) Analyze HBV s183 TCR for Vβ28 gene expression by PCR.</p>
<p>(E) Show morphology of T cell differentiation on various days.</p>
<p>(F) Show flow cytometric analysis of the iPSC-derived cells on day 28, gating on CD3<sup>+</sup>CD8<sup>+</sup> cells, and analyze for the expression of CD8 and TCRVβ28. Data shown are representative of three individual experiments.</p>
</figcaption>
</figure>
<ol type="a">
<li>Platinum-E (Plat-E) packaging cell line is used to generate a pseudovirus for following retroviral transduction.</li>
<li>Seed 3 × 10<sup>6</sup> Plat-E cells into a 100 mm culture dish containing 10 mL of RPMI complete media with FBS and penicillin/streptomycin 1 day prior to plasmid transfection.</li>
<li>On day 0, transfect Plat-E cells with HBV TCR-MiDR retroviral vector by using GeneJammer transfection reagent.</li>
<li>On day 1, take 2 mL of 0.1% Gelatin on to the well of the plate. Keep them in incubator at least for 30 min. After 30 min, remove gelatin from the plate by pipetting. Seed 1 × 10<sup>6</sup> iPSCs into one well of a 0.1% gelatin-coated 24-well plate containing 2 mL of DMEM media with 15% FBS.</li>
<li>On day 2, collect pseudovirus-containing supernatant from transfected Plat-E culture and pass the supernatant through a 0.4 μm filter to exclude potential debris and contaminants.</li>
<li>Perform retroviral transduction in a centrifuge at 500 × <i>g</i> for 1 h in the presence of 5 μg/mL polybrene at 32°C. Use the fresh virus-containing supernatant to get the highest transduction efficiency.</li>
<li>After transduction, place cells in a 32°C, 5% CO<sub>2</sub> incubation overnight (14–16 h).</li>
<li>On day 3, repeat the second transduction as described above. Meanwhile, pre-coat a 6-well plate with irSNL76/7 feeder cells with α-MEM media at 37˚C for future use.</li>
<li>On day 4, collect transduced iPSCs by trypsinization, centrifuge at 400 × <i>g</i> for 5 min, and seed into the 6-well plate pre-coated with irSNL76/7 feeder cells with DMEM media supplemented with 15% FBS at 37˚C.</li>
<li>At confluence, collect cells by trypsinization, centrifuge at 400 × <i>g</i> for 5 min, and process for cell sorting. GFP and DsRED double positive cells will be sorted by a 17-color BD FACS Aria SORP cell sorter. Culture sorted cells (HBV-iPSCs) on fresh irSNL76/7 feeder cells for future use.</li>
<li>OP9-DL4 and OP9-DL1/DL4 cell lines are generated as described above but by using DL4-MiG and DL1/DL4-MiG constructs, respectively. Transduced cells are sorted by the FACSAria SORP based on GFP expression.</li>
</ol>
</li>
<li>Gene-transduced iPSCs were co-cultured with OP9-DL1/DL4 cells expressing Notch ligands (both DL1 and DL4) molecules in the presence of rFlt3L and rIL-7.
<ol type="a">
<li>On day 0, seed 5 × 10<sup>4</sup> HBV TCR gene-transduced iPSCs into a 100 mm culture dish coated with a confluent monolayer of OP9 (or other OP9-related cell lines) cells in 20% FBS in α-MEM medium at 37˚C.</li>
<li>On day 3, change culture medium.</li>
<li>On day 5, collect cells by trypsinization and centrifuge at 400 × <i>g</i> for 5 min before incubating with α-MEM medium on a fresh 100 mm culture dish for 30 min in a 37°C incubator to exclude feeding cells.</li>
<li>Collect and count floating cells, and transfer 5 × 10<sup>5</sup> cells to a fresh culture dish containing a confluent monolayer of feeding cells in 20% FBS α-MEM medium. Add the cytokine mFlt-3L (final concentration: 5 ng/mL) into the culture at 37˚C.</li>
<li>On day 8, gently pipette down loosely attached cells and then wash the feeding layer with 10 mL of PBS to get the maximal recovery of partially differentiated HBV-iPSCs.</li>
<li>After harvesting cells from the co-culture, centrifuge cells at 400 × <i>g</i> for 5 min and resuspend them in 20 mL of 20% FBS α-MEM medium supplemented with mFlt-3L (5 ng/mL) and IL-7 (1 ng/mL).</li>
<li>Transfer cells into a 6-well culture plate coated with confluent OP9-feeder cells. Usually HBV TCR-iPSCs recovered from one 100 mm culture dish will be transferred into one well of a 6-well plate.</li>
<li>From day 10, change medium every other day (20% FBS α-MEM medium supplemented with 5 ng/mL mFlt-3L and 1 ng/mL IL-7).</li>
<li>Replace OP9 feeder cell plates every 4–6 days depending on the growth of the feeder cells.</li>
</ol>
</li>
<li>Visualization of the DsRed expression upon gene transduction by fluorescence microscopy (<a href="#fig1">Figure 1</a>B) and sorted GFP<sup>+</sup>DsRed<sup>+</sup> cells (<a href="#fig1">Figure 1</a>C).</li>
<li>Confirmation of HBV TCR insertion into the transduced cells by PCR amplification (<a href="#fig1">Figure 1</a>D) and gene sequencing.</li>
<li>After 7 days of culture with the OP9-DL1/DL4 cells, iPSCs differentiated into mesoderm-like cells and showed characteristic, non-adherent grape-like cluster morphology on day 14. On day 22, lymphocyte-like cells spread fully on the culture plates (<a href="#fig1">Figure 1</a>E).</li>
<li>On day 28 of in vitro co-culture, the iPSC-derived cells substantially expressed CD3 and Ag-specific TCR, the T cell markers.</li>
<li>Flow cytometric analysis of CD3<sup>+</sup>CD8<sup>+</sup> populations showed that the HBV s183 but no OVA TCR transduction dramatically increased the generation of HBV-specific CD8<sup>+</sup> T cells (CD8<sup>+</sup> TCRVβ28<sup>+</sup>; <a href="#fig1">Figure 1</a>F).</li>
</ol>
<h3 id="sec2.3">Functional assessment of HBV viral Ag-specific iPSC-CTLs (days 28–30)</h3>
<ol start="10">
<li>On day 28 of in vitro co-culture, CD4<sup>−</sup>CD8<sup>+</sup> single-positive (SP) iPSC-CTLs were isolated and stimulated them by T-depleted splenocytes pulsed with s183 peptide for indicated time period and assessed cytokine production.</li>
<li>The iPSC-CTLs produced large amounts of IL-2 and IFN-γ, as detected by intracellular staining (<a href="#fig2">Figure 2</a>A) or ELISA (<a href="#fig2">Figure 2</a>B) according to the manufacturer instruction and displayed Ag-specific cytotoxicity (<a href="#fig2">Figure 2</a>C), which were similar as HBV TCR gene-transduced CTLs.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/367-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Perform functional analysis of HBV viral Ag-specific iPSC-CTLs</div>
<p>On day 28 of <i>in vitro</i> co-culture, sort the SP CD8<sup>+</sup>s183 TCR pentamer<sup>+</sup> iPSC-T cells. Stimulate the iPSC-T cells and CD8<sup>+</sup> T cells transduced with MiDR-s183 TCR with s183 peptide (FLLTRILTI)-pulsed T-depleted splenocytes (APCs) from HHD mice.</p>
<p>(A) Show intracellular staining of IL-2 and IFN-γ after 7 h (gated on CD8<sup>+</sup> cells) (T/APCs = 1:4).</p>
<p>(B) Present ELISA of IL-2 and IFN-γ after 40 h.</p>
<p>(C) Measure T cell cytotoxicity after co-culture for 6 h using the 7-AAD/CFSE cell-mediated cytotoxicity assay kit. Data shown are representative of three individual experiments.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec2.4">Development of HBV replication in mice (weeks 1–8)</h3>
<ol start="12">
<li>HLA-A2.1 transgenic (HHD) mice were infected with HBV by hydrodynamic injection of 10 mg of pAAV/HBV1.2 DNA plasmid through the tail veins of mice.</li>
<li>Total HBV DNA was resuspended in 1 mL of PBS and delivered into the liver within 3 s. For each animal, 10 μg of pAAV/HBV 1.2 DNA dissolved in 8% body weight of PBS was injected. Before the injection, all animals were anesthetized using isoflurane administered by isoflurane vaporizer. Briefly, anesthetized mice were positioned, so one of the lateral veins are visible. Viral DNA was re suspended in 1 mL of PBS and loaded in 3 ML of sterile syringe with a sterile 27 gauze needle. Position the syringe parallel to the tail with the needle pointing toward the body of the mouse and the bevel of the needle facing up. Proceed with the injection only if the needle is correctly inserted, which is evident from the needle sliding in with no resistance. Press down on plunger in one, continuous motion and inject the full volume of liquid within 3–7 s.</li>
<li>Serum level of HBsAg were monitored regularly from the serum of retroorbital blood. The serum was collected from the infected mice on different period of time, serum RNA was collected, and then proceeded for mRNA expression to measure the viral concentration.</li>
<li>DNA was isolated from the blood and 100 ng of DNA was used to detect HBV DNA replication using real-time PCR (RT-PCR) followed the manufacturer protocol.</li>
<li>HBV replication were detected in the serum of HHD mice (C57BL/6 background; <a href="#fig3">Figure 3</a>A) from day 3 to 35. DNA replication peaked on day 7 and then reduced gradually. HBV DNA was not cleared from the serum until day 35.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/367-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Induce HBV replication in HHD mice by hydrodynamic injection</div>
<p>Perform hydrodynamic tail vein injection of HBV plasmid on HHD mice by injecting 10 μg of plasmid in 8% body weight of PBS.</p>
<p>(A) Show serum HBV copies. On indicated time points after injection, isolate the serum from the blood and extract DNA for RT-PCR.</p>
<p>(B) Present serum HBs Ag.</p>
<p>(C) Show serum HBe Ag.</p>
<p>(D) Show serum ALT. On indicated time points after injection, isolate the serum from the blood and determine protein expression by ELISA.</p>
<p>(E) Present liver tissue histology. Euthanize mice on day 8 after HBV infection. Isolate and stain liver samples for histologic examination.</p>
<p>(F) Show quantitation of HBs Ag<sup>+</sup> cells on various days. Data shown are representative of three individual experiments (n = 5).</p>
</figcaption>
</figure>
</li>
<li>We also examined HBs Ag (<a href="#fig3">Figure 3</a>B), HBe Ag (<a href="#fig3">Figure 3</a>C), and alanine aminotransferase (ALT) (<a href="#fig3">Figure 3</a>D) from the serum by ELISA on various days. Similar to DNA replication, HBs Ag peaked on day 7 and then dropped slowly.</li>
<li>Intra-hepatic viral DNA transcription and replication were detected from the liver. Mice were sacrificed at different days of post infection, and their livers were isolated and prepared for HBV surface protein staining. In the early period post infection, expression of HBV surface protein was high (approximately 40% of all liver cells are HBs Ag<sup>+</sup>) and correlates with DNA replication data (<a href="#fig3">Figures 3</a>E and 3F). During the whole period, we detected HBV protein expression following hydrodynamic injection, although the protein expression was lower than that in the initial period.</li>
<li>Inflammatory cell infiltration was detected in the liver after viral infection. Liver samples were processed with hematoxylin and eosin (H&amp;E) staining to detect the inflammatory cells. We observed the robust infiltration of inflammatory cells into the liver in the early period followed by gradually reduction (<a href="#fig3">Figure 3</a>E, lower panel).</li>
</ol>
<h3 id="sec2.5">Detection of accumulated HBV viral Ag-specific iPSC-CTLs in the liver</h3>
<ol start="20">
<li>To exert the cytotoxic effects, HBV viral Ag-specific iPSC-CTLs need to accumulate in the liver in which viral replication initiates and expands, and this requires Ag specificity (<a href="#bib7">Tang et al., 2004</a>).</li>
<li>To accurately express human HBV TCR on mouse iPSCs, we used murine-human hybrid TCR in which the original human constant region was replaced by that of mouse. Also, for the potential recognition of human TCR in mice, we used HHD mice that express human HLA-A2.1 but lack murine major histocompatibility complex (MHC) class I molecules.</li>
<li>Following the hydrodynamic injection of pAAV/HBV1.2 DNA plasmid and adoptive cell transfer of 2 x 10<sup>6</sup> Ag-specific iPSC-CTLs after 1 week of DNA injection, we used flow cytometry to analyze the expression of HBV-specific TCR on CD8<sup>+</sup> T cells. In the spleens and livers of mice receiving HBV-specific iPSC-CTLs, CD8<sup>+</sup> T cells expressing HBV- specific TCR were clearly visualized (3.06/11.87 = 25.8% and 1.74/4.5 = 38.7% of CD8<sup>+</sup> populations, respectively) but was barely detected in those mice receiving the control cells, i.e., OVA-specific iPSC-CTLs (<a href="#fig4">Figure 4</a>A). Using immunofluorescence staining, we further visualized HBV-specific TCR expression on CD8<sup>+</sup>T cells in the livers but did not detect its expression on those of mice receiving OVA-specific iPSC- CTLs (<a href="#fig4">Figure 4</a>B).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/367-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Show accumulation of HBV viral Ag-specific iPSC-CTLs in the liver with HBV replication</div>
<p>Euthanize mice at day 8 after the adoptive transfer of T cells. Harvest the spleens or livers and isolate the intra-hepatic lymphocytes by enzymatic digestion.</p>
<p>(A) Analyze accumulation of HBV-specific T cells by flow cytometry. Make single cell suspension from the intra-hepatic lymphocytes and stain cells with fluorochrome-conjugated antibodies. Gate on CD8<sup>+</sup> T cells to analyze HBV-specific TCR.</p>
<p>(B) Show immunohistochemistry. Stain liver slides with both fluorochrome-conjugated CD8 antibody and s183 TCR pentamer, and examine under a fluorescent microscope. Green color indicates CD8 and red color indicates HBV-specific TCR. Data are representative of five mice per group of three independent experiments.</p>
</figcaption>
</figure>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>We anticipated that HBV viral Ag-specific iPSC-CTLs will reduce HBV replication <i>in vivo</i>. In order to this, we adoptively transferred HBV viral Ag-specific iPSC-CTLs to detect whether they could prevent or reduce viral replication. Mice were intravenously (i.v.) administrated pAAV/HBV1.2 DNA plasmid via hydrodynamic tail vein injection, and in the following week were i.v. adoptively transferred with HBs<sub>183–91</sub> or OVA<sub>257–264</sub>-specific DsRed<sup>+</sup>CD8<sup>+</sup> iPSC-CTLs. The viral titer was measured by RT-PCR from the serum at various time points after the adoptive transfer of T cells.</p>
<p>The expected outcomes were as followed.</p>
<p>Viral replication was significantly reduced at all time points in the mice receiving HBV viral Ag-specific T cells as compared with the mice receiving the control OVA-specific iPSC-CTLs (<a href="#fig5">Figure 5</a>A).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/367-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Demonstrate <i>in vivo</i> reduction of HBV replication by viral Ag-specific iPSC-CTLs</div>
<p>(A) Show serum HBV copies. At the indicated time points after the adoptive transfer of T cells, isolate the serum from the blood and extract DNA for RT-PCR analysis.</p>
<p>(B) Present liver tissue histology. Euthanize mice on day 8 after the adoptive transfer of T cells. Isolate liver samples and stain for histologic examination. The upper panel shows HBs Ag protein expression (↑) in infected mice (IHC staining) and the lower panel shows the inflammatory cell infiltration (H&amp;E staining).</p>
<p>(C) Present quantitation of HBs Ag positive cells at various days. Data shown are three individual experiments (n = 5).</p>
</figcaption>
</figure>
<p>We also examined the HBV surface protein expression in the livers in the above setting of treatment. Mice were euthanized at various days after the HBV injection, and the liver samples were isolated for histologic examination. Samples were stained for HBV surface protein and examined under a microscope. We observed that HBV surface protein was substantially decreased in the mice receiving HBV viral Ag-specific iPSC-CTLs, as compared with the mice receiving control cells (<a href="#fig5">Figures 5</a>B and 5C), and this is associated with inflammatory cell infiltration visualized by H&amp;E staining.</p>
<p>The mechanism behind the reduction of HBV replication by HBV viral Ag-specific iPSC-CTLs were revealed.</p>
<p>To evaluate the extent to which the antiviral effects reduce HBV replication, we examined the production of the antiviral cytokines IFN-γ and TNF-α from the intra-hepatic lymphocytes of mice receiving HBV viral Ag-specific or OVA-specific iPSC-CTLs. Intra-hepatic lymphocytes were isolated from both groups of mice and stained for IFN-γ and TNF-α. CD8<sup>+</sup> T cells were gated, and their productions of IFN-γ and TNF-α were determined by flow cytometry.</p>
<p>The number of IFN-γ-producing CD8<sup>+</sup> T cells in the livers was considerably higher in mice receiving HBV viral Ag-specific iPSC-CTLs than in mice receiving the control OVA-specific cells (3.89/22.69 = 17.1% versus &lt;1%; <a href="#fig6">Figures 6</a>A and 6C, upper panel). A similar observation was obtained in the number of TNF-α producing CD8<sup>+</sup> T cells (11.85/29.75 = 37.3% versus &lt;1%; <a href="#fig6">Figures 6</a>A and 6C, lower panel).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/367-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Identify the critical role of IFN-γ and TNF-α by viral Ag-specific iPSC-CTLs in the reduction of HBV replication</div>
<p>(A) Show IFN-γ or TNF-α producing CD8<sup>+</sup> T cells analyzed by intracellular staining and flow cytometric analysis.</p>
<p>(B) Present immunohistochemistry. Stain liver slides with fluorescence-conjugated CD8 and IFN-γ or TNF-α antibodies, and examine under a fluorescent microscope (red, CD8<sup>+</sup>; green, IFN-γ<sup>+</sup> or TNF-α<sup>+</sup>).</p>
<p>(C) Show percentage of IFN-γ or TNF-α producing CD8<sup>+</sup> T cells.</p>
<p>(D) Show IFN-γ and TNF-α producing CD8<sup>+</sup> T cells analyzed by intracellular staining and flow cytometric analysis, gating on CD8<sup>+</sup> populations. Data are representative of five mice per group of three independent experiments.</p>
</figcaption>
</figure>
<p>We prepared the liver samples for immunofluorescence staining for detection of IFN-γ or TNF-α-producing CD8<sup>+</sup> T cells. More IFN-γ or TNF-α-producing CD8<sup>+</sup> T cells were observed in the livers of mice receiving HBV viral Ag-specific iPSC-CTLs than that of mice receiving OVA-specific cells (<a href="#fig6">Figure 6</a>B).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Generation of a large number of highly reactive Ag-specific T cells are crucial for adoptive T cell transfer (ACT)-based immunotherapy of various diseases, including cancers (e.g., liver cancer) and autoimmune diseases (e.g., systemic lupus erythematous). Under the appropriate circumstance, iPSCs can produce almost all types of cells in the body, including T cells. Thus, iPSCs provide a chance to obtain a renewable source of healthy T cells for treating a wide array of diseases. However, the optimal circumstance for development of T cells from iPSCs (i.e., iPSC-T cells) has not been fully defined.</p>
<p>The <i>in vitro</i> generation of viral Ag-specific T lymphocytes from stem cells has not been optimized, and the iPSC-derived CD8<sup>+</sup> T cells did not differentiate at the same time. In the current study, we developed viral Ag-specific iPSC-CTLs and used them as adoptive cell transfer to reduce HBV replication in a murine model. We sorted DsRed<sup>+</sup>CD8<sup>+</sup> cells for adoptive cell transfer, and the DsRed<sup>+</sup>CD8<sup>+</sup> cells might include CD4<sup>+</sup>CD8<sup>+</sup> (DP) cells and other CD8<sup>+</sup> premature lymphocytes. In chronic HBV infection, the viral genome forms a stable mini-chromosome, the covalently closed circular DNA (cccDNA) that can persist throughout the lifespan of the hepatocyte. Targeting the clearance of the viral mini-chromosome may result in a cure of chronic HBV infection. But after the adoptive cell transfer, the premature lymphocytes might change CD8 expression. Nevertheless, the HBV-specific CTLs can be deleted or dysfunctional or succumb to exhaustion in patients with chronic HBV infection (<a href="#bib2">Benechet and Iannacone, 2017</a>, <a href="#bib5">Kawashima et al., 2018</a>). Moreover, recent studies showed that priming by hepatocytes, CD8<sup>+</sup> T cells differentiated into dysfunctional HBV-specific CTLs, with partial overlap with those of exhausted or tolerant T cells; thus, these HBV-specific CTLs could not be rescued by treatment with immune checkpoint inhibitors such as anti-PDL1 or CD40-mediated myeloid dendritic cells (mDCs)-activation (<a href="#bib1">Benechet et al., 2019</a>, <a href="#bib4">Isogawa et al., 2013</a>).</p>
<p>Adoptive cell transfer of the HBV-specific CTLs has been considered as a highly promising treatment for chronic HBV infection. The robust mouse models for studying HBV persistence and therapeutic intervention may be the adenovirus and adeno-associated virus (AAV)-based systems, which mimic chronic HBV infection. However, there are still disadvantages in the mouse models in which HBV viruses are transient replication in the liver, and no real HBV viral infection as well as cccDNA formation. We did not detect the obvious destruction of the hepatocytes (<a href="#fig5">Figure 5</a>B), which might be associated with HBV Ag presentation. It has been suggested that, by priming by Kupffer cells, but not hepatocytes, viral Ag-specific CD8<sup>+</sup> T cells could efficiently differentiate into functional effector cells against HBV infection (<a href="#bib1">Benechet et al., 2019</a>). <i>In vitro</i>, HBV viral Ag-specific iPSC-CTLs showed the ability to induce specified lysis of target cells, in an effector-to-target ratio-specific manner (<a href="#fig2">Figure 2</a>C), which may not represent in a physiological condition <i>in vivo</i>.</p>
<p>HBV viral Ag-specific iPSC-CTLs characteristically are same as other CTLs. After differentiating into T cells, iPSC-derived cells will not be able to indefinitely proliferate. In fact, iPSC-CTLs have similar growth rate and a confluence as normal T cells. We previously tested the <i>in vivo</i> development of Ag-specific CTLs from iPSCs and did not notice any safety issues of this method (<a href="#bib6">Lei et al., 2011</a>)</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1 (cell culture; days 0–5)</h3>
<p>Slow growth of mouse iPSCs</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Cell culture dish need to be coated properly with sufficient amount of gelatin. Coated dish needs to be in the incubator from 30 min to overnight (14–16 h)</p>
<p>Feeder cells should be confluent in all over the dish but not to be over confluent.</p>
<p>Feeder cells should be healthy and freshly cultured.</p>
<p>Fresh iPSCs should thaw with a minimum temperature and seeded on feeder cells immediately.</p>
<p>iPSCs media should be supplemented with proper supplement and media should be warmed.</p>
<p>iPSCs media should be removed carefully and colony should not be agitated.</p>
<h3 id="sec5.3">Problem 2 (generation of HBV viral Ag-specific iPSC-CTLs; days 0–28)</h3>
<p>Inefficient retroviral transduction</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Packaging cells should be healthy and freshly prepared.</p>
<p>Transfection reagent should be in optimized concentration.</p>
<p>Centrifuge and incubator should set in appropriate temperature as indicated in methods.</p>
<p>Polybrene concentration should be optimized.</p>
<h3 id="sec5.5">Problem 3 (development of HBV replication in mice; weeks 1–8)</h3>
<p>Mice may not be sufficiently infected with HBV</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Hydrodynamic delivery of virus is essential for establish HBV mouse model.</p>
<p>Proper syringe and needle should be use.</p>
<p>Large volume of virus suspension should be delivered within 3 s.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Benechet, A.P., De Simone, G., Di Lucia, P., Cilenti, F., Barbiera, G., Le Bert, N., Fumagalli, V., Lusito, E., Moalli, F., Bianchessi, V., et al. (2019). Dynamics and genomic landscape of CD8(+) T cells undergoing hepatic priming. Nature <i>574</i>, 200-205.</p>
<p id="bib2">Benechet, A.P. and Iannacone, M. (2017). Determinants of hepatic effector CD8(+) T cell dynamics. J. Hepatol. <i>66</i>, 228-233.</p>
<p id="bib3">Gehring, A.J., Xue, S.A., Ho, Z.Z., Teoh, D., Ruedl, C., Chia, A., Koh, S., Lim, S.G., Maini, M.K., Stauss, H., and Bertoletti, A. (2011). Engineering virus-specific T cells that target HBV infected hepatocytes and hepatocellular carcinoma cell lines. J. Hepatol. <i>55</i>, 103-110.</p>
<p id="bib9">Haque, M., Lei, F., Xiong, X., Ren, Y., Kumar, A., Das, J.K., Ren, X., Fang, D., de Figueiredo, P., Yang, J.M., and Song, J. (2020). Stem Cell-Derived Viral Antigen-Specific T Cells Suppress HBV Replication through Production of IFN-γ and TNF-⍺. iScience <i>23</i>, 101333.</p>
<p id="bib4">Isogawa, M., Chung, J., Murata, Y., Kakimi, K., and Chisari, F.V. (2013). CD40 activation rescues antiviral CD8(+) T cells from PD-1-mediated exhaustion. PLoS Pathog. <i>9</i>, e1003490.</p>
<p id="bib5">Kawashima, K., Isogawa, M., Hamada-Tsutsumi, S., Baudi, I., Saito, S., Nakajima, A., and Tanaka, Y. (2018). Type I interferon signaling prevents hepatitis B virus-specific T cell responses by reducing antigen expression. J. Virol. <i>92</i>, e01099-18.</p>
<p id="bib6">Lei, F., Zhao, B., Haque, R., Xiong, X., Budgeon, L., Christensen, N.D., Wu, Y., and Song, J. (2011). In vivo programming of tumor antigen-specific T lymphocytes from pluripotent stem cells to promote cancer immunosurveillance. Cancer Res. <i>71</i>, 4742-4747.</p>
<p id="bib7">Tang, Q., Henriksen, K.J., Bi, M., Finger, E.B., Szot, G., Ye, J., Masteller, E.L., McDevitt, H., Bonyhadi, M., and Bluestone, J.A. (2004). In vitro-expanded antigen-specific regulatory T cells suppress autoimmune diabetes. J. Exp. Med. <i>199</i>, 1455-1465.</p>
<p id="bib8">Yu, J., Hu, K., Smuga-Otto, K., Tian, S., Stewart, R., Slukvin, and Thomson, J.A. (2009). Human induced pluripotent stem cells free of vector and transgene sequences. Science <i>324</i>, 797-801.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Jianxun Song (<a href="mailto:jus35@tamu.edu">jus35@tamu.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate any unique materials or reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code. The authors confirm that the data supporting the findings of this study are available within the article.</p>
<h3>Acknowledgments</h3>
<p>The authors thank Dr. Pei-Jer Chen from National Taiwan University Hospital for providing the pAAV/HBV1.2 construct and Dr. Adam J. Gehring from Toronto General Hospital Research Institute for providing the s183-specific TCR genes. This work was supported by the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> grants R01AI121180, R01CA221867, and R21AI128325 to J.S.</p>
<h3>Author Contributions</h3>
<p>J.S. designed the experiments, analyzed data, and contributed to the writing of the paper. M.H., F.L., and J.K.D. performed the experiments. X.X. and M.H. analyzed data.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>