<article>
<section>
<h1 id="header">Analyzing muscle structure and function throughout the larval instars in live <i>Drosophila</i>
</h1>
<p><time datetime="2021-01-22">Published: January 22, 2021</time></p>
<p>Mridula Balakrishnan,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Whitney J. Sisso,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Mary K. Baylies<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Biochemistry, Cell &amp; Developmental Biology, and Molecular Biology (BCMB) program, Weill Cornell Graduate School of Medical Sciences, New York, NY 10065, USA</p>
<p id="aff2"><sup>2</sup>Developmental Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:mridula.balakrishnan@mpi-bn.mpg.de">mridula.balakrishnan@mpi-bn.mpg.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:bayliesm@mskcc.org">bayliesm@mskcc.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100291">10.1016/j.xpro.2020.100291</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p><i>Drosophila</i> larval musculature is a genetically and optically accessible system to study muscle development. Each larval muscle is a single fiber with conserved cytoarchitecture, including its sarcomere structure and composition. Here, we present a workflow for systematically analyzing muscle structure and function at discrete larval stages, as well as throughout the larval instars, using both newly developed and adapted methods.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Balakrishnan et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detailed methods for preparation and analysis of <i>Drosophila</i> larval muscle structure</li>
<li>Methods for muscle structure and function analysis in the same larva over time</li>
<li>Steps for analyzing sarcomere size and organization</li>
<li>Pitfalls and troubleshooting approaches described</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/423-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Larval muscle structure and function have been examined using numerous techniques at different instars. Previous techniques have examined muscle structure by larval dissection, fixation, and immunostaining, by heat-fixing larvae expressing fluorescent constructs, or by anesthetizing a larva at different instars to examine changes in muscle structure throughout development (<a href="#bib4">Brent et al., 2009</a>; <a href="#bib11">Kakanj et al., 2020</a>; <a href="#bib17">Schnorrer et al., 2010</a>; <a href="#bib7">Choi et al., 2014</a>; <a href="#bib10">Fernandes and Schöck, 2014</a>). Similarly, a low-cost technique to examine larval muscle function has been described previously (<a href="#bib5">Brooks et al., 2016</a>).</p>
<p>Here we describe a detailed workflow for systematically analyzing both muscle structure and function either at discrete larval stages or throughout the larval instars, using both newly developed and adapted methods. In comparison to previously published methods which enable visualization of muscle structure and function at discrete stages, our protocol can be used to observe changes in muscle structure or muscle function in the same larva throughout its development. We describe in detail the techniques used to isolate, store, and grow individual larva over time; likewise, we explain the methods used to quantify muscle and sarcomere structures such as muscle length, sarcomere width, and number from the embryonic period (when sarcomeres are established), as well as larval velocity, a proxy for muscle function, over time. We detail the procedures to collect and maintain the larva individually as they develop through the instars. This isolation of the individual larva is essential for monitoring changes in muscle structure and larval locomotion in a single larva throughout development. We recommend that the materials listed be acquired and that the following steps are followed.</p>
<h3 id="sec1.1">Setting up laying pots</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<ol>
<li>Set up laying pots and embryo collection plates
<ol type="a">
<li>Prepare Laying pot
<ol type="i">
<li>Punch 100 mL plastic beaker (for 6 cm wide petri dishes) with holes to facilitate air exchange and prevent condensation. Holes are made using a hot syringe needle.</li>
</ol>
</li>
<li>Prepare Embryo collection plates (apple juice plates)
<ol type="i">
<li>Microwave 1,500 mL of ddH<sub>2</sub>O, 50 g of granulated sugar, and 45 g of agar to form a homogenous solution. Add 500 mL of apple juice once the solution reaches 65°C, then add 40 mL of 10% Tegosept in 100% ethanol. Pour solution into 6 cm or 10 cm wide petri dishes.</li>
</ol>
</li>
<li>Prepare Yeast paste
<ol type="i">
<li>Stir distilled water with dry baker’s yeast to make a paste. Store at 4°C.</li>
</ol>
</li>
<li>Spread a small quantity of yeast paste on a solidified apple juice plate.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Collecting flies for laying pots
<ol type="a">
<li>Raise flies of the desired genotypes in a light-dark incubator (12 h light:12 h dark) at 25°C.</li>
<li>Collect virgin females (♀) and males (♂) of the respective genotypes and store them separately.</li>
<li>Add flies to the laying pots and seal with embryo collection plate and rubber bands.
<div class="note">
<span class="note-title">Note:</span> The number of male flies added is half the number of female virgins added, e.g., for approximately 60 virgin females, 30–35 male flies are added to a laying pot.</div>
</li>
<li>Place laying pots in a 25°C in a light-dark incubator, changing the agar plates every 24 h.
<div class="note">
<span class="note-title">Note:</span> Prior to embryo collection, place laying pots at 25°C incubator for 2 days in order for the flies to acclimatize and mate, e.g., if planning on collecting embryos from Monday, set up laying pot in 25°C incubator on Friday evening changing plates every 24 h.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Larval collection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="3">
<li>Change the apple juice plates of your laying pots in ∼20 h from your last exchange.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Changing the apple juice plates within ∼20 h of your last exchange only applies to the days you plan on collecting larvae for experiments. On all other days, changing plates every 24 h is sufficient.</div>
<ol start="4">
<li>Examine the collection plate to ensure that there are no larvae present and then shift the plate with embryos to 25°C.</li>
<li>Examine collection plate after 2 h for any larvae that have hatched and isolate the newly hatched 1<sup>st</sup> instar larvae.</li>
<li>Transfer the 1<sup>st</sup> instar larvae into fresh vials with food and incubate at 25°C in a light-dark incubator.</li>
<li>Remove larvae at the desired instar stage (2<sup>nd</sup>, early 3<sup>rd</sup> or late 3<sup>rd</sup> instar) by floating them away from the food using 15% sucrose solution. To do this, pour the sucrose solution into the vial and gently disturb the food at the bottom of the vial using a spatula. Floating larvae can be collected and transferred using a paintbrush.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Store 15% sucrose at 4°C to prevent contamination by microorganisms.</div>
<ol start="8">
<li>Clean larvae gently by immersing and swishing them in a watch glass containing ddH<sub>2</sub>O. Transfer cleaned larvae into a fresh apple juice plate at room temperature.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For further information and images on laying pot setup and larval collection please refer to <a href="#bib1">Azevedo et al. (2016)</a>.</div>
<h3 id="sec1.3">Setting up chambers/plates with food for larval isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="9">
<li>Empty food from fresh vials (∼3–4) or a bottle into a beaker (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/423-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Larval isolation plate setup</div>
<p>(A) Empty fly food from bottle into a glass beaker using spatula.</p>
<p>(B) Microwave until food completely melts.</p>
<p>(C) Add water to food.</p>
<p>(D) Mix food well.</p>
<p>(E) Pipette ~2.5 mL of food into each well of a 6-well plate.</p>
<p>(F) Once food solidifies transfer a larva into each well. Seal plate with plastic wrap and punch holes using syringe needle.</p>
</figcaption>
</figure>
</li>
<li>Heat food in a microwave in short bursts until food has completely melted (<a href="#fig1">Figures 1</a>B and 1C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Add ∼25 mL of water to food once melted to ensure that food does not dry out.</div>
<ol start="11">
<li>Pipette ∼2.5 mL of food into each well of a 6-well plate (<a href="#fig1">Figures 1</a>D and 1E).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Food in each well should sufficiently cover the bottom of the well and be approximately 0.25 cm thick.</div>
<ol start="12">
<li>After food has cooled and solidified, transfer a larva into each well using a paintbrush.</li>
<li>Seal plate using self-sealing plastic wrap and punch holes in the wrap over each well using a fine syringe needle (<a href="#fig1">Figure 1</a>F).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Holes are punched to facilitate air exchange, around 10 holes per well is sufficient. If the holes are too large, larvae can crawl out of the holes.</div>
<ol start="14">
<li>Incubate plate at 25°C in a light-dark incubator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To prevent food from drying out, place plates in a sealable container (either Pyrex/plastic) lined with wet paper towels. Refresh towels as they dry out.</div>
<h3 id="sec1.4">Anesthetizing larvae to visualize muscle structure at different larval stages</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="15">
<li>Set up laying pots of the desired genotypes.</li>
<li>Set up larval collection plates for each of the genotypes.</li>
<li>Fill wells of 6-well culture plate with Kwan Loong Medicated Oil, an anesthetic, and cover plate. Detailed steps on larval anesthetization and the subsequent imaging of their musculature are explained in the <a href="#step-by-step-method-details">Step-by-step method details</a> section.</li>
</ol>
<h3 id="sec1.5">Imaging platform to visualize larval movement</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<ol start="18">
<li>Set up imaging platform.
<ol type="a">
<li>Platform should be wide enough to accommodate a 10 cm apple juice plate at the bottom (<a href="#fig2">Figure 2</a>A). For dimensions of platform see <a href="#materials-and-equipment">Materials and equipment</a> section (<a href="#fig2">Figures 2</a>A and 2B).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/423-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Imaging platform setup and dimensions</div>
<p>(A) Lateral view image of the locomotion platform along with dimensions of platform legs and platform height.</p>
<p>(B) Top view image of the locomotion platform along with dimensions of platform surface used to hold smartphone used to acquire larval crawling movies.</p>
</figcaption>
</figure>
</li>
<li>Ensure adequate number of 10 cm apple juice plates are at room temperature (1 plate per genotype/10 larvae). Detailed steps on larval locomotion analysis and its quantification are explained in the <a href="#step-by-step-method-details">Step-by-step method details</a> section.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend collecting embryos and larvae from laying pots for no more than one week as fertility of flies decreases significantly after a week.</div>
<div class="note">
<span class="note-title">Note:</span> While in our experiments we incubate embryos, larvae, and flies at 25°C, incubation at lower (i.e., 18°C) and higher temperatures (i.e., 28°C) can also be done. However, development is delayed (18°C) or accelerated (28°C) at these temperatures, and hence larval instars can progress slowly or quickly (<a href="#bib15">Pulver and Berni, 2012</a>).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Active dry yeast</td>
<td>Fleischmann's</td>
<td>Cat# 2192</td>
</tr>
<tr>
<td>Agar</td>
<td>Moorehead and Company</td>
<td>Cat# 700B</td>
</tr>
<tr>
<td>Tegosept (hydrobenzoic acid methyl ester)</td>
<td>Sigma</td>
<td>Cat# H5501</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Fisher Scientific</td>
<td>Cat# BP220-1</td>
</tr>
<tr>
<td>Apple juice</td>
<td>Monarch</td>
<td>Cat# 515815</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Fisher Scientific</td>
<td>Cat# 04-355-223</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Fisher Scientific</td>
<td>Cat# BP2291</td>
</tr>
<tr>
<td>Kwan looong medicated oil</td>
<td>Kwan looong</td>
<td><a href="http://www.kwanloongoil.com/">http://www.kwanloongoil.com/</a></td>
</tr>
<tr>
<td>10× PBS</td>
<td>Fisher Scientific</td>
<td>Cat# BP3994</td>
</tr>
<tr>
<td>Bleach, concentrated Clorox (30966)</td>
<td>Owens and Minor</td>
<td>Cat# 5406</td>
</tr>
<tr>
<td>Halocarbon oil 700</td>
<td>Sigma</td>
<td>Cat# H8898</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>D. melanogaster</i>: <i>Mhc-Gal4</i>
</td>
<td>Bloomington <i>Drosophila</i> Stock Center</td>
<td>BDSC# 67044</td>
</tr>
<tr>
<td>
<i>D. melanogaster</i>: <i>UAS-2x-GFP</i>
</td>
<td>Bloomington <i>Drosophila</i> Stock Center</td>
<td>BDSC # 6874</td>
</tr>
<tr>
<td>
<i>D. melanogaster</i>: <i>Zasp66::GFP (ZCL0663)</i>
</td>
<td>Bloomington <i>Drosophila</i> Stock Center</td>
<td>BDSC #6824</td>
</tr>
<tr>
<td>
<i>D. melanogaster</i>:<i>α-actinin::GFP</i>
</td>
<td>Bloomington <i>Drosophila</i> Stock Center</td>
<td>BDSC #51573</td>
</tr>
<tr>
<td>
<i>D. melanogaster</i>: <i>UAS-moesin::mCherry</i>
</td>
<td>(<a href="#bib12">Millard and Martin, 2008</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ/Fiji</td>
<td>NIH</td>
<td>
<a href="https://fiji.sc">https://fiji.sc</a> ; RRID: SCR_002285</td>
</tr>
<tr>
<td>GraphPad Prism</td>
<td>GraphPad</td>
<td>
<a href="http://www.graphpad.com">http://www.graphpad.com</a>;RRID:SCR_002798</td>
</tr>
<tr>
<td>Photoshop</td>
<td>Adobe</td>
<td>
<a href="https://www.adobe.com/products/photoshop.html">https://www.adobe.com/products/photoshop.html</a>; RRID:SCR_014199</td>
</tr>
<tr>
<td>Illustrator</td>
<td>Adobe</td>
<td>
<a href="http://www.adobe.com/products/illustrator.html">http://www.adobe.com/products/illustrator.html</a>; RRID:SCR_010279</td>
</tr>
<tr>
<td>Excel</td>
<td>Microsoft</td>
<td>
<a href="https://www.microsoft.com/en-gb/">https://www.microsoft.com/en-gb/</a>; RRID:SCR_016137</td>
</tr>
<tr>
<td>MPEG Streamclip</td>
<td>Squared 5</td>
<td><a href="http://www.squared5.com/">http://www.squared5.com/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>6-well cell culture plates</td>
<td>Thermo Scientific</td>
<td>Cat# 130184</td>
</tr>
<tr>
<td>Precision Glide Sub-Q Needle 26G × 5/8 inch</td>
<td>Becton, Dickinson and Company</td>
<td>Cat# 305115</td>
</tr>
<tr>
<td>6 cm petri dish (60 mm × 15 mm)</td>
<td>Fisher Scientific</td>
<td>Cat# 08-757-100B</td>
</tr>
<tr>
<td>10 cm petri dish (100 mm × 15 mm)</td>
<td>Fisher Scientific</td>
<td>Cat# FB0875712</td>
</tr>
<tr>
<td>Fly vials</td>
<td>Crystalgen</td>
<td>Cat# 23-5056</td>
</tr>
<tr>
<td>Fly bottles</td>
<td>Crystalgen</td>
<td>Cat# 5065</td>
</tr>
<tr>
<td>Spoonula</td>
<td>Fisher Scientific</td>
<td>Cat# S35888</td>
</tr>
<tr>
<td>Paintbrush size 0</td>
<td>Blick</td>
<td>Cat# 05565-1000</td>
</tr>
<tr>
<td>Beaker (150 mL glass)</td>
<td>Fisher Scientific</td>
<td>Cat# 02-255-25A</td>
</tr>
<tr>
<td>Beaker (100 mL plastic)</td>
<td>Globe Scientific</td>
<td>Cat# 3641</td>
</tr>
<tr>
<td>Forceps (#5 Dumont)</td>
<td>Fine Science Tools</td>
<td>Cat# 11251-20</td>
</tr>
<tr>
<td>Microscope glass slide (3 × 1 × 1 mm)</td>
<td>Fisher Scientific</td>
<td>Cat# 12-544-3</td>
</tr>
<tr>
<td>Coverslip (22 mm × 40 mm)</td>
<td>Fisher Scientific</td>
<td>Cat# 12-544-BP</td>
</tr>
<tr>
<td>Plastic wrap (Press ’n Seal)</td>
<td>Glad</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Imaging platform</h3>
<p>The imaging platform was 3D printed to the dimensions as indicated (<a href="#fig2">Figures 2</a>A and 2B).</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Analyzing muscle function throughout larval development</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–45 min/day ×4 days</div>
<p>The steps below describe how to setup and record larval movement for muscle function analysis. Refer to steps 15–18 in the <a href="#before-you-begin">Before you begin</a> section before starting.</p>
<ol>
<li>Isolate 1<sup>st</sup> instar larvae of your desired genotypes from your vial/plate. Clean larvae gently by immersing and swishing them in a watch glass containing ddH<sub>2</sub>O and transfer to a clean apple juice plate.</li>
<li>Set up imaging platform as shown (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/423-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Assessing muscle function through larval locomotion</div>
<p>(A) Set up imaging platform, agar plate, and smartphone as indicated.</p>
<p>(B) Image of optional dartboard paper used for recording final larval movement.</p>
<p>(C) Setup of dartboard paper placed under agar plate.</p>
<p>(D) Image of agar plate and ruler beside it to set scale during the experiments.</p>
<p>(E) Place larva at the center of the plate/dart board paper and record movement.</p>
</figcaption>
</figure>
</li>
<li>Place a fresh 10 cm wide apple juice agar plate below the imaging platform (<a href="#fig3">Figure 3</a>A).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> An additional aid to help record larval crawling data is a dartboard paper, a circular piece of paper the same size as agar plate divided into concentric circles. Each circle in the dartboard is numbered, recording how far a larva moves with respect to the dartboard can prevent further confusion when handling large numbers of larvae (<a href="#fig3">Figures 3</a>B and 3C).</div>
<ol start="4">
<li>Place a ruler/measuring tape alongside the 10 cm plate. Acquire an image of the agar plate with ruler beside it (<a href="#fig3">Figure 3</a>D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The ruler or measuring tape is required to set scale and measure distance traveled by the larvae in standard units (cm, mm, etc.)</div>
<ol start="5">
<li>Place larva from the first well of genotype of interest at the center of apple juice agar plate (<a href="#fig3">Figure 3</a>E).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is not required to provide an attractant for the larvae, the larvae crawl both in the presence and absence of an attractant with no significant difference in velocities observed between the two groups.</div>
<ol start="6">
<li>Start recording larval movement using a smartphone or camera for 45 s.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Choose a time period for tracking larval movement, e.g., 30, 45, 60 s, etc., based on your preferences. On selection of a time period, keep the time constant throughout all tracking experiments. We do not recommend choosing a time under 30 s as larvae often need time to acclimatize and begin moving. Allowing the larvae 2 s to acclimatize before recording their movement is beneficial.</div>
<ol start="7">
<li>Stop recording larval movement once 45 s is over or when the larva reaches the edge of the plate, whichever comes first.</li>
<li>Remove larva from plate using paintbrush and transfer back to well 1. Transfer larva from well 2 to the plate and repeat steps 5–7 until all larvae of the genotype/plate have been recorded.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our experiments we tracked 8–10 larvae per genotype.</div>
<ol start="9">
<li>Change apple juice plate before tracking larvae from the next genotype.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As larvae crawl on the apple juice, plate they leave their tracks on the plate. Over time, subsequent larvae follow the same tracks skewing the results. Track a maximum of 8 larvae on a single plate.</div>
<ol start="10">
<li>Repeat steps 4–8 till all larvae from all genotypes/plates are tracked. Place plates with larvae back at 25°C.</li>
<li>After 24 h, isolate 2<sup>nd</sup> instar larvae and repeat steps 1–10. Repeat the process every 24 h till larvae reach late 3<sup>rd</sup> instar stage.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Day 1, 1<sup>st</sup> instar; day 2, 2<sup>nd</sup> instar; day 3, early 3<sup>rd</sup> instar; day 4, late 3<sup>rd</sup> instar.</div>
<p>For more details on how temperature can influence development, please refer to <a href="#fig4">Figure 4</a>A.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/423-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. <i>Drosophila</i> development and larval musculature</div>
<p>(A) Illustration depicting the influence of temperature on <i>Drosophila</i> development.</p>
<p>(B) Image of a heat-fixed 3<sup>rd</sup> instar larva expressing Tropomyosin-GFP with the different thoracic and abdominal hemisegments labeled. Scale bar, 100 μm.</p>
<p>(C) Schematic diagram depicting muscles from abdominal hemisegments (A1–A5) of the larva when viewed externally.</p>
<p>(D) Schematic diagram depicting muscles from abdominal hemisegments (A1–A5) of the larva when viewed internally.</p>
<p>Muscle names are as follows: muscle position (D, dorsal; V, ventral; L, lateral) followed by orientation (A, acute; L, longitudinal; O, oblique; T, transverse); SBM, segment border muscle.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> While currently there is no evidence to suggest that larval movement is affected by circadian rhythms, ensure that larval tracking occurs during the light cycle of the larva (<a href="#bib16">Sawin et al., 1994</a>). It is recommended to track the larvae at the same time every day.</div>
<ol start="12">
<li>Once late 3<sup>rd</sup> instar larvae have been tracked, dissect, fix, stain, and mount each larva individually as described in <a href="#bib1">Azevedo et al. (2016)</a>. Examine muscle structure and count number of muscles with structural defects.</li>
<li>Quantify muscle structure defects by scoring the number of defective muscles present in the larval abdominal hemisegments A2-A7 (<a href="#fig4">Figures 4</a>B–4D). The affected muscles were divided into different classes/groups based on severity of defects. The percentages of muscles affected were plotted as graphs, and statistical significance was calculated using two-way ANOVA.</li>
<li>The above experiment was repeated again using a different group of larvae. Quantification of muscle function and larval tracking analysis are explained in detail in the <a href="#quantification-and-statistical-analysis">Quantification and statistical analysis</a> section.</li>
</ol>
<h3 id="sec3.2">Anesthetizing larvae to visualize muscle structure at different larval stages</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–45 min/day × 4 days</div>
<p>The steps below describe how to anesthetize a larva and image its musculature throughout larval development.</p>
<ol start="15">
<li>Isolate 1<sup>st</sup> instar larvae of desired genotype and clean larvae gently by immersing and swishing them in a watch glass containing ddH<sub>2</sub>O.</li>
<li>Transfer and place larva on the lid of 6-well plate containing Kwan Loong Medicated Oil , such that the larva is placed directly over the well containing the oil.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure that the larva does not dry out on the lid. Adding droplets of water to the larva using a paintbrush can help prevent larva from drying out.</div>
<ol start="17">
<li>Incubate larva in the chamber until it is anesthetized, i.e., stops moving when disturbed with a paintbrush.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Incubation times of larvae greatly vary, so need optimization. Here, we present sample incubation times experienced with a larva; e.g., 1<sup>st</sup> instar, 3–5 min; 2<sup>nd</sup> instar, 5–8 min; early 3<sup>rd</sup> instar, 10–12 min; late 3<sup>rd</sup> instar, 15 min.</div>
<ol start="18">
<li>Transfer anesthetized larvae to a microscope slide containing 50% glycerol and mount with a cover slip.</li>
<li>Record orientation of mounted larva (e.g., dorsal side up or down, head and mouth left or right) and identify the hemisegments of the larval musculature to be imaged. Image hemisegment(s) and record hemisegment numbers.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> While imaging, take care not to squish or squeeze the larva, as it can lead to larval death.</div>
<ol start="20">
<li>After imaging, carefully remove coverslip from larva, remove larva, and wash gently with 1× PBS, and then with ddH<sub>2</sub>O.</li>
<li>Transfer larva to clean apple juice plate until it awakens. Once larva begins moving transfer larva to the first well of larval collection plate.</li>
<li>Isolate second larva and repeat steps 15–21, until all larvae of desired genotypes have been anesthetized and imaged.</li>
<li>Move larval collection plate at 25°C in light-dark incubator.</li>
<li>After 24 h isolate 2<sup>nd</sup> instar larvae and repeat steps 15–23. Repeat the process every 24 h till larvae reach late 3<sup>rd</sup> instar stage.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When imaging the same larva at successive instars, ensure that the larva is mounted in the same orientation as done for the 1<sup>st</sup> instar larva. Image the same hemisegments imaged in the 1<sup>st</sup> instar larva.</div>
<div class="note">
<span class="note-title">Note:</span> Day 1, 1<sup>st</sup> instar; day 2, 2<sup>nd</sup> instar; day 3, early 3<sup>rd</sup> instar; day 4, late 3<sup>rd</sup> instar.</div>
<h3 id="sec3.3">Visualizing and quantifying sarcomere structure and size from the late embryonic stage throughout the larval stages using heat fixation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min/day ×4 days</div>
<p>These steps describe how to analyze and image muscle structure analysis at embryonic and larval stages.</p>
<ol start="25">
<li>Set up laying pots of appropriate genotype containing fluorescent reporter(s) (e.g., Zasp-GFP) at 25°C.</li>
<li>Label and change apple juice plates on laying pots within 20 h, keeping the old ones for embryo and larval collection.</li>
<li>Isolate 1<sup>st</sup> instar larvae and distribute larvae into 3 different vials and an apple juice plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We isolated and transferred ∼10 larvae into each vial and apple juice plate.</div>
<ol start="28">
<li>Use the remaining embryos on the plate for collection.
<ol type="a">
<li>Pour enough water to cover embryos on plate and remove dead flies using paintbrush.</li>
<li>Gently disturb the embryos on the plate using a paintbrush and then transfer them to a collection basket.</li>
<li>Place collection baskets in a plate with 50% bleach and dechorionate embryos for 4 min at room temperature.</li>
<li>Rinse the embryos in the collection basket with distilled water repeatedly to remove any traces of bleach from the embryos.</li>
</ol>
</li>
<li>Heat Fixation
<ol type="a">
<li>Embryos: Dip collection baskets with embryos into a water bath at 65°C for 1–2 s, ensuring that embryos are submerged in water.</li>
<li>Larvae: Pick up larvae of any developmental stage by the tail or head using forceps and submerge into a water bath at 65°C for 1–2 s.</li>
</ol>
</li>
<li>Mounting
<ol type="a">
<li>Embryos: Transfer embryos from the collection basket to a microscope slide using a paintbrush and then spread on the slide. Mount in either halocarbon oil or 50% glycerol, and cover with a coverslip.</li>
<li>Larvae: Transfer larvae onto a microscope slide containing halocarbon oil or 50% glycerol. Orient larvae such that their dorsal sides (containing trachea) are either parallel or perpendicular to the long axis of the slide, and cover with a cover slip.</li>
</ol>
</li>
<li>Image the embryos or larvae immediately using a microscope (<a href="#fig5">Figures 5</a>A and 5B).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/423-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Visualization and quantification of muscle and sarcomere structure</div>
<p>(A) Image of sarcomeres and muscle structure in a heat-fixed stage 17 embryo expressing Zasp-GFP (green). Scale bar, 20 μm.</p>
<p>(B) Image of heat-fixed 2<sup>nd</sup> instar larva expressing Zasp-GFP. Zasp-GFP marks sarcomeres. White rectangle highlights a single hemisegments in the larva. Scale bar, 20 μm.</p>
<p>(C) High-magnification image of a single hemisegments from a heat-fixed 3<sup>rd</sup> instar larva opened in Fiji for further analysis. White box highlights 2 ventral longitudinal (VL) muscles, VL1 and VL2, used for muscle and sarcomere structure analysis. Scale bar, 25 μm.</p>
<p>(D) Using line tool in Fiji. White line is drawn from one end of VL muscle to the other. Scale bar, 25 μm.</p>
<p>(E) Measuring length of the line using Measure function under Analyze tool, generates information about line draw in a new window. Red box highlights the length of the line, i.e., muscle length which is then plotted. Scale bar, 25 μm.</p>
<p>(F) Image displaying the results of measuring sarcomere length using the line tool. Red box highlights the length of the line, i.e., sarcomere length which is then plotted.</p>
<p>(G) Image of a larval muscle segment expressing Zasp-GFP to count number of sarcomeres using Cell counter plugin. Red box highlights the number sarcomeres counted, i.e., number of blue dots on the muscle, in a segment of the muscle.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Both embryos and larvae were imaged on an inverted LSM 700 laser-scanning confocal microscope using either the 10×/0.30 NA or 20×/0.8 NA air objectives.</div>
<ol start="32">
<li>Further analysis of muscle structure such as muscle length, sarcomere size, and number can be performed upon image acquisition.
<ol type="a">
<li>Muscle length
<ol type="i">
<li>Open image of muscle in Fiji (<a href="#fig5">Figure 5</a>C).</li>
<li>Using Line tool draw a line from one end of the muscle to other (<a href="#fig5">Figure 5</a>D).
<div class="note">
<span class="note-title">Note:</span> Sometimes it is difficult to determine muscle ends from a single fluorescent reporter line. Using an additional marker such as moesin-mCherry (<a href="#bib12">Millard and Martin, 2008</a>) aids in visualization of the muscle ends.</div>
</li>
<li>To register each measurement, select: Analyze&gt; Measure (a new window will pop up when this is assessed for the first time) (<a href="#fig5">Figure 5</a>E).</li>
<li>Copy all measurements into Excel, and plot graphs using wither Excel or GraphPad Prism. Statistical significance of muscle length between genotypes at different stages were calculated using multiple t tests.</li>
</ol>
</li>
<li>Sarcomere width
<ol type="i">
<li>Open image of muscle in Fiji.
<div class="note">
<span class="note-title">Note:</span> To measure sarcomere width or number, it is preferable to use a fluorescent Z-disc marker such as Zasp-GFP or α-actinin-GFP (<a href="#bib8">Clark and Kadrmas, 2013</a>).</div>
</li>
<li>Using a line tool draw a line from one Z-disc to the next Z-disc. Register each measurement by Select: Analyze&gt; Measure. Repeat the step by drawing a line from each Z-disc to its neighbor throughout the length of the muscle (<a href="#fig5">Figure 5</a>F).</li>
<li>Copy all measurement values into an Excel sheet. Calculate average sarcomere size by averaging the measurements of sarcomere size from a single muscle. Repeat the above steps for subsequent muscles.
<div class="note">
<span class="note-title">Note:</span> Alternatively, another option is to draw a line from one end of the muscle to the other. Register the measurement by selecting Analyze &gt; Measure. Count the number of Z discs, i.e., sarcomere number. To calculate average sarcomere size in the muscle, use the following formula: Muscle length/Sarcomere number.</div>
</li>
<li>In our experiments, we calculated average sarcomere sizes for at least 10 different muscles from 5 different larvae.</li>
</ol>
</li>
<li>Sarcomere number
<ol type="i">
<li>Open image of muscle in Fiji.</li>
<li>Open the Cell counter plugin by selecting Plugins&gt; Cell Counter (a new window will pop up when this plugin is selected). Select initialize option followed by Type 1 option in the pop-up window.</li>
<li>Begin clicking all the Z lines within the muscle. Once all the Z lines are counted, the number next to the Type 1 option in the pop-up window indicates the number of sarcomeres.</li>
<li>Select Type 2 option and then quantify the sarcomere number in the next muscle (<a href="#fig5">Figure 5</a>G). Once all muscles in an image are quantified, open new image and repeat above steps.</li>
<li>Copy all measurement values into an Excel sheet. Calculate average sarcomere number by averaging the number of sarcomeres from different muscles. Graphs were plotted in Excel or in GraphPad Prism.</li>
<li>In our experiments, we calculated sarcomere numbers from at least 10 different muscles from 5 different larvae for each genotype. Statistical significance of sarcomere number between genotypes and/or at different stages were calculated using either Student’s t test or multiple t tests.</li>
</ol>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol describes how <i>Drosophila</i> larval muscle function and structure can be quantified and analyzed, respectively. Quantification of larval crawling velocity by tracking larval movement serves as a proxy for muscle function and can help understand how mutations in genes involved in muscle development and function can affect muscle contraction, thereby impairing larval movement. In control larvae, larval locomotion velocity increases as the larvae progress through the instars [<a href="#fig5">Figure 5</a>I, (<a href="#bib2">Balakrishnan et al., 2020</a>)]. This increase in locomotion velocity throughout larval stages is consistent with an increase in muscle size that occurs during the instars (<a href="#bib2">Balakrishnan et al., 2020</a>, <a href="#bib9">Demontis and Perrimon, 2009</a>, <a href="#bib3">Bawa et al., 2020</a>). Mutations in genes affecting larval locomotion thus far have predominantly decreased locomotion, with very few mutations identified that increase locomotion of the larva (<a href="#bib6">Camuglia et al., 2018</a>). Measuring larval velocity throughout the larval instars can identify when the first defects in muscle function are observed, and if these defects coincide with the defects observed in muscle structure.</p>
<p>Examining muscle structure in control larvae highlights that both sarcomere and muscle structure, which are established in the late embryonic stages, are maintained throughout the larval stages with an increase in sarcomere number as muscles grow (<a href="#bib2">Balakrishnan et al., 2020</a>). In control muscles, sarcomere size remains the same as muscles increase in size (<a href="#bib2">Balakrishnan et al., 2020</a>). However, muscles harboring mutations often show defects in sarcomere/muscle structure, which are observed as early as the embryonic stage or 1<sup>st</sup> and 2<sup>nd</sup> larval instars (<a href="#bib2">Balakrishnan et al., 2020</a>). While muscle defects observed in the embryonic musculature are indicative of perturbations in muscle formation, defects during the 1<sup>st</sup> or 2<sup>nd</sup> instar stages are suggestive of defects in muscle growth and/or maintenance (<a href="#bib2">Balakrishnan et al., 2020</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">Analyzing muscle function throughout larval development</h3>
<ol>
<li>Convert video files to image sequence format (jpeg) using a video converting software such as MPEG Streamclip. Ensure that all jpeg files from the same video file are saved in one folder (<a href="#fig6">Figures 6</a>A and 6B).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/423-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Quantification of muscle function through larval locomotion analysis</div>
<p>(A) List of folders and larval crawling videos from a genotype. Red box highlights a folder containing all files of a single larva.</p>
<p>(B) List of images (.jpeg) obtained from converting a larval crawling video into an image sequence format. Each .jpeg file is a single second of the larval crawling video.</p>
<p>(C) Set scale for the experiment by drawing a line of 1 cm on ruler.</p>
<p>(D) Scale is set by equating distance in pixels of the line to its metric length of 1 cm.</p>
<p>(E) Manual Tracking option under the Tracking plugin is selected, z calibration is entered, and track is added.</p>
<p>(F) Dialog box displaying results after tracking the first second of larval crawling. Note that the velocity is −1.00 as this is the reference point of the larva and this value is discarded during average velocity calculation (highlighted by red box) .</p>
<p>(G) Dialog box displaying results from tracking first 2 s of larval crawling.</p>
<p>(H) Dialog box displaying results from tracking the 45 s of larval crawling, along with the path taken by the larva. Red box highlights the velocity column that is used to average the velocity of the larva that is then plotted.</p>
<p>(I) Crawling velocities of larvae at different larval stages. p values by multiple t tests. <i>Mhc-Gal4</i> is used to drive expression of <i>UAS-2x-GFP</i> in the muscles. Error bars, mean ±SEM.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When converting from video to image sequence, format select output as 1 frame/s. This will result in every folder (1 per larva) containing files of each frame, e.g., 45 jpeg files if larva was imaged for 45 s.</div>
<ol start="2">
<li>To set scale, open image of apple juice plate with ruler beside it by dragging it into the Fiji toolbar. Select line tool and draw line along ruler for a distance of 1 cm. Select: Analyze&gt;Set Scale. In the dialog box set known distance to 1.00 and Unit of length as cm. Note the distance in pixels, e.g., 112. Select the Global option. The scale is indicated at the bottom of the box, e.g., 112 pixel/cm. Accept all changes (<a href="#fig6">Figures 6</a>C and 6D).</li>
<li>To begin manual larval tracking, Select: Plugins &gt; Tracking &gt; Manual Tracking. In the parameters section of the dialog box Time interval is 1 s and z calibration is 1/112 (distance in pixels) (<a href="#fig6">Figure 6</a>E).</li>
<li>Open folder containing jpeg images of larval crawling by dragging it into the Fiji toolbar and check the use Virtual Stack option in the dialog box. Once the larval sequence loads, select the Add Track option in the manual tracking dialog box.</li>
<li>To begin tracking, select Add Track in the manual tracking window. Then identify the mouth parts of the larva (black structures) and click on it. A dialog box will appear that displays the X and Y coordinates along with velocity and distance traveled at that time point. The image also automatically moves to the next frame (<a href="#fig6">Figure 6</a>F).</li>
<li>In the subsequent frames, repeatedly click on the mouth parts of the larva again till the last frame. The last frame will display the track/path taken by the larva along with a dialog box containing all the measurements (<a href="#fig6">Figure 6</a>G).</li>
<li>Copy all measurements and paste them in an Excel sheet. Calculate average velocity (cm/s) by averaging velocity values from Slice 2-45. For velocity in mm/s multiple average velocity(cm/s) × 10 (<a href="#fig6">Figure 6</a>H).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The first velocity measurement is excluded from the quantification as the measurement is negative and is used as the starting point of the larval analysis.</div>
<ol start="8">
<li>Repeat the above steps for the remaining larvae and the other genotypes.</li>
<li>Average velocities (mm/s) were plotted using GraphPad Prism. Statistical significance between genotypes at different larval stages were calculated using multiple t tests (<a href="#fig6">Figure 6</a>I).</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Muscle function can be affected by defects in the musculature or in the nervous system, particularly in the motor neurons that innervate the musculature. Hence, while assessing zygotic mutants for muscle function using the larval crawling assay, it is important to confirm any observed changes in muscle function with muscle structure analysis. It is also important to follow up any observed differences in muscle function in zygotic mutants with mutants where the gene is specifically knocked down in the muscle.</p>
<p>Examination of muscle structure through heat fixation is limited by the availability of fluorescent transgenes. To analyze muscle structure in detail, larval dissection, followed by fixation and staining of musculature in larval fillet should be performed [as described in (<a href="#bib1">Azevedo et al., 2016</a>, <a href="#bib4">Brent et al., 2009</a>)]. Dissection is technically challenging in the 1<sup>st</sup> and 2<sup>nd</sup> instar larvae.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Larval development extends for over 4 days</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Handling the larvae every day, as in the larval locomotion or anesthetization protocols, inflicts stress on the larvae and thereby delays their development, i.e., pupariation. Collecting age matched larvae of the same genotype that are not handled everyday can help by size matching experimental larvae to determine the stage.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Mutants show delayed development.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Some mutants are developmentally delayed, which can be further exacerbated when anesthetizing and imaging the larvae daily. Under these circumstances it is important to stage the larvae based on the differentiation of the mouth hooks or anterior and posterior spiracles (<a href="#bib13">Okada, 1963</a>; <a href="#bib18">Schubiger et al., 1998</a>; <a href="#bib19">Vaufrey et al., 2018</a>). For example, with respect to staging larvae based on spiracles, 1<sup>st</sup> instar larvae have white posterior spiracles, 2<sup>nd</sup> instar larvae have orange posterior spiracles and apparition of club shaped anterior spiracles, early 3<sup>rd</sup> instar have branched anterior spiracles, late 3<sup>rd</sup> instar larvae have everted anterior spiracles, and wandering 3<sup>rd</sup> instar larvae are found out of the food (<a href="#bib14">Park et al., 2002</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Azevedo, M., Schulman, V.K., Folker, E., Balakrishnan, M., and Baylies, M. (2016). Imaging approaches to investigate myonuclear positioning in <i>Drosophila</i>. Methods Mol. Biol. <i>1411</i>, 291-312. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref1">View at publisher</a></p>
<p id="bib2">Balakrishnan, M., Yu, S.F., Chin, S.M., Soffar, D.B., Windner, S.E., Goode, B.L., and Baylies, M.K. (2020). Cofilin loss in <i>Drosophila</i> muscles contributes to muscle weakness through defective sarcomerogenesis during muscle growth. Cell Rep. <i>32</i>, 107893. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref2">View at publisher</a></p>
<p id="bib3">Bawa, S., Brooks, D.S., Neville, K.E., Tipping, M., Sagar, M.A., Kollhoff, J.A., Chawla, G., Geisbrecht, B.V., Tennessen, J.M., Eliceiri, K.W., and Geisbrecht, E.R. (2020). <i>Drosophila</i> TRIM32 cooperates with glycolytic enzymes to promote cell growth. eLife <i>9</i>, e52358. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref3">View at publisher</a></p>
<p id="bib4">Brent, J.R., Werner, K.M., and McCabe, B.D. (2009). <i>Drosophila</i> larval NMJ dissection. J. Vis. Exp. 1107. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref4">View at publisher</a></p>
<p id="bib5">Brooks, D.S., Vishal, K., Kawakami, J., Bouyain, S., and Geisbrecht, E.R. (2016). Optimization of wrMTrck to monitor <i>Drosophila</i> larval locomotor activity. J. Insect Physiol. <i>93-94</i>, 11-17. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref5">View at publisher</a></p>
<p id="bib6">Camuglia, J.M., Mandigo, T.R., Moschella, R., Mark, J., Hudson, C.H., Sheen, D., and Folker, E.S. (2018). An RNAi based screen in <i>Drosophila</i> larvae identifies fascin as a regulator of myoblast fusion and myotendinous junction structure. Skelet. Muscle <i>8</i>, 12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref6">View at publisher</a></p>
<p id="bib7">Choi, B.J., Imlach, W.L., Jiao, W., Wolfram, V., Wu, Y., Grbic, M., Cela, C., Baines, R.A., Nitabach, M.N., and McCabe, B.D. (2014). Miniature neurotransmission regulates <i>Drosophila</i> synaptic structural maturation. Neuron <i>82</i>, 618-634. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref7">View at publisher</a></p>
<p id="bib8">Clark, K.A. and Kadrmas, J.L. (2013). <i>Drosophila melanogaster</i> muscle LIM protein and alpha-actinin function together to stabilize muscle cytoarchitecture: a potential role for Mlp84B in actin-crosslinking. Cytoskeleton (Hoboken) <i>70</i>, 304-316. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref8">View at publisher</a></p>
<p id="bib9">Demontis, F. and Perrimon, N. (2009). Integration of Insulin receptor/Foxo signaling and dMyc activity during muscle growth regulates body size in <i>Drosophila</i>. Development <i>136</i>, 983-993. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref9">View at publisher</a></p>
<p id="bib10">Fernandes, I. and Schöck, F. (2014). The nebulin repeat protein Lasp regulates I-band architecture and filament spacing in myofibrils. J. Cell Biol. <i>206</i>, 559-572. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref10">View at publisher</a></p>
<p id="bib11">Kakanj, P., Eming, S.A., Partridge, L., and Leptin, M. (2020). Long-term in vivo imaging of <i>Drosophila</i> larvae. Nat. Protoc. <i>15</i>, 1158-1187. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref11">View at publisher</a></p>
<p id="bib12">Millard, T.H. and Martin, P. (2008). Dynamic analysis of filopodial interactions during the zippering phase of <i>Drosophila</i> dorsal closure. Development <i>135</i>, 621-626. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref12">View at publisher</a></p>
<p id="bib13">Okada, T. (1963). Caenogenetic differentiation of mouth hooks in drosophilid larvae. Evolution <i>17</i>, 84-98. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref13">View at publisher</a></p>
<p id="bib14">Park, Y., Filippov, V., Gill, S.S., and Adams, M.E. (2002). Deletion of the ecdysis-triggering hormone gene leads to lethal ecdysis deficiency. Development <i>129</i>, 493-503. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref14">View at publisher</a></p>
<p id="bib15">Pulver, S.R. and Berni, J. (2012). The fundamentals of flying: simple and inexpensive strategies for employing <i>Drosophila</i> genetics in neuroscience teaching laboratories. J. Undergrad. Neurosci. Educ. <i>11</i>, A139-A148. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref15">View at publisher</a></p>
<p id="bib16">Sawin, E.P., Dowse, H.B., Hamblen-Coyle, M.J., Hall, J.C., and Sokolowski, M.B. (1994). A lack of locomotor activity rhythms in <i>Drosophila melanogaster</i> larvae (Diptera: Drosophilidae). J. Insect Behav. <i>7</i>, 249-262. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref16">View at publisher</a></p>
<p id="bib17">Schnorrer, F., Schönbauer, C., Langer, C.C., Dietzl, G., Novatchkova, M., Schernhuber, K., Fellner, M., Azaryan, A., Radolf, M., Stark, A., Keleman, K., and Dickson, B.J. (2010). Systematic genetic analysis of muscle morphogenesis and function in <i>Drosophila</i>. Nature <i>464</i>, 287-291. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref17">View at publisher</a></p>
<p id="bib18">Schubiger, M., Wade, A.A., Carney, G.E., Truman, J.W., and Bender, M. (1998). Drosophila EcR-B ecdysone receptor isoforms are required for larval molting and for neuron remodeling during metamorphosis. Development <i>125</i>, 2053-2062. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref18">View at publisher</a></p>
<p id="bib19">Vaufrey, L., Balducci, C., Lafont, R., Prigent, C., and Le Bras, S. (2018). Size matters! Aurora A controls <i>Drosophila</i> larval development. Dev. Biol. <i>440</i>, 88-98. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30278-1/sref19">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Mary Baylies (<a href="mailto:bayliesm@mskcc.org">bayliesm@mskcc.org</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and Code Availability</h4>
<p>This protocol did not generate new data or code.</p>
<h3>Acknowledgments</h3>
<p>We would like to thank the Baylies lab members for helpful discussions. This work was supported by the United States <a href="https://doi.org/10.13039/100000002">National Institute of Arthritis, Musculoskeletal and Skin Diseases</a> (R21AR067361, R01AR108981) and the National Institute of General Medicine (R01GM121971) to M.K.B.</p>
<h3>Author Contributions</h3>
<p>M.B. and M.K.B. wrote the manuscript with input from W.J.S. W.J.S. took images for all the protocols and helped in protocol optimization.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>