<article>
<section>
<h1 id="header">OmicsVolcano: software for intuitive visualization and interactive exploration of high-throughput biological data</h1>
<p><time datetime="2021-01-22">Published: January 22, 2021</time></p>
<p>Irina Kuznetsova,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">10</a>,<a href="#cor1">*</a></sup> Artur Lugmayr,<sup><a href="#aff4">4</a>,<a href="#aff5">5</a></sup> Oliver Rackham,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff6">6</a>,<a href="#aff7">7</a>,<a href="#aff8">8</a></sup> and Aleksandra Filipovska<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff7">7</a>,<a href="#aff8">8</a>,<a href="#aff9">9</a>,<a href="#fn2">11</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Harry Perkins Institute of Medical Research, QEII Medical Centre, Nedlands, WA 6009, Australia</p>
<p id="aff2"><sup>2</sup>ARC Centre of Excellence in Synthetic Biology, QEII Medical Centre, Nedlands, WA 6009, Australia</p>
<p id="aff3"><sup>3</sup>Centre for Medical Research, The University of Western Australia, QEII Medical Centre, Nedlands, WA 6009, Australia</p>
<p id="aff4"><sup>4</sup>Umea University, Department of Computing Science, 901 87 Umea, Sweden</p>
<p id="aff5"><sup>5</sup>Edith Cowan University, School of Science, AI and Optimization Research Group, Joondalup, WA 6027, Australia</p>
<p id="aff6"><sup>6</sup>School of Pharmacy and Biomedical Sciences, Curtin University, Bentley, WA 6102, Australia</p>
<p id="aff7"><sup>7</sup>Curtin Health Innovation Research Institute, Curtin University, Bentley, WA 6102, Australia</p>
<p id="aff8"><sup>8</sup>Telethon Kids Institute, Perth Children's Hospital, Northern Entrance, 15 Hospital Avenue, Nedlands, WA 6009, Australia</p>
<p id="aff9"><sup>9</sup>School of Molecular Sciences, The University of Western Australia, Crawley, WA 6009, Australia</p>
<p id="fn1"><sup>10</sup>Technical contact</p>
<p id="fn2"><sup>11</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:irina.kuznetsova@perkins.org.au">irina.kuznetsova@perkins.org.au</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:aleksandra.filipovska@uwa.edu.au">aleksandra.filipovska@uwa.edu.au</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100279">10.1016/j.xpro.2020.100279</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Advances in omics technologies have generated exponentially larger volumes of biological data; however, their analyses and interpretation are limited to computationally proficient scientists. We created OmicsVolcano, an interactive open-source software tool to enable visualization and exploration of high-throughput biological data, while highlighting features of interest using a volcano plot interface. In contrast to existing tools, our software and user-interface design allow it to be used without requiring any programming skills to generate high-quality and presentation-ready images.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A free and open-source tool for interactive exploration of omics data</li>
<li>Immediate visualization of gene and protein changes on a volcano plot</li>
<li>Visualization and generation of figures with gene ontologies linked to cellular processes</li>
<li>Data exploration and visualization of user-defined cellular and molecular processes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/417-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Significant advances have been made in biological sciences through the generation of data from genomic, transcriptomic, proteomic and metabolomic analyses - jointly referred to as omics technologies (<a href="#bib9">Sandhu et al., 2018</a>). The amount of data varies depending on the type of omics platform and recently there have been concerted efforts toward integrating omics datasets to provide global insights into cellular function (<a href="#bib30">Yan et al. 2017</a>; <a href="#bib5">Cambiaghi et al. 2016</a>). The major limitation of omics technologies has become the interpretation and visualization of the analyzed datasets in a coherent and user-friendly manner that would be suitable for users from diverse fields without any computational skills. Volcano plots effectively visualize significantly increased and reduced changes in entire omics datasets. There are two major limitations in the use of volcano plots: the requirement for computational skills in R to summarize and visualize the data in volcano plots; and the lack of flexibility to interactively highlight or discover specific sets of changes or processes. Furthermore, each addition or change of a highlighted gene or protein requires manual generation of a new plot, making the process of producing volcano plots time consuming for computational biologists.</p>
<p>Existing tools to date, such as EnhancedVolcano, VolcanoR, and msVolcano (<a href="#bib3">Blighe et al. 2019</a>; <a href="#bib16">Naumov et al., 2017</a>; <a href="#bib23">Singh et al. 2016</a>) are capable of plotting omics data as volcano plots and have specific functionalities. These allow genes of choice to be labeled, and some of the programs - such as msVolcano - enable significance testing. Additional software tools, such as DEIVA, are capable of carrying out enrichment analyses. This functionality is also part of software packages such as DAVID (<a href="#bib13">Huang et al. 2008</a>; <a href="#bib14">Huang et al. 2009</a>; <a href="#bib12">Harshbarger et al. 2017</a>). We sought to build upon these approaches and provide interactive volcano plots where sets of genes or proteins that have been classified as part of specific cellular processes could be selectively highlighted. Our goal was to enable biologists without any computational expertise to view, visualize, and explore specific cellular or molecular processes.</p>
<p>OmicsVolcano has been designed for biologists without any programing experience and provides an easy-to-use web-based tool. One of OmicsVolcano’s strengths is its capability to allow interactive exploration of omics data that allows users to focus on the biological aspects of the data. The interactive graphics that are part of OmicsVolcano enhance the impact of the findings and put them into a physiological context. Those include the ability of the user to highlight primary changes, to visualize a group of genes and proteins that are related to a specific cellular process or multiple cellular processes, and to examine their cellular localizations. In each case information about selected genes or proteins is presented in a table below the graph. OmicsVolcano generates high-quality and publication-ready images in scalable vector graphic (SVG) format.</p>
<h3 id="sec1.1">Overview</h3>
<p>The OmicsVolcano software consists of a set of scripts, functions, and modules which search for the presence of duplicated gene names in the input data; add numerical extensions to duplicated gene names; visualize the data as interactive volcano plots; and filter data to significant values based on information provided in the input file. The input file is provided by the user, and consists of five columns that represent identification numbers (IDs), gene symbols, gene descriptions, log fold changes, and adjusted p values. The software package also contains reference files for mitochondrial processes and cellular compartments for the human and mouse genomes. The software uses the input data and processes or cellular compartment information to create volcano plots in a user-friendly way (<a href="#fig1">Figure 1</a>). The software tool allows an interactive interpretation of the data through a web interface, which is easy and intuitive to use.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/417-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The OmicsVolcano home page</div>
<p>The software requires four steps to generate the interactive omics-data visualization plots. The “file” option allows the upload of the input data as an ASCII file which contains IDs, gene names, gene descriptions, log2 fold changes, and adjusted p values. The “explore” option provides several main core functionalities of the software: plot, custom gene or protein selection, mitochondrial processes, multiple process visualization, and cellular compartment visualization. Customization of the statistical significance and threshold of the y-axis, are performed by adjustments of the slider widgets. Additional options, e.g., “upload a gene file”, “insert a list of genes”, “select organism”, and “show mitochondrial process” widgets allow the customization of the data exploration processes. The “export” option enables the export of data as tables or graphics in various pre-defined formats. A manual is available through the “help” option, and the software package version including additional information about the software can be found in the “about” option.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Software download and prerequisites</h3>
<p>A pre-installed R software environment is required to use OmicsVolcano. The easiest option to run OmicsVolcano is to use RStudio. Both, R and RStudio can be downloaded from <a href="https://www.r-project.org/">https://www.r-project.org/</a> and <a href="https://rstudio.com/products/rstudio/download/">https://rstudio.com/products/rstudio/download/</a>, respectively. We encountered some rare cases of R installation environments, where it is required to adjust the file extension settings either to capitalized or lower key letters. In these cases, the extension of source files should be either capitalized or set to lower key letters (e.g., as “config.R” instead of “config.r”).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>RNA sequencing data</td>
<td>GEO</td>
<td>GSE105406, GSE111228</td>
</tr>
<tr>
<td>Proteomics data</td>
<td>PRIDE</td>
<td>PXD015521</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>R version 3.6.1 and 4.02</td>
<td>Team, R.C. 2019 R: A language and environment for statistical computing</td>
<td><a href="https://www.R-project.org/">https://www.R-project.org/</a></td>
</tr>
<tr>
<td>shiny version 1.4.0</td>
<td>
<a href="#bib6">Chang et al., 2019</a>. shiny: Web Application Framework for R</td>
<td>
<a href="https://CRAN.R-project.org/package=shiny">https://CRAN.R-project.org/package=shiny</a>.</td>
</tr>
<tr>
<td>shinydashboard version 0.7.1</td>
<td>
<a href="#bib8">Chang and Ribeiro, 2018</a>. shinydashboard: Create Dashboards with “Shiny”</td>
<td><a href="https://CRAN.R-project.org/package=shinydashboard">https://CRAN.R-project.org/package=shinydashboard</a></td>
</tr>
<tr>
<td>shinydashboardPlus version 0.7.1</td>
<td>
<a href="#bib11">Granjon, 2020</a>. shinydashboardPlus: Add More “AdminLTE2” Components to “shinydashboard”</td>
<td><a href="https://CRAN.R-project.org/package=shinydashboardPlus">https://CRAN.R-project.org/package=shinydashboardPlus</a></td>
</tr>
<tr>
<td>shinyWidgets version 0.5.0</td>
<td>(<a href="#bib33">Perrier et al., 2020</a>). shinyWidgets: Custom Inputs Widgets for Shiny</td>
<td><a href="https://CRAN.R-project.org/package=shinyWidgets">https://CRAN.R-project.org/package=shinyWidgets</a></td>
</tr>
<tr>
<td>shinythemes version 1.1.2</td>
<td>
<a href="#bib7">Chang, 2018</a>. shinythemes: Themes for Shiny</td>
<td>
<a href="https://CRAN.R-project.org/package=shinythemes">https://CRAN.R-project.org/package=shinythemes</a>.</td>
</tr>
<tr>
<td>shinyjs version 2.0.0</td>
<td>
<a href="#bib2">Attali, 2020</a>. shinyjs: Easily Improve the User Experience of Your Shiny Apps in Seconds</td>
<td><a href="https://CRAN.R-project.org/package=shinyjs">https://CRAN.R-project.org/package=shinyjs</a></td>
</tr>
<tr>
<td>dplyr version 0.8.3</td>
<td>
<a href="#bib25">Wickham et al., 2019</a>. dplyr: A Grammar of Data Manipulation</td>
<td><a href="https://CRAN.R-project.org/package=dplyr">https://CRAN.R-project.org/package=dplyr</a></td>
</tr>
<tr>
<td>plotly version 4.9.1</td>
<td>
<a href="#bib21">Sievert, 2018</a>. plotly for R</td>
<td><a href="https://plotly-r.com">https://plotly-r.com</a></td>
</tr>
<tr>
<td>ggplot2 version 3.2.1</td>
<td>
<a href="#bib26">Wickham, 2016</a>. ggplot2: Elegant Graphics for Data Analysis. Springer-Verlag New York</td>
<td><a href="https://ggplot2.tidyverse.org">https://ggplot2.tidyverse.org</a></td>
</tr>
<tr>
<td>crosstalk version 1.0.0</td>
<td>
<a href="#bib10">Cheng, 2016</a>. crosstalk: Inter-Widget Interactivity for HTML Widgets</td>
<td><a href="https://CRAN.R-project.org/package=crosstalk">https://CRAN.R-project.org/package=crosstalk</a></td>
</tr>
<tr>
<td>DT version 0.12</td>
<td>
<a href="#bib29">Xie et al., 2020</a>. DT: A Wrapper of the JavaScript Library “DataTables”</td>
<td><a href="https://CRAN.R-project.org/package=DT">https://CRAN.R-project.org/package=DT</a></td>
</tr>
<tr>
<td>svglite version 1.2.3</td>
<td>
<a href="#bib28">Wickham et al., 2020</a>. svglite: An “SVG” Graphics Device</td>
<td><a href="https://CRAN.R-project.org/package=svglite">https://CRAN.R-project.org/package=svglite</a></td>
</tr>
<tr>
<td>stringr version 1.4.0</td>
<td>
<a href="#bib27">Wickham, 2019</a>. stringr: Simple, Consistent Wrappers for Common String Operations</td>
<td><a href="https://CRAN.R-project.org/package=stringr">https://CRAN.R-project.org/package=stringr</a></td>
</tr>
<tr>
<td>config version 0.3</td>
<td>
<a href="#bib1">Allaire, 2018</a>. config: Manage Environment Specific Configuration Values</td>
<td><a href="https://CRAN.R-project.org/package=config">https://CRAN.R-project.org/package=config</a></td>
</tr>
<tr>
<td>colourpicker version 1.1.0</td>
<td><a href="https://cran.r-project.org/web/packages/colourpicker/index.html">https://cran.r-project.org/web/packages/colourpicker/index.html</a></td>
<td><a href="https://CRAN.R-project.org/package=colourpicker">https://CRAN.R-project.org/package=colourpicker</a></td>
</tr>
<tr>
<td>gridExtra version 2.3</td>
<td><a href="https://cran.r-project.org/web/packages/gridExtra/index.html">https://cran.r-project.org/web/packages/gridExtra/index.html</a></td>
<td><a href="https://cran.r-project.org/web/packages/gridExtra/index.html">https://cran.r-project.org/web/packages/gridExtra/index.html</a></td>
</tr>
<tr>
<td>OmicsVolcano</td>
<td><a href="https://github.com/IrinaVKuznetsova/OmicsVolcano">https://github.com/IrinaVKuznetsova/OmicsVolcano</a></td>
<td>this manuscript</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>OmicsVolcano is available as open-source software and is hosted at GitHub: <a href="https://github.com/IrinaVKuznetsova/OmicsVolcano">https://github.com/IrinaVKuznetsova/OmicsVolcano</a>. The software implementation is written in R (3.6.1) using the following packages: <i>shiny</i> (1.4.0) enabling the creation of an intuitive and interactive web application; <i>ggplot2</i> (3.2.1) and <i>plotly</i> (4.9.1) enabling the creation of the interactive volcano plot graphics; <i>dplyr</i> (0.8.3) enabling the dataset aggregation and analysis; <i>DT</i> (0.11) enabling the display of the R data objects as tables in HTML format; and <i>crosstalk</i> (1.0.0) providing interactions between R objects (<a href="#bib25">Wickham et al. 2019</a>; <a href="#bib6">Chang et al. 2019</a>; <a href="#bib21">Sievert 2018</a>; <a href="#bib29">Xie et al. 2020</a>; <a href="#bib26">Wickham 2016</a>; <a href="#bib24">Team 2019</a>; <a href="#bib10">Cheng 2016</a>), and <i>colourpicker</i> enabling a color palette (<a href="https://cran.r-project.org/web/packages/colourpicker/index.html">https://cran.r-project.org/web/packages/colourpicker/index.html</a>). A full list of utilized packages can be found in the <a href="#key-resources-table">Key resources table</a>. The application is hosted as an open-source package in the GitHub repository.</p>
<p>The test input data consisted of RNA or protein fold changes and adjusted p values from RNA sequencing and proteomic analyses that were processed as described previously (<a href="#bib20">Rudler et al. 2019</a>; <a href="#bib22">Siira et al. 2018</a>; <a href="#bib18">Perks et al., 2018</a>). The input file columns were designated as: “<i>ID</i>,” “<i>GeneSymbol</i>,” “<i>Description</i>,” “<i>Log2FC</i>,” and “<i>AdjPValue</i>” (an example format of the input file is attached to the software browser widget and also shown in the OmicsVolcano GitHub web-page). Specific information about mitochondrial processes was based on our previous studies (<a href="#bib15">Kühl et al. 2017</a>), MitoXplorer (<a href="#bib31">Yim et al. 2019</a>) and combined with MitoCarta 2.0 (<a href="#bib4">Calvo et al. 2015</a>; <a href="#bib17">Pagliarini et al. 2008</a>). Cellular compartment localization information was retrieved from the Human Protein Atlas database available from http://www.proteinatlas.org (<a href="#bib32">Thul et al., 2017</a>).</p>
<p>The software is based on R and uses R packages shown in the <a href="#key-resources-table">Key resources table</a>.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The software has been tested for both R version 3.6.1 and version 4.0.0. It also has been tested on Windows and Mac OS platforms (<a href="#tbl1">Table 1</a>).</div>
<table id="tbl1">
<caption>Table 1. Operating environments on which the software was tested</caption>
<thead>
<tr>
<th>Operating System</th>
<th>Version</th>
</tr>
</thead>
<tbody>
<tr>
<td>Windows</td>
<td>Windows 10</td>
</tr>
<tr>
<td>Mac OS</td>
<td>Mojave version 10.14.6</td>
</tr>
</tbody>
</table>
<ul>
<li>Recommended hardware: minimal 4 Gb memory. Memory requirements may increase with input data size.</li>
<li>Processors: 1 required, 2 recommended.</li>
<li>Example data are provided with the software package. User input files for omics datasets should be formatted as a tab or as a semicolon separated file in ASCII/text format. The file should contain five columns with the column names as shown in <a href="#tbl2">Table 2</a>: ID, GeneSymbol, Description, Log2FC, AdjPValue.</li>
<li>Column names are case-sensitive and require a header row for the input file. Thus, when preparing input files for OmicsVolcano, it is essential to provide a header row. The following rows contain various values, which will be processed by the OmicsVolcano software.</li>
</ul>
<table id="tbl2">
<caption>Table 2. Input file example</caption>
<thead>
<tr>
<th>ID</th>
<th>Gene Symbol</th>
<th>Description</th>
<th>Log2FC</th>
<th>AdjPValue</th>
</tr>
</thead>
<tbody>
<tr>
<td>Q4U4S6</td>
<td>Xirp2</td>
<td>Xin actin-binding repeat-containing protein 2 OS=Mus musculus OX=10090 GN=Xirp2 PE=1 SV=1</td>
<td>6.64</td>
<td>1.33E-08</td>
</tr>
<tr>
<td>Q497D7</td>
<td>Rpl30fo</td>
<td>Rpl30 protein OS=Mus musculus OX=10090 GN=Rpl30 PE=2 SV=1</td>
<td>2.14</td>
<td>0.8</td>
</tr>
<tr>
<td>Q9CPP6</td>
<td>Ndufa5</td>
<td>NADH dehydrogenase [ubiquinone] 1 alpha subcomplex subunit 5 OS=Mus musculus OX=10090 GN=Ndufa5 PE=1 SV=3</td>
<td>-1.52</td>
<td>6.24E-08</td>
</tr>
<tr>
<td>P09055</td>
<td>Itgb1</td>
<td>Integrin beta-1 OS=Mus musculus OX=10090 GN=Itgb1 PE=1 SV=1</td>
<td>0.08</td>
<td>6.29E-08</td>
</tr>
<tr>
<td>…</td>
<td>…</td>
<td>…</td>
<td>...</td>
<td>…</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Step 1: Installing and initializing the OmicsVolcano software tool</h3>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;3 min</div>
<ol>
<li>Download the OmicsVolcano software from the GitHub repository as zip folder.</li>
<li>Unzip the folder into a directory of your choice, such as onto the Desktop.</li>
<li>Open RStudio.</li>
<li>Open the R script OmicsVolcano_App.r through RStudio, by selecting “File,” then select “Open File” and select OmicsVolcano_App.r that will be loaded into the RStudio interface.</li>
<li>Once the OmicsVolcano_App.r is loaded on the interface press the “Run App” button to load the OmicsVolcano software. Ensure that the “Run in Window” or "Run External" selection is checked when you press “Run App” (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/417-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Initializing the OmicsVolcano software in R studio</div>
<p>Screen capture indicating how to select the run app in R studio to open the software.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Check that RStudio is working correctly. <a href="#sec6.1">Troubleshooting 1</a>
</div>
<div class="critical">
<span class="critical-title">Critical:</span> Mac OS users are required to install the XQuartz app from <a href="https://www.xquartz.org">https://www.xquartz.org</a>.</div>
<div class="note">
<span class="note-title">Note:</span> Once the software is downloaded and functional, steps 1–5 do not need to be repeated for future and continued use of the OmicsVolcano software.</div>
<h3 id="sec3.2">Step 2: Visualizing omics data</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<p>There are three menu options under the Home Page heading that enable the visualization and exploration of the data.</p>
<ol start="6">
<li>Press the <b>File – Open…</b> tab to upload a user provided input file in a .txt format, not applicable (NA) or empty fields are allowed.
<div class="note">
<span class="note-title">Note:</span> A demo file can be used for a test run. The demo file designated demofile.txt can be downloaded by selecting <b>File – Open… - Download demo file to local PC</b>. This demo file will be automatically downloaded in the user-selected folder and then used to visualize the data.</div>
<ol type="a">
<li>If the file data is not formatted according to the instructions described in the <a href="#materials-and-equipment">Materials and equipment</a> then an error message will be displayed: “File Loading Error in Data Input File! Please check the number of columns or select the correct field separator character. Alternatively, review the Help Page for the file input format requirements.”</li>
<li>If the data has duplicates, for example in proteomic data sets when there are multiple isoforms of proteins, the default option of the software is to add a numerical extension for each duplicate in order of their appearance in the file provided. This is done so that duplicate names of proteins are not lost when they are visualized. However, if there are no duplicates in the data then the “Check for Duplicates” option can be unselected in the <b>File - Open… - Check for Duplicates.</b>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Press <b>Open…</b>, select input file, and the data will appear in a table with the headings used in the input file. The headings can be toggled to order the data alphabetically, in an ascending or descending order of log2 fold changes, or based on the adjusted p values. <a href="#sec6.3">Troubleshooting 2</a>
</li>
<li>Select File Separator. <a href="#sec6.5">Troubleshooting 3</a>
</li>
<li>To visualize the data select <b>Explore</b> tab and then select <b>Plot.</b> This generates a volcano plot where the significantly increased and reduced data points are colored in red and blue, respectively. Non-significant data points are shown in gray (<a href="#fig3">Figure 3</a>). Any data point can be labeled by clicking on the plotted data points (see 12). <a href="#sec6.7">Troubleshooting 4</a>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/417-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. A volcano plot example with specific interactively selected gene labels</div>
<p>Transcriptomic data are visualized where significantly increased transcripts are shown in light red and significantly decreased transcripts in light blue. Non-significant transcripts are shown in gray. Three increased and three decreased transcripts are indicated in the plot as examples, showing the software’s capability to label individual transcripts of interest.</p>
</figcaption>
</figure>
</li>
<li>The significance and vertical thresholds can be adjusted from the right-side panel named <b>Threshold.</b>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> The software offers the option to choose statistical significance, which is recommended to be either 0.05 or 0.01. Another option is to adjust the log2 fold change threshold, which is set to +/-1 as the default value.</div>
<ol start="11">
<li>Select a data point on the volcano plot by pressing on it within the plot to provide a label for the identity of the gene or protein. Multiple data points can be labeled by holding <b>Shift</b> in Windows or <b>Command</b> in Mac platforms. The information for the selected data point can be seen in the table below the volcano plot. <a href="#sec6.9">Troubleshooting 5</a>
</li>
<li>The table below the volcano plot has two tabs, the <b>Input Data</b> tab shows all the data and the <b>Signific</b> tab lists all the significantly changing data. The <b>Search</b> box within the table enables the user to query specific gene or protein names.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The data input file has to follow the format shown in <a href="#tbl2">Table 2</a> above. <a href="#sec6.11">Troubleshooting 6</a> and <a href="#sec6.13">7</a>
</div>
<div class="note">
<span class="note-title">Note:</span> The <b>Help</b> tab provides a brief guide to the use of the software. The <b>About</b> tab provides information about the authors, license, used packages and their versions, and how to cite the software.</div>
<h3 id="sec3.3">Step 3: Exploring omics data</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–30 min</div>
<p>The <b>Explore</b> tab has four additional features that enable interactive searches for specific genes or processes linked to specific genes.</p>
<ol start="13">
<li>Select <b>Custom Gene List</b> feature:
<ol type="a">
<li>Inserting a list of genes enables the user to manually input a user-specified list of genes that will appear on the volcano plot with gene or protein labels and their information in the table below the plot (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/417-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Volcano custom plot example including user-defined searches</div>
<p>Transcriptomic data are visualized with significantly increased transcripts shown in light red and significantly decreased transcripts shown in light blue. Non-significant transcripts are shown in gray. User-specified transcripts or proteins can be searched in the right-hand box under “Custom Gene list” or uploaded in a user-defined file. The file-based import is useful when a large number of transcripts or proteins are to be searched and visualized in the plot. Dark red color indicates if they are significantly increased, and dark blue if they are significantly reduced within the dataset.</p>
</figcaption>
</figure>
</li>
<li>The <b>Upload a gene file</b> option enables the user to provide their own gene ontology lists from an enrichment analysis to highlight specific genes in a biological process, molecular function, or cellular compartment or a user-specified list of genes within a file. The file is rather simple, and each line of the file represents a gene name used for a process (see list below). The file does not require a header row.</li>
</ol>
</li>
</ol>
<table id="undtbl1">
<thead></thead>
<tbody>
<tr>
<td rowspan="5">Example:</td>
<td>Ndufs2</td>
</tr>
<tr>
<td>Gatc</td>
</tr>
<tr>
<td>Cox7a1</td>
</tr>
<tr>
<td>lmnb1</td>
</tr>
<tr>
<td>Ndufa8</td>
</tr>
</tbody>
</table>
<ol start="14">
<li>The <b>Mitochondrial Processes</b> feature highlights all mitochondria specific genes or proteins in either bright red for the upregulated genes or proteins, or bright blue for the downregulated gene or proteins. Non-mitochondrial genes or proteins are shown as faded red or blue for those that are significantly increased or decreased (<a href="#fig5">Figure 5</a>). <a href="#limitations">Limitations</a>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/417-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Volcano plot showing mitochondrial processes</div>
<p>Transcriptomic data are visualized and mitochondrial transcripts are highlighted in dark red, dark blue, and dark gray, if they are significantly increased, decreased, or unchanged, respectively. Specific processes can be selected from the dropdown menu and these will be highlighted in either dark red or dark blue depending on their change within the dataset.</p>
</figcaption>
</figure>
</li>
<li>An additional feature of the<b>Mitochondrial Processes</b> dropdown menu enables the exploration of 40 different processes linked to mitochondrial function present in the user-provided data. Selection of a specific process highlights a set of genes or proteins related to that mitochondrial process and shows them in bright red for the upregulated genes or proteins, or bright blue for the downregulated gene or proteins. This feature enables a fast identification of specific processes and genes or proteins that are significantly altered in the entire dataset (<a href="#fig5">Figure 5</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The <b>Custom Gene List</b> features enables users to expand the scope of cellular and molecular processes that can be added to the software in addition to <b>Mitochondrial Processes</b>. The <b>Mitochondrial Processes</b> feature is an example of curated gene ontologies linked to MitoCarta 2.0 genes. <a href="#sec6.15">Troubleshooting 8</a>
</div>
<ol start="16">
<li>The <b>Multiple Mitochondrial Processes</b> function enables the selection and user-defined color selection of multiple processes to highlight all the changes in the dataset that are of interest to the user (<a href="#fig6">Figure 6</a>). The user should check the process of interest and select the color for the process and repeat this until all processes required by the user are selected then press “Apply” to show them on the plot as shown in <a href="#fig6">Figure 6</a>. The right-hand panel shows the figure legend revealing the process linked to each selected color and the table below lists all the genes and their descriptions along with the colors representing them on the plot. <a href="#sec6.17">Troubleshooting 9</a>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/417-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Volcano plot showing specific selection and color coding of multiple mitochondrial processes</div>
<p>This dropdown menu allows the user to select specific processes and custom colors for each process and visualizes them on the plot. This feature enables additional multiple processes to be visualized at the same time.</p>
</figcaption>
</figure>
</li>
<li>The<b>Cellular Localization</b> dropdown menu enables the exploration of different cellular compartments related to the changes in the user-provided data. This feature enables a fast identification of genes or proteins that are significantly altered in the entire dataset and related to a specific cellular location (<a href="#fig7">Figure 7</a>). In the example, we identified that the loss of a specific gene with unknown function caused the downregulation of genes involved in endoplasmic reticulum function, providing a valuable and fast insight into the role of our gene of interest (<a href="#fig7">Figure 7</a>). This can be applied to any dataset to quickly identify changes in different cellular compartments. <a href="#sec6.17">Troubleshooting 9</a>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/417-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Volcano plot showing cellular localizations</div>
<p>This feature enables the user to explore changes in specific cellular locations by selecting specific cellular compartments from the dropdown menu. The selected cellular compartments are visualized depending on the changes (increased in dark red and decreased in dark blue) in the input file related to the selected cellular compartment (in this case the endoplasmic reticulum).</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.4">Step 4: Exporting visualized data</h3>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;1 min</div>
<p>The final features of our software are the download options of the completed volcano plot and associated tables. All interactive plots can be downloaded in a vector format as SVG files directly from the plot by pressing the camera-like icon. The image is saved to the default PC location. Static plots generated by <b>Custom Gene List</b> can be exported in numerous different formats. Tables associated with the generated volcano plot can be downloaded in txt and csv formats depending on the requirements of the end-user.</p>
<ol start="18">
<li>Select the <b>Export</b> function and choose either <b>Plot</b> or <b>Table.</b>
<ol type="a">
<li>Select <b>Plot</b> and in the first dropdown menu <b>Custom Gene List</b> can be exported as png, jpeg, or tiff for an image and SVG or pdf for a vector file.</li>
<li>Select <b>Table</b> and in the first dropdown menu and then select the type of table to export. In the second dropdown menu choose to save the tabulated data in csv or txt format.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Hovering the cursor over the volcano plot reveals a camera icon in the top right corner of the volcano plot. Selecting the camera icon enables a shortcut to download the volcano plot in SVG format to the PC default location. Selecting the camera icon enables downloading of volcano plots from each feature on OmicsVolcano. Users using the Ubuntu operating system can use this feature to download their plots.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>We created OmicsVolcano to provide a simple to use tool for biologists to explore and highlight changes in genes, transcripts and proteins in an interactive manner and visualize them for presentations and publications (<a href="#fig1">Figure 1</a>). To date, visualization of omics datasets has been restricted to computational experts, while our software by-passes any requirements for computational skills - empowering scientists from any discipline and skill level to explore their data.</p>
<p>The software requires minimal data input consisting of gene or protein IDs, fold change, and adjusted p values to visualize the data in an interactive volcano plot. The input file can be provided by the user in a simple text format. Once the input file is uploaded in the software, two simple point-and-click features enable the user to choose the plot threshold and to define the changes based on their significance, typically users would choose 0.01 or 0.05, however, the toggle provides users the freedom to select the significance of their choice. These functions enable immediate visualization of the changes on a volcano plot, where the significantly downregulated genes or proteins are highlighted in blue and the significantly upregulated genes, transcripts or proteins are highlighted in red, compared to the remaining genes, transcripts or proteins that are not significantly changed that are shown in gray.</p>
<p>The software provides a table below the volcano plot with additional information on the gene or protein ID, gene symbol, log fold change, adjusted p value and description of the gene/protein function for the entire dataset. The volcano plot is interactive such that each point shown on the plot can be selected, highlighted with its related gene or protein name and the information related to that specific point can be viewed in the table below. Conversely, any gene or protein can be searched in the table and selecting it from the table will highlight it in a specific color on the volcano plot and provide its name on the plot. These features are unique compared to static volcano plots graphed in R where the users cannot identify which gene is related to specific point on the plot unless they have computational expertise to use R to select specific genes or proteins to highlight, making it laborious and time consuming to re-plot the volcano graph whenever a new set of genes, transcripts or proteins need to be examined.</p>
<p>The software functionalities named “Plot” and “Custom Gene List” are broadly applicable to explore datasets of any genome, transcriptome or proteome. The “Plot” - enables exploration of the entire volcano plot with the ability to select and label any point of interest. The “Custom Gene List” function enables users to type in or upload a file with their choice of gene, transcript or protein names in a text format, enabling users from diverse fields to explore processes specific to their research interests.</p>
<p>The “Mitochondrial Process” function enables users to explore changes in mouse or human mitochondria. For this function we have provided a built-in option to analyze 40 different processes related to mitochondrial function as an example, to demonstrate practically how effective our software is at interactively highlighting changes in genes, transcripts or proteins in a specific process. The selection of a specific mitochondrial function highlights the changes in genes, transcripts or proteins linked to the specified process on the plot. The table below the volcano plot provides information on the names and descriptions of the highlighted genes or proteins. To add an additional layer of information within the “processes” selection we highlight the up- and downregulated genes, transcripts or proteins in bright red and blue colors from the specified process, while the remaining significantly up- or downregulated genes, transcripts or proteins remain shown in faded red and blue. This feature highlights the genes, transcripts or proteins in the specified process that are significantly changed compared to the overall identified changes. The “Mitochondrial Process” function exemplifies the flexibility of our software and users can upload their own user-defined cellular and molecular processes associated with specified genes, RNAs or proteins to enable exploration of other cellular processes in OmicsVolcano. The “Multiple Mitochondrial Processes” are an additional feature providing users with the ability to choose multiple processes and color code them specifically. The “Cellular Localization” feature enables the exploration of entire datasets to reveal changes in gene expression or protein levels related to specific compartments within the cell. This enables users to quickly draw conclusions about their findings related to specific parts of the cell. Finally, the generated volcano plots can be downloaded in vector format, that can be further edited if required, whereas the tables can be downloaded in txt or csv format that are publication ready.</p>
<p>In summary, the main features of OmicsVolcano are: (1) a free and open-source software under GNU General Public license available from GitHub as an interactive web application to be run from RStudio; (2) interactive exploration of omics-generated datasets; (3) a data export functionality enabling further examination using other software tools or production of graphics for reports and scientific publications; (4) exploration of processes linked to mitochondrial function or cellular compartment; (5) an intuitive and easy-to-use interface for biologists of any field and (6) broad applicability to any cellular process or organism by providing customizable list of genes, transcripts, proteins, or metabolites associated with specific processes. The software interface enables scientists without any computational skills to independently analyze their data, highlight specific cellular processes, genes, transcripts or proteins of interest and visualize them within minutes. The software is designed to be extendable, so that additional functionalities can easily be added in future releases.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The software is limited to presenting omics data in volcano plots, however, these plots have become the visualization method of choice in the omics field due to their intuitive representation of complex data. Our software provides an interactive interface that enables users to select and highlight as many or as few data points of interest without having to generate new plots every time a new data point is selected – opening up the use of volcano plots to all interested users.</p>
<p>The software can be adapted to highlight specific gene ontologies by providing user-provided input files, which is an added advantage. The software enables visualization of one dataset compared to another but not multidimensional analyses. This is a common feature of volcano plots and most comparisons in biological datasets, however, alternative methods should be used for visualizing data that require multidimensional comparisons. The software has built-in human and mouse reference files that will be updated by the software developers annually.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Software versions and operating system specific requirements.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Make sure that R and RStudio versions are 3.6.1 or greater.</p>
<p>Mac OS users need to install the XQuartz app from <a href="https://www.xquartz.org">https://www.xquartz.org</a>.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Wrong source file formatting, typos, and incorrect column names.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>In the case of incorrect source file formatting, the software will display an error message to notify that there is a typographical error or incorrect column name. One typical example message for incorrectly formatted source files is: "Please check the name of the 2nd column. It is GeneSymbol, not Genesymbol."</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Incorrect field separator characters or incorrect number of columns in source files.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>In File, check that the correct field separator is chosen in the <b>File Separator</b> tab.</p>
<p>OmicsVolcano will display the error message: "Please check number of columns or select correct field separator character. Alternatively, review the Help Page for the file input format."</p>
<h3 id="sec6.7">Problem 4</h3>
<p>A gene or protein is present in the input source file but not visualized by the software.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This gene or protein has no numeric value assigned and is represented as NA or empty field in the “LogFC” or “AdjPValue” columns. Therefore, it cannot be shown on the plot.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>No blue and/or red values in the plot – only gray values are visualized.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The data may not have significantly changing genes or proteins. Also, verify and carefully choose significance and vertical thresholds.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Gray values are located below the significance threshold and are not visualized.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>The input file should contain all values. Do not prefilter the input file for significantly changing genes or proteins only.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Frozen browser page.</p>
<h3 id="sec6.14">Potential solution</h3>
<p>Re-fresh (reload) the browser page.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>Undo selected genes.</p>
<p>I have changed my mind and would like to undo selected gene(s). Note, this is only relevant for the features “<b>Plot</b><b>"</b><b>,</b><b>"</b><b>Mitochondrial Process”</b> and <b>“Cellular Localization.”</b></p>
<h3 id="sec6.16">Potential solution</h3>
<p>Double click on the plot image and select desired gene(s) again or re-fresh (reload) browser page and select desired gene(s) again.</p>
<p>Alternatively, visualize genes of interest with the <b>Custom Gene List</b> feature located at the <b>Plot</b> tab. Create a file that contains the gene names that should be visualized, upload this file in the <b>Custom Gene List</b> located on the right-hand side.</p>
<h3 id="sec6.17">Problem 9</h3>
<p>Visualize multiple genes and identify their location.</p>
<p>I would like to visualize multiple genes, but I do not know where they are located on the plot. Note, this is relevant for the features “<b>Plot”</b> and <b>“Mitochondrial Process”</b> only.</p>
<h3 id="sec6.18">Potential solution</h3>
<p>Use the <b>Search</b> tab in the table below the plot. Write the required gene name and press the row with the gene information. The table row will be highlighted in blue, and the selected gene label will appear on the plot. Continue this way, as many times as required.</p>
<p>Alternatively, use the feature “<b>Custom Gene List.”</b> Create a file that contains gene names that have to be visualized, upload this file in the “<b>Custom Gene List</b>” located on the right-hand side.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Allaire, J.J. (2018). config: Manage Environment Specific Configuration Values (https://CRAN.R-project.org/package=config). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref1">View at publisher</a></p>
<p id="bib2">Attali, Dean. (2020). shinyjs: Easily Improve the User Experience of Your Shiny Apps in Seconds (https://CRAN.R-project.org/package=shinyjs). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref2">View at publisher</a></p>
<p id="bib3">Blighe, K., Rana, S., and Lewis, M. (2019). EnhancedVolcano: publication-ready volcano plots with enhanced colouring and labeling. <a class="external-link" href="https://github.com/kevinblighe/EnhancedVolcano">https://github.com/kevinblighe/EnhancedVolcano</a>.</p>
<p id="bib4">Calvo, S.E., Clauser, K.R., and Mootha, V.K. (2015). MitoCarta2.0: an updated inventory of mammalian mitochondrial proteins. Nucleic Acids Res. <i>44</i>, D1251-D1257. <a class="external-link" href="https://doi.org/10.1093/nar/gkv1003">https://doi.org/10.1093/nar/gkv1003</a></p>
<p id="bib5">Cambiaghi, A., Ferrario, M., and Masseroli, M. (2016). Analysis of metabolomic data: tools, current strategies and future challenges for omics data integration. Brief. Bioinform. <i>18</i>, 498-510. <a class="external-link" href="https://doi.org/10.1093/bib/bbw031">https://doi.org/10.1093/bib/bbw031</a></p>
<p id="bib6">Chang, W., Cheng, C., Allaire, J.J., Xie, Y., McPherson, J., Otto, M., Thornton, J., Farkas, A., Jehl, S., Petre, S., et al. (2019). shiny: web application framework for R. <a class="external-link" href="https://cran.r-project.org/web/packages/shiny/index.html">https://cran.r-project.org/web/packages/shiny/index.html</a>.</p>
<p id="bib7">Chang, W. (2018). shinythemes: Themes for Shiny (https://CRAN.R-project.org/package=shinythemes). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref7">View at publisher</a></p>
<p id="bib8">Chang, W. and Ribeiro, B.B. (2018). shinydashboard: Create Dashboards with ’Shiny’ (https://CRAN.R-project.org/package=shinydashboard). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref8">View at publisher</a></p>
<p id="bib9">Sandhu, C., Qureshi, A., and Emili, A. (2018). Panomics for precision medicine. Trends Mol. Med. <i>24</i>, 85-101. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref9">View at publisher</a></p>
<p id="bib10">Cheng, J. (2016). crosstalk: inter-widget interactivity for HTML widgets. <a class="external-link" href="https://CRAN.R-project.org/package=crosstalk">https://CRAN.R-project.org/package=crosstalk</a>.</p>
<p id="bib11">Granjon, D. (2020). shinydashboardPlus: Add More ’AdminLTE2’ Components to ’shinydashboard (https://CRAN.R-project.org/package=shinydashboardPlus). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref11">View at publisher</a></p>
<p id="bib12">Harshbarger, J., Kratz, A., and Carninci, P. (2017). DEIVA: a web application for interactive visual analysis of differential gene expression profiles. BMC Genomics <i>18</i>. <a class="external-link" href="https://doi.org/10.1186/s12864-016-3396-5">https://doi.org/10.1186/s12864-016-3396-5</a></p>
<p id="bib13">Huang, D.W., Sherman, B.T., and Lempicki, R.A. (2008). Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. Nucleic Acids Res. <i>37</i>, 1-13. <a class="external-link" href="https://doi.org/10.1093/nar/gkn923">https://doi.org/10.1093/nar/gkn923</a></p>
<p id="bib14">Huang, D.W., Sherman, B.T., and Lempicki, R.A. (2009). Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nat. Protoc. <i>4</i>, 44-57. <a class="external-link" href="https://doi.org/10.1038/nprot.2008.211">https://doi.org/10.1038/nprot.2008.211</a></p>
<p id="bib15">Kühl, I., Miranda, M., Atanassov, I., Kuznetsova, I., Hinze, Y., Mourier, A., Filipovska, A., and Larsson, N.G. (2017). Transcriptomic and proteomic landscape of mitochondrial dysfunction reveals secondary coenzyme Q deficiency in mammals. eLife <i>6</i>, e30952. <a class="external-link" href="https://doi.org/10.7554/eLife.30952">https://doi.org/10.7554/eLife.30952</a></p>
<p id="bib16">Naumov, V., Balashov, I., Lagutin, V., Borovikov, P., and Alexeev, A. (2017). VolcanoR - web service to produce volcano plots and do basic enrichment analysis. bioRxiv <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref16">View at publisher</a></p>
<p id="bib17">Pagliarini, D.J., Calvo, S.E., Chang, B., Sheth, S.A., Vafai, S.B., Ong, S.E., Walford, G.A., Sugiana, C., Boneh, A., Chen, W.K., et al. (2008). A mitochondrial protein compendium elucidates complex I disease biology. Cell <i>134</i>, 112-123. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref17">View at publisher</a></p>
<p id="bib18">Perks, K.L., Rossetti, G., Kuznetsova, I., Hughes, L.A., Ermer, J.A., Ferreira, N., Busch, J.D., Rudler, D.L., Spahr, H., Schöndorf, T., et al. (2018). PTCD1 is required for 16S rRNA maturation complex stability and mitochondrial ribosome assembly. Cell Rep. <i>23</i>. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2018.03.033">https://doi.org/10.1016/j.celrep.2018.03.033</a></p>
<p id="bib33">Perrier, V., Meyer, F., Granjon, D., Fellows, I., Davis, W., and Matthews, S. <a class="external-link" href="https://cran.r-project.org/web/packages/shinyWidgets/index.html">https://cran.r-project.org/web/packages/shinyWidgets/index.html</a>.</p>
<p id="bib20">Rudler, D.L., Hughes, L.A., Perks, K.L., Richman, T.R., Kuznetsova, I., Ermer, J.A., Abudulai, L.N., Shearwood, A.J., Viola, H.M., Hool, L.C., et al. (2019). Fidelity of translation initiation is required for coordinated respiratory complex assembly. Sci. Adv. <i>5</i>, eaay2118. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref20">View at publisher</a></p>
<p id="bib21">Sievert, C. (2018). plotly for R. <a class="external-link" href="https://plotly-r.com">https://plotly-r.com</a>.</p>
<p id="bib22">Siira, S.J., Rossetti, G., Richman, T.R., Perks, K., Ermer, J.A., Kuznetsova, I., Hughes, L., Shearwood, A.J., Viola, H.M., Hool, L.C., et al. (2018). Concerted regulation of mitochondrial and nuclear non-coding RNAs by a dual-targeted RNase Z. EMBO Rep. <i>19</i>, e46198. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref22">View at publisher</a></p>
<p id="bib23">Singh, S., Hein, M.Y., and Stewart, A.F. (2016). msVolcano: A flexible web application for visualizing quantitative proteomics data. Proteomics <i>16</i>, 2491-2494. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref23">View at publisher</a></p>
<p id="bib24">Team, R.C. (2019). R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref24">View at publisher</a></p>
<p id="bib25">Wickham, H., François, R., Henry, L., and Müller, K. (2019). dplyr: A Grammar of Data Manipulation. <a class="external-link" href="https://cran.r-project.org/web/packages/dplyr/index.html">https://cran.r-project.org/web/packages/dplyr/index.html</a>.</p>
<p id="bib32">Thul, P.J., Åkesson, L., Wiking, M., Mahdessian, D., Geladaki, A., Ait Blal, H., Alm, T., Asplund, A., Björk, L., Breckels, L.M., et al. (2017). A subcellular map of the human proteome. Science <i>356</i>. <a class="external-link" href="https://doi.org/10.1126/science.aal3321">https://doi.org/10.1126/science.aal3321</a></p>
<p id="bib26">Wickham, H. (2016). ggplot2: Elegant Graphics for Data Analysis (Springer-Verlag). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref26">View at publisher</a></p>
<p id="bib27">Wickham, H. (2019). stringr: Simple, Consistent Wrappers for Common String Operations (https://CRAN.R-project.org/package=stringr). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref27">View at publisher</a></p>
<p id="bib28">Wickham, H., Henry, L., Lin Pedersen, T., Luciani, T.J., Decorde, M., and Lise, V. (2020). svglite: An ‘SVG’ Graphics Device (https://CRAN.R-project.org/package=svglite). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30266-5/sref28">View at publisher</a></p>
<p id="bib29">Xie, Y., Cheng, J., and Tan, X. (2020). DT: A Wrapper of the JavaScript Library ’DataTables. <a href="https://CRAN.R-project.org/package=DT">https://CRAN.R-project.org/package=DT</a>.</p>
<p id="bib30">Yan, J., Risacher, S.L., Shen, L., and Saykin, A.J. (2017). Network approaches to systems biology analysis of complex disease: integrative methods for multi-omics data. Brief. Bioinform. <i>19</i>, 1370-1381. <a class="external-link" href="https://doi.org/10.1093/bib/bbx066">https://doi.org/10.1093/bib/bbx066</a></p>
<p id="bib31">Yim, A., Koti, P., Bonnard, A., Marchiano, F., Dürrbaum, M., Garcia-Perez, C., Villaveces, J., Gamal, S., Cardone, G., Perocchi, F., et al. (2019). mitoXplorer, a visual data mining platform to systematically analyze and visualize mitochondrial expression dynamics and mutations. Nucleic Acids Res. <i>48</i>, 605-632. <a class="external-link" href="https://doi.org/10.1093/nar/gkz1128">https://doi.org/10.1093/nar/gkz1128</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Aleksandra Filipovska (<a href="mailto:aleksandra.filipovska@uwa.edu.au">aleksandra.filipovska@uwa.edu.au</a>).</p>
<h4>Materials Availability</h4>
<p>The software generated in this study is freely available from: <a href="https://github.com/IrinaVKuznetsova/OmicsVolcano">https://github.com/IrinaVKuznetsova/OmicsVolcano</a></p>
<h4>Data and Code Availability</h4>
<p>The example datasets and code generated during this study are available at.: <a href="https://github.com/IrinaVKuznetsova/OmicsVolcano">https://github.com/IrinaVKuznetsova/OmicsVolcano</a>. All data generated or analyzed during this study are included in this published article.</p>
<h3>Acknowledgments</h3>
<p>This project was supported by fellowships and project grants from the <a href="https://doi.org/10.13039/501100000925">National Health and Medical Research Council</a> (APP1058442, APP1045677, APP1041582, APP1023460, APP1005030, APP1043978 to A.F. and O.R.), the <a href="https://doi.org/10.13039/501100000923">Australian Research Council</a> (DP180101656 to A.F. and O.R.), and the Cancer Council of Western Australia (to A.F. and O.R.). I.K. is supported by a UWA Postgraduate Scholarship. The funders had no role in the design of the study, collection, analysis, and interpretation of data or in the preparation of the manuscript. We thank members of the Filipovska and Rackham laboratories for testing the software and providing valuable advice.</p>
<h3>Author Contributions</h3>
<p>I.K. contributed to the algorithm development, script implementation, software design, and manuscript writing. A.L. contributed to the supervision of the software design, software implementation, and manuscript writing. O.R. supervised the software design and manuscript writing. A.F. acted as a principal supervisor of the project and contributed to each stage of the project and manuscript writing. All authors read and approved the final manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>