<article>
<section>
<h1 id="header">Low-input ATAC&amp;mRNA-seq protocol for simultaneous profiling of chromatin accessibility and gene expression</h1>
<p><time datetime="2021-08-27">Published: August 27, 2021</time></p>
<p>Ruifang Li,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup> Sara A. Grimm,<sup><a href="#aff2">2</a></sup> and Paul A. Wade<sup><a href="#aff3">3</a></sup></p>
<p id="aff1"><sup>1</sup>Epigenetics Innovation Lab, Center for Epigenetics Research, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</p>
<p id="aff2"><sup>2</sup>Integrative Bioinformatics Support Group, Epigenetics and Stem Cell Biology Laboratory, National Institute of Environmental Health Sciences, Research Triangle Park, NC 27709, USA</p>
<p id="aff3"><sup>3</sup>Eukaryotic Transcriptional Regulation Group, Epigenetics and Stem Cell Biology Laboratory, National Institute of Environmental Health Sciences, Research Triangle Park, NC 27709, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:lir1@mskcc.org">lir1@mskcc.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100764">10.1016/j.xpro.2021.100764</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We present a simple, fast, and robust protocol (low-input ATAC&amp;mRNA-seq) to simultaneously generate ATAC-seq and mRNA-seq libraries from the same cells in limited cell numbers by coupling a simplified ATAC procedure using whole cells with a novel mRNA-seq approach that features a seamless on-bead process including direct mRNA isolation from the cell lysate, solid-phase cDNA synthesis, and direct tagmentation of mRNA/cDNA hybrids for library preparation. It enables dual-omics profiling from limited material when joint epigenome and transcriptome analyses are needed.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref1">Li et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A simple and fast dual-omics profiling protocol for low-input samples</li>
<li>Simultaneous measurement of open chromatin and gene expression from the same cells</li>
<li>Suitable for limited numbers of cells ranging from 5,000 to 20,000 cells</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/968-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps for using 10,000 mouse embryonic stem cells (mESCs). However, we have also used this protocol for 5,000 and 20,000 mESCs with adjustment of reaction volumes in some of the steps.</p>
<h3 id="sec1.1">Make sure the required materials and equipment are ready to use</h3>
<ol>
<li>If needed, order and prepare reagents and kits per manufacturer’s instructions. The required reagents and kits are listed in <a href="#key-resources-table">key resources table</a> below.
<ol type="a">
<li>Confirm kit contents and make sure that you have the required components in sufficient amount.</li>
<li>Order and resuspend ATAC-seq PCR primers to 10 μM working concentration.
<div class="note">
<span class="note-title">Note:</span> The sequences of universal i5 primer and index i7 primers were published previously (<a href="#bib2">Buenrostro et al., 2015</a>).</div>
</li>
<li>Prepare 1% Digitonin – Digitonin (Promega cat# G9441) is supplied at 2% in DMSO. Dilute 1:1 with nuclease-free water to make a 1% (100 ×) stock solution. It can be kept at −20°C for up to 6 months. Avoid more than 5 freeze-thaw cycles.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Set up and save the thermocycler programs to be run in this protocol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See the detailed settings below in “<a href="#materials-and-equipment">materials and equipment</a>” section.</div>
<ol start="3">
<li>The day before the experiment, confirm that the Dynabeads® Oligo (dT)25 is in liquid suspension. If the Dynabeads® have dried in the vial, resuspend the beads in the buffer they are supplied in by placing the vial on a mixer overnight at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Store the vials of Dynabeads® Oligo (dT)25 upright to avoid drying of the beads.</div>
<ol start="4">
<li>If multiple samples are processed at the same time, calculate in advance the total amounts of ATAC transposition mixture and reverse transcription reaction mix needed for the experiment.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See instructions below in step 2 and step 23.</div>
<ol start="5">
<li>On the day of the experiment, pre-chill microcentrifuge to 4°C and set thermomixer to 37°C.</li>
<li>To prevent RNase contamination, follow best laboratory practices for handling RNA during reagents preparation and during the experiment; for example, wear disposable gloves and change them frequently, use sterile and RNase-free tubes and pipette tips with aerosol barriers, clean pipettors and the bench with RNaseZap® solution (Invitrogen cat# AM9780).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Unless a safe stopping point is specified in the protocol, proceed immediately to the next step.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Digitonin</td>
<td>Promega</td>
<td>Cat# G9441</td>
</tr>
<tr>
<td>TWEEN® 20</td>
<td>Sigma-Aldrich</td>
<td>Cat# 11332465001</td>
</tr>
<tr>
<td>DPBS, no calcium, no magnesium</td>
<td>Gibco™</td>
<td>Cat# 14190250</td>
</tr>
<tr>
<td>SUPERase•In™ RNase Inhibitor (20 U/μL)</td>
<td>Invitrogen™</td>
<td>Cat# AM2696</td>
</tr>
<tr>
<td>Water, nuclease-free</td>
<td>Thermo Scientific™</td>
<td>Cat# R0581</td>
</tr>
<tr>
<td>Lithium chloride solution (8M)</td>
<td>Sigma-Aldrich</td>
<td>Cat# L7026-100ML</td>
</tr>
<tr>
<td>Ethylenediaminetetraacetic acid solution (0.5 M EDTA)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 03690-100ML</td>
</tr>
<tr>
<td>Sodium Acetate Solution (3 M), pH 5.2</td>
<td>Thermo Scientific™</td>
<td>Cat# R1181</td>
</tr>
<tr>
<td>NEBNext® High-Fidelity 2× PCR Master Mix</td>
<td>NEB</td>
<td>Cat# M0541S</td>
</tr>
<tr>
<td>AMPure XP beads</td>
<td>Beckman Coulter</td>
<td>Cat# A63881</td>
</tr>
<tr>
<td>Ethyl alcohol, Pure</td>
<td>Sigma-Aldrich</td>
<td>Cat# E7023-500ML</td>
</tr>
<tr>
<td>Trypan Blue Solution, 0.4%</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15250061</td>
</tr>
<tr>
<td>RNaseZap™ RNase Decontamination Solution</td>
<td>Invitrogen™</td>
<td>Cat# AM9780</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Illumina Tagment DNA Enzyme and Buffer Large Kit</td>
<td>Illumina</td>
<td>Cat# 20034198</td>
</tr>
<tr>
<td>Dynabeads™ mRNA DIRECT™ Micro Purification Kit</td>
<td>Invitrogen™</td>
<td>Cat# 61021</td>
</tr>
<tr>
<td>SuperScript™ IV First-Strand Synthesis System</td>
<td>Invitrogen™</td>
<td>Cat# 18091050</td>
</tr>
<tr>
<td>Nextera XT DNA Library Preparation Kit</td>
<td>Illumina</td>
<td>Cat# FC-131-1024</td>
</tr>
<tr>
<td>Nextera XT Index Kit (24 indexes, 96 samples)</td>
<td>Illumina</td>
<td>Cat# FC-131-1001</td>
</tr>
<tr>
<td>MinElute PCR Purification Kit (250)</td>
<td>QIAGEN</td>
<td>Cat# 28006</td>
</tr>
<tr>
<td>Qubit™ dsDNA HS Assay Kit</td>
<td>Invitrogen™</td>
<td>Cat# Q32851</td>
</tr>
<tr>
<td>D1000 ScreenTape</td>
<td>Agilent</td>
<td>Cat# 5067-5582</td>
</tr>
<tr>
<td>D1000 Reagents</td>
<td>Agilent</td>
<td>Cat# 5067-5583</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Low-input ATAC&amp;mRNA-seq data</td>
<td>This paper</td>
<td>GEO: GSE165478</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>E14 mouse embryonic stem cells</td>
<td>Kristian Helin Lab</td>
<td>RRID: CVCL_C320</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Universal i5 primer: AATGATACGGCGACCACCGAGATCTACACTCGTCGGCAGCGTCAGATGTG</td>
<td><a href="#bib9">Buenrostro et al., 2013</a></td>
<td>Ad1_noMX</td>
</tr>
<tr>
<td>index i7 primer_Ad2.31: CAAGCAGAAGACGGCATACGAGATTAACTTATGTCTCGTGGGCTCGGAGATGTG</td>
<td><a href="#bib2">Buenrostro et al. 2015</a></td>
<td>v2_Ad2.31_ATAAGTTA</td>
</tr>
<tr>
<td>index i7 primer_Ad2.32: CAAGCAGAAGACGGCATACGAGATCGAGTGATGTCTCGTGGGCTCGGAGATGTG</td>
<td><a href="#bib2">Buenrostro et al. 2015</a></td>
<td>v2_Ad2.32_ATCACTCG</td>
</tr>
<tr>
<td>index i7 primer_Ad2.34: CAAGCAGAAGACGGCATACGAGATCTACCATTGTCTCGTGGGCTCGGAGATGTG</td>
<td><a href="#bib2">Buenrostro et al. 2015</a></td>
<td>v2_Ad2.34_AATGGTAG</td>
</tr>
<tr>
<td>index i7 primer_Ad2.35: CAAGCAGAAGACGGCATACGAGATACGTGCTCGTCTCGTGGGCTCGGAGATGTG</td>
<td><a href="#bib2">Buenrostro et al. 2015</a></td>
<td>v2_Ad2.35_GAGCACGT</td>
</tr>
<tr>
<td>index i7 primer_Ad2.36: CAAGCAGAAGACGGCATACGAGATTGACGAAAGTCTCGTGGGCTCGGAGATGTG</td>
<td><a href="#bib2">Buenrostro et al. 2015</a></td>
<td>v2_Ad2.36_TTTCGTCA</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Cutadapt v1.12</td>
<td><a href="#bib7">Martin, 2011</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Bowtie2 v2.1.0</td>
<td><a href="#bib5">Langmead and Salzberg, 2012</a></td>
<td>N/A</td>
</tr>
<tr>
<td>samtools v1.3.1</td>
<td><a href="#bib6">Li et al., 2009</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Picard tools v1.110</td>
<td><a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a></td>
<td>N/A</td>
</tr>
<tr>
<td>BEDtools v2.24.0</td>
<td><a href="#bib8">Quinlan and Hall, 2010</a></td>
<td>N/A</td>
</tr>
<tr>
<td>BedGraphToBigWig</td>
<td><a href="http://hgdownload.soe.ucsc.edu/admin/exe/">http://hgdownload.soe.ucsc.edu/admin/exe/</a></td>
<td>N/A</td>
</tr>
<tr>
<td>STAR v2.5</td>
<td><a href="#bib4">Dobin et al., 2013</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Refrigerated microcentrifuge</td>
<td>Eppendorf</td>
<td>Cat# 5425 R</td>
</tr>
<tr>
<td>MicroCL 21 Microcentrifuge</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 75002466</td>
</tr>
<tr>
<td>Ohaus™ Frontier™5306 Mini Centrifuge</td>
<td>Fisher Scientific</td>
<td>Cat# 02-112-305</td>
</tr>
<tr>
<td>DynaMag™-2 Magnet</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12321D</td>
</tr>
<tr>
<td>Magnetic Separation Rack, 0.2 mL Tubes</td>
<td>EpiCypher</td>
<td>Cat# 10-0008</td>
</tr>
<tr>
<td>96-Well Aluminum Block For 0.2 mL Tubes</td>
<td>Universal Medical</td>
<td>Cat# 81001</td>
</tr>
<tr>
<td>Tube rotator</td>
<td>Labnet</td>
<td>Cat# H5500</td>
</tr>
<tr>
<td>ThermoMixer</td>
<td>Eppendorf</td>
<td>Cat# 5384000020</td>
</tr>
<tr>
<td>Qubit Fluorometer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# Q33216</td>
</tr>
<tr>
<td>2200 TapeStation System</td>
<td>Agilent</td>
<td>Cat# G2964AA</td>
</tr>
<tr>
<td>Applied Biosystems ProFlex PCR System</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 4484073</td>
</tr>
<tr>
<td>Countess™ II Automated Cell Counter</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AMQAX1000</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol uses Qiagen MinElute PCR Purification Kit (cat# 28006) to purify ATAC-DNA. However, Zymo DNA Clean and Concentrator-5 Kit (cat# D4014) can be used instead.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol uses Agilent TapeStation to assess library size distribution. Alternatively, Agilent Bioanalyzer can be used.</div>
<h3 id="sec2.1">Thermocycler program setup</h3>
<ul>
<li>Save the following “RT” program on the thermal cycler:</li>
</ul>
<p>Choose the preheat lid option and set to 100°C</p>
<p>Set the reaction volume to 20 μL</p>
<table id="undtbl1">
<thead>
<tr>
<th colspan="4">Incubation conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial incubation</td>
<td>50°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Reverse transcription</td>
<td>55°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<ul>
<li>Save the following “TAG” program on the thermal cycler:</li>
</ul>
<p>Choose the preheat lid option and set to 100°C</p>
<p>Set the reaction volume to 20 μL</p>
<table id="undtbl2">
<thead>
<tr>
<th colspan="4">Incubation conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tagmentation</td>
<td>55°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>10°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<ul>
<li>Save the following “NXT PCR” program on the thermal cycler:</li>
</ul>
<p>Choose the preheat lid option and set to 100°C</p>
<p>Set the reaction volume to 50 μL</p>
<table id="undtbl3">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Gap filling</td>
<td>72°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>10 s</td>
<td rowspan="3">13–15 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>10°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<ul>
<li>Save the following “ATAC PCR” program on the thermal cycler:</li>
</ul>
<p>Choose the preheat lid option and set to 100°C</p>
<p>Set the reaction volume to 50 μL</p>
<table id="undtbl4">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Gap filling</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">8–10 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>63°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<h3 id="sec2.2">Recipes</h3>
<ul>
<li>ATAC transposition mixture (10 μL for 10,000 cells)</li>
</ul>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>16.5 μL</td>
</tr>
<tr>
<td>nuclease-free H2O</td>
<td>n/a</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>TD Buffer</td>
<td>1×</td>
<td>25 μL</td>
</tr>
<tr>
<td>10% Tween-20</td>
<td>1%</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>1% Digitonin</td>
<td>0.01%</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>TDE1</td>
<td>n/a</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>SUPERase•In™ RNase Inhibitor (20 U/μL)</td>
<td>1 U/μL</td>
<td>2.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare freshly at step 2 of the experiment. Store on ice before use.</div>
<ul>
<li>Reverse transcription reaction mix (20-μL reaction)</li>
</ul>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DEPC-treated water</td>
<td>n/a</td>
<td>12 μL</td>
</tr>
<tr>
<td>5× SSIV Buffer</td>
<td>1×</td>
<td>4 μL</td>
</tr>
<tr>
<td>10 mM dNTP mix (10 mM each)</td>
<td>0.5 mM each</td>
<td>1 μL</td>
</tr>
<tr>
<td>100 mM DTT</td>
<td>5 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Ribonuclease Inhibitor (40 U/μL)</td>
<td>2 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td>SuperScript™ IV Reverse Transcriptase (200 U/μL)</td>
<td>10 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare freshly at step 23 of the experiment. Store on ice before use.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">ATAC tagmentation using whole cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 50–60 min</div>
<p>This step uses Illumina Tagment DNA Enzyme and Buffer Kit (Cat# 20034198) to fragment and tag open chromatin regions with adapter sequences.</p>
<div class="note">
<span class="note-title">Note:</span> This step is based on Omni-ATAC protocol (<a href="#bib3">Corces et al., 2017</a>) with modifications.</div>
<div class="critical">
<span class="critical-title">Critical:</span> This step requires single cell suspension, as cell clumps can cause inaccurate cell counts and inefficient cell permeabilization.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Cell viability needs to be above 90% and preferably around 95%.</div>
<div class="critical">
<span class="critical-title">Critical:</span> This step relies on a one-step membrane permeabilization and transposition. Different cell types may need different detergent or concentration for proper cell permeabilization. However, the combination of 0.01% digitonin and 0.1% Tween-20 is a good starting point.</div>
<ol>
<li>Cell preparation
<ol type="a">
<li>Wash the harvested mESCs once in cold DPBS, and then resuspend the cell pellet in cold DPBS.</li>
<li>Count the cells with a manual or automated cell counter. Trypan blue staining is recommended to assess cell viability.</li>
<li>Gently mix the cell suspension and pipette a volume corresponding to 1×10<sup>4</sup> cells into a new 1.5 mL Eppendorf tube.</li>
</ol>
</li>
<li>Centrifuge the tube containing 1×10<sup>4</sup> cells at 500 × <i>g</i> for 5 min at 4°C in a fixed angle centrifuge.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> While waiting for the centrifuge to stop, prepare ATAC transposition mixture as below and keep it on ice before use.</div>
<table id="undtbl7">
<caption>ATAC transposition mixture (10 μL for 10,000 cells)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>3.3 μL</td>
</tr>
<tr>
<td>nuclease-free H2O</td>
<td>n/a</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>TD Buffer</td>
<td>1×</td>
<td>5 μL</td>
</tr>
<tr>
<td>10% Tween-20</td>
<td>1%</td>
<td>0.1 μL</td>
</tr>
<tr>
<td>1% Digitonin</td>
<td>0.01%</td>
<td>0.1 μL</td>
</tr>
<tr>
<td>Tagment DNA Enzyme (TDE1)</td>
<td>n/a</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>SUPERase•In™ RNase Inhibitor (20 U/μL)</td>
<td>1 U/μL</td>
<td>0.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The volumes given here are for a single sample with 1<b>×</b>10<sup>4</sup> mESCs. For multiple samples, scale up and include 10% excess volume for pipetting losses.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The ratio of cell number to TDE1 is crucial. 10 μL transposition mixture is used for 1<b>×</b>10<sup>4</sup> mESCs. Based on the actual cell number of the sample, adjust the volume of ATAC transposition mixture to use proportionally.</div>
<ol start="3">
<li>Remove all the supernatant carefully using two pipetting steps (aspirate down to 100 ul with a P1000 pipette then remove final 100 ul with a P200 pipette).</li>
<li>Resuspend the cell pellet in 10 μL ATAC transposition mixture by pipetting up and down 6 times.</li>
<li>Incubate the transposition reaction at 37°C for 30 min in an Eppendorf Thermomixer with shaking at 1000 RPM.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> While waiting for the incubation to complete, please prepare the following.</div>
<p>Prepare EDTA-LiCl mixture by mixing 88 μL of 0.5 M EDTA with 275 μL of 8M LiCl. The mixture is stable at room temperature (21°C–23°C) and can be reused in the future.</p>
<p>Prepare reagents for direct mRNA isolation from cells following step 7.</p>
<p>Thaw the components of SuperScript™ IV First-Strand Synthesis System that are frozen.</p>
<p>For each sample, prepare a mixture of 560 μL of Buffer PB (from Qiagen MinElute PCR Purification Kit) with 10 μL of 3 M sodium acetate, pH 5.2 in a labeled Eppendorf tube. Keep the mixture at room temperature (21°C–23°C) before use in step 12.</p>
<ol start="6">
<li>Immediately upon completion of the incubation, quick spin the sample tube, then add 1 μL of EDTA-LiCl mixture (0.1 volume of the ATAC transposition mixture used in step 4) to stop the transposition reaction.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Proceed immediately to step 8.</div>
<h3 id="sec3.2">Direct mRNA isolation from the tagmented cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15–20 min</div>
<p>This step uses Dynabeads™ mRNA DIRECT™ Micro Purification Kit (Cat# 61021) to isolate mRNA directly from the tagmented cells.</p>
<div class="note">
<span class="note-title">Note:</span> This step follows the manufacturer's manual with minor changes. <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/DynabeadmRNADIRECTMicro_UG_Rev004_20120514.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/DynabeadmRNADIRECTMicro_UG_Rev004_20120514.pdf</a>
</div>
<ol start="7">
<li>Prepare reagents
<ol type="a">
<li>Prepare buffers from Dynabeads™ mRNA DIRECT™ Micro Purification Kit
<ol type="i">
<li>Bring all buffers except the 10 mM Tris-HCl to room temperature before use. Store the 10 mM Tris-HCl on ice or at 2°C–8°C before use.</li>
<li>Confirm that the Lysis/Binding Buffer has not precipitated. If any precipitation is observed, warm to room temperature and shake to dissolve.</li>
</ol>
</li>
<li>Wash Dynabeads® Oligo (dT)25 before use
<ol type="i">
<li>Resuspend Dynabeads® Oligo (dT)25 by vortex for more than 30 s.
<div class="note">
<span class="note-title">Note:</span> Keep the Dynabeads® Oligo (dT)25 in liquid suspension during handling and storage to avoid reduced performance. If the Dynabeads® Oligo (dT)25 becomes dried out, resuspend the beads in the buffer they are supplied in by placing the vial on a roller or equivalent overnight at 2°C–8°C to restore their complete functionality.</div>
</li>
<li>Pipette 20 μL Dynabeads® Oligo (dT)25 into a 1.5 mL RNase-free Eppendorf tube.
<div class="note">
<span class="note-title">Note:</span> Use 20 μL Dynabeads® per sample with 1<b>×</b>10<sup>4</sup> cells. For multiple samples, include 5–10% excess volume for pipetting losses.</div>
</li>
<li>Add 1 mL of Lysis/Binding Buffer and mix.</li>
<li>Place the tube on a magnet for 1 min, then remove and discard the supernatant.</li>
<li>Remove the tube from the magnet, then resuspend the beads in 20 μL of Lysis/Binding Buffer (the same volume as the Dynabeads® used in step ii).</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Add 100 μL Lysis/Binding Buffer to the tagmented cells from step 6. Perform a repeated passage of the solution through a pipette tip to obtain complete lysis.</li>
<li>Add 20 μL pre-washed Dynabeads® Oligo (dT)25 to the cell lysate and pipette up and down 2–3 times to mix.</li>
<li>Place the tube on a sample rotator at room temperature (21°C–23°C) for 5 min with continuous rotation to allow the mRNA to anneal to the Dynabeads®.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> While waiting, prepare reverse transcription reaction mix following step 23.</div>
<ol start="11">
<li>Quick spin the sample tube then place it on the magnet for 1 min.</li>
<li>Transfer the supernatant into the Eppendorf tube containing Buffer PB and sodium acetate mixture prepared earlier.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Don’t discard the supernatant, as it contains the tagmented DNA for ATAC-seq library preparation. Store the mixture at room temperature before proceeding to step 42.</div>
<ol start="13">
<li>Remove the sample tube from the magnet and resuspend the Dynabeads®-mRNA complex in 100 μL Washing Buffer A by careful pipetting.</li>
<li>Place the sample tube on the magnet for 1 min, then remove and discard the supernatant.</li>
<li>Repeat steps 13 and 14 once.</li>
<li>Resuspend the Dynabeads®-mRNA complex in 100 μL Washing Buffer B.</li>
<li>Transfer the suspension into a RNase-free PCR tube.</li>
<li>Place the PCR tube on a magnet for 1 min, then discard the supernatant.</li>
<li>Resuspend the Dynabeads®-mRNA complex in 100 μL Washing Buffer B.</li>
<li>Place the PCR tube on the magnet for 1 min, then discard the supernatant.</li>
<li>Remove the tube from the magnet and resuspend the Dynabeads®-mRNA complex in 100 μL ice-cold 10 mM Tris-HCl.</li>
<li>Put the tube on ice and immediately proceed to step 24.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Don’t elute the isolated mRNA from the beads. The Dynabeads®–mRNA complex is used directly for reverse transcription.</div>
<h3 id="sec3.3">Solid-phase cDNA synthesis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20–25 min</div>
<p>This step uses SuperScript™ IV First-Strand Synthesis System (Cat# 18091050) for first-strand cDNA synthesis on beads with the bead-bound oligo (dT) as primer, resulting in mRNA/cDNA hybrids covalently linked to the Dynabeads.</p>
<div class="note">
<span class="note-title">Note:</span> This step follows the manufacturer's manual with minor changes. <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/SSIV_First_Strand_Synthesis_System_UG.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/SSIV_First_Strand_Synthesis_System_UG.pdf</a>
</div>
<ol start="23">
<li>Prepare reverse transcription reaction mix as below and keep it on ice before use.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Pre-warm the 5× SSIV Buffer to room temperature before use. Vortex and briefly centrifuge the buffer prior to preparing the reverse transcription reaction mix.</div>
<table id="undtbl8">
<caption>Reverse transcription reaction mix (20-μL reaction)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DEPC-treated water</td>
<td>n/a</td>
<td>12 μL</td>
</tr>
<tr>
<td>5× SSIV Buffer</td>
<td>1×</td>
<td>4 μL</td>
</tr>
<tr>
<td>10 mM dNTP mix (10 mM each)</td>
<td>0.5 mM each</td>
<td>1 μL</td>
</tr>
<tr>
<td>100 mM DTT</td>
<td>5 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Ribonuclease Inhibitor (40 U/μL)</td>
<td>2 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td>SuperScript™ IV Reverse Transcriptase (200 U/μL)</td>
<td>10 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The volumes given here are for a single 20-μL reverse transcription reaction. For multiple reactions, scale up and include 10% excess volume for pipetting losses.</div>
<ol start="24">
<li>Immediately before adding the reverse transcription reaction mix, place the tube containing the Dynabeads®-mRNA complex on the magnet for 1 min, then discard the supernatant.</li>
<li>Resuspend the Dynabeads®-mRNA complex in 20 μL reverse transcription reaction mix.</li>
<li>Incubate the reaction in a thermocycler with the “RT” program, 50°C for 5 min followed by 55°C for 10 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When using a thermostable reverse transcriptase and the bead-bound oligo (dT) as primer for first-strand cDNA synthesis, an initial incubation at 50°C for 5 minutes is necessary before proceeding at the recommended reaction temperature.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Proceed immediately once the incubation completes.</div>
<div class="note">
<span class="note-title">Note:</span> While waiting, carry out the following.</div>
<p>Thaw Amplicon Tagment Mix (ATM), Tagment DNA Buffer (TD), Nextera PCR Master Mix (NPM), and index adapters on ice. These reagents are from Illumina Nextera XT DNA Library Preparation Kit and Nextera XT Index Kit.</p>
<p>Purify ATAC-DNA from step 12 following instructions in step 42.</p>
<ol start="27">
<li>Quick spin and place the sample tube on the magnet for 30 s, then discard the supernatant.</li>
<li>Remove the tube from the magnet and resuspend the beads in 100 μL of ice-cold 10 mM Tris-HCl.</li>
<li>Place the tube on the magnet for 1 min, then discard the supernatant.</li>
<li>Repeat steps 28 and 29 once.</li>
<li>Remove the tube from the magnet and put it on a cold metal block.</li>
<li>Add 5 μL ice-cold 10 mM Tris-HCl directly on the beads, and immediately proceed to step 33.</li>
</ol>
<h3 id="sec3.4">mRNA-seq library preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 50–60 min</div>
<p>This step uses Illumina Nextera XT DNA Library Preparation Kit (Cat# FC-131-1024) to directly tagment mRNA/cDNA hybrids on beads and then amplify the tagmented cDNA with PCR using index 1 (i7) adapter and index 2 (i5) adapter in Nextera XT Index Kit (Cat# FC-131-1001).</p>
<div class="note">
<span class="note-title">Note:</span> This step follows the manufacturer's manual with minor changes. <a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/samplepreps_nextera/nextera-xt/nextera-xt-library-prep-reference-guide-15031942-05.pdf">https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/samplepreps_nextera/nextera-xt/nextera-xt-library-prep-reference-guide-15031942-05.pdf</a>
</div>
<ol start="33">
<li>Add 10 μL Tagment DNA Buffer (TD) and 5 μL Amplicon Tagment Mix (ATM) directly on the beads.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Tagmentation is sensitive to the input amount<b>.</b> The volumes given here have been validated for mRNA/cDNA hybrids generated from 1–2<b>×</b>10<sup>4</sup> mESCs.</div>
<ol start="34">
<li>Resuspend the beads by pipetting up and down 10 times to mix.</li>
<li>Place the sample tube in the preprogrammed thermocycler and run the “TAG” program (55°C for 5 min, then hold at 10°C).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> When the program reaches 10°C, immediately proceed to step 36 to inactivate the transposome.</div>
<ol start="36">
<li>Quick spin the sample tube, then add 5 μL Neutralize Tagment Buffer (NT) and mix by pipetting up and down 10 times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Check NT for precipitates. If present, vortex until all particulates are resuspended.</div>
<ol start="37">
<li>Incubate the sample tube at room temperature (21°C–23°C) for 5 min.</li>
<li>Add 5 μL i7 adapter, 5 μL i5 adapter, 15 μL Nextera PCR Master Mix (NPM) to the sample, then mix by pipetting up and down 10 times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Open only one index adapter tube at a time to prevent misplacing caps. If using Index Adapter Plate, add 10 μL pre-paired i7 and i5 index adapters. If multiple samples are processed at the same time, please follow Illumina Index Adapters Pooling Guide to select index adapters for each sample.</div>
<ol start="39">
<li>Place the sample tube in the preprogrammed thermocycler and run the “NXT PCR” program.</li>
</ol>
<table id="undtbl9">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Gap filling</td>
<td>72°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>10 s</td>
<td rowspan="3">13–15 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>10°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The PCR cycles needed is dependent on the input amount.</div>
<div class="note">
<span class="note-title">Note:</span> While waiting for the PCR program to complete, set up ATAC PCR reaction following step 43.</div>
<ol start="40">
<li>Once “NXT PCR” program is complete, quick spin the sample tube then place it on the magnet for 1 min.</li>
<li>Transfer the supernatant containing the amplified mRNA-seq library into a new PCR tube.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If you are stopping, store the supernatant at 2°C–8°C for up to 2 days or −20°C for longer time.</div>
<h3 id="sec3.5">ATAC-seq library preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–40 min</div>
<p>This step uses Qiagen MinElute PCR Purification Kit (Cat# 28006) to purify ATAC-DNA from step 12, and then amplifies the purified DNA with a limited-cycle PCR using NEBNext® High-Fidelity 2× PCR Master Mix (Cat# M0541S) and 10 μM ATAC-seq PCR primers (<a href="#bib2">Buenrostro et al., 2015</a>).</p>
<div class="note">
<span class="note-title">Note:</span> ATAC-DNA purification (step 42) follows the manufacturer's manual. <a href="https://www.qiagen.com/us/resources/download.aspx?id=521feb84-9cfa-45ea-9a8f-54c2999aa9a9&amp;lang=en">https://www.qiagen.com/us/resources/download.aspx?id=521feb84-9cfa-45ea-9a8f-54c2999aa9a9&amp;lang=en</a>
</div>
<div class="note">
<span class="note-title">Note:</span> PCR amplification of the purified ATAC-DNA (Step 43) follows Omni-ATAC protocol (<a href="#bib3">Corces et al., 2017</a>) with slight modifications.</div>
<ol start="42">
<li>Purify ATAC-DNA from step 12 using Qiagen MinElute PCR Purification Kit.
<ol type="a">
<li>Mix and apply the sample to the MinElute column.</li>
<li>Centrifuge for 1 min at 17,900 × <i>g</i> at room temperature in a microcentrifuge.</li>
<li>Discard flow-through and place the MinElute column back into the same tube.</li>
<li>Add 750 μL Buffer PE to the MinElute column and centrifuge at 17,900 × <i>g</i> for 1 min.
<div class="note">
<span class="note-title">Note:</span> Add ethanol (96%–100%) to Buffer PE before use (see bottle label for volume).</div>
</li>
<li>Discard flow-through and place the MinElute column back in the same tube. Centrifuge the column for an additional 1 min at maximum speed.
<div class="critical">
<span class="critical-title">Critical:</span> Residual ethanol from Buffer PE will not be completely removed unless the flowthrough is discarded before this additional centrifugation.</div>
</li>
<li>Place the MinElute column in a clean 1.5 mL microcentrifuge tube.</li>
<li>Add 11 μL Buffer EB (10 mM Tris·Cl, pH 8.5) to the center of the membrane to elute DNA. Let the column stand for 1 min, and then centrifuge for 1 min.
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the elution buffer is dispensed directly onto the center of the membrane for complete elution of bound DNA.</div>
</li>
<li>Repeat step 42 g once.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If you are stopping, store the purified ATAC-DNA at −20°C.</div>
</li>
</ol>
</li>
</ol>
<ol start="43">
<li>Amplify the purified ATAC-DNA with NEBNext® High-Fidelity 2× PCR Master Mix and ATAC-seq PCR primers (<a href="#bib2">Buenrostro et al., 2015</a>).
<ol type="a">
<li>Add 20 μL DNA, 2.5 μL universal i5 primer (10 μM), 2.5 μL index i7 primer (10 μM), and 25 μL NEBNext HiFi 2× PCR Master Mix into a PCR tube.
<div class="note">
<span class="note-title">Note:</span> If multiple samples are processed at the same time, make sure that each sample uses a different index i7 primer (<a href="#bib2">Buenrostro et al., 2015</a>). If ATAC-seq and mRNA-seq libraries will be sequenced together, please follow Illumina Index Adapters Pooling Guide to select the indexes compatible with those used in step 38 for mRNA-seq libraries.</div>
</li>
<li>Mix, quick spin, then place the PCR tube in the preprogrammed thermocycler and run the “ATAC PCR” program.
<div class="note">
<span class="note-title">Note:</span> The PCR cycles needed is dependent on the input amount. The optimal PCR cycles can be determined by qPCR amplification after initial 5 cycles according to Omni-ATAC protocol (<a href="#bib3">Corces et al., 2017</a>).</div>
</li>
</ol>
</li>
</ol>
<table id="undtbl10">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Gap filling</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">8–10 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>63°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If you are stopping, store the PCR product at 2°C–8°C for up to 2 days or −20°C for longer time.</div>
<h3 id="sec3.6">Clean up mRNA-seq and ATAC-seq libraries</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 35–40 min</div>
<p>This step uses Agencourt AMPure XP beads (Cat# A63881) to clean up the amplified mRNA-seq library from step 41 and ATAC-seq library from step 43 with single-sided bead purification.</p>
<div class="note">
<span class="note-title">Note:</span> Bring AMPure XP beads to room temperature for at least 30 minutes before use.</div>
<ol start="44">
<li>Vortex AMPure XP beads and add 1.8× beads to each sample (90 μL beads for 50 μL PCR).</li>
<li>Gently pipette the entire volume up and down 10 times to mix thoroughly.</li>
<li>Incubate at room temperature (21°C–23°C) for 15 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Thaw Illumina Resuspension Buffer (RSB) at room temperature.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare fresh 80% ethanol 400 μL per sample.</div>
<ol start="47">
<li>Place the PCR tubes on the magnetic stand for 5 min or until the liquid is clear.</li>
<li>Remove and discard all supernatant without disturbing the beads.</li>
<li>With the PCR tubes on the magnetic stand, add 200 μL freshly prepared 80% ethanol without disturbing the beads.</li>
<li>Incubate at room temperature for 30 s, then remove and discard all the supernatant.</li>
<li>Repeat steps 49 and 50 for a total of two ethanol washes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use a P10 pipette to remove and discard residual ethanol after the second wash.</div>
<ol start="52">
<li>With the PCR tubes on the magnetic stand, air-dry the beads at room temperature (21°C–23°C) for 8 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not overdry the beads, as it may result in lower DNA recovery. If the beads appear cracked, they have been overdried.</div>
<ol start="53">
<li>Add 32.5 μL Resuspension Buffer (RSB) directly on the beads.</li>
<li>Remove the tubes from the magnetic stand, then resuspend the beads thoroughly by gently pipetting the entire volume up and down 10 times.</li>
<li>Incubate at room temperature (21°C–23°C) for 2 min.</li>
<li>Place the PCR tubes on the magnetic stand for 5 min or until the liquid is clear.</li>
<li>Transfer 30 μL of clear supernatant into a new Eppendorf tube labeled with corresponding library name.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If you are stopping, store the purified libraries at −20°C.</div>
<h3 id="sec3.7">Library quality control and sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20–30 min</div>
<p>This step checks library quality and quantity followed by Illumina sequencing.</p>
<ol start="58">
<li>Quantify the libraries using Qubit® Fluorometer with Qubit dsDNA HS Assay Kit (Cat# Q32851) according to manufacturer’s User Guide (<a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/Qubit_dsDNA_HS_Assay_UG.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/Qubit_dsDNA_HS_Assay_UG.pdf</a>).</li>
<li>Determine library size distribution using Agilent TapeStation with D1000 ScreenTape &amp; Reagents following the manufacturer’s User Manual (<a href="https://www.agilent.com/cs/library/usermanuals/public/ScreenTape_D1000_QG.pdf">https://www.agilent.com/cs/library/usermanuals/public/ScreenTape_D1000_QG.pdf</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Alternatively, Agilent Bioanalyzer can be used.</div>
<ol start="60">
<li>Sequence the libraries on Illumina sequencer following the manufacturer’s instructions.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Paired-end sequencing is recommended with sequencing depth following ENCODE guideline for bulk ATAC-seq and mRNA-seq.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Low-input ATAC&amp;mRNA-seq protocol simultaneously generates ATAC-seq and mRNA-seq libraries from the same cells. The library yield largely correlates with the input cell number and PCR cycles used for library amplification. Using 1×10<sup>4</sup> mESCs, the ATAC-seq library yield was ∼ 275 ng with 10 cycles of PCR, and the mRNA-seq library yield was ∼ 125 ng with 15 cycles of PCR. Regarding library size distribution, the ATAC-seq library exhibited nucleosome laddering pattern (<a href="#fig1">Figure 1</a>A), and the mRNA-seq library showed a single peak around 250 bp (<a href="#fig1">Figure 1</a>B). After sequencing, the ATAC-seq data displayed good signal enrichment at open chromatin regions (e.g., transcription start sites), while the mRNA-seq data showed read coverage at expressed genes (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/968-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Fragment size distribution of the libraries</div>
<p>Representative TapeStation traces of ATAC-seq (A) and mRNA-seq (B) libraries.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/968-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. UCSC genome browser view of ATAC-seq and mRNA-seq coverage tracks</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>ATAC-seq fastq files were filtered to remove all entries with a mean base quality score below 20 for either read in the read pair. Adapters were removed via Cutadapt v1.12 (<a href="#bib7">Martin, 2011</a>) with parameters "-a CTGTCTCTTATA -A CTGTCTCTTATA -O 5 -q 0 -m 30". The remaining filtered and trimmed read pairs were mapped against the mm10 reference assembly via Bowtie2 v2.1.0 (<a href="#bib5">Langmead and Salzberg, 2012</a>) with parameters “-X 2000 --fr --end-to-end --very-sensitive”, followed by filtering with samtools v1.3.1 (<a href="#bib6">Li et al., 2009</a>) at MAPQ5. Reads mapped to chrM were ignored in all downstream analysis. Duplicate mapped read pairs were removed by Picard tools MarkDuplicates.jar (v1.110) (<a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>) with parameter “REMOVE_DUPLICATES=TRUE”. Only the 9 bp at the 5′ end of each read was retained for downstream analysis. Coverage tracks for genome browser views were generated using BEDtools v2.24.0 (<a href="#bib8">Quinlan and Hall, 2010</a>) genomeCoverageBed with depth normalized to 10 million read ends per sample and then converted to bigWig format with UCSC utility bedGraphToBigWig (<a href="http://hgdownload.soe.ucsc.edu/admin/exe/">http://hgdownload.soe.ucsc.edu/admin/exe/</a>).</p>
<p>RNA-seq fastq files were filtered to remove all entries with a mean base quality score below 20 for either read in the read pair. Filtered read pairs were mapped against the mm10 reference assembly via STAR v2.5 (<a href="#bib4">Dobin et al., 2013</a>) with parameters "--outSAMattrIHstart 0 --outFilterType BySJout --alignSJoverhangMin 8 --limitBAMsortRAM 55000000000 --outSAMstrandField intronMotif --outFilterIntronMotifs RemoveNoncanonical". Coverage tracks were generated with STAR v2.5 with parameters “--runMode inputAlignmentsFromBAM --outWigType bedGraph --outWigStrand Unstranded --outWigNorm RPM”, followed by conversion to bigWig format via UCSC utility bedGraphToBigWig.</p>
<div class="note">
<span class="note-title">Note:</span> Use appropriate reference genome assembly according to the species of the samples.</div>
<div class="note">
<span class="note-title">Note:</span> Alternatively, BWA-MEM can be used for alignment of ATAC-seq data.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Low-input ATAC&amp;mRNA-seq protocol requires fresh live cells and is thus not suitable for frozen samples. The mRNA-seq library generated with this protocol is not strand specific.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low yield of ATAC-seq library (step 58)</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Below are the possible causes and potential solutions.</p>
<p>Inefficient cell permeabilization.</p>
<ol type="a">
<li>Make sure the cells are in single-cell suspension. Large cell clumps are difficult to permeabilize. (step 1)</li>
<li>The detergent and its concentration may need optimization for the cell type of interest. To ensure success, please first determine the optimal condition for cell permeabilization by trypan blue staining. Briefly, treat the cells with varying detergent and concentration, then examine the percentage of blue cells in trypan blue staining to empirically determine the minimal concentration required for efficient permeabilization (&gt;95% blue cells). (step 4)</li>
</ol>
<p>Low transposition efficiency.</p>
<p>Tn5 transposase requires magnesium ions as cofactor in the transposition reaction. To get rid of chelating agent EDTA in the cell suspension, wash the cells in PBS buffer without EDTA before transposition reaction. (step 1)</p>
<p>Insufficient PCR cycles.</p>
<p>Increase the PCR cycles for ATAC-seq library amplification. (step 43b)</p>
<p>Low recovery during library cleanup.</p>
<p>Don’t overdry the AMPure XP beads after the ethanol washes. (step 52)</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Poor signal-to-noise ratio of ATAC-seq data</p>
<h3 id="sec7.4">Potential solution</h3>
<p>The possible causes and potential solutions are as below.</p>
<p>Low cell viability.</p>
<p>Make sure the cells are alive with more than 90% viability. (step 1)</p>
<p>Suboptimal amount of TDE1 used.</p>
<p>Optimize TDE1 amount to use for the cell type of interest at a given cell number. (step 4)</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Low yield of mRNA-seq library (step 58)</p>
<h3 id="sec7.6">Potential solution</h3>
<p>The possible causes and solutions are as below.</p>
<p>RNA degradation.</p>
<ol type="a">
<li>Follow best practices of RNA handling to avoid RNase contamination during the experiment.</li>
<li>Consider adding more RNase inhibitor in ATAC transposition mixture and reverse transcription reaction mix. (step 4 and 23)</li>
</ol>
<p>Overtagmentation or undertagmentation of mRNA/cDNA hybrids.</p>
<p>Overtagmentation leads to fragments too small to be recovered by AMPure beads purification, while undertagmentation results in fragments too large to be amplified efficiently by PCR. Successful tagmentation highly depends on the ratio of the sample input to the amount of Amplicon Tagment Mix (ATM) used. Try different amount of ATM for the cell type of interest at a given cell number. (step 33)</p>
<p>Insufficient PCR cycles.</p>
<p>Increase the PCR cycles for mRNA-seq library amplification. (step 39)</p>
<p>Low recovery during library cleanup.</p>
<p>Don’t overdry the AMPure XP beads after the ethanol washes. (step 52)</p>
<h3 id="sec7.7">Problem 4</h3>
<p>The fragment size of mRNA-seq library is too small or too large (step 59)</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Avoid overtagmentation or undertagmentation of mRNA/cDNA hybrids as mentioned above. (step 33)</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Excessive large DNA fragments (&gt; 1 Kb) in ATAC-seq library (step 59)</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Conduct size selection with AMPure XP beads to remove large fragments. (step 44)</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Li, R., Grimm, S.A., and Wade, P.A. (2021). A simple and robust method for simultaneous dual-omics profiling with limited numbers of cells. Cell Rep. Methods <i>1</i>, 100041. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref1">View at publisher</a></p>
<p id="bib9">Buenrostro, J.D., Giresi, P.G., Zaba, L.C., Chang, H.Y., and Greenleaf, W.J. (2013). Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nat. Methods <i>10</i>, 1213-1218. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/optLyviADPj5Y">View at publisher</a></p>
<p id="bib2">Buenrostro, J.D., Wu, B., Litzenburger, U.M., Ruff, D., Gonzales, M.L., Snyder, M.P., Chang, H.Y., and Greenleaf, W.J. (2015). Single-cell chromatin accessibility reveals principles of regulatory variation. Nature <i>523</i>, 486-490. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref2">View at publisher</a></p>
<p id="bib3">Corces, M.R., Trevino, A.E., Hamilton, E.G., Greenside, P.G., Sinnott-Armstrong, N.A., Vesuna, S., Satpathy, A.T., Rubin, A.J., Montine, K.S., Wu, B., et al. (2017). An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. Nat. Methods <i>14</i>, 959-962. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref3">View at publisher</a></p>
<p id="bib4">Dobin, A., Davis, C.A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., and Gingeras, T.R. (2013). STAR: ultrafast universal RNA-seq aligner. Bioinformatics <i>29</i>, 15-21. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref4">View at publisher</a></p>
<p id="bib5">Langmead, B. and Salzberg, S.L. (2012). Fast gapped-read alignment with Bowtie 2. Nat. Methods <i>9</i>, 357-359. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref5">View at publisher</a></p>
<p id="bib6">Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R., and Genome Project Data Processing Subgroup. (2009). The sequence alignment/map format and SAMtools. Bioinformatics <i>25</i>, 2078-2079. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref6">View at publisher</a></p>
<p id="bib7">Martin, M. (2011). Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet J. <i>17</i>, 10-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref7">View at publisher</a></p>
<p id="bib8">Quinlan, A.R. and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics <i>26</i>, 841-842. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00470-6/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Ruifang Li (<a href="mailto:lir1@mskcc.org">lir1@mskcc.org</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The datasets generated during this study are available at NCBI Gene Expression Omnibus (GEO) with accession number GSE165478.</p>
<h3>Acknowledgments</h3>
<p>We thank Dr. Hua Wang in the Helin Lab for providing the mESCs. This work was supported by donations to the Center for Epigenetics Research from the Metropoulos Family Foundation and the <a href="https://doi.org/10.13039/100001246">Ambrose Monell Foundation</a> and by the Intramural Research Program of the <a href="https://doi.org/10.13039/100000002">NIH</a>, National Institute of Environmental Health Sciences (ES101965 to P.A.W.).</p>
<h3>Author contributions</h3>
<p>R.L. conceived the project and performed the experiments. R.L. and S.A.G. analyzed and interpreted the data. R.L. wrote the manuscript, and P.A.W. and S.A.G. reviewed and edited the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>