<article>
<section>
<h1 id="header">A zygote-based assay to evaluate intranuclear shuttling in <i>S. cerevisiae</i>
</h1>
<p><time datetime="2021-08-13">Published: August 13, 2021</time></p>
<p>Alan Michael Tartakoff<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Pathology Department and Cell Biology Program, Case Western Reserve University, Cleveland, OH 44106, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:amt10@case.edu">amt10@case.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100736">10.1016/j.xpro.2021.100736</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>It is often necessary to learn whether macromolecules occupy a fixed place in cells. This protocol makes it possible to learn whether individual nucleolar proteins in <i>S. cerevisiae</i> remain in place or depart from and return to the nucleolus. The protocol uses early zygotes in which parental nucleoli are separate for at least one hour. The protocol demonstrates that the localization of many nucleolar proteins is in fact highly dynamic. Photobleaching is not required.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib21">Tartakoff et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>The mobility of proteins within the yeast nucleus can be judged by constructing zygotes</li>
<li>The assay requires minimal equipment other than a sensitive fluorescent microscope</li>
<li>The protocol can be completed within one day once appropriate cells are available</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/922-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Many intracellular structures that are spatially separate are highly dynamic. Individual components may move within the structure or repeatedly depart and return. Moreover, if multiple copies of "the same" structure are present, components may exchange between them. Ongoing departure and return of components provides a means by which their composition can be adjusted.</p>
<p>The procedure described here is based on yeast zygote formation. Similar procedures would be applicable in other situations in which heterotypic cell-cell fusion (and nuclear fusion) can be achieved. The procedure requires cells that express fluorescent proteins of interest but does not rely on photobleaching. Related procedures have been used to learn whether proteins shuttle in and out of the nucleus (<a href="#bib11">Liu et al., 1996</a>, <a href="#bib6">Flach et al., 1994</a>, <a href="#bib5">Elion, 2002</a>). We explain how this approach can be used to evaluate the dynamics of nucleolar proteins when - after nuclear fusion (karyogamy) - parental nucleoli remain separate for more than one hour (<a href="#bib23">Tartakoff and Jaiswal, 2009</a>). If a protein exhibits internucleolar shuttling during this time interval, we conclude that the protein normally departs from and returns to the nucleolus. In corresponding control crosses we learn whether any apparent shuttling occurs in the absence of karyogamy. It is striking that such impermanence is characteristic of several protein/RNA assemblages in the nucleus (<a href="#bib13">Phair and Misteli, 2000</a>).</p>
<h3 id="sec1.1">Recurrent procedures</h3>
<ol>
<li>Conduct all steps at either room temperature or at 30°C.</li>
<li>Use synthetic media for all experiments, as described in the Cold Spring Harbor Manual (<a href="#bib2">Amberg et al., 2005</a>).</li>
<li>Culture cells in 24-well plasticware dishes that are shaken.</li>
<li>Sediment cell cultures for 5 s at 5,000 <b>×</b> g in a table-top microfuge after transfer to Eppendorf tubes.</li>
<li>Use 0.5 mL volumes for all cell incubations and washes.</li>
</ol>
<h3 id="sec1.2">Strain construction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–10 days.</div>
<p>To our knowledge, strains of distinct backgrounds can be used. We have used haploid strains derived from W303 or S288C. Strains of higher ploidy can presumably also be used if their mating types are appropriate.</p>
<ol start="6">
<li>Tag the protein(s) to be followed (e.g., the snoRNP protein,Sik1/Nop56, or the large ribosomal subunit assembly factor, Mak11) with a FP-fluorophore.</li>
<li>Ensure that the tagged protein continues to perform its normal functions when tagged. The resulting "donor" strains are typically crossed with an "acceptor" strain that does not express the same fluorophore. If comparative kinetics of relocation are to be investigated in single crosses, two distinct proteins can be tagged with different fluorophores, e.g., GFP <i>vs</i> mRFP, mCherry or CFP. Both can be expressed in the same cell or they can be expressed in separate cells of different mating type. Transfer thus can be followed in either parallel or anti-parallel fashion. We routinely use centromeric plasmids to express proteins of interest, tag them by covalent integration (<a href="#bib12">Longtine et al., 1998</a>, <a href="#bib10">Janke et al., 2004</a>, <a href="#bib18">Sidebottom et al., 2017</a>, <a href="#bib9">Gardner and Jaspersen, 2014</a>), acquire strains from other labs, or purchase strains (e.g., the GFP collection from Invitrogen/Life Technologies).</li>
<li>If the rate of ongoing synthesis and nuclear import make it impossible to focus on copies of the protein that existed before the cross was initiated, use strains in which expression of the tagged protein can be repressed.</li>
<li>For this purpose, if (over)expression is from a galactose-inducible <i>GAL1/10</i> promoter, pregrow cells in medium with 2% galactose and then transfer them to medium with 2% glucose during the cross. If expression is from a methionine-repressible <i>MET3</i> promoter, pregrow cells in medium lacking methionine and then transfer them to complete synthetic medium in which the concentration of methionine is 20 μg/mL (<a href="#bib2">Amberg et al., 2005</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Strains such as those in the commercially available GFP-tagged collection and deletion collection often carry the <i>met15Δ</i> deletion (e.g. if they are derived from BY4741/4742). They nevertheless grow in synthetic medium lacking methionine. This may be due to traces of methionine that are said to be present in commercial sources of leucine.</div>
<ol start="10">
<li>To evaluate the possible contribution of new synthesis or export to the cytoplasm followed by reimport, perform control crosses under conditions in which parental nuclei do not fuse. This can be achieved using donor or acceptor strains that carry the <i>kar1Δ15</i> mutation in the spindle pole body protein, Kar1. These cells progress normally through the mitotic cell cycle and form zygotes with good efficiency (<a href="#bib24">Vallen et al., 1992</a>). Nevertheless, nuclear fusion is strongly delayed. Moreover, when karyogamy does occur, fusion of spindle pole bodies is blocked, and progeny can be aneuploid (<a href="#bib22">Tartakoff et al., 2018</a>, <a href="#bib4">Dutcher, 1981</a>). <a href="#fig1">Figure 1</a> schematizes the progress of both [wt x wt] crosses and [<i>kar1</i> x wt] crosses.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/922-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Diagram of steps of cell and nuclear fusion in a [wt x wt] cross and a [<i>kar1Δ15</i> x wt] cross</div>
<p>In the illustrated time-courses, the first bud forms at a medial site, as is unusually the case (<a href="#bib26">Zapanta Rinonos et al., 2014</a>). The red rectangles indicate the zone of contact at which cell-cell fusion will occur. The distinct nucleolar proteins are represented by different colors. [wt x wt]: Note that the two nucleoli remain well-separated from each other after nuclear fusion, but that the color markers. interchange. [<i>kar1</i> x wt]: Note that nuclear fusion is delayed even though the nuclei may contact each other. The two colored nucleolar markers do not intermix. In the final images for the [<i>kar1</i> x wt] cross, a single dividing nucleus is illustrated. The second nucleus (although not illustrated) remains in the body of the zygote and divides at the same time (<a href="#bib26">Zapanta Rinonos et al., 2014</a>).</p>
</figcaption>
</figure>
<ol type="a">
<li>To introduce the <i>kar1Δ15</i> mutation into a <i>ura3</i> host transform <i>ura3</i> cells ("pop-in") with the integrating <i>URA3</i> plasmid pMR1593/pAT1457 (<a href="#bib24">Vallen et al., 1992</a>), after its linearization by cutting within <i>KAR1</i> using BglI, followed by column purification according to Qiagen. The Ura+ transformants that are selected have two tandem copies of <i>KAR1</i> (wt and mutant) flanking the selective marker. Use lithium acetate for transformation, but other procedures should work well. Use approximately 1/10 of a miniprep to transform a 0.5 mL culture of cells (O.D. = 2).</li>
<li>Grow the transformants in liquid medium under non-selective conditions for 10–20 h.</li>
<li>Plate 0.5 mL of the cell mixture on 5′-FOA plates to select for uracil auxotrophy (<i>ura-</i>) that usually results from recombination ("pop-out" events) that excises one copy of <i>KAR1</i> (<a href="#bib3">Boeke et al., 1987</a>, <a href="#bib2">Amberg et al., 2005</a>).<br> Collect individual colonies with a toothpick.</li>
<li>Grow them as 0.1 mL liquid cultures in preparation of using a bioassay to learn which form of <i>KAR1</i> is retained.
<ol type="i">
<li>Bioassay: Mix 20 μL of the cultures with 10–20 μL from a culture of wildtype cells of the opposite mating type that express a nuclear marker, e.g., Htb2-mRFP (ATY2289). Pipette the entire volume onto the surface of a Petri dish with complete medium.</li>
<li>After 3–5 h, wash the cells off the plates using 0.5 mL of growth medium and examine them by epifluorescence to evaluate nuclear fusion. Strains of interest characteristically produce zygotes that have two fluorescent nuclei, indicative of the block of karyogamy and implying that they carry the <i>kar1Δ15</i> mutation, rather than having a single large nucleus.
<div class="note">
<span class="note-title">Note:</span> In this example, the nuclear marker, Htb2-mRFP, routinely becomes present in all nuclei, even if introduced by a single parent.</div>
</li>
<li>Cells that carry the <i>kar1Δ15</i> mutation can be frozen at −80°C after mixing with an equal volume of 30% glycerol in growth medium.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Brightness verification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days.</div>
<ol start="11">
<li>With a toothpick, recover a sample of cells stored at −80°C.</li>
<li>Transfer into synthetic growth medium and shake the culture.</li>
<li>After 10 h, examine the cells in an inverted fluorescent microscope to ensure that the fluorescent nucleolar signal(s) are readily visible. Brightness is critical since cell-cell fusion generally reduces fluorescent intensity by half, as is illustrated below.</li>
</ol>
<h3 id="sec1.4">Preparation of petri dishes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h.</div>
<ol start="14">
<li>Autoclave synthetic medium containing 2% agar (<a href="#bib2">Amberg et al., 2005</a>).</li>
<li>Pour the medium into Petri dishes.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Leave the plates uncovered under a laminar flow hood for at least one hr at room temperature to allow residual liquid to evaporate, thereby ensuring that the haploid cells plated on them will dry down quickly and therefore be crowded together. If excess moisture remains, mating mixtures plated on them will remain in suspension and zygotes will form with poor efficiency.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point the plates can be stored for up to one week at 10°C.</div>
<h3 id="sec1.5">Preparation of agarose pads and their use</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min.</div>
<ol start="16">
<li>Add Agarose to growth medium (1.5% final).</li>
<li>Bring to a boil and vortex.</li>
<li>Apply 0.3 mL samples to the surface of a microscope slide that is immediately overlaid with a second slide.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> These slides can be kept in a horizontal position for 1–2 days in a humid slide box in a refrigerator.</div>
<ol start="19">
<li>When the cross has reached an appropriate point for examination, remove the upper slide by gentle horizontal sliding, thereby exposing two surfaces. 0.5–1.0 μL samples of cell pellets can be applied to either or both surfaces.</li>
<li>Add a 1 mm (No. 1) coverslip on top of the cells. For time-lapse imaging, seal the coverslip with vaseline to reduce evaporation. The procedure for preparing agarose-pads for imaging is illustrated in (<a href="#bib27">Zapanta Rinonos et al., 2012</a>).</li>
</ol>
<h3 id="sec1.6">Cell pre-growth</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day.</div>
<p>Mix equal volumes of actively growing cultures. Their final OD<sub>600</sub> should not exceed 0.5. If possible, adenine should be included in the medium to minimize autofluorescence (see below). As necessary, they can be grown in inducing medium, i.e., using galactose in place of glucose or in methionine-free medium.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>
<i>S. cerevisiae</i> strains and their derivatives</td>
<td>Invitrogen/Thermo Fisher</td>
<td>Yeast Deletion and Yeast GFP Collections</td>
</tr>
<tr>
<td>
<i>S. cerevisiae</i> MAT α Htb2-mRFP</td>
<td>Laboratory stock</td>
<td>ATY2289</td>
</tr>
<tr>
<td>
<i>S. cerevisiae</i> MAT α mRFP-HDEL</td>
<td>Laboratory stock</td>
<td>ATY3196</td>
</tr>
<tr>
<td>MAT α <i>kar1Δ15</i> mRFP-HDEL</td>
<td>Laboratory stock</td>
<td>ATY6618</td>
</tr>
<tr>
<td>MAT a Mak11-GFP Sik1-mRFP</td>
<td>Laboratory stock</td>
<td>ATY8300</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides or recombinant proteins</td>
</tr>
<tr>
<td>Media for cell growth</td>
<td>Sigma</td>
<td>Cold Spring Harbor Yeast Genetics Course Manual (<a href="#bib2">Amberg et al., 2005</a>)</td>
</tr>
<tr>
<td>Agarose</td>
<td>Invitrogen/Thermo Fisher</td>
<td>15510-027</td>
</tr>
<tr>
<td>Formaldehyde in water, 37%</td>
<td>Fisher Chemicals</td>
<td>F79</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pKar2frag-mRFP-HDELCut with Xho1 to integrate at <i>URA3</i>
</td>
<td>(<a href="#bib8">Gao et al., 2005</a>)</td>
<td>pTiKmRFP/pAT1180</td>
</tr>
<tr>
<td>ss-GFP-HDEL (<i>URA3</i>/CEN)</td>
<td>(<a href="#bib15">Prinz et al., 2000</a>)</td>
<td>pWP1055/pAT1182</td>
</tr>
<tr>
<td>Yip5 carrying the <i>kar1Δ15</i> mutationCut with BglII to integrate at <i>URA3,</i> then counterselect on 5′-FOA</td>
<td>M. Rose</td>
<td>pMR1593/pAT1457</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>SoftWorX deconvolution software</td>
<td>Applied Precision, Inc.</td>
<td><a href="http://www.sussex.ac.uk/gdsc/intranet/pdfs/softWoRx%20user%20manual.pdf">http://www.sussex.ac.uk/gdsc/intranet/pdfs/softWoRx user manual.pdf</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Titer plate shaker</td>
<td>Thermo Fisher</td>
<td>Labline 4625</td>
</tr>
<tr>
<td>Sterile toothpicks</td>
<td>Local market</td>
<td>Local market</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>A Deltavision RT epifluorescence microscope with an automated stage (Applied Precision, Inc) is used for fluorescent imaging with a 100<b>×</b> objective.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Basic shuttling assays</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day.</div>
<p><u>To follow a tagged protein that is expressed constitutively from its endogenous promoter</u>.</p>
<ol>
<li>Grow two strains of cells (see above), one of which expresses a fluorescent nucleolar protein. One is MAT a and the other is MAT α.</li>
<li>Combine aliquots of both parental cells (e.g., 100 μL of each). Vortex.</li>
<li>Sediment the cell mixtures and resuspend in synthetic medium with 2% glucose at OD600 ≈ 1.</li>
<li>Apply 25 μL samples of these mating mixtures to the surface of Petri dishes containing solid 2% agar in synthetic medium.</li>
<li>Leave open in a laminar-flow hood. Liquid should be absorbed in &lt; 15 min. The dishes are then covered. The overall timing of the cross is measured from the moment when the mating mixture is applied to the surface of the Petri dish.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> More time can be required for absorption if the volume of the samples is increased. In this event, the experiment will not be suitable for collecting kinetic information. See <a href="#troubleshooting">troubleshooting</a>.</div>
<ol start="6">
<li>After 2 h at room temperature, recover the cell mixture by washing with synthetic medium and gentle pipetting up and down 4–5<b>×</b>. Under optimal conditions, many pre-zygotes and zygotes should be visible at this time; however, a majority of haploid cells will remain.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Samples can be vortexed briefly but should not be sonicated. See <a href="#troubleshooting">troubleshooting</a>.</div>
<ol start="7">
<li>Sediment the cells. Without removing the supernatant, recover 0.5–1.0 μL samples of the cell pellet and apply this volume to the surface of freshly-opened Agarose pads, cover with a coverslip, and observe. For imaging in excess of 15 min duration, trim the pads to the dimensions of the coverslip with a single-edge razor blade (after adding the cells) and seal them with vaseline to minimize drying.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid nail polish or mounting media that include organic solvents since they will kill the cells.</div>
<ol start="8">
<li>To restrict attention to relatively early events, we examine zygotes that show no sign of bud emergence that coincides with onset of DNA synthesis (<a href="#bib17">Sena et al., 1976</a>). We then focus on early zygotes that show evidence of nuclear fusion (redistribution of ER/NE markers). In our experience, once transfer of label begins, equilibration is reached within 10–15 min (<a href="#bib21">Tartakoff et al., 2021</a>). As is described below, control experiments need to be performed in parallel.</li>
</ol>
<p>In these mating protocols, cell encounters, fusion and karyogamy are asynchronous, due to the variable proximity of potential mating partners and the fact that cell fusion can occur only when cells have reached the beginning of the cell cycle (<a href="#bib20">Tartakoff, 2015</a>, <a href="#bib7">Gammie and Rose, 2002</a>). Cells can be presynchronized, e.g., by exposure to α-factor which is then washed away when the two cell types are mixed. Alternatively, one can recover spent medium from cultures of MAT a and MAT α cells and use them to culture cells of the opposite mating type for 1–2 h. In our experience these procedures do not obviously increase the efficiency of zygote formation.</p>
<p>To follow tagged proteins whose expression can be repressed.</p>
<ol start="9">
<li>Pregrow one or both parental cells that expresses a tagged protein of interest from a repressible promoter, using a medium that allows expression (e.g., with galactose or lacking methionine).</li>
<li>When the mating mixture is prepared, adjust the composition of the plate used for the cross to repress transcription, e.g., by addition of glucose or methionine. For the many nucleolar proteins that we have studied - judging from the restriction of label to a single nucleolus in [<i>kar1Δ15</i> x wt] crosses - the rate of synthesis appears to be sufficiently low so that new synthesis is not a concern over the relevant time period (<a href="#bib21">Tartakoff et al., 2021</a>) - see below.</li>
</ol>
<p>Regardless of whether protein expression is repressed, intranuclear shuttling can be judged by determining whether the signal(s) that initially was in only one of the cells has become equally visible at both extremities of the elongated zygotic nucleus (i.e., in both of the nucleoli). In [wt x wt] crosses, <i>cis</i>-to-<i>trans</i> redistribution of nucleolar proteins begins once the two nuclei have contacted each other and in fact can be detected even before their zone of contact has visibly dilated (<a href="#bib23">Tartakoff and Jaiswal, 2009</a>).</p>
<p>According to the goal of the experiments, evaluate possible shuttling either at a single time point or by time-lapse imaging. When only a single time point is needed, the samples can be fixed, as follows:</p>
<ol start="11">
<li>Pipette the mating mixture off the plate with complete medium.</li>
<li>Add 1/10 volume of formaldehyde from a 37% stock in water, shaking for 5 min at room temperature.</li>
<li>Wash once with growth medium to quench free aldehydes.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store samples in water at this point for 1–2 days at 4°C before examination.</div>
<h3 id="sec3.2">Control experiments</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day.</div>
<p>To learn whether any appearance of fluorescent signal in the <i>trans</i> nucleus results from continued synthesis or nucleo-cytoplasmic shuttling, we can include two controls:</p>
<p><u>First control.</u></p>
<ol start="14">
<li>To learn whether transfer of label occurs before nuclear fusion, include a marker of the lumen of the nuclear envelope (and ER). This is either mRFP-HDEL or GFP-HDEL. The C-terminal HDEL peptide signal in conjunction with its receptor serves to retrieve cargo (mRFP, GFP) to the ER/NE if it reaches the Golgi (<a href="#bib16">Semenza et al., 1990</a>). We find that the "HDEL signal" itself redistributes almost instantaneously upon cell-cell fusion We therefore transform cells with a centromeric plasmid encoding GFP-HDEL (pWP1055/pAT1182)(<a href="#bib15">Prinz et al., 2000</a>), or an integrating plasmid encoding mRFP-HDEL (pTiKmRFP-HDEL/pAT1180) (<a href="#bib8">Gao et al., 2005</a>). We routinely include a HDEL marker in both [wt x wt] and [<i>kar1</i> x wt] crosses.</li>
</ol>
<p>Since many ribosome assembly factors, in addition to being concentrated in the nucleolus, are present at low levels throughout the nucleoplasm or actually line the nuclear envelope (<a href="#bib21">Tartakoff et al., 2021</a>), <i>cis-trans</i> continuity can often be evident without including an additional label (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/922-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Equilibration of the nucleolar protein, Mak11-GFP, in a [wt x wt] cross</div>
<p>Mating mixtures were applied to the surface of synthetic medium Petri dishes for 2, 3 or 4 h at room temperature and then examined. The brackets designate prezygotes and zygotes. The number of zygotes showing no transfer (a), transfer between separate nucleoli (b) or transfer and nucleolar fusion (c) is tabulated in the lower corners of the images. When relocation occurs, both nucleolar crescents become equally labeled. At 4 h, bud emergence is seldom visible. <u>Strains</u>: ATY8300, ATY3196.</p>
</figcaption>
</figure>
<p><u>Second control.</u></p>
<ol start="15">
<li>Judge the possible contribution of nucleocytoplasmic shuttling and new synthesis by following GFP-tagged AFs in crosses with a <i>kar1Δ15</i> mutant, as described above. None of the dozens of nucleolar proteins that we have studied show labeling of both nuclei in such dikaryons over 2.5 h (<a href="#bib21">Tartakoff et al., 2021</a>) - <a href="#fig3">Figure 3</a>.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/922-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative observations from a [<i>kar1Δ15</i> x wt] cross after 4 h</div>
<p>One of the parental cells expressed Mak11-GFP and Sik1-mRFP, while the other expressed mRFP-HDEL. The pairs of nuclei are encircled by the mRFP-HDEL signal that also highlights the ER at the periphery of each cell. In each zygote, one nucleus shows the strong Mak11-GFP and Sik1-mRFP signals. The other nucleus is either unlabeled or is labeled much more weakly as designated (∗). The scale bar is 5 microns. er: endoplasmic reticulum, N1/N2: nuclei, v: vacuole.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.3">Imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min–2 h</div>
<p>The number of early zygotes that can be imaged in a single field is difficult to control.</p>
<ol start="16">
<li>Collect images of 3−10 fields and examine at least 25 zygotes per cross.</li>
<li>Use a Deltavision RT epifluorescence microscope with an automated stage (Applied Precision, Inc) and an oil immersion objective (Olympus UPlanApo 100<b>×</b>/1.40; ∞/0.17/FN26.5).</li>
<li>Capture z-stacks at 0.5–1.0 μm intervals using a CCD digital camera (Photometrics CoolSnap HQ). % transmission = 100%, exposure time 0.05–1 s, field 1040<b>×</b>1040 pixels, binning 1<b>×</b>1.</li>
<li>Examine through-focal series spanning a total of 6 μ in all cases.</li>
<li>Remove out-of-focus light using the Softworks deconvolution program (<a href="http://www.sussex.ac.uk/gdsc/intranet/pdfs/softWoRx">http://www.sussex.ac.uk/gdsc/intranet/pdfs/softWoRx</a> user manual.pdf).</li>
<li>Repeat experiments 2–4 <b>×</b>. See <a href="#troubleshooting">troubleshooting</a>.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>To evaluate intranuclear shuttling we initiate crosses on synthetic medium plates and either let the cross proceed for 2 h and then examine the cell mixture immediately, or - alternatively - begin time-lapse examination. In this case, we typically examine at least 25 such zygotes and score them with regard to <i>cis-trans</i> equilibration of signal. The distribution of the zygotes in the image field will depend on their concentration in the sample that was applied to the Agarose pad. If too concentrated, they overlap and it is difficult to have multiple zygotes (or even both ends of the same zygote) in a single focal plane.</p>
<p>Time-lapse allows one to witness early stages of prezygote formation, as well as progression. In such experiments we find that the t<sub>1/2</sub> for Mak11-GFP and Sik1-mRFP equilibration is 5–15 min after nuclear fusion (<a href="#bib21">Tartakoff et al., 2021</a>). The presumably anoxic conditions under the coverslip are not obviously problematic, judging from the continued nuclear fusion and budding.</p>
<p>According to the time of incubation prior to being applied to the slide, a variable fraction of the zygotes will have progressed through the classic stages of 1) shmoos with a polarized nucleus (in which the spindle pole body faces the shmoo tip and the nucleolus is at the opposite extremity), 2) prezygotes (in which a flat interface has been established between the apices of both cells), 3) early "linear" zygotes (in which the nuclei make symmetric contact at their apices), 4) later "linear" zygotes (in which nuclei have fused together but there still is no sign of bud emergence), and 5) mature zygotes (in which the nucleoli have merged together and there are single buds at various stages of growth). Zygotes bud repeatedly, as do haploid and diploid cells (<a href="#bib19">Stone et al., 2000</a>, <a href="#bib26">Zapanta Rinonos et al., 2014</a>, <a href="#bib20">Tartakoff, 2015</a>, <a href="#bib25">Ydenberg and Rose, 2008</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Cells do not form many zygotes.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>The most important consideration is that the parental cells are growing rapidly (mid-log phase or more dilute) when they are crossed. Even if they have not reached saturation density, it is important to dilute them 5–10<b>×</b> into fresh medium and shake the cultures for 3–4 h before initiating a cross. It is not advisable to initiate cultures from stored samples and study them without 10 h growth, although the presence of dead bystander cells does not obviously interfere with zygote formation.</p>
<p>A major consideration is the speed with which the mating mixtures are absorbed into the surface of Petri dishes containing growth medium. If more than 30 min is required, plates should be dried for a longer period of time than the standard 1 h.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Endogenous fluorescence is strong.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Cells with lesions along the adenine biosynthetic pathway (<i>ade2</i> or <i>ade3</i> mutants) accumulate fluorescent pigments in their vacuoles and this can interfere with imaging. The problem is most pronounced if cells approach saturation. In general, the vacuolar signal becomes minimal if adenine is added to 200 μg/mL during cell growth.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Photobleaching.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Especially the "mRFP-HDEL" signal becomes photobleached during time-lapse imaging. If it is used, it therefore is generally inadvisable to collect z-stacks of images at less than 15 min intervals.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Foreign cells.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Crosses should be made between two clonal populations of cells. If any other cells are seen, the parental cultures should be recloned by "streaking to singles." This is achieved by using three successive toothpicks to streak the cells orthogonally onto complete medium in Petri dishes. The streaking results in an array of colonies since the first streak is horizontal at the top of the plate, the second streaks are vertical (intersecting the initial streak), and the third back-and-forth streaks are again horizontal, intersecting the second streaks. The plate is then incubated 1–2 days. When colonies appear, grow up several clones, test them individually, and verify their genotypes.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Zygotes are not in a single image plane.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>In recovering the mating mixtures from Petri dishes, a certain amount of agar is also recovered. This can make it difficult for the coverslip to be horizonal, In this event, we recommend using Agarose in place of agar for the cross itself. It is also possible to conduct crosses "under medium" by allowing mating mixtures in 0.1–0.5 mL to settle and form a sub-confluent monolayer at the bottom of a plastic well in a multi-well plate, which is then left immobile (<a href="#bib26">Zapanta Rinonos et al., 2014</a>).</p>
<p>Optical uniformity can also be improved by being careful not to apply an excessive number of cells to the Agarose pads used for visualizaton..</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The lack of synchrony makes certain types of quantitative analysis problematic. Quantitation can be achieved by counting the number of early zygotes that show equilibration of the labeled protein; however, we find that the results are already very clear-cut at a qualitative level when the labeled proteins do (or do not) redistribute to the two ends of the zygote. This is meaningful since in the [<i>kar1</i> x wt] control crosses - even when the incubations are prolonged - there is little or no redistribution (<a href="#fig3">Figure 3</a>). An alternative quantitative protocol could use time-lapse imaging; however, this requires that the zygotes tolerate imaging for 1–3 h. A complicating consideration is that progressive photobleaching will occur.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Assays that depend on zygote formation are inappropriate for certain yeast strains. For example, the remodeling of the cell surface that is intrinsic to establishment of <i>cis-trans</i> continuity during zygote formation requires control of cell wall integrity to ensure protection against osmotic stress. A variety of yeast mutants cannot exert this control and therefore are not suitable for efficient formation of normal zygotes (<a href="#bib20">Tartakoff, 2015</a>, <a href="#bib1">Aguilar et al., 2007</a>, <a href="#bib14">Philips and Herskowitz, 1997</a>). Indeed, even crosses of wildtype cells show occasional evidence of incomplete fusion or what appears to be osmotic rupture.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aguilar, P.S., Engel, A., and Walter, P. (2007). The plasma membrane proteins Prm1 and Fig1 ascertain fidelity of membrane fusion during yeast mating. Mol. Biol. Cell <i>18</i>, 547-556. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref1">View at publisher</a></p>
<p id="bib2">Amberg, D., Burke, D., and Strathern, D. (2005). Methods in Yeast Genetics: A Cold Spring Harbor Laboratory Course Manual, 2005 Edition (Cold Spring Harbor Press). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref2">View at publisher</a></p>
<p id="bib3">Boeke, J.D., Trueheart, J., Natsoulis, G., and Fink, G.R. (1987). 5-Fluoroorotic acid as a selective agent in yeast molecular genetics. Methods Enzymol. <i>154</i>, 164-175. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref3">View at publisher</a></p>
<p id="bib4">Dutcher, S.K. (1981). Internuclear transfer of genetic information in kar1-1/KAR1 heterokaryons in Saccharomyces cerevisiae. Mol. Cell. Biol. <i>1</i>, 245-253. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref4">View at publisher</a></p>
<p id="bib5">Elion, E.A. (2002). How to monitor nuclear shuttling. Methods Enzymol. <i>351</i>, 607-622. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref5">View at publisher</a></p>
<p id="bib6">Flach, J., Bossie, M., Vogel, J., Corbett, A., Jinks, T., Willins, D.A., and Silver, P.A. (1994). A yeast RNA-binding protein shuttles between the nucleus and the cytoplasm. Mol. Cell. Biol. <i>14</i>, 8399-8407. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref6">View at publisher</a></p>
<p id="bib7">Gammie, A.E. and Rose, M.D. (2002). Assays of cell and nuclear fusion. Methods Enzymol. <i>351</i>, 477-498. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref7">View at publisher</a></p>
<p id="bib8">Gao, X.D., Tachikawa, H., Sato, T., Jigami, Y., and Dean, N. (2005). Alg14 recruits Alg13 to the cytoplasmic face of the endoplasmic reticulum to form a novel bipartite UDP-N-acetylglucosamine transferase required for the second step of N-linked glycosylation. J. Biol. Chem. <i>280</i>, 36254-36262. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref8">View at publisher</a></p>
<p id="bib9">Gardner, J.M. and Jaspersen, S.L. (2014). Manipulating the yeast genome: deletion, mutation, and tagging by PCR. Methods Mol. Biol. <i>1205</i>, 45-78. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref9">View at publisher</a></p>
<p id="bib10">Janke, C., Magiera, M.M., Rathfelder, N., Taxis, C., Reber, S., Maekawa, H., Moreno-Borchart, A., Doenges, G., Schwob, E., Schiebel, E., and Knop, M. (2004). A versatile toolbox for PCR-based tagging of yeast genes: new fluorescent proteins, more markers and promoter substitution cassettes. Yeast <i>21</i>, 947-962. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref10">View at publisher</a></p>
<p id="bib11">Liu, Y., Liang, S., and Tartakoff, A.M. (1996). Heat shock disassembles the nucleolus and inhibits nuclear protein import and poly(A)+ RNA export. EMBO J. <i>15</i>, 6750-6757. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref11">View at publisher</a></p>
<p id="bib12">Longtine, M.S., Mckenzie, A., Demarini, D.J., Shah, N.G., Wach, A., Brachat, A., Philippsen, P., and Pringle, J.R. (1998). Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae. Yeast <i>14</i>, 953-961. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref12">View at publisher</a></p>
<p id="bib13">Phair, R.D. and Misteli, T. (2000). High mobility of proteins in the mammalian cell nucleus. Nature <i>404</i>, 604-609. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref13">View at publisher</a></p>
<p id="bib14">Philips, J. and Herskowitz, I. (1997). Osmotic balance regulates cell fusion during mating in Saccharomyces cerevisiae. J. Cell Biol. <i>138</i>, 961-974. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref14">View at publisher</a></p>
<p id="bib15">Prinz, W.A., Grzyb, L., Veenhuis, M., Kahana, J.A., Silver, P.A., and Rapoport, T.A. (2000). Mutants affecting the structure of the cortical endoplasmic reticulum in Saccharomyces cerevisiae. J. Cell Biol. <i>150</i>, 461-474. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref15">View at publisher</a></p>
<p id="bib16">Semenza, J.C., Hardwick, K.G., Dean, N., and Pelham, H.R. (1990). ERD2, a yeast gene required for the receptor-mediated retrieval of luminal ER proteins from the secretory pathway. Cell <i>61</i>, 1349-1357. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref16">View at publisher</a></p>
<p id="bib17">Sena, E., Welch, J., and Fogel, S. (1976). Nuclear and mitochondrial DNA replication during zygote formation and maturation in yeast. Science <i>194</i>, 433-435. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref17">View at publisher</a></p>
<p id="bib18">Sidebottom, A., Thornton, A., Tompson, L., Belur, J., Tilley, N., and Bowers, K. (2017). A systematic review of tagging as a method to reduce theft in retail environments. Crime Sci. <i>6</i>, 7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref18">View at publisher</a></p>
<p id="bib19">Stone, E.M., Heun, P., Laroche, T., Pillus, L., and Gasser, S.M. (2000). MAP kinase signaling induces nuclear reorganization in budding yeast. Curr Biol <i>10</i>, 373-382. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref19">View at publisher</a></p>
<p id="bib20">Tartakoff, A.M. (2015). Cell biology of yeast zygotes, from genesis to budding. Biochim. Biophys. Acta <i>1853</i>, 1702-1714. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref20">View at publisher</a></p>
<p id="bib21">Tartakoff, A.M., Chen, L., Raghavachari, S., Gitiforooz, D., Dhinakaran, A., Ni, C.L., Pasadyn, C., Mahabaeleshwar, G.H., Pasadyn, V., and Woolford, J.L. (2021). The nucleolus as a polarized coaxial cable in which the rDNA axis is surrounded by dynamic subunit-specific phases. Curr. Biol. <i>31</i>, 2507-2519.e4. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref21">View at publisher</a></p>
<p id="bib22">Tartakoff, A.M., Dulce, D., and Landis, E. (2018). Delayed Encounter of Parental Genomes Can Lead to Aneuploidy in Saccharomyces cerevisiae. Genetics <i>208</i>, 139-151. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref22">View at publisher</a></p>
<p id="bib23">Tartakoff, A.M. and Jaiswal, P. (2009). Nuclear fusion and genome encounter during yeast zygote formation. Mol. Biol. Cell <i>20</i>, 2932-2942. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref23">View at publisher</a></p>
<p id="bib24">Vallen, E.A., Hiller, M.A., Scherson, T.Y., and Rose, M.D. (1992). Separate domains of KAR1 mediate distinct functions in mitosis and nuclear fusion. J. Cell Biol. <i>117</i>, 1277-1287. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref24">View at publisher</a></p>
<p id="bib25">Ydenberg, C.A. and Rose, M.D. (2008). Yeast mating: a model system for studying cell and nuclear fusion. Methods Mol. Biol. <i>475</i>, 3-20. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref25">View at publisher</a></p>
<p id="bib26">Zapanta Rinonos, S., Rai, U., Vereb, S., Wolf, K., Yuen, E., Lin, C., and Tartakoff, A.M. (2014). Sequential logic of polarity determination during the haploid-to-diploid transition in Saccharomyces cerevisiae. Eukaryot. Cell <i>13</i>, 1393-1402. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref26">View at publisher</a></p>
<p id="bib27">Zapanta Rinonos, S., Saks, J., Toska, J., Ni, C.L., and Tartakoff, A.M. (2012). Flow cytometry-based purification of S. cerevisiae zygotes. J. Vis. Exp. <i>67</i>, e4197. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00443-3/sref27">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. A.M.Tartakoff, <a href="mailto:amt10@case.edu">amt10@case.edu</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate datasets<i>.</i></p>
<h3>Acknowledgments</h3>
<p>The funding sources R01GM089872 (T.A.M.) and the Visconsi family are acknowledged for support. This publication was made possible by the Clinical and Translational Science Collaborative of Cleveland, UL1TR002548 from the <a href="https://doi.org/10.13039/100006108">National Center for Advancing Translational Sciences</a> (NCATS) component of the National Institutes of Health, and NIH Roadmap for Medical Research. Its contents are solely the responsibility of the authors and do not necessarily represent the official views of the NIH. Collaborators Mayura Umapathy and Eric Yuen provided technical help. The author thanks N.Dean, W-K.Huh, F.Kizito, B.Luttge, D.MacDonald, W.Prinz, and M.Rose for materials and advice. The DeltaVision microscope is managed through the Molecular and Cellular Virology Core of the <a href="https://doi.org/10.13039/100006967">CFAR</a> P30CA43703.</p>
<h3>Author contributions</h3>
<p>A.M.T. conceived of the project and conducted the experiments.</p>
<h3>Declaration of interests</h3>
<p>The author declares no competing interests.</p>
</section>
</article>