<article>
<section>
<h1 id="header">Protocol for proteogenomic dissection of intronic splicing enhancer interactome for prediction of individualized cancer prognosis</h1>
<p><time datetime="2021-02-12">Published: February 12, 2021</time></p>
<p>Li Wang,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> John A. Wrobel,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Ling Xie,<sup><a href="#aff1">1</a></sup> and Xian Chen<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biochemistry &amp; Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</p>
<p id="aff2"><sup>2</sup>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:xianc@email.unc.edu">xianc@email.unc.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100338">10.1016/j.xpro.2021.100338</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Inter- or intra-patient tumor heterogeneity hinders the discovery of biomarkers for predicting individualized prognosis. Here, we present a protocol for an alternative splicing activity-based proteogenomic approach for identification of candidate prognostic markers in cancer cell lines and human breast cancer specimens. The pull-down of protein complexes with intronic splicing enhancer (ISE) probes is followed by tandem mass spectrometry (MS/MS) peptide sequencing. The proteogenomic analysis of data from these ISE-MS/MS assays identifies new prognostic markers that can be utilized to stratify patients with poor prognosis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib12">Wang et al. (2018)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for LC-MS/MS analysis of ISE interactomes from cancer cell lines or tissues</li>
<li>Protocol for proteogenomic identification of ISE interactors of prognostic significance</li>
<li>Protocol for protein extraction, ISE pull-downs, MS/MS, and proteogenomic analysis</li>
<li>Protocols applicable to cancer cell lines and clinical specimens</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/473-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Preparations of cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–1 h</div>
<ol>
<li>Prepare buffers and solutions in advance. Make sure there is a sufficient cell culture medium (500 mL) for cell growth. Here we use triple-negative breast cancer (TNBC) cell line MDA-MB-231 as an example, which can be maintained in DMEM high glucose medium supplemented with 10% fetal bovine serum and antibiotics (100 IU/mL penicillin and 100 μg/mL streptomycin)</li>
<li>Thaw the cryopreserved cells quickly by gently swirling the vial in a 37°C water bath and dilute them with warm medium in a plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Passage the cells at least twice before using them, since they need time to recover and resume regular cell cycle and cell metabolism.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>ESRP1</td>
<td>Thermo Fisher</td>
<td>PA5-25833 RRID: AB_2543333</td>
</tr>
<tr>
<td>ESRP2</td>
<td>Thermo Fisher</td>
<td>PA5-32143 RRID: AB_2549616</td>
</tr>
<tr>
<td>QKI</td>
<td>Bethyl</td>
<td>A300-183A RRID: AB_2173160</td>
</tr>
<tr>
<td>SNRPA1</td>
<td>Proteintech</td>
<td>17368-1-AP RRID: AB_2193724</td>
</tr>
<tr>
<td>SNRPB2</td>
<td>Proteintech</td>
<td>13512-1-AP RRID: AB_2302390</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human breast cancer tissues</td>
<td>UNC Lineberger Tissue Procurement Center</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM high glucose</td>
<td>Gibco</td>
<td>11965092</td>
</tr>
<tr>
<td>FBS</td>
<td>Gibco</td>
<td>26140079</td>
</tr>
<tr>
<td>Penicillin and streptomycin</td>
<td>Gibco</td>
<td>15140122</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td>Sigma</td>
<td>P8340</td>
</tr>
<tr>
<td>PMSF</td>
<td>Acros Organics</td>
<td>215740100</td>
</tr>
<tr>
<td>IGEPAL CA-630</td>
<td>Sigma</td>
<td>I3021</td>
</tr>
<tr>
<td>BCA assay kit</td>
<td>Thermo Fisher</td>
<td>23225</td>
</tr>
<tr>
<td>NeutrAvidin agarose</td>
<td>Thermo Fisher</td>
<td>29201</td>
</tr>
<tr>
<td>Dynabeads M-270 streptavidin</td>
<td>Thermo Fisher</td>
<td>65305</td>
</tr>
<tr>
<td>Sequence grade trypsin</td>
<td>Promega</td>
<td>V511C</td>
</tr>
<tr>
<td>Iodoacetamide</td>
<td>Sigma</td>
<td>I1149</td>
</tr>
<tr>
<td>DL-Dithiothreitol</td>
<td>Sigma</td>
<td>D0632</td>
</tr>
<tr>
<td>Solid phase extraction disk C18 (octadecyl)</td>
<td>Empore 3M</td>
<td>2215</td>
</tr>
<tr>
<td>MS grade trifluoroacetic acid</td>
<td>Thermo Fisher</td>
<td>85183</td>
</tr>
<tr>
<td>0.1% formic acid</td>
<td>Thermo Fisher</td>
<td>LS118-212</td>
</tr>
<tr>
<td>Acetonitrile and 0.1% formic acid</td>
<td>Thermo Fisher</td>
<td>LS120-1</td>
</tr>
<tr>
<td>HPLC grade methanol</td>
<td>Thermo Fisher</td>
<td>A452SK-4</td>
</tr>
<tr>
<td>HPLC grade water</td>
<td>Thermo Fisher</td>
<td>W5-4</td>
</tr>
<tr>
<td>ProtoGel 30%</td>
<td>National Diagnostics</td>
<td>EC-890</td>
</tr>
<tr>
<td>4× ProtoGel resolving buffer</td>
<td>National Diagnostics</td>
<td>EC-892</td>
</tr>
<tr>
<td>ProtoGel stacking buffer</td>
<td>National Diagnostics</td>
<td>EC-893</td>
</tr>
<tr>
<td>Ammonium persulfate</td>
<td>Sigma</td>
<td>A3678</td>
</tr>
<tr>
<td>N,N,N′,N′-Tetramethylethylenediamine (TEMED)</td>
<td>Sigma</td>
<td>T9281</td>
</tr>
<tr>
<td>PageRuler prestained protein ladder</td>
<td>Thermo Scientific</td>
<td>26616</td>
</tr>
<tr>
<td>ProSignal peroxide/dura solution</td>
<td>Genesee Scientific</td>
<td>GSC-929-D10</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>MS Raw data</td>
<td>ProteomeXchange Consortium</td>
<td>PXD008642</td>
</tr>
<tr>
<td>R scripts and data</td>
<td>Mendeley Data</td>
<td><a href="https://doi.org/10.17632/df32tkb89c.1">https://doi.org/10.17632/df32tkb89c.1</a></td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>MCF10A</td>
<td>ATCC</td>
<td>CRL-10317 <b>RRID:CVCL_0598</b>
</td>
</tr>
<tr>
<td>T47D</td>
<td>ATCC</td>
<td>HTB-133 <b>RRID:CVCL_0553</b>
</td>
</tr>
<tr>
<td>MCF7</td>
<td>ATCC</td>
<td>HTB-22 RRID: CVCL_0031</td>
</tr>
<tr>
<td>MDA-MB-231</td>
<td>ATCC</td>
<td>HTB-26 RRID: CVCL_0062</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Control 12 CAUAGCAGAUUGCAUCAUACAU</td>
<td>Synthesized by IDT</td>
<td>Standard desalted RNA oligo</td>
</tr>
<tr>
<td>ISE-kkkA (group A) GGGTTTCGGGTTTCGGGTTT</td>
<td>Synthesized by IDT</td>
<td>Standard desalted RNA oligo</td>
</tr>
<tr>
<td>ISE-kkkB (group B) GGTGGTCGTTGGTCTGGTTG</td>
<td>Synthesized by IDT</td>
<td>Standard desalted RNA oligo</td>
</tr>
<tr>
<td>ISE-kkkC (group C) TTTGGGCTTTGGGCTATTGG</td>
<td>Synthesized by IDT</td>
<td>Standard desalted RNA oligo</td>
</tr>
<tr>
<td>ISE-kkkD (group D) AGGGGGCGAGGGGCGGGGGT</td>
<td>Synthesized by IDT</td>
<td>Standard desalted RNA oligo</td>
</tr>
<tr>
<td>ISE-kkkE (group E) GGTATTCGGTATTCAGGTAT</td>
<td>Synthesized by IDT</td>
<td>Standard desalted RNA oligo</td>
</tr>
<tr>
<td>ISE-kkkF (group F) GTAACGCGGTAACCGGTAAG</td>
<td>Synthesized by IDT</td>
<td>Standard desalted RNA oligo</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>The R Project</td>
<td><a href="https://www.r-project.org/">https://www.r-project.org/</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Maxquant version 1.5.2.8</td>
<td>Maxquant</td>
<td>N/A</td>
</tr>
<tr>
<td>Perseus version 1.5.1.6</td>
<td>Perseus</td>
<td>N/A</td>
</tr>
<tr>
<td><b>Other</b></td>
<td>Eksigent</td>
<td>N/A</td>
</tr>
<tr>
<td>Ultra2D nanoLC system</td>
<td>Thermo Fisher</td>
<td>N/A</td>
</tr>
<tr>
<td>LTQ Orbitrap Velos mass spectrometer</td>
<td>SCIEX</td>
<td>5016752</td>
</tr>
<tr>
<td>NanoLC trap column</td>
<td>Labconco</td>
<td>7670000</td>
</tr>
<tr>
<td>FreeZone 2.5 L freeze dry system</td>
<td>Thermo Scientific</td>
<td>3451</td>
</tr>
<tr>
<td>Low-retention microcentrifuge tubes</td>
<td>Wheaton</td>
<td>35753</td>
</tr>
<tr>
<td>Dounce tissue grinders, 1 mL</td>
<td>Hamilton</td>
<td>1122-01</td>
</tr>
<tr>
<td>Plunger assembly N, RN, LT, C for model 1702</td>
<td>GL Sciences</td>
<td>5010-21514</td>
</tr>
<tr>
<td>Centrifuge adapter</td>
<td>Bio-Rad</td>
<td>12003154</td>
</tr>
<tr>
<td>ChemiDoc MP imaging system</td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Cell resuspension buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>50 mM</td>
<td>1 M pH 8.0</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>5 M</td>
<td>1.55 mL</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td>1×</td>
<td>100×</td>
<td>500 μL add before use</td>
</tr>
<tr>
<td>PMSF</td>
<td>0.5 mM</td>
<td>100 mM</td>
<td>250 μL add before use</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Add to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This buffer can be stored in 4°C for up to 1 year. Make new buffer if precipitation is observed.</div>
<table id="undtbl2">
<caption>2× Cell lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>50 mM</td>
<td>1 M pH 8.0</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>5 M</td>
<td>1.55 mL</td>
</tr>
<tr>
<td>CA630</td>
<td>1%</td>
<td>100%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>NaN<sub>3</sub>
</td>
<td>15 mM</td>
<td>1 M</td>
<td>0.76 mL</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td>1×</td>
<td>100×</td>
<td>500 μL add before use</td>
</tr>
<tr>
<td>PMSF</td>
<td>1 mM</td>
<td>100 mM</td>
<td>500 μL add before use</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Add to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This buffer can be stored in 4°C for up to 1 year. Make new buffer if precipitation is observed. Mixing one part (v/v) of cell resuspension buffer and one part (v/v) of 2× lysis buffer will make 1× cell lysis buffer for the protein extraction from clinical tissue in the following step 9.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Sodium azide is poisonous, exposure to solid NaN<sub>3</sub> can be fatal. Make sure to use personal protective equipment when making a stock solution.</div>
<table id="undtbl3">
<caption>Digestion buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Volume/weight</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>50 mM</td>
<td>1 M pH 8.0</td>
<td>50 μL</td>
</tr>
<tr>
<td>Urea</td>
<td>2 M</td>
<td> </td>
<td>0.12 g</td>
</tr>
<tr>
<td>DTT</td>
<td>1 mM</td>
<td>1 M</td>
<td>1 μL add before use</td>
</tr>
<tr>
<td>Trypsin</td>
<td>5 μg/mL</td>
<td>1 μg/μL</td>
<td>5 μL add before use</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Add to 1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare digestion buffer as needed. To prepare 1 mL buffer, dissolve 0.12 g urea in 700 μL ddH<sub>2</sub>O, add Tris-HCl, and finally top up to 1 mL with ddH<sub>2</sub>O.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Urea solutions should always be freshly prepared and used, as urea solution may develop a significant concentration of reactive cyanate ions upon storage. 1 M DTT stock solution is harmful, avoid contact with skin and eyes, and wear protective goggles and gloves.</div>
<table id="undtbl4">
<caption>Elution buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>50 mM</td>
<td>1 M pH 8.0</td>
<td>50 μL</td>
</tr>
<tr>
<td>Urea</td>
<td>2 M</td>
<td> </td>
<td>0.12 g</td>
</tr>
<tr>
<td>Iodoacetamide</td>
<td>5 mM</td>
<td>550 mM</td>
<td>10 μL add before use</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Add to 1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Iodoacetamide is unstable and light sensitive, solution should always be kept in the dark.</div>
<table id="undtbl5">
<caption>Buffer B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetonitrile</td>
<td>50% (v/v)</td>
<td>100%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Trifluoroacetic acid</td>
<td>0.1%</td>
<td>100%</td>
<td>10 μL</td>
</tr>
<tr>
<td>HPLC grade water</td>
<td> </td>
<td> </td>
<td>Add to 10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: Trifluoroacetic acid is highly corrosive to respiratory system, use a fume hood and gloves.</div>
<table id="undtbl6">
<caption>Buffer A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Trifluoroacetic acid</td>
<td>0.1%</td>
<td>100%</td>
<td>10 μL</td>
</tr>
<tr>
<td>HPLC grade water</td>
<td> </td>
<td> </td>
<td>Add to 10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution<i>:</i> Trifluoroacetic acid is highly corrosive to respiratory system, use a fume hood and gloves.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Protein extraction from breast cancer cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This section details how to extract proteins from cancer cell lines. Procedures for cell lines and clinical samples can be performed at same time or different times.</p>
<ol>
<li>Harvest and pellet cells (MCF10A, T47D, MCF7, MDA-MB0231) when cell confluency reaches about 90%.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> One 10 cm plate of cells or 10 million cells are sufficient for one pull-down experiment. <b>CRITICAL:</b> Do not let the cells overgrow or reach 100% confluency, some cell lines change morphology and other characteristics if overgrown.</div>
<div class="critical">
<span class="critical-title">Critical:</span> All the procedures should be performed on ice.</div>
<ol start="2">
<li>Remove the cell culture medium and wash the cells with pre-chilled cold PBS buffer once, then add another 1 mL cold PBS, gently harvest cells form the plate using a cell scraper and transfer the cell suspension to a 1.5 mL centrifuge tube.</li>
<li>Pellet the cells by centrifugation at 1,800 rpm for 5 min at 4°C. Discard the supernatant carefully without disturbing the cell pellet.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The cell pellets can be stored at this point at −80°C up to 1 year.</div>
<ol start="4">
<li>Resuspend the cells in 0.75 mL resuspension buffer.</li>
<li>Add 0.75 mL 2× lysis buffer and mix thoroughly in the same 1.5 mL centrifuge tube. 0.5% detergent CA630 will dissolve cell membranes and release cytoplasmic protein.</li>
<li>Centrifuge at 12,000 × <i>g</i> for 20 min in 4°C.</li>
<li>Transfer the supernatant that contains soluble protein to a new centrifuge tube and discard the pellet.</li>
<li>Determine protein concentration by BCA assay.</li>
</ol>
<h3 id="sec3.2">Protein extraction from human breast cancer tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This section details how to extract proteins from clinical tissue. These procedures can be performed at same time or different times.</p>
<ol start="9">
<li>Slightly thaw the clinical tissue specimen on ice, and cut out ~0.2 g of the tissue using a scalpel blade or surgical scissor for one experiment.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This method is only valid for breast cancer tissue, other types of tissues may use alternative method to extract total protein. A variety of methods are applicable, including sonication, French pressure cell, grinding, glass bead vortexing, detergent lysis, or frozen tissue crushing (<a href="#bib8">Scopes, 2013</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> All steps should be performed in cold room and samples should sit on ice.</div>
<div class="note">
<span class="note-title">Note:</span> Clinical tissue specimen was stored in −80°C freezer, it is very hard to cut the tissue when it is frozen. Let it thaw on ice for about 5 min or until it gets softer.</div>
<ol start="10">
<li>Cut the tissue into small pieces using surgical scissor on a clean plate.</li>
<li>Transfer the tissue into a pre-chilled 1 mL glass cylinder of Dounce homogenizer. Add 1 mL 1× lysis buffer.</li>
<li>Homogenization is performed by 10–15 passes of tight pestle up and down.</li>
<li>Transfer the sample solution to a new 1.5 mL centrifuge tube, and centrifuge at 12,000 × <i>g</i> for 20 min in 4°C.</li>
<li>Keep the supernatant and repeat the centrifugation, the supernatant contains soluble protein.</li>
<li>Protein concentration is determined by BCA assay.</li>
</ol>
<h3 id="sec3.3">Intronic splicing enhancer (ISE) pull-down</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section describes the method of the intronic splicing enhancer (ISE) pull-down. Please see <a href="#fig1">Figure 1</a> for a step-by-step protocol.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/473-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Step-by-step protocol of ISE pull-down</div>
</figcaption>
</figure>
<ol start="16">
<li>After BCA protein concentration measurement, 1 mg of protein solution (0.2–0.5 mL lysate contain 1 mg total protein) extracted from cell lines or clinical tissue will be used for each pull-down with non-ISE control probe or ISE probe, so at least 2 mg of protein are required for one set of experiment.</li>
<li>Add 10 μL biotinylated non-ISE control RNA probe or ISE probes (0.75 μM) to protein solution and incubate at 4°C for 2 h or overnight. A short incubation time (e.g., 2 h) can significantly reduce no-specific binding, and incubation overnight will get maximal recovery of target protein and low abundant interactors. With a negative control such as non-ISE control RNA to rule out non-specific binding proteins , an overnight incubation is recommended for this pull-down experiment.</li>
<li>Add 50 μL NeutrAvidin agarose and incubate for an additional 2 h at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Wash the agarose twice with PBS and twice with cell lysis buffer before adding to the protein solution.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Be careful when removing wash buffer, try not to lose bead and use pipet tips with a narrow end if possible.</div>
<ol start="19">
<li>Gently spin down the agarose at 500 × <i>g</i> for 1 min at 4°C, remove the supernatant. The ISE RNA-protein complex is captured by the agarose.</li>
<li>Wash the agarose 3 times with 1 mL lysis buffer to remove non-specific binding proteins.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is very easy to lose agarose beads during the wash steps, use extra caution when removing the wash buffer.</div>
<ol start="21">
<li>Wash the agarose additional 5 times with 1 mL resuspension buffer to minimize the amount of residual detergent.</li>
<li>Transfer 10% of the beads into a new sample tube and elute this 10% of beads with 50 μL SDS sample buffer by boiling at 95°C for 5 min for immunoblotting and the rest of the 90% of beads will be processed for MS analysis in the next step.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Detergent needs to be removed prior to MS analysis, because it contaminates the HPLC column and MS instrument and also suppresses the signal of peptide. Resuspension buffer does not contain any detergent, can solely be used as wash buffer without PMSF and protease inhibitor cocktail.</div>
<div class="critical">
<span class="critical-title">Critical:</span> In the final wash, after adding wash buffer, transfer buffer and agarose beads to a new, low-retention tube, then spin down the beads again and remove the wash buffer completely by using a long gel loading tip which has an outer diameter&lt;0.3 mm to prevent loss of beads.</div>
<ol start="23">
<li>Perform western blotting to check the known interactors of ISE probes in the pull-down product before going for MS analysis.</li>
</ol>
<p>e.g., Known interactors including ESRP1 (~7 0kDa), ESRP2 (~90 kDa), TJP3 (~130 kDa), QKI (~40 kDa), LYAR (~45 kDa), SNRPB2 (~26 kDa), SRPK1 (~95k Da), SNRPA1 (~28 kDa), could be separated well on a 10% SDS-PAGE gel. For further details, please refer to (<a href="#bib12">Wang et al., 2018</a>)</p>
<h3 id="sec3.4">Sample preparation for mass spectrometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This protocol describes trypsin digestion on agarose beads and peptide desalting for LC-MS/MS. Please see <a href="#fig2">Figure 2</a> for the step-by-step protocol.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/473-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Flowchart of sample preparation for on-beads trypsin digestion and LC-MS/MS analysis</div>
</figcaption>
</figure>
<ol start="24">
<li>On-beads tryptic digestion for LC-MS/MS. Add 125 μL trypsin digestion buffer containing 5 ng/μL trypsin and 1 mM DTT to the 90% of beads from step 22 and incubate with beads at room temperature on a mixer for 30 min. Trypsin will quickly digest the large protein complex into peptides.</li>
<li>Centrifuge at 500 × <i>g</i> for 1 min at room temperature to bring down the beads, transfer the supernatant that contains pre-digested protein or peptides to a new, low-retention tube.</li>
<li>One additional elution is applied by adding 250 μL elution buffer containing 5 mM Iodoacetamide (IAA) and incubated on a mixer for 10 min at room temperature in dark.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> IAA is light sensitive, to keep the reaction in dark, wrap the rack or cover of the mixer with aluminum foil.</div>
<ol start="27">
<li>Spin down the beads, then transfer the supernatant/elute to a new tube.</li>
<li>Repeat the elution with IAA one more time and combine all supernatants/elutes in one tube.</li>
<li>Proteins were digested overnight at room temperature.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples at this point can be stored at −80°C freezer for up to one year.</div>
<ol start="30">
<li>Peptide cleanup and desalting with C18 StageTips, see <a href="#fig3">Figure 3</a> for the setups.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/473-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. The preparation of StageTips</div>
</figcaption>
</figure>
</li>
<li>Production a single layer C18 StageTips with 200 μL low-retention tips as described in previously published protocol (<a href="#bib7">Rappsilber et al., 2007</a>). In practice, StageTips are made by placing a small portion of C18 materials in a clean 200-mL pipette tip. The C18 material is stamped out using a blunt-ended syringe needle and then released inside a 200-mL pipette tip using plunger. Adding an adapter and collection tube will automate all the steps on a bench top centrifuge (see <a href="#fig3">Figure 3</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> One layer of C18 materials cut by the needle has a capacity up to 30 μg of peptide, use multiple layers if more material is loaded.</div>
<ol start="32">
<li>Acidify (pH~3) the peptides in the elute with 3 μL (0.5%) trifluoroacetic acid (TFA). pH measurement is not necessary here.</li>
<li>Mix them well and centrifuge at 12000 × <i>g</i> for 5 min at 4°C to pellet any precipitation after the pH change. Keep the supernatant.</li>
<li>Conditioning and equilibration of the spin StageTips. Wet the StageTip once with 50 μL methanol and centrifuge at 5,000 rpm for 1 min to let it pass through the C18 material. Discard the liquid in the collection tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Caution<i>:</i> Methanol is toxic and flammable.</div>
<ol start="35">
<li>Wash the StageTip once with 50 μL buffer B and spin at 5,000 rpm for 1 min and discard the liquid.</li>
<li>Equilibrate the StageTip twice with 50 μL buffer A and discard the liquid. The StageTips is now conditioned and ready for use.</li>
<li>To perform peptide binding, replace a new collection tube and load the supernatant from step 33 on to the StageTip and centrifuge at 5,000 rpm for 2~3 min. Retain elute as flow through fraction.</li>
<li>Replace a new collection tube. Wash the StageTip with 200 μL buffer A and spin at 5,000 rpm for 3 min and discard the liquid.</li>
<li>Repeat step 38 twice and discard the liquid.</li>
<li>Replace a new collection tube, load 100 μL buffer B on to the StageTip to elute the peptide by spinning at 5,000 rpm for 2–3 min at room temperature.</li>
<li>No need to replace collection tube, repeat the elution one more time and combine the peptides from all elution.</li>
<li>Concentrate and dry the eluate of a reversed-phase StageTip using a freeze dryer or SpeedVac. It takes approximately 1 h to finish the process.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dried peptides can be stored at −80°C indefinitely.</div>
<div class="note">
<span class="note-title">Note:</span> As in step 37, the flow through fraction can be retained just in case. Dried peptide at the bottom of the tube may not be visible depends on the amount of the peptide content.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> C18 ZipTip Pipette Tips can replace the use of C18 Stage Tips.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Always use low-retention tubes and tips without additives or coatings for maximum recovery of samples for all sample preparations for mass spectrometry.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not keep the organic solvent in the tube for too long, which will lead to plastic polymer contamination to the samples. Use glass tubes or vials instead for long term storage.</div>
<h3 id="sec3.5">Mass spectrometry (LC-MS/MS) analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This step describes LC-MS/MS sequencing of tryptic peptides from ISE pull-downs</p>
<ol start="43">
<li>Dissolve the dried peptide samples in 30 μL 0.1% formic acid, vortex for 1 min and centrifuge at 12,000 × <i>g</i> for 5 min at 4°C.</li>
<li>Transfer 25 μL the peptide sample to a glass sample vial.</li>
<li>Inject 4 μL sample into the C18 column and peptides were first loaded on to a 2 mm × 0.5 mm reverse-phase (RP) C18 trap column at a flow rate of 1 μL/min, then eluted and fractionated on a 25 cm C18 RP column (360 μm × 75 μm × 3 μm) with a gradient at a constant flow rate of 250 nL/min over 180 min linear gradient showing below.</li>
</ol>
<table id="undtbl7">
<caption>LC-MS/MS Setup HPLC reverse-phase chromatography gradient</caption>
<thead>
<tr>
<th>Time(min)</th>
<th>% A</th>
<th>% B</th>
<th>Flow rate (nL/min)</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>95</td>
<td>5</td>
<td>250</td>
</tr>
<tr>
<td>150</td>
<td>60</td>
<td>40</td>
<td>250</td>
</tr>
<tr>
<td>160</td>
<td>20</td>
<td>80</td>
<td>250</td>
</tr>
<tr>
<td>170</td>
<td>20</td>
<td>80</td>
<td>250</td>
</tr>
<tr>
<td>175</td>
<td>85</td>
<td>15</td>
<td>250</td>
</tr>
<tr>
<td>180</td>
<td>85</td>
<td>15</td>
<td>250</td>
</tr>
</tbody>
</table>
<ol start="46">
<li>The Velos LTQ Orbitrap was operated in the positive-ion mode with a data-dependent automatic switch between survey Full MS scan (m/z 300–1,800) (externally calibrated to a mass accuracy of &lt;5 ppm and a resolution of 60,000 at m/z 400) and CID MS/MS acquisition of the top 15 most intense ions. Detailed MS setting is provided in the following table.</li>
</ol>
<table id="undtbl8">
<caption>LTQ Obitrap Velos mass spectrometer parameters</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>Polarity</td>
<td>Positive</td>
</tr>
<tr>
<td>MS analyzer</td>
<td>Orbitrap</td>
</tr>
<tr>
<td>Data acquisition mode</td>
<td>DDA</td>
</tr>
<tr>
<td><b>Full MS</b></td>
<td>Profile</td>
</tr>
<tr>
<td>Orbitrap resolution</td>
<td>60,000 at 400 m/z</td>
</tr>
<tr>
<td>MS scan range (m/z)</td>
<td>300–1,800</td>
</tr>
<tr>
<td>AGC target</td>
<td>1 × 10<sup>6</sup>
</td>
</tr>
<tr>
<td><b>MS/MS</b></td>
<td>Centroid</td>
</tr>
<tr>
<td>MS analyzer</td>
<td>Ion trap</td>
</tr>
<tr>
<td>Mass range</td>
<td>Normal</td>
</tr>
<tr>
<td>Scan rate</td>
<td>Normal</td>
</tr>
<tr>
<td>AGC target</td>
<td>1 × 10<sup>4</sup>
</td>
</tr>
<tr>
<td>Number of dependent scans</td>
<td>Top 15</td>
</tr>
<tr>
<td>Microscans</td>
<td>1</td>
</tr>
<tr>
<td>Activation type</td>
<td>CID</td>
</tr>
<tr>
<td>Normalized collision energy</td>
<td>35</td>
</tr>
<tr>
<td>Charge exclusion</td>
<td>+1</td>
</tr>
<tr>
<td>Dynamic exclusion</td>
<td>60 s</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Multiple RNA cis-acting splicing regulatory elements, such as intronic splicing enhancers (ISEs), that interact with trans-acting splicing proteins (transfactors)(<a href="#bib13">Wang et al., 2012</a>) regulate the diversity and specificity of aberrant alternative splicing that promotes tumor growth, survival, and metastasis. The ISE library contains six biotinylated RNA oligomers corresponding to known ISEs, and a scrambled RNA sequence was served as a non-specific control. A successful ISE pull-down should be able to capture a lot of known ISE RNA interacting proteins (please refer to (<a href="#bib12">Wang et al., 2018</a>) for further details) compared with non-ISE RNA; Such as luminal subtype specific interactors/transfactors ESRP1, ESPR2,TJP3 and basal subtype specific interactors QKI, LYAR, SNRPB2, SRPK1, SNRPA1. An immune blotting experiment is recommended to check those known factors in the pull-down product before doing further MS analysis.</p>
<p>After finishing label-free quantification analysis of MS data, we should be able to identify cluster of transfactors that are specific to each subtypes from both homogeneous tumor cell lines and heterogeneous clinical tissue, such as the luminal and basal subtype specific transfactors showed in the reference (<a href="#bib12">Wang et al., 2018</a>). The heterogeneous tumor tissue such as breast cancer tissue, are a mixed population of cells from different subtypes, and signal of the tissue proteomic data are often averaged. A cross-referencing scheme involving multiple breast cancer or tumor cell lines and clinical tissues of similar subtypes is designed to help sort out subtype specific biomarkers.</p>
<p>In the proteogenomic analysis, a majority (more than 50%) of the ISE transfactors should show a PAM50-Subtypic Interaction-Correlated Expression Pattern (iCEP).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">MS data analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol>
<li>Mass spectral processing and peptide identification were performed on the Andromeda search engine in MaxQuant software (Version 1.5.2.8)(<a href="#bib10">Tyanova et al., 2016a</a>) against a human UniProt database. All searches were conducted with a defined modification of cysteine carbamidomethylation, with methionine oxidation and protein amino-terminal acetylation as dynamic modifications. Peptides were confidently identified using a target-decoy approach with a peptide false discovery rate (FDR) of 1% and a protein FDR of 5%. A minimum peptide length of 7 amino acids was required, maximally two missed cleavages were allowed, initial mass deviation for precursor ion was up to 7 ppm, and the maximum allowed mass deviation for fragment ions was 0.5 Da. A label-free quantification was also performed, a match between runs option was enabled and time window at 0.7 min. Data processing and statistical analysis were performed on Perseus (Version 1.5.1.6)(<a href="#bib11">Tyanova et al., 2016b</a>). Protein quantitation was performed on three biological replicates, and a two-sample t-test statistics was performed with a p value of 5% to report statistically significant expression fold-changes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use latest version of MaxQuant and Perseus software for data analysis.</div>
<h3 id="sec5.2">TCGA analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>The R software environment for statistical computing was used for the data analysis.</p>
<p>The sample R scripts and input data files are available at Mendeley Data.</p>
<ol start="2">
<li>Download and unpack brca_tcga_pub.tar.gz file for Breast Invasive Carcinoma (TCGA, Nature 2012) dataset (<a href="#bib5">The Cancer Genome Atlas Network, 2012</a>) from the cBioPortal (<a href="https://www.cbioportal.org/">https://www.cbioportal.org/</a>) (<a href="#bib1">Cerami et al., 2012</a>, <a href="#bib4">Gao et al., 2013</a>). The following files will be used in the overall TCGA analysis:</li>
</ol>
<p>data_clinical_patient.txt</p>
<p>data_clinical_sample.txt</p>
<p>cases_complete.txt (in the case_lists subfolder)</p>
<p>data_expression_merged_median_Zscores.txt</p>
<p>data_mRNA_median_Zscores.txt</p>
<p>data_CNA.txt</p>
<p>Other breast cancer studies from the cBioPortal can be consider for this analysis. For instance, we performed a similar analysis (<a href="#bib14">Wrobel et al., 2019</a>) to the one described here using the Breast Invasive Carcinoma (TCGA, Cell 2015)(<a href="#bib2">Ciriello et al., 2015</a>) and Breast Cancer (METABRIC, Nature 2012 &amp; Nat Commun 2016)(, <a href="#bib6">Pereira et al., 2016</a>, <a href="#bib3">Curtis et al., 2012</a>) from the cBioPortal.</p>
<ol start="3">
<li>Also construct a csv file of a gene list of ISE transfactors with the gene name in column 1 and the breast cancer subtype determined from the analysis of the experimental proteomic data in column 2. For our analysis, the ISE_transfactor_gene_list.csv file contains 34 Basal-like ISE transfactor genes and 20 luminal ISE transfactor genes.</li>
</ol>
<h3 id="sec5.3">Interaction-correlated expression pattern (iCEP)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>Identify ISE transfactors showing a PAM50-Subtypic Interaction-Correlated Expression Pattern (iCEP), where basal ISE transfactor genes have significantly higher mRNA expression in Basal-like TCGA samples compared to luminal TCGA samples and luminal ISE transfactor genes have significantly higher mRNA expression in luminal TCGA samples compared to Basal-like TCGA samples.</p>
<p>This analysis will use these input files:</p>
<p>ISE_transfactor_gene_list.csv</p>
<p>data_clinical_sample.txt</p>
<p>cases_complete.txt</p>
<p>data_expression_merged_median_Zscores.txt</p>
<p>data_mRNA_median_Zscores.txt</p>
<p>The iCEP_mRNA.R file provides a sample R script for this analysis.</p>
<ol start="4">
<li>The input files are loaded into R using the read.table() function. The TCGA clinical data (found in data_clinical_sample.txt) is filtered for sample IDs in the cases_complete.txt file (463 samples with mutation, CNA, and expression data). For our study we filtered for complete case samples, although this is not necessary. The data_expression_merged_median_Zscores.txt file contains mRNA expression Z-scores compared to diploid tumors (diploid for each gene) for the TGCA samples used in this analysis. However, this file contains the Entrez Gene Id, but does not list the HUGO Gene Symbol. The data_mRNA_median_Zscores.txt file contains both the HUGO Gene Symbol and Entrez Gene Id, and is used to provide the HUGO Gene Symbol based on matching the Entrez Gene Id in the data_expression_merged_median_Zscores.</li>
<li>A Mann-Whitney-Wilcoxon test was performed to judge differences in expression levels between Basal-like and luminal (both Luminal A and Luminal B) TCGA samples based on their PAM50 classification in the TCGA clinical data file for each of the ISE transfactor genes. The p value was calculated using the R wilcox.test() function with the “alternative” parameter set to "two.sided." For the test, we included all genes in the TCGA data set (n = ~17,000) to obtain an adjusted p value for multiple comparisons for the ISE interactor genes. The adjusted p values were calculated by submitting all the p values determined for each gene from the individual test to the p.adjust() function in R using the “fdr” method. We considered an adjusted p value &lt;0.05 to be significant. Basal ISE transfactors with a significant adjusted p value and a mean expression of the Basal-like TCGA samples greater than the mean expression of the luminal TCGA samples show an Interaction-Correlated Expression Pattern (For further details, please refer to (<a href="#bib12">Wang et al., 2018</a>)). Luminal ISE transfactors with a significant adjusted p value and a mean expression of the luminal TCGA samples greater than the mean expression of the Basal-like TCGA samples show an Interaction-Correlated Expression Pattern.</li>
</ol>
<p>The R script produces an output csv file with the results.</p>
<h3 id="sec5.4">CNV correlation with mRNA expression of individual ISE transfactor genes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>This analysis will use these input files:</p>
<p>ISE_transfactor_gene_list.csv</p>
<p>data_clinical_sample.txt</p>
<p>cases_complete.txt</p>
<p>data_expression_merged_median_Zscores.txt</p>
<p>data_mRNA_median_Zscores.txt</p>
<p>data_CNA.txt</p>
<p>The mRNA_gistic.R file provides a sample R script for this analysis.</p>
<ol start="5">
<li>The input files are loaded into R using the read.table() function. The TCGA clinical data (found in data_clinical_sample.txt) is filtered for sample IDs in the cases_complete.txt file (463 samples with mutation, CNA, and expression data). For our study we filtered for complete case samples, although this is not necessary. The data_expression_merged_median_Zscores.txt file contains mRNA expression Z-scores compared to diploid tumors (diploid for each gene) for the TGCA samples. However, this file contains the Entrez Gene Id, but does not list the HUGO Gene Symbol. The data_mRNA_median_Zscores.txt file contains both the HUGO Gene Symbol and Entrez Gene Id, and is used to provide the HUGO Gene Symbol based on matching the Entrez Gene Id in the data_expression_merged_median_Zscores. The data_CNA.txt file contains gene level putative copy number from GISTIC 2.0. Values: −2 = homozygous deletion; −1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</li>
<li>A Mann-Whitney Wilcoxon test was performed to judge differences in mRNA expression levels between copy number loss (GISTIC value of −1 or −2) and diploid (GISTIC value of 0) and separately between copy number gain (GISTIC value of 1 or 2) and diploid (GISTIC value of 0). The p value was calculated using the R wilcox.test() function with the “alternative” parameter set to "two.sided." For the test, we included all genes in the TCGA data set (n = ~17,000) to obtain an adjusted p value for multiple comparisons for the ISE interactor genes. The adjusted p values were calculated by submitting all the p values determined for each gene from the individual test to the p.adjust function in R using the “fdr” method. We considered an adjusted p value &lt;0.05 to be significant. Cases comparing loss to diploid with a significant adjusted p value and the average mRNA expression is lower for loss compared with diploid were considered to show a correlation between copy number and mRNA expression. Cases comparing gain to diploid with a significant adjusted p value and the average mRNA expression is greater for gain compared with diploid were considered to show a correlation between copy number and mRNA expression (For further details, please refer to (<a href="#bib12">Wang et al., 2018</a>)).This analysis can be performed on all TCGA samples or filtered for Basal-like or luminal samples based on the sample PAM50 classification in the TCGA clinical data.</li>
</ol>
<p>The R script produces an output csv file with the results.</p>
<h3 id="sec5.5">Survival analysis for mRNA over-expression</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>This analysis will use these input files:</p>
<p>ISE_transfactor_gene_list.csv</p>
<p>data_clinical_patient.txt</p>
<p>data_clinical_sample.txt</p>
<p>cases_complete.txt</p>
<p>data_expression_merged_median_Zscores.txt</p>
<p>data_mRNA_median_Zscores.txt</p>
<p>The KM_mRNA.R file provides a sample R script for this analysis.</p>
<ol start="7">
<li>The input files are loaded into R using the read.table() function. The data_clinical_patient (containing the OS_MONTHS and OS_STATUS data for the Kaplan-Meier survival analysis) is merged with the data_clinical_sample (containing the PAM50 classification). The merged TCGA clinical data is filtered for sample IDs in the cases_complete.txt file (463 samples with mutation, CNA, and expression data). For our study we filtered for complete case samples, although this is not necessary. The data_expression_merged_median_Zscores.txt file contains mRNA expression Z-scores compared to diploid tumors (diploid for each gene) for the TGCA samples. However, this file contains the Entrez Gene Id, but does not list the HUGO Gene Symbol. The data_mRNA_median_Zscores.txt file contains both the HUGO Gene Symbol and Entrez Gene Id, and is used to provide the HUGO Gene Symbol based on matching the Entrez Gene Id in the data_expression_merged_median_Zscores.</li>
<li>Kaplan-Meier (KM) statistical analysis for differences of overall survival based on mRNA expression of ISE transfactor genes among TCGA BRCA samples was performed using the “survival” R package(<a href="#bib9">Therneau and Grambsch, 2000</a>). Kaplan-Meier estimator and log rank tests were performed using the survival functions Surv, survfit, and survdiff. Cox proportional hazard survival analysis was performed using the survival function coxph. We used KM analysis to assess a correlation between patient-specific mRNA expression alterations of ISE transfactor genes and poor clinical outcome. In the KM analysis, patients with high mRNA expression (z-score &gt; 1) (the altered group) were compared to patients with lower mRNA expression (z-score &lt;= 1) (the unaltered group) for a specific ISE transfactor. We defined a poor clinical outcome for cases where the log rank p value is less than 0.05 and the hazard ratio (HR) is above 1. When we performed KM analysis for a specific PAM50 subtype, we defined the altered group as samples for the specific PAM50 subtype with high mRNA expression (z-score &gt; 1) and the unaltered group as samples for the specific PAM50 subtype with lower mRNA expression (z-score &lt;= 1) plus all the samples not belonging to the specific PAM50 subtype with or without high mRNA expression. For an example KM plot, please refer to (<a href="#bib12">Wang et al., 2018</a>).</li>
</ol>
<p>The R script produces an output csv file with the results.</p>
<h3 id="sec5.6">Survival analysis for copy-number alterations</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>This analysis will use these input files:</p>
<p>ISE_transfactor_gene_list.csv</p>
<p>data_clinical_patient.txt</p>
<p>data_clinical_sample.txt</p>
<p>cases_complete.txt</p>
<p>data_CNA.txt</p>
<p>The KM_gistic.R file provides a sample R script for this analysis.</p>
<ol start="9">
<li>The input files are loaded into R using the read.table() function. The data_clinical_patient (containing the OS_MONTHS and OS_STATUS data for the Kaplan-Meier survival analysis) is merged with the data_clinical_sample (containing the PAM50 classification). The merged TCGA clinical data is filtered for sample IDs in the cases_complete.txt file (463 samples with mutation, CNA, and expression data). For our study we filtered for complete case samples, although this is not necessary. The data_CNA.txt file contains gene level putative copy number from GISTIC 2.0. Values: −2 = homozygous deletion; −1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</li>
</ol>
<ol start="10">
<li>Kaplan-Meier statistical analysis for differences of overall survival based on copy-number GISTIC values of ISE transfactor genes among TCGA BRCA samples was performed using the “survival” R package(<a href="#bib9">Therneau and Grambsch, 2000</a>). Kaplan-Meier estimator and log rank tests were performed using the survival functions Surv, survfit, and survdiff. Cox proportional hazard survival analysis was performed using the survival function coxph. We used KM analysis to assess a correlation between patient-specific copy-number alterations of ISE transfactor genes and poor clinical outcome.<br> KM analysis can be performed separately where the altered group is defined as:
<ol type="a">
<li>patient samples that have a copy number gain (GISTIC values of 1 or 2)</li>
<li>patient samples that have a copy number loss (GISTIC values of −1 or −2)</li>
<li>patient samples that are diploid (GISTIC value = 0).</li>
</ol>
</li>
</ol>
<p>The unaltered group will be the remaining patient samples not in the altered group.</p>
<ol start="11">
<li>We defined a poor clinical outcome for the altered group in cases where the log rank p value is less than 0.05 and the hazard ratio (HR) is above 1. For example KM plots, please refer to (<a href="#bib12">Wang et al., 2018</a>).</li>
</ol>
<p>The R script produces an output csv file with the results.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Label-free quantification method is cheap and cost effective, but it relies heavily on the sample preparation and performance of HPLC and mass spectrometry instrument, the systematic errors in those processes would affect the confidence and accuracy of the end results. We would recommend either do more biological replicates, minimum of 3 are required, or using different quantification methods, such as SILAC or TMT.</p>
<p>The sample size of the clinical specimen is relatively small for this study, a more optimized size will assures an adequate power to identify statistical significance. A power analysis can be used to estimate the minimum sample size required for an experiment.</p>
<p>The clinical follow-up time is relatively short in the TCGA dataset we used for the Kaplan-Meier survival analysis and this may impact the prognostic value. We found that it might be useful to perform Kaplan-Meier survival analysis using additional clinical datasets.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Agarose beads lost during the wash steps.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Magnetic NeutrAvidin or StrepAvidin beads is also a good choice for pull-down; The easy and efficient collection of beads in magnetic fields allows for easy rinsing and removal of excess reagents. A magnet will be required for Magnetic bead collection.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Clinical tumor specimen is too hard to be cut into small pieces and extract protein from it.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Use alternative method, such as grinding of the frozen tissue in a mortar cooled on liquid nitrogen, then dissolve the tissue powder with lysis buffer.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>In some cases an ISE transfactor gene may not be found in the TCGA mRNA expression dataset, which will prevent PAM50-Subtypic Interaction-Correlated Expression Pattern (iCEP) and Kaplan-Meier survival analysis of the ISE transfactor.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Use GeneCards (<a href="https://www.genecards.org/">https://www.genecards.org/</a>) to see if there is an alias for the gene that might be found in the TCGA dataset. Also, see if the ENTREZ ID for the gene is present in TCGA dataset, which can be used to access mRNA expression data.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cerami, E., Gao, J., Dogrusoz, U., Gross, B.E., Sumer, S.O., Aksoy, B.A., Jacobsen, A., Byrne, C.J., Heuer, M.L., and Larsson, E. (2012). The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov. <i>2</i>, 401-404. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref1">View at publisher</a></p>
<p id="bib2">Ciriello, G., Gatza, M.L., Beck, A.H., Wilkerson, M.D., Rhie, S.K., Pastore, A., Zhang, H., McLellan, M., Yau, C., and Kandoth, C. (2015). Comprehensive molecular portraits of invasive lobular breast cancer. Cell <i>163</i>, 506-519. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref2">View at publisher</a></p>
<p id="bib3">Curtis, C., Shah, S.P., Chin, S.-F., Turashvili, G., Rueda, O.M., Dunning, M.J., Speed, D., Lynch, A.G., Samarajiwa, S., and Yuan, Y. (2012). The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. Nature <i>486</i>, 346-352. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref3">View at publisher</a></p>
<p id="bib4">Gao, J., Aksoy, B.A., Dogrusoz, U., Dresdner, G., Gross, B., Sumer, S.O., Sun, Y., Jacobsen, A., Sinha, R., and Larsson, E. (2013). Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Science Signal. <i>6</i>, pl1. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref4">View at publisher</a></p>
<p id="bib6">Pereira, B., Chin, S.-F., Rueda, O.M., Vollan, H.-K.M., Provenzano, E., Bardwell, H.A., Pugh, M., Jones, L., Russell, R., and Sammut, S.-J. (2016). The somatic mutation profiles of 2,433 breast cancers refine their genomic and transcriptomic landscapes. Nature Commun. <i>7</i>, 1-16. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref6">View at publisher</a></p>
<p id="bib7">Rappsilber, J., Mann, M., and Ishihama, Y. (2007). Protocol for micro-purification, enrichment, pre-fractionation and storage of peptides for proteomics using StageTips. Nature Protoc. <i>2</i>, 1896. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref7">View at publisher</a></p>
<p id="bib8">Scopes, R.K. (2013). Protein purification: principles and practice (Springer Science &amp; Business Media). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref8">View at publisher</a></p>
<p id="bib5">The Cancer Genome Atlas Network. (2012). Comprehensive molecular portraits of human breast tumours. Nature <i>490</i>, 61. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref5">View at publisher</a></p>
<p id="bib9">Therneau, T.M. and Grambsch, P.M. (2000). Modeling survival data: extending the Cox model (Springer). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref9">View at publisher</a></p>
<p id="bib10">Tyanova, S., Temu, T., and Cox, J. (2016a). The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. Nature Protoc. <i>11</i>, 2301. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref10">View at publisher</a></p>
<p id="bib11">Tyanova, S., Temu, T., Sinitcyn, P., Carlson, A., Hein, M.Y., Geiger, T., Mann, M., and Cox, J. (2016b). The Perseus computational platform for comprehensive analysis of (prote) omics data. Nat. Method. <i>13</i>, 731-740. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref11">View at publisher</a></p>
<p id="bib12">Wang, L., Wrobel, J.A., Xie, L., Li, D., Zurlo, G., Shen, H., Yang, P., Wang, Z., Peng, Y., and Gunawardena, H.P. (2018). Novel RNA-affinity proteogenomics dissects tumor heterogeneity for revealing personalized markers in precision prognosis of Cancer. Cell Chem. Biol. <i>25</i>, 619-633.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref12">View at publisher</a></p>
<p id="bib13">Wang, Y., Ma, M., Xiao, X., and Wang, Z. (2012). Intronic splicing enhancers, cognate splicing factors and context-dependent regulation rules. Nat. Struct. Mol. Biol. <i>19</i>, 1044. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref13">View at publisher</a></p>
<p id="bib14">Wrobel, J.A., Xie, L., Wang, L., Liu, C., Rashid, N., Gallagher, K.K., Xiong, Y., Konze, K.D., Jin, J., and Gatza, M.L. (2019). Multi-omic dissection of oncogenically active epiproteomes identifies drivers of proliferative and invasive breast tumors. iScience <i>17</i>, 359-378. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00045-9/sref14">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Xian Chen (<a href="mailto:xianc@email.unc.edu">xianc@email.unc.edu</a>).</p>
<h4>Materials Availability</h4>
<p>No new materials are generated in this study.</p>
<h4>Data and Code Availability</h4>
<p>The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD008642<i>.</i> The R scripts and data files used for the analysis of the TCGA datasets are available online at Mendeley Data (<a href="https://doi.org/10.17632/df32tkb89c.1">https://doi.org/10.17632/df32tkb89c.1</a>).</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants NIH R01 GM133107-01, UNC University Cancer Research Fund (UCRF), and 1U24CA160035-01 from the <a href="https://doi.org/10.13039/100000054">National Cancer Institute</a> Clinical Proteomic Tumor Analysis Consortium (CPTAC), awarded to X.C. This invention is protected by US Provisional Patent Application Serial No. 62/481,383 that was filed by University of North Carolina-Chapel Hill. X.C. is the founder of TransChromix, LLC.</p>
<h3>Author Contributions</h3>
<p>L.W. wrote the sample preparation protocol of cell lines and clinical tissue pull-down and MS analysis, prepared the figures, and conceived the research questions; J.A.W. developed the software, performed KM analysis of clinical TCGA data and proteogenomic analysis, and wrote the method. L.X. prepared samples and performed LFQ MS/MS analysis and wrote the methods. X.C. conceived and planned this project and contributed to the writing of the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>