<article>
<section>
<h1 id="header">Quantitative analysis of dense-core vesicle fusion in rodent CNS neurons</h1>
<p><time datetime="2021-02-05">Published: February 5, 2021</time></p>
<p>Alessandro Moro,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Rein I. Hoogstraaten,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Claudia M. Persoon,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Matthijs Verhage,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Ruud F. Toonen<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Functional Genomics, Center for Neurogenomics and Cognitive Research (CNCR), Vrije Universiteit (VU) Amsterdam, de Boelelaan 1085, 1081 HV Amsterdam, the Netherlands</p>
<p id="aff2"><sup>2</sup>Department of Clinical Genetics, UMC Amsterdam, the Netherlands</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ruud.toonen@cncr.vu.nl">ruud.toonen@cncr.vu.nl</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100325">10.1016/j.xpro.2021.100325</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Neuropeptides are essential signaling molecules secreted by dense-core vesicles (DCVs). They contribute to information processing in the brain, controlling a variety of physiological conditions. Defective neuropeptide signaling is implicated in several psychiatric disorders. Here, we provide a protocol for the quantitative analysis of DCV fusion events in rodent neurons using pH-sensitive DCV fusion probes and custom-written analysis algorithms. This method can be used to study DCV fusion mechanisms and is easily adapted to investigate fusion principles of other secretory organelles.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib10">Persoon et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for quantitative analysis of neuropeptide exocytosis in rodent CNS neurons</li>
<li>Generation and lentiviral infection of neurons on prepatterned astrocyte microdot islands</li>
<li>Visualization of neuronal DCV fusion events using pH-sensitive fusion probes</li>
<li>Quantitative analysis of fusion events using custom-written algorithms</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/454-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the experimental steps to quantify DCV fusion events in single cultured neurons grown on microdot islands of astrocytes using pH-sensitive DCV fusion probes. The protocol covers the production of lentiviral particles, preparation of single neuron cultures on astrocyte microdots, image acquisition, and data analysis using custom-written algorithms.</p>
<h3 id="sec1.1">Lentivirus particle production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>For efficient infection of cultured neurons use lentiviral infection to express your gene of interest. In this protocol we use the DCV fusion probes NPY-pHluorin and BDNF-pHluorin, and synapsin-mCherry as live synapse marker. All constructs are driven by a human synapsin promoter for neuron specific expression.</p>
<p>These steps describe lentiviral particle production according to <a href="#bib8">Naldini et al. (1996)</a>. We use a 2<sup>nd</sup> generation lenti viral production system (see <a href="https://www.addgene.org/guides/lentivirus/">https://www.addgene.org/guides/lentivirus/</a> for info on the use of 2<sup>nd</sup> or 3<sup>rd</sup> generation systems).</p>
<ol>
<li>Plate 293T cells from an 80% confluent T175 flask in 245 mm square tissue culture (TC) treated-dishes in DMEM complete (please refer to “<a href="#materials-and-equipment">Materials and equipment</a>” for media recipes). Incubate in a 37°C, 5% CO<sub>2</sub> incubator for 1 day, or until 50% confluence.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> 293T cell passage number should be &lt; 25.</div>
<ol start="2">
<li>For each lenti viral batch (lenti-hSyn-NPY-pHluorin, lenti-hSyn-BDNF-pHluorin, or lenti-hSyn-synapsin-mCherry), transfect cells with a plasmid mix in PBS containing a VSV-G envelope plasmid (e.g., p.MD2.G, 11.4 μg), packaging plasmid (e.g., pCMVΔR8.2, 22.4 μg) and a transfer plasmid containing your gene of interest (e.g., pRRL, 45.8 μg) using polyethyleneimine (PEI; 240 μg). Make up a total volume of 5 mL PBS per plate with a DNA to PEI ratio of 1:3.</li>
<li>Replace media with 100 mL Opti-MEM without fetal calf serum (FCS) after 1 day.</li>
<li>Harvest conditioned medium 40 h after transfection.</li>
<li>Centrifuge supernatant using an Amicon spin filter (100 kDa cutoff) at 4,000 × <i>g</i> for 20–30 min.</li>
<li>Transfer the ± 150 μL of concentrated virus to a tube, dilute in 1 mL PBS and filter sterilize through a 0.2 μm syringe filter.</li>
<li>Typical viral titers obtained with this method range from 5 × 10<sup>8</sup> IFU/mL to 1 × 10<sup>9</sup> IFU/mL.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Viral particles should be aliquoted in single use 0.5 mL tubes, 25–50 μL per tube, and stored at −80°C. Do not re-freeze. Thawed aliquots can be stored at 4°C for up to 4 weeks without loss of efficiency.</div>
<h3 id="sec1.2">Coverslip preparation for neuronal culture on astrocyte microdots</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="8">
<li>Etch 18 mm glass coverslips in 1 N HCl for 12–16 h, wash with H<sub>2</sub>O followed by 1 h in 1 N NaOH.</li>
<li>Rinse extensively with H<sub>2</sub>O and briefly in 70% EtOH. Place coverslips, once air dried, in 12-well plates.</li>
<li>Apply 0.5 mL of agarose type II-A (1.5 gr/l in H<sub>2</sub>O) to cover the entire surface of each coverslip. Remove agarose with a 1 mL pipet within 30 s and let the coverslip air dry.</li>
<li>Prepare the Poly-D-lysine (PDL)-collagen-acetic acid-mix (1:1:3). PDL (0.5 mg/mL; stored at −20°C), collagen (∼3.5 mg/mL; stored at 4°C), acetic acid (17 mM; stored at 4°C).</li>
<li>Put 500 μL of mix onto a filter paper (3 mm cellulose chromatography paper, Whatman) and suck off the extra fluid, the paper should be wet, but not soaked.</li>
<li>Put a rubber stamp containing a grid of 250 μm diameter dots (<a href="#bib2">Burgalossi et al., 2012</a>) on the paper (do not push) and then on the agarose-treated coverslip (give a little pressure).</li>
<li>UV-sterilize the plates for 30 min. Plates can be stored up to 2 weeks at 19°C–22°C in a sterile environment.</li>
</ol>
<h3 id="sec1.3">Culture single primary hippocampal neurons on astrocyte microdots</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 weeks</div>
<p>Astrocyte microdot and primary hippocampal culture preparations have been extensively described (<a href="#bib2">Burgalossi et al., 2012</a>). This is a brief overview highlighting the most important steps.</p>
<ol start="15">
<li>Astrocyte cell preparation: Under a dissection microscope, isolate cortices of two P1 rat pups in Hanks+ medium.</li>
<li>Incubate for 45 min in 5 mL of papain containing ES medium at 37°C with gentle agitation.</li>
<li>Remove ES and add 5 mL of IS medium and incubate for another 15 min. Remove IS and add 2 mL DMEM+ medium. Triturate cortices with a fire-polished Pasteur pipette.</li>
<li>Plate the single cell suspension in T175 flasks in 35 mL DMEM+ and incubate at 37°C, 5% CO<sub>2</sub>.</li>
<li>Refresh medium after 1 day.</li>
<li>When astrocytes are confluent (± 7 days), dissociate the cells from the flask with trypsin-EDTA, count the cells and plate at a density of 6,000 cells/well of a 12 well plate in DMEM+.</li>
<li>Incubate the astrocyte culture in a 37°C, 5% CO<sub>2</sub> incubator for 3–4 days, or until confluent on the microdots (<a href="#fig1">Figure 1</a>A). Replace DMEM+ with NB+ medium.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/454-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Neuronal culture and microscope setup</div>
<p>(A) Example of astrocyte microdot island. Scale bar, 100 μm.</p>
<p>(B) Example of hippocampal neuron, indicated by the arrowhead, 2 days after plating (DIV2) on astrocyte microdot. Scale bar, 100 μm.</p>
<p>(C) Example of hippocampal neuron (arrowhead) at DIV10 on astrocyte microdot. Scale bar, 100 μm</p>
<p>(D) Example of hippocampal neuron (arrowhead) at DIV14 on astrocyte microdot. Scale bar, 100 μm.</p>
<p>(E) Live-cell imaging setup with pinch-valve controlled perfusion system, electrodes, and perfusion barrel.</p>
<p>(F) Example of a coverslip mounted on the microscope stage with perfusion and suction needles, electrodes, and barrel system to deliver Tyrode’s NH<sub>4</sub>Cl, or other solutions.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> DMEM+ should be replaced with NB+ medium on the day of neuronal culture, or 1 day before.</div>
<ol start="22">
<li>Under a dissection microscope, isolate E18-P1 mouse hippocampi in Hanks+ medium. Add 0.025% Trypsin to the Hanks+ solution and incubate hippocampi for 20 min at 37°C, triturate the hippocampi, count the cells and plate neurons at a density of 1,000 cells/well of a 12 well plate in NB+ medium.</li>
<li>Incubate astrocyte-neuron co-cultures at 37°C, 5% CO<sub>2</sub> for up to 21 days.</li>
</ol>
<h3 id="sec1.4">Infect neurons with DCV fusion reporter constructs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<p>This step labels neuronal DCVs with super-ecliptic pHluorin to monitor DCV fusion.</p>
<ol start="24">
<li>At day-in-vitro (DIV) 9–10, check the quality of island cultures under a bright-field microscope. The majority of the astrocyte islands should be intact and confluent and contain no more than one hippocampal neuron (<a href="#fig1">Figure 1</a>C).</li>
<li>Infect neurons by adding 0.5–3 μL of lentivirus particles encoding for BDNF- or NPY-pHluorin to label DCVs and synapsin-mCherry to label synapses into each well.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Titrate new lentiviral batches by infecting DIV9–10 neurons with a limiting dilution series. In our hands, 0.5 to 3 μL of a virus solution (∼5 × 10<sup>8</sup> IFU/mL) prepared according to our lentivirus particle production protocol will result in infection of &gt;90% of neurons imaged using a fluorescence microscope at DIV14. For each lentiviral batch the optimal concentration needs to be assessed. Batches will remain stable for at least 12 months at −80°C and for up to 4 weeks when stored at 4°C. For lenti-hSyn-NPY-pHluorin and lenti-hSyn-BDNF-pHluorin, many moving and stationary puncta should be visible upon Tyrode's NH<sub>4</sub>Cl perfusion. For lenti-hSyn-synapsin-mCherry, immobile puncta (presynaptic terminals) should be visible. Select the optimal viral titer for use in live imaging experiments.</div>
<ol start="26">
<li>Leave 1 or 2 coverslips per condition uninfected. These will be used for Fluo5 Ca<sup>2+</sup> imaging at DIV14.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Infection of additional lentiviral constructs (Cre, rescue constructs) can be performed at earlier or later timepoints. However, the total volume of viral solution should not exceed 20 μL.</div>
<h3 id="sec1.5">Prepare solutions for live-cell imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="27">
<li>Prepare Tyrode's solution and Tyrode's solution containing NH<sub>4</sub>Cl (refer to “<a href="#materials-and-equipment">Materials and equipment</a>” for recipes).
<ol type="a">
<li>Dissolve all salts in dH<sub>2</sub>O at 19°C–22°C.</li>
<li>Adjust the pH to 7.4 with 1 N NaOH.</li>
<li>Check the osmolarity and adjust to 280 mOsmol with glucose.</li>
<li>Filter sterilize and store at 4°C.</li>
<li>Bring to 19°C–22°C before live-cell imaging.</li>
</ol>
</li>
<li>Prepare 2 mM Fluo5-AM aliquots of 5 μL by dissolving Fluo5-AM in DMSO.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Agarose type II-A</td>
<td>Sigma</td>
<td>Cat# A9918</td>
</tr>
<tr>
<td>Laminin</td>
<td>Sigma</td>
<td>Cat# L2020</td>
</tr>
<tr>
<td>Poly-D-lysine</td>
<td>Sigma</td>
<td>Cat# P6407</td>
</tr>
<tr>
<td>Rat tail collagen</td>
<td>BD Biosciences</td>
<td>Cat# 354236</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>VWR</td>
<td>Cat# 20104.334</td>
</tr>
<tr>
<td>B-27</td>
<td>Gibco</td>
<td>Cat# 17504-044</td>
</tr>
<tr>
<td>PBS</td>
<td>Gibco</td>
<td>Cat# 14190-094</td>
</tr>
<tr>
<td>Polyethyleneimine</td>
<td>Polysciences</td>
<td>Cat# 23966-1</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Fisher</td>
<td>Cat# 12087549</td>
</tr>
<tr>
<td>Hanks</td>
<td>Sigma</td>
<td>Cat# H9394</td>
</tr>
<tr>
<td>HEPES 1 M</td>
<td>Gibco</td>
<td>Cat# 15630-056</td>
</tr>
<tr>
<td>DMEM + UltraGlutamine I</td>
<td>Lonza</td>
<td>Cat# LO BE12-604F/U1/12</td>
</tr>
<tr>
<td>FCS</td>
<td>Gibco</td>
<td>Cat# 10270</td>
</tr>
<tr>
<td>NEAA</td>
<td>Life Technologies</td>
<td>Cat# 11140-035</td>
</tr>
<tr>
<td>Penicillin/streptomycin</td>
<td>Gibco</td>
<td>Cat# 15630-056</td>
</tr>
<tr>
<td>L-Cysteine</td>
<td>Sigma</td>
<td>Cat# C7352</td>
</tr>
<tr>
<td>EDTA</td>
<td>AppliChem</td>
<td>Cat# A2937 0500</td>
</tr>
<tr>
<td>Neurobasal medium</td>
<td>Gibco</td>
<td>Cat# 21103-049</td>
</tr>
<tr>
<td>Glutamax</td>
<td>Gibco</td>
<td>Cat# 35050-038</td>
</tr>
<tr>
<td>Papain</td>
<td>Worthington</td>
<td>Cat# 3127</td>
</tr>
<tr>
<td>Trypsin-EDTA</td>
<td>Gibco</td>
<td>Cat# 25300-054</td>
</tr>
<tr>
<td>Trypsin</td>
<td>Gibco</td>
<td>Cat# 1509-046</td>
</tr>
<tr>
<td>Albumin</td>
<td>AppliChem</td>
<td>Cat# A1391 0100</td>
</tr>
<tr>
<td>Trypsin inhibitor</td>
<td>Sigma</td>
<td>Cat# T9523</td>
</tr>
<tr>
<td>NaOH</td>
<td>Sigma</td>
<td>Cat# S5881</td>
</tr>
<tr>
<td>HCl</td>
<td>Sigma</td>
<td>Cat# 38283</td>
</tr>
<tr>
<td>EtOH</td>
<td>VWR</td>
<td>Cat# 83804.360</td>
</tr>
<tr>
<td>NaCl</td>
<td>Fisher</td>
<td>Cat# 15673410</td>
</tr>
<tr>
<td>KCl</td>
<td>SERVA</td>
<td>Cat# 7447-40-7</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>·2H<sub>2</sub>O</td>
<td>Sigma</td>
<td>Cat# C7902</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>·6H<sub>2</sub>O</td>
<td>Sigma</td>
<td>Cat# M9272</td>
</tr>
<tr>
<td>HEPES (1 M)</td>
<td>Fisher</td>
<td>Cat# 15630-056</td>
</tr>
<tr>
<td>Glucose-H<sub>2</sub>O</td>
<td>VWR</td>
<td>Cat# 1040741000</td>
</tr>
<tr>
<td>NH<sub>4</sub>Cl</td>
<td>Sigma</td>
<td>Cat# 12125-02-9</td>
</tr>
<tr>
<td>Fluo-5F-AM</td>
<td>Molecular Probes</td>
<td>Cat# F14222</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Sigma</td>
<td>Cat# D2438</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Mouse: primary neurons from hippocampus, cortex, or striatum</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Rat: primary astrocytes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>293T cells</td>
<td>ATCC</td>
<td>CRL-11268</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: <i>Rab3ABCD</i> null embryonic day 18 (E18) pups</td>
<td><a href="#bib11">Schlüter et al., 2004</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Mouse: <i>Rim1/2</i> conditional null mutant E18 pups</td>
<td>
<a href="#bib5">Kaeser et al., 2011</a>; <a href="#bib6">Kaeser et al., 2008</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td>Rat: Wistar (Crl:WI) P1 pups</td>
<td>Charles River</td>
<td>Strain code: 003</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>hSyn(pr):NPY-pHluorin</td>
<td><a href="#bib12">Van de Bospoort et al., 2012</a></td>
<td>N/A</td>
</tr>
<tr>
<td>hSyn(pr):rBDNF-pHluorin</td>
<td><a href="#bib3">de Wit et al., 2009</a></td>
<td>N/A</td>
</tr>
<tr>
<td>hSyn(pr):Synapsin-mCherry</td>
<td>(<a href="#bib10">Persoon et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>p.MD2.G (encoding VSV-G envelope)</td>
<td><a href="#bib8">Naldini et al., 1996</a></td>
<td>Addgene: #12259</td>
</tr>
<tr>
<td>pCMVΔR8.2 (2<sup>nd</sup> gen. packaging encoding HIV-1 Gag, Pol, Tat, and Rev proteins)</td>
<td><a href="#bib8">Naldini et al., 1996</a></td>
<td>Addgene: #12263</td>
</tr>
<tr>
<td>pRRL (2<sup>nd</sup>/3<sup>rd</sup> gen. transfer)</td>
<td><a href="#bib8">Naldini et al., 1996</a></td>
<td>Addgene: #12252</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>MATLAB</td>
<td>MathWorks</td>
<td><a href="https://mathworks.com">https://mathworks.com</a></td>
</tr>
<tr>
<td>ImageJ</td>
<td>NIH</td>
<td>
<a href="https://imagej.net/">https://imagej.net/</a> RRID:SCR_003070</td>
</tr>
<tr>
<td>DCV_pHluorin</td>
<td>N/A</td>
<td><a href="https://github.com/alemoro/">https://github.com/alemoro/</a></td>
</tr>
<tr>
<td>Microsoft Excel</td>
<td>Microsoft</td>
<td><a href="https://www.microsoft.com">https://www.microsoft.com</a></td>
</tr>
<tr>
<td>NIS Elements</td>
<td>Nikon Instruments</td>
<td>
<a href="https://www.nikoninstruments.com/Products/Software">https://www.nikoninstruments.com/Products/Software</a>RRID:SCR_014329</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>0.2 μm syringe filters</td>
<td>Fisher</td>
<td>Cat# 10331075</td>
</tr>
<tr>
<td>T175 flask</td>
<td>Greiner</td>
<td>Cat# 690175</td>
</tr>
<tr>
<td>TC-treated culture dishes</td>
<td>Fisher</td>
<td>Cat# 10350602</td>
</tr>
<tr>
<td>Amicon spin filter</td>
<td>Millipore</td>
<td>Cat# UFC910096</td>
</tr>
<tr>
<td>3 mm cellulose chromatography paper, Whatman</td>
<td>VWR</td>
<td>Cat# 514-8013</td>
</tr>
<tr>
<td>18 mm glass coverslip</td>
<td>VWR</td>
<td>Cat# 631-1342</td>
</tr>
<tr>
<td>12 well plates</td>
<td>Greiner</td>
<td>Cat# 665165</td>
</tr>
<tr>
<td>NikonTi-E Eclipse</td>
<td>Nikon</td>
<td><a href="https://nikon.com">https://nikon.com</a></td>
</tr>
<tr>
<td>EMCCD DU-897 camera</td>
<td>Andor</td>
<td><a href="https://andor.com">https://andor.com</a></td>
</tr>
<tr>
<td>Master-8</td>
<td>AMPI</td>
<td><a href="https://ampi.co.il">https://ampi.co.il</a></td>
</tr>
<tr>
<td>Stimulus isolator A-395</td>
<td>WPI</td>
<td><a href="https://wpiinc.com">https://wpiinc.com</a></td>
</tr>
<tr>
<td>Pinch valve</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>50 mL perfusion syringes</td>
<td>VWR</td>
<td>Cat# 613-5398</td>
</tr>
<tr>
<td>Exadrop</td>
<td>B. Braun</td>
<td>Cat# 4061306</td>
</tr>
<tr>
<td>Tubing</td>
<td>Sigma</td>
<td>Cat# 58703</td>
</tr>
<tr>
<td>Luer Lock male</td>
<td>Brunschwig</td>
<td>Cat# CT58.1</td>
</tr>
<tr>
<td>Luer Lock female</td>
<td>Brunschwig</td>
<td>Cat# CT62.1</td>
</tr>
<tr>
<td>Glass capillary</td>
<td>GmbH</td>
<td>Cat# GB150F-8P</td>
</tr>
<tr>
<td>Platinum electrodes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Dissection microscope</td>
<td>Nikon</td>
<td>SMZ745</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>DCV_pHluorin code</td>
<td><a href="#bib10">Persoon et al., 2019</a></td>
<td><a href="https://github.com/alemoro">https://github.com/alemoro</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>DMEM complete medium (DMEM+)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM + UltraGlutamine I</td>
<td>1×</td>
<td>440</td>
</tr>
<tr>
<td>FCS</td>
<td>10%</td>
<td>50</td>
</tr>
<tr>
<td>NEAA</td>
<td>1%</td>
<td>5</td>
</tr>
<tr>
<td>Penicillin/streptomycin</td>
<td>50,000 U/mL</td>
<td>5</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500</b></td>
</tr>
</tbody>
</table>
<p>Store at 4°C for up to 1 month. Prepare 50 mL aliquots to reduce rounds of heating (37°C) and cooling, and risk of contamination.</p>
<table id="undtbl2">
<caption>Hanks+</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hanks</td>
<td>1×</td>
<td>990</td>
</tr>
<tr>
<td>HEPES 1 M</td>
<td>10 mM</td>
<td>10</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1,000</b></td>
</tr>
</tbody>
</table>
<p>Store at 19°C–22°C in the dark for up to 24 months.</p>
<table id="undtbl3">
<caption>Enzyme solution (ES)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM + UltraGlutamine I</td>
<td>1×</td>
<td>245</td>
</tr>
<tr>
<td>L-Cysteine</td>
<td>50 mg</td>
<td>n/a</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>100 mM</td>
<td>2.5</td>
</tr>
<tr>
<td>EDTA</td>
<td>50 mM</td>
<td>2.5</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>250</b></td>
</tr>
</tbody>
</table>
<p>Store at −20°C in 5 mL aliquots for up to 12 months.</p>
<p>Preparation:</p>
<ul>
<li>Defrost ES (5 mL) and add 20–25 U/mL papain.</li>
<li>Incubate at 37°C until papain is dissolved (solution should be clear).</li>
<li>Bubble 15 min with carbogen (solution should go from purple to more pinkish) and sterilize with 0.2 μm filter before use.</li>
</ul>
<table id="undtbl4">
<caption>Inhibitor solution (IS)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM + UltraGlutamine 1</td>
<td>1×</td>
<td>225</td>
</tr>
<tr>
<td>FCS</td>
<td>10%</td>
<td>25</td>
</tr>
<tr>
<td>Albumin</td>
<td>625 mg</td>
<td>n/a</td>
</tr>
<tr>
<td>Trypsin inhibitor</td>
<td>625 mg</td>
<td>n/a</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>250</b></td>
</tr>
</tbody>
</table>
<p>Store at −20°C for up to 12 months.</p>
<table id="undtbl5">
<caption>Complete neurobasal medium (NB+)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neurobasal medium</td>
<td>1×</td>
<td>479.25</td>
</tr>
<tr>
<td>B-27 Supplement</td>
<td>2%</td>
<td>10</td>
</tr>
<tr>
<td>HEPES 1 M</td>
<td>17 mM</td>
<td>9</td>
</tr>
<tr>
<td>Glutamax</td>
<td>1%</td>
<td>1.25</td>
</tr>
<tr>
<td>Penicillin/streptomycin</td>
<td>50,000 U/mL</td>
<td>0.5</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500</b></td>
</tr>
</tbody>
</table>
<p>Store at 4°C for up to 1 month. Prepare 50 mL aliquots to reduce rounds of heating (37°C) and cooling, and risk of contamination.</p>
<table id="undtbl6">
<caption>Tyrode’s imaging medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>119</td>
<td>n/a</td>
</tr>
<tr>
<td>KCl</td>
<td>2.5</td>
<td>n/a</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>2</td>
<td>n/a</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>2</td>
<td>n/a</td>
</tr>
<tr>
<td>HEPES</td>
<td>25</td>
<td>n/a</td>
</tr>
<tr>
<td>Glucose</td>
<td>30</td>
<td>n/a</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
</tr>
</tbody>
</table>
<p>Store at 4°C for up to 6 months.</p>
<table id="undtbl7">
<caption>Tyrode’s NH<sub>4</sub>Cl medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>69</td>
<td>n/a</td>
</tr>
<tr>
<td>KCl</td>
<td>2.5</td>
<td>n/a</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>2</td>
<td>n/a</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>2</td>
<td>n/a</td>
</tr>
<tr>
<td>NH<sub>4</sub>Cl</td>
<td>50</td>
<td>n/a</td>
</tr>
<tr>
<td>HEPES</td>
<td>25</td>
<td>n/a</td>
</tr>
<tr>
<td>Glucose</td>
<td>30</td>
<td>n/a</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
</tr>
</tbody>
</table>
<p>Store at 4°C for up to 6 months.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Microscope preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>Here, directions are provided to setup a Nikon Ti-E Eclipse inverted microscope system for live-cell imaging experiments. These steps are similar for other microscopes and acquisition software such as Metamorph, AxioVision, and Olympus Zen. Please refer to the manufacturer's instructions how to send a TTL signal to the Master-8.</p>
<p>Live imaging experiments are performed on a Nikon Ti-E Eclipse inverted microscope system fitted with a Confocal A1R (LU4A Laser) unit and an EMCCD (Andor DU-897), a 40× oil objective (NA 1.3) and appropriate filter sets. NIS Elements software (version 4.30) controls the microscope and image acquisition. Electrical field stimulation is applied via two parallel platinum electrodes controlled by a Master-8 (AMPI) connected to a stimulus isolator (A-395, World Precision Instruments). Superfusion of NH<sub>4</sub>Cl containing Tyrode’s through a glass capillary placed just above the neuron is controlled with a pinch valve (<a href="#fig1">Figure 1</a>E).</p>
<ol>
<li>In the image acquisition software NIS Elements open ND Acquisition and select time. Set the interval as time between frames to 0.5 s for 2 Hz image acquisition, and the duration of the image acquisition.</li>
<li>Set TTL out to trigger the Master-8 at the start of the image acquisition.</li>
<li>Set up a Master-8 paradigm (<a href="#fig2">Figure 2</a>):
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/454-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Master 8 setup</div>
<p>Graphical representation of the Master 8 interface and stimulation paradigm used in this protocol. Channel 1 is connected to the computer (PC), channel 3 is connected to the stimulus isolator, and channel 4 is connected to the pinch valve system for Tyrode’s NH<sub>4</sub>Cl application. Channels 2, 5, 6, 7, 8 are not connected to an external device. Channel 1 receives the TTL signal from the PC at the start of the time-lapse recording. After a 30 s (L) delay, channel 1 delivers a 1 ms (D) trigger to channel 2 and channel 4. Channel 2 responds with 16 (M) 1 ms (D) pulses every 1.5 s (I) that triggers channel 3 with 50 (M) 1 ms (D) pulses every 20 ms (I). Thus, the communication between channel 2 and 3 generates the 16 trains of 50 action potentials at 50 Hz. Channel 4 responds 50 s (L) after input from channel 1 by delivering a 5 s (D) trigger to the pinch valve to start Tyrode's NH<sub>4</sub>Cl perfusion.</p>
</figcaption>
</figure>
<ol type="a">
<li>Start with a delay of 30 s to record baseline fluorescence.</li>
<li>Send 50, 1 ms pulses at 50 Hz to the stimulus isolator for 16 times with 0.5 s between bursts.</li>
<li>Add a delay of 26 s to record post-stimulus fusion events.</li>
<li>Send a 10 s pulse to the pinch valve controller to start superfusion of NH<sub>4</sub>Cl containing Tyrode’s 26 s after stimulation.</li>
</ol>
</li>
<li>Fill Tyrode’s and Tyrode’s NH<sub>4</sub>Cl syringes for gravity flow delivery via software controlled pinch valves (<a href="#fig1">Figure 1</a>E).</li>
</ol>
<h3 id="sec3.2">Live-cell imaging, Ca<sup>2+</sup> imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min to 1 h</div>
<p>This step visualizes intracellular Ca<sup>2+</sup> elevations upon stimulation and subsequent buffering from the cytosol. It provides a read out of the culture’s physiological state and is typically performed at the start of each DCV fusion imaging session.</p>
<ol start="5">
<li>Incubate one coverslip containing uninfected DIV14 neurons with Fluo5-AM (2 μΜ final concentration) for 10 min at 37°C, 5% CO<sub>2</sub>.</li>
<li>Place the coverslip into an imaging chamber and add 500 μL of Tyrode’s solution.</li>
</ol>
<ol start="7">
<li>Prepare neurons for live-cell imaging at the microscope setup (<a href="#fig1">Figure 1</a>F).
<ol type="a">
<li>Place the imaging chamber in the stage holder and bring the neurons in focus using a 40× oil objective in DIC or fluorescence mode (<a href="#fig1">Figure 1</a>D).</li>
<li>Establish Tyrode's perfusion of 0.5 mL/min by placing one perfusion needle at one end and a suction needle at the opposite end of the imaging chamber.</li>
<li>Place two platinum electrodes connected to the stimulus isolator at opposite sides of an astrocyte microdot containing a single neuron.
<div class="critical">
<span class="critical-title">Critical:</span> Make sure that the distance between the electrodes spans one astrocyte microdot (± 0.5mm) and are placed in such a way that the center of your field of view lies between the electrodes. Lower the electrodes as far as possible, just above the glass coverslip but do not touch the coverslip.</div>
</li>
<li>Adjust the 488nm laser power and camera exposure time for optimal dynamic range.
<div class="critical">
<span class="critical-title">Critical:</span> Make sure that the camera pixels are far from saturation since stimulation will typically induce a 100-fold increase in fluorescence.</div>
</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Start time-lapse recording (image acquisition at 2Hz, or higher) and trigger the Master-8 via TTL pulse.</li>
<li>Repeat steps 7c–8 for a maximum of 5 times, or for no longer than 1 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to leave at least one astrocyte microdot between a previously stimulated neuron and the next to guarantee that each neuron is naive at the start of each recording.</div>
<ol start="10">
<li>Assess Ca<sup>2+</sup> influx and efflux of each neuron by plotting intensity over time of multiple regions of interests (ROIs) drawn on the neurites (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/454-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Neuronal live-cell imaging</div>
<p>(A) Top, typical neuron after incubation with the Ca<sup>2+</sup> reporter Fluo5-AM. Insets show zoom of a neurite before field stimulation (B), during stimulation (S) and 2 s after stimulation (R). Bottom, average Fluo5-AM signal during a live-cell imaging experiment consisting of 30 s baseline recording (B), 16 trains of 50 AP at 50 Hz stimulation (S, light blue bars), and 50 s recovery (R). Note the sharp rise and fall of the calcium signal. Scale bar, 20 μm in all.</p>
<p>(B) Top, typical neuron with NPY-pHluorin fusion events, calculated using the Standard Deviation Z-projection option in Fiji, with a zoom of a neurite before field stimulation (B), during stimulation (S), after stimulation (R), and during Tyrode’s NH<sub>4</sub>Cl superfusion (N). Bottom, five example intensity traces showing typical NPY-pHluorin fusion events (start indicated with green arrow head). The intensity was calculated in 3 × 3 pixel ROIs. Scale bar, 20 μm in all.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> If you also use synapsin-mCherry you can draw ROIs on synapsin-mCherry positive puncta to image Ca<sup>2+</sup> signals in synapses specifically.</div>
<h3 id="sec3.3">Live-cell imaging, DCV fusion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This step visualizes single fusion events of pHluorin-labeled DCVs upon stimulation.</p>
<ol start="11">
<li>Place one coverslip containing DIV14 neurons infected with NPY-pHluorin (or BDNF-pHluorin) into an imaging chamber and add 500 μL Tyrode's solution.</li>
<li>Similar to step 7, establish perfusion, position the platinum electrodes and the glass capillary for Tyrode’s NH<sub>4</sub>Cl superfusion close to the neuron in focus.</li>
<li>Start a time-lapse recording with Tyrode’s NH<sub>4</sub>Cl superfusion at the end of the acquisition to dequench NPY-pHluorin in all DCVs (<a href="#fig3">Figure 3</a>B).</li>
</ol>
<h3 id="sec3.4">DCV fusion events detection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min/cell</div>
<p>This step provides information on semi-automatic analysis of DCV fusion events in time-lapse recordings. DCV fusion events are detected based on the standard deviation of the signal intensity throughout the stimulation period. After the automatic detection, it is possible to manually inspect the ROIs for false positives, or to manually add new ROIs.</p>
<ol start="14">
<li>Install the DCV_pHluorin toolset by extracting the DCVpHluorin.zip file from Github in the Fiji.app/macros/toolsets folder. Start Fiji and run the toolset via selecting DCV_pHluorin in the toolset dropdown menu (<a href="#fig4">Figure 4</a>A)
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/454-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. DCV_pHluorin installation and workflow</div>
<p>(A) Fiji main interface, locate the DCV_pHluorin toolset on the toolset dropdown menu, “≫”.</p>
<p>(B) Fiji main interface after loading the DCV_pHluorin toolset.</p>
<p>(C) Visualization of the DCV_pHluorin toolset submenu.</p>
<p>(D) DCV_pHluorin main workflow. Trapezoid blocks represent steps where the user is required to perform an action.</p>
</figcaption>
</figure>
<ol type="a">
<li>After the first installation, the toolset needs to be initialized by opening the “Option menu” (<a href="#fig4">Figures 4</a>B and 4C).</li>
<li>As a final step, install the “Check ROIs” plugin via Plugin &gt; Install Plugin… in Fiji interface. Locate the “DCVpHluorin_CheckROIs.groovy” in the Fiji.app/macros/toolsets/DCVpHluorin folder.</li>
</ol>
</li>
</ol>
<ol start="15">
<li>Open a time-lapse file in Fiji and manually inspect it for focus drift, bleaching, and stage movements. Open the ROI Manager and start the analysis of DCV fusion events via selecting “Fusion detection” under the “Start Analysis Menu,” and adjust the parameters as follows (<a href="#fig3">Figure 3</a>D):
<ol type="a">
<li>Movie normalization method. “B&amp;W Opening” is set as default, use “Baseline subtraction” for noisy images.</li>
<li>Set the “Baseline frames” and “Start of NH4 (frame)” according to your time-lapse acquisition settings. In this case 30 and 160, respectively.</li>
<li>Select the “Number of stimulations” and the “Stimulation time (in frames)” accordingly. For more accurate measurements, it is recommended to round the duration of the stimulation period to the closest multiple of 10. For example, the 16 times 50 APs at 50 Hz last for 24 s (48 frames), so the “Stimulation time (frame)” is set to 50 frames. The program will then ask to insert the “Start of the stimulation (frame)” at a later stage.
<div class="note">
<span class="note-title">Note:</span> in this case, the 16 times 50 APs at 50 Hz is considered as 1 stimulation. However, it is possible to split one stimulation in multiple blocks. Dividing the stimulation in sub-stimulations improves the detection but is more computationally intensive, thus slower and prone to memory errors.</div>
</li>
<li>Adjust the “Smoothing radius” to improve the signal difference between background and foreground, and to reduce scatter noise.</li>
<li>Enter the value for the “SNR” (signal to noise evaluation) as a number between 0 and 255, indicating the difference in signal between foreground and background.
<div class="note">
<span class="note-title">Note:</span> it is possible to manually determine the value of the SNR by selecting “Evaluate SNR.” This will prompt the image used to detect the potential location of the DCV fusion events. Run the Fiji “Find Maxima” function with “Preview point selection” and adjust the value of the “prominence” until the majority of puncta are selected without selecting background noise. Close the “Find Maxima” and report the number on the “Evaluate SNR” dialog. The program will then restart.</div>
</li>
<li>If the “Baseline subtraction” method was selected, insert a desired “Detection threshold” as number of standard deviations above the mean intensity signal. Only pixels with higher intensity values will be considered for ROI placement.</li>
<li>Enter a desired “Cleaning threshold” as number of standard deviations above the mean intensity signal. This threshold is calculated as a rolling standard deviation with the value of “Baseline frames” as window size.</li>
<li>Adjust the “ROI size” to match the observed size of the DCV fusion events.</li>
<li>To avoid multiple detection of the same fusion event, set the “Maximum pxs overlap between ROIs” to be at most 1/3 of the total pixel area.</li>
</ol>
</li>
</ol>
<ol start="16">
<li>Evaluate the outcome of the detection by playing the time-lapse video. To visualize all detected regions, make sure that the “Show All” option in the ROI Manager is selected.
<ol type="a">
<li>Add any missing regions with the “Place ROIs tool;”
<div class="note">
<span class="note-title">Note:</span> After the initial fusion detection, the program will name the ROI in the format of ROIID-EventStart, where ROIID is a 4 digits identifier which starts at “0000,” and the EventStart is the frame number where the signal rose above the detection threshold. For manually placed events the number corresponds to the frame selected by the user. EventStart can be repeated multiple times.</div>
</li>
<li>Regions can be deleted at this step with “Alt + click” inside a ROI.
<div class="optional">
<span class="optional-title">Optional:</span> The user can evaluate each ROI intensity and detection via the “Check ROIs” in the “ROIs Interaction Menu” (<a href="#fig5">Figure 5</a>A). The program will report the Intensity over Time (in frames), with the start of the DCV fusion event indicated with a red circle. Here the user can adjust the detected frame, remove, or add DCV fusion events.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/454-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. DCV_pHluorin example and result tables</div>
<p>(A) Example Fiji interface after running the detection of DCV fusion events. In red, Fiji main interface. In yellow, the time-lapse window, the arrowhead indicates the currently selected ROI. In light-green the ROI Manager, each ROI is saved as a four digits identifier, an “-” and the fusion start frame number. In dark-green the dialog to modify the current event, from the ROIs Interaction menu &gt; Check ROIs. In light blue the intensity over time plot of the selected ROI, the red circle indicates the start of the fusion event.</p>
<p>(B) Explanation of the “Vesicle Parameters” result table.</p>
<p>(C) Explanation of the “Pool Values” result table.</p>
</figcaption>
</figure>
<ol start="17">
<li>Save the ROI Manager file via the “Save ROIs” in the “ROIs Interaction Menu.” Alternatively, the intensity of the individual ROIs can be calculated via the “Measure ROIs” function, which will report a comma separated file with the different ROIs as columns and the time-lapse frames as rows. This file can be stored for further use.</li>
<li>Once the DCV fusion events are detected and the ROI Manager file is saved, calculate the total number of pHluorin-labeled DCVs by doing the following:
<ol type="a">
<li>Make sure that the ROI Manager is open and empty, without any selections on the image;</li>
<li>On the “Start Analysis Menu” select “Get NH4 pool.” An option dialog will appear with the same options as in step 15.</li>
<li>The important parameters are the “Start of NH4 (frame)” and “SNR.” If the value of the SNR is set to 0, the program will calculate the SNR value based on the intensity during NH<sub>4</sub>Cl superfusion. Alternatively, it is possible to manually determine the SNR value selecting “Evaluate SNR” as in step 15e.</li>
</ol>
</li>
<li>Evaluate the outcome of the detection by observing both the pHluorin response in the time-lapse, as well as on the “MAX” figure, which is the normalized maximum intensity projection used to detect the pHluorin positive DCVs. The program will automatically save the ROI Manager file and the list of ROI intensities in the same folder as the image file.</li>
<li>If synapsin-mCherry was used to identify synapses it is possible to create a synapse mask with the following steps:
<ol type="a">
<li>Duplicate the image and invert the gray values, in order to have a black object on white background;</li>
<li>Perform a morphological opening by applying a minimum filter, a maximum filter, and a Gaussian blur filter of the same radius, preferably of roughly the same size as the synapses in pixels;</li>
<li>Subtract the morphological opened image from the original and threshold the result using the “Triangle” method from Fiji “Auto threshold;”</li>
<li>Manually inspect the synapse mask and delete any extra objects.</li>
</ol>
</li>
<li>Once the detection for all recordings is completed, collect all the data via the “ROIs Frame readers Menu” and select “From Folder.” A dialog will appear asking the “baseline frames” used to calculate the baseline intensity level to estimate the DCV fusion duration, the start of the NH<sub>4</sub>Cl superfusion, and whether to calculate the “Synaptic location” of the fusion events, and the estimation of the total amount of pHluorin-labeled DCVs (“Pool Estimation”). This will result in a table with one entry per single DCV fusion event, and one table with three estimations of the total DCV pool (<a href="#fig5">Figure 5</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During data collection the program will look for the ROI Manager files of the DCV fusion events, then it will automatically calculate the DCV fusion duration and the DCV fusion intensity. If “Synaptic location” was selected, the program will open the synapse mask associated to the time-lapse and calculate the distance between the center of the DCV fusion event to the closest synaptic area. For better performance it is convenient to organize the data so that all time-lapse files and their ROI Manager files are in one single folder. Another folder will contain the NH<sub>4</sub>Cl response csv files and the associated ROI Manager files, and a third folder with the images of the synapse mask.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A single hippocampus will yield between 1.5–3.0 × 10<sup>5</sup> neurons/mL, so dissecting half a brain is enough for one experiment. After 2 days in culture (DIV2), neurons attached to the astrocyte microdots forming neurites can be observed (<a href="#fig1">Figure 1</a>B). At DIV9, neurons should have a visible, complex dendritic arborization populating roughly 2/3 of the astrocyte microdot. The astrocyte microdot islands should be round with smooth edges and fully populated with astrocytes. Some microdot islands will have no neurons or harbor multiple neurons. For quantitative DCV fusion analysis select the island harboring one single neuron.</p>
<p>During a typical Fluo-5 based live-cell imaging experiment, neurons should have a diffuse, low fluorescence signal, just above the signal from the underlying astrocytes. During stimulation, the signal should uniformly increase in the neuron, with the soma as the brightest spot, indicating Ca<sup>2+</sup> influx into the cell. In response to neurotransmitter release from the neuron, it is possible to observe a wave of increase in fluorescence from the underlying astrocytes. Ca<sup>2+</sup> efflux from the neurons should be fast, and visible for all of the 16 trains of APs, reaching a 70%–80% fluorescence intensity decay at the end of the stimulation.</p>
<p>For NPY-pHluorin based live-cell imaging experiments, the neurons will have a faint diffuse fluorescence signal that distinguishes the neurites from the underlying astrocytes. The soma will have a higher signal intensity due to the NPY-pHluorin molecules transiting through the <i>trans-</i>Golgi network, which is not acidic. Some non-acidic, pHluorin-labeled vesicles can be observed, which are trafficking along the neurites, as well as some bright puncta that could reflect pHluorin accumulation at the plasma membrane, the extracellular matrix, or in the astrocytes. During stimulation, DCV fusion events should be visible during the recordings, however depending on the intensity scaling of the recording preview, this signal might not directly be visible by eye. During Tyrode’s NH<sub>4</sub>Cl superfusion, the entire DCV population should be visible within 1 s from the opening of the pinch valve with vesicles moving along the neurites.</p>
<p>Using the 16 times 50 APs at 50 Hz stimulation paradigm, the expected number of DCV fusion events per cell is between 30 and 200, however it is possible to observe numbers up to 1,000 and below 10. The intensity of the individual fusion events, a measure of the amount of NPY-pHluorin inside the vesicle, ranges between 1.5 and 3 ΔF/F<sub>0</sub> units above the baseline.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">Data exclusion criteria</h3>
<ol>
<li>Neurons should respond to electrical stimulation with a sharp rise in intracellular calcium levels that should decrease rapidly after stimulation back to baseline levels. Based on our experience, if more than 30% of the tested neurons do not show Ca<sup>2+</sup> influx or efflux upon electrical stimulation, it is recommended to discard the culture. Ca<sup>2+</sup> is the essential trigger for DCV fusion (<a href="#bib4">Hartmann et al., 2001</a>; <a href="#bib3">de Wit et al., 2009</a>). Therefore, insufficient Ca<sup>2+</sup> influx or efflux contributes to variation in DCV fusion numbers. In addition, maintaining a proper Ca<sup>2+</sup> concentration gradient is essential for neuronal health (<a href="#bib1">Brini et al., 2014</a>).</li>
<li>Individual time-lapse recordings are excluded from quantification If:
<ol type="a">
<li>There are no detectable puncta after Tyrode’s NH<sub>4</sub>Cl superfusion. Neurons typically contain between a 1,400 and 18,000 NPY-pHluorin-labeled DCVs (<a href="#bib9">Persoon et al., 2018</a>) that appear as bright, ± 0.6 μm (3 px) puncta during Tyrode’s NH<sub>4</sub>Cl superfusion.</li>
<li>There is noticeable focus drift.</li>
<li>Cell debris, or other fluorescence objects are hovering on top of the neuron.</li>
<li>There is a large number of non-acidic, already bright, pHluorin-labeled vesicles in the neuron.</li>
<li>DCVs are not visibly trafficking upon Tyrode’s NH<sub>4</sub>Cl superfusion.</li>
</ol>
</li>
<li>For correct total DCV pool estimates it is important that pixels (outside the soma) are not saturated upon NH<sub>4</sub>Cl superfusion. Time-lapse recordings are excluded if this is the case.</li>
</ol>
<h3 id="sec5.2">DCV fusion events data extraction</h3>
<ol start="4">
<li>Import the data generated with DCV_pHluorin into a computational software, such as MATLAB, R, or Python.</li>
<li>The “Fusion Start” value corresponds to the frame where a DCV fusion event was detected. To get the time in s of the start of the fusion event use (<i>FusionStart</i> - 1)/<i>F</i> where F ≡ Imaging frequency. Then, generate a histogram of the DCV fusion events per cell.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use an adequate bin size to represent the data. The ultimate goal is to avoid having too many 0 data points per bin. For 2 Hz data, a binning of 0.5 s or 1.5 s is recommended. This is because with 0.5 s the binning is per frame, while with 1.5 s the binning spans from one train of APs to the next one.</div>
<ol start="6">
<li>Calculate the cumulative count based on the histogram.</li>
<li>Extract the total number of DCV fusion events per cell and normalize it to the total DCV pool to obtain the DCV Release Fraction. DCV_pHluorin offers 3 different pool calculations, “Raw Pool,” “Fused Corrected,” and “Puncta Corrected.” It is important to carefully choose which one to use based on the overall quality of the signal during Tyrode’s NH<sub>4</sub>Cl superfusion. If the intensity per individual DCV fusion event is comparable to the intensity of single DCV puncta during Tyrode’s NH<sub>4</sub>Cl superfusion, then the “Fused Corrected” offers the best approximation of the number of pHluorin-labeled DCVs. If the intensity per individual DCV fusion event is lower than single DCV puncta during Tyrode’s NH<sub>4</sub>Cl superfusion, then it is suggested to use the “Puncta Corrected.”</li>
<li>Each individual neuron is considered as n = 1 for statistical evaluation. It is recommended to analyze at least 30 neurons derived from at least 3 independent cultures.</li>
<li>The different conditions are evaluated according to statistical standards. If the data is normally distributed, the difference in mean is evaluated using a Student t-test for experiments evaluating 2 conditions, or analysis of variance (ANOVA) followed by Tukey’s <i>post-hoc</i> correction for evaluating 3 or more conditions. If the data is not normally distributed the equivalent nonparametric tests, Kruskal-Wallis test and Mann Whitney U test, are used.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> It is possible to normalize to the total number of fusion events to observe differences in fusion kinetics between different conditions. DCV_pHluorin also generates several parameters which might be of interest, such as DCV fusion event duration, DCV fusion event location (synaptic or extra-synaptic), and the distance of the DCV fusion event from the closest synaptic location and the closest other DCV fusion location.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is optimized for the quantification of the number of DCV fusion events, their duration, and localization, in isolated neurons. Because of the heterogeneity of the neuronal cell types present in the culture, detecting small (&lt;20%) effects might be challenging. Therefore, it is recommended to perform a power analysis to calculate the number of independent experiments required to obtain sufficient power. In our experience, a significant difference of 30% or more requires 30 neurons from 3 independent cultures. Selection of a specific neuronal cell type by expressing DCV fusion reporters under a specific promoter, such as somatostatin, parvalbumin (for GABAergic neurons), or CaMKII (for mature glutamatergic neurons) may reduce variability.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Neurons do not respond to electrical stimulation (step 8 in <a href="#step-by-step-method-details">Step-by-step method details</a>).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>First, make sure that the electrodes are ∼ 1 mm from the surface of the coverslip and that they are surrounding the neuron in the field of view. If neurons are still not responding to electrical stimulation, calibrate the stimulus strength required to depolarize the membrane. To do so, label neurons with the calcium indicator Fluo5-AM and stimulate with 1 ms pulses starting at 100 mA, and decrease the current until the minimum current is reached that reliably elicits Ca<sup>2+</sup> influx during 1 ms pulses.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>No DCV fusion upon stimulation (step 13 in <a href="#step-by-step-method-details">Step-by-step method details</a>).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>DCV fusion is variable between neurons. Therefore image at least 10 neurons before concluding that there is no DCV fusion upon stimulation. It is also important to exclude low expression of the reporter, by carefully inspecting the response to Tyrode’s NH<sub>4</sub>Cl application. The response should be fast, reaching a maximum intensity within 1 s after the application, and the entire neurite arbor should contain labeled vesicles. It is also important to notice that high expression levels of the reporter could cause more DCVs to be less acidic. This will also affect the number of DCV fusion events upon stimulation. Therefore, it is suggested to titer the number of lentivirus particles to add to the culture as the minimum amount that reliably infects ∼90% of the neurons on a coverslip.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>DCV_pHluorin detects the same DCV fusion event in multiple ROIs (step 16 in <a href="#step-by-step-method-details">Step-by-step method details</a>).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>It is possible to observe multiple DCV fusion events less than 1 px apart in regions where the synapse density is higher, or with multiple neurite crossings. To avoid detection of the same DCV fusion event in multiple ROIs, it is recommended to set the “Maximum pxs overlap between ROIs” to be at most 1/3 of the total area of the ROIs. In case of 3 × 3 px ROIs, the “Maximum pxs overlap between ROIs” would then be of 3 pxs. By doing so, even if one DCV fusion event might span multiple ROIs, only the ROIs that contains all the pixels associated with the DCV fusion event will have a mean intensity value high enough to be detected. It is recommended to manually inspect potential overlap of ROIs via the “Check ROIs” function, especially in fusion dense areas.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Non-homogeneous fluorescence signal intensity in different regions of the neurite arbor (steps 10, 13, and 15 in <a href="#step-by-step-method-details">Step-by-step method details</a>).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Different signal intensities across different regions of the neurite arbor could be caused by a miss-aligned illumination path. To test if this is the case, it is recommended to visualize a uniformly distributed protein, such as cytosolic GFP. The problem should be solved by aligning the light path.</p>
<p>If the uneven illumination, or signal intensity, leads to overrepresentation of the higher signal areas, it is recommended to adjust the “Movie normalization” settings. By using the “Baseline subtraction” Fiji will calculate the average intensity value of the first X frames (X set by user in “Baseline Frames” option). It will subtract this average intensity value from all frames. This method works well for noisy images, but it is sensitive to uneven signal intensities, thus favoring regions with high signal.</p>
<p>An additional option for "movie normalization" is the "B&amp;W opening" option. This uses the information of the pixels surrounding the pixels of interest by a value “Smoothing radius” to first perform a maximum filter, then a minimum filter, and lastly a Gaussian Blur filter. Since the “B&amp;W Opening” scales pixels based on local information compared to the entire image, it is more suitable for images with uneven illumination or signal intensity. In addition to both normalization methods, the detection is done on the standard deviation of the signal over time. Using the standard deviation ensures that high intensity regions are not overrepresented compared to low intensity regions.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Non-acidic structures are visible before NH<sub>4</sub>Cl perfusion (step 13 in <a href="#step-by-step-method-details">Step-by-step method details</a>).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>As stated in problem 2, too high expression levels of the DCV reporter construct could cause DCVs to be less acidic. In addition, when designing new DCV reporter constructs by tagging other neuropeptides with pHluorin it is possible to observe mistargeting to other organelles. For example, mistargeting to reticular or tubular structures has been observed for NPY tagged with EGFP (<a href="#bib7">Nagai et al., 2002</a>). To test if your DCV reporter of interest targets to DCVs it important to first validate the sequence of your construct. Next, it is recommended to verify efficient targeting to DCVs by co-immunolabeling with endogenous DCV markers (<a href="#bib9">Persoon et al., 2018</a>). If your reporter construct is indeed mistargeting it is recommended to try and reduce the construct length to a minimal functional construct (for example only containing the signal peptide and N-terminal domain of the pre-propeptide) and subsequently validate the correct targeting.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Brini, M., Calì, T., Ottolini, D., and Carafoli, E. (2014). Neuronal calcium signaling: function and dysfunction. Cell. Mol. Life Sci. <i>71</i>, 2787-2814. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref1">View at publisher</a></p>
<p id="bib2">Burgalossi, A., Jung, S., Man, K.M., Nair, R., Jockusch, W.J., Wojcik, S.M., Brose, N., and Rhee, J.S. (2012). Analysis of neurotransmitter release mechanisms by photolysis of caged Ca2+ in an autaptic neuron culture system. Nat. Protoc. <i>7</i>, 1351-1365. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref2">View at publisher</a></p>
<p id="bib3">de Wit, J., Toonen, R.F., and Verhage, M. (2009). Matrix-dependent local retention of secretory vesicle cargo in cortical neurons. J. Neurosci. <i>29</i>, 23-37. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref3">View at publisher</a></p>
<p id="bib4">Hartmann, M., Heumann, R., and Lessmann, V. (2001). Synaptic secretion of BDNF after high-frequency stimulation of glutamatergic synapses. EMBO J. <i>20</i>, 5887-5897. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref4">View at publisher</a></p>
<p id="bib5">Kaeser, P.S., Deng, L., Wang, Y., Dulubova, I., Liu, X., Rizo, J., and Südhof, T.C. (2011). RIM proteins tether Ca<sup>2+</sup> channels to presynaptic active zones via a direct PDZ-domain interaction. Cell <i>144</i>, 282-295. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref5">View at publisher</a></p>
<p id="bib6">Kaeser, P.S., Kwon, H.B., Chiu, C.Q., Deng, L., Castillo, P.E., and Südhof, T.C. (2008). RIM1α and RIM1β are synthesized from distinct promoters of the RIM1 gene to mediate differential but overlapping synaptic functions. J. Neurosci. <i>28</i>, 13435-13447. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref6">View at publisher</a></p>
<p id="bib7">Nagai, T., Ibata, K., Park, E.S., Kubota, M., Mikoshiba, K., and Miyawaki, A. (2002). A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications. Nat. Biotechnol. <i>20</i>, 87-90. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref7">View at publisher</a></p>
<p id="bib8">Naldini, L., Blömer, U., Gallay, P., Ory, D., Mulligan, R., Gage, F.H., Verma, I.M., and Trono, D. (1996). In vivo gene delivery and stable transduction of nondividing cells by a lentiviral vector. Science <i>272</i>, 263-267. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref8">View at publisher</a></p>
<p id="bib9">Persoon, C.M., Moro, A., Nassal, J.P., Farina, M., Broeke, J.H., Arora, S., Dominguez, N., van Weering, J.R., Toonen, R.F., and Verhage, M. (2018). Pool size estimations for dense-core vesicles in mammalian CNS neurons. EMBO J. <i>37</i>, e99672. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref9">View at publisher</a></p>
<p id="bib10">Persoon, C.M., Hoogstraaten, R.I., Nassal, J.P., van Weering, J.R., Kaeser, P.S., Toonen, R.F., and Verhage, M. (2019). The RAB3-RIM pathway is essential for the release of neuromodulators. Neuron <i>104</i>, 1065-1080. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref10">View at publisher</a></p>
<p id="bib11">Schlüter, O.M., Schmitz, F., Jahn, R., Rosenmund, C., and Südhof, T.C. (2004). A complete genetic analysis of neuronal Rab3 function. J. Neurosci. <i>24</i>, 6629-6637. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref11">View at publisher</a></p>
<p id="bib12">van de Bospoort, R., Farina, M., Schmitz, S.K., de Jong, A., de Wit, H., Verhage, M., and Toonen, R.F. (2012). Munc13 controls the location and efficiency of dense-core vesicle release in neurons. J. Cell Biol. <i>199</i>, 883-891. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00032-0/sref12">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Ruud F. Toonen (<a href="mailto:ruud.toonen@cncr.vu.nl">ruud.toonen@cncr.vu.nl</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents. Plasmids generated in this study are available upon request.</p>
<h4>Data and Code Availability</h4>
<p>The DCV_pHluorin code is available in GitHub at <a href="https://github.com/alemoro">https://github.com/alemoro</a></p>
<h3>Acknowledgments</h3>
<p>The authors thank Robbert Zalm for cloning and producing lentiviral particles, Desiree Schut and Lisa Laan for astrocyte and neuronal cultures, Joke Wortel for animal breeding, Joost Hoetjes for genotyping, and members of the CNCR DCV project team for fruitful discussions. This work is supported by an <a href="https://doi.org/10.13039/501100000781">ERC</a> Advanced Grant (322966) of the European Union (to M.V.)</p>
<h3>Author Contributions</h3>
<p>R.I.H. and A.M. acquired data. R.I.H., A.M., C.M.P., M.V., and R.F.T. designed figures and wrote the protocol.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>