<article>
<section>
<h1 id="header">Extracting meaningful circuit-based calcium dynamics in astrocytes and neurons from adult mouse brain slices using single-photon GCaMP imaging</h1>
<p><time datetime="2021-02-05">Published: February 5, 2021</time></p>
<p>Brent Asrican<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> and Juan Song<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pharmacology, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</p>
<p id="aff2"><sup>2</sup>Neuroscience Center, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:basrican@email.unc.edu">basrican@email.unc.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:juansong@email.unc.edu">juansong@email.unc.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100306">10.1016/j.xpro.2021.100306</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Confocal, multiphoton, or other advanced microscopy techniques produce high-quality datasets of calcium activity in live tissue. However, researchers without access to such expensive equipment can still produce meaningful observations from single-photon datasets. Here, we describe a protocol to extract meaningful features of both somatic neuronal and membranous astrocytic calcium dynamics obtained from charge-coupled device (CCD)-based camera setups, typical of electrophysiology rigs and highly relevant for investigating neuronal and astrocytic involvement in brain circuitry.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Asrican et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>
<i>In vivo</i> expression of Ca<sup>2+</sup> indicator GCaMP in astrocytes and neurons in mouse brain</li>
<li>Camera-based single-photon methodologies can provide meaningful calcium data</li>
<li>Region-based analyses extract relevant neuronal and astrocytic feature sets</li>
<li>Protocol is applicable without access to confocal or multiphoton microscopy</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/446-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Neuronal calcium has long been associated with the levels of activity, and recording somatic calcium is a reasonable proxy for action potential frequency. However, in astrocytes, recent indications are that calcium signaling in the processes may be functionally more relevant to synaptic functions than the dynamics of the astrocyte soma (<a href="#bib4">Bindocci et al., 2017</a>; <a href="#bib6">Khakh and McCarthy, 2015</a>). Here, we use AAV delivery of GCaMP6 or GCaMP7 constructs in genetically defined neuronal populations, or use a LCK-tagged GCaMP6 construct (<a href="#bib10">Shigetomi et al., 2010</a>) to target the plasma membrane of astrocytes. Along with delivery of floxed chemogenetic DREADDs (<a href="#bib1">Armbruster et al., 2007</a>), we stimulate neuronal circuitry in CCK-Cre mice, and use a scientific-grade complementary metal-oxide-semiconductor (sCMOS) camera to acquire fluorescence datasets in hippocampal CCK interneurons, granule cells, mossy cells, or astrocytes.</p>
<h3 id="sec1.1">Viral injection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks prior</div>
<ol>
<li>Acquire male and female recombinant mice of about 2–6 months of age. CCK-Cre mice (Jax) will be used in this example to demonstrate stimulation of local CCK interneurons and imaging of neuronal and astrocyte calcium.</li>
<li>AAVs are available from commercial sources, and can be injected into brain regions of interest, such as into the hilus of the dentate gyrus.
<ol type="a">
<li>Obtain and mix viral aliquots. Typical stocks from Addgene are on the order of 1 × 10<sup>12</sup> to 1 × 10<sup>13</sup> viral genomes/mL.
<ol type="i">
<li>AAV2-hSyn-DIO-hM3Dq-mCherry (Addgene) is a Cre-dependent version of an excitatory DREADD virus, which in our case will be expressed in CCK type interneurons for circuit stimulation. Aliquot and store at −80°C.</li>
<li>AAV2/5-GFAP-GCaMP6-LCK (Addgene), uses the GFAP promoter to restrict expression of the GCaMP6 indicator to astrocyte cell types. Aliquot and store at −80°C.</li>
<li>AAV9-Syn-DIO-jGCaMP7f (Addgene) is a Cre-dependent version of the newer version of GCaMP, which in our case will be expressed in CCK type interneurons. We have found that GCaMP7 works better than GCaMP6 in these cells. Aliquot and store at −80°C.</li>
<li>AAVDJ-CaMKII-GCaMP6s (Stanford) expresses GCaMP6 in most glutamatergic neurons, and in dentate gyrus, and will express in granular cells as well as hilar mossy cells.</li>
</ol>
</li>
<li>Mix a small volume (∼5–10 μL) of <u>one</u> of the following combinations of AAVs: All vials must be kept on ice.
<ol type="i">
<li>
<u>DIO-hM3Dq with GFAP-GCaMP6-LCK</u>. Use a final concentration of 1.4 × 10<sup>11</sup> of the DREADD, and 3 × 10<sup>12</sup> of the GFAP-GCaMP-LCK virus in sterile PBS.</li>
<li>
<u>DIO-hM3Dq with DIO-GCaMP7</u>. Use a final dilution of concentration of 1.4 × 10<sup>11</sup> of the DREADD, and 1.3 × 10<sup>12</sup> of the DIO-GCaMP7 virus in sterile PBS.</li>
<li>
<u>DIO-hM3Dq with CaMKII-GCaMP6</u>. Use a final concentration of 1.4 × 10<sup>11</sup> of the DREADD, and 5 × 10<sup>11</sup> of the CaMKII-GCaMP6 virus in sterile PBS.</li>
</ol>
</li>
</ol>
</li>
<li>Inject viral constructs into CCK-Cre mice using a stereotaxis apparatus. A surgical scope with adjustable zoom, and digital display that reports coordinates of the stereotaxis arm is needed to properly target the injection.
<ol type="a">
<li>Prepare the animal.
<ol type="i">
<li>Anesthetize animals according to institutional guidelines. Inhalable isoflurane at regulated concentration and rate is recommended. Determine deep state of sleep by non-reactivity to a forceful toe pinch.</li>
<li>Affix the animal to stereotaxis apparatus that employs a bite plate and ear bars. Adjust the head position to be approximately straight and level. Continuous isoflurane should be administered by nose cone while in the stereotaxis apparatus. A heat pad under the animal can assist in maintaining body temperature. Breathing rate should be periodically observed, and adjust the anesthesia concentration as needed.</li>
<li>Shave, or use depilatory cream to remove hair on the scalp.</li>
<li>Follow institutional guidelines on sterile surgical techniques (autoclaving tools, betadine scrubbing, and ethanol cleaning of skin at surgical site).</li>
<li>Make a single ∼1.5 cm incision along the midline of the head to open the scalp and underlying layers, exposing the top of the skull. This should expose a region slightly larger than the area running from bregma to lambda.</li>
<li>Mount a motorized drill in a vertical position with a surgical bit on the stereotaxic arm. Approach the skull, and use the tip of the drill bit at the skull surface to confirm correct alignment and levelness of the animal. Adjust the animal position until:<br> Movement of the drill bit in “Y” direction follows the midline (yaw is correct).<br> The “Z” position at bregma and lambda are equivalent (pitch is level).<br> The “Z” position at the surface of the scull at deflections of about 2 mm left and right of the midline at a point midway between bregma and lambda are equivalent (roll is level).</li>
</ol>
</li>
<li>Deliver virus.
<ol type="i">
<li>Drill a small 1 mm burr hole in each hemisphere at AP −2.0 mm, ML ± 1.5 mm from Bregma.</li>
<li>Mount a micropipette pulled to a fine tip into a Hamilton syringe, and fill with corn oil. Micropipettes can be created by heating capillary glass on a micropipette forge. Tips can be snapped with fine forceps to a diameter that allows extrusion of liquid.</li>
<li>Remove drill attachment and replace with microinjection apparatus containing the Hamilton syringe.</li>
<li>Pull up &gt; 600 nL of virus into the tip of the micropipette using an injection pump.</li>
<li>Re-zero the coordinates at bregma, and approach burr holes. Inject virus using the following settings:<br> Position AP −2.0 mm, ML ± 1.5 mm, DV −2.1 mm.<br> Injection speed of 75 nL/min.<br> Injection volume of 300 nL per injection site.</li>
<li>Activate the pump to initiate the injection.<br> Wait 5 min after cessation of pump to reduce leakage.<br> Slowly retract the needle ∼0.1 mm.<br> Wait another 1 min before carefully retracting the rest of the way.</li>
</ol>
</li>
<li>Close the wound, allow animal to recover, and monitor postoperatively according to institutional requirements.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Confirm accuracy of viral injections. Occasionally combining two viruses has unexpected effects, and the ratios of the two must be adjusted to get desired expression levels. It is suggested to try a variety of dilutions to find the concentrations that work best. GCaMP expression can be detected in fixed tissue using standard antibody staining against GFP if necessary. Expression of hM3Dq can be confirmed by presence red fluorescence due to the mCherry reporter.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> An alternative to micropipette injection is to use an injection needle made for a Hamilton syringe (ex: 33GA). We have found, however, that appropriately small micropipettes induce less injury along the needle track due to the reduced diameter. Care must be taken to prevent clogging at the tip. Validate that the virus level is decreasing during the injection by observing movement of a small bubble at the interface of oil and virus.</div>
<h3 id="sec1.2">Prepare stock solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day prior</div>
<ol start="4">
<li>Healthy brain slices require careful preparation of cutting solutions, recovery solutions, and recording solutions. These stocks can be prepared ahead of time, and then adjusted for osmolarity and pH on the day of recording.
<ol type="a">
<li>Prepare 200 mL of 1 molar stocks of:
<ol type="i">
<li>MgSO<sub>4</sub>
</li>
<li>CaCl<sub>2</sub>
</li>
<li>MgCl<sub>2</sub>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Salt stocks can remain at 20°C–24°C for several months, but check for cloudiness or contamination. Remake as necessary.</div>
</li>
<li>Prepare the following solutions listed in the Materials and equipment section.
<ol type="i">
<li>10× HEPES stock</li>
<li>1× NMDG solution</li>
<li>1× HEPES solution</li>
<li>10× ACSF stock</li>
</ol>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>AAV2-hSyn-DIO-hM3Dq-mCherry</td>
<td>Addgene</td>
<td>RRID: AddGene_44361</td>
</tr>
<tr>
<td>AAV2/5-GFAP-GCaMP6-LCK</td>
<td>Addgene</td>
<td>RRID: AddGene_52924</td>
</tr>
<tr>
<td>AAVDJ-CaMKII-GCaMP6s</td>
<td>Stanford Vector Core</td>
<td>Cat# GVVC-AAV-88</td>
</tr>
<tr>
<td>AAV9-Syn-DIO-jGCaMP7f-WPRE</td>
<td>Addgene</td>
<td>RRID: AddGene_104488</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>HEPES</td>
<td>Sigma</td>
<td>Cat# H3375</td>
</tr>
<tr>
<td>N-Methyl-D-glucamine (NMDG)</td>
<td>Sigma</td>
<td>Cat# M2004</td>
</tr>
<tr>
<td>Sodium ascorbate</td>
<td>Sigma</td>
<td>Cat# A7631</td>
</tr>
<tr>
<td>Thiourea</td>
<td>Sigma</td>
<td>Cat# T8656</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>Sigma</td>
<td>Cat# P5280</td>
</tr>
<tr>
<td>N-Acetyl-L-cysteine</td>
<td>Sigma</td>
<td>Cat# A7250</td>
</tr>
<tr>
<td>CNO</td>
<td>NIH</td>
<td>Cat# C-929</td>
</tr>
<tr>
<td>TTX</td>
<td>Tocris</td>
<td>Cat# 1069</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: CCK-IRES-Cre</td>
<td>Jackson Laboratory</td>
<td>RRID: IMSR_JAX:012706</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Micromanager</td>
<td>Micro-manager.org</td>
<td>RRID: SCR_000415</td>
</tr>
<tr>
<td>FIJI</td>
<td>Image J</td>
<td>RRID: SCR_002285</td>
</tr>
<tr>
<td>Igor Pro</td>
<td>Wavemetrics</td>
<td>RRID: SCR_000325</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Small animal stereotaxis apparatus</td>
<td>Kopf</td>
<td>Cat# Model 942</td>
</tr>
<tr>
<td>Stereo zoom microscope</td>
<td>Leica</td>
<td>Cat# M80</td>
</tr>
<tr>
<td>Rotary drill</td>
<td>Foredom</td>
<td>Cat# MH-170</td>
</tr>
<tr>
<td>Capillary glass</td>
<td>WPI</td>
<td>Cat# 1B100-4</td>
</tr>
<tr>
<td>Microforge (microelectrode puller)</td>
<td>Narishige</td>
<td>Cat# PC-10</td>
</tr>
<tr>
<td>Infusion pump</td>
<td>Kd Scientific</td>
<td>Cat# 78-8130</td>
</tr>
<tr>
<td>Gastight syringe 5 μL</td>
<td>Hamilton</td>
<td>Cat# 80016</td>
</tr>
<tr>
<td>33 gauge needle (optional)</td>
<td>Hamilton</td>
<td>Cat# 7762-06</td>
</tr>
<tr>
<td>Osmometer</td>
<td>Wescor</td>
<td>Cat# Vapro 5520</td>
</tr>
<tr>
<td>pH meter with temperature probe</td>
<td>Fisher</td>
<td>Cat# Accumet AE150</td>
</tr>
<tr>
<td>Semiautomatic vibratome slicer</td>
<td>Leica</td>
<td>Cat# VT1200S</td>
</tr>
<tr>
<td>SliceScope recording microscope</td>
<td>Scientifica</td>
<td>Cat# SliceScope Pro 1000</td>
</tr>
<tr>
<td>40× water fluorescence objective</td>
<td>Olympus</td>
<td>Cat# LUMPLFLN40XW</td>
</tr>
<tr>
<td>RC-27LD ultra quiet imaging chamber</td>
<td>Warner</td>
<td>Cat# 64-1532</td>
</tr>
<tr>
<td>Slice anchor</td>
<td>Warner</td>
<td>Cat# 64-1551</td>
</tr>
<tr>
<td>Accuflow flow regulator infusion set</td>
<td>EMED Technologies</td>
<td>Cat# FLOW-100</td>
</tr>
<tr>
<td>Optimos sCMOS camera</td>
<td>Q-Imaging</td>
<td>Cat# 01-OPTIMOS-R-M-16-C</td>
</tr>
<tr>
<td>Blue LED (470 nm)</td>
<td>CoolLED</td>
<td>Cat# pE-100</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>10× HEPES stock (store at 4°C; lasts about 1 month)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final (1×) concentration (mM)</th>
<th>10× concentration (mM)</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>KCl</td>
<td>2.5</td>
<td>25</td>
<td>1.86 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1.25</td>
<td>12.5</td>
<td>1.50 g</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>30</td>
<td>300</td>
<td>25.2 g</td>
</tr>
<tr>
<td>HEPES</td>
<td>20</td>
<td>200</td>
<td>47.6 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
<td>Add to 1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>1× NMDG solution (for cutting; store at 4°C; lasts about 1 week). NMDG solution reduces excitotoxicity during the slicing procedure by replacement of sodium ions (<a href="#bib12">Ting et al., 2014</a>)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock (mM)</th>
<th>Final concentration (mM)</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× HEPES stock</td>
<td>10×</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>NMDG</td>
<td>–</td>
<td>92</td>
<td>17,960 mg</td>
</tr>
<tr>
<td>Glucose</td>
<td>–</td>
<td>25</td>
<td>4,505 mg</td>
</tr>
<tr>
<td>Sodium ascorbate</td>
<td>–</td>
<td>5</td>
<td>990 mg</td>
</tr>
<tr>
<td>Thiourea</td>
<td>–</td>
<td>2</td>
<td>152.2 mg</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>–</td>
<td>3</td>
<td>330 mg</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>
</td>
<td>1,000</td>
<td>10</td>
<td>10 mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>1,000</td>
<td>0.5</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
<td>∗Add to 900 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> NMDG solution comes out very alkaline. Adjust the stock to about pH 7.5–8 using HCl. This typically requires about 10–12 mL of 6 N HCl. Final adjustments to pH will be made on day of recording.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare NMDG slightly overconcentrated (bring ddH<sub>2</sub>O to about 900 mL when making this stock). Final adjustments to volume will be made on the day of recording based on osmolarity.</div>
<table id="undtbl3">
<caption>1× HEPES Holding (for recovery. Store at 4°C; lasts about 1 week) (<a href="#bib12">Ting et al., 2014</a>)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock (mM)</th>
<th>Final concentration (mM)</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× HEPES stock</td>
<td>10×</td>
<td>–</td>
<td>100 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>–</td>
<td>92</td>
<td>5,372 mg</td>
</tr>
<tr>
<td>Glucose</td>
<td>–</td>
<td>25</td>
<td>4,505 mg</td>
</tr>
<tr>
<td>Sodium ascorbate</td>
<td>–</td>
<td>5</td>
<td>990 mg</td>
</tr>
<tr>
<td>Thiourea</td>
<td>–</td>
<td>2</td>
<td>152.2 mg</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>–</td>
<td>3</td>
<td>330 mg</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>
</td>
<td>1,000</td>
<td>2</td>
<td>2 mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>1,000</td>
<td>2</td>
<td>2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
<td>∗Add to 900 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare HEPES Holding slightly overconcentrated (bring ddH<sub>2</sub>O volume to only 900 mL when making this stock). Final adjustments to volume will be made on the day of recording based on osmolarity.</div>
<table id="undtbl4">
<caption>10× ACSF Stock (for recording. Store at 4°C; lasts about 2 months).</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final (1×) concentration (mM)</th>
<th>10× concentration (mM)</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>125</td>
<td>1,250</td>
<td>73.05 g</td>
</tr>
<tr>
<td>KCl</td>
<td>2.5</td>
<td>25</td>
<td>1.863 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1.25</td>
<td>12.5</td>
<td>1.499 g</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>26</td>
<td>260</td>
<td>21.84 g</td>
</tr>
<tr>
<td>D-Glucose</td>
<td>20</td>
<td>200</td>
<td>36.03 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
<td>Add to 1 L</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Acute brain slice preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5–3 h</div>
<p>Preparing tissue for calcium imaging follows the exact same protocol as for electrophysiological recordings with the exception of the addition of 12 mM N-acetyl-L-cysteine (NAC) to the NMDG and HEPES holding solutions. This has the benefit of preserving glutathione production and improves calcium imaging in acute brain slices (<a href="#bib12">Ting et al., 2014</a>).</p>
<ol>
<li>Final preparation of slicing solutions. (∼ 25 min)
<ol type="a">
<li>Measure 180 mL of NMDG stock solution and 90 mL of HEPES holding solution.</li>
<li>Mix in 392 and 196 mg of NAC to NMDG and HEPES solutions respectively.</li>
<li>Bubble solutions with 95% O<sub>2</sub> and 5% CO<sub>2</sub> for 10 min.</li>
<li>pH the NMDG using HCl and/or KOH to 7.24</li>
<li>pH the HEPES solution using HCl and/or NaOH to 7.24.</li>
<li>Adjust osmolarity to 300 mOsm.
<ol type="i">
<li>Measure the slightly overconcentrated solutions with an osmometer</li>
<li>Reduce osmolarity by adding volumes of ddH<sub>2</sub>O.<br> For NMDG; volume needed (mL) ∼= (Osm × 180/300 – 180 ) × 0.9<br> For HEPES; volume needed (mL) ∼= (Osm × 90/300 – 90) × 0.9</li>
<li>Typically this will bring the final volume to ∼ 200 mL of NMDG, and 100 mL of HEPES.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Dissection and Sectioning. (∼ 45 min)
<ol type="a">
<li>Setup dissection tools and vibratome and water bath.
<ol type="i">
<li>Bring water bath to 35°C.</li>
<li>Assemble useful tools: including, large scissors (for removing head), small scissors (for cutting skin and skull), blunt forceps, spatula, single edge razor blade; tray and Styrofoam lid for pinning and dissection; syringe for cardiac perfusion.</li>
<li>Pack vibratome outer-chamber with ice. Fill inner-chamber with 100 mL of the ice-cold NMDG solution. Bubble with 95% O<sub>2</sub> and 5% CO<sub>2.</sub>
</li>
<li>Place remaining 100 mL of ice-cold NMDG in slice-holding chamber, and keep on ice. Bubble with 95% O<sub>2</sub> and 5% CO<sub>2</sub>. Reserve a small amount for cardiac perfusion.</li>
</ol>
</li>
<li>Anesthetize the research animal according to institutional requirements. Vaporized isoflurane will anesthetize the mice and slow the breathing rate. Animals should be alive, but non-responsive to serious toe pinch. An inverted 15 mL centrifuge tube containing a tissue moistened with additional isoflurane placed over the animal’s snout will ensure a continued anesthetized state during cardiac perfusion procedure.</li>
<li>Pin animal, expose heart, puncture right atrium and perfuse ∼10 mL of ice-cold NMDG solution into left ventricle.</li>
<li>Quickly remove head, open skull, and remove brain directly into the ice-cold NMDG. Let sit for 1 min.</li>
<li>Remove to a towel and bisect hemispheres. Block brain for semi-transverse sections (about 22.5 degrees from coronal, and angled slightly off vertical, inward/downward) with single edge razor blade.</li>
<li>Dab on towel, and glue the cut cerebellar side to the vibratome plate. Insert plate into vibratome.</li>
<li>Slice at 280 μm thickness, using speed ∼ 0.16–0.20 mm/s and amplitude of 1.6 mm. Slow sectioning while cutting through the hippocampus will preserve integrity of the hippocampal layers.</li>
<li>
<u>Gently</u> collect sections containing hippocampus into a slice holder containing ice-cold NMDG using inverted/cut transfer pipette, or gentle paintbrush.</li>
</ol>
</li>
<li>Warming. (∼ 10 min)
<ol type="a">
<li>Transfer the NMDG containing slice holder with sections from the ice, into the 35°C water bath.</li>
<li>Continue bubbling in the water bath for 8–10 min. Do not over-warm. It is recommended to shield from light to preserve fluorescence.</li>
</ol>
</li>
<li>Recovery. (∼ 1.5 h)
<ol type="a">
<li>Remove holder from water bath.</li>
<li>Carefully discard the NMDG solution, and replace with 20°C–24°C, bubbled HEPES holding solution. Ensure that sections are submerged and loosely separated. Continue bubbling at 20°C–24°C for 1–1.5 h. Take care that bubbling is not so vigorous as to cause turbulence/tumbling of the sections. Continue shielding from light with sheets of aluminum foil.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All materials and equipment involved in the preparation of acute brain slices <u>must not</u> have come in contact with fixative (paraformaldehyde or equivalent). Reserve glassware, dissection tools, and vibratome for “live tissue only.” Do not share equipment with lab personnel that use fixative or other harsh chemicals, as it will seriously compromise your tissue. It is recommended not to use detergents or use department dishwashers. Thorough rinsing in water and scrubbing with reserved glassware brush will suffice. Application of 70% ethanol may be used to keep bacteria from growing on tools or slice chambers.</div>
<div class="critical">
<span class="critical-title">Critical:</span> GCaMP indicators are light sensitive and will bleach if exposed to intense light. Slice in low light conditions if possible, and ensure that sectioned tissue is shielded from room illumination by covering water-baths and slice-storage chambers with aluminum foil during recovery. Brain slices from juvenile animals are typically healthier and offer good expression of GCaMP indicator. Tissue from aged animals may be less healthy, but this is not limiting. See <a href="#bib13">Ting et al. (2018</a>) for additional tips on preparing slices from aging animals.</div>
<h3 id="sec3.2">Calcium imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–7 h</div>
<p>The techniques to keep brain slices alive and heathy for calcium imaging are identical to those used for electrophysiological recordings. Illumination of tissue with a blue LED and collecting green fluorescence is done using a standard GFP filter-set.</p>
<p>The sCMOS camera used here is controlled via Micromanager software, which can also interface with the SliceScope microscope stage (Scientifica). Many popular microscope cameras, such as Hamamatsu or Photometrics cameras, use CCD or sCMOS sensors that offer high sensitivity and speed, and are excellent for detecting fluorescent calcium signals. Most devices are already compatible with the Micromanager software (<a href="https://micro-manager.org/wiki/Device_Support">https://micro-manager.org/wiki/Device_Support</a>).</p>
<ol start="5">
<li>Prepare the 1× ACSF recording solution. Calcium events are very numerous in astrocytes, while very sparse in granule cells. It is often beneficial to adjust the levels of Ca<sup>2+</sup> and Mg<sup>2+</sup> depending on which cells you will observe to slightly increase or decrease network excitability.
<ol type="a">
<li>Combine in ddH<sub>2</sub>O
<ol type="i">
<li>For Astrocytes:
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Stock (mM)</th>
<th>Final concentration (mM)</th>
<th>Amount for 500 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× ACSF stock</td>
<td>10×</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>1,000</td>
<td>1.8</td>
<td>900 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>1,000</td>
<td>1.5</td>
<td>750 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
<td>Add to 500 mL</td>
</tr>
</tbody>
</table>
</li>
<li>For granule cells and mossy cells:
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Stock (mM)</th>
<th>Final concentration (mM)</th>
<th>Amount for 500 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× ACSF stock</td>
<td>10×</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>1,000</td>
<td>2.2</td>
<td>1,100 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>1,000</td>
<td>1.1</td>
<td>550 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
<td>Add to 500 mL</td>
</tr>
</tbody>
</table>
</li>
<li>For CCK Cells:
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Stock (mM)</th>
<th>Final concentration (mM)</th>
<th>Amount for 500 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× ACSF stock</td>
<td>10×</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>1,000</td>
<td>2</td>
<td>1,000 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>1,000</td>
<td>1.3</td>
<td>650 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
<td>Add to 500 mL</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>Bubble for at least 10 min at 20°C–24°C with 95% O<sub>2</sub> and 5% CO<sub>2</sub>.</li>
<li>pH to 7.24.</li>
<li>Adjust osmolarity to ∼ 305 mOsm.</li>
<li>Begin perfusion of ACSF on the microscope, continuing to bubble, and use an IV dripper and flow regulator to control the perfusion rate to about 1–2 mL/min.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Identify regions to image.
<ol type="a">
<li>Place a single slice in upright microscope recording chamber. Secure with a slice anchor.</li>
<li>Locate the dentate gyrus with a 10× air objective.</li>
<li>Switch to a long-working distance water-dipping objective with 40× magnification. Confirm the presence of mCherry<sup>+</sup> virally targeted CCK neurons containing DREADD construct, and green GCaMP expression in neurons or astrocytes. GCaMP has a low baseline fluorescence, which can be observed as a diffuse green signal. Typically, spread of injected AAV2/5-GCaMP is not spatially restricted as tightly as AAV2-hM3Dq virus, and expression can be found in cells beyond the hilar region, presumably due to differences in capsid or infectability of different cell types. See <a href="#fig1">Figure 1</a> for typical live-fluorescence appearance of hM3Dq-mCherry and GCaMP expression in astrocytes or neurons in the dentate gyrus.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/446-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Typical fluorescence appearance in live tissue of hM3Dq-mCherry targeted CCK interneurons along with GCaMP constructs targeted to astrocytes, mossy cells/granule cells, or CCK interneurons from CCK-Cre animals</div>
<p>(Top) Diffuse and low basal level fluorescence in astrocytes due to the targeting of GCaMP6 construct to the fine processes with the LCK construct. (Middle) CaMKII-GCaMP6 targets glutamatergic cells of the DG including mossy cells and granule cells. (Bottom) CCK neurons expressing both hM3Dq and GCaMP7, with partial colocalization. GCL, granule cell layer; scale bar, 50 μm. See <a href="#bib2">Asrican et al. (2020)</a> for additional examples and resolutions.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Set up an imaging session that acquires data at two different XY stage positions and extends for several different depths into the slice. Adjust the number of cycles to last approximately 10 min.
<div class="note">
<span class="note-title">Note:</span> Calcium events in acute slice preparation can last several seconds at 20°C–24°C, depending on the specific GCaMP variant that is utilized (fast or slow) and the cell type. Since the frame-rates of most cameras greatly exceed the duration of these calcium events, some time resolution can be sacrificed in order to gain additional image planes. Adjustment of specific imaging parameters may be required for your application.</div>
The main limitation on time resolution of image acquisition is the refocusing and movement of the XY translational stage. The following parameters worked well for a Scientifica microscope and Micromanager software and gave an effective time resolution of 2.85 s. While some fast events may be missed, this is acceptable for detecting the majority of slow buffered calcium events.
<table id="undtbl8">
<thead>
<tr>
<th>Imaging settings</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td># of time points</td>
<td>210</td>
</tr>
<tr>
<td>frame interval (delay)</td>
<td>(minimum)</td>
</tr>
<tr>
<td># of Z-planes</td>
<td>5</td>
</tr>
<tr>
<td>Z-Step Size</td>
<td>15 μm</td>
</tr>
<tr>
<td># XY-positions</td>
<td>2</td>
</tr>
</tbody>
</table>
<ol type="a">
<li>Power the camera and stage and then start the Micromanager software. If not already done, setup and save a configuration file that can interact with:
<ol type="i">
<li>The CCD camera.</li>
<li>The XY stage position.</li>
<li>The Z focus position.</li>
</ol>
</li>
<li>Set camera parameters for 100 ms exposure with no binning.</li>
<li>Open the Multi-Dimensional Acquisition Panel in MicroManager.</li>
<li>Enable “Time Points,” and set the image count to 210 images, 1 ms interval.</li>
<li>Enable “Z-Stacks (Slices)” and Select “Relative Z” mode. Adjust settings to use 5 imaging planes by entering the following parameters:
<ol type="i">
<li>Enter 0 for “Start Z.”</li>
<li>Enter -60 for “End Z.”</li>
<li>Enter 15 for “Step size.”</li>
</ol>
</li>
<li>Disable “Channels” checkbox.</li>
<li>Enable “Multiple Positions (XY)”, and open the “Stage Position List.”</li>
<li>Start “Live View” and turn on the blue LED.</li>
<li>Move the XY Stage to the surface of the slice in a field of view of interest. Push “Mark” to store it as Position0.</li>
<li>Move the XY Stage to a second, non-overlapping region of the tissue. Mark it as Position1.</li>
<li>Stop Live View and turn off the LED.</li>
<li>Minimize room lights or cover the microscope cage with black-cloth. Allow the slice to equilibrate for 5–10 min to reduce mechanical drift. Recheck positioning to ensure stability of the slice.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Acquire calcium data.
<ol type="a">
<li>Once baseline drugs are added for 5–10 min (for example, 1 μM TTX) and slice is confirmed to be mechanically stable, turn on the blue LED at the lowest setting that provides reasonable signal. As the fluorescent signal does indeed bleach over time, and cellular activities may be susceptible to phototoxicity, it is critically important not to over-illuminate the tissue.</li>
<li>Push “Acquire!” to start collecting Data_1.</li>
<li>When complete, turn off the LED and briefly review the dataset. Save the data to disk with an appropriate filename.</li>
<li>Add CNO to the ACSF reservoir to begin circuit stimulation of CCK cells. Wait 10 min to allow drugs to reach the slice and begin to activate DREADD receptors.</li>
<li>When time has nearly expired, recheck the stage position and focus, and readjust the stored positions to match the beginning of Data_1 as needed.</li>
<li>Push “Acquire!” to start collecting Data_2, and save as above.</li>
<li>Wash out drugs and change the slice to continue with additional samples.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Include various pharmacological agents such as glutamate receptor antagonists in the bath starting at baseline to help determine mechanisms of action. For examples, see <a href="#bib2">Asrican et al., (2020</a>).</div>
<div class="optional">
<span class="optional-title">Optional:</span> Some researchers may wish to acquire calcium data during the 10 min delay in step 8d, or perhaps an additional 10 min period of washout. In this case there will be a total of 3–4 datasets from each slice.</div>
<div class="note">
<span class="note-title">Note:</span> A highly sensitive CCD camera is useful when imaging calcium, so that the intensity of the illumination light can be minimized. An Optimos sCMOS camera from Q-Imaging is used in this example. While not used here, TTL control of the LED, or addition of a mechanical shutter in the light path can prevent excitation of the sample during non-imaging episodes of the acquisition, such as during focusing or stage-translation.</div>
<div class="note">
<span class="note-title">Note:</span> Halogen lamps as illumination sources have some intrinsic fluctuations in intensity and result in signal disturbances when sensitive cameras are used. LEDs are typically much more reliable in terms of constant output and have the advantage of also permitting control via TTL signals.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Neurons and astrocytes respond to neuronal activity, and the resulting changes in frequencies and amplitudes of calcium signals can be quantified by region-based analysis algorithms. Datasets from Micromanager software are stored as OME-TIFFs using the open-source Bio Formats standard (<a href="#bib7">Linkert et al., 2010</a>) and can be opened using the FIJI extensions (<a href="#bib9">Schindelin et al., 2012</a>) in ImageJ. As datasets are very large (tens of GBs), a reasonably large and fast solid state drive (SSD) is needed to quickly store the data, and additional RAM may facilitate the analysis. Analysis here is accomplished with a combination of manual identification of responding regions in FIJI/ImageJ (see <a href="#fig2">Figure 2</a>), and quantification using custom procedures in Igor Pro. See <a href="#fig3">Figures 3</a> and <a href="#fig4">4</a> for representative traces.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<ol>
<li>Open in FIJI/ImageJ.
<ol type="a">
<li>Import a dataset in FIJI by using the File -&gt; Import -&gt; Bio Formats menu.</li>
<li>Select only one position to analyze a time.</li>
<li>Adjust brightness/contrast to enhance visibility of signal changes.</li>
</ol>
</li>
<li>Define regions of interest (ROIs). See <a href="#fig2">Figure 2</a>.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/446-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Drawing regions of interest on an astrocytic domain using FIJI/ImageJ</div>
<p>(Left) Three frames of a calcium imaging time series show a calcium event arriving in the middle frame, and disappearing by the third frame. Top row shows a hand-drawn analysis region in yellow around the responding domain. Bottom, the same frames as above, but with an annular shaped surround region, drawn in blue, that will be used for background subtraction. Scale bar, 10 μm. (Right) Representative fluorescence traces from the corresponding region and surround ROIs.</p>
</figcaption>
</figure>
<ol type="a">
<li>Open the ROI Manager from the Tool Menu.</li>
<li>Scroll through the time series to visually identify regions where there is an increase in fluorescence. Adjust the z-position to find the best plane at which the response is seen.</li>
<li>Draw an ROI around the region at the best focal plane using the freehand tool. For neurons, typically this will be a responding soma. For astrocytes, this area is less well defined, and may encompass a subcellular region of the processes of an astrocyte. Add the region to the ROI Manager.</li>
<li>Without changing the image display, draw an annular ROI that surrounds the original ROI. This will serve as a background subtraction region. Add this annulus to the ROI Manager.</li>
<li>Repeat steps 2.c and 2.d sequentially to form an alternating list of regions and surrounds, taking care to identify all active signals in the dataset.</li>
<li>Close the current dataset, and open the next one in that sequence (ex: Data_2), maintaining the previously drawn ROIs.</li>
<li>Continue adding additional ROIs and the annular ROIs to the ROI Manager until each of the datasets in the series has been scrutinized.</li>
<li>Once the entire sequence of datasets for a given position have had the regions defined, save the ROISet to disk by selecting the More -&gt; Save buttons in the ROI Manager.</li>
<li>Select the More -&gt; List buttons, to create a list of the Overlay Elements, which includes the z-stack position that is defined for each ROI. Save the Overlay Elements as a text file.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Manual definition of background regions can be somewhat subjective. It is possible that other calcium signals may be detected inside the annular background regions if care is not taken to avoid this. This can be especially problematic in densely packed regions if labeling is not sparse enough, such as within the granule cell layer where cells are abutted next to one another. Many researchers in this case often draw a single “background” region where no such signals are expected. A useful alternative method for background subtraction that has been used recently is to subtract the intensity of the lower percentile of pixels from the whole dataset (<a href="#bib3">Ayaz et al., 2019</a>). For neurons that are more distributed, such as hilar cells of the dentate gyrus, or for astrocytes which are typically non-overlapping, we have found that individually drawn annular regions are most advantageous. The benefits of individual annular regions are that: (1) a non-signaling background region of the tissue may not be readily available; (2) illumination intensities across the field of view may not be uniform due to tissue scattering, uneven expression of reporters, or unique characteristics of the LED; and (3) contaminating out-of-focus signals from cells in other focal planes occurs locally can be therefore be subtracted appropriately.</div>
<ol start="3">
<li>Criteria for Inclusion/Exclusion.
<ol type="a">
<li>Manual definition of ROIs depends on the analyzer’s ability to identify fluorescence fluctuations by eye. Some adjustment of brightness and contrast levels can assist in identification of fluorescent transients. Include any region (cellular or subcellular) that has a visible (increasing) signal in at least one of the datasets. Occasionally, unhealthy cells may maintain a high fluorescent state, only to suddenly lose intensity. This type of decreasing signal does not reflect a responsive cell and should be excluded.</li>
<li>If the tissue is drifting such that the cell does not remain within the ROI boundary, there are many drift correction algorithms built for ImageJ which may work.</li>
<li>If the drifting is too severe to correct, however, discard the datasets.</li>
<li>Similarly, if the fluorescent signal bleaches completely, or becomes suddenly unresponsive, discard the data sets. Typically, signal photobleaching/ phototoxicity only occurs when light intensities are too high due to attempted compensation for low expression of the calcium sensor, or if excessively repeated image acquisition of same tissue is attempted (&gt;3 attempts).</li>
</ol>
</li>
<li>Measure intensities.
<ol type="a">
<li>Define the measurement parameter in FIJI/ImageJ by going to the Analyze -&gt; Set Measurements menu. Select only “Mean gray value.”</li>
<li>Get fluorescence intensity measurements by sequentially:
<ol type="i">
<li>Opening the each of the dataset (Data_1, Data_2 etc…).</li>
<li>Use the More -&gt; “Multi Measure” buttons in the ROI Manager to analyze.</li>
<li>Save the output file with an appropriate and unique filename.</li>
</ol>
</li>
</ol>
</li>
<li>Quantify in Igor Pro. Once measurements are made for each dataset, import them into an analysis environment, such as Igor Pro, MATLAB, or a custom program which allows development and running of customized analysis routines for calculating ΔF/F values and detecting events. See <a href="#fig3">Figure 3</a>.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/446-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Analysis of time series calcium traces</div>
<p>(A) Depiction of raw fluorescence traces for 5 focal planes for the ROI, the annular background region, and the background-subtracted fluorescence (arbitrary units). X axis reports the datapoint number in the trace. Red indicates the focal plane defined as the “best” focal plane for this ROI.</p>
<p>(B) Calculation of ΔF/F values for the best focal plane defined as (F − F<sub>0</sub>)/F<sub>0</sub>. F<sub>0</sub> is calculated by smoothing the background-subtracted fluorescence and then taking a running minimum.</p>
<p>(C) Preparation of the ΔF/F trace for peak detection. The ΔF/F trace is heavily smoothed to find the red shape of the trace. The difference between the ΔF/F and the smoothed trace reveals transients without any baseline shift. A two-cycle thresholding that considers the mean and standard deviation of the 5<sup>th</sup> through 95<sup>th</sup> percentile of datapoints reveals the signal component (red) above the noise. Peak detection algorithms can then proceed using this threshold.</p>
</figcaption>
</figure>
<ol type="a">
<li>Import Measurement data. Open .csv files (delimited text data), from each dataset in the sequence. ImageJ writes .csv files as delimited text, where columns correspond to each ROI that is defined in the ROI Manager, and rows correspond to the 1,050 image frames (5 z_planes and 210 time_points).</li>
<li>Background subtraction. Since ROIs are defined in FIJI/ImageJ as alternating Region_ROIs and Annular_backgrounds:
<ol type="i">
<li>Extract the odd columns into a new array called Regions.</li>
<li>Extract the even columns into a new array called Backgrounds.</li>
<li>Subtract Background measurements from Region measurements.</li>
<li>Prohibit negative fluorescence by setting a lower limit of 0 fluorescence.</li>
</ol>
</li>
<li>Best Plane Selection. If careful selection of the optimal z-plane is made during ROI creation, the appropriate z-values are written in the Overlay Elements text file.
<ol type="i">
<li>Import the Overlay Elements file as a delimited text file.</li>
<li>Extract only the odd rows from the Overlay Elements.</li>
<li>Use the column entitled “Z” to get the numerical value corresponding to the z-plane to use for each ROI.</li>
<li>Extract the correct z-plane from the background-subtracted measurements and save each ROI as a 210 point time series.</li>
</ol>
</li>
<li>Create F<sub>0</sub>. The penultimate step before calculating ΔF/F is the creation of the F<sub>0</sub> trace, which helps account for bleaching of the signal over time, as well as other non-signal related changes in baseline.
<ol type="i">
<li>Duplicate the 210 point ROI time series just created in step 5.c.iv.</li>
<li>Run a boxcar smoothing with boxsize = 20 points.</li>
<li>For each timepoint, find the minimum smoothed value within a n-back window of 150 points. No negative fluorescence values are permitted.</li>
</ol>
</li>
<li>Calculate ΔF/F trace.
<ol type="i">
<li>Use the background-subtracted values created in step 5.c.iv as the F values.</li>
<li>Use the min_smoothed values created in step 5.d.iii as the F<sub>0</sub> values.</li>
<li>Execute a point by point computation of (F – F<sub>0</sub>)/F<sub>0</sub> to get the ΔF/F trace.</li>
</ol>
</li>
</ol>
</li>
<li>Quantify frequencies of events. Event detection requires recognizing signal in the ΔF/F trace that rise above noise levels. Parameters for such detection may need to be carefully adjusted and will depend on the particularities of the equipment used. See <a href="#fig3">Figures 3</a> and <a href="#fig4">4</a>.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/446-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Quantification of calcium event frequency</div>
<p>(A) Representative ΔF/F traces from an astrocyte dataset, and (B) from a neuronal (granule cell) dataset. Traces are labeled with blue “plus” symbols whenever an event is detected by the peak detection algorithm. The number of detected peaks divided by the time gives the event frequency.</p>
</figcaption>
</figure>
<ol type="a">
<li>Trace concatenation. Extracting features from ΔF/F traces for a given ROI requires treating all conditions of that ROI equally. Begin by concatenating ΔF/Fs from each of Data_1 and Data_2 etc. for each ROI trace. (For example, if the experiment consisted of (1) baseline, (2) CNO, and (3) wash conditions; concatenate those three ΔF/Fs for each ROI).</li>
<li>Suppress unwanted detection of extra datapoints inside slow signals for the upcoming peak detection algorithms.
<ol type="i">
<li>Smooth the concatenated ΔF/Fs by boxcar smoothing with box size = 200 points to get the general shape of the slow fluorescence responses.</li>
<li>Take the difference of the ΔF/F data and the smoothed data to find the acute signal fluctuations. Smooth with a single-pass gaussian filter.</li>
</ol>
</li>
<li>Determine a cutoff threshold based on a 2-cycle analysis of the fluctuation data to determine signal and noise.
<ol type="i">
<li>Sort fluctuation datapoints by intensity.</li>
<li>Collect the 5<sup>th</sup> through 95<sup>th</sup> percentiles of the sorted data.</li>
<li>Calculate the median and standard deviation for this middle 90%.</li>
<li>Create an initial cutoff level at F<sub>Cut</sub> = median + 0.25 × SD.</li>
<li>Recollect all fluctuation datapoints beneath F<sub>cut</sub> level (non-signaling fraction).</li>
<li>Take the 5<sup>th</sup> through 95<sup>th</sup> percentiles of only these points.</li>
<li>Calculate a new median and standard deviation.</li>
<li>Create a revised more stringent cutoff level at F<sub>Cut</sub> = median + 1 SD.</li>
</ol>
</li>
<li>Run a peak detection algorithm with a minimum threshold set to the greater of F<sub>cut</sub> or 0.2 ΔF/F units. The built-in peak detection in Igor Pro finds local maximums using the first and second derivatives, with a user defined box size. A box size of 5 datapoints provides reasonable detection of events at this time-scale.
<ol type="i">
<li>Visual inspection of the detected events will help determine appropriateness of the algorithms. Adjustable parameters include:
<ul>
<li>Smoothing size.</li>
<li>Standard deviation adjustment factor (currently set to 1 SD).</li>
<li>Box size of the peak detection algorithm.</li>
<li>The min threshold parameter of 0.2 ΔF/F units.</li>
</ul>
</li>
<li>Once parameters are set, standardize them for the entire dataset to avoid any bias in analysis.</li>
</ol>
</li>
<li>Re-split the data into the separate conditions, and count the detected events. Divide by the time (approximately 10 min) to get the frequency of events in each ROI for baseline, CNO, and other optional conditions.</li>
<li>The mean frequency of events per minute at baseline and during stimulation can be calculated and plotted as bars. It is useful to plot individual datapoints connected with lines to show associations.</li>
<li>Statistical analysis comparing baseline to CNO conditions should use paired t tests, as the data come from 2 measurements of the same ROI.</li>
</ol>
</li>
<li>Quantifying size of events. In addition to frequency of events, amplitude measurements or area measurements are often reported.
<ol type="a">
<li>Total area under the ΔF/F curve can be easily calculated by integrating the sum of the datapoints of the ΔF/F trace.</li>
<li>Restrict area measurement to be ≥ 0.</li>
<li>The mean area at baseline and during stimulation can be calculated and plotted as bars. It is useful to plot individual datapoints connected with lines to show associations.</li>
<li>Statistical analysis comparing baseline to CNO conditions should use paired t tests, as the data come from 2 measurements of the same ROI.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Amplitude measurements can be difficult to interpret from single-photon datasets, as light scattering from different focal depths, or differences in expression of GCaMP indicator make absolute measurements of event amplitudes unreliable. Instead, use relative comparisons, making sure only to compare the effects of treatments “within ROIs.”</div>
<ol start="8">
<li>Calculate the fold change. It is useful to know what the fold change in frequency or area is for each ROI due to circuit activation (comparing baseline to CNO condition).
<ol type="a">
<li>Fold Change in frequency is calculated as:
<ol type="i">
<li>Frequency<sub>2</sub>/Frequency<sub>1</sub>.</li>
<li>As frequencies can often be zero during Frequency<sub>1</sub>, infinite fold changes should be recalculated as 2 × Frequency<sub>2</sub>/minimum_Frequency<sub>1</sub> (i.e., 1 event/imaging time period).</li>
<li>Fold changes become zero if there are no events in Frequency<sub>2.</sub>
</li>
</ol>
</li>
<li>Fold change in Area is calculated as Area<sub>2</sub>/Area<sub>1</sub>.</li>
</ol>
</li>
<li>Slice-wise comparisons. Recording many events within a slice, and then repeating the acquisition from multiple slices or animals is essentially a nested-study design. In order to account for bias that may exist within a particular slice due to peculiarities in local connectivity (such as one particularly active brain slice with an over-abundance of detected regions), we must determine whether changes in event frequency or event amplitude are independent of the sample origins. The correct statistical analysis process would be to use a Linear Mixed Effects model that takes the fixed and random effects of the study into account (for example, see <a href="#bib5">Bojarskaite et al., 2020</a>; <a href="#bib11">Stobart et al., 2018</a>). Such analysis is available using the fitlme function in MATLAB, or the lmer function in the lme4 package in R. We can also get a good idea of how similar or different individual slices are by doing statistical comparisons in a slice-wise manner. Using the fold changes calculated above, the percentage of regions displaying increasing or decreasing calcium responses can compared.
<ol type="a">
<li>For all slices that are subjected to a given experimental condition, gather the Fold Changes, as calculated in step 8.</li>
<li>For one slice at a time, count the number of values with increasing (Fold &gt; 1), unchanging (Fold = 1), and decreasing (Fold &lt; 1) change.</li>
<li>Dividing by the total number or regions in that slice and multiplying by 100 will provide the corresponding %increasing, %unchanging, and %decreasing regions for that slice.</li>
<li>Compute the mean and standard error for all the %increases, %unchangings, and %decreases.</li>
<li>A stacked bar graph of these means should add up to 100%, and should include the error bars at the tops of each subsection of the bar. See (<a href="#bib2">Asrican et al., 2020</a>) for examples.</li>
<li>Statistics for comparing different conditional manipulations can use unpaired t tests, if particular parts (the increasing part, for example) are changing, or a Chi-squared test if the entire distribution should be compared.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Unlike frequency, area measurements do not have discrete counting of events, and therefore will not ever have fold changes exactly equal to 1. The %unchanging category should either be removed, or the criteria for %unchanging should have a defined “similarity” range acceptable enough to be considered unchanged.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Somatic calcium events in neurons should currently be considered a proxy for neuronal action potentials and do not necessarily equal the spike rate. However, relative changes in calcium signals should follow the same trends as spiking, such that increases in calcium event frequency correlate with increases in spike rates.</p>
<p>Despite the well-known existence of dynamic calcium signals in astrocytes, there has been some controversy on the physiological role. Much of the confusion seems to be attributed to limiting analysis of only IP3 mediated calcium, or only consideration of somatic events (<a href="#bib6">Khakh and McCarthy, 2015</a>; <a href="#bib8">Savtchouk and Volterra, 2018</a>). By including the LCK tag on the GCaMP construct, we can circumvent some of this controversy and ask how signals in the fine processes are affected by neuronal circuitry.</p>
<p>Higher resolution techniques such as 2-photon or confocal microscopy could, of course, provide much finer details that the wide-field fluorescent technique lacks. For example, single-photon techniques that collect out-of-focus light from astrocytes above or below the plane of interest could obscure the ability to assign signals accurately. Despite this, it is still possible to extract meaningful signals from such datasets, provided that enough regions are included in analysis, enough time is allowed to collect longer data sets, and that the ROIs are drawn as carefully as possible.</p>
<p>Finally, there may be certain types of astrocyte calcium signals, such as growing or propagating signals, that are not able to be detected by region-based analysis routines. In this case, more sophisticated intensity-based algorithms, such as the ones developed by (<a href="#bib5">Bojarskaite et al., 2020</a>; or <a href="#bib14">Wang et al., 2019</a>) initially for analyzing multiphoton datasets in astrocytes, might be beneficial. It remains to be seen if such techniques can be adapted for use on wide-field, camera-based, datasets.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Calcium signals are weak or unstable and are unable to be reliably detected during analysis.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Signal to noise ratio can be the biggest difficulty to overcome in analysis of calcium events (steps 5–7 in Quantification and Statistical Analysis). During analysis, defining the best ROI to draw and optimizing the parameters in step 2 in “Quantification and Statistical Analysis” will help, but if the data set is sub-optimal, it will be more difficult. Some analysis parameters in step 5.d.ii-iii and in step 6.d.i in “Quantification and Statistical Analysis” can be adjusted for particular datasets.</p>
<p>Rundown of calcium fluorescence can be caused by too high an intensity of illumination for these longer data sets, or due to poor slice quality. Improving health of the slices, limiting unnecessary illumination and light intensity, and good transgene expression (see below) are all critical for getting quality data sets.</p>
<p>Physical instability of the tissue can also cause extraneous signal fluctuations, which are often due to expansion or contraction of the brain slice in solution. Allow some time, (∼5–10 min) after placing a slice into the perfusate within the recording chamber before initiating imaging sessions to allow for equilibration. A slice anchor is also useful to provide stability and prevent drift.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Poor or insufficient viral expression in brain slices (step 6.c in Step-by-step method details).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Ensure injection coordinates are correct. A test injection of a dye can quickly inform on targeting accuracy. Confirm genotype of mouse if using a Cre animal. Fresh virus may be needed, or additional days post injection to express a transgene may be necessary if expression is low. Preliminary titration of the viral titer is recommended. Occasionally, a different viral serotype may be beneficial. Alternatively, there are transgenic mice that are available commercially that express floxed GCaMP indicators (ex: from Jackson Laboratory).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Calcium datasets are prohibitively large. Data storage for the experiments described here may be a significant limitation.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>The procedures described here use full frame images during acquisition. It may be necessary to bin pixels in order to provide a smaller dataset. However, this does sacrifice some imaging resolution. Additional hard drives may be needed to store and transfer files off of the acquisition computer. Sufficient RAM for loading data during analysis is required. Subsampling of the data is one possible post hoc solution if resources are limited. FIJI also provides a “virtual” mode of importing data, which streams the data from disk rather than loading entire datasets at once.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Armbruster, B.N., Li, X., Pausch, M.H., Herlitze, S., and Roth, B.L. (2007). Evolving the lock to fit the key to create a family of G protein-coupled receptors potently activated by an inert ligand. Proc. Natl. Acad. Sci. U S A <i>104</i>, 5163-5168. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref1">View at publisher</a></p>
<p id="bib2">Asrican, B., Wooten, J., Li, Y.D., Quintanilla, L., Zhang, F., Wander, C., Bao, H., Yeh, C.Y., Luo, Y.J., Olsen, R., et al. (2020). Neuropeptides modulate local astrocytes to regulate adult hippocampal neural stem cells. Neuron <i>108</i>, 349-366. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref2">View at publisher</a></p>
<p id="bib3">Ayaz, A., Stauble, A., Hamada, M., Wulf, M.A., Saleem, A.B., and Helmchen, F. (2019). Layer-specific integration of locomotion and sensory information in mouse barrel cortex. Nat. Commun. <i>10</i>, 2585. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref3">View at publisher</a></p>
<p id="bib4">Bindocci, E., Savtchouk, I., Liaudet, N., Becker, D., Carriero, G., and Volterra, A. (2017). Three-dimensional Ca(2+) imaging advances understanding of astrocyte biology. Science <i>356</i>, eaai8185. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref4">View at publisher</a></p>
<p id="bib5">Bojarskaite, L., Bjornstad, D.M., Pettersen, K.H., Cunen, C., Hermansen, G.H., Abjorsbraten, K.S., Chambers, A.R., Sprengel, R., Vervaeke, K., Tang, W., et al. (2020). Astrocytic Ca(2+) signaling is reduced during sleep and is involved in the regulation of slow wave sleep. Nat. Commun. <i>11</i>, 3240. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref5">View at publisher</a></p>
<p id="bib6">Khakh, B.S. and McCarthy, K.D. (2015). Astrocyte calcium signaling: from observations to functions and the challenges therein. Cold Spring Harb. Perspect. Biol. <i>7</i>, a020404. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref6">View at publisher</a></p>
<p id="bib7">Linkert, M., Rueden, C.T., Allan, C., Burel, J.M., Moore, W., Patterson, A., Loranger, B., Moore, J., Neves, C., Macdonald, D., et al. (2010). Metadata matters: access to image data in the real world. J. Cell Biol. <i>189</i>, 777-782. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref7">View at publisher</a></p>
<p id="bib8">Savtchouk, I. and Volterra, A. (2018). Gliotransmission: beyond black-and-white. J. Neurosci. <i>38</i>, 14-25. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref8">View at publisher</a></p>
<p id="bib9">Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref9">View at publisher</a></p>
<p id="bib10">Shigetomi, E., Kracun, S., and Khakh, B.S. (2010). Monitoring astrocyte calcium microdomains with improved membrane targeted GCaMP reporters. Neuron Glia Biol. <i>6</i>, 183-191. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref10">View at publisher</a></p>
<p id="bib11">Stobart, J.L., Ferrari, K.D., Barrett, M.J.P., Gluck, C., Stobart, M.J., Zuend, M., and Weber, B. (2018). Cortical circuit activity evokes rapid astrocyte calcium signals on a similar timescale to neurons. Neuron <i>98</i>, 726-735.e4. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref11">View at publisher</a></p>
<p id="bib12">Ting, J.T., Daigle, T.L., Chen, Q., and Feng, G. (2014). Acute brain slice methods for adult and aging animals: application of targeted patch clamp analysis and optogenetics. Methods Mol. Biol. <i>1183</i>, 221-242. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref12">View at publisher</a></p>
<p id="bib13">Ting, J.T., Lee, B.R., Chong, P., Soler-Llavina, G., Cobbs, C., Koch, C., Zeng, H., and Lein, E. (2018). Preparation of acute brain slices using an optimized N-methyl-D-glucamine protective recovery method. J. Vis. Exp. <i>132</i>, 53825. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref13">View at publisher</a></p>
<p id="bib14">Wang, Y., DelRosso, N.V., Vaidyanathan, T.V., Cahill, M.K., Reitman, M.E., Pittolo, S., Mi, X., Yu, G., and Poskanzer, K.E. (2019). Accurate quantification of astrocyte and neurotransmitter fluorescence dynamics for single-cell and population-level physiology. Nat. Neurosci. <i>22</i>, 1936-1944. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00013-7/sref14">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Juan Song, <a href="mailto:juansong@email.unc.edu">juansong@email.unc.edu</a>.</p>
<h4>Materials Availability</h4>
<p>No new mouse lines or viral strains were generated in this study. The particular animals and reagents used here are reported in the <a href="#key-resources-table">Key resources table</a>.</p>
<h4>Data and Code Availability</h4>
<p>Calcium data sets from this study have not been deposited publicly, but samples may be requested from the lead contact or the technical contact. Further information on specific analysis code written in Igor Pro may be made from the technical contact, Brent Asrican, <a href="mailto:basrican@email.unc.edu">basrican@email.unc.edu</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank the authors of <a href="#bib2">Asrican et al. (2020)</a> for application of this protocol. This work was supported by grants awarded to J.S. from <a href="https://doi.org/10.13039/100000968">American Heart Association</a>, United States; <a href="https://doi.org/10.13039/100001391">Whitehall Foundation</a>, United States; <a href="https://doi.org/10.13039/100000002">NIH</a> (MH111773-01, MH122692-01 AG058160, NS104530), United States; and <a href="https://doi.org/10.13039/100009670">NARSAD</a> Young Investigator Award (26017) to B.A from the Brain &amp; Behavior Research Foundation, United States.</p>
<h3>Author Contributions</h3>
<p>J.S. conceived and oversaw the original project published in <a href="#bib2">Asrican et al. (2020)</a>. B.A. developed the protocols described here and authored this manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>