<article>
<section>
<h1 id="header">Process control and <i>in silico</i> modeling strategies for enabling high density culture of human pluripotent stem cells in stirred tank bioreactors</h1>
<p><time datetime="2021-12-10">Published: December 17, 2021</time></p>
<p>Felix Manstein,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Kevin Ullmann,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Wiebke Triebert,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Robert Zweigerdt<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Leibniz Research Laboratories for Biotechnology and Artificial Organs (LEBAO), Hannover, Germany</p>
<p id="aff2"><sup>2</sup>REBIRTH Cluster of Excellence, Hannover Medical School, Carl-Neuberg-Str. 1, 30625 Hannover, Germany</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:manstein.felix@mh-hannover.de">manstein.felix@mh-hannover.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:zweigerdt.robert@mh-hannover.de">zweigerdt.robert@mh-hannover.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100988">10.1016/j.xpro.2021.100988</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The routine therapeutic and industrial applications of human pluripotent stem cells (hPSCs) require their constant mass supply by robust, efficient, and economically viable bioprocesses. Our protocol describes the fully controlled expansion of hPSCs in stirred tank bioreactors (STBRs) enabling cell densities of 35 × 10<sup>6</sup> cells/mL while reducing culture medium consumption by 75%. This is achieved by <i>in silico</i> process modeling and computable upscaling. We provide a detailed blueprint for systematic process development of hPSCs and their progenies.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1002/sctm.20-0453">Manstein et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Optimized protocol for high-density expansion of pluripotent stem cells</li>
<li>Guideline for adoption of the <i>in silico</i> model</li>
<li>Universal upscaling strategy for aggregate-based processes in stirred bioreactors</li>
<li>Blueprint for advancing process automation and lineage differentiation strategies</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Several human pluripotent stem cell (hPSC)-based therapy concepts have now progressed towards first-in-man studies (<a href="#bib7">Kobold et al., 2020</a>). However, leading investigators highlighted the current need for more efficient culture processes for generating pluripotent hPSCs - the “raw material” for lineage-directed differentiations - at clinically relevant quantities, quality and at commercially viable conditions (<a href="#bib1">Ackermann et al., 2018</a>; <a href="#bib15">Stevens and Murry, 2018</a>; <a href="#bib5">Halloin et al., 2019</a>; <a href="#bib14">Sahabian et al., 2021</a>).</p>
<p>In 2019, we published a basic bioreactor-based expansion protocol for hPSCs (<a href="#bib9">Manstein et al., 2019</a>). More recently, (<a href="#bib10">Manstein et al., 2021</a>) we could show the overarching importance of pH control for maintaining hPSC proliferation, in line with the established knowledge from biotechnology that the proliferation of continuous cell lines is diminished at pH levels below 7.0. Moreover, in contrast to continuous cell lines such as CHO and BHK cells, hPSCs do not consume lactate, which they extensively produce. This leads to the combinatorial challenge of rising glucose demands further boosting lactate secretion, consequently driving pH stabilizing base supplementation, together elevating osmolality levels. Given these closely interlinked effects of culture strategies and the cell metabolism, we found that perfusion feeding is the most appropriate approach for enabling the controlled high-density cultivation of hPSCs grown aggregates in suspension. Perfusion feeding allows for the proliferation- and density- specific adaption of the media perfusion rate, thus stabilizing culture conditions by providing fresh nutrients, while simultaneously diluting growth-inhibiting metabolites such as lactate. In addition, we demonstrate in this paper how the mean aggregate size and the aggregate size distribution can be effectively controlled by the stirring speed; this approach can be utilized for computable process upscaling and transition to larger bioreactor systems as well.</p>
<p>Considering the above outlined challenges, this protocol describes a strategy for hPSC bioprocessing enabling the 70-fold expansion of suspension-seeded cells within 7 days in stirred tank bioreactors (STBRs) ultimately yielding 35 × 10<sup>6</sup> cells/mL. This expansion rate and cell density is achieved by combining the advantages of perfusion feeding, the tight control of critical process parameters such as pH and dissolved oxygen (DO) as well as uninterrupted nutrient supply facilitated by <i>in silico</i> process modelling. In addition, a universally applicable upscaling strategy is revealed, accounting for bioreactor system-dependent hardware features such as the vessel- and impeller- dimensions.</p>
<p>The proposed culture strategy is thoroughly validated with three independent hiPSC lines, strongly suggesting its universal applicability and similar efficiency for any hiPSC or hESC lines at hand and its potential utility for pluripotent stem cell lines from other species as well, which may also be of interest to the envisioned production of <i>in vitro</i> meat (<a href="#bib13">Rubio et al., 2020</a>).</p>
<p>For the successful protocol application, investigators should be generally trained in relevant mammalian cell culture techniques, including cryopreservation and thawing, sterile cell handling and passaging as well as established analytical standard techniques such as cell counting and viability staining.</p>
<p>Additional knowhow in bioreactor and process technologies such as: bioreactor assembly, bioreactor sterilization and handling under the sterile hood, suspension culture inoculation and process initiation/ execution as well as process monitoring and analysis.</p>
<h3 id="sec1.1">Background and guidance for applying the <i>in silico</i> model</h3>
<p>Guided by our step-by-step protocol, investigators are enabled to recapitulate this advanced culture strategy and to achieve equivalent results using their specific hPSC line and respective STBR system at hand, without major adaptations. However, following this blueprint (model code can be found under <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Mmc1.pdf">Data S1</a>), preexisting wet-lab data established in an individual laboratory can also be used to eventually draft and optimize their own, customized bioreactor- and cell-line specific <i>in silico</i> model.</p>
<p>The <i>in silico</i> model underlying our optimized process (<a href="#bib10">Manstein et al., 2021</a>), is based on classical Monod-kinetics (<a href="#bib12">Monod, 1949</a>) and considers:</p>
<ol>
<li>key process factors that serve as so-called “Monod-variables”
<ol type="a">
<li>glucose-concentration</li>
<li>lactate-concentration</li>
<li>glutamine-concentration</li>
<li>perfusion rate</li>
<li>osmolality</li>
<li>aggregate size</li>
</ol>
</li>
<li>“Monod- (like-) constants, which serve as limiting constants for each respective variable.</li>
<li>cell-specific rates, which must be derived from existing process data, including:
<ol type="a">
<li>cell specific growth rate</li>
<li>consumption rates for glucose and glutamine</li>
<li>production rate for lactate</li>
<li>cell line-dependent aggregate formation rate, which was found to be divers between individual hiPSC lines.</li>
</ol>
</li>
</ol>
<p>Potential starting values to establish such a model are displayed in <a href="#tbl1">Table 1</a>.</p>
<table id="tbl1">
<caption>Table 1. Exemplary kinetic model parameters</caption>
<thead>
<tr>
<th>Model parameter</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>K<sub>Glc</sub> [mM]</td>
<td>1.5</td>
</tr>
<tr>
<td>K<sub>Lac</sub> [mM]</td>
<td>50</td>
</tr>
<tr>
<td>K<sub>Gln</sub> [mM]</td>
<td>0.01</td>
</tr>
<tr>
<td>K<sub>Agg</sub> [μm]</td>
<td>350/2</td>
</tr>
<tr>
<td>K<sub>Osm</sub> [mOsm/kg]</td>
<td>500</td>
</tr>
<tr>
<td>μ [d<sup>−1</sup>]</td>
<td>1.35</td>
</tr>
<tr>
<td>q<sub>Glc</sub> [× 10<sup>−8</sup> mmol × cell<sup>−1</sup> × d<sup>−1</sup>]</td>
<td>1.474</td>
</tr>
<tr>
<td>q<sub>Lac</sub> [× 10<sup>−8</sup> mmol × cell<sup>−1</sup> × d<sup>−1</sup>]</td>
<td>2.37</td>
</tr>
<tr>
<td>q<sub>Gln</sub> [× 10<sup>−9</sup> mmol × cell<sup>−1</sup> × d<sup>−1</sup>]</td>
<td>1.856</td>
</tr>
<tr>
<td>agg<sub>f</sub> [–]</td>
<td>0.95</td>
</tr>
<tr>
<td>agg<sub>g</sub> [–]</td>
<td>0.25</td>
</tr>
</tbody>
</table>
<p>The general workflow to establish/adjust the <i>in silico</i> model to a specific process is as follows (and displayed in <a href="#fig1">Figure 1</a>):</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic outline of the Monod-Model process development strategy</div>
<p>In general, the process starts with already existing wet-lab data, which is an existing process that is supposed to be optimized. From this data, key variables which are regarded to majorly influence the process need to be identified. These variables are quite often metabolites like glucose, but also can be more complex like osmolality or even aggregate size (for aggregate dependent cultures). For these values so called “Monod-constants” need to be defined which are limiting values/ concentrations at which cell growth is impacted. Based on this the first stage model can be built. Afterwards this model needs to be challenged, first in silico and afterwards in the wet-lab. This means that key variables need to be altered in order to affect cell growth. Afterwards the impact of the alteration between the in silico model and the wet-lab need to be compared. With a high probability, the model does not align with the wet-lab data. In that case, the system needs to be reevaluated in order to identify new variables that need to be added, or to adjust the already existing monod-constants. Afterwards the second stage model is generated. From now on the steps of model alteration in silico and in the wet-lab followed by reevaluation need to be repeated until wet-lab and in silico data overlap.</p>
</figcaption>
</figure>
<ol start="4">
<li>Establish a first stage model with potential starting values from <a href="#tbl1">Table 1</a> or the individual process</li>
<li>Challenge the model by applying alterations of key process parameters <i>in silico.</i>
</li>
<li>Evaluation of the alterations <i>in situ</i> in the laboratory. The practical testing of the model will display its potential limitations.</li>
<li>Adaption the Monod- (like-) constants so that a second stage model better reflects the outcome of the <i>in situ</i> experiments. Potentially, the addition of new process-relevant factors must be considered as well, which may have been missing the first stage.</li>
<li>Repeat this model-challenging-and-adjustments” strategy until the <i>in silico</i> model and its predictions properly reflect the real wet lab results and process pattern.</li>
</ol>
<p>To better illustrate the adjustment process the following description is given: For example, the Monod constant for lactate (K<sub>Lac</sub>) has been set to a relatively high level of 200 mM (rather than a more typical value of 50 mM). In consequence, the model may suggest the supplementation of high glucose concentration (for achieving high cell yields) in compliance with high lactate concentration, due to the high value defined for K<sub>Lac</sub>. Most likely, however, respective wet-lab experiments would result in poor cell proliferation in reply to detrimental lactate concentrations accumulating over time and thus inferior cell yields, but even without reaching the maximum lactate concentration level predicted by the model. As a result, the observed wet-lab data will serve as the basis for adapting the next stage model, reflecting reduction of K<sub>Lac</sub>; consequently, the new model should better reflect the cell growth and lactate concentration curves observed by the wet-lab experiments.</p>
<h3 id="sec1.2">Preparation of cultivation flasks for adherent culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<ol start="9">
<li>Preparation of Geltrex-stocks and coating of T-flasks
<ol type="a">
<li>Preparation of aliquots from stock vials
<ol type="i">
<li>Thaw vial for 12 h on ice in the fridge at 4°C.</li>
<li>Aliquot à 125 μL and store at −80°C for up to 12 month.</li>
</ol>
</li>
<li>Coating of flasks
<ol type="i">
<li>Thaw an aliquot of 125 μL into 50 mL DMEM/F12 and mix.</li>
<li>1 mL of Geltrex-DMEM/F12 mix is sufficient for 5 cm<sup>2</sup> surface.</li>
<li>Incubate flasks in the incubator at 37°C for 1 h and store subsequently at 4°C for up to one month.</li>
</ol>
</li>
</ol>
</li>
<li>Alternative: Preparation of Vitronectin-stocks and coating of T-flasks
<ol type="a">
<li>Preparation of aliquots from stock vials
<ol type="i">
<li>Thaw vial at 20°C–22°C.</li>
<li>Aliquot à 250 μL and store at −80°C for up to 12 month.</li>
</ol>
</li>
<li>Coating of flasks
<ol type="i">
<li>Thaw an aliquot of 250 μL into 50 mL PBS and mix.</li>
<li>1 mL of Vitronectin-PBS mix is sufficient for 10 cm<sup>2</sup> surface.</li>
<li>Incubate flasks in the incubator at 37°C for 1 h and store subsequently at 4°C for up to one month.</li>
</ol>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>SSEA3 Monoclonal Antibody (MC-631), DyLight 650, Dilution 1:50</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#MA1-020-D650</td>
</tr>
<tr>
<td>SSEA-4 Antibody, anti-human, VioBlue, REAfinity, Dilution 1:25</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-098-366</td>
</tr>
<tr>
<td>TRA-1-60 Antibody, anti-human, PE, REAfinity, Dilution 1:25</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-122-921</td>
</tr>
<tr>
<td>Oct3/4 Isoform A Antibody, anti-human/mouse, PE, REAfinity, Dilution 1:25</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-123-771</td>
</tr>
<tr>
<td>Nanog Antibody, anti-human, APC, REAfinity, Dilution 1:25</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-120-774</td>
</tr>
<tr>
<td>Ki-67 Antibody, anti-human/mouse, PE-Vio 770, REAfinity, Dilution 1:25</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-120-419</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>PBS (10 ×), pH 7.4</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#70011036</td>
</tr>
<tr>
<td>StemPro Accutase Cell Dissociation Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A1110501</td>
</tr>
<tr>
<td>Versene Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15040033</td>
</tr>
<tr>
<td>DMEM/F-12, HEPES</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11330057</td>
</tr>
<tr>
<td>Pluronic F-68 Non-ionic Surfactant (100×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#S6014</td>
</tr>
<tr>
<td>Geltrex LDEV-Free, hESC-Qualified, Reduced Growth Factor Basement Membrane Matrix</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A1413302</td>
</tr>
<tr>
<td>CTS Vitronectin (VTN-N) Recombinant Human Protein</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A27940</td>
</tr>
<tr>
<td>L-Glutamine 200 mM (100 ×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#25030-024</td>
</tr>
<tr>
<td>D-(+)-Glucose Hybri-Max</td>
<td>Sigma-Aldrich</td>
<td>Cat#G5146</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>Sigma-Aldrich</td>
<td>Cat#24040032</td>
</tr>
<tr>
<td>Na<sub>2</sub>SeO<sub>3</sub>
</td>
<td>Sigma-Aldrich</td>
<td>Cat#S5261</td>
</tr>
<tr>
<td>Insulin solution human</td>
<td>Sigma-Aldrich</td>
<td>Cat#I9278</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A9418</td>
</tr>
<tr>
<td>Transferrin human recombinant</td>
<td>Sigma-Aldrich</td>
<td>Cat#T3705</td>
</tr>
<tr>
<td>Ascorbic acid 2-phosphate</td>
<td>Sigma-Aldrich</td>
<td>Cat#A8960</td>
</tr>
<tr>
<td>Sigmacote</td>
<td>Sigma-Aldrich</td>
<td>Cat#SL2</td>
</tr>
<tr>
<td>Recombinant Human TGF-β1</td>
<td>PeproTech</td>
<td>Cat#100-21C</td>
</tr>
<tr>
<td>Animal-Free Recombinant Human FGF-basic</td>
<td>PeproTech</td>
<td>Cat#AF-100-18B</td>
</tr>
<tr>
<td>y-27632 dihydrochloride</td>
<td>Tocris Bioscience</td>
<td>Cat#1254</td>
</tr>
<tr>
<td>FIX &amp; PERM Kit</td>
<td>Dianova</td>
<td>Cat#GAS-002-1</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HSC1285_T-iPS2 (<a href="#bib6">Hartung et al., 2013</a>)</td>
<td>Hannover Medical School (MHH)</td>
<td>MHHi006-A</td>
</tr>
<tr>
<td>hHSC_Iso4_ADCF_SeViPS2 (Phönix) (<a href="#bib4">Haase et al., 2017</a>)</td>
<td>Hannover Medical School (MHH)</td>
<td>MHHi001-A</td>
</tr>
<tr>
<td>CD34+hPBHSC_GMPDU_SeV-iPS8 (<a href="#bib3">Haase et al., 2019</a>)</td>
<td>Hannover Medical School (MHH)</td>
<td>MHHi008-A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo v10 software</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>Fiji ImageJ</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>DASware control</td>
<td>Eppendorf</td>
<td>Cat#78600167</td>
</tr>
<tr>
<td>Berkeley Madonna</td>
<td>University of California at Berkeley</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>10, 25 and 50 mL serological pipettes</td>
<td>Sarstedt</td>
<td>Cat#86.1254.001, Cat#86.1685.001, Cat#86.1689.001</td>
</tr>
<tr>
<td>15 and 50 mL conical tube</td>
<td>Greiner Bio-One</td>
<td>Cat#188261, Cat#227261</td>
</tr>
<tr>
<td>centrifuge tubes 500 mL</td>
<td>Corning Life Sciences</td>
<td>Cat#CLS431123-36EA</td>
</tr>
<tr>
<td>10 and 200 μL sterile filter pipette tips</td>
<td>Starlab</td>
<td>Cat#S1120-3810, Cat#S1120-8810</td>
</tr>
<tr>
<td>1000 μL sterile filter pipette tips</td>
<td>Sarstedt</td>
<td>Cat#70.762.211</td>
</tr>
<tr>
<td>T75 and T175 flasks</td>
<td>Greiner Bio-One</td>
<td>Cat#658175, Cat#660175</td>
</tr>
<tr>
<td>1.5 and 2.0 mL Eppendorf Safe-Lock Tubes</td>
<td>Eppendorf</td>
<td>Cat#0030120086, Cat#0030120094</td>
</tr>
<tr>
<td>500 mL bottle top filter</td>
<td>TPP</td>
<td>Cat#99505</td>
</tr>
<tr>
<td>Syringe filter</td>
<td>Carl Roth</td>
<td>Cat#P666.1</td>
</tr>
<tr>
<td>20 mL Syringe</td>
<td>B. Braun</td>
<td>Cat#4606205V</td>
</tr>
<tr>
<td>Cell culture cabinet/laminar flow (HERAsafe HS18)</td>
<td>Heraeus Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>Benchtop centrifuge (Heraeus Fresco 17)</td>
<td>Thermo Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Benchtop centrifuge (Heraeus Multifuge 3 S-R)</td>
<td>Thermo Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>MACSQuant Analyzer 10 Flow Cytometer</td>
<td>Miltenyi Biotec</td>
<td>N/A</td>
</tr>
<tr>
<td>Vi-CELL XR</td>
<td>Beckman Coulter</td>
<td>N/A</td>
</tr>
<tr>
<td>Humidified incubator (MCO-20AIC)</td>
<td>Sanyo</td>
<td>N/A</td>
</tr>
<tr>
<td>Water bath</td>
<td>GFL</td>
<td>Cat#1003</td>
</tr>
<tr>
<td>Terg-a-zyme enzyme detergent pack</td>
<td>Sigma-Aldrich</td>
<td>Cat#Z273287-1EA</td>
</tr>
<tr>
<td>Disposable syringes, Omnifix 5 ml with Luer-lock</td>
<td>B. Braun</td>
<td>Cat#8728810F</td>
</tr>
<tr>
<td>Technical Buffer pH 4.01</td>
<td>WTW</td>
<td>Cat#108 800</td>
</tr>
<tr>
<td>Technical Buffer pH 7.00</td>
<td>WTW</td>
<td>Cat#108 802</td>
</tr>
<tr>
<td>Hamilton Storage Solution</td>
<td>Hamilton</td>
<td>Cat#238931</td>
</tr>
<tr>
<td>EasyFerm Bio PHI K8 120</td>
<td>Hamilton</td>
<td>Cat#243632-1513</td>
</tr>
<tr>
<td>OxyFerm FDA 225</td>
<td>Hamilton</td>
<td>Cat#237452</td>
</tr>
<tr>
<td>DASbox Mini bioreactor system for cell culture applications</td>
<td>Eppendorf</td>
<td>Cat#76DX04CC</td>
</tr>
<tr>
<td>DASbox Mini bioreactor vessel for cell culture applications</td>
<td>Eppendorf</td>
<td>Cat#76DS0250ODSS</td>
</tr>
<tr>
<td>DASbox exhaust system</td>
<td>Eppendorf</td>
<td>Cat#76DXOFF</td>
</tr>
<tr>
<td>DASbox exhaust condenser, Peltier</td>
<td>Eppendorf</td>
<td>Cat#76DXCOND</td>
</tr>
<tr>
<td>DASbox overhead drive</td>
<td>Eppendorf</td>
<td>Cat#76DXOHD</td>
</tr>
<tr>
<td>Pitched-Blade Impeller, 8-blade, 60° pitch, stainless steel, O.D. 34 mm, I.D. 5 mm</td>
<td>Eppendorf</td>
<td>Cat#78100604</td>
</tr>
<tr>
<td>Holding Sleeve, for 8-blade impeller, stainless steel with set screw, I.D. 8 mm, for shaft with O.D. 5 mm</td>
<td>Eppendorf</td>
<td>Cat#78100595</td>
</tr>
<tr>
<td>Compression Fitting, complete, with Pg 13.5 male thread, I.D. 12 mm</td>
<td>Eppendorf</td>
<td>Cat#78532284</td>
</tr>
<tr>
<td>Triple Port, Pg 13.5 thread, 3 tubes with O.D. 4 mm × L 85 mm, all parts included</td>
<td>Eppendorf</td>
<td>Cat#78706414</td>
</tr>
<tr>
<td>Pipe, stainless steel, with barb, O.D. 4 mm/I.D. 2 mm, L 225 mm</td>
<td>Eppendorf</td>
<td>Cat#78107023</td>
</tr>
<tr>
<td>Compression Fitting, complete, with Pg 13.5 male thread, I.D. 6 mm</td>
<td>Eppendorf</td>
<td>Cat#78532283</td>
</tr>
<tr>
<td>L-Sparger, stainless steel, complete, O.D. 6 mm, L 300 mm, W 63 mm</td>
<td>Eppendorf</td>
<td>Cat#77102022</td>
</tr>
<tr>
<td>Pump Head Tubing, for DASGIP MP8 pump, Bioprene, I.D. 0.5/W 1.05 mm, female/female</td>
<td>Eppendorf</td>
<td>Cat#78510118</td>
</tr>
<tr>
<td>Pump Head Tubing, for DASGIP MP8 pump, Bioprene, I.D. 1.0/W 1.05 mm, male/female</td>
<td>Eppendorf</td>
<td>Cat#78510109</td>
</tr>
<tr>
<td>Feed Line, with 2× Luer lock fittings, male/male, C-Flex, I.D. 0.8 mm, L 1 m</td>
<td>Eppendorf</td>
<td>Cat#78510309</td>
</tr>
<tr>
<td>Feed Line, with 2× Luer lock fittings, male/male, C-Flex, I.D. 0.8 mm, L 2 m</td>
<td>Eppendorf</td>
<td>Cat#78510310</td>
</tr>
<tr>
<td>Sampling Accessory, with swabable valve</td>
<td>Eppendorf</td>
<td>Cat#78510145</td>
</tr>
<tr>
<td>Polytetrafluoroethylene (PTFE) membrane inline vent filter; pore size 0.2 μm</td>
<td>mdi Membrane Technologies</td>
<td>Cat#ITFX0801BBXX109</td>
</tr>
<tr>
<td>Silicone tubing, inner diameter (i.d.) 1.0 mm, outer diameter (o.d.) 3.0 mm</td>
<td>Carl Roth</td>
<td>Cat#HC61.1</td>
</tr>
<tr>
<td>Silicone tubing, inner diameter (i.d.) 4.0 mm, outer diameter (o.d.) 6.0 mm</td>
<td>Carl Roth</td>
<td>Cat#HC65.2</td>
</tr>
<tr>
<td>hose reduction piece</td>
<td>Carl Roth</td>
<td>Cat#CT46.1</td>
</tr>
<tr>
<td>Female Luer-lock connector</td>
<td>Carl Roth</td>
<td>Cat#CT62.1</td>
</tr>
<tr>
<td>Male Luer-lock plug</td>
<td>Carl Roth</td>
<td>Cat#CT70.1</td>
</tr>
<tr>
<td>Connector, straight, female luer lock/tubing nipple, 4.8 mm</td>
<td>VWR</td>
<td>Cat#INFIPP-LFS48</td>
</tr>
<tr>
<td>Screw cap GL 45 with 2 hose connectors and EPDM gasket</td>
<td>Landgraf Laborsysteme (HLL)</td>
<td>Cat#102112807</td>
</tr>
<tr>
<td>Media bottle,250ml, two 9mm hose nozzles</td>
<td>Landgraf Laborsysteme (HLL)</td>
<td>Cat#L14040250</td>
</tr>
<tr>
<td>DASGIP Parallel Bioreactor System, for cell culture</td>
<td>Eppendorf</td>
<td>Cat#76DG04CCBB</td>
</tr>
<tr>
<td>DASGIP Vessel, DS1000ODSS, 350 mL – 1.0 L, 2× GL45 side arms</td>
<td>Eppendorf</td>
<td>Cat#76DS1000ODSS</td>
</tr>
<tr>
<td>Pitched-Blade Impeller, 8-blade, 60° pitch, stainless steel, O.D. 53 mm, I.D. 8 mm</td>
<td>Eppendorf</td>
<td>Cat#78100605</td>
</tr>
<tr>
<td>Holding Sleeve, for 8-blade impeller, stainless steel with set screw, I.D. 10 mm, for shaft with O.D. 8 mm</td>
<td>Eppendorf</td>
<td>Cat#78100597</td>
</tr>
<tr>
<td>Pump Head Tubing, for DASGIP® MP8 pump, Peripren, I.D. 2.0/W 0.8 mm, female/female</td>
<td>Eppendorf</td>
<td>Cat#78510237</td>
</tr>
<tr>
<td>EasyFerm Bio PHI K8 225</td>
<td>Hamilton</td>
<td>Cat#243632-1543</td>
</tr>
<tr>
<td>Media bottle,1000ml, two 9mm hose nozzles</td>
<td>Landgraf Laborsysteme (HLL)</td>
<td>Cat#L14040101</td>
</tr>
<tr>
<td>Sintered glass sparger, AD 6 × 15mm, Por.3, pore size 16–40 μm, glass tube AD 4 × 0.8 mm, L 35mm</td>
<td>Eppendorf</td>
<td>Cat#78903230</td>
</tr>
<tr>
<td>Stainless Steel Flange</td>
<td>Eppendorf</td>
<td>Cat#78107292</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<ul>
<li>E8 basis medium (10 L)
<ul>
<li>While working under the flow food, pour 20 bottles a 0.5 L of DMEM/F12 into a 10L glass bottle, while keeping the original bottles sterile.</li>
<li>Add 5.43 g NaHCO<sub>3</sub>, adjust the pH to 7.4 with 5 M NaOH (should be 3–5 mL). Check the osmolality (should be 315–320 mOsm/kg).</li>
<li>Filter to sterilize, aliquot a 500 mL in the old bottles and store at 4°C for up to 6 month.</li>
</ul>
</li>
<li>E8 feed basis medium I (10 L)
<ul>
<li>Follow the instructions for E8 basis medium, and additionally add 30 g Glucose. Osmolality should be 335–340 mOsm/kg.</li>
</ul>
</li>
<li>E8 feed basis medium II (10 L)
<ul>
<li>Follow the instructions for E8 basis medium, and additionally add 45 g Glucose. Osmolality should be 350–355 mOsm/kg.</li>
</ul>
</li>
<li>Rho-kinase inhibitor (RI) Y-27632 stocks (1000×, 10 mM)
<ul>
<li>Dissolve 50 mg Y-27632 in 14.7 mL pure, filter to sterilize, aliquot a 1000 μL store at – 20°C for up to 12 month.</li>
</ul>
</li>
<li>bFGF stocks (1000×, 100 μg/mL)
<ul>
<li>Add 300 μL of 10% Bovine serum albumin (BSA)-solution to 29.7 mL PBS to make a 0.1% BSA-solution.</li>
<li>Dissolve 3000 μg recombinant bFGF (dehydrolyzed powder) in 30.0 mL of the 0.1 BSA-solution, filter to sterilize, aliquot a 1000 μL store at – 80°C for up to 12 month.</li>
</ul>
</li>
<li>TGF-β1 stocks (1000 ×, 2 μg/mL)
<ul>
<li>Add 650 μL of 10% human albumin solution + 520 μL of 1 M HCl to 128.83 mL PBS to make a 0.05% human albumin/ 4 mM HCl solution and filter to sterilize.</li>
<li>Dissolve 250 μg recombinant TGF-β1 (dehydrolyzed powder) in 125.0 mL of the prepared human albumin/HCl solution, aliquot a 1000 μL store at – 80°C for up to 12 month.</li>
</ul>
</li>
<li>Transferin stocks (1000 ×, 10.7 mg/mL)
<ul>
<li>Dissolve 1.0 g recombinant Transferin (dehydrolyzed powder) in 93.46 mL PBS, filter to sterilize, aliquot a 1000 μL store at – 80°C for up to 12 month.</li>
</ul>
</li>
<li>L-ascorbic acid 2-phosphat stocks (1000 ×, 64 mg/mL)
<ul>
<li>Dissolve 5.0 g L-ascorbic acid 2-phosphat in 78.13 mL PBS, filter to sterilize, aliquot a 1000 μL store at – 20°C.</li>
</ul>
</li>
<li>Sodium Selenite stocks (10000 ×, 140 μg/mL)
<ul>
<li>Dissolve 0.014 g Sodium Selenite in 100 mL PBS, filter to sterilize, aliquot a 100 μL and store at – 20°C for up to 12 month.</li>
</ul>
</li>
</ul>
<table id="undtbl1">
<caption>E8 adherent medium (for adherent (pre-) culture)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>E8 basis medium</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
<tr>
<td>FGF2 (100 μg/mL)</td>
<td>100 μg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>TGF-β1 (2 μg/mL)</td>
<td>2 μg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Insulin (10 mg/mL)</td>
<td>20 mg/L</td>
<td>1 mL</td>
</tr>
<tr>
<td>Ascorbic acid-2-phosphate (64 mg/mL)</td>
<td>64 mg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Transferrin (10.7 mg/mL)</td>
<td>10.7 mg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Sodium Selenite (140 μg/mL)</td>
<td>14 μg/L</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>RI (Y-27632) (10 mM; only after passaging)</td>
<td>10 μM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>503.05 mL (503.55 mL)</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Stable under the following storage conditions: 4°C and maximum storage time of 2 weeks.</div>
<table id="undtbl2">
<caption>E8 suspension medium (for bioreactor inoculation)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>E8 basis medium</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
<tr>
<td>FGF2 (100 μg/mL)</td>
<td>100 μg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>TGF-β1 (2 μg/mL)</td>
<td>2 μg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Insulin (10 mg/mL)</td>
<td>20 mg/L</td>
<td>1 mL</td>
</tr>
<tr>
<td>Ascorbic acid-2-phosphate (64 mg/mL)</td>
<td>64 mg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Transferrin (10.7 mg/mL)</td>
<td>10.7 mg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Sodium Selenite (140 μg/mL)</td>
<td>14 μg/L</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>RI (Y-27632) (10 mM)</td>
<td>10 μM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Pluronic™ F-68 Non-ionic Surfactant (10%)</td>
<td>0.1%</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>508.55 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Stable under the following storage conditions: 4°C and maximum storage time of 2 weeks.</div>
<table id="undtbl3">
<caption>E8 full feed medium I (for perfusion feeding from day 1 – day 4)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>E8 basis feed medium I</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
<tr>
<td>FGF2 (100 μg/mL)</td>
<td>100 μg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>TGF-β1 (2 μg/mL)</td>
<td>2 μg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Insulin (10 mg/mL)</td>
<td>20 mg/L</td>
<td>1 mL</td>
</tr>
<tr>
<td>Ascorbic acid-2-phosphate (64 mg/mL)</td>
<td>64 mg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Transferrin (10.7 mg/mL)</td>
<td>10.7 mg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Sodium Selenite (140 μg/mL)</td>
<td>14 μg/L</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>Pluronic™ F-68 Non-ionic Surfactant (10%)</td>
<td>0.1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>L-Glutamine (200 mM)</td>
<td>+ 2 mM (4.5 mM in total)</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>513.05 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Stable under the following storage conditions: 4°C and maximum storage time of 2 weeks.</div>
<table id="undtbl4">
<caption>E8 full feed medium II (for perfusion feeding from day 4 – day 7)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>E8 basis feed medium II</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
<tr>
<td>FGF2 (100 μg/mL)</td>
<td>100 μg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>TGF-β1 (2 μg/mL)</td>
<td>2 μg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Insulin (10 mg/mL)</td>
<td>20 mg/L</td>
<td>1 mL</td>
</tr>
<tr>
<td>Ascorbic acid-2-phosphate (64 mg/mL)</td>
<td>64 mg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Transferrin (10.7 mg/mL)</td>
<td>10.7 mg/L</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Sodium Selenite (140 μg/mL)</td>
<td>14 μg/L</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>Pluronic™ F-68 Non-ionic Surfactant (10%)</td>
<td>0.1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>L-Glutamin (200 mM)</td>
<td>+ 2.5 mM (5.0 mM in total)</td>
<td>7.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>515.55 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Stable under the following storage conditions: 4°C and maximum storage time of 2 weeks.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Thawing and adherent (pre-) culture of hPSCs for bioreactor inoculation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 9 days</div>
<p>The aim of this part of the protocol is to describe well-defined adherent (two dimensional, 2D) culture of hPSCs to generate the required quality and quantity of cells required for inoculating the suspension culture (3D) in bioreactors at the desired culture scale. The procedure described here details the generation of hPSCs sufficient to inoculate 4 × bioreactors at 150 mL scale, or alternatively 1 × bioreactor at 500 mL scale. When aiming at larger culture scales, one or two additional cell passages might be required to produce the respective cell numbers in 2D.</p>
<ol>
<li>Thawing of cells
<ol type="a">
<li>Start with a high quality aliquot i.e., a vial of frozen hPSCs representing ∼3 × 10<sup>6</sup> cells (previously generated e.g., by own adherent culture used for cryo-preservation).</li>
<li>Thaw the vial at 37°C in a water bath until only a small rest of ice is visually observable.</li>
<li>Spray the vial with ethanol and place it under a flow hood and dilute the suspension in 10-fold volume of DMEM/12 in a 15 mL conical tube.</li>
<li>Centrifuge at 300 × g for 3 min. Discard the supernatant and resuspend in 15 mL E8 adherent medium + 10 μM Y27632. Remove the coating medium from one coated T75 flask and transfer the cell suspension into the flask.</li>
<li>Place the T-flask in an incubator at 37°C and 5% CO<sub>2</sub>.</li>
</ol>
</li>
<li>After 48 h of incubation, replace the old medium with 15 mL fresh E8 adherent medium and place the flask back in an incubator.</li>
<li>Passaging of cells
<ol type="a">
<li>72 h after seeding (ideally not later), the cells should be passaged to ensure that cells remain at proliferative conditions optimal for this protocol; at this stage the culture confluence should be at ∼60–80% (exemplary pictures are shown in <a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Exemplary microscopical pictures of Monolayer cell growth</div>
<p>(A–E) Exemplary bright field images of a monolayer culture on Geltrex and in E8, directly after seeding (A), on day 1 (B), day 2 (C), day 3 (D) and of the flask after cell detachment in order to highlight good cell detachment (E) (scale bars = 200 µm).</p>
</figcaption>
</figure>
</li>
<li>For passaging, wash cells with PBS, completely remove any residue of PBS and add 0.04 mL/cm<sup>2</sup> StemPro™ Accutase™ and incubate for 3 min at 37°C. Check for cell detachment by tilting the flask.</li>
<li>Dilute Accutase with the 3-fold volume of DMEM/F12, transfer into a conical tube and centrifuge at 300 × g for 3 min. Discard the supernatant, resuspend in 5 mL E8 adherent medium + 10 μM Y27632 and determine the cell count with the Vi-CELL XR (alternatively manually via Trypan blue exclusion and Neubauer chamber (<a href="#bib16">Zweigerdt et al., 2011</a>)).</li>
<li>Remove the coating medium from three coated T175 flasks, add 35 mL E8 adherent medium + 10 μM Y27632 (0.2 mL/cm<sup>2</sup>) containing 3.5 × 10<sup>6</sup> cells per flask (corresponding to 2 × 10<sup>4</sup> cells/cm<sup>2</sup>).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If Vitronectin (instead of Geltrex) is used for the adherent culture, Versene might be used for cell detachment/passaging alternatively to Accutase; the procedure is identical to the steps above but the incubation time should be expanded to 10 min (as compared to 3 min suggested for Accutase treatment).</div>
<ol start="4">
<li>After 48 h of incubation, replace the medium with 35 mL fresh E8 adherent medium per flask and place the flask back in an incubator.</li>
<li>72 h after seeding, the cells should have reached ∼ 60–80% confluence and must be passaged to ensure optimal proliferative conditions for this protocol.
<ol type="a">
<li>For passaging, wash cells with PBS, completely remove any residue of PBS and add 0.04 mL/cm<sup>2</sup> StemPro™ Accutase™ and incubate for 3 min at 37°C. Check and support cell detachment by tilting of the flask.</li>
<li>Dilute Accutase with the 3-fold volume of DMEM/F12, transfer into a conical tube and centrifuge at 300 × g for 3 min. Discard the supernatant, resuspend in 25 mL E8 adherent medium + 10 μM Y27632 and determine the cell density with the Vi-CELL XR (alternatively manually via Trypan blue exclusion and Neubauer chamber).</li>
<li>Remove the coating medium from twelve coated T175 flasks, add 35 mL E8 adherent medium + 10 μM Y27632 (0.2 mL/cm<sup>2</sup>), and 3.5 × 10<sup>6</sup> cells per flask (2 × 10<sup>4</sup> cells/cm<sup>2</sup>).</li>
</ol>
</li>
<li>After 48 h of incubation, replace the old medium with 35 mL fresh E8 adherent medium per flask and place the flasks back in an incubator.</li>
<li>Observe the cells 72 h after seeding. The confluence should be ∼60%–80% and the medium should be slightly yellow and clear. For the inoculation of 4 × bioreactors at 150 mL scale, or 1 × bioreactor at 500 mL scale, the next major step “Bioreactor inoculation” should be followed; otherwise (to generate more cells for inoculating multiple bioreactors and/ or larger scale processes) repeat the steps 3–5, while respectively increased the number of T175 flasks.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The seeding density into T-flasks might require slight (cell line-dependent) adjustment for your hPSC line at hand. However, it is highly critical that cells do not exceed the suggested ∼60%–80% confluence within the 72 h suggested for passaging; higher confluence and/ or exceeding the passaging time beyond 72 h may negatively impact on cell viability after bioreactor inoculation and subsequently reduce the process robustness, efficiency and overall protocol reproducibility. Moreover, even if the cells are not fully detached after 3 minutes of Accutase treatment, do not prolong this incubation step, to avoid detrimental effects on cell viability and proliferation; if the confluence and the passaging interval is following the above-described range, 3 min of Accutase treatment is resulting an optimal detachment and dissociation.</div>
<h3 id="sec3.2">Preparation and assembly of the bioreactor system (DASbox; for 100–250 mL process scale)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days before bioreactor inoculation</div>
<ol start="8">
<li>Siliconizing of the bioreactor glass vessel
<ol type="a">
<li>Place the bioreactor glass vessel under the flow hood and repeatedly pipet 1 mL of Sigmacote to the vessel wall (to at least two-thirds of height) and bottom until no liquid is left. This step is necessary to decrease the chance of cells to the vessel to an increased hydrophobicity of the glass. Let the vessel dry for at least 12 h at 20°C–22°C.</li>
<li>Rinse the bioreactor vessel with water in order to remove any Sigmacote residues.</li>
</ol>
</li>
<li>Pump calibration (exemplarily shown in <a href="#fig3">Figure 3</a>)
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Display of the pump calibration process</div>
<p>(A) Exemplary image of a pump calibration process.</p>
<p>(B) Exemplary of the software during pump calibration.</p>
</figcaption>
</figure>
<ol type="a">
<li>Assemble the base, feed and waste tubing. Important: when the pump head tubing is placed in the pump heads, one tubing end must be long enough to allow placement of tube-attached bottles under the flow hood while being still attached to the bioreactor.
<ol type="i">
<li>The pump head tubing for the base tubing should have an inner diameter of 0.5 mm. For both the feed and waste tubing, in addition to the 0.5 mm pump head tubing a 1.0 mm pump head tubing must be attached.</li>
</ol>
</li>
<li>Place the pump head tubing of base, feed and waste (starting with the bigger diameter for feed and waste) into the pump heads. Use the DASware software for pump calibration. Prime the lines with pure water.</li>
<li>Weigh empty conical tubes with caps and place the tubing outlets in each conical tube.</li>
<li>Start dispensing at a defined flow rate, which is applied during the cultivation.</li>
<li>Weigh the filled conical with caps and insert the weight of the dispensed water to the software interface.</li>
<li>For feed and waste lines, note the calibration values and repeat steps b–e while using the smaller diameter pump head tubing.</li>
</ol>
</li>
<li>pH probe calibration
<ol type="a">
<li>Remove the pH probe from the storage solution, examine the pH probe’s diaphragm for damages and attach the probe to the cable of the bioreactor</li>
<li>Rinse with water, dry with a tissue without rubbing on the diaphragm, and place both the pH probe and the respective temperature probe into the pH 7.00 calibration buffer. It is important to ensure that the diaphragm is fully submerged in the buffer.</li>
<li>After pH and temperature readings have stabilized, calibrate the offset</li>
<li>Repeat steps b and c using pH 4.01 calibration buffer for slope calibration.</li>
</ol>
</li>
<li>Assembly of the bioreactor
<ol type="a">
<li>Assemble the head plate considering the schematic in <a href="#fig4">Figures 4</a> and <a href="#fig5">5</a>.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Schematic overview of the headplate arrangement for both DASbox and DASGIP Bioblock bioreactor systems</div>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Schematic overview of the assembly of bottles to the DASbox bioreactor</div>
<p>(A) Exemplary images of the connection of bottles to the bioreactor. Starting with the waste bottle, followed by feed, base and inoculation.</p>
<p>(B) Side and top view of the assembled bioreactor in order to highlight connections.</p>
</figcaption>
</figure>
</li>
<li>To enable sampling without process interruption, add a piece of silicone tubing (i.d. 1.0 mm) with a Mohr pinchcock clamp, a female Luer-Lock connector and a Luer-Lock sampling valve to the sample port. Fix connections with cable ties.</li>
<li>Exhaust and inlet air filter:
<ol type="i">
<li>Add a piece of silicone tubing (i.d. 4.0 mm) with a vent filter to the exit port of the exhaust gas line.</li>
<li>Assemble a piece of small silicone tubing (i.d. 1.0 mm) with a piece of larger silicone tubing (i.d. 4.0 mm) with the help of a hose reduction piece.</li>
<li>Attach a vent filter to the larger tube and connect everything to the headspace gassing port (for overlay gassing); fix connections with cable ties.</li>
</ol>
</li>
<li>Attach the impeller (Pitched-Blade Impeller, 8-blade, 60° pitch) at the lowest-possible position of the impeller shaft and tighten with screws in the holding sleeve.</li>
<li>Attach the sintered glass frit with a small piece of tubing to the dip tube and position it directly above the impeller. It will function as a cell retention filter while perfusion.</li>
<li>Attach base-, feed- and waste-tubing with bottles and respective ports at the bioreactor head plate. Note that the longer part of the tubing should always be attached to the bottle (but not to the head plate).</li>
<li>To prepare the inoculation bottle:
<ol type="i">
<li>attach a silicone tubing (i.d. 4.0 mm; o.d. 6.0 mm) with the hose nozzle at the bottom of the inoculation bottle (Media bottle with two 9 mm hose nozzles).</li>
<li>Subsequently, add a piece of silicone tubing (i.d. 4.0 mm) with a vent filter to the second hose nozzle at the top of the bottle.</li>
<li>Attach the Inoculation Bottle to the respective port at the bioreactor.</li>
</ol>
</li>
</ol>
</li>
<li>Bioreactor sterilization
<ol type="a">
<li>Cover the bottom of the bioreactor vessel with pure water in order to allow steam autoclaving and to prevent the pH probes from drying out.</li>
<li>Wrap vent filters with aluminum foil to protect from humidity. Close silicone tubing of the sampling port with the Mohr pinchcock clamp.</li>
<li>Autoclave bioreactor at 120°C for 20 min.</li>
<li>Place the bioreactor vessel into the bioreactor station and connect to the overhead drive, exhaust condenser, temperature sensor, gas supply to the overlay port, and connect the cables of the pH and DO probe.</li>
<li>Remove the water and replace it with 100 mL of sterile PBS with the help of the inoculation bottle. This step should be conducted as soon as possible to prevent the pH probe’s diaphragm from draining.</li>
<li>Place the base tubing in pump B, the feed tubing in pump C and the waste tubing in pump D.</li>
<li>Fill the base bottle with 100 mL of NaHCO<sub>3</sub> and prime the tubing until one drop of base is dropping into the bioreactor.</li>
</ol>
</li>
<li>Dissolved oxygen sensor calibration
<ol type="a">
<li>Start overlay gassing with 21% O<sub>2</sub>, temperature control at 37°C and stirring at 80 rpm. Let the DO probe polarize for at least 6 h.</li>
<li>The next day, perform one-point slope calibration at 100% DO.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Additionally apply gassing with pure N<sub>2</sub> for another 6 h followed by zero point calibration.</div>
<ol start="14">
<li>Software set-up (shown in <a href="#fig6">Figure 6</a>)
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Display of the software set up for bioreactor start</div>
</figcaption>
</figure>
<ol type="a">
<li>To set up the software first click on “New Workflow”</li>
<li>Under the tab “Default Template” select Control (Cell Cultivation: pH Control (CO2/base); DO Control (XO2, F)). Then select the respective number of reactors to be used and press “open”.</li>
<li>Remove the tick at pH Calibration and DO Calibration, change the name of the process from “Manager X” to your specific process. Subsequently select the bioreactor units to be used; <b>Optional:</b> name each unit and press “Next”</li>
<li>In the next window, change the settings by pressing on “Feed C”, the tab “Pump C (Sub.)” and change “V.Mode” to “Ignore”; repeat for “Feed D”. Next press “pH” and the tab ”pH Control” and change the values at “Min” to “−25%” and at “Max” to “10 mL/h”. Subsequently select the tab “Pump B (Base)” and change “V.Mode” to “Ignore”. Finally press “Next”.</li>
<li>Copy the Script by clicking on the individual Unit and press on “Vessel”, select the tab “Scripting”, paste the Script (available under <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Mmc2.pdf">Data S2</a>), adjust the volume to your specific working volume and press “Apply”.</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Preparation and assembly of bioreactor system for larger process scale (DASGIP Bioblock; for 300–1000 mL process scale)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days before bioreactor inoculation</div>
<ol start="15">
<li>The Preparation of the DASGIP Bioblock is analogous to the preparation of the DASbox. Therefore, only changes between the DASGIP Bioblock and the DASbox preparation will be described here.</li>
<li>Change to 8. a.
<ol type="a">
<li>Use 1.5 mL of Sigmacote.</li>
</ol>
</li>
<li>Change to 9. a. i.
<ol type="a">
<li>The tube diameter for feed and waste should be 1.0 mm + 2.0 mm.</li>
</ol>
</li>
<li>Change to 12. e.
<ol type="a">
<li>Fill the bioreactor with 400 mL of PBS after autoclaving.</li>
</ol>
</li>
<li>Change to 12. g.
<ol type="a">
<li>Fill the Base bottle with 400 mL of NaHCO<sub>3</sub>.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Bioreactor inoculation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>This section describes how inoculate the bioreactor after the bioreactor was set-up along the description in “Preparation and assembly of the Bioreactor system” (shown in <a href="#fig7">Figure 7</a>)</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Exemplary images of how an inoculation bottle is used to transfer media and cell suspension into the bioreactor</div>
</figcaption>
</figure>
<ol start="20">
<li>Place the inoculation bottle on a lower level than the bioreactor and transfer the PBS from the bioreactor into the inoculation bottle, by gravity. Tilt the bioreactor to further transfer remaining PBS. Place the inoculation bottle (while being attached to the bioreactor) under the flow hood and discard the PBS.</li>
<li>Add 110 mL of fresh E8 suspension medium for bioreactor inoculation to the inoculation bottle, place the inoculation bottle on a higher level than the bioreactor and let it flow by gravity into the bioreactor.</li>
<li>Initiate bioreactor agitation, gassing and temperature control.</li>
<li>Dissociate the cells from the T-flasks as described in 5; here, the use of a 500 mL centrifugation tube (instead of 50 mL conical tubes) facilitates the process. Resuspend the cell pellet in 100 mL of fresh E8 suspension medium for bioreactor inoculation and determine the cell density of the resulting cell suspension applying the Vi-CELL XR.</li>
<li>Calculate the volume of your cell suspension that contains 75 million cells (this cell number is required to achieve the final cell density 0.5 × 10<sup>6</sup> cells/mL in 150 mL process scale) and transfer the calculated volume into the bioreactor utilizing the inoculation bottle, as described in 21.</li>
<li>Add the remaining volume of fresh E8 suspension medium for bioreactor inoculation to fill up to 150 mL to the inoculation bottle and transfer the medium into the bioreactor; this step is required to flush the tubing, thereby avoiding any cell loss in the tubing.</li>
<li>Using the software interface, click on Inoculation and Press Start. This step is critical since it defines the (inoculation-) timing for the entire process script. Bioreactor settings are displayed in <a href="#tbl2">Table 2</a>.
<table id="tbl2">
<caption>Table 2. Bioreactor settings (exemplarily for a 150 mL DASbox bioreactor)</caption>
<thead>
<tr>
<th>Stirring</th>
<th>Temperature</th>
<th>Gassing</th>
<th>pH</th>
<th>DO</th>
<th>Perfusion</th>
</tr>
</thead>
<tbody>
<tr>
<td>80 rpm (exact calculation based on <a href="#fd13">Equations 13</a> + <a href="#fd14">14</a>) ccw</td>
<td>37°C</td>
<td>0.9 sL/h (reactor volume × 6), 21% O<sub>2</sub>, 5% CO<sub>2</sub>
</td>
<td>7.1</td>
<td>40%</td>
<td>see <a href="#tbl3">Table 3</a>
</td>
</tr>
</tbody>
</table>
<table id="tbl3">
<caption>Table 3. Applied feed rates and feed media for respective culture periods</caption>
<thead>
<tr>
<th>Culture day</th>
<th>Feed rate in working volumes per day [1/d]</th>
<th>Feed medium</th>
</tr>
</thead>
<tbody>
<tr>
<td>0–1</td>
<td>0</td>
<td>N/A</td>
</tr>
<tr>
<td>1–2</td>
<td>1</td>
<td>E8 full feed medium I</td>
</tr>
<tr>
<td>2–3</td>
<td>1.5</td>
<td>E8 full feed medium I</td>
</tr>
<tr>
<td>3–4</td>
<td>3</td>
<td>E8 full feed medium I</td>
</tr>
<tr>
<td>4–5</td>
<td>4</td>
<td>E8 full feed medium II</td>
</tr>
<tr>
<td>5–6</td>
<td>6</td>
<td>E8 full feed medium II</td>
</tr>
<tr>
<td>6–7</td>
<td>7</td>
<td>E8 full feed medium II</td>
</tr>
</tbody>
</table>
</li>
<li>As described in “Sampling from the bioreactor”, draw a first process sample at 30 min after inoculation.
<ol type="a">
<li>The sample can be directly measured via the Vi-CELL XR to test/confirm the desired inoculation cell density aiming at 0.5 × 10<sup>6</sup> cells/mL; no enzymatic dissociation of the sample is necessary at this point since the cells have not yet attached into aggregates.</li>
</ol>
</li>
<li>Fill the feed bottle of the bioreactor with E8 full feed medium I and II; two options can be applied:
<ol type="a">
<li>If a dedicated fridge is available, the feed bottle can be placed in the fridge at 4°C. Such cooling allows depositing the entire feed medium required for the first 4 days of process duration.
<ol type="i">
<li>Required volume calculation: process scale multiplied with sum of working volumes during this time span (see <a href="#tbl2">Table 2</a>) i.e., 150 × 5.5 = 825 mL of E8 full feed medium I. After 4 days of process duration, additional 2.55 L (at the 150 mL scale) of E8 full feed medium II need to be filled in the feed bottle.</li>
</ol>
</li>
<li>Without the cooling option, only 150 mL of E8 full feed medium I required for the next 24 h of feeding should be deposited in the feed bottle at 20°C–22°C. Subsequently, the feed bottle needs to be refilled on a daily basis based on the applied feed rate (see <a href="#tbl2">Table 2</a>).</li>
</ol>
</li>
<li>After filling the feed bottle, prime the feed lines.</li>
<li>Due to high cell growth, headspace gassing is not sufficient for the whole process duration. Therefore, it needs to be changed to submerse gassing, with a lower gassing rate in order not to damage aggregates by rising bubbles.
<ol type="a">
<li>After 4 days of process duration. Go to the bioreactor vessel script as described in 14.e. and change the line “Dim FSP as double = VR ∗ 6” to “Dim FSP as double = VR ∗ 1”. Subsequently detach the gassing from the overlay port and attach it to the submerged port in order to supply more oxygen to the culture.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Conduct the process equivalent to steps 20–30 applying the DASGIP Bioblock bioreactor system enabling process scales ranging at 300–1000 mL.</div>
<h3 id="sec3.5">Sampling from the bioreactor</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h, daily</div>
<p>This section describes how to draw cell suspension samples from the bioreactor for monitoring cell aggregation and growth kinetics without process interruption. Sampling and sample analysis should be conducted every 24 h (or more frequently on demand).</p>
<ol start="31">
<li>Remove the cap from the bioreactor sample port and spray with 70% Ethanol.</li>
<li>Connect a sterile syringe to the valve, create a slight underpressure by pulling the plunger, open the Mohr pinchcock clamp, slowly draw 1 mL of cell suspension into the syringe and close the clamp immediately.</li>
<li>After the syringe has been unplugged, discard the cell suspension in order to remove residues that accumulated in the sample line since the last sampling. Subsequently, draw another 2.5 mL of sample volume. Fill the sample into a well of a low attachment 6 well plate. Clean the sample port with 70% ethanol before closing the cap.</li>
<li>To assess aggregate morphological appearance and systematic size determination, conduct microscopic images of the hPSC aggregates directly after sampling. Subsequently, transfer 2 mL of the suspension sample into tubes and centrifuge at 300 × g for 3 min. The supernatant can be stored at −20°C before metabolic analysis e.g., to assess glucose, lactate or amino acid contents.</li>
<li>For cell counting, wash the obtained cell pellet carefully with 1 mL PBS (avoiding aggregate loss), add 500 μL of Accutase, mix well and incubate for 3 min at 37°C. Centrifuge at 300 × g for 3 min, discard the supernatant, resuspend in 2 mL PBS and determine the cell density with the Vi-CELL XR. Cells can be used for additional analysis such as gene expression patterning, cell cycle assessment or flow cytometry-based monitoring of pluripotency marker expression.</li>
</ol>
<h3 id="sec3.6">Flow cytometry-based analysis of pluripotency markers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="36">
<li>Use dissociated cells from the previous step at ∼150000 cells/ well of a 96 well plate.</li>
<li>The pluripotency-associated surface markers SSEA-3, SSEA-4 and TRA-1-60 can be triple stained in 50 μL PBS/well applying the following dilutions: 1:25 for SSEA-4 and TRA-1-60, 1:50 (for SSEA-3). Incubate for 12 min at 20°C–22°C.</li>
<li>For the intracellular pluripotency- and proliferation-associated markers OCT-3/4, NANOG and KI-67, respectively, treat ∼150000 cells/ well with FIX&amp;PERM® Solution A for 15 min at 20°C–22°C. Conduct triple staining in 50 μL PBS + FIX&amp;PERM® Solution B (1:1 ratio) for 12 min at 20°C–22°C using 1:25 dilutions of each antibody.</li>
</ol>
<h3 id="sec3.7">Cleaning of the bioreactor</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This section describes how to properly clean the bioreactor vessels and tubing before re-usage.</p>
<ol start="39">
<li>For process termination, remove the cell suspension using the inoculation bottle as described in 20. The cells can be further processed as required.</li>
<li>Fill the bioreactor with 200 mL of Terg-a-zyme® solution, set the stirring at 150 rpm and incubate at 37°C for at least 12 h.</li>
<li>Detach all tubing from the bioreactor, place the detached ends into a 500 mL beaker with ∼500 mL of distilled water and pump backwards into the attached feed-, waste- and base- bottles for 2 h.</li>
<li>Detach the tubes from the bottles and place the other end of the tubes into the distilled water-filled beaker to pump in circles for at least 12 h.</li>
<li>Rinse all bioreactor ports with Terg-a-zyme® solution. Rinse the cell retention filter thoroughly to avoid future clogging.</li>
<li>On the next day, remove the Terg-a-zyme® solution from the bioreactor and replace with distilled water, rinse the ports with distilled water and incubate for 4 h.</li>
<li>Remove tubes from the beaker. Continue to pump for 30 min in order to remove liquid residues from tubes.</li>
<li>Unscrew DO and pH sensors from the head plate. Place the pH sensor back into the storage solution.</li>
<li>Remove the water from the bioreactor vessel and pat dry.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Application of the protocol should enable a cell culture-trained investigator establishing a highly efficient hPSC expansion process characterized by the exponential proliferation of seeded cells between process days 2–5, while further promoting the substantial increase in cell numbers until the suggested process termination on day 7 (potentially followed by re-seeding or initiation of differentiation). The final cell density expected with this protocol is ∼35 × 10<sup>6</sup> cells/mL on day 7, representing a ∼70-fold expansion of the inoculation density of 0,5 × 10<sup>6</sup> cells/mL (<a href="#fig8">Figure 8</a>A). Starting with the DO setting of 100%, the triggered DO control (set to 40%) is expected to starts at around 30 h post process inoculation (<a href="#fig8">Figure 8</a>B) and the triggered pH control (set to pH 7.1) shall starts at around 20 h post inoculation (<a href="#fig8">Figure 8</a>C). Due to the high cell densities achieved over time, gassing – which is initially performed via the headspace only - needs to be conducted in a submerged manner (described under 42.) at later stages, which results in a “stroke-like DO pattern” on process day 4, followed by stabilization around 40 % DO thereafter (<a href="#fig7">Figure 7</a>B). Notably, on day ∼5–6, even submerged gassing with our bioreactor setup is insufficient to keep the DO at the envisioned 40 %, therefore dropping down to ∼10 % DO on days 6–7 due to the high cell density-induced oxygen consumption. However, the yielded hPSCs at the proposed process endpoint (day 7) are expected to express high levels of pluripotency-associated markers TRA-1-60, SSEA-4, OCT-3/4, NANOG in &gt;95% of the cell whilst &gt;75% of the cells should express SSEA-3, a marker which we have observed as being most sensitive to improper culture condition (<a href="#fig9">Figure 9</a>C). In addition to the expression of pluripotency-associated markers, cells should display a stable and normal karyotype when assessed at process endpoint (<a href="#fig9">Figure 9</a>D).</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Representative expected results for bioprocesses conducted after this protocol</div>
<p>(A) Viable cell density (dots) and viability (squares) for representative processes.</p>
<p>(B) Representative online process parameter measurement of dissolved oxygen (DO) value over process time.</p>
<p>(C) Representative online process parameter measurement of pH value over process time.</p>
</figcaption>
</figure>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Representative expected results for bioprocesses conducted after this protocol</div>
<p>(A) Representative light microscopy pictures of process-derived aggregate samples on days 1, 3, 5 and 7 for 3 different cell lines highlight cell homogeneity within the cultures and the cell line dependent differences in size of formed aggregates(scale bars = 200 μm).</p>
<p>(B) Distribution of aggregate diameters over the cultivation time.</p>
<p>(C) Representative flow cytometry plots of cells harvested at process endpoint (day 7) showing the pluripotency-associated surface markers TRA-1-60, SSEA-4 and SSEA-3 and transcription factors OCT-3/4 and NANOG as well as the proliferation marker KI-67 (isotype controls shown in gray).</p>
<p>(D) Representative karyotype of cells cultured for 7 days under the conditions described here. Figure adapted from (<a href="#bib10">Manstein et al., 2021</a>).</p>
</figcaption>
</figure>
<p>Another characteristic of the process is the induction of a homogeneous aggregate suspension, reaching a (somewhat cell line-dependent) mean aggregate diameter of ∼150 μm–350 μm on process day 7 (<a href="#fig9">Figures 9</a>A and 9B). Our observations based on the application of numerous independent cell lines in this protocol suggest that the relative substantial differences in the average aggregate diameter (observed at a given stirring rate on a given process day; <a href="#fig8">Figure 8</a>B) represents cell line-dependent properties of hPSC. On the molecular level, this might be a consequence of the expression level and/ or processing characteristics of E-cadherin or other cell adhesion molecules, as previously suggested (<a href="#bib8">Konze et al., 2014</a>). However, the hPSC aggregates generated by our protocol can be directly applied for lineage-directed differentiation strategies as we have readily demonstrated for cardiomyocytes (<a href="#bib5">Halloin et al., 2019</a>; <a href="#bib10">Manstein et al., 2021</a>). Notably, however, when aiming at lineage-directed differentiation, we do suggest applying hPSC aggregates harvested already on day 3–4 of the process. In contrast to day 7, the cells proliferate exponentially on process day 3–4 and the aggregate diameter is typically below 300 μm, suggesting no/ low diffusion limits to the cells in the core of aggregates. These facts should therefore support both the efficient re-seeding of day 3–4 derived cells (e.g., for “seed train-based” hPSC mass expansion in continuously increasing scales) as well as the robust differentiation of day 3–4 derived aggregates into a cell type of interest, when applying relevant differentiation strategies.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The below listed equations serve specific requirements in the frame of this protocol as follows.</p>
<ol>
<li>To calculate the cell specific rates that are essential to create the herein used <i>in silico</i> model(s), apply the <a href="#fd1">Equations 1</a>, <a href="#fd2">2</a>, <a href="#fd3">3</a>, <a href="#fd4">4</a>, <a href="#fd5">5</a>, <a href="#fd6">6</a>, <a href="#fd7">7</a>, and <a href="#fd8">8</a>.</li>
<li>Use <a href="#fd9">Equations 9</a>, <a href="#fd10">10</a>, and <a href="#fd11">11</a> to compare how well the respective <i>in silico</i> model compares to the wet-lab results; the smaller the prediction error, the better the model represents the wet-lab data.</li>
<li>
<a href="#fd12">Equation 12</a> describes how the stirring speed must be adapted when aiming at (up)scaling from one to another bioreactor system. However, please note that this calculation is only valid when using impellers of the same geometry across different systems and if the impeller-to-vessel-diameter ratio is constant as well.</li>
<li>Use <a href="#fd13">Equations 13</a> + <a href="#fd14">14</a> to calculate and adapt the stirring speed to the applied process volume in the DASbox or the DASGIP Bioblock systems.</li>
</ol>
<p>The Specific growth rate <math> <mi>μ</mi> </math> [1/d] can be calculated as</p>
<p id="fd1" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 1)</mtext></mtd> <mtd> <mrow> <mi>μ</mi> <mo>=</mo> <mrow> <mo>(</mo> <mfrac> <mrow> <msub> <mi>X</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <msub> <mi>X</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mrow> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> <mo>−</mo> <msub> <mi>t</mi> <mi>n</mi> </msub> </mrow> </mfrac> <mo>)</mo> </mrow> <mo>×</mo> <mfrac> <mn>1</mn> <msub> <mover> <mi>X</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>where <i>X</i> is the cell concentration [cells/L] at the given time point <i>t</i> and <math> <mrow> <mover> <mi>X</mi> <mo>¯</mo> </mover> </mrow> </math> the mean cell concentration [cells/ L] calculated as</p>
<p id="fd2" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 2)</mtext></mtd> <mtd> <mrow> <msub> <mover> <mi>X</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>=</mo> <mfrac> <mrow> <msub> <mi>X</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <msub> <mi>X</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mrow> <mi>ln</mi> <mrow> <mo>(</mo> <msub> <mi>X</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>)</mo> </mrow> <mo>−</mo> <mi>ln</mi> <mrow> <mo>(</mo> <msub> <mi>X</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> <mo>)</mo> </mrow> </mrow> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>Specific substrate metabolite consumption rates <i>qsMet</i> [pmol/ (cell × d)] for the process days without medium change (d0–d1) can be calculated as</p>
<p id="fd3" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 3)</mtext></mtd> <mtd> <mrow> <mi>qsMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>=</mo> <mo>−</mo> <mrow> <mo>(</mo> <mfrac> <mrow> <mi>sMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <mi>sMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mrow> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> <mo>−</mo> <msub> <mi>t</mi> <mi>n</mi> </msub> </mrow> </mfrac> <mo>)</mo> </mrow> <mo>×</mo> <mfrac> <mn>1</mn> <msub> <mover> <mi>X</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>where s<i>Met</i> is the metabolite concentration [pmol/L], <i>t</i> the process time point [d].</p>
<p>In the perfused cultures (d1–d7), the specific substrate metabolite consumption rate can be calculated as</p>
<p id="fd4" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 4)</mtext></mtd> <mtd> <mrow> <mi>qsMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>=</mo> <mo>−</mo> <mrow> <mo>[</mo> <mrow> <mrow> <mo>(</mo> <mfrac> <mrow> <mi>sMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <mi>sMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mrow> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> <mo>−</mo> <msub> <mi>t</mi> <mi>n</mi> </msub> </mrow> </mfrac> <mo>)</mo> </mrow> <mo>+</mo> <mfrac> <mi>F</mi> <mi>V</mi> </mfrac> <mrow> <mo>(</mo> <mrow> <msub> <mover> <mi>sMet</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <mi>sMe</mi> <msub> <mi>t</mi> <mi>f</mi> </msub> </mrow> <mo>)</mo> </mrow> </mrow> <mo>]</mo> </mrow> <mo>×</mo> <mfrac> <mn>1</mn> <msub> <mover> <mi>X</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>where <i>F</i> is the flow rate of feed and waste stream [L/day], <i>V</i> the culture volume [L], s<i>Met</i><sub><i>f</i></sub> the substrate metabolite concentration in the feed stream [pmol/ L] and <math> <mrow> <mover> <mrow> <mi>s</mi> <mi>M</mi> <mi>e</mi> <mi>t</mi> </mrow> <mo>¯</mo> </mover> </mrow> </math> is the mean substrate metabolite concentration in the culture calculated as</p>
<p id="fd5" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 5)</mtext></mtd> <mtd> <mrow> <msub> <mover> <mi>sMet</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>=</mo> <mfrac> <mrow> <mi>sMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <mi>sMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mrow> <mi>ln</mi> <mrow> <mo>(</mo> <mrow> <mi>sMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> </mrow> <mo>)</mo> </mrow> <mo>−</mo> <mi>ln</mi> <mrow> <mo>(</mo> <mrow> <mi>sMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mo>)</mo> </mrow> </mrow> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>Specific waste metabolite production rates <i>qwMet</i> [pmol/ (cell × d)] for the process days without medium change are calculated accordingly as</p>
<p id="fd6" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 6)</mtext></mtd> <mtd> <mrow> <mi>qwMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>=</mo> <mrow> <mo>(</mo> <mfrac> <mrow> <mi>wMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <mi>wMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mrow> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> <mo>−</mo> <msub> <mi>t</mi> <mi>n</mi> </msub> </mrow> </mfrac> <mo>)</mo> </mrow> <mo>×</mo> <mfrac> <mn>1</mn> <msub> <mover> <mi>X</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>where <i>wMet</i> is the waste metabolite concentration [pmol/ L].</p>
<p>In the perfused cultures, the specific waste metabolite production rate is calculated as</p>
<p id="fd7" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 7)</mtext></mtd> <mtd> <mrow> <mi>qwMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>=</mo> <mrow> <mo>[</mo> <mrow> <mrow> <mo>(</mo> <mfrac> <mrow> <mi>wMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <mi>wMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mrow> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> <mo>−</mo> <msub> <mi>t</mi> <mi>n</mi> </msub> </mrow> </mfrac> <mo>)</mo> </mrow> <mo>+</mo> <mfrac> <mi>F</mi> <mi>V</mi> </mfrac> <msub> <mover> <mi>wMet</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> </mrow> <mo>]</mo> </mrow> <mo>×</mo> <mfrac> <mn>1</mn> <msub> <mover> <mi>X</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>where <math> <mrow> <mover> <mrow> <mi>w</mi> <mi>M</mi> <mi>e</mi> <mi>t</mi> </mrow> <mo>¯</mo> </mover> </mrow> </math> is the mean waste metabolite concentration in the culture calculated as</p>
<p id="fd8" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 8)</mtext></mtd> <mtd> <mrow> <msub> <mover> <mi>wMet</mi> <mo>¯</mo> </mover> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>=</mo> <mfrac> <mrow> <mi>wMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> <mo>−</mo> <mi>wMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mrow> <mi>ln</mi> <mrow> <mo>(</mo> <mrow> <mi>wMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mrow> <mi>n</mi> <mo>+</mo> <mn>1</mn> </mrow> </msub> </msub> </mrow> <mo>)</mo> </mrow> <mo>−</mo> <mi>ln</mi> <mrow> <mo>(</mo> <mrow> <mi>wMe</mi> <msub> <mi>t</mi> <msub> <mi>t</mi> <mi>n</mi> </msub> </msub> </mrow> <mo>)</mo> </mrow> </mrow> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>To calculate the prediction error of a modeling approach, first the relative error for each individual data point pair (<i>e</i><sub><i>ri</i></sub>) needs to be calculated as follows, where <i>x</i><sub><i>wl</i></sub> are wet-lab and <i>x</i><sub><i>m</i></sub> are modeled data points, with <math> <mrow> <msub> <mi>x</mi> <mrow> <mi>w</mi> <mi>l</mi> <mspace></mspace> <mi>m</mi> <mi>a</mi> <mi>x</mi> </mrow> </msub> </mrow> </math> as the maximal wet-lab data point:</p>
<p id="fd9" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 9)</mtext></mtd> <mtd> <mrow> <msub> <mi>e</mi> <msub> <mi>r</mi> <mi>i</mi> </msub> </msub> <mo>=</mo> <mfrac> <mrow> <mo>|</mo> <mrow> <msub> <mi>x</mi> <mi>wl</mi> </msub> <mo>−</mo> <msub> <mi>x</mi> <mi>m</mi> </msub> </mrow> <mo>|</mo> </mrow> <msub> <mi>x</mi> <mi>wl max</mi> </msub> </mfrac> <mo>×</mo> <mn>100</mn> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>The average error <math> <mrow> <msub> <mover> <mi>e</mi> <mo>¯</mo> </mover> <mi>r</mi> </msub> </mrow> </math> provides the value on how well the model represents the wet-lab data. The average error and the standard deviation for the average error <math> <mrow> <mi>S</mi> <msub> <mi>D</mi> <msub> <mover> <mi>e</mi> <mo>¯</mo> </mover> <mi>r</mi> </msub> </msub> </mrow> </math> for each data set can be calculated as follows:</p>
<p id="fd10" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 10)</mtext></mtd> <mtd> <mrow> <msub> <mover> <mi>e</mi> <mo>¯</mo> </mover> <mi>r</mi> </msub> <mo>=</mo> <mfrac> <mrow> <msubsup> <mrow> <mo>∑</mo> </mrow> <mrow> <mi>i</mi> <mo>=</mo> <mn>1</mn> </mrow> <mi>n</mi> </msubsup> <msub> <mi>e</mi> <msub> <mi>r</mi> <mi>i</mi> </msub> </msub> </mrow> <mi>n</mi> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p id="fd11" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 11)</mtext></mtd> <mtd> <mrow> <mi>S</mi> <msub> <mi>D</mi> <msub> <mover> <mi>e</mi> <mo>¯</mo> </mover> <mi>r</mi> </msub> </msub> <mo>=</mo> <msqrt> <mrow> <mfrac> <mn>1</mn> <mrow> <mi>n</mi> <mo>−</mo> <mn>1</mn> </mrow> </mfrac> <mo>×</mo> <munderover> <mrow> <mo>∑</mo> </mrow> <mrow> <mi>i</mi> <mo>=</mo> <mn>1</mn> </mrow> <mi>n</mi> </munderover> <msup> <mrow> <mo>(</mo> <mrow> <msub> <mi>e</mi> <msub> <mi>r</mi> <mi>i</mi> </msub> </msub> <mo>−</mo> <msub> <mover> <mi>e</mi> <mo>¯</mo> </mover> <mi>r</mi> </msub> </mrow> <mo>)</mo> </mrow> <mn>2</mn> </msup> </mrow> </msqrt> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>The impeller rotational speed <i>N</i> [rps] of the new vessel <i>o</i>, can be calculated with the impeller diameter <i>D</i> [m], the impeller rotational speed and the vessel liquid volume <i>V</i> [m<sup>3</sup>] of the new and the previously used vessel <i>i</i>:</p>
<p id="fd12" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 12)</mtext></mtd> <mtd> <mrow> <msub> <mi>N</mi> <mi>o</mi> </msub> <mo>=</mo> <mroot> <mrow> <msubsup> <mi>N</mi> <mi>i</mi> <mn>3</mn> </msubsup> <mo>×</mo> <mfrac> <msub> <mi>V</mi> <mi>o</mi> </msub> <msub> <mi>V</mi> <mi>i</mi> </msub> </mfrac> <mo>×</mo> <mfrac> <msubsup> <mi>D</mi> <mi>i</mi> <mn>5</mn> </msubsup> <msubsup> <mi>D</mi> <mi>o</mi> <mn>5</mn> </msubsup> </mfrac> </mrow> <mn>3</mn> </mroot> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>For calculation of the impeller rotational speed at a given liquid volume in the DASbox the following equation can be used:</p>
<p id="fd13" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 13)</mtext></mtd> <mtd> <mrow> <mi>N</mi> <mo>=</mo> <mspace></mspace> <mn>150.5</mn> <mspace></mspace> <mo>×</mo> <mspace></mspace> <msup> <mi>V</mi> <mfrac> <mn>1</mn> <mn>3</mn> </mfrac> </msup> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>For calculation of the impeller rotational speed at a given liquid volume in the DASGip Bioblock the following equation can be used:</p>
<p id="fd14" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 14)</mtext></mtd> <mtd> <mrow> <mi>N</mi> <mo>=</mo> <mspace></mspace> <mn>76.5</mn> <mspace></mspace> <mo>×</mo> <mspace></mspace> <msup> <mi>V</mi> <mfrac> <mn>1</mn> <mn>3</mn> </mfrac> </msup> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>As noted above, it is expected that the here described expansion strategy should yield similar cell densities for other hPSC lines that are of interest to any other laboratory. However, it cannot be excluded that the aggregation of individual cell lines is either poor (thus limiting/precluding the proposed suspension culture approach) or, in contrast, results in the excessive uncontrolled clumping of hPSCs, limiting proliferation and pluripotency. Moreover, cell line specific metabolic properties might differ from the cell lines tested by us, requiring the adjustment of cell specific metabolic rates. Those can be retrospectively adapted and integrated into our <i>in silico</i> modeling approach (according to <a href="#fd3">Equations 3</a>, <a href="#fd4">4</a>, <a href="#fd5">5</a>, <a href="#fd6">6</a>, <a href="#fd7">7</a>, and <a href="#fd8">8</a>) after conducting preliminary wet lab experiments (to generate the required database) applying the culture strategies described here.</p>
<p>As another limitation, it should be noted that the model described here is only validated for the key variables (i.e., glucose-, glutamine- and lactate- levels, osmolality and aggregate size) investigated by us. Thus, future adaptation e.g., to media properties such as temperature-induced decay of FGF-2 levels may be required, respectively.</p>
<p>Also of note: The upscaling strategy described here is only valid when 1) applying an impeller design representing the same (or highly equivalent) geometry and when 2) the impeller-to-vessel diameter ratio remains essentially constant.</p>
<p>As a final limitation, our <i>in silico</i> process model is only validated for modeling the growth kinetics of hPSCs at the pluripotent state but not yet adapted to modeling the complex transition or pluripotent hPSCs along their differentiation into specific lineages.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The monolayer culture is too dense on the passaging day i.e., exceeding 80% confluence (step 5). In addition, the media is bright yellow and excessive dead cells and debris are microscopically observable in the supernatant.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Improper cell numbers (i.e., too high) were seeded. Ensure proper counting and dilution of your cells before seeding. Eventually, reduce the cell number used for seeding as this adaptation may be required for your line. In addition, keep track of the splitting rhythm: It should always be maintained for three days. Eventually check your cell line for normal karyotype or even test the genomic integrity at higher resolution applying respective technologies (<a href="#bib2">Baker et al., 2016</a>) to exclude undesired cell transformation.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>The monolayer culture is below ∼60% of confluence and only small cell colonies are observed after the suggested three days of passage duration (step 5).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>This suggests seeding of low cell numbers: Ensure proper counting and dilution of your cells before seeding. Eventually increase the cell number used for seeding as this adaptation maybe required for your hPSC line. Remember to add RI when passaging hPSCs. In addition, check the quality of the growth factors i.e., FGF-2, TGF-ß or Insulin (<a href="#bib11">Massai et al., 2017</a>) in the E8 adherent medium and eventually use fresh medium/ growth factors. Eventually try alternative suppliers of critical media components, in particular of the growth factors highlighted above. Keep track of the splitting rhythm to be maintained strictly for three days (see <a href="#sec7.1">problem 1</a>).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The monolayer culture appears differentiated, which should be validated by microscopic assessment (<a href="#bib16">Zweigerdt et al., 2011</a>) or by flow cytometry (<a href="#step-by-step-method-details">step-by-step method details</a>, steps 36–38 <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>) on the day of passaging (i.e., on day 3, step 5).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Critically check the quality of your growth factors stocks (origin/supplier, shelf-life, concentration) used to generate the E8 adherent medium and eventually use fresh medium or ensure growth factor replacement; do not conduct repeated freeze/thaw cycles of your growth factor stocks.</p>
<p>In addition, keep track of the feeding and splitting rhythm, do not prolong these beyond three days, and avoid exceeding 80% of cell confluence (see <a href="#sec7.1">problems 1</a>, <a href="#sec7.3">2</a> and <a href="#fig2">Figure 2</a>D). Regularly check your culture for the level of pluripotency markers expression by flow cytometry (<a href="#step-by-step-method-details">step-by-step method details</a>, <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>, steps 36–38).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>24 h after bioprocess inoculation, cells did not form any or very little aggregates (after step 29).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Ensure proper quality of your 2D culture used for process inoculation by considering <a href="#sec7.1">problems 1</a>, <a href="#sec7.3">2</a>, and <a href="#sec7.5">3</a> above. Moreover, avoid extensive incubation time with Accutase (i.e., do not exceed 3 min; see step 5) when generating single cells for cell passaging in 2D and for generating the cell suspension for bioreactor inoculation.</p>
<p>Reevaluate the culture media to ensure proper supplementation of RI and Pluronic™ F-68 in the E8 suspension medium (for bioreactor inoculation). In addition, verify that the stirring speed was set according to step 26 and <a href="#fd12">Equations 12</a>, <a href="#fd13">13</a>, and <a href="#fd14">14</a>.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Cell aggregates remain smaller than the scope exemplified in <a href="#fig9">Figure 9</a>B throughout the process.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Reduce the stirring speed in adaptation to your hPSC line but keeping in mind that stirring speed reduction may also increase the heterogeneity of the aggregate diameter distribution; do not fall below the minimal stirring speed of 50 rpm at 150 mL process volume with respect to the DASbox system. Note that the minimum speeds for other culture volumes can be calculated according to <a href="#fd12">Equations 12</a>, <a href="#fd13">13</a>, and <a href="#fd14">14</a>.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Cell aggregates become large than the scope exemplified in <a href="#fig9">Figure 9</a>B throughout the process</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Increase the stirring speed according to your specific culture process and hPSC line in hand, but do not exceed a maximum speed of 120 rpm at 150 mL process scale for the DASbox system, as exemplified for this protocol. Based on our experience, higher stirring speed may harm your cells and will reduce cell viability and yield (<a href="#bib10">Manstein et al., 2021</a>). The maximum stirring speeds for other culture volumes (and potentially bioreactor systems) can be calculated according to <a href="#fd12">Equations 12</a> or <a href="#fd13">13</a>. Ensure the supplementation of Pluronic™ F-68 to your E8 suspension medium (for bioreactor inoculation), E8 full feed medium I and II as this is of key importance to protect cells from shear-induced damage.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>The cell density at 24 h after bioreactor inoculation is (substantially) below 0.4 × 10<sup>6</sup> cells/mL (i.e., &gt;20 % below the intended seeding density of 0.5 × 10<sup>6</sup> cells/mL; see process step 24).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Ensure proper attention to the hPSC culture used for inoculation by referring to <a href="#sec7.1">problems 1</a>, <a href="#sec7.3">2</a>, and <a href="#sec7.5">3</a> above. Reevaluate the E8 suspension medium (for bioreactor inoculation) in particular for the proper concentration and quality of RI and Pluronic™ F-68.</p>
<h3 id="sec7.15">Problem 8</h3>
<p>Extensive cell clumping eventually plugging narrow gaps in the bioreactor setup such as gaps between the pH or DO probe and the vessel wall. The formation of extensive cell clumps or even cell layers attaching to the bottom of the vessel or other parts inside the bioreactor (i.e., probes, sampling ports, etc.) is observed. The (localized) accumulation and extensive clumping of cells may entirely deplete the cell suspension and subsequently disrupt the bioprocess (after step 29).</p>
<h3 id="sec7.16">Potential solution</h3>
<p>Reevaluate that proper coating of the vessel was performed according to step 8 of this protocol.</p>
<p>Make sure that during bioreactor assembly proper gaps between the respective probe(s) and the vessel wall remain; ensure to place the bioreactor lid properly, which is a prerequisite to avoid any direct attachment of any installations inside the bioreactor and the vessel wall.</p>
<h3 id="sec7.17">Problem 9</h3>
<p>Application of an alternative bioreactor system and/or impeller type (compared to the model described here, step 8 and following) may result in improper cell aggregation, poor cell yields etc.</p>
<h3 id="sec7.18">Potential solution</h3>
<p>Generally, performing the protocol in other bioreactor systems will require systematic testing and modifications. Using another impeller design, it should be noted as a “rule of thumb” that the application of a lower number of blades (an 8-blade pitched impeller is used here; see <a href="#key-resources-table">key resources table</a>), will typically require an increase in the stirring speed compared to this protocol and vice versa (higher number of blades requesting lower stirring speed).</p>
<h3 id="sec7.19">Problem 10</h3>
<p>The pH/DO sensor providing no/ or (apparently) wrong signals (following step 14).</p>
<h3 id="sec7.20">Potential solution</h3>
<p>Check that the connection of cables and probes is dry and well connected.</p>
<p>Ensure that the time when the pH sensor is not submerged in the liquid is minimized; extended time periods can lead to damage of the pH sensor requiring replacement. In this case, the sensor needs to be replaced, recalibrated and sterilized before use.</p>
<p>The electrolyte of the DO sensor may be depleted after autoclaving. In this case, replacement of the membrane and the electrolyte is required following manufacturer’s instructions; ensure autoclaving of the reactor after fixing any probe-related or other issues.</p>
<h3 id="sec7.21">Problem 11</h3>
<p>Improper media prime feed or base lines due to pump failure (step 29).</p>
<h3 id="sec7.22">Potential solution</h3>
<p>Check that all luer connections are tight and no tubing or luer connector is broken. If no connection issues are observed, it might be helpful to use a pipette boy for applying some external pressure to the vent filter of the respective bottle, thereby pushing the liquid into the reactor; this handling requires temporary removal of the pump head to allow liquid flow; ensure subsequent reintroduction of the pump head, respectively.</p>
<h3 id="sec7.23">Problem 12</h3>
<p>Increasing bioreactor liquid volumes are accumulating during perfusion mode (following step 29).</p>
<h3 id="sec7.24">Potential solution</h3>
<p>Check the calibration of pumps B, C, D by using the DASWare Software, clicking on the respective pump, choosing the tab “Adjust” and checking the value at “F.Cal”. The value should be at ∼100 1/mL for a tube diameter of 0.5 mm and at ∼33 1/mL for a tube diameter of 1.0 mm. Compare the values for all 3 pumps. In case of major differences between the calibrations, adjust the values to their mean.</p>
<p>Check that all luer connections are tight and no tubing or luer connector is broken.</p>
<p>Check whether the retention filter is blocked. This becomes visible when small bubbles rise from the waste port, suggesting that the retention filter needs to be changed. In this case, stop temperature, pH and DO controls, agitation and pumps in the DASWare software, disconnect the overhead drive, exhaust condenser, temperature sensor, gas supply, and disconnect the cables to the pH and DO probe. Take the tubings out of the pump heads and place the whole bioreactor under the flow hood. Take another retention filter, that was autoclaved separately and replace the blocked filter with the new one by opening the bioreactor with the help of forceps. Subsequently, close the bioreactor, place it back into the station, reconnect all respective lines and restart temperature, pH and DO controls, agitation and pumps.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ackermann, M., Kempf, H., Hetzel, M., Hesse, C., Hashtchin, A.R., Brinkert, K., Schott, J.W., Haake, K., Kuhnel, M.P., Glage, S., et al. (2018). Bioreactor-based mass production of human iPSC-derived macrophages enables immunotherapies against bacterial airway infections. Nat. Commun. <i>9</i>, 5088. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref1">View at publisher</a></p>
<p id="bib2">Baker, D., Hirst, A.J., Gokhale, P.J., Juarez, M.A., Williams, S., Wheeler, M., Bean, K., Allison, T.F., Moore, H.D., Andrews, P.W., and Barbaric, I. (2016). Detecting genetic mosaicism in cultures of human pluripotent stem cells. Stem Cell Rep. <i>7</i>, 998-1012. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref2">View at publisher</a></p>
<p id="bib3">Haase, A., Glienke, W., Engels, L., Göhring, G., Esser, R., Arseniev, L., and Martin, M. (2019). GMP-compatible manufacturing of three iPS cell lines from human peripheral blood. Stem Cell Res <i>35</i>, 101394. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref3">View at publisher</a></p>
<p id="bib4">Haase, A., Göhring, G., and Martin, U. (2017). Generation of non-transgenic iPS cells from human cord blood CD34(+) cells under animal component-free conditions. Stem Cell Res <i>21</i>, 71-73. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref4">View at publisher</a></p>
<p id="bib5">Halloin, C., Schwanke, K., Loebel, W., Franke, A., Szepes, M., Biswanath, S., Wunderlich, S., Merkert, S., Weber, N., Osten, F., et al. (2019). Continuous WNT control enables advanced hPSC cardiac processing and prognostic surface marker identification in chemically defined suspension culture. Stem Cell Rep. <i>13</i>, 775. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref5">View at publisher</a></p>
<p id="bib6">Hartung, S., Schwanke, K., Haase, A., David, R., Franz, W.M., Martin, U., and Zweigerdt, R. (2013). Directing cardiomyogenic differentiation of human pluripotent stem cells by plasmid-based transient overexpression of cardiac transcription factors. Stem Cells Dev <i>22</i>, 1112-1125. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref6">View at publisher</a></p>
<p id="bib7">Kobold, S., Guhr, A., Mah, N., Bultjer, N., Seltmann, S., Seiler Wulczyn, A.E.M., Stacey, G., Jie, H., Liu, W., Loeser, P., and Kurtz, A. (2020). A manually curated database on clinical studies involving cell products derived from human pluripotent stem cells. Stem Cell Rep. <i>15</i>, 546-555. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref7">View at publisher</a></p>
<p id="bib8">Konze, S.A., van Diepen, L., Schroeder, A., Olmer, R., Moeller, H., Pich, A., Weißmann, R., Kuss, A.W., Zweigerdt, R., and Buettner, F.F. (2014). Cleavage of E-cadherin and β-catenin by calpain affects Wnt signaling and spheroid formation in suspension cultures of human pluripotent stem cells. Mol. Cell Proteomics <i>13</i>, 990-1007. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref8">View at publisher</a></p>
<p id="bib9">Manstein, F., Halloin, C., and Zweigerdt, R. (2019). Human pluripotent stem cell expansion in stirred tank bioreactors. Methods Mol. Biol. <i>1994</i>, 79-91. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref9">View at publisher</a></p>
<p id="bib10">Manstein, F., Ullmann, K., Kropp, C., Halloin, C., Triebert, W., Franke, A., Farr, C.M., Sahabian, A., Haase, A., Breitkreuz, Y., et al. (2021). High density bioprocessing of human pluripotent stem cells by metabolic control and in silico modeling. Stem Cells Transl. Med. <a class="external-link" href="https://doi.org/10.1002/sctm.20-0453">https://doi.org/10.1002/sctm.20-0453</a></p>
<p id="bib11">Massai, D., Bolesani, E., Diaz, D.R., Kropp, C., Kempf, H., Halloin, C., Martin, U., Braniste, T., Isu, G., Harms, V., et al. (2017). Sensitivity of human pluripotent stem cells to insulin precipitation induced by peristaltic pump-based medium circulation: considerations on process development. Sci. Rep. <i>7</i>, 3950. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref11">View at publisher</a></p>
<p id="bib12">Monod, J. (1949). The growth of bacterial cultures. Annu. Rev. Microbiol. <i>3</i>, 371-394. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref12">View at publisher</a></p>
<p id="bib13">Rubio, N.R., Xiang, N., and Kaplan, D.L. (2020). Plant-based and cell-based approaches to meat production. Nat. Commun. <i>11</i>, 6276. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref13">View at publisher</a></p>
<p id="bib14">Sahabian, A., Dahlmann, J., Martin, U., and Olmer, R. (2021). Production and cryopreservation of definitive endoderm from human pluripotent stem cells under defined and scalable culture conditions. Nat. Protoc. <i>16</i>, 1581-1599. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref14">View at publisher</a></p>
<p id="bib15">Stevens, K.R. and Murry, C.E. (2018). Human pluripotent stem cell-derived engineered tissues: clinical considerations. Cell Stem Cell <i>22</i>, 294-297. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref15">View at publisher</a></p>
<p id="bib16">Zweigerdt, R., Olmer, R., Singh, H., Haverich, A., and Martin, U. (2011). Scalable expansion of human pluripotent stem cells in suspension culture. Nat. Protoc. <i>6</i>, 689-700. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00694-8/sref16">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Robert Zweigerdt (<a href="mailto:zweigerdt.robert@mh-hannover.de">zweigerdt.robert@mh-hannover.de</a>), or technical contact, Felix Manstein (<a href="mailto:Manstein.felix@mh-hannover.de">Manstein.felix@mh-hannover.de</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The published article by Manstein et al. includes all datasets generated or analyzed during this study. All codes necessary to reproduce the data are provided in this protocol.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Mmc1.pdf">Data S1. Berkeley Madonna model code for high density expansion of hPSCs, related to step 1 of the before you begin section</a></p>
<p><a id="mmc2" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1227-Mmc2.pdf">Data S2. DASware script for high density expansion of hPSCs, related to step 14 e</a></p>
<h3>Acknowledgments</h3>
<p>The work received funding from the German Research Foundation (<a href="https://doi.org/10.13039/501100001659">DFG</a>): Cluster of Excellence REBIRTH EXC 62/3, ZW64/4-1, ZW 64/4-2, KFO311/ZW64/7-1; the German Ministry for Education and Science (<a href="https://doi.org/10.13039/501100002347">BMBF</a>): 01EK1602A, 13N14086, 01EK1601A, 655 13XP5092B, 031L0249; and “Förderung aus Mitteln des Niedersächsischen Vorab” (grant: ZN3340).</p>
<h3>Author contributions</h3>
<p>F.M., K.U., W.T., and R.Z. developed the protocol; F.M., K.U., and W.T. performed the experiments and analyzed the data; F.M. and R.Z. wrote the manuscript; all authors approved the final paper.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>