<article>
<section>
<h1 id="header">Extracellular vesicle and particle isolation from human and murine cell lines, tissues, and bodily fluids</h1>
<p><time datetime="2020-12-25">Published: December 25, 2020</time></p>
<p>Linda Bojmar,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">15</a>,<a href="#fn2">16</a></sup> Han Sang Kim,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#fn1">15</a></sup> Gabriel C. Tobias,<sup><a href="#aff1">1</a>,<a href="#aff4">4</a></sup> Fanny A. Pelissier Vatter,<sup><a href="#aff1">1</a></sup> Serena Lucotti,<sup><a href="#aff1">1</a></sup> Kofi Ennu Gyan,<sup><a href="#aff1">1</a>,<a href="#aff5">5</a></sup> Candia M. Kenific,<sup><a href="#aff1">1</a></sup> Zurong Wan,<sup><a href="#aff1">1</a>,<a href="#aff6">6</a></sup> Kyung-A Kim,<sup><a href="#aff3">3</a></sup> DooA Kim,<sup><a href="#aff3">3</a></sup> Jonathan Hernandez,<sup><a href="#aff7">7</a>,<a href="#aff8">8</a></sup> Virginia Pascual,<sup><a href="#aff6">6</a>,<a href="#aff9">9</a></sup> Todd E. Heaton,<sup><a href="#aff10">10</a>,<a href="#aff11">11</a></sup> Michael P. La Quaglia,<sup><a href="#aff10">10</a>,<a href="#aff11">11</a></sup> David Kelsen,<sup><a href="#aff12">12</a></sup> Tanya M. Trippett,<sup><a href="#aff10">10</a></sup> David R. Jones,<sup><a href="#aff13">13</a></sup> William R. Jarnagin,<sup><a href="#aff7">7</a></sup> Irina R. Matei,<sup><a href="#aff1">1</a></sup> Haiying Zhang,<sup><a href="#aff1">1</a>,<a href="#cor1">*</a></sup> Ayuko Hoshino,<sup><a href="#aff1">1</a>,<a href="#aff14">14</a>,<a href="#fn1">15</a>,<a href="#cor2">**</a></sup> and David Lyden<sup><a href="#aff1">1</a>,<a href="#fn3">17</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Children’s Cancer and Blood Foundation Laboratories, Departments of Pediatrics, and Cell and Developmental Biology, Drukier Institute for Children’s Health, Meyer Cancer Center, Weill Cornell Medicine, New York, NY 10021, USA</p>
<p id="aff2"><sup>2</sup>Department of Biomedical and Clinical Sciences, Linköping University, Linköping 58183, Sweden</p>
<p id="aff3"><sup>3</sup>Yonsei Cancer Center, Division of Medical Oncology, Department of Internal Medicine, Graduate School of Medical Science, Brain Korea 21 Project, Severance Biomedical Science Institute, Yonsei University College of Medicine, Seoul, Korea</p>
<p id="aff4"><sup>4</sup>School of Physical Education and Sport, University of Sao Paulo, Sao Paulo, Brazil</p>
<p id="aff5"><sup>5</sup>Tri-Institutional PhD Program in Computational Biology and Medicine, New York, NY, USA</p>
<p id="aff6"><sup>6</sup>Department of Pediatrics, Weill Cornell Medicine, New York, NY, USA</p>
<p id="aff7"><sup>7</sup>Hepatopancreatobiliary Service, Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p>
<p id="aff8"><sup>8</sup>Surgical Oncology Program, National Cancer Institute, National Institutes of Health, Bethesda, MD, USA</p>
<p id="aff9"><sup>9</sup>Drukier Institute for Children's Health, Weill Cornell Medicine, New York, NY, USA</p>
<p id="aff10"><sup>10</sup>Department of Pediatrics, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p>
<p id="aff11"><sup>11</sup>Pediatric Surgical Service, Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p>
<p id="aff12"><sup>12</sup>Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p>
<p id="aff13"><sup>13</sup>Thoracic Surgery Service, Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p>
<p id="aff14"><sup>14</sup>School of Life Science and Technology, Tokyo Institute of Technology, Yokohama, Japan</p>
<p id="fn1"><sup>15</sup>These authors contributed equally</p>
<p id="fn2"><sup>16</sup>Technical Contact</p>
<p id="fn3"><sup>17</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:haz2005@med.cornell.edu">haz2005@med.cornell.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:ayukohoshino@bio.titech.ac.jp">ayukohoshino@bio.titech.ac.jp</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:dcl2001@med.cornell.edu">dcl2001@med.cornell.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100225">10.1016/j.xpro.2020.100225</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We developed a modified protocol, based on differential ultracentrifugation (dUC), to isolate extracellular vesicles and particles (specifically exomeres) (EVPs) from various human and murine sources, including cell lines, surgically resected tumors and adjacent tissues, and bodily fluids, such as blood, lymphatic fluid, and bile. The diversity of these samples requires robust and highly reproducible protocols and refined isolation technology, such as asymmetric-flow field-flow fractionation (AF4). Our isolation protocol allows for preparation of EVPs for various downstream applications, including proteomic profiling.</p>
<p>For complete details on the use and execution of this protocol, please refer to Hoshino et al. (2020).</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>EVP isolation from human cell lines, tissues, and bodily fluids</li>
<li>First protocol describing EVP isolation from human tumor tissues and matched adjacent tissues</li>
<li>Improved EVP isolation and total EVP protein yield by dissecting tissues into pieces</li>
<li>AF4 fractionation to further analyze EVP samples</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/336-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the isolation of small and heterogeneous EVPs from tissue culture supernatants (conditioned media) derived from cell lines or tissue explants as well as bodily fluids, such as blood, lymphatic fluid, and bile. Below are listed the different sample types, their generation, and pre-processing before EVP isolation.</p>
<h3 id="sec1.1">Cell culture conditions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<p>All cell lines are mycoplasma-tested prior to EVP isolation.</p>
<ol>
<li>Prepare appropriate cell culture media, i.e., RPMI1640 or DMEM supplemented with penicillin (100 U/mL), streptomycin (100 mg/mL) and 10% EVP-depleted fetal bovine serum (FBS).
<ol type="a">
<li>We routinely prepare our own EVP-depleted fetal bovine serum (FBS). FBS is depleted of EVPs by ultracentrifugation at 100,000 × <i>g</i> for 70 min at 10°C and sterile-filtered through a 0.2 μm low protein-binding membrane. The EVP-depleted FBS can be stored in aliquots at −20°C (short term) and −80°C (long term).</li>
</ol>
</li>
<li>Culture cells at 37°C, 5% CO<sub>2</sub> in EVP-depleted media for 3–4 days until 80% confluence.</li>
<li>Collect conditioned culture media for EVP isolation. See <a href="#tbl1">Table 1</a> for typical cell culture conditions and EVP yields.
<table id="tbl1">
<caption>Table 1. Examples of typical cell culture conditions and EVP yields</caption>
<thead>
<tr>
<th>Cell line</th>
<th>Type of culture dish, growth area (cm<sup>2</sup>)</th>
<th>Cultured media volume (mL)</th>
<th>Cell number at seeding</th>
<th>Total EVP protein yield</th>
</tr>
</thead>
<tbody>
<tr>
<td>MDA-MB-231</td>
<td>T175 flask, 175 cm<sup>2</sup>
</td>
<td>25–30 mL</td>
<td>2.0 × 10<sup>6</sup> cells</td>
<td>14.0 μg</td>
</tr>
<tr>
<td>MDA231-LM2-4175</td>
<td>T175 flask, 175 cm<sup>2</sup>
</td>
<td>25–30 mL</td>
<td>1.8 × 10<sup>6</sup> cells</td>
<td>17.0 μg</td>
</tr>
<tr>
<td>4T1</td>
<td>150 mm plate, 148 cm<sup>2</sup>
</td>
<td>25–30 mL</td>
<td>2.0 × 10<sup>6</sup> cells</td>
<td>50.0 μg</td>
</tr>
<tr>
<td>PANC-1</td>
<td>150 mm plate, 148 cm<sup>2</sup>
</td>
<td>25–30 mL</td>
<td>8.3 × 10<sup>6</sup> cells</td>
<td>20.4 μg</td>
</tr>
<tr>
<td>AsPC-1</td>
<td>150 mm plate, 148 cm<sup>2</sup>
</td>
<td>25–30 mL</td>
<td>9.0 × 10<sup>6</sup> cells</td>
<td>26.0 μg</td>
</tr>
</tbody>
</table>
<p class="table-legend">Abbreviation: EVP, extracellular vesicle and particle</p>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> EVP-depleted FBS is also commercially available (Gibco #A2720801). We recommend using the EVP-depleted FBS prepared using exactly the same method or from the same commercial provider for any comparison study and to test each lot for cell growth and EVP production in a cell culture system.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid fully confluent cell cultures and maintain consistent culture conditions. For each cell line seeding number/density and culture time (48 to 72 h) need to be empirically determined depending on growth rate. If using trypsin to harvest adherent cells, remember to wash it off before seeding. Assess cell viability using a suitable method, i.e., manual cell counting and trypan blue exclusion, automatic cell counting with dead cell discrimination or flow cytometry with annexin V or propidium iodide, at the time of media collection.</div>
<h3 id="sec1.2">Tissue explant culture reagents and tools</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="4">
<li>Prepare explant culture media: RPMI1640 supplemented with penicillin (100 U/mL) and streptomycin (100 mg/mL).
<ol type="a">
<li>Media needs to be warmed to ~15°C–20°C prior to use.</li>
</ol>
</li>
<li>Sterilize surgical instruments (forceps, scalpel holder and scissors) by autoclaving (120°C, 40 min, dry cycle), otherwise use disposable instruments.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For short-term (&lt;16–18 h) tissue explant cultures use serum-free media to avoid introducing bovine proteins and EVPs into the preparations. EVP-depleted media can also be used depending on downstream applications and experimental aim, especially for long-term cell cultures, such as explant outgrowth or organoid culture.</div>
<h3 id="sec1.3">Harvest of murine organs for EVP yield optimization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 20–30 min per mouse</div>
<ol start="6">
<li>Mouse organ harvesting.
<ol type="a">
<li>Mice should be anesthetized with isoflurane (3.0% isoflurane at 2.0 L/min oxygen) delivered through a nose cone and placed in a supine position before perfusing.</li>
<li>Perfuse mouse organs and flush blood out by injecting a bolus of 30 mL PBS/5 mM EDTA into the left ventricle using a 26 G × 1/2 needle and a 30 mL syringe.</li>
<li>Remove the organs of interest under aseptic conditions and immerse them in ice-cold sterile PBS; place on ice until further dissection. Dissect organs in the biosafety cabinet.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Organ perfusion and harvesting can be performed after CO<sub>2</sub> euthanasia, or other IACUC approved methods, such as terminal anesthesia (pentobarbital), if an efficient workflow can be maintained.</div>
<div class="critical">
<span class="critical-title">Critical:</span> A minimum of 5–10 mice is required to isolate EVPs from normal organs. Tumor-bearing organs have higher EVP yields and 2–3 animals could be enough, depending on tumor type and size. Matched organs should be pooled to obtain sufficient amounts of EVPs for further characterization, such as nanoparticle tracking analysis (NTA), electron microscopy (EM), and protein quantification (Bicinchoninic Acid Assay - BCA). Timely processing of tissues within 30 min of harvest is required to ensure maximal EVP yield.</div>
<h3 id="sec1.4">Collection of blood and other bodily fluids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min (blood samples), sample-dependent for other bodily fluids</div>
<p>Blood collection from patients and healthy subjects should typically be performed in the morning, or after fasting. Phlebotomy training is required to perform the peripheral blood draw.</p>
<p>Bile collection can be performed by needle cannulation of the common bile duct at the time of surgery. Bile is snap-frozen and stored at −80°C until analysis.</p>
<p>Lymphatic fluid can be collected after radical lymphadenectomy from routinely used collection drainage. To ensure that the sample of lymph fluid does not contain any surgical debris, only the fluid released between 24 and 48 h should be collected (the fluid collected in the first 24 h is discarded).</p>
<p>Downstream analysis of bodily fluid-derived EVPs follows the same isolation steps (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/336-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Workflow for EVP isolation from cell culture, tissue explants, and bodily fluids</div>
</figcaption>
</figure>
<ol start="7">
<li>Collect the venous blood sample in EDTA (purple top) blood collection tube(s).
<ol type="a">
<li>Invert the tube slowly, several times, immediately upon collection, to mix the blood with the anticoagulant and prevent clot formation.
<div class="note">
<span class="note-title">Note:</span> One collection tube of 10 mL of whole blood typically yields approximately 5 mL of plasma, which is usually sufficient for downstream applications, such as BCA, NTA, EM, proteomic mass spectrometry (see further details in “<a href="#expected-outcomes">Expected outcomes</a>” section). We recommend a minimum starting volume of no less than 1 mL of bodily fluid to ensure sufficient EVP yield.</div>
</li>
<li>Place the tube of blood on ice to keep it cold during eventual transport.
<div class="critical">
<span class="critical-title">Critical:</span> Occasionally invert the blood collection tube to avoid red blood cell lysis or clotting during transport.</div>
<div class="note">
<span class="note-title">Note:</span> Blood collection from mice can be performed from the retro-orbital capillary sinus by using a heparinized capillary tube. Alternatively, blood can be collected via cardiac puncture in syringes containing anti-coagulants at the time of sacrifice. Approximately 50% of the blood volume will be represented by plasma and murine blood should be processed in the same manner as described for human samples. Cardiac puncture allows collection of 0.5–1.2 mL of blood per animal. Retro-orbital collection allows for around 0.5 mL. We recommend to use a minimum 1 mL of plasma which yields approximately 5 μg of EVPs. It is thus necessary to combine the plasma of multiple mice to obtain enough plasma EVPs for analysis.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.5">Preparation of the AF4 instrument</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>EVPs can be further separated into exomeres and distinct extracellular vesicle subpopulations by utilizing asymmetric-flow field-flow fractionation (AF4) technology (<a href="#bib14">Zhang et al., 2018</a>). A detailed protocol encompassing the working principles, AF4 method development and troubleshooting, comparison with other technologies, limitations, and applications has been recently reported (<a href="#bib15">Zhang and Lyden, 2019</a>). Here, we will focus our method description on preparation of the instruments prior to fractionation and sample processing and recovery. In brief, we use the Eclipse AF4 (Wyatt Technology) system to fractionate samples in a short channel (144 mm length, Wyatt Technology). The channel is equipped with a 10 kDa MWCO Regenerated Cellulose membrane (Millipore) on the accumulation bottom wall and a 490 mm spacer. An Agilent Iso pump, autosampler, and fraction collector are included for running buffer, loading samples, and collecting fractions, respectively. Online real-time monitors include the Agilent Multiple Wavelength Detector (280 nm) and Wyatt Technology DAWN HELEOS II (multi-angle light scattering-MALS detector, at 664 nm) with QELS (dynamic light scattering-DLS detector) installed at detector 12 (100°).</p>
<div class="optional">
<span class="optional-title">Optional:</span> Assemble a new membrane for the short channel.</div>
<ol start="8">
<li>Equilibrate the system by running PBS at 1 mL/min for a minimum of 30 min.</li>
<li>Switch on lasers for all monitors ~15 min prior to sample loading.</li>
<li>Switch on thermostats for both the autosampler and fraction collector and set temperature at 4°C, keeping samples and fractions at 4°C throughout the whole fractionation process.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Coat the membrane with BSA by running 30–40 μg BSA 2–3 times using the following method (this step is only needed when a new membrane is installed).</div>
<ol start="10">
<li style="list-style-type: none;">Forward channel flow: 1 mL/min
<ul>
<li>Elution, 2 min, Vx (cross flow) 1.5 mL/min;</li>
<li>Focus, 1 min, Vx 1.5 mL/min;</li>
<li>Focus + Injection, 2 min, Vx 1.5 mL/min, injection flow 0.2 mL/min;</li>
<li>Elution, 15 min, Vx 3 mL/min;</li>
<li>Elution, 5 min, Vx 0 mL/min;</li>
<li>Elution + Injection, 5 min, Vx 0 mL/min;</li>
</ul>
</li>
<li>Pre-chill 96-well fraction collection plates by storing them in a cold room before use. Before starting the fractionation process, install two plates in the designated positions (pre-set in the Chemstation operating program).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For long-term storage (&gt;1-2 weeks), the system should be maintained in 20% ethanol. Flushing the system first with a large volume of water is essential before switching to PBS buffer to avoid precipitation. The PBS buffer should be filtered through 0.1 or 0.2 μm filter units. An inline filter can be used as well. For short-term storage (&lt;1-2 weeks), the system can be stored in water (preferred) or PBS and run in a Night Rinse mode (0.2 mL/min).</div>
<div class="note">
<span class="note-title">Note:</span> The short channel membrane can perform well without a notable difference in particle separation resolution for multiple runs. A new membrane should be installed when obvious shifting of the elution peaks and/or noisy baseline occurs.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human cancer and adjacent normal tissues, blood, bile, and lymphatic fluid</td>
<td>Memorial Sloan Kettering Cancer Center and Yonsei Cancer Center</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Sterile PBS</td>
<td>VWR</td>
<td>Cat#45000-446</td>
</tr>
<tr>
<td>BSA</td>
<td>Sigma</td>
<td>Cat#A1900</td>
</tr>
<tr>
<td>Premium-grade FBS</td>
<td>VWR</td>
<td>Cat#97068-085</td>
</tr>
<tr>
<td>Gibco fetal bovine serum, exosome-depleted</td>
<td>Fisher Scientific</td>
<td>Cat#A2720801</td>
</tr>
<tr>
<td>RPMI</td>
<td>Corning</td>
<td>Cat#10-040-CV</td>
</tr>
<tr>
<td>DMEM</td>
<td>Corning</td>
<td>Cat#10-013-CV</td>
</tr>
<tr>
<td>L-Glutamine, 100×</td>
<td>Corning</td>
<td>Cat#25-005-CI</td>
</tr>
<tr>
<td>Penicillin–streptomycin solution, 50×</td>
<td>Corning</td>
<td>Cat#30-001-CI</td>
</tr>
<tr>
<td>TrypLE</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12604-039P</td>
</tr>
<tr>
<td>UltraPure 0.5 M EDTA, pH 8.0</td>
<td>Corning</td>
<td>Cat#15575020</td>
</tr>
<tr>
<td>Isoflurane solution (formerly IsoThesia)</td>
<td>Covetrus</td>
<td>Cat#11695067772</td>
</tr>
<tr>
<td>Deuterium oxide</td>
<td>Sigma</td>
<td>Cat# 1133660500</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Pierce BCA Protein Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#23225</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Human: MDA-MB-231</td>
<td>ATCC</td>
<td>HTB-26</td>
</tr>
<tr>
<td>Human: MDA-MB-4175</td>
<td>Dr. J. Massague (<a href="#bib16">Minn et al., 2005</a>)</td>
<td>n/a</td>
</tr>
<tr>
<td>Human: PANC-1</td>
<td>ATCC</td>
<td>CRL-1469</td>
</tr>
<tr>
<td>Human: AsPC-1</td>
<td>ATCC</td>
<td>CRL-1682</td>
</tr>
<tr>
<td>Mouse: B16-F10</td>
<td>ATCC</td>
<td>CRL-6475</td>
</tr>
<tr>
<td>Mouse: 4T1</td>
<td>ATCC</td>
<td>CRL-2539</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: wild-type BALB/c</td>
<td>Jackson Laboratory</td>
<td>Cat#000651</td>
</tr>
<tr>
<td>Mouse: wild-type C57BL/6</td>
<td>Jackson Laboratory</td>
<td>Cat#000664</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Astra 6 with an Eclipse module integrated to operate flow</td>
<td>Wyatt Technology</td>
<td>n/a</td>
</tr>
<tr>
<td>ChemStation</td>
<td>Agilent Technologies</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Ultracentrifuge</td>
<td>Beckman Coulter <a href="https://www.beckmancoulter.com/">https://www.beckmancoulter.com/</a>
</td>
<td>Optima XE/XPE, Cat#A94469</td>
</tr>
<tr>
<td>Fixed-angle rotor 50.4Ti</td>
<td>Beckman Coulter</td>
<td>Cat# 347299</td>
</tr>
<tr>
<td>Fixed-angle rotor 70Ti</td>
<td>Beckman Coulter</td>
<td>Cat# 337922</td>
</tr>
<tr>
<td>Fixed-angle rotor 45Ti</td>
<td>Beckman Coulter</td>
<td>Cat# 339160</td>
</tr>
<tr>
<td>Ultracentrifugation tube (45Ti rotor)</td>
<td>Beckman Coulter</td>
<td>Cat#355628</td>
</tr>
<tr>
<td>Ultracentrifugation tube (70Ti rotor)</td>
<td>Beckman Coulter</td>
<td>Cat#355631</td>
</tr>
<tr>
<td>Ultracentrifugation tube (50.4Ti rotor)</td>
<td>Beckman Coulter</td>
<td>Cat#355645</td>
</tr>
<tr>
<td>500 mL Supor MachV PES filter units</td>
<td>VWR</td>
<td>Cat#73520-984</td>
</tr>
<tr>
<td>Integra disposable scalpel blades no. 10</td>
<td>Miltex</td>
<td>REF 4-410</td>
</tr>
<tr>
<td>Mouse Surgical Kit</td>
<td>Kent Scientific</td>
<td>INMOUSEKIT</td>
</tr>
<tr>
<td>26 G × 1/2 in PrecisionGlide needle</td>
<td>Becton Dickinson</td>
<td>Cat#305111</td>
</tr>
<tr>
<td>Cell strainer, 40 μm</td>
<td>Corning</td>
<td>Cat#CLS431750</td>
</tr>
<tr>
<td>Millex-GV Filter, 0.22 μm</td>
<td>Millipore</td>
<td>Cat# SLGVV255F</td>
</tr>
<tr>
<td>96-well plate</td>
<td>VWR</td>
<td>Cat#62406-081</td>
</tr>
<tr>
<td>Blue screw caps</td>
<td>Agilent</td>
<td>Cat#5182-0717</td>
</tr>
<tr>
<td>Screw cap vials</td>
<td>Agilent</td>
<td>Cat#5182-0714</td>
</tr>
<tr>
<td>Vial insert, 250 μL pulled point glass</td>
<td>Agilent</td>
<td>Cat#5183-2085</td>
</tr>
<tr>
<td>96-well plate, 1.0 mL, polypropylene</td>
<td>Agilent</td>
<td>Cat#8010-0534</td>
</tr>
<tr>
<td>Sealing tape, clear polyolefin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#232701</td>
</tr>
<tr>
<td>Amicon Ultra-15 Centrifugal Filter Unit with Ultracel-30 membrane</td>
<td>Millipore</td>
<td>Cat#UFC903024</td>
</tr>
<tr>
<td>Amicon Ultra-4 Centrifugal Filter Unit with Ultracel-30 membrane</td>
<td>Millipore</td>
<td>Cat#UFC803024</td>
</tr>
<tr>
<td>Millipore Reg. Cellulose membrane 10KD SC</td>
<td>Wyatt technology</td>
<td>Cat#4057</td>
</tr>
<tr>
<td>Inline filter membrane (0.1 μm)</td>
<td>Wyatt Technology</td>
<td>Cat#1871</td>
</tr>
<tr>
<td>1260 Infinity analytical- and preparative-scale fraction collectors</td>
<td>Agilent</td>
<td>Cat# G1364C</td>
</tr>
<tr>
<td>1290 thermostat</td>
<td>Agilent</td>
<td>Cat#G1330B</td>
</tr>
<tr>
<td>1260 Infinity standard autosampler</td>
<td>Agilent</td>
<td>Cat#G1329B</td>
</tr>
<tr>
<td>1260 Infinity multiple wavelength detector</td>
<td>Agilent</td>
<td>Cat#G1365D</td>
</tr>
<tr>
<td>1260 Infinity isocratic pump</td>
<td>Agilent</td>
<td>Cat#G1310B</td>
</tr>
<tr>
<td>TG-14 HPLC vacuum degasser</td>
<td>GASTORR</td>
<td>Cat# TG-14</td>
</tr>
<tr>
<td>DAWN HELEOS II with QELS installed at detector 12 (100°)</td>
<td>Wyatt Technology</td>
<td>n/a</td>
</tr>
<tr>
<td>Eclipse AF4</td>
<td>Wyatt Technology</td>
<td>n/a</td>
</tr>
<tr>
<td>Short channel</td>
<td>Wyatt Technology</td>
<td>n/a</td>
</tr>
<tr>
<td>Tabletop Heraeus Multifuge x3R centrifuge</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#75004501</td>
</tr>
<tr>
<td>Microcentrifuge</td>
<td>Eppendorf</td>
<td>Cat#5424R</td>
</tr>
<tr>
<td>AccuScan GO UV/vis microplate spectrophotometer</td>
<td>Fisher Scientific</td>
<td>Cat#14-377-579</td>
</tr>
<tr>
<td>BD microtainer, capillary blood collection tube hematology K2 EDTA additive</td>
<td>Becton Dickinson</td>
<td>Cat#365974</td>
</tr>
<tr>
<td>Micro-hematocrit capillary tubes</td>
<td>Kimble Chase</td>
<td>Cat#41B2501</td>
</tr>
<tr>
<td>BD Vacutainer Red Top blood collection tube</td>
<td>Becton Dickinson</td>
<td>Cat#366434G</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Sucrose/deuterium oxide mixture for density purification</h3>
<p>A sucrose/deuterium oxide mixture can be used to further purify EVP samples by a density cushion.</p>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>30%</td>
<td>15 g</td>
</tr>
<tr>
<td>Tris (Trizma base, Sigma)</td>
<td>20 mM</td>
<td>1.2 g</td>
</tr>
<tr>
<td>D<sub>2</sub>O</td>
<td>n/a</td>
<td>Fill to 48 mL</td>
</tr>
<tr>
<td>HCl</td>
<td>n/a</td>
<td>1–2 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p>To prepare the sucrose/deuterium oxide mixture (density 1.210 g/cm<sup>3</sup>), weigh sucrose and Tris and add D<sub>2</sub>O. Shake until completely dissolved and measure the pH of the mixture. Adjust the pH to 7.4 with HCl and correct the volume to 50 mL. Filter the mixture through a 0.22 μm filter into a sterile tube. The prepared sucrose mixture will be stable for one month at 4°C. After this period, measure the pH and adjust to 7.4.</p>
<div class="critical">
<span class="critical-title">Critical:</span> HCl is highly corrosive and irritant and can cause serious eye and skin damage and respiratory irritation. Handle HCl in a chemical hood and use protective clothing, goggles, and gloves.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend keeping ultracentrifuge rotors in 4°C cold rooms and setting the temperature of the ultracentrifuge to 10°C to extend the life expectancy of the machine.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Alternatives to ultracentrifugation with sucrose/deuterium oxide cushion for further clean-up and purification of EVP samples are ultrafiltration, size-exclusive chromatography, two-phase isolation, magnetic capture, immunoprecipitation, density gradient (such as Optiprep) and microfluidic devices. The outcome of these alternatives has to be determined by the end user. As an alternative for fixed-angle rotors mentioned in this protocol, swing buckets could also be used but specific handling, such as supernatant removal and pellet re-suspension will have to be optimized by the user.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Isolation of EVPs from cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5–4 h (time increases with number of samples)</div>
<p>Perform serial centrifugation and ultracentrifugation steps to isolate EVPs from conditioned cell culture media.</p>
<ol>
<li>Harvest culture media and remove cells, debris, apoptotic bodies, and contaminants through serial centrifugation.
<ol type="a">
<li>Centrifuge sample at 500 × <i>g</i> for 10 min at 4°C to pellet cells and debris.</li>
<li>Collect the supernatant without disturbing the cell/debris pellet. This can be done by quickly pouring off the supernatant into a new container/tube or carefully pipetting off the supernatant. It is essential to work as quickly as possible during these steps, as the pellet will become dislodged over time.
<div class="note">
<span class="note-title">Note:</span> For this and all subsequent steps, supernatants and re-suspended EVP pellets should be kept on ice between isolation steps.</div>
<div class="note">
<span class="note-title">Note:</span> A step of 3,000 × <i>g</i> centrifugation for 20 min at 4°C should be added for cell culture supernatants before short-term storage (&lt;24 h) at 4°C or before freezing to remove additional cell debris contaminants. This step has to be performed if further analysis of the post-12,000 × <i>g</i> microvesicle sample is planned.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Conditioned media supernatants can be stored at −80°C for long-term storage or at 4 °C for short-term (&lt;24 h) storage before ultracentrifugation. Minor differences have been observed between EVPs isolated from fresh versus frozen samples in terms of yield and purity. However, frozen samples are often used to allow for retrospective studies, collaborative studies between different institutions and batch processing of test samples/subjects and controls.</div>
</li>
<li>Centrifuge the supernatant at 12,000 × <i>g</i> for 20 min at 4°C to remove apoptotic bodies and aggregates and larger vesicles that could otherwise be co-isolated with EVPs and “contaminate” the EVP preparation.
<div class="note">
<span class="note-title">Note:</span> This step can be performed in the ultracentrifuge or tabletop centrifuge depending on volume and availability of the ultracentrifuge. We recommend performing the 12,000 × <i>g</i> spin right before ultracentrifugation since contaminating material could develop by freeze-thaw cycles or extended storage times.</div>
</li>
<li>Collect supernatant without disturbing the pellet and transfer to new/clean ultracentrifuge tube by pipetting, leaving a small volume behind to avoid contamination from pellet.
<div class="critical">
<span class="critical-title">Critical:</span> Wash newly purchased ultracentrifugation tubes with soap and water and rinse them extensively in ddH<sub>2</sub>O before the first EVP isolation, since the factory coating on the inside of the tube will prevent the EVP pellet from adhering to the tube. After use, the tubes should be washed extensively and sterilized by autoclaving if required. Cleaning of ultracentrifugation tubes was performed as per standard cleaning procedure for laboratory glassware and utensils used for molecular and cellular biology applications. We extensively wash the tubes in soap and hot water while scrubbing with a brush. The tubes are then thoroughly rinsed with tap water to remove any traces of soap, followed by additional rinses in ddH<sub>2</sub>O. If EVPs are isolated for <i>in vitro</i> or <i>in vivo</i> work rather than molecular biology, the tubes can be autoclaved, this however shortens the life of the tubes. No contamination between samples has ever been observed using these procedures. The tube used depends on the volume of starting material rather than the type of specimen.</div>
<div class="note">
<span class="note-title">Note:</span> It is important to avoid disturbing the pellet with the pipette tip in these centrifugation steps since the aim is to remove pelleted contamination; instead, leave a minimal amount of media above the pellet to avoid dissociating the pellet. Remember to work as quickly as possible during these steps as the pellet will become dislodged over time.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Isolate EVPs by ultracentrifugation
<ol type="a">
<li>Centrifuge supernatant at 100,000 × <i>g</i> for 70 min at 10°C in a fixed-angle rotor in a Beckman Coulter ultracentrifuge (see <a href="#materials-and-equipment">Materials and equipment</a>) to collect EVPs.
<div class="critical">
<span class="critical-title">Critical:</span> It is very important to accurately balance the tubes in the ultracentrifuge. An unbalanced rotor can cause spillage, contamination, run failure, or even damage the rotor/ultracentrifuge.</div>
<div class="note">
<span class="note-title">Note:</span> Before inserting tubes into the rotor, we recommend marking an area on the bottom of the tube where the pellet is expected to form, since it may not be easily visible. Avoid disturbing the pelleted area when discarding the supernatant and pay careful attention to it for successful retrieval of your EVP pellet (<a href="#fig2">Figure 2</a>).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/336-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Careful handling of the EVP pellet</div>
<p>(A) After ultracentrifugation, carefully decant supernatant and (B and C) re-suspend the pellet in a small volume of PBS (0.5–1 mL depending on tube size) by pipetting and flushing the marked area of the lower side of the ultracentrifugation tube. Avoid generating bubbles.</p>
</figcaption>
</figure>
</li>
<li>Discard the supernatant by decanting or carefully pipetting without disturbing the pellet. If pipetting off the supernatant, keep the tip of the pipet on the side of the tube opposite from the pellet. If decanting the supernatant, be sure to pour the supernatant off the side of the tube opposite the pellet and use one quick, careful motion to invert the tube.
<div class="optional">
<span class="optional-title">Optional:</span> Store supernatant if interested in EVP-depleted media for measurement of other secreted factors, such as cytokines.</div>
<div class="note">
<span class="note-title">Note:</span> In this and subsequent steps it is important to work quickly and remove the supernatant as fast as possible after the ultracentrifuge has stopped to prevent dissociation of the pellet. After removing the supernatant, leave the pellet untouched and place the inverted tube in a rack for 1–3 min to avoid backflow of liquid. This ensures minimal carryover of contaminants and prevents dissociation of the pellet. Remove drops of liquid from tube walls by vacuum aspiration or using a Kimwipe, if necessary. <a href="#sec6.1">Troubleshooting 1</a>.</div>
</li>
<li>To wash the pellet, re-suspend it in 500 μL of cold PBS by pipetting and flushing the marked area at the bottom of the ultracentrifugation tube. Ensure the pellet is re-suspended well so contaminants trapped in the pellet (free, abundant proteins) are washed away. Transfer the re-suspended pellet to a new ultracentrifuge tube and add 3 mL (tube #355645), 20 mL (tube #355631) or 44 mL (tube #355628) of cold PBS. We recommend using the same size tube throughout the protocol (i.e., depending on starting volume) since the starting volume affects the yield. Thus, small volumes, 1–3 mL should be spun in a small tube where these fewer EVPs can more readily be collected and concentrated. <a href="#sec6.3">Troubleshooting 2</a>.</li>
<li>Centrifuge the re-suspended pellet at 100,000 × <i>g</i> for 70 min at 10°C to collect the washed EVP pellet.</li>
<li>Discard the PBS supernatant by decanting and remove all liquid by gently tapping the inverted tube to avoid backflow.</li>
<li>Re-suspend the EVP pellet in 100 μL (see note for range below) of cold PBS and store at −80°C. Reserve 10 μL aliquots for downstream quantification and characterization, such as protein quantification, NTA, and EM, to avoid freeze-thaw cycles.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The final re-suspension volume can be adjusted to create an appropriate concentration. As low as 50 μL or up to 500 μL can be used for high yields depending on the total EVP yield. Because cell lines vary in their EVP production, we recommend that users empirically establish their own EVP yield for each cell type. Final concentrations will depend on downstream applications but should be above the lower limit of detection of the BCA assay or other assay for protein quantification. If desired, the EVP pellet can be re-suspended directly in cell lysis buffer for downstream RNA or protein applications.</div>
<div class="optional">
<span class="optional-title">Optional:</span> After thawing of stored EVP samples and before downstream applications, depending on the type of downstream analysis, an additional centrifugation at 12,000 × <i>g</i> for 5 min at 4°C can be performed to remove potential aggregates/precipitates formed during storage.</div>
<h3 id="sec3.2">Isolation of EVPs from tissue samples</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section describes culture of dissected tissue explants and serial centrifugation steps to isolate EVPs from conditioned media. Fresh human tumor tissues and peritumoral adjacent tissues were obtained from patients surgically treated at Memorial Sloan Kettering Cancer Center (MSKCC) and Yonsei Cancer Center (YCC). The ages of patients ranged from 3 to 88 years old. All individuals or legal guardians provided informed consent for tissue donation according to protocols approved by the institutional review board of MSKCC, WCM, and YCC (IRB 11-033A, 16-774, 16-1514, and 15-015, MSKCC; IRB 0604008488, WCM; IRB 4-2019-0811, YCC). The study is compliant with all relevant ethical regulations regarding research involving human participants. All mouse experiments were performed in accordance with Institutional Animal Care and Use Committees (IACUC) and American Association for Laboratory Animal Science (AAALAS) guidelines (Weill Cornell Medicine animal protocol 0709-666A). Six to eight weeks old wild-type C57BL/6 or BALB/c female mice were purchased from Jackson Laboratories.</p>
<ol>
<li>Collect the tumor tissue and adjacent tissue in ice-cold PBS at the time of surgery and transfer them on ice to the class II biosafety cabinet within 30 min.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Tissue weight can be used to normalize the EVP yield. Work quickly or in PBS to avoid drying out tissue and decreasing EVP yield.</div>
<div class="note">
<span class="note-title">Note:</span> Viability will vary between tissue explant cultures depending on a number of factors, such as surgical approach, time to culture, and tissue composition. Viability can be assessed using suitable methods such as manual cell counting and trypan blue exclusion, automatic cell counting with dead cell discrimination dyes or flow cytometry with annexin V or propidium iodide. The process of making single cell suspensions can also affect the viability. We have compared our dissected tissue explants to whole tissue explants and found similar, slightly enhanced, viability in the dissected cultures (<a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/336-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Comparison of cell viability between whole tissue versus tissue dissected into smaller pieces</div>
<p>(A) Nanosight (NTA) profiles and protein concentration by bicinchoninic acid (BCA) assay for human colon that was cultured as whole tissue or dissected into smaller pieces.</p>
<p>(B) Immunoblotting of lysates (20 μg of proteins) from baseline tissue (directly after collection, timepoint 0 h) immediately placed into lysing buffer without culturing, whole tissue, and dissected tissue (after 12 h culture) derived from the human colon. PARP antibody (#9542, Cell Signaling Technology [CST]), cleaved caspase-3 antibody (#9661S, CST), GAPDH antibody (#2118S, CST) were used for immunoblotting.</p>
<p>(C) Comparison of human colon whole tissue and dissected tissue cell viability as determined by dual Acridine Orange / Propidium Iodide fluorescence and brightfield optics, using the LUNA automated cell counter. Live cells are green and dead cells are red.</p>
<p>(D) Comparison of human colon whole tissue and dissected tissue cell viability by apoptotic flow staining shows similar viability between baseline tissue directly after collection and whole and dissected tissues after 12 h tissue explant culture.</p>
<p>(E) Comparison of cell viability between baseline and 24 h cultures of mouse lung tissue explant.</p>
<p>(F) Cell viability, as determined by flow cytometry with LIVE/DEAD™ Fixable Blue Dead Cell Stain, shows increased viability in dissected tissue cultures compared to whole tissue culture from mice.</p>
</figcaption>
</figure>
<ol start="2">
<li>Wash tissues in cold PBS × 3 and quickly cut the tissues into 2 mm<sup>3</sup>-sized pieces. Rinse tissue pieces again in PBS before plating if needed to remove debris and/or blood.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Heterogeneous tumor and adjacent tissues might require dissection of specific areas of interest, such as removal of adipose tissue, tumor and dissection of specific structures (i.e. tumor margin), and thus, trained personnel is required.</div>
<ol start="3">
<li>Place tissue pieces in a 6-well plate (&lt;10 pieces) or a 10-cm dish (10–15 pieces).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to limit the number of dissected tissues per dish to no more than 15 pieces, which will minimize loss of tissue viability.</div>
<ol start="4">
<li>If possible, allow tissue pieces to adhere to the plastic dishes for a few minutes before adding media. Pipet media slowly against the side of the dish to avoid disturbing the tissue pieces. Ensure enough media is added to cover the tissue pieces and try to minimize detachment from the plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After initial media collection, consider adding EVP-depleted FBS and other essential growth factors to culture media if establishing long-term viable cultures is desired. Alternatively, organoids can be generated from the fresh tissue and EVPs can be continuously collected from these organoid cultures.</div>
<ol start="5">
<li>Carefully transfer plates to 37°C incubator with 5% CO<sub>2</sub>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep in a separate incubator reserved for <i>ex vivo</i> work, away from routine mycoplasma-free cell cultures, if feasible.</div>
<ol start="6">
<li>After 16–18 h, carefully collect culture media without disturbing tissue explants, especially if continuous culturing is planned. Keep the collected media on ice.</li>
<li>Centrifuge explant culture media according to the centrifugation steps described above for EVP isolation from cell cultures.
<ol type="a">
<li>Centrifuge the culture media at 500 × <i>g</i> for 10 min at 4°C. Remove supernatant to a new tube.</li>
<li>Centrifuge the supernatant at 3,000 × <i>g</i> for 20 min at 4°C. Remove supernatant to a new tube.</li>
<li>Centrifuge the supernatant at 12,000 × <i>g</i> for 20 min at 4°C, in either a tabletop centrifuge or in the ultracentrifuge, depending on volume.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Given the complex nature of tissue samples and the presence of significant debris compared to cell lines, always perform all three pre-ultracentrifugation steps (500 × <i>g</i>, 3,000 × <i>g</i>, and 12,000 × <i>g</i>) mentioned above to remove insoluble material. Additional filtrations and purification steps can be added as described below.</div>
<ol start="8">
<li>Isolate EVPs by ultracentrifugation at 100,000 × <i>g</i> for 70 min at 10°C.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> If further purification of a tissue sample EVP pellet is desired, re-suspend the EVP pellet in 2.5 mL of PBS and slowly overlay onto a 0.5 mL sucrose/deuterium oxide cushion layer by tilting the tube to 45°; then centrifuge at 100,000 × <i>g</i> for 70 min at 10°C (<a href="#fig4">Figure 4</a>). Aspirate the lower fraction using an 18 G needle and 3 mL syringe. Ensure not to disturb the layers or eventual pellet and collect a volume similar to the amount of sucrose/deuterium oxide mixture added (0.5 mL here). Note: the amount of sucrose/deuterium oxide mixture will depend on the volume of the sample/ultracentrifugation tube size used and should be used in a ratio of 1:5 sucrose/D<sub>2</sub>O:EVP/PBS suspension (<a href="#bib4">Lamparski et. al, 2002</a>). The end user will need to confirm that sucrose density-cushion purified EVPs retain their functional characteristics. <a href="#sec6.5">Troubleshooting 3</a>.</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/336-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Ultracentrifugation of tissue sample EVPs over a sucrose/deuterium oxide density layer</div>
<p>(A) EVPs re-suspended in PBS could be further purified by overlaying onto a sucrose/deuterium oxide cushion and centrifuging at 100,000 × <i>g</i>, resulting in a lower EVP-enriched layer that could be aspirated from the upper layer containing any potential contaminants.</p>
<p>(B) Aspiration of the lower fraction using an 18G needle and 3 mL syringe.</p>
</figcaption>
</figure>
<ol start="9">
<li>Re-suspend EVP pellet in 3 mL of PBS to wash and collect by ultracentrifugation at 100,000 × <i>g</i> for 70 min at 10°C.</li>
<li>Discard the supernatant and re-suspend collected EVP pellet in 100 μL of PBS.</li>
<li>Store at −80°C. Save 10 μL aliquots for protein quantification, NTA, and EM to avoid freeze-thaw cycles.</li>
</ol>
<h3 id="sec3.3">Isolation of EVPs from blood samples and other bodily fluids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>This section describes differential centrifugation steps to isolate EVPs from whole blood or bodily fluids, such as lymphatic fluid, and bile. Human blood and bodily fluids were obtained from patients treated at Memorial Sloan Kettering Cancer Center (MSKCC). The ages of patients ranged from 1 to 83 years old. All individuals or legal guardians provided informed consent for blood/fluid donation according to protocols approved by the institutional review board of MSKCC (IRB 11-033A, 16-774, 16-1514, and 15-015; MSKCC, IRB 0604008488, WCM). The study is compliant with all relevant ethical regulations regarding research involving human participants. All mouse experiments were performed in accordance with Institutional Animal Care and Use Committees (IACUC) and American Association for Laboratory Animal Science (AAALAS) guidelines (Weill Cornell Medicine animal protocol 0709-666A). Six to eight weeks old wild-type C57BL/6 or BALB/c female mice were purchased from Jackson Laboratories.</p>
<ol>
<li>Remove cells, debris, apoptotic bodies, and other contaminants by serial centrifugation steps.
<ol type="a">
<li>Centrifuge the EDTA tube containing the blood at 500 × <i>g</i> for 10 min at 4°C. The blood will separate into a lower layer containing red blood cells (RBC) and white blood cells (WBC) and an upper layer of plasma
<div class="note">
<span class="note-title">Note:</span> The plasma should be yellow and clear but not red, which is a sign of RBC lysis at collection that could potentially affect and contaminate the EVP preparation. Following blood collection in anti-coagulant-coated tubes, we follow the manufacturer’s protocol, and roll/invert tubes several times, to ensure the blood is mixed with the anti-coagulant, thus minimizing clotting and hemolysis. However, if visual observation of hemolysis is not sufficient and the user would like to confirm by quantitative measurement, we recommend the Nature protocol by Dr. Olivier Wever and Dr. An Hendrix using a spectrophotometer to confirm the absence of hemolysis by the lack of a spectrophotometric absorbance peak for free hemoglobin at 414 nm (<a href="#bib13">Tulkens et al., 2020</a>). Additionally, the presence of large amounts of lipids, as indicated by a cloudy appearance, might indicate the failure of fasting in subjects prior to blood collection. Consider excluding such samples.</div>
<div class="note">
<span class="note-title">Note:</span> For bile samples, and other very viscous samples, such as pancreatic duct fluid, add ice-cold PBS at a ratio of 1:1 and homogenize the mixture by repeated pipetting. Follow the same isolation steps as for blood.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is important not to disturb the pellet after each centrifugation step. Instead, collect most of the plasma while leaving a small residual volume above the pellet. We recommend handling a maximum of 10 plasma samples at one time since additional samples will increase the delay and the risk of EVP loss.</div>
</li>
<li>Collect the plasma by pipetting the top layer without touching the interphase layer and transfer into a new tube.</li>
<li>Centrifuge the plasma at 3,000 × <i>g</i> for 20 min at 4°C. This step will remove platelets that could otherwise be co-isolated with EVPs and contaminate the EVP preparation.
<div class="note">
<span class="note-title">Note:</span> Platelet numbers can be assessed by classic hemocytometer counting, cell counting with size exclusion or more precisely through flow cytometry of the platelet suspension stained for CD61 (platelet glycoprotein IIb, broadly expressed on resting and activated platelets).</div>
</li>
<li>Collect the plasma and discard the platelet-rich pellet if not needed.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The plasma can be frozen at −80°C for long-term storage.</div>
</li>
<li>Centrifuge the plasma at 12,000 × <i>g</i> for 20 min at 4°C in either a tabletop microcentrifuge or in an ultracentrifuge.
<div class="note">
<span class="note-title">Note:</span> This step will remove apoptotic bodies and larger vesicles that could otherwise be co-isolated with EVPs and contaminate the EVP preparation.</div>
<div class="optional">
<span class="optional-title">Optional:</span> The sample can be further processed to remove contaminants, such as lipids, by filtering the cleared post-12,000 × <i>g</i> plasma supernatant through a 1.2 μm membrane.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend performing the 12,000 × <i>g</i> spin right before the ultracentrifugation steps, especially for plasma and other bodily fluids as well as tissue-derived media, since protein aggregates and precipitates could develop through freeze-thaw cycles or over extended waiting times.</div>
<div class="optional">
<span class="optional-title">Optional:</span> After the 12,000 × <i>g</i> spin, save 100 μL aliquots of cleared plasma for EVP quantification and characterization by NTA.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Isolate EVPs by ultracentrifugation.
<ol type="a">
<li>Transfer the plasma supernatant to an ultracentrifuge tube (see <a href="#materials-and-equipment">Materials and equipment</a>). For plasma, we recommend the 50.4Ti rotor with the #355645 tube for volumes up to 4 mL.</li>
<li>Centrifuge sample at 100,000 × <i>g</i> for 70 min at 10°C in a fixed angle rotor in a Beckman Coulter ultracentrifuge (see <a href="#materials-and-equipment">Materials and equipment</a>).</li>
<li>Decant or carefully pipet off the plasma, leaving the pellet untouched; place the inverted tube in a tube rack to drain and to avoid liquid backflow. Note the volume of plasma for calculating EVP numbers and protein per milliliter plasma.</li>
</ol>
</li>
<li>Re-suspend the EVP pellet in 500 μL of ice-cold PBS and transfer to a second ultracentifuge tube.</li>
<li>Add 3.5 mL of PBS to wash the pellet.</li>
<li>Centrifuge the sample at 100,000 × <i>g</i> for 70 min at 10°C.</li>
<li>Discard the supernatant by decanting and ensure there is no liquid backflow onto the pellet.</li>
<li>Re-suspend the pellet in 100 μL of PBS. Store at −80°C. Save aliquots for protein quantification, NTA, and EM to avoid freeze-thaw cycles.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> EVPs can also be isolated from serum (in a red top blood collection tube without anticoagulant, Cat#366434G) using the same procedure; however, serum EVPs and plasma EVPs are not equivalent.</div>
<div class="note">
<span class="note-title">Note:</span> The complex nature of plasma and heterogeneity between patients makes EVP isolation challenging. Additional filtering and PBS dilutions can be made but could also introduce loss or bias. Optimization protocols are being developed to decrease the inter-sample variation in EVP isolation from blood. See further discussion under “<a href="#limitations">Limitations</a>” section.</div>
<h3 id="sec3.4">EVP sample preparation for AF4</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>EVPs isolated from different sources via ultracentrifugation are pre-processed for AF4 analysis.</p>
<ol>
<li>Adjust the protein concentration of freshly isolated or thawed frozen EVPs to 0.5–1 μg/μL with PBS.</li>
<li>Pre-clear the EVP suspension by centrifuging in a tabletop microcentrifuge at 12,000 × <i>g</i> at 4°C for 5 min to remove large protein and nanoparticle aggregates.</li>
<li>Transfer the supernatant into a pre-chilled screw cap vial. A pulled point glass vial insert can be used for samples with small volume.</li>
<li>Place the sample vial into the designated position in the pre-chilled autosampler.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To achieve the best resolution and yield, we have determined empirically that 40–100 μg of EVP protein is required for EVPs isolated from conditioned media of cell culture for AF4 processing (with our instrument settings). Due to the size of the sample injection loop, which determines the maximum sample volume to be loaded, a sample concentration range of 0.4–1 μg/μL is required for AF4. However, for EVPs derived from other sources, these parameters need to be optimized empirically due to different EVP complexity.</div>
<h3 id="sec3.5">AF4 of EVPs and data collection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 70 min</div>
<p>EVPs are separated based on their hydrodynamic sizes via AF4, and real-time data collection is carried out for concentration and size determination.</p>
<ol>
<li>Astra 6 (Wyatt Technology) is used for data acquisition and analysis, including MALS, DLS, and UV signals. Open a new experiment with the current configuration of the instrument, and set parameters for data collection: MALS data collection interval is 1 s and QELS interval is 2 s; total data collection time is 60 min. Save the file and click “RUN.” Data collection automatically starts once triggered by the autosampler sample loading.</li>
<li>Chemstation software (Agilent Technologies) with an integrated Eclipse module (Wyatt Technology) is used to operate the AF4. First, program a new method or load the desired method from the method library. The running method we have developed and optimized for EVP separation is the following:
<ul>
<li>Forward channel flow: 1 mL/min</li>
<li>Elution, 2 min, Vx 0.5 mL/min;</li>
<li>Focus, 1 min, Vx 0.5 mL/min;</li>
<li>Focus + Injection, 2 min, Vx 0.5 mL/min, injection flow 0.2 mL/min;</li>
<li>Elution, 45 min, Vx a linear gradient decreasing from 0.5 to 0 mL/min;</li>
<li>Elution, 5 min, Vx 0 mL/min;</li>
<li>Elution + Injection, 5 min, Vx 0 mL/min;</li>
</ul>
</li>
<li>In Chemstation, set the input sample volume in the autosampler module (&lt;=100 μL), and set the parameters for fraction collection in the fraction collector module. Here, we collect fractions by time slice (0.5 min/fraction).</li>
<li>Click “Run a single sample.” The sample will be automatically loaded and fractionated accordingly, and data collection by Astra 6 will start once the sample loading is done.</li>
<li>Fractions are collected into 96-well plates based on time slice by the Agilent Fraction Collector.</li>
<li>After all samples have been processed for the day, switch off all lasers and thermostats for both autosamplers and fraction collector. Switch on the COMET of the Dawn Heleos II detector for ~30 min to clean the flow cell. Change the buffer to water and run the Night Rinse mode ~16–18 h. For long-term storage, the system can be stored in 20% ethanol after running a large volume of water through the system.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> A blank control of the running buffer (i.e., PBS) can be run using the same running method in parallel to the EVP samples. This control can assist in evaluating instrument performance, such as the background noise level and baseline drifting.</div>
<div class="note">
<span class="note-title">Note:</span> Samples more than 100 μg or of a volume larger than 100 μL can be sequentially fractionated via AF4 using the same running method and fractions can be pooled together according to their hydrodynamic sizes.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Collected fraction plates can be sealed with adhesive tapes and kept on ice in 4°C for 2–3 days. However, we recommend proceeding to the next step of fraction concentration and characterization shortly after fraction collection if time allows.</div>
<h3 id="sec3.6">Fraction concentration and characterization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min or longer (timeframe varies based on the total amount of input samples to be analyzed)</div>
<p>The size of EVP fractions is analyzed using the Astra 6 software and fractions of similar size are pooled together for further concentration and downstream characterization.</p>
<ol>
<li>In Astra 6, adjust the baseline of data collected from each monitor in the experiment just completed and save the changes.</li>
<li>Open a new window of EASI Graph in Astra 6, and plot the hydrodynamic radius (Rh) of particles vs Time. Please refer to <a href="#bib15">Zhang and Lyden (2019)</a> for the underlying principle of hydrodynamic radius determination and fractionation efficiency evaluation using the Autocorrelation function. Both QELS and UV absorbance are displayed on a relative scale to illustrate the relative abundance of each subpopulation in the sample. A single peak or multiple peak regions can be selected for analysis simultaneously.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Representative individual fractions can be first concentrated as described in steps 34–36 and a small aliquot can be subjected to negative staining TEM analysis for quick morphology examination.</div>
<ol start="3">
<li>Guided by the particle size, abundance, and morphology, adjacent fractions with similar properties are then pooled together into 50 mL Falcon tubes and placed on ice.</li>
<li>Pre-wet the Millipore centrifugal filter columns with Ultracel-30 membrane (30 kDa cutoff) with cold PBS (fill to full) and spin in tabletop clinical centrifuge at 3,700 × <i>g</i> at 4°C for 5 min. Discard the residual liquid in the filter unit and flow-through in the collection tube and place the filter column back into the collection tube.</li>
<li>Transfer the pooled fractions into the filter column (top part) and spin at 3,700 × <i>g</i> at 4°C for 7–8 min. Discard the flow-through but keep the samples retained in upper chamber of the filter column (the concentrated sample).</li>
<li>Repeat step 34 until each pooled set is concentrated to the desired volume using a single filter column.</li>
<li>Mix the concentrated samples gently and transfer to Eppendorf tubes on ice. For better recovery, an additional volume of PBS can be added to rinse the filter column and combined with the sample.</li>
<li>Record the volume of the final product and remove an aliquot for BCA assay and/or NTA analysis to determine the protein concentration and/or particle concentration. A small aliquot (~1 μg) can be saved to verify particle integrity and morphology by TEM.</li>
<li>Use the EVP subsets immediately for downstream analysis or keep on ice for short-term storage. Aliquots can be kept frozen at −80°C for long-term storage.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For EVP samples from a specific type of source material that has never been analyzed previously by AF4, it is critical to examine the morphology of EVPs present in representative individual fractions by TEM first, before pooling fractions together for further processing and analysis. NTA analysis is not applicable for exomeres, which, due to their size, are below the detection limit of NTA technology (minimum size detected is 50 nm).</div>
<div class="note">
<span class="note-title">Note:</span> An alternative approach to concentrate a large volume of pooled fractions is ultracentrifugation at 100,000 × <i>g</i> at 10°C for 70 min. The pelleted EVP subsets can then be re-suspended in the desired volume of PBS or other types of solution as required by downstream analysis.</div>
<div class="note">
<span class="note-title">Note:</span> This AF4 fractionation protocol is developed and optimized for EVPs isolated via dUC method from conditioned media of cell culture. For complex samples, such as plasma-derived EVPs, further preparation of samples prior to AF4 fractionation may be required.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Due to the recent interest in understanding the role of EVPs in inter-organ communication and their value as biomarkers, the development of methods that could improve and optimize EVP isolation protocols from different sources, such as blood and tissues, is critical.</p>
<p>Cell lines, as well as their EVPs, remain most valuable tools for functional studies, including cancer research. However, we have shown that cell line EVPs do not always recapture the complexity and heterogeneity of tissue samples, such as tumor explants (<a href="#bib17">Hoshino et al., 2020</a>). While a cell line will produce EVPs specific for that tumor cell type, tumor explants contain a mixture of EVPs derived from the tumor cells, tumor stroma, and infiltrating immune cells. Similarly, bodily fluids also contain EVPs derived from distant organs, such as the liver and bone marrow. The lower complexity of cell line-derived EVPs, compared to tissue explants or bodily fluids, makes them suitable for targeted approaches, such as immunoaffinity-based isolation methods. Cultured cell lines lack the tissue organization and microenvironment, which may alter cell physiology and lead to differences in EVP content and secretion, but they allow the study of EVPs derived from a single cell type source. To overcome these shortcomings, patient-derived organoids can be used for EVP isolation, allowing us to partially recapitulate the tissue architecture while limiting the sources of EVPs to one or two cell types. Another advantage of using cell lines is the possibility to scale up cultures to obtain large amounts of EVPs for downstream applications. For example, with our protocol, the commonly used, commercially available human breast cancer cell line MDA-MB-231 (2 × 10<sup>6</sup> cells, 4 days of culture) yields around 15 μg of EVP protein per 25 mL of media, and the well-known murine melanoma cell line B16-F10 yields 35 μg of EVP protein per 15 mL of media after three days of a culture, initially seeded with 1.5 × 10<sup>6</sup> cells. The amounts of EVPs will vary largely between cell lines and culture conditions and therefore the scale of the culture will have to be empirically determined for each experiment depending on the amount of EVPs required for downstream applications.</p>
<p>For optimization of EVP isolation from tissue samples, we used organs harvested from healthy wild-type mice, murine metastatic lung tumor tissue, and human adjacent colon tissue (<a href="#fig5">Figure 5</a>). We compared EVP yields from whole tissue/organs to dissected (2 mm<sup>3</sup>) pieces and performed the following downstream characterization: total EVP protein amounts were measured by BCA; particle number, size and concentration were measured by NTA; and morphology was determined by TEM. Combining 5 BALB/c mice per group, we found that dissecting tissue explants into smaller pieces resulted in an increase in EVP protein yields measured by BCA assay in all tested tissues (<a href="#tbl2">Table 2</a>). Similar results were observed when EVP number per μg of tissue was measured by NTA, showing an increase in the number of EVPs in tissue explants that had been dissected into pieces. The more pronounced increase in EVPs detected by BCA, compared to NTA suggests that the increase in exomeres, or small protein particles, which are below the detection limit of NTA, is higher relative to the increase in larger extracellular vesicles/exosomes (this has to be further confirmed by AF4 experiments). Since downstream analysis protocols including proteomics and functional characterization require at least 10 μg of EVP protein, and NTA and TEM require at least 1 μg, increasing the yield of the tissue explant protocol ensures the feasibility of further experiments, while decreasing the number of animals required (meeting the 3 Rs principle - replacement, reduction, refinement), and reducing the cost of animal acquisition and maintenance. Taken together, these results demonstrate that dissecting tissue explants into smaller pieces before culture for EVP isolation improves EVP yield. However, it remains unclear whether dissected tissues versus whole tissue explants, and the tissue explant culture process itself, can alter tissue biology, promoting changes in the EVP cargo.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/336-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Dissecting tissues into 2 mm<sup>3</sup> pieces improves EVP yields without affecting EVP morphology</div>
<p>(A) Nanosight (NTA) profiles for whole organs and tissue dissected into smaller pieces for murine normal lung and metastasis-bearing lung.</p>
<p>(B) Transmission electron microscopy images of the same samples as in b, normal lung (upper row) and metastasis-bearing lung (lower row).</p>
<p>(C and D) (C) Particle size for EVPs from normal lung and metastasis-bearing lung and (D) EVP protein yields from whole tissue versus tissue dissected into smaller pieces. n = 3, mean and SD is shown; ∗p &lt; 0.05.</p>
</figcaption>
</figure>
<table id="tbl2">
<caption>Table 2. Increased EVP yields from dissected tissue explant cultures versus whole tissue cultures</caption>
<thead>
<tr>
<th rowspan="2">Tissue</th>
<th colspan="2">EVP concentration by BCA (ng/μL)</th>
<th colspan="2">EVP size by NTA (nm)</th>
<th colspan="2">The number of EVP particles by NTA (particles/mL)</th>
</tr>
<tr>
<th>Whole</th>
<th>Dissected</th>
<th>Whole</th>
<th>Dissected</th>
<th>Whole</th>
<th>Dissected</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="7">Human</td>
</tr>
<tr>
<td>Colon</td>
<td>353.3</td>
<td>741.8</td>
<td>109 (±3)</td>
<td>107 (±2)</td>
<td>1.3 × 10<sup>8</sup> (±1.0 × 10<sup>7</sup>)</td>
<td>3.1 × 10<sup>8</sup> (±2.0 × 10<sup>7</sup>)</td>
</tr>
<tr>
<td colspan="7">Mouse</td>
</tr>
<tr>
<td>Brain</td>
<td>29.1</td>
<td>329.4</td>
<td>134 (±6)</td>
<td>144 (±4)</td>
<td>3.2 × 10<sup>8</sup> (±2.9 × 10<sup>7</sup>)</td>
<td>4.9×10<sup>8</sup> (±1.8 × 10<sup>7</sup>)</td>
</tr>
<tr>
<td>Kidney</td>
<td>12.0</td>
<td>42.2</td>
<td>140 (±4)</td>
<td>159 (±16)</td>
<td>6.8 × 10<sup>8</sup> (±7.6×10<sup>7</sup>)</td>
<td>2.6 × 10<sup>8</sup> (±1.9×10<sup>7</sup>)</td>
</tr>
<tr>
<td>Liver</td>
<td>83.1</td>
<td>247.2</td>
<td>116 (±5)</td>
<td>143 (±14)</td>
<td>2.3 × 10<sup>8</sup> (±1.0 × 10<sup>7</sup>)</td>
<td>3.5 × 10<sup>8</sup> (±2.9 × 10<sup>7</sup>)</td>
</tr>
<tr>
<td>Lung</td>
<td>11.1</td>
<td>209.8</td>
<td>120 (±1)</td>
<td>115 (±1)</td>
<td>4.7 × 10<sup>9</sup> (±5.4 × 10<sup>7</sup>)</td>
<td>5.6 × 10<sup>9</sup> (±3.9 × 10<sup>7</sup>)</td>
</tr>
<tr>
<td>Pancreas</td>
<td>44.6</td>
<td>188.5</td>
<td>178 (±27)</td>
<td>202 (±53)</td>
<td>7.2 × 10<sup>7</sup> (±8.1 × 10<sup>6</sup>)</td>
<td>1.4 × 10<sup>8</sup> (±1.5 × 10<sup>7</sup>)</td>
</tr>
<tr>
<td>Spleen</td>
<td>7.7</td>
<td>92.0</td>
<td>145 (±14)</td>
<td>163 (±27)</td>
<td>6.1 × 10<sup>8</sup> (±2.9 × 10<sup>7</sup>)</td>
<td>5.3×10<sup>8</sup> (±2.0 × 10<sup>7</sup>)</td>
</tr>
<tr>
<td>Thymus</td>
<td>37.0</td>
<td>145.1</td>
<td>135 (±5)</td>
<td>138 (±11)</td>
<td>4.4 × 10<sup>8</sup> (±4.6 × 10<sup>7</sup>)</td>
<td>2.6 × 10<sup>8</sup> (±1.2 × 10<sup>8</sup>)</td>
</tr>
<tr>
<td>White adipose tissue</td>
<td>0.2</td>
<td>1,299.6</td>
<td>134 (±8)</td>
<td>130 (±10)</td>
<td>9.9 × 10<sup>7</sup> (±1.2 × 10<sup>7</sup>)</td>
<td>6.4 × 10<sup>8</sup> (±6.2 × 10<sup>7</sup>)</td>
</tr>
<tr>
<td>Brown adipose tissue</td>
<td>7.1</td>
<td>138.6</td>
<td>113 (±8)</td>
<td>153 (±16)</td>
<td>2.2 × 10<sup>8</sup> (±3.3 × 10<sup>7</sup>)</td>
<td>2.0 × 10<sup>8</sup> (±2.3 × 10<sup>7</sup>)</td>
</tr>
</tbody>
</table>
<p class="table-legend">EVP size is similar between the two conditions. EVP protein concentration was measured by BCA (ng/μL, left), EVP size (nm, middle) and concentration (number of EVP/μg tissue) were measured by Nanosight (right) for human colon tissue and murine pancreas, white adipose tissue (WAT), brown adipose tissue (BAT), lung, brain, spleen, thymus, kidney, and liver.</p>
<p>Bodily fluids are often rich in EVPs and 1 mL of fluid is sufficient to isolate enough material for downstream characterization and proteomics. Blood, as the most easily accessible fluid, can yield from a few micrograms to 20 μg EVP protein per mL plasma whereas bile could be even richer and contain up to double the amount of EVP protein (and ~1 × 10<sup>8</sup> EVPs/mL) compared to plasma. The amounts will vary based on health status and other variables, such as cancer stage, treatment, etc.</p>
<p>AF4 separates EVPs based on their hydrodynamic size within a large dynamic range. As reported in our recent work, AF4 facilitates the successful separation of distinct subsets of extracellular vesicles/exosomes and the identification of exomeres (<a href="#bib14">Zhang et al., 2018</a>). A representative AF4 fractionation profile of B16F10- derived EVPs and TEM images of pooled fractions corresponding to exomeres, Exo-S, and Exo-L are shown in <a href="#fig6">Figure 6</a>.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/336-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Representative AF4 separation of EVPs prepared from B16F10 cells using differential ultracentrifugation</div>
<p>(A) Representative AF4 fractionation profile showing hydrodynamic radius (Rh) and fractograms of UV absorbance and QELS plotted against time.</p>
<p>(B) Representative negative staining TEM images of combined fractions resolved by AF4 for corresponding exomeres, Exo-S, and Exo-L subpopulations. Colored arrows: representative EVPs in each subpopulation (adapted from <a href="#bib14">Zhang et al., 2018</a>).</p>
</figcaption>
</figure>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/336-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Example of “contaminated” samples purified over a sucrose/deuterium oxide density layer</div>
<p>Aggregates and insoluble material could contaminate the EVP pellet, especially in complex samples, such as tissue explants or bodily fluids. Purification methods, such as density cushion, could be performed to improve sample purity.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Although the differential ultracentrifugation (dUC) protocol provides a robust methodology to process and analyze a large number of specimens from different sources and with a wide range of sample volumes, it presents several limitations. dUC separates particles based on their hydrodynamic size and density but the resolution and the resultant EVP purity is limited. Only several groups of large vesicles, microparticles, and small EVPs (relatively enriched for exomeres and exosomes) are isolated. Moreover, EVPs in each group will be heterogeneous and contamination by other non-EVP proteins will likely occur (<a href="#bib11">Théry et al., 2006</a>; <a href="#bib10">Tauro et al., 2012</a>). Furthermore, the composition of bodily fluids, such as blood plasma, is much more complex than most other types of specimens, such as conditioned media from cell culture. One major complication is the high abundance in plasma of albumin, lipoproteins and other soluble proteins. These proteins/complexes cannot be efficiently eliminated from the small EVP fraction by the dUC method. Moreover, some of the lipoproteins are within the size range of EVPs (e.g., LDL 18–28 nm in diameter; IDL, 25–50 nm; VLDL 30–80 nm; Chylomicrons 75–600 nm). The presence of high amounts of contaminants can interfere with specific downstream analysis, such as multi-omics profiling. Therefore, removal of these abundant contaminants by coupling dUC with other separation strategies, such as immunoaffinity capture or size-exclusion chromatography, based on distinct biophysical/biochemical properties of EVPs is necessary to further improve the quality of EVP preparations and expand the dynamic range of the analysis for EVP-based biomarker discovery. Accordingly, a larger volume of starting material is usually required if additional purification procedures are included to yield enough EVP for final analysis.</p>
<p>In the past decade, a variety of strategies have been developed for EVP isolation. Besides the dUC method described here, examples include immunoaffinity capture (IAC), ultrafiltration (UF), size-exclusion chromatography (SEC), polymer-based precipitation, and microfluidics (<a href="#bib11">Théry et al., 2006</a>; <a href="#bib2">Chen et al., 2010</a>; <a href="#bib7">Merchant et al., 2010</a>; <a href="#bib5">Lässer et al., 2012</a>; <a href="#bib10">Tauro et al., 2012</a>; <a href="#bib3">Jørgensen et al., 2013</a>; <a href="#bib8">Nakai et al., 2016</a>; <a href="#bib6">Ko et al., 2017</a>; <a href="#bib9">Reátegui et al., 2018</a>; <a href="#bib1">Brennan et al., 2020</a>). We recently reported the application of AF4 for small EVP fractionation with improved resolution (<a href="#bib14">Zhang et al., 2018</a>; <a href="#bib15">Zhang and Lyden, 2019</a>). To choose the most suitable method for biomarker discovery using plasma samples, especially for potential clinical translation, several critical factors have to be taken into consideration, including the available specimen volume/size and the proposed downstream analysis. A compromise may have to be made between purity and yield. Labor- and time-efficiency are important factors to consider, as well. According to the MISEV2018 guidelines, dUC falls within the category of “intermediate recovery, intermediate specificity” (<a href="#bib12">Théry et al., 2018</a>). Given that dUC is reproducible and commonly used, can be quickly and easily mastered by researchers, and does not require specialized expertise or equipment, it is still the most accessible methodology for EVP isolation. This is very important for studies requiring cross-comparison and validation by multiple centers or studies involving sample collection and processing over a relatively long timespan. Another reason for dUC remaining the gold standard for EVP isolation is that no chemicals are used, thereby preserving the EV integrity, which is particularly important for downstream functional studies.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Variability in EVP yield.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Despite using the same ultracentrifuge manufacturer, model, and rotors, we have observed variability in EVP yield between different ultracentrifuges. Since each ultracentrifuge is unique, if possible, always use the same machine, or if it is necessary to use different or several machines, compare yields between machines and ensure test samples and controls are prepared using the same machine.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Low EVP yield.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>One option is to use a smaller sized tube to concentrate the collection of the EVPs in a smaller target area at the bottom of the tube and avoid loss by washing in large amounts of PBS. Another possibility is that the EVP pellet may be lost by dissolving into supernatant during prolonged handling times between steps. Work quickly or handle fewer samples at a time to avoid this problem.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Especially when isolating EVPs from human samples, one can expect a variety of contaminants, such as tissue debris and protein aggregates, and therefore extensive processing is necessary to remove them and to achieve satisfactory quality of EVPs.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>One approach for the purification of limited sample quantities is the use of a density separation step (see description in protocol and <a href="#fig7">Figure 7</a>). Other options are filtration and additional pre-ultracentrifugation spins, or extensive washes, all of which will lead to some loss of material. For sucrose/deuterium oxide cushion purification up to 50% loss in total EVP protein can be expected but depending on downstream applications, specific EVP populations can also be enriched. Purification steps should be included when the EVP product will be submitted for downstream “omics” analyses for which higher purity of the product is required. We recommend using the sucrose cushion for tissue explants where the collected EVP pellet still contains aggregates after the washing step, which are typically observed for certain organs, such as pancreas and liver. This also depends partly on disease condition and needs to be determined by the investigator.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Brennan, K., Martin, K., FitzGerald, S.P., O'Sullivan, J., Wu, Y., Blanco, A., Richardson, C., and Mc Gee, M.M. (2020). A comparison of methods for the isolation and separation of extracellular vesicles from protein and lipid particles in human serum. Sci. Rep. <i>10</i>, 1039.</p>
<p id="bib2">Chen, C., Skog, J., Hsu, C.H., Lessard, R.T., Balaj, L., Wurdinger, T., Carter, B.S., Breakefield, X.O., Toner, M., and Irimia, D. (2010). Microfluidic isolation and transcriptome analysis of serum microvesicles. Lab Chip <i>10</i>, 505-511.</p>
<p id="bib17">Hoshino, A., Kim, H.S., Bojmar, L., Gyan, K.E., Cioffi, M., Hernandez, J., Zambirinis, C.P., Rodrigues, G., Molina, H., Heissel, S., et al. (2020). Extracellular Vesicle and Particle Biomarkers Define Multiple Human Cancers. Cell <i>182</i>, 1044-1061.</p>
<p id="bib3">Jørgensen, M., Bæk, R., Pedersen, S., Søndergaard, E.K., Kristensen, S.R., and Varming, K. (2013). Extracellular Vesicle (EV) Array: microarray capturing of exosomes and other extracellular vesicles for multiplexed phenotyping. J. Extracell. Vesicles <i>2</i>, 20920.</p>
<p id="bib4">Lamparski, H.G., Metha-Damani, A., Yao, J.Y., Patel, S., Hsu, D.H., Ruegg, C., and Le Pecq, J.B. (2002). Production and characterization of clinical grade exosomes derived from dendritic cells. J. Immunol. Methods <i>270</i>, 211-226.</p>
<p id="bib5">Lässer, C., Eldh, M., and Lötvall, J. (2012). Isolation and characterization of RNA-containing exosomes. J. Vis. Exp. <i>2012</i>, e3037.</p>
<p id="bib6">Ko, J., Bhagwat, N., Black, T., Yee, S.S., Na, Y.J., Fisher, S., Kim, J., Carpenter, E.L., Stanger, B.Z., and Issadore, D. (2017). miRNA profiling of magnetic nanopore–isolated extracellular vesicles for the diagnosis of pancreatic cancer. Cancer Res. <i>78</i>, 3688-3697.</p>
<p id="bib7">Merchant, M.L., Powell, D.W., Wilkey, D.W., Cummins, T.D., Deegens, J.K., Rood, I.M., McAfee, K.J., Fleischer, C., Klein, E., and Klein, J.B. (2010). Microfiltration isolation of human urinary exosomes for characterization by MS. Proteomics Clin. Appl. <i>4</i>, 84-96.</p>
<p id="bib16">Minn, A.J., Gupta, G.P., Siegel, P.M., Bos, P.D., Shu, W., Giri, D.D., Viale, A., Olshen, A.B., Gerald, W.L., and Massague, J. (2005). Genes that mediate breast cancer metastasis to lung. Nature <i>436</i>, 518-524.</p>
<p id="bib8">Nakai, W., Yoshida, T., Diez, D., Miyatake, Y., Nishibu, T., Imawaka, N., Naruse, K., Sadamura, Y., and Hanayama, R. (2016). A novel affinity-based method for the isolation of highly purified extracellular vesicles. Sci. Rep. <i>6</i>, 33935.</p>
<p id="bib9">Reátegui, E., van der Vos, K.E., Lai, C.P., Zeinali, M., Atai, N.A., Aldikacti, B., Floyd, F.P., Khankhel, A.H., Thapar, V., Hochberg, F.H., et al. (2018). Engineered nanointerfaces for microfluidic isolation and molecular profiling of tumor-specific extracellular vesicles. Nat. Commun. <i>9</i>, 175.</p>
<p id="bib10">Tauro, B.J., Greening, D.W., Mathias, R.A., Ji, H., Mathivanan, S., Scott, A.M., and Simpson, R.J. (2012). Comparison of ultracentrifugation, density gradient separation, and immunoaffinity capture methods for isolating human colon cancer cell line LIM1863-derived exosomes. Methods <i>56</i>, 293-304.</p>
<p id="bib11">Théry, C., Amigorena, S., Raposo, G., and Clayton, A. (2006). Isolation and characterization of exosomes from cell culture supernatants and biological fluids. Curr. Protoc. Cell Biol.</p>
<p id="bib12">Théry, C., Witwer, K.W., Aikawa, E., Jose Alcaraz, M., Anderson, J.D., Andriantsitohaina, R., Antoniou, A., Arab, T., Archer, F., et al. (2018). Minimal information for studies of extracellular vesicles 2018 (MISEV2018): a position statement of the International Society for Extracellular Vesicles and update of the MISEV2014 guidelines. J. Extracell. Vesicles <i>7</i>, 1535750.</p>
<p id="bib13">Tulkens, J., De Wever, O., and Hendrix, A. (2020). Analyzing bacterial extracellular vesicles in human body fluids by orthogonal biophysical separation and biochemical characterization. Nat. Protoc. <i>15</i>, 40-67.</p>
<p id="bib14">Zhang, H., Freitas, D., Kim, H.S., Fabijanic, K., Li, Z., Chen, H., Mark, M.T., Molina, H., Martin, A.B., Bojmar, L., et al. (2018). Identification of distinct nanoparticles and subsets of extracellular vesicles by asymmetric flow field-flow fractionation. Nat. Cell Biol. <i>20</i>, 332-343.</p>
<p id="bib15">Zhang, H. and Lyden, D. (2019). Asymmetric-flow field-flow fractionation technology for exomere and small extracellular vesicle separation and characterization. Nat. Protoc. <i>14</i>, 1027-1053.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, David Lyden (<a href="mailto:dcl2001@med.cornell.edu">dcl2001@med.cornell.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The protocol includes all datasets generated or analyzed during this study. The study did not generate new unique code.</p>
<h3>Acknowledgments</h3>
<p>We thank the Electron Microscopy &amp; Histology services of the Weill Cornell Medicine Microscopy &amp; Image Analysis Core for TEM with <a href="https://doi.org/10.13039/100000002">NIH</a>, United States Shared Instrumentation Grant (S10RR027699) and Medical Illustration &amp; Design, part of the Medical Research Support Services of <a href="https://doi.org/10.13039/501100008005">Yonsei University College of Medicine</a>, South Korea, for all artistic support related to this work. We are grateful for the great AF4 technical support from Wyatt Technology. The authors gratefully acknowledge support from: the <a href="https://doi.org/10.13039/100000054">National Cancer Institute</a>, United States CA224175 (D.L.), CA210240 (D.L.), CA232093 (D.L.), CA163117 and CA207983 (D.L.), CA163120 (D.L.), CA169416 (D.L.), CA169538 (D.L.), CA218513 (D.L., H.Z.) and AI144301 (D.L., V.P.); the <a href="https://doi.org/10.13039/100000005">United States Department of Defense</a>, United States W81XWH-13-1-0425 (D.L.), W81XWH-13-1-0427, W81XWH-13-1-0249 (D.L.), and W81XWH-14-1-0199 (D.L.); <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a>, United States/WCM <a href="https://doi.org/10.13039/100012729">CTSC</a>, United States (NIH/<a href="https://doi.org/10.13039/100006108">NCATS</a> (UL1TR00457) (H.Z.); NIH/NCATS (UL1TR002384) (D.L. and H.Z.); the <a href="https://doi.org/10.13039/100006792">Hartwell Foundation</a>, United States (D.L.); the Thompson Family Foundation (D.L., D.K.); the STARR Consortium I9-A9-056 (D.L., H.Z.) and I8-A8-123 (D.L.); the Pediatric Oncology Experimental Therapeutics Investigator’s Consortium (D.L., T.M.T.); <a href="https://doi.org/10.13039/100001445">Alex’s Lemonade Stand Foundation</a>, United Sates (D.L.); the <a href="https://doi.org/10.13039/100001006">Breast Cancer Research Foundation</a>, United States (D.L.); the Feldstein Medical Foundation (D.L.); the Tortolani Foundation (D.L.); the Clinical &amp; Translational Science Center (D.L., H.Z.); the Mary Kay Ash Charitable Foundation (D.L., I.M.); the Malcolm Hewitt Weiner Foundation (D.L.); the Manning Foundation (DL., A.H.); the Daniel P. and Nancy C. Paduano Family Foundation (D.L); the James Paduano Foundation (D.L.); the Sohn Foundation (D.L.); the AHEPA Vth District <a href="https://doi.org/10.13039/100002002">Cancer Research Foundation (D.L., L.B., S.L.)</a>; the Daedalus Fund Selma and Lawrence Ruben Science to Industry Bridge Award (D.L.); the <a href="https://doi.org/10.13039/100001542">Children’s Cancer and Blood Foundation</a>, United States (D.L.); Susan G. Komen Postdoctoral Fellowship PDF15331556, JST <a href="https://doi.org/10.13039/501100009023">PRESTO, Japan</a> 30021, and <a href="https://doi.org/10.13039/501100001691">JSPS</a> KAKENHI, Japan JP19K23743 (A.H.); the <a href="https://doi.org/10.13039/501100003725">National Research Foundation of Korea</a>, South Korea (NRF) grant funded by the Korea government (<a href="https://doi.org/10.13039/501100014188">MSIT</a>) (2019R1C1C1006709, 2018R1A5A2025079, and 2020M3F7A1094093); a grant of the Korea Health Technology R&amp;D Project through the <a href="https://doi.org/10.13039/501100003710">Korea Health Industry Development Institute</a> (KHIDI), funded by the <a href="https://doi.org/10.13039/501100003625">Ministry of Health &amp; Welfare</a>, South Korea (KHIDIHI19C1015010020); "The Alchemist Project" through the <a href="https://doi.org/10.13039/501100003052">Ministry of Trade, Industry and Energy</a> (MOTIE, Korea) (20012443), and Severance Hospital Research fund for Clinical excellence (SHRC) (C-2020-0032 and C-2020-0025) (H.S.K.); The Swedish Cancer Society Pancreatic Cancer Fellowship, the Lions International Postdoctoral fellowship and the Sweden-America stipend (L.B.); the <a href="https://doi.org/10.13039/501100001807">São Paulo Research Foundation</a> (FAPESP) São Paulo, Brazil (2017/07117-7) (G.C.T); the <a href="https://doi.org/10.13039/501100001711">Swiss National Science Foundation</a> (SFNS), Switzerland Postdoc Mobility grant (P2SKP3_174785 and P400PB_186791) (F.A.V.); and the <a href="https://doi.org/10.13039/100000005">DoD</a> PRCRP Horizon Award, United States (W81XWH-19-PRCRP-HA) (S.L).</p>
<h3>Author Contributions</h3>
<p>L.B., A.H., and H.S.K designed the experimental approach, coordinated the project, interpreted the data, and wrote the manuscript. F.P.V., G.T., S.L., J.H., D.K., K.K. and Z.W. performed optimization experiments. T.E.H., M.P.Q., D.K., T.M.T., D.R.J., and W.R.J. provided human samples. K.E.G., C.M.K., V.P., I.M., and H.Z. contributed to data interpretation and wrote the manuscript. D.L. led the project, interpreted the data, and wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>D.L., A.H., H.S.K., and L.B. have filed a US patent application related to this work.</p>
</section>
</article>