<article>
<section>
<h1 id="header">Purification of micronuclei from cultured cells by flow cytometry</h1>
<p><time datetime="2021-03-12">Published: March 12, 2021</time></p>
<p>Eléonore Toufektchan<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and John Maciejowski<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Molecular Biology Program, Sloan Kettering Institute, New York, NY 10065, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:toufekte@mskcc.org">toufekte@mskcc.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:maciejoj@mskcc.org">maciejoj@mskcc.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100378">10.1016/j.xpro.2021.100378</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Micronuclei are aberrant nuclear compartments that form when chromosomes or chromosome fragments fail to incorporate into a primary nucleus during mitotic exit. Ruptures at the micronuclear envelope are associated with DNA damage and activation of immune sensing pathways. To gain insights into these processes, we have developed a method to purify ruptured micronuclei. This method paves the way toward understanding the consequences of micronuclear envelope rupture.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib10">Mohr et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>An optimized protocol for purifying micronuclei with ruptured nuclear envelopes</li>
<li>Use of fluorescent markers enables flow sorting of distinct populations of micronuclei</li>
<li>Preservation of micronuclear protein and DNA content for functional characterization</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/516-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Cell lines generation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 weeks before the experiment</div>
<p>This protocol aims to purify micronuclei from cultured cells by combining sucrose gradient separation (<a href="#bib3">Cummings, 1972</a>; <a href="#bib8">Ly et al., 2017</a>; <a href="#bib12">Shimizu et al., 1996</a>) and flow cytometry sorting (<a href="#bib11">Nüsse and Kramer, 1984</a>; <a href="#bib1">Abend et al., 1996</a>) to isolate micronuclei with a ruptured nuclear envelope. Existing methods can use density gradient centrifugation to separate micronuclei from primary nuclei but cannot distinguish between distinct types of micronuclei (<a href="#bib8">Ly et al., 2017</a>; <a href="#bib12">Shimizu et al., 1996</a>). In order to distinguish between intact and ruptured micronuclei we mark cells with GFP-cGAS (cyclic GMP-AMP synthase), which has previously been shown to rapidly localize to micronuclei upon micronuclear envelope rupture (<a href="#bib2">Bartsch et al., 2017</a>; <a href="#bib4">Dou et al., 2017</a>; <a href="#bib6">Harding et al., 2017</a>; <a href="#bib9">Mackenzie et al., 2017</a>; <a href="#bib13">Yang et al., 2017</a>). Micronuclei marked with GFP-cGAS can be identified and isolated using FACS. Therefore, a critical first step of this protocol is the generation of cells overexpressing GFP-cGAS. The H2B histone fused to the fluorescent protein mCherry must also be overexpressed in the cells to identify chromatin.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other fluorescently tagged proteins can be overexpressed and used to sort micronuclei. For example, GFP-BAF (<a href="#bib5">Halfmann et al., 2019</a>; <a href="#bib7">Liu et al., 2018</a>; <a href="#bib14">Young et al., 2020</a>), which localizes to sites of nuclear envelope rupture, can – in principle – also be used to mark and sort ruptured micronuclei.</div>
<ol>
<li>Generate a cell line overexpressing the fluorescent protein Histone H2B-mCherry to mark chromatin.
<ol type="a">
<li>Use a retroviral or lentiviral plasmid to stably express H2B-mCherry in your cell line of interest.</li>
<li>Infected cells should be flow-sorted to select mCherry positive cells.</li>
<li>Generation of a clonal population of cells is recommended.</li>
</ol>
</li>
<li>Generate a cell line overexpressing the fluorescent protein GFP-cGAS and a second cell line overexpressing both H2B-mCherry and GFP-cGAS.
<ol type="a">
<li>Use a retroviral or lentiviral plasmid to stably express GFP-cGAS in your cell line of interest.</li>
<li>Infected cells should be sorted to select GFP positive cells or mCherry and GFP double positive cells.</li>
<li>Generation of clonal population of cells is recommended.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We established this protocol using the parental cell lines HEK293T or HeLa. These cells are cultured in DMEM high glucose media supplemented with 1% Penicillin/Streptomycin and 10% FBS. The use of another parental cell line is possible but may need adjustments and further troubleshooting to achieve the same quality of micronuclei purifications as described in this protocol.</div>
<div class="note">
<span class="note-title">Note:</span> The generation of clonal cell lines permits the homogeneous overexpression of the fluorescently tagged proteins H2B-mCherry and GFP-cGAS.</div>
<div class="note">
<span class="note-title">Note:</span> The cell lines can be generated once, and stocks should be made before proceeding to the next steps of the protocol.</div>
<h3 id="sec1.2">Culture cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 days before the experiment</div>
<ol start="3">
<li>Thaw cells overexpressing the fluorescent proteins H2B-mCherry and GFP-cGAS at least 10 days before the experiment and seed them on an appropriate container (e.g., 1–2 × 10<sup>6</sup> cells in a 10-cm dish). Also thaw parental cells and cells overexpressing only H2B-mCherry to be used as controls for flow cytometry sorting.</li>
<li>Expand H2B-mCherry GFP-cGAS cells by passaging every 3 days to obtain 7.5–15 × 10<sup>7</sup> cells when starting the experiment. Control cells should be expanded to reach 5 × 10<sup>7</sup> cells when starting the experiment.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use large square cultures dishes (245 × 245 × 25 mm dishes) to expand the cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Homogeneous expression of the fluorescent markers should be verified by microscopy and/or flow cytometry before expanding the cells.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>cGAS</td>
<td>Cell Signaling Technology</td>
<td>Cat#15102; RRID:AB_2732795</td>
</tr>
<tr>
<td>GFP</td>
<td>Santa Cruz</td>
<td>Cat#sc-9996; RRID:AB_627695</td>
</tr>
<tr>
<td>mCherry</td>
<td>Abcam</td>
<td>Cat#ab213511; RRID:AB_2814891</td>
</tr>
<tr>
<td>SMC1</td>
<td>Bethyl Laboratories</td>
<td>Cat#A300-055A; RRID:AB_2192467</td>
</tr>
<tr>
<td>Goat anti-mouse IgG Alexa Fluor Plus 680</td>
<td>Invitrogen</td>
<td>Cat#A32729; RRID:AB_2633278</td>
</tr>
<tr>
<td>Goat anti-rabbit IgG Alexa Fluor Plus 800</td>
<td>Invitrogen</td>
<td>Cat#A32735; RRID:AB_2633284</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Aqua Hold Pap Pen 2</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#23-769-533</td>
</tr>
<tr>
<td>Cytochalasin B</td>
<td>Cayman Chemical Company</td>
<td>Cat#11328</td>
</tr>
<tr>
<td>DAPI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#62247</td>
</tr>
<tr>
<td>DMEM + high glucose (4,500 mg/L)</td>
<td>MSKCC Media Core Facility</td>
<td>N/A</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#26-400-044</td>
</tr>
<tr>
<td>Odyssey blocking buffer (TBS)</td>
<td>LI-COR Biosciences</td>
<td>Cat#927-50000</td>
</tr>
<tr>
<td>NP-40 Surfact-Amps Detergent</td>
<td>Thermo Scientific</td>
<td>Cat#85124</td>
</tr>
<tr>
<td>Paraformaldehyde</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AC416785000</td>
</tr>
<tr>
<td>Penicillin/streptomycin (1,000×)</td>
<td>Gemini Bio</td>
<td>Cat#400-109</td>
</tr>
<tr>
<td>ProLong Gold antifade reagent</td>
<td>Invitrogen</td>
<td>Cat#P36934</td>
</tr>
<tr>
<td>Protease inhibitor tablets, EDTA-free</td>
<td>Thermo Scientific</td>
<td>Cat#A32965</td>
</tr>
<tr>
<td>Reversine</td>
<td>Cayman Chemical Company</td>
<td>Cat#10004412</td>
</tr>
<tr>
<td>Spermidine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AC132740010</td>
</tr>
<tr>
<td>Spermine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AC132750010</td>
</tr>
<tr>
<td>Trypsin 0.05% EDTA 0.02% in Hanks' balanced salt solution (HBSS) with phenol red and without calcium/magnesium</td>
<td>MSKCC Media Core Facility</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>HEK293T</td>
<td>ATCC</td>
<td>Cat#ACS-4500</td>
</tr>
<tr>
<td>HEK293T H2B-mCherry #4</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>HEK293T H2B-mCherry #4 GFP-cGAS #1</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>HeLa</td>
<td>ATCC</td>
<td>Cat#CRM-CCL-2</td>
</tr>
<tr>
<td>HeLa H2B-mCherry #2</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>HeLa H2B-mCherry #2 GFP-cGAS #1</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Phoenix</td>
<td>Laboratory of Titia de Lange</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pQCXIB H2B-mCherry</td>
<td>This paper</td>
<td>Addgene #164244</td>
</tr>
<tr>
<td>pTK GFP-cGAS</td>
<td>Laboratory of James Chen</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Cytofunnel</td>
<td>Shandon</td>
<td>Cat#5991039</td>
</tr>
<tr>
<td>Cytospin 4 cytocentrifuge</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A78300003</td>
</tr>
<tr>
<td>FACSAria III cell sorter</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>JS-5.3 swinging bucket rotor</td>
<td>Beckman</td>
<td>Cat#368690</td>
</tr>
<tr>
<td>Ministrainer 40 μm</td>
<td>pluriSelect</td>
<td>Cat#43-10040-46</td>
</tr>
<tr>
<td>Nitrocellulose membrane</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10600002</td>
</tr>
<tr>
<td>Novex WedgeWell Tris Glycine Mini gels</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#XP08160BOX</td>
</tr>
<tr>
<td>Non-Stick RNase-free 1.5 mL microfuge tubes</td>
<td>Ambion</td>
<td>Cat#AM12450</td>
</tr>
<tr>
<td>Nunc Square BioAssay dishes</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#166508</td>
</tr>
<tr>
<td>Tissue grind pestle size 40 mL</td>
<td>Neta Scientific</td>
<td>KMBL-885302-0040</td>
</tr>
<tr>
<td>Tissue grind tube size 40 mL</td>
<td>Neta Scientific</td>
<td>KMBL-885303-0040</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Solution preparation</h3>
<table id="undtbl1">
<caption>Lysis buffer, pH 8.5</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Add to 500 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>1 M pH 8</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Magnesium acetate</td>
<td>1 M</td>
<td>2 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>Calcium chloride</td>
<td>1 M</td>
<td>3 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>Sucrose</td>
<td>1 M</td>
<td>0.32 M</td>
<td>160 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>0.5 M pH 8</td>
<td>0.1 mM</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>NP-40</td>
<td>10%</td>
<td>0.1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>~ 327.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Adjust the pH of the lysis buffer to 8.5. Store at 4°C for up to 6 months.</p>
<p>Immediately before use, supplement the lysis buffer with: Cytochalasin B (final concentration 10 μg/mL), Dithiothreitol (DTT; final concentration 1 mM), spermine (final concentration 0.15 mM), spermidine (final concentration 0.75 mM) and protease inhibitors (see Other solutions below).</p>
<table id="undtbl2">
<caption>Sucrose 1.8 M buffer, pH 8</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Add to 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>1 M pH 8</td>
<td>10 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Magnesium Acetate</td>
<td>1 M</td>
<td>5 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>0.5 M pH 8</td>
<td>0.1 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>Sucrose</td>
<td>n/a</td>
<td>1.8 M</td>
<td>616.14 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p>To dissolve the sucrose powder, place the solution in a water bath heated to 60°C for 30 min and mix the bottle every 5–10 min. Store at 4°C for up to 6 months.</p>
<p>Immediately before use, supplement the Sucrose 1.8 M Buffer with: DTT (final concentration 1 mM), spermine (final concentration 0.15 mM), spermidine (final concentration 0.75 mM), bovine serum albumin (BSA; final concentration 0.03%) and protease inhibitors (see Other solutions below).</p>
<table id="undtbl3">
<caption>Sucrose 1.6 M buffer, pH 8</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Add to 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>1 M pH 8</td>
<td>10 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Magnesium acetate</td>
<td>1 M</td>
<td>5 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>0.5 M pH 8</td>
<td>0.1 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>Sucrose</td>
<td>n/a</td>
<td>1.6 M</td>
<td>546.18 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p>To dissolve the sucrose powder, place the solution in a water bath heated to 60°C for 30 min and mix the bottle every 5–10 min. Store at 4°C for up to 6 months.</p>
<p>Immediately before use, supplement the sucrose 1.6 M buffer with: DTT (final concentration 1 mM), spermine (final concentration 0.15 mM), spermidine (final concentration 0.75 mM), BSA (final concentration 0.03%) and protease inhibitors (see Other solutions below).</p>
<h3 id="sec2.2">Supplemented PBS 1× buffer</h3>
<p>Supplemented PBS 1× must be prepared fresh and kept at 4°C. Immediately before use, supplement the amount of PBS 1× necessary for the experiment with: BSA (final concentration 0.03%), NP-40 (final concentration 0.1%) and protease inhibitors (see Other Solutions below).</p>
<p>The protease inhibitors should be dissolved separately in a small volume of PBS and filtered with a 0.45 μm filter before addition to the final solution of Supplemented PBS 1×.</p>
<p>DAPI can be added to the Supplemented PBS 1×, if specified in the protocol (final concentration 2 μg/mL).</p>
<table id="undtbl4">
<caption>PFA 16% stock solution, pH 7.2–7.3</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Add to 50 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>PFA</td>
<td>n/a</td>
<td>16%</td>
<td>8 g</td>
</tr>
<tr>
<td>NaOH</td>
<td>2 M</td>
<td>8 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>10×</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p>PFA 16% stock solution can be prepared in advance. Add 8 g of paraformaldehyde to 10 mL ddH<sub>2</sub>O and add 200 μL 2 M NaOH. Warm this solution in a water bath heated to 65°C for 30 min and shake the tube every 10 min. Then, add 5 mL of 10× PBS and 25–30 mL of ddH<sub>2</sub>O to the solution. Mix the final solution by inverting. Adjust the pH to 7.2–7.3 by adding small amounts of HCl solution. Adjust the solution to the final volume of 50 mL with ddH<sub>2</sub>O. Keep the solution at 4°C for up to 2 weeks. Alternatively, aliquots can be stored at −20°C, but freezing/thawing should be avoided.</p>
<table id="undtbl5">
<caption>Laemmli 4× stock solution, pH 6.8</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Add to 50 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>1 M pH 7</td>
<td>200 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>100%</td>
<td>40%</td>
<td>20 mL</td>
</tr>
<tr>
<td>SDS</td>
<td>n/a</td>
<td>8%</td>
<td>4 g</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>n/a</td>
<td>0.04%</td>
<td>0.02 g</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>100%</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p>Laemmli 4× Stock Solution can be prepared in advance. Mix all the reagents in a beaker, except for bromophenol blue. Adjust the pH to 6.8 by adding small amounts of HCl. Then, add bromophenol blue powder and ddH<sub>2</sub>O to a final volume of 50 mL and mix well. The solution is stable at 20°C–25°C.</p>
<table id="undtbl6">
<caption>Other Solutions</caption>
<thead>
<tr>
<th>Name</th>
<th>Stock concentration</th>
<th>Storage</th>
</tr>
</thead>
<tbody>
<tr>
<td>Calcium chloride</td>
<td>1 M</td>
<td rowspan="5">stable at 20°C–25°C</td>
</tr>
<tr>
<td>EDTA (ethylenediaminetetraacetate)</td>
<td>0.5 M pH 8</td>
</tr>
<tr>
<td>Magnesium acetate</td>
<td>1 M</td>
</tr>
<tr>
<td>Sucrose</td>
<td>1 M</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>1 M pH 8</td>
</tr>
<tr>
<td>BSA (bovine serum albumin)</td>
<td>30%, soluble in ddH<sub>2</sub>O</td>
<td rowspan="6">stable at −20°C for up to 6 months, make aliquots to avoid repeated freezing and thawing</td>
</tr>
<tr>
<td>Cytochalasin B</td>
<td>10 mg/mL, soluble in DMSO</td>
</tr>
<tr>
<td>DTT (dithiothreitol)</td>
<td>1 M, soluble in ddH<sub>2</sub>O</td>
</tr>
<tr>
<td>Reversine (Mps1 inhibitor)</td>
<td>5 mM, soluble in DMSO</td>
</tr>
<tr>
<td>Spermidine</td>
<td>1 M, soluble in ddH<sub>2</sub>O</td>
</tr>
<tr>
<td>Spermine</td>
<td>1 M, soluble in ddH<sub>2</sub>O</td>
</tr>
</tbody>
</table>
<p>To prepare the BSA 30% solution: warm the solution in a water bath heated to 37°C for 30 min to 1 h and invert the tube gently every 10 min. Do not vortex the solution in order to avoid excessive frothing. Filter the solution with a 0.45 μm filter when all the powder is completely dissolved.</p>
<h3 id="sec2.3">Microscopy setup</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>Prior to flow cytometry sorting, microscopy can be used to verify the success of micronuclei-primary nuclei separation after cell lysis (Cell Lysis and Sucrose Density Gradient). Images are acquired on a DeltaVision Elite system equipped with a DV Elite CMOS camera, microtiter stage, and ultimate focus module. We collect Z-stacks through the samples at 0.2 μm increments.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Many other fluorescent imaging systems will be suitable for this analysis.</div>
<p>Acquire images using the desired objective (we use a 60×/1.42 oil objective) and the 405 nm excitation filter (DAPI). Be sure to choose appropriate acquisition times and neutral density filters that avoid signal saturation.</p>
<h3 id="sec2.4">Sucrose density gradient preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–15 min</div>
<p>The sucrose density gradient should be prepared immediately before use.</p>
<p>Use a 25 mL pipette to add 15 mL of Sucrose 1.6 M Buffer to the bottom of a 50 mL conical tube. Then, add 20 mL of Sucrose 1.8 M Buffer in the conical tube, by putting the tip of the pipette to the top inner side of the conical tube and slowly pipette the solution (use the slowest speed of the pipette controller).</p>
<div class="critical">
<span class="critical-title">Critical:</span> If the Sucrose 1.8 M Buffer is pipetted too fast, it will not mix with the Sucrose 1.6 M Buffer appropriately and fail to form a density gradient.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The volumes described for the sucrose density gradient are optimal for 10 mL of lysate:sucrose mixture (sucrose density gradient). If you have a significantly lower volume of lysed cells, you should adapt the amounts of the sucrose buffers accordingly (for example, if you have 5 mL of lysate:sucrose mixture, use a sucrose density gradient made of 7.5 mL sucrose 1.6 M buffer and 10 mL sucrose 1.8 M buffer). Similarly, if you have a significantly higher volume of lysed cells, you should prepare several sucrose density gradients (several tubes) and process the lysed cells in several batches.</div>
<h3 id="sec2.5">Flow cytometry sorting setup</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>All the sorting experiments were established with the BD FACSAria III Cell Sorter with the 100 μm nozzle. The sorter needs to have a refrigerated collection tube holder.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The Sony SH800S can also be used to sort the samples. Other cell sorters are likely suitable for this application but have not been tested.</div>
<div class="note">
<span class="note-title">Note:</span> the use of the 100 μm nozzle ensures a mild stream in order to prevent extensive damage to micronuclei during the sort.</div>
<p>Perform quality controls and Accudrop and select the desired filtered sets:</p>
<ul>
<li>DAPI channel: Laser 405–450/40</li>
<li>GFP channel: Laser 488–530/30</li>
<li>mCherry Channel: Laser 561–610/20</li>
</ul>
<p>All the channels, as well as forward scatters (FSC), must be in logarithmic view. Set the thresholds for FSC and for DAPI to the lowest threshold value allowed by the sorter. Typically, FSC threshold is set to voltage 500 V and DAPI threshold is set to voltage 300 V.</p>
<p>Adjust the voltages for each channel, including FSC, DAPI, GFP, and mCherry channels, to ensure that the positive populations are present on the displayed scale, using the following controls (all the channels must be in logarithmic view):</p>
<ul>
<li>Parental cells (not overexpressing any fluorescent proteins) untreated</li>
<li>Parental cells treated with Reversine for 24–48 h</li>
<li>Cells overexpressing the fluorescent protein H2B-mCherry and treated with Reversine for 24–48 h to induce micronuclei formation</li>
<li>Cells overexpressing both H2B-mCherry and GFP-cGAS and treated with Reversine for 24–48 h.</li>
</ul>
<p>First use the parental cells treated with Reversine to set the voltages of FSC and DAPI. Then, use the cells overexpressing the fluorescent proteins to set the voltage of the GFP and the mCherry channels.</p>
<p>Set the gates to isolate the micronuclei population (see <a href="#fig1">Figure 1</a>):</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/516-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Selecting flow cytometry gates to isolate distinct micronuclei populations</div>
<p>(A) Graph displaying all events detected by flow cytometry. Gates drawn to isolate micronuclei are indicated. In gate 1: blue dots represent total micronuclei; red dots represent intact micronuclei and green dots represent ruptured micronuclei.</p>
<p>(B) Graph displaying the events of gate 1 (A) and illustrating the gates drawn to isolate the ruptured micronuclei events (gate 3).</p>
</figcaption>
</figure>
<ul>
<li>Display <b>all events</b> in a graph [y-axis = FSC-A – x-axis = DAPI-A] (see <a href="#fig1">Figure 1</a>A)</li>
<li>Draw <b>gate 1</b> around the population of <b>total micronuclei</b> (use the parental cells untreated or treated with Reversine to identify this population): micronuclei should have a lower DAPI signal intensity and a lower FSC than the primary nuclei</li>
<li>Display <b>gate 1</b> in a graph [y-axis = mCherry-A – x-axis = GFP-A] (see <a href="#fig1">Figure 1</a>B)</li>
<li>Draw <b>gate 2</b> around the population of <b>intact micronuclei</b> (i.e., with no or very low GFP signal intensity): gate 2 should represent 15%–30% events of gate 1</li>
<li>Draw <b>gate 3</b> around the population of <b>ruptured micronuclei</b> (i.e., with high GFP signal intensity): gate 2 should represent 5%–10% events of gate 1</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Micronuclei display a wide variety of diameters, ranging from very small micronuclei (0.5–1 μm) to large micronuclei (10-15 μm). <b>Gate 1</b> can be drawn to span from low FSC (500 V) to high FSC (1,500 V) in order to collect both small and large micronuclei. Nevertheless, it is not possible to guarantee that all micronuclei will be collected.</div>
<div class="note">
<span class="note-title">Note:</span> The percent of events described for <b>gate 2</b> and <b>gate 3</b> applies to HeLa and HEK293T cells. These numbers were determined using immunofluorescence after micronuclei sorting to confirm that GFP-cGAS only localizes in micronuclei from <b>gate 3</b> and is absent from micronuclei in <b>gate 2</b>. The percentages can vary when different parental cell lines are used.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The samples should never be vortexed. Vortexing the sample can lead to irreversible damage of the nuclei and micronuclei, which can compromise the outcome of the experiment.</div>
<div class="note">
<span class="note-title">Note:</span> To homogenize the samples, gently invert the FACS tubes 5 times. If one sample is sorted for longer than 2 h, the micronuclei will tend to sediment at the bottom of the tube. If this occurs, pause flow sorting and homogenize again by inverting the tube before resuming sorting.</div>
<div class="note">
<span class="note-title">Note:</span> Other events can be detected in the graph [y-axis = FSC-A – x-axis = DAPI-A]: events with high DAPI and high FSC represent primary nuclei; events with low DAPI and high FSC represent non-lysed cells or large cell debris; events with low DAPI and low FSC represent the undissolved reagents in the buffer (such as protease inhibitors, BSA or NP-40); events with high DAPI and low FSC represent the small cell debris or undissolved reagents in the buffer (such as DAPI stain). To limit the frequency of these events, filter the PBS 1× solution. Check the <a href="#troubleshooting">troubleshooting</a> section for more details.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>The protocol was established in HEK293T and HeLa cells overexpressing H2B-mCherry and GFP-cGAS fluorescent proteins. These cell lines should be used for best results. If other parental cell lines are used (either another parental cell line or other fluorescent proteins), the protocol should be adapted at each step, as indicated below.</p>
<h3 id="sec3.1">Cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<p>In the next steps, the cell culture steps necessary for the preparation of the samples are described in detail.</p>
<ol>
<li>Seed the cells overexpressing H2B-mCherry and GFP-cGAS fluorescent proteins 24 h before drug treatment at desired density on an appropriate container (i.e., 3–10 of 245 × 245 × 25 mm dishes; 2.5 × 10<sup>7</sup> cells per dish).</li>
<li>For control samples, seed 2 dishes of parental cells, as well as cells overexpressing only H2B-mCherry (1 dish per cell line), 24 h before drug treatment at desired density on an appropriate container (1–2 × 2.5 × 10<sup>7</sup> cells per sample in a 245 × 245 × 25 mm dish).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The final volume of media contained in the 245 × 245 × 25 mm dishes should be 80 mL per dish for ideal cell growth.</div>
<div class="critical">
<span class="critical-title">Critical:</span> 2–6 × 10<sup>8</sup> cells are typically necessary to isolate sufficient micronuclei for most downstream applications. However, culturing more than 10 × 245 × 245 × 25 mm dishes of cells at once is not advised, as it will significantly increase the time for sample preparation and sorting.</div>
<h3 id="sec3.2">Drug treatment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>In the next steps, drug treatment with an Mps1 inhibitor is described in detail.</p>
<ol start="3">
<li>Treat all cultured cells (except one dish of parental cells) with the Mps1 inhibitor Reversine at the final concentration of 0.5 μM (1:10,000 dilution of the stock solution).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other chemical inhibitors of Mps1, such as AZ3146 or BAY-1217389, or spindle poisons, such as nocodazole or taxol, can also be used to perturb chromosome segregation during mitosis and induce micronuclei formation.</div>
<ol start="4">
<li>Culture cells at 37°C for 24–48 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The duration of the drug treatment with Reversine has been adapted for high levels of micronuclei accumulation in HEK293T (48 h culture post-treatment) or HeLa cells (24 h culture post-treatment). If different parental cells are used, or if a milder induction of micronuclei is desired, the appropriate duration of the treatment should be established in advance by microscopy of DAPI stained cells.</div>
<div class="note">
<span class="note-title">Note:</span> The drug treatment induces the formation of a high number of micronuclei (1–6 micronuclei per cell), which is necessary to increase the yield of the experiment. If no drug treatment is used, the yield of the experiment will be significantly reduced.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Drug treatment will partially affect cell growth and can lead to cell death. Use a higher seeding density to avoid major cell loss. If parental cells other that HEK293T or HeLa cells are used, cell viability after drug treatment and seeding density should be established in advance.</div>
<h3 id="sec3.3">Cell samples collection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 h</div>
<p>In the next steps, the collection of samples from cultured cells is described in detail.</p>
<ol start="5">
<li>Harvest the cultured cells (cell lines expressing the fluorescent proteins H2B-mCherry and GFP-cGAS ± drug treatment; control cells ± drug treatment) in 50 mL conical tubes.
<ol type="a">
<li>Wash cells with 30 mL PBS 1× per cell culture dish.</li>
<li>Add 20 mL of pre-warmed (37°C) Trypsin per cell culture dish.
<div class="note">
<span class="note-title">Note:</span> The cells will not detach easily and form more aggregates if the volume of Trypsin used in the 245 × 245 × 25 mm dishes is lower than 20 mL.</div>
</li>
<li>Incubate the dishes at 37°C for 10 min to enhance Trypsin activity.</li>
<li>After 10 min, verify that the cells are detached from the dish using a microscope. If the cells are not detached and/or form aggregates, place the dish back at 37°C and incubate for 2–5 min more.</li>
<li>Add 10 mL of pre-warmed (37°C) fresh culture medium and collect the cells in a 50 mL conical tube. Use one 50 mL conical tube per culture dish.</li>
<li>Wash the dish with an additional 10 mL of fresh culture medium and add this volume to the conical tube.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If working with more than 5 culture dishes, process in batches of 1–5 dishes at a time, to avoid prolonged incubation in Trypsin.</div>
<ol start="6">
<li>Collect a sample of trypsinized cells (one for each cell line ± drug treatment) for cell counting.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use the Beckman Z1 Coulter Particle Counter to count the cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Accurate estimates of the cell number in each sample is essential. See <a href="#tbl1">Table 1</a> (<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>) to determine how to process the samples in the next steps of the experiment.</div>
<table id="tbl1">
<caption>Table 1. Cell sample harvesting and lysis</caption>
<thead>
<tr>
<th>Sample</th>
<th>Total cells</th>
<th>Washing medium 1 and 2 (mL)</th>
<th>Medium + cytochalasin B (mL)</th>
<th>Lysis buffer (mL)</th>
<th>Sucrose 1.8 M buffer (mL)</th>
<th>Lysate 1:1 sucrose mixture (mL)</th>
<th>Number of sucrose density gradients</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sample</td>
<td>Number of cells</td>
<td>10<sup>8</sup> cells per 10 mL medium</td>
<td>10<sup>8</sup> cells per 10 mL medium</td>
<td>10<sup>8</sup> cells per 5 mL lysis buffer</td>
<td>= lysis buffer volume</td>
<td>= lysate + sucrose 1.8 M buffer (1:1)</td>
<td>10 mL lysate 1:1 sucrose mixture/1 gradient</td>
</tr>
<tr>
<td>Example</td>
<td>6 × 10<sup>8</sup>
</td>
<td>60</td>
<td>60</td>
<td>30</td>
<td>30</td>
<td>60</td>
<td>6 gradients</td>
</tr>
</tbody>
</table>
<ol start="7">
<li>Centrifuge the harvested cells at 300 × <i>g</i> for 5 min (at 20°C–25°C).</li>
<li>Discard the supernatant and keep the pelleted cells in the 50 mL conical tube.</li>
<li>Wash the pellet in fresh culture medium without FBS (first wash).
<ol type="a">
<li>Resuspend the cell pellet in 2 mL of fresh culture medium using a 1000 μL pipette tip.</li>
<li>Add fresh culture medium without serum to reach the washing buffer volume indicated in <a href="#tbl1">Table 1</a> (<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If one cell sample has been collected in several conical tubes, the cell pellets can be pooled before resuspension in the final volume of culture medium without serum.</div>
<ol start="10">
<li>Centrifuge the cells at 300 × <i>g</i> for 5 min (at 20°C–25°C).</li>
<li>Discard the first wash and keep the pellet in the conical tube.</li>
<li>Wash the pellet in fresh culture medium without FBS (second wash).</li>
<li>Centrifuge the cells at 300 × <i>g</i> for 5 min (at 20°C–25°C).</li>
<li>Discard the second wash and keep the pellet in the conical tube.</li>
<li>Resuspend the cell pellet in pre-warmed (37°C) culture medium without FBS supplemented with cytochalasin B (final concentration of 10 μg/mL). Cytochalasin B is used to help facilitate separation of micronuclei from nuclei.
<ol type="a">
<li>Resuspend the cell pellet in 1 mL of fresh culture medium supplemented with Cytochalasin B using a 1000 μL pipette tip before adding the rest of the medium to reach the final desired volume (See <a href="#tbl1">Table 1</a>; <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</li>
</ol>
</li>
<li>Incubate the cells in the medium supplemented with Cytochalasin B for 30 min at 37°C.</li>
<li>Centrifuge the cells at 300 × <i>g</i> for 5 min (at 4°C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cytochalasin B is an actin polymerization inhibitor which helps efficiently dissociate the micronuclei from their primary nuclei prior and during cell lysis (<a href="#bib12">Shimizu et al., 1996</a>).</div>
<h3 id="sec3.4">Cell lysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<p>In the next steps, lysis of the collected cell samples is described in detail.</p>
<div class="critical">
<span class="critical-title">Critical:</span> From step 18 onward all work should be performed on ice.</div>
<ol start="18">
<li>Place the conical tubes containing the pelleted samples on ice.</li>
<li>Discard the culture medium supplemented with Cytochalasin B and keep the cell pellet on ice.</li>
<li>Resuspend the cell pellet in ice-cold lysis buffer.
<ol type="a">
<li>Resuspend the cell pellet using a 5 mL pipette with lysis buffer by gently pipetting up and down 5 times.</li>
<li>Add additional volume of lysis buffer to reach the final volume of solution (See <a href="#tbl1">Table 1</a>; <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</li>
</ol>
</li>
<li>Transfer the cell lysate into a Dounce-homogenizer.</li>
<li>Gently Dounce-homogenize the sample by using a loose-fitting pestle (10 strokes).</li>
<li>Transfer the cell lysate back into the 50 mL conical tube.</li>
</ol>
<p><b><i>Optional</i> verification step:</b> Homogenize the lysate. Collect 50 μL of the lysate and mix with an equal volume of PBS 1× + 0.5 μg/mL of DAPI. Examine the sample under a fluorescence microscope. The primary nuclei and micronuclei should be released from other cellular compartments and the micronuclei and primary nuclei should be separated from each other (see <a href="#fig2">Figure 2</a>A; <a href="#troubleshooting">troubleshooting</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/516-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Verification steps of the micronuclei purification protocol</div>
<p>(A) Immunofluorescence for GFP (cGAS) and DAPI staining of the lysate obtained after the lysis and Dounce homogenization of cells. Scale bar, 25 μm.</p>
<p>(B) Immunofluorescence for GFP (cGAS) and DAPI staining of the different fractions obtained after the centrifugation of the cell lysate on the sucrose gradient. Fraction 1, cell debris; fraction 2, micronuclei; fraction 3, primary nuclei. Scale bar, 25 μm. Red arrows indicate intact micronuclei (GFP-cGAS-negative). Green arrows indicate ruptured micronuclei (GFP-cGAS-positive).</p>
</figcaption>
</figure>
<ol start="24">
<li>Mix the cell lysate with an equal volume of Sucrose 1.8 M Buffer: [Lysate 1:1 Sucrose].</li>
</ol>
<h3 id="sec3.5">Sucrose density gradient</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–45 min</div>
<p>In the next steps, the processing of the cell lysates through the sucrose density gradient is described in detail.</p>
<ol start="25">
<li>Prepare the sucrose density gradient (Sucrose Density Gradient Preparation) in a 50 mL conical tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> One sucrose density gradient (in a 50 mL conical tube) is sufficient to process 10 mL of the mixture [Lysate 1:1 Sucrose] prepared in step 24. Prepare as many sucrose density gradient tubes as necessary to process all of the sample.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Never invert the conical tube containing the sucrose density gradient to avoid disturbing the gradient.</div>
<ol start="26">
<li>Slowly pipette 10 mL of the mixture [Lysate 1:1 Sucrose 1.8 M Buffer] on top of the 2-layered Sucrose Density Gradient.</li>
<li>Centrifuge the samples in a JS-5.3 swinging bucket rotor at 944 × <i>g</i> for 20 min at 4°C.</li>
</ol>
<ol start="28">
<li>Collect the following fractions (see <a href="#fig2">Figure 2</a>B):<br> - Fraction 1 = upper 2 mL (cell debris)<br> - Fraction 2 = following 5 mL (micronuclei)<br> - Fraction 3 = following 38 mL (primary nuclei)
<ol type="a">
<li>Discard fractions 1 and 3.</li>
<li>Transfer Fraction 2, containing the micronuclei, to a new 50 mL conical tube.</li>
<li>Keep the new conical tube on ice.</li>
</ol>
</li>
</ol>
<p><b><i>Optional</i> verification step:</b> Before discarding any fraction, collect 50 μL of each fraction and mix it to an equal volume of PBS 1× + 0.5 μg/mL of DAPI. Examine the samples under a fluorescence microscope. Fraction 1 should contain very small particles and some micronuclei. Fraction 2 should contain micronuclei and some small primary nuclei (<a href="#fig2">Figure 2</a>B; <a href="#troubleshooting">troubleshooting</a>). Micronuclei should be separate from primary nuclei. Fraction 3 should contain primary nuclei, as well as aggregates of nuclei and large cell debris.</p>
<div class="note">
<span class="note-title">Note:</span> The fraction volumes described above are optimal for samples made using a sucrose density gradient of 35 mL (15 mL Sucrose 1.6 M Buffer + 20 mL Sucrose 1.8 M Buffer) and 10 mL of mixture [Lysate 1:1 Sucrose]. If one of the sucrose density gradients used in the experiment has different proportions, the volumes of the fractions should be adapted accordingly.</div>
<h3 id="sec3.6">Micronuclei sample preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–45 min</div>
<p>In the next steps, the preparation of micronuclei before flow cytometry sorting is described in detail.</p>
<ol start="29">
<li>Dilute fraction 2 1:5 with ice-cold supplemented PBS 1×.</li>
<li>Centrifuge the samples in a JS-5.3 swinging bucket rotor at 944 × <i>g</i> for 20 min at 4°C.</li>
<li>Discard the supernatant carefully but leave around 500 μL of buffer with the pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The pellet should be visible.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not invert the conical tube when discarding the supernatant to avoid losing the micronuclei pellet. Similarly, strong aspiration should be avoided. We prefer using a 10 mL pipette and a pipette controller to discard the supernatant.</div>
<ol start="32">
<li>Resuspend the micronuclei pellet in ice-cold supplemented PBS 1× containing 2 μg/mL DAPI. (See <a href="#tbl1">Table 1</a>).
<ol type="a">
<li>Resuspend the pellet using a 1000 μL pipette tip by gently pipetting up and down 5 times.</li>
</ol>
</li>
<li>Filter the samples through a 40 μm ministrainer into FACS tubes.
<ol type="a">
<li>Do not force the passage of the sample through the filter. Forcing sample through the filter can damage micronuclei or result in delays during flow sorting.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Filters with a size range between 20 μm and 40 μm can be used with similar results. However, filters classically used for flow cytometry cell sorting (which are larger than 40 μm) should be avoided.</div>
<ol start="34">
<li>Keep filtered samples on ice in the dark.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Micronuclei samples should be sorted by flow cytometry (see following steps) as soon as possible. Delays may result in a need for samples to be re-filtered.</div>
<h3 id="sec3.7">Flow cytometry sorting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–6 h</div>
<p>In the next steps, the sorting of the different sub-populations of micronuclei is described in detail.</p>
<ol start="35">
<li>Sort the desired samples with a BD FACSAria III Cell Sorter (Flow Cytometry Sorting Setup).
<ol type="a">
<li>Use the control samples to draw the gates of total, intact, and ruptured micronuclei. Draw an additional gate for primary nuclei if desired (see <a href="#fig1">Figure 1</a>).</li>
<li>Wash the system by running water between each sample for 1 min.</li>
<li>Add ice-cold supplemented PBS 1× in the collection tubes before collecting the samples and invert the tubes as many times as necessary to coat the interior of the tubes with the buffer.
<div class="note">
<span class="note-title">Note:</span> We typically use 1.5 mL conical tubes for downstream immunofluorescence applications and 15 mL conical tubes for downstream immunoblotting applications.</div>
</li>
<li>Invert the sample FACS tubes 5 times to homogenize the sample.
<div class="critical">
<span class="critical-title">Critical:</span> Never vortex the tubes containing nuclei and/or micronuclei samples to prevent irreversible damage to the samples.</div>
</li>
<li>Sort the desired gates at a flow rate 1,500–3,000 events/s, mode 4-way purity.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A flow rate faster than 3,000 events/s can lead to increased risk of error during sorting.</div>
<div class="note">
<span class="note-title">Note:</span> The 4-way purity mode is slower than the other sorting modes available, but it will sort samples with the lowest error rate.</div>
<ol start="36">
<li>Put the collection tubes on ice after sorting until further processing.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The sorting of ruptured micronuclei can take up to several hours if the number of micronuclei desired is high and/or if the samples contain low levels of micronuclei.</div>
<h3 id="sec3.8">Processing of sorted samples</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–90 min</div>
<p>In the next steps, the processing of the sorted micronuclei is described in detail.</p>
<ol start="37">
<li>Sorted samples can be processed differently according to further applications:
<ol type="a">
<li>Immunofluorescence:
<ol type="i">
<li>Collect 300,000 micronuclei (or 100,000 primary nuclei) in 1.5 mL low-binding collection tubes.
<div class="note">
<span class="note-title">Note:</span> 300,000 micronuclei are the ideal number of micronuclei to sort in order to have a concentrated sample to image by microscopy. However, the immunofluorescence can be performed on less micronuclei (150,000 minimum).</div>
</li>
<li>Fix the samples with PFA: add the appropriate volume of PFA 16% to the sample tube to reach a final concentration of 2%–4%.</li>
<li>Invert the tubes 5 times and incubate at 20°C–25°C for 15 min.</li>
<li>Transfer the tubes into cytospin funnels (1 funnel can contain up to 500 μL).</li>
<li>Cytospin the samples onto microscopy slides at 1000 rpm for 10 min at 20°C–25°C (see manufacturer’s instruction for using the Cytospin).</li>
<li>Let the slides air-dry at 20°C–25°C for at least 15 h and then perform immunofluorescence.</li>
</ol>
</li>
<li>Western blotting, mass spectrometry, DNA extraction:
<ol type="i">
<li>Collect the samples in 15 mL conical tubes: collect 500,000 micronuclei per protein sample for western blotting; 2–3,000,000 micronuclei per protein sample for proteomic analysis; 500,000–2,000,000 micronuclei per DNA sample.</li>
<li>Add BSA (final concentration 0.03%) and NP-40 (final concentration 0.1%) to the samples.</li>
<li>Centrifuge in a JS-5.3 swinging bucket rotor at max speed for 20 min at 4°C.</li>
<li>Discard the supernatant carefully and keep the pellet which should be visible.
<div class="critical">
<span class="critical-title">Critical:</span> Do not invert the tubes or use aspiration to discard the supernatant after centrifugation. Use pipettes only and avoid disturbing the pellet.</div>
</li>
<li>Resuspend the pellet in 500 μL ice-cold supplemented PBS 1× buffer and transfer into a 1.5 mL low-binding tube.</li>
<li>Centrifuge at max speed for 20 min at 4°C.</li>
<li>Discard the supernatant carefully and keep the pellet.
<div class="critical">
<span class="critical-title">Critical:</span> If the pellet is not visible, do not discard the supernatant and add BSA. Then, Centrifuge the sample again for 20 min (<a href="#troubleshooting">troubleshooting</a>).</div>
</li>
<li>Store the pellets at −80°C until performing the functional assay.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Functional assays can be performed 1 day to several months after storing the samples at −80°C, or cytospining on a microscopy slide.</div>
<h3 id="sec3.9">Functional assay example: immunoblotting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>In the next steps, the quantitative immunoblotting of the sorted micronuclei is described in detail.</p>
<ol start="38">
<li>Thaw micronuclei samples on ice.</li>
<li>Lyse samples in 1× Laemmli buffer at 2.5 × 10<sup>6</sup> micronuclei/μL.</li>
<li>Denature the lysates at 95°C for 5 min.</li>
<li>Sonicate the samples using a Bioruptor 300 on the high intensity setting for 8 cycles 30 s ON/30 s OFF.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If sonication is not available the DNA can be sheared using a 1 mL syringe until the lysate is no longer viscous.</div>
<ol start="42">
<li>Lysate from 0.5 × 10<sup>6</sup> micronuclei can be loaded in each lane of a polyacrylamide gel, resolved by SDS-PAGE and transferred to nitrocellulose membranes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We omit protein quantification after lysis and sonication to avoid loss of sample.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Protein extraction followed by BCA (bicinchoninic acid) assay can be performed to quantify the protein concentration. However, the BCA assay is not very sensitive and requires high volumes of micronuclei samples to detect proteins.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend using a pre-cast gradient 8%–16% gel to maximize the separation of proteins in order to detect the highest number of immunoblotting targets on the membrane.</div>
<ol start="43">
<li>Block the membranes in Odyssey blocking buffer in TBS (for Li-COR immunoblotting).</li>
<li>Dilute primary antibodies in blocking buffer supplemented with 0.2% Tween and incubated with membranes for at least 15 h at 4°C.</li>
<li>Wash the membranes in TBS supplemented with Tween 0.5% (TBS-T) 3 × 5 min.</li>
<li>Use secondary antibodies (Alexa Fluor 680 and 800) at 1:10,000 dilutions in blocking buffer supplemented with 0.2% Tween and incubated with membranes for 1 h at 20°C–25°C.</li>
<li>Wash the membranes in TBS supplemented with Tween 0.5% (TBS-T) 3 × 5 min.</li>
<li>Wash the membranes one additional time in TBS buffer without Tween for 5 min.</li>
<li>Dry the membranes before imaging.</li>
<li>Measure fluorescence using an infrared imaging scanner (Odyssey; LI-COR) according to the manufacturer’s instructions.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Non-quantitative immunoblotting can be performed. Membranes are blocked in TBS-T and milk 5% solution. The primary antibodies and the horseradish-peroxidase-conjugated secondary antibodies are diluted in TBS-T. Imaging of the membranes is performed using enhanced chemiluminescence.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Sucrose gradient centrifugation should result in a near homogeneous population of micronuclei with limited contamination from primary nuclei and cell debris. During flow cytometry sorting, gate 1 (see <a href="#fig1">Figure 1</a>A) should contain 50%–80% of the total events detected by the flow cytometry sorter. Then, if the purification of ruptured and intact micronuclei is desired, the intact micronuclei in gate 2 should represent 15%–30% of the total micronuclei (gate 1) and the ruptured micronuclei (gate 3) should represent 5%–10% of the gate 1 events (see <a href="#fig1">Figure 1</a>B).</p>
<p>As an example of a successful experiment, if 5 × 10<sup>8</sup> cells are collected to prepare samples to sort, the average yield of ruptured micronuclei to expect after sorting is between 500,000 and 1,000,000, with a sorting duration of about 2–4 h.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p><a href="#tbl1">Table 1</a> is necessary to determine the volumes of washing mediums, medium supplemented with cytochalasin B, lysis buffer, and sucrose buffer needed to process the cells based on the number of cells collected after trypsinization. In addition, this table helps to establish the number of sucrose density gradients to prepare in order to process all the lysed cells.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Yield of micronuclei is limited to the number of cells that can be cultured and collected to prepare the samples. If the cell line selected to prepare the samples cannot be expanded easily (for example, if the drug treatment to induce micronuclei limits or stops cell growth), the number of cells to culture has to be extended to collect the required number of micronuclei for further functional assays. Alternatively, the number of purification experiments can be multiplied to reach the desired number of micronuclei. Micronuclear envelopes are likely damaged during cell lysis which can lead to the leakage of some small proteins (such as NLS tagged with GFP or mCherry fluorescent markers). Hence, some proteins may be lost from micronuclei during purification.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>When verifying the efficiency of the cell lysis by microscopy, the primary nuclei and micronuclei seem to still be encapsulated by the cell membrane.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>The choice and final concentration of detergent (0.1% NP-40) in the lysis buffer has been established for the cell lines HEK293T and HeLa. If you are adapting this protocol to another cell line, it is possible that the concentration of NP-40 is too low, or that another detergent, such as Triton X-100, will be more appropriate. You can try increasing concentrations of NP-40, or of another detergent, to establish the right conditions for the cells lines you are using.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>When verifying the efficiency of cell lysis by microscopy, the micronuclei are not detached from the primary nuclei.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>This problem can arise when the Dounce homogenization was unsuccessful. Repeated Dounce homogenization may improve micronuclei separation. Alternatively, if using cell lines different from HEK293T or HeLa, the choice or final concentration of detergent in the lysis buffer can be inappropriate (see <a href="#sec7.1">problem 1</a>). Omitting the Cytochalasin B treatment (steps 15 and 16) can also cause the micronuclei to fail to separate from the primary nuclei.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The Fractions collected after the centrifugation of the sucrose density gradient do not contain the expected particles (i.e., debris in Fraction 1, micronuclei in Fraction 2, and primary nuclei in Fraction 3).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>The Fraction volumes described in the <a href="#step-by-step-method-details">step-by-step method details</a> section step 28 are optimal for samples prepared in HEK293T or HeLa cells. If another parental cell line is used, the volumes of each Fractions should be adapted as followed:</p>
<p>Collect Fractions of 1 mL in 1.5 mL tubes starting from the top of the sucrose density gradient until the 15<sup>th</sup> mL (Fraction 1 = upper 1 mL; Fraction 2 = following 1 mL; Fraction 3 = following 1 mL; etc.).</p>
<p>Collect 50 μL of each Fraction and mix it to an equal volume of PBS 1× + 0.5 μg/mL of DAPI.</p>
<p>Examine the samples under a fluorescence microscope. Identify Fractions that contain micronuclei but do not contain primary nuclei or cell debris. These Fractions can be pooled and processed as Fraction 2 described in the <a href="#step-by-step-method-details">step-by-step method details</a> section step 28.b.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>The samples sorted by flow cytometry are not pure and contaminated by events from other gates.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>There are several different “sort precision modes” of sorting by flow cytometry on the BD FACSAria III Cell Sorter. To ensure the purity of the micronuclei samples and avoid the contamination by unwanted events (such as primary nuclei or micronuclei from different gates), you should use the “4-way purity” mode. This precision mode is not frequently used for cell sorting, so the setup has to be verified before every sort experiment.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>The flow cytometer does not detect the micronuclei population of events in the gate graph [y-axis = FSC-A – x-axis = DAPI-A] (see <a href="#fig1">Figure 1</a>).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Classical settings for cell sorting on the BD FACSAria III Cell Sorter set a threshold of detection for the FSC of 1000 V and above on the graphical scale. This means that all the events detected with a FSC below 1000 will not be displayed on the graph. Since, the majority of micronuclei are detected within a range of 300–1000 on the FSC channel, the threshold needs to be lowered to display the micronuclei on the graph.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>After centrifugation of the flow cytometry samples, the micronuclei pellets are not visible or very small.</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Even a large number of micronuclei may not give rise to a large pellet after centrifugation.</p>
<p>If a pellet is not visible, keep the supernatant and increase the concentration of BSA in the sample (maximum 0.3%) and/or NP-40 (maximum 0.1%). Mix the samples by inverting several times (do not vortex) and centrifuge again. Be careful not to exceed 0.1% NP-40 in the sample as it will induce lysis of the micronuclei.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>No signal is detected by immunoblotting in the micronuclei sorted samples.</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Collecting enough micronuclei for one or several replicates of protein samples is challenging, especially when isolating ruptured micronuclei. The ideal number of micronuclei for one sample to load on a polyacrylamide gel for immunoblotting is 500,000 micronuclei. If the number of micronuclei collected after one experiment is not sufficient, you can pellet the sample, store it at −80°C and pool samples until the desired number of micronuclei are isolated. We recommend using primary nuclei as controls for verifying the efficiency of the immunoblotting experiment (50,000–100,000 primary nuclei per lane). Whole cell extract can also be used as loading control for this experiment (10,000–100,000 cells per lane).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Abend, M., Frombeck, S., Baaske, C., and van Beuningen, D. (1996). Isolation of micronuclei: high yield by sucrose gradient versus maximum purity by cell sorting. Mutat. Res. Environ. Mutagen. Relat. Sub. <i>360</i>, 23-28. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref1">View at publisher</a></p>
<p id="bib2">Bartsch, K., Knittler, K., Borowski, C., Rudnik, S., Damme, M., Aden, K., Spehlmann, M.E., Frey, N., Saftig, P., Chalaris, A., and Rabe, B. (2017). Absence of RNase H2 triggers generation of immunogenic micronuclei removed by autophagy. Hum. Mol. Genet. <i>26</i>, 3960-3972. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref2">View at publisher</a></p>
<p id="bib3">Cummings, D.J. (1972). Isolation and partial characterization of macro- and micronuclei from paramecium aurelia. J. Cell Biol. <i>53</i>, 105-115. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref3">View at publisher</a></p>
<p id="bib4">Dou, Z., Ghosh, K., Vizioli, M.G., Zhu, J., Sen, P., Wangensteen, K.J., Simithy, J., Lan, Y., Lin, Y., Zhou, Z., et al. (2017). Cytoplasmic chromatin triggers inflammation in senescence and cancer. Nature <i>550</i>, 402-406. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref4">View at publisher</a></p>
<p id="bib5">Halfmann, C.T., Sears, R.M., Katiyar, A., Busselman, B.W., Aman, L.K., Zhang, Q., O’Bryan, C.S., Angelini, T.E., Lele, T.P., and Roux, K.J. (2019). Repair of nuclear ruptures requires barrier-to-autointegration factor. J. Cell Biol. <i>218</i>, 2136-2149. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref5">View at publisher</a></p>
<p id="bib6">Harding, S.M., Benci, J.L., Irianto, J., Discher, D.E., Minn, A.J., and Greenberg, R.A. (2017). Mitotic progression following DNA damage enables pattern recognition within micronuclei. Nature <i>548</i>, 466-470. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref6">View at publisher</a></p>
<p id="bib7">Liu, S., Kwon, M., Mannino, M., Yang, N., Renda, F., Khodjakov, A., and Pellman, D. (2018). Nuclear envelope assembly defects link mitotic errors to chromothripsis. Nature <i>561</i>, 551-555. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref7">View at publisher</a></p>
<p id="bib8">Ly, P., Teitz, L.S., Kim, D.H., Shoshani, O., Skaletsky, H., Fachinetti, D., Page, D.C., and Cleveland, D.W. (2017). Selective Y centromere inactivation triggers chromosome shattering in micronuclei and repair by non-homologous end joining. Nat. Cell Biol. <i>19</i>, 68-75. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref8">View at publisher</a></p>
<p id="bib9">Mackenzie, K.J., Carroll, P., Martin, C.-A., Murina, O., Fluteau, A., Simpson, D.J., Olova, N., Sutcliffe, H., Rainger, J.K., Leitch, A., et al. (2017). cGAS surveillance of micronuclei links genome instability to innate immunity. Nature <i>548</i>, 461-465. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref9">View at publisher</a></p>
<p id="bib10">Mohr, L., Toufektchan, E., von Morgen, P., Chu, K., Kapoor, A., and Maciejowski, J. (2021). ER-directed TREX1 limits cGAS activation at micronuclei. Mol. Cell. <a class="external-link" href="https://doi.org/10.1016/j.molcel.2020.12.037">https://doi.org/10.1016/j.molcel.2020.12.037</a></p>
<p id="bib11">Nüsse, M. and Kramer, J. (1984). Flow cytometric analysis of micronuclei found in cells after irradiation. Cytometry <i>5</i>, 20-25. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref11">View at publisher</a></p>
<p id="bib12">Shimizu, N., Kanda, T., and Wahl, G.M. (1996). Selective capture of acentric fragments by micronuclei provides a rapid method for purifying extrachromosomally amplified DNA. Nat. Genet. <i>12</i>, 65. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref12">View at publisher</a></p>
<p id="bib13">Yang, H., Wang, H., Ren, J., Chen, Q., and Chen, Z.J. (2017). cGAS is essential for cellular senescence. Proc. Natl. Acad. Sci. U S A <i>114</i>, E4612-E4620. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref13">View at publisher</a></p>
<p id="bib14">Young, A.M., Gunn, A.L., and Hatch, E.M. (2020). BAF facilitates interphase nuclear membrane repair through recruitment of nuclear transmembrane proteins. MBoC <i>31</i>, 1551-1560. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00085-X/sref14">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, John Maciejowski (<a href="mailto:maciejoj@mskcc.org">maciejoj@mskcc.org</a>).</p>
<h4>Materials Availability</h4>
<p>Cell lines generated in this study and listed in the <a href="#key-resources-table">key resources table</a> are available from John Maciejowski. Plasmids used in this study and listed in the <a href="#key-resources-table">key resources table</a> are available on Addgene (<a href="https://www.addgene.org">https://www.addgene.org</a>).</p>
<h4>Data and Code Availability</h4>
<p>The datasets and original source data generated during this study are available at Mendeley Data (<a href="https://doi.org/10.17632/d8p53cv3ry.1">https://doi.org/10.17632/d8p53cv3ry.1</a>).</p>
<h3>Acknowledgments</h3>
<p>Work in the J.M. lab is supported by the <a href="https://doi.org/10.13039/100000054">NCI</a> (R00CA212290), the <a href="https://doi.org/10.13039/100000875">Pew Charitable Trusts</a>, the Starr Cancer Consortium, the Geoffrey Beene and Ludwig Centers at <a href="https://doi.org/10.13039/100007052">MSKCC</a>, and MSKCC core grant P30-CA008748.</p>
<h3>Author Contributions</h3>
<p>E.T. developed the micronucleus purification method and wrote the manuscript. J.M. supervised method development and edited the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>J.M. has received consulting fees from Ono Pharmaceutical Co. His spouse is an employee of and has equity in Bristol Myers Squibb.</p>
</section>
</article>