<article>
<section>
<h1 id="header">Delivering defined amounts of purified protein with high precision into living cells</h1>
<p><time datetime="2021-01-22">Published: January 22, 2021</time></p>
<p>Kristen Skruber,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Tracy-Ann Read,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Eric A. Vitriol<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Anatomy and Cell Biology, University of Florida, College of Medicine, Gainesville, FL 32610, USA</p>
<p id="aff2"><sup>2</sup>Center for Translational Research in Neurodegenerative Disease, University of Florida, College of Medicine, Gainesville, FL 32610, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:skruberk@ufl.edu">skruberk@ufl.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:evitriol@ufl.edu">evitriol@ufl.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100272">10.1016/j.xpro.2020.100272</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we detail a protocol using electroporation to precisely deliver defined amounts of purified protein into CAD cells. This method allows one million cells to be electroporated with protein simultaneously, with high delivery efficiency and low cell death. Further, by circumventing the normal biosynthetic pathway, proteins can be studied without the complication of post-translational modifications and before a transcriptional response can be initiated. This protocol will be useful for any researcher who is interested in protein concentration-dependent cellular phenotypes.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib10">Skruber et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Precise amounts of purified protein can be delivered into cells by electroporation</li>
<li>Delivered protein is proportional to bath concentration in electroporation reaction</li>
<li>Protein retains functionality after electroporation</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/413-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Preparing cells for electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>Cells chosen for protein delivery should be in log-phase growth and at a low passage number. If cells are maintained under selection by an antibiotic, such as puromycin, the antibiotic should ideally be removed 24 h prior to the electroporation to increase cell viability. Cells should also be routinely tested for mycoplasma infection, which could interfere with electroporation. In this protocol we use cath.a differentiated (CAD) cells (<a href="#bib6">Qi et al., 1997</a>) which can be transfected with &gt;99% electroporation efficiency (percentage of cells which uptake protein) and &lt;10% cell death.</p>
<h3 id="sec1.2">Determining optimal electroporation parameters</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>Electroporation conditions vary by cell type and need to be optimized prior to performing experiments. We recommend first using an inert fluorescent tracer such as fluorescently labeled dextran (see <a href="#fig1">Figure 1</a>) to determine optimal conditions for cell viability and electroporation efficiency. Options for measuring cell viability include colorimetric assays such trypan blue exclusion, or MTS (3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium). Optimization of electroporation parameters is described in detail below.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/413-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Quantifying delivery of electroporated protein</div>
<p>(A) Quantifying protein delivery using imaging (step 4). On the left are representative images of cells electroporated with 10 kDa dextran-FITC. The dextran-FITC bath concentration in the electroporation chamber is indicated. Images are scaled identically and pseudocolored based on the included lookup table to convey relative fluorescent intensities. On the right is a box-and-whisker plot depicting the quantification of mean cellular 10 kDa dextran-FITC fluorescence as a function of its bath concentration in the electroporation chamber. Individual data points are plotted along with the mean and 95% confidence intervals. The R<sup>2</sup> value is the linear fit through the mean value of fluorescence intensity for each bath concentration. N = 42, 59, 53, 69 for 20 μM, 50 μM, 100 μM, 200 μM, respectively.</p>
<p>(B) Quantifying protein delivery with a co-electroporated tracer using imaging (step 4). On the right are representative images of cells co-electroporated with Thymosin β4-647 and 10 kDa dextran-FITC. The bath concentration in the electroporation chamber is indicated. Images are scaled identically and pseudocolored based on the included lookup table to convey relative fluorescent intensities. On the right is a box-and-whisker plot depicting the quantification of mean cellular intensities of Thymosin β4-647 and 10 kDa dextran-FITC fluorescence as a function of bath concentration in the electroporation chamber. Individual data points are plotted along with the mean and 95% confidence intervals. The R<sup>2</sup> value is the linear fit through the mean value of fluorescence intensity for each bath concentration. N = 77, 69, 74, 83 for 25 μM, 50 μM, 100 μM, 150 μM, respectively. Note that co-electroporation efficiency is comparable to (A) the electroporation of a single protein.</p>
<p>(C) Quantification of protein delivery by fluorometry (step 5). Three biological replicates of cells were transfected with the indicated bath concentration of Thymosin β4-647. Immediately after electroporation cells were washed, collected, and raw fluorescence units are plotted as a function of concentration. The R<sup>2</sup> value is the linear fit through the mean value of fluorescence intensity for each bath concentration.</p>
<p>(D) Quantification of protein delivery by western blot (step 6). Western blot and quantification of expression levels of profilin 1 in control and PFN1 KO cells after electroporation with the designated concentration of purified profilin 1. The concentrations reflect the bath concentration of profilin 1 in the electroporation chamber. Profilin 1 expression was normalized to GAPDH. Four biological replicates were used for each condition. Individual data points are plotted along with the mean and 95% confidence intervals. Scale bars, 10 μm. (A) and (D) are reprinted with permission from <a href="#bib10">Skruber et al. (2020)</a>.</p>
</figcaption>
</figure>
<h3 id="sec1.3">Preparing protein for cellular delivery</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Successful delivery of protein through electroporation depends on the protein preparation. The protein should be of high quality, meaning that it is of high purity (&gt;99.5%), with minimal contamination, and maintained at conditions empirically determined to maintain protein activity. Protein quality should be assessed through a minimum of two methods. Analyze protein samples with SDS-PAGE and Coomassie Brilliant Blue or Silver stain to confirm that the protein runs as a single band and thus has not degraded. Further validation by amino acid analysis (LC-MS/MS) will eliminate the possibility of a contaminant that co-migrates with the protein of interest and will also accurately determine concentration.</p>
<p>Protein concentration must be validated for each protein after purification. The most common method is measuring absorbance with a UV-VIS spectrophotometer. Measuring absorbance at 280 nm with a Nanodrop provides a good estimate of the protein concentration. If that equipment is not available, Bicinchoninic acid assay (BCA) is a highly sensitive assay that works over a large range of amino acid compositions. Protein samples analyzed through BCA are compared against a protein standard such as bovine serum albumin (BSA) (<a href="#tbl1">Table 1</a>). However, DTT, chelating agents like EDTA, or high detergent concentrations (SDS &gt;1.25% or Triton X-100 &gt;0.25%) in the buffer are unsuitable for this method. Alternatives to BCA include the Lowry and Bradford assays. BCA and Lowry assays both work on the principle of copper-protein chelation and are thus sensitive to buffers with chelating agents present such as EDTA. The Lowry assay is also sensitive to the presence of detergents, but it has a higher tolerance for reducing agents than BCA. The Bradford assay trades compatibility with detergents, reducing agents, and chelating agents for reduced sensitivity over a large range of amino acid compositions. See <a href="#troubleshooting">Troubleshooting</a> for additional details.</p>
<table id="tbl1">
<caption>Table 1. Example BCA data used to determine the concentration of Thymosin β4</caption>
<thead>
<tr>
<th>Bovine Serum Albumin mg/mL</th>
<th>Trial 1</th>
<th>Trial 2</th>
<th>Trial 3</th>
</tr>
</thead>
<tbody>
<tr>
<td>2.0</td>
<td>0.841</td>
<td>0.836</td>
<td>0.805</td>
</tr>
<tr>
<td>1.0</td>
<td>0.498</td>
<td>0.465</td>
<td>0.476</td>
</tr>
<tr>
<td>0.5</td>
<td>0.317</td>
<td>0.292</td>
<td>0.324</td>
</tr>
<tr>
<td>0.25</td>
<td>0.209</td>
<td>0.195</td>
<td>0.205</td>
</tr>
<tr>
<td>0.12</td>
<td>0.156</td>
<td>0.151</td>
<td>0.155</td>
</tr>
<tr>
<td>0.0</td>
<td>0.097</td>
<td>0.096</td>
<td>0.101</td>
</tr>
<tr>
<td>Thymosin β4</td>
<td>0.426</td>
<td>0.476</td>
<td>0.561</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> The protein buffer is an important variable in the electroporation reaction, as high salt concentrations can induce arcing and cause cell death. Protein buffers suitable for electroporation are low salt. For instance phosphate buffered saline (PBS) and HEPES buffer have proven suitable electroporation buffers (<a href="#bib1">Alex et al., 2019</a>). Electroporation with protein buffer and an inert fluorescent tracer should be performed prior to protein delivery. If cell viability or electroporation efficiency is low, the protein should be dialyzed into a buffer more suitable for electroporation. See <a href="#troubleshooting">Troubleshooting</a> for more detail.</div>
<h3 id="sec1.4">Preparing coverslips</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 26 h</div>
<p>For imaging, coverslips need to be cleaned and coated with appropriate reagents to help the cells adhere to them. Below we detail coating the coverslips with laminin as in (<a href="#bib10">Skruber et al., 2020</a>); however, the specific reagents (ex. fibronectin, poly-d-lysine) and incubation times/temperatures will vary depending on the cell type used.</p>
<ol>
<li>Generating “squeaky clean” imaging-ready coverslips.
<ol type="a">
<li>Place 1,000 coverslips directly in a bath sonicator. Alternately, coverslips can be placed in a beaker which can then be filled and placed in a sonicator.</li>
<li>Add diH<sub>2</sub>O + Alconox lab detergent to fill line.</li>
<li>Sonicate for 1 h.</li>
<li>Remove detergent, wash with ddH<sub>2</sub>O.</li>
<li>Repeat steps c and d 10×.</li>
<li>Add 75% EtOH to fill line, sonicate for 1 h.</li>
<li>Remove 75% EtOH.</li>
<li>Store coverslips in 75% EtOH in sealed container.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> As long as the container is opened under only sterile conditions, coverslips can be stored for up to 2 years.</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h, 15 min</div>
<ol start="2">
<li>Coating coverslips with laminin.
<ol type="a">
<li>Remove (6) coverslips from 75% EtOH under sterile conditions and place one into each well of a 6-well sterile plate filled with ∼2 mL DPBS. Coverslips will float until EtOH evaporates.</li>
<li>Once coverslips sink to the bottom, aspirate DPBS and wash with 1–2 mL fresh DPBS. Repeat 2×.</li>
<li>Add 990 μL of DPBS/well.</li>
<li>Thaw laminin (Sigma-Aldrich, 10 μg/mL) on ice, briefly vortex, and then pulse centrifuge to collect at the bottom of the tube.</li>
<li>Dispense 10 μL of laminin per well and gently rotate plate to mix. Do not leave laminin for an extended time at 20°C–25°C.</li>
<li>Place the 6-well plate into an incubator at 37°C, 5% CO<sub>2</sub> for 2 h.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Profilin1 C56B8 rabbit mAb</td>
<td>Cell Signaling</td>
<td>#3246</td>
</tr>
<tr>
<td>Rabbit anti-GAPDH (2118)</td>
<td>Cell Signaling</td>
<td>Cat# 2118S</td>
</tr>
<tr>
<td>Goat anti-rabbit Alexa Fluor 680</td>
<td>Li-Cor</td>
<td>P/N: 926-68021</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Thymosin β4-647</td>
<td>(<a href="#bib11">Yarmola et al., 2000</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Profilin-1</td>
<td>(<a href="#bib4">Henty-Ridilla et al., 2017</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>16% Paraformaldehyde</td>
<td>Electron Microscopy Sciences</td>
<td>CAT# 15710</td>
</tr>
<tr>
<td>cOmplete Protease Inhibitor Cocktail</td>
<td>Roche</td>
<td>Cat#11697498001</td>
</tr>
<tr>
<td>Laminin from Engelbreth-Holm-Swarm murine sarcoma basement membrane</td>
<td>Sigma-Aldrich</td>
<td>Cat# L2020-1MG</td>
</tr>
<tr>
<td>Alexa Fluor Phalloidin 568</td>
<td>Thermo Fisher</td>
<td>A12380</td>
</tr>
<tr>
<td>Coomassie brilliant blue</td>
<td>Millipore Sigma</td>
<td>B8522</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Sigma-Aldrich</td>
<td>D2650</td>
</tr>
<tr>
<td>Dextran, fluorescein 10,000 mw anionic</td>
<td>Thermo Fisher</td>
<td>D1821</td>
</tr>
<tr>
<td>Dextran, fluorescein 40,000 mw anionic</td>
<td>Thermo Fisher</td>
<td>D1845</td>
</tr>
<tr>
<td>Orange G</td>
<td>Sigma-Aldrich</td>
<td>O7252</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Pierce bicinchoninic acid (BCA) assay</td>
<td>Thermo Scientific</td>
<td>Cat#232225</td>
</tr>
<tr>
<td>Qubit Protein Assay Kit</td>
<td>Thermo Fisher</td>
<td>Q33212</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Cath.a differentiated (CAD) cells</td>
<td>ATCC</td>
<td>CRL-11179</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Fiji ImageJ</td>
<td>NIH</td>
<td><a href="https://imagej.net/Fiji">https://imagej.net/Fiji</a></td>
</tr>
<tr>
<td>“SubtractMeasuredBackround” Macro</td>
<td>NIH</td>
<td><a href="https://imagej.nih.gov/ij/macros/SubractMeasuredBackground.txt">https://imagej.nih.gov/ij/macros/SubractMeasuredBackground.txt</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Neon transfection system</td>
<td>Invitrogen</td>
<td>MPK5000S</td>
</tr>
<tr>
<td>XCell II blot module</td>
<td>Invitrogen</td>
<td>EI9051</td>
</tr>
<tr>
<td>XCell SureLock mini-cell electrophoresis</td>
<td>Invitrogen</td>
<td>EI0001</td>
</tr>
<tr>
<td>Qubit 4 fluorometer</td>
<td>Thermo Fisher</td>
<td>Q33238</td>
</tr>
<tr>
<td>Fisherbrand Superfrost microscope slides</td>
<td>Fisher Scientific</td>
<td>12-550-15</td>
</tr>
<tr>
<td>Prolong Diamond antifade mountant</td>
<td>Invitrogen</td>
<td>REF#P36961</td>
</tr>
<tr>
<td>4%–20% Tris-glycine gels</td>
<td>Invitrogen</td>
<td>Cat#XP04200BOX</td>
</tr>
<tr>
<td>Amicon Ultra centrifugal filter</td>
<td>Millipore Sigma</td>
<td>UFC203024</td>
</tr>
<tr>
<td>Deutsche Spiegelglas coverslips 25 mm</td>
<td>Carolina</td>
<td>Item #633057</td>
</tr>
<tr>
<td>DMEM/F-12 with HEPES</td>
<td>Thermo Fisher</td>
<td>Catalog # 11330032</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>For electroporation we use the Neon transfection system. The Neon uses a tip-based capillary probe to maximize viability and electroporation efficiency (<a href="#bib5">Kim et al., 2008</a>). The Neon is also attractive for protein electroporation since the reaction volume is only 10 μL. Systems with larger volume electroporation chambers will require significantly more purified protein. Alternative systems to the Neon include Lonza’s Amaxa/Amaxa 4D or Bio-Rad’s Gene Pulser Xcell.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Protein electroporation</h3>
<p>Here, we provide a protocol for electroporation of protein into cells. For establishing parameters that yield high electroporation efficiency, we recommend first optimizing the cell line for electroporation by using an inert fluorescent standard, such as dextran. After electroporation conditions are optimized with the standard, the protein of interest can then be introduced by co-loading with dextran (step 1, <a href="#fig1">Figure 1</a>). Labeling the protein of interest highly increases the possibilities for quantification of its delivery (steps 4–6), however labeling could interfere with protein function. If the protein is unlabeled, electroporation efficiency can be established by performing labeling the cells with antibodies against the protein of interest. For CAD cells, a semi-confluent 10 cm dish (approximate 1×10<sup>6</sup> cells) yields enough cells for the equivalent of two electroporations or a 10 μL suspension of Total Cell Volume (<a href="#tbl2">Table 2</a>). For each condition, “Total Volume” reflects the amount needed for two electroporations. This technique minimizes electroporation variability. We found it best to maintain the same volume of electroporation buffer (buffer R) across different protein concentrations (see <a href="#tbl2">Table 2</a> for example protocol).</p>
<table id="tbl2">
<caption>Table 2. Example electroporation protocol for unlabeled protein with fluorescent tracer</caption>
<thead>
<tr>
<th>Stock concentration of protein (μM)</th>
<th>Desired final concentration of protein (μM)</th>
<th>Volume of protein (μL)</th>
<th>Volume of R buffer (μL)</th>
<th>Fluorescent tracer volume (μL)</th>
<th>Total cell suspension volume (μL)</th>
<th>Total electroporation volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>0 (buffer only)</td>
<td>0</td>
<td rowspan="6">6.25</td>
<td rowspan="6">7.25</td>
<td rowspan="6">1.5</td>
<td rowspan="6">10</td>
<td rowspan="6">25</td>
</tr>
<tr>
<td>200</td>
<td>50</td>
</tr>
<tr>
<td>40</td>
<td>10</td>
</tr>
<tr>
<td>80</td>
<td>20</td>
</tr>
<tr>
<td>200</td>
<td>50</td>
</tr>
<tr>
<td>400</td>
<td>100</td>
</tr>
</tbody>
</table>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h, 30 min</div>
<ol>
<li>Transfecting cells with fluorescent dextran or protein
<ol type="a">
<li>Take cell culture media and 6-well plate with laminin-coated coverslips out of the incubator. Under sterile conditions, remove the DPBS and add 2 mL of pre-warmed media to each well of a 6 well plate. Place the plate back in the incubator.</li>
<li>Thaw dextran or protein on ice in the dark (e.g., covered with foil). If delivering protein, first preclarify by centrifugation (&gt;100,000 × <i>g</i>, 20 min) to remove aggregates. For an example of co-electroporation with protein and dextran see <a href="#tbl2">Table 2</a>.</li>
<li>Add the required buffer volumes (see <a href="#tbl2">Table 2</a>.) in sterile 0.5 mL Eppendorf tubes.</li>
<li>Harvest cells from (3) 60%–80% confluent 10 cm dishes by trypsinization (1.5 mL, 500 μL/dish, 3 min).
<div class="note">
<span class="note-title">Note:</span> Alternately, cells could be harvested with a cell scraper.</div>
</li>
<li>Inhibit trypsin with (6 mL, 2 mL/dish) 8% FBS in DMEM/F12 media.</li>
<li>Combine harvested cells into one 15 mL falcon tube and gently centrifuge (150 × <i>g</i>, 3 min) to remove trypsin.</li>
<li>Wash pellet in DPBS (3 mL) being careful to at least dislodge or fully resuspend the pellet upon DPBS addition.</li>
<li>Centrifuge as before, add DPBS, and then remove DPBS. Repeat 1×.</li>
<li>Remove 6-well plate from incubator.</li>
<li>Add dextran/protein to total electroporation volume.</li>
<li>Remove DPBS then resuspend pellet gently in the minimal amount of electroporation buffer R. For example, for six electroporations with 10 μL of cell suspension per electroporation, the minimal amount needed would be 65 μL to provide enough volume to minimize bubble formation. We recommend adding an additional electroporation’s worth of cell suspension (10 μL) in case of arcing, an additional 10 μL for counting, and an additional 10 μL for the non-electroporation control. Gently mix the cell suspension so cells are evenly distributed.</li>
<li>Use 10 μL of cell volume to count cells, cells should be in the range of 3–5 × 10<sup>6</sup> cells/ mL. Dilute in buffer R if cell count is too high.</li>
<li>Add 10 μL of total cell suspension volume to each Eppendorf tube using a 20 μL pipette tip. Stir the solution once gently with a pipette tip and then mix by pipetting up and down once, taking care to avoid bubble formation.</li>
<li>Draw the solution into one 10 μL Neon Pipette Tip (Thermo Fisher). Tip should then be inverted to visualize any air bubbles. If bubbles have formed, dispense suspension back in the Eppendorf tube and redraw. If bubbles still remain, pulse centrifuge and resuspend.</li>
<li>Use one single 1,400 v 20 ms pulse for electroporation. The pipette tip contents are dispensed into the 6 well plate with coverslips and media following completion of the pulse.</li>
<li>Use one 10 μL Neon Pipette Tip (Thermo Fisher) twice per condition. Change the tip between concentrations.
<div class="critical">
<span class="critical-title">Critical:</span> The time cells and delivery materials are suspended in buffer should be minimized; work quickly while taking care to ensure that bubbles do not form. If bubbles are not eliminated, the electroporation reaction could arc resulting in high cell death.</div>
</li>
<li>Place 6 well plate in an incubator (37°C, 5% CO<sub>2</sub>) for 2 h to allow cells to adhere.</li>
<li>Plate a non-electroporated control from the total cell suspension volume from step k (30 mm dish) simultaneously with electroporated samples.
<div class="timing">
<span class="timing-title">Timing:</span> 48 h, 30 min</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Preparing slides for imaging
<ol type="a">
<li>Prepare 6 mL of 4% PFA in PBS from 16% paraformaldehyde (1.5 mL of cold 16% PFA is added to 4.5 mL of PBS).</li>
<li>After 2 h, remove cells from incubator and confirm adherence by light microscopy.</li>
<li>For fixation, quickly aspirate the media and add 1 mL of 4% PFA/well. Fix cells for 10 min at 20°C–25°C in the dark.</li>
<li>Remove the PFA and safely dispose of it.</li>
<li>Dispense PBS liberally into each well to a volume sufficient to cover the cells. Remove the PBS and gently add more to each well to remove traces of PFA. Repeat 1×.</li>
<li>Leave PBS in wells for 5 min in the dark.</li>
<li>Label and prepare slide (25 × 75 mm, Superfrost). Invert a bottle of ProLong Diamond Antifade mounting media and dispense one large drop.</li>
<li>While tilting the 6 well plate, use forceps to carefully grab each coverslip. While keeping track of which face the cells are adhered to, gently drain the coverslip of excess PBS through capillary action by touching the edge of the coverslip close to a KimWipe or absorbent towel, being careful not to touch the face of the coverslip with cells. Place each coverslip on the mounting media so that cells are facing down toward the slide.</li>
<li>Keep slides in the dark and undisturbed for 48 h.</li>
<li>After 48 h, seal slides with a minimal amount of clear nail polish and allow the nail polish to dry completely before imaging.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Keep slides in the dark at 4°C for up to 2 months.</div>
<h3 id="sec3.2">Assessing delivery of purified proteins</h3>
<p>Two important measures of protein electroporation are the fraction of electroporated cells and the amount of protein delivered per cell. Below are three orthogonal options for assessing the delivery of purified protein into cells. For assessing the amount of delivered protein, we recommend that at least two different methods are used, including a method that quantifies delivery on a per cell basis, such as quantification by microscopy. We also recommend performing at least three biological replicates of the assays listed below to determine if protein delivery is consistent from day to day.</p>
<ol start="3">
<li>Determining electroporation efficiency. We used the EVOS XL digital inverted microscope equipped with the 20× 0.5 NA objective to measure efficiency. However, any microscope capable of transmitted light (ex. phase or DIC) and epifluorescence imaging should be sufficient. For these measurements, you want to use lower magnification (ex. 10× or 20×) to fit as many cells in a field of view as possible.
<ol type="a">
<li>Locate a field of view in the transmitted light channel, then focus and capture image.</li>
<li>Without changing the field of view, switch to the fluorescence channel and capture an image.</li>
<li>Repeat and save in tiff format.</li>
<li>Electroporation efficiency can be counted manually by counting the total number of cells (transmitted light channel) and dividing it by the number of fluorescent cells or quantified as described below using the freely available Fiji (ImageJ) software.</li>
<li>For the transmitted light channel, binarize each image (Process/Binary/Make Binary).
<ol type="i">
<li>Binarizing the image will often fragment cells. If this is the case, erode the binarized image (Process/Binary/Erode) until the cells appear whole.</li>
<li>Convert binarized image to mask (Process/Binary/Convert to mask).
<div class="note">
<span class="note-title">Note:</span> If thresholding the binarized transmitted light channel is difficult or cells are not captured, a plasma membrane stain such as CellMask HCS stain (Thermo Fisher) or Vybrant Di cell-labeling (Thermo Fisher) could alternately be used instead of the transmitted light channel to locate the cell edge. Lastly, cells could be circled by hand, however, this last option introduces the most bias.</div>
</li>
</ol>
</li>
<li>To measure electroporation efficiency, analyze particles (Analyze/Analyze Particles), under “Show” select “Outlines.” To ensure that cell debris is not counted as a cell, limit the size by selecting a threshold number for cell size (for example 10-Infinity). Count cells using the results generated in Analyze Particles for both the light and fluorescence channels.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Quantifying protein delivery using imaging. In this approach, electroporated cells are imaged on a microscope. The benefit to this method is that cell-to-cell variability of protein delivery is established in addition to mean changes of the entire population. We used the EVOS XL digital inverted microscope equipped with the 20× 0.5 NA objective to measure fluorescence intensity. However, any microscope capable of epifluorescence should be sufficient. For whole cell measurements, it is best to use a lower magnification objective with a large depth of focus.
<ol type="a">
<li>Locate cells with a range of fluorescence intensities.</li>
<li>Select settings for both light intensity and exposure time that provide good signal for both high and low fluorescence intensity cells. Ensure that highly fluorescent cells do not saturate the signal while retaining good signal-to-noise ratio for lower intensity cells. Since increasing either exposure time or light power will increase the signal, elect to increase exposure time over light power to reduce bleaching. Perform for both channels.</li>
<li>Locate cells suitable for single-cell measurements: center them in the field of view, focus, and then capture the image. Cells should not be touching other cells. Once established, do not change imaging settings.
<div class="note">
<span class="note-title">Note:</span> It is important that all imaging for each biological replicate is performed on the same day, using identical imaging parameters (light intensity, exposure time). This will minimize the potential that equipment variability (ex. light source) to affect a result.</div>
</li>
<li>For measurement of two probes, make sure that the stage is not moved when capturing both channels.</li>
<li>Capture images for 30–40 cells and save in tiff format.</li>
<li>Open Fiji software and import all images from one data set as an image sequence (File/Import/Image Sequence).</li>
<li>Create one maximum intensity Z-projection of the image sequence (Image/Stacks/Z-Project-Select: Max IP).</li>
<li>To measure background on a large number of images, select the maximum intensity Z-projection of the image sequence and locate an area devoid of cells or signal. To ensure the area is only background, adjust the brightness and contrast by reducing the maximum intensity (Image/Adjust/Brightness&amp;Contrast). Use the rectangle tool (located in the toolbar) to draw a small box. While holding down shift, identify at least one more area free of cells/signal to draw another box, preferably at a far distance from the first box. When done, two areas should be selected on the stack that are devoid of any signal to generate a representative value for the background. Alternately, background can be subtracted from individual images, one at a time by using the rectangle tool to draw a small box in an area devoid of cells. Measure the mean intensity of the background (Analyze/Measure) for each image and then manually subtract that value from the image (Process/Math/Subtract). If background is subtracted manually, proceed to step l.</li>
<li>Select the window for the image sequence and Shift + E (alternately, Edit/Selection/Restore Selection) to transfer the drawn boxes to the image sequence. This places all the boxes in the same location for each image in the sequence, and every box should represent an area devoid of cells.</li>
<li>Close the maximum intensity projection.</li>
<li>Download and then run the background subtraction macro (Plugins/Macros/Run/”File location”/Backgroundsubtract.txt) (For link see <a href="#key-resources-table">Key resources table</a>).
<div class="note">
<span class="note-title">Note:</span> For simultaneous measurement of two channels, background subtract each channel. To use one channel to aid in detection of the other (<a href="#fig1">Figure 1</a>B), use the brighter channel to locate and threshold the cell boundary. Transfer the selection from one channel to the other using Shift + E (alternately, Edit/Selection/Restore Selection) to transfer the drawn boxes to the image sequence.</div>
</li>
<li>In the “Set Measurements” (Analyze/Set Measurements) window select the following: Area, Integrated Density, Mean Gray Value, Median. If measuring two fluorescence channels, each channel should have identical area measurements per cell.
<div class="note">
<span class="note-title">Note:</span> There are multiple metrics available for fluorescence quantification. For quantifying the delivery of a fluorescent probe that diffusively localizes throughout the cytosol, we recommend using mean fluorescent intensity. Mean intensity is the sum pixel intensity divided by cell area and normalizes the measurement to account for any changes in area. It is a measure of the fluorescent probe concentration. Integrated density is a sum pixel intensity within the measured area and therefore measures the amount of fluorescent probe. Integrated density can alternately be calculated by multiplying the mean intensity by the area. Median pixel value can be used to display large variances in fluorescence distribution. Taking into consideration the localization of the probe as well as its distribution will help establish which metric will be the most informative.</div>
</li>
<li>Check a few of the images in the sequence to make sure the background was subtracted by drawing a box in an area devoid of cells. Take a measurement by Ctrl + M (alternately, Analyze/Measure) and checking the mean value. By selecting both an area for background and for a cell you can establish a signal: noise ratio.</li>
<li>To measure individual cells, threshold the signal (Image/Adjust/Threshold). Optimize the thresholding so that cells are selected. If background subtraction was done correctly, the signal-to-noise ratio is high, and the protein is diffusively distributed in the cytoplasm, then the entire cell should be outlined (see Alternatives). Once the cell is thresholded, use the wand tool (located in the toolbar) to select the cell and Ctrl + M (Analyze/Measure) to measure. Additionally, the ROI Manager (Analyze/Tools/ROI Manager) can be used to measure cells and save a record of the analysis for future use.
<div class="note">
<span class="note-title">Note:</span> If the protein is unlabeled, delivery can be assessed by performing immunostaining using an antibody that is specific for that protein. If the protein is endogenously expressed in the cells that are being electroporated, then electroporation should be assessed in cells were the gene encoding for the protein of interest is knocked out. Alternatively, an epitope tag could be added to the electroporated protein. As with any immunolabeling experiment, care should be taken to ensure that they antibodies are specifically interacting with their intended target. Protein delivery can also be indirectly by co-electroporating a fluorescent dextran that is similar in size to the protein being introduced (<a href="#fig1">Figure 1</a>B), though co-electroporation of both probes needs to first be documented with an immunostaining experiment against the protein of interest.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Fluorescence activated cell sorting could alternatively be used to assess protein delivery to individual cells. This method has the advantage of being able to analyze a much greater number of cells. Though, information about cell morphology that may be of importance will be lost.</div>
</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Quantifying protein delivery using fluorometry. In this approach, electroporated cells are lysed and their fluorescence is measured in a fluorometer. This requires that the delivered protein is fluorescent. As this is a bulk assay, it does not give a measure of cell-to-cell variability. Thus, it is recommended that this assay is combined with one that does such as quantitative imaging (step 4). Also, it only allows a relative comparison to be made between groups electroporated with different amounts of protein (<a href="#fig1">Figure 1</a>C).
<ol type="a">
<li>Pre-warm 10 mL aliquot of media (DMEM/F-12 with 10% FBS) in an incubator (37°C, 5% CO<sub>2</sub>) and then divide 5 mL among (6) 1.5 mL Eppendorf tubes.</li>
<li>Thaw Alexa647-tagged Thymosin β4 or your choice of high quality fluorescently tagged protein or molecule.</li>
<li>Follow the protocol of step 1 with 2–3 times the starting cell volume (6–9 60%–80% confluent 10 cm dishes). Double the electroporation volumes for four electroporations/condition instead of two.</li>
<li>Take extra care to thoroughly resuspend the total cell volume and then count cells. Aliquot total cell volume equally among all conditions. Make a control for electroporation by mixing fluorescently tagged protein with lysate but do not electroporate.</li>
<li>After electroporation, deliver cells to the Eppendorf tubes and partially close the lids to allow for gas exchange. Allow cells to rest for 10 min in an incubator (37°C, 5% CO<sub>2</sub>).</li>
<li>Centrifuge Eppendorf tubes for 3 min, 150 × <i>g</i>, at 20°C–25°C. Remove media and replace with 1 mL of DPBS taking care to gently resuspend the pellet. Repeat the process of centrifugation, gentle re-suspension and washing, and DPBS removal 3×. Leave the pellet in DPBS.
<div class="note">
<span class="note-title">Note:</span> This process can be performed under sterile or non-sterile conditions.</div>
</li>
<li>Suspend each pellet in the same volume of a fluorometry-suitable protein buffer. The Qubit system comes with its own protein buffer (Component B).</li>
<li>Using a properly calibrated Qubit 4 Fluorometer or equivalent instrument, select the “Fluorometer” function and choose the Red (635 nm) excitation wavelength.
<div class="note">
<span class="note-title">Note:</span> Check the excitation wavelength for the fluorophore of your protein or molecule of interest.</div>
</li>
<li>Record relative fluorescence units for each sample and graph as a function of protein concentration in the electroporation chamber.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Quantifying protein delivery by western blot. This approach is similar to fluorometry (step 5), in that it is a bulk method that quantifies mean protein delivery among cells in a population. However, it offers a greater flexibility in what proteins are able to be measured since they do not have to be fluorescently labeled. This method does require that the protein of interest can be differentiated on the western blot, either because it runs at a different size than the endogenous protein or has an epitope tag that allows it to be specifically identified. As with fluorometry, it is recommended that an approach that quantifies cell-to-cell variability such as quantitative fluorescence imaging (step 4) is also performed. It is also important to know the electroporation efficiency (step 3) to accurately interpret the results.
<ol type="a">
<li>Dispense 4–5 mL/dish of cell culture media (DMEM/F-12 with 10% FBS) in (6) 60 mm tissue culture dishes (4–5 mL/dish) and place dishes in an incubator for a minimum of 30 min prior to electroporation. Pre-warm a 50 mL aliquot of media to 37°C in a water or bead bath.</li>
<li>Follow the protocol of step 1 with 2–3 times the starting cell volume (6–9 60%–80% confluent 10 cm dishes). Double the electroporation volumes for four electroporations/condition instead of two. Additionally, use control cells to establish endogenous levels of protein.</li>
<li>After electroporation, allow cells to adhere for 2 h in an incubator (37°C, 5% CO<sub>2</sub>).</li>
<li>Prepare for cell lysis:
<ol type="i">
<li>Make a 2 mL RIPA buffer solution (50 mM Tris-HCl pH 7.4, 150 mM NaCl, 1% Triton X-100, 0.5% sodium deoxycholate, 0.1% sodium dodecyl sulfate) with 80 μL of solubilized 25× Roche Protease inhibitor cocktail. Keep on ice.</li>
<li>Aliquot 10 mL of DPBS and keep on ice.</li>
<li>Label (12) 1.5 mL Eppendorf tubes and keep on ice.</li>
<li>Pre-chill a microcentrifuge capable of speeds up to 10,000 × <i>g</i> to 4°C.</li>
</ol>
</li>
<li>Check for adherence and electroporation in a light and fluorescent microscope, respectively.</li>
<li>Harvest cells.
<ol type="i">
<li>On ice, aspirate media and rinse cells with ice-cold DPBS. Aspirate DPBS and replace with 250 μL of RIPA buffer. Cells should incubate on ice with RIPA buffer for 5 min prior to mechanical lysis.</li>
<li>By using a cell scraper and tilting the plate, collect cells at the bottom of the dish. Carefully alternate scraping and rotating the dish. Be sure to work quickly and keep the cells on ice.</li>
<li>Using a 1 mL pipette, transfer the RIPA buffer solution with cells to the chilled and labeled Eppendorf tube.</li>
<li>Start a timer counting up.</li>
<li>Repeat this for all cells. Do not leave cells out of the incubator longer than 5 min unless they are being harvested.</li>
<li>Work quickly and ensure that the first harvested plate gets the same amount of time on ice in RIPA as the last as monitored by the timer.</li>
</ol>
</li>
<li>Generate whole cell lysate by mechanical lysis of cells with repeated passage (5×) through a 27G needle. Alternately, cells can be lysed via probe-tip sonicator. Either method should be performed on ice.</li>
<li>Centrifuge lysate (10 min, 10,000 × <i>g</i>, 4°C). Pelleted membrane and debris should be seen at the bottom of the tube after centrifugation.</li>
<li>For the Li-Cor system, prepare 6× SDS buffer stained with Orange G (40% glycerol, 6% sodium dodecyl sulfate, 0.5% Orange G, 300 mM Tris-HCl pH 6.8, store at 20°C–25°C).</li>
<li>During centrifugation step h, prepare SDS buffer by adding 5% β-mercaptoethanol (12.5 μL β- mercaptoethanol to 237.5 μL 6× SDS buffer).</li>
<li>After centrifugation, take 30 μL of lysate from each sample and load 10 μL of each sample to three wells onto the same 96 well plate for protein quantification. Keep samples on ice.</li>
<li>Transfer 145 μL of lysate from each sample to the Eppendorf tubes labeled in step diii and dilute with 30 μL of the prepared 6× SDS buffer. Keep all samples on ice until loading.</li>
<li>Take a 96 well plate and load with BSA standards. Dispense 10 μL of each sample to three wells.</li>
<li>Add 190 μL of BCA reagent to all wells using a multi-channel pipette.</li>
<li>Incubate 96 well plate with BCA working reagent for a minimum of 30 min at 37°C. Then read absorbance values with a plate reader. Calculate requisite volumes for loading protein samples. Make sure to multiply by the dilution factor to account for the addition of SDS buffer (<a href="#tbl1">Table 1</a>).</li>
<li>Denature samples at 95°C for 5 min in a heating block. Store samples directly on ice after denaturing and then centrifuge (5 min, 5,000 × <i>g</i>, 4°C).
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples can be stored at −80°C for up to 1 month at this point or processed as described below.</div>
</li>
<li>Make 1× running buffer (1.44% glycine, 0.3% tris base, 0.1% sodium dodecyl sulfate (SDS)) and 1× transfer buffer (7.5% glycine, 1.21% tris base) in 20% methanol added last. Store the transfer buffer aliquots at −20°C.</li>
<li>Load the gel in XCell SureLock Mini-Cell Electrophoresis or equivalent system.
<ol type="i">
<li>Prepare a gel (Novex tris-glycine 1.0 mm mini) for SDS-PAGE. In 1× running buffer, load 10 μL of WesternSure Pre-Stained Chemiluminescent Protein Ladder (Li-Cor) into the left-most lane.</li>
<li>Load 10 μg of each sample into the subsequent lanes, keep track of which lanes receive which samples.</li>
<li>Load empty lanes with 2 μL of 6× SDS buffer.</li>
</ol>
</li>
<li>Run gel at constant voltage until loading dye is close to the bottom of the gel (∼2 h, 120 V at 20°C–25°C).</li>
<li>1 h and 45 min after the start of running the gel, wet a polyvinylidene difluoride (PVDF) membrane in a glass container with enough 100% methanol (MetOH) to cover the entire membrane. Dispose of the MetOH safely (Note: MetOH for wetting membranes can be reused) and wash the membrane 2× in de-ionized water (dH<sub>2</sub>O). On the second wash, shake the membrane in dH<sub>2</sub>O for 2 min. Remove the water and wash the membrane in transfer buffer on an orbital shaker or rocker. Ensure the membrane does not dry out.</li>
<li>Pre-wet the filter pads, blotting paper, and gel in ice-cold transfer buffer for a wet transfer in the XCell II Blotting or equivalent module. Assemble stack for transfer (from bottom to top: filter pad, blotting paper, gel, membrane, blotting paper, filter pad) and transfer on ice at constant current to the PVDF membrane (50 min, 100 mA).</li>
<li>After transfer, rinse the membrane 2× in dH<sub>2</sub>O. Then, block membrane for 15 min in 5% bovine serum albumin (BSA) in 1× tris buffered saline (TBS) at 20°C–25°C (2.5 g/50 mL). Dilute primary antibodies in 5% BSA, 0.1% Tween-20, then add to membrane. Incubate primary antibody while shaking at 4°C overnight.</li>
<li>The next day, wash membrane in 0.1% Tween-20 in TBS 3× for 30 min. Incubate membrane with Li-Cor secondary antibodies diluted in 5% BSA, 0.1% Tween-20 in TBS for 2 h at 20°C–25°C while shaking. The membrane is then washed in 0.1% Tween-20 in TBS 3× for 30 min, and then washed in 1× TBS 2×.</li>
<li>Visualize western and quantify the bands according to manufactures instructions.
<div class="note">
<span class="note-title">Note:</span> Use a quantitative western analysis such as via the Li-Cor system to maintain linearity in detection. Quantitative western blotting can also be performed to determine how much protein is delivered in reference to endogenous levels. If establishing intracellular concentrations is desired, load a range of purified protein concentrations to generate a standard curve. To calculate concentration of protein per cell from western blot results, both the cell volume and the number of cells loaded per well are required.</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Using the protein delivery assessment methods described in steps 6–8, we have shown that the amount of delivered protein is linearly proportional to the bath concentration in the electroporation reaction for FITC-labeled dextran (dextran-FITC) (<a href="#fig1">Figure 1</a>A), Alexa 647-labeled thymosin β4 (Tβ4-647) (<a href="#fig1">Figure 1</a>B), Tβ4-647 that has been co-electroporated with dextran-FITC (<a href="#fig1">Figure 1</a>C), and profilin 1 (<a href="#fig1">Figure 1</a>D) (<a href="#bib10">Skruber et al., 2020</a>). Further, we have shown that proteins are functional after delivery by electroporation. Shown in <a href="#fig2">Figure 2</a> is the effect of Tβ4-647 electroporation on polymerized actin levels. Since Tβ4 sequesters actin monomers (<a href="#bib7">Safer et al., 1991</a>), increasing Tβ4 concentration in cells should reduce the amount of polymerized actin. Electroporating cells with a 125 μM bath concentration of Tβ4 reduces polymerized actin levels approximately 30% (<a href="#fig2">Figure 2</a>). Additionally we have demonstrated that electroporating physiological levels of profilin 1 can rescue actin defects seen in PFN1 KO cells (<a href="#bib10">Skruber et al., 2020</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/413-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Protein delivered by electroporation retains its functionality</div>
<p>(A) Representative staining of Thymosin β4-647 (125 μM) after electroporation. Cells were transfected, then fixed and stained with 488-phalloidin after permeabilization.</p>
<p>(B) Phalloidin intensity is used as a measure of polymerized actin.</p>
<p>N = 140, 121 for cells electroporated with buffer or Thymosin β4-647 (125 μM), respectively. Scale bar, 10 μm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol must be optimized for each cell line and each transfected protein. Some cell lines may be less suitable for electroporation, resulting in low electroporation efficiency or high cell death. Similarly, different proteins may be unsuitable for electroporation based on their size, conformation, amino acid composition, or required buffer for proper folding. See <a href="#troubleshooting">Troubleshooting</a> for a discussion of these topics.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem</h3>
<p>High cell death after electroporation (step 1).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Maximizing cell viability generally may require sacrificing electroporation efficiency and vice-versa (<a href="#bib9">Sherba et al., 2020</a>). We chose to optimize electroporation efficiency and accounted for low levels (∼10%) of routine cell death during electroporation. However, high (∼50%) cell death is a problem that compromises the experiment. If high cell death is seen in comparison to the non-electroporated control (step 1), start by ensuring that no antibiotics are present in the media post-electroporation. Antibiotics in the media hinder recovery and may be cytotoxic in the 1–2 h electroporation recovery period. Increasing serum (for instance from 8% to 10%) may additionally aid in post-electroporation recovery. Arcing is also a major cause of reduction in cell viability. Arcing causes a flash of light that can be easily visualized in the Neon tip during the electroporation. If loss of viability is only present in certain conditions, it is possible that arcing occurred but was missed visually. The Amaxa device reports when arcing has occurred to clarify this possibility. Researchers should be vigilant during electroporation to ensure arcing does not occur and steps should be taken to prevent it (see below). If high cell death is seen in comparison to no protein control or correlates with increasing volumes of added protein, then protein buffer composition may be a cause of excessive cell death. Concentrate the protein and determine the new concentration. As the protein buffer is mixed with the electroporation buffer R, there is a level of tolerance for less than ideal buffer compositions. The ratio between the buffer required for protein stability and the ideal buffer for electroporation must be experimentally optimized. Lastly, some have found that washing with DPBS or treating cells with low concentrations of trypsin after electroporation helps to deal with post-electroporation recovery (<a href="#bib1">Alex et al., 2019</a>).</p>
<h3 id="sec6.3">Problem</h3>
<p>Excessive arcing during electroporation (step 1).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Excessive arcing during electroporation is most likely due to bubbles in the electroporation tip. After mixing the total electroporation solution the sample can be pulse centrifuged and very gently resuspended. Excessive salt in the protein buffer could also lead to excessive arcing. Desalting the protein, dialyzing into a different buffer, and reconcentrating may provide a more suitable electroporation reaction. Additional sources of arcing could include contaminants in the electroporation mix, perhaps due to incomplete media removal prior to electroporation.</p>
<h3 id="sec6.5">Problem</h3>
<p>Low electroporation efficiency (steps 1 and 3).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The most likely solution is to optimize the electroporation settings by varying the pulse length and voltage. A short, high voltage pulse, a long, low voltage pulse, or a protocol with two successive pulses are good starting points. A complete range of conditions can be found in the documentation for the electroporation system as provided by the manufacturer. The quality of cells used for electroporation can also affect electroporation efficiency. Cells should be recently passaged and in log growth phase. Cells which are attached to each other may pose a problem. This could be solved by light trituration in the presence of trypsin during the cell collection phase. Understanding the time course of degradation of the protein may also be important during troubleshooting. Knowing whether the experiment is taking place during the time period prior to protein turn-over may be important and could be performed with a combination translation and proteasome inhibitor (such as cycloheximide and MG-132). Lastly, establishing electroporation efficiency with an inert fluorescence dextran of the same approximate size of the protein can help determine if the desired electroporation is possible.</p>
<h3 id="sec6.7">Problem</h3>
<p>The delivered protein appears to have no effect after electroporation.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>First, check the protein preparation and its concentration. Protocols for purification are highly specific for each protein and may require strict adherence or troubleshooting, see protocols for profilin (<a href="#bib4">Henty-Ridilla et al., 2017</a>) and Thymosin β4 (<a href="#bib11">Yarmola et al., 2000</a>) for reference. For determination of protein concentration, the protein buffer composition should be known and the blank or protein standard used should be solubilized in a buffer identical to the protein sample. Re-establishing protein concentration after high speed centrifugation will ensure only solubilized protein is measured. If protein is no longer soluble or is aggregating, then empirically determine ideal storage conditions and acquire a fresh protein preparation. Establishing the range of protein concentration relative to endogenous levels may be an important insight into both the function of the protein and to the physiological relevance of the experiment. Protein degradation kinetics and lifetimes can be established to understand whether the experiment is taking place during the time period prior to turn-over. However, we recommend erring on minimizing time post-electroporation as per the protocol to avoid this confounding factor.</p>
<p>In our experience, after optimization of electroporation parameters, protein preparation is the largest determinant of functionality. While protein purification protocols are specific for each protein, knowledge about general strategies for purification may aid in enhancing protein quality (<a href="#bib8">Scopes, 2013</a>). Alternatively, users of this protocol may opt to buy commercially available purified proteins or outsource the purification step. Cell quality is also an important factor in protocol optimization. 1×10<sup>6</sup> cells can be transfected with high efficiency and high viability as long as they are in log-phase growth and at low passage number. Protein suitability for electroporation must be determined experimentally as variables like size and charge may alter their ability to be electroporated. We have electroporated proteins up to 40 kDa (<a href="#bib10">Skruber et al., 2020</a>), and others have shown that electroporation is possible for 70 kDa (<a href="#bib2">Demiryurek et al., 2015</a>) and 148 kDa (<a href="#bib3">Graziadei et al., 1991</a>) sized molecules. Little is known about the effect of protein charge on electroporated, but we have observed quantitative delivery of both acidic (<a href="#fig1">Figure 1</a>B) and basic (<a href="#fig1">Figure 1</a>D) proteins.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Alex, A., Piano, V., Polley, S., Stuiver, M., Voss, S., Ciossani, G., Overlack, K., Voss, B., Wohlgemuth, S., Petrovic, A., et al. (2019). Electroporated recombinant proteins as tools for in vivo functional complementation, imaging and chemical biology. eLife <i>8</i>, e48287.</p>
<p id="bib2">Demiryurek, Y., Nickaeen, M., Zheng, M., Yu, M., Zahn, J.D., Shreiber, D.I., Lin, H., and Shan, J.W. (2015). Transport, resealing, and re-poration dynamics of two-pulse electroporation-mediated molecular delivery. Biochim. Biophys. Acta <i>1848</i>, 1706-1714.</p>
<p id="bib3">Graziadei, L., Burfeind, P., and Bar-Sagi, D. (1991). Introduction of unlabeled proteins into living cells by electroporation and isolation of viable protein-loaded cells using dextran-fluorescein isothiocyanate as a marker for protein uptake. Anal. Biochem. <i>194</i>, 198-203.</p>
<p id="bib4">Henty-Ridilla, J.L., Juanes, M.A., and Goode, B.L. (2017). Profilin directly promotes microtubule growth through residues mutated in amyotrophic lateral sclerosis. Curr. Biol. <i>27</i>, 3535-3543.e4.</p>
<p id="bib5">Kim, J.A., Cho, K., Shin, M.S., Lee, W.G., Jung, N., Chung, C., and Chang, J.K. (2008). A novel electroporation method using a capillary and wire-type electrode. Biosens. Bioelectron. <i>23</i>, 1353-1360.</p>
<p id="bib6">Qi, Y., Wang, J.K., McMillian, M., and Chikaraishi, D.M. (1997). Characterization of a CNS cell line, CAD, in which morphological differentiation is initiated by serum deprivation. J. Neurosci. <i>17</i>, 1217-1225.</p>
<p id="bib7">Safer, D., Elzinga, M., and Nachmias, V.T. (1991). Thymosin beta 4 and Fx, an actin-sequestering peptide, are indistinguishable. J. Biol. Chem. <i>266</i>, 4029-4032.</p>
<p id="bib8">Scopes, R.K. (2013). Protein Purification: Principles and Practice (Springer Science &amp; Business Media).</p>
<p id="bib9">Sherba, J.J., Hogquist, S., Lin, H., Shan, J.W., Shreiber, D.I., and Zahn, J.D. (2020). The effects of electroporation buffer composition on cell viability and electro-transfection efficiency. Sci. Rep. <i>10</i>, 3053.</p>
<p id="bib10">Skruber, K., Warp, P.V., Shklyarov, R., Thomas, J.D., Swanson, M.S., Henty-Ridilla, J.L., Read, T.A., and Vitriol, E.A. (2020). Arp2/3 and mena/vasp require profilin 1 for actin network assembly at the leading edge. Curr. Biol. <i>30</i>, 2651-2664.e5.</p>
<p id="bib11">Yarmola, E.G., Somasundaram, T., Boring, T.A., Spector, I., and Bubb, M.R. (2000). Actin-latrunculin A structure and function. Differential modulation of actin-binding protein function by latrunculin A. J. Biol. Chem. <i>275</i>, 28120-28127.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Eric Vitriol (<a href="mailto:evitriol@ufl.edu">evitriol@ufl.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze [datasets/code].</p>
<h3>Acknowledgments</h3>
<p>This project was supported by a <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (NIH) Pathway to Independence Award (R00 NS087104) and an NIH Maximizing Investigators’ Research Award for Early Stage Investigators (R35 GM137959) to E.A.V.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, K.S. and E.A.V.; Methodology, K.S. and E.A.V.; Formal Analysis, K.S. and E.A.V.; Investigation, K.S. and E.A.V.; Resources, K.S. and E.A.V.; Writing, K.S., T.A.R., and E.A.V.; Visualization, K.S. and E.A.V.; Project Administration, E.A.V.; Supervision, E.A.V.; Funding Acquisition, E.A.V.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>