<article>
<section>
<h1 id="header">Optimized protocol for isolation of high-quality single cells from the female mouse reproductive tract tissues for single-cell RNA sequencing</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Rajendra Kumar Gurumurthy,<sup><a href="#aff1">1</a></sup> Naveen Kumar,<sup><a href="#aff2">2</a></sup> and Cindrilla Chumduri<sup><a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Biology, Max Planck Institute for Infection Biology, Berlin 10117, Germany</p>
<p id="aff2"><sup>2</sup>Chair of Microbiology, University of Würzburg, Würzburg 97074, Germany</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:cindrilla.chumduri@uni-wuerzburg.de">cindrilla.chumduri@uni-wuerzburg.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100970">10.1016/j.xpro.2021.100970</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Single-cell RNA sequencing (scRNA-seq) is a powerful tool for enumerating the gene expression dynamics at single-cell resolution. Various organs comprising distinct cellular composition and architecture require unique approaches for highly viable single-cell preparation and reliable sequencing results. Here, we describe an optimized protocol for isolating the female reproductive tract (FRT), dissecting different FRT regions, and preparing high-viability single cells from the uterine endocervix and ectocervix to generate a complete molecular cell atlas by scRNA-seq for studying normal physiology and disease.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00676-6/sref1">Chumduri et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Dissection of the mouse female reproductive tract (FRT)</li>
<li>Separation of the uterine ectocervix, endocervix, and transition zone</li>
<li>An optimized protocol for single-cell preparation from the uterine cervix</li>
<li>Enables characterization of FRT tissues at single-cell resolution</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1188-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>All experiments involving animals were performed according to regulations for animal research and approvals from national legal, institutional, and local authorities at Max Planck Institute for Infection Biology.</p>
<h3 id="sec1.1">Background</h3>
<p>This protocol describes the procedure of dissection of the female reproductive tract (FRT) from mice and subsequent dissociation of tissues to isolate single cells for the single-cell RNA sequence (scRNA seq) analysis. This protocol is mainly developed for the mouse tissues, but it can be adopted for human tissue samples. Isolated tissue should be processed immediately for the scRNA seq analysis to avoid degradation of RNA.</p>
<h3 id="sec1.2">Prepare before beginning the experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol>
<li>Keep all the buffers and solutions ready (for details, see <a href="#key-resources-table">key resources table</a>, <a href="#materials-and-equipment">materials and equipment</a> sections).</li>
<li>Autoclave 2 sets of dissection tools and 1.5 mL Microfuge tubes.</li>
</ol>
<h3 id="sec1.3">Prepare the following on the day of the experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="3">
<li>Keep the dissection platform and a dissection microscope ready.</li>
<li>Keep the autoclaved dissection sets, including forceps and scissors, Petri dishes, and scalpel/blades ready for performing the dissection.</li>
<li>Keep a Falcon tube containing 70% ethanol and one Falcon tube with sterile PBS for cleaning dissection tools.</li>
<li>Pre-cool the PBS, keep the 50 mL collection tubes, 40 μm cell strainer, and ice buckets ready.</li>
<li>Turn on the cell culture hood at least 15 min before starting the work.</li>
<li>Turn on the water bath and set the temperature to 37°C.</li>
<li>Cooldown the centrifuges to 4°C.</li>
<li>Thaw the collagenase type II solution on ice and prewarm collagenase type II (see <a href="#materials-and-equipment">materials and equipment</a> section) and TrypLE solutions (see <a href="#key-resources-table">Key resources table</a>) at 37°C.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DPBS, no calcium, no magnesium</td>
<td>Gibco</td>
<td>Cat#14190-169</td>
</tr>
<tr>
<td>Hank's balanced salt solution (HBBS)</td>
<td>Gibco</td>
<td>Cat#14025-092</td>
</tr>
<tr>
<td>Collagenase Type II</td>
<td>Merck</td>
<td>Cat#234155</td>
</tr>
<tr>
<td>BSA</td>
<td>Carl Roth</td>
<td>Cat#8076.3</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Merck</td>
<td>Cat#1.00983.2511</td>
</tr>
<tr>
<td>TrypLE</td>
<td>Gibco</td>
<td>Cat#12605-028</td>
</tr>
<tr>
<td>Trypan blue solution, 0.4%</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15250061</td>
</tr>
<tr>
<td>Autoclaved distilled water</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Chromium Single Cell A Chip Kit</td>
<td>10× Genomics</td>
<td>Cat#PN-120236</td>
</tr>
<tr>
<td>Chromium Single Cell 3' Library &amp; Gel Bead Kit v2</td>
<td>10× Genomics</td>
<td>Cat#PN-120237</td>
</tr>
<tr>
<td>Agilent High Sensitivity DNA Kit</td>
<td>Agilent</td>
<td>Cat#5067-4626</td>
</tr>
<tr>
<td>NovaSeq6000 S1 Reagent Kit (100 cycles)</td>
<td>Illumina</td>
<td>Cat#20012865</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>C57bl/6J wild type mice</td>
<td>The Jackson Laboratory</td>
<td><a href="https://www.jax.org/">https://www.jax.org/</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Seurat v4.0.4</td>
<td>Satija Lab</td>
<td>RRID:SCR_016341</td>
</tr>
<tr>
<td>R v4.1.1</td>
<td>R Project for Statistical Computing</td>
<td>RRID:SCR_001905</td>
</tr>
<tr>
<td colspan="3">Others</td>
</tr>
<tr>
<td>0.22 μm filter</td>
<td>Merck Millipore</td>
<td>Cat#SLGP033RS</td>
</tr>
<tr>
<td>Pipette tips, 10 μL</td>
<td>Sarstedt</td>
<td>Cat#1181</td>
</tr>
<tr>
<td>Pipette tips, 200 μL</td>
<td>Sarstedt</td>
<td>Cat# 3030</td>
</tr>
<tr>
<td>Pipette tips, 1000 μL</td>
<td>Sarstedt</td>
<td>Cat#1181</td>
</tr>
<tr>
<td>Biosphere Filter tips 20 μL</td>
<td>Sarstedt</td>
<td>Cat#70.1114.210</td>
</tr>
<tr>
<td>Biosphere Filter tips 200 μL</td>
<td>Sarstedt</td>
<td>Cat#70.760.216</td>
</tr>
<tr>
<td>Biosphere Filter tips 1000 μL</td>
<td>Sarstedt</td>
<td>Cat#70.762.211</td>
</tr>
<tr>
<td>Sterile scissors</td>
<td>B. Braun</td>
<td>Cat#BC050R</td>
</tr>
<tr>
<td>Bent forceps</td>
<td>Bochem Lab Supply</td>
<td>Cat#1130</td>
</tr>
<tr>
<td>100 mm Petri dish</td>
<td>TPP</td>
<td>Cat#P93150</td>
</tr>
<tr>
<td>Parafilm</td>
<td>Bemis</td>
<td>Cat#PM996</td>
</tr>
<tr>
<td>15 mL Falcon tubes</td>
<td>Sarstedt</td>
<td>Cat#62.554.100</td>
</tr>
<tr>
<td>50 mL Falcon tubes</td>
<td>Sarstedt</td>
<td>Cat#62.547.004</td>
</tr>
<tr>
<td>Neubauer counting chamber</td>
<td>Marienfeld</td>
<td>Cat#0640210</td>
</tr>
<tr>
<td>Microcentrifuge tubes 1.5 mL</td>
<td>Sarstedt</td>
<td>Cat#72.690.001</td>
</tr>
<tr>
<td>40 μm cell strainer</td>
<td>BD Falcon</td>
<td>Cat#352340</td>
</tr>
<tr>
<td>Scalpel blade</td>
<td>Carl Roth</td>
<td>Cat#Ex75</td>
</tr>
<tr>
<td>26 G needle</td>
<td>VWR</td>
<td>Cat#612-0147</td>
</tr>
<tr>
<td>High-speed centrifuge</td>
<td>Eppendorf</td>
<td>Cat#5810 R</td>
</tr>
<tr>
<td>Water bath</td>
<td>GFL</td>
<td>Cat#1002</td>
</tr>
<tr>
<td>Cell culture hood</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10110910</td>
</tr>
<tr>
<td>Eppendorf centrifuge</td>
<td>Eppendorf</td>
<td>Cat#5417R</td>
</tr>
<tr>
<td>Benchtop orbital shaker</td>
<td>Thermo Scientific</td>
<td>Cat#SHKE420HP</td>
</tr>
<tr>
<td>Bright-field microscope</td>
<td>OLYMPUS</td>
<td>Cat#IX50</td>
</tr>
<tr>
<td>10× Genomics Chromium Controller</td>
<td>10× Genomics</td>
<td>Cat#PN110203</td>
</tr>
<tr>
<td>SimpliAmp Thermal Cycler</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A24811</td>
</tr>
<tr>
<td>2100 Bioanalyzer instrument</td>
<td>Agilent</td>
<td>Cat#G2939BA</td>
</tr>
<tr>
<td>NovaSeq 6000 System</td>
<td>Illumina</td>
<td>Cat#20012850</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Refer material safety data sheets of chemicals before reagent preparation and take protective measures whenever handling hazardous chemicals. Follow biosafety precautions when handling biological material.</p>
<table id="undtbl1">
<caption>Collagenase Type II solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hank's balanced salt solution (HBBS)</td>
<td>N/A</td>
<td>200 mL</td>
</tr>
<tr>
<td>Collagenase Type II</td>
<td>0.5 mg/mL</td>
<td>100 mg</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>200 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Make aliquots of the required volume in 15 mL Falcon tubes. Aliquots can be stored at −20°C for up to 2 years. Avoid repeated freezing and thawing since it decreases the enzyme activity.</div>
<table id="undtbl2">
<caption>4% BSA in PBS (100× stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DPBS</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>4%</td>
<td>400 mg</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Reagent can be stored at −20°C for more than a year. Dilute 4% BSA in PBS 1:100 with DPBS before use at a final concentration of 0.04%.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Isolation of female mouse reproductive tract and dissection of ectocervix and endocervix for single-cell preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol>
<li>Euthanize experimental mice using the appropriate ethically approved technique.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For single-cell sequencing, we combine cells isolated from three mice and process each FRT tissue region separately.</div>
<ol start="2">
<li>Sterilize the mouse by spraying with 70% ethanol on the whole-body surface.</li>
<li>Firmly immobilize the euthanized mouse on the dissection platform, with the ventral side facing up, using 26G needles by spreading the legs and pinning all the 4 feet to the dissection platform (<a href="#fig1">Figure 1</a>A and <a href="#mmc1">Methods video S1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1188-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Preparation and isolation of mouse FRT</div>
<p>(A) Euthanized mice prepared for dissection.</p>
<p>(B) Dissected mouse with exposed genital tract (left panel) with a yellow arrow highlighting the FRT and the isolated FRT (right panel).</p>
<p>(C) The different FRT regions are separated.</p>
</figcaption>
</figure>
</li>
<li>Make an incision in the skin and underlying abdominal muscles extending down to the genital part using sterile scissors and pin the opened skin to the dissection platform.</li>
<li>Clear off any connective tissue from the FRT (<a href="#fig1">Figure 1</a>B).</li>
<li>Using bent forceps, carefully lift out the entire genital tissue by holding at the middle part of the cervix region (harder part of the cervix) and cut out the tissue from the area connected to the vagina (<a href="#mmc1">Methods video S1</a>).</li>
<li>Place the FRT tissue in an empty 100 mm Petri dish and remove any associated adipose and connective tissue (<a href="#fig1">Figure 1</a>B, and <a href="#mmc1">Methods video S1</a>).</li>
<li>Transfer the tissue to a 15 mL Falcon tube containing 10 mL ice-cold PBS and wash tissue by inverting the tube several times.</li>
<li>Transfer the tissue to a sterile 100 mm Petri dish using forceps.</li>
<li>Separate ectocervix and endocervix from the junction as indicated in <a href="#fig1">Figure 1</a>C and <a href="#mmc1">Methods video S1</a> using a scalpel blade based on the physical characteristics of the tissue as follows.
<ol type="a">
<li>First, locate the ectocervix based on its hard, dense, and fibrous texture and proximity to the vagina. Cut the vaginal region from the ectocervix.</li>
<li>Then choose a region close to the place of uterine horns bifurcation and dissect endocervical tissue.</li>
<li>Dissect out an area physically overlapping with the defined ectocervix and the endocervix, called the transition zone (TZ).</li>
<li>Place different FRT tissue regions in separate 100 mm Petri dishes.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use a separate scalpel blade to cut the ectocervix and endocervix region from the cervix junction to avoid contamination of cells between tissues.</div>
<div class="note">
<span class="note-title">Note:</span> From this step on, the tissue from different cervical regions can be processed in parallel.</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1188-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Preparation and isolation of mouse FRT, related to steps 3–10</div>
<p>Shown are the steps in preparing the euthanized mice for dissection on the dissection platform, dissection, isolation of the FRT, and separation of ectocervix and endocervix from the junction as indicated in <a href="#fig1">Figure 1</a>C</p>
</figcaption>
</figure>
<h3 id="sec3.2">Cell isolation from the uterine cervix for single-cell RNA sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to prepare single-cell suspensions within a short time and maintain the cells in a cool condition where appropriate to ensure high-quality RNA for the 10× Genomics sequencing.</div>
<ol start="11">
<li>By holding tissue with forceps, mince into small pieces using sterile scissors (less than 0.5 mm<sup>3</sup> thickness) (<a href="#fig2">Figures 2</a>A and 2B) (<a href="#sec6.1">troubleshooting 1</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1188-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Single-cell preparation from the endocervix and ectocervix tissue</div>
<p>(A) Uterine Cervix tissue collected in PBS in a petri dish.</p>
<p>(B) Cervix tissue minced into small pieces with scissors and scalpel blade.</p>
<p>(C and D) Minced tissue is transferred to collagenase type II containing Falcon tube for tissue digestion (C) and the Falcon tube containing digested tissue after collagenase type II treatment (D).</p>
<p>(E) Digested tissue is incubated with TrypLE for cell dissociation.</p>
<p>(F) Filtration of the dissociated cells to remove cell clumps.</p>
<p>(G and H) Cells isolated from uterine cervix tissue and imaged at 10× (G) and 20× (H) magnification.</p>
</figcaption>
</figure>
</li>
<li>Transfer the minced tissue with a pipette into a 15 mL Falcon tube containing 3 mL of collagenase type II (0.5 mg/mL) per tube pre-warmed at 37°C (<a href="#fig2">Figure 2</a>C).</li>
<li>Incubate the tubes containing collagenase type II for 45 min at 37°C, 180 rpm in a horizontal position in an orbital shaker incubator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Seal the cap of the Falcon tube with Parafilm to avoid leakage of solution and loss of cells.</div>
<ol start="14">
<li>After digestion, gently pipette up and down 20 times with a 1 mL pipette to dissociate cell aggregates (<a href="#fig2">Figure 2</a>D) (<a href="#sec6.1">troubleshooting 1</a>).</li>
<li>Centrifuge the suspension at 1000 × g for 6 min at 4°C.</li>
<li>Discard the supernatant carefully and retain the pellet.</li>
<li>Add 3 mL of pre-warmed TrypLE and resuspend the cell pellet using a 5 mL pipette boy by pipetting up and down 5 times.</li>
<li>Incubate for 15 min at 37°C, 180 rpm, in an orbital shaker by placing the tube in the horizontal position.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Seal the Falcon tube cap with Parafilm to avoid leakage of solution and loss of cells.</div>
<ol start="19">
<li>Gently pipette up and down 20 times with a 1 mL pipette to dissociate cell aggregates (<a href="#fig2">Figure 2</a>E).</li>
<li>Add 5 mL of ice-cold PBS, centrifuge at 1000 × g for 6 min at 4°C.</li>
<li>Discard the supernatant and retain the pellet.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Re-suspend the pellet in 5 mL of ice-cold PBS containing 0.04% BSA, mix by inverting the tube several times.</div>
<ol start="22">
<li>Pass the solution into a pre-assembled 50 mL Falcon tube topped with a 40 μm cell strainer placed on ice to filter out tissue debris (<a href="#fig2">Figure 2</a>F) (<a href="#sec6.3">troubleshooting 2</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Pre-wetting the filter with the PBS used for the cells can ease the passage of cells.</div>
<ol start="23">
<li>Centrifuge at 1000 × g for 6 min at 4°C, discard the supernatant.</li>
<li>Resuspend the cells in 3 mL ice-cold PBS containing 0.04% BSA and count the cells using a Neubauer counting chamber (<a href="#fig2">Figure 2</a>G) (<a href="#sec6.5">troubleshooting 3</a> and <a href="#sec6.7">4</a>).</li>
<li>Proceed immediately to step 28−29 for single-cell sequencing and to avoid loss of cell viability.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the cell type, viability may significantly decrease or form clumps (such as peripheral blood mononuclear cells) when cells are kept in suspension for a prolonged time before use. Non-viable and dying cells generally contain more fragmented RNA that may not be efficiently captured by 10× Genomics Single Cell Solutions and may, therefore, decrease cell recovery efficiency.</div>
<h3 id="sec3.3">Optional: Cell viability assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<ol start="26">
<li>Take 10 μL of cell suspension from step 25 and mix thoroughly with 10 μL of trypan blue.</li>
<li>Load 10 μL onto the Neubauer chamber to determine the cell viability.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the samples have less than 70% viability, dead cells may be removed by following the dead cells removal protocol from 10× Genomics <a href="https://support.10xgenomics.com/single-cell-gene-expression/sample-prep/doc/demonstrated-protocol-removal-of-dead-cells-from-single-cell-suspensions-for-single-cell-rna-sequencing">https://support.10xgenomics.com/single-cell-gene-expression/sample-prep/doc/demonstrated-protocol-removal-of-dead-cells-from-single-cell-suspensions-for-single-cell-rna-sequencing</a>
</div>
<ol start="28">
<li>Centrifuge at 1000 × g for 6 min at 4°C, discard the supernatant.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Re-suspend the cell pellet in the required volume of ice-cold PBS containing 0.04% BSA to obtain a cell concentration of 700 to 1200 cells/ μL, which is the optimal range of cell concentration for 10× Chromium Controller.</div>
<ol start="29">
<li>Immediately following cell counting, proceed to the 10× Genomics protocol for single library preparation and sequencing.
<ol type="a">
<li>Perform single-cell partition into nanolitre-scale Gel-Bead-In-Emulsions (GEMs) and reverse transcription using Single-Cell 3′ Reagent Kit v2 and library construction for Illumina sequencing (<a href="https://support.10xgenomics.com/single-cell-gene-expression/library-prep/doc/user-guide-chromium-single-cell-3-reagent-kits-user-guide-v2-chemistry">https://support.10xgenomics.com/single-cell-gene-expression/library-prep/doc/user-guide-chromium-single-cell-3-reagent-kits-user-guide-v2-chemistry</a>).
<div class="note">
<span class="note-title">Note:</span> The data in our manuscript <a href="#bib1">Chumduri et al. (2021)</a> was generated using the V2 kit. However, a new kit with v3 Chemistry has become available currently (<a href="https://support.10xgenomics.com/single-cell-gene-expression/library-prep?filter=chromium">https://support.10xgenomics.com/single-cell-gene-expression/library-prep?filter=chromium</a>). We recommend following the manufacturer's instructions for using the newer versions of the kits as they become available.</div>
<ol type="i">
<li>Load approximately 13,000 cells per sample onto the 10× Genomics Chromium Controller chip to recover approximately 6,000 cells (<a href="#bib1">Chumduri et al., 2021</a>).
<div class="note">
<span class="note-title">Note:</span> Users can choose other loading cell numbers and expected target cell recovery depending on the need and budget. 10× Genomics recommends loading 13,000 cells for 7500 target cell recovery (<a href="#sec6.9">troubleshooting 5</a>).</div>
</li>
<li>Perform reverse transcription of single-cell GEMs to produce barcoded cDNA and amplify the cDNA using a SimpliAmp Thermal Cycler (Applied Biosystems) or similar Thermal Cycler.</li>
<li>Quantify the cDNA library and check the quality using a Bioanalyzer with a High Sensitivity DNA kit (Agilent).</li>
</ol>
</li>
<li>Proceed to sequence in paired-end mode with S1 100-cycles kit using Novaseq 6000 sequencer (Illumina) with approximately 400 million reads allocated per sample with at least 70,000 reads per cell.
<div class="note">
<span class="note-title">Note:</span> Users can choose other platforms depending on sample size, the required sequencing depth, and the budget. For details concerning quality control and computational analysis of the scRNA sequence data, refer to (<a href="#bib1">Chumduri et al., 2021</a>).</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This single-cell isolation protocol is optimized to obtain all the cell types of the cervical tissue, including epithelial cells, stromal cells, endothelial and immune cells, and their subpopulations (<a href="#fig3">Figure 3</a>) (<a href="#bib1">Chumduri et al., 2021</a>). With this protocol, approximately 600,000, 550,000, 200,000 cells can be obtained from each 14-weeks-old mouse ectocervix, endocervix, and transition zone, respectively. However, it should be noted that the cell yield can vary depending on the age of the mice. This protocol allows single-cell preparation within a short time, which is critical to obtain high-quality RNA for the 10× Genomics sequencing. This protocol could also be implemented to isolate single cells from the other parts of the mouse FRT and human FRT tissue biopsies by adjusting collagenase type II digestion lengths. This protocol enables an unbiased representation of all the cell types of the tissue to investigate the cellular and microenvironmental changes during development, healthy homeostasis, and pathogenesis.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1188-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. scRNA-seq from the endocervix and ectocervix tissue of the FRT</div>
<p>(A) Uniform manifold approximation and projection (UMAP) of the cellular subclusters from healthy mouse ectocervix and endocervix color-coded by sampled tissue.</p>
<p>(B) UMAP plot of cell types identified in the endocervix and ectocervix tissue using scRNA-seq.</p>
<p>(C) UMAP of epithelial subclusters color-coded by the epithelial type (Columnar; Squamous).</p>
<p>Figures regenerated from endocervix and ectocervix scRNA seq data from <a href="#bib1">Chumduri et al. (2021</a>).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Single-cell sequencing could be expensive to perform separate sequencing of multiple replicates from different mice or donors. However, cells from different replicates can be pooled after separately isolating single cells. Such an approach, while providing a better representation of biological variation from different mice/donors, can also significantly reduce the sequencing costs. Profiling of single cells from multiple samples individually, although provides high resolution with respect to coverage of cells and deeper sequencing depth per cell, the overall cost per sample is still significantly high. To further reduce the costs, single cells prepared from multiple samples parallelly can be uniquely barcoded in suspension and pooled prior to 10× Genomics single-cell library preparation to perform MULTI-seq as described by (<a href="#bib3">McGinnis et al., 2019</a>). The proportion of cells from different tissue compartments of the organ can vary. For instance, epithelial cell abundance will be much lesser than the stromal cells in the endocervix compared to ectocervix. Therefore, by pooling the cells from multiple mice, one can ensure the overall abundance of individual cell types of the tissues. For studies focusing on specific cell types, single-cell suspension generated in this protocol can be used to enrich specific cell types (example, epithelial cells) with Fluorescence-activated cell sorting. While single-cell sequencing provides the global view of cell types and their transcriptional profiles at the single-cell resolution, the information on the spatial organization of the cell types within the tissues is lost during cell preparation. Thus, the application of rapidly evolving spatial transcriptomics (<a href="#bib2">Marx, 2021</a>) combined with sequencing might be an alternative if the spatial information of the cells with the tissue is to be investigated.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The tissue is intact after collagenase type II treatment after step 14.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>This could be because the tissue is not minced into a smaller size in step 11. Mince the tissue into the smallest possible pieces (less than 0.5 mm<sup>3</sup>).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Cell clumps retained in the filter after step 22.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This could be because cells were not dissociated properly. Resuspending the pellet completely by pipetting gently up and down 10 times more before passing the cells through the cell strainer.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Low yield of cells in the final single-cell suspension after step 25.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The possible reason could be errors in centrifugation, discarding of supernatant or resuspending the cell pellet. Maintain the correct centrifugation speed and temperature, make sure that the cell pellet is not disturbed while removing the supernatant, and take care of resuspending the pellet gently in an appropriate volume of ice-cold PBS containing 0.04% BSA.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>A high number of dead cells after the final single-cell suspension after step 25.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>The possible reason can be improper handling of cells during pipetting, centrifugation, resuspension of cells, not maintaining appropriate temperatures, or slow execution of the protocol. Handle the pipetting steps gently, maintain the correct temperature and centrifugation speeds and follow the timings as indicated in the protocol.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>A low number of cells identified from single-cell sequencing after step 32.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The possible reason is an error in the cell counting leading to the loading of too few cells onto the microfluidic chip. Maintain the single-cell stock concentration between 700 to 1200 cells / μL before loading onto the 10× Chromium controller. Further, the target cell recovery can vary with tissue and cell types.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chumduri, C., Gurumurthy, R.K., Berger, H., Dietrich, O., Kumar, N., Koster, S., Brinkmann, V., Hoffmann, K., Drabkina, M., Arampatzi, P., et al. (2021). Opposing Wnt signals regulate cervical squamocolumnar homeostasis and emergence of metaplasia. Nat. Cell Biol. <i>23</i>, 184-197. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00676-6/sref1">View at publisher</a></p>
<p id="bib2">Marx, V. (2021). Method of the Year: spatially resolved transcriptomics. Nat. Methods <i>18</i>, 9-14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00676-6/sref2">View at publisher</a></p>
<p id="bib3">McGinnis, C.S., Patterson, D.M., Winkler, J., Conrad, D.N., Hein, M.Y., Srivastava, V., Hu, J.L., Murrow, L.M., Weissman, J.S., Werb, Z., et al. (2019). MULTI-seq: sample multiplexing for single-cell RNA sequencing using lipid-tagged indices. Nat. Methods <i>16</i>, 619-626. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00676-6/sref3">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. Cindrilla Chumduri (<a href="mailto:cindrilla.chumduri@uni-wuerzburg.de">cindrilla.chumduri@uni-wuerzburg.de</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code. The scRNA seq data reported in <a href="#bib1">Chumduri et al. (2021</a>) has been deposited in GEO under accession number GSE128987.</p>
<h3>Acknowledgments</h3>
<p>We thank Pon Ganish Prakash for regenerating the UMAPs of scRNA-seq data published in <a href="#bib1">Chumduri et al. (2021</a>) and movie compilation and Shilpa Mary Kurian for helping acquire the pictures of the different steps of the protocol. C.C. is funded by the University of Wuerzburg and DFG (GRK 2157). N.K. is funded by <a href="https://doi.org/10.13039/501100001659">DFG</a>-DAAD and DFG (GRK2157).</p>
<h3>Author contributions</h3>
<p>C.C. and R.K.G. developed the protocol. C.C., R.K.G., and N.K. carried out experiments. C.C. made the illustrations and graphical abstract. R.K.G., N.K., and C.C. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>