<article>
<section>
<h1 id="header">Serine/threonine ligation-assisted chemical synthesis of HMGA1a protein with site-specific post-translational modifications</h1>
<p><time datetime="2021-08-27">Published: August 27, 2021</time></p>
<p>Tongyao Wei,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Heng Liu,<sup><a href="#aff1">1</a></sup> Hongxiang Wu,<sup><a href="#aff1">1</a></sup> Fan Pu,<sup><a href="#aff1">1</a></sup> and Xuechen Li<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Chemistry, State Key Laboratory of Synthetic Chemistry, The University of Hong Kong, Hong Kong, P.R. China</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:wei_tongyao@163.com">wei_tongyao@163.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:xuechenl@hku.hk">xuechenl@hku.hk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100777">10.1016/j.xpro.2021.100777</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Dissecting the function of proteins’ post-translational modifications (PTMs) is seriously hindered by the difficulty in obtaining the homogeneous protein with the PTMs of interest. Chemical protein synthesis offers a great potential to overcome this limitation. Here, a detailed protocol is introduced for chemical synthesis of HMGA1a protein with site-specific modifications via Ser/Thr ligation strategy, by which we can systematically study the function of the triple phosphorylation (3pSer) in the HMGA1a acidic tail.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00483-4/sref1">Wei et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Proteins with PTMs can be produced by ligation-assisted chemical synthesis</li>
<li>Phosphorylated HMGA1a proteins were chemical synthesized</li>
<li>Binding partners can be profiled by streptavidin-biotin affinity pull-down assay</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin (Thompson and Muir, 2019)</h2>
<p>Over the past decades, a variety of new and refined proteomics technologies have emerged, which significantly improve our abilities to profile the PTMs spatiotemporally. Nowadays, Identification of PTMs even has become commercially available. In addition, the public literature databases (e.g., PubMed, Uniprot) enable us to find the interested PTMs on proteins of interest. To study the interested PTMs, accessing homogenous proteins with site-specific PTMs is crucial. Direct extraction from cells or enzymatic PTM installation on the recombinant protein still remains challenging. Instead, protein chemical synthesis has emerged as an effective way to introduce PTMs site-selectively and homogeneously.</p>
<p>Solid-phase peptide synthesis (SPPS) can efficiently prepare the polypeptides chemically without complicated setups. However, due to the truncated sequences accumulated from the incomplete coupling, SPPS generally is not used to prepare peptides with more than 50 amino acids. Despite many efforts being made to expand the SPPS′ capability, convergent synthesis, which bypasses the formidable challenge of SPPS, has become the most common way to accessing long peptide sequences (up to proteins). Specifically, chemical peptide ligation technologies, such as native chemical ligation (NCL) and its variants, KAHA ligation, Serine/Threonine ligation (STL), and cysteine/penicillamine ligation (CPL), can selectively and specifically forge a natural peptidic bond between the C- and N-terminus of two unprotected peptides. Therefore, a protein containing noncanonical elements can be re-constructed by stitching peptides together via several rounds of peptide ligation (<a href="#bib2">Tan et al., 2020</a>).</p>
<p>Of the ligation technologies developed to date, the C-terminal salicylaldehyde ester-based chemoselective condensation, STL, has been widely used for synthesis of cyclic peptides and proteins. (<a href="#bib3">Zhang et al., 2013</a>; <a href="#bib2">Tan et al., 2020</a>). This reaction occurs between the N-terminal serine or threonine or cysteine peptide fragment and the peptide fragment with C-terminal salicylaldehyde ester to afford an <i>N,O/S</i>-benzylidene-acetal-linked intermediate in a pyridine/acetic acid mixture. After acidolysis with TFA/H<sub>2</sub>O, the native amide bond linkage will be formed at the ligation site (<a href="#bib2">Tan et al., 2020</a>). It is noteworthy that proteins with more than 400 residues are still not routinely reachable by chemical total synthesis, since it requires many rounds of ligation. Finally, the total yield will be extremely low.</p>
<p>In this protocol, we describe the procedure to chemically produce the homogenous HMGA1a with the triple phosphorylation on its acidic tail by STL. HMGA1a with other PTMs also can be achieved by this protocol (e.g., methylation and acetylation). High-Mobility Group A1a (HMGA1a) is one of non-histone chromatin proteins and involved in many cell nuclear processes (gene transcription, DNA replication and repair, heterochromatin organization, etc.) (<a href="#bib4">Reeves, 2001</a>). Accumulating studies have found that like Histone protein, HMGA1a carries a variant of protein posttranslational modifications (PTMs) including methylation, acetylation, phosphorylation and so on (<a href="#fig1">Figure 1</a>A). However, unlike various defined functions on the histone protein, most of PTMs’ functions on HMGA1 still remain to be illustrated (<a href="#bib5">Zhang and Wang, 2008</a>). Here, we focused on the triple phosphorylation on the acidic tail.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Analysis of ligation sites</div>
<p>(A) PTMs of HMGA1a and disconnection sites for chemical ligation are shown. The synthesis involves C-to-N sequential Ser/Thr ligations. This figure is reprinted with permission from <a href="#bib1">Wei et al., 2021</a>.</p>
<p>(B) Effect of C-terminal amino acid on STL by a model study. The reaction conversions were calculated after 2 h. The Lys, Asp, and Glu ester is hard to make. ∗: Cys is SStBu protected. The data was adopted from <a href="#bib6">Liu and Li, 2018</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-P53 (1:200)</td>
<td>Santa Cruz</td>
<td>Cat# sc-126; RRID: AB_628082</td>
</tr>
<tr>
<td>Anti-mouse IgG (1:2000)</td>
<td>Cell Signaling Technology</td>
<td>Cat# 7076S; RRID: AB_330924</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>2-Chlorotrityl chloride resin</td>
<td>CSBio</td>
<td>Cat# AR-FCRX-0010</td>
</tr>
<tr>
<td>Fmoc-Ala-OH</td>
<td>CSBio</td>
<td>Cat# AR-FALA-0010</td>
</tr>
<tr>
<td>Fmoc-Ser(tBu)-OH</td>
<td>CSBio</td>
<td>Cat# AR-FSER-0010</td>
</tr>
<tr>
<td>Fmoc-Glu(OtBu)-OH</td>
<td>CSBio</td>
<td>Cat# AR-FGLU-0010</td>
</tr>
<tr>
<td>Fmoc-Thr(tBu)-OH</td>
<td>CSBio</td>
<td>Cat# AR-FTHR-0010</td>
</tr>
<tr>
<td>Fmoc-Lys(Boc)-OH</td>
<td>CSBio</td>
<td>Cat# AR-FLYS-0010</td>
</tr>
<tr>
<td>Fmoc-Gln(Trt)-OH</td>
<td>CSBio</td>
<td>Cat# AR-FGLN-0010</td>
</tr>
<tr>
<td>Fmoc-Pro-OH</td>
<td>CSBio</td>
<td>Cat# AR-FPRO-0010</td>
</tr>
<tr>
<td>Fmoc-Leu-OH</td>
<td>CSBio</td>
<td>Cat# AR-FLEU-0010</td>
</tr>
<tr>
<td>Fmoc-Asp(OtBu)-OH</td>
<td>CSBio</td>
<td>Cat# AR-FASP-0010</td>
</tr>
<tr>
<td>Fmoc-Gly-OH</td>
<td>CSBio</td>
<td>Cat# AR-FGLY-0010</td>
</tr>
<tr>
<td>Fmoc-Arg(Pbf)-OH</td>
<td>CSBio</td>
<td>Cat# AR-FARG-0010</td>
</tr>
<tr>
<td>Fmoc-Val-OH</td>
<td>CSBio</td>
<td>Cat# AR-FVAL-0010</td>
</tr>
<tr>
<td>Fmoc-Asn(Trt)-OH</td>
<td>CSBio</td>
<td>Cat# AR-FASN-0010</td>
</tr>
<tr>
<td>Fmoc-Ile-OH</td>
<td>CSBio</td>
<td>Cat# AR-FILE-0010</td>
</tr>
<tr>
<td>Boc-Ser(tBu)-OH</td>
<td>CSBio</td>
<td>Cat# AR-BSER-0006</td>
</tr>
<tr>
<td>Fmoc-Gly(Hmb)-OH</td>
<td>Santa Cruz</td>
<td>Cat# sc-285793</td>
</tr>
<tr>
<td>Fmoc-Ser(HPO3Bzl)-OH</td>
<td>Bidepharm</td>
<td>Cat# BD18628-5g</td>
</tr>
<tr>
<td>Hexafluorophosphate azabenzotriazole tetramethyl uronium (HATU)</td>
<td>GL Biochem</td>
<td>Cat# 00703</td>
</tr>
<tr>
<td>Dichloromethane (DCM)</td>
<td>Fisher Chemical</td>
<td>Cat# D/1852/17</td>
</tr>
<tr>
<td>Trifluoroacetic acid (TFA)</td>
<td>J&amp;K</td>
<td>Cat# 101398</td>
</tr>
<tr>
<td>2,2,2-Trifluoroethanol (TFE)</td>
<td>J&amp;K</td>
<td>Cat# 269931</td>
</tr>
<tr>
<td>N,N-Dimethylformamide (DMF)</td>
<td>AQA</td>
<td>Cat# DF-1161-2500</td>
</tr>
<tr>
<td>Benzotriazol-1-yloxytripyrrolidinophosphonium hexafluorophosphate (PyBOP)</td>
<td>Chem-Impex</td>
<td>Cat# 02276</td>
</tr>
<tr>
<td>N, N-Diisopropylethylamine (DIEA)</td>
<td>Energy Chem</td>
<td>Cat# W3200145000</td>
</tr>
<tr>
<td>Acetic acid (AcOH)</td>
<td>DUKSAN</td>
<td>Cat# 3839</td>
</tr>
<tr>
<td>CH3CN</td>
<td>DUKSAN</td>
<td>Cat# 2957</td>
</tr>
<tr>
<td>Piperidine</td>
<td>Merck</td>
<td>Cat# 8.22299.0500</td>
</tr>
<tr>
<td>Phenol</td>
<td>Energy Chem</td>
<td>Cat# W6105175000</td>
</tr>
<tr>
<td>Hydroxybenzotriazole (HOBt)</td>
<td>GL Biochem</td>
<td>Cat# 00602</td>
</tr>
<tr>
<td>N-Methyl-2-pyrrolidone (NMP)</td>
<td>DUKSAN</td>
<td>Cat# 674</td>
</tr>
<tr>
<td>4-Dimethylaminopyridine (DMAP)</td>
<td>Energy Chem</td>
<td>Cat# B040170</td>
</tr>
<tr>
<td>Triethylamine (Et3N)</td>
<td>Energy Chem</td>
<td>Cat# B0100655000</td>
</tr>
<tr>
<td>Acetic anhydride (Ac2O)</td>
<td>Energy Chem</td>
<td>Cat# A0105110250</td>
</tr>
<tr>
<td>Triisopropylsilane (Tips)</td>
<td>TCI</td>
<td>Cat# T1533</td>
</tr>
<tr>
<td>Diethyl ether</td>
<td>RCI Labscan</td>
<td>Cat# AR1044B</td>
</tr>
<tr>
<td>Potassium cyanide (KCN)</td>
<td>Sigma</td>
<td>Cat# 207810-25G</td>
</tr>
<tr>
<td>Pyridine</td>
<td>Sigma</td>
<td>Cat# 270970-1L</td>
</tr>
<tr>
<td>Ninhydrin</td>
<td>Energy Chemical</td>
<td>Cat# A050135</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Scharlab</td>
<td>Cat# ET00232500</td>
</tr>
<tr>
<td>Hydrazine monohydrate</td>
<td>TCI</td>
<td>Cat# H0172</td>
</tr>
<tr>
<td>Hexane</td>
<td>AQA</td>
<td>Cat# HX-1251-020L</td>
</tr>
<tr>
<td>Triethylamine (TEA)</td>
<td>Energy Chemical</td>
<td>Cat# B010065</td>
</tr>
<tr>
<td>Diethanolamine (DEA)</td>
<td>Sigma</td>
<td>Cat# 471216-2.5L</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>Transgen</td>
<td>Cat# FG701-01</td>
</tr>
<tr>
<td>DMEM</td>
<td>Thermo Fisher</td>
<td>Cat# 11995-040</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Thermo Fisher</td>
<td>Cat# 10270</td>
</tr>
<tr>
<td>Trypsin</td>
<td>Transgen Biotech</td>
<td>Cat# FG301-01</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (10,000 U/mL)</td>
<td>Thermo Fisher</td>
<td>Cat# 15140122</td>
</tr>
<tr>
<td>Pierce Protease Inhibitor Mini Tablets, EDTA-free</td>
<td>Thermo Fisher</td>
<td>Cat# A32955</td>
</tr>
<tr>
<td>NP-40</td>
<td>Beyotime</td>
<td>Cat# ST366</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma</td>
<td>Cat# G5516</td>
</tr>
<tr>
<td>BCA assay</td>
<td>Beyotime</td>
<td>Cat# P0012</td>
</tr>
<tr>
<td>High Capacity Streptavidin Agarose</td>
<td>Thermo Fisher</td>
<td>Cat# 20359</td>
</tr>
<tr>
<td>NaCl</td>
<td>Sigma</td>
<td>Cat# S3014</td>
</tr>
<tr>
<td>Tris(hydroxymethyl)aminomethane (Tris)</td>
<td>Macklin</td>
<td>Cat# T822117</td>
</tr>
<tr>
<td>PVDF membrane</td>
<td>Immobilon</td>
<td>Cat# ISEQ00010</td>
</tr>
<tr>
<td>HMGA1a</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>Biotin-HMGA1a</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>HMGA1a-3pSer</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>Biotin-HMGA1a-3pSer</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HEK293T</td>
<td>HUEN, Michael Shing-Yan Laboratory, HKU</td>
<td>N/A</td>
</tr>
<tr>
<td>MCF-7</td>
<td>Chi-Ming Che Laboratory, HKU</td>
<td>N/A</td>
</tr>
<tr>
<td>NB-4</td>
<td>Chi-Ming Che Laboratory, HKU</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>UniDec</td>
<td>University of Oxford</td>
<td>Version 1.0.10 (<a href="https://github.com/michaelmarty/UniDec/releases">https://github.com/michaelmarty/UniDec/releases</a>)</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>LC-MS (ACQUITY UPLC H-Class PLUS System)</td>
<td>Waters</td>
<td>Cat# 720003268en</td>
</tr>
<tr>
<td>Preparative HPLC</td>
<td>Waters</td>
<td> </td>
</tr>
<tr>
<td>Hei-VAP Core Rotary Evaporator</td>
<td>Heidolph</td>
<td>Cat# 02-321-081</td>
</tr>
</tbody>
</table>
<p class="table-legend">[Resin and amino acids from CSBio are recommended for peptide synthesis]</p>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Solution preparation</h3>
<table id="undtbl1">
<caption>[10 mM KCN solution]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Potassium cyanide (KCN)</td>
<td>10 mM</td>
<td>16.5 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td><b>n/a</b></td>
<td>25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[Can be store at 25°C up to one year]</p>
<table id="undtbl2">
<caption>[Kaiser test solution A]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KCN solution (10 mM)</td>
<td>0.2 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>Pyridine</td>
<td>98%</td>
<td>49 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[Can be store at 25°C up to one year]</p>
<table id="undtbl3">
<caption>[Kaiser test solution B]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ninhydrin</td>
<td>5%</td>
<td>1 g</td>
</tr>
<tr>
<td>Ethanol</td>
<td><b>n/a</b></td>
<td>20 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[Can be store at 25°C up to one year]</p>
<table id="undtbl4">
<caption>[Kaiser test solution C]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Phenol</td>
<td><b>n/a</b></td>
<td>20 g</td>
</tr>
<tr>
<td>Ethanol</td>
<td><b>n/a</b></td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[Can be store at 25°C up to one year]</p>
<div class="critical">
<span class="critical-title">Critical:</span> Freshly mix 10 μL solution A, 10 μL solution B, and 10 μL solution C together in a 1.5 mL Eppendorf tube before use. And take several grains of resin into 30 μL Kaiser solution. Followed by heating for 3 min at 110°C. If the coupling is finished, the solution will remain yellow; otherwise, the solution will turn to a dark blue color.</div>
<table id="undtbl5">
<caption>[Lysis buffer]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (1M); pH 7.4</td>
<td>20 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>NaCl (1.5M)</td>
<td>150 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>NP-40</td>
<td>1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>PMSF (100 mM)</td>
<td>1 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>43 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[Can be store at 4°C for up to one year, Pierce Protease inhibitor Mini Tablet (1 tablet for 5 mL lysis buffer) will be added before use]</p>
<table id="undtbl6">
<caption>[4 <b>×</b> loading buffer]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (1M); pH 6.8</td>
<td>200 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>SDS</td>
<td>8%</td>
<td>4 g</td>
</tr>
<tr>
<td>DTT</td>
<td>40 mM</td>
<td>3.1 g</td>
</tr>
<tr>
<td>Bromphenol blue</td>
<td>0.4%</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>40%</td>
<td>20 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>20 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[Can be store at 25°C up to one year, can be diluted accordingly]</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Analyze the ligation sites and synthesis design</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 h]</div>
<p>This section describes the sequence analysis for disconnection sites.</p>
<ol>
<li>Analyze ligation sites.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> HMGA1a is rich with Ser and Thr residues, which provides multiple options for conducting peptide ligation. Therefore, STL strategy was employed to generate the full length HMGA1a (<a href="#fig1">Figure 1</a>A).</div>
<div class="note">
<span class="note-title">Note:</span> The C-terminal Gly, Ala, Ser, Gln, Thr, Phe, and Cys(SStBu) salicylaldehyde esters generally give higher conversions under the standard STL ligation conditions (<a href="#fig1">Figure 1</a> B), Therefore, we chose G37ˆT38 and G62ˆS63 as the disconnection sites for peptide ligation.</div>
<div class="note">
<span class="note-title">Note:</span> Before peptide fragment synthesis, we can use some peptide hydrophobicity /hydrophilicity analysis tools (e.g. <a href="https://www.peptide2.com/N_peptide_hydrophobicity_hydrophilicity.php">https://www.peptide2.com/N_peptide_hydrophobicity_hydrophilicity.php</a>) to evaluate the hydrophobicity, as very hydrophobic peptide sequence will aggregate during peptide elongation and results in the failure of peptide preparation. If so, it is better to choose another ligation site with less hydrophobic fragments. Alternatively, removable N-(2-hydroxy-4-methoxy benzyl) (Hmb) tag can be used to improve synthesis of the hydrophobic peptides.</div>
<ol start="2">
<li>Protection design.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> HMGA1a synthesis involved three fragments via two ligations in the C-to-N direction; thus, the N-terminal Thr of the fragment 2 was temporarily protected by the Fmoc group. Otherwise, the N-terminus of fragment 2 may also react during ligation reaction.</div>
<h3 id="sec3.2">HMGA1a fragment 3 preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 weeks]</div>
<p>This section describes the preparation of HMGA1a (63–106) peptide fragment along with the removable N-(2-hydroxy-4-methoxy benzyl) (Hmb) tag (<a href="#fig2">Figure 2</a>). Peptide fragments can be readily produced by peptide synthesizer (e.g., CEM Liberty Blue). Alternatively, we can prepare them manually via solid phase peptide synthesis. Typically, one equivalent 2-chlorotrityl chloride resin reacts with four equivalents of the first amino acid and coupling reagent in DMF. HATU/DIEA were used as the coupling reagents.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. The sequence of fragment 3</div>
<p>This figure is reprinted with permission from <a href="#bib1">Wei et al., 2021</a>.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Hmb tag is introduced to increase the peptide solubility.</div>
<ol start="3">
<li>Prepare peptide fragment 3 following the standard Fmoc-SPPS protocol (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. The overview of SPPS scheme</div>
<p>(A) SPPS flow chart.</p>
<p>(B) The overview of the reaction vessel for manual SPPS.</p>
<p>(C) Set-up of coupling, deprotection, and wash step.</p>
</figcaption>
</figure>
<ol type="a">
<li>Resin Swelling and first amino acid coupling.
<ol type="i">
<li>Weight 200 mg 2-chlorotrityl chloride resin (resin loading: 0.4 mmol/g) into the reaction vessel.</li>
<li>Add 5mL anhydrous DCM to vessel and shake for 15 min.</li>
<li>Dry the resin on a vacuum pump.</li>
<li>Add 5mL anhydrous DCM, the first amino acid (Fmoc protected) of the C-terminus of the fragment (0.32 mmol) and 56 μL DIEA (0.32 mmol). And shake at 25°C for 2 h to generate the first amino acid anchored resin.</li>
<li>Dry the resin on a vacuum pump.</li>
<li>Wash by 10 mL DCM and 10 mL DMF for 3 times, respectively.</li>
<li>The obtained resin is swelled in 5 mL DCM containing 100 μL methanol and 56 μL DIEA (0.32 mmol).</li>
<li>The mixture is shaken for 30 min.</li>
<li>The resin is washed by DCM for 3 times and dry for next step.</li>
</ol>
</li>
<li>Deprotection: Add 5 mL 20% piperidine in DMF (v/v) to the reaction vessel. Shake for 15 min at 25°C.</li>
<li>Coupling: Weight corresponding (0.32 mmol) Fmoc protected amino acid, 121.6 mg HATU (0.32 mmol) in a 15 mL conical tube, dissolved by 5 mL DMF and followed by adding 56 μL DIEA (0.32 mmol). Transfer the mixture into the reaction vessel. Shake for 2 h at 25°C.
<div class="note">
<span class="note-title">Note:</span> Coupling time can be elongate to 12 h.</div>
</li>
<li>Wash: Dry the resin and wash by 10 mL DMF <b>×</b> 3 times, 10 mL DCM <b>×</b> 3 times and 10 mL DMF <b>×</b> 3 times. Finally, the resin is dried for next step.</li>
<li>Deprotection: Add 5 mL 20% piperidine in DMF (v/v) to the reaction vessel. Shake for 15 min at 25°C.</li>
<li>Wash: Dry the resin and wash by 10 mL DMF × 3 times, 10 mL DCM × 3 times and 10 mL DMF × 3 times. Finally, the resin is dried for next amino acid coupling (go to step c).
<div class="note">
<span class="note-title">Note:</span> Repeated step c-f until all amino acids are coupled and then proceed to step g.</div>
</li>
<li>Cleavage from resin.
<ol type="i">
<li>Achieving side chain unprotected peptide: (Before the cleavage step, the peptide bound resin should wash by 10 mL DCM for 3 times) After completion of the synthesis, cleavage and global side chain deprotection can be achieved in the same manner by adding 10 mL cleavage cocktail A (TFA/H<sub>2</sub>O/phenol (90/5/5, v/v/v)) and shaking for 1h at 25°C. After cleavage, the resin is filtered, and combined filtrates are blown-off under a stream of compressed air. The crude product is precipitated by 40 mL cold diethyl ether, and followed by centrifugation immediately at 4000 <i>g</i> for 5 min. Then the ether is subsequently decanted. The obtained solid is ready for HPLC purification.</li>
<li>Achieving sidechain full-protected peptide: (Before the cleavage step, the peptide bound resin should wash by 10 mL DCM for 3 times) Cleavage of peptide with sidechain protection can be achieved by adding 10 mL mild acidic cleavage cocktail B (CH<sub>2</sub>Cl<sub>2</sub>/2,2,2-trifluoroethanol (TFE)/AcOH (8/1/1, v/v/v)) and shaking for 1 h at 25°C. After cleavage, the resin is filtered, and the combined filtrates are evaporated under Rotavap. The resulting oily crude product is further dissolved by 10 mL DCM and 30 mL hexane, followed by evaporation under vacuum to co-evaporate all acetic acid with hexane. The co-evaporation is repeated for 5 times to give a powder, which can be directly used for the preparation of the C-terminal peptide salicylaldehyde ester.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The coupling from Glu92-Glu95 need to be monitored by the Kaiser test to make sure the coupling is finished as the continuous Glu residues cause strong steric hindrance and is difficult to couple. Also, decrease the loading of the resin to 0.3 mmol/g to reduce the steric hindrance during the coupling.</div>
<div class="note">
<span class="note-title">Note:</span> Coupling Fmoc-Ser(HPO<sub>3</sub>Bzl)-OH at Ser98, Ser101, and Ser102 using HATU/HOBt (4 equivalent) in DMF/NMP (1/1, v/v) as coupling reagent.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Coupling Fmoc-Gly(Hmb)-OH at Gly96 for 12 h. Fmoc-Gly(Hmb)-OH is commercially available, or refer to the synthetic route at <a href="#bib1">Wei et al., 2021</a>.</div>
<ol start="4">
<li>After coupling of the last Boc-Ser(tBu)-OH, treat resin with 10 mL 2% hydrazine in DMF (v/v) and shake for 30 min to liberate hydroxyl group on Hmb.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The hydroxyl group of Hmb will form an ester bond with the amino acids’ carboxyl group during each coupling, but it can be removed in the following deFmoc step by piperidine/DMF. In the last coupling, we do not perform the deprotection as we use Boc-Ser(tBu)-OH. Boc can be removed at cleavage step 8 without additional treatment. So, we need use hydrazine to remove the amino acids on Hmb. If the last amino acid is Fmoc protected, we need do the deFmoc step. Then, step 4 can be skipped.</div>
<ol start="5">
<li>Dry the resin by connecting the reaction vessel to vacuum pump and rinse by DCM.</li>
<li>Treat resin with 10 mL Ac<sub>2</sub>O/Et<sub>3</sub>N/DMAP (100 μL/300 μL/5 mg) in DCM and shake for 30 min to cap the hydroxyl group of Hmb.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Hmb will be cleaved by the following treatment of concentrated TFA if the hydroxyl group is unprotected.</div>
<ol start="7">
<li>Dry the resin by connecting the reaction vessel to vacuum pump and rinse by DCM.</li>
<li>Cleave the peptides from resin by treating resin with 10 mL TFA/H<sub>2</sub>O/Tips/Phenol (92.5/2.5/2.5/2.5, v/v/v/v) for 3 h.</li>
<li>The resin-cleavage solution mixture is filtered through the vessel filter frit by directly pushing the plunger into a 50 mL centrifugation tube, and the cleavage solution is concentrated under a stream of compressed air to give the crude product.</li>
<li>Crude product is further precipitated by adding 40mL cold ether, followed by centrifugation immediately (4000 <i>g</i>, 5 min at 25°C).</li>
<li>Repeat step 10 twice.</li>
<li>The solid is dissolved in 10 mL 10% CH<sub>3</sub>CN/H<sub>2</sub>O. The solution is filtered by 0.45 μm membrane filter to give a clear solution.</li>
<li>Pipette 20 μL of the step 12 solution and dilute it to 200 μL by adding 180 μL 10% ACN/H<sub>2</sub>O. The sample is filtered by 0.45 μm membrane filter to give a clear filtrate which is place into a UPLC sample vial.</li>
<li>The step 13 sample vial is further place into a UPLC-MS sample chamber (maintaining at 15 <sup>o</sup>C) at corresponding position. Create a new injection method in the UPLC-MS Masslynx software, the detail parameters are as follow: MS File is 250-3072 8 min, MS Tune File is MS Tune 30 V, and the Inlet File are 15–60% 8 min (15–60% ACN/H<sub>2</sub>O), 10–50% 8 min (10–50% ACN/H<sub>2</sub>O), and 10–40% 8 min (10–40% ACN/H<sub>2</sub>O). The gradient which can give the best resolution for the sample is determined.</li>
<li>After Step 14, the 10–40% 8 min (10–40% ACN/H<sub>2</sub>O) gives the best resolution of the sample in the UPLC-MS, each of sample component has shown a symmetric peak in the UPLC-MS UV spectrum.</li>
<li>The reverse phase column is connected to a Waters HPLC system. After that, a 10–40% CH<sub>3</sub>CN/H<sub>2</sub>O gradient over 40min method, according to the step 15, (UV absorption 205 nm, flow rate 20 mL/min) is loaded for 10 mL, and the pressure of the system should be maintained between 115 to 125 Psi.</li>
<li>The step 12 solution is injected into the loop of HPLC system, then trigger its sample injector and click the “real time” button of the HPLC software (MassLynx) to give a UV monitoring curve.</li>
<li>Collect all the HPLC elution by 15 mL tube based on the UV peak, and 20 μL of sample aliquot, taking from each of the 15 mL tube, are injected into the UPLC-MS.</li>
<li>Those tubes with one peak and desired mass spectrum are collected and subjected to liquid nitrogen freezing and lyophilization to give the product as white powder.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The freeze-dried product can be store at −80°C for several months.</div>
<h3 id="sec3.3">HMGA1a fragment 2 preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 weeks]</div>
<p>This section describes the preparation of HMGA1a (38–62) peptide fragment along with the N-terminal Fmoic and C-terminal salicylaldehyde ester (<a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. The sequence of fragment 2</div>
<p>This figure is reprinted with permission from <a href="#bib1">Wei et al., 2021</a>.</p>
</figcaption>
</figure>
<ol start="20">
<li>Prepare peptide fragment 2 following standard Fmoc-SPPS protocol (<a href="#fig3">Figure 3</a>).</li>
<li>After the coupling last Fmoc-Thr(tBu)-OH, the resin obtain from the SPPS is dried by vacuum and rinsed with DCM.</li>
<li>Treat resin with 10 mL DCM/AcOH/TFE (8/1/1, v/v/v) for 1 h.</li>
<li>The resin is filtered, and the cleavage solution is evaporated under reduce pressure.</li>
<li>The resulting oily crude product is further dissolved DCM and hexane, followed by evaporation under vacuum to remove all acetic acid.</li>
<li>Repeated co-evaporation for 5 times to give solid powder.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The powder can be directly used for the C-terminal salicylaldehyde ester preparation without HPLC purification.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The solid product can be store at −80°C for several months.</div>
<ol start="26">
<li>Dissolved peptide/PyBOP/DIEA (1/3/6, mol/mol/mol) in DCM (final concentration of peptide: 8.5 mM) and stir for 10 min.</li>
<li>20 equivalent 2-(dimethoxymethyl)phenol is added and stir for 12 h.</li>
<li>The DCM of the reaction mixture is removed by evaporation to give a colorless oil.</li>
<li>Treat the above oil with 10 mL TFA/H<sub>2</sub>O/Phenol (95/2.5/2.5, v/v/v) for 1.5 h.</li>
<li>The cleavage solution is concentrated under a stream of compressed air.</li>
<li>Crude products are precipitated by 40 mL cold ether, followed by centrifugation immediately (4000 <i>g</i>, 5 min at 25°C).</li>
<li>The solid is dissolved in 10 mL 20% CH<sub>3</sub>CN/H<sub>2</sub>O. The solution is filtered by 0.45 μm membrane filter and check the ligation product by LC/MS as step 13–14. Then, purify the product by HPLC (20–50% CH<sub>3</sub>CN/H<sub>2</sub>O over 40 min) as step 16–19.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The freeze-dried product can be store at −80°C for several months.</div>
<h3 id="sec3.4">HMGA1a fragment 1 preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 weeks]</div>
<p>This section describes the preparation of HMGA1a (1–37) peptide fragment along with the C-terminal salicylaldehyde ester (<a href="#fig5">Figure 5</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. The sequence of fragment 1</div>
<p>This figure is reprinted with permission from <a href="#bib1">Wei et al., 2021</a>.</p>
</figcaption>
</figure>
<ol start="33">
<li>Prepare peptide fragment 1 following standard Fmoc-SPPS protocol (<a href="#fig3">Figure 3</a>).</li>
</ol>
<ol start="34">
<li>N-terminus coupling
<ol type="a">
<li>Install a biotin at N-terminus: after the coupling of last Fmoc-Ser-OH and the deprotection of its Fmoc protecting group, the D-biotin can be directly coupled after Ser1.
<div class="note">
<span class="note-title">Note:</span> The biotin coupling procedure is the same as the amino acid coupling.</div>
</li>
<li>Install an acetyl at N-terminus: after the coupling and deprotecting the last Fmoc-Ser-OH, the resin is treated with Ac<sub>2</sub>O/ Triethylamine (TEA) (1/1, mol/mol) in 5 mL DCM, and with 5% mol DMAP as the catalyst at 25°C for 2 h.</li>
</ol>
</li>
</ol>
<ol start="35">
<li>Dry the resin and rinse by DCM.</li>
<li>Treat resin with 10 mL DCM/AcOH/TFE (v/v/v, 8/1/1) for 1.5 h.</li>
<li>The procedure of installing the salicylaldehyde ester is the same as that of fragment 2 (step 23–31).</li>
<li>The solid is dissolved in 10 mL 10% CH<sub>3</sub>CN/H<sub>2</sub>O. The solution is filtered by 0.45 μm membrane filter to give a clear solution and check the ligation product by LC/MS as step 13–14. Then, purify the product by HPLC (10–40% CH<sub>3</sub>CN/H<sub>2</sub>O over 40 min) as step 16–19.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The freeze-dried product can be store at −80°C for several months.</div>
<h3 id="sec3.5">Ligation between HMGA1a fragment 2 and 3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [9 h]</div>
<p>This section describes the first round ligation between fragment 2 and 3 (<a href="#fig6">Figure 6</a>A).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. LC/MS analysis of ligation product between fragment 2 and 3</div>
<p>(A) The sequence of ligation product.</p>
<p>(B) The UV trace of the ligation mixture after acidolysis step (the sample was taken from step 44). We can see the ligation product and unconsumed fragment 2 and 3 from the UV trace. ∗ the C-terminal salicylaldehyde ester will be hydrolyzed slowly during the ligation to give the fragment 2 without C-terminal salicylaldehyde ester.</p>
<p>(C) The UV trace of the ligation product after HPLC purification (the sample was token after step 45). The purified product was shown as one peak.</p>
<p>(D) The mass spectra of the peak in UV trace showed the correct mass of product. The molecular weight of desired product (C<sub>344</sub>H<sub>570</sub>N<sub>105</sub>O<sub>123</sub>P<sub>3</sub>) is 8237.88 Da. P<sub>1</sub> [M+4H]<sup>4+</sup> m/z = 2060.48; P<sub>2</sub> [M+5H]<sup>5+</sup> m/z = 1648.58; P<sub>3</sub> [M+6H]<sup>6+</sup> m/z = 1373.98; P<sub>4</sub> [M+7H]<sup>7+</sup> m/z = 1177.84; P<sub>5</sub> [M+8H]<sup>8+</sup> m/z = 1030.74; P<sub>6</sub> [M+9H]<sup>9+</sup> m/z = 916.32; P<sub>7</sub> [M+10H]<sup>10+</sup> m/z = 824.79. Found: 2060.89; 1648.56; 1374.25; 1177.84; 1030.78; 916.49; 824.72.</p>
<p>This figure is reprinted with permission from <a href="#bib1">Wei et al., 2021</a>.</p>
</figcaption>
</figure>
<ol start="39">
<li>Weight 21 mg (1 equivalent) of fragment 3 and 18 mg (1.5 equivalent) of fragment 2 in a 4 mL glass bottle.</li>
<li>400 μL pyridine/AcOH (1/12, mol/mol) is added (the final concentration of fragment 1 is 10 mM). The reaction is stirred for 6 h at 25°C.</li>
<li>Add reaction solution into 40 mL cold ether, followed by centrifugation immediately (4000 <i>g</i>, 5 min at 25°C).</li>
<li>Solid is further washed twice by 40 mL cold ether.</li>
<li>Treat solid with 400 μL TFA/H<sub>2</sub>O/Tips (95/2.5/2.5, v/v/v) for 30 min.</li>
<li>Add reaction solution into 40 mL cold ether, followed by centrifugation immediately (4000 <i>g,</i> 5 min at 25°C).</li>
<li>The solid is dissolved in 10 mL 5% CH<sub>3</sub>CN/H<sub>2</sub>O. The solution is filtered by 0.45 μm membrane filter and check the ligation product by LC/MS (<a href="#fig6">Figure 6</a>B) as step 13–14. Then, purify the product by HPLC (5–40% CH<sub>3</sub>CN/H<sub>2</sub>O over 40 min) as step 16–19 (<a href="#fig6">Figures 6</a>C and 6D).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The freeze-dried product can be store at −80°C for several months.</div>
<h3 id="sec3.6">DeAc on Hmb of fragment (2+3)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2.5 h]</div>
<p>This section describes the removal of acetyl group on Hmb tag.</p>
<ol start="46">
<li>Dissolve 17 mg (1 equivalent) fragment (2+3) in 420 μL H<sub>2</sub>O (final concentration of peptide: 5 mM).</li>
<li>Add 3.4 μL hydrazine monohydrate (30 equivalent) into solution and stir for 2 h at 25°C.</li>
</ol>
<h3 id="sec3.7">DeFmoc on Thr38 fragment (2+3)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [4 h]</div>
<p>This section describes the removal of Fmoc on Thr38 for next round ligation.</p>
<ol start="48">
<li>Directly add 378 μL of H<sub>2</sub>O/CH<sub>3</sub>CN (1/1, v/v) + 42 μL DEA into solution and stir for another 2 h at 25°C.</li>
<li>The reaction is quenched by adding 8.4 μL TFA and the solution is filtered by 0.45 μm membrane filter and purified by HPLC (5–40% CH<sub>3</sub>CN/H<sub>2</sub>O over 40 min) as step 16–19. Finally, the product is freeze-dried.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The freeze-dried product can be store at −80°C for several months.</div>
<h3 id="sec3.8">Ligation between HMGA1a fragment 1 and (2+3)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [6 h]</div>
<p>This section describes the second round ligation between fragment 1 and (2+3).</p>
<ol start="50">
<li>Weight 5 mg (1 equivalent) of fragment (2+3) and 4.5 mg (1.5 equivalent) of fragment 1 in a 1 mL glass vial.</li>
<li>100 μL pyridine/AcOH (1/12, mol/mol) is added (the final concentration of fragment 1 is 10 mM). The reaction is stirred for 3 h at 25°C.</li>
<li>Add reaction solution into 40 mL cold ether, followed by centrifugation immediately (4000 <i>g</i>, 5 min at 25°C).</li>
<li>Solid is further washed twice by 40 mL cold ether.</li>
<li>Treat solid with 100 μL TFA/H<sub>2</sub>O/Tips (95/2.5/2.5, v/v/v) for 30 min.</li>
<li>Add reaction solution into 40 mL cold ether, followed by centrifugation immediately (4000 <i>g</i>, 5 min at 25°C).</li>
<li>The solid is dissolved in 10 mL 5% CH<sub>3</sub>CN/H<sub>2</sub>O. The solution is filtered by 0.45 μm membrane filter and check the ligation product by LC/MS as step 13–14. Then, purify the product by HPLC (5–40% CH<sub>3</sub>CN/H<sub>2</sub>O over 40 min) as step 16–19 (<a href="#fig7">Figure 7</a>).
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. HMGA1a with triple phosphorylation were chemical synthesized</div>
<p>(A) UV trace and mass spectra of Biotin-HMGA1a-3pSer. Up: UV trace of purified product was shown as one peak; Bottom: Mass spectra of the peak in UV trace showed the correct mass of product. The molecular weight of desired product (C<sub>492</sub>H<sub>841</sub>N<sub>164</sub>O<sub>177</sub>P<sub>3</sub>S) is 12011.09 Da. P<sub>1</sub> [M+6H]<sup>6+</sup> m/z = 2002.85; P<sub>2</sub> [M+7H]<sup>7+</sup> m/z = 1716.87; P<sub>3</sub> [M+8H]<sup>8+</sup> m/z = 1502.39; P<sub>4</sub>[M+9H]<sup>9+</sup> m/z = 1335.57; P<sub>5</sub> [M+10H]<sup>10+</sup> m/z = 1202.11; P<sub>6</sub> [M+11H]<sup>11+</sup> m/z = 1092.92; P<sub>7</sub> [M+12H]<sup>12+</sup> m/z = 1001.92; P<sub>8</sub>[M+13H]<sup>13+</sup> m/z = 924.93; P<sub>9</sub> [M+14H]<sup>14+</sup> m/z = 858.94; P<sub>10</sub> [M+15H]<sup>15+</sup> m/z = 801.74; P<sub>11</sub> [M+16H]<sup>16+</sup> m/z = 750.69; P<sub>12</sub>[M+17H]<sup>17+</sup> m/z = 707.54; P<sub>13</sub> [M+18H]<sup>18+</sup> m/z = 668.28. Found: 2003.15; 1716.89; 1502.26; 1336.07; 1202.14; 1093.01; 1002.08; 925.04; 859.10; 801.87; 751.75; 707.72; 668.70.</p>
<p>(B) UV trace and mass spectra of HMGA1a-3pSer. Up: UV trace of purified product was shown as one peak; Bottom: Mass spectra of the peak in UV trace showed the correct mass of product. The molecular weight of desired product (C<sub>484</sub>H<sub>829</sub>N<sub>162</sub>O<sub>176</sub>P<sub>3</sub>)molecular weight is 11826.84 Da. P<sub>1</sub> [M+6H]<sup>6+</sup> m/z = 1972.11; P<sub>2</sub> [M+7H]<sup>7+</sup> m/z = 1690.52; P<sub>3</sub> [M+8H]<sup>8+</sup>m/z = 1479.33.39; P<sub>4</sub> [M+9H]<sup>9+</sup> m/z = 1315.07; P<sub>5</sub> [M+10H]<sup>10+</sup> m/z = 1183.67; P<sub>6</sub> [M+11H]<sup>11+</sup> m/z = 1076.15; P<sub>7</sub> [M+12H]<sup>12+</sup> m/z = 986.55; P<sub>8</sub> [M+13H]<sup>13+</sup> m/z = 910.74; P<sub>9</sub> [M+14H]<sup>14+</sup> m/z = 845.76; P<sub>10</sub> [M+15H]<sup>15+</sup> m/z = 789.44; P<sub>11</sub> [M+16H]<sup>16+</sup> m/z = 740.17; P<sub>12</sub> [M+17H]<sup>17+</sup> m/z = 696.69. Found: 1971.82; 1690.72; 1479.40; 1315.16; 1183.76; 1076.25; 986.76; 910.82; 845.80; 789.59; 740.40; 696.80.The raw data of mass spectrum (txt format) was loaded on UniDec for deconvolution to generate the mass of product (Bin Every: 1.0 Da; Sample Mass Every: 1.0 Da; Peak FWHM: 1.0 Th; Peak Detection Range: 5.0 Da; Peak Detection Threshold: 0.1)</p>
<p>This figure is reprinted with permission from <a href="#bib1">Wei et al., 2021</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The freeze-dried product or stock solution by dissolving product in H<sub>2</sub>O at 10 mg/mL can be store at −80°C for several months.</div>
<div class="note">
<span class="note-title">Note:</span> The biotin-HMGA1a (wt) can be prepared in the same way.</div>
<h3 id="sec3.9">Pull-down experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 day]</div>
<p>This section describes the pull-down experiment by synthetic biotinylated HMGA1a proteins.</p>
<ol start="57">
<li>Prepare 3 plates (10 cm) of cells with 90% confluency.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As we will detect the P53 using according antibody, we used HEK293T, NB-4, and MCF-7 cell lines. We can use different cell lines for different purposes.</div>
<ol start="58">
<li>Rinse cell by 2 mL pre-warm PBS. Repeat twice.</li>
<li>Lyse cells on ice by adding 1 mL cold lysis buffer with protease inhibitor for each plate. Scratch and incubate for 10 min.</li>
<li>Collect cell lysate and centrifuge at 13000 <i>g</i> for 10 min at 4°C.</li>
<li>Carefully collect the supernatant by pipette. Alternatively, the bottom pellet can be resuspended by 100 μL lysis buffer and subject several seconds of sonication and repeat step 60 and combine the cell lysate together.</li>
<li>Quantify the concentration of cell lysate by BCA assay. Adjust the concentration to 2 mg/mL.</li>
<li>Set up the pull-down assay in 1.5 mL eppendorf tubes as below:</li>
</ol>
<table id="undtbl7">
<thead>
<tr>
<th>Tube entry</th>
<th>1</th>
<th>2</th>
<th>3</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cell lysate /mg</td>
<td>2</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>Biotin-HMGA1a /μg</td>
<td>-</td>
<td>20</td>
<td>-</td>
</tr>
<tr>
<td>Biotin-HMGA1a-3pSer /μg</td>
<td>-</td>
<td>-</td>
<td>20</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The amounts of biotin-HMGA1a probes can be varied empirically.</div>
<ol start="64">
<li>Incubate the samples for 12 h at 4°C with an overhead rotation.</li>
</ol>
<ol start="65">
<li>Prepare the streptavidin agarose resin.
<ol type="a">
<li>Carefully suspend 50 μL resin by 1 mL cold lysis buffer.</li>
<li>Centrifuge at 1000g for 3 min at 4°C.
<div class="critical">
<span class="critical-title">Critical:</span> The centrifugal machine should be set to soft acceleration and deacceleration from this step.</div>
</li>
<li>Carefully discard the supernatant by pipette.</li>
<li>Repeat step a-c twice.</li>
<li>After washing, the resin is finally suspended in 50 μL cold lysis buffer.</li>
</ol>
</li>
</ol>
<ol start="66">
<li>Apply 15 μL pre-equilibrated resin slurry to each pull-down tube and incubate for another 4 h at 4°C with overhead rotation.</li>
<li>Centrifuge at 1000g for 3 min at 4°C.</li>
<li>Carefully discard the supernatant by pipette.</li>
<li>Wash the resin twice carefully as step 65a-c.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The pull-down samples are ready for LC-MS/MS sample preparation to identify the new binders which specifically interact with HMGA1a or HMGA1a-3pSer. This protocol did not descripted it. More detail can be found at <a href="#bib1">Wei et al., 2021</a>.</div>
<ol start="70">
<li>Add 50 μL 1 × loading buffer and boil sample at 100°C for 10 min.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The boiled samples can be store at −80°C for several months.</div>
<ol start="71">
<li>SDS-PAGE/western blot analysis (<a href="#fig8">Figure 8</a>).
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/963-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Pull-down P53 from indicated cell lysates by biotin-HMGA1a-3pSer and biotin-HMGA1a</div>
<p>P53 was probed by anti-P53 and HMGA1a proteins were visualized by Coomassie blue staining. This figure is reprinted with permission from <a href="#bib1">Wei et al., 2021</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Spin down the resin and loading supernatant samples on SDS-PAGE.</div>
<div class="note">
<span class="note-title">Note:</span> Biotin-HMGA1a protein will appear at ∼16 kDa on SDS-PAGE as it contains many positive charged residues.</div>
<div class="note">
<span class="note-title">Note:</span> Mono-streptavidin protein can be released from resin after boiling and appear at ∼12 kDa on SDS-PAGE.</div>
<div class="note">
<span class="note-title">Note:</span> Pull-down of binders is one of applications of synthetic proteins. Besides, the synthetic proteins can be used in surface plasmon resonance (SPR) or isothermal titration calorimetry (ITC) to quantify the binding affinity.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Chemical protein synthesis could serve as a good tool to study the PTMs of interest. This protocol generalizes the procedure of chemical synthesis of HMGA1a by Ser/Thr ligation, which opens up more possibilities to identify other unknown function of PTMs on HMGA1a.</p>
<p>A successful reproduction of the synthesis should have relatively high purity (<a href="#fig7">Figure 7</a>) and good yield in milligram scale. Biotin-streptavidin mediated pull-down strategy has been the most common way to identify new binders from cell lysate. In our case, we could see P53 always bind HMGA1a instead of the triple phosphorylated variant in the chosen cell lines (<a href="#fig8">Figure 8</a>). This result proved that the triple phosphorylation could serve as regulatory switch to control the interaction between HMGA1a and P53.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Chemical protein synthesis provides an effective solution to introduce PTMs into proteins site-selectively. However, there are some gaps still awaiting to be filled. 1). Proteins with more than 400 residues are still not routinely reachable by chemical protein synthesis, since it requires many rounds of ligation. Finally, the total yield will be extremely low. In this case, a protein semi-synthesis approach can be applied, in which the synthetic parts with the desired PTMs and the recombinant parts can be ligated together. This approach significantly extends our capacity to generate the big protein in a cost-effective manner. More detailed information about this strategy can be found at <a href="#bib7">Thompson and Muir, 2019</a>. 2). The protein after chemical synthesis is a denatured form, which needs a refolding step to generate the functional protein. HMGA1a is an intrinsic disordered protein and does not require refolding operation. 3). Membrane proteins or hydrophobic sequence are currently very difficult to be synthesized because of the aggregation in SPPS and low solubility in ligation solvent. Try to change the ligation sites to adjust the hydrophobicity of peptide fragment or introduce solubilizing tags. 4). Delivery of a synthetic protein into cell is still challenging, which limits the application in cell for <i>in vivo</i> studies.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Peptides are not pure after step 3.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>In most cases, the truncation species are the main impurities. Apply the Kaiser test to monitor the coupling if needed. Double or triple coupling at potential troublesome sites (e.g., Arg, Glu) is a good way to solve this problem: after finishing the first-round coupling, wash resin and recouple the same amino acid for one more time.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Hydrolysis was observed after step 32 and 38.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>After the successful preparation of the peptide salicylaldehyde ester, HPLC purification should be performed immediately, as the salicylaldehyde ester is more stable in powder form.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Peptide fragments are not soluble in the ligation buffer in step 40 and 51.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Low solubility always causes low ligation efficiency. All HMGA1a fragments have good solubility in pyridine/AcOH (1/12, mol/mol) buffer. However, there may be some other peptides are not soluble. In these cases, we can vary the ratio of pyridine and AcOH (e.g., 1:1, 1:3, 1:9), or dissolve peptides in a small volume of Dimethylformamide (DMF) or Hexafluoro-2-propanol (HFIP) firstly, then dilute it with pyridine/AcOH buffer to the desired concentration. Normally, these cosolvents do not negatively affect the ligation efficiency.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Ligation efficiency is low when checking the ligation mixture at step 45 and 56.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>If the peptide fragments are not soluble well in ligation buffer, please refer problem 3 and try to dissolve the peptide fragment firstly.</p>
<p>If the peptide fragments are soluble, please further check the C-terminal amino acid of peptide ester part. Some amino acids will significantly hinder the Ser/Thr ligation because of their bulky sidechain (<a href="#fig1">Figure 1</a>B). If so, re-analyze the protein sequence and change the ligation sites.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Fail to detect the interested protein after pull-down experiment through western blot in step 71.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>In our case, we detected P53 in the chosen cell lines (<a href="#fig8">Figure 8</a>). However, we also failed to identify P53 in some cell lines (e.g., Hela and H1299), because these two cell lines barely express P53 without external stimuli. Therefore, it is always good to include the cell lysate input sample as a positive control for western blot detection.</p>
<p>If the expression level of interested proteins is normal or high, we need check the amount of probes, incubation time, binding buffer. Normally, more probes, longer incubation time and less salts in binding buffer will result in higher signal in western blot.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Wei, T., Liu, H., Chu, B., Blasco, P., Liu, Z., Tian, R., Li, D.X., and Li, X. (2021). Phosphorylation-regulated HMGA1a-P53 interaction unveils the function of HMGA1a acidic tail phosphorylations via synthetic proteins. Cell Chem. Biol. <i>28</i>, 722-732. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00483-4/sref1">View at publisher</a></p>
<p id="bib2">Tan, Y., Wu, H., Wei, T., and Li, X. (2020). Chemical protein synthesis: advances, challenges, and outlooks. J. Am. Chem. Soc. <i>142</i>, 20288-20298. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00483-4/sref2">View at publisher</a></p>
<p id="bib3">Zhang, Y., Xu, C., Lam, H.Y., Lee, C.L., and Li, X. (2013). Protein chemical synthesis by serine and threonine ligation. Proc. Natl. Acad. Sci. U S A <i>110</i>, 6657. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00483-4/sref3">View at publisher</a></p>
<p id="bib4">Reeves, R. (2001). Molecular biology of HMGA proteins: hubs of nuclear function. Gene <i>277</i>, 63-81. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00483-4/sref4">View at publisher</a></p>
<p id="bib5">Zhang, Q. and Wang, Y. (2008). High mobility group proteins and their post-translational modifications. Biochim. Biophys. Acta <i>1784</i>, 1159-1166. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00483-4/sref5">View at publisher</a></p>
<p id="bib6">Liu, H. and Li, X. (2018). Serine/threonine ligation: origin, mechanistic aspects, and applications. Acc. Chem. Res. <i>51</i>, 1643-1655. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00483-4/sref6">View at publisher</a></p>
<p id="bib7">Thompson, R.E. and Muir, T.W. (2019). Chemoenzymatic semisynthesis of proteins. Chem. Rev. <i>120</i>, 3051-3126. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00483-4/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Xuechen Li (<a href="mailto:xuechenl@hku.hk">xuechenl@hku.hk</a>).</p>
<h4>Materials availability</h4>
<p>Requests for reagents may be sent to Xuechen Li (<a href="mailto:xuechenl@hku.hk">xuechenl@hku.hk</a>).</p>
<h4>Data and code availability</h4>
<p>No data and code were developed in this study.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the <a href="https://doi.org/10.13039/501100002920">Research Grants Council</a> (17309616 and C7017-18G) of Hong Kong, the <a href="https://doi.org/10.13039/100007219">National Science Foundation of China</a> (21672180 and 91753101), and the Area of Excellence Scheme of the <a href="https://doi.org/10.13039/501100001839">University Grants Committee</a> of Hong Kong (AoE/P-705/16).</p>
<h3>Author contributions</h3>
<p>X.L., T.W., H.L., H.W., and F.P. established this protocol. X.L, T.W., and H.W. wrote this manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>