<article>
<section>
<h1 id="header">Highly Efficient CRISPR-Associated Protein 9 Ribonucleoprotein-Based Genome Editing in <i>Euglena gracilis</i>
</h1>
<p><time datetime="2020-03-27">Published: March 27, 2020</time></p>
<p>Toshihisa Nomura,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">7</a>,<a href="#fn2">8</a>,<a href="#cor1">*</a></sup> Mizuki Yoshikawa,<sup><a href="#aff2">2</a></sup> Kengo Suzuki,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> and Keiichi Mochida<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>RIKEN Center for Sustainable Resource Science, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan</p>
<p id="aff2"><sup>2</sup>RIKEN Baton Zone Program, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan</p>
<p id="aff3"><sup>3</sup>euglena Co., Ltd., 5-33-1 Shiba, Minato-ku, Tokyo 108-0014, Japan</p>
<p id="aff4"><sup>4</sup>Kihara Institute for Biological Research, Yokohama City University, 641-12 Maioka-cho, Totsuka-ku, Yokohama, Kanagawa 244-0813, Japan</p>
<p id="aff5"><sup>5</sup>Graduate School of Nanobioscience, Yokohama City University, 1-7-29 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan</p>
<p id="aff6"><sup>6</sup>Institute of Plant Science and Resources, Okayama University, 2-20-1 Chuo, Kurashiki, Okayama 710-0046, Japan</p>
<p id="fn1"><sup>7</sup>Technical Contact</p>
<p id="fn2"><sup>8</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:toshihisa.nomura@riken.jp">toshihisa.nomura@riken.jp</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:keiichi.mochida@riken.jp">keiichi.mochida@riken.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100023">10.1016/j.xpro.2020.100023</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p><i>Euglena gracilis</i>, a unicellular phytoflagellate microalga, is a promising biomaterial for foods, feeds, and biofuels. However, targeted mutagenesis in this species has been a long-standing challenge. We recently developed a transgene-free, highly efficient, genome editing method for <i>E. gracilis</i> using CRISPR/Cas9 ribonucleoproteins (RNPs). Our method achieved mutagenesis rates of approximately 80% or more through an electroporation-based direct delivery of Cas9 RNPs. Therefore, this method is suitable for basic research and industrial applications, such as the breeding of <i>Euglena</i>.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib7">Nomura et al. (2019)</a>.</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for an efficient genome editing method using Cas9 RNPs in <i>Euglena gracilis</i>
</li>
<li>Transgene-free targeted mutagenesis is possible via direct delivery of Cas9 RNPs</li>
<li>Highly efficient knock-in of short sequences into the target site is possible</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/70-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<div class="critical">
<span class="critical-title">Critical:</span> All procedures in this protocol are performed aseptically on a clean bench.</div>
<h3>Culture of <i>Euglena gracilis</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–7 days</div>
<ol>
<li>
<i>Euglena gracilis</i> Z strain was cultured using KH medium (<a href="#bib6">Koren, 1967</a>) adjusted to pH 5.5 with potassium hydroxide on a rotary shaker (120 rpm) at 28°C under continuous light (50 μmol photons m<sup>−2</sup>s<sup>−1</sup>) conditions.</li>
<li>The culture was maintained by inoculating 100 μL into 50 mL of fresh KH medium every week (<a href="#fig1">Figures 1</a>A and 1C).</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/70-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Liquid Culture of <i>Euglena gracilis</i>
</div>
<p>Just after the <i>E. gracilis</i> cells were transferred to fresh KH medium (A), after culturing for 3 days (B), and 7 days (C).</p>
</figcaption>
</figure>
<ol start="3">
<li>For the direct delivery of Cas9 ribonucleoproteins (RNPs) by electroporation, <i>E. gracilis</i> cells that have been cultured for 3 days are used (<a href="#fig1">Figure 1</a>B).</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>Alt-R CRISPR-Cas9 tracrRNA</td>
<td>Integrated DNA Technologies</td>
<td>1072532</td>
</tr>
<tr>
<td>Alt-R CRISPR-Cas9 crRNA</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>Alt-R S.p. Cas9 Nuclease V3</td>
<td>Integrated DNA Technologies</td>
<td>1081058</td>
</tr>
<tr>
<td>Ultramer DNA Oligos</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>Kaneka Easy DNA Extraction Kit v.2</td>
<td>Kaneka</td>
<td>KN-T110005</td>
</tr>
<tr>
<td>Tks Gflex DNA Polymerase</td>
<td>Takara Bio</td>
<td>R060A</td>
</tr>
<tr>
<td>CloneJET PCR Cloning Kit</td>
<td>Thermo Scientific</td>
<td>K1231</td>
</tr>
<tr>
<td>Alt-R Genome Editing Detection Kit</td>
<td>Integrated DNA Technologies</td>
<td>1075931</td>
</tr>
<tr>
<td>(NH4)<sub>2</sub>HPO<sub>4</sub>
</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>016-03325</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>169-04245</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>・7H<sub>2</sub>O</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>131-00405</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>038-24985</td>
</tr>
<tr>
<td>Fe<sub>2</sub> (SO<sub>4</sub>)<sub>3</sub>・nH<sub>2</sub>O</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>091-02832</td>
</tr>
<tr>
<td>MnCl<sub>2</sub>・4H<sub>2</sub>O</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>139-00722</td>
</tr>
<tr>
<td>ZnSO<sub>4</sub>・7H<sub>2</sub>O</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>264-00402</td>
</tr>
<tr>
<td>CoSO<sub>4</sub>・7H<sub>2</sub>O</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>032-03802</td>
</tr>
<tr>
<td>Na<sub>2</sub>MoO<sub>4</sub>・2H<sub>2</sub>O</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>196-02472</td>
</tr>
<tr>
<td>CuSO<sub>4</sub>・5H<sub>2</sub>O</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>039-04412</td>
</tr>
<tr>
<td>Vitamin B<sub>1</sub> (Thiamin)</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>201-00852</td>
</tr>
<tr>
<td>Vitamin B<sub>12</sub>
</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>226-00343</td>
</tr>
<tr>
<td>10% Sulfuric acid</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>198-11705</td>
</tr>
<tr>
<td>Potassium hydroxide</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>168-21815</td>
</tr>
<tr>
<td colspan="3">Experimental Models: Organisms/Strains</td>
</tr>
<tr>
<td><i>Euglena gracilis Z </i></td>
<td>Institute of Applied Microbiology (IAM) culture collection</td>
<td>IAM E-6, NIES-48</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>BioTRON</td>
<td>NK systems</td>
<td>LPH-411SP</td>
</tr>
<tr>
<td>Shake-LR</td>
<td>TAITEC</td>
<td>0054809-000</td>
</tr>
<tr>
<td>T100 Thermal Cycler</td>
<td>Bio-Rad</td>
<td>#1861096</td>
</tr>
<tr>
<td>NEPA21 Super Electroporator</td>
<td>Nepa Gene</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<h3>Design of the Target Sequence for crRNA Synthesis</h3>
<p>Normally, the target sequence used is 20 bp upstream of the protospacer adjacent motif sequence (5’-NGG-3’). The GC content of the target sequence should be 40%–60%. It is recommended to design two or more target sequences for one target gene.</p>
<h3>Preparation of Stock crRNA and tracrRNA Solutions (100 μM) with Nuclease-free Duplex Buffer</h3>
<p>Alt-R CRISPR-Cas9 crRNA (CRISPR RNA) and tracrRNA (trans-activating crRNA) were synthesized by Integrated DNA Technologies (IDT), and a 100-μM solution was prepared using nuclease-free duplex buffer (IDT).</p>
<div class="note">
<span class="note-title">Note:</span> The stock solution can be stored at −20°C.</div>
<h3>Preparation of the Modified CM Medium (<a href="#bib1">Cramer and Myers, 1952</a>)</h3>
<ul>
<li>Prepare a modified CM medium with the following composition.</li>
</ul>
<table>
<caption>Composition of Modified CM Medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Weight per L</th>
</tr>
</thead>
<tbody>
<tr>
<td>(NH4)<sub>2 </sub>HPO<sub> 4</sub>
</td>
<td>1.0 g</td>
</tr>
<tr>
<td>KH<sub>2 </sub>PO<sub>4 </sub>
</td>
<td>1.0 g</td>
</tr>
<tr>
<td>MgSO<sub>4 </sub>・7H<sub> 2</sub>O</td>
<td>0.2 g</td>
</tr>
<tr>
<td>CaCl<sub>2 </sub>
</td>
<td>0.02 g</td>
</tr>
<tr>
<td>Fe<sub>2 </sub> (SO <sub>4</sub>) <sub>3</sub> ・nH<sub>2 </sub>O</td>
<td>3 mg</td>
</tr>
<tr>
<td>MnCl<sub>2 </sub>・4H<sub> 2</sub>O</td>
<td>1.8 mg</td>
</tr>
<tr>
<td>ZnSO<sub>4 </sub>・7H<sub> 2</sub>O</td>
<td>0.4 mg</td>
</tr>
<tr>
<td>CoSO<sub>4 </sub>・7H<sub> 2</sub>O</td>
<td>1.5 mg</td>
</tr>
<tr>
<td>Na<sub>2 </sub>MoO<sub> 4</sub>・2H <sub>2</sub>O</td>
<td>0.2 mg</td>
</tr>
<tr>
<td>CuSO<sub>4 </sub>・5H<sub> 2</sub>O</td>
<td>0.02 mg</td>
</tr>
<tr>
<td>Vitamin B<sub> 1</sub>
</td>
<td>0.1 mg</td>
</tr>
<tr>
<td>Vitamin B<sub> 12</sub>
</td>
<td>0.0005 mg</td>
</tr>
</tbody>
</table>
<p>Adjust the pH to 5.5 with 10% sulfuric acid.</p>
<p>Autoclave at 121°C for 15 min.</p>
<div class="note">
<span class="note-title">Note:</span> CM medium can be stored at 20°C–25°C.</div>
<h3>Preparation of Electroporation Solution</h3>
<ul>
<li>Prepare a 0.3-M sucrose solution and sterilize using a 0.2-μm pore size syringe filter.</li>
<li>Mix the modified CM medium and the filter sterilized sucrose solution at a ratio of 3:2 (v/v).</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> The electroporation solution can be stored at 20°C–25°C.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol can be implemented using other thermal cyclers, plant growth chambers, and orbital shakers with equivalent device performance as the above equipment.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Preparation of Cas9 RNPs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This step describes the procedure to prepare Cas9 RNP complexes for electroporation.</p>
<ol>
<li>Equal amounts of crRNA and tracrRNA solution (100 μM stock) were mixed in a 0.2-mL PCR tube. An example of the reagent amounts used for one electroporation reaction is shown in <a href="#tbl1">Table 1</a>.</li>
</ol>
<table id="tbl1">
<caption>Alt-R CRISPR-Cas9 crRNA Reagent Volumes Used in Each Electroporation Reaction</caption>
<thead>
<tr>
<th>Reagent (μM)</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Alt-R CRISPR-Cas9 crRNA (100)</td>
<td>0.6</td>
</tr>
<tr>
<td>Alt-R CRISPR-Cas9 tracrRNA (100)</td>
<td>0.6</td>
</tr>
<tr>
<td>Total volume</td>
<td>1.2</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>The mixture was heated at 95°C for 5 min and then cooled to 20°C using a thermal cycler with a ramp rate setting of −0.1°C/s.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Heated crRNA-tracrRNA complex can be cooled on the bench at 20°C–25°C.</div>
<ol start="3">
<li>Alt-R S.p. Cas9 Nuclease V3 (IDT, 62 μM solution) was added to the cooled gRNA complex. Examples of the amount of reagent used for one electroporation reaction are shown in <a href="#tbl2">Table 2</a>.</li>
</ol>
<table id="tbl2">
<caption>crRNA-tracrRNA Complex and Alt-R CRISPR-Cas9 crRNA Reagent Volumes Used in Each Electroporation</caption>
<thead>
<tr>
<th>Reagent (μM)</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>crRNA-tracrRNA complex</td>
<td>1.2</td>
</tr>
<tr>
<td>Alt-R S.p. Cas9 Nuclease V3 (62)</td>
<td>0.8</td>
</tr>
<tr>
<td>Total volume</td>
<td>2.0</td>
</tr>
</tbody>
</table>
<ol start="4">
<li>To form the Cas9 RNP complexes, the mixture was incubated for 15 min at 20°C–25°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The Cas9 RNP complex can be stored at −80°C for 6 months.</div>
<h3>Direct Delivery of Cas9 RNPs by Electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–20 min</div>
<p>This step describes the procedure to deliver Cas9 RNPs to <i>E. gracilis</i> cells by electroporation.</p>
<ol start="5">
<li>
<i>E. gracilis</i> was cultured in KH medium (pH 5.5) for 3 days (<a href="#fig1">Figure 1</a>B), and 1 mL of the cell culture was transferred to a 1.5 mL tube (<a href="#fig2">Figure 2</a>A).</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/70-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Preparation of the Suspension of E. gracilis Cells for Electroporation</div>
<p>Transfer of cultured <i>E. gracilis</i> cells (A), <i>E. gracilis</i> cells pellet after centrifugation (B), Resuspended <i>E. gracilis</i> cells (C).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> The optical density at 600 nm of 10-fold dilution of <i>E. gracilis</i> after 3 days of culture is approximately 0.5–0.7.</div>
<ol start="6">
<li>The cell culture sample was centrifuged at 400 × <i>g</i> for 30 s (<a href="#fig2">Figure 2</a>B), and the supernatant was removed.</li>
<li>The cells were then resuspended with the addition of 1 mL of electroporation solution.</li>
<li>The cell suspension was centrifuged at 400 × <i>g</i> for 30 s, and the supernatant was removed.</li>
<li>The cells were resuspended in electroporation solution at a concentration of 1 × 10<sup>6</sup> cells/mL (<a href="#fig2">Figure 2</a>C).</li>
<li>Then, 48 μL of the <i>E. gracilis</i> suspension was transferred to 0.2 mL new tube (<a href="#fig3">Figure 3</a>A).</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/70-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Preparation Procedure for Cas9 RNPs Introduction by Electroporation</div>
<p>Transfer of resuspended <i>E. gracilis</i> cells (A), Addition of Cas9 RNPs (B), Transfer of Cas9 RNPs and cell suspension mixture to cuvette (C), Tapping of the cuvette (D).</p>
</figcaption>
</figure>
<ol start="11">
<li>Finally, 2 μL of Cas9 RNP complex solution was added and the solution was mixed several times by pipetting (<a href="#fig3">Figure 3</a>B). Alternatively, to prevent the loss of Cas9 RNPs solution, the <i>E. gracilis</i> suspension can be directly transferred to a tube containing 2 μL of the RNP complex solution.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The amount of Cas9 RNPs is important. If the RNP amount is reduced, the genome editing efficiency will decrease.</div>
<ol start="12">
<li>The Cas9 RNPs and <i>E. gracilis</i> suspension mixture was transferred to a 2-mm gap cuvette (EC-002; Nepa Gene) (<a href="#fig3">Figure 3</a>C).</li>
<li>The cuvette was tapped to drop the Cas9 RNP and cell suspension mixture to the bottom of the cuvette (<a href="#fig3">Figure 3</a>D).</li>
<li>A NEPA21 Super Electroporator (Nepa Gene) was used for the introduction of the RNP complexes. The electroporation conditions are shown in <a href="#tbl3">Table 3</a> and <a href="#fig4">Figure 4</a>A.</li>
</ol>
<table id="tbl3">
<caption>Settings for the NEPA21 Super Electroporator</caption>
<thead>
<tr>
<th> </th>
<th>Poring Pulse</th>
<th>Transfer Pulse</th>
</tr>
</thead>
<tbody>
<tr>
<td>Voltage (V)</td>
<td>300</td>
<td>20</td>
</tr>
<tr>
<td>Pulse length (ms)</td>
<td>5</td>
<td>50</td>
</tr>
<tr>
<td>Pulse interval (ms)</td>
<td>50</td>
<td>50</td>
</tr>
<tr>
<td>Number of pulses</td>
<td>2</td>
<td>5</td>
</tr>
<tr>
<td>Decay rate (%)</td>
<td>40</td>
<td>40</td>
</tr>
<tr>
<td>Polarity switching</td>
<td>+</td>
<td>±</td>
</tr>
</tbody>
</table>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/70-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Operation Procedure for the Electroporator</div>
<p>Settings of the electroporator (A), Insertion of the cuvette into the electrode chamber (B), Measurement of the impedance value (C), Initiation of electroporation (D).</p>
</figcaption>
</figure>
<ol start="15">
<li>The cuvette was inserted into the electrode chamber (<a href="#fig4">Figure 4</a>B), and the “Ω” switch was pressed. The impedance value was usually approximately 0.4–0.5 kΩ (<a href="#fig4">Figure 4</a>C).</li>
<li>The “Start” switch was pressed to initiate electroporation (<a href="#fig4">Figure 4</a>D).</li>
<li>After electroporation, 1 mL of KH medium (pH 5.5) was added to the cuvette (<a href="#fig5">Figures 5</a>A and 5B).</li>
</ol>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/70-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Procedure of Recovery Culture after Electroporation</div>
<p>Ejection of the cuvette from the electrode chamber (A), Addition of KH medium (B), Transfer of cell suspension to well plate (C), Covering of culture plate with aluminum foil (D) and recovery culture on a rotary shaker (E).</p>
</figcaption>
</figure>
<ol start="18">
<li>The cell suspension was transferred to a 12-well plate (<a href="#fig5">Figure 5</a>C) and cultured on a rotary shaker (120 rpm) at 28°C under dark conditions (by covering with aluminum foil) for 24–72 h (<a href="#fig5">Figures 5</a>D and 5E).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Seal the plate with parafilm and surgical tape to prevent evaporation of the culture medium.</div>
<h3>Single-Stranded Oligodeoxynucleotide-Mediated Knockin (Optional)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<p>The high-efficiency knockin of short sequences (∼50 bp) is possible by introducing the Cas9 RNPs and single-stranded oligodeoxynucleotides (ssODNs) with 50 bp upstream and downstream homology sequence arms of the target site (<a href="#bib7">Nomura et al., 2019</a>).</p>
<ol start="19">
<li>ssODNs were synthesized by IDT, and a 200 μM solution was prepared with nuclease-free duplex buffer (IDT)</li>
<li>For electroporation, 2 μL of RNP complex solution and 1 μL of 200 μM ssODN stock solution were added to 47 μL of the <i>E. gracilis</i> suspension.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is possible to rewrite the target base using an ssODN containing the nucleotide substitutions. In this case, to prevent re-cleavage by Cas9 RNPs, two or more base mismatches are required in the seed region of the target sequence in ssODN.</div>
<h3>Detection of Targeted Mutagenesis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h or more</div>
<p>For the detection of mutagenesis at on-target sites in Cas9 RNP-introduced <i>E. gracilis</i>, a T7 endonuclease I (T7EI) mismatch cleavage assay was performed using an Alt-R Genome Editing Detection Kit (IDT).</p>
<ol start="21">
<li>The DNA template was extracted from the small <i>E. gracilis</i> cells pellet (<a href="#fig6">Figure 6</a>A) using a Kaneka Easy DNA Extraction Kit v.2 (Kaneka).</li>
</ol>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/70-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Example of T7 Endonuclease I Mismatch Cleavage Assay</div>
<p>Small <i>E. gracilis</i> cells pellet after recovery culture for DNA extraction (A) and example of T7E1 assay results (B).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Alternatively, another DNA extraction kit can be used.</div>
<ol start="22">
<li>DNA fragments, including the target sites, were amplified by Tks Gflex DNA Polymerase (Takara Bio) using the specific primer set.</li>
<li>Using the amplified DNA fragments, the T7EI treatment was performed according to the product manufacturer’s instructions.</li>
<li>The digested DNA fragments were monitored using 1% agarose gel electrophoresis (<a href="#fig6">Figure 6</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Because T7EI does not recognize single-base indels, the T7EI assay estimates a lower editing efficiency rate.</div>
<h3>Pure-Line (Single-Cell-Derived Clone) Isolation from a Single <i>E. gracilis</i> Cell</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week or more</div>
<p>This step describes how to obtain a single-cell-derived clone in <i>E. gracilis</i>.</p>
<ol start="25">
<li>Following electroporation after 24–72 h culture in KH medium, a single cell was isolated using a micro pick-and-place system (Nepa Gene) (<a href="#fig7">Figures 7</a>A and 7B).</li>
</ol>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/70-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Procedure of Pure-Line Isolation from Single Cell of <i>E. gracilis</i>
</div>
<p>A single <i>E. gracilis</i> cell isolation using a micro pick-and-place system (A and B), Culture from isolated single <i>E. gracilis</i> cell (C and D).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Contamination can be suppressed by using KH medium at pH 3.5 for culture.</div>
<h3>Genotyping by Sanger Sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This step describes Sanger sequencing for genotyping of <i>E. gracilis</i>.</p>
<ol start="27">
<li>The DNA template was extracted from a 5–10-μL cell pellet volume of <i>E. gracilis</i> using the Kaneka Easy DNA Extraction Kit v.2 (Kaneka).</li>
<li>DNA fragments, including the target sites, were amplified by Tks Gflex DNA Polymerase using the specific primer set for target site.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A PCR product length of approximately 500–1000 bp is suitable for genotyping.</div>
<ol start="29">
<li>To clone the PCR products, we used a CloneJET PCR cloning kit (Thermo Fisher Scientific).</li>
<li>Colonies were then selected with the vector containing the insert DNA by colony PCR.</li>
<li>Each plasmid was extracted using a plasmid DNA extraction Mini Kit (FAVORGEN), and sequences were determined using Sanger sequencing.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p><i>Euglena gracilis</i>, a unicellular phototrophic protist, is a promising material for foods, feeds, and biofuels. However, the development of a targeted mutagenesis method in this species has been a long-standing challenge. Among the current genetic manipulation technologies, genome editing by the direct delivery of RNPs has various advantages, including time effectiveness, a low cytotoxicity, a high efficiency, and the reduction of off-target effects (<a href="#bib5">Jeon et al., 2017</a>). In our method, insertion and/or deletion (Indel) mutations rate of 77.7%–90.1% has been detected by amplicon sequencing in two different target sequences in the <i>EgGSL2 </i> gene (<a href="#bib7">Nomura et al., 2019</a>). Therefore, our developed RNP-based genome editing in <i>E. gracilis</i> opens up new avenues to reveal the functions of genes. In addition, based on our developed method, it will be possible to develop convenient CRISPR-based technology for the basic research of <i>E. gracilis</i>, such as chromosome visualization and epigenome editing (<a href="#bib2">Doudna and Charpentier, 2014</a>; <a href="#bib8">Wang et al., 2016</a>). Furthermore, RNP-based genome editing without transgenes is suitable for industrial use because it can potentially bypass the regulation of genetically modified organisms. Thus, RNP-based genome editing in <i>E. gracilis</i> paves the way for improving its industrially relevant traits to promote production of bio-based material.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Because the published genome sequence information of <i>E. gracilis</i> strain Z1 appears to be fragmented (<a href="#bib3">Ebenezer et al., 2019</a>), it has been difficult to precisely assess the off-target effects of genome editing in <i>E. gracilis </i>. This limitation will be resolved by improving the genome assembly of <i>E. gracilis</i>. To confirm a phenotypic change by on-target in genome-edited strains, several target sites for a target gene should be examined. Off-target effect can be suppressed by reducing the amount of RNPs and using a truncated (∼18 bp) target sequence (<a href="#bib4">Fu et al. 2014</a>). Furthermore, off-target can be reduced by the use of S.p. HiFi Cas9 Nuclease V3 (IDT).</p>
<p>In case a different strain of <i>E. gracilis</i> is used, efficiency may decrease due to differences in cell properties, such as growth rate. Therefore, in such cases, the protocol for each strain needs to be optimized. In addition, there is a possibility that parts of the genomic DNA sequence vary depending on the strain of <i>E. gracilis</i> used. Before designing the target sequence, it is also necessary to obtain information on the surrounding DNA sequence of the target site by cloning and sequencing.</p>
<p>The maximum length of single-stranded DNAs that can be knocked in in our protocol remains undetermined. It is generally considered that the homology arm needs to be lengthened according to the length of the DNA to be knocked in. Although the present method has enabled us to precisely knockin a DNA fragment of at least 50 bp in length, an improvement in our method for the knockin of longer DNAs in <i>E. gracilis</i> will provide opportunities to engineer <i>E. gracilis</i> cells to enhance its productivity and function.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem</h3>
<p>Low mutagenesis efficiency</p>
<h3>Potential Solution</h3>
<p>In cases when the genome-edited cells exhibit a slow growth phenotype, the mutagenesis efficiency could be decreased due to the faster growth of wild-type cells. This problem can be solved by earlier cell isolation (e.g., 24 h) after the introduction of Cas9 RNPs. On the other hand, if the performance of the designed target sequence is poor it can be improved by doubling the amount (4 μL/total 50 μL reaction) of Cas9 RNPs used for electroporation.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cramer, M., and Myers, J. (1952). Growth and photosynthetic characteristics of <i>Euglena gracilis</i>. Arch. Mikrobiol. <i>17</i>, 384-402.</p>
<p id="bib2">Doudna J.A., and Charpentier, E. (2014). Genome editing. The new frontier of genome engineering with CRISPR-Cas9. Science <i>346</i>, 1258096.</p>
<p id="bib3">Ebenezer, T.E., Zoltner, M., Burrell, A., Nenarokova, A., Novak Vanclova, A.M.G., Prasad, B., Soukal, P., Santana-Molina, C., O’Neill, E., Nankissoor, N.N., et al. (2019). Transcriptome, proteome and draft genome of <i>Euglena gracilis</i>. BMC Biol. <i>17</i>, 11.</p>
<p id="bib4">Fu, Y., Sander, J.D., Reyon, D., Cascio, V.M, &amp; Joung, J.K. (2014). Improving CRISPR-Cas nuclease specificity using truncated guide RNAs. Nat. Biotechnol. <i>32</i>, 279.</p>
<p id="bib5">Jeon, S., Lim, J.-M., Lee, H.-G., Shin, S.-E., Kang, N.K., Park, Y.-I., Oh, H.-M., Jeong, W.-J., Jeong, B.-R., Chang, Y.K. (2017). Current status and perspectives of genome editing technology for microalgae. Biotechnol. Biofuels <i>10</i>, 267.</p>
<p id="bib6">Koren, L. (1967). High-yield media for photosynthesizing <i>Euglena gracilis</i> Z. J. Protozool. <i>14</i>, 17.</p>
<p id="bib7">Nomura, T., Inoue, K., Uehara-Yamaguchi, Y., Yamada, K., Iwata, O., Suzuki, K., Mochida, K. (2019). Highly efficient transgene-free targeted mutagenesis and single-stranded oligodeoxynucleotide-mediated precise knock-in in the industrial microalga <i>Euglena gracilis</i> using Cas9 ribonucleoproteins. Plant Biotechnol. J. <i>17</i>, 2032-2034.</p>
<p id="bib8">Wang, H., La Russa, M., Qi, L.S. (2016). CRISPR/Cas9 in genome editing and beyond. Ann. Rev. Biochem. <i>85</i>, 227-264.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>We thank Aya Ide and Hiromi Ojima for their support in the acquisition of the photos in this experimental protocol.</p>
<h3>Author Contributions</h3>
<p>Methodology, T.N.; Investigation, T.N. and M.Y.; Resources, K.S., Writing – Original Draft, T.N. and K.M.</p>
<h3>Declaration of Interests</h3>
<p>K.S. is an employee and shareholder of euglena Co., Ltd. T.N., K.S., and K.M. have applied for a patent for this method.</p>
</section>
</article>