<article>
<section>
<h1 id="header">Protocol for assay of transposase accessible chromatin sequencing in non-model species</h1>
<p><time datetime="2021-02-19">Published: February 19, 2021</time></p>
<p>Stephen Kissane,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Vignesh Dhandapani,<sup><a href="#aff1">1</a></sup> and Luisa Orsini<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>School of Biosciences, University of Birmingham, Birmingham B15 2TT, UK</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:l.orsini@bham.ac.uk">l.orsini@bham.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100341">10.1016/j.xpro.2021.100341</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The assay for transposase accessible chromatin (ATAC-seq) is a method for mapping genome-wide chromatin accessibility. Coupled with high-throughput sequencing, it enables integrative epigenomics analyses. ATAC-seq requires direct access to cell nuclei, a major challenge in non-model species such as small invertebrates, whose soft tissue is surrounded by a protective exoskeleton. Here, we present modifications of the ATAC-seq protocol for applications in small crustaceans, extending applications to non-model species.</p>
<p>For complete information on the use and execution of this protocol, please refer to <a href="#bib4">Buenrostro et al. (2013)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>ATAC-seq modified protocol for applications in non-model species</li>
<li>Transposase titration identifies thresholds for optimal transposition at lower costs</li>
<li>Minimal number of cells/tissue is identified for low input reactions</li>
<li>Proof of concept ATAC-seq analysis for the waterflea <i>Daphnia magna</i>
</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/475-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The assay for transposase accessible chromatin (ATAC-seq) has been optimized for mammalian cells to map genome-wide chromatin changes, allowing simultaneous interrogation of nucleosome positions in regulatory sites and genome-wide chromatin accessibility (<a href="#bib4">Buenrostro et al., 2013</a>; <a href="#bib5">Buenrostro et al., 2015</a>). More recently, the protocol has been applied to other model species, such as the fruitfly to discover transcriptional stages of aging in the brain (<a href="#bib8">Davie et al., 2018</a>), and mice to identify tissue-specific transcription factors (<a href="#bib19">Liu et al., 2019</a>). However, to investigate the chromatin rearrangement within cells, a direct <i>in vitro</i> transposition of sequencing adapters into native chromatin is needed. This requires between 500 and 5,000 free nuclei from cell suspensions obtained from blood or soft tissue; the sensitivity of detection diminishes with smaller input and quality of the input material, limiting the application of ATAC-seq in non-model species (<a href="#bib4">Buenrostro et al., 2013</a>).</p>
<p>Here, we present a modified ATAC-seq protocol for applications in small crustaceans, in which a chitin exoskeleton obstructs access to cell nuclei. The protocol is optimized for low input material, relevant for applications in non-model species in which starting material may be of limited quantity. The current costs of ATAC-seq are prohibitive for population-level applications because of the costs of the transposase enzyme. We perform a titration experiment demonstrating that the amount of the enzyme transposase required to obtain high-quality <i>in vitro</i> transposition can be significantly reduced without affecting data quality. We demonstrate the application of the modified protocol on the biomedical and ecological model species <i>Daphnia</i>. Like the model species <i>Drosophila</i>, <i>Daphnia</i> enjoys many technical advantages over vertebrate models: they are easy and inexpensive to culture in the laboratory, have a short life cycle, and produce large numbers of externally laid embryos. In addition, <i>Daphnia</i> has a parthenogenetic life cycle, allowing the rearing of populations of isogenic individuals (clones) from a single genotype. These properties enabled us to test different sample inputs and to quantify the quality of output without confounding factors associated with genetic variation among genotypes.</p>
<h3 id="sec1.1">Experimental design considerations</h3>
<p>The first step of this protocol requires the collection of sufficient tissue from <i>Daphnia</i> genotypes. Depending on the experimental design, multiple genotypes may be used. As epigenomic patterns may be influenced by environmental factors, special care should be adopted in controlling for environmental and developmental variation among genotypes and biological replicates. Before samples collection, <i>Daphnia</i> genotypes should be maintained in common garden conditions for at least two generations to reduce interference from maternal effect. When multiple genotypes are used, they should be synchronized and sampled at the same developmental stages. To prevent variation introduced by sex-specific expression, only females should be collected, following direct inspection of the transparent <i>Daphnia</i> body under a stereomicroscope. Populations of identical clones from the same genotype at the same developmental stage can be used to achieve sufficient starting material for nuclei extraction.</p>
<h3 id="sec1.2">Material preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5 h</div>
<p>Before starting the protocol, ensure that the following reagents and equipment are available:</p>
<ol>
<li>80% ethanol solution corresponding to 1 mL per sample.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is recommended that a fresh ETOH solution is prepared daily to reduce alteration of ETOH final concentration due to evaporation.</div>
<ol start="2">
<li>Lysis tubes for tissue homogenization with Geno/Grinder or other suitable tissue homogenizer. These are prepared by adding 25–30 ceramic beads of 1.4 mm diameter to sterile tubes. Alternatively, purchase sterile tubes prefilled with ceramic beads of 1.4 mm diameter.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To avoid sample contamination, tubes are filled with ceramic beads under a laminar flow hood and stored in sealed sterile bags.</div>
<ol start="3">
<li>Ensure that two heat blocks are available and set at 65°C and 37°C, respectively.</li>
<li>Ensure that a centrifuge suitable for Eppendorf tubes is at 4°C for nuclei purification (step 2 below).</li>
<li>Ensure that ATAC sequencing primers are available (<a href="#tbl1">Table 1</a>). Primers are used at a final concentration of 25 μM per sample.
<table id="tbl1">
<caption>Table 1. ATAC primers</caption>
<thead>
<tr>
<th>Primer</th>
<th>Sequence F primer 5′-3′</th>
<th>Sequence R primer 5′-3′</th>
</tr>
</thead>
<tbody>
<tr>
<td>ATAC-1</td>
<td>CAAGCAGAAGACGGCATACGAGATGACACAGTGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACGACACAGTTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-2</td>
<td>CAAGCAGAAGACGGCATACGAGATGCATAACGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACGCATAACGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-3</td>
<td>CAAGCAGAAGACGGCATACGAGATACAGAGGTGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACACAGAGGTTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-4</td>
<td>CAAGCAGAAGACGGCATACGAGATCCACTAAGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACCCACTAAGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-5</td>
<td>CAAGCAGAAGACGGCATACGAGATTGTTCCGTGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTGTTCCGTTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-6</td>
<td>CAAGCAGAAGACGGCATACGAGATGATACCTGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACGATACCTGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-7</td>
<td>CAAGCAGAAGACGGCATACGAGATAGCCGTAAGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACAGCCGTAATCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-8</td>
<td>CAAGCAGAAGACGGCATACGAGATCTCCTGAAGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACCTCCTGAATCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-9</td>
<td>CAAGCAGAAGACGGCATACGAGATACGAATCCGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACACGAATCCTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-10</td>
<td>CAAGCAGAAGACGGCATACGAGATAATGGTCGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACAATGGTCGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-11</td>
<td>CAAGCAGAAGACGGCATACGAGATCGCTACATGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACCGCTACATTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-12</td>
<td>CAAGCAGAAGACGGCATACGAGATCCTAAGTCGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACCCTAAGTCTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-13</td>
<td>CAAGCAGAAGACGGCATACGAGATTTGCTTGGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTTGCTTGGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-14</td>
<td>CAAGCAGAAGACGGCATACGAGATCCTGTCAAGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACCCTGTCAATCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-15</td>
<td>CAAGCAGAAGACGGCATACGAGATAGCCTATCGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACAGCCTATCTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-16</td>
<td>CAAGCAGAAGACGGCATACGAGATTGATCACGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTGATCACGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-17</td>
<td>CAAGCAGAAGACGGCATACGAGATCCACATTGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACCCACATTGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-18</td>
<td>CAAGCAGAAGACGGCATACGAGATTCGAGAGTGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCGAGAGTTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-19</td>
<td>CAAGCAGAAGACGGCATACGAGATGGTCGTATGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACGGTCGTATTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-20</td>
<td>CAAGCAGAAGACGGCATACGAGATACAGGCATGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACACAGGCATTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-21</td>
<td>CAAGCAGAAGACGGCATACGAGATGTGATCCAGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACGTGATCCATCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-22</td>
<td>CAAGCAGAAGACGGCATACGAGATTTCGTACGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTTCGTACGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-23</td>
<td>CAAGCAGAAGACGGCATACGAGATATGACAGGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACATGACAGGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-24</td>
<td>CAAGCAGAAGACGGCATACGAGATCGACCTAAGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACCGACCTAATCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-25</td>
<td>CAAGCAGAAGACGGCATACGAGATTATGGCACGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTATGGCACTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-26</td>
<td>CAAGCAGAAGACGGCATACGAGATATAACGCCGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACATAACGCCTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-27</td>
<td>CAAGCAGAAGACGGCATACGAGATGTAGTACCGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACGTAGTACCTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-28</td>
<td>CAAGCAGAAGACGGCATACGAGATCGCGTATTGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACCGCGTATTTCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-29</td>
<td>CAAGCAGAAGACGGCATACGAGATATCCACGAGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACATCCACGATCGTCGGCAGCGTCAGATGTG</td>
</tr>
<tr>
<td>ATAC-30</td>
<td>CAAGCAGAAGACGGCATACGAGATTAACGTCGGTCTCGTGGGCTCGGAGATGT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTAACGTCGTCGTCGGCAGCGTCAGATGTG</td>
</tr>
</tbody>
</table>
<p class="table-legend">List of forward (F) and reverse (R) primer sequences used in the sequencing of ATAC-seq data on an Illumina platform. Primers used at 25μM final concentration are shown in 5′-3′ orientation.</p>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For convenience, primers at a final concentration of 25 μM are pre-aliquoted in a 96-well plate to enable automated library preparation with a liquid handling robot or manual library preparation using a multichannel pipette.</div>
<div class="note">
<span class="note-title">Note:</span> To reduce index hopping between samples, a unique combination of paired end primers per sample should be preferred (<a href="#tbl1">Table 1</a>). Index hopping has impacted NGS technologies from the time sample multiplexing was developed. It causes the incorrect assignment of libraries from the expected index to a different index in the multiplexed pool. Using unique dual indexes over combinatorial dual indexes alleviates this problem (<a href="#bib16">Kircher et al., 2012</a>)</div>
<ol start="6">
<li>Prepare the following solutions using sterile Milli-Q water and store up to 3 months at 18°C–20°C</li>
</ol>
<table id="undtbl1">
<caption>Tris-HCl solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)/weight (g)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>1,000</td>
<td>7.88 g</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>NaCl solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)/weight (g)</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>5,000</td>
<td>14.61 g</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>MgCl<sub>2</sub> solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)/weight (g)</th>
</tr>
</thead>
<tbody>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>300</td>
<td>1.43 g</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>RSB buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>1,000</td>
<td>0.25</td>
</tr>
<tr>
<td>NaCl</td>
<td>5,000</td>
<td>0.05</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>300</td>
<td>0.25</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>24.45</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>25</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals</td>
</tr>
<tr>
<td>Buffer EB</td>
<td>Qiagen</td>
<td>CAT# 19086</td>
</tr>
<tr>
<td>Ceramic beads (1.4 mm diameter; 325 g)</td>
<td>Qiagen</td>
<td>CAT# 13113-325</td>
</tr>
<tr>
<td>∗Ethanol absolute, molecular biology grade (200 proof)</td>
<td>Fisher Scientific</td>
<td>CAT# 10517694</td>
</tr>
<tr>
<td>High sensitivity D1000 reagents</td>
<td>Agilent</td>
<td>CAT# 5067- 5585</td>
</tr>
<tr>
<td>High sensitivity D1000 ScreenTape</td>
<td>Agilent</td>
<td>CAT# 5067- 5584</td>
</tr>
<tr>
<td>Magbio HighPrep PCR beads</td>
<td>Auto Q Biosciences</td>
<td>CAT# AQ 60500</td>
</tr>
<tr>
<td>Illumina Tagment DNA TDE1 enzyme and buffer kits</td>
<td>Illumina</td>
<td>CAT# 20034198</td>
</tr>
<tr>
<td>Liquid Proteinase K</td>
<td>Fisher Scientific</td>
<td>CAT# NC1442588</td>
</tr>
<tr>
<td>∗MasterPure Complete DNA and RNA Purification Kit</td>
<td>Lucigen</td>
<td>CAT# MC85200</td>
</tr>
<tr>
<td>∗MgCl<sub>2</sub>
</td>
<td>Sigma-Aldrich</td>
<td>CAT# M8266-100G</td>
</tr>
<tr>
<td>∗NaCl</td>
<td>Sigma-Aldrich</td>
<td>CAT# S7653-250G</td>
</tr>
<tr>
<td>NEBNext High-Fidelity 2× PCR Master Mix</td>
<td>New England Biolabs</td>
<td>CAT# M0541L</td>
</tr>
<tr>
<td>Qubit 1× dsDNA HS Assay Kit</td>
<td>Invitrogen</td>
<td>CAT# Q33231</td>
</tr>
<tr>
<td>∗RNase A</td>
<td>Qiagen</td>
<td>CAT# 19101</td>
</tr>
<tr>
<td>Screw cap tubes (2 mL)</td>
<td>VWR International</td>
<td>CAT# 211-0432</td>
</tr>
<tr>
<td>iTaq Universal SYBR Green Supermix</td>
<td>Bio-Rad</td>
<td>CAT# 172-5121</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>Sigma-Aldrich</td>
<td>CAT# 93363-500G</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Sigma-Aldrich</td>
<td>CAT# P9416-100ML</td>
</tr>
<tr>
<td>Water DEPC treated</td>
<td>National Diagnostics</td>
<td>EC-625</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>Daphnia magna</i> LRV0_1</td>
<td>University of Birmingham</td>
<td>NA</td>
</tr>
<tr>
<td>
<i>Daphnia magna</i> LRV2_1</td>
<td>University of Birmingham</td>
<td>NA</td>
</tr>
<tr>
<td>
<i>Daphnia magna</i> LRV3.5_15</td>
<td>University of Birmingham</td>
<td>NA</td>
</tr>
<tr>
<td>
<i>Daphnia magna</i> LRV13_3</td>
<td>University of Birmingham</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>fastqc (V0.11.9)</td>
<td>(<a href="#bib1">Andrews, 2010</a>)</td>
<td><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc">https://www.bioinformatics.babraham.ac.uk/projects/fastqc</a></td>
</tr>
<tr>
<td>Trimmomatic (v0.32)</td>
<td>(<a href="#bib3">Bolger et al., 2014</a>)</td>
<td>=<a href="http://www.usadellab.org/cms/?pagetrimmomatic">http://www.usadellab.org/cms/?pagetrimmomatic</a>
</td>
</tr>
<tr>
<td>Galaxy pipeline</td>
<td>(<a href="#bib2">Batut et al., 2018</a>)</td>
<td><a href="https://training.galaxyproject.org/">https://training.galaxyproject.org/</a></td>
</tr>
<tr>
<td>multiqc</td>
<td>(<a href="#bib11">Ewels et al., 2016</a>)</td>
<td><a href="https://github.com/ewels/MultiQC/">https://github.com/ewels/MultiQC/</a></td>
</tr>
<tr>
<td>ataqv</td>
<td>(<a href="#bib21">Orchard et al., 2020</a>)</td>
<td><a href="https://github.com/ParkerLab/ataqv">https://github.com/ParkerLab/ataqv</a></td>
</tr>
<tr>
<td>Bowtie2 (v2.2.6)</td>
<td>(<a href="#bib17">Langmead et al., 2019</a>)</td>
<td><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td>
</tr>
<tr>
<td>Samtools</td>
<td>(<a href="#bib18">Li et al., 2009</a>)</td>
<td><a href="http://www.htslib.org/doc/samtools.html">http://www.htslib.org/doc/samtools.html</a></td>
</tr>
<tr>
<td>Picard tools</td>
<td>Online resource</td>
<td><a href="https://broadinstitute.github.io/picard/">https://broadinstitute.github.io/picard/</a></td>
</tr>
<tr>
<td>Qualimap 2</td>
<td>(<a href="#bib20">Okonechnikov et al., 2016</a>)</td>
<td><a href="http://qualimap.conesalab.org/">http://qualimap.conesalab.org/</a></td>
</tr>
<tr>
<td>deeptools alignmentSieve</td>
<td>(<a href="#bib23">Ramirez et al., 2018</a>)</td>
<td><a href="https://deeptools.readthedocs.io/en/develop/content/tools/alignmentSieve.html">https://deeptools.readthedocs.io/en/develop/content/tools/alignmentSieve.html</a></td>
</tr>
<tr>
<td>MACS2 callpeak</td>
<td>(<a href="#bib12">Gaspar, 2018</a>)</td>
<td><a href="https://hbctraining.github.io/Intro-to-ChIPseq/lessons/05_peak_calling_macs.html">https://hbctraining.github.io/Intro-to-ChIPseq/lessons/05_peak_calling_macs.html</a></td>
</tr>
<tr>
<td>bedtools</td>
<td>(<a href="#bib22">Quinlan, 2014</a>)</td>
<td><a href="https://bedtools.readthedocs.io/en/latest/">https://bedtools.readthedocs.io/en/latest/</a></td>
</tr>
<tr>
<td>Hypergeometric Optimization of Motif EnRichment (HOMER)</td>
<td>(<a href="#bib9">Duttke et al., 2019</a>)</td>
<td><a href="http://homer.ucsd.edu/homer/motif/">http://homer.ucsd.edu/homer/motif/</a></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> An asterisk (∗) marks chemicals or reagents that are harmful or toxic. In the following hazard and storage requirements are identified:</div>
<p>Ethanol: highly flammable, cause possible fire flashpoint, potential carcinogen. Necessary precautions should be adopted when handling; storage of large volumes should be in fire-proof cabinets.</p>
<p>MgCl<sub>2</sub>: Irritant and corrosive. Protective gloves and lab coat should be wore at all times.</p>
<p>NaCl: Irritant. Protective gloves and lab coat should be wore at all times.</p>
<p>RNase A: May cause allergy or asthma if breathed. Avoid inhalation.</p>
<p>Masterpure kit: Danger, causes skin irritation, serious eye damage, harmful to aquatic life. Protective gloves and lab coat should be wore at all times. Disposal in clinical waste</p>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Equipment required is listed:</p>
<table id="undtbl5">
<thead>
<tr>
<th> </th>
<th>Source</th>
<th>Identifier</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Equipment</td>
</tr>
<tr>
<td>Fine Balance</td>
<td>VWR</td>
<td>LA2541</td>
</tr>
<tr>
<td>Centrifuge compatible for Eppendorf tubes (5424R)</td>
<td>Eppendorf</td>
<td>5404000332</td>
</tr>
<tr>
<td>2020 Geno/Grinder</td>
<td>Spex Sample Prep</td>
<td>n/a</td>
</tr>
<tr>
<td>NanoDrop 8000</td>
<td>Labtech</td>
<td>ND-8000-GL</td>
</tr>
<tr>
<td>Nexus Gradient</td>
<td>Eppendorf</td>
<td>6331000041</td>
</tr>
<tr>
<td>Qubit 3.0 Fluorometer</td>
<td>Invitrogen</td>
<td>Q33216</td>
</tr>
<tr>
<td>Tapestation 2200</td>
<td>Agilent</td>
<td>G2964AA</td>
</tr>
<tr>
<td>Thermomixer C</td>
<td>Eppendorf</td>
<td>5382000031</td>
</tr>
<tr>
<td colspan="3">Optional equipment</td>
</tr>
<tr>
<td>Biomek FxP<sup><a href="#dtbl6fna">a</a></sup>
</td>
<td>Beckman Coulter</td>
<td>A31844</td>
</tr>
<tr>
<td>Bioruptor Pico-sonication device</td>
<td>Diagenode</td>
<td>B01060010</td>
</tr>
</tbody>
</table>
<p id="dtbl6fna" class="table-footnote"><sup>a</sup>The ATAC-seq protocol was optimized manually on a small number of samples. After this optimization step, a liquid handling Biomek FxP workstation was used to prepare large sample numbers. Any liquid handling robotic workstation can be used.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>The modified ATAC protocol has the following main steps:</p>
<h3 id="sec3.1">Cells extraction from tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>Step 1 enables the extraction of cells from tissues by shearing tissue and creating a cells suspension.</p>
<ol>
<li>Flash-freeze 7 last instar female <i>Daphnia</i> juveniles in 2 mL VWR Screw Cap tubes containing 25–30 ceramic beads of 1.4 mm diameter. Store at –80°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The frozen tissue can be stored at −80° before further processing.</div>
<ol start="2">
<li>Thaw samples on ice.</li>
<li>Add 300 μL MasterPure Tissue and Cell Lysis Solution with 2.8 μL of proteinase K (50 U/mL).</li>
<li>Shear samples using an automated tissue homogenizer (e.g., Geno/Grinder) at 1,750 rpm for 30 s.</li>
<li>Place tubes in 65°C for 15 min, inverting 3 times during the incubation.</li>
<li>Cool samples at 37°C for 5 min.</li>
<li>Add 4 μL of 1 mg/mL RNase A.</li>
<li>Incubate at 37°C for 30 min.</li>
<li>Place on ice for 5 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The amount of tissue used to obtain sufficient nuclei is determined experimentally. A titration experiment using different amounts of tissue is recommended for this optimization step. For the protocol optimization, we used pools of last instar Daphnia juveniles ranging from 7 to 60 individuals. Please refer to the <a href="#expected-outcomes">Expected outcomes</a> section below for a description of results of the titration experiment.</div>
<h3 id="sec3.2">Nuclei purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5 h</div>
<p>Step 10 enables the separation of cell material and proteins from the nuclei</p>
<ol start="10">
<li>Add 175 μL MPC protein precipitation reagent (MasterPure Kit) and mix well by gentle pipetting</li>
<li>Centrifuge at 10,000 × <i>g</i> for 10 min in a refrigerated centrifuge at 4°C</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If chromatin deconvolution is not efficient following the steps above, a pico-sonication step for DNA shearing (e.g., Bioruptor, Covaris) can be introduced. For the optimization of this protocol, two settings were used on the Bioraptor: 600 bp and 150 bp. No appreciable differences were observed in the DNA libraries and in the sequencing results between these two settings (see <a href="#expected-outcomes">Expected outcomes</a>).</div>
<ol start="12">
<li>Transfer supernatant to a clean tube and store at –20°C</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The purified nuclei can be stored at −20° until further processing.</div>
<ol start="13">
<li>Thaw samples on ice and transfer 1/3 of supernatant into tubes containing 1 mL of resuspension buffer (RSB; MasterPure Kit) and 0.1% Tween 20.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All supernatant can be transferred into the resuspension buffer. It is possible to determine whether the whole resuspension is needed by testing DNA library quality when a proportion of the supernatant is used. Using only a fraction of the supernatant may provide sufficient material for technical replicates. .</div>
<ol start="14">
<li>Centrifuge at 500 × <i>g</i> for 10 min in a fixed-angle pre-chilled 4°C centrifuge to pellet the nuclei.</li>
<li>Discard Supernatant.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The pelleted nuclei can be stored at −20° until further processing.</div>
<h3 id="sec3.3">Transposition reaction optimization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>This step is only used in the optimization stage to reduce the amount of transposase enzyme without affecting data quality. A titration of the manufacturer recommended transposase enzyme concentration were used to identify the minimum amount required for a high-quality ATAC library. The results of this titration experiment are in <a href="#expected-outcomes">Expected outcomes</a>.</p>
<ol start="16">
<li>Immediately following the nuclei preparation, resuspend the pellet in different dilutions of the transposase reaction mix [25 μL 2× TD buffer; 0.63 μL transposase (100 nM final concentration); 22.5 μL of nuclease-free water].</li>
<li>Test different transposase concentrations. Here we tested four concentrations of the transposase enzyme as follows:
<ol type="a">
<li>1/1 of the volume recommended: 25 μL 2× TD Buffer; 2.5 μL transposase; 22.5 μL of nuclease-free water.</li>
<li>1/2 of the volume recommended: 25 μL 2× TD Buffer; 1.25 μL transposase; 23.75 μL of nuclease-free water.</li>
<li>1/4 of the volume recommended: 25 μL 2× TD Buffer; 0.63 μL transposase; 24.37 μL of nuclease-free water.</li>
<li>1/8 of the volume recommended: 25 μL 2× TD Buffer; 0.32 μL transposase; 24.68 μL of nuclease-free water.</li>
</ol>
</li>
<li>Incubate the transposition reaction at 37°C for 30 min in a thermomixer using 1,000 rpm mixing speed</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Longer incubation times may be tested to improve the transposition efficiency. In the optimization phase, incubation of 16–20 h was tested. The Tapestation traces did not show significant difference between the two incubation times (see <a href="#expected-outcomes">Expected outcomes</a>).</div>
<ol start="19">
<li>Store samples at −20°C until further processing</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The transposition reaction can be stored overnight or up to few days at −20° before progressing with the purification step.</div>
<ol start="20">
<li>Purify samples using a 1:1 volume Magbio HighPrep PCR beads (Auto Q Biosciences) as follows:
<ol type="a">
<li>Place Magbio HighPrep PCR beads at 18°C–20°C for 30 min</li>
<li>Add nuclease-free water to the ligation reaction bringing the reaction volume to 100 μL.
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the final volume is 100 μL before adding Magbio HighPrep PCR beads.</div>
</li>
<li>Add 100 μL (1.0×) of resuspended Magbio HighPrep PCR beads and mix well using a vortex mixer or a pipette (pipet gently at least 10 times).</li>
<li>Incubate for 5 min at 18°C–20°C.</li>
<li>Spin tubes in a microcentrifuge and place on a magnetic rack to separate beads from the supernatant. After the solution clears (about 5 min), discard the supernatant that contains unwanted fragments.
<div class="critical">
<span class="critical-title">Critical:</span> Do not discard the beads, which bind to the target fragments.</div>
</li>
<li>Add 200 μL of freshly prepared 80% ethanol while the tubes are on the magnetic rack. Incubate at 18°C–20°C for 30 s.</li>
<li>Carefully remove the supernatant without disturbing the beads and discard.</li>
<li>Repeat washing step f.</li>
<li>Spin briefly and place tubes onto the magnetic rack.</li>
<li>Remove residual ETOH without disturbing the beads and air dry for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> Do not exceed 5 min; over drying the beads may result in lower recovery of target DNA.</div>
</li>
<li>Remove tubes from the rack and elute DNA in 23 μL Buffer EB. Mix well on a vortex mixer or by pipetting, and incubate for 2 min at 18°C–20°C.</li>
<li>Place tubes onto the magnetic rack until the solution appear clear.</li>
<li>Transfer 20 μL of supernatant to a clean tube and discard the beads.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The purified samples can be stored overnight and up to few days at −20° before the transposition reaction.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Transposition reaction and purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>Following the optimization in steps 16–20, in this step the transposition reaction is completed using the optimal transposase enzyme concentration, followed by a purification step.</p>
<ol start="21">
<li>Immediately following the nuclei preparation, resuspend the pellet in the transposase reaction mix optimized in step 3 [25 μL 2× TD buffer; 0.63 μL transposase (100 nM final concentration); 22.5 μL of nuclease free water].</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The pellet has to be mixed gently at least six times.</div>
<div class="note">
<span class="note-title">Note:</span> The transposase concentration used here is optimized for tissue obtained from 7 last instar Daphnia juveniles.</div>
<ol start="22">
<li>Incubate the transposition reaction at 37°C for 30 min in a thermomixer using 1,000 rpm mixing speed.</li>
<li>Store samples at −20°C until further processing</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The transposition reaction can be stored at −20° until further processing.</div>
<ol start="24">
<li>Purify samples using a 1:1 volume Magbio HighPrep PCR beads (Auto Q Biosciences) as follows:
<ol type="a">
<li>Place Magbio HighPrep PCR beads at 18°C–20°C for 30 min</li>
<li>Add nuclease-free water to the ligation reaction bringing the reaction volume to 100 μL.
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the final volume is 100 μL before adding Magbio HighPrep PCR beads.</div>
</li>
<li>Add 100 μL (1.0×) of resuspended Magbio HighPrep PCR beads and mix well using a vortex mixer or a pipette (pipet gently at least 10 times).</li>
<li>Incubate for 5 min at 18°C–20°C.</li>
<li>Spin tubes in a microcentrifuge and place on a magnetic rack to separate beads from the supernatant. After the solution clears (about 5 min), discard the supernatant that contains unwanted fragments.
<div class="critical">
<span class="critical-title">Critical:</span> Do not discard the beads, which bind to the target fragments.</div>
</li>
<li>Add 200 μL of freshly prepared 80% ethanol while the tubes are on the magnetic rack. Incubate at room temperature for 30 s.</li>
<li>Carefully remove the supernatant without disturbing the beads and discard.</li>
<li>Repeat washing step f.</li>
<li>Spin briefly and place tubes onto the magnetic rack.</li>
<li>Remove residual ETOH without disturbing the beads and air dry for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> Do not exceed 5 min; over drying the beads may result in lower recovery of target DNA.</div>
</li>
<li>Remove tubes from the rack and elute DNA in 23 μL Buffer EB. Mix well on a vortex mixer or by pipetting, and incubate for 2 min at 18°C–20°C.</li>
<li>Place tubes onto the magnetic rack until the solution appear clear.</li>
<li>Transfer 20 μL of supernatant to a clean tube and discard the beads.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The purified samples can be stored at −20°C before the amplification step.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Library amplification optimization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5 h</div>
<p>An optimization of the PCR cycling of transposed DNA is advised as suboptimal amplification or overamplification can introduce biases in the DNA fragments distribution within the libraries. For example, in case of over amplification small DNA fragments are preferentially amplified. Consequently, the DNA libraries suffer from reduced complexity.</p>
<p>Complete 5 PCR cycles following the cycling conditions below and using NEBNext High-Fidelity PCR master mix and 25 μM Nextera PCR primers: 25 μL NEBNext High-Fidelity 2× PCR Master Mix; 10 μL transposed DNA; 2.5 μL (25 μM) Forward Primer (<a href="#tbl1">Table 1</a>); 2.5 μL (25 μM) Reverse Primer; 10 μL nuclease-free H<sub>2</sub>O.</p>
<ol start="25">
<li>Optimization step 1: PCR cycling
<ol type="a">
<li>1 cycle
<ol type="i">
<li>30 s, 98°C</li>
</ol>
</li>
<li>5 cycles
<ol type="i">
<li>10 s, 98°C</li>
<li>30 s, 63°C</li>
<li>60 s, 72°C</li>
<li>Hold at 4°C until optimization step 2 is complete.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="26">
<li>Optimization step 2: qPCR to determine the optimal number of PCR cycles<br> Perform a qPCR (5 μL NEBNext High-Fidelity 2× PCR Master Mix; 5 μL of amplified DNA from optimization step 1; 0.25 μL (25 μM) Nextera Forward PCR Primer; 0.25 μL (25 μM) Nextera Reverse PCR Primer; 0.09 μL 100 x SYBR Green I; 4.41 μL nuclease-free H<sub>2</sub>O) with the following cycling program:
<ol type="a">
<li>1 cycle
<ol type="i">
<li>30 s, 98°C</li>
</ol>
</li>
<li>20 cycles
<ol type="i">
<li>10 s, 98°C</li>
<li>30 s, 63°C</li>
<li>60 s, 72°C</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="27">
<li>Calculate the total number of PCR cycles and compete step 25. To determine the optimal number of PCR cycles, plot the qPCR fluorescent signal, identify the value corresponding to 1/3 of the plateau fluorescent value (<a href="#fig1">Figure 1</a>, red line). The number of PCR cycles to complete the optimization step 25 above is the number of cycles that intersects the threshold (red line). If this value falls between two cycles, the lower cycle should be preferred.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/475-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Library amplification optimization via qPCR</div>
<p>Fluorescence plot used in point 5 of the ATAC protocol to optimize DNA library amplification steps and prevent over or underamplification, which can causes biases in the DNA fragment representation within libraries. To determine the optimal number of PCR cycles, we plot the qPCR fluorescent signal, and identify the value corresponding to 1/3 of the plateau fluorescent value (red line). The number of PCR cycles to achieve an optimal amplification is the number of cycles that intersects the threshold (red line). If this value falls between two cycles, the lower cycle should be preferred. The vertical red line is the number of cycles that intersects the threshold.</p>
</figcaption>
</figure>
</li>
</ol>
<ol start="28">
<li>Purify amplified libraries using a 1:1 volume Magbio HighPrep PCR beads (Auto Q Biosciences) as follows:
<ol type="a">
<li>Place Magbio HighPrep PCR beads at 18°C–20°C for 30 min</li>
<li>Add nuclease-free water to the ligation reaction bringing the reaction volume to 100 μL.
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the final volume is 100 μL before adding Magbio HighPrep PCR beads.</div>
</li>
<li>Add 100 μL (1.0×) of resuspended Magbio HighPrep PCR beads and mix well using a vortex mixer or a pipette (pipet gently at least 10 times).</li>
<li>Incubate for 5 min at 18°C–20°C.</li>
<li>Spin tubes in a microcentrifuge and place on a magnetic rack to separate beads from the supernatant. After the solution clears (about 5 min), discard the supernatant that contains unwanted fragments.
<div class="critical">
<span class="critical-title">Critical:</span> Do not discard the beads, which bind to the target fragments.</div>
</li>
<li>Add 200 μL of freshly prepared 80% ethanol while the tubes are on the magnetic rack. Incubate at 18°C–20°C for 30 s.</li>
<li>Carefully remove the supernatant without disturbing the beads and discard.</li>
<li>Repeat washing step f.</li>
<li>Spin briefly and place tubes onto the magnetic rack.</li>
<li>Remove residual ETOH without disturbing the beads and air dry for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> Do not exceed 5 min; over drying the beads may result in lower recovery of target DNA.</div>
</li>
<li>Remove tubes from the rack and elute DNA in 23 μL Buffer EB. Mix well on a vortex mixer or by pipetting, and incubate for 2 min at 18°C–20°C.</li>
<li>Place tubes onto the magnetic rack until the solution appear clear.</li>
<li>Transfer 20 μL of supernatant to a clean tube and discard the beads.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> the purified libraries can be stored at -20<b>°</b>C before the amplification step</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.6">Library amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>In this step, DNA libraries are amplified to generate sufficient material for next generation sequencing (NGS). This step should follow an optimization of the PCR conditions as explained in point 5.</p>
<ol start="29">
<li>Following DNA purification, amplify DNA libraries using NEBNext High-Fidelity PCR master mix and 25 μM Nextera PCR primers: 25 μL NEBNext High-Fidelity 2× PCR Master Mix; 10 μL transposed DNA; 2.5 μL 25 μM Forward PCR Primer; 2.5 μL 25 μM Reverse PCR Primer; 10 μL nuclease-free H<sub>2</sub>O (<a href="#tbl1">Table 1</a>), using the following PCR cycling:
<ol type="a">
<li>1 cycle
<ol type="i">
<li>5 min, 72°C</li>
<li>30 s, 98°C</li>
</ol>
</li>
<li>12 cycles
<ol type="i">
<li>10 s, 98°C</li>
<li>30 s, 63°C</li>
<li>60 s, 72°C</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The optimal number of cycles for non-saturated libraries obtained from 7 last instar Daphnia individuals is 12. The number of cycles may change with the input material and should be optimized using step 29 above.</div>
<ol start="30">
<li>Purify amplified libraries using a 1:1 volume Magbio HighPrep PCR beads (Auto Q Biosciences) as follows:
<ol type="a">
<li>Place Magbio HighPrep PCR beads at 18°C–20°C for 30 min</li>
<li>Add nuclease-free water to the ligation reaction bringing the reaction volume to 100 μL.
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the final volume is 100 μL before adding Magbio HighPrep PCR beads.</div>
</li>
<li>Add 100 μL (1.0×) of resuspended Magbio HighPrep PCR beads and mix well using a vortex mixer or a pipette (pipet gently at least 10 times).</li>
<li>Incubate for 5 min at 18°C–20°C.</li>
<li>Spin tubes in a microcentrifuge and place on a magnetic rack to separate beads from the supernatant. After the solution clears (about 5 min), discard the supernatant that contains unwanted fragments.
<div class="critical">
<span class="critical-title">Critical:</span> Do not discard the beads, which bind to the target fragments.</div>
</li>
<li>Add 200 μL of freshly prepared 80% ethanol while the tubes are on the magnetic rack. Incubate at 18°C–20°C for 30 s.</li>
<li>Carefully remove the supernatant without disturbing the beads and discard.</li>
<li>Repeat washing step f.</li>
<li>Spin briefly and place tubes onto the magnetic rack.</li>
<li>Remove residual ETOH without disturbing the beads and air dry for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> Do not exceed 5 min; over drying the beads and may result in lower recovery of target DNA<b>.</b>
</div>
</li>
<li>Remove tubes from the rack and elute DNA in 23 μL Buffer EB. Mix well on a vortex mixer or by pipetting, and incubate for 2 min at 18°C–20°C.</li>
<li>Place tubes onto the magnetic rack until the solution appear clear.</li>
<li>Transfer 20 μL of supernatant to a clean tube and discard the beads.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The purified libraries can be stored at −20°C before next generation sequencing.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.7">Next generation sequencing</h3>
<p>In this step, DNA libraries are prepared for next generation sequencing using Illumina or Illumina- compatible platforms.</p>
<ol start="31">
<li>Assess the quality and size of the ATAC libraries using a High Sensitivity D1000 Screentape (Agilent - 5067- 5584) and High Sensitivity D1000 Reagents (Agilent - 5067- 5585) on a Tapestation 2200 (Agilent, G2964AA).</li>
<li>Quantitate libraries on a Qubit 3.0 Fluorometer (Invitrogen, Q33216) using Qubit 1× dsDNA HS Assay Kit (Invitrogen, Q33231). Use the average library size from the Tapestation QC test to determine the final molar concentration. Other quantitation approaches can be used (e.g., qPCR). Here, we used Qubit combined with Tapestation traces because it allows a higher throughput and is more cost-effective. Information from the Tapestation traces allow to assess the fragment distribution within libraries. A bioanalyzer can be used in alternative to the Tapestation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Tapestation traces appear smooth with a peak at 200–400 bp, depending on the insert size used in the library preparation (see below). The same libraries run on a bioanalyzer tend to show a more laddered profile (data not shown). The profile can vary among samples with some samples showing a wider peak; however, these difference do not affect the quality of the libraries.</div>
<ol start="33">
<li>Prepare equimolar pools of ATAC libraries ensuring that different sets of unique paired indexes are used for different libraries.</li>
<li>Assess the quality and size of the ATAC library pool on Tapestation 2200 (Agilent, G2964AA) using High Sensitivity D1000 Screentape (Agilent - 5067- 5584) and High Sensitivity D1000 Reagents (Agilent - 5067- 5585).</li>
<li>Quantitate ATAC library pool using Qubit 3.0 Fluorometer (Invitrogen, Q33216) and Qubit 1× dsDNA HS Assay Kit (Invitrogen, Q33231).</li>
<li>Optional step. If necessary, pools can be concentrated using a 1:1 HighPrep bead purification described above (step 32).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The pooled libraries can be stored at −20°C until further processing.</div>
<ol start="37">
<li>Run prepared 2 × 250 bp ATAC libraries on an Illumina or other compatible platform to generate 40 million reads per sample. A HiSeqXten was used here.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A test run on an Illumina MiSeq platform is advised to validate the protocol before processing large sample numbers. The libraries constructed using different concentrations of the transposase enzyme (see <a href="#expected-outcomes">Expected outcomes</a>) were sequenced at low coverage. The normalized ATAC-seq traces from these low coverage libraries were compared with a high coverage sample (40 million reads) at four scaffolds (see <a href="#expected-outcomes">Expected outcomes</a>).</div>
<h3 id="sec3.8">Bioinformatics analysis: ATAC-seq standards and processing pipeline</h3>
<p>This step describes the processing pipeline used to perform quality checks on the ATAC sequences and to generate summary metrics, for which we followed the ENCODE guidelines (<a href="https://www.encodeproject.org/atac-seq/">https://www.encodeproject.org/atac-seq/</a>), where possible.</p>
<div class="note">
<span class="note-title">Note:</span> Metrics that require a chromosomal-level assembly of the reference genome could not be calculated as the reference genome of D. magna 2.4 is not resolved at chromosomal level.</div>
<ol start="38">
<li>Perform an initial sequence quality check using fastqc (V0.11.9) for each fastq files (<a href="#bib1">Andrews, 2010</a>).</li>
<li>Use Trimmomatic (v0.32) (<a href="#bib3">Bolger et al., 2014</a>) to remove Illumina adapters and low quality reads, as well as reads shorter than 50 bp using the following parameters: “ILLUMINACLIP:List_of_Adapaters.fa:2:30:10 LEADING:30 TRAILING:30 MINLEN:50 HEADCROP:10.”</li>
<li>Validate libraries, estimate sample replicate bias and quantify read quality following the gold standard of ENCODE (<a href="https://www.encodeproject.org/atac-seq">https://www.encodeproject.org/atac-seq</a>), where possible, and the ATAC-seq data analysis recommendations by galaxy projects (<a href="#bib2">Batut et al., 2018</a>) using multiqc (<a href="#bib11">Ewels et al., 2016</a>) and ataqv (<a href="#bib21">Orchard et al., 2020</a>).</li>
<li>After indexing your reference genome, use Bowtie2 (v2.2.6) (<a href="#bib17">Langmead et al., 2019</a>) ultra-fast aligner to map reads that passed the quality threshold to the reference genome using very sensitive search mode settings as follows: “-D 20 -R 3 -N 0 -L 20 -i S,1,0.50” .</li>
<li>Apply Samtools (V1.4) (<a href="#bib18">Li et al., 2009</a>) to convert Bowtie2 mapped SAM files to compressed binary format BAM; sort and index for further analysis.</li>
<li>Remove PCR duplicates from the sorted BAM files using the Broad Institute Picard tools (2.10.5) (<a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>).</li>
<li>Filter out r that map onto the mitochondrial DNA (NCBI: NC_026914.1) before calling ATAC peaks.</li>
<li>Quantify mapping quality statistics and visualize results using Qualimap 2 (<a href="#bib20">Okonechnikov et al., 2016</a>) (<a href="http://qualimap.conesalab.org/">http://qualimap.conesalab.org/</a>).</li>
<li>Shift aligned reads “4 , -5” and “5, -4” for positive and negative strand reads, respectively, using deeptools alignmentSieve (<a href="#bib23">Ramirez et al., 2018</a>).</li>
<li>Use MACS2 callpeak (<a href="#bib12">Gaspar, 2018</a>) to call the peaks from each replicate BAM file and sample; QC matrices are calculated on unmerged technical or biological replicates. For some downstream analyses (e.g., comparative peak analysis) replicates can be merged.</li>
<li>Filter, combine (where relevant) and convert peaks for visualization using bigwig in bedtools (<a href="#bib22">Quinlan, 2014</a>) and/or custom R scripts.</li>
<li>Quantify replicates concordance by calculating the Irreproducible Discovery Rate (IDR), and following the ENCODE processing pipeline for replicated or unreplicated samples as necessary (<a href="https://github.com/kundajelab/idr">https://github.com/kundajelab/idr</a>).</li>
<li>Visualize high-quality genome browser tracks on the focal species reference genome using Deeptools pyGenomeTracks (<a href="#bib23">Ramirez et al., 2018</a>).</li>
<li>Identify and extract Transcription Start Site (TSS) from the reference gene annotation file into deeptool compatible bed format. Scale peak traces across samples using deeptools computed matrix and visualize TSS heatmaps using plotHeatmap (<a href="#bib23">Ramirez et al., 2018</a>).</li>
<li>Search for enriched known motifs at transcription start sites (TSS) using HOMER discovery algorithm for regulatory elements (Hypergeometric Optimization of Motif EnRichment; <a href="#bib9">Duttke et al., 2019</a>). A search window spanning 200 bp upstream from the peak center with up to 12 bp motif length can be used following (<a href="#bib13">Heinz et al., 2010</a>).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Expected outcomes at critical steps of the protocol and optimization steps are described.</p>
<h3 id="sec4.1">Optimizing DNA input for ATAC-seq in <i>Daphnia</i> (related to step 1)</h3>
<p>The amount of starting material for ATAC-seq libraries was a critical step in our protocol optimization. As compared to mammalian or vertebrate models, the crustacean <i>Daphnia</i> has limited starting material. The whole adult organism measures less than 4 mm. <i>Daphnia</i> provides many of the felicities of invertebrate model species: they are easy and inexpensive to culture in the laboratory, have a short life cycle, and produce large numbers of externally laid embryos. <i>Daphnia</i> has a parthenogenetic life cycle, in which sexual and asexual reproduction alternate (<a href="#bib10">Ebert, 2005</a>). Sexual recombination results in early stage embryos that arrest their development and enter dormancy (<a href="#bib15">Kerfoot and Weider, 2004</a>). Dormant embryos can be revived and propagated clonally under standard laboratory conditions, allowing the rearing of populations of isogenic individuals (clones) from a single genotype (<a href="#bib7">Cambronero Cuenca and Orsini, 2018</a>). To determine the optimal amount of tissue for high-quality ATAC libraries, we took advantage of <i>Daphnia</i> clonality. We performed a titration experiment using 7, 15, 30 and 60 last instar female <i>Daphnia</i> juveniles from the same genotype (<a href="#fig2">Figure 2</a>). The quality of ATAC libraries was assessed for: i) animal tissue manually homogenized using a pestle; ii) tissue treated with a tissue homogenizer (Geno/Grinder); and iii) tissue treated with automated tissue homogenizer followed by DNA shearing A successful ATAC library is expected to result in a clear peak between 200–300 bp. A larger size peak can be visible, representing unfragmented genomic DNA (&gt;1,000 bp).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/475-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Optimizing DNA input for ATAC-seq</div>
<p>A titration experiment was completed to identify the optimal input material (tissue) to generate a robust ATAC-seq library. The results of this titration experiment are shown as tape station traces of ATAC-seq libraries obtained from different starting material: 7, 15, 30, and 60 last instar <i>Daphnia</i> female juveniles, from a single genotype. These different starting materials were tested with three tissue homogenization techniques: manual homogenization with a pestle, automated tissue homogenizer (Geno/Grinder) and a combination of automated tissue homogenizer (Geno/Grinder) and DNA sharing using a pico-sonicator (Bioruptor). The scale on the y axis may differ. The scale on the x axis shows size in base pairs (bp).</p>
</figcaption>
</figure>
<p>Libraries obtained from tissue treated with an automated tissue homogenizer (Geno/Grinder) were more replicable in quality and quantity of material obtained than libraries obtained from tissue homogenized manually (<a href="#fig2">Figure 2</a>). The combined use of a tissue homogenizer and a picosonicator did not visibly improve the quality of the libraries when 7 and 15 individual <i>Daphnia</i> were used (<a href="#fig2">Figure 2</a>), whereas it improved the quality of the ATAC libraries when more than 30 individuals were used (<a href="#fig2">Figure 2</a>). Even when tissue homogenizer combined with DNA shearing was used, the ATAC libraries were of lower quality when more than 30 individuals were used as compared to &lt;15 individual, likely because of suboptimal homogenization of tissue and suboptimal access to free nuclei (<a href="#fig2">Figure 2</a>; 60 individuals).</p>
<h3 id="sec4.2">Optimizing transposase concentration (related to step 3)</h3>
<p>The prohibitive costs of the enzyme transposase impose serious limitations to the application of open chromatin analysis in population epigenomics. We perform a titration experiment to determine the minimal amount of transposase enzyme needed to generate good quality ATAC-seq libraries. Dilutions corresponding to 100%, 50%, 25%, and 12% of the manufacturer suggested concentrations were used. These dilutions were tested on libraries prepared with a tissue homogenizer step and a 16–20 h (overnight) transposase reaction (<a href="#fig3">Figures 3</a>A–3D) as well as on libraries prepared using a tissue homogenizer and a picosonicator (<a href="#fig3">Figures 3</a>E–3H). There were no appreciable differences among 100%, 50%, and 25% dilutions and between the two methods used (<a href="#fig3">Figure 3</a>). The amount of libraries obtained was also comparable (<a href="#fig3">Figure 3</a>). However, whereas the 12% dilution did not produce an acceptable quality library with transposase alone, it produce an acceptable quality library in the transposase combined with a picosonicator step (<a href="#fig3">Figures 3</a>D and 3H). For the 100% and 25% dilutions, the transposase incubation time was reduced to 30 min as per manufacturer guidelines (<a href="#fig3">Figures 3</a>A1 and 3C1). The quality of these libraries was comparable to the ones obtained with16–20 h (overnight) incubation (<a href="#fig3">Figures 3</a>A–3D), even if they resulted in a comparatively lower yield.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/475-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Optimizing transposase dilution in ATAC-seq libraries</div>
<p>Population epigenomics is largely unexplored due to the prohibitive costs of the transposase enzyme. Here we show the outcome of a titration experiment for different dilution of the transposase enzyme suggested by the manufacturer. ATAC-seq tape station traces are shown for libraries obtained with different dilutions of transposase: 100%, 50%, 25%, and 12% of the manufacturer suggested concentration. (A–D) ATAC libraries obtained with an over/night (O/N) incubation of the transposition reaction preceded by an automated tissue homogenization step; (E–H) libraries obtained with an over/night (O/N) incubation of the transposition reaction preceded by an automated tissue homogenization step combined with pico-sonication (DNA shearing); A1 and C1: ATAC libraries obtained for 100% and 25% dilutions of the transposase with an incubation of the transposition reaction of 30 min.</p>
</figcaption>
</figure>
<h4 id="sec4.2.1">Validation of ATAC-seq modified protocol (related to step 7)</h4>
<p>ATAC-seq libraries obtained from the titration experiments in point b (<a href="#fig3">Figure 3</a>) were sequenced at low coverage to assess whether they produced qualitatively comparable results, namely replicable peak traces. The same genotype sequenced at high depth of coverage (40 million reads) was used as reference in this analysis (<a href="#fig4">Figure 4</a>, R). We then compared chromatin profiles in the reference and in the low coverage samples. ATAC fragments were consistently observed across the low coverage and the reference samples, indicating that the tested range of DNA input and the dilutions of transposase tested did not significantly alter the quality of ATAC libraries and ATAC-seq peak traces (<a href="#fig4">Figure 4</a>). We did not do any quantitative analyses on these data because of the low depth of sequencing.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/475-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Normalized ATAC-seq traces for four scaffolds on the <i>Daphnia magna</i> genome</div>
<p>Normalized ATAC-seq traces shown for 4 scaffolds on the <i>Daphnia magna</i> genome (v2.4) obtained from the ATAC-seq libraries in <a href="#fig3">Figure 3</a>. These traces are obtained from low coverage libraries and compared to a high coverage library obtained from the same <i>Daphnia</i> genotype R, used as reference.</p>
</figcaption>
</figure>
<p>To validate the modified ATAC-seq protocol for downstream applications, we constructed ATAC-seq libraries using 25% dilution of transposase and 7 last instar individuals from 3 <i>D. magna</i> genotypes. Taking advantage of the properties of <i>Daphnia</i>, namely the opportunity to generate clonal copies of each genotype, we sequenced 6 ATAC-seq libraries with high depth of coverage (40 million reads per sample) from an exposure experiment in which clonal replicates of three genotypes (LRV2_1; LRV3.5_15 and LRV13_3) were either maintained in control conditions (not exposed), or exposed to 8 μg/L of a carbamate insecticide. This insecticide has been previously shown to have an adverse effect on <i>Daphnia</i> fitness within a single generation (<a href="#bib6">Cambronero et al., 2018</a>; <a href="#bib14">Jansen et al., 2011</a>). The six libraries were first used to generate data quality metrics, and then for peak and motif analyses.</p>
<h5 id="sec4.2.1.1">Metrics</h5>
<p>We assessed the quality of our ATAC sequences using the golden standard metrics of the ENCODE project (<a href="https://www.encodeproject.org/atac-seq/">https://www.encodeproject.org/atac-seq/</a>) and the ATAC-seq data analysis recommendations by galaxy projects, where possible. ENCODE metrics that require a chromosomal-level genome assembly could not be used. The metrics calculated following the ENCODE standard are shown in the following. Additional sequence metrics are in <a href="#tbl2">Table 2</a>. All metrics support that the modified ATAC-seq protocol optimized here can be used for population epigenomics of invertebrates.</p>
<table id="tbl2">
<caption>Table 2. Sequence metrics</caption>
<thead>
<tr>
<th>Sample name</th>
<th>No. of raw paired end reads</th>
<th>No. of QC paired end reads</th>
<th>Insert size median</th>
<th>Coverage mean</th>
<th>Mapped reads %</th>
<th>No. of reads mapped to the mitochondria</th>
<th>No. of peaks</th>
</tr>
</thead>
<tbody>
<tr>
<td>LRV13_3R1_C</td>
<td>18,635,973 x 2</td>
<td>16,135,365 x 2</td>
<td>151</td>
<td>24,5574</td>
<td>91.33%</td>
<td>17,974</td>
<td>61,285</td>
</tr>
<tr>
<td>LRV13_3-R2C</td>
<td>29,340,610 x 2</td>
<td>25,005,090 x 2</td>
<td>153</td>
<td>37,6782</td>
<td>91.84%</td>
<td>19,888</td>
<td>65,281</td>
</tr>
<tr>
<td>LRV13_3R1_I</td>
<td>16,668,003 x 2</td>
<td>12,773,098 x 2</td>
<td>131</td>
<td>16,3577</td>
<td>89.55%</td>
<td>24,572</td>
<td>59,735</td>
</tr>
<tr>
<td>LRV13_3R2_I</td>
<td>13,709,892 x 2</td>
<td>10,100,948 x 2</td>
<td>123</td>
<td>12,4835</td>
<td>88.91%</td>
<td>21,810</td>
<td>60,661</td>
</tr>
<tr>
<td>LRV2_1R1_C</td>
<td>12,140,926 x 2</td>
<td>10,137,006 x 2</td>
<td>197</td>
<td>16,1945</td>
<td>91.36%</td>
<td>3,280</td>
<td>28,993</td>
</tr>
<tr>
<td>LRV2_1R2_C</td>
<td>14,302,926 x 2</td>
<td>11,238,267 x 2</td>
<td>168</td>
<td>15,8192</td>
<td>90.47%</td>
<td>7,696</td>
<td>43,685</td>
</tr>
<tr>
<td>LRV2_1R1_I</td>
<td>12,388,306 x 2</td>
<td>7,082,046 x 2</td>
<td>111</td>
<td>7,4666</td>
<td>83.23%</td>
<td>21,434</td>
<td>61,512</td>
</tr>
<tr>
<td>LRV2_1R2_I</td>
<td>13,813,597 x 2</td>
<td>9,790,744 x 2</td>
<td>129</td>
<td>11,1734</td>
<td>83.20%</td>
<td>21,059</td>
<td>58,803</td>
</tr>
<tr>
<td>LRV3_5_15R1_C</td>
<td>18,483,584 x 2</td>
<td>13,824,157 x 2</td>
<td>155</td>
<td>18,9622</td>
<td>93.69%</td>
<td>8,435</td>
<td>51,683</td>
</tr>
<tr>
<td>LRV3_5_15R2_C</td>
<td>25,080,326 x 2</td>
<td>20,644,243 x 2</td>
<td>174</td>
<td>30,8094</td>
<td>93.78%</td>
<td>7,970</td>
<td>48,057</td>
</tr>
<tr>
<td>LRV3_5_15R1_I</td>
<td>14,942,965 x 2</td>
<td>10,405,452 x 2</td>
<td>129</td>
<td>13,1903</td>
<td>92.63%</td>
<td>20,251</td>
<td>55,355</td>
</tr>
<tr>
<td>LRV3_5_15R2_I</td>
<td>13,019,186 x 2</td>
<td>8,165,030 x 2</td>
<td>129</td>
<td>10,0822</td>
<td>92.68%</td>
<td>17,601</td>
<td>53,166</td>
</tr>
</tbody>
</table>
<p class="table-legend">QC metrics for NGS data generated from the high coverage samples (LRV2_1; LRV3.5_15 and LRV13_3) obtained from an exposure experiment in which clonal replicates were either exposed to the insecticide Carabaryl (8μg/L) or maintained in non-stress (control) conditions. C = control; I = insecticide carbaryl; R1 = technical replicate 1; R2 = technical replicate 2.</p>
<table id="undtbl6">
<thead>
<tr>
<th>ENCODE standard</th>
<th>Current paper</th>
</tr>
</thead>
<tbody>
<tr>
<td>Experiments should have two or more biological replicates or at least two technical replicates</td>
<td>Three biological replicated and two technical replicates per samples</td>
</tr>
<tr>
<td>Each replicate should have 25 million on-duplicate, non-mitochondrial aligned reads</td>
<td>Reads per replica ranged between 14 and 50 million reads (Average read per replica: 26 million reads)</td>
</tr>
<tr>
<td>Percentage of mapped reads should be greater than 95%, though values &gt;80% may be acceptable</td>
<td>Percentage of mapped reads ranged between 83.2% and 93.8%</td>
</tr>
<tr>
<td>Irreproducible Discovery Rate (IDR) should be less than 2</td>
<td>IDR &lt;1.6</td>
</tr>
<tr>
<td>The number of peaks within a replicated peak file should be &gt;150,000, though values &gt;100,000 may be acceptable</td>
<td>&gt;50,000 (corrected for the genome size of <i>D. magna</i>, 130 Mb; this value is over the quality threshold indicated)</td>
</tr>
<tr>
<td>The number of peaks within an IDR peak file should be &gt;70,000, though values &gt;50,000 may be acceptable</td>
<td>&gt; 11,700 (corrected for the genome size of <i>D. magna</i>, 130 Mb; this value is over the quality threshold indicated)</td>
</tr>
</tbody>
</table>
<h5 id="sec4.2.1.2">Peak and motif analysis</h5>
<p>We used the six high coverage samples to identify divergence in chromatin accessibility driven by exposure to a carbamate insecticide (carbaryl). To visualize changes in chromatin signatures, we compare the normalized ATAC-seq traces of exposed and unexposed genotypes (<a href="#fig5">Figure 5</a>). As <i>Daphnia</i> has a parthenogenetic life cycle, genotypes are genetically distinct but can be propagated clonally, therefore clonal copies of the same genotype can be exposed to different environments. Genome-wide counts of chromatin motifs were quantified in exposed and non-exposed genotypes and summarized in a principal component analysis (<a href="#fig6">Figure 6</a>). Treated and control animals were clearly distinguishable in the PCA plot (<a href="#fig6">Figure 6</a>A).Treatment-specific ATAC traces were identified in a Venn diagram including all genotypes (control and treatment) (<a href="#fig6">Figure 6</a>B).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/475-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Normalized ATAC-seq traces of three <i>Daphnia</i> genotypes exposed to the insecticide carbaryl</div>
<p>Normalized ATAC traces are shown for three genotypes of <i>D. magna</i> (LRV2_1; LRV3.5_15 and LRV13_3) for a representative scaffold of ca. 400 kb (Scaffold 2244). For each genotype a clonal replicate exposed to 8 μg/L of the insecticide carbaryl (I) and a non-exposed clonal replicate (C) are shown. These traces are obtained from high coverage libraries (40 million reads).</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/475-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Chromatin accessibility patterns in <i>Daphnia magna</i>
</div>
<p>(A and B) (A) Principal component analysis of genome-wide accessibility variation at three <i>D. magna</i> genotypes (LRV2_1; LRV3.5_15 and LRV13_3). Clonal replicates of the three genotypes exposed to the insecticide carbaryl (I; 8 μg/L) as well as non-exposed clonal replicates (C) are shown; (B) Venn diagram showing unique ATAC normalized traces identified in <i>D. magna</i> genotypes exposed to the insecticide carbaryl (I), as well as traces shared between treated (I) and untreated (C) genotypes.</p>
</figcaption>
</figure>
<p>A heatmap of the genome-wide TSS motifs, scaled using deeptools and visualized with plotHeatmap, show a distinct pattern for exposed and non-exposed copies of the same genotypes, identifying a clear regulatory response to the experimental perturbation imposed by the insecticide carbaryl (<a href="#fig7">Figure 7</a>). A list of known motifs searched against the HOMER discovery algorithm for regulatory elements (Hypergeometric Optimization of Motif EnRichment; (<a href="#bib9">Duttke et al., 2019</a>)) are listed in <a href="#tbl3">Table 3</a> for control and exposed samples (LRV2_1; LRV3.5_15 and LRV13_3).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/475-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Heatmaps of transcription start sites (TSS)</div>
<p>Distribution of genome-wide DNA motifs shared among three <i>Daphnia magna</i> genotypes exposed to the insecticide carbaryl (8 μg/L) (LRV3.5_15I, LRV2_1I and LRV13_3I) and maintained in non-stressful conditions (LRV3.5_15C, LRV2_1C, and LRV13_3C). Peak traces across samples were scaled using deeptools and visualized with plotHeatmap. TSS were scaled to include 1 kb up and downstream of the TSS. Similarity decreases from red to blue.</p>
</figcaption>
</figure>
<table id="tbl3">
<caption>Table 3. Known motifs enrichment results</caption>
<thead>
<tr>
<th>Motif name</th>
<th>Consensus</th>
<th>Log p value</th>
<th>q value (Benjamini)</th>
<th># of target sequences with motif (of 10,325)</th>
<th>% of target sequences with motif</th>
<th># of background sequences with motif (of 32,685)</th>
<th>% of background sequences with motif</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="8">Control</td>
</tr>
<tr>
<td>NFIL3(bZIP)/HepG2-NFIL3-ChIP-Seq(Encode)/Homer</td>
<td>VTTACGTAAYNNNNN</td>
<td>−7.14E+02</td>
<td>0</td>
<td>2,868</td>
<td>27.77%</td>
<td>4,448.9</td>
<td>13.61%</td>
</tr>
<tr>
<td>Foxh1(Forkhead)/hESC-FOXH1-ChIP-Seq(GSE29422)/Homer</td>
<td>NNTGTGGATTSS</td>
<td>−1.28E+03</td>
<td>0</td>
<td>3,435</td>
<td>33.27%</td>
<td>4,471.3</td>
<td>13.68%</td>
</tr>
<tr>
<td>AT2G20400(G2like)/colamp-AT2G20400-DAP-Seq(GSE60143)/Homer</td>
<td>DNVGAATATTCBNHN</td>
<td>−6.69E+02</td>
<td>0</td>
<td>1,164</td>
<td>11.27%</td>
<td>1,050.5</td>
<td>3.21%</td>
</tr>
<tr>
<td>AT1G76880(Trihelix)/col-AT1G76880-DAP-Seq(GSE60143)/Homer</td>
<td>ACGGTAAAAW</td>
<td>−6.13E+02</td>
<td>0</td>
<td>2,177</td>
<td>21.08%</td>
<td>3,130.6</td>
<td>9.58%</td>
</tr>
<tr>
<td>At2g03500(G2like)/col-At2g03500-DAP-Seq(GSE60143)/Homer</td>
<td>WWAGAATATTCT</td>
<td>−5.68E+02</td>
<td>0</td>
<td>1,315</td>
<td>12.73%</td>
<td>1,457.9</td>
<td>4.46%</td>
</tr>
<tr>
<td>At5g29000(G2like)/col-At5g29000-DAP-Seq(GSE60143)/Homer</td>
<td>RGAATATTCYHH</td>
<td>−5.35E+02</td>
<td>0</td>
<td>1370</td>
<td>13.27%</td>
<td>1,618.3</td>
<td>4.95%</td>
</tr>
<tr>
<td>JGL(C2H2)/col-JGL-DAP-Seq(GSE60143)/Homer</td>
<td>ACYTTCAGTT</td>
<td>−4.94E+02</td>
<td>0</td>
<td>3,956</td>
<td>38.31%</td>
<td>7,971.9</td>
<td>24.39%</td>
</tr>
<tr>
<td>Foxa3(Forkhead)/Liver-Foxa3-ChIP-Seq(GSE77670)/Homer</td>
<td>BSNTGTTTACWYWGN</td>
<td>−3.54E+02</td>
<td>0</td>
<td>1,701</td>
<td>16.47%</td>
<td>2,746.3</td>
<td>8.40%</td>
</tr>
<tr>
<td>FOXP1(Forkhead)/H9-FOXP1-ChIP-Seq(GSE31006)/Homer</td>
<td>NYYTGTTTACHN</td>
<td>−3.12E+02</td>
<td>0</td>
<td>2,146</td>
<td>20.78%</td>
<td>3,964.4</td>
<td>12.13%</td>
</tr>
<tr>
<td>At3g04030(G2like)/col-At3g04030-DAP-Seq(GSE60143)/Homer</td>
<td>DRGAATCT</td>
<td>−2.84E+02</td>
<td>0</td>
<td>1,903</td>
<td>18.43%</td>
<td>3,465.8</td>
<td>10.60%</td>
</tr>
<tr>
<td>At1g25550(G2like)/colamp-At1g25550-DAP-Seq(GSE60143)/Homer</td>
<td>NAGATTCY</td>
<td>−2.79E+02</td>
<td>0</td>
<td>1,921</td>
<td>18.60%</td>
<td>3,530.5</td>
<td>10.80%</td>
</tr>
<tr>
<td>FOXK2(Forkhead)/U2OS-FOXK2-ChIP-Seq(E-MTAB-2204)/Homer</td>
<td>SCHTGTTTACAT</td>
<td>−2.58E+02</td>
<td>0</td>
<td>2,290</td>
<td>22.18%</td>
<td>4,558.8</td>
<td>13.95%</td>
</tr>
<tr>
<td>FUS3(ABI3VP1)/col-FUS3-DAP-Seq(GSE60143)/Homer</td>
<td>DNNWTNTGCATGKNN</td>
<td>−2.36E+02</td>
<td>0</td>
<td>1,323</td>
<td>12.81%</td>
<td>2,240.2</td>
<td>6.85%</td>
</tr>
<tr>
<td>At1g68670(G2like)/colamp-At1g68670-DAP-Seq(GSE60143)/Homer</td>
<td>WNWWHNRAAGATTCT</td>
<td>−2.30E+02</td>
<td>0</td>
<td>1,419</td>
<td>13.74%</td>
<td>2,489.4</td>
<td>7.62%</td>
</tr>
<tr>
<td>Foxo1(Forkhead)/RAW-Foxo1-ChIP-Seq(Fan_et_al.)/Homer</td>
<td>CTGTTTAC</td>
<td>−2.13E+02</td>
<td>0</td>
<td>3,355</td>
<td>32.49%</td>
<td>7,721.5</td>
<td>23.63%</td>
</tr>
<tr>
<td>GTL1(Trihelix)/colamp-GTL1-DAP-Seq(GSE60143)/Homer</td>
<td>WWTTTACCKY</td>
<td>−1.89E+02</td>
<td>0</td>
<td>3,023</td>
<td>29.28%</td>
<td>6,935.8</td>
<td>21.22%</td>
</tr>
<tr>
<td>Foxo3(Forkhead)/U2OS-Foxo3-ChIP-Seq(E-MTAB-2701)/Homer</td>
<td>DGTAAACA</td>
<td>−1.86E+02</td>
<td>0</td>
<td>2,570</td>
<td>24.89%</td>
<td>5,689.8</td>
<td>17.41%</td>
</tr>
<tr>
<td>FoxD3(forkhead)/ZebrafishEmbryo-Foxd3.biotin-ChIP-seq(GSE106676)/Homer</td>
<td>TGTTTAYTTAGC</td>
<td>−1.81E+02</td>
<td>0</td>
<td>2,115</td>
<td>20.48%</td>
<td>4,483.8</td>
<td>13.72%</td>
</tr>
<tr>
<td>OCT:OCT-short(POU,Homeobox)/NPC-OCT6-ChIP-Seq(GSE43916)/Homer</td>
<td>ATGCATWATGCATRW</td>
<td>−1.65E+02</td>
<td>0</td>
<td>2,091</td>
<td>20.25%</td>
<td>4,511</td>
<td>13.80%</td>
</tr>
<tr>
<td>At5g04390(C2H2)/col200-At5g04390-DAP-Seq(GSE60143)/Homer</td>
<td>AGTGANDN</td>
<td>−1.55E+02</td>
<td>0</td>
<td>6473</td>
<td>62.69%</td>
<td>17,722.3</td>
<td>54.23%</td>
</tr>
<tr>
<td>Foxa2(Forkhead)/Liver-Foxa2-ChIP-Seq(GSE25694)/Homer</td>
<td>CYTGTTTACWYW</td>
<td>−1.48E+02</td>
<td>0</td>
<td>2,418</td>
<td>23.42%</td>
<td>5,514.6</td>
<td>16.87%</td>
</tr>
<tr>
<td>At3g24120(G2like)/col-At3g24120-DAP-Seq(GSE60143)/Homer</td>
<td>NWWAGMATMW</td>
<td>−1.21E+02</td>
<td>0</td>
<td>5,036</td>
<td>48.77%</td>
<td>13,498.4</td>
<td>41.30%</td>
</tr>
<tr>
<td>AT5G60130(ABI3VP1)/col-AT5G60130-DAP-Seq(GSE60143)/Homer</td>
<td>WTTYTAAGVAAA</td>
<td>−1.20E+02</td>
<td>0</td>
<td>3,896</td>
<td>37.73%</td>
<td>10,024.7</td>
<td>30.67%</td>
</tr>
<tr>
<td>At2g01060(G2like)/colamp-At2g01060-DAP-Seq(GSE60143)/Homer</td>
<td>AGATKCBNWW</td>
<td>−1.17E+02</td>
<td>0</td>
<td>5,572</td>
<td>53.96%</td>
<td>15,213.2</td>
<td>46.55%</td>
</tr>
<tr>
<td>RBFox2(?)/Heart-RBFox2-CLIP-Seq(GSE57926)/Homer</td>
<td>TGCATGCA</td>
<td>−1.10E+02</td>
<td>0</td>
<td>2,618</td>
<td>25.35%</td>
<td>6,366.5</td>
<td>19.48%</td>
</tr>
<tr>
<td>AT5G47660(Trihelix)/colamp-AT5G47660-DAP-Seq(GSE60143)/Homer</td>
<td>AWTTTTACCG</td>
<td>−1.10E+02</td>
<td>0</td>
<td>3,348</td>
<td>32.42%</td>
<td>8,493.7</td>
<td>25.99%</td>
</tr>
<tr>
<td>Fox:Ebox(Forkhead,bHLH)/Panc1-Foxa2-ChIP-Seq(GSE47459)/Homer</td>
<td>NNNVCTGWGYAAACASN</td>
<td>−9.71E+01</td>
<td>0</td>
<td>1,766</td>
<td>17.10%</td>
<td>4,066.8</td>
<td>12.44%</td>
</tr>
<tr>
<td>STZ(C2H2)/colamp-STZ-DAP-Seq(GSE60143)/Homer</td>
<td>HNBTCACT</td>
<td>−9.54E+01</td>
<td>0</td>
<td>6,447</td>
<td>62.43%</td>
<td>18,252.9</td>
<td>55.85%</td>
</tr>
<tr>
<td>At3g12730(G2like)/colamp-At3g12730-DAP-Seq(GSE60143)/Homer</td>
<td>AAGATTCT</td>
<td>−8.99E+01</td>
<td>0</td>
<td>2,458</td>
<td>23.80%</td>
<td>6,079.2</td>
<td>18.60%</td>
</tr>
<tr>
<td>AT5G45580(G2like)/colamp-AT5G45580-DAP-Seq(GSE60143)/Homer</td>
<td>ADRGAATCTH</td>
<td>−7.41E+01</td>
<td>0</td>
<td>3,038</td>
<td>29.42%</td>
<td>7,938.1</td>
<td>24.29%</td>
</tr>
<tr>
<td>GT2(Trihelix)/colamp-GT2-DAP-Seq(GSE60143)/Homer</td>
<td>AMGGTAAAWWWN</td>
<td>−7.10E+01</td>
<td>0</td>
<td>2,944</td>
<td>28.51%</td>
<td>7694.4</td>
<td>23.54%</td>
</tr>
<tr>
<td>FOXK1(Forkhead)/HEK293-FOXK1-ChIP-Seq(GSE51673)/Homer</td>
<td>NVWTGTTTAC</td>
<td>−6.86E+01</td>
<td>0</td>
<td>2,808</td>
<td>27.19%</td>
<td>7,319.6</td>
<td>22.40%</td>
</tr>
<tr>
<td>FOXM1(Forkhead)/MCF7-FOXM1-ChIP-Seq(GSE72977)/Homer</td>
<td>TRTTTACTTW</td>
<td>−6.43E+01</td>
<td>0</td>
<td>2,428</td>
<td>23.51%</td>
<td>6,249.7</td>
<td>19.12%</td>
</tr>
<tr>
<td>AT2G38300(G2like)/col-AT2G38300-DAP-Seq(GSE60143)/Homer</td>
<td>ADRGAATGTT</td>
<td>−6.01E+01</td>
<td>0</td>
<td>2,429</td>
<td>23.52%</td>
<td>6,299.9</td>
<td>19.28%</td>
</tr>
<tr>
<td>FoxL2(Forkhead)/Ovary-FoxL2-ChIP-Seq(GSE60858)/Homer</td>
<td>WWTRTAAACAVG</td>
<td>−4.43E+01</td>
<td>0</td>
<td>2,395</td>
<td>23.19%</td>
<td>6,396.6</td>
<td>19.57%</td>
</tr>
<tr>
<td>AP-2gamma(AP2)/MCF7-TFAP2C-ChIP-Seq(GSE21234)/Homer</td>
<td>SCCTSAGGSCAW</td>
<td>−4.38E+01</td>
<td>0</td>
<td>205</td>
<td>1.99%</td>
<td>322.3</td>
<td>0.99%</td>
</tr>
<tr>
<td>AP-2alpha(AP2)/Hela-AP2alpha-ChIP-Seq(GSE31477)/Homer</td>
<td>ATGCCCTGAGGC</td>
<td>−4.31E+01</td>
<td>0</td>
<td>150</td>
<td>1.45%</td>
<td>207.7</td>
<td>0.64%</td>
</tr>
<tr>
<td>KAN2(G2like)/colamp-KAN2-DAP-Seq(GSE60143)/Homer</td>
<td>ATATTCTY</td>
<td>−4.10E+01</td>
<td>0</td>
<td>2,289</td>
<td>22.17%</td>
<td>6,126.4</td>
<td>18.75%</td>
</tr>
<tr>
<td>FOXA1(Forkhead)/MCF7-FOXA1-ChIP-Seq(GSE26831)/Homer</td>
<td>WAAGTAAACA</td>
<td>−2.48E+01</td>
<td>0</td>
<td>2,438</td>
<td>23.61%</td>
<td>6,834.6</td>
<td>20.91%</td>
</tr>
<tr>
<td>Foxf1(Forkhead)/Lung-Foxf1-ChIP-Seq(GSE77951)/Homer</td>
<td>WWATRTAAACAN</td>
<td>−1.78E+01</td>
<td>0</td>
<td>2,621</td>
<td>25.38%</td>
<td>7,539.7</td>
<td>23.07%</td>
</tr>
<tr>
<td>KANADI1(Myb)/Seedling-KAN1-ChIP-Seq(GSE48081)/Homer</td>
<td>ARGAATAWWN</td>
<td>−1.77E+01</td>
<td>0</td>
<td>2,421</td>
<td>23.45%</td>
<td>6,932.4</td>
<td>21.21%</td>
</tr>
<tr>
<td>EBF(EBF)/proBcell-EBF-ChIP-Seq(GSE21978)/Homer</td>
<td>DGTCCCYRGGGA</td>
<td>−1.45E+01</td>
<td>0</td>
<td>29</td>
<td>0.28%</td>
<td>31.1</td>
<td>0.10%</td>
</tr>
<tr>
<td>EBF2(EBF)/BrownAdipose-EBF2-ChIP-Seq(GSE97114)/Homer</td>
<td>NABTCCCWDGGGAVH</td>
<td>−1.41E+01</td>
<td>0</td>
<td>178</td>
<td>1.72%</td>
<td>384.3</td>
<td>1.18%</td>
</tr>
<tr>
<td>Dorsal(RHD)/Embryo-dl-ChIP-Seq(GSE65441)/Homer</td>
<td>GGGAAAAMCCCG</td>
<td>−1.39E+01</td>
<td>0</td>
<td>130</td>
<td>1.26%</td>
<td>262.8</td>
<td>0.80%</td>
</tr>
<tr>
<td>MYB73(MYB)/col-MYB73-DAP-Seq(GSE60143)/Homer</td>
<td>NNNNHAACNGHHDHN</td>
<td>−1.38E+01</td>
<td>0</td>
<td>4,006</td>
<td>38.80%</td>
<td>11,938.3</td>
<td>36.53%</td>
</tr>
<tr>
<td>E2F4(E2F)/K562-E2F4-ChIP-Seq(GSE31477)/Homer</td>
<td>GGCGGGAAAH</td>
<td>−1.35E+01</td>
<td>0</td>
<td>421</td>
<td>4.08%</td>
<td>1,055</td>
<td>3.23%</td>
</tr>
<tr>
<td>Tlx?(NR)/NPC-H3K4me1-ChIP-Seq(GSE16256)/Homer</td>
<td>CTGGCAGSCTGCCA</td>
<td>−1.23E+01</td>
<td>0.0001</td>
<td>117</td>
<td>1.13%</td>
<td>238.7</td>
<td>0.73%</td>
</tr>
<tr>
<td>HDG7(HB)/col-HDG7-DAP-Seq(GSE60143)/Homer</td>
<td>WGCATTTAATGC</td>
<td>−1.20E+01</td>
<td>0.0001</td>
<td>509</td>
<td>4.93%</td>
<td>1,324.4</td>
<td>4.05%</td>
</tr>
<tr>
<td>AT2G40260(G2like)/colamp-AT2G40260-DAP-Seq(GSE60143)/Homer</td>
<td>WAAAYATTCTTT</td>
<td>−1.19E+01</td>
<td>0.0001</td>
<td>2,664</td>
<td>25.80%</td>
<td>7,829.9</td>
<td>23.96%</td>
</tr>
<tr>
<td>LBD2(LOBAS2)/colamp-LBD2-DAP-Seq(GSE60143)/Homer</td>
<td>TCCGAWTTTTTCGGN</td>
<td>−1.14E+01</td>
<td>0.0002</td>
<td>547</td>
<td>5.30%</td>
<td>1,442.8</td>
<td>4.41%</td>
</tr>
<tr>
<td>RKD2(RWPRK)/colamp-RKD2-DAP-Seq(GSE60143)/Homer</td>
<td>GACKTTTCRDCTTCC</td>
<td>−1.07E+01</td>
<td>0.0004</td>
<td>741</td>
<td>7.18%</td>
<td>2,020.9</td>
<td>6.18%</td>
</tr>
<tr>
<td>ZNF528(Zf)/HEK293-ZNF528.GFP-ChIP-Seq(GSE58341)/Homer</td>
<td>AGAAATGACTTCCCT</td>
<td>−9.87E+00</td>
<td>0.001</td>
<td>6</td>
<td>0.06%</td>
<td>2.4</td>
<td>0.01%</td>
</tr>
<tr>
<td>ZNF143|STAF(Zf)/CUTLL-ZNF143-ChIP-Seq(GSE29600)/Homer</td>
<td>ATTTCCCAGVAKSCY</td>
<td>−9.78E+00</td>
<td>0.0011</td>
<td>77</td>
<td>0.75%</td>
<td>151.6</td>
<td>0.46%</td>
</tr>
<tr>
<td>LRF(Zf)/Erythroblasts-ZBTB7A-ChIP-Seq(GSE74977)/Homer</td>
<td>AAGACCCYYN</td>
<td>−8.93E+00</td>
<td>0.0025</td>
<td>317</td>
<td>3.07%</td>
<td>813</td>
<td>2.49%</td>
</tr>
<tr>
<td>FOXA1(Forkhead)/LNCAP-FOXA1-ChIP-Seq(GSE27824)/Homer</td>
<td>WAAGTAAACA</td>
<td>−7.56E+00</td>
<td>0.0095</td>
<td>2,801</td>
<td>27.13%</td>
<td>8,400.2</td>
<td>25.70%</td>
</tr>
<tr>
<td>Pax8(Paired,Homeobox)/Thyroid-Pax8-ChIP-Seq(GSE26938)/Homer</td>
<td>GTCATGCHTGRCTGS</td>
<td>−6.85E+00</td>
<td>0.019</td>
<td>98</td>
<td>0.95%</td>
<td>223.7</td>
<td>0.68%</td>
</tr>
<tr>
<td>THRa(NR)/C17.2-THRa-ChIP-Seq(GSE38347)/Homer</td>
<td>GGTCANYTGAGGWCA</td>
<td>−6.79E+00</td>
<td>0.0199</td>
<td>106</td>
<td>1.03%</td>
<td>246</td>
<td>0.75%</td>
</tr>
<tr>
<td>EBF1(EBF)/Near-E2A-ChIP-Seq(GSE21512)/Homer</td>
<td>GTCCCCWGGGGA</td>
<td>−6.50E+00</td>
<td>0.0261</td>
<td>122</td>
<td>1.18%</td>
<td>291.4</td>
<td>0.89%</td>
</tr>
<tr>
<td>LOB(LOBAS2)/col-LOB-DAP-Seq(GSE60143)/Homer</td>
<td>CGCCGKAWWTTHCGS</td>
<td>−6.18E+00</td>
<td>0.0353</td>
<td>188</td>
<td>1.82%</td>
<td>479.4</td>
<td>1.47%</td>
</tr>
<tr>
<td>At3g60580(C2H2)/col-At3g60580-DAP-Seq(GSE60143)/Homer</td>
<td>WTTYTACT</td>
<td>−6.16E+00</td>
<td>0.0354</td>
<td>5,763</td>
<td>55.81%</td>
<td>17,780</td>
<td>54.40%</td>
</tr>
<tr>
<td>GEI-11(Myb?)/cElegans-L4-GEI11-ChIP-Seq(modEncode)/Homer</td>
<td>CCGACAYYTYACGGG</td>
<td>−6.02E+00</td>
<td>0.0402</td>
<td>47</td>
<td>0.46%</td>
<td>95.1</td>
<td>0.29%</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th>Motif Name</th>
<th>Consensus</th>
<th>Log p value</th>
<th>q value (Benjamini)</th>
<th># of target sequences with motif (of 6,508)</th>
<th>% of target sequences with motif</th>
<th># of background sequences with motif (of 22,227)</th>
<th>% of background sequences with motif</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="8">Exposed to carbaryl</td>
</tr>
<tr>
<td>Foxh1(Forkhead)/hESC-FOXH1-ChIP-Seq(GSE29422)/Homer</td>
<td>NNTGTGGATTSS</td>
<td>−1.58E+03</td>
<td>0</td>
<td>3,274</td>
<td>50.31%</td>
<td>4,276.4</td>
<td>19.24%</td>
</tr>
<tr>
<td>NFIL3(bZIP)/HepG2-NFIL3-ChIP-Seq(Encode)/Homer</td>
<td>VTTACGTAAYNNNNN</td>
<td>−1.16E+03</td>
<td>0</td>
<td>2,668</td>
<td>41.00%</td>
<td>3,562.3</td>
<td>16.03%</td>
</tr>
<tr>
<td>AT2G20400(G2like)/colamp-AT2G20400-DAP-Seq(GSE60143)/Homer</td>
<td>DNVGAATATTCBNHN</td>
<td>−8.36E+02</td>
<td>0</td>
<td>1,138</td>
<td>17.49%</td>
<td>923.1</td>
<td>4.15%</td>
</tr>
<tr>
<td>At5g29000(G2like)/col-At5g29000-DAP-Seq(GSE60143)/Homer</td>
<td>RGAATATTCYHH</td>
<td>−7.31E+02</td>
<td>0</td>
<td>1,316</td>
<td>20.22%</td>
<td>1,361</td>
<td>6.12%</td>
</tr>
<tr>
<td>At2g03500(G2like)/col-At2g03500-DAP-Seq(GSE60143)/Homer</td>
<td>WWAGAATATTCT</td>
<td>−7.29E+02</td>
<td>0</td>
<td>1,264</td>
<td>19.42%</td>
<td>1,267.8</td>
<td>5.71%</td>
</tr>
<tr>
<td>AT1G76880(Trihelix)/col-AT1G76880-DAP-Seq(GSE60143)/Homer</td>
<td>ACGGTAAAAW</td>
<td>−6.95E+02</td>
<td>0</td>
<td>2,061</td>
<td>31.67%</td>
<td>3,039.7</td>
<td>13.68%</td>
</tr>
<tr>
<td>JGL(C2H2)/col-JGL-DAP-Seq(GSE60143)/Homer</td>
<td>ACYTTCAGTT</td>
<td>−6.35E+02</td>
<td>0</td>
<td>3,297</td>
<td>50.66%</td>
<td>6,576.1</td>
<td>29.59%</td>
</tr>
<tr>
<td>At3g04030(G2like)/col-At3g04030-DAP-Seq(GSE60143)/Homer</td>
<td>DRGAATCT</td>
<td>−4.18E+02</td>
<td>0</td>
<td>1,548</td>
<td>23.79%</td>
<td>2,473.8</td>
<td>11.13%</td>
</tr>
<tr>
<td>At1g25550(G2like)/colamp-At1g25550-DAP-Seq(GSE60143)/Homer</td>
<td>NAGATTCY</td>
<td>−4.04E+02</td>
<td>0</td>
<td>1,546</td>
<td>23.76%</td>
<td>2,505</td>
<td>11.27%</td>
</tr>
<tr>
<td>FUS3(ABI3VP1)/col-FUS3-DAP-Seq(GSE60143)/Homer</td>
<td>DNNWTNTGCATGKNN</td>
<td>−3.83E+02</td>
<td>0</td>
<td>1,145</td>
<td>17.59%</td>
<td>1,616.2</td>
<td>7.27%</td>
</tr>
<tr>
<td>Foxa3(Forkhead)/Liver-Foxa3-ChIP-Seq(GSE77670)/Homer</td>
<td>BSNTGTTTACWYWGN</td>
<td>−3.44E+02</td>
<td>0</td>
<td>1,643</td>
<td>25.25%</td>
<td>2,928.1</td>
<td>13.18%</td>
</tr>
<tr>
<td>At5g04390(C2H2)/col200-At5g04390-DAP-Seq(GSE60143)/Homer</td>
<td>AGTGANDN</td>
<td>−3.06E+02</td>
<td>0</td>
<td>4,722</td>
<td>72.56%</td>
<td>12,862.7</td>
<td>57.88%</td>
</tr>
<tr>
<td>At1g68670(G2like)/colamp-At1g68670-DAP-Seq(GSE60143)/Homer</td>
<td>WNWWHNRAAGATTCT</td>
<td>−3.06E+02</td>
<td>0</td>
<td>1,248</td>
<td>19.18%</td>
<td>2,052.4</td>
<td>9.24%</td>
</tr>
<tr>
<td>OCT:OCT-short(POU,Homeobox)/NPC-OCT6-ChIP-Seq(GSE43916)/Homer</td>
<td>ATGCATWATGCATRW</td>
<td>−2.76E+02</td>
<td>0</td>
<td>1,888</td>
<td>29.01%</td>
<td>3,839.3</td>
<td>17.28%</td>
</tr>
<tr>
<td>FOXP1(Forkhead)/H9-FOXP1-ChIP-Seq(GSE31006)/Homer</td>
<td>NYYTGTTTACHN</td>
<td>−2.64E+02</td>
<td>0</td>
<td>2,013</td>
<td>30.93%</td>
<td>4,242.2</td>
<td>19.09%</td>
</tr>
<tr>
<td>FOXK2(Forkhead)/U2OS-FOXK2-ChIP-Seq(E-MTAB-2204)/Homer</td>
<td>SCHTGTTTACAT</td>
<td>−2.47E+02</td>
<td>0</td>
<td>2,081</td>
<td>31.98%</td>
<td>4,514.6</td>
<td>20.32%</td>
</tr>
<tr>
<td>At2g01060(G2like)/colamp-At2g01060-DAP-Seq(GSE60143)/Homer</td>
<td>AGATKCBNWW</td>
<td>−2.25E+02</td>
<td>0</td>
<td>4,320</td>
<td>66.38%</td>
<td>11,893</td>
<td>53.52%</td>
</tr>
<tr>
<td>FoxD3(forkhead)/ZebrafishEmbryo-Foxd3.biotin-ChIP-seq(GSE106676)/Homer</td>
<td>TGTTTAYTTAGC</td>
<td>−2.22E+02</td>
<td>0</td>
<td>1,821</td>
<td>27.98%</td>
<td>3,889.1</td>
<td>17.50%</td>
</tr>
<tr>
<td>GTL1(Trihelix)/colamp-GTL1-DAP-Seq(GSE60143)/Homer</td>
<td>WWTTTACCKY</td>
<td>−2.10E+02</td>
<td>0</td>
<td>2,489</td>
<td>38.25%</td>
<td>5,930.3</td>
<td>26.69%</td>
</tr>
<tr>
<td>Foxo1(Forkhead)/RAW-Foxo1-ChIP-Seq(Fan_et_al.)/Homer</td>
<td>CTGTTTAC</td>
<td>−2.02E+02</td>
<td>0</td>
<td>2,670</td>
<td>41.03%</td>
<td>6,536.3</td>
<td>29.41%</td>
</tr>
<tr>
<td>Foxo3(Forkhead)/U2OS-Foxo3-ChIP-Seq(E-MTAB-2701)/Homer</td>
<td>DGTAAACA</td>
<td>−1.81E+02</td>
<td>0</td>
<td>2,309</td>
<td>35.48%</td>
<td>5,549.6</td>
<td>24.97%</td>
</tr>
<tr>
<td>STZ(C2H2)/colamp-STZ-DAP-Seq(GSE60143)/Homer</td>
<td>HNBTCACT</td>
<td>−1.75E+02</td>
<td>0</td>
<td>4,561</td>
<td>70.08%</td>
<td>13,110.5</td>
<td>59.00%</td>
</tr>
<tr>
<td>At3g24120(G2like)/col-At3g24120-DAP-Seq(GSE60143)/Homer</td>
<td>NWWAGMATMW</td>
<td>−1.70E+02</td>
<td>0</td>
<td>3,705</td>
<td>56.93%</td>
<td>10,141.5</td>
<td>45.64%</td>
</tr>
<tr>
<td>AT5G60130(ABI3VP1)/col-AT5G60130-DAP-Seq(GSE60143)/Homer</td>
<td>WTTYTAAGVAAA</td>
<td>−1.60E+02</td>
<td>0</td>
<td>3,003</td>
<td>46.14%</td>
<td>7,879.9</td>
<td>35.46%</td>
</tr>
<tr>
<td>RBFox2(?)/Heart-RBFox2-CLIP-Seq(GSE57926)/Homer</td>
<td>TGCATGCA</td>
<td>−1.56E+02</td>
<td>0</td>
<td>2,151</td>
<td>33.05%</td>
<td>5,222.6</td>
<td>23.50%</td>
</tr>
<tr>
<td>At3g12730(G2like)/colamp-At3g12730-DAP-Seq(GSE60143)/Homer</td>
<td>AAGATTCT</td>
<td>−1.37E+02</td>
<td>0</td>
<td>1,807</td>
<td>27.77%</td>
<td>4,304.9</td>
<td>19.37%</td>
</tr>
<tr>
<td>Foxa2(Forkhead)/Liver-Foxa2-ChIP-Seq(GSE25694)/Homer</td>
<td>CYTGTTTACWYW</td>
<td>−1.32E+02</td>
<td>0</td>
<td>2,170</td>
<td>33.34%</td>
<td>5,447.6</td>
<td>24.52%</td>
</tr>
<tr>
<td>AT5G47660(Trihelix)/colamp-AT5G47660-DAP-Seq(GSE60143)/Homer</td>
<td>AWTTTTACCG</td>
<td>−1.18E+02</td>
<td>0</td>
<td>2,630</td>
<td>40.41%</td>
<td>7,002</td>
<td>31.51%</td>
</tr>
<tr>
<td>Fox:Ebox(Forkhead,bHLH)/Panc1-Foxa2-ChIP-Seq(GSE47459)/Homer</td>
<td>NNNVCTGWGYAAACASN</td>
<td>−1.08E+02</td>
<td>0</td>
<td>1,541</td>
<td>23.68%</td>
<td>3,701.6</td>
<td>16.66%</td>
</tr>
<tr>
<td>AT5G45580(G2like)/colamp-AT5G45580-DAP-Seq(GSE60143)/Homer</td>
<td>ADRGAATCTH</td>
<td>−9.23E+01</td>
<td>0</td>
<td>2,378</td>
<td>36.54%</td>
<td>6,416</td>
<td>28.87%</td>
</tr>
<tr>
<td>FOXM1(Forkhead)/MCF7-FOXM1-ChIP-Seq(GSE72977)/Homer</td>
<td>TRTTTACTTW</td>
<td>−8.64E+01</td>
<td>0</td>
<td>2,118</td>
<td>32.54%</td>
<td>5,644.6</td>
<td>25.40%</td>
</tr>
<tr>
<td>GT2(Trihelix)/colamp-GT2-DAP-Seq(GSE60143)/Homer</td>
<td>AMGGTAAAWWWN</td>
<td>−8.22E+01</td>
<td>0</td>
<td>2,462</td>
<td>37.83%</td>
<td>6,782.8</td>
<td>30.52%</td>
</tr>
<tr>
<td>KAN2(G2like)/colamp-KAN2-DAP-Seq(GSE60143)/Homer</td>
<td>ATATTCTY</td>
<td>−7.27E+01</td>
<td>0</td>
<td>1,751</td>
<td>26.91%</td>
<td>4,618.3</td>
<td>20.78%</td>
</tr>
<tr>
<td>FOXK1(Forkhead)/HEK293-FOXK1-ChIP-Seq(GSE51673)/Homer</td>
<td>NVWTGTTTAC</td>
<td>−5.79E+01</td>
<td>0</td>
<td>2,358</td>
<td>36.23%</td>
<td>6,705.1</td>
<td>30.17%</td>
</tr>
<tr>
<td>AT2G38300(G2like)/col-AT2G38300-DAP-Seq(GSE60143)/Homer</td>
<td>ADRGAATGTT</td>
<td>−5.36E+01</td>
<td>0</td>
<td>1,736</td>
<td>26.67%</td>
<td>4,761.6</td>
<td>21.43%</td>
</tr>
<tr>
<td>FoxL2(Forkhead)/Ovary-FoxL2-ChIP-Seq(GSE60858)/Homer</td>
<td>WWTRTAAACAVG</td>
<td>−3.55E+01</td>
<td>0</td>
<td>2,059</td>
<td>31.64%</td>
<td>6,026.5</td>
<td>27.12%</td>
</tr>
<tr>
<td>FOXA1(Forkhead)/MCF7-FOXA1-ChIP-Seq(GSE26831)/Homer</td>
<td>WAAGTAAACA</td>
<td>−3.52E+01</td>
<td>0</td>
<td>2,147</td>
<td>32.99%</td>
<td>6,318.9</td>
<td>28.44%</td>
</tr>
<tr>
<td>KANADI1(Myb)/Seedling-KAN1-ChIP-Seq(GSE48081)/Homer</td>
<td>ARGAATAWWN</td>
<td>−3.16E+01</td>
<td>0</td>
<td>1,833</td>
<td>28.17%</td>
<td>5,351</td>
<td>24.08%</td>
</tr>
<tr>
<td>MYB73(MYB)/col-MYB73-DAP-Seq(GSE60143)/Homer</td>
<td>NNNNHAACNGHHDHN</td>
<td>−1.30E+01</td>
<td>0.0001</td>
<td>2,749</td>
<td>42.24%</td>
<td>8,766.4</td>
<td>39.45%</td>
</tr>
<tr>
<td>AP-2gamma(AP2)/MCF7-TFAP2C-ChIP-Seq(GSE21234)/Homer</td>
<td>SCCTSAGGSCAW</td>
<td>−1.29E+01</td>
<td>0.0001</td>
<td>64</td>
<td>0.98%</td>
<td>116.8</td>
<td>0.53%</td>
</tr>
<tr>
<td>FOXA1(Forkhead)/LNCAP-FOXA1-ChIP-Seq(GSE27824)/Homer</td>
<td>WAAGTAAACA</td>
<td>−1.13E+01</td>
<td>0.0003</td>
<td>2,454</td>
<td>37.71%</td>
<td>7,822</td>
<td>35.20%</td>
</tr>
<tr>
<td>AP-2alpha(AP2)/Hela-AP2alpha-ChIP-Seq(GSE31477)/Homer</td>
<td>ATGCCCTGAGGC</td>
<td>−1.01E+01</td>
<td>0.001</td>
<td>46</td>
<td>0.71%</td>
<td>83</td>
<td>0.37%</td>
</tr>
<tr>
<td>EBF1(EBF)/Near-E2A-ChIP-Seq(GSE21512)/Homer</td>
<td>GTCCCCWGGGGA</td>
<td>−8.99E+00</td>
<td>0.0029</td>
<td>55</td>
<td>0.85%</td>
<td>109.4</td>
<td>0.49%</td>
</tr>
<tr>
<td>Foxf1(Forkhead)/Lung-Foxf1-ChIP-Seq(GSE77951)/Homer</td>
<td>WWATRTAAACAN</td>
<td>−8.66E+00</td>
<td>0.004</td>
<td>2,300</td>
<td>35.34%</td>
<td>7,385.2</td>
<td>33.23%</td>
</tr>
<tr>
<td>At3g60580(C2H2)/col-At3g60580-DAP-Seq(GSE60143)/Homer</td>
<td>WTTYTACT</td>
<td>−8.25E+00</td>
<td>0.0058</td>
<td>3,791</td>
<td>58.25%</td>
<td>12,469.4</td>
<td>56.11%</td>
</tr>
<tr>
<td>SeqBias: GCW-triplet</td>
<td>GCWGCWGCWGCW</td>
<td>−8.06E+00</td>
<td>0.0069</td>
<td>6,440</td>
<td>98.96%</td>
<td>21,878</td>
<td>98.46%</td>
</tr>
<tr>
<td>STOP1(C2H2)/colamp-STOP1-DAP-Seq(GSE60143)/Homer</td>
<td>DTATCTGGKGRAGGT</td>
<td>−7.26E+00</td>
<td>0.015</td>
<td>152</td>
<td>2.34%</td>
<td>396.6</td>
<td>1.78%</td>
</tr>
<tr>
<td>At2g33710(AP2EREBP)/colamp-At2g33710-DAP-Seq(GSE60143)/Homer</td>
<td>WTKGCGGCKR</td>
<td>−6.01E+00</td>
<td>0.0516</td>
<td>597</td>
<td>9.17%</td>
<td>1,821.2</td>
<td>8.20%</td>
</tr>
<tr>
<td>LOB(LOBAS2)/col-LOB-DAP-Seq(GSE60143)/Homer</td>
<td>CGCCGKAWWTTHCGS</td>
<td>−5.92E+00</td>
<td>0.0551</td>
<td>81</td>
<td>1.24%</td>
<td>199.5</td>
<td>0.90%</td>
</tr>
</tbody>
</table>
<p class="table-legend">ATAC motifs found in the control (non-exposed) and in carabaryl exposed samples. The motif name; the consensus motif sequence; the log p value; the corrected Benjamini q value; the Number of Target Sequences (including their percentage); and the number of Background Sequences with Motifs (including their percentage) are shown.</p>
<p>Overall, our results show that the modified ATAC protocol presented here is compatible with applications in population epigenomics of <i>D. magna</i>, with possible application in other non-model invertebrates.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Many steps in the optimized protocol are temperature sensitive (e.g., enzymatic reactions). If incubation temperatures are not respected and if room temperature exceeds 30°C, the protocol may be unreliable. Furthermore, tissue storage conditions may be a cause of failure of the protocol due to degradation of the input material. Animal tissue should be used fresh or flash frozen immediately after collection for later use. Improperly stored tissue may lead to unreliable outputs. The use of DNA exacted from human cell lines is advised as positive control to assess reliability of critical steps in the protocol.</p>
<p>The protocol has been optimized for the crustacean <i>Daphnia magna</i>. Although transferability to other crustaceans is largely expected, it is advised that transposase enzyme, staring material and quality of ATAC libraries is assessed for other organisms before the essay is applied to large sample numbers.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1: inefficient DNA fragmentation (related to step 2)</h3>
<p>The transposition reaction efficiency may be diminished or have reduced replicability because of inefficient DNA fragmentation. This may occur in species with very compact genomes.</p>
<h3 id="sec6.2">Potential solution: mechanic fragmentation</h3>
<p>A mechanic DNA fragmentation step (e.g., pico-sonication) may be introduced prior to the transposition reaction to improve evenness of fragmentation and accessibility of the tagmentase, enhancing replicability and efficiency of the transposition reaction. An example of how mechanic fragmentation may improve DNA shearing is in <a href="#fig2">Figure 2</a>. Using a tissue homogenizer improves the quality of the ATAC-seq libraries; the combination of a tissue homogenizer and DNA shearing using a picosonicator improves the quality of the libraries when excess tissue may limit accessibility to the nuclei.</p>
<h3 id="sec6.3">Problem 2: low transposition efficiency (related to step 3)</h3>
<p>Excess transposition or low efficiency of the transposition reaction may affect the identification of chromatin motifs.</p>
<h3 id="sec6.4">Potential solutions</h3>
<p>Each suggested solution below or a combination thereof can be used to improve the transposition efficiency.</p>
<p>Different amounts of transpositions enzyme may be used. In the optimization of this protocol, different dilutions of the transposase suggested concentration were used: 100%, 50%, 25%, and 12% (<a href="#fig3">Figure 3</a>). A concentration corresponding to 25% of the manufacturer suggested concentration resulted in good quality ATAC-seq Daphnia libraries. Different concentrations may be used for other organisms.</p>
<p>Different incubation times may be tested for the transposition reaction. In the optimization of this protocol we used 16–20 h long incubation as well as 4h incubation times for the transposase reaction. Whereas no appreciable differences were observed in the Daphnia libraries, this may differ in other species.</p>
<p>Access to chromatin may be improved by mechanic fragmentation. Access to chromatin within nuclei can be enhanced by a mechanic fragmentation of the animal tissue. Including a mechanic fragmentation improves chromatic accessibility (<a href="#fig2">Figure 2</a>; tissue homogenizer) resulting in better quality libraries as compared to libraries obtained from manual fragmentation (<a href="#fig2">Figure 2</a>; pestle).</p>
<p>It is advisable to perform a protocol optimization on a reduced number of samples prior to processing large sample numbers.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Andrews, S. (2010). FastQC: a quality control tool for high throughput sequence data. <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>.</p>
<p id="bib2">Batut, B., Hiltemann, S., Bagnacani, A., Baker, D., Bhardwaj, V., Blank, C., Bretaudeau, A., Brillet-Gueguen, L., Cech, M., Chilton, J., et al. (2018). Community-driven data analysis training for biology. Cell Syst. <i>6</i>, 752-758.e1. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref2">View at publisher</a></p>
<p id="bib3">Bolger, A.M., Lohse, M., and Usadel, B. (2014). Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics <i>30</i>, 2114-2120. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref3">View at publisher</a></p>
<p id="bib4">Buenrostro, J.D., Giresi, P.G., Zaba, L.C., Chang, H.Y., and Greenleaf, W.J. (2013). Transposition of native chromatin for multimodal regulatory analysis and personal epigenomics. Nat. Methods <i>10</i>, 1213-1218. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref4">View at publisher</a></p>
<p id="bib5">Buenrostro, J.D., Wu, B., Chang, H.Y., and Greenleaf, W.J. (2015). ATAC-seq: a method for assaying chromatin accessibility genome-wide. Curr. Protoc. Mol. Biol. <i>109</i>, 21 29 21-21 29 29. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref5">View at publisher</a></p>
<p id="bib6">Cambronero, C.M., Marshall, H., De Meester, L., Davidson, A., Beckerman, A.P., and Orsini, L. (2018). Predictability of the impact of multiple stressors on the keystone species <i>Daphnia</i>. Sci. Rep. <i>8</i>, 17572. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref6">View at publisher</a></p>
<p id="bib7">Cambronero Cuenca, M. and Orsini, L. (2018). Resurrection of dormant <i>Daphnia magna</i>: protocol and applications. J. Vis. Exp. <i>131</i>, e56637. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref7">View at publisher</a></p>
<p id="bib8">Davie, K., Janssens, J., Koldere, D., De Waegeneer, M., Pech, U., Kreft, L., Aibar, S., Makhzami, S., Christiaens, V., Bravo Gonzalez-Blas, C., et al. (2018). A single-cell transcriptome atlas of the aging <i>Drosophila</i> brain. Cell <i>174</i>, 982-998.e20. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref8">View at publisher</a></p>
<p id="bib9">Duttke, S.H., Chang, M.W., Heinz, S., and Benner, C. (2019). Identification and dynamic quantification of regulatory elements using total RNA. Genome Res. <i>29</i>, 1836-1846. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref9">View at publisher</a></p>
<p id="bib10">Ebert, D. (2005). Ecology, Epidemiology, and Evolution of Parasitism in Daphnia (National Library of Medicine (US), National Center for Biotechnology). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref10">View at publisher</a></p>
<p id="bib11">Ewels, P., Magnusson, M., Lundin, S., and Kaller, M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics <i>32</i>, 3047-3048. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref11">View at publisher</a></p>
<p id="bib12">Gaspar, J.M. (2018). Improved peak-calling with MACS2. bioRxiv <a class="external-link" href="https://doi.org/10.1101/496521">https://doi.org/10.1101/496521</a></p>
<p id="bib13">Heinz, S., Benner, C., Spann, N., Bertolino, E., Lin, Y.C., Laslo, P., Cheng, J.X., Murre, C., Singh, H., and Glass, C.K. (2010). Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Mol. Cell <i>38</i>, 576-589. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref13">View at publisher</a></p>
<p id="bib14">Jansen, M., De Meester, L., Cielen, A., Buser, C.C., and Stoks, R. (2011). The interplay of past and current stress exposure on the water flea <i>Daphnia</i>. Funct. Ecol. <i>25</i>, 974-982. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref14">View at publisher</a></p>
<p id="bib15">Kerfoot, W.C. and Weider, L.J. (2004). Experimental paleoecology (resurrection ecology): Chasing Van Valen’s Red Queen hypothesis. Limnol. Oceanogr. <i>49</i>, 1300-1316. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref15">View at publisher</a></p>
<p id="bib16">Kircher, M., Sawyer, S., and Meyer, M. (2012). Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform. Nucleic Acids Res. <i>40</i>, 2513-2524. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref16">View at publisher</a></p>
<p id="bib17">Langmead, B., Wilks, C., Antonescu, V., and Charles, R. (2019). Scaling read aligners to hundreds of threads on general-purpose processors. Bioinformatics <i>35</i>, 421-432. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref17">View at publisher</a></p>
<p id="bib18">Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R., and Genome Project Data Processing, S. (2009). The sequence alignment/map format and SAMtools. Bioinformatics <i>25</i>, 2078-2079. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref18">View at publisher</a></p>
<p id="bib19">Liu, C., Wang, M., Wei, X., Wu, L., Xu, J., Dai, X., Xia, J., Cheng, M., Yuan, Y., Zhang, P., et al. (2019). An ATAC-seq atlas of chromatin accessibility in mouse tissues. Sci. Data <i>6</i>, 65. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref19">View at publisher</a></p>
<p id="bib20">Okonechnikov, K., Conesa, A., and Garcia-Alcalde, F. (2016). Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data. Bioinformatics <i>32</i>, 292-294. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref20">View at publisher</a></p>
<p id="bib21">Orchard, P., Kyono, Y., Hensley, J., Kitzman, J.O., and Parker, S.C.J. (2020). Quantification, dynamic visualization, and validation of bias in ATAC-seq data with ataqv. Cell Syst <i>10</i>, 298-306 e294. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref21">View at publisher</a></p>
<p id="bib22">Quinlan, A.R. (2014). BEDTools: the Swiss-army tool for genome feature analysis. Curr Protoc Bioinformatics <i>47</i>, 11.12.1-11.12.34. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref22">View at publisher</a></p>
<p id="bib23">Ramirez, F., Bhardwaj, V., Arrigoni, L., Lam, K.C., Gruning, B.A., Villaveces, J., Habermann, B., Akhtar, A., and Manke, T. (2018). High-resolution TADs reveal DNA sequences underlying genome organization in flies. Nat. Commun. <i>9</i>, 189. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00048-4/sref23">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Luisa Orsini (<a href="mailto:l.orsini@bham.ac.uk">l.orsini@bham.ac.uk</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and Code Availability</h4>
<p>The ATAC-seq data generated during the optimization of this protocol can be found at PRJNA669393 in the NCBI repository.</p>
<h3>Acknowledgments</h3>
<p>The sequencing data were generated by EnviSion, Environmental BioSequencing and BioComputing. This work was supported by a <a href="https://doi.org/10.13039/501100000270">NERC</a> highlights grant (NE/N016777/1). We thank Caroline Sewell for technical support in the <i>Daphnia</i> facility at the University of Birmingham.</p>
<h3>Author Contributions</h3>
<p>S.K. performed the protocol optimization with input from L.O. V.D. performed data analysis. L.O. conceived and coordinated the experiments. All authors contributed to the manuscript writing.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests</p>
</section>
</article>