<article>
<section>
<h1 id="header">Analysis of epicardial genes in embryonic mouse hearts with flow cytometry</h1>
<p><time datetime="2021-02-26">Published: February 26, 2021</time></p>
<p>Andia Nicole Redpath,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Irina-Elena Lupu,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Nicola Smart<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>British Heart Foundation Centre of Regenerative Medicine, Burdon Sanderson Cardiac Science Centre, Department of Physiology, Anatomy &amp; Genetics, University of Oxford, South Parks Road, Oxford OX1 3PT, UK</p>
<p id="fn1"><sup>2</sup>These authors contributed equally</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:andia.redpath@dpag.ox.ac.uk">andia.redpath@dpag.ox.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:nicola.smart@dpag.ox.ac.uk">nicola.smart@dpag.ox.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100359">10.1016/j.xpro.2021.100359</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Genetic markers used to define discrete cell populations are seldom expressed exclusively in the population of interest and are, thus, unsuitable when evaluated individually, especially in the absence of spatial and morphological information. Here, we present fluorescence <i>in situ</i> hybridization for flow cytometry to allow simultaneous analysis of multiple marker genes at the single whole-cell level, exemplified by application to the embryonic epicardium. The protocol facilitates multiplexed quantification of gene and protein expression and temporal changes across specific cell populations.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Lupu et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol to characterize co-expression of genes in the epicardium and its derivatives</li>
<li>Dissociation of embryonic mouse hearts into a reliable single-cell suspension</li>
<li>Multiplexed RNA-ISH staining paired with cell lineage- and immuno-labeling</li>
<li>Single-cell resolution analysis of epicardial marker expression by flow cytometry</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/492-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Set up timed mating pairs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 weeks</div>
<p>This step details an example mating combination used to attain epicardial lineage tracing by employing the tamoxifen-inducible Cre-<i>loxP</i> system (CreERT2). In brief, tamoxifen treatment results in nuclear translocation of the tissue-specific driven CreERT2, where it will interact with <i>loxP</i> sites and excise a <i>STOP</i> cassette upstream of a reporter protein gene. After excision, the reporter e.g., a fluorophore, is permanently and constitutively expressed in the parent cell and its derivatives (<a href="#bib4">Meilhac and Buckingham, 2018</a>).</p>
<ol>
<li>Select mice with relevant genotype (at age 10–30 weeks) to be paired for timed breeding.
<ol type="a">
<li>For epicardial lineage tracing, males homozygous for <i>Rosa26tdTom</i> (<a href="#bib3">Madisen et al., 2010</a>) and heterozygous for <i>Wt1CreERT2</i> (<a href="#bib6">Zhou et al., 2008</a>) should be crossed with C57BL/6 females to achieve embryos heterozygous for <i>Rosa26tdTom</i> and heterozygous for <i>Wt1CreERT2.</i> Pairing is optimally performed in the late afternoon to maximize mating during the night, and confirmation of vaginal plugs early the next morning.</li>
<li>Note the morning that a vaginal plug is observed, separate females from mating, and refer to this date as embryonic day 0.5 (E0.5).
<ol type="i">
<li>Calculate subsequent days to schedule tamoxifen induction and harvest time for specific embryonic stages.</li>
<li>Harvest may be scheduled for E11.5, when epicardium formation is complete; E13.5, when most epicardium-derived cells (EPDCs) emerge; E15.5, when epicardial epithelial-mesenchymal transition (epiEMT) is complete; and E17.5, when EPDCs have differentiated to fibroblasts and mural cells.</li>
</ol>
</li>
</ol>
</li>
<li>Tracing of the epicardial lineage in <i>R26tdTom;Wt1CreERT2</i> embryos is achieved when pregnant females are treated with 80 mg/kg (mouse body weight) tamoxifen by oral gavage at E9.5. This protocol was shown to tdTomato-label 89.1% of WT1+ cells (<a href="#bib2">Lupu et al., 2020</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Users may refer to The Jackson Laboratory website (<a href="https://www.jax.org">https://www.jax.org</a>) for document advising steps for successful timed mating.</div>
<div class="note">
<span class="note-title">Note:</span> Users may refer to The Procedures with Care website (<a href="http://www.procedureswithcare.org.uk/oral-gavage-in-the-mouse/">http://www.procedureswithcare.org.uk/oral-gavage-in-the-mouse/</a>) for a video demonstrating the oral gavage procedure in mice. In this study, flexible disposable animal feeding tubes (Instech Laboratories, Cat# FTP-20-30) were similarly used to administer tamoxifen orally.</div>
<h3 id="sec1.2">Prepare dissociation reagents and buffers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="3">
<li>Reconstitute components of the Neonatal Heart Dissociation Kit (Miltenyi Biotec) before start of experiment.</li>
<li>Prepare buffers required for enzymatic dissociation of embryonic hearts.
<ol type="a">
<li>DMEM supplemented with 10% FBS, cell culture medium required for termination of the enzymatic reaction.</li>
<li>PEB buffer (PBS, pH 7.4, 0.5% bovine serum albumin (BSA), and 2 mM EDTA), buffer required for the red blood cell lysis step.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Users should prepare buffers (step 4) freshly on the day. If prepared the day before, ensure preparation is carried out under aseptic conditions and buffers stored at 2°C–8°C.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-mouse CD31 BV421 (390)</td>
<td>BioLegend</td>
<td>Cat# 102423RRID: AB_2562186</td>
</tr>
<tr>
<td>Anti-cardiac troponin T BV421 (13-11)</td>
<td>BD Biosciences</td>
<td>Cat# 565618 RRID: AB_2739306</td>
</tr>
<tr>
<td>Anti-Wilms tumor protein Alexa Fluor 488 (CAN-R9(IHC)-56-2)</td>
<td>Abcam</td>
<td>Cat# ab202635</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Zombie Aqua Fixable Viability Kit</td>
<td>BioLegend</td>
<td>Cat# 423101</td>
</tr>
<tr>
<td>Cell staining buffer</td>
<td>BioLegend</td>
<td>Cat# 420201</td>
</tr>
<tr>
<td>Red blood cell lysis solution (10×)</td>
<td>Miltenyi Biotec</td>
<td>Cat# 130-094-183</td>
</tr>
<tr>
<td>AbC Total Antibody Compensation Bead Kit</td>
<td>Thermo Fisher Scientific/Invitrogen</td>
<td>Cat# A10497</td>
</tr>
<tr>
<td>Wt1 - Alexa Fluor 488</td>
<td>Thermo Fisher Scientific</td>
<td>Assay ID: VB4-13886-PF</td>
</tr>
<tr>
<td>Tbx18 - Alexa Fluor 488</td>
<td>Thermo Fisher Scientific</td>
<td>Assay ID: VB4-3113124-PF</td>
</tr>
<tr>
<td>Tbx18 - Alexa Fluor 568</td>
<td>Thermo Fisher Scientific</td>
<td>Assay ID: VB10-3283954-PF</td>
</tr>
<tr>
<td>Sema3d - Alexa Fluor 647</td>
<td>Thermo Fisher Scientific</td>
<td>Assay ID: VB1-3044712-PF</td>
</tr>
<tr>
<td>Sema3d - Alexa Fluor 750</td>
<td>Thermo Fisher Scientific</td>
<td>Assay ID: VB6-3214456-PF</td>
</tr>
<tr>
<td>Tcf21 - Alexa Fluor 750</td>
<td>Thermo Fisher Scientific</td>
<td>Assay ID: VB6-3197774-PF</td>
</tr>
<tr>
<td>Scx - Alexa Fluor 647</td>
<td>Thermo Fisher Scientific</td>
<td>Assay ID: VB1-3028071-PF</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Neonatal Heart Dissociation Kit, mouse and rat</td>
<td>Miltenyi Biotec</td>
<td>Cat# 130-098-373</td>
</tr>
<tr>
<td>PrimeFlow RNA Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 88-18005-210</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: <i>Rosa26tdTom</i>: B6.Cg Gt(ROSA)26Sor<sup>tm14(CAG−tdTomato)Hze</sup>/J</td>
<td>The Jackson Laboratory</td>
<td>MGI ID:3809524; JAX Stock No: 007914; RRID: IMSR_JAX:007914</td>
</tr>
<tr>
<td>Mouse: <i>Wt1CreERT2</i>: Wt1<sup>tm2(cre/ERT2)Wtp</sup>/J</td>
<td>The Jackson Laboratory</td>
<td>MGI ID:3801682; JAX Stock No: 010912; RRID: IMSR_JAX:010912</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>BD FACSDiva</td>
<td>BD Biosciences</td>
<td><a href="https://www.bdbiosciences.com/">https://www.bdbiosciences.com/</a></td>
</tr>
<tr>
<td>FlowJo V10</td>
<td>FlowJo LLC</td>
<td>
<a href="https://www.flowjo.com">https://www.flowjo.com</a>; RRID: SCR_008520</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Dissecting microscope with fluorescence filters (RFP)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Light microscope</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Dissection scissors, forceps no 5 and no 55</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>gentleMACS Octo dissociator</td>
<td>Miltenyi Biotec</td>
<td>Cat# 130-095-937</td>
</tr>
<tr>
<td>gentleMACS C-tubes</td>
<td>Miltenyi Biotec</td>
<td>Cat# 130-096-334</td>
</tr>
<tr>
<td>Cell strainer 70 μm</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Eppendorf 5804R series centrifuge (with cooling)</td>
<td>Eppendorf</td>
<td>N/A</td>
</tr>
<tr>
<td>Deepwell plate rotor A-2-DWP</td>
<td>Eppendorf</td>
<td>Cat# 5804740009</td>
</tr>
<tr>
<td>HybEZ oven</td>
<td>Advanced Cell Diagnostics</td>
<td>Cat# PN 321710/321720</td>
</tr>
<tr>
<td>HybEZ oven humidity control tray</td>
<td>Advanced Cell Diagnostics</td>
<td>Cat# PN 310012</td>
</tr>
<tr>
<td>PrimeFlow 96-well plate</td>
<td>Thermo Fisher Scientific/Invitrogen</td>
<td>Cat# 44-17005-46</td>
</tr>
<tr>
<td>BD LSRFortessa X-20 flow cytometer: fitted with UV 355 nm, violet 405 nm, blue 488 nm, yellow-green 561 nm, red 640 nm lasers</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>Vortex</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Cooled microcentrifuge for 1.5–2.0 mL tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Multichannel manual pipettor</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Hemocytometer</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>V-bottom 96-well polypropylene plate</td>
<td>N/A</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Tamoxifen stock</h3>
<p>For 10 mL of 10 mg/mL tamoxifen stock, dissolve 100 mg tamoxifen in 1 mL of ethanol on a heated shaker at ≥37°C. Preheat 9 mL of Peanut Oil to ≥37°C, before adding to the dissolved tamoxifen-ethanol solution. Tamoxifen will sometimes come out of solution with addition of peanut oil, vortexing briefly may be necessary before returning to the heated shaker until fully dissolved. Store at −20°C in 1 mL aliquots for up to 6 months.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure tamoxifen stocks are prepared under aseptic conditions since the solution will be administered to animals.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Harvest of embryonic hearts: day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15–20 min per litter</div>
<p>This step details how to take embryonic heart samples for enzymatic dissociation.</p>
<ol>
<li>Following culling of the pregnant female mouse, using approved methods such as cervical dislocation, remove the uterine horn containing mouse embryos.
<ol type="a">
<li>Place dissected uterus in a Petri dish containing ice cold PBS. Separate each embryo by pinching between implantation sites/embryos with a pair of forceps.</li>
<li>Separate the muscular uterine wall, membranes, and visceral yolk sac to expose the embryo (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/492-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Processing of embryonic hearts</div>
<p>(A) Inspect embryonic anatomy to assess and confirm the developmental stage.</p>
<p>(B) Excise heart from embryo, remove the outflow tract.</p>
<p>(C) Remove all traces of the pericardium; especially present on hearts of later developmental stages, ≥E15.5.</p>
<p>(D) Submerge hearts in enzyme mix and mount C-tube to automated dissociator.</p>
<p>(E) Neutralize enzymatic reaction and filter cell suspension.</p>
<p>(F) Small faint red cell pellet is visible before RBC lysis.</p>
<p>(G) Small white cell pellet is visible after RBC lysis.</p>
<p>(H) Count viable cells (bright) with 0.4% trypan blue solution; cell shape and size may vary between cells.</p>
<p>RA, right atrium; LA, left atrium; RV, right ventricle; LV, left ventricle; OFT, outflow tract; peri, pericardium.</p>
</figcaption>
</figure>
<ol type="i">
<li>Inspect embryonic anatomy to assess and confirm the developmental stage. Users may refer to the eMouseAtlas (<a href="https://www.emouseatlas.org">https://www.emouseatlas.org</a>) for assistance.</li>
</ol>
</li>
</ol>
</li>
<li>Excise hearts from embryos, remove the outflow tract (<a href="#fig1">Figure 1</a>B) and pericardium (<a href="#fig1">Figure 1</a>C), and transfer directly into pre-cooled C-tubes containing cold PBS. Hearts from embryos with <i>R26tdTom;Wt1CreERT2</i>/+ genotype may be separated from Cre- littermates by inspecting hearts under the red fluorescence channel.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If lineage tracing is desired, ensure reporter-labeled hearts are separated from unlabeled hearts. Regardless of whether unlabeled hearts are used in downstream staining, ensure some are taken and dissociated for single-color compensation (where cells are stained for just one of the markers, repeat for each color) and fluorescence minus one (FMO) (where one of the fluorochrome-conjugated antibodies/RNA probes used in the staining combination is left out, repeat for each color). This study describes a multiparameter staining protocol, and it is standard practice to prepare FMO controls to accurately determine gating of positively stained populations. Users may refer to <a href="#bib1">Cossarizza et al., 2019</a> for further details on the importance and preparation of FMO controls.</div>
<div class="note">
<span class="note-title">Note:</span> C-tubes containing hearts are kept on ice (+4°C) throughout harvest. If multiple litters are to be harvested and users anticipate hearts needing to be on ice for longer than 20 min, we recommend C-tubes contain PBS supplemented with 2% FBS. Time on ice should not exceed 1 h but instead stagger litter harvests to ensure hearts are processed as quickly as possible.</div>
<div class="note">
<span class="note-title">Note:</span> If mice carry other mutant alleles relevant to the experiment, take embryonic tail samples for genotyping.</div>
<div class="note">
<span class="note-title">Note:</span> During harvest remove Neonatal Heart Dissociation Kit enzymes to thaw on ice.</div>
<h3 id="sec3.2">Processing embryonic hearts: day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5–2 h</div>
<p>Enzymatic dissociation of embryonic hearts and preparation of single-cell suspensions.</p>
<ol start="3">
<li>Embryonic heart digestion
<ol type="a">
<li>Prepare enzyme mix as per manufacturer’s instructions (Neonatal Heart Dissociation Kit protocol). 2.5 mL enzyme mix is required per digestion.
<div class="note">
<span class="note-title">Note:</span> Keep embryonic hearts intact (atria + ventricles) and refer to <a href="#tbl1">Table 1</a> for estimated number of cells per heart at each embryonic stage. Pooling of tissue is required for E11.5 and E13.5, and should be carried out before digestion, assuming identical embryonic genotypes can be ensured.</div>
<table id="tbl1">
<caption>Table 1. Estimated number of cells per heart at each embryonic stage</caption>
<thead>
<tr>
<th>Embryonic stage</th>
<th>Estimated number of cells per heart</th>
<th>Tissue pooling before dissociation</th>
</tr>
</thead>
<tbody>
<tr>
<td>E11.5</td>
<td>80,000–100,000</td>
<td>advisable, pool at least 5 hearts</td>
</tr>
<tr>
<td>E13.5</td>
<td>250,000–300,000</td>
<td>advisable, pool at least 3 hearts</td>
</tr>
<tr>
<td>E14.5</td>
<td>350,000–400,000</td>
<td>advisable, pool at least 2 hearts</td>
</tr>
<tr>
<td>E15.5</td>
<td>500,000–600,000</td>
<td>not required unless staining more than 2 target probe panels</td>
</tr>
<tr>
<td>E17.5</td>
<td>1,200,000–1,700,000</td>
<td>not required unless staining more than 4 target probe panels</td>
</tr>
</tbody>
</table>
</li>
<li>As per manufacturer’s instructions, remove supernatant (PBS) carefully from C-tubes containing embryonic hearts using a fine tip plastic Pasteur pipette.</li>
<li>Add 2.5 mL of enzyme mix to each C-tube, invert tubes ensuring all hearts are submerged in solution and attach onto the sleeve of the gentleMACS Octo Dissociator with Heaters (<a href="#fig1">Figure 1</a>D).</li>
<li>Run program 37C_mr_NHDK_1. The duration of this program is 60 min.
<div class="note">
<span class="note-title">Note:</span> If in possession of the gentleMACS dissociator (without heaters), follow the manufacturer’s alternative instructions. Enzymatic reactions may also take place on heated racks/incubator at 37°C, with gentle mechanical agitation using a (regular tip) plastic Pasteur pipette for 30 s; agitate every 15 min, for a total of 2 agitations during the 45 min digestion.</div>
</li>
<li>After the dissociator program is complete, detach C-tubes and add 7.5 mL of cold culture medium (DMEM supplemented with 10% FBS) to terminate enzymatic reaction.</li>
<li>Centrifuge C-tubes briefly at 100 × <i>g</i> for 1 min to collect solution from C-tube caps, and maximize sample transfer.</li>
<li>Resuspend sample with a plastic Pasteur pipette and immediately transfer to a 50 mL tube fitted with a 70 μm strainer (<a href="#fig1">Figure 1</a>E).</li>
<li>Wash C-tubes and strainers with 3 mL of cold cell culture medium to maximize sample transfer. Use the thumb press end of a 2 mL syringe plunger rod as a pestle to push any remaining tissue clumps through the strainer. Use a pipette to collect everything from the underside of the strainer before discarding.</li>
<li>Discard strainers and centrifuge cell suspension at 600 × <i>g</i> for 5 min at +10°C.</li>
<li>Remove supernatant completely and resuspend cell pellet in 500 μL of cold PEB buffer.</li>
<li>Transfer the 500 μL cell suspension to 15 mL tubes. Wash 50 mL tubes with another 500 μL of cold PEB and transfer to corresponding tube to maximize sample transfer. Each cell suspension will make up a total volume of 1 mL. Immediately proceed to the next step.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Red blood cell lysis (<a href="#fig1">Figures 1</a>F and 1G)
<ol type="a">
<li>Prepare 1× red blood cell lysis solution (Miltenyi Biotec) with ultrapure DNase/RNase-free distilled water.</li>
<li>Add 10 mL of 1× red blood cell lysis solution to each sample.</li>
<li>Incubate for 2 min at 20°C–24°C.</li>
<li>Centrifuge at 600 × <i>g</i> for 5 min at +10°C. In the meantime, prepare Enzyme A solution in PBS.</li>
<li>Remove supernatant and gently resuspend cell pellet in 1 mL Enzyme A solution, by resuspending twice with a pipette. After all samples are resuspended, make up to 10 mL volume by adding 9 mL Enzyme A solution to each sample.</li>
<li>Centrifuge at 600 × <i>g</i> for 5 min at +10°C.</li>
<li>Remove supernatant and resuspend cell pellet in 500 μL cold cell staining buffer (BioLegend). Keep samples on ice (+2°C to +4°C).</li>
<li>Take a sample of cells to perform manual cell counts using a hemocytometer (<a href="#fig1">Figure 1</a>H; advised ratio of cell suspension to 0.4% trypan blue solution, 1:1). <a href="#sec7.1">Troubleshooting 1</a> and <a href="#sec7.3">2</a>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Automated cell counters may be used if users are confident that dead cells, debris, and any remaining erythrocytes are excluded from the final count. Users must ensure automated counters count all viable cells that vary in size and shape, for example more mature cardiomyocytes will display reduced circularity and may be falsely identified as debris. Optimize counter settings to match manual counts before adopting for regular use. Normally, this protocol results in &gt;94% viable cells.</div>
<div class="note">
<span class="note-title">Note:</span> Users will need to set a maximum number of cells to stain to match cell numbers between samples. We advise setting a cell number maximum, to allow for control during staining and data acquisition. For example, if the total cell number in sample A = 260,000, sample B = 400,000 and sample C = 300,000, one can set a maximum of 100,000 cells per target probe panel. Thus, the sum of excess cells from sample A, B, and C (=360,000) can be removed and allocated to controls, i.e., FMOs and compensation. This example is suitable for 2 staining panels. Late embryonic stages will yield more cells, thus the user may raise the maximum number of cells per target probe panel.</div>
<div class="critical">
<span class="critical-title">Critical:</span> No fewer than 100,000 cells should be allocated per target probe panel for each experimental/biological sample.</div>
<h3 id="sec3.3">Staining cells: day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5 h</div>
<p>Staining cells for viability, and antibody staining of cell surface and intracellular antigens.</p>
<ol start="5">
<li>Fixable viability dye staining
<ol type="a">
<li>Add 10 mL PBS to each sample, invert to mix, and centrifuge at 600 × <i>g</i> for 5 min at +4°C.</li>
<li>Remove supernatant and resuspend cell pellet in 300 μL diluted Zombie Aqua Fixable Viability Dye solution (1:1,000 in PBS).</li>
<li>Transfer samples to 1.5 mL tubes and incubate for 30 min at 20°C–24°C, protected from light.</li>
<li>Add 1 mL of cold cell staining buffer, centrifuge at 500 × <i>g</i> for 5 min at +4°C, and discard supernatant. Proceed to cell surface antibody staining procedure.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Before staining for viability take some cells from your allocated control samples to generate compensation controls for the live/dead dye and fluorescent reporter protein (e.g., tdTomato). Users may take cells from several embryonic stages to create unstained, Aqua dye, and tdTomato single-color compensation controls. Ensure unstained and Aqua dye controls are taken from reporter negative samples. <a href="#tbl2">Table 2</a> summarizes the advised minimum number of cells required for each control.</div>
<table id="tbl2">
<caption>Table 2. Minimum number of cells required for each control</caption>
<thead>
<tr>
<th>Single-color control</th>
<th>Minimum number of cells to stain</th>
<th>Source of control cells - embryonic stage</th>
<th>FMO control</th>
<th>Minimum number of cells to stain</th>
<th>Source of control cells - embryonic stage</th>
</tr>
</thead>
<tbody>
<tr>
<td>Unstained</td>
<td>100,000</td>
<td>any or mixture of stages</td>
<td>BV421 anti-CD31</td>
<td>50,000</td>
<td>any or mixture of stages</td>
</tr>
<tr>
<td>Aqua Zombie Dye–“Dead”</td>
<td>100,000</td>
<td>any or mixture of stages</td>
<td>BV421 anti-cTnT</td>
<td>50,000</td>
<td>any or mixture of stages</td>
</tr>
<tr>
<td>tdTomato</td>
<td>50,000</td>
<td>any or mixture of stages</td>
<td>Alexa Fluor 488 anti-WT1</td>
<td>50,000</td>
<td>∗in accordant to samples–prepare fmo controls for each embryonic stage analyzed</td>
</tr>
<tr>
<td> </td>
<td> </td>
<td> </td>
<td>Alexa Fluor target probes</td>
<td>50,000</td>
<td>∗as above</td>
</tr>
<tr>
<td> </td>
<td> </td>
<td> </td>
<td>tdTomato</td>
<td>50,000</td>
<td>∗as above</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> To generate the Aqua positive single-color compensation control, add 200 μL 70% ethanol to the 100 μL cell suspension. Vortex samples twice in 5 s pulses and incubate for 2 min at 20°C–24°C. Add 500 μL of PBS and centrifuge at 800 × <i>g</i> for 5 min at +4°C. Discard supernatant and resuspend in 100 μL residual volume, and repeat PBS wash. Discard as much supernatant as possible without disrupting the white and “fluffy” cell pellet/smear. Stain with Aqua fixable viability dye, wash and fix sample in IC fixation buffer (add 100 μL IC fixation buffer to 100 μL stained cell suspension). All single-color compensation controls should be fixed before storing at 2°C–8°C, protected from light.</div>
<ol start="6">
<li>Antibody staining for the cell surface marker CD31/PECAM-1, to stain coronary and lymphatic endothelial cells and endocardium.
<ol type="a">
<li>Resuspend cell pellet in 100 μL diluted BV421 anti-CD31 antibody (1:50 in cell staining buffer). Incubate for 30 min on ice (+2°C to +4°C), protected from light.</li>
<li>Add 1 mL of cold cell staining buffer, centrifuge at 500 × <i>g</i> for 5 min at +4°C, and discard supernatant, keeping 100 μL residual volume.</li>
<li>Resuspend cells in residual volume. Proceed to cell fixation and permeabilization for staining of intracellular antigens.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If cells exceed 5 × 10<sup>6</sup>, adjust staining volume to 200 μL.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Before staining for surface markers, take some cells from your allocated control samples to generate FMOs.</div>
<div class="note">
<span class="note-title">Note:</span> Coronary endothelial cells (CECs) express Wt1 from E11.5, and lymphatic endothelial cells (LECs) express Sema3d (<a href="#bib2">Lupu et al., 2020</a>). Despite utilizing a protocol for epicardial lineage tracing, with tamoxifen induction at E9.5 for highest epicardium-labeling efficiency and specificity in <i>Wt1CreERT2</i> mice, approximately 3.4% CECs are directly labeled in E17.5 hearts. Thus, it is important to distinguish (i.e., exclude) endothelial populations during analysis of the epicardium and EPDC populations.</div>
<ol start="7">
<li>Fixation, permeabilization, and antibody staining for the intracellular marker cTnT, to stain cardiomyocytes.
<ol type="a">
<li>Add 1 mL of PrimeFlow RNA fixation buffer 1, invert to mix and incubate for 30 min on ice (+2°C to +4°C), protected from light.</li>
<li>Centrifuge at 800 × <i>g</i> for 5 min at +4°C and discard supernatant, keeping 100 μL residual volume. Resuspend cells in residual volume by vortexing.</li>
<li>Add 1 mL of PrimeFlow RNA permeabilization buffer supplemented with RNase inhibitors, invert to mix and centrifuge at 800 × <i>g</i> for 7 min at +4°C. Discard supernatant, keeping 100 μL residual volume. Resuspend cells in residual volume by vortexing.</li>
<li>Repeat wash as in step c.</li>
<li>Add 100 μL diluted BV421 anti-cTnT antibody (1:100 in PrimeFlow RNA permeabilization buffer + RNase inhibitors) to 100 μL cell suspension (final Ab dilution 1:200). Incubate for 30 min on ice (+2°C to +4°C), protected from light.</li>
<li>Repeat Perm wash as in step c.</li>
<li>Add 1 mL of PrimeFlow RNA fixation buffer 2, invert to mix, and incubate for 12–20 h at 2°C–8°C, protected from light.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Fixation and permeabilization steps are consistent with the manufacturer’s instructions (PrimeFlow RNA Assay kit) with minor adjustments to centrifugation time to accommodate low cell numbers.</div>
<div class="note">
<span class="note-title">Note:</span> Users may include Alexa Fluor 488 anti-WT1 antibody at step e to stain for WT1 protein (Final Ab dilution 1:100). This allows comparison between transcript and protein level of <i>Wt1</i> in samples, as previously executed in (<a href="#bib2">Lupu et al., 2020</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Before staining for intracellular markers, take some cells from your allocated control samples to generate FMOs.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is critical to use PrimeFlow RNA Tubes and/or PrimeFlow 96-well plate, when carrying out the PrimeFlow RNA assay.</div>
<h3 id="sec3.4">Target probe hybridization and signal amplification: day 2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8.5 h</div>
<p>Staining cells for epicardial gene transcripts (mRNA) using a commercially available <i>in situ</i> hybridization assay for flow cytometry.</p>
<ol start="8">
<li>Hybridization of <i>Wt1</i>, <i>Sema3d</i>, <i>Tbx18</i>, <i>Tcf21</i>, and <i>Scx</i> probes. Multiplexed target probe combinations (staining panels) are exemplified in <a href="#tbl3">Tables 3</a> and <a href="#tbl4">4</a>, to aid division of cell suspensions to control (FMO) and experimental samples/wells.
<table id="tbl3">
<caption>Table 3. Staining panels</caption>
<thead>
<tr>
<th>Staining panel</th>
<th>Aqua</th>
<th>BV421</th>
<th>Alexa Fluor 488</th>
<th>Alexa Fluor 568 or tdTomato</th>
<th>Alexa Fluor 647</th>
<th>Alexa Fluor 750</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td>Zombie Dye</td>
<td>Anti-CD31Anti-cTnT</td>
<td>Wt1 probe</td>
<td>Tbx18 probe</td>
<td>Sema3d probe</td>
<td>Tcf21 probe</td>
</tr>
<tr>
<td>B</td>
<td>Zombie Dye</td>
<td>Anti-CD31Anti-cTnT</td>
<td>Wt1 probe</td>
<td>tdTomato</td>
<td>Sema3d probe</td>
<td>Tcf21 probe</td>
</tr>
<tr>
<td>C</td>
<td>Zombie Dye</td>
<td>Anti-CD31Anti-cTnT</td>
<td>Wt1 probe</td>
<td>Tbx18 probe</td>
<td>Scx probe</td>
<td>Sema3d probe</td>
</tr>
<tr>
<td>D</td>
<td>Zombie Dye</td>
<td>Anti-CD31Anti-cTnT</td>
<td>Tbx18 probe</td>
<td>tdTomato</td>
<td>Scx probe</td>
<td>Sema3d probe</td>
</tr>
<tr>
<td>E</td>
<td>Zombie Dye</td>
<td>Anti-CD31Anti-cTnT</td>
<td>Anti-WT1 antibody</td>
<td>tdTomato</td>
<td>Wt1 probe</td>
<td>–</td>
</tr>
</tbody>
</table>
<table id="tbl4">
<caption>Table 4. 96-well experimental template for 3 staining panels and 2 embryonic stages</caption>
<thead>
<tr>
<th> </th>
<th colspan="6"><i>R26tdTom; Wt1CreERT2/+</i></th>
<th colspan="6"><i>R26tdTom; +/+ OR WT</i></th>
</tr>
</thead>
<tbody>
<tr>
<td> </td>
<td>1</td>
<td>2</td>
<td>3</td>
<td>4</td>
<td>5</td>
<td>6</td>
<td>7</td>
<td>8</td>
<td>9</td>
<td>10</td>
<td>11</td>
<td>12</td>
</tr>
<tr>
<td>A</td>
<td>AF488FMOE11.5 (B)</td>
<td>TomFMOE11.5 (B)</td>
<td>AF647FMOE11.5 (B)</td>
<td>AF750FMOE11.5 (B)</td>
<td> </td>
<td> </td>
<td>AF488FMOE11.5 (A)</td>
<td>AF568FMOE11.5 (A)</td>
<td>AF647FMOE11.5 (A)</td>
<td>AF750FMOE11.5 (A)</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>B</td>
<td>AF488FMOE11.5 (D)</td>
<td>TomFMOE11.5 (D)</td>
<td>AF647FMOE11.5 (D)</td>
<td>AF750FMOE11.5 (D)</td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>C</td>
<td>AF488FMOE13.5 (B)</td>
<td>TomFMOE13.5 (B)</td>
<td>AF647FMOE13.5 (B)</td>
<td>AF750FMOE13.5 (B)</td>
<td>AquaFMO (B)</td>
<td>BV421FMO (B)</td>
<td>AF488FMOE13.5 (A)</td>
<td>AF568FMOE13.5 (A)</td>
<td>AF647FMOE13.5 (A)</td>
<td>AF750FMOE13.5 (A)</td>
<td>AquaFMO (A)</td>
<td>BV421FMO (A)</td>
</tr>
<tr>
<td>D</td>
<td>AF488FMOE13.5 (D)</td>
<td>TomFMOE13.5 (D)</td>
<td>AF647FMOE13.5 (D)</td>
<td>AF750FMOE13.5 (D)</td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>E</td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>F</td>
<td>E11.5 #1 (B)</td>
<td>E11.5 #1 (D)</td>
<td>E13.5 #1 (B)</td>
<td>E13.5 #1 (D)</td>
<td> </td>
<td> </td>
<td>E11.5 #1 (A)</td>
<td>E13.5 #1 (A)</td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>G</td>
<td>E11.5 #2 (B)</td>
<td>E11.5 #2 (D)</td>
<td>E13.5 #2 (B)</td>
<td>E13.5 #2 (D)</td>
<td> </td>
<td> </td>
<td>E11.5 #2 (A)</td>
<td>E13.5 #2 (A)</td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>H</td>
<td>E11.5 #3 (B)</td>
<td>E11.5 #3 (D)</td>
<td>E13.5 #3 (B)</td>
<td>E13.5 #3 (D)</td>
<td> </td>
<td> </td>
<td>E11.5 #3 (A)</td>
<td>E13.5 #3 (A)</td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table>
<ol type="a">
<li>Following fixation, centrifuge samples at 800 × <i>g</i> for 7 min at 20°C–24°C. Discard supernatant, keeping 100 μL residual volume. Resuspend cells in residual volume by vortexing.</li>
<li>Add 1 mL of PrimeFlow RNA wash buffer, invert to mix and centrifuge samples at 800 × <i>g</i> for 7 min at 20°C–24°C. Discard supernatant, keeping 100 μL residual volume. Resuspend cells in residual volume by vortexing.</li>
<li>Repeat wash as in step b.</li>
<li>If users aim to stain 100,000 cells per target probe panel, adjust total volume for each sample using PrimeFlow RNA wash buffer to achieve cell concentration of 500,000 cells/mL. Divide sample into staining panels by transferring 200 μL of cell suspension into the corresponding well of a PrimeFlow 96-well plate. <a href="#sec7.5">Troubleshooting 3</a>
</li>
<li>Centrifuge plate at 1,000 × <i>g</i> for 7 min at 20°C–24°C (use an adhesive plate film to cover). Discard supernatant by inverting the plate in a single motion with sufficient force, then blot plate by gently placing it on a paper towel. Briefly vortex to resuspend cells in the ~10 μL residual volume.</li>
<li>Add 50 μL of PrimeFlow wash buffer to each well, then add 50 μL diluted target probes (1:20 in Target Probe Diluent) to their corresponding wells (refer to <a href="#tbl3">Table 3</a> for advice on panels and <a href="#tbl4">Table 4</a> for 96-well template), and pipet to mix.</li>
<li>Incubate plate with the lid on for 2 h at 40°C. If using the HybEZ oven, place plate into the center of the humidity control tray.</li>
<li>Continue protocol as per manufacturer’s instructions in Appendix 7 from step 26 (PrimeFlow RNA assay), however users should modify the centrifugation time to 7 min and add 50 μL of PrimeFlow RNA PreAmp Mix, PrimeFlow RNA Amp Mix, and PrimeFlow RNA Label Probes, respectively, to 50 μL cell suspension.</li>
<li>Following signal amplification and washes, store samples in IC fixation buffer (100 μL cell suspension in PrimeFlow RNA storage buffer + 100 μL IC fixation buffer). Samples should be stored at 2°C–8°C, kept dark, and using an adhesive plate film. We advise acquiring data within 3 days.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Allocate wells for corresponding target probe FMOs and divide control sample accordingly. Ensure no fewer than 50,000 cells are used per target probe FMO control sample. FMO controls are treated the same way as experimental samples and should be prepared in parallel (<a href="#tbl2">Tables 2</a> and <a href="#tbl4">4</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> While unstained, Aqua dye, and tdTomato single-color compensation controls are prepared on day 1 using cells; BV421, Alexa Fluor 488, Alexa Fluor 568, Alexa Fluor 647, and Alexa Fluor 750 single-color controls are typically prepared on day 2 using appropriate compensation beads. Bead-based compensation controls are prepared as per manufacturer’s instructions (PrimeFlow RNA assay): (a) add fluorochrome-conjugated antibody (dilution equivalent to sample staining) or 5 μL PrimeFlow Compensation Control to 100 μL compensation bead suspension and incubate for 15–30 min, on ice (+2°C to +4°C), protected from light. (b) Add 1 mL of PBS and centrifuge at 600 × <i>g</i> for 5 min at +4°C. (c) Discard supernatant and resuspend beads in 100 μL residual volume. (d) Add 100 μL IC Fixation buffer to the 100 μL stained bead suspension. All single-color compensation controls should be fixed before storing at +4°C, protected from light.</div>
<div class="note">
<span class="note-title">Note:</span> We advise staining cells in bulk numbers (1.5 mL tubes) during antibody staining of cell surface and intracellular antigens, however if staining panels are more complex and users are working with a considerable number of samples, they may choose to transfer their samples earlier into PrimeFlow 96-well plates. <a href="#sec7.5">Troubleshooting 3</a>
</div>
<div class="note">
<span class="note-title">Note:</span> If samples ≥1 × 10<sup>6</sup> cells per well, we advise following the manufacturer’s instructions on volumes (e.g., diluted target probes) and centrifugation time.</div>
<div class="note">
<span class="note-title">Note:</span> If litters are to be harvested over several days, and users wish to compare/quantify gene target levels between samples, we advise limiting tissue collection to 3 consecutive days and carrying out the target probe hybridization for all samples concurrently. To achieve this, samples may be stored in PrimeFlow RNA wash buffer with RNase inhibitors at 2°C–8°C, in the dark, after step 8c (washing of immunostained fixed samples). Samples may be stored for a maximum of 2 days. However, it is important to note that fluorochrome-conjugated antibody brightness will dim with time and should be considered during analysis.</div>
<h3 id="sec3.5">Data collection: day 3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–6 h</div>
<p>Data collection using BD LSRFortessa X-20 flow cytometer: fitted with UV 355 nm, violet 405 nm, blue 488 nm, yellow-green 561 nm, and red 640 nm lasers and plate-based acquisition (HTS).</p>
<ol start="9">
<li>Before accessing the flow cytometer:
<ol type="a">
<li>Wash compensation controls with cell staining buffer and resuspend in 300 μL cell staining buffer. Transfer to 5 mL flow cytometry tubes.</li>
<li>Centrifuge PrimeFlow 96-well plate containing samples and FMOs at 1,000 × <i>g</i> for 7 min at +4°C, and discard supernatant. Resuspend cells in 220 μL cell staining buffer per well, and transfer to V-bottom 96-well polypropylene plates for data collection. <a href="#sec7.7">Troubleshooting 4</a>
</li>
<li>Place tubes and plate on ice (+2°C to +4°C) and protect from light.</li>
</ol>
</li>
<li>Set up PMT voltages for forward-scatter, side-scatter (FSC/SSC) and fluorescence channels.
<ol type="a">
<li>Refer to example in <a href="#tbl5">Table 5</a> for cytometer laser/filter configuration.
<table id="tbl5">
<caption>Table 5. Example of cytometer configuration to be used for data collection</caption>
<thead>
<tr>
<th> </th>
<th>Aqua Zombie Dye</th>
<th>BV421</th>
<th>Alexa Fluor 488</th>
<th>Alexa Fluor 568</th>
<th>tdTomato</th>
<th>Alexa Fluor 647</th>
<th>Alexa Fluor 750</th>
</tr>
</thead>
<tbody>
<tr>
<td>Laser (nm)</td>
<td>355</td>
<td>405</td>
<td>488</td>
<td>561</td>
<td>561</td>
<td>640</td>
<td>640</td>
</tr>
<tr>
<td>Filter</td>
<td>525/35</td>
<td>450/50</td>
<td>530/30</td>
<td>610/20</td>
<td>586/15</td>
<td>670/30</td>
<td>780/60</td>
</tr>
</tbody>
</table>
</li>
<li>To set up FSC/SSC, use an unstained cell sample that has undergone the PrimeFlow assay. This protocol tends to increase autofluorescence and FSC/SSC properties will differ from traditional immunostaining protocols. Ensure that you are collecting all cells of interest on scale and eliminating debris from acquisition.</li>
<li>Ensure that fluorescence channel signals are on scale and adjust PMT voltages accordingly. Use unstained, single-color stained, and full panel samples specifically prepared to help adjust PMT voltages.</li>
</ol>
</li>
<li>Set compensation.
<ol type="a">
<li>For Aqua dye compensation, acquire and record at least 1,000 positively single-color stained cells. Remove sample, then replace with unstained cell sample and append recording. This will result in 2 signals, a positive and a negative peak fluorescence signal, in the same fluorescence channel recording. Repeat this for tdTomato single-color control. The remaining compensation samples should be acquired as usual.</li>
<li>Check compensation settings and proceed to plate-based acquisition (HTS).</li>
</ol>
</li>
<li>Collect and analyze samples.
<ol type="a">
<li>Set up plate-based acquisition, ensure the V-bottom setting is selected. The throughput mode should be set to “Standard.” It is important to keep sample flow rate and sample volume acquired (e.g., 200 μL) consistent between wells. Set number of mixes and wash volume to maximum. There is no need to set a maximum number of events to record in the analysis tab; acquire and record all possible events in your total cells gate (P1). The number of events recorded will be similar between samples as users: (a) stain a set number of cells per sample, (b) resuspend samples in a set volume of buffer (220 μL) prior data collection, and (c) acquire a set volume (200 μL) during data collection.</li>
<li>Cells may have settled to the bottom of the well, resuspend with a multichannel before loading plate. Start collection and analysis of FMOs and samples.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If users intend on quantifying and comparing transcript levels between samples, they must collect and analyze samples during the same flow cytometry session.</div>
<div class="critical">
<span class="critical-title">Critical:</span> We advise to prepare and acquire FMOs for each developmental stage, and for each independent experiment.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Successful dissociation of embryonic hearts will establish a single-cell suspension demonstrating &gt;94% viability with an expected yield per heart as summarized in <a href="#tbl1">Table 1</a>. In this study, E14.5 embryonic hearts derived from <i>R26tdTom; Wt1CreERT2/+</i> (labeled) and <i>R26tdTom;+/+</i> (unlabeled or WT) embryos underwent the PrimeFlow RNA assay protocol. A gating strategy was generated to maximize exclusion of unwanted cells, i.e., dead cells and other major cardiac cell populations, and subsequent detection of our population of interest, i.e., epicardium and epicardium-derived cells.</p>
<p>Following exclusion of dead cells and doublets, we then excluded cardiomyocytes (cTnT<sup>+</sup>) and endothelial cells (CD31<sup>+</sup>; CECs and endocardium) by gating the cTnT<sup>−</sup>CD31<sup>−</sup> fraction. For reporter-labeled hearts, we subsequently gated tdTomato<sup>+</sup> cells, a fraction composed of epicardium and epicardium-derived cells only. Expression of five epicardial genes was simultaneously assessed within the tdTomato<sup>+</sup> population by using two different target probe combinations: Wt1/Sema3d/Tcf21 (<a href="#tbl3">Table 3</a>; staining panel B) and Tbx18/Scx/Sema3d (<a href="#tbl3">Table 3</a>; staining panel D), with Sema3d serving to ensure comparability between probe sets. Subsequent gating of <i>Wt1+Tcf21+Sema3d+</i> (<a href="#fig2">Figure 2</a>A) and <i>Tbx18+Sema3d+Scx+</i> (<a href="#fig2">Figure 2</a>B), demonstrated a major proportion of tdTomato+ cells with overlapping expression of <i>Wt1</i>, <i>Tcf21</i>, <i>Tbx18</i>, <i>Sema3d</i>, and <i>Scx</i>. The spatiotemporal expression of these genes was previously characterized in the developing heart, demonstrating significant overlap of expression in the epicardium with a gradual decrease in levels of transcription as the epicardium quiesces (<a href="#bib2">Lupu et al., 2020</a>). Epicardium-derived cells (EPDCs) invading the myocardium–particularly progenitor cells which are yet to commit to mural or fibroblast cells–demonstrate a rapid downregulation and loss of <i>Wt1</i>, <i>Sema3d</i>, <i>Tbx18</i>, and <i>Scx</i>. In contrast, <i>Tcf21</i> is upregulated in EPDCs (<a href="#bib2">Lupu et al., 2020</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/492-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Gating strategy for assessing expression of <i>Wt1</i>, <i>Tcf21</i>, <i>Tbx18</i>, <i>Sema3d</i>, and <i>Scx</i> in epicardium-lineage traced cells of E14.5 hearts (<i>R26tdTom; Wt1CreERT2/+</i>)</div>
<p>Epi, epicardium; EPDCs, epicardium-derived cells.</p>
</figcaption>
</figure>
<p>Hearts from embryos of wild-type genotype do not benefit from reporter-labeled epicardium and EPDCs. However, epicardial cells may be identified through overlapping expression of these markers when assessed simultaneously. Notably, if assessed alone, these markers cannot be used to identify the epicardium, due to their varying expression in non-epicardial compartments and differentiated EPDCs throughout development (<a href="#bib2">Lupu et al., 2020</a>). Following exclusion of unwanted cells, gating <i>Wt1</i><sup>+</sup><i>Sema3d</i><sup>+</sup><i>Tbx18</i><sup>+</sup><i>Tcf21</i><sup>+</sup> cells within the cTnT<sup>−</sup>CD31<sup>−</sup> fraction, enriches for the epicardium (<a href="#fig3">Figure 3</a>). However, it is important to note that Wt1<sup>low</sup>Sema3d<sup>low</sup>Tbx18<sup>low</sup>Tcf21<sup>high</sup> (<a href="#fig3">Figure 3</a>, #) may comprise subepicardial cells (early EPDCs).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/492-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Gating strategy for assessing expression of <i>Wt1</i>, <i>Tcf21</i>, <i>Tbx18</i>, <i>Sema3d</i>, and <i>Scx</i> in the cTnT<sup>−</sup>CD31<sup>−</sup> fraction of E14.5 hearts (no lineage tracing, <i>R26tdTom; +/+</i>)</div>
<p>Epi, epicardium; EPDCs, epicardium-derived cells; # contains early EPDCs normally located in the subepicardium.</p>
</figcaption>
</figure>
<p>Alternatively, the protocol in this study may be used to:</p>
<ol>
<li>Assess other genes expressed in the epicardium and EPDCs, and quantify percentage positive cells for a particular gene of interest.</li>
<li>Assess transcriptional changes between conditions.</li>
<li>Validate candidate genes from single-cell RNA-seq datasets and quantify population percentage.</li>
<li>Compare transcript versus protein expression in epicardium-lineage traced cells (e.g., <i>Wt1</i> vs WT1) and reporter-labeling efficiency (<a href="#bib2">Lupu et al., 2020</a>).</li>
<li>Assess level of target gene expression, for example after tamoxifen-induced recombination in epicardium-specific knockout lines.</li>
<li>Generate a single-cell suspension for scRNA-seq. Resuspend cells in a more appropriate buffer at step 4g, and proceed to prepare cells for plate or droplet-based sequencing.</li>
</ol>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Quantify percentage (%) positive from live cells or from parent population (e.g., tdTomato+). Gated cell counts should be exported from your FlowJo workspace, and then percentages calculated accordingly.</p>
<p>Quantify mean fluorescence intensity to represent gene expression level in the gated epicardial population. Gate epicardial cells, then the geometric mean calculated for each target probe should be exported from your FlowJo workspace. Finally, subtract the geometric mean of your respective FMO control sample to calculate the gMFI.</p>
<p>Percentage positive cells and gMFI for different experimental samples may be compared using a statistical test suitable for the particular experimental design.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol requires a minimum of 100,000 cells per staining panel for each biological replicate, meaning more hearts need to be collected for early stages such as E11.5 and E13.5. Due to the nature of the protocol, i.e., multiple staining and washing steps, and cell permeabilization, the user should expect to lose a significant number of cells which will be evident during data collection. Furthermore, it is necessary to create FMO controls specific for each developmental stage, to assist with gating of RNA target associated fluorophores. In some cases, the signal shift is small between positive and negative cells (e.g., Tcf21–Alexa Fluor 750), thus FMOs are critical. However, this means that more cells are required for each experiment to complete a full array of FMO controls. This protocol may be unsuccessful and data inconclusive if the dissociated tissue yields too few cells for experimental and control sample staining.</p>
<p>Target probe types vary in brightness and may limit users to specific types if low transcript expression is expected, e.g., Scx should be allocated to Alexa Fluor 647 (type 1) with high sensitivity. Alexa Fluor 750 has intermediate-to-low sensitivity, thus we advise to use target probes against highly expressed genes in this fluorescence channel e.g., Tcf21. This may limit target probe combinations (<a href="#sec7.9">troubleshooting 5</a>). Notably, to quantify and compare level of target expression between embryonic stages or conditions, hybridization and data collection need to be carried out for all samples on the same day. Target probe types should also be kept consistent, as the number of branches of the branched-DNA technology may vary between types (e.g., Tbx18 type 1 vs Tbx18 type 4 target probe), causing differences in signal intensity independent of fluorophore brightness or cytometer settings. Finally, some fluorochromes are not compatible with the PrimeFlow RNA assay and were not used in this study. Refer to manufacturer’s manual for compatibility if users wish to use other fluorochrome-conjugated antibodies.</p>
<p>This protocol is strengthened by the use of an epicardial lineage mouse model, and we strongly advise temporal labeling to confidently assess gene expression in the epicardium and epicardium-derived cells. In wild-type hearts, the epicardium can be enriched by gating cTnT<sup>−</sup>CD31<sup>−</sup><i>Wt1</i><sup>+</sup><i>Sema3d</i><sup>+</sup> cells, however this population becomes harder to distinguish at later embryonic stages. This is due to the epicardium becoming a smaller fraction of the cTnT<sup>−</sup>CD31<sup>−</sup> population, and <i>Wt1</i> and <i>Sema3d</i> expression being downregulated as the epicardium quiesces. Furthermore, only 2 available fluorescence channels are left to nominate new PrimeFlow target probes for assessing other genes. <i>Upk3b,</i> previously shown to be exclusively expressed in proepicardial organ (PEO) and epicardial cells during heart development (<a href="#bib2">Lupu et al., 2020</a>; <a href="#bib5">Rudat et al., 2014</a>) may be used to gate the epicardium in wild-type hearts, and thus freeing fluorescence channels to assess 3 other RNA targets simultaneously.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low yield of cells after dissociation and sample preparation.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>This problem may be a result of: 1) prolonged incubation with enzyme mix, 2) enzyme mix prepared incorrectly, or 3) prolonged incubation in RBC lysis buffer. To avoid this:</p>
<ul>
<li>Terminate enzymatic dissociation immediately after the Dissociator program is complete.</li>
<li>Increase FBS in the culture medium (e.g., DMEM + 20% FBS) and ensure it is cold on use.</li>
<li>If working with many samples, stagger the RBC lysis step between samples to ensure the 2 min incubation is not exceeded. May also use cold RBC lysis to reduce effectiveness.</li>
</ul>
<h3 id="sec7.3">Problem 2</h3>
<p>Low cell viability after dissociation and sample preparation.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Viability should be assessed early on at cell counting in step 4h, since cells with compromised viability will stain blue and should not exceed 10% positive cells. Reduced viability may be a result of reasons encountered in problem 1, but also due to prolonged incubation of hearts on ice before dissociation. To avoid this:</p>
<ul>
<li>Stagger harvest accordingly to limit time hearts spend on ice (+2°C to +4°C) to 1 h.</li>
<li>Supplement PBS with 2% FBS if hearts will be on ice (+2°C to +4°C) for longer than 20 min.</li>
</ul>
<h3 id="sec7.5">Problem 3</h3>
<p>No events acquired during data collection.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>This problem can arise if incompatible plastic tubes or plates are used during target probe hybridization and signal amplification.</p>
<ul>
<li>Only use PrimeFlow RNA 1.5 mL tubes or PrimeFlow RNA 96-well plate during these steps.</li>
</ul>
<h3 id="sec7.7">Problem 4</h3>
<p>Acquisition of sample volume is unstable or there are many events recorded that display scatter properties suggestive of micro bubbles or debris.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>This problem can arise if samples are in Storage buffer and/or PrimeFlow 96-well plates during data acquisition. We have found that Storage buffer precipitates will be recorded as events during data collection, and PrimeFlow 96-well plates can cause issues with plate-based acquisition fluidics.</p>
<ul>
<li>Centrifuge cells and resuspend in cell staining buffer on data collection day.</li>
<li>Transfer cells to a more appropriate 96-well plate such as V-bottom 96-well polypropylene plate for plate-based acquisition.</li>
</ul>
<h3 id="sec7.9">Problem 5</h3>
<p>No positive staining of certain RNA target probes or all target probes.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>This problem can arise if select genes are not expressed in your cells, or the PrimeFlow assay protocol was carried out unsuccessfully. In addition, RNA quality may be compromised in your samples.</p>
<ul>
<li>Users may use RNA target probes against housekeeping genes such as <i>ACTB</i>, to act as positive controls to determine protocol success.</li>
<li>Test another target probe type against your RNA target of interest to ensure lack of signal is not due to use of intermediate-to-low sensitivity Alexa Fluor 488 and Alexa Fluor 750 probes for lowly expressed genes.</li>
</ul>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cossarizza, A., Chang, H., Radbruch, A., Acs, A., Adam, D., Adam-Klages, S., Agace, W.W., Aghaeepour, N., Akdis, M., Allez, M., et al. (2019). Guidelines for the use of flow cytometry and cell sorting in immunological studies (second edition). Eur. J. Immunol. <i>49</i>, 1457-1973. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00066-6/sref1">View at publisher</a></p>
<p id="bib2">Lupu, I.-E., Redpath, A.N., and Smart, N. (2020). Spatiotemporal analysis reveals overlap of key proepicardial markers in the developing murine heart. Stem Cell Reports <i>14</i>, 770-787. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00066-6/sref2">View at publisher</a></p>
<p id="bib3">Madisen, L., Zwingman, T.A., Sunkin, S.M., Oh, S.W., Zariwala, H.A., Gu, H., Ng, L.L., Palmiter, R.D., Hawrylycz, M.J., Jones, A.R., et al. (2010). A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. Nat. Neurosci. <i>13</i>, 133-140. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00066-6/sref3">View at publisher</a></p>
<p id="bib4">Meilhac, S.M. and Buckingham, M.E. (2018). The deployment of cell lineages that form the mammalian heart. Nat. Rev. Cardiol. <i>15</i>, 705-724. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00066-6/sref4">View at publisher</a></p>
<p id="bib5">Rudat, C., Grieskamp, T., Röhr, C., Airik, R., Wrede, C., Hegermann, J., Herrmann, B.G., Schuster-Gossler, K., and Kispert, A. (2014). Upk3b is dispensable for development and integrity of urothelium and mesothelium. PLoS One <i>9</i>, e112112. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00066-6/sref5">View at publisher</a></p>
<p id="bib6">Zhou, B., Ma, Q., Rajagopal, S., Wu, S.M., Domian, I., Rivera-Feliciano, J., Jiang, D., von Gise, A., Ikeda, S., Chien, K.R., et al. (2008). Epicardial progenitors contribute to the cardiomyocyte lineage in the developing heart. Nature <i>454</i>, 109-113. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00066-6/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Nicola Smart (<a href="mailto:nicola.smart@dpag.ox.ac.uk">nicola.smart@dpag.ox.ac.uk</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents or mouse lines.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate new datasets or codes.</p>
<h3>Acknowledgments</h3>
<p>We thank Prof William Pu, Harvard, for the <i>Wt1</i>CreERT2 line; the Sir William Dunn School of Pathology Flow Cytometry facility for advice and use of equipment; and Biomedical Services staff for animal husbandry. We thank Dr Corinna Scholz, Field Application Scientist Flow Cytometry at Thermo Fisher Scientific for advice on PrimeFlow RNA assay applications. This work was funded by the <a href="https://doi.org/10.13039/501100000274">British Heart Foundation</a> (BHF): DPhil Studentship (FS/15/68/32042), Project grant (PG/16/27/32114), and BHF Ian Fleming Senior Basic Science Research Fellowship (FS/13/4/30045 and FS/19/32/34376).</p>
<h3>Author Contributions</h3>
<p>Methodology, A.N.R. and I.-E.L.; Investigation, A.N.R. and I.-E.L.; Formal Analysis, A.N.R. and I.-E.L.; Visualization, A.N.R.; Writing – Original Draft, A.N.R.; Writing – Review &amp; Editing, A.N.R., I.-E.L., and N.S.; Funding Acquisition, N.S.; Supervision, A.N.R. and N.S.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>