<article>
<section>
<h1 id="header">Secret3D Workflow for Secretome Analysis</h1>
<p><time datetime="2020-12-18">Published: December 18, 2020</time></p>
<p>Vittoria Matafora<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Angela Bachi<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>IFOM- FIRC Institute of Molecular Oncology, 20139 Milan, Italy</p>
<p id="fn1"><sup>2</sup>Technical Contact</p>
<p id="fn2"><sup>3</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:vittoria.matafora@ifom.eu">vittoria.matafora@ifom.eu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:angela.bachi@ifom.eu">angela.bachi@ifom.eu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100162">10.1016/j.xpro.2020.100162</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Secretome analysis is crucial to unravel extracellular processes. However, secreted proteins are difficult to detect in mass-spectrometry-based analysis due to contamination of serum proteins deriving from cell culture media and to high glycosylation, which hampers tryptic digestion. Secret3D workflow is an optimized protocol for the global analysis of secretome from <i>in vitro</i> cultured cells. It allows efficient and robust protein identification and quantitation and provides information on putative N-glycosylation sites of the secreted proteins.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib4">Matafora et al. (2020)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Secret3D workflow allows the analysis of secreted proteins from <i>in vitro</i> cultured cells</li>
<li>De-glycosylation and double digestion enhance protein identification and quantification</li>
<li>Identification of putative glycosylation sites of the secreted proteins</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/325-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before You Begin</h2>
<h3 id="sec1.1">Prepare Medium and Heavy Cell Media for Cell Culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> can be made before starting cell culture</div>
<ol>
<li>Use Dulbecco Modified Eagle's Medium DMEM deprived of arginine and lysine + 5% FBS Dialyzed + 2 mM L-Glutamine or RPMI-1640 deprived of arginine and lysine + 10% FBS Dialyzed + 2 mM L-Glutamine according to cell type and supplemented with medium- and heavy-labeled amino acids, added at the concentration originally contained in the media. The medium culture contains arginine (L-Arg <sup>13</sup>C<sub>6</sub>-<sup>14</sup>N<sub>4</sub>) and lysine (L-Lys-4,4,5,5-D4) and the heavy culture contains arginine (L-Arg <sup>13</sup>C<sub>6</sub>-<sup>15</sup>N<sub>4</sub>) and lysine (L-Lys <sup>13</sup>C<sub>6</sub>-<sup>15</sup>N<sub>2</sub>) amino acids (<a href="#bib5">Ong et al., 2002</a>).</li>
</ol>
<h3 id="sec1.2">Incorporate Medium- and Heavy-Labeled Amino Acids into the Proteome of Cultured Cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<ol start="2">
<li>Cells are cultured in the labeled medium for at least five cell divisions to obtain full incorporation of the labeled amino acids in their proteome.</li>
</ol>
<h3 id="sec1.3">Perform Incorporation Test</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="3">
<li>Analyze the proteome of a small aliquot of cell lysate to check the level of incorporation of labeled arginine and lysine (<a href="#bib3">Harsha et al., 2008</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The incorporation level should be at least 95%, otherwise the quantitation is biased toward the light condition. If the cells are not fully incorporated, the culture in medium and heavy cell media should be continued until full incorporation is reached.</div>
<h3 id="sec1.4">Starve the Cells in Serum-free Medium</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="4">
<li>Equal numbers of cells are split into 15 cm dishes at roughly 50% confluence. Once cell lines reach ∼70% confluence, one 15-cm dishes of each cell line is washed 3× with PBS and 3× with serum-free media. Cells are starved in serum-free medium for 18 h.</li>
</ol>
<h3 id="sec1.5">Check Cells Viability upon Serum Starvation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="5">
<li>Collect the medium for the analysis of the secreted proteins (secretome). The remaining adherent cells are collected by trypsinization and used for viability assay.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Viability ≥95% is accepted, avoid dead cells that might contaminate the secretome.</div>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>Heavy L-arginine-<sup>13</sup>C<sub>6</sub>,<sup>15</sup>N<sub>4</sub> HCl</td>
<td>Isotec (Sigma)</td>
<td>Cat#608033-500 MG</td>
</tr>
<tr>
<td>Heavy L-lysine-<sup>13</sup>C<sub>6</sub>,<sup>15</sup>N<sub>2</sub> 2HCl</td>
<td>Cambridge Isotope Laboratories</td>
<td>Cat#CILCLM-291-0.25</td>
</tr>
<tr>
<td>Medium L-arginine-<sup>13</sup>C<sub>6</sub> HCl</td>
<td>Cambridge Isotope Laboratories</td>
<td>Cat#CILCLM-2265-05</td>
</tr>
<tr>
<td>Medium L-lysine-4,4,5,5-D<sub>4</sub> 2HCl</td>
<td>Cambridge Isotope Laboratories</td>
<td>Cat#CILCLM-2640-0.5</td>
</tr>
<tr>
<td>2-Chloroacetamide</td>
<td>Sigma-Aldrich</td>
<td>Cat#C0267</td>
</tr>
<tr>
<td>Ammonium bicarbonate</td>
<td>Sigma-Aldrich</td>
<td>Cat#09830</td>
</tr>
<tr>
<td>Acetonitrile</td>
<td>Fisher Chemical</td>
<td>Cat#A955-4</td>
</tr>
<tr>
<td>Formic acid</td>
<td>Fisher Chemical</td>
<td>Cat#A117-50</td>
</tr>
<tr>
<td>Tris hydrochloride Tris-HCl</td>
<td>Sigma-Aldrich</td>
<td>Cat#10812846001</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Sigma-Aldrich</td>
<td>Cat#S7653-250G</td>
</tr>
<tr>
<td>Phosphate buffered saline PBS</td>
<td>Sigma-Aldrich</td>
<td>Cat#P5493</td>
</tr>
<tr>
<td>TCEP</td>
<td>Thermo scientific</td>
<td>Cat#20490</td>
</tr>
<tr>
<td>Buffer G</td>
<td>New England Biolabs</td>
<td>Cat#P0704S</td>
</tr>
<tr>
<td>PNGase F</td>
<td>New England Biolabs</td>
<td>Cat#P0704S</td>
</tr>
<tr>
<td>Trypsin, Sequencing Grade, modified</td>
<td>ROCHE</td>
<td>Cat#11 418 025 001</td>
</tr>
<tr>
<td>Endoproteinase Glu-C Sequencing Grade</td>
<td>ROCHE</td>
<td>Cat#11420399001</td>
</tr>
<tr>
<td>DMEM, high glucose, no glutamine, no lysine, no arginine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A1443101</td>
</tr>
<tr>
<td>RPMI Medium (lysine and arginine depleted)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#88365</td>
</tr>
<tr>
<td>FBS Dialyzed USA Origin</td>
<td>Invitrogen</td>
<td>Cat#26400044</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>Bradford</td>
<td>Applied Biosystems</td>
<td>Cat#4368814</td>
</tr>
<tr>
<td colspan="3">Deposited Data</td>
</tr>
<tr>
<td>.RAW files were deposited together with all peptides identified and parameters used for the analysis</td>
<td>Peptide Atlas repository</td>
<td><a href="http://www.peptideatlas.org/PASS01358">http://www.peptideatlas.org/PASS01358</a></td>
</tr>
<tr>
<td colspan="3">Experimental Models: Cell Lines</td>
</tr>
<tr>
<td>IGR37</td>
<td>DSMZ</td>
<td>Cat#ACC-237</td>
</tr>
<tr>
<td>IGR39</td>
<td>DSMZ</td>
<td>Cat#ACC-239</td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>Microsoft Excel 2010</td>
<td>Microsoft Office</td>
<td>N/A</td>
</tr>
<tr>
<td>MaxQuant</td>
<td>Max Planck Institute of Biochemistry</td>
<td><a href="https://www.maxquant.org/maxquant/">https://www.maxquant.org/maxquant/</a></td>
</tr>
<tr>
<td>Perseus</td>
<td>Max Planck Institute of Biochemistry</td>
<td><a href="https://www.maxquant.org/perseus/">https://www.maxquant.org/perseus/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<p>Tris 100 mM buffered to pH 8.0 (Tris-HCl), urea 8 M is required. This buffer is for the solubilization of secreted proteins.</p>
<p>TCEP solution 1 M and chloroacetamide 1 M in Tris-HCl are required to reduce and alkylate disulfide bonds of cysteine residues.</p>
<p>Ammonium bicarbonate 50 mM is required to solubilize trypsin, phosphate buffer pH 7.8 is used to solubilize Glu-C, while NaCl 0.5 M solution is required to elute peptides from the filter.</p>
<p>Solvent A (2% ACN, 0.1% formic acid) and solvent B (80% ACN, 0.1% formic acid) are required for nanoLC separation of the purified peptides.</p>
<p>The Bradford Assay can be used for protein quantification, but any other similar methods could also be used.</p>
<p>For mass spectrometry analysis, high-resolution Q-Exactive HF mass spectrometer connected to the EASY-nLC 1000 HPLC system (Thermo Fisher Scientific) allows the proteomic analysis of secreted proteins, but also other high-resolution mass spectrometers could be used.</p>
<p>Data analysis software required: Xcalibur 4.0, MaxQuant 1.6, Perseus 1.5. MaxQuant software (1.5.2.8) and Perseus software (version 1.5.6.0) are used for the analysis showed below; however, other versions are suited for the quantitative analysis.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-By-Step Method Details</h2>
<h3 id="sec3.1">Seeding Cells (Day 0)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 30 min</div>
<p>This step describes how to seed the medium- or heavy-labeled IGR37 or IGR39 melanoma cell line.</p>
<ol>
<li>Remove and discard culture medium from the pre-cultured IGR37 or IGR39 melanoma cells in 15 cm dish.</li>
<li>Wash attached cells with 10 mL of autoclaved phosphate buffered saline (PBS) for three times to remove serum (<a href="#troubleshooting">Troubleshooting 1</a>).</li>
<li>Add 1.0 mL of 0.05% Trypsin-EDTA solution, incubate at 37°C for 3–5 min and observe cells under an inverted microscope. If required, extend the incubation time until all the cells detach from the plastic dish.</li>
<li>Add 9.0 mL of complete medium- or heavy-labeled growth medium and aspirate cells by gently pipetting.</li>
<li>Transfer the cell suspension into a 15 mL conical centrifuge tube and centrifuge at 250 × <i>g</i> for 5 min at 25°C (<a href="#troubleshooting">Troubleshooting 2</a>).</li>
<li>Remove and discard culture medium and suspend the cell pellet in 1.0 mL complete medium- or heavy-labeled growth medium and check the cell number using a cell counter.</li>
<li>Seed 1 × 10<sup>6</sup> cells in 10 mL complete medium- or heavy-labeled growth medium into one 15 cm dish.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> generally, by seeding 1 × 10<sup>6</sup> cells more than 2 × 10<sup>6</sup> cells are collected at day 2. If not, seed more than one plate for each sample.</div>
<div class="note">
<span class="note-title">Note:</span> depending on the cell type, 1 × 10<sup>6</sup> cells may look sparse, in this case seed at least at 50% confluency.</div>
<h3 id="sec3.2">Cells Starvation in Serum-free Medium (Day 1)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This step describes how to eliminate serum proteins from the medium.</p>
<ol start="8">
<li>Remove and discard the culture medium.</li>
<li>Wash the attached cells with 10 mL of autoclaved PBS for three times.</li>
<li>Wash the attached cells with 10 mL of medium- or heavy-labeled growth medium deprived of fetal bovine serum (FBS) for three times.</li>
<li>Add 10 mL of medium- or heavy-labeled growth medium FBS deprived, starve the cells for 18 h.</li>
</ol>
<h3 id="sec3.3">Secretome Preparation (Day 2)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>This step describes how to prepare the secreted proteins extract.</p>
<ol start="12">
<li>Collect the medium from 2 × 10<sup>6</sup> cells previously starved in serum-free medium for 18 h. Centrifuge at 250 × <i>g</i> for 5 min to eliminate dead cells. Filter the supernatant with 0.22 μm filters.</li>
<li>Mix the heavy and medium secretomes 1:1. Concentrate the secreted proteins by using microcon filters with 10 kDa cutoff (Millipore) (capacity 15 mL) until 500 μL. Centrifuge at 4,500 × <i>g</i> for 10 min (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/325-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Concentration and Buffer Exchange of Secreted Proteins</div>
<p>(A) Concentration of the cell culture medium. The solution in the lower part of the Falcon tube is discarded, while secreted proteins, present in the upper part, are concentrated to 500 μL.</p>
<p>(B) Further concentration of the secretome. The solution at the bottom of the tube is discarded, while secreted proteins, present in the upper part of the filter and indicated by the arrow, are concentrated to 30 μL.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> if the volume of the combined secretomes exceeds the microcon capacity, load the sample in steps of 15 mL each until the entire volume is loaded on the filter.</div>
<ol start="14">
<li>Denature proteins by changing the buffer with urea 8 M Tris-HCl 100 mM at pH 8 (urea buffer) as follows. Add urea buffer on the top of the microcon filter, centrifuge at 9,300 × <i>g</i> for 10 min and discard the flow through. Perform at least three washes with 5 mL of urea buffer or until the solution become transparent. Transfer the secreted protein extract (generally about 500 μL) into an Eppendorf tube and sonicate with BIORUPTOR (3 cycles: 30 s on/30 s off).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the sonication step is important to destroy proteins aggregates and to release proteins from extracellular vesicles allowing proteins to be properly solubilized.</div>
<ol start="15">
<li>Quantify proteins concentration by using Bradford assay. <a href="#troubleshooting">Troubleshooting 3</a>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> in our experiments, 2 × 10<sup>6</sup> cells produce more than 30 μg of secreted proteins. This amount is sufficient for the MS analysis.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The secreted protein extract can be stored at −80°C for at least several weeks.</div>
<h3 id="sec3.4">Secretome De-glycosylation and Proteolytic Digestion (Day 2)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes how to reduce and alkylate cysteines of secreted proteins and how to perform proteins de-glycosylation and double digestion.</p>
<ol start="16">
<li>By using smaller (about 400 μL capacity) microcon filters with 10 kDa cutoff (Millipore), reduce the volume of 30 μg of secreted proteins to 30 μL. Add the secreted proteins on the top of the filter (<a href="#fig1">Figure 1</a>B). Centrifuge 10 min at 9,300 × <i>g</i> at 25°C. Discard the flow through. As the secretome will be more than 400 μL, load the secreted proteins in steps of 400 μL each until the entire sample is loaded. Leave the proteins on the filter and perform cysteines reduction and alkylation adding TCEP 10 mM and 2-Chloroacetamide 40 mM in urea buffer on the top of the filter and incubate for 30 min at 25°C. Specifically, add 200 μL of urea buffer to the filter plus 8 μL of 2-Chloroacetamide 1 M and 2 μL of TCEP 1 M. After 30 min, centrifuge 10 min at 9,300 × <i>g</i> at 25°C. Discard the flow through.</li>
<li>Exchange buffer by centrifugation at 9,300 × <i>g</i> for 10 min with 200 μL of buffer G (dilution buffer provided with the enzyme) and add 1.5 μL PNGase F (500 units/μl) for 1 h in 100 μL of buffer G at 25°C following manufacturer’s instruction.</li>
<li>Exchange again buffer by centrifugation at 9,300 × <i>g</i> for 10 min with 300 μL ammonium bicarbonate 50 mM for two times and digest with trypsin (1:50 = enzyme: secreted proteins) dissolved in 100 μL ammonium bicarbonate 50 mM for 12–18 h at 37°C.</li>
<li>Peptides are recovered at the bottom of the microcon filters by centrifugation at 9,300 × <i>g</i> for 10 min and by adding two consecutive washes of 50 μL of NaCl 0.5 M.</li>
<li>Exchange again buffer by centrifugation at 9,300 × <i>g</i> for 10 min with 300 μL phosphate buffer (pH 7.8) for two times and add Glu-C (1:50 = enzyme: secreted proteins) dissolved in 100 μL of phosphate buffer (pH 7.8) on the top of the filters to further digest polypeptides.</li>
<li>Peptides are recovered at the bottom of the microcon filters by centrifugation at 9,300 × <i>g</i> for 10 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> trypsin- and Glu-C-digested peptides are analyzed separately at the MS and pooled together in MaxQuant analysis.</div>
<h3 id="sec3.5">Peptides Purification and LC-MS/MS (Day 4)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days or more (depending on the number of samples to be analyzed)</div>
<p>This step describes how to purify peptides and LC-MS/MS analysis.</p>
<ol start="22">
<li>Eluted peptides are purified on a C18 StageTip (<a href="#bib7">Rappsilber et al., 2007</a>). 3 μg of digested sample are injected onto a quadrupole Orbitrap Q-Exactive HF mass spectrometer (Thermo Scientific).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The peptide samples can be stored at −20°C for a few weeks.</div>
<ol start="23">
<li>Peptides separation is achieved on a linear gradient from 95% solvent A (2% ACN, 0.1% formic acid) to 55% solvent B (80% ACN, 0.1% formic acid) over 75 min (<a href="#fig2">Figure 2</a>) and from 55% to 100% solvent B in 3 min at a constant flow rate of 0.25 μL/min on UHPLC Easy-nLC 1000 (Thermo Scientific) where the LC system is connected to a 23-cm fused-silica emitter of 75 μm inner diameter (New Objective, Inc. Woburn, MA, USA), packed in-house with ReproSil-Pur C18-AQ 1.9 μm beads (Dr Maisch Gmbh, Ammerbuch, Germany) using a high-pressure bomb loader (Proxeon, Odense, Denmark).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/325-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. TIC (Upper Panel) and Base Peak Chromatogram (Lower Panel) of a Typical Secretome MS Analysis</div>
</figcaption>
</figure>
</li>
<li>The mass spectrometer is operated in DDA mode: dynamic exclusion enabled (exclusion duration = 15 s), MS1 resolution = 70,000, MS1 automatic gain control target = 3 × 10<sup>6</sup>, MS1 maximum fill time = 60 ms, MS2 resolution = 17,500, MS2 automatic gain control target = 1 × 10<sup>5</sup>, MS2 maximum fill time = 60 ms, and MS2 normalized collision energy = 25. For each cycle, one full MS1 scan range = 300–1,650 m/z (<a href="#fig2">Figure 2</a>), was followed by 12 MS2 scans using an isolation window of 2.0 m/z.</li>
<li>MS analysis is performed as reported in <a href="#bib4">Matafora et al., 2020</a>. Raw MS files are processed with MaxQuant software, making use of the Andromeda search engine (<a href="#bib1">Cox et al., 2011</a>). MS/MS peak lists are searched against the UniProtKB complete proteome database depending on the samples origin. Searches are performed on both trypsin- and Glu-C-digested peptides analyzed together for each sample by selecting, as fixed modification, alkylation of cysteine by carbamidomethylation, while, as variable modifications, oxidation of methionine, N-terminal acetylation, and N-Deamination. For R and K amino acids, multiplicity is set to 2 and modifications including medium- heavy-labeled R and K are considered. Trypsin and Glu-C specificity are used with up to two missed cleavages allowed. Mass tolerance is set to 5 ppm and 10 ppm for parent and fragment ions, respectively. Andromeda generates a reverse decoy database used to calculate the False Discovery Rate (FDR), which is set to &lt;0.01 for peptide spectrum matches (PSMs). For identification, a minimum of two peptides identified per protein is required, of which at least one has to be unique to the protein group. The minimum ratio count used for quantification is 2 (<a href="#troubleshooting">Troubleshooting 4</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> LC-MS/MS parameters and post data analysis may vary depending on the instruments and software available in each laboratory.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>Generally, from 2 × 10<sup>6</sup> cells as starting material, the amount of secreted proteins recovered is about 30 μg. The amount of recovered proteins strongly depends on the cell type, i.e., proficient secretory cells or non-secretory cells.</p>
<p>With the Secret3D workflow, the MS analysis of the digested proteins usually leads to the identification of roughly 2,000 proteins. We have analyzed more than ten different cell lines and the number of proteins identified is comparable (<a href="#bib4">Matafora et al., 2020</a>).</p>
<p>Here we report as an example a list of the top 50 most abundant proteins identified by Secret3D in IGR37 melanoma metastatic cells (heavy-labeled) versus IGR39 primitive melanoma cells (medium labeled) (<a href="#tbl1">Table 1</a>). Notably, both cell lines are labeled either medium or heavy, therefore the only source of light labeled proteins is the serum. MaxQuant provides the list of proteins identified and quantified in heavy and medium conditions, but also the light condition. By comparing H/L and M/L ratios contaminant proteins coming from serum can be monitored in both samples, while, analyzing the H/M ratios, the IGR37 and IGR39 secretome abundances can be directly compared. For each protein identified, the number of H/M ratio counts is the number of peptide spectrum matches for which heavy/medium pairs were detected; the higher is this number, the better is the quantitation. Moreover, intensity values give an estimation of proteins abundance in each sample, this intensity is the summed intensity of all peptide matches for each protein (modified or not unique peptides are not taken in account).</p>
<table id="tbl1">
<caption>Table 1. Example of Quantitation and Analysis of the Top 50 Proteins Identified by Secret3D in IGR37 Cells (Heavy Labeled) and IGR39cells (Medium Labeled)</caption>
<thead>
<tr>
<th>Protein ID</th>
<th>Protein Name</th>
<th>Gene Name</th>
<th>Intensity Ratio H/M</th>
<th>Count Ratio H/M</th>
<th>Intensity M</th>
<th>Intensity H</th>
</tr>
</thead>
<tbody>
<tr>
<td>P02649</td>
<td>apolipoprotein E</td>
<td>APOE</td>
<td>2.80E+04</td>
<td>178</td>
<td>5.83E+09</td>
<td>6.45E+11</td>
</tr>
<tr>
<td>Q08380</td>
<td>galectin-3-binding protein</td>
<td>LGALS3BP</td>
<td>2.30E+00</td>
<td>53</td>
<td>2.22E+11</td>
<td>2.83E+11</td>
</tr>
<tr>
<td>P07093-2</td>
<td>glia-derived nexin</td>
<td>SERPINE2</td>
<td>2.30E+00</td>
<td>30</td>
<td>1.83E+11</td>
<td>1.91E+11</td>
</tr>
<tr>
<td>P11047</td>
<td>laminin subunit γ-1</td>
<td>LAMC1</td>
<td>1.10E+00</td>
<td>43</td>
<td>2.02E+11</td>
<td>1.38E+11</td>
</tr>
<tr>
<td>P08670</td>
<td>vimentin</td>
<td>VIM</td>
<td>1.10E+00</td>
<td>52</td>
<td>2.63E+11</td>
<td>1.22E+11</td>
</tr>
<tr>
<td>O43852</td>
<td>calumenin</td>
<td>CALU</td>
<td>1.60E+00</td>
<td>48</td>
<td>1.56E+11</td>
<td>1.21E+11</td>
</tr>
<tr>
<td>O60568</td>
<td>procollagen-lysine,2-oxoglutarate 5-dioxygenase 3</td>
<td>PLOD3</td>
<td>3.00E+00</td>
<td>53</td>
<td>8.55E+10</td>
<td>1.16E+11</td>
</tr>
<tr>
<td>C9J2H1</td>
<td>inter-α-trypsin inhibitor heavy chain H5</td>
<td>ITIH5</td>
<td>9.00E+00</td>
<td>27</td>
<td>3.25E+10</td>
<td>1.12E+11</td>
</tr>
<tr>
<td>G3XAI2</td>
<td>laminin subunit β-1</td>
<td>LAMB1</td>
<td>1.20E+00</td>
<td>70</td>
<td>1.48E+11</td>
<td>9.18E+10</td>
</tr>
<tr>
<td>Q96RW7</td>
<td>hemicentin-1</td>
<td>HMCN1</td>
<td>1.70E+00</td>
<td>9</td>
<td>9.24E+10</td>
<td>9.12E+10</td>
</tr>
<tr>
<td>P10451-4</td>
<td>osteopontin</td>
<td>SPP1</td>
<td>7.20E+00</td>
<td>10</td>
<td>3.17E+10</td>
<td>9.08E+10</td>
</tr>
<tr>
<td>P07237</td>
<td>Protein disulfide-isomerase</td>
<td>P4HB</td>
<td>1.20E+00</td>
<td>78</td>
<td>1.55E+11</td>
<td>8.57E+10</td>
</tr>
<tr>
<td>A0A0A0MTC7</td>
<td>laminin subunit α-4</td>
<td>LAMA4</td>
<td>1.00E+00</td>
<td>19</td>
<td>1.50E+11</td>
<td>8.46E+10</td>
</tr>
<tr>
<td>P62805</td>
<td>histone H4</td>
<td>HIST1H4A</td>
<td>3.80E+01</td>
<td>23</td>
<td>3.27E+09</td>
<td>7.89E+10</td>
</tr>
<tr>
<td>P06733</td>
<td>α-enolase</td>
<td>ENO1</td>
<td>1.50E+00</td>
<td>50</td>
<td>1.42E+11</td>
<td>7.85E+10</td>
</tr>
<tr>
<td>P60709</td>
<td>actin, cytoplasmic 1</td>
<td>ACTB</td>
<td>1.10E+00</td>
<td>97</td>
<td>1.96E+11</td>
<td>7.85E+10</td>
</tr>
<tr>
<td>P24821</td>
<td>tenascin</td>
<td>TNC</td>
<td>7.60E-01</td>
<td>61</td>
<td>1.84E+11</td>
<td>7.65E+10</td>
</tr>
<tr>
<td>Q6UVK1</td>
<td>chondroitin sulfate proteoglycan 4</td>
<td>CSPG4</td>
<td>3.10E+00</td>
<td>67</td>
<td>4.17E+10</td>
<td>7.17E+10</td>
</tr>
<tr>
<td>P02787</td>
<td>serotransferrin</td>
<td>TF</td>
<td>1.40E+01</td>
<td>22</td>
<td>7.38E+09</td>
<td>7.16E+10</td>
</tr>
<tr>
<td>U3KQK0</td>
<td>histone H2B</td>
<td>HIST1H2BN</td>
<td>3.40E+01</td>
<td>17</td>
<td>3.16E+09</td>
<td>7.08E+10</td>
</tr>
<tr>
<td>A0A087WTA8</td>
<td>collagen α-2(I) chain</td>
<td>COL1A2</td>
<td>2.70E+01</td>
<td>9</td>
<td>7.01E+09</td>
<td>6.57E+10</td>
</tr>
<tr>
<td>P05067-11</td>
<td>amyloid β A4 protein</td>
<td>APP</td>
<td>1.60E+00</td>
<td>11</td>
<td>5.80E+10</td>
<td>6.44E+10</td>
</tr>
<tr>
<td>P11021</td>
<td>78 kDa glucose-regulated protein</td>
<td>HSPA5</td>
<td>9.40E-01</td>
<td>60</td>
<td>1.29E+11</td>
<td>6.03E+10</td>
</tr>
<tr>
<td>Q16610</td>
<td>extracellular matrix protein 1</td>
<td>ECM1</td>
<td>3.00E+00</td>
<td>10</td>
<td>3.19E+10</td>
<td>5.74E+10</td>
</tr>
<tr>
<td>P30101</td>
<td>protein disulfide-isomerase A3</td>
<td>PDIA3</td>
<td>1.00E+00</td>
<td>65</td>
<td>1.17E+11</td>
<td>5.70E+10</td>
</tr>
<tr>
<td>J3KPS3</td>
<td>fructose-bisphosphate aldolase A</td>
<td>ALDOA</td>
<td>1.20E+00</td>
<td>39</td>
<td>1.12E+11</td>
<td>5.58E+10</td>
</tr>
<tr>
<td>Q8IUX7</td>
<td>adipocyte enhancer-binding protein 1</td>
<td>AEBP1</td>
<td>3.70E+00</td>
<td>11</td>
<td>2.56E+10</td>
<td>5.55E+10</td>
</tr>
<tr>
<td>P00338</td>
<td>L-lactate dehydrogenase A chain</td>
<td>LDHA</td>
<td>8.20E-01</td>
<td>54</td>
<td>1.64E+11</td>
<td>5.45E+10</td>
</tr>
<tr>
<td>P07996</td>
<td>Thrombospondin-1</td>
<td>THBS1</td>
<td>1.70E-01</td>
<td>49</td>
<td>5.50E+11</td>
<td>5.09E+10</td>
</tr>
<tr>
<td>P98160</td>
<td>basement membrane-specific heparan sulfate proteoglycan</td>
<td>HSPG2</td>
<td>1.60E+00</td>
<td>12</td>
<td>5.51E+10</td>
<td>5.01E+10</td>
</tr>
<tr>
<td>D3DQH8</td>
<td>SPARC</td>
<td>SPARC</td>
<td>8.40E-01</td>
<td>16</td>
<td>1.15E+11</td>
<td>4.93E+10</td>
</tr>
<tr>
<td>P01023</td>
<td>α-2-macroglobulin</td>
<td>A2M</td>
<td>3.50E-01</td>
<td>88</td>
<td>2.69E+11</td>
<td>4.87E+10</td>
</tr>
<tr>
<td>P62258</td>
<td>14-3-3 protein epsilon</td>
<td>YWHAE</td>
<td>9.20E-01</td>
<td>28</td>
<td>1.19E+11</td>
<td>4.72E+10</td>
</tr>
<tr>
<td>A0A087X0S5</td>
<td>collagen α-1(VI) chain</td>
<td>COL6A1</td>
<td>3.40E-01</td>
<td>49</td>
<td>2.63E+11</td>
<td>4.38E+10</td>
</tr>
<tr>
<td>A0A0A6YY95</td>
<td>C-type lectin domain family 11 member A</td>
<td>CLEC11A</td>
<td>1.20E+02</td>
<td>5</td>
<td>1.30E+09</td>
<td>4.31E+10</td>
</tr>
<tr>
<td>P26038</td>
<td>moesin</td>
<td>MSN</td>
<td>1.10E+00</td>
<td>34</td>
<td>7.93E+10</td>
<td>4.29E+10</td>
</tr>
<tr>
<td>O94985</td>
<td>calsyntenin-1</td>
<td>CLSTN1</td>
<td>1.10E+00</td>
<td>10</td>
<td>6.47E+10</td>
<td>4.14E+10</td>
</tr>
<tr>
<td>Q16769</td>
<td>glutaminyl-peptide cyclotransferase</td>
<td>QPCT</td>
<td>1.30E+03</td>
<td>47</td>
<td>571420000</td>
<td>4.01E+10</td>
</tr>
<tr>
<td>K7EK07</td>
<td>Histone H3</td>
<td>H3F3B</td>
<td>7.30E+01</td>
<td>12</td>
<td>1.79E+09</td>
<td>3.92E+10</td>
</tr>
<tr>
<td>P18065</td>
<td>insulin-like growth factor-binding protein 2</td>
<td>IGFBP2</td>
<td>1.50E+00</td>
<td>12</td>
<td>5.21E+10</td>
<td>3.89E+10</td>
</tr>
<tr>
<td>P00558</td>
<td>phosphoglycerate kinase 1</td>
<td>PGK1</td>
<td>2.60E+00</td>
<td>30</td>
<td>3.54E+10</td>
<td>3.81E+10</td>
</tr>
<tr>
<td>P07900</td>
<td>heat shock protein HSP 90-alpha</td>
<td>HSP90AA1</td>
<td>9.40E-01</td>
<td>47</td>
<td>1.17E+11</td>
<td>3.76E+10</td>
</tr>
<tr>
<td>P11142</td>
<td>heat shock cognate 71 kDa protein</td>
<td>HSPA8</td>
<td>8.50E-01</td>
<td>56</td>
<td>9.86E+10</td>
<td>3.70E+10</td>
</tr>
<tr>
<td>P13667</td>
<td>protein disulfide-isomerase A4</td>
<td>PDIA4</td>
<td>1.10E+00</td>
<td>40</td>
<td>6.33E+10</td>
<td>3.65E+10</td>
</tr>
<tr>
<td>Q99988</td>
<td>growth/differentiation factor 15</td>
<td>GDF15</td>
<td>1.50E+03</td>
<td>63</td>
<td>1.58E+09</td>
<td>3.41E+10</td>
</tr>
<tr>
<td>O43707</td>
<td>α-actinin-4</td>
<td>ACTN4</td>
<td>5.30E-01</td>
<td>52</td>
<td>1.66E+11</td>
<td>3.41E+10</td>
</tr>
<tr>
<td>P07195</td>
<td>L-lactate dehydrogenase B chain</td>
<td>LDHB</td>
<td>9.70E-01</td>
<td>40</td>
<td>8.36E+10</td>
<td>3.29E+10</td>
</tr>
<tr>
<td>P02545-2</td>
<td>prelamin-A/C</td>
<td>LMNA</td>
<td>3.30E-01</td>
<td>61</td>
<td>1.95E+11</td>
<td>3.11E+10</td>
</tr>
<tr>
<td>Q96JB6</td>
<td>lysyl oxidase homolog 4</td>
<td>LOXL4</td>
<td>1.70E+00</td>
<td>4</td>
<td>2.75E+10</td>
<td>3.03E+10</td>
</tr>
</tbody>
</table>
<p>Proteins identified by Secret3D were verified as belonging to the extracellular space (<a href="#fig3">Figure 3</a>A) by mapping the identified proteins in the subcellular localization database Compartments (<a href="https://compartments.jensenlab.org/Search">https://compartments.jensenlab.org/Search</a>). Moreover, most of the identified proteins were also found in the human cancer secretome database (HCSD) (<a href="#bib2">Feizi et al., 2015</a>), attesting that Secret3D is suitable for secretome analysis (<a href="#fig3">Figure 3</a>B). Further, about one third of the secreted proteins identified were also predicted as secreted or transmembrane proteins in the Human Proteins Atlas database (<a href="http://www.proteinatlas.org">www.proteinatlas.org</a>) (<a href="#fig3">Figure 3</a>C) (<a href="#bib8">Uhlén, et al. 2015</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/325-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Subcellular Component Analysis of Secreted Proteins Identified by Secret3D Workflow</div>
<p>(A) Subcellular localization analysis performed by using the Compartments database.</p>
<p>(B) Venn diagram comparing the secreted proteins identified by Secret3D with the human cancer secretome database.</p>
<p>(C) Venn diagram comparing the secreted proteins identified by Secret3D and predicted secreted and transmembrane proteins in the Human Proteins Atlas Database.</p>
</figcaption>
</figure>
<p>Secreted3D is also able to provide information about the putative N-glycosylation sites of the identified secretome. Modified peptides are about 25%–30% of all identified peptides (<a href="#fig4">Figure 4</a>). As spontaneous deamination of asparagine can occur also without PNGase F participation, the glycosylated sites identified by N-deamidation in MS analysis need further validation (<a href="#bib6">Palmisano et al. 2012</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/325-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Analysis of Asn Deamidated Peptides in IGR37 versus IGR39 Cell Lines</div>
<p>Upper panel: Venn diagram indicating the modified peptides and total peptides identified. Lower panel: MS/MS spectrum of a tenascin glycosylated peptide.</p>
</figcaption>
</figure>
<p>Moreover, Secret3D leads to an increased protein sequence coverage compared to traditional methods, which do not include de-glycosylation and double digestion (<a href="#fig5">Figure 5</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/325-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Sequence Coverage Analysis</div>
<p>Comparison of the sequence coverage of the proteins identified in IGR37 versus IGR39 cell lines by Secret3D or by a method without de-glycosylation and double digestion. t test: ∗∗∗∗p &lt; 0.00001.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and Statistical Analysis</h2>
<p>SILAC ratios from DDA .raw files are calculated by MaxQuant software. The statistical analysis is performed by using Perseus software included in the MaxQuant package. t test, Volcano plot, and ANOVA statistical analyses are performed depending on the experimental design. FDR &lt;0.05 or p-value &lt;0.05 is applied. KEGG enrichment pathway analysis is performed via EnrichR (<a href="http://amp.pharm.mssm.edu/Enrichr">http://amp.pharm.mssm.edu/Enrichr</a>), using the Gene ID of the identified proteins.</p>
<p>As an example, we report here the Volcano plot of the secretome of metastatic (H) versus primitive (L) melanoma cell lines (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/325-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Example of Volcano Plot Representation of Significantly Modulated Proteins (Red Dots)</div>
<p>Intensity Ratio H/L of the proteins identified in metastatic (H) versus primitive (L) melanoma cell lines and p values are reported in the graph. The analysis was performed on three biological replicates. t test analysis, FDR &lt;0.05, was used to select significant proteins.</p>
</figcaption>
</figure>
<p>At least three biological replicates for each condition are required.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Secret3D protocol can be applied only to cell lines that are able to grow in conditioned medium. There are some cell lines that suffer when BSA is removed. Therefore, we recommend verifying cell viability upon conditioning. Viability has to be at least 95%. This protocol is not suited for tissues. This protocol is used to compare secretome from two different samples (medium versus heavy) but can modified for larger experiments including more than two conditions (see paragraph below).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Secretome analysis of more than two conditions or multiple cell lines (step 2).</p>
<h3 id="sec7.2">Potential Solution</h3>
<p>All the cells are heavy labeled, are run separately at the MS, and are treated in MaxQuant as for label free analysis (considering only heavy-labeled proteins).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Cell viability is under 95% (step 5).</p>
<h3 id="sec7.4">Potential Solution</h3>
<p>Reduce the conditioning time, 12 h instead of 18 h.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Cells secrete low amount of proteins (step 15).</p>
<h3 id="sec7.6">Potential Solution</h3>
<p>Start with more cells or increase time of conditioning, checking cell viability.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Secretome contains albumin contamination (step 25).</p>
<h3 id="sec7.8">Potential Solution</h3>
<p>Increase washing before cell starvation.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cox, J., Neuhauser, N., Michalski, A., Scheltema, R.A., Olsen, J.V., and Mann, M. (2011). Andromeda: a peptide search engine integrated into the MaxQuant environment. J. Proteome Res. <i>10</i>, 1794-1805.</p>
<p id="bib2">Feizi, A., Banaei-Esfahani, A., and Nielsen, J. (2015). HCSD: the human cancer secretome database. Database (Oxford) <i>2015</i>, bav051.</p>
<p id="bib3">Harsha, H.C., Molina, H., and Pandey, A. (2008). Quantitative proteomics using stable isotope labeling with amino acids in cell culture. Nat. Protoc. <i>3</i>, 505-516.</p>
<p id="bib4">Matafora, V., Farris, F., Restuccia, U., Tamburri, S., Martano, G., Bernardelli, C., Sofia, A., Pisati, F., Casagrande, F., Lazzari, L., Marsoni, S., Bonoldi, E., and Bachi, A. (2020). Amyloid aggregates accumulate in melanoma metastasis modulating YAP activity. EMBO Rep. <i>21</i>, e50446.</p>
<p id="bib5">Ong, S.E., Blagoev, B., Kratchmarova, I., Kristensen, D.B., Steen, H., Pandey, A., and Mann, M. (2002). Stable isotope labeling by amino acids in cell culture, SILAC, as a simple and accurate approach to expression proteomics. Mol. Cell. Proteomics <i>1</i>, 376-386.</p>
<p id="bib6">Palmisano, G., Melo-Braga, M.N., Engholm-Keller, K., Parker, B.L., and Larsen, M.R. (2012). Chemical deamidation: a common pitfall in large-scale N-linked glycoproteomic mass spectrometry-based analyses. J. Proteome Res. <i>11</i>, 1949-1957.</p>
<p id="bib7">Rappsilber, J., Mann, M., and Ishihama, Y. (2007). Protocol for micro-purification, enrichment, pre-fractionation and storage of peptides for proteomics using StageTips. Nat. Protoc. <i>2</i>, 1896-1906.</p>
<p id="bib8">Uhlén, M., Fagerberg, L., Hallström, B.M., Lindskog, C., Oksvold, P., Mardinoglu, Å., Sivertsson, A., Kampf, C., Sjöstedt, E., Asplund, A., et al. (2015). Tissue-based map of the human proteome. Science <i>347</i>, 1260419.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by Angela Bachi (<a href="mailto:angela.bachi@ifom.eu">angela.bachi@ifom.eu</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate any unique materials or reagents.</p>
<h4>Data and Code Availability</h4>
<p>Proteomic datasets produced in this study are available in the following databases: Proteomics Identification database PeptideAtlas <a href="http://www.peptideatlas.org/">http://www.peptideatlas.org/</a>, code PASS01358.</p>
<h3>Acknowledgments</h3>
<p>This work has been supported by <a href="https://doi.org/10.13039/501100005010">AIRC</a> IG 18607 and IG 14578 to Angela Bachi.</p>
<h3>Author Contributions</h3>
<p>V.M. was responsible for conducting the experiments and manuscript preparation. A.B. provided oversight and insight during the experiments and manuscript preparation.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>