<article>
<section>
<h1 id="header">Protocol for Comprehensive Synthetic Lethality Screens</h1>
<p><time datetime="2020-03-06">Published: March 6, 2020</time></p>
<p>Damia Romero-Moya<sup><a href="#aff1">1</a></sup> and Jeroen P. Roose<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Anatomy, University of California San Francisco (UCSF), 513 Parnassus Avenue, Room HSW-1326, San Francisco, CA 94143-0452, USA</p>
<p id="fn1"><sup>2</sup>Technical Contact</p>
<p id="fn2"><sup>3</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:jeroen.roose@ucsf.edu">jeroen.roose@ucsf.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100016">10.1016/j.xpro.2020.100016</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we provide a detailed protocol for synthetic lethality screens in a Jurkat T cell leukemia line using cell death as the readout measuring the combinatorial effect of a pan-PI3K inhibitor (GDC0941) with specific gene depletion by shRNA. We describe the use of an ultra-complex shRNA library, coverage considerations, time frames, protocol details, and bottlenecks with images to facilitate similar approaches. We discuss how this protocol resource can be readily adapted by investigators.</p>
<p>For complete details on the use and execution of this protocol, please refer to (<a href="#bib1">Mues et al., 2019</a>).</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol for synthetic lethality screens in non-adherent cells</li>
<li>Considerations and background information to consider before starting</li>
<li>Step-by-step protocol details with example images for easy application</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/63-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<p>Here, we describe the use of a non-adherent, human Jurkat T cell leukemia cell line that we can easily expand to 1.6x10<sup>8</sup> cells, which can be efficiently transduced with 1.1x10<sup>8</sup> lentiviral infectious units to achieve a MOI (multiplicity of infection) of 0.7 (In general, a MOI of 0.3 -0.9 is usually recommended for screens).</p>
<ol>
<li>Thaw Jurkat cell line from N2 in a 37°C water bath.</li>
<li>Resuspend cells at 2x10<sup>5 </sup> cells/ml with Jurkat media</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep cell density between 10<sup>5</sup> -10<sup>6</sup> cells/ml, this ensures efficient transduction.</div>
<p><a href="#fig1">Figure 1</a>A</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/63-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Jurkat</div>
<p>(A) Representative image of healthy Jurkat cell line under the microscope.</p>
</figcaption>
</figure>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other cell lines could be used, if the lentiviral infection rate is established before use. Identification and validation of a cell platform that allows you to consider the previous parameters are critical. Can you obtain large numbers of cells? Do these cells die faithfully in your assay? Can you routinely and consistently infect your cells to efficiently introduce your shRNA- or sgRNA- guides? In our Mues <i>et al.</i> study (<a href="#bib1">Mues et al., 2019</a>) we validate one specific synthetic lethality combination in 28 different cancer cell lines, which demonstrated that this Jurkat line is, in principal, suitable to identify synthetic lethality that also impacts other cancer types and adherent cells.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> We describe the use of cell death as a screening tool, but fluorescent reporters can work very well too. You should design an appropriate screening assay to answer your specific biological question.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> We describe the use of FACS-based assays, but microscopy could also be used. Keep in mind that the quality of experimental read-out and its discriminating power is very important and depends strongly on the method selected. For example, we favor FACS-based assays over microscopy because we can apply well-defined gates with FACS. High throughput is also an important consideration.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> In our screen, we used a high coverage shRNA library with 2000X coverage. There are several alternatives available (e.g., either shRNA or more novel sgRNA, such as for CRISPRi or CRISPRa platforms, which can be used to inhibit (CRISPRi) or activate (CRISPRa) gene expression respectively). The key issue in choosing a library is being able to work with high coverage. This avoids population-skewing, over- or under- estimating single target genes, and off-target complication. Many publications use at least 1000X coverage.</div>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>pAkt (Ser473), h</td>
<td>Cell Signaling Tech.</td>
<td>CST #4058</td>
</tr>
<tr>
<td>pS6 (Ser235/236), h</td>
<td>Cell Signaling Tech.</td>
<td>CST #2211</td>
</tr>
<tr>
<td>Annexin V-APC</td>
<td>eBioscience</td>
<td>BMS306APC-20</td>
</tr>
<tr>
<td>PE Donkey Anti-Rabbit IgG</td>
<td>Jackson Immuno</td>
<td>7711-116-152</td>
</tr>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>GDC-0941 (Pictislib)</td>
<td>Selleckchem</td>
<td>S1065</td>
</tr>
<tr>
<td>Sodium Azide (NaN3)</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>calcium chloride (CaCl2 x 2H2O)</td>
<td>Sigma-Aldrich</td>
<td>C8106</td>
</tr>
<tr>
<td>Methanol</td>
<td>Fisher Chemical</td>
<td>A412</td>
</tr>
<tr>
<td>32% PFA</td>
<td>VWR</td>
<td>15714-S</td>
</tr>
<tr>
<td>DMSO</td>
<td>Fisher Chemical</td>
<td>BP321</td>
</tr>
<tr>
<td>2-propanol</td>
<td>Sigma-Aldrich</td>
<td>190764</td>
</tr>
<tr>
<td>Propidium Iodide</td>
<td>Molecular Probes</td>
<td>P3566</td>
</tr>
<tr>
<td>Acridine Orange</td>
<td>Sigma-Aldrich</td>
<td>235474</td>
</tr>
<tr>
<td>Puromycin Dihydrochloride</td>
<td>MP Biomedicals</td>
<td>0210055225</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>QIAamp DNA Blood Maxi Kit</td>
<td>QIAGEN</td>
<td>51192</td>
</tr>
<tr>
<td>Buffer AW1</td>
<td>QIAGEN</td>
<td>19081</td>
</tr>
<tr>
<td>Buffer AW2</td>
<td>QIAGEN</td>
<td>19072</td>
</tr>
<tr>
<td>QIAquick Gel Extraction Kit</td>
<td>QIAGEN</td>
<td>28115</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>QIAGEN</td>
<td>R28104</td>
</tr>
<tr>
<td>PstI-HF</td>
<td>New England Biolabs</td>
<td>R3140T</td>
</tr>
<tr>
<td>HiFi Phusion polymerase</td>
<td>New England Biolabs</td>
<td>M0530L</td>
</tr>
<tr>
<td colspan="3">Experimental Models: Cell Lines</td>
</tr>
<tr>
<td>Human: JURKAT cells</td>
<td>Lab of Jeroen Roose</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>TruSeq Index Primers (The barcoding primers are proprietary to Illumina and were shared with the UCSF <br>Genomics Core with the constraint not to publish the individual sequences.)</td>
<td>Illumina</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>UCSF EXPANDed RNAi library</td>
<td>Lab of Michael McManus, <br>UCSF ViraCore</td>
<td><a href="#bib2">Bassik et al., 2009</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Flow cytometer</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Automatic or manual cell counter</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Water bath</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Centrifuge for 1.5, 15, 50ml tubes and 96 well plates</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Vacuum system with manifold</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Thermocycler</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Agarose/DNA gel station</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Cold room (4°C)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>HyClone™ RPMI 1640 Media with L-Glutamine</td>
<td>GE Healthcare LifeScience</td>
<td>SH30027.02</td>
</tr>
<tr>
<td>1M HEPES, 100 ml</td>
<td>UCSF Cell culture Facility</td>
<td>INVZR929</td>
</tr>
<tr>
<td>1.2ml Microtiter tubes</td>
<td>ThermoFisher Scientific</td>
<td>3492</td>
</tr>
<tr>
<td>123count™ eBeads Counting Beads</td>
<td>Invitrogen</td>
<td>01-1234</td>
</tr>
<tr>
<td>Roller Bottle</td>
<td>Corning</td>
<td>430849</td>
</tr>
<tr>
<td>GeneRuler 1kbP Plus DNA ladder</td>
<td>ThermoFisher Scientific</td>
<td>SM1333</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<h3>Jurkat Media</h3>
<ul>
<li>RPMI with</li>
<li>L-Glutamine</li>
<li>10% FBS</li>
<li>1%P/S</li>
<li>1% HEPES</li>
</ul>
<h3>Drug Reconstitution</h3>
<p>Reconstitute 1mg of GDC0941 at 10mM with 194 μl of DMSO</p>
<p>Store at -20°C for long term storage.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid freeze/thaw cycles</div>
<h3>FACS Buffer</h3>
<ul>
<li>PBS</li>
<li>2mM EDTA</li>
<li>2% FBS</li>
<li>0.09% NaN<sub>3</sub>
</li>
</ul>
<h3>QIAamp DNA Blood Maxi Kit</h3>
<p>Resuspend Protease according to manufacturer’s protocol.</p>
<p>Make 500μl Protease aliquots with H2O and store at -20°C</p>
<div class="note">
<span class="note-title">Note:</span> We recommend you order more wash buffers AW1 and AW2, which will be used for the vacuum protocol.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Titer Drug Concentration</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min (<a href="#fig2">Figure 2</a>)</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/63-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Drug Preparation</div>
<p>(A) Cartoon of serial dilution to prepare 3-fold drug dilution.</p>
</figcaption>
</figure>
<p>This step allows you to determine what drug concentration (GDC0941) to use in your screen.</p>
<ol>
<li>Prepare 3-fold serial dilution (9μM, 3μM, 1μM, 333nM, 111nM, 37nM, 12nM, DMSO) at 2X</li>
</ol>
<ol type="a">
<li>Thaw a small aliquot of GDC0941 while you prepare tubes</li>
<li>Set up 2 tubes with 750μl of Jurkat media and 6 tubes with 500μl of Jurkat media</li>
<li>Add 1.35μl of 10mM GDC0941 to one tube containing 750μl and mix. This will be 9 μM</li>
<li>Transfer 250 μl from 9 μM tube to a tube containing 500 μl and mix. This will be 3 μM.</li>
<li>Transfer 250 μl from 3 μM tube to a tube containing 500 μl and mix. This will be 1 μM.</li>
<li>Repeat previous dilution until 12nM tube.</li>
<li>For DMSO control, add 1.35μl DMSO to the last tube containing 750μl</li>
<li>Transfer 200μl of each inhibitor concentration into two adjacent wells (for duplicates) of a 48 well plate</li>
</ol>
<h3>Analyzing Cell Proliferation with Flow Cytometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2h</div>
<p>This step describes how to obtain the cell counts with flow cytometry with live/dead/apoptotic discrimination</p>
<ol start="2">
<li>Prepare Jurkat cell line.</li>
</ol>
<ol type="a">
<li>Count cells according to your preferred method</li>
<li>Resuspend 2x10 <sup>6</sup>cells at 400x10<sup>3</sup> cells/ml with Jurkat media</li>
<li>Add 200μl of cells to 48 well plate containing inhibitors</li>
<li>Let the cells proliferate in the incubator for 3 days at 37°C</li>
</ol>
<ol start="3">
<li>Set up reagents for 20 samples. Final volume will be 200μl.</li>
</ol>
<ol type="a">
<li>10X CaCl2 in PBS (20mM). <a href="#tbl1">Table 1</a>
</li>
</ol>
<table id="tbl1">
<caption>Table 1. CaCl2 and Acridine Orange Solutions</caption>
<thead>
<tr>
<th>[Stock]</th>
<th> </th>
</tr>
</thead>
<tbody>
<tr>
<td>1M CaCl2</td>
<td>1.47 g CaCl2 x2 H2O in 10ml ddH20</td>
</tr>
<tr>
<td>1mM Acridine Orange (AO)</td>
<td>2.65 mg AO in 10ml ddH2O</td>
</tr>
</tbody>
</table>
<ol start="2" type="a">
<li>10X Acridine Orange (AO, 200nM) and Propidium Iodide (PI, 2μM). Add 20μl of 10μM AO and 20μl of 100μM PI stocks to 1ml PBS. <a href="#tbl1">Table 1</a>
</li>
<li>10X Annexin V–APC in FACS Buffer. 1μl of Annexin V + 19μl of FACS Buffer for each sample.</li>
<li>123count eBeads Counting Beads. Remember manufacturer exact counts (∼10<sup>6</sup>beads/ml)</li>
</ol>
<ol start="4">
<li>Prepare a pre-mix transferring 200μl of each of the reagents (10X CaCl2 in PBS (20mM), 10X AO/PI (200nM/2μM), 10X Annexin-V in FACS Buffer (1:19), eBioscience 123 count Beads) into 1ml tube.</li>
<li>Add 80μl of the previous pre-mix into 16 1.2ml Micro titer tubes</li>
<li>Add 120μl of each well of cell proliferation experiment seeded 3 days ago into previous 1.2ml Micro titer tubes containing the pre-mix. Keep tubes on ice.</li>
<li>Run the samples in a Flow cytometer machine (<a href="#fig3">Figure 3</a>A) with the appropriate Lasers and filters. E.g. AO in FITC, PI in mCherry, Annexin V in APC, Beads in Pacific Blue and AmCyan</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/63-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Flow Cytometry Analysis</div>
<p>(A) Picture of a flow machine suitable for the analysis.</p>
<p>(B) Representative flow cytometry panel of Live/Dead proliferation experiment. Beads are gated on Pacific Blue vs AMcyan. Nucleated cells are gated on Acridine Orange (FITC<sup>+</sup>). Apoptotic cells are gated on Annexin V<sup>+</sup> Propidium Iodide (PI)<sup>-</sup>. Dead cells are gated Propidium Iodide (PI)<sup>+</sup>.</p>
<p>(C) Representative phospho-flow analysis of baseline pAKT (red) and pS6 (green) levels on Jurkat.</p>
</figcaption>
</figure>
<h3>Flow Cytometry Analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> variable</div>
<p>This step describes a proposal of gating strategy in order to reduce variability. (<a href="#fig3">Figure 3</a>B)</p>
<ol start="8">
<li>No gating in FCS/SSC</li>
<li>All events in Pacific Blue and AmCyan. Gate double positive beads</li>
<li>All events in FITC and mCherry. Gate on AO+ (nucleated cells)</li>
<li>AO+ events in mCherry and APC. Make quad gate: live cells (PI-Annexin V-), apoptotic cells (PI- Annexin V+) dead cells (PI+ Annexin V+)</li>
<li>Get counts of each parameter</li>
<li>Cell concentration (cells/ml) = bead concentration x (cell count x bead volume) / (bead count x cell volume). For example, Cell concentration (cells/ml) = ∼10<sup>6</sup> beads/ml x (cell count x 20μl) / (beads count x 120μl)</li>
</ol>
<h3>Signaling Experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes how to analyze the effect of 1h of GDC0941 in PI3K signaling by measuring levels of pAKT and pS6</p>
<ol start="14">
<li>Prepare Drug concentration as above.</li>
<li>Resuspend 8x10<sup>6 </sup>Jurkat cells with 4ml of Jurkat media (2x10<sup>6</sup> cells/ml)</li>
<li>Add 400μl of each GDC0941 concentration (9μM, 3μM, 1μM, 333nM, 111nM, 37nM, 12nM, DMSO) in a different well of a 12 well plate</li>
<li>Add 400μl of Jurkat cells into each well containing GDC0941. Mix well and incubate for 1h at 37°C.</li>
<li>Add 200μl of 10% PFA and incubate at RT for 5 min. Transfer cells to 1.5ml tube</li>
<li>Centrifuge 5 min at 500G, remove supernatant, resuspend cells by flicking.</li>
<li>Add 300μl of 90% ice-cold methanol and incubate at -20°C at least 12-16h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples here can be stored at -20 for up to 2 months.</div>
<ol start="21">
<li>Add 1ml of FACS buffer and incubate 5 min to rehydrate</li>
</ol>
<ol type="a">
<li>Prepare primary antibody</li>
</ol>
<ol type="i">
<li>pAKT S473: 495μl of FACS buffer + 5μl of pAKT antibody</li>
<li>pS6 S235/236: 499μl of FACS buffer + 1μl of pS6 antibody</li>
<li>Control: 500μl of FACS buffer</li>
</ol>
<ol start="22">
<li>Centrifuge cells for 2 min at 2500G, remove supernatant, resuspend cells by flicking</li>
<li>Add 150μl of FACS buffer</li>
<li>Split each condition (50μl) in 3 wells of a U-shaped 96 well plate</li>
<li>Add 150μl of FACS buffer</li>
<li>Centrifuge for 2 min at 2500G, remove supernatant, resuspend cells by flicking</li>
<li>Add 50μl of each antibody mix (pAKT, pS6 and control) to each condition.</li>
<li>Incubate 30 min at RT</li>
</ol>
<ol type="a">
<li>Prepare 2ary antibody</li>
<li>Donkey anti-rabbit-PE: 1386μl of FACS buffer + 14ul of antibody</li>
</ol>
<ol start="29">
<li>Wash cells twice with 200μl of FACS buffer</li>
<li>Add 50μl of 2ary antibody to each well</li>
<li>Incubate 15 min in the darkness at RT</li>
<li>Wash twice with 200μl of FACS buffer at 2500G for 2 min</li>
<li>Resuspend the cells with 150μl of FACS buffer and acquire on a flow cytometer.</li>
<li>Analyze your data e.g. in histograms that depict the levels of pAkt or pS6 (<a href="#fig3">Figure 3</a>C).</li>
</ol>
<h3>Lentiviral Transduction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–12 days</div>
<p>This step describes the infection of Jurkat cell line containing the shRNA library with at least 2000X coverage. The shRNA-encoding vector has mCherry reporter gene and Puromycin selection gene.</p>
<ol start="35">
<li>Jurkat cell line is expanded in regular flasks until desired cell number is achieved</li>
<li>Day -9: shRNA library contains ∼1,800 genes with 55,000 shRNAs. To use a 2000X coverage 1.1x10<sup>8 </sup> lentiviral infectious units are seeded with 1.6x10<sup>8</sup> Jurkat cells to achieve a MOI of 0.7 in 80ml total in a roller bottle (<a href="#fig4">Figure 4</a>A).</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/63-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Jurkats Containing the shRNA Library Are Resistant to Puromycin after Treatment</div>
<p>(A) Picture of a roller bottle in the roller bottle specific incubator.</p>
<p>(B) Representative flow cytometry histogram of Puromycin/mCherry selection from untransduced cells, day -7, day -3 and day 0.</p>
</figcaption>
</figure>
<ol start="37">
<li>Leave cells with lentivirus 12-16h in the incubator.</li>
<li>Day -8: Next morning add 90ml of fresh Jurkat media.</li>
<li>Day -7: Analyze mCherry expression by flow cytometry and start treatment with 2μg/ml of Puromycin (previously tittered: protocol not described). Split cells when necessary adding 2μg/ml of Puromycin keeping cell density between 10<sup>5</sup> and 10 <sup>6</sup> cells/ml to ensure they are in exponential growth.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not use less than 1.1x10<sup>8</sup> cells. Coverage might be compromised</div>
<ol start="40">
<li>Day -3: Analyze mCherry expression by flow cytometry.</li>
<li>Day 0: Analyze mCherry expression by flow cytometry. Cells should be &gt;90% positive. (<a href="#fig4">Figure 4</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If less than 90% of the cells are mCherry positive, continue the puromycin selection.</div>
<ol start="42">
<li>Cryopreserve at least 1.1x10<sup>8</sup> cells at T0.</li>
</ol>
<h3>Cell Growth and Selection Screen</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 weeks</div>
<p>Expand Jurkat cell line with GDC0941 or DMSO for ∼3 weeks to allow for shRNA-induced cell death (synthetic lethality). Note that enrichment for shRNA species can also be picked up.</p>
<ol start="43">
<li>Day 0: Add 1.1x10<sup>8</sup> cells into 2 different roller bottles and fill up to 250ml of fresh Jurkat media with 1μg/ml of Puromycin. One bottle will have GDC0941 at 1.4μM, the concentration at which cell growth is reduced by 20% and phospho-signaling is reduced according to drug titration. 35μl of GDC0941 at 10mM will be added to the bottle. Same amount of DMSO will be added to the control bottle</li>
<li>Day 1: Split the 250ml of each bottle of GDC0941 and DMSO into 2 roller bottles (125 each bottle), to ensure there is not cell saturation. Add 125ml of new fresh media to each bottle with 17.5μl of GDC0941 at 10mM or DMSO and 1μg/ml of Puromycin.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Puromycin can be reduced at 0.5-1μg/ml to maintain the selection</div>
<ol start="45">
<li>Day 3: Count and check cell viability. Split cells down to 5.5x10<sup>7</sup> cells each bottle, add fresh Jurkat media up to 250ml with 1μg/ml of Puromycin and 1.4μM of GDC0941 or DMSO.</li>
<li>Repeat previous step every two or three days according to your cell growth. Keeping them under 2x10<sup>6</sup>cells/ml</li>
<li>On day 22, snap freeze. Count cells, centrifuge 1.2x10<sup>8</sup> cells 5min at 1500rpm, discard supernatant, transfer them into a 5ml cryovial, centrifuge and discard supernatant again. Resuspend cells in remaining liquid and snap freeze at liquid N2 for 1 min before you keep it at -80 for long term storage.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> we chose a 22-day screening period. This is somewhat arbitrary and could be shortened. The reason to allow for a longer selection period here was two-fold. First, we use a very modest concentration of PI3K inhibitor as backdrop. Second, we wanted to be sure to catch synthetic lethality that was not instantly but could take several days.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> You can cryopreserve cells during the splitting days if you would like to preserve material from time points while the screen is ongoing.</div>
<h3>gDNA Extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>This step describes the extraction of gDNA from frozen pellet.</p>
<ol start="48">
<li>Use the QIAamp DNA Blood Maxi Kit. Protocol is slightly modified from the manufacturer. Before to start, warm water bath to 70°C, thaw an aliquot of Protease and set up the vacuum manifold connectors (<a href="#fig5">Figure 5</a>).</li>
</ol>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/63-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Vacuum Manifold for DNA Extraction</div>
<p>(A) Vacuum manifold connected to the vacuum system for DNA extraction.</p>
</figcaption>
</figure>
<ol start="49">
<li>Thaw cell pellet at RT.</li>
<li>Resuspend cells completely when thawed.</li>
<li>Add 8.5ml of PBS and mix while transferring solution to a 50ml tube.</li>
<li>Add 500μl of Protease and mix.</li>
<li>Add 12 ml of buffer AL.</li>
<li>Invert 15 times to mix, then shake vigorously for 1 min. DO NOT VORTEX.</li>
<li>Close tube tightly and incubate 15min at 70°C, shake every 5 min.</li>
<li>Add 10ml of 100% EtOH.</li>
<li>Invert 15 times to mix, then shake vigorously for 1 min. DO NOT VORTEX.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each sample is divided in 2 columns.</div>
<ol start="58">
<li>Add half of volume (∼15ml) to each column. Taking care not moisten the rim.</li>
<li>Open the vacuum ad let the lysate pass through the column.</li>
<li>Wash with 15ml of AW1 – 30-60min.</li>
<li>Wash with 15ml of AW2 – 15-30 min.</li>
<li>Dry each membrane individually opening the valve for 10s.</li>
<li>Wipe rim with dry clean tissue.</li>
<li>Dry all columns for 30 min with vacuum.</li>
<li>Transfer columns into 50ml tubes.</li>
<li>Add 600μl of AE buffer, incubate for 5 min, centrifuge 5min at 3500g.</li>
<li>Add 600μl of AE buffer, incubate for 5 min, centrifuge 5min at 3500g.</li>
<li>Add 400μl of AE buffer, incubate for 5 min, centrifuge 10min at 3500g.</li>
<li>Final volume ∼1100-1200μl.</li>
<li>Transfer elution into a 1.5ml tube.</li>
<li>Determine DNA concentration.</li>
</ol>
<h3>gDNA Digestion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 16–18 h</div>
<p>This step describes the digest of whole gDNA to excise the band of interest.</p>
<ol start="72">
<li>In a 2ml tube add the following:</li>
</ol>
<ul>
<li>1100μl of gDNA.</li>
<li>123μl of CutSmart Buffer.</li>
<li>8μl of PstI-HF restriction enzyme.</li>
</ul>
<ol start="73">
<li>Incubate 12-16h at 37°C in a rotation.</li>
<li>Estimate digestion by running 2μl of digested DNA and 1μl of undigested in a 1.5% agarose gel.</li>
<li>You are expecting a smear on the digested gDNA (<a href="#fig6">Figure 6</a>A).</li>
</ol>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/63-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Illustrated Steps for gDNA Extraction</div>
<p>(A) Representative agarose gel with undigested DNA (left) and PstI-HF digestion 12-16h.</p>
<p>(B) An excised piece of agarose inside 10ml syringe (left) and same piece slurred down (right).</p>
<p>(C) Representative picture of an orange gel extraction buffer (left) and yellow gel extraction buffer (right).</p>
<p>(D) 15ml tube with the end cut (left) and same tube inside columns (right).</p>
<p>(E) 15ml tube with the cut end inside the columns sealed with parafilm and ensembled in the vacuum manifold (left) same but with yellow extraction buffer (right).</p>
</figcaption>
</figure>
<h3>DNA Gel Extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes how to excise the gel portion to extract the size of interest.</p>
<ol start="76">
<li>Prepare a 0.6% agarose gel in a large caster. Make sure the gel has 3 lanes. 1 big lane in the middle which will hold 1400μl and 1 to each side for the ladder.</li>
<li>Once the gel is solidified, transfer it to the cold room (4°C).</li>
<li>Carefully pull out the combs and check for integrity.</li>
<li>Add 130μl of 10X DNA loading dye to digested DNA.</li>
<li>Load along with 1kb ladder and run the gel at 130V <u>in the cold room</u> (4°C) - 1-2h.</li>
<li>Cut band of interest out of the gel. In this protocol is 2.2kb (Between 1.5 and 4kb markers).</li>
<li>Weight the fragment of gel. From this moment we will assume the gel’s weight is 10g.</li>
<li>Place the fragment of the gel in a 10ml syringe.</li>
<li>Push the gel slurry down using the syringe into a 50ml tube (<a href="#fig6">Figure 6</a>B).</li>
<li>Place it at -20°C for at least 12-16h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Sample can be stored at -20°C for longer if desired.</div>
<ol start="86">
<li>Thaw it at RT – 30-60 min.</li>
<li>Use QIAquick Gel extraction kit using vacuum Manifold. Little modifications of the protocol are made.</li>
<li>Add 30 ml of QC buffer to the 50ml tube.</li>
<li>Incubate the tube at 50°C for 5-10 min or until the gel is melted. Mix to accelerate the process.</li>
<li>After the gel has dissolved completely, check that the color of the mixture is yellow. If the mixture is orange or violet add 10 μl of 3M sodium acetate pH 5 until the solution turns yellow (<a href="#fig6">Figure 6</a>C).</li>
<li>Use 2 columns for each sample. Take a new clean 15ml tube cut the bottom part of them with a clean blade. Making a hole. Place the tube inside the column (<a href="#fig6">Figure 6</a>D).</li>
<li>Seal the 15ml tube and the column with parafilm in order to add the solution in fewer steps (<a href="#fig6">Figure 6</a>E).</li>
<li>Add half of the volume (∼20ml) to each 15ml tube (<a href="#fig6">Figure 6</a>E).</li>
<li>Let the solution go through the column using the vacuum.</li>
<li>Remove the 15ml tubes.</li>
<li>Wash twice with 500μl of QG buffer.</li>
<li>Wash 3 times with 800μl of PE buffer.</li>
<li>Vacuum for 5 min to dry the membrane.</li>
<li>Transfer the column to a 2ml collection tube.</li>
<li>Centrifuge at Max speed for 5 min and discard the collection tube.</li>
<li>Place the column to a new 1.5ml low DNA bind tube.</li>
<li>Elute each column twice with 55μl of EB and combine to one tube.</li>
<li>Final volume ∼200μl.</li>
<li>Determine DNA concentration.</li>
<li>Run 2μl of the elution in a 1.5% agarose gel to confirm DNA extraction. A smear is a good sign.</li>
</ol>
<h3>DNA Amplification and Barcoding</h3>
<div class="timing">
<span class="timing-title">Timing:</span> variable</div>
<p>This step describes the amplification of shRNA from the proviral DNA and barcoded for sequencing.</p>
<ol start="106">
<li>DNA concentration is adjusted to 40ng/μl</li>
<li>PCR optimization for temperature and cycles</li>
</ol>
<ol type="a">
<li>Prepare small amount of primers at 10μM for optimization. TrueSeqIndex forward and a common revers primer are used for barcode</li>
<li>PCR are performed using NEB HiFi Phusion polymerase</li>
<li>PCR pre-mix for 5 well to test temperature. Prepare on ice and add in the following order. <a href="#tbl2">Table 2</a>
</li>
</ol>
<table id="tbl2">
<caption>Table 2. PCR Optimization Calculations for Temperature</caption>
<thead>
<tr>
<th> </th>
<th>Volume (μl)</th>
</tr>
</thead>
<tbody>
<tr>
<td>H20</td>
<td>44</td>
</tr>
<tr>
<td>GC buffer</td>
<td>20</td>
</tr>
<tr>
<td>10mM dNTPs</td>
<td>2</td>
</tr>
<tr>
<td>10μM TrueSeqIndex forward</td>
<td>5</td>
</tr>
<tr>
<td>10μM TrueSeq reverse</td>
<td>5</td>
</tr>
<tr>
<td>DMSO</td>
<td>3</td>
</tr>
<tr>
<td>Phusion polymerase</td>
<td>1</td>
</tr>
<tr>
<td>DNA</td>
<td>20 (160ng/reaction)</td>
</tr>
</tbody>
</table>
<ol start="4" type="a">
<li>Split 20μl to 4 different 0.2ml PCR tubes. Pre-heat the thermocycler at 98°C prior to add the tubes. Skip this step once the tubes are in the thermocycler (<a href="#fig7">Figure 7</a>A)</li>
</ol>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/63-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Indexing PCR</div>
<p>(A) Thermocycler with a gradient of temperatures used to choose the appropriate annealing temperature and number of cycles for indexing.</p>
<p>(B) Photo of the PCR cycle used to assess the right temperature of annealing (left &amp; right).</p>
</figcaption>
</figure>
<ol start="5" type="a">
<li>Cycling protocol (<a href="#fig7">Figure 7</a>B)</li>
</ol>
<ul>
<li>Stage 1 - 98°C - Forever</li>
<li>Stage 2 - 98°C – 3 min</li>
<li>Stage 3; step 1 - 98°C – 15s</li>
<li>Stage 3; step 2 - Four gradients of temperatures according to your primers – 30s</li>
<li>Stage 3; step 3 - 72°C – 30 s</li>
<li>Repeat stage 3, 29 times</li>
<li>Stage 4; step 1 - 72°C – 5min</li>
<li>Stage 4; step 2 - 4°C – Forever</li>
</ul>
<ol start="6" type="a">
<li>Run samples at 1% agarose and choose the temperature that show the neat 270bp band</li>
<li>PCR pre-mix for 10 well to test cycles. Prepare on ice and add in the following order. <a href="#tbl4">Table 4</a>
</li>
</ol>
<table id="tbl4">
<caption>Table 4. PCR Optimization Calculations for Cycles</caption>
<thead>
<tr>
<th> </th>
<th>Volume (μl)</th>
</tr>
</thead>
<tbody>
<tr>
<td>H20</td>
<td>88</td>
</tr>
<tr>
<td>GC buffer</td>
<td>40</td>
</tr>
<tr>
<td>10mM dNTPs</td>
<td>4</td>
</tr>
<tr>
<td>10μM TrueSeqIndex forward</td>
<td>10</td>
</tr>
<tr>
<td>10μM TrueSeq reverse</td>
<td>10</td>
</tr>
<tr>
<td>DMSO</td>
<td>6</td>
</tr>
<tr>
<td>Phusion polymerase</td>
<td>2</td>
</tr>
<tr>
<td>DNA</td>
<td>40 (160ng/reaction)</td>
</tr>
</tbody>
</table>
<ol start="8" type="a">
<li>Split 20μl to 9 different 0.2ml PCR tubes. Pre-heat the thermocycler at 98°C prior to add the tubes. Skip this step once the tubes are in the thermocycler</li>
<li>Cycling protocol</li>
</ol>
<ul>
<li>Stage 1 - 98°C – Forever</li>
<li>Stage 2 - 98°C – 3 min</li>
<li>Stage 3; step 1 - 98°C – 15s</li>
<li>Stage 3; step 2 - Previously step chosen temperature – 30s</li>
<li>Stage 3; step 3 - 72°C – 30 s</li>
<li>Repeat stage 3, 30 times</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> Remove from the thermocycles one tube at cycles 10, 12, 14, 16, 18, 20, 22, 24, 28 and put them on ice until the cycle reach 30. Then add all the tubes again to the thermocycles to finish the last elongation step</div>
<ul>
<li>Stage 4; step 1 - 72°C – 5min</li>
<li>Stage 4; step 2 - 4°C – Forever</li>
</ul>
<ol start="108">
<li>Run samples at 1% agarose and choose the cycle that first shows a single 270bp band before higher and lower band form.</li>
<li>Now you are ready to scale up reaction. Multiply previous volumes for 75x. <a href="#tbl3">Table 3</a>
</li>
</ol>
<table id="tbl3">
<caption>Table 3. Scale Up PCR Calculations</caption>
<thead>
<tr>
<th> </th>
<th>Volume (μl)</th>
</tr>
</thead>
<tbody>
<tr>
<td>H20</td>
<td>660</td>
</tr>
<tr>
<td>GC buffer</td>
<td>300</td>
</tr>
<tr>
<td>10mM dNTPs</td>
<td>30</td>
</tr>
<tr>
<td>10μM TrueSeqIndex forward</td>
<td>75</td>
</tr>
<tr>
<td>10μM TrueSeq reverse</td>
<td>75</td>
</tr>
<tr>
<td>DMSO</td>
<td>445</td>
</tr>
<tr>
<td>Phusion polymerase</td>
<td>15</td>
</tr>
<tr>
<td>DNA</td>
<td>300 (160ng/reaction)</td>
</tr>
</tbody>
</table>
<ol start="110">
<li>Divide into twelve 0.2ml tubes (120μl each). Pre-heat the thermocycler at 98°C prior to add the tubes. Skip this step once the tubes are in the thermocycler</li>
<li>ycling protocol</li>
</ol>
<ul>
<li>Stage 1 - 98°C - Forever</li>
<li>Stage 2 - 98°C – 3 min</li>
<li>Stage 3; step 1 - 98°C – 15s</li>
<li>Stage 3; step 2 - Temperature – 30s</li>
<li>Stage 3; step 3 - 72°C – 30 s</li>
<li>Repeat stage 3, “n” times</li>
<li>Stage 4; step 1 - 72°C – 5min</li>
<li>Stage 4; step 2 - 4°C – Forever</li>
</ul>
<ol start="112">
<li>Combine all PCR products from previous 12 tubes and run 15μl in a 1% agarose gel</li>
<li>Confirm there is a neat 270bp band</li>
</ol>
<h3>PCR Product Concentration</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This step reduces the volume of the PCR reaction. QIAquick PCR purification kit is used.</p>
<ol start="114">
<li>Mix 1.5ml of PCR reaction + 7.5ml of Buffer PB</li>
<li>If the color of the mixture is orange or violet add 10μl of 3M sodium acetate pH 5 and mix. Repeat until color of the mix will turn yellow (<a href="#fig6">Figure 6</a>C)</li>
<li>Run each sample into 2 columns. Place a column into a 2ml collection tube</li>
<li>Add 750μl of solution into each column</li>
<li>Centrifuge for 30-60s at Max speed and discard flow-through</li>
<li>Add 750μl of solution again and centrifuge until all solution run through the columns</li>
<li>Wash twice with 1ml of PE buffer.</li>
<li>Centrifuge for 30-60s at Max speed.</li>
<li>Place the column into a new 2ml collection tube and centrifuge for 1 min to remove residual wash buffer.</li>
<li>Place each column in a clean low-DNA binding tube.</li>
<li>Add 55μl EB buffer to each column.</li>
<li>Incubate for 1 min at RT and centrifuge for 1 min at max speed.</li>
<li>Combine elution from both columns. Final volume will be 110μl.</li>
</ol>
<h3>PCR Purification from Agarose Gel</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6h</div>
<p>This step describes the isolation of PCR product from agarose gel.</p>
<div class="note">
<span class="note-title">Note:</span> Different techniques can be used to isolate index DNA band from a gel. Mues et al. (<a href="#bib1">Mues et al., 2019</a>) described a purification using polyacrylamide gel and isolation from a gel with electro-elution. We have since had better success with agarose gel purification, which we describe here.</div>
<ol start="127">
<li>Prepare a 0.6% agarose gel.</li>
<li>Once the gel is solidified, transfer it to the cold room (4°C)</li>
<li>Add 11μl of 10X loading dye to the 110μl eluted previously</li>
<li>Load along with a ladder (i.e. GeneRuler 1kb Plus DNA Ladder) and run the gel at 130V in the cold room (4°C)</li>
<li>Cut band of interest out of the gel. In this protocol it is 270 bp fragment and safe for extraction.</li>
<li>Weigh the fragment of gel. From this moment we will assume the gel weight is 1g.</li>
<li>Use QIAquick Gel extraction kit using microcentrifuge protocol. Slight modifications are made.</li>
<li>Add 3 ml of QC buffer to the 15ml tube.</li>
<li>Incubate the tube at 50°C for 5-10 min or until the gel is melted. Mix to accelerate the process.</li>
<li>After the gel has dissolved completely, check that the color of the mixture is yellow. If mixture is orange or violet add 10 μl of 3M sodium acetate pH 5 until the solution turns yellow (<a href="#fig6">Figure 6</a>C).</li>
<li>Add 1ml of 2-propanol to increase the yield. Only works for DNA fragments &lt;400bp and &gt;4kb.</li>
<li>Place the column into 2ml collection tube.</li>
<li>Add 800μl of the solution and centrifuge for 1 min at max speed.</li>
<li>Discard flow-through and place the column back to the collection tube.</li>
<li>Add 800μl more of the solution and centrifuge for 1 min. Repeat as much as you need.</li>
<li>Wash once with 500μl of QG buffer.</li>
<li>Wash twice with 800μl of PE buffer.</li>
<li>Centrifuge at max speed for 2 min and discard the collection tube.</li>
<li>Place the column to a new 1.5ml low DNA bind tube.</li>
<li>Elute each column twice with 50μl of EB and combine.</li>
<li>Determine DNA concentration.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>This protocol is meant to be used as a reference to design other screenings. We described the detailed methodological steps, drug titration, lentiviral transduction, DNA extraction and amplification to inform the reader what to expect. The “Before you begin” section contains considerations that are important when one aims to make adaptations to the described screen. If the considerations of cell line of choice, rigorous screening assay (e.g. cell death), and higher than 1000x coverage are followed properly, the research can expect to successfully identify synthetic lethality pairs, as we did in our Mues et al. study (<a href="#bib1">Mues et al., 2019</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This specific protocol cannot be adapted to all cell lines. The use of adherent cell lines will need a different approach for expansion of cell numbers, lentiviral transduction, and/or monitoring cell growth. Cell doubling times, confluency of adherent cells, available growth factors, and frequency of drug administration should be considered for each individual cell type of choice.</p>
<p>Library coverage is always a limitation; the higher the level of coverage, the lower the noise to data ratio, and the more confident the generated data sets become. Usually 1000X coverage is assumed as sufficient for experiments using libraries. In our Mues et al. study (<a href="#bib1">Mues et al., 2019</a>) we used 2000X coverage to increase the number of shRNA in the screening and obtain more statistical power. A screen should be designed where aspects like manageable set-up and affordable experimentation are rationally weighed against the highest possible coverage. A popular solution to this dilemma is to perform screen for families of genes, a selection of signaling pathways, or a collection of cell biological processes instead of whole genome screens.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bassik, MC, Lebbink, RJ, Churchman, LS, Ingolia, NT, Patena, W, LeProust, EM, Schuldiner, M, Weissman, JS, McManus, MT, 2009. Rapid creation and quantitative monitoring of high coverage shRNA libraries. Nat Methods <i>6</i>, 443–445.</p>
<p id="bib2">Mues, M., Karra, L., Romero-Moya, D., Wandler, A., Hangauer, M. J., Ksionda, O., Thus, Y., Lindenbergh, M., Shannon, K., Mcmanus, M. T. &amp; Roose, J. P. 2019. High-complexity shRNA libraries and PI3 kinase inhibition in cancer: high-fidelity synthetic lethality predictions. Cell Rep., <i>27</i>, 631-647.e5.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>We thank the members of the Roose lab and particularly Marsilius Mues for excellent record keeping. Research support for our activities came from a <a href="https://doi.org/10.13039/100009858">Gabrielle’s Angel Foundation</a> grant, an <a href="https://doi.org/10.13039/100001445">Alex’ Lemonade Stand Foundation</a> Innovator Award, the NIH/<a href="https://doi.org/10.13039/100000054">NCI</a> (R01 CA187318), and the NIH/<a href="https://doi.org/10.13039/100000050">NHLBI</a> (R01 HL120724) (all to J.P.R.).</p>
<h3>Author Contributions</h3>
<p>Writing – Original Draft, D.R.-M.; Writing – Review &amp; Editing, D.R.-M. and J.P.R.; Funding Acquisition, J.P.R.</p>
<h3>Declaration of Interests</h3>
<p>J.P.R. is a co-founder and scientific advisor of Seal Biosciences, Inc. and on the scientific advisory committee for the Mark Foundation for Cancer research. The other author has no financial interests to declare.</p>
</section>
</article>