<article>
<section>
<h1 id="header">Optimized protocol for gene editing in adipocytes using CRISPR-Cas9 technology</h1>
<p><time datetime="2021-01-29">Published: January 29, 2021</time></p>
<p>Yan Qiu<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> and Qiurong Ding<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>CAS Key Laboratory of Nutrition, Metabolism and Food Safety, Shanghai Institute of Nutrition and Health, Shanghai Institutes for Biological Sciences, University of Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai 200031, P. R. China</p>
<p id="aff2"><sup>2</sup>Institute for Stem Cell and Regeneration, Chinese Academy of Sciences, Beijing 100101, P.R. China</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yqiusibs@hotmail.com">yqiusibs@hotmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:qrding@sibs.ac.cn">qrding@sibs.ac.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100307">10.1016/j.xpro.2021.100307</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We present a detailed protocol for gene editing in adipocytes using the CRISPR-Cas technology. This protocol describes sgRNA design, preparation of lentiCRISPR-sgRNA vectors, functional validation of sgRNAs, preparation of lentiviruses, and lentiviruses transduction in adipocytes. Moreover, an optimized method of gene editing using the lentiCRISPRv2 vector expressing two sgRNAs targeting two different genes has also been described.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib6">Qiu et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Gene editing protocol in adipocytes using CRISPR-Cas technology</li>
<li>Double gene editing in adipocytes using optimized protocol</li>
<li>High-efficiency sgRNA will produce cleavage bands on agarose gel</li>
<li>High-efficiency sgRNA will significantly decrease target protein expression</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/438-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The RNA-guided CRISPR/Cas9 (clustered regularly interspaced short palindromic repeats (CRISPR)-associated protein 9) technology has been widely used in gene editing. CRISPR/Cas9 system consists of the nuclease Cas9 and the guide RNA, which guides Cas9 to cleave DNA and produce DNA double-stranded breaks (DSBs) at a specific genomic locus. The generated DSBs can be repaired via the NHEJ (non-homologous end joining) or HDR (homology directed recombination) pathway, both of which can be adopted to mediate a desired editing outcome.</p>
<p>This protocol describes the use of lentiCRISPR mediated gene knockout in pre-adipocytes, which can be then differentiated into mature adipocytes for functional studies.</p>
<div class="note">
<span class="note-title">Note:</span> This protocol can also be applied to adipocyte cell lines.</div>
<h3 id="sec1.1">sgRNA design and synthesis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>sgRNA design. Input target genomic DNA sequence. sgRNAs can be designed using the online CRISPR Design Tool. For example, <a href="http://tools.genome-engineering.org">http://tools.genome-engineering.org</a> (<a href="#bib7">Ran et al., 2013</a>), Broad Institute GPP (<a href="#bib8">Sanson et al., 2018</a>, <a href="#bib1">Doench et al., 2016</a>), Synthego (<a href="https://design.synthego.com/%23/">https://design.synthego.com/#/</a>), etc.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> sgRNAs can also be designed manually.</div>
<ol type="a">
<li>Obtain the gene coding sequence (CDS) from the NCBI gene database (<a href="https://www.ncbi.nlm.nih.gov">https://www.ncbi.nlm.nih.gov</a>) by searching the gene name in the designated assembly.</li>
<li>Copy the CDS and import the CDS into the UCSC genome browser (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>) to obtain the corresponding genomic sequence.</li>
<li>Select the most common exon among all isoforms to obtain the DNA sequence.
<div class="note">
<span class="note-title">Note:</span> In this protocol, we do not consider the functional difference of different isoforms, thus recommend to select the most common exon among all isoforms for sgRNA design; in case researchers are interested in functional differences between different isoforms, targeting isoform specific exon would be a better choice.</div>
</li>
<li>Single-guide RNA (sgRNA) sequences can be manually selected by identifying the 20-bp sequence directly upstream of any 5′-NGG (protospacer adjacent motif, PAM) or downstream of any 5′-CCN.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended to select sgRNAs with high specificity, which can be assessed by searching in the UCSC genome browser (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>). Typically, design three or more sgRNAs for the first time according to the target gene coding region. Then select sgRNA with the highest editing activity for gene editing in pre-adipocytes.</div>
<ol start="2">
<li>sgRNA synthesis.
<ol type="a">
<li>Design the sgRNA oligonucleotide primers as follows:<br> sgRNA Forward: 5′-CACCGNNNNNNNNNNNNNNNNNNNN-3′, sgRNA Reverse: 5′-AAACNNNNNNNNNNNNNNNNNNNNC-3′.
<div class="note">
<span class="note-title">Note:</span> The highlighted region represents the 20-nt sgRNA sequence; the forward primer is 20-nt upstream of 5′-NGG, and the reverse primer complements the 20 highlighted nucleotides of the forward primer.</div>
</li>
<li>Order the designed sgRNA oligonucleotides.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>lentiCRISPRv2</td>
<td>Addgene</td>
<td>Cat# 52961</td>
</tr>
<tr>
<td>pMDLg/pRRE</td>
<td>Addgene</td>
<td>Cat# 12251</td>
</tr>
<tr>
<td>pRSV-Rev</td>
<td>Addgene</td>
<td>Cat# 12253</td>
</tr>
<tr>
<td>pMD2.G</td>
<td>Addgene</td>
<td>Cat# 12259</td>
</tr>
<tr>
<td>pUC57-MU6-gRNA</td>
<td>Genewiz</td>
<td>n/a</td>
</tr>
<tr>
<td>pCW-Cas9</td>
<td>Addgene</td>
<td>Cat# 50661</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>T7 Endonuclease I</td>
<td>NEB</td>
<td>Cat# M0302L</td>
</tr>
<tr>
<td>BsmBI</td>
<td>NEB</td>
<td>Cat# R0580</td>
</tr>
<tr>
<td>NEbuffer 3.1</td>
<td>NEB</td>
<td>n/a</td>
</tr>
<tr>
<td>NEbuffer 2</td>
<td>NEB</td>
<td>n/a</td>
</tr>
<tr>
<td>BbsI-HF</td>
<td>NEB</td>
<td>Cat# R3539S</td>
</tr>
<tr>
<td>CutSmart</td>
<td>NEB</td>
<td>n/a</td>
</tr>
<tr>
<td>EcoRI</td>
<td>NEB</td>
<td>Cat# R3101S</td>
</tr>
<tr>
<td>KOD buffer</td>
<td>TOYOBO</td>
<td>Cat# KOD-401</td>
</tr>
<tr>
<td>Ligation high</td>
<td>TOYOBO</td>
<td>Cat# LGK-101</td>
</tr>
<tr>
<td>2× Taq Master Mix (Dye)</td>
<td>TransGen Biotech</td>
<td>Cat# CW0682</td>
</tr>
<tr>
<td>Polyethylenimine (PEI)</td>
<td>Polysciences</td>
<td>Cat# 23966-1</td>
</tr>
<tr>
<td>Polybrene</td>
<td>Merck</td>
<td>Cat# TR-1003-50UL</td>
</tr>
<tr>
<td>Gelatin</td>
<td>Santa Cruz</td>
<td>Cat# sc-218567</td>
</tr>
<tr>
<td>Insulin</td>
<td>Santa Cruz</td>
<td>Cat# sc-29062</td>
</tr>
<tr>
<td>T3</td>
<td>Sigma</td>
<td>Cat# T-2877</td>
</tr>
<tr>
<td>Indomethacin</td>
<td>Sigma</td>
<td>Cat# I-7378</td>
</tr>
<tr>
<td>Dexamethasone</td>
<td>Sigma</td>
<td>Cat# D-4902</td>
</tr>
<tr>
<td>IBMX</td>
<td>Sigma</td>
<td>Cat# I-5879</td>
</tr>
<tr>
<td>Rosiglitazone</td>
<td>Santa Cruz</td>
<td>Cat# sc-20279</td>
</tr>
<tr>
<td>DMEM</td>
<td>Gibco</td>
<td>Cat# 8117254</td>
</tr>
<tr>
<td>FBS</td>
<td>Gibco</td>
<td>Cat# 16000044</td>
</tr>
<tr>
<td>Puromycin dihydrochloride</td>
<td>BioVision</td>
<td>Cat# 295160</td>
</tr>
<tr>
<td>RIPA buffer</td>
<td>Millipore</td>
<td>Cat# 20188</td>
</tr>
<tr>
<td>SDS-PAGE gel</td>
<td>Epizyme</td>
<td>Cat# PG112</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Beyotime Biotechnology</td>
<td>Cat# ST007</td>
</tr>
<tr>
<td>DTT</td>
<td>Sangon Biotech</td>
<td>Cat# A100281-0005</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat# 73509461</td>
</tr>
<tr>
<td>SDS</td>
<td>Amresco</td>
<td>Cat# 0227-1KG</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat# 10010618</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>Sangon Biotech</td>
<td>Cat# A500922-0025</td>
</tr>
<tr>
<td>Agarose</td>
<td>Biowest</td>
<td>Cat# BY-R0100</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>OXOID</td>
<td>Cat# LP0021</td>
</tr>
<tr>
<td>Tryptone</td>
<td>OXOID</td>
<td>Cat# LP0042</td>
</tr>
<tr>
<td>NaCl</td>
<td>Sangon Biotech</td>
<td>Cat# 7647-14-5</td>
</tr>
<tr>
<td>Agar powder</td>
<td>Solarbio</td>
<td>Cat# A8190</td>
</tr>
<tr>
<td>Calcium chloride dihydrate</td>
<td>Sigma</td>
<td>Cat# C7902</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>TIANprep Mini Plasmid Kit</td>
<td>TIANGEN</td>
<td>Cat# DP103</td>
</tr>
<tr>
<td>TIANamp Genomic DNA Kit</td>
<td>TIANGEN</td>
<td>Cat# DP304-03</td>
</tr>
<tr>
<td>Gel Extraction Kit</td>
<td>CWBIO</td>
<td>Cat# CW2302M</td>
</tr>
<tr>
<td>qPCR Lentivirus Titration Kit</td>
<td>Abm</td>
<td>Cat# LV900</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>HEK293T</td>
<td>Cell Bank, Type Culture Collection Committee, Chinese Academy of Sciences</td>
<td>n/a</td>
</tr>
<tr>
<td>NIH3T3</td>
<td>Cell Bank, Type Culture Collection Committee, Chinese Academy of Sciences</td>
<td>n/a</td>
</tr>
<tr>
<td>Brown adipocytes</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>White adipocytes</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>0.22-μm vacuum filter</td>
<td>Millipore</td>
<td>Cat# SLGP033RB</td>
</tr>
<tr>
<td>0.45-μm vacuum filter</td>
<td>Millipore</td>
<td>Cat# SLHV033RB</td>
</tr>
<tr>
<td>Beckman JA25.50 rotor</td>
<td>Beckman</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Trans5α Chemically Competent Cell</td>
<td>TransGen</td>
<td>Cat# 431675</td>
</tr>
<tr>
<td>NEB Stable competent cell</td>
<td>NEB</td>
<td>Cat# C3040I</td>
</tr>
<tr>
<td>One Shot StbI3 chemically competent cell</td>
<td>Invitrogen</td>
<td>Cat# C737303</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>sgRNA forward: 5′-CACCGNNNNNNNNNNNNNNNNNNNN-3′</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>sgRNA reverse: 5′-AAACNNNNNNNNNNNNNNNNNNNNC-3′</td>
<td>n/a</td>
<td>n/a</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<ul>
<li>Induction medium for differentiation of brown pre-adipocytes</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume to add</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>n/a</td>
<td>45 mL</td>
</tr>
<tr>
<td>10% FBS</td>
<td>n/a</td>
<td>n/a</td>
<td>5 mL</td>
</tr>
<tr>
<td>Insulin</td>
<td>5 mg/mL</td>
<td>1 μg/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>T3</td>
<td>10 μM</td>
<td>1 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>Indomethacin</td>
<td>0.125 M</td>
<td>0.125 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Dexamethasone</td>
<td>1 mM</td>
<td>1 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>IBMX</td>
<td>0.5 M</td>
<td>0.5 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Rosiglitazone</td>
<td>1 mM</td>
<td>1 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Always prepare freshly before use.</div>
<ul>
<li>Maintenance medium for differentiation of brown pre-adipocytes</li>
</ul>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume to add</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>n/a</td>
<td>45 mL</td>
</tr>
<tr>
<td>10% FBS</td>
<td>n/a</td>
<td>n/a</td>
<td>5 mL</td>
</tr>
<tr>
<td>Insulin</td>
<td>5 mg/mL</td>
<td>1 μg/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>T3</td>
<td>10 μM</td>
<td>1 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Always prepare freshly before use.</div>
<ul>
<li>Preparation of PEI stock</li>
</ul>
<p>Add linear PEI (50 mg) to 50 mL sterile water in a 50-mL conical tube. Place the tube at 55°C in a water bath and vortex every 10 min until PEI is completely dissolved. The PEI solution is cooled to room temperature (∼25°C), filter sterilized using a 0.22-μm vacuum filter and stored at 4°C for up to 5 months or longer.</p>
<ul>
<li>Preparation of LB buffer</li>
</ul>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast extract</td>
<td>5 g</td>
</tr>
<tr>
<td>Tryptone</td>
<td>10 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>10 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p>LB buffer is autoclaved at 121°C for 15 min. Add antibiotics after the LB buffer is cooled adequately. The LB buffer can be stored at room temperature (∼25°C) for up to 1 month or longer.</p>
<div class="note">
<span class="note-title">Note:</span> The use of antibiotics depends on the vectors that used. When culture LentiCRISPRv2-based vector, add ampicillin to the LB buffer before use. The final concentration of ampicillin is 100 μg/mL.</div>
<ul>
<li>Preparation of LB agar plates</li>
</ul>
<p>To prepare LB agar plates, add 15 g agar powder to 1 L LB buffer and autoclave for 15 min at 121°C. After adequate cooling, add antibiotics to the LB buffer, and pour into the petri dishes. Allow to solidify. The LB agar plates can be stored at 4°C for up to 45 days or longer.</p>
<ul>
<li>Preparation of 3× SDS-PAGE loading buffer</li>
</ul>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl (pH 6.8)</td>
<td>7.5 mL</td>
</tr>
<tr>
<td>DTT</td>
<td>2.313 g</td>
</tr>
<tr>
<td>SDS</td>
<td>3 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>15 mL</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>0.015 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p>The 3× SDS-PAGE loading buffer can be stored at room temperature (∼25°C) for up to 1 month or longer.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of lentiCRISPR-sgRNA vector</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> sgRNA can also be delivered by retrovirus or by electroporation. If you already have the established retroviral vector, directly jump to step 6 of the sgRNA screening part for instructions on how to screening sgRNAs with high editing activity. Skip the part on lentiviruses preparation and infection (steps 4–6), if you are preparing to deliver genes via electroporation.</div>
<ol>
<li>Day 1. Part One-Preparation of sgRNA oligonucleotide inserts.
<ol type="a">
<li>Dissolve the sgRNA oligonucleotides in ddH<sub>2</sub>O and dilute to a final concentration of 10 μM. Set up the following reaction for each pair of sgRNA oligonucleotides:
<table id="undtbl5">
<thead>
<tr>
<th>Component</th>
<th>Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>sgRNA-F (10 μM)</td>
<td>1</td>
</tr>
<tr>
<td>sgRNA-R (10 μM)</td>
<td>1</td>
</tr>
<tr>
<td>KOD buffer (10×)</td>
<td>10</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>88</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>100</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The KOD buffer is from the KOD-Plus-Neo kit (TOYOBO, # KOD-401).</div>
</li>
<li>Anneal these sgRNA oligonucleotides at 95°C for 5 min, and cool the mixture to room temperature (∼25°C).</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Day 1. Part Two-Digestion of the lentiCRISPRv2 plasmid.
<div class="note">
<span class="note-title">Note:</span> This step can also be done before step 1.</div>
<ol type="a">
<li>The lentiCRISPRv2 plasmid is obtained as a gift from Feng Zhang (Addgene plasmid #52961; RRID: Addgene_52961): Digest the lentiCRISPRv2 plasmid (1 μg) with 0.5 μL BsmBI enzyme at 55°C for 1 h. Run samples on 1% (wt/vol) agarose gel to verify the appropriately sized fragment (12,994-nt DNA fragment). Successful digestion should yield 12,994-nt and 1,879-nt DNA fragment.
<table id="undtbl6">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>lentiCRISPRv2</td>
<td>1 μg</td>
</tr>
<tr>
<td>BsmBI</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>NEbuffer3.1 (10×)</td>
<td>2 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>to 20 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Day 1. Part Three-Cloning of sgRNA into the lentiCRISPRv2 vector.
<ol type="a">
<li>Set up the ligation reaction for each sgRNA as following:
<table id="undtbl7">
<thead>
<tr>
<th>Component</th>
<th>Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>BsmBI-digested lentiCRISPRv2 from step 2</td>
<td>0.3</td>
</tr>
<tr>
<td>sgRNA from step 1</td>
<td>5.7</td>
</tr>
<tr>
<td>Ligation high</td>
<td>3</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>9 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Incubate the ligation reaction at room temperature (∼25°C) for 2 h.</li>
<li>Transform the ligation reaction into a Trans5α chemically competent cell. Incubate the mixture on ice for 30 min, followed by heat shock at 42°C for 30 s, and immediately re-incubate on ice for 2 min.
<div class="note">
<span class="note-title">Note:</span> It is recommended to transform the lentiviral plasmids in NEB Stable competent cell or One Shot StbI3 chemically competent cell (Invitrogen) to reduce potential homologous recombination.</div>
</li>
<li>Add 100 μL LB buffer to the ligation mixture from step 3c, then incubate the mixture at 37°C for 1 h in a shaker.</li>
<li>Add the mixture from step 3d onto an LB plate with 100 μg/mL ampicillin. Incubate the LB plate at 37°C for 12–16 h.
<div class="critical">
<span class="critical-title">Critical:</span> Prepare a negative control plate (ligation of BsmBI-digested lentiCRISPRv2 alone without annealed sgRNA oligo insert).</div>
</li>
<li>Day 2: Assess the plate for colony growth. Typically, multiple colonies should grow on the lentiCRISPR-sgRNA (sgRNA inserted into lentiCRISPRv2 plasmid) plate, and the negative control plate should not have colony growth.</li>
<li>Select two or three colonies from the lentiCRISPR-sgRNA plate to confirm the insertion of sgRNA into lentiCRISPRv2 vector. Grow each single colony in 3 mL LB buffer with 100 μg/mL ampicillin and shake the LB buffer at 37°C for 6 h.
<div class="note">
<span class="note-title">Note:</span> The time for incubation is flexible.</div>
</li>
<li>Isolate plasmid DNA from the bacterial cultures using a TIANprep Mini Plasmid Kit, according to the manufacturer’s instructions (<a href="https://www.tiangen.com/asset/imsupload/up0062501001604555834.pdf">https://www.tiangen.com/asset/imsupload/up0062501001604555834.pdf</a>).</li>
<li>Day 3. Verify the CRISPR vector, designated as lentiCRISPR-sgRNA1, by sequencing. Sequence each colony from the U6 promoter using the U6 primer. The 20-bp sgRNA sequence should be inserted between the U6 promoter and the remainder of the sgRNA scaffold. Further details and sequence of the lentiCRISPRv2 vector can be found at <a href="http://n2t.net/addgene:52961">http://n2t.net/addgene:52961</a>.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Screening sgRNAs with high editing activity</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 11 days</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to screen for sgRNAs with high editing activity in HEK293T cells (for gene editing in human cells) and NIH3T3 cells (for gene editing in mouse cells).</div>
<ol start="4">
<li>Preparation of lentiCRISPR-sgRNA virus
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<ol type="a">
<li>HEK293T cells culture. Culture cells in DMEM supplemented with 10% FBS at 37°C and 5% CO<sub>2</sub>.</li>
<li>Day 1. Prepare cells for transfection. Plate HEK293T cells onto 10-cm plates in DMEM supplemented with 10% FBS 16–24 h prior to transfection. Typically, plate ∼4 × 10<sup>6</sup> cells in a 10-cm plate (∼5 × 10<sup>4</sup> cells/cm<sup>2</sup>).
<div class="note">
<span class="note-title">Note:</span> It is recommended to transfect the plasmid mix when cells are 80%–90% confluent.</div>
</li>
<li>Day 2. On the day of transfection, replace the growth medium with 6–8 mL DMEM 4–6 h before transfection. Prepare polyethylenimine (PEI) transfection reagents for each 10-cm plate as following:
<div class="critical">
<span class="critical-title">Critical:</span> It is important to perform this step gently and carefully because HEK293T cells can be easily detached. It is also recommended to pre-coat cell culture plates with poly-L-lysine or collagen to prevent cell detachment.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> PEI-based transfection can be replaced with calcium phosphate or lipofection-based approaches.</div>
<ol type="i">
<li>Mix the lentiviral plasmid with the packaging vectors (total ∼25 μg plasmid DNA), and subsequently dilute the mixture in 300 μL DMEM by pipetting up and down for a few times.
<table id="undtbl8">
<thead>
<tr>
<th>Component</th>
<th>Amount (μg)</th>
</tr>
</thead>
<tbody>
<tr>
<td>pMDLg/pRRE</td>
<td>7.35</td>
</tr>
<tr>
<td>pMD2.G</td>
<td>3.95</td>
</tr>
<tr>
<td>pRSV-Rev</td>
<td>2.85</td>
</tr>
<tr>
<td>lentiCRISPR-sgRNA</td>
<td>11.25</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>25.4</b></td>
</tr>
</tbody>
</table>
</li>
<li>Mix the PEI stock gently before use. Thereafter, add 75 μL PEI (the ratio of PEI volume (μL) to the amount of total DNA (μg) is 1:3) to 300 μL DMEM and mix gently by pipetting up and down for a few times.</li>
<li>Add the PEI/DMEM solution (375 μL) to the diluted DNA/DMEM mixture and mix immediately by pipetting up and down for a few times or vortexing.</li>
<li>Incubate the mixture at room temperature (∼25°C) for 15 min, and add the mixture to each plate with gentle rocking for 2 or 3 times.</li>
<li>After 4–6 h, replace DMEM with growth medium.</li>
<li>Days 4 and 5. Collect the virus-containing medium 48 h and 72 h after transfection. Thereafter, filter the medium with a 0.45 μm vacuum filter, and store the virus-containing medium at 4°C for immediate use. For longer term storage at −80°C, the virus-containing medium can be concentrated by centrifugation with a Beckman JA25.50 rotor (capacity for eight tubes) at 48,384 × <i>g</i> for 2 h at 4°C.
<div class="note">
<span class="note-title">Note:</span> Lentivirus can be dissolved in cold PBS or DMEM. Usually, for one 50 mL tube, it is recommended to dissolve the virus with 200 μL cold PBS or DMEM. For more detailed information about the production and purification of lentiviruses, please refer to (<a href="#bib9">Tiscornia et al., 2006</a>, <a href="#bib2">Dull et al., 1998</a>).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Lentivirus can also be concentrated via polyethylene glycol (PEG)-based system (<a href="#bib4">Larcombe et al., 2019</a>). Troubleshooting 1 (Low titers of lentiviruses)</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="5">
<li>Lentivirus titration (: Titer the concentrated lentivirus with a commercial qPCR Lentivirus Titration Kit, according to the manufacturer’s instructions (<a href="https://www.abmgood.com/pub/media/productdocument/document//l/v/lv900_datasheet_v18.pdf">https://www.abmgood.com/pub/media/productdocument/document//l/v/lv900_datasheet_v18.pdf</a>). Typically, the lentivirus titer of a 10-cm plate is ∼1 × 10<sup>7</sup> pfu.</li>
</ol>
<ol start="6">
<li>Lentiviruses transduction in NIH3T3 cells
<div class="timing">
<span class="timing-title">Timing:</span> 6 days</div>
<ol type="a">
<li>Day 1. Plate NIH3T3 cells in a 12-well plate 1 day before transduction.</li>
<li>Day 2. Infect cells with lentiviruses at a MOI of ∼30 and allow lentivirus-infected cells to grow for another 2 days.</li>
<li>Day 4. Treat cells with 3.5 μg/μL puromycin for 2 days to remove non-infected cells.</li>
<li>Day 6. Collect cells for genomic DNA extraction.
<div class="note">
<span class="note-title">Note:</span> Polybrene can be added to increase transduction efficiency.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not forget to set up control groups (cells infected with lentiviruses that carry no sgRNA or express only GFP). At least include one of them as a control, and it is recommended to include both as controls.</div>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Surveyor assay to validate sgRNA function
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<div class="note">
<span class="note-title">Note:</span> The Cas9 and sgRNA in cells can cause genomic insertion or deletion (indel) mutations, which can be detected via either the surveyor assay or sequencing. The surveyor primers can be designed via the online Primer Design Tool. For example, Primer 3.0 Web (<a href="http://primer3.ut.ee">http://primer3.ut.ee</a>).</div>
<ol type="a">
<li>Extract genomic DNA from different group cells using the TIANamp Genomic DNA Kit, according to the manufacturers’ instruction (<a href="https://www.tiangen.com/asset/imsupload/up0484377001604554551.pdf">https://www.tiangen.com/asset/imsupload/up0484377001604554551.pdf</a>).</li>
<li>Set up the PCR reaction as follows:
<table id="undtbl9">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Surveyor-F (10 μM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Surveyor-R (10 μM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>2× Taq</td>
<td>50 μL</td>
</tr>
<tr>
<td>Genomic DNA</td>
<td>500 ng</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>to 100 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>100 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Design PCR primers for amplification (Surveyor-F/R) of the target region with the predicted SpCas9 cut site in the center. It is recommended to design a primer pair that will produce a ∼600 bp band.</div>
Then perform PCR using the following cycling conditions:
<table id="undtbl10">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
</thead>
<tbody>
<tr>
<td>Steps</td>
<td>Temperature</td>
<td>Time</td>
<td>Cycles</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>94°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">25–35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>58°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>forever</td>
<td> </td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The time for extension is mainly determined by the length of DNA product and the DNA polymerase used.</div>
</li>
<li>Run the PCR products on a 1% (wt/vol) agarose gel. Purify the PCR product using a Gel Extraction Kit, according to the manufacturers’ instruction (<a href="https://www.cwbiotech.com/uploads/websitepdf/216c4037-3eae-4ac2-b86a-762277a7adc1.pdf">https://www.cwbiotech.com/uploads/websitepdf/216c4037-3eae-4ac2-b86a-762277a7adc1.pdf</a>).</li>
<li>Anneal the PCR products as follows:
<table id="undtbl11">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNA product from step 7c</td>
<td>250 ng</td>
</tr>
<tr>
<td>NEbuffer 2 (10×)</td>
<td>2 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>to 20 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Put the mixture at 95°C for 5 min.</div>
<div class="note">
<span class="note-title">Note:</span> Take equal amounts of DNA product from different lentiviruses-treated cells (lentiviruses carrying sgRNA or no sgRNA) for analysis.</div>
</li>
<li>Digest the annealed DNA fragments with 0.5 μL T7 Endonuclease I (T7EI) enzyme at 37°C for 1 h.</li>
<li>Run samples on a 1% (wt/vol) agarose gel.</li>
<li>Determine the cleavage efficiency by quantifying gel bands without overexposing them. A high-efficiency sgRNA will result in obvious cleavage bands, whereas the negative controls should have no cleavage bands. Troubleshooting 4 (No obvious cleavage bands after T7 endonuclease I digestion).</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Optional steps</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<ol start="8">
<li>Two sgRNAs can be cloned into one lentiCRISPRv2 vector. The lentiCRISPRv2 vector can be modified to target two different genes simultaneously.
<ol type="a">
<li>Day 1. Prepare the following reaction mixture to digest 1 μg of pUC57-MU6-gRNA plasmid DNA:
<table id="undtbl12">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>pUC57-MU6-gRNA</td>
<td>1 μg</td>
</tr>
<tr>
<td>BbsI-HF</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>CutSmart (10×)</td>
<td>2 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>to 20 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
Digest the pUC57-MU6-gRNA plasmid with 0.5 μL BbsI enzyme at 37°C for 2.5 h and run the product on a 1% (wt/vol) agarose gel to verify successful fragment.
<div class="note">
<span class="note-title">Note:</span> The pUC57-MU6-gRNA plasmid can also be digested with BbsI enzyme at 37°C for 8–12 h.</div>
</li>
<li>Days 1–3. Clone another sgRNA into the pUC57-MU6-gRNA plasmid, which is called pUC57-sgRNA2. The protocol is identical to that in step 3.</li>
<li>Day 3. Part One. Digest the pUC57-sgRNA2 vector with EcoRI enzyme at 37°C for 1 h. Run samples on a 1% (wt/vol) agarose gel to verify successful fragment (the 430-bp-long product), which is designated as Mouse U6-sgRNA2. Successful reactions should yield a 430-bp fragment and a 2,713-bp fragment.</li>
<li>Day 3. Part Two. Digest the lentiCRISPR-sgRNA1 vector (from step 3) with EcoRI enzyme at 37°C for 1 h. Run samples on a 1% (wt/vol) agarose gel to verify successful fragment.</li>
<li>Day 3. Part Three. Ligate the Mouse U6-sgRNA2 fragment from step 8c into the digested lentiCRISPR-sgRNA1 from step 8d. Set up the ligation reaction as follows:
<table id="undtbl13">
<thead>
<tr>
<th>Component</th>
<th>Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>lentiCRISPR-sgRNA1 from step 8d</td>
<td>0.3</td>
</tr>
<tr>
<td>Mouse U6-sgRNA2 fragment from step 8c</td>
<td>5.7</td>
</tr>
<tr>
<td>Ligation high</td>
<td>3</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>9 μL</b></td>
</tr>
</tbody>
</table>
Days 3–5. All other steps are identical to step 3.
<div class="note">
<span class="note-title">Note:</span> It can also be achieved by delivering two sgRNAs from two separate viruses when using higher MOI for infection to simultaneously knockout two genes in one cell.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Lentiviral transfection of adipocytes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days + differentiation</div>
<ol start="9">
<li>Pre-adipocytes culture and transduction
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<div class="note">
<span class="note-title">Note:</span> This protocol does not consider the effect of a specific gene on adipocytes differentiation. To exclude possible effect of a gene during adipocyte differentiation, we recommend the use of inducible CRISPR/Cas9 system, for example, the pCW-Cas9 (Addgene, #50661), to start gene editing after adipocyte differentiation in mature adipocytes. Infect pre-adipocytes with the inducible CRISPR/Cas9 system to introduce the Cas9 and sgRNA cassettes to cells, and induce the Cas9 expression with doxycycline after differentiation to knockout certain genes of interest.</div>
<ol type="a">
<li>Maintain brown pre-adipocytes and white pre-adipocytes in DMEM supplemented with 20% FBS at 37°C and 5% CO<sub>2</sub>. Passage the cells at 1:6 or more every 2 days.
<div class="note">
<span class="note-title">Note:</span> Plates can be pre-coated with 0.1% gelatin to prevent cell detachment. The volume of 0.1% gelatin added is as follows: 12-well plate, 1 mL/well, 6-well plate, 1.5 mL/well, and 10-cm culture dish, 6 mL/dish.</div>
</li>
<li>Plate pre-adipocytes in 6-well plates for lentiviral infection. After approximately 8 h, add lentiviruses (carry sgRNA) to cells (day 1). Add ∼6 × 10<sup>5</sup> viral particles to 2 × 10<sup>5</sup> cells, which typically resulted in 20%–30% of cells infected (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/438-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Lentiviral infection and puromycin treatment</div>
<p>Representative images show the density variation of brown pre-adipocytes after lentiviral infection from day 1 to day 3 (A); or after puromycin treatment from day 4 to day 6 (B). The lentiCRISPRv2 vector contains the puromycin cassette while the lenti-GFP vector does not. Thus, cells infected with lentiCRISPRv2 viruses are resistant to puromycin treatment, and cells infected with lenti-GFP viruses are puromycin-irresistant. Scale bar, 200 μm.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended to set up control groups (cells infected with lentiviruses that carry no sgRNA or express GFP).</div>
<div class="note">
<span class="note-title">Note:</span> Polybrene (8 μg/mL) can be added to increase transduction efficiency.</div>
</li>
<li>After 48 h of transduction, treat cells with 1.5 μg/μL puromycin for 2 days to remove non-infected cells (<a href="#fig1">Figure 1</a>B).</li>
<li>After 2 days of puromycin treatment, collect cells and plate cells in a new 12-well plate for differentiation. Troubleshooting 2 (The number of infected pre-adipocytes might be variable, thereby resulting in experimental variability)
<div class="note">
<span class="note-title">Note:</span> It is recommended to allow these infected cells to grow in growth medium for one more day before differentiation.</div>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Adipocyte differentiation.
<div class="note">
<span class="note-title">Note:</span> The protocol for the differentiation of white pre-adipocytes is similar to that for the differentiation of brown pre-adipocytes, with the exception that no T3 and 5 μg/mL insulin is used during the whole differentiation process (<a href="#bib5">Qiu et al., 2018</a>, <a href="#bib3">Feng et al., 2019</a>). Thus, this protocol mainly describes the brown pre-adipocytes differentiation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare all reagents for adipocyte differentiation before use. Aliquot stock solutions into small volumes. Avoid repeatedly freezing and thawing.</div>
<ol type="a">
<li>When brown pre-adipocytes grow confluent (termed “day 0”), remove the growth medium and add induction medium to induce adipogenesis.
<div class="note">
<span class="note-title">Note:</span> The induction medium should be freshly prepared before use.</div>
</li>
<li>On day 3, remove the induction medium and add the maintenance medium.
<div class="note">
<span class="note-title">Note:</span> Do not wash cells with PBS. The maintenance medium should be freshly prepared before use.</div>
</li>
<li>Change the maintenance medium every 2 days. On day 6 or 7, Typically more than 90% of cells are differentiated. Troubleshooting 3 (Low differentiation potential of pre-adipocytes)
<div class="note">
<span class="note-title">Note:</span> Change the maintenance medium daily if it appears yellow.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Functional validation of sgRNAs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<ol start="11">
<li>Functional testing for the detection of indel mutations by surveyor assay (Timing: 1 day): Confirm the function of sgRNA in adipocytes by detecting the indel mutations generated by specific sgRNAs. The protocol is identical to that in step 7. Troubleshooting 4 (No obvious cleavage bands after T7 endonuclease I digestion)</li>
</ol>
<ol start="12">
<li>Functional testing to determine protein expression
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol type="a">
<li>Collect the well-differentiated adipocytes in 1.5 mL tube. For one well of 12-well plate, add 60 μL RIPA buffer into the tube, mix gently by pipetting up and down for a few times, and put the mixture on ice for 15 min.
<div class="note">
<span class="note-title">Note:</span> The volume of RIPA buffer can be flexible. For one well of 12-well plate, it is recommended to add 60 μL RIPA buffer to lysis cells.</div>
</li>
<li>Centrifuge 15 min at 15,294 × <i>g</i>, and transfer the lysis buffer (middle layer) to a new 1.5 mL tube (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/438-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic view of the layers after centrifugation (step 12b)</div>
<p>The upper layer is lipid, the middle layer is the lysis buffer, and the lower layer is the cell debris.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> This step can be repeated to remove lipids. Troubleshooting 5 (Lipid droplets in mature adipocytes reduce protein extraction efficiency)</div>
</li>
<li>Add 3× SDS-PAGE loading buffer to each tube, and incubate the mixture at 95°C for 5 min.
<div class="note">
<span class="note-title">Note:</span> The volume of 3× SDS-PAGE loading buffer depends on the volume of RIPA buffer in step 12a. For 60 μL RIPA buffer, add 30 μL 3× SDS-PAGE loading buffer.</div>
</li>
<li>Place the solution on ice for immediate use or store the solution at −20°C or −80°C for future use.</li>
<li>Run the protein samples on a SDS-PAGE gel, and detect the target protein expression using corresponding antibodies.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>When running the DNA samples on 1% (wt/vol) agarose gel, high-efficiency sgRNA will generate obvious cleavage bands, whereas the negative controls will have no cleavage bands (<a href="#fig3">Figure 3</a>). When running protein samples on SDS-PAGE gel, high-efficiency sgRNA will result in significantly decreased target protein expression.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/438-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Surveyor assay and western blot results illustrating the editing activity of selected sgRNAs</div>
<p>(A) Assessment of sgRNA cleavage efficiency by surveyor assay. sgRNA that results in obvious cleavage bands indicates high-efficiency sgRNA, such as sgRNA1 and 2 showed here. sgRNA that hardly generates any cleavage bands indicate low-efficiency sgRNA, such as sgRNA3 showed here. The sgRNA efficiency is quantified by ImageJ. (B) Western blot confirming the reduction in protein levels resulted by selected sgRNA. Figure reprinted with permission from <a href="#bib6">Qiu et al. (2020</a>).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>In this protocol, as gene editing is performed in a group of cells, it usually results in a knockdown effect, as in some cells with a certain percentage, CRISPR editing results in indels with multiples of 3 nt, thus resulting in deletion or insertion of several amino acids, but not frameshift or early protein truncation. In order to ensure a knockout effect, single clones must be isolated with sequencing confirmed with a frameshift indel mutation.</p>
<p>Another limitation of this protocol is that genes are edited in pre-adipocytes but not directly in mature adipocytes. It is possible that knockout of certain gene may have functional consequences to adipocyte differentiation. If researchers want to exclude possible effect of a gene during adipocyte differentiation, we recommend using inducible CRISPR/Cas9 system, for example, the pCW-Cas9 (Addgene, #50661), to start gene editing after adipocyte differentiation in mature adipocytes. This works by infecting pre-adipocytes with the inducible CRISPR/Cas9 system to introduce the Cas9 and sgRNA cassettes to cells, and induce the Cas9 expression with doxycycline after differentiation to knockout certain genes of interest.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low titers of lentiviruses (step 4).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>This might occur due to multiple reasons: first, poor HEK293T cell health. It is recommended to use low-passage cells. We usually discard cells that have been passaged for more than 15 times.</p>
<p>Second, low transfection efficiency of plasmids. Check the quality of transfection reagents or change to other more-efficient transfection reagents. Improper cell density (too low or too high) also results in low transfection efficiency.</p>
<p>Third, the time for transfection. According to our experience, transfect plasmids 16–24 h after seeding will improve lentiviruses titers.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The number of infected pre-adipocytes might be variable (step 9d), thereby resulting in experimental variability.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This might occur due to multiple reasons. The viral titer might not be accurate. Therefore, titer the lentiviral samples with a commercial qPCR Lentivirus Titration Kit before use.</p>
<p>If the target gene affects cell growth, the density of pre-adipocytes for different groups may be different. To determine this, seed the same number of cells for different groups in a 12-well plate to induce differentiation.</p>
<p>The concentration of puromycin might be different in each sample. To avoid this, prepare a puromycin stock solution and then make diluted samples to ensure homogeneity. Avoid addition of puromycin individually.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Low differentiation potential of pre-adipocytes (step 10).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>This might occur due to multiple reasons. First, poor cell health can alter growth rate or morphology. Cells that have more than 15 passages should not be used. A higher number of cell divisions might decrease the differentiation capacity.</p>
<p>Second, the activity of compounds used for differentiation is low. Aliquot to avoid repeated freezing and thawing.</p>
<p>Third, improper cell density. Cell density in a too low or too high level may affect differentiation.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>No obvious cleavage bands after T7 endonuclease I digestion (step 7g and step 11).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This might occur due to low editing activity of sgRNA. Therefore, it is recommended to design at least three sgRNAs and screen for sgRNAs with high editing activity. The sgRNA with the highest editing activity should be selected to transfect cells.</p>
<p>Second, low activity of T7EI can cause incomplete digestion. The editing activity of sgRNA can also be determined by sequencing the PCR product generated using surveyor primers.</p>
<p>Third, the time required for T7EI digestion might be short. Therefore, the PCR products can be incubated with T7EI enzyme for a longer duration to achieve complete digestion.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Lipid droplets in mature adipocytes reduce protein extraction efficiency (step 12b).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The lipid droplet content is positively correlated with the maturity of adipocytes. Remove the lipid droplets after addition of RIPA buffer to cells. Moreover, it is recommended to repeat this step until no obvious lipid droplets are observed.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Doench, J.G., Fusi, N., Sullender, M., Hegde, M., Vaimberg, E.W., Donovan, K.F., Smith, I., Tothova, Z., Wilen, C., Orchard, R., et al. (2016). Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nat. Biotechnol. <i>34</i>, 184-191. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref1">View at publisher</a></p>
<p id="bib2">Dull, T., Zufferey, R., Kelly, M., Mandel, R.J., Nguyen, M., Trono, D., and Naldini, L. (1998). A third-generation lentivirus vector with a conditional packaging system. J. Virol. <i>72</i>, 8463-8471. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref2">View at publisher</a></p>
<p id="bib3">Feng, Z., Wei, Y., Zhang, Y., Qiu, Y., Liu, X., Su, L., Liang, N., Yin, H., and Ding, Q. (2019). Identification of a rhodanine derivative BML-260 as a potent stimulator of UCP1 expression. Theranostics <i>9</i>, 3501-3514. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref3">View at publisher</a></p>
<p id="bib4">Larcombe, M.R., Manent, J., Chen, J., Mishra, K., Liu, X., and Nefzger, C.M. (2019). Production of high-titer lentiviral particles for stable genetic modification of mammalian cells. Methods Mol. Biol. <i>1940</i>, 47-61. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref4">View at publisher</a></p>
<p id="bib5">Qiu, Y., Sun, Y., Xu, D., Yang, Y., Liu, X., Wei, Y., Chen, Y., Feng, Z., Li, S., Reyad-Ul Ferdous, M., et al. (2018). Screening of FDA-approved drugs identifies sutent as a modulator of UCP1 expression in brown adipose tissue. EBioMedicine <i>37</i>, 344-355. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref5">View at publisher</a></p>
<p id="bib6">Qiu, Y., Yang, Y., Wei, Y., Liu, X., Feng, Z., Zeng, X., Chen, Y., Liu, Y., Zhao, Y., Chen, L., et al. (2020). Glyburide regulates UCP1 expression in adipocytes independent of KATP channel blockade. iScience <i>23</i>, 101446. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref6">View at publisher</a></p>
<p id="bib7">Ran, F.A., Hsu, P.D., Wright, J., Agarwala, V., Scott, D.A., and Zhang, F. (2013). Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. <i>8</i>, 2281-2308. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref7">View at publisher</a></p>
<p id="bib8">Sanson, K.R., Hanna, R.E., Hegde, M., Donovan, K.F., Strand, C., Sullender, M.E., Vaimberg, E.W., Goodale, A., Root, D.E., Piccioni, F., and DOENCH, J.G. (2018). Optimized libraries for CRISPR-Cas9 genetic screens with multiple modalities. Nat. Commun. <i>9</i>, 5416. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref8">View at publisher</a></p>
<p id="bib9">Tiscornia, G., Singer, O., and Verma, I.M. (2006). Production and purification of lentiviral vectors. Nat. Protoc. <i>1</i>, 241-245. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00014-9/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources should be directed to and will be fulfilled by the Lead Contact, Dr. Qiurong Ding (<a href="mailto:qrding@sibs.ac.cn">qrding@sibs.ac.cn</a>).</p>
<h4>Materials Availability</h4>
<p>All unique materials generated from this study are available from the Lead Contact with a complete Materials Transfer Agreement.</p>
<h4>Data and Code Availability</h4>
<p>We did not generate any unique datasets or data.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants from the <a href="https://doi.org/10.13039/501100013290">National Key R&amp;D Program of China</a> (2017YFA0102800, 2017YFA0103700), the <a href="https://doi.org/10.13039/501100002367">Strategic Priority Research Program of the Chinese Academy of Sciences</a> (XDA16030402), and the <a href="https://doi.org/10.13039/501100001809">National Natural Science Foundation of China</a> (31670829, 31971063).</p>
<h3>Author Contributions</h3>
<p>Conceptualization, Y.Q. and Q.D.; Investigation, Y.Q.; Writing – Original Draft, Y.Q.; Writing – Review &amp; Editing, Y.Q. and Q.D.; Funding Acquisition, Q.D.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>