<article>
<section>
<h1 id="header">Simultaneous recording of behavioral and neural responses of free-moving nematodes <i>C. elegans</i>
</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Hirofumi Sato,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Hirofumi Kunitomo,<sup><a href="#aff1">1</a></sup> Xianfeng Fei,<sup><a href="#aff2">2</a></sup> Koichi Hashimoto,<sup><a href="#aff3">3</a></sup> and Yuichi Iino<sup><a href="#aff1">1</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biological Sciences, Graduate School of Science, The University of Tokyo, 113-0033 Tokyo, Japan</p>
<p id="aff2"><sup>2</sup>Department of Intelligent Information, Faculty of Science and Technology, Tohoku Bunka Gakuen University, 981-8551 Sendai, Japan</p>
<p id="aff3"><sup>3</sup>Department of System Information Sciences, Graduate School of Information Sciences, Tohoku University, 980-8579 Sendai, Japan</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:hisato@bs.s.u-tokyo.ac.jp">hisato@bs.s.u-tokyo.ac.jp</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:iino@bs.s.u-tokyo.ac.jp">iino@bs.s.u-tokyo.ac.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101011">10.1016/j.xpro.2021.101011</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>To reveal the neural mechanisms that control animal behavior, it is necessary to link the neural responses to behavioral changes and interpret them. We have developed a protocol to simultaneously record the behavior and neural activity of freely moving <i>C. elegans</i> by combining a microfluidic device and a tracking stage. Here we detail the protocol for the experiment, with an example of behavioral and neural responses of nematodes to salt concentration changes.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref8">Sato et al. (2021)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Simultaneous recording of neural responses and behavior of freely moving <i>C. elegans</i>
</li>
<li>A format for efficiently analyzing behavior and neural activity in salt chemotaxis</li>
<li>Using microfluidic devices to manipulate the magnitude and timing of stimulation</li>
<li>Applicable to behavioral analysis for various stimuli</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1245-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>In this experiment, <i>C. elegans</i> adults are to be used. For cultivating animals, we use the standard method, using Nematode Growth Medium (NGM) agar plates and <i>E. coli</i> OP50 strain (<a href="#bib3">Brenner, 1974</a>). It is known that <i>C. elegans</i> is attracted to a salt concentration at which the animal experiences food (or the animals were "conditioned" at the salt concentration; (<a href="#bib6">Kunitomo et al., 2013</a>; <a href="#bib8">Sato et al., 2021</a>). Animals conditioned with high salt concentrations show migration toward high salt concentrations, while those conditioned with low salt concentrations show migration toward low salt concentrations. We describe experiments aimed at observing the behavior and neural activity of nematodes depending on the conditioned salt concentration. Therefore, the nematodes are conditioned at various salt concentrations before the assay is performed. To condition the nematodes, it is necessary to prepare NGM plates with various salt concentrations. Therefore, in this section, we describe how to prepare the plates and reagents necessary for the experiment.</p>
<h3 id="sec1.1">NGM condition plates preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>The animals used in the experiment should be conditioned the day before the observation. Standard NGM plates contain 50 mM NaCl. Therefore, here we prepare NGM plates containing 0 mM, 50 mM, and 100 mM NaCl. These NGM plates are osmotically adjusted by glycerol as in previous studies (<a href="#bib6">Kunitomo et al., 2013</a>).</p>
<ol>
<li>Put 2.5 g HIPOLYPEPTONE (or polypeptone from any supplier) and 17 g agar in a 2 L glass bottle and add 5.84 g (100 mM) or 2.92 g (50 mM) or 0 g (0 mM) NaCl.</li>
<li>Add 4.02 mL (5.07 g), 7.68 mL (16.81 g) or 11.33 mL (28.55 g) of glycerol for 100 mM, 50 mM, or 0 mM NaCl plates, respectively, to adjust the osmolarity.</li>
<li>Add 1 mL cholesterol solution (5 mg/mL diluted in ethanol) and then fill the bottle up to 1000 mL with milli-Q water.</li>
<li>Sterilize by autoclaving for 20 min at 121°C. Also sterilize the KPO<sub>4</sub> buffer (1 M stock solution, pH 6.0, see tables in <a href="#materials-and-equipment">materials and equipment</a>), MgSO<sub>4</sub> (1 M stock solution) and CaCl<sub>2</sub> (1 M stock solution).</li>
<li>After the autoclaved bottle cools to below 60°C, add 25 mL KPO<sub>4</sub> buffer, 1 mL MgSO<sub>4</sub> solution, and 1 mL CaCl<sub>2</sub> solution.</li>
<li>Dispense 10 mL of the NGM into each 60 mm Petri dish.</li>
<li>Place the NGM at room temperature (20°C–25°C) overnight to solidify.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The NGM plates should not be dried because it will change the salt concentration it contains. It is recommended to store the plates in airtight containers (4°C, up to 1 month).</div>
<table id="undtbl1">
<caption>0 mM NGM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Polypeptone</td>
<td>2.5 mg/mL</td>
<td>2.5 g</td>
</tr>
<tr>
<td>Agar</td>
<td>17 mg/mL</td>
<td>17 g</td>
</tr>
<tr>
<td>5 mg/mL Cholesterol solution</td>
<td>5 mg/L</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M MgSO<sub>4</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M CaCl<sub>2</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M KPO<sub>4</sub> buffer</td>
<td>25 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>310 mM</td>
<td>22.66 mL</td>
</tr>
<tr>
<td>Milli Q water</td>
<td>N/A</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>50 mM NGM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>50 mM</td>
<td>2.92 g</td>
</tr>
<tr>
<td>Polypeptone</td>
<td>2.5 mg/mL</td>
<td>2.5 g</td>
</tr>
<tr>
<td>Agar</td>
<td>17 mg/mL</td>
<td>17 g</td>
</tr>
<tr>
<td>5 mg/mL Cholesterol solution</td>
<td>5 mg/L</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M MgSO<sub>4</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M CaCl<sub>2</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M KPO<sub>4</sub> buffer</td>
<td>25 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>210 mM</td>
<td>15.35 mL</td>
</tr>
<tr>
<td>Milli Q water</td>
<td>N/A</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>100 mM NGM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>100 mM</td>
<td>5.84 g</td>
</tr>
<tr>
<td>Polypeptone</td>
<td>2.5 mg/mL</td>
<td>2.5 g</td>
</tr>
<tr>
<td>Agar</td>
<td>17 mg/mL</td>
<td>17 g</td>
</tr>
<tr>
<td>5 mg/mL Cholesterol solution</td>
<td>5 mg/L</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M MgSO<sub>4</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M CaCl<sub>2</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M KPO<sub>4</sub> buffer</td>
<td>25 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>110 mM</td>
<td>8.04 mL</td>
</tr>
<tr>
<td>Milli Q water</td>
<td>N/A</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<h3 id="sec1.2">Bacterial food source</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>NGM conditioning plates are seeded with <i>E. coli</i> OP50 as a food source.</p>
<ol start="8">
<li>Grow <i>E. coli</i> OP50 as single colonies on LB agar plates at 37C for 16 h.</li>
<li>Pick a single colony of <i>E. coli</i> OP50 and culture it in LB medium without NaCl (see table in <a href="#materials-and-equipment">materials and equipment</a>) at 37C for 16 h.</li>
<li>Apply 50 μL of <i>E. coli</i> OP50 culture medium to each NGM plate in a 60 mm Petri dish, and spread the bacteria using a glass spreader. Then culture the bacteria at 37C for 16 h.</li>
</ol>
<h3 id="sec1.3">Preparation of NaCl solutions at various concentrations</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>In this experiment, NaCl solutions containing different concentrations of NaCl are used, and switching between them stimulates the animals. We usually mix a 0 mM NaCl solution with a 100 mM NaCl solution to make a solution with the required NaCl concentration. The osmolarity of these NaCl solutions is adjusted to about 350 mOSM by glycerol, as in previous studies (<a href="#bib6">Kunitomo et al., 2013</a>; <a href="#bib9">Suzuki et al., 2008</a>). As an example, since the osmolarity of a 100 mM NaCl solution is higher than that of a 0 mM NaCl solution due to the higher amount of 100 mM salt (= 200 mOSM), the osmolarity is adjusted by adding 200 mM more glycerol to the 0 mM NaCl solution.</p>
<ol start="11">
<li>Add 2.92 g (100 mM) or 0 g (0 mM) NaCl. Then add 0.1 g gelatin to prevent nematodes from sticking to the PDMS device and pipette tips.</li>
<li>Add 2.54 g (100 mM NaCl) or 14.28 g (0 mM NaCl) of glycerol to adjust the osmolarity and fill up the bottle to 500 mL with miliRO water.</li>
<li>Sterilize by autoclaving for 20 min at 121°C. Also sterilize the KPO<sub>4</sub> buffer (1 M stock solution, pH 6.0, see tables in <a href="#materials-and-equipment">materials and equipment</a>) and MgSO<sub>4</sub> (1 M stock solution) and CaCl<sub>2</sub> (1 M stock solution)</li>
<li>After the autoclaved bottle cools to below 60°C, add 12.5 mL KPO<sub>4</sub> buffer, 0.5 mL MgSO<sub>4</sub> solution, and 0.5 mL CaCl<sub>2</sub> solution.</li>
</ol>
<table id="undtbl4">
<caption>0 mM NaCl solutions</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M MgSO<sub>4</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M CaCl<sub>2</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M KPO<sub>4</sub> buffer</td>
<td>25 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>Gelatin</td>
<td>0.2 mg/mL</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>310 mM</td>
<td>22.66 mL</td>
</tr>
<tr>
<td>Milli Q water</td>
<td>N/A</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>100 mM NaCl solutions</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>100 mM</td>
<td>5.84 g</td>
</tr>
<tr>
<td>1 M MgSO<sub>4</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M CaCl<sub>2</sub> solution</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M KPO<sub>4</sub> buffer</td>
<td>25 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>Gelatin</td>
<td>0.2 mg/mL</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>110 mM</td>
<td>8.04 mL</td>
</tr>
<tr>
<td>Milli Q water</td>
<td>N/A</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">These solutions can be stored at 4°C for up to 1 month.</p>
<h3 id="sec1.4">Preparation of nematodes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<ol start="15">
<li>Place four adult hermaphrodite nematodes on an NGM plate (60 mm petri dish) with food (<i>E. coli</i>).</li>
<li>Incubate the nematodes at 20°C for 4 days.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The width of the microfluidic chamber used in this experiment is adjusted to the body size of the nematodes at 5 days post hatching, raised at 20°C. Therefore, in this experiment, nematodes at 4 days (about 96 h) post hatching are mainly used, and they are then conditioned overnight (&gt; 6 h). If too young or too old nematodes are used, their body size may not fit into the microfluidic device, making it difficult to quantify their movements.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Escherichia coli, OP50</td>
<td>Caenorhabditis Genetics Center</td>
<td>OP50</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Sodium Chloride</td>
<td>MANAC</td>
<td>9317</td>
</tr>
<tr>
<td>Hipolypeptone (or any polypeptone)</td>
<td>WAKO</td>
<td>392-02115</td>
</tr>
<tr>
<td>Agar</td>
<td>TAISHO TECHNOS</td>
<td>17071902</td>
</tr>
<tr>
<td>Magnesium sulfate (MgSO<sub>4</sub>) heptahydrate</td>
<td>Sigma-Aldrich</td>
<td>19-0480-5</td>
</tr>
<tr>
<td>Calcium chloride (CaCl<sub>2</sub>)</td>
<td>WAKO</td>
<td>033-25035</td>
</tr>
<tr>
<td>Cholesterol</td>
<td>WAKO</td>
<td>034-03002</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Sigma-Aldrich</td>
<td>64-17-5</td>
</tr>
<tr>
<td>Monopotassium phosphate (KH<sub>2</sub>PO<sub>4</sub>)</td>
<td>Sigma-Aldrich</td>
<td>24-5260-5</td>
</tr>
<tr>
<td>Dipotassium phosphate (K<sub>2</sub>HPO<sub>4</sub>)</td>
<td>Sigma-Aldrich</td>
<td>24-5240-5</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma-Aldrich</td>
<td>12-1120-5</td>
</tr>
<tr>
<td>LB Broth</td>
<td>Sigma-Aldrich</td>
<td>L3522-250G</td>
</tr>
<tr>
<td>M9 buffer</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Fluorescein</td>
<td>Sigma-Aldrich</td>
<td>46955-1G-F</td>
</tr>
<tr>
<td>Gelatin</td>
<td>Nacalai Tesque</td>
<td>16631-05</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>C. elegans</i> wild-type N2</td>
<td>Caenorhabditis Genetics Center</td>
<td>N2 (Bristol)</td>
</tr>
<tr>
<td><i>C. elegans N2; Ex[gcy-5p::gcamp6s; gcy-5p::mCherry; lin-44p::gfp]</i></td>
<td><a href="#bib8">Sato et al. (2021)</a></td>
<td>JN3500</td>
</tr>
<tr>
<td><i>C. elegans N2; Ex[npr-9p::gcamp6s; npr-9p::mCherry; lin-44p::gfp]</i></td>
<td><a href="#bib8">Sato et al. (2021)</a></td>
<td>JN3329</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>MetaMorph</td>
<td>Molecular Devices</td>
<td>MetaMorph Version7.7.4.0</td>
</tr>
<tr>
<td>Worm tracking software</td>
<td>Dr. Koichi Hashimoto/HAWKVISION</td>
<td>N/A</td>
</tr>
<tr>
<td>ImageJ</td>
<td>National Institutes of Health (NIH)</td>
<td>Java 1.8.0_172</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>The code for image analysis</td>
<td><a href="#bib8">Sato et al. (2021)</a></td>
<td>Mendeley Data: <a href="https://doi.org/10.17632/xcxngcy982.1">https://doi.org/10.17632/xcxngcy982.1</a>
</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Petri plates</td>
<td>IWAKI</td>
<td>1010-060</td>
</tr>
<tr>
<td>Terumo syringe, 50 mL</td>
<td>TERUMO</td>
<td>SS-50ESZ</td>
</tr>
<tr>
<td>Terumo syringe, 10 mL</td>
<td>TERUMO</td>
<td>SS-10SZ</td>
</tr>
<tr>
<td>Three-way stopcock</td>
<td>TERUMO</td>
<td>TS-TL2K</td>
</tr>
<tr>
<td>Polyethylene tube, inner diameter φ 0.58 mm</td>
<td>Imamura</td>
<td>01630-05</td>
</tr>
<tr>
<td>Stainless steel tube (cut 23G stainless steel tube into 1cm lengths)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Syringe pump</td>
<td>kd Scientific</td>
<td>78-8110</td>
</tr>
<tr>
<td>Peristaltic pump</td>
<td>ATTO corporation</td>
<td>SJ-1211</td>
</tr>
<tr>
<td>Optical microscope</td>
<td>OLYMPUS</td>
<td>BX51</td>
</tr>
<tr>
<td>Tracking stage (with control software)</td>
<td>HAWKVISION</td>
<td>HV-SAU02</td>
</tr>
<tr>
<td>EM-CCD Camera</td>
<td>HAMAMATSU</td>
<td>C9743-13 or successor model</td>
</tr>
<tr>
<td>Digital camera for infrared light imaging.</td>
<td>Point Grey</td>
<td>GRAS-03K2M-C</td>
</tr>
<tr>
<td>Glass chip 34 × 45 × 1.1</td>
<td>TOSHIN RIKO</td>
<td>N/A (Custom made, details to follow)</td>
</tr>
<tr>
<td>PDMS microfluidic chip (modified from <a href="#bib1">Albrecht and Bargmann, 2011</a>)</td>
<td>YODAKA</td>
<td>U944 (Custom made, details to follow)</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl6">
<caption>1 M KPO<sub>4</sub> buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>0.87 M</td>
<td>118.14 g</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>0.13 M</td>
<td>22.96 g</td>
</tr>
<tr>
<td>Milli Q water</td>
<td>N/A</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl7">
<caption>1 M MgSO<sub>4</sub> stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>MgSO<sub>4</sub>·7H<sub>2</sub>O</td>
<td>1 M</td>
<td>246.47 g</td>
</tr>
<tr>
<td>Milli Q water</td>
<td>N/A</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl8">
<caption>1 M CaCl<sub>2</sub> stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>1 M</td>
<td>147.01 g</td>
</tr>
<tr>
<td>Milli Q water</td>
<td>N/A</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<h3 id="sec2.1">PDMS microfluidic device</h3>
<p>The microfluidic device is made of polydimethylsiloxane (PDMS) and is a modified version of the one developed at previous studies (<a href="#bib1">Albrecht and Bargmann, 2011</a>; <a href="#fig1">Figures 1</a>A and 1B). The overall size is 40 mm × 25 mm, with a microfluidic chamber of 20 mm square. The microfluidic chamber is lined with pillars of 0.2 mm in diameter, and the distance between the pillars is 0.1 mm. There are also four holes with a diameter of 0.5 mm, each of which is connected to the microfluidic chamber.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1245-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Components of microfluidic device</div>
<p>(A) Picture of PDMS microfluidic chip.</p>
<p>(B) Structural diagram of PDMS microfluidic chip. It consists of four holes for inserting tubes and a microfluidic chamber of 20 mm square. Inside the microfluidic chamber, pillars with 0.2 mm diameter are placed at 0.1 mm intervals. The comb-shaped parts in the enlarged image have been modified from the original version (<a href="#bib1">Albrecht and Bargmann, 2011</a>).</p>
<p>(C) Picture of two glass plates. One of the two plates has four holes in it.</p>
<p>(D) Schematic of a glass plate with holes. There are four holes with 1 mm diameter. The positions of the four holes are arranged so that they overlap with the holes in the PDMS microfluidic chip.</p>
</figcaption>
</figure>
<h3 id="sec2.2">Glass plates</h3>
<p>The glass plates are common microscope slides, and the size is 45 mm × 34 mm, and are used in a set of two (<a href="#fig1">Figure 1</a>C). One of the two plates has four holes (1 mm in diameter), the position of which overlaps with the holes in the PDMS microfluidic device (<a href="#fig1">Figures 1</a>B and 1D). The material of the glass plates is the same as that of common glass slides for microscopes.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p><i>C. elegans</i> is known to exhibit behavioral changes and neural responses to a variety of stimuli (<a href="#bib2">Bargmann et al., 1993</a>; <a href="#bib4">Gray et al., 2004</a>; <a href="#bib5">Hilliard et al., 2005</a>; <a href="#bib7">Mori and Ohshima, 1995</a>; <a href="#bib10">Ward et al., 2008</a>). This tracking imaging system using a PDMS microfluidic device allows us to simultaneously observe animal behavior and neural responses to stimuli. This protocol targets the experience-dependent salt chemotaxis of <i>C. elegans</i>. It can also be applied to other stimuli (e.g., water-soluble chemicals, odors, osmotic stimuli, etc.) if they can be channeled through the microfluidic system (<a href="#bib1">Albrecht and Bargmann, 2011</a>). The protocol is divided into four stages: 1: conditioning, 2: assembly of the microfluidic device and introduction of the nematodes, 3: administration of the stimuli and recording of movement and neural responses, and 4: quantification of behavior and neural responses.</p>
<h3 id="sec3.1">Conditioning of animals</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This assay aims to measure the behavior and neural activity of nematodes in response to salt concentration change stimuli, and to investigate whether these responses are altered by the salt concentration that the nematodes have previously experienced. Therefore, initially, the nematodes are conditioned with various salt concentrations.</p>
<ol>
<li>Wash nematodes from one 6 cm plate (make sure that there is still food left, and the nematodes are not starving, and more than 200 adult nematodes are present) into a 1.5 mL Eppendorf tube using 1 mL of M9 buffer.</li>
<li>After the nematodes have settled to the bottom of the tube, remove the supernatant, and wash again with 1 mL of M9 buffer.</li>
<li>After the nematodes settle to the bottom of the tube, remove the supernatant, and transfer the nematodes to an NGM condition plate (0, 50, or 100 mM NaCl) with a lawn of <i>E. coli.</i>
</li>
<li>Incubate at 20°C overnight (at least 6 h or more) for conditioning.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> When transferring nematodes to an NGM condition plate, do not bring in too much M9 buffer to minimize its effect on the salt concentration of the plate. A standard amount is about 10–20 μL.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If you are using strains with extrachromosomal arrays which are not 100% heritable for your experiment, you can pick up about 100 nematodes carrying the arrays and transfer them to an NGM condition plate.</div>
<h3 id="sec3.2">Assembly of the microfluidic device</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>In this experiment, nematodes were placed in a PDMS microfluidic chamber and stimulated by switching the solution flowing in the chamber. A chip made of PDMS is sandwiched between two glass plates, and the stimulating solution flows through holes in the glass plates and the chip (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1245-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic diagram of how to assemble a microfluidic device</div>
<p>(A) Summary of each tube connecting to the microfluidic chip.</p>
<p>(B) Pictures from each stage of assembly. First, sandwich the microfluidic chip between two glass plates. At this time, the holes in the microfluidic chip and the holes in the glass plate are placed so that they overlap. Next, insert steel pins into each hole and hold the glass plates in place with binder clips. Then, connect polyethylene tubes to each pin.</p>
</figcaption>
</figure>
<ol start="5">
<li>Wash a glass plate, a perforated glass plate, and a PDMS microfluidic chip with 100% ethanol and air-dry them.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Store the glass plates and PDMS chips in ethanol to avoid their getting moldy.</div>
<ol start="6">
<li>Place the PDMS chip between the glass plates (<a href="#fig2">Figure 2</a>B left). Align the holes in the perforated glass plate with the holes for inserting the polyethylene tube (φ 0.58 mm) in the PDMS chip. Then insert the stainless steel tubes into the four holes of the chip.</li>
<li>Fix the glass plate with a binder clip (<a href="#fig2">Figure 2</a>B right top).</li>
<li>Vacuum pump for 15 min to evaporate.</li>
<li>Mix 0 mM NaCl solution and 100 mM NaCl solution to make 50 mL of 50 mM NaCl solution and 25 mM NaCl solution.</li>
<li>Add 0.5 μL of 0.5 μM fluorescein solution to 25 mM NaCl solution (final concentration: 1 nM).</li>
<li>Put 40 mL each of 50 mM NaCl buffer and 25 mM NaCl buffer into respective 50 mL Terumo syringe set it on the feed pump, and connect polyethylene tube to the syringes (<a href="#fig3">Figure 3</a>A, NaCl solution 1 and 2).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1245-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Overview diagram of the experimental setup</div>
<p>(A) Overview of the microfluidic device and each polyethylene tube connected to it. There are two syringes for NaCl solution (① and ②, 1 m length each), one syringe for nematodes introduction (③, 0.5 m length), and one peristaltic pump for waste fluid connected (④, 0.5 m length).</p>
<p>(B) Schematic diagram of the imaging system. An infrared camera is used to acquire bright field images and an EM-CCD camera is used to acquire fluorescence images. The bright field images are used to control the tracking stage so that the nematode is kept in the center of the field of view.</p>
</figcaption>
</figure>
<ol type="a">
<li>Put 10 mL of 50 mM NaCl buffer into a 10 mL syringe and connect the tube (<a href="#fig3">Figure 3</a>A, nematode introduction syringe). This syringe is used for nematode introduction to the microfluidic chamber. Flow path from the nematode introduction syringe is opened and closed using a three-way stopcock.</li>
<li>Connect a syringe with the NaCl solution to one side of the flow path and a polyethylene tube to the other side that will be later connected to PDMS. Attach another syringe to the third flow path and use it to remove air from the tube.</li>
</ol>
</li>
<li>Connect the polyethylene tubes from each syringe and pump (①–④) to the steel tubes of the PDMS microfluidic device, and turn on all pumps and open all tubes to allow the solution to flow (<a href="#fig2">Figures 2</a>B and <a href="#fig3">3</a>A).
<ol type="a">
<li>There are four polyethylene tubes in total. Two tubes are for NaCl solution (<a href="#fig3">Figure 3</a>A ① and ②), one for introducing nematodes (<a href="#fig3">Figure 3</a>A ③), and one for waste fluid (<a href="#fig3">Figure 3</a>A ④).</li>
<li>When the chamber is filled with NaCl solution, close all the tubes. The flow rate of each pump should be set to 100 μL/min.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure that the solution has completely filled the microfluidic chamber. If air bubbles are present in the chamber, they may cause failure of solution exchange or prevent nematode movement during imaging. If air bubbles in the chamber are hard to be removed, turn off the peristaltic pump attached to the waste.</div>
<h3 id="sec3.3">Introduction of nematodes into the microfluidic chamber</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–15 min</div>
<ol start="13">
<li>Put 1 mL of 50 mM NaCl solution in a 1.5 mL Eppendorf tube and put 10–20 nematodes picked from the conditioning plate.</li>
<li>Take nematodes with a P1000 micropipette and put it into the NaCl solution in a nematodes-introduction syringe (10 mL Terumo syringe) (<a href="#fig3">Figure 3</a>A ③).</li>
<li>Open the tube from the nematode-introduction syringe (<a href="#fig3">Figure 3</a>A ③), turn on the peristaltic pump for waste fluid (<a href="#fig3">Figure 3</a>A ④), and introduce nematodes into the microfluidic chamber. After the nematode enters the chamber, close the tube from the nematode-introduction syringe and turn off the waste pump.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The number of nematodes to be introduced into the microfluidic chamber should be about 10. If the number of nematodes is too small, it may be difficult to find suitable ones for imaging. On the other hand, if there are too many nematodes, they may collide with each other, making it difficult to track them.</div>
<h3 id="sec3.4">Stimulation of animals and measurement of behaviors and neural activities</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>An infrared camera and an EM-CCD camera are used to acquire bright field images and fluorescence images, respectively (<a href="#fig3">Figure 3</a>B). Based on the bright field images, the tracking stage is controlled according to the animal's movement, and the behavior and neural activity of the moving animal are acquired.</p>
<ol start="16">
<li>Turn on the peristaltic pump for waste and open the tube of 50 mM NaCl solution.</li>
<li>Launch the tracking stage, the nematode tracking software, and the fluorescence imaging software (MetaMorph). Then, select the head of the nematode and start tracking. At the same time, start capturing brightfield and fluorescence images.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The nematode tracking software uses the pattern around the selected point as a template, and tracks the animal based on it. Therefore, if the template is set to include the pharynx and part of the intestine, which have a characteristic shape, the tracking will be successful (<a href="#mmc1">Methods video S1</a>). In addition, it is better to select a worm that is not moving too fast or too slow (about 50–100 μm/s) to obtain good quality data.</div>
<ol start="18">
<li>25 s after the start of imaging, close the tube of 50 mM NaCl solution and simultaneously open the tube of 25 mM NaCl solution.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Depending on the position of each nematode in the microfluidic chamber, there is a time lag between the opening and closing of the tube and actual delivery of the stimulus (salt concentration change) to the nematode. In this experiment, fluorescein was added to a 25 mM NaCl solution. Therefore, the timing of solution switching can be identified by imaging the fluorescence of fluorescein.</div>
<ol start="19">
<li>75 s after the start of imaging, close the tube of 25 mM NaCl solution and open the tube of 50 mM NaCl solution at the same time.
<ol type="a">
<li>At 125 s after the start of shooting, close the tube of 50 mM NaCl solution again and open the tube of 25 mM NaCl solution at the same time.</li>
<li>At 180 s after the start of imaging, end the measurement, close the tube of 25 mM NaCl solution and open the tube of 50 mM NaCl solution.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this experiment, the nematodes are stimulated to change salt concentration from</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1245-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Simultaneous recording of velocity and ASER activity in response to salt concentration change, related to steps 17–19</div>
<p>Bright field image on the left and fluorescence image on the right. In the bright field image, the area surrounded by white squares (the pharynx and part of the intestine of the nematode) is used as a template for tracking. Since fluorescein is added to the 25 mM NaCl solution, it is possible to identify the timing of solution switching in the GCaMP channel.</p>
</figcaption>
</figure>
<p>50 mM → 25 mM → 50 mM → 25 mM (<a href="#fig4">Figure 4</a>A). The nematode's forward/backward state lasts for about several seconds to several tens of seconds. Therefore, in this experiment, the stimulus switching interval was set to 50 s. Since nematodes are attracted to the NaCl concentration at conditioning, changes away from and toward the NaCl concentration at conditioning are applied as stimuli. For example, for a nematode conditioned to 50 mM NaCl concentration, a change in NaCl concentration from 50 mM to 25 mM is a change away from the conditioned concentration, and a change from 25 mM to 50 mM is a change toward the conditioned concentration.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1245-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. The salt stimuli used in the experiment and the responses to them</div>
<p>(A) Time course for switching salt solutions during the experiment. Since it takes about 30 s from the entrance to the exit of the microfluidic chamber, the actual timing at which the stimulus is delivered to nematodes depends on their position in the microfluidic chamber.</p>
<p>(B) Comparison of the magnitude of the velocity change and neural response of ASER to salt concentration down-step (50 mM–25 mM) and up-step (25 mM–50 mM) between the first animal and the second and following animals. These animals were conditioned with 100 mM NaCl. Each graph represents the differences between mean value of 10 s windows before and after stimulus. Each dot represents the data of one animal. Mean ± S. E. M. (NS) not significant, Welch’s test.</p>
</figcaption>
</figure>
<ol start="20">
<li>Select another nematode in the chamber and measure again by steps 17–19. Up to about 5 animals should be measured in one experiment.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this experiment, no significant differences in neural responses or behavioral changes due to repeated stimulus exposure were detected (<a href="#fig4">Figure 4</a>B). If different stimuli or experimental conditions are to be used, it is necessary to estimate whether there is an effect of repeated stimulation. To avoid the effects of repeated stimulation, return to step 5 after measuring the first animal.</div>
<ol start="21">
<li>After the experiment, dismantle the microfluidic device and clean it with ethanol. To continue the imaging again, start from step 5.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Data on velocity and neural activity are collected for a single recording (<a href="#fig5">Figures 5</a> and <a href="#fig6">6</a>). The velocity of the nematode is expected to vary depending on whether the stimulus received is attractive or aversive. For example, nematodes conditioned with 100 mM NaCl showed backward movement in response to salt concentration decrease (<a href="#fig5">Figure 5</a>). In contrast, nematodes conditioned with 0 mM NaCl showed backward movement in response to salt concentration increase (<a href="#fig6">Figure 6</a>). In addition, each neuron is thought to exhibit its own characteristic activity pattern.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1245-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Velocity and ASER activity in response to salt concentration changes</div>
<p>The data are from one animal, and this animal was conditioned with 100 mM NaCl. The animal is exposed to 50 mM NaCl solution in the white part of the graph and 25 mM in the gray part. The upper panel is a graph of velocity change, where negative values indicate backward movement. The lower panel shows the change in intracellular calcium concentration in the ASER neuron. This animal showed backward movement when the salt concentration decreased, and ASER was also activated for the decrease in salt concentration.</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1245-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Velocity and AIB activity in response to salt concentration changes</div>
<p>The data are from one animal, and this animal was conditioned with 0 mM NaCl. The animal is exposed to 50 mM NaCl solution in the white part of the graph and 25 mM in the gray part. The upper panel is a graph of velocity change, where negative values indicate backward movement. The lower panel shows the change in intracellular calcium concentration in the AIB neuron. This animal showed backward movement when the salt concentration increased, and AIB was activated at the same moment as the backward movement.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>In order to quantify behavior and neural activity from the acquired images, it is necessary to analyze them ad hoc.</p>
<h3 id="sec5.1">Quantification of the behavior of nematodes</h3>
<ol>
<li>To quantify the behavior (movement speed) of nematodes, use bright field images. For each image, calculate the distance and direction of movement by comparing the deviation from the image in the previous frame.</li>
<li>Next, divide the distance traveled by the time per frame (30 ms) to obtain the speed of the nematode's movement at that time.</li>
<li>Furthermore, detect and compare the direction of the nematode's head in each image with the direction of movement to determine whether the nematode is moving forward or backward.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is recommended to take a moving average of about 1 s because the velocity data is noisy. The code used for these analyses are available in the following repository: Mendeley Data: <a href="https://doi.org/10.17632/xcxngcy982.1">https://doi.org/10.17632/xcxngcy982.1</a>.</div>
<h3 id="sec5.2">Quantification of neural activities</h3>
<p>To quantify neural activity, we use fluorescent images: after subtracting the background of the image using ImageJ, we quantify the fluorescence of the target area using the Track Objects function of MetaMorph or Analyze particles function of ImageJ. The fluorescence of fluorescein is used to determine the timing of the stimulus (solution exchange) to the nematode.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The width of the microchamber is adjusted to the body of an adult nematode (5 days post hatching, raised at 20°C). Therefore, it may be difficult to track the movement of larvae or mutants with thin bodies. If you want imaging of thin-bodied nematodes, you need to design a narrower chamber (Ordinarily, the spacing between pillars in a microfluidic chamber is 100 μm, and this space is to be narrower.).</p>
<p>It is possible to record from nematodes on an agar plate instead of in a microchamber, but in this case it is necessary to be cautious about the shift in the z-plane, because agar plates may not always be flat.</p>
<p>Since there is no food in the microchamber, prolonged imaging may have starvation effects on the nematodes. Therefore, it is not recommended to take images for more than 30 min after introducing nematodes into the microchamber.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Air bubbles are formed in the microfluidic chamber during imaging (step 16–20).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>This problem may arise at step 16–20. This may be because the speed of the waste pump (<a href="#fig3">Figure 3</a>A ④) is faster than that of the feed pumps (<a href="#fig3">Figure 3</a>A, ① and ②). Therefore, the problem can be solved by reducing the speed of the waste pump or increasing the speed of the feed pumps. However, if the pumping speed is too fast, the movement of the nematode will be affected, so it is recommended to reduce the speed of the liquid waste pump if possible.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Nematodes do not enter the microfluidic chamber (step 15).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>This problem may arise at step 15. Push the nematode-introduction syringe slowly with the piston to make it easier for the nematode to enter the microfluidic chamber. Be careful not to push too hard or the liquid will leak from the side of the chamber.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The nematode does not move in the microfluidic chamber/ moves too fast (step 17–19).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>This problem may arise at step 17–19. If the strain you are using is not a mutant with abnormal movement, the cause may be that the body width of the nematode is larger/smaller than the microfluidic chamber. If the nematodes do not move in the chamber, it is recommended to use younger nematodes, and if they move too fast, it is recommended to use older nematodes. A standard size is about the same as a wild-type (N2) at 5 days after hatching. If it is necessary to use animals for a certain number of days post hatching, it is recommended to design a thicker or narrower chamber (See also the “<a href="#limitations">limitations</a>” section.).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Solution switching does not occur/is slow (step 17–19).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>This problem may arise at step 17–19. First, make sure that the flow path is not clogged. Also, the microfluidic channel has a distance from the entrance of the solution to the outlet for the waste solution. Therefore, if the nematodes are close to the outlet, it will take about 30 s for the solution to reach the nematodes after switching the solution. If it takes much longer, it may be improved by increasing the speed of the pump for feeding the solution.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Solution leaks from the microfluidic chamber (step 12–20).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>This problem may arise at step 12–20. This is mainly due to the speed of the feed pumps (<a href="#fig3">Figure 3</a>A, ① and ②) being too much faster than the waste pump (<a href="#fig3">Figure 3</a>A ④). First, make sure that the flow path for the waste fluid is not clogged. Next, the speed of the feed pumps is reduced or the speed of the waste pump is increased, which can be a solution. In addition, using stronger binder clips to fix the microfluidic chip and the glass plates may be a solution.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Albrecht, D.R. and Bargmann, C.I. (2011). High-content behavioral analysis of <i>Caenorhabditis elegans</i> in precise spatiotemporal chemical environments. Nat. Methods <i>8</i>, 599-606. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref1">View at publisher</a></p>
<p id="bib2">Bargmann, C.I., Hartwieg, E., and Horvitz, H.R. (1993). Odorant-selective genes and neurons mediate olfaction in <i>C. elegans</i>. Cell <i>74</i>, 515-527. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref2">View at publisher</a></p>
<p id="bib3">Brenner, S. (1974). The genetics of <i>Caenorhabditis elegans</i>. Genetics <i>77</i>, 71-94. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref3">View at publisher</a></p>
<p id="bib4">Gray, J.M., Karow, D.S., Lu, H., Chang, A.J., Chang, J.S., Ellis, R.E., Marietta, M.A., and Bargmann, C.I. (2004). Oxygen sensation and social feeding mediated by a <i>C. elegans</i> guanylate cyclase homologue. Nature <i>430</i>, 317-322. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref4">View at publisher</a></p>
<p id="bib5">Hilliard, M.A., Apicella, A.J., Kerr, R., Suzuki, H., Bazzicalupo, P., and Schafer, W.R. (2005). In vivo imaging of <i>C. elegans</i> ASH neurons: cellular response and adaptation to chemical repellents. EMBO J. <i>24</i>, 63-72. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref5">View at publisher</a></p>
<p id="bib6">Kunitomo, H., Sato, H., Iwata, R., Satoh, Y., Ohno, H., Yamada, K., and Iino, Y. (2013). Concentration memory-dependent synaptic plasticity of a taste circuit regulates salt concentration chemotaxis in <i>Caenorhabditis elegans</i>. Nat. Commun. <i>4</i>, 2210. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref6">View at publisher</a></p>
<p id="bib7">Mori, I. and Ohshima, Y. (1995). Neural regulation of thermotaxis in <i>C. elegans</i>. Nature <i>376</i>, 344-348. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref7">View at publisher</a></p>
<p id="bib8">Sato, H., Kunitomo, H., Fei, X., Hashimoto, K., and Iino, Y. (2021). Glutamate signaling from a single sensory neuron mediates experience-dependent bidirectional behavior in <i>Caenorhabditis elegans</i>. Cell Rep. <i>35</i>, 109177. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref8">View at publisher</a></p>
<p id="bib9">Suzuki, H., Thiele, T.R., Faumont, S., Ezcurra, M., Lockery, S.R., and Schafer, W.R. (2008). Functional asymmetry in <i>Caenorhabditis elegans</i> taste neurons and its computational role in chemotaxis. Nature <i>454</i>, 114-117. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref9">View at publisher</a></p>
<p id="bib10">Ward, A., Liu, J., Feng, Z., and Xu, X.Z.S. (2008). Light-sensitive neurons and channels mediate phototaxis in <i>C. elegans</i>. Nat. Neurosci. <i>11</i>, 916-922. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00717-6/sref10">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Yuichi Iino (<a href="mailto:iino@bs.s.u-tokyo.ac.jp">iino@bs.s.u-tokyo.ac.jp</a>).</p>
<h4>Materials availability</h4>
<p>All unique/stable reagents generated in this study are available from the lead contact without restriction.</p>
<h4>Data and code availability</h4>
<p>Tracking-imaging software is commercially available along with the tracking-stage (HAWKVISION). The code for image analysis is available from the following repository: Mendeley Data: <a href="https://doi.org/10.17632/xcxngcy982.1">https://doi.org/10.17632/xcxngcy982.1</a>. For more information about the code, please contact the author. All relevant data are available from the authors upon request.</p>
<h3>Acknowledgments</h3>
<p>We thank the Caenorhabditis Genetics Center and the laboratory of Cornelia I. Bargmann for providing strains and the design of microfluidic devices. We also thank members of the Y.I. laboratory for helpful comments and advice with the experiments. This work was supported by the <a href="https://doi.org/10.13039/501100003382">Core Research for Evolutional Science and Technology</a> program (CREST) “Creation of Fundamental Technologies for Understanding and Control of Biosystem Dynamics,” Grant JPMJCR12W1, of the <a href="https://doi.org/10.13039/501100002241">Japan Science and Technology Agency</a>; Japan Society for the Promotion of Science KAKENHI Grant-in-Aid for Scientific Research (S) Grant JP17H06113; Grants-in-Aid for Innovative Area "Artificial Intelligence and Brain Science" (19H04980) and Japan Society for the Promotion of Science KAKENHI Grant-in-Aid for Early-Career Scientists Grant 19K16286, The salt Science Foundation No. 2043, and Center for Integrative Science of Human Behavior (CiSHuB).</p>
<h3>Author contributions</h3>
<p>H.S., H.K., and Y.I. designed experiments; H.S. performed experiments; H.K., X.F., and K.H. contributed new reagents/analytic tools; H.S. analyzed data; and H.S., H.K., and Y.I. wrote the paper.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>