<article>
<section>
<h1 id="header">Generation and functional characterization of murine mammary organoids</h1>
<p><time datetime="2021-08-27">Published: August 27, 2021</time></p>
<p>Hon Yan Kelvin Yip<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Antonella Papa<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Cancer Program, Monash Biomedicine Discovery Institute and Department of Biochemistry and Molecular Biology, Monash University, Melbourne, VIC 3800, Australia</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:antonella.papa@monash.edu">antonella.papa@monash.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100765">10.1016/j.xpro.2021.100765</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>3D cultures of mammary epithelial cells purified from murine models provide a unique resource to study genetically defined breast cancer and response to targeted therapies. Here, we describe step-by-step experimental procedures for the successful establishment of murine mammary organoid lines isolated from mammary glands or mammary tumors driven by mutations in components of the PI3K pathway. These detailed protocols also include procedures to perform assays that can be adopted to screen response to drug treatments and to inform better therapies.</p>
<p>For details on potential applications and use of this protocol, please refer to <a href="https://doi.org/10.1016/j.molcel.2020.09.027">Yip et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol to establish murine mammary organoids from normal and tumor tissues</li>
<li>Expansion, cryopreservation, and recovery of organoid lines enable bio-banking</li>
<li>Molecular and lineage markers reveal maintenance of physiologic differentiation</li>
<li>Cell proliferation and cell death assays allow screening in response to treatments</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The experimental procedures included in these protocols have been optimized to isolate and culture mammary organoids derived from female mice in a C57BL/6J genetic background. Adjustments in incubation times and volumes of reagents can make these protocols suitable also for highly side-branched mouse strains such as the 129 mice. Virgin female mice between 10–12 weeks of age have been used to isolate mammary epithelial cells, and mice of up to 6 months of age for isolation of tumor-derived mammary epithelial cells. Some of the required reagents and solutions used in this protocol can be prepared in advance and need to be stored as indicated; other reagents will need to be made fresh on the day of the experiment. Please refer to the <a href="#key-resources-table">key resources table</a> for a complete list of materials and equipment.</p>
<h3 id="sec1.1">Preparation of reagents and solutions</h3>
<p>Reagents to prepare in advance:</p>
<ol>
<li>50 μg/mL Epidermal Growth Factor (EGF) solution (2500×)
<ol type="a">
<li>E.g. Dissolve 100 μg of lyophilized EGF in 2 mL of sterile phosphate-buffered saline 1× (PBS-1<b>×</b>) + 0.1% (w/v) bovine serum albumin (BSA) to make a 50 μg/mL stock solution.</li>
<li>Aliquot 50 μL of EGF (50 μg/mL) in individual tubes and store at −20°C for up to 12 months.</li>
</ol>
</li>
<li>50 μg/mL Fibroblast Growth Factor (FGF)2 solution (2500×)
<ol type="a">
<li>E.g. Dissolve 100 μg of lyophilized FGF2 in 2 mL of sterile 5 mM Tris (pH7.6) + 0.1% (w/v) BSA to make a 50 μg/mL stock solution.</li>
<li>Aliquot 50 μL of FGF2 (50 μg/mL) in individual tubes and store at −20°C for up to 12 months.</li>
</ol>
</li>
<li>10 mM Y-27632 solution (1000×)
<ol type="a">
<li>E.g. Dissolve 10 mg of lyophilized Y-27632 in 3.037 mL of filter-sterile (0.2 μm filter) distilled water (dH<sub>2</sub>O) to make a 10 mM stock solution.</li>
<li>Aliquot 100 μL Y-27632 (10 mM) in individual tubes and store at −20°C for up to 12 months.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Due to the short half-life of these growth factors and inhibitor, the Organoid culture medium is made fresh every week in order to preserve stability and activity of compounds.</div>
<ol start="4">
<li>Matrigel
<ol type="a">
<li>Thaw a new bottle of Matrigel in a 4°C cold room on ice for 14–16 h, or on ice at room temperature (20°C–25°C), for 1–2 h.</li>
<li>The day after, transfer the bottle of Matrigel into a tissue culture hood, gently swirl the bottle and open the lid.</li>
<li>Using a pre-chilled 1 mL pipette tip, mix the Matrigel up-and-down 4–5 times, and transfer 1 mL aliquots in individual Eppendorf tubes.</li>
<li>Store Matrigel aliquots at −20°C, or at −80°C if using polypropylene or other compatible tubes that can withstand low temperatures.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For storage of reagents, follow manufacturer’s recommendations. Minimize freeze-thaw cycles to preserve integrity of reagents.</div>
<div class="critical">
<span class="critical-title">Critical:</span> For organoid cultures, it is recommended to use Matrigel containing high proteins concentration, i.e. &gt; 10 mg/mL, calculated with the Lowry method, and low endotoxin levels, i.e. ∼ 2EU/mL. Matrigels with low protein contents, due to batch-to-batch variability, are not recommended for organoids cultures as they may impair cellular growth.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Due to the high viscosity, it is critically important to thoroughly mix the Matrigel before dispensing into aliquots. Always pre-chill and wet pipette tips by pipetting ice-cold PBS-1<b>×</b> a few times before coming into contact with Matrigel. This will minimize loss of reagent as Matrigel can stick to the inside and outside of pipette tips.</div>
<div class="critical">
<span class="critical-title">Critical:</span> After thawing, Matrigel must be held on ice until the time of plating.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol has been optimized using the Corning® Matrigel Growth Factor Reduced (GFR) Basement Membrane Matrix; however, alternative basement membrane matrices have been generated and used to grow mouse and human organoids (<a href="#bib4">Dekkers et al., 2021</a>; <a href="#bib12">Sachs et al., 2018</a>). Matrigel is a naturally derived gel extracted from the Engelbreth-Holm-Swarm (EHS) mouse sarcoma and, as such, batch-to-batch variability in proteins content can occur. When using a new batch of Matrigel for the first time, it is suggested to perform comparative proliferation assays (e.g. CellTiter-Glo 3D proliferation assay) using a previously characterized organoid line to assess variations in growth rates across biological/technical replicates.</div>
<p>Regents prepared freshly every time on the day of tissue isolation:</p>
<ol start="5">
<li>10,000 U/mL Collagenase type III (33×)
<ol type="a">
<li>Weigh out 45 mg (equivalent to 10,000 U) of lyophilized collagenase and dissolve in 1 mL of basal DMEM/F12 medium to prepare a 10,000 U/mL stock solution.</li>
<li>Pipette up and down to aid dissolution.</li>
</ol>
</li>
<li>10 mg/mL Dispase II solution (2×)
<ol type="a">
<li>Dissolve 10 mg (equivalent to 18U) of lyophilized Dispase II for every mL of basal DMEM/F12 medium to make up 10 mg/mL stock.</li>
</ol>
</li>
<li>5 mg/mL DNase I solution (10×)
<ol type="a">
<li>Dissolve 5 mg (equivalent to 2000 U) of lyophilized DNase I in 1 mL of basal DMEM/F12 medium to prepare a 5 mg/mL stock solution.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not vortex solutions containing enzymes.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Animal-Free Recombinant Human EGF</td>
<td>PeproTech</td>
<td>Cat# AF-100-15</td>
</tr>
<tr>
<td>Animal-Free Recombinant Human FGF-Basic (FGF2)</td>
<td>PeproTech</td>
<td>Cat# AF-100-18B</td>
</tr>
<tr>
<td>Insulin-Transferrin-Selenium-Ethanolamine (ITS-X)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 51500056</td>
</tr>
<tr>
<td>Y-27632 dihydrochloride (ROCK inhibitor)</td>
<td>Tocris</td>
<td>Cat# 1254</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>Sigma-Aldrich</td>
<td>Cat# P4458</td>
</tr>
<tr>
<td>
<sup>1</sup>Recombinant Human R-spondin 1</td>
<td>R&amp;D Systems</td>
<td>Cat# 4645-RS</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Assay Matrix</td>
<td>Cat# ASFBS-FR</td>
</tr>
<tr>
<td>Gentamicin (10 mg/mL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15710064</td>
</tr>
<tr>
<td>DMEM/F-12 medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 21331020</td>
</tr>
<tr>
<td>Trypsin (2.5%)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15090046</td>
</tr>
<tr>
<td>TrypLE™ Express Enzyme (1×), phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12605028</td>
</tr>
<tr>
<td>Collagenase III</td>
<td>Worthington</td>
<td>Cat# LS004182</td>
</tr>
<tr>
<td>Dispase II</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 17105041</td>
</tr>
<tr>
<td>Corning® Matrigel® Growth Factor Reduced (GFR) Basement Membrane Matrix</td>
<td>Corning</td>
<td>Cat# 356231</td>
</tr>
<tr>
<td>Hoechst 33342 solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 62249</td>
</tr>
<tr>
<td>Propidium iodide (PI)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# P3566</td>
</tr>
<tr>
<td>
<sup>2</sup>eBioscience™ 1× RBC Lysis Buffer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 00-4333-57</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Millipore</td>
<td>Cat# 1029521000</td>
</tr>
<tr>
<td>DAPI</td>
<td>Sigma-Aldrich</td>
<td>Cat# D9542</td>
</tr>
<tr>
<td>BYL719 (alpelisib)</td>
<td>Selleckchem</td>
<td>Cat# S2814</td>
</tr>
<tr>
<td>DNase I</td>
<td>Sigma-Aldrich</td>
<td>Cat# DN25</td>
</tr>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rat monoclonal Alexa Fluor 488 anti-mouse CD31 (clone 390) (1:400)</td>
<td>BioLegend</td>
<td>Cat# 102414; RRID:AB_493408</td>
</tr>
<tr>
<td>Rat monoclonal Alexa Fluor 488 anti-mouse CD45 (clone 30F-11) (1:400)</td>
<td>BioLegend</td>
<td>Cat# 103122; RRID:AB_493531</td>
</tr>
<tr>
<td>Rat monoclonal Alexa Fluor 488 anti-mouse TER-119/Erythroid Cells (clone TER-119) (1:400)</td>
<td>BioLegend</td>
<td>Cat# 116215; RRID:AB_493402</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD49f (Integrin alpha 6) Monoclonal Antibody (clone eBioGoH3 (GoH3)), PE (1:800)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12-0495-82; RRID:AB_891474</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD326 (EpCAM) (clone G8.8), APC (1:400)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 17-5791-80; RRID:AB_2734965</td>
</tr>
<tr>
<td>Mouse monoclonal anti-Estrogen Receptor a (clone 1D5) (1:200)</td>
<td>Agilent</td>
<td>Cat# M704729-2; RRID:AB_2101946</td>
</tr>
<tr>
<td>Mouse monoclonal purified anti-Cytokeratin 8 (1:200)</td>
<td>BioLegend</td>
<td>Cat# 904804; RRID:AB_2616821</td>
</tr>
<tr>
<td>Rabbit polyclonal purified anti-Keratin 14 (1:200)</td>
<td>BioLegend</td>
<td>Cat# 905304; RRID:AB_2616896</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-Ki-67 (clone SP6) (1:200)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# MA5-14520; RRID:AB_10979488</td>
</tr>
<tr>
<td>Rabbit Monoclonal anti-Progesterone Receptor A/B (clone D8Q2J) (1:800)</td>
<td>Cell Signaling Technology</td>
<td>Cat# 8757;RRID: AB_2797144</td>
</tr>
<tr>
<td>Mouse Monoclonal anti-Actin, a-Smooth Muscle (clone 1A4) (1:1000)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 2547;RRID: AB_476701</td>
</tr>
<tr>
<td>Phalloidin-iFluor 555 Reagent (1:1000)</td>
<td>Abcam</td>
<td>Cat# ab176756</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>CellTiter-Glo® 3D Cell Viability Assay</td>
<td>Promega</td>
<td>Cat# G9681</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: <i>Pik3calatH1047R/+</i> line</td>
<td>
<a href="#bib15">Tikoo et al., 2012</a>;<a href="#bib20">Yip et al., 2020</a>
</td>
<td>n/a</td>
</tr>
<tr>
<td>Mouse: <i>PtenC124S/+</i> line</td>
<td><a href="#bib11">Papa et al., 2014</a></td>
<td>n/a</td>
</tr>
<tr>
<td>Mouse: <i>PtenC124S Pik3caH1047R- Tg(Krt8-cre/ERT2)17Blpn/J</i>
</td>
<td><a href="#bib20">Yip et al., 2020</a></td>
<td>n/a</td>
</tr>
</tbody>
</table>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Given the high sequences homology between human and murine EGF and FGF2, recombinant murine versions of these growth factors could be used as an alternative.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> R-spondin 1 conditioned medium (CM) can be used as an alternative to recombinant R-spondin 1. In our assays, we have used conditioned media generated by the Monash Organoid Program. Conditioned media are an economical and convenient way to produce reagents because they can be generated in large batches, and can be stored as frozen aliquots at −20°C (<a href="#bib5">Drost et al., 2016</a>). Activity of R-spondin 1 CM can be tested by comparing growth rates of organoid cultures grown under serial dilutions of newly generated condition medium, to rates of cultures maintained under defined concentrations of recombinant R-spondin 1 (<a href="#bib8">Jarde et al., 2018</a>).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> In our assays, we have used in-house made Red blood cell lysis buffer prepared as follows: 156 mM ammonium chloride, 0.1 mM EDTA and 12 mM sodium bicarbonate.</div>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Basal DMEM/F12 medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>GlutaMax (100×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin (5000 Unit) /Streptomycin (5 mg per mL)</td>
<td>100 U/mL / 100 μg/mL</td>
<td>10 mL</td>
</tr>
<tr>
<td>DMEM/F12 medium</td>
<td>1×</td>
<td>485 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>Digestion medium I</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Collagenase III (10,000 U/mL)</td>
<td>300 U/mL</td>
<td>0.3 mL</td>
</tr>
<tr>
<td>Trypsin (2.5%)</td>
<td>0.25%</td>
<td>1 mL</td>
</tr>
<tr>
<td>FBS (100%)</td>
<td>5%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Gentamicin (10 mg/mL)</td>
<td>10 μg/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>Basal DMEM/F12 medium</td>
<td>1×</td>
<td>8.19 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>Digestion medium II</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dispase II (10 mg/mL)</td>
<td>5 mg/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>DNase I (5 mg/mL)</td>
<td>0.5 mg/mL</td>
<td>1 mL</td>
</tr>
<tr>
<td>Basal DMEM/F12 medium</td>
<td>1×</td>
<td>4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>Organoid culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EGF (50 μg/mL)</td>
<td>20 ng/mL</td>
<td>4 μL</td>
</tr>
<tr>
<td>FGF2 (50 μg/mL)</td>
<td>20 ng/mL</td>
<td>4 μL</td>
</tr>
<tr>
<td>ITS-X (100×)</td>
<td>1×</td>
<td>100 μL</td>
</tr>
<tr>
<td>R-spondin1 CM (100%)</td>
<td>2.5%</td>
<td>400 μL</td>
</tr>
<tr>
<td>Y-27632 (10 mM)</td>
<td>10 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>Basal DMEM/F12 medium</td>
<td>1×</td>
<td>9.482 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>Staining medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Propidium iodide (1 mg/mL)</td>
<td>10 μg/mL</td>
<td>100 μL</td>
</tr>
<tr>
<td>Hoechst 33342 (12.3 mg/mL)</td>
<td>10 μg/mL</td>
<td>8 μL</td>
</tr>
<tr>
<td>Organoid culture medium</td>
<td>1×</td>
<td>9.892 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Digestion medium I and II, and staining medium are prepared fresh before use. Basal DMEM/F12 medium can be kept at 4°C for a month, but the Organoid culture medium must be made fresh every week to preserve stability and activity of critical reagents (e.g. Y-27632).</div>
<div class="note">
<span class="note-title">Note:</span> The CellTiter-Glo® 3D Cell Viability Assays is optimized to process 3D organoid cultures. However, the original CellTiter-Glo assay, generally used to process 2D cell cultures, could also be used to quantify cell viability of 3D organoid cultures. We refer the users to the manufacturer website (<a href="https://www.promega.com.au">https://www.promega.com.au</a>) for details on the application and use of the CellTiter-Glo assay for 3D cultures.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Mammary cell isolation</h3>
<h4 id="sec3.1.1">Day 1</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<div class="note">
<span class="note-title">Note:</span> To minimize loss of epithelial cells during isolation (e.g. pellet resuspension), we recommend using gentle vortexing instead of pipetting with serological pipettes. Also, pipette tips should be pre-wetted 3 times with a 5% serum-containing medium before pipetting mammary epithelial cells.</div>
<ol>
<li>Prepare Collagenase Type III, Dispase II and DNase I stock solutions as indicated in <a href="#sec1.1">preparation of reagents and solutions</a>.</li>
<li>Dissection of mammary fat pads:
<ol type="a">
<li>Euthanize one mouse at a time using laboratory standard procedures (e.g., carbon dioxide chamber), and as approved by the Institutional Animal Care and Use Committee (IACUC).</li>
<li>Generously spray the ventral side of the mouse with 70% ethanol and rub against the hair to ensure the skin is thoroughly wiped.</li>
<li>Transfer the mouse into a biosafety cabinet.</li>
<li>Place the mouse on a clean tray, and tape/pin the feet to expose the ventral side up.</li>
<li>Using clean tools cut one ventral midline from the inguinal nipples towards the thoracic nipples, along the skin and without cutting the peritoneum. Make two lateral incisions from the central midline towards each side of the mouse flanks, left and right, then lift and peel the skin to expose the inguinal (4<sup>th</sup> and 5<sup>th</sup> pairs) mammary fat pads (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Dissociation of mammary fad pads</div>
<p>(A) Anatomy of mammary fat pads (MFPs) from a C57BL/6J wild-type mouse, with the 4<sup>th</sup> right MFP displaying the inguinal lymph node (LN), encircled.</p>
<p>(B) A pair of MFPs (4<sup>th</sup> pair, left and right) is laid on a 100 mm dish and minced through vertical and horizontal cuts (black dotted lines) using a razor blade.</p>
<p>(C) Image showing appearance of optimally minced MFPs.</p>
<p>(D) Minced MFPs are enzymatically digested with Digestion medium I to release mammary fragments.</p>
<p>(E) After centrifugation, a pellet containing mammary fragments plus stromal tissue (white layer), and red blood cells (RBC, red layer) is visible at the bottom of the tube.</p>
<p>(F) After RBC lysis, a white cell pellet containing mammary fragments is collected. On the top right corner, references to experimental steps described in the main text.</p>
</figcaption>
</figure>
</li>
<li>Locate the lymph nodes within the mammary fat pads and use a pair of forceps to squeeze and remove them. This will minimize contamination from immune cells (See note).</li>
<li>Using clean scissors and tweezers, cut and remove the inguinal fat pads from each side of the mouse flank and place them in a 100 mm tissue culture dish, containing 5 mL of basal DMEM/F12 medium (+ 5% FBS), at room temperature. From now on, only use sterile and tissue-culture grade reagents and plastics (<a href="#fig1">Figure 1</a>B).</li>
<li>Quickly proceed with the remaining mice and collect all mammary fat pads up to this point.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Dissecting tools (scissors, tweezers, and forceps) should be autoclaved or left submerged in 100% ethanol for 30 minutes (min), in a tissue culture hood before use.</div>
<div class="note">
<span class="note-title">Note:</span> It is preferable to maintain mammary fat pads at 20°C–25°C in DMEM/F12 medium (+ 5% FBS) and not on ice, while collecting remaining samples. This will reduce thermal damage to cells and proteins caused by repeated changes in temperatures, i.e. from ∼37<sup>o</sup>C mouse body temperature to ice-temperature (∼4°C), and from ice-temperature to 37<sup>o</sup>C for enzymatic digestions.</div>
<div class="note">
<span class="note-title">Note:</span> All mammary fat pads can be used to purify mammary epithelial cells. However, inguinal mammary fat pads (4<sup>th</sup> and 5<sup>th</sup> pairs) have fewer lymph nodes and blood vessels than thoracic mammary fat pads, and are a preferable source of mammary epithelial cells.</div>
<ol start="3">
<li>Isolation of mammary epithelial fragments:
<div class="note">
<span class="note-title">Note:</span> The following steps are performed at 20°C–25°C in a tissue culture hood, unless otherwise specified, and refer to freshly collected tissue. Organoids can be also derived from frozen samples (<a href="#bib16">Walsh et al., 2016</a>).</div>
<ol type="a">
<li>Holding mammary fat pads with tweezers, dip each of them into a conical tube containing 30 mL of PBS-1× and rinse off hairs and blood, if present. Use fresh PBS-1× per mouse.</li>
<li>Place two pairs of fat pads from the same mouse on a fresh 100 mm dish, and mince the glands using a sterile razor blade. Hold the razor vertically and perform perpendicular cuts, alternating vertical and horizontal cuts (<a href="#fig1">Figure 1</a>C). Mince two pairs of fat pads at the time for 3 min, in order to obtain a homogeneous tissue mixture with minimal cellular damage (See <a href="#troubleshooting">troubleshooting</a>-problem 1 and <a href="#fig7">Figure 7</a>).</li>
<li>Transfer minced tissue from the same mouse into a 50 mL tube containing 10 mL of Digestion medium I; shake at 200 rpm in a 37°C orbital shaker for 40 min.</li>
<li>At the end of the digestion, vortex the tube for 5 s at medium speed, then transfer the content into a clean 15 mL tube (<a href="#fig1">Figure 1</a>D).
<div class="note">
<span class="note-title">Note:</span> Pre-warm the Digestion medium I in a 37°C waterbath. 10 mL of Digestion medium I are sufficient to digest and isolate epithelial cells from 4 mammary fat pads collected from one single mouse, e.g. 4<sup>th</sup>–5<sup>th</sup> pairs. Longer digestions (e.g. 5 additional min) may be needed for 129 mouse strains. Also, volume and times of digestion should be scaled according to the number of mammary fat pads digested.</div>
<div class="note">
<span class="note-title">Note:</span> Single mammary fat pads from C57BL/6J mice, irrespective of their anatomical location, contain a sufficient number of mammary epithelial cells (∼80,000 cells) to establish organoid lines. Larger mammary fat pads, e.g. 4<sup>th</sup> pair, can also be divided in two parts where one half can be used for mammary organoid extraction, and the other half to perform additional analyses, e.g. pathological assessment, immunostaining.</div>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Enrichment of mammary fragments:
<ol type="a">
<li>Centrifuge the 15 mL tube at 450 × <i>g</i> for 10 min. A white layer containing fat should be visible at the top (<a href="#fig1">Figure 1</a>E).</li>
<li>Transfer the supernatant, including the fat layer, into a new collecting tube, and keep aside.
<div class="note">
<span class="note-title">Note:</span> The supernatant can be re-digested in case of low fragments yield (See <a href="#troubleshooting">troubleshooting</a><b>-Problem 1</b>).</div>
</li>
<li>Tap and flick the centrifuged tube to dislodge the tissue pellet at the bottom.</li>
<li>Resuspend pellet in 5 mL of fresh basal DMEM/F12 medium (+ 5% FBS); this will also stop the enzymatic digestion.</li>
<li>Close the lid and invert the tube a few times to mix the solution.</li>
<li>Transfer 1 mL of tissue suspension into a well of a 24-well plate and observe the digested tissue under a bright field microscope (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative images of mammary fragments isolated from mammary fat pads (MFPs)</div>
<p>(A) Poor isolation of mammary fragments (black arrow heads) is obtained with insufficient mincing and/or digestion. Few mammary fragments are found interspersed in Digestion medium I together with few terminal end buds (red arrow heads) and undigested stroma (blue arrows).</p>
<p>(B) Bright field microscopy image showing complete MFP digestion after incubation with Digestion medium I. MFP are fully digested and mammary fragments released. Inset shows appearance of isolated mammary fragments.</p>
<p>Scale bar = 100 μm (A, B, zoom-in image).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Verify that at least hundreds fragments have been isolated after digestion I, i.e. with Digestion medium I (<a href="#fig2">Figure 2</a>), and before proceeding to the next step. See <a href="#troubleshooting">troubleshooting</a> if too few epithelial fragments (<b>Problem 1</b>) or large amounts of stromal fragments (<b>Problem 2</b>) are visible.</div>
</li>
<li>Transfer the 1 mL suspension of mammary fragments from the 24-well back into the centrifuge tube and combine with the remaining 4 mL of suspension.</li>
<li>Rinse the wells with 1 mL of basal DMEM/F12 medium (+5% FBS) to collect any residual fragments, and combine them with the 5 mL from step g.
<div class="note">
<span class="note-title">Note:</span> A layer of red blood cells (RBC) can be found at the bottom of the pellet. If no RBC are present, skip to <b>Step n</b>.</div>
</li>
<li>Centrifuge the tube at 450 × <i>g</i> for 5 min.</li>
<li>Aspirate and discard the supernatant.</li>
<li>Tap and flick the tube to dislodge the tissue pellet and expose the red blood cells.</li>
<li>Add 2 mL of RBC lysis buffer and incubate for 1 min at 20°C–25°C.</li>
<li>Add 8 mL of PBS-1× (+1% FBS) to dilute the RBC lysis buffer and pipet up and down a few times.
<div class="note">
<span class="note-title">Note:</span> A low-speed centrifugation (<b>Step 4.n</b>) will separate the dissociated stromal and dead cells at the top of the suspension, while enriching for the desired epithelial fragments in the white pellet at the bottom.</div>
</li>
<li>Centrifuge the tube at 200 × <i>g</i> for 1 min (<a href="#fig1">Figure 1</a><b>F</b>).</li>
<li>Aspirate and discard the supernatant.</li>
<li>Add 2 mL of TrypLE and digest the pellet containing fragments for 5 min in a 37°C waterbath.</li>
<li>Add 6 mL of PBS-1× (+ 1% FBS) to stop the enzymatic reactions, close the lid and invert the tube 3–4 times, and then spin at 450 × <i>g</i> for 5 min.</li>
<li>Aspirate supernatant and resuspend the pellet with 2 mL of Digestion medium II. Place the tube in a 37°C waterbath for 5 min.
<div class="note">
<span class="note-title">Note:</span> Pre-warm TrypLE and Digestion medium II in a 37°C waterbath prior use. 2 mL of Digestion medium II is enough for 2 pairs of mammary fat pads.</div>
<div class="note">
<span class="note-title">Note:</span> To aid cell dissociation, tap and invert the tube a few times every minute during the incubation Steps (p) and (r).</div>
<div class="note">
<span class="note-title">Note:</span> If single cell isolation is required, such as for flow cytometric analyses, extend the enzymatic digestion in Step (p) and (r) to 10 min. Additional steps are included as <i>Alternative</i><i>s</i> at the bottom of the section for samples preparation for flow cytometry.</div>
</li>
<li>Add 6 mL of 0.1% FBS in PBS and invert the tube 3–4 times. Centrifuge the sample at 450 × <i>g</i> for 5 min.</li>
<li>Aspirate supernatant and resuspend the cell pellet (containing aggregates) in 1 mL of Organoid culture medium.</li>
<li>Pipette 20 μL of aggregates suspension into a well containing 20 μL of trypan blue and count the numbers of cell aggregates using a hemocytometer.
<div class="note">
<span class="note-title">Note:</span> Mammary fragments obtained after enzymatic digestions (<b>Step 4.r</b>) are referred to as cell aggregates (&lt;10 cells).</div>
<div class="note">
<span class="note-title">Note:</span> On average ∼500,000 cell aggregates, between epithelial and stromal cells, are obtained from 4 fat pads (4<sup>th</sup> and 5<sup>th</sup> pairs) from a 10–12 weeks old female mouse.</div>
<div class="note">
<span class="note-title">Note:</span> High concentration of FBS in washing solutions can be detrimental to the viability of primary epithelial cells, therefore it is recommended to progressively reduce the concentration of FBS from 5% to 0.1% when resuspending cell pellets. A reduced percentage of FBS, i.e. 0.1%, is however still required to maximize cell recovery during centrifugations.</div>
</li>
<li>
<b>Alternatives:</b> To generate mammary organoids from fluorescent activated cell sorting (FACS)-sorted, single cell populations of mammary epithelial cells, i.e., luminal and basal, perform the following steps before plating:
<ol type="i">
<li>After step 4.s, resuspend cell pellet at 1 × 10<sup>6</sup> cells per 100 μL of FACS buffer (1% FBS in PBS-1×) and incubate at 20°C–25°C for 20 min with diluted fluorochrome-conjugated antibodies directed at lineage markers: CD31, CD45, TER119, EpCAM and CD49f in 1.5 mL tubes.</li>
<li>Wash cells with 1 mL of FACS buffer twice, centrifuge at 500 × <i>g</i> for 3 min.</li>
<li>Resuspend each cell pellet in 200 μL of FACS buffer containing 1 μg/mL DAPI in FACS compatible tubes. Samples are ready for cell sorting (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Organoids derived from FACS-sorted mammary epithelial cells. Mammary cells were dissociated following experimental steps included in this protocol, up to step 4.s. and steps in <i>A</i><i>lternatives</i>
</div>
<p>(A) Cells were then stained with EpCAM and CD49f to distinguish between luminal and basal epithelial cell populations. P1 gate (green) indicates epithelial cells, whereas red dots represent stromal cells.</p>
<p>(B) Sorted luminal and basal cells were counted, plated in Matrigel domes, and let grow into organoids. Representative images showing organoid colonies after 18 days of culture are provided. Three different types of organoid colonies can be identified and are generally referred to as solid colonies (left), cystic colonies with bright reflection (middle), and budding colonies with protruding cells (right) (<a href="#bib7">Jamieson et al., 2017</a>).</p>
<p>Scale bar = 50 μm.</p>
</figcaption>
</figure>
</li>
<li>Collect sorted cells and plate as described below.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Culturing of mammary epithelial cells in Matrigel</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<div class="note">
<span class="note-title">Note:</span> Mammary organoid colonies plated in 24-well plates will require ∼one-week time to become confluent before they can be passaged (<a href="#fig4">Figure 4</a>).</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Mammary organoid colonies at Day 6 of culture</div>
<p>(A) Image of murine mammary organoids ready to be passaged.</p>
<p>(B) Mammary organoid shown at higher magnification.</p>
<p>Scale bar = 200 μm (A) and 50 μm (B).</p>
</figcaption>
</figure>
<ol start="5">
<li>Coating of 24-well plates before plating:
<ol type="a">
<li>Thaw Matrigel (in a 4°C cold room on ice for 14–16 h, or on ice at 20°C–25°C for 1–2 h) (See <a href="#sec1.1">preparation of reagents and solutions</a>, point 4). 10 μL of Matrigel will be used to coat each well of a 24-well plate.</li>
<li>Wet pipette tips with pre-chilled PBS-1× and mix the Matrigel solution by pipetting up and down 4–5 times before use.</li>
<li>Open and place the required number of 24-well plates in the tissue culture hood.</li>
<li>Pipette 10 μL of Matrigel in the center of each well to form a flat layer of ∼1 cm in diameter, and then transfer them to a 37<sup>o</sup>C cell incubator for 10 min.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Matrigel coating layer will prevent mammary epithelial cells from adhering to the bottom of the plate where they would grow in 2D-monolayer. Regular tissue culture grade can be used instead of ultra-low attachment plates.</div>
<ol start="6">
<li>Plating mammary organoids: After counting the overall number of isolated cell aggregates, proceed with plating. Once plated, mammary organoids are maintained in a humidified cell incubator at 37<sup>o</sup>C and 5% CO<sub>2</sub>.
<ol type="a">
<li>For each well of a 24-well plate, pipette a volume of cell suspension containing 30,000 cell aggregates, and transfer them into a 1.5 mL centrifuge tube containing 9 times the volume of Matrigel. For example, 5 μL of cell suspension in 45 μL of Matrigel would generate a Matrigel dome of 50 μL in each well. Maximum size dome recommended for 24-wells is 100 μL.</li>
<li>To plate more than one well, prepare a master suspension by multiplying the volume of cell suspension for one single well by the number of wells intended to plate, plus 1 extra well to account for dead volume lost during pipetting. Transfer the cell suspension into a 1.5 mL centrifuge tube containing 9 times volume of Matrigel, pipette up and down to mix well and dispense the cell suspension in each pre-coated well. As an example, to plate 5 wells with a 50 μL dome, each containing 30,000 cells in 5 μL, consider pipetting 30 μL of cell suspension (5 μL × 6 wells) into 270 μL of Matrigel. Plate 50 μL in each pre-coated well.</li>
<li>Keep tubes containing Matrigel on ice until plating.
<div class="note">
<span class="note-title">Note:</span> The ratio between the volume of cell aggregates suspension (V<sub>C</sub>) containing 30,000 cell aggregates, and the volume of Matrigel (V<sub>M</sub>) should be maintained at 1:9 (V<sub>C</sub>:V<sub>M</sub>) in order to allow optimal growth of organoid colonies and to allow Matrigel to set. If cell recovery is low, and the suspension of aggregates is too diluted i.e. less than 30,000 cell aggregates per 5 μL medium, it is recommended to centrifuge the sample, aspirate off the supernatant, and resuspend the sample in smaller volumes.</div>
<div class="note">
<span class="note-title">Note:</span> Studies have shown that plating cell aggregates (&lt;10 cells), rather than single cell suspension, generates organoid colonies in a shorter time, as cell aggregates more rapidly adapt to the tissue culture environment than single cells do. Cell aggregates also display higher cell viability, possibly owing to the retention of positive cell-to-cell interactions (<a href="#bib9">Miyoshi and Stappenbeck, 2013</a>; <a href="#bib13">Sato et al., 2011</a>).</div>
</li>
<li>Take the pre-coated 24-well plate (step 5) from the incubator and place it in the tissue culture hood.</li>
<li>Pipette and mix the cell aggregates/Matrigel suspension a few times before plating.</li>
<li>Position the pipette vertically over the center of a well and pipette the Matrigel containing cells in the middle of the well. This should form a small Matrigel dome.</li>
<li>Plate as many wells as necessary repeating the steps above.</li>
<li>Gently transfer the 24-well plate to a 37<sup>o</sup>C cell incubator for 30 min to allow Matrigel to set.</li>
<li>Once set, take the 24-well plate back to the hood and gently overlay 1 mL of pre-warmed Organoid culture medium in each well, then return the plate to the 37<sup>o</sup>C cell incubator.</li>
<li>Change culture medium every 2 days. Avoid touching or disturbing the cell dome while pipetting the medium off, and carefully add the 1 mL of pre-warmed fresh medium (<a href="#fig5">Figure 5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Plating of mammary organoids</div>
<p>(A) Bright field microscopy image showing optimal density of mammary organoids plated in a 24 well (30,000 aggregates in 50 μL of Matrigel and Organoid Medium). At Day2 of culture, mammary organoids appear as small, single spheres.</p>
<p>(B) Image showing densely plated mammary organoids. In over-plated cultures, mammary organoids appear very close to each other and interfere with each other growth rates.</p>
<p>Scale bar = 400 μm.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> On the day of plating, the ROCK inhibitor Y-27632 is included in the medium to prevent anoikis. Loss of cell-to-cell interactions can induce death of primary epithelial cells after detachment from the gland basement membrane, i.e. anoikis (<a href="#bib14">Sato et al., 2009</a>; <a href="#bib17">Watanabe et al., 2007</a>).</div>
<div class="note">
<span class="note-title">Note:</span> See <a href="#troubleshooting">troubleshooting</a>-Problem 3 if 2D cells are visible at the bottom of the wells.</div>
</li>
<li>
<b><i>Alternative</i></b> <b>for tumor-derived cells:</b> The experimental steps described in this protocol can also be used to isolate and establish 3D-cultures of tumor epithelial cells (tumoroids) derived from mammary tumors (<a href="#fig6">Figure 6</a>) (<a href="#bib6">Duarte et al., 2018</a>). In our experimental settings, we have successfully generated mammary tumoroids from tumors driven by cancer mutations in the tumor suppressor PTEN and the proto-oncogene PI3K (<a href="#bib20">Yip et al., 2020</a>). For tumor samples:
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Generation of tumor-derived 3D organoids</div>
<p>Workflow describing experimental steps for the extraction of tumor cells, and plating of 3D mammary tumoroids. See text for details.</p>
</figcaption>
</figure>
<ol type="i">
<li>Process mice as in Step 2. Dissection of mammary fat pads.</li>
<li>Dissect out mammary tumors, place them in a 100 mm dish and remove normal-looking mammary fat pads.</li>
<li>Processing one sample at the time, use a razor blade to cut tumors in small portions and mince as described in Step 3. Isolation of mammary epithelial fragments, until it reaches a consistent texture (<a href="#fig7">Figure 7</a>). Optional: portions of un-minced samples can be snap-frozen in liquid nitrogen and stored at −80<sup>o</sup>C for further analyses (e.g., protein and DNA extraction).
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Images of minced tissues</div>
<p>Mammary fat pads from 2 months old wild-type mouse (top), and mammary tumor (bottom) from 6-month-old mPTEN/mPI3K mouse, were collected, placed on a 100 mm dish, and minced with razor blades. Insufficient tissue mincing results in non-homogeneous tissue appearance, where aggregates of fat tissue and blood vessels are still visible (middle figures). Optimal tissue mincing generates a homogenous suspension of fragments, where histological aggregates can no longer be identified (right). Scale bar = 1 cm.</p>
</figcaption>
</figure>
</li>
<li>Incubate minced tumor tissue in 10 mL of Digestion medium I for one hour in a 37°C orbital shaker at 200 rpm.</li>
<li>Centrifuge to collect cell pellet and rinse it once in 5 mL of DMEM/F12 + 5% FBS.</li>
<li>Collect pellet and follow Step 4.k to remove RBC, if present.</li>
<li>Digest each sample containing tumor fragments with 3 mL of TrypLE first, followed by a digestion with 3 mL of Digestion medium II, sequentially.</li>
<li>Resuspend each cell pellet in 5 mL of Organoid culture medium and follow from step 4.u for counting of tumor fragments.</li>
<li>Plate 30,000 cell aggregates in a 50 μL Matrigel dome; let the dome set and overlay 1 mL of Organoid culture medium.</li>
<li>Culture and passage as in step 7 (<a href="#fig8">Figure 8</a>).
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Tumor-derived mammary organoids at Day 4 of culture</div>
<p>(A) Bright field microscopy image of mammary organoids derived from tumors of mice harboring the loss-of-function PTEN C124S mutation plus the oncogenic PI3K H1047R mutation (mPTEN/mPI3K).</p>
<p>(B) A mPTEN/mPI3K tumor-derived organoid, tumoroid, at a higher magnification.</p>
<p>Scale bar = 400 μm (A) and 50 μm (B).</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Passaging of mammary organoids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Organoid cultures plated as fragments (30,000 cell aggregates per well of a 24-well plate) reach confluency in ∼6 days (<a href="#fig4">Figure 4</a>). A well in which &gt; 70% of organoid colonies has reached ∼ 200 μm in size should be considered confluent. At this stage, organoids need to be passaged and re-plated in fresh Matrigel. On average, organoids are passaged every week at 1:4 to 1:6 dilutions. One confluent well of a 24-well plate contains on approximately 200,000 cells, which can generate 60 μg of total protein lysates.</p>
<ol start="7">
<li>Passaging of mammary organoids:
<div class="note">
<span class="note-title">Note:</span> Thaw fresh aliquots of Matrigel on ice in a 4°C cold room for 14–16 hours, or on ice at 20°C–25°C for 1–2 hours.</div>
<ol type="a">
<li>Aspirate the culture medium without touching the cell dome, using a vacuum aspirator.</li>
<li>Add 1 mL of TrypLE to each confluent well and gently break the Matrigel by pipetting up and down 5–6 times.</li>
<li>Incubate the 24-well plate at 37°C in a cell incubator for 2 min.</li>
<li>Take the plate out and place in the tissue culture hood, and mechanically dissociate mammary organoids into smaller cell aggregates by pipetting several times to assist the enzymatic digestion.</li>
<li>Return the plate to the incubator for another 3 min of digestion.
<div class="note">
<span class="note-title">Note:</span> The majority of organoid colonies will be fully digested into aggregates (∼10 cells) using standard incubation time, i.e. 5 min. However, with large organoid colonies (&gt;300 μm in diameter), 1–2 additional min of incubation with TrypLE may be required. Monitor the progression of the digestion under the microscope to identify optimal incubation time. Do not digest organoids to single cells as this may affect cell viability and may extend time of recovery after plating. Use a 1-mL pipette tip to pipette the suspension and facilitate cell dissociation.</div>
</li>
<li>At the end of the digestion, transfer the cell suspension into a 15 mL tube and add 3 mL of 0.1% FBS in basal DMEM/F12 medium to stop the TrypLE.</li>
<li>Centrifuge the tubes at 450 × <i>g</i> for 5 min.</li>
<li>Aspirate the supernatant, resuspend the cell pellet in 1 mL of Organoid culture medium and count the number of cell aggregates.</li>
<li>Transfer 30,000 cell aggregates in Organoid culture medium to a 1.5 mL tube containing 9 volumes of Matrigel (See step 6. Plating mammary organoids).</li>
<li>Plate the dome of cell aggregates in the middle of a well in a fresh 24-well plate, pre-coated with Matrigel (See step 5 for the coating steps).
<div class="note">
<span class="note-title">Note:</span> The frequency with which organoids are passaged should be determined empirically based on the growth rates of the newly established organoids. This may vary depending on the genetic background of the cells, as well as on the presence of oncogenic mutations, which may increase growth rates.</div>
<div class="note">
<span class="note-title">Note:</span> Mammary organoids generated with this protocol maintain relevant molecular and lineage markers (<a href="#fig9">Figure 9</a>) and can be passaged and used for assays until passage 5–6. See <a href="#limitations">limitations</a> section for details.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Molecular and lineage characterization of mammary organoids</div>
<p>(A) Formalin-fixed paraffin-embedded (FFPE) sections of wild-type mammary organoids (5 μm) were generated and immuno-stained at passage 5 with the indicated markers: Ki67, proliferative marker; Estrogen Receptor (ER) and Progesterone Receptor (PR), markers of luminal epithelial cells. Arrows point to positive stains.</p>
<p>(B) Immunofluorescence showing presence of luminal epithelial cells in wild-type and PTEN<sup>CS/+</sup> mutant mammary organoids indicated by positivity to keratin 8 (red); the presence of basal cells is indicated by the keratin 14 (white) and smooth muscle actin (SMA)-alpha (white) positive staining. Phalloidin (red) and DAPI (blue) are used as counterstains for F-actin and DNA, respectively. SMA-alpha, phalloidin and DAPI stains were generated as 3D whole mount staining and image acquired with a Lecia SP8 confocal microscope.</p>
<p>Scale bar = 50 μm.</p>
</figcaption>
</figure>
<h3 id="sec3.4">Cryopreservation of mammary organoids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> for freezing/thawing: 10 min per well of a 24-well plate</div>
<div class="note">
<span class="note-title">Note:</span> From the time of initial plating, mammary organoids derived from crude preparation will require 1-week time to grow and become confluent, before they can be frozen. Confluent wells can also be processed to generate formalin-fixed paraffin embedded (FFPE) blocks (<a href="#bib6">Duarte et al., 2018</a>), which can be sectioned and used to determine the cellular composition of organoid colonies through immunocytochemistry or immunofluorescence assays (<a href="#fig9">Figure 9</a>).</div>
<ol start="8">
<li>Freezing organoid cultures
<ol type="a">
<li>Prepare organoid freezing medium by making up a solution of 90% FBS and 10% Dimethyl Sulfoxide (DMSO), at room temperature.</li>
<li>Set a cryopreservation container filled with propan-2-ol aside, at 20°C–25°C.</li>
<li>Gently aspirate the medium off the 24-well plates containing confluent organoids and dissociate organoids into cell aggregates as in step 7.a to 7.g.</li>
<li>After centrifugation, resuspend each cell pellet in 0.5 mL of freezing medium and transfer it into a cryovial.</li>
<li>Transfer the cryopreservation container with the cryovials to a −80°C freezer.</li>
<li>The day after, transfer cryovials into a liquid nitrogen tank for long-term storage.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A confluent well of 24-well plate organoids can be diluted 1:3 and frozen into 3 cryovials. If desired, under-confluent wells containing smaller organoid colonies (<a href="#fig5">Figure 5</a><b>A</b>) can be mechanically dissociated using a 1 mL pipette tip, and frozen without trypsinization or dilution.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to freeze organoids as aggregates and not as single cells as this will generate more rapid recovery and higher cell viability.</div>
<h3 id="sec3.5">Recovery of frozen mammary organoids in culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min per well of a 24-well plate</div>
<ol start="9">
<li>Thawing of cryopreserved organoids
<div class="note">
<span class="note-title">Note:</span> Thaw fresh aliquots of Matrigel on ice in a 4<i>°</i>C cold room for 14–16 hours, or on ice at 20°C–25<i>°</i>C for 1–2 hours. The following steps describe the procedure to thaw a single cryovial.</div>
<ol type="a">
<li>Warm the Organoid culture medium in a 37°C waterbath for 10 min, then place it in the hood.</li>
<li>Place the cryovial containing frozen organoids in a 37°C waterbath for 2–3 min, or until almost fully thawed.</li>
<li>Transfer the 0.5 mL of organoids in freezing medium into a 15 mL tube containing 5 mL of basal DMEM/F12 medium.</li>
<li>Centrifuge at 450 × <i>g</i> for 5 min.</li>
<li>Aspirate supernatant and leave a layer of basal DMEM/F12 medium to cover the pellet.</li>
<li>Resuspend the cell pellet in a solution containing 1 part of Organoid culture medium and 9 parts of Matrigel.</li>
<li>Plate between 50 and 100 μL of organoids/Matrigel mixture per well, as described in step 6. Plate content from one cryovial in three wells (<a href="#fig10">Figure 10</a>).
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Recovery of mammary organoids from cryopreservation</div>
<p>Appearance of small mammary organoids derived from cryopreserved fragments, after 2 days of culture. Scale bar = 200 μm.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.6">Growth assays of mammary organoids in 96-well plate format</h3>
<div class="timing">
<span class="timing-title">Timing:</span> for plating a whole 96-well plate: 1.5 h</div>
<p>The following protocol describes steps optimized to study growth rates of mammary organoids derived from mouse models with single or compound mutations in PTEN and PI3K (<a href="#fig11">Figures 11</a> and <a href="#fig12">12</a>). This protocol has also been used to test sensitivity to inhibitors directed at components of the PI3K pathway (Please refer to <a href="#bib20">Yip et al., 2020</a>). Briefly, the day after plating, mammary organoid lines were treated with various PI3K and AKT inhibitors, at different concentrations, for 4 days, and drugs were refreshed every two days. At the end of each treatment, growth rate of mammary organoids were processed with the Promega CellTiter –Glo 3D assay, and according to the <a href="https://www.promega.com.au/-/media/files/resources/protocols/technical-manuals/101/celltiter-glo-3d-cell-viability-assay-protocol.pdf?rev=88083aa3f7284e898ff0f218aa3c6b59&amp;sc_lang=en">manufacturer’s instructions</a>.</p>
<figure id="fig11"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig11.jpg" alt="Fig11.jpg">
<figcaption>
<div class="figcaption-title">Figure 11. Cell growth assay of PTEN-wild-type and PTEN-mutant mammary organoids</div>
<p>PTEN wild-type (PTEN<sup>+/+</sup>) and PTENC124S mutant (PTEN<sup>CS/+</sup>) mammary organoids were plated in 96-well plates and grown for 6 days in Organoid culture medium.</p>
<p>(A) Images showing PTEN<sup>+/+</sup> and PTEN<sup>CS/+</sup> mammary organoids at Day 6 of culture. Scale bar = 50 μm.</p>
<p>(B) At the end of the experiment, cell numbers were quantified using the CellTiter-Glo 3D, and growth rates presented as relative fold changes over Day 0. Data represent mean values with individual data points ± standard deviation.</p>
</figcaption>
</figure>
<figure id="fig12"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig12.jpg" alt="Fig12.jpg">
<figcaption>
<div class="figcaption-title">Figure 12. Cell growth assay of mammary organoids under drug treatment</div>
<p>Mammary organoid lines were generated from wild-type and mutant mice: PTEN<sup>CS/+</sup> mouse, K8-PI3K<sup>HR/+</sup> mouse, and PTEN<sup>CS/+</sup>; K8-PI3K<sup>HR/+</sup> mutant mouse. The PTENC124S mutation is a loss-of-function mutation that inhibits PTEN catalytic function. In K8-PI3K<sup>HR/+</sup> mice, the PI3KH1047R mutation is driven by the tamoxifen-induced Cre-recombinase under the <i>cytokeratin</i> <i>8</i> <i>promoter</i>, <i>K8-CreERT2,</i> (K8-PI3K<sup>HR/+</sup>).</p>
<p>(A) Image of organoids plated in 96-well plate and grown in Organoid medium for 4 days. Scale bar = 1000 μm.</p>
<p>(B) Mammary organoids with the indicated genetic mutations were plated in 96-wells and treated for 4 days with increasing concentrations of the p110α-specific PI3K inhibitor BYL719. Response to treatment was assessed using the CellTiter-Glo 3D assay, and values normalized over untreated wells, within the same genotype. Data represent mean values with individual data points ± standard deviation shown as triplicate wells.</p>
</figcaption>
</figure>
<ol start="10">
<li>Plating of mammary organoids in 96-well plates for growth assay
<div class="note">
<span class="note-title">Note:</span> Thaw fresh aliquots of Matrigel on ice in a 4<i>°</i>C cold room for 14–16 hours, or on ice at 20°C–25°C for 1–2 hours.</div>
<ol type="a">
<li>Pre-coat each of the required wells of a 96-well plate with 5 μL of Matrigel and incubate the plate in a 37°C incubator for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> Do not exceed 5 min incubation as this would dry out the coating layer and prevent an even plating of the cell/Matrigel suspension. After 5 min incubation, leave the plate in the cell culture hood, at room temperature.</div>
<div class="critical">
<span class="critical-title">Critical:</span> When using 96-well plates, avoid plating cells in the outer wells to circumvent the “edge effect” and evaporation of culturing medium. It is suggested to fill the empty wells with 200 μL of sterilised water to minimize loss of volume in experimental wells.</div>
</li>
<li>Take the 24-well plate containing the organoid cultures and proceed to count the number of cell aggregates as in step 7.</li>
<li>Prepare a cell/Matrigel suspension with a concentration of 8000 cell aggregates per 20 μL total volume.</li>
<li>Using a P-20 tip, transfer 20 μL of cell/Matrigel mixture into each well of a 96-well plate.</li>
<li>Gently and slowly pipette the 20 μL of cell/Matrigel suspension right at the center of the well, on top of the Matrigel coating layer. The suspension will spread evenly within a few seconds.</li>
<li>Leave the 96-well plate at 37<sup>o</sup>C in a cell incubator to set for 20 min.</li>
<li>Once the Matrigel has set, take the 96-well plate back to the hood and overlay 200 μL of Organoid culture medium per well.</li>
<li>Incubate the 96-well plate at 37<sup>o</sup>C in a cell incubator for 6 days and refresh Organoid culture medium every 2 days (<a href="#fig11">Figure 11</a>A).</li>
<li>Measure growth of organoid colonies (e.g., CellTiter-Glo 3D assay) (<a href="#fig11">Figure 11</a>B).
<div class="optional">
<span class="optional-title">Optional:</span> (j-m): Growth assay in response to drug treatments.</div>
The day after plating (step 10.g) gently aspirate the 200 μL of Organoid culture medium off the well without disturbing the cell dome.</li>
<li>Prepare a master mix containing the drug concentration to be tested diluted in Organoid culture medium. Additional wells treated with vehicles should be included as controls.</li>
<li>Pipette the Organoid culture medium off and add 200 μL of master mix in each well of a 96-well plate.</li>
<li>Incubate the 96-well plate at 37<sup>o</sup>C in a cell incubator for 4 days
<div class="note">
<span class="note-title">Note:</span> treatment can be refreshed every 2 days.</div>
</li>
<li>At the end of the experiment, measure growth of organoid colonies in response to treatment (e.g., CellTiter-Glo 3D assay) (<a href="#fig12">Figure 12</a>).
<div class="note">
<span class="note-title">Note:</span> The number of mammary organoids plated for drug treatment can vary depending on the growth rates of each experimental organoid line, and also on the length of the treatment. The optimal numbers of organoids to be plated should be tested and determined empirically. Also see <a href="#troubleshooting">troubleshooting</a><b>-Problem 4</b> for inconsistent results across technical replicates.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> To test response of mammary organoids to drug treatments, reduced Matrigel concentration; 2%–5% Matrigel in growth medium can be used to limit usage of reagents (<a href="#bib18">Wrenn et al., 2020</a>; <a href="#bib22">Zhang et al., 2016</a>).</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.7">Cell death assay of mammary organoids using propidium iodide (PI) and hoechst 33342 staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min</div>
<p>The following protocol can be utilized to quantify percentage of dead cells in mammary organoids using fluorescent dyes, such as propidium iodide (PI) and Hoechst 33342 staining. The experimental steps have been optimized for 96-well organoid cultures, and the percentage of cell death (PI positive cells) is normalized over the number of live cells (Hoechst 33342 positive cells). Any fluorescence plate readers equipped with appropriate filters (indicated below) can be used to measure PI and Hoechst 33342 signals.</p>
<ol start="11">
<li>Cell death quantification of mammary organoid cultures
<div class="note">
<span class="note-title">Note:</span> It is recommended to use clear-bottom black wall microplates for fluorescence reading to minimize signal interference from neighbouring wells.</div>
<div class="note">
<span class="note-title">Note:</span> For PI staining, plate extra wells of organoids as positive control. Before staining, treat the additional well with 0.1% Triton X-100 diluted in Organoid culture medium for 1 min. Stain the positive control and the experimental samples at once. Triton X-100 is a non-ionic surfactant that permeabilizes the plasma membrane and exposes the DNA, thus facilitating PI labelling.</div>
<ol type="a">
<li>Following drug treatment, or at the end of a growth assay experiment, replace the Organoid culture medium with 200 μL of staining medium (See <a href="#materials-and-equipment">materials and equipment</a> for preparation).</li>
<li>Place the organoids in a 37°C cell incubator and let them stain for 30 min.</li>
<li>Remove staining medium without disturbing the cell dome and add 200 μL of PBS-1×.</li>
<li>Measure the fluorescent PI signal using a multi-mode microplate reader (e.g., CLARIOstar PLUS microplate reader).
<ol type="i">
<li>Adjust focal height.</li>
<li>Measure fluorescence signals using the following parameters: excitation and emission wavelengths for PI: 535 nm and 617 nm; and excitation and emission wavelengths for Hoechst 33342: 361 nm and 486 nm.</li>
</ol>
<ol start="3" type="i">
<li>Measure PI first and Hoechst 33342 next (<a href="#fig13">Figure 13</a>).
<figure id="fig13"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig13.jpg" alt="Fig13.jpg">
<figcaption>
<div class="figcaption-title">Figure 13. Cell death assay of mammary organoids</div>
<p>Wild-type mammary organoids were cultured in Organoid culture medium and supplemented with increasing concentrations of noggin conditioned medium for 6 days. Effect on cell death was assayed as described in step 11, using propidium iodide (PI) and Hoechst33342 stain. Wells treated with 0.1% Triton-X 100 in Organoid culture medium were used as positive control. Data revealed no significant difference in the ratio of dead versus live cells upon noggin administration, indicating no beneficial effect on organoids survival upon noggin addition. a.u. = arbitrary unit. Data represent mean ± standard deviation of triplicate wells.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> PI and Hoechst 33342 are both DNA dyes. Live cells with intact cell membranes are impermeable to PI but permeable to Hoechst 33342. Therefore, the relative intensity of the dyes can be used as a way to discriminate between dead cells and live cells (See <a href="#troubleshooting">troubleshooting</a><b>-Problem 5</b>).</div>
<div class="note">
<span class="note-title">Note:</span> Using this cell death protocol, mammary organoids can be sequentially processed with the CellTiter-Glo 3D assay to also measure growth rates, if desired.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>We have provided step-by-step protocols suitable for the extraction of mammary epithelial cells from young and aged female mice, and have included steps that allow the successful plating and expansion of mammary organoid lines. In vitro cultures of mammary organoids maintain critical features that recapitulate morphologic structures of a mammary gland, as confirmed by the positivity to lineage and molecular markers detected through immuno-staining (<a href="#fig9">Figure 9</a>)(<a href="#bib7">Jamieson et al., 2017</a>; <a href="#bib10">Nguyen-Ngoc et al., 2015</a>). Following these experimental procedures, we have established several mammary organoid lines with a success rate of 100%, using both wild type and cells derived from mouse models with various cancer-associated mutations in PTEN and/or PI3K (<a href="#fig3">Figure 3</a>, <a href="#fig4">4</a>, and <a href="#fig8">8</a>). By passaging and expanding organoid cultures using our protocols, it is possible to collect total protein lysates for Western blot analyses, and to also perform imaging studies through <i>in situ</i> immunofluorescence of fixed organoids, or immunohistochemistry on paraffin-embedded organoids (see <a href="#fig9">Figure 9</a>)(<a href="#bib2">Cheung et al., 2013</a>; <a href="#bib12">Sachs et al., 2018</a>; <a href="#bib20">Yip et al., 2020</a>). These techniques allow a comprehensive characterization of the molecular properties of newly established mammary epithelial cells with distinct genetic alterations, and cultured in a 3D-setting, and can be used to monitor the status of clinically-relevant molecular markers (e.g., ER, <a href="#fig9">Figure 9</a>), over time. The generation and exploitation of 3D mammary organoid cultures can provide an important avenue for the identification of more effective anti-cancer treatments (<a href="#bib21">Yip and Papa, 2021</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>For cell growth and cell death assays of mammary organoid cultures, raw data of triplicate wells were averaged, and standard deviations represented by error bars calculated with GraphPad Prism 8. Graphs and data panels were generated using GraphPad Prism 8.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Mammary organoids can be passaged several times (4–5 times over few weeks) thus allowing expansion and generation of a number of cells sufficient to perform multiple functional assays. However, after initial passages (up to passage 5), a significant decrease in the proliferative capacity of organoid lines can occur. In <a href="#fig14">Figure 14</a>, we show this condition in growth assays of wild-type and PTEN<sup>C124S/+</sup> mutant mammary organoids, cultured over 9 passages. To overcome this loss of proliferation, and exhaustion of organoid lines, laboratories working on intestinal organoids have proposed adding cytokines to their organoid cultures (<a href="#bib14">Sato et al., 2009</a>), in particular noggin, a bone-morphogenetic protein 4 (BMP4) inhibitor secreted by mesenchymal cells in the intestine. Noggin is also added to the culture medium of human mammary organoids (<a href="#bib12">Sachs et al., 2018</a>). Yet, the role of BMP4 signaling in mammary epithelial cells is still unclear and contrasting results have been reported (<a href="#bib3">Choi et al., 2019</a>; <a href="#bib19">Yan et al., 2021</a>). The addition of noggin to the organoid culture medium may help reduce cell death while sustaining self-renewal capacity of stem cells in vitro, thus extending the number of cell passaging. In testing the growth of mammary organoids cultured with or without the addition of noggin conditioned medium, in our experiments we have found no significant improvements in terms of cell viability over time, i.e., dead vs live cells, in our mammary organoids (<a href="#fig13">Figure 13</a>). Because of this, we have not included noggin in our Organoid culture medium.</p>
<figure id="fig14"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/959-Fig14.jpg" alt="Fig14.jpg">
<figcaption>
<div class="figcaption-title">Figure 14. Growth rates of wild-type and mutant mammary organoids</div>
<p>A total of 7,000 mammary epithelial cells (P1) were cultured in 96-well plates for 7 days followed by trypsinization and re-plating into new 96-wells (7,000 cells, per well; P2). Serial passages were performed and cell numbers counted at day 7 of culture using a hemocytometer. The reduction in the proliferative potential observed after passage 4 (P4) indicates that a loss of stem/progenitor cell populations may have occurred. Representative images of PTEN<sup>+/+</sup> mammary organoids at P2 and P8 are shown. Scale bar = 400 μm. Each datapoint represents a single well.</p>
</figcaption>
</figure>
<p>In this protocol, mammary organoid cultures are established from crude extraction of epithelial cells from mouse mammary glands. Contamination from other cell types such as neuronal and stromal cells can be observed during the initial plating, Passage 0, P0. However, based on immunofluorescence staining using keratin 8 and keratin 14, markers of luminal and basal epithelial cells, only mammary epithelial cells successfully proliferate through passage 1. It is therefore recommended not to use cells at Passage 0 for functional assays. Alternatively, mammary epithelial cells can be extracted from mammary glands, FACS-sorted (see step 4.r, plus Notes and <i>A</i><i>lternatives</i>), and then plated in Matrigel. FACS-sorted mammary epithelial cells will require longer time (∼2 weeks) than mammary epithelial cell aggregates to form organoids of comparable sizes.</p>
<p>The cell death assay presented in this protocol is a rapid and cost-effective way to detect death of organoid cells (<a href="#bib1">Bode et al., 2019</a>). However, propidium iodide (PI) can only detect dying cells with a compromised plasma membrane. Using this approach, early cell death events, where plasma membrane is still intact, will remain undetected. As an alternative, commercially available apoptosis kit e.g., <a href="https://www.promega.com.au/products/cell-health-assays/apoptosis-assays/caspase-glo-3-7-3d-assay/?catNum=G8981">Caspase-Glo 3/7 3D assay</a> (Promega) can be used to detect early apoptotic markers.</p>
<p>The volume of Matrigel used to perform drug screen with organoids can scale up quickly and become costly. As an alternative, organoids suspensions in low Matrigel concentration (e.g., 2%), diluted in growth medium, have been used and could be considered as an option to reduce usage of reagents and associated costs (<a href="#bib18">Wrenn et al., 2020</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low yield of isolated mammary epithelial fragments (See step 4.f).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Low yield of mammary fragments can be observed due to incomplete digestion of mammary fat pads. (step 4.f). To overcome this, it is recommended to recover the samples after digestion I, (step 4.b) and repeat the incubation step with the same Digestion medium I for 10 additional min at 37<sup>o</sup>C, without adding extra Digestion medium I. After centrifugation, remove the supernatant, combine the new cell pellet with the previous one and proceed with step 4.g.</p>
<p>It is possible that an inefficient mincing of the mammary fat pads (step 3.b) also contributes to the low release of mammary epithelial fragments from the mammary gland (<a href="#fig2">Figures 2</a> and <a href="#fig7">7</a>). Insufficient tissue mincing can leave epithelial cells trapped in large fragments surrounded by stromal tissue, which impairs the subsequent enzymatic digestions. If the yield of isolated mammary epithelial fragments is low, even after two rounds of digestion with Digestion medium I, it is recommended to extend the tissue mincing from 3 to 5 min. A prolonged mincing step beyond 5 min is however not suggested as this may decrease the overall cell viability.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Large amounts of stromal fragments among mammary fragments (See step 4.f).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Stromal fragments (e.g., nerve fragments) are usually larger than epithelial fragments and can be removed using a 100 μm cell strainer to enrich for mammary epithelial fragments.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>2D-cell growth contamination of 3D-organoid cultures (See step 6.j).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>If the mammary organoid fragments are plated too densely, or become over-confluent, it is possible to observe that some epithelial cells migrate through the Matrigel coating layer and attach to the bottom of the plate, where they grow in 2D-monolayer. In this instance, it is recommended to disrupt the culture by pipetting the Organoid culture medium multiple times with a P-1000 pipette in order to break the Matrigel dome. It is important to avoid touching and scraping the bottom of the well to avoid resuspending the 2D cell layer. Transfer the mixture of organoids, Matrigel and medium into a 15 mL tube containing 5 mL of 0.1% FBS in basal DMEM/F12 medium. Centrifuge the suspension at 450 × <i>g</i> for 5 min and aspirate the supernatant. Resuspend the cell pellet in 300 μL with a solution containing 1 part of Organoid culture medium and 9 parts of Matrigel, and plate on Matrigel-coated plates as three cell domes, in three wells.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>High variability in the measurement of growth assays across replicates.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>When using plate readers for the measurement of cell growth rates, an incomplete lysis of organoid colonies can cause variability and inconsistency across wells of technical replicates. To avoid this, after adding the lysis buffer, place the 24-well plate on a bi-directional orbital shaker and shake the organoid colonies (e.g., 700 rpm for 5 min with a CLARIOstar PLUS plate reader). Avoid lysing organoids by pipetting up and down with pipette tips as this may generate air bubbles which will interfere with the reading of the luminescence signal.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Low cell death reading.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>It is recommended to use a positive control e.g., Triton X-100 (see step 11, Note 2) to empirically determine gain and technical parameters of the plate reader and to establish a positive signal-to-noise ratio. 30 min incubation with PI and Hoechst 33342 are sufficient to stain organoids in Matrigel.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bode, K.J., Mueller, S., Schweinlin, M., Metzger, M., and Brunner, T. (2019). A fast and simple fluorometric method to detect cell death in 3D intestinal organoids. Biotechniques <i>67</i>, 23-28. <a class="external-link" href="https://doi.org/10.2144/btn-2019-0023">https://doi.org/10.2144/btn-2019-0023</a></p>
<p id="bib2">Cheung, K.J., Gabrielson, E., Werb, Z., and Ewald, A.J. (2013). Collective invasion in breast cancer requires a conserved basal epithelial program. Cell <i>155</i>, 1639-1651. <a class="external-link" href="https://doi.org/10.1016/j.cell.2013.11.029">https://doi.org/10.1016/j.cell.2013.11.029</a></p>
<p id="bib3">Choi, S., Yu, J., Park, A., Dubon, M.J., Do, J., Kim, Y., Nam, D., Noh, J., and Park, K.S. (2019). BMP-4 enhances epithelial mesenchymal transition and cancer stem cell properties of breast cancer cells via Notch signaling. Sci Rep. <i>9</i>, 11724. <a class="external-link" href="https://doi.org/10.1038/s41598-019-48190-5">https://doi.org/10.1038/s41598-019-48190-5</a></p>
<p id="bib4">Dekkers, J.F., van Vliet, E.J., Sachs, N., Rosenbluth, J.M., Kopper, O., Rebel, H.G., Wehrens, E.J., Piani, C., Visvader, J.E., Verissimo, C.S., et al. (2021). Long-term culture, genetic manipulation and xenotransplantation of human normal and breast cancer organoids. Nat. Protoc. <i>16</i>, 1936-1965. <a class="external-link" href="https://doi.org/10.1038/s41596-020-00474-1">https://doi.org/10.1038/s41596-020-00474-1</a></p>
<p id="bib5">Drost, J., Karthaus, W.R., Gao, D., Driehuis, E., Sawyers, C.L., Chen, Y., and Clevers, H. (2016). Organoid culture systems for prostate epithelial and cancer tissue. Nat. Protoc. <i>11</i>, 347-358. <a class="external-link" href="https://doi.org/10.1038/nprot.2016.006">https://doi.org/10.1038/nprot.2016.006</a></p>
<p id="bib6">Duarte, A.A., Gogola, E., Sachs, N., Barazas, M., Annunziato, S., de Ruiter, J.R., Velds, A., Blatter, S., Houthuijzen, J.M., van de Ven, M., et al. (2018). BRCA-deficient mouse mammary tumor organoids to study cancer-drug resistance. Nat. Methods <i>15</i>, 134-140. <a class="external-link" href="https://doi.org/10.1038/nmeth.4535">https://doi.org/10.1038/nmeth.4535</a></p>
<p id="bib7">Jamieson, P.R., Dekkers, J.F., Rios, A.C., Fu, N.Y., Lindeman, G.J., and Visvader, J.E. (2017). Derivation of a robust mouse mammary organoid system for studying tissue dynamics. Development <i>144</i>, 1065-1071. <a class="external-link" href="https://doi.org/10.1242/dev.145045">https://doi.org/10.1242/dev.145045</a></p>
<p id="bib8">Jarde, T., Kerr, G., Akhtar, R., and Abud, H.E. (2018). Modelling intestinal carcinogenesis using in vitro organoid cultures. Methods Mol. Biol. <i>1725</i>, 41-52. <a class="external-link" href="https://doi.org/10.1007/978-1-4939-7568-6_4">https://doi.org/10.1007/978-1-4939-7568-6_4</a></p>
<p id="bib9">Miyoshi, H. and Stappenbeck, T.S. (2013). In vitro expansion and genetic modification of gastrointestinal stem cells in spheroid culture. Nat. Protoc. <i>8</i>, 2471-2482. <a class="external-link" href="https://doi.org/10.1038/nprot.2013.153">https://doi.org/10.1038/nprot.2013.153</a></p>
<p id="bib10">Nguyen-Ngoc, K.V., Shamir, E.R., Huebner, R.J., Beck, J.N., Cheung, K.J., and Ewald, A.J. (2015). 3D culture assays of murine mammary branching morphogenesis and epithelial invasion. Methods Mol. Biol. <i>1189</i>, 135-162. <a class="external-link" href="https://doi.org/10.1007/978-1-4939-1164-6_10">https://doi.org/10.1007/978-1-4939-1164-6_10</a></p>
<p id="bib11">Papa, A., Wan, L., Bonora, M., Salmena, L., Song, M.S., Hobbs, R.M., Lunardi, A., Webster, K., Ng, C., Newton, R.H., et al. (2014). Cancer-associated PTEN mutants act in a dominant-negative manner to suppress PTEN protein function. Cell <i>157</i>, 595-610. <a class="external-link" href="https://doi.org/10.1016/j.cell.2014.03.027">https://doi.org/10.1016/j.cell.2014.03.027</a></p>
<p id="bib12">Sachs, N., de Ligt, J., Kopper, O., Gogola, E., Bounova, G., Weeber, F., Balgobind, A.V., Wind, K., Gracanin, A., Begthel, H., et al. (2018). A living biobank of breast cancer organoids captures disease heterogeneity. Cell <i>172</i>, 373-386 e310. <a class="external-link" href="https://doi.org/10.1016/j.cell.2017.11.010">https://doi.org/10.1016/j.cell.2017.11.010</a></p>
<p id="bib13">Sato, T., van Es, J.H., Snippert, H.J., Stange, D.E., Vries, R.G., van den Born, M., Barker, N., Shroyer, N.F., van de Wetering, M., and Clevers, H. (2011). Paneth cells constitute the niche for Lgr5 stem cells in intestinal crypts. Nature <i>469</i>, 415-418. <a class="external-link" href="https://doi.org/10.1038/nature09637">https://doi.org/10.1038/nature09637</a></p>
<p id="bib14">Sato, T., Vries, R.G., Snippert, H.J., van de Wetering, M., Barker, N., Stange, D.E., van Es, J.H., Abo, A., Kujala, P., Peters, P.J., and Clevers, H. (2009). Single Lgr5 stem cells build crypt-villus structures in vitro without a mesenchymal niche. Nature <i>459</i>, 262-265. <a class="external-link" href="https://doi.org/10.1038/nature07935">https://doi.org/10.1038/nature07935</a></p>
<p id="bib15">Tikoo, A., Roh, V., Montgomery, K.G., Ivetac, I., Waring, P., Pelzer, R., Hare, L., Shackleton, M., Humbert, P., and Phillips, W.A. (2012). Physiological levels of Pik3ca(H1047R) mutation in the mouse mammary gland results in ductal hyperplasia and formation of ERalpha-positive tumors. PLoS One <i>7</i>, e36924. <a class="external-link" href="https://doi.org/10.1371/journal.pone.0036924">https://doi.org/10.1371/journal.pone.0036924</a></p>
<p id="bib16">Walsh, A.J., Cook, R.S., Sanders, M.E., Arteaga, C.L., and Skala, M.C. (2016). Drug response in organoids generated from frozen primary tumor tissues. Sci. Rep. <i>6</i>, 18889. <a class="external-link" href="https://doi.org/10.1038/srep18889">https://doi.org/10.1038/srep18889</a></p>
<p id="bib17">Watanabe, K., Ueno, M., Kamiya, D., Nishiyama, A., Matsumura, M., Wataya, T., Takahashi, J.B., Nishikawa, S., Nishikawa, S., Muguruma, K., and Sasai, Y. (2007). A ROCK inhibitor permits survival of dissociated human embryonic stem cells. Nat. Biotechnol. <i>25</i>, 681-686. <a class="external-link" href="https://doi.org/10.1038/nbt1310">https://doi.org/10.1038/nbt1310</a></p>
<p id="bib18">Wrenn, E.D., Moore, B.M., Greenwood, E., McBirney, M., and Cheung, K.J. (2020). Optimal, large-scale propagation of mouse mammary tumor Organoids. J. Mammary Gland Biol. Neoplasia <i>25</i>, 337-350. <a class="external-link" href="https://doi.org/10.1007/s10911-020-09464-1">https://doi.org/10.1007/s10911-020-09464-1</a></p>
<p id="bib19">Yan, G., Dai, M., Zhang, C., Poulet, S., Moamer, A., Wang, N., Boudreault, J., Ali, S., and Lebrun, J.J. (2021). TGFbeta/cyclin D1/Smad-mediated inhibition of BMP4 promotes breast cancer stem cell self-renewal activity. Oncogenesis <i>10</i>, 21. <a class="external-link" href="https://doi.org/10.1038/s41389-021-00310-5">https://doi.org/10.1038/s41389-021-00310-5</a></p>
<p id="bib20">Yip, H.Y.K., Chee, A., Ang, C.S., Shin, S.Y., Ooms, L.M., Mohammadi, Z., Phillips, W.A., Daly, R.J., Cole, T.J., Bronson, R.T., et al. (2020). Control of glucocorticoid receptor levels by PTEN establishes a failsafe mechanism for tumor suppression. Mol. Cell <i>80</i>, 279-295 e278. <a class="external-link" href="https://doi.org/10.1016/j.molcel.2020.09.027">https://doi.org/10.1016/j.molcel.2020.09.027</a></p>
<p id="bib21">Yip, H.Y.K. and Papa, A. (2021). Signaling pathways in cancer: therapeutic targets, combinatorial treatments, and new developments. Cells <i>10</i>. <a class="external-link" href="https://doi.org/10.3390/cells10030659">https://doi.org/10.3390/cells10030659</a></p>
<p id="bib22">Zhang, Z., Christin, J.R., Wang, C., Ge, K., Oktay, M.H., and Guo, W. (2016). Mammary-stem-cell-based somatic mouse models reveal breast cancer drivers causing cell fate dysregulation. Cell Rep. <i>16</i>, 3146-3156. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2016.08.048">https://doi.org/10.1016/j.celrep.2016.08.048</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Antonella Papa (<a href="mailto:antonella.papa@monash.edu">antonella.papa@monash.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze any datasets or code.</p>
<h3>Acknowledgments</h3>
<p>The authors would like to thank the Monash Animal Research Platform for animal care and the Monash Organoid Program, the Monash Micro Imaging Facility, Biochemistry Imaging Facility, and the Monash FlowCore for technical assistance and reagents. Graphical abstract and <a href="#fig6">Figure 6</a> were created with <a href="http://BioRender.com">BioRender.com</a>. A.P. is supported by a <a href="https://doi.org/10.13039/100008018">Victorian Cancer Agency</a> (VCA) Mid-Career Research Fellowship (MCRF20027).</p>
<h3>Author contributions</h3>
<p>Conceptualization, H.Y.K.Y. and A.P.; methodology, H.Y.K.Y. and A.P.; formal analysis, H.Y.K.Y.; investigation, H.Y.K.Y. and A.P.; resources, A.P.; data curation, H.Y.K.Y.; writing, H.Y.K.Y. and A.P.; funding acquisition, A.P.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>