<article>
<section>
<h1 id="header">Protocol to maintain single functional mouse hematopoietic stem cells <i>in vitro</i> without cell division</h1>
<p><time datetime="2021-11-05">Published: November 5, 2021</time></p>
<p>Miriam Belmonte<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> and David G. Kent<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Wellcome MRC Cambridge Stem Cell Institute, University of Cambridge, Hills Road, Cambridge CB2 0XY, UK</p>
<p id="aff2"><sup>2</sup>Department of Haematology, University of Cambridge, Cambridge CB2 0XY, UK</p>
<p id="aff3"><sup>3</sup>York Biomedical Research Institute, Department of Biology, University of York, York YO10 5DD, UK</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:miriam.belmonte7@gmail.com">miriam.belmonte7@gmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:david.kent@york.ac.uk">david.kent@york.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100927">10.1016/j.xpro.2021.100927</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol details the isolation and <i>in vitro</i> maintenance of single hematopoietic stem cells (HSCs) in the absence of the bone marrow niche. The HSCs do not divide over 7 days and fully retain their long-term functional capacity in transplantation assays. Following hibernation culture, HSCs can be used to study quiescence exit and can be genetically manipulated as single cells prior to division.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2021.04.002">Oedekoven et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>
<i>In vitro</i> maintenance of single HSCs in the absence of a hematopoietic niche</li>
<li>
<i>In vitro</i> system to study quiescence exit in single HSCs</li>
<li>Ability to manipulate HSCs via gene modification at the single-cell level</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1141-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Hematopoietic stem cells are thought to reside at the apex of the hematopoietic hierarchy and possess the unique capacity to both self-renew (perpetuating themselves as an undifferentiated population) and differentiate (generating the correct numbers and types of mature cell progeny to perform the necessary functions of the blood system). The development of clonal approaches to purify blood stem and progenitor cell populations has enabled significant breakthroughs in the field including the formal documentation of HSC functional heterogeneity and the development of a wide array of molecular tools to study HSCs at the single cell level. This has also led to a range of studies aimed at mimicking the HSC environment <i>in vitro</i> in order to develop protocols that would successfully expand hematopoietic populations in a dish.</p>
<p>In Oedekoven et al., we described a powerful culture system which maintains mouse and human HSCs as single cells, while preserving their functional and molecular properties, allowing a 7-day culture window to study HSCs in a distinct physiological context (<a href="#bib9">Oedekoven et al., 2021</a>). These “hibernation cultures” keep HSCs from dividing by removal of key mitogens SCF and TPO from standard conditions. Optimization and utilization of such <i>in vitro</i> cultures would have a range of applications for manipulating HSCs at the single-cell level in both experimental and clinical research.</p>
<p>The protocol below describes the specific steps for using serum-supplemented medium. However, we have also validated the hibernation protocol in the serum-free culture media published in <a href="#bib11">Wilkinson et al. (2019)</a> as reported in <a href="#bib9">Oedekoven et al. (2021)</a>.</p>
<h3 id="sec1.1">Prepare washing buffer (PBS 2%FCS)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol>
<li>Mix all reagents listed below:</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>-</td>
<td>490mL</td>
</tr>
<tr>
<td>Fetal Calf Serum</td>
<td>2%</td>
<td>10mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>500mL</b></td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Filter using a 0.2μm filter and store at 4°C for up to 1 month</li>
</ol>
<h3 id="sec1.2">Prepare culture medium</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="3">
<li>To prepare 10 mL of medium (amount for loading 100μL/well of a 96-well plate), mix all the reagents listed in the table below except for IL-11 (which will be added to the plate after sorting the cells)</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>StemSpan SFEM</td>
<td>-</td>
<td>-</td>
<td>8.78</td>
</tr>
<tr>
<td>Fetal Calf Serum</td>
<td>-</td>
<td>10%</td>
<td>1</td>
</tr>
<tr>
<td>Penicillin / Streptomycin</td>
<td>100 units/mL</td>
<td>1%</td>
<td>0.1</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>200mM</td>
<td>1%</td>
<td>0.1</td>
</tr>
<tr>
<td>10<sup>−4</sup>M 2-Mercaptoethanol</td>
<td>-</td>
<td>0.2%</td>
<td>0.02</td>
</tr>
<tr>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>Interleukin 11 (IL-11)</td>
<td>10ng/mL</td>
<td>20 ng/mL</td>
<td>0.02</td>
</tr>
</tbody>
</table>
<ol start="4">
<li>Filter the medium using a 0.2μm filter before use
<ol type="a">
<li>Store the medium at 4°C for up to 1 week, if needed</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Pre-load medium a U-bottom 96-well plate</h3>
<ol start="5">
<li>Load 50μL of the culture media (described above) per well into U-bottom 96-well plate(s).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the rest of the medium at 4°C as this will be added after the cell sort using a 2X concentration of IL-11. This is to avoid the wastage of recombinant cytokines in the event that there is a technical failure at the cell sorter and to synchronize the stimulation time of the cytokine).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-mouse CD45 FITC (dilution 1:100)</td>
<td>BD Biosciences</td>
<td>clone 30-F11</td>
</tr>
<tr>
<td>Anti-mouse EPCR PE (dilution 1:100)</td>
<td>STEMCELL</td>
<td>clone RMEPCR1560</td>
</tr>
<tr>
<td>Anti-mouse CD150 PE-Cy7 (dilution 1:100)</td>
<td>Biolegend</td>
<td>clone TC15-12F12.2</td>
</tr>
<tr>
<td>Anti-mouse CD48 APC (dilution 1:100)</td>
<td>Biolegend</td>
<td>clone HM48-1</td>
</tr>
<tr>
<td>Anti-mouse Sca-1 BV421 (dilution 1:200)</td>
<td>Biolegend</td>
<td>clone D7</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>PBS</td>
<td>Sigma</td>
<td>Catalog #D8537</td>
</tr>
<tr>
<td>Fetal Calf Serum (FCS)<i>FCS should be screened for high efficiency haematopoietic colony growth</i>
</td>
<td>Sigma</td>
<td>Catalog #F7524</td>
</tr>
<tr>
<td>StemSpan</td>
<td>STEMCELL Technologies</td>
<td>Catalog #09650</td>
</tr>
<tr>
<td>Ammonium Chloride (NH<sub>4</sub>Cl)</td>
<td>STEMCELL Technologies</td>
<td>Catalog #07850</td>
</tr>
<tr>
<td>2mM L-Glutamine</td>
<td>Life Technologies</td>
<td>Catalog #25030081</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Fisher Scientific</td>
<td>Catalog #11528926</td>
</tr>
<tr>
<td>Penicillin and Streptomycin (Pen/Strep)</td>
<td>Sigma</td>
<td>Catalog #P4333</td>
</tr>
<tr>
<td>7-Aminoactinomycin D (1mg/mL solution in DMSO)</td>
<td>Life Technologies</td>
<td>Catalog #A1310</td>
</tr>
<tr>
<td>Recombinant mouse IL-11</td>
<td>Bio-Techne</td>
<td>Catalog #418-ML</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>EasySep Mouse Hematopoietic Progenitor Enrichment kit</td>
<td>STEMCELL Technologies</td>
<td>Catalog # 19856</td>
</tr>
<tr>
<td>EasySep Magnet</td>
<td>STEMCELL Technologies</td>
<td>Catalog # 18000</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>C57BL/6 mice, 8–12 weeks old</td>
<td>Charles River Laboratories</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Dissecting instruments (scissors and forceps)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Mortar and pestle</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>50μm filters</td>
<td>Wolf Laboratories</td>
<td>Catalog #04-0042-2317</td>
</tr>
<tr>
<td>5 mL polypropylene tubes</td>
<td>VWR International</td>
<td>Catalog #352063</td>
</tr>
<tr>
<td>5 mL polystyrene tubes</td>
<td>Scientific Laboratory Supplies</td>
<td>Catalog #352054</td>
</tr>
<tr>
<td>50 mL conical tubes</td>
<td>VWR International</td>
<td>Catalog #525-0402</td>
</tr>
<tr>
<td>U-bottom 96-well plates</td>
<td>Scientific Laboratory Supplies</td>
<td>Catalog #353077</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Mouse bone marrow cell preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 2–3 h, depending on the number of animals/samples</div>
<p>This section describes the preparation of murine bone marrow cell suspension and magnetic bead depletion of progenitor cells (optional) by negative selection using the EasySep™ Mouse Hematopoietic Progenitor Cell Enrichment Kit (STEMCELL).</p>
<ol>
<li>Harvest the bone marrow cells from femora, tibiae and pelvic bones removed from the mice by crushing the bones in 10mL PBS 2%FCS using a mortar and pestle. For specific animals where maximal cells are required, BM can also be obtained from the spine and forearms</li>
<li>Transfer the cell suspension into a 50 mL conical polypropylene centrifuge tube</li>
<li>Centrifuge the cell suspension for 5 min at 300 x g at room temperature</li>
<li>Remove the supernatant and resuspend the cells in 3mL PBS/FCS</li>
<li>Lyse the red blood cells by adding 5 mL Ammonium Chloride (NH<sub>4</sub>Cl, STEMCELL) incubating the cells on ice for 10 min, vortexing after the first 5 min</li>
<li>Fill the tube up to 50mL with PBS/FCS and centrifuge for 5 min at 300 x g at room temperature</li>
<li>Resuspend the cells in 500μL/mouse PBS/FCS and transfer into a polystyrene round-bottom tube
<ol type="a">
<li>Add 1mL of PBS/FCS to the empty 50mL conical polypropylene centrifuge tube from step 6 and use these leftover cells for preparing control samples of non-depleted cells. This is important to assist the user in setting gates for sorting as well as for determining the absolute frequencies of subpopulations of interest, since these cannot be measured from the enriched sample. Filter the suspension using a 50-μm filter. Use 100μL of this suspension into each of the FACS control tubes (unstained, 7AAD only, FITC only, PE only, APC only, PE-Cy7 only, BV421 only)
<div class="optional">
<span class="optional-title">Optional:</span> Perform lineage depletion by using EasySep mouse hematopoietic progenitor cell enrichment kit (STEMCELL). The lineage depletion step will reduce sorting time and reagent costs, especially when isolating HSCs from multiple mice. We typically use the Hematopoietic Enrichment Cocktail and EasySep technology from STEMCELL Technologies, however, other commercial suppliers of lineage depletion kits exist. Alternatively, the lineage depletion step could by replaced using directly conjugated antibodies which allow the exclusion of the differentiated population during the cell sorting (CD5, CD11b, CD19, CD45R/B220, Ly6G/C(Gr-1), TER119). This, however, will increase sorting time and reagent costs, compared to performing the lineage-depletion step prior to cell sorting.</div>
Manufacturer protocol was followed with minor modifications to prioritize yield over purity in this pre-sort “debulking” step:
<ol type="i">
<li>Add 10μL/mouse EasySep Mouse Hematopoietic Progenitor Cell (HSPC) Isolation cocktail and incubate for 15 min, on ice.</li>
<li>Add 15μL/mouse EasySep Streptavidin RapidSpheres and incubate for 15 min on ice</li>
<li>Add PBS/FCS to a total of 2.5mL</li>
<li>Insert the tube into the EasySep Magnet and incubate for 3 min at room temperature</li>
<li>Transfer the supernatant into a new tube and repeat the magnet incubation step one additional time</li>
<li>Centrifuge the cell suspension for 5 min at 300 x g at room temperature</li>
<li>Resuspend the cells in 100μL/mouse PBS/FCS and transfer into a polypropylene tube to perform antibody staining</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Antibody staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 40–50 min</div>
<p>This section describes the incubation of bone marrow samples with fluorescence-tagged antibodies and the preparation of compensation controls.</p>
<ol start="8">
<li>Add the following antibodies to the FACS control tubes:</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Label</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>CD45</td>
<td>FITC</td>
<td>1:100</td>
</tr>
<tr>
<td>CD48</td>
<td>APC</td>
<td>1:100</td>
</tr>
<tr>
<td>CD150</td>
<td>PE-Cy7</td>
<td>1:100</td>
</tr>
<tr>
<td>EPCR</td>
<td>PE</td>
<td>1:100</td>
</tr>
<tr>
<td>Sca-1</td>
<td>BV421</td>
<td>1:200</td>
</tr>
</tbody>
</table>
<ol start="9">
<li>Add all antibodies at the concentration specified above to the sample tubes (making an antibody master mix is recommended if more than one sample)</li>
<li>Vortex the cells in each tube and incubate on ice in the dark for 30 min</li>
<li>Wash the cells by adding 1mL of PBS/FCS to each tube and centrifuging for 5 min at 300 x g at room temperature</li>
<li>Resuspend the control tubes in 300μL PBS/FCS and add 1:1000 7AAD to the 7AAD-only control tube</li>
<li>Resuspend the sample tube in 500μL/mouse 1:1000 7AAD in PBS/FCS and filter the suspension using a 50-μm filter into a new polypropylene tube. Add 500μL 1:1000 7AAD in PBS/FCS to the original tube to make sure to rinse out the remaining cells and transfer them to the new tube through the 50-μm filter</li>
</ol>
<h3 id="sec2.3">Sorting of ESLAM Sca-1<sup>high</sup> LT-HSCs</h3>
<p>This section describes the workflow to isolate single HSCs using fluorescent-activated cell sorting (FACS).</p>
<ol start="14">
<li>Use the unstained control to set voltages of each channel on the flow sorter and use 7AAD-only control to set the viability gates. Use the single stain controls to set up the compensation for each detection channel</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Compensation beads can substitute for single stain controls, but it is important to remember that they will not account for background staining or autofluorescence and it is advisable to check the full panel in an initial setup phase.</div>
<ol start="15">
<li>Apply the compensation settings to the sample tubes</li>
<li>Load the sample tube on the flow sorter and set gates as shown in <a href="#fig1">Figure 1</a>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1141-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Representative gating strategy to purify ESLAM Sca-1<sup>high</sup> LT-HSCs</div>
<p>On a SSC-A vs FSC-A plot, gate a region which excludes most of the dead cells and debris and very large cells. Gate on single cells using pulse width and FSC-A as shown. Exclude dead cells by gating 7AAD-negative cells. To set the SLAM population, gate CD48<sup>-</sup> CD150<sup>+</sup> cells, being careful not to go too high into the CD48<sup>dim</sup> population. From the SLAM population, gate CD45<sup>+</sup>EPCR<sup>+</sup>cells with a relatively strict EPCR<sup>+</sup> gate (initially described as EPCR<sup>++</sup>)(<a href="#bib6">Kent et al., 2009</a>). To set the ESLAM Sca-1<sup>high</sup> population which has an increased proportion of LT-HSCs but reduced overall yield, gate Sca-1<sup>high</sup> cells (described in <a href="#bib12">Wilson et al., 2015</a>).</p>
</figcaption>
</figure>
</li>
<li>Use the single-cell deposition unit of the sorter to place 1 cell into each of the wells of U-bottom 96-well plates, each well having been preloaded with 50μL media (described above in “Before you begin” section)</li>
<li>Load 50μL media (described above) supplemented with a 2X cytokine concentration to bring the final medium to 1X cytokines</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The cell suspension can be stained for longer than 30 minutes on ice, but it is recommended not to exceed ∼1hour in order to avoid overstaining the samples and increasing background signal. For a quicker preparation of single stain controls, tubes can be incubated for a shorter time (∼15 minutes) at room temperature, but this is not advisable for primary samples where room temperature incubation may alter the function of cells.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Given the long preparation time required to obtain cell suspension pre-sorting, bone marrow harvest can be performed the day before the sort and the cell suspension can be stored in 10%FCS/PBS at 4°C overnight. If considering this option, consider storing the sample in a larger liquid volume (∼5mL/mouse) and then spinning at 300 x g for 7 minutes followed by addition of crude DNase I to the pellet in the morning before undertaking lineage depletion and/or antibody staining described above.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec3.1">Mouse bone marrow cell preparation</h3>
<p>Cell suspension obtained from normal mice should look pinkish and be a bit cloudy. After the first centrifugation step the cell pellet will look reddish, but following the RBC lysis step and centrifugation, the supernatant should look clear and the pellet will be whitish, with red edges.</p>
<h3 id="sec3.2">Preparation of cells for cell sorting</h3>
<p>The final cell suspension should be contained in a volume of ∼1mL/mouse processed and filtered before sorting, as described above, to remove any clumps of cells which might clog the flow instrument. When resuspended in 2%FCS/PBS, the cell suspension should look translucent.</p>
<h3 id="sec3.3">Sorting of ESLAM Sca-1<sup>high</sup> LT-HSCs</h3>
<p>Representative flow cytometric plots for a typical ESLAM Sca-1<sup>high</sup> cell sorting procedure are shown in <a href="#fig1">Figure 1</a>. Viable cells are gated based on their FSC and SSC properties and their exclusion of 7AAD. CD48<sup>+</sup> cells are excluded by gating the bottom of the three major populations. These cells are then visualized in a CD45/EPCR plot and the double positive cells are selected. CD150<sup>+</sup>CD48<sup>-</sup>EPCR<sup>+</sup>Sca-1<sup>high</sup> cells are then selected, which represents ∼0.003% of the whole bone marrow cells in an adult C57BL/6 mouse.</p>
<h3 id="sec3.4">Single-cell culture of HSC</h3>
<p>In order to visualize the cells under the microscope, adjust the field of view to look at the entire well, then focus on the center of the well by using the coarse adjustment. In 96 well U-bottom plates, this is where the cell typically settles a few hours after sort. To facilitate the visualization of the single cells under the microscope, the plate can be spun in a centrifuge for 5 min at 240 x g to allow the cells to be collected at the center of the well (cells may not appear as spherical immediately after spinning). Alternatively, to facilitate visual identification, the first well can be used to sort 100 viable cells to allow establishment of the field of view for subsequent wells. Between 20 and 40% of single cells will survive 7 days of culture in the hibernation condition, compared to ∼98% survival in 300ng/mL SCF-supplemented media. ∼99% of the input HSCs cultured in the hibernation condition will not divide during the 7-day culture period and will be maintained as single cells (<a href="#fig2">Figures 2</a> and <a href="#fig3">3</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1141-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Cells in hibernation conditions have reduced survival and do not divide</div>
<p>Adapted from Oedekoven et al. (<a href="#bib9">Oedekoven et al., 2021</a>). Left panel: HSC survival in hibernation conditions is decreased compared to SCF-supplemented medium (+SCF n=355, 5 biological replicates; -SCF n=1722, 7 biological replicates). Right panel: number of clones that had divided (numbers of wells with &gt;2 cells were scored to determine) at day 7 post-isolation. Only culture conditions without SCF retained HSCs as single cells. Bars show mean with SEM. Unpaired t-test: ∗p&lt;0.05, ∗∗p&lt;0.01, ∗∗∗p&lt;0.001.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1141-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Time-lapse imagine of single HSCs in hibernation cultures</div>
<p>The three pictures show an individual well imaged on day 1, 3 and 7 of culture in the hibernation condition. Single cells do not divide over the course of the 7-day hibernation culture. Scale bar represents 50um. See also the supplemental video in Oedekoven et al. (<a href="#bib9">Oedekoven et al., 2021</a>)</p>
</figcaption>
</figure>
<h3 id="sec3.5">Single cells for future manipulation</h3>
<p>In <a href="#bib9">Oedekoven et al., 2021</a>, we showed that single HSCs could have virus delivered to them without undergoing division (<a href="#bib9">Oedekoven et al., 2021</a>). However, there are a wide range of other studies that could be imagined, including monitoring symmetric versus asymmetric partitioning of cellular components using high resolution imaging (<a href="#bib5">Filippi, 2021</a>; <a href="#bib10">Wang et al., 2021</a>), single cell immunostaining (<a href="#bib4">Ema et al., 2007</a>), or screening of compounds for improved survival or induction of cell division.</p>
<p>60–70% of single HSCs cultured in the hibernation condition for 7 days are able to generate clones in colony forming cell assays at day 14, with the majority generating at least three different mature cell types (as determined by flow cytometry for cell surface markers Ly6g (Gr-1), CD11b (Mac-1), Ter-119 and CD41) (<a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1141-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Single HSCs in hibernation culture retain multi-potency</div>
<p>Adapted from Oedekoven et al. (<a href="#bib9">Oedekoven et al., 2021</a>). Left panel: Colony forming cell assay efficiency for single HSCs in hibernating culture (n=230, 6 biological replicates). Right panel: ∼80% of single cells generate colonies of at least three lineages in colony forming cell assays (n=166, 3 biological replicates). Colonies are defined as MK (containing cells positive for CD41, a megakaryocyte marker), GM (containing cells positive for granulocyte/monocyte markers Gr-1 and CD11b), GEM (containing cells positive for granulocyte/monocyte markers Gr-1 and CD11b, GMM (positive for GM and MK markers), and GEMM (positive for GM, MK and erythrocyte markers). Bars show mean with SEM.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<h3 id="sec4.1">Low survival in hibernation culture</h3>
<p>HSC survival in hibernation culture is decreased in the absence of SCF compared to SCF-supplemented medium, as shown in <a href="#fig2">Figure 2</a>. Further optimization of the hibernation culture and/or the addition of additional supplement (e.g., Rho kinase inhibitors (<a href="#bib2">Claassen et al., 2009</a>), fibronectin, PVA (<a href="#bib11">Wilkinson et al., 2019</a>)) could improve the survival <i>in vitro</i> and allow larger-scale studies, without requiring the use of larger number of animals to obtain higher cell output.</p>
<h3 id="sec4.2">Potential selection of a specific HSC phenotype</h3>
<p>In Oedekoven et al. (<a href="#bib9">Oedekoven et al., 2021</a>), we investigated whether differences in HSC subtypes (in accordance with <a href="#bib3">Dykstra et al., 2007</a>) were observed between hibernating and freshly isolated HSCs in single cell transplantation experiments. Although it would need to be confirmed in a larger number of recipients, the current data suggest that hibernation conditions might preferentially retain α-HSC (<a href="#bib9">Oedekoven et al., 2021</a>) which is also consistent with the higher levels of CD150 expression previously reported (<a href="#bib1">Beerman et al., 2010</a>; <a href="#bib8">Morita et al., 2010</a>). This could indicate that the α-HSC subtype is more resilient in the hibernation culture and that the cytokine deprivation might be selecting for the more myeloid-biased HSCs and this property may also be related to the delayed engraftment that is observed in α-HSCs.</p>
<h3 id="sec4.3">Long-term survival of HSCs in hibernation culture</h3>
<p>Cell survival decreases overtime, and reduces after 7 days of culture, making longer term cultures in this current condition challenging (<a href="#fig5">Figure 5</a>). We have tested up to 10 days in some experiments without observing an additional decrease in cell survival, however, we are unable to say whether extended culture beyond 10 days would also permit the same subset of HSCs to survive.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1141-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Time course of survival over 7 days of SCF deprivation</div>
<p>SCF-deprivation decreased HSC survival in culture compared to SCF-supplemented conditions, and only ∼25–35% remain viable single cells up to 7 days in culture (+SCF, 5 biological replicates; -SCF, 5 biological replicates). Bars show mean with SEM.</p>
</figcaption>
</figure>
<h3 id="sec4.4">Proliferation of HSCs in hibernation culture</h3>
<p>Our data show that over 99% of HSCs cultured in hibernation condition remain as single cells over the 7-day hibernation culture. However, a small fraction of single cells (∼0.8%) do undergo division. In order to confirm that none of the cells which survived on day 7 have undergone division followed by death of one daughter cells, we recommend performing daily counts or time-lapse imaging to monitor potential cell division/cell death.</p>
<h3 id="sec4.5">Gene expression changes</h3>
<p>Gene expression profiling of single LT-HSCs in 7-day hibernation conditions revealed a high retention of G0/G1 cell cycle genes, as well as self-renewal regulators of the HSC-related MolO signature (<a href="#bib12">Wilson et al., 2015</a>), suggesting that such genes are indeed likely to be essential for HSC function and confirming the functional HSC identity of hibernating HSCs at the transcriptional level. However, some molecular differences between hibernating and freshly isolated HSCs were observed, and a number of regulators were not expressed in hibernating HSCs (e.g., members of the AP1 complex). These observations suggest that the hibernation cultures might drive the reduced expression of these factors. That said, it is important to note that SCF-stimulation of the hibernating HSCs did not re-initiate their expression, potentially disfavoring this hypothesis (<a href="#bib9">Oedekoven et al., 2021</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Insufficient time on the sorter to obtain the required cell number and/or insufficient number of cells obtained after sorting.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Insufficient time/cells might be caused by insufficient number of input cells. Consider harvesting more bone marrow (more mice) and performing the optional pre/sort lineage depletion (e.g., using EasySep Lineage Depletion cocktail from STEMCELL Technologies). This step will help decreasing the number of contaminating cells and will increase the absolute yields. If performing single cell sorting, it is advisable not to increase the speed of cell sort beyond ∼2000 cells/s in order to avoid too many aborted decisions in the single cell/droplet setting on the flow sorter.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>High number of 7AAD-positive cells at the sorter.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Depending on the final panel of antibodies/fluorochromes used, there may be some compensation issue causing cells to appear 7AAD positive. This is most likely to be the case if these cells are directly above the viable cell fraction in FSC (as events lower in FSC are likely to be dead). Check with a flow operator and adjust the flow cytometer’s settings and compensation where necessary. If the cells were indeed dead, some possible approaches can be taken including keeping the cells and plates on ice and not leaving cells overnight. It would also be worth checking reagents to ensure that they are at the correct concentrations and not inadvertently driving cell lysis (e.g., 10× PBS versus 1× PBS).</p>
<h3 id="sec5.5">Problem 3</h3>
<p>Difficulty finding the single cells in individual wells under the microscope.</p>
<h3 id="sec5.6">Potential solution</h3>
<p>There are two possible issues: 1) the cells were not sorted properly or 2) the cells are present, but you cannot find them. If the cells were not sorted correctly in the wells, check with the flow operator the flow cytometer’s settings and calibration used (including sort reports) and consider performing a set of optimization experiments with cells (not beads) to ensure that the single cell deposition unit is functioning properly. If the cells are there, but you cannot find them, another possible explanation is that the medium is full of debris which make hard to distinguish the cells. We recommend always filtering the media before sorting into it, by using a 0.22μm filter and to sort ∼100 viable cells into well A1 to allow efficient finding of the focal plane for visualizing cells. We also suggest to always prepare a plate loaded with media (as above) with 300ng/mL Stem Cell Factor (SCF), as good control for the sorter/sorting. In this latter scenario, we would expect to obtain over 90% survival and cell division to occur following 24–48 h (<a href="#bib7">Kent et al., 2013</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Beerman, I., et al. (2010). Functionally distinct hematopoietic stem cells modulate hematopoietic lineage potential during aging by a mechanism of clonal expansion. Proc. Natl. Acad. Sci. U S A <i>107</i>, 5465-5470. <a class="external-link" href="https://doi.org/10.1073/pnas.1000834107">https://doi.org/10.1073/pnas.1000834107</a></p>
<p id="bib2">Claassen, D.A., Desler, M.M., and Rizzino, A. (2009). ROCK inhibition enhances the recovery and growth of cryopreserved human embryonic stem cells and human induced pluripotent stem cells. Mol. Reprod. Dev. <i>76</i>, 722. <a class="external-link" href="https://doi.org/10.1002/MRD.21021">https://doi.org/10.1002/MRD.21021</a></p>
<p id="bib3">Dykstra, B., et al. (2007). Long-term propagation of distinct hematopoietic differentiation programs in vivo. Cell Stem Cell <i>1</i>, 218-229. <a class="external-link" href="https://doi.org/10.1016/j.stem.2007.05.015">https://doi.org/10.1016/j.stem.2007.05.015</a></p>
<p id="bib4">Ema, H., et al. (2007). Adult mouse hematopoietic stem cells: purification and single-cell assays. Nat. Protoc. <i>1</i>, 2979-2987. <a class="external-link" href="https://doi.org/10.1038/nprot.2006.447">https://doi.org/10.1038/nprot.2006.447</a></p>
<p id="bib5">Filippi, M.D. (2021). Hematopoietic stem cell (HSC) divisional memory: the journey of mitochondrial metabolism through HSC division. Exp. Hematol. <i>96</i>, 27-34. <a class="external-link" href="https://doi.org/10.1016/J.EXPHEM.2021.01.006">https://doi.org/10.1016/J.EXPHEM.2021.01.006</a></p>
<p id="bib6">Kent, D.G., et al. (2009). Prospective isolation and molecular characterization of hematopoietic stem cells with durable self-renewal potential. Blood <i>113</i>, 6342-6350. <a class="external-link" href="https://doi.org/10.1182/blood-2008-12-192054">https://doi.org/10.1182/blood-2008-12-192054</a></p>
<p id="bib7">Kent, D.G., et al. (2013). Self-renewal of single mouse hematopoietic stem cells is reduced by JAK2V617F without compromising progenitor cell expansion. PLoS Biol. <i>11</i>, e1001576, Edited by M. A. Goodell. <a class="external-link" href="https://doi.org/10.1371/journal.pbio.1001576">https://doi.org/10.1371/journal.pbio.1001576</a></p>
<p id="bib8">Morita, Y., Ema, H., and Nakauchi, H. (2010). Heterogeneity and hierarchy within the most primitive hematopoietic stem cell compartment. J. Exp. Med. <i>207</i>, 1173-1182. <a class="external-link" href="https://doi.org/10.1084/jem.20091318">https://doi.org/10.1084/jem.20091318</a></p>
<p id="bib9">Oedekoven, C.A., et al. (2021). Hematopoietic stem cells retain functional potential and molecular identity in hibernation cultures. Stem Cell Reports <i>16</i>, 1614-1628. <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2021.04.002">https://doi.org/10.1016/j.stemcr.2021.04.002</a></p>
<p id="bib10">Wang, W., et al. (2021). Cytokine combinations for human blood stem cell expansion induce cell type- and cytokine-specific signaling dynamics. Blood. . <a class="external-link" href="https://doi.org/10.1182/blood.2020008386">https://doi.org/10.1182/blood.2020008386</a></p>
<p id="bib11">Wilkinson, A.C., et al. (2019). Long-term ex vivo haematopoietic-stem-cell expansion allows nonconditioned transplantation. Nature <i>571</i>, 117-121. <a class="external-link" href="https://doi.org/10.1038/s41586-019-1244-x">https://doi.org/10.1038/s41586-019-1244-x</a></p>
<p id="bib12">Wilson, N.K., et al. (2015). Combined single-cell functional and gene expression analysis resolves heterogeneity within stem cell populations. Cell Stem Cell <i>16</i>, 712-724. <a class="external-link" href="https://doi.org/10.1016/j.stem.2015.04.004">https://doi.org/10.1016/j.stem.2015.04.004</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, David G. Kent <a href="mailto:david.kent@york.ac.uk">david.kent@york.ac.uk</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The data sets supporting this protocol, and used in Figures, have not been deposited in a public repository but are available from the corresponding author upon request.</p>
<h3>Acknowledgments</h3>
<p>We thank all of the authors of the original study (<a href="#bib9">Oedekoven et al., 2021</a>) from which this protocol was generated. We thank Reiner Schulte, Chiara Cossetti, and Gabriela Grondys-Kotarba of the CIMR Flow Cytometry core for technical assistance and suggestions; Sally Thomas and the Central Biomedical Services unit staff; and Mairi Shepherd for providing some of the illustrations contained in the graphical abstract. Work in the D.G.K. laboratory is supported by an <a href="https://doi.org/10.13039/100010663">ERC</a> Starting Grant (ERC-2016-STG–715371), an MRC-<a href="https://doi.org/10.13039/100009619">AMED</a> joint award (MR/V005502/1), a CR-UK Programme Foundation Award (DCRPGF\100008), and a core support grant to the Wellcome MRC Cambridge Stem Cell Institute.</p>
<h3>Author contributions</h3>
<p>M.B. analyzed the data reported in this study and generated the graphs and figures. M.B. and D.G.K. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>