<article>
<section>
<h1 id="header">Assessment of dopaminergic neuron degeneration in a <i>C. elegans</i> model of Parkinson’s disease</h1>
<p><time datetime="2022-04-01">Published: April 1, 2022</time></p>
<p>Konstantinos Palikaras,<sup><a href="#aff1">1</a>,<a href="#fn1">6</a>,<a href="#cor1">*</a></sup> Tanima SenGupta,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">6</a></sup> Hilde Nilsen,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#cor2">**</a></sup> and Nektarios Tavernarakis<sup><a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#fn2">7</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Physiology, School of Medicine, National and Kapodistrian University of Athens, 11527 Athens, Greece</p>
<p id="aff2"><sup>2</sup>Department of Clinical Molecular Biology, University of Oslo, Oslo, Norway</p>
<p id="aff3"><sup>3</sup>Department of Clinical Molecular Biology, Akershus University Hospital, Lørenskog, Norway</p>
<p id="aff4"><sup>4</sup>Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas, Heraklion, 71300 Crete, Greece</p>
<p id="aff5"><sup>5</sup>Department of Basic Sciences, Faculty of Medicine, University of Crete, Heraklion, 71300 Crete, Greece</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:palikarask@med.uoa.gr">palikarask@med.uoa.gr</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:hilde.nilsen@medisin.uio.no">hilde.nilsen@medisin.uio.no</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:tavernarakis@imbb.forth.gr">tavernarakis@imbb.forth.gr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101264">10.1016/j.xpro.2022.101264</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Transgenic <i>Caenorhabditis elegans</i> that expresses the full-length wild-type human α-synuclein in dopaminergic neurons provides a well-established Parkinson’s disease (PD) nematode model. Here, we present a detailed protocol to monitor and dissect the molecular underpinnings of age-associated neurodegeneration using this PD nematode model. This protocol includes preparation of nematode growth media and bacterial food sources, as well as procedures for nematode growth, synchronization, and treatment. We then describe procedures to assess dopaminergic neuronal death <i>in vivo</i> using fluorescence imaging.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref23">SenGupta et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A Parkinson’s disease nematode model to study α-synuclein-mediated neurotoxicity</li>
<li>Comprehensive approach for scoring cell death of dopaminergic neurons in <i>C. elegans</i>
</li>
<li>Genetic tools to investigate the tissue specific effects on neurodegeneration</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1527-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">
<i>C. elegans</i> strains and culture conditions</h3>
<p>We followed standard procedures for nematode maintenance (<a href="#bib25">Stiernagle, 2006</a>). <i>C. elegans</i> strains were grown on nematode growth medium (NGM) plates seeded with the non-pathogenic <i>Escherichia coli</i> OP50-1 and HT115(DE3) bacteria. Animals were cultured at 20°C. The following strains were used to evaluate dopaminergic neuronal loss: BY273: <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>α-synuclein<sup>wt</sup>], RB877: <i>nth-1(ok724)III</i>, IR2355: <i>nth-1(ok724)III</i>; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>α-synuclein<sup>wt</sup>]. To investigate the tissue specific effects on neurodegeneration, we used the following transgenic animals: dopaminergic neuron-specific RNAi UA196: <i>sid-1(pk3321); baln11</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>α-synuclein<sup>wt</sup>]; <i>baln33</i>[p<sub><i>dat-1</i></sub>SID-1; p<sub><i>myo-2</i></sub>mCherry], pan-neuronal RNAi IR2531: <i>sid-1(pk3321)V</i>; <i>uIs69</i>[p<sub><i>unc-119</i></sub>SID-1; p<sub><i>myo-2</i></sub>mCherry]V; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>α-synuclein<sup>wt</sup>], hypodermis-specific RNAi IR2945: <i>rde-1(ne219)V</i>; <i>kzIs9</i>[p<sub><i>lin-26</i></sub>RDE-1; p<sub><i>lin-26</i></sub>NLS::GFP; <i>rol-6(su1006)</i>]; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>α-synuclein<sup>wt</sup>], intestine-specific RNAi IR2947: <i>rde-1(ne219)V</i>; <i>kbIs7</i>[p<sub><i>nhx-2</i></sub>RDE-1; <i>rol-6(su1006)</i>]; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>α-synuclein<sup>wt</sup>].</p>
<h3 id="sec1.2">Preparation of NGM plates</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<ol>
<li>Weigh and mix 3 g of NaCl, 2.5 g bacto-peptone, 0.2 g streptomycin and 17 g agar in a 1 L glass bottle.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> To avoid any heat-inactivation of streptomycin, it can be added after autoclaving, when the media is cooled down. A conical flask can be used instead of the bottle.</div>
<div class="note">
<span class="note-title">Note:</span> The streptomycin-resistant OP50-1 <i>E. coli</i> strain is used. For the preparation of the RNAi agar plates do not add streptomycin. The HT115(DE3) <i>E. coli</i> stain is not resistant to streptomycin.</div>
<ol start="2">
<li>Add 900 mL distilled water.</li>
<li>Place a magnetic stir bar into the bottle and close its cap. Autoclave for 30 min.</li>
<li>Place the autoclaved bottle on the stirrer and air-cool it to 55°C–60°C.</li>
<li>Add 1 mL MgSO<sub>4</sub> (1 M stock solution), 1 mL cholesterol (5 mg/mL stock solution), 1 mL CaCl<sub>2</sub> (1 M stock solution), 1 mL nystatin (10 mg/mL stock solution), 25 mL KPO<sub>4</sub> (1 M stock solution).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Nystatin is an anti-fungal agent. Add 500 μL ampicillin (10 mg/mL stock solution) in the medium for RNAi agar plates.</div>
<ol start="6">
<li>Add distilled sterile water up to 1 L.</li>
<li>Use a peristaltic pump and dispense the NGM medium into petri dishes. Add 10 mL of NGM medium per petri dish (60 × 15 mm diameter).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A liquid pipette can be used instead of a peristaltic pump.</div>
<ol start="8">
<li>Leave the NGM plates to solidify.</li>
<li>Place the NGM plates at room temperature (22°C–25°C) for a day before use.</li>
<li>Upside-down the NGM plates to avoid moisture condensation on the lid and store them for up to 3 weeks at 4°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Longer storage of NGM plates could affect the salt concentration due to excessive moisture evaporation. Therefore, the same batch of NGM plates should be used for the entire set of experiments.</div>
<h3 id="sec1.3">Bacterial food source</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<h4 id="sec1.3.1">OP50-seeded NGM plates</h4>
<ol start="11">
<li>Streak <i>E. coli</i> OP50 bacteria onto LB agar plate.</li>
<li>Incubate the plates at 37°C overnight (∼18 h).</li>
<li>Pick a single OP50 bacterial colony from an LB agar plate by using a sterilized toothpick (or inoculation needle) and place it in a flask containing 50 mL LB medium and incubate it in a shaking incubator at 37°C for 8 h.</li>
<li>Place 200 μL of OP50 culture on NGM plates.</li>
<li>Swirl and let the plates dry at room temperature overnight.</li>
</ol>
<h4 id="sec1.3.2">HT115(DE3)-seeded RNAi agar plates</h4>
<ol start="16">
<li>Streak HT115(DE3) <i>E. coli</i> strains expressing the empty vector pL4440 (control) and the pL4440 containing the sequence of <i>nth-1</i> gene (<i>nth-1</i>) onto LB agar plates containing 100 μg/mL ampicillin and 10 μg/mL tetracycline.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> HT115(D3) <i>E. coli</i> strain is tetracycline resistant due to <i>rnc14::Tn10</i> allele. The RNAi vector pL4440 confers resistance to ampicillin. Thus, RNAi agar plates should contain both ampicillin and tetracycline.</div>
<ol start="17">
<li>Incubate the plates at 37°C overnight (∼18 h).</li>
<li>Pick a bacterial colony of HT115(DE3) bacteria from each condition (control and <i>nth-1</i>) and place them in separate bacteriological culture tubes containing 5 mL LB medium, 5 μL ampicillin (stock solution 100 mg/mL) and 5 μL tetracycline (stock solution 10 mg/mL).</li>
<li>Incubate the tubes in a shaking incubator at 37°C overnight (~18 h).</li>
<li>Prepare different bacteriological tubes for each condition, and add 5 mL LB medium and 5 μL ampicillin (stock solution 100 mg/mL).</li>
<li>Add 350 μL (70 μL per 1 mL LB/ampicillin) of each overnight culture into separate bacteriological culture tubes containing 5 mL LB medium and 5 μL ampicillin (stock solution 100 mg/mL).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Nematodes physiology is affected by high-dose of tetracycline (<a href="#bib28">Vangheel et al., 2014</a>). Thus, tetracycline concentration should be reduced during the preparation of RNAi bacterial cultures.</div>
<ol start="22">
<li>Incubate the cultures in a shaking incubator at 37°C for 4 h until OD<sub>600</sub> will be 0.5–0.8.</li>
<li>Add 1–2 mM IPTG (20 mM stock solution) in each culture and proceed directly with plates seeding.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> IPTG can be added in the medium of RNAi agar plate after autoclaving. Fresh IPTG-containing RNAi plates should be prepared every 2 weeks because IPTG efficiency declines over time.</div>
<ol start="24">
<li>Place 200 μL of each culture (control and <i>nth-1</i>) on RNAi agar plates.</li>
<li>Swirl and let the plates dry at room temperature overnight.</li>
</ol>
<h3 id="sec1.4">Synchronizing <i>C. elegans</i> populations</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8 days</div>
<ol start="26">
<li>Transfer 10 L4 nematode larvae of each strain in separate OP50-seeded NGM plates. Prepare two plates per genotype/condition.</li>
<li>Incubate and let the nematodes to develop and grow at 20°C.</li>
<li>After 4–5 days, the plates contain mixed population with the presence of plenty gravid adult worms. Wash the plates with 2 mL M9 buffer and collect the animals in sterile 1.5 mL tubes.</li>
<li>Let the animals to settle down by gravity for 1 min and remove the liquid.</li>
<li>Add 500 μL freshly made bleaching solution and mix the samples.</li>
<li>Vortex the solution for 20 s. Repeat vortexing every minute until the worms are completely dissolved.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not keep the nematodes for more than 5 min in the bleaching solution, as it might affect viability of embryos.</div>
<ol start="32">
<li>Spin down the samples for 30 s at 2,000 g using a table-top centrifuge.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Centrifuge the samples for 1 min at 180 g.</div>
<ol start="33">
<li>Discard the supernatant and keep the pellet.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Remove carefully the supernatant without disturbing the egg-pellet.</div>
<ol start="34">
<li>Wash the egg-pellet with 1 mL of sterile M9 buffer.</li>
<li>Spin down the samples for 30 s at 2,000 g using a table-top centrifuge.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Centrifuge the samples for 1 min at 180 g.</div>
<ol start="36">
<li>Discard the supernatant and keep the pellet.</li>
<li>Repeat twice steps 34–36.</li>
<li>Add 200 μL of sterile M9 buffer and solubilize the pellet.</li>
<li>Dispense the egg solution to OP50-seeded NGM plates.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the gene of interest has not been knocked down using RNAi before, examine whether its knockdown could affect animals’ development to avoid any severe developmental arrest prior to the experiments. Then, eggs can be placed on HT115(DE3)-seeded RNAi agar plates enhancing the silencing of the gene of interest.</div>
<ol start="40">
<li>Incubate the plates at 20°C.</li>
<li>After 3 days, the plates are full of L4 stage nematodes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Several mutations or RNAi treatments might interfere with normal <i>C. elegans</i> development. Thus, any developmental delay or arrest should be taken into consideration for nematodes synchronization prior to any experiment, when animals of different genetic backgrounds are used for neurodegeneration assessment.</div>
<h3 id="sec1.5">Preparation of <i>C. elegans</i> strains</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–7 days</div>
<ol start="42">
<li>Transfer 20–30 L4 larvae per OP50-seeded NGM plate or HT115(DE3)-seeded RNAi agar plate.</li>
<li>Incubate the plates at 20°C.</li>
<li>Transfer the nematodes to freshly seeded NGM or RNAi agar plates every two days and incubate them at 20°C.</li>
<li>After the respective days, use 1, 3 and 7-day-old transgenic nematodes for microscopic examination and monitor dopaminergic neuron survival.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> To avoid worm picking use a 40 μm cell-strainer to separate adults from L1-L2 larvae. Wash the plate with M9 buffer and pass the solution through a 40 μm cell-strainer. L1-L2 larvae will pass through the filter, and adults will remain. Turn the cell-strainer to opposite direction and wash the remained adults with 500 μL of sterile M9 buffer. Carefully collect the adults directly in a freshly seeded NGM plate. This approach has to be done daily; otherwise, the progeny will grow and will be retained by the cell-strainer alongside the adults.</div>
<div class="note">
<span class="note-title">Note:</span> Use at least three separate plates containing transgenic worms for each experimental condition.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Non-starved and well-fed animals should be used. Avoid overcrowding that could lead to starvation, which influences organism physiology.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>OP50 <i>E. coli</i>
</td>
<td>Caenorhabditis Genetics Center</td>
<td>OP50-1</td>
</tr>
<tr>
<td>HT115(DE3) <i>E. coli</i>
</td>
<td>Caenorhabditis Genetics Center</td>
<td>HT115(DE3)</td>
</tr>
<tr>
<td>pL4440 in HT115(D3) <i>E. coli</i>
</td>
<td>Tavernarakis lab</td>
<td>Tavernarakis lab #6</td>
</tr>
<tr>
<td>
<i>nth-1</i> in pL4440 in HT115(DE3) <i>E. coli</i>
</td>
<td>Tavernarakis lab</td>
<td>Tavernarakis lab #2328</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Agar</td>
<td>Sigma-Aldrich</td>
<td>Cat# 05040</td>
</tr>
<tr>
<td>Bacto-peptone</td>
<td>BD, Bacto<sup>TM</sup>
</td>
<td>Cat# 211677</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>EMD Millipore</td>
<td>Cat# 106404</td>
</tr>
<tr>
<td>Magnesium sulfate (MgSO<sub>4</sub>)</td>
<td>Sigma-Aldrich</td>
<td>Cat# M7506</td>
</tr>
<tr>
<td>Cholesterol</td>
<td>SERVA Electrophoresis</td>
<td>Cat# 17101.01</td>
</tr>
<tr>
<td>Calcium chloride dehydrate (CaCl<sub>2</sub> 2H<sub>2</sub>O)</td>
<td>Sigma-Aldrich</td>
<td>Cat# C5090</td>
</tr>
<tr>
<td>di-Potassium hydrogen phosphate (K<sub>2</sub>HPO<sub>4</sub>)</td>
<td>EMD Millipore</td>
<td>Cat# 137010</td>
</tr>
<tr>
<td>Potassium dihydrogen phosphate (KH<sub>2</sub>PO<sub>4</sub>)</td>
<td>EMD Millipore</td>
<td>Cat# 104873</td>
</tr>
<tr>
<td>di-Sodium hydrogen phosphate (Na<sub>2</sub>HPO<sub>4</sub>)</td>
<td>EMD Millipore</td>
<td>Cat# 106586</td>
</tr>
<tr>
<td>Ethanol absolute</td>
<td>Sigma-Aldrich</td>
<td>Cat# 1070174000</td>
</tr>
<tr>
<td>Streptomycin</td>
<td>Sigma-Aldrich</td>
<td>Cat# S6501</td>
</tr>
<tr>
<td>Nystatin</td>
<td>Sigma-Aldrich</td>
<td>Cat# N3503</td>
</tr>
<tr>
<td>Tetracycline hydrochloride</td>
<td>PanReac AppliChem</td>
<td>Cat# A2228. 0025</td>
</tr>
<tr>
<td>Ampicillin sodium salt</td>
<td>PanReac AppliChem</td>
<td>Cat# A0839.0100</td>
</tr>
<tr>
<td>Isopropyl β-D-1-thiogalactopyranoside (IPTG)</td>
<td>Sigma-Aldrich</td>
<td>Cat# I5502</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>Sigma-Aldrich</td>
<td>Cat# Y0875</td>
</tr>
<tr>
<td>Tryptone</td>
<td>Sigma-Aldrich</td>
<td>Cat# T7293</td>
</tr>
<tr>
<td>AgaPure<sup>TM</sup> Agarose</td>
<td>Canvax</td>
<td>Cat# AG006</td>
</tr>
<tr>
<td>Sodium hypochlorite (NaOCl) solution</td>
<td>EMD Millipore</td>
<td>Cat# 105614</td>
</tr>
<tr>
<td>Levamisole hydrochloride</td>
<td>Sigma-Aldrich</td>
<td>Cat# PHR1798</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>a-synuclein<sup>wt</sup>]</td>
<td>R. Blakely Lab</td>
<td>BY273</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: N2; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP]</td>
<td>Tavernarakis / Nilsen lab</td>
<td>IR2514</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: <i>nth-1(ok724)III</i>
</td>
<td>Caenorhabditis Genetics Center</td>
<td>RB877</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: <i>nth-1(ok724)III</i>; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>a-synuclein<sup>wt</sup>]</td>
<td>Tavernarakis / Nilsen lab</td>
<td>IR2355</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: <i>sid-1(pk3321)</i>; <i>baln11</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>a-synuclein<sup>wt</sup>]; <i>baln33</i>[p<sub><i>dat-1</i></sub>SID-1; p<sub><i>myo-2</i></sub>mCherry]</td>
<td>Caldwell Lab</td>
<td>UA196</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: <i>sid-1(pk3321)V</i>; <i>uIs69</i>[p<sub><i>unc-119</i></sub>SID-1; p<sub><i>myo-2</i></sub>mCherry]V; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>a-synuclein<sup>wt</sup>]</td>
<td>Tavernarakis lab</td>
<td>IR2531</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: <i>rde-1(ne219)V</i>; <i>kzIs9</i>[p<sub><i>lin-26</i></sub>RDE-1; p<sub><i>lin-26</i></sub>NLS::GFP; <i>rol-6(su1006)</i>]; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>a-synuclein<sup>wt</sup>]</td>
<td>Tavernarakis lab</td>
<td>IR2945</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: <i>rde-1(ne219)V</i>; <i>kbIs7</i>[p<sub><i>nhx-2</i></sub>RDE-1; <i>rol-6(su1006)</i>]; <i>Is</i>[p<sub><i>dat-1</i></sub>GFP; p<sub><i>dat-1</i></sub>a-synuclein<sup>wt</sup>]</td>
<td>Tavernarakis lab</td>
<td>IR2947</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pL4440 (control or empty vector)</td>
<td>Fire lab</td>
<td>Addgene Plasmid #1654</td>
</tr>
<tr>
<td>
<i>nth-1</i> in pL4440</td>
<td>Tavernarakis lab</td>
<td>Tavernarakis lab #2328</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Zen</td>
<td>Zeiss</td>
<td><a href="https://www.zeiss.com/microscopy/us/products/microscope-software/zen-lite.html">https://www.zeiss.com/microscopy/us/products/microscope-software/zen-lite.html</a></td>
</tr>
<tr>
<td>EVOS FL AUTO 2 software</td>
<td>Thermo Fisher Scientific</td>
<td><a href="https://www.thermofisher.com/gr/en/home/technical-resources/software-downloads/evos-fl-auto2-imaging-system-software-download.html">https://www.thermofisher.com/gr/en/home/technical-resources/software-downloads/evos-fl-auto2-imaging-system-software-download.html</a></td>
</tr>
<tr>
<td>GraphPad Prism software package</td>
<td>GraphPad Software Inc., San Diego, USA</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Incubators for stable temperature (20 &amp; 37°C)</td>
<td>BIOBASE</td>
<td>BJPX – B80II</td>
</tr>
<tr>
<td>Nikon dissecting stereomicroscope</td>
<td>Nikon</td>
<td>SZM645</td>
</tr>
<tr>
<td>Zeiss epifluorescence stereomicroscope</td>
<td>Zeiss</td>
<td>Zeiss SteReo Lumar V12</td>
</tr>
<tr>
<td>EVOS cell imaging systems</td>
<td>Thermo Fisher scientific</td>
<td>EVOS FL Auto 2</td>
</tr>
<tr>
<td>Zeiss confocal microscope</td>
<td>Zeiss</td>
<td>Zeiss LSM 710</td>
</tr>
<tr>
<td>Microscope slides 75 × 25 × 1</td>
<td>Marienfeld-Superior</td>
<td>Cat# 1000612</td>
</tr>
<tr>
<td>Microscope cover glass 18 × 18</td>
<td>Marienfeld-Superior</td>
<td>Cat# 0101030</td>
</tr>
<tr>
<td>Petri plates, 60 × 15 mm</td>
<td>Sigma-Aldrich</td>
<td>Cat# P5481</td>
</tr>
<tr>
<td>Petri plates, 92 × 16 mm</td>
<td>Sigma-Aldrich</td>
<td>Cat# P5481</td>
</tr>
<tr>
<td>Cell Strainer 40 μm,</td>
<td>pluriSelect</td>
<td>43-57040-50</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>NGM medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>3 g</td>
<td>50 mM</td>
</tr>
<tr>
<td>Bacto-peptone</td>
<td>2.5 g</td>
<td>2.5 mg/mL</td>
</tr>
<tr>
<td>streptomycin</td>
<td>0.2 g</td>
<td>0.2 mg/mL</td>
</tr>
<tr>
<td>Agar</td>
<td>17 g</td>
<td>17 mg/mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>900 mL</td>
<td>–</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>
<sup><a href="#dtbl1fnlowast">a</a></sup><b>Add up to 1 L</b>
</td>
<td><b>–</b></td>
</tr>
</tbody>
</table>
<p id="dtbl1fnlowast" class="table-footnote"><sup>a</sup>Autoclave 900 mL NGM medium and cool it to 55°C–60°C, and add 1 mL MgSO<sub>4</sub> (1 M stock solution; final concentration: 1 mM), 1 mL cholesterol (5 mg/mL stock solution; final concentration: 5 μg/mL), 1 mL 1 mL CaCl<sub>2</sub> (1 M stock solution; final concentration: 1 mM), 1 mL nystatin (10 mg/mL stock solution; final concentration: 10 μg/mL), 25 mL KPO<sub>4</sub> (1 M stock solution; final concentration: 25 mM). Fill with sterilized ddH<sub>2</sub>O up to 1 L.</p>
<table id="undtbl2">
<caption>M9 buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>3 g</td>
<td>3 mg/mL</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>6 g</td>
<td>6 mg/mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>5 g</td>
<td>5 mg/mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>up to 1 L</td>
<td>–</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td><b>–</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave M9 buffer, and add 1 mL MgSO<sub>4</sub> (1 M stock solution) to 1 L M9 buffer so that the final concentration of MgSO<sub>4</sub> is 1 mM. Store M9 buffer for up to 2 months at 4°C.</div>
<table id="undtbl3">
<caption>1 M KPO<sub>4</sub> buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>102.2 g</td>
<td>0.75 M</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>57.06 g</td>
<td>0.32 M</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>1 L</td>
<td>–</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td><b>–</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave and store KPO<sub>4</sub> buffer (pH:6) at room temperature (RT). Store KPO<sub>4</sub> buffer for up to 2 months at RT.</div>
<table id="undtbl4">
<caption>Bleaching solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaOH (5N)</td>
<td>1 mL</td>
<td>0.5 N</td>
</tr>
<tr>
<td>5% Sodium hypochlorite (NaOCl) solution</td>
<td>2 mL</td>
<td>25%</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>7 mL</td>
<td>–</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>10 mL</b></td>
<td><b>–</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store bleaching solution for a week at room temperature.</p>
<table id="undtbl5">
<caption>Nystatin stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nystatin</td>
<td>10 mg/mL</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Ethanol</td>
<td>70%</td>
<td>35 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>15 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store nystatin stock solution for up to 5 months at 4°C.</p>
<table id="undtbl6">
<caption>Ampicillin stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ampicillin sodium salt</td>
<td>10 mg/mL</td>
<td>1 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store ampicillin stock solution for up to 6 months at −20°C</p>
<table id="undtbl7">
<caption>Tetracycline stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tetracycline hydrochloride</td>
<td>10 mg/mL</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Ethanol</td>
<td>70%</td>
<td>35 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>15 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store tetracycline stock solution for up to 3 months at −20°C.</p>
<table id="undtbl8">
<caption>Levamisole solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Levamisole hydrochloride</td>
<td>0.5 M</td>
<td>1.2 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store levamisole stock solution for up to 5 months at 4°C.</p>
<table id="undtbl9">
<caption>M9/levamisole solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Levamisole (0.5 M)</td>
<td>20 mM</td>
<td>400 μL</td>
</tr>
<tr>
<td>M9 buffer</td>
<td>N/A</td>
<td>15 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>15 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store M9/levamisole stock solution for up to 2 weeks at 4°C.</p>
<table id="undtbl10">
<caption>LB liquid medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>5 g</td>
<td>5 mg/mL</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>5 g</td>
<td>5 mg/mL</td>
</tr>
<tr>
<td>Tryptone</td>
<td>10 g</td>
<td>10 mg/mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>up to 1 L</td>
<td>–</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td><b>–</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave and store the LB medium for up to 3 weeks at room temperature.</p>
<table id="undtbl11">
<caption>LB agar plates</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>5 g</td>
<td>5 mg/mL</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>5 g</td>
<td>5 mg/mL</td>
</tr>
<tr>
<td>Tryptone</td>
<td>10 g</td>
<td>10 mg/mL</td>
</tr>
<tr>
<td>Agar</td>
<td>15 g</td>
<td>15 mg/mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>up to 1 L</td>
<td>–</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td><b>–</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave the LB agar medium. Air-cool the medium to 55°C–60°C. Pour 18 mL LB agar medium per petri dish (92 × 16 mm). Store the LB agar plates for up to 3 weeks at 4°C.</p>
<div class="note">
<span class="note-title">Note:</span> Prepare 100 μg/mL ampicillin and/or 10 μg/mL tetracycline LB agar plates: Add 170 μL ampicillin (100 mg/mL stock solution) and/or 17 μL tetracycline (10 mg/mL stock solution) on LB agar plates and spread the plates by using a sterilized glass spreader.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Agarose pad preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>This step describes how to prepare 2% agarose pads combined with M9/levamisole buffer that will be used for mounting the nematodes. Several methods (e.g., agarose pads, polystyrene nanoparticles, microfluidic chips and anesthetics) have been developed and utilized to immobilize and mount nematodes for long- or short-term imaging (<a href="#bib5">Dong et al., 2018</a>; <a href="#bib9">Kim et al., 2013</a>; <a href="#bib14">Mondal et al., 2016</a>; <a href="#bib15">Mondal and Koushika, 2014</a>). The use of agarose pads is the most common methods because it is a simple, low-cost and versatile technique.</p>
<ol>
<li>Weigh 0.5 g of agarose in a 50 mL glass beaker.</li>
<li>Add 25 mL of M9 buffer.</li>
<li>Place and heat the mixture in a microwave until the agarose will be dissolved.</li>
<li>Stir the mixture periodically and keep it warm on a heating plate.</li>
<li>Place an empty microscope slide on the bench.</li>
<li>Add a drop of 30 μL of agarose solution (2% final concentration) in the middle of the slide.</li>
<li>Take a second microscope slide and place it on the top of the agarose drop and press down gently to flatten it.</li>
<li>After 30 s remove carefully the top microscope slide.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Several agarose pads can be prepared (<a href="#bib19">Ramachandran et al., 2015</a>; <a href="#bib21">Rieckher and Tavernarakis, 2017</a>; <a href="#bib29">Walston and Hardin, 2010</a>; <a href="#bib30">Wang et al., 2021</a>). Leave the top microscope slide as a cover to eliminate the evaporation and preserve the agarose pads humidity longer (∼ 30 min).</div>
<ol start="9">
<li>Proceed with the sample preparation.</li>
</ol>
<h3 id="sec3.2">Mounting nematodes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–10 min</div>
<p>This step describes the mounting process of transgenic nematodes on the agarose pads before the image acquisition.</p>
<ol start="10">
<li>Add 10 μL 20 mM M9/levamisole buffer on the agarose pad.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Levamisole is an agonist of cholinergic receptors and influences directly neuronal function and physiology (<a href="#bib4">Culetto et al., 2004</a>; <a href="#bib6">Fleming et al., 1997</a>; <a href="#bib10">Kim et al., 2001</a>; <a href="#bib18">Podbilewicz and Gruenbaum, 2006</a>). Thus, lower concentration of levamisole (e.g., 1 or 5 mM) can be used, especially for experiments that require extended periods of live-cell imaging.</div>
<ol start="11">
<li>Use an eyelash, which is glued on a toothpick or platinum wire worm picker, to pick and transfer the transgenic nematodes into the M9/levamisole droplet. Transfer 15–20 animals per drop.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Transfer carefully the transgenic animals one by one into the droplet to avoid nematodes injury or even death. Old nematodes are more sensitive to mechanical forces.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A worm pick can be used instead of eyelash to transfer the transgenic animals into the droplet.</div>
<ol start="12">
<li>Place gently a coverslip on the top of the transgenic nematodes. <a href="#sec6.1">Troubleshooting 1</a>.</li>
<li>Use nail polish and seal the coverslip on the agarose pad.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Sealing of the coverslips is required for the maintenance of humidity during the imaging process. Sealing is highly recommended for long-term imaging experiments.</div>
<ol start="14">
<li>Proceed to microscopic examination of the specimens.</li>
</ol>
<h3 id="sec3.3">Image acquisition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="15">
<li>Use an epifluorescence (e.g., EVOS FL AUTO 2) or a confocal (e.g., Zeiss LSM 710) microscope combined with a camera.</li>
<li>Place the prepared slides under the microscope.</li>
<li>Properly focus and detect single transgenic nematodes co-expressing green fluorescent protein (GFP) and α-syn in dopaminergic neurons.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1527-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Dopaminergic neuronal circuit in <i>C. elegans</i>
</div>
<p>(A) Dopaminergic neuronal circuit consists of 8 neurons the <i>C. elegans</i> hermaphrodite. Transgenic nematodes expression cytosolic GFP under the <i>dat-1</i> promoter display two pairs of CEPs and a pair of ADE neurons in the anterior part and a pair of PDE neurons in the posterior part of the nematode body.</p>
<p>(B) CEPs and ADEs neurons form a well-structure network in the head region (i). Dopaminergic neuronal circuit is gradually deteriorated with age in transgenic animals expressing α-synuclein. CEPs and ADEs neurons present dendritic or outgrowths loss (ii), entire loss of their cell bodies (iii), axonal and some blebbing (iv). Remnants of neuronal cell bodies (asterisks), intact neuronal processes (arrows) and axonal beading (arrowheads) are depicted. Scale bars, 500 μm and 50 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The hermaphroditic nematode <i>C. elegans</i> has 8 dopaminergic neurons, 6 (CEPs and ADEs) in the head region and 2 (PDEs) in the middle body (<a href="#fig1">Figure 1</a>A). The male nematodes display additional 6 dopaminergic neuronal cells, which are located in the tail (<a href="#bib26">Sulston et al., 1975</a>).</div>
<ol start="18">
<li>Capture z-stack images of the head and the middle body region by using 20× objective lens. <a href="#sec6.3">Troubleshooting 2</a>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> EVOS FL AUTO 2 software and Zeiss ZEN (black edition) were used for image acquisition.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use the same imaging settings (e.g., lens, magnifiers, filter exposure time, laser intensity, gain etc.) during the entire microscopic examination for all the condition that will be used for analysis.</div>
<ol start="19">
<li>Save the acquired images.</li>
<li>Process the acquired z-stack images with the Zeiss ZEN software (black version), to obtain the maximum intensity projection (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1527-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Maximum intensity projection by using the Zeiss ZEN software</div>
<p>Step 1. Open an acquired Z-stack image (.czi file) with the Zeiss ZEN software (black version; <a href="https://www.zeiss.com/microscopy">https://www.zeiss.com/microscopy</a>); Step 2. In the processing tab under Method section select “Maximum intensity projection”; Step 3. Select the Z-stack image and press select button followed by apply button to create the Maximum intensity projection image; Step 4. Export the maximum projection intensity image to your respective drive.</p>
</figcaption>
</figure>
</li>
<li>Proceed to the analysis of the acquired images.</li>
</ol>
<h3 id="sec3.4">Scoring degeneration of dopaminergic neurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="22">
<li>Open the acquired images by using ZEN software.</li>
<li>Examine the transgenic nematodes and evaluate dopaminergic neuronal loss by the following complementary methods:
<ol type="a">
<li>Measure average fluorescence pixel intensity from dopaminergic neurons (CEPs, ADEs, PDEs) expressing GFP under the dopaminergic neuron specific promoter using the Zeiss ZEN software (blue version).</li>
<li>Open an acquired image with ZEN Blue software, click graphics, select draw spine contour, mark the GFP positive neuronal cell body, and note the mean intensity value.
<div class="note">
<span class="note-title">Note:</span> Create similar shape or size of region of interests (ROIs) to improve and increase the measurements’ accuracy.</div>
</li>
<li>Follow this step to score the intensity for each CEP, ADE and PDE neurons (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1527-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Measure average fluorescence pixel intensity by using the Zeiss ZEN software</div>
<p>Step 1. Open an acquired maximum intensity projection image with the Zeiss ZEN software (blue version; <a href="https://www.zeiss.com/microscopy">https://www.zeiss.com/microscopy</a>); Step 2. Click the graphics tab and select “draw spine contour”; Step 3. Draw the contour around CEPs or ADEs neurons expressing cytosolic GFP; Step 4. The pen symbol marks the end of contour and provides area and intensity mean values, which can be noted in excel file for further analysis.</p>
</figcaption>
</figure>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Image analysis software, such as FIJI (<a href="https://imagej.net/software/fiji/">https://imagej.net/software/fiji/</a>) or Qupath (<a href="https://qupath.github.io">https://qupath.github.io</a>), could be alternatively utilized to quantify the average fluorescence pixel intensity from dopaminergic neurons.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use transgenic animals (IR2514) expressing only the cytosolic GFP under the <i>dat-1</i> promoter to examine whether its activity is affected by the respective treatments or genetic backgrounds.</div>
</li>
<li>Score neurodegeneration by monitoring specific morphological features of dopaminergic neurons in the head region. <i>C. elegans</i> contains 3 pairs of head dopaminergic neurons, which form well-structured neuronal processes (<a href="#fig1">Figure 1</a>Bi). Nematodes expressing α-synuclein in dopaminergic neurons display age-dependent degeneration that is characterized by dendritic or outgrowths loss (<a href="#fig1">Figure 1</a>Bii), entire loss of neuronal cell bodies (<a href="#fig1">Figure 1</a>Biii), axonal and some blebbing (<a href="#fig1">Figure 1</a>Biv). Count the neuronal cells bodies with wild type morphology to signify the survival of dopaminergic neurons in the total number of imaged animals. Use 3 different plates with 15–20 animals per strain/condition in each experimental set up. <a href="#sec6.5">Troubleshooting 3</a>, <a href="#sec6.7">4</a>, and <a href="#sec6.9">5</a>.
<div class="note">
<span class="note-title">Note:</span> The assessment of the morphological alterations in dopaminergic neurons is a subjective method. Thus, (1) positive controls should always be included in each experimental set up and (2) each experiment must be conducted in a double-blind manner.</div>
</li>
</ol>
</li>
</ol>
<ol start="24">
<li>Import and analyze the data by using a statistical software package (e.g., GraphPad Prism).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Repeat each assay at least three times.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Transgenic animals expressing the human α-synuclein together with the cytosolic GFP in dopaminergic neurons are a well-characterized and established PD nematode model (<a href="#bib3">Cooper and Van Raamsdonk, 2018</a>). The accumulation of α-synuclein aggregates promotes the gradual degeneration of dopaminergic neurons with age (<a href="#fig4">Figures 4</a>A and <a href="#fig4">4</a>B).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1527-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Neurodegeneration assessment in PD nematode model</div>
<p>(A–F) Transgenic animals co-expressing the human α-synuclein protein and cytoplasmic GFP in dopaminergic neurons display age-dependent degeneration signified by (A) altered cellular morphology and (B) decreased GFP intensity (n = 35 nematodes per condition; ns p&gt;0.05, ∗∗∗p&lt;0.001; one-way ANOVA followed by Bonferroni’s multiple comparison test). Scale bars, 50 μm and 5 μm. Pan-neuronal (C) and dopaminergic neuron (D) specific depletion of NTH-1 protect against α-synuclein-mediated toxicity, whereas knocking down of <i>nth-1</i> in hypodermis (E) and intestine (F) does not promote the survival of dopaminergic neurons (n= 5 biological replicates, 40 animals per condition; ∗∗∗p&lt;0.001; unpaired <i>t-</i>test). Error bars denote SEM.</p>
</figcaption>
</figure>
<p>Recently, the current protocol was used to demonstrate an intricate link between the base excision repair (BER) pathway efficiency and PD pathophysiology (<a href="#bib23">SenGupta et al., 2021</a>). The BER deficient <i>nth-1(ok724)</i> mutants display enhanced neuroprotection against α-synuclein during ageing (<a href="#fig4">Figures 4</a>A and <a href="#fig4">4</a>B). To examine whether NTH-1 DNA glycosylase acts in a cell-autonomous manner and regulates neuronal viability, PD nematodes were subjected to RNAi against <i>nth-1</i> in specific tissues. Interestingly, pan-neuronal or dopaminergic neuron-specific knock down of NTH-1 is sufficient to facilitate neuronal survival (<a href="#fig4">Figures 4</a>C and <a href="#fig4">4</a>D), whereas hypodermal or intestinal RNAi against <i>nth-1</i> do not provide any neuroprotective effect (<a href="#fig4">Figures 4</a>E and <a href="#fig4">4</a>F).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The neurodegeneration assessment assay described here in <i>C. elegans</i> PD model is quite robust with high reproducibility. However, live cell imaging of <i>C. elegans</i> PD neurons can be challenging for the first time. Therefore, it is advisable to train the eye by following the neurons in a few settings. As the scoring is subjective, (1) the use of well-known inducers or inhibitors of α-synuclein-induced neurodegeneration and (2) blinded repeats are the optimum solution to avoid observer bias (<a href="#bib3">Cooper and Van Raamsdonk, 2018</a>; <a href="#bib13">Maulik et al., 2017</a>; <a href="#bib16">Offenburger et al., 2018</a>).</p>
<p>During slide preparation, coverslip must be placed gently to avoid air bubbles and rupturing of animals. Imaging should be performed as quickly as possible after slide preparation to avoid dehydration of the worms, which can significantly reduce the quality of images. In addition, worms might not survive long-term imaging. Thus, time must be calibrated and planned well.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Excessive rupture of transgenic nematode bodies is happening upon the placement of the coverslip (step 12).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>To avoid excessive rapture of nematodes, increase either the volume of M9/levamisole drop or the number of the animals in the drop.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Low image quality (e.g., blurry image) due to (A) increased fluorescence intensity during the acquisition process or (B) extensive bubble formation (step 18).</p>
<h3 id="sec6.4">Potential solution</h3>
<ul>
<li>The presence of the residual bacteria during the worm picking and transferring into the M9/levamisole droplet should be reduced. Therefore, the use of the eyelash is recommended to decrease the bacterial load in the specimen. Alternatively, worms could also be picked to an NGM (or RNAi) plate with no bacterial lawn &amp; left to move around for ∼30 min prior to transfer with the eyelash if residual bacteria persist.</li>
<li>Place gently and slowly the coverslip on the top of the sample. Oblique-angle lowering down is recommended to avoid bubble formation.</li>
</ul>
<h3 id="sec6.5">Problem 3</h3>
<p>Although 1-day-old control transgenic animals co-expressing GFP and α-synuclein under the <i>dat-1</i> promoter do not display dopaminergic neuronal loss, excessive degeneration can be observed in young transgenic nematodes (step 23).</p>
<h3 id="sec6.6">Potential solution</h3>
<ul>
<li>Reduce the scan and imaging time. Long-term imaging process could induce photodamage and neuronal death. In case of long-term or time-lapse imaging process, 10% agarose pads combined with polystyrene nanoparticles could be alternatively used (<a href="#bib9">Kim et al., 2013</a>; <a href="#bib20">Rieckher et al., 2018</a>).</li>
<li>Remove dead and censored nematodes from the imaging process. Neuronal morphology can be affected by several genetics and environmental factors, including internal hatching, starvation, temperature fluctuations among others, that influence animals’ viability.</li>
<li>Use freshly prepared agarose pads and M9/levamisole solution to maintain humidity throughout the imaging process. Sample dehydration alters the morphological features of neurons.</li>
<li>Avoid the use of sodium azide (NaN3) as an anesthetic. Sodium azide could induce necrotic cell death even at low concentration upon long-term exposure (<a href="#bib1">Artal-Sanz et al., 2006</a>; <a href="#bib22">Sato et al., 2008</a>).</li>
<li>Long-term cultivation of nematodes in the laboratory results in the accumulation of random genomic mutations that could affect animals’ physiology. Every three months, thaw and renew <i>C. elegans</i> strains to maintain their genetic background.</li>
</ul>
<h3 id="sec6.7">Problem 4</h3>
<p>7-day-old control transgenic animals co-expressing GFP and α-synuclein do not present increased levels of neurodegeneration (step 23).</p>
<h3 id="sec6.8">Potential solution</h3>
<ul>
<li>Avoid temperature fluctuations and use well-fed animals. Nematodes have to be grown under optimal physiological conditions. Several stress conditions, such as starvation and short-periods of heat shock, are shown to promote neuroprotection (<a href="#bib8">Griffin et al., 2019</a>; <a href="#bib11">Kourtis et al., 2012</a>; <a href="#bib24">Steinkraus et al., 2008</a>).</li>
<li>Do not cultivate continuously <i>C. elegans</i> strains. Regularly thaw new nematodes to avoid genetic drift. Newly thawed nematodes should be cultured for at least three generation before being utilized in any experimental set up.</li>
</ul>
<h3 id="sec6.9">Problem 5</h3>
<p>The <i>E. coli</i> OP50 and HT115(D3<i>)</i> bacterial strains differentially affect α-synuclein-induced neurodegeneration in transgenic nematodes of the same age and/or genetic background (step 23).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The <i>E. coli</i> OP50 and HT115(D3<i>)</i> are two distinct bacterial strains, which differ in their nutrient and metabolite composition (<a href="#bib2">Coolon et al., 2009</a>; <a href="#bib7">Gracida and Eckmann, 2013</a>; <a href="#bib12">MacNeil et al., 2013</a>; <a href="#bib17">Pang and Curran, 2014</a>). Therefore, these two distinct bacterial food sources differentially impact <i>C. elegans</i> gene expression, cellular responses and physiology (<a href="#bib7">Gracida and Eckmann, 2013</a>; <a href="#bib12">MacNeil et al., 2013</a>; <a href="#bib17">Pang and Curran, 2014</a>; <a href="#bib27">Urrutia et al., 2020</a>; <a href="#bib32">Zhou et al., 2019</a>). To avoid these differential diet effects, RNAi feeding protocol can be performed by using a genetically engineered OP50 strain enabling the silencing of gene of interest (OP50i) (<a href="#bib31">Xiao et al., 2015</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Artal-Sanz, M., Samara, C., Syntichaki, P., and Tavernarakis, N. (2006). Lysosomal biogenesis and function is critical for necrotic cell death in Caenorhabditis elegans. J. Cell Biol. <i>173</i>, 231-239. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref1">View at publisher</a></p>
<p id="bib2">Coolon, J.D., Jones, K.L., Todd, T.C., Carr, B.C., and Herman, M.A. (2009). Caenorhabditis elegans genomic response to soil bacteria predicts environment-specific genetic effects on life history traits. PLoS Genet. <i>5</i>, e1000503. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref2">View at publisher</a></p>
<p id="bib3">Cooper, J.F. and Van Raamsdonk, J.M. (2018). Modeling Parkinson's disease in C. elegans. J. Parkinsons Dis. <i>8</i>, 17-32. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref3">View at publisher</a></p>
<p id="bib4">Culetto, E., Baylis, H.A., Richmond, J.E., Jones, A.K., Fleming, J.T., Squire, M.D., Lewis, J.A., and Sattelle, D.B. (2004). The Caenorhabditis elegans unc-63 gene encodes a levamisole-sensitive nicotinic acetylcholine receptor alpha subunit. J. Biol. Chem. <i>279</i>, 42476-42483. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref4">View at publisher</a></p>
<p id="bib5">Dong, L., Cornaglia, M., Krishnamani, G., Zhang, J., Mouchiroud, L., Lehnert, T., Auwerx, J., and Gijs, M.A.M. (2018). Reversible and long-term immobilization in a hydrogel-microbead matrix for high-resolution imaging of Caenorhabditis elegans and other small organisms. PLoS One <i>13</i>, e0193989. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref5">View at publisher</a></p>
<p id="bib6">Fleming, J.T., Squire, M.D., Barnes, T.M., Tornoe, C., Matsuda, K., Ahnn, J., Fire, A., Sulston, J.E., Barnard, E.A., Sattelle, D.B., et al. (1997). Caenorhabditis elegans levamisole resistance genes lev-1, unc-29, and unc-38 encode functional nicotinic acetylcholine receptor subunits. J. Neurosci. <i>17</i>, 5843-5857. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref6">View at publisher</a></p>
<p id="bib7">Gracida, X. and Eckmann, C.R. (2013). Fertility and germline stem cell maintenance under different diets requires nhr-114/HNF4 in C. elegans. Curr. Biol. <i>23</i>, 607-613. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref7">View at publisher</a></p>
<p id="bib8">Griffin, E.F., Scopel, S.E., Stephen, C.A., Holzhauer, A.C., Vaji, M.A., Tuckey, R.A., Berkowitz, L.A., Caldwell, K.A., and Caldwell, G.A. (2019). ApoE-associated modulation of neuroprotection from Abeta-mediated neurodegeneration in transgenic Caenorhabditis elegans. Dis. Model. Mech. <i>12</i>, dmm037218. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref8">View at publisher</a></p>
<p id="bib9">Kim, E., Sun, L., Gabel, C.V., and Fang-Yen, C. (2013). Long-term imaging of Caenorhabditis elegans using nanoparticle-mediated immobilization. PLoS One <i>8</i>, e53419. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref9">View at publisher</a></p>
<p id="bib10">Kim, J., Poole, D.S., Waggoner, L.E., Kempf, A., Ramirez, D.S., Treschow, P.A., and Schafer, W.R. (2001). Genes affecting the activity of nicotinic receptors involved in Caenorhabditis elegans egg-laying behavior. Genetics <i>157</i>, 1599-1610. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref10">View at publisher</a></p>
<p id="bib11">Kourtis, N., Nikoletopoulou, V., and Tavernarakis, N. (2012). Small heat-shock proteins protect from heat-stroke-associated neurodegeneration. Nature <i>490</i>, 213-218. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref11">View at publisher</a></p>
<p id="bib12">MacNeil, L.T., Watson, E., Arda, H.E., Zhu, L.J., and Walhout, A.J. (2013). Diet-induced developmental acceleration independent of TOR and insulin in C. elegans. Cell <i>153</i>, 240-252. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref12">View at publisher</a></p>
<p id="bib13">Maulik, M., Mitra, S., Bult-Ito, A., Taylor, B.E., and Vayndorf, E.M. (2017). Behavioral phenotyping and pathological indicators of Parkinson's disease in C. elegans models. Front. Genet. <i>8</i>, 77. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref13">View at publisher</a></p>
<p id="bib14">Mondal, S., Hegarty, E., Martin, C., Gokce, S.K., Ghorashian, N., and Ben-Yakar, A. (2016). Large-scale microfluidics providing high-resolution and high-throughput screening of Caenorhabditis elegans poly-glutamine aggregation model. Nat. Commun. <i>7</i>, 13023. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref14">View at publisher</a></p>
<p id="bib15">Mondal, S. and Koushika, S.P. (2014). Microfluidic devices for imaging trafficking events in vivo using genetic model organisms. Methods Mol. Biol. <i>1174</i>, 375-396. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref15">View at publisher</a></p>
<p id="bib16">Offenburger, S.L., Ho, X.Y., Tachie-Menson, T., Coakley, S., Hilliard, M.A., and Gartner, A. (2018). 6-OHDA-induced dopaminergic neurodegeneration in Caenorhabditis elegans is promoted by the engulfment pathway and inhibited by the transthyretin-related protein TTR-33. PLoS Genet. <i>14</i>, e1007125. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref16">View at publisher</a></p>
<p id="bib17">Pang, S. and Curran, S.P. (2014). Adaptive capacity to bacterial diet modulates aging in C. elegans. Cell Metab. <i>19</i>, 221-231. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref17">View at publisher</a></p>
<p id="bib18">Podbilewicz, B. and Gruenbaum, Y. (2006). Live imaging of Caenorhabditis elegans: preparation of samples. CSH Protoc. <i>6</i>, pdb.prot4601. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref18">View at publisher</a></p>
<p id="bib19">Ramachandran, P.V., Mutlu, A.S., and Wang, M.C. (2015). Label-free biomedical imaging of lipids by stimulated Raman scattering microscopy. Curr. Protoc. Mol. Biol. <i>109</i>, 30.33.31-30.33.17. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref19">View at publisher</a></p>
<p id="bib20">Rieckher, M., Markaki, M., Princz, A., Schumacher, B., and Tavernarakis, N. (2018). Maintenance of proteostasis by P body-mediated regulation of eIF4E availability during aging in Caenorhabditis elegans. Cell Rep. <i>25</i>, 199-211.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref20">View at publisher</a></p>
<p id="bib21">Rieckher, M. and Tavernarakis, N. (2017). P-body and stress granule quantification in Caenorhabditis elegans. Bio Protoc. <i>7</i>, e2108. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref21">View at publisher</a></p>
<p id="bib22">Sato, E., Suzuki, T., Hoshi, N., Sugino, T., and Hasegawa, H. (2008). Sodium azide induces necrotic cell death in rat squamous cell carcinoma SCC131. Med. Mol. Morphol. <i>41</i>, 211-220. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref22">View at publisher</a></p>
<p id="bib23">SenGupta, T., Palikaras, K., Esbensen, Y.Q., Konstantinidis, G., Galindo, F.J.N., Achanta, K., Kassahun, H., Stavgiannoudaki, I., Bohr, V.A., Akbari, M., et al. (2021). Base excision repair causes age-dependent accumulation of single-stranded DNA breaks that contribute to Parkinson disease pathology. Cell Rep. <i>36</i>, 109668. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref23">View at publisher</a></p>
<p id="bib24">Steinkraus, K.A., Smith, E.D., Davis, C., Carr, D., Pendergrass, W.R., Sutphin, G.L., Kennedy, B.K., and Kaeberlein, M. (2008). Dietary restriction suppresses proteotoxicity and enhances longevity by an hsf-1-dependent mechanism in Caenorhabditis elegans. Aging Cell <i>7</i>, 394-404. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref24">View at publisher</a></p>
<p id="bib25">Stiernagle, T. (2006). Maintenance of C. elegans (WormBook), pp. 1-11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref25">View at publisher</a></p>
<p id="bib26">Sulston, J., Dew, M., and Brenner, S. (1975). Dopaminergic neurons in the nematode Caenorhabditis elegans. J. Comp. Neurol. <i>163</i>, 215-226. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref26">View at publisher</a></p>
<p id="bib27">Urrutia, A., Garcia-Angulo, V.A., Fuentes, A., Caneo, M., Legue, M., Urquiza, S., Delgado, S.E., Ugalde, J., Burdisso, P., and Calixto, A. (2020). Bacterially produced metabolites protect C. elegans neurons from degeneration. PLoS Biol. <i>18</i>, e3000638. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref27">View at publisher</a></p>
<p id="bib28">Vangheel, M., Traunspurger, W., and Spann, N. (2014). Effects of the antibiotic tetracycline on the reproduction, growth and population growth rate of the nematode Caenorhabditis elegans. Nematology <i>16</i>, 19-29. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref28">View at publisher</a></p>
<p id="bib29">Walston, T. and Hardin, J. (2010). An agar mount for observation of Caenorhabditis elegans embryos. Cold Spring Harb Protoc. <i>2010</i>, pdb prot5540. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref29">View at publisher</a></p>
<p id="bib30">Wang, X., Li, T., Hu, J., Feng, Z., Zhong, R., Nie, W., Yang, X., and Zou, Y. (2021). In vivo imaging of a PVD neuron in Caenorhabditis elegans. STAR Protoc. <i>2</i>, 100309. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref30">View at publisher</a></p>
<p id="bib31">Xiao, R., Chun, L., Ronan, E.A., Friedman, D.I., Liu, J., and Xu, X.Z. (2015). RNAi interrogation of dietary modulation of development, metabolism, behavior, and aging in C. elegans. Cell Rep. <i>11</i>, 1123-1133. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref31">View at publisher</a></p>
<p id="bib32">Zhou, J.J., Chun, L., and Liu, J.F. (2019). A comprehensive understanding of dietary effects on C. elegans physiology. Curr. Med. Sci. <i>39</i>, 679-684. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00144-7/sref32">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Prof. Nektarios Tavernarakis (<a href="mailto:tavernarakis@imbb.forth.gr">tavernarakis@imbb.forth.gr</a>).</p>
<h4>Materials availability</h4>
<p>All materials are available upon request.</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze any datasets.</p>
<h3>Acknowledgments</h3>
<p>We thank R. Blakely and G. Caldwell for sharing the BY273 and UA196 strains, respectively. Some of the nematode strains used in this work were provided by the Caenorhabditis Genetics Center, which is funded by the <a href="https://doi.org/10.13039/100000097">National Center for Research Resources</a> of the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a>. This work was supported by the South East Norway Regional Health Authority (Grant no. 2015029). T.S. was supported by an institutional grant from <a href="https://doi.org/10.13039/501100012446">Akershus University Hospital</a>. K.P. is supported by Fondation Santé. N.T. is funded by grants from the <a href="https://doi.org/10.13039/100010663">European Research Council</a> (ERC –GA695190– MANNA), the European Commission Framework Programmes, and the Greek Ministry of Education.</p>
<h3>Author contributions</h3>
<p>K.P. and T.S. conducted experiments; K.P. and T.S wrote the manuscript; H.N. and N.T wrote and edited the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>