<article>
<section>
<h1 id="header">Optimized protocol to create deletion in adherent cell lines using CRISPR/Cas9 system</h1>
<p><time datetime="2021-10-29">Published: October 29, 2021</time></p>
<p>Umer Farooq<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a></sup> and Dimple Notani<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>National Centre for Biological Sciences, Tata Institute for Fundamental Research, Bangalore, Karnataka 560065, India</p>
<p id="aff2"><sup>2</sup>The University of Trans-Disciplinary Health Sciences and Technology, IVRI road, Bangalore, Karnataka 560064, India</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:dnotani@ncbs.res.in">dnotani@ncbs.res.in</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100857">10.1016/j.xpro.2021.100857</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The CRISPR/Cas9 system is a powerful tool for genome editing and is adaptable for a wide range of applications. Here, we have put together a step-by-step protocol for generating knockout cell lines (coding or non-coding region) using CRISPR/Cas9 tool. The protocol below has been tested on adherent cell lines such as HeLa and MCF7. However, it may easily be adapted to other adherent cell lines with minor variations.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref4">Farooq et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>CRISPR-Cas9 protocol optimized for deletions in adherent and immortalized cell lines</li>
<li>The protocol covers complete flow from sgRNA cloning to the validation of deletion</li>
<li>This simple protocol can be used in low-resource settings</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1121-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 3 days</div>
<p>The CRISPR/Cas9 system employs single guide RNAs (sgRNAs) to direct Cas9, an endonuclease protein, to a specific region of DNA (<a href="#bib1">Adli, 2018</a>). When Cas9 is recruited, it creates a double-stranded break in the DNA at the specified region (<a href="#bib6">Jinek et al., 2013</a>). A pair of sgRNAs flanking a region on either side would cause the deletion of that region. This system has been engineered to accommodate a wide range of applications. Currently, this technology is used for genome editing, transcriptional activation and repression, epigenetic modifications, and live DNA/RNA imaging, among other things (<a href="#bib5">Gilbert et al., 2013</a>; <a href="#bib7">Maeder et al., 2013</a>; <a href="#bib2">Amabile et al., 2016</a>; <a href="#bib8">Qin et al., 2017</a>). This protocol explains how to create specific knockout cell lines step by step. It has been tested on adherent cancerous cell lines such as HeLa (<a href="#bib4">Farooq et al., 2021</a>). However, with minor modifications, it can be easily applied to other adherent cell lines.</p>
<p>The plasmids used in the process will be described first, followed by the design and cloning of sgRNAs. Then we go over how to make viral particles, how to use these viruses to transduce cells, and how to isolate and expand single cells. Finally, we discuss how to use PCRs to screen and validate single cell colony deletion. (Please refer to graphical abstract for these steps).</p>
<p><b>Plasmids</b></p>
<p>In this protocol following plasmids are used: lentiCas9-Blast for Cas9 (Addgene #52962); pgRNA-humanized (Addgene #44248) for sgRNA cloning; pCMV-VSVG (Addgene #8454) and psPAX2 (Addgene #12260) for packaging of viruses.</p>
<p><b>Prepare the plasmids (lentiCas9-Blast; pgRNA-humanized; pCMV-VSVG and psPAX2);</b></p>
<p>To improve transfection efficiency and obtain high-titer viral particles, high-quality plasmid preparation is needed. To amplify the plasmids, we use the DH5α E. coli strain. We prepare plasmids with the QIAGEN Plasmid Midi Kit. The supplier's protocol that comes with the kit should be followed when preparing the plasmids.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Although we used the DH5α strain, we recommend either Stbl3 (Thermo C737303) or NEB Stable E. coli strain (NEB C3040I) for lentiviral plasmid amplification. Lentiviral constructs have a proclivity for recombination, and these strains have been modified to have a lower recombination rate. Furthermore, when compared to DH5α, these strains produce higher yields for lentiviral constructs.</div>
<p><b>Coat the cell culture dishes with poly-D-lysine</b></p>
<p>Add 1 mL of 0.1 mg/mL poly-D-lysine to 35 mm culture dish. Swirl the dish to completely cover it. Incubate for 2–3 min at room temperature (25°C). Allow the dish to dry completely after removing the poly-D-lysine solution. Wash the dish twice with 1 mL of DPBS and allow it to dry completely.</p>
<h3 id="sec1.1">Genomic DNA isolation buffer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 2 h</div>
<p>The recipe for the genomic DNA isolation buffer is as follows:</p>
<table id="undtbl1">
<caption>Genomic DNA Isolation Buffer Recipe</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5M NaCl</td>
<td>100 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1M Tris-Cl (pH 8.0)</td>
<td>10 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>0.5M EDTA (pH 8.0)</td>
<td>25 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>0.1 mg/mL</td>
<td>X μl</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>43.5 mL</td>
</tr>
<tr>
<td>Total</td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">The genomic DNA isolation buffer should be stored at room temperature (25°C) without proteinase K. It can be stored for two weeks.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Proteinase K is a labile enzyme. It must be added to the buffer freshly.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and viral strains</td>
</tr>
<tr>
<td>DH5α Competent Cells</td>
<td>Thermo</td>
<td>18265017</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides and recombinant proteins</td>
</tr>
<tr>
<td>DMEM</td>
<td>Gibco</td>
<td>10569–010</td>
</tr>
<tr>
<td>FBS</td>
<td>Gibco</td>
<td>26140079</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Gibco</td>
<td>15070–063</td>
</tr>
<tr>
<td>DPBS</td>
<td>Gibco</td>
<td>D8537</td>
</tr>
<tr>
<td>Puromycin</td>
<td>Gibco</td>
<td>A11138-03</td>
</tr>
<tr>
<td>Poly-D-Lysine</td>
<td>Sigma</td>
<td>P6407</td>
</tr>
<tr>
<td>0.25% Trypsin-EDTA (1<b>×</b>)</td>
<td>Gibco</td>
<td>25200–072</td>
</tr>
<tr>
<td>BstXI</td>
<td>NEB</td>
<td>R0113L</td>
</tr>
<tr>
<td>XhoI</td>
<td>NEB</td>
<td>R0146S</td>
</tr>
<tr>
<td>Buffer 3.1</td>
<td>NEB</td>
<td>B7203S</td>
</tr>
<tr>
<td>1 Kb Plus DNA Ladder</td>
<td>NEB</td>
<td>N3200</td>
</tr>
<tr>
<td>Ampicillin sodium salt</td>
<td>Sigma</td>
<td>A0166</td>
</tr>
<tr>
<td>Hexadimethrine bromide (Polybrene)</td>
<td>Sigma</td>
<td>H9268</td>
</tr>
<tr>
<td>LB broth</td>
<td>HiMedia</td>
<td>M575</td>
</tr>
<tr>
<td>LB broth with agar</td>
<td>Sigma</td>
<td>L3272</td>
</tr>
<tr>
<td>Lipofectamine 2000</td>
<td>Invitrogen</td>
<td>11668–019</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>NEB</td>
<td>M02–02M</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Gibco</td>
<td>31985–070</td>
</tr>
<tr>
<td>Kapa Taq DNA Polymerase</td>
<td>Kapa Biosystems</td>
<td>KK1014</td>
</tr>
<tr>
<td>Agarose</td>
<td>Sigma</td>
<td>A9539</td>
</tr>
<tr>
<td>KOD Hot Start DNA Polymerase</td>
<td>Merck</td>
<td>71086–4</td>
</tr>
<tr>
<td>Proteinase K, recombinant, PCR Grade</td>
<td>Roche</td>
<td>17106500</td>
</tr>
<tr>
<td>Phenol:chloroform:isoamyl (PCI) alcohol</td>
<td>Ambion</td>
<td>AM9732</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>NEB</td>
<td>M0202L</td>
</tr>
<tr>
<td>T4 DNA Ligase Buffer</td>
<td>NEB</td>
<td>B0202S</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>QIAGEN Plasmid Midi Kit</td>
<td>QIAGEN</td>
<td>12143</td>
</tr>
<tr>
<td>QIAprep Spin Miniprep Kit</td>
<td>QIAGEN</td>
<td>27106</td>
</tr>
<tr>
<td>QIAquick Gel Extraction Kit</td>
<td>QIAGEN</td>
<td>28706</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>QIAGEN</td>
<td>28106</td>
</tr>
<tr>
<td>QIAamp DNA Mini Kit</td>
<td>QIAGEN</td>
<td>51304</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>HeLa</td>
<td>ATCC</td>
<td>CCL-2</td>
</tr>
<tr>
<td>HEK293FT</td>
<td>Thermo</td>
<td>R70007</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Universal reverse oligo</td>
<td>Sigma</td>
<td>CTAGTACTCGAGAAAAAAAGCACCGACTCGGTGCCAC</td>
</tr>
<tr>
<td>sgRNA sequencing oligo</td>
<td>Sigma</td>
<td>CCCTGCCCCGGTTAATTTGC</td>
</tr>
<tr>
<td>PCRin oligos</td>
<td>Sigma</td>
<td>N/A</td>
</tr>
<tr>
<td>PCRout oligos</td>
<td>Sigma</td>
<td>N/A</td>
</tr>
<tr>
<td>63-Nucleotide-long DNA guide sequence</td>
<td>Sigma</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pgRNA-humanized</td>
<td>Addgene</td>
<td>#44248</td>
</tr>
<tr>
<td>lentiCas9-Blast</td>
<td>Addgene</td>
<td>#52962</td>
</tr>
<tr>
<td>pCMV-VSVG</td>
<td>Addgene</td>
<td>#8454</td>
</tr>
<tr>
<td>psPAX2</td>
<td>Addgene</td>
<td>#12260</td>
</tr>
<tr>
<td colspan="3">Others</td>
</tr>
<tr>
<td>96-Well cell culture plate</td>
<td>Thermo</td>
<td>167314</td>
</tr>
<tr>
<td>48-Well cell culture plate</td>
<td>Thermo</td>
<td>150687</td>
</tr>
<tr>
<td>10 cm Cell culture dish</td>
<td>Thermo</td>
<td>150466</td>
</tr>
<tr>
<td>6 well Cell culture plate</td>
<td>Thermo</td>
<td>140675</td>
</tr>
<tr>
<td>35 mm Dishes</td>
<td>Thermo</td>
<td>150318</td>
</tr>
<tr>
<td>15 mL Centrifuge tube</td>
<td>Tarsons</td>
<td>546021</td>
</tr>
<tr>
<td>50 mL Centrifuge tube</td>
<td>Tarsons</td>
<td>546041</td>
</tr>
<tr>
<td>1.5 mL Centrifuge tube</td>
<td>Tarsons</td>
<td>500010</td>
</tr>
<tr>
<td>Petri dish 90 mm</td>
<td>Tarsons</td>
<td>460090</td>
</tr>
<tr>
<td>0.45 Micron filters</td>
<td>Millipore</td>
<td>SLHV033RS</td>
</tr>
<tr>
<td>Reagent reservoirs</td>
<td>Thermo</td>
<td>8094</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Designing and cloning of sgRNAs in pgRNA-humanized vector</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [∼10 Days]</div>
<p>To delete a specific genomic DNA region, a pair of sgRNAs targeting either side of the deletion region is required. First, we will go over how to use the CRISPOR tool to design guide sequences complementary to the flanks of the genomic region to be knocked out. Once the guide sequences are designed, we will walk you through the process of cloning them in the pgRNA-humanized vector.</p>
<ol>
<li>
<b>Designing guide sequences for region of interest.</b> The steps outlined below can be used to obtain the nucleotide sequence of the target region:
<ol type="a">
<li>Search the region of interest in the designated assembly of UCSC genome browser (<a href="https://genome.ucsc.edu/">https://genome.ucsc.edu/</a>).</li>
<li>Select your target region.</li>
<li>Obtain the flanking region sequence of targeted DNA by selecting the flanks.</li>
<li>Click on ‘View’ option.</li>
<li>In the dropdown menu, click on ‘DNA’ and then click on ‘Get DNA’.
<div class="critical">
<span class="critical-title">Critical:</span> We recommend designing at least two separate pairs of guide sequences for each target.</div>
<div class="note">
<span class="note-title">Note:</span> The size of the flanks would depend on the objective of experiment. For deletion of a gene or a non-coding region, larger flanks (50–100 bp) can be selected. For smaller deletions like deletion of transcription factor binding motifs, guide sequences should be designed very close to region to be deleted, thus limiting the flexibly of selecting the flanks.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is not necessary to target the complete gene in order to knock it out. Targeting one or a few exons along with promoter is sufficient to perturb the gene.</div>
</li>
<li>Paste this sequence into the CRISPOR tool (<a href="http://crispor.tefor.net">http://crispor.tefor.net</a>) (<a href="#bib3">Concordet and Haeussler, 2018</a>) and choose the correct genome (organism) and PAM that is recognized by the Cas9 protein in use.
<div class="note">
<span class="note-title">Note:</span> The Cas9 protein used in the study recognizes NGG as the PAM site.</div>
</li>
<li>Look for guide sequences with high specificity and few predicted off-targets.</li>
<li>Add the following nucleotide sequences to your guide sequence:<br> GGAGAACCACCTTGTTGG<u>NNNNNNNNNNNNNNNNNNNN</u>GTTTTAGAGCTAGAAATAGCAAGTT (where (N)<sub>20</sub> is the guide sequence).</li>
<li>Get the 63-nucleotide long DNA guide sequence synthesized.
<div class="note">
<span class="note-title">Note:</span> 63 nucleotide long guide sequences can be synthesized in desalted form. We routinely purchase these oligos from local vendors but they can be purchased from IDT (Integrated DNA Technology).</div>
<div class="note">
<span class="note-title">Note:</span> We have successfully created lines with deletions ranging from 1 Kb to 3 Kb with this protocol. However, it can be used to knockout smaller or larger regions as well.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>
<b>Oligos designing for validation of knockout clones.</b> To screen the colonies for deletion, oligos for PCR validation are to be designed. Design two oligo sets, one within the target region (PCRin) and the other outside the target region (PCRout), as shown in the <a href="#fig1">Figure 1</a>. Get the designed surveyor oligos synthesized.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1121-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic illustrating the design of oligos for PCR validation of knockout clones</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Oligos can be designed manually or with the help of a primer designing tool. The knockout amplicon of less than 500 bp is recommended.</div>
<ol start="3">
<li>Cloning of guide sequences in pgRNA-humanized vector
<ol type="a">
<li>For sgRNA PCR amplification, the designed 63 nucleotides long guide sequence is used as a forward oligo, universal reverse oligo (See <a href="#key-resources-table">key resources table</a>) is used as a reverse oligo and the pgRNA-humanized vector as a template (<a href="#fig2">Figure 2</a>). To amplify the full-length sgRNA, set up a 100 μL PCR reaction with any proofreading polymerase. For KOD DNA Polymerase following reaction is used:
<table id="undtbl2">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>10<b>×</b> Buffer</td>
<td>10 μL</td>
<td>1<b>×</b>
</td>
</tr>
<tr>
<td>25 mM MgSO<sub>4</sub>
</td>
<td>6 μL</td>
<td>1.5 mM</td>
</tr>
<tr>
<td>dNTPs (2 mM each)</td>
<td>10 μL</td>
<td>0.2 mM</td>
</tr>
<tr>
<td>10 uM Forward Oligo (guide sequence with overhangs)</td>
<td>3 μL</td>
<td>0.3 μM</td>
</tr>
<tr>
<td>10 μM Universal Reverse Oligo</td>
<td>3 μL</td>
<td>0.3 μM</td>
</tr>
<tr>
<td>Template (pgRNA-humanized vector)</td>
<td>X μl</td>
<td>30 ng</td>
</tr>
<tr>
<td>KOD Polymerase</td>
<td>2 μL</td>
<td>0.02 U/μL</td>
</tr>
<tr>
<td>Water</td>
<td>X μl</td>
<td> </td>
</tr>
<tr>
<td>Total Volume</td>
<td>100 μL</td>
<td> </td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Universal reverse oligo can be used for amplification of any sgRNA that is to be cloned in pgRNA-humanized vector.</div>
</li>
<li>Run 5 μL of the PCR product on 1.8% agarose gel. An amplicon of 133 bp size will be visible (<a href="#fig3">Figure 3</a>).</li>
<li>Clean up the remaining PCR product with the QIAquick PCR Purification Kit after verifying the success of PCR.</li>
<li>Elute the DNA in 25 μL of nuclease-free water.</li>
<li>Set up the digestion of purified PCR product with BstXI and XhoI restriction enzymes as follows:
<table id="undtbl3">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Purified PCR Product</td>
<td>20 μL</td>
<td>X ng</td>
</tr>
<tr>
<td>BstXI</td>
<td>1 μL</td>
<td>10 units</td>
</tr>
<tr>
<td>XhoI</td>
<td>1 μL</td>
<td>20 units</td>
</tr>
<tr>
<td>Buffer 3.1</td>
<td>2.5 μL</td>
<td>1<b>×</b>
</td>
</tr>
<tr>
<td>Water</td>
<td>X μl</td>
<td>N/A</td>
</tr>
<tr>
<td>Total volume</td>
<td>25 μL</td>
<td> </td>
</tr>
</tbody>
</table>
</li>
<li>Similarly, set up the digestion of pgRNA-humanized vector with BstXI and XhoI restriction enzymes to linearize it:
<table id="undtbl4">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNA</td>
<td>X μl</td>
<td>1 μg</td>
</tr>
<tr>
<td>BstXI</td>
<td>1 μL</td>
<td>10 units</td>
</tr>
<tr>
<td>XhoI</td>
<td>1 μL</td>
<td>20 units</td>
</tr>
<tr>
<td>Buffer 3.1</td>
<td>2.5 μL</td>
<td>1<b>×</b>
</td>
</tr>
<tr>
<td>Water</td>
<td>X μl</td>
<td>N/A</td>
</tr>
<tr>
<td>Total volume</td>
<td>25 μL</td>
<td> </td>
</tr>
</tbody>
</table>
</li>
<li>Incubate the digestion mixture overnight (14–16 h) in water bath set at 37°C.</li>
<li>Run both the digested vector and insert on 1.0% and 1.8% agarose gels respectively (<a href="#fig4">Figures 4</a>A and 4B).</li>
<li>Excise the gel slices containing DNA fragments (8.3 kb for vector and approx. 125 bp for sgRNA) from the respective gels.</li>
<li>Extract the DNA from the gel slices with QIAquick Gel Extraction Kit from Qiagen. To extract DNA from the gel, follow the instructions included with the kit.</li>
<li>To integrate the digested sgRNAs into the pgRNA-humanized vector, set up a ligation reaction as follows:
<table id="undtbl5">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Digested pgRNA humanized vector</td>
<td>X μl</td>
<td>50 ng</td>
</tr>
<tr>
<td>Digested sgRNA</td>
<td>X μl</td>
<td>2.25 ng (3:1 Ratio)</td>
</tr>
<tr>
<td>T4 DNA Ligase Buffer</td>
<td>2 μL</td>
<td>1<b>×</b>
</td>
</tr>
<tr>
<td>T4 DNA Ligase (NEB)</td>
<td>0.5 μL</td>
<td>200 U</td>
</tr>
<tr>
<td>Water</td>
<td>X μl</td>
<td>N/A</td>
</tr>
<tr>
<td>Total Volume</td>
<td>20 μL</td>
<td> </td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Remember to set up a control ligation reaction that doesn't include the digested sgRNA.</div>
</li>
<li>Incubate the ligation samples overnight (14–16 h) at 16°C.</li>
<li>Transform the ligated plasmid into DH5α strain of E. coli and plate it on agar plate with 100 μg/mL ampicillin.</li>
<li>Next day, inoculate at least 10 colonies from the agar plate into tubes containing 3 mL LB broth supplemented with 100 ug/mL of ampicillin. Allow the bacteria to grow at 37°C for overnight (14–16 h) with constant shaking.</li>
<li>Using the QIAGEN Plasmid Mini kit, isolate the plasmid from the inoculated colonies. Follow the protocol that comes with the kit.</li>
<li>For screening the positive clones, set up a digestion reaction (as in Step 3e) with BstXI and XhoI (scale down the reaction to 15 μL).
<div class="note">
<span class="note-title">Note:</span> In order to clearly see the release, minimum 1 ug of plasmid DNA should be taken for restriction digestion.</div>
</li>
<li>Run the digested samples on 1.8% agarose gel (<a href="#fig5">Figure 5</a>).</li>
<li>Perform Sanger sequencing on clones that show approx. 125 bp fragment release. Confirm that the clone contains your desired sgRNA with no mutations.
<div class="critical">
<span class="critical-title">Critical:</span> Test whether your sgRNAs work before proceeding for the rest of the protocol. Transfect the cell line of interest with lentiCas9 Blast and pgRNA-humanized containing your sgRNAs. Select the cells with puromycin (3 μg/ml) for 48 h, harvest the cells and isolate genomic DNA and set up PCRs with PCRout oligos. Presence of both wild type and knockout (faint) bands confirms that the sgRNAs are functional. In that case one can proceed with the rest of the protocol.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1121-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Graphical representation of the PCR amplification and cloning of sgRNAs in pgRNA-humanized vector</div>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1121-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Agarose gel image showing the amplified full length sgRNA. The amplicon is visible at 133 bp</div>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1121-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Agarose gel images showing A) linearized pgRNA-humanized vector and B) sgRNA double digested with BstXI and XhoI</div>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1121-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Post screening gel image showing the approx. 125 bp release from 8 different positive clones</div>
</figcaption>
</figure>
<h3 id="sec2.2">Production of lentiviruses in HEK293FT cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼6 days</div>
<p>This section describes how lentiviruses carrying sgRNA constructs and lentiCas9-Blast vector are made. Once the lentiviruses are ready, the subsequent steps will guide you to transduce the cells.</p>
<ol start="4">
<li>
<b>Cell culture.</b> HEK293FT and HeLa cell lines are maintained/cultured in DMEM supplemented with 10% FBS (v/v) and 5% PenStrep (v/v) at 37°C with 5% CO<sub>2</sub>. Passage both cell lines every 3 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Early passage (less than 30) HEK293FT cell line should be used for viral production.</div>
<div class="critical">
<span class="critical-title">Critical:</span> HEK293FT cells should be cultured on poly-D lysine-treated plates before transfection because these cells detach from the untreated plate surface following transfection.</div>
<ol start="5">
<li>Cell preparation for transfection (Late Evening)
<ol type="a">
<li>Aspirate medium and add 2 mL of 0.25% Trypsin-EDTA to 10 cm dish of HEK293FT cells. Incubate the plate for 1–2 min at 37°C.</li>
<li>Remove trypsin from the plate carefully to avoid disturbing the cells. Gently tap the plate to dislodge the cells.</li>
<li>Collect and resuspend cells in 3 mL of fresh culture media.</li>
<li>Seed 0.8 × 10<sup>6</sup> cells on poly-D lysine-coated 35 mm dishes ∼14 h before transfection. Place the plate back into incubator.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> At the time of transfection, the cells should be 60–65% confluent. Both low and high confluency will reduce the transfection efficiency, and as a result, also the viral titer.</div>
<ol start="6">
<li>Plasmid transfection (Early Morning) Transfection mixture of DNA and Lipofectamine 2000 is to be prepared in Opti-MEM medium as follows:
<ol type="a">
<li>Add 100 μL Opti-MEM and 3 μg (0.6 μg each of psPAX2, pCMV-VSVG, pgRNA-humanized sgRNA1, pgRNA-humanized sgRNA2 and lentiCas9-Blast) DNA to tube A.</li>
<li>Add 150 μL Opti-MEM and 6 μL lipofectamine 2000 to tube B.</li>
<li>Mix the contents of both tubes thoroughly by pipetting.</li>
<li>Add the contents of tube B to tube A and mix thoroughly.</li>
<li>Short spin tube A to collect the liquid from the tube walls.</li>
<li>Incubate the transfection mixture at room temperature (25°C) for 25 min.</li>
<li>Meanwhile, remove the spent medium from the plate to be transfected and add 0.75 mL of fresh Opti-MEM to it.</li>
<li>Add the transfection mixture (tube A) to the cells after 25 min. Swirl the plate to evenly distribute the mixture. Incubate the cells for 6 h at 37°C.</li>
<li>Remove the transfection mixture from the cells after 6 h of transfection. Add 2 mL fresh media to the plate and incubate it for 48 h in the incubator.
<div class="note">
<span class="note-title">Note:</span> The above-mentioned transfection protocol is for 35 mm dishes. If larger dishes are used, the recipe should be scaled up accordingly.</div>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Viral Particle collection and transduction
<ol type="a">
<li>After 48 h, take the plates out of the incubator and transfer the medium containing viral particles to a sterile 15 mL falcon tube (Falcon A).</li>
<li>Store the Falcon A at 4°C and incubate the cells for next 24 h with 1 mL of fresh media at 37°C.</li>
<li>Seed 0.7 × 10<sup>6</sup> HeLa cells (or any cell line of interest) in 35 mm dish approx. 14 h prior to transduction.</li>
<li>Remove the media from HEK293FT cell plate after 24 h and transfer it to Falcon A.</li>
<li>Filter the entire viral particle-containing media by passing it through 0.45-micron syringe-driven filters. Collect the filtered media in fresh sterile 15 mL tubes.</li>
<li>Check the volume of filtered media you have and add 8 μg/mL polybrene.</li>
<li>Add 2 mL of this filtered media to HeLa cells in 35 mm dishes (or any other cell line of interest).</li>
<li>Remove the medium after 12–14 h of transduction. Incubate the cells for 24 h with 2 mL fresh media.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If you can't work with/make viruses, cells can be transfected with lentiCas9 Blast and pgRNA-humanized vector with lipofectamine. Because the efficiency of transfection is less than transduction, in that situation approximately 45%–60% of cells will die after puromycin selection.</div>
<h3 id="sec2.3">Selection of sgRNA-positive cells and seeding of single cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼20 days</div>
<p>The selection of cells transduced with sgRNAs, followed by single cell seeding in the 96 well plates is discussed in this section.</p>
<ol start="8">
<li>Selection of cells transfected with sgRNAs
<ol type="a">
<li>To select the sgRNA positive cells, add 3 μg/mL puromycin to the transduced cells for 48 h.</li>
<li>Remove the media and wash the cells twice with DPBS to remove any dead cells.</li>
<li>Add 0.3 mL trypsin and incubate at 37°C for 1–2 min.</li>
<li>Carefully remove the trypsin and tap the plate to dislodge the cells.</li>
<li>Collect the cells and resuspend them in 2 mL of fresh media without puromycin.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Puromycin concentration varies from cell line to cell line. Use the concentration which is desired for cell line of your interest.</div>
<ol start="9">
<li>Single cells seeding in 96 well plates
<ol type="a">
<li>Using a cell counter, count the number of cells per mL of media. Count the cells at least three times and if the difference between the three readings is small, the average should be used to minimize the variation.</li>
<li>Add the media and cells to the sterile reagent reservoirs.</li>
<li>Dilute the cells such that 100 μL of DMEM contains only one cell.</li>
<li>Using the 100 μL multi-channel pipette, add 100 μL of DMEM-containing cells to the 96 well plate.
<div class="critical">
<span class="critical-title">Critical:</span> Seed at least five 96 well plates to get clones with homozygous deletion for aneuploid cell lines.</div>
<div class="note">
<span class="note-title">Note:</span> Instead of seeding single cells by dilution method, cell sorters can be used to seed cells in 96 well plates. Sorting can be done using the mCherry marker, which is coded from the pgRNA humanized vector along with puromycin.</div>
</li>
<li>The next day, observe each well under light microscope at 20<b>×</b> resolution to label the wells containing single cell.</li>
<li>Disregard the wells that have no cell or more than one cell.</li>
<li>Allow the single cells to form colonies.</li>
<li>Change the media every 3–4 days, taking care not to lose the cells.
<div class="note">
<span class="note-title">Note:</span> It takes approx. 15–20 days for HeLa cells to form colonies that are big enough to be processed further.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If the single cells aren't visible the following day, wait 2–3 days for the cells to form minute colonies that can be easily seen under a microscope. Mark the wells that contain single colonies.</div>
<div class="note">
<span class="note-title">Note:</span> At least 25–30 wells in a 96-well plate would contain single cells. From five 96-well plates, 100–125 single clones are expected for final screening.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.4">Screening of knockout clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 5 days</div>
<p>This final section will cover the genomic DNA isolation from the cells, followed by PCRs to screen for the colonies carrying homozygous knockout of the region of interest.</p>
<ol start="10">
<li>Genomic DNA isolation
<ol type="a">
<li>Add 50 μL of trypsin to the colonies and transfer them to 48-well plates with 0.5 mL media. Allow the cells to grow for 2–3 days.</li>
<li>Trypsinize the cells once they are 80–90% confluent by adding 100 μL trypsin to each well. Resuspend the cells in 1 mL media.</li>
<li>Transfer half of the cells to 1.5 mL tubes for screening, and the other half should be seeded in the same wells.</li>
<li>Spin the tubes at 600 rcf for 5 min. Discard the supernatant and remove the excess media from the tubes by inverting them on a tissue paper.</li>
<li>Resuspend the pellet in 300 μL of genomic DNA isolation buffer. Pipette thoroughly until the sample's viscosity is reduced. Incubate the samples overnight (14–16 h) at 50°C.</li>
<li>Add an equal volume of Phenol:Chloroform:Isoamyl (PCI) alcohol to the sample and vortex for 1 min.</li>
<li>Spin the tubes at 13500 rcf for 12 min at 4°C.</li>
<li>Collect the aqueous phase with care without disturbing the interphase, and transfer it to a new 1.5 mL tube.
<div class="note">
<span class="note-title">Note:</span> Genomic DNA should also be isolated from wild type cells. The nature of clones must be determined by comparison to this control.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If the aqueous phase appears viscous, add more digestion buffer to the sample and repeat the PCI extraction.</div>
</li>
<li>To the sample, add 1/10th volume of 3M sodium acetate (pH 5.5) and 2.5 volumes of absolute ethanol. Vortex the tubes and place them in a −20°C freezer for an hour.</li>
<li>Pellet the precipitated DNA by centrifuging the tubes for 12 min at 4°C at 13500 rcf.</li>
<li>To remove residual salt, wash the pellet twice with 70% ethanol. Decant the ethanol and allow the pellet to air dry for 15–20 min.</li>
<li>Dissolve the DNA pellet in 100 μL TE buffer.
<div class="note">
<span class="note-title">Note:</span> If necessary, RNA contaminants can be removed by treating the sample with RNase A.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Extracted DNA can be stored at −20°C for further processing.</div>
</li>
</ol>
</li>
</ol>
<ol start="11">
<li>Validation of knockout clones
<div class="note">
<span class="note-title">Note:</span> In our study, we used Kapa Taq DNA polymerase for validation PCRs. Other Taq-based polymerases can also be used, but the supplier's protocol must be followed in that case.</div>
<ol type="a">
<li>Prepare the PCR mix for the validation of clones (PCRout) as follows:
<table id="undtbl6">
<thead>
<tr>
<th>Components</th>
<th>15 μL reaction</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>10<b>×</b> Buffer A or B</td>
<td>1.5 μL</td>
<td>1<b>×</b>
</td>
</tr>
<tr>
<td>10 mM dNTP</td>
<td>0.3 μL</td>
<td>200 μM</td>
</tr>
<tr>
<td>10 uM Forward Oligo</td>
<td>0.6 μL</td>
<td>0.4 μM</td>
</tr>
<tr>
<td>10 uM Reverse Oligo</td>
<td>0.6 μL</td>
<td>0.4 μM</td>
</tr>
<tr>
<td>5 U/μL Kapa Taq DNA Polymerase</td>
<td>0.06 μL</td>
<td>1 U</td>
</tr>
<tr>
<td>Water</td>
<td>Up to 15 μL</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</li>
<li>Add 2 μL of genomic DNA obtained from Step 10l to the PCR mix.
<div class="critical">
<span class="critical-title">Critical:</span> The concentration of DNA in manually isolated samples cannot be accurately measured. If the desired amplicon is not observed, more volume of template DNA can be taken.</div>
</li>
<li>Adjust the PCR to the following parameters:
<table id="undtbl7">
<thead>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Duration</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>Initial denaturation</b></td>
<td>95°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td><b>Denaturation</b></td>
<td>95°C</td>
<td>30 S</td>
<td rowspan="3">40 cycles</td>
</tr>
<tr>
<td><b>Annealing</b></td>
<td>Tm- 5°C</td>
<td>30 S</td>
</tr>
<tr>
<td><b>Extension</b></td>
<td>72°C</td>
<td>1 min/kb</td>
</tr>
<tr>
<td><b>Final extension</b></td>
<td>72°C</td>
<td>1 min/kb</td>
<td>1</td>
</tr>
</tbody>
</table>
</li>
<li>Run all of the samples on an agarose gel (<a href="#fig6">Figures 6</a>A and 6B).
<div class="critical">
<span class="critical-title">Critical:</span> Determine the percentage of gel based on the expected size of amplicon.</div>
<table id="undtbl8">
<thead>
<tr>
<th>Amplicon size</th>
<th>Gel percentage</th>
</tr>
</thead>
<tbody>
<tr>
<td>&gt;3 Kb</td>
<td>0.8%</td>
</tr>
<tr>
<td>1–3 Kb</td>
<td>1%</td>
</tr>
<tr>
<td>500 bp – 1 Kb</td>
<td>1.2%</td>
</tr>
<tr>
<td>100 bp – 500 bp</td>
<td>1.5%</td>
</tr>
<tr>
<td>&lt;100 bp</td>
<td>2%</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Only the wild type amplicon will be visible in the control sample. Clones with no deletion will have an amplicon similar to the control. Heterozygous clones will produce two distinct amplicons: 1. wild type and 2. knockout amplicon. The homozygous deletion clones show a single amplicon of expected size after deletion.</div>
</li>
<li>Complement the above PCR results with another round of PCRs with PCRin oligo set to confirm the outcome of the clones.
<div class="critical">
<span class="critical-title">Critical:</span> The control sample will only show one amplicon of the expected molecular weight. Clones with no deletion and heterozygous deletions will also produce an amplicon of the same size as the control sample. Clones with homozygous deletion will not show any amplification.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If the knockout region codes for a protein, the clones can be further validated using western blotting. qRT PCR can be used to test the expression of regions that are transcribed into RNA.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The PCR product from the validated knockout clones should be sent for Sanger sequencing to rule out the possibility of nucleotide insertion or more than expected deletion at the junction of deleted region (<a href="#fig7">Figure 7</a>). Sequence multiple clones for further molecular characterization of deleted line.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1121-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Post PCR validation (PCRout) gel images</div>
<p>(A) Gel image showing amplicons from wild type cells and heterozygous deletion clones.</p>
<p>(B) Gel image showing amplicon from wild type cells and homozygous deletion clone (Figure 6B is used from <a href="#bib4">Farooq et al., 2021</a>).</p>
</figcaption>
</figure>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1121-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. A representative Sanger sequencing chromatogram showing the absence of the deleted region.</div>
<p>Used from <a href="#bib4">Farooq et al., 2021</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>We were able to generate knockout lines of both non-coding elements and coding genes using this method. This protocol has been tested primarily with HeLa cell lines, but it also works well with MCF7 cell lines. This protocol can also be applied to other adherent cell lines. The efficiency of obtaining knockout clones, however, would differ from cell line to cell line. The efficiency of this protocol would also be affected by the cell ploidy level.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol requires co-transfecting at least five different plasmids, reducing the likelihood of obtaining a knockout clone. This limitation can be overcome by using a different vector system that can hold multiple sgRNAs. This would vastly improve protocol efficiency.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>No deletion amplicon observed (step 11)</p>
<h3 id="sec5.2">Potential solution</h3>
<p>There are numerous reasons why a knockout amplicon is not obtained. One possible explanation is that one of the sgRNAs in a pair is ineffective. Rule out this possibility by using the second set of sgRNAs. If the second set of sgRNAs also fails to produce a deletion amplicon, double-check your sgRNA design. Cas9 from various bacterial strains requires a different PAM site. Check to see if you selected the correct bacterial strain when designing the sgRNAs.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>No homozygous knockout line obtained after screening (step 11)</p>
<h3 id="sec5.4">Potential solution</h3>
<p>There are a variety of explanations why a homozygous clone cannot be obtained. One possible explanation is that the clones obtained are not from a single cell. When seeding the cells in a 96-well plate, try breaking up the clumps into single cells. Even, when labeling the wells that contain only single cells, be very careful. Another possibility is that your region of interest is critical for cell viability, and the cells with the homozygous deletion do not survive during the process.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>All cells are dead after antibiotic selection (step 8)</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Puromycin concentration varies between cell lines. Check the optimal puromycin concentration for your cell line ahead of time. This issue could also be caused by a low virus titer. Before transduction, determine the viral titer. If you have a low viral titer, try concentrating viruses from multiple dishes. Low titer can occur for a variety of reasons, including: 1. Plasmid transfection efficiency is low. This could be due to poor plasmid quality. It can also occur as a result of too high or too low cell confluency. 2. The HEK293FT cells in poor health at the time of transfection would also affect the transfection efficiency. 3. The cell line of interest does not tolerate constitutive expression of Cas9 protein. This can be avoided by transiently delivering lentiCas9 blast to the cells. Furthermore, using vectors in which Cas9 expression is inducible rather than constitutive would also help. In such cases, vectors like pCW-Cas9 (#50661) should be used.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>During screening, the PCR does not work for all samples (step 11).</p>
<h3 id="sec5.8">Potential solution</h3>
<p>This could be because the samples carry high salt and residual phenol that inhibit the PCR reaction. Purify these samples using Phenol-chloroform-Isoamyl alcohol once more. The pellet should be washed twice with 70% ethanol. This would aid in the removal of salt.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>Sanger sequencing reveals the presence of a few random nucleotides at the junction of deleted region (step 11).</p>
<h3 id="sec5.10">Potential solution</h3>
<p>These added nucleotides are likely the result of DNA repair process. Send more homozygous knockout clones obtained during screening for Sanger sequencing. Choose the clone that does not have any random nucleotide mutations or has the fewest of all.</p>
<h3 id="sec5.11">Problem 6</h3>
<p>PCRout does not show any wildtype amplicon in some samples, whereas PCRin shows an amplicon of the desired size (step 11).</p>
<h3 id="sec5.12">Potential solution</h3>
<p>This implies that the clones are heterozygous in nature. The region of interest is deleted from some copies of the chromosomes (A problem with aneuploid cells) but not from others. PCRin assay should not show any amplicon for a clean homozygous clone. Clones with only knockout amplicon in PCRout (wild type amplicon should be absent) and no amplicon in PCRin assays should be chosen.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Adli, M. (2018). The CRISPR tool kit for genome editing and beyond. Nat. Commun. <i>9</i>, 1-13. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref1">View at publisher</a></p>
<p id="bib2">Amabile, A., Migliara, A., Capasso, P., Biffi, M., Cittaro, D., Naldini, L., and Lombardo, A. (2016). Inheritable Silencing of endogenous genes by hit-and-run targeted epigenetic editing. Cell <i>167</i>, 219-232. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref2">View at publisher</a></p>
<p id="bib3">Concordet, J.P. and Haeussler, M. (2018). CRISPOR: intuitive guide selection for CRISPR/Cas9 genome editing experiments and screens. Nucleic Acids Res. <i>46</i>, W242-W245. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref3">View at publisher</a></p>
<p id="bib4">Farooq, U., Saravanan, B., Islam, Z., Walavalkar, K., Singh, A.K., Jayani, R.S., Meel, S., Swaminathan, S., and Notani, D. (2021). An interdependent network of functional enhancers regulates transcription and EZH2 loading at the INK4a/ARF locus. Cell Rep. <i>34</i>, 12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref4">View at publisher</a></p>
<p id="bib5">Gilbert, L.A., Larson, M.H., Morsut, L., Liu, Z., Brar, G.A., Torres, S.E., Stern-Ginossar, N., Brandman, O., Whitehead, E.H., Doudna, J.A., et al. (2013). CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes. Cell <i>154</i>, 442-451. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref5">View at publisher</a></p>
<p id="bib6">Jinek, M., East, A., Cheng, A., Lin, S., Ma, E., and Doudna, J. (2013). RNA-programmed genome editing in human cells. eLife <i>2</i>, e00471. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref6">View at publisher</a></p>
<p id="bib7">Maeder, M.L., Linder, S.J., Cascio, V.M., Fu, Y., Ho, Q.H., and Joung, J.K. (2013). CRISPR RNA-guided activation of endogenous human genes. Nat. Methods <i>10</i>, 977-979. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref7">View at publisher</a></p>
<p id="bib8">Qin, P., Parlak, M., Kuscu, C., Bandaria, J., Mir, M., Szlachta, K., Singh, R., Darzacq, X., Yildiz, A., and Adli, M. (2017). Live cell imaging of low- and non-repetitive chromosome loci using CRISPR-Cas9. Nat. Commun. <i>8</i>, 14725. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00563-3/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dimple Notani (<a href="mailto:dnotani@ncbs.res.in">dnotani@ncbs.res.in</a>)</p>
<h4>Materials availability</h4>
<p>No unique reagent was generated in this study.</p>
<h4>Data and code availability</h4>
<p>No unique datasets or data were generated in this study.</p>
<h3>Acknowledgments</h3>
<p>We acknowledge support of the <a href="https://doi.org/10.13039/501100001502">Department of Atomic Energy</a>, Government of India, under project no.12-R&amp;D-TFR-5.04–0800, Department of Biotechnology (BT/PR38396/GET/119/305/2020), and intramural funds from <a href="https://doi.org/10.13039/501100005879">NCBS</a>-TIFR. D.N. is an EMBO Global Investigator. We also acknowledge the funding support from Welcome-IA: IA/1/14/2/501539 (to D.N.). U.F. is supported by <a href="https://doi.org/10.13039/501100001412">CSIR</a> (Govt. of India) senior research fellowship. We are grateful to Henrietta Lacks and to her family members for their contributions to biomedical research</p>
<h3>Author contributions</h3>
<p>The protocol is written by U.F. with feedback from D.N. Authors thank DN Lab members for input.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>