<article>
<section>
<h1 id="header">Luciferase reporter assays to monitor interferon signaling modulation by SARS-CoV-2 proteins</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Maximilian Hirschenberger,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Manuel Hayn,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Alexandre Laliberté,<sup><a href="#aff1">1</a></sup> Lennart Koepke,<sup><a href="#aff1">1</a></sup> Frank Kirchhoff,<sup><a href="#aff1">1</a></sup> and Konstantin Maria Johannes Sparrer<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Molecular Virology, Ulm University Medical Center, 89081, Ulm, Baden-Württemberg, Germany</p>
<p id="fn1"><sup>2</sup>These authors contributed equally</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:Maximilian.Hirschenberger@uni-ulm.de">Maximilian.Hirschenberger@uni-ulm.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:Konstantin.Sparrer@uni-ulm.de">Konstantin.Sparrer@uni-ulm.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100781">10.1016/j.xpro.2021.100781</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We present a protocol for analyzing the impact of SARS-CoV-2 proteins in interferon signaling using luciferase reporter assays. Here, the induction of defined promoters can be quantitatively assessed with high sensitivity and broad linear range. The results are similar to those obtained using qPCR to measure endogenous mRNA induction. The assay requires stringent normalization and confirmation of the results in more physiological settings. The protocol is adaptable for other viruses and other innate immune stimuli.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.celrep.2021.109126">Hayn et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Measure the impact of viral proteins on innate immune activation</li>
<li>Rapid, cost-effective, and reliable procedure that can be adapted to other viruses</li>
<li>Flexible system can be adjusted for different types of immune stimuli</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1023-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Here, we apply luciferase reporter gene assays to rapidly determine the impact of SARS-CoV-2 proteins on interferon (IFN) signaling pathways (for comprehensive reviews see (<a href="#bib14">Platanias, 2005</a>; <a href="#bib15">Randall and Goodbourn, 2008</a>; <a href="#bib20">Stark et al., 1998</a>; <a href="#bib21">Stetson and Medzhitov, 2006</a>)). Bioluminescent reporter systems such as the Luciferase reporter system allow quantitative analysis of protein levels. In principle the emission of light is catalyzed by a bioluminescent reporter which requires a specific substrate and usually energy (ATP). Light emission from the bioluminescent protein can be quantified accurately and corresponds to the amount of protein present in the sample over a high dynamic range. Luciferases are ideal reporters, as their light signal is not dependent on fluorescence, thus avoiding auto-fluorescent effects. In addition, they are not endogenously expressed in human cells. Luciferase-based reporter gene assays to measure gene expression have been used since the late 1980s for various purposes (<a href="#bib4">de Wet et al., 1987</a>; <a href="#bib7">Fujita et al., 1993</a>). However, they are still relevant and popular today as flexible and rapid tools for quantification of gene expression.</p>
<p>In reporter systems, the reporter (here: Luciferase of <i>Photinus pyralis</i>, Firefly Luciferase or short F-Luc) is an exogenous coding region introduced in a reporter plasmid downstream of a promoter of interest, whose activity shall be monitored. The promoters that are induced can be derived from endogenous promoters or be artificially constructed. As an example we analyzed the activity of two selected artificial promoters: The interferon response element (ISRE) stimulated by type I and III IFN (<a href="#bib20">Stark et al., 1998</a>) and the gamma-activated sequence (GAS) induced upon type II IFN signaling. Upon stimulation of cells with type I or III IFNs, signaling cascades are activated that eventually lead to the activation of the transcription factors STAT1 and STAT2. Heterodimers of activated STAT1 and STAT2 together with IRF9 form the ISGF3 complex, that binds to the ISRE regions of the promoter and subsequently drive the transcription of F-Luc (<a href="#fig1">Figure 1</a>). The artificial ISRE reporter (<a href="#bib16">Sanchez et al., 1998</a>) contains 5 repeats of the ISGF3 binding site (ISRE, TAGTTTCACTTTCCC) (<a href="#bib20">Stark et al., 1998</a>) and a minimal TATA box to drive transcription. Similarly, type II interferons drive activation and assembly of activated STAT1 homodimers that assemble on GAS promoter sequences and subsequently drive expression of the F-Luc from the reporter plasmid (<a href="#fig1">Figure 1</a>). The pGAS reporter plasmid (<a href="#bib1">Briken et al., 1995</a>; <a href="#bib16">Sanchez et al., 1998</a>; <a href="#bib20">Stark et al., 1998</a>) contains four repeats of the gamma-activated sequence (GAS, AGTTTCATATTACTCTAAATC) followed by a minimal TATA box with low basal activity. Repetition of the elements allow increased sensitivity (<a href="#fig2">Figures 2</a>A and 2B).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1023-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic depiction of the principle of the assay</div>
<p>Interferons are recognized by dedicated receptors that induce intracellular signaling cascades which eventually culminate in the activation of STAT transcription factors by phosphorylation. Activated STAT binds to enhancer sequences (e.g., ISRE or GAS, see <a href="#fig2">Figure 2</a>) driving the expression of a Luciferase reporter. The amosunt of reporter can be quantified by luminescence. Induction of the reporter gene can be inhibited at multiple steps of the signaling cascade by proteins of SARS-CoV-2. STAT, signal transducer and activator of transcription. Luc, Luciferase. ISRE, Interferon stimulated response element. Gas, gamma-activated sequence. For more details, see the step-by-step description below.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1023-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic depiction of the promoter structures of the pISRE (A) and pGAS (B) reporter constructs</div>
<p>ISRE, Interferon stimulated response element. TSS, transcription start site. F-Luc, Firefly Luciferase. GAS, gamma-activated sequence.</p>
</figcaption>
</figure>
<p>In principle, this protocol can also be applied to quantify other innate immune pathways, such as the activation of innate immune sensors (<a href="#bib19">Sparrer et al., 2012</a>). In addition, it can be easily adapted to assess the impact of any library of viral proteins on innate immune activation. Here, the procedure is described for analyzing the impact of SARS-CoV-2 proteins on type I, II, and III IFN signaling in HEK293T cells using 48-well plates. A similar method was used by us in A549, HeLa, and Vero E6 cells and it may be adapted to other cell types. Cell lines stably containing reporter have been published or are commercially available (e.g., via Invivogen) as an alternative approach to co-transfecting the reporter plasmid. However, co-transfection of the reporter generally allows measuring promoter activity in the cells that express the viral protein. This is especially important if the assay is used in cells that cannot be transfected to high level. Other luciferases for quantitative readout may be used such as the Gaussia luciferase (G-Luc, from <i>Gaussia princeps</i>) or the Renilla luciferase (R-Luc, from <i>Renilla reniformis</i>). Alternatively, other quantitative reporter systems e.g., based on alkaline phosphatases may also be used instead of luciferases.</p>
<div class="note">
<span class="note-title">Note:</span> In this protocol, we use G-Luc and R-Luc constructs for normalization as they use different substrates for light emission. Thus, their activity can be measured using a commercially available kit separately in the same sample, allowing stringent normalization.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Any cell lines used should be free of mycoplasma and tested on a regular basis (e.g. once a month). Mycoplasma contamination will impair meaningful analysis of IFN signaling assays. Further, cell lines should only be passaged for a limited number of passages. Isolate the plasmids with endotoxin removing kits and store aliquots at −20°C. Before using the plasmids, verify the sequence by sequencing and we recommend measuring the concentration using fluorescence-based assays (e.g. via the Qubit Fluorometer) or alternatively spectrophotometry.</div>
<p>Before you begin, there are several things to consider.</p>
<h3 id="sec1.1">Choice of cell model</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 days</div>
<p>The choice of the right model cell line is crucial as that will define the activity of the pathway you are looking at. Please note that another consideration needs to be how good the cells can be transfected. Higher transfection rates usually equal more significant results. We generally use human embryonic kidney cells 293T (HEK293T) cells as they are readily transfectable and most IFN signaling pathways work. Cell lines with specific defects may be used for specialized application, e.g., to test the impact of cellular proteins of the IFN antagonism by viral protein or to single our specific signaling cascades.</p>
<ol>
<li>Browse the literature to determine whether it is known that the targeted pathway(s) are functional in your cell line of choice. For most commonly used cell lines like HEK293T cells, these analyses have been done and are available. Please note, especially in HEK293T cells some innate immune pathways (e.g., STING signaling or TLR signaling) are non-functional which requires exogenous overexpression of the deficient components.</li>
</ol>
<ol start="2">
<li>If the functionality of the targeted pathway is unknown, or was not tested before, one could assess it via qPCR. This step aims to allow the reader to recapitulate the basic steps for qPCRs, detailed protocols can be found e.g., in (<a href="#bib9">Hayn et al., 2021</a>) or in the MiQe guidelines (<a href="#bib2">Bustin et al., 2009</a>). Before you begin, you should select an IFN stimulated gene (ISG) suitable for the type of IFN you aim to use based on the literature and order the corresponding primer/probe for qPCR.
<ol type="a">
<li>Seed approximately 4×10<sup>4</sup> cells of the cell line of choice in a 48-well plate in 250 μL DMEMxxx, adjust according to cell size. The intended density for treatment is 60%–70%.
<div class="note">
<span class="note-title">Note:</span> DMEMxxx denotes Dulbecco’s Modified Eagle Medium (DMEM) supplemented with 10% (v/v) fetal bovine serum (FBS), 100 U/ml penicillin, 100 μg/ml streptomycin and 2 mM L-glutamine. See also the <a href="#sec2.2">buffers and media</a> section.</div>
</li>
<li>Treat the cells with the desired IFN 18 h post seeding. A good starting concentration is 1000 IU/mL, which generally assures a robust stimulation of the cells.
<div class="note">
<span class="note-title">Note:</span> Higher concentrations may be necessary if there is no stimulation observed in the initial experiment, as signaling components may only be expressed at low levels in some cell lines.</div>
</li>
<li>Lyse the cells in RNA lysis buffer and proceed with RNA purification according to the manufacturers protocol e.g., using the ZymoResearch Quick-RNA Kits or Qiagen RNeasy Kits (<a href="https://files.zymoresearch.com/protocols/_r1054_r1055_quick-rna_miniprep_kit.pdf">https://files.zymoresearch.com/protocols/_r1054_r1055_quick-rna_miniprep_kit.pdf</a> or <a href="https://www.qiagen.com/us/resources/download.aspx?id=14e7cf6e-521a-4cf7-8cbc-bf9f6fa33e24&amp;lang=en">https://www.qiagen.com/us/resources/download.aspx?id=14e7cf6e-521a-4cf7-8cbc-bf9f6fa33e24&amp;lang=en</a>).
<div class="note">
<span class="note-title">Note:</span> For OneStep qPCR a separate cDNA synthesis is not required. Priming and cDNA synthesis is completed in the first step in the Thermocycler using the amplification primers. Normalization on GAPDH will be performed.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Primers should be chosen to be exon-spanning. Thus, DNA digestion is optional, however it may be performed to get rid of DNA contaminants in the RNA isolation. DNA removal is absolutely required if non-exon-spanning primers are used for qPCR.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Digest with DNAse I to remove genomic DNA, according to the manufacturer’s protocol (Digestions steps are included in the kits: ZymoResearch Quick-RNA Kits or Qiagen RNeasy Kits; <a href="https://files.zymoresearch.com/protocols/_r1054_r1055_quick-rna_miniprep_kit.pdf">https://files.zymoresearch.com/protocols/_r1054_r1055_quick-rna_miniprep_kit.pdf</a> or <a href="https://www.qiagen.com/us/resources/download.aspx?id=14e7cf6e-521a-4cf7-8cbc-bf9f6fa33e24&amp;lang=en">https://www.qiagen.com/us/resources/download.aspx?id=14e7cf6e-521a-4cf7-8cbc-bf9f6fa33e24&amp;lang=en</a>)</div>
</li>
<li>Determine the relative amount of RNA using OneStep qPCR (e.g., SuperScript III One-Step Quantitative RT-PCR System, Thermo Fisher) for your target gene normalized to a control mRNA, e.g., GAPDH. cDNA synthesis using the specific primers provided and qPCR amplification are performed in one pipetting step.</li>
<li>For example, upon stimulation with type I IFNs the OAS1 mRNA should be induced as a model interferon stimulated gene (ISG) in HEK293T cells (<a href="#fig3">Figures 3</a>A and 3B). Treatment with type II IFNs should increase CXCL10 mRNA levels (<a href="#fig3">Figure 3</a>C).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1023-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Quantitative PCR and F-Luc reporter gene assay comparing reporter and endogenous gene induction</div>
<p>(A–C) HEK293T cell were transfected with the pISRE-F-Luc (A and B) or pGAS-F-Luc constructs (C) and ISG (OAS1, A,B; CXCL10, (C) mRNA induction was analyzed by qPCR, 18 h post treatment with 500 IU/mL IFN-α2 (A, yellow), 1000 IU/mL IFN-β (B, purple) or 500 IU/mL IFN-γ (C, green). ISG mRNA levels were normalized to GAPDH mRNA levels. All inductions are calculated relatively to the mock (=untreated) control. n=3±SD (biological replicates) for F-Luc reporter gene assay and n=2-3±SD (technical replicates) for qPCR measurements.</p>
</figcaption>
</figure>
</li>
<li>Calculate induction of the target gene upon stimulation using the ΔΔCT method (<a href="#bib2">Bustin et al., 2009</a>; <a href="#bib11">Livak and Schmittgen, 2001</a>). Compare to the reporter gene assay which was performed in parallel.
<div class="critical">
<span class="critical-title">Critical:</span> All inductions were recapitulated in reporter gene assays using either the pISRE-F-Luc construct or the pGAS-F-Luc construct (<a href="#fig3">Figure 3</a>). Please note that many promoters used in reporter assays are artificial and the exact fold induction values may differ. For further information on analysis, interpretation and set up of qPCR experiments please refer to the MIQE guidelines (<a href="#bib2">Bustin et al., 2009</a>).</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Choice and amount of reporter/IFN</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>The reporter should match the IFN used and the resulting readout has to be above background but below saturation levels (<a href="#fig4">Figure 4</a>). Common reporter promoters and suggested amounts for transfection in HEK293T cells for different types of IFNs are listed in <a href="#tbl1">Table 1</a>. All protocols were optimized for the use of F-Luc as a reporter and other Luciferases as internal controls.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1023-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Determining an optimal amount of reporter/stimulant</div>
<p>(A) Quantification of F-Luc activity in HEK293T cells transfected with indicated amounts (0 ng, 50 ng, 100 ng, 200 ng) of pISRE-F-Luc or pGAS-F-Luc plasmid. n=3±SEM (biological replicates).</p>
<p>(B) Quantification of F-Luc activity in HEK293T cells transfected with pGAS-F-Luc (top panel) or pISRE-F-Luc (bottom panel) after treatment with increasing concentrations of IFN-γ (green) or IFN-λ1 (blue). Saturation levels are indicated (dotted line). n=3±SEM (biological replicates).</p>
<p>(C) Data from (B) was normalized on co-expressed R-Luc controlled by a GAPDH promoter. n=3±SEM (biological replicates).</p>
</figcaption>
</figure>
<table id="tbl1">
<caption>Table 1. Examples of IFNs and respective reporter constructs/promoters and suggested amounts</caption>
<thead>
<tr>
<th>Stimulus</th>
<th>Reporter</th>
<th>Suggested amount IFN</th>
<th>Suggested amount reporter</th>
<th>Readout</th>
</tr>
</thead>
<tbody>
<tr>
<td>IFN-α</td>
<td>pISRE</td>
<td>500 U/mL</td>
<td>100 ng</td>
<td>24 h p.st.</td>
</tr>
<tr>
<td>IFN-β</td>
<td>pISRE</td>
<td>1000 U/mL</td>
<td>100 ng</td>
<td>8 h p.st.</td>
</tr>
<tr>
<td>IFN-γ</td>
<td>pGAS</td>
<td>400 U/mL</td>
<td>100 ng</td>
<td>48 h p.st.</td>
</tr>
<tr>
<td>IFN-λ1</td>
<td>pISRE</td>
<td>20 ng/mL</td>
<td>100 ng</td>
<td>24 h p.st.</td>
</tr>
</tbody>
</table>
<p>Transfect HEK293T cells in a 48-well plate with increasing amounts of the reporter plasmid and one well with empty vector plasmid.</p>
<div class="note">
<span class="note-title">Note:</span> Ideally, the background induction caused by transfection of the reporter should be slightly above the background of non-transfected cells, we suggest 1.2–1.5-fold higher than background. For example, in HEK293T cells an ideal amount for transfection of pISRE-F-Luc plasmid is 100 ng at 48-well plate scale (<a href="#tbl1">Table1</a> and <a href="#fig3">Figure 3</a>).</div>
<ol start="3">
<li>Determining the right amount of reporter construct. Please follow the main protocol below.</li>
</ol>
<ol start="4">
<li>Determining the right amount and timing of stimulation (IFN). A detailed protocol for transfection and luciferase measurement is given below. In brief:
<ol type="a">
<li>Seed 4×10<sup>4</sup> HEK293T cells (or cell line of choice) in 48-well plates in 250 μL DMEMxxx.</li>
<li>Use set amount of the reporter plasmid (as determined in step 1) and one well with empty vector plasmid. Add empty plasmid to all transfections to transfect the same amount of total DNA in each well.</li>
<li>Transfect according to the procedure described in detail below.</li>
<li>Stimulate the wells with increasing amounts of stimulant (IFN).</li>
<li>Harvest at different timepoints post stimulation (for suggestions see <a href="#tbl1">Table 1</a>). As the luciferase accumulates, longer stimulation may lead to stronger signals.</li>
<li>Quantify luciferase activity.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Both the amount of reporter plasmid and the amount of stimulant (IFN) need to be titrated to choose the optimal amounts for the final experiment. Ideally, use a concentration for the final assay that is stimulating in the mid-range, i.e. shows a robust fold induction (&gt;2-fold) but has not reached saturation yet. Some reporter plasmids may produce high background levels (<a href="#fig4">Figure 4</a>).</div>
<h3 id="sec1.3">Choice of the internal control</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>To normalize the raw F-Luc values (<a href="#fig5">Figure 5</a>D) when measuring the impact of SARS-CoV-2 proteins on IFN signaling, we used the CellTiter-Glo method (<a href="#fig5">Figure 5</a>A). This allows to adjust the luciferase activity to the cell viability as some proteins, as outlined above, interfere with other means of normalization (<a href="#fig4">Figures 4</a>B and 4C). However, our protocol allows quantification of several different normalization methods, thus e.g., pGAPDH-R-Luc was always recorded in parallel. pGAPDH-R-Luc expresses R-Luc controlled by the housekeeping gene GAPDH promoter and thus can be used to control for transfection efficiency. We recommend using multiple methods of normalization. We do not recommend using a secreted normalization system (e.g., G-Luc, <a href="#fig5">Figure 5</a>C) as that varied a lot between samples. We recommend adding an in parallel transfected fluorescence reporter as a control, to assess transfection efficiency e.g., via microscopy.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1023-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Comparison of normalization methods</div>
<p>HEK293T cells overexpressing indicated viral proteins and the pISRE-F-Luc reporter construct plus one normalization construct as indicated.</p>
<p>(A) Cell viability determined by CellTiter-Glo quantification in the supernatant 48 h post transfection.</p>
<p>(B) Quantification of the in-cell R-Luc activity expressed from co-transfected pGAPDH-R-Luc.</p>
<p>(C) Quantification of G-Luc activity in the supernatant. The pTAL-G-Luc construct was co-expressed.</p>
<p>(D) Quantification of the in-cell activity of ISRE-controlled F-Luc. n=3±SEM (biological triplicates).</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Every reporter gene assay needs to be normalized on transfection efficiency and controlled for cell viability. However, normalizations can cause problems. We recommend normalization to a promoter not affected by IFN treatment. Here, we included the GAPDH promoter. However, it is important to determine whether expression of the viral protein impacts expression of the normalization reporter. For example, some viral proteins (like SARS-CoV-2 ORF3a, <a href="#fig5">Figure 5</a>) interfere with vesicle trafficking or secretion of proteins (<a href="#bib6">Freundt et al., 2010</a>; <a href="#bib9">Hayn et al., 2021</a>; <a href="#bib12">Miao et al., 2021</a>, p. 3; <a href="#bib25">Zhang et al., 2021</a>), thus a normalization based on a secreted luciferase is not applicable. Other proteins like e.g. Nsp1 of SARS-CoV-2 (<a href="#fig4">Figure 4</a>) induce a translation shutdown (<a href="#bib18">Schubert et al., 2020</a>; <a href="#bib23">Thoms et al., 2020</a>) thus effectively reducing every gene translation including GAPDH promoter dependent expression of Renilla luciferase (R-Luc). In these cases, a normalization on GAPDH driven R-Luc is misleading. In general, the signal of the internal control should not vary drastically between the different samples and the control and cell viability/cytotoxicity should always be assessed.</div>
<p>If the viability of the sample (as measured by CellTiter-Glo) is below 75% we would highly recommend discarding the data as it can be misleading. Please see the <a href="#troubleshooting">troubleshooting</a> section below. We recommend to monitor cytotoxicity at 24–48 h post transfection of a vector for single overexpression of the viral protein, in the absence of an IFN stimulus. Do not include viral proteins in the analysis that indiscriminately reduce cell viability drastically within 24 h.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT Or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>IRDye 800CW Goat anti-Rat IgG Secondary Antibody (1:20,000)</td>
<td>LI-COR Biosciences</td>
<td>Cat#926–32219; RRID:AB_1850025</td>
</tr>
<tr>
<td>IRDye 680RD Goat anti-Rabbit IgG Secondary Antibody (1:20,000)</td>
<td>LI-COR Biosciences</td>
<td>Cat#926–68071; RRID:AB_10956166</td>
</tr>
<tr>
<td>Monoclonal rat anti-GAPDH antibody, W17079A (1:1,000)</td>
<td>BioLegend</td>
<td>Cat#607902; RRID:AB_2734503</td>
</tr>
<tr>
<td>Polyclonal rabbit anti-SARS-CoV-2 Nsp3 antibody (1:1,000)</td>
<td>GeneTex</td>
<td>Cat#GTX135614; RRID:AB_2887505</td>
</tr>
<tr>
<td>Polyclonal rabbit anti-strep II-tag antibody, (1:2,000)</td>
<td>Abcam</td>
<td>Cat#ab76949; RRID:AB_1524455</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides and recombinant proteins</td>
</tr>
<tr>
<td>1% Casein in PBS</td>
<td>Thermo Scientific</td>
<td>Cat#37528</td>
</tr>
<tr>
<td>5× Passive lysis buffer</td>
<td>Promega</td>
<td>Cat#E1941</td>
</tr>
<tr>
<td>CellTiter-Glo Luminescent Cell Viability Assay</td>
<td>Promega</td>
<td>Cat#G7570</td>
</tr>
<tr>
<td>Coelenterazine (native-CTZ)</td>
<td>PFK Biotech</td>
<td>Cat#102173</td>
</tr>
<tr>
<td>Dual-Glo luciferase assay system</td>
<td>Promega</td>
<td>Cat#E2920</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle Medium (DMEM)</td>
<td>Gibco</td>
<td>Cat#41965039</td>
</tr>
<tr>
<td>Erythrosin B Stain</td>
<td>BioCat GmbH</td>
<td>Cat#L13002</td>
</tr>
<tr>
<td>fetal bovine serum (FBS)</td>
<td>Gibco</td>
<td>Cat#10270106</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma-Aldrich</td>
<td>Cat#G5516</td>
</tr>
<tr>
<td>IFN-α2</td>
<td>Sigma-Aldrich</td>
<td>Cat#SRP4594</td>
</tr>
<tr>
<td>IFN-β</td>
<td>Merck</td>
<td>Cat#IF014</td>
</tr>
<tr>
<td>IFN-γ</td>
<td>Sigma-Aldrich</td>
<td>Cat#I3265</td>
</tr>
<tr>
<td>IFN-λ1</td>
<td>R&amp;D Systems</td>
<td>Cat#1598-IL</td>
</tr>
<tr>
<td>Immobilon-FL PVDF membrane</td>
<td>Merck Millipore</td>
<td>Cat#IPVH00010</td>
</tr>
<tr>
<td>KCl</td>
<td>Applichem</td>
<td>Cat#A2939</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>Applichem</td>
<td>Cat#A1043</td>
</tr>
<tr>
<td>L-glutamine</td>
<td>PAN-Biotech</td>
<td>Cat#P04-80100</td>
</tr>
<tr>
<td>MES-SDS running buffer (20×)</td>
<td>Alfa Aesar</td>
<td>Cat#J62138</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Merck</td>
<td>Cat#106580</td>
</tr>
<tr>
<td>NaCl</td>
<td>Merck</td>
<td>Cat#106404</td>
</tr>
<tr>
<td>NuPAGE 4–12% Bis-Tris Gels</td>
<td>Invitrogen</td>
<td>Cat#NP0321BOX</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Gibco</td>
<td>Cat#31985047</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>PAN-Biotech</td>
<td>Cat#P06-07050</td>
</tr>
<tr>
<td>Phosphate-Buffered Saline (PBS)</td>
<td>Gibco</td>
<td>Cat#14190094</td>
</tr>
<tr>
<td>Quick-RNA Microprep Kit</td>
<td>Zymo Research</td>
<td>Cat#R1051</td>
</tr>
<tr>
<td>SuperScript III Platinum One-Step qRT-PCR Kit</td>
<td>Thermo Fisher</td>
<td>Cat#11732088</td>
</tr>
<tr>
<td>TransIT-LT1 Transfection Reagent</td>
<td>Mirus</td>
<td>Cat#MIR2300</td>
</tr>
<tr>
<td>Pierce Rapid Gold BCA Protein Assay Kit</td>
<td>Thermo Fisher</td>
<td>cat#A53227</td>
</tr>
<tr>
<td>Trypsin 0.05%/EDTA 0.02%</td>
<td>PAN-Biotech</td>
<td>Cat#P10-023100</td>
</tr>
<tr>
<td>TWEEN 20</td>
<td>Sigma-Aldrich</td>
<td>Cat#P1379</td>
</tr>
<tr>
<td>β-mercaptoethanol</td>
<td>Sigma-Aldrich</td>
<td>Cat#444203</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Human: HEK293T</td>
<td>ATCC</td>
<td>Cat#CRL3216; RRID:CVCL_0063</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>TaqMan Gene Expression Assays CXCL10 Primer probe</td>
<td>Thermo Fisher</td>
<td>Cat#Hs00171042_m1</td>
</tr>
<tr>
<td>TaqMan Gene Expression Assays OAS1 Primer probe</td>
<td>Thermo Fisher</td>
<td>Cat#Hs00973637_m1</td>
</tr>
<tr>
<td>Human GAPD (GAPDH) Endogenous Control (VIC/TAMRA probe)</td>
<td>Thermo Fisher (Applied Biosystems)</td>
<td>Cat#4310884E</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pCR3-Ig-MeV-V</td>
<td>Karl-Klaus Conzelmann</td>
<td>(<a href="#bib13">Pfaller and Conzelmann, 2008</a>; <a href="#bib19">Sparrer et al., 2012</a>)</td>
</tr>
<tr>
<td>pGAPDH_PROM_01_Renilla SP Luciferase</td>
<td>SwitchGear Genomics</td>
<td>Cat#S721624</td>
</tr>
<tr>
<td>pGAS-Luc</td>
<td>Stratagene, Agilent</td>
<td>Cat#219093</td>
</tr>
<tr>
<td>pISRE-Luc</td>
<td>Stratagene, Agilent</td>
<td>Cat#219092</td>
</tr>
<tr>
<td>pLVX-EF1alpha-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-E-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-M-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-N-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp10-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp11-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp12-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp1-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp13-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp14-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp15-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp16-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp2-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp4-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp5-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp6-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp7-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp8-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Nsp9-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf10-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf3a-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf3c-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf6-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf7a-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf7b-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf8-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf9b-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-Orf9c-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pLVX-EF1alpha-nCoV2019-S-2xStrep-IRES-Puro</td>
<td>David Gordon and Nevan Krogan</td>
<td>(<a href="#bib8">Gordon et al., 2020</a>)</td>
</tr>
<tr>
<td>pTAL-Gaussia luciferase</td>
<td>Daniel Sauter</td>
<td>(<a href="#bib17">Sauter et al., 2013</a>)</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Corel DRAW 2017</td>
<td>Corel Corporation</td>
<td><a href="http://www.coreldraw.com">www.coreldraw.com</a></td>
</tr>
<tr>
<td>GraphPad PRISM 9</td>
<td>GraphPad Software, Inc.</td>
<td><a href="http://www.graphpad.com">www.graphpad.com</a></td>
</tr>
<tr>
<td>LI-COR Image Studio Lite Version 5.0</td>
<td>LI-COR Biosciences</td>
<td><a href="http://www.licor.com">www.licor.com</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Equipment</h3>
<p>For analysis the following devices were used: Orion II microplate Luminometer (Berthold) for luminescence measurements (0.1 s measurement per well). Odyssey 9120 infrared imager (LI-COR) to image Western blots. Quantitative PCR was performed using the StepOnePlus Real-Time PCR System (Applied Biosystems). For Western blotting, Thermo Scientific (Running Chambers) and VWR (Blotting) devices were used.</p>
<p>The lysates for bioluminescence measurements were transferred to Nunc F96 MicroWell white plates (#236105 ,Thermo Scientific).</p>
<h3 id="sec2.2">Buffers and media</h3>
<table id="undtbl1">
<caption>6× SDS-PAGE loading buffer</caption>
<thead>
<tr>
<th>Reagent (dilute in water)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris–HCl pH 6.8</td>
<td>187.5 mM</td>
</tr>
<tr>
<td>Glycerol</td>
<td>75 %</td>
</tr>
<tr>
<td>2-mercaptoethanol</td>
<td>15 %</td>
</tr>
<tr>
<td>SDS</td>
<td>6 %</td>
</tr>
<tr>
<td>Orange G</td>
<td>0.3 %</td>
</tr>
</tbody>
</table>
<p class="table-legend">store long-term (for a year) at −20°C, short-term (a week) at 20°C–25°C.</p>
<table id="undtbl2">
<caption>DMEMxxx (use sterile filtered reagents)</caption>
<thead>
<tr>
<th>Reagent (dilute in sterile water)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dulbecco’s modified Eagle medium (DMEM)</td>
<td>1×</td>
</tr>
<tr>
<td>FCS (heat-inactivated)</td>
<td>10%</td>
</tr>
<tr>
<td>L-glutamine</td>
<td>2 mM</td>
</tr>
<tr>
<td>Optional: streptomycin</td>
<td>100 mg/mL</td>
</tr>
<tr>
<td>Optional: penicillin</td>
<td>100 U/mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">store at 4°C (maximum for 2–3 weeks) or 20°C–25°C (maximum a week). We typically discourage use of antibiotics in mammalian cell culture media during assays.</p>
<table id="undtbl3">
<caption>Coelenterazine substrate</caption>
<thead>
<tr>
<th>Reagent (dilute in MeOH)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Coelenterazine (native-CTZ)</td>
<td>1 mg/mL</td>
</tr>
<tr>
<td>HCl</td>
<td>0.01%</td>
</tr>
</tbody>
</table>
<p class="table-legend">Aliquot and store at −80°C for a maximum of 6 months.</p>
<table id="undtbl4">
<caption>PBS-T</caption>
<thead>
<tr>
<th>Reagents (dilute in water)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tween 20</td>
<td>0.2%</td>
</tr>
<tr>
<td>20× PBS</td>
<td>1×</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 20°C–25°C for a maximum of a week, discard if a precipitate is formed.</p>
<table id="undtbl5">
<caption>20× PBS pH 7.4</caption>
<thead>
<tr>
<th>Reagents (dilute in water)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>2.74 M</td>
</tr>
<tr>
<td>KCl</td>
<td>54 mM</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>200 mM</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>36 mM</td>
</tr>
</tbody>
</table>
<p class="table-legend">The pH needs to be adjusted to 7.4. Store at 20°C–25°C for a maximum of a year.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Cell seeding and transfection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>During this step HEK293T cells are seeded in 48-well plates for the subsequent TransIT-LT1 transfection (Mirus Bio) of the reporter and viral plasmids. We recommend to replace the cells after a limited number of passaging steps (10–15) with freshly thawed cryostocks of a low passage.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The cells have to be free of mycoplasma or bacterial contamination, see also <a href="#before-you-begin">before you begin</a> section.</div>
<ol>
<li>Detach HEK293T cells with trypsin and count living cells using dye exclusion (Erythrosin B or Trypan blue Stain). Adjust the concentration and volume of the trypsinized cells in DMEMxxx for the seeding.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Avoid over-trypsinization. Typically, we prepare 10% extra volume of trypsinized cells to ensure that every well receives the correct number of cells and there is a little leftover after seeding.</div>
<ol start="2">
<li>Seed 4×10<sup>4</sup> cells in 48-well plates in 250 μL DMEMxxx per well to achieve 60–70% confluency the next day. Discard the leftover of the seeding solution.</li>
<li>Incubate for 18 h at 37°C and 5% CO<sub>2</sub> in a humid atmosphere.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Plan your seeding well and seed enough wells with cells to ensure that samples can be transfected at least in triplicates.</div>
<div class="critical">
<span class="critical-title">Critical:</span> For an optimal transfection efficiency, cell confluency should be between 60 - 70%. Transfection efficiency suffers if the cells are too confluent and cell viability is dramatically decreased if the cells are too sparsely seeded.</div>
<ol start="4">
<li>Transfect the HEK293T cells 18 h post seeding using TransIT-LT1 (<a href="https://www.mirusbio.com/products/transfection/transit-lt1-transfection-reagent">https://www.mirusbio.com/products/transfection/transit-lt1-transfection-reagent</a>). Alternatively, other Lipofection reagents, e.g., Lipofectamin 3000 may also be used.</li>
<li>Prepare the mix of DNA constructs in 25 μL Opti-MEM for one 48-well according to <a href="#tbl2">Table 2</a>. The transfection mixture may also be prepared in PBS or any other buffer that does not contain FCS, which would impair formation of the transfection complexes.
<table id="tbl2">
<caption>Table 2. Components of the transfection mixture</caption>
<thead>
<tr>
<th>Component</th>
<th>Amount per well</th>
</tr>
</thead>
<tbody>
<tr>
<td>Reporter pISRE-F-Luc <u>or</u> pGAS-F-Luc</td>
<td>100 ng100 ng</td>
</tr>
<tr>
<td>Internal control construct (e.g., pGAPDH-R-Luc)</td>
<td>20 ng</td>
</tr>
<tr>
<td>Viral protein (in pLVX1)</td>
<td>250 ng</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td><b>370 ng</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Fill up each mix to the same amount of total DNA with e.g., empty vector.</p>
</li>
<li>Add TransIT-LT1 (3 μL TransIT-LT1 per μg DNA, for 370 ng DNA add 1.1 μL TransIT-LT1/ 48-well) to the DNA-Opti-MEM mixture and mix by pipetting up and down.</li>
<li>Incubate for 15 min at 20°C–25°C.
<div class="critical">
<span class="critical-title">Critical:</span> Transfection complexes are stable for approximately 30 minutes in the reaction mixture. Finish your transfection before.</div>
</li>
<li>Carefully add the transfection mixture dropwise to the cells. There is no need to wash the cells before addition of the transfection mixture.
<div class="optional">
<span class="optional-title">Optional:</span> To reduce cytotoxic effects, change the medium very carefully to fresh growth medium 4–6 h post transfection.</div>
<div class="note">
<span class="note-title">Note:</span> If this assay is performed with a large number of samples, we recommend using a master mix to ensure that each sample received the same amount of the F-Luc reporter plasmid and the normalization control plasmids.</div>
</li>
</ol>
<h3 id="sec3.2">Stimulation with IFNs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8–48 h</div>
<p>This step describes how interferon signaling is induced in HEK293T cells co-transfected with the respective reporter plasmid, the normalization plasmid and plasmids expressing viral proteins as described above.</p>
<ol start="9">
<li>Stimulate the cells with IFNs</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Amount of IFN and timing depend on the respective pathway that will be induced and should be determined before you begin with the experiment. Our suggestions are given in <a href="#tbl1">Table 1</a>.</div>
<ol start="10">
<li>Replace the growth medium in the wells with 200 μL per well of DMEMxxx containing the respective IFN in the appropriate amount (see <a href="#tbl1">table 1</a> and <a href="#before-you-begin">before you begin</a>).</li>
<li>Replace the growth medium of the negative control (no stimulation) with 200 μL per well of DMEMxxx lacking the stimulant.</li>
<li>Incubate for 8–48 h (suggestion given in <a href="#tbl1">Table 1</a>), depending on the stimulant used.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Monitor the cells for cell death by visual inspection, the CellTiter-Glo assay (performed later) will give a quantitative evaluation of cell viability.</div>
<h3 id="sec3.3">Quantification of luciferase activity</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>This step describes how signaling of type I, II and III interferons was quantified using F-Luc reporters controlled by the respective promoters. In addition, three alternative normalization measurements are described, based on either cell viability (CellTiter-Glo), R-Luc expression (pGAPDH-R-Luc), or G-Luc expression (pTAL-G-Luc). Cell viability measurements are based on ATP metabolization in the cell lysate. Dead cells metabolize less. The pGAPDH-R-Luc construct harbors the R-Luc reporter controlled by the human GAPDH promoter sequence. The plasmid pTAL-G-Luc contains a TATA-like promoter (pTAL) region from the Herpes simplex virus thymidine kinase (HSV-TK) that is not responsive to NF-κB (<a href="#bib17">Sauter et al., 2013</a>).</p>
<div class="critical">
<span class="critical-title">Critical:</span> R-Luc and F-Luc can be measured in the same well. You need at least 50 μL cell lysate (25μL for F-Luc/R-Luc and 25μL for cell viability) if normalization to cell viability is required, since it is not possible to measure F-Luc and cell viability in the same sample well. G-Luc measurements are performed in the supernatants of the samples. Thus, supernatants have to be transferred to fresh white 96-well plates (Nunc F96 MicroWell white plates) prior to lysis of the cells (step 13). Additionally, it is essential to use a luminometer with automated reagent injectors to perform this assay as the Gaussia substrate is processed quickly and therefore all samples should be measured directly after substrate addition.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Cells and supernatant can be stored separately at −20°C to −80°C for a few days before measuring luciferase activity.</div>
<ol start="13">
<li>Cell lysis
<ol type="a">
<li>Remove supernatant and add 100 μL 1× Passive Lysis Buffer per 48-well. Keep the supernatant if G-Luc normalization will be performed, otherwise discard.</li>
<li>Shake gently for 15 min at 20°C–25°C until a fluffy white precipitate is formed.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Freeze and thaw the plate at −20°C to improve cell lysis and have a more uniform signal.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The cell lysates in 1<b>×</b> PLB can be stored at −20°C to −80°C for a few days before measuring luciferase activity.</div>
<ol start="14">
<li>To analyze the reporter gene expression, perform a <b>Dual-Glo Luciferase Assay</b> using the modified manufactures instructions: (<a href="https://www.promega.de/Products/luciferase-assays/Reporter-Assays/Dual_Glo-Luciferase-Assay-System/?catNum=E2920">https://www.promega.de/Products/luciferase-assays/Reporter-Assays/Dual_Glo-Luciferase-Assay-System/?catNum=E2920</a>)</li>
<li>Transfer 25 μL of cell lysate into a type white 96-well plate.</li>
<li>Add one volume (25 μL) of the Dual-Glo Luciferase Reagent. Shake cautiously and incubate for at least 10 min at 20°C–25°C in the dark.</li>
<li>Measure the F-Luc signal with a Luminometer using standard settings, 0.1 s measurement time per well on an Orion II microplate Luminometer (Berthold). The measurement should be performed within 30 min.</li>
<li>Quantify the signal of your internal control (normalization) of choice.
<ol type="a">
<li>For <b>R-Luc</b> normalization:
<ol type="i">
<li>R-Luc measurements are performed in the same sample as the F-Luc assay. Reuse the plate from step 17.</li>
<li>Add one volume (25 μL) of <b>Dual-Glo Stop &amp; Glo Reagent.</b> Shake cautiously and incubate for at least 10 min at 20°C–25°C in the dark.</li>
<li>Measure the R-Luc signal with a Luminometer using standard settings, 0.1 s measurement time per well on an Orion II microplate Luminometer (Berthold). The measurement should be performed within 30 min.</li>
</ol>
</li>
<li>For <b>Cell Viability</b> normalization<b>:</b> Perform <b>CellTiter-Glo Luminescent Cell Viability Assay</b> using modified manufactures instructions: (<a href="https://www.promega.de/products/cell-health-assays/cell-viability-and-cytotoxicity-assays/celltiter_glo-luminescent-cell-viability-assay/?catNum=G7570">https://www.promega.de/products/cell-health-assays/cell-viability-and-cytotoxicity-assays/celltiter_glo-luminescent-cell-viability-assay/?catNum=G7570</a>)
<ol type="i">
<li>Transfer 25 μL of cell lysate into a type white 96-well plate.</li>
<li>Add one volume (25 μL) of CellTiter-Glo Reagent.</li>
<li>Shake cautiously and incubate for at least 10 min at 20°C–25°C in the dark.</li>
<li>Measure luciferase signal using the Orion II microplate Luminometer (Berthold) within 30 min.</li>
</ol>
</li>
<li>For <b>G-Luc</b> normalization:
<ol type="i">
<li>Prepare Coelenterazine solution as described in the <a href="#sec2.2">buffers and media</a> section of the <a href="#materials-and-equipment">materials and equipment</a> chapter and store it as 100 μL aliquots at −80°C.</li>
<li>Prepare Gaussia substrate by thawing one aliquot (100 μL) and mixing it with 20 mL 1× PBS. This amount is sufficient for measuring two full 96-well plates plates.</li>
<li>Prime the luminometer (Orion II microplate Luminometer, Berthold):
<ul>
<li>Insert an external priming container/plate into the luminometer.</li>
<li>Place the reagent injector cable in a tube with ultrapure water.</li>
<li>Prime the injectors with water using standard settings.</li>
<li>Place the reagent injector cable in a tube containing the previously prepared Gaussia substrate.</li>
<li>Prime the injectors with the Gaussia substrate as recommended by the manufacturer of your device.</li>
<li>Remove the external priming container/plate.</li>
</ul>
</li>
<li>Transfer 25 μL of supernatant into a type white 96-well plate and agitate gentle to evenly distribute the sample within the well.</li>
<li>Set volume of injected substrate to 50 μL substrate per sample and record G-Luc activity over a period of 1 s per sample with a delay of 2 s following substrate addition.</li>
<li>Insert the plate and start the measurement.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Confirm expression levels of viral proteins by western blotting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 days</div>
<p>To assess the expression of the viral proteins, immunoblotting analysis can be used. Ideally, expression analysis is done before the reporter gene assay to allow adjustment of transfected DNA amounts of the viral gene expression plasmids to ensure similar expression levels. We additionally recommend to perform expression analysis of the viral proteins after every reporter gene assay. Using the same protein tag on each protein ensures similar detection via a tag-directed antibody. We have used the strep II tag, but any similar setup like (3×)FLAG or (3×)HA tags are feasible. We recommend having the viral proteins expressed from the same vector, ideally codon-optimized to avoid drastic differences in expression levels.</p>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Lysates may be stored at −20 to −80°C for weeks to months.</div>
<ol start="19">
<li>Use the cell lysates from step 13 to confirm proper expression of the viral proteins by western blotting. A more detailed protocol for Western blotting is available elsewhere (<a href="#bib9">Hayn et al., 2021</a>; <a href="#bib10">Koepke et al., 2020</a>).</li>
<li>Transfer 30 μL of the lysates to a tube and pellet the cell debris at 20,000 g, 4°C for 20 min.</li>
<li>Quantify protein amount using a BCA assay (e.g., Pierce Rapid Gold BCA Protein Assay Kit; <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0017135_PierceRapidGoldBCAProteinAssayKit_UG.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0017135_PierceRapidGoldBCAProteinAssayKit_UG.pdf</a>) according to the manufacturer’s instructions and adjust samples to the same protein concentration using 1×Passive Lysis Buffer. Usually we adjust to 1 μg/μL.</li>
<li>Transfer a volume equal to 10μg of total protein to fresh tubes and add 6× SDS-PAGE Loading buffer to reach a final concentration of 1×.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of protein can be decreased or increased depending on the expected expression levels. Typically, 10μg of total protein are a good starting point. Equal amounts of each sample need to be taken.</div>
<ol start="23">
<li>Boil the lysates at 95°C for 10 min to denature the proteins.</li>
<li>Vortex briefly, spin down and load one lysate per pocket in a NuPAGE 4%–12% Bis-Tris Gel. Load a pre-stained protein marker to estimate the size of the viral proteins.</li>
<li>Run the gel at 90 V for 90 min in 1× MES-SDS running buffer.</li>
<li>After the electrophoresis, transfer the proteins to a PVDF membrane using semi-dry western blotting, as detailed in (<a href="#bib10">Koepke et al., 2020</a>).</li>
<li>After blotting, disassemble the setup and place the membrane in 1% casein in PBS to block any free protein binding sites for 60 min at 20°C–25°C while gently agitating.</li>
<li>Incubate the membrane with an anti-strep II-tag antibody (Polyclonal rabbit anti-strep II-tag antibody, 1 to 2,000 diluted in PBS-T with 0.05% Casein) at 4°C for 16–24 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As the anti-strep-II antibody and the anti-GAPDH antibody (used in step 34) were raised in different species, they may be combined in one step for staining. However, we have found that separate staining reduces the background in the Western Blots.</div>
<ol start="29">
<li>Wash the membrane three times for 5 min in PBS-T.</li>
<li>Incubate the membrane with a secondary antibody (IRDye 680RD goat anti-rabbit IgG secondary antibody, 1 to 20,000 diluted in PBS-T with 0.05% Casein) for 30 min at 20°C–25°C.</li>
<li>Wash the membrane three times for 5 min in PBS-T.</li>
<li>Quantify the fluorescence of the secondary antibody using a Li-Cor Odyssey Scanner.</li>
<li>Wash the membrane three times for 5 min in PBS-T.</li>
<li>Incubate the membrane with an anti-GAPDH antibody (Monoclonal rat anti-GAPDH antibody, 1 to 1,000 diluted in PBS-T with 0.05% Casein) for 1.5 h at 20°C–25°C.</li>
<li>Wash the membrane three times for 5 min in PBS-T.</li>
<li>Incubate the membrane with a secondary antibody (IRDye 800CW goat anti-rat IgG secondary antibody, 1 to 20,000 diluted in PBS-T with 0.05% Casein) for 30 min at 20°C–25°C.</li>
<li>Quantify the fluorescence of the secondary antibody using a Li-Cor Odyssey Scanner.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All viral proteins should be expressed to similar levels to allow proper comparison of their effects on the signaling pathways. See below for hints if similar expression levels are not reached.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The expected outcome is a bar plot showing the quantitative impact of SARS-CoV-2 proteins on IFN signaling (<a href="#fig6">Figure 6</a>). The raw data of <a href="#fig4">Figure 4</a>D was normalized on cell viability (CellTiter-Glo, <a href="#fig4">Figure 4</a>A) and statistics were calculated to indicate significant modulators of type I IFN signaling (<a href="#fig6">Figure 6</a>). The data is displayed relative to the vector controls, either with the stimulated vector control set to 100% (<a href="#fig6">Figure 6</a>A) or as fold induction relative to the non-stimulated vector control (<a href="#fig6">Figure 6</a>B). Expression blots are provided in (<a href="#bib9">Hayn et al., 2021</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1023-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Expected outcome of the luciferase assays</div>
<p>Quantification of the in-cell activity of ISRE-controlled F-Luc normalized to cell viability (CellTiter-Glo).</p>
<p>(A and B) (A) Percentage promoter activities compared to the stimulated vector control or (B), fold induction values relative to the non-stimulated vector control were calculated. n=3±SEM (biological triplicates). Samples are compared as indicated or to the stimulated vector control. Student’s t-test with Welch’s correction. ∗,p&lt;0.05, ∗∗,p&lt;0.01, ∗∗∗, p&lt;0.001. Not significant is not indicated.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The data was acquired using an Orion II microplate luminometer (Berthold). To include the reading in an analysis it has to fulfill the following criteria:</p>
<ul>
<li>The fold induction should be significant (mock vs. IFN treatment).</li>
<li>Fold induction of the stimulation should be ideally above 5-fold.</li>
<li>All luciferase readings should be above background (usually at &gt;10<sup>3</sup>).</li>
<li>All readings are not saturated (check your instrument setting).</li>
<li>All controls (here, Measles virus (MeV) V protein was used as a positive control which efficiently antagonizes IFN signaling (<a href="#bib13">Pfaller and Conzelmann, 2008</a>; <a href="#bib19">Sparrer et al., 2012</a>) inhibit the signaling as expected.</li>
</ul>
<p>If the criteria are not met, please refer to the ‘<a href="#before-you-begin">before you begin</a>’ or the ‘<a href="#troubleshooting">troubleshooting</a>’ section of this protocol.</p>
<div class="note">
<span class="note-title">Note:</span> Viral proteins may inhibit the promoter of interest to slightly lower levels than the non-treated mock control by reducing the background signaling of the pathway. If the viral protein induces a host cell shutoff or a general inhibition of transcription/translation, this would be one of the indicators. However, even a general shutoff of cellular gene expression may contribute to immune evasion, albeit not as a specific inhibition of components of the IFN signaling pathways.</div>
<p>Sample data is provided in <a href="#tbl3">Table 3</a>. Use the raw reading to perform the following calculations:</p>
<table id="tbl3" class="dense-table">
<caption>Table 3. Example of data measured on a plate reader</caption>
<thead>
<tr>
<th> </th>
<th colspan="3">CellTiter-glo values</th>
<th colspan="3">F-Luc values (background subtracted)</th>
<th colspan="3">Ratio F-Luc/CellTiter-Glo</th>
</tr>
<tr>
<th>Transfected expression construct:</th>
<th>Replicate 1</th>
<th>Replicate 2</th>
<th>Replicate 3</th>
<th>Replicate 1</th>
<th>Replicate 2</th>
<th>Replicate 3</th>
<th>Replicate 1</th>
<th>Replicate 2</th>
<th>Replicate 3</th>
</tr>
</thead>
<tbody>
<tr>
<td>Vector (not-stimulated)</td>
<td>1362230</td>
<td>1565560</td>
<td>1500490</td>
<td>1450</td>
<td>1510</td>
<td>1560</td>
<td>0.001064</td>
<td>0.000965</td>
<td>0.00104</td>
</tr>
<tr>
<td>Vector</td>
<td>1331190</td>
<td>1428330</td>
<td>1679530</td>
<td>226560</td>
<td>238610</td>
<td>240170</td>
<td>0.170194</td>
<td>0.167055</td>
<td>0.142998</td>
</tr>
<tr>
<td>MeV V</td>
<td>1204400</td>
<td>1302960</td>
<td>1520620</td>
<td>8810</td>
<td>9450</td>
<td>9480</td>
<td>0.007315</td>
<td>0.007253</td>
<td>0.006234</td>
</tr>
<tr>
<td>NSP1</td>
<td>1034860</td>
<td>1010910</td>
<td>1314860</td>
<td>2010</td>
<td>2460</td>
<td>3110</td>
<td>0.001942</td>
<td>0.002433</td>
<td>0.002365</td>
</tr>
<tr>
<td>NSP2</td>
<td>1350470</td>
<td>1279680</td>
<td>1480560</td>
<td>242850</td>
<td>321050</td>
<td>321780</td>
<td>0.179826</td>
<td>0.250883</td>
<td>0.217337</td>
</tr>
<tr>
<td>NSP4</td>
<td>1317270</td>
<td>1425710</td>
<td>1634980</td>
<td>263370</td>
<td>285120</td>
<td>259060</td>
<td>0.199936</td>
<td>0.199985</td>
<td>0.158448</td>
</tr>
<tr>
<td>NSP5</td>
<td>1329170</td>
<td>1224430</td>
<td>1341000</td>
<td>17710</td>
<td>17040</td>
<td>16590</td>
<td>0.013324</td>
<td>0.013917</td>
<td>0.012371</td>
</tr>
<tr>
<td>NSP6</td>
<td>1652360</td>
<td>1422720</td>
<td>1553980</td>
<td>288370</td>
<td>329320</td>
<td>329990</td>
<td>0.17452</td>
<td>0.231472</td>
<td>0.212352</td>
</tr>
<tr>
<td>NSP7</td>
<td>1485790</td>
<td>1281010</td>
<td>1371270</td>
<td>184350</td>
<td>174400</td>
<td>153450</td>
<td>0.124075</td>
<td>0.136143</td>
<td>0.111904</td>
</tr>
<tr>
<td>NSP8</td>
<td>1107500</td>
<td>1294790</td>
<td>1168750</td>
<td>223180</td>
<td>187600</td>
<td>198390</td>
<td>0.201517</td>
<td>0.144888</td>
<td>0.169745</td>
</tr>
<tr>
<td>NSP9</td>
<td>1300610</td>
<td>1216560</td>
<td>1300440</td>
<td>66540</td>
<td>70800</td>
<td>67780</td>
<td>0.051161</td>
<td>0.058197</td>
<td>0.052121</td>
</tr>
<tr>
<td>NSP10</td>
<td>1258980</td>
<td>1225790</td>
<td>1347270</td>
<td>35370</td>
<td>37350</td>
<td>28890</td>
<td>0.028094</td>
<td>0.03047</td>
<td>0.021443</td>
</tr>
<tr>
<td>NSP11</td>
<td>1354470</td>
<td>1081270</td>
<td>1292670</td>
<td>107890</td>
<td>101730</td>
<td>92780</td>
<td>0.079655</td>
<td>0.094084</td>
<td>0.071774</td>
</tr>
<tr>
<td>NSP12</td>
<td>1252870</td>
<td>1377740</td>
<td>1413930</td>
<td>298450</td>
<td>272720</td>
<td>250010</td>
<td>0.238213</td>
<td>0.197947</td>
<td>0.176819</td>
</tr>
<tr>
<td>NSP13</td>
<td>1403890</td>
<td>1353110</td>
<td>1470630</td>
<td>13100</td>
<td>11270</td>
<td>10640</td>
<td>0.009331</td>
<td>0.008329</td>
<td>0.007235</td>
</tr>
<tr>
<td>NSP14</td>
<td>864250</td>
<td>832900</td>
<td>879820</td>
<td>20220</td>
<td>15930</td>
<td>18500</td>
<td>0.023396</td>
<td>0.019126</td>
<td>0.021027</td>
</tr>
<tr>
<td>NSP15</td>
<td>1347210</td>
<td>1293850</td>
<td>1296670</td>
<td>105790</td>
<td>114850</td>
<td>117120</td>
<td>0.078525</td>
<td>0.088766</td>
<td>0.090324</td>
</tr>
<tr>
<td>S</td>
<td>1140010</td>
<td>1223150</td>
<td>1051670</td>
<td>97460</td>
<td>116430</td>
<td>117480</td>
<td>0.08549</td>
<td>0.095189</td>
<td>0.111708</td>
</tr>
<tr>
<td>ORF3a</td>
<td>890390</td>
<td>739520</td>
<td>851310</td>
<td>43040</td>
<td>49220</td>
<td>49530</td>
<td>0.048338</td>
<td>0.066557</td>
<td>0.058181</td>
</tr>
<tr>
<td>ORF3b</td>
<td>1137650</td>
<td>1064850</td>
<td>963730</td>
<td>55500</td>
<td>56900</td>
<td>61940</td>
<td>0.048785</td>
<td>0.053435</td>
<td>0.064271</td>
</tr>
<tr>
<td>E</td>
<td>953870</td>
<td>966710</td>
<td>970660</td>
<td>28070</td>
<td>31450</td>
<td>33820</td>
<td>0.029427</td>
<td>0.032533</td>
<td>0.034842</td>
</tr>
<tr>
<td>M</td>
<td>774430</td>
<td>748550</td>
<td>866140</td>
<td>64730</td>
<td>74490</td>
<td>72170</td>
<td>0.083584</td>
<td>0.099512</td>
<td>0.083324</td>
</tr>
<tr>
<td>ORF6</td>
<td>1064100</td>
<td>993390</td>
<td>1031010</td>
<td>2650</td>
<td>2400</td>
<td>2630</td>
<td>0.00249</td>
<td>0.002416</td>
<td>0.002551</td>
</tr>
<tr>
<td>ORF7a</td>
<td>1496400</td>
<td>1294270</td>
<td>1295250</td>
<td>43990</td>
<td>49550</td>
<td>44270</td>
<td>0.029397</td>
<td>0.038284</td>
<td>0.034179</td>
</tr>
<tr>
<td>ORF7b</td>
<td>1178080</td>
<td>999530</td>
<td>945810</td>
<td>12360</td>
<td>12230</td>
<td>12120</td>
<td>0.010492</td>
<td>0.012236</td>
<td>0.012814</td>
</tr>
<tr>
<td>ORF8</td>
<td>1364570</td>
<td>1250760</td>
<td>1172740</td>
<td>33350</td>
<td>33980</td>
<td>29450</td>
<td>0.02444</td>
<td>0.027167</td>
<td>0.025112</td>
</tr>
<tr>
<td>N</td>
<td>1544600</td>
<td>1432180</td>
<td>1280710</td>
<td>418770</td>
<td>430120</td>
<td>388390</td>
<td>0.271119</td>
<td>0.300325</td>
<td>0.303261</td>
</tr>
<tr>
<td>ORF9b</td>
<td>1454850</td>
<td>1364790</td>
<td>1401950</td>
<td>155300</td>
<td>167400</td>
<td>152140</td>
<td>0.106746</td>
<td>0.122656</td>
<td>0.10852</td>
</tr>
<tr>
<td>ORF9c</td>
<td>1500700</td>
<td>1289060</td>
<td>1153010</td>
<td>71230</td>
<td>68740</td>
<td>67010</td>
<td>0.047465</td>
<td>0.053326</td>
<td>0.058117</td>
</tr>
<tr>
<td>ORF10</td>
<td>1519340</td>
<td>1424720</td>
<td>1292400</td>
<td>86330</td>
<td>84610</td>
<td>76260</td>
<td>0.056821</td>
<td>0.059387</td>
<td>0.059006</td>
</tr>
</tbody>
</table>
<ol>
<li>Subtract the background (empty wells) from the values. Background measurements should be below 1000 RLU/s.</li>
<li>Calculate the ratio of F-Luc to R-Luc, G-Luc or CellTiter-Glo for each individual sample.</li>
<li>Normalize either to the stimulated vector control (100%) or the non-stimulated vector control (1) by dividing the normalized values with the average of the control.</li>
<li>Display the data using GraphPad Prism 9 including error bars (standard deviation or standard error of the mean).</li>
<li>Statistical analyses may be calculated using student’s t-tests with Welch’s correction.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on what treatments are being used, a one-way or two-way ANOVA may be more appropriate with post-hoc Dunnet’s test.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Luciferase Reporter Gene assays have a few general limitations which should be noted. As the reporter constructs and viral protein expression constructs need to be transfected into cells, the transfection rates reached need to be relatively high. However, since the impact of the viral proteins on the cascades is measured only in the transfected cells the background of stimulation from non-transfected cells is relatively low. Thus, transfection and expression rates just need to meet the criteria as outlined in the ‘<a href="#before-you-begin">before you begin</a>’ section. Alternatively, cell lines stably expressing reporters (e.g., ISRE-luciferase) are commercially available and can be used with stimulants/inhibitors that can be added to the supernatant.</p>
<p>The genomic context of the endogenous genes is not reflected by the reporter, as it is delivered as an exogenous plasmid. Furthermore, enhancer elements and upstream/downstream promoter elements regulating the expression of the endogenous gene are usually not present in the reporter plasmid. Generally, 5′UTRs and 3′UTRs are omitted as well. However, that allows reduction of the used promoter sequence to known components that can be dissected with higher specificity or, as in the case of the artificial ISRE promoter, be used to maximize the expression and sensitivity of the construct by repetition of transcription factor binding sites. Activity of the respective IFN stimulated pathway is limited by receptor and signaling component expression on the model cell line used. However, this limitation also applies to endogenous genes. Furthermore, the activity of a viral protein might be occluded if a required cellular factor is not present in sufficient quantities.</p>
<p>Overexpression of a viral protein may not reflect the situation and expression levels during a genuine virus infection. Notably, these reporter assays may also be performed in infected cells to analyze the impact of viral gene expression on IFN signaling. However, the situation is much more complex as the infection may lead to additional stimulation of the reporter on top of the IFN treatment. Many viral genes form complexes with each other, which is not reflected by overexpressing them individually. This limitation may be easily overcome by co-transfecting multiple viral protein expression plasmids.</p>
<p>In summary, reporter assays should be generally used for medium to high-throughput approaches, where fast and convenient readouts are necessary. To determine the impact of an individual SARS-CoV-2 protein on IFN signaling, please follow-up with more detailed mechanistic experiments using endogenous readouts, e.g., STAT phosphorylation or qPCR. A confirmation of the effects observed in reporter gene assays with endogenous readouts is highly warranted.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The normalization is changing the results of the assay or samples are out of range after normalization (<a href="#step-by-step-method-details">step-by-step method details</a> step 18 a-c and <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</p>
<h3 id="sec7.2">Potential solution(s)</h3>
<ol>
<li>Use another promoter for your internal control. For example, if cellular promoters like GAPDH are impacted by viral gene expression, other promoters like the Herpes Simplex-Virus 1 (HSV-1) Thymidin kinase promoter (TK) or the Human cytomegalovirus (CMV) immediate early enhancer and promoter can be used for normalization purposes to assess transfection efficiency.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Some viral promoters like the commonly used CMV promoter include NF-κB binding sites that may enhance gene expression driven by these promoters upon IFN treatment.</div>
<ol start="2">
<li>Use another internal control (e.g., G-Luc).</li>
<li>Normalize to cell viability (e.g., CellTiter-Glo) to control for differences in cell number/viability. If that system conflicts with your reporter, alternative methods such as MTT assays (<a href="#bib24">Wang et al., 2010</a>) may be used.</li>
<li>For some viral proteins any normalization may not be appropriate as fundamental changes in the cell are induced, e.g., translation inhibition or host cell shutoffs. In fact, equal downregulation of reporter and normalization luciferases may indicate that a more general mechanism is employed by the viral protein, that is not specific for the IFN signaling cascade. In this case, we recommend to switch to other types of assays.</li>
<li>The viral protein may target the Luciferase. Here we recommend to examine the endogenous situation to avoid the bias of reporters. The luciferase reporter results have to be interpreted with caution and may even be misleading.</li>
</ol>
<h3 id="sec7.3">Problem 2</h3>
<p>Expression of some viral proteins leads to cell death (<a href="#before-you-begin">Before you begin</a>: Choice of the internal control and <a href="#step-by-step-method-details">step-by-step method details</a> step 18 a-c).</p>
<h3 id="sec7.4">Potential solution(s)</h3>
<ol>
<li>Reduce the amount of transfected viral protein, use a titration to assess which concentration is tolerated by your model cells.</li>
<li>In case the cell death is not excessive a normalization by cell viability (CellTiter-Glo, see above) is advised.</li>
<li>Switch model cell lines, some cell types are less prone to cell death upon overexpression of certain viral proteins.</li>
<li>Use truncations/mutations of the protein, that do not kill the cells but may retain their activity in the IFN pathway.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The function to induce cell death may be linked to the function of the viral protein as an antagonist of the IFN pathway.</div>
<h3 id="sec7.5">Problem 3</h3>
<p>The fold induction of the reporter in the positive control is very low, e.g., in the range between 1 and 2-fold (Before you begin: Choice and amount of reporter/stimulant).</p>
<h3 id="sec7.6">Potential solution(s)</h3>
<ol>
<li>Use a higher amount of the reporter construct. Before you start always titrate the optimal amount of reporter construct (see ‘<a href="#before-you-begin">before you begin</a>’ section).</li>
<li>Increase the amount and/or extend the incubation time of the stimulant. Before you start, always titrate the optimal amount and incubation time for the respective stimulus (see ‘<a href="#before-you-begin">before you begin</a>’ section).</li>
<li>Use a different lot of the cytokine as lot activities may vary.</li>
<li>The signaling cascade has low activity in the cell line of choice (e.g., HEK293T cells). You may want to switch to a different cell model, or alternatively additionally overexpress missing or low expressed signaling components, e.g., recreating the TLR pathway in HEK293T cells by overexpression of MyD88 and IRF7 (<a href="#bib13">Pfaller and Conzelmann, 2008</a>) or overexpressing STING in HEK293T cells to monitor DNA detection by cGas (<a href="#bib3">Chiang et al., 2018</a>; <a href="#bib5">Diner et al., 2013</a>; <a href="#bib22">Sun et al., 2013</a>). However, please note, that this usually leads to background stimulation of the pathway and may mask effects of the viral proteins. Make sure to examine whether the expression levels of the co-expressed signaling component are equal in all samples.</li>
</ol>
<h3 id="sec7.7">Problem 4</h3>
<p>There are drastic differences in the expression of the viral proteins occluding a comparison of the impact on IFN signaling (<a href="#step-by-step-method-details">step-by-step method details</a> step 32).</p>
<h3 id="sec7.8">Potential solution(s)</h3>
<ol>
<li>Use codon-optimized expression vectors for the viral proteins and the same expression vector.</li>
<li>Titer the different proteins and adjust the transfected DNA amounts to adjust expression levels.</li>
</ol>
<h3 id="sec7.9">Problem 5</h3>
<p>None of the viral proteins tested changes promoter induction levels (<a href="#step-by-step-method-details">step-by-step method details</a> step 32, <a href="#before-you-begin">before you begin</a>: Choice of cell model and Choice and amount of reporter/stimulus, <a href="#step-by-step-method-details">step-by-step method details</a> step 17).</p>
<h3 id="sec7.10">Potential solution(s)</h3>
<ol>
<li>Check expression of viral proteins and, if applicable, increase the amount of transfected viral protein expression plasmid.</li>
<li>Revisit the amounts of stimulus used (see ‘<a href="#before-you-begin">before you begin</a>’ section), current concentrations may have reached saturation/overstimulation.</li>
<li>Check whether the luminometer has reached saturation and dilute samples if necessary.</li>
</ol>
</section>
<section>
<h2 id="q-and-a">Q&amp;A</h2>
<p class="questions"><b>Q:</b> Could you please provide the nucleotide sequence of plasmid pISRE-F-Luc, or let me know where I can find it?</p>
<p class="answer"><b>A:</b> The pISRE-F-Luc plasmid was obtain from Stratagene/Agilent (Cat#219092). The company provides the nucleotide sequence of the enhancer elements and a plasmid map, which can be found under the following link: <a href="https://www.agilent.com/cs/library/usermanuals/public/219073.pdf">https://www.agilent.com/cs/library/usermanuals/public/219073.pdf</a>.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Briken, V., Ruffner, H., Schultz, U., Schwarz, A., Reis, L.F., Strehlow, I., Decker, T., and Staeheli, P. (1995). Interferon regulatory factor 1 is required for mouse Gbp gene activation by gamma interferon. Mol. Cell. Biol. <i>15</i>, 975-982. <a class="external-link" href="https://doi.org/10.1128/MCB.15.2.975">https://doi.org/10.1128/MCB.15.2.975</a></p>
<p id="bib2">Bustin, S.A., Benes, V., Garson, J.A., Hellemans, J., Huggett, J., Kubista, M., Mueller, R., Nolan, T., Pfaffl, M.W., Shipley, G.L., et al. (2009). The MIQE guidelines: minimum information for publication of quantitative real-time PCR experiments. Clin. Chem. <i>55</i>, 611-622. <a class="external-link" href="https://doi.org/10.1373/clinchem.2008.112797">https://doi.org/10.1373/clinchem.2008.112797</a></p>
<p id="bib3">Chiang, J.J., Sparrer, K.M.J., van Gent, M., Lässig, C., Huang, T., Osterrieder, N., Hopfner, K.-P., and Gack, M.U. (2018). Viral unmasking of cellular 5S rRNA pseudogene transcripts induces RIG-I-mediated immunity. Nat. Immunol. <i>19</i>, 53-62. <a class="external-link" href="https://doi.org/10.1038/s41590-017-0005-y">https://doi.org/10.1038/s41590-017-0005-y</a></p>
<p id="bib4">de Wet, J.R., Wood, K.V., DeLuca, M., Helinski, D.R., and Subramani, S. (1987). Firefly luciferase gene: structure and expression in mammalian cells. Mol. Cell. Biol. <i>7</i>, 725-737. <a class="external-link" href="https://doi.org/10.1128/mcb.7.2.725-737.1987">https://doi.org/10.1128/mcb.7.2.725-737.1987</a></p>
<p id="bib5">Diner, E.J., Burdette, D.L., Wilson, S.C., Monroe, K.M., Kellenberger, C.A., Hyodo, M., Hayakawa, Y., Hammond, M.C., and Vance, R.E. (2013). The innate immune DNA sensor cGAS produces a noncanonical cyclic dinucleotide that activates human STING. Cell Rep. <i>3</i>, 1355-1361. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2013.05.009">https://doi.org/10.1016/j.celrep.2013.05.009</a></p>
<p id="bib6">Freundt, E.C., Yu, L., Goldsmith, C.S., Welsh, S., Cheng, A., Yount, B., Liu, W., Frieman, M.B., Buchholz, U.J., Screaton, G.R., et al. (2010). The open reading frame 3a protein of severe acute respiratory syndrome-associated coronavirus promotes membrane rearrangement and cell death. J. Virol. <i>84</i>, 1097-1109. <a class="external-link" href="https://doi.org/10.1128/JVI.01662-09">https://doi.org/10.1128/JVI.01662-09</a></p>
<p id="bib7">Fujita, T., Nolan, G.P., Liou, H.-C., Scott, M.L., and Baltimore, D. (1993). The candidate proto-oncogene bcl-3 encodes a transcriptional coactivator that activates through NF-KB p50 homodimers. Genes Dev <i>7</i>, 1354-1363. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00487-1/sref7">View at publisher</a></p>
<p id="bib8">Gordon, D.E., Jang, G.M., Bouhaddou, M., Xu, J., Obernier, K., White, K.M., O’Meara, M.J., Rezelj, V.V., Guo, J.Z., Swaney, D.L., et al. (2020). A SARS-CoV-2 protein interaction map reveals targets for drug repurposing. Nature <i>583</i>, 459-468. <a class="external-link" href="https://doi.org/10.1038/s41586-020-2286-9">https://doi.org/10.1038/s41586-020-2286-9</a></p>
<p id="bib9">Hayn, M., Hirschenberger, M., Koepke, L., Nchioua, R., Straub, J.H., Klute, S., Hunszinger, V., Zech, F., Prelli Bozzo, C., Aftab, W., et al. (2021). Systematic functional analysis of SARS-CoV-2 proteins uncovers viral innate immune antagonists and remaining vulnerabilities. Cell Rep. <i>35</i>, 109126. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2021.109126">https://doi.org/10.1016/j.celrep.2021.109126</a></p>
<p id="bib10">Koepke, L., Winter, B., Grenzner, A., Regensburger, K., Engelhart, S., van der Merwe, J.A., Krebs, S., Blum, H., Kirchhoff, F., and Sparrer, K.M.J. (2020). An improved method for high-throughput quantification of autophagy in mammalian cells. Sci. Rep. <i>10</i>, 12241. <a class="external-link" href="https://doi.org/10.1038/s41598-020-68607-w">https://doi.org/10.1038/s41598-020-68607-w</a></p>
<p id="bib11">Livak, K.J. and Schmittgen, T.D. (2001). Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods <i>25</i>, 402-408. <a class="external-link" href="https://doi.org/10.1006/meth.2001.1262">https://doi.org/10.1006/meth.2001.1262</a></p>
<p id="bib12">Miao, G., Zhao, H., Li, Y., Ji, M., Chen, Y., Shi, Y., Bi, Y., Wang, P., and Zhang, H. (2021). ORF3a of the COVID-19 virus SARS-CoV-2 blocks HOPS complex-mediated assembly of the SNARE complex required for autolysosome formation. Dev. Cell <i>56</i>, 427-442.e5. <a class="external-link" href="https://doi.org/10.1016/j.devcel.2020.12.010">https://doi.org/10.1016/j.devcel.2020.12.010</a></p>
<p id="bib13">Pfaller, C.K. and Conzelmann, K.-K. (2008). Measles virus V protein is a decoy substrate for IkappaB kinase alpha and prevents Toll-like receptor 7/9-mediated interferon induction. J. Virol. <i>82</i>, 12365-12373. <a class="external-link" href="https://doi.org/10.1128/JVI.01321-08">https://doi.org/10.1128/JVI.01321-08</a></p>
<p id="bib14">Platanias, L.C. (2005). Mechanisms of type-I- and type-II-interferon-mediated signalling. Nat. Rev. Immunol. <i>5</i>, 375-386. <a class="external-link" href="https://doi.org/10.1038/nri1604">https://doi.org/10.1038/nri1604</a></p>
<p id="bib15">Randall, R.E. and Goodbourn, S. (2008). Interferons and viruses: an interplay between induction, signalling, antiviral responses and virus countermeasures. J. Gen. Virol. <i>89</i>, 1-47. <a class="external-link" href="https://doi.org/10.1099/vir.0.83391-0">https://doi.org/10.1099/vir.0.83391-0</a></p>
<p id="bib16">Sanchez, T., Xu, L., Buchanan, M., and Zheng, C.-F. (1998). Optimizing transfection conditions for studying signal transduction pathways. Strategies. 52-53. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00487-1/sref16">View at publisher</a></p>
<p id="bib17">Sauter, D., Hotter, D., Engelhart, S., Giehler, F., Kieser, A., Kubisch, C., and Kirchhoff, F. (2013). A rare missense variant abrogates the signaling activity of tetherin/BST-2 without affecting its effect on virus release. Retrovirology <i>10</i>, 85. <a class="external-link" href="https://doi.org/10.1186/1742-4690-10-85">https://doi.org/10.1186/1742-4690-10-85</a></p>
<p id="bib18">Schubert, K., Karousis, E.D., Jomaa, A., Scaiola, A., Echeverria, B., Gurzeler, L.-A., Leibundgut, M., Thiel, V., Mühlemann, O., and Ban, N. (2020). SARS-CoV-2 Nsp1 binds the ribosomal mRNA channel to inhibit translation. Nat. Struct. Mol. Biol. <i>27</i>, 959-966. <a class="external-link" href="https://doi.org/10.1038/s41594-020-0511-8">https://doi.org/10.1038/s41594-020-0511-8</a></p>
<p id="bib19">Sparrer, K.M.J., Pfaller, C.K., and Conzelmann, K.-K. (2012). Measles virus C protein interferes with Beta interferon transcription in the nucleus. J. Virol. <i>86</i>, 796-805. <a class="external-link" href="https://doi.org/10.1128/JVI.05899-11">https://doi.org/10.1128/JVI.05899-11</a></p>
<p id="bib20">Stark, G.R., Kerr, I.M., Williams, B.R., Silverman, R.H., and Schreiber, R.D. (1998). How cells respond to interferons. Annu. Rev. Biochem. <i>67</i>, 227-264. <a class="external-link" href="https://doi.org/10.1146/annurev.biochem.67.1.227">https://doi.org/10.1146/annurev.biochem.67.1.227</a></p>
<p id="bib21">Stetson, D.B. and Medzhitov, R. (2006). Type I Interferons in Host Defense. Immunity <i>25</i>, 373-381. <a class="external-link" href="https://doi.org/10.1016/j.immuni.2006.08.007">https://doi.org/10.1016/j.immuni.2006.08.007</a></p>
<p id="bib22">Sun, L., Wu, J., Du, F., Chen, X., and Chen, Z.J. (2013). Cyclic GMP-AMP Synthase is a Cytosolic DNA Sensor that Activates the Type-I Interferon Pathway. Science <i>339</i>. <a class="external-link" href="https://doi.org/10.1126/science.1232458">https://doi.org/10.1126/science.1232458</a></p>
<p id="bib23">Thoms, M., Buschauer, R., Ameismeier, M., Koepke, L., Denk, T., Hirschenberger, M., Kratzat, H., Hayn, M., Mackens-Kiani, T., Cheng, J., et al. (2020). Structural basis for translational shutdown and immune evasion by the Nsp1 protein of SARS-CoV-2. Science <i>369</i>, 1249-1255. <a class="external-link" href="https://doi.org/10.1126/science.abc8665">https://doi.org/10.1126/science.abc8665</a></p>
<p id="bib24">Wang, P., Henning, S.M., and Heber, D. (2010). Limitations of MTT and MTS-based assays for measurement of antiproliferative activity of green tea polyphenols. PLoS One <i>5</i>, e10202. <a class="external-link" href="https://doi.org/10.1371/journal.pone.0010202">https://doi.org/10.1371/journal.pone.0010202</a></p>
<p id="bib25">Zhang, Y., Sun, H., Pei, R., Mao, B., Zhao, Z., Li, H., Lin, Y., and Lu, K. (2021). The SARS-CoV-2 protein ORF3a inhibits fusion of autophagosomes with lysosomes. Cell Discov. <i>7</i>, 1-12. <a class="external-link" href="https://doi.org/10.1038/s41421-021-00268-z">https://doi.org/10.1038/s41421-021-00268-z</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Konstantin Sparrer (<a href="mailto:konstantin.sparrer@uni-ulm.de">konstantin.sparrer@uni-ulm.de</a>).</p>
<h4>Materials availability</h4>
<p>All unique reagents generated in this study are listed in the <a href="#key-resources-table">key resources table</a> and available upon reasonable request from the lead contact or the technical contact.</p>
<h4>Data and code availability</h4>
<p>The published article includes all data sets generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>We thank Jana-Romana Fischer and Birgit Ott for excellent technical assistance. We would like to acknowledge the library of SARS-CoV-2 expression plasmids which was generously provided by Nevan Krogan (University of California, San Francisco). This study was supported by <a href="https://doi.org/10.13039/501100001659">DFG</a> grants to F.K. (CRC1279, SPP1923), K.M.J.S. (CRC1279, SPP1923, SP1600/4-1, SP1600/6-1), a COVID-19 research grant of the <a href="https://doi.org/10.13039/501100003542">Ministry of Science, Research and the Arts</a> (MWK) Baden-Württemberg (to F.K.), as well as the <a href="https://doi.org/10.13039/501100002347">Federal Ministry of Education and Research (BMBF)</a> to F.K. (Restrict SARS-CoV-2) and K.M.J.S. (IMMUNOMOD). M.Hi., M.Ha., and L.K. are part of the international graduate school of molecular medicine, Ulm (IGradU).</p>
<h3>Author contributions</h3>
<p>M.Hi., M.Ha., L.K., A.L. performed the experimental work, arranged the figures, and provided comments to the manuscript. M.Hi., M.Ha., F.K. and K.M.J.S. conceived the study, planned experiments, and wrote the manuscript. All authors reviewed and approved the manuscript.</p>
<h3>Declaration of interests</h3>
<p>No competing interest to declare.</p>
</section>
</article>