<article>
<section>
<h1 id="header">A proximity labeling protocol to probe proximity interactions in <i>C. elegans</i>
</h1>
<p><time datetime="2021-12-03">Published: December 3, 2021</time></p>
<p>Ariana D. Sanchez<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Jessica L. Feldman<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biology, Stanford University, Stanford, CA 94305, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:feldmanj@stanford.edu">feldmanj@stanford.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100986">10.1016/j.xpro.2021.100986</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Enzyme-catalyzed proximity labeling (PL) has emerged as a critical approach for identifying protein-protein proximity interactions in cells; however, PL techniques were not historically practical in living multicellular organisms due to technical limitations. Here, we present a protocol for applying PL to living <i>C. elegans</i> using the biotin ligase mutant enzyme TurboID. We demonstrated PL in a tissue-specific and region-specific manner by focusing on non-centrosomal MTOCs (ncMTOCs) of intestinal cells. This protocol is useful for targeted <i>in vivo</i> protein network profiling.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref9">Sanchez et al. (2021)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>TurboID enabled biotin-based proximity labeling protocol for <i>C. elegans</i>
</li>
<li>Experimental design guidelines for proximity labeling in <i>C. elegans</i>
</li>
<li>A step-by-step TurboID protocol from transgene design to protein identification</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1205-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Experimental design considerations</h3>
<p>This protocol describes the steps for conducting biotin-ligase based proximity labeling in living <i>C. elegans</i> using TurboID in intestinal cells (<a href="#bib9">Sanchez et al., 2021</a>). We have also previously shown examples of proximity labeling using miniTurbo (<a href="#bib1">Branon et al., 2018</a>).</p>
<p>Before beginning, consider which experimental and control samples best fit your biological question to guide plasmid and strain construction. Due to our interests in defining components of non-centrosomal microtubule organizing centers (ncMTOCs), our experimental <i>C. elegans</i> sample expressed the microtubule minus end binding protein PTRN-1 tagged with HA::TurboID specifically in intestinal cells. PTRN-1 localizes to the apical ncMTOC and not to any other part of intestinal cells, making it an ideal bait protein. Experiments should have at least two control samples that will aid in the identification of non-specific biotinylated proteins and contaminants in the mass spectrometry datasets: 1) A control for non-specific biotinylation. In our case, we controlled for cytoplasmic biotinylation by expressing cytosolic HA::TurboID in intestinal cells; 2) A control for endogenously biotinylated proteins (e.g., carboxylases) and background contaminants. Ideally, this strain would be close to your experimental strain, but lacking TurboID expression. In our case, this control was N2.</p>
<p>When designing our <i>C. elegans</i> codon-optimized TurboID constructs, we considered the following parameters:</p>
<p><i><u>Epitope tag:</u></i> We designed fusion constructs with a small HA epitope tag to minimize disruption to TurboID activity. Lysine-rich epitope tags like FLAG should be avoided (<a href="#bib2">Cho et al., 2020</a>).</p>
<p><i><u>Promoter:</u></i> Consider the desired tissue-specificity, timing, and level of expression in choosing a promoter. As ncMTOC formation is well characterized in the intestine (<a href="#bib4">Feldman and Priess 2012</a>), we chose the intestine-specific promoter <i>ges-1</i>, which was reported to have high expression that begins in the embryo and continues into adulthood (<a href="#bib7">Pauli et al., 2006</a>).</p>
<p><i><u>Tag Location:</u></i> Tag the bait protein in a manner that does not affect the endogenous localization or function of your protein of interest. In our studies, we tagged the C-terminus of PTRN-1 with HA::TurboID because the localization pattern was similar to endogenous localization and previous reports showed that C-terminally-tagged PTRN-1 rescues phenotypes of null <i>ptrn-1</i> mutants (<a href="#bib8">Richardson et al., 2014</a>; <a href="#bib10">Wang et al., 2015</a>).</p>
<p><i><u>Expression Level:</u></i> In PL experiments, protein expression levels are important. In <i>C. elegans,</i> there are many types of tagged expression approaches which vary in level of expression, including high copy arrays with subsequent integration, single copy insertions, or insertion of a tag into the endogenous locus using CRISPR/Cas9. Regardless of the approach, generating integrated <i>C. elegans</i> transgenic lines is ideal for consistency and maximizing efficiency of large-scale experiments. We see significant differences in biotinylation efficiency in integrated high copy arrays versus single copy insertions: High-copy array-based expression provided proper protein localization, albeit with minor cytoplasmic signal, and significant biotinylation levels detectable by IF staining and western blot analysis; Single copy insertions provided facile transgene integration, but no detectable biotinylation signal by immunofluorescence (IF) staining nor by western blot analysis.</p>
<p>Finally, consider the approach you will take to identify biotinylated proteins via mass spectrometry that best fits your experimental goals. We do not outline these specific steps as it is critical to determine the best approaches for identification of biotinylated proteins with your mass spectrometry facility or collaborator.</p>
<h3 id="sec1.2">Design and build <i>C. elegans</i> strains</h3>
<div class="timing">
<span class="timing-title">Timing:</span> minimum 12 days</div>
<ol>
<li>Design and build TurboID fusion constructs</li>
</ol>
<p>Design primers to build plasmids with the desired epitope tag and codon-optimized TurboID. The <i>C. elegans</i> codon-optimized TurboID is available at Addgene (plasmid # 118220). Constructs are listed in the key resources table.</p>
<div class="note">
<span class="note-title">Note:</span> Consider using miniTurbo instead of TurboID if the experimental design requires a smaller enzyme with a different activity profile.</div>
<div class="note">
<span class="note-title">Note:</span> Our TurboID strains were maintained at 20°C on NGM (nematode growth medium) plates with OP50 bacteria.</div>
<div class="note">
<span class="note-title">Note:</span> Sequence the entire TurboID construct before proceeding to ensure that mutations were not introduced during the cloning process. We have noticed reversion of some of the original TurboID mutations after culturing in bacteria, especially at nucleotide 632 (C-&gt;G) of the <i>C. elegans</i> codon-optimized TurboID sequence which results in a P149R substitution.</div>
<ol start="2">
<li>Microinjection of plasmids into <i>C. elegans</i>
</li>
</ol>
<p>Microinject DNA into the cytoplasm of the syncytial gonad to establish a transgenic line of high-copy extrachromosomal arrays (<a href="#bib3">Evans 2006</a>). We injected into an N2 wild-type <i>C. elegans</i> strain with 50 ng/μL of the TurboID plasmid and 2.5 ng/μL of a fluorescent co-injection marker <i>myo-2p::mCherry</i>. Refer to troubleshooting <a href="#sec6.1">problem 1</a> for additional information.</p>
<div class="note">
<span class="note-title">Note:</span> We do not recommend selecting rol-6 as the co-injection marker because Rol worms do not access bacteria as efficiently as wild-type moving worms.</div>
<h3 id="sec1.3">Diagnostic check of TurboID localization and biotinylation activity</h3>
<div class="timing">
<span class="timing-title">Timing:</span> minimum 10 days</div>
<ol start="3">
<li>Grow strains on plates with either biotin-rich bacteria (OP50 or NA22) or on plates with biotin-depleted bacteria (MG1655bioB::kan) (<a href="#bib6">Ortega-Cuellar et al., 2010</a>) for two generations. By growing strains in either condition and comparing the resulting biotinylation levels, you can assess the level of TurboID-induced biotinylation that is obtained from feeding worms biotin-rich bacteria. Additionally, as we have previously shown that feeding worms the biotin-depleted bacteria reduces the biotinylation levels produced by TurboID (<a href="#bib1">Branon et al., 2018</a>), the biotin-depleted bacteria can allow for acute exposure to biotin if necessary.
<ol type="a">
<li>Prepare plates with biotin-rich <i>E. coli</i> (OP50 or NA22)
<ol type="i">
<li>Pick one colony of bacteria from an LB plate into 100 mL of LB broth and grow for 16 h at 37°C without agitation.</li>
<li>Seed OP50 on NGM plates or NA22 on peptone-rich NGM plates.</li>
<li>Incubate the plates at 20°C–23°C for 2 days to dry.</li>
<li>If necessary, store at 4°C.</li>
</ol>
</li>
<li>Prepare plates with biotin auxotrophic <i>E. coli</i> (MG1655bioB::kan)
<ol type="i">
<li>Pick one colony of bacteria from an LB plate with kanamycin into 100 mL of LB broth with kanamycin (100 μg/mL) and grow for 16 h at 37°C.</li>
<li>Centrifuge 35 mL of MG1655bioB::kan at 3,000 RPM for 10 min.</li>
<li>Wash the bacteria in 20 mL of M9 buffer twice.</li>
<li>Fill tube up to 17.5 mL with M9 and seed mixture onto NGM plates.</li>
<li>Incubate plates at 20°C–23°C for 2 days to dry.</li>
<li>If necessary, store at 4°C.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To prepare NGM agar, combine 3 g NaCl, 17 g agar, and 2.5 g peptone and 975 mL water. Autoclave the solution, cool down to 55°C, then add 1 mL of CaCl<sub>2</sub> (1 M stock), 1 mL of cholesterol (5 mg/mL stock in 95% ethanol), 1 mL uracil, and 25 mL potassium phosphate buffer (1 M stock).</div>
<div class="note">
<span class="note-title">Note:</span> To prepare peptone-rich NGM plates, combine 1.2 g NaCl, 25 g agar, and 20 g peptone and 975 mL water. Autoclave the solution, cool down to 55°C, then add 25 mL potassium phosphate buffer (1 M stock), 1 mL MgSO<sub>4</sub> (1 M stock), and 1 mL of cholesterol (5 mg/mL stock in 95% ethanol).</div>
<div class="note">
<span class="note-title">Note:</span> Washing away the LB broth from the MG1655 <i>bioB</i>::kan bacteria before plating on NGM plates is critical for removing excess biotin.</div>
<div class="note">
<span class="note-title">Note:</span> Do not agitate bacteria when growing for 16 hours.</div>
<ol start="4">
<li>Probe the localization of TurboID and its biotinylation activity by immunofluorescence (IF) staining and imaging (<a href="#fig1">Figure 1</a>A). Refer to troubleshooting <a href="#sec6.3">problems 2</a> and <a href="#sec6.5">3</a> for additional information.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1205-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. TurboID provides tissue- and region-specific promiscuous biotinylation in <i>C. elegans</i>
</div>
<p>(A) Immunofluorescence of fixed <i>C. elegans</i> comma stage embryos marking TurboID (anti-HA, red) and biotinylated proteins (streptavidin-488, green) for the indicated genotype. Worms were fed either biotin-depleted bacteria (biotin (-), MG1655 <i>bioB</i>::kan) or biotin-rich bacteria (biotin (+), OP50). The white dotted line marks the intestine of the embryo and arrowheads mark the apical ncMTOC. Note that the localization of the fusion protein (anti-HA) and the biotinylation pattern match closely. Scale bar= 10 μm.</p>
<p>(B) Immunoblotting for biotinylated proteins (streptavidin-488, green) in L4/adult worm lysates (input) of wild-type (N2), PTRN-1::HA::TurboID ("P:T"), or HA::TurboID ("T"). Whole worm lysates were subjected to affinity purification with streptavidin-covered magnetic beads ("biotin A.P.") to enrich for biotinylated proteins. TurboID proteins (arrowheads) and endogenously biotinylated proteins (asterisks) are indicated. Note that the range and amount of biointylated proteins is higher in the "P:T" and "T" lanes compared to N2. Additionally, note the absence of biotinylated proteins in the flow-through lanes indicating a high efficiency of capture of biotinylated proteins by the streptavidin-covered magnetic beads.</p>
</figcaption>
</figure>
<ol type="a">
<li>Incubate 25–30 gravid adults in M9 buffer to obtain appropriate stage embryos</li>
<li>Cut open adult worms on a glass 22 × 40 mm coverslip to release embryos and then invert the coverslip perpendicularly over a poly-lysine coated glass microscope slide stamped with 3 Teflon spacers (Thermo Fisher Scientific).</li>
<li>Freeze slides on dry ice and permeabilize embryos by swiftly removing the coverslip and immediately dipping the slide into a Coplin jar with 100% MeOH at –20°C for 5–10 min.</li>
<li>Wash the slides twice in 1× PBS, then once in 1× PBST (PBS plus 0.1% Tween) for 5 min each at 20°C–23°C.</li>
<li>To incubate the slides in anti-HA primary antibody (Abcam, 1:200 diluted in 1× PBST), add 10–15 μL of the antibody to a 22 × 40 mm coverslip and gently invert coverslip onto the slide.</li>
<li>Incubate for 16 h at 4°C in a humid chamber to stain for the epitope tag.</li>
<li>Remove the coverslip by gently dipping the slide into a small container of 1× PBS solution until the coverslip floats off.</li>
<li>Wash the slides in 1× PBST three times for 5 min each at 20°C–23°C.</li>
<li>Add 10–15 μL of a secondary antibody mixture to a 22 × 40 mm coverslip and gently invert the coverslip onto the slide. The secondary antibody mixture consists of CY3-anti-mouse secondary antibody (Jackson Immunoresearch Laboratories, 1:200), Streptavidin Alexa Fluor 488 (Invitrogen, 1:200) to stain for biotinylated proteins, and DAPI (Sigma, 1:10,000) to stain for nuclei, each diluted in 1× PBST.</li>
<li>Incubate the slides for one hour at 37°C in a humid chamber. Note that from now on the samples are light-sensitive, so incubations should proceed with minimal exposure to light.</li>
<li>Repeat step <b>g.</b> and then wash the slides once in 1× PBST and twice in 1× PBS for 5 min each at 20°C–23°C.</li>
<li>Remove excess PBS, mount each sample in 10 μL of Vectashield (Vector Laboratories), gently invert a 22 × 22 mm coverslip onto the sample (be careful not to introduce air bubbles), and then seal the coverslip onto the slide with nail polish to immobilize the coverslip and protect the sample from desiccation. The samples can be stored at 4°C for several months.</li>
<li>Image the samples with fluorescence microscopy. We used a Nikon Ti-E inverted microscope (Nikon Instruments) with a Yokagawa X1 spinning disk and an Andor Ixon Ultra back thinned EM-CCD camera controlled by NIS Elements at a z-sampling rate of 0.5 μm.</li>
<li>Determine the localization pattern of the TurboID constructs and the biotinylation signals compared to controls and between samples fed with biotin-rich or biotin-depleted bacteria. The localization of the fusion protein (HA) and the biotinylation pattern should closely match (<a href="#fig1">Figure 1</a>A).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not agitate samples as agitation will liberate embryos from the glass slides.</div>
<div class="note">
<span class="note-title">Note:</span> Use the appropriate staining technique for the <i>C. elegans</i> stage of development. This approach is specific to embryos.</div>
<ol start="5">
<li>Probe the expression level of the TurboID constructs and the resulting biotinylation patterns by western blot analysis. Refer to troubleshooting <a href="#sec6.7">problem 4</a> for additional information.
<ol type="a">
<li>Pick 50 adult worms and transfer to Eppendorf tubes containing 1 mL of M9 buffer. If the TurboID construct is expressed from an extrachromosomal array, only pick array-positive worms.</li>
<li>Wash the worms with 1 mL of M9 buffer until the supernatant is clear of bacteria. This typically takes about two washes.</li>
<li>Remove excess M9 buffer until approximately 50 μL of M9 remains.</li>
<li>Add 50 μL of 4× sample buffer (supplemented with 1:10 β-mercaptoethanol).</li>
<li>Boil the samples at 95°C for 10 min.</li>
<li>Vortex the samples for 10–20 s and then centrifuge at 13,000×<i>g</i> for 5 min at 4°C.</li>
<li>Load equal volume of lysate onto a 4–20% Mini-PROTEAN TGX PAGE gel (Bio-Rad) or another SDS polyacrylamide gel. If using HRP detection methods, run two separate gels, one to detect biotinylated proteins and one to detect the TurboID construct. Running a single gel will suffice if you are using fluorescent detection methods as we did in our protocol.</li>
<li>Transfer the samples to a nitrocellulose membrane (0.4 μm, Bio-Rad).</li>
<li>Stain the membrane with Ponceau S solution to check the quality of the protein transfer. Then, remove the Ponceau S stain by rinsing the membrane with water.</li>
<li>Block the membrane with 5% milk in 1× PBST for 30 min to 1 h at 20°C–23°C or for 16 h at 4°C.</li>
<li>Incubate the membrane with anti-HA primary antibody (1:5000 in 1× PBST, rat monoclonal, Roche) for 1 h at 20°C–23°C.</li>
<li>Wash the membrane three times with 1× PBST for 5 min each.</li>
<li>To detect proteins on a fluorescence imaging system (e.g., LI-COR), incubate the membrane with a secondary antibody (1:5000, goat anti-rat IRDye 680RD, Licor) and streptavidin-IR Dye (1:5000, 800CW, Licor) in 1× PBST for 1 h at 20°C–23°C.</li>
<li>Wash the membrane three times with 1× PBST for 5 min each.</li>
<li>Image the blot on LI-COR Odyssey CLx or an equivalent technology.</li>
</ol>
</li>
</ol>
<h3 id="sec1.4">Production of stably transmitting transgenic lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 weeks</div>
<p>Extrachromosomal arrays that behave well in the immunofluorescence and western blotting assays should be integrated in order to achieve 100% transmission, which will facilitate subsequent larger-scale biochemical experiments. Several methods have been demonstrated to be effective to integrate extrachromosomal arrays and we used a gamma irradiation-based approach.</p>
<ol start="6">
<li>Evaluate the transmission rate and developmental time of each transgenic line.
<ol type="a">
<li>For each transgenic line, bleach 10 fluorescent gravid adults onto 10 separate OP50-seeded NGM plates to synchronize developmental stage.</li>
<li>Observe progeny each day to evaluate developmental timing and transmission rate. Note the time at which the strain takes to reach the L4 stage in order to prepare for the gamma irradiation treatment. Choose the transgenic lines with ∼50% transmission rate for subsequent integration.</li>
</ol>
</li>
<li>For each strain that you desire to integrate, pick 140 array-positive L4 hermaphrodites onto 1–3 60 mm OP50-seeded NGM plate(s).</li>
<li>Place the plates in a cesium irradiator and irradiate with 3800 Rad gamma irradiation following the instrument-specific instructions.</li>
<li>Transfer worms to 60 mm OP50-seeded NGM plates, preferably 5 fluorescent worms per plate for a total of 28 plates per strain and incubate at 15°C until the food is exhausted.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Wrap the plates in Parafilm to avoid excessive loss of moisture.</div>
<ol start="10">
<li>When the food is recently exhausted, transfer worms from each plate to a new set of 60 mm OP50-seeded NGM plates and incubate ∼2 days.</li>
<li>For each plate, transfer 15 fluorescent worms to individual 35 mm OP50-seeded NGM plates for a total of 420 plates per strain.</li>
<li>Screen the progeny for lines with 100% fluorescence before the food is exhausted.</li>
<li>Outcross the integrated lines by mating to N2 males.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As some arrays integrate more easily than others, we advise attempting to integrate 2–3 extrachromosomal lines per construct.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>rat anti-HA</td>
<td>Roche</td>
<td>Cat#11867423001</td>
</tr>
<tr>
<td>goat anti-rat IRDye 680RD</td>
<td>LI-COR Biosciences</td>
<td>Cat#925-68076; Lot#C61115-06</td>
</tr>
<tr>
<td>streptavidin-IRDye 800CW</td>
<td>LI-COR Biosciences</td>
<td>Cat#925-32230; Lot#C60913-04</td>
</tr>
<tr>
<td>mouse anti-HA</td>
<td>Abcam</td>
<td>Cat#ab130275 [16B12]; Lot#GR250145-5</td>
</tr>
<tr>
<td>CY3-anti-mouse</td>
<td>Jackson Immunoresearch Laboratories</td>
<td>Cat#115-165-166; Lot#117091</td>
</tr>
<tr>
<td>streptavidin Alexa Fluor 488</td>
<td>Invitrogen</td>
<td>Ref#532354; Lot#1719656</td>
</tr>
<tr>
<td>DAPI</td>
<td>Sigma-Aldrich</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>E. coli</i> (MG1655 <i>bioB</i>::kan) biotin auxotrophic food</td>
<td>Dr. John E. Cronan, University of Illinois</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>E. coli</i> OP50 standard food</td>
<td>CAENORHABDITIS GENETICS CENTER</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>E. coli</i> NA22 standard food</td>
<td>CAENORHABDITIS GENETICS CENTER</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Pierce 660nm Protein Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#22662</td>
</tr>
<tr>
<td>Pierce Streptavidin Magnetic Beads</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#88817</td>
</tr>
<tr>
<td>4–20% Mini-PROTEAN TGX PAGE gel</td>
<td>Bio-Rad Laboratories</td>
<td>Cat#4561093</td>
</tr>
<tr>
<td>Nitrocellulose membrane 0.4μm</td>
<td>Bio-Rad Laboratories</td>
<td>Cat#162-0115</td>
</tr>
<tr>
<td>Fast prep beads, Lysing matrix C, 1.0 mm silica spheres</td>
<td>MP Biomedicals</td>
<td>Cat#MP116912100</td>
</tr>
<tr>
<td>4× Laemmli Sample Buffer</td>
<td>Bio-Rad Laboratories</td>
<td>Cat#1610747</td>
</tr>
<tr>
<td>Beckman Coulter ultracentrifuge tubes (13 × 51 mm)</td>
<td>Fisher Scientific</td>
<td>Cat#NC9529688</td>
</tr>
<tr>
<td>HALT Protease Inhibitor Cocktail, EDTA free (100×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#78447</td>
</tr>
<tr>
<td>Ponceau S solution</td>
<td>Sigma-Aldrich</td>
<td>Cat#P7170</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>C. elegans</i> strain N2 wild-type</td>
<td>CGC</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>C. elegans</i> strain JLF291: <i>wowEx68</i>(<i>myo-2</i>p::<i>mCherry</i>::<i>unc-54</i> 3'UTR; <i>ges-1</i>p::<i>3xHA</i>::<i>TurboID</i>::<i>unc-54</i> 3'UTR)</td>
<td><sup>25</sup></td>
<td>CGC</td>
</tr>
<tr>
<td>
<i>C. elegans</i> strain JLF337: <i>wowIs24</i>(<i>myo-2</i>p::<i>mCherry</i>::<i>unc-54</i> 3'UTR;<i>ges-1</i>p::<i>ptrn-1a</i>::<i>3xHA</i>::<i>TurboID</i>::<i>unc-54</i> 3'UTR)</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>C. elegans</i> strain JLF371: <i>wowIs26</i>(<i>myo-2</i>p::<i>mCherry</i>::<i>unc-54</i> 3'UTR; <i>ges-1</i>p::<i>ptrn-1a</i>::<i>3xHA</i>::<i>TurboID</i>::<i>unc-54</i> 3'UTR)</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>C. elegans</i> strain JLF372: <i>wowIs17</i>(<i>myo-2</i>p::<i>mCherry</i>::<i>unc-54</i> 3'UTR; <i>ges-1</i>p::<i>3xHA</i>::<i>TurboID</i>::<i>unc-54</i> 3'UTR)</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Plasmid pAS31 [<i>ges-1</i>p::<i>3xHA</i>::<i>TurboID</i>::<i>unc-54</i> 3'UTR]</td>
<td>Addgene</td>
<td>Addgene plasmid # 118220; <a href="http://n2t.net/addgene:118220">http://n2t.net/addgene:118220</a>; RRID: Addgene_118220</td>
</tr>
<tr>
<td>Plasmid pAS33 [<i>ges-1</i>p::<i>ptrn-1a</i>::<i>3xHA</i>::<i>TurboID</i>::<i>unc-54</i> 3'UTR]</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pCFJ90 [<i>myo-2</i>p::<i>mcherry</i>::<i>unc-54</i> 3'UTR]</td>
<td>Addgene</td>
<td>Addgene plasmid # 19327; <a href="http://n2t.net/addgene:19327">http://n2t.net/addgene:19327</a>; RRID:Addgene_19327</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Image J (version 2.1.0)</td>
<td>NIH</td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td>NIS elements</td>
<td>Nikon Instruments</td>
<td><a href="https://www.microscope.healthcare.nikon.com/products/software/nis-elements">https://www.microscope.healthcare.nikon.com/products/software/nis-elements</a></td>
</tr>
<tr>
<td>Image Studio Software</td>
<td>LI-COR Biosciences</td>
<td><a href="https://www.licor.com">https://www.licor.com</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl2">
<caption>High-SDS RIPA lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 8.0</td>
<td>50 mM</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
</tr>
<tr>
<td>SDS</td>
<td>1% (wt/vol)</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.5% (wt/vol)</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>1% (vol/vol)</td>
</tr>
<tr>
<td>Leupeptin</td>
<td>2.5 mg/mL</td>
</tr>
<tr>
<td>Pepstatin</td>
<td>5 mg/mL</td>
</tr>
<tr>
<td>PMSF</td>
<td>1mM</td>
</tr>
<tr>
<td>HALT</td>
<td>1×</td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh in ddH<sub>2</sub>O the day of use</p>
<table id="undtbl3">
<caption>SDS-free RIPA lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 8.0</td>
<td>50 mM</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.5% (wt/vol)</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>1% (vol/vol)</td>
</tr>
<tr>
<td>Leupeptin</td>
<td>2.5 mg/mL</td>
</tr>
<tr>
<td>Pepstatin</td>
<td>5 mg/mL</td>
</tr>
<tr>
<td>PMSF</td>
<td>1 mM</td>
</tr>
<tr>
<td>HALT</td>
<td>1×</td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh in ddH<sub>2</sub>O the day of use</p>
<table id="undtbl4">
<caption>RIPA lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 8.0</td>
<td>50 mM</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
</tr>
<tr>
<td>SDS</td>
<td>0.2% (wt/vol)</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.5% (wt/vol)</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>1% (vol/vol)</td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh in ddH<sub>2</sub>O the day of use</p>
<div class="note">
<span class="note-title">Note:</span> Prepare all lysis buffers fresh on the day of use.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare the stock solution of Tris-HCl (pH 8.0), dissolve Tris into water and adjust the pH to 8.0 with hydrochloric acid (HCl). This buffer can be stored for several months at 20°C–23°C.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare stock solution of PMSF, dissolve PMSF in isopropanol. This solution can be stored at –20°C for several months.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare stock solution of Leupeptin, dissolve Leupeptin in water. This solution can be stored at –20°C for several months.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare stock solution of Pepstatin, dissolve Pepstatin in DMSO. This solution can be stored at –20°C for several months.</div>
<table id="undtbl5">
<caption>NGM agar plates</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>3 g</td>
</tr>
<tr>
<td>Agar</td>
<td>17 g</td>
</tr>
<tr>
<td>Peptone</td>
<td>2.5 g</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>975 mL autoclave the solution, cool down the solution to 55°C, then add the following ingredients.</td>
</tr>
<tr>
<td>1 M CaCl<sub>2</sub>
</td>
<td>1 mL</td>
</tr>
<tr>
<td>5 mg/mL Cholesterol in 95% ethanol</td>
<td>1 mL</td>
</tr>
<tr>
<td>Uracil</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M Potassium phosphate buffer</td>
<td>25 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store NGM agar plates at 4°C in sealed boxes and use within one month.</div>
<table id="undtbl6">
<caption>Peptone-rich NGM agar plates</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>1.2 g</td>
</tr>
<tr>
<td>Agar</td>
<td>25 g</td>
</tr>
<tr>
<td>Peptone</td>
<td>20 g</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>975 mL autoclave the solution, cool down the solution to 55°C, then add the following ingredients.</td>
</tr>
<tr>
<td>1 M MgSO<sub>4</sub>
</td>
<td>1 mL</td>
</tr>
<tr>
<td>5 mg/mL Cholesterol in 95% ethanol</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M Potassium phosphate buffer</td>
<td>25 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store peptone-rich NGM agar plates at 4°C in sealed boxes and use within one month.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Grow <i>C. elegans</i> on a large scale and collect strains</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8 days</div>
<p>To prepare for large scale experiments, expand the <i>C. elegans</i> strain populations then collect and freeze worms at –80°C. The amount of sample to be collected will depend on the scale and developmental stage desired for any given experiment.</p>
<ol>
<li>Synchronize and expand the <i>C. elegans</i> strains. Refer to troubleshooting <a href="#sec6.9">problem 5</a> for additional information.
<ol type="a">
<li>Transfer freshly starved L1 worms onto 150 mm NA22-seeded peptone-rich NGM plates.</li>
<li>Once the hermaphrodites are gravid but not yet starved, bleach them to collect embryos and expand synchronized populations.
<ol type="i">
<li>Wash gravid hermaphrodites off of plates with M9 buffer.</li>
<li>Centrifuge the worms in conical tubes at 850×<i>g</i> for 1 min.</li>
<li>Remove the supernatant.</li>
<li>Add freshly prepared bleach (98 mL water, 6 mL 5N KOH, 12 mL sodium hypochlorite (10–15% stock)) and gently invert for approximately 7 min.</li>
<li>Centrifuge the samples at 850×<i>g</i> for 1 min.</li>
<li>Remove the supernatant.</li>
<li>To wash away the bleach, add M9 buffer and gently invert the tube for 5–15 s.</li>
<li>Repeat steps v. - vii. at least four times to completely wash away any residual bleach from the worms.</li>
<li>Plate approximately 100,000 embryos per 150 mm NA22-seeded peptone-rich NGM plate.</li>
<li>Incubate the worms at 25°C.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The bleach solution should be prepared fresh the day of use.</div>
<div class="note">
<span class="note-title">Note:</span> Do not allow the bleach treatment to last more than 8–10 min. Excessive exposure to bleach will damage or kill the embryos and thus reduce the yield of viable embryos.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Growing worms at higher temperatures (25°C) increases biotinylation activity compared to lower temperatures (15°C–20°C).</div>
<ol start="2">
<li>Collect and freeze worms at the desired developmental stage.
<ol type="a">
<li>Wash worms off of the plates with M9 buffer.</li>
<li>Centrifuge worms in conical tubes at 850×<i>g</i> for 1 min.</li>
<li>Remove the supernatant and add back fresh M9 buffer.</li>
<li>Repeat steps b. – c. until supernatant is clear of bacteria (approximately five times).</li>
<li>For the final wash of worms with M9 buffer, allow the worms to settle to the bottom of the tube by gravity. To expedite this step, place the worms on ice.</li>
<li>Aliquot the worms in desired amounts into fresh Eppendorf tubes.</li>
<li>Let the worms settle on ice.</li>
<li>Remove as much M9 buffer supernatant as possible.</li>
<li>Freeze the pellets in liquid nitrogen and store them at –80°C.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Frozen pellets can be stored at –80°C for several months.</div>
<div class="note">
<span class="note-title">Note:</span> Performing thorough washes to remove bacteria from <i>C. elegans</i> samples is critical for the quality of the subsequent lysis and biotinylation enrichment steps.</div>
<div class="note">
<span class="note-title">Note:</span> When collecting larval or adult stages of worms, place them on ice to accelerate the rate of worms settling to the bottom of the tube. Placing samples on ice also slows the activity of biotin ligases at any stage of worm development.</div>
<div class="note">
<span class="note-title">Note:</span> Avoid future freeze-thaws by aliquoting small amounts into tubes. In our case, we froze 500 μL worm pellets.</div>
<h3 id="sec3.2">Lyse <i>C. elegans</i> strains and enrich biotinylated proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>To collect biotinylated proteins, first lyse the worms and then capture biotinylated proteins on streptavidin-covered magnetic beads. The steps for capturing biotinylated proteins have been adapted (<a href="#bib1">Branon et al., 2018</a>; <a href="#bib5">Hung et al., 2016</a>).</p>
<ol start="3">
<li>Before proceeding, prepare all lysis buffers and place them on ice.
<ol type="a">
<li>
<i><u>High-SDS RIPA lysis buffer</u></i>: 50 mM Tris-HCl (pH 8.0), 150 mM NaCl, 1% (wt/vol) SDS, 0.5% (wt/vol) sodium deoxycholate, 1% (vol/vol) Triton X-100, 2.5 mg/mL Leupeptin, 5 mg/mL Pepstatin, 1 mM PMSF, 1× HALT solution in water.</li>
<li>
<i><u>SDS-free RIPA lysis buffer</u></i>: 50 mM Tris-HCl (pH 8.0), 150 mM NaCl, 0.5% (wt/vol) sodium deoxycholate, 1% (vol/vol) Triton X-100, 2.5 mg/mL Leupeptin, 5 mg/mL Pepstatin, 1 mM PMSF, 1× HALT solution in water.</li>
<li>
<i><u>RIPA lysis buffer</u></i>: 50 mM Tris-HCl (pH 8.0), 150 mM NaCl, 0.2% (wt/vol) SDS, 0.5% (wt/vol) sodium deoxycholate, 1% (vol/vol) Triton X-100 in water.</li>
</ol>
</li>
<li>Transfer worm pellets from –80°C to ice.</li>
<li>Add 500 μL of high SDS RIPA lysis buffer for every 500 μL of worm pellet.</li>
<li>Once the pellet is thawed to 4°C, transfer the entire sample to a tube containing silica spheres (for L4 or adult stages we used "lysing matrix C" 1.0 mm silica spheres, MP Biomedicals).</li>
<li>Agitate at 6.5 m/s for 20 s pulses with 5-min breaks on ice (FastPrep-24 Classic) for a total of three times.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> After agitation, take a small aliquot to check the status of lysis under a dissecting microscope.</div>
<ol start="8">
<li>Avoiding the silica spheres, transfer the entire sample to clean tubes appropriate for sonication. You might need to add additional high SDS RIPA lysis buffer to ensure that the total volume of liquid is set for proper submersion of the sonication apparatus.</li>
<li>Sonicate at 18% at 5 s intervals with 5 s breaks for 50 s total in a 4°C water bath (Branson Sonifier SFX250).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> After sonication treatment, take a small aliquot to check the status of lysis under a dissecting microscope. Refer to troubleshooting <a href="#sec6.11">problem 6</a> for additional information.</div>
<ol start="10">
<li>Add SDS-free RIPA buffer to achieve a final SDS concentration of 0.2%.</li>
<li>Gently rotate the lysate on a slow spinning wheel for 1 h at 4°C.</li>
<li>Transfer the samples to ultracentrifuge tubes (13 × 51 mm), and spin them at 4°C for 30 min at 51,000 RPM in a TLA-100.3 ultracentrifuge rotor (Beckman).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use larger ultracentrifuge tubes when handling larger volumes of sample.</div>
<ol start="13">
<li>Carefully collect the liquid layer (avoid the lipid layer) and transfer to a clean Eppendorf tube.</li>
<li>Take 2% of liquid fraction (this is the "input" fraction) and save it on ice.</li>
<li>Measure the protein concentration for each sample using the Pierce 660-nm Protein Assay (Thermo Fisher Scientific). At this point of the protocol, we calculated an average protein concentration of 1.6 mg/mL. Note that this is from a 500 μL pellet of worms that has gone through several dilution steps.</li>
<li>Wash the streptavidin-coated magnetic beads (Pierce Streptavidin Magnetic Beads, Thermo Fisher Scientific, #88817) by adding 1 mL of RIPA lysis buffer, gently rotating on a spinning wheel for 5 min, then pelleting the beads with a magnetic rack (Thermo Fisher) and discarding the supernatant. Repeat a total of 2–3 times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When handling streptavidin magnetic beads, use pipette tips that are 200 μL or larger with the tips cut off with a clean razor blade.</div>
<ol start="17">
<li>Add the optimized volume of streptavidin-coated magnetic beads to 1 mg of protein for each sample. We used 125 μL beads to achieve a ratio of 8 μg protein/1 μL beads.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The ratio of protein concentration to bead volume is important to optimize with bead titration tests. For example, the use of more streptavidin beads than necessary can increase the number of nonspecific binders, thus creating background. Conversely, the use of less streptavidin beads than necessary will exclude biotinylated proteins.</div>
<ol start="18">
<li>If samples are at unequal volumes, add additional RIPA lysis buffer to achieve similar volumes across samples. This step is to bring all samples to similar concentrations and will ease sample mixing.</li>
<li>Gently rotate the lysate on a slow spinning wheel at 4°C for 16 h.</li>
<li>Store the rest of the lysate at –80°C.</li>
<li>Pellet the beads using a magnetic rack and remove the supernatant.</li>
<li>Set aside 2% of the supernatant (this is the "flow-through" fraction) and save it on ice.</li>
<li>Wash each bead sample with buffers to remove nonspecific binders. Keep the samples on ice between washes.
<ol type="a">
<li>Add 1 mL of cold RIPA buffer, gently rotate at 20°C–23°C on a spinning wheel for 2 min, then pellet beads using a magnetic rack and discard the supernatant.</li>
<li>Add 1 mL of cold 1M KCl, gently rotate at 20°C–23°C on a spinning wheel for 2 min, then pellet beads using a magnetic rack and discard the supernatant.</li>
<li>Add 1 mL of cold 0.1 M Na<sub>2</sub>CO<sub>3</sub>, gently rotate at 20°C–23°C on a spinning wheel for 2 min, then pellet beads using a magnetic rack and discard the supernatant.</li>
<li>Add 1 mL of 2 M urea, gently rotate at 20°C–23°C on a spinning wheel for 2 min, then pellet beads using a magnetic rack and discard the supernatant.</li>
<li>Add 1 mL of 4 M urea, gently rotate at 20°C–23°C on a spinning wheel for 2 min, then pellet beads using a magnetic rack and discard the supernatant.</li>
<li>Add 1 mL of cold RIPA buffer, gently rotate at 20°C–23°C on a spinning wheel for 2 min then pellet beads using a magnetic rack and discard the supernatant.</li>
<li>Repeat step <b>f.</b> twice total.</li>
<li>Add 1 mL of cold PBS, gently rotate at 20°C–23°C on a spinning wheel for 2 min then pellet beads using a magnetic rack and discard supernatant.</li>
<li>Repeat step <b>h.</b> 5 times total.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Each wash reagent and the frequency of washes can be optimized to ensure that only proteins covalently tagged with biotin are enriched.</div>
<div class="note">
<span class="note-title">Note:</span> Urea solutions should be prepared in 10 mM Tris-HCl (pH 8.0). Dissolve urea in the Tris buffer and adjust the pH with hydrochloric acid (HCl). These solutions should be made fresh on the day of use.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare stock solutions of KCl, and Na<sub>2</sub>CO<sub>3</sub> in water; these can be stored for several months at 20°C–23°C.</div>
<div class="note">
<span class="note-title">Note:</span> At this point, biotinylated proteins that were enriched with the streptavidin magnetic beads can now be identified. The steps for identifying biotinylated proteins are not outlined in this protocol as the specific steps will depend on your experimental design considerations and the mass spectrometry protocols available to you. Contact your mass spectrometry facility to discuss a protocol for identifying captured biotinylated proteins. Also note that you might need to scale up the experiment if the mass spec protocol calls for a higher amount of protein.</div>
<ol start="24">
<li>To run samples for western blot analysis, elute the biotinylated proteins from the beads. Biotin is included in this step to elute biotinylated proteins more effectively.
<ol type="a">
<li>Add 3× sample buffer supplemented with 1:10 β-mercaptoethanol and 2 mM biotin.</li>
<li>Boil at 95°C for 10 min.</li>
<li>Vortex beads briefly then cool samples on ice.</li>
<li>Centrifuge the tubes briefly to bring down condensation.</li>
<li>Pellet the beads using a magnetic rack.</li>
<li>Collect eluate (this will be the biotinylation eluate fraction).</li>
<li>Place the biotinylation eluate fraction on ice.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To make a 100 mM stock solution of biotin, dissolve biotin in DMSO. Stock aliquots can be stored at –20°C for several months.</div>
<ol start="25">
<li>Add 4× sample buffer (supplemented with 1:10 β-mercaptoethanol) to each input and flow-through fractions. Dilute 3 parts sample with 1 part of 4× sample buffer.</li>
<li>Boil at 95°C for 10 min.</li>
<li>Place the samples back on ice.</li>
<li>Run equal volumes of input, flow-through, and biotinylation eluate samples for each genotype on one SDS polyacrylamide gel for subsequent western blot analysis to assess transgene construct expression, overall biotinylation patterns and levels, and to check that biotinylated proteins are enriched in the eluate and are not present in the flow-through.
<ol type="a">
<li>Transfer the samples to a nitrocellulose membrane (0.4 μm, Bio-Rad)</li>
<li>Stain the membrane with Ponceau S solution to check the quality of the protein transfer. Then, remove the Ponceau S stain by rinsing the membrane with water.</li>
<li>Block the membrane with 5% milk in 1× PBST solution for 30 min to 1 h at 20°C–23°C or for 16 h at 4°C.</li>
<li>To detect proteins on a fluorescent imaging system (e.g., LI-COR), incubate the membrane with anti-HA primary antibody (1:5000 in 1× PBST, rat monoclonal, Roche) 1 h at 20°C–23°C.</li>
<li>Wash the membrane three times with 1× PBST for 5 min each.</li>
<li>Incubate the membrane with a secondary antibody (1:5000, goat anti-rat IRDye 680RD, Licor) and streptavidin-IR Dye (1:5000, 800CW, Licor) in 1× PBST for 1 h at 20°C–23°C.</li>
<li>Wash the membrane three times with 1× PBST for 5 min each.</li>
<li>Image the blot on LI-COR Odyssey CLx or an equivalent technology.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All samples during and after worm lysis should remain at 4°C.</div>
<div class="optional">
<span class="optional-title">Optional:</span> You can split your sample to run on two SDS polyacrylamide gels such that one is used for western blotting and the other is used for silver stain. The silver stain is used to visualize all of the protein bands in the biotinylation eluate of each sample, which is useful for troubleshooting (see troubleshooting <a href="#sec6.17">problem 9</a>).</div>
<div class="note">
<span class="note-title">Note:</span> We recommend using glass pipettes to handle large quantities of worms because worms stick less to the side of glass tips compared to various plastics.</div>
<div class="note">
<span class="note-title">Note:</span> Refer to troubleshooting <a href="#sec6.13">problems 7</a>, <a href="#sec6.15">8</a>, and <a href="#sec6.17">9</a> for additional information.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Successful TurboID based proximity labeling in <i>C. elegans</i> can be assessed by several factors: 1) A clear enrichment of biotinylated proteins over background in the western blotting assay, including biotinylation of the TurboID fusion protein (<a href="#fig1">Figure 1</a>B). Background endogenously biotinylated proteins in the worm are most highly enriched at ∼75 kDa and ∼125 kDa (<a href="#bib11">Watts et al., 2018</a>); 2) IF staining of the epitope tag and of biotinylation with a streptavidin conjugate will show whether the construct correctly localizes and whether biotinylation activity is regionally localized (<a href="#fig1">Figure 1</a>A); 3) Mass spectrometry results show an enrichment of proteins predicted to be expressed in the tissue of choice. We used previously published RNAseq datasets to make this judgement; 4) Mass spectrometry results show some expected proteins known to interact with the bait, although the absence of a known protein interactor does not necessarily mean the protocol was unsuccessful; 5) The endogenous localization of proximity interactors is similar to the localization of the bait protein of interest.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Although this protocol is useful for identifying protein-protein proximity interactions, it has several key limitations. First, this protocol is limited by the level of biotin ligase expression as we have found that overexpression of biotin ligase enzymes in the intestine leads to higher biotinylation activity than a single copy insertion. Performing a diagnostic check of biotinylation levels by IF staining and western blotting will be key for assessing this potential limitation. Second, from our protocol we did not pinpoint the developmental time that biotinylation was occurring. If developmental timing is important for your protocol, it is important to assay biotinylation activity with IF staining and western blotting at different stages of <i>C. elegans</i> development. To narrow biotinylation activity to a tighter developmental time window, strains can be grown on biotin-depleted bacteria and then fed biotin-rich bacteria at the desired developmental stage. Third, it is important to keep in mind that TurboID is limited by its predicted 10 nm labeling radius, the amount of available lysines in the vicinity of TurboID, the total amount of free biotin available, and restricted access to sterically inaccessible proteins (<a href="#bib1">Branon et al., 2018</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Microinjection of plasmids leads to low array transmission (<a href="#before-you-begin">before you begin</a>, step 2).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Try lowering the concentration of the co-injection marker and/or the TurboID plasmid.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The TurboID construct shows proper localization by IF staining but biotinylation signal over background is not detected (<a href="#before-you-begin">before you begin</a>, step 4).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Investigate whether the low biotinylation activity is due to low levels of TurboID expression and/or low levels of free biotin in the worm. Try increasing the expression of the TurboID construct by changing the promoter, further codon-optimizing of TurboID, high copy array expression, and/or increasing the worm growth temperature. If the problem of low biotinylation levels persists, it is likely that the level of available biotin in the worm is too low. After growing worms on biotin-rich bacteria, try incubating worms in M9 buffer supplemented with excess biotin or in liquid bacteria culture supplemented with excess biotin.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>There is biotinylation signal over background, but the biotinylation signal looks diffuse by IF staining (<a href="#before-you-begin">before you begin</a>, step 4).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>It is critical that both the TurboID construct and the biotinylation signal overlap as much as possible. If the IF staining indicates that biotinylation activity is not enriched to the target subcellular region, then try targeting TurboID with an alternative bait protein. If an alternative bait is lacking, the mass spectrometry data from the control samples can be used as a guide to filter out background biotinylated proteins in the experimental sample.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The TurboID construct is poorly expressed (<a href="#before-you-begin">before you begin</a>, step 5).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Tune expression level by changing the promoter. Additionally, we have shown that growing worms at a higher temperature increases expression level (<a href="#bib1">Branon et al., 2018</a>). Lastly, try further codon-optimization of the TurboID sequence.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Worm bleaching is not effective for expanding and synchronizing the worm population (Method Step 1).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>First ensure that the bleach solution is made fresh the day of use. Second, check that there is enough bleach solution available for the number of gravid adults. If worms are not disintegrated within ∼7 min, then a larger volume of bleach needs to be used. Third, after plating the embryos of bleached adults, check the following day for viability. A low rate of embryo survival could indicate excessive bleach treatment.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Worm lysis is not effective (Method Step 9).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>There are many approaches to worm lysis and the procedure can be optimized for the particular stage of worm development and the tools on hand. The lysis steps outlined in this protocol are ideal for L4 or adult stages of worms. Alternative worm lysis approaches include flash freezing worms in liquid nitrogen and subsequently grinding up worms into a fine powder with a mortar and pestle. Additionally, if you have access to a cell disrupter (Constant systems cell disruptor; Model MC/BA), this instrument can also be used for lysing <i>C. elegans</i> at various stages of development.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>The 'flow-through' lanes on the western blot contain biotinylated proteins (Method Step 28).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>This is likely an indication that all of the biotinylated proteins are not fully captured on the streptavidin beads. Use more beads to capture more biotinylated proteins. Optimize the streptavidin bead to lysate ratio by testing various bead volumes for a given concentration of worm lysate.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>Poor enrichment of biotinylated proteins by streptavidin-covered magnetic beads (Method Step 28).</p>
<h3 id="sec6.16">Potential solution</h3>
<p>This problem could be the result of too few streptavidin beads used for the enrichment step. Alternatively, the elution step could be optimized by increasing the boiling time. Lastly, it is possible that there was too much biotin in the initial worm sample prior to lysis. This is why thoroughly washing the worms of excess bacteria prior to lysis is critical.</p>
<h3 id="sec6.17">Problem 9</h3>
<p>The negative control sample (the worm strain without TurboID) shows too much material eluted from the streptavidin beads. For example, the biotinylated proteins eluted from the streptavidin beads look similar for the experimental and negative control sample on the silver stain gel (Method Step 28).</p>
<h3 id="sec6.18">Potential solution</h3>
<p>This is likely an indication that the streptavidin beads are capturing too many nonspecific binders. This problem can be addressed in two different ways: 1) make the streptavidin bead wash steps more stringent. Try increasing the total number of washes, the wash volumes, or the duration of each wash step. Additionally, try including a wash step with a higher concentration of urea; 2) optimize the streptavidin bead to lysate ratio. If too many nonspecific binders are captured, reduce the amount of streptavidin beads used.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Branon, T.C., Bosch, J.A., Sanchez, A.D., Udeshi, N.D., Svinkina, T., Carr, S.A., Feldman, J.L., Perrimon, N., and Ting, A.Y. (2018). Efficient proximity labeling in living cells and organisms with TurboID. Nat. Biotechnol. <i>6</i>, e02357. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref1">View at publisher</a></p>
<p id="bib2">Cho, K.F., Branon, T.C., Udeshi, N.D., Myers, S.A., Carr, S.A., and Ting, A.Y. (2020). Proximity labeling in mammalian cells with TurboID and split-TurboID. Nat. Protoc. <i>15</i>, 3971-3999. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref2">View at publisher</a></p>
<p id="bib3">Evans, T.C. (2006). Transformation and microinjection (April 6, 2006). In The C. elegans Research Community, WormBook, ed. (WormBook). <a class="external-link" href="https://doi.org/10.1895/wormbook">https://doi.org/10.1895/wormbook</a></p>
<p id="bib4">Feldman, J.L. and Priess, J.R. (2012). A Role for the centrosome and PAR-3 in the hand-off of MTOC function during epithelial polarization. Curr. Biol. <i>22</i>, 575-582. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref4">View at publisher</a></p>
<p id="bib5">Hung, V., Udeshi, N.D., Lam, S.S., Loh, K.H., Cox, K.J., Pedram, K., Carr, S.A., and Ting, A.Y. (2016). Spatially resolved proteomic mapping in living cells with the engineered peroxidase APEX2. Nat. Protoc. <i>11</i>, 456-475. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref5">View at publisher</a></p>
<p id="bib6">Ortega-Cuellar, D., Hernandez-Mendoza, A., Moreno-Arriola, E., Carvajal-Aguilera, K., Perez-Vazquez, V., Gonzalez-Alvarez, R., and Velazquez-Arellano, A. (2010). Biotin starvation with adequate glucose provision causes paradoxical changes in fuel metabolism gene expression similar in rat (<i>Rattus norvegicus</i>), nematode (<i>Caenorhabditis elegans</i>) and yeast (<i>Saccharomyces cerevisiae</i>). J. Nutrigenet. Nutrigenom. <i>3</i>, 18-30. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref6">View at publisher</a></p>
<p id="bib7">Pauli, F., Liu, Y., Kim, Y.A., Chen, P., and Kim, S.K. (2006). Chromosomal clustering and GATA transcriptional regulation of intestine-expressed genes in <i>C. elegans</i>. Development <i>133</i>, 287-295. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref7">View at publisher</a></p>
<p id="bib8">Richardson, C.E., Spilker, K.A., Cueva, J.G., Perrino, J., Goodman, M.B., and Shen, K. (2014). PTRN-1, a microtubule minus end-binding CAMSAP homolog, promotes microtubule function in <i>Caenorhabditis elegans</i> neurons. eLife <i>3</i>, e01498. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref8">View at publisher</a></p>
<p id="bib9">Sanchez, A.D., Branon, T.C., Cote, L.C., Papagiannakis, A., Liang, X., Pickett, M.A., Shen, K., Jacobs-Wagner, C., Ting, A.Y., and Feldman, J.L. (2021). Proximity labeling reveals non-centrosomal microtubule-organizing center components required for microtubule growth and localization. Curr. Biol. CB <i>31</i>, 3586-3600.e11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref9">View at publisher</a></p>
<p id="bib10">Wang, S., Wu, D., Quintin, S., Green, R.A., Cheerambathur, D.K., Ochoa, S.D., Desai, A., and Oegema, K. (2015). NOCA-1 functions with γ-tubulin and in parallel to Patronin to assemble non-centrosomal microtubule arrays in <i>C. elegans</i>. eLife <i>4</i>, e08649. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref10">View at publisher</a></p>
<p id="bib11">Watts, J.S., Morton, D.G., Kemphues, K.J., and Watts, J.L. (2018). The biotin-ligating protein BPL-1 is critical for lipid biosynthesis and polarization of the <i>Caenorhabditis elegans</i> embryo. J. Biol. Chem. <i>293</i>, 610-622. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00692-4/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jessica Feldman (<a href="mailto:feldmanj@stanford.edu">feldmanj@stanford.edu</a>).</p>
<h4>Materials availability</h4>
<p>DNA controls and transgenic <i>C. elegans</i> strains generated in this study are available from the Lead Contact, Dr. Jessica Feldman, upon request.</p>
<h4>Data and code availability</h4>
<ul>
<li>All data reported in this paper will be shared by the lead contact upon request.</li>
<li>Requests for any additional information required to conduct the protocol reported in this paper should be directed to the lead contact or the technical contact.</li>
</ul>
<h3>Acknowledgments</h3>
<p>We thank Tess C. Branon and Alice Y. Ting for a wonderful collaborative effort. Some of the nematode strains used in this work were provided by the Caenorhabditis Genetic Center, which is funded by the <a href="https://doi.org/10.13039/100000002">NIH</a> Office of Research Infrastructure Programs (P40 OD010440). This work was supported by an <a href="https://doi.org/10.13039/100000002">NIH</a> New Innovator Award DP2GM119136-01 and R01GM133950 awarded to J.L.F. A.D.S. was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> Training Grant 2T32GM007276.</p>
<h3>Author contributions</h3>
<p>Conceptualization, J.L.F. and A.D.S.; methodology, J.L.F. and A.D.S; formal analysis, A.D.S.; Writing – original draft, A.D.S.; writing – review &amp; editing, J.L.F. and A.D.S.; funding acquisition, J.L.F. and A.D.S.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>