<article>
<section>
<h1 id="header">Generation of recombinant vaccinia virus and analysis of virus-induced cell death</h1>
<p><time datetime="2021-10-08">Published: October 8, 2021</time></p>
<p>Zhijun Liu,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup> Kidong Kang,<sup><a href="#aff1">1</a></sup> and Francis Ka-Ming Chan<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Immunology, Duke University School of Medicine, DUMC 3010, Durham, NC 27710, USA</p>
<p id="fn1"><sup>2</sup>Present address: National Local Joint Engineering Research Center for Precision Surgery &amp; Regenerative Medicine and Surgical Engineering, First Affiliated Hospital of Xi’an Jiaotong University, Xi’an, China</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:franciskaming.chan@duke.edu">franciskaming.chan@duke.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:lzj9840824@163.com">lzj9840824@163.com</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100871">10.1016/j.xpro.2021.100871</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Vaccinia virus is a large double-stranded DNA virus that is widely used to express foreign genes from different origins. We generated recombinant vaccinia virus that expresses a viral inhibitor to examine its effect on virus-induced necroptosis. We provide a detailed protocol to describe the generation of recombinant vaccinia virus, validation of protein expression, and determination of necroptosis using live cell imaging. This approach can be adapted to examine the effect of other cell death regulators on virus-induced cell death.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00577-3/sref3">Liu et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Construction of targeting vectors for generation of recombinant vaccinia virus</li>
<li>Selection of recombinant vaccinia virus based on plaque size</li>
<li>Amplification of recombinant vaccinia virus</li>
<li>Validation of target protein expression and examine the effect on virus-induced cell death</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1059-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the generation of recombinant vaccinia virus expressing the viral inducer of RIPK3 degradation (vIRD) and its effect on virus-induced necroptosis in the murine fibrosarcoma L929. Before we begin, we need to generate a targeting vector that contains vIRD. There are several different strategies for generating recombinant vaccinia virus (<a href="#bib2">Cotter et al., 2017</a>). We describe a method based on that described by Blasco and Moss (<a href="#bib1">Blasco and Moss, 1995</a>). This method uses the vaccinia virus strain vRB12, which contains a deletion of the F13L (aka <i>vp37</i>) gene. F13L facilitates cell-to-cell infection. Thus, vRB12 forms small plaques on monolayer cell infection (<a href="#bib1">Blasco and Moss, 1995</a>). Normal plaque size is restored when the F13L gene is repaired through homologous recombination with the pRB21 plasmid. Introduction of any gene of interest in pRB21 will simultaneously allow foreign gene expression and restoration of large plaque formation. This method therefore permits easy visual selection of recombinant viruses based on plaque size. Because vaccinia virus exhibits wide range of cell tropism, this protocol can also be adapted for infection of other cell types such as primary bone marrow derived macrophages, J2 virus-transformed macrophages, mouse embryonic fibroblasts, and other transformed cell lines.</p>
<div class="note">
<span class="note-title">Note:</span> Laboratory personnel are advised to be vaccinated prior to working with vaccinia virus. Virus-associated waste should be neutralized with diluted bleach solution prior to disposal.</div>
<h3 id="sec1.1">Design PCR primers for HiFi DNA assembly</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>We use the NEBbuilder HiFi Assembly platform to create the targeting vector for generation of the recombinant virus. However, other cloning methods can also be used. We recommend the use of molecular biology program such as Snapgene (<a href="https://www.snapgene.com/">https://www.snapgene.com/</a>) for design of the cloning strategy. We use the plasmid pEGFP-C1-vIRD as template for the PCR reaction (<a href="#bib3">Liu et al., 2021</a>). The vIRD sequence is derived from the vIRD ortholog in cowpox virus (CPXV). The N-terminal GFP fusion allows use to track virus-infected cells by GFP fluorescence.</p>
<ol>
<li>Design PCR primers for the open reading frame of GFP-vIRD.</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th> </th>
<th>Primer sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>Forward primer</td>
<td>5′-ATTCCTGCAGGCTAGCCACCATGGTGAGCAAGGGCGAG-3′</td>
</tr>
<tr>
<td>Reverse primer</td>
<td>5′-ATTTAGGCCTCCATGGATCAATATGGGTAATGCTTG-3′</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The sequences in red correspond to the overlapping sequence in the pRB21 vector, while the sequences in black correspond to the sequence in GFP-vIRD. The overlapping pRB21 sequences is required for the HiFi DNA assembly platform. NEB recommends the overlapping sequence to be 15–20 bases long, which we found to give optimal results in most cases. Unlike cloning into mammalian expression vectors, it is not necessary to include Kozak sequence before the ATG start codon. More information on the principles of HiFi DNA assembly can be found in the NEB website (<a href="https://www.neb.com/applications/cloning-and-synthetic-biology/dna-assembly-and-cloning/nebuilder-hifi-dna-assembly">https://www.neb.com/applications/cloning-and-synthetic-biology/dna-assembly-and-cloning/nebuilder-hifi-dna-assembly</a>).</div>
<h3 id="sec1.2">Cloning of GFP-vIRD into the pRB21 targeting vector</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 days</div>
<ol start="2">
<li>Linearize the pRB21 vector with the restriction enzymes HindIII and Xmal.</li>
<li>Assemble the PCR reaction mixture per manufacturer’s instructions for the Q5® Hot Start High-Fidelity 2X Master Mix (<a href="https://www.neb.com/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491">https://www.neb.com/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491</a>). Perform PCR reaction using primers in step 1 with pEGFP-C1-vIRD as template.</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">25 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>58°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>2 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<ol start="4">
<li>Run the linearized vector and PCR products on 1% DNA agarose gel.</li>
<li>Excise the 5.5 kb linearized pRB21 band and the 2.64 kb band of the PCR product containing the DNA from the agarose gel. Use Zymoclean Gel DNA Recovery Kit to extract DNA from the excised gel.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other commercial kits such as the Qiagen DNA extraction kit can also be used.</div>
<ol start="6">
<li>Assemble the HiFi DNA assembly reaction using the linearized vector from step 2 and the PCR fragment from step 3. Refer to the manufacturer’s instructions (<a href="https://www.neb.com/protocols/2014/11/26/nebuilder-hifi-dna-assembly-reaction-protocol">https://www.neb.com/protocols/2014/11/26/nebuilder-hifi-dna-assembly-reaction-protocol</a>) for details on the reaction conditions.</li>
<li>Transform the HiFi DNA assembly product in step 6 into NEB 5-alpha competent <i>E. coli.</i>
<ol type="a">
<li>Thaw out a tube of NEB-5 alpha competent <i>E. coli</i> on ice.</li>
<li>Add 1 μL of the reaction product in step 6 to the competent E. coli.</li>
<li>Incubate on ice for 30 min</li>
<li>Heat shock the DNA-<i>E. coli</i> mixture at 42<sup>o</sup>C for 30 s.</li>
<li>Place on ice for 5 min.</li>
<li>Add 950 μL of Luria Broth (LB) to the tube.</li>
<li>Place the transformed E. coli in a 37<sup>o</sup>C shaker incubator. Set the shaker at 225 rpm.</li>
<li>Incubate for 1 h in the 37<sup>o</sup>C shaker.</li>
</ol>
</li>
<li>Spread 100 μL of the transformed <i>E. coli</i> on LB agar plate supplemented with 50 μg/mL Ampicillin.</li>
<li>Incubate the plate at 37<sup>o</sup>C for 14–20 h.</li>
<li>Pick colonies and grow them up in 5 mL LB containing 50 μg/mL Ampicillin for 14–20 h in a 37<sup>o</sup>C incubator.</li>
<li>Extract DNA from bacterial cultures using standard commercial kits (e.g., Qiagen miniprep kit).</li>
<li>Perform diagnostic restriction enzyme digestion to identify clones with the correct GFP-vIRD insert. For GFP-vIRD, we used BamHI and EcoRI, which will release the insert fragment of 2.64 kb.</li>
<li>Confirm the fidelity of the insert DNA sequence by Sanger sequencing. Primers are chosen from the 5′ and 3′ end of the cloning site to ensure full coverage of the insert.</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th> </th>
<th>Sequencing primer</th>
</tr>
</thead>
<tbody>
<tr>
<td>vp37 primer</td>
<td>5′-GAGAGAGATTGGGTGAGCTCAC-3</td>
</tr>
<tr>
<td>M13R universal primer</td>
<td>5′-CAGGAAACAGCTATGAC-3′</td>
</tr>
</tbody>
</table>
<h3 id="sec1.3">Preparation of cell culture media and other solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<table id="undtbl4">
<caption>Complete MEM-10 (store at 4<sup>o</sup>C for maximum of 6 months)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Heat-inactivated fetal bovine serum</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>L-glutamine (200 mM)</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>HEPES pH 7.2 (1 M)</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Non-essential amino acids (100X)</td>
<td>1X</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin (10,000 U/mL)</td>
<td>100 U/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>MEM</td>
<td>n/a</td>
<td>430 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>Complete DMEM-10 (store at 4<sup>o</sup>C for maximum of 6 months)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Heat-inactivated fetal bovine serum</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>L-glutamine (200 mM)</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>HEPES pH 7.2 (1 M)</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin (10,000 U/mL)</td>
<td>100 U/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>435 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>MEM-2.5 (store at 4<sup>o</sup>C for maximum of 6 months)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Heat-inactivated fetal bovine serum</td>
<td>2.5%</td>
<td>12.5 mL</td>
</tr>
<tr>
<td>L-glutamine (200 mM)</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>HEPES pH 7.2 (1 M)</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Non-essential amino acids (100X)</td>
<td>1X</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin (10,000 U/mL)</td>
<td>100 U/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>MEM</td>
<td>n/a</td>
<td>467.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl7">
<caption>MEM-2.5 with 2.5% methyl cellulose (store at 4<sup>o</sup>C for maximum of 6 months)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Methyl cellulose</td>
<td>2.5%</td>
<td>12.5 g</td>
</tr>
<tr>
<td>Heat-inactivated fetal bovine serum</td>
<td>2.5%</td>
<td>12.5 mL</td>
</tr>
<tr>
<td>L-glutamine (200 mM)</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>HEPES pH 7.2 (1 M)</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Non-essential amino acids (100X)</td>
<td>1X</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin (10,000 U/mL)</td>
<td>100 U/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>MEM</td>
<td>n/a</td>
<td>467.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td> </td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Sterilize the methyl cellulose powder and the stir bar by autoclave. Add MEM to the methyl cellulose with the stir bar in the autoclaved medium bottle. Methyl cellulose is viscous and will require 2–3 days of stirring at 4<sup>o</sup>C with magnetic stir bar to completely dissolve it. After the methyl cellulose is dissolved, the other ingredients can be added to the medium.</div>
<table id="undtbl8">
<caption>Crystal violet staining buffer (store at 20–25<sup>o</sup>C for maximum of 1 year)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Crystal violet</td>
<td>0.1%</td>
<td>0.1 g</td>
</tr>
<tr>
<td>10% Formalin</td>
<td>1%</td>
<td>10 mL</td>
</tr>
<tr>
<td>100% ethanol</td>
<td>10%</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>80 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl9">
<caption>RIPA lysis buffer (store at 4<sup>o</sup>C for maximum of 1 year)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-Cl (pH8.0)</td>
<td>25 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>5M NaCl</td>
<td>150 mM</td>
<td>3 mL</td>
</tr>
<tr>
<td>500 mM EDTA</td>
<td>0.5 mM</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>10% Sodium deoxycholate</td>
<td>0.5%</td>
<td>5 mL</td>
</tr>
<tr>
<td>50X Complete protease inhibitor cocktail</td>
<td>1X</td>
<td>2 mL</td>
</tr>
<tr>
<td>100X Phosphatase inhibitor cocktail</td>
<td>1X</td>
<td>85.4 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>42 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Protease and phosphatase inhibitor cocktails are labile and should be added to the RIPA lysis buffer just before use.</div>
<div class="critical">
<span class="critical-title">Critical:</span> BS-C-1 cells, an epithelial cell line of the African green monkey kidney origin, show pronounced density-dependent cell growth in tissue culture. Above a confluent cell density of approximately 1.5 × 10<sup>5</sup> cells/per cm<sup>2</sup>, cell growth and infectivity of the cells are significantly affected (<a href="#bib4">Sutherst et al., 1978</a>). Therefore, it is important to maintain sub-confluent culture condition to avoid overgrowth.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-GFP antibody (use at 1:1,000 dilution)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# MA5-1526-HRP; RRID: AB_2537651</td>
</tr>
<tr>
<td>β-Actin antibody (use at 1:3,000 dilution)</td>
<td>Cell Signaling Technology</td>
<td>Cat# 3700; RRID: AB_2242334</td>
</tr>
<tr>
<td>Anti-mouse IgG conjugated to HRP (use at 1:5,000 dilution)</td>
<td>Jackson ImmunoResearch</td>
<td>Cat# 115-005-003; RRID: AB_2338447</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>rVACV-GFP-vIRD</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>vRB12</td>
<td>Bernard Moss</td>
<td>N/A</td>
</tr>
<tr>
<td>NEB 5-alpha competent <i>E. coli</i>
</td>
<td>New England Biolabs</td>
<td>C2987H</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Opti-MEM Reduced Serum Medium</td>
<td>Gibco</td>
<td>Cat# 31985-070</td>
</tr>
<tr>
<td>Fetal Bovine Serum (Characterized)</td>
<td>HyClone</td>
<td>Cat# SH30071.03</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Gibco</td>
<td>Cat# 25030-164</td>
</tr>
<tr>
<td>Non-essential amino acids</td>
<td>Gibco</td>
<td>Cat# 11140</td>
</tr>
<tr>
<td>HEPES, pH7.2</td>
<td>Gibco</td>
<td>Cat# 15630</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.05%), phenol red</td>
<td>Gibco</td>
<td>Cat# 25300-054</td>
</tr>
<tr>
<td>Sterile PBS</td>
<td>VWR</td>
<td>Cat# 45000-446</td>
</tr>
<tr>
<td>DMEM</td>
<td>Corning</td>
<td>Cat# 10-013-CV</td>
</tr>
<tr>
<td>MEM</td>
<td>Corning</td>
<td>Cat# 10-009-CV</td>
</tr>
<tr>
<td>Penicillin–streptomycin solution, 503</td>
<td>Corning</td>
<td>Cat# 30-001-CI</td>
</tr>
<tr>
<td>Methyl cellulose</td>
<td>Sigma</td>
<td>Cat# M0512</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Sigma</td>
<td>Cat# A9393</td>
</tr>
<tr>
<td>Paraformaldehyde Solution, 4% in PBS, Thermo Scientific™</td>
<td>Thermo Scientific</td>
<td>Cat# J19943K2</td>
</tr>
<tr>
<td>Crystal violet</td>
<td>Sigma</td>
<td>Cat# C0775</td>
</tr>
<tr>
<td>Complete Protease Inhibitor Cocktail</td>
<td>Sigma</td>
<td>Cat# 11697498001</td>
</tr>
<tr>
<td>Phosphatase Inhibitor Cocktail</td>
<td>Sigma</td>
<td>Cat# P2850</td>
</tr>
<tr>
<td>4%–20% SurePAGE Bis-Tris</td>
<td>GenScript</td>
<td>Cat# M00657</td>
</tr>
<tr>
<td>LDS Sample Buffer (4X)</td>
<td>Novax</td>
<td>Cat# NP008</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Incucyte® Cytotox Green Reagent</td>
<td>Essen BioScience</td>
<td>Cat# 4633</td>
</tr>
<tr>
<td>Incucyte® Cytotox Red Reagent</td>
<td>Essen BioScience</td>
<td>Cat# 4632</td>
</tr>
<tr>
<td>TransIT®-LT1 Transfection Reagent</td>
<td>Mirus Bio</td>
<td>Cat# MIR2300</td>
</tr>
<tr>
<td>Q5® Hot Start High-Fidelity 2X Master Mix</td>
<td>New England Biolabs</td>
<td>Cat# M0494</td>
</tr>
<tr>
<td>Zymoclean Gel DNA Recovery Kit</td>
<td>Zymo Research</td>
<td>Cat# D4002</td>
</tr>
<tr>
<td>NEBuilder® HiFi DNA Assembly Cloning Kit</td>
<td>New England Biolabs</td>
<td>Cat# M5520</td>
</tr>
<tr>
<td>QIAGEN Miniprep Kit</td>
<td>QIAGEN</td>
<td>Cat# 27106</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>L929</td>
<td>ATCC</td>
<td>Cat# CCL-1; RRID: CVCL_0462</td>
</tr>
<tr>
<td>Vero</td>
<td>ATCC</td>
<td>Cat# CRL-1586; RRID: CVCL_0059</td>
</tr>
<tr>
<td>BS-C-1</td>
<td>ATCC</td>
<td>Cat# CCL-26; RRID: CVCL_0607</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>PCR primers for vIRD (forward): ATTCCTGCAGGCTAGCCACCATGGTGAGCAAGGGCGAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>PCR primers for vIRD (reverse): ATTTAGGCCTCCATGGATCAATATGGGTAATGCTTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Sequencing primers (vp37): GAGAGAGATTGGGTGAGCTCAC</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Sequencing primers (M13R): CAGGAAACAGCTATGAC</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pRB21</td>
<td>Bernard Moss</td>
<td>N/A</td>
</tr>
<tr>
<td>pRB21-GFP-vIRD</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Incucyte 2020B</td>
<td>Essen BioScience</td>
<td><a href="https://www.essenbioscience.com/en/products/incucyte/incucyte-s3/">https://www.essenbioscience.com/en/products/incucyte/incucyte-s3/</a></td>
</tr>
<tr>
<td>SnapGene</td>
<td>SnapGene</td>
<td><a href="https://www.snapgene.com/">https://www.snapgene.com/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Vaccinia virus infection and DNA transfection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<p>In this procedure, we will infect BS-C-1 cells with vRB12, followed by transfection of pRB21 containing our gene of interest. BS-C-1 cells are chosen because they form large visible plaques on monolayer infection (<a href="#fig1">Figure 1</a>). This first step will generate heterogeneous virions. Further selection will be necessary to isolate recombinant vaccinia virus for subsequent subcloning and expansion.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1059-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Workflow for recombinant vaccinia virus generation</div>
</figcaption>
</figure>
<ol>
<li>Infection with vRB12
<ol type="a">
<li>Seed BS-C-1 cells at 2.5 <b>×</b> 10<sup>5</sup> cells per well in 1 mL Complete MEM-10 medium on 12-well tissue culture plate.</li>
<li>Culture cells for 16–20 h in humidified incubator at 37<sup>o</sup>C and 5% CO<sub>2</sub>. The cells should be at 90% or higher confluency at the time of infection.</li>
<li>Thaw vRB12 stock virus on ice.</li>
<li>Dilute virus to 0.5 <b>×</b> 10<sup>5</sup> pfu/mL using MEM-2.5 medium.</li>
<li>Aspirate medium from the cell monolayer. Replace it with 0.5 mL diluted virus from 1d.</li>
<li>Incubate at 37°C and 5% CO<sub>2</sub> incubator for 2 h.</li>
</ol>
</li>
<li>DNA transfection
<ol type="a">
<li>Prewarm Opti-MEM and LT1 transfection reagent to 22<sup>o</sup>C.</li>
<li>Prepare transfection master mix containing 100 μL Opti-MEM, 3 μL TransIT®-LT1 Transfection Reagent and 1 μg pRB21-GFP-vIRD.</li>
<li>Incubate the DNA transfection mixture at 22<sup>o</sup>C for 15 min.</li>
<li>Retrieve virus-infected cells from step 1f.</li>
<li>Remove medium from virus-infected BS-C-1 cells.</li>
<li>Wash the infected cells twice by rinsing the cells with 1 mL of 1X PBS.</li>
<li>Add 1 mL MEM-2.5 medium to the cells.</li>
<li>Add the DNA transfection mixture dropwise to the cells.</li>
<li>Gently rock the cells back and forth to evenly mix the transfection mixture with the cells.</li>
<li>Incubate the cells in a 37°C, 5% CO<sub>2</sub> incubator for 24 h.</li>
<li>Replace medium with 1 mL MEM-2.5 medium.</li>
<li>Incubate the cells for 2 days in a 37°C, 5% CO<sub>2</sub> incubator.</li>
</ol>
</li>
<li>Harvest of viral supernatant
<ol type="a">
<li>Prewarm Opti-MEM in 37°C water bath.</li>
<li>Remove 500 μL of culture medium from each well.</li>
<li>Dislodge the infected cells in the remaining culture medium using disposable cell scraper.</li>
<li>Transfer the cell suspension to a 2 mL sterile screw-top microcentrifuge tubes.</li>
<li>Place the cells in in dry ice for 5 min, followed by placing them in a 37°C water bath for 5 min. Vortex for 5 s.</li>
<li>Repeat the freeze/thaw two more times for a total of three freeze/thaw cycles.</li>
<li>Store the viral supernatants at −80<sup>o</sup>C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The viral supernatants will contain cellular debris. It is advisable to remove the debris before proceeding to selection of recombinant virus. This can be achieved by centrifugation for 5 minutes at 650 <b>×</b> <i>g</i> at 22<sup>o</sup>C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Disposable items such as tissue culture plates, pipet tips, Eppendorf tubes that have been exposed to virus or virus-infected materials should be treated with 0.05% diluted bleach solution prior to disposal.</div>
<h3 id="sec2.2">Selection of recombinant virus</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>The procedure here describes the selection of recombinant vaccinia virus based on visual plaque size. A total of three rounds of selection will be needed to ensure isolation of homogeneous population of recombinant viral progenies (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1059-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Workflow for selection of recombinant virus</div>
</figcaption>
</figure>
<ol start="4">
<li>Seed BS-C-1 cells at 5 <b>×</b> 10<sup>5</sup> cells in 2 mL complete MEM-10 medium per well on 6-well plate. Incubate in a 37°C, 5% CO<sub>2</sub> incubator for 24 h.</li>
<li>Replace medium with 1 mL of pre-warmed MEM-2.5 medium per well.</li>
<li>Thaw viral supernatants from step 3 on ice. To each well, dispense 100 μL, 10 μL, 1 μL or 0.1 μL of the thawed viral supernatants to the cell monolayer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Since the titer of the viral supernatant is unknown, the 10-fold serial dilution of the virus ensures that large distinct plaques can be easily identified. If the plaque count is too high, further dilutions can be used.</div>
<ol start="7">
<li>Incubate in a 37°C, 5% CO<sub>2</sub> incubator for 2 h.</li>
<li>Remove medium and replace it with 2 mL pre-warmed MEM-2.5 containing 2.5% methylcellulose.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The methylcellulose is viscous and is included to minimize spreading of viral progenies from the original infected cells. To avoid air bubbles, add the MEM-2.5% methylcellulose at an angle along the edge of the well.</div>
<ol start="9">
<li>Incubate in a 37°C, 5% CO<sub>2</sub> incubator for 48 h.</li>
<li>Dispense 0.5 mL of pre-warmed MEM-2.5 medium to sterile Eppendorf tubes.</li>
<li>Retrieve the plate from step 9 from the incubator. Identify large plaques, which should be visible after 48–72 h of infection. Mark the location of the plaques at the bottom of the plate using a marker pen.</li>
<li>Using 200 μL pipet tip attached to a P200 pipette, carefully scrape the plaque area with the tip while drawing up the content of the plaque.</li>
<li>Transfer the content to Eppendorf tube containing 500 μL MEM-2.5.</li>
<li>Vortex the Eppendorf tube.</li>
<li>Subject the tube to three cycles of freeze/thaw three times as described in step 3.</li>
<li>Save the released virus at −80<sup>o</sup>C freezer until next round of selection.</li>
<li>Repeat steps 4–16 two more times for a total of three rounds of plaque selection. The additional rounds of subcloning ensures purity of the resulting viral stocks.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The methylcellulose helps to prevent spreading of viruses within the well and to allow clear distinct plaques to form. Therefore, avoid disturbing the plate during the 48-hour incubation period. In some cases, longer incubation time of up to 72 hours may be required for clear large plaques to form. We typically collect three to five plaques for each round of selection.</div>
<h3 id="sec2.3">Amplification of recombinant virus</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 weeks</div>
<p>This protocol describes the stepwise amplification and production of recombinant vaccinia virus.</p>
<h4 id="sec2.3.1">6-Well plate amplification</h4>
<ol start="18">
<li>Seed 5 <b>×</b> 10<sup>5</sup> BS-C-1 cells in 2 mL complete MEM-10 per well in a 6-well plate. Incubate at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
<li>Dilute 250 μL of recombinant virus from step 17 with 750 μL MEM-2.5.</li>
<li>Dispense the diluted virus to the BS-C-1 cells.</li>
<li>Incubate at 37°C, 5% CO<sub>2</sub> for 30 min.</li>
<li>Add 1 mL of MEM-2.5 to each well, incubate in 37°C, 5% CO<sub>2</sub> incubator for 48 h.</li>
<li>Remove 1 mL of medium.</li>
<li>Scrape the cells in the remaining 1 mL of medium with disposable cell scraper.</li>
<li>Transfer the cells and supernatant to sterile Eppendorf tube.</li>
<li>Subject the infected cell mixture to three cycles of freeze/thaw. Vortex the tube after each thaw cycle.</li>
<li>Save the viral supernatant in −80<sup>o</sup>C freezer.</li>
</ol>
<h4 id="sec2.3.2">T25 flask amplification</h4>
<ol start="28">
<li>Seed 1.5 <b>×</b> 10<sup>6</sup> of BS-C-1 cells in complete MEM medium in a T25 flask. Incubate at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
<li>Dilute 500 μL of amplified recombinant virus from step 27 with 1.5 mL MEM-2.5.</li>
<li>Dispense the diluted virus to the BS-C-1 cells in T25 flask.</li>
<li>Incubate virus with cells at 37°C, 5% CO<sub>2</sub> for 30 min.</li>
<li>Add 3 mL of MEM-2.5 to the flask.</li>
<li>Incubate at 37°C, 5% CO<sub>2</sub> for 48 h.</li>
<li>Scrape the cells with disposable cell scraper. Transfer the supernatant to 15 mL conical tube.</li>
<li>Spin the cells down at 1800 <b>×</b> <i>g</i> for 5 min. Resuspend the cell pellet in 1 mL MEM-2.5.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Biosafety caps should be used for the rotor buckets during centrifugation to prevent leakage of infected materials.</div>
<ol start="36">
<li>Subject the cell mixture to three cycles of freeze/thaw. Vortex the tube after each thaw cycle.</li>
<li>Save the viral supernatant in −80<sup>o</sup>C freezer.</li>
</ol>
<h3 id="sec2.4">Validation of protein expression by recombinant vaccinia virus</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<div class="note">
<span class="note-title">Note:</span> We typically will validate protein expression after expansion of the virus in T25 flasks. However, validation of protein expression can also be performed at the 6-well stage.</div>
<ol start="38">
<li>Seed 2.5 <b>×</b> 10<sup>5</sup> L929 cells in 1 mL of complete DMEM-10 medium in each well of a 12-well plate. Incubate at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
<li>Add 10 μL of the amplified recombinant virus from step 37 with 1 mL MEM-2.5.</li>
<li>Add the diluted virus to the cells. Incubate at 37°C, 5% CO<sub>2</sub> for 2 h.</li>
<li>Replace medium with 1 mL of complete DMEM-10 medium per well.</li>
<li>Incubate infected cells at 37°C, 5% CO<sub>2</sub> for 18 h.</li>
<li>Wash the infected cells by rinsing the cells twice with 1 mL 1X PBS at 4<sup>o</sup>C.</li>
<li>Add 100 μL RIPA lysis buffer supplemented with 1X Complete protease inhibitor cocktail and 1X phosphatase inhibitor cocktail.</li>
<li>Transfer the cells to Eppendorf tube.</li>
<li>Homogenate the cell lysate by pressing it through an insulin syringe five times.</li>
<li>Boil in 1X LDS Sample buffer for 5 min.</li>
<li>Load the cell lysates on 4%–20% SurePAGE Bis-Tris precast gels.</li>
<li>Transfer the resolved cell lysates onto nitrocellulose membrane.</li>
<li>Perform Western blot using anti-GFP antibody conjugated to HRP. Representative result is shown in <a href="#fig3">Figure 3</a>.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1059-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Confirmation of vIRD expression by Western blot</div>
<p>L929 cells were infected with the indicated viruses for 18 h. Expression of GFP-vIRD was determined by Western blot. Expression of actin was used to confirm equal loading of proteins.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec2.5">Large-scale production of recombinant vaccinia virus stock</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="51">
<li>Seed 1 <b>×</b> 10<sup>7</sup> BS-C-1 cells in complete MEM-10 medium in T175 flask. Incubate at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
<li>Dilute 500 μL of recombinant virus from step 37 with 4.5 mL MEM-2.5.</li>
<li>Add the diluted virus to the cells in the flask. Incubate infected cells at 37°C, 5% CO<sub>2</sub> for 30 min.</li>
<li>Add 25 mL of MEM-2.5 to each flask. Incubate at 37°C, 5% CO<sub>2</sub> for 48 h.</li>
<li>Scrape cells with disposable cell scraper and transfer the culture supernatant to 50 mL conical tube.</li>
<li>Spin down the cells at 1800 <b>×</b> <i>g</i> for 5 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Biosafety caps should be used for the rotor buckets during centrifugation to prevent leakage of infected materials.</div>
<ol start="57">
<li>Resuspend cell pellet in 2 mL MEM-2.5.</li>
<li>Subject the infected cell mixture to three cycles of freeze/thaw. Vortex the tube after each thaw cycle.</li>
<li>Aliquot the viral supernatant into 2 mL screw cap freezing vials.</li>
<li>Save the viral supernatant in −80<sup>o</sup>C freezer.</li>
</ol>
<h3 id="sec2.6">Determination of viral titer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>Before using the recombinant virus to determine the effect of virus-induced cell death, it is important to determine the viral tier using monolayer plaque assay (<a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1059-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Workflow for viral titer determination</div>
</figcaption>
</figure>
<ol start="61">
<li>Seed 5 <b>×</b> 10<sup>5</sup> Vero cells per well in 2 mL complete MEM-10 medium in 6-well plate. Incubate at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
<li>Thaw the virus from step 60 on ice.</li>
<li>Perform serial dilution of the virus stock as follows:
<ol type="a">
<li>Add 10 μL of stock virus solution to 990 μL of MEM-2.5. Mix by gently pipetting the content several times. This is the 1:100 (10<sup>−2</sup>) dilution.</li>
<li>Transfer 20 μL of the diluted virus in step 63a with 180 μL of MEM-2.5. Mix by gently pipetting the content several times. This is the 1:1,000 (10<sup>−3</sup>) dilution.</li>
<li>Prepare the 10<sup>−4</sup>, 10<sup>−5</sup>, 10<sup>−6</sup> and 10<sup>−7</sup> dilutions as in step 63b.</li>
</ol>
</li>
<li>Aspirate the culture medium in the 6-well plate. Replace with 1 mL MEM-2.5 for each well.</li>
<li>Add 100 μL of the serially diluted virus from step 63 to each well, leaving the last well as control without virus. Incubate at 37°C, 5% CO<sub>2</sub> for 2 h.</li>
<li>Remove the medium. Add 2 mL complete MEM to each well. Incubate at 37°C, 5% CO<sub>2</sub> for 48 h.</li>
<li>Wash cells twice with 2 mL 1X PBS.</li>
<li>Add 1 mL 4% paraformaldehyde solution in PBS. Fix the cells for 30 min at 22<sup>o</sup>C.</li>
<li>Aspirate the paraformaldehyde solution. Add 1 mL of crystal violet staining solution to each well. Rock the plate gently to ensure that the solution covers the whole well.</li>
<li>Aspirate the crystal violet solution. Rinse the wells under a running tap to remove the crystal violet solution.</li>
<li>Count the number of plaques in the wells with clear and distinct plaques. A representative image of the clear plaques is shown in <a href="#fig5">Figure 5</a>.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1059-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Representative plaque assay</div>
<p>Note that the plaques in the wells with lowest dilution factors were too numerous and not discrete enough for accurate counting. In this example, only the 10<sup>−6</sup> and 10<sup>−7</sup> wells were counted. The PFUs from the two wells were averaged to obtain the final PFU.</p>
</figcaption>
</figure>
</li>
<li>Calculate the plaque forming unit (PFU) using the formula:</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>P</mi> <mi>F</mi> <mi>U</mi> <mo>=</mo> <mspace></mspace> <mi>N</mi> <mi>u</mi> <mi>m</mi> <mi>b</mi> <mi>e</mi> <mi>r</mi> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>p</mi> <mi>l</mi> <mi>a</mi> <mi>q</mi> <mi>u</mi> <mi>e</mi> <mi>s</mi> <mspace></mspace> <mi>x</mi> <mfrac> <mn>1</mn> <mrow> <mi>d</mi> <mi>i</mi> <mi>l</mi> <mi>u</mi> <mi>t</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> <mspace></mspace> <mi>f</mi> <mi>a</mi> <mi>c</mi> <mi>t</mi> <mi>o</mi> <mi>r</mi> </mrow> </mfrac> <mspace></mspace> <mi>x</mi> <mfrac> <mn>1</mn> <mrow> <mi>v</mi> <mi>o</mi> <mi>l</mi> <mi>u</mi> <mi>m</mi> <mi>e</mi> <mspace></mspace> <mrow> <mo>(</mo> <mi>m</mi> <mi>l</mi> <mo>)</mo> </mrow> </mrow> </mfrac> </mrow> </math></p>
<h3 id="sec2.7">Determination of virus-induced cell death</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This protocol describes the functional characterization of the effect of expression of the necroptosis inhibitor vIRD in vaccinia virus-induced necroptosis (<a href="#fig6">Figure 6</a>). We use the Incucyte live cell imaging system to measure cell death. However, other methods can be used to enumerate cell death in response to infection by the recombinant vaccinia virus.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1059-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Workflow of cell death measurement using live cell imaging</div>
</figcaption>
</figure>
<ol start="73">
<li>Seed 1.25 x 10<sup>5</sup> L929 cells in 1 mL complete DMEM medium per well in 24-well plate. Incubate at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> L929 cells produce autocrine TNF in response to virus infection (<a href="#bib3">Liu et al., 2021</a>). Therefore, there is no need to add exogenous TNF to the infected cells. However, other cells such as mouse embryonic fibroblasts do not produce autocrine TNF. In this situation, recombinant TNF will have to be added to the infected cells to trigger necroptosis.</div>
<ol start="74">
<li>Thaw the recombinant vaccinia virus rVACV-GFP-vIRD on ice. In this example, we will infect the cells with a multiplicity of infection (moi) of 2.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We found that moi of 2 leads to about 30% virus-induced cell death by 24 hours post-infection. This allows us to measure the effect of genes that increase or decrease virus-induced cell death. However, the moi should be determined empirically based on the amount of autocrine TNF produced and the sensitivity to virus-induced necroptosis. If the cell line does not produce autocrine TNF, cell death can be induced with exogenous TNF.</div>
<ol start="75">
<li>To 300 μL DMEM-2.5, add the required volume of rVACV-GFP-vIRD using the formula below:</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>V</mi> <mi>o</mi> <mi>l</mi> <mi>u</mi> <mi>m</mi> <mi>e</mi> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>s</mi> <mi>t</mi> <mi>o</mi> <mi>c</mi> <mi>k</mi> <mspace></mspace> <mi>v</mi> <mi>i</mi> <mi>r</mi> <mi>u</mi> <mi>s</mi> <mspace></mspace> <mi>n</mi> <mi>e</mi> <mi>e</mi> <mi>d</mi> <mi>e</mi> <mi>d</mi> <mspace></mspace> <mrow> <mo>(</mo> <mi>m</mi> <mi>l</mi> <mo>)</mo> </mrow> <mo>=</mo> <mfrac> <mrow> <mi>M</mi> <mi>O</mi> <mi>I</mi> <mspace></mspace> <mi>x</mi> <mspace></mspace> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mspace></mspace> <mi>n</mi> <mi>u</mi> <mi>m</mi> <mi>b</mi> <mi>e</mi> <mi>r</mi> </mrow> <mrow> <mi>P</mi> <mi>F</mi> <mi>U</mi> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>s</mi> <mi>t</mi> <mi>o</mi> <mi>c</mi> <mi>k</mi> <mspace></mspace> <mrow> <mo>(</mo> <mi>m</mi> <mi>l</mi> <mo>)</mo> </mrow> </mrow> </mfrac> </mrow> </math></p>
<ol start="76">
<li>Replace the culture medium with DMEM-2.5 containing the virus from step 75. Incubate at 37°C, 5% CO<sub>2</sub> for 2 h.</li>
<li>Remove the medium containing the virus. Replace it with 500 μL complete DMEM containing 100 nM Incucyte® Cytotox Red Dye.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Incucyte system can measure green and red fluorescence. Since the infected cells will express GFP, Cytotox Red is used. Otherwise, cell death can be measured using green fluorescent dyes such as Incucyte® Cytotox Green or Yoyo-1.</div>
<ol start="78">
<li>Scan the plate using the Incucyte live cell imaging system every hour for up to 24 h. Representative images of GFP expression by the virus and cell death by Cytotox Red is shown in <a href="#fig7">Figure 7</a>.
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1059-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Representative Incucyte images of L929 cells infected with the indicated recombinant viruses</div>
<p>Cells were infected with the indicated virus for 24 h in the presence of cytotox red. Infected cells are marked by green fluorescence. Note that cells infected with rVACV-GFP were Cytotox Red, which indicates cell death. By contrast, rVACV-GFP-vIRD infected cells had little cytotox red signals, which is consistent with virus-induced RIPK3 degradation and resistance to autocrine TNF-induced necroptosis.</p>
</figcaption>
</figure>
</li>
<li>Analyze the results using the Incucyte software version 2020B.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cell death is calculated by dividing the red fluorescent area (dying cells) over the green fluorescent area (infected cells) using the Incucyte software. If non-fluorescent viruses are used, the phase area can be used instead of the green fluorescent area. Alternatively, cell death can be calculated using the number of fluorescent objects. Other methods of cell death measurement such as flow cytometry can also be used for cell death measurement.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The procedures described in this protocol is expected to successfully generate recombinant vaccinia virus that express the gene-of-interest, an example of which is shown in <a href="#fig3">Figure 3</a> for rVACV-GFP-vIRD. By repairing the gene <i>vp37</i>, the recombinant virus is expected to generate large plaque on cell monolayer. An example of this is shown in <a href="#fig5">Figure 5</a>. Finally, expression of the cell death inhibitor vIRD causes RIPK3 degradation and is expected to inhibit necroptosis. By tracking infected cells with GFP and cell death with Cytotox Red, we show that TNF-induced necroptosis was inhibited by vIRD (<a href="#fig7">Figure 7</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Vaccinia virus is a cytolytic virus that will ultimately cause cell death. In our experience, we typically observe about 30% virus-induced cell death in L929 cells by empty recombinant vaccinia virus (<a href="#bib3">Liu et al., 2021</a>). The dosage of the virus used will affect the kinetics and extent of virus-induced cell death. Thus, investigators who plan to use this approach to assess the function of pathogen-encoded cell death inhibitors will need to consider these factors in their experimental design. Empty recombinant virus that does not encode the cell death regulator should always be used as control to compare the effect of any cell death regulator expressed by the recombinant virus.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Poor protein expression by the recombinant virus (step 50).</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Poor protein expression can result from contamination of virus that does not contain the gene-of-interest. In this case, additional rounds of selection can be performed to ensure homogeneous population of recombinant virus. Althernatively, other methods of generation of recombinant vaccinia virus that use different promoters can be used (<a href="#bib2">Cotter et al., 2017</a>). For example, recombinant vaccinia virus using recombination at the thymidine kinase locus is a widely popular option.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>Too many plaques formed in the plaque assay (step 71).</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Further dilution of the virus stocks should be done to achieve more accurate plaque count and determination of the viral titer.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>Too few plaques formed during selection of recombinant virus (step 71).</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Optimize the incubation time during vRB12 infection or transfection efficiency (e.g., changing the amount of pRB21 used in the transfection).</p>
<h3 id="sec5.7">Problem 4</h3>
<p>Undercounting cell death by Incucyte imaging (step 79).</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Image segmentation using the Incucyte often results in undercounting of cell death. To circumvent this problem, fluorescent dyes that label the cell such as the Nuclight Rapid Red Dye from Essen Bio can be used to provide a more accurate cell count. Another alternative will be to independently validate the results using a different cell death measurement method such as flow cytometric staining with Annexin V and Propidium iodide (<a href="#bib5">Wallberg et al., 2016</a>).</p>
<h3 id="sec5.9">Problem 5</h3>
<p>Excessive and rapid virus-induced cell death (step 79).</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Excessive cell death is often seen when cells are infected with high moi (&gt; 10) or if there is cell debris in the viral supernatant. To reduce virus-induced cell death, lower moi can be used. Centrifugation of the viral supernatant before use will also help to reduce cell death caused by cellular debris contamination.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Blasco, R. and Moss, B. (1995). Selection of recombinant vaccinia viruses on the basis of plaque formation. Gene <i>158</i>, 157-162. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00577-3/sref1">View at publisher</a></p>
<p id="bib2">Cotter, C.A., Earl, P.L., Wyatt, L.S., and Moss, B. (2017). Preparation of cell cultures and vaccinia virus stocks. Curr. Protoc. Protein Sci. <i>89</i>, 5.12.1-5.12.18. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00577-3/sref2">View at publisher</a></p>
<p id="bib3">Liu, Z., Nailwal, H., Rector, J., Rahman, M.M., Sam, R., Mcfadden, G., and Chan, F.K. (2021). A class of viral inducer of degradation of the necroptosis adaptor RIPK3 regulates virus-induced inflammation. Immunity <i>54</i>, 247-258.e7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00577-3/sref3">View at publisher</a></p>
<p id="bib4">Sutherst, R.W., Wagland, B.M., and Roberts, J.A. (1978). The effect of density on the survival of Boophilus microplus on previously unexposed cattle. Int. J. Parasitol. <i>8</i>, 321-324. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00577-3/sref4">View at publisher</a></p>
<p id="bib5">Wallberg, F., Tenev, T., and Meier, P. (2016). Analysis of apoptosis and necroptosis by fluorescence-activated cell sorting. Cold Spring Harb. Protoc. <i>2016</i>. pdb.prot087387. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00577-3/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Francis Chan (<a href="mailto:franciskaming.chan@duke.edu">franciskaming.chan@duke.edu</a>).</p>
<h4>Materials availability</h4>
<p>Recombinant viruses and plasmids generated from this study are available from the lead contact upon completion of Materials Transfer Agreement.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze any dataset or code.</p>
<h3>Acknowledgments</h3>
<p>This work is supported by <a href="https://doi.org/10.13039/100000060">National Institute of Allergy and Infectious Diseases</a> grant AI 148302. We thank Grant McFadden for reagents, advice, and technical assistance.</p>
<h3>Author contributions</h3>
<p>Z.L., K.K., and F.K.-M.C. wrote the manuscript. Z.L. provided the data.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>