<article>
<section>
<h1 id="header">Minimally invasive protocols and quantification for microglia-mediated perineuronal net disassembly in mouse brain</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Alessandro Venturino<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Sandra Siegert<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Science and Technology (IST) Austria, Am Campus 1, 3400 Klosterneuburg, Austria</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:alessandro.venturino@ist.ac.at">alessandro.venturino@ist.ac.at</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sandra.siegert@ist.ac.at">sandra.siegert@ist.ac.at</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101012">10.1016/j.xpro.2021.101012</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Enzymatic digestion of the extracellular matrix with chondroitinase-ABC reinstates juvenile-like plasticity in the adult cortex as it also disassembles the perineuronal nets (PNNs). The disadvantage of the enzyme is that it must be applied intracerebrally and it degrades the ECM for several weeks. Here, we provide two minimally invasive and transient protocols for microglia-enabled PNN disassembly in mouse cortex: repeated treatment with ketamine-xylazine-acepromazine (KXA) anesthesia and 60-Hz light entrainment. We also discuss how to analyze PNNs within microglial endosomes-lysosomes.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/J.CELREP.2021.109313">Venturino et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>PNN disassembly with repeated ketamine-mediated anesthesia in mice</li>
<li>60 Hz light entrainment as an alternative approach</li>
<li>Strategy to quantify density of PNN-coated cells</li>
<li>Determine PNN material enriched within microglial lysosomes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1234-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>We outline two protocols for microglia-enabled PNN disassembly in the primary visual cortex, which we have applied in C57BL/6J mice of both sexes at the age of 8–12 weeks.</p>
<h3 id="sec1.1">Protocol 1: KXA (ketamine-xylazine-acepromazine)</h3>
<p>The first protocol (option 1) is based on repeated exposure to ketamine-mediated anesthesia. Ketamine is an anesthetic that amongst others blocks the NMDA receptor (<a href="#bib25">Zorumski et al., 2016</a>) and preferentially acts on parvalbumin-positive neurons (<a href="#bib6">Behrens et al., 2007</a>).</p>
<p>In our lab, we have applied this protocol with the following parameters: (1) C57BL/6J (8–12 weeks of age, both sexes), 1×, 2×, 3×, 6× KXA for primary visual cortex (V1) and primary somatosensory cortex (S1) in a repetition-dependent manner. We observed for both, V1 and S1, approximately 80% and up to 90% PNN loss with 3× and 6× KXA, respectively (<a href="#bib22">Venturino et al., 2021</a>); (2) C57BL/6N (8–12 weeks of age, both sexes), 3× KXA for V1 and S1. C57BL/6N mice showed 80% reduction of PNN density after 3× KXA treatment, in line with the result obtained with C57BL/6J mice (<a href="#bib22">Venturino et al., 2021</a>).</p>
<div class="note">
<span class="note-title">Note:</span> Common mouse models for labeling microglia like CX<sub>3</sub>CR<sub>1</sub><sup>CRE-ERT2</sup> or CX<sub>3</sub>CR<sub>1</sub><sup>GFP</sup> are knock-in models that will contain a CX<sub>3</sub>CR<sub>1</sub> knock-out allele, when used as heterozygous animals, or CX<sub>3</sub>CR<sub>1</sub> fully knocked out, when homozygous. This might alter the KXA-mediated microglia response as <i>e.g.</i> CX<sub>3</sub>CR<sub>1</sub> is involved in synapse pruning and injury responses (<a href="#bib16">Paolicelli et al., 2011</a>; <a href="#bib8">Freria et al., 2017</a>)<i>.</i>
</div>
<div class="note">
<span class="note-title">Note:</span> Our animal protocol has so far only covered the combination of ketamine-xylazine-acepromazine (KXA) but the procedure also works with 3× KX in the V1 (<a href="#bib22">Venturino et al., 2021</a>).</div>
<div class="note">
<span class="note-title">Note:</span> 20% of PNN recovers within 3 days after the last 3× KXA injection and reaches full coverage by 14 days for S1 and V1 of C57BL/6J mice (<a href="#bib22">Venturino et al., 2021</a>). The recovery kinetics may differ between the mouse model and strain background.</div>
<h3 id="sec1.2">Protocol 2: 60-Hz light entrainment</h3>
<p>The second protocol (option 2) focuses on the visual stimulation protocol and uses 60 Hz light flickering entrainment. Here, the pulsed light mimics ketamine-induced gamma oscillation, which has been previously described in the mammalian brain (<a href="#bib2">Akeju et al., 2016</a>). We applied this protocol in C57BL/6J mice (8–12 weeks, both sexes), 2 h daily for 5 days for V1. We did not observe PNN loss when we applied the same paradigm with 8 or 40 Hz pulsed light compared to controls (treated with non-flickering light with the same light intensity). As a note, it has been shown that gamma oscillations induced by 40 Hz light stimulation can propagate to other brain regions such as hippocampus (CA1) (<a href="#bib11">Iaccarino et al., 2016</a>) and frontal cortex (<a href="#bib12">Jones et al., 2019</a>), where this stimulation paradigm triggers microglia-mediated engulfment of amyloid-β plaques in murine models of Alzheimer disease (<a href="#bib11">Iaccarino et al., 2016</a>; <a href="#bib15">Martorell et al., 2019</a>).</p>
<p>For both protocols, we identified microglia as the main cell type for promoting PNN degradation as described in <a href="#bib22">Venturino et al. (2021)</a>. Both protocols failed to dismantle PNNs in microglia-depleted mice, where approximately 80% of the microglial cells were depleted with PLX 5622-enriched food administered 1.5 weeks prior and during both treatment regimens (<a href="#bib22">Venturino et al., 2021</a>). The condition for protocol 1 were C57BL/6J (8–12 weeks of age, both sexes), 3× KXA for V1 and S1) and protocol 2, C57BL/6J (8–12 weeks of age, both sexes), 2 h daily for 5 days in V1.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Before starting with the experiments, it is important for the users to have the approval of the animal procedures from the ethical committee of their institute and all relevant authorities.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rat α-CD68 (1:250)</td>
<td>Bio-Rad AbD Serotec Limited</td>
<td>Cat#MCA1957, clone FA-11, Lot 1807, RRID:AB_322219</td>
</tr>
<tr>
<td>Goat α-Iba1 (1:200)</td>
<td>Abcam</td>
<td>Cat#ab5076, Lot FR3288145-1, RRID:AB_2224402</td>
</tr>
<tr>
<td>Rabbit a-Iba1 (1:750)</td>
<td>GeneTex</td>
<td>Cat#GTX100042, Lot 41556, RRID:AB_1240434</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>
<i>Wisteria floribunda</i> lectin – fluorescein-labeled (1:200)</td>
<td>Szabo-Scandic</td>
<td>Cat#VECFL-1351, Lot ZE0611</td>
</tr>
<tr>
<td>
<i>Wisteria floribunda</i> lectin – biotinylated (1:200)</td>
<td>Szabo-Scandic</td>
<td>Cat#VECB-1355, Lot ZE0424</td>
</tr>
<tr>
<td>Hoechst 33342 (1:10.000)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#H3570</td>
</tr>
<tr>
<td>Isoflurane</td>
<td>Zoetis</td>
<td>Cat#6089373</td>
</tr>
<tr>
<td>Ketamine</td>
<td>MSD Animal Health</td>
<td>Cat#A137A01</td>
</tr>
<tr>
<td>Xylazine</td>
<td>AniMedica</td>
<td>Cat#7630517</td>
</tr>
<tr>
<td>Acepromazine</td>
<td>VANA GmbH</td>
<td>Cat#18F211</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: C57BL/6J</td>
<td>The Jackson Laboratory</td>
<td>Cat#000664</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>MATLAB R2019b</td>
<td>MathWorks</td>
<td>RRID:SCR_001622</td>
</tr>
<tr>
<td>Imaris</td>
<td>Bitplane Imaris</td>
<td>9.3.1</td>
</tr>
<tr>
<td>Arduino IDE for PC or Mac</td>
<td><a href="https://www.arduino.cc/en/main/software">https://www.arduino.cc/en/main/software</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Arduino UNO</td>
<td>RS Components</td>
<td>Cat#769-7409</td>
</tr>
<tr>
<td>Breadboard</td>
<td>RS Components</td>
<td>Cat#102-9147</td>
</tr>
<tr>
<td>Transistor</td>
<td>RS Components</td>
<td>Cat#485-7721</td>
</tr>
<tr>
<td>200 Ω resistor</td>
<td>RS Components</td>
<td>Cat#174-3016</td>
</tr>
<tr>
<td>Jumper wires</td>
<td>RS Components</td>
<td>Cat#634-8651</td>
</tr>
<tr>
<td>White LED strip</td>
<td>RS Components</td>
<td>Cat#153-3638</td>
</tr>
<tr>
<td>Box 62 l with lid (black)</td>
<td>OBI</td>
<td>Cat#4969176</td>
</tr>
<tr>
<td>Soldering set</td>
<td>OBI</td>
<td>Cat#4162129</td>
</tr>
<tr>
<td>Insulating tape</td>
<td>OBI</td>
<td>Cat#7613235</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>The stock solution of ketamine (100 mg/mL), xylazine (20 mg/mL) and acepromazine (10 mg/mL) are obtained from the company. The drugs are in saline solution in “ready-to-use” formulation. The drugs have to be stored between 2°C and 8°C. The expiration date indicated on the vial should not be exceeded.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Option 1. PNN disassembly by repeated KXA-induced anesthesia</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 weeks</div>
<p>Here, we describe first how to prepare a ready-to-use KXA solution, followed by performing the intraperitoneal injection, and finally we outline how to check the level of anesthesia and how to maintain the animals during the anesthetized state.</p>
<p>The drug dosages used for mice are: (1) ketamine 100 mg/kg; (2) xylazine 10 mg/kg; (3) acepromazine 3 mg/kg.</p>
<div class="note">
<span class="note-title">Note:</span> The effect of PNN loss also occurs with just ketamine and xylazine. On the other hand, acepromazine reduces the death rate and improves the surgical plane (<a href="#bib4">Arras et al., 2001</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ketamine at this dosage must be combined with xylazine to prevent uncontrolled muscular contraction.</div>
<ol>
<li>Prepare the ketamine-xylazine-acepromazine (KXA) solution.
<ol type="a">
<li>Combine 2 mL of ketamine, 1 mL of xylazine, and 0.6 mL of acepromazine in 3 mL of physiological saline solution containing 0.9% (w/v) NaCl.</li>
<li>Vortex for 30 s.</li>
<li>The KXA solution should look yellow without any precipitates and can be stored for up to 2 weeks at 4°C. <a href="#sec7.1">Troubleshooting 1</a>
</li>
</ol>
</li>
<li>Weigh the mice on a fine scale.</li>
<li>Calculate the injection dosage for each mouse. A mouse receives 33 μL/10 g weight e.g., a mouse weighing 30 g receives 99 μL of the KXA solution (<a href="#bib9">Harkness and Wagner, 1995</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The drug dosage must be adjusted for each mouse. Even if mice are the same sex or come from the same litter, there may be fluctuations in the body weight.</div>
<ol start="4">
<li>Prepare a 1 mL syringe equipped with a 26G needle with the appropriate amount of KXA solution.</li>
</ol>
<ol start="5">
<li>Inject intraperitoneally. A schematic of the repeated KXA treatment protocol can be found in <a href="#fig1">Figure 1</a>A)
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1234-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. PNN disassembly by repeated KXA-induced anesthesia</div>
<p>(A) Experimental timeline for repeated KXA application including the expected PNN loss and recovery rate after KXA exposure. KXA, ketamine-xylazine-acepromazine. PNN, perineuronal net. S1, primary somatosensory cortex. V1, primary visual cortex.</p>
<p>(B) Mouse receiving injection. Left, restraining position for <i>intraperitoneal</i> (<i>i.p.</i>) injection. Arrow, position of the tail between the ring and the pinky of the operator. Right, the injection procedure, in which the needle is inserted with a 45° angle into the lower quadrant of the abdomen.</p>
</figcaption>
</figure>
<ol type="a">
<li>Grab the mouse at the neck and turn it around (<a href="#fig1">Figure 1</a>B).</li>
<li>Insert the needle at 45° into the peritoneal cavity (<a href="#fig1">Figure 1</a>B).
<div class="critical">
<span class="critical-title">Critical:</span> Do not inject on the <i>linea alba</i>. This might cause lesions to the bladder.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure that the solution does not leak out of the mouse.</div>
<div class="note">
<span class="note-title">Note:</span> After the injection, no solution should accumulate and be visible under the skin.</div>
</li>
<li>Turn the mouse back, head down, and gently release it back into its home cage.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Tilting the mouse head down helps push the internal organs towards the head, which prevents damage from the injection.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The injection must be performed intraperitoneally with the needle inserted directly into the peritoneal cavity. In our experience and with the anesthetic dosage used in this protocol, misplacement will cause inefficient anesthesia induction. As a note, it has been shown that ketamine anesthesia can be achieved with subcutaneous administration in mice with double the concentration used here (<a href="#bib14">Levin-Arama et al., 2016</a>).</div>
<ol start="6">
<li>Monitor the state of anesthesia.
<ol type="a">
<li>After the injection, the mouse will undergo a hyperexcitation phase for the first 1–2 min marked by increased running activity. Then, the mouse will slowly become unconscious, and the anesthesia will begin. It might take 5–10 min to reach the induction of the surgical plane, which will last for approximately 40 min. <a href="#sec7.3">Troubleshooting 2</a>
</li>
<li>Apply eye ointment to prevent dehydration of the sclera.</li>
<li>Confirm the anesthesia by checking the following parameters (<a href="#bib21">Tsukamoto et al., 2015</a>):
<ol type="i">
<li>Absence of the toe pinch reflex around 10 min after induction.</li>
<li>Decrease in respiratory frequency.</li>
<li>No responses to noxious stimuli.</li>
<li>Flaccid paralysis.</li>
<li>Absence of whisker movement.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To prevent hypothermia, the mouse should be kept at 37°C with a heating pad or a red lamp for the entire time (induction, deep anesthesia and recovery phase).</div>
<div class="note">
<span class="note-title">Note:</span> Regularly check temperature with a thermometer and adjust the heating device accordingly.</div>
<ol start="7">
<li>After complete recovery, return the mouse to the animal facility with food and water <i>ad libitum.</i>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If many animals are housed in the same cage, make sure that all animals are awake, as they otherwise become aggressive to the ones that are still waking up.</div>
<div class="note">
<span class="note-title">Note:</span> To avoid potential impact of circadian rhythm on brain activity (<a href="#bib10">Hor et al., 2019</a>), we always injected the animals at the same time (in our case 9 a.m.) and collected the tissue 4 h after the last treatment.</div>
<ol start="8">
<li>Repeat steps 1–7 every 2–3 days at least two times (up to 6 times) to reach the desired level of reduction in the number of PNN-coated cells (<a href="#fig1">Figure 1</a>A). Expected results for 6× KXA treatment can be found in <a href="#fig2">Figures 2</a>A and 2B.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1234-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Expected results for PNN loss and recovery with repeated KXA-induced anesthesia</div>
<p>(A) Wisteria floribunda agglutinin (WFA)-labeled parasagittal brain sections after 6× saline (left), and 6× KXA treatment (right). Scale bar: 1000 μm and 800 μm, respectively. Color intensity reflects PNN density with blue and white low and high density, respectively. CB, cerebellum. CP, caudato putamen. FR, frontal cortex. IC, inferior colliculus. M1, primary motor cortex. PIR, piriform cortex. S1, primary somatosensory cortex. V1, primary visual cortex.</p>
<p>(B and C) PNN density in S1. Absolute number of PNN-coated cells ± SEM after repeated KXA treatment (B) and after 3× KXA withdrawal (C). Triangle, female. Circle, male. Linear regression model with selected post-hoc comparison. ∗∗p &lt; 0.01. ∗∗∗p &lt; 0.001, <sup>ns</sup>p &gt; 0.05. SEM: standard error of the mean.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We typically performed our analyses 4 h after the last injection.</div>
<div class="note">
<span class="note-title">Note:</span> 20% of PNN recovers within 3 days after the last 3× KXA injection and reaches full coverage by 14 days for S1 (<a href="#fig2">Figure 2</a>C) and V1 of C57BL/6J mice (<a href="#bib22">Venturino et al., 2021</a>).</div>
<div class="note">
<span class="note-title">Note:</span> PNN recovery kinetics may differ between the mouse model and strain background.</div>
<div class="note">
<span class="note-title">Note:</span> We observed a significant reduction in the number of PNNs in S1 and V1 after repeated KXA anesthesia (<a href="#fig2">Figure 2</a>A). The effect of this treatment may not be the same in all brain regions because ketamine does not act equally throughout the brain (<a href="#bib3">Alda et al., 1994</a>). For example, we found PNN removal in the hippocampus only after 6× KXA (<a href="#bib22">Venturino et al., 2021</a>).</div>
<h3 id="sec3.2">Option 2. PNN disassembly by 60-Hz light entrainment</h3>
<p>This section describes first how to build and program a 60 Hz light stimulation device, and then how to verify the stimulation frequency. Steps 9–17 outline the building of the hardware, steps 18–24 the software.</p>
<h3 id="sec3.3">Part 1. Build light stimulation device for PNN disassembly with 60 Hz</h3>
<div class="timing">
<span class="timing-title">Timing:</span> up to 1 h</div>
<p>An overview of the circuit and the components needed for the assembly of the light stimulation device can be found in <a href="#fig3">Figures 3</a>A and 3B.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1234-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Construction of the light stimulation device</div>
<p>(A) Schematic of the circuit designed with the software Fritzing.</p>
<p>(B) Overview of the necessary tools.</p>
<p>(C–I) Step-by-step constructions of building the electric circuit. (C) Transistor is inserted on the breadboard. White arrow, ground pin. (D) Electric connection built between the transistor ground (blue arrow) and the breadboard ground (orange arrow). (E) The resistor is connected with the left transistor pin (blue arrow) to the breadboard ground (orange arrow). (F) The orange cable connects the Arduino ground port (orange arrow) to the breadboard ground pin (blue arrow). (G) The yellow cable connects the Arduino digital port 5 (orange arrow) to the left transistor pin (blue arrow). (H) The red LED voltage cable is connected to the Arduino Vin port (red arrow). The grey jumper wire to the transistor exit pin (blue arrow). (I) Soldering the red voltage wire from the LED strip to a jumper wire.</p>
<p>(J) Screenshot of Arduino IDE software user-interface for the board programming.</p>
<p>(K and L) Code “check” and “upload” Arduino IDE icons, respectively.</p>
<p>(M) Oscilloscope reading of the Arduino digital port 5 output showing 60 Hz square wave function.</p>
<p>(N) Adjust a mouse cage to the center of the stimulation box for equal illumination.</p>
</figcaption>
</figure>
<ol start="9">
<li>Stick the LED strip horizontally along the inner walls of a black plastic box (60 × 40 × 32 cm, 62 l volume). The LED strip goes round the box in 6 layers with 4 cm space in between.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Add the LED stripes in a way from top to bottom to allow equal illumination.</div>
<ol start="10">
<li>The circuit is built as outlined in <a href="#fig3">Figure 3</a>A.</li>
<li>Take the breadboard and insert the transistor. The ground pin of the transistor is on the right (<a href="#fig3">Figure 3</a>C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The ground pin of the transistor is often indicated on the chip with the letter “G”.</div>
<ol start="12">
<li>Connect the ground of the breadboard (blue symbol “−”) with the ground of the transistor (<a href="#fig3">Figure 3</a>D).</li>
<li>Put a 200 Ω resistor between the ground of the breadboard and the left pin of the transistor (<a href="#fig3">Figure 3</a>E).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A resistor is critical for the functionality of the transistor.</div>
<ol start="14">
<li>Connect one ground port of the Arduino board (indicated with the letters GND) to the ground of the breadboard (blue symbol “−”) (<a href="#fig3">Figure 3</a>F).</li>
<li>Connect the digital port 5 of the Arduino board to the left pin of the transistor (<a href="#fig3">Figure 3</a>G).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The resistor should be in front of the cable that goes to the left pin of the transistor.</div>
<div class="note">
<span class="note-title">Note:</span> The number of the Arduino digital port depends on the programming code. We used digital port 5 in our code to send the signal to the LEDs.</div>
<ol start="16">
<li>Connect the voltage wires from the LED strip as follows:
<ol type="a">
<li>the red cable is connected to the Arduino board at the Vin (voltage in) port, and</li>
<li>the black cable to the middle pin of the transistor (<a href="#fig3">Figure 3</a>H).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the cable is a fine-stranded conductor, solder a jumper wire on the cable to allow a stable connection with the transistor on the breadboard (<a href="#fig3">Figure 3</a>I).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Insulate the solder joint with electrical tape.</div>
<ol start="17">
<li>The system is connected to a 12 V power supply via the jack plug of the Arduino.</li>
<li>Download and install the Arduino IDE for PC or Mac from this link: <a href="https://www.arduino.cc/en/main/software">https://www.arduino.cc/en/main/software</a>
</li>
<li>Connect the Arduino board to the laptop with a USB cable type A/B.</li>
<li>Open the Arduino IDE (<a href="#fig3">Figure 3</a>J) and select the Arduino board in <i>Tools &gt; Board &gt; Arduino/Genuino UNO.</i>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure that the correct Arduino board is selected.</div>
<ol start="21">
<li>Copy and paste the following codes:
<ol type="a">
<li>for the 60 Hz flickering in the Arduino IDE text editor:
<div class="textbox">
<p><code>int ledPin = 5;//digital input from the board</code></p>
<p><code>void setup ()</code></p>
<p><code>{</code></p>
<p><code>  pinMode(ledPin, OUTPUT);</code></p>
<p><code>}</code></p>
<p><code>void loop()</code></p>
<p><code>{</code></p>
<p><code>  digitalWrite(ledPin, HIGH);//light on, no fading</code></p>
<p><code>  delay(8.3);//delay in ms</code></p>
<p><code>  digitalWrite(ledPin, LOW);//light off, no fading</code></p>
<p><code>  delay(8.3);//delay in ms</code></p>
<p><code>}</code></p>
</div>
</li>
<li>for the control with the same light intensity without 60 Hz flickering.</li>
</ol>
</li>
</ol>
<div class="textbox">
<p><code>int ledPin = 5;//digital input from the board</code></p>
<p><code>void setup ()</code></p>
<p><code>{</code></p>
<p><code>  pinMode(ledPin, OUTPUT);</code></p>
<p><code>}</code></p>
<p><code>void loop()</code></p>
<p><code>{</code></p>
<p><code>  digitalWrite(ledPin, HIGH);//keep light on</code></p>
<p><code>}</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> In this script, we use digital port 5 to send the signal to the LEDs. If another digital port is used, the script has to be modified accordingly.</div>
<ol start="22">
<li>Verify that the script is functional by clicking on the “check” icon (<a href="#fig3">Figure 3</a>K).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the script doesn’t work the Arduino IDE will show a red error message reporting in which line of the code the problem is. <a href="#sec7.5">Troubleshooting 3</a>
</div>
<ol start="23">
<li>Select the output port for uploading the code in <i>Tools &gt; Port.</i> In our example, the port is COM5.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure that the port number corresponds to your Arduino.</div>
<ol start="24">
<li>Upload the code from step 21 by clicking on the “upload” icon (<a href="#fig3">Figure 3</a>L).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Arduino board memorizes previously uploaded code. Each time the board is powered up it will execute the code automatically.</div>
<ol start="25">
<li>Confirm with an oscilloscope that the frequency (60 Hz) and the shape (square wave) of the stimulation is correct (<a href="#fig3">Figure 3</a>M).</li>
<li>Take a mouse cage, adjust it to the center of the stimulation box, and mark the position with a tape (<a href="#fig3">Figure 3</a>N).</li>
</ol>
<h3 id="sec3.4">Part 2. PNN disassembly with 60 Hz light entrainment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<p>This section outlines the light stimulation protocol for inducing PNN disassembly with 60 Hz light entrainment with the previously built device. An overview of the 60 Hz light stimulation protocol and the expected results can be found in <a href="#fig4">Figure 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1234-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. PNN disassembly by 60-Hz light entrainment with expected PNN loss</div>
<p>(A) Experimental timeline for the 60 Hz light entrainment. V1, primary visual cortex.</p>
<p>(B) Wisteria floribunda agglutinin (WFA)-labeled coronal brain sections of mice treated for 5 days with light control (no flickering, left) or 60 Hz (right). HP, hippocampus. Scale bar, 200 μm. Zoom-in, 50 μm.</p>
<p>(C) Percentage of change in mean PNN-coated cells ± SEM in V1. Triangle, females. Circles, males. Two-sample t test. ∗∗∗p &lt; 0.001. SEM, standard error of the mean.</p>
</figcaption>
</figure>
<ol start="27">
<li>Acclimatize animals after they have arrived from the animal facility for 1 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To avoid potential impact of circadian rhythm on brain activity (<a href="#bib10">Hor et al., 2019</a>), we always stimulated the animals at the same time (in our case from 9 to 11 a.m.) and collected the tissue four hours after the last stimulation.</div>
<ol start="28">
<li>Remove all enrichment from the cage before stimulation to prevent the mouse from hiding.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The animals should always have unlimited access to food and water.</div>
<ol start="29">
<li>Position the animal cage in the marked position in the center of the box.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Up to 4 animals can be simultaneously stimulated.</div>
<ol start="30">
<li>Start the Arduino software as described in the previous section in steps 21–24 and select the stimulation protocol for either 60 Hz or control non-flickering light but with the same light intensity.</li>
<li>Expose the mice to light stimulation for 2 h per day for 5 consecutive days (<a href="#fig4">Figure 4</a>A).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During the stimulation the box lid was closed.</div>
<div class="note">
<span class="note-title">Note:</span> This test paradigm causes approximately 40% PNN reduction in V1 (<a href="#fig4">Figures 4</a>B and 4C). We have no insights into whether prolonged treatment will further reduce PNN and how fast the PNN recovers. According to the recovery rate after 3× KXA administration, the PNN recovers by 20% within 3 days after KXA withdrawal (<a href="#fig2">Figure 2</a>C).</div>
<div class="note">
<span class="note-title">Note:</span> We typically perfused the animal 4 hours after the last stimulation and performed image analysis. For detailed protocols see <a href="#bib22">Venturino et al. (2021)</a>. Alternatively, this protocol could be explored for <i>in vivo</i> electrophysiological recordings in awake mice, behavioral studies, and <i>in vivo</i> imaging experiments to name a few.</div>
<div class="note">
<span class="note-title">Note:</span> The stimulation device generates electrical noise, therefore further modifications, such as isolating the electrical connections with copper mesh, are required to enable <i>in-vivo</i> electrophysiology during the 60 Hz light stimulation. <a href="#sec7.7">Troubleshooting 4</a>
</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Both repeated ketamine anesthesia and 60 Hz light entrainment protocols led to a significant reduction in the number of PNN-coated cells (<a href="#fig2">Figures 2</a>B and <a href="#fig4">4</a>C). KXA treatment is effective in V1 and S1. After 2×, 3×, and 6× treatment with KXA, we found a 50%, 80%, and 90% reduction in PNN-coated cells in the S1, respectively (<a href="#fig2">Figure 2</a>B). For the 60 Hz light entrainment, we found a 40% reduction in PNN-coated cells in the V1 (<a href="#fig4">Figure 4</a>C), which corresponds approximately to 2× KXA treatment.</p>
<p>Microglial cells in these cortical areas will contain PNN fragments within their lysosomes (<a href="#fig6">Figures 6</a>G and 6H). For further details refer to <a href="#bib22">Venturino et al. (2021)</a>.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Here, we describe first how to quantify the PNN density in a semi-automatic way, and second how to analyze the amount of PNN material within microglial lysosomes in 3D using Imaris software. The images are 40× z-stack confocal images immunostained for PNNs with WFA (<i>Wisteria floribunda</i> lectin), for microglia with Iba1 (ionized calcium-binding adaptor molecule 1), and for lysosomes/endosomes with CD68 antibodies. For further details about tissue preparation, histology, immunostaining, and confocal imaging settings see <a href="#bib22">Venturino et al. (2021)</a>.</p>
<h3 id="sec5.1">PNN density quantification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> up to 30 min</div>
<p>In the following example, we use data from the primary visual cortex (V1) of a female C57BL/6J mouse exposed to control light.</p>
<ol>
<li>Software requirements:
<ol type="a">
<li>Get access to Imaris and obtain a license: <a href="https://imaris.oxinst.com/packages">https://imaris.oxinst.com/packages</a>.</li>
<li>Get access to MATLAB and obtain a license here: <a href="https://www.mathworks.com/products/get-matlab.html">https://www.mathworks.com/products/get-matlab.html</a>.</li>
<li>Activate the MATLAB plugins for Imaris following this tutorial: <a href="https://imaris.oxinst.com/learning/view/article/enabling-imagej-fiji-and-matlab-plugins-in-imaris.%20">https://imaris.oxinst.com/learning/view/article/enabling-imagej-fiji-and-matlab-plugins-in-imaris.</a>
</li>
<li>To perform the described analysis, the following functions and plugins are required:
<ol type="i">
<li>the Imaris spot function,</li>
<li>the surface rendering functions, and the surface-surface coloc MATLAB plugin.</li>
<li>The MATLAB plugins for Imaris, called XTension, can be downloaded from the Imaris open repository at this link: <a href="https://imaris.oxinst.com/open/">https://imaris.oxinst.com/open/</a>. Once the MATLAB plugins have been downloaded, add them to Imaris by selecting File &gt; Preferences &gt; Custom Tools.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The folder with the plugins can be linked by using the “Add” button.</div>
<p>The folder that includes the .exe and .xml files must contain the file ImarisLib.jar.</p>
<div class="note">
<span class="note-title">Note:</span> If this file is not present, you can copy the .jar file included with Imaris. This file is available in the path where Imaris was installed on drive C: C:\ProgramFiles\Bitplane\Imarisx64 9.3.1\XT\rtmatlab\ImarisLib.jar. At the end of this process, restart Imaris and the plugins will appear in the Image Processing dropdown menu.</div>
<ol start="2">
<li>Convert the confocal image file to an .ims file using the Imaris converter software.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure that during the conversion the voxel size dimensions are not lost. <a href="#sec7.9">Troubleshooting 5</a>
</div>
<ol start="3">
<li>Open the converted file in Imaris.</li>
<li>Apply filters by clicking the Image Processing icon (<a href="#fig5">Figure 5</a>A) on the toolbar and selecting <i>Median 3×3×3</i> and <i>Background Subtraction</i> for each individual channel.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1234-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. PNN density quantification protocol with IMARIS spot function</div>
<p>Screenshots of Imaris software user-interface for spot function.</p>
<p>(A) Imaris icon for image processing.</p>
<p>(B) Measurement of PNN diameter of Wisteria floribunda agglutinin (WFA)-labeled cell in <i>Slice</i> mode. Scale bar, 20 μm.</p>
<p>(C) Imaris icon for spot function.</p>
<p>(D) Setting the spot detection parameters.</p>
<p>(E–G) The spot function automatically detects spots indicated by grey balls. Scale bars: 20 and 50 μm, respectively. Arrows, PNN-coated cells, where the software failed to automatically detect. (F) Imaris icon for edit mode. Those spots are added manually (G).</p>
<p>(H) The number of detected spots can be exported with the statistic function. Arrow, Imaris icon for statistics.</p>
</figcaption>
</figure>
</li>
<li>Visualize the PNN channel. <a href="#sec7.11">Troubleshooting 6</a>
</li>
<li>Measure the diameter of PNNs in slice-visualization mode (<a href="#fig5">Figure 5</a>B). The length of the line is in the top right corner. Use the left navigation bar to go through the z stack.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use the mouse wheel to zoom in.</div>
<ol start="7">
<li>Add a spot function by clicking on the spot icon (<a href="#fig5">Figure 5</a>C) and entering the diameter measured in step 6 (<a href="#fig5">Figure 5</a>D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend to use spot function instead of surfaces to count the absolute number of PNN because the surfaces can be fragmented during the rendering process, which would be counted as a single surface and artificially increase the numbers.</div>
<div class="note">
<span class="note-title">Note:</span> We refrain from measuring the PNN intensity signal for quantifying changes in density because this method is prone to artifacts which can be introduced by fluctuation of the laser power during the images acquisition, bleaching of the fluorophores during the scanning, different staining quality between brain slices, etc. (<a href="#bib23">Waters, 2009</a>).</div>
<div class="note">
<span class="note-title">Note:</span> An alternative to the spot function can be the quantification of the area or volume occupied by the PNN, which can be done in ImageJ/Fiji without additional plugins needed. The 3D quantification of the PNN phagocytosed by microglia, based on Imaris surface function, will be discussed in the following step-by-step method paragraph.</div>
<ol start="8">
<li>Click “Next” to visualize the automatically-detected spots (<a href="#fig5">Figure 5</a>E). By changing the threshold of the histogram in the lower left corner, new spots can be automatically added.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Some PNNs will not be detected. They can be manually added, as described next.</div>
<ol start="9">
<li>Click “Next” to end the detection.</li>
<li>Go to the edit mode by clicking on the pencil icon (<a href="#fig5">Figure 5</a>F).</li>
<li>Manually add the missed spots by positioning the mouse cursor on the missed PNNs, and pressing <i>Shift + left mouse click</i> (<a href="#fig5">Figure 5</a>G). Remove misplaced spots by pressing <i>Shift + left mouse click</i> on the spot.</li>
<li>Read the number of the detected spots in the statistic function (<a href="#fig5">Figure 5</a>H). In this example, 36 spots were detected.</li>
<li>Calculate the PNN density as <math> <mrow> <mi>d</mi> <mo>=</mo> <mfrac> <mrow> <mtext>n </mtext> <mrow> <mo>(</mo> <mtext>PNN</mtext> <mo>)</mo> </mrow> </mrow> <mrow> <mtext>V z</mtext> <mo>−</mo> <mtext>stack</mtext> </mrow> </mfrac> <mspace></mspace> <mtext>where V z</mtext> <mo>−</mo> <mtext>stack</mtext> <mo>=</mo> <mtext>x∗y∗z</mtext> </mrow> </math> in spreadsheet software (for example Excel) and report the result as the number of PNN-coated cells per mm<sup>3</sup>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Spot counting is also possible with open-source programs. The number of PNN-coated cells can be calculated using the spot counter plugin (<a href="https://github.com/nicost/spotCounter/">https://github.com/nicost/spotCounter/</a>) in ImageJ/Fiji (<a href="https://fiji.sc/">https://fiji.sc/</a>). Another possibility is the Spot Detection Utilities plug in (<a href="https://gitlab.pasteur.fr/bia/spot-detection-utilities">https://gitlab.pasteur.fr/bia/spot-detection-utilities</a>) for the software Icy (<a href="http://icy.bioimageanalysis.org/">http://icy.bioimageanalysis.org/</a>), which allows spot counting and 3D-tracking.</div>
<h3 id="sec5.2">Quantification of the amount of PNN material within microglial lysosomes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> up to 30 min</div>
<p>In the following example, we will use data from the primary somatosensory cortex (S1) of a male C57BL/6J mouse treated with 2× KXA.</p>
<div class="note">
<span class="note-title">Note:</span> In our experiments, we use the lysosomal/endosomal marker CD68 for the quantification of the PNN material inside microglia. Other lysosomal markers are available such as Lamp1, Lamp2a, and Lamp2b. It is also possible to omit the lysosomal marker and focus on PNN material within microglia.</div>
<ol start="14">
<li>Convert the confocal image file to an .ims file using the Imaris converter software.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure that during the conversion the voxel size dimensions are not lost.</div>
<ol start="15">
<li>Load the file into Imaris.</li>
<li>If necessary, apply filters such as median filter, background subtraction, or similar using the Image Processing function (<a href="#fig5">Figure 5</a>A).</li>
<li>Activate the microglia channel and create a new surface by clicking on the surface icon (<a href="#fig6">Figure 6</a>A).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1234-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Determine PNN fragments within microglial CD68 with IMARIS surface rendering</div>
<p>Screenshots of Imaris software user-interface for surface rendering.</p>
<p>(A) Creating a surface module for Iba1-stained microglia (green). Arrow, Imaris icon for surface function.</p>
<p>(B) Setting of the microglia surface parameter.</p>
<p>(C) Microglia surface rendering with local contrast. Scale bar, 20 μm.</p>
<p>(D) Application of the <i>Area</i> filter to remove rendering artifacts.</p>
<p>(E and F) Apply <i>Surface-surface coloc</i> plugin for surface.</p>
<p>(G and H) (G) Example image of colocalized surfaces for microglia(green), PNN (magenta), and CD68 (blue) with zoom-in (H). Scale bar, 20 μm. Zoom-in, 10 μm.</p>
<p>(I) Imaris icon for statistics.</p>
</figcaption>
</figure>
</li>
<li>Click “Next”, select the microglia channel in the drop-down menu and enter the surface detail resolution. In our analysis, we used a surface detail of 0.2 μm. Select “Background subtraction (local contrast)” as the thresholding method (<a href="#fig6">Figure 6</a>B) and click “Next”.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The “surface detail” changes as a function of the resolution of the confocal image.</div>
<div class="note">
<span class="note-title">Note:</span> Using the local contrast as the thresholding method prevents the creation of artifacts due to fluctuations in the staining intensity. <a href="#sec7.11">Troubleshooting 7</a>
</div>
<ol start="19">
<li>Cover the microglia volume with the surface by moving the threshold histogram toward the left side (<a href="#fig6">Figure 6</a>C) and click “Next”.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The little surface artifact that might occur will be filtered out in the next step.</div>
<div class="note">
<span class="note-title">Note:</span> The same values of thresholding can be used when the images are acquired at the same day, the laser power was stable, and the staining quality being comparable within the brain slice. <u>Note</u>: If the analysis is done with Imaris, we recommend to select the <i>local contrast</i> instead of the <i>intensity</i> for thresholding during the surface rendering process, to prevent artifact formation, which would result in overestimation of the 3D surfaces.</div>
<ol start="20">
<li>Correct for the surface artifacts by applying a filter. In the filter type drop-down menu select “Area” and apply a filter of 10 μm<sup>2</sup> (<a href="#fig6">Figure 6</a>D). Click “Next” to end the surface creation process.</li>
<li>Repeat steps 17–20 for the PNN and the CD68 channels.</li>
<li>Rename the surfaces as <i>Microglia</i>, <i>PNN</i> and <i>CD68</i> by double clicking on them.</li>
<li>Perform “Resample 3D” to speed up the computation. Click on <i>Edit &gt; Resample 3D</i> and enter 50 as the new size of the z axis. Click “OK”.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> “Resample 3D” reduces the voxel density in a data set to increase the computation speed.</div>
<div class="note">
<span class="note-title">Note:</span> Resampling reduces the number of voxels in a grid but keeps the original relationship between the voxels.</div>
<ol start="24">
<li>Select the microglia and CD68 surfaces and click on <i>Image Processing &gt; Surfaces Functions &gt; Surface-surface Coloc</i> to open the MATLAB plug in.</li>
<li>In the MATLAB window, select the surfaces to colocalize (<a href="#fig6">Figure 6</a>E) and click “OK”.</li>
<li>In the next pop-up window, click on “Smoothing”.</li>
<li>Enter the smoothing factor. The smoothing factor must be double the image voxel size (in our case: 0.4 μm). Click “OK”.</li>
<li>At the end of the computation, a new surface will be created. Rename it as <i>Microglia Cd68 Coloc.</i>
</li>
<li>Select the <i>PNN</i> and the <i>Microglia Cd68 Coloc</i> surfaces and repeat the colocalization process as described in steps 24–27 (<a href="#fig6">Figure 6</a>F).</li>
<li>At the end of the computation, a new surface will be created. Rename it as <i>PNN within Microglia CD68.</i>
</li>
<li>Uncheck the Volume module and verify the presence of colocalized surfaces (<a href="#fig6">Figures 6</a>G and 6H).</li>
<li>Export the statistics file from all the created surfaces by clicking on the statistics icon (<a href="#fig6">Figure 6</a>I). Save the file in <i>.</i>xls or .csv format.</li>
<li>In a spreadsheet software (e.g., Excel), calculate the percentage of microglial volume occupied by CD68 and the percentage of the CD68 volume within microglia occupied by PNN.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The created <i>surface area</i> or <i>volume</i> can be exported as an Imaris statistics file (see point 19) and used; for example, to estimate the PNN covered surface after the treatments.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The repeated ketamine treatment is less invasive than other PNN-removal strategies such as injecting chondroitinase into the brain (<a href="#bib18">Pizzorusso et al., 2002</a>; <a href="#bib19">Romberg et al., 2013</a>; <a href="#bib7">Carulli et al., 2016</a>). On the other hand, the ketamine dosage is still high, making the translation to clinical practice challenging. Lower, repeated ketamine dosage at dissociative concentration is commonly used to obtain a schizophrenic-like rat model (<a href="#bib13">Kaushik et al., 2020</a>). These data suggest that the effects of ketamine on the brain might be dose-dependent and its actual effect is less obvious. Apart the well-described inhibition of the NMDA receptors (<a href="#bib5">Autry et al., 2011</a>), ketamine seems to have a plethora of different target sites, from sodium channels to cholinergic, dopamine, and opioid receptors (<a href="#bib24">Wray and Rasenick, 2019</a>) (<a href="#bib20">Sleigh et al., 2014</a>): this makes characterization of the molecular and cellular pathways involved challenging.</p>
<p>60 Hz light entrainment successfully mimics the gamma oscillations induced by ketamine (<a href="#bib1">Ahnaou et al., 2017</a>; <a href="#bib2">Akeju et al., 2016</a>). With <i>in-vivo</i> electrophysiological recordings in awake, freely-moving mice, we observed strong 60 Hz oscillations evoked in the V1 (<a href="#bib22">Venturino et al., 2021</a>), suggesting that the visual cortex is tuned to the stimulation frequency. How far 60 Hz oscillations can spread in the brain is still not clear. 40 Hz light-induced oscillations have also been recorded in the hippocampus (<a href="#bib11">Iaccarino et al., 2016</a>) and in the prefrontal cortex (<a href="#bib12">Jones et al., 2019</a>), suggesting that light-induced oscillation can also reach other cortical and subcortical regions.</p>
<p>A limitation of the quantification side of this protocol is that we performed our analysis on a commercial software provided by the Bioimaging facility of IST Austria. Imaris combines an intuitive, user-friendly software interface to represent and analyze 3D images. Custom-made scripts from MatLab or python can be implemented as plugins in Imaris XTension.</p>
<p>Alternative strategies to visualize and analyze 3D images and surfaces provides VAA3D (<a href="#bib17">Peng et al., 2010</a>) (<a href="https://alleninstitute.org/what-we-do/brain-science/research/products-tools/vaa3d/">https://alleninstitute.org/what-we-do/brain-science/research/products-tools/vaa3d/</a>) and the python-based Napari (<a href="https://napari.org/tutorials/fundamentals/installation.html">https://napari.org/tutorials/fundamentals/installation.html#installing-as-a-bundled-app</a>), which is based on an open source plugin (<a href="https://www.napari-hub.org/">https://www.napari-hub.org/</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The KXA solution is not clear (step 1).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Prepare a fresh KXA solution for every experiment. If the precipitates are visible, filter the solution using 0.2 μm syringe filters.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>The mouse does not become anesthetized and reflexes are still present (step 5–6).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Make sure that the correct dosage of KXA was administered and that the injection was performed <i>i.p.</i> If you can see an accumulation of liquid under the skin, then the solution was not injected in the peritoneal cavity. In this case, exclude the mouse from the study.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The Arduino code doesn’t work (step 22).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>There are several reasons why the code might not work. Here are possible reasons: first, the parenthesis is open but not closed; or second, the semicolon is missing at the end of the command line.</p>
<p>In case, the error message says “COM Port not found”, navigate in the Arduino IDE to <i>Tools &gt; Port</i> and make sure that the COM Port is checked.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>The 60 Hz light stimulation device generates electrical noise during <i>in-vivo</i> electrophysiology recordings (step 31).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>To perform <i>in-vivo</i> electrophysiology during the light stimulation, it is necessary to electrically isolate every solder connection, building a small Faraday cage around each. To do this, wrap a small piece of copper shielding mesh around the connection and fix it with electrical tape. Isolate the transistor in the same way.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>The voxel dimensions are lost during the conversion process (step 2 in the “<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>”).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Check the voxel dimension in the metadata file of the confocal file and enter the correct voxel sizes in Imaris by pressing <i>CTRL + I</i>.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>The channel adjustments tool is not visible, the channels cannot be activated or deactivated (step 5 in the “<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>”).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Navigate to <i>Edit &gt; Show Display Adjustment</i> or click <i>CTRL + D</i>.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>The surface rendering with the local contrast thresholding method is not precise because the staining signal is low (step 18 in the “<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>”).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>If you have a low signal-to-noise ratio from the staining, we recommend to select <i>Brightness</i> as the thresholding method during the Imaris surface rendering process, although this method is less precise in signal detection.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ahnaou, A., Huysmans, H., Biermans, R., Manyakov, N.V., and Drinkenburg, W.H.I.M. (2017). Ketamine: differential neurophysiological dynamics in functional networks in the rat brain. Transl. Psychiatry <i>7</i>, e1237. <a class="external-link" href="https://doi.org/10.1038/tp.2017.198">https://doi.org/10.1038/tp.2017.198</a></p>
<p id="bib2">Akeju, O., Song, A.H., Hamilos, A.E., Pavone, K.J., Flores, F.J., Brown, E.N., and Purdon, P.L. (2016). Electroencephalogram signatures of ketamine anesthesia-induced unconsciousness. Clin. Neurophysiol. <i>127</i>, 2414-2422. <a class="external-link" href="https://doi.org/10.1016/j.clinph.2016.03.005">https://doi.org/10.1016/j.clinph.2016.03.005</a></p>
<p id="bib3">Alda, S., Fujiwara, N., and Shimoji, K. (1994). Differential regional effects of ketamine on spontaneous and glutamate-induced activities of single CNS neurones in rats. Br. J. Anaesth. <i>73</i>, 388-394. <a class="external-link" href="https://doi.org/10.1093/bja/73.3.388">https://doi.org/10.1093/bja/73.3.388</a></p>
<p id="bib4">Arras, M., Autenried, P., Rettich, A., Spaeni, D., and Rülicke, T. (2001). Optimization of intraperitoneal injection anesthesia in mice: drugs, dosages, adverse effects, and anesthesia depth. Comp. Med. <i>51</i>, 443-456. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00718-8/sref4">View at publisher</a></p>
<p id="bib5">Autry, A.E., Adachi, M., Nosyreva, E., Na, E.S., Los, M.F., Cheng, P.F., Kavalali, E.T., and Monteggia, L.M. (2011). NMDA receptor blockade at rest triggers rapid behavioural antidepressant responses. Nat. Nat. <i>475</i>, 91-96. <a class="external-link" href="https://doi.org/10.1038/nature10130">https://doi.org/10.1038/nature10130</a></p>
<p id="bib6">Behrens, M.M., Ali, S.S., Dao, D.N., Lucero, J., Shekhtman, G., Quick, K.L., and Dugan, L.L. (2007). Ketamine-induced loss of phenotype of fast-spiking interneurons is mediated by NADPH-oxidase. Science <i>318</i>, 1645-1647. <a class="external-link" href="https://doi.org/10.1126/science.1148045">https://doi.org/10.1126/science.1148045</a></p>
<p id="bib7">Carulli, D., Kwok, J.C.F., and Pizzorusso, T. (2016). Perineuronal nets and CNS plasticity and repair. Neural Plast. <i>2016</i>, 4327082. <a class="external-link" href="https://doi.org/10.1155/2016/4327082">https://doi.org/10.1155/2016/4327082</a></p>
<p id="bib8">Freria, C.M., Hall, J.C.E., Wei, P., Guan, Z., McTigue, D.M., and Popovich, P.G. (2017). Deletion of the fractalkine receptor, CX3CR1, improves endogenous repair, axon sprouting, and synaptogenesis after spinal cord injury in mice. J. Neurosci. <i>37</i>, 3568-3587. <a class="external-link" href="https://doi.org/10.1523/JNEUROSCI.2841-16.2017">https://doi.org/10.1523/JNEUROSCI.2841-16.2017</a></p>
<p id="bib9">Harkness, E.J. and Wagner, E. (1995). The Biology and Medicine of Rabbits and Rodents (Williams&amp; Wilkins). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00718-8/sref9">View at publisher</a></p>
<p id="bib10">Hor, C.N., Yeung, J., Jan, M., Emmenegger, Y., Hubbard, J., Xenarios, I., Naef, F., and Franken, P. (2019). Sleep–wake-driven and circadian contributions to daily rhythms in gene expression and chromatin accessibility in the murine cortex. Proc. Natl. Acad. Sci. <i>116</i>, 25773-25783. <a class="external-link" href="https://doi.org/10.1073/PNAS.1910590116">https://doi.org/10.1073/PNAS.1910590116</a></p>
<p id="bib11">Iaccarino, H.F., Singer, A.C., Martorell, A.J., Rudenko, A., Gao, F., Gillingham, T.Z., Mathys, H., Seo, J., Kritskiy, O., Abdurrob, F., et al. (2016). Gamma frequency entrainment attenuates amyloid load and modifies microglia. Nature <i>540</i>, 230-235. <a class="external-link" href="https://doi.org/10.1038/nature20587">https://doi.org/10.1038/nature20587</a></p>
<p id="bib12">Jones, M., McDermott, B., Oliveira, B.L., O’Brien, A., Coogan, D., Lang, M., Moriarty, N., Dowd, E., Quinlan, L., McGinley, B., et al. (2019). Gamma band light stimulation in human case studies: groundwork for potential Alzheimer’s disease treatment. J. Alzheimer’s Dis. <i>70</i>, 171-185. <a class="external-link" href="https://doi.org/10.3233/JAD-190299">https://doi.org/10.3233/JAD-190299</a></p>
<p id="bib13">Kaushik, R., Lipachev, N., Matuszko, G., Kochneva, A., Dvoeglazova, A., Becker, A., Paveliev, M., and Dityatev, A. (2020). Fine structure analysis of perineuronal nets in the ketamine model of schizophrenia. Eur. J. Neurosci. 1-17. <a class="external-link" href="https://doi.org/10.1111/ejn.14853">https://doi.org/10.1111/ejn.14853</a></p>
<p id="bib14">Levin-Arama, M., Abraham, L., Waner, T., Harmelin, A., Steinberg, D.M., Lahav, T., and Harlev, M. (2016). Subcutaneous compared with intraperitoneal ketamine–xylazine for anesthesia of mice. Am. Assoc. Lab. Anim. Sci. <i>55</i>, 794. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00718-8/sref14">View at publisher</a></p>
<p id="bib15">Martorell, A.J., Paulson, A.L., Suk, H.J., Abdurrob, F., Drummond, G.T., Guan, W., Young, J.Z., Kim, D.N.W., Kritskiy, O., Barker, S.J., et al. (2019). Multi-sensory gamma stimulation ameliorates Alzheimer’s-associated pathology and improves cognition. Cell <i>177</i>, 256-271.e22. <a class="external-link" href="https://doi.org/10.1016/j.cell.2019.02.014">https://doi.org/10.1016/j.cell.2019.02.014</a></p>
<p id="bib16">Paolicelli, R.C., Bolasco, G., Pagani, F., Maggi, L., Scianni, M., Panzanelli, P., Giustetto, M., Ferreira, T.A., Guiducci, E., Dumas, L., et al. (2011). Synaptic pruning by microglia is necessary for normal brain development. Science <i>333</i>, 1456-1458. <a class="external-link" href="https://doi.org/10.1126/science.1202529">https://doi.org/10.1126/science.1202529</a></p>
<p id="bib17">Peng, H., Ruan, Z., Long, F., Simpson, J.H., and Myers, E.W. (2010). V3D enables real-time 3D visualization and quantitative analysis of large-scale biological image data sets. Nature Biotechnology <i>28</i>, 348-353. <a class="external-link" href="https://doi.org/10.1038/nbt.1612">https://doi.org/10.1038/nbt.1612</a></p>
<p id="bib18">Pizzorusso, T., Medini, P., Berardi, N., Chierzi, S., Fawcett, J.W., and Maffei, L. (2002). Reactivation of ocular dominance plasticity in the adult visual cortex. Science <i>298</i>, 1248-1251. <a class="external-link" href="https://doi.org/10.1126/science.1072699">https://doi.org/10.1126/science.1072699</a></p>
<p id="bib19">Romberg, C., Yang, S., Melani, R., Andrews, M.R., Horner, A.E., Spillantini, M.G., Bussey, T.J., Fawcett, J.W., Pizzorusso, T., and Saksida, L.M. (2013). Depletion of perineuronal nets enhances recognition memory and long-term depression in the perirhinal cortex. J. Neurosci. <i>33</i>, 7057-7065. <a class="external-link" href="https://doi.org/10.1523/JNEUROSCI.6267-11.2013">https://doi.org/10.1523/JNEUROSCI.6267-11.2013</a></p>
<p id="bib20">Sleigh, J., Harvey, M., Voss, L., and Denny, B. (2014). Ketamine - more mechanisms of action than just NMDA blockade. Trends Anaesth. Crit. Care. 76-81. <a class="external-link" href="https://doi.org/10.1016/j.tacc.2014.03.002">https://doi.org/10.1016/j.tacc.2014.03.002</a></p>
<p id="bib21">Tsukamoto, A., Serizawa, K., Sato, R., Yamazaki, J., and Inomata, T. (2015). Vital signs monitoring during injectable and inhalant anesthesia in mice. Exp. Anim. <i>64</i>, 57-64. <a class="external-link" href="https://doi.org/10.1538/expanim.14-0050">https://doi.org/10.1538/expanim.14-0050</a></p>
<p id="bib22">Venturino, A., Schulz, R., Jesús-Cortés, H. De, Maes, M.E., Nagy, B., Reilly-Andújar, F., Colombo, G., Cubero, R.J.A., Uiterkamp, F.E.S., Bear, M.F., et al. (2021). Microglia enable mature perineuronal nets disassembly upon anesthetic ketamine exposure or 60-Hz light entrainment in the healthy brain. Cell Rep. <i>36</i>, 109313. <a class="external-link" href="https://doi.org/10.1016/J.CELREP.2021.109313">https://doi.org/10.1016/J.CELREP.2021.109313</a></p>
<p id="bib23">Waters, J.C. (2009). Accuracy and precision in quantitative fluorescence microscopy. J. Cell Biol. <i>185</i>, 1135. <a class="external-link" href="https://doi.org/10.1083/JCB.200903097">https://doi.org/10.1083/JCB.200903097</a></p>
<p id="bib24">Wray, N.H. and Rasenick, M.M. (2019). NMDA-receptor independent actions of ketamine: a new chapter in a story that’s not so old. Neuropsychopharmacology <i>44</i>, 220-221. <a class="external-link" href="https://doi.org/10.1038/s41386-018-0201-y">https://doi.org/10.1038/s41386-018-0201-y</a></p>
<p id="bib25">Zorumski, C.F., Izumi, Y., and Mennerick, S. (2016). Ketamine: NMDA receptors and beyond. J. Neurosci. <i>36</i>, 11158-11164. <a class="external-link" href="https://doi.org/10.1523/JNEUROSCI.1547-16.2016">https://doi.org/10.1523/JNEUROSCI.1547-16.2016</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. Sandra Siegert (<a href="mailto:sandra.siegert@ist.ac.at">sandra.siegert@ist.ac.at</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or codes.</p>
<h3>Acknowledgments</h3>
<p>This research was supported by the <a href="https://doi.org/10.13039/100010663">European Research Council</a> (grant 715571 to S.S.). We thank Rouven Schulz, Michael Schunn, Claudia Gold, Gabriel Krens, Sarah Gorkiewicz, Margaret Maes, Jürgen Siegert, Marco Benevento, and Sara Oakeley for comments on the manuscript and the IST Austria Bioimaging Facility for the technical support.</p>
<h3>Author contributions</h3>
<p>Conceptualization, A.V., S.S.; writing – original draft: A.V., S.S.; funding acquisition: S.S.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests. A.V. and S.S. disclose an international patent application (PCT/EP2020/079365).</p>
</section>
</article>