<article>
<section>
<h1 id="header">In-yeast reconstruction of the African swine fever virus genome isolated from clinical samples</h1>
<p><time datetime="2021-09-10">Published: September 10, 2021</time></p>
<p>Fabien Labroussaa,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">6</a>,<a href="#fn3">7</a>,<a href="#cor1">*</a></sup> Kemal Mehinagic,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a></sup> Valentina Cippa,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Matthias Liniger,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Hatice Akarsu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Nicolas Ruggli,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">5</a></sup> and Joerg Jores<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Veterinary Bacteriology, University of Bern, Bern 3001, Switzerland</p>
<p id="aff2"><sup>2</sup>Department of Infectious Diseases and Pathobiology (DIP), Vetsuisse Faculty, University of Bern, Bern 3001, Switzerland</p>
<p id="aff3"><sup>3</sup>Institute of Virology and Immunology (IVI), Sensemattstrasse 293, Mittelhäusern 3147, Switzerland</p>
<p id="aff4"><sup>4</sup>Graduate School for Cellular and Biomedical Sciences, University of Bern, Bern 3001, Switzerland</p>
<p id="fn1"><sup>5</sup>These authors contributed equally</p>
<p id="fn2"><sup>6</sup>Technical contact</p>
<p id="fn3"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:fabien.labroussaa@vetsuisse.unibe.ch">fabien.labroussaa@vetsuisse.unibe.ch</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:joerg.jores@vetsuisse.unibe.ch">joerg.jores@vetsuisse.unibe.ch</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100803">10.1016/j.xpro.2021.100803</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes a synthetic genomics pipeline to clone and engineer the entire 190-kbp genome of the African swine fever virus (ASFV) genotype II in yeast using transformation-associated recombination cloning. The viral genome was cloned using DNA directly extracted from a clinical sample. In addition, the precise deletion of a non-essential gene and its replacement by a synthetic reporter gene cassette are presented. This protocol is applicable to other ASFV genotypes and other large DNA viruses.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Use of TAR for the individual cloning of five ASFV sub-genomic fragments in yeast</li>
<li>Chemical synthesis of both 5′ and 3′ ITR genomic regions</li>
<li>Replacement of the C962R gene by reporter genes (eGFP or secNLuc)</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/985-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>ASFV causes a devastating hemorrhagic disease in pigs with a mortality often reaching up to 100% (<a href="#bib17">Sánchez-Cordón et al., 2018</a>). ASFV is rapidly spreading throughout Europe and Asia, already responsible for the loss of approximatively a third of the world’s pig population. The development of effective treatments and vaccines are hindered due to knowledge gaps regarding the biology of the virus (<a href="#bib18">Sánchez et al., 2019</a>).</p>
<p>ASFV is a large DNA virus belonging to the <i>Asfarviridae</i> family with a genome size ranging between 170 and 190 kbp (<a href="#bib6">Dixon et al., 2013</a>), a size problematic to be stably maintained in <i>E. coli</i>-based systems commonly used to generate and mutagenize molecular clones. The generation of defined ASFV mutants still relies on homologous recombination methods mainly involving eukaryotic cells such as swine macrophages (<a href="#bib3">Borca et al., 2021</a>; <a href="#bib7">Gallardo et al., 2018</a>; <a href="#bib11">Monteagudo et al., 2017</a>; <a href="#bib13">O’Donnell et al., 2015</a>), although the use of CRISPR-Cas9 technology was recently implemented (<a href="#bib2">Borca et al., 2018</a>). In addition, ASFV replication in cell culture has been reported to result in adaptive single nucleotide mutations, genome deletions and rearrangements associated with phenotype changes (<a href="#bib3">Borca et al., 2021</a>; <a href="#bib9">Krug et al., 2015</a>).</p>
<p>Here we describe a yeast-based protocol for the cloning and engineering of an ASFV isolate belonging to the genotype II, the genotype responsible for the current pandemic in Europe and Asia. This pipeline relies on the transformation-associated recombination (TAR) cloning strategy, which was originally developed for the selective isolation and maintenance of large eukaryotic DNA regions in <i>Saccharomyces cerevisiae</i> (<a href="#bib10">Larionov et al., 1996</a>). The TAR cloning method uses the superior capacities of the yeast to perform <i>in vivo</i> homologous recombination and diverts it in order to recombine several overlapping DNA fragments together. This system was adopted to reconstruct the genome of the first synthetic mycoplasma cell (Syn 1.0; <a href="#bib8">Gibson et al., 2010</a>), but was also applied to other large DNA viruses such as the Herpes simplex virus type 1 (HSV-1; <a href="#bib14">Oldfield et al. 2017</a>) and, more recently, to RNA viruses including the SARS-CoV-2 (<a href="#bib19">Thao et al., 2020</a>).</p>
<p>This protocol includes all the necessary steps from the extraction of ASFV genomic DNA from clinical samples up to the reconstruction of full-length ASFV genomes, natural or engineered, in the yeast <i>S. cerevisiae</i>. This protocol does not include a rescue system, which has to be developed and adapted separately for the different viruses to be tackled.</p>
<h3 id="sec1.1">Biosafety</h3>
<p>In Switzerland, ASFV is a risk group 4 pathogen and hence all work involving live virus was carried out in the BSL-3Ag containment facility of the Institute of Virology and Immunology in Mittelhäusern (Switzerland). After inactivation, the sera from ASFV-infected pigs were transferred out of the BSL-3Ag environment to standard BSL-2 laboratories, where the extraction of the viral genomic DNA and all the following steps of this protocol were performed. Animal experiments were conducted at the IVI in compliance with the animal welfare regulation of Switzerland under the cantonal license BE18/2019.</p>
<h3 id="sec2.1">Design of the ASFV genome reconstruction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼2 h (user dependent)</div>
<p>The <i>in silico</i> design used in this protocol involves the fragmentation of the ASFV genome into seven sub-genomic fragments as presented in <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/985-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. TAR cloning design for the reconstruction of the ASFV genome</div>
<p>The genome was divided into seven sub-genomic fragments. Fragments 1 and 2, encompassing highly repetitive ITRs, were chemically-synthetized. Both contain an artificial 50-bp sequence stretch (in pink) at their 5′ and 3′-ends, respectively. These 50-bp stretches will be clipped off via restriction endonucleases indicated by the asterisks after full-length genome assembly. Positions indicated for each of the seven sub-genomic fragments correspond to the ASFV nucleotide sequence as provided in GenBank.</p>
</figcaption>
</figure>
<ol>
<li>The 5′ and 3′-ends of the ASFV genome, consisting of inverted-terminal-repeats (ITRs), are chemically-synthetized (GenScript) in order to prevent any illegitimate recombination events during the final reassembly in yeast.
<ol type="a">
<li>Both synthetic fragments, named Fragment 1 and 2 respectively, are ligated into the vector pUC57 using the unique <i>EcoRV</i> restriction site, clone in <i>E. coli</i> and sequence-verified.</li>
<li>They contain two <i>SmaI</i> restriction sites at their 5′ and 3‘-ends, allowing their excision from the pUC57 vector.</li>
<li>Fragment 1 is a 2,090-bp dsDNA fragment consisting of a 50-bp unique region (randomly designed for site-directed TAR cloning purposes), followed by an <i>I-SceI</i> restriction site and by the first 2,010 nucleotides of the ASFV genome.</li>
<li>Fragment 2 consists of the last 1,736 nucleotides of the ASFV genome followed by an <i>I-SceI</i> restriction site and by another 50-bp unique region (also randomly designed for TAR cloning).</li>
<li>Their corresponding sequences are provided as supplementary material.</li>
</ol>
</li>
<li>The other five sub-genomic fragments ranging in size from 12.4 to 49.6 kbp were isolated using the TAR cloning method. Their corresponding start and end positions are displayed in <a href="#fig1">Figure 1</a>.
<ol type="a">
<li>Fragment 4, 5 and 6 mainly contain genes encoding proteins involved in virus replication and structure.</li>
<li>Fragments 3 and 7 encompass genes included in multi-gene family (MGF) 100, 110, 360 and 505, most of which still have an unknown function (<a href="#bib6">Dixon et al., 2013</a>; <a href="#bib20">Zhu et al., 2020</a>).</li>
<li>Alternative designs consisting of different numbers of sub-genomic fragments with different 5’ and/or 3′-ends are generally possible, but have not been investigated here. As a general rule of thumb, fragmentation designs aiming for a higher number of shorter sub-genomic fragments require more experimental time (both in term of design and isolation of these genomic intermediates) but will ultimately result in more flexible assemblies.</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Design of the primers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h–2 h (user dependent)</div>
<p>Primers used throughout the protocol are listed in the <a href="#key-resources-table">key resources table</a> (here).</p>
<ol start="3">
<li>In addition to their 3′-binding regions with the pCC1BAC-Ura3(-His3) TAR vectors, all the primers used for TAR cloning contain a 50-bp region (in bold) complementary to the 5′-end (reverse primers) or the 3′-end (forward primers) of the ASFV sub-genomic fragment of interest. Such overlaps (also called hooks) are required for the specific and legitimate homologous recombination in yeast.</li>
<li>The presence of an <i>I-SceI</i> restriction site (underlined) allows the linearization of the ASFV sub-genomic fragments after their isolation and purification as yeast artificial chromosomes (YACs).</li>
<li>Due to their length, all primers used for TAR cloning are purified on polyacrylamide gel electrophoresis (PAGE purified) (Microsynth, Balgach, Switzerland). All other primers used for the PCR-based screening and multiplex PCRs of the different yeast clones were only desalted after synthesis.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Stellar™ competent <i>E. coli</i>
</td>
<td>Takara Bio</td>
<td>636763</td>
</tr>
<tr>
<td>TransforMax™ Epi300™ Electrocompetent <i>E. coli</i>
</td>
<td>Epicentre/Lucigen</td>
<td>EC300110</td>
</tr>
<tr>
<td>ASFV isolate Georgia 2007</td>
<td>LMA, Tbilisi (Georgia)</td>
<td>This work</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>S. cerevisiae</i> VL6-48N</td>
<td>JCVI</td>
<td><a href="#bib12">Noskov et al., 2002</a></td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pCC1BAC-His3/Ura3</td>
<td>JCVI</td>
<td><a href="#bib8">Gibson et al., 2010</a></td>
</tr>
<tr>
<td>pUC57-P_Pol-secNluc _C962R-int</td>
<td>GenScript</td>
<td>This work</td>
</tr>
<tr>
<td>pUC57-P_Pol-eGFP_C962R-int</td>
<td>GenScript</td>
<td>This work</td>
</tr>
<tr>
<td>pUC57-Fragment1</td>
<td>GenScript</td>
<td>This work</td>
</tr>
<tr>
<td>pUC57-Fragment2</td>
<td>GenScript</td>
<td>This work</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>TAR cloning (ASFV WT)ASFTARhook-1R: <b>gtctagctagcatctagtatgaatcg</b><b>gattcatggcatgcatgcaatcgt</b><u>TAGGGATAACAGGGTAAT</u>GTCGTGACTGGGAAAACCCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-2F: <b>taacctgacattgcgatccgtaca</b><b>attgcatgctggaactacctgatcag</b><u>TAGGGATAACAGGGTAAT</u>GATCCTCTAGAGTCGACCTGCAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-3R: <b>tgttaatatatgaagaatcattgt</b><b>ttcacatctatggggtaaacatttag</b><u>TAGGGATAACAGGGTAAT</u>GTCGTGACTGGGAAAACCCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-3F: <b>cagaacatttagataattgaga</b><b>gattactttccatacttgttaagctttt</b><u>TAGGGATAACAGGGTAAT</u>GATCCTCTAGAGTCGACCTGCAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-4R: <b>atagatactcaaaagacagt</b><b>cgttcatgacgtaagattgtcgctttaatc</b><u>TAGGGATAACAGGGTAAT</u>GTCGTGACTGGGAAAACCCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-4F: <b>caggctgtgaagccagaaggcctcctgaccttgatga</b><b>aggtcgtacatga</b><u>TAGGGATAACAGGGTAAT</u>GATCCTCTAGAGTCGACCTGCAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-5R: <b>ttagtaaacagccttcggagcacg</b><b>aactgttcatcgtatttaaaaaataa</b><u>TAGGGATAACAGGGTAAT</u>GTCGTGACTGGGAAAACCCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-5F: <b>ttcagcgtctagcagcggtttcagctt</b><b>ggcaagatgcgctgagtggtagt</b><u>TAGGGATAACAGGGTAAT</u>GATCCTCTAGAGTCGACCTGCAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-6R: <b>cccgtgcggtaggttttcgtgaa</b><b>ccgataaatgttttagaaatcatttaa</b><u>TAGGGATAACAGGGTAAT</u>GTCGTGACTGGGAAAACCCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-6F: <b>agcaatatcctcttctatctcgca</b><b>atcctcctcctccatttccatagtgt</b><u>TAGGGATAACAGGGTAAT</u>GATCCTCTAGAGTCGACCTGCAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-7R: <b>cggaatcttcatccgacgatgagtcc</b><b>tattcactttatgatagtttctaa</b><u>TAGGGATAACAGGGTAAT</u>GTCGTGACTGGGAAAACCCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-7F: <b>ctctctgatggtgacaaatctccgataggaatatatg</b><b>acgtaacataatt</b><u>TAGGGATAACAGGGTAAT</u>GATCCTCTAGAGTCGACCTGCAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>TAR cloning (Fragment 4ΔC962R)ASFTARhook-4R: <b>atagatactcaaaagacagtcg</b><b>ttcatgacgtaagattgtcgctttaatc</b><u>TAGGGATAACAGGGTAAT</u>GTCGTGACTGGGAAAACCCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFTARhook-4F2: <b>cattaggacctctcccgcccatttaaatttttagtttctacaataataaa</b><u>TAGGGATAACAGGGTAAT</u>GATCCTCTAGAGTCGACCTGCAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Screening of yeast transformants (internal fragments)ASFVint-3F: ccttagtagcggcagatacc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-3R: atggacaggtttcaatgctcg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-4F: tcgcatttcgtgttgaaatacg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-4R: gcattcctgcctattaatgtgc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-5F: gttcaagtggtggaggctc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-5R: agccttccaacgtgttgttgc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-6F: gagtgcacgtatcagattacg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-6R: tcgtggtgttcaaggtaatcg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-7F: gcattaatgaaagctgtacagg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVint-7R: gttgaagtccatgaatctctgg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Screening of yeast transformants (junctions) pCC1jct-F1: ccattcagctgcgcaactg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg3-R: tataagcttactgaagccatcc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg3-F: ctgattaaagcgacaatcttacg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pCC1jct-R1: cttccatgtcggcagaatgc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg4-R: ctaccaaaacctctctacatgc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg4-F: ccaactccttagggaatatcc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg5-R: cgatatggacgatgtccagc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg5-F: tttcggcatatccagcctcc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg6-R: ctccgagctgcacttttacg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg6-F: agtattattagaaatggctgtcg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg7-R: agagattctcctgttattgtgg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg7-F: tgacctgtagtacgtatgatgg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Screening of final ASFV constructs (multiplex PCR for junctions)Multiplex PCR 1 pCC1jct-F1: ccattcagctgcgcaactg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg1-R: tgaatcggattcatggcatgc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg3-F2: gttggcaacaatccacagacg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg4-R2: cagtctttacaggaaacatgg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg4-F: ccaactccttagggaatatcc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg5-R: cgatatggacgatgtccagc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg2-F: tgacattgcgatccgtacaattgc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pCC1jct-R1: cttccatgtcggcagaatgc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Multiplex PCR 2ASFVFg1-F: taaaagagaagagtctggactg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg3-R: tataagcttactgaagccatcc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg5-F: tttcggcatatccagcctcc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg6-R: ctccgagctgcacttttacg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg6-F: agtattattagaaatggctgtcg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg7-R: agagattctcctgttattgtgg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg7-F: tgacctgtagtacgtatgatgg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVFg2-R: acgtatagttgataaaagtgctgg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Screening for eGFP and secNluc reporters integrationASFVreporter-F: gattataaagagtaactcgtagagg</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>ASFVreporter-R: acatgttacgtacagttcacttcc</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>YPDA</td>
<td>Takara Bio</td>
<td>630306</td>
</tr>
<tr>
<td>SD broth (with 2% glucose)</td>
<td>Formedium</td>
<td>CSM0205</td>
</tr>
<tr>
<td>-His single Drop-out</td>
<td>Formedium</td>
<td>DCS0071</td>
</tr>
<tr>
<td>-Ura single Drop-out</td>
<td>Formedium</td>
<td>DCS0161</td>
</tr>
<tr>
<td>Minimal SD Agar Base</td>
<td>Takara Bio</td>
<td>630412</td>
</tr>
<tr>
<td>Difco LB Agar, Lennox</td>
<td>DB</td>
<td>240110</td>
</tr>
<tr>
<td>Dulbecco’s Phosphate Buffered Saline (10<b>×</b>), no calcium, no magnesium</td>
<td>Thermo Fischer</td>
<td>14200067</td>
</tr>
<tr>
<td>Lithium acetate dihydrate</td>
<td>Sigma-Aldrich</td>
<td>L4158</td>
</tr>
<tr>
<td>ssDNA sodium salt</td>
<td>Sigma-Aldrich</td>
<td>D1626</td>
</tr>
<tr>
<td>PEG3350</td>
<td>Sigma-Aldrich</td>
<td>P4338-500G</td>
</tr>
<tr>
<td>Chelex 100 sodium form</td>
<td>Sigma-Aldrich</td>
<td>95577-100G-F</td>
</tr>
<tr>
<td>Glass beads, acid washed</td>
<td>Sigma-Aldrich</td>
<td>G8772-100G</td>
</tr>
<tr>
<td>PrimeSTAR GXL DNA polymerase</td>
<td>Takara Bio</td>
<td>R050B</td>
</tr>
<tr>
<td>GoTaq G2 Green Master Mix</td>
<td>Promega</td>
<td>M7823</td>
</tr>
<tr>
<td>Multiplex PCR Kit</td>
<td>QIAGEN</td>
<td>206143</td>
</tr>
<tr>
<td>Zymolyase®-100T</td>
<td>AMSBIO</td>
<td>120493-1</td>
</tr>
<tr>
<td>Glycoblue™ Coprecipitant (15 mg/mL)</td>
<td>Thermo Fischer</td>
<td>AM9516</td>
</tr>
<tr>
<td>Ribonucleic acid, transfer from baker's yeast</td>
<td>Sigma</td>
<td>R5636-1ML</td>
</tr>
<tr>
<td>Sodium acetate 3M, pH5.2</td>
<td>Thermo Fischer</td>
<td>R1181</td>
</tr>
<tr>
<td>RedSafe™ Nucleic Acid Staining Solution</td>
<td>iNtRON</td>
<td>21141</td>
</tr>
<tr>
<td>
<i>Dpn</i>I</td>
<td>NEB</td>
<td>R0176L</td>
</tr>
<tr>
<td>
<i>Sma</i>I</td>
<td>NEB</td>
<td>R0141L</td>
</tr>
<tr>
<td>I-<i>Sce</i>I</td>
<td>NEB</td>
<td>R0694L</td>
</tr>
<tr>
<td>CopyControl™ Induction solution</td>
<td>Lucigen</td>
<td>CCIS125</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>MagAttract HMW DNA Kit</td>
<td>QIAGEN</td>
<td>67563</td>
</tr>
<tr>
<td>QIAprep Spin Miniprep Kit</td>
<td>QIAGEN</td>
<td>27106</td>
</tr>
<tr>
<td>QIAGEN Plasmid Midi Kit</td>
<td>QIAGEN</td>
<td>12143</td>
</tr>
<tr>
<td>QIAGEN Large-Construct Kit</td>
<td>QIAGEN</td>
<td>12462</td>
</tr>
<tr>
<td>High Pure PCR Product Purification Kit</td>
<td>Merck</td>
<td>11732668001</td>
</tr>
<tr>
<td>QIAquick Gel Extraction Kit</td>
<td>QIAGEN</td>
<td>28704</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Optima™ L-90K Ultracentrifuge with SW41Ti swinging-bucket rotor</td>
<td>Beckman Coulter</td>
<td>N/A</td>
</tr>
<tr>
<td>Biospectrophotometer</td>
<td>Eppendorf</td>
<td>6135000009</td>
</tr>
<tr>
<td>Lab Armor Bead Bath</td>
<td>LabArmor</td>
<td>M706</td>
</tr>
<tr>
<td>Thermomixer C</td>
<td>Eppendorf</td>
<td>5382000015</td>
</tr>
<tr>
<td>UltraSlim LED Transilluminator</td>
<td>MaestroGen</td>
<td>SLB-01W</td>
</tr>
<tr>
<td>Ultra-Clear Centrifuge Tubes (14 <b>×</b> 89 mm, 13.2 mL)</td>
<td>Beckman Coulter</td>
<td>344059</td>
</tr>
<tr>
<td>Gene Pulser Xcell™ Electroporation System</td>
<td>Bio-Rad</td>
<td>165–2660/68</td>
</tr>
<tr>
<td>Roti®-Store yeast cryo vials</td>
<td>Carl Roth</td>
<td>X983.1</td>
</tr>
<tr>
<td>Roti®-Store yeast cryo vials (for <i>E. coli</i>)</td>
<td>Carl Roth</td>
<td>P730.1</td>
</tr>
<tr>
<td>Gene Pulser®/MicroPulser™ Electroporation Cuvettes, 0.1 cm gap</td>
<td>Bio-Rad</td>
<td>1652089</td>
</tr>
<tr>
<td>X-tracta Gel Extractor Tool</td>
<td>Sigma-Aldrich</td>
<td>Z722390</td>
</tr>
<tr>
<td>Baffled flasks ROTILABO®, straight neck, 500 mL or 2 L</td>
<td>Carl Roth</td>
<td>LY96.1 or LY98.1</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Benchling</td>
<td>N/A</td>
<td><a href="https://benchling.com">https://benchling.com</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th colspan="3">Zymolyase solution</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Zymolyase-100T</td>
<td>10 mg/mL</td>
<td>200 mg</td>
</tr>
<tr>
<td>Glycerol (50% v/v)</td>
<td>25% (v/v)</td>
<td>10 mL</td>
</tr>
<tr>
<td>Tris-HCl (1M, pH7.5)</td>
<td>50 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Solution can be stored in 0.5 mL-aliquots at −20°C up to 1 year. Avoid excessive freezing/thawing cycles.</p>
<table id="undtbl2">
<thead>
<tr>
<th colspan="3">SPEM solution</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>.2H<sub>2</sub>O</td>
<td>7.75 mM</td>
<td>0.69 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>.2H<sub>2</sub>O</td>
<td>2.31 mM</td>
<td>0.18 g</td>
</tr>
<tr>
<td>EDTA (0,5M, pH7,5)</td>
<td>10 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Sorbitol</td>
<td>1 M</td>
<td>91 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 500 mL</td>
</tr>
<tr>
<td colspan="3">Sterilized on 0.22 μm filter</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 20°C–25°C up to 1 year.</p>
<table id="undtbl3">
<thead>
<tr>
<th colspan="3">STC solution</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sorbitol</td>
<td>1 M</td>
<td>18.2 g</td>
</tr>
<tr>
<td>Tris (1M, pH7.5)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> (1M)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td colspan="3">Sterilized on 0.22 μm filter</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 20°C–25°C for up to 3 months.</p>
<table id="undtbl4">
<thead>
<tr>
<th colspan="3">SOS solution</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sorbitol</td>
<td>1 M</td>
<td>9.1 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>n/a</td>
<td>0.125 g</td>
</tr>
<tr>
<td>Bacto peptone</td>
<td>n/a</td>
<td>0.25 g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> (1M)</td>
<td>0.6 μM</td>
<td>300 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td colspan="3">Sterilized on 0.22 μm filter</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 20°C–25°C up to 1 year.</p>
<table id="undtbl5">
<thead>
<tr>
<th colspan="3">Sorbitol plates</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sorbitol</td>
<td>1 M</td>
<td>91 g</td>
</tr>
<tr>
<td>SD Base Agar</td>
<td>n/a</td>
<td>23.35 g</td>
</tr>
<tr>
<td>-His (or –Ura)</td>
<td>n/a</td>
<td>0.35 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 500 mL</td>
</tr>
<tr>
<td colspan="3">Adjust the pH to 5.8 and autoclave 121°C 15min</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>25 mL/plate</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store upside down at 4°C up to 2/3 weeks.</p>
<table id="undtbl6">
<thead>
<tr>
<th colspan="3">PEG solution</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PEG 8000</td>
<td>n/a</td>
<td>10 g</td>
</tr>
<tr>
<td>Tris-HCl (1M, pH7.5)</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> 1M</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td colspan="3">Adjust the pH to 7.5 and sterilized on 0.22 μm filter</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh for every transformation and place at 20°C–25°C during the experiment.</p>
<table id="undtbl7">
<thead>
<tr>
<th colspan="3">TOP agar</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sorbitol</td>
<td>1 M</td>
<td>45.5 g</td>
</tr>
<tr>
<td>SD base</td>
<td>n/a</td>
<td>6.72 g</td>
</tr>
<tr>
<td>-His (or -Ura)</td>
<td>n/a</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Bacto agar</td>
<td>n/a</td>
<td>7.5 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 250 mL</td>
</tr>
<tr>
<td colspan="3">Adjust the pH to 5.8 and autoclave 121°C 15min</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>12.5 mL /plate</b></td>
<td><b>250 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh on the morning of each transformation and place at 55°C in a water/bead bath until needed.</p>
<table id="undtbl8">
<thead>
<tr>
<th colspan="3">SD-Ura (or-His) plates</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>SD Base Agar</td>
<td>n/a</td>
<td>46.7 g</td>
</tr>
<tr>
<td>-His (or -Ura)</td>
<td>n/a</td>
<td>0.77 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td colspan="3">Adjust the pH to 5.8 and autoclave 121°C 15min</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store upside down at 4°C up to 2/3 weeks.</p>
<table id="undtbl9">
<thead>
<tr>
<th colspan="3">LB-chl<sup>R</sup><sub>12.5</sub> plates</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>LB Agar</td>
<td>n/a</td>
<td>35 g</td>
</tr>
<tr>
<td>Chloramphenicol (50 mg/mL)</td>
<td>12.5 μg/mL</td>
<td>250 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store upside down at 4°C up to 2/3 weeks.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec4.1">Concentration of ASFV particles and isolation of high molecular weight viral genomic DNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This step describes the concentration of viral particles from the serum of domestic pigs experimentally infected experimentally with ASFV via ultracentrifugation and the subsequent isolation of high molecular weight viral DNA (vDNA). Organ material from a field case of ASF in Western Georgia in June 2007 was obtained from Tinatin Onashvili and Cezar Machitidze, Laboratory of the Ministry of Agriculture (LMA), Tbilisi, Georgia. Spleen homogenate was used to infect six 6-week-old male specific pathogen-free Large White pigs by oro-nasal (three pigs) or intramuscular (three pigs) application. Sera were collected five days after infection for further processing in this protocol. On the fifth day after infection, the serum of ASFV- infected pigs contained ∼10<sup>8</sup> ASFV genome equivalents and was processed as described below. The same protocol can be applied for isolation of vDNA from cell-culture supernatants containing viral particles.</p>
<ol>
<li>Form a sucrose cushion by adding 2 mL of a 20% sucrose solution in a 13.2 mL ultra-clear centrifuge tube (Beckman Coulter, see <a href="#key-resources-table">key resources table</a>)</li>
<li>Overlay the sucrose with 7 mL of ASFV-infected pig serum by gently pipetting down the side of the tube</li>
<li>Carefully add phosphate buffer saline (PBS) (see <a href="#key-resources-table">key resources table</a>) up to 2–3 mm from the top of the tube.</li>
<li>Centrifuge at 50,000 <b>×</b> <i>g</i> for 90 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Familiarize yourself with the user manual and safety instructions before using the rotor and operating the ultracentrifuge.</div>
<ol start="5">
<li>Carefully discard the supernatant by pipetting without disturbing the pellet. Usually a discrete pale-white pellet is visible.</li>
<li>Resuspend the pellet in 200 μL of PBS.</li>
<li>Extract the virus DNA using the MagAttract HMW DNA Kit (Qiagen) according to manufacturer’s instructions (see <a href="https://www.qiagen.com/us/resources/resourcedetail?id=3e331b6c-0742-483b-b74f-a7be2d4c1116&amp;lang=en">here</a>).</li>
</ol>
<h3 id="sec4.2">Transformation-associated recombination (TAR) cloning of ASFV sub-genomic fragments</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day (+ 2 days of incubation after yeast transformation)</div>
<p>This section describes all the necessary steps required for the individual isolation of the different ASFV sub-genomic fragments in yeast. It encompasses the generation of the different TAR vectors carrying the appropriate hook sequences up to their individual co-transformation in yeast spheroplasts along with the previously isolated high-quality ASFV genomic DNA. Each of the five sub-genomic fragments has to be isolated individually but all the transformations can be performed in parallel.</p>
<ol start="8">
<li>Insertion of the overlapping regions (hooks) and I-<i>Sce</i>I restrictions sites in the pCC1BAC-Ura3 TAR vector
<ol type="a">
<li>pCC1BAC-Ura3 plasmid (10 ng/μL) was used as DNA template in the following PCR reaction.
<table id="undtbl10">
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount (for one reaction)</th>
</tr>
<tr>
<th>ddH<sub>2</sub>O</th>
<th>n/a</th>
<th>32 μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>5<b>×</b> PrimeSTAR DNA Buffer</td>
<td>1<b>×</b>
</td>
<td>10 μL</td>
</tr>
<tr>
<td>dNTP mixture (2.5 mM each)</td>
<td>200 μM each</td>
<td>4 μL</td>
</tr>
<tr>
<td>Forward primer (10 μM)</td>
<td>0.2 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Reverse primer (10 μM)</td>
<td>0.2 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>PrimeSTAR GXL DNA polymerase</td>
<td>1.25 U</td>
<td>1 μL</td>
</tr>
<tr>
<td>DNA template (TAR vector)</td>
<td>10 ng</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl11">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature (°C)</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>98</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98</td>
<td>10 s</td>
<td rowspan="3">30</td>
</tr>
<tr>
<td>Annealing</td>
<td>52</td>
<td>15 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68</td>
<td>10 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>68</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>10</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<p class="table-legend">PrimeSTAR GXL DNA polymerase was selected as it consistently resulted in specific amplicons. Alternative high-fidelity DNA polymerases such as the KOD Hot Start DNA polymerase (#71086, Merck) can also be used.</p>
</li>
<li>PCR products were separated and visualized on a 0.8% agarose gel containing RedSafe 1<b>×</b>, which is a substitute for ethidium bromide.</li>
<li>
<b>Optional (but recommended):</b> Add 1 μL of <i>Dpn</i>I restriction enzyme directly to the PCR mixture without prior treatment and incubate for 1 h at 37°C in a temperature-controlled heat block to digest template DNA. After 1 h, add another 1 μL of <i>Dpn</i>I and incubate for an extra hour.</li>
<li>
<i>Dpn</i>I-treated PCR products are purified using the High Pure PCR product purification kit (Roche) following manufacturer’s recommendations (<a href="https://www.sigmaaldrich.com/deepweb/assets/sigmaaldrich/product/documents/184/921/hppcrpkrobul.pdf">here</a>).</li>
<li>Purified PCR products are quantified individually by measuring the absorbance at 260 nm using the μCuvette® G1.0 of the Biospectrophotomer (Roche) and concentrations are adjusted to 100 ng/μL when possible.</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Yeast spheroplasts preparation
<ol type="a">
<li>Start an overnight culture of <i>S. cerevisiae</i> VL6-48N in 10 mL YPDA (pH 6.5) medium at 30°C under agitation (200 rpm).</li>
<li>The next morning, measure the OD<sub>600nm</sub> and use the appropriate volume to start a fresh 100-mL <i>S. cerevisiae</i> VL6-48N culture in YPDA with an initial OD<sub>600nm</sub> of ∼0.2.</li>
<li>When the OD<sub>600nm</sub> of this new culture reaches ∼2, collect yeast cells by centrifugation at 1,750 <b>×</b> <i>g</i> for 3 min, discard the supernatant and resuspend the pellet in 20 mL of 1M sorbitol. Incubate overnight at 4°C.</li>
<li>Collect the yeast cells by centrifugation at 1,750 <b>×</b> <i>g</i> for 3 min at 4°C. Resuspend the pellet in 10 mL of SPEM solution (see recipe) in a 50-mL sterile centrifuge tube.</li>
<li>Add 20 μL of ß-mercaptoethanol and 20 μL of Zymolyase®-100T solution (see recipe) and incubate at 30°C with gentle agitation (80 rpm) until spheroplasts are ready.
<div class="critical">
<span class="critical-title">Critical:</span> The formation of spheroplasts needs to be assessed by comparing the OD<sub>600nm</sub> measurements of a 1/10<sup>th</sup> dilution of the spheroplast solution in a 1M sorbitol solution (intact spheroplasts) versus in a 2% SDS solution (lysed spheroplasts). Spheroplasts are considered ready when the ratio between the two readings is comprised between 3 and 4. Ideally, OD<sub>600nm</sub> of 0.8 and 0.2 should be obtained when cells are diluted in 1M sorbitol and 2% SDS solutions, respectively.</div>
</li>
<li>When spheroplasts are ready, add immediately 40 mL of 1M sorbitol and mix gently by inversion.</li>
<li>Collect the yeast cells by centrifugation at 1,200 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Gently resuspend the pellet in 20 mL of 1M sorbitol using a 25-mL serological pipette. Add another 30 mL of 1M sorbitol and gently invert the tube 2–3 times.</li>
<li>Collect spheroplasts by centrifugation at 1,200 <b>×</b> <i>g</i> for 5 min and resuspend the pellet in 2 mL STC (see recipe). Incubate at room temperature for 10 min.</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Spheroplast transformation
<ol type="a">
<li>Mix 100 μL of yeast spheroplast with 5 μg of ASFV genomic DNA and 250 ng of the PCR-amplified pCC1BAC-Ura3 TAR vector containing the appropriate hooks (i.e., sharing overlaps with the ASFV sub-genomic fragment to clone). Incubate at room temperature for 10 min.
<div class="note">
<span class="note-title">Note:</span> One transformation is performed per tube, which corresponds to the individual TAR cloning of one ASFV sub-genomic fragments. In total, seven independent transformations need to be performed in parallel in order to individually cloned all ASFV sub-genomic fragments described here.</div>
</li>
<li>Add 0.5 mL of PEG solution (see recipe). Mix by rotating the tube very gently. Incubate at room temperature for 20 min.</li>
<li>Collect the cells at 2,500 <b>×</b> <i>g</i> for 5 min. Carefully remove the supernatant and resuspend the pellet in 700 μL SOS solution (see recipe). Incubate for 2 h at 30°C.</li>
<li>Mix the spheroplasts with 12 mL of melted TOP agar medium (see recipe), previously equilibrated at 55°C in a bead/water bath. Quickly pour the mixture onto Sorbitol-Ura plates (see recipe) and allow it to solidify for few seconds at room temperature.</li>
<li>Incubate at 30°C for 2 days.
<div class="note">
<span class="note-title">Note:</span> If no transformants are observed, see the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.1">problem 1</a>).</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec4.3">PCR-based screening of yeast transformants</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section describes the PCR-mediated identification of yeast transformants carrying the sub-genomic ASFV fragments of interest. <a href="#fig2">Figure 2</a> summarizes the different steps involved in this section. The first step consists in the replication of individual yeast colonies on a new selection plate to ensure adequate propagation. The first screening relies on the amplification of a short internal region of the desired fragment, which is performed on pools of eight yeast colonies to allow a large but rapid initial pre-screening. The same PCR is then carried out on all individual colonies of previously tested positive pools to identify individual positive yeast clones. The second and third PCRs target the junctions between the TAR vector and the ASFV sub-genomic fragments.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/985-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Flowchart describing the steps required for the PCR-based screening of yeast transformants</div>
<p>Yeast colonies obtained on Sorb-Ura plates are replicated individually onto a fresh SD-Ura plates. A first GC extraction is performed on eight patches pooled together into a single tube (“Pool” GC preparation). New GC extractions are then carried out on each of the eight patches from a positive pool (“Individual” GC preparations). These steps are repeated for every TAR-cloned sub-genomic fragment.</p>
</figcaption>
</figure>
<ol start="11">
<li>Pick isolated colonies from the original Sorbitol-Ura plates and patch each of them onto fresh SD-Ura plates. A total of 32 colonies are usually patched per plate using a 32-square grid PetriSticker™ (Sigma-Aldrich). Incubate at 30°C for 24 h.</li>
<li>Extract yeast (extra)-chromosomal DNA using the glass bead Chelex 100 preparation (GC prep) method as described elsewhere (<a href="#bib1">Blount et al., 2016</a>).
<ol type="a">
<li>Collect a ∼2-mm<sup>2</sup> surface of a yeast patch with the tip of a 20-uL sterile pipette tip and placed into an 1.5mL-Eppendorf tube containing 100 μL of a 5% Chelex 100 solution (resuspended in deionized water) and add acid washed glass beads to half total sample volume.</li>
<li>Repeat step a) for 7 additional patches (in order to make a pool of 8 colonies)</li>
<li>Vortex at 1,400 rpm for 4 min.</li>
<li>Incubate for 2 min at 99°C and centrifuge at 18,000 <b>×</b> <i>g</i> for 1 min.</li>
<li>Transfer 30–40 μL of the supernatant to a clean tube. Store at 4°C for 24–48 h or at −20°C for a longer period.</li>
<li>Use 1 μL of the supernatant as DNA template in each of the subsequent screening PCRs</li>
</ol>
</li>
</ol>
<ol start="13">
<li>PCR amplifications of i) the internal genomic DNA region and ii) the two junctions between the TAR vector and ASFV gDNA fragment are performed. Amplicons obtained from pooled clones and individual clones are displayed in <a href="#fig3">Figure 3</a>.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/985-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. PCR-based screening and identification of yeast clones carrying an ASFV sub-genomic fragment</div>
<p>(A) Agarose gels showing the amplification products of the internal DNA regions located on ASFV sub-genomic fragment 3 (left) and sub-genomic fragment 4 (right). PCRs were carried on pools (P) of eight colonies. Pools indicated in red are considered positives.</p>
<p>(B) Amplification products obtained when the 5′ and 3′ junctions between the TAR vector and each of the two sub-genomic fragments were assessed. Plus and minus signs indicate the positive (ASFV gDNA) and negative (water) controls for each of the PCRs, respectively. GeneRuler 100 bp Plus DNA ladder (Thermo Scientific) was used as marker (M).</p>
</figcaption>
</figure>
<ol type="a">
<li>PCR reactions are performed using the GoTaq® G2 Green Master mix (Promega) according to manufacturer’s recommendations (<a href="https://ch.promega.com/resources/protocols/product-information-sheets/g/gotaq-g2-green-master-mix-protocol/">see here</a>). The set-up for each reaction is presented in the following table:
<table id="undtbl12">
<thead>
<tr>
<th colspan="2">Set-up PCR reaction (for one reaction) (GoTaq G2 green master mix)</th>
</tr>
</thead>
<tbody>
<tr>
<td>GoTaq® G2 Green Master Mix, 2<b>×</b>
</td>
<td>12.5μL</td>
</tr>
<tr>
<td>upstream primer, 10μM</td>
<td>2 μL</td>
</tr>
<tr>
<td>downstream primer, 10μM</td>
<td>2 μL</td>
</tr>
<tr>
<td>DNA template (GC prep, step 12)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td>7.5 μL</td>
</tr>
<tr>
<td>Total</td>
<td>25 μL</td>
</tr>
</tbody>
</table>
</li>
<li>PCRs are run following the cycling conditions described below:
<table id="undtbl13">
<thead>
<tr>
<th colspan="4">PCR cycling conditions (GoTaq G2 green master mix)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Steps</td>
<td>Temperature</td>
<td>Time</td>
<td>Cycles</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>95</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95</td>
<td>15 s</td>
<td rowspan="3">30</td>
</tr>
<tr>
<td>Annealing</td>
<td>50</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>10</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<div class="optional">
<span class="optional-title">Optional:</span> Sanger sequencing of the different PCR products is recommended to confirm the identity of the ASFV sub-genomic fragment cloned and that the junctions do not contain any SNPs that would be detrimental for the subsequent steps.</div>
</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Select one or two yeast clones containing the desired fragment for each of the five TAR cloning experiments for further use. Save these yeast clones at −80°C in Roti®Store yeast cryo vials following manufacturer’s conditions (<a href="https://www.carlroth.com/ch/en/cryo-conservation/rotistore-yeast-cryo-vials/p/x983.1">see here</a>).</li>
</ol>
<p>If no positive yeast transformants are obtained, see <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.3">problem 2</a>).</p>
<h3 id="sec4.4">Multicopy plasmid induction in <i>E. coli</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section describes the extraction of the yeast artificial chromosomes (YACs) containing ASFV sub-genomic fragments from yeast followed by their subsequent transformation into <i>E. coli</i>. The multicopy induction system present on the pCC1BAC-Ura3 TAR vector is then used to purify larger amounts of plasmids/YACs required for the final reassembly in yeast.</p>
<ol start="15">
<li>YAC extraction from yeast based on the QIAGEN mini kit
<ol type="a">
<li>Resuspend ∼half of the remaining yeast patch from positive clones (from step 16) into 250 μL of P1 solution (included in the QIAGEN mini kit), 25 μL of Zymolyase solution (see recipe) and 2.5 μL of β-mercaptoethanol. Vortex for 10 s and incubate the mixture at 37°C for 30 min.</li>
<li>Vortex for another 10 s and incubate for 30 min at 37°C.</li>
<li>Add 250 μL of P2 solution and follow all the next steps according to manufacturer’s instructions (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-plasmid-kits/?catno=12125">here</a>) up to the elution step.</li>
<li>For the elution step, add 30 μL of elution buffer into the column and let sit for 5 min. Place the column into a fresh microcentrifuge tube and centrifuge for 1 min at 18,000 <b>×</b> <i>g</i> at room temperature.</li>
</ol>
</li>
<li>
<i>E. coli</i> transformation using Transformax™ EPI300™ electrocompetent cells
<ol type="a">
<li>Chill 1-mm electroporation cuvettes and sterile Eppendorf tubes on ice for 15 min (one cuvette and one tube per transformation).</li>
<li>Add 3 μL of the previously extracted plasmid solution to 35 μL of freshly thawed Transformax™ EPI300™ electrocompetent <i>E. coli</i> cells in a pre-chilled tube.</li>
<li>Transfer the mixture to the pre-chilled 1-mm electroporation cuvette and electroporate following manufacturer’s instructions. In our case, we used the Gene Pulser Electroporation system (Bio-Rad) with the following parameters: 2 kV; 25 μFD and 200 Ohms.</li>
<li>Add immediately 950 μL of SOC medium and incubate for 1 h at 37°C under agitation (200 rpm).</li>
<li>Transfer the mixture to a fresh sterile microcentrifuge tube and centrifuge for 30 s at 18,000 <b>×</b> <i>g</i>, discard the supernatant.</li>
<li>Resuspend the pellet in 200 μL of fresh SOC medium and plate on selective medium (LB-Agar plates containing 12.5 μg/mL chloramphenicol). Transformants are usually observed after 24 h of incubation at 37°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> An additional 24 hours incubation might ease the picking of the <i>E. coli</i> colonies in some cases.</div>
<ol start="17">
<li>PCR verification of the <i>E. coli</i> clones.
<ol type="a">
<li>Resuspend one <i>E. coli</i> colony per ASFV sub-genomic fragment in a fresh microcentrifuge tube containing 10 μL of deionized water.</li>
<li>Use 1 μL of the mixture as DNA template for each of the two junction PCRs as previously described in step 18.</li>
<li>Use the remaining 8 μL to start a 12-mL culture in LB-Chl<sup>R</sup><sub>12.5</sub> for each positive <i>E. coli</i> clone tested. Incubate for 12–14 h at 37°C under agitation (220 rpm).</li>
<li>Save one or two <i>E. coli</i> clones at −80°C using 750 μL of the overnight cultures mixed with glycerol (15% final concentration).</li>
</ol>
</li>
</ol>
<ol start="18">
<li>Multicopy induction and plasmid purification from <i>E. coli</i> clones
<ol type="a">
<li>Use the remaining 10 mL of the overnight culture (from step 22) to start a 100-mL LB-Chl<sup>R</sup><sub>12.5</sub> culture into a 250-mL Erlenmeyer baffled flask to ensure proper aeration.</li>
<li>Add 100 μL of CopyControl™ induction solution to the culture and incubate for 5 h at 37°C under agitation (220 rpm).
<div class="critical">
<span class="critical-title">Critical:</span> Proper aeration of the culture is extremely important to ensure high yields of plasmid after purification.</div>
</li>
<li>Centrifuge the culture for 20 min at 4,250 <b>×</b> <i>g</i> at room temperature. Pellets can be stored at −20°C.</li>
<li>Plasmids were extracted using the QIAGEN midiprep plasmid purification kit following manufacturer’s instructions (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-plasmid-kits/?catno=12125">here</a>).</li>
<li>Quantify DNA concentration using a biospectrophotometer.</li>
</ol>
</li>
</ol>
<h3 id="sec4.5">Full-length reassembly of ASFV clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day (+2 days after yeast transformation)</div>
<p>This section describes the reconstruction of full-length ASFV genomes in yeast. It consists in the co-transformation of <i>S. cerevisiae</i> with a mixture of precipitated DNA containing all five overlapping ASFV sub-genomic fragments as well as the two chemically-synthetized fragments 1 and 2. This section also contains the final screening step of the reconstructed ASFV genomes confirming the presence of all the junctions between the seven different ASFV sub-genomic fragments.</p>
<ol start="19">
<li>Gel purification of the two synthetic fragments 1 and 2
<ol type="a">
<li>Five micrograms of the pUC57-Fragment1 plasmid were <i>Sma</i>I-digested in five 50-μL total reaction volume each containing 1 μg of DNA. Digestions were performed following manufacturer’s instructions (<a href="http://nebcloner.neb.com/">NEB</a>). The same protocol was applied to the digestion of the pUC57-Fragment2 plasmid.</li>
<li>Digested products were loaded on a 1% TAE agarose gel containing RedSafe™ 1<b>×</b>. Electrophoresis was carried out for 1 h at 50 V (3.85 V/cm).</li>
<li>The DNA bands corresponding to the linearized synthetic constructs were excised from the gel using an X-tracta gel extractor tool (Sigma-Aldrich) after short exposure to blue light using an LED transilluminator (MaestroGen).
<div class="note">
<span class="note-title">Note:</span> Blue light should be preferred over ultraviolet light to avoid DNA damage during the purification process</div>
</li>
<li>The excised gel band was placed in a 1.5-mL microcentrifuge tube and DNA was eluted in 30 μL final volume using the QIAQuick gel extraction kit following manufacturer’s instructions (<a href="https://www.qiagen.com/us/shop/pcr/qiaquick-gel-extraction-kit/">QIAGEN</a>).</li>
<li>Quantify DNA concentrations using a biospectrophotometer. One hundred fmoles of each of the <i>Sma</i>I-digested Fragment1 and Fragment2 are used for step 26.</li>
</ol>
</li>
</ol>
<ol start="20">
<li>Linearization of the YACs containing the different ASFV sub-genomic fragments
<ol type="a">
<li>Digest 50 fmoles of each of the ASFV sub-genomic fragments individually by I-<i>Sce</i>I following manufacturer’s instructions (<a href="http://nebcloner.neb.com/">here</a>). If DNA concentration is too low, multiple 50-μL reactions can be performed in parallel.</li>
<li>After 1 h restriction time, add 1 μL of I-<i>Sce</i>I to the reaction and incubate for an extra hour.</li>
</ol>
</li>
<li>Precipitation of DNA prior yeast transformation
<ol type="a">
<li>All digested ASFV fragments (synthetic and TAR-cloned ones) are pooled together in a microcentrifuge tube.</li>
<li>The total volume (V<sub>DNA</sub>) of all digested ASFV fragments is calculated. Digested ASFV fragments is precipitated in a solution containing 1/10<sup>th</sup> V<sub>DNA</sub> of isopropanol; 1/10<sup>th</sup> V<sub>DNA</sub> of 3M NaAc; 1 μL of Glycoblue™ coprecipitant (ThermoFisher) and 1 μL of tRNA (Sigma-Aldrich).</li>
<li>Centrifuge the mixture at 12,500 <b>×</b> <i>g</i> for 30 min at 4°C and discard the supernatant.</li>
<li>Resuspend the pellet in 700 μL of 70% ethanol solution.</li>
<li>Centrifuge at 12,500 <b>×</b> <i>g</i> for 15 min at 4°C and discard the supernatant.</li>
<li>Resuspend the pellet in 30 μL of TE buffer (10 mM Tris-HCl pH8; 1 mM EDTA). Store at 4°C if used immediately or at −20°C for a longer period.</li>
</ol>
</li>
<li>Yeast spheroplast co-transformation with all ASFV sub-genomic fragments and final pCC1BAC-His3 TAR vector
<ol type="a">
<li>Yeast spheroplasts were prepared as previously described (step 14).</li>
<li>Mix yeast spheroplast with 250 ng of PCR-amplified pCC1BAC-His3 and the 30 μL of all seven precipitated ASFV fragments.</li>
<li>The following steps were performed as previously described in step 15 with the exception that yeast cells were plated on Sorbitol-His plates instead of Sorbitol-Ura. Transformants can be observed after 24 h and colonies are big enough to be picked after 48 h.</li>
</ol>
</li>
</ol>
<ol start="23">
<li>Screening of yeast transformants using multiplex PCR
<ol type="a">
<li>Patch yeast colonies onto a fresh SD-His plate.</li>
<li>Extract yeast DNA using the GC-prep method as previously described (step 17)</li>
<li>Use 1 μL as DNA template and set up two independent multiplex PCR reactions for individual colonies using the Qiagen® Multiplex PCR kit (Qiagen) following manufacturer’s instructions (<a href="https://www.qiagen.com/us/shop/pcr/qiagen-multiplex-pcr-kit/">here</a>). Expected profiles for ASFV full-length clones are presented in <a href="#fig4">Figure 4</a>. All primers included in each multiplex PCR can be found in the primer table here. The set-up for each reaction is presented in the following table:
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/985-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Identification of positive full-length clones using multiplex PCRs screening</div>
<p>Two independent multiplex PCRs were performed on full-length assembled ASFV constructs with two sets of primers in order to ensure the presence of all junctions between the different ASFV sub-genomic fragments. Amplification profiles expected for the multiplex Set1 (top) and multiplex Set2 (bottom) can be observed for positive clones (in red). Expected sizes of the PCR products are indicated on the left side of the gels. GeneRuler 100 bp Plus DNA ladder (Thermo Scientific) was used as marker (M).</p>
</figcaption>
</figure>
<table id="undtbl14">
<thead>
<tr>
<th colspan="2">Set-up PCR reaction (for one reaction) (multiplex PCR kit Qiagen)</th>
</tr>
</thead>
<tbody>
<tr>
<td>2<b>×</b> QIAGEN Multiplex PCR Master Mix</td>
<td>12.5 μL</td>
</tr>
<tr>
<td>10<b>×</b> primer mix, 2 μM each primer</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>RNase-free water</td>
<td>9 μL</td>
</tr>
<tr>
<td>DNA template (GC prep, step 23b)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Total</td>
<td>25 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Cycling conditions for the two PCRs are described in the table below:
<table id="undtbl15">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>95</td>
<td>15 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95</td>
<td>1 min</td>
<td rowspan="3">35</td>
</tr>
<tr>
<td>Annealing</td>
<td>50</td>
<td>2.5 min</td>
</tr>
<tr>
<td>Extension</td>
<td>72</td>
<td>1.5 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>68</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>10</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
</ol>
<ol start="24">
<li>Select one or two yeast positive transformants for further use. Biobank these yeast clones at −80°C in Roti®Store yeast cryo vials following manufacturer’s conditions (<a href="https://www.carlroth.com/ch/en/cryo-conservation/rotistore-yeast-cryo-vials/p/x983.1">see here</a>).</li>
</ol>
<p>If no positive yeast transformants are obtained, see <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.9">problem 5</a>).</p>
<ol start="25">
<li>Large-scale preparation of full-length ASFV genomic DNA
<ol type="a">
<li>YAC extraction from yeast
<ol type="i">
<li>Start a 500-mL culture of <i>S. cerevisiae</i> containing full-length ASFV construct in SD-His at 30°C under agitation in a 2 L baffled Erlenmeyer flask.</li>
<li>When OD<sub>600nm</sub> reaches ∼2, centrifuge the culture at 4,250 <b>×</b> <i>g</i> for 20 min at 4°C.</li>
<li>Resuspend the pellet in a solution containing 20 mL of P1 solution (included in the QIAGEN® Large-construct kit), 2 mL of Zymolyase solution (see recipe) and 200 μL of β-mercaptoethanol. Vortex for 30 s and incubate the mixture at 37°C for 1 h.</li>
<li>Vortex for another 30 s and incubate for another 1 h min at 37°C.</li>
<li>DNA extraction is performed using the Qiagen® Large-construct kit following manufacturer’s instructions (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-large-construct-kit/">here</a>)</li>
</ol>
</li>
<li>Plasmid extraction from <i>E. coli</i>
<ol type="i">
<li>Repeat steps 20 and 21 with an <i>S. cerevisiae</i> clone carrying a full-length ASFV construct.</li>
<li>Proceed to DNA extraction using the Qiagen® Large-construct kit starting with a 300-mL <i>E. coli</i> culture following manufacturer’s instructions (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-large-construct-kit/">here</a>).</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Alternatively, agarose plugs containing intact ASFV chromosome-sized DNA can be prepared following manufacturer’s instructions (<a href="https://www.bio-rad.com/en-ch/product/chef-genomic-dna-plug-kits?ID=21f77550-a849-4d32-b90f-c9428fc7b425">here</a>). Such plugs can be prepared from either <i>E. coli</i> or yeast cultures using the CHEF Bacterial Genomic DNA plug kit (Bio-Rad, 1703592) or CHEF Yeast Genomic DNA plug kit (Bio-Rad, 1703593), respectively.</div>
<h3 id="sec4.6">Genetic engineering of ASFV clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This section describes the necessary steps for the engineering of the ASFV genome using the previously isolated sub-genomic fragments. Here, we substituted one non-essential gene in Fragment 4 (namely the C962R gene; <a href="#fig5">Figure 5</a>, in dark gray) (<a href="#bib16">Ramirez-Medina et al., 2020</a>) and replaced it by chemically-synthetized DNA cassettes carrying reporter genes as a proof of concept.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/985-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Design for the replacement of the ASFV C962R gene with synthetic DNA cassettes carrying the eGFP or secNLuc reporter genes</div>
<p>The ASFV sub-genomic fragment 4 excluding the C962R gene (Fragment 4ΔC962R) was first isolated using TAR cloning. Two DNA cassettes, namely the P_Pol-eGFP_C962R-int or P_Pol-secNLuc_C962R-int, were chemically synthesized and cloned in the pUC57 plasmid. They carry either the eGFP and secreted Luciferease reporter genes and contain overlaps with their neighboring fragments during genome reassembly, namely Fragment 4ΔC962R and Fragment 5. Asterisks indicate the presence of <i>Sma</i>I restriction sites used for linearization. All genetic features included in the synthetic DNA cassettes are detailed in step 31.</p>
</figcaption>
</figure>
<ol start="26">
<li>Design of the chemically-synthetized DNA cassettes carrying the reporter genes
<ol type="a">
<li>
<b><i>Sma</i>I restrictions sites (indicated with asterisks in</b> <a href="#fig5">Figure 5</a><b>):</b> Two <i>Sma</i>I restrictions sites were added at the 5′ and 3′ ends of the DNA cassettes to ensure linearization and purification from the pUC57 backbone plasmid.</li>
<li>
<b>5′-overlap (</b><a href="#fig5">Figure 5</a><b>, in light gray):</b> a region of 250 bp, overlapping the ASFV genome upstream of the C962R coding sequence included in the ASFV sub-genomic fragment 4, was added to ensure legitimate recombination in yeast.</li>
<li>
<b>Promoter region (</b><a href="#fig5">Figure 5</a><b>, in yellow):</b> the promoter region of the DNA polymerase (G1211R) of the ASFV Armenia isolate was used as previously reported (<a href="#bib15">Portugal et al., 2017</a>).</li>
<li>
<b>Reporter sequences:</b> the respective 720-bp and 600-bp nucleotide sequence of the enhanced GFP (eGFP) (<a href="#fig5">Figure 5</a>, in green) and secreted Luciferase (secNLuc) (<a href="#fig5">Figure5</a>, in pink) obtained from Promega were added in frame with the promoter region.</li>
<li>
<b>Terminator sequence (</b><a href="#fig5">Figure 5</a><b>, in red):</b> the 281-bp terminator sequence of the thymidine kinase (TK, L60) was added after the reporter sequences in both constructs (<a href="#bib15">Portugal et al., 2017</a>).</li>
<li>
<b>3′-overlap (</b><a href="#fig5">Figure 5</a><b>, in orange):</b> a 387-bp region consisting of the last 65 bp of the 3′UTR of the C962R open reading frame (<a href="#bib4">Cackett et al., 2020</a>). The rest of the sequence, overlapping with the ASFV sub-genomic fragment 5, was added to ensure legitimate recombination in yeast.</li>
</ol>
</li>
<li>Isolation of the ASFV sub-genomic fragment 4 deleted for the C962R gene
<ol type="a">
<li>Generation of pCC1BAC-Ura3 with appropriate hooks. A PCR amplification of the pCC1BAC-Ura3 TAR vector was carried out with primers ASFTARhook4-R/F2 as reported in step 8.</li>
<li>Yeast clones carrying the ASFV sub-genomic fragment 4 deleted for C962R were obtained using TAR cloning as reported in steps 9 and 10.</li>
<li>Repeat steps 11 to 18 in order to obtain purified YACs containing the ASFV sub-genomic fragment 4 deleted for C962R.</li>
</ol>
</li>
<li>Full-length reassembly of genetically engineered ASFV genomes carrying eGFP or secNLuc reporter genes
<ol type="a">
<li>Repeat steps 24 to 30 including one of the two chemically-synthetized DNA cassettes, namely pUC57-P_Pol-secNluc _C962R-int or pUC57-P_Pol-eGFP_C962R-int, previously gel purified along with all the required ASFV sub-genomic fragments.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The main outcome of this protocol is to provide a fast and flexible genetic engineering platform for the modification of genomes derived from large DNA viruses, such as the ASFV, to the scientific community. All full-length constructs (wild-type full-length ASFV construct and its respective eGFP and secNLuc versions), as well as all the individual ASFV sub-genomic fragments described in the protocol will be made available to the scientific community upon request.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>One current limitation of the system is the need to clone all sub-genomic fragments in <i>E. coli</i> prior to the final reassembly in yeast. This is due to the difficulties encountered when large YACs have to be purified directly from <i>S. cerevisiae</i>. Indeed, YACs such as the ones described here are maintained as single copy plasmid in yeast. Along with the purification issues due to yeast genomic contamination, it is technically challenging to produce large quantities of pure YACs from yeast cultures. Propagation of the constructs in <i>E. coli</i> can be associated with SNPs introductions or stability issues. Efforts are ongoing to circumvent these difficulties so that a “yeast-only” system can be achieved.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No colonies are obtained after yeast spheroplast transformation (step 15)</p>
<h3 id="sec7.2">Potential solutions</h3>
<p>Make sure that the ratio observed in step 9.e during spheroplast formation is correct. A significantly different ratio will affect the competency of the yeast preparation. If needed, a second culture can be started and used as backup in case the first culture fails to provide the appropriate ratio.</p>
<p>Check that the proper auxotrophic marker was used to prepare the agar plates.</p>
<p>Ensure that the PEG solution has been freshly prepared and that the pH of the all solutions is adequate.</p>
<p>The amount of DNA to use for the yeast transformation should not exceed 10%–20% of the total reaction volume. A large DNA volume will negatively impact the transformation efficiency.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>No positive yeast clones obtained after the PCR-based screening (step 18)</p>
<h3 id="sec7.4">Potential solutions</h3>
<p>Always confirm that the TAR vector has been specifically amplified. Always include a negative control in each transformation experiment consisting of the PCR-amplified TAR vector only. If a high number of colonies is present on the negative control plate, consider doing another round of <i>Dpn</i>I treatment or gel-purify your TAR vector before use.</p>
<p>Ensure that the primers used to PCR amplify the TAR vector have been designed correctly. As these primers are typically ∼80-nt long, it is generally recommended to add an extra PAGE purification step after synthesis.</p>
<p>The sequences used for the overlapping regions should be unique and should not consist of highly repetitive regions that can trigger illegitimate recombination during TAR cloning.</p>
<p>Check the quality of the purified viral DNA. This is of great importance, especially when large sub-genomic DNA fragments are to be TAR-cloned.</p>
<p>Ensure that the GC extractions as well as the designed primers do not alter the amplification of the targeted internal regions by running appropriate PCR controls including a positive one using ASFV genomic DNA as template.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Low concentration of DNA after induction in <i>E. coli</i> (step 23)</p>
<h3 id="sec7.6">Potential solutions</h3>
<p>Make sure that the aeration of the <i>E. coli</i> culture is optimal. To do so, make sure to use baffled Erlenmeyer flasks able to contain at least 5 times the volume of culture actually used.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Incorrect DNA profiles after I-<i>Sce</i>I digestion (step 25)</p>
<h3 id="sec7.8">Potential solutions</h3>
<p>Check for the presence and correct sequence of each the I-<i>Sce</i>I restriction sites by Sanger sequencing after PCR amplification. If problems are observed, use a different <i>E. coli</i> clone.</p>
<p>Determine the presence of possible genomic DNA contamination in your samples by loading each of them on a 0.8% agarose gel (as shown in <a href="#fig6">Figure 6</a>). Genomic DNA contamination might artificially decrease the concentration of plasmid DNA in your samples. If so, start a new plasmid DNA preparation and reduce the initial volume of the culture to 50 mL.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/985-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Restrictions profiles of ASFV subgenomic fragments extracted from yeast or <i>E. coli</i>
</div>
<p>Agarose gel showing plasmid DNA extracted from <i>S. cerevisiae</i> (YAC) or <i>E. coli</i> (BAC) undigested (-) or digested with I-<i>Sce</i>I (+) for two different ASFV sub-genomic fragments. Red asterisks indicated the presence of genomic DNA in the plasmid extractions, which is particularly present and problematic when plasmids are extracted from yeast cultures. The expected digestion pattern is shown by the two red arrows corresponding to the linearized ASFV sub-genomic fragments (top) and TAR vector (bottom), respectively, and only obtained with <i>E. coli</i> derived plasmid DNA. The Lambda DNA/HindIII ladder (ThermoFischer) (Thermo Scientific) was used as marker (M).</p>
</figcaption>
</figure>
<p>Mechanical shearing might be observed on a 0.8% agarose gel as a DNA smear. Make sure to slowly and carefully mix your plasmid DNA preparations by pipetting at little as possible or use wide-bore tips if necessary.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>No positive full-length clones after screening using the multiplex PCRs (step 28)</p>
<h3 id="sec7.10">Potential solutions</h3>
<p>Make sure that all the PCRs included in the multiplex PCR reactions are working when performed individually using the genomic DNA as DNA template.</p>
<p>Make sure that each sub-genomic fragment overlap its two neighboring fragments by at least 50 bp.</p>
<p>Make sure that all sub-genomic fragments were included in the pool of DNA transformed in yeast.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Blount, B.A., Driessen, M.R.M., and Ellis, T. (2016). GC preps: Fast and easy extraction of stable yeast genomic DNA. Sci. Rep. <i>6</i>, 26863. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref1">View at publisher</a></p>
<p id="bib2">Borca, M.V., Holinka, L.G., Berggren, K.A., and Gladue, D.P. (2018). CRISPR-Cas9, a tool to efficiently increase the development of recombinant African swine fever viruses. Sci. Rep. <i>8</i>, 3154. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref2">View at publisher</a></p>
<p id="bib3">Borca, M.V., Rai, A., Ramirez-Medina, E., Silva, E., Velazquez-Salinas, L., Vuono, E., Pruitt, S., Espinoza, N., and Gladue, D.P. (2021). A cell culture-adapted vaccine virus against the current pandemic African swine fever virus strain. J. Virol. <i>95</i>, e0012321. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref3">View at publisher</a></p>
<p id="bib4">Cackett, G., Matelska, D., Sýkora, M., Portugal, R., Malecki, M., Bähler, J., Dixon, L., and Werner, F. (2020). The African swine fever virus transcriptome. J. Virol. <i>94</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref4">View at publisher</a></p>
<p id="bib5">Chapman, D.A.G., Darby, A.C., da Silva, M., Upton, C., Radford, A.D., and Dixon, L.K. (2011). Genomic analysis of highly virulent Georgia 2007/1 isolate of African swine fever virus. Emerg. Infect. Dis. <i>17</i>, 599-605. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref5">View at publisher</a></p>
<p id="bib6">Dixon, L.K., Chapman, D.A.G., Netherton, C.L., and Upton, C. (2013). African swine fever virus replication and genomics. Virus Res. <i>173</i>, 3-14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref6">View at publisher</a></p>
<p id="bib7">Gallardo, C., Sánchez, E.G., Pérez-Núñez, D., Nogal, M., de León, P., Carrascosa, Á.L., Nieto, R., Soler, A., Arias, M.L., and Revilla, Y. (2018). African swine fever virus (ASFV) protection mediated by NH/P68 and NH/P68 recombinant live-attenuated viruses. Vaccine <i>36</i>, 2694-2704. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref7">View at publisher</a></p>
<p id="bib8">Gibson, D.G., Glass, J.I., Lartigue, C., Noskov, V.N., Chuang, R.-Y., Algire, M.A., Benders, G.A., Montague, M.G., Ma, L., Moodie, M.M., et al. (2010). Creation of a bacterial cell controlled by a chemically synthesized genome. Science <i>329</i>, 52-56. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref8">View at publisher</a></p>
<p id="bib9">Krug, P.W., Holinka, L.G., O’Donnell, V., Reese, B., Sanford, B., Fernandez-Sainz, I., Gladue, D.P., Arzt, J., Rodriguez, L., Risatti, G.R., et al. (2015). The progressive adaptation of a georgian isolate of African swine fever virus to vero cells leads to a gradual attenuation of virulence in swine corresponding to major modifications of the viral genome. J. Virol. <i>89</i>, 2324-2332. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref9">View at publisher</a></p>
<p id="bib10">Larionov, V., Kouprina, N., Graves, J., Chen, X.N., Korenberg, J.R., and Resnick, M.A. (1996). Specific cloning of human DNA as yeast artificial chromosomes by transformation-associated recombination. Proc. Natl. Acad. Sci. U S A <i>93</i>, 491-496. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref10">View at publisher</a></p>
<p id="bib11">Monteagudo, P.L., Lacasta, A., López, E., Bosch, L., Collado, J., Pina-Pedrero, S., Correa-Fiz, F., Accensi, F., Navas, M.J., Vidal, E., et al. (2017). BA71ΔCD2: a new recombinant live attenuated African swine fever virus with cross-protective capabilities. J. Virol. <i>91</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref11">View at publisher</a></p>
<p id="bib12">Noskov, V., Kouprina, N., Leem, S.-H., Koriabine, M., Barrett, J.C., and Larionov, V. (2002). A genetic system for direct selection of gene-positive clones during recombinational cloning in yeast. Nucleic Acids Res. <i>30</i>, e8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref12">View at publisher</a></p>
<p id="bib13">O’Donnell, V., Holinka, L.G., Gladue, D.P., Sanford, B., Krug, P.W., Lu, X., Arzt, J., Reese, B., Carrillo, C., Risatti, G.R., et al. (2015). African swine fever virus georgia isolate harboring deletions of MGF360 and MGF505 genes is attenuated in swine and confers protection against challenge with virulent parental virus. J. Virol. <i>89</i>, 6048. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref13">View at publisher</a></p>
<p id="bib14">Oldfield, L.M., Grzesik, P., Voorhies, A.A., Alperovich, N., MacMath, D., Najera, C.D., Chandra, D.S., Prasad, S., Noskov, V.N., Montague, M.G., et al. (2017). Genome-wide engineering of an infectious clone of herpes simplex virus type 1 using synthetic genomics assembly methods. Proc. Natl. Acad. Sci. U S A <i>114</i>, E8885-E8894. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref14">View at publisher</a></p>
<p id="bib15">Portugal, R.S., Bauer, A., and Keil, G.M. (2017). Selection of differently temporally regulated African swine fever virus promoters with variable expression activities and their application for transient and recombinant virus mediated gene expression. Virology <i>508</i>, 70-80. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref15">View at publisher</a></p>
<p id="bib16">Ramirez-Medina, E., Ramirez-Medina, E., Vuono, E.A., Vuono, E.A., Rai, A., Rai, A., Pruitt, S., Pruitt, S., Silva, E., Silva, E., et al. (2020). The C962R ORF of African swine fever strain georgia is non-essential and not required for virulence in swine. Viruses <i>12</i>, 676. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref16">View at publisher</a></p>
<p id="bib17">Sánchez-Cordón, P.J., Montoya, M., Reis, A.L., and Dixon, L.K. (2018). African swine fever: A re-emerging viral disease threatening the global pig industry. Vet. J. <i>233</i>, 41-48. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref17">View at publisher</a></p>
<p id="bib18">Sánchez, E.G., Pérez-Núñez, D., and Revilla, Y. (2019). Development of vaccines against African swine fever virus. Virus Res <i>265</i>, 150-155. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref18">View at publisher</a></p>
<p id="bib19">Thao, T.T.N., Labroussaa, F., Ebert, N., V’kovski, P., Stalder, H., Portmann, J., Kelly, J., Steiner, S., Holwerda, M., Kratzel, A., et al. (2020). Rapid reconstruction of SARS-CoV-2 using a synthetic genomics platform. Nature <i>582</i>, 1-8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref19">View at publisher</a></p>
<p id="bib20">Zhu, Z., Chen, H., Liu, L., Cao, Y., Jiang, T., Zou, Y., and Peng, Y. (2020). Classification and characterization of multigene family proteins of African swine fever viruses. Brief. Bioinform <i>22</i>, bbaa380. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00509-8/sref20">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to Dr. Fabien Labroussaa. Email: <a href="mailto:fabien.labroussaa@vetsuisse.unibe.ch">fabien.labroussaa@vetsuisse.unibe.ch</a></p>
<h4>Materials availability</h4>
<p>All plasmids and resources generated during this study will be made available upon request. Recipients will be asked for proofs concerning their capacities to work with BSL-2 and BSL-3 agents before any shipment of material.</p>
<h4>Data and code availability</h4>
<p>The genomic sequence of the abovementioned ASFV isolate was determined using Illumina sequencing performed at the Next-Generation Sequencing platform (University of Bern). The sequence is identical to the isolate Georgia 2007/1 (GenBank: FR682468.2) previously published (<a href="#bib5">Chapman et al., 2011</a>).</p>
<p>This protocol does not report original code.</p>
<h3>Acknowledgments</h3>
<p>We thank Tinatin Onashvili and Cezar Machitidze (Laboratory of the Ministry of Agriculture, Tbilisi, Georgia) for kindly providing the original ASFV-infected organ material used in this study. Sequencing has been done at the Next Generation Sequencing Platform at the University of Bern (Tosso Leeb and Pamela Nicholson). The cloning and genome assembly-related works, as well as F.L., V.C., H.A., and J.J., were funded by intramural funds of the <a href="https://doi.org/10.13039/100009068">University of Bern</a>. The experimental infection, isolation of gDNA, next generation sequencing, as well as K.M., L.M., and N.R. were funded by the <a href="https://doi.org/10.13039/501100006454">Swiss Federal Food Safety and Veterinary Office</a> (Grant No. 1.19.02).</p>
<h3>Author contributions</h3>
<p>F.L., N.R., and J.J. designed the study, F.L., K.M., and V.C. performed the experiments, M.L. helped in the design of the constructs, and H.A. analyzed the sequencing results. F.L. drafted the protocolincluding the figures. All authors reviewed and approved the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>