<article>
<section>
<h1 id="header">Quantification of drug-induced fractional killing using high-throughput microscopy</h1>
<p><time datetime="2021-01-22">Published: January 22, 2021</time></p>
<p>Zintis Inde,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Jason Rodencal,<sup><a href="#aff1">1</a></sup> and Scott J. Dixon<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biology, Stanford University, 327 Campus Dr, Stanford, CA 94305, USA</p>
<p id="fn1"><sup>2</sup>Present address: Department of Environmental Health, Harvard School of Public Health, 677 Huntington Ave, Boston, MA 02115, USA</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:zinde@hsph.harvard.edu">zinde@hsph.harvard.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sjdixon@stanford.edu">sjdixon@stanford.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100300">10.1016/j.xpro.2021.100300</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Anti-cancer drugs kill only a fraction of cells within a population at any given time. Here, we describe a protocol to quantify drug-induced fractional killing over time using high-throughput imaging. This protocol can be used to compare the effect of hundreds of conditions in parallel. We show how this protocol can be used to examine fractional killing in response to inhibitors of the mitogen-activated protein kinase pathway.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Inde et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for high-throughput analysis of drug-induced fractional killing <i>in vitro</i>
</li>
<li>Detailed methods for comparing fractional killing between conditions</li>
<li>Description of MEK1/2 inhibitor-induced fractional killing variability</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/431-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Generate mKate2-expressing cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>To assess drug-induced fractional killing, it is necessary to count both live and dead cells within the population (<a href="#bib2">Inde et al., 2020</a>). Live cells can be conveniently detected via expression of a nuclear-localized fluorescent protein, mKate2. As described below, this enables live cells to be counted using a microscope; these counts are necessary to compute fractional killing in the main protocol.</p>
<div class="note">
<span class="note-title">Note:</span> The methods for generating cell lines and analyzing data described in this protocol should be generalizable to any imaging platform. For imaging live and dead cells, this protocol utilizes an Incucyte microscope installed in a standard tissue culture incubator maintained at 37°C with 5% CO<sub>2</sub> (<a href="#fig1">Figure 1</a>A). Specific imaging parameters may need to be adjusted if using a different imaging platform.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/431-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Generation of mKate2<sup>+</sup> cell lines</div>
<p>(A) An Incucyte installed in a standard tissue culture incubator for the STACK protocol.</p>
<p>(B) HT-1080 cells seeded in a tissue culture plate after adhering to the plate surface. Cells are imaged using the Incucyte 10× objective in the phase-contrast channel. Cells were infected with NLR lentivirus at a MOI of ~3 (right) or vehicle control (left). Cells were then treated with 1 μg/mL puromycin for 24 h (bottom). Scale bar, 100 μm.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> The protocol is optimized for adherent cell lines, which remain in a single focal plane in culture. It is conceivable that this protocol could be applied to non-adherent cell lines, but additional optimization would be required, including for the centrifugation of culture plates to collect cells at the bottoms of wells prior to imaging.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Cell should be thawed and cultured in appropriate culture medium, as recommended by the suppler. Choice of culture medium or additives should not affect the execution of the protocol. As with any cell-based assay, using early passage cells is preferred. However, the protocol will work with cells at any passage number. The protocol is compatible with cell culture requiring coated culture vessels (e.g., Matrigel); confirm successful Incucyte imaging of cells in coated plates before beginning.</div>
<ol>
<li>Determine antibiotic selection dose for cell line(s) of interest.
<ol type="a">
<li>Seed cells in 6 wells of a 12-well culture plate. Seeding density should result in approximately 75% confluence the following day when antibiotic is added.</li>
<li>Allow cells to adhere to plate 18–24 h.</li>
<li>Apply antibiotic for selection in a dose response.
<ol type="i">
<li>Dilute antibiotic in culture media to a dose of 10 μg/mL. Dilute 1:1 in culture media four times to yield a two-fold dose response ranging from 625 ng/mL to 10 μg/mL.</li>
<li>Aspirate media from cultured cells and replace with antibiotic selection media. Replace normal media in remaining well for an untreated control.</li>
<li>Monitor cells daily with a standard light microscope or by imaging using an Incucyte. For most cell lines, puromycin selection will be complete within 48 h.</li>
<li>Determine the minimum antibiotic dose at which selection is complete in uninfected cells (<a href="#fig1">Figure 1</a>B). If death is not apparent from morphological features of the selected cells, SYTOX Green may be used, as described below, to confirm cell death.</li>
<li>Note the minimum dose required to kill &gt;99% of uninfected cells and use this dose for selection of NucLight Red-infected cell lines in step 3.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Lentiviral NucLight Red vectors are available from Sartorius carrying either puromycin- or bleomycin-selectable markers. These vectors express mKate2 fluorescent protein fused to a nuclear localization sequence that targets this protein into the nucleus. Protein import into the nucleus is an ATP-dependent process, and therefore only active in live cells. NucLight Green vectors are also available expressing GFP, if desired. Puromycin is more rapidly lethal to mammalian cells than bleomycin. Selection for bleomycin resistance using bleomycin-family antibiotic zeocin often requires passaging cells several times in antibiotic-containing medium. Thus, if compatible with other experimental conditions, selection with puromycin is generally preferable. Alternatively, or in addition, infected cells may be selected using the optional fluorescence activated cell sorting (FACS) step below. FACS sorting may also be utilized to enrich for high-expressing mKate2 positive (i.e.’<sup>+</sup>’) cells.</div>
<ol start="2">
<li>Infect cell line(s) with NucLight Red lentivirus.
<ol type="a">
<li>Seed cell line(s) in 12 well culture plate at approximately 25% confluence. Seed two wells per cell line, one for infection and one as a mock-infected control to confirm later antibiotic selection.</li>
<li>Allow cells to adhere to plate 18–24 h.</li>
<li>Consult packaging or supplier website to determine viral titer of NucLight Red lentivirus, expressed in transduction units (TU). Calculate volume of virus to be applied to cells to yield a multiplicity of infection (m.o.i.) of approximately 3.
<ol type="i">
<li>Multiply the number of cells per well by 3 to obtain the necessary TU. Depending on the proliferation rate of the cell line in question, the cell number will be greater than the number of cells seeded; multiply the number of cells by 1.5 for an approximate cell number at time of infection.</li>
<li>Based on the titer of the lentivirus, calculate the volume of virus required for each well (desired TU divided by TU/mL)</li>
</ol>
</li>
<li>On the day of infection, prepare polybrene and lentivirus solutions in culture media.
<ol type="i">
<li>Polybrene stocks can be made at 1,000× in water at a concentration of 8 mg/mL (store at −20°C for up to 1 year). Dilute to 2× in a volume of 500 μL media per infection or control well.</li>
<li>Dilute calculated volume of virus in 500 μL media per infection well.</li>
</ol>
</li>
<li>Aspirate media from culture plate.</li>
<li>Add 500 μL media containing diluted 2× polybrene per well.</li>
<li>To infection well(s), add 500 μL diluted virus media. To uninfected control well(s), add 500 μL culture media without virus.</li>
<li>Monitor expression of mKate2 over the following days by imaging using an Incucyte or another imaging platform. mKate2 expression should be apparent after 24–48 h.</li>
<li>The day following infection, remove and properly dispose of virus- and polybrene-containing media from all wells. Replace with normal culture media.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Lentiviral infection and handling of cells post-infection should be performed with appropriate biosafety level 2+ precautions. Personal protective equipment (PPE) should be worn while performing infections and while handling viral media or culture plates with infected cells. Freshly diluted 10% bleach solution should be prepared daily, and viral media-contaminated waste should be submerged fully in 10% bleach to decontaminate for 20 min before disposing in standard biohazard waste. Virus precautions should be maintained, even after infection media has been removed, until infected cells have been expanded to a new culture vessel and the infection plate has been disposed of.</div>
<ol start="3">
<li>Select infected cells.
<ol type="a">
<li>Two days after infection, prepare 1 mL antibiotic media per well at the concentration determined in step 1.</li>
<li>Remove and properly dispose of media from wells of infection plate.</li>
<li>Add antibiotic selection media to all wells.
<div class="optional">
<span class="optional-title">Optional:</span> Infected cells can be expanded before selection, and a portion of infected, unselected cells can be separately maintained in culture until successful selection is confirmed.</div>
</li>
<li>Monitor selection and confluence of infected cells by imaging using the Incucyte.</li>
<li>If cells reach 100% confluence during selection, passage cells to a 6 well culture plate while maintaining selection. Trypsinizing and reseeding cells in antibiotic-containing media will increase the sensitivity of cells to antibiotic; if necessary, reseed cells in antibiotic-free media and resume selection once cells have adhered to new plate.</li>
<li>Maintain identical passaging and selection procedures for infected cells and uninfected controls. Selection is complete when no viable cells are present in uninfected control wells (<a href="#fig1">Figure 1</a>B, bottom left).</li>
<li>Expand selected mKate2<sup>+</sup> cells in normal culture media. If desired, cryopreserve vials of mKate2<sup>+</sup> cells for later use.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Frozen cells can be maintained in this state indefinitely until investigators are prepared to undertake the next steps in this protocol.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If preferred, or if antibiotic selection is not possible, select NucLight Red-infected cells by FACS. mKate2 fluorescence is detectable in the PE-Texas Red channel found in most FACS instruments; its excitation/emission maxima are 588/633. Before sorting, infected cells should be expanded so that at least 10,000 mKate2<sup>+</sup> cells can be isolated for further expansion. An uninfected control population can be used to gate all mKate2<sup>+</sup> cells, or the desired population can be selected based simply on the distribution of mKate2 signal in the infected cells.</div>
<div class="note">
<span class="note-title">Note:</span> Either of these approaches, bulk sorting of mKate2<sup>+</sup> cells or antibiotic selection, will yield a polyclonal mKate2<sup>+</sup> population for downstream applications. In other words, the precise number of transgene insertions and the location of these insertions in the genome will vary among the infected cells. If polyclonality is not desirable for the downstream applications of interest, mKate2-expressing cell lines may be produced by single-cell sorting of infected cells from most cell lines (not all cell lines are suitable for single-cell cloning). After sorting and expansion of clonal cell lines from the infected population, these clonal cell lines may be examined for mKate2 expression and other phenotypes of interest, then further utilized as described below.</div>
<div class="note">
<span class="note-title">Note:</span> Investigators may wish to validate that mKate2 expression does not affect cell death phenotypes. In our experience, mKate2 expression does not alter sensitivity to lethal stimuli. However, this can be confirmed by applying a given lethal stimulus to both the parental and mKate2-expressing cell lines in parallel and looking for similar responses using an independent readout of cell death (e.g., Annexin-V positivity).</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Optimize the detection of live and dead cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<div class="critical">
<span class="critical-title">Critical:</span> For initial experiments, a 96 well format typically allows for easy setup and produces robust data. The cell dilutions suggested here yield cell densities of 10,000, 5,000, 2,500, and 1,250 cells/well. If necessary, pilot experiments can be repeated for further optimization of seeding density. To wit, initial cell seeding density can impact the detection of live and dead cells, and potentially the lethality of different drugs, as discussed (<a href="#bib1">Forcina et al., 2017</a>). Cell density may also impact cell proliferation rate, which may in turn alter the response to a given lethal stimulus. In all cases, the seeding density selected should be low enough that red objects can be reliably segmented by the analysis software, as described below using the Incucyte analyzer as a specific example. After determining an appropriate initial cell seeding density and optimizing an analyzer for a given cell line, experiments can be scaled for higher throughput (384 well format) or larger culture areas (12 well, 6 well, or other formats).</div>
<ol start="4">
<li>Determine appropriate seeding density for plate type and selected experimental design.
<ol type="a">
<li>For a 96 well plate experiment, seed a range of cell densities in a test plate.
<ol type="i">
<li>Trypsinize mKate2<sup>+</sup> cells, then quench the trypsinization reaction with medium and count the cells suspended in medium. Resuspend in culture medium at 50,000 cells/mL. Dilute 1:1 in media 3× to yield cell concentrations of 25,000, 12,500, and 6,250 cells/mL. Prepare sufficient cell suspension to test a range of doses for lethal compound(s) of interest at each seeding density.</li>
<li>Dispense cells into tissue culture plate in a volume of 200 μL/well. To prevent uneven distribution of cells when they adhere to the plate, centrifuge plate briefly (place in a tabletop swing bucket centrifuge, start spin, and stop when speed reaches approximately 55 × <i>g</i>) before placing in tissue culture incubator 18–24 h.</li>
</ol>
</li>
<li>Replace seeding media with treatment media.
<ol type="i">
<li>Add SYTOX Green dye to appropriate culture media at a concentration of 20 nM. SYTOX Green is a live cell impermeable nucleic acid dye first characterized in (<a href="#bib3">Langsrud and Sundheim, 1996</a>). Uptake of SYTOX Green following plasma membrane permeabilization marks dead cells.</li>
<li>Prepare treatment media by diluting compounds of interest in SYTOX Green-containing media. To test multiple doses, perform serial dilutions of compounds in SYTOX Green-containing media. Include vehicle control treatment(s) as appropriate.</li>
<li>Invert seeded plate and flick sharply into a basin to remove seeding media. Adherent cells should not detach and will remain adhered to the plate. Working quickly to prevent wells from drying completely, replace with 200 μL of treatment media.</li>
</ol>
</li>
<li>Begin Incucyte imaging of plate.
<ol type="i">
<li>Place plate in open slot of Incucyte. Ensure that plate is seated securely in slot and level along all plate edges.</li>
<li>In Incucyte Zoom software, select appropriate culture plate definition and scanning pattern. Add plate to repeating scan schedule to scan once every 4 h.</li>
</ol>
</li>
<li>Monitor death and proliferation of cells over time (<a href="#fig2">Figure 2</a>A). Stop experiment when untreated wells have reached full confluence and started to die or when death induced by compound(s) of interest is complete.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/431-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Optimization of Incucyte analyzer</div>
<p>(A) HT-1080<sup>N</sup> cells following initial seeding and after treatment with vehicle control (DMSO) or the lethal proteasome inhibitor bortezomib for 48 h. Scale bar, 100 μm. Cells are imaged using the 10× objective; SYTOX green marks dead cells in the green channel.</p>
<p>(B) Screenshots of Incucyte software showing results of low (top) and high (bottom) edge sensitivity setting on object counting. Scale bars, 20 μm.</p>
<p>(C) Screenshots of Incucyte software showing optimized masking for red, green, and overlap objects. Scale bars, 100 μm.</p>
</figcaption>
</figure>
</li>
<li>If necessary, repeat optimization experiment, adjusting plate format, seeding density, compound treatments, or scan timing to adequately capture phenomena of interest.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Many lethal compounds can be used as robust controls for cell death induction and to optimize the Incucyte analyzer. A selection of suitable lethal compounds is listed in the <a href="#key-resources-table">Key resources table</a>: bortezomib is a proteasome inhibitor that induces apoptotic cell death; staurosporine is a natural product that induces apoptosis through inhibition of multiple kinases; erastin2 is an inhibitor of the cystine/glutamate antiporter system x<sub>c</sub><sup>-</sup> and induces ferroptotic (i.e., non-apoptotic) cell death.</div>
<ol start="5">
<li>Optimize analyzer for mKate2<sup>+</sup> cell line
<ol type="a">
<li>Select images from seeding optimization experiment(s) and add to a new image collection. Include sufficient variety of images to ensure robust object recognition across conditions: varying confluence, varying cell viability, and both drug-treated and vehicle control-treated cells. Note that drug treatment may alter cell morphology (and therefore object recognition) even if viability is unaffected.</li>
<li>Generate new analyzer based on image collection. Include quantification of mKate2<sup>+</sup>, SYTOX Green<sup>+</sup>, and overlap<sup>+</sup> objects. (Overlap<sup>+</sup> objects are identified as objects in both the red and green channels by the analyzer. Typically, these are dead cells that have not yet fully lost mKate2 fluorescence.) Recommended settings include adaptive color processing and setting min and max area filters to determine counting of red and green objects. General guidelines for adjusting analyzer parameters are discussed in the following note.</li>
<li>Set edge sensitivity such that red nuclei and green puncta are counted only once. Setting sensitivity too high may result in objects being segmented and counted multiple times (<a href="#fig2">Figure 2</a>B).</li>
<li>Examine analyzer masks and determine quality of object recognition. Adjust parameters as necessary to reduce false negative and false positive objects (<a href="#fig2">Figure 2</a>C).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Incucyte image analyzer software has two main components for each color channel: parameters and cleanup. We use the no background subtraction, adaptive parameter, with a threshold adjustment of 10 green calibrated units (GCU) in the green channel and 1 red calibrated unit (RCU) in the red channel. This means that while the background brightness in each image is identified, it is not explicitly subtracted from the intensities of the identified objects. Instead, an object is called as long as it is 10 GCUs above the background threshold in the green channel or 1 RCU above the threshold in the red channel. The green channel GCU threshold is set higher because we have observed a green “haze” in many images (as in <a href="#fig2">Figure 2</a>A, left). True SYTOX green positive objects are very bright in comparison, and so this strict threshold cutoff correctly identifies green puncta while excluding the haze.</div>
<div class="note">
<span class="note-title">Note:</span> We also use the edge split setting, which is recommended for separating closely spaced objects. The edge sensitivity is set to −7 in the green channel and −3 in the red channel. A lower edge sensitivity means that the analyzer is less likely to segment pixels into multiple objects (less likely to “call” multiple red or green objects when there is only one true object) whereas a higher edge sensitivity means that the analyzer is more likely to segment objects. For example, in <a href="#fig2">Figure 2</a>B an edge sensitivity of −3 allows for proper segmentation of red nuclei, whereas an edge sensitivity of +10 over-segments the nuclei and counts multiple red objects erroneously. Our analyzer has a lower edge sensitivity in the green channel than the red channel (−7 vs −3) because it is often necessary to separate closely packed red objects. For example, a very confluent well may have cells whose nuclei are pushed closely together. The red nuclei in this scenario should be properly counted as independent objects.</div>
<div class="note">
<span class="note-title">Note:</span> For cleanup settings, the user can set min and max thresholds for area, eccentricity, mean intensity, and integrated intensity. In the green channel, we set an area filter for a minimum of 40 μm<sup>2</sup> and a maximum of 750 μm<sup>2</sup>. In the red channel, we restrict the area from 40 μm<sup>2</sup> to 8,100 μm<sup>2</sup>. We have found that the stricter area cutoff in the green channel also prevents erroneous counting of the green “haze” mentioned above. The eccentricity setting will define a range of roundness for objects. We do not recommend setting eccentricity threshold cutoffs. Mean and integrated intensity settings will allow for cutoffs based on the intensity levels of called objects. It can be valuable to set minimum thresholds to prevent the analyzer calling very faint objects. Particularly, this can be useful in the red channel, to exclude dead cells that still retain very faint mKate2+ signal. We set a minimum mean intensity of 5.00 A.U. in the red channel, where objects less intense are not counted by the analyzer.</div>
<div class="note">
<span class="note-title">Note:</span> Cells may behave differently if they are from different tissues of origin, or if they are exposed to different drugs or seeding conditions. Therefore, there is no true “one size fits all” setting for the image analyzer. Overall, we recommend that the user alter each of these parameters as appropriate such that red and green objects are appropriately counted by the analyzer for a given lethal treatment of interest.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Incucyte NucLight Red Lentivirus (EF1α, Puro)</td>
<td>Sartorius</td>
<td>Cat#4625</td>
</tr>
<tr>
<td>Incucyte NucLight Red Lentivirus (EF1α, Bleo)</td>
<td>Sartorius</td>
<td>Cat#4627</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Puromycin dihydrochloride</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A1113803</td>
</tr>
<tr>
<td>Zeocin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#R25001</td>
</tr>
<tr>
<td>Hexadimethrine bromide (polybrene)</td>
<td>Millipore Sigma</td>
<td>Cat#H9268</td>
</tr>
<tr>
<td>SYTOX green</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#S7020</td>
</tr>
<tr>
<td>Bortezomib</td>
<td>Fisher Scientific</td>
<td>Cat#NC0587961</td>
</tr>
<tr>
<td>Staurosporine</td>
<td>ApexBio</td>
<td>Cat#A8192</td>
</tr>
<tr>
<td>Erastin2</td>
<td>Cayman Chemical</td>
<td>Cat#27087</td>
</tr>
<tr>
<td>Pimasertib</td>
<td>Selleck Chemicals</td>
<td>Cat#S1475</td>
</tr>
<tr>
<td>Trametinib</td>
<td>Selleck Chemicals</td>
<td>Cat#S2673</td>
</tr>
<tr>
<td>AZD8055</td>
<td>Selleck Chemicals</td>
<td>Cat#S1555</td>
</tr>
<tr>
<td>S63845</td>
<td>Selleck Chemicals</td>
<td>Cat#21131</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>A549<sup>N</sup>
</td>
<td><a href="#bib1">Forcina et al., 2017</a></td>
<td>N/A</td>
</tr>
<tr>
<td>A549<sup>N</sup> Control</td>
<td><a href="#bib2">Inde et al., 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td>A549<sup>N</sup> MCL1<sup>LOF</sup>
</td>
<td><a href="#bib2">Inde et al., 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td>U-2 OS<sup>N</sup>
</td>
<td><a href="#bib1">Forcina et al., 2017</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Calu-6<sup>N</sup>
</td>
<td><a href="#bib2">Inde et al., 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td>HT-29<sup>N</sup>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>H23<sup>Cas9; N</sup>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>SW48<sup>N</sup>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>T98G<sup>N</sup>
</td>
<td><a href="#bib1">Forcina et al., 2017</a></td>
<td>N/A</td>
</tr>
<tr>
<td>SW620<sup>N</sup>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>NCI-H1299<sup>N</sup>
</td>
<td><a href="#bib2">Inde et al., 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td>NCI-H2291<sup>N</sup>
</td>
<td><a href="#bib2">Inde et al., 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Prism</td>
<td>GraphPad Software</td>
<td>N/A</td>
</tr>
<tr>
<td>Excel</td>
<td>Microsoft</td>
<td>N/A</td>
</tr>
<tr>
<td>IncuCyte Base Software</td>
<td>Sartorius</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Incucyte microscope</td>
<td>Sartorius</td>
<td>N/A</td>
</tr>
<tr>
<td>96-well plate</td>
<td>Corning</td>
<td>Cat# 3904</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Conduct Incucyte experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> varies by length of drug treatment, but typically 2–5 days</div>
<p>In this step, cell seeding, treatment, and imaging are conducted to generate raw counts of live and dead cells at each timepoint over the course of an experiment. These counts provide the basis for further analysis of cell death kinetics in subsequent steps.</p>
<ol>
<li>Seed mKate2<sup>+</sup> cells for treatment.
<ol type="a">
<li>Trypsinize, quench, and count mKate2<sup>+</sup> cells. Resuspend in culture media for seeding at the density determined in pilot experiments.</li>
<li>Seed cells in appropriate tissue culture vessel, centrifuge 55 × <i>g</i> for 5 s, and place in tissue culture incubator. Allow to adhere 18–24 h.</li>
</ol>
</li>
<li>Apply drug treatment.
<ol type="a">
<li>Add SYTOX Green dye to appropriate culture medium at a concentration of 20 nM.</li>
<li>Prepare treatment media by diluting compounds of interest in SYTOX Green-containing medium. If desired, perform serial dilutions of compounds to test multiple doses. Include vehicle control treatment(s) as appropriate.</li>
<li>Invert seeded plate and flick sharply into a basin to remove seeding medium. Working quickly to prevent wells from drying completely, replace with 200 μL of treatment medium.</li>
</ol>
</li>
<li>Begin Incuycte imaging of plate.
<ol type="a">
<li>Place plate in open slot of Incucyte. Ensure that plate is seated securely in slot and level along all plate edges.</li>
<li>In Incucyte Zoom software, select the plate definition for the specific vessel (e.g., Corning 96 well, Cat# 3904) being employed in the experiment from the list of preset option. Add the plate to the repeating scan schedule to scan once every 4 h.</li>
</ol>
</li>
<li>Complete Incucyte experiment and obtain raw counts for analysis.
<ol type="a">
<li>Stop experiment at endpoint determined in pilot experiments.</li>
<li>Run analysis of collected images using cell line-optimized analyzer.</li>
<li>Export time course data for each treatment condition for three object count parameters: SYTOX Green<sup>+</sup> objects per mm2, mKate2<sup>+</sup> objects per mm2, and overlap objects per mm2.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Data can be further analyzed immediately or stored electronically until a later time.</div>
<h3 id="sec2.2">Analyze fractional killing kinetics</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This step describes the processing of raw object counts to calculate cell death kinetic parameters. Specifically, raw counts are converted to a lethal fraction (LF) metric as described in (<a href="#bib1">Forcina et al., 2017</a>). Curve fitting of LF values over the course of an experiment allows for calculation of the death onset (D<sub>O</sub>) and death rate (D<sub>R</sub>) parameters. D<sub>R</sub> values are a useful means to quantify the rate of drug-induced fractional killing (FK), as described in (<a href="#bib2">Inde et al., 2020</a>). These parameter values can then be compared between diverse conditions acquired in parallel.</p>
<ol start="5">
<li>Calculate and plot LF for each time point in each experimental condition. Refer to “<a href="#quantification-and-statistical-analysis">Quantification and statistical analysis</a>” for a sample dataset.
<div class="optional">
<span class="optional-title">Optional:</span> Examine the overlap<sup>+</sup> counts for the tested conditions. For some cell line/drug combinations, or individual timepoints, these may be a substantial fraction of all mKate2<sup>+</sup> object. If overlap<sup>+</sup> counts are high (&gt; 5% of all mKate2<sup>+</sup> counts), generate corrected mKate2<sup>+</sup> object counts by subtracting the overlap object count from the corresponding mKate2 object count at each timepoint.</div>
<ol type="a">
<li>Using the raw or corrected mKate2<sup>+</sup> counts, calculate LF according to the formula below. MaxSG is the maximum value for SYTOX Green<sup>+</sup> counts observed up to the current timepoint.<br> <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/431-Si1.jpg" alt="Si1.jpg">
</li>
<li>Using GraphPad Prism software, plot the LF data over time in an XY scatter plot</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Fit one phase association curve to data. This curve, also referred to as a lag-exponential death (LED) model, includes a plateau phase (up until D<sub>O</sub>) where little or no death occurs and an exponential phase (after D<sub>O</sub>) where death begins at a given rate (D<sub>R</sub>) and then slows over time.
<ol type="a">
<li>In the Prism software, create a new analysis from the data table containing the LF data.</li>
<li>In the “Create New Analysis” window, select “Nonlinear regression (curve fit)” under the “XY analyses” category.</li>
<li>In the “Parameters: Nonlinear Regression” window, in the “Model” tab, select “Plateau followed by one phase association” under the “Exponential” category.</li>
<li>Also, in the “Parameters: Nonlinear Regression” window, in the “Constrain” tab, change the Constraint Type for X0 (corresponding to D<sub>R</sub>) to “No constraint.”</li>
<li>Click “OK” to perform the curve-fitting analysis. A new “Results” sheet will appear when the analysis is complete.</li>
</ol>
</li>
<li>Extract kinetic parameters from fitted curve.
<ol type="a">
<li>In the Results sheet corresponding to the LF data table, confirm a successful curve fit. If the nonlinear regression analysis is unable to determine parameters for the curve, the column will be labeled “Ambiguous” and will not provide 95% CI values. In these cases, often where little death is observed in the treated cells, the exponential curve fit is not appropriate for the data, and D<sub>R</sub> and D<sub>O</sub> values should not be used for comparisons with other conditions.</li>
<li>For each time course, kinetic parameters correspond to calculated “Best-fit values” for the fitted curves. X0 corresponds to D<sub>O</sub> in hours; Y0 corresponds to the baseline LF preceding D<sub>O</sub>; Plateau corresponds to LF at the conclusion of the experiment; and K corresponds to D<sub>R</sub>.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Kinetic parameters extracted from the fitted curves should reasonably match qualitative observations about the observed onset of death in the population and the rate of death after onset. STACK analysis and kinetic parameter extraction allows comparison of drug-induced death kinetics across different drugs (alone or in combination), doses, and cell lines. Example data adapted in part from (<a href="#bib2">Inde et al., 2020</a>) are shown in <a href="#fig3">Figure 3</a>.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/431-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Utilizing kinetic parameters to compare FK across genotypes and drug combinations</div>
<p>(A) Cell death over time in A549<sup>N</sup> WT and MCL1 CRISPR-mutated loss of function (LOF) cells. Cells are treated with lethal (5 μM) or sublethal (500 nM) dose of the MEK inhibitor pimasertib to induce cell death, and the effect of combination with an mTOR inhibitor (AZD8055, 500 nM) or MCL1 inhibitor (S63845, 5 μM) is tested in each genetic background. Data represent mean ± SD from three independent biological replicates.</p>
<p>(B) Extracted cell death kinetic parameter values and their interpretations for the curves presented in A.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Here we present a sample dataset generated from the treatment of ten cell lines treated for 120 h with dose responses for three drugs: two MEK inhibitors (pimasertib and trametinib) and a proteasome inhibitor (bortezomib) (<a href="#fig4">Figure 4</a>). To illustrate data processing steps, raw and processed data are shown for a 5 μM dose of pimasertib in H2291 lung carcinoma cells (<a href="#tbl1">Table 1</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/431-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Utilizing kinetic parameters to compare FK for lethal compounds across cell lines</div>
<p>(A) Example lethal fraction (LF) values for ten cancer cell lines treated with lethal compounds over 120 h. Data represent mean ± SD from three independent biological replicates. Raw data for a representative replicate of pimasertib-treated H2291<sup>N</sup> cells are shown in <a href="#tbl1">Table 1</a>.</p>
<p>(B) Kinetic parameters calculated from selected curves depicted in (A).</p>
</figcaption>
</figure>
<table id="tbl1">
<caption>Table 1. Raw object counts and calculated lethal fraction values</caption>
<thead>
<tr>
<th rowspan="2">Time</th>
<th colspan="2">Raw data</th>
<th>Calculated values</th>
</tr>
<tr>
<th>SYTOX Green<sup>+</sup> counts/mm<sup>2</sup>
</th>
<th>mKate2<sup>+</sup> counts/mm<sup>2</sup>
</th>
<th>Lethal fraction (LF)</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>1.856391</td>
<td>118.809</td>
<td>0.015</td>
</tr>
<tr>
<td>4</td>
<td>2.320489</td>
<td>132.2679</td>
<td>0.017</td>
</tr>
<tr>
<td>8</td>
<td>1.392293</td>
<td>154.0805</td>
<td>0.015</td>
</tr>
<tr>
<td>12</td>
<td>1.856391</td>
<td>147.5831</td>
<td>0.015</td>
</tr>
<tr>
<td>16</td>
<td>1.392293</td>
<td>161.0419</td>
<td>0.014</td>
</tr>
<tr>
<td>20</td>
<td>1.392293</td>
<td>168.0034</td>
<td>0.014</td>
</tr>
<tr>
<td>24</td>
<td>4.17688</td>
<td>175.429</td>
<td>0.023</td>
</tr>
<tr>
<td>28</td>
<td>4.17688</td>
<td>180.0699</td>
<td>0.023</td>
</tr>
<tr>
<td>32</td>
<td>5.105075</td>
<td>185.6391</td>
<td>0.027</td>
</tr>
<tr>
<td>36</td>
<td>7.425564</td>
<td>186.5673</td>
<td>0.038</td>
</tr>
<tr>
<td>40</td>
<td>14.38703</td>
<td>189.816</td>
<td>0.070</td>
</tr>
<tr>
<td>44</td>
<td>13.92293</td>
<td>187.9596</td>
<td>0.071</td>
</tr>
<tr>
<td>48</td>
<td>18.56391</td>
<td>195.8493</td>
<td>0.087</td>
</tr>
<tr>
<td>52</td>
<td>20.4203</td>
<td>190.2801</td>
<td>0.097</td>
</tr>
<tr>
<td>56</td>
<td>23.66899</td>
<td>184.2468</td>
<td>0.114</td>
</tr>
<tr>
<td>60</td>
<td>27.84587</td>
<td>184.7109</td>
<td>0.131</td>
</tr>
<tr>
<td>64</td>
<td>40.8406</td>
<td>189.816</td>
<td>0.177</td>
</tr>
<tr>
<td>68</td>
<td>38.05602</td>
<td>191.6724</td>
<td>0.176</td>
</tr>
<tr>
<td>72</td>
<td>48.26617</td>
<td>192.6006</td>
<td>0.200</td>
</tr>
<tr>
<td>76</td>
<td>52.90715</td>
<td>185.175</td>
<td>0.222</td>
</tr>
<tr>
<td>80</td>
<td>58.94042</td>
<td>183.7827</td>
<td>0.243</td>
</tr>
<tr>
<td>84</td>
<td>64.97369</td>
<td>175.8931</td>
<td>0.270</td>
</tr>
<tr>
<td>88</td>
<td>84.00169</td>
<td>181.4622</td>
<td>0.316</td>
</tr>
<tr>
<td>92</td>
<td>83.5376</td>
<td>175.429</td>
<td>0.324</td>
</tr>
<tr>
<td>96</td>
<td>95.60414</td>
<td>176.8212</td>
<td>0.351</td>
</tr>
<tr>
<td>100</td>
<td>95.14005</td>
<td>173.1085</td>
<td>0.356</td>
</tr>
<tr>
<td>104</td>
<td>100.2451</td>
<td>168.0034</td>
<td>0.374</td>
</tr>
<tr>
<td>108</td>
<td>106.7425</td>
<td>160.1137</td>
<td>0.400</td>
</tr>
<tr>
<td>112</td>
<td>129.9474</td>
<td>164.7547</td>
<td>0.441</td>
</tr>
<tr>
<td>116</td>
<td>138.7652</td>
<td>168.0034</td>
<td>0.452</td>
</tr>
<tr>
<td>120</td>
<td>136.9088</td>
<td>159.6496</td>
<td>0.465</td>
</tr>
</tbody>
</table>
<p class="table-legend">Data are shown for H2291<sup>N</sup> cells treated with 5 μM pimasertib at the indicated timepoints.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>As demonstrated in the example dataset above, robust calculation of D<sub>O</sub> and D<sub>R</sub> parameter values can only be performed for conditions where sufficient lethality is observed during the course of the experiment, typically with a final lethal fraction of greater than 0.2 (<a href="#bib1">Forcina et al., 2017</a>). Thus, a sufficiently lethal dose of drug must be applied, and the experiment must run long enough for lethality to be observed. Empirically, we have observed that when these conditions are met, D<sub>O</sub> and D<sub>R</sub> can be calculated robustly even if the time course does not run until the lethal fraction values have reached a final plateau. Similarly, these parameters can typically be calculated accurately using a scanning interval of 4 h or longer; however, in some cases, such as very rapid lethal treatments, a shorter measurement interval (e.g., 1 h) may better capture fast kinetics.</p>
<div class="note">
<span class="note-title">Note:</span> In addition to extracted kinetic parameter values, other information derived from these curves, including the maximum lethal fraction values and the area under the lethal fraction curve could potentially be of use in summarizing and comparing fractional killing under different lethal conditions. These alternative values, which are easily extracted from lethal fraction data over time, may provide a useful surrogate measure of fractional killing for cases where cell death exhibits low or unusual kinetics over time that cannot be effectively captured by the described curve-fitting approach.</div>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Heterogeneous expression of mKate2 in viable cells (<a href="#before-you-begin">Before you begin</a>, step 3)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Reapply antibiotic selection until more uniform marker expression is observed or employ the alternative FACS-based selection strategy described after step 3g under “<a href="#before-you-begin">Before you begin</a>.”</p>
<h3 id="sec6.3">Problem 2</h3>
<p>False positive or false negative objects recognized by Incucyte image analyzer (<a href="#before-you-begin">Before you begin</a>, step 5)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Repeated adjustments to analyzer parameters may be required until a final version is established for a given cell line. Accurate quantification of mKate2<sup>+</sup> objects is particularly difficult at 100% confluence or greater; cell seeding densities or experiment time courses may also be modified to prevent overcrowding of cells and subsequent quantification errors.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Changes in cell morphology occur during or after cell death (<a href="#before-you-begin">Before you begin</a>, step 5)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>If necessary, analyzer should be optimized for each combination of cell line and lethal stimulus, with parameters set to count mKate2<sup>+</sup> objects even if their shape is affected by drug treatment. After death, some dead cells may lose SYTOX Green fluorescence over time. This should not affect LF calculations, since they are based on the maximum SYTOX Green values observed up to a given timepoint, as described (<a href="#bib1">Forcina et al., 2017</a>).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Non-specific SYTOX Green signal (<a href="#step-by-step-method-details">Step-by-step method details</a>, step 2)</p>
<h3 id="sec6.8">Potential solution</h3>
<p>20 nM is an appropriate SYTOX Green concentration for almost all cell lines. In some rare cases, this concentration may yield dim green signal in the cytoplasm of viable cells; in these cases, lower doses of SYTOX Green may be tested. Non-specific SYTOX Green signal may also indicate the presence of contamination such as mycoplasma.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Other non-specific fluorescence (<a href="#step-by-step-method-details">Step-by-step method details</a>, step 2)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Some tested compounds may be autofluorescent and therefore interfere with imaging, typically producing false positives for SYTOX Green fluorescent objects. Autofluorescent compounds can be preliminarily identified by identifying conditions where SYTOX Green counts are high at early timepoints, without obvious evidence for cell death. Subsequent visual inspection can confirm that this fluorescence in the green channel is not due to true positive dead cells. In some cases, autofluorescent material is present outside cells. Conditions with autofluorescent compounds may either be removed from consideration or analyzed based on the live cell counts alone.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Forcina, G.C., Conlon, M., Wells, A., Cao, J.Y., and Dixon, S.J. (2017). Systematic quantification of population cell death kinetics in mammalian cells. Cell Syst. <i>4</i>, 600-610.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00007-1/sref1">View at publisher</a></p>
<p id="bib2">Inde, Z., Forcina, G.C., Denton, K., and Dixon, S.J. (2020). Kinetic heterogeneity of cancer cell fractional killing. Cell Reports <i>32</i>, 107845. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00007-1/sref2">View at publisher</a></p>
<p id="bib3">Langsrud, S. and Sundheim, G. (1996). Flow cytometry for rapid assessment of viability after exposure to a quaternary ammonium compound. J. Appl. Bacteriol. <i>81</i>, 411-418. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00007-1/sref3">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Requests for further information should be directed to and will be fulfilled by the Technical Contact, Zintis Inde (<a href="mailto:zinde@hsph.harvard.edu">zinde@hsph.harvard.edu</a>). Requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Scott J. Dixon (<a href="mailto:sjdixon@stanford.edu">sjdixon@stanford.edu</a>).</p>
<h4>Materials Availability</h4>
<p>Cell lines expressing nuclear mKate2 are available from the Lead Contact.</p>
<h4>Data and Code Availability</h4>
<p>There is no additional data associated with this protocol.</p>
<h3>Acknowledgments</h3>
<p>We thank Leslie Magtanong and other members of the Dixon lab for comments on this work. S.J.D. is supported by the <a href="https://doi.org/10.13039/100000002">NIH</a> (1R01GM122923).</p>
<h3>Author Contributions</h3>
<p>Conceptualization, Z.I. and S.J.D.; Methodology, Z.I. and J.R.; Writing – Original Draft, Z.I.; Writing – Review &amp; Editing, Z.I., J.R., and S.J.D.; Funding Acquisition and Supervision, S.J.D.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>