<article>
<section>
<h1 id="header">Denaturing cross-linking immunoprecipitation to identify footprints for RNA-binding proteins</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Michael Rosenberg,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Vered Levy,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Verena K. Maier,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Barry Kesner,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Roy Blum,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Jeannie T. Lee<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, MA 02114, USA</p>
<p id="aff2"><sup>2</sup>Department of Genetics, Harvard Medical School, Boston, MA 02114, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:lee@molbio.mgh.harvard.edu">lee@molbio.mgh.harvard.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100819">10.1016/j.xpro.2021.100819</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The isolation of protein-RNA complexes in the “denaturing cross-linked RNA immunoprecipitation” (dCLIP) protocol is based on biotin-tagging proteins of interest, UV cross-linking RNA to protein in vivo, RNase protection assay, and isolating RNA-protein complexes under denaturing conditions over a streptavidin column. Insofar as conventional antibody-based CLIP assays have been challenging to apply to Polycomb complexes, dCLIP has been applied successfully and yields small RNA footprints from which de novo motif analysis can be performed to identify RNA binding motifs.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.cels.2017.09.014">Rosenberg et al. (2017)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>dCLIP biotags a protein of interest to identify cross-linked RNA interactors in vivo</li>
<li>Biotin-streptavidin purification system enables denaturing washing conditions</li>
<li>dCLIP is successfully applied to chromatin-modifying protein complexes</li>
<li>dCLIP allows high-resolution mapping of RNA binding sites and de novo motif analysis</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1018-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Prepare stable cell lines expressing BirA biotinylase and AVI-tagged protein/s of interest</h3>
<p>We utilize the in vivo biotinylation system described previously in Kim et al. (<a href="#bib5">Kim et al., 2009</a>). Prepare 2 types of stable cell lines: (1) Parental line expressing V5-His-tagged bacterial biotin ligase BirA (BirAV5His) and (2) Experimental cell line/s using the parental line to express both bacterial biotin ligase BirA and the protein of interest fused to biotag (GLNDIFGAQKIEWHE) or a similar AVI-tag (GLNDIFEAQKIEWHE). These peptides are specifically recognized as a substrate by BirA and undergo biotinylation on the lysine residue.</p>
<div class="note">
<span class="note-title">Note:</span> We used puromycin and neomycin resistance for cell line selection, but different combinations of selection markers can be utilized.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Generate a biotagged protein of interest by knocking-in a biotinylation tag on the N-terminus or C-terminus of the endogenous gene via CRISPR-assisted genome editing in BirA-expressing parental cell line.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The Strep-Tag system (IBA-Lifesciences, <a href="https://www.iba-lifesciences.com/strep-tag">https://www.iba-lifesciences.com/strep-tag</a>) can be utilized to achieve a cost-effective high-affinity binding without the requirement for expressing two transgenic proteins. Strep-Tag serves as a mimic for biotin and occupies the same binding pocket on the specifically designed streptavidin. Omitting the need to express BirA enzyme can reduce the potential background signal due to off-target BirA activity.</div>
<div class="timing">
<span class="timing-title">Timing:</span> 4-8 Weeks</div>
<ol>
<li>Subclone gene/s of interest into an expression vector with biotinylation tag and selection marker for puromycin resistance</li>
<li>Generate parental stable cell line expressing BirA.
<ol type="a">
<li>Grow cells to 70% confluence in T-75 flasks.</li>
<li>Trypsinize cells to create a single-cell suspension in PBS.</li>
<li>Electroporate to transfect cells with pEF1aBirAV5His vector in PBS. Use GenePulser II (Bio-Rad) according to the manufacturer’s instructions (<a href="https://www.bio-rad.com/en-us/product/gene-pulser-xcell-electroporation-systems?ID=b1a35eb3-d55c-47b3-aaf3-95e4d1d85848#fragment-doc">https://www.bio-rad.com/en-us/product/gene-pulser-xcell-electroporation-systems?ID=b1a35eb3-d55c-47b3-aaf3-95e4d1d85848#fragment-doc</a>). Use the following cell number/DNA ratio (Ratios may differ for other cell lines).
<ol type="i">
<li>2 × 10<sup>7</sup> ES mouse cells (EL 16.7, (<a href="#bib7">Lee and Lu, 1999</a>)) with 30 μg linearized vector.</li>
<li>1 × 10<sup>7</sup> HEK293 or MEF cells with 15 μg linearized vector.</li>
</ol>
</li>
<li>Culture cells in growth media supplemented with 300 μg/mL Geneticin reagent (G418) to select for transfected cells.</li>
<li>Confirm transfection and expression of recombinant BirA by PCR genotyping and Western blotting with specific antibodies. Isolate clones with the highest expression of BirA to use as a parental line.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Generate recombinant protein cell line/s.
<ol type="a">
<li>Electroporate to transfect parental cells (BirA expressing line) with pEF1a-FlagBiotag-Protein of Interest - PGKpuro or pBrCAG-Avi-GFP-Protein of Interest vectors carrying the protein of interest fused to biotinylation tag or AVI-tag respectively. Prepare control cell lines expressing the empty vector (pEF1a-FlagBiotag- PGKpuro) or Avi-tagged GFP only (pBrCAG-Avi-GFP). Use GenePulser II as in step 2c.<br> Use the following cell number/DNA ratio (ratios may differ for other cell lines).
<ol type="i">
<li>2 × 10<sup>7</sup> ES mouse cells with 30 μg linearized vector (EL 16.7, (<a href="#bib7">Lee and Lu, 1999</a>)).</li>
<li>1 × 10<sup>7</sup> HEK293 or MEF cells with 15 μg linearized vector.</li>
</ol>
</li>
<li>Culture cells in growth media supplemented with 1 μg/mL Puromycin and 300 μg/mL Geneticin (G418) to select for transfected cells.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A protein of interest fused to a mutated biotinylation tag (<a href="#bib4">Furuyama and Henikoff, 2006</a>) can be used as an alternative approach for the empty vector or GFP only controls.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend conducting calibration experiments with a range of antibiotic concentrations to determine the minimal concentration that kills the entire cell population within 2–3 days. In our experience, insufficient antibiotic concentration results in a large number of false-positive clones. The instructions for the “killing curve” experiment can be found elsewhere (<a href="http://www.sigmaaldrich.com/technical-documents/articles/biology/antibiotic-kill-curve.html">https://www.sigmaaldrich.com/technical-documents/articles/biology/antibiotic-kill-curve.html</a>)</div>
<ol start="4">
<li>Confirm transfection and expression of recombinant proteins by PCR genotyping and Western blotting with specific antibodies to the recombinant proteins.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To minimize non-specific cross-linking to RNA species. We recommend choosing clones with overall protein expression (recombinant + endogenous) that is most similar to the physiological level, as depicted in <a href="#fig1">Figure 1</a>
</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1018-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The method for selection of clonal cell lines for dCLIP experiments</div>
<p>Representative Western blotting for clonal cell lines derived from mouse embryonic fibroblasts with stable expression of BirA and Avi-GFP-EZH2 or Avi-GFP alone (control). Protein extracts prepared from each clonal cell line were probed with specific antibodies against EZH2 and GAPDH (loading control) proteins. Following densitometry analysis, the intensity ratios between Avi-GFP-EZH2 and endogenous EZH2 and between total EZH2 (transgenic + endogenous) and GAPDH were computed. Note that expression of transgenic EZH2 resulted in reduced expression of the endogenous EZH2, thus leveraging a total amount of EZH2 protein between control and Avi-GFP-EZH2 expressing cells. Clones# 1C, C14 (highlighted in Bold), that exhibited total EZH2 / GAPDH ratio closest to the control cell lines, were chosen for subsequent dCLIP experiments.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Ezh2 (D2C9) XP rabbit monoclonal antibody, 1:1000 dilution</td>
<td>Cell Signaling Technologies Cat# 5246</td>
<td>RRID:AB_10694683</td>
</tr>
<tr>
<td>GAPDH (14C10) rabbit monoclonal antibody, 1:5000 dilution</td>
<td>Cell Signaling Technologies Cat# 2118</td>
<td>RRID:AB_561053</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>pEF1aBirAV5His</td>
<td>Dr. Stuart Orkin, Harvard Medical School</td>
<td>n/a</td>
</tr>
<tr>
<td>pEF1-Flag-Biotag</td>
<td>Dr. Stuart Orkin, Harvard Medical School</td>
<td>n/a</td>
</tr>
<tr>
<td>pBrCAG-Avi-GFP-IRES-Puro</td>
<td>Gift from Dr. Mitinori Saitou, Kyoto University, Japan</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Geneticin G418</td>
<td>Life Technologies</td>
<td>Cat# 10131035</td>
</tr>
<tr>
<td>Puromycin</td>
<td>Life Technologies</td>
<td>Cat# A1113803</td>
</tr>
<tr>
<td>Roche cOmplete Mini EDTA-free Protease Inhibitor Cocktail Tablets</td>
<td>Sigma-Aldrich</td>
<td>Cat# 4693159001</td>
</tr>
<tr>
<td>Protease Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat# P8340</td>
</tr>
<tr>
<td>Roche Protector RNAse Inhibitor, 40U/μL</td>
<td>Sigma-Aldrich</td>
<td>Cat# 3335402001</td>
</tr>
<tr>
<td>TURBO™ DNase, 2 U/μL</td>
<td>Life Technologies</td>
<td>Cat# AM2239</td>
</tr>
<tr>
<td>RNase I, 10 U/μL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# EN0601</td>
</tr>
<tr>
<td>SUPERaseIn RNase Inhibitor, 20 U/μL</td>
<td>Life Technologies</td>
<td>Cat# AM2694</td>
</tr>
<tr>
<td>Dynabeads™ Protein G for Immunoprecipitation</td>
<td>Invitrogen</td>
<td>Cat# 10004D</td>
</tr>
<tr>
<td>Dynabeads™ MyOne™ Streptavidin C1</td>
<td>Invitrogen</td>
<td>Cat# 65001</td>
</tr>
<tr>
<td>ATP, [γ-32P]-6000Ci/mmol 10mCi/mL, 250 μCi</td>
<td>PerkinElmer</td>
<td>Cat# BLU002Z250UC</td>
</tr>
<tr>
<td>ATP solution, 100 mM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R0441</td>
</tr>
<tr>
<td>T4 Polynucleotide Kinase, 10U/μL</td>
<td>New England Biolabs</td>
<td>Cat# M0201S</td>
</tr>
<tr>
<td>LongAmp_Taq 2X Master Mix</td>
<td>New England Biolabs</td>
<td>Cat# M0287S</td>
</tr>
<tr>
<td>GlycoBlue™ Coprecipitant, 15 mg/mL</td>
<td>Invitrogen</td>
<td>Cat# AM9516</td>
</tr>
<tr>
<td>SuperScript III Reverse Transcriptase</td>
<td>Invitrogen</td>
<td>Cat# 18080044</td>
</tr>
<tr>
<td>Trizol Reagent</td>
<td>Life Technologies</td>
<td>Cat# 15596018</td>
</tr>
<tr>
<td>Acid-Phenol:Chloroform with IAA, 125:24:1, pH 4.5</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM9722</td>
</tr>
<tr>
<td>Novex NuPAGE™ LDS Sample Buffer (4X)</td>
<td>Life Technologies</td>
<td>Cat# NP0007</td>
</tr>
<tr>
<td>Western Lightning Plus-ECL</td>
<td>PerkinElmer</td>
<td>Cat# NEL103001EA</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>NEBNext® Small RNA Library Prep Set for Illumina® (Multiplex Compatible)</td>
<td>New England Biolabs</td>
<td>Cat# E7330S</td>
</tr>
<tr>
<td>Bioanalyzer High Sensitivity DNA Analysis</td>
<td>Agilent</td>
<td>Cat# 5067-4626</td>
</tr>
<tr>
<td>KAPA Library Quantification Kit</td>
<td>Roche</td>
<td>Cat# KK4844</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>ES mouse cells (EL 16.7 (129/Cas)</td>
<td>Lee Lab</td>
<td>n/a</td>
</tr>
<tr>
<td>Irradiated mouse embryonic fibroblast feeder cells</td>
<td>Lee Lab</td>
<td>n/a</td>
</tr>
<tr>
<td>HEK293</td>
<td>ATCC</td>
<td>CRL-1573</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Amersham Protran 0.2 μm nitrocellulose membrane</td>
<td>GE Healthcare</td>
<td>Cat# 10600011</td>
</tr>
<tr>
<td>Glass microfiber filter</td>
<td>Whatman</td>
<td>Cat# 1823010</td>
</tr>
<tr>
<td>Costar SpinX column</td>
<td>Corning</td>
<td>Cat# 8161</td>
</tr>
<tr>
<td>Novex NuPAGE™ 4%–12%, Bis-Tris, 1.5 mm, Mini Protein Gel</td>
<td>Life Technologies</td>
<td>Cat# NP0335PK2</td>
</tr>
<tr>
<td>Gene Pulser II Electroporation System</td>
<td>Bio-Rad</td>
<td>Cat# 165-2105</td>
</tr>
<tr>
<td>UV Stratalinker 1800</td>
<td>Stratagene</td>
<td>n/a</td>
</tr>
<tr>
<td>Thermomixer - Mixer HC</td>
<td>USA Scientific</td>
<td>Cat# 8012-0000</td>
</tr>
<tr>
<td>5prime phase-lock gel heavy 2 mL,</td>
<td>Fisher Scientific</td>
<td>Cat# 2302830</td>
</tr>
<tr>
<td>TapeStation System</td>
<td>Agilent</td>
<td>Cat# G2992AA</td>
</tr>
<tr>
<td>DNA LoBind tube 1.5 mL</td>
<td>Eppendorf</td>
<td>Cat# 022431021</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">PBS lysis buffer</h3>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS pH 7.4 (10×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1M)</td>
<td>1 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> (0.1M)</td>
<td>0.1 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Sodium Deoxycholate (10%)</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Nonidet-P-40 (10%)</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>39.9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 12 months</p>
<h3 id="sec2.2">10 M Urea stock solution</h3>
<p>Add 24 g urea powder to 21.3 mL ddH<sub>2</sub>O in 50 mL tube, vortex and incubate at 30°C on rotatory shaker until complete dissolving. Then, equilibrate to 21°C–24°C. Store at 21°C–24°C for up to 6 monhts.</p>
<h3 id="sec2.3">PBS-urea wash buffer</h3>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS pH 7.4 (10×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>Urea (10M)</td>
<td>8 M</td>
<td>40 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>4.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 21°C–24°C for up to 12 months.</p>
<h3 id="sec2.4">PBS-SDS wash buffer</h3>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS pH 7.4 (10×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>2%</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>35 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 21°C–24°C for up to 12 months.</p>
<h3 id="sec2.5">PBS high salt wash buffer</h3>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS pH 7.4 (10×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>NaCl (5M)</td>
<td>850 mM</td>
<td>8.5 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Sodium Deoxycholate (10%)</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Nonidet-P-40 (10%)</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>28.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 12 months</p>
<h3 id="sec2.6">PBS low salt wash buffer</h3>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS pH 7.4 (10×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>NaCl (5M)</td>
<td>150 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Sodium Deoxycholate (10%)</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Nonidet-P-40 (10%)</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>35.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 12 months</p>
<h3 id="sec2.7">DNase mix: Prepared fresh</h3>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNase buffer (10×)</td>
<td>1×</td>
<td>16 μL</td>
</tr>
<tr>
<td>Turbo DNase (2 u/μL)</td>
<td>0.1 u/μL</td>
<td>8 μL</td>
</tr>
<tr>
<td>Protease Inhibitor Cocktail (100×)</td>
<td>1×</td>
<td>1.6 μL</td>
</tr>
<tr>
<td>Roche Protector RNAse inhibitor (40U/ μL)</td>
<td>0.4U/ μL</td>
<td>1.6 μL</td>
</tr>
<tr>
<td>SUPERaseIn RNase Inhibitor (20 U/μL)</td>
<td>0.1 U/μL</td>
<td>0.8 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>132 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>160 μL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.8">Low pH PNK buffer</h3>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 6.48 (1M)</td>
<td>70 mM</td>
<td>2.8 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1M)</td>
<td>10 mM</td>
<td>0.4 mL</td>
</tr>
<tr>
<td>DTT (1M)</td>
<td>5 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>36.6 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 6 months</p>
<h3 id="sec2.9">Low-pH PNK mix: Prepared fresh</h3>
<table id="undtbl8">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Low pH PNK buffer</td>
<td>1×</td>
<td>80 μL</td>
</tr>
<tr>
<td>T4 PNK 10 U/μL,</td>
<td>0.24 U/ μL</td>
<td>2 μL</td>
</tr>
<tr>
<td>Roche Protector RNAse inhibitor (40 U/ μL)</td>
<td>0.95 U/ μL</td>
<td>2 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>84 μL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.10">PNK buffer</h3>
<table id="undtbl9">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 7.4 (1M)</td>
<td>50 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1M)</td>
<td>10 mM</td>
<td>0.4 mL</td>
</tr>
<tr>
<td>DTT (1M)</td>
<td>5 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>NP40 (20%)</td>
<td>0.5%</td>
<td>2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>35.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 6 months</p>
<h3 id="sec2.11">PNK mix: Prepared fresh</h3>
<table id="undtbl10">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PNK buffer</td>
<td>1×</td>
<td>80 μL</td>
</tr>
<tr>
<td>10 mM ATP</td>
<td>0.86 mM</td>
<td>8 μL</td>
</tr>
<tr>
<td>T4 PNK 10 U/μL,</td>
<td>0.32 U/ μL</td>
<td>3 μL</td>
</tr>
<tr>
<td>Roche Protector RNAse inhibitor (40 U/ μL)</td>
<td>0.86 U/ μL</td>
<td>2 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>93 μL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.12">PNK mix 2 for membrane elution: Prepared fresh</h3>
<table id="undtbl11">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PNK buffer</td>
<td>1×</td>
<td>80 μL</td>
</tr>
<tr>
<td>ATP, [γ-32P]6000 Ci/mmol, 10 mCi/mL</td>
<td>0.45 μCi/μL</td>
<td>4 μL</td>
</tr>
<tr>
<td>T4 PNK 10 U/μL,</td>
<td>0.34 U/μL</td>
<td>3 μL</td>
</tr>
<tr>
<td>Roche Protector RNAse inhibitor (40 U/ μL)</td>
<td>0.9 U/μL</td>
<td>2 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>89 μL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.13">PNK wash buffer</h3>
<table id="undtbl12">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 7.4 (1M)</td>
<td>50 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>NaCl (5M)</td>
<td>150 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>EDTA (0.5M)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>NP40 (20%)</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>42.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 12 months</p>
<h3 id="sec2.14">Proteinase K buffer</h3>
<table id="undtbl13">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 8.0 (1M)</td>
<td>100 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>NaCl (5M)</td>
<td>200 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>EDTA (0.5M)</td>
<td>5 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>42 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 21°C–24°C for up to 12 months.</p>
<h3 id="sec2.15">PK solution: Prepared fresh</h3>
<table id="undtbl14">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 7.4 (1M)</td>
<td>100 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>NaCl (5M)</td>
<td>50 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>EDTA (0.5M)</td>
<td>10 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Proteinase K (20 mg/mL)</td>
<td>4 mg/mL</td>
<td>200 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>670 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.16">PK-urea solution: Prepared fresh</h3>
<table id="undtbl15">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 7.4 (1M)</td>
<td>100 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>NaCl (5M)</td>
<td>50 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>EDTA (0.5M)</td>
<td>10 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Urea (10M)</td>
<td>7 M</td>
<td>700 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>170 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.17">20× NuPage MOPS/SDS running buffer pH 7.7</h3>
<table id="undtbl16">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>MOPS</td>
<td>1 M</td>
<td>104.6 gr</td>
</tr>
<tr>
<td>Tris base</td>
<td>1 M</td>
<td>60.6 gr</td>
</tr>
<tr>
<td>SDS</td>
<td>2%</td>
<td>10 gr</td>
</tr>
<tr>
<td>EDTA</td>
<td>20 mM</td>
<td>3 gr</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>to 500mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 3 months</p>
<h3 id="sec2.18">20× NuPage transfer buffer pH 7.2</h3>
<table id="undtbl17">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bicine</td>
<td>0.5 M</td>
<td>10.2 gr</td>
</tr>
<tr>
<td>Bis-Tris (free base)</td>
<td>0.5 M</td>
<td>13.1 gr</td>
</tr>
<tr>
<td>EDTA</td>
<td>20 mM</td>
<td>0.75 gr</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>to 125 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 12 months</p>
<h3 id="sec2.19">DNase mix 2: Prepared fresh</h3>
<table id="undtbl18">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNase buffer (10×)</td>
<td>1×</td>
<td>5 μL</td>
</tr>
<tr>
<td>Turbo DNase (2 u/μL)</td>
<td>0.04 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td>Roche Protector RNase inhibitor (40 U/ μL)</td>
<td>0.8 U/ μL</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>43 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">In vivo UV cross-linking of RNA-protein complexes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–1 h</div>
<p>This step creates a snapshot of protein-RNA interactions at a set time. Covalent cross-linking stabilizes the connection between protein and RNA residues situated in close proximity inferring the interaction between them.</p>
<ol>
<li>Grow cells to full confluence in 15 cm tissue culture dishes. Prepare one 15 cm confluent plate per one dCLIP pull-down<b>.</b> We recommend using 2 plates per sample type since, as described below, we use two RNase concentrations per every clonal cell line.</li>
<li>Wash cells with 10 mL ice-cold PBS.</li>
<li>Irradiate cells covered in 5 mL ice-cold PBS in opened lid dish using 254 nm wavelength.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Expose monolayer cells to 150 mJ/cm<sup>2</sup> and ES cell colonies to 400 mJ/cm<sup>2</sup>
</div>
<ol start="4">
<li>Add 5 mL of ice-cold PBS and scrape cells into 15 mL tubes.</li>
<li>Centrifuge to pellet cells 5 min, 1,000 g at 4°C. Discard supernatant.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Shock-freeze cell pellets in liquid nitrogen and store at −80°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> We strongly advise conducting a small-scale dCLIP experiment prior to entering a large-scale library preparation step (See Small-scale dCLIP protocol, (<a href="#bib10">Rosenberg et al., 2017</a>)). The small-scale protocol is similar to the membrane elution protocol described below with the following modifications. The lysate from one cross-linked 15 cm plate is divided into four equal portions that are treated with No RNase, 100 u/mL, 10 u/mL, and 2u/mL RNase respectively. The washing steps are conducted with 500 μl volumes and the 3′-end dephosphorylation step is omitted. The volumes of DNase mix and PNK mix are downscaled 4-fold. Finally, only <sup>32</sup>P-labeled ATP is utilized at the phosphorylation step. A successful small-scale experiment for Chromobox Protein Homolog 7 (CBX7) protein is shown in <a href="#fig2">Figure 2</a>A. Notice the clear pattern of radiolabeled RNA fragments cross-linked to CBX7 protein in dCLIP-CBX7 ( <a href="#fig2">Figure 2</a>A) compared to extremely high background signal which masked the specific pattern of CBX7 cross-linked RNAs in conventional CLIP-CBX7 (<a href="#fig2">Figures 2</a>B and 2C). Another important control for small-scale experiments is a No UV control, where a dCLIP protocol is conducted on cells not treated with UV, and thus are presumed to lack a radioactive signal emanating from cross-linked RNA fragments, shown in <a href="#fig2">Figure 2</a>D (<a href="#bib9">Rosenberg et al., 2021</a>). In <a href="#fig3">Figure 3</a>, we provide an example of a small-scale dCLIP experiment for Embryonic Ectoderm Development (EED) protein that did not exhibit a detectable dCLIP signal (<a href="#bib9">Rosenberg et al., 2021</a>) despite being recognized as a potential RNA-binding protein in other studies (<a href="#bib14">Long et al., 2017</a>). For that reason, we did not proceed further to a large-scale dCLIP experiment with EED protein.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1018-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. dCLIP vs Conventional CLIP of Polycomb proteins in mouse ES cells</div>
<p>(A) Representative autoradiography dCLIP experiment. The red arrowhead indicates the Bio-tagged-CBX7 signal in two clonal cell lines, 3E and 6F, expressing nearly physiological levels of Bio-tagged-CBX7. The excised membrane area is marked in white.</p>
<p>(B) Representative autoradiography of CLIP experiment using specific antibodies against CBX7. Rabbit IgG was used as a control. The expected size of CBX7 protein is marked with a red arrowhead. A high background signal was visible at 40 kDa level and was not cleared by up to 1M salt washes.</p>
<p>(C) Representative autoradiography of CLIP experiment with anti-HA tag antibody. The 6C and 12D clonal cell lines express HA-tagged-CBX7. The red arrowhead marks HA-CBX7 related signal. Note the presence of strong background signal of anti-HA antibody, similar to anti-CBX7 antibody in (B).</p>
<p>(D) Representative dCLIP experiments for EZH2. Left panel, autoradiography of dCLIP experiment. Right panel, Western blot with the anti-GFP antibody. Red arrows, GFP-Biotagged-EZH2 signal. Ezh2-4A and Ezh2-5A are two clonal 16.7 mES cell lines expressing nearly physiological levels of GFP-Biotagged-EZH2. Green arrowhead – Avi-GFP protein.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1018-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Embryonic Ectoderm Development (EED) protein as an example for protein without dCLIP-detectable RNA binding</div>
<p>(A) Representative dCLIP experiment with N-terminally – Flag-Biotagged EED protein. Left panel, autoradiography of dCLIP experiment. Right panel, Western blot with anti-FLAG antibody. Red arrow, FLAG-Biotagged-EED signal.</p>
<p>(B) Representative dCLIP experiment with C-terminally – HA-Biotagged EED protein. Left panel, autoradiography of dCLIP experiment. Right panel, Western blot with the anti-HA antibody. Red arrow, HAG-Biotagged-EED signal. mEed-4H and mEed-2G are two clonal cell lines expressing nearly physiological levels of FLAG-Biotagged-EED. mEed-11B and mEed-12B are two clonal cell lines nearly expressing physiological levels of HA-Biotagged-EED. Note the lack of EED-specific dCLIP signal in both panels.</p>
</figcaption>
</figure>
<h3 id="sec3.2">Protected protein-RNA “footprint” preparation by partial RNase digestion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–1.5 h</div>
<p>To determine the sequence of RNA binding sites in the RNA-protein complexes, RNA molecules covalently bound to the protein of interest are trimmed to a short size range to create a characteristic RNA ‘footprint’.</p>
<div class="critical">
<span class="critical-title">Critical:</span> RNA fragments shorter than 50 bp are difficult to align unambiguously to the genome; therefore, it is essential to prevent excessive RNase digestion. To create the appropriate ‘Footprint’ RNA size range we tested several RNase concentrations in a preceding small-scale experiment and determined that in order to achieve an optimal size distribution of cross-linked RNA fragments, two RNase concentrations, 10 U/mL and 2 U/mL should suffice. With two cell lysates per clonal cell line, each lysate was treated with 10 U/mL and 2 U/mL RNase respectively. The eluted RNA fragments from the same clonal line are later combined prior to the library construction as described below.</div>
<ol start="6">
<li>Pre-thaw 2 cell pellets per sample type at 4°C and resuspend each pellet in 1.25 mL of ice-cold PBS-lysis buffer supplemented with 1 tablet of Complete-mini EDTA-free tablet, 40 U/mL Protector RNase inhibitor, 1mM Dithiothreitol (DTT).</li>
<li>Rotate 25 min at 4°C.</li>
<li>Spin down and add 25 μL TurboDNase 2 U/μL. Roughly estimate the sample volume.</li>
<li>Add RNase I 100 U/μL to a final concentration of 10 U/mL and 2 U/mL (See Critical note above)</li>
<li>Incubate 15 min in a 37°C. Gently mix samples every 5 min.</li>
</ol>
<ol start="11">
<li>Spin down, add 48 U SUPERaseIN, and adjust the SDS concentration to a final 0.1% to terminate RNase I activity.
<div class="note">
<span class="note-title">Note:</span> SUPERaseIN is used in this step for its ability to inhibit RNase I, compared to other common RNase inhibitors. 0.1% SDS is also intended to inactivate RNase I according to manufacturer instructions.</div>
<div class="optional">
<span class="optional-title">Optional:</span> We observed that pulling down proteins that are tightly bound to chromatin, such as EZH1 and JARID2 is facilitated by using a higher concentration of SDS.</div>
<ol type="a">
<li>In step 11, adjust the SDS concentration to 1% SDS instead of 0.1% and rotate samples for 5 min at 21°C–24°C.</li>
<li>Continue with steps 12–15 at 21°C–24°C <b>instead of</b> 4°C.</li>
</ol>
</li>
</ol>
<ol start="12">
<li>Centrifuge samples twice, 10 min, 21,130g at 4°C to remove residual cell debris.</li>
<li>Transfer supernatant to new 2 mL tubes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In the next steps magnetic Protein G Dynabeads (G-beads) are used for pre-clearing of the lysate from the proteins and nucleic acids binding non-specifically to magnetic beads whereas magnetic Dynabeads MyOne Streptavidin C1 (C1-beads) are used for isolating the RNA-biotinylated protein complexes from cell lysates. Prior to use, thoroughly mix beads and aliquot the volume needed for the total number of samples. Prepare 80 μL beads per sample and use a magnetic separator to separate beads from the supernatant. All bead washes are done with ice-cold buffer at 21°C–24°C.</div>
<ol start="14">
<li>Pre-clearing of cell lysates.
<ol type="a">
<li>Aliquot the volume G-beads needed for the total number of samples. Remove supernatant.</li>
<li>Wash G-beads twice with 1 mL of PBS-lysis buffer.</li>
<li>Resuspend G-beads in 150 μL PBS-lysis buffer per sample multiplied by the number of samples.</li>
<li>Transfer 150 μL G-beads to 2 mL tubes. Remove supernatant.</li>
<li>Add cell lysates to pre-clearing beads. Mix well and rotate 1 h at 4°C.</li>
</ol>
</li>
<li>Pull-down biotinylated proteins with C1-beads.
<ol type="a">
<li>Wash C1-beads 3 times in 1 mL PBS-lysis buffer + 0.5% BSA.</li>
<li>Resuspend C1-beads in 150 μL PBS lysis buffer + 0.5% BSA per sample multiplied by the number of samples.</li>
<li>Transfer 150 μL C1-beads to 2 mL tubes. Remove supernatant.</li>
<li>Capture pre-clearing G-beads (step 14e) on the magnetic separator and transfer the supernatant to C1-beads tubes.</li>
<li>Mix well and rotate for 2 h at 4°C.</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Denaturing stringent washes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–1.5 h</div>
<p>Using the biotin-streptavidin high interaction strength property (K<sub>d</sub>=10<sup>−15</sup> M, (<a href="#bib13">Weber et al., 1989</a>)), the dCLIP protocol was uniquely designed to enable extremely stringent washes. The washing step efficiently removes non-covalently bound RNA species and proteins thus significantly reducing the background signal commonly observed in conventional CLIP protocols that rely on antibody-antigen interactions ( <a href="#fig2">Figure 2</a>).</p>
<div class="note">
<span class="note-title">Note:</span> All wash steps are done in 1.2 mL volume rotating for 5 minutes. Use a magnetic separator to capture the beads and discard the supernatant. The wash buffers do not include RNase inhibitors and rely on the presence of 0.1% SDS (or higher concentration) which irreversibly inactivate RNase I (See manufacturer’s instructions, <a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%253A%252F%252Fassets.thermofisher.com%252FTFS-Assets%252FLSG%252Fmanuals%252FMAN0012009_RNase_I_UG.pdf&amp;title=VXNlciBHdWlkZTogUk5hc2UgSQ==">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2FMAN0012009_RNase_I_UG.pdf&amp;title=VXNlciBHdWlkZTogUk5hc2UgSQ==</a>).</div>
<ol start="16">
<li>Wash samples twice in PBS-Urea wash buffer at 21°C–24°C.</li>
<li>Wash samples twice PBS-SDS wash buffer at 21°C–24°C.</li>
<li>Wash once in PBS high-salt wash buffer at 4°C.</li>
<li>Wash once in PBS low-salt wash buffer at 4°C.</li>
</ol>
<h3 id="sec3.4">DNase treatment</h3>
<p>The dCLIP protocol distinctively includes three DNase treatment steps. The first DNase treatment in step 8 promotes the dissociation of proteins tightly bound to chromatin such as CBX7 (<a href="#bib3">El Messaoudi-Aubert et al., 2010</a>). The second DNase treatment, similar to conventional CLIP protocols, aims to remove all DNA linked to RNA-protein of interest complexes since the UV crosslinking step crosslinks nucleic acid residues, RNA and DNA alike, to associated proteins. In the third DNase treatment, added here, the naked RNA molecules at the end of the precipitation step are treated to remove any DNA contaminates immediately prior to library construction (<a href="#bib1">Darnell, 2012</a>).</p>
<ol start="20">
<li>Resuspend the beads in 800 μL 1xDNase buffer and transfer to 1.5 mL tubes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step aims to remove the remaining detergents used in previous wash steps that might inhibit the DNase activity.</div>
<ol start="21">
<li>Rotate 5 min at 4°C. Capture C1-beads and discard the supernatant.</li>
<li>Resuspend the C1-beads in 160 μL of DNase mix and rotate 30 min at 37°C.</li>
<li>Spin down briefly to collect the liquid retained in the lid back to the main volume, capture C1-beads, and discard the supernatant.</li>
<li>Wash C1-beads once in PBS low-salt wash buffer. Rotate 5 min at 4°C.</li>
</ol>
<h3 id="sec3.5">3′-end dephosphorylation, 5′-end phosphorylation, and RNA extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5–2.5 h</div>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h for membrane RNA elution</div>
<p>The 3′-end dephosphorylation and 5′-phosphorylation (steps 25 and 26) prepare the RNA fragments for proper ligation of 5′ and 3′ adapters in the <a href="#sec3.6">library construction</a> section.</p>
<div class="critical">
<span class="critical-title">Critical:</span> RNA extraction can be done directly from C1-beads or after running the samples on an SDS-PAGE gel. Direct RNA elution from C1-beads can be done on cell lines growing in monolayer such as HEK293, mouse embryonic fibroblasts, and differentiating mouse ES cells. It is recommended to use the optional Membrane RNA elution step for undifferentiated mouse ES colonies grown on a feeder layer (<a href="#bib8">McElroy and Reijo Pera, 2008</a>).</div>
<p>In our experience, there’s a significantly higher background due to non-specific RNA binding when undifferentiated mouse ES colonies RNA is eluted directly from beads, as shown in <a href="#fig4">Figure 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1018-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Comparison between bead elution and membrane elution dCLIP methods</div>
<p>dCLIP libraries for CBX7 were prepared in Day 0 undifferentiated ES cells growing in colonies on feeder layer (A) and differentiating Day 7 ES cells growing in monolayer (B). The libraries were sequenced, aligned to reference mouse genome and the RNA binding profiles for CBX7-expressing cells vs control cells were generated and visualized in Integrative Genomic Viewer. Dusp9 RNA dCLIP profile in Day 0 cells (A) and Day 7 cells (B) were examined to assess differences between two RNA extraction methods – elution directly from beads vs. SDS-PAGE purification, nitrocellulose membrane transfer, and elution of RNA from the membrane. Note that control cells exhibited a very high background signal after beads elution protocol had been applied in Day 0 cells but not in Day 7 cells. RING1 And YY1 Binding Protein (RYBP) protein served as a control for dCLIP specificity since, compared to CBX7, RYBP is predicted to possess low RNA binding activity (<a href="#bib11">Tavares et al., 2012</a>).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> All washes are done for 5 min, rotating at 21°C–24°C.</div>
<ol start="25">
<li>3′ end dephosphorylation
<ol type="a">
<li>Wash C1-beads once in 1 mL Low pH PNK buffer. Discard supernatant.</li>
<li>Incubate C1-beads in 80 μL Low-pH-PNK mix 20 min at 37°C, shaking at 1,000 rpm for 15 s every 2 min.</li>
<li>Capture C1-beads and discard the supernatant.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If choosing the Membrane RNA elution option use the Optional steps to replace steps 26 and 27.</div>
<ol start="26">
<li>RNA 5′ end phosphorylation.
<ol type="a">
<li>Wash C1-beads once in 1 mL PNK buffer. Discard supernatant.</li>
<li>Incubate C1-beads in 80 μL PNK mix 20 min at 37°C, shaking samples at 1,000 rpm for 15 s every 2 min</li>
<li>Capture C1-beads and discard the supernatant.
<div class="optional">
<span class="optional-title">Optional:</span> RNA 5′ end phosphorylation for membrane RNA elution.</div>
<div class="note">
<span class="note-title">Note:</span> Since RNA 5′ end phosphorylation for membrane RNA elution involves radioactive material, the work should be performed according to the safety regulations specified by the institution or facility where the work is conducted.</div>
All washes are done for 5 min, rotating at 4°C.</li>
<li>Wash C1-beads once in 1 mL of PNK buffer.</li>
<li>Incubate C1-beads in 80 μL PNK mix 2–10 min at 37°C.</li>
<li>Add 8 μL 10 mM ATP and incubate 20 min at 37°C.</li>
<li>Capture C1-beads and discard the supernatant.</li>
<li>Wash C1-beads 3 times with 1 mL of ice-cold PNK wash buffer.</li>
<li>Incubate C1-beads in 85 μL 1xSDS sample buffer 5 min at 95°C (80 mM Tris pH 6.8, 2% SDS, 100 mM DTT, 10% Glycerol)
<div class="optional">
<span class="optional-title">Optional:</span> The 1xSDS sample buffer can be substituted with 85 μL NuPage 3xLDS sample buffer. Incubate at 80°C for 10 min. 1xNuPage-LDS buffer is not sufficient to disrupt streptavidin – biotin interactions at 80°C thus precluding an elution of protein-RNA complexes.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store samples at −20°C</div>
</li>
<li>To continue reheat samples for 10 min at 80°C or 5 min 95°C (depending on the sample buffer used)</li>
<li>Skip steps 27 and 28 and continue with the ‘Optional: Membrane RNA elution’ steps.</li>
</ol>
</li>
</ol>
<ol start="27">
<li>RNA extraction from C1-beads
<ol type="a">
<li>Wash C1-beads twice with 1 mL PBS-Urea wash buffer</li>
<li>Wash C1-beads once with 1 mL Proteinase K buffer</li>
<li>Incubate C1-beads in 200 μL Proteinase K buffer supplemented with 1 mg/mL Proteinase K, 30 min at 55°C, rotating.</li>
</ol>
</li>
</ol>
<ol start="28">
<li>RNA precipitation
<div class="optional">
<span class="optional-title">Optional:</span> Membrane RNA elution</div>
<ol type="a">
<li>RNA transfer to nitrocellulose membrane
<ol type="i">
<li>Load 40 μL per lane in two adjacent lanes on 1.5 mm NuPage 4%–12% Bis-Tris gradient gels. Run gels in NuPage MOPS/SDS running buffer on 200 V.</li>
<li>Transfer samples to nitrocellulose membrane for 1.5 h, 140 mA at 4°C in NuPage 1× transfer buffer + 12% methanol.</li>
<li>Expose the membrane to a phosphoimager screen.</li>
</ol>
</li>
<li>RNA elution from the membrane
<div class="critical">
<span class="critical-title">Critical:</span> The excised membrane pieces containing protein-RNA complexes should start in the area at least 10 kda above the size of protein of interest (corresponding to a protein of interest covalently bound to RNA fragments larger than 30 bp. Excise a similar area from control samples (Illustrated in <a href="#fig2">Figure 2</a>A, white box).</div>
<ol type="i">
<li>Excise membrane pieces (two lanes/sample). Cut into small pieces of 0.5–2 mm with a clean blade on the surface of sterile plastic 100 mm dish, and transfer to 1.5 mL low-binding 1.5 mL tubes.</li>
<li>Incubate membrane pieces in 200 μL PK solution for 20 min at 55°C with constant shaking at 1,200 rpm in Thermomixer.</li>
<li>Add 200 μL of PK-urea solution to membrane pieces. Incubate 20 min at 55°C with constant vortexing at 1,200 rpm.</li>
<li>Spin the samples 1 min at maximum speed to pellet membrane pieces and transfer supernatant to phase-lock tubes.</li>
<li>Proceed to RNA precipitation steps 28 c-o.</li>
<li>Dissolve the RNA in 25 μL DNase mix 2. Then combine samples of the same cell type that were treated with different RNase concentrations (see step 9) to a final volume of 50 μL.</li>
<li>Proceed to steps 28 q-s.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this step samples from the same cell type (see step 9) that were treated with different RNase concentrations are combined at a final volume of 400 μL.</div>
<ol start="3" type="a">
<li>Centrifuge phase-lock gel 2 mL tubes, 30 s 16,000 <i>g</i> to pellet gel.</li>
<li>Load 400 μL eluted RNA samples.</li>
<li>Add 400 μL acidic phenol-chloroform and shake vigorously.</li>
<li>Centrifuge 5 min 16,000 <i>g</i> at 21°C–24°C.</li>
<li>Repeat steps c-d.</li>
<li>Transfer the upper aqueous phase to a new low-binding 1.5 mL tube.</li>
<li>Add 40 μL 3M Sodium Acetate, 1 μL Glycoblue, and 1 mL 100% ethanol,
<div class="critical">
<span class="critical-title">Critical:</span> Adding GlycoBlue is essential for visualizing the small RNA pellet before the elution step. The RNA pellet can easily detach from the tube’s low-binding surface during steps involving the removal of the supernatant.</div>
</li>
<li>Mix well and incubate 16 h at −20°C.</li>
<li>Centrifuge 30 min 13,523 <i>g</i> at 4°C. Discard supernatant.</li>
<li>Wash pellets in 1 mL 75% ethanol.</li>
<li>Centrifuge 10 min 13,523 <i>g</i> at 4°C. Discard supernatant.</li>
<li>Spin down and manually remove the remaining ethanol with a pipettor and 20 μL tip without lifting the pellet.</li>
<li>Air dry pellets 5–10 min at 21°C–24°C or 5 min in the laminar flow hood.</li>
<li>Dissolve pellets in 50 μL DNase mix 2 .</li>
<li>Incubate 30 min at 37°C.</li>
<li>Add 950 μL Trizol reagent and follow manufacturer instructions for RNA extraction. Add 0.5 μL GlycoBlue to the precipitation step.</li>
<li>Elute RNA pellets in 8 μL nuclease-free ddH2O.</li>
</ol>
</li>
</ol>
<h3 id="sec3.6">Library construction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>Library construction is the first step of NextGen sequencing. In this step, multiplex adapters are added to uniquely barcode each RNA molecule and the samples are prepared to enable a high-throughput short-read sequencing on the Illumina platform.</p>
<ol start="29">
<li>For library preparation use Multiplex Compatible NEBNext Small RNA Library Prep according to manufacturer instructions with the following modifications (<a href="https://www.neb.com/protocols/2018/03/27/protocol-for-use-with-nebnext-small-rna-library-prep-set-for-illumina-e7300-e7580-e7560-e7330">https://www.neb.com/protocols/2018/03/27/protocol-for-use-with-nebnext-small-rna-library-prep-set-for-illumina-e7300-e7580-e7560-e7330</a>):
<ol type="a">
<li>Use 7 μL RNA for library preparation.</li>
<li>Dilute all adapters and primers 12-fold, since the initial amount of eluted RNA in the dCLIP protocol is significantly less than the initial amount specified by the manufacturer.</li>
<li>Use 20 μL of adapter-ligated RNA for cDNA preparation. Take 8 μL for the No-RT negative controls.
<div class="note">
<span class="note-title">Note:</span> Out of convenience, we replaced the kit’s M-MuLV reverse transcriptase and Murine RNase inhibitor, with SuperScript III Reverse Transcriptase (Life Technologies) and Protector RNase inhibitor (Roche) respectively.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store the cDNA at −20°C .</div>
</li>
<li>Amplify cDNA with Illumina multiplexed primers for 25 cycles using LongAmp_Taq 2X Master Mix.
<div class="note">
<span class="note-title">Note:</span> No-RT negative controls were amplified for 35 cycles to control for contamination.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store the amplified DNA at −20°C .</div>
</li>
<li>PCR product isolation and size selection.
<ol type="i">
<li>Run PCR products on 6% TBE-acrylamide gel.</li>
<li>Excise gel pieces ranging from 150 bp to 600 bp. Avoid cutting close to the adapter dimers area (<a href="#fig5">Figure 5</a>A).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1018-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Representative size profile for amplified dCLIP library</div>
<p>(A and B) Size selection of amplified dCLIP libraries for CBX7 is shown in (A). Amplified libraries were subjected to electrophoresis on 6% TBE-acrylamide gels and the area corresponding to the white rectangle was excised from each lane to avoid contamination with adapter dimers. PCR products were eluted and analyzed on the Bioanalyzer using a High Sensitivity DNA chip (B).</p>
<p>(C) Following alignment of the sequencing reads obtained from dCLIP samples, strand-specific enriched peaks (depicting RNA-binding sites) from three replicate dCLIP libraries for CBX7 were pooled, and overlapped peaks were merged into long enrichment regions in a strand-specific manner. Length distribution frequency of the enriched dCLIP peaks, as well as mean, median, and standard deviation were calculated.</p>
</figcaption>
</figure>
</li>
<li>Transfer to 1.5 mL low-binding tubes with 400 μL Gel elution buffer (NEB) and crush using a 1 mL syringe plunger.</li>
<li>Rotate overnight at 21°C–24°C to elute the PCR products from a gel.
<div class="optional">
<span class="optional-title">Optional:</span> Elute DNA for 2 hrs at 37°C, or 4 hrs at 30°C.</div>
</li>
<li>Spin down and load the sup on a glass microfiber filter placed in a Costar SpinX tube to remove the remaining gel pieces. Centrifuge 1 min 15,871 <i>g</i> at 21°C–24°C.</li>
<li>Follow Ethanol precipitation steps 28 g-m.
<div class="note">
<span class="note-title">Note:</span> long 16 h incubation after adding ethanol is not necessary in this step. 1 h incubation on −20°C is sufficient.</div>
</li>
<li>Resuspend pellets in 12 μL of nuclease-free water.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> AMPure beads can be used for size selection and isolation of amplified DNA, although we have not tried it in our hands for dCLIP protocol. Follow the manufacturer’s recommendations for the appropriate beads to sample ratios (<a href="https://www.beckman.com/reagents/genomic/cleanup-and-size-selection/pcr/bead-ratio">https://www.beckman.com/reagents/genomic/cleanup-and-size-selection/pcr/bead-ratio</a>).</div>
<ol start="30">
<li>Library QC
<ol type="a">
<li>Run 1 μL of each sample on a High Sensitivity DNA chip or Tapestation to determine sample size distribution. A typical size profile is shown in <a href="#fig5">Figure 5</a>B.</li>
<li>Use the KAPA Illumina Library Quantification kit to quantify the PCR products.</li>
</ol>
</li>
<li>Pool 3 to 5 samples to one multiplexed sample. Combine equivalent amounts of 1.5 nM–2 nM of each sample.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To obtain sufficient coverage, we used one lane of Illumina HiSeq 2000 apparatus for library sequencing to render 200–240 million 50 bp paired-end reads.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A successful outcome of dCLIP experiment is shown in <a href="#fig2">Figures 2</a>, <a href="#fig4">4</a>, and <a href="#fig5">5</a>. First, the clear radioactive signal at and above the size of protein of interest, compared to control must be obtained, as demonstrated in <a href="#fig2">Figures 2</a>A and 2D for CBX7 and EZH2 proteins respectively. The unsuccessful small-scale dCLIP experiment is shown in <a href="#fig3">Figure 3</a> for EED protein, and in this case, we do not recommend proceeding further. The RNase-protection step in dCLIP protocol enables mapping of RNA-binding sites with superior resolution and accuracy (<a href="#fig4">Figures 4</a> and <a href="#fig5">5</a>C). A similar high resolution was achieved with additional proteins (<a href="#bib9">Rosenberg et al., 2021</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>For protein expression analysis presented in <a href="#fig1">Figure 1</a>, protein bands were developed using Western Lightning Plus -ECL Kit (Perkin-Elmer), and the signal intensity was analyzed using Chemidoc MP Imaging System (Bio-Rad) and ImageLab Ver. 5.2.1 software (Bio-Rad). Exposures were captured on different time points using ChemiDoc cumulative signal option to avoid signal saturation. Densitometry analysis was conducted using ImageLab software (Bio-Rad). All dCLIP experiments were performed in at least n=2 biological replicates (two separate clonal cell lines). No replicates were excluded from analyses presented, and all attempts to replicate the data were successful.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The dCLIP RNA library is represented by non-abortive reverse transcription products only. Protein-RNA covalent bonds created by UV cross-linking are resistant to Proteinase K treatment, resulting in protein residues or small peptides remaining at cross-linking points in eluted RNA. In course of reverse transcription, reverse-transcriptase can either pass protein-RNA cross-linking sites uninterrupted, bypass a protein-RNA cross-linking sites by introducing a random nucleotide, or be stalled and fall off, resulting in abortive cDNA species (<a href="#bib2">Darnell, 2010</a>; <a href="#bib6">Konig et al., 2011</a>).</p>
<p>Small RNA library protocol employed in the dCLIP method relies on adapter ligation to both sides of RNA fragment. Thus, for dCLIP fragments to be amplified from both adapters after cDNA synthesis, reverse transcription should go into completion. Lastly, contamination by natively biotinylated proteins that interact directly with streptavidin beads may occur.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Inefficient pull-down of biotinylated protein of interest in step 14.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>The efficient pull-down of AVI-tagged protein depends on (1) the accessibility of the AVI-tag to the BirA biotin ligase inside the cells and (2) the accessibility of the biotinylated tag to the streptavidin beads during the pull-down step. First, the AVI tag can be fused either to the N-terminus or the C-terminus of the protein of interest, with subsequent assessment of pull-down efficiency. To improve the accessibility of streptavidin to the biotinylated tag, we suggest increasing the SDS concentration to 1% at the end of the RNase digestion step. This should increase protein unfolding resulting in improved accessibility to the biotinylated protein tag.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>A high background signal in control dCLIP samples compared to dCLIP experimental samples following dCLIP, beads elution, library sequencing, and mapping to the reference genome (<a href="#fig4">Figure 4</a>A, beads-eluted samples of EL16.7 Day 0 cells).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>To minimize the background signal we highly recommend conducting the membrane elution option. When choosing this option one needs to take all required precautions that involve working with radioactive isotopes as specified by radiation safety regulations. Resolving the protein-RNA complexes on gel aids the isolation of a sample subset that corresponds to the RNA-protein of interest complexes (<a href="#fig2">Figure 2</a>A). Furthermore, only RNA species associated with proteins are expected to bind to a nitrocellulose membrane. Non-covalent bound RNAs will diffuse to the surrounding buffer during the transfer step resulting in higher sample purity.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>An excessive amount of adapter dimers in the amplified library – step 29d, <a href="#fig5">Figure 5</a>A.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>One of the main reasons for an excessive amount of adapter dimers is the substantial loss of input RNA throughout the procedure. Consequently, we consistently utilized GlycoBlue reagent to stain RNA pellets in order to avoid removing a pellet during a supernatant collection after ethanol precipitation and centrifugation steps. Also, avoid using low-binding tubes during RNA extraction and 3<sup>rd</sup> DNAse digestion steps because of the very loose attachment of RNA pellets to the sides.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Contamination with bacterial ribosomal RNA during the RNA elution from membrane step.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>We and others noticed that some nitrocellulose membrane brands are the source of bacterial ribosomal RNA contamination after membrane extraction (<a href="#fig5">Figure 5</a>A. blank sample, (<a href="#bib12">Van Nostrand et al., 2017</a>)). We highly recommend using the nitrocellulose membranes brands recommended by Van Nostrand and colleagues to reduce a background signal (Amersham Protran 0.2 μm nitrocellulose membrane).</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Small-scale dCLIP experiment renders very weak radioactive signal compared to No UV control.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>While a very weak dCLIP signal compared to No UV control can serve as evidence that the protein of interest is an extremely weak RNA binder, it is important to confirm that there is no technical failure on the execution side, especially for inexperienced users. Hence, to establish the dCLIP protocol in the lab, we recommend choosing a known RNA-binding protein for pilot dCLIP experiments. Even a conventional CLIP experiment with a specific antibody could suffice. In our case, we conducted a conventional CLIP assay with MOV10 RNA helicase and confirmed that we had been able to obtain a strong radioactive signal at the desired location before continuing to the dCLIP protocol.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Darnell, R. (2012). CLIP (cross-linking and immunoprecipitation) identification of RNAs bound by a specific protein. Cold Spring Harb Protoc. <i>2012</i>, 1146-1160. <a class="external-link" href="https://doi.org/10.1101/pdb.prot072132">https://doi.org/10.1101/pdb.prot072132</a></p>
<p id="bib2">Darnell, R.B. (2010). HITS-CLIP: panoramic views of protein-RNA regulation in living cells. Wiley Interdiscip. Rev. RNA <i>1</i>, 266-286. <a class="external-link" href="https://doi.org/10.1002/wrna.31">https://doi.org/10.1002/wrna.31</a></p>
<p id="bib3">El Messaoudi-Aubert, S., Nicholls, J., Maertens, G.N., Brookes, S., Bernstein, E., and Peters, G. (2010). Role for the MOV10 RNA helicase in polycomb-mediated repression of the INK4a tumor suppressor. Nat. Struct. Mol. Biol. <i>17</i>, 862-868. <a class="external-link" href="https://doi.org/10.1038/nsmb.1824">https://doi.org/10.1038/nsmb.1824</a></p>
<p id="bib4">Furuyama, T. and Henikoff, S. (2006). Biotin-tag affinity purification of a centromeric nucleosome assembly complex. Cell Cycle <i>5</i>, 1269-1274. <a class="external-link" href="https://doi.org/10.4161/cc.5.12.2889">https://doi.org/10.4161/cc.5.12.2889</a></p>
<p id="bib5">Kim, J., Cantor, A.B., Orkin, S.H., and Wang, J. (2009). Use of in vivo biotinylation to study protein-protein and protein-DNA interactions in mouse embryonic stem cells. Nat. Protoc. <i>4</i>, 506-517. <a class="external-link" href="https://doi.org/10.1038/nprot.2009.23">https://doi.org/10.1038/nprot.2009.23</a></p>
<p id="bib6">Konig, J., Zarnack, K., Rot, G., Curk, T., Kayikci, M., Zupan, B., Turner, D.J., Luscombe, N.M., and Ule, J. (2011). iCLIP--transcriptome-wide mapping of protein-RNA interactions with individual nucleotide resolution. J. Vis. Exp. <a class="external-link" href="https://doi.org/10.3791/2638">https://doi.org/10.3791/2638</a></p>
<p id="bib7">Lee, J.T. and Lu, N. (1999). Targeted mutagenesis of Tsix leads to nonrandom X inactivation. Cell <i>99</i>, 47-57. <a class="external-link" href="https://doi.org/10.1016/s0092-8674(00)80061-6">https://doi.org/10.1016/s0092-8674(00)80061-6</a></p>
<p id="bib14">Long, Y., Bolanos, B., Gong, L., Liu, W., Goodrich, K.J., Yang, X., Chen, S., Gooding, A.R., Maegley, K.A., Gajiwala, K.S., et al. (2017). Conserved RNA-binding specificity of polycomb repressive complex 2 is achieved by dispersed amino acid patches in EZH2. Elife <i>6</i>, e31558. <a class="external-link" href="https://doi.org/10.7554/eLife.31558">https://doi.org/10.7554/eLife.31558</a></p>
<p id="bib8">McElroy, S.L. and Reijo Pera, R.A. (2008). Preparation of mouse embryonic fibroblast feeder cells for human embryonic stem cell culture. CSH Protoc. <i>2008</i>. <a class="external-link" href="https://doi.org/10.1101/pdb.prot5041">https://doi.org/10.1101/pdb.prot5041</a></p>
<p id="bib9">Rosenberg, M., Blum, R., Kesner, B., Aeby, E., Garant, J.M., Szanto, A., and Lee, J.T. (2021). Motif-driven interactions between RNA and PRC2 are rheostats that regulate transcription elongation. Nat. Struct. Mol. Biol. <i>28</i>, 103-117. <a class="external-link" href="https://doi.org/10.1038/s41594-020-00535-9">https://doi.org/10.1038/s41594-020-00535-9</a></p>
<p id="bib10">Rosenberg, M., Blum, R., Kesner, B., Maier, V.K., Szanto, A., and Lee, J.T. (2017). Denaturing CLIP, dCLIP, pipeline identifies discrete RNA footprints on chromatin-associated proteins and reveals that CBX7 targets 3' UTRs to regulate mRNA expression. Cell Syst <i>5</i>, 368-385.e15. <a class="external-link" href="https://doi.org/10.1016/j.cels.2017.09.014">https://doi.org/10.1016/j.cels.2017.09.014</a></p>
<p id="bib11">Tavares, L., Dimitrova, E., Oxley, D., Webster, J., Poot, R., Demmers, J., Bezstarosti, K., Taylor, S., Ura, H., Koide, H., et al. (2012). RYBP-PRC1 complexes mediate H2A ubiquitylation at polycomb target sites independently of PRC2 and H3K27me3. Cell <i>148</i>, 664-678. <a class="external-link" href="https://doi.org/10.1016/j.cell.2011.12.029">https://doi.org/10.1016/j.cell.2011.12.029</a></p>
<p id="bib12">Van Nostrand, E.L., Nguyen, T.B., Gelboin-Burkhart, C., Wang, R., Blue, S.M., Pratt, G.A., Louie, A.L., and Yeo, G.W. (2017). Robust, cost-effective profiling of RNA binding protein targets with single-end enhanced crosslinking and immunoprecipitation (seCLIP). Methods Mol. Biol. <i>1648</i>, 177-200. <a class="external-link" href="https://doi.org/10.1007/978-1-4939-7204-3_14">https://doi.org/10.1007/978-1-4939-7204-3_14</a></p>
<p id="bib13">Weber, P.C., Ohlendorf, D.H., Wendoloski, J.J., and Salemme, F.R. (1989). Structural origins of high-affinity biotin binding to streptavidin. Science <i>243</i>, 85-88. <a class="external-link" href="https://doi.org/10.1126/science.2911722">https://doi.org/10.1126/science.2911722</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jeannie T. Lee (<a href="mailto:lee@molbio.mgh.harvard.edu">lee@molbio.mgh.harvard.edu</a>).</p>
<h4>Materials availability</h4>
<p>All unique/stable reagents generated in this study are available from the lead contact with a completed materials transfer agreement.</p>
<h4>Data and code availability</h4>
<p>This study did not generate a new dataset or code for analysis.</p>
<h3>Acknowledgments</h3>
<p>We thank the lab members for helpful advice and many important discussions. We also give special thanks to U. Kim and A. Maisonet for excellent technical assistance with deep sequencing. This work was funded by the <a href="https://doi.org/10.13039/100000011">HHMI</a> and <a href="https://doi.org/10.13039/100000002">NIH</a> R01-HD097665 to J.T.L.</p>
<h3>Author contributions</h3>
<p>M.R., V.K.M., and J.T.L. conceived the idea of utilizing <i>in vivo</i> biotinylation to improve the conventional CLIP protocol. M.R. designed the dCLIP method, with V.K.M.’s initial involvement. B.K. aligned sequencing data. R.B. devised the bioinformatics pipeline and performed the computational analysis for the identification of consensus motifs. M.R., V.L., and J.T.L. drafted figures and wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>A patent related to dCLIP has been filed by the MGH. J.T.L. is a cofounder of Translate Bio and Fulcrum and an Advisor to Skyhawk Therapeutics.</p>
</section>
</article>