<article>
<section>
<h1 id="header">Generation of mixed murine organoids to model cellular interactions</h1>
<p><time datetime="2021-12-10">Published: December 10, 2021</time></p>
<p>Ana Krotenberg Garcia,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Jacco van Rheenen,<sup><a href="#aff1">1</a></sup> and Saskia Jacoba Elisabeth Suijkerbuijk<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Pathology, Oncode Institute, Netherlands Cancer Institute, Amsterdam 1066 CX, The Netherlands</p>
<p id="fn1"><sup>2</sup>Present address: Division of Developmental Biology, Institute of Biodynamics and Biocomplexity, Department of Biology, Faculty of Science, Utrecht University, Utrecht, The Netherlands</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:a.krotenberg.garcia@nki.nl">a.krotenberg.garcia@nki.nl</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:s.j.e.suijkerbuijk@uu.nl">s.j.e.suijkerbuijk@uu.nl</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100997">10.1016/j.xpro.2021.100997</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Cell competition is a mechanism of interaction that dictates cell selection based on differences in cellular fitness. We designed a protocol to generate mixed murine organoids and enteroid monolayers used to study such complex cellular interactions in a mammalian system. This protocol is dedicated to follow the behavior of different cell populations over time, using (time-lapse) microscopy or transcriptome/proteome analysis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.celrep.2021.109307">Krotenberg Garcia et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol to generate 3D mixed murine organoids and enteroid monolayers</li>
<li>Analysis of complex cellular interactions, such as cell competition</li>
<li>Follow the behavior of different cell populations over time</li>
<li>Adaptable to other organoid cultures derived from different tissues</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>In tissues, cellular interactions are essential for quality control, both during development and in homeostasis. In particular, competitive interactions can drive selection and thereby determine cell fate, growth and survival (<a href="#bib19">Vishwakarma and Piddini, 2020</a>). This mechanism was historically modeled in <i>Drosophila</i> (<a href="#bib16">Simpson, 1979</a>; <a href="#bib17">Suijkerbuijk et al., 2016</a>), Madin–Darby canine kidney (MDCK) epithelial cells (<a href="#bib7">Hogan et al., 2009</a>; <a href="#bib13">Norman et al., 2012</a>) and more recently inducible mouse models (<a href="#bib9">Kon et al., 2017</a>; <a href="#bib3">Flanagan et al., 2021</a>; <a href="#bib6">Hill et al., 2021</a>; <a href="#bib12">van Neerven et al., 2021</a>). Here, we describe a readily adaptable method to generate mixed organoid cultures, which mimic the close interaction between different cell populations.</p>
<p>This protocol describes the generation of mixed organoids and enteroid monolayers from two types of murine small intestine organoid cultures; wild-type cells (<a href="#bib10">Krotenberg Garcia et al., 2021</a>) and <i>Apc</i><sup>−/−</sup><i>Kras</i><sup>G12D/WT</sup><i>Trp53</i><sup>-/R172H</sup> cancer cells (<a href="#bib4">Fumagalli et al., 2017</a>). Previously we have extended the use of the protocol to other colorectal cancer lines (<a href="#bib8">Jackstadt et al., 2019</a>; <a href="#bib10">Krotenberg Garcia et al., 2021</a>). Based on the flexibility of the protocol it can be adapted to generate mixed organoids from other cystic-growing cultures derived from different tissues. The individual organoid cultures described in this protocol were derived from different donor mice with a similar genetic background (C57BL/6JRj). However, we have previously generated mixed organoids from cultures with a different genetic background and did not observe aberrant behavior.</p>
<p>For successful execution of this protocol and subsequent analysis, labeling of both cell populations is essential. In particular, when loss (out-competition) of one population is expected. Here we describe the use of wild-type intestine cells labeled with membrane-bound tdTomato (derived from mTmG transgenic mice (<a href="#bib11">Muzumdar et al., 2007</a>)) and cancer cells labeled with Dendra2 (introduced by lentiviral transduction (<a href="#bib5">Heinz et al., 2020</a>)).</p>
<p>Furthermore, the ratio of cells that are added to the mixing procedure is dependent on the characteristics of the original organoid cultures. For the intestinal model described here a 2:1 ratio (wild-type : cancer) is optimal. This ratio, which was based on the difference in proliferation rate between the individual cell populations and validated experimentally, provides equal competition potential to both populations. In addition, other characteristics such as cell ratios that naturally occur in organ tissue can serve as a guideline.</p>
<p>This protocol has broad applications and can be used to study cellular interactions on an organoid and population basis. Involvement of specific molecular pathways can be revealed through a combination with chemical or genetic manipulation or downstream analysis such as (single cell) mRNA sequencing or proteomics.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>anti-Phospho-c-Jun (Ser73) (D47G9) (Dilution 1:500)</td>
<td>Cell Signaling TECHNOLOGY</td>
<td>3270; RRID:AB_2129575</td>
</tr>
<tr>
<td>anti-Cleaved Caspase-3 (Asp175) (Dilution 1:400)</td>
<td>Cell Signaling TECHNOLOGY</td>
<td>9661; RRID:AB_2341188 06-570</td>
</tr>
<tr>
<td>anti-Aldolase B + Aldolase C antibody [EPR3138Y] (Dilution 1:300)</td>
<td>Abcam</td>
<td>ab75751; RRID:AB_2226682</td>
</tr>
<tr>
<td>anti-Lysozyme (EC 3.2.1.17) (Dilution 1:500)</td>
<td>Agilent</td>
<td>A0099; RRID:AB_2341230</td>
</tr>
<tr>
<td>anti-Olfm4 (D6Y5A) (1:100)</td>
<td>Cell Signaling TECHNOLOGY</td>
<td>39141; RRID:AB_2650511</td>
</tr>
<tr>
<td>Chicken anti-Rabbit, Alexa Fluor 647 (Dilution 1:500)</td>
<td>Thermo Fisher Scientific</td>
<td>A21443; RRID:AB_2535861</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Cultrex PathClear Reduced Growth Factor Basement Membrane Extract Type 2</td>
<td>R&amp;D Systems</td>
<td>3533-005-02</td>
</tr>
<tr>
<td>Advanced DMEM F/12</td>
<td>Thermo Fisher Scientific</td>
<td>12634-010</td>
</tr>
<tr>
<td>HEPES</td>
<td>Thermo Fisher Scientific</td>
<td>15630-056</td>
</tr>
<tr>
<td>Penicillin/streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>15140-122</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>Thermo Fisher Scientific</td>
<td>35050-068</td>
</tr>
<tr>
<td>R-spondin1</td>
<td>prepared in house</td>
<td>n/a</td>
</tr>
<tr>
<td>Noggin</td>
<td>prepared in house</td>
<td>n/a</td>
</tr>
<tr>
<td>B-27</td>
<td>Thermo Fisher Scientific</td>
<td>17504-044</td>
</tr>
<tr>
<td>hEGF</td>
<td>PeproTech</td>
<td>AF-100-15</td>
</tr>
<tr>
<td>N-acetylcysteine</td>
<td>Sigma-Aldrich</td>
<td>A9165</td>
</tr>
<tr>
<td>Y-27632</td>
<td>AbMole BioScience</td>
<td>M1817</td>
</tr>
<tr>
<td>CHIR-99021</td>
<td>Tocris Bioscience</td>
<td>4423</td>
</tr>
<tr>
<td>TryplE</td>
<td>Thermo Fisher Scientific</td>
<td>12605-010</td>
</tr>
<tr>
<td>DPBS, no calcium, no magnesium</td>
<td>Thermo Fisher Scientific</td>
<td>14190144</td>
</tr>
<tr>
<td>PBS tablets</td>
<td>Gibco</td>
<td>18912014</td>
</tr>
<tr>
<td>Paraformaldehyde 16% (w/v) in aqueous solution methanol-free</td>
<td>Alfa Aesar</td>
<td>43368.9M; CAS RN:50-00-0</td>
</tr>
<tr>
<td>BSA</td>
<td>Roche</td>
<td>10735094001</td>
</tr>
<tr>
<td>4′,6-Diamidino-2-phenylindole Dihydrochloride (DAPI)</td>
<td>Toronto Research Chemicals</td>
<td>D416050; CAS RN: 28718-90-3</td>
</tr>
<tr>
<td>TX-100</td>
<td>Sigma-Aldrich</td>
<td>X-100; CAS RN: 9036-19-5</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>RNA-seq data</td>
<td><a href="#bib10">Krotenberg Garcia et al., 2021</a></td>
<td>NCBI GEO:GSE176027</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Murine WT small intestine organoidsGt(ROSA) 26Sortm4(ACTB-tdTomato,-EGFP)Luo/JMus</td>
<td><a href="#bib10">Krotenberg Garcia et al., 2021</a></td>
<td>n/a</td>
</tr>
<tr>
<td>Murine CRC organoidsVillin-CreERT2Apcfl/flKrasG12D/WTTr53fl/R1</td>
<td><a href="#bib4">Fumagalli et al., 2017</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FIJI</td>
<td><a href="https://imagej.net/">https://imagej.net/</a></td>
<td>2.1.0/1.53h 9.0.0</td>
</tr>
<tr>
<td>FlowJo 10.6.1</td>
<td>BD Biosciences</td>
<td><a href="https://www.flowjo.com/">https://www.flowjo.com/</a></td>
</tr>
<tr>
<td>Imaris</td>
<td>Oxford Instruments</td>
<td>9.3.1</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Cooled centrifuge</td>
<td>Eppendorf</td>
<td>Eppendorf Centrifuge 5702 RSerienr: 5703II619246</td>
</tr>
<tr>
<td>Eppendorf centrifuge</td>
<td>Eppendorf</td>
<td>Eppendorf AG 5424Serienr: 5424FR273064</td>
</tr>
<tr>
<td>Tissue culture microscope equipped with fluorescent light source</td>
<td>Thermo Fisher Scientific</td>
<td>EVOS FL</td>
</tr>
<tr>
<td>Inverted confocal microscope</td>
<td>Leica Microsystems</td>
<td>TCS SP8</td>
</tr>
<tr>
<td>Spinning disk confocal microscope</td>
<td>Andor</td>
<td>Dragonfly</td>
</tr>
<tr>
<td>Cell counter</td>
<td>Westburg</td>
<td>Model: LUNA IISerienr: LUC-04-00126</td>
</tr>
<tr>
<td>Pasteur Capillary Pipette, short size 150mm</td>
<td>VWR</td>
<td>612-1798</td>
</tr>
<tr>
<td>5 mL Polystyrene Round-Bottom Tube with 35 μm Cell-Strainer Cap</td>
<td>Falcon</td>
<td>352235</td>
</tr>
<tr>
<td>5 mL Polystyrene Round-Bottom Tube</td>
<td>Falcon</td>
<td>352063</td>
</tr>
<tr>
<td>15 mL tubes</td>
<td>Sarstedt</td>
<td>62554502</td>
</tr>
<tr>
<td>1.5 mL Safe-Lock Tubes</td>
<td>Eppendorf</td>
<td>0030120.086</td>
</tr>
<tr>
<td>24 well CELLSTAR® plate, polystyrene</td>
<td>Greiner Bio-One</td>
<td>662165</td>
</tr>
<tr>
<td>12 well CELLSTAR® plate, polystyrene</td>
<td>Greiner Bio-One</td>
<td>665165</td>
</tr>
<tr>
<td>6 well CELLSTAR® plate, polystyrene</td>
<td>Greiner Bio-One</td>
<td>657165</td>
</tr>
<tr>
<td>8 well chambered μ-Slide, glass-bottom</td>
<td>IBIDI</td>
<td>80827</td>
</tr>
<tr>
<td>96 well μ-Plate, ibiTreat #1.5 polymer coverslip</td>
<td>IBIDI</td>
<td>89626</td>
</tr>
<tr>
<td>96 well SensoPlate, glass-bottom</td>
<td>Greiner Bio-One</td>
<td>655892</td>
</tr>
<tr>
<td>384 well microplate, tissue culture treated</td>
<td>Greiner Bio-One</td>
<td>781091</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Culture media recipes</h3>
<p>According to <a href="#bib15">Sato et al. (2009)</a> and <a href="#bib18">Thorne et al. (2018)</a></p>
<table id="undtbl1">
<caption>Basic culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Advanced DMEM/F-12</td>
<td>n/a</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td>HEPES</td>
<td>1 M</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>100×</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>10.000 U/mL</td>
<td>100 U/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>515 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare in a sterile environment, store at 4°C for up to 2 months.</p>
<table id="undtbl2">
<caption>ENR culture medium (ENR: EGF, Noggin and RSPO1)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Basic culture medium</td>
<td>n/a</td>
<td>n/a</td>
<td>38.87 mL</td>
</tr>
<tr>
<td>B-27</td>
<td>50×</td>
<td>2%</td>
<td>1 mL</td>
</tr>
<tr>
<td>N-Acetyl-L-Cysteine</td>
<td>500 mM</td>
<td>1.25 mM</td>
<td>125 μL</td>
</tr>
<tr>
<td>R-spondin1 Conditioned Medium (CM)</td>
<td>n/a</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Noggin Conditioned Medium (CM)</td>
<td>n/a</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>human Epidermal Growth Factor (hEGF)</td>
<td>0.5 mg/mL</td>
<td>50 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare in a sterile environment, store at 4°C for up to 2 weeks.</p>
<div class="note">
<span class="note-title">Note:</span> For in house production of conditioned media refer to <a href="#bib2">Broutier et al. (2016)</a>. Alternatively, Noggin-FC fusion protein and R-Spondin 3-FC fusion protein conditioned medium can be used (U-Protein Express BV, Cat. No.#N002 and #R001).</div>
<table id="undtbl3">
<caption>Enteroid plating medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Basic culture medium</td>
<td>n/a</td>
<td>n/a</td>
<td>38.77 mL</td>
</tr>
<tr>
<td>B-27</td>
<td>50×</td>
<td>2%</td>
<td>1 mL</td>
</tr>
<tr>
<td>N-Acetyl-L-Cysteine</td>
<td>500 mM</td>
<td>1.25 mM</td>
<td>125 μL</td>
</tr>
<tr>
<td>R-spondin1 CM</td>
<td>n/a</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Noggin CM</td>
<td>n/a</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>hEGF</td>
<td>0.5 mg/mL</td>
<td>50 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>CHIR-99021</td>
<td>3 mM</td>
<td>3 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Y-27632</td>
<td>10 mM</td>
<td>10 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare in a sterile environment, store at 4°C for up to 2 weeks.</p>
<h3 id="sec2.2">Solution recipes</h3>
<table id="undtbl4">
<caption>Fixation solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Paraformaldehyde (methanol-free)</td>
<td>16%</td>
<td>4%</td>
<td>10 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>2× from tablets</td>
<td>1×</td>
<td>20 mL</td>
</tr>
<tr>
<td>MilliQ water</td>
<td>n/a</td>
<td>n/a</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at -20°C for up to 1 month, avoid freeze thaw cycles.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Paraformaldehyde is hazardous; wear gloves, work in chemical safety hood and dispose waste in accordance with local regulations.</div>
<table id="undtbl5">
<caption>Blocking solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>n/a</td>
<td>5%</td>
<td>2.5 g</td>
</tr>
<tr>
<td>TX-100</td>
<td>10%</td>
<td>0.2%</td>
<td>1 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>1× from tablets</td>
<td>1×</td>
<td>49 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 1 month.</p>
<table id="undtbl6">
<caption>Antibody incubation solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>n/a</td>
<td>2.5%</td>
<td>1.25 g</td>
</tr>
<tr>
<td>TX-100</td>
<td>10%</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>1× from tablets</td>
<td>1×</td>
<td>49.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50. mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 1 month.</p>
<table id="undtbl7">
<caption>Washing solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TX-100</td>
<td>10%</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>1× from tablets</td>
<td>1×</td>
<td>49.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at room temperature (18°C–25°C) for up to 6 months.</p>
<table id="undtbl8">
<caption>FACS buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>dPBS</td>
<td>n/a</td>
<td>n/a</td>
<td>9.75 mL</td>
</tr>
<tr>
<td>B-27</td>
<td>50×</td>
<td>2%</td>
<td>200 μL</td>
</tr>
<tr>
<td>N-Acetyl-L-Cysteine</td>
<td>500 mM</td>
<td>1.25 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>hEGF</td>
<td>0.5 mg/mL</td>
<td>50 ng/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>Y-27632</td>
<td>10 mM</td>
<td>10 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>DAPI</td>
<td>1 mg/mL</td>
<td>1 μg/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare in a sterile environment, store at 4°C for up to 2 weeks.</p>
<h3 id="sec2.3">Media and reagent preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ul>
<li>Thaw Reduced Growth Factor Basement Membrane Extract Type 2 (BME2) on ice or in a fridge overnight.</li>
<li>Incubate unpackaged 6, 12 and 24 well plates (depending on the type of experiment) untreated polystyrene culture plates in a tissue culture incubator at 37°C for at least 48h prior to use to promote formation of BME2 droplets during plating.</li>
<li>Prepare all buffers and media according to the tables in the “Materials and equipment” section and store appropriately (see notes).</li>
<li>Prepare glass Pasteur pipets by fire-polishing the tip of the pipets using a Bunsen burner (<a href="#fig1">Figure 1</a>). At least one pipet per individual culture should be prepared, spares are recommended.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Preparation of glass Pasteur pipets</div>
<p>(A) Fire-polish a glass Pasteur pipet by placing the tip in the flame of a Bunsen burner for a couple of seconds while continuously rolling the pipet.</p>
<p>(B and C) An example of a glass Pasteur pipet before (B) and after (C) fire-polishing. Scale bar = 1 mm</p>
</figcaption>
</figure>
</li>
<li>Cool centrifuges to 4°C.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Instead of BME2 other culture matrices (e.g. Matrigel or Collagen) can be used. Select appropriate matrix depending on organoid culture of choice. Specific characteristics of certain matrices might interfere with downstream analysis. For example, Collagen does not dissolve well in the fixation solution (Step 33) and might need additional permeabilization steps, or Matrigel containing phenol-red might cause problems with auto-fluorescence in time-lapse experiments. Optimization is recommended when alternative matrices are used.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Organoid culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days</div>
<p>This section describes the culture steps in preparation of small intestine wild-type and cancer organoids for generation of mixed cultures. For complete details on the isolation and culture of murine small intestine organoids please refer to (<a href="#bib15">Sato et al., 2009</a>; <a href="#bib14">Sato and Clevers, 2012</a>; <a href="#bib1">Andersson-Rolf et al., 2014</a>). For optimal and efficient execution of this protocol it is essential that the organoids have adapted to the experimental culture medium and are in a healthy state (<a href="#fig2">Figure 2</a>) prior to the start of the experiment.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Input organoid cultures</div>
<p>(A and B) Mature murine wild-type (A) and Cancer (B) organoid cultures passaged 3 days prior to the start of the experiment. Pay attention to the morphology of the organoids, which should have a smooth epithelium and lack apoptotic cells in their lumen. 2–6 crypt-like structures are optimal for wild-type small intestine cultures. (A) Membrane-bound tdTomato labelled wild-type cells (<a href="#bib10">Krotenberg Garcia et al., 2021</a>) were derived from healthy small intestine tissue. (B) Dendra2 labelled cancer cells were derived from <i>Apc</i><sup>−/−</sup><i>Kras</i><sup>G12D/WT</sup><i>Trp53</i><sup>-/R172H</sup> small intestinal tumors (<a href="#bib4">Fumagalli et al., 2017</a>).</p>
<p>Scale bars = 100 μm</p>
</figcaption>
</figure>
<ol>
<li>Culture the organoids in 3D in BME2 for at least 7 days in ENR culture medium. There is no maximum duration of this adaptation time. Passage the cultures when the lumen fills with apoptotic cells, approximately once every 3–4 days.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is essential that the experimental culture medium is uniform for all organoid cultures in the experiment and contains a combination of all factors that are required for the individual cultures. For example, the cancer population described in this protocol does not require hEGF or RSPO1, but is cultured in ENR medium, in order to avoid phenotypic changes during the experiment that could be caused by growth factors.</div>
<ol start="2">
<li>Passage organoid cultures 2–3 days prior to the start of the experiment until mature and healthy organoids are formed (<a href="#fig2">Figure 2</a>). Refer to <a href="#tbl2">Table 2</a> for guidance on input material.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We have not observed differences in outcome between early and late passage cultures. However, depending on the organoid cultures of choice the use of early passages might be required.</div>
<h3 id="sec3.2">Preparation of organoids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15–30 min</div>
<p>This section describes the preparation of small intestine wild-type and cancer organoids for mixed cultures. After Step 7, organoids are ready to proceed to generation of 3D mixed organoids (Part A) or generation of mixed enteroid monolayers (Part B).</p>
<div class="note">
<span class="note-title">Note:</span> Organoid cultures are very sensitive to fluctuations in temperature. Therefore, we recommend to use ice-cold reagents, keep cells on ice in between steps and use centrifuges that are cooled to 4°C throughout the protocol.</div>
<ol start="3">
<li>Harvest organoids by scraping the bottom of the well and pipetting up and down with a p1000 pipette using ice-cold Basic culture medium.</li>
<li>Collect the organoid suspension in 15 mL centrifuge tubes and keep on ice.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> To ensure harvest of all organoids, it is recommended to wash the well with another 1 mL of ice-cold Basic medium and collect leftovers.</div>
<ol start="5">
<li>Centrifuge at 300×<i>g</i> at 4°C for 5 min. Three clear layers should be formed; medium, BME2 and the pellet of organoids (top-to-bottom, <a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Preparation of organoids</div>
<p>Overview of the three phases formed by centrifugation of an organoid suspension. Medium (top), BME2 (middle) and organoid pellet (bottom).</p>
</figcaption>
</figure>
</li>
<li>Carefully remove the medium and BME2 layers without disrupting the organoid pellet.</li>
<li>Resuspend the pellet in 1 mL of ice-cold Basic culture medium.</li>
</ol>
<h3 id="sec3.3">Part A. Generation of 3D mixed organoids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>This section describes the main steps to generate 3D mixed organoids from small intestine wild-type and cancer cells. Once the 3D mixed organoids are formed, they can be used in different techniques such as cell sorting, immuno-fluorescence and time-lapse imaging, described later in this protocol.</p>
<ol start="8">
<li>Use a fire-polished glass Pasteur pipet to mechanically disrupt the organoids. Pipette up and down 10–20× while pressing the pipet tip against the bottom of the tube, until no clear structures are visible by eye. The suspension now contains clumps of 50–150 cells (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Generation of organoid clump suspension</div>
<p>Bright field examples of mechanically disrupted organoid cultures. Depicted are fragments that need more disruption (left), of optimal size (middle, indicated by arrow heads) or were disrupted too much (right).</p>
<p>Scale bar = 100 μm</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Immediately before use, sterilize glass Pasteur pipets by briefly passing them through a flame and coat the inside walls of the pipet by aspiration of Basic culture medium to avoid attachment of the organoids.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Optimal breaking of the organoids is essential. Therefore, it is recommended to monitor the status at a microscope after every 5–10× of pipetting. Stop the breaking of the organoids before they are single cells (<a href="#fig4">Figure 4</a>).</div>
<ol start="9">
<li>Centrifuge at 300×<i>g</i> at 4°C for 5 min.</li>
<li>Aspirate supernatant and resuspend the pellets in 300 μL ice-cold Basic culture medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The final ratio of cells in the mixed organoids is dependent on the input of the different cell populations (see <a href="#tbl2">Table 2</a>). Therefore, it is important to compare the pellet size and adjust the amount of Basic culture medium that is added in Step 10 accordingly (add 300 μL to the smallest pellet, increase volume X times for a X times bigger pellet).</div>
<ol start="11">
<li>Prepare one Eppendorf tube for each of the conditions (Pure population A, Pure population B and Mix Population A : B) and add the correct amount of cell clump suspension (<a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Recommended volumes of cell clump suspension for mixing of 3D organoids</caption>
<thead>
<tr>
<th>Condition</th>
<th>Ratio</th>
<th>Pure population A</th>
<th>Pure population B</th>
<th>Mix population A : B</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>1:1</td>
<td>
<b>Vial 1A:</b> 100 μL</td>
<td>
<b>Vial 1B:</b> 100 μL</td>
<td>
<b>Vial 1AB:</b> 100 μL : 100 μL</td>
</tr>
<tr>
<td>2</td>
<td>2:1</td>
<td>
<b>Vial 2A:</b> 100 μL</td>
<td>
<b>Vial 2B:</b> 50 μL</td>
<td>
<b>Vial 2AB:</b> 100 μL : 50 μL</td>
</tr>
<tr>
<td>3</td>
<td>3:1</td>
<td>
<b>Vial 3A:</b> 100 μL</td>
<td>
<b>Vial 3B:</b> 33 μL</td>
<td>
<b>Vial 3AB:</b> 100 μL : 33 μL</td>
</tr>
<tr>
<td>4</td>
<td>4:1</td>
<td>
<b>Vial 4A:</b> 100 μL</td>
<td>
<b>Vial 4B:</b> 25 μL</td>
<td>
<b>Vial 4AB:</b> 100 μL : 25 μL</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The efficiency of generating mixed organoids is not 100%. Therefore, it is recommended to include duplicates of mixed conditions in order to generate equal numbers of pure and mixed organoids.</div>
<div class="note">
<span class="note-title">Note:</span> The optimal ratio for mixing is dependent on the characteristics of the individual cultures, such as differences proliferation rate. It is recommended to include multiple ratios and validate the optimal mix conditions experimentally.</div>
<ol start="12">
<li>Centrifuge at 300×<i>g</i> at 4°C for 5 min</li>
<li>Carefully aspirate the supernatant with a p200 pipette and resuspend the small pellet in 10 μL of ENR medium.</li>
<li>Incubate the Eppendorf vials containing the concentrated cell clump suspensions at 37°C for 30 min to promote cell aggregation.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Aggregation of cells in a small volume is essential for efficient formation of mixed organoids. Do not increase the volume to more than 25 μL.</div>
<ol start="15">
<li>After the incubation time, dilute the aggregates with ENR medium (see <a href="#tbl2">Table 2</a> for volume).
<table id="tbl2">
<caption>Table 2. Recommended input and plating conditions for mixing of 3D organoids</caption>
<thead>
<tr>
<th colspan="3">Input per well for four conditions</th>
<th colspan="3">Aggregation per condition</th>
<th colspan="2">Plating per condition</th>
</tr>
<tr>
<th>Plate type</th>
<th>Size</th>
<th>No. organoids</th>
<th>Volume</th>
<th>+ENR medium</th>
<th>+BME2</th>
<th>No. drops</th>
<th>Drop size</th>
</tr>
</thead>
<tbody>
<tr>
<td>24-well (Greiner)</td>
<td>1.9 cm<sup>2</sup>
</td>
<td>250</td>
<td>5 μL</td>
<td>10 μL</td>
<td>30 μL</td>
<td>3</td>
<td>12 μL</td>
</tr>
<tr>
<td>12-well (Greiner)</td>
<td>3.9 cm<sup>2</sup>
</td>
<td>500</td>
<td>10 μL</td>
<td>15 μL</td>
<td>50 μL</td>
<td>6</td>
<td>12 μL</td>
</tr>
<tr>
<td>6-well (Greiner)</td>
<td>9.6 cm<sup>2</sup>
</td>
<td>1000–1500</td>
<td>20 μL</td>
<td>50 μL</td>
<td>140 μL</td>
<td>15</td>
<td>12 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Add double the volume of BME2 (compared to the diluted aggregates) to achieve a final ratio of 1:2 (see <a href="#tbl2">Table 2</a> for volume) and keep on ice.</li>
<li>Plate drops of 12 μL per well of an imaging-compatible plate. Output and drop size are based on use of a 96-well imaging plate (IBIDI). Adapt when different plates are used.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For sorting of cells from 3D organoids it is recommended to plate mixed organoids in a 6 well format to increase yield</div>
<div class="note">
<span class="note-title">Note:</span> The input material is indicated per one well, this is sufficient for four conditions (1× each of pure populations and 2× mix)</div>
<div class="note">
<span class="note-title">Note:</span> The volumes in <a href="#tbl2">Table 2</a> are guidelines. To avoid crowding it is recommended to plate one drop in a separate dish and monitor organoid density (<a href="#fig5">Figure 5</a>). Adjust the volume if needed.</div>
<div class="note">
<span class="note-title">Note:</span> The indicated drop size is optimized for plating of a single drop per well of a 96-well imaging plate (IBIDI). It is recommended to adjust the size when other plate types are used.</div>
<ol start="18">
<li>Invert the plate and let the drops solidify in a 37°C incubator for 10–30 min.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Plating density</div>
<p>Bright field examples of cell aggregates that are plated at an optimal density (left) or a density that is too concentrated (right).</p>
<p>Scale bar = 100 μm</p>
</figcaption>
</figure>
</li>
<li>Carefully add 150 μL room temperature ENR medium per well, do not disrupt the drops, and return the plate to the incubator (37°C, 5% CO2).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this stage chemical manipulation can be started, for example by addition of small molecule inhibitors to the culture medium.</div>
<h3 id="sec3.4">Part B. Generation of mixed enteroid monolayers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>This section describes the main steps to generate mixed enteroid monolayers (<a href="#bib18">Thorne et al., 2018</a>) from individual small intestine wild-type and cancer cells. Once the monolayers are formed, they can be used in different techniques such as immuno-fluorescence and time-lapse imaging, described later in this protocol.</p>
<ol start="20">
<li>Coat imaging plates:
<ol type="a">
<li>Dilute BME2 to a final concentration of 0.8 mg/mL in ice-cold Basic culture medium and keep on ice.</li>
<li>Add 50–150 μL of diluted BME2 per well (see <a href="#tbl3">Table 3</a> for appropriate volume) and incubate for 1 h at 37°C
<table id="tbl3">
<caption>Table 3. Recommended input and plating conditions for mixed enteroid monolayers</caption>
<thead>
<tr>
<th colspan="3" rowspan="2">Well format</th>
<th colspan="6">Number of cells /well</th>
</tr>
<tr>
<th colspan="2">Ratio 1:1</th>
<th colspan="2">Ratio 1:2</th>
<th colspan="2">Ratio 1:4</th>
</tr>
<tr>
<th>Type</th>
<th>Size</th>
<th>Volume</th>
<th>A</th>
<th>B</th>
<th>A</th>
<th>B</th>
<th>A</th>
<th>B</th>
</tr>
</thead>
<tbody>
<tr>
<td>384 (Greiner)</td>
<td>0.10 cm<sup>2</sup>
</td>
<td>50 μL</td>
<td>3000</td>
<td>3000</td>
<td>3000</td>
<td>1500</td>
<td>3000</td>
<td>750</td>
</tr>
<tr>
<td>96 (Greiner)</td>
<td>0.34 cm<sup>2</sup>
</td>
<td>100 μL</td>
<td>10000</td>
<td>10000</td>
<td>10000</td>
<td>5000</td>
<td>10000</td>
<td>2500</td>
</tr>
<tr>
<td>96 (IBIDI)</td>
<td>0.56 cm<sup>2</sup>
</td>
<td>150 μL</td>
<td>15000</td>
<td>15000</td>
<td>15000</td>
<td>7500</td>
<td>15000</td>
<td>3750</td>
</tr>
</tbody>
</table>
</li>
<li>Wash the coated wells twice with Basic culture medium and use the plate within 2 h. Keep the plates at room temperature until use.</li>
</ol>
</li>
<li>Continue with resuspended pellet (Step 7) and use a fire-polished glass Pasteur pipet to mechanically disrupt the organoids by briefly pipetting up and down.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Immediately before use, sterilize glass Pasteur pipets by briefly passing them through a flame and coat the inside walls of the pipet by aspiration of Basic culture medium to avoid attachment of the organoids.</div>
<ol start="22">
<li>Centrifuge at 300×<i>g</i> at 4°C for 5 min.</li>
<li>Aspirate the supernatant and add 1 mL of TrypLE at room temperature.</li>
<li>Dissociate the broken organoids by pipetting up and down using a fire-polished glass Pasteur pipet.</li>
<li>Once a single cell suspension is formed (after ± 3 min at room temperature) immediately add 10 mL of cold Basic culture medium and keep on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> TrypLE is sufficiently inactivated by dilution. Additional inactivation steps (e.g. addition of serum or an inhibitor) might be required when other dissociation reagents are used. Check recommendations of the supplier.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Some organoid cultures are very sensitive to dissociation. It is recommended to reduce the incubation time in TrypLE to less than 5 min. This can be achieved by processing few samples simultaneously and repeated monitoring at a microscope until a single cell suspension is reached (similar to the right panel in <a href="#fig4">Figure 4</a>).</div>
<ol start="26">
<li>Centrifuge at 300×<i>g</i> at 4°C for 5 min.</li>
<li>Aspirate the supernatant and add 200 μL of ice-cold Enteroid plating medium.</li>
<li>Count cells using a Bürker Türk Counting Chamber or automated cell counter</li>
<li>Dilute and plate cells in ice-cold Enteroid plating medium in an imaging compatible plate according to <a href="#tbl3">Table 3</a>.</li>
<li>Let the cells adhere in an incubator (37°C, 5% CO<sub>2</sub>).</li>
<li>Replace the medium with 37°C ENR medium after 16–24 h and refresh every 2–3 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this stage chemical manipulation can be started, for example by addition of small molecule inhibitors to the culture medium.</div>
<h3 id="sec3.5">Immuno-fluorescence staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section describes how to proceed with Immuno-fluorescence staining of mixed organoids or enteroid monolayers. The indicated volumes are based on use of a 96-well imaging plate (IBIDI) and can be adapted accordingly when different plates are used.</p>
<ol start="32">
<li>Carefully aspirate culture medium without disturbing the drop of BME2 or the monolayer.</li>
<li>Add 100 μL Fixation solution (see “<a href="#materials-and-equipment">materials and equipment</a>” section) and incubate at room temperature for 20 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is not required to include an additional wash step prior to fixation.</div>
<ol start="34">
<li>Aspirate the Fixation solution once the drop of BME2 is completely dissolved (<a href="#fig6">Figure 6</a>) and wash twice with 100 μL PBS.
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Fixation of mixed organoids</div>
<p>Drop of BME2 containing mixed 3D organoids before fixation (left) and after dissolving in fixation solution (right). The BME2 drop, before and after disruption, are outlined by a dashed line.</p>
<p>Scale bar = 1 mm</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the drop is not completely dissolved, the fixation can be prolonged for up to 30 min.</div>
<div class="note">
<span class="note-title">Note:</span> The organoids are now attached to the bottom of the well.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> After fixation organoids can be stored in PBS for 2/3 weeks, protected from light at 4°C before continuing with immuno-staining.</div>
<ol start="35">
<li>Add 100 μL blocking solution (see “<a href="#materials-and-equipment">materials and equipment</a>” section) and incubate at room temperature for at least 30 min to block and permeabilize organoids.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step can be prolonged for several hours when convenient.</div>
<ol start="36">
<li>Add the appropriate primary antibodies diluted in Antibody incubation solution (50 μL per well) and incubate at 4°C protected from light overnight (8–16 h) or for up to several days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See “<a href="#expected-outcomes">expected outcomes</a>” section for examples of antibodies that can be used.</div>
<ol start="37">
<li>Wash three times for 5 min with 100 μL Washing solution at room temperature.</li>
<li>Add the appropriate secondary antibodies diluted in Antibody incubation solution (50 μL per well) and incubate at room temperature protected from light for at least one hour.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this step other dyes such as DAPI or phalloidin can be included. Selection of secondary antibodies should be based on the expression of fluorophores by the organoid cultures to avoid cross-talk.</div>
<div class="note">
<span class="note-title">Note:</span> This step can be prolonged for up to several days. Incubate at 4°C when exceeding 5 hours.</div>
<ol start="39">
<li>Wash three times with 100 μL Washing solution for 5 min each at room temperature.</li>
<li>Rinse twice with 100 μL PBS and leave the organoids in PBS.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Stained organoids can be can be stored in the dark at 4°C. To prevent loss of signal, we recommend proceeding with image analysis within one week. Ensure the organoids do not dry out by adding enough PBS and/or sealing plates with parafilm.</div>
<ol start="41">
<li>Proceed to imaging using a platform of choice. A confocal based platform is recommended, using a high NA 20–25× objective.</li>
<li>Acquire Z-stacks, see notes for recommendations.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For 3D mixed organoids the thickness of Z-slices should be determined based on the detail of information that is needed for further analysis. For example, a thickness of 5 μm is sufficient for analysis at a cellular resolution. For optimal 3D reconstruction without loss of data, a system calculated optical slice thickness can be used.</div>
<div class="note">
<span class="note-title">Note:</span> For enteroid monolayers, it is recommended to acquire Z-stacks covering the total thickness of the monolayer (approximately 20 μm) to avoid loss of 3D information. Data can be further analyzed via 3D reconstruction or Z-projection.</div>
<h3 id="sec3.6">Time-lapse imaging of mixed organoids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–4 days</div>
<p>This section describes how to proceed with time-lapse imaging of mixed organoids or enteroid monolayers. Ensure cells are plated in an imaging-compatible plate at Step 17 or 29.</p>
<ol start="43">
<li>Refresh the medium 1 day after plating.</li>
<li>Add PBS to the surrounding empty wells to avoid evaporation.</li>
<li>Proceed to imaging using a platform of choice that is equipped with a CO<sub>2</sub> and 37°C incubator. A (spinning-disk) confocal based platform is recommended, using a high NA 20–25× objective.</li>
<li>Acquire images with a maximum time interval of one hour and a maximum Z-slice thickness of 5 μm for tracking of single cells in 3D.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Organoids can be imaged for up to 72 hours. A change of medium is necessary for longer experiments.</div>
<h3 id="sec3.7">Sorting cells from mixed organoids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 h</div>
<p>This section describes how to process previously mixed 3D organoids for cell sorting.</p>
<ol start="47">
<li>Harvest organoids by scraping the bottom of the well and pipetting up and down with a p1000 pipette using ice-cold Basic culture medium.</li>
<li>Collect the organoid suspension in 15 mL centrifuge tubes.</li>
<li>Centrifuge at 300×<i>g</i> at 4°C for 5 min. Three clear layers should be formed; medium, BME2 and the pellet of organoids (top-to-bottom, <a href="#fig3">Figure 3</a>).</li>
<li>Carefully remove the medium and BME2 layers without disrupting the organoid pellet.</li>
<li>Resuspend the pellet in 1 mL of ice-cold Basic culture medium.</li>
<li>Mechanically disrupt the organoids by briefly pipetting up and down using a fire-polished glass Pasteur pipet.</li>
<li>Centrifuge at 300×<i>g</i> at 4°C for 5 min.</li>
<li>Aspirate the supernatant and add 1 mL of TrypLE at room temperature.</li>
<li>Dissociate the broken organoids by pipetting up and down using a fire-polished glass Pasteur pipet.</li>
<li>Once a single cell suspension is formed (after ± 3 min) immediately add 10 mL of ice-cold Basic culture medium and keep on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> TrypLE is sufficiently inactivated by dilution. Additional inactivation steps (e.g. addition of serum or an inhibitor) might be required when other dissociation reagents are used. Check recommendations of the supplier.</div>
<ol start="57">
<li>Centrifuge at 300×<i>g</i> at 4°C for 5 min.</li>
<li>Resuspend pellet in 250 μL ice-cold FACS buffer and pass through a 35 μm cell strainer.</li>
<li>Keep cells on ice and immediately proceed to cell sorting using a platform of choice.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Our protocol describes the generation of mixed 3D organoids and enteroid monolayers from individual cell populations. We have not tried to derived mixed enteroid monolayers directly from mixed 3D organoids. During the course of the experiment growth and progression of the cultures can be monitored using a tissue culture microscope equipped with fluorescent light source. When generating mixed cultures of populations of different cellular fitness, a gradual loss of the weaker cell population is expected (<a href="#fig7">Figure 7</a>). The 3D cultures of wild-type small intestine and <i>Apc</i><sup>−/−</sup><i>Kras</i><sup>G12D/WT</sup><i>Trp53</i><sup>-/R172H</sup> cancer cells mixed in a 2:1 ratio described here, consist of ± 40% wild-type cells 1 day after mixing and this decreases to ±10% on day 4 (<a href="#bib10">Krotenberg Garcia et al., 2021</a>). The described enteroid monolayers mixed in a 2:1 ratio (wild-type : cancer) are composed of ±20% wild-type cells the moment a confluent monolayer has formed (<a href="#bib10">Krotenberg Garcia et al., 2021</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Progression of mixed 3D organoids and enteroid monolayers</div>
<p>(A–D) Monitoring growth of pure (A) and mixed (B) 3D organoids using a fluorescent tissue culture microscope and pure (C) and mixed (D) enteroid monolayers using a stage-calibrated confocal microscope for up to one week. Cancer cells are visualized in green. The wild-type population (magenta) is gradually lost from the cultures over time.</p>
<p>Scale bars = 100 μm</p>
</figcaption>
</figure>
<p>These mixed cultures can be used to study cellular interactions with a variety of methods. Immuno-fluorescence staining can reveal cell competition-induced changes in cellular behavior and morphology. We have previously observed that wild-type small intestine cells revert to a fetal-like state (<a href="#bib10">Krotenberg Garcia et al., 2021</a>). This coincides which a diminished formation of crypt-like structures and reduced presence of cell types that are normally present in adult intestinal tissue, such as Paneth and intestinal stem cells (<a href="#fig8">Figures 8</a>A–8C). In addition, activation of signaling pathways can be studied using immuno-fluorescence. Wild-type cells are elimination via apoptosis and this is driven by active JNK signaling (<a href="#bib10">Krotenberg Garcia et al., 2021</a>), which can be visualized using cl-CASP3 and cJUN-pS73 antibodies (<a href="#fig8">Figures 8</a>D and 8E; <a href="#mmc1">Methods video S1</a>). Specific changes in cellular behavior can be tracked by time-lapse microscopy (<a href="#fig9">Figures 9</a>A and 9B; <a href="#mmc2">Methods video S2</a> and <a href="#mmc3">S3</a>). Cell sorting is used to detect changes on a population level (<a href="#fig10">Figure 10</a>A). Sorted cells can subsequently be used for further downstream analysis such as (mRNA) sequencing or proteomics. Such analysis has revealed a reversion of wild-type cells to a fetal-like state (<a href="#bib10">Krotenberg Garcia et al., 2021</a>) and reduced expression of intestinal stem cell markers (<a href="#fig10">Figure 10</a>B).</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Analysis of cellular interactions using immuno-fluorescence</div>
<p>Examples of analysis of interactions between cells in mixed 3D organoids and enteroid monolayers by immuno-fluorescence.</p>
<p>(A–C) Immuno-fluorescence staining of 3D-reconstructed pure (top) and mixed (bottom) organoids. Cancer (green) or wild-type (magenta) cells are visualized. The organoids were stained for lysozyme (A, green, Paneth cells), Aldolase B (B, magenta, enterocytes) or OLFM4 (C, magenta, intestinal stem cells), nuclei are visualized with DAPI (blue).</p>
<p>(D and E) Immuno-fluorescence staining of 3D-reconstructed mixed organoid (D) and mixed monolayer (E). Cancer (green) and wild-type (magenta) cells are visualized. The organoids were stained for cJUN-pS73 (D, grey) or cl-CASP3 (E, grey); nuclei are visualized with DAPI (blue). Also see <a href="#mmc1">Methods video S1</a>.</p>
<p>Scale bars = 50 μm</p>
</figcaption>
</figure>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Analysis of cellular interactions using time-lapse microscopy</div>
<p>(A) Time-lapse series of a 3D-reconstructed mixed organoid, started one day after mixing. Cancer (green) and wild-type (magenta) cells are visualized. Also see <a href="#mmc2">Methods video S2</a>.Time-lapse series of control treated 3D reconstructed mixed intestinal organoid, related to steps 43–46.</p>
<p>(B) Time-lapse series of a mixed monolayer. Image acquisition was started after a full monolayer had formed, approximately three days after plating and the ratio shifted towards the cancer population. Cancer (green) and wild-type (magenta) cells are visualized. The arrow indicates an eliminated WT cell, which shrinks and is lost from the culture. Also see <a href="#mmc3">Methods video S3</a>.Time-lapse series of a competing enteroid monolayer, related to steps 43–46.</p>
<p>Scale bars = 50 μm</p>
</figcaption>
</figure>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Analysis of cellular interactions using cell sorting</div>
<p>(A) Flow cytometry sorting of wild-type and cancer cells from pure and mixed cultures, the numbers in the corners display the percentage of sorted cells.</p>
<p>(B) Heatmap of the fold change of genes that are differentially expressed in wild-type cells upon mixing (Log2). Genes that are expressed by specific adult intestinal cell types are indicated.</p>
</figcaption>
</figure>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. 3D-reconstruction of confocal images of a mixed organoid, related to Figure 8D and steps 32–42</div>
<p>Cancer (green) and wild-type (magenta) cells are visualized. The organoid was stained for cJUN-pS73 (grey) and nuclei with DAPI (blue)</p>
</figcaption>
</figure>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Mmc2.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S2. Time-lapse series of control treated 3D reconstructed mixed intestinal organoid, related to Figure 9A and steps 43–46</div>
<p>Cancer (green) and wild-type (magenta) cells are visualized</p>
</figcaption>
</figure>
<figure id="mmc3"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1217-Mmc3.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S3. Time-lapse series of a competing enteroid monolayer, related to Figure 9B and steps 43–46</div>
<p>Cancer (green) and wild-type (magenta) cells are visualized. Arrow indicates an example of a wild-type cells that shrinks and is eliminated</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Our protocol describes 3D mixed organoids and enteroid monolayers that are generated from adult stem cell derived cystic-growing organoid cultures. These cultures only contain epithelial cells and lack other cell types such as mesenchyme, neurons and immune cells. Therefore, the current protocol cannot be used to study the influence of a microenvironment. Extensive optimization is required to adapt this protocol for these purposes.</p>
<p>Currently, the maximum duration of the experiment is limited to the growth and passage requirements of the organoid cultures. The optimal duration for the mixed 3D cultures described in this protocol is three days and the maximum length is five days, by which all wild-type cells are eliminated. The mixed enteroid monolayers are viable for at least ten days, however most wild-type cells are lost within seven days after a full monolayer is formed. It is not possible to passage the organoids while preserving the mixed cellular interactions. Propagation of the long-term effects of cellular interactions can possibly be achieved by re-mixing of cells that were sorted from mixed populations.</p>
<p>Generation of viable mixed organoids from cultures that have incompatible basic growth conditions can be problematic. For example, when one of the essential growth factors for one of the cell populations has a negative impact on the health of another cell population. In this instance, a shared experimental medium cannot easily be designed and redefinition of the minimal medium composition is required. In addition, extreme differences in the proliferation rate of the individual cultures will impair generation of sufficient mixed organoids. Here, adaptation of the experimental design is necessary. For example, by mixing cell clumps of the slow dividing population with fast growing single cells.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Drops of BME2 attach to side of well (Step 17)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Ensure unpackaged tissue culture plates are incubated in a heated CO2 incubator for at least 48 h prior to use. Choose a smaller drop size if the problem persists.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>(Mixed) organoids adhere to the bottom of the plate (Step 18).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Invert the plate while drops are solidifying and/or use low-attachment/uncoated tissue culture plates</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Only a low percentage of organoids are mixed.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Reduce the volume of medium that is used during aggregation (Step 13). Ensure the organoids are optimally broken (<a href="#fig4">Figure 4</a>). Increase the amount of input material.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Mixed organoids fuse during the experiment.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Plate less dense cultures after aggregation by adding higher volumes of medium and BME2 (Step 16, also see <a href="#fig5">Figure 5</a>).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>It is difficult to distinguish the two different cell populations during FACS or microscopy analysis (see “<a href="#expected-outcomes">expected outcomes</a>”).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Ensure both cell populations are differentially fluorescently labelled and this is bright enough for your analysis platform of choice (see “<a href="#before-you-begin">before you begin</a>”). In addition, a counterstain can be used in order to detect all cells in mixed organoids (e.g., DAPI or expression of a fluorescently-tagged histone H2B variant).</p>
<h3 id="sec6.11">Problem 6</h3>
<p>One of the cell populations is lost before the end of the experiment (<a href="#fig7">Figure 7</a>)</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Adapt the input ratio during mixing (<a href="#tbl1">Table 1</a>) and/or ensure cells are mixed as clumps, not single cells (<a href="#fig4">Figure 4</a>).</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Fluorescent signal is not equally distributed after immuno-fluorescence staining (see “<a href="#expected-outcomes">expected outcomes</a>”).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>Ensure BME2 drops are fully dissolved by fixation solution (<a href="#fig6">Figure 6</a>). Prolong permeabilization (Step 35) and/or increase the percentage of TX-100 in the Blocking solution.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>Quality of images is reduced or lost during time-lapse image acquisition (Steps 43–46).</p>
<h3 id="sec6.16">Potential solution</h3>
<p>Ensure the microscope is heated and incubate the plate in the system for at least 30 min before set-up the session to avoid drift. Carefully determine maximum Z-dimension before selecting positions to ensure the organoids stay within reach during acquisition and the objective is not damaged. Ensure the system is equipped with a C02 incubator and check flow rate. Use a dry objective for prolonged time-lapse imaging to prevent loss of water or oil immersion. Alternatively, a water pump can be installed to keep a stable water immersion at the objective. Ensure, photo-toxicity and photo-bleaching are kept at a minimum level by optimization of acquisition settings (e.g., laser power, scanning time).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Andersson-Rolf, A., Fink, J., Mustata, R.C., and Koo, B.-K. (2014). A video protocol of retroviral infection in primary intestinal organoid culture. J. Visualized Experiments <i>90</i>, 51765. <a class="external-link" href="https://doi.org/10.3791/51765">https://doi.org/10.3791/51765</a></p>
<p id="bib2">Broutier, L., Andersson-Rolf, A., Hindley, C.J., Boj, S.F., Clevers, H., Koo, B.K., and Huch, M. (2016). Culture and establishment of self-renewing human and mouse adult liver and pancreas 3D organoids and their genetic manipulation. Nat. Protoc. <i>11</i>, 1724-1743. <a class="external-link" href="https://doi.org/10.1038/nprot.2016.097">https://doi.org/10.1038/nprot.2016.097</a></p>
<p id="bib3">Flanagan, D.J., Pentinmikko, N., Luopajärvi, K., Willis, N.J., Gilroy, K., Raven, A.P., Mcgarry, L., Englund, J.I., Webb, A.T., Scharaw, S., et al. (2021). NOTUM from Apc-mutant cells biases clonal competition to initiate cancer. Nature. <a class="external-link" href="https://doi.org/10.1038/s41586-021-03525-z">https://doi.org/10.1038/s41586-021-03525-z</a></p>
<p id="bib4">Fumagalli, A., Drost, J., Suijkerbuijk, S.J.E., van Boxtel, R., de Ligt, J., Offerhaus, G.J., Begthel, H., Beerling, E., Tan, E.H., Sansom, O.J., et al. (2017). Genetic dissection of colorectal cancer progression by orthotopic transplantation of engineered cancer organoids. Proc. Natl. Acad. Sci. U S A <i>114</i>, E2357-E2364. <a class="external-link" href="https://doi.org/10.1073/pnas.1701219114">https://doi.org/10.1073/pnas.1701219114</a></p>
<p id="bib5">Heinz, M.C., Oost, K.C., and Snippert, H.J.G. (2020). Introducing the stem cell ASCL2 reporter STAR into intestinal organoids. STAR Protoc. <i>1</i>, 100126. <a class="external-link" href="https://doi.org/10.1016/j.xpro.2020.100126">https://doi.org/10.1016/j.xpro.2020.100126</a></p>
<p id="bib6">Hill, W., Zaragkoulias, A., Salvador-Barbero, B., Parfitt, G.J., Alatsatianos, M., Padilha, A., Porazinski, S., Woolley, T.E., Morton, J.P., Sansom, O.J., and Hogan, C. (2021). EPHA2-dependent outcompetition of KRASG12D mutant cells by wild-type neighbors in the adult pancreas. Curr. Biol. 1-11. <a class="external-link" href="https://doi.org/10.1016/j.cub.2021.03.094">https://doi.org/10.1016/j.cub.2021.03.094</a></p>
<p id="bib7">Hogan, C., Dupré-Crochet, S., Norman, M., Kajita, M., Zimmermann, C., Pelling, A.E., Piddini, E., Baena-López, L.A., Vincent, J.P., Itoh, Y., et al. (2009). Characterization of the interface between normal and transformed epithelial cells. Nat. Cell Biol. <i>11</i>, 460-467. <a class="external-link" href="https://doi.org/10.1038/ncb1853">https://doi.org/10.1038/ncb1853</a></p>
<p id="bib8">Jackstadt, R., van Hooff, S.R., Leach, J.D., Cortes-Lavaud, X., Lohuis, J.O., Ridgway, R.A., Wouters, V.M., Roper, J., Kendall, T.J., Roxburgh, C.S., et al. (2019). Epithelial NOTCH signaling rewires the tumor microenvironment of colorectal cancer to drive poor-prognosis subtypes and metastasis. Cancer Cell <i>36</i>, 319-336.e7. <a class="external-link" href="https://doi.org/10.1016/j.ccell.2019.08.003">https://doi.org/10.1016/j.ccell.2019.08.003</a></p>
<p id="bib9">Kon, S., Ishibashi, K., Katoh, H., Kitamoto, S., Shirai, T., Tanaka, S., Kajita, M., Ishikawa, S., Yamauchi, H., Yako, Y., et al. (2017). Cell competition with normal epithelial cells promotes apical extrusion of transformed cells through metabolic changes. Nat. Cell. Biol. <i>19</i>, 530-541. <a class="external-link" href="https://doi.org/10.1038/ncb3509">https://doi.org/10.1038/ncb3509</a></p>
<p id="bib10">Krotenberg Garcia, A., Fumagalli, A., Le, H.Q., Jackstadt, R., Lannagan, T.R.M., Sansom, O.J., van Rheenen, J., and Suijkerbuijk, S.J.E. (2021). Active elimination of intestinal cells drives oncogenic growth in organoids. Cell. Rep. <i>36</i>, 109307. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2021.109307">https://doi.org/10.1016/j.celrep.2021.109307</a></p>
<p id="bib11">Muzumdar, M.D., Tasic, B., Miyamichi, K., Li, L., and Luo, L. (2007). A global double-fluorescent Cre reporter mouse. Genesis <i>45</i>, 593-605. <a class="external-link" href="https://doi.org/10.1002/dvg.20335">https://doi.org/10.1002/dvg.20335</a></p>
<p id="bib12">van Neerven, S.M., de Groot, N.E., Nijman, L.E., Scicluna, B.P., van Driel, M.S., Lecca, M.C., Warmerdam, D.O., Kakkar, V., Moreno, L.F., Vieira Braga, F.A., et al. (2021). Apc-mutant cells act as supercompetitors in intestinal tumour initiation. Nature. <a class="external-link" href="https://doi.org/10.1038/s41586-021-03558-4">https://doi.org/10.1038/s41586-021-03558-4</a></p>
<p id="bib13">Norman, M., Wisniewska, K.A., Lawrenson, K., Garcia-Miranda, P., Tada, M., Kajita, M., Mano, H., Ishikawa, S., Ikegawa, M., Shimada, T., and Fujita, Y. (2012). Loss of Scribble causes cell competition in mammalian cells. J. Cell Sci. <i>125</i>, 59-66. <a class="external-link" href="https://doi.org/10.1242/jcs.085803">https://doi.org/10.1242/jcs.085803</a></p>
<p id="bib14">Sato, T. and Clevers, H. (2012). Primary mouse small intestinal epithelial cell cultures. In Methods in Molecular Biology, pp. 319-328. <a class="external-link" href="https://doi.org/10.1007/978-1-62703-125-7_19">https://doi.org/10.1007/978-1-62703-125-7_19</a></p>
<p id="bib15">Sato, T., Vries, R.G., Snippert, H.J., Van De Wetering, M., Barker, N., Stange, D.E., Van Es, J.H., Abo, A., Kujala, P., Peters, P.J., and Clevers, H. (2009). Single Lgr5 stem cells build crypt-villus structures in vitro without a mesenchymal niche. Nature <i>459</i>, 262-265. <a class="external-link" href="https://doi.org/10.1038/nature07935">https://doi.org/10.1038/nature07935</a></p>
<p id="bib16">Simpson, P. (1979). Parameters of cell competition in the compartments of the wing disc of Drosophila. Develop. Biol. <i>69</i>, 182-193. <a class="external-link" href="https://doi.org/10.1016/0012-1606(79)90284-7">https://doi.org/10.1016/0012-1606(79)90284-7</a></p>
<p id="bib17">Suijkerbuijk, S.J.E., Kolahgar, G., Kucinski, I., and Piddini, E. (2016). Cell competition drives the growth of intestinal adenomas in Drosophila. Curr. Biol. <i>26</i>, 428-438. <a class="external-link" href="https://doi.org/10.1016/j.cub.2015.12.043">https://doi.org/10.1016/j.cub.2015.12.043</a></p>
<p id="bib18">Thorne, C.A., Chen, I.W., Sanman, L.E., Cobb, M.H., Wu, L.F., and Altschuler, S.J. (2018). Enteroid monolayers reveal an autonomous WNT and BMP circuit controlling intestinal epithelial growth and organization. Develop. Cell <i>44</i>, 624-633.e4. <a class="external-link" href="https://doi.org/10.1016/j.devcel.2018.01.024">https://doi.org/10.1016/j.devcel.2018.01.024</a></p>
<p id="bib19">Vishwakarma, M. and Piddini, E. (2020). Outcompeting cancer. Nat. Rev. Cancer. <a class="external-link" href="https://doi.org/10.1038/s41568-019-0231-8">https://doi.org/10.1038/s41568-019-0231-8</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Saskia J.E. Suijkerbuijk (<a href="mailto:s.j.e.suijkerbuijk@uu.nl">s.j.e.suijkerbuijk@uu.nl</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents, plasmids, or organoid lines</p>
<h4>Data and code availability</h4>
<p>This study did not generate new unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was financially supported by <a href="https://doi.org/10.13039/501100004622">Dutch Cancer Society</a> Fellowship BUIT-2013-5847 (to S.J.E.S.) and Young Investigator Grant 11491/2018-1 (to S.J.E.S. and A.K.G) and the CancerGenomics.nl and the Doctor Josef Steiner Foundation (to J.v.R). We thank Marjolijn Mertz, Lenny Brocks and the NKI BioImaging facilty for technical support.</p>
<h3>Author contributions</h3>
<p>Methodology and Investigation, A.K.G and S.J.E.S. Writing – Original Draft, A.K.G and S.J.E.S. Writing – Review &amp; Editing, A.K.G and S.J.E.S.; Funding Acquisition, S.J.E.S and J.v.R.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>