<article>
<section>
<h1 id="header">CRISPR-Cas9-mediated induction of large chromosomal inversions in human bronchial epithelial cells</h1>
<p><time datetime="2022-03-25">Published: March 25, 2022</time></p>
<p>Andriani Angelopoulou,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">8</a></sup> Angelos Papaspyropoulos,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">8</a></sup> Argyris Papantonis,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#cor1">*</a></sup> and Vassilis G. Gorgoulis<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#aff7">7</a>,<a href="#fn2">9</a>,<a href="#fn3">10</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Molecular Carcinogenesis Group, Department of Histology and Embryology, Faculty of Medicine, National and Kapodistrian University of Athens, 11527 Athens, Greece</p>
<p id="aff2"><sup>2</sup>Biomedical Research Foundation, Academy of Athens, 11527 Athens, Greece</p>
<p id="aff3"><sup>3</sup>Translational Epigenetics Group, Institute of Pathology, University Medical Center Göttingen, 37075 Göttingen, Germany</p>
<p id="aff4"><sup>4</sup>Center for Molecular Medicine Cologne (CMMC), University of Cologne, 50931 Cologne, Germany</p>
<p id="aff5"><sup>5</sup>Division of Cancer Sciences, School of Medical Sciences, Faculty of Biology, Medicine &amp; Health, University of Manchester, M20 4GJ Manchester, UK</p>
<p id="aff6"><sup>6</sup>Center for New Biotechnologies and Precision Medicine, Medical School, National and Kapodistrian University of Athens, 11527 Athens, Greece</p>
<p id="aff7"><sup>7</sup>Faculty of Health and Medical Sciences, University of Surrey, GU2 7YH Surrey, UK</p>
<p id="fn1"><sup>8</sup>These authors contributed equally</p>
<p id="fn2"><sup>9</sup>Technical contact</p>
<p id="fn3"><sup>10</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:argyris.papantonis@med.uni-goettingen.de">argyris.papantonis@med.uni-goettingen.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:vgorg@med.uoa.gr">vgorg@med.uoa.gr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101257">10.1016/j.xpro.2022.101257</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The <i>in vitro</i> recapitulation of chromosomal rearrangements is a necessary tool for understanding malignancy at the molecular level. Here, we describe the targeted induction of a large chromosomal inversion (&gt;3.7 Mbp) through CRISPR-Cas9-mediated genome editing. As inversions occur at low frequency following Cas9 cleavage, we provide a detailed screening approach of FACS-sorted, single-cell-derived clonal human bronchial epithelial cell (HBEC) cultures. The protocol provided is tailored to HBECs; however, it can be readily applied to additional adherent cellular models.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref9">Zampetidis et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Large chromosomal inversions engineered with CRISPR-Cas9 technology in the HBEC system</li>
<li>Step-by-step protocol of gRNA design and cloning into Cas9-encoding vectors</li>
<li>Clonal propagation of FACS-sorted single cells following non-liposomal transfection</li>
<li>Description of primer design strategy and PCR screening for inversion validation</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1518-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps of inducing a targeted chromosomal inversion in human bronchial epithelial cells (HBECs).</p>
<h3 id="sec1.1">Design of gRNAs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–1 h</div>
<p>In order to obtain a chromosomal inversion, two independent chromosomal breaks must be induced. Therefore, two different gRNAs are required to direct Cas9 binding and cleavage at two separate genomic positions. Specifically, each breakpoint should be the target of an individual gRNA. In our study, we aimed at recapitulating an inversion naturally occurring in cancerous cells that have escaped from oncogene-induced senescence, as identified by Whole Genome Sequencing (WGS) (<a href="#bib9">Zampetidis et al., 2021</a>). Based on the precise inversion coordinates (start/end), two gRNAs are designed, one for each breakpoint, following the steps below (see <a href="#fig1">Figure 1</a>):</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1518-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Flow chart of the gRNA design process using the UCSC browser and CRISPOR algorithm</div>
<p>Based on the breakpoint position (red highlight), the DNA sequence around it (blue highlight) can be obtained using the UCSC browser. The acquired sequence is then imported to CRISPOR serving as target for the design of gRNAs. The gRNAs predicted to have the highest specificity scores and the smallest distance from the breakpoint are chosen and modified with the proper overhangs before ordering.</p>
</figcaption>
</figure>
<ol>
<li>The DNA sequence used as target for the design of the two gRNAs can be obtained via the UCSC Genome Browser (<a href="https://genome.ucsc.edu/">https://genome.ucsc.edu/</a>).
<ol type="a">
<li>Choose the genome version of interest. In our case, “Human GRCh38/hg38” was used.</li>
<li>Based on the known position of inversion breakpoint 1, mark the genomic area ranging from 100-150 bp before to 100–150 bp after the breakpoint. For example, if breakpoint 1 is located at chr3:2,920,305, in the UCSC Genome Browser search box type “chr3:2,920,205–2,920,405” to mark the required chromosomal area and click on “Go”.</li>
<li>Select “View” on the UCSC Genome Browser toolbar and click on the “DNA” option.</li>
<li>In the new window, click on “Get DNA” in order to obtain the exact DNA sequence. This is the sequence required to design gRNA primers using the CRISPOR algorithm (see step 2a, below).</li>
<li>Repeat steps 1a–1d for breakpoint 2 of the inversion.</li>
</ol>
</li>
<li>To design the gRNAs use the CRISPOR algorithm (<a href="http://crispor.tefor.net/">http://crispor.tefor.net/</a>):
<ol type="a">
<li>Enter the DNA sequence obtained from step 1d for breakpoint 1. Make sure the reference genome matches the one used in the UCSC Browser (step 1a) and subsequently select a Protospacer Adjacent Motif (PAM) that is identifiable by the type of Cas9 enzyme encoded by the transfection vector. If SpCas9 is expressed by the transfection vector, selected the 20 bp-NGG PAM format. Click “Submit” to obtain candidate gRNA sequences targeting the template DNA.</li>
<li>The CRISPOR algorithm by default ranks candidate gRNA sequences from highest to lowest specificity, as this is the critical parameter. From the list of candidate gRNAs appearing on a new page, choose the guide sequences with the highest Massachusetts Institute of Technology (MIT) and Cutting Frequency Determination (CFD) specificity scores (<a href="#bib3">Doench et al., 2016</a>; <a href="#bib4">Hsu et al., 2013</a>; <a href="#bib8">Tycko et al., 2019</a>). These scores evaluate candidate gRNAs in terms of potential off-target effects, with the highest MIT/CFD scores corresponding to the smallest number of potential off-target events. Among guide sequences with similar MIT/CFD scores, select the ones offering the minimum distance between the Cas9 cleavage site and the known breakpoint 1 site. Regarding the Cas9 cleavage site, SpCas9 normally induces a blunt cut between the 17th and 18th base in the target sequence (3–4 bp 5′ of the PAM) (<a href="#bib5">Jinek et al., 2012</a>).</li>
<li>Repeat steps 2a and 2b for the DNA sequence corresponding to breakpoint 2.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Αvoid selecting sequences that are indicated as “Inefficient” or displaying “High GC content” by the CRISPOR algorithm, as this may significantly compromise target cleavage.</div>
<div class="note">
<span class="note-title">Note:</span> More than one gRNAs per each breakpoint need to be designed and tested for optimal efficiency.</div>
<ol start="3">
<li>Before ordering the gRNAs as standard desalted oligos, the selected gRNA sequences should be modified as follows:
<ol type="a">
<li>Make sure the PAM sequence is not included in the gRNA oligos to be ordered. Remove the PAM sequence from gRNAs before proceeding to the next steps.
<div class="critical">
<span class="critical-title">Critical:</span> The PAM sequence is located downstream of the target DNA sequence and is essential for Cas9 binding and cleavage, but it should <u>not</u> be part of the gRNA oligos, to avoid unwanted Cas9-mediated cleavage of the gRNA itself.</div>
<div class="note">
<span class="note-title">Note:</span> As an alternative to removing the PAM sequence from gRNAs before ordering, random nucleotide mutations may be introduced to the PAM sequence so that it is not in an NGG format. This way Cas9-mediated cleavage will be prevented.</div>
</li>
<li>Each gRNA sequence yielded by the CRISPOR algorithm serves as the Forward (FW) gRNA sequence. Generate the reverse complement of the FW gRNA sequence to obtain the Reverse (REV) gRNA sequence. The following online tool can be used to directly obtain the reverse complement of an input sequence: <a href="https://www.bioinformatics.org/sms/rev_comp.html">https://www.bioinformatics.org/sms/rev_comp.html</a>
</li>
<li>Add to both gRNA sequences (FW and REV) the proper overhangs in order to render them compatible for cloning into the gRNA scaffold of the transfection vector. In our experimental setup, two PX330-based plasmids were used as cloning vectors: pSpCas9(BB)-2A-GFP (PX458) (#48138; Addgene) (<a href="#bib7">Ran et al., 2013</a>) and pU6-(BbsI)_CBh-Cas9-T2A-mCherry (#64324; Addgene) (<a href="#bib2">Chu et al., 2015</a>), therefore before ordering the gRNAs we made the following nucleotide additions:
<ol type="i">
<li>CACC<u>G</u> to the 5′ end of the FW gRNA sequence.</li>
<li>AAAC to the 5′ end and <u>C</u> to the 3′ end of the REV gRNA sequence.
<div class="critical">
<span class="critical-title">Critical:</span> Addition of <u>G</u> in the FW sequence and <u>C</u> in the REV sequence should be done only if the FW sequence does not start with G, as the initiation of transcription at the RNA Pol III U6 promoter is enhanced by the presence of a <u>G</u> at the transcription start site.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Design of primers for identifying the inversion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–1 h</div>
<p>The successful generation of the inversion can be identified by PCR using specific primer combinations (<a href="#fig2">Figure 2</a>A). Initially, two pairs of primers are designed: one pair of Forward (F1) and Reverse (R1) primers amplifying wild-type genomic DNA before the induction of breakpoint 1, and a second pair of Forward (F2) and Reverse (R2) primers amplifying wild-type genomic DNA before the induction of breakpoint 2. In the case that the combination of F1/F2 or R1/R2 primers yields a PCR product, at least one allele has been inverted (<a href="#fig2">Figure 2</a>A). In detail:</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1518-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Primer design for identification of the inversion</div>
<p>(A) Schematic illustration of the position of the primers around breakpoints.</p>
<p>(B) UCSC browser image depicting the flanking region on the left side of the gRNA1 (yellow highlight) and on the right side (green highlight), based on which the F and R primers are designed, respectively. Flanking regions are selected at least 150 bp away from the Cas9 cleavage site to ensure proper amplification.</p>
</figcaption>
</figure>
<ol start="4">
<li>To acquire the DNA sequence serving as template for the design of primers:
<ol type="a">
<li>Use the UCSC Genome Browser (<a href="https://genome.ucsc.edu/">https://genome.ucsc.edu/</a>). Go to “Tools” on the Toolbar and Click on “BLAT” (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1518-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Step-by-step guide to acquire the DNA sequence serving as template for the design of primers</div>
<p>(A–C) Visualization of (A). Step 4a (B). Step 4b and (C). Step 4c of the “<a href="#sec1.2">design of primers for identifying the inversion</a>” section of the protocol.</p>
</figcaption>
</figure>
</li>
<li>Insert the gRNA sequence in FASTA format (<a href="#fig3">Figure 3</a>B). For example, for breakpoint 1, insert the respective gRNA sequence (gRNA1) as follows:
<div class="textbox">
<p><code>“</code></p>
<p><code>&gt;gRNA1</code></p>
<p><code>GCACCATGGAAACTGAGATA</code></p>
<p><code>”</code></p>
</div>
</li>
<li>Click on “Submit” and in the new window choose “browser”. This will allow visualization of the DNA sequence corresponding to gRNA1 (step 4b) (<a href="#fig3">Figure 3</a>C).</li>
<li>As F1 and R1 primers should be located &gt;150 bp away from the breakpoint, zoom out (by clicking on the “10×” button, <a href="#fig3">Figure 3</a>C) and select the flanking regions spanning &gt;150 bp away from each end of the gRNA sequence (see <a href="#fig2">Figure 2</a>B).</li>
<li>To acquire the left flanking sequence click on “View”→“DNA” and “Get DNA”. This area will be used as a template to design primer F1.</li>
<li>To acquire the right flanking sequence repeat step 4e on the right side of gRNA1. This sequence will be used as a template to design primer R1.</li>
<li>Carry out the above Steps (4a–f) separately for breakpoint 2 to acquire the DNA templates to also design primers F2 and R2.</li>
</ol>
</li>
<li>The primer designing tool of NCBI (<a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a>) can be used to design the primers:
<ol type="a">
<li>Enter in the “PCR template” box the left flanking sequence for gRNA1 obtained from step 4.</li>
<li>Modify the settings as follows:
<ul>
<li>Primer melting temperatures (Tm): 59°C–61°C</li>
<li>Database: RefSeq representative genomes</li>
<li><u>Advanced settings:</u></li>
<li>Primer size: 19–22 nt</li>
<li>Primer GC content (%): 45–75</li>
</ul>
</li>
<li>Repeat steps 5a and 5b for the right flanking sequence of gRNA1, as well as for the two flanking sequences of breakpoint 2.
<div class="critical">
<span class="critical-title">Critical:</span> Selected primers should target regions at least 150 bp away from the Cas9 cleavage site to ensure correct amplification of the desired DNA fragment containing the breakpoint. The reason is that repair of the cleaved site occurs randomly (through Non-Homologous End Joining (NHEJ)) and may variably affect the DNA sequence proximal to the cleaved site.</div>
<div class="note">
<span class="note-title">Note:</span> As per convention, F primers are derived from left flanking sequences and R primers from right flanking sequences of the gRNAs. F and R primers need to fully encompass their respective breakpoint.</div>
<div class="note">
<span class="note-title">Note:</span> In the case that no primers are returned using the above settings (step 5b), the PCR template sequence can be expanded up to ∼1,000 bp around each gRNA (<a href="#fig2">Figure 2</a>B).</div>
<div class="note">
<span class="note-title">Note:</span> Choose more than one set of primers for each breakpoint in order to facilitate the screening process, as well as for validation purposes.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Choosing gRNA expression vectors</h3>
<p>Two plasmid vectors, both expressing Cas9 but different fluorescent markers, are chosen for the cloning of the two gRNAs. gRNA sequences are processed in order to be able to be effectively cloned into the selected vectors, as indicated in step 3c. In our experimental setting, the gRNAs designed at steps 1–3 to target the breakpoints of the inversion were cloned separately into pSpCas9(BB)-2A-GFP (PX458) (#48138; Addgene) (<a href="#bib7">Ran et al., 2013</a>) and pU6-(BbsI)_CBh-Cas9-T2A-mCherry (#64324; Addgene) (<a href="#bib2">Chu et al., 2015</a>) (<a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1518-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Empty backbones of Cas9/GFP (left) or Cas9/mCherry (right) expression vectors</div>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Fluorescent labeling of these vectors facilitates single cell isolation by FACS, as described below.</div>
<h3 id="sec1.4">Cloning of gRNAs into vectors</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<ol start="6">
<li>Annealing of oligos and cloning steps were based on the protocol of the Zhang lab (<a href="https://media.addgene.org/data/plasmids/62/62988/62988-attachment_i-jdFt6Gm-ft.pdf">https://media.addgene.org/data/plasmids/62/62988/62988-attachment_i-jdFt6Gm-ft.pdf</a>):
<ol type="a">
<li>
<u>Digest each plasmid with BbsI.</u> The restriction cutting site of BbsI is located next to the gRNA scaffold in both plasmids. Set up the reactions as follows and incubate for 30 min at 37°C:
<table id="undtbl1">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Plasmid (1 μg)</td>
<td>X μl</td>
</tr>
<tr>
<td>10× NEBuffer</td>
<td>5 μL</td>
</tr>
<tr>
<td>BbsI (10 units)</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>up to 50 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Run the reaction in a 0.8% agarose gel and extract the digested plasmid using the Zymoclean Gel DNA Recovery Kit (<a href="https://files.zymoresearch.com/protocols/_d4001t_d4001_d4002_d4007_d4008_zymoclean_gel_dna_recovery_kit.pdf">https://files.zymoresearch.com/protocols/_d4001t_d4001_d4002_d4007_d4008_zymoclean_gel_dna_recovery_kit.pdf</a>) and elute in 6 μL Elution buffer. Recipes for 0.8% and 1.3% agarose gels used in this protocol are provided in the “<a href="#materials-and-equipment">materials and equipment</a>” section.</li>
<li>Phosphorylation and annealing of oligos.
<ol type="i">
<li>Set up the reaction for each pair of oligos, as follows:
<table id="undtbl2">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Oligo 1 (100 μM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Oligo 2 (100 μM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>10× T4 DNA Ligase Reaction Buffer</td>
<td>1 μL</td>
</tr>
<tr>
<td>T4 PNK (10 units)</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH20</td>
<td>6 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Incubate at 37°C for 30 min followed by incubation at 95°C for 5 min and then ramp down to 25°C at 5°C/minute.</li>
<li>The phosphorylated and annealed oligos can be stored at −20°C.</li>
</ol>
</li>
<li>Prepare the ligation reaction as follows and incubate for 60 min at 15°C–22°C.
<table id="undtbl3">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>BbsI digested plasmid from step 5 (50 ng)</td>
<td>X μl</td>
</tr>
<tr>
<td>Phosphorylated and annealed oligo duplex (diluted 1:200) from step 6c</td>
<td>1 μL</td>
</tr>
<tr>
<td>5× T4 DNA Ligase Buffer</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>ddH20</td>
<td>up to 10 μL</td>
</tr>
<tr>
<td><b>Subtotal</b></td>
<td><b>10 μL</b></td>
</tr>
<tr>
<td>T4 ligase (1 unit)</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>11 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>After the ligation step, each gRNA should be incorporated into the gRNA scaffold position (<a href="#fig4">Figure 4</a>). Validate the cloning results by Sanger sequencing using the U6 primer.
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended that the ligation reactions be directly used for bacterial transformation to increase transformation efficiency, alternatively they could be stored at 4°C until the following day.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.5">Bacterial transformation and plasmid DNA isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="7">
<li>To transform bacterial strains and yield the maximum amount of plasmid DNA:
<ol type="a">
<li>Add half of the ligation reaction (5.5 μL) from step 6d directly into a tube containing ∼50 μL of DH5a competent cells (#18258012) thawed on ice, tap gently and incubate for 30 min on ice.</li>
<li>Heat-shock the competent cells at 42°C for 30 s.</li>
<li>Incubate on ice for 10 min.</li>
<li>Add 1 mL LB medium and incubate at 37°C on a shaking platform at 140 rpm for 45 min.</li>
<li>Centrifuge the tube at 1,500 × <i>g</i> for 5 min. Discard the supernatant but keep 100 μL to resuspend the bacterial pellet.</li>
<li>Plate the mixture on a 100 mm LB agar plate containing 100 μg/mL ampicillin and incubate the plate upside down for 12–15 h at 37°C.</li>
<li>Using a pipette tip, transfer single colonies from the plate in separate tubes, each one containing 3 mL LB broth and 100 μg/mL ampicillin.</li>
<li>Incubate for ∼6 h at 37°C, shaking at 140 rpm.</li>
<li>In a flask (#4980-300) containing 200 mL LB broth with 100 μg/mL ampicillin add 200 μL from the liquid bacterial culture and incubate for 12–15 h at 37°C, shaking at 140 rpm.</li>
<li>Pellet the culture by centrifugation for 10 min at 2,800 × <i>g</i> rpm at 4°C.</li>
<li>Remove the supernatant and store the pellet at −20°C for processing later, or directly proceed to plasmid extraction using the PureLink™ HiPure Plasmid Filter Maxiprep Kit (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Fpurelink_hipure_plasmid_filter_purification_man.pdf">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Fpurelink_hipure_plasmid_filter_purification_man.pdf</a>).</li>
<li>Measure the concentration of each purified plasmid using a NanoDrop spectrophotometer (Q3000).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In the case that bacterial transformation of the cut vector alone yields multiple colonies, Calf intestinal alkaline phosphatase (CIP) (#M0290) could be added to the digested plasmids to prevent self-ligation of linearized cloning vectors.</div>
<h3 id="sec1.6">Cell culture of HBECs before transfection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–15 days</div>
<ol start="8">
<li>Human Bronchial Epithelial Cells (HBECs) are cultured based on the protocol by Komseli et al. (<a href="#bib6">Komseli et al., 2018</a>). Specifically:
<ol type="a">
<li>Thaw one HBEC cryovial containing ∼2 × 10<sup>6</sup> cells and transfer the content in a 15 mL Falcon tube containing 5 mL of Keratinocyte Serum-Free Medium supplemented with 50 μg/mL Bovine Pituitary Extract and 5 ng/mL human epidermal growth factor (EGF).</li>
<li>Spin down at 500 × <i>g</i> at 15°C–22°C, remove completely the supernatant and resuspend the cell pellet in 10 mL Keratinocyte Serum-Free Medium (without antibiotics). The content is transferred in a 100 mm culture dish suitable for adherent cell culture (#664160).
<div class="critical">
<span class="critical-title">Critical:</span> HBECs must be cultured in serum-free medium, as serum exerts toxic effects severely affecting their proliferation and survival.</div>
<div class="note">
<span class="note-title">Note:</span> Avoid using antibiotics in the Keratinocyte medium, as they may induce stress.</div>
</li>
<li>Replenish the culture medium every other day.</li>
</ol>
</li>
<li>When the cells are confluent:
<ol type="a">
<li>Remove Keratinocyte medium and wash once with 1× PBS.</li>
<li>Trypsinize cells in 1 mL trypsin/EDTA (#15400054) and incubate at 37°C for 5 min.</li>
<li>Add an equal (or larger) amount of trypsin neutralizer (#R002100) and split cells by transferring 1/3 or 1/2 of the final volume in a 15 mL tube.
<div class="critical">
<span class="critical-title">Critical:</span> Trypsin neutralizer is essential for inactivating trypsin, as serum is not contained in the medium.</div>
</li>
<li>Pellet down the cells by centrifugation at 500 × <i>g</i> for 5 min at 15°C–22°C. Remove all the supernatant.</li>
<li>Resuspend the pellet in Keratinocyte medium and subsequently seed cells in 100 mm cell culture plates.
<div class="critical">
<span class="critical-title">Critical:</span> Do not exceed a 1:3 dilution when splitting, as cell viability may dramatically decrease.</div>
</li>
<li>Repeat steps 9a–9e every time cells become confluent to maintain a growing HBEC culture.
<div class="critical">
<span class="critical-title">Critical:</span> HBECs should be cultured for at least 10 days after thawing to achieve good transfection efficiency.</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>MAX Efficiency™ DH5α Competent Cells</td>
<td>Invitrogen</td>
<td>Cat# 18258012</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>BbsI</td>
<td>New England Biolabs (NEB)</td>
<td>Cat# R0539S</td>
</tr>
<tr>
<td>NEBuffer™ r2.1</td>
<td>New England Biolabs (NEB)</td>
<td>Supplied with BbsI</td>
</tr>
<tr>
<td>Agarose</td>
<td>Sigma-Aldrich</td>
<td>Cat# A4018</td>
</tr>
<tr>
<td>SYBR Safe DNA gel stain</td>
<td>Invitrogen</td>
<td>Cat# S33102</td>
</tr>
<tr>
<td>DNA ladder 100 bp</td>
<td>New England Biolabs (NEB)</td>
<td>Cat# N3231S</td>
</tr>
<tr>
<td>10× T4 DNA Ligase Reaction Buffer</td>
<td>New England Biolabs (NEB)</td>
<td>Cat# B0202S</td>
</tr>
<tr>
<td>T4 PNK</td>
<td>New England Biolabs (NEB)</td>
<td>Cat# M0201S</td>
</tr>
<tr>
<td>5× T4 DNA Ligase Buffer</td>
<td>Invitrogen</td>
<td>Cat# 46300018</td>
</tr>
<tr>
<td>T4 DNA ligase</td>
<td>Invitrogen</td>
<td>Cat# 15224017</td>
</tr>
<tr>
<td>LB broth</td>
<td>CONDA</td>
<td>Cat# 777491</td>
</tr>
<tr>
<td>Agar</td>
<td>PanReac-AppliChem</td>
<td>Cat# A0949</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>PanReac-Applichem</td>
<td>Cat# A0839</td>
</tr>
<tr>
<td>Keratinocyte Serum-Free Medium</td>
<td>Invitrogen</td>
<td>Cat# 17005-075</td>
</tr>
<tr>
<td>PBS 1×</td>
<td>Biowest</td>
<td>Cat# L0615-500</td>
</tr>
<tr>
<td>0.5% Trypsin/EDTA 10×</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15400054</td>
</tr>
<tr>
<td>Trypsin neutralizer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R002100</td>
</tr>
<tr>
<td>KCl</td>
<td>PanReac-Applichem</td>
<td>Cat# 131494</td>
</tr>
<tr>
<td>TRIS</td>
<td>PanReac-Applichem</td>
<td>Cat# A1086</td>
</tr>
<tr>
<td>MgCl2</td>
<td>Apollo Scientific</td>
<td>Cat# IN1017</td>
</tr>
<tr>
<td>IGEPAL CA-630</td>
<td>Sigma-Aldrich</td>
<td>Cat# I3021</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Sigma-Aldrich</td>
<td>Cat# P1379</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Invitrogen</td>
<td>Cat# AM2546</td>
</tr>
<tr>
<td>FuGENE ® HD Transfection Reagent</td>
<td>Promega</td>
<td>Cat# E2311</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11058021</td>
</tr>
<tr>
<td>Pen/Strep</td>
<td>PAN-Biotech</td>
<td>Cat# P06-07100</td>
</tr>
<tr>
<td>10× Dream Taq Buffer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# B71</td>
</tr>
<tr>
<td>dNTPs</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R0181</td>
</tr>
<tr>
<td>Dream Taq Polymerase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# EP0702</td>
</tr>
<tr>
<td>Calf intestinal alkaline phosphatase (CIP)</td>
<td>New England Biolabs (NEB)</td>
<td>Cat# M0290</td>
</tr>
<tr>
<td>Glacial acetic acid</td>
<td>PanReac-AppliChem</td>
<td>Cat# 131008</td>
</tr>
<tr>
<td>EDTA</td>
<td>CALBIOCHEM-NOVAGEN</td>
<td>Cat# 4005-OP</td>
</tr>
<tr>
<td>DAPI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 62247</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Zymoclean Gel DNA Recovery Kit</td>
<td>Zymo research</td>
<td>Cat# D4001</td>
</tr>
<tr>
<td>PureLink™ HiPure Plasmid Filter Maxiprep Kit</td>
<td>Invitrogen</td>
<td>Cat# K210017</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>QIAGEN</td>
<td>Cat# 28104</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HBEC-CDC6 Tet-ON</td>
<td><a href="#bib6">Komseli et al. (2018)</a></td>
<td>Species: Human; Cell line sex: female. Parental cells are known as HBEC-3KT.</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>gRNA1 Fw</td>
<td> </td>
<td>CACCGCACCATGGAAACTGAGATA</td>
</tr>
<tr>
<td>gRNA1 Rev</td>
<td> </td>
<td>AAACTATCTCAGTTTCCATGGTGC</td>
</tr>
<tr>
<td>gRNA2 Fw</td>
<td> </td>
<td>CACCGAGCACACAAATGCTCAAAGC</td>
</tr>
<tr>
<td>gRNA2 Rev</td>
<td> </td>
<td>AAACGCTTTGAGCATTTGTGTGCTC</td>
</tr>
<tr>
<td>hU6 Primer</td>
<td> </td>
<td>GGGCCTATTTCCCATGATTCCT</td>
</tr>
<tr>
<td>Primer F1</td>
<td> </td>
<td>GGCTCCCGTGCTCAAAGTAT</td>
</tr>
<tr>
<td>Primer R1</td>
<td> </td>
<td>TGTTAGCATGATGTGCCCCT</td>
</tr>
<tr>
<td>Primer F2</td>
<td> </td>
<td>TTGAGGAGCCAGAGGCAAAG</td>
</tr>
<tr>
<td>Primer R2</td>
<td> </td>
<td>AGATCATACTTCCCCCACTCCT</td>
</tr>
<tr>
<td>Primer F3</td>
<td> </td>
<td>AAGCTCTGAAGCCAATCCCC</td>
</tr>
<tr>
<td>Primer R3</td>
<td> </td>
<td>TTGGTTTGGTTCCCTGCACA</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pSpCas9(BB)-2A-GFP (PX458)</td>
<td><a href="#bib7">Ran et al. (2013)</a></td>
<td>pSpCas9(BB)-2A-GFP (PX458) was a gift from Feng Zhang (Addgene plasmid cat# 48138 ; <a href="http://n2t.net/addgene:48138">http://n2t.net/addgene:48138</a>; RRID:Addgene_48138)</td>
</tr>
<tr>
<td>pU6-(BbsI)_CBh-Cas9-T2A-mCherry</td>
<td><a href="#bib2">Chu et al. (2015)</a></td>
<td>pU6-(BbsI)_CBh-Cas9-T2A-mCherry was a gift from Ralf Kuehn (Addgene plasmid cat# 64324; <a href="http://n2t.net/addgene:64324">http://n2t.net/addgene:64324</a>; RRID:Addgene_64324)</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>CRISPOR</td>
<td><a href="http://crispor.tefor.net/">http://crispor.tefor.net/</a></td>
<td>CRISPOR, RRID:SCR_015935</td>
</tr>
<tr>
<td>Primer-BLAST</td>
<td><a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a></td>
<td>Primer-BLAST, RRID:SCR_003095</td>
</tr>
<tr>
<td>UCSC Genome Browser</td>
<td><a href="https://genome.ucsc.edu/">https://genome.ucsc.edu/</a></td>
<td>UCSC Genome Browser, RRID:SCR_005780</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>100 mm culture dish</td>
<td>Greiner Bio-One</td>
<td>Cat# 664160</td>
</tr>
<tr>
<td>Neubauer glass chamber</td>
<td>Marienfeld Superior</td>
<td>Cat# 0640010</td>
</tr>
<tr>
<td>6-well plate</td>
<td>Greiner Bio-One</td>
<td>Cat# 657160</td>
</tr>
<tr>
<td>96-well plates</td>
<td>Greiner Bio-One</td>
<td>Cat# 650180</td>
</tr>
<tr>
<td>24-well plates</td>
<td>Greiner Bio-One</td>
<td>Cat# 662160</td>
</tr>
<tr>
<td>12-well plates</td>
<td>Greiner Bio-One</td>
<td>Cat# 665180</td>
</tr>
<tr>
<td>300 mL flasks</td>
<td>Life sciences</td>
<td>Cat# 4980-300</td>
</tr>
<tr>
<td>Q3000 UV Spectrophotometer</td>
<td>Quawell</td>
<td>N/A</td>
</tr>
<tr>
<td>FACS Aria III</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>The Quick Lysis Buffer recipe was used for genomic DNA preparation in step 5 of the “<a href="#sec3.4">screening for inverted clone validation</a>” section.</p>
<table id="undtbl4">
<caption>Quick lysis buffer recipe (for quick genomic DNA preparation)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>500 mM KCl</td>
<td>50 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>500 mM TRIS pH: 8.3</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>5 mM MgCl<sub>2</sub>
</td>
<td>2.5 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>NP40</td>
<td>0.45%</td>
<td>225 μL</td>
</tr>
<tr>
<td>Tween 20</td>
<td>0.45%</td>
<td>225 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td>18.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
<tr>
<td>20 mg/mL Proteinase K</td>
<td>0.4 μg/μL</td>
<td>1 μL per 50 μL Lysis Buffer</td>
</tr>
</tbody>
</table>
<p class="table-legend">Quick lysis buffer can be stored at 4°C for up to 3 weeks without Proteinase K, which should be added on the day of the cell lysis.</p>
<table id="undtbl5">
<caption>10× TAE (for agarose gel preparation)</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>0.4 M</td>
<td>48.4 g</td>
</tr>
<tr>
<td>Glacial acetic acid</td>
<td>1.14%</td>
<td>11.4 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>12 mM</td>
<td>3.7 g</td>
</tr>
<tr>
<td>dH20</td>
<td> </td>
<td>up to 1 L dH2O</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">10× TAE buffer can be stored at 15°C–22°C for several months. 10× TAE is diluted to 1× working solution.</p>
<table id="undtbl6">
<caption>Agarose gel recipes</caption>
<thead>
<tr>
<th>Component</th>
<th>1.8% agarose gel</th>
<th>1.3% agarose gel</th>
</tr>
</thead>
<tbody>
<tr>
<td>Agarose</td>
<td>0.8 g</td>
<td>1.3 g</td>
</tr>
<tr>
<td>1× TAE</td>
<td>up to 100 mL</td>
<td>up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>100 mL</b></td>
<td><b>100 mL</b></td>
</tr>
<tr>
<td>SYBR Safe DNA gel stain</td>
<td>10 μL</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Agarose gels should ideally be used immediately after preparation or stored in the dark at 4°C, in 1× TAE buffer for up to 24 h following preparation.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Seeding and transfection of HBECs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol>
<li>
<u>Seeding</u> of HBECs prior to transfection:
<ol type="a">
<li>Harvest HBECs as described in steps 9 (a–e) of the “<a href="#sec1.6">cell culture of HBECs before transfection</a>” section above.</li>
<li>Count cells using a Neubauer glass chamber.</li>
<li>Seed ∼8 × 10<sup>4</sup> cells in each well of a 6-well plate containing 2 mL keratinocyte medium to subsequently transfect the cells with the gRNA/Cas9 expressing vectors.</li>
</ol>
</li>
<li>Two days after plating, the keratinocyte medium is replenished and double transfection with the gRNA/Cas9 expressing vectors can be performed. HBECs are transfected with 1.5–2.5 μg of each plasmid DNA using the FuGENE ® HD Transfection Reagent (#E2311).
<ol type="a">
<li>Prepare the Transfection reagent: DNA mixture in a tube using a 4:1 FuGENE® HD Transfection Reagent: DNA ratio, and add the mixture in 200 μL Opti-MEM (#11058021). For example, 3 μg of total plasmid DNA mixed with 12 μL FuGENE are added in 200 μL Opti-MEM.</li>
<li>Resuspend by thorough pipetting and incubate for 15 min at 15°C–22°C.</li>
<li>Add the mixture dropwise in each transfection well containing 2 mL keratinocyte medium, in antibiotic- and serum-free conditions.
<div class="critical">
<span class="critical-title">Critical:</span> Avoid using antibiotics in the transfection medium, as they may reduce the HBEC transfection efficiency due to increased cellular stress.</div>
<div class="note">
<span class="note-title">Note:</span> Use only one gRNA per breakpoint per transfection.</div>
<div class="note">
<span class="note-title">Note:</span> Other adherent cell lines which are normally cultured in the presence of serum (e.g., Li-Fraumeni) can easily be transfected using serum-containing medium, as serum does not interfere with the FuGENE reagent.</div>
</li>
<li>After 24 h, HBECs can be harvested to assess plasmid expression and/or used in downstream analyses.
<div class="critical">
<span class="critical-title">Critical:</span> To minimize stress and achieve high co-transfection efficiency up to 35%, transfect HBECs (or other challenging cell lines) at a confluency of ∼80% (i.e., <u>two days</u> after seeding cells at the density indicated in step 1c.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Single-cell sorting by fluorescence-activated cell sorting (FACS)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="3">
<li>Following transfection, HBECs are subjected to FACS sorting:
<ol type="a">
<li>24 h after transfection, trypsinize and collect HBECs as described in steps 9a–9d of the “<a href="#sec1.6">cell culture of HBECs before transfection</a>” section.</li>
<li>Wash once with 1× PBS and spin down at 500 × <i>g</i> for 5 min at 15°C–22°C.</li>
<li>After discarding the supernatant, resuspend the cell pellet in 500 μL Opti-MEM or Keratinocyte medium and keep on ice until sorting.
<div class="note">
<span class="note-title">Note:</span> Cellular viability can be estimated by adding 4′,6-Diamidino-2-Phenylindole (DAPI) (at a final concentration of 500–1,000 ng/mL) into a portion of the transfected cells resuspended in 500 μL 1× PBS, directly before sorting. Only the DNA of non-viable cells is stained by DAPI due to the inability of DAPI to pass through intact cell membranes of living cells. In the case of HBECs the percentage of dead cells was estimated at 4%–7% of the population.</div>
</li>
<li>Perform single-cell sorting of the cell suspension using a flow cytometer (BD FACS Aria III). Adjust the settings such that only double-positive single cells expressing both GFP and mCherry (GFP+/mCherry+) will be sorted.</li>
<li>Prepare 96-well plates (#650180) for cell sorting, by adding in each well 100 μL keratinocyte medium supplemented with 1% Pen/Strep to avoid contamination during FACS. Prepare at least 4 × 96-well plates.</li>
<li>Sort cells by plating one cell per well of a 96-well plate.
<div class="critical">
<span class="critical-title">Critical:</span> Given that the percentage of single cells surviving after single cell sorting is very low (approximately 30%), it is crucial to fill as many 96-well plates as possible, as this will facilitate the screening procedure and minimize experimental risk.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Propagation of clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 weeks</div>
<ol start="4">
<li>HBECs that survived after seeding into 96-well plates as single cells will gradually start proliferating, giving rise to separate clones in each well.
<ol type="a">
<li>Allow single cell cultures to recover by incubating undisturbed in humid 37°C/5% CO<sub>2</sub> chamber for 7 days after FACS.</li>
<li>Add 100 μL antibiotic-free keratinocyte medium and start monitoring the clones under the microscope.</li>
<li>Replenish the medium every week: Remove carefully the medium from each well using a 100 μL pipette or a multi-channel pipette, trying not to touch the bottom and slowly add 100 μL fresh Keratinocyte medium.</li>
<li>Once single cell-derived clones reach full confluency, the screening procedure can commence.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The use of a multi-channel pipette is highly recommended for easier and faster handling of the clones.</div>
<h3 id="sec3.4">Screening for inverted clone validation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>DNA is first extracted from each clone separately and the presence of the inversion is queried via PCR.</p>
<ol start="5">
<li>For quick preparation of genomic DNA (<a href="#bib1">Charette and Cosson, 2004</a>):
<ol type="a">
<li>After aspirating the medium and washing once with 1× PBS, harvest the cells derived from each well of the 96-well plates by adding 30 μL Trypsin/EDTA 1× and incubating for 5 min at 37°C.</li>
<li>Add an equal or larger volume of Trypsin Neutralizer Solution and resuspend thoroughly by pipetting.</li>
<li>For each clonal culture, transfer half of the amount of solution containing the trypsinized cells in a new well of a 96-well plate containing 200 μL (or more) of keratinocyte medium.
<div class="critical">
<span class="critical-title">Critical:</span> For HBECs and other sensitive cell lines which normally require centrifugation for total trypsin removal, the addition of a larger amount of medium (≥ 200 μl) in the new 96-well plates will yield equivalent results in regards to propagation.</div>
</li>
<li>Lyse the cells which were not transferred into new 96-well plates by directly adding 30 μL of Quick Lysis Buffer (see <a href="#materials-and-equipment">materials and equipment</a> section above) and transfer the lysate in an Eppendorf tube.</li>
<li>Heat for 45 min at 60°C, followed by 10 min at 80°C for Proteinase K inactivation.</li>
<li>The lysate contains the genomic DNA, and 4 μL of the lysate can be used directly as template for PCR.</li>
<li>The rest of the lysate can be stored at −20°C for 1–2 weeks or −80°C for long-term storage.</li>
</ol>
</li>
<li>For the screening process:
<ol type="a">
<li>Set up the PCR reactions using F1/F2 and R1/R2 primers (each pair in a separate reaction) to verify the presence of the inversion and exclude “non-inverted” clones. Set up the reaction for each clone as described below:
<table id="undtbl7">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNA template</td>
<td>4 μL</td>
</tr>
<tr>
<td>10× Dream Taq Buffer</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>10 mM dNTPs</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>10 μM primer mix</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Dream Taq Polymerase (1.25 units)</td>
<td>0.25 μL</td>
</tr>
<tr>
<td>ddH20</td>
<td>up to 25 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>25 μL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Before screening, it is essential that PCR conditions are optimized for the set of primers used. It is recommended to first use the F1/R1 and F2/R2 primer combinations on wild-type genomic DNA template, which should result in an unambiguous PCR product corresponding to the wild-type alleles. For guidance regarding the design of primers for efficient targeting, please refer to the “<a href="#sec1.2">design of primers for identifying the inversion</a>” section above.</div>
</li>
<li>Run the reactions in a 1.3% agarose gel (for recipe see Table in the “<a href="#materials-and-equipment">materials and equipment</a>” section prepared with SYBR Safe DNA gel stain (#S33102). The clones yielding a PCR product (at ∼770 bp and ∼1,300 bp for F1/F2 and R1/R2, respectively, in our experimental setup) harbor at least one inverted allele (<a href="#fig5">Figure 5</a>A).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1518-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Validation of the engineered inversion through PCR</div>
<p>(A and B) Agarose gel images representing: (A) PCR screening of CRISPR-Cas9-engineered clones using F1+F2 primers. Clone 13 possibly harbors the desired inversion. (B) Identification of heterozygosity regarding the inversion; F1+R1: amplification of wild-type product at 921 bp, F2+R2: amplification of wild-type product at 1,100 bp, F3+R3: amplification of the <i>BHLHE40</i> gene located between the breakpoint sites at 856 bp, F1+F2: amplification of inverted breakpoint 1 region at ∼770 bp, R1+R2: amplification of inverted breakpoint 2 region at ∼1,300 bp. The presence of bands for both wild-type and inverted regions in the same clones indicates heterozygosity.</p>
</figcaption>
</figure>
</li>
<li>To assess heterozygosity, run PCR reactions as described in step 6a on the same clones, this time using the F1/R1 and F2/R2 primer combinations (again, each primer combination in a separate reaction). Amplification with these sets of primers will reveal the presence of a wild-type allele in addition to the inverted one previously identified, which indicates heterozygosity (<a href="#fig5">Figure 5</a>B). Absence of amplification using the F1/R1 and F2/R2 primer combinations indicates homozygosity regarding the inversion.
<table id="undtbl8">
<caption>PCR cycling conditions</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">30 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>63°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>15 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>To conclusively validate the engineering of the desired inverted area, PCR products should be first purified using the QIAquick PCR Purification Kit (#28104, <a href="https://www.qiagen.com/us/resources/download.aspx?id=95f10677-aa29-453d-a222-0e19f01ebe17&amp;lang=en">https://www.qiagen.com/us/resources/download.aspx?id=95f10677-aa29-453d-a222-0e19f01ebe17&amp;lang=en</a>) and submitted for Sanger sequencing using the F1 or F2 primer for the inverted breakpoint 1 and R1 or R2 for the inverted breakpoint 2.
<div class="note">
<span class="note-title">Note:</span> The inversion occurs at low frequency after CRISPR-Cas9 editing (&lt;1.5% in HBECs), therefore a large number of clones should be screened via PCR to identify the few ones harboring the inversion.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Propagation of the inverted clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 weeks</div>
<p>The clones identified as “inverted” should be gradually expanded in the case of HBECs or other sensitive cell lines. Specifically, cells should be initially transferred from each well of a 96-well plate to a well of a 24-well plate and then to a 12-well plate before the cells can be safely transferred to a 6-well plate. This is an essential step to avoid additional stress of the cells growing in the presence of Trypsin/EDTA and Neutralizer (see step 5c in the “<a href="#sec3.4">screening for inverted clone validation</a>” section), which cannot be directly removed, as centrifugation of such a small pellet is not recommended at this critical step, to prevent cell loss.</p>
<div class="critical">
<span class="critical-title">Critical:</span> HBECs should be transferred to larger wells only upon reaching confluency, as sub-confluent HBEC cultures exhibit lower viability and propagate at a slower rate.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Estimated yield of plasmid concentration on the NanoDrop spectrophotometer after bacterial transformation and use of the PureLink™ HiPure Plasmid Filter Maxiprep Kit (see Section “<a href="#sec1.5">bacterial transformation and plasmid DNA isolation</a>”): 0.5–2 μg/μL. Estimated percentage of GFP+/mCherry+ HBEC single cells surviving following FACS sorting: 30%. Estimated DNA yield following PCR product purification using the QIAquick PCR Purification Kit (see Section “<a href="#sec3.4">screening for inverted clone validation</a>”, step 6d): 50–150 ng/μL. Estimated inversion efficiency (clones containing at least one “inverted” allele) in the surviving single-cell deriving HBEC cultures: &lt;1.5%.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>One possible limitation of the described protocol stems from the fact that “inverted” clones derive from single cells sorted through FACS. As HBECs are non-cancerous cells whose survival may be compromised in sub-confluent culture conditions, it is likely that a proportion of cells harboring inversions fail to propagate beyond the single cell stage. This will likely have an impact on the yield of “inverted” clones during the screening process. Expanding the protocol to cell lines demanding a higher level of confluency than HBECs to maintain survival may, therefore, require further optimization.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Single cells sorted by FACS display compromised survival. (<a href="#step-by-step-method-details">step-by-step method details</a>, step 3).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Single-cell isolation using FACS is highly recommended, because it is a relatively straightforward method to acquire clonal cultures without the addition of selection chemicals which would potentially confer toxic effects. Single cell survival may partially improve by adjusting various parameters related to the FACS sorting process <i>per se</i>, such as using fresh buffer solutions, adjusting the nozzle size depending on the cell line and maintaining cells in the required temperature until the time of sorting. Those actions will contribute towards minimizing the stress possibly induced by external factors.</p>
<p>If optimizing FACS conditions is not sufficient to improve single cell survival, a more laborious approach may be implemented, which involves sorting clumps of cells rather than single cells and subsequently performing serial dilutions in order to achieve single cell cultures at a later stage. This, of course, may lead to a significant workload increase as the risk of cell clumps not consisting exclusively of GFP+/mCherry+ double positive cells is higher.</p>
<p>In case FACS sorting cannot be implemented for non-HBEC cell lines, manual serial dilution could be applied to obtain single cells, however this method cannot exclude negative or single-positive cells from being propagated.</p>
<p>Antibiotic selection may be a potential alternative route to FACS sorting of non-HBEC cell lines, provided that the plasmid vectors carry antibiotic resistance cassettes (preferably different from one another, to facilitate the selection process) and the cell line of choice can tolerate antibiotic treatment. This method may be more time-efficient compared to single cell FACS sorting.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The inverted amplicon is not detectable. (<a href="#step-by-step-method-details">step-by-step method details</a>, step 6).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>In the case that the primers (F1/R1 and F2/R2) cannot amplify the wild-type genomic region, make sure that the PCR conditions, and especially the annealing temperature (Tm), have been properly optimized based on the Tm of the individual primers (see “<a href="#sec1.2">design of primers for identifying the inversion</a>” section). If primer design parameters seem to be met but there is still no PCR product, we recommend setting up the PCR reactions trying other sets of primers.</p>
<p>In the case that the F1/R1 and F2/R2 primers are able to amplify the wild-type genomic regions, but their combinations (F1/F2 and R1/R2) fail to yield any PCR product corresponding to the inverted regions, this is an indication that the gRNAs used may be inefficient and we recommend using the next best gRNA candidate sequence based on the CRISPOR criteria (see “<a href="#sec1.1">design of gRNAs</a>” section, step 2).</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The inverted breakpoints have been identified by PCR, but after clonal propagation the amplicons are gradually withdrawn (<a href="#step-by-step-method-details">step-by-step method details</a>, “<a href="#sec3.5">propagation of the inverted clones</a>” section).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>This could potentially be attributed to the presence of more than one cells plated in individual wells of the 96-well plates during FACS sorting. FACS generally ensures that single cells are sorted; however, a doublet may be occasionally plated instead of a single cell. Sorting cells in as many as possible 96-well plates offers the possibility to reduce such contaminations.</p>
<p>Another possibility is related to cross-contamination during cell culturing, which may occur when handling wild-type and “inverted” clones at the same time. This can be largely prevented with the systematic use of filter pipette tips, to prevent unwanted transfer of cells among different cultures.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Only one of the inversion breakpoint regions (F1/F2 or R1/R2) is amplified by PCR (<a href="#step-by-step-method-details">step-by-step method details</a>, step 6).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This may possibly occur as a result of an extensive break, preventing the binding of primers to a region proximal to the breakpoint. A possible solution is to design a new set of primers binding further away from the breakpoint (see “<a href="#sec1.2">Design of primers for identifying the inversion</a>” section).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Lipofectamine-based transfection approaches may lead to insufficient transfection efficiency or be highly toxic for non-transformed cells (<a href="#step-by-step-method-details">step-by-step method details</a>, step 2).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>To minimize this problem, we used the FuGene transfection reagent (#E2311), which is a non-liposomal formulation yielding higher efficiency and reduced toxicity compared to other lipofectamine reagents which were also tried on HBECs. Another potential solution to the problem would be the implementation of virus-based approaches, including lenti or adenoviral vectors. Those virus-based approaches would likely increase transfection efficiency, enabling replication of results and validation of a chromosomal inversion.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Charette, S.J. and Cosson, P. (2004). Preparation of genomic DNA from dictyostelium discoideum for PCR analysis. Biotechniques <i>36</i>, 574-575. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref1">View at publisher</a></p>
<p id="bib2">Chu, V.T., Weber, T., Wefers, B., Wurst, W., Sander, S., Rajewsky, K., and Kuhn, R. (2015). Increasing the efficiency of homology-directed repair for CRISPR-Cas9-induced precise gene editing in mammalian cells. Nat. Biotechnol. <i>33</i>, 543-548. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref2">View at publisher</a></p>
<p id="bib3">Doench, J.G., Fusi, N., Sullender, M., Hegde, M., Vaimberg, E.W., Donovan, K.F., Smith, I., Tothova, Z., Wilen, C., Orchard, R., et al. (2016). Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nat. Biotechnol. <i>34</i>, 184-191. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref3">View at publisher</a></p>
<p id="bib4">Hsu, P.D., Scott, D.A., Weinstein, J.A., Ran, F.A., Konermann, S., Agarwala, V., Li, Y., Fine, E.J., Wu, X., Shalem, O., et al. (2013). DNA targeting specificity of RNA-guided Cas9 nucleases. Nat. Biotechnol. <i>31</i>, 827-832. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref4">View at publisher</a></p>
<p id="bib5">Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J.A., and Charpentier, E. (2012). A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Science <i>337</i>, 816-821. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref5">View at publisher</a></p>
<p id="bib6">Komseli, E.S., Pateras, I.S., Krejsgaard, T., Stawiski, K., Rizou, S.V., Polyzos, A., Roumelioti, F.M., Chiourea, M., Mourkioti, I., Paparouna, E., et al. (2018). A prototypical non-malignant epithelial model to study genome dynamics and concurrently monitor micro-RNAs and proteins in situ during oncogene-induced senescence. BMC Genomics <i>19</i>, 37. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref6">View at publisher</a></p>
<p id="bib7">Ran, F.A., Hsu, P.D., Wright, J., Agarwala, V., Scott, D.A., and Zhang, F. (2013). Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. <i>8</i>, 2281-2308. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref7">View at publisher</a></p>
<p id="bib8">Tycko, J., Wainberg, M., Marinov, G.K., Ursu, O., Hess, G.T., Ego, B.K., Aradhana, Li, A., Truong, A., Trevino, A.E., et al. (2019). Mitigation of off-target toxicity in CRISPR-Cas9 screens for essential non-coding elements. Nat. Commun. <i>10</i>, 4063. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref8">View at publisher</a></p>
<p id="bib9">Zampetidis, C.P., Galanos, P., Angelopoulou, A., Zhu, Y., Polyzou, A., Karamitros, T., Kotsinas, A., Lagopati, N., Mourkioti, I., Mirzazadeh, R., et al. (2021). A recurrent chromosomal inversion suffices for driving escape from oncogene-induced senescence via subTAD reorganization. Mol. Cell <i>81</i>, 4907-4923. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00137-X/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Professor Vassilis G Gorgoulis (<a href="mailto:vgorg@med.uoa.gr">vgorg@med.uoa.gr</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The published article includes all datasets and code generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>We would like to thank Dr. Panagiota Kafasla and Dr. Paschalis Sideras for technical support. V.G.G. is financially supported by the European Union Horizon 2020 Research and Innovation Program under Marie Sklodowska-Curie grant agreement 722729 (SYNTRAIN). V.G.G. is also supported by the National Public Investment Program of the Ministry of Development and Investment/<a href="https://doi.org/10.13039/501100003448">General Secretariat for Research and Technology</a> in the framework of the Flagship Initiative to address SARS-CoV-2 (2020SE01300001); the Welfare Foundation for Social &amp; Cultural Sciences (KIKPE), Athens, Greece; an H. Pappas donation; grants 775 (Hippo) and 3782 (PACOREL) from the <a href="https://doi.org/10.13039/501100013209">Hellenic Foundation for Research and Innovation</a> (HFRI); and NKUA-SARG grant 70/3/8916. A.A. is co-funded by the <a href="https://doi.org/10.13039/501100013209">Hellenic Foundation for Research and Innovation</a> (HFRI) under the 3rd Call for HFRI PhD Fellowships (fellowship number: 5747). A. Papaspyropoulos is co-funded by the <a href="https://doi.org/10.13039/501100005411">Foundation for Education and European Culture</a> (IPEP). A. Papantonis is supported by the <a href="https://doi.org/10.13039/501100001659">Deutsche Forschungsgemeinschaft</a> via the Clinical Research Unit KFO5002 project grant (CO 1568/2-1) and the Priority Program SPP2202 (grant PA 2456/11-1).</p>
<h3>Author contributions</h3>
<p>A.A. and A. Papaspyropoulos contributed equally to this work. A.A. and A. Papaspyropoulos contributed to writing the protocol. A. Papantonis and V.G.G. contributed to writing the protocol and supervised the work.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>