<article>
<section>
<h1 id="header">Isolation of human ESC-derived cardiac derivatives and embryonic heart cells for population and single-cell RNA-seq analysis</h1>
<p><time datetime="2021-02-12">Published: February 12, 2021</time></p>
<p>Federica Santoro,<sup><a href="#aff1">1</a></sup> Kenneth R. Chien,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#cor1">*</a></sup> and Makoto Sahara<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn1">5</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm 171 77, Sweden</p>
<p id="aff2"><sup>2</sup>Department of Medicine, Karolinska Institutet, Stockholm 141 86, Sweden</p>
<p id="aff3"><sup>3</sup>Institution of Immunology, Genetics and Pathology, Uppsala Universitet, Uppsala 751 85, Sweden</p>
<p id="aff4"><sup>4</sup>Department of Surgery, Yale University School of Medicine, New Haven, CT 06510, USA</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kenneth.chien@ki.se">kenneth.chien@ki.se</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:makoto.sahara@ki.se">makoto.sahara@ki.se</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100339">10.1016/j.xpro.2021.100339</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The combination of population and single-cell RNA sequencing analysis using human embryonic stem cell (hESC) differentiation and developmental tissues is a powerful approach to elucidate an organ-specific cellular and molecular atlas in human embryogenesis. This protocol describes (1) cardiac-directed differentiation and isolation of hESC-derived cardiac derivatives with fluorescence-activated cell sorting, (2) isolation of human embryonic heart-derived single cardiac cells, and (3) construction of cDNA libraries with Smart-seq2. These allow for the preparation of human developmental samples for comprehensive transcriptional analysis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib6">Sahara et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Efficient cardiac-directed differentiation of hESCs</li>
<li>Isolation of hESC-derived cardiac derivatives with fluorescence-activated cell sorting</li>
<li>Isolation of human embryonic heart-derived single cardiac cells</li>
<li>Construction of cDNA libraries for both population and single-cell RNA sequencing</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/472-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Maintenance of hESC culture</h3>
<ol>
<li>Prepare Matrigel-coated 6-well and 12-well plates:
<ol type="a">
<li>Thaw frozen Matrigel (BD Biosciences) at 4°C for 12–16 h. Aliquot and freeze again directly.
<div class="note">
<span class="note-title">Note:</span> Store the aliquots at −20°C for up to 6 months. The use of pre-cooled pipets, tips, and tubes are recommended when handling Matrigel, to avoid forming a gel.</div>
</li>
<li>Resuspend thawed Matrigel in cold DMEM/F12 at a 1:100 ratio (e.g., 120 μL of Matrigel in 12 mL of DMEM/F12).</li>
<li>Add 1 mL (or 0.5 mL) of diluted Matrigel per well of a 6-well (or 12-well) plate (approximately 7–10 μg/cm<sup>2</sup>). Ensure that the entire surface in each well is covered.</li>
<li>Store the coated plates at 4°C for up to 2 weeks.</li>
<li>Before use, incubate the coated plate at 37°C for 30–45 min. Then, after aspirating Matrigel solutions, the plate is ready to seed fresh hESCs for cell maintenance (on 6-well plates) or differentiation (on 12-well plates).</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Culture and maintain the human ESC lines (e.g., ES03) on Matrigel-coated 6-well plates in a humidified incubator at 37°C and 5% CO<sub>2</sub>. Cells are fed daily with mTeSR-1 medium and passaged every 5–7 days by incubating with Dispase (for 10–15 min at 37°C) or Accutase (for 5–10 min at 37°C). For the first 24 h after subculturing, the media should be supplemented with 5 μM Y-27632 (a ROCK inhibitor).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The Essential 8 medium and vitronectin (diluted at 1:100 in PBS)-coated plates, can be used as substitutes for mTeSR-1 medium and Matrigel-coated plates for maintenance culture of hESCs. When using the Essential 8 and vitronectin system, Versene solution (for 5–10 min at 37°C) is recommended to use as a non-enzymatic cell dissociation reagent for passaging the cells.</div>
<ol start="3">
<li>Before starting differentiation experiments, check if the cultured hESCs entirely express pluripotency markers (e.g., Oct3/4) via immunocytochemistry or flow cytometry, and verify that the hESCs show a normal karyotype using standard G-band karyotype analysis (<a href="#bib8">Tidball et al., 2017</a>; <a href="#bib7">Sun et al., 2020</a>).</li>
</ol>
<h3 id="sec1.2">Human embryonic/fetal heart experiments</h3>
<ol start="4">
<li>For human embryonic/fetal heart experiments, an Institutional Review Board (IRB)-approved ethical permission and approved informed consents must be first acquired. Human embryonic/fetal heart samples (e.g., 4.5 to 10 weeks of the gestation stages) should be obtained from authorized sources using an IRB-approved protocol (<a href="#bib6">Sahara et al., 2019</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Human samples are considered to be potentially infectious and should be handled in Biosafety Level II cabinets.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>See <a href="#tbl1">Table 1</a> for antibodies for flow cytometry</td>
<td>This paper</td>
<td><a href="#tbl1">Table 1</a></td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human embryonic/fetal heart tissues</td>
<td>Authorized sources in Karolinska University Hospital (Sweden)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>mTeSR1</td>
<td>STEMCELL Technologies</td>
<td>Cat.# 05850</td>
</tr>
<tr>
<td>Essential 8 medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# A1517001</td>
</tr>
<tr>
<td>RPMI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 11875119</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 11320033</td>
</tr>
<tr>
<td>DPBS (1×) (no calcium, no magnesium)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 14190144</td>
</tr>
<tr>
<td>HEPES (1 M)</td>
<td>Gibco</td>
<td>Cat.# 11560496</td>
</tr>
<tr>
<td>Matrigel</td>
<td>BD Biosciences</td>
<td>Cat.# 354234</td>
</tr>
<tr>
<td>Vitronectin</td>
<td>Thermo Fisher</td>
<td>Cat.# A14700</td>
</tr>
<tr>
<td>Dispase</td>
<td>STEMCELL Technologies</td>
<td>Cat.# 07913</td>
</tr>
<tr>
<td>Accutase</td>
<td>STEMCELL Technologies</td>
<td>Cat.# 07920</td>
</tr>
<tr>
<td>Versene solution</td>
<td>Gibco</td>
<td>Cat.# 15040066</td>
</tr>
<tr>
<td>Y-27632</td>
<td>Tocris</td>
<td>Cat.# 1254</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>Sigma-Aldrich</td>
<td>Cat.# SML1046</td>
</tr>
<tr>
<td>CHIR98014</td>
<td>Sigma-Aldrich</td>
<td>Cat.# SML1094</td>
</tr>
<tr>
<td>IWP-2</td>
<td>Tocris</td>
<td>Cat.# 3533</td>
</tr>
<tr>
<td>Wnt-C59</td>
<td>Tocris</td>
<td>Cat.# 5148</td>
</tr>
<tr>
<td>B-27 Supplement (50×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 17504044</td>
</tr>
<tr>
<td>B-27 Supplement (50×), minus insulin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# A1895601</td>
</tr>
<tr>
<td>DMSO</td>
<td>Sigma-Aldrich</td>
<td>Cat.# 2650</td>
</tr>
<tr>
<td>Hydrochloric acid (37%)</td>
<td>Sigma-Aldrich</td>
<td>Cat.# 320331</td>
</tr>
<tr>
<td>Saponin</td>
<td>Sigma-Aldrich</td>
<td>Cat.# 84510</td>
</tr>
<tr>
<td>Horse serum</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 1605130</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>Sigma-Aldrich</td>
<td>Cat.# A2153</td>
</tr>
<tr>
<td>Paraformaldehyde</td>
<td>Sigma-Aldrich</td>
<td>Cat.# 158127</td>
</tr>
<tr>
<td>RNaseOUT ribonuclease inhibitor (40 U/μL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 10777019</td>
</tr>
<tr>
<td>Collagenase type 2</td>
<td>Worthington</td>
<td>Cat.# 4176</td>
</tr>
<tr>
<td>Protease type XXIV</td>
<td>Sigma-Aldrich</td>
<td>Cat.# P8038</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>Sigma-Aldrich</td>
<td>Cat.# 12103C</td>
</tr>
<tr>
<td>TrypLE Express enzyme</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 12604013</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat.# X100</td>
</tr>
<tr>
<td>dNTP mix</td>
<td>New England Biolabs</td>
<td>Cat.# N0447S</td>
</tr>
<tr>
<td>SuperScript II reverse transcriptase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 18064014</td>
</tr>
<tr>
<td>KAPA HiFi HotStart ReadyMix</td>
<td>KAPA Biosystems</td>
<td>Cat.# KK2601</td>
</tr>
<tr>
<td>AMPure XP beads</td>
<td>Beckman Coulter</td>
<td>Cat.# A63881</td>
</tr>
<tr>
<td>DAPI solution (1.0 mg/mL)</td>
<td>BD Pharmingen</td>
<td>Cat.# 564907</td>
</tr>
<tr>
<td>PI solution (100 μg/2 mL)</td>
<td>BD Pharmingen</td>
<td>Cat.# 5564463</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RecoverAll Total Nucleic Acid Isolation Kit</td>
<td>Ambion</td>
<td>Cat.# AM1975</td>
</tr>
<tr>
<td>Smart-Seq2</td>
<td>(<a href="#bib5">Picelli et al., 2013</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Agilent RNA 6000 Nano Kit</td>
<td>Agilent</td>
<td>Cat.# 5067-1511</td>
</tr>
<tr>
<td>Agilent RNA 6000 Nano Chips</td>
<td>Agilent</td>
<td>Cat.# 5067-1529</td>
</tr>
<tr>
<td>Agilent High-Sensitivity DNA Kit</td>
<td>Agilent</td>
<td>Cat.# 5067-4626</td>
</tr>
<tr>
<td>Agilent High-Sensitivity DNA Chips</td>
<td>Agilent</td>
<td>Cat.# 5067-4627</td>
</tr>
<tr>
<td>Nextera XT DNA Library Preparation Kit</td>
<td>Illumina</td>
<td>Cat.# FC-131-1024</td>
</tr>
<tr>
<td>Nextera XT Index Kit (24 or 96 indexes for 96 or 384 samples)</td>
<td>Illumina</td>
<td>Cat.# FC-131-1001, # FC-131-2001</td>
</tr>
<tr>
<td>Qubit High-Sensitivity DNA Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# Q32854</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Population RNA-seq data</td>
<td>This paper</td>
<td>E-MTAB-7537</td>
</tr>
<tr>
<td>Single-cell RNA-seq data</td>
<td>This paper</td>
<td>PRJNA510181</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Human ESC: ES03 line</td>
<td>WiCell</td>
<td>Cat.# ES03</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>oligo-dT primer:5′-AAGCAGTGGTATCAACGCAGAGTACT<sub>30</sub>VN-3′</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Template-switching oligo:5′-AAGCAGTGGTATCAACGCAGAGTACATrGrG+G-3′ (∗rG: riboguanosine; +G: Locked Nucleic Acid-modified guanosine)</td>
<td>QIAGEN</td>
<td>N/A</td>
</tr>
<tr>
<td>IS-PCR primer: 5′-AAGCAGTGGTATCAACGCAGAGT-3′</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FACS Diva</td>
<td>BD Biosciences</td>
<td>RRID:SCR_001456</td>
</tr>
<tr>
<td>FlowJo</td>
<td>Tree Star</td>
<td>RRID:SCR_008520</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>TC20 automated cell counter</td>
<td>Bio-Rad</td>
<td>Cat.# 1450102</td>
</tr>
<tr>
<td>Fine small scissor</td>
<td>Fine Science Tools</td>
<td>No. 14090-09</td>
</tr>
<tr>
<td>Fine spring scissor</td>
<td>Fine Science Tools</td>
<td>No. 15000-10</td>
</tr>
<tr>
<td>Fine forceps</td>
<td>Dumont</td>
<td>No. 11251-20, 11251-35</td>
</tr>
<tr>
<td>Magnetic Stand-96</td>
<td>Ambion</td>
<td>Cat.# AM10027</td>
</tr>
<tr>
<td>FACSARIA III cell sorter</td>
<td>BD Biosciences</td>
<td>RRID:SCR_016695</td>
</tr>
<tr>
<td>Bioanalyzer instrument</td>
<td>Agilent</td>
<td>RRID:SCR_018043</td>
</tr>
<tr>
<td>Qubit fluorometer</td>
<td>Thermo Fisher Scientific</td>
<td>RRID:SCR_018095</td>
</tr>
<tr>
<td>Hiseq 2500 system</td>
<td>Illumina</td>
<td>RRID:SCR_016383</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>RPMI/B27-ins medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI</td>
<td>–</td>
<td>490 mL</td>
</tr>
<tr>
<td>B-27 Supplement (50×), minus insulin</td>
<td>1×</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>RPMI/B27 medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI</td>
<td>–</td>
<td>490 mL</td>
</tr>
<tr>
<td>B-27 Supplement (50×)</td>
<td>1×</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>Y-27632, 5 mM (stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Y-27632</td>
<td>–</td>
<td>10 mg</td>
</tr>
<tr>
<td>Sterile DPBS</td>
<td>–</td>
<td>6.24 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>6.24 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve, aliquot, and store at −20°C for up to 6 months or at 4°C for up to 2 weeks. Add in a medium at a 1:1,000 ratio (working concentration: 5 μM).</div>
<table id="undtbl4">
<caption>CHIR99021, 30 mM (stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CHIR99021</td>
<td>–</td>
<td>25 mg</td>
</tr>
<tr>
<td>DMSO</td>
<td>–</td>
<td>1.79 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>1.79 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve, aliquot, and store at −20°C for up to 6 months or at 4°C for up to 2 weeks. Add in a medium at a 1:2,500 ratio (working concentration: 12 μM).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Another GSK-3β inhibitor CHIR98014 can be used as a substitute for CHIR99021 and applied as follows: dissolve 5 mg of CHIR98014 (Sigma) in 2.06 mL of DMSO (stock concentration: 5 mM). Aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks. Add in a medium at a 1:5,000 ratio (working concentration: 1 μM).</div>
<table id="undtbl5">
<caption>IWP2, 5 mM (stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>IWP2</td>
<td>–</td>
<td>10 mg</td>
</tr>
<tr>
<td>DMSO</td>
<td>–</td>
<td>4.29 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>4.29 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve by incubating the mixture at 37°C for 10 min. Aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks. Add in a medium at a 1:1,000 ratio (working concentration: 5 μM).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Another Wnt inhibitor Wnt-C59 can be used as a substitute for IWP2 and applied as follows: dissolve 10 mg of Wnt-C59 (Tocris) in 2.64 mL of DMSO (stock concentration: 10 mM). Aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks. Add in a medium at a 1:5,000 ratio (working concentration: 2 μM).</div>
<table id="undtbl6">
<caption>FACS buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>3%</td>
<td>3 mL</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>–</td>
<td>97 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve and filter the above. Aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks.</div>
<table id="undtbl7">
<caption>Blocking/staining solution A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Horse (or goat) serum</td>
<td>10%</td>
<td>10 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>–</td>
<td>90 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve and filter the above. Aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks.</div>
<table id="undtbl8">
<caption>Blocking/staining solution B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Horse (or goat) serum</td>
<td>10%</td>
<td>10 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>Saponin</td>
<td>0.1%</td>
<td>100 μg</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>–</td>
<td>90 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve and filter the above. Aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks.</div>
<table id="undtbl9">
<caption>4% paraformaldehyde (PFA) solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PFA</td>
<td>4%</td>
<td>4 g</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>–</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve 4 g of PFA in 100 mL PBS using microwave. Adjust the solution pH to 7.4 with 1 N NaOH and 1 N HCl. Aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks.</div>
<table id="undtbl10">
<caption>Permeabilization solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton X-100</td>
<td>0.1%</td>
<td>100 μL</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>–</td>
<td>99.9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Filter, aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks.</div>
<table id="undtbl11">
<caption>Heart-primary solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES (1 M)</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>NaCl solution (5 M)</td>
<td>35 mM</td>
<td>3.5 mL</td>
</tr>
<tr>
<td>Glucose (D(+), MW = 180.16)</td>
<td>10 mM</td>
<td>0.901 g</td>
</tr>
<tr>
<td>Sucrose (MW = 342.30)</td>
<td>134 mM</td>
<td>22.93 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub> (MW = 141.96)</td>
<td>16 mM</td>
<td>1.136 g</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub> (MW = 84.01)</td>
<td>25 mM</td>
<td>1.050 g</td>
</tr>
<tr>
<td>KCl (MW = 74.56)</td>
<td>7.75 mM</td>
<td>0.289 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub> (MW = 136.09)</td>
<td>1.18 mM</td>
<td>0.080 g</td>
</tr>
<tr>
<td>HCl (37%)</td>
<td>–</td>
<td>Up to pH 7.4</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>–</td>
<td>Up to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare and dissolve the above. Filter and store at 4°C for up to 3 months.</div>
<table id="undtbl12">
<caption>10% bovine serum albumin (BSA) solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>10%</td>
<td>10 g</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>–</td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve 100 mg/mL of BSA in PBS. Aliquot and store at −20°C for up to 6 months or at 4°C for up to 2 weeks.</div>
<table id="undtbl13">
<caption>Collagenase A solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Collagenase type II</td>
<td>200 U/mL</td>
<td>4.2 mg</td>
</tr>
<tr>
<td>Protease type XXIV</td>
<td>6 U/mL</td>
<td>7.74 mg</td>
</tr>
<tr>
<td>10% BSA solution</td>
<td>0.5%</td>
<td>600 μL</td>
</tr>
<tr>
<td>Heart-primary solution</td>
<td>–</td>
<td>11.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>12 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve and filter the above in each experimental case. This solution cannot be stocked.</div>
<table id="undtbl14">
<caption>Collagenase B solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Collagenase type II</td>
<td>400 U/mL</td>
<td>24 mg</td>
</tr>
<tr>
<td>Heart-primary solution</td>
<td>–</td>
<td>40 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve and filter the above in each experimental case. This solution cannot be stocked.</div>
<table id="undtbl15">
<caption>Cell suspension buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TrypLE Express enzyme</td>
<td>–</td>
<td>5 mL</td>
</tr>
<tr>
<td>DPBS (1×)</td>
<td>–</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl16">
<caption>Cell lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton X-100</td>
<td>0.2%</td>
<td>2 μL</td>
</tr>
<tr>
<td>RNase inhibitor (40 U/μL)</td>
<td>2 U/μL</td>
<td>50 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>–</td>
<td>948 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl17">
<caption>RT mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (in 10 μL)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>SuperScript II reverse transcriptase (200 U/μL)</td>
<td>10 U/μL</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>RNase inhibitor (40 U/μL)</td>
<td>1 U/μL</td>
<td>0.25 μL</td>
</tr>
<tr>
<td>SuperScript II first-strand buffer (5×)</td>
<td>1×</td>
<td>2 μL</td>
</tr>
<tr>
<td>DTT (100 mM)</td>
<td>5 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Betaine (5 M)</td>
<td>1 M</td>
<td>2 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>6 mM</td>
<td>0.06 μL</td>
</tr>
<tr>
<td>Template-switching oligo (100 μM)</td>
<td>1 μM</td>
<td>0.10 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>–</td>
<td>0.29 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>5.7 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl18">
<caption>PCR pre-amplification mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>First-strand reaction (from step 39)</td>
<td>–</td>
<td>10 μL</td>
</tr>
<tr>
<td>KAPA HiFi HotStart Ready Mix (2×)</td>
<td>1×</td>
<td>12.5 μL</td>
</tr>
<tr>
<td>IS-PCR primer (10 μM)</td>
<td>0.1 μM</td>
<td>0.25 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>–</td>
<td>2.25 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>25 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl19">
<caption>Tagmentation reaction solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNA from PCR (from step 42l)</td>
<td>–</td>
<td>500 pg</td>
</tr>
<tr>
<td>Tagment DNA buffer (2×)</td>
<td>1×</td>
<td>10 μL</td>
</tr>
<tr>
<td>Tagment DNA enzyme mix</td>
<td>–</td>
<td>5 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>–</td>
<td>Up to 20 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> A tagment DNA buffer (2×) and a tagment DNA enzyme mix are supplied in the Nextera XT DNA Library Preparation kit (Illumina).</div>
<table id="undtbl20">
<caption>Enrichment PCR solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tagmented DNA (from step 44d)</td>
<td>–</td>
<td>25 μL</td>
</tr>
<tr>
<td>Nextera PCR Master Mix (2×)</td>
<td>1×</td>
<td>15 μL</td>
</tr>
<tr>
<td>Index 1 primers (N7xx)</td>
<td>–</td>
<td>5 μL</td>
</tr>
<tr>
<td>Index 2 primers (N5xx)</td>
<td>–</td>
<td>5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>–</td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> A Nextera PCR Mater Mix is supplied in the Nextera XT DNA Library Preparation kit (Illumina), and Index 1 and 2 primers are supplied in the Nextera XT Index kit (24 or 96 indexes for 96 or 384 samples).</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Cardiac-directed differentiation of hESCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 14–16 days</div>
<p>Cardiac differentiation of hESCs is conducted with the Wnt signaling modulators, based on previously published protocols (<a href="#bib1">Burridge et al., 2014</a>; <a href="#bib4">Lian et al., 2013</a>) with minor modifications (<a href="#fig1">Figure 1</a>). This approach can generate efficiently cTnT<sup>+</sup> cardiomyocytes occupying 60%–80% of the total differentiated cells in 10–12 days, via the ISL1<sup>+</sup> multipotent cardiac progenitors (MCPs) and cardiac intermediates on day 3 and day 6 of differentiation, respectively. This protocol also generates small subsets of other cardiac derivatives, such as pacemaker, smooth muscle, and endothelial cells (<a href="#bib6">Sahara et al., 2019</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/472-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. hESC cardiac differentiation protocol based on Wnt signaling modulation</div>
<p>CM, cardiomyocyte; MCP, multipotent cardiac progenitor; ME, medium exchange.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> For efficient differentiation, it is highly recommended to use hESCs with less than 50–60 passages.</div>
<ol>
<li>Four days before differentiation (day −4): Dissociate hESCs cultured on maintenance plates with 80%–90% confluency into single cells, and seed them on differentiation plates:
<ol type="a">
<li>Aspirate the old mTeSR1 medium on Matrigel-coated 6-well plates and wash the cells with DPBS once.</li>
<li>Add 1 mL of Accutase to each well. Put the plate in a 37°C, 5% CO<sub>2</sub> incubator for 5–10 min.</li>
<li>Add 1 mL of mTeSR1 or fetal bovine serum (FBS) to each well. Collect all the cells into a 15-mL conical tube by gently pipetting.
<div class="note">
<span class="note-title">Note:</span> Before use, warm all medium, serum, and solutions at 20°C–25°C for more than 20–30 min.</div>
</li>
<li>Count the total cell number using an automated cell counter or a hemocytometer.</li>
<li>Centrifuge the pooled cells at 200 × <i>g</i> for 5 min at 20°C–25°C.</li>
<li>Aspirate the supernatant and resuspend the cells in mTeSR1 supplemented with 5 μM Y-27632 (a ROCK inhibitor) at an appropriate cell density (e.g., 0.5–1 million cells per mL).</li>
<li>Seed 250,000–500,000 dissociated cells in a final volume of 1 mL media (mTeSR1 + 5 μM Y-27632) per well of Matrigel-coated 12-well plates.
<div class="critical">
<span class="critical-title">Critical:</span> The optimal seeding density on plates for efficient cardiac differentiation is variable and dependent on cell lines and passage numbers. The pilot experiments are required to determine it in each experimental case (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec5.1">Problem 1</a>).</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Days −3 to −1: The medium is replaced daily with 1 mL of fresh mTeSR1 per well of the 12-well plate.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This initial cell growing period can be shortened from a total of 4 days to 2 days. If shortened, seed 750,000 to 1.2 million dissociated cells per well of Matrigel-coated 12-well plates in the beginning. The optimal seeding density should be determined empirically in each case.</div>
<ol start="3">
<li>Day 0: Aspirate the old mTeSR1 medium and add 2 mL of RPMI medium supplemented with B27 minus insulin (RPMI/B27-ins) and with 12 μM CHIR99021 (a GSK-3β inhibitor) per well of the 12-well plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The optimal concentration of CHIR99021 may vary a little depending on cell lines and other conditions, but it is usually 6–14 μM (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec5.1">Problem 1</a>).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> 1 μM CHIR98014 can be used as a substitute for CHIR99021. The optimal concentration of CHIR98014 may vary a little depending on cell lines and other conditions but is usually 0.7–1.2 μM.</div>
<ol start="4">
<li>Day 1 (24 h after the addition of CHIR99021 on day 0): Aspirate the RPMI/B27-ins medium with CHIR99021 and add 2 mL of fresh RPMI/B27-ins medium per well of the 12-well plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The RPMI/B27-ins medium with CHIR99021 should be replaced into fresh RPMI/B27-ins medium without CHIR99021 precisely after 24 h.</div>
<ol start="5">
<li>Day 3 (72 h after the addition of CHIR99021 on day 0): Prepare and add the semi-conditioned medium supplemented with IWP2 (a Wnt inhibitor):
<ol type="a">
<li>Collect 1 mL of the old medium per well of the 12-well plate into a 50-mL conical tube.</li>
<li>Add 1 mL of fresh RPMI/B27-ins per well into the conical tube (Mix old and fresh RPMI/B27-ins medium at a 1:1 ratio).</li>
<li>Add 2 μL of 5 mM IWP2 into 2 mL of the semi-conditioned medium from step 5b (final concentration 5 μM).</li>
<li>Aspirate the remaining 1 mL of the old medium per well and add 2 mL of the semi-conditioned medium supplemented with IWP2.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> 2 μM (working concentration) of Wnt-C59 can be used as a substitute for 5 μM IWP2.</div>
<ol start="6">
<li>Day 5 (96 h after the addition of CHIR99021 on day 0): Aspirate the semi-conditioned medium with IWP2 and add 2 mL of fresh RPMI/B27-ins medium per well of the 12-well plate.</li>
<li>Day 7: Aspirate the old RPMI/B27-ins medium and add 2 mL of fresh RPMI medium supplemented with B27 (RPMI/B27) per well of the 12-well plate.</li>
<li>Thereafter, the medium is replaced into 2 mL of fresh RPMI/B27 per well of the 12-well plate every 2 or 3 days.</li>
<li>Beating cardiomyocytes start to appear in the culture, typically at day 8–9 of cardiac differentiation. Robust and broad spontaneous contractions are observed from day 10–12 onward.</li>
</ol>
<h3 id="sec3.2">Isolation of hESC-derived cardiac subpopulations with FACS</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–6 h</div>
<p>Following the <i>in vitro</i> cardiac differentiation as noted above, a variety of hESC-derived cardiac subpopulations that involve early MCPs to cardiac intermediates and differentiated cardiac cells can be isolated at different time points. During differentiation, ISL1<sup>+</sup> cardiac progenitors first appear on day 3 and peak by day 6 when ISL1<sup>+</sup> cells occupy 80%–90% of the total differentiated cells. These cells mainly differentiate into beating cardiomyocytes along with generating small subsets of other differentiated cell types, such as pacemaker, smooth muscle, and endothelial cells from day 8–10 onward. The isolation of the hESC-derived cardiac subpopulations is conducted using fluorescence-activated cell sorting (FACS) and lineage-specific markers’ antibodies for cell staining on days 3 (for MCPs), 6 (for cardiac intermediates), and 10 (for differentiated cardiac cells) of the differentiation, respectively (<a href="#fig1">Figures 1</a> and <a href="#fig2">2</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/472-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Gating strategies for sorting hESC-derived cardiac subpopulations</div>
<p>Representative FACS plots showing examples of gating strategies for the isolation of hESC-derived cardiac derivatives.</p>
<p>(A) The first gating of cells on the FSC and SSC plots to discard cell debris and doublets (each time).</p>
<p>(B–D) Gating the ISL1<sup>+</sup>PDGFRα<sup>+</sup> MCPs (day 3; B), the ISL1<sup>+</sup>Lin<sup>+</sup> cardiac intermediates (day 6; C), and the ISL1<sup>−</sup>Lin<sup>+</sup> differentiated cardiac cells (day 10; D). The plots of cells stained with isotype controls are also shown, respectively. CI, cardiomyocyte intermediate; CM, cardiomyocyte; EC, endothelial cell; EI, endothelial intermediate; MCP, multipotent cardiac progenitor; PI, pacemaker intermediate; PM, pacemaker cell; SMC, smooth muscle cell; SMI, smooth muscle intermediate.</p>
</figcaption>
</figure>
<ol start="10">
<li>On each of the specified days (i.e., days 3, 6, and 10 of cardiac differentiation):
<ol type="a">
<li>Aspirate the old medium on the differentiation 12-well plates and wash the cells with DPBS once.</li>
<li>Add 0.5 mL of Accutase to each well of the 12-well plate. Put the plate in a 37°C, 5% CO<sub>2</sub> incubator for 5–10 min.</li>
<li>Add 0.5 mL of mTeSR1 or FBS to each well. Collect all the cells into a 15-mL conical tube by gently pipetting.</li>
<li>Centrifuge the pooled cells at 200 × <i>g</i> for 5 min at 20°C–25°C.</li>
<li>Aspirate the supernatant and resuspend the cells in 100 μL of DPBS.</li>
<li>Count the total cell number using an automated cell counter or a hemocytometer.</li>
<li>Centrifuge the cells at 200 × <i>g</i> for 5 min at 20°C–25°C.</li>
<li>Aspirate the supernatant and resuspend the cells in Blocking/Staining Solution A.</li>
<li>Divide the cell solution into separate 1.5 mL tubes for unstained and single color-stained controls, and multicolor-stained samples.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The cell concentration (cell number per volume) of each tube for the single color-stained controls and multicolor-stained samples should be the same (e.g., 1–2 × 10<sup>6</sup> cells per 100-200 μL of Blocking/Staining Solution A) (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec5.3">Problem 2</a>).</div>
<ol start="11">
<li>Staining for cell surface markers:
<ol type="a">
<li>Do blocking of the cells in Blocking/Staining Solution A on the shaker at 4°C for 15 min.</li>
<li>Centrifuge the cells at 200 × <i>g</i> for 5 min at 20°C–25°C.</li>
<li>Aspirate the supernatant and resuspend the cells in 100–200 μL of the primary antibodies’ solution (Blocking/Staining Solution A + appropriately diluted primary antibodies for cell surface antigens; e.g., anti-platelet derived growth factor receptor-α [PDGFRα]) (<a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Antibodies to sort hESC-derived cardiac subpopulations via FACS</caption>
<thead>
<tr>
<th> </th>
<th>Cell surface or intracellular antigen</th>
<th>Company</th>
<th>Catalog number</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="5">Primary antibody</td>
</tr>
<tr>
<td>Cardiac troponin T/TNNT2 FITC</td>
<td>intracellular</td>
<td>Abcam</td>
<td>ab105439</td>
<td>1:200</td>
</tr>
<tr>
<td>CD31 APCCY7</td>
<td>cell surface</td>
<td>BD Biosciences</td>
<td>563653</td>
<td>1:150</td>
</tr>
<tr>
<td>HCN4 PE</td>
<td>cell surface</td>
<td>StreeMarq Biosciences</td>
<td>SMC-320D-R-PE</td>
<td>1:200</td>
</tr>
<tr>
<td>ISL1</td>
<td>intracellular</td>
<td>DSHB</td>
<td>39.4D5</td>
<td>1:100</td>
</tr>
<tr>
<td>ISL1 PE</td>
<td>intracellular</td>
<td>BD Biosciences</td>
<td>562547</td>
<td>1:100</td>
</tr>
<tr>
<td>PDGFRα Alexa Fluor (AF) 647</td>
<td>cell surface</td>
<td>BD Biosciences</td>
<td>562798</td>
<td>1:100</td>
</tr>
<tr>
<td>SMMHC</td>
<td>intracellular</td>
<td>Biomedical Technologies</td>
<td>BT-562</td>
<td>1:20</td>
</tr>
<tr>
<td>OCT3/4</td>
<td>intracellular</td>
<td>Santa Cruz</td>
<td>sc-5279</td>
<td>1:50</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th> </th>
<th>Targeted primary antibody</th>
<th>Company</th>
<th colspan="2">Catalog number</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="6">Secondary antibody</td>
</tr>
<tr>
<td>Goat anti-mouse IgG2b secondary antibody, AF 647</td>
<td>ISL1 (39.4D5)</td>
<td>Thermo Fisher Scientific</td>
<td>A-21242</td>
<td colspan="2">1:1,000</td>
</tr>
<tr>
<td>Goat anti-rabbit IgG (H+L) secondary antibody, AF 405</td>
<td>SMMHC (BT-562)</td>
<td>Thermo Fisher Scientific</td>
<td>A-31556</td>
<td colspan="2">1:500</td>
</tr>
<tr>
<td>Goat anti-mouse IgG2b secondary antibody, AF 488</td>
<td>ISL1 (39.4D5), OCT3/4 (sc-5279)</td>
<td>Thermo Fisher Scientific</td>
<td>A-21141</td>
<td colspan="2">1:1,000</td>
</tr>
<tr>
<td colspan="6">Isotype control</td>
</tr>
<tr>
<td>Mouse IgG1 FITC</td>
<td>TNNT2 FITC (ab105439)</td>
<td>Abcam</td>
<td>Ab91356</td>
<td colspan="2">1:20</td>
</tr>
<tr>
<td>Mouse IgG1 APCCY7</td>
<td>CD31 APCCY7 (563653)</td>
<td>BD Biosciences</td>
<td>557873</td>
<td colspan="2">1:150</td>
</tr>
<tr>
<td>Mouse IgG1 PE</td>
<td>HCN4 PE (SMC-320D-R-PE)</td>
<td>Thermo Fisher Scientific</td>
<td>12-4714-42</td>
<td colspan="2">1:20</td>
</tr>
<tr>
<td>Mouse IgG2b</td>
<td>ISL1 (39.4D5)</td>
<td>Thermo Fisher Scientific</td>
<td>14-4732-82</td>
<td colspan="2">1:100</td>
</tr>
<tr>
<td>Mouse IgG1 PE</td>
<td>ISL1 PE (562547)</td>
<td>BD Biosciences</td>
<td>554680</td>
<td colspan="2">1:50</td>
</tr>
<tr>
<td>Mouse IgG2a AF 647</td>
<td>PDGFRα AF647 (562798)</td>
<td>BD Biosciences</td>
<td>557715</td>
<td colspan="2">1:100</td>
</tr>
<tr>
<td>Rabbit IgG</td>
<td>SMMHC (BT-562)</td>
<td>Thermo Fisher Scientific</td>
<td>02-6102</td>
<td colspan="2">1:20</td>
</tr>
<tr>
<td>Mouse IgG2b AF 488</td>
<td>OCT3/4 (sc-5279)</td>
<td>Santa Cruz</td>
<td>sc-3892</td>
<td colspan="2">1:50</td>
</tr>
</tbody>
</table>
</li>
<li>Incubate the cells on the shaker at 4°C for 30–60 min.</li>
<li>Centrifuge the cells at 200 × <i>g</i> for 5 min at 20°C–25°C and wash the cells with 300 μL of DPBS.</li>
<li>Repeat step 11e.</li>
<li>Centrifuge the cells at 200 × <i>g</i> for 5 min at 20°C–25°C and aspirate the supernatant.</li>
</ol>
</li>
</ol>
<ol start="12">
<li>After cell surface markers’ staining (step 11), one of the following steps (a, b, or c) should be adopted:
<ol type="a">
<li>If either staining for inner cell proteins or staining with a secondary antibody is not planned, resuspend the cells in 300 μL of FACS buffer and filter the cell solutions into a 5 mL FACS tube. Keep the FACS tubes on ice until detection. Go to step 18.
<div class="note">
<span class="note-title">Note:</span> For live cell sorting, the cells should be stained with DAPI (4′,6-diamidino-2-phenylindole; a final concentration: 0.05–0.2 μg/mL) or PI (propidium iodide; a final concentration: 2–5 μg/mL) immediately before the FACS running, so that live intact cells can be gated as DAPI- or PI-negative populations.</div>
</li>
<li>If the primary antibody used in step 11 is not conjugated with a fluorochrome, cells should be stained with a fluorochrome-conjugated secondary antibody (e.g., Alexa Fluor 405-, 488-, 555-, and/or 647-conjugated [Molecular Probes]; typically diluted at 1:500–1,000 in 500–1,000 μL of Blocking/Staining Solution A) specific to the appropriate species on the shaker at 4°C for 15 min. Thereafter, return to step 11e–g, and then follow either step 12a or 12c.</li>
<li>If staining for inner cell proteins is concurrently planned, go to step 13.
<div class="note">
<span class="note-title">Note:</span> In the case of the isolation of hESC-derived cardiac subpopulations described here, the step 12c should be adopted because the progenitor marker ISL1 is an inner cell protein and requires cell fixation for staining.</div>
</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Fix the cells with a 4% paraformaldehyde (PFA) solution on the shaker at 20°C–25°C for 20–25 min. Wrap the cell tubes in aluminum foil.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After cells are once stained with fluorochrome-conjugated antibodies, protect the cells from the light at all times.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare all the buffers and solutions at and after fixation (in steps 13–18) to be supplemented with a RNase inhibitor (e.g., RNaseOUT Ribonuclease inhibitor [Thermo Fisher]) at a 1:100 ratio for effective RNA extraction from the fixed and FACS-sorted cells. This is also applied to PBS used for cell washing (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec5.5">Problem 3</a>).</div>
<ol start="14">
<li>Centrifuge the cells at 400 × <i>g</i> for 5 min at 20°C–25°C. Aspirate the supernatant and resuspend the cells in 1 mL of Permeabilization solution.</li>
<li>Repeat step 14.</li>
<li>Centrifuge the cells at 400 × <i>g</i> for 5 min at 20°C–25°C. Aspirate the supernatant and resuspend the cells in 0.5 mL of Blocking/Staining Solution B.</li>
<li>Staining for inner cell proteins:
<ol type="a">
<li>Do blocking of the cells in Blocking/Staining Solution B on the shaker at 4°C for 15 min. Wrap the cell tubes in aluminum foil.</li>
<li>Centrifuge the cells at 400 × <i>g</i> for 5 min at 20°C–25°C.</li>
<li>Aspirate the supernatant and resuspend the cells in 100–200 μL of the primary antibodies’ solution (Blocking/Staining Solution B + appropriately diluted primary antibodies for inner cell proteins; e.g., anti-ISL1) (<a href="#tbl1">Table 1</a>).</li>
<li>Incubate the cells on the shaker at 4°C for 30–60 min.</li>
<li>Centrifuge the cells at 400 × <i>g</i> for 5 min at 20°C–25°C and wash the cells with 300 μL of PBS.</li>
<li>Repeat step 17e.</li>
<li>Centrifuge the cells at 400 × <i>g</i> for 5 min at 20°C–25°C and aspirate the supernatant.</li>
<li>Stain the cells with a fluorochrome-conjugated secondary antibody (typically diluted at 1:500–1,000 in 500–1,000 μL of Blocking/Staining Solution B) specific to the appropriate species on the shaker at 4°C for 15 min.</li>
<li>Centrifuge the cells at 400 × <i>g</i> for 5 min at 20°C–25°C and wash the cells with 300 μL of PBS.</li>
<li>Repeat step 17i.</li>
<li>Centrifuge the cells at 400 × <i>g</i> for 5 min at 20°C–25°C. Aspirate the supernatant, and resuspend the cells in 300 μL of FACS buffer. Filter the cell solutions into a 5 mL FACS tube. Keep the FACS tubes on ice until detection.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For the appropriate gating strategy at FACS, the following stained controls should be prepared in addition to unstained and single color-stained controls. Run and analyze all these controls with samples at the same time (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec5.3">Problem 2</a>):</div>
<div>Isotype primary antibody (derived from the same species and immunoglobulin type)-stained control(s) (<a href="#tbl1">Table 1</a>)</div>
<div>Only the secondary antibody-stained control(s)</div>
<div>Positively and negatively stained cell-type control(s) (For example, undifferentiated hESCs are often used for the latter.)</div>
<ol start="18">
<li>Analyze and sort the stained cells using a flow cytometer and cell sorter (e.g., FACSARIA III [BD Biosciences]) equipped with a 100 μm nozzle at 4°C temperature. Keep the flow rate between 800–1,200 events per second.
<ol type="a">
<li>Use the unstained and single color-stained controls to set appropriate PMT voltages and adjust compensation using a cell analyzing software (e.g., FACS Diva [BD Biosciences]).</li>
<li>Gate single cells with the FSC-A versus SSC-A plot, the FSC-W versus FSC-H plot, and the SSC-W versus SSC-H plot to discard cell debris and doublets (<a href="#fig2">Figure 2</a>A). When performing live cell sorting, gate DAPI- or PI-negative cell populations (<i>see step 12a Note</i>).</li>
<li>Create appropriate sorting gates for the samples through concurrently analyzing and referring to the isotype antibody-stained and the marker-positive or negative cell-type controls.</li>
<li>Sort the ISL1<sup>+</sup>PDGFRα<sup>+</sup> MCPs (day 3), the ISL1<sup>+</sup>Lin (i.e., TNNT2, HCN4, SMMHC, or PECAM1)<sup>+</sup> cardiac intermediates (day 6), and the ISL1<sup>-</sup>Lin<sup>+</sup> differentiated cardiac cells such as cardiomyocytes, pacemaker, smooth muscle, and endothelial cells (CM, PM, SMC, and EC; day 10) (<a href="#fig2">Figures 2</a>B–2D). Sort more than 10,000 (preferably, ≥20,000) fixed cells of each population into pre-coated 1.5 mL tubes containing 100 μL of PBS (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec5.5">Problem 3</a>).</li>
<li>Immediately after sorting, centrifuge the cells in 1.5 mL tubes at 400 × <i>g</i> for 5 min at 4°C and aspirate the supernatant. Go to step 19.</li>
<li>Further analyze the flowcytometry data with the FlowJo software (Tree Star).</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">RNA extraction from fixed, stained, and FACS-sorted cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 h</div>
<p>Total RNA of the intracellular stained and FACS-sorted cells is extracted using an efficient protocol customized for fixed cells (<a href="#bib3">Hrvatin et al., 2014</a>) with minor modifications (i.e., incubation time length and temperature in protease digestion [step 19]).</p>
<div class="critical">
<span class="critical-title">Critical:</span> Before starting to work with RNA, clean the lab environments including a bench, pipettors, and other instruments with an RNase decontamination solution (e.g., RNaseZap solution [Ambion]).</div>
<ol start="19">
<li>Protease digestion: Resuspend the cell pellets (from step 18e) in 100 μL of Digestion Buffer with 4 μL of Protease (both supplied in the RecoverAll Total Nucleic Acid Isolation Kit [Ambion]; <a href="https://www.thermofisher.com/order/catalog/product/AM1975#/AM1975">https://www.thermofisher.com/order/catalog/product/AM1975#/AM1975</a>) in 1.5 mL tubes and incubate the mixture in heat blocks or a water bath at 50°C for 1 h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> After protease digestion, the cell lysates can be snap-frozen and stored in a freezer at −80°C for 2–4 weeks until the following RNA isolation.</div>
<ol start="20">
<li>RNA isolation is conducted according to the manufacturer’s instructions (RecoverAll Total Nucleic Acid Isolation Kit [Ambion]), as described shortly below:
<ol type="a">
<li>Combine 120 μL of Isolation Additive (<i>supplied in the kit</i>) and 275 μL of 100% ethanol.</li>
<li>Add 395 μL of the Isolation Additive/ethanol mixture into the cell lysate from step 19. Mix by gently pipetting.</li>
<li>Pipet the cell lysate/ethanol mixture (from step 20b) onto a filter cartridge placed in a collection tube (<i>both supplied in the kit</i>).</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s to pass the mixture through the filter.</li>
<li>Discard the flow-through and re-set the filter cartridge in the same collection tube.</li>
<li>Add 700 μL of Wash 1 (<i>supplied in the kit</i>) to the filter cartridge.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s. Discard the flow-through and re-set the filter cartridge in the same collection tube.</li>
<li>Add 500 μL of Wash 2/3 (<i>supplied in the kit</i>) to the filter cartridge.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s. Discard the flow-through and re-set the filter cartridge in the same collection tube.</li>
<li>Centrifuge the assembly at 10,000 × <i>g</i> for an additional 30 s to remove residual fluid from the filter.</li>
</ol>
</li>
<li>RNA purification with DNase digestion is conducted according to the manufacturer’s instructions (RecoverAll Total Nucleic Acid Isolation Kit [Ambion]), as described shortly below:
<ol type="a">
<li>Combine 4 μL of DNase, 6 μL of 10× DNase buffer, and 50 μL of nuclease-free water (<i>all supplied in the kit</i>).</li>
<li>Add 60 μL of DNase mix (from step 21a) to the center of the filter cartridge (from step 20j). Cap the tube and incubate for 30 min at 20°C–25°C.</li>
<li>Add 700 μL of Wash 1 to the filter cartridge. Incubate for 60 s at 20°C–25°C.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s. Discard the flow-through and re-set the filter cartridge in the same collection tube.</li>
<li>Add 500 μL of Wash 2/3 to the filter cartridge.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s. Discard the flow-through and re-set the filter cartridge in the same collection tube.</li>
<li>Repeat steps 21e and 21f.</li>
<li>Centrifuge the assembly at 10,000 × <i>g</i> for 1 min to remove residual fluid from the filter.</li>
<li>Transfer the filter cartridge to a fresh collection tube.</li>
<li>Apply 20–60 μL of Elution Solution (<i>supplied in the kit</i>) or nuclease-free water at 20°C–25°C to the center of the filter cartridge, and close the cap.</li>
<li>Place the sample at 20°C–25°C for 1 min.</li>
<li>Centrifuge for 1 min at maximum speed to pass the mixture through the filter. The eluate contains the RNA.</li>
</ol>
</li>
<li>The integrity and concentration of extracted RNAs are measured with the Agilent RNA 6000 Nano kit and the 2100 Bioanalyzer instrument (Agilent) according to the manufacturer’s instructions (<a href="https://www.agilent.com/en/product/automated-electrophoresis/bioanalyzer-systems/bioanalyzer-instrument">https://www.agilent.com/en/product/automated-electrophoresis/bioanalyzer-systems/bioanalyzer-instrument</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The RNA can be stored in a freezer at −80°C for 2–4 weeks until cDNA library construction (from step 36 onward).</div>
<h3 id="sec3.4">Isolation of human embryonic/fetal heart-derived single cardiac cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>This step enzymatically dissociates cardiac cells from each compartment of fresh human embryonic/fetal hearts (<a href="#fig3">Figure 3</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/472-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Workflow of the isolation of human embryonic/fetal heart-derived single cardiac cells</div>
<p>E/F, embryonic/fetal; OFT, outflow tract.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> All the procedures here, except for digestion steps and centrifugation, should be performed under a UV-sterilized cabinet with laminar flow (Biosafety Level II) using standard aseptic precautions. Keep the experimental environment free from RNases and contaminated DNA.</div>
<ol start="23">
<li>Following surgery, quickly transfer human embryonic/fetal hearts (e.g., at 4.5 to 10 weeks of the gestation stages) into 35 mm dishes filled with 5 mL of DMEM/F12 on ice.</li>
<li>Under a microscope in the hood, dissect the hearts carefully into 3 compartments such as outflow tract, ventricle, and atria with removing fatty and connective tissues using fine small and spring scissors (straight type) and forceps (standard and 45° tips). Transfer the regions separately into different 35 mm dishes filled with 5 mL of the Heart-Primary solution on ice.</li>
<li>Cut each region into as small pieces as possible, using the scissors and forceps.</li>
<li>Transfer cut pieces of each region into 3 mL of the Collagenase A solution in a 15 mL tube, separately.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare fresh collagenase A and B solutions every time.</div>
<ol start="27">
<li>Pre-digestion: Place the tubes into a water bath at 37°C with shaking for 10 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The pre-digestion step removes red blood cells and cell debris from the samples.</div>
<ol start="28">
<li>Centrifuge the tubes at 200 × <i>g</i> for 3 min. Discard the supernatant carefully in each tube.</li>
<li>Add 3 mL of the Collagenase B solution into the 15 mL tubes. Gently tap the bottom of the tubes.</li>
<li>Digestion: Place the tubes into a water bath at 37°C with shaking for 25 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the water bath does not have an automated shaking function, shake the tubes manually for 1 min every 5 min.</div>
<ol start="31">
<li>Transfer and pass the suspension including dissociated single cells through a 40-μm cell strainer into a new 50 mL tube filled with 1 mL of FBS for neutralization.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If there are still clearly visible residual tissues, add another 3 mL of the Collagenase B solution into the same 15 mL tube after transferring the supernatant, and repeat step 30. Finally, the dissociated cell suspensions from the first and second digestion steps can be combined (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec5.7">Problem 4</a>).</div>
<ol start="32">
<li>Centrifuge the 50 mL tubes at 200 × <i>g</i> for 3 min. Discard the supernatant carefully.</li>
<li>Resuspend the cells in 200 μL of Cell Suspension Buffer (containing TrypLE Express Enzyme and DPBS at a 1:1 ratio) for manually picking single cells (Go to step 34), or 300 μL of FACS buffer for sorting single cells with FACS (Go to step 35).</li>
</ol>
<ol start="34">
<li>Manual picking of single cells:
<ol type="a">
<li>Transfer the cell solutions (in Cell Suspension Buffer) into a new 35 mm dish.</li>
<li>Using a micro capillary pipette under a microscope in the hood, pick single cells in the volume of 0.3 μL and transfer each single cell into a 0.2-mL thin-walled PCR tube containing 2 μL of Cell Lysis Buffer (i.e., nuclease-free water containing 0.2% Triton X-100 and 2 U/μL of a RNase inhibitor), 1 μL of 10 μM oligo-dT primer (5′-AAGCAGTGGTATCAACGCAGAGTACT<sub>30</sub>VN-3′) and 1 μL of 10 mM dNTP mix.
<div class="note">
<span class="note-title">Note:</span> The 0.2-mL thin-walled PCR tubes containing 2 μL of Cell Lysis Buffer, 1 μL of 10 μM oligo-dT primer and 1 μL of 10 mM dNTP mix per one tube should be prepared before picking cells.</div>
</li>
<li>After isolating, the single cells in the 0.2-mL thin-walled PCR tubes are snap-frozen on dry ices and stored in a freezer at −80°C for 2–4 weeks until cDNA library construction (from step 37 onward).
<div class="critical">
<span class="critical-title">Critical:</span> All the procedures from the heart digestion to picking single cells (steps 23–34) should be completed within 3–4 h, to prevent loss of the cell viabilities. Typically, 150–200 single cells from each region (i.e., outflow tract, ventricle, or atria) can be obtained during the time interval (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec5.9">Problem 5</a>).</div>
</li>
</ol>
</li>
</ol>
<ol start="35">
<li>Single-cell sorting via FACS:
<ol type="a">
<li>Filter the cell solutions (in FACS buffer) into a 5 mL FACS tube. Keep the FACS tubes on ice until detection.</li>
<li>Using a cell sorter (e.g., FACSARIA III) and following step 18b, DAPI- or PI-negative live cells are gated and sorted as one cell per well on 96- or 384-well plates where each well contains 2 μL of Cell Lysis Buffer, 1 μL of 10 μM oligo-dT primer and 1 μL of 10 mM dNTP mix.
<div class="note">
<span class="note-title">Note:</span> The 96- or 384-well plates containing 2 μL of Cell Lysis Buffer, 1 μL of 10 μM oligo-dT primer and 1 μL of 10 mM dNTP mix in each well should be prepared in advance and can be stored at −20°C for 2–4 weeks. After the solutions in the plate are thawed on ice, centrifuge the plate at 200 × <i>g</i> for 30 s before sorting cells.</div>
</li>
<li>After sorting, the 96- or 384-well plates are snap-frozen on dry ices and stored in a freezer at −80°C for 2–4 weeks until cDNA library construction (from step 37 onward).</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Construction of cDNA libraries with Smart-Seq2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–12 h</div>
<p>cDNA libraries of the hESC-derived cardiac bulk populations and the human embryonic/fetal heart-derived single cardiac cells are generated with the Smart-Seq2 method (<a href="#bib5">Picelli et al., 2013</a>) with minor modifications (i.e., we also apply Smart-Seq2 to bulk RNA samples when minor cell populations obtained with very low amounts of RNA due to a fewer cell number and cell fixation before RNA extraction are included). The generated cDNA libraries are ready for sequencing.</p>
<div class="note">
<span class="note-title">Note:</span> For bulk populations’ RNA samples (from step 22), go to step 36, and for single-cell lysates (from step 34 or 35), go to step 37 directly.</div>
<ol start="36">
<li>Transfer 100 pg of total RNA from bulk populations (from step 22) into a 0.2-mL thin-walled PCR tube, containing 1 μL of 10 μM oligo-dT primer and 1 μL of 10 mM dNTP mix. Add nuclease-free water to a total volume of 4.3 μL in each sample.</li>
<li>Denature: RNA solutions (from step 36) or single-cell lysates (from step 34 or 35) are denatured at 72°C for 3 min and immediately placed on ice afterward.</li>
<li>Add 5.7 μL of the reverse transcription (RT) mix, containing 0.5 μL of SuperScript II reverse transcriptase (200 U/μL) and 0.1 μL of 100 μM Template-switching oligo (5′-AAGCAGTGGTATCAACGCAGAGTACATrGrG+G-3′), to each sample (a final reaction volume of 10 μL). Mix the reaction by gently pipetting and spin down the samples at 600 × <i>g</i> for 15 s at 20°C–25°C.</li>
<li>Reverse transcription: Incubate the sample tubes in a thermal cycler as the following condition: 42°C for 90 min, 10 cycles of “50°C for 2 min and 42°C for 2 min,” and 70°C for 15 min. Set and keep the thermal cycler at 4°C after the reaction terminated.</li>
<li>Add 15 μL of the PCR pre-amplification mix, containing 12.5 μL of KAPA HiFi HotStart Ready Mix (2×, KAPA Biosystems) and 0.25 μL of 10 μM IS-PCR primer (5′-AAGCAGTGGTATCAACGCAGAGT-3′), to each sample (a final reaction volume of 25 μL). Mix the reaction by gently pipetting and spin down the samples at 600 × <i>g</i> for 15 s at 20°C–25°C.</li>
<li>PCR pre-amplification: Incubate the sample tubes in a thermal cycler as the following condition: 98°C for 3 min, 15 cycles (for bulk population RNA) or 18 cycles (for single cells) of “98°C for 20 s, 67°C for 15 s, and 72°C for 6 min,” and 72°C for 5 min. Set and keep the thermal cycler at 4°C after the reaction terminated.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The PCR products can be stored in a freezer at −20°C for 1–2 months until the following procedures.</div>
<ol start="42">
<li>PCR purification: Purify the PCR products using AMPure XP beads (Beckman Coulter):
<ol type="a">
<li>Equilibrate AMPure XP beads at 20°C–25°C for 15 min. Then, vortex well.</li>
<li>Add 25 μL of AMPure XP beads (1:1 ratio) to each sample from step 41. Mix the reaction by pipetting and transfer the solutions to a 96-well plate.</li>
<li>Incubate the mixture for 8 min at 20°C–25°C.</li>
<li>Place the 96-well plate on the magnetic stand (e.g., Magnetic Stand-96 [Ambion]) for 5 min.</li>
<li>Discard the clear liquid carefully.</li>
<li>Wash the beads with 200 μL of 80% ethanol solution. Incubate the samples for 30 s and then discard the ethanol.</li>
<li>Repeat the washing (step 42f).</li>
<li>Dry the beads by leaving the plate at 20°C–25°C for 5 min.</li>
<li>Add 17.5 μL of nuclease-free water and resuspend the beads.</li>
<li>Incubate the plate without the magnetic stand for 2 min.</li>
<li>Place the plate on the magnetic stand for 2 min.</li>
<li>Pipette 15 μL of the clear supernatant without disturbing the beads and transfer it to a 0.2-mL thin-walled PCR tube.</li>
</ol>
</li>
<li>First quality check of the constructed cDNA library: Check the library size distribution using a High-Sensitivity DNA chip and a Bioanalyzer (Agilent).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The expected average library size should be around 1.5–2.0 kb with few short (&lt;500 bp) fragments (<a href="#fig4">Figure 4</a>A).</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/472-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Examples of cDNA library profiles obtained by the Bioanalyzer system</div>
<p>(A) Representative examples of good quality cDNA (left) and degraded cDNA (right) after the PCR pre-amplification.</p>
<p>(B) Representative example of successfully tagmented and enriched cDNA after the final enrichment PCR.</p>
</figcaption>
</figure>
<ol start="44">
<li>Tagmentation reaction:
<ol type="a">
<li>Set up the tagmentation reaction solution, containing 500 pg of cDNA (from step 42l), 10 μL of a tagment DNA buffer (2×) and 5 μL of a tagment DNA enzyme mix (Nextera XT DNA Library Preparation kit [Illumina]), to a final reaction volume of 20 μL on ice. Mix the solution by pipetting.</li>
<li>Perform the tagmentation reaction in a thermal cycler at 55°C for 5 min. Set and keep the thermal cycler at 4°C after the reaction terminated.</li>
<li>To strip off the tagment enzyme, add 5 μL of NT buffer (Nextera XT DNA Library Preparation kit) to each sample. Mix the solution by pipetting.</li>
<li>Incubate the mixture for 5 min at 20°C–25°C.</li>
</ol>
</li>
<li>Final enrichment PCR:
<ol type="a">
<li>Prepare the enrichment PCR solution (a final reaction volume of 50 μL), containing 25 μL of tagmented DNA (from step 44d), 15 μL of Nextera PCR Master Mix, 5 μL of Index 1 primers (N7xx) and 5 μL of Index 2 primers (N5xx), using the Nextera XT DNA Library Preparation and Index kits (Illumina).</li>
<li>Perform PCR in a thermal cycler as the following condition: 72°C for 3 min, 95°C for 30 s, 12 cycles of “95°C for 10 s, 55°C for 30 s, and 72°C for 30 s,” and 72°C for 5 min. Set and keep the thermal cycler at 4°C after the reaction terminated.</li>
</ol>
</li>
<li>PCR purification: Repeat step 42 (a to l) but use 30 μL of AMPure XP beads (at a 0.6:1 ratio). Finally, transfer 15 μL of the clear supernatant into a new 1.5-mL polyallomer tube.</li>
<li>Quality check of the final cDNA library: Measure the concentration of each library by a Qubit High-Sensitivity DNA kit and a Qubit fluorometer (Invitrogen) according to the manufacturer’s instructions (<a href="https://www.thermofisher.com/se/en/home/industrial/spectroscopy-elemental-isotope-analysis/molecular-spectroscopy/fluorometers/qubit/qubit-fluorometer.html">https://www.thermofisher.com/se/en/home/industrial/spectroscopy-elemental-isotope-analysis/molecular-spectroscopy/fluorometers/qubit/qubit-fluorometer.html</a>). Check the size distribution of the tagmented library using a High-Sensitivity DNA chip and a Bioanalyzer (Agilent).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The expected size of the tagmented library is around 300–900 bp (<a href="#fig4">Figure 4</a>B).</div>
<ol start="48">
<li>Library pooling:
<ol type="a">
<li>Calculate the molarity of the final library using the concentration obtained with the Qubit system and the average library size obtained with the Bioanalyzer (from step 47).</li>
<li>Dilute each library to a final concentration of 4 nM. Pool equal nanomoles of each sample, confirming that none of them has the same combination of Index 1 (N7xx) and Index 2 (N5xx) adapters.</li>
<li>Measure again the concentration and the library size distribution of the pooled libraries by the Qubit and Bioanalyzer systems. Adjust the concentration to 4 nM, if necessary.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After this step, the constructed cDNA library is ready for sequencing. Perform single-end or paired-end sequencing of the libraries using a next-generation sequencer (e.g., HiSeq 2500 [Illumina]; <a href="http://www.illumina.com/">http://www.illumina.com/</a>) according to the manufacturer’s and the sequence core facility’s instructions.</div>
<h3 id="sec3.6">Expected outcomes</h3>
<p>By applying the cardiac differentiation protocol (<a href="#fig1">Figure 1</a>) and the multicolor cell separation approach with lineage-specific markers’ antibodies (<a href="#tbl1">Table 1</a>), each of the stage-specific cardiac lineages, such as the ISL1<sup>+</sup>PDGFRα<sup>+</sup> MCPs (day 3), the ISL1<sup>+</sup>Lin (i.e., TNNT2, HCN4, SMMHC, or PECAM1)<sup>+</sup> cardiac intermediates (day 6), and the ISL1<sup>−</sup>Lin<sup>+</sup> differentiated cardiac cells (CM, PM, SMC, and EC) (day 10) can be isolated via FACS (<a href="#fig2">Figure 2</a>). Under successful cardiac differentiation, while a great number of cells (e.g., approximately 1–2 million cells from one 12-well culture plate) of the cardiomyocyte lineage (ISL1<sup>+</sup>TNNT2<sup>+</sup> [day 6] and ISL1<sup>−</sup>TNNT2<sup>+</sup> [day 10]) can be sorted, around 10,000 cells in each of the other minor populations (i.e., pacemaker, smooth muscle, and endothelial lineages) can be sorted from one 12-well culture plate. Using the RNA extraction protocol described here, we usually observe the good quality of RNA profiles from more than 10,000 (preferably, ≥20,000) fixed and FACS-sorted cells in each population (<a href="#fig5">Figure 5</a>). With the heart digestion and isolation approach (<a href="#fig3">Figure 3</a>) in this protocol, followed by the cDNA library construction procedure with the Smart-Seq2, the good quality of cDNA library profiles can be obtained from 500–1,000 single cardiac cells derived from one human embryonic/fetal heart (<a href="#fig4">Figure 4</a>). At the end of this protocol, the population and single-cell libraries are ready for sequencing. After sequencing and getting raw sequenced FASTQ files, downstream bioinformatics analysis will allow us to uncover developmental cellular hierarchies and a molecular atlas of individually specified cells in more detail through combining both the population and single-cell transcriptomic data (<a href="#bib6">Sahara et al., 2019</a>). The approach described here could be applicable to the studies investigating not only cardiogenesis but also other organs’ development in human embryogenesis.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/472-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Examples of RNA profiles of fixed and FACS-sorted cells with different cell numbers</div>
<p>Results obtained by the Bioanalyzer system. The RNA integrity number (RIN) scores: (A) 8.6, (B) 8.0, and (C) N/A.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>RNA degradation often happens, especially in the lowest amount of RNA samples derived from single cells and/or fixed and FACS-sorted rare cell populations. This hampers construction of the valid cDNA libraries, which is necessary for proper RNA sequencing and transcriptomic analysis. Therefore, the study described in this protocol requires to repeat the experiments many times, to optimize the conditions in each step, and to submit a great number of samples from the precious materials such as human embryonic/fetal tissues to sequencing, which means that these attempts are highly expensive and time consuming.</p>
<p>Although the cardiac differentiation protocol based on the Wnt signaling modulation (<a href="#fig1">Figure 1</a>) is efficient to differentiate hESCs toward the cardiac lineage (cardiac progenitors, intermediates, and cardiomyocytes), it is important to keep in mind that this approach is still somewhat artificial and may not reflect the <i>in vivo</i> authentic cardiac development precisely.</p>
<p>The limited cell numbers in the single-cell analysis by the Smart-Seq2 method may still bias the results. Especially, it might be difficult to detect rare and/or sensitive cell types and genes among the analyzed samples. Most recently, the Smart-Seq3 method has been reported to increase sensitivity for detecting transcripts per cell, compared to Smart-Seq2 (<a href="#bib2">Hagemann-Jenson et al., 2020</a>). We envision that our protocols described here could be adapted to Smart-Seq3 as well without major procedural changes, which may overcome those shortcomings.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>The ESCs do not differentiate efficiently into cardiomyocytes.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>First, in every 12-well plates of cardiac differentiation, leave 1–2 wells without harvesting cells and continue the differentiation culture in those wells until 12–14 days of differentiation, in order to ensure that the cells on the plate could differentiate properly into beating cardiomyocytes by observing robust and broad spontaneous contractions from day 10–12 onward.</p>
<p>When not well differentiated, check if the hESCs maintained and used for differentiation entirely express pluripotency markers (e.g., Oct3/4) via immunocytochemistry or flow cytometry, and verify that the hESCs show a normal karyotype using standard G-band karyotype analysis (<a href="#bib8">Tidball et al., 2017</a>; <a href="#bib7">Sun et al., 2020</a>). Optimize the starting cell density and the concentration of CHIR99021 (or CHIR98014) for the differentiation, since these parameters are varied and dependent on the cultured cell lines, passage numbers, and other conditions. On day 1, 3, and 5 in cardiac differentiation, change the medium precisely 24, 72, and 96 h after the addition of CHIR99021 (or CHIR98014) on day 0. Use the freshly prepared mediums and chemicals (e.g., Y-27632, CHIR99021, and IWP2).</p>
<h3 id="sec5.3">Problem 2</h3>
<p>The FACS analysis shows that some cell population is under-stained OR over-stained.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Prepare the FACS tubes of single color-stained controls and multicolor-stained samples, all of which contain the stained cells with the same cell number per volume (e.g., 1–2 × 10<sup>6</sup> cells per 100–200 μL of solution). For the appropriate gating, various staining controls, such as the isotype primary antibody-stained control, only the secondary antibody-stained control, and positively and negatively stained cell-type controls should be prepared and analyzed at the same time in addition to unstained and single color-stained controls. Optimize the dilution ratio of the antibodies in each case.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>The RNA profiles of the fixed and FACS-sorted cell populations show low quality (RNA degradation).</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Check if all the buffers and solutions at and after fixation (in steps 13–18) including PBS used for cell washing are supplemented with a RNase inhibitor at a 1:100 ratio. Sort as many fixed cells as possible (preferably, ≥20,000). For this, prepare ≥2–3 of the 12-well plates for cardiac differentiation at the same time, and collect the cells from those plates all together on the designated days.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>The human embryonic/fetal heart tissues are not well digested, and the number of the picked or sorted single cardiac cells is low.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Cut the heart regions into as small pieces as possible under a microscope, using fine scissors and forceps. Prepare the collagenase A and B solutions freshly every time (Do not stock them into the freezer). The incubation time with the collagenase B solution can be extended by 30–40 min. If needed, repeat the digestion step using the collagenase B solution (step 30) and combine all the dissociated cell suspensions in the end.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>The constructed cDNA library profiles of the single cardiac cells show low quality (RNA degradation, etc.).</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Perform all the single-cell experiments in a specified area that is thoroughly cleaned and decontaminated from RNases and DNA. Complete the steps of the heart digestion and isolation of the single cardiac cells by manual picking or FACS sorting in the shortest time (within 3–4 h, at longest). After picking or sorting, immediately put the cells in cell lysis buffer onto dry ices and store at −80°C until cDNA library construction. Work fast and smoothly with the freshly prepared reaction solutions at each step of the Smart-seq2 (step 37 onward).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Burridge, P.W., Matsa, E., Shukla, P., Lin, Z.C., Churko, J.M., Ebert, A.D., Lan, F., Diecke, S., Huber, B., Mordwinkin, N.M., et al. (2014). Chemically defined generation of human cardiomyocytes. Nat. Methods <i>11</i>, 855-860. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00046-0/sref1">View at publisher</a></p>
<p id="bib2">Hagemann-Jensen, M., Ziegenhain, C., Chen, P., Ramsköld, D., Hendriks, G.J., Larsson, A.J.M., Faridani, O.R., and Sandberg, R. (2020). Single-cell RNA counting at allele and isoform resolution using Smart-seq3. Nat. Biotechnol. <i>38</i>, 708-714. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00046-0/sref2">View at publisher</a></p>
<p id="bib3">Hrvatin, S., Deng, F., O'Donnell, C.W., Gifford, D.K., and Melton, D.A. (2014). MARIS: method for analyzing RNA following intracellular sorting. PLoS One <i>9</i>, e89459. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00046-0/sref3">View at publisher</a></p>
<p id="bib4">Lian, X., Zhang, J., Azarin, S.M., Zhu, K., Hazeltine, L.B., Bao, X., Hsiao, C., Kamp, T.J., and Palecek, S.P. (2013). Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/beta-catenin signaling under fully defined conditions. Nat. Protoc. <i>8</i>, 162-175. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00046-0/sref4">View at publisher</a></p>
<p id="bib5">Picelli, S., Björklund, Å.K., Faridani, O.R., Sagasser, S., Winberg, G., and Sandberg, R. (2013). Smart-seq2 for sensitive full-length transcriptome profiling in single cells. Nat. Methods <i>10</i>, 1096-1098. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00046-0/sref5">View at publisher</a></p>
<p id="bib6">Sahara, M., Santoro, F., Sohlmér, J., Zhou, C., Witman, N., Leung, C.Y., Mononen, M., Bylund, K., Gruber, P., and Chien, K.R. (2019). Population and single-cell analysis of human cardiogenesis reveals unique LGR5 ventricular progenitors in embryonic outflow tract. Dev. Cell <i>48</i>, 475-490. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00046-0/sref6">View at publisher</a></p>
<p id="bib7">Sun, L., Li, J., Li, E., Niu, S., Qin, Z., Zhi, Q., Zhao, J., Xiong, H., Li, Y., Jian, L., et al. (2020). CRISPR/Cas9 mediated establishment of a human CSRP3 compound heterozygous knockout hESC line to model cardiomyopathy and heart failure. Stem Cell Res. <i>49</i>, 102077. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00046-0/sref7">View at publisher</a></p>
<p id="bib8">Tidball, A.M., Dang, L.T., Glenn, T.W., Kilbane, E.G., Klarr, D.J., Margolis, J.L., Uhler, M.D., and Parent, J.M. (2017). Rapid generation of human genetic loss-of-function iPSC lines by simultaneous reprogramming and gene editing. Stem Cell Reports <i>9</i>, 725-731. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00046-0/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Makoto Sahara (<a href="mailto:makoto.sahara@ki.se">makoto.sahara@ki.se</a>).</p>
<h4>Materials Availability</h4>
<p>This protocol does not generate new unique reagents.</p>
<h4>Data and Code Availability</h4>
<p>Population RNA-seq data are available in the ArrayExpress database at EMBL-EBI (accession number: E-MTAB-7537; <a href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</a>). Single-cell RNA-seq data are available in the Sequence Read Archive (SRA) (accession number: PRJNA510181; <a href="http://www.ncbi.nlm.nih.gov/sra/">www.ncbi.nlm.nih.gov/sra/</a>).</p>
<h3>Acknowledgments</h3>
<p>This work was supported by research grants from the <a href="https://doi.org/10.13039/501100004063">Knut and Alice Wallenberg Foundation</a> (K.R.C.; 2013.0028), <a href="https://doi.org/10.13039/501100004359">Vetenskapsrådet</a> (K.R.C. and M.S.; 541-2013-8351 and 2019-01359), AstraZeneca Pharmaceuticals (K.R.C.), <a href="https://doi.org/10.13039/501100003793">Swedish Heart and Lung Foundation</a> (M.S.; 20150421 and 20190380), and <a href="https://doi.org/10.13039/501100004047">Karolinska Institutet</a> (K.R.C. and M.S.). F.S. was supported by an <a href="https://doi.org/10.13039/100004410">EMBO</a> long-term fellowship (ALTF 620-2014).</p>
<h3>Author Contributions</h3>
<p>M.S. and K.R.C. conceived and supervised the concepts, ideas, and experimental plans of the study. F.S. and M.S. performed all experiments. F.S., M.S., and K.R.C. interpreted all data. M.S. wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>