<article>
<section>
<h1 id="header">Domain-specific biochemical and serological characterization of SARS-CoV-2 nucleocapsid protein</h1>
<p><time datetime="2021-10-29">Published: October 29, 2021</time></p>
<p>Chao Wu,<sup><a href="#aff1">1</a>,<a href="#cor5">*****</a></sup> Abraham J. Qavi,<sup><a href="#aff1">1</a>,<a href="#fn1">9</a></sup> Austin B. Moyle,<sup><a href="#aff2">2</a>,<a href="#fn1">9</a></sup> Nicole D. Wagner,<sup><a href="#aff2">2</a>,<a href="#fn1">9</a></sup> Asmaa Hachim,<sup><a href="#aff3">3</a>,<a href="#fn1">9</a></sup> Niloufar Kavian,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#fn1">9</a></sup> Aidan R. Cole,<sup><a href="#aff1">1</a>,<a href="#fn1">9</a></sup> Joyce Sweeney-Gibbons,<sup><a href="#aff6">6</a>,<a href="#fn1">9</a></sup> Henry W. Rohrs,<sup><a href="#aff2">2</a></sup> J.S. Malik Peiris,<sup><a href="#aff3">3</a>,<a href="#aff7">7</a></sup> Christopher F. Basler,<sup><a href="#aff6">6</a></sup> Michael L. Gross,<sup><a href="#aff2">2</a></sup> Sophie A. Valkenburg,<sup><a href="#aff3">3</a>,<a href="#cor3">***</a></sup> Christopher W. Farnsworth,<sup><a href="#aff1">1</a>,<a href="#cor4">****</a></sup> Gaya K. Amarasinghe,<sup><a href="#aff1">1</a>,<a href="#fn2">10</a>,<a href="#fn3">11</a>,<a href="#cor1">*</a></sup> and Daisy W. Leung<sup><a href="#aff1">1</a>,<a href="#aff8">8</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pathology and Immunology, Washington University School of Medicine in St. Louis, St. Louis, MO 63110, USA</p>
<p id="aff2"><sup>2</sup>Department of Chemistry, Washington University in St. Louis, St. Louis, MO 63110, USA</p>
<p id="aff3"><sup>3</sup>HKU-Pasteur Research Pole, School of Public Health, The University of Hong Kong, Hong Kong, China</p>
<p id="aff4"><sup>4</sup>Université Paris Descartes, Sorbonne Paris Cité, Faculté de Médecine, Assistance Publique–Hôpitaux de Paris, Hôpital Universitaire Paris Centre, Centre Hospitalier Universitaire Cochin, Service d’Immunologie Biologique, Paris, France</p>
<p id="aff5"><sup>5</sup>Institut Cochin, INSERM U1016, Université Paris Descartes, Sorbonne Paris Cité, Paris, France</p>
<p id="aff6"><sup>6</sup>Center for Microbial Pathogenesis, Institute for Biomedical Sciences, Georgia State University, Atlanta, GA 30303, USA</p>
<p id="aff7"><sup>7</sup>Division of Public Health Laboratory Sciences, School of Public Health, Li Ka Shing Faculty of Medicine, The University of Hong Kong, Hong Kong, China</p>
<p id="aff8"><sup>8</sup>Department of Internal Medicine, Washington University School of Medicine in St. Louis, St. Louis, MO 63110, USA</p>
<p id="fn1"><sup>9</sup>These authors contributed equally</p>
<p id="fn2"><sup>10</sup>Technical contact</p>
<p id="fn3"><sup>11</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:gamarasinghe@wustl.edu">gamarasinghe@wustl.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:dwleung@wustl.edu">dwleung@wustl.edu</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:sophiev@hku.hk">sophiev@hku.hk</a></p>
<p id="cor4"><sup>****</sup>Correspondence: <a href="mailto:cwfarnsworth@wustl.edu">cwfarnsworth@wustl.edu</a></p>
<p id="cor5"><sup>*****</sup>Correspondence: <a href="mailto:chaowu@wustl.edu">chaowu@wustl.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100906">10.1016/j.xpro.2021.100906</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Nucleocapsid proteins are essential for SARS-CoV-2 life cycle. Here, we describe protocols to gather domain-specific insights about essential properties of nucleocapsids. These assays include dynamic light scattering to characterize oligomerization, fluorescence polarization to quantify RNA binding, hydrogen-deuterium exchange mass spectrometry to map RNA binding regions, negative-stain electron microscopy to visualize oligomeric species, interferon reporter assay to evaluate interferon signaling modulation, and a serology assay to reveal insights for improved sensitivity and specificity. These assays are broadly applicable to RNA-encapsidated nucleocapsids.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref20">Wu et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Developed biophysical assays to characterize SARS-CoV-2 N oligomerization</li>
<li>Quantified RNA binding using fluorescence polarization</li>
<li>Mapped RNA binding site with hydrogen-deuterium exchange mass spectrometry</li>
<li>Defined clinical insights from serology and interferon reporter assays</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Human samples</h3>
<p>The collection of human plasma must be approved through Institutional Review Boards prior to any studies. Human samples used with these protocols were approved by the Human Research Protection Office at Washington University in St. Louis and the Institutional Review Board of The Hong Kong University and the Hong Kong Island West Cluster of Hospitals.</p>
<h3 id="sec1.2">Construction of expression plasmid</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<ol>
<li>Perform PCR to amplify desired SARS-CoV-2 N constructs with corresponding primers.</li>
</ol>
<p>For duplicate PCR reactions, use:</p>
<table id="undtbl1">
<thead>
<tr>
<th colspan="2">PCR reactions before adding enzyme</th>
</tr>
<tr>
<th>Volume</th>
<th>Reagents in PCR</th>
</tr>
</thead>
<tbody>
<tr>
<td>62.8 μL</td>
<td>ddH<sub>2</sub>O</td>
</tr>
<tr>
<td>10 μL</td>
<td>10<b>×</b> KOD buffer #1 or #2 depending on product size</td>
</tr>
<tr>
<td>10 μL</td>
<td>DMSO</td>
</tr>
<tr>
<td>4 μL</td>
<td>MgCl<sub>2</sub> (25 mM)</td>
</tr>
<tr>
<td>2 μL</td>
<td>dNTPs (2 mM)</td>
</tr>
<tr>
<td>1 μL</td>
<td>gene template (∼ 100 ng/μL)</td>
</tr>
<tr>
<td>5 μL</td>
<td>forward primer (50 ng/μL)</td>
</tr>
<tr>
<td>5 μL</td>
<td>reverse primer (50 ng/μL)</td>
</tr>
</tbody>
</table>
<p>Split into two PCR tubes, add into each tube:</p>
<table id="undtbl2">
<thead>
<tr>
<th colspan="2">Enzyme needed for each PCR reaction</th>
</tr>
<tr>
<th>Volume</th>
<th>Reagents in PCR</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.1 μL</td>
<td>KOD enzyme</td>
</tr>
</tbody>
</table>
<p>Total reaction volume in each tube should be 50 μL.</p>
<p>Then initiate PCR reaction with following program:</p>
<table id="undtbl3">
<thead>
<tr>
<th colspan="4">PCR cycling conditions using KOD enzyme</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td rowspan="3">30 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>50°C</td>
<td>8 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure there are no restriction enzyme sites inside gene of interest. If there are, mutate internal restriction enzyme sites before cloning constructs. Double check that there is no stop codon within the reverse primer prior to use, as the expression vector has a C-terminal tag. Extension time depends on size of the product amplified. <a href="https://www.emdmillipore.com/US/en/product/KOD-DNA-Polymerase,EMD_BIO-71085">Please refer to Millipore Sigma for specific information on KOD polymerase</a>.</div>
<ol start="2">
<li>Run DNA agarose gel (1%) for ∼ 25 min at 80 V.</li>
<li>Cut out desired DNA bands with visual aid using UV lamp and clean up gel bands with <i>Promega Wizard® SV Gel and PCR Clean-Up System.</i>
</li>
<li>Perform restriction enzyme digestion of expression vector (pET28a-CterTev His6) and amplified gene of interest.</li>
</ol>
<p>For each digestion reaction, use:</p>
<table id="undtbl4">
<thead>
<tr>
<th colspan="2">Digestion reaction</th>
</tr>
<tr>
<th>Volume</th>
<th>Reagents in digestion reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 μL</td>
<td>10<b>×</b> NEB buffer 3.1</td>
</tr>
<tr>
<td>1 μL</td>
<td>BamHI</td>
</tr>
<tr>
<td>1 μL</td>
<td>NdeI</td>
</tr>
<tr>
<td>43 μL</td>
<td>Purified PCR products or vector</td>
</tr>
</tbody>
</table>
<p class="table-legend">Incubate at 37°C for 1 h.</p>
<ol start="5">
<li>For digested vector, remove phosphate group from 5′ end using calf-intestinal alkaline phosphatase for 1 h at 37°C.</li>
<li>Run DNA gel and cut out desired DNA bands for vector and gene of interest.</li>
<li>Gel purify digested vector and amplified gene of interest.</li>
<li>Ligate amplified gene of interest into expression vector.</li>
</ol>
<p>For each ligation reaction, use</p>
<table id="undtbl5">
<thead>
<tr>
<th colspan="2">Ligation reaction</th>
</tr>
<tr>
<th>Volume</th>
<th>Reagents in ligation reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>4 μL</td>
<td>ddH<sub>2</sub>O</td>
</tr>
<tr>
<td>1 μL</td>
<td>10<b>×</b> ligation buffer</td>
</tr>
<tr>
<td>3 μL</td>
<td>digested gene of interest</td>
</tr>
<tr>
<td>1 μL</td>
<td>digested expression vector</td>
</tr>
<tr>
<td>1 μL</td>
<td>T4 DNA ligase</td>
</tr>
</tbody>
</table>
<p class="table-legend">Incubate at 4°C, 12–18 h.</p>
<ol start="9">
<li>Transform DH-5α competent cells with ligation reaction and plate onto LB plates with 50 μg/mL kanamycin. Incubate at 37°C for 12–18 h.</li>
<li>Pick single colony and inoculate into 10 mL LB supplemented with 50 μg/mL kanamycin.</li>
<li>Perform miniprep (<i>Promega Wizard® Plus SV Minipreps</i>) and send plasmids for sequence confirmation.</li>
</ol>
<h3 id="sec1.3">Bacteria growth in LB media</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="12">
<li>Transform 100 μL <i>E. coli</i> BL21 (DE3) competent cells with 1 μL of expression plasmid (∼ 100 ng/ μL) for each construct by heat shock.</li>
<li>Add 900 μL LB media and incubate at 37°C for 1 h.</li>
<li>Spin down cell pellet at 6000 <b>×</b> <i>g</i> for 10 min, and pipette out 900 μL supernatant.</li>
<li>Resuspend remaining media and spread evenly on LB agar plate with 50 μg/mL kanamycin. Incubate at 37°C for 12–18 h.</li>
<li>Pick single colony and inoculate into 10 mL of LB media supplemented with 50 μg/mL kanamycin. Culture at 37°C with shaking until media is turbid.</li>
<li>Transfer culture into 2 L LB media supplemented with 50 μg/mL kanamycin and culture for 4–5 h.</li>
<li>At OD<sub>600</sub> of 0.6–0.7, induce protein expression with 0.5 mM IPTG after cooling down media on ice for 15 min. Continue to shake at 18°C for 12–14 h.</li>
<li>Harvest cells.
<ol type="a">
<li>Harvest cell culture by centrifugation at 6000 × <i>g</i> for 10 min at 10°C.</li>
<li>Using post-induction gel sample, run SDS-PAGE and stain with Coomassie blue to determine expression outcome.</li>
<li>Gently discard the supernatant.</li>
<li>Resuspend cells in 45 mL lysis buffer containing newly added 2-mercaptoethanol (BME) and protease inhibitors (see <a href="#materials-and-equipment">materials and equipment</a> for recipe of lysis buffer).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For RNA binding nucleocapsids, the concentration of salt is crucial in purification outcome. Higher concentration of NaCl is required for purifying RNA-free nucleocapsids.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The harvested cells can be stored at −80°C for years.</div>
<h3 id="sec1.4">Purification of nucleocapsid protein</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day per protein</div>
<p>The C-terminal His tag nucleocapsid proteins are purified using Ni affinity chromatography and size exclusion chromatography (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Protein purification of SARS-CoV-2 nucleocapsid</div>
<p>C-terminal tagged nucleocapsid proteins are purified using Ni affinity chromatography (A) and size exclusion chromatography (B). This purification strategy generates reasonably pure proteins for a series of constructs with one day per protein time cost. Adapted from “His-tagged Protein Purification” and “Protein Purification”, by <a href="http://Biorender.com">Biorender.com</a> (2020). Retrieved from <a href="https://app.biorender.com/biorender-templates">https://app.biorender.com/biorender-templates</a>.</p>
</figcaption>
</figure>
<ol start="20">
<li>After diluting thawed pellet into 45 mL lysis buffer, pass cells through an Avestin EmulsiFlex-C5 homogenizer without and with pressure three times.</li>
<li>Centrifuge sample at 30,000 <b>×</b> <i>g</i> at 10°C for 40 min. Gently transfer the supernatant to a clean container.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Before proceeding to chromatography, filter the supernatant with a 1.2 μm filter to remove large particles.</div>
<ol start="22">
<li>Prewash the column with three column volumes of Ni buffer B and three column volumes of Ni buffer A (see <a href="#materials-and-equipment">materials and equipment</a> for recipe). Apply the supernatant to the Ni-NTA column.</li>
<li>Wash out nonspecific binders with three column volumes of Ni buffer A.</li>
<li>Elute bound protein with a step gradient of 40% Ni buffer B and a step gradient of 100% Ni buffer B.</li>
<li>Perform SDS-PAGE to determine purity.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For nucleocapsids that bind RNA with high affinity and tend to phase separate, low salt conditions are avoided to maximize chance of RNA-free species purification.</div>
<ol start="26">
<li>Concentrate Ni elution to 5 mL using centrifugal filter units (Millpore Amicon® Ultra).</li>
<li>Inject concentrated protein into Superdex 200 size exclusion column, pre-equilibrated with size exclusion buffer (see <a href="#materials-and-equipment">materials and equipment</a> for recipe).</li>
<li>Evaluate fraction purity using Coomassie staining of SDS-PAGE.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Despite the use of high salt concentration during purification, purified nucleocapsid protein may contain multiple species, including RNA-free and RNA-bound (<a href="#bib17">Su et al., 2018</a>; <a href="#bib20">Wu et al., 2021</a>). To evaluate heterogeneity, measure A260/A280 ratios of samples. A ratio below 0.6 is considered RNA-free.</div>
<ol start="29">
<li>Pool together and concentrate separately fractions within each peak before flash freezing in liquid nitrogen for storage.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> When aliquoting, use a range of volumes (20, 50, 100, 200, 500 μL) to minimize unnecessary freeze-thaw cycles for experiments requiring varying amounts of protein.</div>
<h3 id="sec1.5">Pre-HDX quality control</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>For successful analysis of proteins by HDX-MS, a homogenous and high purity sample is required. We generally find that in-house expressed and purified protein is higher in quality than commercially available counterparts. Because the instrument and personnel time investments for the HDX-MS experiments and data analysis are considerable, quality control steps are recommended to maximize the likelihood of success. Although biochemical assays may provide coarse-grained information pertaining to the state and identity of the protein sample, using other high resolution mass spectrometry measurements, described below, will be beneficial for quality control in this application.</p>
<ol start="30">
<li>Characterize/confirm sequence and heterogeneity of the protein construct by using denaturing LC-MS.
<ol type="a">
<li>Denature the protein with low pH (such as 0.1% formic acid), organic (&lt; 5% acetonitrile), reducing agents (such as TCEP), and/or denaturants.</li>
<li>Inject the denatured protein onto an HPLC for desalting and elution into a mass spectrometer (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. LC-MS diagram</div>
<p>(A) Overview of protein sample submitted to HDX prior to MS analysis.</p>
<p>(B) Sample and LC solvent paths for valve position 1, allowing for sample loading into sample loop or elution from desalting column through protease columns (generating peptides), separated by C18 column, and into the MS.</p>
<p>(C) Sample and LC solvent paths for valve position 2, moving protein from sample loop onto desalting column.</p>
<p>Figure created with <a href="http://Biorender.com">Biorender.com</a>.</p>
</figcaption>
</figure>
</li>
<li>Process data by using deconvolution software (e.g., Intact by Protein Metrics, Inc.) or manually to identify heterogeneity.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The differential HDX-MS method described here relies on bottom-up LC-MS/MS analysis to resolve local differences in deuterium uptake. Any heterogeneity (e.g., from mutations, post-translational modifications) will interfere with mapping results and could result in convoluted HDX results reporting on the mixture.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If the oligomeric state is not clarified by other techniques, characterize oligomeric state and confirm binding using native MS.</div>
<h3 id="sec1.6">LC-MS valve setup</h3>
<div class="timing">
<span class="timing-title">Timing:</span> protease column preparation: 2 days; valve set up: 30 min</div>
<p>Assemble a valve setup for online protease digestion, peptide desalting, analytical separation, and elution into a mass spectrometer (<a href="#fig2">Figure 2</a>).</p>
<ol start="31">
<li>Prepare a protease column(s) for online, post-HDX digestion (<a href="#bib19">Wang et al., 2002</a>).
<ol type="a">
<li>Immobilize acidic protease(s) on POROS-20AL beads.
<ol type="i">
<li>Add 20 mg of sodium cyanoborohydride dissolved in 1 mL of 2 M sodium sulfate to a 2 mL solution containing 80 mg of the protease in 50 mM sodium citrate buffer (pH 5), and incubate for 10 min at 18°C–22°C with gentle rocking.
<div class="critical">
<span class="critical-title">Critical:</span> Sodium cyanoborohydride is a flammable solid, reactive with water, and exposure can cause serious burns. Pay special attention to the safety data sheet for proper storage and handling.</div>
</li>
<li>To this mixture, add 600 mg of POROS-20AL beads, adjust the rocker speed to suspend the slurry, and incubate 5–10 min at 18°C–22°C.</li>
<li>Slowly add 2.125 mL of 2 M sodium sulfate (∼ 500 μL every 5 min).</li>
<li>Incubate for 12–16 h at 18°C–22°C with rocking.</li>
<li>Quench the reaction with 1 mL of 1 M ethanolamine and incubate for 2 h at 18°C–22°C.</li>
<li>Filter the coupled beads and wash with 50 mM citrate buffer (pH 5), 1 M NaCl in citrate buffer (pH 5), 50 mM citrate buffer (pH 5), and water/0.1% formic acid. Resuspend beads in water/0.1% formic acid for packing into a column.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Protease coupled beads are available for purchase.</div>
<div class="note">
<span class="note-title">Note:</span> Once coupled, the beads can be stored at 4°C in water/0.1% formic acid for several months.</div>
</li>
</ol>
</li>
<li>Pack beads with an immobilized protease into clean stainless-steel columns (we used 2 mm inner diameter <b>×</b> 20 mm length column).</li>
<li>Wash the packed column for 12–18 h directly into waste with water/0.1% formic acid at 50 μL/min.
<div class="note">
<span class="note-title">Note:</span> With careful use, packed protease columns can be reused for months. After usage, the ends should be capped to keep the beds wet, and the column stored at 4°C. We recommend using a model protein to evaluate the health of the protease column before each use.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Prepacked protease columns are available for purchase.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Instead of online digestion, in-solution protein digestion can be performed with a large excess of proteases, but this increases H/D back-exchange and produces protease peptide fragments congesting the spectra.</div>
</li>
</ol>
</li>
</ol>
<ol start="32">
<li>Incorporate a C8 column for trapping and desalting the digested peptide products.</li>
<li>Use a C18 column for analytical separation of the product peptides.</li>
<li>Couple the LC apparatus to a mass spectrometer capable of both MS and MS/MS (in this case a Bruker MaXis II HM Q-TOF).</li>
<li>During acquisition, maintain the valves, columns, and lines (except the protease column(s)) in an ice bath to reduce back-exchange.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Any mass spectrometer capable of MS and MS/MS can be used for mass analyses.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-human IgG secondary Ab labeled with HRP (1:5000)</td>
<td>Southern Biotech</td>
<td>Cat#905209</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>E. coli BL21(DE3)</td>
<td>Agilent</td>
<td>Cat#200131</td>
</tr>
<tr>
<td>SeV</td>
<td>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY USA</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human Plasma, Patient with Confirmed SARS-CoV-2 Infection</td>
<td>Barnes-Jewish Hospital, St. Louis, MO, USA</td>
<td>N/A</td>
</tr>
<tr>
<td>Human Plasma, Patient with Confirmed SARS-CoV-2 Infection</td>
<td>Hong Kong University and HK Island West Cluster of Hospitals, Hong Kong, PRC</td>
<td>N/A</td>
</tr>
<tr>
<td>Human Plasma, Patient with negative test</td>
<td>Barnes-Jewish Hospital, St. Louis, MO, USA</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>LC-MS grade formic acid</td>
<td>CovaChem</td>
<td>Cat#PI 85171;CAS 64-18-6</td>
</tr>
<tr>
<td>Phosphate buffered saline tablets</td>
<td>Millipore Sigma</td>
<td>Cat#P4417</td>
</tr>
<tr>
<td>Deuterium oxide (D, 99.9%)</td>
<td>Cambridge Isotope Laboratories</td>
<td>Cat#DLM-4-100;CAS 7789-20-0;</td>
</tr>
<tr>
<td>Urea</td>
<td>Millipore Sigma</td>
<td>Cat#U4883;CAS: 57-13-6</td>
</tr>
<tr>
<td>2% Uranyl Acetate</td>
<td>Ted Pella</td>
<td>19481</td>
</tr>
<tr>
<td>KOD DNA polymerase</td>
<td>Millipore Sigma</td>
<td>71085</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Dual-Glo luciferase kit</td>
<td>Promega</td>
<td>Cat#E2920</td>
</tr>
<tr>
<td>Promega Wizard® SV Gel and PCR Clean-Up System</td>
<td>Promega</td>
<td>A9282</td>
</tr>
<tr>
<td>Promega Wizard® Plus SV Minipreps</td>
<td>Promega</td>
<td>A1460</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Human: HEK293T cells</td>
<td>ATCC</td>
<td>CRL-3216</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>20nt ssRNA sequence: UUUCACCUCCCUUUCAGUUU</td>
<td>GenScript</td>
<td>N/A</td>
</tr>
<tr>
<td>19nt slRNA sequence: GGAAGAUUAAUAAUUUUCC</td>
<td>GenScript</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Plasmid pET28a-CterTEV His6 SARS-CoV-2 N 1-419</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pET28a-CterTEV His6 SARS-CoV-2 N 44-419</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pET28a-CterTEV His6 SARS-CoV-2 N 44-369</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pET28a-CterTEV His6 SARS-CoV-2 N 44-247</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pET28a-CterTEV His6 SARS-CoV-2 N 44-176</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pET28a-CterTEV His6 SARS-CoV-2 N 248-369</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pCAGGS MLAV VP35</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pCAGGS SARS-CoV-2 N 1-419</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pCAGGS SARS-CoV-2 N 44-369</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pCAGGS SARS-CoV-2 N 44-176</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pCAGGS SARS-CoV-2 N 248-419</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid pCAGGS SARS-CoV-2 N 248-369</td>
<td>This work</td>
<td>N/A</td>
</tr>
<tr>
<td>IFN-β promoter-firefly luciferase reporter plasmid</td>
<td>Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY USA</td>
<td>N/A</td>
</tr>
<tr>
<td>pRL-TK Renilla luciferase reporter plasmid</td>
<td>Promega</td>
<td>E2231</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Intact Mass™</td>
<td>Protein Metrics Incorporated</td>
<td>Version 3.11</td>
</tr>
<tr>
<td>Byonic™</td>
<td>Protein Metrics Incorporated</td>
<td>Version 3.11</td>
</tr>
<tr>
<td>Byologic™</td>
<td>Protein Metrics Incorporated</td>
<td>Version 3.11</td>
</tr>
<tr>
<td>DataAnalysis v 4.4</td>
<td>Bruker Daltonics</td>
<td>Version 4.4</td>
</tr>
<tr>
<td>HDExaminer</td>
<td>Sierra Analytics Incorporated</td>
<td>Version 2.5.1</td>
</tr>
<tr>
<td>Origin</td>
<td>OriginLab</td>
<td>Version 7</td>
</tr>
<tr>
<td>PRISM</td>
<td>GraphPad</td>
<td>Version 7</td>
</tr>
<tr>
<td>Dynamics Software</td>
<td>Wyatt</td>
<td>Version 7</td>
</tr>
<tr>
<td>Gen5 Software</td>
<td>BioTek</td>
<td>Version 3</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>ZORBAX Eclipse XDB C8 trap column (2.1 <b>×</b> 15 mm)</td>
<td>Agilent Technologies</td>
<td>Cat#975700-936</td>
</tr>
<tr>
<td>XSelect CSH C18 XP (130 Å, 2.5 μm, 2.1 <b>×</b> 50 mm)</td>
<td>Waters Corporation</td>
<td>Cat#186006101</td>
</tr>
<tr>
<td>MaXis II 4G Q-TOF</td>
<td>Bruker Daltonics</td>
<td>N/A</td>
</tr>
<tr>
<td>MaXis II HM Q-TOF</td>
<td>Bruker Daltonics</td>
<td>N/A</td>
</tr>
<tr>
<td>EmulsiFlex-C5 homogenizer</td>
<td>Avestin</td>
<td>N/A</td>
</tr>
<tr>
<td>Glow-Discharged Copper Grid, 200 mesh</td>
<td>Ted Pella</td>
<td>01840-F</td>
</tr>
<tr>
<td>JEM-1400 plus TEM</td>
<td>JEOL</td>
<td>N/A</td>
</tr>
<tr>
<td>Phosphor-Scintillated 12-bit CCD Camera</td>
<td>AMT</td>
<td>XR111</td>
</tr>
<tr>
<td>DynaPro-PlateReader II</td>
<td>Wyatt Technologies Corporation</td>
<td>N/A</td>
</tr>
<tr>
<td>Cytation5 Plate Reader</td>
<td>BioTek</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl6">
<caption>Lysis buffer</caption>
<thead>
<tr>
<th>Reagent (stock concentration)</th>
<th>Final concentration</th>
<th>Amount (mL) for 1000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 7.5 (2 M)</td>
<td>20 mM</td>
<td>10</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>1 M</td>
<td>200</td>
</tr>
<tr>
<td>Imidazole (2 M)</td>
<td>20 mM</td>
<td>10</td>
</tr>
<tr>
<td>2-mecaptoethanol (BME) (14.3 M)</td>
<td>5 mM</td>
<td>0.35</td>
</tr>
<tr>
<td>Antipain (1 mg/mL)</td>
<td>1 μg/mL</td>
<td>1</td>
</tr>
<tr>
<td>Benzamidine (1 mg/mL)</td>
<td>1 μg/mL</td>
<td>1</td>
</tr>
<tr>
<td>Leupepsin (1 mg/mL)</td>
<td>1 μg/mL</td>
<td>1</td>
</tr>
<tr>
<td>Pepstain (0.5 mg/mL)</td>
<td>1 μg/mL</td>
<td>2</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>774.65</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>1</b><b>,</b><b>000</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 6 months. Add protease inhibitors and BME right before use.</p>
<table id="undtbl7">
<caption>Ni buffer A</caption>
<thead>
<tr>
<th>Reagent (stock concentration)</th>
<th>Final concentration</th>
<th>Amount (mL) for 1,000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 7.5 (2 M)</td>
<td>20 mM</td>
<td>10</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>1 M</td>
<td>200</td>
</tr>
<tr>
<td>Imidazole (2 M)</td>
<td>20 mM</td>
<td>10</td>
</tr>
<tr>
<td>2-mecaptoethanol (BME) (14.3 M)</td>
<td>5 mM</td>
<td>0.35</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>779.65</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>1</b><b>,</b><b>000</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 6 months. Add BME right before use.</p>
<table id="undtbl8">
<caption>Ni buffer B</caption>
<thead>
<tr>
<th>Reagent (stock concentration)</th>
<th>Final concentration</th>
<th>Amount (mL) for 1,000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 7.5 (2 M)</td>
<td>20 mM</td>
<td>10</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>1 M</td>
<td>200</td>
</tr>
<tr>
<td>Imidazole (2 M)</td>
<td>500 mM</td>
<td>250</td>
</tr>
<tr>
<td>2-mecaptoethanol (BME) (14.3 M)</td>
<td>5 mM</td>
<td>0.35</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>539.65</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>1</b><b>,</b><b>000</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 6 months. Add BME right before use.</p>
<table id="undtbl9">
<caption>Size exclusion buffer</caption>
<thead>
<tr>
<th>Reagent (stock concentration)</th>
<th>Final concentration</th>
<th>Amount (mL) for 1,000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES pH 7.5 (1 M)</td>
<td>25 mM</td>
<td>25</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>500 mM</td>
<td>100</td>
</tr>
<tr>
<td>Glycerol (80%)</td>
<td>5%</td>
<td>62.5</td>
</tr>
<tr>
<td>TCEP (0.5 M)</td>
<td>2 mM</td>
<td>4</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>808.5</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>1</b><b>,</b><b>000</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterilize with 0.22 μm filter. Store at 4°C for up to 6 months.</p>
<table id="undtbl10">
<caption>DLS buffer</caption>
<thead>
<tr>
<th>Reagent (stock concentration)</th>
<th>Final concentration</th>
<th>Amount (mL) for 1,000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES pH 7.5 (1 M)</td>
<td>25 mM</td>
<td>25</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>150 mM</td>
<td>30</td>
</tr>
<tr>
<td>Glycerol (80%)</td>
<td>5%</td>
<td>62.5</td>
</tr>
<tr>
<td>TCEP (0.5 M)</td>
<td>2 mM</td>
<td>4</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>878.5</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>1</b><b>,</b><b>000</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterilize with 0.22 μm filter. Store at 4°C for up to 6 months.</p>
<table id="undtbl11">
<caption>RP-LC Solvent A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HPLC Grade water</td>
<td>99.9% (%v/v)</td>
<td>999 mL</td>
</tr>
<tr>
<td>MS-Grade Formic Acid</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>1</b> <b>L</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–22°C for no longer than 1 month.</p>
<table id="undtbl12">
<caption>RP-LC Solvent B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HPLC Grade water</td>
<td>19.9% (%v/v)</td>
<td>199 mL</td>
</tr>
<tr>
<td>HPLC Grade acetonitrile</td>
<td>80%</td>
<td>800 mL</td>
</tr>
<tr>
<td>MS-Grade Formic Acid</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>1</b> <b>L</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–22°C for no longer than 1 month.</p>
<table id="undtbl13">
<caption>HDX H<sub>2</sub>O buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 <b>×</b> Phosphate buffered saline tablet</td>
<td>10 mM Phosphate Buffer137 mM NaCl2.7 mM KCl pH 7.4</td>
<td>1 tablet</td>
</tr>
<tr>
<td>HPLC Grade Water</td>
<td>n/a</td>
<td>200 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for no longer than 1 year.</p>
<table id="undtbl14">
<caption>HDX D<sub>2</sub>O buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 <b>×</b> Phosphate buffered saline tablet</td>
<td>10 mM Phosphate Buffer137 mM NaCl2.7 mM KCl pH 7.0 (pD 7.4)</td>
<td>1 tablet</td>
</tr>
<tr>
<td>deuterium oxide (99.9%)</td>
<td>n/a</td>
<td>200 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Aliquot and store frozen (−20°C) for no longer than 1 year.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Adjust the pH with concentrated HCl or NaOH to minimize incorporation of exchangeable hydrogen into the deuterated buffer. Alternatively, you can use DCl or NaOD diluted with D<sub>2</sub>O. Ensure recording the pH of the final solution.</div>
<table id="undtbl15">
<caption>HDX Quench buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Urea (8 M)</td>
<td>4 M</td>
<td>25 mL</td>
</tr>
<tr>
<td>1<b>×</b>PBS</td>
<td>5 mM Phosphate Buffer68.5 mM NaCl1.35 mM KCl pH 2.4</td>
<td>25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Aliquot and store frozen (−20°C) for no longer than 1 year</p>
<div class="critical">
<span class="critical-title">Critical:</span> The kinetic minimum for protein backbone amide H/D exchange is at pH ∼2.5 (<a href="#bib12">Narang et al., 2020</a>); after quenching the exchange, HDX should be minimized, and back exchange with the solvent minimized. Test the pH of a 2:3 mixture of HDX H<sub>2</sub>O buffer:HDX quench buffer solution to ensure the pH is ∼2.5 and record the value.</div>
<div class="note">
<span class="note-title">Note:</span> The appropriate pH is also dependent upon the protease activity. Some sacrifices to H/D back-exchange may be necessary if alternative proteases to pepsin are used.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A variety of quench buffers may be prepared for optimizing the digestion conditions.</div>
<table id="undtbl16">
<caption>MS data acquisition parameters</caption>
<thead>
<tr>
<th>Method node</th>
<th>Parameter</th>
<th>Setting</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">MS and MS/MS</td>
</tr>
<tr>
<td>Mode</td>
<td>Spectra File Type</td>
<td>save line and profile spectra</td>
</tr>
<tr>
<td> </td>
<td>Ion Polarity</td>
<td>positive</td>
</tr>
<tr>
<td> </td>
<td>Mass Range</td>
<td>50–4000 m/z</td>
</tr>
<tr>
<td> </td>
<td>Rolling Average</td>
<td>Off</td>
</tr>
<tr>
<td> </td>
<td>Spectra Rate</td>
<td>2.00 Hz</td>
</tr>
<tr>
<td>ESI Source</td>
<td>End Plate Offset</td>
<td>500 V</td>
</tr>
<tr>
<td> </td>
<td>Capillary</td>
<td>4500 V</td>
</tr>
<tr>
<td> </td>
<td>Nebulizer</td>
<td>2.0 Bar</td>
</tr>
<tr>
<td> </td>
<td>Dry Gas</td>
<td>5.0 L/min</td>
</tr>
<tr>
<td> </td>
<td>Dry Temp</td>
<td>250 C</td>
</tr>
<tr>
<td>Tune Tab</td>
<td>Transfer Funnel 1 RF</td>
<td>400.0 Vpp</td>
</tr>
<tr>
<td> </td>
<td>Transfer isCID Energy</td>
<td>0.0 eV</td>
</tr>
<tr>
<td> </td>
<td>Transfer Multipole RF</td>
<td>400.0 Vpp</td>
</tr>
<tr>
<td> </td>
<td>Quadrupole Ion Energy</td>
<td>3.0 V</td>
</tr>
<tr>
<td colspan="3">MS/MS</td>
</tr>
<tr>
<td>Auto MS/MS</td>
<td>Precursor Ion List</td>
<td>include</td>
</tr>
<tr>
<td> </td>
<td>Mass Range</td>
<td>300.00–4000.00</td>
</tr>
<tr>
<td> </td>
<td>Width</td>
<td>± 0.5</td>
</tr>
<tr>
<td> </td>
<td>Cycle Time</td>
<td>3.0 s</td>
</tr>
<tr>
<td> </td>
<td>Exclude after</td>
<td>1 spectrum</td>
</tr>
<tr>
<td> </td>
<td>Release after</td>
<td>2.00 min</td>
</tr>
<tr>
<td> </td>
<td>Reconsider Precursor if Curr./Prev. Intens.</td>
<td>checked</td>
</tr>
<tr>
<td> </td>
<td>Reconsider Precursor if Curr./Prev. Intens.</td>
<td>3</td>
</tr>
<tr>
<td>MS/MS Preference</td>
<td>Preferred Mass Range (checked)</td>
<td>300.00–2200.00</td>
</tr>
<tr>
<td> </td>
<td>Width</td>
<td>± 0.5</td>
</tr>
<tr>
<td> </td>
<td>Charge State Range</td>
<td>1–6</td>
</tr>
<tr>
<td> </td>
<td>Exclude Singly</td>
<td>unchecked</td>
</tr>
<tr>
<td> </td>
<td>Exclude Unknown</td>
<td>unchecked</td>
</tr>
<tr>
<td>Acquisition (Precursor</td>
<td>MS Spectra Rate</td>
<td>2.00 Hz</td>
</tr>
<tr>
<td>Acquisition Control)</td>
<td>MS/MS Low (per 1000 sum.)</td>
<td>25000 cts, 2.00 Hz</td>
</tr>
<tr>
<td> </td>
<td>MS/MS Low</td>
<td>57175 cts.</td>
</tr>
<tr>
<td> </td>
<td>MS/MS High (per 1000 sum.)</td>
<td>250000 cts., 5.00 Hz</td>
</tr>
<tr>
<td> </td>
<td>MS/MS High</td>
<td>571750 cts.</td>
</tr>
<tr>
<td> </td>
<td>MS/MS Total Cycle Time Range</td>
<td>3 s</td>
</tr>
<tr>
<td> </td>
<td>MS/MS Absolute Threshold</td>
<td>1144 cts.</td>
</tr>
</tbody>
</table>
<table id="undtbl17">
<caption>CID collision energy list</caption>
<thead>
<tr>
<th>Type</th>
<th>Mass</th>
<th>Width</th>
<th>CE</th>
<th>Charge state</th>
</tr>
</thead>
<tbody>
<tr>
<td>Base</td>
<td>300</td>
<td>3</td>
<td>21</td>
<td>3</td>
</tr>
<tr>
<td>Base</td>
<td>300</td>
<td>3</td>
<td>26</td>
<td>2</td>
</tr>
<tr>
<td>Base</td>
<td>300</td>
<td>3</td>
<td>34</td>
<td>1</td>
</tr>
<tr>
<td>Base</td>
<td>500</td>
<td>4.8</td>
<td>28</td>
<td>3</td>
</tr>
<tr>
<td>Base</td>
<td>500</td>
<td>4.8</td>
<td>34</td>
<td>2</td>
</tr>
<tr>
<td>Base</td>
<td>500</td>
<td>4.8</td>
<td>39</td>
<td>1</td>
</tr>
<tr>
<td>Base</td>
<td>1,000</td>
<td>6</td>
<td>36</td>
<td>3</td>
</tr>
<tr>
<td>Base</td>
<td>1,000</td>
<td>6</td>
<td>40</td>
<td>2</td>
</tr>
<tr>
<td>Base</td>
<td>1,000</td>
<td>6</td>
<td>52</td>
<td>1</td>
</tr>
<tr>
<td>Base</td>
<td>2,000</td>
<td>9</td>
<td>40</td>
<td>3</td>
</tr>
<tr>
<td>Base</td>
<td>2,000</td>
<td>9</td>
<td>45</td>
<td>2</td>
</tr>
<tr>
<td>Base</td>
<td>2,000</td>
<td>9</td>
<td>55</td>
<td>1</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Dynamic light scattering (DLS)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>DLS measures hydrodynamic properties of purified RNA-free nucleocapsid protein to monitor oligomerization using a Wyatt DynaPro PlateReader-III. This assay is useful to characterize particles with a wide range of hydrodynamic radii from 1 nm to 1000 nm (<a href="#fig3">Figure 3</a>) (<a href="#bib16">Stetefeld et al., 2016</a>; <a href="#bib17">Su et al., 2018</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Dynamic light scattering assay for size characterization</div>
<p>Brownian motion of particles produces dynamic light scattering signals. These intensity fluctuations contain useful information and diffusion coefficients from autocorrelation analysis that can be converted to size.</p>
</figcaption>
</figure>
<ol>
<li>Thaw purified protein on ice.</li>
<li>Filter DLS buffer with a 0.22 μm filter.</li>
<li>Dilute protein solution into 2 mg/mL. Prepare 120 μL of sample for each protein, sufficient for technical triplicates. With Corning® Low Volume 384-well Clear Flat Bottom plate (Corning 3540), each well requires 35 μL of sample.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To adjust salt concentration, prepare buffer without salt for a simple calculation. First calculate the amount of buffer without salt needed for dilution, then dilute with DLS buffer to target for desired protein concentration.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Centrifuge sample for 5 min at 4°C using a tabletop centrifuge at top speed to remove aggregates.</div>
<ol start="4">
<li>Load 35 μL sample into each well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Before loading, check for presence of dust in wells. Cover plate with aluminum foil whenever possible when not loading to reduce likelihood of dirt getting inside.</div>
<div class="critical">
<span class="critical-title">Critical:</span> While loading, place pipette tip on the bottom of well and pipet slowly. Avoid generating bubbles. Gently shake plates if bubbles are trapped inside and remove bubbles by pipetting. It is important to get rid of bubbles before experiments.</div>
<ol start="5">
<li>Set up experiment for acquisition.
<ol type="a">
<li>Correlation function high cutoff set to 60 ms.</li>
<li>Acquire well images set to Yes.</li>
<li>DLS Acq time set to 5 s.</li>
<li>DLS Number Acq set to 10.</li>
<li>Auto-attenuation time limits set to 60 s.</li>
</ol>
</li>
<li>When analyzing data, access Regularization Graph and use triplicates to reach consensus.
<ol type="a">
<li>Remove peaks with less than 1% mass.</li>
<li>Align triplicates to check consistency for each major peak.</li>
<li>Determine statistics of measured radius across triplicates.</li>
<li>Repeat with newly prepared sample to check consistency across runs.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> DLS is extremely sensitive to dirt and bubbles. Use well images to check sample conditions and repeat when inconsistency arises. Please refer to Wyatt for additional guidance on instrument specific protocols.</div>
<h3 id="sec3.2">Fluorescence polarization (FP) assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h for step 7</div>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h for step 13</div>
<p>The FP assay uses fluorescently labeled RNA and titrates increasing concentrations of protein for affinity estimation based on polarization change of fluorescence label upon binding (<a href="#fig4">Figure 4</a>). Due to the high sensitivity of this technique, as low as 1 nM FITC-labeled RNA can be used, making it a useful probe for high affinity binding system (<a href="#bib8">Liu et al., 2017</a>; <a href="#bib17">Su et al., 2018</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Fluorescence polarization assay for nucleocapsid-RNA binding</div>
<p>(A) Principle for fluorescence polarization based RNA binding assay. Fluorescence labeled RNA molecules tumble fast and do not lead to preferential orientation. This results in depolarized emission that can be recorded with equal parallel and perpendicular fluorescence intensity. This corresponds to low FP values. After binding with protein, labeled RNA tumble slowly resulting in residual orientation that can be recorded with increased fluorescence polarization. By titrating protein concentration, the varying FP values will yield a binding curve that can be fit for a K<sub>D</sub>. (B) Fluorescence polarization assay in 96 well plate. First, a protein dilution series is prepared. Depending on binding mechanism which results in different curve shape, choose a suitable dilution factor to cover both unbound and bound side of baselines. A 2.5 fold dilution usually is a good starting point for a 12 point dilution series. After the dilution series are prepared, load protein first into wells and then labeled RNA. Cover solution containing labeled RNA with aluminum foil whenever possible to reduce quenching. A 10 min shake and incubation is recommended before reading with plate reader. Adapted from “Fluorescence Polarization Assay”, by <a href="http://Biorender.com">Biorender.com</a> (2020). Retrieved from <a href="https://app.biorender.com/biorender-templates">https://app.biorender.com/biorender-templates</a>.</p>
</figcaption>
</figure>
<ol start="7">
<li>Prepare protein titration series.
<ol type="a">
<li>Determine highest concentration point and fold of dilution. For nM affinity binding, a 2.5 fold dilution series with 12 concentration points starting with 10 μM as highest concentration and 0.42 nM as lowest concentration covers the entire binding curve. This step usually requires optimization.
<div class="note">
<span class="note-title">Note:</span> FITC labeled RNA at 1 nM concentration in this assay setup results in reasonable signal. Optimize RNA concentration with a known RNA binder so Z-factor is above 0.5. <math> <mrow> <mi>Z</mi> <mo>=</mo> <mn>1</mn> <mo>−</mo> <mfrac> <mrow> <mn>3</mn> <mrow> <mo>(</mo> <mi>σ</mi> <mrow> <mo>(</mo> <mi>p</mi> <mo>)</mo> </mrow> </mrow> <mo>+</mo> <mi>σ</mi> <mrow> <mo>(</mo> <mi>n</mi> <mo>)</mo> </mrow> <mo>)</mo> </mrow> <mrow> <mi>μ</mi> <mrow> <mo>(</mo> <mi>p</mi> <mo>)</mo> </mrow> <mo>−</mo> <mi>μ</mi> <mrow> <mo>(</mo> <mi>n</mi> <mo>)</mo> </mrow> </mrow> </mfrac> </mrow> </math>, <math> <mrow> <mi>μ</mi> </mrow> </math> and <math> <mrow> <mi>σ</mi> </mrow> </math>are means and standard deviations of positive (p) and negative (n) controls.</div>
</li>
<li>Prepare 120 μL of highest concentration point for the entire dilution series (each concentration point needs a volume of 25 μL and duplicates need 60 μL). Binding buffer is same as DLS buffer has 150 mM NaCl.</li>
<li>Pipette 90 μL binding buffer into each of 11 Eppendorf tubes. Take out 60 μL from highest concentration tube and mix well into the next concentration point. Repeat 11 times until entire dilution series are prepared.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Prepare labeled RNA stock.
<ol type="a">
<li>Calculate amount of RNA needed for each duplicate run. 700 μL at 2 nM (1 nM final concentration after mixing with protein) are prepared.</li>
<li>Cover tubes with aluminum foil to protect from light.</li>
</ol>
</li>
<li>Load protein dilution series into wells of a 96 well plate (Corning 3650). Each titration series is in one row. For example, duplicate runs of one experiment will take row A and B.</li>
<li>Load 25 μL of RNA into each well.</li>
<li>Load buffer and RNA only controls in row C.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Load multiple controls to estimate uncertainty associated with detection.</div>
<ol start="12">
<li>Read with Cytation 5 Cell Imaging Multi-mode Reader.
<ol type="a">
<li>Set excitation to 485 nm and emission to 528 nm with a bandpass of 20 nm.</li>
<li>Set read height to 8.5 mm.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Optical probe height has a considerable effect on the signal to noise ratio. Therefore, read height may need optimization for best performance.</div>
<ol start="13">
<li>Fit K<sub><i>D</i></sub> using Origin.
<ol type="a">
<li>Determine FP values using Gen5 Analysis software. More information on analysis using Gen5 Analysis can be found on the BioTek website.</li>
<li>Averages and standard deviations for titration points are calculated in Excel.</li>
<li>Binding curves are fit using Origin. See <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a> for details (<a href="#bib8">Liu et al., 2017</a>; <a href="#bib17">Su et al., 2018</a>). Alternative software such as GraphPad PRISM can also be for analysis.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each experiment containing duplicates are repeated at least three times. Statistics about K<sub>D</sub> values are reported across these technical repeats.</div>
<h3 id="sec3.3">Negative-stain electron microscopy</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min per grid (including loading and imaging) for step 22</div>
<p>This step describes preparation of nucleocapsid-RNA negative-stain sample for electron microscopy (EM) (<a href="#fig5">Figure 5</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Negative-stain electron microscopy</div>
<p>Freshly thawed proteins samples are loaded onto glow-discharged grids. After blotting with filter paper, grids are washed in water before stained with 2% UA solution. Air dried grids are then imaged with electron microscope. Adapted from “EM”, by <a href="http://Biorender.com">Biorender.com</a> (2020). Retrieved from <a href="https://app.biorender.com/biorender-templates">https://app.biorender.com/biorender-templates</a>.</p>
</figcaption>
</figure>
<ol start="14">
<li>Glow discharge carbon-coated copper grids (Ted Pella, 01840-F).
<ol type="a">
<li>Place grids onto a glass slide with carbon film side facing up.</li>
<li>Put the glass slide on sample stage of Leica EM ACE600.</li>
<li>Glow discharge for 45 s with following settings: Current (12 mA), Air (2.0E-1 mbar), Working Distance (50 mm), and Tilt (0°C).</li>
</ol>
</li>
<li>Apply 2 μL of sample at 1 mg/mL diluted in PBS to each grid. Wait for 1 min.</li>
<li>Blot the grid with filter paper for 3 s.</li>
<li>On a piece of parafilm, pipette three droplets of ddH<sub>2</sub>O and two droplets of 2% uranyl acetate (UA) solution. Each droplet has a volume of 20 μL.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Though the external radiological and chemical hazards of UA are relatively low, it is acutely toxic if swallowed or inhaled and may cause damage to organs through prolonged or repeated exposure. Review Safety Data Sheet and use with precautions.</div>
<p>Prepare fresh 2% UA from 4% UA stock solution and spin for 1 min at top speed using a tabletop centrifuge. Prepare 4% UA stock every 6 months.</p>
<ol start="18">
<li>Face the grid onto the ddH<sub>2</sub>O droplet to wash for 30 s and blot the grid for 3 s.</li>
<li>Repeat two more times with ddH<sub>2</sub>O.</li>
<li>Repeat two more times with 2% UA.</li>
<li>Airdry the grid before storing in a grid box.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure grids are completely dry before imaging.</div>
<ol start="22">
<li>Image grids with electron microscope. Each negative-stained grid is loaded onto the side-entry holder of JEOL JEM-1400plus Transmission Electron Microscope operating at 120 kV and recorded with an AMT XR111 high-speed 4K <b>×</b> 2K pixel phosphor-scintillated 12-bit CCD camera at 3,000 to 40,000 magnification.</li>
</ol>
<h3 id="sec3.4">Enzyme-linked immunosorbent assay (ELISA) for human plasma</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days for step 23</div>
<p>This section outlines how to perform an ELISA with patient plasma samples to screen for serological responses (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. ELISA overview</div>
<p>A 96-well plate is coated with N-protein fragments. The plate is subsequently blocked with protein and exposed to patient sera. A secondary antibody, an anti-human IgG conjugated to horseradish peroxidase (HRP) is then added to the plate for read-out. Created with <a href="http://Biorender.com">Biorender.com</a>.</p>
</figcaption>
</figure>
<ol start="23">
<li>Recombinant N proteins were coated on 96 well flat bottom immunosorbent plates at a concentration of 500 ng/mL in 100 μL coating buffer (PBS with 53% Na<sub>2</sub>CO<sub>3</sub> and 42% NaHCO<sub>3</sub>) at 4°C for 12–18 h.
<ol type="a">
<li>An additional plate was coated with a non-specific protein mixture to measure background binding of each sample.
<ol type="i">
<li>Non-specific protein mixture consisted of PBS with 5% fetal bovine serum (FBS).</li>
</ol>
</li>
</ol>
</li>
<li>Diluted plasma samples (1:100) were bound for 2 h.
<ol type="a">
<li>Samples were diluted in dilution buffer (PBS with 0.05% Tween-20 and 0.1% FBS) for a total volume of 200 μL.</li>
</ol>
</li>
<li>The 96 well plate was rinsed with 150 μL of wash buffer (PBS with 0.1% Tween-20).</li>
<li>Both 96 well plates were exposed to an anti-human IgG secondary antibody labeled with HRP (Invitrogen) diluted at 1:5000 in 20 mL dilution buffer (PBS with 0.05% tween-20 and 0.1% FBS).</li>
<li>Plates were read-out with R&amp;D substrate (Cat#DY999) as per <a href="https://resources.rndsystems.com/pdfs/datasheets/duoanc.pdf?v=20210908&amp;_ga=2.41750306.1448719793.1631140276-1192254123.1631140276">manufacturer’s recommendation on a Wallac spectrophotometer</a>. Incubation was between 18°C–22°C for 20 min while protecting from light. The absorbance was detected at 450 nm.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Patient plasma samples carry risk of pathogens, not limited to hepatitis B (HBV), hepatitis C (HCV), and human immunodeficiency virus (HIV). Therefore, proper PPE must be utilized when handling any samples (<a href="#bib18">Twitchell, 2003</a>).</div>
<h3 id="sec3.5">IFN-β promoter reporter assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This assays measures the ability of transfected proteins of interest to influence an interferon response induced by Sendai virus (SeV) (<a href="#fig7">Figure 7</a>) (<a href="#bib11">Messaoudi et al., 2015</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. IFN-β reporter assay</div>
<p>HEK-293T cells are transfected with a plasmid containing an IFN-β promoter-firefly luciferase reporter, a control plasmid containing a Renilla luciferase reporter, and an expression plasmid for one of the studied N-protein variants. Subsequently, cells are infected with Sendai virus (SeV), which stimulates the Interferon signaling pathway. N-protein variants inhibit the Interferon response pathway. Following transfection and infection, cells are lysed, and luciferase activity evaluated. Created with <a href="http://Biorender.com">Biorender.com</a>.</p>
</figcaption>
</figure>
<ol start="28">
<li>Seeding HEK-293T cells.
<ol type="a">
<li>HEK-293T cells were prepared from a stock solution of cells.
<ol type="i">
<li>Cells were split between 70%–90% confluency.</li>
</ol>
</li>
<li>HEK-293T cell media was aspirated from the flask.</li>
<li>Cells were exposed to EDTA-Trypsin for 5 min.</li>
<li>DMEM + 10% was added to the cell flask.</li>
<li>Trypsinized cells, split 5 <b>×</b> 10<sup>4</sup> into a 96 well plate.</li>
</ol>
</li>
<li>24 h later, HEK-293T cells (5 <b>×</b> 10<sup>4</sup>) were co-transfected using Lipofectamine 2000 with:
<ol type="a">
<li>25 ng of an IFN-β promoter-firefly luciferase reporter plasmid</li>
<li>25 ng of pRL-TK Renilla luciferase reporter plasmid</li>
<li>125, 12.5, and 1.25 ng of the indicated viral protein expression plasmid</li>
</ol>
</li>
<li>24 h after transfection, cells were mock-treated or treated with SeV (15 hemaglutination units/mL).</li>
<li>18 h post-treatment or post-infection, cells were lysed.</li>
<li>Cell lysates were analyzed for luciferase activity using a Dual-Luciferase reporter system.
<ol type="a">
<li>The Promega Dual-luciferase system utilizes enzymatic substrates for both Renilla luciferase and Firefly Luciferase.</li>
<li>Preparation of Buffers.
<ol type="i">
<li>Resuspend the Luciferase Assay Substrate in Luciferase Assay Buffer II.</li>
<li>Dilute the stock solution of Stop &amp; Glo® 50x to 1x with Stop &amp; Glo® Buffer.</li>
</ol>
</li>
<li>Read-out of 96-well plate.
<ol type="i">
<li>Add 20 μL of cell lysates prepared above.</li>
</ol>
</li>
<li>Assays were performed in triplicate.</li>
</ol>
</li>
<li>Viral protein expression was confirmed by Western blot analysis.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important that all reagents used in cell culture are kept sterile (<a href="#bib3">Coté, 1998</a>). Prior to readout with the luciferase, verify that the LAR II and the Stop &amp; Glo® Reagents have been warmed to 18°C–22°C. Luciferase activity is highly variable (<a href="#bib15">Repele and Manu, 2019</a>), and it is recommended that an auto-injector is used for injection of reagents.</div>
<h3 id="sec3.6">Differential HDX-MS structural characterization of RNA binding</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Mapping: 1 day; HDX-MS: 1 day per protein state</div>
<p>Hydrogen-deuterium exchange mass spectrometry (HDX-MS) provides sensitive, high-resolution measurement of solvent accessible surface area and changes in protein dynamics (<a href="#bib10">Masson et al., 2019</a>). We and others successfully used HDX-MS to characterize host-pathogen interactions (<a href="#bib13">Pei et al., 2021</a>; <a href="#bib14">Raghuvamsi et al., 2021</a>), monitor individual host and viral protein dynamics and interactions (<a href="#bib1">Batra et al., 2021</a>; <a href="#bib17">Su et al., 2018</a>; <a href="#bib4">Courouble et al., 2021</a>)<i>,</i> map antibody-antigen interactions (<a href="#bib7">Huang et al., 2018</a>; <a href="#bib2">Chen et al., 2019</a>), and determine binding sites of DNA/RNA with proteins (<a href="#bib9">Long et al., 2017</a>). This protocol outlines a method to pinpoint viral protein RNA binding.</p>
<ol start="34">
<li>Perform pre-HDX mapping.
<ol type="a">
<li>To ∼ 100 pmol of the protein of interest (2 μL), dilute 10:1 with H2O buffer (+ 18 μL).</li>
<li>Dilute 2:3 with a selected quench buffer.
<div class="note">
<span class="note-title">Note:</span> For more dilute protein samples, scale up the volumes if the ratios are retained.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Flash freeze quenched solutions for ease in the HDX-MS analysis.</div>
</li>
<li>Inject into the LC-MS apparatus for digestion, desalting, and LC-MS/MS analysis
<ol type="i">
<li>Digest and desalt the protein with 200 mL/min solvent A using valve position 2 (<a href="#fig2">Figure 2</a>) for 3 min (or 3 trap column volumes).</li>
<li>Analytically separate and elute the digested protein fragments by using valve position 1 (<a href="#fig2">Figure 2</a>) and an organic gradient (solvent A – solvent B; for an example gradient, see below).
<table id="undtbl18">
<caption>Example linear gradient for reversed-phase peptide elution</caption>
<thead>
<tr>
<th>Time (min)</th>
<th>% A</th>
<th>% B</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>95</td>
<td>5</td>
</tr>
<tr>
<td>0.5</td>
<td>95</td>
<td>5</td>
</tr>
<tr>
<td>11</td>
<td>50</td>
<td>50</td>
</tr>
<tr>
<td>13</td>
<td>0</td>
<td>100</td>
</tr>
<tr>
<td>13.5</td>
<td>0</td>
<td>100</td>
</tr>
<tr>
<td>15</td>
<td>95</td>
<td>5</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Include a high organic pulse (i.e., 100% solvent B) at the end of the LC gradient to decrease carryover from run to run.</div>
<div class="note">
<span class="note-title">Note:</span> The LC gradient may need to be adjusted for differences in protein hydrophobicity or to improve temporal separation of peptides/chromatogram congestion.</div>
</li>
</ol>
</li>
<li>Process the LC-MS/MS results to optimize digestion efficiency using Byonic and Byologic (see <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a> below for details) to compare:
<ol type="i">
<li>Protein coverage</li>
<li>Peptide length</li>
<li>Peptide abundance</li>
<li>Count of identified peptides</li>
<li>Redundancy
<div class="note">
<span class="note-title">Note:</span> Peptides with a 5–20 amino acid length are recommended for HDX-MS. Redundancy can be measured numerically as a function of the average number of overlapping peptides across all residues and tabulated in the suggested comparison. We also recommend visual inspection of redundancy in the context of protein coverage.</div>
</li>
</ol>
</li>
<li>Repeat with a range of quench conditions and protease columns/column orientations (<a href="#bib5">Cravello et al., 2003</a>) (see <a href="#fig8">Figure 8</a>A for an example of digestion optimization logic). Optimization of temperature, quench incubation time, addition of other chaotropic reagents can be done.</li>
<li>Repeat the mapping experiment with the chosen conditions, but with added RNA to confirm the RNA is not affecting the digestion.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> As long as mapping conditions (i.e., MS settings and protease choice) are replicated during the HDX-MS experiment, HDX-MS can be completed at any time assuming sample integrity. Proteins can typically be stored for months at −80 °C.</div>
<div class="note">
<span class="note-title">Note:</span> If the protein contains disulfide bonds, a reducing agent should be added to the quench. Reduction of disulfide bonds may necessitate incubation at elevated temperature for 1–3 minutes. Incubation, especially at elevated temperatures, increases H/D back-exchange; care should be taken to minimize back exchange.</div>
<div class="note">
<span class="note-title">Note:</span> Denaturation using other chaotropic agents such as guanidine HCl may be used to better denature the protein. The efficacy of such denaturants on improving protein digestion is protein-dependent.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Screening of mapping conditions defines the feasibility and resolution of the HDX-MS experiment. Carefully optimize the mapping results before proceeding with HDX-MS. For an example of mapping logic, see table below and <a href="#fig8">Figure 8</a>A.</div>
<table id="undtbl19">
<caption>Example screening of quench and digestion conditions</caption>
<thead>
<tr>
<th> </th>
<th>Denaturant (starting conc.)</th>
<th>Protease column (s)</th>
<th>Coverage (%)</th>
<th>Average length (aa)</th>
<th>ID’d peptide count</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>3 M Urea</td>
<td>Pepsin</td>
<td>100%</td>
<td>28.2 (± 0.8)</td>
<td>341 (± 7)</td>
</tr>
<tr>
<td>2</td>
<td>3 M Urea</td>
<td>FXIII</td>
<td>100%</td>
<td>20.86 (± 0.04)</td>
<td>577 (± 3)</td>
</tr>
<tr>
<td>3</td>
<td>3 M Urea</td>
<td>FXIII-Pepsin</td>
<td>100%</td>
<td>18.3 (± 0.4)</td>
<td>550 (± 20)</td>
</tr>
<tr>
<td>4</td>
<td>3 M Urea</td>
<td>Pepsin-FXIII</td>
<td>100%</td>
<td>20.18 (± 0.04)</td>
<td>556 (± 4)</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
</ol>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1122-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. HDX expected outcome</div>
<p>(A) (i) Representative MS<sup>2</sup> spectrum of positively identified peptide from the mapping experiment, interpreted from b- and y- ion fragments that provide sequence information. Blue, N-terminal-facing notches in the peptide sequence indicate identification of b-ions; red, C-terminal-facing notches in the peptide sequence indicate identification of y-ions. A screenshot of data from Byologic software. Peptide map spanning SARS-CoV-2 N protein NTD-LKR-CTD showing coverage from digestion by pepsin (ii, black) and pepsin-FXIII columns (iii, orange). iv) Comparison of effect of protease selection on peptide length. In general, pepsin (black) generates longer peptides than pepsin-FXIII columns in series (red).</p>
<p>(B) (i) LC extracted ion chromatograms, indicated by red triangle bookends, for a peptide submitted to incubation in D<sub>2</sub>O for 0 (nondeuterated control), 10, 30, 300, 3600 s (top to bottom), demonstrating consistent elution times with time. The mass spectra (ii) associated with these peaks demonstrate increased deuteration as a function of time. (iii) Graphing the centroid shift allows for the comparison of unbound (black) and the RNA-bound protein (red), indicating protection (left) or no change (right) for peptide generated by pepsin-FXIII digestion in the bound state. (iv) A Woods’ plot includes cumulative differences from all time points for the peptides to be viewed in one image, with statistics (p &lt; 0.1, ∗; p &lt; 0.05, ∗∗; p &lt; 0.01, ∗∗∗), enabling an overall/global view/overview of changes in HDX across the entire protein sequence. Changes in deuteration upon RNA-binding mapped on a known structure of N protein NTD (PDB: 6M3M) (v). Meaningful decreases shown in blue. Raw data screenshots from PMI Byos and HDExaminer softwares. Woods’ plot and N protein NTD structure (iv and v) from <a href="#bib20">Wu et al. (2021)</a>.</p>
</figcaption>
</figure>
<ol start="35">
<li>Equilibrate unbound and bound stock solutions. Protein/ligand concentrations and buffer conditions should be consistent across all HDX samples. A higher concentration stock should be generated for both bound and unbound states such that 2 μL of stock diluted with 18 μL D2O buffer yields the same final protein concentration.
<ol type="a">
<li>Prepare the unbound protein stock solution (∼25 μM).</li>
<li>Prepare the bound protein stock solution (∼25 μM) with 1:1 RNA.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> The amount of RNA added may be increased to maximize the percentage of protein bound to RNA.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare and acquire the non-deuterated control and all HDX time points in at least duplicate.</div>
<ol start="36">
<li>Prepare the non-deuterated control (i.e., 0 s).
<ol type="a">
<li>To 2 μL of the unbound protein stock, add 18 μL H<sub>2</sub>O buffer.</li>
<li>Stop HDX with addition of 30 μL quench solution.</li>
<li>Flash freeze, and store solution at −80°C.</li>
</ol>
</li>
</ol>
<ol start="37">
<li>Prepare HDX labeled protein (<a href="#fig2">Figure 2</a>A).
<ol type="a">
<li>To 2 μL of unbound protein stock on ice, add 18 μL D<sub>2</sub>O buffer (to maximize HDX at 90% D).</li>
<li>Incubate at range of times logarithmic: 4 time point: 10, 30, 300, 3600 s.
<div class="optional">
<span class="optional-title">Optional:</span> More time points may be added (e.g., 60, 900, 14400 s) to define better HDX kinetics.</div>
</li>
<li>Quench HDX with addition of 30 μL quench solution for final volume of 50 μL (same as sample loop).</li>
<li>Flash freeze in liquid nitrogen.</li>
</ol>
</li>
</ol>
<ol start="38">
<li>Repeat all with the bound protein stock solution.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All times should be controlled! Deviations in timing can lead to differential back-exchange extents and poor precision. Deviations in observed LC elution times result in more manual adjustment of extraction window during processing.</div>
<div class="critical">
<span class="critical-title">Critical:</span> HDX-MS is sensitive to fluctuations in temperature, pH, and ionic strength. All buffer conditions should be controlled, and all reagents should be pre-equilibrated at the temperature they will be used to ensure no unwanted deviations across the experiment.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Inject immediately if protein aggregates upon flash freeze.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store flash frozen HDX samples at −80°C for up to 1 month.</div>
<ol start="39">
<li>Perform LC-MS analysis of the HDX labeled protein states.
<ol type="a">
<li>Thaw flash frozen HDX samples at 37°C for 30 s just prior to injection (consistency is essential).</li>
<li>Inject the protein and analyze as noted for mapping above for MS analysis (without MS/MS).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Deuteriums are prone to scrambling, especially with collision induced dissociation. Other dissociation techniques can be used for fragmentation without scrambling (<a href="#bib6">Ferguson et al., 2007</a>).</div>
<ol start="40">
<li>To reduce carryover, wash sample loop with at least 3 volumes (i.e., 150 μL for a 50 μL loop) with water/0.1% formic acid, acetonitrile + 0.1% formic acid, and again water + 0.1% formic acid between injections.</li>
<li>To monitor carryover extents, run a blank between samples.
<ol type="a">
<li>H<sub>2</sub>O blank: 20 μL H<sub>2</sub>O buffer + 30 μL quench.</li>
<li>D<sub>2</sub>O blank: 2 μL H<sub>2</sub>O buffer + 18 μL D<sub>2</sub>O buffer + 30 μL quench.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Negative-stain EM is particularly useful to visualize oligomeric species with higher resolution of the electron microscope compared to the light microscope. For nucleocapsids bound to RNA, it is common to observe loose coils where oligomers of nucleocapsids are linked with copurified RNA as well as more condensed form helical assembly, depending on specific constructs and solvent environment. Due to inherent conformational flexibility, oligomers of nucleocapsid protein can adopt different forms, such as rings and tubes of different lengths. In the case of SARS-CoV-2 N, upon RNA binding and dilution into 150 mM NaCl, we observed additional species of N-RNA which phase separates into liquid droplets of μm size. These droplets are a separate phase from solvent and lack internal details. They might be relevant to RNA packaging.</p>
<p>HDX-MS mapping of the protein-RNA binding system with appropriate selection of protease and quench conditions generates a set of peptides spanning the protein (<a href="#fig8">Figure 8</a>A). Positively identified peptides are selected prior to exchange based on high quality MS2 spectra whereby fragment ions essentially sequence the peptide (<a href="#fig8">Figure 8</a>Ai). The efficacy of mapping may be checked by peptide coverage maps to afford quick comparison (e.g., pepsin vs. FXIII-pepsin, <a href="#fig8">Figures 8</a>Aii and 8Aiii, respectively). Because 5–20 amino-acid peptides are optimum for maximizing HDX-MS resolution, peptide length as a metric should be compared across mapping conditions (<a href="#fig8">Figure 8</a>Aiv). Taken together, coverage and peptide length are two metrics for optimization of HDX-MS conditions.</p>
<p>Differential HDX-MS of a two-state system measures meaningful differences for peptides containing residues involved in RNA binding (<a href="#fig8">Figure 8</a>B). Timing of the HDX-apparatus for repeatability of LC conditions should generate chromatograms with consistent elution times. Specifically, the extracted ion chromatogram (XIC/EIC) of an identified peptide should show maintenance of elution time, shape, and relative abundance across samples undergoing differential HDX (<a href="#fig8">Figure 8</a>Bi). If the exchange kinetics are appropriately measured in the HDX experiment, the extracted mass spectrum for the LC peak will demonstrate an increase in deuteration, whereby the centroid will shift with time (<a href="#fig8">Figure 8</a>Bii). Kinetic plots, representing peptide deuteration as a function of incubation time will show changes in deuteration for the protein <i>bound state</i> relative to the <i>unbound state</i> for those peptides that include RNA binding sites (decreased HDX) and/or changes in backbone flexibility/hydrogen bonding (increased and/or decreased HDX) (<a href="#fig8">Figure 8</a>Biii, left). Conversely, there will be no meaningful difference in HDX for the two states for those peptides covering regions of the protein not affected by RNA binding (<a href="#fig8">Figure 8</a>Biii, right). The cumulative difference (defined in <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a> section) for all kinetics plots are plotted across the protein sequence, allowing the user to visualize dynamics for the entire system (<a href="#fig8">Figure 8</a>Biv). Finally, meaningful differences in deuteration are represented on known structures to draw conclusions about differential H-D kinetics in the context of structure (<a href="#fig8">Figure 8</a>Bv).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">Curve fitting of fluorescence polarization binding data</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Titrations curves of varying protein receptor concentration were fitted using the following user defined equation in Origin.</p>
<p class="math-ml"><math> <mrow> <mi>F</mi> <mo>=</mo> <mrow> <mo>(</mo> <mi>Fb</mi> <mo>−</mo> <mi>Ff</mi> <mo>)</mo> </mrow> <mo>×</mo> <mfrac> <mrow> <mrow> <mo>(</mo> <mrow> <mi>KD</mi> <mo>+</mo> <mi>L</mi> <mo>+</mo> <mi>R</mi> </mrow> <mo>)</mo> </mrow> <mo>−</mo> <msqrt> <mrow> <msup> <mrow> <mo>(</mo> <mi>KD</mi> <mo>+</mo> <mi>L</mi> <mo>+</mo> <mi>R</mi> <mo>)</mo> </mrow> <mn>2</mn> </msup> <mo>−</mo> <mn>4</mn> <mo>×</mo> <mi>R</mi> <mo>×</mo> <mi>L</mi> </mrow> </msqrt> </mrow> <mrow> <mn>2</mn> <mo>×</mo> <mi>R</mi> </mrow> </mfrac> </mrow> </math></p>
<p>Fb is fluorescence polarization value when FITC-ssRNA is saturated with protein receptor and Ff is fluorescence polarization value when FITC-ssRNA is alone. KD is the dissociate constant. L and R are total RNA and protein concentration, respectively.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Origin was used for curve fitting. However, any other software, e.g. GraphPad Prism, that is capable of nonlinear curve fitting can be used.</div>
<h3 id="sec5.2">Processing pre-HDX mapping results using Protein Metrics software</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Several LC-MS/MS peptide identification processing softwares are available (e.g., Mascot, PEAKs, Proteome Discoverer), and any that identifies peptides from MS/MS analysis and reports the LC retention time for the peptide can be used.</div>
<ol>
<li>Process each LC-MS/MS mapping datafile using Byonic™.
<ol type="a">
<li>Import raw data and FASTA sequence files.</li>
<li>Search data using <i>non specific</i> [<i>sic</i>] <i>(slowest) digestion specificity,</i> as is appropriate for nonspecific digestion of proteins with Pepsin/FXIII proteases.</li>
<li>Check newly-generated Byonic file for protein coverage by using peptides with good MS<sup>2</sup> fragmentation.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Import Byonic™ results into Byologic®.
<ol type="a">
<li>Use the <i>Byonic Import</i> function to search directly Byonic data for LC retention window information.</li>
<li>Validate MS2 spectra for good fragmentation and appropriate assignment to LC peaks.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If data quality or coverage are poor, do not move on; consider evaluating additional mapping conditions.</div>
<ol start="3">
<li>Export parameters for comparison of the quench and digestion conditions (<a href="#fig8">Figure 8</a>A).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Consider, given that peptides containing 5–20 amino acid residues are most useful for HDX, evaluating and selecting optimum quench and digestions conditions.</div>
<ol start="4">
<li>Export a csv file for all peptides with the chosen quench and digestion conditions.</li>
<li>Within the csv file, remove all columns except sequence, XIC apex (or retention time), charge, and score. This is the working mapping file for processing HDX results.</li>
</ol>
<h3 id="sec5.3">Processing HDX-MS results using HDExaminer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Several HDX processing software packages are available (e.g., HDX Workbench, HX Express) and can be used.</div>
<ol start="6">
<li>Start new project from the protein FASTA sequence.</li>
<li>Import the peptide list from the mapping results created in Byologic in step 5, previous section.</li>
<li>Import the raw LC-MS datafiles; adjust incubation times and percent deuterium to correspond with each data file.</li>
<li>For each peptide, adjust LC extraction windows to cover the same retention time across all timepoints. Watch for contaminating ions in the <i>m/z</i> window, as they perturb the fitting for <i>m/z</i> centroid calculation (and resulting deuteration values).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Care should be taken to ensure the same extraction window for each peptide for all time points and all states (i.e., unbound/bound).</div>
<div class="note">
<span class="note-title">Note:</span> Some peptides should be deleted from the analysis pool on basis of quality of extracted ion chromatogram and/or MS of isotopologues. Export peptide pool results.</div>
<h3 id="sec5.4">Woods’ plot generation and statistical analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="10">
<li>Calculate cumulative differences in HDX (<a href="#fig3">Figure 3</a>A) as the sum of all differences in D at all time points:</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>Summed</mi> <mspace></mspace> <mo>Δ</mo> <mi>HX</mi> <mo>=</mo> <mspace></mspace> <mo>∑</mo> <mrow> <mo>(</mo> <msub> <mi>D</mi> <mrow> <mspace></mspace> <mi>t</mi> <mo>,</mo> <mspace></mspace> <mi>bound</mi> </mrow> </msub> <mo>)</mo> </mrow> <mo>−</mo> <mo>∑</mo> <mrow> <mo>(</mo> <msub> <mi>D</mi> <mrow> <mspace></mspace> <mi>t</mi> <mo>,</mo> <mspace></mspace> <mi>unbound</mi> </mrow> </msub> <mo>)</mo> </mrow> </mrow> </math></p>
<p>where D<sub>t</sub> is the deuterium uptake in Da at each time point for the respective state (unbound or bound).</p>
<div class="note">
<span class="note-title">Note:</span> <i>Positive</i> summed <i>ΔHX</i> values indicate the HDX for a given peptide is <i>higher</i> in the bound state than the unbound (i.e., binding increased solvent accessibility or decreased amide hydrogen bonding). <i>Negative</i> summed <i>ΔHX</i> values indicate the HDX for a given peptide was <i>lower</i> in the bound state than the unbound state (i.e., binding decreased solvent accessibility or increased amide hydrogen bonding).</div>
<ol start="11">
<li>Calculate the error of the HDX measurement for each differential peptide result as 3 times the propagated error from the standard deviations at each time point and for both states.</li>
</ol>
<ol start="12">
<li>Calculate global HDX significance limits.
<ol type="a">
<li>Calculate the pooled standard deviation for the entire dataset for each state:
<p class="math-ml"><math> <mrow> <msub> <mi>s</mi> <mi>pooled</mi> </msub> <mo>=</mo> <mspace></mspace> <msqrt> <mfrac> <mrow> <mrow> <mo>∑</mo> <msup> <mi>s</mi> <mn>2</mn> </msup> </mrow> </mrow> <mi>N</mi> </mfrac> </msqrt> </mrow> </math></p>
where <i>s</i> is all standard deviations across the dataset and N is the number of data points (the number of timepoints ∗ the number of peptides).
<div class="note">
<span class="note-title">Note:</span> Peptides for which standard deviations were not calculated at any time point should be excluded from the dataset.</div>
</li>
<li>From the pooled standard deviations of both states, calculate the standard error of the mean (SEM):
<p class="math-ml"><math> <mrow> <mi>SE</mi> <msub> <mi>M</mi> <mrow> <mo>Δ</mo> <mover> <mi>HX</mi> <mo>¯</mo> </mover> </mrow> </msub> <mo>=</mo> <mspace></mspace> <msqrt> <mrow> <mfrac> <msubsup> <mi>s</mi> <mrow> <mi>p</mi> <mo>,</mo> <mi>unbound</mi> </mrow> <mn>2</mn> </msubsup> <msub> <mi>n</mi> <mi>unbound</mi> </msub> </mfrac> <mo>+</mo> <mfrac> <msubsup> <mi>s</mi> <mrow> <mi>p</mi> <mo>,</mo> <mi>bound</mi> </mrow> <mn>2</mn> </msubsup> <msub> <mi>n</mi> <mi>bound</mi> </msub> </mfrac> </mrow> </msqrt> </mrow> </math></p>
where n is the number of experimental technical replicates (in this case 2 for both).</li>
<li>Using the SEM and t-values for a two-tailed Student’s t-distribution, p &lt; 0.1, 0.05, and 0.01, and 2 degrees of freedom (for duplicates of two states, 2 + 2 - 2), calculate the confidence interval or global <i>ΔHX</i> significance limit(s):
<p class="math-ml"><math> <mrow> <mo>±</mo> <mi>C</mi> <msub> <mi>I</mi> <mrow> <mo>Δ</mo> <mover> <mrow> <mi>H</mi> <mi>X</mi> </mrow> <mo>¯</mo> </mover> </mrow> </msub> <mo>=</mo> <mi>t</mi> <mspace></mspace> <mo>×</mo> <mi>S</mi> <mi>E</mi> <msub> <mi>M</mi> <mrow> <mo>Δ</mo> <mover> <mrow> <mi>H</mi> <mi>X</mi> </mrow> <mo>¯</mo> </mover> </mrow> </msub> </mrow> </math></p>
<br> Peptide HDX differences are only considered statistically significant if the propagated error does not cross 0 and the HDX differences exceed the global significance limit(s).</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Generate HDX summary and HDX data tables.</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>HDX-MS modifies amide hydrogens on the entire protein surface as a function of solvent accessibility to probe changes in backbone protein dynamics and hydrogen bonding occurring the in higher order structure. Owing to the risk of back-exchange necessitating expedited sample handling and special considerations for low pH digestion, regions of the protein in critical regions of interest may not digest well, and the user may wish to probe solvent accessibility of specific protein side chains.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Broad peaks in size exclusion chromatography (<a href="#sec1.4">purification of nucleocapsid protein</a>, step 27).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Since only one affinity tag column is used before size exclusion chromatography, contaminants might be a problem to cause broad peaks in size exclusion chromatography. Due to the RNA binding and aggregation-prone nature of N, especially for longer constructs, it is crucial to maintain high salt concentration in lysis and Ni column steps. This helps remove nonspecific binding. In addition, when choosing molecular weight cutoff of concentrators, it is better to use concentrators of highest molecular weight cutoff to improve purity. In addition, keep everything on ice and purify as fast as possible to minimize degradation.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Introduction of bubbles while loading (<a href="#sec3.1">dynamic light scattering</a>, step 4).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Avoiding generating bubbles in the sample in the first place. This can be done by sample centrifugation. While loading, touch the bottom of the well, then slowly release sample inside while pulling up pipette tip. Leave the last bit of sample and do not pipette everything out. Practice loading with buffer and BSA protein sample first. If bubbles are introduced into wells, gently tap the plate to let bubbles float to surface and remove bubbles with pipette tip.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Mapping digestion shows missing sections of protein (i.e., low coverage), poor peptide redundancy, low peptide signal (<a href="#sec3.6">differential HDX-MS structural characterization of RNA binding</a>, step 34).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>These are indications that mapping requires further optimization before starting HDX experiments. One or both of the following may be required: (1) adjust quench solution components (e.g., chaotropic reagent, salt concentration, reducing agent). (2) Incorporate additional protease on-line with original protease or use alternative protease or preparation.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>No meaningful differences in H-D are observed between two protein states (<a href="#sec3.6">differential HDX-MS structural characterization of RNA binding</a>, step 37).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>HDX is reporting on all states of the target protein in solution. If complex formation is incomplete, differences in HDX are diluted. To ensure more protein binding, the user can increase the relative concentration of the RNA. It may be possible that the time range of the experiment is not the same as the protein dynamics/kinetics of HDX. When both protein states are highly exchanged across all timepoints, decrease the temperature for HDX (e.g., perform on ice); for cases where both protein states undergo slow exchange with time, increase the HDX temperature. It is also possible that back exchange is occurring during separation and measurement. Ensure that minimal time is involved for measurement and that sample is kept at low T. Check the pH of the final solution after quench to ensure a value corresponding nearly to the kinetic minimum at pH 2.6.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Peptide XIC elution times drift, making HDX-MS data processing more time consuming (<a href="#sec3.6">differential HDX-MS structural characterization of RNA binding</a>, step 39).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Consistent timing of sample injection, start of LC gradient, and operation of MS file acquisition are critical to maintain a consistent measurement of XIC elution time. We recommend the use of a handheld timer to set alarms reminding the user to operate each piece of equipment in manual mode. Given that peptide elution times are also a function temperature; ensure constant temperature in the HDX-MS apparatus by maintaining an ice water slush for the duration of the experiment.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Batra, J., Mori, H., Small, G.I., Anantpadma, M., Shtanko, O., Mishra, N., Zhang, M., Liu, D., Williams, C.G., Biedenkopf, N., et al. (2021). Non-canonical proline-tyrosine interactions with multiple host proteins regulate Ebola virus infection. EMBO J. <i>40</i>, e105658. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref1">View at publisher</a></p>
<p id="bib2">Chen, G., Karauzum, H., Long, H., Carranza, D., Holtsberg, F.W., Howell, K.A., Abaandou, L., Zhang, B., Jarvik, N., Ye, W., et al. (2019). Potent neutralization of staphylococcal enterotoxin B in vivo by antibodies that block binding to the T-cell receptor. J. Mol. Biol. <i>431</i>, 4354-4367. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref2">View at publisher</a></p>
<p id="bib3">Coté, R.J. (1998). Aseptic technique for cell culture. Curr. Protoc. Cell Biol. <i>00</i>, 1.3.1-1.3.10. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref3">View at publisher</a></p>
<p id="bib4">Courouble, V.V., Dey, S.K., Yadav, R., Timm, J., Harrison, J.J.E.K., Ruiz, F.X., Arnold, E., and Griffin, P.R. (2021). Revealing the structural plasticity of SARS-CoV-2 nsp7 and nsp8 using structural proteomics. J. Am. Soc. Mass Spectrom. <i>32</i>, 1618-1630. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref4">View at publisher</a></p>
<p id="bib5">Cravello, L., Lascoux, D., and Forest, E. (2003). Use of different proteases working in acidic conditions to improve sequence coverage and resolution in hydrogen/deuterium exchange of large proteins. Rapid Commun. Mass. Spectrom. <i>17</i>, 2387-2393. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref5">View at publisher</a></p>
<p id="bib6">Ferguson, P.L., Pan, J., Wilson, D.J., Dempsey, B., Lajoie, G., Shilton, B., and Konermann, L. (2007). Hydrogen/deuterium scrambling during quadrupole time-of-flight MS/MS analysis of a zinc-binding protein domain. Anal. Chem. <i>79</i>, 153-160. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref6">View at publisher</a></p>
<p id="bib7">Huang, R.Y.C., Krystek, S.R., Felix, N., Graziano, R.F., Srinivasan, M., Pashine, A., and Chen, G. (2018). Hydrogen/deuterium exchange mass spectrometry and computational modeling reveal a discontinuous epitope of an antibody/TL1A Interaction. mAbs <i>10</i>, 95-103. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref7">View at publisher</a></p>
<p id="bib8">Liu, G., Nash, P.J., Johnson, B., Pietzsch, C., Ilagan, M.X.G., Bukreyev, A., Basler, C.F., Bowlin, T.L., Moir, D.T., Leung, D.W., and Amarasinghe, G.K. (2017). A sensitive in vitro high-throughput screen to identify pan-filoviral replication inhibitors targeting the VP35–NP interface. ACS Infect. Dis. <i>3</i>, 190-198. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref8">View at publisher</a></p>
<p id="bib9">Long, Y., Bolanos, B., Gong, L., Liu, W., Goodrich, K.J., Yang, X., Chen, S., Gooding, A.R., Maegley, K.A., Gajiwala, K.S., et al. (2017). Conserved RNA-binding specificity of polycomb repressive complex 2 is achieved by dispersed amino acid patches in EZH2. eLife <i>6</i>, e31558. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref9">View at publisher</a></p>
<p id="bib10">Masson, G.R., Burke, J.E., Ahn, N.G., Anand, G.S., Borchers, C., Brier, S., Bou-Assaf, G.M., Engen, J.R., Englander, S.W., Faber, J., et al. (2019). Recommendations for performing, interpreting and reporting hydrogen deuterium exchange mass spectrometry (HDX-MS) experiments. Nat. Methods <i>16</i>, 595-602. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref10">View at publisher</a></p>
<p id="bib11">Messaoudi, I., Amarasinghe, G.K., and Basler, C.F. (2015). Filovirus pathogenesis and immune evasion: insights from Ebola virus and Marburg virus. Nat. Rev. Microbiol. <i>13</i>, 663-676. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref11">View at publisher</a></p>
<p id="bib12">Narang, D., Lento, C., and Wilson, J.D. (2020). HDX-MS: an analytical tool to capture protein motion in action. Biomedicines <i>8</i>, 224. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref12">View at publisher</a></p>
<p id="bib13">Pei, J., Wagner, N.D., Zou, A.J., Chatterjee, S., Borek, D., Cole, A.R., Kim, P.J., Basler, C.F., Otwinowski, Z., Gross, M.L., et al. (2021). Structural basis for IFN antagonism by human respiratory syncytial virus nonstructural protein 2. Proc. Natl. Acad. Sci. U S A <i>118</i>. e2020587118. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref13">View at publisher</a></p>
<p id="bib14">Raghuvamsi, P.V., Tulsian, N.K., Samsudin, F., Qian, X., Purushotorman, K., Yue, G., Kozma, M.M., Hwa, W.Y., Lescar, J., Bond, P.J., et al. (2021). SARS-CoV-2 S protein:ACE2 interaction reveals novel allosteric targets. eLife <i>10</i>, e63646. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref14">View at publisher</a></p>
<p id="bib15">Repele, A. and Manu. (2019). Robust normalization of luciferase reporter data. Methods Protoc. <i>2</i>, 62. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref15">View at publisher</a></p>
<p id="bib16">Stetefeld, J., Mckenna, S.A., and Patel, T.R. (2016). Dynamic light scattering: a practical guide and applications in biomedical sciences. Biophys. Rev. <i>8</i>, 409-427. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref16">View at publisher</a></p>
<p id="bib17">Su, Z., Wu, C., Shi, L., Luthra, P., Pintilie, G.D., Johnson, B., Porter, J.R., Ge, P., Chen, M., Liu, G., et al. (2018). Electron cryo-microscopy structure of ebola virus nucleoprotein reveals a mechanism for nucleocapsid-like assembly. Cell <i>172</i>, 966-978.e12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref17">View at publisher</a></p>
<p id="bib18">Twitchell, K.T. (2003). Bloodborne pathogens. What you need to know--Part I. AAOHN J. <i>51</i>, 38-45, quiz 46-7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref18">View at publisher</a></p>
<p id="bib19">Wang, L., Pan, H., and Smith, D.L. (2002). Hydrogen exchange-mass spectrometry: optimization of digestion conditions∗. Mol. Cell Proteomics <i>1</i>, 132-138. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref19">View at publisher</a></p>
<p id="bib20">Wu, C., Qavi, A.J., Hachim, A., Kavian, N., Cole, A.R., Moyle, A.B., Wagner, N.D., Sweeney-Gibbons, J., Rohrs, H.W., Gross, M.L., et al. (2021). Characterization of SARS-CoV-2 nucleocapsid protein reveals multiple functional consequences of the C-terminal domain. iScience <i>24</i>, 102681. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00612-2/sref20">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. Gaya Amarasinghe (<a href="mailto:gamarasinghe@wustl.edu">gamarasinghe@wustl.edu</a>).</p>
<h4>Materials availability</h4>
<p>Plasmids in this study are available with a completed Materials Transfer Agreement Request for these reagents by submitting to Dr. Gaya Amarasinghe (<a href="mailto:gamarasinghe@wustl.edu">gamarasinghe@wustl.edu</a>).</p>
<h4>Data and code availability</h4>
<p>The protocol includes all data sets generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>We thank members of the Amarasinghe, Leung, Gross, Farnsworth, Valkenburg, Peiris, and Basler groups for critical feedback and support. We thank Bruker for mass spectrometry technical and instrument support, and Protein Metrics (R42GM121133 to H.W.R.) and Sierra Analytics for providing proteomic and HDX data analysis software. Research was supported by Fast Grant #2161 (Emergent Ventures) to G.K.A., by Fast Grant Fast Grants Award #2158 to C.F.B, and <a href="https://doi.org/10.13039/100000002">NIH</a> grants (P01AI120943, R01AI123926 to G.K.A. and C.F.B; R01AI107056 to D.W.L.; P41GM103422 and R24GM136766 to M.L.G.; R01AI143292 and R01AI148663 to C.F.B.). S.A.V. was supported by COVID190115 and COVID190126 Health and Medical Research Fund, <a href="https://doi.org/10.13039/501100005407">Food and Health Bureau</a>, Hong Kong and <a href="https://doi.org/10.13039/100000002">NIH</a>/NIAID CEIRS contract HHSN272201400006C. A.J.Q. is supported by a <a href="https://doi.org/10.13039/100000002">NIH</a> T32 training grant (T32CA009547). Negative-stain EM was performed through the use of Washington University Center for Cellular Imaging (WUCCI) supported by <a href="https://doi.org/10.13039/100013395">Washington University School of Medicine</a>, The Children’s Discovery Institute of <a href="https://doi.org/10.13039/100007268">Washington University</a> and St. Louis Children’s Hospital (CDI-CORE-2015-505 and CDI-CORE-2019-813), and the <a href="https://doi.org/10.13039/100007338">Foundation for Barnes-Jewish Hospital</a> (3770 and 4642).</p>
<h3>Author contributions</h3>
<p>C.W., G.K.A., and D.W.L. conceived the overall project. M.L.G. provided the MS facilities for HDX. All authors were integral to the design and execution of the study. C.W., A.J.Q., A.B.M., N.D.W., G.K.A., and D.W.L. wrote the initial draft with significant input and editing from all authors.</p>
<h3>Declaration of interests</h3>
<p>M.L.G. is an unpaid member of the scientific advisory board of Protein Metrics and GenNext Technologies, two companies commercializing protein footprinting software and tools.</p>
</section>
</article>