<article>
<section>
<h1 id="header">Protocol to select conformation-specific intracellular antibodies for targeted protein degradation in an engineered cell line</h1>
<p><time datetime="2021-01-08">Published: January 8, 2021</time></p>
<p>Nicolas Bery,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Gilles Favre,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Aurélien Olichon<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Centre de Recherche en Cancérologie de Toulouse (CRCT), UMR1037 INSERM - Université Toulouse III - ERL5294 CNRS, Toulouse, 31100, France</p>
<p id="aff2"><sup>2</sup>Département de Biologie, Institut Claudius Regaud, Toulouse, 31100, France</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:nicolas.bery@inserm.fr">nicolas.bery@inserm.fr</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:aurelien.olichon@inserm.fr">aurelien.olichon@inserm.fr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100249">10.1016/j.xpro.2020.100249</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we provide a protocol for the selection of conformation-specific intracellular antibody degraders using a cell-based screening method. We applied this protocol to select antibody-based degraders targeting the active form of the small GTPase RHOB (i.e., RHOB-GTP) using an engineered H2882 cell line. The protocol can be used to study the function of RHOB active conformation in various cellular settings. This protocol can be broadly applied to select any kind of intracellular antibody degraders, regardless of conformational state.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib1">Bery et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Antibodies are cloned into a degrader entry vector after a phage display selection</li>
<li>Degraders are screened in a stable antigen-fluorescent protein fusion cell line</li>
<li>This protocol will provide antibody-based degraders to study protein functions</li>
<li>This protocol could be applied to any protein binders and antigens of interest</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/347-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used in a publication to select intracellular antibodies as macromolecule degraders of the small GTPase RHOB’s active conformation to study RHOB-GTP function in cells (<a href="#bib1">Bery et al., 2019a</a>). We set up a cell-based screening that allows the direct monitoring in cells of RHOB-GTP degradation by nanobodies isolated after four rounds of phage display selection. The protocol for selecting conformational RHO antibodies by phage display has been previously described (<a href="#bib5">Chinestra et al., 2012</a>). To quantify the decay reflecting the antigen degradation, clonal stable cell lines with homogenous expression of the antigen or controls of selectivity need to be prepared before starting the screen. In our hands, the establishment of stable cell lines with homogenous expression of the target is mandatory. Notably, we were not successful in a previous attempt to screen the antibody-based degraders by transient transfection, due to high cell to cell heterogeneity in ectopic expression of both the antigen and the degrader. The antigen of interest (here RHOB-GTP) was fused to the histone H2B-mCherry to visualize the degradation by fluorescent microscopy and to localize it to the nucleus where it can accumulate without apparent toxicity. Any antigen of interest can be fused to a fluorescent reporter, such as mCherry, localized or not to a specific sub-cellular compartment (e.g., nucleus).</p>
<p>Various negative control cell lines were also produced to enable the selection of conformation-specific RHOB-GTP degraders. We used a constitutively active mutant of RHOB (RHOB<sup>Q63L</sup>) to mimic RHOB-GTP conformation and an inactive mutant (RHOB<sup>T19N</sup>) to mimic RHOB-GDP conformation. This protocol has a broad applicability as it can be applied to select any kind of intracellular antibody degraders (conformation-specific or not). In fact, we previously implemented this protocol as proof-of-concept to isolate (non-conformational) anti-GFP antibody-based degraders already identified as functional intracellular antibodies (i.e., that bind to GFP fusion in cells). We checked their ability to degrade H2B-GFP antigen in HeLa S3 H2B-GFP stable cell line (<a href="#bib10">Moutel et al., 2016</a>). Furthermore, this protocol is not limited to the antibody format and it could be applied to protein binders with a different scaffold such as DARPins (<a href="#bib4">Binz et al., 2003</a>, <a href="#bib2">Bery et al., 2019b</a>), monobodies (<a href="#bib8">Koide et al., 1998</a>, <a href="#bib13">Spencer-Smith et al., 2017</a>) or affimers (<a href="#bib14">Tiede et al., 2017</a>).</p>
<h3 id="sec1.1">Experimental considerations</h3>
<p>In this protocol, two entry vectors are used and were described elsewhere (<a href="#bib1">Bery et al., 2019a</a>). A schematic diagram of the antibody-based degrader entry vector is shown in <a href="#fig1">Figure 1</a> (top panel). This construct is a fusion between the SlmbFBOX domain from <i>Drosophila</i> and an antibody of choice (or any other kind of protein binders). IRES-MITO-GFP is used as fluorescent reporter to visualize transfected cells. MITO is a mitochondrial targeting sequence (MTS) from subunit VIII of human cytochrome c oxidase fused to a GFP (Green Fluorescent Protein).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/347-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic diagrams of the vectors used in this protocol</div>
<p>(Top) FBOX-sdAb-IRES-MITO-GFP is the antibody-based degrader entry vector (top vector). NcoI/NotI are the restriction sites used to insert any antibody of interest in fusion with the FBOX domain. GFP could be replace by any other fluorescent protein of interest by using AgeI/Acc65I restriction sites. sdAb: single domain antibody. (Bottom) H2B-mCherry-IRES-Zeocin is the antigen entry vector to express the antigen (here RHOB) in fusion to the histone H2B and the red fluorescent protein mCherry (middle and bottom vectors). Any antigen of interest can be inserted in this vector using KpnI/BamHI restriction sites. The different restriction sites used to construct these vectors are shown on the schematic diagrams.</p>
</figcaption>
</figure>
<p>The other vector employed in this protocol is the nuclear red fluorescent antigen entry vector (RHOB is the antigen of interest here). The red fluorescent protein mCherry is in fusion with histone H2B and therefore linked to the chromatin (nuclear signal). Schematic diagrams of the different vectors are shown <a href="#fig1">Figure 1</a> (bottom panel). Any antigen of interest can be inserted between KpnI and BamHI restriction sites to generate a nuclear and red fluorescent-bound antigen. Stable clones can be selected in eucaryote cells via the Zeocin resistance marker.</p>
<h3 id="sec1.2">Determine the concentration of antibiotic for the stable cell line establishment (antibiotic kill curve)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<ol>
<li>Plate 40,000 H2882 cells per well of a 12-well plate with RPMI medium supplemented with 10% FBS.</li>
<li>The next day, change the medium with 1 mL of medium containing the following concentrations of Zeocin (μg mL<sup>−1</sup>): 0; 50; 100; 200; 300; 400; 500; 600; 700; 800; 900; 1,000.</li>
<li>Change the medium every 3 to 4 days with fresh antibiotic.</li>
<li>After 10 to 14 days of antibiotic selection, choose the concentration of antibiotic which is twice the one that kills all the cells (e.g., at 50 μg mL<sup>−1</sup> of Zeocin all H2882 cells were dead after 14 days, therefore we choose 100 μg mL<sup>−1</sup> as concentration of Zeocin for the selection of H2882 stable cell lines).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Any antibiotic working in eucaryote cells can be used here depending on the backbone of the vector (e.g., Neomycin/G418, blasticidin, hygromycin, puromycin, …).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-RHOB</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat#sc-180; RRID: AB_2179110</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-mCherry</td>
<td>Biovision</td>
<td>Cat#5993-100, RRID: AB_1975001</td>
</tr>
<tr>
<td>Mouse monoclonal anti-RHOB</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat#sc-8048, RRID: AB_628219</td>
</tr>
<tr>
<td>Goat anti-mouse Alexa Fluor 488</td>
<td>Thermo Fisher</td>
<td>Cat# A-11001RRID: AB_2534069</td>
</tr>
<tr>
<td>Mouse monoclonal anti-α-tubulin</td>
<td>Sigma-Aldrich</td>
<td>Cat#T9026; RRID: AB_477593</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>TG1</td>
<td>Euromedex</td>
<td>Cat#LU-60502-1</td>
</tr>
<tr>
<td>DH5α</td>
<td>New England Biolabs</td>
<td>Cat#C2987I</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Zeocin</td>
<td>Invitrogen</td>
<td>Cat#R25001</td>
</tr>
<tr>
<td>Mowiol 4-88</td>
<td>Sigma</td>
<td>Cat#81381-50G</td>
</tr>
<tr>
<td>Paraformaldehyde, 16% solution</td>
<td>MP Biomedicals</td>
<td>Cat#02199983-CF</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>JetPrime transfection reagent</td>
<td>Polyplus Transfection</td>
<td>Cat#114-15</td>
</tr>
<tr>
<td>NucleoSpin Plasmid (Miniprep Kit)</td>
<td>Machery Nagel</td>
<td>Cat#740588.250</td>
</tr>
<tr>
<td>NucleoBond Xtra Midi Plus (Midiprep Kit)</td>
<td>Machery Nagel</td>
<td>Cat#740412.50</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Human: H2882</td>
<td>(<a href="#bib11">Sato et al., 2007</a>)</td>
<td>RRID: CVCL_5158</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>p-H2B-mCherry-RHOB<sup>Q63L</sup> ΔCAAX-IRES-Zeo</td>
<td>(<a href="#bib1">Bery et al., 2019a</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>p-H2B-mCherry-RHOB<sup>T19N</sup> ΔCAAX-IRES-Zeo</td>
<td>(<a href="#bib1">Bery et al., 2019a</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>p-H2B-mCherry-IRES-Zeo</td>
<td>(<a href="#bib1">Bery et al., 2019a</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>p-FBOX-sdAb-6xHis-myc-IRES-MITO-GFP</td>
<td>(<a href="#bib1">Bery et al., 2019a</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>NIS Elements v3</td>
<td>Nikon</td>
<td><a href="https://www.nikon.com/products/microscope-solutions/support/download/software/imgsfw/nis-br_v3221432.htm">https://www.nikon.com/products/microscope-solutions/support/download/software/imgsfw/nis-br_v3221432.htm</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Cell culture</h3>
<p>Here we specify the reagent used for the cell culture of H2882 cells (wild-type and stable clones):</p>
<div class="note">
<span class="note-title">Note:</span> Cell culture is performed in a sterile tissue culture hood under aseptic conditions and cells are maintained in a humidified 37°C incubator with 5% CO<sub>2</sub>.</div>
<p>H2882 Cell Medium: RPMI supplemented with 10% FBS with or without 100 μg mL<sup>−1</sup> Zeocin.</p>
<table id="undtbl1">
<caption>Paraformaldehyde (PFA) solution</caption>
<thead>
<tr>
<th>Reagent (stock concentration)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Paraformaldehyde (16%)</td>
<td>3.7%</td>
<td>5.55 mL</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>n/a</td>
<td>18.45 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>24 mL</b></td>
</tr>
</tbody>
</table>
<p>3.7% PFA is stored at 22°C for 1 week.</p>
<div class="critical">
<span class="critical-title">Critical:</span> PFA is a carcinogen, mutagen and toxic reagent and should be handled under a chemical fume hood and waste should be discard in a specific bin.</div>
<table id="undtbl2">
<caption>Mowiol preparation</caption>
<thead>
<tr>
<th>Reagent (stock concentration)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glycerol (100%)</td>
<td>25%</td>
<td>6 g</td>
</tr>
<tr>
<td>Mowiol (powder)</td>
<td>100 g L<sup>−1</sup>
</td>
<td>2.4 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>6 mL</td>
</tr>
<tr>
<td>Tris-HCl pH 8.5 (0.2 M)</td>
<td>0.1 M</td>
<td>12 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>24 mL</b></td>
</tr>
</tbody>
</table>
<ul>
<li>Add 6 mL of milli-Q water to the glycerol and Mowiol powder and leave 2 h at 22°C.</li>
<li>Add 12 mL of 0.2 M Tris-HCl pH 8.5.</li>
<li>Incubate 3 h in a water bath at 60°C, mixing regularly.</li>
<li>Centrifugation of the solution to remove any non-solubilized element (5 min at 5,000 × <i>g</i>).</li>
<li>Make aliquots in 1.5 mL Eppendorf tubes.</li>
<li>Store at −20°C for several months.</li>
</ul>
<h3 id="sec2.2">Equipment for the screen</h3>
<p>MoFlo Astrios (Flow cytometer, post-screen analysis of the antibody-based degraders)</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Any flow cytometer that has a green and a red laser can be used for the post-screen analysis.</div>
<p>Nikon Eclipse 90i: used for the immunofluorescence of the stable cell lines.</p>
<p>Inverted Nikon Eclipse Ti: used for the cell-based screen.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Any epifluorescence microscope device can be used for the immunofluorescence analysis.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Transfection of H2882 cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>Transfecting H2882 cells that are <i>RHOB</i><sup>-/-</sup> cells with H2B-mCherry-RHOB<sup>Q63L</sup>ΔCAAX plasmid will allow the selection of clones stably expressing this fluorescent localized antigen (see <a href="#before-you-begin">Before you begin</a> section and <a href="#fig1">Figure 1</a>, bottom panel). This will make the screen easier and more reliable than transient transfection of the antigen. If the degrader needs to be specific of a conformation or an isoform, then several stable cell lines can be done. Before starting the antibiotic selection of the cell line, an antibiotic kill curve needs to be performed to determine the concentration of antibiotic to use that kill all the cells (see <a href="#before-you-begin">Before you begin</a> section).</p>
<ol>
<li>Plate 40,000 cells per well of a 24-well plate (in 0.5 mL of RPMI supplemented with 10% FBS) to get the cells at 80% confluence the next day.</li>
<li>24 h later, transfect the cells with the JetPRIME reagent and the plasmid(s) of your choice (i.e., nuclear red fluorescent antigen entry vector). Add 0.75 μg of DNA to 50 μL of JetPRIME buffer, vortex 10 s and centrifuge 3 s at 2,000 × <i>g</i>. Add 1.5 μL of JetPRIME reagent, vortex 10 s and centrifuge 3 s at 2,000 × <i>g</i>. Incubate at 22°C for 10 min and then add the mix drop by drop on the cells.</li>
<li>4 h later, change the medium with 0.5 mL of fresh medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Any cell lines of interest that can be easily transfected (for the cell-based screen) could be used to make stable clones such as HEK293T or HeLa cells. Any suitable transfection reagent can also be used.</div>
<h3 id="sec3.2">Selection of H2882 stable clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 weeks</div>
<p>Once the cells are transfected, the antibiotic selection process can start. Generally, after 3 to 4 weeks of antibiotic selection (depending of the antibiotic used), clones should appear and be picked.</p>
<ol start="4">
<li>72 h after the transfection, detached the cells.
<ol type="a">
<li>Aspirate the medium, wash with 0.5 mL of PBS and add 0.3 mL of trypsin.</li>
<li>Leave the cells at 37°C for 3–5 min, until they are detached. Add 0.7 mL of complete medium and resuspend the cells by pipetting up and down carefully (1 mL of resuspended cells).</li>
</ol>
</li>
<li>Plate 1/50 (i.e., 20 μL of the 1 mL of resuspended cells), 1/150 (6.7 μL of the 1 mL) and the rest of the cells (973.3 μL) in 10 mL of medium containing your antibiotic at the concentration determined with the killing curve in 100 mm plates.</li>
<li>Every three / four days, change the medium of the plates with fresh antibiotic.</li>
<li>After 3 to 4 weeks (at least with Zeocin), clones (i.e., colonies of cells) should appear on the plate. When clones are 3–6 mm large, use sterile 8 mm cloning cylinders to pick the colonies.
<ol type="a">
<li>Aspirate the medium, wash the 100 mm plate with 5 mL of warm PBS.</li>
<li>Put the cloning cylinder around the colony of cells and press gently on the top of the cylinder to create a seal between the cylinder and the plate.</li>
<li>Add 50 μL of trypsin and wait under the tissue culture hood for 3 to 5 min, until the cells detached. You can pipet carefully up and down to force the cells to detach.</li>
<li>Prepare a 96-well plate with 200 μL of complete medium with antibiotic.</li>
<li>Once detached, pipet the 50 μL of trypsinized cells and add it into one well of the 96-well plate. One clone per well.</li>
</ol>
</li>
<li>Amplify the clones by increasing the surface of culture every time they are confluent: from 96-well to 48-well to 12-well, until a dozens of aliquots (2 × 10<sup>6</sup> cells per vial) can be frozen as stock and stored in liquid nitrogen.</li>
</ol>
<h3 id="sec3.3">Characterization of H2882 stable clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>Clonal stable cell lines have to be checked for the proper expression and localization of the antigen in cells by immunofluorescence and western blot.</p>
<ol start="9">
<li>Check the expression of the antigen by western blot (using either anti-RHOB, 1/500, and/or anti-mCherry, 1/500, antibodies incubated 18 h) (see <a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/347-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Analysis of the expression of the antigen (RHOB) by western blot</div>
<p>Wild-type (WT) and stable H2882 clones transfected with the H2B-mCherry-RHOB<sup>Q63L</sup> expression vector and selected with antibiotics were lysed and analyzed by western blot to check the expression of the antigen (RHOB) and the fluorescent reporter (mCherry) using anti-RHOB and anti-mCherry antibodies. α-Tubulin is the loading control (incubated 1 h at 1/10,000 dilution).</p>
</figcaption>
</figure>
</li>
<li>Check the correct localization of the fluorescent antigen (here nuclear mCherry-RHOB) by immunofluorescence (see <a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/347-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Localization of the antigen in cells by immunofluorescence</div>
<p>H2882 cells stably expressing the histone H2B-mCherry, H2B-mCherry-RHOB<sup>Q63L</sup>, or H2B-mCherry-RHOB<sup>T19N</sup> were stained with an anti-RHOB primary antibody and an Alexa-488 secondary antibody (green channel). DAPI is used to stain the DNA (blue channel) and mCherry is directly visualized on the red channel. Scale bars, 10 μm.</p>
</figcaption>
</figure>
<ol type="a">
<li>Plate cells on a coverslip, the day after, fix the cells with 3.7% PFA, permeabilize the cells, block them and incubate the appropriate primary (mouse anti-RHOB: 1/50, 3 h incubation at 22°C) and secondary antibodies (goat anti-mouse Alexa Fluor 488, 1/400, 1 h incubation at 22°C) and mount the coverslip on a slide with mounting medium containing DAPI (Mowiol, see <a href="#materials-and-equipment">Materials and equipment</a> section).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The clone used for the cell-based screen should ideally express a medium level of fluorescent antigen: enough to detect it easily by microscopy but not too high to be still able to observe a degradation by microscopy (here H2B-mCherry-RHOB, see clones 2–4 and 6 on <a href="#fig1">Figure 1</a> as good examples).</div>
<h3 id="sec3.4">Creation of a sub-library of intracellular antibody-based degraders</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>The phage display technology selects antibodies specific for an antigen. The protocol for selection and enrichment of conformational antibodies targeting GTP-bound RHO by phage display has been previously described elsewhere (<a href="#bib5">Chinestra et al., 2012</a>). Usually three to four rounds of panning are sufficient to get specific antibodies according to the stringency of the competitive or washing steps (<a href="#bib7">Haque and Tonks, 2012</a>). Then, antibody fragments are usually tested first as recombinant antibody fragments by various <i>in vitro</i> methods such as ELISA, western blot, or immunofluorescence. Then, some hits are further sub-cloned into a mammalian expression vector to be express as an intracellular antibody but only few antibodies performed best both in <i>in vitro</i> assays and as intrabodies. To directly identify the best intracellular antibody-based degraders, namely that express well in the reducing intracellular environment and remain functional (i.e., that deplete specifically their target antigen), we set up a cell-based screen after the last round of phage display by sub-cloning the library of antibodies obtained after this last round of phage display panning in an antibody-based degrader entry vector (described in <a href="#before-you-begin">Before you begin</a> section and in <a href="#bib1">Bery et al., 2019a</a>). Importantly, this protocol is not limited to phage display and could be applied to other similar panning enrichment methods involving large libraries of protein binders (e.g., single domain antibodies, DARPins, monobodies, …).</p>
<ol start="11">
<li>From the last round of phage display, directly grow TG1 <i>E. coli</i> containing the modified pHEN2 vector from NaLi-H1 library (<a href="#bib10">Moutel et al., 2016</a>) in 100 mL of LB with ampicillin and 0.5% glucose for 5 h at 37°C (until Optical Density (OD) ≈ 0.1–0.2).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to not grow 16–18 h, and be careful that OD not exceed 0.2 to avoid overgrowth of some antibody clones and loss of diversity.</div>
<ol start="12">
<li>Make a high purity medium scale preparation of plasmid DNA from the culture using any midiprep kit (i.e., Machery Nagel or Qiagen).</li>
<li>Digest 2–3 μg of DNA by NcoI and NotI enzymes 2 h at 37°C. This will digest the coding sequence of the antibodies from the modified pHEN2 vector.
<ol type="a">
<li>Run a 1% agarose gel to separate the digested DNA fragments</li>
<li>Purify the NcoI/NotI digested DNA (band around 400 bp) with a gel extract DNA purification kit (i.e., Machery Nagel or Qiagen).</li>
</ol>
</li>
<li>Digest the entry vector pFBOX-X-IRES-MITO-GFP by NcoI and NotI (see <a href="#fig1">Figure 1</a>).</li>
<li>Ligate the polyclonal pool of DNA fragments coding for the antibodies selected after the last round of phage display into the pFBOX-X-IRES-MITO-GFP vector using the T4 DNA ligase.</li>
<li>Transform competent DH5α bacteria or any cloning stain with the ligation mix and plate on several agar plates with 100 μg mL<sup>−1</sup> ampicillin.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> A 50% glycerol stock of each bacterial colony can be done by growing for 18 h the cells in a 96-well plate (one colony per well of a deep-well 96-well plate) containing 500 μL of LB complemented with 100 μg mL<sup>−1</sup> of ampicillin. The next day, add 500 μL of 100% glycerol in each well, mix each well to homogenize and store the plate at −80°C. Several plates can be done and be used as starting point for the miniprep step described in the next step.</div>
<h3 id="sec3.5">Cell-based screening of intracellular antibody-based degraders</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>The polyclonal pool of single domain antibodies (sdAb) selected after the last round of phage display is now in fusion with the FBOX domain along with a fluorescent reporter MITO-GFP. This reporter will allow the monitoring of transfected cells (useful for flow cytometry or microscopy analyses for instance). Each clone will be transfected for 48 h in the stable cell line expressing the fluorescent antigen of choice (here H2882 cells expressing H2B-mCherry-RHOB<sup>Q63L</sup> mutant). Each well will be observed using an inverted epifluorescence microscope. The cells expressing a FBOX-sdAb fusion will have green mitochondria, and in these cells, the red fluorescent nuclear signal will be examined.</p>
<ol start="17">
<li>The next day, start as many minipreps as needed for the screen by inoculating a colony into 3 mL of LB medium with 100 μg mL<sup>−1</sup> ampicillin. A total of 300 minipreps was performed in (<a href="#bib1">Bery et al., 2019a</a>). This can be scaled up or down (see <a href="#limitations">Limitations</a> section).
<ol type="a">
<li>Miniprep are done using a miniprep kit (Machery Nagel or Qiagen).</li>
<li>To check the diversity of the sub-library, 50 clones have been sent for sequencing.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Minipreps can be stored at −20°C and the screen can be performed over weeks if necessary.</div>
<ol start="18">
<li>In each well of 24-well plates, seed 30,000 H2882 cells stably expressing the target antigen H2B-mCherry-RHOB<sup>Q63L</sup>.</li>
<li>The next day, transfect 0.75 μg of plasmid DNA of interest (coding for FBOX-sdAb-IRES-MITO-GFP, see <a href="#fig1">Figure 1</a>, top panel) with 1.5 μL of JetPRIME reagent. One plasmid per well, as many wells as needed for the screen (i.e., 300 clones will require 300 wells).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to include a negative control (antibody not binding the antigen of interest) in the screen to assess the effect of an antibody non-degrading the antigen (as shown in <a href="#fig4">Figure 4</a>).</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/347-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Expected results from the cell-based screening of conformational antibody-based degraders</div>
<p>(A) Representative images obtained from the screen after transfection of the single domain antibody (sdAb)-based degrader plasmid (FBOX-sdAb-IRES-MITO-GFP) into the stable cell lines H2B-mCherry-RHOB<sup>Q63L</sup> and the negative stable cell line H2B-mCherry. White arrows indicate transfected cells (green mitochondria) where a decrease of the mCherry fluorescence is only observed with the FBOX-sdAb2 in the H2B-mCherry-RHOB<sup>Q63L</sup> stable cell line (FBOX-sdAb1 is a negative control).</p>
<p>(B) Analysis of the conformation of the positive hits by flow cytometry. H2882 stables clones: H2B-mCherry (white bars), H2B-mCherry-RHOB<sup>T19N</sup> (gray bars), and H2B-mCherry-RHOB<sup>Q63L</sup> (black bars) are transfected with a negative FBOX-sdAb (not degrading) or a positive FBOX-sdAb (degrading). The intensity of mCherry fluorescence in the green transfected cells is analyzed by flow cytometry and compared to the normalized negative control. Data represented in (B) are means ± standard deviation (SD) of three independent experiments.</p>
</figcaption>
</figure>
<ol start="20">
<li>48 h later, the cells are fixed 15 min with 3.7% of PFA, rinse three times with PBS and observed under an inverted microscope. See <a href="#fig4">Figure 4</a>A left-hand side panel for representative images expected from such a screen.</li>
<li>All the positive hits are subsequently screened against the negative stable cell lines expressing H2B-mCherry to confirm the specificity of the antibody-based degraders. See <a href="#fig4">Figure 4</a>A right-hand side panel.</li>
<li>To check the conformational specificity of the hits, a counter-screen on a negative cell line (here the H2B-mCherry-RHOB<sup>T19N</sup> cell line) is performed by flow cytometry. See <a href="#fig4">Figure 4</a>B.</li>
<li>Finally, positive hits which efficiently degrade the target but not the controls, are selected for further characterization on their capacity to efficiently degrade the endogenous antigen in more physiological cellular models.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol will allow the isolation of antibody-based degraders that works in the intracellular environment. Hence, it will be possible to use the antibody without the E3 ligase as tracking or inhibitory tools rather than only being a degrader. In addition, the antibody-based degraders can be expressed in cell lines of interest by lentivectors or other virus-based vectors, with inducible promoter if required, to study the function of the specific conformation of the target.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Images from the cell-based screen described in this protocol (see <a href="#fig4">Figure 4</a>A) could be quantified by using software such as ImageJ or Photoshop. By determining the fluorescence intensity in the nucleus of transfected cells versus cells transfected with a negative control, a percentage of degradation could be obtained.</p>
<p>We rather quantified the degradation percentage using flow cytometry after the cell-based screen, which allows the quantification of a large number of cells (more than 10, 000) in a very fast fashion. We normalized the intensity of the mCherry fluorescence from GFP positive cells (i.e., green mitochondria) transfected with an antibody-based degrader to the intensity of the mCherry fluorescence from the GFP positive cells transfected with a non-degrader negative control antibody (see <a href="#fig4">Figure 4</a>B as example).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>A limitation of this protocol would be the level of the screen’s throughput described here. It is a relatively low throughput screen and could be turn into a medium throughput screen using automation. Minipreps can be performed in 96 wells, followed by 96 well reverse transfection, and screening could be performed with a high content imager, such as Operetta or Arrayscan, or a flow cytometer, both in 96-well plate format. In these cases, quantification should be performed using the device dedicated software. Such screening methods would further provide a direct quantification of the degradation for each clone tested.</p>
<p>The number of clones to screen to get a degrader depends on various parameters such as the amino acid sequence diversity of the clones obtained after the phage display (i.e., number of clones with a different amino acid sequence) but also on the antibody library used for the phage display (e.g., favorable or not for the selection of intracellular antibodies). Of note, the clones’ diversity will also be influenced by the phage display panning conditions, the nature of the antigen used, … We screened 300 clones for 30 positive clones but only 8 hits had a different amino acid sequence which corresponds to a ≈2.5% hit rate. Therefore, using a higher throughput technique will increase the number of clones screened and potentially the number of positive hits.</p>
<p>Importantly, while the antibody-based degraders isolated with this screen degrade their target in an engineered stable cell line, further depletion of their endogenous untagged target in a model cell line or organism could give rise to several issues : (i) the cell line of interest should be easy to transfect or transduce, (ii) the cell line should express an antigen at a level easily revealed with a quantitative assay, (iii) the cell line should express the E3 ligase complex that associate with the Fbox domain for efficient degradation. Accordingly, we and others have shown that the type of E3 ligase used for antibody-based degraders is important (<a href="#bib3">Bery et al., 2020</a>, <a href="#bib12">Shin et al., 2015</a>, <a href="#bib9">Lim et al., 2020</a>), therefore, it might be possible to improve the degradation efficiency by changing the FBOX with another E3 ligase such as the VHL (<a href="#bib6">Fulcher et al., 2016</a>) or SPOP (<a href="#bib12">Shin et al., 2015</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Too many clones in the establishment of stable cell lines (related to Selection of H2882 stable clones, steps 4 and 5).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Make more dilutions of the cells (1/200, 1/500, …).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>No clones appear in the establishment of stable cell lines after 3 to 4 weeks of antibiotic selection (related to Selection of H2882 stable clones, steps 4 and 5).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Check the transfection efficiency by microscopy (fluorescent reporter) before adding the antibiotic. Use an easy to transfect cell line such as HEK293T or HeLa.</p>
<p>If it does not work, lentivectors could be used to efficiently and stably express an antigen of interest.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>No detectable MITO-GFP transfection reporter for some clones in the cell-based screen.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Increase the concentration of the plasmid miniprep by doubling the bacterial culture volume, and reducing the miniprep elution volume to the minimum according to manufacturer recommendation. Choose a more efficient transfection reagent.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>No positive degraders selected (related to Cell-based screening of intracellular antibody-based degraders, steps 19 and 20).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Increase the number of antibody-based degraders screened and check the diversity of the antibodies selected after the phage display by sequencing. If the diversity is too low (over representation of few clones), then use the antibodies from the previous round of phage display after checking the diversity of that round by sequencing.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>The antibody-based degraders work in the engineered stable cell line but not on the endogenous target in a cell line of interest (related to Cell-based screening of intracellular antibody-based degraders, step 23).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Switching FBOX domain to the one of a more efficient E3 ligase in the cell line used (e.g., VHL or SPOP) could resolve this issue.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bery, N., Keller, L., Soulie, M., Gence, R., Iscache, A.L., Cherier, J., Cabantous, S., Sordet, O., Lajoie-Mazenc, I., Pedelacq, J.D., et al. (2019a). A targeted protein degradation cell-based screening for nanobodies selective toward the cellular RHOB GTP-bound conformation. Cell Chem. Biol. <i>26</i>, 1544-1558.e6.</p>
<p id="bib2">Bery, N., Legg, S., Debreczeni, J., Breed, J., Embrey, K., Stubbs, C., Kolasinska-Zwierz, P., Barrett, N., Marwood, R., Watson, J., et al. (2019b). KRAS-specific inhibition using a DARPin binding to a site in the allosteric lobe. Nat. Commun. <i>10</i>, 2607.</p>
<p id="bib3">Bery, N., Miller, A., and Rabbitts, T. (2020). A potent KRAS macromolecule degrader specifically targeting tumours with mutant KRAS. Nat. Commun. <i>11</i>, 3233.</p>
<p id="bib4">Binz, H.K., Stumpp, M.T., Forrer, P., Amstutz, P., and Pluckthun, A. (2003). Designing repeat proteins: well-expressed, soluble and stable proteins from combinatorial libraries of consensus ankyrin repeat proteins. J. Mol. Biol. <i>332</i>, 489-503.</p>
<p id="bib5">Chinestra, P., Lajoie-Mazenc, I., Faye, J.C., and Favre, G. (2012). Use of phage display for the identification of molecular sensors specific for activated Rho. Methods Mol. Biol. <i>827</i>, 283-303.</p>
<p id="bib6">Fulcher, L.J., Macartney, T., Bozatzi, P., Hornberger, A., Rojas-Fernandez, A., and Sapkota, G.P. (2016). An affinity-directed protein missile system for targeted proteolysis. Open Biol. <i>6</i>, 160255.</p>
<p id="bib7">Haque, A. and Tonks, N.K. (2012). The use of phage display to generate conformation-sensor recombinant antibodies. Nat. Protoc. <i>7</i>, 2127-2143.</p>
<p id="bib8">Koide, A., Bailey, C.W., Huang, X., and Koide, S. (1998). The fibronectin type III domain as a scaffold for novel binding proteins. J. Mol. Biol. <i>284</i>, 1141-1151.</p>
<p id="bib9">Lim, S., Khoo, R., Peh, K.M., Teo, J., Chang, S.C., Ng, S., Beilhartz, G.L., Melnyk, R.A., Johannes, C.W., Brown, C.J., et al. (2020). bioPROTACs as versatile modulators of intracellular therapeutic targets including proliferating cell nuclear antigen (PCNA). Proc. Natl. Acad. Sci. U S A <i>117</i>, 5791-5800.</p>
<p id="bib10">Moutel, S., Bery, N., Bernard, V., Keller, L., Lemesre, E., De Marco, A., Ligat, L., Rain, J.C., Favre, G., Olichon, A., and Perez, F. (2016). NaLi-H1: a universal synthetic library of humanized nanobodies providing highly functional antibodies and intrabodies. eLife <i>5</i>, e16228.</p>
<p id="bib11">Sato, N., Fukui, T., Taniguchi, T., Yokoyama, T., Kondo, M., Nagasaka, T., Goto, Y., Gao, W., Ueda, Y., Yokoi, K., et al. (2007). RhoB is frequently downregulated in non-small-cell lung cancer and resides in the 2p24 homozygous deletion region of a lung cancer cell line. Int. J. Cancer <i>120</i>, 543-551.</p>
<p id="bib12">Shin, Y.J., Park, S.K., Jung, Y.J., Kim, Y.N., Kim, K.S., Park, O.K., Kwon, S.H., Jeon, S.H., Trinh LE, A., Fraser, S.E., et al. (2015). Nanobody-targeted E3-ubiquitin ligase complex degrades nuclear proteins. Sci. Rep. <i>5</i>, 14269.</p>
<p id="bib13">Spencer-Smith, R., Koide, A., Zhou, Y., Eguchi, R.R., Sha, F., Gajwani, P., Santana, D., Gupta, A., Jacobs, M., Herrero-Garcia, E., et al. (2017). Inhibition of RAS function through targeting an allosteric regulatory site. Nat. Chem. Biol. <i>13</i>, 62-68.</p>
<p id="bib14">Tiede, C., Bedford, R., Heseltine, S.J., Smith, G., Wijetunga, I., Ross, R., Alqallaf, D., Roberts, A.P., Balls, A., Curd, A., et al. (2017). Affimer proteins are versatile and renewable affinity reagents. eLife <i>6</i>, e24903.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Aurélien Olichon (<a href="mailto:aurelien.olichon@inserm.fr">aurelien.olichon@inserm.fr</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>No data or code were generated in this study.</p>
<h3>Acknowledgments</h3>
<p>This work is supported by a grant from <a href="https://doi.org/10.13039/501100002915">Fondation pour la Recherche Médicale</a> (FRM) (Equipe labellisée FRM [DEQ20170839117]). N.B. is supported by a fellowship from the <a href="https://doi.org/10.13039/501100004431">Fondation de France</a> (no. 00097692).</p>
<h3>Author Contributions</h3>
<p>Conceptualization: N.B., G.F., and A.O.; Methodology: N.B., G.F., and A.O.; Writing: N.B., G.F., and A.O.; Supervision and Funding Acquisition: A.O. and G.F.</p>
<h3>Declaration of Interests</h3>
<p>N.B., G.F., and A.O. are co-inventors on the patent PTC/EP2016/052136, concerning the discovery of RHO-GTP single domain antibodies and their applications.</p>
</section>
</article>