<article>
<section>
<h1 id="header">A high-throughput genetic screening protocol to measure lipid bilayer stress-induced unfolded protein response in <i>Saccharomyces cerevisiae</i>
</h1>
<p><time datetime="2021-10-01">Published: October 1, 2021</time></p>
<p>Nurulain Ho,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup> Wei Sheng Yap,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> and Guillaume Thibault<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn4">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>School of Biological Sciences, Nanyang Technological University, 60 Nanyang Drive, Singapore 637551, Singapore</p>
<p id="aff2"><sup>2</sup>Institute of Molecular and Cell Biology, A∗STAR, Singapore, 138673, Singapore</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Present address: The Cancer Science Institute of Singapore, National University of Singapore, 14 Medical Drive, Singapore 117599</p>
<p id="fn3"><sup>5</sup>Technical contact</p>
<p id="fn4"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:csihn@nus.edu.sg">csihn@nus.edu.sg</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:thibault@ntu.edu.sg">thibault@ntu.edu.sg</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100868">10.1016/j.xpro.2021.100868</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The endoplasmic reticulum (ER) stress is defined by the accumulation of unfolded proteins at the ER and perturbation at the ER membrane, known as lipid bilayer stress (LBS). In turn, ER stress triggers the unfolded protein response (UPR) to restore ER homeostasis. Here, we provide a modified protocol based on the synthetic genetic array analysis in <i>Saccharomyces cerevisiae</i> to identify genetic perturbations that induce the UPR by LBS. This method is adaptable to other canonical stress pathways.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref3">Ho et al. (2020)</a>, <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref4">Jonikas et al. (2009)</a> and <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref1">Baryshnikova et al. (2010)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Generation and validation of <i>IRE1</i> and <i>IRE1ΔLD</i> query strains with a UPR reporter</li>
<li>Detailed protocol of query strains mated to the yeast deletion library using SGA</li>
<li>High-throughput measurement of reporter fluorescence levels by flow cytometry</li>
<li>Data analysis to identify gene deletions activating the UPR by lipid bilayer stress</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1054-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>To identify genes that are required to maintain endoplasmic reticulum (ER) membrane homeostasis, we employed a genome-wide high-throughput screen using an engineered <i>Saccharomyces cerevisiae</i> Ire1 sensor that activates the unfolded protein response (UPR) exclusively by lipid bilayer stress (LBS). The engineered sensor lacks the proteotoxic stress-sensing luminal domain (Ire1ΔLD). This query strain was mated to the yeast deletion library using synthetic genetic array methodology. A parallel screen was carried out using a query strain expressing full-length Ire1. Both query strains contain an integrated reporter system in which GFP is driven by the UPR element (<i>UPRE</i>)-containing promoter and mCherry is driven by a constitutive promoter.</p>
<h3 id="sec1.1">Generating Ire1 and Ire1ΔLD query strains and library preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>Both the <i>IRE1</i> and <i>IRE1</i>Δ<i>LD</i> genes are genomically integrated in YMS612 strain with the dominant hygromycin resistance marker (<i>Hyg</i><sup><i>R</i></sup>) (<a href="#fig1">Figure 1</a>A and <a href="#key-resources-table">key resources table</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1054-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Genomic integration and validation of the query strains</div>
<p>(A) Schematic representation of <i>IRE1</i> and <i>IRE1ΔLD</i> used in the study. The luminal domain (LD) of <i>IRE1</i> is truncated from amino acids 31 to 526 to generate <i>IRE1ΔLD</i>.</p>
<p>(B and C) Typical fluorescence images of query strains <i>IRE1</i> (B) and <i>IRE1ΔLD</i> (C) with genomically integrated <i>UPREpr-GFP</i> and <i>TEF2pr-mCherry</i>. <i>IRE1</i> but not <i>IRE1ΔLD</i> query strains are GFP positive upon DTT treatment. Scale bar: 10 μm.</p>
</figcaption>
</figure>
<ol>
<li>Genomic integration of <i>IRE1</i> and <i>IRE1</i>Δ<i>LD</i> with <i>Hyg</i><sup><i>R</i></sup>
<ol type="a">
<li>PCR-mediated amplification of <i>IRE-Hyg</i><sup><i>R</i></sup> and <i>IRE1ΔLD-Hyg</i><sup><i>R</i></sup> from plasmid pGT0453 and pGT0454</li>
<li>Amplify <i>IRE1-Hyg</i><sup><i>R</i></sup> and <i>IRE1ΔLD-Hyg</i><sup><i>R</i></sup> cassettes with primer pair HN148 and HN149 using <i>Phusion</i> polymerase following <a href="https://www.neb.sg/protocols/0001/01/01/pcr-protocol-m0530">manufacturer’s protocol</a>.</li>
<li>Inoculate a single colony of YMS612 strain in 10 mL of YPD media and grow 16 h at 30°C to an absorbance at 600 nm of 0.6–0.8 OD/mL.</li>
<li>Pellet at 1,000 × <i>g</i>, 5 min, wash with 20 mL LiOAc mix and resuspend in 200 μL LiOAc mix.</li>
<li>Add 150 μL of washed cells to 700 μL PLATE mix, 10 μL of denatured ssDNA and 2 μg of purified PCR product and incubate 1 h at 30°C.</li>
<li>Add 118.6 μL DMSO to the transformation mixture and heat shock 22 min at 42°C.</li>
<li>Pellet the cells for 15 s at maximum speed.</li>
<li>Resuspend the pellet in 200 μL of YPD and incubate for 1.5 h at 30°C.</li>
<li>Pellet the cells for 15 s at maximum speed and remove most of the supernatant.</li>
<li>Plate onto YPD plates containing 200 μg/mL Hyg.</li>
<li>Positive clones will appear after 48–72 h incubation at 30°C.</li>
<li>Successful integration of <i>IRE1</i> (YGT1228) and <i>IRE1ΔLD</i> (YGT1202) will yield PCR fragments of 6054 bp and 4285 bp, respectively, by amplifying the genomic DNA with primer pair HN148 and HN149.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The selected strains can be stored at −80°C as glycerol stocks until ready to proceed to the next steps.</div>
<div class="note">
<span class="note-title">Note:</span> The genotyping strategy at step 1l is only insert-specific. Therefore, the subsequent functional validation steps are important. Otherwise, a flanking primer can be used to replace either HN148 or HN149 for orientation-specific genotyping.</div>
<ol start="2">
<li>Validate the functionality of Ire1 and Ire1ΔLD in the query strains by measuring the UPR activation using confocal fluorescence microscopy:
<ol type="a">
<li>Grow YGT1228 and YGT1202 strains each in two tubes containing 500 μL of YPD media at 30°C to mid-log phase.
<div class="critical">
<span class="critical-title">Critical:</span> The absorbance at 600 nm should not exceed 0.4 OD/mL to prevent UPR activation in these cells.</div>
</li>
<li>Add 0.5 μL of 1 M DTT (final concentration of 1 mM) into each tube and incubate for 1 h to induce the UPR.</li>
<li>Coat coverslips evenly with 1 mg/mL of ConA and incubate at ∼22°C for 15 min.</li>
<li>Rinse the coverslips three times with 500 μL of sterile ddH<sub>2</sub>O.</li>
<li>Place cells on coated coverslips mounted onto Attofluor cell chambers.</li>
<li>Set up three acquisition channels to acquire:<br> (1) Brightfield<br> (2) GFP (excitation 488 nm, emission 507 nm)<br> (3) mCherry (excitation 532 nm, emission 600 nm)</li>
<li>Image cells using a Zeiss LSM 710 confocal microscope with a 100× 1.4 NA oil plan-Apochromat objective (<a href="#fig1">Figures 1</a>B and 1C).</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Sterilizing the pin tools
<div class="note">
<span class="note-title">Note:</span> All pinning steps are performed using a 96 Floating E-Clip style Pin Multi-Blot Replicator (V&amp;P Scientific). Alternatively, a ROTOR HDA (Singer Instruments), a BioMatrix Colony Processing Robot, or any other desired pinning tools can be used.</div>
<ol type="a">
<li>Set up five sterile reservoirs containing the following: (1) 30 mL of sterile ddH<sub>2</sub>O, (2) 40 mL of 10% bleach, (3) 50 mL of sterile ddH<sub>2</sub>O, (4) 70 mL of sterile ddH<sub>2</sub>O, and (5) 90 mL of 95% ethanol.</li>
<li>Soak the pin replicator for 1 min in the reservoir containing 30 mL of sterile ddH<sub>2</sub>O to remove the cells from the pins.</li>
<li>Immerse the pin replicator in 10% bleach for approximately 20 s.</li>
<li>Rinse off the bleach from the pins by transferring the replicator to the reservoir containing 50 mL of sterile ddH<sub>2</sub>O and then to reservoir containing 70 mL of ddH<sub>2</sub>O.</li>
<li>Immerse the replicator pins in 95% ethanol for 30 s.</li>
<li>Shake off the excess ethanol and flame the pin replicator.</li>
<li>Allow the pin replicator to cool for 3 min before use.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure thorough removal of cells after each pinning step to avoid cross contamination. The hand pinner must be sterilized after each use.</div>
<ol start="4">
<li>Constructing the deletion mutant array in 384-density array format
<ol type="a">
<li>Take out the desired microplates containing glycerol stocks of the deletion array from −80°C.</li>
<li>Immediately place the plate on dry ice to ensure gentle thawing of the glycerol stocks. Do not allow the glycerol stocks to thaw out completely.</li>
<li>Carefully peel off the aluminum sealer from each 96-well plate.
<div class="critical">
<span class="critical-title">Critical:</span> Be cautious not to cross-contaminate wells.</div>
</li>
<li>Using the pin replicator, copy the array on a single-well YPD agar plate containing 200 μg/mL G418. Use the library copier VP 381 as a guide for printing the library (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1054-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Construction of the 384-format mutant array guided by the library copier VP381</div>
<p>Yeast deletion library colonies grown on the source plates (YPD agar plate containing G418) were created by inoculation from frozen glycerol stock in 96-format arrays. Before the subsequent pinning steps, fit the destination plate (YPD agar plate containing G418) into the middle of the library copier VP381. To condense four 96-format arrays into a single 384-format array, pin the colonies on the first source plate using a sterile 96-pin replicator and replicate onto the destination plate aligned to the ‘A’ alignment holes. Repeat the pinning step with the next three source plates aligned to the ‘B’, ‘C’, and ‘D’ alignment holes and transfer to the same destination plate. The resulting 384-format mutant array will then be used for subsequent SGA steps to generate the final mutant array expressing the UPR sensors and reporters. Each 384-format array can be expanded to four 96-format liquid culture for flow cytometry acquisition.</p>
</figcaption>
</figure>
</li>
<li>Reseal the master plates with sterile aluminum seals and immediately return to −80°C.</li>
<li>Incubate the plates at ∼22°C for 2 days.</li>
<li>Use the 96 Floating E-Clip style Pin Multi-Blot Replicator to array the 96-density format colonies into 384-density array format with the help of the library copier VP 381. Incubate the colonies at ∼22°C for 1 day.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The potent UPR inducer gene deletion <i>opi3</i>Δ strain and wild-type (WT) strains are included in each 96 well plate. Typically, each library microplates contain at least 2 empty wells that were used to include positive (<i>opi3</i>Δ) and negative (WT) controls. <i>OPI3</i> deletion was selected because it causes high UPR by lipid bilayer stress in both query strains <i>IRE1</i> and <i>IRE1ΔLD</i> (<a href="#bib3">Ho et al., 2020</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Freshly grown colonies of the 384-density deletion array should be generated for each batch of the genetic screen.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Bacto agar</td>
<td>Becton Dickinson</td>
<td>214010</td>
</tr>
<tr>
<td>Bacto peptone</td>
<td>Becton Dickinson</td>
<td>211677</td>
</tr>
<tr>
<td>Bacto yeast extract</td>
<td>Becton Dickinson</td>
<td>212750</td>
</tr>
<tr>
<td>Adenine sulfate</td>
<td>Sigma-Aldrich</td>
<td>A2786</td>
</tr>
<tr>
<td>Tyrosine</td>
<td>Sigma-Aldrich</td>
<td>T8566</td>
</tr>
<tr>
<td>Isoleucine</td>
<td>Sigma-Aldrich</td>
<td>I7403</td>
</tr>
<tr>
<td>Phenylalanine</td>
<td>Sigma-Aldrich</td>
<td>P5482</td>
</tr>
<tr>
<td>Glutamic acid</td>
<td>Sigma-Aldrich</td>
<td>G8415</td>
</tr>
<tr>
<td>Aspartic acid</td>
<td>Sigma-Aldrich</td>
<td>A7219</td>
</tr>
<tr>
<td>Valine</td>
<td>Sigma-Aldrich</td>
<td>V0513</td>
</tr>
<tr>
<td>Threonine</td>
<td>Sigma-Aldrich</td>
<td>T8441</td>
</tr>
<tr>
<td>Serine</td>
<td>Sigma-Aldrich</td>
<td>S4311</td>
</tr>
<tr>
<td>Tryptophan</td>
<td>Sigma-Aldrich</td>
<td>T8941</td>
</tr>
<tr>
<td>Leucine</td>
<td>Sigma-Aldrich</td>
<td>L8912</td>
</tr>
<tr>
<td>Inositol</td>
<td>Sigma-Aldrich</td>
<td>I5125</td>
</tr>
<tr>
<td>
<i>p</i>-Aminobenzoic acid</td>
<td>Sigma-Aldrich</td>
<td>A9878</td>
</tr>
<tr>
<td>Alanine</td>
<td>Sigma-Aldrich</td>
<td>A7469</td>
</tr>
<tr>
<td>Asparagine</td>
<td>Sigma-Aldrich</td>
<td>A4159</td>
</tr>
<tr>
<td>Cysteine</td>
<td>Sigma-Aldrich</td>
<td>C7352</td>
</tr>
<tr>
<td>Glutamine</td>
<td>Sigma-Aldrich</td>
<td>49419</td>
</tr>
<tr>
<td>Glycine</td>
<td>Sigma-Aldrich</td>
<td>G8790</td>
</tr>
<tr>
<td>Proline</td>
<td>Sigma-Aldrich</td>
<td>P5607</td>
</tr>
<tr>
<td>Difco Yeast Nitrogen Base without Amino Acids (YNB w/o AA)</td>
<td>Becton Dickinson</td>
<td>291930</td>
</tr>
<tr>
<td>Difco Yeast Nitrogen Base without Amino Acids and Ammonium Sulfate (YNB w/o AA, AS)</td>
<td>Becton Dickinson</td>
<td>233520</td>
</tr>
<tr>
<td>L-Glutamic acid monosodium salt hydrate (MSG)</td>
<td>Sigma-Aldrich</td>
<td>G1626</td>
</tr>
<tr>
<td>D-Glucose monohydrate (Glucose)</td>
<td>Duchefa Biochemie</td>
<td>G0802-1000</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Promega</td>
<td>H5433</td>
</tr>
<tr>
<td>DMSO</td>
<td>Sigma-Aldrich</td>
<td>D8418</td>
</tr>
<tr>
<td>Hygromycin B (Hyg)</td>
<td>Nacalai Tesque</td>
<td>07296-11-E</td>
</tr>
<tr>
<td>Geneticin (G418 sulfate) (50 mg/mL)</td>
<td>Gibco</td>
<td>10131035</td>
</tr>
<tr>
<td>L-canavanine sulfate crystalline (canavanine)</td>
<td>Sigma Aldrich</td>
<td>C9758</td>
</tr>
<tr>
<td>S-(2-Aminoethyl)-L-cysteine hydrochloride (thialysine)</td>
<td>Sigma-Aldrich</td>
<td>A2636</td>
</tr>
<tr>
<td>
<i>Taq</i> DNA polymerase</td>
<td>New England Biolabs</td>
<td>M0273</td>
</tr>
<tr>
<td>Standard <i>Taq</i> reaction buffer</td>
<td>New England Biolabs</td>
<td>B9014</td>
</tr>
<tr>
<td>
<i>Phusion</i> high-fidelity DNA polymerase</td>
<td>New England Biolabs</td>
<td>M0530</td>
</tr>
<tr>
<td>
<i>Phusion</i> HF buffer</td>
<td>New England Biolabs</td>
<td>B0518</td>
</tr>
<tr>
<td>10 mM Deoxynucleotide (dNTP) Solution Mix</td>
<td>New England Biolabs</td>
<td>N0447</td>
</tr>
<tr>
<td>Poly(ethylene glycol), average MW 3350</td>
<td>Sigma-Aldrich</td>
<td>P4338</td>
</tr>
<tr>
<td>Lithium acetate</td>
<td>Sigma-Aldrich</td>
<td>517992</td>
</tr>
<tr>
<td>EDTA</td>
<td>Bio-Rad</td>
<td>1610729</td>
</tr>
<tr>
<td>Tris</td>
<td>Bio-Rad</td>
<td>1610719</td>
</tr>
<tr>
<td>Commercial bleach</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Salmon sperm DNA (ssDNA)</td>
<td>Sigma-Aldrich</td>
<td>31149</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>Gold Biotechnology</td>
<td>DTT50</td>
</tr>
<tr>
<td>Concanavalin A Type IV (ConA)</td>
<td>Sigma-Aldrich</td>
<td>C2010</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>MAT</i>α<i>, UPRE-GFP::URA TEF2pr-mCherry::MET15 his3Δ1 leu2Δ0 lys2 met15Δ0 ura3Δ0 can1Δ::STE2pr-</i>Sp<i>_his5 lyp1Δ::STE3pr-LEU2 cyh2</i>, BY4741 background</td>
<td>(<a href="#bib2">Haass et al., 2007</a>)</td>
<td>YMS612</td>
</tr>
<tr>
<td>
<i>ire1::IRE1-Hyg</i><sup><i>R</i></sup>, YMS612 background</td>
<td>(<a href="#bib3">Ho et al., 2020</a>)</td>
<td>YGT1228</td>
</tr>
<tr>
<td>
<i>ire1::iRE1ΔLD-Hyg</i><sup><i>R</i></sup>, YMS612 background</td>
<td>(<a href="#bib3">Ho et al., 2020</a>)</td>
<td>YGT1202</td>
</tr>
<tr>
<td>Yeast Mat-A Haploid deletion clones</td>
<td>Thermo Fisher Scientific</td>
<td>95401.H2P</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Forward primer for <i>IRE1-Hyg</i><sup><i>R</i></sup><i>/IRE1ΔLD-Hyg</i><sup><i>R</i></sup> cassettes: <b>ACAAAGAAGTAATGAACTTAAATGCTATTATACAG</b>
</td>
<td>IDT</td>
<td>HN148</td>
</tr>
<tr>
<td>Reverse primer for <i>IRE1-Hyg</i><sup><i>R</i></sup><i>/IRE1ΔLD-Hyg</i><sup><i>R</i></sup> cassettes: <b>CCGTCCCAAACATTGTCATAGATTC</b>
</td>
<td>IDT</td>
<td>HN149</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>
<i>pRS313-IRE1-Hyg</i> <sup><i>R</i></sup>
</td>
<td>This study</td>
<td>pGT453</td>
</tr>
<tr>
<td>
<i>pRS313-IRE1ΔLD-Hyg</i> <sup><i>R</i></sup>
</td>
<td>This study</td>
<td>pGT454</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo 10.8.0</td>
<td>Becton Dickinson</td>
<td><a href="https://www.flowjo.com/solutions/flowjo">https://www.flowjo.com/solutions/flowjo</a></td>
</tr>
<tr>
<td>FACSDiVA v 8.0</td>
<td>Becton Dickinson</td>
<td><a href="https://www.bdbiosciences.com/en-eu/instruments/research-instruments/research-software/flow-cytometry-acquisition/facsdiva-software">https://www.bdbiosciences.com/en-eu/instruments/research-instruments/research-software/flow-cytometry-acquisition/facsdiva-software</a></td>
</tr>
<tr>
<td>ImageJ</td>
<td>NIH</td>
<td><a href="https://imagej.nih.gov/ij/index.html">https://imagej.nih.gov/ij/index.html</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Attofluor cell chamber</td>
<td>Thermo Fisher Scientific</td>
<td>A7816</td>
</tr>
<tr>
<td>Cover glass, circle, 25 mm</td>
<td>Fisher Scientific</td>
<td>12-545-102P</td>
</tr>
<tr>
<td>One-well plate, sterile</td>
<td>Greiner</td>
<td>670180</td>
</tr>
<tr>
<td>Microplate, 96 wells, sterile</td>
<td>Greiner</td>
<td>655161</td>
</tr>
<tr>
<td>Lid for microplate, sterile</td>
<td>Greiner</td>
<td>656161</td>
</tr>
<tr>
<td>Silverseal sealer, aluminum</td>
<td>Greiner</td>
<td>676090</td>
</tr>
<tr>
<td>96 Floating E-Clip style Pin Multi-Blot Replicator</td>
<td>V&amp;P Scientific</td>
<td>VP 408FS2AS</td>
</tr>
<tr>
<td>Library copier</td>
<td>V&amp;P Scientific</td>
<td>VP 381</td>
</tr>
<tr>
<td>384 Solid Pin Multi-Blot Replicator</td>
<td>V&amp;P Scientific</td>
<td>VP 384F</td>
</tr>
<tr>
<td>LSRFortessa X-20 with automated high-throughput sampler (HTS)</td>
<td>Becton Dickinson</td>
<td>N/A</td>
</tr>
<tr>
<td>LSM 710 confocal microscope with a 100× 1.4 NA oil plan-Apochromat objective</td>
<td>Zeiss</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Stock solutions</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>50% Glucose</td>
<td>50% (w/v)</td>
</tr>
<tr>
<td>100 mg/mL Canavanine, store at −20°C</td>
<td>100 mg/mL</td>
</tr>
<tr>
<td>100 mg/mL Thialysine, store at −20°C</td>
<td>100 mg/mL</td>
</tr>
<tr>
<td>100 mg/mL Hyg, store at 4°C</td>
<td>100 mg/mL</td>
</tr>
<tr>
<td>1 M Lithium acetate</td>
<td>1 M</td>
</tr>
<tr>
<td>50% PEG</td>
<td>50% (w/v)</td>
</tr>
<tr>
<td>10 mg/mL ssDNA, store at −20°C</td>
<td>10 mg/mL</td>
</tr>
<tr>
<td>10× Tris-EDTA buffer, pH7.4</td>
<td>100 mM Tris, 10 mM EDTA</td>
</tr>
<tr>
<td>1 M DTT, store at −20°C</td>
<td>1 M</td>
</tr>
<tr>
<td>10% bleach</td>
<td>10% (v/v)</td>
</tr>
<tr>
<td>1 mg/mL ConA, store at −20°C,</td>
<td>1 mg/mL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>PLATE mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Lithium acetate</td>
<td>100 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>10× Tris-EDTA, pH7.4</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>50% PEG</td>
<td>40%</td>
<td>40 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterile filter before use. Prepare fresh before use.</p>
<table id="undtbl3">
<caption>LiOAc Mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Lithium acetate</td>
<td>100 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>10× Tris-EDTA</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>Sterile ddH<sub>2</sub>O</td>
<td>40%</td>
<td>40 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterile filter before use. Prepare fresh before use.</p>
<table id="undtbl4">
<caption>10× Amino acid supplement drop-out mixture (drop out His, Arg, Lys, Ura, Met)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Adenine sulfate</td>
<td>1 g/L</td>
<td>1 g</td>
</tr>
<tr>
<td>Tyrosine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Isoleucine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Phenylalanine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Glutamic acid</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Aspartic acid</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Valine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Threonine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Serine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Tryptophan</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Leucine</td>
<td>3.6 g/L</td>
<td>3.6 g</td>
</tr>
<tr>
<td>Inositol</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>p-Aminobenzoic acid</td>
<td>0.1 g/L</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Alanine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Asparagine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Cysteine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Glutamine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Glycine</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Proline</td>
<td>0.7 g/L</td>
<td>0.7 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterile filter before use. Store at 4<sup>°</sup>C, use within a month.</p>
<table id="undtbl5">
<caption>YPD medium with or without G418 and/or Hyg</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast extract</td>
<td>1% (w/v)</td>
<td>10 g</td>
</tr>
<tr>
<td>Peptone</td>
<td>2% (w/v)</td>
<td>20 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>Up to 960 mL</td>
</tr>
<tr>
<td>Autoclave</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>50% D-Glucose monohydrate</td>
<td>2% (w/v)</td>
<td>40 mL</td>
</tr>
<tr>
<td>50 mg/mL G418 (optional)</td>
<td>200 μg/mL</td>
<td>4 mL∗</td>
</tr>
<tr>
<td>100 mg/mL Hyg (optional)</td>
<td>200 μg/mL</td>
<td>2 mL∗</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗Adjust ddH<sub>2</sub>O accordingly before autoclaving. Store at 20<sup>°</sup>C or 4<sup>°</sup>C with antibiotics, use within a month.</p>
<table id="undtbl6">
<caption>YPD agar with or without G418 and/or Hyg</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast extract</td>
<td>1% (w/v)</td>
<td>10 g</td>
</tr>
<tr>
<td>Peptone</td>
<td>2% (w/v)</td>
<td>20 g</td>
</tr>
<tr>
<td>Agar</td>
<td>2% (w/v)</td>
<td>20 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>Up to 954 mL</td>
</tr>
<tr>
<td>Autoclave</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>50% Glucose</td>
<td>2% (w/v)</td>
<td>40 mL</td>
</tr>
<tr>
<td>50 mg/mL G418 (optional)</td>
<td>200 μg/mL</td>
<td>4 mL∗</td>
</tr>
<tr>
<td>100 mg/mL Hyg (optional)</td>
<td>200 μg/mL</td>
<td>2 mL∗</td>
</tr>
<tr>
<td>Pour plates</td>
<td>–</td>
<td>–</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗Adjust ddH<sub>2</sub>O accordingly before autoclaving. Store at 20<sup>°</sup>C or 4<sup>°</sup>C with antibiotics, use within a month.</p>
<table id="undtbl7">
<caption>Sporulation amino acids supplement powder</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Histidine</td>
<td>2 g</td>
</tr>
<tr>
<td>Leucine</td>
<td>10 g</td>
</tr>
<tr>
<td>Uracil</td>
<td>2 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4<sup>°</sup>C protected from light, use within a month.</p>
<table id="undtbl8">
<caption>Enriched sporulation agar</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Potassium acetate</td>
<td>1% (w/v)</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>0.1% (w/v)</td>
<td>1 g</td>
</tr>
<tr>
<td>D-Glucose monohydrate</td>
<td>0.05% (w/v)</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Sporulation amino acids supplement powder</td>
<td>0.01% (w/v)</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Agar</td>
<td>2% (w/v)</td>
<td>20 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>Up to 996 mL</td>
</tr>
<tr>
<td>Autoclave</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>50 mg/mL G418</td>
<td>200 μg/mL</td>
<td>4 mL</td>
</tr>
<tr>
<td>Pour plates</td>
<td>–</td>
<td>–</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4<sup>°</sup>C, use within a month.</p>
<table id="undtbl9">
<caption>SD-His/Arg/Lys/Ura/Met containing canavanine/thialysine</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>YNB (w/o AA)</td>
<td>6.7 g/L</td>
<td>6.7 g</td>
</tr>
<tr>
<td>Agar</td>
<td>2% (w/v)</td>
<td>20 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>Up to 859 mL</td>
</tr>
<tr>
<td>Autoclave</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>50% Glucose</td>
<td>2% (w/v)</td>
<td>40 mL</td>
</tr>
<tr>
<td>10× Amino acid supplement drop-out mixture</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>100 mg/mL Canavanine</td>
<td>50 μg/mL</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>100 mg/mL Thialysine</td>
<td>50 μg/mL</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Pour plates</td>
<td>–</td>
<td>–</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4<sup>°</sup>C, use within a month.</p>
<table id="undtbl10">
<caption>SD<sub>MSG</sub>-His/Arg/Lys/Ura/Met containing canavanine/thialysine/G418</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>YNB (w/o AA, AS)</td>
<td>1.7 g/L</td>
<td>1.7 g</td>
</tr>
<tr>
<td>MSG</td>
<td>1 g/L</td>
<td>1 g</td>
</tr>
<tr>
<td>Agar</td>
<td>2% (w/v)</td>
<td>20 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>Up to 855 mL</td>
</tr>
<tr>
<td>Autoclave</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>50% Glucose</td>
<td>2% (w/v)</td>
<td>40 mL</td>
</tr>
<tr>
<td>10× Amino acid supplement drop-out mixture</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>100 mg/mL Canavanine</td>
<td>50 μg/mL</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>100 mg/mL Thialysine</td>
<td>50 μg/mL</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>50 mg/mL G418</td>
<td>200 μg/mL</td>
<td>4 mL</td>
</tr>
<tr>
<td>Pour plates</td>
<td>–</td>
<td>–</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4<sup>°</sup>C, use within a month.</p>
<table id="undtbl11">
<caption>SD<sub>MSG</sub>-His/Arg/Lys/Ura/Met containing canavanine/thialysine/G418/Hyg</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>YNB (w/o AA, AS)</td>
<td>1.7 g/L</td>
<td>1.7 g</td>
</tr>
<tr>
<td>MSG</td>
<td>1 g/L</td>
<td>1 g</td>
</tr>
<tr>
<td>Agar</td>
<td>2% (w/v)</td>
<td>20 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>Up to 853 mL</td>
</tr>
<tr>
<td>Autoclave</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>50% Glucose</td>
<td>2% (w/v)</td>
<td>40 mL</td>
</tr>
<tr>
<td>10× Amino acid supplement drop-out mixture</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>100 mg/mL Canavanine</td>
<td>50 μg/mL</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>100 mg/mL Thialysine</td>
<td>50 μg/mL</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>50 mg/mL G418</td>
<td>200 μg/mL</td>
<td>4 mL</td>
</tr>
<tr>
<td>100 mg/mL Hyg</td>
<td>200 μg/mL</td>
<td>2 mL</td>
</tr>
<tr>
<td>Pour plates</td>
<td>–</td>
<td>–</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4<sup>°</sup>C, use within a month.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Selection of query strains mated to the yeast deletion library</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 weeks</div>
<div class="note">
<span class="note-title">Note:</span> Keeping the environment as sterile as possible at the bench is critical. Pinning steps leave plates vulnerable to contaminations.</div>
<ol>
<li>Freshly grow the query strains <i>IRE1</i> (YGT1228) and <i>IRE1Δ</i>LD (YGT1202) from frozen glycerol stocks.
<ol type="a">
<li>Streak out the query strains on YPD agar plates containing 200 μg/mL Hyg and incubate at 30°C for 2–3 days.</li>
<li>Inoculate a single colony into 5 mL of YPD liquid medium. Incubate at 30°C for 2 days using a culture rotator (40 rpm).</li>
<li>Prepare the query strain lawns to start the screen.
<ol type="i">
<li>Using a spreader, evenly spread 800 μL of each saturated liquid culture onto a single-well YPD agar plate.</li>
<li>Repeat and prepare a total of thirteen query strain lawns for a genome-wide screen with thirteen condensed 384-density format deletion arrays.</li>
<li>Allow the lawns to dry and incubate at 30°C for 2 days.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Perform both <i>IRE1</i> and <i>IRE1</i>ΔLD query strain screens in parallel for fair comparison of downstream data.</div>
<ol start="2">
<li>Mate query strains with the gene deletion arrays (<a href="#fig2">Figures 2</a> and <a href="#fig3">3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1054-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Reporter-synthetic genetic array functional genomic screen for UPR activation</div>
<p>(A) A <i>MAT</i>α query strain containing either one of the two integrated recombinant UPR sensors selectable by Hygromycin (Hyg) due to the dominant <i>Hyg</i><sup><i>R</i></sup> selectable marker. The query strain also possesses a <i>MAT</i>a-specific reporter (<i>can1</i>Δ<i>::STE2pr-</i>Sp<i>_his5</i>), a recessive marker (<i>lyp1</i>Δ) to counter-select diploid population, and the UPR reporter with auxotrophic markers <i>MET15</i> and <i>URA3</i>. Each yeast strain of the <i>MAT</i>a array inoculated from the yeast deletion library contains a single gene deletion replaced by the dominant <i>Kan</i><sup><i>R</i></sup> selectable marker. The query strain is mated to the array strains on YPD agar plates.</p>
<p>(B) Successfully mated strains yield heterozygous diploid with <i>Hyg</i><sup><i>R</i></sup> and <i>Kan</i><sup><i>R</i></sup> phenotype that grow on YPD agar plates containing Hyg and G418.</p>
<p>(C) Sporulation of the diploid strains occur upon transfer onto the enriched sporulation agar plates, resulting in the formation of haploid meiotic progeny.</p>
<p>(D) Selection of the <i>MAT</i>a meiotic haploid progeny on SD-His/Arg/Lys/Ura/Met agar plates containing canavanine/thialysine.</p>
<p>(E) <i>Kan</i><sup>R</sup> meiotic haploid progeny selection on SD<sub>MSG</sub>-His/Arg/Lys/Ura/Met agar plates containing canavanine/thialysine/G418.</p>
<p>(F) <i>Kan</i><sup>R</sup> (<i>xxx</i>Δ) and <i>Hyg</i><sup>R</sup> (<i>IRE1</i> or <i>IRE1</i>Δ<i>LD)</i> haploids are selected with the same media as in (E) with the additional supplementation of Hyg (SD<sub>MSG</sub>-His/Arg/Lys/Ura/Met containing canavanine/thialysine/G418/Hyg).</p>
</figcaption>
</figure>
<ol type="a">
<li>Pin the query strain (<i>MAT</i>α, <i>Hyg</i><sup><i>R</i></sup>) from the lawn onto fresh single-well YPD agar plates using the 384 Solid Pin Multi-Blot Replicator.</li>
<li>Pin the 384-density deletion library (<i>MAT</i>a, <i>Kan</i><sup><i>R</i></sup>) on top of the query strain.</li>
<li>Incubate at ∼22°C for 1 day.</li>
</ol>
</li>
<li>Select for diploids by pinning the colonies onto single-well YPD agar plates containing G418/Hyg. Incubate the plates 2 days at 30°C.</li>
<li>Sporulate the selected diploids by pinning the colonies onto single-well enriched sporulation agar plates. Incubate 7 days at 22°C.</li>
<li>Select for <i>MAT</i>a meiotic haploid progenies by pinning the spores onto single-well SD-His/Arg/Lys/Ura/Met agar plates containing canavanine/thialysine. Incubate at 2 days at 30°C.</li>
<li>Select for <i>MAT</i>a, <i>Kan</i><sup><i>R</i></sup> meiotic haploid progenies by pinning the spores onto single-well SD<sub>MSG</sub>-His/Arg/Lys/Ura/Met agar plates containing canavanine/thialysine/G418 solid medium. Incubate 2 days at 30°C.</li>
<li>Select for <i>MAT</i>a, <i>Kan</i><sup><i>R</i></sup>, <i>Hyg</i><sup><i>R</i></sup> meiotic haploid progenies by pinning the haploids onto single-well SD<sub>MSG</sub>-His/Arg/Lys/Ura/Met agar plates containing canavanine/thialysine/G418/Hyg. Incubate at 2 days at 30°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The resulting selected progenies will contain either <i>IRE1</i> or <i>IRE1</i>Δ<i>LD</i>, marked by <i>Hyg</i><sup><i>R</i></sup>, and a <i>Kan</i><sup><i>R</i></sup> cassette that replace a non-essential gene.</div>
<div class="note">
<span class="note-title">Note:</span> Ammonium sulfate interferes with G418 selection. It is replaced with monosodium glutamic acid (MSG) at steps 6 and 7.</div>
<h3 id="sec3.2">Measurement of reporter fluorescence levels by flow cytometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="8">
<li>Preparation of cell cultures for flow cytometry
<ol type="a">
<li>Inoculate the selected haploid progenies from the 384-density format arrays into four 96 well plates using the 96 Floating E-Clip style Pin Multi-Blot Replicator. Each well should be prefilled with 200 μL of YPD liquid media (<a href="#fig2">Figure 2</a>).</li>
<li>Grow 12–16 h at 30°C to allow cell cultures to reach saturation.</li>
<li>Using a multi-channel pipette, dilute the cell cultures to a density of 0.05 OD/mL while maintaining the 200 μL volume in each well.</li>
<li>Incubate the diluted cells 4.5 h at 30°C.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The absorbance at 600 nm should not exceed 0.4 OD/mL to prevent UPR activation in these cells.</div>
<ol start="9">
<li>Measure the fluorescence levels of GFP and mCherry reporters from each well with an automated high-throughput sampler (HTS) connected to the LSRFortessa X-20.
<ol type="a">
<li>It is important to resuspend the cells with a multi-channel pipette before reading with the flow cytometer to prevent clogging of the instrument.</li>
<li>Acquire cells at low speed.</li>
<li>Excite GFP and mCherry at 488 and 561 nm and collect through a 505 and 595 nm long-pass filter and a 530/30 and 610/20 band pass filter, respectively.</li>
<li>One 96-well plate is read within 5 min, with a read of 10,000 cells per well.</li>
<li>Use the software FACSDiVA v 8.0 to acquire data in .fcs file format. Read files with the software FlowJo 10.8.0 (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1054-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Typical flow cytometry gating workflow to quantify UPR activity</div>
<p>(A) Density plot of the recorded events was gated (gate 1, left panel) to include most intact or viable cells but not ruptured or dead cells. This population was then replotted in another density plot (right panel) where the sequential gate 2 was defined to isolate single cells. Only the gate 2 subpopulation was included in the subsequent fluorescence quantification.</p>
<p>(B) Overlay histograms displaying the fluorescent activity of the constitutive reporter mCherry (left panel) and the inducible UPR reporter GFP (right panel) in the gate 2 subpopulation of <i>IRE1</i> query strain. In this example, the blue population was treated with DMSO (carrier) and red population with the UPR inducer, DTT, respectively. The median of the GFP was normalized to the median of mCherry to determine the relative UPR activity.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The potent UPR inducer gene deletion <i>opi3</i>Δ strain and wild-type (WT) strains are included in each 96 well plate. Typically, each library microplates contain at least 2 empty wells that were used to include positive (<i>opi3</i>Δ) and negative (WT) controls. <i>OPI3</i> deletion was selected because it causes high UPR by lipid bilayer stress in both query strains <i>IRE1</i> and <i>IRE1ΔLD</i> (<a href="#bib3">Ho et al., 2020</a>).</div>
<h3 id="sec3.3">Data analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="10">
<li>Normalizing the GFP fluorescence levels using Microsoft Excel.
<ol type="a">
<li>Calculate the GFP/mCherry ratio signal for each mutant (m) strain (GFP<sub>m</sub>/mCherry<sub>m</sub>).</li>
<li>Calculate the median of GFP/mCherry ratio signal for each plate (GFP<sub>median</sub>/mCherry<sub>median</sub>)</li>
<li>Calculate the UPR reporter level of each mutant strain (Equation <a href="#fd1">1</a>)
<p id="fd1" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 1)</mtext></mtd> <mtd> <mrow> <mi>U</mi> <mi>P</mi> <mi>R</mi> <mi>r</mi> <mi>e</mi> <mi>p</mi> <mi>o</mi> <mi>r</mi> <mi>t</mi> <mi>e</mi> <mi>r</mi> <mi>l</mi> <mi>e</mi> <mi>v</mi> <mi>e</mi> <mi>l</mi> <mi>s</mi> <mo>=</mo> <mi>l</mi> <mi>o</mi> <mi>g</mi> <mn>2</mn> <mrow> <mo>[</mo> <mrow> <mrow> <mo>(</mo> <mfrac> <mrow> <mi>G</mi> <mi>F</mi> <msub> <mi>P</mi> <mi>m</mi> </msub> </mrow> <mrow> <mi>m</mi> <mi>C</mi> <mi>h</mi> <mi>e</mi> <mi>r</mi> <mi>r</mi> <msub> <mi>y</mi> <mi>m</mi> </msub> </mrow> </mfrac> <mo>)</mo> </mrow> <mi>x</mi> <mrow> <mo>(</mo> <mfrac> <mfrac> <mn>1</mn> <mrow> <mi>G</mi> <mi>F</mi> <msub> <mi>P</mi> <mrow> <mi>m</mi> <mi>e</mi> <mi>d</mi> <mi>i</mi> <mi>a</mi> <mi>n</mi> </mrow> </msub> </mrow> </mfrac> <mrow> <mi>m</mi> <mi>C</mi> <mi>h</mi> <mi>e</mi> <mi>r</mi> <mi>r</mi> <msub> <mi>y</mi> <mrow> <mi>m</mi> <mi>e</mi> <mi>d</mi> <mi>i</mi> <mi>a</mi> <mi>n</mi> </mrow> </msub> </mrow> </mfrac> <mo>)</mo> </mrow> </mrow> <mo>]</mo> </mrow> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
</li>
<li>Exclude any knockout candidates from the analysis with cell counts of 350 cells/well or less.</li>
<li>UPR reporter levels equal or more than 1.5-fold change were considered positive hits.</li>
</ol>
</li>
</ol>
<ol start="11">
<li>Validation of selected positive hits
<ol type="a">
<li>Grow each selected strain in 500 μL of YPD media to a cell density of up to an absorbance of 0.4 OD/mL.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The selected strains can be stored at −80°C as glycerol stocks until ready to proceed to next steps.</div>
<ol type="i">
<li>Measure the relative GFP fluorescence using the LSRFortessa X-20 flow cytometer by exciting GFP and mCherry at 488 and 561 nm and collect through a 505 and 595 nm long-pass filter and a 530/30 and 610/20 band pass filter, respectively (<a href="#fig5">Figures 5</a>A and 5B).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1054-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Validation of positive hits from the screens</div>
<p>(A and B) Typical validation of positive hits by flow cytometry. The reporter-synthetic genetic array mutants carrying single deletion of <i>SCJ1</i> and <i>SPC2</i>, respectively, were passed through the flow cytometer. Indicated by the GFP gate, disruption of <i>SCJ1</i> resulted in strong UPR activation in both query strains <i>IRE1</i> and <i>IRE1</i>Δ<i>LD</i> (top left panel), whereas disruption of <i>SPC2</i> weakly induced the UPR in both of the query strains (middle left panel). The signal of the reference reporter mCherry was consistent across the samples’ right panels. The data was analyzed using the same strategy as described in <a href="#fig3">Figure 3</a>.</p>
<p>(C and D) Typical validation of positive hits by confocal fluorescence microscopy. The same reporter-synthetic genetic array mutants from <a href="#fig4">Figure 4</a>A were validated by confocal microscopy. A strong cytosolic expression of GFP was observed in the <i>scj1</i>Δ mutants and a relatively weaker GFP signal was observed in the <i>spc2</i>Δ mutants. Scale bar: 10 μm.</p>
</figcaption>
</figure>
</li>
<li>Place cells on slides coated with 1 mg/mL ConA mounted onto Attofluor cell chambers. Image cells using a Zeiss LSM 710 confocal microscope with a 100× 1.4 NA oil plan-Apochromat objective (<a href="#fig5">Figures 5</a>C and 5D).</li>
</ol>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Disruption of ER membrane integrity activates the UPR in an Ire1 luminal domain independent manner. This protocol is designed to identify gene deletions that activate the UPR to identify cellular processes necessary for ER membrane integrity by comparing the <i>IRE1</i> and <i>IRE1ΔLD</i> query strains. Out of the 4,847 mutants screened, we identified 629 and 958 gene deletions that activate the UPR in an Ire1 LD-dependent and -independent manner, respectively (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1054-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Number of deletion mutants activating the UPR in both query strains</div>
<p>From the screen, 448 gene deletions activated the UPR in <i>IRE1</i> query strain, 777 activated the UPR in <i>IRE1ΔLD</i> query strain while 181 gene deletions activated the UPR in both query strains.</p>
</figcaption>
</figure>
<p>We expect most deleted genes identified in <i>IRE1</i> query strains to exhibit functions related to protein folding and protein quality control at the ER. For instance, functions should include protein <i>N-</i> and <i>O-</i>glycosylation, ER-associated protein degradation (ERAD). Additionally, genes with functions related to protein trafficking and lipid biosynthesis should be identified.</p>
<p>Out of the 181 genes that specifically activated the UPR by lipid bilayer stress, we found that loss of <i>ARV1</i>, <i>GET1</i>, <i>PMT2, OPI3</i>, <i>SCJ1</i>, <i>SPC2</i>, and <i>STE24</i> activated the UPR independently of Ire1 LD as previously reported (<a href="#bib6">Promlek et al., 2011</a>). The deletion of other genes such as vacuolar protein sorting <i>VPS8</i>, <i>VPS29</i>, <i>VPS61</i>, <i>VPS63</i>, and <i>VPS72</i>, highly activated the UPR, confirming their requirement for ER membrane integrity (<a href="#fig7">Figure 7</a>) (<a href="#bib5">Markgraf et al., 2009</a>). Another important process is the ERAD machinery. The ERAD component Hrd1 forms a ubiquitin-gated protein conducting channel for the retro-translocation of misfolded ER luminal protein across the ER lipid bilayer. Given that Hrd1-Hrd3 is part of an ER membrane integrated complex (<a href="#bib7">Schoebel et al., 2017</a>; <a href="#bib8">Wu et al., 2020</a>), this complex might regulate ER membrane integrity. In conclusion, our findings suggest that components of vesicular trafficking and the ERAD are necessary to maintain ER membrane integrity.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1054-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Deletion of genes encoding proteins linked to diverse ER functions activate the UPR by LBS</div>
<p>Heatmap representing the levels of UPR activation caused by the deletion of selected genes in <i>IRE1</i> and <i>IRE1</i>Δ<i>LD</i> query strains. In summary, these genes have been classified into functions associated to the proteostasis network, multiple steps of vesicular trafficking, fatty acid and phosphatidylcholine (PC) biosynthesis. ER stress occurs possibly through membrane associated aberrancies causing UPR activation. These genes were selected and classified based on their gene ontology functions (ER membrane associated), involvement in the UPR and on the degree of UPR activation.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This screen was performed manually which made it technically challenging to complete in one iteration, considering both query strains had to be screened in parallel. As a result, we first screened deletion library plates 1 to 20, followed by deletion library plates 21 to 40 and lastly, deletion library plates 41 to 51 and 70 to 71. Therefore, it is important to include a negative and positive control to each screened deletion library plate. A researcher would benefit from having access to a robotic pinner and accompanying software to accelerate and ease this part of the protocol.</p>
<p>Genes that are essential in the absence of Ire1 LD will be missing from the screen of <i>IRE1</i>Δ<i>LD</i> query strain. As the UPR will not be activated in <i>IRE1</i>Δ<i>LD</i> query strain during proteotoxic stress, some diploids might not yield any selected progenies due to synthetic lethality or the growth of the resulting progeny might be beyond optimal for the screen.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Difficulty in generating transformed query strains (before you begin section, step 1).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Ensure that YMS612 cells are freshly activated from glycerol stock and then re-inoculated and grown to an absorbance of 0.6–0.8 OD/mL at 600 nm before transformation. Use freshly made reagents for all transformation steps.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Cell density is too low for fluorescence confocal acquisition (before you begin section, step 2; step-by-step method details section, step 11).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>The absorbance at 600 nm of the cell culture should never exceed 0.4 OD/mL to avoid the activation of the UPR in unstressed cells. For slow growing mutants, concentrate the cells by spinning down a large volume of the cell suspension at low speed and resuspend in a smaller volume before adhering cells to the coated slides.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Uneven transfer of yeast cells during the pinning steps (before you begin section, step 4; step-by-step method details section, steps 2–7).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Avoid using excessive agar media when making the agar plate as it may lead to uneven surface along the edges, causing the pins to puncture the agar. Otherwise, a floating-pin replicator can be used to facilitate the pinning steps.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Contamination during the SGA steps (step-by-step method details section, steps 1–7).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Even with applied aseptic techniques, contamination may still occur due to the relatively long period of time required for the completion of SGA. The antibiotics used for selections throughout the SGA procedure can be optimized by the user. However, it is important to avoid using antibiotics in excess during SGA as some mutants may exhibit sensitivity to certain antibiotics.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>High level of basal UPR activity causes false positive hits (step-by-step method details section, step 8).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Cell density should not exceed 0.4 OD/mL (absorbance at 600 nm) for flow cytometry. Subculture the cells if they have grown beyond 0.4 OD/mL or re-inoculate liquid media from the mutant arrays. Overgrowing cells will induce the UPR even in wild-type cells due to nutrient depletion. Overgrown cells should be diluted at lower density and grown for at least 8 h without exceeding 0.4 OD/mL to allow the UPR to be deactivated to the basal level. It should also be noted that the query strains induce high expression of free GFP proteins which in turn are very stable regardless of UPR deactivation. Therefore, we highly recommend to re-inoculate the media from the mutant arrays.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Baryshnikova, A., Costanzo, M., Dixon, S., Vizeacoumar, F.J., Myers, C.L., Andrews, B., and Boone, C. (2010). Synthetic genetic array (SGA) analysis in Saccharomyces cerevisiae and Schizosaccharomyces pombe. Methods Enzymol. <i>470</i>, 145-179. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref1">View at publisher</a></p>
<p id="bib2">Haass, F.A., Jonikas, M., Walter, P., Weissman, J.S., Jan, Y.N., Jan, L.Y., and Schuldiner, M. (2007). Identification of yeast proteins necessary for cell-surface function of a potassium channel. Proc. Natl. Acad. Sci. U S A. <i>104</i>, 18079-18084. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref2">View at publisher</a></p>
<p id="bib3">Ho, N., Yap, W.S., Xu, J., Wu, H., Koh, J.H., Goh, W.W.B., George, B., Chong, S.C., Taubert, S., and Thibault, G. (2020). Stress sensor Ire1 deploys a divergent transcriptional program in response to lipid bilayer stress. J. Cell Biol <i>219</i>, e201909165. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref3">View at publisher</a></p>
<p id="bib4">Jonikas, M.C., Collins, S.R., Denic, V., Oh, E., Quan, E.M., Schmid, V., Weibezahn, J., Schwappach, B., Walter, P., Weissman, J.S., and Schuldiner, M. (2009). Comprehensive characterization of genes required for protein folding in the endoplasmic reticulum. Science <i>323</i>, 1693-1697. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref4">View at publisher</a></p>
<p id="bib5">Markgraf, D.F., Ahnert, F., Arlt, H., Mari, M., Peplowska, K., Epp, N., Griffith, J., Reggiori, F., and Ungermann, C. (2009). The CORVET subunit Vps8 cooperates with the Rab5 homolog Vps21 to induce clustering of late endosomal compartments. Mol. Biol. Cell <i>20</i>, 5276-5289. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref5">View at publisher</a></p>
<p id="bib6">Promlek, T., Ishiwata-Kimata, Y., Shido, M., Sakuramoto, M., Kohno, K., and Kimata, Y. (2011). Membrane aberrancy and unfolded proteins activate the endoplasmic reticulum stress sensor Ire1 in different ways. Mol. Biol. Cell <i>22</i>, 3520-3532. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref6">View at publisher</a></p>
<p id="bib7">Schoebel, S., Mi, W., Stein, A., Ovchinnikov, S., Pavlovicz, R., Dimaio, F., Baker, D., Chambers, M.G., Su, H., Li, D., et al. (2017). Cryo-EM structure of the protein-conducting ERAD channel Hrd1 in complex with Hrd3. Nature <i>548</i>, 352-355. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref7">View at publisher</a></p>
<p id="bib8">Wu, X., Siggel, M., Ovchinnikov, S., Mi, W., Svetlov, V., Nudler, E., Liao, M., Hummer, G., and Rapoport, T.A. (2020). Structural basis of ER-associated protein degradation mediated by the Hrd1 ubiquitin ligase complex. Science <i>368</i>, eaaz2449. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00574-8/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Guillaume Thibault (<a href="mailto:thibault@ntu.edu.sg">thibault@ntu.edu.sg</a>).</p>
<h4>Materials availability</h4>
<p>Strains and plasmids described in this study are available upon request.</p>
<h4>Data and code availability</h4>
<p>This study did not generate nor analyze datasets.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the Nanyang Assistant Professorship Program from <a href="https://doi.org/10.13039/501100001475">Nanyang Technological University</a>, the <a href="https://doi.org/10.13039/501100001381">National Research Foundation Singapore</a> under its National Research Foundation of Singapore together with the <a href="https://doi.org/10.13039/501100001809">National Natural Science Foundation of China</a> (NRF-NSFC) joint research grant call (NRF2018NRFNSFC003SB-006) and the <a href="https://doi.org/10.13039/501100001459">Ministry of Education – Singapore</a> Academic Research Fund Tier 1 (2019-T1-002-011), and the Nanyang Technological University Research Scholarship to N.H. (predoctoral fellowship).</p>
<h3>Author contributions</h3>
<p>Conceptualization, G.T.; methodology, N.H. and W.S.Y.; investigation, N.H. and W.S.Y.; writing – original draft, N.H., W.S.Y., and G.T.; writing – review &amp; editing, W.S.Y. and G.T.; funding acquisition, N.H. and G.T.; supervision, G.T.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>