<article>
<section>
<h1 id="header">Evaluating cytokine production by flow cytometry using brefeldin A in mice</h1>
<p><time datetime="2021-01-08">Published: January 8, 2021</time></p>
<p>Stephen B. Kovacs,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">6</a>,<a href="#cor1">*</a></sup> Changhoon Oh,<sup><a href="#aff4">4</a></sup> Youssef Aachoui,<sup><a href="#aff4">4</a>,<a href="#fn1">5</a></sup> and Edward A. Miao<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">5</a>,<a href="#fn3">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Immunology, Duke University, Durham, NC 27710, USA</p>
<p id="aff2"><sup>2</sup>Department of Molecular Genetics and Microbiology, Duke University, Durham, NC 27710, USA</p>
<p id="aff3"><sup>3</sup>Department of Microbiology and Immunology, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</p>
<p id="aff4"><sup>4</sup>Department of Microbiology and Immunology, Center for Microbial Pathogenesis and Host Responses, University of Arkansas for Medical Sciences, Little Rock, AR 72205, USA</p>
<p id="fn1"><sup>5</sup>These authors contributed equally</p>
<p id="fn2"><sup>6</sup>Technical contact</p>
<p id="fn3"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:stephen_kovacs@med.unc.edu">stephen_kovacs@med.unc.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:edward.miao@duke.edu">edward.miao@duke.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100244">10.1016/j.xpro.2020.100244</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Characterizing cytokine production <i>in situ</i> is important for properly understanding immunologic responses. Cytokine reporter mice are limited by the need to cross markers into various knockout backgrounds and by availability of reporters of interest. To overcome this, we utilize injection of brefeldin A into mice to enable flow cytometric analysis of <i>in situ</i> cytokine production during a bacterial infection. While we evaluate IFN-γ production during <i>Burkholderia thailandensis</i> infection, this protocol can be applied to other cytokines and other mouse models.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib4">Kovacs et al. (2020)</a> and <a href="#bib5">Liu and Whitton (2005)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Brefeldin A and flow cytometry can be used to detect cytokine production in mice</li>
<li>This methodology can be applied to mice of any genotype without extra breeding</li>
<li>We discuss how to apply this protocol to other infectious and immune mouse models</li>
<li>This protocol thoroughly discusses advantages and limitations of this methodology</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/344-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The investigator should be comfortable with any procedures involved with their particular in vivo model. Additionally, investigators should be proficient with injecting mice via the intravenous route, and the use of a flow cytometer with large multi-parameter capabilities (e.g., LSR II).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Alexa Fluor (AF) 488 rat anti-mouse CD127 (clone A7R34)</td>
<td>BioLegend</td>
<td>Cat# 135017</td>
</tr>
<tr>
<td>Brilliant Violet (BV) 570 rat anti-mouse CD3 (clone 17A2)</td>
<td>BioLegend</td>
<td>Cat# 100225</td>
</tr>
<tr>
<td>APC-Cy7 rat anti-human/mouse CD45R (clone RA3-6B2)</td>
<td>BioLegend</td>
<td>Cat# 103223</td>
</tr>
<tr>
<td>APC-Cy7 rat anti-mouse Ly6G (clone 1A8)</td>
<td>BioLegend</td>
<td>Cat# 127623</td>
</tr>
<tr>
<td>PE-Cy7 rat anti-mouse CD90.2 (clone 53-2.1)</td>
<td>BioLegend</td>
<td>Cat# 140309</td>
</tr>
<tr>
<td>BV650 mouse anti-mouse NK1.1 (clone PK136)</td>
<td>BioLegend</td>
<td>Cat# 108735</td>
</tr>
<tr>
<td>BV421 Armenian hamster anti-mouse TCRγδ (clone GL3)</td>
<td>BioLegend</td>
<td>Cat# 118119</td>
</tr>
<tr>
<td>AF594 rat anti-mouse CD4 (clone GK1.5)</td>
<td>BioLegend</td>
<td>Cat# 100446</td>
</tr>
<tr>
<td>PE rat anti-mouse CD8 (clone 53-6.7)</td>
<td>BioLegend</td>
<td>Cat# 100708</td>
</tr>
<tr>
<td>APC rat anti-mouse IFN-γ (clone XMG1.2)</td>
<td>BioLegend</td>
<td>Cat# 505810</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>Burkholderia thailandensis,</i> strain E264-1 (ATCC 700388 strain E264 that was previously passaged through <i>Casp1</i><sup>−/−</sup><i>Casp11</i><sup>−/−</sup> mice)</td>
<td><a href="#bib11">Aachoui et al., 2015</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>LIVE/DEAD Fixable Blue Dead Cell Stain Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# L23105</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: Wild type (WT): C57BL/6J</td>
<td>The Jackson Laboratory</td>
<td>JAX# 000664</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo (we used version 9.9.6 for all figures in this protocol but recommend the most recent version, version 10.7)</td>
<td>TreeStar</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Fixation/Permeabilization Solution kit</td>
<td>BD Biosciences</td>
<td>554714</td>
</tr>
<tr>
<td>Brilliant Stain Buffer</td>
<td>BD Biosciences</td>
<td>563794</td>
</tr>
<tr>
<td>UltraComp eBeads Compensation Beads</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 01-2222</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Extracellular antibody master mix</caption>
<thead>
<tr>
<th>Antibody</th>
<th>Dilution factor</th>
</tr>
<tr>
<th>Diluent</th>
<th>Volume per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>BV650 anti-mouse NK1.1</td>
<td>1:80</td>
</tr>
<tr>
<td>BV570 anti-mouse CD3</td>
<td>1:40</td>
</tr>
<tr>
<td>BV421 anti-mouse TCRγδ</td>
<td>1:40</td>
</tr>
<tr>
<td>AF488 anti-mouse CD127</td>
<td>1:320</td>
</tr>
<tr>
<td>PE-Cy7 anti-mouse CD90</td>
<td>1:400</td>
</tr>
<tr>
<td>APC-Cy7 anti-mouse Ly6G</td>
<td>1:1280</td>
</tr>
<tr>
<td>APC-Cy7 anti-human/mouse CD45R</td>
<td>1:320</td>
</tr>
<tr>
<td>AF594 anti-mouse CD4 (optional)</td>
<td>1:400</td>
</tr>
<tr>
<td>PE anti-mouse CD8 (optional)</td>
<td>1:160</td>
</tr>
<tr>
<td>Brilliant Stain Buffer</td>
<td>50 μL</td>
</tr>
<tr>
<td>FACS Buffer</td>
<td>Add until total volume is 100 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> CD4 and CD8 antibodies are included as optional because these antibodies did provide adequate delineation of CD4 and CD8 T cell populations, the fluorophore-marker combinations were not optimized, resulting in minor compensation issues. See <a href="#expected-outcomes">Expected outcomes</a> for more details.</div>
<div class="note">
<span class="note-title">Note:</span> All sample data included in figures in this protocol used FACS buffer as diluent without Brilliant Stain Buffer added. However, we recommend using Brilliant Stain Buffer in this panel or any other panel that utilizes multiple Brilliant Violet dyes to avoid potential nonspecific staining due to aggregation of these dyes that could affect your interpretation of your data. FACS buffer should be used when your panel includes one or fewer Brilliant Violet or other polymer-based dyes.</div>
<table id="undtbl2">
<caption>Intracellular antibody master mix</caption>
<thead>
<tr>
<th>Antibody</th>
<th>Dilution factor</th>
</tr>
</thead>
<tbody>
<tr>
<td>APC anti-mouse IFN-γ</td>
<td>1:400</td>
</tr>
<tr>
<td><b>Diluent</b></td>
<td>BD Biosciences Perm/Wash buffer</td>
</tr>
<tr>
<td>Total volume (per sample)</td>
<td>100 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>ACK lysis buffer recipe</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NH<sub>4</sub>Cl</td>
<td>150 mM</td>
<td>4.01 g</td>
</tr>
<tr>
<td>KHCO<sub>3</sub>
</td>
<td>10 mM</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>EDTA</td>
<td>0.1 mM</td>
<td>18.6 mg</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>500 mL</b></td>
</tr>
<tr>
<td colspan="3">Titrate pH to 7.2–7.4; Store at room temperature (25°C–27°C)</td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>FACS buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EDTA (0.5 M)</td>
<td>4 mM</td>
<td>4 mL</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>0.5%</td>
<td>2.5 g</td>
</tr>
<tr>
<td>PBS</td>
<td>N/A</td>
<td>496 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>500 mL</b></td>
</tr>
<tr>
<td colspan="3">Store at 4°C</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Prepare bacterial inoculum</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>For this protocol, we will describe staining for <i>in situ</i> cytokine production by lymphocytes during <i>Burkholderia thailandensis</i> infection. The inoculation protocol should be adjusted to fit the infection or treatment protocol of interest.</p>
<ol>
<li>Grow a bacterial culture.
<ol type="a">
<li>Two to three days prior to infection, streak out <i>Burkholderia thailandensis</i> from frozen stock onto a Luria-Bertani (LB) plate, and grow for one to two days at 37°C.
<div class="note">
<span class="note-title">Note:</span> Plates of streaked <i>B. thailandensis</i> can be stored for up to a week at 4°C.</div>
</li>
<li>Inoculate a 1.5 mL liquid LB culture with a single colony and grow for 16 h at 37°C in a shaking incubator to achieve stationary phase.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> If mice will not be infected immediately at the end of this 16 h incubation, place the bacteria on ice until ready to infect mice.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Prepare the inoculum.
<ol type="a">
<li>Pellet the 1.5 mL liquid culture of bacteria by spinning in a microcentrifuge at 3,000 × <i>g</i> for 1 min.</li>
<li>Remove the supernatant and resuspend in 1 mL PBS to wash the bacteria. Spin at 3,000 × <i>g</i> for 1 min. Repeat for a total of two PBS washes. Resuspend in 1 mL PBS.</li>
<li>Estimate the concentration of bacteria using an OD<sub>600</sub> as determined using a spectrometer.
<ol type="i">
<li>An OD<sub>600</sub> of 1.00 is approximately 1 × 10<sup>9</sup> CFU/mL of <i>B. thailandensis.</i> The relationship between OD<sub>600</sub> and CFU is approximately linear below OD<sub>600</sub> of 1.00. Dilute the bacteria until its OD<sub>600</sub> is less than 1.00.</li>
<li>Calculate concentration using the following equation:<br> Concentration (in CFU/mL) = OD<sub>600</sub> × 1 × 10<sup>9</sup> CFU/mL x dilution factor</li>
</ol>
</li>
<li>Dilute the inoculum with PBS to 2 × 10<sup>7</sup> CFU per 200 μL (the volume for each inoculum injection).</li>
<li>Verify inoculum via spot dilution plating.
<ol type="i">
<li>Create a total of eight (including undiluted) 1:10 serial dilutions of the inoculum in PBS.</li>
<li>Plate 10 μL of each dilution in triplicate on an LB plate using a spot plating method (or plate 100 μL if using a whole plate method).</li>
<li>Incubate overnight (14–16 h) at 37°C.</li>
<li>Looking at the dilution which has approximately 10–50 colonies per spot, count and average the number of colonies per spot.</li>
<li>Calculate actual inoculum using the following formula:<br> Inoculum (in CFU/mouse) = average # colonies x dilution factor x 20</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Infect and treat mice with brefeldin A</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>We next infect the mice and, after waiting enough time to allow immune responses to establish, treat mice with the Golgi blocker brefeldin A (BFA). Since most cytokines are processed and secreted through the classical secretory pathway that passes through the Golgi apparatus, BFA prevents the secretion of cytokines, causing their accumulation within the immune cells for eventual detection by flow cytometry. Thus, treating mice with BFA allows for the evaluation of <i>in situ</i> cytokine production without requiring the use of reporter mice. Consequently, mice of any genetic background can be used, allowing the evaluation of the effects of knocking out genes on cytokine production without prior crossing of knockout mice onto a reporter background. For example, we successfully evaluated cytokine production in both wild type and <i>Rag1</i><sup>−/−</sup> mice (<a href="#bib4">Kovacs et al., 2020</a>).</p>
<p>Our timings for this experiment were based on known kinetics of <i>B. thailandensis</i> infection in wild type mice. The time point in each mouse model will vary, but the timing of injecting BFA 6 h prior to euthanizing the mice is important to allow for adequate accumulation of cytokines within cells.</p>
<ol start="3">
<li>Inject mice intraperitoneally with 200 μL of the inoculum created in step 2. Uninfected control mice should be injected with 200 μL of PBS as a mock infection.</li>
</ol>
<ol start="4">
<li>Twelve hours after infection, treat mice with BFA.
<ol type="a">
<li>Dilute BFA stock.
<ol type="i">
<li>BFA stock should be reconstituted in DMSO to a concentration of 20 mg/mL.
<div class="note">
<span class="note-title">Note:</span> BFA stock can safely be stored at −20°C for at least a year.</div>
</li>
<li>Further dilute working solution to 0.5 mg/mL in PBS. Keep this solution at room temperature (RT; 25°C–27°C) in order to avoid injecting a large volume of cold liquid into the mice. Do not reuse working solution.</li>
</ol>
</li>
<li>Inject 500 μL of working solution of BFA intravenously into the tail vein of both infected and uninfected mice.
<div class="note">
<span class="note-title">Note:</span> This volume was used in the original publication by Liu et al. (<a href="#bib5">Liu and Whitton, 2005</a>). We did not attempt to optimize the injection volume, as this volume worked for our studies. Considerations include whether the volume used enhances BFA exposure throughout the body or reduces cytotoxic effects by dilution of BFA (<a href="#sec7.3">Troubleshooting 2</a>).</div>
</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Six hours after injecting BFA, euthanize mice and promptly harvest spleens per your institution’s animal guidelines. Keep spleens in PBS and on ice until all samples are ready to process.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> While keeping the spleens in PBS is sufficient over a short duration when harvesting spleens and immediately proceeding to the next steps, consider keeping spleens in serum-free media such as DMEM for longer durations to avoid loss of cell viability. Cells can be maintained in this media until RBC lysis.</div>
<div class="note">
<span class="note-title">Note:</span> Until cells have been fixed, cells are still alive and actively producing cytokines, albeit in a manner no longer reflective of the <i>in situ</i> setting. While the cell populations and their cytokine productions are likely stable at this point for a short period, it is advisable to proceed to the next steps through cell fixation (step 27) as soon as possible.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The exact timing of these injections will depend on the infection and model being tested. In this case, we have optimized the time points of our infection (18 h total infection time) to view cytokine production during innate immune responses to <i>B. thailandensis</i> with a infective dose of 2 × 10<sup>7</sup> CFU. Indeed, if the inoculum of <i>B. thailandensis</i> is lowered to 1 × 10<sup>4</sup> CFU in wild type mice, the infection is cleared faster and we think that cytokine production returns to baseline by 18 h post infection, resulting in a lack of staining (<a href="#fig1">Figure 1</a>, compared to robust IFN-γ signal in <a href="#fig2">Figure 2</a>). Thus, it is important to optimize the timing of your BFA injections and tissue harvests for your specific model (<a href="#sec7.1">Troubleshooting 1</a>).</div>
<h3 id="sec3.3">Create a single cell suspension of splenocytes</h3>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/344-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Choosing correct experimental conditions and timing is essential for observing robust cytokine signal</div>
<p>When infecting with a large bacterial inoculum (2 × 10<sup>7</sup> CFU), a robust IFN-γ signal is observed at 18 h post infection. When using smaller bacterial inoculum (10<sup>4</sup> CFU), IFN-γ signal is similar to background levels at 18 h post infection, a time point at which infection with this lower inoculum has already been cleared.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/344-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Analysis of flow cytometric data</div>
<p>(A) Gating strategy used to characterize what cell types constitute the IFN-γ-producing cells. Italicized labels indicate what cell population is being gated.</p>
<p>(B) Representative results at 18 h post infection with 2 × 10<sup>7</sup> CFU <i>B. thailandensis</i>.</p>
<p>(C) Representative results demonstrating gating of ILC1s within the CD3<sup>−</sup>NK1.1<sup>+</sup> subset.</p>
<p>(D) Representative results demonstrating TCRγδ and CD4 and CD8 subpopulations of the CD3<sup>+</sup>NK1.1<sup>−</sup> subset when including the optional CD4 and CD8 antibodies. Panels from this figure are reused with permission from all authors from our previously published paper (<a href="#bib4">Kovacs et al., 2020</a>).</p>
</figcaption>
</figure>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>We next need to create a single cell suspension so that we can stain them for flow cytometry.</p>
<ol start="6">
<li>Place the spleen on a 70 micron cell strainer fitted on top of a 50 mL conical tube.</li>
<li>Using the plunger of a 1 mL syringe, gently press the spleen against the strainer to mechanically disrupt the tissue. Continue until the only remaining tissue on the strainer is white and fatty in texture, occasionally adding PBS to ensure that the tissue and cells do not dry out.</li>
<li>Wash the remaining cells off the strainer and into the underlying 50 mL conical using 10 mL of PBS. Dispose of the strainer and add PBS to the conical for a total volume of 30 mL.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If you are interested in immune cells present in other organs, the processing protocol may differ. Analogous procedures can be used for immune cell-dense organs like lymph nodes; however, more fibrous organs will likely require additional steps such as more intense physical manipulations and/or treatment with proteases like collagenase. For example, see the methods section of (<a href="#bib10">Weizman et al., 2017</a>) for descriptions of how to process mouse liver and lungs.</div>
<ol start="9">
<li>Spin the cells at 800 × <i>g</i>, 4°C, for 10 min using a standard centrifuge.</li>
<li>Discard the supernatant and resuspend in 5 mL of ACK RBC lysis buffer. Incubate on ice for 5 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The timing of incubating your cells for 5 min is important to avoid potential loss of cell viability (<a href="#sec7.3">Troubleshooting 2</a>).</div>
<ol start="11">
<li>Add 35 mL of PBS to stop lysis. Spin at 800 × <i>g</i>, 4°C, for 10 min using a standard centrifuge.</li>
<li>Discard the supernatant and resuspend in 5 mL of PBS.</li>
<li>Filter through a new 70 micron cell strainer fitted on top of a new 50 mL conical to remove aggregates of dead cells that formed after lysis.</li>
<li>
<b>Key step:</b> Count cells using a cell counter or hemocytometer and record the data. These counts will be used to aliquot for staining in the next steps as well as for calculating absolute counts of subsets in the final analyses.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If you desire to localize the cells that are producing cytokine within the organ, you can instead fix the spleen in 4% PFA and perform immunofluorescent staining using an analogous protocol published by Mazet et al (<a href="#bib6">Mazet et al., 2019</a>).</div>
<h3 id="sec3.4">Stain splenocytes and perform flow cytometric analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>We next stain our cells for flow cytometry. Under normal circumstances (without BFA treatment), most cytokines are synthesized and promptly secreted by the cell. This causes the total pool of cytokine protein within the cell to be very low, despite high transcription and translation. Brefeldin A halts intracellular vesicle formation and transportation of proteins from the endoplasmic reticulum to the Golgi apparatus, inhibiting the secretory pathway and resulting in accumulation of translated proteins within the cell. This greatly increases the sensitivity of detection with antibodies specific for the cytokine of interest.</p>
<ol start="15">
<li>Plate 1–2 × 10<sup>6</sup> cells in 200 μL PBS for each mouse into a well on a polypropylene, round-bottom 96-well plate.</li>
<li>Prepare a well for a live/dead (L/D) compensation tube.
<ol type="a">
<li>Take 1 to 2 million splenocytes and split into two Eppendorf tube.</li>
<li>Heat shock one of the tubes by heating it at 55°C for 5 min followed by 2 min on ice. The other tube should remain on ice the entire time.</li>
<li>Mix the cells from the two Eppendorf tubes together and plate onto the sample plate. This ensures that the compensation tube will have sizable populations of both live and dead cells to serve as positive and negative compensation controls.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Instead of using cells for L/D compensation, you can use amine reactive beads (for example, see Arc Amine Reactive Compensation Bead Kit (cat# A10628) from Thermo Fischer). If using these beads, you can make this compensation tube alongside the other compensation tubes in step 34.</div>
<ol start="17">
<li>Centrifuge the plate at 450 × <i>g</i>, RT, for 5 min. Discard the supernatant.</li>
<li>Resuspend each well with 200 μL of PBS. Centrifuge the plate at 450 × <i>g</i>, RT, for 5 min and discard the supernatant.</li>
<li>Resuspend each well with 100 μL of L/D stain in PBS for 20 min at RT in the dark
<ol type="a">
<li>To make L/D stain, add 1 μL of L/D stock per 1 mL of PBS</li>
</ol>
</li>
<li>Add 100 μL FACS buffer to each well. Centrifuge the plate at 450 × <i>g</i>, RT, for 5 min and discard the supernatant.</li>
<li>Resuspend each sample in 100 μL of extracellular staining master mix, and the L/D compensation well in 100 μL of PBS. Stain for 30 min at RT in the dark.</li>
<li>Add 100 μL of FACS buffer to each well. Centrifuge the plate at 450 × <i>g</i>, RT, for 5 min and discard the supernatant.</li>
<li>Resuspend each well with 200 μL of FBS buffer. Centrifuge the plate at 450 × <i>g</i>, RT, for 5 min and discard the supernatant.</li>
<li>Add 100 μL of BD Biosciences Fix/Perm to each well. Incubate at 4°C for 20 min in the dark.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the cytokine being stained, this fix/perm step may need to be extended to 45 min to 1 h to get optimal intracellular staining of the cytokine. For IFN-γ, we found that 20 min was sufficient (<a href="#sec7.1">Troubleshooting 1</a>).</div>
<ol start="25">
<li>Add 100 μL of BD Biosciences Perm/Wash buffer to each well. Centrifuge the plate at 1,250 × <i>g</i>, 4°C, for 5 min and discard the supernatant.</li>
<li>Resuspend each well with 200 μL of BD Biosciences Perm/Wash buffer. Centrifuge the plate at 1,250 × <i>g</i>, 4°C, for 5 min and discard the supernatant.</li>
<li>Resuspend each well with 200 μL of BD Biosciences Perm/Wash buffer.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Once all cells have been fixed, samples can be stored at 4°C in the dark overnight in the Perm/Wash buffer. Intracellular staining can be resumed the following day with no significant change in staining.</div>
<ol start="28">
<li>Centrifuge plate at 1,250 × <i>g</i>, 4°C, for 5 min and discard the supernatant.</li>
<li>Resuspend each sample in 100 μL of intracellular staining master mix and the L/D compensation well in 100 μL of BD Biosciences Perm/Wash buffer. Stain for 1 h at 4°C in the dark.</li>
<li>Resuspend each well with 100 μL of BD Biosciences Perm/Wash buffer. Centrifuge the plate at 1,250 × <i>g</i>, 4°C, for 5 min and discard the supernatant.</li>
<li>Resuspend each well with 200 μL of FBS buffer. Centrifuge the plate at 1,250 × <i>g</i>, RT, for 5 min and discard the supernatant.</li>
<li>Resuspend in 200 μL of FACS buffer. Transfer to a flow tube with a filter.</li>
</ol>
<ol start="33">
<li>Prepare compensation tubes.
<ol type="a">
<li>Add 1 μL of antibody to 1 drop of UltraComp eBeads Compensation Beads.
<div class="note">
<span class="note-title">Note:</span> These beads can bind mouse, rat, and hamster-derived antibodies, which works with all antibodies used in the panel included in this protocol. However, if you use different antibodies in your panel, ensure that the beads are compatible with all antibodies being used. Beads compatible with antibodies derived from other species are also commercially available. Alternatively, if enough cells present the marker for the antibody, you can stain splenocytes with the antibody alone to use as a compensation tube.</div>
</li>
<li>Vortex briefly.</li>
<li>Incubate at RT in the dark for 15 min.</li>
<li>Add 150 μL of FACS buffer and centrifuge at 450 × <i>g</i>.</li>
<li>Resuspend in 200 μL of FACS buffer and transfer to a flow tube.
<div class="note">
<span class="note-title">Note:</span> For conventional flow cytometry (such as the flow we performed using an LSRII), beads offer an easy way to generate compensation samples that usually result in accurate compensation. However, if using a full spectrum flow cytometer, then the slight differences between fluorescent spectra of fluorophores on beads compared to cells could result in unmixing errors. Thus, these users will need to determine if there are any differences between single color controls using beads and cells, and they should use cells as compensations controls if there is a difference.</div>
</li>
</ol>
</li>
</ol>
<ol start="34">
<li>Keep samples at 4°C in the dark until ready to run on a flow cytometer.</li>
<li>Run samples through flow cytometer.</li>
<li>Analyze results using FlowJo.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>If successful, using the gating schema displayed in <a href="#fig2">Figure 2</a>A, only minimal amounts of cytokine staining will be detected in uninfected controls, which represents the basal level of cytokine production (<a href="#sec7.5">Troubleshooting 3</a>). A robust signal should be detected from the infected mice where cells are strongly producing cytokine (<a href="#fig2">Figure 2</a>B). Frequency of cells can be determined directly from flow cytometry analyses. The gating schema used will enable analysis of T cells (CD3<sup>+</sup>), NK cells (CD3<sup>−</sup>NK1.1<sup>+</sup>CD127<sup>−</sup>), and ILC1s (CD3<sup>−</sup>NK1.1<sup>+</sup>CD127<sup>+</sup>CD90<sup>+</sup> or CD3<sup>−</sup>NK1.1<sup>−</sup>CD127<sup>+</sup>CD90<sup>+</sup>) (<a href="#fig2">Figures 2</a>A–2C). Gamma-delta T cells can also be identified within the CD3<sup>+</sup> population (<a href="#fig2">Figure 2</a>D).</p>
<p>We have also expanded the panel with CD4 and CD8 (see optional antibodies), allowing the identification of CD4 and CD8 T cells within the CD3<sup>+</sup> population. While compensations issues begin to appear since we did not optimize the fluorophore-marker combinations in the panel expanded to include CD4 and CD8, we include these antibodies in this protocol as an option since they were sufficient to delineate the CD4 and CD8 populations (<a href="#fig2">Figure 2</a>D). Further optimization of which fluorophore is assigned to each marker can be done to avoid these compensation issues.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Frequency is calculated using FlowJo analysis software. In addition to frequency, absolute counts for each population being examined should be calculated. This calculation is important because differences in frequency can result from either changes in that population of cells (e.g., more NK cells are present in the spleen) or changes in the other populations (e.g., fewer T cells will make NK cell frequency increase). Absolute counts allow determination of the causes of changes in frequencies. This can easily be performed by multiplying the total splenocyte count determined immediately after forming the single cell suspension for each sample (see step 14) by the frequencies of each gate used to gate on the population. For example, to calculate absolute number of NK cells in the spleen that produce IFN-γ using the gating from <a href="#fig2">Figure 2</a>A, we did the following:<br> Absolute # of NK cells producing IFN - γ in spleen =<br> 5.75 × 10<sup>7</sup> splenocytes × 0.976 (singlet freq) × 0.62 (lymphocyte freq) × 0.822 (live cell freq) × 0.286 (non - excluded cell freq) × 0.0234 (IFN - γ - producing cell freq) × 0.618 (NK cell freq)<br> = 1.18 10<sup>5</sup> NK cells</p>
<p>Using the “Frequency of Total” statistic in FlowJo for a given population, this equation can be simplified to:<br> Absolute count = splenocyte count x Freq of Total (for given population)</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>BFA is a Golgi blocker that inhibits the secretory pathway. This has several implications. First, BFA only inhibits release of cytokines that go through the secretory pathway. Members of the IL-1 family (e.g., IL-1β, IL-18, IL-33) do not encode a secretion signal, are stored in the cytosol, and are released by cell lysis or membrane permeabilization rather than through the secretory pathway (<a href="#bib3">Dinarello, 2017</a>). Therefore, this methodology will not assess release of these cytokines. Indeed, it would give the opposite result: positive staining would indicate the lack of release of IL-1 family members. Similarly, some cell types contain pre-formed cytokine not intended for immediate release. Notably, this includes mast cells which store pre-formed TNF until another stimulus promotes its release (<a href="#bib8">Mukai et al., 2018</a>). Neither the methodology reported here nor the use of cytokine reporter mice will be able to differentiate the pre-formed cytokine from cytokine that is created and immediately secreted on demand.</p>
<p>Second, release of cytokines, chemokines, and other proteins is important for a diverse array of immune functions. Thus, adding BFA likely alters kinetics and dynamics of infections and immune responses. However, these effects of BFA could be minimized by only treating mice with BFA after the immune response to be studied has had time to establish and by limiting the treatment time with BFA to 6 h.</p>
<p>In addition, as a Golgi blocker, BFA can be toxic to cells. It is therefore important to monitor for cell death using live/dead markers. In our <i>B. thailandensis</i> infection model, BFA toxicity appears minimal. However, it is possible that some populations of cells may be more sensitive to BFA in other murine models, which could result in a selective loss in that population that may be difficult to detect unless that particular cell type was specifically examined.</p>
<p>Finally, sensitivity for detecting a cytokine response will be limited by the robustness of the cytokine response, the size of the population being looked at, and the accessibility of the tissue to blood flow (and therefore to the BFA that is injected intravenously). Low prevalence of responding cells could be particularly apparent when examining non-lymphoid tissues. This could be rectified by enriching for immune cells and removing stromal cells. Furthermore, this is a limitation that likely affects this BFA method of assessing <i>in situ</i> cytokine production more than the use of a reporter mouse. The fluorophores used in reporter mice often have longer half-lives than those of cytokines, meaning that more can accumulate within individual cells (<a href="#bib1">Bouabe, 2012</a>; <a href="#bib2">Corish and Tyler-Smith, 1999</a>; <a href="#bib7">Mohrs et al., 2001</a>). This increases the signal and improves the sensitivity of the reporter mice compared to BFA-treated mice (at the cost of being less able to resolve cytokine secretion that occurred over shorter periods of time).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No cytokine signal</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Optimization may be required for each model to determine the time point that maximizes the likelihood for capturing a cytokine signal. With the proper timing, this protocol can be easily applied to a diverse array of models. The main stipulations are that you wait a sufficient amount of time to establish an immune response prior to BFA injection to minimize the effect of BFA on the immune response (see <a href="#limitations">Limitations</a> for more details), and you treat mice with BFA for approximately 6 h immediately prior to harvesting the mouse.</p>
<p>Another potential cause of a lack of cytokine signal is that BFA does not always efficiently block cytokine secretion. In these cases, another Golgi blocker, monensin, may be more successful at inhibiting secretion of the cytokine. Indeed, Sun et al. reported successfully treating mice intravenously with 0.5 mg of monensin in 500 μL of PBS instead of BFA to successfully visualize IL-10 production (<a href="#bib9">Sun et al., 2009</a>). For reference on which cytokines are known to be efficiently inhibited by brefeldin A versus monensin, see the Cytokine Detection subsection of the Techniques section on the following website: <a href="https://www.bdbiosciences.com/us/applications/research/intracellular-flow/">https://www.bdbiosciences.com/us/applications/research/intracellular-flow/</a>.</p>
<p>Finally, while we found 20 min fix/perm (step 24) was sufficient for staining of IFN-γ, this duration of time may be insufficient for other cytokines. Thus, consider extending the fix/perm step up to 45 min or 1 h.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Low cell number</p>
<h3 id="sec7.4">Potential solution</h3>
<p>If this is accompanied by excessive cell death per the live/dead staining, this could have resulted from lysing too long with ACK buffer. Large clumps forming after treatment with ACK buffer could suggest this as a culprit. In this case, ensure samples do not remain in ACK buffer too long. If working with many samples, lyse only a small group at once and then proceed to next group once PBS has been added to ACK-treated samples to stop reaction. Alternatively, using too high of a concentration of BFA for too long may result in cell toxicity. In this case, lower concentration of BFA used or see if treating mice with BFA for a shorter duration reduces cytotoxicity without significantly reducing the signal of your cytokine.</p>
<p>If low cell numbers are not accompanied by excessive cell death, this could have resulted from plating the wrong number of splenocytes per well for staining or loss of cells during washing steps (e.g., aspiration of pellet). Verify that you are correctly counting the number of splenocytes obtained and correctly calculating the volume of cell suspension to add into each well for staining.</p>
<p>Lastly, some mouse lines deficient in lymphocytes, such as <i>Rag1</i><sup>−/−</sup><i>,</i> do not have many splenocytes. Therefore, a low cell count may be expected for these mouse lines. In these cases, the goal is to collect and analyze as many cells as possible.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>High background signal for cytokine in uninfected control</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Even if you utilize the optimized dilutions suggested for antibody master mix included in this protocol, batch variation could cause increased background for a given amount of antibody used. Therefore, titrate the antibody to determine the best concentration to use. In addition, utilize a fluorescent minus one (FMO) control in which you stain with all of the antibodies in the panel except for the one with which you are having trouble. This will enable you to see if there is a compensation issue that needs to be corrected as well as improve your gating strategy by allowing more accurate discrimination between positive and negative signals.</p>
</section>
<section>
<h2 id="q-and-a">Q&amp;A</h2>
<p class="questions"><b>Q:</b> What is the vendor and catalogue number for the Brefeldin A you used in "Evaluating cytokine production by flow cytometry using Brefeldin A in mice"?</p>
<p class="answer"><b>A:</b> We purchased it from Sigma, catalog number B6542-25MG.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib11">Aachoui, Y., Kajiwara, Y., Leaf, I.A., Mao, D., Ting, J.P.-Y., Coers, J., Aderem, A., Buxbaum, J.D., and Miao, E.A. (2015). Canonical Inflammasomes Drive IFN-γ to Prime Caspase-11 in Defense against a Cytosol-Invasive Bacterium. Cell Host and Microbe <i>18</i>, 320-332.</p>
<p id="bib1">Bouabe, H. (2012). Cytokine reporter mice: the special case of IL-10. Scand. J. Immunol. <i>75</i>, 553-567.</p>
<p id="bib2">Corish, P. and Tyler-Smith, C. (1999). Attenuation of green fluorescent protein half-life in mammalian cells. Protein Eng. Des. Sel. <i>12</i>, 1035-1040.</p>
<p id="bib3">Dinarello, C.A. (2017). Overview of the IL-1 family in innate inflammation and acquired immunity. Immunol. Rev. <i>281</i>, 8-27.</p>
<p id="bib4">Kovacs, S.B., Oh, C., Maltez, V.I., McGlaughon, B.D., Verma, A., Miao, E.A., and Aachoui, Y. (2020). Neutrophil caspase-11 is essential to defend against a cytosol-invasive bacterium. Cell Rep. <i>32</i>, 107967.</p>
<p id="bib5">Liu, F. and Whitton, J.L. (2005). Cutting edge: re-evaluating the in vivo cytokine responses of CD8+ T cells during primary and secondary viral infections. J. Immunol. <i>174</i>, 5936-5940.</p>
<p id="bib6">Mazet, J.M., Chiodetti, A.L., Mahale, J.N., and Gérard, A. (2019). Imaging of in situ interferon gamma production in the mouse spleen following <i>Listeria monocytogenes</i> infection. JoVE 1-12.</p>
<p id="bib7">Mohrs, M., Shinkai, K., Mohrs, K., and Locksley, R.M. (2001). Analysis of type 2 immunity in vivo with a bicistronic IL-4 reporter. Immunity <i>15</i>, 303-311.</p>
<p id="bib8">Mukai, K., Tsai, M., Saito, H., and Galli, S.J. (2018). Mast cells as sources of cytokines, chemokines, and growth factors. Immunol. Rev. <i>282</i>, 121-150.</p>
<p id="bib9">Sun, J., Madan, R., Karp, C.L., and Braciale, T.J. (2009). Effector T cells control lung inflammation during acute influenza virus infection by producing IL-10. Nat. Med. <i>15</i>, 277-284.</p>
<p id="bib10">Weizman, O.-E., Adams, N.M., Schuster, I.S., Krishna, C., Pritykin, Y., Lau, C., Degli-Esposti, M.A., Leslie, C.S., Sun, J.C., and O’Sullivan, T.E. (2017). ILC1 confer early host protection at initial sites of viral infection. Cell <i>171</i>, 795-808.e12.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Dr. Edward Miao, <a href="mailto:edward.miao@duke.edu">edward.miao@duke.edu</a>.</p>
<h4>Materials availability</h4>
<p>The bacteria strain used in this paper is available upon request to lead contact.</p>
<h4>Data and code availability</h4>
<p>This paper did not generate any new datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grants AI097518, AI133236, AI139304, AI119073, AI136920 (E.A.M.), F30AI142990 (S.K.), and 1K22AI132489-01 (Y.A.), and NIH/<a href="https://doi.org/10.13039/100000057">NIGMS</a> award P20-GM103625 (Y.A.). E.A.M. is supported by the Yang Biomedical Scholars Award. The UNC Flow Cytometry Core Facility is supported in part by an <a href="https://doi.org/10.13039/100000054">NCI</a> Center Core Support Grant (P30CA016086) to the <a href="https://doi.org/10.13039/100008615">UNC Lineberger Comprehensive Cancer Center</a>. We also acknowledge the UAMS Center for Microbial Pathogenesis and Host Inflammatory Responses (CMPHIR) for continued support and the UAMS Flow Cytometry Core Facility for technical assistance.</p>
<h3>Author Contributions</h3>
<p>Y.A. and E.A.M. led the project. S.B.K. optimized the protocol and performed the experiments. C.O. independently validated the protocol.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>