<article>
<section>
<h1 id="header">Sequencing, cloning, and antigen binding analysis of monoclonal antibodies isolated from single mouse B cells</h1>
<p><time datetime="2021-03-19">Published: March 19, 2021</time></p>
<p>Charlotte Viant,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup> Amelia Escolano,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Spencer T. Chen,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> and Michel C. Nussenzweig<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup></p>
<p id="aff1"><sup>1</sup>Laboratory of Molecular Immunology, The Rockefeller University, New York, NY 10065, USA</p>
<p id="aff2"><sup>2</sup>Howard Hughes Medical Institute (HHMI), The Rockefeller University, New York, NY 10065, USA</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:cviant@rockefeller.edu">cviant@rockefeller.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100389">10.1016/j.xpro.2021.100389</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The analysis of B cell receptors (BCR) from single B cells is crucial to understanding humoral immune responses. Here, we describe a protocol for the sequencing, cloning, and characterization of antibody genes that encode BCRs. We used this method to analyze the BCRs of different mouse B cell populations for somatic hypermutations, clonal and phylogenic relationships, and their affinity for cognate antigen.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib7">Viant et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol describes single-cell sorting of mouse B cells into 96-well plates</li>
<li>Performs nested PCRs using reverse-transcribed cDNA of sorted single cells</li>
<li>Cloning of paired antibodies by sequence and ligation-independent cloning method</li>
<li>BCR analysis: somatic hypermutations, clonal/phylogenic relationships, antigen affinity</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/522-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the steps to isolate B cells, sequence their antibody genes and produce monoclonal Fabs (<a href="#fig1">Figure 1</a>). All the steps need to be plan in advance and adapted to the user’s experiment purpose. The current protocol is an adaptation of the methods described by (<a href="#bib8">von Boehmer et al., 2016</a>) and (<a href="#bib3">Escolano et al., 2019</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/522-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Diagram showing an overview of the protocol to isolate B cells and produce monoclonal antibodies</div>
</figcaption>
</figure>
<h3 id="sec1.1">Experimental design consideration</h3>
<ol>
<li>The experimental design and mouse strain will vary according to the user’s proposed study. All mouse experiments will need to comply with protocols approved by a local animal ethics committees. We used <i>S1pr2</i><sup>CreERT2/+</sup><i>R26</i><sup>ZSGreen/+</sup> (<a href="#bib5">Madisen et al., 2010</a>; <a href="#bib6">Shinnakasu et al., 2016</a>) mice immunized with an HIV-1 Envelope derived protein antigen (TM4-Core, (<a href="#bib2">Dosenovic et al., 2015</a>)). In these mice, the CreERT2-recombinase is expressed under the regulation of the S1pr2 promoter. Upon tamoxifen administration, the CreERT2-recombinase is translocated to the nucleus, which subsequently results in permanent expression of the ZSGreen fluorescent protein in <i>S1pr2</i> expressing cells. Different CreERT2-recombinase constructs have different expression levels and relative deletion efficiencies on targeted flox alleles. The user should titrate the tamoxifen dose and injection route for specific mouse strains. For the <i>S1pr2</i><sup>CreERT2/+</sup><i>R26</i><sup>ZSGreen/+</sup> mouse, we determined that one dose of 12 mg by oral gavage resulted in efficient ZSGreen expression by the S1pr2<sup>+</sup> cells.</li>
<li>The protocol for cell staining and the appropriate gating strategy will be designed and tested in advance and according to the user’s experimental needs, to clearly identify the populations of interest. Here we are using a panel of antibodies optimized for cell sorting of ZSGreen<sup>+</sup> B cells: dump channel (NK1.1, CD4, CD8, dead cell marker), B220, GL7, CD38, CD95 (<a href="#fig2">Figure 2</a>B: Gating strategy). This method can be adapted to any B cell population.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/522-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Sorting strategies and sequencing analysis</div>
<p>(A) Schematic representation of the experiment: S1pr2<sup>CreERT2/+</sup> R26<sup>ZSGreen/+</sup> mice immunized with TM4-Core (footpad injection) at day 0, tamoxifen gavage on day 9, sort of ZSGreen<sup>+</sup> B cells on day 14 form popliteal LN.</p>
<p>(B) FACS plots showing the gating strategy used to sort ZSGreen<sup>+</sup> B cells (singlets, Dump<sup>−</sup>, B220<sup>+</sup>, ZSGreen<sup>+</sup>).</p>
<p>(C) FACS plots showing 96 sorted ZSGreen<sup>+</sup> B cells. FACS index sorting allows the retrospective identification of each single cell by recording the level of expressions of the different markers. Thus, memory B cells (ZSGreen<sup>+</sup> CD38<sup>+</sup> GL7<sup>−</sup>) and GC cells (ZSGreen<sup>+</sup> CD38<sup>−</sup> GL7<sup>+</sup>) can be identified among the sorted ZSGreen<sup>+</sup> B cells.</p>
<p>(D) Pie charts depicting the distribution of antibody sequences (IgH + IgK) from GC B cells from 3 individual lymph nodes. The number in the inner circle indicates the number of sequences analyzed. White slices indicate sequences isolated only once, and gray or colored slices are proportional to the number of clonally related sequences.</p>
<p>(E) Graph shows number of somatic mutations (nucleotides, IgH + IgL) in the antibodies from the colored clones identified in (C).</p>
<p>(F) Trees show phylogenetic relationships between IgH and IgK sequences from the colored clone identified in LN 3 in (C).</p>
<p>(G) Graphs show biolayer interferometry traces obtained upon loading the sensors with TM4-Core and subsequently immersing them into solutions containing Fabs. Curves in red represent the binding of the anti-HIV-1 broadly neutralizing antibody 3BNC60m (<a href="#bib2">Dosenovic et al., 2015</a>) Fab, the positive control. Curves in gray represent the binding of ED38 (<a href="#bib4">Gitlin et al., 2016</a>) Fab, the negative control.</p>
<p>(H) Graphs show biolayer interferometry traces obtained upon loading the sensors with individual Fabs and subsequently immersing them into solutions containing TM4-Core. Curves in blue represent the binding of 3BNC60m Fab, the positive control. Curves in gray represent the binding of ED38 Fab, the negative control.</p>
</figcaption>
</figure>
</li>
<li>To efficiently and specifically amplify antibody genes from single B cells, we used nested PCR. This method involves two successive PCR reactions with different sets of primers. The first set of primers is designed to anneal upstream from the second set. The primers used in this protocol were designed to amplify IgM and IgG antibody genes from C57BL6 mice (<a href="#tbl1">Table 1</a>). Amplification of antibody genes from other mouse strains or species, as well as amplification of other antibody isotypes, will require a different set of primers that the user needs to design in advance.
<table id="tbl1">
<caption>Table 1. Primers for sequencing</caption>
<thead></thead>
<tbody>
<tr>
<td>AGGAACTGCAGGTGTCC</td>
<td>1st PCR IgH forward 1mFH_I</td>
</tr>
<tr>
<td>CAGCTACAGGTGTCCACTCC</td>
<td>1st PCR IgH forward 1mFH_II</td>
</tr>
<tr>
<td>TGGCAGCARCAGCTACAGG</td>
<td>1st PCR IgH forward 1mFH_III</td>
</tr>
<tr>
<td>CTGCCTGGTGACATTCCCA</td>
<td>1st PCR IgH forward 1mFH_IV</td>
</tr>
<tr>
<td>CCAAGCTGTGTCCTGTC</td>
<td>1st PCR IgH forward 1mFH_V</td>
</tr>
<tr>
<td>TTTTAAAAGGTGTCCAGKGT</td>
<td>1st PCR IgH forward 1mFH_VI</td>
</tr>
<tr>
<td>CCTGTCAGTAACTRCAGGTGTCC</td>
<td>1st PCR IgH forward 1mFH_VII</td>
</tr>
<tr>
<td>TTTTAAAAGGGGTCCAGTGT</td>
<td>1st PCR IgH forward 1mFH_VIII</td>
</tr>
<tr>
<td>CGTTCCTGGTATCCTGTCT</td>
<td>1st PCR IgH forward 1mFH_IX</td>
</tr>
<tr>
<td>ATGAAGTTGTGGYTRAACTGG</td>
<td>1st PCR IgH forward 1mFH_X</td>
</tr>
<tr>
<td>TGTTGGGGCTKAAGTGGG</td>
<td>1st PCR IgH forward 1mFH_XI</td>
</tr>
<tr>
<td>AGAAGGTGTGCACACCGCTGGAC</td>
<td>1st PCR IgH reverse 1mRG</td>
</tr>
<tr>
<td>AGGGGGCTCTCGCAGGAGACGAGG</td>
<td>1st PCR IgH REVERSE 1mRM</td>
</tr>
<tr>
<td>RGTGCAGATTTTCAGCTTCCTGCT</td>
<td>1st PCR IgK forward 1mFK_I</td>
</tr>
<tr>
<td>TGGACATGAGGGCYCCTGCTCAGT</td>
<td>1st PCR IgK forward 1mFK_II</td>
</tr>
<tr>
<td>CTSTGGTTGTCTGGTGTTGAYGGA</td>
<td>1st PCR IgK forward 1mFK_III</td>
</tr>
<tr>
<td>GTTGCTGCTGCTGTGGCTTACA</td>
<td>1st PCR IgK forward 1mFK_IV</td>
</tr>
<tr>
<td>GTATCTGGTACCTGTGG</td>
<td>1st PCR IgK forward 1mFK_V</td>
</tr>
<tr>
<td>TGCCTGTTAGGCTGTTGGTGCT</td>
<td>1st PCR IgK forward 1mFK_VI</td>
</tr>
<tr>
<td>GCTCAGTTCCTTGGTCTCCTGTTGC</td>
<td>1st PCR IgK forward mFK_VII</td>
</tr>
<tr>
<td>TGGGTGCTGCTGCTCTGGGT</td>
<td>1st PCR IgK forward 1mFK_VIII</td>
</tr>
<tr>
<td>CAGTTCCTGTTTCTGTTARTGCTCTGG</td>
<td>1st PCR IgK forward 1mFK_IX</td>
</tr>
<tr>
<td>TGCTCTGGTTATATGGTGCTGATGGG</td>
<td>1st PCR IgK forward 1mFK_X</td>
</tr>
<tr>
<td>ACTGAGGCACCTCCAGATGTT</td>
<td>1st PCR IgK reverse 1mRK</td>
</tr>
<tr>
<td>GGGAATTCGAGGTGCAGCTGCAGGAGTCTGG</td>
<td>2st PCR IgH forward 2mFG</td>
</tr>
<tr>
<td>GCTCAGGGAARTAGCCCTTGAC</td>
<td>2st PCR IgH reverse 2mRG</td>
</tr>
<tr>
<td>AGGGGGAAGACATTTGGGAAGGAC</td>
<td>2st PCR IgH Reverse 2mRM</td>
</tr>
<tr>
<td>GAYATTGTGMTSACMCARWCTMCA</td>
<td>2st PCR IgK forward 2mFK</td>
</tr>
<tr>
<td>TGGGAAGATGGATACAGTT</td>
<td>2st PCR IgK reverse 2mRK</td>
</tr>
</tbody>
</table>
<p class="table-legend">See <a href="#bib8">von Boehmer et al. (2016)</a>.</p>
<table id="tbl2">
<caption>Table 2. Primers for cloning PCR</caption>
<thead></thead>
<tbody>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCGAAAWTGTGCTCACCCAGTC</td>
<td>IgK forward T4hmFK_I</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCCAAATTGTTCTCACCCAGTC</td>
<td>IgK forward T4hmFK_II</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCRACATTGTGCTGACCCAATC</td>
<td>IgK forward T4hmFK_III</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCGAAACAACTGTGACCCAGTC</td>
<td>IgK forward T4hmFK_IV</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCGATATTGTGATGACSCAGGC</td>
<td>IgK forward T4hmFK_V</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCRRTRTTGTGATGACCCARAC</td>
<td>IgK forward T4hmFK_VI</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCGATATCCAGATGACACAGAC</td>
<td>IgK forward T4hmFK_VII</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCGACATTGTGATGACMCAGTC</td>
<td>IgK forward T4hmFK_VIII</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCCGACATCCAGATGACHCAGTC</td>
<td>IgK forward T4hmFK_IX</td>
</tr>
<tr>
<td>GAAGACAGATGGTGCAGCCACCGTACGTTTCAGCTCCAGCTTGGTCCC</td>
<td>IgK reverse T4hRK_I</td>
</tr>
<tr>
<td>GAAGACAGATGGTGCAGCCACCGTACGTTTTATTTCCAGTCTGGTCCC</td>
<td>IgK reverse T4hRK_II</td>
</tr>
<tr>
<td>GAAGACAGATGGTGCAGCCACCGTACGTTTKATTTCCARCTTKGTSCC</td>
<td>IgK reverse T4hRK_III</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTGAWGTGCAGCTGGTGGAGTC</td>
<td>IgH forward T4hmFH_I</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTCAGGTGCAGCTGAAGSAGTC</td>
<td>IgH forward T4hmFH_II</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTGARGTGAAGCTGGTGGARTC</td>
<td>IgH forward T4hmFH_III</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTCAGGTCCAACTGCAGCAGCC</td>
<td>IgH forward T4hmFH_IV</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTSAGGTYCAGCTGCARCAGTC</td>
<td>IgH forward T4hmFH_V</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTCAAGTGCAGATGAAGGAGTC</td>
<td>IgH forward T4hmFH_VI</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTCAGATCCAGTTGGYGCAGTC</td>
<td>IgH forward T4hmFH_VII</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTCAGGTCCAACTCCAGCAGCC</td>
<td>IgH forward T4hmFH_VIII</td>
</tr>
<tr>
<td>CTAGTAGCAACTGCAACCGGTGTACATTCTCAGGTGCAACTGAAGCAGTC</td>
<td>IgH forward T4hmFH_IX</td>
</tr>
<tr>
<td>CCGATGGGCCCTTGGTCGACGCTGAGGAGACGGTGACCGTGG</td>
<td>IgH reverse T4hRG_I</td>
</tr>
<tr>
<td>CCGATGGGCCCTTGGTCGACGCTGAGGAGACTGTGAGAGTGG</td>
<td>IgH reverse T4hRG_II</td>
</tr>
<tr>
<td>CCGATGGGCCCTTGGTCGACGCTGAGGAGACAGTGACCAGAG</td>
<td>IgH reverse T4hRG_III</td>
</tr>
<tr>
<td>CCGATGGGCCCTTGGTCGACGCTGAGGAGACGGTGACTGAGG</td>
<td>IgH reverse T4hRG_IV</td>
</tr>
<tr>
<td>GCTTCGTTAGAACGCGGCTAC</td>
<td>5′ Ab sense</td>
</tr>
</tbody>
</table>
<p class="table-legend">See <a href="#bib8">von Boehmer et al. (2016)</a>.</p>
</li>
<li>To clone the antibody heavy- and light-chain genes into expression vectors we used the Sequence and Ligation-Independent Cloning (SLIC) method. The SLIC cloning method allows the assembly of multiple DNA fragments in a single reaction using <i>in vitro</i> homologous recombination and single-strand annealing. We recommend that the user prepare a stock of the appropriate linearized vectors in advance.</li>
<li>HEK293-6E suspension cells developed by the NCR Biotechnology Research Institute (NRC-BRI, Montréal, Canada) are used for antibody production. These cells proved to be excellent tools for transient transfection and subsequent high-titer production of recombinant proteins. They grow in suspension in FreeStyle 293 expression medium supplemented with penicillin and streptomycin (10,000 U/mL). The user needs to start the HEK293-6E cell culture several days before use. The cells need at least 2 passages before transfection and can be kept in culture for one month.</li>
</ol>
<p>To illustrate this protocol, we used the following experiment as an example (<a href="#fig2">Figures 2</a>A–2E): three <i>S1pr2</i><sup>CreERT2/+</sup><i>R26</i><sup>ZSGreen/+</sup> mice were immunized with TM4-Core (day 0: Footpad immunization, 5 μg) and treated with tamoxifen 9 days later (one dose, oral gavage, 12 mg). At day 12, we sorted ZGreen<sup>+</sup> B cells from individual popliteal LN and Germinal Center (GC) cells were identified by their expression of GL7 and CD95 and the absence of CD38 (data obtained from the sort index file). GC cells antibodies were sequenced and we analyzed their somatic hypermutations, clonality, and phylogenic relationships.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mice: <i>S1pr2</i>-ERT2cre (male and female 7 to 10 weeks)</td>
<td>T. Kurosaki</td>
<td><a href="#bib6">Shinnakasu et al., 2016</a></td>
</tr>
<tr>
<td>Mice: <i>Rosa</i>-ZSGreen (male and female 7 to 10 weeks)</td>
<td>Jackson Laboratory</td>
<td>Stock No 007914</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>HEK293-6E</td>
<td>National Research Council of Canada</td>
<td>NRC file 11565</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Subcloning Efficiency DH5α Competent Cells</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#18265017</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Tamoxifen</td>
<td>Sigma</td>
<td>Cat#T5648</td>
</tr>
<tr>
<td>Corn oil</td>
<td>Sigma</td>
<td>Cat#C8267</td>
</tr>
<tr>
<td>HIV-1 TM4-Core</td>
<td>A.T. McGuire and L. Stamatatos (Fred Hutchinson Cancer Research Center, Seattle) (<a href="#bib2">Dosenovic et al., 2015</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Imject alum</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#77161</td>
</tr>
<tr>
<td>DPBS 1× (-mg, -ca)</td>
<td>Gibco</td>
<td>Cat#M02900</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>GE Healthcare Life Sciences</td>
<td>Cat#SH30910.03</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>Invitrogen</td>
<td>Cat#15575020</td>
</tr>
<tr>
<td>ACK lysing buffer</td>
<td>Gibco</td>
<td>Cat#A1049201</td>
</tr>
<tr>
<td>Live/dead marker Zombie NIR</td>
<td>BioLegend</td>
<td>Cat#423106</td>
</tr>
<tr>
<td>TCL buffer</td>
<td>QIAGEN</td>
<td>Cat#1031576</td>
</tr>
<tr>
<td>2-β-Mercaptoethanol</td>
<td>Sigma</td>
<td>Cat#M3148</td>
</tr>
<tr>
<td>HotStarTaq DNA polymerase (25,000)</td>
<td>QIAGEN</td>
<td>Cat#203209</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>QIAGEN</td>
<td>Cat#1039498</td>
</tr>
<tr>
<td>dNTP set (100 mM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10-297-018</td>
</tr>
<tr>
<td>PCR buffer 10×</td>
<td>QIAGEN</td>
<td>Cat#1005479</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma</td>
<td>Cat#S0389-5KG</td>
</tr>
<tr>
<td>Cresol red</td>
<td>Sigma</td>
<td>Cat#C-9877</td>
</tr>
<tr>
<td>Tergitol type NP40</td>
<td>Sigma</td>
<td>Cat#NP40S</td>
</tr>
<tr>
<td>SalI-HF</td>
<td>New England Biolabs</td>
<td>Cat#R3138L</td>
</tr>
<tr>
<td>BsiWI</td>
<td>New England Biolabs</td>
<td>Cat#R3553L</td>
</tr>
<tr>
<td>AgeI</td>
<td>New England Biolabs</td>
<td>Cat#R3552L</td>
</tr>
<tr>
<td>Alkaline phosphatase, calf intestinal (CIP)</td>
<td>New England Biolabs</td>
<td>Cat#M0290</td>
</tr>
<tr>
<td>Gel loading dye purple 6×</td>
<td>New England Biolabs</td>
<td>Cat#B7024S</td>
</tr>
<tr>
<td>NucleoSpin Gel and PCR Clean-up</td>
<td>Machery-Nagel</td>
<td>Cat#740609.250</td>
</tr>
<tr>
<td>Purified BSA 100×</td>
<td>New England Biolabs</td>
<td>Cat#B9001S</td>
</tr>
<tr>
<td>NEB2</td>
<td>New England Biolabs</td>
<td>Cat#B7002S</td>
</tr>
<tr>
<td>T4 DNA polymerase</td>
<td>New England Biolabs</td>
<td>M0203L</td>
</tr>
<tr>
<td>LB Agar</td>
<td>BD Biosciences</td>
<td>Cat#244510</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A9518-100G</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Decon Labs</td>
<td>Cat#2716</td>
</tr>
<tr>
<td>RNase inhibitor</td>
<td>Promega</td>
<td>Cat#N2615</td>
</tr>
<tr>
<td>Freestyle 293 Expression Medium</td>
<td>Gibco</td>
<td>Cat#12-338-026</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>Gibco</td>
<td>Cat#15-140-122</td>
</tr>
<tr>
<td>Steriflip 50 mL 0.22μm</td>
<td>Millipore Sigma</td>
<td>Cat#SCGP00525</td>
</tr>
<tr>
<td>PEI MAX - transfection grade linear polyethylenimine hydrochloride (MW 40,000)</td>
<td>Polysciences</td>
<td>Cat#24765-1</td>
</tr>
<tr>
<td>EZ-Link NHS-PEG4-Biotin</td>
<td>Thermo Scientific</td>
<td>Cat#21330</td>
</tr>
<tr>
<td>Streptavidin-BV711</td>
<td>BD Biosciences</td>
<td>Cat#563262</td>
</tr>
<tr>
<td>Imidazole</td>
<td>Sigma</td>
<td>Cat#1370980100</td>
</tr>
<tr>
<td>Tris pH 7.5</td>
<td>Sigma</td>
<td>Cat#10708976001</td>
</tr>
<tr>
<td>NaCl</td>
<td>Sigma</td>
<td>Cat#7647-14-5</td>
</tr>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-mouse CD16/32 (ratmAb 2.4G2, mouse Fc block) 1/500 (final: 1 μg/mL)</td>
<td>BD Biosciences</td>
<td>Cat#553141</td>
</tr>
<tr>
<td>Anti-mouse CD95-PE-Cy7 (Jo2) 1/200 (final: 1 μg/mL)</td>
<td>BD Biosciences</td>
<td>Cat#557653</td>
</tr>
<tr>
<td>Anti-mouse CD38-PB (90) 1/100 (final: 5 μg/mL)</td>
<td>BioLegend</td>
<td>Cat#102719</td>
</tr>
<tr>
<td>Anti-mouse B220-BV605 (RA3-6B2) 1/200 (final: 1 μg/mL)</td>
<td>BioLegend</td>
<td>Cat#103244</td>
</tr>
<tr>
<td>Anti-mouse T and B cell activation antigen-e660 (GL7) 1/100 (final: 2 μg/mL)</td>
<td>eBiosciences</td>
<td>Cat#50-5902-82</td>
</tr>
<tr>
<td>Anti-mouse CD4-eF780 (RM4-5) 1/200 (final: 1 μg/mL)</td>
<td>eBiosciences</td>
<td>Cat#47-0042-82</td>
</tr>
<tr>
<td>Anti-mouse CD8-eF780 (53-6.7) 1/200 (final: 1 μg/mL)</td>
<td>eBiosciences</td>
<td>Cat#47-0081-82</td>
</tr>
<tr>
<td>Anti-mouse NK1.1-eF780 (PK136) 1/200 (final: 1 μg/mL)</td>
<td>eBiosciences</td>
<td>Cat#47-5941-82</td>
</tr>
<tr>
<td>Anti-mouse F4/80-eF780 (BM8) 1/200 (final: 1 μg/mL)</td>
<td>eBiosciences</td>
<td>Cat#47-4801-82</td>
</tr>
<tr>
<td>3BNC60m Fab</td>
<td><a href="#bib2">Dosenovic et al., 2015</a></td>
<td>N/A</td>
</tr>
<tr>
<td>ED38 Fab</td>
<td>(<a href="#bib10">Wardemann et al., 2003</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Random primers</td>
<td>Invitrogen</td>
<td>Cat#48190-011</td>
</tr>
<tr>
<td>Primers for sequencing (see <a href="#tbl1">Table 1</a>)</td>
<td><a href="#bib8">von Boehmer et al., 2016</a></td>
<td>See <a href="#tbl1">Table 1</a>
</td>
</tr>
<tr>
<td>Primers for cloning PCR (see <a href="#tbl2">Table 2</a>)</td>
<td><a href="#bib8">von Boehmer et al., 2016</a></td>
<td>See <a href="#tbl2">Table 2</a>
</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RNAClean XP beads (RNA-SPRI beads)</td>
<td>Beckman Coulter</td>
<td>Cat#A63987</td>
</tr>
<tr>
<td>SuperScript III First-Strand Synthesis kit</td>
<td>Invitrogen</td>
<td>Cat#18080400</td>
</tr>
<tr>
<td>QIAquick 96 PCR Purification Kit</td>
<td>QIAGEN</td>
<td>Cat#28183</td>
</tr>
<tr>
<td>Ni Sepharose 6 Fast Flow</td>
<td>GE Healthcare</td>
<td>Cat#17-5318-06</td>
</tr>
<tr>
<td>NucleoBond Xtra Maxi</td>
<td>Macherey-Nagel</td>
<td>Cat#740414.100</td>
</tr>
<tr>
<td>Poly-Prep columns</td>
<td>Bio-Rad</td>
<td>Cat#7311553</td>
</tr>
<tr>
<td>Kinetics buffer 10×</td>
<td>ForteBio</td>
<td>Cat#18-1105</td>
</tr>
<tr>
<td>Biosensor high precision streptavidin (SAX)</td>
<td>ForteBio</td>
<td>Cat#18-5119</td>
</tr>
<tr>
<td>Biosensor anti-human FAB2G</td>
<td>ForteBio</td>
<td>Cat#18-5127</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>96-Well microplates polypropylene black</td>
<td>Greiner bio-one</td>
<td>Cat#655209</td>
</tr>
<tr>
<td>Insulin syringes</td>
<td>BD Biosciences</td>
<td>Cat#329461</td>
</tr>
<tr>
<td>Plastic feeding tubes 20ga</td>
<td>Instech Laboratories</td>
<td>Cat#FTP-20-38</td>
</tr>
<tr>
<td>1 mL syringes with BD Luer-Lok</td>
<td>BD Biosciences</td>
<td>Cat#309628</td>
</tr>
<tr>
<td>Cell strainer 70 μm Nylon</td>
<td>Falcon</td>
<td>Cat#352350 10172888300050</td>
</tr>
<tr>
<td>DynaMag-96 Side Magnet</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12331D</td>
</tr>
<tr>
<td>BD FACSAria III cell sorter</td>
<td>BD</td>
<td>N/A</td>
</tr>
<tr>
<td>SimpliAmp thermal cycler</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A24811</td>
</tr>
<tr>
<td>Amicon Ultra-4 centrifugal filter unit with Ultracel-10 membrane</td>
<td>Millipore Sigma</td>
<td>Cat#UFC801096</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism version 7</td>
<td>GraphPad Software</td>
<td>
<a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a>RRID: SCR_002798</td>
</tr>
<tr>
<td>FlowJo v10.3.5</td>
<td>FlowJo</td>
<td>
<a href="https://www.flowjo.com/">https://www.flowjo.com</a> RRID: SCR_008520</td>
</tr>
<tr>
<td>Adobe Illustrator CC 2018</td>
<td>BD Biosciences</td>
<td>
<a href="https://www.adobe.com/uk/products/illustrator.html">https://www.adobe.com/uk/products/illustrator.html</a>RRID: SCR_014198</td>
</tr>
<tr>
<td>IgBlast tool (NCBI)</td>
<td>NCBI</td>
<td>
<a href="https://www.ncbi.nlm.nih.gov/igblast/">https://www.ncbi.nlm.nih.gov/igblast/</a>RRID: SCR_002873</td>
</tr>
<tr>
<td>Octet Software Version 10.0</td>
<td>Fortebio</td>
<td><a href="https://www.fortebio.com/products/octet-systems-software">https://www.fortebio.com/products/octet-systems-software</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>20 mM Imidazole buffer, kept at room temperature (20°C–22°C) with no time limit</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Imidazole</td>
<td>20 mM</td>
<td>0.13 g</td>
</tr>
<tr>
<td>Tris pH 7.5</td>
<td>20 mM</td>
<td>0.24 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>300 mM</td>
<td>17.53 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>30 mM Imidazole buffer, kept at room temperature (20°C–22°C) with no time limit</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Imidazole</td>
<td>30 mM</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Tris pH 7.5</td>
<td>20 mM</td>
<td>0.24 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>300 mM</td>
<td>17.53 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>250 mM Imidazole buffer, kept at room temperature (20°C–22°C) with no time limit</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Imidazole</td>
<td>250 mM</td>
<td>17.02 g</td>
</tr>
<tr>
<td>Tris pH 7.5</td>
<td>20 mM</td>
<td>0.24 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>300 mM</td>
<td>17.53 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Mouse immunization and treatments</h3>
<div class="critical">
<span class="critical-title">Critical:</span> Experiments involving mice should be performed according to all relevant governmental and institutional guidelines and regulations, in compliance with protocols approved by local animal ethics committees.</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol>
<li>Footpad (FP) immunization at experimental day 0.
<ol type="a">
<li>Antigen precipitation in alum.
<ol type="i">
<li>Dilution of the antigen at the chosen concentration in PBS 1× and Alum at a 2:1 ratio (25 μL/FP with 5 μg of TM4-Core diluted in 8.33 μL Alum and 16.7 μL PBS 1×).</li>
<li>Rotate at 4°C for 45 min.</li>
</ol>
</li>
<li>Injection of adjuvanted antigen in the footpad of anesthetized mice (isoflurane anesthesia) using 0.5 mL insulin syringe (25 μL/FP).</li>
</ol>
</li>
</ol>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="2">
<li>Tamoxifen treatment. The day of injection, the concentration and the number of doses will vary according to the user’s proposed study. In the experiment illustrated in <a href="#fig2">Figure 2</a>A, we injected one dose (12 mg) 9 days after immunization.
<ol type="a">
<li>Dissolve tamoxifen in Corn oil at 60 mg/mL. Incubate in a rotator at 60°C for 15 min minimum. (Troubleshooting 2)</li>
<li>Oral gavage (200 μL/mice) with plastic feeding tube and 1 mL syringe.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Single-cell index sorting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<div class="critical">
<span class="critical-title">Critical:</span> Cell staining, cytometer compensations, and gating strategies need to be optimized before the experiment.</div>
<ol start="3">
<li>Preparation of B cell suspensions.
<ol type="a">
<li>Euthanize mice according to institutional guidelines.</li>
<li>Collect the mouse popliteal lymph nodes (LNs): secure the mice with the knee down to expose the popliteal fossa. Carefully separate the LN from surrounding adipose tissues and muscles using micro-dissecting tweezers and forceps (<a href="#fig3">Figure 3</a>). Put the LN in a 70-μm polystyrene sterile cell strainer immersed in FACS Buffer (FB: PBS 1× pH 7.2, 10% Fetal Bovine Serum (FBS), 2 mM EDTA, 4°C) and placed in a Petri dish on ice.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/522-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Dissection mouse popliteal lymph nodes</div>
<p>Red arrow: position popliteal LN, right footpad.</p>
</figcaption>
</figure>
</li>
<li>Disaggregate the LN with the help of a 3 mL syringe plunger (<a href="#fig3">Figure 3</a>).</li>
<li>Filter the cell suspensions into a 15 mL conical tube using a new 70-μm polystyrene sterile cell strainer. Bring the final volume to 10 mL with cold FB.</li>
<li>Centrifuge the cells at 350 × <i>g</i> for 5 min at 4°C. Carefully remove the supernatant.</li>
<li>Lyse red blood cells by resuspending the pellet in 1 mL of ACK Lysis Buffer for 1 min at 4°C. Stop the reaction with 10 mL of FB.</li>
<li>Centrifuge the cells at 350 × <i>g</i> for 5 min at 4°C. Carefully remove the supernatant.</li>
<li>Resuspend the cells in 500 μL of FB and transfer them to a 5 mL polystyrene round-bottom tube.</li>
</ol>
</li>
<li>B cell staining.
<ol type="a">
<li>Centrifuge at 350 × <i>g</i> for 5 min at 4°C.</li>
<li>Add anti-mouse CD16/32 to block Fc receptors (1:500; final concentration at 1 μg/mL) in FB for 15 min at 4°C.</li>
<li>Centrifuge the tubes at 350 × <i>g</i> for 5 min at 4°C. Carefully remove the supernatant.</li>
<li>Add fluorescently labeled antibodies and Dead Cells Marker (DCM) in PBS 1× pH 7.2, 2 mM EDTA (without adding FBS as it decreases the DCM labeling efficiency; DCM staining protocol is modified from the manufacturer’s instructions) for 30 min at 4°C (<a href="#fig2">Figure 2</a>B: labeling of ZSGreen<sup>+</sup> B cells).</li>
<li>Wash the cells with 2 mL of FB and centrifuge at 350 × <i>g</i> for 5 min at 4°C. Carefully remove the supernatant.</li>
<li>Resuspend the cells in 500 μL of FB. Protect from the light and keep at 4°C</li>
</ol>
</li>
<li>Single B cell sort (gating strategy: <a href="#fig2">Figures 2</a>B and 2C).
<ol type="a">
<li>Add 5 μL of lysis buffer (TCL with 1% 2-β-mercaptoethanol) into each well of a 96-well PCR plate. Prepare an ice bucket with dry ice to immediately freeze the plates containing the sorted B cells.</li>
<li>Using a BD Aria III cell sorter, or equivalent, proceed with the compensation setup and set the gate strategy.</li>
<li>Sort single B cells into individual wells of the 96-well plates containing the lysis buffer. In the experiment illustrated in <a href="#fig2">Figures 2</a>A–2C we sorted between 3 and 4 96-well plates of ZSGreen<sup>+</sup> B cells from one popliteal LN.</li>
<li>After completion, immediately seal the 96-well plate and place it on dry ice.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Plates can be stored at −80°C for several years.</div>
<h3 id="sec3.3">mRNA purification and reverse transcription</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5 h</div>
<p>After isolation of single B cells, single-cell RNA is purified using RNA-SPRI paramagnetic beads. The RNA-SPRI beads need to be brought to room temperature (20°C–22°C) at least 30 min in advance. The reverse transcription mix 1 and 2 can be prepared in advance and stored at 4°C until use.</p>
<div class="critical">
<span class="critical-title">Critical:</span> It is critical to work in a DNA and RNase free area to prevent potential contamination and RNA degradation. It is recommended to work in a laminar flow hood specifically designated for antibody gene amplification from single cells, to use sterile filter pipette tips and to thoroughly clean all pipets and surfaces with RNase Away.</div>
<ol start="6">
<li>Bring RNA-SPRI beads at room temperature (20°C–22°C) for 30 min and mix thoroughly by vortexing to resuspend any magnetic particles that may have settled.</li>
<li>Thaw the plate containing the sorted B cells on ice for 5 min. Centrifuge at 400 × <i>g</i> for 30 s at 4°C and add 10 μL of nuclease-free water.</li>
<li>Add 33 μL of RNA-SPRI beads/well and mix by pipetting. We used a 1: 2.2 ratio of Lysate and RNA-SPRI beads to allow the purification of RNA molecules of both larger and smaller molecular weights. Incubate at RT (20°C–22°C) for 10 min.</li>
<li>Prepare wash buffer: 80% ethanol in nuclease-free water.</li>
<li>Place the plate on a DynaMag-96 side magnet. Incubate for 5 min at RT (20°C–22°C) or until beads are clearly attached to the wall of the tube next to the magnet.</li>
<li>Remove the supernatant with a multichannel pipet without disturbing the bead pellet.</li>
<li>Add 125 μL of 80% ethanol and wash the beads by moving the plate to an adjacent magnet column a total of 4 times. Allow the beads to completely move from side to side of the tube wall to ensure proper washing. Carefully remove the ethanol and repeat the washes two more times for a total of 3 washes.</li>
<li>Air dry the bead pellets for 8–10 min.</li>
<li>Elute RNA from beads with 11 μL of reverse transcription (RT) Mix-1 per well. Pipette up and down several times to resuspend the bead pellet.</li>
</ol>
<table id="undtbl4">
<thead>
<tr>
<th>RT Mix 1 (1 plate)</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>1,087.125</td>
</tr>
<tr>
<td>Random primers (0.3 mg/mL)</td>
<td>58.75</td>
</tr>
<tr>
<td>NP40 (10%)</td>
<td>58.75</td>
</tr>
<tr>
<td>RNasin Plus (RNase Inhibitor) (40 U/μL)</td>
<td>23</td>
</tr>
</tbody>
</table>
<ol start="15">
<li>Incubate the plates for 3 min at 65°C in a thermocycler.</li>
<li>Add 7 μL of RT Mix-2 per well.</li>
</ol>
<table id="undtbl5">
<thead>
<tr>
<th>RT Mix 2 (1 plate)</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>235.75</td>
</tr>
<tr>
<td>5× Superscript Buffer</td>
<td>345</td>
</tr>
<tr>
<td>dNTP mix (25 mM)</td>
<td>57.5</td>
</tr>
<tr>
<td>DTT (100 mM)</td>
<td>115</td>
</tr>
<tr>
<td>RNasin Plus (RNase Inhibitor) (40 U/μL)</td>
<td>23</td>
</tr>
<tr>
<td>Superscript III (200 U/μL)</td>
<td>28.75</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> The reverse transcription mix 1 and 2 can be prepared in advance and stored at 4°C until use.</div>
<ol start="17">
<li>Incubate the plate in a thermocycler using the following program:</li>
</ol>
<table id="undtbl6">
<thead>
<tr>
<th colspan="3">PCR cycling conditions</th>
</tr>
<tr>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>42°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>25°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>50°C</td>
<td>60 min</td>
<td>1</td>
</tr>
<tr>
<td>94°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<ol start="18">
<li>Add 10 μL/well of nuclease-free water. Proceed to immunoglobulin gene amplification by PCR or store the plate containing cDNA at −20°C or −80°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Plates can be stored for several years.</div>
<h3 id="sec3.4">First PCR amplification of the heavy- and light-chain genes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5 h</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use sterile filter pipette tips for all reactions. All surfaces should be sprayed with DNA/RNAse away. Resuspend lyophilized primers (<a href="#tbl1">Table 1</a>) at 50 μM in RNase free water. Prepare a working solution mix by mixing equal volumes of each primer.</div>
<ol start="19">
<li>Thaw the cDNA plate on ice.</li>
<li>The variable heavy and light chain genes are amplified in separate PCR reactions. Prepare PCR mixes for the first step of PCR amplification (PCR 1) of IgH and IgK genes.</li>
</ol>
<table id="undtbl7">
<thead>
<tr>
<th>PCR 1 mix (1 plate)</th>
<th>Volume (μL)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>3,328</td>
<td> </td>
</tr>
<tr>
<td>10× Buffer</td>
<td>384</td>
<td>1×</td>
</tr>
<tr>
<td>dNTPs mix (25 mM)</td>
<td>48</td>
<td>0.3 mM</td>
</tr>
<tr>
<td>5′F primers (50 μM)</td>
<td>23</td>
<td>0.3 μM</td>
</tr>
<tr>
<td>3′R primers (50 μM)</td>
<td>15</td>
<td>0.2 μM</td>
</tr>
<tr>
<td>HotStarTaq DNA polymerase (5 U/μL)</td>
<td>42</td>
<td>0.05 U/μL</td>
</tr>
</tbody>
</table>
<p>Primers (<a href="#tbl1">Table 1</a>):</p>
<p>The forward primers are a mix of 9 or 10 different primers. In order to have sufficient amount of each primer in the reaction we use a slightly higher concentration of the forward mix compared to the concentration of the single reverse primer.</p>
<p>5′ F Primers Heavy chain: 1mFHI / 1mFHII / 1mFHIII / 1mFHIV / 1mFHV / 1mFHVI / 1mFHVII / 1mFHVIII / 1mFHIX</p>
<p>3′ R Primers Heavy chain: 1mRHG (IgG) / 1mRHC</p>
<p>5′ F Primers Light chain: 1mFkA / 1mFkB / 1mFkC / 1mFkD / 1mFkE / 1mFkF / 1mFkG / 1mFkH / 1mFkI / 1mFkJ</p>
<p>3′ R Primer Light chain: 1mRk</p>
<ol start="21">
<li>Add 38 μL/well of PCR 1 mix in a new 96-well PCR plate.</li>
<li>Centrifuge the cDNA plate at 400 × <i>g</i> for 1 min at 4°C.</li>
<li>Transfer 3 μL of cDNA solution from the cDNA plate to the same position in the new 96-well plate containing PCR1 mix.</li>
<li>Place the plate in a thermocycler and incubate according to the following program:</li>
</ol>
<table id="undtbl8">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>15 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">50</td>
</tr>
<tr>
<td>Annealing</td>
<td>46°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>55 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<ol start="25">
<li>Proceed to the second step of amplification (PCR 2) or store the PCR1 plate at −20°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Plates can be stored at −20°C for several years.</div>
<h3 id="sec3.5">Second PCR amplification of the heavy- and light-chain genes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5 h</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use sterile filter pipette tips for all reactions. All surfaces should be sprayed with DNA/RNAse away. Resuspend lyophilized primers (<a href="#tbl1">Table 1</a>) at 50 μM in RNase free water. Prepare a working solution mix by mixing equal volumes of each primer.</div>
<ol start="26">
<li>Thaw the PCR 1 plate for 5 min on ice.</li>
<li>Prepare PCR mixes for the second step of PCR amplification (PCR 2) of IgH and IgK genes.</li>
</ol>
<table id="undtbl9">
<thead>
<tr>
<th>PCR 2 mix (1 plate)</th>
<th>Volume (μL)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>2,536</td>
<td> </td>
</tr>
<tr>
<td>Loading buffer</td>
<td>800</td>
<td> </td>
</tr>
<tr>
<td>10× Buffer</td>
<td>384</td>
<td>1×</td>
</tr>
<tr>
<td>dNTPs mix (25 mM)</td>
<td>48</td>
<td>0.3 mM</td>
</tr>
<tr>
<td>5′F primers (50 μM)</td>
<td>15</td>
<td>0.2 μM</td>
</tr>
<tr>
<td>3′R primers (50 μM)</td>
<td>15</td>
<td>0.2 μM</td>
</tr>
<tr>
<td>HotStarTaq DNA polymerase (5 U/μL)</td>
<td>42</td>
<td>0.05 U/μL</td>
</tr>
</tbody>
</table>
<p>Loading buffer: dissolve 20 g sucrose in nuclease-free water to a final volume of 50 mL and sprinkle a few grains of cresol red dye.</p>
<p>Primers (<a href="#tbl1">Table 1</a>):</p>
<p>5′ F Primer Heavy chain: 2mFG</p>
<p>3′ R Primers Heavy chain: 2mRG (IgG) 1mRHC</p>
<p>5′ F Primer Light chain: IgK-Fw1-Fw2</p>
<p>3′ R Primer Light chain: 2mRk</p>
<ol start="28">
<li>Add 38 μL/well of PCR 2 mix in a new 96-well PCR plate.</li>
<li>Centrifuge the PCR 1 plate at 400 × <i>g</i> for 1 min at 4°C.</li>
<li>Transfer 4 μL of PCR 1 from each well to the same position in the PCR 2 plate.</li>
<li>Place the plate in a thermocycler and incubate according to the following program:</li>
</ol>
<table id="undtbl10">
<caption>IgG/IgM:</caption>
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>15 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">50 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>55 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<table id="undtbl11">
<caption>IgK:</caption>
<thead>
<tr>
<th colspan="4">PCR Cycling Conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>15 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">50 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>46°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>55 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<ol start="32">
<li>Run 5 μL of the product of PCR 2 in a 2% agarose gel. The expected PCR product is ~500 bp for IgH and ~450 bp for IgK.</li>
</ol>
<p>Sequence the PCR 2 products by Sanger sequencing using the reverse primer of PCR 2. In the experiment illustrated in <a href="#fig2">Figures 2</a>A–2D we sequenced between 77 to 124 GC cells paired antibodies from 3 to 4 96-well plates of ZSGreen<sup>+</sup> GC cells sorted from one popliteal LN</p>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Plates can be stored at 4°C for several days.</div>
<h3 id="sec3.6">PCR amplification of heavy- and light-chain genes for cloning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5 h</div>
<p>The cloned insert is obtained amplifying the product of the first nested PCR to add sequences homologous to the plasmid’s site of insertion. The primers used for the cloning PCR have adaptor sequences adding at the end of the inserts the restriction site homolog to the vectors. In this protocol, we are using AgeI and SalI for heavy chain and AgeI and BsiWI for K chain.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Use sterile filter pipette tips for all reactions. All surfaces should be sprayed with DNA/RNase away. Resuspend lyophilized primers at 50 μM in RNase free water. Prepare a working solution mix by mixing equal volumes of each primer.</div>
<ol start="33">
<li>Thaw the first PCR plate for 5 min on ice.</li>
<li>The variable heavy and light chain genes are amplified by separate PCR reactions. Prepare PCR mix for the cloning PCR for each chain.</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th>Cloning PCR mix (1 plate)</th>
<th>Volume (μL)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>2,536</td>
<td> </td>
</tr>
<tr>
<td>Loading buffer</td>
<td>800</td>
<td> </td>
</tr>
<tr>
<td>10× Buffer</td>
<td>384</td>
<td>1×</td>
</tr>
<tr>
<td>dNTP (25 mM)</td>
<td>48</td>
<td>0.3 mM</td>
</tr>
<tr>
<td>5′F primers (50 μM)</td>
<td>15</td>
<td>0.3 μM</td>
</tr>
<tr>
<td>3′R primers (50 μM)</td>
<td>15</td>
<td>0.2 μM</td>
</tr>
<tr>
<td>HotStarTaq DNA polymerase (5 U/μL)</td>
<td>42</td>
<td>0.05 U/μL</td>
</tr>
</tbody>
</table>
<p>Primers (<a href="#tbl2">Table 2</a>):</p>
<p>5′ F Primers Heavy chain: T4hmFH_I / T4hmFH_II / T4hmFH_III / T4hmFH_IV / T4hmFH_V / T4hmFH_VI / T4hmFH_VII / T4hmFH_VIII / T4hmFH_IX</p>
<p>3′ R Primers Heavy chain: T4hmRG_I / T4hmRG_II / T4hmRG_III / T4hmRG_IV</p>
<p>5′ F Primers Light chain: T4hmFK_I / T4hmFK_II / T4hmFK_III / T4hmFK_IV / T4hmFK_V / T4hmFK_VI / T4hmFK_VII / T4hmFK_VIII / T4hmFK_IX</p>
<p>3′ R Primers Light chain: T4hmRK_I / T4hmRK_II / T4hmRK_III</p>
<ol start="35">
<li>Add 38 μL/well of cloning PCR mix in a new 96-well PCR plate.</li>
<li>Centrifuge the PCR 1 plate at 400 × <i>g</i> for 1 min at 4°C.</li>
<li>Add 4 μL of PCR 1 from each well to the same position in the Cloning PCR plate.</li>
<li>Place the plate in a thermocycler and run the following program:</li>
</ol>
<table id="undtbl13">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>15 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">50 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>50°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>55 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<ol start="39">
<li>Load 5 μL of the cloning PCR product from each well in a 2% agarose gel. Run the gel at 140 V for 20 min. The expected product is ~500 bp for the heavy chain and ~450 bp for the light chain.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Plates can be stored at 4°C for several days.</div>
<h3 id="sec3.7">Sequence and ligation-independent cloning (SLIC) of heavy and light chain genes in expression vectors</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<div class="critical">
<span class="critical-title">Critical:</span> Preset thermocyclers at 25°C and 42°C and bring LB plates containing the appropriate antibiotic to room temperature (20°C–22°C) (e.g., ampicillin).</div>
<ol start="40">
<li>Plasmid linearization: the expression vector containing the corresponding restrictions sites specific for IgH (AgeI and SalI) or IgK (AgeI and BSiWI) insertion need to be linearized with the appropriate restriction enzymes to allows the homologous recombination with the insert.
<ol type="a">
<li>Enzymatic digestion with the appropriate restriction enzymes. In this method, we used AgeI and SalI to linearize the plasmid containing the IgG constant region and AgeI and BsiWI to linearize the plasmid containing the IgK constant region.
<table id="undtbl14">
<thead>
<tr>
<th>Enzymatic digestion (1 reaction)</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>To final 50 μL</td>
</tr>
<tr>
<td>Vector</td>
<td>40 μg</td>
</tr>
<tr>
<td>Enzyme 1 (20,000 U/mL)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Enzyme 2 (20,000 U/mL)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Enzyme 10× Buffer</td>
<td>5 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Incubate at 37°C, overnight (12–16 h).</li>
<li>Add 1 μL of Alkaline phosphatase, calf intestinal (CIP) (stock 10,000 U/mL) to de-phosphorylate the 5′ and 3′ ends of the linearized vector. This will prevent the relegation of the plasmid. Incubate at 37°C for 30 min.</li>
<li>Purify the linearized plasmid. Run the digestion mix in a 1% agarose gel and purify the linearized vector from the gel by cutting the corresponding DNA band under the UV light and using a DNA purification system (NucleoSpin Gel and PCR Clean-up) according to manufacturer’s instructions (<a href="https://www.mn-net.com/media/pdf/f0/ee/cf/SP-NucleoSpin-Gel-and-PCR-Clean-up-vacuum-processing.pdf)">https://www.mn-net.com/media/pdf/f0/ee/cf/SP-NucleoSpin-Gel-and-PCR-Clean-up-vacuum-processing.pdf</a>).</li>
</ol>
</li>
</ol>
<ol start="41">
<li>Insert purification. Purify the products of the cloning PCR using the most suitable method in function of the number of samples. (e.g., QIAquick 96 PCR Purification Kit)</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Purified PCR products can be stored at 4°C for several days or at −20°C for years.</div>
<ol start="42">
<li>Sequence and Ligation-Independent Cloning (SLIC)
<ol type="a">
<li>Prepare the following ligation mix on ice.
<table id="undtbl15">
<thead>
<tr>
<th>Ligation mix (1 reaction)</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>To final 8.5 μL</td>
</tr>
<tr>
<td>Linearized vector</td>
<td>40 ng</td>
</tr>
<tr>
<td>BSA (10×)</td>
<td>1 μL</td>
</tr>
<tr>
<td>NEBuffer 2 (10×)</td>
<td>1 μL</td>
</tr>
<tr>
<td>T4 DNA Polymerase (3,000 U/mL)</td>
<td>0.5 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Mix 8.5 μL of ligation mix with 1/10 of the final volume of purified PCR product in a new 96-well PCR plate on ice.
<div class="critical">
<span class="critical-title">Critical:</span> The reaction must be prepared on ice to prevent the premature exonuclease activity of T4 polymerase.</div>
</li>
<li>Incubate the plates for 2.5 min at 25°C in thermocycler.
<div class="critical">
<span class="critical-title">Critical:</span> The incubation time has been optimized to generate overhangs of the appropriate length to ensure efficient SLIC.</div>
</li>
<li>Immediately place the plate on ice for at least 10 min to stop the reaction.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Plates can be stored at 4°C for several days.</div>
</li>
</ol>
</li>
</ol>
<ol start="43">
<li>Transformation.
<ol type="a">
<li>Thaw chemically competent DH5α cells on ice. Pipette 30 μL of competent cells into individual wells of a new 96-well plate.</li>
<li>While on ice add 4 μL of the ligation reaction to the competent cells, making sure the cells and DNA are in contact. Do not mix. Incubate on ice for 5 min.</li>
<li>Proceed to heat shock the bacteria at 42°C for 40 s in a thermocycler.</li>
<li>Immediately transfer the plate to ice and incubate for at least 1 min.</li>
<li>Add 40 μL of LB medium. Place the plate on a shaker at a 45° angle and incubate at 210 rpm and 37°C for 40 min.</li>
<li>Streak transformed cells on prewarmed LB-antibiotic agar plates.</li>
<li>Incubate LB-antibiotic agar plates over night at 37°C.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> LB plates can be stored at 4°C for several days.</div>
<h3 id="sec3.8">Screening of bacteria colonies and sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use sterile filter pipette tips for all reactions. All surfaces should be sprayed with DNA/RNAse away. Resuspend lyophilized primers at 50 μM in RNase free water. Prepare a working solution mix by mixing equal volumes of each primer.</div>
<ol start="44">
<li>Screen bacterial colonies by PCR using the 5′Ab sense forward primer and the corresponding IgH and IgK reverse primers used in PCR 2.</li>
<li>Prepare the colony PCR mixes.</li>
</ol>
<table id="undtbl16">
<thead>
<tr>
<th>Colony PCR mix (1 plate)</th>
<th>Volume (μL)</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>1,737</td>
<td> </td>
</tr>
<tr>
<td>Loading buffer</td>
<td>500</td>
<td> </td>
</tr>
<tr>
<td>10× Buffer</td>
<td>250</td>
<td>1×</td>
</tr>
<tr>
<td>dNTPs (25 mM)</td>
<td>13</td>
<td>0.1 mM</td>
</tr>
<tr>
<td>5′Ab sense (50 μM)</td>
<td>15</td>
<td>0.3 μM</td>
</tr>
<tr>
<td>3′R primer (50 μM)</td>
<td>15</td>
<td>0.3 μM</td>
</tr>
<tr>
<td>HotStarTaq DNA polymerase (5 U/μL)</td>
<td>20</td>
<td>0.04 U/μL</td>
</tr>
</tbody>
</table>
<p>Primers (<a href="#tbl2">Table 2</a>):</p>
<p>5′ F Primer: 5′ Ab sense</p>
<p>3′ R Primer: Reverse IgH or IgK primers from PCR 2</p>
<ol start="46">
<li>Add 25 μL/well of Colony PCR mix in a new 96-well PCR plate.</li>
<li>Using sterile pipette tips, pick individual bacterial colonies and introduce the tips in individual wells of a 96-well plate containing the colony PCR mix. It is recommended to pick 3 colonies per ligation reaction. Leave the tips inside the PCR mix until the next step.</li>
<li>Prepare a bacterial stock for each reaction by streaking each tip from the colony PCR plate on a LB-antibiotic agar plate. It is suggested to use a numbered grid to organize the bacteria stock on the agar plate.</li>
<li>Incubate LB-antibiotic agar plates at 37°C overnight (12–16 h).</li>
<li>Run the colony PCR plate in a thermocycler following the program:</li>
</ol>
<table id="undtbl17">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>15 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>57°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>50 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<ol start="51">
<li>Load 5 μL of Colony PCR product of each well in a 2% agarose gel. Run the gel at 140 V for 20 min. The expected product is ~500 bp for the heavy chain and ~450 bp for the light chain.</li>
<li>Sequence the colony PCR products by Sanger sequencing using the 5′Ab sense primer</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> LB–ampicillin agar and colony PCR plates can be stored at 4°C for several days.</div>
<ol start="53">
<li>Analyze the sequences of the Colony PCR products to identify correct sequences</li>
</ol>
<h3 id="sec3.9">Antibody DNA preparation and transfection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8 days</div>
<ol start="54">
<li>DNA preparation: Grow individual bacterial cultures carrying the heavy and light chain plasmids in LB media with antibiotic (e.g., ampicillin 0.1 mg/mL) at 37°C and 220 rpm, overnight (12–16 h). Purify plasmid DNA from the bacteria cultures using the preferred commercial DNA isolation system (NucleoBond Xtra Maxi) and follow the manufacturer’s instructions (<a href="https://www.mn-net.com/media/pdf/ed/82/0f/Instruction-NucleoBond-Xtra.pdf)">https://www.mn-net.com/media/pdf/ed/82/0f/Instruction-NucleoBond-Xtra.pdf</a>). Alternatively, the heavy and light chain cultures from a paired antibody can be mixed 50:50 and processed together resulting in a mixed preparation of heavy and light chain plasmid DNAs.</li>
</ol>
<ol start="55">
<li>Transfection of HEK293-6E cells:
<div class="critical">
<span class="critical-title">Critical:</span> Transfections should be done under a laminar flow hood. The cells used for antibody production are HEK293-6E cells. They grow in suspension in FreeStyle 293 expression medium supplemented with penicillin and streptomycin (10 000 U/mL).</div>
<ol type="a">
<li>Use HEK293-6E cell cultures in the range of 8.0 × 10E5 to 1.2 × 10E6 cells per mL for transfection.</li>
<li>Use 50 mL of cell culture in a 125 mL Erlenmeyer flask:
<ol type="i">
<li>Mix 25 μg of heavy chain DNA with 25 μg of light chain DNA or alternatively 50 μg of pre-combined heavy and light chain DNA and bring to a total of 2.3 mL with PBS 1×. Vortex.</li>
<li>Add 170 μL of 0.045% (wt/vol) polyethylenimine (PEI 0.34% of cell culture volume to be transfected). Vortex the mixture for at least 15 s.</li>
<li>Add the mixture to 50 mL of the HEK293-6E cell culture.</li>
</ol>
</li>
<li>Incubate for 5–10 days at 130 rpm, 37°C, 95% humidity and 8.0% CO<sub>2</sub>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.10">Fab purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="56">
<li>After 5–10 d, transfer the transfected HEK293-6E cell cultures to 50 mL tubes and centrifuge at 5,000 × <i>g</i> for 30 min at 4°C. Filter the supernatant using a 0.22-μm membrane filter to eliminate debris.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The supernatant can be kept at 4°C for several days. For longer times, it is recommended to add Azide (0.02%–0.05%) to prevent bacterial growth.</div>
<ol start="57">
<li>Purify Fabs by using Ni-beads according to the manufacturer’s instructions (<a href="https://www.sigmaaldrich.com/catalog/product/sigma/ge17531806?lang=en&amp;region=US)">https://www.sigmaaldrich.com/catalog/product/sigma/ge17531806?lang=en&amp;region=US</a>).<br> The Fab present in the culture supernatant are purified with Ni Sepharose beads because they are histidine-tagged. Ni-beads are recommended for high-resolution purification of histidine-tagged proteins, providing sharp peaks and concentrated eluate.
<ol type="a">
<li>To activate the Ni-beads and remove the ethanol-based storage buffer: add 500 μL of beads in a poly-prep chromatography column. Wash by flowing through the column 5 mL of distilled water. Add 1 mL of 20 mM Imidazole buffer (20 mM Imidazol, 20 mM Tris pH 7.5, 300 mM NaCl in water) and let go through by gravity. Add the end cap to the column and resuspend the beads with 500 μL of 20 mM Imidazole buffer, then transfer the beads to a new tube.</li>
<li>Add 200 μL of beads to 50 mL of cell supernatant containing the Fabs, and incubate the mixture overnight (12–16 h) at 4°C in a spinning wheel.</li>
</ol>
</li>
</ol>
<ol start="58">
<li>Equilibrate poly-prep chromatography columns with 5 mL of PBS 1×.</li>
<li>Centrifuge the supernatants for 10 min at 300 × <i>g</i> and 4°C to pellet the beads. Carefully decant 2/3 of the supernatant, and pour the remaining supernatant containing the beads into the column. Let pass through the column by gravity.</li>
<li>Wash one time with 1 mL of 20 mM Imidazole buffer followed by one time with 30 mM Imidazole buffer (30 mM Imidazol, 20 mM Tris pH 7.5, 300 mM NaCl in water).</li>
<li>Elute in 1.5 mL Eppendorf tubes using 600 μL of 250 mM Imidazole buffer (250 mM Imidazol, 20 mM Tris pH 7.5, 300 mM NaCl in water).</li>
<li>Depending on the subsequent use, it will be recommended to do buffer exchange with PBS 1× using 10K centrifugal filters according to the manufacturer’s instructions. (<a href="https://www.emdmillipore.com/US/en/product/Amicon-Ultra-4-Centrifugal-Filter-Unit,MM_NF-UFC801024?ReferrerURL=https%3A%2F%2Fwww.google.fr%2F&amp;bd=1">https://www.emdmillipore.com/US/en/product/Amicon-Ultra-4-Centrifugal-Filter-Unit,MM_NF-UFC801024?ReferrerURL=https%3A%2F%2Fwww.google.fr%2F&amp;bd=1</a>)</li>
<li>The Fab productivity is determined by measurement with nanodrop and PAGE analysis.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Fabs can be stored at 4°C for several years.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">Sequencing analysis</h3>
<p>The analysis of antibody sequences (<a href="#fig2">Figures 2</a>D–2F) provides the following information:</p>
<p>Clonal families: Clones are defined by antibodies sharing the same heavy and light germline and by calculating the hamming distance for the junction region (toolkit: Change-O) (<a href="#fig2">Figure 2</a>D).</p>
<p>Number of mutations: The number of mutations in the heavy and light chain genes of the isolated antibodies are determined by comparing the nucleotide sequences of their V genes with the corresponding germline V genes. (<a href="#fig2">Figure 2</a>E). Because non templated nucleotides (n-nucleotides) are added to the VDJ junctions during VDJ recombination and it is not possible to determine whether these nucleotides where subsequently somatically mutated, the junctions and DJ genes were not considered for the mutation analysis.</p>
<p>Phylogeny: phylogenetic relationships between antibody sequences can be analyzed (<a href="#fig2">Figure 2</a>F). The IgH and IgK antibody sequences are merged and aligned and subsequently analyzed using Gctree (<a href="#bib1">DeWitt et al., 2018</a>). If analyzing different antibody sequences from the same clone, the unmutated V gene sequence of the clone can be used for outgroup rooting.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The purified Fabs can be used to determine antigen binding affinities using Biolayer interferometry (BLI) (<a href="#fig2">Figures 2</a>G and 2H).</p>
<p>The following protocol was used to determine the binding affinity/avidity of a Fab to the TM4-Core HIV-1 trimer (<a href="#bib7">Viant et al., 2020</a>). BLI is an optical technique where binding events between the antigen and the Fab result in an increase in optical thickness on the tip of the biosensor that can be measured as a wavelength shift from the reference surface (<a href="#fig2">Figures 2</a>G and 2H). A basic kinetic assay can be split in four steps: the baseline (which must be flat), the loading (multiple concentrations of the loading protein need to be tested to find the best loading density), the association (several concentrations need to be tested) and the dissociation (the time of the dissociation step should be at least twice the one of the association).</p>
<ol>
<li>Affinity measurement (<a href="#fig2">Figure 2</a>G): To document monovalent interactions the HIV-1 TM4-Core trimer was immobilized on the biosensor chip and subsequently exposed to the cloned Fab.
<ol type="a">
<li>Antigen biotinylation. We randomly biotinylated TM4-Core using a 1:1 ratio of biotin versus TM4-Core. A low ratio of biotin in the reaction was selected to prevent epitope masking. We used the EZ-Link NHS-PEG4-Biotin kit and followed manufacturer’s instructions (<a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0016360_2161299_EZ_Link_NHS_PEG4_Biotin_UG.pdf)">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0016360_2161299_EZ_Link_NHS_PEG4_Biotin_UG.pdf</a>). We removed un-conjugated biotin using Zebra desalting columns (provide in the kit).</li>
<li>BLI assay with high precision streptavidin biosensor: (1) baseline: 60 s immersion in 1× kinetic buffer (kinetics buffer 10× diluted in PBS1×) (2) loading: 200 s immersion in a solution with biotinylated trimeric TM4-core in 1× kinetic buffer at 400 nM. (3) baseline: 200 s immersion in buffer. (4) Association: 300 s immersion in solution with Fab 10 μg/mL (5) dissociation: 600 s immersion in buffer. Curve fitting was performed using the Data analysis software (ForteBio).</li>
<li>All measurements of Antigen-Fab binding need to be corrected by subtracting the signal obtained in the presence of antigen but in absence of Fab.</li>
</ol>
</li>
<li>Avidity measurement (<a href="#fig2">Figure 2</a>H): To model polyvalent antigen-antibody interactions we coated the biosensor chip with the Fab to create a multivalent surface and exposed the chip to the HIV-1 TM4-Core trimer in solution.
<ol type="a">
<li>BLI assay with FAB2G biosensor: (1) baseline: 60 s immersion in 1× kinetic buffer (2) loading: 200 s immersion in a solution with Fab in 1× kinetic buffer at 40 μg/mL. (3) baseline: 200 s immersion in buffer. (4) Association: 300 s immersion in solution with TM4-core in 1× kinetic buffer at 400 nM (5) dissociation: 600 s immersion in buffer. Curve fitting was performed using the Data analysis software (ForteBio).</li>
<li>All measurements of Fab/TM4-Core binding need to be corrected by subtracting the signal obtained in the presence of Fab but in absence of TM4-Core.</li>
</ol>
</li>
</ol>
<p>Alternatively, binding experiments can be done by Surface Plasmon Resonance (SPR). SPR is a well-established technology developed 25 years ago, with higher sensitivity for small molecule detection than the more recent BLI method. However, BLI is sensitive enough to analyze antibody binding and has the advantage of being able to run up to 8 samples at the same time. In addition, BLI biosensors can be reused and the analyzed sample recovered, reducing the cost in material and antibody production.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The efficiency of Ig gene amplification by nested PCR is 40%–60%, therefore a significant number of cells needs to be processed in order to obtain a significant number of paired heavy- and light-chain genes to clone.</p>
<p>The efficiency of the nested PCR varies depending on the V genes used by the isolated B cells. It is possible that the heavy or light chain of a particular clone is not efficiently amplified by the PCR primers.</p>
<p>In this method, the B cells are not sorted based on their ability to bind antigen. Thus, even if the antibodies are specific for the antigen, the affinity of the produced Fab can be too low to be detected by BLI assays. It is possible to adapt the sort strategy to isolate antigen binding B cells (<a href="#bib9">Wang et al., 2020</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Some of the reagents are not available (<a href="#key-resources-table">key resources table</a>).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>We listed reagents that we tested and used but some other equivalent products can certainly be use. We recommend the users to test any new regents before starting the experiment.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Tamoxifen is hard to dissolve (step 2).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Vortex the tamoxifen before incubation. Incubate the tamoxifen in a rotator at 60°C for 15 min or more and vortex every 10–15 min. It has been reported that to avoid precipitation problems, tamoxifen can be dissolved in a 10% EtOH corn oil solution.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Very few DNA bands after PCR 2 (step 32).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>It is possible that the antibody sequences are not covered by the standard PCR forward and reverse primers. The primers have to be adapted to the mouse strain, antibody isotype, leader sequence, and V gene family.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Nested PCR yields positive bands in the negative control wells and the same antibody sequence is found across many wells (step 32).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>These results indicate that there is a contamination with an IgH or IgK DNA in the PCR reaction mix. The nested PCR protocol involves 100 cycles of amplification. This high number of cycles will amplify traces of any contaminant antibody DNA in the PCR mix. Contaminations should be avoided by working in a clean, nucleic acid-free area. Avoid working with antibody DNA preparations in the same area and the same day of antibody gene amplification by nested PCR.</p>
<p>If a contamination is detected, discard all the reagents and prepare fresh mixes.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Upon bacterial transformation of the SLIC product, we obtain bacterial colonies but they do not contain the insert of interest. Alternatively, upon bacterial transformation of the SLIC product, we do not obtain bacterial colonies (step 53).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>The linearization of the vector may have been inadequate. Repeat the digestion of the expression vector and remember to use CIP after vector digestion to prevent relegation.</p>
<p>Alternatively, there may be a problem with the insert or the vector: insert ratio used in the SLIC reaction. Confirm that the insert purification was efficient. Increase the amount of insert in the SLIC reaction.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>An antibody successfully sequenced after PCR 2 fails to be amplified by the cloning PCR (step 52).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>The antibody sequence obtained in PCR 2 can be ordered as a gene fragment including flanking restriction sites that will allow regular ligation in the corresponding expression vector.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>No production of antibodies (step 63).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>The antibodies sequences can be out of frame or miss-paired. Sequence the IgH and IgK plasmids to make sure that the cloning and the plasmid amplification have been done properly.</p>
<h3 id="sec7.15">Problem 8</h3>
<p>Fab affinity for the antigen is too low to be detected by BLI assays (<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</p>
<h3 id="sec7.16">Potential solution</h3>
<p>To increase the sensitivity of the assay, the kinetic experiment needs to allow several Fabs (or antibodies) to bind to one antigen. In our experiments, Fabs coated on the biosensor can bind to the same TM4-Core trimer. In addition, the antigen can be multimerized to enhance the avidity effect.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">DeWitt, W.S., Mesin, L., Victora, G.D., Minin, V.N., and Matsen, F.A.t. (2018). Using genotype abundance to improve phylogenetic inference. Mol. Biol. Evol. <i>35</i>, 1253-1265. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref1">View at publisher</a></p>
<p id="bib2">Dosenovic, P., von Boehmer, L., Escolano, A., Jardine, J., Freund, N.T., Gitlin, A.D., McGuire, A.T., Kulp, D.W., Oliveira, T., Scharf, L., et al. (2015). Immunization for HIV-1 broadly neutralizing antibodies in human Ig knockin mice. Cell <i>161</i>, 1505-1515. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref2">View at publisher</a></p>
<p id="bib3">Escolano, A., Gristick, H.B., Abernathy, M.E., Merkenschlager, J., Gautam, R., Oliveira, T.Y., Pai, J., West, A.P., Barnes, C.O., Cohen, A.A., et al. (2019). Immunization expands B cells specific to HIV-1 V3 glycan in mice and macaques. Nature <i>570</i>, 468-473. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref3">View at publisher</a></p>
<p id="bib4">Gitlin, A.D., von Boehmer, L., Gazumyan, A., Shulman, Z., Oliveira, T.Y., and Nussenzweig, M.C. (2016). independent roles of switching and hypermutation in the development and persistence of B lymphocyte memory. Immunity <i>44</i>, 769-781. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref4">View at publisher</a></p>
<p id="bib5">Madisen, L., Zwingman, T.A., Sunkin, S.M., Oh, S.W., Zariwala, H.A., Gu, H., Ng, L.L., Palmiter, R.D., Hawrylycz, M.J., Jones, A.R., et al. (2010). A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. Nat. Neurosci. <i>13</i>, 133-140. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref5">View at publisher</a></p>
<p id="bib6">Shinnakasu, R., Inoue, T., Kometani, K., Moriyama, S., Adachi, Y., Nakayama, M., Takahashi, Y., Fukuyama, H., Okada, T., and Kurosaki, T. (2016). Regulated selection of germinal-center cells into the memory B cell compartment. Nat. Immunol. <i>17</i>, 861-869. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref6">View at publisher</a></p>
<p id="bib7">Viant, C., Weymar, G.H.J., Escolano, A., Chen, S., Hartweger, H., Cipolla, M., Gazumyan, A., and Nussenzweig, M.C. (2020). Antibody affinity shapes the choice between memory and germinal center b cell fates. Cell <i>183</i>, 1298-1311.e11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref7">View at publisher</a></p>
<p id="bib8">von Boehmer, L., Liu, C., Ackerman, S., Gitlin, A.D., Wang, Q., Gazumyan, A., and Nussenzweig, M.C. (2016). Sequencing and cloning of antigen-specific antibodies from mouse memory B cells. Nat. Protoc. <i>11</i>, 1908-1923. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref8">View at publisher</a></p>
<p id="bib9">Wang, Z., Merkenschlager, J., Chen, S.T., Oliveira, T.Y., Ramos, V., Gordon, K.M., Yao, K.H., Jankovic, M., Nussenzweig, M., and Escolano, A. (2020). Isolation of single HIV-1 Envelope specific B cells and antibody cloning from immunized rhesus macaques. J. Immunol. Methods <i>478</i>, 112734. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/sref9">View at publisher</a></p>
<p id="bib10">Wardemann, H., Yurasov, S., Schaefer, A., Young, J.W., Meffre, E., and Nussenzweig, M.C. (2003). Predominant autoantibody production by early human B cell precursors. Science <i>301</i>, 1374-1377. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00096-4/opt3NraUUIgIZ">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for reagents and resources should be directed to and will be fulfilled by the lead contact Charlotte Viant (<a href="mailto:cviant@rockefeller.edu">cviant@rockefeller.edu</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents. The mouse lines obtained from other laboratories are described below and may require a Material Transfer Agreement (MTA) with the providing scientists.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate any unique datasets or codes<b>.</b></p>
<h3>Acknowledgments</h3>
<p>We thank T. Eisenreich for help with mouse colony management, K. Yao for technical help, K. Gordon for cell sorting, V. Ramos and T.Y. Oliveira for the bioinformatic analysis, all the members of the Nussenzweig laboratory for discussion, Dr. T. Kurosaki for S1PR2 mice, and A.T. McGuire and L. Stamatatos for providing the HIV-1 TM4-Core protein. This work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> (1P01AI138212), NIH (P01AI100148), NIH (5R37 AI037526), and NIH Center for HIV/AIDS Vaccine Immunology and Immunogen Discovery (CHAVID) (1UM1AI144462-01) to M.C.N. M.C.N. is an HHMI investigator.</p>
<h3>Author Contributions</h3>
<p>C.V. performed experiments. C.V, A.E., and S.T.C. improved the protocol. C.V., A.E., and M.C.N. wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>M.C.N. is an inventor on the patent for 3BNC60. The rights to 3BNC60 have been licensed to Gilead by Rockefeller University.</p>
</section>
</article>