<article>
<section>
<h1 id="header">Rapid preparation of human blood plasma for bottom-up proteomics analysis</h1>
<p><time datetime="2021-10-08">Published: October 8, 2021</time></p>
<p>Evgenia Shishkova<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> and Joshua J. Coon<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>National Center for Quantitative Biology of Complex Systems, Madison, WI 53706, USA</p>
<p id="aff2"><sup>2</sup>Department of Biomolecular Chemistry, University of Wisconsin - Madison, Madison, WI 53706, USA</p>
<p id="aff3"><sup>3</sup>Morgridge Institute for Research, Madison, WI 53715, USA</p>
<p id="aff4"><sup>4</sup>Department of Chemistry, University of Wisconsin - Madison, Madison, WI 53706, USA</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:shishkova@wisc.edu">shishkova@wisc.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:jcoon@chem.wisc.edu">jcoon@chem.wisc.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100856">10.1016/j.xpro.2021.100856</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol offers step-by-step instructions for preparation of raw blood plasma for liquid chromatography - tandem mass spectrometry (LC-MS/MS) analysis in clinical proteomics studies. The technique is simple, robust, and reproducible, and the entire transformation from plasma proteins to desalted tryptic peptides takes only 3–4 h. The protocol ensures efficient denaturation of native proteases that, in combination with the speediness of the procedure, prevents non-specific and irreproducible cleavage of digested peptides. The protocol can be adopted for large-scale studies and automation.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref18">Overmyer et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Instructions for preparation of human blood plasma for proteomics LC-MS/MS analysis</li>
<li>Transformation from plasma proteins to desalted tryptic peptides takes only 3–4 h</li>
<li>Rapid thermal denaturation of proteins minimizes native protease activity</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1062-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Thawing frozen plasma</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–120 min</div>
<ol>
<li>
<i>If plasma is not frozen, skip this step.</i> If the plasma sample is frozen, thaw on ice or in a refrigerator. Gently invert the tube to mix every 30 min. Depending on the total volume of plasma and the type of container, this step could take several hours.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> all human plasma samples are considered biohazardous and should be handled with extreme caution to prevent exposure to blood-borne pathogens. Consult your laboratory’s biosafety protocol and contact your institution’s Biosafety Committee if unsure of the best practices. In general, all procedures that may induce aerosol formation and release, such as tube opening, vortexing, and heating, must be performed inside a Class II Biosafety cabinet, up to <b>step 6</b> of the protocol. All relevant personal protective equipment (PPE), such as goggles, disposable gowns, disposable gloves, and others, as specified by the laboratory’s biosafety protocol, must be worn.</div>
<div class="critical">
<span class="critical-title">Critical:</span> strictly avoid vortexing, rapid shaking, or centrifuging plasma at high speeds (&gt; 2,000 × <i>g</i>). Harsh handling can induce <i>in vitro</i> coagulation and precipitation of plasma components, causing sample-to-sample irreproducibility in identification and quantitation of the involved proteins (see <a href="#sec6.1">problems 1</a> and <a href="#sec6.3">2</a> in <a href="#troubleshooting">troubleshooting</a><b>)</b>. Make notes on the color and consistency of the samples to reference later if needed.</div>
<h3 id="sec1.2">Getting equipment ready</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 30 min (dependent on specific equipment used)</div>
<ol start="2">
<li>Heat a sand bath (dry block heater) to 110°C.</li>
<li>Heat an incubator to 37°C.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human plasma, NaHep, pooled gender</td>
<td>BioIVT</td>
<td>Cat# HMN378062</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Lysyl Endopeptidase (Lys-C)</td>
<td>Wako Chemicals</td>
<td>Cat# NC9242798</td>
</tr>
<tr>
<td>Sequencing Grade Modified Trypsin</td>
<td>Promega</td>
<td>Cat# V5111</td>
</tr>
<tr>
<td>Acetonitrile (Optima™ LC/MS)</td>
<td>Fisher Scientific</td>
<td>Cat# A9554 (CS)</td>
</tr>
<tr>
<td>Methanol (Optima™ LC/MS)</td>
<td>Fisher Scientific</td>
<td>Cat# A454SK-4</td>
</tr>
<tr>
<td>Water (Optima™ LC/MS)</td>
<td>Fisher Scientific</td>
<td>Cat# W6-4</td>
</tr>
<tr>
<td>TCEP HCl (tris(2-carboxyethyl)phosphine hydrochloride) for biotechnology</td>
<td>VWR</td>
<td>Cat# VWRVK831-10G</td>
</tr>
<tr>
<td>2-Chloroacetamide, ≥98.0% (HPLC)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 22790-250G-F</td>
</tr>
<tr>
<td>Pierce™ Trifluoroacetic Acid (TFA), Sequencing grade</td>
<td>Thermo Scientific</td>
<td>Cat# 28901</td>
</tr>
<tr>
<td>Guanidine hydrochloride, for molecular biology, ≥99%</td>
<td>Sigma-Aldrich</td>
<td>Cat# G3272-500G</td>
</tr>
<tr>
<td>Invitrogen™ Tris (1 M), pH 8.0, RNase-free</td>
<td>Fisher Scientific</td>
<td>Cat# AM9856</td>
</tr>
<tr>
<td>Urea powder, BioReagent, for molecular biology, suitable for cell culture</td>
<td>Sigma-Aldrich</td>
<td>Cat# U5378-1KG</td>
</tr>
<tr>
<td>Pierce™ Formic Acid, LC-MS Grade</td>
<td>Thermo Scientific</td>
<td>Cat# PI28905</td>
</tr>
<tr>
<td>tert-Butyl methyl ether, 1 L</td>
<td>Sigma-Aldrich</td>
<td>Cat# 443808-1L</td>
</tr>
<tr>
<td>cOmplete™, Mini, EDTA-free Protease Inhibitor Cocktail</td>
<td>Roche</td>
<td>Cat# 11836170001</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Quantitative Colorimetric Peptide Assay</td>
<td>Pierce</td>
<td>Cat# 23275</td>
</tr>
<tr>
<td>BCA Protein Assay Kit - Reducing Agent Compatible</td>
<td>Pierce</td>
<td>Cat# 23250</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Strata™-X 33 μm Polymeric Reversed Phase, 10 mg / well, 96-Well Plates</td>
<td>Phenomenex</td>
<td>Part# 8E-S100-AGB</td>
</tr>
<tr>
<td>Strata™-X 33 μm Polymeric Reversed Phase, 10 mg / 1 mL, Tubes , 100/Pk</td>
<td>Phenomenex</td>
<td>Part# 8B-S100-AAK</td>
</tr>
<tr>
<td>Strata™-X 33 μm Polymeric Reversed Phase, 30 mg / 1 mL, Tubes , 100/Pk</td>
<td>Phenomenex</td>
<td>Part# 8B-S100-TAK</td>
</tr>
<tr>
<td>Strata® 96-Well Plate Manifold, Universal, w/ Vacuum Gauge</td>
<td>Phenomenex</td>
<td>Part # AH0-8950</td>
</tr>
<tr>
<td>Whatman® Panpeha™ pH indicator strips, pH range 0 to 14, 200/pk</td>
<td>Millipore Sigma</td>
<td>Cat# WHA10360005</td>
</tr>
<tr>
<td>PrecisionGlide™ Hypodermic Conventional Needles</td>
<td>BD</td>
<td>Cat# 14-826-10</td>
</tr>
<tr>
<td>Heratherm Compact Incubator</td>
<td>Lab Depot, Inc.</td>
<td>Cat# 50125590</td>
</tr>
<tr>
<td>Digital Dry Baths/Block Heater with open container style block filled with sand</td>
<td>Thermo Scientific</td>
<td>Cat# 88870001</td>
</tr>
<tr>
<td>3′ Purifier Logic+ Class II A2 Biological Safety Cabinet with 10" sash opening</td>
<td>Labconco Corporation</td>
<td>Cat# 302310001</td>
</tr>
<tr>
<td>Ducted Chemical Fume Hood with a Vertical Sash</td>
<td>Labconco Corporation</td>
<td>Cat# 110410000</td>
</tr>
<tr>
<td>NanoDrop™ One/OneC Microvolume UV-Vis Spectrophotometer</td>
<td>Thermo Scientific</td>
<td>Cat# ND-ONE-W</td>
</tr>
<tr>
<td>Q700 Sonicator</td>
<td>QSonica</td>
<td>Cat# Q700-110</td>
</tr>
<tr>
<td>ELx808™ Absorbance Microplate Reader</td>
<td>BioTek</td>
<td>n/a</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> using chemicals produced by manufacturers other than the ones specified in the Key Resource Table is permitted, if they are of highest purity, sequence grade, and are LC/MS grade. The exception is sequencing grade modified trypsin (Promega), which is a key to the success of the procedure and should not be substituted with other similar products.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> using the specific brands of the compact incubator and the dry heater is not essential; they can be substituted for any pieces of equipment capable of reaching the required temperature and securely holding 1.5 mL conical-bottom tubes. Likewise, any Class II Biosafety cabinet, chemical fume hood, temperature-controlled sonicator, and microplate reader with similar specifications can be deployed, in place of the ones listed in the Key Resource Table.</div>
<table id="undtbl1">
<caption>10x stock of 100 mM tris(2-carboxyethyl) phosphine hydrochloride (TCEP) and 400 mM 2-chloroacetamide (CAA)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TCEP (&gt;99% purity)</td>
<td>100 mM</td>
<td>286.65 mg</td>
</tr>
<tr>
<td>2-Chloroacetamide (≥98.0% purity)</td>
<td>400 mM</td>
<td>374.04 mg</td>
</tr>
<tr>
<td>LC/MS grade water</td>
<td>n/a</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[The 10x stock can be aliquoted into ten 1 mL portions and stored frozen at −40°C or colder for up to 1 year].</p>
<div class="caution">
<span class="caution-title">Caution:</span> Tris(2-carboxyethyl) phosphine hydrochloride is corrosive and can cause severe skin and eye burns. 2-chloroacetamide is toxic upon ingestion, may cause an allergic skin reaction, and a suspected reproductive toxin. Appropriate PPE (<i>e.g.</i>, gloves, goggles, and a laboratory coat) should be used while working with these substances.</div>
<table id="undtbl2">
<caption>Lysis buffer A (LBA)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Guanidine hydrochloride (≥99% purity)</td>
<td>6 M</td>
<td>28.66 g</td>
</tr>
<tr>
<td>10x stock 100 mM TCEP and 400 mM CAA</td>
<td>1x (10 mM TCEP and 40 mM CAA)</td>
<td>5 mL</td>
</tr>
<tr>
<td>Tris (1 M, pH 8)</td>
<td>100 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>LC/MS grade water</td>
<td>n/a</td>
<td>40 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[The buffer can be stored at ambient temperature on a lab bench for up to 1 month].</p>
<div class="caution">
<span class="caution-title">Caution:</span> Guanidine hydrochloride and Tris are skin and eye irritants. Appropriate PPE (<i>e.g.</i>, gloves, goggles, and a laboratory coat) should be used while working with these substances.</div>
<table id="undtbl3">
<caption>Lysis buffer B (LBB)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Urea (&gt; 99% purity)</td>
<td>8 M</td>
<td>4.8 mg</td>
</tr>
<tr>
<td>Tris (1 M, pH 8)</td>
<td>100 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>10x stock 100 mM TCEP and 400 mM CAA</td>
<td>1x (10 mM TCEP and 40 mM CAA)</td>
<td>1 mL</td>
</tr>
<tr>
<td>LC/MS grade water</td>
<td>n/a</td>
<td>Up to 10 mL (watch out for volume displacement)</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[The buffer must be made fresh daily before the procedure and cannot be stored].</p>
<div class="critical">
<span class="critical-title">Critical:</span> LBB should be prepared <i>fresh daily</i> before the procedure. Never store LBB due to undesirable reactivity with primary amines.</div>
<table id="undtbl4">
<caption>LysC solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>LysC (lyophilized)</td>
<td>1 μg/ μL</td>
<td>Lot specific</td>
</tr>
<tr>
<td>LC/MS grade water</td>
<td>n/a</td>
<td>Determined as described below</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>Determined as described below</td>
</tr>
</tbody>
</table>
<p class="table-legend">[The reagent can be aliquoted into 20 or 50 μL portions and stored frozen at -40°C or colder for up to 1 year].</p>
<ul>
<li>Obtain specific activity measurement from the product label (typically between 10–12 activity units per vial, for example 11.7 AU/vial).</li>
<li>Using the product Lot number, obtain online the Certificate of Analysis for the specific batch of enzyme your product was manufactured as.</li>
<li>From the Certificate of Analysis, determine the activity-to-enzyme mass conversion factor (for example, 4.2 AU/mg).</li>
<li>Multiple specific activities per vial by the activity-to-enzyme mass conversion factor, to determine the mass of enzyme in the vial. For example, a vial with 11.7 AU of LysC at 4.2 AU/mg contains 2.925 mg of enzyme.</li>
<li>Resuspend the lyophilized enzyme at final concentration of 1 μg/ μL by gently pipetting water and inverting the vial 10–20 times. In the above example, adding 2.925 mL of water produces the correct final concentration.</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> Do not vortex enzymes (LysC and trypsin) when resuspending or thawing. Mix only by gentle pipetting or inversion.</div>
<table id="undtbl5">
<caption>Dilution buffer (100 mM Tris in water, pH 8)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (1 M, pH 8)</td>
<td>100 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>LC/MS grade water</td>
<td>n/a</td>
<td>22.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[The buffer must be made fresh daily before the procedure and cannot be stored].</p>
<div class="critical">
<span class="critical-title">Critical:</span> Dilution buffer should be prepared <i>fresh</i> before the procedure. Never store 100 mM Tris in H<sub>2</sub>O due to a possible contamination with biological materials.</div>
<table id="undtbl6">
<caption>Desalting solvent: cartridge washing (0.1% trifluoroacetic acid (TFA) in water)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Trifluoroacetic acid (&gt; 99% purity)</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>LC/MS grade water</td>
<td>n/a</td>
<td>499.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[The reagent can be stored at ambient temperature on a lab bench for up to 3 months. With prolonged storage, check pH of the solvent prior to experiments using pH strips and use only if pH of less than 2 is observed].</p>
<div class="caution">
<span class="caution-title">Caution:</span> Trifluoroacetic acid is an irritant and corrosive to the skin, eyes, and respiratory tract tissues, acutely toxic by ingestion and inhalation, and corrosive to metals. Appropriate PPE and safety equipment (<i>e.g.</i>, gloves, goggles, a laboratory coat, and a chemical fume hood) should be used while working with this acid.</div>
<table id="undtbl7">
<caption>Desalting solvent: peptide elution (80% acetonitrile (ACN) in 0.1% trifluoroacetic acid (TFA) in H<sub>2</sub>O)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Trifluoroacetic acid (&gt; 99% purity)</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>LC/MS grade acetonitrile (&gt;99% purity)</td>
<td>80%</td>
<td>400</td>
</tr>
<tr>
<td>LC/MS grade water</td>
<td>n/a</td>
<td>99.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[The reagent can be stored at ambient temperature on a lab bench for up to 1 month. With prolonged storage, check pH of the solvent prior to experiments using pH strips and use only if pH of less than 2 is observed].</p>
<div class="caution">
<span class="caution-title">Caution:</span> Acetonitrile is highly flammable and considered acutely toxic solvent upon skin exposure. Appropriate PPE (<i>e.g.</i>, laboratory coat, goggles, and gloves) should be worn when working with this solvent. A chemical fume hood should be used when working with large volumes of this solvent.</div>
<table id="undtbl8">
<caption>Peptide resuspension solvent (0.2% formic acid in water)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Formic acid (&gt; 99% purity)</td>
<td>0.2%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>LC/MS grade water</td>
<td><b>n/a</b></td>
<td>249.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>250 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">[The reagent can be stored at ambient temperature on a lab bench for up to 3 months. With prolonged storage, check pH of the solvent prior to experiments using pH strips and use only if pH of less than 2 is observed].</p>
<div class="caution">
<span class="caution-title">Caution:</span> Formic acid is flammable, an irritant, corrosive to skin, eyes, and respiratory tract, acutely toxic by ingestion and inhalation, and corrosive to metals. Appropriate PPE and safety equipment (<i>e.g.</i>, gloves, goggles, a laboratory coat, and a chemical fume hood) should be used while working with this acid.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Protein denaturation and extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–40 min</div>
<p>Blood plasma is a protein-rich biofluid (60–80 mg/mL for healthy individuals; <a href="#bib1">Anderson and Anderson 2002</a>; <a href="#bib12">Leeman et al., 2018</a>) that does not require lengthy treatment to release proteins, making it quite amenable to proteomic analysis. However, plasma contains numerous native proteases that, if not denatured effectively, can cleave proteins in the non-specific and unpredictable manner. Rapid thermal denaturation, the 1<sup>st</sup> step of the procedure, ensures that the plasma proteins are later extracted reproducibly and to completion, and the activity of the native proteases does not interfere with the downstream steps of the sample preparation. Guanidine hydrochloride is selected as a chaotropic agent because of its thermostability, relative chemical inertness at high temperatures, and ability to solubilize proteins across a broad range of the hydrophobicity scale.</p>
<ol>
<li>Label one 1.5 mL conical-bottom Eppendorf tube per plasma sample and transfer a 50 μL aliquot of LBA.</li>
<li>While working in a Biosafety cabinet, gently invert thawed plasma samples 3–5 times, examine their appearance, and make notes on each sample (see <a href="#sec6.1">problems 1</a> and <a href="#sec6.3">2</a> in <a href="#troubleshooting">troubleshooting</a>).</li>
<li>While working in a Biosafety cabinet, denature plasma samples.
<ol type="a">
<li>Pipet 5 μL of plasma into the prepared tubes with LBA.</li>
<li>Vortex for 5 s at 2,000–3,200 × <i>g.</i>
</li>
<li>Spin down to ensure all liquid is at the bottom of the tubes.</li>
<li>Place the tubes into the heating block (sand bath) at 100°C–110°C for 10 min.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Continuous heating of the samples for more than 10 min could cause thermal damage, protein coagulation, and undesirable chemical modifications (see <a href="#sec6.5">problem 3</a> in <a href="#troubleshooting">troubleshooting</a><b>)</b>.</div>
<div class="note">
<span class="note-title">Note:</span> Make sure that the tube labels do not peel or wipe off when heated in a sand bath. Mark both tube bodies and caps.</div>
<div class="note">
<span class="note-title">Note:</span> To avoid solidification of plasma components, the optimal ratio of plasma proteins to volume of LBA should be kept under 10:1 of μg plasma protein to μL LLA.</div>
<ol start="4">
<li>Take the tubes out and let them cool down to ambient temperature on the benchtop.</li>
<li>Precipitate proteins by addition of methanol. After this step the plasma samples are no longer considered infectious and could be handled outside the Biosafety cabinet.
<ol type="a">
<li>Add a 450 μL volume of methanol (up to ∼90% methanol vol/vol) to each tube and vortex for 5 s at 2,000–3,200 × <i>g.</i>
</li>
<li>Centrifuge down for 6 min at 13,000 × <i>g</i> at ambient temperature.</li>
</ol>
</li>
</ol>
<div class="caution">
<span class="caution-title">Caution:</span> Methanol is highly flammable, a suspected fetal toxin, an eye irritant, and considered an acutely and chronically toxic solvent. Appropriate PPE (<i>e.g.</i>, a laboratory coat, goggles, and gloves) should be worn when working with this solvent. A chemical fume hood should be used when working with large volumes of this solvent.</div>
<div class="note">
<span class="note-title">Note:</span> Avoid overspinning and excessively compressing the protein pellet, which could make it difficult to resuspend in the subsequent steps.</div>
<div class="alternative">
<span class="alternative-title">Alternative:</span> The exact timing and speed of centrifugation are not critical. If the high-speed centrifuge is not available, the samples could be spun at lower speed for a longer time (<i>e.g.</i>, 20 min at 5,000 × <i>g</i>).</div>
<ol start="6">
<li>At this point slanted white-to-yellow protein pellets should be visible at the bottom of the tubes (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1062-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Photograph of protein pellet formed upon addition of methanol in steps 4 and 5</div>
</figcaption>
</figure>
<ol type="a">
<li>Decant supernatant while avoiding disturbing the pellets and air-dry the tubes on the sides for 3–5 min.</li>
<li>Invert the tubes and tap gently to remove the remaining supernatant.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not overdry the pellets; a small amount of residual methanol will not interfere with the following steps of the procedure. Overdrying, however, could make resuspending the pellets challenging.</div>
<ol start="7">
<li>Resuspend in digestion buffer LBB.
<ol type="a">
<li>Add a 100 μL aliquot of freshly made LBB to each tube.</li>
<li>Vortex at 2,000–3,200 × <i>g</i> to resuspend the pellets (∼5–15 min). If visible pellets persist, sonicate in a temperature-controlled ultrasonic water bath at 4°C–30°C, until they fully resuspend.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> never heat peptides and proteins in urea-containing buffers to over 60°C, as at high temperatures urea can rapidly react with and covalently modify primary amines, such N-termini of proteins and lysine residues. These modifications change mass of peptides and proteins and interfere with their identification during database searching.</div>
<div class="optional">
<span class="optional-title">Optional:</span> when working with plasma of healthy individuals, peptide yields stated in <a href="#expected-outcomes">expected outcomes</a> should be readily observed, and the procedure can be carried out exactly as described, without the need for protein concentration determination step. During analysis of plasma of severely ill patients, however, a wider range of protein concentrations could exist (see <a href="#expected-outcomes">expected outcomes</a> and <a href="#sec6.11">problem 6</a> in <a href="#troubleshooting">troubleshooting</a>). To account for that, researchers may choose to perform protein concentration determination using commercial assays, such as Reducing Agent Compatible BCA Protein Assay Kit from Pierce, according to the manufacturer’s instructions.</div>
<h3 id="sec3.2">Protein digestion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5 hrs</div>
<p>Trypsin digestion is a workhorse of bottom-up proteomics (<a href="#bib20">Vandermarliere et al., 2013</a>). Pre-digestion with another protease, LysC, is often performed to boost digestion efficiency and to reduce the number of missed cleavages, especially at lysine residues that are preceded by prolines. When sequencing grade enzymes are used, this reaction is generally specific and reproducible and proceeds mostly to completion. Special attention, however, should be paid to pH and ionic strength of the digestion buffers, as trypsin activity is at its peak at pH 6.5–8.5 and concentrations of denaturing agents, such as urea, below 1.5 M (see <a href="#sec6.7">problem 4</a> in <a href="#troubleshooting">troubleshooting</a>). LysC is not as sensitive to buffer conditions and hence can withstand higher concentrations of urea.</p>
<ol start="8">
<li>Add LysC enzyme for the 1<sup>st</sup> step of sequential digestion.
<ol type="a">
<li>Pipet 7 μL LysC solution (∼1:50 enzyme: protein ratio) and gently invert tubes 5 times to mix.</li>
<li>Centrifuge at &lt;2,000 × <i>g</i> at ambient temperature to collect liquid at the bottom of the tubes.</li>
<li>Place the tubes into an incubator at 37°C for 1 h.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the required mass of enzymes and the reported enzyme-to-protein ratio of 1:50 (LysC in <b>step 8</b> and trypsin in <b>step 9</b>) is calculated based on the expected average yield of peptides (see <a href="#expected-outcomes">expected outcomes</a> for more information). Samples with moderately higher and lower concentrations could be processed without the need for adjustments in volumes of enzymes added, as enzyme-to-protein ratios between 1:25 and 1:100 are equally effective.</div>
<ol start="9">
<li>Add trypsin enzyme for the 2<sup>nd</sup> step of sequential digest.
<ol type="a">
<li>Dilute LBB (8 M urea) down to below 1.5 M urea with a 450 μL volume of freshly made 100 mM Tris (pH 8).</li>
<li>Add 17.5 μL trypsin (∼1:50 enzyme: protein ratio) and invert 5 times to mix.</li>
<li>Centrifuge at &lt;2,000 × <i>g</i> at ambient temperature to collect liquid at the bottom of the tubes.</li>
<li>Place the tubes back into an incubator at 37°C for at least 1 h (see <a href="#sec6.7">problem 4</a> in <a href="#troubleshooting">troubleshooting</a>).</li>
</ol>
</li>
<li>To stop enzymatic activity and to prepare the proteins for latter solid-phase extraction (SPE), acidify the samples.
<ol type="a">
<li>Add a 50 μL volume of 10% TFA (to the final concentration of ∼1% TFA by volume).</li>
<li>Vortex for 5 s at &gt;2,000 × <i>g</i> at ambient temperature.</li>
<li>Confirm that pH of the samples has decreased below 2 by pipetting 2 μL of the acidified mixture on pH indicator strips.</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Peptide desalting and purification via solid phase extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Solid phase extraction (SPE) is a popular technique for purification of digested peptides and removal of chemicals and buffer components that may interfere with chromatographic separations, electrospray ionization, and/ or MS analysis (<a href="#bib2">Bladergroen and van der Burgt 2015</a>). Numerous products featuring diverse solid phase chemistries are currently available, most of which work through the same basic steps: loading of peptides, washing of interfering compounds, and elution of purified peptides. Strata™-X 33 μm Polymeric Reversed Phase cartridges, suggested for use in this protocol, offer the advantage of speed and ease of implementation but could be substituted with many other SPE products (Sep-Pak from Waters, ZIPTIP® C18 pipette tips from Merck Millipore, among others).</p>
<ol start="11">
<li>Prepare the desalting setup (see <a href="#fig2">Figure 2</a> for illustrations to <b>steps 11–17</b>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1062-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Illustration of the desalting set-up using Strata™-X 33 μm Polymeric Reversed Phase cartridges, described in steps 11–17</div>
</figcaption>
</figure>
<ol type="a">
<li>Use a needle to pierce through caps of 50 mL conical tubes that will serve to hold waste eluents (up to six SPE cartridges per tube) with at least two holes per cap.</li>
<li>Use the same needle to pierce through the caps of 1.5 mL conical-bottom Eppendorf tubes (one tube per plasma sample) with at least two holes per cap.</li>
<li>Label the sides of the 1.5 mL tubes but do not write on the caps. These tubes will serve to collect eluted peptides.</li>
<li>Place freshly uncapped needles through the holes of the waste 50 mL tubes and insert Strata™-X 33 μm Polymeric Reverse Phase cartridges 10 mg/ 1 mL onto the needles (see <a href="#sec6.9">problem 5</a> in <a href="#troubleshooting">troubleshooting</a>).</li>
<li>Place freshly uncapped needles through the holes in the caps of 1.5 mL conical-bottom Eppendorf tubes.</li>
</ol>
</li>
<li>Prime the cartridges by washing with a 1 mL of 100% ACN. Let liquid flow through into a waste container unassisted, with the force of gravity only, until dry.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> when simultaneously desalting more than 16 samples, the use of Strata™-X 33 μm Polymeric Reversed Phase, 10 mg / well cartridges in the 96-well plate format is recommended. The needles and collection tubes are not required in this case, and purified peptides (<b>step 17</b>) could be collected into a commercial 96-well plate, such as TrueTaper® 96-Well Collection Plate from Analytical.</div>
<div class="note">
<span class="note-title">Note:</span> avoid using mechanical force, such as plungers, to push liquid through the SPE cartridges and let the liquid drip through unassisted, as much as feasible. If slow drip of liquid is observed though the column prior to loading the sample (<b>step 12</b>), discard the column and use a new one.</div>
<ol start="13">
<li>Wash cartridges with a 1 mL volume of 0.1% TFA in water and let liquid flow through until dry into waste.</li>
<li>Load digested peptides.
<ol type="a">
<li>Spin acidified samples down for 5 min at 10,000 × <i>g.</i>
</li>
<li>While avoiding pipetting any precipitates, load samples onto the prepared SPE cartridges.</li>
<li>Wait for the liquid to flow through into waste.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> when the protocol is followed closely and the recommended concentration of sample is used, no major issues with clogging of SPE cartridges are expected. However, in rare cases, especially when working with many samples in the 96-well format, extra negative pressure may be needed to draw liquid through the clogged packing bed. In this case, the use of plunges or vacuum manifolds, such as Strata® 96-Well Plate Manifold with Vacuum Gauge, could be considered.</div>
<ol start="15">
<li>Wash with a 1 mL volume of 0.1% TFA in water and let the drip flow through into waste.</li>
<li>Move cartridges containing washed peptides to collection tubes with fresh needles, while leaving the used needles in Falcon tubes for future reuse if desired.</li>
<li>Elute peptides by pipetting a 300 μL volume of 80% ACN in 0.1% TFA and collect the flow-through, which contains purified peptides.</li>
<li>Lyophilize or vacuum centrifuge to dry eluted peptides.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> lyophilized peptides could be frozen at this point and stored dry for prolonged periods of time (days to months) at temperature below −20°C.</div>
<div class="note">
<span class="note-title">Note:</span> this protocol is designed with label-free quantification in mind, such as MaxLFQ (<a href="#bib5">Cox et al., 2014</a>) and parallel reaction monitoring (<a href="#bib19">Peterson et al., 2012</a>). However, if other types of peptide quantification are desired, for example multiplexed quantification with isobaric tagging, desalted and dried peptides produced at the end of <b>step 18</b> can be carried further to perform chemical tagging. Likewise, resultant tryptic peptides could be fractionated or enriched for post-translational modifications (PTMs), such as phosphorylation or glycosylation. Please see <a href="#sec6.9">problem 5</a> in <a href="#troubleshooting">troubleshooting</a> for more ideas.</div>
<ol start="19">
<li>Resuspend the samples.
<ol type="a">
<li>Add a 300 μL volume of 0.2% formic acid in LC/MS grade water.</li>
<li>Vortex at 2,000–3,200 × <i>g</i> for 10–15 min.</li>
<li>Centrifuge for 10 min at 13,000 × <i>g</i> at 4°C.</li>
</ol>
</li>
<li>Obtain peptide concentration measurements of the samples using NanoDrop™ or Pierce™ Quantitative Colorimetric Peptide Assay, according to manufacturer’s instructions.</li>
<li>Transfer volumes of samples necessary for analysis into autosampler vials compatible with the liquid chromatography system of choice; if upon spinning in <b>step 19</b> precipitation was observed, avoid disturbing the precipitate and pipet only the supernatant.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the described protocol is agnostic to the specifics of LC/MS-MS analysis that follows, as long as the method is designed to analyze tryptic peptides. Numerous separation strategies, data acquisition methods, and instrument types could be used. Please access the following references for more details: <a href="#bib6">Gallien et al. (2015)</a>, <a href="#bib7">Geyer et al. (2016)</a>, <a href="#bib14">Meier et al. (2018)</a>, <a href="#bib15">Messner et al. (2020)</a>, and <a href="#bib18">Overmyer et al. (2020)</a>.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Protein concentration in human blood plasma could vary from person to person, depending on the individual’s health status, hydration, medication use, and other factors (<a href="#bib1">Anderson and Anderson 2002</a>; <a href="#bib16">Nakayasu et al., 2021</a>). However, when working with pooled plasma of healthy individuals, such as pooled gender human plasma from BioIVT or NIST 1950 plasma standard, this protocol on average yields ∼331.5 μg (relative standard deviation across eight replicates of 7.1%) of tryptic peptides from a 5 μL volume of sample (<a href="#fig3">Figure 3</a>A and <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1062-Mmc1.xlsx">Table S1</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1062-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Experimental peptide yields obtained using the protocol.</div>
<p>(A) Histogram of experimental peptide yields obtained from eight technical replicates of control samples (5 μL volumes of pooled gender human plasma from BioIVT), analyzed in <a href="#bib18">Overmyer et al. (2020)</a>.</p>
<p>(B) Histogram of experimental peptide yields obtained from 5 μL volumes of 128 unique patient blood plasma samples, analyzed in <a href="#bib18">Overmyer et al. (2020)</a>.</p>
</figcaption>
</figure>
<p>The initial concentration of plasma proteins, of course, affects the final peptide yields. The histogram in <a href="#fig3">Figure 3</a>B displays the results of an examination of plasma protein concentration in the population of 128 individuals of highly variable health statuses (<a href="#bib18">Overmyer et al., 2020</a>). The plot illustrates that plasma peptide yields could exhibit greater variability in diverse clinical samples than in samples obtained from healthy individuals. Hence, we recommend including technical control samples of pooled plasma alongside of clinical plasma samples, to monitor all steps of the protocol based on the achievement of the expected peptide yields. Please also see <a href="#sec6.11">problem 6</a> in <a href="#troubleshooting">troubleshooting</a> for further discussion of clinical samples of varying protein concentrations.</p>
<p>If both protein concentration determination assay (Optional after <b>step 7</b>) and peptide concentration determination assay (<b>step 20</b>) were performed, not greater than 10–20% losses of material are expected. For example, if the original amount of protein in the sample prior to digestion is 350 μg, at least 280 μg of tryptic peptides should be yielded at the end of the procedure.</p>
<p>Further, no more than 10–15% of identified peptide sequences should contain missed cleavages of internal lysine and arginine residues, and the percent of missed cleavages detected should be consistent (within 10%) across all samples prepared simultaneously (please see <a href="#sec6.7">problem 4</a> in <a href="#troubleshooting">troubleshooting</a>). Lastly, if the procedure is carried out correctly, virtually no identified peptide sequences should be products of semi-tryptic cleavage, i.e., sequences with amino acids at one or both ends that do not correspond to expected products of tryptic digest of the original proteins (please see <a href="#sec6.5">problem 3</a> in <a href="#troubleshooting">troubleshooting</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol assumes that the plasma samples to be analyzed are of high quality and have not been compromised during collection, handling, and storage. Although we offer some tips and suggestions on how to identify and handle problematic samples (see <a href="#sec6.1">problems 1</a> and <a href="#sec6.3">2</a> in <a href="#troubleshooting">troubleshooting</a>), special care should be taken to avoid having to work with them in the first place, as no amount of computational manipulation can fully compensate for poor sample quality, for example, due to <i>in vitro</i> hemolysis or coagulation.</p>
<p>The protocol is equally suitable for plasma samples collected in the presence of citrate, heparin, and di- and tri-potassium—EDTA as anticoagulants. We, however, have not specifically assessed protocol’s compatibility with plasma samples containing sodium fluoride/ potassium oxalate or enzymatic inhibitors, such as pyridostigmine. Although we do not foresee any obvious issues due to the presence of these compounds, we urge future users to evaluate robustness and reproducibility of the protocol prior to working with the samples containing these anticoagulants.</p>
<p>As the identity of anticoagulation agents and the type of tubes used during plasma collection impacts the plasma proteome (<a href="#bib21">Ilies et al., 2017</a>; <a href="#bib9">Halvey et al., 2021</a>), all samples to be compared should be collected using the same methodology. Our protocol offers no solutions for issues that may arise due to differences in plasma collection procedures.</p>
<p>Due to the addition of guanidinium chloride and thermal treatment of samples, the described preparation procedure is not compatible with analyses of small molecules, such as lipids or metabolites, and nucleic acids. If multi-omic analysis is desired, separate aliquots of plasma should be used for proteomic and other analyses.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The quality of the plasma samples is compromised due to mistakes in handling made <i>prior</i> or <i>during</i> the sample preparation procedure (<b>step 3</b>). For example, the plasma samples could be hemolyzed as indicated by an orange or reddish color, suggesting contamination with lysed red blood cells, or contain large amounts of visible blood clots and solid fibrils.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>To the best of our knowledge, removing red blood cell contamination or re-dissolving clots formed <i>in vitro</i> is not possible, and it is best to exercise extreme caution during collection and handling of raw plasma. However, such issues could be at least partially mitigated during data analysis. Proteins originating from contaminating red blood cells or affected by clotting can be identified following the methodology described in <a href="#bib8">Geyer et al. (2019)</a> and consequently excluded from consideration for further biological interpretation. Similarly, in large-scale studies compromised samples could be identified and excluded from downstream data analysis.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Plasma samples, as noted during examination in <b>step 2</b>, are lipemic and display high turbidity, <i>i.e.</i>, appear milky and opaque due to the presence of high concentrations of triglycerides and lipoprotein particles.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Lipemia is described as visible turbidity in serum or plasma samples caused by high concentrations of triglycerides and other lipids, most commonly due to patient non-compliance with fasting prior to blood collection and health conditions, such as alcohol abuse and diabetes (<a href="#bib17">Nikolac 2014</a>; <a href="#bib3">Castro-Castro et al., 2018</a>). High concentrations of lipids could interfere with analytical and clinical assays, and in case of proteomic analysis, can impede SPE and separation via liquid chromatography, causing peak broadening, contamination in MS, and column clogging. Protein extraction with 90% methanol (<b>step 5</b>) should effectively remove most lipids; however, if extremely high amounts of lipids are present, employing methyl-tert-butyl ether (MTBE) for more efficient lipid extraction is recommended. Lipid extraction with MTBE demonstrates excellent recovery of all major lipid classes, including triglycerides, from a wide variety of sample types (<a href="#bib13">Matyash et al., 2008</a>). A recent study by <a href="#bib10">He et al. (2021)</a> also assures compatibility of MTBE lipid extraction with the proteomics sample preparation, akin to the one described in this protocol.</p>
<p>The <b>step 5</b> of the procedure should be modified as follows:</p>
<ol start="6">
<li>Precipitate proteins by addition of the methyl-tert-butyl ether (MTBE)/ methanol mixture. After this step the plasma samples are no longer considered infectious and could be handled outside the Biosafety cabinet.
<ol type="a">
<li>Add a 220 μL volume of MTBE and 65 μL volume of methanol to each tube and vortex for 5 s.</li>
<li>Centrifuge down for 6 min at 13,000 × <i>g</i> at ambient temperature.</li>
</ol>
</li>
</ol>
<div class="caution">
<span class="caution-title">Caution:</span> MTBE is highly flammable and considered acutely toxic upon respiratory tract, skin, and eye exposure. It is also a suspected cancer agent and may cause drowsiness and dizziness. Appropriate PPE (<i>e.g.</i>, laboratory coat, goggles, and gloves) should be worn when working with this solvent. A chemical fume hood should be used when working with large volumes of this solvent.</div>
<h3 id="sec6.5">Problem 3</h3>
<p>A large number of identified peptide sequences are semi-tryptic, <i>i.e.</i>, at one or both ends contain amino acids that do not correspond to the expected products of tryptic digest of the original proteins.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Such sequences containing semi-tryptic (partially tryptic) cleavages typically result from non-specific activity of native plasma proteases. Native protease activity could be reduced by prolonged and more aggressive protein denaturation (<b>step 4</b>). As mentioned above, continuous heating for more than 10 min is not recommended. However, the researcher can heat for 10 min multiple times, while allowing the samples to cool down to ambient temperature between the heating cycles (<i>e.g.</i>, 10 min heating, followed by 10 min cooling, followed by another round of 10 min heating).</p>
<p>Although rapid and efficient denaturing of samples is the preferred route and should be generally sufficient (<a href="#bib4">Clifton et al., 2011</a>), addition of protease inhibitors, such as cOmplete™ Mini EDTA-free Protease Inhibitor Cocktail from Roche, could also be considered. Note, besides inhibiting native protease, they also interfere with LysC and trypsin activity, and the amount of these enzymes may have to be increased, for example to final enzyme mass: protein mass ratio of 1:25.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>A greater than expected (&gt;15%) number of identified peptide sequences contain missed cleavages, <i>i.e.</i>, internal lysine and arginine residues, and/ or the percentage of missed cleaved sequences varies between samples by more than 10%.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Tryptic peptides with missed cleavages are sequences that contain lysine and arginine residues at positions other than C-terminus. As trypsin normally displays sequence-specific activity, some missed cleavages are always present, particularly in the vicinity of negatively charged amino acids or in lysine-rich stretches (<a href="#bib20">Vandermarliere et al., 2013</a>). In anticipation of this, when searching proteomics data acquired on samples prepared using this protocol, up to 2–3 missed cleavages should be allowed. In general, the percent of missed cleavages between samples of the same type prepared using the same protocol should be consistent.</p>
<p>An unusually high number of missed cleavages suggest low activity of sequencing enzymes – trypsin and/ or LysC. To prevent this issue, the researchers should only use freshly thawed, high-quality sequencing grade enzymes and avoid vortexing and centrifuging them at high speeds (&gt;2,000 × <i>g</i>).</p>
<p>Additionally, to increase digestion efficiency the following could be tried in <b>steps 8–10</b>: (i) the enzyme to protein ratio could be increase from 1:50 (enzyme mass: protein mass) to 1:25 or higher; (ii) the digestion buffer can be further diluted to the final concentration of urea below 1 M (<i>e.g</i>., 0.5 or 0.8 M); (iii) pH of the digestion buffer (LBB) should be checked and adjusted if needed to 7–8 with a small volume of 1 M Tris (pH = 8); and (iv) digestion time with each enzyme could be increased from 1 h to 2–4 h.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>The amount of peptides yielded from 5 μL of plasma (∼331.5 μg for healthy plasma donors) is not sufficient for desired downstream analysis and sample manipulation.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Some researchers may need to generate a large amount of plasma peptides to be used as a long-term technical control. Others may want to pre-fractionate, chemically label, or enrich plasma peptides for PTMs. In these cases, the amount of material yielded from 5 μL of plasma may not be sufficient. Note, up to 500 μg of plasma peptide could be prepared following this protocol without the need for any modifications.</p>
<p>Alternatively, the protocol permits straightforward scaling of the yields with proportionate increases of plasma and reagent volumes. For example, processing of 50 μL aliquot of plasma requires increasing all reagent volumes by ten-fold. Lengths of treatments, such as thermolysis (<b>step 4</b>) and digestion (<b>steps 8–10</b>), should be kept constant.</p>
<p>Additionally, a change must be made to the selection of SPE cartridges (<b>steps 11–17</b>). Strata™-X 33 μm Polymeric Reverse Phase cartridges <i>10 mg/ 1 mL</i>, the product recommended for use by this protocol, possesses limited loading capacity of up to 500 μg (20:1 ratio of stationary phase mass to bound peptide mass). Desalting a mass of peptides in excess of the cartridge’s loading capacity causes peptide losses and reduction in the number of peptide and protein identifications. If greater than 500 μg of peptides are expected to be yielded, similar products with larger packing beds, <i>e.g.</i>, Strata™-X 33 μm Polymeric Reverse Phase cartridges 3<i>0 mg/ 1 mL</i> or 6<i>0 mg/ 3 mL,</i> should be used instead. The volumes of equilibration, washing, and elution solvents should be scaled according to the increase in the mass of the packing bed. For example, if 30 mg/ 1 mL cartridges are used, they should be equilibrated and washed with 3 mL of 100% ACN and 0.1% TFA in water, respectively, and eluted with 900 μL volume of 80% ACN in 0.1% TFA.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Protein determination assay (Optional after <b>step 7</b>) was performed, and protein concentration in the patient plasma was found to be considerably outside of the expected range for healthy individuals with anticipated average peptide yields of ∼331.5 μg from the 5 μL aliquot of plasma.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>This protocol can be applied to samples with lower protein concentrations without any adjustments. As explained in <a href="#sec6.9">problem 5</a> in <a href="#troubleshooting">troubleshooting</a>, the protocol permits processing of up to 500 μg of plasma peptides without any changes to the procedure. The histogram in <a href="#fig3">Figure 3</a>B illustrates that less than 1% of the examined 128 human plasma samples, originated from patients of various health statuses (<a href="#bib18">Overmyer et al., 2020</a>) and contained amounts of proteins that exceeded that. Processing such concentrated samples necessitates adjustments to the procedure, as described in <a href="#sec6.9">problem 5</a> in <a href="#troubleshooting">troubleshooting</a>.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Anderson, N.L. and Anderson, N.G. (2002). The human plasma proteome: history, character, and diagnostic prospects. Mol. Cell. Proteomics <i>1</i>, 845-867. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref1">View at publisher</a></p>
<p id="bib2">Bladergroen, M.R. and van der Burgt, Y.E.M. (2015). Solid-phase extraction strategies to surmount body fluid sample complexity in high-throughput mass spectrometry-based proteomics. J. Anal. Methods Chem. <i>2015</i>, 250131. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref2">View at publisher</a></p>
<p id="bib3">Castro-Castro, M.J., Candás-Estébanez, B., Esteban-Salán, M., Calmarza, P., Arrobas-Velilla, T., Romero-Román, C., Pocoví-Mieras, M., and Aguilar-Doreste, J.Á. (2018). Removing lipemia in serum/plasma samples: a multicenter study. Ann. Lab. Med. <i>38</i>, 518-523. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref3">View at publisher</a></p>
<p id="bib4">Clifton, J., Huang, F., Rucevic, M., Cao, L., Hixson, D., and Josic, D. (2011). Protease inhibitors as possible pitfalls in proteomic analyses of complex biological samples. J. Proteomics <i>74</i>, 935-941. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref4">View at publisher</a></p>
<p id="bib5">Cox, J., Hein, M.Y., Luber, C.A., Paron, I., Nagaraj, N., and Mann, M. (2014). Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. Mol. Cell. Proteomics <i>13</i>, 2513-2526. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref5">View at publisher</a></p>
<p id="bib6">Gallien, S., Kim, S.Y., and Domon, B. (2015). Large-scale targeted proteomics using internal standard triggered-parallel reaction monitoring (IS-PRM). Mol. Cell. Proteomics <i>14</i>, 1630-1644. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref6">View at publisher</a></p>
<p id="bib7">Geyer, P.E., Kulak, N.A., Pichler, G., Holdt, L.M., Teupser, D., and Mann, M. (2016). Plasma proteome profiling to assess human health and disease. Cell Syst. <i>2</i>, 185-195. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref7">View at publisher</a></p>
<p id="bib8">Geyer, P.E., Voytik, E., Treit, P.V., Doll, S., Kleinhempel, A., Niu, L., Müller, J.B., Buchholtz, M., Bader, J.M., Teupser, D., et al. (2019). Plasma Proteome Profiling to detect and avoid sample-related biases in biomarker studies. EMBO Mol. Med. <i>11</i>, 1-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref8">View at publisher</a></p>
<p id="bib9">Halvey, P., Farutin, V., Koppes, L., Gunay, N.S., Pappas, D.A., Manning, A.M., and Capila, I. (2021). Variable blood processing procedures contribute to plasma proteomic variability. Clin. Proteomics <i>18</i>, 5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref9">View at publisher</a></p>
<p id="bib10">He, Y., Rashan, E.H., Linke, V., Shishkova, E., Hebert, A.S., Jochem, A., Westphall, M.S., Pagliarini, D.J., Overmyer, K.A., and Coon, J.J. (2021). Multi-omic single-shot technology for integrated proteome and lipidome analysis. Anal. Chem. <i>93</i>, 4217-4222. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref10">View at publisher</a></p>
<p id="bib21">Ilies, M., Iuga, C.A., Loghin, F., Dhople, V.M., Thiele, T., Völker, U., and Hammer, E. (2017). Impact of blood sample collection methods on blood protein profiling studies. Clin Chim Acta <i>471</i>, 128-134. <a class="external-link" href="https://doi.org/10.1016/j.cca.2017.05.030">https://doi.org/10.1016/j.cca.2017.05.030</a></p>
<p id="bib12">Leeman, M., Choi, J., Hansson, S., Storm, M.U., and Nilsson, L. (2018). Proteins and antibodies in serum, plasma, and whole blood—size characterization using asymmetrical flow field-flow fractionation (AF4). Anal. Bioanal. Chem. <i>410</i>, 4867-4873. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref12">View at publisher</a></p>
<p id="bib13">Matyash, V., Liebisch, G., Kurzchalia, T.V., Shevchenko, A., and Schwudke, D. (2008). Lipid extraction by methyl-terf-butyl ether for high-throughput lipidomics. J. Lipid Res. <i>49</i>, 1137-1146. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref13">View at publisher</a></p>
<p id="bib14">Meier, F., Geyer, P.E., Virreira Winter, S., Cox, J., and Mann, M. (2018). BoxCar acquisition method enables single-shot proteomics at a depth of 10,000 proteins in 100 minutes. Nat. Methods <i>15</i>, 440-448. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref14">View at publisher</a></p>
<p id="bib15">Messner, C.B., Demichev, V., Wendisch, D., Michalick, L., White, M., Freiwald, A., Textoris-Taube, K., Vernardis, S.I., Egger, A.S., Kreidl, M., et al. (2020). Ultra-high-Throughput clinical proteomics reveals classifiers of COVID-19 infection. Cell Syst. <i>11</i>, 11-24.e4. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref15">View at publisher</a></p>
<p id="bib16">Nakayasu, E.S., Gritsenko, M., Piehowski, P.D., Gao, Y., Orton, D.J., Schepmoes, A.A., Fillmore, T.L., Frohnert, B.I., Rewers, M., Krischer, J.P., et al. (2021). Tutorial: best practices and considerations for mass-spectrometry-based protein biomarker discovery and validation. Nat. Protoc. . <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref16">View at publisher</a></p>
<p id="bib17">Nikolac, N. (2014). Lipemia: causes, interference mechanisms, detection and management. Biochem. Med. <i>24</i>, 57-67. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref17">View at publisher</a></p>
<p id="bib18">Overmyer, K.A., Shishkova, E., Miller, I.J., Balnis, J., Bernstein, M.N., Peters-Clarke, T.M., Meyer, J.G., Quan, Q., Muehlbauer, L.K., Trujillo, E.A., et al. (2020). Large-scale multi-omic analysis of COVID-19 severity. Cell Syst. <i>12</i>, 23-40.e7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref18">View at publisher</a></p>
<p id="bib19">Peterson, A.C., Russell, J.D., Bailey, D.J., Westphall, M.S., and Coon, J.J. (2012). Parallel reaction monitoring for high resolution and high mass accuracy quantitative, targeted proteomics. Mol. Cell. Proteomics <i>11</i>, 1475-1488. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref19">View at publisher</a></p>
<p id="bib20">Vandermarliere, E., Mueller, M., and Martens, L. (2013). Getting intimate with trypsin, the leading protease in proteomics. Mass Spectrom. Rev. <i>32</i>, 453-465. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00562-1/sref20">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by Prof. Joshua J Coon at <a href="mailto:jcoon@chem.wisc.edu">jcoon@chem.wisc.edu</a>.</p>
<h4>Materials availability</h4>
<p>Not applicable.</p>
<h4>Data and code availability</h4>
<p>The protocol includes all relevant datasets. For other associated data and examples of protocol’s implementation, please see <a href="#bib18">Overmyer et al. (2020)</a>.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1062-Mmc1.xlsx">Table S1. Experimental peptide yields obtained from 5 μL volumes of control pooled and 128 unique patient blood plasma samples, analyzed in Overmyer et al. (2020), related to step 20 and expected outcomes</a></p>
<h3>Acknowledgments</h3>
<p>We thank Annie Jen for help with figure preparation. This work was supported by funding to the National Center for Quantitative Biology of Complex Systems (5P41GM108538, J.J.C.).</p>
<h3>Author contributions</h3>
<p>E.S. developed the method, analyzed data, prepared figures, and composed the manuscript. J.J.C. obtained funding, supervised the work, and edited the manuscript.</p>
<h3>Declaration of interests</h3>
<p>E.S. declares no competing interest. J.J.C. is a consultant for Thermo Fisher Scientific.</p>
</section>
</article>