<article>
  <section>
    <h1 id="header">
      UnCovid: A versatile, low-cost, and open-source protocol for SARS-CoV-2
      RNA detection
    </h1>
    <p><time datetime="2021-11-05">Published: November 5, 2021</time></p>
    <p>
      Roberto Alcántara,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup>
      Katherin Peñaranda,<sup><a href="#aff1">1</a></sup> Gabriel
      Mendoza-Rojas,<sup><a href="#aff1">1</a></sup> Jose A. Nakamoto,<sup><a href="#aff1">1</a></sup>
      Eva Dueñas,<sup><a href="#aff1">1</a></sup> Daniela Alvarez,<sup><a href="#aff1">1</a></sup>
      Vanessa Adaui,<sup><a href="#aff1">1</a></sup> and Pohl Milón<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup>
    </p>
    <p id="aff1">
      <sup>1</sup>Centre for Research and Innovation, Health Sciences Faculty,
      Universidad Peruana de Ciencias Aplicadas (UPC), Lima 15023, Peru
    </p>
    <p id="fn1"><sup>2</sup>Technical contact</p>
    <p id="fn2"><sup>3</sup>Lead contact</p>
    <p id="cor1">
      <sup>*</sup>Correspondence:
      <a href="mailto:pcmeralc@upc.edu.pe">pcmeralc@upc.edu.pe</a>
    </p>
    <p id="cor2">
      <sup>**</sup>Correspondence:
      <a href="mailto:pmilon@upc.edu.pe">pmilon@upc.edu.pe</a>
    </p>
    <p>
      <span class="open-access">Open Access</span> • DOI:
      <a href="https://doi.org/10.1016/j.xpro.2021.100878">10.1016/j.xpro.2021.100878</a>
    </p>
  </section>
  <section>
    <h2 id="summary">Summary</h2>
    <p>
      Here, we describe a detailed step-by-step protocol to detect SARS-CoV-2
      RNA using RT-PCR-mediated amplification and CRISPR/Cas-based
      visualization. The optimized assay uses basic molecular biology equipment
      such as conventional thermocyclers and transilluminators for qualitative
      detection. Alternatively, a fluorescence plate reader can be used for
      quantitative measurements. The protocol detects two regions of the
      SARS-CoV-2 genome in addition to the human RNaseP sample control. Aiming
      to reach remote regions, this work was developed to use the portable
      molecular workstation from BentoLab.
    </p>
    <p>
      For complete details on the use and execution of this protocol, please
      refer to
      <a href="https://doi.org/10.1016/j.crmeth.2021.100093" class="external-link">Alcántara et al., 2021</a>.
    </p>
    
    
    
    
  <div class="highlights">
<h3>Highlights</h3>
<ul>
      <li>
        Molecular detection of SARS-CoV-2 using RT-PCR coupled to CRISPR/Cas
        technology
      </li>
      <li>
        Compatibility for applying UnCovid as a Laboratory-developed test (LDT)
      </li>
      <li>Protocol validated with clinical samples</li>
      <li>
        Versatility for developing molecular detection of other RNA viruses
      </li>
    </ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure>
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-GA.jpg" alt="GraphicalAbstract.jpg">
    </figure>
</div></section>
  <section>
    <h2 id="before-you-begin">Before you begin</h2>
    <p>
      The UnCovid protocol couples nucleic acid amplification using RT-PCR with
      detection of the amplified DNA mediated by CRISPR/Cas12a (<a href="#bib7">Chen et al., 2018</a>). Cas12a is a DNA nuclease that uses an adaptor RNA (CRISPR RNA) to
      identify the target DNA to be cleaved (<a href="#bib19">Zetsche et al., 2015</a>). Upon cleavage of the target double strand DNA (dsDNA), Cas12a
      activates a promiscuous off target activity that can cut single stranded
      DNA (ssDNA) (<a href="#bib13">Jinek et al., 2012</a>). This off target
      activity is used to cleave a dually labeled ssDNA. i.e., a ssDNA labeled
      with a donor fluorophore and a silent acceptor (quencher) results in
      measurable increase of fluorescence upon Cas12a-mediated cleavage. The
      Föster Resonance Energy Transfer (FRET) between the dyes is released due
      to a distance increase between the dyes. The UnCovid protocol is intended
      to be used with locally produced enzymes (M-MLV reverse transcriptase, Taq
      DNA polymerase, and LbCas12a) or their commercial counterparts of your
      preference. Albeit the crRNA can be chemically synthetized and purchased,
      <i>in vitro</i> transcription is most cost efficient. This segment of the
      protocol describes the production, purification, quantification, and
      storage of crRNA for CRISPR/Cas reaction. For details about enzyme
      production refer to (<a href="#bib17">Mendoza-Rojas et al., 2021</a>).
    </p>
    <h3 id="sec1.1">General experimental considerations</h3>
    <ol>
      <li>
        This protocol is intended to be executed by experienced personnel. Users
        require to have a basic molecular biology training and experience, be
        accustomed with proper pipetting technique and trained for good clinical
        laboratory practice (GCLP).
      </li>
      <li>
        Good clinical laboratory practices (GCLP) are recommended for use of
        this protocol. A 90 min online training in GCLP is available at the
        following link:
        <a href="https://www.cdc.gov/labtraining/training-courses/good-lab-practices-molecular-genetics-testing.html">https://www.cdc.gov/labtraining/training-courses/good-lab-practices-molecular-genetics-testing.html</a>.
      </li>
      <li>
        Correct use of basic equipment is necessary to avoid cross
        contamination. Avoid using the same gloves across all areas, amplicon
        contamination must be strongly contained (<a href="#bib8">Davidi et al., 2021</a>).
      </li>
      <li>
        It is recommended to work with a unidirectional flow of samples
        considering “clean” (preparing reactions, sample manipulation) and
        “dirty” lab areas (where the thermal cyclers are located, and for
        post-processing steps (electrophoresis)). A Basic Molecular Biology
        Moduls for Laboratory Practice is available at the following link:
        <a href="https://www.cdc.gov/labtraining/training-courses/basic-molecular-biology/laboratory-practice.html">https://www.cdc.gov/labtraining/training-courses/basic-molecular-biology/laboratory-practice.html</a>
      </li>
    </ol>
    <h3 id="sec1.2">crRNA purification</h3>
    <div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
    <ol start="5">
      <li>
        <i>In vitro</i> transcription reaction <i>In vitro</i> transcription
        (IVT) requires a double-stranded DNA (dsDNA) template containing the T7
        promoter sequence. Here, we used the following dsDNA templates to
        transcribe crRNAs, complemental to two SARS-CoV-2 genes and for the
        human RNaseP as sample quality control:
        <figure id="fx5">
          <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Fx5.jpg" alt="Fx5.jpg">
          <figcaption>
            <div class="figcaption-title">
              ∗The T7 promoter sequence is highlighted in red.
            </div>
          </figcaption>
        </figure>
        <div class="note">
          <span class="note-title">Note:</span> Use the
          <i>in vitro</i> transcription kit of your choice to prepare crRNA.
          Here, the TranscriptAid T7 High Yield Transcription Kit (Thermo
          Fisher) was used.
        </div>
        <div class="critical">
          <span class="critical-title">Critical:</span> Pay attention to the
          following general recommendations to avoid RNase contamination (<a href="#bib10">Green and Sambrook, 2019</a>): (1) Maintain a separate pipette set, pipette tips, and any other
          material required for RNA lab work. (2) Use gloves at all times.
          Change them frequently. (3) Preferably use RNase-free water (i.e.,
          nuclease-free water aliquots) to prepare reactions or buffers. (4)
          Clean working surfaces, pipettes, and equipment with 20% bleach or
          alternative commercial solutions. To avoid accelerated deterioration
          of any plastics and metals, wipe down with 70% ethanol after using 20%
          bleach.
        </div>
        <ol type="a">
          <li>
            Dissolve the dsDNA templates using nuclease-free water to 100 μM
            concentration.
          </li>
          <li>Thaw all reagents and maintain the enzyme mix on ice.</li>
          <li>
            Prepare the 25 mM NTPs (ATP, GTP, CTP, and UTP) mix by adding an
            equal volume of each nucleotide (100 mM stock) in a single tube.
          </li>
          <li>
            Prepare the master mix for 40 μL reactions as follows:
            <table id="undtbl2">
              <thead>
                <tr>
                  <th>Reagent</th>
                  <th>Initial concentration</th>
                  <th>Final concentration</th>
                  <th>Volume (μL) per reaction</th>
                  <th>Volume (μL) for 3 reactions</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>5<b>×</b> Reaction buffer</td>
                  <td>5<b>×</b>
</td>
                  <td>1<b>×</b>
</td>
                  <td>8</td>
                  <td>24</td>
                </tr>
                <tr>
                  <td>NTP mix</td>
                  <td>25 mM</td>
                  <td>10 mM</td>
                  <td>16</td>
                  <td>48</td>
                </tr>
                <tr>
                  <td>Enzyme mix</td>
                  <td>-</td>
                  <td>-</td>
                  <td>4</td>
                  <td>12</td>
                </tr>
                <tr>
                  <td>
                    Nuclease-free water <sup><a href="#dtbl2fna">a</a></sup>
                  </td>
                  <td>-</td>
                  <td>-</td>
                  <td>11</td>
                  <td>33</td>
                </tr>
                <tr>
                  <td>Final volume</td>
                  <td></td>
                  <td></td>
                  <td>39</td>
                  <td>117</td>
                </tr>
              </tbody>
            </table>
            <p class="table-footnote" id="dtbl2fna">
              <sup>a</sup>Water volume can be adjusted depending on template
              volume
            </p>
          </li>
          <li>Dispense 39 μL of master mix in a 1.5 mL tube.</li>
          <li>
            Add 4 μg of the respective dsDNA template to the reaction tube.
            Here, one microliter of a 100 μM dsDNA stock (100 pmoles) was used.
          </li>
          <li>Incubate the reaction at 37°C for 3 h.</li>
        </ol>
      </li>
    </ol>
    <ol start="6">
      <li>
        crRNA purification
        <div class="note">
          <span class="note-title">Note:</span> Use the RNA purification kit of
          your choice to purify the transcribed crRNA. Here, the Direct-zol RNA
          Miniprep purification Kit from Zymo Research was used.
        </div>
        <ol type="a">
          <li>
            Add 3 volumes (i.e., 120 μL for a 40 μL reaction) of TRI-Reagent®
            (Zymo Research) or other similar product (TRIzol®, RNAzol®,
            TriPureTM, or TriSureTM) to each volume of the
            <i>in vitro</i> transcription reaction. Mix thoroughly and spin
            briefly.
          </li>
          <li>
            Add 1 volume (i.e., 160 μL) of 95%–100% cold ethanol. Mix thoroughly
            and spin briefly.
          </li>
          <li>
            Transfer the mixture to the spin column and centrifuge at 10000
            <b>×</b> <i>g</i> for 30 s. Discard the flow-through (FT).
          </li>
          <li>
            Add 400 μL of RNA wash buffer and centrifuge at 10000 <b>×</b>
            <i>g</i> for 30 s. Discard the FT.
            <div class="critical">
              <span class="critical-title">Critical:</span> DNase I treatment is
              recommended for DNA template removal. Deoxyribonuclease I (DNase
              I) cuts both single-stranded and double-stranded DNA. Recombinant
              DNase I is typically used for selectively degrading DNA in the
              presence of RNA. One application of the DNase I treatment is
              removal of DNA template after <i>in vitro</i> transcription prior
              to downstream applications. Here, the DNase I and digestion buffer
              are provided with the Direct-zol RNA Miniprep kit (Zymo Research,
              Cat # E1010). The lyophilized enzyme can be reconstituted with
              275 μL of MilliQ water. DNase I digestion step is performed on the
              spin column. Prepare the working stock of DNase I by mixing 75 μL
              of DNA digestion buffer (provided in the kit) with 5 μL of DNase I
              (6 U/μL stock concentration, 0.38 U/μL final concentration) (Zymo
              Research, Cat # E1010). Alternatively, use DNase I in 1<b>×</b>
              DNase I buffer (10 mM Tris-HCl, 2.5 mM MgCl<sub>2</sub>, 0.5 mM
              CaCl<sub>2</sub>, pH 7.6 25°C).
            </div>
          </li>
          <li>
            Add 80 μL of DNase I working stock to the column and incubate at
            25°C for 15 min.
          </li>
          <li>
            Add 400 μL of RNA pre-wash buffer and centrifuge at 10000 <b>×</b>
            <i>g</i> for 30 s. Discard the FT.
          </li>
          <li>
            Add 700 μL of RNA wash buffer and centrifuge at 10000 <b>×</b>
            <i>g</i> for 30 s. Discard the FT.
          </li>
          <li>Transfer the column to a clean collection 1.5 mL tube</li>
          <li>
            Add 80 μL of nuclease-free water. Incubate for 2–5 s, and centrifuge
            at 10000 <b>×</b> <i>g</i> for one minute. Collect the tube and
            place it on ice.
          </li>
          <li>
            Quantify the RNA by measuring the absorbance at 260 nm (A260) in a
            spectrophotometer. Here, the NanoDrop One microvolume UV-Vis
            spectrophotometer was used (Thermo Fisher Scientific). An A260 nm
            absorbance of 1 is equivalent to 40 μg/mL single-stranded RNA.
          </li>
          <li>
            Calculate the molar concentration using the corresponding molecular
            weight (ORF1ab = 13.5 kDa, N = 13 kDa, RNaseP = 13 kDa).
          </li>
          <li>
            Dilute your RNA stocks to 10–20 μM using pre-cooled nuclease-free
            water, mix briefly and spin down.
          </li>
          <li>
            Dispense 20 μL aliquots in 0.5–1.5 mL tubes and store at −80°C.
          </li>
        </ol>
      </li>
    </ol>
  </section>
  <section>
    <h2 id="key-resources-table">Key resources table</h2>
    <table id="krt">
      <thead>
        <tr>
          <th>REAGENT or RESOURCE</th>
          <th>SOURCE</th>
          <th>IDENTIFIER</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td colspan="3">Biological samples</td>
        </tr>
        <tr>
          <td>Nasopharyngeal swabs</td>
          <td>Instituto de Investigación Nutricional, Lima, Peru</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            Quantitative PCR (qPCR) control RNA from heat-inactivated
            SARS-CoV-2, Isolate USA-WA1/2020
          </td>
          <td>Centers for Disease Control and Prevention</td>
          <td>BEI Resources NR-52347</td>
        </tr>
        <tr>
          <td colspan="3">Chemicals, peptides, and recombinant proteins</td>
        </tr>
        <tr>
          <td>Potassium chloride</td>
          <td>Supelco</td>
          <td>Cat # 104936</td>
        </tr>
        <tr>
          <td>Magnesium chloride hexahydrate</td>
          <td>Supelco</td>
          <td>Cat # 105833</td>
        </tr>
        <tr>
          <td>Triton X-100</td>
          <td>Sigma-Aldrich</td>
          <td>Cat # 10789704001</td>
        </tr>
        <tr>
          <td>Trehalose</td>
          <td>Sigma-Aldrich</td>
          <td>Cat # T9449-25G</td>
        </tr>
        <tr>
          <td>Sodium chloride</td>
          <td>Supelco</td>
          <td>Cat # 106404</td>
        </tr>
        <tr>
          <td>Glycerol 85%</td>
          <td>Supelco</td>
          <td>Cat # 104094</td>
        </tr>
        <tr>
          <td>EDTA</td>
          <td>Merck Millipore</td>
          <td>Cat # 324503</td>
        </tr>
        <tr>
          <td>Tris, Hydrochloride, Molecular Biology Grade</td>
          <td>Merck Millipore</td>
          <td>Cat # US1648317</td>
        </tr>
        <tr>
          <td>Tris base ULTROL grade</td>
          <td>Calbiochem</td>
          <td>Cat # 643811</td>
        </tr>
        <tr>
          <td>Dithiothreitol (DTT)</td>
          <td>Thermo Fisher</td>
          <td>Cat # R0682</td>
        </tr>
        <tr>
          <td>dNTP Mix (10 mM each)</td>
          <td>Thermo Fisher</td>
          <td>Cat # R0194</td>
        </tr>
        <tr>
          <td>SafeGreen</td>
          <td>abm</td>
          <td>Cat # G108-G</td>
        </tr>
        <tr>
          <td>TriTrack DNA loading dye 6<b>×</b>
</td>
          <td>Thermo Fisher</td>
          <td>Cat # R1161</td>
        </tr>
        <tr>
          <td>Agarose powder</td>
          <td>Cleaver Scientific</td>
          <td>Cat # CSL-AG500</td>
        </tr>
        <tr>
          <td>OmniPur Water DEPC treated, sterile, nuclease-free</td>
          <td>Sigma-Aldrich</td>
          <td>Cat # 9610-1L</td>
        </tr>
        <tr>
          <td>DNaseI</td>
          <td>PanReac Applichem</td>
          <td>Cat # A3778</td>
        </tr>
        <tr>
          <td>GeneRuler 100 bp DNA Ladder</td>
          <td>Thermo Scientific</td>
          <td>Cat # SM0242</td>
        </tr>
        <tr>
          <td>Bovine Serum Albumin, lyophilized powder</td>
          <td>Sigma-Aldrich</td>
          <td>Cat # A9418-100G</td>
        </tr>
        <tr>
          <td colspan="3">Critical commercial assays</td>
        </tr>
        <tr>
          <td>TranscriptAid T7 High Yield Transcription Kit</td>
          <td>Thermo Fisher</td>
          <td>Cat # K0441</td>
        </tr>
        <tr>
          <td>Direct-zol RNA Miniprep</td>
          <td>Zymo Research</td>
          <td>Cat # R2051</td>
        </tr>
        <tr>
          <td>2<b>×</b> One-Step RT-PCR Master Mix</td>
          <td>Norgen BioTek</td>
          <td>Cat # 28113</td>
        </tr>
        <tr>
          <td>QIAamp viral RNA Mini Kit</td>
          <td>Qiagen</td>
          <td>Cat # 52906</td>
        </tr>
        <tr>
          <td colspan="3">Deposited data</td>
        </tr>
        <tr>
          <td>
            <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Mmc1.xlsx">Data S1</a>
            and
            <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Mmc2.xlsx">S2</a>
          </td>
          <td>This paper</td>
          <td>
            <a href="https://doi.org/10.17632/8m8z37v8xz.1">https://doi.org/10.17632/8m8z37v8xz.1</a>
          </td>
        </tr>
        <tr>
          <td colspan="3">Oligonucleotides</td>
        </tr>
        <tr>
          <td>
            dsDNA oligo: crRNA sequence N target SARS-CoV-2
            TAATACGACTCACTATAGGTAATTTCTACTAAGTGTAGATCCCCCAGCGCTTCAGCGTTC
          </td>
          <td><a href="#bib4">Broughton et al.,2020</a></td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            dsDNA oligo: crRNA sequence ORF1ab target SARS-CoV-2
            TAATACGACTCACTATAGGTAATTTCTACTAAGTGTAGATTTAGAGACGGTTGGGAAATTG
          </td>
          <td>This paper and <a href="#bib1">Alcántara et al., 2021</a>
</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            dsDNA oligo: crRNA sequence RNaseP target human sample control
            TAATACGACTCACTATAGGTAATTTCTACTAAGTGTAGATAATTACTTGGGTGTGACCCT
          </td>
          <td><a href="#bib4">Broughton et al.,2020</a></td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            ssDNA oligo: primer forward N target SARS-CoV-2
            TACAAACATTGGCCGCAAATTGC
          </td>
          <td>This paper and <a href="#bib1">Alcántara et al., 2021</a>
</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            ssDNA oligo: primer reverse N target SARS-CoV-2 CCAATGCGCGACATTCCG
          </td>
          <td>This paper and <a href="#bib1">Alcántara et al., 2021</a>
</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            ssDNA oligo: primer forward ORF1ab target SARS-CoV-2
            GTTGTTCAGTTGACTTCGC
          </td>
          <td>This paper and <a href="#bib1">Alcántara et al., 2021</a>
</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            ssDNA oligo: primer reverse ORF1ab target SARS-CoV-2
            GACAATTTCACAAGCACAGG
          </td>
          <td>This paper and <a href="#bib1">Alcántara et al., 2021</a>
</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            ssDNA oligo: primer forward RNaseP target human sample control
            ACTCAGCCATCCACATCC
          </td>
          <td>This paper and <a href="#bib1">Alcántara et al., 2021</a>
</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>
            ssDNA oligo: primer reverse RNaseP target human sample control
            CACCCTCAATGCAGAGTC
          </td>
          <td><a href="#bib4">Broughton et al.,2020</a></td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>reporter_FQ: /56-FAM/TTATT/3IABkFQ/</td>
          <td><a href="#bib7">Chen et al. (2018)</a></td>
          <td>N/A</td>
        </tr>
        <tr>
          <td colspan="3">Software and algorithms</td>
        </tr>
        <tr>
          <td>ImageJ</td>
          <td><a href="#bib21">Schneider et al., 2012</a></td>
          <td>
            <a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a>
          </td>
        </tr>
        <tr>
          <td>GraphPad Prism version 9</td>
          <td>GraphPad Software, La Jolla, CA</td>
          <td><a href="http://www.graphpad.com">www.graphpad.com</a></td>
        </tr>
        <tr>
          <td>Stata v14.0</td>
          <td>StataCorp, College Station, TX</td>
          <td><a href="https://www.stata.com/">https://www.stata.com/</a></td>
        </tr>
        <tr>
          <td colspan="3">Other</td>
        </tr>
        <tr>
          <td>
            96-well flat clear bottom black polystyrene TC-treated microplates
          </td>
          <td>Corning®</td>
          <td>Cat # 3603</td>
        </tr>
        <tr>
          <td>NanoDrop One microvolume UV-Vis spectrophotometer</td>
          <td>Thermo Fisher</td>
          <td>Cat # ND-ONE-W</td>
        </tr>
        <tr>
          <td>safeVIEW: LED/Blue Light Transilluminator</td>
          <td>Cleaver Scientific</td>
          <td>Cat # SAFEVIEW</td>
        </tr>
        <tr>
          <td>Cytation<sup>TM</sup> 5 Cell Imaging Multi-Mode Reader</td>
          <td>BioTek Instruments</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>Synergy<sup>TM</sup> H1 Hybrid Multi-Mode Reader</td>
          <td>BioTek Instruments</td>
          <td>N/A</td>
        </tr>
      </tbody>
    </table>
  </section>
  <section>
    <h2 id="materials-and-equipment-(optional)">
      Materials and equipment (optional)
    </h2>
    <ul>
      <li>Buffers:</li>
    </ul>
    <table id="undtbl3">
      <caption>
        10<b>×</b>
        CrB Buffer
      </caption>
      <thead>
        <tr>
          <th>Stocks</th>
          <th>Final concentration</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Tris-HCl (1 M) (pH 7.9, 25°C)</td>
          <td>100 mM</td>
          <td>2.5 mL</td>
        </tr>
        <tr>
          <td>NaCl (1 M)</td>
          <td>100 mM</td>
          <td>2.5 mL</td>
        </tr>
        <tr>
          <td>Nuclease-free water</td>
          <td>n/a</td>
          <td>20 mL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>n/a</b></td>
          <td><b>25 mL</b></td>
        </tr>
      </tbody>
    </table>
    <p class="table-legend">∗Store at −20°C</p>
    <table id="undtbl4">
      <caption>
        1<b>×</b>
        CrB1 Buffer
      </caption>
      <thead>
        <tr>
          <th>Stocks</th>
          <th>Final concentration</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>10<b>×</b> CrB Buffer</td>
          <td>1<b>×</b>
</td>
          <td>3.9 mL</td>
        </tr>
        <tr>
          <td>Nuclease-free water</td>
          <td>n/a</td>
          <td>100 μL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>n/a</b></td>
          <td><b>4 mL</b></td>
        </tr>
      </tbody>
    </table>
    <p class="table-legend">∗Always use fresh working stocks.</p>
    <table id="undtbl5">
      <caption>
        1<b>×</b>
        CrB2 Buffer
      </caption>
      <thead>
        <tr>
          <th>Stocks</th>
          <th>Final concentration</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>1<b>×</b> CrB1 Buffer</td>
          <td>1<b>×</b>
</td>
          <td>3340 L</td>
        </tr>
        <tr>
          <td>MgCl<sub>2</sub> (1 M)</td>
          <td>17.65 mM</td>
          <td>60 μL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>n/a</b></td>
          <td><b>3.4 mL</b></td>
        </tr>
      </tbody>
    </table>
    <p class="table-legend">∗Always use fresh stocks.</p>
    <div class="note">
      <span class="note-title">Note:</span> If using locally produced enzymes,
      prepare the following buffers
    </div>
    <table id="undtbl6">
      <caption>
        5<b>×</b>
        RPB Buffer
      </caption>
      <thead>
        <tr>
          <th>Stocks</th>
          <th>Final concentration</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Tris-HCl (1 M) (pH 8.4, 25°C)</td>
          <td>250 mM</td>
          <td>6.25 mL</td>
        </tr>
        <tr>
          <td>KCl (1 M)</td>
          <td>375 mM</td>
          <td>9.375 mL</td>
        </tr>
        <tr>
          <td>MgCl<sub>2</sub> (1 M)</td>
          <td>15 mM</td>
          <td>375 μL</td>
        </tr>
        <tr>
          <td>Trehalose dihydrate</td>
          <td>50%</td>
          <td>12.5 g</td>
        </tr>
        <tr>
          <td>DTT (1 M)</td>
          <td>50 mM</td>
          <td>1.25 mL</td>
        </tr>
        <tr>
          <td>EDTA (0.5 M)</td>
          <td>0.5 mM</td>
          <td>25 μL</td>
        </tr>
        <tr>
          <td>Nuclease-free water</td>
          <td>n/a</td>
          <td>to 25 mL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>n/a</b></td>
          <td><b>25 mL</b></td>
        </tr>
      </tbody>
    </table>
    <p class="table-legend">∗Storage at −20°C.</p>
    <table id="undtbl7">
      <caption>
        4<b>×</b>
        RPB Buffer
      </caption>
      <thead>
        <tr>
          <th>Stocks</th>
          <th>Final concentration</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>5<b>×</b> RPB Buffer</td>
          <td>4<b>×</b>
</td>
          <td>200 μL</td>
        </tr>
        <tr>
          <td>10 mM dNTPs mixture</td>
          <td>1.6 mM</td>
          <td>40 μL</td>
        </tr>
        <tr>
          <td>Nuclease-free water</td>
          <td>n/a</td>
          <td>10 μL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>n/a</b></td>
          <td><b>250 μL</b></td>
        </tr>
      </tbody>
    </table>
    <p class="table-legend">
      ∗Storage at – 20°C. Each 250 μL aliquot yields 50 RT-PCR reactions.
    </p>
    <table id="undtbl8">
      <caption>
        MB6 buffer
      </caption>
      <thead>
        <tr>
          <th>Stocks</th>
          <th>Final concentration</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Tris-HCl (1 M) (pH 8.0, 25°C)</td>
          <td>50 mM</td>
          <td>1.25 mL</td>
        </tr>
        <tr>
          <td>NaCl (1 M)</td>
          <td>100 mM</td>
          <td>2.5 mL</td>
        </tr>
        <tr>
          <td>DTT (1 M)</td>
          <td>5 mM</td>
          <td>125 μL</td>
        </tr>
        <tr>
          <td>EDTA (0.5 M)</td>
          <td>0.1 mM</td>
          <td>5 μL</td>
        </tr>
        <tr>
          <td>Triton X-100</td>
          <td>0.1%</td>
          <td>25 μL</td>
        </tr>
        <tr>
          <td>Glycerol</td>
          <td>50%</td>
          <td>12.5 mL</td>
        </tr>
        <tr>
          <td>Nuclease-free water</td>
          <td>n/a</td>
          <td>8.5 mL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>n/a</b></td>
          <td><b>25 mL</b></td>
        </tr>
      </tbody>
    </table>
    <p class="table-legend">∗Storage at −20°C</p>
    <ul>
      <li>Oligonucleotides: The following primers were used at 5 μM stocks:</li>
    </ul>
    <div class="note">
      <span class="note-title">Note:</span> Dissolve the primers using
      nuclease-free water to 100 μM concentration. Then dilute to 5 μM in an
      appropriate volume.
    </div>
    <table id="undtbl9">
      <thead>
        <tr>
          <th>Gene</th>
          <th>Orientation</th>
          <th>Name</th>
          <th>Sequence (5’ – 3′)</th>
          <th>Amplicon Size (bp)</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td rowspan="2">ORF1ab</td>
          <td>Forward</td>
          <td>FCb3</td>
          <td>GTTGTTCAGTTGACTTCGC</td>
          <td rowspan="2">168</td>
        </tr>
        <tr>
          <td>Reverse</td>
          <td>RCb2</td>
          <td>GACAATTTCACAAGCACAGG</td>
        </tr>
        <tr>
          <td rowspan="2">N</td>
          <td>Forward</td>
          <td>FN1</td>
          <td>TACAAACATTGGCCGCAAATTGC</td>
          <td rowspan="2">131</td>
        </tr>
        <tr>
          <td>Reverse</td>
          <td>RN2</td>
          <td>TCATCCAATTTGATGGCACC</td>
        </tr>
        <tr>
          <td rowspan="2">RNAseP</td>
          <td>Forward</td>
          <td>FR1</td>
          <td>ACTCAGCCATCCACATCC</td>
          <td rowspan="2">175</td>
        </tr>
        <tr>
          <td>Reverse</td>
          <td>RR2</td>
          <td>CACCCTCAATGCAGAGTC</td>
        </tr>
      </tbody>
    </table>
    <ul>
      <li>
        Additional reagents:
        <ul>
          <li>Working stock 2 mg/mL BSA, dissolved in Nuclease-free water</li>
          <li>Working stock 1 M MgCl<sub>2</sub>
</li>
          <li>Nuclease-free water</li>
          <li>
            56-FAM/TTATT/3IABkFQ dually-labeled probe should be dissolved at
            100 μM concentration in nuclease-free water. HPLC-purified probe is
            recommended and can be purchased from any oligo synthesis supplier.
          </li>
        </ul>
      </li>
    </ul>
    <div class="note">
      <span class="note-title">Note:</span> If using locally produced enzymes,
      then prepare a working 20<b>×</b> stock of enzymes as follows:
    </div>
    <ul>
      <li>
        Enzymes:
        <ul>
          <li>
            20<b>×</b> working stock M-MLV reverse transcriptase: Dilute your
            enzyme stock in MB6 buffer to a final concentration of 34 ng/μL.
          </li>
          <li>
            20<b>×</b> working stock Taq DNA polymerase: Dilute your enzyme
            stock to a final concentration of 32 ng/μL.
          </li>
        </ul>
      </li>
      <li>
        Equipment:
        <ul>
          <li>
            The protocol described here for SARS-CoV-2 detection can be used in
            any conventional thermal cycler. Here, we used the T-100 thermal
            cycler (Bio-Rad) (Cat. No. 1861096) and the portable BentoLab (<a href="https://www.bento.bio/">https://www.bento.bio/</a>).
          </li>
          <li>
            Fluorescence signal visualization used the microplate reader Synergy
            H1 (BioTek Instruments) for a quantitative readout, or a
            transilluminator safeVIEW: LED/Blue Light (Cleaver Scientific) for a
            qualitative readout.
          </li>
          <li>
            RT-PCR product visualization uses 5% agarose gel electrophoresis. In
            this protocol, the multiSUB Choice, Wide Midi Horizontal
            Electrophoresis System was used (Cleaver Scientific).
          </li>
        </ul>
      </li>
    </ul>
  </section>
  <section>
    <h2 id="step-by-step-method-details">Step-by-step method details</h2>
    <p>
      The present protocol describes the procedure for a conventional RT-PCR
      reaction with locally produced enzymes (<a href="#bib17">Mendoza-Rojas et al., 2021</a>) coupled to CRISPR/Cas12a detection using a fluorescence microplate
      reader, or direct fluorescence visualization (<a href="#bib1">Alcántara et al., 2021</a>). This protocol is intended to be used for SARS-CoV-2 RNA detection.
      However, the described methodology could be set up as a base protocol for
      molecular detection of nucleic acid molecules (RNA/DNA) of any other
      pathogen or microorganism of interest.
    </p>
    <div class="note">
      <span class="note-title">Note:</span> In this protocol, we describe RT-PCR
      and CRISPR/Cas recipes for 32 reactions, to fit in the BentoLab portable
      system. All the examples in the article correspond to a 32 reactions
      workflow (28 samples + controls). Scale the required volume of each
      reagent depending on the total number of samples to test or if a 96-well
      thermocycler is available.
    </div>
    <h3 id="sec3.1">Sample RNA extraction</h3>
    <p>
      This protocol has been validated with nasopharyngeal swab samples. Use
      your locally approved viral RNA extraction or other approved commercial
      alternative kits. Here, the viral RNA material was extracted using the
      QIAamp viral RNA Mini kit (QIAGEN).
    </p>
    <ol>
      <li>
        To perform RNA extraction, 200 μL of sample or negative extraction
        control (NEC) was used according to the manufacturer protocol (<a href="https://www.qiagen.com/us/resources/download.aspx?id=c80685c0-4103-49ea-aa72-8989420e3018&amp;lang=en">https://www.qiagen.com/us/resources/download.aspx?id=c80685c0-4103-49ea-aa72-8989420e3018&amp;lang=en</a>). Proceed with the Spin protocol or Vacuum protocol depending on
        vacuum pump and manifold availability.
      </li>
      <li>Elute RNA with 100 μL of nuclease-free water.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> A negative extraction control (NEC)
      should be included during the RNA extraction step to evaluate
      contamination during sample handling. A NEC could be a pool of verified
      negative patient samples, commercial alternatives (i.e., human DNA in
      stabilizing synthetic matrix, such as the one available in the HDPCR<sup>TM</sup>
      SARS-CoV-2 assay (ChromaCode)), or even molecular-grade nuclease-free
      water.
    </div>
    <h3 id="sec3.2">One-step RT-PCR</h3>
    <div class="timing">
      <span class="timing-title">Timing:</span> two hours from reaction
      preparation to results
    </div>
    <p>
      This protocol was standardized to amplify two gene SARS-CoV-2 targets, at
      the 5′- and 3′-end of the genome, as well as a region of the RNAseP human
      gene as RNA extraction control using a one-step RT-PCR reaction. Examples
      used M-MLV reverse transcriptase (RT) and Taq DNA polymerase produced in
      the laboratory. Commercial enzymes can be used as an alternative.
    </p>
    <div class="alternatives">
      <span class="alternatives-title">Alternatives:</span> The following
      commercial kits have been evaluated in replacement of the enzymes
      mentioned here, M-MLV reverse transcriptase and/or Taq-polymerase.
      Commercial kits can be used as one- or two-step RT-PCR reactions without
      interfering with the CRISPR/Cas detection assay. For one-step reactions we
      tested the 2<b>×</b> One-Step RT-PCR Master Mix (Cat. No. 28113, Norgen
      Biotek). For two-step reactions we tested RevertAid First Strand cDNA
      Synthesis Kit (Cat. No. K1622, ThermoFisher) coupled with DreamTaq Green
      PCR Master Mix (2<b>×</b>) (Cat. No. K1081, ThermoFisher) or Taq DNA
      polymerase with Standard Taq Buffer (Cat. No. M0273S, NEB). Other
      commercial enzymes are expected to be suitable for nucleic acid
      amplification followed by detection with the proposed CRISPR/Cas system
      described in this protocol.
    </div>
    <p>
      The one-step RT-PCR reaction produces first a DNA copy (cDNA) of the
      target RNA molecules extracted from nasopharyngeal swabs or saliva
      samples. After this reverse transcription step, the cDNA template is
      amplified by the Taq DNA polymerase to generate dsDNA copies of the target
      region of interest (<a href="#bib2">Álvarez-Fernández, 2013</a>;
      <a href="#bib6">Chaubal et al., 2018</a>;
      <a href="#bib9">Graham et al., 2021</a>;
      <a href="#bib16">Mahony et al., 2011</a>).
    </p>
    <div class="note">
      <span class="note-title">Note:</span> Each gene target is amplified
      individually by RT-PCR.
    </div>
    <ol start="3">
      <li>
        Master mix preparation
        <div class="critical">
          <span class="critical-title">Critical:</span> It is strongly
          recommended to prepare the master mix in a different laboratory area
          from where clinical samples are manipulated or where the samples are
          analyzed. Amplicon contamination can severely compromise your results.
          Follow the general recommendations for molecular analysis of clinical
          samples (WHO. <a href="#bib20">World Health Organization. (2018)</a>)
        </div>
        <div class="pause-point">
          <span class="pause-point-title">Pause point:</span> Thaw all reagents
          on ice, while pre-heating a thermoblock or thermocycler at 50°C.
        </div>
        <ol type="a">
          <li>
            Prepare the master mix according to the following recipe (i.e.,
            scale the required volume of each reagent depending on the total
            number of samples to test). Here we work with 33 reactions
            corresponding to a final volume of 660 μL considering a 2 μL sample
            to be analyzed and a total volume of 20 μL for each reaction.
            <table id="undtbl10">
              <thead>
                <tr>
                  <th>Reagent</th>
                  <th>Initial concentration</th>
                  <th>Final concentration</th>
                  <th>Volume (μL) per reaction</th>
                  <th>Volume (μL) for 33 reactions</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>RPB4<b>×</b>
</td>
                  <td>4<b>×</b>
</td>
                  <td>1<b>×</b>
</td>
                  <td>5</td>
                  <td>165</td>
                </tr>
                <tr>
                  <td>M-MLV RT</td>
                  <td>20<b>×</b>
</td>
                  <td>1<b>×</b>
</td>
                  <td>1</td>
                  <td>33</td>
                </tr>
                <tr>
                  <td>Taq polymerase</td>
                  <td>20<b>×</b>
</td>
                  <td>1<b>×</b>
</td>
                  <td>1</td>
                  <td>33</td>
                </tr>
                <tr>
                  <td>Reverse primer</td>
                  <td>5 μM</td>
                  <td>0.2 μM</td>
                  <td>0.8</td>
                  <td>26.4</td>
                </tr>
                <tr>
                  <td>Forward primer</td>
                  <td>5 μM</td>
                  <td>0.2 μM</td>
                  <td>0.8</td>
                  <td>26.4</td>
                </tr>
                <tr>
                  <td>Nuclease-free water</td>
                  <td>-</td>
                  <td>-</td>
                  <td>9.4</td>
                  <td>310.2</td>
                </tr>
                <tr>
                  <td><b>Final volume</b></td>
                  <td></td>
                  <td></td>
                  <td><b>18</b></td>
                  <td><b>594</b></td>
                </tr>
              </tbody>
            </table>
            <div class="note">
              <span class="note-title">Note:</span> It is recommended to prepare
              1-2 additional reactions due to some volume loss by pipetting.
            </div>
            <div class="critical">
              <span class="critical-title">Critical:</span> We strongly
              recommend preparing and maintaining the master mix on ice (or use
              an IsoFreeze® PCR rack) to avoid unspecific product amplification
              (i.e., primer dimerization) (<a href="#bib9">Graham et al., 2021</a>).
            </div>
          </li>
          <li>Mix by gently vortexing and spin down for few seconds.</li>
          <li>
            Dispense 18 μL in PCR tubes (0.2 mL). Individual tubes or PCR strips
            can be used.
          </li>
          <li>
            Securely close all tubes and spin down the tubes for few seconds.
          </li>
        </ol>
      </li>
    </ol>
    <p>Place the tubes on ice (or using an IsoFreeze® PCR rack)</p>
    <div class="critical">
      <span class="critical-title">Critical:</span> To avoid amplicon
      contamination, the addition of RNA samples extracted from clinical samples
      should be performed in a dedicated laboratory area, pipette set, and BSL2
      hood. This area should be restricted to any incoming amplified material.
      Gloves and coats should also be dedicated to this area.
    </div>
    <ol start="4">
      <li>
        Add 2 μL of extracted RNA samples to each reaction (step 2). To evaluate
        potential reaction contamination and reaction performance, a no template
        control (NTC), a NEC (step 1), and a positive control should be included
        in each assay, respectively.
      </li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> For the NTC, nuclease-free water
      must be added to the respective PCR tube. For the positive control,
      genomic RNA, RNA standards (e.g., as the ones available in the BEI
      Resources catalogue (<a href="https://www.beiresources.org/">https://www.beiresources.org/</a>)) or sample pools from certified positive samples for SARS-CoV-2 can be
      used.
    </div>
    <ol start="5">
      <li>
        Insert the tubes or strips into the thermocycler from the BentoLab
        system (alternative thermocyclers can be used instead) (<a href="#fig1">Figure 1</a>).
        <figure id="fig1">
          <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Fig1.jpg" alt="Fig1.jpg">
          <figcaption>
            <div class="figcaption-title">
              Figure 1. Recommended plate layout for 32 reactions to be
              processed in the BentoLab
            </div>
          </figcaption>
        </figure>
      </li>
    </ol>
    <div class="critical">
      <span class="critical-title">Critical:</span> Thermocyclers must be in a
      dedicated area. Ensure that all tubes or strips are sealed properly to
      avoid cross-contamination between samples and to protect samples from
      evaporation.
    </div>
    <ol start="6">
      <li>Run the following RT-PCR program:</li>
    </ol>
    <table id="undtbl11">
      <thead>
        <tr>
          <th colspan="4">PCR cycling conditions</th>
        </tr>
        <tr>
          <th>Step</th>
          <th>Temperature</th>
          <th>Time</th>
          <th>Cycles</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Reverse transcription</td>
          <td>50°C</td>
          <td>20 min</td>
          <td>1</td>
        </tr>
        <tr>
          <td>Initial denaturation</td>
          <td>95°C</td>
          <td>5 min</td>
          <td>1</td>
        </tr>
        <tr>
          <td>Denaturation</td>
          <td>95°C</td>
          <td>5 s</td>
          <td rowspan="2">45 cycles</td>
        </tr>
        <tr>
          <td>Annealing/Extension</td>
          <td>55°C</td>
          <td>30 s</td>
        </tr>
        <tr>
          <td>Hold</td>
          <td>10°C</td>
          <td colspan="2">Forever</td>
        </tr>
      </tbody>
    </table>
    <div class="note">
      <span class="note-title">Note:</span> Total run time 1:40 hours
    </div>
    <div class="pause-point">
      <span class="pause-point-title">Pause point:</span> Meanwhile the
      CRISPR/Cas reaction can be prepared (see CRISPR/Cas reaction below).
    </div>
    <ol start="7">
      <li>
        Store the RT-PCR products at 4°C for short term or at −20°C for mid/long
        term until the CRISPR/Cas detection assay is performed.
      </li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> Optional, the RT-PCR product can be
      visualized by agarose gel electrophoresis. RT-PCR products are expected to
      be 168 bp for the ORF1ab target, 131 bp for the N target, and 175 bp for
      the RNAseP target (<a href="#fig2">Figure 2</a>). If RT-PCR product bands
      are not visible, see
      <a href="#troubleshooting">troubleshooting</a> section –
      <a href="#sec7.1">problem 1</a>.
    </div>
    <figure id="fig2">
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Fig2.jpg" alt="Fig2.jpg">
      <figcaption>
        <div class="figcaption-title">
          Figure 2. RT-PCR products from positive and negative controls for
          ORF1ab, N and RNAseP targets
        </div>
        <p>
          The amplification products were run on a 5% agarose gel at 70 V for
          60 min. AmpliSize Molecular Ruler (Cat. No. 1708200, Bio-Rad) was used
          as ladder. Products were visualized using SafeGreen (Cat. No. G108-G,
          abm) in the loading buffer TriTrack DNA loading dye 6<b>×</b> (Cat.
          No. R1161, ThermoFisher).
        </p>
      </figcaption>
    </figure>
    <h3 id="sec3.3">CRISPR/Cas12a-mediated detection</h3>
    <div class="timing">
      <span class="timing-title">Timing:</span> 35 minutes from reaction
      preparation to results
    </div>
    <p>
      This protocol was standardized to detect two <i>loci</i> at the 5′- and
      3′-end of the SARS-CoV-2 genome, respectively, and a region of the RNAseP
      human gene. The following crRNA sequences were used for protocol
      standardization and validation:
    </p>
    <table id="undtbl12">
      <thead>
        <tr>
          <th>Target</th>
          <th>Sequence (5’ – 3′)</th>
          <th>length (bp)</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>ORF1ab</td>
          <td>UAAUUUCUACUAAGUGUAGAUUUAGAGACGGUUGGGAAAUUG</td>
          <td>42</td>
        </tr>
        <tr>
          <td>N</td>
          <td>UAAUUUCUACUAAGUGUAGAUCCCCCAGCGCUUCAGCGUUC</td>
          <td>41</td>
        </tr>
        <tr>
          <td>RNAseP</td>
          <td>UAAUUUCUACUAAGUGUAGAUAAUUACUUGGGUGUGACCCU</td>
          <td>41</td>
        </tr>
      </tbody>
    </table>
    <p>
      The LbCas12a enzyme detects a specific DNA sequence by recognizing the
      base pairing between crRNA and the target (<a href="#bib7">Chen et al., 2018</a>; <a href="#bib18">Swarts et al., 2017</a>). Upon recognition of the
      target DNA, Cas12a undergoes structural changes in its RuvC active site
      that result in an unspecific trans-cleavage activity on any available
      single-stranded DNA (ssDNA) molecule in the vicinity (<a href="#bib7">Chen et al., 2018</a>; <a href="#bib15">Li et al., 2019</a>). This has been adapted for
      <i>in vitro</i> detection of a target DNA of interest for diagnostic
      proofs of concept (<a href="#bib4">Broughton et al., 2020</a>;
      <a href="#bib7">Chen et al., 2018</a>). The CRISPR/Cas12a assay uses a
      double-labeled reporter probe (i.e., with a fluorophore and quencher) that
      is cleaved upon Cas12a trans-activity only in the presence of the specific
      target DNA in the reaction, thereby generating a fluorescent signal that
      can be detected with a fluorescence plate reader or by direct
      visualization using a transilluminator (<a href="#bib3">Bonini et al., 2020</a>; <a href="#bib14">Kumar et al., 2020</a>).
    </p>
    <div class="note">
      <span class="note-title">Note:</span> The following example concentrations
      and volumes are indicated for the previous 32 RT-PCR reactions for the
      ORF1ab target that include 28 unknown samples, two positive controls and
      two no template controls (NTC). Each target RT-PCR product (ORF1ab, N, and
      RNaseP) is evaluated individually using the CRISPR/Cas12a assay. An Excel
      template (Mendeley Data:
      <a href="https://doi.org/10.17632/8m8z37v8xz.1">https://doi.org/10.17632/8m8z37v8xz.1</a>
      Reaction recipe template) is provided to calculate the necessary volume of
      the required reagents for crRNA re-folding and Cas12a master mix
      preparation.
    </div>
    <div class="note">
      <span class="note-title">Note:</span> All buffers required in this segment
      must be prepared in advance.
    </div>
    <ol start="8">
      <li>Thaw all reagents on ice</li>
    </ol>
    <div class="pause-point">
      <span class="pause-point-title">Pause point:</span> Pre-heating a
      thermoblock at 65°C.
    </div>
    <ol start="9">
      <li>
        crRNA re-folding
        <ol type="a">
          <li>
            Prepare the working stock by diluting the crRNA stock in
            nuclease-free water to get a final concentration of 1.3 μM (see
            <a href="#before-you-begin">before you begin</a>, Step 6I, for crRNA
            production).
          </li>
          <li>Mix gently by vortexing or tapping, and spin briefly.</li>
          <li>Incubate at 65°C for 10 min.</li>
          <li>Cool at 25°C for 10 min.</li>
          <li>Mix gently by vortexing or tapping. Then spin briefly.</li>
        </ol>
      </li>
    </ol>
    <ol start="10">
      <li>
        10<b>×</b> Cas12a complex
        <ol type="a">
          <li>
            Prepare the 10<b>×</b> Cas12a complex according to the following
            recipe (i.e., scale the required volume of each reagent depending on
            the total number of samples to test or your stocks’ concentration).
            <table id="undtbl13">
              <thead>
                <tr>
                  <th>Reagent</th>
                  <th>Initial concentration</th>
                  <th>Final concentration</th>
                  <th>Volume (μL) per reaction</th>
                  <th>
                    Volume (μL) for 33 reactions<sup><a href="#dtbl14fna">a</a></sup>
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>Cas12a</td>
                  <td>5.4 μM</td>
                  <td>0.1μM</td>
                  <td>0.184</td>
                  <td>6.1</td>
                </tr>
                <tr>
                  <td>Re-folded crRNA</td>
                  <td>1.3 μM</td>
                  <td>0.15 μM</td>
                  <td>1.164</td>
                  <td>38.4</td>
                </tr>
                <tr>
                  <td>Probe</td>
                  <td>100 μM</td>
                  <td>3 μM</td>
                  <td>0.3</td>
                  <td>9.9</td>
                </tr>
                <tr>
                  <td>1<b>×</b> CrB1</td>
                  <td>-</td>
                  <td>-</td>
                  <td>8.351</td>
                  <td>275.6</td>
                </tr>
                <tr>
                  <td>Final volume</td>
                  <td>-</td>
                  <td></td>
                  <td></td>
                  <td>330</td>
                </tr>
              </tbody>
            </table>
            <p class="table-footnote" id="dtbl14fna">
              <sup>a</sup>Considering one additional reaction due to volume loss
              by pipetting.
            </p>
          </li>
          <li>
            Protect from light by wrapping the tube with aluminum foil or using
            dark tubes, if available.
          </li>
          <li>Incubate the 10<b>×</b> Cas12a complex at 25°C for 10 min.</li>
        </ol>
      </li>
    </ol>
    <ol start="11">
      <li>
        CRISPR/Cas12a reaction preparation
        <ol type="a">
          <li>
            Each CRISPR/Cas detection reaction is prepared according to the
            following scheme:
            <table id="undtbl14">
              <thead>
                <tr>
                  <th>PipettingOrder</th>
                  <th>Reagent</th>
                  <th>Initial concentration</th>
                  <th>Final concentration</th>
                  <th>Final volume (μL) per assay</th>
                  <th>Notes</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>1</td>
                  <td>CrB2 (17.65 mM MgCl<sub>2</sub>)</td>
                  <td>1<b>×</b>
</td>
                  <td>1<b>×</b>
</td>
                  <td>85</td>
                  <td rowspan="2">Mix separately</td>
                </tr>
                <tr>
                  <td>2</td>
                  <td>RT-PCR product</td>
                  <td>-</td>
                  <td>-</td>
                  <td>5</td>
                </tr>
                <tr>
                  <td>3</td>
                  <td>Cas12a complex</td>
                  <td>10<b>×</b>
</td>
                  <td>1<b>×</b>
</td>
                  <td>10</td>
                  <td>Add right before measuring</td>
                </tr>
                <tr>
                  <td></td>
                  <td>Final volume</td>
                  <td>-</td>
                  <td>-</td>
                  <td>100</td>
                  <td></td>
                </tr>
              </tbody>
            </table>
            <div class="note">
              <span class="note-title">Note:</span> CrB2 buffer contains
              17.65 mM MgCl<sub>2</sub>, which upon addition of the RT-PCR
              product and the 10<b>×</b> Cas12a complex brings MgCl<sub>2</sub>
              to 15 mM.
            </div>
          </li>
          <li>
            Dispense 85 μL of 1<b>×</b> CrB2 in separate tubes, PCR strips, or a
            96-well PCR plate.
          </li>
          <li>Add 5 μL of the RT-PCR product (Step 7).</li>
          <li>
            Mix gently by tapping or using a plate mixer if available. If
            working with PCR tubes or strips, spin briefly.
            <div class="note">
              <span class="note-title">Note:</span> When working with a
              multichannel pipette consider increasing the volumes of the
              first two pipetting steps to avoid inconsistent pipetting. 102 μL
              CrB2 and 6 μL RT-PCR product for a total of 108 μL from which
              90 μL are taken with the multichannel pipette for further
              analysis.
            </div>
          </li>
          <li>
            Dispense 10 μL of the 10<b>×</b> Cas12a complex in every well of the
            96-well microplate (Step 10c) (<a href="#fig3">Figure 3</a>).
            <figure id="fig3">
              <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Fig3.jpg" alt="Fig3.jpg">
              <figcaption>
                <div class="figcaption-title">
                  Figure 3. Recommended plate layout for CRISPR/Cas reactions in
                  a 96-well microplate
                </div>
                <p>
                  For each target 32 reactions are included (28 unknown
                  samples + duplicates of positive controls, negative
                  extraction, and no template controls).
                </p>
              </figcaption>
            </figure>
          </li>
          <li>
            Add 90 μL of the diluted RT-PCR products according to the above
            plate layout (Step 11c).
          </li>
        </ol>
      </li>
    </ol>
    <p>
      Alternative for the Cas12a complex to set up CRISPR/Cas reactions: the
      EnGen® Lba Cas12a (Cpf1) (Cat. No. M0653S – M0653T, NEB) can be used with
      the NEB buffer 1<b>×</b>.
    </p>
    <ol start="12">
      <li>
        Outcome readout A quantitative and a qualitative approach have been
        evaluated using a microplate reader or by direct fluorescence
        visualization in microtubes or plates, respectively. The fluorescence is
        produced by the cleavage of the double-labeled probe
        (56-FAM/TTATT/3IABkFQ) due to target recognition by the CRISPR/Cas12a
        system.
        <ol type="a">
          <li>
            Quantitative readout
            <div class="note">
              <span class="note-title">Note:</span> The assay was tested using
              the Synergy<sup>TM</sup> H1 Hybrid Multi-Mode microplate reader
              (BioTek), with software Gen5 v3.10.
            </div>
            <ol type="i">
              <li>
                Set-up the instrument according to the following parameters:
                <table id="undtbl15">
                  <thead></thead>
                  <tbody>
                    <tr>
                      <td><b>General</b></td>
                      <td></td>
                    </tr>
                    <tr>
                      <td>Plate type</td>
                      <td>Select the respective black plate</td>
                    </tr>
                    <tr>
                      <td>Temperature</td>
                      <td>25°C</td>
                    </tr>
                    <tr>
                      <td><b>Kinetic step</b></td>
                      <td></td>
                    </tr>
                    <tr>
                      <td>Runtime</td>
                      <td>35 min</td>
                    </tr>
                    <tr>
                      <td>Interval</td>
                      <td>1 min</td>
                    </tr>
                    <tr>
                      <td>
                        <b>Read method</b><sup><a href="#dtbl16fna">a</a></sup>
                      </td>
                      <td></td>
                    </tr>
                    <tr>
                      <td>Detection method</td>
                      <td>Fluorescence intensity</td>
                    </tr>
                    <tr>
                      <td>Read type</td>
                      <td>Endpoint/Kinetic</td>
                    </tr>
                    <tr>
                      <td>Optics type</td>
                      <td>Monochromators</td>
                    </tr>
                    <tr>
                      <td>Excitation (nm)</td>
                      <td>491</td>
                    </tr>
                    <tr>
                      <td>Emission (nm)</td>
                      <td>525</td>
                    </tr>
                    <tr>
                      <td>Gain</td>
                      <td>85</td>
                    </tr>
                    <tr>
                      <td>Read height</td>
                      <td>7 mm</td>
                    </tr>
                  </tbody>
                </table>
                <p class="table-footnote" id="dtbl16fna">
                  <sup>a</sup>If using an equipment that works with a variable
                  bandwidth (i.e., Cytation 5 Multi-Mode reader, Biotek)
                  consider using the following wavelengths: ex 491±9 nm, and em
                  525±21 nm. If your instrument uses excitation and emission
                  filters, use the corresponding set for Fluorescein (Usually
                  pre-set as GFP)
                </p>
              </li>
              <li>
                Transfer the microplate to the reading instrument and proceed
                with the measurement.
              </li>
            </ol>
          </li>
          <li>
            Qualitative readout
            <div class="note">
              <span class="note-title">Note:</span> The protocol was tested
              using the transilluminator - safeVIEW: LED/Blue Light (470 nm blue
              LEDs) (Cleaver Scientific).
            </div>
            <ol type="i">
              <li>
                Once the CRISPR/Cas12a reaction is prepared, incubate without
                light exposure for 30 min.
              </li>
              <li>
                Transfer the microplate to the transilluminator and close the
                filter tap.
              </li>
              <li>
                Register the results of direct fluorescence visualization using
                a photo camera or a smartphone.
                <div class="note">
                  <span class="note-title">Note:</span> Using a black cabinet,
                  box, or dark room improves the direct visualization of the
                  results.
                </div>
              </li>
            </ol>
          </li>
        </ol>
      </li>
    </ol>
  </section>
  <section>
    <h2 id="expected-outcomes">Expected outcomes</h2>
    <p>
      Regarding the fluorescence detection in unknown samples, the fluorescence
      must rapidly increase over time for positive samples. Negative samples and
      reaction controls do not show a marked increase. However, they can
      slightly increase over time. An appropriate variable to compare results is
      the fluorescence ratio between the analyzed sample over the non-template
      control (NTC) (see the
      <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>
      section)(<a href="#bib1">Alcántara et al., 2021</a>;
      <a href="#bib12">Hou et al., 2020</a>). The fluorescence ratio should be
      higher for positive samples than for the negative ones. There is a
      positive dependence between fluorescence ratios and the viral RNA load of
      the sample, higher RNA loads derive in higher fluorescence ratios (<a href="#fig4">Figure 4</a>A). In the validation study with 100 clinical samples, the median
      fluorescence ratio for positive samples was higher for ORF1ab target
      (11.4) than the one observed for the N target (7.9) (<a href="#fig4">Figure 4</a>B) (<a href="#bib1">Alcántara et al., 2021</a>). For both detection
      targets, positive samples show a fluorescence ratio over 4-fold with
      respect to negative samples. For results interpretation, the fluorescence
      ratio of the RNaseP human gene must be considered (see below).
    </p>
    <figure id="fig4">
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Fig4.jpg" alt="Fig4.jpg">
      <figcaption>
        <div class="figcaption-title">
          Figure 4. Fluorescence ratio distribution for clinical samples in this
          example
        </div>
        <p>
          Data from 50 SARS-CoV-2 positive and 50 negative samples are shown.
        </p>
        <p>
          (A) Fluorescence distribution for SARS-CoV-2 positive samples sorted
          by viral RNA load for the ORF1ab and N targets.
        </p>
        <p>
          (B) Fluorescence distribution for SARS-CoV-2 positive and negative
          samples for the two targets (ORF1ab, N).
        </p>
        <p>
          (C) RNaseP fluorescence ratio distribution for SARS-CoV-2 samples,
          sorted by viral RNA load, positive and, negative controls. Tukey
          boxplots in A-C indicate the median, 25%, and 75% percentiles.
          Whiskers indicate minimun and maximun values excluding outliers (full
          circles).
        </p>
      </figcaption>
    </figure>
    <p>
      The fluorescence signal of the RNaseP gene must increase over time for
      both SARS-CoV-2 positive and negative clinical samples. Fluorescence
      ratios of the RNaseP target over the NTC control are expected to be around
      25 independently of the viral RNA load (<a href="#fig4">Figure 4</a>C).
      Fluorescence ratios for the RNaseP target are higher than the ones
      observed for SARS-CoV-2 targets. In the validation study, the median
      fluorescence ratio for the 100 tested clinical samples for RNaseP was
      around 25 (<a href="#bib1">Alcántara et al., 2021</a>). If the
      fluorescence ratio is less than two, the assay must be repeated to rule
      out any technical error. If the repeated test fluorescence ratio is less
      than two, the sample quality must be checked, or a new sample should be
      required (see <a href="#troubleshooting">troubleshooting</a> section).
    </p>
    <p>
      Fluorescence ratios for positive controls must be higher than the negative
      controls. In the validation study with 100 clinical samples, a mean
      fluorescence ratio of 13.96 ± 1.72 RFU and 0.64 ± 0.14 RFU were reported
      for the positive and negative controls for the ORF1ab target,
      respectively. Likewise, an average fluorescence ratio of 8.37 ± 0.35 RFU
      and 1.05 ± 0.43 RFU were calculated for the positive and negative controls
      for the N target, respectively (<a href="#fig5">Figure 5</a>). If
      fluorescence is not observed, see the
      <a href="#troubleshooting">troubleshooting</a> section –
      <a href="#sec7.3">problems 2</a> and/or <a href="#sec7.5">3</a>.
    </p>
    <figure id="fig5">
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Fig5.jpg" alt="Fig5.jpg">
      <figcaption>
        <div class="figcaption-title">
          Figure 5. Fluorescence ratio for positive and negative controls
        </div>
        <p>
          Fluorescence ratio data represent the mean of five independent
          experiments. Error bars show standard deviation.
        </p>
      </figcaption>
    </figure>
    <p>
      Finally, the observation of fluorescence is indicative of a positive
      result for SARS-CoV-2. Positive samples show a high fluorescence intensity
      (<a href="#fig6">Figure 6</a>). If weak fluorescence signal is visualized,
      see the <a href="#troubleshooting">troubleshooting</a> section –
      <a href="#sec7.7">problem 4</a>.
    </p>
    <figure id="fig6">
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Fig6.jpg" alt="Fig6.jpg">
      <figcaption>
        <div class="figcaption-title">
          Figure 6. Visual detection of the amplified N target using
          CRISPR/Cas12a-dependent fluorescence signal
        </div>
        <p>
          RT-PCR products of positive and negative SARS-CoV-2 samples were
          observed in a blue light LED transilluminator.
        </p>
      </figcaption>
    </figure>
  </section>
  <section>
    <h2 id="quantification-and-statistical-analysis">
      Quantification and statistical analysis
    </h2>
    <p>
      In the quantitative approach, the fluorescence ratios over the NTC are
      evaluated to determine positivity.
    </p>
    <div class="note">
      <span class="note-title">Note:</span> An Excel template (Mendeley Data:
      <a href="https://doi.org/10.17632/8m8z37v8xz.1">https://doi.org/10.17632/8m8z37v8xz.1</a>. Fluorescence ratios calculation template) can be used to calculate the
      fluorescence ratio of each sample. Sheet 1 is the original readout report
      and exported as an Excel file from the Biotek instruments. The template
      must be modified if using a different instrument and software. Sheet 2
      shows the normalized fluorescence ratios between the sample and the NTC
      control.
    </div>
    <ol>
      <li>
        Export the raw data for all samples as an Excel spreadsheet. Relative
        fluorescence unit (RFU) time courses will appear in one column for each
        sample/well (<a href="#tbl1">Table 1</a>).
        <table id="tbl1">
          <caption>
            Table 1. Example of raw data measured on a plate reader
          </caption>
          <thead>
            <tr>
              <th>Time</th>
              <th>Temp.</th>
              <th colspan="6">Raw data</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>-</td>
              <td>-</td>
              <td>Sample Example</td>
              <td>Sample Example</td>
              <td>Positive control</td>
              <td>Positive control</td>
              <td>NEC</td>
              <td>NTC</td>
            </tr>
            <tr>
              <td colspan="2">Plate wells in File S2</td>
              <td>C5</td>
              <td>C7</td>
              <td>E8</td>
              <td>F8</td>
              <td>G8</td>
              <td>H8</td>
            </tr>
            <tr>
              <td>0:00:00</td>
              <td>25°C</td>
              <td>3255</td>
              <td>2323</td>
              <td>5298</td>
              <td>4210</td>
              <td>2388</td>
              <td>2182</td>
            </tr>
            <tr>
              <td>00:15:00</td>
              <td>25°C</td>
              <td>11773</td>
              <td>3051</td>
              <td>24534</td>
              <td>21618</td>
              <td>3633</td>
              <td>3431</td>
            </tr>
            <tr>
              <td>00:30:00</td>
              <td>25°C</td>
              <td>22442</td>
              <td>3759</td>
              <td>47499</td>
              <td>43959</td>
              <td>5366</td>
              <td>5015</td>
            </tr>
          </tbody>
        </table>
      </li>
      <li>
        Normalize the fluorescence signal by subtracting the first value (Time =
        0) for each sample (<a href="#tbl2">Table 2</a>).
        <table id="tbl2">
          <caption>
            Table 2. Example of normalized data measured on a plate reader
          </caption>
          <thead>
            <tr>
              <th>Time</th>
              <th>Temp.</th>
              <th colspan="6">Raw data</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>-</td>
              <td>-</td>
              <td>Sample Example</td>
              <td>Sample Example</td>
              <td>Positive control</td>
              <td>Positive control</td>
              <td>NEC</td>
              <td>NTC</td>
            </tr>
            <tr>
              <td colspan="2">Plate wells in File S2</td>
              <td>C5</td>
              <td>C7</td>
              <td>E8</td>
              <td>F8</td>
              <td>G8</td>
              <td>H8</td>
            </tr>
            <tr>
              <td>00:15:00</td>
              <td>25°C</td>
              <td>8518</td>
              <td>728</td>
              <td>19236</td>
              <td>17408</td>
              <td>1245</td>
              <td>1249</td>
            </tr>
            <tr>
              <td>00:30:00</td>
              <td>25°C</td>
              <td>19187</td>
              <td>1436</td>
              <td>42201</td>
              <td>39749</td>
              <td>2978</td>
              <td>2833</td>
            </tr>
          </tbody>
        </table>
      </li>
      <li>
        Calculate the mean for fluorescence values of the positive control.
      </li>
      <li>
        Calculate the fluorescence ratio for each control and sample using the
        following equation:
      </li>
    </ol>
    <figure id="si1" style="width: 127px">
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Si1.jpg" alt="Si1.jpg">
    </figure>
    <p>
      Divide the fluorescence signal (RFU) of test samples (positive control or
      NEC) by the NTC values, both measured at the timepoint 30 min (<a href="#tbl3">Table 3</a>).
    </p>
    <table id="tbl3">
      <caption>
        Table 3. Example of fluorescence ratio for normalized data measured on a
        plate reader
      </caption>
      <thead>
        <tr>
          <th>Time</th>
          <th>Temp.</th>
          <th colspan="6">Raw data</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>-</td>
          <td>-</td>
          <td>Sample Example</td>
          <td>Sample Example</td>
          <td>Positive control</td>
          <td>Positive control</td>
          <td>NEC</td>
          <td>NTC</td>
        </tr>
        <tr>
          <td colspan="2">Plate wells in File S2</td>
          <td>C5</td>
          <td>C7</td>
          <td>E8</td>
          <td>F8</td>
          <td>G8</td>
          <td>H8</td>
        </tr>
        <tr>
          <td>00:30:00</td>
          <td>25°C</td>
          <td>6.604</td>
          <td>0.494</td>
          <td>14.525</td>
          <td>13.681</td>
          <td>1.025</td>
          <td>0.975</td>
        </tr>
      </tbody>
    </table>
    <ol start="5">
      <li>
        Determine positivity following the algorithm shown in
        <a href="#fig7">Figure 7</a>,
        <figure id="fig7">
          <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Fig7.jpg" alt="Fig7.jpg">
          <figcaption>
            <div class="figcaption-title">Figure 7. Test decision scheme</div>
            <p>
              Cut-off values were calculated by a Receiver Operating
              Characteristic (ROC) curve analysis (<a href="#bib11">Hanley and McNeil, 1982</a>). The selected cut-off values were those that reported the
              highest percentage of correctly classified samples according to a
              ROC curve analysis with 100 samples (<a href="#bib1">Alcántara et al., 2021</a>). ∗ Test again if symptoms occur. If symptoms already present
              RT-qPCR recommended. ∗∗ If an invalid result is obtained, see
              <a href="#troubleshooting">troubleshooting</a> section –
              <a href="#sec7.9">problem 5</a>.
            </p>
          </figcaption>
        </figure>
      </li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> The method using this protocol
      showed a Limit of Detection (LoD) of 10<sup>2</sup> ge/μL using SARS-CoV-2
      genomic RNA (NR-52347, BEI Resources). During the validation process five
      out of 50 clinical samples showed false negative results. All discordant
      samples showed Cq values more than 33 (Alcántara <i>et al.,</i> 2021).
      However, Cq value distribution in infected people peaked between 23 – 25
      Cq values (<a href="#bib5">Buchan et al., 2020</a>). In that context, the
      described protocol here should be able to detect all clinical samples.
    </div>
  </section>
  <section>
    <h2 id="limitations">Limitations</h2>
    <p>
      This protocol has been validated only with clinical upper-respiratory
      samples (n = 100) and performed in one country (Peru). Although no
      mutations have been reported in the SARS-CoV-2 target sequences used in
      this protocol (ORF1ab and N), the test sensitivity could vary depending on
      local circulating SARS-CoV-2 strains. The detection targets, used in this
      protocol, have been selected considering genome regions with low
      variability. We cannot exclude that false negative results could arise due
      to future variants of SARS-CoV-2. We recommend conducting a local clinical
      validation before using it as a laboratory-developed test (LDT) for
      clinical analysis. Negative results do not exclude COVID-19 disease.
      Molecular tests are part of the diagnostic algorithms worldwide that
      include other variables such as symptoms, contact to diagnosed patients,
      among others. False negative results can be produced by errors during
      sample manipulation and processing. This protocol is not authorized by any
      governmental institution yet. A technical guide to validate in-house
      diagnostic tests can be downloaded from the FIND website (<a href="https://www.finddx.org/reports-and-landscapes/in-house-test-development-for-molecular-detection-of-sars-cov-2-en-fr-pt/">https://www.finddx.org/reports-and-landscapes/in-house-test-development-for-molecular-detection-of-sars-cov-2-en-fr-pt/</a>). Other outcome readouts such as lateral flow assay on paper strips have
      not been validated with this protocol.
    </p>
  </section>
  <section>
    <h2 id="troubleshooting">Troubleshooting</h2>
    <h3 id="sec7.1">Problem 1</h3>
    <p>No amplification for the positive control in RT-PCR (RT_PCR_outcome)</p>
    <p>
      The lack of RT-PCR products for the positive control indicates that a
      technical error has occurred during the preparation of the master mix or
      the RT-PCR running step. Also, that the quality of the reagents could be
      compromised. Low or null yield amplification can produce a low
      fluorescence signal that could be misinterpreted as a negative result.
    </p>
    <h3 id="sec7.2">Potential solution</h3>
    <p>
      Check that the used cycling conditions are correct. Storage conditions of
      the reagents, enzymes and aliquots of the controls must be checked.
      Replace the working stock of reagents, enzymes or aliquots of the controls
      if necessary.
    </p>
    <h3 id="sec7.3">Problem 2</h3>
    <p>
      High fluorescence signal in the no template control and/or negative
      extraction control (Fluorescence_outcome)
    </p>
    <p>
      A high fluorescence signal in the NTC indicates contamination during the
      CRISPR/Cas reaction or more likely during the amplification step (RT-PCR).
      A high fluorescence signal in the NTC gives lower fluorescence ratios that
      could be misinterpreted as negative results in the test samples (false
      negative). In a five-day assay (to assess repeatability), a normalized
      fluorescence signal average (i.e., time zero subtracted) of 5171 ± 821 RFU
      (a.u.) and 2725 ± 411 RFU (a.u.) were obtained for both NTC controls of N
      and ORF1ab targets, respectively, at time point 30 min. Similarly, a
      normalized fluorescence signal average of 5350 ± 234 RFU (a.u.) and 1754 ±
      418 RFU (a.u.) were observed for both NEC control of N and ORF1ab targets,
      respectively. However, these values may change depending on the instrument
      used and current setup.
    </p>
    <h3 id="sec7.4">Potential solution</h3>
    <p>
      First, it is recommended to repeat the assay in order to rule out cross
      contamination between samples or contamination in any reagent used in the
      CRISPR/Cas reaction. At the same time, it is recommended to run an
      electrophoresis to visualize any amplicon contamination in the NTC or NEC
      reaction. If amplicons are observed, contamination in reagents, equipment
      (i.e., pipettes or tips) or lab area used in the RT-PCR must be
      considered. Replace the working stock of reagents (i.e., primers, buffer,
      dNTP, or enzymes), and tips. Clean the pipettes and working areas with a
      commercial solution such as LookOut® DNA erase spray (Sigma-Aldrich) or
      1–2% hypochlorite solution. Cleaning procedures of equipment and lab areas
      must be considered as part of the working routine.
    </p>
    <h3 id="sec7.5">Problem 3</h3>
    <p>Low fluorescence signal in positive controls (Fluorescence_outcome)</p>
    <p>
      Low fluorescence signal in positive controls could be due to technical
      errors in the reaction preparation (i.e., miscalculation of reagents or
      enzymes concentration, improper pipetting dispense, etc.). Also, that the
      quality of the reagents or enzymes may be compromised. Low fluorescent
      signals could compromise the analysis of unknown samples, affecting the
      calculated ratios and the test results. This problem can arise from the
      RT-PCR step or the CRISPR/Cas-mediated detection. In a five-day assay (to
      assess repeatability), a normalized fluorescence signal average (i.e.,
      time zero subtracted) of 43029 ± 6598.40 RFU (a.u.) and 37837.8 ± 3583.51
      RFU (a.u.) were obtained for both SARS-CoV-2 N and ORF1ab targets,
      respectively, for a time readout of 30 min. However, the reported values
      may change depending on the instrument used and/or the reading setup.
    </p>
    <h3 id="sec7.6">Potential solution</h3>
    <p>
      To rule out a potential lack of RT-PCR amplification, check the reaction
      products by agarose gel electrophoresis. If RT-PCR products are absent see
      <a href="#sec7.1">problem 1</a>. If amplified products are observed, the
      problem can be solved by focusing on the CRISPR/Cas reaction. Check the
      reagents and enzyme calculation, equipment setup, and reagents and enzymes
      storage conditions. Replace the working reagents and/or enzymes stocks if
      necessary. If a different fluorophore is used in the double-labeled probe,
      equipment setup must be standardized before evaluating unknown clinical
      samples.
    </p>
    <h3 id="sec7.7">Problem 4</h3>
    <p>
      Weak fluorescence during direct visualization readout
      (Visualization_outcome)
    </p>
    <p>
      Naked-eyed detection has been validated using a blue-light
      transilluminator. High fluorescence signal was reported for high- (Cq &lt;
      25) and mid-viral load samples (Cq 25–31) (<a href="#fig6">Figure 6</a>).
      Using other transilluminators (e.g., UV-light transilluminator) showed
      weaker fluorescence signal even for high- and mid- viral load samples.
    </p>
    <h3 id="sec7.8">Potential solution</h3>
    <p>
      It is recommendable to test direct visual readout for control reactions
      before testing clinical unknown samples, if using other than a blue-light
      transilluminator. If available, top filter or dark box can be used to
      improve direct fluorescence visualization.
    </p>
    <h3 id="sec7.9">Problem 5</h3>
    <p>Invalid results (Interpretation_outcome)</p>
    <p>
      Invalid results are obtained when the fluorescence signal in the RNAseP
      target is low, producing fluorescence ratios &lt; 2. Low fluorescence
      ratios can be obtained if there is a high fluorescence signal in the NTC
      or a low signal in the reactions with the test samples.
    </p>
    <h3 id="sec7.10">Potential solution</h3>
    <p>
      Verify the presence of RT-PCR products to rule out an amplification
      problem by agarose gel electrophoresis. If RT-PCR products are absent see
      <a href="#sec7.1">problem 1</a>. If amplified products are observed, the
      problem can be solved by focusing on the CRISPR/Cas reaction. Repeat the
      CRISPR/Cas reaction to rule out any technical error during the preparation
      of the Cas12a master mix. Verify the storage conditions of the Cas12a and
      crRNA stocks to assure the reagents quality. If the repeated fluorescence
      ratio result is &lt; 2 or RT-PCR products are not adequately amplied, low
      sample quality must be considered, and a new sample must be requested.
    </p>
  </section>
  <section>
    <h2 id="references">References</h2>
    <p id="bib1">
      Alcántara, R., Peñaranda, K., Mendoza, G., Nakamoto, J.A., Martins-Luna,
      J., Valle, J.D., Adaui, V., and Milón, P. (2021). Unlocking SARS-CoV-2
      detection in low- and middle-income. Cell Reports Methods. .
      <a href="https://doi.org/10.1016/j.crmeth.2021.100093" class="external-link">https://doi.org/10.1016/j.crmeth.2021.100093</a>
    </p>
    <p id="bib2">
      Álvarez-Fernández, R. (2013). Chapter one explanatory chapter PCR primer
      design. Methods Enzymol. <i>529</i>, 1-21.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref2" class="external-link">View at publisher</a>
    </p>
    <p id="bib3">
      Bonini, A., Poma, N., Vivaldi, F., Kirchhain, A., Salvo, P., Bottai, D.,
      Tavanti, A., and Francesco, F.D. (2020). Advances in biosensing: the
      CRISPR/Cas system as a new powerful tool for the detection of nucleic
      acids. J. Pharm. Biomed. <i>192</i>, 113645.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref3" class="external-link">View at publisher</a>
    </p>
    <p id="bib4">
      Broughton, J.P., Deng, X., Yu, G., Fasching, C.L., Servellita, V., Singh,
      J., Miao, X., Streithorst, J.A., Granados, A., Sotomayor-Gonzalez, A.,
      et al. (2020). CRISPR–Cas12-based detection of SARS-CoV-2. Nat.
      Biotechnol. <i>38</i>, 870-874.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref4" class="external-link">View at publisher</a>
    </p>
    <p id="bib5">
      Buchan, B.W., Hoff, J.S., Gmehlin, C.G., Perez, A., Faron, M.L.,
      Munoz-Price, L.S., and Ledeboer, N.A. (2020). Distribution of SARS-CoV-2
      PCR cycle threshold values provide practical insight into overall and
      target-specific sensitivity among symptomatic patients. Am. J. Clin.
      Pathol. <i>154</i>, aqaa133.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref5" class="external-link">View at publisher</a>
    </p>
    <p id="bib6">
      Chaubal, G., Sarkale, P., Kore, P., and Yadav, P. (2018). Development of
      single step RT-PCR for detection of Kyasanur forest disease virus from
      clinical samples. Heliyon <i>4</i>, e00549.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref6" class="external-link">View at publisher</a>
    </p>
    <p id="bib7">
      Chen, J.S., Ma, E., Harrington, L.B., Costa, M.D., Tian, X., Palefsky,
      J.M., and Doudna, J.A. (2018). CRISPR-Cas12a target binding unleashes
      indiscriminate single-stranded DNase activity. Science <i>360</i>,
      eaar6245.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref7" class="external-link">View at publisher</a>
    </p>
    <p id="bib8">
      Davidi, D., Fitzgerald, S., Glaspell, H.L., Jalbert, S., Klapperich, C.M.,
      Landaverde, L., Maheras, S., Mattoon, S.E., Britto, V.M., Nguyen, G.T.,
      et al. (2021). Amplicon residues in research laboratories masquerade as
      COVID-19 in surveillance tests. Cell Rep. Methods <i>1</i>, 100005.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref8" class="external-link">View at publisher</a>
    </p>
    <p id="bib9">
      Graham, T.G.W., Dugast-Darzacq, C., Dailey, G.M., Nguyenla, X.H., Dis,
      E.V., Esbin, M.N., Abidi, A., Stanley, S.A., Darzacq, X., and Tjian, R.
      (2021). Open-source RNA extraction and RT-qPCR methods for SARS-CoV-2
      detection. PLoS One <i>16</i>, e0246647.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref9" class="external-link">View at publisher</a>
    </p>
    <p id="bib10">
      Green, M.R. and Sambrook, J. (2019). How to win the battle with RNase.
      Cold Spring Harb. Protoc. <i>2019</i>. .
      <a href="https://doi.org/10.1101/pdb.top101857" class="external-link">https://doi.org/10.1101/pdb.top101857</a>
    </p>
    <p id="bib11">
      Hanley, J.A. and McNeil, B.J. (1982). The meaning and use of the area
      under a receiver operating characteristic (ROC) curve. Radiology
      <i>143</i>, 29-36.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref11" class="external-link">View at publisher</a>
    </p>
    <p id="bib12">
      Hou, T., Zeng, W., Yang, M., Chen, W., Ren, L., Ai, J., Wu, J., Liao, Y.,
      Gou, X., Li, Y., et al. (2020). Development and evaluation of a rapid
      CRISPR-based diagnostic for COVID-19. PLoS Pathog. <i>16</i>, e1008705.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref12" class="external-link">View at publisher</a>
    </p>
    <p id="bib13">
      Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J.A., and
      Charpentier, E. (2012). A programmable dual-RNA–guided DNA endonuclease in
      adaptive bacterial immunity. Science <i>337</i>, 816-821.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref13" class="external-link">View at publisher</a>
    </p>
    <p id="bib14">
      Kumar, P., Malik, Y.S., Ganesh, B., Rahangdale, S., Saurabh, S., Natesan,
      S., Srivastava, A., Sharun, K., Yatoo, M.I., Tiwari, R., et al. (2020).
      CRISPR-cas system: an approach with potentials for COVID-19 diagnosis and
      therapeutics. Front. Cell. Infect. Microbiol. <i>10</i>, 576875.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref14" class="external-link">View at publisher</a>
    </p>
    <p id="bib15">
      Li, L., Li, S., Wu, N., Wu, J., Wang, G., Zhao, G., and Wang, J. (2019).
      HOLMESv2: a CRISPR-cas12b-assisted platform for nucleic acid detection and
      DNA methylation quantitation. ACS Synth. Biol. <i>8</i>, 2228-2237.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref15" class="external-link">View at publisher</a>
    </p>
    <p id="bib16">
      Mahony, J.B., Petrich, A., and Smieja, M. (2011). Molecular diagnosis of
      respiratory virus infections. Crit. Rev. Clin. Lab. Sci. <i>48</i>,
      217-249.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref16" class="external-link">View at publisher</a>
    </p>
    <p id="bib17">
      Mendoza-Rojas, G., Sarabia-Vega, V., Sanchez-Castro, A., Tello, L.,
      Cabrera-Sosa, L., Nakamoto, J.A., Peñaranda, K., Adaui, V., Alcántara, R.,
      Milón, P., et al. (2021). A low-cost and open-source protocol for
      expression of key molecular biology enzymes. Star Protoc. .
      <a href="https://doi.org/10.1016/j.xpro.2021.100899" class="external-link">https://doi.org/10.1016/j.xpro.2021.100899</a>
    </p>
    <p id="bib21">
      Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to
      ImageJ: 25 years of image analysis. Nature Methods <i>9</i>, 671-675.
      <a href="https://doi.org/10.1038/nmeth.2089" class="external-link">https://doi.org/10.1038/nmeth.2089</a>
    </p>
    <p id="bib18">
      Swarts, D.C., van der Oost, J., and Jinek, M. (2017). Structural basis for
      guide RNA processing and seed-dependent DNA targeting by CRISPR-cas12a.
      Mol. Cell <i>66</i>, 221-233.e4.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref18" class="external-link">View at publisher</a>
    </p>
    <p id="bib20">
      World Health Organization. (2018). Dos and Dont’s for molecular testing.
      Available at
      <a href="https://www.who.int/teams/global-malaria-programme/case-management/diagnosis/nucleic-acid-amplification-based-diagnostics/dos-and-don-ts-for-molecular-testing" class="external-link">https://www.who.int/teams/global-malaria-programme/case-management/diagnosis/nucleic-acid-amplification-based-diagnostics/dos-and-don-ts-for-molecular-testing</a>
      (Accessed: 19 Sep 2020).
    </p>
    <p id="bib19">
      Zetsche, B., Gootenberg, J.S., Abudayyeh, O.O., Slaymaker, I.M., Makarova,
      K.S., Essletzbichler, P., Volz, S.E., Joung, J., van der Oost, J., Regev,
      A., et al. (2015). Cpf1 is a single RNA-guided endonuclease of a class 2
      CRISPR-cas system. Cell <i>163</i>, 759-771.
      <a href="http://refhub.elsevier.com/S2666-1667(21)00584-0/sref19" class="external-link">View at publisher</a>
    </p>
  </section>
  <section>
    <h2 id="article-info">Article info</h2>
    <h3>Resource availability</h3>
    <h4>Lead contact</h4>
    <p>
      Further information and requests for resources and reagents should be
      directed to and will be fulfilled by the Lead Contact, Pohl Milón (<a href="mailto:pmilon@upc.pe">pmilon@upc.pe</a>).
    </p>
    <h4>Materials availability</h4>
    <p>
      All unique reagents developed in this work can be purchased at any gene
      synthesis company (i.e., primers and crRNA templates).
    </p>
    <h4>Data and code availability</h4>
    <p>
      Supplemental data files have been deposited to Mendeley Data:
      <a href="https://doi.org/10.17632/8m8z37v8xz.1">https://doi.org/10.17632/8m8z37v8xz.1</a>.
    </p>
    <h3>Supplemental information</h3>
    <p>
      <a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Mmc1.xlsx">Data S1. Reaction recipe template, related to CRISPR/Cas12a-mediated
        detection section</a>
    </p>
    <p>
      <a id="mmc2" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1145-Mmc2.xlsx">Data S2. Fluorescence ratio calculation template, related to
        quantification and statistical analysis section</a>
    </p>
    <h3>Acknowledgments</h3>
    <p>
      We are very thankful to Dr. Marcos Milla for donating equipment that was
      used in this study and others. We would also like to thank all laboratory
      members of the Adaui and Milón groups for their help, support, and great
      working atmosphere. This work was supported by grants from the
      <a href="https://doi.org/10.13039/501100010751">Fondo Nacional de Desarrollo Científico, Tecnológico y de Innovación
        Tecnológica</a>
      [070-2020-FONDECYT] to V.A. and [036-2019-FONDECYT-BM-INC.INV] to P.M.
    </p>
    <h3>Author contributions</h3>
    <p>
      Conceptualization and methodology, R.A., K.P., V.A., and P.M.;
      investigation, R.A., K.P., J.A.N., G.M.R., E.D., and D.A.; formal
      analysis, R.A., K.P., J.A.N., and P.M., writing – original draft,
      supervision, and visualization, R.A., K.P., and P.M., writing – review
      &amp; editing, R.A., K.P., J.A.N., G.M.R., E.D., D.A., V.A., and P.M.;
      project administration, P.M.
    </p>
    <h3>Declaration of interests</h3>
    <p>The authors declare no competing interests.</p>
  </section>
</article>