<article>
<section>
<h1 id="header">Functional proteomics protocol for the identification of interaction partners in <i>Tetrahymena thermophila</i>
</h1>
<p><time datetime="2021-03-05">Published: March 5, 2021</time></p>
<p>Syed Nabeel-Shah,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff6">6</a>,<a href="#fn1">7</a>,<a href="#cor1">*</a></sup> Jyoti Garg,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a></sup> Pata-Eting Kougnassoukou Tchara,<sup><a href="#aff4">4</a>,<a href="#aff5">5</a></sup> Ronald E. Pearlman,<sup><a href="#aff3">3</a></sup> Jean-Philippe Lambert,<sup><a href="#aff4">4</a>,<a href="#aff5">5</a></sup> and Jeffrey Fillingham<sup><a href="#aff1">1</a>,<a href="#fn2">8</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Chemistry and Biology, Ryerson University, 350 Victoria Street, Toronto, ON M5B 2K3, Canada</p>
<p id="aff2"><sup>2</sup>Donnelly Centre, University of Toronto, Toronto, ON, M5S 3E1, Canada</p>
<p id="aff3"><sup>3</sup>Department of Biology, York University, 4700 Keele Street, Toronto, ON, M3J 1P3, Canada</p>
<p id="aff4"><sup>4</sup>Department of Molecular Medicine, Cancer Research Center, Big Data Research Center, Université Laval, Quebec City, QC, Canada</p>
<p id="aff5"><sup>5</sup>CHU de Québec Research Center, CHUL, 2705 Laurier Boulevard, Quebec City, QC, G1V 4G2, Canada</p>
<p id="aff6"><sup>6</sup>Department of Molecular Genetics, University of Toronto, Toronto, ON, M5S 1A8, Canada</p>
<p id="fn1"><sup>7</sup>Technical contact</p>
<p id="fn2"><sup>8</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:nabeel.haidershah@mail.utoronto.ca">nabeel.haidershah@mail.utoronto.ca</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:jeffrey.fillingham@ryerson.ca">jeffrey.fillingham@ryerson.ca</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100362">10.1016/j.xpro.2021.100362</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We describe an optimized protocol for one-step affinity purification of FZZ-tagged proteins followed by mass spectrometry analysis for the identification of protein-protein interactions in the ciliate protozoan <i>Tetrahymena thermophila</i>. The FZZ epitope tag contains 2 protein A moieties (ZZ) and a 3xFLAG separated by a TEV cleavage site, which can also be employed in tandem affinity purification. This protocol is versatile and is suitable to use for other common epitope tags and can be adapted for other ciliates.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib7">Garg et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>
<i>Tetrahymena</i> FZZ-tagged proteins are affinity purified with M2 agarose or magnetic beads</li>
<li>Purified affinity material is analyzed by LC-MS/MS to identify interaction partners</li>
<li>
<i>SAINTexpress</i> discriminates between true interaction partners and false positives</li>
<li>This protocol has been validated for many nuclear proteins and has broad applications</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/502-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Using this protocol, we have performed affinity purification followed by mass spectrometry analysis (AP-MS) to study FZZ-tagged nuclear proteins (<a href="#bib17">Saettone et al., 2019</a>; <a href="#bib22">Wahab et al., 2020</a>) including histones (<a href="#bib2">Ashraf et al., 2019</a>; <a href="#bib12">Nabeel-Shah et al., 2020a</a>), histone chaperones, (<a href="#bib6">Garg et al., 2013</a>; <a href="#bib13">Nabeel-Shah et al., 2020b</a>; <a href="#bib16">Saettone et al., 2018</a>), and transcriptional regulatory Mediator-complex in <i>Tetrahymena</i> (<a href="#bib7">Garg et al., 2019</a>). This protocol can also be used for identifying protein-protein interactions for cytoplasmic proteins.</p>
<p><i>Tetrahymena</i> is nuclear dimorphic with a large polyploid somatic nucleus (<b>MAC</b>ronucleus) and a smaller diploid germinal nucleus (<b>MIC</b>ronucleus) in a common cytoplasm. This protocol is optimized for <i>Tetrahymena</i> cells expressing C-terminal FZZ-tagged proteins (<a href="#bib5">Couvillion and Collins, 2012</a>) from their endogenous MAC loci. To engineer endogenously tagged <i>Tetrahymena</i> cell lines (<a href="#fig1">Figure 1</a>A), refer to biolistic transformation protocol as detailed in (<a href="#bib3">Bruns and Cassidy-Hanley, 2000</a>). Alternatively, validated antibodies against the native proteins or other epitope tags of interest such as GFP and HA, can be utilized. The optimum time of incubation for affinity purification may vary depending upon the antibody used.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/502-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overview of homologous recombination mediated generation of endogenously FZZ tagged <i>Tetrahymena thermophila</i>
</div>
<p>(A) Schematic of the plasmid employed for the endogenous tagging of genes at their 3′ UTR with an FZZ tag. Two separate ~1 kb DNA fragments up and downstream of the predicted stop codon of the gene of interest are cloned into the FZZ tagging vector (pBKS-FZZ; kind gift of Dr. K. Collins, University of California, Berkeley). One micrometer gold particles (60 mg/mL; Bio-Rad) are coated with at least 5 μg of the transforming plasmid DNA and particles are introduced into the <i>Tetrahymena</i> MAC using biolistic transformation with a PDS-1000/He Biolistic particle delivery system (Bio-Rad). The FZZ epitope tag is inserted into desired MAC locus via cellular homologous recombination machinery. The transformants are selected using paromomycin (60 μg/mL). MAC homozygosity is achieved by growing the cells in increasing concentrations of paromomycin to a final concentration of 1 mg/mL.</p>
<p>(B) Western blotting analysis indicating the recovery of affinity purified (AP) FZZ-tagged proteins. Input, 1% of the initial volume (step 13), and affinity purified material, 1% of the final eluate (step 31), was loaded on a 10% SDS-PAGE gel and transferred to a nitrocellulose membrane. The membrane was probed with anti-FLAG antibody (1:5,000 dilution) to examine the recovery of the bait. No signal was detected in the untagged control cells. Anti-actin (1:10,000) was used as a loading control. Goat anti-mouse IgG H&amp;L secondary antibody was used at 1:5,000 dilution. Ordinate is molecular weight.</p>
</figcaption>
</figure>
<p>Prior to the experiment, prepare the lysis buffer and store at 4°C. Protease inhibitors should be added immediately before the cell lysis.</p>
<p>All steps are performed on ice or in a cold room or refrigerated chamber at 4°C unless stated otherwise.</p>
<h3 id="sec1.1">Prepare cell pellets</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol>
<li>In parallel, grow <i>Tetrahymena</i> cells expressing the FZZ epitope-tagged gene of interest and the untagged control cells in 500 mL of 1× SPP media, overnight (~12–14 h) at 30°C with gentle shaking (~200 rpm).</li>
<li>Next day, pellet the cells in 50 mL Falcon tubes. Harvest cells at mid-log phase growth at a final concentration of ~3  ×  10<sup>5</sup> cells/mL.</li>
<li>Centrifuge the cells at 1,000 × <i>g</i> for 1 min. Immediately remove the supernatant by aspiration and resuspend in 50 mL of 10 mM Tris-HCl pH 7.4.</li>
<li>Centrifuge again and immediately remove the supernatant.</li>
<li>Snap-freeze the pelleted cells in liquid nitrogen and store at −80°C until the day of the experiment.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Culture vessels must be kept clean and sterile. The use of dedicated flasks for <i>Tetrahymena</i> culture is strongly recommended to avoid microbiological contamination.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not vortex the intact cells during resuspension to avoid damaging them.</div>
<div class="note">
<span class="note-title">Note:</span> <i>Tetrahymena</i> can grow vegetatively or on mixing equal number of cells of different mating type, paired cells undergo conjugation involving meiosis, genetic exchange, nuclear development, genome reorganization, and irreversible genome silencing. To identify conjugation-specific protein-protein interactions, <i>Tetrahymena</i> cells of different mating types of inbred line B (e.g., CU428 [Mpr/Mpr (VII, mp-s)] and B2086 [Mpr+/Mpr+ (II, mp-s)]), expressing the protein of interest, are grown to mid-log phase and starved in 10 mM Tris-HCl pH 7.4 for 16 to 24 h. Mix equal number of starved cells to initiate conjugation. Observe under a dissecting microscope at 10× magnification to confirm the onset of conjugation. There must be 90% paired cells 3 h after mixing the cells to be used in an experiment. Pellet the cells at the desired time point.</div>
<div class="note">
<span class="note-title">Note:</span> For conjugation cell pellets: expression from MAC occurs only until ~8 h post-mixing; for rare cases when the protein of interest is expressed only later during conjugation it may be necessary to epitope tag the MIC locus.</div>
<div class="note">
<span class="note-title">Note:</span> Wild-type <i>Tetrahymena</i> cells without an epitope tag, referred to as an untagged control, are used as negative control.</div>
<div class="note">
<span class="note-title">Note:</span> Cell pellets should be prepared for at least two biological replicates per condition tested to permit statistical analysis of the results. Control samples should be prepared and processed in parallel to experimental samples to facilitate the identification of batch effects.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse ANTI-FLAG</td>
<td>Sigma</td>
<td>F1804</td>
</tr>
<tr>
<td>Packed M2-agarose</td>
<td>Sigma</td>
<td>A2220</td>
</tr>
<tr>
<td>ANTI-Actin</td>
<td>GenScript</td>
<td>RRID: AB_914102</td>
</tr>
<tr>
<td>Goat anti-mouse IgG H&amp;L (Alexa Fluor 488)</td>
<td>Abcam</td>
<td>ab150113</td>
</tr>
<tr>
<td>ANTI-FLAG M2 magnetic beads (optional: alternative to M2 agarose)</td>
<td>Sigma</td>
<td>M8823</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Protease inhibitor cocktail for yeast</td>
<td>Sigma</td>
<td>P8215</td>
</tr>
<tr>
<td>NP-40</td>
<td>Thermo Scientific</td>
<td>85124</td>
</tr>
<tr>
<td>Phenylmethylsulfonyl fluoride (PMSF)</td>
<td>Thermo Scientific</td>
<td>36978</td>
</tr>
<tr>
<td>TRIZMA base</td>
<td>Sigma</td>
<td>T1503</td>
</tr>
<tr>
<td>Benzonase</td>
<td>Sigma</td>
<td>E8263</td>
</tr>
<tr>
<td>Ammonium hydroxide (NH<sub>4</sub>OH) (14.5 M)</td>
<td>Carolina Biological Supply Co</td>
<td>844011</td>
</tr>
<tr>
<td>Magnesium chloride</td>
<td>Invitrogen</td>
<td>AM9530G</td>
</tr>
<tr>
<td>Calcium chloride</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>2× Laemmli buffer</td>
<td>Sigma</td>
<td>S3401-10VL</td>
</tr>
<tr>
<td>Proteomics grade, dimethylated trypsin</td>
<td>Sigma</td>
<td>T6567</td>
</tr>
<tr>
<td>Fe-EDTA</td>
<td>Sigma</td>
<td>E6760</td>
</tr>
<tr>
<td>Proteose peptone</td>
<td>Sigma</td>
<td>82450</td>
</tr>
<tr>
<td>Glucose</td>
<td>Sigma</td>
<td>47829</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>Sigma</td>
<td>Y1625</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>T. thermophila</i>: B2086 [Mpr+/Mpr+ (II mp-s)]</td>
<td>
<i>Tetrahymena</i> Stock Center of Cornell University, Ithaca NY<a href="http://tetrahymena.vet.cornell.edu/">http://tetrahymena.vet.cornell.edu/</a>
</td>
<td>n/a</td>
</tr>
<tr>
<td>
<i>T. thermophila</i>: CU428 [Mpr/Mpr (VII mp-s )]</td>
<td>
<i>Tetrahymena</i> Stock Center of Cornell University, Ithaca NY<a href="http://tetrahymena.vet.cornell.edu/">http://tetrahymena.vet.cornell.edu/</a>
</td>
<td>n/a</td>
</tr>
<tr>
<td>
<i>T. thermophila</i>: Med31-FZZ: B2086 [Mpr+/Mpr+ (mp-s, mt II)]</td>
<td><a href="#bib7">Garg et al. 2019</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Refrigerated microcentrifuge</td>
<td>Eppendorf</td>
<td>5417r</td>
</tr>
<tr>
<td>Magnetic separation rack (optional: when using magnetic beads)</td>
<td>Thermo Scientific</td>
<td>12321D</td>
</tr>
<tr>
<td>Benchtop orbital shaker model MaxQ 4000</td>
<td>Thermo Scientific</td>
<td>SHKE4000</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Standard solutions</h3>
<p>Prepare 1 M Tris-HCl pH 8.0. Use concentrated hydrochloric acid (HCl) to adjust the pH.</p>
<p>Prepare 10% NP-40 solution by diluting the stock in sterile water.</p>
<p>Prepare 5 M NaCl solution using sterile water.</p>
<div class="note">
<span class="note-title">Note:</span> HCl causes serious burns. To protect against HCl vapors, handle under chemical hood.</div>
<h3 id="sec2.2">2× lysis buffer</h3>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>600 mM</td>
<td>6 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 8.0</td>
<td>40 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>1 mM</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>42 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Add protease inhibitor cocktail and 100 mM PMSF immediately prior to cell lysis. Dissolve PMSF in isopropanol at 17.4 mg/mL. Dilute to a final volume of 10 mL with isopropanol. Aliquots should be stored at −20°C.</div>
<div class="note">
<span class="note-title">Note:</span> PMSF can cause chemical burns of skin and eyes. Caution should be exercised. Prepare this reagent in a chemical hood.</div>
<p><b>Storage:</b> Store lysis buffer at 4°C.</p>
<h3 id="sec2.3">IPP300 wash buffer</h3>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>300 mM</td>
<td>3 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 8.0</td>
<td>10 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>10% NP-40</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>46 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p><b>Storage:</b> Store IPP300 buffer at 4°C.</p>
<h3 id="sec2.4">IPP100 wash buffer</h3>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>100 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 8.0</td>
<td>10 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>10% NP-40</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>48 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p><b>Storage:</b> Store IPP100 buffer at 4°C.</p>
<h3 id="sec2.5">2 mM CaCl<sub>2</sub>/20 mM Tris solution</h3>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M CaCl<sub>2</sub>
</td>
<td>2 mM</td>
<td>0.01 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 8.0</td>
<td>20 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>48.9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p><b>Storage:</b> Store CaCl<sub>2</sub>/Tris buffer at 4°C.</p>
<h3 id="sec2.6">0.5 M NH<sub>4</sub>OH</h3>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>14.5 M NH<sub>4</sub>OH</td>
<td>0.5 M</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>28 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>29 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> NH<sub>4</sub>OH is corrosive with strong ammonia odor. Handle in a fume hood.</div>
<h3 id="sec2.7">1× SPP media for <i>Tetrahymena</i> growth</h3>
<p>2% proteose peptone</p>
<p>0.1% yeast extract</p>
<p>0.2% glucose</p>
<p>0.003% sequestrene (Fe-EDTA), can be replaced with 33 μM FeCl<sub>3</sub></p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Thawing the cell pellets</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5 h</div>
<ol>
<li>Thaw the cell pellets on ice. Have a small bucket of water at room temperature (~22°C) and swirl the cell pellet in it every few minutes to assist the thawing process. Expect 20 to 30 min to completely thaw the cell pellets.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Be careful toward the end of thawing the pellets. There are usually some frozen bits left in the middle of the thawed cells. Thaw them completely while ensuring the cell pellets remain cold.</div>
<h3 id="sec3.2">Cell lysis: soluble cell lysate preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5 h</div>
<p>Thawed cells are lysed, and the cell lysate is treated with a nuclease and clarified.</p>
<ol start="2">
<li>To 25 mL of 2× lysis buffer, add 500 μL protease inhibitors for yeast (Sigma) + 200 μL of 100 mM PMSF (stock prepared in isopropanol).</li>
<li>Estimate the cell pellet volume by using a 10 mL pipette. For 500 mL of cell culture, the pellet size should be between 2–3 mL.</li>
<li>Add an equal volume of ice-cold 2× lysis buffer supplemented with protease inhibitors. Mix by gently pipetting up and down. The solution should become viscous due to lysed nuclei and release of chromatin.</li>
<li>Adjust the total volume to 15 mL using 1× lysis buffer (dilute the 2× lysis buffer to 1× lysis buffer using ice-cold sterile ddH<sub>2</sub>O).</li>
<li>Add 10% NP-40 to a final concentration of 0.2% (e.g., 300 μL to 15 mL total lysis solution).</li>
<li>Keep the lysate on ice for 10 min and check for viscosity using a wide-bore P1000 pipette tip (or cut a P1000 tip-head with a sterile razor blade before use). The solution should be viscous at this point.</li>
<li>Add 5 μL of Benzonase (Sigma) to digest the released chromatin, genomic DNA, and RNA resulting in an enhanced solubility of chromatin-associated protein complexes.</li>
<li>Rotate in the cold room using a Nutator (or any other suitable rotator) for 30 min.</li>
<li>Divide the cell lysate into 10 pre-chilled 1.5 mL Eppendorf tubes (1.5 mL lysate in each tube).</li>
<li>Centrifuge the lysate at 16,100 × <i>g</i> for 30 min at 4°C.</li>
<li>Pool the clarified cell lysates by transferring the supernatant to 15 mL Falcon tubes (pre-chilled on ice). Pool the cell lysate by pouring the lysate from each 1.5 mL Eppendorf tube into a single 15 mL Falcon tube.</li>
<li>Save 50 μL of the cell lysate in a separate Eppendorf tube as input. Flash freeze in liquid nitrogen immediately, and store at −80°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Many protease inhibitor cocktails include EDTA. Concentrations of higher than 1 mM EDTA will inhibit Benzonase activity.</div>
<div class="note">
<span class="note-title">Note:</span> Viscosity should be eliminated after Benzonase treatment. If samples remain difficult to manipulate, add additional Benzonase or incubate for a longer time to ensure that viscosity is eliminated.</div>
<div class="note">
<span class="note-title">Note:</span> Centrifugation of the cell lysate can also be performed in a Sorvall-type SA600 rotor @ 12,000 rpm @ 4ᵒC for 30 min followed by transfer to a fresh 15 mL pre-chilled Falcon tube.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The lysate can be flash-frozen after step 11 and can be kept at −80°C until further use. To preserve protein-protein interactions, add glycerol to 20% final concentration to the lysis buffer.</div>
<h3 id="sec3.3">Affinity purification using M2 agarose beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 h</div>
<p>Affinity purification using M2 agarose beads (i.e., anti-FLAG monoclonal antibody covalently attached to agarose by hydrazide linkage) is carried out to isolate the FZZ-tagged protein (bait) and its interaction partners (prey). Take out the agarose beads and slowly warm them on ice for 10 min. Gently rotate in your hand to mix the bead slurry.</p>
<ol start="14">
<li>Transfer 100 μL of the of anti-FLAG M2 agarose slurry (50 μL of packed beads) to a pre-chilled 15 mL Falcon tube per purification.</li>
<li>Wash with 10 mL ice-cold 1× lysis buffer by gently resuspending the beads.</li>
<li>Centrifuge at 960 × <i>g</i> for 2 min at 4°C. Aspirate the supernatant.</li>
<li>Repeat the washing step 3 times.</li>
<li>Resuspend the beads in 50 μL of 1× lysis buffer (final volume will be 100 μL).</li>
<li>Transfer the pooled supernatant from step 11 to the 15 mL Falcon tube containing M2 agarose beads.</li>
<li>Incubate for 2 h at 4°C with end-to-end rotation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Alternatively, immunomagnetic beads can be used. Transfer 100 μL protein G Dynabeads per purification to a 1.5 mL Eppendorf tube. These beads are uniform 2.8 μm superparamagnetic beads with recombinant Protein G covalently coupled to the surface. Separate Dynabeads using a magnetic rack (~15–30 s). Remove the supernatant. Resuspend the beads in 1 mL 1× lysis buffer, put back on the magnetic rack, and remove the supernatant again. Repeat this step 3 times and finally resuspend the Dynabeads in 100 μL 1× lysis buffer. Add 5 μg of anti-FLAG antibody to 100 μL of protein G Dynabeads and rotate at room temperature (~22°C) for 1 h or overnight at 4°C. After incubation, magnetically separate the beads. Remove the supernatant and wash 3 times with 1 mL 1× lysis buffer. Resuspend in 100 μL 1× lysis buffer and transfer to the pooled supernatant from step 11. Incubate for 2–4 h at 4°C with end-to-end rotation.</div>
<p>We have tested this protocol using both anti-GFP and anti-FLAG antibodies. Validated antibodies against endogenous proteins can also be used with this protocol. For the FZZ or FLAG tagged proteins, ANTI-FLAG M2 Magnetic Beads can also be used. In this case, omit the antibody-Protein G beads conjugation step. Wash 100 μL M2 magnetic beads 3 times with 1× lysis buffer, resuspend in 100 μL 1× lysis buffer and add to the pooled supernatant from step 11. Incubate at 4°C with end-to-end rotation</p>
<div class="note">
<span class="note-title">Note:</span> In addition to using untagged cells, a negative control without antibodies should be used if the purifications are being carried out using a specific antibody against endogenous protein.</div>
<div class="note">
<span class="note-title">Note:</span> For agarose beads, always cut the end of the pipette tip using a sterile razor blade prior to pipetting the beads to minimize shearing of the beads. Alternatively, wide-bore tips can be used to handle agarose beads.</div>
<h3 id="sec3.4">Washing and elution of the affinity purified proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>Elution of FZZ-tagged proteins (or alternative epitope tags, e.g., GFP) and associated interactions partners can be performed either with Laemmli buffer or ammonium hydroxide (NH<sub>4</sub>OH). Prepare and pre-chill the following buffers: IPP300, IPP100 with NP-40, IPP100 without NP-40.</p>
<ol start="21">
<li>Wash the beads once with high-salt IPP300 to remove any weakly bound protein contaminants. Gently resuspend the beads in 15 mL IPP300, rotate 5 min in the cold room, centrifuge for 2 min at 960 × <i>g.</i>
</li>
<li>Wash the beads as above twice with 15 mL IPP100-NP-40 buffer.</li>
<li>Wash the beads as above twice with 15 mL IPP100 without NP-40. These washes remove the NP-40 detergent.</li>
<li>After the last wash, resuspend the M2-agarose beads in 750 μL of CaCl<sub>2</sub>/Tris solution and transfer to a 1.5 mL Eppendorf tube.</li>
<li>Centrifuge for 1 min at 2,700 × <i>g</i> at room temperature (~22°C).</li>
<li>Remove the supernatant.</li>
<li>Resuspend the beads in 500 μL of 0.5 M NH<sub>4</sub>OH prepared fresh from the concentrated stock solution.</li>
<li>Rotate for 20 min at room temperature (~22°C).</li>
<li>Centrifuge for 2 min at room temperature at 2,700 × <i>g.</i>
</li>
<li>Transfer the supernatant to a new pre-chilled 1.5 mL Eppendorf tube.</li>
<li>Transfer 50 μL to a separate pre-chilled 1.5 mL Eppendorf tube. This will be used later in Western blotting analysis to examine the recovery of the bait.</li>
<li>Flash freeze all samples in liquid nitrogen and store at −80°C until further use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After washing with CaCl<sub>2</sub>/Tris solution, the bead pellet often shrinks slightly. The CaCl<sub>2</sub>/Tris wash is performed to remove residual salt from the beads.</div>
<div class="note">
<span class="note-title">Note:</span> For magnetic beads, the CaCl<sub>2</sub>/Tris wash can be omitted.</div>
<div class="note">
<span class="note-title">Note:</span> For elution using Laemmli buffer, add 1× Laemmli buffer to the beads after the washing steps. Heat at 95°C using a heat block or boiling water for 5 min and remove the supernatant, which can be directly used for SDS-PAGE and associated Western blotting.</div>
<div class="critical">
<span class="critical-title">Critical:</span> NH<sub>4</sub>OH has a strong odor, is hazardous, and causes irritation of the skin. Handle NH<sub>4</sub>OH in a fume hood.</div>
<h3 id="sec3.5">Western blotting analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>Western blotting analysis is performed using the input (step 13) and affinity purified material (50 μL saved fraction from step 31) to confirm the recovery of bait proteins (<a href="#fig1">Figure 1</a>B).</p>
<ol start="33">
<li>Load input and affinity purified fractions in duplicates on a 10% SDS-PAGE gel.</li>
<li>Electrophorese the gel and transfer the proteins to a nitrocellulose or PVDF membrane according to manufacturer’s instructions or other established protocols (<a href="#bib11">Mahmood and Yang, 2012</a>).</li>
<li>Use anti-FLAG antibody (1:5,000 dilution) for immunoblot analysis. The expected outcome is shown in <a href="#fig1">Figure 1</a>B.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Silver staining can be performed to visualize the affinity purified FZZ-tagged protein (bait) and its interaction partners (prey) (e.g., <a href="#bib6">Garg et al., 2013</a>).</div>
<h3 id="sec3.6">Mass spectrometry and statistical analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> several days</div>
<p>For researchers without direct access to mass spectrometers, mass spectrometry services can be contracted from a facility available to the researcher, e.g., The Donnelly Mass Spectrometry Center (DMSC) (<a href="https://ccbr.utoronto.ca/donnelly-mass-spec-facility">https://ccbr.utoronto.ca/donnelly-mass-spec-facility</a>) or Network Biology Collaborative Centre (<a href="https://nbcc.lunenfeld.ca/">https://nbcc.lunenfeld.ca/</a>). If such a service is employed, details for submitting samples should be arranged ahead of time with the facility.</p>
<ol start="36">
<li>Speedvac the purified samples to dryness with no heat. For ~450 μL of purified material in 0.5 M NH<sub>4</sub>OH, expect 2–3 h of drying.</li>
<li>Resuspend the dried material in 100 μL of freshly prepared 50 mM ammonium bicarbonate pH 8.0.</li>
<li>Add 0.75 μg of proteomic grade dimethylated trypsin resuspended in 20 mM Tris-HCl pH 8.0 at a concentration of 0.1 μg/μL and incubate at 37⁰C for ~15 h with gentle lateral mixing.</li>
<li>Add an extra 0.25 μg of proteomic grade dimethylated trypsin and incubate for an additional ~3 h at 37°C with gentle lateral mixing.</li>
<li>Speedvac the purified samples to dryness with no heat. For ~110 μL of digested material, expect 1–2 h of drying.</li>
<li>Prior to mass spectrometry data acquisition, the samples are cleaned using C<sub>18</sub> Stage-Tip (<a href="#bib14">Rappsilber et al., 2007</a>) to remove salt and detergents, contaminants that interfere with the mass spectrometry data acquisition.</li>
<li>The dried desalted peptides are now ready for mass spectrometry analysis. They should be stored at −80°C until data acquisition.</li>
<li>The mass spectrometry data acquisition and analysis will vary based on the mass spectrometer to be employed. Recently, we have employed an Orbitrap Fusion (Thermo Fisher Scientific) equipped with a nanoelectrospray ion source coupled to a Dionex UltiMate 3000 nanoRSLC chromatography system (Thermo Fisher Scientific). We find that a 60 min LC linear gradient is appropriate for the complexity of the peptide samples under analysis generated by this protocol. The resulting raw mass spectrometry files are converted to .mzML using ProteoWizard (<a href="#bib1">Adusumilli and Mallick, 2017</a>) and searched with Mascot (v2.3.02; Matrix Science). The spectra are searched with the RefSeq database (version 45, January 24th, 2011) acquired from NCBI against a total of 24,770 <i>Tetrahymena thermophila</i> sequences.</li>
<li>To identify statistically significant interaction partners, <i>SAINTexpress</i> [v3.61; (<a href="#bib19">Teo et al., 2014</a>)] is used. <i>SAINTexpress</i> calculates the probability value of each potential protein-protein interaction being enriched over the estimated background distribution. Default parameters should be employed in this analysis with as many controls as possible. Typically, we aim to have at least as many control replicates as samples in our <i>SAINTexpress</i> analyses.</li>
<li>High-confidence interaction partners identified with <i>SAINTexpress</i> are those with a false discovery rate (FDR) of ≤0.01 (<a href="#fig2">Figures 2</a>A and 2B).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/502-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Overview of the optimized protocol for the identification of statistically significant protein-protein interaction in <i>Tetrahymena thermophila</i>
</div>
<p>(A) Schematic of the sample preparation and mass spectrometry acquisition and analysis.</p>
<p>(B) Overview of the <i>SAINTexpress</i> output file showing each prey protein identified along with their average spectral count (AvgSpec), the spectral count identified for the prey protein in each control sample (Ctrl Count) and the related statistical measure of the interaction (AvgP and FDR).</p>
<p>(C) Network representation of AP-MS identified interaction partners. Data are filtered using SAINTexpress against several control purifications. Interaction partners that pass the statistical threshold of FDR ≤0.01 are shown. Edge width is proportional to average spectral counts for each prey. Note: To generate this network, AP-MS data for MED31-FZZ was utilized, as reported in <a href="#bib7">Garg et al. (2019)</a>.</p>
</figcaption>
</figure>
</li>
<li>High-confidence interaction partners can be visualized as a protein-protein interaction network using Cytoscape (<a href="#bib4">Cline et al., 2007</a>) (<a href="#fig2">Figure 2</a>C) or as heat maps or dot plots using Prohits-viz (<a href="#bib8">Knight et al., 2017</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To minimize contaminants observed in the LC-MS/MS analysis, “LC-MS grade” or better-quality solvents should be employed in the preparation of all buffers to be used in the digestion and clean-up of purified material.</div>
<div class="note">
<span class="note-title">Note:</span> <i>SAINTexpress</i> analyses can be performed through the crapome portal (<a href="http://www.crapome.org">www.crapome.org</a>), using a standalone version available at <a href="http://saint-apms.sourceforge.net/Main.html">http://saint-apms.sourceforge.net/Main.html</a>, or through the ProHits LIMS software (<a href="#bib9">Liu et al., 2016</a>).</div>
<div class="note">
<span class="note-title">Note:</span> As an alternative to <i>SAINTexpress</i>, users can employ the non-parametric method ComPASS (<a href="#bib18">Sowa et al., 2009</a>), which uses a normal distribution model, or simpler two sample t-test, in the Perseus software for example (<a href="#bib20">Tyanova et al., 2016</a>), to filter false positives from their raw mass spectrometry data.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol is optimized for the validation of FZZ-tagged cell lines and their use for the isolation of FZZ-tagged proteins (bait) and their physical interaction partners (prey) in <i>Tetrahymena thermophila</i>. Typical Western blot results are shown in <a href="#fig1">Figure 1</a>B. Mass spectrometry analysis is used to identify the co-purifying interaction partners (<a href="#fig2">Figure 2</a>A). Application of <i>SAINTexpress</i> identifies high-confidence interaction partners that are significantly enriched over the background (<a href="#fig2">Figure 2</a>B). The identified interaction partners can be visualized as a network using Cytoscape (<a href="#fig2">Figure 2</a>C). The use of this protocol to identify protein-protein interactions has yielded useful insights into the function of numerous proteins in <i>Tetrahymena thermophila</i>, including MED31 (<a href="#bib7">Garg et al., 2019</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol utilizes endogenously tagged proteins as a bait in AP-MS experiments. We have not tested this protocol for epitope-tagged proteins that are being ectopically expressed in <i>Tetrahymena</i> but foresee no problem in doing so based on our functional proteomics experience. While this protocol relies on high-salt stringent washing steps to minimize the background, it is still possible that some non-specific proteins remain in the eluate. We therefore recommend the use of alternative methods such as reciprocal co-immunoprecipitation followed by Western blotting to validate the AP-MS detected protein-protein interactions of interest before embarking on their biological characterization.</p>
<p>Use of specific antibodies against endogenous proteins is also possible, however, it should be employed only after the validation of the antibody. In particular, testing the antibody specificity using knockout cell lines is recommended.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Bait protein levels are low in the final elute.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>This problem can be addressed by either scaling up the amount of starting material or increasing the amount of antibody employed.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The bait protein is absent in Western blots.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Ensure that cells are expressing the epitope-tagged version of the protein of interest. This can be confirmed using either indirect immunofluorescence or Western blotting analysis of whole cell extracts prior to performing AP-MS. If you are using a specific antibody against an endogenous protein, ensure that your antibody is suitable for immunoblot analysis. Applications for commercially available antibodies are typically available from the supplier.</p>
<p>Since this protocol uses endogenously tagged proteins as baits, it is possible that the protein of interest is weakly expressed. In such cases, scaling up the starting material might resolve the issue.</p>
<p>While a rare occurrence, the FZZ tag might result in inappropriate localization or improper folding for certain proteins. In such cases, altering orientation of the tag, e.g., from C terminus to N terminus, and/or use of an alternative epitope tag, such as GFP, should be explored to resolve the issue.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>There are too many background interactions detected after AP-MS.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>This may be addressed by performing a two-step affinity purification as detailed in <a href="#bib6">Garg et al. 2013</a>. It is worth nothing that tandem affinity purification will drastically reduce the number of protein-protein interactions detected. In our experience, modern mass spectrometers provide enough analysis depth to not require a two-step affinity purification for most bait proteins.</p>
<p>Another option is to increase the number of control purifications which might help in filtering the non-specific interactions. As there are as yet no curated databases of AP-MS contaminant proteins available for <i>Tetrahymena</i>, we recommend using control AP-MS data sets that we have published previously (<a href="#bib2">Ashraf et al., 2019</a>; <a href="#bib6">Garg et al., 2013</a>, <a href="#bib7">2019</a>; <a href="#bib12">Nabeel-Shah et al., 2020a</a>; <a href="#bib16">Saettone et al., 2018</a>) if users cannot generate more of their own controls. These data can be freely accessed from publicly available proteomic repositories such as MassIVE (<a href="http://massive.ucsd.edu">http://massive.ucsd.edu</a>). Their accession numbers are provided in our previous reports. Additionally, it is possible to virtually compress control samples in the <i>SAINTexpress</i> analysis to create a “worst-case scenario” in which only the most abundant identification in a set of controls is considered. We suggest to not compress controls by more than half of their total number while maintaining a number of controls equal to at least the biological replicate employed to ensure that the background contaminants are effectively modeled by <i>SAINTexpress</i>.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The bait was detected but no interaction partners were identified.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>We have encountered this problem for some classes of proteins including kinases, phosphatases, and methyltransferases which often do not possess interaction domains or surfaces. Based on our experience in human cells, we recommend using proximity-labeling based methods, such as BioID (<a href="#bib15">Roux et al., 2012</a>), which do not necessitate protein-protein interactions to be maintained throughout the biochemical purification. BioID has previously been implemented and published for <i>Tetrahymena thermophila</i> (<a href="#bib10">Louka et al., 2018</a>; <a href="#bib21">Urbanska et al., 2018</a>) and we have successfully used it in our studies (unpublished observations).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Biological replicates generate very different results after mass spectrometry acquisition.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>A lack of reproducibility may be caused by issues throughout the protocol. One possible explanation is the very different level of starting material. Care should be taken to harvest a consistent number of cells for each replicate. If a drastically different number of cells are to be employed, normalize the total protein per sample. Ideally, all samples would saturate the affinity resin during the incubation with lysates which helps minimize variation between replicates. Another possibility is that the purified material is too complex and saturates the mass spectrometer. A simple verification of this possibility is to define the number of MS/MS spectra acquired throughout the data acquisition. If the number of MS/MS spectra reach the maximum duty cycle of the method employed, the length of the LC gradient should be increased and/or a faster mass spectrometer should be employed.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Adusumilli, R. and Mallick, P. (2017). Data conversion with proteoWizard msConvert. Methods Mol. Biol. <i>1550</i>, 339-368. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref1">View at publisher</a></p>
<p id="bib2">Ashraf, K., Nabeel-Shah, S., Garg, J., Saettone, A., Derynck, J., Gingras, A.-C., Lambert, J.-P., Pearlman, R.E., and Fillingham, J. (2019). Proteomic analysis of histones H2A/H2B and variant Hv1 in <i>Tetrahymena thermophila</i> reveals an ancient network of chaperones. Mol. Biol. Evol. <i>36</i>, 1037-1055. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref2">View at publisher</a></p>
<p id="bib3">Bruns, P.J. and Cassidy-Hanley, D. (2000). Biolistic transformation of macro- and micronuclei. Methods Cell Biol. <i>62</i>, 501-512. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref3">View at publisher</a></p>
<p id="bib4">Cline, M.S., Smoot, M., Cerami, E., Kuchinsky, A., Landys, N., Workman, C., Christmas, R., Avila-Campilo, I., Creech, M., Gross, B., et al. (2007). Integration of biological networks and gene expression data using Cytoscape. Nat. Protoc. <i>2</i>, 2366-2382. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref4">View at publisher</a></p>
<p id="bib5">Couvillion, M.T. and Collins, K. (2012). Biochemical approaches including the design and use of strains expressing epitope-tagged proteins. Methods Cell Biol. <i>109</i>, 347-355. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref5">View at publisher</a></p>
<p id="bib6">Garg, J., Lambert, J.P., Karsou, A., Marquez, S., Nabeel-Shah, S., Bertucci, V., Retnasothie, D.V., Radovani, E., Pawson, T., Gingras, A.C., et al. (2013). Conserved Asf1-importinβ physical interaction in growth and sexual development in the ciliate <i>Tetrahymena thermophila</i>. J. Proteomics <i>94</i>, 311-326. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref6">View at publisher</a></p>
<p id="bib7">Garg, J., Saettone, A., Nabeel-Shah, S., Cadorin, M., Ponce, M., Marquez, S., Pu, S., Greenblatt, J., Lambert, J.-P., Pearlman, R.E., et al. (2019). The Med31 conserved component of the divergent mediator complex in tetrahymena thermophila participates in developmental regulation. Curr. Biol. <i>29</i>, 2371-2379.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref7">View at publisher</a></p>
<p id="bib8">Knight, J.D.R., Choi, H., Gupta, G.D., Pelletier, L., Raught, B., Nesvizhskii, A.I., and Gingras, A.C. (2017). ProHits-viz: A suite of web tools for visualizing interaction proteomics data. Nat. Methods <i>14</i>, 645-646. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref8">View at publisher</a></p>
<p id="bib9">Liu, G., Knight, J.D.R., Zhang, J.P., Tsou, C.-C., Wang, J., Lambert, J.-P., Larsen, B., Tyers, M., Raught, B., Bandeira, N., et al. (2016). Data independent acquisition analysis in ProHits 4.0. J. Proteomics <i>149</i>, 64-68. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref9">View at publisher</a></p>
<p id="bib10">Louka, P., Vasudevan, K.K., Guha, M., Joachimiak, E., Wloga, D., Tomasi, R.F.X., Baroud, C.N., Dupuis-Williams, P., Galati, D.F., Pearson, C.G., et al. (2018). Proteins that control the geometry of microtubules at the ends of cilia. J. Cell Biol. <i>217</i>, 4298-4313. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref10">View at publisher</a></p>
<p id="bib11">Mahmood, T. and Yang, P.C. (2012). blot: technique, theory, and trouble shooting. N. Am. J. Med. Sci. <i>4</i>, 429-434. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref11">View at publisher</a></p>
<p id="bib12">Nabeel-Shah, S., Ashraf, K., Saettone, A., Garg, J., Derynck, J., Lambert, J.P., Pearlman, R.E., and Fillingham, J. (2020a). Nucleus-specific linker histones Hho1 and Mlh1 form distinct protein interactions during growth, starvation and development in <i>Tetrahymena thermophila</i>. Sci. Rep. <i>10</i>, 1-14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref12">View at publisher</a></p>
<p id="bib13">Nabeel-Shah, S., Garg, J., Saettone, A., Ashraf, K., Lee, H., Wahab, S., Ahmed, N., Fine, J., Derynck, J., Ponce, M., et al. (2020b). Functional characterization of RebL1 highlights the evolutionary conservation of oncogenic activities of the RBBP4/7 orthologue in <i>Tetrahymena thermophila</i>. BioRxiv. <a class="external-link" href="https://doi.org/10.1101/2020.11.07.372946">https://doi.org/10.1101/2020.11.07.372946</a></p>
<p id="bib14">Rappsilber, J., Mann, M., and Ishihama, Y. (2007). Protocol for micro-purification, enrichment, pre-fractionation and storage of peptides for proteomics using StageTips. Nat. Protoc. <i>2</i>, 1896-1906. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref14">View at publisher</a></p>
<p id="bib15">Roux, K.J., Kim, D.I., Raida, M., and Burke, B. (2012). A promiscuous biotin ligase fusion protein identifies proximal and interacting proteins in mammalian cells. J. Cell Biol. <i>196</i>, 801-810. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref15">View at publisher</a></p>
<p id="bib16">Saettone, A., Garg, J., Lambert, J.-P., Nabeel-Shah, S., Ponce, M., Burtch, A., Thuppu Mudalige, C., Gingras, A.-C., Pearlman, R.E., and Fillingham, J. (2018). The bromodomain-containing protein Ibd1 links multiple chromatin-related protein complexes to highly expressed genes in <i>Tetrahymena thermophila</i>. Epigenetics Chromatin <i>11</i>, 10. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref16">View at publisher</a></p>
<p id="bib17">Saettone, A., Nabeel-Shah, S., Garg, J., Lambert, J.-P., Pearlman, R.E., and Fillingham, J. (2019). Functional proteomics of nuclear proteins in <i>Tetrahymena thermophila</i>: a review. Genes (Basel) <i>10</i>, 333. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref17">View at publisher</a></p>
<p id="bib18">Sowa, M.E., Bennett, E.J., Gygi, S.P., and Harper, J.W. (2009). Defining the human deubiquitinating enzyme interaction landscape. Cell <i>138</i>, 389-403. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref18">View at publisher</a></p>
<p id="bib19">Teo, G., Liu, G., Zhang, J., Nesvizhskii, A.I., Gingras, A.-C., and Choi, H. (2014). SAINTexpress: improvements and additional features in significance analysis of INTeractome software. J. Proteomics <i>100</i>, 37-43. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref19">View at publisher</a></p>
<p id="bib20">Tyanova, S., Temu, T., Sinitcyn, P., Carlson, A., Hein, M.Y., Geiger, T., Mann, M., and Cox, J. (2016). The perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods <i>13</i>, 731-740. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref20">View at publisher</a></p>
<p id="bib21">Urbanska, P., Joachimiak, E., Bazan, R., Fu, G., Poprzeczko, M., Fabczak, H., Nicastro, D., and Wloga, D. (2018). Ciliary proteins Fap43 and Fap44 interact with each other and are essential for proper cilia and flagella beating. Cell. Mol. Life Sci. <i>75</i>, 4479-4493. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref21">View at publisher</a></p>
<p id="bib22">Wahab, S., Saettone, A., Nabeel-Shah, S., Dannah, N., and Fillingham, J. (2020). Exploring the histone acetylation cycle in the protozoan model <i>Tetrahymena thermophila</i>. Front. Cell Dev. Biol. <i>8</i>, 509. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00069-1/sref22">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jeffrey Fillingham (<a href="mailto:jeffrey.fillingham@ryerson.ca">jeffrey.fillingham@ryerson.ca</a>).</p>
<h4>Materials Availability</h4>
<p>This description did not generate new reagents unique to this work that have not already been described and are available.</p>
<h4>Data and Code Availability</h4>
<p>The study did not generate any new datasets or code not already described including their availability.</p>
<h3>Acknowledgments</h3>
<p>Work in the Fillingham and Lambert laboratories was support by the Natural Sciences and Engineering Research Council of Canada (<a href="https://doi.org/10.13039/501100000038">NSERC</a>) Discovery Grants RGPIN-2015-06448 and RGPIN-2017-06124, respectively. J.F. was the recipient of the <a href="https://doi.org/10.13039/100007891">Ryerson University</a> Health Research Fund. Work in the Pearlman laboratory was supported by Canadian Institutes of Health Research (<a href="https://doi.org/10.13039/501100000024">CIHR</a>) MOP13347 and <a href="https://doi.org/10.13039/501100000038">NSERC</a> Discovery Grant 539509. P.-E.K.T. was supported by a Bourse de formation Desjardins pour la recherche et l’innovation from the Fondation du CHU de Québec and a Bourse Distinction Luc Bélanger from the Cancer Research Center – <a href="https://doi.org/10.13039/100007867">Université Laval</a>. J.-P.L. holds a Junior 1 salary award from the Fonds de Recherche du Québec-Santé (FRQ-S) and was also supported through a John R. Evans Leaders Fund from the <a href="https://doi.org/10.13039/501100000196">Canada Foundation for Innovation</a> (37454). The authors would like to thank other laboratory members, particularly Alejandro Saettone and Kanwal Ashraf, for their help in optimizing this protocol.</p>
<h3>Author Contributions</h3>
<p>S.N.-S. wrote, implemented, and optimized the protocol. J.G. edited the manuscript and worked in optimization of the protocol with feedback from S.N.-S. P.-E.K.T. edited the manuscript and figure. R.E.P. edited the manuscript and contributed funding and infrastructure. J.-P.L. provided funds and infrastructure to perform experiments, contributed to the implementation of the protocol, edited the manuscript, and advised during the optimization. J.F. provided funds and infrastructure to perform experiments, edited the manuscript, and advised during the optimization.</p>
<h3>Declaration of Interests</h3>
<p>Authors declare no competing interests.</p>
</section>
</article>