<article>
<section>
<h1 id="header">Calcium imaging of adult-born neurons in freely moving mice</h1>
<p><time datetime="2021-01-08">Published: January 8, 2021</time></p>
<p>Alvaro Carrier-Ruiz,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff4">4</a></sup> Yuki Sugaya,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> Deependra Kumar,<sup><a href="#aff3">3</a></sup> Pablo Vergara,<sup><a href="#aff3">3</a></sup> Iyo Koyanagi,<sup><a href="#aff3">3</a></sup> Sakthivel Srinivasan,<sup><a href="#aff3">3</a></sup> Toshie Naoi,<sup><a href="#aff3">3</a></sup> Masanobu Kano,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Masanori Sakaguchi<sup><a href="#aff3">3</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Neurophysiology, Graduate School of Medicine, The University of Tokyo, Tokyo 113-0033, Japan</p>
<p id="aff2"><sup>2</sup>International Research Center for Neurointelligence (WPI-IRCN), The University of Tokyo Institutes for Advanced Study (UTIAS), Tokyo 113-0033, Japan</p>
<p id="aff3"><sup>3</sup>International Institute for Integrative Sleep Medicine (WPI-IIIS), University of Tsukuba, Tsukuba, Ibaraki 305-0006, Japan</p>
<p id="aff4"><sup>4</sup>Department of Neuroscience, Karolinska Institute, Stockholm 17165, Sweden</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ysugaya@m.u-tokyo.ac.jp">ysugaya@m.u-tokyo.ac.jp</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sakaguchi.masa.fp@alumni.tsukuba.ac.jp">sakaguchi.masa.fp@alumni.tsukuba.ac.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100238">10.1016/j.xpro.2020.100238</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Adult-born neurons (ABNs) in the dentate gyrus bestow unique cellular plasticity to the mammalian brain. We recently found that the activity of ABNs during sleep is necessary for memory consolidation. Here, we describe our method for Ca<sup>2+</sup> imaging of ABN activity using a miniaturized fluorescent microscope and sleep recordings. As preparatory surgery and post-recording data processing can be major obstacles, we provide detailed descriptions and problem-solving tips.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib13">Kumar et al. (2020)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Express a Ca<sup>2+</sup> indicator, GCaMP3, to label adult-born neurons</li>
<li>Perform surgery, Ca<sup>2+</sup> imaging, sleep recording, and fear conditioning</li>
<li>Process data to track the same adult-born neurons across lengthy recording sessions</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>All experiments were performed in accordance with the Science Council of Japan’s Guidelines for Proper Conduct of Animal Experiments. Experimental protocols were approved by the Institutional Animal Care and Use Committee at the University of Tsukuba. C57BL/6 mice (Jackson Laboratory) were bred in our colony at the University of Tsukuba and maintained on a 12-h light/dark cycle (lights on 9 am to 9 pm) with ad libitum access to food and water. All mice were group-housed with 2–5 mice/cage, and only male mice were used in the experiments. To induce transgene expression in ABNs, mice were treated with tamoxifen at 7 weeks of age. At ∼9–12 weeks of age, mice were anesthetized with isoflurane and fixed in a stereotaxic frame for the surgery demonstrated in the videos.</p>
<h3 id="sec1.1">Generate transgenic mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 months</div>
<ol>
<li>Obtain at least two transgenic mouse lines: one that expresses tamoxifen-inducible Cre recombinase in neural stem/progenitor cells, and one that expresses a fluorescent calcium (Ca<sup>2+</sup>) indicator protein after exposure to Cre recombinase.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> We successfully used Nestin-CreERT2 transgenic mice from The Jackson Laboratory (stock #016261), in which Cre recombinase is expressed in Nestin-positive cells in the adult brain by tamoxifen induction. Alternatively, Ascl1-CreERT2 mice (The Jackson Laboratory, stock #012882) could be used to specifically target transient amplifying progenitors (<a href="#bib10">Kim et al., 2007</a>, <a href="#bib22">Yang et al., 2015</a>), although we do not have experience using this line.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend using the GCaMP3 mouse line Ai38 (RCL-GCaMP3) to express the fluorescent Ca<sup>2+</sup> indicator in adult-born neurons (ABNs). Although next-generation indicators, such as GCaMP6f and GCaMP6s, are usually preferable, their mouse lines (<a href="#bib14">Madisen et al., 2015</a>) show very low or absent expression of Cre-inducible transgene when expressed in ABNs after crossing with Nestin-CreERT2 mice. For example, crossing Nestin-CreERT2 mice with Ai96 (RCL-GCaMP6s) or Ai94 (TRE-LSL-GCaMP6s)/RCL-tTA mice (The Jackson Laboratory, stock #008600) does not induce expression of GCaMP6 (unpublished results). We speculate that expression of recently developed GCaMP sensors in early neural progenitors interferes with Ca<sup>2+</sup> signaling mechanisms essential for immature ABN survival.</div>
<div class="note">
<span class="note-title">Note:</span> Instead of using mouse transgenic lines, induction of Ca<sup>2+</sup> indicators and/or Cre recombinase could be achieved by viral transduction. However, most adeno-associated viruses (AAVs) are not suitable for infecting neural stem/progenitor cells due to their low infection rate (<a href="#bib12">Kotterman et al., 2015</a>) and potential cytotoxicity (<a href="#bib9">Johnston et al., 2020</a>). Although AAV-mediated expression of GCaMP6 was reported in combination with two AAVs (<a href="#bib3">Danielson et al., 2016</a>), this previous study did not confirm expression of GCaMP6 in progenitor cells.</div>
<ol start="2">
<li>Cross transgenic lines, in accordance with their corresponding guidelines, to generate mice in which expression of GCaMP3 is initiated by tamoxifen-inducible Cre recombinase (i.e., CreERT2) in adult neural stem/progenitor cells (i.e., nestin:GCaMP3 mice).</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Tamoxifen</td>
<td>Merck</td>
<td>T5648-1G</td>
</tr>
<tr>
<td>Isoflurane inhalation solution</td>
<td>Pfizer</td>
<td>n/a</td>
</tr>
<tr>
<td>Ibuprofen</td>
<td>Wako, Japan</td>
<td>094-02643</td>
</tr>
<tr>
<td>Xylocaine injection 1% with epinephrine (lidocaine)</td>
<td>Aspen Japan</td>
<td>n/a</td>
</tr>
<tr>
<td>OCT mounting medium</td>
<td>Sakura Finetek</td>
<td>4583</td>
</tr>
<tr>
<td>Vectashield</td>
<td>Vector</td>
<td>H-1000</td>
</tr>
<tr>
<td>4% paraformaldehyde phosphate buffer solution</td>
<td>Nacalai Tesque</td>
<td>09154-85</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Nacalai Tesque</td>
<td>30404-45</td>
</tr>
<tr>
<td>Sunflower oil</td>
<td>Sigma</td>
<td>1642347</td>
</tr>
<tr>
<td>Corn oil</td>
<td>Sigma</td>
<td>C8267</td>
</tr>
<tr>
<td>Flunixin meglumine</td>
<td>Fujita</td>
<td>FLUNIXIN INJ. 10%</td>
</tr>
<tr>
<td>Antibiotics (sulfadiazine/trimethoprim)</td>
<td>Kyoritsu Seiyaku</td>
<td>Tribrissen injection</td>
</tr>
<tr>
<td>Tarivid ophthalmic ointment 0.3% (Ofloxacin)</td>
<td>Santen Pharmaceutical</td>
<td>1319722M1056</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mice: C57BL/6-Tg(Nes-cre/ERT2)KEisc/J</td>
<td>The Jackson Laboratory</td>
<td>016261</td>
</tr>
<tr>
<td>Mice: Ai38(RCL-GCaMP3)</td>
<td>The Jackson Laboratory</td>
<td>029043</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>nVista HD</td>
<td>Inscopix</td>
<td>n/a</td>
</tr>
<tr>
<td>Inscopix image decompressor</td>
<td>Inscopix</td>
<td>n/a</td>
</tr>
<tr>
<td>EEG and EMG recording and analysis software</td>
<td>KISSEI COMTEC</td>
<td>Vital Recorder and Sleep Sign</td>
</tr>
<tr>
<td>Mosaic</td>
<td>Inscopix</td>
<td>n/a</td>
</tr>
<tr>
<td>MATLAB</td>
<td>MathWorks</td>
<td>2016b</td>
</tr>
<tr>
<td>IgorPro 8</td>
<td>WaveMetrics</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>∗GRIN lens probe, 1.0-mm diameter, ~4.0-mm length</td>
<td>Inscopix</td>
<td>1050-004595</td>
</tr>
<tr>
<td>∗Rectangular metal frame</td>
<td>Narishige</td>
<td>CF-10</td>
</tr>
<tr>
<td>∗Baseplate</td>
<td>Inscopix</td>
<td>1050-004638</td>
</tr>
<tr>
<td>∗Miniscope</td>
<td>Inscopix</td>
<td>nVista(2.0)</td>
</tr>
<tr>
<td>∗Baseplate cover</td>
<td>Inscopix</td>
<td>1050-004639</td>
</tr>
<tr>
<td>∗Miniscope gripper</td>
<td>Inscopix</td>
<td>1050-002199</td>
</tr>
<tr>
<td>∗Dummy miniscope</td>
<td>Inscopix</td>
<td>1050-003762</td>
</tr>
<tr>
<td>∗∗Electrodes for EEG recording</td>
<td>Yamazaki</td>
<td>ϕ1.0 × 2.0</td>
</tr>
<tr>
<td>∗∗Electrodes for EMG recording</td>
<td>Cooner Wire</td>
<td>AS633</td>
</tr>
<tr>
<td>∗∗Flat cable for EEG/EMG signal transfer</td>
<td>Hitachi Cable</td>
<td>20528-ST LF</td>
</tr>
<tr>
<td>∗∗EEG/EMG signal amplifier</td>
<td>Kyoto Biotex</td>
<td>n/a</td>
</tr>
<tr>
<td>∗∗Analog-to-digital convertor for EEG/EMG signal</td>
<td>Contec</td>
<td>AD16-16U(PCIEV)</td>
</tr>
<tr>
<td>∗∗4-pin header for EEG/EMG socket</td>
<td>Hirose</td>
<td>A3B-4PA-2DSA(71)</td>
</tr>
<tr>
<td>Fear conditioning setup</td>
<td>Med Associates</td>
<td>n/a</td>
</tr>
<tr>
<td>Isoflurane vaporizer</td>
<td>Shinano Seisakusyo</td>
<td>SN-487-OT</td>
</tr>
<tr>
<td>Stereotaxic frame</td>
<td>Narishige</td>
<td>SR-6M</td>
</tr>
<tr>
<td>Drill</td>
<td>Minitor</td>
<td>Minimo</td>
</tr>
<tr>
<td>Vacuum pump</td>
<td>Iwaki</td>
<td>VPUMP-140</td>
</tr>
<tr>
<td>Surgical microscope</td>
<td>Leica</td>
<td>Leica GZ6</td>
</tr>
<tr>
<td>Dental cement liquid</td>
<td>Shofu</td>
<td>PROVINICE 204310644</td>
</tr>
<tr>
<td>Dental cement powder</td>
<td>Shofu</td>
<td>PROVINICE 096017656</td>
</tr>
<tr>
<td>Loctite cyanoacrylate adhesive</td>
<td>Loctite</td>
<td>Loctite 454</td>
</tr>
<tr>
<td>Super glue</td>
<td>Konishi</td>
<td>Alon Alfa A</td>
</tr>
<tr>
<td>Silicone</td>
<td>Shofu</td>
<td>DENTSILICONE-V</td>
</tr>
<tr>
<td>Head-fixing support</td>
<td>Narfishige</td>
<td>MAG-1</td>
</tr>
<tr>
<td>Perista perfusion pump</td>
<td>ATTO</td>
<td>AC2110</td>
</tr>
<tr>
<td>Cryostat</td>
<td>Leica Microsystems</td>
<td>Leica CM 1850</td>
</tr>
<tr>
<td>Confocal fluorescent microscope</td>
<td>Olympus</td>
<td>FV1200</td>
</tr>
<tr>
<td>Glass electrodes</td>
<td>Harvard Apparatus</td>
<td>GC150F-7.5</td>
</tr>
<tr>
<td>Ultrasonic bath</td>
<td>SND</td>
<td>US-101</td>
</tr>
<tr>
<td>Glass electrode puller</td>
<td>Narishige</td>
<td>PC-10</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Data and code underlying the results described in this manuscript</td>
<td>https://github.com/vergaloy/ABNs-Calcium-extraction</td>
<td>n/a</td>
</tr>
<tr>
<td>Raw data from the related article (<a href="#bib13">Kumar et al., 2020</a>)</td>
<td>https://data.mendeley.com/datasets/gfbdv5kfrz/1</td>
<td>n/a</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗Materials only for imaging surgery.<br> ∗∗Materials only for EEG/EMG surgery.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Induction of Ca<sup>2+</sup> sensor expression in ABNs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–8 h</div>
<p>Express GCaMP3 via tamoxifen-inducible Cre recombinase in adult neural stem/progenitor cells.</p>
<ol>
<li>Prepare 20 mg/mL tamoxifen solution.
<ol type="a">
<li>Heat corn oil to 42°C for 30 min.</li>
<li>Dissolve tamoxifen in corn oil at a concentration of 20 mg/mL in an ultrasonic bath (generic consumer glassware is sufficient) or with continuous shaking at 37°C. Store the prepared solution at 4°C.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> As tamoxifen is sensitive to light, it must be made and stored away from light in an amber or foil-wrapped tube.</div>
<div class="note">
<span class="note-title">Note:</span> Tamoxifen may take 1–7 h to completely dissolve depending on the dissolution method.</div>
<ol start="2">
<li>Inject tamoxifen into nestin:GCaMP3 mice.
<ol type="a">
<li>Select an appropriate number of adult nestin:GCaMP3 mice.
<div class="note">
<span class="note-title">Note:</span> The definition of “adult” is usually determined by sexual maturity, which is commonly accepted to be ∼6 weeks of age for mice. Older or younger mice can be used if needed. However, older mice have lower levels of adult neurogenesis, which decreases the success rate of imaging, whereas younger mice might be too delicate to survive the necessary surgical procedures.</div>
</li>
<li>Administer tamoxifen (120 mg/kg) into the intraperitoneal cavity five times at 1- or 2-day intervals. Store the solution at 4°C until the next injection.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid using tamoxifen that has been stored for more than 1 week. The efficiency of tamoxifen-induced recombination is severely impaired when using old or light-exposed tamoxifen.</div>
<div class="note">
<span class="note-title">Note:</span> The duration of the injection period is correlated with the degree of developmental variation of labeled ABNs. That is, the longer the injection period, the higher the chance of recording ABNs with different properties resulting from differences in their maturational stages.</div>
<h3 id="sec2.2">Maturation of labeled ABNs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> dependent on experimental design</div>
<p>Allow ABNs to mature after tamoxifen injection.</p>
<ol start="3">
<li>Wait a pre-determined period of time to allow GCaMP3-labeled ABNs to mature.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The length of the waiting period depends on the target age of ABNs at the time of Ca<sup>2+</sup> imaging. Keep in mind that 1–2 weeks is needed to prepare for imaging. For example, if the target age of ABNs is 6 weeks, wait 4 weeks after tamoxifen injection to move on to the next step. If the target age of ABNs is 1 week or less, consider performing lens implantation before tamoxifen injection. So far, we have successfully performed Ca<sup>2+</sup> imaging of ABNs as early as 3 days after the last tamoxifen injection.</div>
<div class="note">
<span class="note-title">Note:</span> GCaMP3 expression can be verified by examining brain sections from a tamoxifen-injected mouse using a fluorescence microscope (<a href="#fig1">Figure 1</a>) before the next step.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Adult-born neurons (ABNs) expressing the calcium (Ca<sup>2+</sup>) sensor GCaMP3</div>
<p>Optical section of the dentate gyrus (DG) with confocal microscopy showing ABNs expressing the Ca<sup>2+</sup> sensor GCaMP3 induced by tamoxifen injection. The letters g, h, and ml represent the granule cell layer, hilus, and molecular layer, respectively. The arrowhead indicates the subgranular zone, and the dotted line indicates the border between the DG and cornu ammonis 1 (CA1). Scale bar, 100 μm.</p>
</figcaption>
</figure>
<h3 id="sec2.3">GRIN lens and EEG/EMG electrode implantation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h per mouse</div>
<p>Surgically implant a gradient-index (GRIN) lens and electroencephalogram (EEG)/electromyogram (EMG) electrodes into the brains of tamoxifen-injected mice.</p>
<div class="note">
<span class="note-title">Note:</span> We describe a specific protocol for imaging ABNs in the dentate gyrus (DG) in freely moving mice. The original miniscope technology used in this protocol was first developed by <a href="#bib5">Ghosh et al., (2011)</a>. For a more generalized protocol to image other brain regions, please refer to <a href="#bib19">Resendez et al. (2016)</a> for a recent overview. <i>In vivo</i> Ca<sup>2+</sup> imaging of ABNs in the DG was first described by <a href="#bib3">Danielson et al. (2016)</a> in awake head-fixed mice.</div>
<div class="note">
<span class="note-title">Note:</span> We implanted EEG/EMG electrodes to identify sleep stages in combination with ABNs activity. Mammalian sleep contains two major orthogonal states: rapid eye movement (REM) sleep and non-REM (NREM) sleep. Each sleep stage is characterized by prominent neural oscillatory patterns. For example, slow waves in the cortex are abundant during deep NREM sleep, whereas theta oscillations in the hippocampus occur during REM sleep, both of which play critical roles in memory consolidation (<a href="#bib1">Boyce et al., 2016</a>; <a href="#bib15">Marshall et al., 2006</a>). Details of sleep recording and analysis are available in <a href="#bib17">Oishi et al. (2016)</a>.</div>
<ol start="4">
<li>Anesthetize the mouse with an anesthetic of choice (e.g., isoflurane).</li>
</ol>
<ol start="5">
<li>Open a cranial window sufficiently large for lens and electrode implantation.
<ol type="a">
<li>Shave hair from the surgical site.</li>
<li>Secure the head of the mouse in a stereotaxic frame.</li>
<li>Cover the eyes of the mouse with white petrolatum cream to prevent dryness.</li>
<li>Disinfect the skin at the surgical site.</li>
<li>Make a small incision in the skin using a spring scissor along the sagittal line of the skull.</li>
<li>Clean the exposed skull with cotton swabs.</li>
<li>Scratch the skull surface using a drill bit to make it rough, which will facilitate later adhesion of glue and dental cement.</li>
<li>Obtain a flat skull position by matching the height of two parallel points along the sagittal axis (±3 mm lateral from bregma) as well as the height of bregma and lambda points.</li>
<li>Mark the desired stereotaxic coordinates for the GRIN lens and EEG electrodes on the skull surface using the stereotaxic manipulator.</li>
<li>Drill five holes in the skull: one for the GRIN lens, two over one hemisphere for EEG electrodes, and two over the contralateral hemisphere for anchoring electrodes (i.e., not attached to EEG wires; <a href="#fig2">Figure 2</a>, <a href="#mmc1">Methods Video S1</a>, 00:00–00:30).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Positions of the four screws and hole for the gradient-index (GRIN) lens</div>
<p>The blue arrow indicates the GRIN lens position. The green arrowhead indicates bregma.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> The electrode holes should be just large enough to accommodate the tip of the electrodes. In <a href="#fig2">Figure 2</a>, the holes were drilled above the frontal and parietal cortices on both sides of skull as an example: anteroposterior (AP) +1.5 mm and −3 mm and mediolateral (ML) ±1.5 mm and ±1.7 mm, respectively. The anchoring electrodes prevent the implant from becoming detached at later points in the experimental protocol.</div>
<figure id="mmc1"><video controls="controls">
                <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Mmc1.mp4" type="video/mp4"></source>
                Your browser does not support HTML5 video.
              </video>
<figcaption>
<div class="figcaption-title">Methods Video S1. Drilling holes for implanting the GRIN lens and EEG electrodes, refer to steps 5j and k and 6a</div>
</figcaption>
</figure>
</li>
<li>Make a cranial window for GRIN lens implantation using a drill (<a href="#mmc1">Methods Video S1</a>, 00:30–00:41).
<div class="note">
<span class="note-title">Note:</span> The cranial window should be just large enough to accommodate the GRIN lens implant. We recommend using an Inscopix GRIN lens 1 mm in diameter and 4 mm in length to record ABNs in the dorsal DG. In this case, a 1.2-mm<sup>2</sup> cranial window centered at AP −2 mm and ML +0.7 mm works well.</div>
</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Implant the EEG electrodes, anchoring screws, and GRIN lens.
<ol type="a">
<li>Insert the EEG electrodes and anchoring screws epidurally (i.e., not completely penetrating the skull; <a href="#fig2">Figure 2</a>, <a href="#mmc1">Methods Video S1</a>, 00:42–01:31).</li>
<li>For the lens hole, aspirate cortical tissue above the region of interest using a glass pipette connected to a vacuum pump in circular movements from the center to the periphery of the cranial window (<a href="#mmc2">Methods Video S2</a>, 00:00–00:36). While aspirating the cortex, the brain tissue will first have a homogeneous appearance. Stop aspiration when white matter tracts of the corpus callosum and alveus hippocampus are visible as identified by striations (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. White matter tracts with white striations exposed during aspiration of cortical tissue</div>
<p>The area of white matter tracts is surrounded by the light blue dotted line. The cranial window (blue dotted line) is larger than the exposed area. Scale bar, 1 mm.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> From this point onward, some bleeding is normal during the procedure. Continuously irrigate the exposed brain tissue with sterile saline while aspirating the cortex (<a href="#mmc2">Methods Video S2</a>, 00:13–00:23).</div>
<div class="critical">
<span class="critical-title">Critical:</span> A surgical microscope is needed to observe distinctions in brain structure during surgery. In our experience, damaging the CA1 by aspiration usually destroys the structural organization of the DG.</div>
<figure id="mmc2"><video controls="controls">
                <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Mmc2.mp4" type="video/mp4"></source>
                Your browser does not support HTML5 video.
              </video>
<figcaption>
<div class="figcaption-title">Methods Video S2. Aspirating cortical tissue for GRIN lens implantation, refer to step 6b and c</div>
</figcaption>
</figure>
</li>
<li>After reaching the corpus callosum fibers, keep the exposed brain tissue irrigated with sterile saline while aspirating any blood present in the region to maintain visibility (<a href="#mmc2">Methods Video S2</a>, 00:37–00:53).</li>
<li>Attach a GRIN lens to a stereotaxic manipulator using a clip holder (<a href="#fig4">Figure 4</a>) and align it above the center of the cranial window (<a href="#mmc3">Methods Video S3</a>, 00:00–00:15).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Setup for GRIN lens insertion</div>
<p>A GRIN lens (blue arrow) is held by a clip holder (green arrow) and inserted into the hippocampus.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> We made a lens holder by gluing a generic spring clip holder to the base of a manipulator bar. We recommend holding the GRIN lens at the tip (∼0.5-mm length) of the clip holder and keeping the lens straight (<a href="#mmc3">Methods Video S3</a>, 00:00–00:09).</div>
<figure id="mmc3"><video controls="controls">
                <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Mmc3.mp4" type="video/mp4"></source>
                Your browser does not support HTML5 video.
              </video>
<figcaption>
<div class="figcaption-title">Methods Video S3. Implanting the GRIN lens, refer to step 6d–h</div>
</figcaption>
</figure>
</li>
<li>Carefully lower the lens using 0.1-mm dorso-ventral steps into the hippocampus. The final coordinate of the objective surface of the GRIN lens is 1.3 mm lower than the top of the skull (<a href="#mmc3">Methods Video S3</a>, 00:16–00:34).
<div class="note">
<span class="note-title">Note:</span> We lower the lens at a rate of ∼500 μm/min. Tissue swelling and relaxation affect the quality of subsequent recordings. We do not recommend using a smaller diameter lens because the resulting reduction in field of view makes it substantially more difficult to observe a population of ABNs that is sparse in both number and activity.</div>
</li>
<li>Fix the GRIN lens to the skull and all four screws with a thin layer of cyanoacrylate adhesive (Loctite 454, referred to henceforth as Loctite glue) and allow it to cure for ∼5–10 min (<a href="#mmc3">Methods Video S3</a>, 00:35–00:53).
<div class="note">
<span class="note-title">Note:</span> We find that adding dental cement liquid on top of Loctite glue rapidly solidifies the adhesive and reduces this step to a few minutes.</div>
</li>
<li>Release the lens from the stereotaxic manipulator (<a href="#mmc3">Methods Video S3</a>, 00:54–01:00).</li>
<li>Cover the skull using a layer of Loctite glue and dental cement liquid around the insertion point (<a href="#mmc3">Methods Video S3</a>, 01:00–01:23) and allow it to completely cure.</li>
<li>Attach the EEG/EMG socket to the skull using Loctite glue, add a drop of dental cement liquid, and allow it to completely cure (<a href="#mmc4">Methods Video S4</a>, 00:00–00:11).
<div class="critical">
<span class="critical-title">Critical:</span> The EEG/EMG socket should be fixed far enough from the lens to avoid later collision between the socket and microscope.</div>
<figure id="mmc4"><video controls="controls">
                <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Mmc4.mp4" type="video/mp4"></source>
                Your browser does not support HTML5 video.
              </video>
<figcaption>
<div class="figcaption-title">Methods Video S4. Implanting EEG/EMG electrodes, refer to step 6i–7</div>
</figcaption>
</figure>
</li>
<li>Insert two wires into the cervical portion of the trapezoid muscles for EMG recording (<a href="#mmc4">Methods Video S4</a>, 00:12–00:33).</li>
<li>Apply carbon black powder-mixed dental cement (liquid + powder) to any exposed area of the skull as well as to the area where the Loctite glue meets the GRIN lens and EEG/EMG wires and let it solidify for ∼10 min (<a href="#mmc4">Methods Video S4</a>, 00:34–00:49).
<div class="critical">
<span class="critical-title">Critical:</span> Completely cover the EEG/EMG wires with Loctite glue to prevent damage when the mouse scratches its head (<a href="#fig5">Figure 5</a>).</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Electrodes and GRIN lens fixed with dental cement</div>
<p>The skull and electroencephalogram (EEG)/electromyogram (EMG) wires are completely covered with black dental cement. Note that the EEG/EMG socket (green arrow) and GRIN lens (blue arrow) are not covered.</p>
</figcaption>
</figure>
</li>
<li>Apply Loctite glue to the outer border of the dental cement to connect it to the surrounding skin and wait until it completely cures (<a href="#mmc4">Methods Video S4</a>, 00:50–01:00).</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Cover the GRIN lens and surrounding area with silicone to protect it from damage resulting from mouse activity in the home cage (<a href="#fig6">Figure 6</a>, <a href="#mmc4">Methods Video S4</a>, 01:01–01:12).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. GRIN lens and the surrounding area covered with silicone</div>
<p>The silicone is indicated by dotted line.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Attach a rectangular metal frame (12 × 19 × 1 mm) to the skull using dental cement and let it solidify for 5 min. This frame will prevent the mouse from scratching the lens and aid in manipulating the awake mouse when attaching the miniaturized fluorescent microscope (miniscope) and EEG/EMG cables before recording.</div>
<ol start="8">
<li>Finish the surgical procedure.
<ol type="a">
<li>Stop anesthesia.</li>
<li>Remove the mouse from the stereotaxic holder.</li>
<li>Administer postoperative 5% glucose solution and ibuprofen (30 mg/kg).</li>
<li>Place the mouse in a clean cage partially covering a heating pad to prevent hypothermia and allow it to recover.</li>
</ol>
</li>
<li>House the operated mouse individually to avoid damage to the implant.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To prepare ibuprofen (30 mg/kg) solution, dissolve 30 mg ibuprofen powder in 100 μL of 100% ethanol, add 1 mL sunflower oil, and evaporate the ethanol by centrifuging.</div>
<h3 id="sec2.4">Postoperative recovery</h3>
<div class="timing">
<span class="timing-title">Timing:</span> at least 1 week</div>
<p>Allow operated mice to rest for at least 1 week with minimal manipulation.</p>
<div class="note">
<span class="note-title">Note:</span> We recommend keeping mice in the sleeping chamber/room during recovery to habituate them to the environment.</div>
<div class="note">
<span class="note-title">Note:</span> Manipulation of mice during this period may cause detachment of the lens and electrodes, as some degree of inflammation might still be present, making the implant more vulnerable to detachment from the skull.</div>
<h3 id="sec2.5">Miniscope baseplate attachment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min per mouse</div>
<p>Attach a magnetic baseplate for the miniscope to lens-implanted mice.</p>
<ol start="10">
<li>Anesthetize the mouse with an anesthetic of choice (e.g., isoflurane).</li>
<li>Secure the head of the mouse in the stereotaxic frame.</li>
<li>Prepare the miniscope.
<ol type="a">
<li>Attach the magnetic baseplate to the miniscope.</li>
<li>Secure the miniscope to its gripper tool and attach it to a micromanipulator.</li>
<li>Connect the recording hardware to the computer.</li>
<li>Position the miniscope above the head of the mouse.</li>
<li>Start the image acquisition software.</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Attach the magnetic baseplate.
<ol type="a">
<li>Remove the silicone protection from the skull of the mouse.</li>
<li>Adjust the position of the miniscope using the micromanipulator, centering it above the implanted lens.</li>
<li>Turn on the LED light of the miniscope and lower it until the top surface of the lens becomes visible through the camera.</li>
<li>Carefully continue lowering the miniscope until the DG becomes visible.
<div class="note">
<span class="note-title">Note:</span> The DG will be recognizable by its abundant vasculature. Due to GCaMP3 expression in ABNs and our implantation method, this will be the first focusable structure.</div>
</li>
<li>From this point, to adjust the imaging focus to the granular/subgranular layer, lower the miniscope ∼50 μm. The vasculature should become slightly out of focus at this point (<a href="#fig7">Figure 7</a>A).
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Fluorescent signals from the DG observed through the GRIN lens</div>
<p>The DG is shown (A) without or (B) with ΔF/F processing. Red arrows indicate vasculature, and the blue arrowhead indicates a Ca<sup>2+</sup> transient in an ABN. Scale bar, 100 μm.</p>
</figcaption>
</figure>
<div class="optional">
<span class="optional-title">Optional:</span> When the outlines of GCaMP3-expressing ABNs can be visualized, we recommend switching to the nVista software ΔF/F function to confirm the presence of dynamic changes in fluorescence corresponding to Ca<sup>2+</sup> transients (<a href="#fig7">Figure 7</a>B).</div>
<div class="note">
<span class="note-title">Note:</span> Due to the low signal-to-noise ratio of GCaMP3 and the small number of ABNs labeled using our protocol, the fluorescence signal from ABNs may be undetectable at this point. It may still be possible to observe ABN activity with post-recording processing of the images. The success rate is ∼50%.</div>
</li>
<li>After choosing the optimal focal plane, secure the baseplate to the skull using dental cement (liquid + power) and allow it to completely cure (<a href="#fig8">Figure 8</a>, <a href="#mmc5">Methods Video S5</a>).
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Black baseplate secured above the GRIN lens using black dental cement</div>
<p>(A) Top view.</p>
<p>(B) Rear view. Note that the screw (blue arrows) and the surface of the baseplate are not covered by dental cement.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> When fixing the baseplate, there are three points at which dental cement should not be directly applied: the surface of the implanted lens, the screw for the magnetic baseplate, and the miniscope.</div>
<div class="note">
<span class="note-title">Note:</span> Fixing the baseplate without gaps will prevent dust and ambient illumination from interfering with the field of view during imaging.</div>
<div class="note">
<span class="note-title">Note:</span> Generic dental cement may shrink to a small degree when it becomes solid. If the position of the baseplate changes significantly, setting the baseplate 5–10 μm above the optimal focal plane before applying dental cement or using C&amp;B Metabond as recommended by Inscopix may be helpful. In addition, the focal plane can be adjusted at the time of recording by turning the microscope as instructed by Inscopix.</div>
<figure id="mmc5"><video controls="controls">
                <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Mmc5.mp4" type="video/mp4"></source>
                Your browser does not support HTML5 video.
              </video>
<figcaption>
<div class="figcaption-title">Methods Video S5. Fixing the baseplate, refer to step 13f</div>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Finish the imaging preparation.
<ol type="a">
<li>After the dental cement has solidified completely, release the miniscope from the gripper tool.</li>
<li>Detach the miniscope from the magnetic baseplate, which should remain attached to the skull.</li>
<li>Attach the baseplate cover to prevent dust on the lens.</li>
<li>Release the mouse and return it to its home cage.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this point, mice are in principle ready for recording.</div>
<h3 id="sec2.6">Habituation for EEG/EMG recording and Ca<sup>2+</sup> imaging during sleep</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼6 h per day for at least 3 days per mouse</div>
<p>Habituate mice to the miniscope and EEG/EMG cables.</p>
<ol start="15">
<li>Attach the dummy miniscope and dummy EEG/EMG cables to the mouse.
<ol type="a">
<li>Hold the mouse head on both sides of the implant (<a href="#mmc6">Methods Video S6</a>, 00:00–00:23).
<div class="critical">
<span class="critical-title">Critical:</span> The mouse will stop moving if its eyes are covered (<a href="#mmc6">Methods Video S6</a>). Avoid applying too much force to the implant, which could cause its detachment later.</div>
<figure id="mmc6"><video controls="controls">
                <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Mmc6.mp4" type="video/mp4"></source>
                Your browser does not support HTML5 video.
              </video>
<figcaption>
<div class="figcaption-title">Methods Video S6. Connecting the miniscope and EEG/EMG cables to the mouse, refer to steps 15–16</div>
</figcaption>
</figure>
</li>
<li>Remove the baseplate cover and attach the dummy miniscope and dummy EEG/EMG cables (<a href="#fig9">Figure 9</a>, <a href="#mmc6">Methods Video S6</a>, 00:24–01:39).
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Dummy miniscope and dummy EEG/EMG cables attached to the mouse</div>
<p>The dummy miniscope and the dummy EEG/EMG cables are indicated by a blue and a green arrow, respectively.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Mice might initially show difficulties moving around with the miniscope and EEG/EMG cables. We find that a minimum of 9 days of habituation is necessary for mice to freely move. As the goal is only to habituate mice, it is not necessary to use the real recording system at this point.</div>
</li>
</ol>
</li>
</ol>
<ol start="16">
<li>Place the mouse inside the sleeping chamber for ∼6 h (<a href="#mmc6">Methods Video S6</a>, 01:40–02:02).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Periodically visually confirm whether the mouse sleeping. Mice should be able to fall asleep more quickly after each habituation session.</div>
<ol start="17">
<li>Finish the habituation protocol.
<ol type="a">
<li>Remove the mouse from the sleeping chamber.</li>
<li>Detach the dummy materials and attach the baseplate cover.</li>
<li>Return the mouse to its home cage.</li>
</ol>
</li>
<li>Repeat the habituation protocol on the following day until the mouse is able to sleep and freely move around inside the sleeping chamber.</li>
</ol>
<h3 id="sec2.7">EEG/EMG recording and Ca<sup>2+</sup> imaging during sleep: day 1, baseline recording</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–6 h per mouse depending on desired imaging period</div>
<div class="note">
<span class="note-title">Note:</span> For illustrative purposes, we present a protocol for cued fear conditioning as previously described (<a href="#bib13">Kumar et al., 2020</a>).</div>
<p>Record EEG/EMG and Ca<sup>2+</sup> transients during sleep.</p>
<ol start="19">
<li>Prepare the mouse for EEG/EMG recording and Ca<sup>2+</sup> imaging.
<ol type="a">
<li>Start the acquisition software and set the imaging parameters.
<div class="note">
<span class="note-title">Note:</span> For consistency, use the same imaging settings/parameters for all mice. For lengthy imaging sessions, we recommend setting the acquisition rate at ∼5 Hz to reduce the file size. For 5 Hz recordings, we find that an LED intensity of 10%–30% of its maximum power for 4 h does not oversaturate any detected pixel. We usually do not observe cell death resulting from phototoxicity during lengthy imaging sessions, but a small degree of photobleaching is inevitable. As the signal-to-noise ratio of GCaMP3 is not as good as that for newer-generation Ca<sup>2+</sup> sensors, we recommend leaving gain at 1 to reduce imaging noise.</div>
</li>
<li>Secure the miniscope and EEG/EMG cables above the recording chamber.
<div class="critical">
<span class="critical-title">Critical:</span> To avoid damage to the cables, ensure that they do not hang close to the mouse, which can bite and damage them. At the same time, there should be sufficient slack on the cables to avoid movement-induced cable twisting and strain.</div>
<div class="note">
<span class="note-title">Note:</span> We placed the EEG/EMG and miniscope cable close to each other so that their rotation axes are similar. During our recording period (maximum ∼4 h), we did not experience substantial tangling of the EEG/EMG and miniscope cables, presumably because we typically perform experiments during the early half of daytime, when mice are often sleeping. If tangling occurs, we recommend using a slip ring for the miniscope with a hole to accommodate the EEG/EMG cable passing through it (or vice versa).</div>
</li>
<li>Remove the baseplate cover and attach the miniscope and EEG/EMG cables.</li>
</ol>
</li>
</ol>
<ol start="20">
<li>Start EEG/EMG recording and Ca<sup>2+</sup> imaging simultaneously.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Periodically monitor the state of the mouse and cables to ensure that that mouse is behaving naturally and there is no twisting of the cables.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend closely monitoring for signs of sleep apnea. This can generally be achieved by monitoring for fragmented sleep during an NREM sleep episode.</div>
<ol start="21">
<li>Finish the recordings.
<ol type="a">
<li>Stop the recordings.</li>
<li>Detach both the miniscope and EEG/EMG cables and reattach the dummy miniscope and EEG cable.</li>
</ol>
</li>
</ol>
<h3 id="sec2.8">EEG/EMG recording and Ca<sup>2+</sup> imaging during sleep: day 2, fear conditioning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–6 h per mouse depending on the research aim</div>
<p>Fear condition mice with tones and foot shocks.</p>
<ol start="22">
<li>Prepare the mouse for the fear conditioning chamber.
<ol type="a">
<li>Remove the baseplate cover and attach the miniscope.</li>
<li>Start the Ca<sup>2+</sup> imaging acquisition software and set the imaging parameters.</li>
<li>Arrange the cables above the conditioning chamber by taking the same precautions as those used for the sleeping chamber.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use the dummy miniscope if the actual recoding equipment is not needed during conditioning.</div>
<ol start="23">
<li>Fear condition the mouse.
<ol type="a">
<li>Place the mouse inside the conditioning chamber.</li>
<li>Perform Ca<sup>2+</sup> imaging for 10 min during the pre-conditioning period (i.e., pre-shock recording).</li>
<li>Stop the imaging and quickly detach the miniscope and reattach the dummy miniscope (&lt;1 min) to avoid a change in either the shock condition or the field of view (due to hitting the miniscope against the wall during shock).</li>
<li>Perform cued fear conditioning (<a href="#mmc7">Methods Video S7</a>).
<figure id="mmc7"><video controls="controls">
                <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Mmc7.mp4" type="video/mp4"></source>
                Your browser does not support HTML5 video.
              </video>
<figcaption>
<div class="figcaption-title">Methods Video S7. Fear conditioning with the miniscope attached to the head, refer to step 23d</div>
</figcaption>
</figure>
</li>
<li>Quickly reattach the miniscope and perform 5 min of Ca<sup>2+</sup> imaging (i.e., post-shock recording).</li>
<li>Finish the recording and return the mouse to the sleeping chamber.</li>
<li>Connect the EEG/EMG cable and start EEG/EMG recording and Ca<sup>2+</sup> imaging simultaneously.</li>
</ol>
</li>
</ol>
<ol start="24">
<li>Record EEG/EMG activity and Ca<sup>2+</sup> transients.
<ol type="a">
<li>Transfer the sleeping chamber to the sleep recording room.
<div class="note">
<span class="note-title">Note:</span> If necessary, it is possible to record the activity of neurons during the foot shocks of fear conditioning (<a href="#bib7">Grewe et al. 2017</a>). In this case, we recommend covering the interior walls of the conditioning chamber with a shock-absorbing material to prevent damage to the miniscope if the mouse hits the walls when reacting to the foot shocks.</div>
<div class="note">
<span class="note-title">Note:</span> Avoid detaching the miniscope and cables from the head of the mouse when transferring it to the sleeping chamber to avoid a change in the field of view. If necessary, disconnect the cables from the computer port and reconnect them after transferring the mouse.</div>
<div class="note">
<span class="note-title">Note:</span> If the mouse is not properly habituated to the experimental manipulation, it is possible that some procedures (e.g., plugging and unplugging the cables/miniscope) could stress the mouse and affect its freezing response. Therefore, we suggest confirming whether freezing is specifically observed in the shocked context. For a no-learning control group, a no shock or immediate shock protocol can be considered.</div>
</li>
<li>Start the EEG/EMG recording and Ca<sup>2+</sup> imaging simultaneously.</li>
</ol>
</li>
</ol>
<ol start="25">
<li>Finish the recording as done on the previous day.</li>
</ol>
<h3 id="sec2.9">EEG/EMG recording and Ca<sup>2+</sup> imaging: day 3, fear conditioning test</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–10 min per mouse</div>
<p>Confirm the establishment of a fear memory.</p>
<ol start="26">
<li>Prepare the mouse for the fear conditioning chamber as done on the previous day.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use the dummy miniscope if the actual recoding equipment is not needed during conditioning.</div>
<ol start="27">
<li>Confirm the establishment of a fear memory.
<ol type="a">
<li>Start EEG/EMG recording and Ca<sup>2+</sup> imaging software.</li>
<li>Place the mouse inside the conditioning chamber.</li>
<li>Start the EEG/EMG recording and Ca<sup>2+</sup> imaging simultaneously.</li>
<li>Leave the mouse inside the chamber for 5–10 min.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Attaching the miniscope during the test tends to reduce the overall amount of freezing (<a href="#bib13">Kumar et al., 2020</a>). To avoid confounding effects, we habituated mice to the weight of the miniscope using a dummy miniscope and cables. A similar approach is widely adopted in tetrode experiments. We found that although mice tend to show low freezing behavior when the miniscope is attached, they freeze only in the learning context (<a href="#bib13">Kumar et al., 2020</a>). If the freezing duration is still too short, longer habituation with the dummy microscope or counterbalancing the microscope should be considered. Alternatively, a slip ring could be used.</div>
<ol start="28">
<li>Finish the recording as done on the previous day.</li>
</ol>
<h3 id="sec2.10">Anatomical confirmation of the recorded signals</h3>
<div class="timing">
<span class="timing-title">Timing:</span> dependent on the desired histological method</div>
<p>Verify the position of the implanted GRIN lens post-mortem.</p>
<ol start="29">
<li>Obtain a brain sample
<ol type="a">
<li>Euthanize the mouse according to institutional guidelines.</li>
<li>Transcardially perfuse the mouse with cold 4% paraformaldehyde (PFA).</li>
<li>Remove the brain and place it inside a tube with 4% PFA at 4°C for 24 h.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The brain can be stored at 4°C for several days. In this case, exchange the PFA for phosphate-buffered saline (PBS) after 24 h and protect the tube from light, but expect a natural decay of the fluorescence signal over time. Long exposure to PFA may interfere with immunohistochemical detection of antigens, some of which are critical for adult neurogenesis research, such as doublecortin (<a href="#bib16">Moreno-Jimenez et al., 2019</a>).</div>
<div class="optional">
<span class="optional-title">Optional:</span> A peristaltic pump can be used for perfusion. Replace the 4% PFA with 20%–30% sucrose solution if a cryostat will be used to slice the brain. After the brain sinks, remove it from the tube and cryopreserve it in an OCT mounting medium block at −80°C.</div>
<ol start="30">
<li>Slice the brain.
<ol type="a">
<li>Slice the brain using a cryostat or vibratome at 50 μm thickness and collect the slices in plastic wells or dishes with desired buffer.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use 60% glycerol/PBS solution for long-term storage of brain slices in a freezer. In case slices must be kept at 4°C, we recommend adding 0.1% sodium azide to the buffer to prevent bacterial contamination.</div>
<ol start="31">
<li>Confirm the location of the implanted lens and GCaMP3 fluorescence signal.
<ol type="a">
<li>Mount the brain slices on microscope slides using fluorescence-preserving medium.</li>
<li>Image the slides with a fluorescence microscope (<a href="#fig10">Figure 10</a>).
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Histological confirmation of GRIN lens position and GCaMP3 signal</div>
<p>The letters g, h, and ml indicate the granule cell layer, hilus, and molecular layer of the DG, respectively. The white line indicates the position of the lens, and the white dotted line indicates the border between the DG and stratum lacunosum moleculare. The expression of GCaMP3 is shown in green. Scale bar, 100 μm.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> GCaMP3 fluorescence is usually sufficiently bright to be detected without immunohistochemical staining. If it is not possible to observe the fluorescence signal due to a particular reason (e.g., acetone pretreatment), a simple antibody stain targeting the GFP molecule is sufficient to confirm GCaMP3 expression. We found that using Malinol mounting medium is an economical option for rapid confirmation of the signal.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Successful completion of this method allows the detection of ABN Ca<sup>2+</sup> transients in freely moving mice. A recording session configured with the parameters described here will produce a raw recording file with a size of ∼1,327 kb/frame. A representative recording video of Ca<sup>2+</sup> transients is provided in our previous article (<a href="#bib13">Kumar et al., 2020</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Here, we present steps for processing raw video files into Ca<sup>2+</sup> fluorescence time series. We use Mosaic software for preprocessing and motion correction and the MATLAB implementation of the constrained non-negative matrix factorization (CNMF)-E algorithm (<a href="#bib21">Zhou et al., 2018</a>) and custom scripts to optimize extraction of ABN Ca<sup>2+</sup> transients (available through the GitHub link below).</p>
<div class="note">
<span class="note-title">Note:</span> We recommend the following system requirements: ≥128 GB RAM; MATLAB 2020 or above; most recent Intel Core i7 processor; 1 TB SSD.</div>
<ol>
<li>Decompress the raw files using the Inscopix image decompressor.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend downsampling the video by a factor of 4 during decompression to save disc space and reduce loading times. Decompressing video files into .hdf5 format substantially improves loading times in Mosaic and MATLAB.</div>
<ol start="2">
<li>Preprocess the video file according to the standard Mosaic workflow (<a href="#mmc8">Methods Video S8</a>, 00:00–00:22).<br>
<ol type="a">
<li>Import the video file into Mosaic software.</li>
<li>Preprocess the file in Mosaic. Select “fix defective pixels” and “fix isolated dropped frames.”</li>
</ol>
<figure id="mmc8"><video controls="controls">
            <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Mmc8.mp4" type="video/mp4"></source>
            Your browser does not support HTML5 video.
          </video>
<figcaption>
<div class="figcaption-title">Methods Video S8. Analyzing Ca<sup>2+</sup> imaging data, refer to quantification and statistical analysis steps 2–5</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Spatial downsampling is not necessary if already done during decompression.</div>
<ol start="3">
<li>Perform motion correction according to the standard Mosaic workflow (<a href="#mmc8">Methods Video S8</a>, 00:22–01:25).
<ol type="a">
<li>Save the file in “.h5” format.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Preprocessing and motion correction can be performed using any version of Inscopix Data Processing Software (<a href="https://www.youtube.com/watch?v=YF4IlcotTrM&amp;feature=emb_logo">https://www.youtube.com/watch?v=YF4IlcotTrM&amp;feature=emb_logo</a>). Motion correction can be performed using CaImAn (<a href="#bib6">Giovannucci et al., 2019</a>) or any other software while considering the specific characteristics of ABN activity as described below.</div>
<div class="note">
<span class="note-title">Note:</span> Ideally, perform optimal motion correction by choosing a reference point in the field of view where most ABNs are located with clear spatial markers (i.e., blood vessels). This is important because movements of the brain occur in three dimensions and might be non-rigid. Thus, perfect motion correction in the entire field of view is not possible in most cases. Instead, choose reference regions that minimize motion in the region where ABNs are located and consider cropping out unnecessary portions of the video, which will reduce the number of false positives and decrease processing time. Moreover, some regions in the field of view may move asynchronously. This can occur when tissue or blood clots get stuck between the lens and focal plane, which creates different planes of movement in the image. Try to avoid analyzing regions that contain desynchronized movement.</div>
<div class="note">
<span class="note-title">Note:</span> Detecting ABN activity by the naked eye can be challenging. This is because ABNs are low in number (∼5–25 ABNs per field of view) and very sparsely active (∼1 transient/min). Moreover, the GCaMP3 signal is weaker than newer-generation GCaMPs. To improve the identification of regions with active ABNs, adjust the contrast to between 40%–95% of the maximum observed signal. An estimation of neuronal location can be obtained in the motion correction panel when you check “subtract spatial mean,” “invert image,” and “apply spatial mean,” which makes ABNs visible as black circular shapes in the field of view.</div>
<div class="critical">
<span class="critical-title">Critical:</span> After performing motion correction, check that no major signs of motion are visually detected. A single run of the motion correction algorithm is usually not sufficient to solve all problems. Instead, try using different combinations of reference region and motion correction type (e.g., translation, rotation). We recommend first using “translation only” as the motion correction type. If there is still motion that was not corrected, check whether this is due to rotations or expansions in the field of view and change the motion correction type accordingly.</div>
<div class="note">
<span class="note-title">Note:</span> For information on how to perform motion correction when tracking the same ABNs through different recording sessions, see the “tracking the same neurons” section in our previous article (<a href="#bib13">Kumar et al., 2020</a>).</div>
<ol start="4">
<li>Extract Ca<sup>2+</sup> transients in MATLAB using CNMF-E (<a href="#mmc8">Methods Video S8</a>, 1:25–6:25).
<ol type="a">
<li>Set up a folder with the “.h5” video files to analyze.</li>
<li>Add the folder with the CNMF-E scripts provided in this article to the MATLAB path.</li>
<li>Open the file “ABNs_PV.m.”</li>
<li>Input the path of the folder with the video files in line 2.</li>
<li>Input the frame rate in line 46.</li>
<li>Run the code.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For short video sessions, run “demo_large_data_1p.m” instead of “ABNs_PV.m.”</div>
<ol start="5">
<li>Perform the following quality controls:
<ol type="a">
<li>Run “neuron.viewNeurons([], neuron.C_raw);” to visually inspect each neuron.
<div class="note">
<span class="note-title">Note:</span> Delete extracted components with temporal traces or spatial shapes that do not correspond to real neurons. If you must delete too many neurons, consider repeating the analysis with higher peak-to-noise ratio (PNR) or local correlation (CORR) thresholds.</div>
</li>
<li>Run “neuron.show_contours(0.6, [], neuron.PNR.∗neuron.Cn, 1)” to see the contours of the extracted components over the PNR image multiplied by the CORR image.
<div class="note">
<span class="note-title">Note:</span> Enlarged non-circular regions usually reflect motion artifacts. If contours are not drawn over several neuron-like regions, this suggests that the PNR or CORR threshold were set too high. If contours are drawn over several non-neuron-like regions, this suggests that the PNR or CORR threshold were set too low.</div>
</li>
<li>Run “implay(cat(2,mat2gray(neuron.PNR_all),mat2gray(neuron.Cn_all)),5);” to see the PNR and CORR images from consecutive temporal segments of 1,000 frames.</li>
<li>Check for signs of motion artifacts and the presence of a visual representation of the mean activity of ABNs across different times.</li>
<li>Run stackedplot(neuron.C_raw); to see several Ca<sup>2+</sup> transients at the same time.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Correlated ensemble activity is common in ABNs; however, perfectly overlapping rising dynamics with square-like shapes may indicate motion artifacts. Check the video in the specific frames to corroborate whether this is real ensemble activity (example in <a href="#mmc8">Methods Video S8</a>).</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> To extract Ca<sup>2+</sup> transients, we use CNMF-E (<a href="#bib21">Zhou et al., 2018</a>) with some modifications. We recommend first reading the original CNMF-E article to understand the main concepts behind the algorithm. We choose CNMF-E because it is particularly good at extracting noisy signals, and the code is easily adapted to a wide range of experimental needs. We made some minor modifications to the original code that are necessary for lengthy recordings (&gt;10 min), which is available in GitHub link below. For shorter recordings, we recommend using the original code.</div>
<p>The parameters that mainly affect CNMF-E results are the minimum local correlation of a pixel and its neighbors (<a href="#fig11">Figure 11</a>A, left; noted as “min_corr”) and the minimum PNR of a pixel (<a href="#fig11">Figure 11</a>A, right; noted as “min_pnr”). These parameters are thresholds used to obtain the first estimation of the spatial and temporal components that will be used to initialize the CNMF algorithm. For sparse data, such as that from ABNs, total video length may affect the estimation of CORR. This happens because in one-photon imaging, pixels associated with a neuron are highly correlated only when that neuron is active. Given that ABNs are usually inactive, increasing the video length may be translated into a poor CORR, especially for files that are several hours long.</p>
<figure id="fig11"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/340-Fig11.jpg" alt="Fig11.jpg">
<figcaption>
<div class="figcaption-title">Figure 11. Analysis of Ca<sup>2+</sup> imaging data</div>
<p>Overlapping batch analysis.</p>
<p>(A) Local correlation (CORR) and peak-to-noise ratio (PNR) image of three neurons in a recording file.</p>
<p>(B) Ca<sup>2+</sup> transients detected and CORR for three neurons in three consecutive batches. Black trace shows the final estimated signal for the entire recording.</p>
<p>(C) Normalized CORR for a 3-h video obtained by analyzing the entire video sequence versus the maximum projection of the CORR images obtained by overlapping batch implementation. Note that CORR of the entire video sequence includes many highly correlated non-circular shapes that are unlikely to represent real ABNs. Analysis of the entire video will introduce several false positives and noisier estimation of Ca<sup>2+</sup> transients in this case.</p>
<p>(D) Spatial and temporal features of Ca<sup>2+</sup> transients extracted by conventional CNMF-E (CC) and overlapping batches (OB) methods. The two transients were randomly chosen from those extracted by CC or OB methods.</p>
<p>(E) Circularities (isoperimetric quotient) of spatial components and PNR of temporal traces.</p>
<p>(F) Example of spatial components extracted from an individual batch (i), the OB method (ii), or the CC method (ii) in the same time period. Because there are more neurons in the entire recording period than in an individual batch, spatial components were weighted by average activity during the period.</p>
<p>(G) Left: Cosine similarity of spatial components for the individual batch versus CC method and individual batch versus OB method. The star symbol in the figure corresponds to the spatial components shown in (F). Right: Temporal correlation of Ca<sup>2+</sup> transients of common neurons for the individual batch versus CC method and individual batch versus OB method. A neuron extracted by the CC or OB method was considered to be the same as that in an individual batch if its spatial component had a cosine similarity &gt;0.8. If more than two pairs of neurons satisfied this condition, the pair with the higher temporal correlation was used for calculations.</p>
<p>(H) Left: percentage of active neurons not detected in an individual batch. Right: PNR of missed neurons.</p>
<p>Data were analyzed by Mann-Whitney tests (E, G) and one-sample t tests (H).</p>
</figcaption>
</figure>
<p>To solve this problem, we segment the video in short overlapping batches of 1,000 frames, which allows us to estimate the spatial component of ABNs from discrete temporal windows in which they are highly active. This is depicted in <a href="#fig11">Figure 11</a>B, in which the CORR and Ca<sup>2+</sup> transients of three close ABNs are shown for three consecutive batches. Note that ABNs display prominent Ca<sup>2+</sup> transients that translate into a higher CORR, particularly in the third batch. Because the spatial component of each neuron is shared across batches, only this batch is necessary to initialize those ABNs, even if other batches have CORR comparable to the background level. Having obtained a good initial estimation of spatial components, the CNMF algorithm can extract temporal traces from other batches, even if its CORR or PNR is below the defined threshold. An overlapping batch approach produces a cleaner estimation of CORR, in contrast to the analysis of the entire video sequence (<a href="#fig11">Figure 11</a>C).</p>
<p>The multi-batch algorithm is implemented in the original CNMF-E article (<a href="#bib21">Zhou et al., 2018</a>); however, we find that in some cases, artifacts are introduced in the concatenation point between batches. Thus, we implement a multi-batch algorithm using overlapping batches. We also include some utility functions to correct for variations in baseline noise between batches and some minor bugs producing empty matrices given the sparse activity of ABNs. Our code contains optimized parameters that can extract Ca<sup>2+</sup> transients from ABNs in most cases. To illustrate the differences between the overlapping batches (OB) method and the conventional CNMF-E (CC) method (i.e., running CNMF-E on the whole video sequence), we compared the spatial and temporal components extracted by both methods (<a href="#fig11">Figure 11</a>D). True-positive neurons are expected to display circular shapes and Ca<sup>2+</sup> transients distinguishable from noise. Neurons extracted by the OB method display higher circularity and PNR than those extracted by the CC method (<a href="#fig11">Figure 11</a>E). Indeed, several neurons extracted by the CC method display spatial components that largely differ from a circular shape. Considering those spatial components with circularities five standard deviations below those extracted by the OB method, we estimated that 34% of components extracted by the CC method are false positives (<a href="#fig11">Figure 11</a>E, left, red dots). Next, we examined how the final temporal and spatial components of the transients extracted by the OB or CC method (analysis of &gt;30,000 frames) differ from those extracted from the analysis of an individual batch (analysis of ∼1,000 frames) (<a href="#fig11">Figure 11</a>F). Components extracted by the OB method were spatially and temporally more similar to those extracted from an individual batch (<a href="#fig11">Figure 11</a>G). We estimate that, on average, 61% of neurons remain undetected in individual batch analysis compared with the OB method (<a href="#fig11">Figure 11</a>H, left) because they display PNR significantly below the detection threshold. These results indicate that the OB method minimizes the number of false positives without compromising true-positive detection.</p>
<ol start="6">
<li>To track ABNs across different sessions, motion correct each session as previously described (<a href="#bib4">Ghandour et al., 2019</a>). In Mosaic software, this is done as follows:
<ol type="a">
<li>Motion correct each session independently.</li>
<li>Extract one frame from one session to use as a reference for other sessions.</li>
<li>Motion correct each session relative to the extracted frame.
<div class="note">
<span class="note-title">Note:</span> Sometimes using different reference frames from different sessions provides better results. Consider the points mentioned in “<a href="#quantification-and-statistical-analysis">Quantification and statistical analysis</a>: step 3.”</div>
</li>
<li>Concatenate the recording sessions, ensuring that sessions are correctly aligned.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Tracing ABNs across several days using tracking algorithms (<a href="#bib20">Sheintuch et al., 2017</a>) may not be possible given the small number of ABNs. However, we were able to track ABNs across different recording sessions within the same day by concatenating different sessions into one file. Spatial markers, such as blood vessels or ABNs with persistent activity, can be used to align different sessions.</div>
<p>Analyzing sessions independently and then identifying the same ABNs based on their footprints assumes that all ABNs can be detected in each recording session. However, this is not always the case, especially considering the sparse activity of ABNs and the low signal-to-noise ratio of GCaMP3. This issue is illustrated in <a href="#fig11">Figure 11</a>B; note that in the first batch, three ABNs are active with CORR comparable to the background local correlation. If lower CORR and PNR thresholds are used to detect these ABNs, many false-positive neurons would be included in the analysis, further hindering the tracking of ABNs. Instead, we initialize these ABNs in a specific period in which they show high activity (e.g., third batch in <a href="#fig11">Figure 11</a>B). The CNMF algorithm is then used to extract Ca<sup>2+</sup> transients from other batches with a lower signal-to-noise ratio. This is particularly important when tracking the same ABNs because some may be more active in some recording sessions than in others and, hence, be wrongly labeled as inactive if these sessions are analyzed independently.</p>
<ol start="7">
<li>After extracting the raster plot of ABN activity, we further analyzed data for each sleep stage in 10-s bins using Sleep Sign software (example raw data are available in <a href="#bib13">Kumar et al., 2020</a>). Alternatively, several other types of software are available for sleep analysis (e.g., Sleep, <a href="#bib2">Combrisson et al., 2017</a>). More details of sleep recording and analysis are available in <a href="#bib17">Oishi et al. (2016)</a>.</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Although GCaMP imaging and miniscope technologies fuel new discoveries of how neuron activity generates specific behaviors in freely moving animals, their limitations are well known.</p>
<p>One limitation is related to the resolution of the z-axis. The miniscope used in our study (<a href="#bib13">Kumar et al., 2020</a>) excites fluorescence molecules without focusing or scanning the light; it does not have a pinhole. Therefore, all fluorescence molecule hundreds of micrometers below the miniscope can be excited and detected at the same time. This can allow researchers to obtain a larger number of ABN signals. However, it is possible that a detected Ca<sup>2+</sup> transient can stem from two ABNs aligned in the z-axis.</p>
<p>Another limitation is that the experimental preparation for DG imaging in freely moving mice results in a partial lesion of the ipsilateral CA1 area when implanting the GRIN lens. Less invasive techniques could be used in the future to prevent possible circuit reorganization induced by surgery, as previously recommended (<a href="#bib8">Hainmueller and Bartos, 2018</a>; <a href="#bib11">Kirschen et al., 2017</a>; <a href="#bib18">Pilz et al., 2016</a>).</p>
<p>Finally, the developing nature of ABNs imposes a time constraint on the experimental schedule. As the activity and behavioral significance of ABNs change as they transition from immature to mature granule cells (e.g., <a href="#bib13">Kumar et al., 2020</a>), it can be challenging to use the same mice in multiple behavioral tasks and ensure that the recorded ABNs have the same physiological properties across the task periods.</p>
<p>A final limitation is the overall cost of the necessary equipment. Using the described hardware makes the experiment faster and easier to perform, but it costs more than $150,000 USD in total. Fortunately, more budget-friendly options are available. A similar miniscope can be built by a researcher using the open-source miniscope platform (UCLA Miniscope). In addition, simpler over-the-counter computers can be used for the experiments, but this would likely increase the data processing time.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>GCaMP3-labeled cells are rare or absent (steps 1, 2, and 31).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>The presence of a small number of GCaMP3-labeled neurons can be due to inefficient Cre induction/recombination by tamoxifen administration. The dose-dependency of GCaMP3 expression should be confirmed in each experimental setting. Always prepare fresh tamoxifen solution, protecting it from light. Ensure that tamoxifen is completely dissolved to achieve a sufficient dose. Also, when injecting, leave the needle tip in the intraperitoneal space for a few seconds to avoid reflux of tamoxifen. The injected dose can be systematically increased to determine whether this solves the problem.</p>
<p>When using different versions of GCaMP, confirm immunohistologically that Cre recombinase is expressed and that there is no cell death induced by cytotoxicity of Ca<sup>2+</sup> sensors in ABNs.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The lens is incorrectly positioned (steps 4–9).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This problem can occur due to variations in different stereotaxic equipment or unintended changes in lens position during implantation. Confirm that the lens is oriented precisely within the hole and re-evaluate the coordinates for each experimental setting.</p>
<p>Implanting the lens into the CA1 generates resistance, which can force the lens out of the brain. Only release the lens from the stereotaxic micromanipulator after the glue/dental cement cures; otherwise, the position of the lens may shift when released from the grip.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The implant detaches from the mouse (steps 6–28).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Implant detachment can occur when it is not well secured to the skull. When applying several layers of different adhesives, ensure that each layer completely cures before applying the next one, as mixing them could compromise their stability. In some cases, standard black cement may compromise the longevity of the implant. One possibility for this case is to use regular light dental cement and cover it with black paint or nail polish to prevent ambient light-derived artifacts. Infection and tissue inflammation can also decrease the adhesive strength of the glue/dental cement; ensure adequate prophylactic practices during surgery and postoperative recovery to prevent this problem. In addition, do not immobilize mice by holding the baseplate, as physical stress at the implant can cause its detachment.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Excessive motion artifacts are present during data acquisition (steps 12–14, 19, 24, and 27).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Ensure that the baseplate is fixed properly on the skull and that the miniscope is correctly attached to the baseplate and tightly secured in place by its screw. Also, low frame rates may increase the amount of motion blur, which can impair performance of the motion correction algorithm.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Fluorescence signals do not show dynamic changes in intensity (steps 13, 20, 24, 27).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>“False” fluorescence signals are usually derived from autofluorescent materials (e.g., damaged tissue or blood cells below the lens) and not from GCaMP3. To prevent these, ensure that the cortex above the target coordinates is completely aspirated and that there is no residual bleeding before implanting the lens. As lowering the lens too quickly into the brain can damage the tissue, ensure that the lens is implanted slowly and allow the tissue to settle after each 150-μm step. If necessary, retract the lens ±25 μm before each step to further alleviate pressure and prevent unnecessary damage.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Data processing is extremely slow or impossible (<a href="#quantification-and-statistical-analysis">Quantification and statistical analysis</a> steps).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>As sleep recordings can be &gt;3 h long, the final compressed file can be &gt;200 GB. To be processed, the file will need to be decompressed and spatially down-sampled (4×), resulting in a ∼216 kb/frame file. However, because some MATLAB scripts require double precision, we recommend that the PC has at least four times more RAM than the down-sampled file size. When it is not possible to reduce the size of the data file by temporal and/or spatial binning, the full-length recording can be split into two shorter files. The initial processing steps can be performed separately for each file, which can later be concatenated before detecting active ABNs using batch implementation and disabling parallel computing (although this can take several hours).</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Few or no ABNs are present in the imaging file (<a href="#quantification-and-statistical-analysis">Quantification and statistical analysis</a> step 4).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>If there are no problems with GCaMP3 expression or lens implantation coordinates, this problem can be solved by changing parameters in the CNMF-E script. Start by modifying the minimum CORR and PNR thresholds. In this case, visual inspection of cell shape and temporal dynamics of Ca<sup>2+</sup> transients become more important, as the rate of false cell detection increases with less stringent parameters.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>MATLAB code crashes during analysis (<a href="#quantification-and-statistical-analysis">Quantification and statistical analysis</a> steps 4–6).</p>
<h3 id="sec6.16">Potential solution</h3>
<p>The code (original or provided) is not bug-proof. In case of an error, read the error message in the MATLAB command window. If no ABN is detected, the code will crash. Visually confirm whether you can detect Ca<sup>2+</sup> activity. If active ABNs can be visually detected in the raw file, consider using a lower PNR or CORR threshold. Also, monitor your RAM usage when running the code. Consider upgrading your hardware or reducing the recording duration.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Boyce, R., Glasgow, S.D., Williams, S., and Adamantidis, A. (2016). Causal evidence for the role of REM sleep theta rhythm in contextual memory consolidation. Science <i>352</i>, 812-816.</p>
<p id="bib2">Combrisson, E., Vallat, R., Eichenlaub, J.B., O'Reilly, C., Lajnef, T., Guillot, A., Ruby, P.M., and Jerbi, K. (2017). Sleep: an open-source python software for visualization, analysis, and staging of sleep data. Front. Neuroinform. <i>11</i>, 60.</p>
<p id="bib3">Danielson, N.B.B., Kaifosh, P., Zaremba, J.D.D., Lovett-Barron, M., Tsai, J., Denny, C.A.A., Balough, E.M.M., Goldberg, A.R.R., Drew, L.J.J., Hen, R., et al. (2016). Distinct contribution of adult-born hippocampal granule cells to context encoding. Neuron <i>90</i>, 101-112.</p>
<p id="bib4">Ghandour, K., Ohkawa, N., Fung, C.C.A.C.A., Asai, H., Saitoh, Y., Takekawa, T., Okubo-Suzuki, R., Soya, S., Nishizono, H., Matsuo, M., et al. (2019). Orchestrated ensemble activities constitute a hippocampal memory engram. Nat. Commun. <i>10</i>, 2637.</p>
<p id="bib5">Ghosh, K.K., Burns, L.D., Cocker, E.D., Nimmerjahn, A., Ziv, Y., Gamal, A.E., and Schnitzer, M.J. (2011). Miniaturized integration of a fluorescence microscope. Nat. Methods <i>8</i>, 871-878.</p>
<p id="bib6">Giovannucci, A., Friedrich, J., Gunn, P., Kalfon, J., Brown, B.L., Koay, S.A., Taxidis, J., Najafi, F., Gauthier, J.L., Zhou, P., Khakh, B.S., Tank, D.W., Chklovskii, D.B., and Pnevmatikakis, E.A. (2019). CaImAn an open-source tool for scalable calcium imaging data analysis. eLife <i>8</i>, e38173.</p>
<p id="bib7">Grewe, B.F., Gründemann, J., Kitch, L.J., Lecoq, J.A., Parker, J.G., Marshall, J.D., Larkin, M.C., Jercog, P.E., Grenier, F., Li, J.Z., Lüthi, A., and Schnitzer, M.J. (2017). Neural ensemble dynamics underlying a long-term associative memory. Nature <i>543</i>, 670-675.</p>
<p id="bib8">Hainmueller, T. and Bartos, M. (2018). Parallel emergence of stable and dynamic memory engrams in the hippocampus. Nature <i>558</i>, 292-296.</p>
<p id="bib9">Johnston, S.T., Parylak, S.L., Kim, S., Mac, N., Lim, C.K., Gallina, I.S., Bloyd, C.W., Newberry, A., Saavedra, C.D., Novák, O., et al. (2020). AAV ablates neurogenesis in the adult murine hippocampus. bioRxiv <a href="https://doi.org/10.1101/2020.01.18.911362">https://doi.org/10.1101/2020.01.18.911362</a></p>
<p id="bib10">Kim, E.J., Leung, C.T., Reed, R.R., and Johnson, J.E. (2007). In vivo analysis of Ascl1 defined progenitors reveals distinct developmental dynamics during adult neurogenesis and gliogenesis. J. Neurosci. <i>27</i>, 12764-12774.</p>
<p id="bib11">Kirschen, G.W.W., Shen, J., Tian, M., Schroeder, B., Wang, J., Man, G., Wu, S., and Ge, S. (2017). Active dentate granule cells encode experience to promote the addition of adult-born hippocampal neurons. J. Neurosci. <i>37</i>, 4661-4678.</p>
<p id="bib12">Kotterman, M.A., Vazin, T., and Schaffer, D.V. (2015). Enhanced selective gene delivery to neural stem cells in vivo by an adeno-associated viral variant. Development <i>142</i>, 1885-1892.</p>
<p id="bib13">Kumar, D., Koyanagi, I., Carrier-Ruiz, A., Vergara, P., Srinivasan, S., Sugaya, Y., Kasuya, M., Yu, T.S., Vogt, K.E., Muratani, M., et al. (2020). sparse activity of hippocampal adult-born neurons during REM sleep is necessary for memory consolidation. Neuron <i>107</i>, 552-565.</p>
<p id="bib14">Madisen, L., Garner, A.R., Shimaoka, D., Chuong, A.S., Klapoetke, N.C., Li, L., van der Bourg, A., Niino, Y., Egolf, L., Monetti, C., et al. (2015). Transgenic mice for intersectional targeting of neural sensors and effectors with high specificity and performance. Neuron <i>85</i>, 942-958.</p>
<p id="bib15">Marshall, L., Helgadóttir, H., Mölle, M., and Born, J. (2006). Boosting slow oscillations during sleep potentiates memory. Nature <i>444</i>, 610-613.</p>
<p id="bib16">Moreno-Jimenez, E.P., Flor-Garcia, M., Terreros-Roncal, J., Rabano, A., Cafini, F., Pallas-Bazarra, N., Avila, J., Llorens-Martin, M., Moreno-Jiménez, E.P., Flor-García, M., et al. (2019). Adult hippocampal neurogenesis is abundant in neurologically healthy subjects and drops sharply in patients with Alzheimer’s disease. Nat. Med. <i>25</i>, 554-560.</p>
<p id="bib17">Oishi, Y., Takata, Y., Taguchi, Y., Kohtoh, S., Urade, Y., and Lazarus, M. (2016). Polygraphic recording procedure for measuring sleep in mice. J. Vis. Exp. e53678.</p>
<p id="bib18">Pilz, G.A.A., Carta, S., Stäuble, A., Ayaz, A., Jessberger, S., Helmchen, F., Stauble, A., Ayaz, A., Jessberger, S., and Helmchen, F. (2016). Functional imaging of dentate granule cells in the adult mouse hippocampus. J. Neurosci. <i>36</i>, 7407-7414.</p>
<p id="bib19">Resendez, S.L., Jennings, J.H., Ung, R.L., Namboodiri, V.M., Zhou, Z.C., Otis, J.M., Nomura, H., McHenry, J.A., Kosyk, O., and Stuber, G.D. (2016). Visualization of cortical, subcortical and deep brain neural circuit dynamics during naturalistic mammalian behavior with head- mounted microscopes and chronically implanted lenses. Nat. Protoc. <i>11</i>, 566-597.</p>
<p id="bib20">Sheintuch, L., Rubin, A., Brande-Eilat, N., Geva, N., Sadeh, N., Pinchasof, O., and Ziv, Y. (2017). Tracking the same neurons across multiple days in Ca<sup>2+</sup> imaging data. Cell Rep. <i>21</i>, 1102-1115.</p>
<p id="bib22">Yang, S.M., Alvarez, D.D., and Schinder, A.F. (2015). Reliable Genetic Labeling of Adult-Born Dentate Granule Cells Using Ascl1 CreERT2 and Glast CreERT2 Murine Lines. J. Neurosci. <i>35</i>, 15379-15390.</p>
<p id="bib21">Zhou, P., Resendez, S.L.L., Rodriguez-Romaguera, J., Jimenez, J.C.C., Neufeld, S.Q.Q., Stuber, G.D.D., Hen, R., Kheirbek, M.A.A., Sabatini, B.L.L., Kass, R.E.E., et al. (2018). Efficient and accurate extraction of in vivo calcium signals from microendoscopic video data. eLife <i>7</i>, 1-37.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Masanori Sakaguchi (<a href="mailto:sakaguchi.masa.fp@alumni.tsukuba.ac.jp">sakaguchi.masa.fp@alumni.tsukuba.ac.jp</a>).</p>
<h4>Materials availability</h4>
<p>We did not generate any unique/stable reagents in this study.</p>
<h4>Data and code availability</h4>
<p>Data and code underlying the results described in this manuscript are available at:</p>
<p><a href="https://github.com/vergaloy/ABNs-Calcium-extraction">https://github.com/vergaloy/ABNs-Calcium-extraction</a></p>
<p>Also, the raw data from the related article (<a href="#bib13">Kumar et al., 2020</a>) are available at:</p>
<p><a href="https://data.mendeley.com/datasets/gfbdv5kfrz/1">https://data.mendeley.com/datasets/gfbdv5kfrz/1</a></p>
<h3>Acknowledgments</h3>
<p>We thank Dr. K.G. Akers for comments on the manuscript. This work was partially supported by grants from the World Premier International Research Center Initiative from <a href="https://doi.org/10.13039/501100001700">MEXT</a>, <a href="https://doi.org/10.13039/501100001695">JST</a> <a href="https://doi.org/10.13039/501100003382">CREST</a> grant no. JPMJCR1655, <a href="https://doi.org/10.13039/501100001691">JSPS</a> KAKENHI grant nos. 16K18359, 15F15408, 15H01276, 15K18332, 26115502, 25116530, JP16H06280, and 20H03552, <a href="https://doi.org/10.13039/100007449">Takeda Science Foundation</a>, Shimadzu Science Foundation, <a href="https://doi.org/10.13039/501100008880">Kanae Foundation</a>, <a href="https://doi.org/10.13039/501100012018">Research Foundation for Opto-Science and Technology</a>, <a href="https://doi.org/10.13039/501100003837">Ichiro Kanehara Foundation</a>, <a href="https://doi.org/10.13039/501100004051">Kato Memorial Bioscience Foundation</a>, <a href="https://doi.org/10.13039/100008695">Japan Foundation for Applied Enzymology</a>, <a href="https://doi.org/10.13039/501100008667">Senshin Medical Research Foundation</a>, <a href="https://doi.org/10.13039/501100012630">Life Science Foundation of Japan</a>, <a href="https://doi.org/10.13039/100008732">The Uehara Memorial Foundation</a>, Brain Science Foundation, <a href="https://doi.org/10.13039/501100004088">Kowa Life Science Foundation</a>, Inamori Research Grants Program, and GSK Japan to M.S.; <a href="https://doi.org/10.13039/501100001691">JSPS</a> KAKENHI grant nos. 25000015 and 18H04012 to M.K.; <a href="https://doi.org/10.13039/100011313">Tokyo Biochemical Research Foundation</a> and <a href="https://doi.org/10.13039/501100001691">JSPS</a> FPD to S.SR.; <a href="https://doi.org/10.13039/501100001691">JSPS</a> FPD to D.K.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, Y.S., M.K., and M.S.; Methodology, A.C.-R., Y.S., D.K., I.K., P.V., S.S., and M.S.; Investigation, A.C.-R., D.K., Y.S., I.K., S.S., and T.N.; Validation, A.C.-R, Y.S., D.K., and P.V.; Formal Analysis, Y.S., A.C.-R., and P.V.; Data Curation, A.C.-R., Y.S., P.V., and M.S.; Writing – Original Draft, A.C.-R and Y.S.; Writing – Review &amp; Editing, Y.S., D.K., A.C.-R, P.V., and M.S.; Funding Acquisition, M.S., M.K., and Y.S.; Resources, M.K., M.S., and T.S.; Supervision, M.K. and M.S.; Project Administration, Y.S., M.K., and M.S. All authors discussed and approved the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>