<article>
<section>
<h1 id="header">Detection of homolog-independent meiotic DNA repair events in <i>C. elegans</i> with the intersister/intrachromatid repair assay</h1>
<p><time datetime="2021-09-10">Published: September 10, 2021</time></p>
<p>Erik Toraason,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Marissa Glover,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Anna Horacek,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a></sup> and Diana E. Libuda<sup><a href="#aff1">1</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Molecular Biology, Department of Biology, University of Oregon, 1229 Franklin Boulevard, Eugene, OR 97403, USA</p>
<p id="aff2"><sup>2</sup>Department of Molecular, Cell, and Developmental Biology, University of California Santa Cruz, Sinsheimer Labs, 1156 High Street, Santa Cruz, CA 95064, USA</p>
<p id="aff3"><sup>3</sup>Section on Gene Expression, Eunice Kennedy Shriver National Institute of Child Health and Human Development, P.O. Box 3006, Rockville, MD 20847, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:etoraaso@uoregon.edu">etoraaso@uoregon.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:dlibuda@uoregon.edu">dlibuda@uoregon.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100801">10.1016/j.xpro.2021.100801</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Accurate repair of DNA double-strand breaks (DSBs) in developing germ cells is critical to promote proper chromosome segregation and to maintain genome integrity. To directly detect homolog-independent (intersister/intrachromatid) meiotic DSB repair, we exploited the genetics and germline physiology of <i>C. elegans</i> to (1) induce a single DSB in nuclei across discrete stages of meiotic prophase I; (2) detect repair of that DSB as a homolog-independent crossover or noncrossover; and (3) sequence the resultant product to assess mechanisms of recombination.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00507-4/sref6">Toraason et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A single double-strand DNA break (DSB) can be induced in <i>C. elegans</i> by heat shock</li>
<li>Assay monitors repair of induced DSB with the sister chromatid or same DNA molecule</li>
<li>Frequency of crossovers and noncrossovers assessed for specific stages of meiosis I</li>
<li>Gene conversion tracts of repaired single DSBs assessed by sequencing</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/984-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The intersister/intrachromatid repair (ICR) assay enables: 1) the targeted induction of a single DSB within nuclei undergoing meiotic prophase I; and, 2) subsequent detection of repair of that DSB with the sister chromatid or same DNA molecule as indicated by GFP fluorescence in resultant progeny (<a href="#fig1">Figure 1</a>, see <a href="#fig1">Figure 1</a> Legend and <a href="#bib6">Toraason et al., 2021</a> for details). Importantly, the <i>C. elegans</i> germ line is organized in a spatial temporal gradient and meiocytes progress both through the germ line and meiotic stages at a known rate, enabling a reverse time course protocol to be performed to assess how DNA repair templates are engaged to repair DSBs induced within specific meiotic stages. In wild-type worms, progeny laid 10–22 h post heat shock are derived from a ‘non-interhomolog window’ at the time of DSB induction, corresponding to late pachytene and diplotene, when the homologous chromosome cannot be engaged to repair DSBs (<a href="#bib6">Toraason et al., 2021</a>, <a href="#bib4">Rosu et al., 2011</a>). Progeny laid 22–58 h following heat stress were within the ‘interhomolog window’, corresponding to meiotic entry through mid/late pachytene, when the homologous chromosome is available as a repair template. Assessing the frequency of ICR assay repair events enables insights into engagement of specific templates for DNA repair, and sequencing of the recombinant loci generated during repair enables assessment of mechanisms of meiotic recombination.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/984-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The ICR Assay enables detection of homolog-independent meiotic recombination events</div>
<p>(A) The ICR Assay is composed of tandem null GFP sequences integrated in exon 6 of <i>unc-5</i> on Chromosome IV. The upstream GFP is driven by a <i>myo-3</i> body wall muscle specific promoter and is truncated, while the downstream GFP is driven by a pharynx specific promoter and is interrupted by a Mos1 transposon sequence. Following heat-shock induction of Mos1 transposase expression, Mos1 excision in the germ line will generate a single DSB (<a href="#bib3">Robert and Bessereau, 2007</a>; <a href="#bib4">Rosu et al., 2011</a>). If the damaged chromatid engages the nonallelic GFP template either on its sister or within the same chromatid for repair either as a noncrossover or crossover, the final repair product will code for a functional GFP protein. The tissue specificity of the GFP will be determined by the repair pathway engaged; a crossover will yield GFP expressed only in the body wall muscle. However, a noncrossover will yield GFP expressed either only in the pharynx, or in the pharynx and body wall muscle due to long-range promoter interactions between the <i>myo-2</i> and <i>myo-3</i> promoter sequences (<a href="#bib2">Okkema et al., 1993</a>).</p>
<p>(B) The tandem GFP sequences in the ICR assay contain polymorphisms, enabling the generation of conversion tracts in recombination events. Diagrams in this figure were repurposed from (<a href="#bib6">Toraason et al., 2021</a>).</p>
</figcaption>
</figure>
<ol>
<li>Calibrate air incubators to 15°C, 20°C, 25°C, and 34°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend the use of an internal thermometer to ensure accuracy. Heat shock using alternative heat sources, such as a water bath, may produce frequencies differing from published results (<a href="#bib6">Toraason et al., 2021</a>).</div>
<ol start="2">
<li>Pour 4 Nematode Growth Media (NGM) plates for each parent hermaphrodite you intend to score in the assay and seed with OP50 <i>E. coli</i> in a small lawn.
<ol type="a">
<li>We recommend 35<b>×</b>10 mm plates be used to facilitate efficient scoring of recombinant progeny, seeded with a single drop from a P1000 pipette in the center of the plate such that the OP50 does not touch the plate walls.</li>
<li>To prevent bacterial lawn overgrowth, keep seeded plates at room temperature (∼18°C–22°C) overnight (∼16–20 h) and then maintain them at 4°C until use. If plates are kept refrigerated, then they should be allowed to return to room temperature (∼18°C–22°C) before use.</li>
</ol>
</li>
<li>If the ICR assay (<a href="#fig1">Figure 1</a>) is being performed in a mutant background to assess the impact of that mutation on intersister/intrachromatid repair, generate three strains (strains #1–3, see below). Details on maintaining <i>C.elegans</i> strains are thoroughly described in <a href="#bib5">Stiernagel, 2006</a>.
<ol type="a">
<li>Strain #1 (a mating stock [a <i>C. elegans</i> culture maintained with both male and hermaphrodite animals] of the strain carrying the mutation)</li>
<li>Strain #2 (a hermaphrodite stock which both; A) carries the mutation of interest; and, B) is homozygous for the ICR assay construct <i>unc-5(lib1)</i> on Chromosome IV and the Mos1 transposase transgene KrIs14 on Chromosome V). Generate strain #2 by crossing the mutation of interest to strain DLW14 (<a href="#key-resources-table">key resources table</a>))</li>
<li>Strain #3 (a hermaphrodite stock carrying the mutation of interest which is also homozygous for the <i>unc-5(e791)</i> allele on Chromosome IV). Generate strain #3 by crossing the mutation of interest to strain CB791 (<a href="#key-resources-table">Key resources table</a>))</li>
</ol>
</li>
<li>If it is unknown whether the rate of meiotic progression in the <i>C. elegans</i> germ line is altered in a mutant of interest, then oocyte progression through the germ line should be assessed by EdU labeling or ovulation rate (<a href="#bib1">Jaramillo-Lambert et al., 2007</a>; <a href="#bib4">Rosu et al., 2011</a>; <a href="#bib6">Toraason et al., 2021</a>).
<ol type="a">
<li>If the mutant in question does exhibit altered meiotic progression, then timepoints should be adjusted such that the first timepoint following heat shock encompasses only progeny which were derived from oocytes in late Pachytene-Diplotene at the time of heat shock.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The protocol described below is for the ICR assay performed in a wild-type background. However, if there is an additional mutation which is introduced that is stable as a homozygote, an identical cross scheme as to the one described below can be performed using strains which are homozygous for the mutation of interest. If the assay is to be done in a mutant context which needs to be maintained as a heterozygote, then the balancer chromosome or genetic markers used to track that mutation will need to be accounted for in the strain construction to ensure that the parent hermaphrodites used in the ICR assay are homozygous for the mutation of interest and heterozygous for both the <i>unc-5(lib1)</i> ICR assay construct and KrIs14 transgene. If an allele of interest is located on the X chromosome and conveys serious deleterious phenotypes, the ICR assay may not be amenable to test its function in intersister/intrachromatid repair, as the mutation will be carried hemizygous in the male during crossing. Further, we envision that our protocol can be amenable to be performed with RNAi treatments to knock down specific gene products, but we have not yet tested out the specific parameters for using the protocol in conjunction with RNAi-based knockdown methods.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>OP50 <i>Escherichia coli</i>
</td>
<td>CGC</td>
<td>OP50</td>
</tr>
<tr>
<td>TOP10 <i>Escherichia coli</i> chemically competent cells</td>
<td>Invitrogen</td>
<td>C4040-06</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>≥99.8% pure Tris base (Tris[hydroxymethyl]aminomethane or Trimethamine)</td>
<td>Bio-Rad</td>
<td>Cat#1610716EDU; CAS 77-86-1</td>
</tr>
<tr>
<td>Dimethylsulfoxide ≥99.9% (DMSO)</td>
<td>VWR</td>
<td>97063-136; CAS 67-68-5</td>
</tr>
<tr>
<td>GeneRuler 1 kb Ladder</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#SM0311</td>
</tr>
<tr>
<td>Hydrochloric Acid, Certified ACS Plus, 36.5–38.0% (HCl)</td>
<td>Thermo Fisher Scientific</td>
<td>40233; CAS 7647-01-0</td>
</tr>
<tr>
<td>IGEPAL® CA-630</td>
<td>Sigma-Aldrich</td>
<td>Cat#I8896; CAS 9002-93-1</td>
</tr>
<tr>
<td>Magnesium chloride (MgCl2)</td>
<td>Sigma-Aldrich</td>
<td>Cat#M8266; CAS 7786-30-3</td>
</tr>
<tr>
<td>OneTaq Quick-Load 2<b>×</b> Master Mix w/ Standard Buffer</td>
<td>New England Biolabs</td>
<td>Cat#M0486</td>
</tr>
<tr>
<td>Potassium chloride (KCl)</td>
<td>VWR</td>
<td>Cat#MK6858-04; CAS 7447-40-7</td>
</tr>
<tr>
<td>Proteinase K, Molecular Biology Grade</td>
<td>New England Biolabs</td>
<td>Cat#P8107S</td>
</tr>
<tr>
<td>Tween® 20</td>
<td>Sigma-Aldrich</td>
<td>Cat#P9416; CAS 9005-64-5</td>
</tr>
<tr>
<td>CO<sub>2</sub>, solid</td>
<td>Airgas</td>
<td>CAS 124-38-9</td>
</tr>
<tr>
<td>Ethanol 190 Proof</td>
<td>Fisher Scientific</td>
<td>CAS 64-17-5</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain CB791 (<i>unc-5(e791)</i> IV)</td>
<td>Caenorhabditis Genetics Center</td>
<td>CB791</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain DLW14 (<i>unc-5(lib1</i> [intersister/intrachromatid repair assay <i>Pmyo-3</i>::GFP(-) + <i>unc-119(+)</i> + <i>Pmyo-2::GFP(Mos1)</i>]) IV; <i>KrIs14(Phsp-16.48</i>::MosTransposase; <i>lin-15B</i>; <i>Punc-122</i>::GFP) V)</td>
<td>Caenorhabditis Genetics Center</td>
<td>DLW14</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain EN909 (<i>KrIs14(Phsp-16.48</i>::MosTransposase; <i>lin-15B</i>; <i>Punc-122</i>::GFP) V)</td>
<td>Caenorhabditis Genetics Center</td>
<td>EN909</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain N2 (wild-type)</td>
<td>Caenorhabditis Genetics Center</td>
<td>N2</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>DLO546 (5′-AGTTGGTAATGGTAGCGACC-3′)</td>
<td>Integrated DNA Technologies (IDT)</td>
<td>DLO546</td>
</tr>
<tr>
<td>DLO822 (5′-ATTTTAACCCTCGGGGTACG-3′)</td>
<td>Integrated DNA Technologies (IDT)</td>
<td>DLO822</td>
</tr>
<tr>
<td>DLO823 (5′-TCCATGCCATGTGTAATCCCA-3′)</td>
<td>Integrated DNA Technologies (IDT)</td>
<td>DLO823</td>
</tr>
<tr>
<td>DLO824 (5′-AGATCCATCTAGAAATGCCGGT-3′)</td>
<td>Integrated DNA Technologies (IDT)</td>
<td>DLO824</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Benchling Align Sequences Tool</td>
<td>Benchling</td>
<td><a href="https://help.benchling.com/en/">https://help.benchling.com/en/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>QIAGEN</td>
<td>28104</td>
</tr>
<tr>
<td>GeneJET Gel Extraction Kit</td>
<td>Thermo Fisher</td>
<td>R1341</td>
</tr>
<tr>
<td>TOPO-TA Cloning Kit</td>
<td>Thermo Fisher</td>
<td>K2040-40</td>
</tr>
<tr>
<td>Axio Zoom.V16 fluorescent dissection microscope (PlanNeoFluar Z 1<b>×</b>/0.25 FWD 56 mm lens paired with Pl 10x/23 eye pieces for up to 112<b>×</b> magnification and Zeiss CL 6000 LED cold-light source illumination)</td>
<td>Zeiss</td>
<td>Axio Zoom.V16</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Worm lysis buffer (stored at −20°C for up to 1 year)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KCl (2.5 M)</td>
<td>50 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Tris pH8.2 (1 M)</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>2.5 mM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>IGEPAL® (10%)</td>
<td>0.45%</td>
<td>45 μL</td>
</tr>
<tr>
<td>Tween20® (10%)</td>
<td>0.45%</td>
<td>45 μL</td>
</tr>
<tr>
<td>Proteinase K (20 mg/mL)</td>
<td>0.03 mg/mL</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>877.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> KCl, Tris, MgCl<sub>2</sub>, IGEPAL, Tween20, and Proteinase K are irritants. Wear gloves when handling these reagents.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Generate ICR assay parent hermaphrodites</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 9 days</div>
<ol>
<li>Cross L4 larval stage male worms from Strain #1 to L4 hermaphrodite worms from Strain #2.
<ol type="a">
<li>Set up a total of 2–4 cross plates each with 2 males mated to 7–10 hermaphrodites.</li>
<li>Maintain plates in a 20°C incubator.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> As all crosses performed in this protocol utilize males which are wild-type moving (non-Unc) and hermaphrodites with defective movement phenotypes (Unc), it is important that an appropriate ratio of males to hermaphrodites be set up to prevent the males from killing the hermaphrodites during mating. In our hands, all crosses between wild-type moving males and Unc hermaphrodites are best performed on 60<b>×</b>15 mm NGM plates seeded with a large lawn of OP50 (∼3–4 drops from a 25 mL serological pipette in the center of the plate without touching the plate edges and allowed to fully dry). For additional security, the hermaphrodites may be placed equidistant around the edge of the OP50.</div>
<ol start="2">
<li>Four days after setting up the cross between Strain #1 and Strain #2, pick wild-type moving (non-Unc) coelomocyte GFP positive L4 male F1 progeny from the crosses set up in step 1 to L4 hermaphrodites from Strain #3.
<ol type="a">
<li>Set up a total of 10–15 cross plates each with 2 males to 7–10 hermaphrodites.</li>
<li>Maintain these new cross plates at 20°C and discard the cross plates from step 1.</li>
</ol>
</li>
<li>Four days after setting up the cross between F1 progeny from cross 1 and hermaphrodites from Strain #3, isolate L4 coelomocyte GFP+ Unc hermaphrodite progeny from the cross plates set up in step 2 by picking them onto NGM plates seeded with OP50. These progeny carry the correct genotype for the ICR assay (<a href="#fig1">Figure 1</a>). We recommend that 30–40 parent hermaphrodites be used in each replicate of the assay.
<ol type="a">
<li>Place the ICR assay hermaphrodites at 15°C until the time of heat shock (Step 4) and discard the mating plates set up in step 2.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The ICR assay L4 hermaphrodites in step 3 should only be picked within a two-hour window 16–18 hours before the desired heat shock time. Maintenance of the hermaphrodites to be used in the ICR assay at 15°C ensures that the hermaphrodites will just have entered reproductive adulthood at the time of heat shock. If a mutant is being used which alters the timeline of developmental progression, then adjust the window in which hermaphrodites are being picked before heat shock to stage them similarly.</div>
<h3 id="sec3.2">Perform the intersister/intrachromatid repair assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 days</div>
<div class="note">
<span class="note-title">Note:</span> A graphical outline of the ICR assay is depicted in <a href="#fig2">Figure 2</a>.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/984-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Graphical timeline of the ICR assay</div>
<p>Presented is a timeline for a hypothetical ICR assay wherein hermaphrodites are being transferred at 8am and 8pm. All times should be adjusted accordingly if a different transfer time is desired. A detailed delination of specific steps is described in ‘perform the intrachromatid/intersister repair assay’ section (steps #4–16) of this document. Horizontal colored rows indicate incubators of the designated temperature at which worms should be incubated for a given step. Vertical solid lines designate the transition between days, while vertical dashed lines indicate 6 h periods. Horizontal arrows indicate that a plate should be transferred from one temperature to another, or maintained at a specific temperature.</p>
</figcaption>
</figure>
<ol start="4">
<li>16–18 h after isolating ICR assay parent hermaphrodites of the correct genotype (see step 3), place them in a pre-calibrated 34°C incubator for 1 h.</li>
<li>Following heat shock, place the hermaphrodites in a 20°C incubator.</li>
<li>10 h after heat shock was initiated, separate heat shocked parent hermaphrodites onto individual NGM plates seeded with OP50 (use the prepared 35 <b>×</b> 10 mm plates seeded with a single drop of OP50 from a P1000 pipette for this and all following timepoints) and maintain them in a 20°C incubator. Mark plates to indicate the timepoint post heat shock (10–22 h) and a unique ID for each parent hermaphrodite.</li>
<li>22 h after heat shock was initiated, transfer parent hermaphrodites from 10–22 h plates to new NGM plates seeded with OP50 (22–34 h plates). Maintain parent hermaphrodites (22–34 h plates) at 20°C and place progeny (10–22 h plates) at 15°C.</li>
<li>34 h after heat shock was initiated, transfer parent hermaphrodites from 22–34 h plates to new NGM plates seeded with OP50 (34–46 h plates). Maintain parent hermaphrodites (34–46 h plates) at 20°C and place progeny (22–34 h plates) at 15°C.</li>
<li>46 h after heat shock was initiated, transfer parent hermaphrodites from 34–46 h plates to new NGM plates seeded with OP50 (46–58 h plates). Maintain parent hermaphrodites (46–58 h plates) at 20°C and place progeny (34–46 h plates) at 15°C.</li>
<li>50–54 h after heat shock was initiated, place 10–22 h plates at 25°C.</li>
<li>58 h after heat shock was initiated, remove parent hermaphrodites from 46–58 h plates and discard them. Place progeny (46–58 h plates) at 15°C.</li>
<li>3 days (∼72 h) after heat shock was initiated, score 10–22 h plates for recombinant progeny (<a href="#fig1">Figures 1</a>A and <a href="#fig3">3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/984-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. ICR assay germline and somatic recombinant phenotypes</div>
<p>Meiotic and somatic recombinants arising from the ICR assay. All worms depicted are in late L4 or early adult stages of development. GFP fluorescence images of recombinant progeny derived from the ICR assay. Scale bars represent 200 μm. White asterisks mark the pharyngeal tip of each nematode and the shape of each worm is delineated by dashed lines. Small blue arrowheads indicate fluorescent coelomocytes, marking that the worm is carrying the Mos1 transposase transgene, and therefore do not indicate a recombinant tissue. Partial GFP fluorescence arising from somatic recombination is marked with large blue arrowheads.</p>
</figcaption>
</figure>
<ol type="a">
<li>Noncrossover meiotic repair events are indicated by the presence of pharynx GFP expression that may or may not be accompanied by body wall GFP expression (<a href="#fig3">Figure 3</a>).</li>
<li>Crossover meiotic repair products are indicated by body wall GFP expression only (<a href="#fig3">Figure 3</a>).</li>
<li>Record the number of nonrecombinant progeny, noncrossover recombinant progeny (pharynx GFP), crossover recombinant progeny (body wall GFP only), unhatched eggs, and unfertilized oocytes laid by each parent hermaphrodite.</li>
<li>If recombinants are identified, then place them onto individual plates for future DNA extraction and sequencing of the recombination event.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Somatic Mos1 transposon excision and subsequent repair in the developing embryo may produce weak GFP expression in portions of the body wall or pharynx tissue (<a href="#fig3">Figure 3</a>). If a putative recombinant is ambiguous as to whether its phenotype is derived from a meiotic or somatic DSB repair event, then place that progeny individually onto an NGM plate seeded with OP50 and allow it to lay progeny. If it is a somatic recombinant, then the GFP phenotype of the parent will not be heritable.</div>
<ol start="13">
<li>After completing scoring of the progeny laid 10–22 h after heat shock, place progeny laid 22–34 and 34–46 h after heat shock at 25°C overnight (∼14–18 h).</li>
<li>4 days (∼96 h) after heat shock was initiated, score 22–34 h and 34–46 h plates for recombinant progeny, as described in step 12.</li>
<li>After completing scoring of the progeny laid 22–34 and 34–46 h after heat shock, place progeny laid 46–58 h after heat shock at 25°C overnight (∼14–18 h).</li>
<li>5 days (∼120 h) after heat shock was initiated, score 46–58 h plates for recombinant progeny, as described in step 12.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> At the time of scoring, no progeny should be reproductively mature, as it is possible that that progeny’s offspring may be included in the progeny counts. Thus, if a plate has one or more progeny older than L4 stage and there are worms on the plate that are also L1/L2 stage, then the plate should not be included in the dataset.</div>
<h3 id="sec3.3">Amplify and sequence ICR assay conversion tracts</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–5 days</div>
<ol start="17">
<li>Lyse individual crossover and noncrossover progeny to obtain genomic DNA. (note: the protocol below indicates the one used in <a href="#bib6">Toraason et al., 2021</a> but this step could be achieved by other <i>C. elegans</i> lysis methods)
<ol type="a">
<li>Place individual progeny in 10 μL of Worm Lysis Buffer.</li>
<li>Freeze thaw 3 times by iteratively placing samples in a dry ice bath with 95% EtOH and a 65°C water bath until the samples are completely frozen or thawed, respectively (∼5–10 s).</li>
<li>Incubate at 60°C for one hour and then at 95°C for 15 min.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples may be held at this step at 4°C before proceeding.</div>
</li>
<li>Dilute each sample by adding 10 μL of deionized H<sub>2</sub>O and store at −20°C.
<div class="critical">
<span class="critical-title">Critical:</span> Direct contact with solid CO<sub>2</sub> can cause injury. Use caution when handling the dry ice bath.</div>
</li>
</ol>
</li>
</ol>
<ol start="18">
<li>PCR amplify noncrossover recombinants using the primers DLO822 and DLO823 (<a href="#key-resources-table">Key resources table</a>, <a href="#fig4">Figure 4</a>A, Primer set A) and crossover recombinants using primers DLO824 and DLO546 (<a href="#key-resources-table">Key resources table</a>, <a href="#fig4">Figure 4</a>B, Primer set B). In our hands, 200 μL of PCR reaction using OneTaq 2xMaster Mix (NEB) and 2% DMSO generated sufficient specific product for sequencing.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/984-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. PCR amplification of ICR assay recombinant loci</div>
<p>(A) PCR primer design schematic for amplification specifically of ICR assay crossover and noncrossover recombinant loci. Primer set ‘a’ is composed of forward primer DLO822 and reverse primer DLO823, while Primer set ‘b’ is composed of forward primer DLO824 and reverse primer DLO546.</p>
<p>(B) PCR amplification of crossover and noncrossover products produces expected products from individual ICR assay recombinants. The dark band below 250 bp in all lanes indicates primer dimers. Gel image and graphic images from this figure are repurposed from (<a href="#bib6">Toraason et al., 2021</a>).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All PCR cycle conditions described in this protocol were optimized for these conditions and should be adapted if alternative polymerases are to be used.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Due to the repetitive structure of the ICR assay locus, inclusion of DMSO in the PCR reaction may improve reaction specificity.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> PCR reactions may be stored at −20°C.</div>
<table id="undtbl2">
<thead>
<tr>
<th colspan="4">PCR cycling conditions – Noncrossover amplification with DLO822 and DLO823</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td>1 cycle</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>53°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>68°C</td>
<td>5 min</td>
<td>1 cycle</td>
</tr>
<tr>
<td>Hold</td>
<td>10°C</td>
<td>Forever</td>
<td> </td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<thead>
<tr>
<th colspan="4">PCR cycling conditions – Crossover amplification with DLO824 and DLO546</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td>1 cycle</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>62°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>68°C</td>
<td>5 min</td>
<td>1 cycle</td>
</tr>
<tr>
<td>Hold</td>
<td>10°C</td>
<td>Forever</td>
<td> </td>
</tr>
</tbody>
</table>
<ol start="19">
<li>Run 10 μL of the total PCR reaction volume for each lysate sample on an agarose gel using a DNA stain to determine reaction specificity. Both noncrossover and crossover reactions should yield a product of ∼950 bp (<a href="#fig4">Figure 4</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If significant off target bands are seen, then the PCR reaction should be further optimized. If small off target bands are observed but robust amplification of the desired target is seen, then run the remaining PCR reaction volume of that sample on a gel and excise the desired band. Purify the DNA amplicon by gel extraction and elute in 10–15 μL of ddH<sub>2</sub>O.</div>
<div class="note">
<span class="note-title">Note:</span> If no off-target products are observed in the PCR reaction, then concentrate the amplicon from the remaining reaction volume by PCR purification and elute in 10–15 μL of ddH<sub>2</sub>O.</div>
<ol start="20">
<li>Determine the sequence of the purified amplicon by Sanger sequencing. For high confidence coverage of the amplicon, we recommend using both DLO822 and DLO823 to sequence amplicons from noncrossover events and DLO824 and DLO546 to sequence amplicons from crossover events.</li>
<li>Compare sequenced amplicons to the reference sequence (provided upon request by the lead contact or accessed on the our lab website: <a href="http://www.libudalab.org">www.libudalab.org</a>) and annotate converted polymorphisms. Benchling alignment software was utilized for sequence comparisons in <a href="#bib6">Toraason et al., 2021</a>, but any sequencing alignment software will be sufficient for this purpose.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>In our hands, performing the ICR assay in a wild-type background produces GFP expressing meiotic recombinants at a rate of ∼0.7% of the total brood (<a href="#fig5">Figure 5</a>, <a href="#bib6">Toraason et al., 2021</a>). Mutants defective in intersister/intrachromatid repair may produce altered ratios of noncrossover and crossover outcomes, or reduced recombinant progeny. For example, <i>xpf-1</i> nuclease mutants produce ICR assay recombinants at ∼2 fold lower frequency than wild type (<a href="#bib6">Toraason et al., 2021</a>). It is possible that the specific recombination frequencies may change depending on lab specific conditions, especially if different maintenance conditions or heat shock methods are employed.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/984-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. ICR assay recombinant progeny counts</div>
<p>(A) Table of progeny counts arising from the wild-type ICR assays performed in (<a href="#bib6">Toraason et al., 2021</a>).</p>
<p>(B) Frequency of meiotic recombination events arising from the ICR assay. Error bars indicate 95% Binomial confidence intervals.</p>
</figcaption>
</figure>
<p>The majority of wild-type intersister/intrachromatid noncrossover tracts that we have sequenced (<a href="#bib6">Toraason et al., 2021</a>) exhibited nucleotide conversions only at the polymorphism most proximal to the Mos1 excision site (<a href="#fig1">Figures 1</a>B and <a href="#fig6">6</a>), but a minority were converted at all polymorphisms. Wild-type intersister/intrachromatid crossover tracts, however, displayed tract lengths greater than or equal to 96 bp (<a href="#bib6">Toraason et al., 2021</a>). Mutants which alter the processing of meiotic recombination intermediates may alter the size of conversion tracts. Alternately, defects in mismatch repair or multiple template engagement may produce ‘restoration tracts’, or unconverted polymorphisms flanked by converted polymorphisms. One such restoration tract was observed in <a href="#bib6">Toraason et al., 2021</a>.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/984-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Example of ICR assay conversion tract analysis</div>
<p>An example noncrossover recombinant derived from an ICR assay exhibiting conversions at the three most proximal polymorphisms to the DSB site. The tract was sequenced and aligned to a reference sequence using Benchling alignment software (above). A green check indicates a polymorphism converted to the upstream GFP repair template sequence, while a red ‘X’ indicates that a polymorphism was not converted. Arrows indicate the positions of these converted polymorphisms on a conversion tract plot (below). Vertical gray lines indicate the position of each ICR assay polymorphism, while the horizontal line represents a single conversion tract. Dark opacity points and lines connect contiguous converted polymorphisms and encompass the minimum tract length, while low opacity lines extend from the final converted polymorphism to the most proximal unconverted polymorphism and represent maximum conversion tract length.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The proportion of homolog-independent meiotic recombinant progeny within a timepoint is calculated as (GFP+ meiotic recombinant progeny/Total Live Progeny scored). Proportions of progeny which are recombinant can be compared between groups by using a standard Fisher’s exact test. In addition to assessing individual timepoints, it can be useful to broadly group timepoints by stages during which the homolog is available as a repair template (22–58 h post heat shock) and stages at which it is not (10–22 h post heat shock).</p>
<p>A useful metric for comparing conversion tracts is the minimum tract length, or the distance between the first and final polymorphism converted (<a href="#fig6">Figure 6</a>). In the ICR assay, the majority of noncrossover conversion tracts we observed were converted only at the most proximal polymorphism to the site of Mos1 transposon excision (<a href="#fig1">Figures 1</a>B and <a href="#fig6">6</a>). Researchers may also compare the proportions of “short” (converted at only one polymorphism) and “long” (converted at multiple polymorphisms) tracts between timepoints or genotypes by Fisher’s exact test.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>As the homolog-independent recombination events detected by the ICR assay are rare, the frequency of recombinants requires large numbers of progeny (ideally &gt;1000 total recombinant and nonrecombinant progeny per timepoint to be compared) for statistical confidence. Researchers should be cautious when planning experiments using the ICR assay which require a large number of pairwise comparisons and should precede these experiments with an appropriate statistical power analysis to determine the number of progeny to be scored for confidence in a negative result. For this same reason, mutations which have weak effects on the frequency of homolog-independent recombination may be challenging to detect in the ICR assay and we recommend performing a standard power analysis statistical test to determine how many worms may need to be assessed for statistically relevant result.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>A reduced number of recombinant progeny are observed at the 10–22 h timepoint (Step 12), and a large number of dead embryos are being laid at 22+ hour timepoints (step 13–16)</p>
<h3 id="sec7.2">Potential solution</h3>
<p>We found that the function of the ICR assay is not stable if the <i>unc-5(lib1)</i> ICR assay construct and KrIs14 Mos1 transposase transgene are inherited from the hermaphrodite parent. Reconstruct your cross scheme to ensure that both components are inherited from a male lineage (steps 1–3).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>A recombinant locus amplifies well for conversion tract analysis (steps 18–19), but the sequencing returns weak or messy results (step 21).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Inheritance of the <i>unc-5(lib1)</i> chromatid carrying an unexcised Mos1 allele from sperm may permit somatic excision and error-prone repair in the ICR assay, complicating accurate conversion tract identification by Sanger sequencing of PCR amplicons directly. We have been able to identify conversion tracts by TOPO cloning the purified PCR amplicon into a vector backbone, transforming the resultant plasmids into TOP-10 cells, and isolating single colonies. We recommend isolating plasmid products from &gt;10 of these colonies for sequencing to generate a consensus sequence of the gene conversion event.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The genetic background (e.g., a meiotic DNA repair mutant) in which the ICR assay is being performed lays few eggs or lays a high number of arrested embryos (steps 12, 14, 16), resulting in very few scorable progeny per parent hermaphrodite.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>To overcome fecundity and fertility defects innate to a genotype of interest, performing the ICR assay with multiple parent hermaphrodites per plate scored for recombinant progeny expedites the process of scoring for recombinants. We have performed the ICR assay with 2–3 parent hermaphrodites per plate without progeny overcrowding and consuming all available OP50 by the time of scoring. (steps 6–9).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>A fluorescent recombinant progeny which was isolated died before time of lysis (step 17).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>If the carcass of the recombinant is still present on the plate, place it into lysis buffer following the recommended protocol. We have been able to amplify recombinant loci from the residual DNA in the corpses of recombinant progeny. Alternately, if the F1 recombinant isolated from the ICR assay is a hermaphrodite and laid F2 progeny before perishing, isolate 1–5 GFP expressing F2 progeny (indicating that they inherited the recombinant locus) and instead lyse them for downstream analysis.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Experimental replicates of the ICR assay with a mutation in the genetic background produce notably different results (e.g., frequencies of recombinants, frequencies of crossover and noncrossover events). (steps 12, 14, 16)</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Some DNA repair mutants are not stable when passaged serially as homozygotes. Your strains may have accumulated mutations, leading to changes in the ICR assay results. Consider reconstructing Strains #1–3 for your genotype of interest with balancer or marker chromosomes (<a href="#before-you-begin">before you begin</a> Step 3). If your mutant of interest is already balanced, confirm the genotypes of Strains #1–3 by PCR. Some DNA repair mutants, and in particular mutants for multiple DNA repair pathways, may exhibit defects in DNA repair associated with balancer chromosomes, resulting in loss of the desired genotype or accumulation of mutations over generations.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Jaramillo-Lambert, A., Ellefson, M., Villeneuve, A.M., and Engebrecht, J.A. (2007). Differential timing of S phases, X chromosome replication, and meiotic prophase in the C. elegans germ line. Dev. Biol. <i>308</i>, 206-221. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00507-4/sref1">View at publisher</a></p>
<p id="bib2">Okkema, P.G., Harrison, S.W., Plunger, V., Aryana, A., and Fire, A. (1993). Sequence requirements for myosin gene expression and regulation in Caenorhabditis elegans. Genetics <i>135</i>, 385-404. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00507-4/sref2">View at publisher</a></p>
<p id="bib3">Robert, V. and Bessereau, J.-L. (2007). Targeted engineering of the Caenorhabditis elegans genome following Mos1-triggered chromosomal breaks. EMBO J. <i>26</i>, 170-183. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00507-4/sref3">View at publisher</a></p>
<p id="bib4">Rosu, S., Libuda, D.E., and Villeneuve, A.M. (2011). Robust crossover assurance and regulated interhomolog access maintain meiotic crossover number. Science <i>334</i>, 1286-1289. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00507-4/sref4">View at publisher</a></p>
<p id="bib5">Stiernagle, T. (2006). Maintenance of <i>C. elegans</i>. In WormBook, The C. elegans Research Community, ed. (WormBook). <a class="external-link" href="https://doi.org/10.1895/wormbook.1.101.1">https://doi.org/10.1895/wormbook.1.101.1</a></p>
<p id="bib6">Toraason, E., Horacek, A., Clark, C., Glover, M.L., Adler, V.L., et al. (2021). Meiotic DNA break repair can utilize homolog-independent chromatid templates in C. elegans. Curr. Biol. <i>31</i>, 1508-1514. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00507-4/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Diana E. Libuda (<a href="mailto:dlibuda@uoregon.edu">dlibuda@uoregon.edu</a>).</p>
<h4>Materials availability</h4>
<p>All strains and reference sequences are available upon request or from the <i>Caenorhabditis</i> Genetics Center (CGC).</p>
<h4>Data and code availability</h4>
<p>Datasets associated with this protocol are available in <a href="#bib6">Toraason et al. (2021)</a>.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> T32GM007413 and Advancing Science in America (<a href="https://doi.org/10.13039/100008227">ARCS</a>) Foundation Award to E.T.; <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> R25HD070817 to A.H.; and <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> R00HD076165 and R35GM128890 to D.E.L. We thank the CGC (funded by <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> Office of Research Infrastructure Programs [NIH] P40 OD010440) for strains.</p>
<h3>Author contributions</h3>
<p>M.G. generated and tested the ICR assay. E.T. optimized the ICR assay. E.T. and A.H. optimized the SNP mapping for the conversion tract analyses with the ICR assay. D.E.L. designed the initial ICR assay. E.T. and D.E.L. wrote the paper.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>