<article>
<section>
<h1 id="header">A proteomics protocol to identify stimulation-induced binding partners dependent on a specific gene in mammalian cells</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Junji Zhu<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> and Wuhan Xiao<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>State Key Laboratory of Freshwater Ecology and Biotechnology, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan 430072, China</p>
<p id="aff2"><sup>2</sup>The Innovation of Seed Design, Chinese Academy of Sciences, Wuhan 430072, China</p>
<p id="aff3"><sup>3</sup>Hubei Hongshan Laboratory, Wuhan 430070, China</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:junjizhu@ihb.ac.cn">junjizhu@ihb.ac.cn</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:w-xiao@ihb.ac.cn">w-xiao@ihb.ac.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100962">10.1016/j.xpro.2021.100962</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Some protein-protein interactions are induced by different kinds of stimulation and are dependent on specific genes. To identify these interaction partners, we present a protocol which utilizes affinity purification of Flag-tagged protein complexes followed by mass-spectrometry-based proteomics to compare stimulation-induced interactomes between wild-type and CRISPR-Cas9-mediated knockout cells. The candidates of interest are identified using bioinformatic analyses and verified by biochemical approaches. This protocol is highly versatile and applies to a variety of cells and different types of stimulation.</p>
<p>For complete details on the use and execution of this protocol, please refer to (<a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00668-7/sref7">Zhu et al., 2021</a>).</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for identifying stimulation-dependent protein-protein interaction</li>
<li>Protocol for identifying a specific gene-dependent interactome</li>
<li>The protocol applies to a variety of cells and different types of stimulation</li>
<li>Bioinformatic and biochemical analyses can help to exclude non-specific interactions</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1172-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used to identify RNA virus-induced PRMT7 associated proteins dependent on MAVS (<a href="#bib7">Zhu et al., 2021</a>). Here, as an example, we overexpressed Flag-PRMT7 in <i>MAVS</i><sup><i>+/+</i></sup> and <i>MAVS</i><sup>−/−</sup> HEK293T cells treated with or without Sendai Virus (SeV) (<a href="#fig1">Figure 1</a>), and then we used mass spectrometry-based proteomics to identify Flag-PRMT7-associated proteins. After identifying PRMT7-associated proteins in each sample, we compared with each other. The proteins enriched in sample B (<i>MAVS</i><sup><i>+/+</i></sup> HEK293T cells with the overexpression of Flag-PRMT7 followed by SeV infection) compared to those in sample A (<i>MAVS</i><sup><i>+/+</i></sup> HEK293T cells with the overexpression of Flag-PRMT7) were considered as virus-induced PRMT7 associated proteins, and the proteins diminished in sample C (<i>MAVS</i><sup>−/−</sup> HEK293T cells with the overexpression of Flag-PRMT7 followed by SeV infection) compared to sample B were considered as PRMT7 associated proteins dependent on MAVS. The common proteins obtained from two independent comparisons were considered as virus-induced PRMT7 associated proteins dependent on MAVS. Alternatively, the main steps of this protocol can also be applied to other cells. Moreover, validated antibodies against endogenous proteins or other epitope tags of interest such as GFP and HA can be utilized for affinity purification.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1172-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overexpression of Flag-PRMT7 in <i>MAVS</i><sup><i>+/+</i></sup> and <i>MAVS</i><sup>−/−</sup> HEK293T cells treated with or without SeV</div>
</figcaption>
</figure>
<h3 id="sec1.1">Generation of specific gene knockout cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼<b>4–6 weeks</b>
</div>
<p>The protocols for generating single-gene knocked out cell line by CRISPR/Cas9 have been described previously (<a href="#bib2">Gao et al., 2021</a>; <a href="#bib4">Qiu and Ding, 2021</a>; <a href="#bib5">Ran et al., 2013</a>). Alternatively, If the associated-proteins of your interest are dependent on multi-genes, this protocol can also be adapted for multi-gene knocked out cell lines, and you can refer to the protocol (<a href="#bib6">Zhou et al., 2020</a>) for the detailed procedures of generating multi-gene knocked out cell lines.</p>
<h3 id="sec1.2">Confirm the expression of plasmids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼<b>3–4 days</b>
</div>
<p>It is recommended to perform Western blot analysis to confirm expression of Flag-PRMT7 before scaling up for immunoprecipitation assays.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse Anti-Flag</td>
<td>Sigma-Aldrich</td>
<td>F1804</td>
</tr>
<tr>
<td>Mouse Anti-SMURF1</td>
<td>Abcam</td>
<td>Ab57573</td>
</tr>
<tr>
<td>Rabbit IgG</td>
<td>Cell Signaling Technology</td>
<td>2729S</td>
</tr>
<tr>
<td>Rabbit Anti-PRMT7(for WB)</td>
<td>Cell Signaling Technology</td>
<td>14762</td>
</tr>
<tr>
<td>Rabbit Anti-PRMT7(for Co-IP)</td>
<td>Novus Biologicals</td>
<td>NBP2-19939</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>
<i>MAVS</i><sup><i>+/+</i></sup> HEK293T</td>
<td><a href="#bib7">Zhu et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>MAVS</i><sup>−<i>/</i>−</sup> HEK293T</td>
<td><a href="#bib7">Zhu et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Protease Inhibitor Cocktail</td>
<td>Bimake</td>
<td>B14001</td>
</tr>
<tr>
<td>Trypsin</td>
<td>Promega Corporation</td>
<td>V9013</td>
</tr>
<tr>
<td>Acetonitrile (CH<sub>3</sub>CN)</td>
<td>Sigma-Aldrich</td>
<td>A3396</td>
</tr>
<tr>
<td>Ammonium Bicarbonate(NH<sub>4</sub>HCO<sub>3</sub>)</td>
<td>Sigma-Aldrich</td>
<td>A6141</td>
</tr>
<tr>
<td>Dithiothreitol(DTT)</td>
<td>Thermo Fisher Scientific</td>
<td>D1532</td>
</tr>
<tr>
<td>Lodoacetamide</td>
<td>Sigma-Aldrich</td>
<td>I1149</td>
</tr>
<tr>
<td>PBS</td>
<td>Monad</td>
<td>RP05002S</td>
</tr>
<tr>
<td>Na<sub>3</sub>VO<sub>4</sub>
</td>
<td>Sigma-Aldrich</td>
<td>450243</td>
</tr>
<tr>
<td>PMSF</td>
<td>Sigma-Aldrich</td>
<td>78830</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Lipofectamine 2000 Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>1668019</td>
</tr>
<tr>
<td>Anti-Flag M2 Agarose beads</td>
<td>Sigma-Aldrich</td>
<td>A2220</td>
</tr>
<tr>
<td>Protein G Sepharose</td>
<td>GE HealthCare Company</td>
<td>17-0618-01</td>
</tr>
<tr>
<td>BCA Kit</td>
<td>Beyotime</td>
<td>P0012</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Flag-PRMT7</td>
<td><a href="#bib7">Zhu et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>SeV</td>
<td>Bo Zhong lab</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software</td>
</tr>
<tr>
<td>pFind (V3.1.6)</td>
<td><a href="http://pfind.ict.ac.cn/software/pFind/">http://pfind.ict.ac.cn/software/pFind/</a></td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th>Culture medium</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>N/A</td>
<td>445 mL</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>L-glutamine</td>
<td>40 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<thead>
<tr>
<th>Lysis buffer(RIPA)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris HCl (pH 7.4)</td>
<td>50 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>NP-40</td>
<td>1%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Na-deoxycholate</td>
<td>0.25%</td>
<td>2.5 g</td>
</tr>
<tr>
<td>NaCl (2.5 M)</td>
<td>150 mM</td>
<td>60 mL</td>
</tr>
<tr>
<td>EDTA (pH 7.4, 0.5 M)</td>
<td>1 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>NaF (0.5 M)</td>
<td>1 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>876 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<thead>
<tr>
<th>2 × SDS sample buffer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris HCl (pH 6.8)</td>
<td>100 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>10%</td>
<td>1 mL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>4%</td>
<td>4 mL</td>
</tr>
<tr>
<td>Bromophenol Blue</td>
<td>0.2%</td>
<td>0.02 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>20%</td>
<td>2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>2 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Transfection of cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 18–24 h</div>
<p><i>MAVS</i><sup><i>+/+</i></sup> and <i>MAVS</i><sup>−/−</sup> HEK293T cells need to be transfected with Flag-PRMT7 before immunoprecipitation. Three 10-cm dishes with 70%–80% confluent cells are used for this experiment. HEK293T cells are cultured in DMEM medium supplemented with 10% (vol/vol) fetal bovine serum (FBS) at 37°C with 5% CO<sub>2</sub>.</p>
<ol>
<li>Prepare cells for transfection
<ol type="a">
<li>Remove old medium from <i>MAVS</i><sup><i>+/+</i></sup> and <i>MAVS</i><sup>−/−</sup> HEK293T cells (with 80–90% confluency) and mildly wash cells once by PBS. Add 2 mL trypsin to each 10-cm dishes, place the dishes in a cell culture incubator for 5 min at 37°C.</li>
<li>Add 4 mL warm (37°C) DMEM medium into each plate to inactivate trypsin. Pipette the cells softly and then transfer the suspension into 15-mL tubes and spin at 600 × g for 5 min.</li>
<li>Remove the supernatant, add 6 mL fresh medium to each tube, and then suspend the cells by pipetting.</li>
<li>Prepare two 10-cm dishes and add 2 mL of <i>MAVS</i><sup><i>+/+</i></sup> HEK293T cells and 8 mL fresh DMEM medium into each plate. Similarly, add 2 mL of <i>MAVS</i><sup>−/−</sup> HEK293T cells and 8 mL fresh DMEM medium into a new plate. Pipette the medium to homogenize cells (∼30% confluency), and put into an incubator at 37°C with 5% CO<sub>2</sub>. After seeded for 16 h, the cells will reach about 70%–80% confluency.</li>
</ol>
</li>
<li>Plasmid transfection
<ol type="a">
<li>Lipofectamine 2000 and 10 μg Flag-PRMT7 are diluted with 100 μL Opti-MEM respectively. After 5-min incubation, mix them gently and incubate for 20 min at room temperature.</li>
<li>Add DNA-Lipofectamine 2000 complexes (100 μL) to each dish, mix gently by rocking the plate back and forth.</li>
<li>6 h later, change the medium with fresh pre-warmed medium, and put back into the incubator at 37°C with 5% CO<sub>2</sub> for 10 h before SeV stimulation. It is not necessary to change medium after transfection unless culture medium turns yellow.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The number of 10-cm dishes may vary depending on your experiments, and the abundance of protein.</div>
<div class="note">
<span class="note-title">Note:</span> The step 2 can be optional if you wish to enrich the endogenous proteins by validated antibodies.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to transfect Flag-empty vector into <i>MAVS</i><sup><i>+/+</i></sup> HEK293T cells as a control for further analysis.</div>
<div class="note">
<span class="note-title">Note:</span> Optionally choose other transfection reagent if needed and use the manufacturer instructions for troubleshooting.</div>
<h3 id="sec3.2">Stimulate the cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼8 h</div>
<p>Cells respond to pathogen-associated molecular patterns by upregulating IFN and IFN-stimulated genes (ISGs). Sendai virus (SeV) is an RNA virus which can trigger IFN response and has been extensively used to study innate antiviral responses. In this protocol, for identifying associated proteins of PRMT7 induced by RNA virus infection, we challenged cells with SeV.</p>
<ol start="3">
<li>Treat the <i>MAVS</i><sup><i>+/+</i></sup> and <i>MAVS</i><sup>−/−</sup> HEK293T cells with or without SeV for 8 h.
<ol type="a">
<li>Dilute SeV (MOI=1) in minimal volume of serum-free medium.</li>
<li>Add diluted SeV to <i>MAVS</i><sup><i>+/+</i></sup> and <i>MAVS</i><sup>−/−</sup> HEK293T cells, and also add the same amount of serum-free medium to another set of <i>MAVS</i><sup><i>+/+</i></sup> HEK293T cells; put the cells in the incubator at 37°C with 5% CO<sub>2</sub> for 8 h.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The types of pathogen-stimulation and treatment time are dependent on your experiments. This step can be optional if your experiment does not need any stimulation.</div>
<h3 id="sec3.3">Cell lysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1.5 h</div>
<p>The transfected cells with or without stimulation are lysed and the protein concentration is measured.</p>
<ol start="4">
<li>After the transfected cell infected with or without SeV for 8 h, transfer dishes from the incubator on ice.</li>
<li>Remove the medium and wash dishes twice with 10 mL PBS.</li>
<li>For each dish, add 1 mL lysis buffer (RIPA) and keep shaking on a horizontal shaker at 4°C for 30 min. Optionally, use cell scrapers to lift cells from dishes if multiple dishes are transfected with the same plasmid, add 1 mL lysis buffer (RIPA) and transfer cell lysate into 1.5-mL microcentrifuge tube.</li>
<li>Centrifuge at 12,000 × g for 15 min at 4°C.</li>
<li>Transfer supernatant into new microcentrifuge tubes.</li>
<li>Measure protein concentration using BCA assay or other available methods.</li>
<li>Add additional lysis buffer (RIPA) to ensure the same concentration for different samples.</li>
<li>Save 50 μL of each lysate as input and store at -80°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prior to experiment, prepare lysis buffer and store at 4°C. 1% NaVO<sub>4</sub>, PMSF and protease inhibitor must be added immediately before use.</div>
<div class="note">
<span class="note-title">Note:</span> Even though RIPA is the most commonly used lysis buffer, sometimes a different buffer may be used, based on the proteins tested.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Be careful about the medium containing SeV. Collect the medium using a container and have it autoclaved before discard.</div>
<h3 id="sec3.4">Affinity purification using Anti-Flag M2 agarose beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 h or overnight</div>
<p>Anti-Flag M2 agarose beads (anti-Flag monoclonal antibody covalently attached to agarose by hydrazide linkage) is used to isolate Flag-tagged protein (bait) and its interactome (prey). Keep agarose beads on ice and slowly warm them at 4°C for 5 min. Gently rotate to mix bead slurry.</p>
<ol start="12">
<li>Prepare enough 1.5-mL microcentrifuge tubes based on your experiments and add 20 μL anti-Flag M2 agarose bead slurry into each tube.</li>
<li>Wash beads with 1 mL lysis buffer by gently resuspending.</li>
<li>Centrifuge at 5000 × g for 1 min at 4°C, and remove supernatant.</li>
<li>Repeat step 13 and 14 three times.</li>
<li>Replace 1 mL cell lysates from step 10 to each 1.5-mL microcentrifuge tubes containing washed beads and put on an end-over-end rotator at 4°C for 3 h or overnight.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For resin transfer, use a scissor to cut the sharp end of a plastic pipette tip to enlarge the tip opening which allows to transfer resin smoothly. Gently rotate or vortex to ensure resuspension of resin before transferring.</div>
<div class="note">
<span class="note-title">Note:</span> Be careful! Don’t transfer any resin when removing supernatant. It is recommended to use a narrow-end pipette tip which can be made by using forceps to pinch the opening of a plastic pipette tip until it is partially closed.</div>
<h3 id="sec3.5">Washing and elution of affinity purified proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>Elution of Flag-tagged proteins and associated complexes can be carried out with SDS-PAGE sample buffer, 3 × Flag peptide or 0.1 M glycine HCl, pH 3.5. In this protocol, we use SDS-PAGE sample buffer as an example. Alternatively, this method can be also applied for other immunomagnetic beads. For further details about other two methods, you can refer to manufacturer instructions of anti-Flag M2 agarose beads (Sigma).</p>
<ol start="17">
<li>Centrifuge the tubes containing lysate and bead at 5000 × g for 1 min at 4°C, and remove supernatant.</li>
<li>Wash beads with 1 mL lysis buffer by gently resuspending and try to remove nonspecific binding proteins.</li>
<li>Centrifuge at 5000 × g for 1 min at 4°C, and remove supernatant.</li>
<li>Repeat step 18 and step 19 three times.</li>
<li>Add 50 μL 2 × SDS sample buffer to each sample.</li>
<li>Boil samples for 5 min.</li>
<li>Centrifuge samples at 5000 × g for 2 min at 4°C and remove supernatants to new 1.5-mL microcentrifuge tubes. Keep on ice, use immediately or store at -80°C.</li>
</ol>
<p>Biochemical analysis of affinity purified proteins</p>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>To confirm affinity purified proteins by anti-Flag M2 beads worked and enriched proteins qualified for mass spectrometry, you can perform Western blot analysis and protein staining, such as silver staining or Coomassie blue staining.</p>
<ol start="24">
<li>Load and run inputs (step 11) and concentrated samples (step 23) on SDS-PAGE gel in duplicate.
<ol type="a">
<li>One set of gel is used for silver staining (<a href="#fig2">Figure 2</a>) or Coomassie blue staining.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1172-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Silver staining from samples A, B, and C shows enrichment of putative FLAG-PRMT7 interaction proteins</div>
</figcaption>
</figure>
</li>
<li>Transfer another set of gel to PVDF membrane and use anti-Flag antibody for Western blot analysis.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this step, it is recommended to make verification for gene knockout using immunoblotting analysis of the inputs from wildtype and knockout cells respectively.</div>
<h3 id="sec3.6">In-gel digestion of proteins separated by SDS-PAGE</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>Once affinity purified proteins are validated, perform SDS-PAGE to separate proteins of each sample and use in-gel digestion to digest the proteins into small peptides which will be used for Mass spectrometry analysis.</p>
<ol start="25">
<li>Excision of Coomassie blue stained bands
<ol type="a">
<li>Separate protein of each sample obtained from step 23 by SDS-PAGE. In this step, we generally use straight percentage gels (10%–12%). Load each sample into two adjacent wells (up to 20 μL/well). Let them run to reach 1.0–1.5 cm in gels (200V for ∼10 min).</li>
<li>Stain gels with Coomassie blue.</li>
<li>Scan gel before cutting out the bands.</li>
<li>Excise bands of each sample from the gel. We generally cut entire lane into 5-12 slices. Once you get the slices cut from a specific lane, further cut into 1.0 × 1.0 mm pieces and place into 1.5 mL microcentrifuge tubes. Next, use a fresh scalpel to cut each slice, and transfer them to a tube using same scalpel. Try to finish this step as quickly as possible, because gels will become stickier and sticker as they stay longer in air.</li>
<li>Wash gel pieces of each sample with 300 μL dH<sub>2</sub>O water for 15 min. Add 300 μL CH<sub>3</sub>CN and wash them for further 15 min. Remove supernatant.</li>
<li>Wash gel pieces of each sample with 300 μL NH<sub>4</sub>HCO<sub>3</sub> (100 mM) for 15 min and discard supernatant.</li>
<li>Add 100 μL CH<sub>3</sub>CN to dehydrate gel pieces of each sample for 5 min. The gel pieces should shrink and look completely white. Remove supernatant.</li>
<li>Dry gel pieces of each sample in a speedvac for 5 min.</li>
</ol>
</li>
<li>Reduction/Alkylation of gel pieces
<ol type="a">
<li>Add 50 μL DTT (10 mM) in 100 mM NH<sub>4</sub>HCO<sub>3</sub> to each sample and incubate at 56°C for 1 h. Discard supernatant.</li>
<li>Add 50 μL of 50 mM freshly prepared iodoacetamide in 100 mM NH<sub>4</sub>HCO<sub>3</sub> to each sample and incubate in dark at room temperature for 30 min. Discard supernatant.</li>
<li>Wash band pieces with 300 μL of 100 mM NH<sub>4</sub>HCO<sub>3</sub> for 15 min. Remove supernatant.</li>
<li>Wash band pieces with 300 μL of 20 mM NH<sub>4</sub>HCO<sub>3</sub>/CH<sub>3</sub>CN (50:50 V/V) for 15 min. Remove supernatant.</li>
<li>Add 100 μL CH<sub>3</sub>CN to dehydrate band pieces for 5 min. Remove supernatant.</li>
<li>Dry band pieces in a speedvac for 5 min.</li>
</ol>
</li>
<li>Digestion of gel pieces
<ol type="a">
<li>Add 20 μL of 12.5 μg/mL trypsin in 20 mM NH<sub>4</sub>HCO<sub>3</sub> to each sample.</li>
<li>Keep gel to dehydrate in trypsin digestion buffer for 30 min.</li>
<li>Incubate at 37°C overnight.</li>
</ol>
</li>
<li>Extraction of peptides
<ol type="a">
<li>Transfer supernatants to new 1.5-mL microcentrifuge tubes. The supernatants containing peptides are used for further analysis.</li>
<li>Add 50 μL of 1% formic acid to each sample and incubate for 20 min.</li>
<li>Transfer supernatant to corresponding tubes at step (a).</li>
<li>Repeat step (a) and (b) one more time.</li>
<li>Add 150 μL CH<sub>3</sub>CN to each sample and incubate for 10 min.</li>
<li>Transfer supernatant to corresponding tubes at step (a).</li>
<li>Use a speedvac to dry peptides in tubes at step (a) completely.</li>
<li>Resuspend invisible pellet with 50 μL of 1% formic acid. To optimize solubilization of peptides, firstly add 10 μL of 5% formic acid and then add 40 μL dH<sub>2</sub>O. Store at -20°C or -80°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Perform all gel washing and extraction steps on a shaker to ensure complete extraction of peptides.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Perform all pipetting steps in a laminal flow hood. Do not crash your head or any other part of your body, and wear gloves and mask during this procedure in order to avoid keratin contamination.</div>
<h3 id="sec3.7">Mass spectrometry and statistical analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> weeks</div>
<p>Small peptides digested from proteins of each sample are analyzed by mass spectrometry to identify the interactome of PRMT7.</p>
<ol start="29">
<li>Tryptic peptides are dissolved in 0.1% formic acid (solvent A), directly load into a home-made reversed-phase analytical column (C18 resin with 3-μm particle size, 250-mm length×75-μm inner diameter, Acclaim PepMap RSLC, Thermo Fisher Scientific) or other suitable column (e.g., C18 resin with 1.9-μm particle size, 250-mm length ×75-μm inner diameter, New Objective, Woburn, MA).</li>
<li>Gradient is comprised of an increase from 5% to 6% solvent B (0.1% formic acid in 98% acetonitrile) over 5.2 min, 6%–33% in 52.8 min and climbing to 90% in 0.2 min then holding at 90% for the last 6.8 min, all hold at a constant flow rate of 400 nL/min on an EASY-nLC 1000 UPLC system.</li>
<li>Peptides are subjected to ESI (Electrospray Ionization) source followed by tandem mass spectrometry (MS/MS) in Q Exactive<sup>TM</sup> HF-X (Thermo) coupled online to the UPLC.</li>
<li>The electrospray voltage applied is 2.4 kV (according to experience). The m/z scan range is 350–1800 for full scan, and intact peptides are detected in the Orbitrap at a resolution of 60,000 (default parameter).</li>
<li>Peptides are then selected for MS/MS using NCE (Normalized Collision Energy) setting as 28 and fragments are detected in Orbitrap at a resolution of 15,000.</li>
<li>A data-dependent procedure that alternated between one MS scan followed by 25 MS/MS scans with 25.0 s dynamic exclusion. Fixed first mass is set as 100 m/z.</li>
<li>All acquired raw data are processed with pFind (V3.1.6) software. Peak lists are searched against NCBI protein database. Four missed cleavages are allowed for trypsin. The precursor and fragment ion mass tolerances are 20 ppm and 20 ppm, respectively. Minimum peptide length is set at 6, while the estimated false discovery rate (FDR) threshold for peptide and protein are specified at maximum 1%. For other parameters in pFind, we use the algorithm defaults.</li>
<li>After analyzing raw data from these samples, PRMT7-associated proteins in each sample are identified. Then the identified proteins in each sample are compared with each other. The proteins enriched in sample B compared to those in sample A are considered as virus-induced PRMT7 associated proteins, and the proteins diminished in sample C compared with sample B are considered as PRMT7 associated proteins dependent on MAVS. The common proteins obtained from the two independent comparisons[(B-A)<b>∩</b>(B-C)] are considered as virus-induced PRMT7 associated proteins dependent on MAVS (<a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. The proteins identified by mass spectrometry</caption>
<thead>
<tr>
<th colspan="2">A</th>
<th colspan="2">B</th>
<th colspan="2">C</th>
<th>B–A</th>
<th>BC</th>
<th>(B–A)∩(B–C)</th>
</tr>
<tr>
<th>Proteins</th>
<th>PSMs</th>
<th>Proteins</th>
<th>PSMs</th>
<th>Proteins</th>
<th>PSMs</th>
<th>Proteins</th>
<th>Proteins</th>
<th>Proteins</th>
</tr>
</thead>
<tbody>
<tr>
<td>KIF11</td>
<td>112</td>
<td>KIF11</td>
<td>74</td>
<td>KIF11</td>
<td>105</td>
<td>SMURF2</td>
<td>SMURF2</td>
<td>SMURF2</td>
</tr>
<tr>
<td>RS27A</td>
<td>31</td>
<td>SMURF2</td>
<td>62</td>
<td>PPM1B</td>
<td>25</td>
<td>EF1A3</td>
<td>EF1A3</td>
<td>EF1A3</td>
</tr>
<tr>
<td>IRS4</td>
<td>27</td>
<td>EF1A3</td>
<td>57</td>
<td>RS27A</td>
<td>24</td>
<td>SMURF1</td>
<td>SMURF1</td>
<td>SMURF1</td>
</tr>
<tr>
<td>K2C1</td>
<td>25</td>
<td>RS27A</td>
<td>37</td>
<td>HNRNPM</td>
<td>23</td>
<td>PPM1B</td>
<td>PRMT5</td>
<td> </td>
</tr>
<tr>
<td>KCTD1</td>
<td>24</td>
<td>SMURF1</td>
<td>37</td>
<td>KCTD5</td>
<td>22</td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>KIC9</td>
<td>23</td>
<td>K2C1</td>
<td>17</td>
<td>K2C1</td>
<td>21</td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>PRMT5</td>
<td>22</td>
<td>IRS4</td>
<td>16</td>
<td>IRS4</td>
<td>20</td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>HNRNPM</td>
<td>20</td>
<td>PRMT5</td>
<td>16</td>
<td>PPM1A</td>
<td>17</td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>RS11</td>
<td>18</td>
<td>PPM1B</td>
<td>14</td>
<td>ATPA</td>
<td>16</td>
<td> </td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For researchers who cannot directly access to mass spectrometers, mass spectrometry services and data analysis can be contracted with a facility or a commercial company available to the researchers.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to perform replicates for sample A, B and C in order to improve the accuracy of results.</div>
<div class="note">
<span class="note-title">Note:</span> You can narrow down the range of proteins based on your research goal to find out the proteins you are interested in before validation, such as you intend to look for E3 ubiquitin ligases, you can specifically pick up the proteins with E3 ubiquitin ligase activity for further analysis.</div>
<h3 id="sec3.8">Biochemical analysis for the candidate validation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>After mass spectrometry and statistical analysis, biochemical analysis, such as immunoprecipitation, must be performed for the validation of candidate proteins. In this protocol, based on our previously study (<a href="#bib7">Zhu et al., 2021</a>), we think that E3 ligase SMURF1 are supposed to be the target among associated- proteins. If you do not have specific target of interest, it is recommended to validate proteins based on their PSMs (Peptide Spectrum Matches).</p>
<ol start="37">
<li>Treat 80%–90% confluent <i>MAVS</i><sup><i>+/+</i></sup> HEK293T cells (four 10-cm dishes including one as control) and <i>MAVS</i><sup>−/−</sup> HEK293T cells (three 10-cm dishes) with SeV for 0 h, 4 h and 8 h.</li>
<li>Cell lysates are prepared following step 4 to step 11.</li>
<li>Preclear cell lysate by combing with 20 μL Protein A/G and 1 μg IgG in a 1.5 mL microcentrifuge tube. Incubate tubes at 4°C with gentle rotation for 30 min.</li>
<li>Centrifuge at 5000 × g for 1 min at 4°C, and remove supernatant to new 1.5-mL microcentrifuge tubes placed on ice.</li>
<li>Add appropriate amount of PRMT7 antibody (1.0 μg–2.0 μg ) to precleared lysates from step 40 (the control is added with the same amount of IgG). Incubate samples at 4°C for 2 h with gentle rotation.</li>
<li>Add 20 μL Protein A/G to each sample. Incubate samples at 4°C for 4 h or overnight with gentle rotation.</li>
<li>Washing and elution of affinity purified proteins following step 17 to step 23.</li>
<li>Load and run inputs and concentrated samples on SDS-PAGE gels.</li>
<li>Transfer gels to PVDF membranes and use indicated antibodies for Western blot analysis (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1172-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Immunoprecipitation analysis of endogenous interaction of PRMT7 and SMURF1 in <i>MAVS</i><sup><i>+/+</i></sup> and <i>MAVS</i><sup>−/−</sup> HEK293T cells</div>
<p>Immunoblot of whole-cell lysates (bottom) and proteins immunoprecipitated with anti-PRMT7 antibody (top) from <i>MAVS</i><sup><i>+/+</i></sup> and <i>MAVS</i>/ HEK293T cells infected with SeV for 0–8 h. Figure reprint with permission from (<a href="#bib7">Zhu et al., 2021</a>).</p>
</figcaption>
</figure>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol is designed to identify a gene-associated proteins which are induced by cellular stimulation and are dependent on a specific gene. Wildtype and CRISPR-Cas9-mediated knockout cells of a specific gene are transfected with Flag-tagged interested gene and followed by stimulation. By affinity purification of interested gene and subsequent analysis by mass spectrometry, the stimulation induced and a specific gene dependent interactome will be identified. Subsequently, validation for the candidates should be performed by biochemical analysis.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While in this protocol, we utilize Flag-tagged proteins as a bait for affinity purification and MS analysis, other epitope tags, such as His and GST can also be utilized for affinity purification. However, we have not tested whether it is suitable for using endogenous protein as a bait, which may largely depend on the enrichment effect of endogenous protein by a specific antibody. Additionally, we use CRISPR-Cas9-mediated knockout cells to identify a specific-gene-dependent interaction protein. Of note, some gene-deficiency will seriously affect the cell viability, this kind of genes may not be suitable for employing this protocol. Moreover, even though we aim to minimize false identification of nonspecific proteins that interact with an interested protein (i.e., use Flag-empty as a negative control and perform bioinformatics analysis by comparing identified proteins between samples to exclude nonspecific proteins), it is still hard to exclude some nonspecific proteins. We therefore highly recommend to use other methods (immunoblots, reciprocal immunoprecipitation and immunofluorescence) for further validation.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Expression of exogenous bait protein is low.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Increase concentration and amount of DNA for transfection following the manufacturer instructions of transfection reagent. Alternatively, try to use other transfection reagents or methods, such as Vigofect (Vigorous Biotech, Beijing, China) or transduction with lentivirus.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>There is no visible affinity purified protein in Western blot analysis, Coomassie blue staining or silver staining.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This may be largely due to insufficient amount of protein used for immunoprecipitation or low enrichment efficiency of affinity purification. In this case, we recommend to scaling up reactions with larger volumes (up to 40 μL per well for 10-well SDS-PAGE gel), reagents and samples, as well as optimize time of incubation for affinity purification. A pilot experiment is recommended before formal affinity purification.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Expressions of exogenous bait protein between wildtype and knockout cells are largely different.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>This may be caused by the difference of cell conditions between wildtype and knockout cells. In this case, we recommend to replace wildtype cells with knockout cells reconstituted by ectopic expression of the gene knocked out in cells (<a href="#bib1">Comstra et al., 2017</a>).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>There are too many non-specific interactions identified by MS in each sample.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This problem can be resolved by using control purification. It may help to filter non-specific interactions. Alternatively, the curated databases of contaminant proteins associated to magnetic bead isolations (<a href="#bib1">Comstra et al., 2017</a>; <a href="#bib3">Mellacheruvu et al., 2013</a>) may also help to resolve this problem partially.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>There are no available antibodies for validating endogenous candidate proteins.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>To make different epitope tags for identified protein and bait protein respectively (e.g., Flag, HA or Myc) and then overexpress them in wild-type and knockout cells followed by treatment with or without stimulation. Subsequently, immunoprecipitation and immunoblotting are conducted with these epitope tag antibodies. This alternative method can help to resolve this problem.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Comstra, H.S., McArthy, J., Rudin-Rush, S., Hartwig, C., Gokhale, A., Zlatic, S.A., Blackburn, J.B., Werner, E., Petris, M., D'Souza, P., et al. (2017). The interactome of the copper transporter ATP7A belongs to a network of neurodevelopmental and neurodegeneration factors. Elife <i>6</i>, e24722. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00668-7/sref1">View at publisher</a></p>
<p id="bib2">Gao, P., Dong, X., Wang, Y., and Wei, G.H. (2021). Optimized CRISPR/Cas9-mediated single nucleotide mutation in adherent cancer cell lines. STAR Protoc. <i>2</i>, 100419. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00668-7/sref2">View at publisher</a></p>
<p id="bib3">Mellacheruvu, D., Wright, Z., Couzens, A.L., Lambert, J.P., St-Denis, N.A., Li, T., Miteva, Y.V., Hauri, S., Sardiu, M.E., Low, T.Y., et al. (2013). The CRAPome: a contaminant repository for affinity purification-mass spectrometry data. Nat. Methods <i>10</i>, 730-736. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00668-7/sref3">View at publisher</a></p>
<p id="bib4">Qiu, Y. and Ding, Q. (2021). Optimized protocol for gene editing in adipocytes using CRISPR-Cas9 technology. STAR Protoc. <i>2</i>, 100307. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00668-7/sref4">View at publisher</a></p>
<p id="bib5">Ran, F.A., Hsu, P.D., Wright, J., Agarwala, V., Scott, D.A., and Zhang, F. (2013). Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. <i>8</i>, 2281-2308. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00668-7/sref5">View at publisher</a></p>
<p id="bib6">Zhou, P., Chan, B.K.C., Wan, Y.K., Yuen, C.T.L., Choi, G.C.G., Li, X., Tong, C.S.W., Zhong, S.S.W., Sun, J., Bao, Y., et al. (2020). A three-way combinatorial CRISPR screen for analyzing interactions among druggable targets. Cell. Rep. <i>32</i>, 108020. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00668-7/sref6">View at publisher</a></p>
<p id="bib7">Zhu, J., Li, X., Cai, X., Zha, H., Zhou, Z., Sun, X., Rong, F., Tang, J., Zhu, C., Liu, X., et al. (2021). Arginine monomethylation by PRMT7 controls MAVS-mediated antiviral innate immunity. Mol. Cell. <i>81</i>, 3171-3186.e3178. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00668-7/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Wuhan Xiao (<a href="mailto:w-xiao@ihb.ac.cn">w-xiao@ihb.ac.cn</a>).</p>
<h4>Materials availability</h4>
<p>All reagents generated in this study are available from the lead contact upon completing a Materials Transfer Agreement.</p>
<h4>Data and code availability</h4>
<p>No unique datasets or codes were generated in this study.</p>
<h3>Acknowledgments</h3>
<p>We acknowledge support from the Strategic Priority Research Program of the Chinese Academy of Sciences (XDA24010308 to W.X.), the National Natural Science Foundation of China (31830101, and 31721005 to W.X.), and the National Key Research and Development Program of China (2018YFD0900602 to W.X.).</p>
<h3>Author contributions</h3>
<p>J.Z. and W.X. conceived and designed the project. J.Z. developed and optimized the protocol. J.Z. and W.X. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>