<article>
<section>
<h1 id="header">Immunoaffinity purification of endogenous proteins from <i>S. cerevisiae</i> for post-translational modification and protein interaction analysis</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Deepika Jaiswal,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Rashi Turniansky,<sup><a href="#aff1">1</a></sup> and Erin M. Green<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biological Sciences, University of Maryland Baltimore County, Baltimore, MD 21250, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:deej@umbc.edu">deej@umbc.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:egreen@umbc.edu">egreen@umbc.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100945">10.1016/j.xpro.2021.100945</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Protein regulation by post-translational modifications and protein-protein interactions is critical to controlling molecular pathways. Here, we describe an immunoaffinity purification approach in <i>Saccharomyces cerevisiae</i>. The protocol uses an endogenously-expressed epitope-tagged protein and can be applied to the identification of post-translational modifications or protein binding partners. The lysine methyltransferase Set5 is used as an example here to purify phosphorylated Set5 and identify phosphosites; however, this approach can be applied to a diverse set of proteins in yeast.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref2">Jaiswal et al. (2020)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Preparation of yeast for immunoaffinity purification of proteins using the FLAG tag</li>
<li>Detailed recommendations on optimization of immunoaffinity purifications</li>
<li>Versatile method for identifying post-translational modifications or protein interactors</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1170-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used in a recent publication (<a href="#bib2">Jaiswal et al., 2020</a>) to study the extent of phosphorylation of the lysine methyltransferase Set5 in <i>Saccharomyces cerevisiae</i>. Following identification of the phosphosites, mutant versions of Set5 were created to determine whether or not phosphorylation impacts methyltransferase activity. These approaches can be adapted to other protein methyltransferases in yeast or other systems, including human methyltransferases (<a href="#bib5">Separovich et al., 2020</a>; <a href="#bib6">Separovich and Wilkins, 2021</a>). This experiment can also be used to identify other post-translational modifications or protein interactors of the target protein, as well as to purify a target protein to be used in enzymatic or other biochemical activity assays. In this protocol, a yeast strain expressing Set5 fused to an N-terminal 3<b>×</b>FLAG tag under the control of its endogenous promoter is used. It is expected that a strain with a FLAG-tagged protein of interest is constructed using standard yeast molecular genetic approaches (<a href="#bib3">Longtine et al., 1998</a>; <a href="#bib4">Moqtaderi and Struhl, 2008</a>) prior to beginning this protocol.</p>
<h3 id="sec1.1">Prepare media and buffers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol>
<li>Prepare YPD medium for growing yeast cultures.
<ol type="a">
<li>Mix medium components in a large beaker with a magnetic stir bar and place on a stir plate until all components are dissolved.</li>
<li>Pour medium into 1 L bottles and autoclave to sterilize.</li>
</ol>
</li>
<li>Prepare the yeast lysis buffer required for generating the whole cell lysates.</li>
<li>The SDS-PAGE gels and buffers can also be prepared ahead of time. Polyacrylamide gels can be stored at 4°C for up to one week and the buffers are stable for several months at 25°C.</li>
</ol>
<h3 id="sec1.2">Prepare yeast strains</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="4">
<li>Streak yeast strains (untagged control strain and FLAG-tagged methyltransferase strain) from glycerol stocks for single colonies on YPD medium plates.
<ol type="a">
<li>Place the plate in a 30°C incubator for two days.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Yeast Extract</td>
<td>Research Products International</td>
<td>Cat# Y20020</td>
</tr>
<tr>
<td>Peptone</td>
<td>Research Products International</td>
<td>Cat# P20250</td>
</tr>
<tr>
<td>Dextrose</td>
<td>Research Products International</td>
<td>Cat# G32045</td>
</tr>
<tr>
<td>HEPES</td>
<td>Fisher Scientific</td>
<td>Cat# BP310-500</td>
</tr>
<tr>
<td>Magnesium acetate</td>
<td>MilliporeSigma</td>
<td>Cat# M5661</td>
</tr>
<tr>
<td>EDTA (Ethylenediaminetetraacetic acid)</td>
<td>MilliporeSigma</td>
<td>Cat# E6758</td>
</tr>
<tr>
<td>EGTA (Ethylene-bis(oxyethylenenitrilo) tetraacetic acid)</td>
<td>MilliporeSigma</td>
<td>Cat# 67-42-5</td>
</tr>
<tr>
<td>Glycerol</td>
<td>MilliporeSigma</td>
<td>Cat# G5516</td>
</tr>
<tr>
<td>3<b>×</b>FLAG peptide</td>
<td>APEeXBIO</td>
<td>Cat# A6001</td>
</tr>
<tr>
<td>PMSF</td>
<td>MilliporeSigma</td>
<td>Cat# P7626</td>
</tr>
<tr>
<td>EZBlock™ Protease Inhibitor Cocktail IV</td>
<td>BioVision</td>
<td>Cat# K279</td>
</tr>
<tr>
<td>PhosSTOP phosphatase inhibitor tablets</td>
<td>MilliporeSigma</td>
<td>Cat# PHOSS-RO</td>
</tr>
<tr>
<td>Igepal® CA-630 (NP-40)</td>
<td>MilliporeSigm</td>
<td>Cat# I8896</td>
</tr>
<tr>
<td>FLAG magnetic beads</td>
<td>MilliporeSigma</td>
<td>Cat# M8823</td>
</tr>
<tr>
<td>Pierce™ Coomassie Plus (Bradford) Protein Assay Reagent</td>
<td>Fisher Scientific</td>
<td>Cat# PI23238</td>
</tr>
<tr>
<td>Tris-Base-Ultrapure</td>
<td>United States Biological</td>
<td>Cat# 77-86-1</td>
</tr>
<tr>
<td>SDS (Sodium Dodecyl Sulfate)</td>
<td>Fisher Scientific</td>
<td>Cat# 50-751-7526</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>MiliporeSigma</td>
<td>Cat# M3148</td>
</tr>
<tr>
<td>Bromophenol blue sodium salt</td>
<td>Fisher Scientific</td>
<td>Cat# AA3263906</td>
</tr>
<tr>
<td>PMSF (Phenylmethanesulfonyl fluoride)</td>
<td>MilliporeSigma</td>
<td>Cat# P7626</td>
</tr>
<tr>
<td>Isopropanol, 99.6%</td>
<td>Fisher Scientific</td>
<td>Cat# AC423830025</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Fisher Scientific</td>
<td>Cat# AC615100010</td>
</tr>
<tr>
<td>SilverQuest Silver staining kit</td>
<td>Fisher Scientific</td>
<td>Cat# LC6070</td>
</tr>
<tr>
<td>Ammonium persulfate (APS)</td>
<td>MilliporeSigma</td>
<td>Cat# A3678</td>
</tr>
<tr>
<td>TEMED (N,N,N′,N′-Tetramethylethylenediamine)</td>
<td>Millipore Sigma</td>
<td>Cat# T7024</td>
</tr>
<tr>
<td>40% Acrylamide/Bis Solution (37.5:1)</td>
<td>Bio-Rad</td>
<td>Cat# 1610148</td>
</tr>
<tr>
<td>Precision Plus Protein™ All Blue Prestained Protein Standards</td>
<td>Bio-Rad</td>
<td>Cat# 1610373</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>S. cerevisiae</i>: strain BY4741 (yEG001)<i>MAT</i>a <i>his3Δ1 leu2Δ0 met15Δ0 ura3Δ0</i>
</td>
<td><a href="#bib2">Jaiswal et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>S. cerevisiae</i>: strain BY4741 (yEG401)<i>MAT</i>a <i>set5::3xFLAG-SET5</i>
</td>
<td><a href="#bib2">Jaiswal et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>NanoDrop 2000 software</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Refrigerated Benchtop Centrifuge</td>
<td>Beckman Coulter</td>
<td>Cat# ALLEGRA X-14R</td>
</tr>
<tr>
<td>Polypropylene Bottles (500 mL)</td>
<td>Beckman Coulter</td>
<td>Cat# 355665</td>
</tr>
<tr>
<td>Liquid Nitrogen</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Dry Ice</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Coffee grinder</td>
<td>Proctor Silex</td>
<td>N/A</td>
</tr>
<tr>
<td>Branson 450 Digital Sonifier</td>
<td>Branson</td>
<td>N/A</td>
</tr>
<tr>
<td>Magnetic tube stand</td>
<td>New England Biolabs</td>
<td>Cat# S1509S</td>
</tr>
<tr>
<td>NanoDrop 2000 and 2000c spectrophotometers</td>
<td>Fisher Scientific</td>
<td>Cat# 13-400-412</td>
</tr>
<tr>
<td>Wide-bore, low-retention tips</td>
<td>Fisher Scientific</td>
<td>Cat# 02-707-600</td>
</tr>
<tr>
<td>15 mL conical tube</td>
<td>Genesee Scientific</td>
<td>Cat# 28-101</td>
</tr>
<tr>
<td>50 mL conical tube</td>
<td>Genesee Scientific</td>
<td>Cat# 28-108</td>
</tr>
<tr>
<td>14 mL polypropylene centrifuge tube</td>
<td>Fisher Scientific</td>
<td>Cat# NC9164146</td>
</tr>
<tr>
<td>1.5 mL disposable cuvettes</td>
<td>Fisher Scientific</td>
<td>Cat# 50-476-476</td>
</tr>
<tr>
<td>Labquake™ Tube Shaker/Rotators</td>
<td>Fisher Scientific</td>
<td>Cat# 13-687-12Q</td>
</tr>
<tr>
<td>Mini-Protean Tetra Cell, 4 gel electrophoresis system</td>
<td>Bio-Rad</td>
<td>Cat# 1658000</td>
</tr>
<tr>
<td>Thermo Scientific™ Nalgene™ Staining Boxes</td>
<td>Fisher Scientific</td>
<td>Cat# 03-484-11A</td>
</tr>
<tr>
<td>Kimtech Kimwipes</td>
<td>Fisher Scientific</td>
<td>Cat# 06-666</td>
</tr>
<tr>
<td>Parafilm</td>
<td>Fisher Scientific</td>
<td>Cat# S37440</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>YPD medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast Extract</td>
<td>1%</td>
<td>10 g</td>
</tr>
<tr>
<td>Peptone</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>Dextrose</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>Deionized water</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Sterilize by autoclaving and store at 25°C for up to 4 weeks.</div>
<table id="undtbl2">
<caption>YPD Plates</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast Extract</td>
<td>1%</td>
<td>10 g</td>
</tr>
<tr>
<td>Peptone</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>Dextrose</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>Agar</td>
<td>2.5%</td>
<td>25 g</td>
</tr>
<tr>
<td>Deionized water</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Sterilize by autoclaving and pour plates while the media is still warm. Allow the plates to solidify and store at 4°C for up to 2 weeks.</div>
<table id="undtbl3">
<caption>Yeast lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES pH 8.0 (1 M)</td>
<td>100 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>MgOAc (1 M)</td>
<td>20 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>10%</td>
<td>25 mL</td>
</tr>
<tr>
<td>EGTA (0.5 M)</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.1 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>deionized water</td>
<td>n/a</td>
<td>Up to 250 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>250 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Filter sterilize and store at 4°C.</div>
<table id="undtbl4">
<caption>IP elution buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast lysis buffer</td>
<td>N/A</td>
<td>240 μL</td>
</tr>
<tr>
<td>FLAG Peptide (5 mg/mL)</td>
<td>0.2 mg/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>250 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Make fresh just before use.</div>
<table id="undtbl5">
<caption>5<b>×</b> SDS sample buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCL pH 6.8 (2 M)</td>
<td>250 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>SDS</td>
<td>10%</td>
<td>1 gm</td>
</tr>
<tr>
<td>Glycerol (100%)</td>
<td>30%</td>
<td>3 mL</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>5%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Bromophenol Blue (0.04%)</td>
<td>0.02%</td>
<td>5 mL</td>
</tr>
<tr>
<td>deionized water</td>
<td>n/a</td>
<td>Up to 10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Make 1 mL aliquots and store at −20°C.</div>
<table id="undtbl6">
<caption>PMSF (100 mM stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PMSF</td>
<td>100 mM</td>
<td>0.17 gm</td>
</tr>
<tr>
<td>Isopropanol</td>
<td>n/a</td>
<td>Up to 10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Incubate briefly at 37°C to dissolve completely. Make 1 mL aliquots and store at −20°C.</div>
<table id="undtbl7">
<caption>APS (10 % w/v stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>APS (ammonium persulfate)</td>
<td>10%</td>
<td>1 gm</td>
</tr>
<tr>
<td>deionized water</td>
<td>n/a</td>
<td>Up to 10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Make 1 mL aliquots and store at −20°C up to one month.</div>
<table id="undtbl8">
<caption>4<b>×</b> resolving gel buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>1.5 M</td>
<td>181.7 gm</td>
</tr>
<tr>
<td>HCl</td>
<td>N/A</td>
<td>∼25–30 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.4%</td>
<td>40 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Stir the Tris in water and once dissolved completely, add HCl dropwise to set the pH to 8.8 measured with a pH meter. Add the SDS and stir to dissolve completely. Bring up the volume to 1000 mL with deionized water. Store at 25°C.</div>
<table id="undtbl9">
<caption>4<b>×</b> stacking gel buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>0.5 M</td>
<td>60.55 gm</td>
</tr>
<tr>
<td>HCl</td>
<td>N/A</td>
<td>∼35–40 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.4%</td>
<td>40 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Stir the Tris in water and once dissolved completely, add HCl dropwise to set the pH to 6.8 measured with a pH meter. Add the SDS and stir to dissolve completely. Bring up the volume to 1000 mL with deionized water. Store at 25°C.</div>
<table id="undtbl10">
<caption>10<b>×</b> SDS gel running buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>250 mM</td>
<td>30.3 gm</td>
</tr>
<tr>
<td>Glycine</td>
<td>1.92 M</td>
<td>144 gm</td>
</tr>
<tr>
<td>SDS</td>
<td>1%</td>
<td>10 gm</td>
</tr>
<tr>
<td>deionized water</td>
<td>n/a</td>
<td>Up to 1000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Stir Tris and glycine in water on a heated stir plate. Once dissolved, turn off the heat and add SDS. Dissolve SDS by stirring and bring up the volume to 1000 mL with deionized water.</div>
<table id="undtbl11">
<caption>8% resolving gel</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>4<b>×</b> resolving gel buffer</td>
<td>1<b>×</b>
</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>40% acrylamide/bis (37.5:1)</td>
<td>8%</td>
<td>1 mL</td>
</tr>
<tr>
<td>10% APS</td>
<td>0.1%</td>
<td>50 μL</td>
</tr>
<tr>
<td>TEMED</td>
<td>0.1%</td>
<td>5 μL</td>
</tr>
<tr>
<td>deionized water</td>
<td>n/a</td>
<td>2.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare mixture just before pouring gel. Add 10% APS and TEMED immediately before pouring the gel.</div>
<table id="undtbl12">
<caption>4% stacking gel</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>4<b>×</b> stacking gel buffer</td>
<td>1<b>×</b>
</td>
<td>0.65 mL</td>
</tr>
<tr>
<td>40% acrylamide/bis (37.5:1)</td>
<td>4%</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>10% APS</td>
<td>0.1%</td>
<td>50 μL</td>
</tr>
<tr>
<td>TEMED</td>
<td>0.1%</td>
<td>5 μL</td>
</tr>
<tr>
<td>deionized water</td>
<td>n/a</td>
<td>1.65 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>2<b>.5 mL</b>
</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare mixture just before pouring gel. Add 10% APS and TEMED immediately before pouring the gel.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Grow and harvest yeast cultures expressing FLAG-tagged protein</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>Grow sufficient amounts of yeast cells expressing the FLAG-tagged methyltransferase to perform the immunoprecipitation. Cells are frozen in droplets in lysis buffer and can be stored at −80°C prior to lysate preparation.</p>
<ol>
<li>Grow appropriate yeast strains (tagged and untagged control, for example yEG001 and yEG401) in YPD for harvesting cell pellets.
<ol type="a">
<li>Inoculate 20 mL of YPD medium with a single colony and grow at 30°C for 16–18 h in a shaking incubator at 220 rpm until saturated.</li>
<li>The next day, measure the optical density at 600 nm (OD<sub>600</sub>) of the yeast culture using a spectrophotometer. Dilute the culture to an OD<sub>600</sub> = 0.2 in 2 L of sterile YPD medium in a new flask.
<ol type="i">
<li>The volume (V) of saturated culture that is used for inoculation of 2 L is calculated using this formula: Volume (L) = (0.2 OD X 2 L)/OD<sub>600</sub> saturated culture</li>
</ol>
</li>
<li>Grow at 30°C until the cells reach OD<sub>600</sub> ∼1.0 (5–6 h).</li>
</ol>
</li>
<li>Harvest cells by centrifugation.
<ol type="a">
<li>Pellet culture in 500 mL bottles by centrifugation (Allegra X014R) at 4300 <b>×</b> <i>g</i> for 30 min at 4°C. Pour off supernatant.</li>
<li>Wash pellet by resuspending in 40 mL ice-cold sterile, deionized water. Transfer to a 50 mL conical centrifuge tube.</li>
<li>Centrifuge at 2850 <b>×</b> <i>g</i> for 5 min at 4°C. Pour off supernatant.</li>
</ol>
</li>
<li>Resuspend cells in a volume of lysis buffer equivalent to the approximate size of the pellet (e.g., If the pellet is ∼5 mL, resuspend in 5 mL of buffer), as shown in <a href="#fig1">Figure 1</a>A<b>.</b>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1170-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Harvesting yeast cells and making yeast suspension beads</div>
<p>(A) Yeast cell pellet in 50 mL conical tube.</p>
<p>(B) Yeast suspension beads collected in a metal mesh strainer after being frozen in liquid nitrogen.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After centrifugation, the cells will be pelleted at the bottom of the conical centrifuge tube. The graduation marking for volume on the conical tube can be used as an approximation for the size of the pellet.</div>
<ol start="4">
<li>Using a 10 mL serological pipette, add the cell suspension drop-wise into liquid nitrogen in a douer or ice bucket, as shown in <a href="#mmc1">Methods video S1</a>.
<ol type="a">
<li>Use a metal strainer or pipette tip rack insert to collect the yeast suspension beads from the liquid nitrogen (<a href="#fig1">Figure 1</a>B) and transfer them to a 50 mL conical tube. Immediately store the tube at −80°C.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The yeast suspension beads can be stored at −80°C for up to 3 weeks prior to use. (Longer storage may also be feasible but has not been tested in our lab.)</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1170-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Preparation of yeast suspension beads from cell lysate by dropping from a pipette into liquid nitrogen in an ice bucket, related to step 4</div>
</figcaption>
</figure>
<h3 id="sec3.2">Prepare the yeast lysate for immunoprecipitation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>In the following steps, the yeast cells were lysed and the extract is clarified prior to performing the immunoprecipitation.</p>
<div class="critical">
<span class="critical-title">Critical:</span> During the lysate preparation, all steps are performed with ice-cold buffers (stored at 4°C and placed on ice prior to use) and the lysate should be kept cold at all times in an ice bucket and/or in a cold room.</div>
<ol start="5">
<li>In the cold room, fill a pre-chilled coffee grinder with a few pellets of dry ice. Add the yeast suspension beads and grind for 1 min until it is a fine powder (<a href="#fig2">Figures 2</a>A and 2B).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1170-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Grinding yeast suspension beads to prepare lysate</div>
<p>(A) Coffee grinder with small pellets of dry ice and yeast suspension beads.</p>
<p>(B) Ground powder of yeast suspension beads and dry ice after 1 min of processing in the coffee grinder.</p>
</figcaption>
</figure>
<ol type="a">
<li>Pour/scrape the ground cells into a 14 mL polypropylene centrifuge tube. Place the tube on ice and allow the dry ice to evaporate.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A similar approach can also be performed with a metal, steel blade blender, although we have found these are more prone to freezing during lysis. If dry ice is not available, liquid nitrogen can also be used, as described elsewhere (<a href="#bib1">DeCaprio and Kohl, 2020</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> For a 5 mL pellet volume, two 1.5 cm long pieces of dry ice work well. The dry ice evaporation time can vary from 60 to 90 min at 4°C. Occasional swirling speeds up the process.</div>
<ol start="6">
<li>Once the extract is mostly liquid, add protease and phosphatase inhibitors and detergent as described:
<ol type="a">
<li>Add 1 mM PMSF and 1<b>×</b> EZBlock Protease Inhibitor Cocktail IV (1:1000 dilution of the stock solution).</li>
<li>Add 1<b>×</b> PhosSTOP phosphatase inhibitor to the lysate from a 10<b>×</b> stock solution following recommendations from the commercial vendor.</li>
<li>Add NP-40 to a final concentration of 0.4% and invert 10 times to mix well.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A 10<b>×</b> stock solution of the PhosSTOP phosphatase inhibitors can be obtained by dissolving one tablet in 1 mL of deionized water. This can be stored at 4°C for 1 month or −20°C for 6 months. Alternatively, the tablet can be directly dissolved into the lysate. To speed the process, the PhosSTOP tablet can be placed on a piece of parafilm and cut into smaller pieces with a clean razor blade before adding to the lysate. One tablet gives a 1<b>×</b> concentration of phosphatase inhibitors in 10 mL of buffer.</div>
<div class="optional">
<span class="optional-title">Optional:</span> To shear nucleic acids in the lysate, sonicate lysates 3 times for 30 seconds each at 15% output with at least 1 minute on ice between each round of sonication. This is recommended for proteins that interact with chromatin or other nucleic acid complexes that may be insoluble following lysate clarification and was performed for the FLAG-Set5 purification used as an example.</div>
<ol start="7">
<li>Clarify the lysate by centrifugation at 3094 <b>×</b> <i>g</i> for 10 min at 4°C.
<ol type="a">
<li>During the centrifugation, prepare the anti-FLAG magnetic beads as follows:
<ol type="i">
<li>Pipet 75 μL (per IP) of 50/50 bead slurry into a microfuge tube (for 2 IPs, use 150 μL).</li>
<li>Add 1 mL of yeast lysis buffer to the tube and mix by inverting 5–6 times.</li>
<li>Place the tube in the magnetic stand and allow beads to collect against the wall of the tube.</li>
<li>Remove supernatant and repeat wash with 1 mL of lysis buffer, mix by inversion. Place it back on the magnetic stand, allow the beads to collect on the wall of the tube, and remove supernatant.</li>
<li>After the final wash, resuspend beads in a bead-equivalent volume of lysis buffer to generate a 50/50 slurry.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If performing more than 2 IPs, use 15 mL conical tubes for bead washes with 3 mL of wash buffer for each wash.</div>
<div class="critical">
<span class="critical-title">Critical:</span> When pipetting beads, use wide-bore, low-retention tips. Also, do not vortex beads. Always use gentle inversion or a rotator for resuspending beads</div>
<ol start="8">
<li>Transfer the supernatant to a 15 mL conical tube and place clarified lysate on ice in an ice bucket.</li>
</ol>
<h3 id="sec3.3">Set up the immunoprecipitation (IP)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>The protein concentration of the lysates is measured to standardize the concentration within the immunoprecipitations (IPs) and the lysates are incubated with the washed anti-FLAG magnetic beads.</p>
<ol start="9">
<li>Determine the protein concentration of the lysate using a Bradford assay.
<ol type="a">
<li>Add 1 mL of Bradford reagent to disposable cuvettes for each protein sample to be tested including a blank control.</li>
<li>Add 1 μL of lysate to the cuvette, cover the top of each cuvette with a small piece of parafilm, and invert the cuvettes 4–6 times to mix the solution well. Incubate the cuvettes at 25°C for 10 min.</li>
<li>Using a visible light spectrophotometer, measure the absorbance at 595 nm for each sample.</li>
<li>Based on the standard curve and the dilution factor used (1:1000), use the absorbance values to calculate the protein concentration for each of the lysates.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A standard curve should be developed using BSA or other protein of known quantity. The standard curve should encompass protein concentrations in the range of 4 mg/mL to 40 mg/mL and the absorbance measured in yeast lysis buffer used for preparing the IPs.</div>
<ol start="10">
<li>Normalize the protein concentration for tagged and untagged protein lysate and make up additional volume with the yeast lysis buffer if needed.
<ol type="a">
<li>Remove 100 μL from the lysate to use as a total lysate sample for future western blot analysis. Add 25 μL of 5<b>×</b> SDS sample buffer, boil for 5 min, and store at −20°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Using the protocol as outlined above, an expected lysate concentration is approximately 20 μg/μL. All lysates used for IPs should be normalized to the same concentration in a volume of 10–13 mL. In this case, the total protein used per IP ranges from 200–260 mg.</div>
<ol start="11">
<li>Add 75 μL of the washed 50/50 slurry of the FLAG magnetic beads to the normalized lysate in 15 mL conical tubes. Rotate the samples on a tube rotator at 8 rpm for 3 h at 4°C.</li>
</ol>
<h3 id="sec3.4">Wash IPs and elute tagged protein using 3xFLAG peptide</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>Following the IP, the beads are washed to remove proteins non-specifically bound to the beads and the FLAG-tagged protein is natively-eluted using 3x FLAG peptide.</p>
<ol start="12">
<li>After 3 h of incubation, place the tubes in the magnetic stand and allow the beads to collect on the wall of the tube. Remove supernatant (flow-through) by pipetting. Save a 100 μL aliquot for western blot analysis.</li>
<li>Wash the beads 3 times with yeast lysis buffer.
<ol type="a">
<li>Add 5 mL of cold yeast lysis buffer to the beads. Rotate for 5 min at 4°C.</li>
<li>Place the tubes in the magnetic stand, allow the beads to collect, and remove the supernatant.</li>
<li>Repeat the wash with 5 mL of yeast lysis buffer.</li>
<li>Repeat the wash as described above except use 5 mL of yeast lysis buffer supplemented with 0.5% NP-40.</li>
<li>Transfer the beads to a new microfuge tube using 500 μL of yeast lysis buffer + 0.5% NP-40. Wash the old tube and pipette tip with an additional 500 μL of buffer and add the buffer and any more beads collected to the new tube.</li>
<li>Place the tubes in the magnetic stand, allow the beads to collect, and remove the supernatant.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use wide-bore, low-retention tips to completely collect all beads and transfer to a new tube.</div>
<ol start="14">
<li>Prepare elution buffer by adding 0.2 mg/mL 3xFLAG peptide (from a 5 mg/mL stock of 3xFLAG peptide) to yeast lysis buffer.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Always prepare the FLAG elution buffer just before use.</div>
<ol start="15">
<li>Add 50 μL of FLAG elution buffer to the tube with FLAG magnetic beads from Step 13. Rotate the samples on a tube rotator at 8 rpm for 30 min at 4°C.
<ol type="a">
<li>Place the tubes in the magnetic stand, allow the beads to collect on the side of the tube, and carefully transfer the supernatant to a new tube, avoiding disturbing the beads. This is FLAG eluate #1.</li>
</ol>
</li>
<li>Perform a second elution by adding 50 μL of FLAG elution buffer to the beads and rotate at 4°C for 30 min.
<ol type="a">
<li>Place the tubes in the magnetic stand, allow the beads to collect on the side of the tube, and carefully transfer the supernatant to a new tube, avoiding disturbing the beads. This is FLAG eluate #2.</li>
</ol>
</li>
<li>Remove 15 μL of Eluate #1 and #2 into new tubes. Add 3.8 μL of 5<b>×</b> SDS sample buffer, boil for 5 min, and store at −20°C for use in SDS-PAGE followed by silver stain or western blotting. Freeze the remaining sample in liquid nitrogen and store at −80°C until mass spectrometry is performed.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The gel analysis samples can be stored at −20°C prior to performing SDS-PAGE and the samples for mass spectrometry can be stored at −80°C prior to processing for mass spectrometry.</div>
<h3 id="sec3.5">SDS-polyacrylamide gel electrophoresis (PAGE) and protein detection by silver stain</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>To verify that the FLAG-tagged protein has been eluted from the beads, use SDS-PAGE followed by silver stain (using Invitrogen SilverQuest Silver staining kit or similar) or other staining procedure for detecting low quantities of protein. A western blot with anti-FLAG antibody can also be performed, however it may not accurately reflect the quantity of protein nor indicate the presence of co-precipitating proteins. This protocol outlines the procedure for hand-casting gels, however commercial pre-cast gels can also be used.</p>
<ol start="18">
<li>Set up the glass plates and casting apparatus for pouring the polyacrylamide gel based on the manufacturer’s instructions associated with the gel apparatus and running box.</li>
<li>Prepare an 8% resolving gel mixture in a 15 mL conical tube using the recipe outlined in the Materials and Equipment section.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The acrylamide percentage of the resolving gel depends on the molecular weight of the target protein. Depending upon the size of the protein one can use a higher or lower percentage resolving gel.</div>
<ol start="20">
<li>Prepare the resolving gel by quickly pouring the mixture from the conical tubes between the glass plates in the casting apparatus, leaving approximately 3 cm space below the top of the short glass plate. This will use approximately 4.5 mL of the gel mixture.
<ol type="a">
<li>Carefully pour a layer of isopropanol over the top of the resolving gel to level out the top of the resolving gel.</li>
<li>Let the resolving gel stand for 20–30 min until it is polymerized.</li>
</ol>
</li>
<li>Once the resolving gel has polymerized, prepare the 4% stacking gel mixture.</li>
<li>To make the stacking gel, carefully remove the isopropanol layer by pouring, performing a quick wash with 1 mL of deionized water, and wick the space to remove any residual isopropanol using a Kimwipe or similar.</li>
<li>Using a pipette, add the stacking gel mixture on top of the resolving gel between the glass plates. Use enough stacking gel mixture to completely fill the gap between the plates.</li>
<li>Quickly place a 10-well comb between the plates and allow the gel to polymerize for approximately 20 min.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The polyacrylamide gels can be poured ahead of time and stored at 4°C or used right away. To store the gel, wrap it in a damp paper towel and then wrap again in plastic wrap, completely sealing all edges so that it does not dry out. Store at 4°C for up to one week.</div>
<ol start="25">
<li>When ready to run the SDS-PAGE, assemble the gel electrophoresis unit and place it in the buffer tank.
<ol type="a">
<li>Make 1 L of 1<b>×</b> SDS running buffer by diluting the 10<b>×</b> SDS running buffer with deionized water. Pour buffer in the chamber until the wells of the gel are covered and to the level indicated on the gel box by the manufacturer.</li>
<li>Load 18 μL of the IP samples in SDS sample buffer and 3 μL of the molecular weight marker.</li>
</ol>
</li>
<li>Attach the gel box to the power supply and run the gel at a constant 200 V for approximately 45 min, or until the dye front reaches the bottom of the gel.</li>
<li>Remove the gel from the running apparatus and carefully open the glass plates. Use a plastic spatula to trim off the stacking gel and, if desired, to remove excess gel outside of the lanes of interest.</li>
<li>Place the gel in a clean staining box and rinse briefly with deionized water.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Nalgene staining box recommended here contains an LDPE plug at the bottom for easy drainage of solutions without having to handle the gel during staining.</div>
<ol start="29">
<li>Follow the steps for silver staining as described in the SilverQuest™ Silver Staining Kit protocol.
<ol type="a">
<li>Following the staining procedure, the gel can be washed several times in deionized water and imaged on a white light imaging platform or dried and stored for future reference.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The silver nitrate in the silver staining kit is hazardous and may cause skin or eye irritation if exposed. In addition, it should be disposed of using proper waste handling procedures according to local and state regulations. It should not be discarded in the sink.</div>
<div class="note">
<span class="note-title">Note:</span> Once elution of the target protein has been confirmed by SDS-PAGE, the samples can be prepared for and subjected to mass spectrometry following protocols relevant to the available facilities and instrumentation. In addition, the goal of identifying phosphosites or other post-translational modifications or protein interaction partners should be considered when choosing conditions for protein digestion and other steps required prior to mass spectrometry. Examples of appropriate methods and protocols are described elsewhere (<a href="#bib2">Jaiswal et al., 2020</a>; <a href="#bib7">Separovich et al., 2021</a>).</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol provides a method for the purification of epitope-tagged factors, such as lysine methyltransferases or other chromatin regulators, from yeast in order to detect post-translational modifications, primarily phosphorylation. Following the purification, it is expected that nanogram quantities of the target protein will be obtained. The purified protein should be compared to a purification from a yeast strain lacking the FLAG epitope tag (untagged) to confirm that the target protein is purified above any background signal. An example purification of a FLAG-tagged version of the lysine methyltransferase Set5 compared to an untagged control is shown in <a href="#fig3">Figure 3</a>. Mass spectrometry analysis of both the untagged and FLAG-tagged purifications should be performed to ensure proper identification of the target protein (particularly if it is a low abundance protein). In the example provided, we sought to identify phosphorylated residues within Set5, however other post-translational modifications as well as protein interaction partners can also be detected using this method. This protocol can also be performed under different growth conditions or in different genetic backgrounds to monitor changes in phosphorylation, other PTMs, or protein interactors due to environmental or genetic changes.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1170-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. SDS-PAGE with silver stain of 3xFLAG-Set5 purification</div>
<p>Lane 1 shows the anti-FLAG IP from the untagged, wild-type strain and lane 2 is the IP from the strain expressing FLAG-Set5. FLAG peptide eluates #1 and #2 were combined in this gel. Molecular weight markers are indicated.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The use of an epitope tag to purify the target protein may cause altered activity of the protein. The catalytic activity of lysine methyltransferases, or other enzymes, may be impacted by the location or sequence of the tag. The epitope tag may also disrupt localization or protein-interactions in the cell, which may impact the state of phosphorylation. For Set5, an N-terminal FLAG tag was used due to predicted disruption of the enzymatic activity of Set5 with a C-terminal tag. The functionality of the tagged alleles should be tested using genetic assays or by following methylation activity in cells, if possible.</p>
<p>This protocol purifies the target protein from a whole cell lysate that is enriched for chromatin proteins due to shearing of nucleic acids with sonication. However, some target proteins may have altered phosphorylation profiles depending on their subcellular localization. The data captured from this protocol will not account for this possibility, as the localization pattern of the target protein is disrupted during the lysis and all protein is pooled together. For example, Set5 shows chromatin binding and also has a large cytoplasmic pool. From this experiment, we cannot determine whether the identified phosphosites are specific to one pool of Set5. Alternative lysis procedures using subcellular fractionation protocols would address this limitation.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The whole cell extract concentration is low and is not sufficient for performing the IP (see Step 9).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>This problem is most likely due to insufficient quantities of yeast cell growth or poor lysis. If the total cell pellet amount is less than described in this protocol due to growth or strain differences, increase the culture volume used for harvesting cells (see Step 1). The cells can also be grown to higher density, however significant physiological changes can occur as cells shift from log-phase growth through the post-diauxic shift to stationary phase. This may alter the abundance and/or post-translational modification status of the target protein.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The target protein is detected by SDS-PAGE but shows high levels of degradation.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This problem may be caused by insufficient inhibition of proteases in the whole cell lysate or due to intrinsic instability of the target protein. The concentration of the yeast protease inhibitor cocktail can be increased in the extract up to a 1:100 dilution (see Step 6). All buffers should be ice-cold when used and the samples stored on ice or in a mixture of ice with some water to keep them cold. In addition, a protease-deficient strain of yeast can be used for performing the purification, such as a strain similar to DDY1810 (MATa, <i>leu2</i>Δ, <i>trp1</i>Δ, <i>ura3-52, prb1-1122, pep4-3, pre1-451</i>) (<a href="#bib8">Shang et al., 2003</a>).</p>
<h3 id="sec6.5">Problem 3</h3>
<p>There is low or undetectable signal for the target FLAG-tagged protein on the silver stain gel or anti-FLAG western blot.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Low or undetectable amounts of purified protein may be due to low levels of the protein in the whole cell extract, inefficiency in the anti-FLAG IP, or poor elution from the anti-FLAG magnetic beads. Some lysine methyltransferases and other chromatin-modifying enzymes are lowly-expressed in yeast. For endogenous proteins expressed at low levels, increasing the culture volume can dramatically improve the recovery of the target protein in the IP (see Step 1). We have successfully performed purifications from cells harvested from 10–12 L of yeast culture.</p>
<p>To test the efficiency of the IP, collect the flow-through sample after the anti-FLAG IP and perform a western blot on this sample compared to the whole cell extract. If the protein is present in equal amounts in each sample, the amount of anti-FLAG beads can be increased for the IP. A series of IPs titrating the amount of beads using the same extract concentration will help determine the appropriate ratio of anti-FLAG beads to the amount of extract used. Increasing the anti-FLAG beads in the IP may increase the background signal obtained, however this can be determined by comparing the IP eluates from tagged and untagged strains on an SDS-PAGE gel stained with silver stain.</p>
<p>It is also possible that the elution of the target protein with the 3xFLAG peptide is inefficient (see Step 15). To test if the FLAG-tagged protein is retained on the anti-FLAG beads but not eluted, the beads can be resuspended in 1<b>×</b> SDS sample buffer and boiled for 5 min. This sample can be subjected to SDS-PAGE and either anti-FLAG western blotting or silver staining to determine the amount of protein retained on the beads. If significant amounts of protein are still associated with the beads, the incubation time with the 3xFLAG peptide may be increased or the number of elution steps may be increased (see Step 15). Alternatively, another elution method, such as using 4% SDS or a low pH buffer, such as 100 mM glycine pH 3.0, can be employed.</p>
<p>Another possible cause of low recovery of the target protein is that the FLAG epitope tag is inaccessible when fused to the protein. This can potentially be addressed by moving the tag to the other terminus (C- or N). Alternatively, a different epitope tag, such as 3xHA, 9<b>×</b>MYC, or GFP can be tried, though this will necessitate using the respective antibodies and alternative elution methods dependent on the tag. Changing the sequence of the tag may alter its accessibility. In addition, if an antibody recognizing the target protein is available, this can also be used but should be carefully validated for specificity.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>There are high levels of background signal in the IP from the untagged yeast strain, making it difficult to detect the endogenously-purified FLAG-tagged protein.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>The background can be reduced by adding sodium chloride to the yeast lysis buffer (up to 500 mM NaCl if the primary goal is phosphorylation or other post-translational modification detection and up to 300 mM if protein interactors are sought). The number of washes can also be increased (see Step 13). If the protein is abundant and the anti-FLAG IP is efficient, the total amount of whole cell extract used in the IP can be decreased to reduce background (see Step 10).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>There is dark or uneven background stain on the silver stained polyacrylamide gel.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>It is critical that reagents and plasticware used in silver staining is free of protein and other contaminants. Use a dedicated staining tray that is only used for silver stain gels and wash thoroughly after use. Use ultra-pure water for washing the gel and making all solutions. Also, perform the washing steps thoroughly and as described for the Silverquest Silver Staining kit (see Steps 28 and 29).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">DeCaprio, J. and Kohl, T.O. (2020). Lysing yeast cells for immunoprecipitation using a coffee grinder. Cold Spring Harb. Protoc. <i>2020</i>, 482-484. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref1">View at publisher</a></p>
<p id="bib2">Jaiswal, D., Turniansky, R., Moresco, J.J., Ikram, S., Ramaprasad, G., Akinwole, A., Wolf, J., Yates, J.R., and Green, E.M. (2020). Function of the MYND Domain and C-terminal region in regulating the subcellular localization and catalytic activity of the SMYD Family lysine methyltransferase Set5. Mol. Cell Biol. <i>40</i>. e00341–19. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref2">View at publisher</a></p>
<p id="bib3">Longtine, M.S., McKenzie, A., Demarini, D.J., Shah, N.G., Wach, A., Brachat, A., Philippsen, P., and Pringle, J.R. (1998). Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae. Yeast <i>14</i>, 953-961. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref3">View at publisher</a></p>
<p id="bib4">Moqtaderi, Z. and Struhl, K. (2008). Expanding the repertoire of plasmids for PCR-mediated epitope tagging in yeast. Yeast <i>25</i>, 287-292. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref4">View at publisher</a></p>
<p id="bib5">Separovich, R.J., Pang, C.N.I., and Wilkins, M.R. (2020). Controlling the Controllers: regulation of histone methylation by Phosphosignalling. Trends Biochem. Sci. <i>45</i>, 1035-1048. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref5">View at publisher</a></p>
<p id="bib6">Separovich, R.J. and Wilkins, M.R. (2021). Ready, SET, Go: post-translational regulation of the histone lysine methylation network in budding yeast. J. Biol. Chem. <i>297</i>, 100939. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref6">View at publisher</a></p>
<p id="bib7">Separovich, R.J., Wong, M.W.M., Chapman, T.R., Slavich, E., Hamey, J.J., and Wilkins, M.R. (2021). Post-translational modification analysis of Saccharomyces cerevisiae histone methylation enzymes reveals phosphorylation sites of regulatory potential. J. Biol. Chem. <i>296</i>, 100192. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref7">View at publisher</a></p>
<p id="bib8">Shang, C., Hazbun, T.R., Cheeseman, I.M., Aranda, J., Fields, S., Drubin, D.G., and Barnes, G. (2003). Kinetochore protein interactions and their regulation by the Aurora kinase Ipl1p. Mol. Biol. Cell <i>14</i>, 3342-3355. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00651-1/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Erin M. Green (<a href="mailto:egreen@umbc.edu">egreen@umbc.edu</a>).</p>
<h4>Materials availability</h4>
<p>All materials associated with this study and the original research study (<a href="#bib2">Jaiswal et al., 2020</a>) are available by request to the lead contact, Erin M. Green (<a href="mailto:egreen@umbc.edu">egreen@umbc.edu</a>).</p>
<h4>Data and code availability</h4>
<p>All data associated with this study and the original research study (<a href="#bib2">Jaiswal et al., 2020</a>) are available within the published articles. The raw data for the mass spectrometry associated with <a href="#bib2">Jaiswal et al. (2020)</a> is deposited in the PRIDE partner repository with the Proteome-Xchange Consortium. The data set identifier is Proteome-Xchange: PXD014756.</p>
<h3>Acknowledgments</h3>
<p>The authors acknowledge members of the Green Lab for technical feedback and comments on the manuscript. We also thank James Moresco and John R. Yates III (Scripps Research Institute) for performing the mass spectrometry analysis associated with the original research published in <a href="#bib2">Jaiswal et al. (2020)</a>. This work is funded by <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (NIH) grants R01GM124342 and R21AG064507 to E.M.G.</p>
<h3>Author contributions</h3>
<p>D.J. and R.T. performed experiments and optimized the protocol. E.M.G. contributed to the experimental design, supervised the work, and obtained funding and resources. D.J. and E.M.G. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>