<article>
<section>
<h1 id="header">Fast <i>in vitro</i> protocol for the visualization and quantitative high-throughput analysis of sprouting angiogenesis by confocal microscopy</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Lanette Kempers,<sup><a href="#aff1">1</a></sup> Ivo van der Bijl,<sup><a href="#aff1">1</a></sup> Anne-Marieke D. van Stalborch,<sup><a href="#aff1">1</a></sup> Bas Ponsioen,<sup><a href="#aff2">2</a></sup> and Coert Margadant<sup><a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Sanquin Research and Landsteiner laboratory, Amsterdam University Medical Center, 1066 CX Amsterdam, The Netherlands</p>
<p id="aff2"><sup>2</sup>Molecular Cancer Research, Center for Molecular Medicine, University Medical Center Utrecht, Utrecht University, 3584 CX Utrecht, The Netherlands</p>
<p id="aff3"><sup>3</sup>Angiogenesis Laboratory, Department of Medical Oncology, Cancer Center Amsterdam, Amsterdam University Medical Center, 1081 HV Amsterdam, The Netherlands</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:c.margadant@amsterdamumc.nl">c.margadant@amsterdamumc.nl</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100690">10.1016/j.xpro.2021.100690</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We describe an optimized, cost-effective, reproducible, and robust protocol to study sprouting angiogenesis in glass-bottom 96-well plates by confocal microscopy, ideal for screening of drug or shRNA libraries. Effective and stable knockdown of gene expression in primary endothelial cells is achieved by lentiviral transduction. Dynamic behavior of individual cells and fluorescent proteins is analyzed by time-lapse imaging, while competitive advantages in tip cell formation are assessed using mixtures of differentially labeled cell populations. Finally, we present a macro for high-throughput analysis.</p>
<p>For complete information on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00397-X/sref1">van der Bijl et al. (2020)</a> and <a class="external-link" href="https://doi.org/10.1007/s10456-021-09788-4">Kempers et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>High-throughput microscopy analysis of sprouting angiogenesis for drug/shRNA screening</li>
<li>Analysis of dynamic cell and protein behavior during sprouting by time-lapse microscopy</li>
<li>Mosaic assays to image competitive advantages in tip cell behavior</li>
<li>Macro for fast automated quantitative analysis</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/994-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Experimental design considerations</h3>
<p>The current protocol is designed to achieve fast and robust human umbilical vein endothelial cell (HUVEC) sprouting in fibrin gels, and to visualize sprouting and tip cell characteristics, such as the formation of filopodia, by confocal microscopy (<a href="#fig1">Figure 1</a>). With the described cell culture set-up, maximal numbers of HUVECs are obtained with a limited number of culture steps, thus allowing to perform sprouting screens within the life-span of primary endothelial cells. Instead of freshly isolating endothelial cells we use commercially available pools of HUVECs, which we expand to the extent that the same batch of cells can be used for a large number of conditions during consecutive experiments. The described sprouting assay allows co-culture with several other cell types and can be used as a platform for the screening of clinical samples as well as drug or short hairpin (sh) RNA libraries, thus providing a unique and versatile set-up for both fundamental research into the regulation of angiogenesis and screening of patient material for diagnostic and therapeutic goals. Homogeneous and stable suppression of target gene expression using shRNAs is achieved by lentiviral transduction followed by puromycin selection, which quickly generates a population of cells that can be used for functional assays during a week. Building on previously established protocols to analyze sprouting angiogenesis (<a href="#bib3">Eglinger et al., 2017</a>; <a href="#bib6">Nakatsu and Hughes, 2008</a>; <a href="#bib7">Nowak-Sliwinska et al., 2018</a>), the assay described here is performed in ‘half-area’ 96-well glass-bottom plates, which require small volumes of reagents, allow fast, robust, and highly efficient sprouting already within 20–48 h, and are suitable for medium- to high-throughput screening. While the focus here is primarily on the initial phases of sprouting, we perform this assay in the absence of fibroblasts, which are required for lumenization (<a href="#bib9">Newman et al., 2011</a>). Furthermore, we also describe how to perform live-cell labeling in this set-up, using cell-permeable fluorescent dyes or expression of fluorescent proteins, as well as mosaic sprouting assays with a mixture of differentially labeled cells. Important criteria for selecting the right kind of microscope, especially when performing live imaging, include phototoxicity, speed/temporal resolution and sensitivity. While we have used confocal systems for the applications described here, confocals are relatively slow and can damage the sample, compared to some other systems. This can be overcome with resonant scanning, at the cost of reduced resolution. Alternatives to confocal systems include light sheet microscopes and two-photon microscopes. While the former have little phototoxicity and are ideal for the imaging of fast dynamics, the latter have superior confocality and penetration depth but are potentially harmful because of the high energy input. As different research questions will require specific adjustments to experimental design, we also present some alternatives and modifications to the protocol and conditions described here.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/994-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Assessment of sprouting angiogenesis by confocal microscopy</div>
<p>(A) Maximum projection of confocal z-stack comprised of 142 z-slices showing sprouting after 48 h, stained with phalloidin to visualize actin filaments (F-actin; pseudocolored <i>magenta</i>) and Hoechst to stain the nuclei (pseudocolored <i>cyan</i>). Bar, 75 μm.</p>
<p>(B) Depth coding (using LUT <i>Spectrum</i>) of the same stack of images.</p>
<p>(C and D) Zooms of insets 1 and 2, showing tip cell morphology in more detail.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Prepare cell culture materials and buffers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~4 h</div>
<ol>
<li>Prepare mammalian cell culture media as described in the tables below.</li>
</ol>
<table id="undtbl1">
<caption>Preparation of Dulbecco’s Modified Eagle’s Medium (DMEM)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>440 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>200 mM L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>D-Glucose</td>
<td>4.5 g/L</td>
<td>2.25 g</td>
</tr>
<tr>
<td>100 mM Sodium pyruvate</td>
<td>1 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin</td>
<td>100 U/mL</td>
<td> </td>
</tr>
<tr>
<td>Streptomycin</td>
<td>100 μg</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">DMEM complete medium can be stored for up to 6 months at 2°C–8°C.</p>
<table id="undtbl2">
<caption>Preparation of Endothelial Growth Medium-2 (EGM-2)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EBM-2</td>
<td>n/a</td>
<td>440 mL</td>
</tr>
<tr>
<td>Bullet kit</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>200 mM L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin</td>
<td>100 U/mL</td>
<td> </td>
</tr>
<tr>
<td>Streptomycin</td>
<td>100 μg</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">EGM-2 complete medium can be stored for up to 4 months at 2°C–8°C.</p>
<table id="undtbl3">
<caption>Preparation of Iscove’s Modified Dulbecco’s Medium (IMDM)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>IMDM</td>
<td>n/a</td>
<td>440 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>200 mM L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin</td>
<td>100 U/mL</td>
<td> </td>
</tr>
<tr>
<td>Streptomycin</td>
<td>100 μg</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">IMDM complete medium can be stored for up to 3 months at 2°C–8°C.</p>
<ol start="2">
<li>Coat cell culture flasks and/or dishes with fibronectin (1 μg/mL) or gelatin (0.1%).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Coating with fibronectin is efficient at 37°C for 60 min, gelatin requires only 10 min. HEK293T cells are poorly adherent and fibronectin coating will facilitate their adhesion and proliferation. HUVEC culture is stimulated by gelatin coating.</div>
<div class="note">
<span class="note-title">Note:</span> For coating of tissue culture plastic for HEK293T cells, fibronectin can also be substituted by serum, which contains large quantities of fibronectin and vitronectin that will be absorbed by the plastic and stimulate integrin-mediated cell adhesion. Serum can be re-used multiple times for this purpose.</div>
<ol start="3">
<li>Prepare buffers and beads as described in the tables below.</li>
</ol>
<table id="undtbl4">
<caption>Preparation of PBS with MgCl2/CaCl2</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td>MgCl2</td>
<td>0.5 mM</td>
<td> </td>
</tr>
<tr>
<td>CaCl2</td>
<td>1 Mm</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">PBS with MgCl2/CaCl2 can be stored for up to 12 months at 15°C–30°C.</p>
<table id="undtbl5">
<caption>Preparation of 4% paraformaldehyde (PFA)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td>PFA</td>
<td>4%</td>
<td>40 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">PFA can be stored for up to 12 months at −20°C.</p>
<div class="note">
<span class="note-title">Note:</span> PFA is flammable, corrosive, toxic, carcinogenic and is an irritant for the skin, eyes, and the respiratory tract. A lab coat, gloves, face mask, and safety goggles should be worn when handling PFA. All steps should be performed inside a fume hood, and waste should be disposed of in a special container.</div>
<div class="critical">
<span class="critical-title">Critical:</span> PFA should be dissolved first in 800 mL PBS inside a fume hood, while continuously stirring. Add NaOH until the solution clears, filter it, adjust the pH to 7.0 with HCl and fill up to 1 L with PBS. Aliquot the solution for long-term storage at −20°C.</div>
<table id="undtbl6">
<caption>PBS/gelatin preparation</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td>Gelatin</td>
<td>0.1%</td>
<td>1 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">PBS-gelatin can be stored for up to 6 months at 4°C.</p>
<div class="note">
<span class="note-title">Note:</span> PBS-gelatin should be sterilized before use in tissue culture, for instance using bottle top filters from Millipore (pore size 0.22 μm).</div>
<table id="undtbl7">
<caption>Bead suspension</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cytodex beads</td>
<td>20 g/L</td>
<td>1 g</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">The beads can be stored for up to 12 months at 20°C–22°C.</p>
<div class="note">
<span class="note-title">Note:</span> This will generate a stock solution of approximately 3 × 10<sup>6</sup> beads/L. The beads should be sterilized before use in an autoclave at 120°C.</div>
<h3 id="sec1.3">HUVEC expansion and storage</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="4">
<li>Warm cell culture media to 37°C before use.</li>
<li>Add a large volume of IMDM containing 10% fetal bovine serum, 2 mM L-glutamine, and 1% penicillin/streptomycin (~400 μL/cm<sup>2</sup>) to a T25 culture flask.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use IMDM here only for economical reasons. Since this medium is discarded (see step 11 below) we prefer to not use EGM-2 for this step.</div>
<ol start="6">
<li>Place the flask at 37°C and 5% CO<sub>2</sub>.</li>
<li>Thaw 1 Lonza cryovial of HUVECs, carefully aspirate cell suspension, and gently pipet into a coated T25.</li>
<li>Distribute the cells over the culture flask by gentle rocking.</li>
<li>Place the flask at 37°C and 5% CO<sub>2</sub> for ~2 h.</li>
<li>2 h after seeding, check whether the cells have adhered and have started to spread over the surface.</li>
<li>Replace the IMDM with warm EGM-2 (~200 μL/cm<sup>2</sup>).</li>
<li>After 2 days, remove the EGM-2 medium and wash cells with a large volume of 37°C PBS.</li>
<li>Add Trypsin/EDTA (~10 μL/cm<sup>2</sup>) and distribute over entire surface.</li>
<li>Place cells back at 37°C and 5% CO<sub>2</sub> for 1 min.</li>
<li>Check under microscope whether cells have rounded up and/or detached.</li>
<li>Gently tap flask to fully detach rounded cells.</li>
<li>Add Trypsin Neutralization Solution (10–20 μL/cm<sup>2</sup>) for ~30 s.</li>
<li>Add EGM-2 and resuspend cells.</li>
<li>Seed cells into a coated T150.</li>
<li>After 2 days, trypsinize the cells as described above and transfer to coated T150s (usually 4–6), depending on cell density.</li>
<li>After 2 days, freeze cells for cryo storage, 25 cm<sup>2</sup> of confluent cells per vial (this yields usually 30–40 vials).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure HUVECs grow sufficiently (<a href="#sec7.1">troubleshooting 1</a><b>)</b>
</div>
<h3 id="sec1.4">Preparation of constructs for lentiviral transduction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>Constructs containing shRNAs are obtained from the TRC MISSION library. Each MISSION shRNA clone is constructed within the lentivirus plasmid vector pLKO.1-puro, containing ampicillin and puromycin resistance genes for selection in bacterial or mammalian cells, respectively, and transformed into <i>Escherichia coli</i> (DH5α). Individual clones are purchased as frozen bacterial glycerol stocks containing Terrific Broth (TB), carbenicillin (100 μg/mL), and 15% glycerol, which can be stored at −80°C in 96-well plates. For isolation of plasmid DNA, bacteria are inoculated in Lennox Broth (LB) containing Ampicillin (100 μg/mL) and grown for 12–16 h at 37°C. The plasmid DNA is purified using the NucleoBond Xtra Midi- or Maxiprep protocol (Macherey-Nagel), according to the manufacturer’s instructions. Constructs needed for lentiviral production are purified from <i>Eschirichia coli</i> Stbl3, grown at 30°C. Required constructs include:</p>
<p><i>pHDM-HgpM2 GAG/POL</i>, encoding the main structural viral proteins and reverse transcriptase</p>
<p><i>pRC-CMV-REV1B</i>, encoding the post-transcriptional regulator Rev1b necessary for efficient synthesis of viral proteins,</p>
<p><i>pHDMG-G VSV ENV</i>, coding for the envelope protein of Vesicular Stomatitis Virus,</p>
<p><i>pHDM-TAT 1B</i>, encoding the Tat1b protein, which facilitates viral entry into target cells</p>
<p><i>pLKO.1-puro</i> encoding the shRNA of interest and puromycin resistance for selection of positive target cells</p>
<div class="note">
<span class="note-title">Note:</span> Sequence confirmation of the insert after plasmid isolation is advised. The following reagents and conditions are optimized for sequencing of shRNA-encoding inserts:</div>
<table id="undtbl8">
<caption>Reaction mix for sequencing reaction</caption>
<thead>
<tr>
<th> </th>
<th>Stock</th>
<th>Final</th>
<th>Dilution</th>
<th>1 RXN</th>
</tr>
</thead>
<tbody>
<tr>
<td>H2O</td>
<td>1×</td>
<td>0.5×</td>
<td>2</td>
<td>6</td>
</tr>
<tr>
<td>DMSO</td>
<td>1×</td>
<td>0.05×</td>
<td>20</td>
<td>1</td>
</tr>
<tr>
<td>Betaine</td>
<td>5M</td>
<td>1M</td>
<td>5</td>
<td>4</td>
</tr>
<tr>
<td>Big Dye Terminator Buffer</td>
<td>5×</td>
<td>1×</td>
<td>5</td>
<td>4</td>
</tr>
<tr>
<td>Big Dye terminator</td>
<td>5×</td>
<td>1×</td>
<td>5</td>
<td>4</td>
</tr>
<tr>
<td>Primer</td>
<td>10 μM</td>
<td>0.25 μM</td>
<td>40</td>
<td>0.5</td>
</tr>
<tr>
<td>DNA</td>
<td>1 μg/mL</td>
<td>0.025 μg/mL</td>
<td>40</td>
<td>0.5</td>
</tr>
</tbody>
</table>
<table id="undtbl9">
<caption>PCR cycling conditions</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>98</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98</td>
<td>30 s</td>
<td> </td>
</tr>
<tr>
<td>Annealing</td>
<td>54</td>
<td>10 s</td>
<td>40</td>
</tr>
<tr>
<td>Extension, 3 s increment</td>
<td>60</td>
<td>2–4 min</td>
<td> </td>
</tr>
</tbody>
</table>
<h3 id="sec1.5">Production of lentivirus-like particles encoding shRNAs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<ol start="22">
<li>Seed human embryonic kidney (HEK) 293T cells 1:4 in fibronectin-coated filter-cap flasks.</li>
<li>The following morning, transfect HEK293T cells with constructs as in <a href="#tbl1">Table 1</a>. The cells should be 50%–80% confluent.
<table id="tbl1">
<caption>Table 1. Transfection mix for the production of lentivirus-like particles</caption>
<thead>
<tr>
<th>
<i>Trans</i>IT + TAT</th>
<th> </th>
<th>cm<sup>2</sup>
</th>
<th>cm<sup>2</sup>
</th>
<th> </th>
</tr>
</thead>
<tbody>
<tr>
<td><b>Surface area per condition</b></td>
<td><b>Ratio</b></td>
<td>1</td>
<td><b>75</b></td>
<td> </td>
</tr>
<tr>
<td><i>pHDMG·G VSV ENV</i></td>
<td>2.0</td>
<td>44</td>
<td>4950</td>
<td>Ng</td>
</tr>
<tr>
<td><i>pHDM·HgpM2 GAG/POL</i></td>
<td>1.0</td>
<td>22</td>
<td>1667</td>
<td>Ng</td>
</tr>
<tr>
<td><i>pRC-CMV-REV1B</i></td>
<td>1.0</td>
<td>22</td>
<td>1667</td>
<td>Ng</td>
</tr>
<tr>
<td><i>pHDM-TAT1B</i></td>
<td>1.0</td>
<td>22</td>
<td>1667</td>
<td>Ng</td>
</tr>
<tr>
<td>
<i>pLKO.1-puro</i> encoding shRNA of interest</td>
<td>13.0</td>
<td>289</td>
<td>21667</td>
<td>Ng</td>
</tr>
<tr>
<td>
<b>Total #ng of DNA/cm</b><sup><b>2</b></sup>
</td>
<td>18.0</td>
<td><b>400</b></td>
<td>30000</td>
<td>Ng</td>
</tr>
<tr>
<td>
<b>Total #μL of <i>Trans</i>IT/cm</b><sup><b>2</b></sup>
</td>
<td> </td>
<td><b>0.40</b></td>
<td>30</td>
<td>mL</td>
</tr>
<tr>
<td>Optimem</td>
<td> </td>
<td>10.00</td>
<td>750</td>
<td>mL</td>
</tr>
</tbody>
</table>
</li>
<li>Dilute TransIT-LT1 (0.4 μL/cm<sup>2</sup>) in Optimem (10 μL/cm<sup>2</sup>) and mix thoroughly.</li>
<li>Add viral constructs (44 ng/cm<sup>2</sup> <i>pHDMG-G VSV ENV</i>, 22ng/cm<sup>2</sup> <i>pHDM-HgpM2 GAG/POL</i>, 22 ng/cm<sup>2</sup> <i>pRC-CMV-REV1B</i>, 22 ng/cm<sup>2</sup> <i>pHDM-TAT1B</i>) to obtain a mastermix.</li>
<li>Add mastermix (10.4 μL/cm<sup>2</sup>) to a tube containing <i>pLKO.1-puro</i> coding for desired shRNA (289 ng/cm<sup>2</sup>).</li>
<li>Leave transfection mix at 20°C–22°C for approximately 20 min in the dark.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not incubate the transfection mixture longer than 30 min.</div>
<ol start="28">
<li>Add transfection mix to the HEK293T cells, place in the incubator at 37°C in the presence of 5% CO<sub>2</sub>.</li>
<li>Remove medium containing the transfection mix ~6 h later, and replace with approximately 100 μL/cm<sup>2</sup> of DMEM.</li>
<li>Place cells back in the incubator at 37°C and 5% CO<sub>2</sub>.</li>
<li>At 48 h after transfection, harvest the culture medium, centrifuge at 500 × <i>g</i> for 5 min, store at 4°C.</li>
<li>At 72 h after transfection, harvest the culture medium, centrifuge at 500 × <i>g</i> for 5 min.</li>
<li>Mix with the 48 h harvest, filter using a 0.45 μm pore filter, aliquot and store at −80°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Protocol steps 28–33 must be performed in a BSL-2 lab. Tight adherence to BSL-2 regulations on sample handling and the disposal of plastic ware and media is required.</div>
<div class="critical">
<span class="critical-title">Critical:</span> A good viral titer is crucial for efficient transduction (<a href="#sec7.3">troubleshooting 2</a>)</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>Eschirichia coli</i> DH5α</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>Eschirichia coli</i> Stbl3</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Aprotinin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A1153</td>
</tr>
<tr>
<td>Betaine</td>
<td>Sigma-Aldrich</td>
<td>B0300</td>
</tr>
<tr>
<td>CaCl2</td>
<td>Merck</td>
<td>CAS 10035-04-08</td>
</tr>
<tr>
<td>CellTracker Green</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#C2925</td>
</tr>
<tr>
<td>CellTracker Red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#C34552</td>
</tr>
<tr>
<td>D-Glucose</td>
<td>Sigma-Aldrich</td>
<td>CAS 50-99-7</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>J.T.Baker</td>
<td>CAS 67-68-5</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle’s Medium (DMEM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#41965-039</td>
</tr>
<tr>
<td>EDTA Triplex III</td>
<td>Merck</td>
<td>CAS 6381-92-6</td>
</tr>
<tr>
<td>Endothelial Basal Cell Growth Medium-2 (EBM-2)</td>
<td>PromoCell</td>
<td>C-22211</td>
</tr>
<tr>
<td>Endothelial Cell Growth Medium-2 with supplements (EGM-2)</td>
<td>PromoCell</td>
<td>C-22011</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS), heat inactivated</td>
<td>Bodinco</td>
<td>5010</td>
</tr>
<tr>
<td>Fibrinogen (Haemocomplettan P)</td>
<td>CSL Behring</td>
<td>Cat#B02BB01</td>
</tr>
<tr>
<td>Fibronectin</td>
<td>Sigma-Aldrich</td>
<td>Cat#F1141</td>
</tr>
<tr>
<td>Gelatin</td>
<td>Sigma-Aldrich</td>
<td>Cat#G1890</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Sigma-Aldrich</td>
<td>Cat#G7513</td>
</tr>
<tr>
<td>Hoechst 33342</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#H3570</td>
</tr>
<tr>
<td>Iscove’s Modified Eagle Medium (IMDM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12440053</td>
</tr>
<tr>
<td>MgCl2</td>
<td>Sigma-Aldrich</td>
<td>CAS 7791-18-6</td>
</tr>
<tr>
<td>Opti-MEM Reduced Serum Medium, GlutaMAX Supplement</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#51985034</td>
</tr>
<tr>
<td>Phosphate-buffered saline (PBS), pH 7.0</td>
<td>Fresenius Kabi</td>
<td>N/A</td>
</tr>
<tr>
<td>Paraformaldehyde (PFA)</td>
<td>Merck</td>
<td>Cat#1.04005</td>
</tr>
<tr>
<td>Phalloidin, Texas Red-conjugated</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#T7471</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>Sigma-Aldrich</td>
<td>P0781</td>
</tr>
<tr>
<td>Polyethylene glycol (PEG) 6000</td>
<td>Sigma-Aldrich</td>
<td>CAS 25322-68-3</td>
</tr>
<tr>
<td>Puromycin</td>
<td>InvivoGen ant-pr-1</td>
<td>ant-pr-1</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11360070</td>
</tr>
<tr>
<td>Thrombin</td>
<td>Sigma-Aldrich</td>
<td>Cat#T1063</td>
</tr>
<tr>
<td>
<i>Trans</i>IT-LT1</td>
<td>Mirus Bio</td>
<td>MIR2360</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat#T8787</td>
</tr>
<tr>
<td>Trypsin Neutralization Solution</td>
<td>Lonza</td>
<td>Cat# CC-5002</td>
</tr>
<tr>
<td>Trypsin</td>
<td>Sigma-Aldrich</td>
<td>Cat#59418C</td>
</tr>
<tr>
<td>VEGF-A (VEGF165), human recombinant</td>
<td>R&amp;D Systems</td>
<td>Cat#293-VE</td>
</tr>
<tr>
<td colspan="3">Experimental models: cells</td>
</tr>
<tr>
<td>Human embryonic kidney (HEK) 293T cells</td>
<td>ATCC</td>
<td>CRL-3216</td>
</tr>
<tr>
<td>Human umbilical vein endothelial cells (HUVECs)</td>
<td>Lonza</td>
<td>C2519A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>
<i>pLKO.1-puro</i> encoding the shRNA of interest and puromycin resistance for selection of positive target cells</td>
<td>TRC Mission Library</td>
<td>N/A</td>
</tr>
<tr>
<td>Control sequence/scrambled sequence</td>
<td>TRC Mission Library</td>
<td>Cat# C002</td>
</tr>
<tr>
<td>
<i>Primer: U6 Forward:</i> 5′-GGACTATCATATGCTTACCG-3′</td>
<td>IDT</td>
<td>Custom Oligo</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>
<i>pHDM-HgpM2 GAG/POL</i>, encoding the main structural viral proteins and reverse transcriptase</td>
<td>Addgene</td>
<td>164441</td>
</tr>
<tr>
<td>
<i>pRC-CMV-REV1B</i>, encoding the post-transcriptional regulator Rev1b necessary for efficient synthesis of viral proteins</td>
<td>Addgene</td>
<td>164443</td>
</tr>
<tr>
<td>
<i>pHDMG-G VSV ENV</i>, coding for the envelope protein of Vesicular Stomatitis Virus</td>
<td>Addgene</td>
<td>164440</td>
</tr>
<tr>
<td>
<i>pHDM-TAT 1B</i>, encoding the Tat1b protein, which facilitates viral entry into target cells</td>
<td>Addgene</td>
<td>164442</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Imaging software pertaining to confocal microscope (used here: Leica LAS X)</td>
<td><a href="https://www.leica-microsystems.com/products/microscope-software/p/leica-las-x-ls/">https://www.leica-microsystems.com/products/microscope-software/p/leica-las-x-ls/</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Optional: <i>Navigator</i> module in Leica LAS X</td>
<td> </td>
<td>N/A</td>
</tr>
<tr>
<td>ImageJ/Fiji</td>
<td><a href="https://imagej.net/Downloads">https://imagej.net/Downloads</a></td>
<td><a href="#bib8">Schindelin et al., 2009</a></td>
</tr>
<tr>
<td>
<i>Sprout Morphology</i> plugin</td>
<td><a href="https://imagej.net/Sprout_Morphology">https://imagej.net/Sprout_Morphology</a></td>
<td><a href="#bib3">Eglinger et al., 2017</a></td>
</tr>
<tr>
<td>Macro “<i>Automated sprout analysis</i>”, generated here</td>
<td><a href="https://github.com/Sprouting-angiogenesis/Automated-sprout-analysis">https://github.com/Sprouting-angiogenesis/Automated-sprout-analysis</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Optional: <i>3D viewer</i> plugin</td>
<td><a href="https://imagej.nih.gov/ij/plugins/3d-viewer/">https://imagej.nih.gov/ij/plugins/3d-viewer/</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Microsoft Excel</td>
<td><a href="https://www.microsoft.com/nl-nl/microsoft-365/excel">https://www.microsoft.com/nl-nl/microsoft-365/excel</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Optional: GraphPad Prism 5.0</td>
<td><a href="http://www.graphpad.com/scientific-software/prism">http://www.graphpad.com/scientific-software/prism</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Cytodex 3 microcarrier beads</td>
<td>Sigma</td>
<td>Cat#C3275</td>
</tr>
<tr>
<td>96-well Half Area High Content Imaging Glass Bottom Microplate</td>
<td>Corning</td>
<td>Cat#4580</td>
</tr>
<tr>
<td>0.45 μm Pore filters</td>
<td>Whatman (GE Healthcare)</td>
<td>Cat#10462100</td>
</tr>
<tr>
<td>0.2 mL Low Profile Thin-walled 8 Tube &amp; Flat Cap Strips</td>
<td>Thermo Scientific</td>
<td>AB-0773</td>
</tr>
<tr>
<td>BigDye Terminator v3.1 Cycle Sequencing Kit</td>
<td>Thermo Fisher Scientific</td>
<td>4337456</td>
</tr>
<tr>
<td>Bottle top filters</td>
<td>Merck Millipore</td>
<td>S2GVU05RE</td>
</tr>
<tr>
<td>Thermocycler</td>
<td>MJ Research</td>
<td>Dyad Disciple</td>
</tr>
<tr>
<td>Autoclave</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>DNA sequencing system</td>
<td>Thermo Fisher Scientific</td>
<td>3730 DNA Analyzer</td>
</tr>
<tr>
<td>Multichannel pipette</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Cell counter</td>
<td>Innovatis</td>
<td>CASY TT</td>
</tr>
<tr>
<td>Confocal setup with climate chamber, multi-position mode, and water pump</td>
<td>Leica SP8</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>In addition to the abovementioned reagents and materials (<a href="#key-resources-table">key resources table</a>), a BSL-1 cell culture facility with incubator and flow cabinet, and including standard tissue culture disposables, is required. Furthermore, a bacterial lab equipped with shaking incubator, as well as materials and reagents for transformation and plasmid isolation is needed. A cell counter (for instance; CASY TT) is used to assess the numbers of endothelial cells to be incubated with Cytodex beads, while a confocal microscope equipped with a CO<sub>2</sub>-supplemented climate chamber, as well as a time-lapse and multi-position module, is required for live imaging. We use a Leica SP8 confocal set-up, but any other confocal microscope with these specifications will be effective. For the production of lentiviral particles and the lentiviral transduction of endothelial cells with shRNAs, BSL-2 facilities are required. Finally, for the analysis, quantification, and storage of data, FIJI software with the indicated plugins, as well as our newly generated macro “<i>Automated sprout analysis</i>” (<a href="#key-resources-table">key resources table</a>) are required, as well as a powerful computer with a fast processor, plenty of hard-disk space, and sufficient RAM (at least 16 GB).</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Instead of Hoechst, DAPI can be used. The CellTracker dyes used here for live imaging can also be substituted for others. A number of different cell-permeable dyes, coupled to a variety of fluorophores, exist that are retained in the cytosol or label distinct subcellular compartments. While this protocol is optimized for the 96-well ‘half area’ plates, the use of imaging plates of different size and/or well number is also possible. The volumes indicated here should then be adjusted accordingly. Finally, other endothelial cells can be used instead of HUVECs, although not all endothelial cells grow and sprout equally efficiently.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Endothelial cell culture and lentiviral transduction for stable knockdown</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~1 week</div>
<ol>
<li>Thaw 1 vial of HUVECs and seed into a gelatin-coated T75.</li>
<li>After 1 or 2 days, HUVECs can be replated as described above into appropriate number of coated T150s, usually 3–4, depending on the pool of HUVECs (some pools grow better than others). This step is for experiments that require large numbers of cells. Otherwise, skip this step and proceed with 3.</li>
<li>After 1 or 2 days, seed HUVECs for lentiviral transduction into coated 6-well plates at 40%–60% confluency, 1 or 2 wells per condition.</li>
<li>The following day, transduce HUVECs with virus-like particles, obtained as described above.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> HUVECs can be transduced with individual shRNAs or with pools of different shRNAs targeting the same gene. The latter is suitable for initial screening purposes. The efficiency of knockdown varies per shRNA, as do the chances of off-target effects. Furthermore, it is recommended to use 2 or 3 different clones per target gene, to reduce the chances of erroneous interpretation of results due to off-target effects. Finally, a scrambled sequence in the same vector should be used as a control.</div>
<ol start="5">
<li>HUVECs are incubated for 12–16 h with the shRNA lentivirus solution (1 mL/well of a 6-wells plate) in DMEM, supplemented with 0.2–1.0 mL of EGM-2.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Supplement with EGM-2, depending on virus concentration and/or efficiency. It is advised to use as little virus as possible to achieve efficient knockdown of target gene expression. Since the shRNA-coding sequence is randomly integrated into the host genome, increasing copy number increases the chance of destroying critical genes, which will affect the cell in ways other than by knockdown of the target gene. Titration of the virus (with puromycin) is recommended.</div>
<ol start="6">
<li>Replace the medium with EGM-2 containing puromycin (1 μg/mL) to select positive cells (see <a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/994-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Generation of HUVECs with stable suppression of gene expression and assessment of the efficiency of knockdown</div>
<p>(A) Phase/contrast images showing HUVECs before and after lentiviral transduction with pLKO.1 and selection with puromycin. Untransduced controls are all dead after two days of puromycin treatment. Bar, 100 μm.</p>
<p>(B) Determination of knockdown efficiency using flow cytometry. Histograms represent cell-surface levels of β1 integrin in HUVECs expressing scrambled sequences (sh_Ctrl) or shRNAs against integrin β1 (sh_β1). Bar, 20 μm.</p>
<p>(C) Confocal images of HUVECs stained for β1 integrin (<i>yellow</i>), F-actin (<i>red</i>), and VE-cadherin (<i>gray</i>).</p>
<p>(D–G) (D) Composite images showing examples of sprouting efficiency in fibrin gels. Bar, 75 μm. Total network length/bead (E), average sprout length (F), and number of sprouts/bead (G) were determined using the protocol described here from 10 beads out of 2 experiments. Data are means ± SEM, statistically significant differences are denoted by ∗ p &lt; 0.05, (unpaired two-tailed t test).</p>
</figcaption>
</figure>
</li>
<li>After 2 days, the selected cells can be used for attachment to beads (see sprouting assay protocol below).</li>
<li>The efficiency of knockdown should be assessed in parallel by flow cytometry (for surface proteins) or Western blotting (for intracellular proteins), using validated antibodies (see <a href="#fig2">Figure 2</a>). If these are not available, knockdown can be confirmed by Q-PCR.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Selection should be finished 2–3 days after the addition of puromycin. Always take 1 well of non-transduced cells along to confirm puromycin-induced cell death. In case the selection is suboptimal, a kill curve should be performed first to establish the ideal puromycin concentration.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure sufficient cells after obtained after selection (<a href="#sec7.5">troubleshooting 3</a>).</div>
<h3 id="sec3.2">Assay to screen for factors regulating sprouting angiogenesis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>This step describes how to achieve fast and robust HUVEC sprouting in fibrin gels, and to visualize sprouting and tip cell characteristics, such as the formation of filopodia, by confocal microscopy. Work in the flow hood to keep cells and materials sterile.</p>
<ol start="9">
<li>Prepare beads
<ol type="a">
<li>Resuspend beads.</li>
<li>Transfer 250 μL of resuspended beads to a sterile 1.5 mL Eppendorf tube.</li>
<li>Allow beads to settle and remove supernatant.</li>
<li>Add 250 μL of EGM-2.</li>
</ol>
</li>
<li>Prepare cells
<ol type="a">
<li>Harvest HUVECs by trypsinization and resuspend in EGM-2.</li>
<li>Count cells using a cell counter.</li>
<li>Adjust cells to a concentration of 660.000 cells/mL (=1 × 10<sup>6</sup> cells/1.5mL).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This is sufficient to coat approximately 1200 beads. To coat less or more beads, change numbers accordingly.</div>
<ol start="11">
<li>Attach cells to beads
<ol type="a">
<li>Add 1.5 mL of cell suspension to a 50 mL tube (=1 × 10<sup>6</sup> cells in total).</li>
<li>Resuspend beads and add 40 μL to the cell suspension.</li>
<li>Place cell/bead mixture at 37°C and 5% CO<sub>2</sub> for 4 h.</li>
<li>Resuspend every 20 min.</li>
</ol>
</li>
</ol>
<ol start="12">
<li>Incubation of beads with cells
<ol type="a">
<li>Add cell/bead mixture to T25.
<div class="note">
<span class="note-title">Note:</span> Don’t fill up the pipette with the entire 1.5 mL; too many cells/beads will stick to the inside of the pipette.</div>
</li>
<li>Add 3.5 mL of EGM-2 to T25.</li>
<li>Place cell/bead mixture at 37°C and 5% CO<sub>2</sub> for 12–16 h.</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Harvest HUVEC-coated beads
<ol type="a">
<li>Resuspend beads and transfer to a 50 mL tube.</li>
<li>Rinse T25 once with 3 mL of EGM-2 and add to the same tube.</li>
<li>Allow beads to settle.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The knockdown of some target genes may affect HUVEC adhesion to Cytodex beads. Visually inspect whether HUVEC adhesion is efficient.</div>
<ol start="14">
<li>Wash HUVEC-coated beads
<ol type="a">
<li>After beads have settled, remove the supernatant, add 1 mL of EGM-2, resuspend and allow beads to settle. Repeat.</li>
</ol>
</li>
</ol>
<ol start="15">
<li>Prepare fibrin gel
<ol type="a">
<li>Add 0.6 mL of 10 mg/mL fibrinogen to 2.4 mL of PBS, then add 18 μL of 15 U/mL aprotinin.</li>
<li>After removing the supernatant from the beads, add fibrinogen/aprotinin/PBS mixture to the 50 mL tube.</li>
<li>Pipet a drop of 1,875 μL of a 10 U/mL thrombin solution in the middle of a well of a glass-bottom 96 well plate.
<div class="note">
<span class="note-title">Note:</span> The plates mentioned here require small volumes of reagents, ideal for testing multiple conditions simultaneously, and the glass bottom is ideal for confocal imaging. However, plates with different properties (such as different well size, or plastic bottom if phase/contrast imaging is used instead of confocal microscopy) can also be used.</div>
</li>
<li>Add 30 μL of the fibrinogen/aprotinin/PBS mixture containing HUVEC-coated beads and resuspend once, very gently.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Avoid air bubbles. Do not move plate.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure the beads are properly resuspended and distributed. While too many beads per well or beads that are too close to one another could be good to study anastomosis, it hinders a proper discrimination of sprouts belonging to individual beads (see <a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/994-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Common problems associated with the assay</div>
<p>Maximum projection of an image stack (A), basal z-slice only (B), and 3-D projection along the x-axis (C), showing HUVECs spreading on the bottom of the plate (<i>arrow</i>). Example of hypersprouting (D), beads too close to each other (E) and beads too close to the well periphery (F). All stainings show F-actin (<i>magenta</i>) and Hoechst (<i>cyan</i>). Maximum projections are generated from stacks of 199-291 z-slices. Bars, 75 μm.</p>
</figcaption>
</figure>
<ol start="16">
<li>Polymerization
<ol type="a">
<li>Allow the fibrin gel to polymerize for 5 min in the flow hood at 20°C–22°C.</li>
<li>Gently place plate in incubator for 15 min.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Avoid air bubbles. Do not move plate!</div>
<ol start="17">
<li>Sprouting
<ol type="a">
<li>In the flow hood, very gently drip 100 μL of EGM-2 on top of the gel.</li>
<li>Place HUVEC/bead/gel mixture at 37°C and 5% CO<sub>2</sub> for 20–48 h.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The relative quantities of thrombin and fibrinogen are crucial for the polymerization and quality of the gels. While poor gel quality can affect sprouting in various ways, a common problem is that beads sink to bottom and cells start to adhere to the bottom of the well (see <a href="#fig3">Figure 3</a>) (<a href="#sec7.7">troubleshooting 4</a>).</div>
<div class="note">
<span class="note-title">Note:</span> The optimal window required for sprouting should be determined first for appropriate experimental conditions. We occasionally observe excessive sprouting already within 48 h, which complicates the analysis and quantification (see <a href="#fig3">Figure 3</a>). Also, it will be more difficult to identify stimulatory effects on sprouting under these circumstances (<a href="#sec7.9">troubleshooting 5</a>).</div>
<h3 id="sec3.3">Multi-position confocal imaging of fixed material</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8–12 h</div>
<p>After the desired time of sprouting (in glass-bottom ‘half-well’ 96-well plates, efficient sprouting is already visible within 20–48 h after initiation of the assay), cells can be fixed for immunofluorescence analysis. For the visualization of individual sprouts and cells, rather than subcellular structures, we use phalloidin to stain actin filaments, and Hoechst to stain nuclei. Both stains are very bright and because of their small size, their delivery to cells in the fibrin gels is much more efficient than that of antibodies. The time required for imaging of an entire plate will depend on the numbers of beads to be imaged per condition, as well as microscope settings such as number of z-slices, resolution, and scan speed.</p>
<ol start="18">
<li>Staining
<ol type="a">
<li>Aspirate the medium and wash once with PBS containing MgCl2/CaCl2.</li>
<li>Fix with 500 μL of 4% PFA for 15 min at 20°C–22°C.</li>
<li>Wash 4 times with PBS.</li>
<li>Permeabilize with 500 μL of 0.5% Triton X-100 in PBS for 5 min at 4°C (this is sufficient for robust delivery of phalloidin, in contrast to antibodies which would require longer permeabilization).</li>
<li>Block non-specific binding with 5% BSA in PBS containing MgCl2/CaCl2 for 2 h at 20°C–22°C.</li>
<li>Wash the gels 3 times with PBS containing MgCl2/CaCl2.</li>
<li>Incubate with Hoechst and phalloidin for 12–16 h at 4°C.</li>
<li>Wash gels 3 times with PBS containing MgCl2/CaCl2 for 10 min and store at 4°C until imaging. Although it is advised to image the plates shortly after staining, the plates can be stored for a number of weeks.
<div class="note">
<span class="note-title">Note:</span> In case of storage, keeping the phalloidin in until imaging increases the quality of staining.</div>
</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> For the visualization of subcellular structures or specific proteins, stainings with antibodies can be performed. For details on immunostaining in fibrin gels see: (<a href="#bib3">Eglinger et al., 2017</a>; <a href="#bib6">Nakatsu and Hughes, 2008</a>)</div>
<ol start="19">
<li>Multi-position confocal imaging
<ol type="a">
<li>Apply two drops of water to the lens of a 20× long-distance water objective, as well as a drop of water on the bottom of the 96-well plate underneath the wells to be filmed.</li>
<li>Place plate on the stage of confocal microscope.
<div class="note">
<span class="note-title">Note:</span> The use of objectives with long working distance is preferred. Best results are obtained with water objectives, which generate less optical aberrations at long working distance, as compared to oil-immersion objectives. Moreover, oil immersion objectives are less suitable for multi-position imaging, as the oil will be distributed all over the plate, increasing the risk of focus loss. For longer imaging periods with water objectives, the use of a water pump is required to maintain water immersion. Use no higher magnification than 20–25× if entire beads with all pertaining sprouts are to be imaged.</div>
</li>
<li>Select the appropriate laser lines (for instance, excitation at 405 nm for Hoechst, and 561/594 nm for Texas Red-conjugated phalloidin).</li>
<li>Determine laser/gain and other confocal settings, for instance image dimensions 1024 × 1024 pixels, 0.75 μm/pixel, pinhole 1 μm, scan speed 400 Hz, bidirectional scanning, and performing sequential scanning between stacks (see <a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/994-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Confocal imaging and mosaic sprouting assays</div>
<p>(A) The confocal set-up used for assays described in this paper requires a module for multi-position imaging and a 37°C climate chamber equipped with CO<sub>2</sub> and humidifier, as well as a water pump for prolonged water immersion imaging.</p>
<p>(B) Screenshot of confocal settings for imaging of sprouts in multi-position mode.</p>
<p>(C) Example of a mosaic assay for simultaneous assessment of differential sprouting properties between conditions. Two different HUVEC populations are differentially labeled using CellTracker dyes (pseudocolored <i>yellow</i> and <i>magenta</i>), mixed in a 1:1 ratio on beads, and allowed to sprout in fibrin gels, whereafter they are fixed and the nuclei stained (<i>gray</i>). Images are maximum projections of confocal z-stacks comprised of 183 slices. Bar, 75 μm.</p>
</figcaption>
</figure>
</li>
<li>Optimize laser and gain settings.</li>
<li>Set the appropriate number of z-slices. Example: at a 1.5 μm z-slice thickness, a typical bead with sprouts of about 300 nm in height will require around 200 z-slices.</li>
<li>Run a pilot stack to confirm or adjust settings.</li>
<li>Select xyz positions and save to positions list. Attempt to image ~10 beads per condition.
<div class="note">
<span class="note-title">Note:</span> The settings mentioned here will enable fast imaging with high-quality results, but several settings including the number of z-slices should be adjusted according to individual demands on image quality or imaging speed.</div>
</li>
<li>Start the experiment. The positions list can be created at the end of the day to let the confocal run for 12–16 h.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The above settings will generate high-quality images with great resolution, but imaging with these settings will take on average 15 min per bead. For the analysis of large numbers of beads, scanning time can be reduced to about 3 min per bead using resonant scanning. The latter option will result in lower resolution but will still generate sufficient detail for proper visualization and quantification.</div>
<div class="note">
<span class="note-title">Note:</span> Because of bead-to-bead heterogeneity and variability in sprouting, it is advised to image multiple beads (5–10) per condition. Potential bias in selecting the beads for imaging could be reduced by performing the selection in a blind manner. Furthermore, unbiased bead selection may be aided using microscope applications that scan the entire well at low magnification (such as Leica Navigator), thus giving an impression of the general efficiency of sprouting per condition. Finally, phase/contrast images can be acquired of the same plate in parallel to confirm differences in sprouting between conditions.</div>
<h3 id="sec3.4">Cell labeling and mosaic sprouting assays to assess differences in tip cell formation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>While the protocol described above allows for visualization of tip cell morphology in detail (see <a href="#fig1">Figure 1</a>), differences in tip cell behavior or competitive advantages in tip cell formation can be assessed more directly in mosaic assays with differentially labeled cell populations, e.g., by using CellTracker as described below.</p>
<ol start="20">
<li>Labeling of HUVECs with cell-permeable CellTracker dyes
<ol type="a">
<li>Trypsinize HUVECs of experimental condition and control.</li>
<li>Count HUVECs and resuspend equal numbers in EGM-2.</li>
<li>Add a different CellTracker dye to each population, for instance CellTracker Green for the experimental condition, and CellTracker Red for the control (both at suggested concentrations of 0.1 μM).</li>
<li>Wash the cells twice with large volumes of PBS.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Perform these steps in the flow hood to keep cells and dyes sterile. CellTracker dyes are dissolved in DMSO as a 1000× solution, according to the manufacturer’s instructions.</div>
<div class="note">
<span class="note-title">Note:</span> CellTracker dyes are cell-permeable but once in the cell, they are converted into cell-impermeable reaction products. They are transferred to daughter cells after division. CellTracker Green produces uniform cell labeling, while CellTracker Red prominently labels vesicular compartments (see <a href="#fig4">Figures 4</a> and <a href="#fig5">5</a>).</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/994-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Live imaging approaches for visualization of tip cell behavior during sprouting</div>
<p>(A) Stills from a single-channel time-lapse movie showing filopodial dynamics in sprouting HUVECs labeled with CellTracker. Images were acquired every 10 min with 61 z-slices per time-point. Bar, 20 μm.</p>
<p>(B) Schematic representation of the construct used for live imaging (<i>top</i>). Stills from a pseudocolored dual-channel time-lapse movie, showing a tip cell expressing H2B-mNeon to visualize the nucleus, and TagRFP-CAAX to track the plasma membrane and endomembranes (<i>bottom</i>). Images were acquired every 15 min with 66 z-slices per time-point. Bar, 10 μm.</p>
<p>(C) Stills from a pseudocolored dual-channel time-lapse sequence showing tip cell dynamics in a mosaic sprouting assay with a 1:1 mixed population of HUVECs differentially stained with distinct CellTracker dyes. Imaging started 20 h after sprouting, images were acquired from 99 z-slices with a 20 min interval. Bar, 75 μm.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Wear protective clothing, gloves, and eye/face protection when handling DMSO or the DMSO dye solution. DMSO easily penetrates the skin and facilitates the absorption of organic molecules into tissues. Dispose of waste and materials in compliance with the appropriate local regulations.</div>
<ol start="21">
<li>Attaching labeled HUVECs to beads
<ol type="a">
<li>Pre-incubate for 30 min in the dark at 37°C.</li>
<li>Mix the two populations in a 1:1 ratio.</li>
<li>Incubate labeled HUVEC mixture with the beads for 12–16 h at 37°C as described in steps 11 and 12.</li>
<li>Harvest beads with labeled HUVECs as described in steps 13 and 14.</li>
<li>Add beads with labeled HUVECs to fibrin gels and start sprouting assay as described in steps 15–17.</li>
</ol>
</li>
</ol>
<ol start="22">
<li>Fix cells at appropriate time-point, e.g., at 20 h after the induction of sprouting.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> CellTracker dyes can induce toxic effects in the long run. Determine cellular health by visual inspection and decide on the optimal time-point to fix the cells. For tip cell competition assays, formation of large sprouts is not necessary and the experiment can be fixed at an early time-point.</div>
<ol start="23">
<li>Label cells after fixation with Hoechst as described above in step 18. Keep plates in the dark to avoid loss of CellTracker signal.</li>
<li>Select the appropriate laser lines on a confocal set-up (for instance; 405 nm for Hoechst, 488 nm for CellTracker Green, and 591 nm for CellTracker Red).</li>
<li>Image the cells as described above (step 19).</li>
<li>Create maximum projections of the z-stacks as described in the <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a> section below. The separate channels will reveal cells of each population, as well as the nuclei of all cells (see <a href="#fig4">Figure 4</a>C).</li>
<li>Quantification of tip cells is performed by counting all tip cells for each color, and dividing this number by the total number of cells in that color. This will correct for potential differences in total cell numbers in each condition.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> First determine the best conditions for labeling (ie concentration, incubation times) with CellTracker dyes (<a href="#sec7.11">troubleshooting 6</a>). Note that prolonged incubation increases dye dilution due to cell division, as well as transfer of dyes to adjacent cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To rule out (indirect) effects of the dye on tip cell formation it is recommended to include a condition where the different CellTracker dyes are swapped between cell populations. Furthermore, it is advised to include a condition in which control cells, labeled with either dye, are mixed in a 1:1 ratio. The number of tip cells in each color should be around 50% in this condition.</div>
<h3 id="sec3.5">Live imaging by time-lapse confocal microscopy</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>In addition to imaging fixed materials, this protocol describes how to generate time-lapse movies from sprouting cells in fibrin gels, which will reveal the dynamic behavior of tip cell filopodia and of individual cells within sprouts. To visualize endothelial cells, they can be labeled using CellTracker dyes, or engineered to express a fluorescent protein. As an example of the latter, we here use a bicistronic construct in a lentiviral vector, encoding mNeonGreen labeled histone 2B (H2B) as well as a CAAX motif tethered to tagRFP (<a href="#bib2">Bolhaqueiro et al., 2018</a>). The C-terminal CAAX motif used here is from K-Ras and integrates specifically in the plasma membrane, thus marking the cell membrane and endocytic/pinocytic compartments derived thereof, while H2B marks the chromatin. Therefore, this approach is useful for the simultaneous imaging of membranes and nuclei in living cells (see <a href="#fig5">Figure 5</a>).</p>
<ol start="28">
<li>Label cells with CellTracker dyes as described above in step 20.</li>
</ol>
<p>Or:</p>
<ol start="29">
<li>Follow steps 22–32 of the section “<a href="#sec1.5">Production of lentivirus-like particles</a>”, but using the lentiviral expression vector of interest, instead of shRNA-encoding <i>pLKO.1-puro.</i>
</li>
<li>Transduce HUVECs as in steps 1–5 of section “<a href="#sec3.1">Endothelial cell culture and lentiviral transduction</a>“.</li>
<li>If desired, select positive cells with antibiotics as described in step 6 of section “<a href="#sec3.1">Endothelial cell culture and lentiviral transduction</a>“.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use a lentiviral vector here for the delivery of our fluorescent protein of interest. Any marker protein can be used in this manner, within the size limit of a lentiviral vector (~10 kilobases in total; virus production is less efficient for inserts &gt; 2kb).</div>
<div class="note">
<span class="note-title">Note:</span> If imaging with lentivirally delivered proteins is initiated shortly after transduction, a BSL-2 imaging facility is required.</div>
<div class="note">
<span class="note-title">Note:</span> CellTracker is easily combined with stable suppression of gene expression using shRNAs. To achieve robust expression of a fluorescent marker protein in stable knockdown cells, we recommend transducing and selecting shRNA-expressing cells first, followed by transduction to express the marker protein. A different antibiotic (for example blasticidin) could be used to select marker protein-expressing cells, but this will often not be necessary given the high degree of transduction efficiency using lentiviral systems, and the fact that positive cells can easily be selected by visual inspection.</div>
<ol start="32">
<li>Attach the fluorescent HUVECs to beads as described in step 21.</li>
<li>Determine the best time-point to start imaging. Generally, sprouting does not occur earlier than 16–18 h after placing the beads in the gels.</li>
<li>Replace the medium approximately 1 h before starting to set up the microscope. The microscope should be prewarmed and atmospherically stabilized at 37°C and 5% CO2.</li>
<li>Select the appropriate laser lines and settings. Appropriate time intervals will depend on the hypothesis, e.g., imaging filopodia dynamics will require shorter intervals than investigating tip cell sprouting or sprout elongation.</li>
<li>Start the experiment as described above in step 19.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Because CellTracker can fade, impose toxicity, or transfer between cells during prolonged incubation, it is advisable to not image CellTracker-loaded cells too long.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to first determine the optimal confocal settings (number of z-slices per position, intervals between images), as well as potential phototoxicity and bleaching effects (<a href="#sec7.13">troubleshooting 7</a>).</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The protocol described here will achieve robust sprouting of endothelial cells in a short time-frame, followed by detailed visualization and quantitative assessment of sprouting characteristics such as length and numbers of sprouts, numbers of individual cells, etc. The ‘half-area’ 96-well plates require limited amounts of reagents and are suitable for medium- to high-throughput screening. Dynamic behavior of individual cells within sprouts can be monitored using fluorescent probes by time-lapse microscopy, while the mosaic assays will assess relative sprouting efficiencies and competitive advantages between control and experimental conditions in the same well.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min–5 h</div>
<p>The protocol below makes use of our newly generated macro “<i>Automated sprout analysis</i>” that reiterates sprouting analysis using the FIJI/ImageJ plugin <i>Sprout morphology</i>, based on a previously described method (<a href="#bib3">Eglinger et al., 2017</a>). Upon importing raw confocal data, the macro will create a maximum projection, despeckle the image, and enhance the contrast by 0.3%. Subsequently the macro will detect beads, sprouts and nuclei according to adjustable settings and store the results in a separate folder. The required time will depend on the number of beads analyzed, binning options, and the computer used.</p>
<ol>
<li>Install ImageJ plugin <i>Sprout morphology.</i>
<ol type="a">
<li>Menu bar → Help → Update…→ Manage update sites → Check Angiogenesis.</li>
<li>Click close and apply changes.</li>
<li>Restart ImageJ/FIJI.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Run macro.
<ol type="a">
<li>Import “<i>Automated sprout analysis</i>” macro in ImageJ</li>
<li>Click Run &gt; the macro will open the window “<i>macro to reiterate sprouting analysis</i>” &gt; click OK</li>
<li>The macro will open a window to select the appropriate file &gt; click OK</li>
<li>The macro will open the “<i>Bio-Formats options</i>” window &gt; select the appropriate settings (<i>Virtual Stack</i> recommended) &gt; click OK</li>
<li>Macro will then open the window “<i>Bio-Formats series options</i>” &gt; select a single position (the sole purpose here is readout of metadata; user can later define positions to be analyzed) &gt; click OK
<div class="note">
<span class="note-title">Note:</span> Bio-Formats will allow direct import of raw data without the need to export them first from the microscopy software. Bio-Formats will retrieve the appropriate/relevant dimensions directly from the metadata.</div>
</li>
<li>Macro will open a Dialog Window to define all desired settings (see <a href="#fig6">Figure 6</a>). Pre-defined settings are shown (and can be retrieved later).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/994-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Screenshot of the settings window from our “<i>Automated sprout analysis</i>” macro</div>
<p>Appropriate settings for the opening, processing, analysis and output of confocal z-stacks of sprouting can be defined here, as described in the main text.</p>
</figcaption>
</figure>
</li>
<li>Name the experiment and select the positions to be analyzed (<i>1</i> in <a href="#fig6">Figure 6</a>)</li>
<li>Define staining and colors of separate channels (<i>2</i>)</li>
<li>If necessary, select options to reduce RAM size, e.g., cropping around the entire sprout outline and/or binning (<i>3</i>).
<div class="note">
<span class="note-title">Note:</span> Binning will speed up analysis and use less memory, but be aware that reduced resolution may impact the results</div>
</li>
<li>Define which channels to use for detection of nuclei, beads and sprouts (<i>4</i>)</li>
<li>Select thresholding settings and options for optimal detection of individual nuclei (segmentation) (<i>5</i>). Standard settings used in this paper are shown.
<div class="note">
<span class="note-title">Note:</span> Appropriate settings for sprout and bead thresholding should be determined and optimized first for individual experiments.</div>
</li>
<li>Select the minimal bead radius (MBR), ranging from 60 to 100 μm (<i>6</i>).
<div class="note">
<span class="note-title">Note:</span> Varying the MBR will affect the detection of the bead, and thereby also the numbers of nuclei in sprouts and sprout length. It is advised to first try all MBRs to determine the optimal value in order to get the best size and most circular shape, which will be clear from the created output file (<a href="#fig7">Figures 7</a>F–7K) (<a href="#sec7.15">troubleshooting 8</a>).</div>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/994-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Example of an output file</div>
<p>The macro described here will generate for each individual bead (position) a file containing the analysis settings and main results (A), the separate channels and merge of the maximum projection (B–D), a depth-coded image with and without the outlined bead and spouts (E and F), the bead and sprout masks (G–I), skeletonized sprouts (J), and the segmented nuclei (H). The image stack used here contained 362 z-slices, separate channels were F-actin (<i>magenta</i>) and nuclei (<i>cyan</i>), and depth coding was performed with the LUT <i>Spectrum</i>.</p>
</figcaption>
</figure>
</li>
<li>Select settings for the output file, such as colors of the bead and sprout outlines (<i>7</i>). The “<i>save combine output</i>” option prompts the generation of a tiff combining all analyses from the current experiment in a multidimensional format, designed to facilitate evaluation of the results (see 3a below). To reduce output file size, use the “<i>bin it</i>” option (this option leaves data analysis itself unaffected).</li>
<li>Finally, the option to return to the pre-defined settings can also be selected (<i>8</i>)
<div class="note">
<span class="note-title">Note:</span> Output data are saved to a folder structure, organized by Experiment ID. On first use, the user will be prompted to select the destination for the output folder, which will from then automatically be used in subsequent analyses.</div>
<div class="note">
<span class="note-title">Note:</span> The time necessary to complete analysis depends on the computer used. For the analysis of large data sets, a fast computer with sufficient RAM (at least 16 GB) is crucial (<a href="#sec7.15">troubleshooting 8</a>).</div>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Analyze data.
<ol type="a">
<li>The macro will store the generated results on the selected drive, in a folder called “<i>Results Sprouting Analysis</i>”. Data from each experiment will be stored in a separate folder, and will contain (among others) a tiff file with two sliders. The bottom slider will enable toggling between all individual positions (beads of the experiment, while the top slider will enable scrolling through all analysis results (see <a href="#fig7">Figure 7</a>), including the name of experiment/condition as well as used settings and obtained results (A), the separate and merged channels of the max projection (B-D), depth coding with and without outlined bead and sprouts (E,F), the bead and sprout masks (G-I), skeletonized sprouts (J), and the segmented nuclei (H).</li>
<li>The macro will also save a basic Excel (xls) file, containing the used settings as well as the results from all positions (# of beads, sprouts, and nuclei, total network length, average sprout length and sprout width). For opening in Excel, choose semicolon as delimiter and use ‘.’ as Decimal separator and ‘’(empty) as Thousands separator.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Graphs can be created directly in Excel or using an alternative program such as GraphPad Prism. The latter also allows direct statistical analysis. Start from data table or graph → <i>Analyze</i> → choose the appropriate method of statistical analysis, depending on experimental set-up and number of conditions.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Technical problems may arise at several steps of the protocol described here, associated for instance with cell culture, sprouting efficiency, staining and multi-position imaging of fixed cells, live-cell microscopy, or the analysis of image stacks. Potential solutions to these issues are detailed in the <a href="#troubleshooting">troubleshooting</a> section. It is important to realize that endothelial cells can differ from one individual to another, and that not all HUVECs proliferate and sprout to the same extent, which is a source of variability between different labs and/or different sets of experiments in the same lab. Moreover, this assay relies on generating confocal z-stacks from multiple different conditions in multi-position imaging mode, which requires an optimal balance between the number of z-slices per position, the number of positions to be imaged, and the resolution (and therefore speed) of imaging. Reductions in z-step size will lead to an increase in the number of z-slices and thus provide more detail, but will also require more imaging time. Also, higher resolution will decrease the speed of imaging and vice versa. Access to confocal microscopes and available imaging time may be limited, although for this type of experiments a positions list can be created at the end of the day to run the experiment for 12–16 h. During live imaging, limitations will include detrimental effects resulting from the use of fluorescent dyes and/or phototoxicity, as well as bleaching of the fluorescent signal over time. Furthermore, there are limitations to the number of z-slices and positions, as well as the time interval in between two consecutive positions, that can be imaged in a multi-position live experiment. Finally, the analysis of image stacks using ImageJ requires a fast device with sufficient memory, and can be painfully slow if a ‘regular’ computer is used.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>HUVECs grow slower than expected (steps 20 and 21 of <a href="#before-you-begin">before you begin</a>).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Cell adhesion and proliferation can be enhanced by coating the flasks with gelatin, fibronectin, or serum. Furthermore, proliferation is optimal when cells are seeded at appropriate densities, i.e., not too sparse and not too dense.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Viral titer is low (steps 32 and 33 of <a href="#before-you-begin">before you begin</a>).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>To increase viral titer, try to enhance the efficiency of transfection. HEK293T cell adhesion and proliferation can be improved by coating the tissue culture dishes. HEK393T cells are best transfected at a density of 50%–80%. Low-passage HEK293T cells produce a higher titer than high-passage cells. Finally, the viral supernatant can be concentrated using PEG 6000 (<a href="#bib5">Kutner et al., 2009</a>). While viral supernatant can be stored at −80°C for a number of weeks to even months, best results are achieved using freshly isolated supernatant.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Few cells are left after selection (step 7).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>The viral titer may be too low, which can be solved by optimizing the transfection procedure in HEK293T cells, or by concentrating the virus as described above. Alternatively, some shRNAs can cause cell death due to off-target effects. Comparison of different shRNAs per target gene is advised. Finally, if the gene of interest is an essential gene, its suppression may impair cell viability and/or proliferation. In this case it is likely that selection occurs for cells that have low levels of knockdown, and thus express residual levels of the protein that rescue compromised proliferation or cell viability.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Gels do not polymerize well and/or are of poor quality (steps 15 and 16).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>The quality and relative amounts of the used fibrinogen and thrombin are essential. Thrombin can be aliquoted and maintained at −80°C, but its activity may decrease over time during prolonged storage. Furthermore, it is recommended to use high-purity fibrinogen preparations. Suboptimal quantities of active thrombin and/or fibrinogen will cause the gel either to polymerize too fast and/or become too rigid, which will impair sprouting, or not polymerize fast/good enough, in which case the beads may sink to the bottom of the well and the cells will start adhering to the tissue culture glass plate (see <a href="#fig3">Figure 3</a>).</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Excessive sprouting is observed after 48 h (hypersprouting), which can complicate analysis (see <a href="#fig3">Figure 3</a>) (step 17).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Determine the optimal conditions in a pilot experiment. In case of excessive sprouting, reduce the time of the assay (fix the cells at an earlier time-point).</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Labeling with CellTracker induces toxicity, or dye transfer between adjacent cells is observed (steps 21 and 22).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Optimal conditions for labeling and use of CellTracker dyes should be assessed first. Too high concentrations can reduce cell viability and performance, while too low concentrations, especially combined with bleaching effects during imaging, will compromise visibility. Moreover, transfer of dyes between adjacent cells can occur. Several dyes are available, fused to a variety of fluorophores and labeling distinct subcellular compartments, and comparison of these will identify the optimal dye for specific experimental conditions.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>The signal is bleaching, cells look unhealthy, or cells do not behave as expected in live imaging assays (step 35).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>The dyes may impair cell performance directly or as a result of phototoxicity during live imaging. First assess the direct effects of the dyes on cell function, by incubation during the same time-frame but without exposure to light. Then aim for the minimal concentration of dye that will still generate sufficient visibility even after long-term imaging. If impaired cell function results from phototoxicity, a solution may be to increase the interval between consecutive images (thus reducing the total number of images), as well as reduce the number of z-slices at each position. On a confocal microscope, resonant scanning may reduce bleaching effects and phototoxicity. Alternatively, other systems like light sheet microscopes could resolve these problems.</p>
<h3 id="sec7.15">Problem 8</h3>
<p>The macro does not generate results or aborts analysis (steps 1 and 2 of <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</p>
<h3 id="sec7.16">Potential solution</h3>
<p>Allocating more RAM to ImageJ is recommended (<i>Edit</i> &gt; <i>Options</i> &gt; <i>Memory and threads</i>). If insufficient RAM is available, the macro will abort analysis. The user can resume analysis by selecting the appropriate position in the Settings Dialog. The macro also indicates when analysis for a given position failed. This is usually because the bead was not properly detected, and can be solved by selecting a different MBR.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">van der Bijl, I., Nawaz, K., Kazlauskaite, U., Stalborch, A.-M. van, Tol, S., Orgaz, A.J., Bout, I. van den, Reinhard, N.R., Sonnenberg, A., and Margadant, C. (2020). Reciprocal integrin/integrin antagonism through kindlin-2 and Rho GTPases regulates cell cohesion and collective migration. Matrix Biol. <i>93</i>, 60-78. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00397-X/sref1">View at publisher</a></p>
<p id="bib2">Bolhaqueiro, A.C.F., Jaarsveld, R.H. van, Ponsioen, B., Overmeer, R.M., Snippert, H.J., and Kops, G.J.P.L. (2018). Live imaging of cell division in 3D stem-cell organoid cultures. Methods Cell Biol. <i>145</i>, 91-106. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00397-X/sref2">View at publisher</a></p>
<p id="bib3">Eglinger, J., Karsjens, H., and Lammert, E. (2017). Quantitative assessment of angiogenesis and pericyte coverage in human cell-derived vascular sprouts. Inflamm. Regen. <i>37</i>, 1-9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00397-X/sref3">View at publisher</a></p>
<p id="bib4">Kempers, L., Wakayama, Y., van der Bijl, I., De Cuyper, I.M., Jongejan, A., Kat, M., van Stalborch, A.D., van Boxtel, A.L., Hubert, M., van Buul, J.D., et al. (2021). The endosomal RIN2/Rab5C machinery prevents VEGFR2 degradation to control gene expression and tip cell identity during angiogenesis. Angiogenesis. <a class="external-link" href="https://doi.org/10.1007/s10456-021-09788-4">https://doi.org/10.1007/s10456-021-09788-4</a></p>
<p id="bib5">Kutner, R.H., Zhang, X.-Y., and Reiser, J. (2009). Production, concentration and titration of pseudotyped HIV-1-based lentiviral vectors. Nat. Protoc. <i>4</i>, 495-505. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00397-X/sref5">View at publisher</a></p>
<p id="bib6">Nakatsu, M.N. and Hughes, C.C. (2008). An optimised three-dimensional in vitro model for the analysis of angiogenesis. Methods Enzym. <i>443</i>, 65-82. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00397-X/sref6">View at publisher</a></p>
<p id="bib9">Newman, A.C., Nakatsu, M.N., Chou, W., Gershon, P.D., and Hughes, C.C. (2011). The requirement for fibroblasts in angiogenesis: fibroblast-derived matrix proteins are essential for endothelial cell lumen formation. Mol Biol Cell. <i>22</i> (<i>20</i>), 3791-3800. <a class="external-link" href="https://doi.org/10.1091/mbc.E11-05-0393">https://doi.org/10.1091/mbc.E11-05-0393</a></p>
<p id="bib7">Nowak-Sliwinska, P., Alitalo, K., Allen, E., Anisimov, A., Aplin, A.C., Auerbach, R., Augustin, H.G., Bates, D.O., Beijnum, J.R. van, Bender, R.H.F., et al. (2018). Consensus guidelines for the use and interpretation of angiogenesis assays. Angiogenesis <i>21</i>, 425-532. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00397-X/sref7">View at publisher</a></p>
<p id="bib8">Schindelin, J., Arganda-Carrera, I., Frise, E., Verena, K., Mark, L., Tobias, P., Stephan, P., Curtis, R., Stephan, S., Benjamin, S., et al. (2009). Fiji - an Open platform for biological image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00397-X/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Coert Margadant (<a href="mailto:c.margadant@amsterdamumc.nl">c.margadant@amsterdamumc.nl</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The ImageJ macro “<i>Automated sprout analysis</i>” is available from GitHub (<a href="https://github.com/Sprouting-angiogenesis/Automated-sprout-analysis">https://github.com/Sprouting-angiogenesis/Automated-sprout-analysis</a>).</p>
<h3>Acknowledgments</h3>
<p>Work in C.M.’s laboratory is/has been supported by research grants from the <a href="https://doi.org/10.13039/501100003246">Netherlands Organisation for Scientific Research</a> (ZonMW Veni 016.146.160) and the Dutch Thrombosis Foundation (2017-01), Sanquin PPOD 17-29, and Landsteiner Foundation for Blood Transfusion Research (<a href="https://doi.org/10.13039/100009425">LSBR)</a> 2019. We further acknowledge support from the Dutch Society for Cell Biology. We thank Ben Morris and Roderick Beijersbergen (Robotics and Screening Center, Netherlands Cancer Institute, Amsterdam, The Netherlands) for their kind gifts of materials, as well as Dirk Geerts (Amsterdam University Medical Center, Amsterdam, The Netherlands) for help with the sequencing protocol for shRNAs. We gratefully acknowledge technical support from Simon Tol, Mark Hoogenboezen, and Erik Mul (Central Facility, Sanquin Research, Amsterdam, The Netherlands), as well as Lenny Brocks and Marjolijn Mertz (Digital Microscopy Facility, Netherlands Cancer Institute, Amsterdam, The Netherlands).</p>
<h3>Author contributions</h3>
<p>L.K., I.v.d.B., and A.-M.D.v.S. conducted experiments and edited the manuscript, L.K. developed the pipeline for image acquisition and processing and performed data analysis, and B.P. wrote the macro for automated analysis. C.M. conceived and supervised the project and wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>