<article>
<section>
<h1 id="header">Generation of TCRα-transduced T cells for adoptive transfer therapy of salmonellosis in mice</h1>
<p><time datetime="2021-03-12">Published: March 12, 2021</time></p>
<p>Anastasiia Kalinina,<sup><a href="#aff1">1</a>,<a href="#cor1">*</a></sup> Alexandra Bruter,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Ludmila Nesterenko,<sup><a href="#aff3">3</a></sup> Ludmila Khromykh,<sup><a href="#aff1">1</a></sup> and Dmitry Kazansky<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Federal State Budgetary Institution "N. N. Blokhin National Medical Research Center of Oncology," the Ministry of Health of the Russian Federation, Moscow 115478, Russia</p>
<p id="aff2"><sup>2</sup>Core Facility Centre, Institute of Gene Biology, Russian Academy of Sciences, Moscow 119334, Russia</p>
<p id="aff3"><sup>3</sup>N. F. Gamaleya National Research Center of Epidemiology and Microbiology, the Ministry of Health of the Russian Federation, Moscow 123098, Russia</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:aakalinina89@gmail.com">aakalinina89@gmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:kazansky1@yandex.ru">kazansky1@yandex.ru</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100368">10.1016/j.xpro.2021.100368</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Adoptive transfer therapy has great potential to treat diseases such as cancer as well as autoimmune and infectious diseases. Identification of chain-centric T cell receptors (TCRs) with the dominant-active antigen-specific α-chains (TCRα) can significantly improve the efficacy of adoptive cell therapy while reducing time, labor, and costs of generation of TCR-modified antigen-specific T cells. This protocol describes how to generate salmonella-specific TCRα-modified mouse T cells by retroviral transduction and evaluate their functional activity <i>in vivo</i> in the mouse model of salmonellosis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib4">Kalinina et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Adoptive T cell therapy has great potential for treatment of infections</li>
<li>Protocol describes generating T cells transduced with dominant pathogen-specific TCRα</li>
<li>Protocol describes usage of specific TCRα-transduced T cells in mice salmonellosis</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/511-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol is the first, describing the therapeutic application of adoptive transfer of T cells modified with the single dominant α-chain of the antigen-specific chain-centric TCRs for the treatment of bacterial infection. Although this protocol was developed and tested in the experimental infectious model of salmonellosis in mice, it represents a versatile technique that could be applied in the treatment of varied pathologies. Our protocol implies universal strategies for identification and cloning of therapeutic dominant-active antigen-specific variants of TCRα, generation of TCRα-transduced T cells, and adoptive transfer methods to test the protective efficiency of pathogen-specific TCRα-modified T cells using <i>in vivo</i> mouse models.</p>
<p>This section describes the specific steps for cloning TCRα-SM1. We used the same protocol for cloning all TCRα-SM genes. Here, we provide primers and PCR conditions used for cloning TCRα-SM1 (<a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a>).</p>
<table id="tbl1">
<caption>Table 1. Primers for cloning</caption>
<thead>
<tr>
<th>Primer</th>
<th>Sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>AgeI_forward</td>
<td>ATTAACCGGT ATGGACAAGATCCTGACAGC</td>
</tr>
<tr>
<td>Unique_forward</td>
<td>CAAGGGTAGCGGGATACAAC</td>
</tr>
<tr>
<td>Unique_reverse</td>
<td>GTATCCCGCTACCCTTGCT</td>
</tr>
<tr>
<td>C-fragment_forward</td>
<td>ATCCTCGGTCTCAGGACAGC</td>
</tr>
<tr>
<td>C-fragment_ reverse</td>
<td>GCTGTCCTGAGACCGAGGAT</td>
</tr>
<tr>
<td>C-fragment_ reverse_SalI</td>
<td>TACTGTCGACTCAACTGGACCACAGCCTC</td>
</tr>
</tbody>
</table>
<table id="tbl2">
<caption>Table 2. PCR conditions</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="4">PCR cycling conditions for V-segment</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>95°С</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°С</td>
<td>40 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>66°С</td>
<td>40 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°С</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°С</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°С</td>
<td colspan="2">forever</td>
</tr>
<tr>
<td colspan="4">PCR cycling Conditions for J-segment</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>95°С</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°С</td>
<td>40 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>68°С</td>
<td>40 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°С</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°С</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°С</td>
<td colspan="2">forever</td>
</tr>
<tr>
<td colspan="4">PCR cycling Conditions for C-segment</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>95°С</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°С</td>
<td>40 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>60°С</td>
<td>40 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°С</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°С</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°С</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> To identify dominant-active salmonella-specific TCRα (TCRα-SM), we generated cDNA libraries of α-chains TCR of salmonella-specific memory T cells, re-stimulated <i>in vitro</i> by the bacterial antigens. cDNA libraries were prepared and raw data was processed as described previously (<a href="#bib2">Egorov et al., 2015</a>; <a href="#bib1">Bolotin et al., 2015</a>; <a href="#bib5">Shugay et al., 2014</a>). For detailed description of cDNA libraries analysis and selection of dominant-active TCRα-SM refer to (<a href="#bib4">Kalinina et al., 2020</a>).</div>
<h3 id="sec1.1">Cloning TCRa-SM1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 17–20 days (4 days for cloning TCRa-SM into the pTZ vector; 7–10 days for sequencing and analyses, depending on a facility; 4 days for cloning TCRa-SM into the MigR1 -PGK vector)</div>
<div class="note">
<span class="note-title">Note:</span> As the template for amplification of V-, J-, and C-segment of TCRa, use cDNA (20 ng) of salmonella-specific memory cells of a B10.D2(R101) mouse that were enriched <i>in vitro</i> by re-stimulation with salmonella. For detailed methods of <i>in vivo</i> generation and <i>in vitro</i> enrichment of salmonella-specific memory T cells refer to (<a href="#bib4">Kalinina et al., 2020</a>).</div>
<ol>
<li>Using primers AgeI_forward and Unique_reverse (10 μM each), amplify V-segment (Fragment 1).</li>
<li>Using primers Unique_forward and C-fragment_ reverse (10 μM each), amplify J-segment and 3`-region of C-segment (Fragment 2).</li>
<li>Using primers C-fragment_forward and C-fragment_reverse_SalI (10 μM each), amplify C-segment (Fragment 3).</li>
<li>Transfer PCR products (Fragment 1, Fragment 2, and Fragment 3) on individual 1.5% agarose gels, run the gels at 6 V/cm.</li>
<li>Cut fragments with the target size from the gel and extract DNA using the QIAquick Gel Extraction Kit (<a href="https://www.qiagen.com/us/spotlight-pages/ias/automated-qpcr-workflow/assay-setup/qiaquick-gel-extraction-kit/">https://www.qiagen.com/us/spotlight-pages/ias/automated-qpcr-workflow/assay-setup/qiaquick-gel-extraction-kit/</a>) or any other kits available. Keep exposure to minimum.</li>
<li>Mix Fragment 1, Fragment 2, and Fragment 3 (20 ng each) at 1:1:1 and use this mix as the template to amplify the full TCRa sequence, using primers AgeI_forward and C-fragment_reverse_SalI (10 μM each).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the amplified Fragment 1, Fragment 2, and Fragment 3 have a pair wise overlap, and this PCR reaction will result in the full length TCRα product.</div>
<div class="note">
<span class="note-title">Note:</span> For possible issues concerning amplification of Fragment 1 and Fragment 2 refer to the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.1">problem 1</a>).</div>
<ol start="7">
<li>Transfer the PCR product on 1.5% agarose gel, run the gel at 6 V/cm, and extract DNA (see step 5).</li>
<li>Mix 20 ng of extracted DNA, 2.0 μL 10× Taq buffer, 2.0 μL dATP (2 mM), and 0.5 μL Taq polymerase to the final volume 20.0 μL, mix by vortexing and incubate 30 min at 72°С.</li>
<li>Mix 10.0 μL of the resulted DNA fragment, containing poly-A, 1.0 μL pTZ vector, 1.0 μL ligation buffer, and 0.5 μL T4 ligase, mix by vortexing and incubate 2 h at 20–22°С.</li>
<li>Transform the competent <i>E. coli</i> with the ligase mix:
<ol type="a">
<li>Thaw one vial of cryopreserved competent <i>E. coli</i> cells on ice.</li>
<li>Transfer 10.0 μL of the ligase mix to <i>E. coli</i> cells, mix by pipetting (2–3 times).</li>
<li>Incubate on ice 20 min.</li>
<li>Incubate 30 s at 42°С, then 5 min on ice.</li>
<li>Add 500 μL of sterile LB broth, mix by pipetting (2–3 times).</li>
<li>Incubate 30 min at 37°С.</li>
<li>Seed <i>E. coli</i> cells on a Petri dish with LB agar, containing ampicillin to the final concentration 1 μg/mL.</li>
<li>Place the Petri dish in incubator at 37°С for 12–16 h.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Transformation can be performed with other commercial kits available.</div>
<ol start="11">
<li>Screen bacterial colonies for positive transformed colonies by PCR, using primers AgeI_forward and C-fragment_ reverse_SalI.</li>
<li>Transfer positive transformed colonies into individual 15 mL centrifuge tubes, containing 5 mL LB and 5 μL ampicillin (1 mg/mL).</li>
<li>Incubate at 37°С for 12–16 h at 180 rpm.</li>
<li>Centrifuge at 10 000 × <i>g</i> for 5 min, remove the supernatant.</li>
<li>Extract DNA using the Wizard Plus SV Minipreps DNA Purification System (Promega) (<a href="https://worldwide.promega.com/-/media/files/resources/protocols/technical-bulletins/0/wizard-plus-sv-minipreps-dna-purification-system-protocol.pdf?la=en">https://worldwide.promega.com/-/media/files/resources/protocols/technical-bulletins/0/wizard-plus-sv-minipreps-dna-purification-system-protocol.pdf?la=en</a>) or any other kits available to you.</li>
<li>Sequence DNA to ensure the full TCRa sequence contains no errors and completely matches the predicted sequence.</li>
<li>Clone the full TCRa sequence into the MigR1-PGK vector:
<ol type="a">
<li>Mix 100 μg DNA, containing the full TCRa sequence (see step 15), 4.0 μL restriction buffer, 2.0 μL SalI restriction enzyme, 2.0 μL AgeI restriction enzyme, and ddH<sub>2</sub>O to the final volume 40.0 μL.</li>
<li>Similarly, mix 1.2 μg MigR1-PGK with the same restriction enzymes in the final volume 40.0 μL.</li>
<li>Incubate the restriction mixes at 37°С for 3 h.</li>
<li>Transfer these restriction mixes on individual 1.5% agarose gels, run the gels at 6 V/cm.</li>
<li>Cut fragments with the target size from the gel and extract DNA (see step 5).</li>
<li>Ligate the full TCRa and linearized MigR1-PGK (see step 9).</li>
<li>Transform the competent <i>E. coli</i> (repeat steps 10 and 11).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the MigR1-PGK vector was generated from the commercial MigR1 plasmid (#27490, Addgene) by replacing IRES to the PGK promoter.</div>
<h3 id="sec1.2">Preparation of plasmid DNA for transfection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="18">
<li>Transfer a positive transformed <i>E. coli</i> colony (see step 17g) into a 50 mL centrifuge tube with 20 mL LB broth, containing 20 μL ampicillin (1 mg/mL).</li>
<li>Incubate at 37°С for 12–16 h at 180 rpm.</li>
<li>Centrifuge tube at 6,000 × <i>g</i> for 15 min at 4°C.</li>
<li>Remove the supernatant and extract DNA using the QIAGEN Plasmid Midi Kit (QIAGEN) (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-plasmid-kits">https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-plasmid-kits</a>) or any other similar kit.</li>
<li>Dissolve DNA in 1 mL ddH<sub>2</sub>O, measure DNA concentration on a spectrophotometer.</li>
<li>To prepare the pCL-Eco plasmid, transform competent <i>E. coli</i> cells with 1 μL pCL-Eco plasmid (see steps 10a, c–h) and repeat steps 18–22.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> One Midiprep yields approximately 100 μg DNA.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Fc block (clone 2.4G2)</td>
<td>BD Pharmingen</td>
<td>Cat#553142</td>
</tr>
<tr>
<td>Anti-mouse CD3 brilliant violet 421 (clone 145-2С11)</td>
<td>BioLegend</td>
<td>Cat#100227</td>
</tr>
<tr>
<td>Anti-mouse TCRa Va2-FITC (clone B20.1)</td>
<td>BioLegend</td>
<td>Cat#127805</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>E. coli</i> XL1blue</td>
<td>Agilent</td>
<td>Cat#200249</td>
</tr>
<tr>
<td>
<i>Salmonella enterica</i> <i>serovar Typhimurium</i> virulent strain IE147</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Taq buffer</td>
<td>Evrogen</td>
<td>Cat#PB008</td>
</tr>
<tr>
<td>DATP</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#R0141</td>
</tr>
<tr>
<td>Taq polymerase</td>
<td>Evrogen</td>
<td>Cat#PK113S</td>
</tr>
<tr>
<td>T4 ligase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15224-017</td>
</tr>
<tr>
<td>SalI restriction enzyme</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#ER0591</td>
</tr>
<tr>
<td>AgeI restriction enzyme</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#ER1461</td>
</tr>
<tr>
<td>LB broth</td>
<td>VWR Life Science AMRESCO</td>
<td>Cat#89500-596</td>
</tr>
<tr>
<td>LB agar</td>
<td>VWR Life Science AMRESCO</td>
<td>Cat#97064-106</td>
</tr>
<tr>
<td>SS agar</td>
<td>Condalab</td>
<td>Cat#1064</td>
</tr>
<tr>
<td>Concanavalin A</td>
<td>Sigma-Aldrich</td>
<td>Cat#C5275</td>
</tr>
<tr>
<td>Hexadimethrine bromide (Polybrene)</td>
<td>Sigma-Aldrich</td>
<td>Cat#107689</td>
</tr>
<tr>
<td>Recombinant mouse interleukin-2</td>
<td>Sigma-Aldrich</td>
<td>Cat#11271164001</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>InsTAclone PCR Cloning Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#K1214</td>
</tr>
<tr>
<td>QIAquick Gel Extraction Kit</td>
<td>QIAGEN</td>
<td>Cat#28704</td>
</tr>
<tr>
<td>QIAGEN Plasmid Midi Kit</td>
<td>QIAGEN</td>
<td>Cat#12143</td>
</tr>
<tr>
<td>Wizard Plus SV Minipreps DNA Purification System</td>
<td>Promega</td>
<td>Cat#A1330</td>
</tr>
<tr>
<td>1× RBC lysis buffer</td>
<td>Invitrogen</td>
<td>Cat#00-4333</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>HEK293T cells</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>B10.D2(R101) mice, females and males (18–22 g, 6–8 weeks of age)</td>
<td>Breeding facility of FSBI "N.N. Blokhin NMRCO," Moscow, Russia</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>MigR1-PGK-TCRa-SM1</td>
<td>this study</td>
<td>n/a</td>
</tr>
<tr>
<td>MigR1-PGK-TCRa-SM14</td>
<td>this study</td>
<td>n/a</td>
</tr>
<tr>
<td>MigR1-PGK-TCRa-SM16</td>
<td>this study</td>
<td>n/a</td>
</tr>
<tr>
<td>MigR1-PGK-TCRa-SM20</td>
<td>this study</td>
<td>n/a</td>
</tr>
<tr>
<td>MigR1-PGK-TCRa-SM21</td>
<td>this study</td>
<td>n/a</td>
</tr>
<tr>
<td>MigR1-PGK-GFP</td>
<td>this study</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>pCL-Eco plasmid</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>Syringe with a blunt 22G gavage needle</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>SilentCrusher M homogenizer</td>
<td>Heidolph</td>
<td>Cat# 595-06000-00</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th>2× HBS buffer (store at −20°C)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES (250 mM), pH 7.05</td>
<td>50 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td>KCl (50 mM)</td>
<td>10 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td>Glucose (60 mM)</td>
<td>12 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td>NaCl (1.4 M)</td>
<td>280 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub> (7.5 mM)</td>
<td>1.5 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>3 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Mix the components of 2× HBS buffer by vortexing, then filter through a 0.20 μm filter. Store at −20°C. Repeated freeze/thaw cycles are permitted and do not alter pH of the buffer. If required, larger amount of HBS buffer can be prepared and aliquoted to 5–10 mL.</div>
<table id="undtbl2">
<thead>
<tr>
<th>CaCl<sub>2</sub> (store at −20°C)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>2.5 M</td>
<td>60 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>225 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>2.5 M</b></td>
<td><b>225 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Filter CaCl<sub>2</sub> solution through a 0.20 μm filter. Store at −20°C.</div>
<table>
<thead>
<tr>
<th>∗ Concanavalin A (store at −20°C)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>ConA</td>
<td>1 mg/mL</td>
<td>1 mg</td>
</tr>
<tr>
<td>RPMI 1640</td>
<td>-</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 mg/mL</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th>∗Polybrene (store at −20°C)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Polybrene</td>
<td>10 mg/mL</td>
<td>100 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>10 mg/mL</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Filter polybrene solution through a 0.20 μm filter. Store at −20°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Concanavalin A may be harmful by inhalation, digestion, or skin absorption. Avoid breathing and contacts with eyes, wear protective gloves and clothing. Polybrene is a possible mild eye and skin irritant, wear protective gloves.</div>
<h3 id="sec2.1">General comments</h3>
<p>Alternative gel purification systems, DNA extraction kits, and plasmids purification kits can be applied in this protocol.</p>
<p>In this protocol, the flow cytometry analysis were performed on a BD FACSCanto II cytometer using the FACSDiva 6.0 software (BD Bioscience). For this, we used Brilliant Violet 421-conjugated anti-CD3 antibodies and FITC-conjugated anti-Va2 antibodies. This analysis can be performed on any available flow cytometry instrument with any combination of fluorochromes.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Retrovirus production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 days</div>
<p>This section describes transfection of HEK293T cells to produce the retrovirus particles that contain the TCRα-SM construct or the control GFP construct. Sterile reagents and aseptic technique should be used throughout the procedure.</p>
<div class="note">
<span class="note-title">Note:</span> <i>In vitro</i> screening of TCRα-SM from the generated cDNA libraries revealed 5 variants of dominant-active salmonella-specific TCRα-SM: TCRα-SM1, TCRα-SM14, TCRα-SM16, TCRα-SM20, and TCRα-SM21 (<a href="#bib4">Kalinina et al., 2020</a>). All these five TCRα-SM variants were used for transduction of activated T cells for adoptive transfer experiments. This section describes production of retroviruses, individually containing these TCRα-SM constructs.</div>
<ol>
<li>Three days prior to transfection, thaw one vial of cryopreserved HEK293T cells in 37°C water bath.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use low-passage HEK293T cells (up to the 10<sup>th</sup> passage) to ensure the high transfection efficacy. Avoid passing HEK293T cells prior to transfection.</div>
<ol start="2">
<li>Seed 1.0 × 10<sup>6</sup> HEK293T cells per a flask in 6 T25 flasks in 5 mL DMEM, supplemented with 4.5 g/L glucose and 10% fetal bovine serum. Grow cells in incubator at 37°C, 5% CO<sub>2</sub> to 60%–80% confluence.
<div class="note">
<span class="note-title">Note:</span> Use only freshly cultured cells for transfection, avoiding cell passing.</div>
<div class="note">
<span class="note-title">Note:</span> One T25 flask of HEK293T cells is sufficient to produce 5 mL of control GFP virus or TCRα-SM virus, enough to transduce 3.0–18.0 × 10<sup>6</sup> activated mouse T cells. If more virus is required, this can be scaled up to T75 or T225.</div>
</li>
<li>In the day of transfection:
<ol type="a">
<li>In a 1.5 mL Eppendorf tube mix 6 μg of pCL-Eco plasmid and 6 μg of MigR1-PGK-TCRa-SM, add sterile ddH<sub>2</sub>O to the final volume 225 μL, and thoroughly mix by vortexing. Then add 25 μL of 2 M CaCL<sub>2</sub> and mix by pipetting. Add 250 μL 2× HBS in a drop-wise manner by gentle agitation. Incubate the resulting mixture for 5 min at 20–22°C. Similarly, prepare the transfection mix of pCL-eco+ MigR1-PGK-GFP.
<div class="critical">
<span class="critical-title">Critical:</span> Add 2× HBS buffer slowly by drops, gently but thoroughly mixing after each drop.</div>
<div class="note">
<span class="note-title">Note:</span> In this case, prepare 5 plasmids mixes of pCL-Eco with MigR1-PGK-TCRa-SM1, MigR1-PGK-TCRa-SM14, MigR1-PGK-TCRa-SM16, MigR1-PGK-TCRa-SM20, or MigR1-PGK-TCRa-SM21 and one plasmid mix of pCL-Eco + MigR1-PGK-GFP.</div>
<div class="note">
<span class="note-title">Note:</span> If more virus is required, transfect T75 or T225 HEK293T flasks, increasing proportionally the amount of plasmids, CaCL<sub>2</sub>, and 2× HBS to the final volume of the transfection mix 1.35 mL and 4.5 mL, respectively.</div>
</li>
<li>Remove the growth medium from T25 flasks, add 5 mL of fresh DMEM for HEK293T (see step 2). Place the flask into a horizontal position to ensure all cells are covered with the medium.</li>
<li>Add one transfection mix (see step 3a) to an individual HEK293T flask in a drop-wise manner, gently agitate the medium to uniformly disturb the transfection mix.</li>
<li>Place HEK293T flasks into incubator at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>24 h post-transfection:
<ol type="a">
<li>Check the transfection efficiency in the HEK293T flask, transfected with pCL-eco + MigR1-PGK-GFP (see step 3a) in a fluorescent microscope (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/511-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The efficiency of HEK293T transfection with pCL-eco + MigR1-PGK-GFP plasmid mix</div>
<p>Scale bar, 100 μm.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> For possible issues concerning the transfection efficacy, refer to the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.3">problem 2</a>).</div>
</li>
<li>Remove the growth medium, containing plasmids from T25 flasks, add 5 mL of fresh DMEM for HEK293T (see step 2).</li>
<li>Place HEK293T flasks into incubator at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>48 h post-transfection:
<ol type="a">
<li>Collect the growth medium from HEK293T flasks in individual 15 mL centrifuge tubes, and centrifuge 200 × <i>g</i> for 5 min to precipitate any residual cells. Filter the supernatant, containing viral particles into new sterile 15 mL tubes to obtain 5 mL of virus batch 1.</li>
<li>Add 5 mL of fresh DMEM (see step 2) to HEK293T flasks, place flasks into incubator for another 24 h.</li>
</ol>
</li>
<li>72 h post -transfection, repeat the step 5a to collect 5 mL of virus batch 2.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> In each step, working with HEK293T handle cells with care, avoiding rough medium flow or agitating to ensure cells remain adherent to a flask.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Supernatants with viral particles, collected in 48 and 72 h post-transfection (see steps 5a and 6; virus batch 1 and virus batch 2, respectively), can be frozen and stored at −70°C for at least 1 month without losing the transduction efficacy. Wrap the tube cap with parafilm before freezing.</div>
<div class="note">
<span class="note-title">Note:</span> When producing retroviruses from T75 or T225, aliquot virus batch 1 and virus batch 2 to 4 mL. Use one aliquot (4 mL) of virus batch 1 and virus batch 2 to transduce up to 18.0 × 10<sup>6</sup> activated mouse T cells. Store virus aliquots at −70°C, avoiding repeated freeze/thaw cycles.</div>
<h3 id="sec3.2">Preparation of cell suspensions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This section describes preparation of single-cell suspensions from the spleen and lymph nodes (LNs) of B10.D2(R101) mice. Sterile reagents and aseptic technique should be used throughout the procedure.</p>
<div class="note">
<span class="note-title">Note:</span> Prepare suspensions of the spleen and LNs cells 24 h prior to transduction, when using freshly collected virus batches (see steps 5 and 6), i.e., 24 h post-transfection of HEK293T cells.</div>
<ol start="7">
<li>Preparation of cell suspensions:
<ol type="a">
<li>Isolate spleen, mesenteric, both inguinal and axillary LNs of a euthanized mouse. Place spleen and LNs in two individual Potter homogenizers, containing 3 mL 1× PBS.</li>
<li>Using conic pestle, gently squeeze cells from the organ stroma.</li>
<li>Filter the resulting cell suspensions into individual 15 mL centrifuge tubes through a nylon filter (50 μm) to remove cell debris.</li>
<li>Centrifuge cell suspensions 200 × <i>g</i> for 5 min at 4°C, remove the supernatant, resuspend LNs cell pellet in 3 mL 1× PBS.</li>
<li>Resuspend splenocytes pellet in 1× RBC Lysis Buffer (Invitrogen) and remove erythrocytes, following the manufacturer`s instructions (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2F00-4333.pdf&amp;title=VGVjaG5pY2FsIERhdGEgU2hlZXQ6IDFYIFJCQyBMeXNpcyBCdWZmZXI=">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2F00-4333.pdf&amp;title=VGVjaG5pY2FsIERhdGEgU2hlZXQ6IDFYIFJCQyBMeXNpcyBCdWZmZXI=</a>).</li>
<li>After erythrocytes lysis, resuspend splenocytes pellet in 3 mL 1× PBS.</li>
<li>Combine resulting suspensions of LNs cells and splenocytes in one 15 mL centrifuge tube.</li>
</ol>
</li>
<li>Centrifuge cells suspension 200 × <i>g</i> for 5 min at 4°C, remove the supernatant.</li>
<li>Resuspend cells in 15 mL RPMI medium, supplemented with 10% fetal bovine serum, 5×10<sup>−5</sup> M 2- mercaptoethanol, 2 mM L-glutamine, 20 mM HEPES, and 10 μg/mL Ciprofloxacin.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Homogenize the spleen and LNs in individual 40 mm Petri dishes in 3 mL 1× PBS, using a 2 mL syringe plunger. Resuspend the cell suspension by pipetting with a 1 mL pipette (5–10 times). Transfer the resulting cell suspension into 15 mL centrifuge tube through a nylon filter and proceed to the steps 7d–g.</div>
<div class="caution">
<span class="caution-title">Caution:</span> Gently squeeze cells from the spleen and LNs to avoid cell death.</div>
<h3 id="sec3.3">Activation of mouse T cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This section describes generation of activated T cells by stimulation of the spleen and LNs cells of B10.D2(R101) mice with Concanavalin A (ConA) and recombinant mouse Interleukin-2 (IL-2).</p>
<ol start="10">
<li>Transfer the entire cell suspension (see step 9) into a T75 flask.</li>
<li>Add 45 μL ConA (1.0 mg/mL) to the final concentration 3.0 μg/mL.</li>
<li>Add 15 μL IL-2 (10,000 U/mL) to the final concentration 10.0 U/mL.</li>
<li>Gently agitate the flask to mix and uniformly disturb ConA and IL-2.</li>
<li>Place the flask into incubator at 37°C, 5% CO<sub>2</sub> for 24 h.</li>
</ol>
<h3 id="sec3.4">Transduction of activated mouse T cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days + 3 days for transduced T cells expansion</div>
<p>This section describes retroviral transduction of activated T cells of B10.D2(R101) mice by two rounds of spinoculation with the virus, containing the target TCRα-SM construct or the GFP construct.</p>
<ol start="15">
<li>24 h post-activation of mouse T cells, collect cells from the T75 flask into sterile 15 mL centrifuge tube.</li>
<li>Centrifuge cell suspension 200 × <i>g</i> for 5 min at 4°C, remove the supernatant.</li>
<li>Resuspend cell pellet in 3 mL complete RPMI medium (see step 9).</li>
<li>Count cells in Goryaev chamber or hemocytometer after trypan blue-eosin staining.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Trypan blue without eosin can be used as well. Use trypan blue-eosin staining to improve dead cells visualization in a light microscope.</div>
<ol start="19">
<li>Resuspend cells in complete RPMI medium (see step 9) to the final concentration 3.0 × 10<sup>6</sup> cells/mL.</li>
<li>Transfer 1 mL of cell suspension, containing 3.0 × 10<sup>6</sup> activated T cells in one well of a 6-well plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To generate control non-transduced T cells, transfer 3.0 × 10<sup>6</sup> activated T cells into a T75 flask in 10 mL complete RPMI medium. Add 15 μL ConA (1.0 mg/mL) to the final concentration 1.5 μg/mL and 10 μL IL-2 (10,000 U/mL) to the final concentration 10.0 U/mL, mix gently. Place the flask into incubator at 37°C, 5% CO<sub>2</sub> for 48 h. Then add 5 mL complete RPMI medium, 7.5 μL ConA (1.0 mg/mL) to the final concentration 1.5 μg/mL and 5 μL IL-2 (10,000 U/mL) to the final concentration 10.0 U/mL, mix gently. Place the flask into incubator for 3 days. Use these non-transduced activated T cells as the negative control for <i>in vivo</i> adoptive transfer experiments.</div>
<ol start="21">
<li>Add 4 mL of virus batch 1 TCRα-SM or GFP (see step 5a) and 3 mL of complete RPMI medium (see step 9) to the final volume of 8 mL per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this case, plate 1 mL of cell suspension (3.0 × 10<sup>6</sup> cells/well) into 6 wells. Add virus batch 1 TCRα-SM1, TCRα-SM14, TCRα-SM16, TCRα-SM20, TCRα-SM21, and GFP to each individual well.</div>
<div class="note">
<span class="note-title">Note:</span> When using frozen virus, thaw one aliquot of virus batch 1 (4 mL) and warm up on a water bath at 37°C before adding to activated T cells.</div>
<ol start="22">
<li>In each well, add 8 μL IL-2 (10,000 U/mL) to the final concentration 10.0 U/mL and 6.4 μL polybrene (10.0 mg/mL) to the final concentration 8.0 μg/mL. Mix gently but thoroughly by pipetting with a 1 mL pipette (2- 3 times).</li>
<li>Wrap the plate lid with parafilm and spinoculate plates 2,000 × <i>g</i> for 2 h at 22°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure that plates are well-balanced. When using the second plate for balancing, fill the corresponding number of wells with 8 mL sterile PBS per a well.</div>
<ol start="24">
<li>Resuspend cells with a 5 mL pipette (5–10 times) in each well.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Thoroughly resuspend cells to avoid cell aggregation that increases cell death. Look through each well in a light microscope to ensure cells are well resuspended.</div>
<ol start="25">
<li>Put the plate into incubator at 37°C, 5% CO<sub>2</sub> for 16–20 h.</li>
<li>Collect all cells from each well in an individual 15 mL sterile centrifuge tube, using a 5 mL pipette.</li>
<li>Centrifuge cells suspension 200 × <i>g</i> for 5 min at 4°C, remove the supernatant.</li>
<li>Resuspend cell pellet in 4 mL complete RPMI medium (see step 9) and seed all cells back into the corresponding well of the plate.</li>
<li>Add 4 mL of virus batch 2 TCRα-SM or GFP (see step 6) to each corresponding well to the final volume of 8 mL per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When using frozen virus, thaw one aliquot of virus batch 2 (see step 6) and warm up on a water bath at 37°C before adding to a well.</div>
<ol start="30">
<li>Repeat steps 22–24.</li>
<li>Put the plate into incubator at 37°C, 5% CO<sub>2</sub> for 2–4 h.</li>
<li>Repeat steps 26 and 27.</li>
<li>Resuspend cells in 10 mL complete RPMI medium (see step 9).</li>
<li>Add 22.5 μL ConA (1.0 mg/mL) and 15 μL IL-2 (10,000 U/mL), mix gently but thoroughly by pipetting with a 5 mL pipette (2–3 times).</li>
<li>Transfer all cell suspensions in individual T75 flasks, add 5 mL complete RPMI medium (see step 9) to the final volume 15 mL in each flask.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the final concentration of ConA and IL-2 is 1.5 μg/mL and 10.0 U/mL, respectively.</div>
<ol start="36">
<li>Put the flasks into incubator at 37°C, 5% CO<sub>2</sub> for 3 days.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Control cell growth in a daily manner to avoid overgrowth. If required, add extra complete RPMI medium, supplemented with 1.5 μg/mL ConA and 10.0 U/mL IL-2. For possible issues concerning viability of transduced T cells, refer to the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.5">problem 3</a>).</div>
<h3 id="sec3.5">Evaluation of transduction efficacy</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This section describes how to evaluate the transduction efficacy using flow cytometry.</p>
<ol start="37">
<li>Collect transduced T cells and non-transduced T cells into individual 15 mL centrifuge tubes (or 50 mL tubes, if extra medium were added to cultured T cells).</li>
<li>Centrifuge cells suspension 200 × <i>g</i> for 5 min at 4°C, remove the supernatant.</li>
<li>Gently resuspend transduced T cells in 5 mL sterile 1× PBS.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep transduced T cells on ice before adoptive transfer to avoid cell death.</div>
<ol start="40">
<li>Count cells in Goryaev chamber or hemocytometer after trypan blue-eosin staining.</li>
<li>Aliquot approximately 3.0 × 10<sup>5</sup> cells from each tube and transfer to round-bottom flow cytometry tubes.</li>
<li>Add 1 mL 1× PBS and centrifuge 200 × <i>g</i> for 5 min at 4°C.</li>
<li>Remove the supernatant, gently resuspend cells in 100 mL 1× PBS.</li>
<li>Add Fc block antibodies (0.1 μg per 10<sup>5</sup> cells), incubate cells for 10 min at 4°C.</li>
<li>Centrifuge tubes 200 × <i>g</i> for 5 min at 4°C, remove the supernatant, and gently resuspend cells in 100 mL 1× PBS.</li>
<li>Add anti-CD3 and anti-Va fluorescent antibodies, incubate cells 30–40 min at 4°C in dark.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prior to the flow cytometry analysis, identify Va families, to which each TCRα belongs. Use the corresponding anti-Va antibodies to stain TCRα transduced T cells. In this case, stain TCRα-SM1 transduced T cells with anti-Va2 antibodies (TCRα-SM1 belongs to the Va2 family).</div>
<div class="note">
<span class="note-title">Note:</span> Use antibodies dilutions, recommended by the manufacturer. It is also possible to preliminary titrate each batch of antibodies to determine the working dilution.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If TCRα belongs to the Va family, to which there are no commercially available antibodies, use the level of GFP expression in the GFP-transduced T cells as the reference to identify the transduction efficacy for TCRα-transduced T cells. In these case, TCRα-SM14 belongs to the Va3 family, TCRα-SM20 to the Va5 family, TCRα-SM16 and TCRα-SM21 to the Va16 family, for all of which there are no commercial antibodies.</div>
<ol start="47">
<li>Add 1 mL 1× PBS and centrifuge 200 × <i>g</i> for 5 min at 4°C.</li>
<li>Resuspend cells in 500 μL 1× PBS, add 10 μL PI (10.0 μg/mL) to all tubes.</li>
<li>Analyze on a flow cytometer (<a href="#fig2">Figure 2</a>):
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/511-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Gating strategy for analysis of the transduction efficacy</div>
<p>(A) Set 3 subsequent gates: gate lymphocytes by parameters SSC-A versus FSC-A (lymphocytes), then gate singlet cells by parameters FSC-H versus FSC-A (single cells), and gate live cells as Percp-Cy5.5<sup>neg</sup> on Percp-Cy5.5_Log histogram.</p>
<p>(B) Set a dotplot CD3 versus Va2 for live singlet cells, gate CD3+ cells.</p>
<p>(C) Set Va2_Log histogram for CD3+ live singlet cells, gate Va2+ cells.</p>
<p>(D) Set FITC_Log histogram for CD3+ live singlet cells, gate FITC<sup>+</sup> cells as GFP+ cells</p>
</figcaption>
</figure>
<ol type="a">
<li>Set three subsequent gates as follows: side (SSC) versus forward (FSC) scatter to gate lymphocytes, then FSC-H versus FSC-A to gate singlets, and Percp-Cy5.5 negative cells to gate live cells (<a href="#fig2">Figure 2</a>A). The Percp-Cy5.5-A channel of BD FacsCanto II is used to detect PI fluorescence. Use log sales for SSC and fluorochromes. Collect 10<sup>5</sup> cells from the lymphocyte gate to analyze the transduction efficacy.</li>
<li>set CD3 versus Va2 dotplot for non-transduced T cells and TCRα-SM1 transduced T cells and gate CD3+ cells (<a href="#fig2">Figure 2</a>B).</li>
<li>set Va2 histogram for CD3+ cells and gate Va2+ cells to evaluate percent of Va2+ T cells (<a href="#fig2">Figure 2</a>C). Subtract the percentage of Va2+ cells in the population of non-transduced T cells from the percentage of Va2+ cells in the population of TCRα -SM1 transduced T cells to evaluate the transduction efficacy. In this case, the transduction efficacy for TCRa-SM1 transduced T cells is 81.4–11.0 = 71.4% (<a href="#fig2">Figure 2</a>C).</li>
<li>To evaluate the transduction efficacy in the population of GFP-transduced T cells, set FITC histogram for CD3+ live singlet cells and gate FITC<sup>+</sup> cells (<a href="#fig2">Figure 2</a>D). In this case, the transduction efficacy for GFP-transduced T cells is 70.1%.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The transduction efficacy for TCRa-SM1 and GFP correlate well, thus, GFP transduction efficacy can be correctly used to assess the transduction efficacy for T cells modified with TCRα, for which there are no available antibodies. The transduction efficacy can vary from 40% to 70%.</div>
<h3 id="sec3.6">Adoptive transfer of transduced T cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h (depending on the number of animals used in the adoptive transfer experiment)</div>
<p>This section describes how to perform intravenous adoptive transfer (via the retro-orbital sinus) of transduced and control (non-transduced) activated T cells to mice.</p>
<div class="note">
<span class="note-title">Note:</span> To evaluate the therapeutic potential of transduced T cells, perform adoptive transfer on day 3 post-infection of mice with virulent <i>Salmonella enterica serovar Typhimurium.</i> To evaluate the prophylactic potential of transduced T cells, perform adoptive transfer 3 days prior to infection of mice (<a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/511-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Schemes of the adoptive transfer experiments</div>
</figcaption>
</figure>
<ol start="50">
<li>Calculate concentration of TCRa-SM transduced T cells as follows:</li>
</ol>
<p class="chem">C<sub>transduced cells</sub> = C<sub>total</sub> × Transduction coefficient,</p>
<p>where C<sub>transduced cells</sub> is the concentration of transduced T cells (×10<sup>6</sup>/mL); C<sub>total</sub> is the total concentration of cells in the corresponding tube (×10<sup>6</sup>/mL) as counted in step 40; transduction coefficient is the transduction efficacy as measured in step 49c,d, divided by 100.</p>
<p><b>Example:</b> Calculation of TCRa-SM1 transduced T cells concentration</p>
<p>Concentration of cells in the corresponding tube (C<sub>total</sub>), 5.0 × 10<sup>6</sup>/mL (counted in step 40)</p>
<p>Transduction efficacy for TCRa-SM1, 71.4% (see step 498c)</p>
<p>Transduction coefficient, 71.4/100 = 0.714</p>
<p>Concentration of TCRa-SM1 transduced T cells (C<sub>transduced cells</sub>) = 5.0 × 0.714 = 3.57 × 10<sup>6</sup>/mL</p>
<ol start="51">
<li>Prepare the mix of TCRa-SM transduced T cells to the final dose 3.0 × 10<sup>5</sup> transduced cells per mouse in 200 μL 1× PBS. In this case, aliquot and mix 1.5 × 10<sup>5</sup> transduced TCRa-SM16 cells and 1.5 × 10<sup>5</sup> transduced TCRa-SM21 cells to the final volume 200 μL per mouse for therapeutic adoptive transfer. Aliquot and mix 1.0 × 10<sup>5</sup> transduced TCRa-SM1, TCRa-SM14, and TCRa-SM20 each to the final volume 200 μL per mouse for prophylactic adoptive transfer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The total dose of adoptively transferred T cells will amount 3.0 × 10<sup>5</sup>/Transduction coefficient per mouse (see step 50). In this case, the total dose of T cells will be 3.0 × 10<sup>5</sup>/0.714 = 4.2 × 10<sup>5</sup> cell per mouse.</div>
<div class="critical">
<span class="critical-title">Critical:</span> 2 or more types of TCRa-SM transduced T cells must be used simultaneously for adoptive transfer to achieve therapeutic (or prophylactic) effect. Never exceed the total dose of adoptively transferred <b>transduced</b> T cells (all types together) above 3.0 × 10<sup>5</sup> cells/mouse to avoid toxic side-effects of activated T cells <i>in vivo</i> (cytokine storm, immunosuppression). Our data showed that adoptive transfer of 3.0 × 10<sup>7</sup> transduced T cells/mouse, followed by infection on day 7 post-adoptive transfer resulted by day 9 post-infection in increased mice mortality, 2–3 log increase in salmonella burden, and splenomegaly with destructive changes in mice injected with either non-transduced or transduced T lymphocytes comparing to mice without the adoptive transfer.</div>
<ol start="52">
<li>Aliquot non-transduced T cells to the final dose of 3.0 × 10<sup>5</sup>/Transduction coefficient per mouse in 200 μL 1× PBS. In this case, the total dose of non-transduced T cells will be 3.0 × 10<sup>5</sup>/0.7 = 4.2 × 10<sup>5</sup> cell per mouse.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare the master mix of non-transduced T cells or TCRa-SM transduced T cells for an intended number of mice (N). In this case, for 5 mice mix 10.7 × 10<sup>5</sup> of total TCRa-SM16 cells and 10.7 × 10<sup>5</sup> of total TCRa-SM21 cells (i.e., 1.5 × 10<sup>5</sup> transduced TCRa cells × 5/Transduction coefficient) in 1.0 mL 1× PBS for therapeutic adoptive transfer. For prophylactic adoptive transfer to 5 mice, mix 7.1 × 10<sup>5</sup> of total TCRa-SM1, TCRa-SM14, TCRa-SM20 cells each (i.e., 1.0 × 10<sup>5</sup> transduced TCRa cells × 5/Transduction coefficient) in 1.0 mL 1× PBS.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare master mix for N+2 mice to consider any technical losses during the adoptive transfer procedure.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not use TCRa-SM transduced T cells for adoptive transfer if the transduction efficacy is below 50% to reduce non-transduced cells burden. This will ensure the high specific therapeutic effect of TCRa-SM transduced T cells while reducing non-specific action of non-transduced T cells.</div>
<ol start="53">
<li>Using an insulin syringe, transfer 200 μL of cell suspension into a mouse through the retro-orbital sinus (<a href="#bib3">Hedrich et al., 2004</a>)</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Thoroughly mix cells before sampling an aliquot. Keep cell suspensions on ice throughout the adoptive transfer procedure to ensure cells viability.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Cells can be injected via a tail vein without affecting the overall efficacy of adoptive transfer therapy.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid bubbles in a syringe while injecting cells into mice. Large air bubbles can cause air embolism and occasionally mouse death.</div>
<h3 id="sec3.7">Preparation of fresh cultures of <i>Salmonella enterica</i> <i>serovar Typhimurium</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section describes preparation of freshly grown cultures of <i>S. enterica serovar Typhimurium</i> virulent strain IE 147, subsequently used for infection of mice.</p>
<ol start="54">
<li>Thaw one vial of cryopreserved <i>S. enterica serovar Typhimurium</i> strain IE 147 in 37°C water bath.</li>
<li>Transfer bacterial cells in 5 mL LB broth and incubate in a thermostatic shaker at 185 rpm at 37°C for 18 h.</li>
<li>Prepare 7 serial 10x dilutions of bacterial culture in sterile 1× PBS as follows: 900 μL PBS + 100 μL of bacterial cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep freshly grown bacterial cells in LB broth at 4°C prior to infecting mice.</div>
<ol start="57">
<li>Seed 500 μL of bacterial cells from 10<sup>−6</sup> and 10<sup>−7</sup> dilutions onto individual Petri dishes with SS agar.</li>
<li>Put Petri dishes into incubator at 37°C for 12 h.</li>
<li>Count a number of colonies on each Petri dish and calculate the concentration of salmonella in LB broth (CFU/mL).</li>
<li>Aliquot 4–6 × 10<sup>6</sup> CFU of bacterial cells and dilute in sterile 1× PBS to the final volume 200 μL per mouse.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare the master mix of bacterial cells for N+2 mice to consider any technical losses during infection procedure.</div>
<div class="critical">
<span class="critical-title">Critical:</span> 4–6 × 10<sup>6</sup> CFU/mouse corresponds to LD<sub>27</sub> of <i>S. enterica serovar Typhimurium</i> strain IE 147 for B10.D2(R101) mice. When using other virulent strains and/or other mice lines, determine LD<sub>50</sub> for each concrete experimental model and use LD<sub>20</sub>–LD<sub>30</sub> to infect mice for adoptive transfer experiments.</div>
<h3 id="sec3.8">Infection of mice with <i>Salmonella enterica serovar Typhimurium</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–1.5 h (depending on the number of animals used in the adoptive transfer experiment)</div>
<p>This section describes how to perform per os (intragastric) infection of mice with <i>S. enterica serovar Typhimurium</i> virulent strain IE 147 (<a href="#bib3">Hedrich et al., 2004</a>).</p>
<ol start="61">
<li>Using a syringe with a blunt 22G gavage needle, slowly administer per os 200 μL suspension of bacterial cells to a mouse.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For a therapeutic adoptive transfer experiment, infect mice 3 days prior to adoptive transfer of T cells. For a prophylactic adoptive transfer experiment, infect mice 3 days post-adoptive transfer of T cells (<a href="#fig3">Figure 3</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Similarly, infect mice without the adoptive transfer of either transduced or non-transduced T cells. Use this group of mice as the background control.</div>
<h3 id="sec3.9">Evaluation of bacterial loads in infected mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section describes how to evaluate the number of bacterial colony forming units (CFU) in the spleen of mice, infected with salmonella.</p>
<ol start="62">
<li>Euthanize mice on day 7 post-infection and aseptically isolate the spleen.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The therapeutic adoptive transfer experiment lasts for 7 days, the prophylactic adoptive transfer for 10 days (<a href="#fig3">Figure 3</a>).</div>
<ol start="63">
<li>Prepare homogenates from each spleen individually using the SilentCrusher M homogenizer in 1 mL sterile 1× PBS.</li>
<li>Prepare one 10× dilution of the resulted spleen homogenate in 1 mL sterile 1× PBS as follows: 900 μL PBS + 100 μL of cell suspension.</li>
<li>Seed 500 μL of the diluted spleen homogenates onto individual Petri dishes with SS agar.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep the rest of undiluted spleen homogenates at 4°C.</div>
<ol start="66">
<li>Put Petri dishes into incubator at 37°C for 18–24 h.</li>
<li>Count a number of bacterial colonies on each Petri dish, calculate bacterial CFU per spleen.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For possible issues concerning the evaluation of bacterial loads, refer to the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.7">problem 4</a>).</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The effect of adoptive transfer therapy of salmonella-infected mice with TCRα-SM- transduced T cells will be assessed by comparing the bacterial loads in the spleen of mice injected with the mix of TCRα-SM- transduced T cells with the bacterial loads in the spleen of mice with adoptively transferred non-transduced T cells and of mice without the adoptive transfer (<a href="#fig4">Figure 4</a>). Refer to <a href="#fig3">Figure 3</a> for the experimental schemes.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/511-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Adoptive transfer therapy with TCRa-SM-transduced T cells results in a significant improvement in control of salmonellosis in mice</div>
<p>(A) Therapeutic adoptive transfer of mix of T cells, transduced with TCRa-SM16 and TCRa-SM21 on day 3 post-infection of mice with <i>S. enterica</i> <i>serovar Typhimurium</i> strain IE 147.</p>
<p>(B) Prophylactic adoptive transfer of mix of T cells, transduced with TCRa-SM1, TCRa-SM14, and TCRa-SM20 3 days before infection of mice with virulent <i>S. enterica</i> <i>serovar Typhimurium</i> strain IE 147. Bars indicate mean ± SD. One-way ANOVA</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Flow cytometry data was analyzed using Flow Jo 7.6. (TreeStar Inc).</p>
<p>Data are presented as mean ± SD. All statistical analyses were performed using one-way analysis of variance (ANOVA). A p-value &lt; 0.05 was considered significant. All statistical analyses were performed using Prism software (v. 8.1.2, GraphPad).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Our protocol was developed for the experimental infectious model of salmonellosis in B10.D2(R101) mice. Preliminary, we analyzed B10.D2(R101) mice susceptibility to <i>Salmonella enterica</i> <i>serovar Typhimurium</i> virulent strain IE147 and determined LD<sub>50</sub> of this strain for this particular mouse line. When using another mice line and/or virulent strain of <i>S. enterica serovar Typhimurium</i>, initial experiments must be set to determine LD<sub>50</sub> of a bacterial strain for a mice line used.</p>
<p>In the protocol of activated mouse T cells transduction, we used five variants of dominant-active salmonella-specific TCRα, originated from memory T cells of B10.D2(R101) mice, immunized with avirulent salmonella. Due to MHC-restriction of TCR, these variants of TCRα can be used for transduction of T lymphocytes and subsequent adoptive transfer only to MHC-matched mice. The MHC haplotype of B10.D2(R101) mice is K<sup>d</sup>I-A<sup>d</sup>I-E<sup>d</sup>D<sup>b</sup>.</p>
<p>To achieve high therapeutic or prophylactic efficacy, several types of TCRα-transduced T cells (at least 2) should be simultaneously transferred to mice. In our work, we evaluated <i>in vivo</i> effects of two mixes of T cells, individually transduced with 5 variants of salmonella-specific TCRα. Additional <i>in vivo</i> experiments using different combinations of these 5 types of TCRα-transduced T cells (or all 5 variants at once) could further improve the efficacy of adoptive transfer therapy and help determine selection patterns for the most effective combinations of distinct TCRα variants. We assume that pre-selected dominant-active TCRα variants should be characterized on the part of their antigen specificity, so the most effective combination of transduced lymphocytes would have several variants of TCRα, each specific to the unique bacterial antigen. In this respect, it is also important to identify the availability, abundance, and immunogenicity of pathogenic antigens to predict what type of TCRα-transduced T cells would be the most effective <i>in vivo</i> under adoptive transfer to an infected host.</p>
<p>Our technique can be potentially applied against various infections if therapeutic dominant-active antigen-specific TCRα against each particular pathogen can be identified and isolated. Further studies with different infectious model systems are required to prove the efficacy of TCRα-transduced T cells adoptive transfer therapy in the treatment of bacterial and viral diseases. If the T cell response is a crucial factor of immunity to a pathogen, we assume that clones with α-chain-centric TCRs could represent a substantial part of pathogen-specific T cells. Considering some physiological features of α-chain TCR (TCRα gene editing and lack of allelic exclusion during rearrangement in the thymus, specific aspects of interaction with MHC-peptide complexes), it seems plausible that generation of α-chain-centric TCRs could be a regular phenomenon. Although identification of TCRβ-centric TCRs was beyond the scope of our studies, analyses of the frequency of generation and the therapeutic potential of T cells with TCRβ-centric TCRs comparing to clones with TCRα-centric TCRs would improve the technology described here.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No PCR products were obtained during amplification of V- and J-segments of TCRα (see section <a href="#sec1.1">cloning TCRα-SM1</a>, steps 1 and 2)</p>
<h3 id="sec7.2">Potential solution</h3>
<p>In cloning each individual TCRα, unique primers are used for amplification of the respective V- and J-segment (Fragment 1 and Fragment 2). For each specific pair of primers, annealing temperature must be optimized. For this, perform analytical PCR with each pair of primers, setting a range of annealing temperature (62°С, 64°С, 66°С, 68°С, 70°С) for each PCR mix individually. Use the total spleen cDNA of a non-immunized syngeneic mouse as the negative template and cDNA of enriched immunized splenocytes as a positive template. The annealing temperature for each combination of primers will be specific if the respective PCR product is obtained only from the positive template, whereas no PCR products are detected from the negative template. Next, perform preparative PCR with the defined cycling conditions to amplify Fragment 1, Fragment 2, and Fragment 3 of each individual TCRα.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Low transfection efficacy (see section <a href="#sec3.1">retrovirus production</a>, step 4)</p>
<h3 id="sec7.4">Potential solution</h3>
<p>pH of the HBS buffer can significantly affect the efficiency of transfection. Make small aliquots (up to 1 mL) of freshly prepared HBS buffer and titrate pH from 7.05 to 7.2 with 0.5 intervals. Use these titrated HBS aliquots to transfect T25 HEK293T cells with MigR1-PGK-GFP (see section <a href="#sec3.1">retrovirus production</a>, steps 1–4). Check the transfection efficacy in 24 h post-transfection in a fluorescent microscope to detect a HBS buffer with the most effective pH. Titrate the HBS buffer stock to adjust its pH to the effective value (<a href="#fig5">Figure 5</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/511-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Titration of 2× HBS buffer</div>
<p>Scale bar, 100 μm.</p>
</figcaption>
</figure>
<h3 id="sec7.5">Problem 3</h3>
<p>Low viability of transduced T cells (see section <a href="#sec3.4">transduction of activated mouse T cells</a>, step 36)</p>
<h3 id="sec7.6">Potential solution</h3>
<p>While retroviral transduction doesn`t affect the viability of modified mouse T cells (<a href="#fig3">Figure 3</a>A), activated lymphocytes are vulnerable to culture conditions and a deficit of growth factors (especially, IL-2). Activated T cells proliferate vigorously, thus, control cell growth in a daily manner. Maintain the confluence of cultured T cells at 60%–70%. If necessary, add an extra complete RPMI 1640 medium, supplemented with ConA and (importantly) IL-2 to the final concentration of 1.5 μg/mL and 10.0 U/mL, respectively. The same rules are applicable for culturing non-transduced (control) T cells. Before the adoptive transfer, evaluate T cells viability in a light microscope after staining with trypan blue-eosin and/or by flow cytometry after PI staining. Do not use T cells with the viability below 70% in the adoptive transfer experiments.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Too many or no bacterial colonies were grown from 10× dilution of the spleen homogenate (see section <a href="#sec3.9">evaluation of bacterial loads in infected mice</a>, step 67)</p>
<h3 id="sec7.8">Potential solution</h3>
<p>If no bacterial colonies were grown, seed the corresponding undiluted spleen homogenate onto SS agar and count bacterial colonies in 24 h. If no colonies are grown from the undiluted spleen homogenate, this indicates complete bacterial clearance in a mouse. This effect can be observed in mice injected with TCRα-SM-transduced T cells, indicating the vivid anti-bacterial protective potential of these modified lymphocytes.</p>
<p>If too many bacterial colonies are grown from 10× diluted spleen homogenate, so they cannot be counted, prepare 2–3 serial 10× dilutions and seed them on SS agar for 24 h. Count bacterial CFU on the Petri dish with the minimal number of colonies and calculate bacterial CFU per spleen, using the dilution index (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/511-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Possible problems in the analysis of bacterial loads</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bolotin, D.A., Poslavsky, S., Mitrophanov, I., Shugay, M., Mamedov, I.Z., Putintseva, E.V., and Chudakov, D.M. (2015). MiXCR: software for comprehensive adaptive immunity profiling. Nat. Methods <i>12</i>, 380-381. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00075-7/sref1">View at publisher</a></p>
<p id="bib2">Egorov, E.S., Merzlyak, E.M., Shelenkov, A.A., Britanova, O.V., Sharonov, G.V., Staroverov, D.B., Bolotin, D.A., Davydov, A.N., Barsova, E., Lebedev, Y.B., et al. (2015). Quantitative profiling of immune repertoires for minor lymphocyte counts using unique molecular identifiers. J. Immunol. <i>194</i>, 6155-6163. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00075-7/sref2">View at publisher</a></p>
<p id="bib3">Hedrich, H., Bullock, G., and Petruz, P. (2004). The Handbook of Experimental Animals: The Laboratory Mouse (Elsevier Academic Press), pp. 527-542. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00075-7/sref3">View at publisher</a></p>
<p id="bib4">Kalinina, A.A., Nesterenko, L.N., Bruter, A.V., Balunets, D.V., Chudakov, D.M., Izraelson, M., Britanova, O.V., Khromykh, L.M., and Kazansky, D.B. (2020). Adoptive immunotherapy based on chain-centric TCRs in treatment of infectious diseases. iScience <i>23</i>, 101854-101879. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00075-7/sref4">View at publisher</a></p>
<p id="bib5">Shugay, M., Britanova, O.V., Merzlyak, E.M., Turchaninova, M.A., Mamedov, I.Z., Tuganbaev, T.R., Bolotin, D.A., Staroverov, D.B., Putintseva, E.V., Plevova, K., et al. (2014). Towards error-free profiling of immune repertoires. Nat. Methods <i>11</i>, 653-655. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00075-7/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Prof. Dmitry Kazansky (<a href="mailto:kazansky1@yandex.ru">kazansky1@yandex.ru</a>).</p>
<h4>Materials Availability</h4>
<p>Requests for plasmids and all other pertinent information can be forwarded to Prof. Dmitry Kazansky.</p>
<h4>Data and Code Availability</h4>
<p>The published article includes all datasets generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>We thank A.V. Belyavskii (Engelhardt Institute of Molecular Biology of the Russian Academy of Sciences, Moscow, Russia) for providing us with the pCL-Eco plasmid. This study was supported by Russian Foundation for Advanced Research Projects (agreement # 6/053/2015-2018).</p>
<h3>Author Contributions</h3>
<p>A.K., A.B., and L.N. drafted and described the protocols. L.K. and D.K. provided the <a href="#key-resources-table">key resources table</a>. A.B., L.N., L.K., and D.K. edited the manuscript. A.K. prepared the figures. A.K. prepared the graphical abstract.</p>
<h3>Declaration of Interests</h3>
<p>The authors have a patent of the Russian Federation related to this work (Kazansky, D.B., Khromykh, L.M., Kalinina, A.A., Silaeva, Y.Y., Zamkova, M.A., Bruter, A.V., Persiyantseva, N.A., Chikileva, I.O., Jolokhava, L.Kh., Nesterenko, L.N. et al. (2018). A method of creating anti-infectious immunologic defense against <i>Salmonella typhimurium</i> and <i>Listeria monocytogenes</i> by transgenesis of T lymphocytes. Patent #2706554).</p>
</section>
</article>