<article>
<section>
<h1 id="header">Expression and purification of tag-free SARS-CoV-2 RNA-dependent RNA polymerase in <i>Escherichia coli</i>
</h1>
<p><time datetime="2021-03-05">Published: March 5, 2021</time></p>
<p>Tyler L. Dangerfield,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Nathan Z. Huang,<sup><a href="#aff1">1</a></sup> and Kenneth A. Johnson<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Biosciences, University of Texas at Austin, Austin, TX 78712, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:tylerdangerfield@utexas.edu">tylerdangerfield@utexas.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:kajohnson@mail.utexas.edu">kajohnson@mail.utexas.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100357">10.1016/j.xpro.2021.100357</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The RNA-dependent-RNA polymerase (RdRp) from SARS-CoV-2 is an important drug target because it is responsible for viral RNA genome replication. Efficient production of recombinant RdRp is important in screening antivirals to treat COVID-19. Here, we present our protocol for expression of tag-free replication complex proteins in <i>E. coli</i> and subsequent purification. Despite the added complexity of multiple purification steps, our methods provide greater activity, yield at lower cost, and are faster than baculovirus expression systems.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Dangerfield et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Soluble expression of tag-free, highly active SARS-CoV-2 polymerase complex in <i>E. coli</i>
</li>
<li>Purification protocol for NSP-7/8/12 RNA polymerase complex is provided</li>
<li>Protocol for expression and purification of supplemental NSP8 included</li>
<li>Purified SARS-CoV-2 RNA polymerase complex catalyzes fast processive RNA replication</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/496-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The SARS-CoV-2 RNA-dependent-RNA polymerase (RdRp) complex is responsible for replication of the viral RNA genome and consists of non-structural proteins (nsp)12, nsp7, and nsp8. The only FDA approved treatment for COVID-19 infections is the nucleoside analog, Remdesivir, highlighting the importance of this enzyme as a target for therapies to combat the viral infection. Many protocols to express tagged RdRp complex proteins in baculovirus-infected insect cells have been published ( <a href="#bib3">Gordon et al., 2020</a>; <a href="#bib4">Hillen et al., 2020</a>; <a href="#bib5">Yin et al., 2020</a>) but these methods are slow, expensive, and have a limited yield. Furthermore, tagged proteins can display different activity than their untagged counterparts, and appropriate controls for the effect of the tag are sorely lacking. Here we present our protocol for co-expression of nsp12/7/8 complex and tag-free nsp8 in <i>E. coli</i> and subsequent purification. The procedure yields approximately 7 mg of highly active enzyme per liter of culture that catalyzes RNA polymerization at a rate sufficient to account replication of the 30 kb viral genome in 2 min (<a href="#bib2">Dangerfield et al., 2020</a>).</p>
<h3 id="sec1.1">Prepare electrocompetent <i>E. coli</i> BL21/ pG-Tf2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>In this step, electrocompetent <i>E. coli</i> BL21 / pG-Tf2 are prepared which will be used to co-express the RdRp complex in <i>E. coli</i> with the aid of chaperone proteins from the pG-Tf2 plasmid.</p>
<ol>
<li>Before starting:
<ol type="a">
<li>Autoclave the following items: 2 L dH<sub>2</sub>O split into 2 flasks, 100 mL of 10% (v/v) glycerol, 4 × 250 mL centrifuge bottles and caps, 1.5 mL microcentrifuge tubes, 250 mL graduated cylinder, 100 mL graduated cylinder, 1 L LB media in 2.8 L baffle-bottomed flasks. Add chloramphenicol to 20 μg/mL after cooling to 25°C.</li>
<li>Chill the following items to 4°C: 2 L autoclaved dH<sub>2</sub>O, 100 mL autoclaved 10% (v/v) glycerol, 4 × 250 mL autoclaved centrifuge bottles and caps, 1.5 mL autoclaved microcentrifuge tubes, 250 mL autoclaved graduated cylinder, 100 mL autoclaved graduated cylinder</li>
</ol>
</li>
<li>Transform BL21 <i>E. coli</i> with the plasmid pG-Tf2 following transformation protocols for the type of competent BL21 <i>E. coli</i> used. Spread the transformation reaction solution onto an LB/Agar plate containing 20 μg/mL chloramphenicol.</li>
<li>Incubate the plate for 16 h at 37°C.</li>
<li>Select a single colony from the plate with the previous day’s transformation and inoculate a 10 mL starter culture of 2× YT media (defined under Materials and Equipment) with 20 μg/mL chloramphenicol. Grow the culture for 16 h at 37°C with shaking at 200 RPM.</li>
<li>Inoculate 1 L of LB media (defined under Materials and Equipment) + 20 μg/mL chloramphenicol with the 10 mL starter culture</li>
<li>Incubate cells with shaking at 30°C, 250 RPM. Measure the OD<sub>600</sub> periodically (every half hour or so until the OD<sub>600</sub> reaches 0.2, then every 15 min).</li>
<li>When the OD<sub>600</sub> reaches 0.35–0.4, immediately put the cells on ice and chill the culture for 25 min, swirling occasionally to ensure even cooling. Place centrifuge bottles on ice at this time and move the needed rotors to the centrifuge to start cooling to 4°C</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to not let the OD <sub>600</sub> get any higher than 0.4. The OD<sub>600</sub> should be carefully monitored and checked often, especially when it gets above 0.2 as the cells grow exponentially. It usually takes about 3 h to reach an OD<sub>600</sub> of 0.35 when using a 10 mL starter culture added to 1 L of medium. It is also very important to keep the cells at 4°C for the remainder of the procedure following growth. The cells and any bottles or solutions that they come in contact with must be pre-chilled to 4°C.</div>
<ol start="8">
<li>Spin 1: Split the 1 L culture into four parts by pouring about 250 mL into ice cold centrifuge bottles. Harvest the cells by centrifugation at 1,000 × <i>g</i> for 20 min at 4°C.</li>
<li>Decant the supernatant and resuspend each pellet in 200 mL of ice cold dH<sub>2</sub>O by gentle swirling</li>
<li>Spin 2: Harvest the cells by centrifugation at 1,000 × <i>g</i> for 20 min at 4°C</li>
<li>Decant the supernatant and resuspend each pellet in 100 mL of ice cold dH<sub>2</sub>O with gentle swirling. Combine the resuspensions into 2 centrifuge bottles</li>
<li>Spin 3: Harvest the cells by centrifugation at 1,000 × <i>g</i> for 20 min at 4°C. At this step chill two 50 mL conical tubes on ice</li>
<li>Decant the supernatant and resuspend each pellet in 40 mL of ice cold 10% (v/v) glycerol with gentle swirling. Transfer each suspension to a 50 mL conical tube</li>
<li>Spin 4: Harvest the cells by centrifugation at 1,000 × <i>g</i> for 20 min at 4°C. Carefully aspirate the supernatant with a sterile Pasteur pipette</li>
<li>Resuspend each pellet in 1 mL of ice cold 10% (v/v) glycerol by gently swirling</li>
<li>Aliquot 50 μL into sterile and cold 1.5 mL microfuge tubes and snap freeze in liquid nitrogen. Store aliquots at −80°C.</li>
</ol>
<div class="hazard-note">
<span class="hazard-note-title">Hazard Note:</span> Dispose of <i>E. coli</i> cultures as recommended by your local biosafety protocols</div>
<h3 id="sec1.2">Transform expression plasmids into <i>E. coli</i> BL21</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>In this step, the expression plasmids are transformed into BL21 <i>E. coli</i> by electroporation. Any transformation protocol can be used for the pcI<sup>ts,ind+</sup> -(NSP8) plasmid into BL21 <i>E. coli</i> but the protocol presented here is for electrocompetent cells prepared as described above.</p>
<ol start="17">
<li>Chill 0.1 mm electroporation cuvettes on ice for at least 15 min</li>
<li>Remove one tube of competent <i>E. coli</i> BL21 and one tube of electrocompetent <i>E. coli</i> BL21/pG-Tf2 made in the previous step from the −80°C freezer and place on ice for 10 min</li>
<li>Add plasmid DNA to appropriate cells
<ol type="a">
<li>Pipette 10 ng of pcI<sup>ts,ind+</sup>-(NSP8) in a volume less than 5 μL into the tube of <i>E. coli</i> BL21. Perform transformation following the protocol of the manufacturer of the competent cells.</li>
<li>Pipette at least 100 ng of pQE-(NSP12)-pcI<sup>ts,ind+</sup> -(NSP7-NSP8) in a volume less than 5 μL into the tube of electrocompetent <i>E. coli</i> BL21/pG-Tf2. The rest of the protocol is for transformation into the electrocompetent <i>E. coli</i> BL21/pG-Tf2.</li>
</ol>
</li>
<li>Transfer the cells/DNA mixture to the chilled electroporation cuvette and gently tap the cuvette on the lab bench to ensure the cells are in the gap in the bottom of the cuvette</li>
<li>Wipe off any water or ice that is on the metal parts of the electroporation cuvette with a Kimwipe (paper tissue). Insert the cuvette into the cuvette holder connected to the Gene Pulser and slide into the safety cover.</li>
<li>Electroporate at 1.8 kV, 25 μF by holding down the left red button and pressing the right red button until the machine beeps. Immediately release the buttons after you hear the beep</li>
<li>Check that the time constant given on the Gene Pulser is greater than 3.8.
<ol type="a">
<li>If the time constant is less than 3.8, try the transformation again or re-make the competent cells.</li>
</ol>
</li>
<li>Immediately add 950 μL of sterile SOC media (defined under Materials and Equipment) to the cuvette and pipette up and down to mix</li>
<li>Transfer the cells to a 1.5 mL microcentrifuge tube and do outgrowth at 30°C for 1 h.</li>
<li>Plate transformations on selective plates
<ol type="a">
<li>Plate <i>E. coli</i> BL21/pcI<sup>ts,ind+</sup> -(NSP8) on LB plates with 100 μg/mL ampicillin</li>
<li>Plate <i>E. coli</i> BL21/pG-Tf2/pQE-(NSP12)-pcI<sup>ts,ind+</sup>-(NSP7-NSP8) on LB plates with 30 μg/mL kanamycin and 20 μg/mL chloramphenicol</li>
</ol>
</li>
<li>Incubate plates at 30°C
<ol type="a">
<li>
<i>E. coli</i> BL21/pcI<sup>ts,ind+</sup>-(NSP8) will show colonies after 16–24 h</li>
<li>
<i>E. coli</i> BL21/pG-Tf2/pQE-(NSP12)-pcI<sup>ts,ind+</sup>-(NSP7-NSP8) will take longer to see colonies, 24–48 h.</li>
</ol>
</li>
</ol>
<div class="hazard-note">
<span class="hazard-note-title">Hazard Note:</span> Dispose of <i>E. coli</i> cultures as recommended by your local biosafety protocols</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>E. coli</i> BL21</td>
<td>New England Biolabs</td>
<td>Cat. # C253OH</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Chloramphenicol</td>
<td>Fisher Scientific</td>
<td>Cat. # BP904-100</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Fisher Scientific</td>
<td>Cat. # BP1760-25</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>Fisher Scientific</td>
<td>Cat. # BP906-5</td>
</tr>
<tr>
<td>Tetracycline</td>
<td>Fisher Scientific</td>
<td>Cat. # BP912-100</td>
</tr>
<tr>
<td>Isopropyl-beta-D-thiogalactoside (IPTG)</td>
<td>Gold Biotechnology</td>
<td>Cat. # L2481C</td>
</tr>
<tr>
<td>Nalidixic acid</td>
<td>Sigma-Aldrich</td>
<td>Cat. # N8878</td>
</tr>
<tr>
<td>Lysozyme</td>
<td>Sigma-Aldrich</td>
<td>Cat. # L4919</td>
</tr>
<tr>
<td>Phenylmethylsulfonyl fluoride (PMSF)</td>
<td>Sigma-Aldrich</td>
<td>Cat. # P7626</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pG-Tf2</td>
<td>Takara Biosciences</td>
<td>Cat. # 3340</td>
</tr>
<tr>
<td>pcI<sup>ts,ind+</sup>-(NSP8)</td>
<td><a href="#bib2">Dangerfield et al., 2020</a></td>
<td>Addgene, 160656</td>
</tr>
<tr>
<td>pQE-(NSP12)-pcI<sup>ts,ind+</sup> -(NSP7-NSP8)</td>
<td><a href="#bib2">Dangerfield et al., 2020</a></td>
<td>Addgene, 160540</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Gene Pulser II</td>
<td>Bio-Rad</td>
<td>Model # 165-2105</td>
</tr>
<tr>
<td>0.1 mm electroporation cuvettes</td>
<td>Bio-Rad</td>
<td>Cat. # 165-2089</td>
</tr>
<tr>
<td>Whatman 0.2 μm PES syringe filters</td>
<td>Millipore Sigma</td>
<td>Cat. # WHA67802502</td>
</tr>
<tr>
<td>Branson Sonifier 450</td>
<td>VWR Scientific</td>
<td>Model # S-450A</td>
</tr>
<tr>
<td>5 mL Q FF columns</td>
<td>Cytiva Life Sciences</td>
<td>Cat. # 17515601</td>
</tr>
<tr>
<td>5 mL SP-FF columns</td>
<td>Cytiva Life Sciences</td>
<td>Cat. # 17515701</td>
</tr>
<tr>
<td>5 mL HiTrap Blue-HP columns</td>
<td>Cytiva Life Sciences</td>
<td>Cat. # 17041301</td>
</tr>
<tr>
<td>Superdex 200 10/300 GL</td>
<td>Cytiva Life Sciences</td>
<td>Model # 17-5175-0</td>
</tr>
<tr>
<td>Q Sepharose-FF Bulk Resin</td>
<td>Cytiva Life Sciences</td>
<td>Cat. # 17051001</td>
</tr>
<tr>
<td>Heparin Sepharose 6 FF bulk resin</td>
<td>Cytiva Life Sciences</td>
<td>Cat. # 17099801</td>
</tr>
<tr>
<td>Amicon 10 kDa MWCO centrifugal concentrators</td>
<td>Millipore Sigma</td>
<td>Cat. # UFC901024</td>
</tr>
<tr>
<td>Amicon 30 kDa MWCO centrifugal concentrators</td>
<td>Millipore Sigma</td>
<td>Cat. # UFC903024</td>
</tr>
<tr>
<td>SpectraPor 1 kDa MWCO dialysis tubing</td>
<td>Spectrum Chemical</td>
<td>Cat. # 888-10966</td>
</tr>
<tr>
<td>NanoDrop One</td>
<td>Thermo Fisher</td>
<td>Cat. # ND-ONE-W</td>
</tr>
<tr>
<td>Roche cOmplete, EDTA-free protease inhibitor tablets</td>
<td>Millipore Sigma</td>
<td>Cat. # 11873580001</td>
</tr>
<tr>
<td>AKTA Pure FPLC</td>
<td>Cytiva Life Sciences</td>
<td>Cat. # 29018226</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th>Solution</th>
<th>Contents</th>
<th>Final volume</th>
<th>Storage</th>
</tr>
</thead>
<tbody>
<tr>
<td>LB media</td>
<td>0.5% (w/v) yeast extract, 1% (w/v) Bacto Tryptone, 1% (w/v) NaCl, appropriate antibiotic after cooling</td>
<td>1 L</td>
<td>25°C, 1 week</td>
</tr>
<tr>
<td>LB + Amp. + agar plates</td>
<td>0.5% (w/v) yeast extract, 1% (w/v) Tryptone, 1% (w/v) NaCl, 1.5% (w/v) Bacto agar, 100 μg/mL ampicillin after cooling</td>
<td>500 mL (~12 plates)</td>
<td>4°C, 1 month</td>
</tr>
<tr>
<td>LB + Chl. + agar plates</td>
<td>0.5% (w/v) yeast extract, 1% (w/v) Tryptone, 1% (w/v) NaCl, 1.5% (w/v) Bacto agar, 20 μg/mL chloramphenicol after cooling</td>
<td>500 mL (~12 plates)</td>
<td>4°C, 1 month</td>
</tr>
<tr>
<td>LB + Chl. + Kan. + agar plates</td>
<td>0.5% (w/v) yeast extract, 1% (w/v) Tryptone, 1% (w/v) NaCl, 1.5% (w/v) Bacto agar; 20 μg/mL chloramphenicol and 30 μg/mL kanamycin after cooling</td>
<td>500 mL (~12 plates)</td>
<td>4°C, 1 month</td>
</tr>
<tr>
<td>2× YT media</td>
<td>1% (w/v) yeast extract, 1.6% (w/v) Bacto Tryptone, 0.5% (w/v) NaCl</td>
<td>10 mL</td>
<td>25°C, 1 week</td>
</tr>
<tr>
<td>SOC media</td>
<td>0.5% (w/v) yeast extract, 2% (w/v) Bacto Tryptone, 0.05% (w/v) NaCl, 20 mM glucose, 10 mM MgCl<sub>2</sub>, 2.5 mM KCl</td>
<td>10 mL</td>
<td>25°C, 1 week</td>
</tr>
<tr>
<td>Terrific broth</td>
<td>2.4% (w/v) yeast extract, 2% (w/v) Bacto Tryptone, 0.4% (v/v) glycerol, 17.2 mM KH<sub>2</sub>PO<sub>4</sub>, 72 mM K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>4 L</td>
<td>25°C, 1 week</td>
</tr>
<tr>
<td>10% glycerol</td>
<td>10% (v/v) glycerol</td>
<td>150 mL</td>
<td>4°C, 1 month</td>
</tr>
<tr>
<td>6× SDS sample buffer</td>
<td>12% (w/v) SDS, 0.06% (w/v) bromophenol blue, 47% (v/v) glycerol, 42 mM Tris-HCl pH 6.8, 9.3% (w/v) DTT</td>
<td>5 mL</td>
<td>−20°C, 1 month</td>
</tr>
<tr>
<td>100 mg/mL ampicillin</td>
<td>10% (w/v) ampicillin, solvent: dH<sub>2</sub>O</td>
<td>3 mL</td>
<td>−20°C, 2 weeks</td>
</tr>
<tr>
<td>30 mg/mL kanamycin</td>
<td>3% (w/v) kanamycin sulfate, solvent: dH<sub>2</sub>O</td>
<td>2 mL</td>
<td>−20°C, 2 weeks</td>
</tr>
<tr>
<td>20 mg/mL chloramphenicol</td>
<td>2% (w/v) chloramphenicol, Solvent: Ethanol</td>
<td>3 mL</td>
<td>−20°C, 2 weeks</td>
</tr>
<tr>
<td>1 M IPTG</td>
<td>23.8% (w/v) IPTG, solvent: dH<sub>2</sub>O</td>
<td>1 mL</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>25 mg/mL nalidixic acid</td>
<td>2.5% (w/v) nalidixic acid, solvent: 0.3 M NaOH</td>
<td>3 mL</td>
<td>make fresh</td>
</tr>
<tr>
<td>10 μg/mL tetracycline</td>
<td>0.001% (w/v) tetracycline, solvent: dH<sub>2</sub>O</td>
<td>1 mL</td>
<td>make fresh</td>
</tr>
<tr>
<td>1 M DTT</td>
<td>15.4% (w/v) DTT, solvent: dH<sub>2</sub>O</td>
<td>50 mL</td>
<td>−20°C, 1 month</td>
</tr>
<tr>
<td>30 mg/mL lysozyme</td>
<td>3% (w/v) lysozyme, solvent: lysis buffer</td>
<td>3 mL</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>800 mM PMSF</td>
<td>13.9% (w/v) PMSF, solvent: acetone</td>
<td>3 mL</td>
<td>make fresh daily</td>
</tr>
<tr>
<td>4% Sodium deoxycholate</td>
<td>4% (w/v) sodium deoxycholate, solvent: dH<sub>2</sub>O</td>
<td>5 mL</td>
<td>4°C, 1 month</td>
</tr>
<tr>
<td>Expression test sample Buffer</td>
<td>66% (v/v) 2× SDS sample buffer, 33% (v/v) 10% glycerol</td>
<td>5 mL</td>
<td>−20°C, 1 month</td>
</tr>
<tr>
<td>Expression test buffer A</td>
<td>50 mM Tris-HCl pH 8, 2 mM EDTA</td>
<td>10 mL</td>
<td>4°C, 1 month</td>
</tr>
<tr>
<td>Expression test buffer B</td>
<td>50 mM Tris-HCl pH 8, 2 mM EDTA, 500 mM NaCl</td>
<td>10 mL</td>
<td>4°C, 1 month</td>
</tr>
<tr>
<td>Lysis buffer-1</td>
<td>50 mM Tris-HCl pH 8.0, 2.5 mM EDTA, 150 mM NaCl, 1 mM DTT</td>
<td>100 mL</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Lysis buffer-2</td>
<td>50 mM Tris-HCl pH 8.0, 2 mM EDTA, 250 mM NaCl, 10% glycerol, 5 mM DTT</td>
<td>100 mL</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Dilution buffer-1</td>
<td>50 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 10% glycerol, 1 mM DTT</td>
<td>3 L</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Dilution buffer-2</td>
<td>50 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 10% glycerol, 5 mM DTT</td>
<td>3 L</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Buffer A-1</td>
<td>50 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 50 mM NaCl, 10% glycerol, 1 mM DTT</td>
<td>4 L</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Buffer A-2</td>
<td>50 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 50 mM NaCl, 10% glycerol, 2 mM DTT</td>
<td>4 L</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Buffer B-1</td>
<td>50 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 1 M NaCl, 10% glycerol, 1 mM DTT</td>
<td>3 L</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Buffer B-2</td>
<td>50 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 1 M NaCl, 10% glycerol, 2 mM DTT</td>
<td>3 L</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Storage buffer-1</td>
<td>50 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 20 mM KCl, 50% glycerol, 1 mM DTT)</td>
<td>2 L</td>
<td>4°C, make fresh</td>
</tr>
<tr>
<td>Storage buffer-2</td>
<td>50 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 50 mM NaCl, 50% glycerol, 1 mM DTT</td>
<td>2 L</td>
<td>4°C, make fresh</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗ The pH of all buffers was measured at 25°C</p>
<div class="critical"><span class="critical-title">Critical:</span></div>
<ul>
<li>EDTA – Harmful if inhaled. Eye irritant. Wear goggles and weigh powder in fume hood.</li>
<li>SDS – Harmful if swallowed or inhaled. Skin and eye irritant. Wear protective gloves and weigh chemical in a fume hood.</li>
<li>Ampicillin – Harmful if inhaled. Potential skin irritant. Wear gloves.</li>
<li>Kanamycin – Harmful if inhaled or ingested. Potential skin irritant. Wear gloves.</li>
<li>Chloramphenicol – Harmful if inhaled. Potential skin irritant. Wear gloves.</li>
<li>IPTG – Harmful if swallowed or inhaled. Potential skin irritant. Wear gloves.</li>
<li>Tetracycline – Harmful if swallowed or inhaled. Potential skin irritant. Wear gloves.</li>
<li>Nalidixic acid – Harmful if swallowed or inhaled. Potential skin irritant. Wear gloves.</li>
<li>DTT – Harmful if swallowed or inhaled. Potential skin irritant. Wear gloves.</li>
<li>PMSF – Very toxic. Harmful if swallowed, inhaled, or applied to skin. Wear gloves, weigh powder in fume hood.</li>
<li>Sodium deoxycholate – Harmful if swallowed or inhaled.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Expression of SARS-CoV-2 nsp8 in <i>E. coli</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>In this step, a 2 L culture of <i>E. coli</i> harboring the plasmid pcI<sup>ts,ind+</sup>-(NSP8) is grown to a high density. Protein expression is then induced by shifting the temperature from 30°C to 42°C for 20 min then to 38°C for 3 h. Finally, the cells are harvested and frozen for later use in protein purification.</p>
<ol>
<li>Inoculate a single colony from the plate containing <i>E. coli</i> BL21/ pcI<sup>ts,ind+</sup>(NSP8) into 8 mL of Terrific broth with 100 μg/mL ampicillin. Incubate for 16 h with shaking at 30°C, 200 rpm.</li>
<li>The following morning pellet the cells by centrifugation at 5,000 × <i>g</i> for 10 min.</li>
<li>Decant the supernatant and resuspend the cells in 6 mL of fresh Terrific broth with 100 μg/mL ampicillin.</li>
<li>Inoculate each of 2 × 1 L of Terrific broth with 100 μg/mL ampicillin, split into 2 × 2.8 L baffle-bottomed flasks, with 3 mL of the resuspended culture from step 3. Incubate the culture at 30°C with shaking at 250 rpm.</li>
<li>Measure the OD<sub>600</sub> of the culture every 30 min until the OD<sub>600</sub> reaches 2.</li>
<li>When the OD<sub>600</sub> reaches 2, remove a “pre-induction” sample by pelleting (10,000 × <i>g</i>, 5 min, 20°C–25°C) a volume of cells equal to 1 mL at an OD<sub>600</sub> of 6. Aspirate the supernatant then store the pellet at −80°C.</li>
<li>Induce expression by moving the cultures to an incubator preheated to 42°C for 20 min and incubate with shaking at 250 rpm.</li>
<li>Move the cultures to an incubator preheated to 38°C and incubate with shaking at 250 rpm for 3 h.</li>
<li>Measure the OD<sub>600</sub> of the cultures. Take a post-induction sample by pelleting a volume of cells equal to 1 mL at an OD<sub>600</sub> of 6. Decant then aspirate the remaining supernatant. Store the pellet at −80°C.</li>
<li>Harvest the cells by centrifugation at 6,000 × <i>g</i> for 20 min at 4°C Decant the supernatant then transfer the pellet to a pre-weighed 50 mL conical tube with a spatula.</li>
<li>Determine the mass of the tube plus pellet then subtract the mass of the empty tube to get the yield of cell pellet. Record the mass on the tube as well as in your notes. Typical yields are 15–20 <i>g</i> of cell pellet per 2 L of growth media.</li>
<li>Store the pellet at −80°C. The pellet is stable for at least one month at −80°C but likely is stable for longer than we have tested.</li>
</ol>
<div class="hazard-note">
<span class="hazard-note-title">Hazard Note:</span> Dispose of <i>E. coli</i> cultures as recommended by your local biosafety protocols</div>
<h3 id="sec3.2">Expression of SARS-CoV-2 nsp12/7/8 in <i>E. coli</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5 days</div>
<p>In this step, <i>E coli</i> harboring the chaperone plasmid pG-Tf2 and the RdRp complex co-expression plasmid pQE-(NSP12)-pcI<sup>ts,ind+</sup>-(NSP7-NSP8) are grown to a high density before inducing expression for 16 h and harvesting the cells by centrifugation.</p>
<ol start="13">
<li>Inoculate a single colony from the plate with <i>E. coli</i> BL21/pG-Tf2/pQE-(NSP12)-pcI<sup>ts,ind+</sup> -(NSP7-NSP8) into an 8 mL starter culture of Terrific broth with 30 μg/mL kanamycin and 20 μg/mL chloramphenicol. Incubate at 30°C with shaking at 200 rpm for 16 h.</li>
<li>The following morning inoculate the starter culture into 1 L of Terrific broth with 30 μg/mL kanamycin and 20 μg/mL chloramphenicol in a 2.8 L baffle-bottomed flask and incubate with shaking at 30°C, 250 rpm.</li>
<li>Periodically check the OD<sub>600</sub> of the culture. When the OD<sub>600</sub> reaches 2, take a pre-induction sample by harvesting a volume of cells equal to 1 mL at an OD<sub>600</sub> of 6 and storing the pellet at −80°C.</li>
<li>Add tetracycline from a 10 μg/mL stock to a final concentration of 10 ng/mL and move the culture to an incubator pre-cooled to 16°C.</li>
<li>Incubate with shaking at 16°C, 250 rpm for 20 min.</li>
<li>Add IPTG to 0.5 mM from a 1 M stock solution. Add nalidixic acid to 50 μg/mL from a 25 mg/mL stock.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make 25 mg/mL nalidixic acid stock fresh daily by dissolving the powder into 0.3 M NaOH.</div>
<ol start="19">
<li>Express for 16 h at 16°C with shaking at 250 rpm.</li>
<li>Measure the OD<sub>600</sub>. Pellet a volume of cells equal to 1 mL at an OD<sub>600</sub> of 6 and store the pellet at −80°C.</li>
<li>Harvest the cells by centrifugation at 6,000 × <i>g</i> for 20 min at 4°C.</li>
<li>Decant the supernatant and transfer the pellet to a pre-weighed 50 mL conical tube with a spatula. Determine the mass of the pellet then store at −80°C.</li>
</ol>
<div class="hazard-note">
<span class="hazard-note-title">Hazard Note:</span> Dispose of <i>E. coli</i> cultures as recommended by your local biosafety protocols</div>
<h3 id="sec3.3">Testing expression levels from large-scale <i>E. coli</i> expressions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5 day</div>
<p>In this step, pre- and post-induction samples collected during protein expression are lysed and separated by SDS-PAGE to determine the level of overexpression and solubility of the proteins of interest before pursuing further purification from the large pellets. The method described here is adapted from (<a href="#bib1">Brandis and Johnson, 2009</a>).</p>
<ol start="23">
<li>For each sample to be analyzed, set up a series of 3 tubes for each time point, each containing 45 μL of Expression Test Sample Buffer. Label the tubes in each set of three as follows:
<ol type="a">
<li>TCP = Total Cell Protein</li>
<li>SOL = Soluble Protein</li>
<li>INS = Insoluble Protein</li>
</ol>
</li>
<li>Thaw pellets collected pre- and post-induction from the large-scale expressions above on ice.</li>
<li>Add 200 μL of Expression Test Buffer A containing 0.3 mg/mL Lysozyme to each pellet. Vortex until the pellets have been uniformly resuspended.</li>
<li>Incubate samples at 37°C for 10 min then place on ice. The solutions should be very viscous due to the chromosomal DNA that is released upon lysis.</li>
<li>Sonicate with the micro-tip (20% duty cycle, 0% output) for 30 – 40 bursts, ensuring the tip is always under the level of the solution in the tube. After sonication, the viscosity of the sample should be much lower.</li>
<li>Add 22.4 μL of 5 M NaCl to each sample. Seal the tubes and vortex.</li>
<li>Remove 15 μL of the sonicate and transfer it to the appropriate “TCP” sample tube, containing 45 μL of Expression Test Sample Buffer.</li>
<li>Spin the remainder of the sonicate in a microcentrifuge tube at 16,000 × <i>g</i> in a tabletop microcentrifuge for 10 min at 20°C–25°C.</li>
<li>Remove 15 μL of the supernatant to the appropriate “SOL” sample tube already containing Expression Test Sample Buffer.</li>
<li>Aspirate off the remaining supernatant.</li>
<li>Add 400 μL of Expression Test Buffer B and resuspend the pellet using the same sonication procedure as in step 27. This is a wash step to remove any residual soluble protein from the pellet and from the walls of the tube.</li>
<li>Spin the suspension at 16,000 × <i>g</i> for 10 min at 20°C–25°C in a tabletop microcentrifuge.</li>
<li>Aspirate and discard the supernatant.</li>
<li>Add 192 μL of Expression Test Buffer B to each sample and resuspend the pellets by sonication as in step 27. Usually only 10 – 12 bursts are needed to resuspend the pellet.</li>
<li>Remove 15 μL of the resuspended pellet solution to the appropriate “INS” tube with Expression Test Sample Buffer.</li>
<li>Heat all samples at 80°C–85°C for 5 min prior to loading 10 – 15 μL per lane for SDS-PAGE.</li>
<li>Run SDS-PAGE gels according to the manufacturers protocol, then stain with Coomassie brilliant blue. We recommend 15% acrylamide gels for resolving overexpressed nsp7 (9.4 kDa) and nsp8 (22 kDa), and 6% acrylamide gels for resolving overexpressed nsp12 (107 kDa).</li>
</ol>
<h3 id="sec3.4">Purification of SARS-CoV-2 nsp8</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>In this step, the frozen pellet obtained from the bacterial expression of nsp8 is lysed, and nsp8 is purified by a number of chromatography steps.</p>
<ol start="40">
<li>Thaw the bacterial pellet from the expression above, on ice.</li>
<li>Resuspend the cell pellet in Lysis Buffer-1 at 5 mL buffer per gram of <i>E. coli.</i>
</li>
<li>Add 800 mM phenylmethylsulfonyl fluoride (PMSF) to a final concentration of 10 mM. Add 30 mg/mL Lysozyme to a final concentration of 0.3 mg/mL and stir the lysate at 25°C for 15 min on a stir plate.</li>
<li>Sonicate the lysate (20% duty, power 5) on ice for 20 min with constant stirring from a stir bar. During sonication, check the sample every 3–5 min to ensure efficient mixing and ice cooling as the sonicator produces heat.</li>
<li>Add 4% sodium deoxycholate to a final concentration of 0.1%. Add 5 M NaCl to a final concentration of 0.5 M. Stir on ice for 30 min.</li>
<li>Centrifuge the lysate at 75,000 × <i>g</i> for 45 min at 4°C.</li>
<li>Filter the supernatants through a 0.2 μm syringe filter.</li>
<li>Measure the conductivity of Buffer A-1 containing 10% Buffer B-1. Dilute the filtered sample with Dilution Buffer-1 until the conductivity of the sample is equal to the conductivity of Buffer A-1 with 10% Buffer B-1 (approximately a 5-fold dilution). Remove an aliquot of the diluted sample into 6× SDS Sample Buffer tor later analysis by SDS-PAGE.</li>
<li>HiTrap Q-FF / HiTrap SP-FF Chromatography: Flow the lysate on to two sequential 5 mL HiTrap Q FF column with the outlet connected to two sequential 5 mL HiTrap SP-FF column, equilibrated in Buffer A-1, at 10 mL/min. Monitor absorbance at 280 nm. Collect the flow through in a separate, labeled container and store at 4°C as a backup.</li>
<li>Wash with 10% Buffer B-1 until the UV <sub>280</sub> reaches baseline (approximately 100 mL). Collect the eluate in a separate container, save a sample for SDS-PAGE analysis, and store at 4°C.</li>
<li>Disconnect the Q columns, leaving the two SP columns connected to the FPLC.</li>
<li>Elute proteins bound to the SP column with a 200 mL gradient from 10%–80% Buffer B at 5 mL/min. Collect 5 mL fractions and store fractions on ice.</li>
<li>Remove aliquots of various fractions with a large A<sub>280</sub> peak in the chromatogram into 6× SDS Sample Buffer. Heat samples at 85°C for 5 min. Load samples onto 15% SDS-PAGE gels, then run the gel until the bromophenol blue reaches the bottom of the gel following the manufacturers protocol. Stain and destain the gel with Coomassie brilliant blue.</li>
<li>Pool fractions containing nsp8.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> At this point, fractions can be left at 4°C for up to 16 h.</div>
<ol start="54">
<li>HiTrap Blue-HP Chromatography: Dilute the pooled fractions with Dilution Buffer-1 until the conductivity of the sample is equal to the conductivity of Buffer A-1.</li>
<li>Load the sample onto a 5 mL HiTrap Blue-HP column, equilibrated in Buffer A-1 at 10 mL/min. Collect the flow through in a separate, labeled container, save a sample for SDS-PAGE, and store at 4°C.</li>
<li>Wash the column with 10% Buffer B at 5 mL/min until the UV<sub>280</sub> reaches baseline, collecting the eluate in a separate, labeled container. Save a sample for SDS-PAGE and store at 4°C.</li>
<li>Elute bound proteins with a 50 mL linear gradient from 10%–80% Buffer B at 5 mL/min, collecting 2 mL fractions.</li>
<li>Analyze samples from the Blue column by SDS-PAGE as described above.</li>
<li>Pool fractions containing nsp8.</li>
<li>Concentrate the pooled fractions by ultrafiltration with Amicon Ultra centrifugal concentrators with a 10 kDa molecular weight cutoff (MWCO) at 4°C until the sample volume is less than 1 mL, following the manufacturer’s instructions.</li>
<li>Size Exclusion Chromatography: Inject the sample onto the Superdex 200 10/300 GL column, equilibrated in 10% Buffer B with the sample loop on the HPLC.</li>
<li>Elute proteins with 10% Buffer B-1, collecting 1 mL fractions.</li>
<li>Analyze fractions by SDS-PAGE as described above.</li>
<li>Pool fractions containing nsp8.</li>
<li>Concentration / Storage: Determine the concentration of nsp8 by absorbance at 280 nm using the extinction coefficient ε<sub>280</sub>: 19,940 M <sup>−1</sup> cm<sup>−1</sup> (<a href="#bib2">Dangerfield et al., 2020</a>).</li>
<li>If necessary, concentrate the sample with Amicon Ultra centrifugal concentrators (10 kDa MWCO) following the manufacturer’s protocol . We concentrated the protein to 300 μM at this step without precipitation problems.</li>
<li>Dialyze the sample into Storage Buffer-1 with 1 kDa MWCO dialysis tubing at 4°C with 2 × 500 mL × 1 h buffer changes.</li>
<li>Remove the sample from the dialysis tubing and determine the concentration of nsp8 by absorbance at 280 nm using the extinction coefficient above.</li>
<li>Divide the purified protein into aliquots, flash freeze into liquid nitrogen and store at −80°C until use.</li>
</ol>
<h3 id="sec3.5">Purification of nsp12/7/8</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>In this step, pellets of <i>E. coli</i> harboring overexpressed nsp12/7/8 complex are lysed and subsequently purified over several chromatography steps.</p>
<ol start="70">
<li>Thaw cells from previous expression and resuspend in Lysis Buffer-2 at 5 mL buffer per gram <i>of E. coli</i> cells.</li>
<li>For each 50 mL of lysis buffer used above in step 70, dissolve one EDTA-free protease inhibitor tablet into 2 mL of Lysis Buffer-2 and add to the sample. Add 800 mM PMSF to a final concentration of 10 mM. Add 4% sodium deoxycholate to a final concentration of 0.1%. Add 30 mg/mL lysozyme to a final concentration of 0.3 mg/mL. Stir at 25°C for 15 min on a stir plate.</li>
<li>Sonicate the lysate on ice for 20 min with stirring (20% duty cycle, power 5). Check ice cooling every 5 min.</li>
<li>Clarify the lysate by centrifugation at 75,000 × <i>g</i> at 4°C for 1 h.</li>
<li>Filter the supernatant through at 0.2 μm syringe filter.</li>
<li>Measure the conductivity of Buffer A-2. Measure the conductivity of the sample and dilute with Dilution Buffer-2 until the conductivity is equal to the conductivity of Buffer A-2 (~ 6 fold dilution). Save a sample for SDS-PAGE analysis.</li>
<li>Q Sepharose-FF Chromatography: Load the sample at 5 mL/min onto a 30 mL Q Sepharose-FF column equilibrated in Buffer A-2, collecting the flow through in a separate, labeled container. Store the flow through at 4°C.</li>
<li>Wash with Buffer A until the UV reaches baseline, collecting the eluate in a separate container. Store at 4°C.</li>
<li>Elute bound proteins with a 300 mL linear gradient from 0%–100% Buffer B-2, collecting 10 mL fractions.</li>
<li>Analyze fractions by SDS-PAGE.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> At this point, fractions can be left at 4°C for up to 16 h.</div>
<ol start="80">
<li>Pool fractions containing nsp12/7/8.</li>
<li>HiTrap Blue-HP Chromatography: Filter the pooled fractions with a 0.2 μm syringe filter.</li>
<li>Dilute the pooled fractions with Dilution Buffer-2 until the conductivity is equal to the conductivity of Buffer A-2.</li>
<li>Load the sample onto the column at 5 mL/min and collect the flow through, containing nsp12/7/8.</li>
<li>Heparin Sepharose 6-FF Chromatography: Load the flow through from the blue column directly on to a 5 mL Heparin Sepharose 6-FF column equilibrated in Buffer A-2 at 5 mL/min and collect the flow through in a separate labeled container. Store at 4°C.</li>
<li>Wash with Buffer A-2 until the UV reaches baseline, collecting the eluate in a separate labeled container. Store at 4°C.</li>
<li>Elute bound proteins with a 50 mL linear gradient of 0%–100% Buffer B-2 at 2.5 mL/min, collecting 5 mL fractions.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> At this point, fractions can be left at 4°C for up to 16 h.</div>
<ol start="87">
<li>Analyze results of column by SDS-PAGE as described above. Pool fractions containing nsp12/7/8 complex.</li>
<li>HiTrap SP-FF Chromatography: Filter the pooled fractions with a 0.2 μm syringe filter. Dilute the pooled fractions with Dilution Buffer-2 until the conductivity is equal to the conductivity of Buffer A-1.</li>
<li>Load the diluted sample onto a 5 mL HiTrap SP-FF column at 5 mL/min, equilibrated in Buffer A-2, collecting the flow through in a container on ice.</li>
<li>Concentrate the flow through from the SP column with Amicon Ultra centrifugal concentrators (30 kDa MWCO) following the manufacturers protocol until the volume is less than 0.5 mL.</li>
<li>Size Exclusion Chromatography: Inject the concentrated sample onto a Superdex 200 10/300 GL column, equilibrated in Buffer A-2 with the sample loop on the FPLC at 0.75 mL/min.</li>
<li>Separate the complexes in the same buffer, collecting 1.5 mL fractions.</li>
<li>Analyze fractions by SDS-PAGE. Pool fractions containing nsp12/7/8 complex.</li>
<li>Concentration / Storage: Concentrate the pooled fractions using Amicon Ultra (30 kDa MWCO) centrifugal concentrators until the volume is approximately 1 mL, following the manufacturers protocols.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We have concentrated the nsp12/7/8 complex up to 165 μM without precipitation, although the maximum concentration obtainable without precipitation is not known.</div>
<ol start="95">
<li>Dialyze the concentrated sample into Storage Buffer-2 with 1 kDa MWCO dialysis tubing at 4°C with two 500 mL buffer changes, 1 h of equilibration for each.</li>
<li>Remove sample from dialysis tubing, determine the final volume, and measure the concentration by absorbance at 280 nm using the extinction coefficient ε <sub>280</sub>: 181,300 M<sup>−1</sup> cm <sup>−1</sup> calculated from the amino acid sequence and assuming one nsp12, one nsp7, and two nsp8 polypeptides per complex.</li>
<li>Flash freeze aliquots in liquid nitrogen and store at −80°C.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The expected outcomes are shown in <a href="#fig1">Figures 1</a>, <a href="#fig2">2</a>, and <a href="#fig3">3</a> .</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/496-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Expression and purification of nsp8</div>
<p>The soluble fraction of the <i>E. coli</i> lyase is shown in the left most lane, followed by a sample taken before loading onto the Q/SP columns, and blue column. The lane labeled MW ladder contains PAGE Ruler Plus molecular weight ladder, with corresponding sizes as follows from small to large: 10 kDa (green), 15 kDa (blue), 25 kDa (red), 35 kDa (blue), 55 kDa (blue), 70 kDa (red), 100 kDa (blue). Lanes to the right of the molecular weight ladder are different dilutions of the final purified protein. Figure reprinted with permission from <a href="#bib2">Dangerfield et al., 2020</a>.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/496-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Expression test for nsp12/7/8</div>
<p>Samples are from an expression test from a large-scale expression of the nsp12/7/8 complex. Molecular weight markers are in the leftmost lane of each gel, with their corresponding molecular weights (in kDa) given to the left of each gel.</p>
<p>(A) 15% SDS-PAGE gel of pre- and post-induction samples. Samples corresponding to the total cell protein (TCP), soluble protein (Sol.) and insoluble protein (Ins.) are shown both before induction (Pre-induction), and after induction (post-induction).</p>
<p>(B) 6% SDS-PAGE gel of pre-and post-induction samples. Samples are labeled as in (A), but higher molecular weight bands can be seen. Figure reprinted with permission from <a href="#bib2">Dangerfield et al. (2020)</a>.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/496-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Gel from nsp12/7/8 purification</div>
<p>The leftmost lane contains a molecular weight marker, with corresponding sizes in kDa given to the left of the gel. Two dilutions of the final purified nsp12/7/8 complex are shown in the right two lanes. Bands of interest are labeled to the right of the gel. Figure reprinted with permission from <a href="#bib2">Dangerfield et al. (2020)</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Calculating the final yield in mg of each protein preparation based on the volume and molar concentration measured by absorbance:</p>
<p>To determine the yield of each protein preparation in mg, use the following equation:</p>
<p class="math-ml"><math> <mrow> <mtext>Yield of Protein </mtext> <mrow> <mo>(</mo> <mtext>mg</mtext> <mo>)</mo> </mrow> <mo>=</mo> <mtext>Volume </mtext> <mrow> <mo>(</mo> <mtext>mL</mtext> <mo>)</mo> </mrow> <mo>×</mo> <mrow> <mo>[</mo> <mtext>Protein</mtext> <mo>]</mo> </mrow> <mrow> <mo>(</mo> <mtext>μM</mtext> <mo>)</mo> </mrow> <mo>×</mo> <mtext>Molecular Weight of Protein</mtext> <mrow> <mo>(</mo> <mtext>s</mtext> <mo>)</mo> </mrow> <mrow> <mo>(</mo> <mfrac> <mtext>g</mtext> <mtext>mol</mtext> </mfrac> <mo>)</mo> </mrow> <mo>×</mo> <msup> <mn>10</mn> <mrow> <mo>−</mo> <mn>6</mn> </mrow> </msup> </mrow> </math></p>
<p>For the nsp8 preparation, the molecular weight is 22,000 g/mol and the typical yield is 25 mg per liter of culture. For the nsp12/7/8 complex preparation, the combined molecular weight of the complex (assuming 2 molecules of nsp8 per molecule of nsp7 and nsp12) is 160,000 g/mol and the typical yield is 7 mg per liter of culture.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While our bacterial expression protocol is significantly less time consuming that baculovirus protein expression, our purification protocols require many more steps than traditional purification of tagged proteins. Also even after multiple chromatography steps, there are still some proteolysis products, visible by SDS-PAGE, in the preparations of both enzymes. Both protein preparations are free from contaminating nuclease activity. Single turnover kinetic analysis shows that the enzyme is greater than 60% active. The purification procedures can likely be further optimized to eliminate these contaminating bands.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No colonies after transforming expression plasmids into <i>E. coli</i> (<a href="#before-you-begin">before you begin</a>, step 27).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Ensure <i>E. coli</i> cells are kept as cold as possible after the growth phase (<a href="#before-you-begin">before you begin</a>, steps 7–16) when preparing competent cells. Add more plasmid DNA before electroporation of competent cells (<a href="#before-you-begin">before you begin</a>, step 19).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Inconsistent band intensity / same bands in soluble fraction and insoluble fraction (<a href="#step-by-step-method-details">step-by-step method details</a>, step 39)</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Ensure all media is aspirated from cell pellets harvested for expression test before freezing samples (<a href="#step-by-step-method-details">step-by-step method details</a>, steps 6, 9, 15, 20). Ensure pellets are completely resuspended during lysis steps (<a href="#step-by-step-method-details">step-by-step method details</a>, steps 25–27). Ensure all buffer is aspirated after removing the soluble fraction sample (<a href="#step-by-step-method-details">Step-by-step method details</a>, step 32). Perform an additional “wash step” before removing insoluble fraction sample ( <a href="#step-by-step-method-details">step-by-step method details</a>, repeat steps 33–35). Load the same volume of each sample when analyzing results by SDS-PAGE (<a href="#step-by-step-method-details">step-by-step method details</a>, step 38).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>No/low expression of nsp8 (<a href="#step-by-step-method-details">step-by-step method details</a>, step 39) .</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Always use a fresh transformation of pcI<sup>ts,ind+</sup>-(NSP8) (<a href="#before-you-begin">before you begin</a>, steps 17–27). Ensure ampicillin plates are made fresh as ampicillin is prone to degradation. Preheat incubators to 30°C, 42°C, and 38°C to ensure efficient heat induction (<a href="#step-by-step-method-details">step-by-step method details</a>, steps 4, 7, 8). The 42°C step can be performed in a preheated water bath for better heat transfer to the cells ( <a href="#step-by-step-method-details">step-by-step method details</a>, step 7).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>No/low expression of nsp12/7/8 complex (<a href="#step-by-step-method-details">step-by-step method details</a>, step 39).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Always use a fresh transformation of pQE-(NSP12)-pcI <sup>ts,ind+</sup>-(NSP7-NSP8) into BL21/pG-Tf2 cells ( <a href="#before-you-begin">before you begin</a>, steps 17–27). Make LB/Chl./Kan. plates fresh to prevent satellite colonies. Ensure the nalidixic acid stock is made fresh daily. Ensure antibiotics and IPTG solutions are prepared as fresh as possible.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>RdRp complex proteins do not bind to a column ( <a href="#step-by-step-method-details">step-by-step method details</a>, steps 48, 55, 76, 83, 84, 89).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Add more of the appropriate Dilution Buffer ( <a href="#step-by-step-method-details">step-by-step method details</a>, steps 47, 54, 75, 82, 88) and check to the conductivity of the sample to ensure it is below the conductivity of the appropriate Buffer A. Re-make FPLC buffers, taking care to add the correct concentration of NaCl.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Brandis, J.W. and Johnson, K.A. (2009). High-cell density shake-flask expression and rapid purification of the large fragment of Thermus aquaticus DNA polymerase I using a new chemically and temperature inducible expression plasmid in <i>Escherichia coli</i>. Protein Expr. Purifi. <i>63</i>, 120-127. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00064-2/sref1">View at publisher</a></p>
<p id="bib2">Dangerfield, T.L., Huang, N.Z., and Johnson, K.A. (2020). Remdesivir is effective in combating COVID-19 because it is a better substrate than ATP for the viral RNA-dependent RNA polymerase. iScience <i>23</i>, 101849. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00064-2/sref2">View at publisher</a></p>
<p id="bib3">Gordon, C.J., Tchesnokov, E.P., Woolner, E., Perry, J.K., Feng, J.Y., Porter, D.P., and Götte, M. (2020). Remdesivir is a direct-acting antiviral that inhibits RNA-dependent RNA polymerase from severe acute respiratory syndrome coronavirus 2 with high potency. J. Biol. Chem. <i>295</i>, 6785-6797. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00064-2/sref3">View at publisher</a></p>
<p id="bib4">Hillen, H.S., Kokic, G., Farnung, L., Dienemann, C., Tegunov, D., and Cramer, P. (2020). Structure of replicating SARS-CoV-2 polymerase. Nature <i>584</i>, 154-156. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00064-2/sref4">View at publisher</a></p>
<p id="bib5">Yin, W., Mao, C., Luan, X., Shen, D.-D., Shen, Q., Su, H., Wang, X., Zhou, F., Zhao, W., Gao, M., et al. (2020). Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir. Science <i>368</i>, 1499. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00064-2/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Requests for further information should be directed to and will be fulfilled by the lead contact, Kenneth A. Johnson ( <a href="mailto:kajohnson@utexas.edu">kajohnson@utexas.edu</a>).</p>
<h4>Materials Availability</h4>
<p>All materials in this study can be obtained from sources given in the <a href="#key-resources-table">key resources table</a>.</p>
<h4>Data and Code Availability</h4>
<p>The published article includes all datasets generated or analyzed during this study</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants from <a href="https://doi.org/10.13039/100000060">NIAID</a> ( 1R01AI110577 to K.A.J.),<a href="https://doi.org/10.13039/100000002">NIH (R01 GM114223</a> 114223 to K.A.J.), and the <a href="https://doi.org/10.13039/100000928">Welch Foundation</a> (F-1604 to K.A.J.).</p>
<h3>Author Contributions</h3>
<p>Conceptualization, K.A.J., T.L.D., and N.Z.H.; Methodology, K.A.J., T.L.D., and N.Z.H.; Formal Analysis, K.A.J. and T.L.D.; Investigation, T.L.D. and N.Z.H.; Resources, K.A.J.; Writing—Original Draft, T.L.D.; Writing—Review and Editing, K.A.J., T.L.D., and N.Z.H.; Visualization, T.L.D.; Supervision, K.A.J.; Project Administration, K.A.J.; Funding Acquisition, K.A.J.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>