<article>
<section>
<h1 id="header">Expression, purification, and characterization of a membrane-associated cyclic oligo-adenylate degrader from <i>Sulfolobus islandicus</i>
</h1>
<p><time datetime="2021-01-29">Published: January 29, 2021</time></p>
<p>Ruiliang Zhao,<sup><a href="#aff1">1</a></sup> Yang Yang,<sup><a href="#aff1">1</a></sup> Ke Yang,<sup><a href="#aff1">1</a></sup> and Wenyuan Han<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>State Key Laboratory of Agricultural Microbiology and College of Life Science and Technology, Huazhong Agricultural University, 430070 Wuhan, China</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:hanwenyuan@mail.hzau.edu.cn">hanwenyuan@mail.hzau.edu.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100299">10.1016/j.xpro.2021.100299</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Type III CRISPR-cas systems initiate cyclic oligo-adenylate (cOA) signaling to initiate immune response. Previously, we identified that a membrane-associated DHH-DHHA1 family protein from <i>Sulfolobus islandicus</i> efficiently degrades cOA. Here, we provide detailed protocols for expression and purification of the protein from its native host and a cOA degradation assay with the purified enzyme. The methodology should be of interest for researchers studying <i>Sulfolobus</i>, membrane-associated proteins, or type III CRISPR-cas systems.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib10">Zhao et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Construct a <i>Sulfolobus</i> strain to express a membrane-associated DHH-DHHA1 protein (MAD)</li>
<li>Purify MAD by detergent treatment followed by chromatography</li>
<li>Analyze the degradation of type III CRISPR second messenger by MAD</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/433-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Make <i>S. islandicus</i> competent cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~10 days</div>
<ol>
<li>Inoculate <i>S. islandicus</i> E233S1 cells (<a href="#bib2">Deng et al., 2009</a>) from −80°C stock into 5 mL of SCVU medium and grow them in a 78°C shaker (the optimum growth temperature of <i>Sulfolobus</i> is 78°C).</li>
<li>When the optical density at 600 nm (OD<sub>600</sub>) of the culture is about 0.6–0.8, transfer 3 mL of the culture into a conical flask containing 30 mL SCVU medium. (see <a href="#troubleshooting">Troubleshooting</a> <a href="#sec6.1">Problem 1</a>)</li>
<li>When the OD<sub>600</sub> of the 30 mL culture is about 0.6–0.8, transfer a specific volume of the culture into 30 mL fresh SCVU medium so that the OD<sub>600</sub> of the new culture is about 0.05. The volume to be transferred (V) can be calculated with the following equation:<br> V*OD<sub>600</sub> = (30 mL + V)*0.05</li>
<li>After 24 h, measure the OD<sub>600</sub> of the culture, which should be ~0.3. Transfer the cells again into a new conical flask containing 30 mL medium and make the initial OD<sub>600</sub> about 0.05.</li>
<li>After 24 h, repeat step 4.</li>
<li>After 24 h, repeat step 4 except that the cells are transferred into 100 mL medium.</li>
<li>After 18 h, measure the OD<sub>600</sub> of the culture, which should be ~0.2. If so, collect the cells by centrifugation at 6,500 × <i>g</i> for 10 min at 22°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the OD<sub>600</sub> of the culture is greater than 0.25, transfer the cells in to 100 mL fresh medium as described above again and repeat step 7 after 18 h.</div>
<ol start="8">
<li>Remove supernatant, resuspend the cells in 40 mL of 20 mM sucrose and collect the cells again.</li>
<li>Repeat step 8 twice.</li>
<li>Remove supernatant gently and resuspend the cells in about 1 mL of 20 mM sucrose. Divide the cell suspension into 50 μL aliquots and store them at −80°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The OD<sub>600</sub> of the cell suspension should be 6–10. To determine the OD<sub>600</sub>, dilute 50 μL of the cell suspension about 10 times, the OD<sub>600</sub> of the diluted cell suspension should be 0.6–1.0.</div>
<div class="note">
<span class="note-title">Note:</span> After washing with 20 mM sucrose, the cell pellet may become loose. Longer centrifugation time may be required to reduce loss of cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If the culture growth rate is slower than expected, repeat the procedure from step 3.</div>
<h3 id="sec1.2">Synthesize radioactive cyclic oligo-adenylate (cOA)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="11">
<li>Mix the components as shown in the table below in an Eppendorf (EP) tube and incubate the mixture at 70°C for 20 min.</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th colspan="3">Reaction components of cOA synthesis assay</th>
</tr>
<tr>
<th>Reaction components</th>
<th>Stock concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Mes (pH 6.0)</td>
<td>200 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>MnCl<sub>2</sub>
</td>
<td>50 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>DTT</td>
<td>50 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>ATP</td>
<td>1 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>target RNA</td>
<td>2 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Cmr-α</td>
<td>200 nM</td>
<td>1 μL</td>
</tr>
<tr>
<td>α<sup>32</sup>P-ATP</td>
<td>3,000 Ci/mmol, 10 mCi/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>DEPC H<sub>2</sub>O</td>
<td> </td>
<td>3 μL</td>
</tr>
<tr>
<td>Total volume</td>
<td colspan="2">10 μL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> α<sup>32</sup>P-ATP is radioactive. The operator has to be trained before the assay and protected during the assay according to a local radiation safety manual.</div>
<div class="note">
<span class="note-title">Note:</span> High quality of Cmr-α complex is very important for the cOA synthesis assay. We previously reported detailed protocols for expression and purification of Cmr-α complex containing uniform crRNA species from its native host (<a href="#bib3">Han et al., 2017</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Since Cmr-α may mediate target RNA cleavage at 22°C, it is recommended that Cmr-α is supplemented into the mixture as the final component.</div>
<div class="note">
<span class="note-title">Note:</span> If the radioactivity of α<sup>32</sup>P-ATP is weak due to long-time storage, more α<sup>32</sup>P-ATP should be supplemented into the mixture.</div>
<div class="note">
<span class="note-title">Note:</span> RNase-free environment is critical for cOA synthesis assay</div>
<ol start="12">
<li>Add 590 μL DEPC H<sub>2</sub>O, heat the sample at 95°C for 5 min, collect all liquid at the bottom of the tube with gentle centrifugation and store the sample at −20°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is strongly recommended to analyze the amount of ATP that has been converted into cOA by denaturing gel electrophoresis and radioautography before further experiments. Detailed protocol is shown in steps 33–39.</div>
<div class="note">
<span class="note-title">Note:</span> The Cmr-α complex generates cyclic tetra-adenylate (cA4) as main product and cyclic tri-adenylate (cA3) as byproduct (<a href="#bib4">Han et al., 2018</a>), which cannot be distinguished by denaturing gel electrophoresis. High performance liquid chromatography (HPLC) and mass spectrometry are required to identify the molecules.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>S. islandicus</i> E233S1</td>
<td>(<a href="#bib2">Deng et al., 2009</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>E. coli</i> DH5α</td>
<td>Novagene</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>α<sup>32</sup>P-ATP</td>
<td>PerkinElmer</td>
<td>Cat#NEG003H250UC</td>
</tr>
<tr>
<td>n-Dodecyl-β-D-maltopyranoside (DDM)</td>
<td>RHAWN</td>
<td>Cat#R012115</td>
</tr>
<tr>
<td>ATP</td>
<td>Invitrogen</td>
<td>Cat#18330019</td>
</tr>
<tr>
<td>2× RNA loading dye</td>
<td>Thermo Scientific</td>
<td>Cat#R0641</td>
</tr>
<tr>
<td>Gelrite</td>
<td>Duchefa Biochemie</td>
<td>Cat#71010-52-1</td>
</tr>
<tr>
<td>Difco casamino acids, vitamin assay (CAA)</td>
<td>Thermo Scientific</td>
<td>Cat#228820</td>
</tr>
<tr>
<td>Goldview</td>
<td>Coolaber</td>
<td>Cat#SL2140-1ml</td>
</tr>
<tr>
<td>4-Morpholineethanesulfonic acid (MES)</td>
<td>Macklin</td>
<td>Cat#M813436</td>
</tr>
<tr>
<td>Niacin</td>
<td>Macklin</td>
<td>Cat#N814565</td>
</tr>
<tr>
<td>Biotin</td>
<td>Macklin</td>
<td>Cat#B6220</td>
</tr>
<tr>
<td>Pantothenate</td>
<td>Macklin</td>
<td>Cat#D805185</td>
</tr>
<tr>
<td>Lipoic acid</td>
<td>Macklin</td>
<td>Cat#A835604</td>
</tr>
<tr>
<td>Folic acid</td>
<td>Macklin</td>
<td>Cat#F809516</td>
</tr>
<tr>
<td>p-Aminobenzoic acid</td>
<td>Macklin</td>
<td>Cat#A800692</td>
</tr>
<tr>
<td>Vitamin B1</td>
<td>Coolaber</td>
<td>Cat#CV11781</td>
</tr>
<tr>
<td>Vitamin B2</td>
<td>Coolaber</td>
<td>Cat#CV11791</td>
</tr>
<tr>
<td>Vitamin B6</td>
<td>Coolaber</td>
<td>Cat#CV11821</td>
</tr>
<tr>
<td>Vitamin B12</td>
<td>Coolaber</td>
<td>Cat#CV11831</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pSeSD</td>
<td>(<a href="#bib7">Peng et al., 2012</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pSeSD-SiRe_0244</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Phanta Max Super-Fidelity DNA Polymerase</td>
<td>Vazyme</td>
<td>Cat#P505-d1</td>
</tr>
<tr>
<td>2XTaq Master Mix (Dye Plus)</td>
<td>Vazyme</td>
<td>Cat#P112-01</td>
</tr>
<tr>
<td>HiPure PCR Pure Mini Kit</td>
<td>Magen</td>
<td>Cat#D2121-03</td>
</tr>
<tr>
<td>HiPure Plasmid Micro Kit</td>
<td>Magen</td>
<td>Cat#P1001-01C</td>
</tr>
<tr>
<td>FastDigest NdeI</td>
<td>Thermo Scientific</td>
<td>Cat#FD0583</td>
</tr>
<tr>
<td>FastDigest SalI</td>
<td>Thermo Scientific</td>
<td>Cat#FD0644</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>Thermo Scientific</td>
<td>Cat#EL0014</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>SiRe_0244-for: 5′-atatccatatggaatattatgcaatagtacataacg-3′</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>SiRe_0244-rev: 5′-gtacgtcgacatacggcgaagaagcgctactc-3′</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>Seq-F: 5′-aactggcggtacatagtggta-3′</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>Target RNA: 5′-uuaagucugguuucccuccaggguaucuaagcuuugaagggg-3′</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ</td>
<td>National Institutes of Health</td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td>UNICON</td>
<td>Cytiva</td>
<td><a href="https://www.cytivalifesciences.com/en/us/shop/chromatography/software/unicorn-7-p-05649">https://www.cytivalifesciences.com/en/us/shop/chromatography/software/unicorn-7-p-05649</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>HisTrap HP 1 mL</td>
<td>Cytiva</td>
<td>Cat#17524701</td>
</tr>
<tr>
<td>Superdex 200 Increase 10/300 GL</td>
<td>Cytiva</td>
<td>Cat#28990944</td>
</tr>
<tr>
<td>Gene pulser/micropulser electroporation cuvettes, 0.1 cm gap</td>
<td>Bio-Rad</td>
<td>Cat#1652089</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">
<i>Sulfolobus</i> medium</h3>
<p>The <i>Sulfolobus</i> medium is prepared following a published recipe with modifications (<a href="#bib11">Zillig et al., 1994</a>). The stock solutions for <i>Sulfolobus</i> medium include mineral salts, Ca/Mg solution, vitamin mixture, 50% sucrose, 20% casamino acid (CAA), 20% D-arabinose and 20% uracil.</p>
<p>The mineral salts contain nine different salt solutions, each of which is prepared separately. The concentrations of them in stocks and the amounts required for 1 L medium are shown in the following table.</p>
<table id="undtbl2">
<thead>
<tr>
<th colspan="3">Mineral salt mixture</th>
</tr>
<tr>
<th>Salts</th>
<th>Concentration (w/v)</th>
<th>Amount for 1 L medium (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>MnCl<sub>2</sub>·4H<sub>2</sub>O</td>
<td>2%</td>
<td>40</td>
</tr>
<tr>
<td>Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>·10H<sub>2</sub>O</td>
<td>2%</td>
<td>105</td>
</tr>
<tr>
<td>ZnSO<sub>4</sub>·7H<sub>2</sub>O</td>
<td>2%</td>
<td>5.5</td>
</tr>
<tr>
<td>CuSO<sub>4</sub>·5H<sub>2</sub>O</td>
<td>0.2%</td>
<td>12.5</td>
</tr>
<tr>
<td>Na<sub>2</sub>MoO<sub>4</sub>·2H<sub>2</sub>O</td>
<td>0.2%</td>
<td>7.5</td>
</tr>
<tr>
<td>VOSO<sub>4</sub>·5H<sub>2</sub>O</td>
<td>0.2%</td>
<td>7.5</td>
</tr>
<tr>
<td>CoSO<sub>4</sub>·7H<sub>2</sub>O</td>
<td>0.2%</td>
<td>2.5</td>
</tr>
<tr>
<td>NiSO<sub>4</sub>·6H<sub>2</sub>O</td>
<td>0.2%</td>
<td>2.5</td>
</tr>
<tr>
<td>FeSO<sub>4</sub>·7H<sub>2</sub>O</td>
<td>2%</td>
<td>100</td>
</tr>
<tr>
<td colspan="2">In total</td>
<td>283</td>
</tr>
</tbody>
</table>
<p class="table-legend">Measure each salt in a beaker and add ~80 mL water to resuspend the salts. For FeSO<sub>4</sub>, add 0.5 M HCl until the salt is dissolved, supplement water up to 100 mL, and filter-sterilize the solution through a 0.22-μm filter.</p>
<table id="undtbl3">
<thead>
<tr>
<th colspan="2">Ca/Mg solution</th>
</tr>
<tr>
<th>Component</th>
<th>Amount (g)</th>
</tr>
</thead>
<tbody>
<tr>
<td>MgCl<sub>2</sub>·6H<sub>2</sub>O</td>
<td>203.3</td>
</tr>
<tr>
<td>Ca(NO<sub>3</sub>)<sub>2</sub>·4H<sub>2</sub>O</td>
<td>70.8</td>
</tr>
</tbody>
</table>
<p class="table-legend">Dissolve the salts in water, add water up to 1 L, and steam-sterilize the solution.</p>
<table id="undtbl4">
<thead>
<tr>
<th colspan="2">Vitamin mixture (200×)</th>
</tr>
<tr>
<th>Component</th>
<th>Amount (mg)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Niacin</td>
<td>20</td>
</tr>
<tr>
<td>Biotin</td>
<td>8</td>
</tr>
<tr>
<td>Pantothenate</td>
<td>20</td>
</tr>
<tr>
<td>Lipoic acid</td>
<td>20</td>
</tr>
<tr>
<td>Folic acid</td>
<td>8</td>
</tr>
<tr>
<td>p-Aminobenzoic acid</td>
<td>20</td>
</tr>
<tr>
<td>Vitamin B1</td>
<td>20</td>
</tr>
<tr>
<td>Vitamin B2</td>
<td>20</td>
</tr>
<tr>
<td>Vitamin B6</td>
<td>20</td>
</tr>
<tr>
<td>Vitamin B12</td>
<td>20</td>
</tr>
</tbody>
</table>
<p class="table-legend">Dissolve the above vitamins in 1 L water and filter-sterilize the solution with a 0.22-μm filter.</p>
<table id="undtbl5">
<thead>
<tr>
<th colspan="3">SCVU</th>
</tr>
<tr>
<th>Component</th>
<th>Amount per liter</th>
<th> </th>
</tr>
</thead>
<tbody>
<tr>
<td>Mineral salt mixture</td>
<td>283 μL</td>
<td rowspan="6">Add water up to 970 mL, adjust pH to 3.0, steam-sterilize the solution.</td>
</tr>
<tr>
<td>Ammonium sulfate</td>
<td>3 g</td>
</tr>
<tr>
<td>Kalium sulfate</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Kalium chloride</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Glycine</td>
<td>0.7 g</td>
</tr>
<tr>
<td>Ca/Mg solution</td>
<td>1 mL</td>
</tr>
<tr>
<td>50% sucrose</td>
<td>4 mL</td>
<td rowspan="4">Added before use.</td>
</tr>
<tr>
<td>20% CAA</td>
<td>10 mL</td>
</tr>
<tr>
<td>Vitamin mixture</td>
<td>5 mL</td>
</tr>
<tr>
<td>20% uracil</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">The genetic host <i>S. islandicus</i> E233S1 is grown in SCVU medium. Transformants are grown in SCV medium that does not contain uracil. STV, where tryptone (2 g/L) replaces CAA, is used for large-scale culture growth. ATV, where 20% D-arabinose (10 mL/L) replaces 50% sucrose, is used to induce protein expression from the arabinose promoter.</p>
<table id="undtbl6">
<thead>
<tr>
<th colspan="3">2× SCV</th>
</tr>
<tr>
<th>Component</th>
<th>Amount per liter</th>
<th> </th>
</tr>
</thead>
<tbody>
<tr>
<td>Mineral salt mixture</td>
<td>566 μL</td>
<td rowspan="6">Add water up to 970 mL, adjust pH to 3.0, steam-sterilize the solution.</td>
</tr>
<tr>
<td>Ammonium sulfate</td>
<td>6 g</td>
</tr>
<tr>
<td>Kalium sulfate</td>
<td>1 g</td>
</tr>
<tr>
<td>Kalium chloride</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Glycine</td>
<td>1.4 g</td>
</tr>
<tr>
<td>Ca/Mg solution</td>
<td>20 mL</td>
</tr>
<tr>
<td>50% sucrose</td>
<td>8 mL</td>
<td rowspan="3">Added before use.</td>
</tr>
<tr>
<td>20% CAA</td>
<td>20 mL</td>
</tr>
<tr>
<td>Vitamin mixture</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To make SCV plates, mix preheated 2× SCV with equal volume of boiled 1.4% gelrite and pour ∼30 mL of the mixture onto a glass dish. Cool the plates at 22°C until the medium becomes solid and store them at 4°C.</p>
<div class="note">
<span class="note-title">Note:</span> The amount of Ca/Mg solution in 2× SCV is 20 times of that in SCV.</div>
<h3 id="sec2.2">Other solutions</h3>
<table id="undtbl7">
<thead>
<tr>
<th>Name</th>
<th>Recipe</th>
</tr>
</thead>
<tbody>
<tr>
<td>Mes (pH 6.0)</td>
<td>Dissolve 3.9 g Mes in 80 mL DEPC H<sub>2</sub>O, adjust pH to 6.0 with 4 M NaOH, add DEPC H<sub>2</sub>O to 100 mL, filter-sterilize the solution through a 0.22 μm filter, divide it into 1 mL aliquots and store them at −20°C</td>
</tr>
<tr>
<td>Incubation medium</td>
<td>3 g ammonium sulfate, 0.5 g kalium sulfate, 0.1 g kalium chloride, 0.7 g glycine in 1 L water, steam-sterilized</td>
</tr>
<tr>
<td>1.4% gelrite</td>
<td>1.4 g gelrite in 100 mL water, steam-sterilized</td>
</tr>
<tr>
<td>0.4% gelrite</td>
<td>0.4 g gelrite in 100 mL water, steam-sterilized</td>
</tr>
<tr>
<td>BufferA</td>
<td>20 mM HEPES pH 7.5, 20 mM imidazole, 500 mM NaCl, filtered</td>
</tr>
<tr>
<td>BufferA1</td>
<td>20 mM HEPES pH 7.5, 20 mM imidazole, 500 mM NaCl, 0.02% DDM, filtered</td>
</tr>
<tr>
<td>BufferB1</td>
<td>20 mM HEPES pH 7.5, 500 mM imidazole, 500 mM NaCl, 0.02% DDM, filtered</td>
</tr>
<tr>
<td>BufferC1</td>
<td>20 mM Tris-HCl pH 7.5, 250 mM NaCl, 0.02% DDM, filtered</td>
</tr>
<tr>
<td>10× TBE buffer</td>
<td>108 g of Tris base, 55 g of boric acid, 7.5 g of EDTA dissolved in about 1 L water, steam-sterilized</td>
</tr>
<tr>
<td>5× SDS-loading dye</td>
<td>250 mM Tris-HCl pH6.8, 10% SDS, 500 mM DTT, 50% glycerol, and 0.5% bromophenol blue dye</td>
</tr>
<tr>
<td>SDS-PAGE running buffer</td>
<td>3.03 g Tris base, 14.4 g glycine, 1 g SDS dissolved in 1 L water</td>
</tr>
<tr>
<td>Gel staining solution</td>
<td>1 g Coomassie brilliant blue R250, 250 mL isopropanol, 100 mL acetic acid, add water up to 1 L</td>
</tr>
<tr>
<td>Gel destaining solution</td>
<td>100 mL ethanol, 100 mL acetic acid, add water up to 1 L</td>
</tr>
</tbody>
</table>
<h3 id="sec2.3">Equipment</h3>
<p>T100 Thermo cycler (Bio-Rad, USA)</p>
<p>Gene Pulser Xcell Electroporation System (Bio-Rad, USA)</p>
<p>JN-02C French press (JNBIO, China)</p>
<p>Peristaltic Pump P-1 (Cytiva, USA)</p>
<p>ÄKTA Pure 25 L Chromatography System (Cytiva, USA)</p>
<p>Mini-PROTEAN Tetra Handcast System (Bio-Rad, USA)</p>
<p>Mini-PROTEAN Tetra Vertical Electrophoresis Cell (Bio-Rad, USA)</p>
<p>BG-verMIDI vertical electrophoresis system (Baijing, China)</p>
<p>Storage Phosphor Screen BAS-IP MS 2025 E (Cytiva, USA)</p>
<p>Fujifilm FLA-5100 (FUJIFILM Life Science, Japan)</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> All the equipment can be replaced by others with the same functions.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Construction of the expression plasmid (pSiRe_0244)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This step is to construct the plasmid that expresses a target gene in <i>S. islandicus</i> based on shuttle vector pSeSD (<a href="#bib7">Peng et al., 2012</a>). The vector contains an engineered arabinose promoter (P<sub>ara</sub>) to drive the transcription of the target gene and sequences encoding two 6xHis tags, allowing either C-terminal or N-terminal His tag as desired (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/433-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Sequences features of the arabinose promoter (P<sub>ara</sub>) and multiple cloning sites (MCS) region of pSeSD</div>
<p>The sequences encoding the two 6xHis tags are indicated. SD, Shine-Dalgarno sequence.</p>
</figcaption>
</figure>
<ol>
<li>Amplify the gene encoding the membrane-associated DHH-DHHA1 family protein (<i>SiRe_0244</i>) by PCR with the genomic DNA of <i>S. islandicus</i> as template using Phanta Max Super-Fidelity DNA Polymerase (Vazyme, China) in T100 Thermo cycler (Bio-Rad, USA). The PCR reaction mixture and cycles are set up as below:</li>
</ol>
<table id="undtbl8">
<thead>
<tr>
<th colspan="2">PCR reaction components</th>
</tr>
<tr>
<th>Component</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>2 × Phanta® Max Buffer</td>
<td>25</td>
</tr>
<tr>
<td>dNTP mix (10 mM each)</td>
<td>1</td>
</tr>
<tr>
<td>SiRe_0244-for (10 μM)</td>
<td>2</td>
</tr>
<tr>
<td>SiRe_0244-rev (10 μM)</td>
<td>2</td>
</tr>
<tr>
<td>template (50 ng/μL E233S1 genomic DNA)</td>
<td>1</td>
</tr>
<tr>
<td>Phanta Max Super-Fidelity Polymerase</td>
<td>1</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>18</td>
</tr>
<tr>
<td>Total</td>
<td>50</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Cycle Sep</th>
<th>Temperature (°C)</th>
<th>Time</th>
<th>Number of Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95</td>
<td>3 min</td>
<td> </td>
</tr>
<tr>
<td>Denaturation</td>
<td>95</td>
<td>30 s</td>
<td rowspan="3">32</td>
</tr>
<tr>
<td>Annealing</td>
<td>55</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72</td>
<td>5 min</td>
<td> </td>
</tr>
<tr>
<td>Hold</td>
<td>12</td>
<td>∞</td>
<td> </td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Purify the PCR product with HiPure PCR Pure Mini Kit (Magen, China) following the manufacturer instruction. Digest the PCR product and the vector pSeSD with NdeI and SalI (Thermo Scientific, USA) in the following reaction mixture at 37°C for 2 h.</li>
</ol>
<table id="undtbl9">
<thead>
<tr>
<th colspan="2">Restriction digestion components</th>
</tr>
<tr>
<th>Component</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× FastDigest buffer</td>
<td>5</td>
</tr>
<tr>
<td>Plasmid/ PCR product</td>
<td>About 1 μg</td>
</tr>
<tr>
<td>FastDigest NdeI</td>
<td>1</td>
</tr>
<tr>
<td>FastDigest SalI</td>
<td>1</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>Up to 50</td>
</tr>
<tr>
<td>Total</td>
<td>50</td>
</tr>
</tbody>
</table>
<ol start="3">
<li>Purify the digested DNA fragments with HiPure PCR Pure Mini Kit and ligate them using T4 DNA Ligase (Thermo Scientific, USA) in the following reaction mixture at 22°C for 2 h.</li>
</ol>
<table id="undtbl10">
<thead>
<tr>
<th colspan="2">Ligation components</th>
</tr>
<tr>
<th>Component</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× T4 DNA Ligase Buffer</td>
<td>1</td>
</tr>
<tr>
<td>Digested PCR product</td>
<td>About 20 ng</td>
</tr>
<tr>
<td>Digested plasmid</td>
<td>About 20 ng</td>
</tr>
<tr>
<td>50% PEG 4000 solution</td>
<td>1</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>1</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>Up to 10</td>
</tr>
<tr>
<td>Total</td>
<td>10</td>
</tr>
</tbody>
</table>
<ol start="4">
<li>Transform the ligated plasmid into <i>E. coli</i> DH5α cells using classical heat shock transformation procedure. Plate the cells onto LB-agar plates containing 100 μg/mL ampicillin.</li>
</ol>
<ol start="5">
<li>Colony PCR verification.
<ol type="a">
<li>Pipette up a colony using a 1 mL tip and scatter the colony in 50 μL H<sub>2</sub>O by pipetting up and down. Use 1 μL of the cell suspension as template for PCR amplification with 2XTaq Master Mix (Vazyme, China) as below. Transfer the residual cell suspension into 1 mL LB medium containing 100 μg/mL ampicillin and culture the cells at 37°C for 6 h with shaking. Usually, 8 colonies are selected for PCR verification.
<table id="undtbl11">
<thead>
<tr>
<th colspan="2">PCR reaction components</th>
</tr>
<tr>
<th>Component</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>2× PCR mixture</td>
<td>15</td>
</tr>
<tr>
<td>Seq-F</td>
<td>1</td>
</tr>
<tr>
<td>SiRe_0244-rev</td>
<td>1</td>
</tr>
<tr>
<td>template</td>
<td>1</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>12</td>
</tr>
<tr>
<td>Total</td>
<td>30</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Cycle Sep</th>
<th>Temperature (°C)</th>
<th>Time</th>
<th>Number of Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95</td>
<td>3 min</td>
<td> </td>
</tr>
<tr>
<td>Denaturation</td>
<td>95</td>
<td>30 s</td>
<td rowspan="3">32</td>
</tr>
<tr>
<td>Annealing</td>
<td>53</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72</td>
<td>5 min</td>
<td> </td>
</tr>
<tr>
<td>Hold</td>
<td>12</td>
<td>∞</td>
<td> </td>
</tr>
</tbody>
</table>
</li>
<li>Run agarose gel electrophoresis to analyze the PCR products. The expected size of the PCR product is ~1.1 kb.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Transfer 0.5 mL of a validated cell suspension into a new EP tube and send the sample for Sanger sequencing (Tsingke, China). Transfer the left cell suspension into 5 mL LB medium containing 100 μg/mL ampicillin and grow the cells at 37°C for 16 h with shaking. Extract the plasmid, i.e., pSeSD-SiRe_0244, with HiPure Plasmid Micro Kit (Magen, China). The plasmid concentration is usually 300–500 ng/μL.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the plasmid yield or quality is low, ethanol precipitation is recommended to concentrate and desalt the plasmid prior to being used in electroporation.</div>
<h3 id="sec3.2">Electroporation of the expression plasmid and validation of transformants</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~2 weeks</div>
<ol start="7">
<li>Thaw a tube of competent cells and mix them with ~500 ng of pSeSD-SiRe_0244. Transfer the mixture into an 1 mm electroporation cuvette. Place the cuvette in the cuvetter chamber of Gene Pulser Xcell Electroporation System (Bio-Rad, USA) and apply electrical pulse at 1.2 kV, 25 μF, 600 Ω.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The pulse time should be ~13 ms. A short pulse time may indicate high electric conductivity in the mixture and lead to failed transformation.</div>
<ol start="8">
<li>Resuspend the cells with 1 mL of pre-warmed incubation medium immediately after electrical pulse, and transfer the cell suspension into an EP tube. Incubate the cells at 75°C for ~1 h without shaking.</li>
<li>Pre-warm the solutions required for plating in 15 mL tubes, including 4 mL of 0.4% gelrite and 4 mL of 2× SCV for each plate. Pipette 1 μL and 10 μL of the cell suspension into tubes containing 2× SCV respectively. Pour 4 mL of 0.4% gelrite into each tube, mix them quickly, and pour the mixture onto a SCV plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the expressed gene is not toxic for <i>Sulfolobus</i>, 1 μL of the cell suspension should generate enough colonies. However, plating of different volumes of the cell suspension is still recommended.</div>
<div class="note">
<span class="note-title">Note:</span> Simultaneously, electroporation of competent cells without any vector and empty vector (pSeSD) should be performed as controls.</div>
<ol start="10">
<li>Place the plates in a plastic box which should then be placed in a 75°C incubator. Maintain moisture inside the box to prevent the plates from drying.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We usually grow <i>Sulfolobus</i> cells on plates at 75°C to prevent plates from melting.</div>
<ol start="11">
<li>After seven days, check the plates. (See <a href="#troubleshooting">Troubleshooting</a> <a href="#sec6.3">Problem 2</a> and <a href="#sec6.5">3</a>). If there are colonies like in <a href="#fig2">Figure 2</a>A. Transfer 6–8 colonies onto new plates by the following procedure:
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/433-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative results derived from electroporation and validation of the transformants</div>
<p>(A) Examples of the plates on the 7<sup>th</sup> day after electroporation. pSeSD-SiRe_0244, pSeSD and H<sub>2</sub>O have been transformed into <i>S. islandicus</i> E233S1. The volumes of the cell suspensions used for plating are indicated above the image.</p>
<p>(B) A representative plate on to which three colonies from (A) have been transferred.</p>
<p>(C) A gel image of culture PCR to verify the strain containing pSeSD-SiRe_0244. The expected size of the PCR product is 1,091 bp. −, negative control with H<sub>2</sub>O as template; +, positive control with pSeSD-SiRe_0244 plasmid as template; 1–8, 8 transformants derived from (A).</p>
</figcaption>
</figure>
<ol type="a">
<li>Transfer about 30 μL H<sub>2</sub>O in an EP tube. Pipette up a colony from the plate using a 1 mL tip and scatter the colony in the 30 μL H<sub>2</sub>O by pipetting up and down.</li>
<li>Transfer the cell suspension onto a new SCV plate.</li>
</ol>
</li>
<li>After 3–5 days, the cells on the plate should form lawn as shown in <a href="#fig2">Figure 2</a>B. Transfer each colony into 5 mL of SCV medium.</li>
<li>After 2–3 days, verify the cultures using PCR by the following procedure:
<ol type="a">
<li>Transfer 300 μL culture into an EP tube. Centrifuge at 8,500 × <i>g</i> for 3 min to collect the cells.</li>
<li>Resuspend the cells with 500 μL H<sub>2</sub>O (the volume of H<sub>2</sub>O may vary depending on the amount of cells) and heat the cell suspension at 95°C for 10 min.</li>
<li>Cool the cell solution on ice and use 1 μL as template for PCR analysis. The PCR reaction mixture and cycle conditions should be the same as step 5.</li>
<li>Run agarose gel electrophoresis to analyze the PCR products (<a href="#fig2">Figure 2</a>C).</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Prepare cell mass for protein purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~10 days</div>
<ol start="14">
<li>Transfer a verified culture into 30 mL STV medium. After 1 or 2 days, at an OD<sub>600</sub> of about 0.6–0.8, inoculate the strain into 100 mL STV medium (10 mL into 100 mL). After 1 or 2 days when the OD<sub>600</sub> is about 0.6–0.8, transfer the 100 mL culture into 1 L STV medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Tryptone is richer than CAA and contains a trace amounts of uracil. Therefore, tryptone is not recommended for growth of transformants obtained through electroporation.</div>
<ol start="15">
<li>After 1 or 2 days when the OD<sub>600</sub> is about 0.6–0.8, transfer the 1 L culture into 8 L ATV medium containing 0.2% D-arabinose instead of sucrose.</li>
<li>Collect cells from 8 L culture by centrifugation at 8,500 × <i>g</i> for 10 min, resuspend them with 100 mL of BufferA, and store the cell suspension at −80°C.</li>
</ol>
<h3 id="sec3.4">Purification of SiRe_0244</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step is to purify His-tagged SiRe_0244 from the cell mass of step 16. The purification procedure includes two main steps: nickel affinity chromatography and size exclusion chromatography (SEC). Before that, the detergent n-Dodecyl-β-D-Maltopyranoside (DDM) is used to treat cell lysate to extract SiRe_0244 from the cell membrane.</p>
<ol start="17">
<li>Thaw the frozen cells in warm water and disrupt the cells by JN-02C French press (JNBIO, China) with the pressure of 1,200 bar.</li>
<li>Supplement DDM into the cell lysate at a final concentration of 0.1% and incubate the lysate at 4°C for 16 h with gentle shaking.</li>
<li>Centrifuge at 18,000 × <i>g</i> for 20 min at 4°C to remove undisrupted cells or cell debris.</li>
<li>During the 20 min, wash Peristaltic Pump P-1 (Cytiva, USA) with 10 mL of 20% ethanol followed by 10 mL filtered water and connect HisTrap HP column (Cytiva, USA) with the pump as shown in <a href="#fig3">Figure 3</a>. Wash the column with 10 mL filtered water and equilibrate the column with 20 mL BufferA1.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/433-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. How to connect the HisTrap column to Peristaltic Pump P-1</div>
<p>Filtered water, BufferA1 or cell lysate can be poured into the beaker, then driven by the pump to flow through the column and finally collected by the tube.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cytiva was previously a part of GE Healthcare Life Sciences.</div>
<ol start="21">
<li>Filter the cleared lysate with 0.22 μm syringe filter and load the lysate onto the column at a speed of about 1 mL/min using Peristaltic Pump P-1. This step takes about 2 h.</li>
<li>During the 2 h, equilibrate Superdex 200 Increase 10/300 GL column (Cytiva, USA) with BufferC1 using ÄKTA Pure 25 L Chromatography System (Cytiva, USA), and re-equilibrate the ÄKTA system with BufferA1.
<ol type="a">
<li>Turn on the ÄKTA system and open the UNICON software. Go to “Manual&gt;Execute Manual Instructions” in the system control window, and use the “Manual instructions” dialog to wash both pumps (A and B) and the system with filtered water.</li>
<li>Connect the Superdex 200 column with the ÄKTA system. Equilibrate the column with filtered water and then BufferC1 at the speed of about 0.5 mL/min until the values of UV, pressure, and conductivity are stable.</li>
<li>Remove the Superdex 200 column from the ÄKTA system and wash pumpA and pumpB with BufferA1 and BufferB1 respectively.</li>
<li>Equilibrate the ÄKTA system with BufferA1.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Operation of the ÄKTA system and the UNICON software should follow the manual instruction of the manufacturer.</div>
<div class="note">
<span class="note-title">Note:</span> Make sure that the end of buffer lines are in the right buffers and covered by the buffers at all time.</div>
<ol start="23">
<li>After loading the lysate onto the HisTrap column, remove the column from Peristaltic Pump P-1 and connect it with the ÄKTA system. Wash the column with 100% BufferA1 (20 mM imidazole) for 50 mL, followed by 91% BufferA1 and 9% BufferB1 (63.2 mM imidazole) for 30 mL. Elute the bound proteins with 20 mL of 41% BufferA1 and 59% Buffer B1 (~300 mM imidazole), and wash the column with 5 mL of 100% BufferB1. Collect the fractions corresponding to the main peak and concentrate them to ~0.5 mL. The profile of the chromatography is shown in <a href="#fig4">Figure 4</a>.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/433-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Nickel affinity chromatography profile</div>
<p>The HisTrap column is washed with 0%, 9%, 59%, and 100% BufferB1 successively. The fractions eluted by 51% BufferB1 (indicated by the red box) are collected for next step of purification.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To accomplish the purification procedure within one day, the fractions of the HisTrap chromatography are not analyzed by SDS-PAGE. However, it is recommended to save aliquots of the fractions for SDS-PAGE on the next day.</div>
<ol start="24">
<li>Centrifuge the concentrated sample at 18,000 × <i>g</i> for 20 min at 4°C to remove protein aggregates.</li>
<li>Meanwhile, remove the HisTrap column from the ÄKTA system, equilibrate the ÄKTA system with BufferC1 again, wash the injection loop with filtered water and then BufferC1, and connect the Superdex 200 column with the ÄKTA system.</li>
<li>Load the concentrated proteins into the injection loop with a 1 mL syringe, set the methods as shown in <a href="#fig5">Figure 5</a>A and start running. The profile of the SEC is shown in <a href="#fig5">Figure 5</a>B. (See <a href="#troubleshooting">Troubleshooting</a> <a href="#sec6.7">Problem 4</a> and <a href="#sec6.9">5</a>)
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/433-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Methods and results of size exclusive chromatography (SEC)</div>
<p>(A) An image of the “Manual instructions” dialog, showing the settings for SEC: flow rate, 0.5 mL/min; injection valve, inject; alarm pre-column pressure, 2.20 MPa; fraction volume, 1 mL; auto zero UV.</p>
<p>(B) SEC profile.</p>
</figcaption>
</figure>
</li>
<li>After running the methods, wash the columns with filtered water and fill them with 20% ethanol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is recommended to clean the HisTrap column following the instructions of the manufacturer if the column is clogged or discolored. In brief, the column is successively washed by 10 mL of stripping buffer (20 mM sodium phosphate, 0.5 M NaCl, 50 mM EDTA, pH 7.4) to strip off Ni<sup>2+</sup> ions, 10 mL of distilled water, 10 mL of 1.5 M NaCl to remove ionically bound proteins, 10 mL of BufferA1, 10 mL of distilled water, 60 mL of 1 M NaOH to remove precipitated protein, 10 mL of BufferA1, 10 mL of distilled water, 60 mL of 0.5% Triton X-100 in 0.1 M acetic acid to remove lipids and lipoproteins and 10 mL of distilled water. Then, the column is recharged with 1 mL of 0.1 M NiSO<sub>4</sub>, washed by 10 mL of distilled water and filled with 20% ethanol. The washing is performed by the Peristaltic Pump P-1 at a flow rate of 1 mL/min.</div>
<h3 id="sec3.5">SDS-PAGE analysis of purified SiRe_0244</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~6 h</div>
<ol start="28">
<li>Cast SDS-PAGE gels using Mini-PROTEAN Tetra Handcast System (Bio-Rad, USA). The components of resolving gel and stacking gel are shown in the table below.</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th colspan="3">12% SDS-PAGE gel components</th>
</tr>
<tr>
<th>Component</th>
<th>Resolving gel (mL)</th>
<th>Stacking gel (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Water</td>
<td>3.3</td>
<td>2.1</td>
</tr>
<tr>
<td>30% acrylamide:bis-acrylamide (29:1)</td>
<td>4.0</td>
<td>0.5</td>
</tr>
<tr>
<td>1.5 M Tris-HCl, 0.4% SDS, pH 8.8</td>
<td>2.5</td>
<td>-</td>
</tr>
<tr>
<td>1 M Tris-HCl, 0.4% SDS, pH 6.8</td>
<td>-</td>
<td>0.38</td>
</tr>
<tr>
<td>10% ammonium persulfate (APS)</td>
<td>0.1</td>
<td>0.03</td>
</tr>
<tr>
<td>TEMED</td>
<td>0.004</td>
<td>0.003</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Acrylamide is toxic. This step should be performed in a fume hood with standard laboratory nitrile gloves and lab coats.</div>
<ol start="29">
<li>Mix 20 μL of each fraction with 5 μL of 5× SDS-loading dye. Heat the mixture at 95°C for 5 min.</li>
<li>Set up the Mini-PROTEAN Tetra Vertical Electrophoresis Cell (Bio-Rad, USA) for running the gel. Fill the tank with SDS-PAGE running buffer, load the samples into the wells and start running.</li>
<li>Stop running when the blue dye almost reaches the bottom of the gel. Carefully separate the gel from the glass plates and place it on a plastic plate. Stain the gel with gel staining solution for ~2 h at 22°C.</li>
<li>Destain the gel with gel destaining solution. Change destaining solution several times until the bands in the gel are visible (<a href="#fig6">Figure 6</a>A).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/433-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Analysis of the fractions from SEC</div>
<p>(A) SDS-PAGE analysis. The expected size of His-tagged SiRe_0244 is 38.7 kDa, corresponding to the band indicated the red arrow.</p>
<p>(B) cOA degradation activity.</p>
<p>The volume of each fraction is indicated above the gels. 7.1, 7.1–8.0 mL.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.6">Activity assay of SiRe_0244</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="33">
<li>Prepare a denaturing polyacrylamide gel.
<ol type="a">
<li>Mix 1.5 mL of 10× TBE, 18 mL of 30% acrylamide and 12.6 g urea in a 50 mL tube. Invert the tube several times and place it in a 37°C incubator to accelerate dissolution of urea.</li>
<li>Set up two glasses to gain the space for the gel using BG-verMIDI vertical electrophoresis system (Baijing, China).</li>
<li>After urea is dissolved, add 150 μL APS and 10 μL TEMED into the mixture, mix them quickly, and pour the mixture into the space between the two glasses. Insert comb immediately.</li>
<li>After the acrylamide is polymerized, remove the comb and wash the wells with 0.5× TBE, pre-run the gel in a PAGE tank at a voltage of 200 V for 30 min.</li>
</ol>
</li>
<li>Mix the reaction components as shown in the table below and divide it into 12 aliquots. Add 1 μL of each fraction from gel filtration chromatography into the aliquots and BufferC1 into the first one as control.</li>
</ol>
<table id="undtbl13">
<thead>
<tr>
<th colspan="3">cOA degradation assay components</th>
</tr>
<tr>
<th>Component</th>
<th>Volume (μL) (for 13 reactions)</th>
<th>Volume (μL) for each reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>Mes Buffer</td>
<td>13</td>
<td>1</td>
</tr>
<tr>
<td>5 mM MnCl<sub>2</sub>
</td>
<td>13</td>
<td>1</td>
</tr>
<tr>
<td>Radioactive cOA</td>
<td>13</td>
<td>1</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>78</td>
<td>6</td>
</tr>
<tr>
<td>Total</td>
<td>117</td>
<td>9</td>
</tr>
</tbody>
</table>
<ol start="35">
<li>Incubate the reactions at 70°C for 4 min. Chill the tubes on ice immediately and add 10 μL of 2× RNA loading dye.</li>
<li>Heat the samples at 95°C for 5 min and chill them on ice for 2 min immediately. Centrifuge for several seconds to collect the liquid at the bottom of the tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Step 34–36 can be accomplished during the time of gel polymerization and prerunning.</div>
<ol start="37">
<li>Wash the wells with 0.5× TBE again and load the samples into the well. Run the gel at a voltage of 300 V.</li>
<li>When the bromophenol band reaches the middle of the gel, Stop running, remove the small glass from the gel, and adhere the gel to an equal-size filter paper. Cover the gel with cling film, place the gel in a cassette and cover the gel with Storage Phosphor Screen BAS-IP MS 2025 E (Cytiva, USA). Place the cassette in a freezer for 16 h.</li>
<li>Scan the screen with Fujifilm FLA-5100 (FUJIFILM Life Science, Japan) to obtain the image (<a href="#fig6">Figure 6</a>B).</li>
</ol>
<h3 id="sec3.7">Quantification analysis of the cOA cleavage assay by ImageJ</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~1 h</div>
<ol start="40">
<li>Open the image from step 39 by ImageJ with the Fiji plugin (<a href="https://imagej.nih.gov">https://imagej.nih.gov</a>) and convert it to grayscale and 8-bit depth.</li>
<li>Draw a rectangle that exactly includes all the species in the first lane and go to “Analyze&gt;Gels&gt;Select First Lane” to set up the first rectangle.</li>
<li>Move the rectangle to next lane and go to “Analyze&gt;Gels&gt;Select Next Lane” until all lanes are included (<a href="#fig7">Figure 7</a>A).
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/433-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Quantification of the cOA degradation activity with ImageJ</div>
<p>(A) Determination of the area for quantification using the Rectangle tool. All the lanes have been included.</p>
<p>(B) Examples of the quantification profiles of the 1<sup>st</sup> lane (control) and 8<sup>th</sup> lane (the 13-mL fraction). The three peaks, which are separated by four short lines, represent cOA, degradation product and ATP respectively.</p>
<p>(C) Quantification data in an Excel spreadsheet. The percentage of product, i.e., [product]/([cOA]+[product]), has been calculated.</p>
<p>(D) Histogram chart of the data from (C).</p>
<p>Lane 1, control; lanes 2–12, fractions from 7.1 mL to 17.1 mL.</p>
</figcaption>
</figure>
</li>
<li>Go to “Analyze&gt;Gels&gt;Plot Lanes,” which generates a new window with a profile plot of each lane (<a href="#fig7">Figure 7</a>B).</li>
<li>Choose the Straight Line selection tool and draw short lines at both sides of a peak manually to enclose the area below it. The area represents the amount of each species of the lane (<a href="#fig7">Figure 7</a>B). (If the peak appears to float above the baseline of the profile plot, draw a line at the bottom of the peak to remove background).</li>
<li>Choose the Wand tracing tool. Click enclosed areas to measure peaks.</li>
<li>Copy the results to an Excel spreadsheet. Calculate [product]/([cOA] + [product]) in each lane to evaluate the activity of SiRe_0244 from each fraction of the size exclusion chromatography (<a href="#fig7">Figure 7</a>C). The data can be displayed in histogram chart (<a href="#fig7">Figure 7</a>D) or in other forms as desired.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Electroporation of pSeSD-SiRe_0244 into <i>S. islandicus</i> E233S generates a strain that expresses His-tagged SiRe_0244 upon arabinose induction. When purified by nickel affinity chromatography, the protein is eluted with 300 mM imidazole (<a href="#fig4">Figure 4</a>). According to the plot of size exclusion chromatography (SEC) and SDS-PAGE analysis (<a href="#fig5">Figures 5</a> and <a href="#fig6">6</a>), SiRe_0244 is present from 8.0 to 14.9 mL in SEC and forms two peaks at ~9 mL and ~14 mL. cOA degradation activity analysis of the fractions derived from SEC shows that cOA is degraded into products that migrate faster than the substrate in the denaturing gel (<a href="#fig6">Figure 6</a>B). Further, the activity of the fractions is roughly correlated to the distribution of SiRe_0244 in the fractions. However, the activity of the peak 2 fractions is generally higher than that of peak 1 fractions (<a href="#fig7">Figure 7</a>D).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>First, expression of <i>Sulfolobus</i> proteins in its native host is time-consuming (about 1 month) compared to protein expression in <i>E. coli</i>. Therefore, only proteins that are not soluble in <i>E. coli</i> are to be expressed in <i>Sulfolobus.</i></p>
<p>Second, the expression level of a specific protein is highly dependent on its inherent characters. If a protein is toxic or prone to be degraded, the yield is usually very low. In this case, the expression level of SiRe_0244 is much lower than that of EstA, which has been used to test the expression ability of pSeSD (<a href="#bib7">Peng et al., 2012</a>).</p>
<p>Third, denaturing gel electrophoresis cannot distinguish different species in cOA or identify the degradation products of cOA. HPLC and mass spectrum is a powerful tool to identify these molecules (<a href="#bib1">Athukoralage et al., 2018</a>; <a href="#bib5">Kazlauskiene et al., 2017</a>; <a href="#bib6">Niewoehner et al., 2017</a>). In addition, end profiling of the degradation products also provides important information in the identity of these molecules (<a href="#bib1">Athukoralage et al., 2018</a>; <a href="#bib10">Zhao et al., 2020</a>).</p>
<p>Fourth, extraction of SiRe_0244 from cell membrane by DDM does not generate uniform SiRe_0244-detergent complex. In our experiments, SiRe_0244 is present in the fractions from 8.0 to 14.9 mL in SEC, forming two peaks at ~9 mL and ~14 mL respectively. Further, the protein from the second peak exhibits higher cOA degradation activity than that in the first. Currently, the reasons for these phenomena are unknown. For further information on screening suitable detergents for a specific protein, please refer to (<a href="#bib9">Stetsenko and Guskov, 2017</a>).</p>
<p>At last, analysis of SiRe_0244 mutants is still required to confirm the cOA degradation activity of SiRe_0244. The protocols for expression and characterization of the mutants are the same as those of the wild type protein.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p><i>Sulfolobus</i> culture grows too slow.</p>
<h3 id="sec6.2">Potential solution</h3>
<ol type="i">
<li>Culture grows too slow at the inoculation step. Try to inoculate the cells from −80°C stock on to a SCVU plate. When the cells form lawn like those in <a href="#fig2">Figure 2</a>B, transfer the cells in liquid medium.</li>
<li>Culture grows too slow when transferred into fresh medium. Several tips can improve <i>Sulfolobus</i> culture grow. (1) Preheat the fresh medium at 78°C. (2) Increase the volume to be transferred into the fresh medium. Alternatively, collect the cells to be transferred by centrifugation, remove old medium, resuspend the cells with fresh medium and transfer them into fresh medium. (4) Supplement with 0.04% yeast extract into the medium (Note: yeast extract contains uracil and should not be used at the electroporation step).</li>
<li>Culture grows too slow in medium containing D-arabinose. That usually happens when induction of a protein is toxic. Grow the culture in sucrose up to the OD<sub>600</sub> of ~0.4, supplement D-arabinose up to 0.2%, and further grow the culture for 24–48 h to induce protein expression.</li>
</ol>
<h3 id="sec6.3">Problem 2</h3>
<p>No colonies on the plates from electroporation of expression plasmid.</p>
<h3 id="sec6.4">Potential solution</h3>
<ol type="i">
<li>If the transformation efficiency of the control plasmid (pSeSD) is also too low (less than 10<sup>3</sup> colonies per μg), repeat the steps for preparation of competent cells and SCV plates.</li>
<li>If the transformation rate of pSeSD is ~10<sup>5</sup> colonies per μg, try to reextract the expression plasmid.</li>
<li>If the transformation rate of pSeSD is ~10<sup>5</sup> colonies per μg and the protein to be expressed has potential to induce cell toxicity, try to replace P<sub>ara</sub> with a mutated version, e.g., S-50/2 promoter (<a href="#bib8">Peng et al., 2009</a>), which expresses a reporter gene at a lower level in glucose, and to replace sucrose with glucose in the medium.</li>
</ol>
<h3 id="sec6.5">Problem 3</h3>
<p>Too much background after electroporation.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>After electroporation, cells can grow on SCV plates to some extent using residual uracil or uracil from dead cells. To reduce background, we recommend to culture cells with SCV medium instead of SCVU at step 6 when making <i>S. islandicus</i> competence cells, which could reduce uracil level in the competence cells.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Too many impurities after SEC.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>If the expression level of a protein is very low and too much cell mass is applied into the HisTrap column, the purification outcome usually contains impurities. Try to modify the pSeSD plasmid to express the protein carrying a 10-His tag and increase the concentration of imidazole at both wash step (step 23) and elution step (step 24).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>No or too low protein yield.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Try to track protein expression and purification by western blot using anti-His antibody as described in our Cell Reports article (<a href="#bib10">Zhao et al., 2020</a>) to check whether the protein is not expressed or lost at a purification step.</p>
<p>If the protein is not expressed, check whether the expression plasmid is lost by PCR as described in step 5. We also recommend to repeat the electroporation assay to obtain a new strain for protein expression.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Athukoralage, J.S., Rouillon, C., Graham, S., Gruschow, S., and White, M.F. (2018). Ring nucleases deactivate type III CRISPR ribonucleases by degrading cyclic oligoadenylate. Nature <i>562</i>, 277-280. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref1">View at publisher</a></p>
<p id="bib2">Deng, L., Zhu, H., Chen, Z., Liang, Y.X., and She, Q. (2009). Unmarked gene deletion and host-vector system for the hyperthermophilic crenarchaeon <i>Sulfolobus islandicus</i>. Extremophiles <i>13</i>, 735-746. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref2">View at publisher</a></p>
<p id="bib3">Han, W., Li, Y., Deng, L., Feng, M., Peng, W., Hallstrom, S., Zhang, J., Peng, N., Liang, Y.X., White, M.F., et al. (2017). A type III-B CRISPR-Cas effector complex mediating massive target DNA destruction. Nucleic Acids Res. <i>45</i>, 1983-1993. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref3">View at publisher</a></p>
<p id="bib4">Han, W., Stella, S., Zhang, Y., Guo, T., Sulek, K., Peng-Lundgren, L., Montoya, G., and She, Q. (2018). A Type III-B Cmr effector complex catalyzes the synthesis of cyclic oligoadenylate second messengers by cooperative substrate binding. Nucleic Acids Res. <i>46</i>, 10319-10330. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref4">View at publisher</a></p>
<p id="bib5">Kazlauskiene, M., Kostiuk, G., Venclovas, C., Tamulaitis, G., and Siksnys, V. (2017). A cyclic oligonucleotide signaling pathway in type III CRISPR-Cas systems. Science <i>357</i>, 605-609. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref5">View at publisher</a></p>
<p id="bib6">Niewoehner, O., Garcia-Doval, C., Rostol, J.T., Berk, C., Schwede, F., Bigler, L., Hall, J., Marraffini, L.A., and Jinek, M. (2017). Type III CRISPR-Cas systems produce cyclic oligoadenylate second messengers. Nature <i>548</i>, 543-548. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref6">View at publisher</a></p>
<p id="bib7">Peng, N., Deng, L., Mei, Y., Jiang, D., Hu, Y., Awayez, M., Liang, Y., and She, Q. (2012). A synthetic arabinose-inducible promoter confers high levels of recombinant protein expression in hyperthermophilic archaeon <i>Sulfolobus islandicus</i>. Appl. Environ. Microbiol. <i>78</i>, 5630-5637. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref7">View at publisher</a></p>
<p id="bib8">Peng, N., Xia, Q., Chen, Z., Liang, Y.X., and She, Q. (2009). An upstream activation element exerting differential transcriptional activation on an archaeal promoter. Mol. Microbiol. <i>74</i>, 928-939. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref8">View at publisher</a></p>
<p id="bib9">Stetsenko, A. and Guskov, A. (2017). An overview of the top ten detergents used for membrane protein crystallization. Crystals <i>7</i>, 197. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref9">View at publisher</a></p>
<p id="bib10">Zhao, R., Yang, Y., Zheng, F., Zeng, Z., Feng, W., Jin, X., Wang, J., Yang, K., Liang, Y.X., She, Q., et al. (2020). A membrane-associated DHH-DHHA1 nuclease degrades type III CRISPR second messenger. Cell Rep. <i>32</i>, 108133. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref10">View at publisher</a></p>
<p id="bib11">Zillig, W., Kletzin, A., Schleper, C., Holz, I., Janekovic, D., Hain, J., Lanzendorfer, M., and Kristjansson, J.K. (1994). Screening for sulfolobales, their plasmids and their viruses in Icelandic solfataras. Syst. Appl. Microbiol. <i>16</i>, 609-628. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00006-X/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Wenyuan Han (<a href="mailto:hanwenyuan@mail.hzau.edu.cn">hanwenyuan@mail.hzau.edu.cn</a>).</p>
<h4>Materials Availability</h4>
<p>Expression plasmid and strain generated in this study are available upon request.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>The research was supported by <a href="https://doi.org/10.13039/501100001809">National Science Foundation of China</a> (grant no. 31970545), <a href="https://doi.org/10.13039/501100007925">Huazhong Agricultural University</a> Scientific &amp; Technological Self-innovation Foundation, and <a href="https://doi.org/10.13039/501100012476">Fundamental Research Funds for Central Universities</a> (grant no. 2662020SKPY001). We thank the Core Facilities at College of Life Science and Technology and research associate Dr. Delin Zhang at Center for Protein Research (CPR), Huazhong Agricultural University, for technical support.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, W.H., R.Z., and Y.Y.; Investigation, R.Z., Y.Y., and K.Y.; Writing – Original Draft, W.H. and R.Z.; Writing – Review &amp; Editing, W.H., R.Z., Y.Y., and K.Y.; Funding Acquisition, W.H.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>