<article>
<section>
<h1 id="header">Screening of CRISPR-Cas9-generated point mutant mice using MiSeq and locked nucleic acid probe PCR</h1>
<p><time datetime="2021-09-24">Published: September 24, 2021</time></p>
<p>Kommireddy Vasu<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Paul L. Fox<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Cardiovascular and Metabolic Sciences, Lerner Research Institute, Cleveland Clinic, Cleveland, OH 44195, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kommirv@ccf.org">kommirv@ccf.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:foxp@ccf.org">foxp@ccf.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100785">10.1016/j.xpro.2021.100785</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>CRISPR-Cas9-mediated, site-directed mutagenesis in mice generates mosaic founder mice with varied efficiency of desired point mutation and other non-homologous end-joined variants. Here, we present a protocol for design, sample preparation, and analysis for identification of mice with the desired mutation. Deep sequencing provides the proportion of reads of a particular allele for each mouse line. Locked nucleic acid probe-based qPCR provides rapid identification of the mutant allele and can be used for genotyping offspring during subsequent breeding for colony establishment.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00491-3/sref5">Vasu et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Deep sequencing of CRISPR-targeted founder mice quantitates multiple locus alleles</li>
<li>Locked nucleic acid qPCR rapidly screens chimeric mutant mice for desired allele</li>
<li>LNA qPCR assay efficiently genotypes mutant mice bearing a single point mutation</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1030-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>The following protocol can be used to screen CRISPR-Cas9-targeted mice by two methods, each with a distinct advantage: (i) Deep sequencing of the founder mice at the target loci by MiSeq allows quantitation of the multiple alleles generated by homology-directed repair (HDR) or by non-homologous end-joining (NHEJ) pathways. (ii) Locked nucleic acid (LNA) probe-based PCR method to identify mice with a specific mutant allele. Both male and female mice of age 10 days to 3 weeks were used in this study. Probe design for LNA probe-based PCR requires prior knowledge of the mutation through prior sequencing in the case of NHEJ, or from knowledge of the repair template used for HDR. These approaches have been used to successfully identify and breed mice generated by CRISPR-Cas9 with a point mutation in <i>Eprs</i> that deletes a portion of the zinc-binding domain (<a href="#bib5">Vasu et al., 2021</a>).</p>
<div class="note">
<span class="note-title">Note:</span> This protocol can also be adapted to other species or cell lines.</div>
<h3 id="sec1.1">Primer design for MiSeq</h3>
<p>We have used a MiSeq v3 flow cell as a Paired-End (PE) 250 cycle run (here referred to as MiSeq250), i.e., 250 cycles in each direction, or 500 cycles total are performed.</p>
<ol>
<li>Input target loci in UCSC genome browser (<a href="https://genome.ucsc.edu/">https://genome.ucsc.edu/</a>) and download the genomic DNA sequence surrounding the CRISPR-Cas9 cut site (<a href="#fig1">Figure 1</a>). Briefly,
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1030-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Screenshot of UCSC genome browser depicting target loci</div>
<p>Chromosome 1q segment of <i>Mus musculus Eprs</i> surrounding exon 32.</p>
</figcaption>
</figure>
<ol type="a">
<li>Select → Genomes → Mouse</li>
<li>Search for gene of interest, e.g., <i>Eprs</i>
</li>
<li>Zoom to region of interest, e.g., exon 32</li>
<li>Select → region of interest, e.g., exon 32</li>
<li>Select → get DNA</li>
<li>Copy and save the sequence</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Using IDT's PrimerQuest Tool (<a href="https://www.idtdna.com/PrimerQuest/Home/Index">https://www.idtdna.com/PrimerQuest/Home/Index</a>), design primers spanning ∼125 nt on either side of the cut site.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Primers can be designed manually. For manual design, the T<sub>m</sub> of the primers should be 60°C–62°C with the following parameters: 0.2 μM oligonucleotides, 50 mM Na<sup>+</sup>, 3 mM Mg<sup>2+</sup>, and 0.8 mM dNTPs.</div>
<ol type="a">
<li>Identify the region of interest from the downloaded genomic sequence.</li>
<li>Forward primer can be manually selected by identifying the 20-bp sequence 125 nt upstream of the cut site. The reverse primer can be selected by identifying the 20-bp sequence 125 nt downstream of the cut site. Determine the T<sub>m</sub> of primers using NEB T<sub>m</sub> calculator (<a href="https://tmcalculator.neb.com/">https://tmcalculator.neb.com/#!/main</a>), or other programs that calculate T<sub>m</sub>. If the forward or reverse primer does not meet the guidelines, an alternative primer pair resulting in an amplicon size range of 250 ± 50 nt is acceptable.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Add barcode amplification anchors to the designed primers as depicted below:</li>
</ol>
<p>Forward primer</p>
<p>TCCCTACACGACGCTCTTCCGATCT – Forward gene-specific primer (R1)</p>
<p>Reverse primer</p>
<p>AGTTCAGACGTGTGCTCTTCCGATCT – Reverse gene-specific primer (R2)</p>
<div class="critical">
<span class="critical-title">Critical:</span> It is strongly recommended that primers are selected with T<sub>m</sub> difference not greater than 5°C, and that GC-rich regions, secondary structures, and repeated sequences are avoided.</div>
<h3 id="sec1.2">Primers and probe design for LNA qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>In this protocol, we have used LNA probes for qPCR because they confer increased structural stability to the hybridized probe, thus leading to higher maximum annealing temperature. Design of LNA-containing probes for allele discrimination employs three primers - two primers for amplification of the target region in both alleles, and a short fluorescent probe that binds in an allele-specific manner. LNA is a nucleic acid analog that contains a 2′-O, 4′-C methylene bridge that locks the ribose in the 3′-endo conformation (<a href="#fig2">Figure 2</a>). Design rules used for incorporation of LNA bases into oligonucleotides are based on models from previous studies (<a href="#bib3">Owczarzy et al., 2011</a>, <a href="#bib2">McTigue et al., 2004</a>). In this method, the LNA probe utilizes the 5′ nuclease assay principle in which a short LNA oligonucleotide labeled with a fluorescent reporter on the 5′ end and a quencher at the 3′ end is used. The fluorescence of the reporter is quenched due to its close proximity to the quencher. During polymerization, thermostable polymerases with 5’ → 3′ exonuclease activity separates the fluorophore from the quencher when the probe is bound to the target. This gives a fluorescence signal proportional to the amount of amplification.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1030-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Comparative structure of DNA and LNA monomers</div>
<p>Nucleoside-specific bridge connecting the 2′ oxygen and 4′ carbon is highlighted (red)</p>
</figcaption>
</figure>
<ol start="4">
<li>Label the sequence downloaded in step 1 as Allele 1. Make the desired point mutation in the sequence and label it Allele 2. In case of NHEJ-based CRISPR editing, this step requires prior knowledge of the outcome of the CRISPR-Cas9 editing. In case of HDR, the probe can be designed based on the repair template. We have validated the method for a specific point mutation generated by HDR (unpublished data).</li>
<li>Design forward and reverse primers spanning the mutation site to generate an amplicon of 75 to 150 bp as described above in Step 2. These primers are non-discriminatory and can amplify either Allele 1 or Allele 2.</li>
<li>Perform NCBI blast analysis (<a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a>) to determine primer pair specificity, secondary structure, and possible presence of annotated SNPs in the region.</li>
</ol>
<ol start="7">
<li>Using the OligoAnalyzer Tool (<a href="https://www.idtdna.com/calc/analyzer">https://www.idtdna.com/calc/analyzer</a>), design the probe sequence for each allele surrounding the mutation site. Briefly, the empirical guidelines for the probe design are:
<ol type="a">
<li>Probe sequence should be relatively short (10–15 nt) compared to the amplification primers.</li>
<li>Select the probe sequence with a GC-content of 30%–60% around the mutation site.</li>
<li>Introduce LNA bases (up to 6) surrounding the mutation site to increase probe T<sub>m</sub>.</li>
<li>Select probe with T<sub>m</sub> higher than the amplification primers.</li>
<li>For each iteration, test the T<sub>m</sub> for the mismatch allele.</li>
<li>Select an LNA probe that exhibits ΔT<sub>m</sub> &gt; 15°C, compared to the mismatch allele.</li>
<li>Look for secondary structure and avoid LNA bases that participate in secondary structures.</li>
<li>Avoid continuous stretch of 4 LNA bases in the sequence.</li>
<li>Avoid G at the 5′end near the fluorophore as it can quench fluorescence.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Custom LNA probe design services are offered by IDT (<a href="https://www.idtdna.com/">https://www.idtdna.com/</a>) and Qiagen (<a href="https://www.qiagen.com/us/">https://www.qiagen.com/us/</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Design of the probes to the same strand of DNA can enhance allele discrimination due to competition between the probes.</div>
</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Selecting from FAM, Cy3, Cy5, TEX, TYE, YAK, and HEX fluorophores for 5′ end labeling, and Iowa Black™ FQ or Black Hole Quencher for the 3′ end as quencher, labeled probes can be purchased from IDT. For multiplexing reactions, the dye selection tool (<a href="https://www.idtdna.com/site/order/qpcr/primetimeprobes/multiplex">https://www.idtdna.com/site/order/qpcr/primetimeprobes/multiplex</a>) can be used for selecting dye combinations for a range of qPCR systems. We have used 5′ Hex (absorbance at 555 nm) and 3′ Iowa Black Fluorophore Quencher (IBFQ) for allele 1, and 5′ FAM (absorbance at 520 nm) and 3′ IBFQ for allele 2 (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1030-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. LNA probe design for detection of wild-type and mutant DNA</div>
<p>LNA probe design used to detect wild-type and mutant mouse tail DNA using FAM and HEX fluorophores and IBFQ quencher. LNA bases are underlined (blue).</p>
<p>Figure reprinted with permission from <a href="#bib5">Vasu et al., 2021</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Fluorophore dyes compatible with the specific qPCR instrument should be selected. Verify with the instrument’s handbook regarding peak emission wavelength detection and channels required for the reference dye. In the present study, Rox dye was used as reference.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DirectPCR Lysis Reagent (Tail)</td>
<td>Viagen Biotech</td>
<td>102-T</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Sigma-Aldrich</td>
<td>P6556</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>Ambion</td>
<td>AM9760G</td>
</tr>
<tr>
<td>2-Propanol</td>
<td>Sigma-Aldrich</td>
<td>I9516</td>
</tr>
<tr>
<td>Ethyl alcohol</td>
<td>Sigma-Aldrich</td>
<td>493538</td>
</tr>
<tr>
<td>Agarose</td>
<td>Sigma-Aldrich</td>
<td>A9539</td>
</tr>
<tr>
<td>TE Buffer</td>
<td>Thermo Fisher</td>
<td>12090015</td>
</tr>
<tr>
<td>Phusion DNA Polymerase.</td>
<td>New England Biolabs</td>
<td>M0530S</td>
</tr>
<tr>
<td>dNTPs</td>
<td>New England Biolabs</td>
<td>N0447S</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Forward Primer</td>
<td>IDT</td>
<td>TCCCTACACGACGCTCTTCCGATCT – Forward gene-specific primer (R1)</td>
</tr>
<tr>
<td>Reverse Primer</td>
<td>IDT</td>
<td>AGTTCAGACGTGTGCTCTTCCGATCT – Reverse gene-specific primer (R2)</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>CRISPR-Cas9 targeted Mice</td>
<td><a href="#bib5">Vasu et al., 2021</a></td>
<td>C57BL/6</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>QIAquick PCR purification kit</td>
<td>QIAGEN</td>
<td>28104</td>
</tr>
<tr>
<td>DNeasy Blood &amp; Tissue Kit</td>
<td>QIAGEN</td>
<td>69504</td>
</tr>
<tr>
<td>PrimeTime® Gene Expression Master Mix</td>
<td>IDT</td>
<td>1055772</td>
</tr>
<tr>
<td>MiSeq Reagent Kit v3</td>
<td>Illumina</td>
<td>MS-102-3003</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>UCSC Genome Browser</td>
<td>UCSC</td>
<td><a href="https://genome.ucsc.edu/">https://genome.ucsc.edu/</a></td>
</tr>
<tr>
<td>PrimerQuest Tool</td>
<td>IDT</td>
<td><a href="https://www.idtdna.com/PrimerQuest/Home/Index">https://www.idtdna.com/PrimerQuest/Home/Index</a></td>
</tr>
<tr>
<td>NEB Tm Calculator</td>
<td>New England Biolabs</td>
<td><a href="https://tmcalculator.neb.com/%23!/main">https://tmcalculator.neb.com/#!/main</a></td>
</tr>
<tr>
<td>Cas-Analyzer</td>
<td>(<a href="#bib4">Park et al., 2017</a>)</td>
<td><a href="http://www.rgenome.net/cas-analyzer/">http://www.rgenome.net/cas-analyzer/</a></td>
</tr>
<tr>
<td>OligoAnalyzer Tool</td>
<td>IDT</td>
<td><a href="https://www.idtdna.com/calc/analyzer">https://www.idtdna.com/calc/analyzer</a></td>
</tr>
<tr>
<td>Dye Selection Tool</td>
<td>IDT</td>
<td><a href="https://www.idtdna.com/site/order/qpcr/primetimeprobes/multiplex">https://www.idtdna.com/site/order/qpcr/primetimeprobes/multiplex</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Microcentrifuge</td>
<td>Eppendorf</td>
<td>5424R</td>
</tr>
<tr>
<td>Plate centrifuge</td>
<td>Midsci</td>
<td>C2000</td>
</tr>
<tr>
<td>Mini-centrifuge</td>
<td>Ohaus</td>
<td>5306</td>
</tr>
<tr>
<td>Thermo-mixer</td>
<td>Eppendorf</td>
<td>4376600</td>
</tr>
<tr>
<td>Thermocycler</td>
<td>Applied Biosystems</td>
<td>2700</td>
</tr>
<tr>
<td>NanoDrop 1000</td>
<td>Thermo Fisher Scientific</td>
<td>ND-1000</td>
</tr>
<tr>
<td>StepOnePlus™ Real-Time PCR System</td>
<td>Thermo Fisher Scientific</td>
<td>4376600</td>
</tr>
<tr>
<td>MiSeq® System</td>
<td>Illumina</td>
<td>SY-410-1003</td>
</tr>
<tr>
<td>Optical Adhesive Film</td>
<td>Applied Biosystems</td>
<td>4311971</td>
</tr>
<tr>
<td>96-well Fast plate</td>
<td>LPS</td>
<td>L238800</td>
</tr>
<tr>
<td>Adhesive Seal Applicator</td>
<td>Thermo Fisher Scientific</td>
<td>AB1391</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Preparation of Genotyping Master Mix:</p>
<p>Thaw PrimeTime® Gene Expression Master Mix and reference dye. Vortex Master Mix reagents thoroughly and add the appropriate amount of reference dye depending on the qPCR system. In our experiments, StepOnePlus™ Real-Time PCR System was used, and 40 μL of reference dye was added per mL of 2× Master Mix. Store at −20°C. Master mix can be stored for up to 6 months without any decrease in efficiency.</p>
<div class="note">
<span class="note-title">Note:</span> Certain qPCR systems, e.g., Agilent Mx4000P™ qPCR System, require low amount of reference dye. In such instruments, add 4 μl of reference dye per ml of 2× Master Mix.</div>
<p>Before each PCR run:</p>
<ul>
<li>Resuspend the assay probe by vortexing, then centrifuge briefly.</li>
<li>Resuspend thawed DNA samples by vortexing, then centrifuge briefly.</li>
<li>Thoroughly mix the Genotyping Master Mix.</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> Ethidium bromide is a DNA-intercalating agent and can cause mutations. It is potentially carcinogenic or teratogenic. Wear full personal protective equipment and handle with care in a designated working area.</div>
<p>Preparation of 20× probe mix:</p>
<ul>
<li>Centrifuge the primer and probe tubes at 750 <i>g</i> for 15 s to bring lyophilized material to the bottom of the tube.</li>
<li>Resuspend using TE buffer (10 mM Tris pH 8.0, 0.1 mM EDTA) to a stock of 100 μM. Calculate volumes based on nmol of synthesized product in the stock tube.</li>
<li>The desired concentration of primers and probe is 500 nM and 250 nM, respectively, in the reaction mix.</li>
<li>Mix the primers and probe to a 20× concentration, i.e., 10 μM and 5 μM, respectively. Make up the volume with TE buffer.</li>
<li>Briefly vortex and centrifuge at 750 <i>g</i> for 5 s.</li>
<li>Probe mix can be stored −20°C for up to 6 months without any decrease in efficiency.</li>
</ul>
<div class="alternatives"><span class="alternatives-title">Alternatives:</span></div>
<p>SYBR Safe DNA Gel Stain may be used as an alternative to ethidium bromide.</p>
<p>In our study, custom LNA probes were purchased from IDT, but they can be purchased from any commercial vendors that synthesize LNA oligonucleotides such as Qiagen. Primers and probes were made to a 20× stock, but they can be made to 40× or 10× without effecting qPCR efficiency. We used a StepOne Plus qPCR instrument from Applied Biosystems. Other RT-PCRs can be programmed to yield similar results. Based on the facilities available at the Genomics Core of Case Western Reserve University (<a href="https://case.edu/medicine/genomics/pricing">https://case.edu/medicine/genomics/pricing</a>), we used MiSeq v3 flow cell as a Paired-End (PE) 250 cycles run (MiSeq250). Based on the region of interest to be sequenced, and core facility available, other modes of MiSeq can be utilized, e.g., PE 75 cycles or PE 150 cycles.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Here we describe two methods to assess CRISPR-Cas9-generated mutations. The MiSeq250 method can characterize both NHEJ- and HDR-generated mutations such as point mutations, and short insertions and deletions. The LNA probe qPCR described here requires prior knowledge of the kind of mutations expected such as those generated by HDR, or identified by MiSeq or by other sequencing methods. Both methods require extraction and preparation of very high-quality DNA.</p>
<h3 id="sec3.1">DNA extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 h</div>
<ol>
<li>The procedure below describes high-yield purification of genomic DNA from mouse tissues.
<ol type="a">
<li>Clip ∼0.5 cm of mouse tail, or other tissue, e.g., ear punch, in compliance with procedures approved by the institute's animal care and handling guidelines and research animal protocols.</li>
<li>Add 200 μL DirectPCR Lysis Reagent containing freshly prepared Proteinase K (0.25–0.5 mg/mL).
<div class="note">
<span class="note-title">Note:</span> Although 200 μl DirectPCR is usually sufficient for complete lysis of ∼0.5 cm tail, poor DNA recovery can be improved by increasing the lysis reagent to 300 μl to give better mixing efficiency.</div>
</li>
<li>Incubate tubes for 16 h at 55°C.
<div class="optional">
<span class="optional-title">Optional:</span> For increased DNA yields, rotate tubes in rotating hybridization oven.</div>
</li>
<li>Inactivate Proteinase K by incubating tubes at 85°C for 45 min.</li>
<li>Centrifuge at 12,000 <i>g</i> for 30 s to remove tissue debris.</li>
<li>Add NaCl to a final concentration of 250 mM, and then add 0.7 volume of isopropanol.</li>
<li>Centrifuge at 12,000 <i>g</i> for 5 min at 4°C. Discard the supernatant.</li>
<li>Wash the DNA pellet with 1 mL of 70% ethanol. Centrifuge at 12,000 <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>Air-dry the pellet for 5–10 min at 22°C. Dissolve in 100 μL of nuclease-free water.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Determine DNA concentration, and 260/280 nm and 260/230 nm ratios by Nanodrop. A 260/280 nm ratio of 1.8 and a 260/230 nm ratio of 2.0–2.2 is considered to be high-quality DNA with low protein contamination. Dilute DNA to 225 ng/μL and store at −20°C. (See <a href="#troubleshooting">troubleshooting 1</a>)</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> DNA can be stored at −20°C without significant reduction in quality.</div>
<div class="note">
<span class="note-title">Note:</span> Genomic DNA also can be isolated by Qiagen DNeasy Blood and Tissue Kit according to the manufacturer’s instructions (<a href="https://www.qiagen.com/br/products/top-sellers/dneasy-blood-and-tissue-kit/">https://www.qiagen.com/br/products/top-sellers/dneasy-blood-and-tissue-kit/</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Crude DNA extracts from rapid protocols, e.g., from heat-inactivated tissue lysates (Tail/toe/ear lysis buffer with Proteinase K) may not work reliably.</div>
<h3 id="sec3.2">Generation of amplicon for MiSeq250</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<ol start="3">
<li>The following steps describe amplification of target loci for MiSeq250 using Phusion DNA polymerase.
<ol type="a">
<li>Assemble all reaction components on ice.
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>to 50 μL</td>
</tr>
<tr>
<td>5× Phusion HF</td>
<td>1 X</td>
<td>10 μL</td>
</tr>
<tr>
<td>10 mM dNTPs</td>
<td>200 μM</td>
<td>1.0 μL</td>
</tr>
<tr>
<td>10 μM Forward primer</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>10 μM Reverse primer</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Genomic DNA</td>
<td>225 ng</td>
<td>variable</td>
</tr>
<tr>
<td>DMSO (optional)</td>
<td>2%</td>
<td>1.0 μL</td>
</tr>
<tr>
<td>Phusion DNA polymerase</td>
<td>1 Unit</td>
<td>0.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>50</b> μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Because Phusion DNA polymerase has 3′→ 5′ exonuclease activity, it is critical to add the polymerase last to prevent primer degradation. The volume of genomic DNA added should be less than 20% of the total reaction volume to increase PCR yield.</div>
<div class="note">
<span class="note-title">Note:</span> 5× Phusion HF Buffer and 5× Phusion GC Buffer are provided with the enzyme. The protocol describes use of HF buffer. For GC-rich templates or primers with secondary structure, GC buffer can improve reaction performance.</div>
</li>
<li>Gently mix the reaction components. Do a quick spin to collect the components at the tube bottom.</li>
<li>Transfer PCR tubes from ice to a preheated PCR instrument with heated lid, and begin thermocycling with the following conditions:
<table id="undtbl3">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">30 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>58°C–68°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">n/a</td>
</tr>
</tbody>
</table>
</li>
<li>Transfer PCR tubes to ice and run the PCR products on a 1% (wt/vol) agarose gel. (See <a href="#troubleshooting">troubleshooting 2</a>)</li>
<li>Purify the PCR product using a PCR purification kit, according to the manufacturers’ instructions (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/qiaquick-pcr-purification-kit/">https://www.qiagen.com/us/products/discovery-and-translational-research/qiaquick-pcr-purification-kit/</a>).</li>
</ol>
</li>
</ol>
<ol start="4">
<li>The PCR products are processed by a genomic service (<a href="https://case.edu/medicine/genomics/">https://case.edu/medicine/genomics/</a>) for secondary PCR involving barcode addition followed by MiSeq. Barcoding refers to addition of unique indices involving short sequence of DNA to the PCR fragments during library preparation. This step allows for multiple libraries to be pooled and sequenced simultaneously during a sequencing run, i.e., multiplexing. The sequencing reads from the pooled libraries are sorted based on the barcode sequence, i.e., demultiplexing. We used a genomics core service with a MiSeq v3 flow cell as a Paired-End (PE) 250 cycle run, i.e., 250 cycles in each direction or 500 cycles total are sequenced.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other MiSeq run modes are compatible with CRISPR edit detection such as PE 75 cycles or PE 150 cycles based on the region of interest, length of amplicon, and the services available.</div>
<h3 id="sec3.3">MiSeq250 data analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 h</div>
<p>Analysis of reads from next-generation sequencing is done using Cas-Analyzer (<a href="#bib4">Park et al., 2017</a>) available at <a href="http://www.rgenome.net/cas-analyzer/">http://www.rgenome.net/cas-analyzer/</a>. Alternate platforms supporting next-generation sequencing datasets, e.g., CRISPResso2, can be used.</p>
<ol start="5">
<li>The format of the raw output data from the MiSeq platform is usually FastQ or gzip-compressed, and both data types are accepted by Cas-Analyzer. Download FastQ files for both reads to a local folder.</li>
<li>Input parameters:
<ol type="a">
<li>Upload sequencing data.</li>
<li>Select nuclease type → single nuclease.</li>
<li>Select nuclease → from the appropriate drop-down menu. In our study, SpCas9 from <i>Streptococcus pyogenes</i>: 5′-NGG-3′ was selected.</li>
<li>Target DNA sequence → input the sgRNA sequence used for CRISPR-Cas9 from 5′ to 3′ without the PAM sequence.</li>
<li>Input donor DNA sequence → enter the donor sequence used for HDR. This is an optional parameter to analyze recombined sequences.</li>
<li>Input comparison range. The default is 70 and can be changed to limit the query sequence comparison to the specified number with the input sequence. (See <a href="#troubleshooting">troubleshooting 3</a>)</li>
<li>Input minimum frequency. The default is 1 and can be changed to filter the number of reads below the set threshold.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">LNA probe qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Before you begin, see ‘<a href="#materials-and-equipment">materials and equipment</a>’ for preparation of Genotyping Master Mix and 20× Probe mix.</p>
<p>The following steps describe allelic discrimination using the StepOne plus qPCR system. (See <a href="#troubleshooting">troubleshooting 4</a>)</p>
<ol start="7">
<li>Prepare the reaction mix.
<ol type="a">
<li>Determine number of reactions to be performed for each assay, including positive and negative controls.
<ol type="i">
<li>Test samples – samples of unknown genotype.</li>
<li>No-template control – DNase-free water (two reactions per plate) to correct for background fluorescence.</li>
<li>Positive control – positive controls for each allele. (See <a href="#troubleshooting">troubleshooting 5</a>)</li>
</ol>
</li>
<li>Add following components for each reaction:
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2× Genotyping Master mix</td>
<td>1 ×</td>
<td>5.0 μL</td>
</tr>
<tr>
<td>20× Probe mix</td>
<td>1 ×</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>-</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5.5 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Allow 10% overage to account for pipetting errors and to avoid air bubbles.</div>
</li>
<li>Briefly vortex.</li>
<li>Quickspin the tubes to remove air bubbles.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Prepare the reaction plate.
<ol type="a">
<li>Dilute each DNA sample, including controls, in nuclease-free water to 22.25 ng/μL and add 4.5 μL per well.
<div class="note">
<span class="note-title">Note:</span> All wells using the probe assay must contain same amount of DNA. A final concentration of at least 20 ng/well is recommended.</div>
</li>
<li>Add 5.5 μL of reaction mix to each well of the 96-well plate.</li>
<li>Seal the plate with adhesive film, then centrifuge briefly to remove air bubbles and to drive the reaction mix to the bottom.
<div class="note">
<span class="note-title">Note:</span> Use a compression pad or other sealer to firmly seal the optical film.</div>
</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Perform qPCR.
<ol type="a">
<li>Instrument and set-up properties using StepOne Software v2.3.
<ol type="i">
<li>Select the instrument type and the plate format (e.g., 96-well).</li>
<li>Choose genotyping as experiment type.</li>
<li>Select the reagent type – Taqman reagents. This selection uses default settings from the built-in Taqman protocol.</li>
<li>Choose the ramp speed.</li>
</ol>
</li>
<li>Plate setup. Define the allele-specific probe and samples for the experiment.
<ol type="i">
<li>Define the fluorescence reporter and the quencher.</li>
<li>Assign the probes to selected wells.</li>
<li>Assign the samples to selected wells.</li>
</ol>
</li>
<li>Run method. Edit the default run method with appropriate T<sub>m</sub>. Include Pre-PCR read to collect the background fluorescence.</li>
</ol>
</li>
<li>Analysis.
<ol type="a">
<li>Analyze the plate layout to identify any flags and errors identified. Review the QC summary. (See <a href="#troubleshooting">troubleshooting 6</a>)</li>
<li>Select amplification plot to show amplification curves in all wells.</li>
<li>Click allele discrimination plot (which is a clustering algorithm) to report the normalized intensities of each probe in each well, and to generate sample data clusters. Briefly,
<ol type="i">
<li>Define the fluorescence reporter and the quencher. Click ‘Experiment’, Select Analysis → Allele Discrimination Plot</li>
<li>If the data is not analyzed by default, click ‘Analyze’.</li>
<li>Click to view the ‘Plate layout tab’ to see and select the wells of interest.</li>
<li>For the wells that are assigned unknown, check for amplification.</li>
</ol>
</li>
<li>Wells with desired genotype are selected for downstream applications.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Allele Discrimination Plot does not assign genotypes when only one type of allele is present in an experiment.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>MiSeq250 analysis: Results summary tabulates number of wild-type reads and insertions and deletions (indels) (<a href="#fig4">Figure 4</a>). Indels and HDR frequency are calculated and position of indels are shown graphically. The number of reads with each type of indel is shown as a downloadable alignment. Based on the MiSeq analysis of number of reads of a desired mutation, mice with highest frequency of mutation are used for back-cross with a wild-type strain. It’s desirable to have greater than 10% read frequency which will result in 1 pup with the desired mutation out of 20 after backcross. For LNA-probe qPCR, mice with mutant probe amplification are selected for further breeding. The LNA-qPCR distinguishes alleles that differ by a single nt, permitting definitive identification of all 3 mouse genotypes from tail DNA (<a href="#fig5">Figure 5</a>). We have successfully used LNA probe qPCR to genotype single nt insertion, two point mutations separated by 7 nt, and a single nt point mutation. Other scenarios, e.g., <math> <mrow> <mo>≥</mo> </mrow> </math>2 nt insertions and deletions, have not been tested.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1030-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Screenshot of Cas-Analyzer software depicting the result summary</div>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1030-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. LNA-based detection of mutant allele</div>
<p>(A) Segregation of DNA from homozygous wild-type (wt) and heterozygous wt/mutant (mut) mice is shown in an allele discrimination plot.</p>
<p>(B) qPCR curves showing amplification and selective detection using wt (left) and mut (right) probe sets.</p>
<p>Figure reprinted with permission from <a href="#bib5">Vasu et al., 2021</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>A potential limitation of the MiSeq method of identification is the requirement for a costly, next-generation sequencing facility. Also, next-generation sequence facilities simultaneously run hundreds of samples, and there can be a relatively long wait time until sufficient sample orders are obtained to permit a run. The MiSeq250 method described here can characterize both NHEJ- and HDR-generated mutations such as point mutations and short insertions and deletions. The LNA-qPCR described here requires prior knowledge of the kind of mutation expected such as those generated by HDR frequently used for knock-in studies. A technical limitation of the LNA-qPCR method is the requirement for a qPCR system with a multiplexing option. Also, optimization of primer design and qPCR parameter determination might be required.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Poor quality of genomic DNA and/or low yield. A low 260/230 ratio indicates chemical contaminants that absorb at 230 nm such as EDTA, phenol, guanidine HCl, and carbohydrates. A low 260/280 ratio indicates protein contaminants. Many factors affect the quality of genomic DNA and yield including incorrect amount of source tissue, tissue storage, or DNA isolation technique (step 2 ).</p>
<h3 id="sec6.2">Potential solution</h3>
<ol>
<li>In case of incomplete digestion of the tissue sample, cut samples into extremely small pieces for rapid and efficient lysis. Allow the sample to remain in the lysis buffer during incubation steps. Large tissue clumps should not be observed after digestion.</li>
<li>Tissue samples stored for long periods at 4°C or −20°C will show degradation of genomic DNA. For samples stored for extended periods, freeze rapidly in liquid nitrogen and store at −80°C.</li>
</ol>
<h3 id="sec6.3">Problem 2</h3>
<p>No PCR amplification. Incorrect design of the primers, or quality of genomic DNA, can influence PCR amplicon generation (step 3).</p>
<h3 id="sec6.4">Potential solution</h3>
<ol>
<li>Analyze primer design parameters and set up a gradient PCR to identify correct T<sub>m</sub> of the primers. Addition of DMSO or use of Phusion GC buffer can improve amplicon generation if the genomic region contains GC-rich sequences.</li>
<li>Improve DNA quality (See <a href="#troubleshooting">troubleshooting 1</a>). As an alternative for DNA isolation, use Qiagen DNeasy Blood and Tissue Kit (<a href="https://www.qiagen.com/br/products/top-sellers/dneasy-blood-and-tissue-kit/">https://www.qiagen.com/br/products/top-sellers/dneasy-blood-and-tissue-kit/</a>).</li>
</ol>
<h3 id="sec6.5">Problem 3</h3>
<p>Cas-Analyzer does not detect significant reads in the pair-end mode, or gives an error message indicating no reads detected (step 6).</p>
<h3 id="sec6.6">Potential solution</h3>
<ol>
<li>Run the program in single-read mode and load each read separately for every sample.</li>
<li>Adjust default parameters, e.g., input comparison range, to detect more reads.</li>
<li>To analyze reads, use alternate platform that supports next-generation sequencing datasets, e.g., CRISPResso2 (<a href="https://crispresso.pinellolab.partners.org/submission">https://crispresso.pinellolab.partners.org/submission</a>) (<a href="#bib1">Clement et al., 2019</a>).</li>
</ol>
<h3 id="sec6.7">Problem 4</h3>
<p>Inadequate or absent LNA-qPCR amplification. Incorrect primer design and poor genomic DNA quality can adversely affect LNA-qPCR amplification (step 7).</p>
<h3 id="sec6.8">Potential solution</h3>
<ol>
<li>Re-evaluate LNA primer design parameters and set up a gradient PCR to determine T<sub>m</sub> of the primers.</li>
<li>Improve DNA quality (See <a href="#troubleshooting">Troubleshooting 1</a>). Alternatively, use Qiagen DNeasy Blood and Tissue Kit to isolate DNA (<a href="https://www.qiagen.com/br/products/top-sellers/dneasy-blood-and-tissue-kit/">https://www.qiagen.com/br/products/top-sellers/dneasy-blood-and-tissue-kit/</a>).</li>
</ol>
<h3 id="sec6.9">Problem 5</h3>
<p>Incorrect genotype identification in controls; incorrect selection of fluorophores appropriate for a particular qPCR instrument could cause fluorescence "bleed-through" (step 7).</p>
<h3 id="sec6.10">Potential solution</h3>
<ol>
<li>Identify the correct fluorophore in the qPCR instrument manual. If necessary, communicate with manufacturer for correct wavelength detection.</li>
<li>Use IDT's multiplex dye selection tool to determine acceptable fluorophores (<a href="https://www.idtdna.com/site/order/qpcr/primetimeprobes/multiplex">https://www.idtdna.com/site/order/qpcr/primetimeprobes/multiplex</a>).</li>
<li>Run probes separately to identify the fluorophore quencher combination with highest signal above background. Use the same quencher type in assays that will be multiplexed.</li>
</ol>
<h3 id="sec6.11">Problem 6</h3>
<p>Multiple quality control flags after the analysis. Improper qPCR set up can lead to flagging of wells (step 10).</p>
<h3 id="sec6.12">Potential solution</h3>
<ol>
<li>Increase the assay volume to minimize pipetting errors.</li>
<li>Set up reactions on ice or use a PCR cooler to maintain 0°C - 4°C temperature for the duration.</li>
</ol>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Clement, K., Rees, H., Canver, M.C., Gehrke, J.M., Farouni, R., Hsu, J.Y., Cole, M.A., Liu, D.R., Joung, J.K., Bauer, D.E., et al. (2019). CRISPResso2 provides accurate and rapid genome editing sequence analysis. Nat. Biotechnol. <i>37</i>, 224-226. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00491-3/sref1">View at publisher</a></p>
<p id="bib2">McTigue, P.M., Peterson, R.J., and Kahn, J.D. (2004). Sequence-dependent thermodynamic parameters for locked nucleic acid (LNA)-DNA duplex formation. Biochemistry <i>43</i>, 5388-5405. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00491-3/sref2">View at publisher</a></p>
<p id="bib3">Owczarzy, R., You, Y., Groth, C.L., and Tataurov, A.V. (2011). Stability and mismatch discrimination of locked nucleic acid-DNA duplexes. Biochemistry <i>50</i>, 9352-9367. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00491-3/sref3">View at publisher</a></p>
<p id="bib4">Park, J., Lim, K., Kim, J.S., and Bae, S. (2017). Cas-analyzer: an online tool for assessing genome editing results using NGS data. Bioinformatics <i>33</i>, 286-288. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00491-3/sref4">View at publisher</a></p>
<p id="bib5">Vasu, K., Ramachandiran, I., Terenzi, F., Khan, D., China, A., Khan, K., Chechi, A., Baleanu-Gogonea, C., Gogonea, V., and Fox, P.L. (2021). The zinc-binding domain of mammalian prolyl-tRNA synthetase is indispensable for catalytic activity and organism viability. iScience <i>24</i>, 102215. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00491-3/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to, and will be fulfilled by, the lead contact, Paul Fox (<a href="mailto:foxp@ccf.org">foxp@ccf.org</a>).</p>
<h4>Materials availability</h4>
<p>All primers generated in this study will be made available on request but may require a completed Materials Transfer Agreement if there is a potential for commercial application.</p>
<h4>Data and code availability</h4>
<p>This study did not generate data sets or code for any algorithms.</p>
<h3>Acknowledgments</h3>
<p>We thank Simone Edelheit and the Case Western Reserve University Genomics Core for MiSeq250 services and discussions. This project was supported by <a href="https://doi.org/10.13039/100000002">N.I.H.</a> grants P01 HL076491, R01 DK123236, R01 DK118085, and R01 AG067146 (to P.L.F.) and by a Clinical and Translational Science Collaborative (CTSC) Core Utilization Pilot Grant Award from <a href="https://doi.org/10.13039/100008564">Case Western Reserve University School of Medicine</a> (to P.L.F.).</p>
<h3>Author contributions</h3>
<p>Conceptualization, methodology, investigation, analysis, writing – original draft, writing - review &amp; editing, K.V; Conceptualization, supervision, project administration, funding acquisition, writing - review &amp; editing, P.L.F.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>