<article>
<section>
<h1 id="header">Tracking neuronal motility in live murine retinal explants</h1>
<p><time datetime="2021-12-10">Published: December 17, 2021</time></p>
<p>Nozie D. Aghaizu,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn3">4</a>,<a href="#cor1">*</a></sup> Katherine M. Warre-Cornish,<sup><a href="#aff1">1</a></sup> Martha R. Robinson,<sup><a href="#aff1">1</a></sup> Robin R. Ali,<sup><a href="#aff1">1</a>,<a href="#fn2">3</a></sup> and Rachael A. Pearson<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#fn4">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>University College London, Institute of Ophthalmology, London EC1V 9EL, UK</p>
<p id="fn1"><sup>2</sup>Present address: Dementia Research Institute, University College London, The Cruciform Building, Gower Street, London WC1E 6BT, UK</p>
<p id="fn2"><sup>3</sup>Present address: Centre for Cell and Gene Therapy, King’s College London, 8th Floor Tower Wing, Guy’s Hospital, London SE1 9RT, UK</p>
<p id="fn3"><sup>4</sup>Technical contact</p>
<p id="fn4"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:nozie.aghaizu@ucl.ac.uk">nozie.aghaizu@ucl.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:rachael.pearson@kcl.ac.uk">rachael.pearson@kcl.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101008">10.1016/j.xpro.2021.101008</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The developing retina undergoes dynamic organizational changes involving significant intra-retinal motility of the encompassing cells. Here, we present a protocol for tracking retinal cell motility in live explanted mouse retinae. Although originally applied to rod and cone photoreceptors, this strategy is applicable to any fluorescently labeled cell in mouse retinae and other similar experimental retinal models. Careful tissue handling is critical for the successful acquisition of high-quality live imaging data. Further instructions for semi-automated <i>in silico</i> data handling are provided.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.celrep.2021.109461">Aghaizu et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for cell motility tracking in live murine retinal explants</li>
<li>Fluorescent labelling of retinal cells using transgenic animals or by transfection</li>
<li>Live-imaging set-up of murine retinae in a 2-photon imaging system</li>
<li>
<i>In silico</i> tracking and quantification of retinal cell motility</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1226-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was designed to track motions of fluorescently labeled retinal cells. In <a href="#bib1">Aghaizu et al. (2021)</a>, cellular labeling was achieved either by using transgenic mouse lines expressing cell-type specific fluorescent reporters or by electroporation of acutely explanted retinae with an expression plasmid driving the expression of a fluorescent protein.</p>
<p>All mice in preparation of this manuscript were used according to the NC3R ARRIVE guidelines. Both male and female mice were used in this study without discrimination. All animal studies were carried out under the Animals (Scientific Procedures) Act 1986 under a project license PPL 70/8120 issued by the UK Government Home Office and conducted in accordance with protocols approved by the Animal Welfare and Ethics Committee of the UCL Institute of Ophthalmology. All animals were killed by cervical dislocation performed by trained personnel (approved under Schedule 1 as a method of humane killing). All efforts were made to minimize the number and suffering of animals used in these experiments.</p>
<p>Male and female mice were group housed in the animal facility at University College London on a standard 12-h light/dark cycle at the same light levels throughout the experimental period. Animals were kept in individually ventilated cages on animal grade wood chips and given access to nesting material and food and water ad libitum.</p>
<h3 id="sec1.1">Preparation of transgenic mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5–4.5 weeks</div>
<ol>
<li>Cell-type specific reporter mice are used for the purpose of having fluorescently labeled rod or cone photoreceptors <i>in vivo.</i> Timed-matings need to be set up 2.5–4.5 weeks prior to retinal live imaging depending on the developmental stage of interest (typically between embryonic day (E) 16.5 and postnatal day (P) 10.5).
<ol type="a">
<li>For fluorescently labeled rod photoreceptors, time-mate <i>Nrl.GFP</i><sup><i>+/+</i></sup> mice (<a href="#bib2">Akimoto et al., 2006</a>). These mice are suitable for homozygotic breeding.</li>
<li>For fluorescently labeled cone photoreceptors, time-mate <i>Chrnb4.EGFP</i> mice (<a href="#bib4">Gong et al., 2003</a>). These mice should be bred as hemizygotes.</li>
<li>For Cre-expressing rod photoreceptors, time-mate <i>Nrl.Cre</i><sup><i>+/−</i></sup> mice (<a href="#bib3">Brightman et al., 2016</a>). These mice should be bred as hemizygotes.</li>
</ol>
</li>
<li>Successful mating is typically accompanied by the presence of a vaginal plug, which marks embryonic day (E) 0.5 following overnight mouse pairing for 14–16 h. The gestation period in mice is usually 19–21 days (strain dependent). Subsequently isolate and live image retinae at desired time point (see ‘<a href="#step-by-step-method-details">step-by-step method details</a>’).</li>
</ol>
<h3 id="sec1.2">Preparation of transfection-based cellular labelling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5–4.5 weeks including husbandry; 2.5 h for dissection, electroporation and culture setup</div>
<ol start="3">
<li>Photoreceptor fluorescent labeling can also be achieved by introducing an expression plasmid of choice into the desired target retinal cells. We used a conditional strategy to limit fluorescent protein expression to rod photoreceptors only. This was achieved by electroporating a Cre-sensitive expression plasmid into acutely explanted retinae from <i>Nrl.Cre</i><sup><i>+/−</i></sup> mice as described below, although this can also be achieved by <i>in vivo</i> electroporation (<a href="#bib5">Matsuda and Cepko, 2004</a>) or AAV viral transduction (<a href="#bib1">Aghaizu et al., 2021</a>).
<ol type="a">
<li>Dissect out retinae from P0.5–1.5 mice obtained from timed mated pregnancies (see ‘<a href="#sec1.1">preparation of transgenic mice</a>’; (1.c)) in ice cold dissection medium (<a href="#fig1">Figure 1</a>A). Remove all non-neural retinal tissues (cornea, sclera, iris, choroid, optic nerve, retinal pigment epithelium, and ciliary body) with exception of the lens, which is left in place for increased tissue stability (<a href="#fig1">Figure 1</a>B).
<div class="critical">
<span class="critical-title">Critical:</span> To avoid contamination, dissection tools (#5 Dumont forceps, Vannas scissors) are to be sterilized with 70 % ethanol before dissection. Furthermore, aim to perform the majority of ‘Preparation steps no. 3’ in a sterile tissue culture hood.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Care should be taken to avoid damaging retinae during dissection as tissue disruptions introduced during this step tend to become amplified during <i>in vitro</i> culture. We found that dissected retinae are most gently transferred by using sterile plastic Pasteur pipettes (supplied with electroporation cuvettes) with a manually widened opening (<a href="#fig1">Figure 1</a>I).</div>
</li>
<li>Transfer retinae to electroporation cuvette containing a 1 μg/μL plasmid DNA solution in PBS, prepared as described in ‘Materials and equipment’ (e.g., <i>shDync1h1/EGFP</i>; <a href="#bib1">Aghaizu et al., 2021</a>) and incubate at room temperature (18°C–21°C) for 5 min.</li>
<li>Orientate retinae within the cuvette such that the apical and lens sides of the retina are aligned with the cuvette electrodes; for optimal electroporation efficiency the apical surface is to be juxtaposed to the anode (<a href="#fig1">Figure 1</a>J).</li>
<li>Use a pulse generator to electroporate retinae with 5 × 30 V square pulses of 50 ms duration and with 950 ms intervals.
<div class="critical">
<span class="critical-title">Critical:</span> Avoid direct contact between electrodes and retinae during electroporation as this may result in substantial tissue damage.</div>
</li>
<li>Allow retinae to recover in pre-equilibrated (37°C/5% CO<sub>2</sub>) dissection medium for 5 min.</li>
<li>Transfer retinae to pre-equilibrated (37°C/5% CO<sub>2</sub>) culture medium for 5 min.</li>
<li>For <i>in vitro</i> retinal explant cultures, transfer up to 3 retinae lens side down onto a Whatman® Nuclepore™ Track-Etched Membrane floating on 2 mL of culture medium in a 6-well tissue culture plate and culture at 37°C/5% CO<sub>2</sub> in a sitting drop of culture medium; 1 retina per drop (<a href="#fig1">Figure 1</a>J).</li>
<li>Re-feed each retina by adding 25 μL culture medium to sitting drop every other day until ready for live imaging.
<div class="note">
<span class="note-title">Note:</span> Transfection efficiency can be assessed on standard epifluorescence microscopes, fluorescent protein permitting. In general, latency times between electroporation and visual fluorescent protein detection depend to a large extent, albeit not exclusively, on promoter strength and fluorescent protein brightness. In our experience, when expression of <i>EGFP</i> and <i>DsRed</i> was driven by a strong promoter (e.g. <i>CMV</i>, <i>CAG</i> or rod cell-specific <i>Nrl</i> promoter (<a href="#bib2">Akimoto et al., 2006</a>)), expression was detectable after 1–2 days and firmly established at 3 days post electroporation.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1226-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Retinal dissection and tissue handling</div>
<p>(A) Enucleated eye from a P0 <i>Nrl.Cre</i><sup><i>+/−</i></sup> mouse.</p>
<p>(B) Neural retina with lens left intact isolated from eyeball.</p>
<p>(C) Lens (cyan arrowhead) separated from neural retina (magenta arrowhead).</p>
<p>(D) Radial cuts (magenta arrowheads) applied to the neural retina. Tissue was also transferred onto the edge of the standard microscopy glass slide.</p>
<p>(E) Neural retina flattened on a glass slide by unfolding the 4 ‘petals’. Excess liquid was removed (compared with (D)).</p>
<p>(F) 0.45 μm MF-Millipore™ nitrocellulose membrane placed on top of flattened neural retina.</p>
<p>(G) Membrane-mounted retina being transferred back into RPMI 1640 medium by dipping the edge of the glass slide into medium. The membrane will gradually become visibly hydrated during this step (magenta arrowhead).</p>
<p>(H) Flat mounted neural retina (magenta dotted outline) on nitrocellulose membrane. Membrane was inverted compared to (G).</p>
<p>(I) Plastic Pasteur pipette manually widened using sterilized scissors.</p>
<p>(J) Schematic depicting optimal orientation of neural retina relative to electroporation electrodes. Magenta plastic circles symbolize plasmid used for electroporation. The subsequent step depicts the neural retinal culture setup, whereby the neural retina is placed on a floating Whatman® Nuclepore™ Track-Etched Membrane in a sitting drop. Scale bars, 1 mm (A–E), 10 mm (F–H).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM/F-12, HEPES, L-Glutamine</td>
<td>Gibco</td>
<td>Cat#11330057</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Gibco</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>Taurine</td>
<td>Sigma-Aldrich</td>
<td>Cat#T0625</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Gibco</td>
<td>Cat#26140079</td>
</tr>
<tr>
<td>10× PBS</td>
<td>Gibco</td>
<td>Cat#70011044</td>
</tr>
<tr>
<td>Ultrapure dH<sub>2</sub>O (DNAse/RNAse free)</td>
<td>Invitrogen</td>
<td>Cat#11538646</td>
</tr>
<tr>
<td>RPMI 1640, phenol red free</td>
<td>Gibco</td>
<td>Cat#11835030</td>
</tr>
<tr>
<td>DMEM<sup>gfp</sup>-2 live imaging medium</td>
<td>Evrogen</td>
<td>Cat#MCK02</td>
</tr>
<tr>
<td>Paraffin oil</td>
<td>Supelco</td>
<td>Cat#76235</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Qiagen plasmid kit (maxi or mega scale)</td>
<td>Qiagen</td>
<td>Cat#12162/12181</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>Nrl.GFP</i><sup><i>+/+</i></sup> mice (B6.Cg-Tg(Nrl-EGFP)1Asw/J)</td>
<td>kind gift of A. Swaroop</td>
<td>RRID:IMSR_JAX:02 1232</td>
</tr>
<tr>
<td>
<i>Nrl.Cre</i><sup><i>+/−</i></sup> mice (<i>C57BL/6J-Tg(Nrl-cre)1Smgc/J</i>)</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:02 8941</td>
</tr>
<tr>
<td>
<i>Chrnb4.EGFP</i> mice (Tg(Chrnb4EGFP)CL200Gsat/Mmnc)</td>
<td>MMRRC</td>
<td>RRID:MMRRC_0002 59-UNC</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pD10 mU6.TL-shCtrl-TL.shDync1h1 / CMV.FL-pA-FL.EGFP (abbreviated: <i>shDync1h1/EGFP</i>)</td>
<td><a href="#bib1">Aghaizu et al. (2021)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientificsoftware/prism/">https://www.graphpad.com/scientificsoftware/prism/</a></td>
</tr>
<tr>
<td>Fiji/ImageJ</td>
<td>National Institutes of Health (NIH)</td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td>IMARIS</td>
<td>Bitplane</td>
<td><a href="http://www.bitplane.com/imaris">http://www.bitplane.com/imaris</a></td>
</tr>
<tr>
<td>MATLAB</td>
<td>MathWorks</td>
<td><a href="https://www.mathworks.com/product%20s/matlab.html">https://www.mathworks.com/product s/matlab.html</a></td>
</tr>
<tr>
<td>Custom-written Matlab scripts</td>
<td><a href="#bib1">Aghaizu et al. (2021)</a></td>
<td><a href="https://github.com/RPearsonLab/Photoreceptor_tracking">https://github.com/RPearsonLab/Photoreceptor_tracking</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Clean bench</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Tissue culture hood</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Tissue culture incubator</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>1.0 mL Pasteur Pipette</td>
<td>Starlab</td>
<td>Cat#E1414-1111</td>
</tr>
<tr>
<td>Dissection microscope</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>6-well tissue culture plate (sterile)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#150239</td>
</tr>
<tr>
<td>Dumont forceps #5 (2×)</td>
<td>Fisher Scientific</td>
<td>Cat#15483542</td>
</tr>
<tr>
<td>Vannas Scissors, Super fine, 8 cm, straight</td>
<td>Fisher Scientific</td>
<td>Cat#50-822-579</td>
</tr>
<tr>
<td>30 G microlance 3 needle</td>
<td>BD</td>
<td>Cat#304000</td>
</tr>
<tr>
<td>Pulse generator</td>
<td>BTX</td>
<td>Cat#ECM 830</td>
</tr>
<tr>
<td>Electroporation cuvette (2 mm gap size)</td>
<td>BTX</td>
<td>Cat#45-0126</td>
</tr>
<tr>
<td>Whatman® Nuclepore™ Track-Etched Membranes (0.2 μm, polycarbonate)</td>
<td>Merck</td>
<td>Cat#WHA10417006</td>
</tr>
<tr>
<td>Microscopic slides</td>
<td>Fisher Scientific</td>
<td>Cat#11562203</td>
</tr>
<tr>
<td>MF nitrocellulose membrane filters</td>
<td>Merck Millipore</td>
<td>Cat#HAWP01300</td>
</tr>
<tr>
<td>Ø5.5 cm tissue culture petri dish</td>
<td>VWR</td>
<td>Cat#734-2318</td>
</tr>
<tr>
<td>Dow Corning® high-vacuum silicone grease</td>
<td>Sigma-Aldrich</td>
<td>Cat#Z273554</td>
</tr>
<tr>
<td>Platinum ring tissue anchor</td>
<td>e.g., Warner Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>Live imaging microscope with 2-photon and environmental control capabilities (temperature, CO<sub>2</sub>)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>DNA solution for electroporation</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Plasmid DNA</td>
<td>1 μg/μL</td>
<td>X</td>
</tr>
<tr>
<td>PBS (10×)</td>
<td>1×</td>
<td>15 μL</td>
</tr>
<tr>
<td>dH<sub>2</sub>O (DNAse/RNAse free)</td>
<td>N/A</td>
<td>Top up to 150 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>150 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">DNA solution was freshly prepared prior to electroporation.</p>
<p>Note that highly concentrated plasmid DNA (ideally &gt; 1 μg/μL) is required and should be produced using Qiagen plasmid kits or similar commercially available kits.</p>
<table id="undtbl2">
<caption>Dissection medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Taurine working solution (50 μM)</td>
<td>150 nM</td>
<td>90 μL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (10,000 U/mL)</td>
<td>10 U/mL</td>
<td>30 μL</td>
</tr>
<tr>
<td>DMEM/F-12, HEPES, L-Glutamine</td>
<td>N/A</td>
<td>29.88 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>30 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Dissection medium was freshly prepared prior to retinal harvest but should be stable at 4°C for 1 week.</p>
<table id="undtbl3">
<caption>Culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Fetal bovine serum (100 %)</td>
<td>5 %</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Taurine working solution (50 μM)</td>
<td>150 nM</td>
<td>30 μL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (10,000 U/mL)</td>
<td>10 U/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>DMEM/F-12, HEPES, L-Glutamine</td>
<td>N/A</td>
<td>9.46 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Culture medium can be stored at 4°C for 1 week.</p>
<table id="undtbl4">
<caption>Live imaging medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Rutin (100×)</td>
<td>1×</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>DMEM<sup>gfp</sup>-2</td>
<td>N/A</td>
<td>9.9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Live imaging medium was freshly prepared prior to imaging.</p>
<p>Note that rutin is supplied as part of DMEM<sup>gfp</sup>-2 kit (Cat#: MCK02).</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Setting up retina for live imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>This step describes the retrieval of retinae either from living mice or from retinal explant cultures at desired developmental stage to set them up for live imaging. Herein contained are details on retinal dissection, tissue mounting, and live imaging culture set up. Live imaging experiments were performed on an upright confocal microscope; for an inverted microscope the setup would have to be modified accordingly.</p>
<ol>
<li>Retrieval of retina. When isolating retinae from live mice or retinal explant cultures, follow steps ‘a.’ or ‘b.’, respectively.
<ol type="a">
<li>Retrieval from live mice.
<ol type="i">
<li>Sacrifice mice by cervical dislocation.</li>
<li>Enucleate eyes and transfer to ice-cold RPMI 1640 medium using forceps (<a href="#fig1">Figure 1</a>A).</li>
<li>Isolate neural retina (tissue composed of cells of the outer nuclear layer, inner nuclear layer, and ganglion cell layer) by dissection under a dissection microscope in ice-cold RPMI 1640 medium by removing all non-neural tissues (cornea, sclera, iris, choroid, lens, optic nerve, retinal pigment epithelium, and ciliary body) using forceps (<a href="#fig1">Figures 1</a>B and 1C).
<div class="critical">
<span class="critical-title">Critical:</span> Upon removal of the lens during retinal dissection, ensure that vitreous is removed as well. Failure to remove vitreous can impede the flattening process (also applies to 1.b.ii).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Care must be taken to remove all pigmented non-neural retinal tissues (retinal pigment epithelium and ciliary body) as the melanin pigment contained in these tissues absorbs the 2-photon infrared light leading to rapid heat-induced tissue damage when the microscope is operated in 2-photon mode (also applied to 1.b.ii).</div>
</li>
</ol>
</li>
<li>Retrieval from retinal explant cultures.
<ol type="i">
<li>Transfer retina from explant culture to ice-cold RPMI 1640 medium.</li>
<li>Isolate neural retina by dissection under a dissection microscope in ice-cold RPMI 1640 medium by removing lens with forceps.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Flattening and mounting retina.
<ol type="a">
<li>Apply four equidistant radial cuts using fine scissors going halfway towards the centre of the retina. This results in ‘petal’ like retinal segments (<a href="#fig1">Figure 1</a>D).</li>
<li>Transfer retina in a small drop of RPMI 1640 medium onto the edge portion of a microscopy glass slide using a sterile plastic Pasteur pipette with manually widened opening (<a href="#fig1">Figure 1</a>I) such that photoreceptors face the glass slide and retinal ganglion cells face the user (<a href="#fig1">Figure 1</a>D).</li>
<li>Carefully flatten retina onto glass slide by unfolding petals using fine forceps. Petals will resist flattening due to natural curvature of the retina. Flattening is aided by removing some of the RPMI 1640 (<a href="#fig1">Figure 1</a>E).
<div class="critical">
<span class="critical-title">Critical:</span> Care must be taken to not introduce tissue damage during flattening and unfolding petals. This may be aided by using a clean, fine brush. Even small tissue disruption may result in visible loss of retinal integrity during live imaging.</div>
<div class="critical">
<span class="critical-title">Critical:</span> (2.c.-3.b.) During these steps, the retina will be exposed to minimal amounts of RPMI 1640 medium at times. Care must be taken to not let the retina dry out.</div>
</li>
<li>After successful flattening, remove most of RPMI 1640 medium using lint-free tissue paper.</li>
<li>Immediately place a 0.45 μm MF-Millipore™ nitrocellulose membrane on the retina (with ganglion cell layer touching the membrane) making sure tissue is securely mounted on the membrane (<a href="#fig1">Figure 1</a>F).</li>
<li>Gently transfer membrane-mounted retina back into RPMI 1640 medium by dipping the edge of the glass slide into medium (<a href="#fig1">Figure 1</a>G). The membrane will gradually become visibly hydrated when contacted by the solution, upon completion of which the membrane can be fully transferred using the dissection tools (<a href="#fig1">Figure 1</a>H).</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Setting up imaging dish for live imaging (<a href="#fig2">Figure 2</a>)
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1226-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Live imaging setup schematic</div>
</figcaption>
</figure>
<ol type="a">
<li>Fix membrane with retina facing up to the bottom of a Ø5.5 cm tissue culture petri dish on a thin layer of vacuum grease.</li>
<li>To ensure secure mounting and to prevent tissue drifting during live imaging, carefully press down membrane onto vacuum grease using blunt tools. Place a platinum ring over the membrane, the added weight of which should further prevent tissue drifting.</li>
<li>Fill the dish with 10 mL DMEM<sup>gfp</sup>-2 live imaging medium.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Live imaging setup and recording</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min set up; up to ∼ 12 h time-lapse recording</div>
<p>The following steps describe setting up the retinal live imaging session (see also <a href="#fig2">Figure 2</a>), at the end of which the user will have generated a time-lapse recording of the fluorescently labelled retinal cells. We routinely used a 2-photon microscope set-up to reduce phototoxicity and to increase photon tissue penetration and <i>z</i> resolution.</p>
<ol start="4">
<li>Transfer imaging dish containing flat-mounted retina to microscope environmental chamber pre-equilibrated to 37°C/5% CO<sub>2</sub>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Allow tissue to acclimatize to 37°C/5% CO<sub>2</sub> for at least 30 min prior to initiating time-lapse recording to prevent temperature gradient induced tissue drifting.</div>
<ol start="5">
<li>Carefully introduce a water-immersion objective pre-sterilized with 70 % ethanol into the live imaging medium. We routinely used 25× and 40× water-immersion objectives (NA = 0.95 and 0.8 respectively).</li>
<li>Affix CO<sub>2</sub> supplying apparatus to the live imaging dish.</li>
<li>Identify suitable field of view to be subjected to time-lapse recording by scanning through microscope eyepiece.</li>
<li>Tune the multiphoton laser source to a wavelength suitable for 2 photon-excitation of the fluorescent protein(s) of interest (e.g., 920 nm for EGFP).</li>
<li>Further, ensure that laser and detector(s) have been properly set up for signal detection.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In 2-photon mode, the pinhole of confocal microscopes should be fully dilated.</div>
<ol start="10">
<li>Apply a layer of paraffin oil, pre-equilibrated to 37°C, on top of the live imaging medium to prevent medium evaporation during time-lapse recording.</li>
<li>Initiate time-lapse recording. In <a href="#bib1">Aghaizu et al. (2021)</a>, we acquired <i>xyzt</i> image series at a resolution of 512 × 512 pixels, at a step size of 1 μm and at 10 min intervals. Imaging duration frequently reached up to 12 h, although this is critically dependent on starting tissue health/integrity.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Any pigmented cells present within the field of view will absorb infrared light (i.e. 2-photon laser light) and rapidly heat up the tissue, rapidly bringing about tissue destruction.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to ensure on occasion that the intended field of view remains in focus; in the event of tissue drift appropriate adjustments are to be made by user.</div>
<div class="note">
<span class="note-title">Note:</span> When imaging photoreceptors, early stages of deteriorating tissue health most frequently presented themselves as individual photoreceptors apically extending out of the outer nuclear layer (see <a href="#fig4">Figure 4</a>A, bottom right panel for an example of apically extruding photoreceptors). Further deterioration usually entails full apical ‘ejection’ of individual and subsequently bigger groups of photoreceptors until tissue integrity is fully compromised.</div>
<h3 id="sec3.3">
<i>In silico</i> processing of time-lapse recording and cell tracking</h3>
<div class="timing">
<span class="timing-title">Timing:</span> minimum 30 mins but can be significantly longer depending on image quality, number of cells to be tracked, and implementation of manual track validation.</div>
<p>Upon completion of time-lapse recording, generated <i>xyzt</i> image series are to be processed and subjected to cell tracking.</p>
<ol start="12">
<li>Process <i>xyzt</i> image series in Fiji/ImageJ (<a href="#bib8">Schindelin et al., 2012</a>) with standard imaging processing tools available, such as noise reduction, to achieve maximum possible image quality and signal to noise ratio. Ensure that processing steps are kept consistent.</li>
<li>Register <i>xyzt</i> image series (<i>in silico</i> correction of tissue drift), as some amount of tissue drifting during time-lapse recording is common. This can be accomplished in Fiji/ImageJ either with the ‘Correct 3D drift’ plug-in (<a href="#bib6">Parslow et al., 2014</a>) for automated drift correction based on automated landmark recognition or the ‘Descriptor-based series registration (2D/3D + t)’ plug-in (<a href="#bib7">Preibisch et al., 2010</a>) in cases where automated registration failed and registration landmarks had to be defined manually.</li>
<li>Export registered <i>xyzt</i> image series as ∗.tif file.</li>
<li>Perform cell tracking using software of choice. In <a href="#bib1">Aghaizu et al. (2021)</a> we used the spot tracking tool within IMARIS software. IMARIS software is able to identify rod cell bodies based on the preceding input of size and location parameters (<a href="#mmc1">Methods video S1</a>; video still shown in <a href="#fig3">Figure 3</a>). Rod somata were typically ellipsoid in shape, measuring ∼ 5 μm along the short (lateral) and ∼ 7.5 μm along the long (radial) axis and are typically located deeper within the tissue compared to the developing bulbous segment structures.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1226-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. IMARIS tracking of rod somal motility in <i>xyzt</i> image stacks obtained from time-lapse recordings</div>
<p>This image is a still captured from <a href="#mmc1">Methods video S1</a>. Scale bar, 50 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Once set up, cell tracking in the IMARIS spot tracking tool is fully automated. Nonetheless, we recommend manually validating the generated trajectories to avoid including data artifacts into subsequent analysis steps.</div>
<ol start="16">
<li>Export completed cell tracking data as ∗.xls file.</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1226-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. IMARIS tracking methodology (see also Figure 2; related to step 15)</div>
</figcaption>
</figure>
<h3 id="sec3.4">Data processing and analysis</h3>
<p>The primary data outputs from the processing and analysis steps will be average mean squared displacement (AvMSD) as well as instantaneous velocity and instantaneous acceleration values for the population of cells analysed. In <a href="#bib1">Aghaizu et al. (2021)</a>, these were generated by importing IMARIS-generated tracking information into Matlab followed by execution of custom-written Matlab scripts (available on the project Github repository on <a href="https://github.com/RPearsonLab/Photoreceptor_tracking">https://github.com/RPearsonLab/Photoreceptor_tracking</a>).</p>
<ol start="17">
<li>Arrange IMARIS-generated tracking data in Microsoft Excel according to <a href="#tbl1">Table 1</a>. ‘#’ denotes numbered data point ID in Excel; ‘Trajectory ID’ denotes unique numbered identifier of individual trajectories; ‘Time index’ denotes time point within time-lapse recording, at which data point was generated (note that this is in integer rather than time units); ‘Position <i>x/y/z</i>’ denotes 3D positional information.
<table id="tbl1">
<caption>Table 1. Example trajectory data</caption>
<thead>
<tr>
<th>#</th>
<th>Trajectory ID</th>
<th>Time index</th>
<th>Position <i>x</i>
</th>
<th>Position <i>y</i>
</th>
<th>Position <i>z</i>
</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1.0</td>
<td>1.0</td>
<td>60.0</td>
</tr>
<tr>
<td>2</td>
<td>1</td>
<td>2</td>
<td>1.0</td>
<td>1.0</td>
<td>59.0</td>
</tr>
<tr>
<td>3</td>
<td>1</td>
<td>3</td>
<td>1.0</td>
<td>1.0</td>
<td>54.0</td>
</tr>
<tr>
<td>4</td>
<td>1</td>
<td>4</td>
<td>1.0</td>
<td>1.0</td>
<td>51.0</td>
</tr>
<tr>
<td>5</td>
<td>1</td>
<td>5</td>
<td>1.0</td>
<td>1.0</td>
<td>45.0</td>
</tr>
<tr>
<td>6</td>
<td>2</td>
<td>1</td>
<td>1.0</td>
<td>1.0</td>
<td>30.0</td>
</tr>
<tr>
<td>7</td>
<td>2</td>
<td>2</td>
<td>1.0</td>
<td>1.0</td>
<td>30.0</td>
</tr>
<tr>
<td>8</td>
<td>2</td>
<td>3</td>
<td>1.0</td>
<td>1.0</td>
<td>30.0</td>
</tr>
<tr>
<td>9</td>
<td>2</td>
<td>4</td>
<td>1.0</td>
<td>1.0</td>
<td>30.0</td>
</tr>
<tr>
<td>10</td>
<td>2</td>
<td>5</td>
<td>1.0</td>
<td>1.0</td>
<td>30.0</td>
</tr>
<tr>
<td>11</td>
<td>3</td>
<td>1</td>
<td>1.0</td>
<td>1.0</td>
<td>10.0</td>
</tr>
<tr>
<td>12</td>
<td>3</td>
<td>2</td>
<td>1.0</td>
<td>1.0</td>
<td>10.1</td>
</tr>
<tr>
<td>13</td>
<td>3</td>
<td>3</td>
<td>1.0</td>
<td>1.0</td>
<td>10.5</td>
</tr>
<tr>
<td>14</td>
<td>3</td>
<td>4</td>
<td>1.0</td>
<td>1.0</td>
<td>10.9</td>
</tr>
<tr>
<td>15</td>
<td>3</td>
<td>5</td>
<td>1.0</td>
<td>1.0</td>
<td>11.0</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Matlab scripts were designed to read-in tracking data in the format specified in <a href="#tbl1">Table 1</a>. The user is advised to either manipulate data to match <a href="#tbl1">Table 1</a> or to modify Matlab script accordingly.</div>
<ol start="18">
<li>Open and run the Matlab script 'MSDforztrackingdata.m' (see Github project repository) to read-in trajectory data and to calculate AvMSD &amp; standard error values. Users will be prompted to navigate to the source trajectory file upon initiation of the script.</li>
</ol>
<p>AvMSD will be calculated according to the equation</p>
<p class="math-ml"><math> <mrow> <mi>M</mi> <mi>S</mi> <mi>D</mi> <mrow> <mo>(</mo> <mrow> <mi>Δ</mi> <mi>t</mi> </mrow> <mo>)</mo> </mrow> <mo>=</mo> <mspace></mspace> <mfrac> <mn>1</mn> <mrow> <mi>N</mi> <mo>−</mo> <mi>n</mi> </mrow> </mfrac> <munderover> <mo>∑</mo> <mrow> <mi>i</mi> <mo>=</mo> <mn>1</mn> </mrow> <mrow> <mi>N</mi> <mo>−</mo> <mi>n</mi> </mrow> </munderover> <msup> <mrow> <mo>[</mo> <mrow> <mi>z</mi> <mrow> <mo>(</mo> <mrow> <mrow> <mo>(</mo> <mrow> <mi>i</mi> <mo>+</mo> <mi>n</mi> </mrow> <mo>)</mo> </mrow> <mi>δ</mi> <mi>t</mi> </mrow> <mo>)</mo> </mrow> <mo>−</mo> <mi>z</mi> <mrow> <mo>(</mo> <mi>i</mi> <mi>δ</mi> <mi>t</mi> <mo>)</mo> </mrow> </mrow> <mo>]</mo> </mrow> <mn>2</mn> </msup> </mrow> </math></p>
<p>where <math> <mrow> <mi>z</mi> <mrow> <mo>(</mo> <mrow> <mi>i</mi> <mi>δ</mi> <mi>t</mi> </mrow> <mo>)</mo> </mrow> </mrow> </math> and <math> <mrow> <mi>z</mi> <mrow> <mo>(</mo> <mrow> <mrow> <mo>(</mo> <mrow> <mi>i</mi> <mo>+</mo> <mi>n</mi> </mrow> <mo>)</mo> </mrow> <mi>δ</mi> <mi>t</mi> </mrow> <mo>)</mo> </mrow> </mrow> </math> are the z positions of a given nucleus at time frames <math> <mrow> <mi>i</mi> <mi>δ</mi> <mi>t</mi> </mrow> </math> and <math> <mrow> <mrow> <mo>(</mo> <mrow> <mi>i</mi> <mo>+</mo> <mi>n</mi> </mrow> <mo>)</mo> </mrow> <mi>δ</mi> <mi>t</mi> </mrow> </math> respectively, n is an integer representing the time interval between those positions and N is the total number of time points within the time-lapse recording. The only script manipulation required to run 'MSDforztrackingdata.m' is to specify the range of trajectory IDs to be analysed from source file (e.g.,: 1, 2, 3,…, n); line 134.</p>
<p class="chem">group = [<code>1</code>, <code>2</code>, <code>3</code>];</p>
<div class="note">
<span class="note-title">Note:</span> The above formula calculates the MSD at all possible time lags and produces averages within individual trajectories as well as the entire group of specified trajectories. For instance, at a time lag of 2 according to <a href="#tbl1">Table 1</a>, all possible MSDs between time indices 1 &amp; 3, 2 &amp; 4, and 3 &amp; 5 for trajectories 1–3 are calculated and averaged to produce the AvMSD value at time lag 2.</div>
<ol start="19">
<li>Upon completion of script 'MSDforztrackingdata.m', the Matlab will have generated result outputs in Vector/matrix format denoted ‘AvMSD’ and ‘standarderrors’ containing AvMSD and standard error values with increasing time lag.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For our cell type of interest (photoreceptors), motility occurred predominantly along the <i>z</i> axis (radial) rather than in <i>xy</i> (lateral) directions. AvMSD and instantaneous velocity calculations in the Matlab script are thus based exclusively on <i>z</i> motility and the user is advised to adjust the Matlab script accordingly for higher dimensional analyses.</div>
<ol start="20">
<li>Open and run 'ztrajectory_gen.m' to convert read-in trajectory data stored in array format into matrix 'ztrajectories'. Individual trajectories are listed by row.</li>
</ol>
<div class="textbox">
<p><code>lengths = ones(length(trajectories),1);</code></p>
<p><code>for i = 1:length(trajectories)</code></p>
<p>  <code>lengths(i) = length(trajectories{i});</code></p>
<p><code>end</code></p>
<p><code>t = max(lengths);</code></p>
<p><code>ztrajectories = zeros(length(trajectories),t);</code></p>
<p><code>for i = 1:length(trajectories)</code></p>
<p>  <code>if trajectories{i}(1,2) &gt; 1</code></p>
<p>    <code>leading = NaN∗ones(length(1:(trajectories{i}(1,2)-1)),1);</code></p>
<p>  <code>else</code></p>
<p>    <code>leading = [];</code></p>
<p>  <code>end</code></p>
<p>  <code>if trajectories{i}(end,2) &lt; t</code></p>
<p>    <code>trailing = NaN∗ones(length(trajectories{i}(end,2):t-1),1);</code></p>
<p>  <code>else</code></p>
<p>    <code>trailing = [];</code></p>
<p>  <code>end</code></p>
<p>  <code>ztrajectories(i,:) = [leading;trajectories{i}(:,5);trailing];</code></p>
<p><code>end</code></p>
</div>
<ol start="21">
<li>Open and run 'Trajectory_analysis.m' for further kinetic analyses (instantaneous velocity, instantaneous acceleration) of trajectory data generated in (17.) and (18.). Upon completion, the Matlab script will have generated result outputs in matrix format denoted ‘dfff2’ and ‘d2fff2’ containing smoothened instantaneous velocity and acceleration values respectively. ‘dyyy’ contains un-smoothened instantaneous velocity data.</li>
</ol>
<div class="textbox">
<p><code>ddd = ztrajectories;</code></p>
<p><code>ncells = size(ddd,1);</code></p>
<p><code>nsamples = size(ddd,2);</code></p>
<p><code>xxx = nan(ncells,nsamples);</code></p>
<p><code>yyy = nan(ncells,nsamples);</code></p>
<p><code>for icell = 1:ncells</code></p>
<p>  <code>x = find(isfinite(ddd(icell,:)));</code></p>
<p>  <code>y = ddd(icell,x);</code></p>
<p>  <code>xq = min(x):max(x);</code></p>
<p>  <code>yq = interp1(x,y,xq,'linear');</code></p>
<p>  <code>xxx(icell,xq) = xq;</code></p>
<p>  <code>yyy(icell,xq) = yq;</code></p>
<p><code>end</code></p>
<p><code>fff = filter( [1 1 1]/3, 1, yyy' )';</code></p>
<p><code>fff(:,[1:3 end-2:end]) = NaN;</code></p>
<p><code>fff = circshift(fff,[0 -1]);</code></p>
<p><code>dyyy = diff(yyy,1,2)./10;</code></p>
<p><code>z = nan(ncells,1);</code></p>
<p><code>dyyy = horzcat(dyyy,z);</code></p>
<p><code>dfff = diff(fff,1,2)./10;</code></p>
<p><code>z = nan(ncells,1);</code></p>
<p><code>dfff = horzcat(dfff,z);</code></p>
<p><code>windowsize = 4;</code></p>
<p><code>b = (1/windowsize)∗ones(1,windowsize);</code></p>
<p><code>a = 1;</code></p>
<p><code>dim = 2;</code></p>
<p><code>dfff2 = filter(b,a,dfff,[],dim);</code></p>
<p><code>dfff2 = circshift(dfff2,[0 -1]);</code></p>
<p><code>d2fff = diff(dfff,1,2);</code></p>
<p><code>z = nan(ncells,1);</code></p>
<p><code>d2fff = horzcat(d2fff,z);</code></p>
<p><code>windowsize = 4;</code></p>
<p><code>b = (1/windowsize)∗ones(1,windowsize);</code></p>
<p><code>a = 1;</code></p>
<p><code>dim = 2; % to filter rows, not columns</code></p>
<p><code>d2fff2 = filter(b,a,d2fff,[],dim);</code></p>
<p><code>d2fff2 = circshift(d2fff2,[0 -1]);</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> Matlab script was custom written for time-lapse recordings acquired at 10 min intervals. This is reflected in instantaneous velocity calculations, where velocity between consecutive frames (μm/10 min) needed to be divided by 10 to obtain instantaneous velocity (μm/min) (lines 59 and 64 in 'Trajectory_analysis.m').</div>
<ol start="22">
<li>Plot and interpret data generated in (17.) and (19.) in software of choice, e.g., Matlab or Graphpad Prism.
<ol type="a">
<li>Plot AvMSD (± standard errors) vs time lag (e.g., 0, 10, 20, 30,…n min). Depending on the nature of motility, AvMSD vs time lag curves will be either linear or curved.
<ol type="i">
<li>A linear AvMSD vs time lag curve is produced if random/non-directional motion dominates in the population of analysed cells. Here, the MSD is a linear function of elapsed time Δt,
<p class="math-ml"><math> <mrow> <mi>M</mi> <mi>S</mi> <mi>D</mi> <mo>=</mo> <mn>2</mn> <mi>R</mi> <mi>D</mi> <mi>Δ</mi> <mi>t</mi> </mrow> </math></p>
with a one-dimensional slope of 2R<i>D = 2D</i>, where R is the dimensionality (in the present study, R = 1) and where <i>D</i> is the coefficient of movement. Apply linear function curve fitting to obtain the slope ‘2D’ and subsequently calculate ‘D’ to quantitatively compare AvMSD steepness.</li>
<li>Directed motions in the analysed cell population result in AvMSD vs time lag curves with positive curvature. Here, the MSD displays a quadratic dependence on elapsed time. Apply quadratic function curve fitting
<p class="math-ml"><math> <mrow> <mi>y</mi> <mo>=</mo> <mi>a</mi> <msup> <mi>x</mi> <mn>2</mn> </msup> </mrow> </math></p>
to obtain the quadratic factor ‘a’ as numerical representation of curve steepness.</li>
</ol>
</li>
<li>Plot instantaneous velocity and acceleration values as histograms to show frequency distributions.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Following successful live imaging the user will have generated a <i>xyzt</i> image stack. The fluorescence signal distribution within the tissue will depend on the exact experimental design. For reference, we provide two examples where rod photoreceptors were labeled using different approaches (<a href="#fig4">Figure 4</a>A): <i>Nrl.Cre</i><sup><i>+/−</i></sup> retina, electroporated at P0.5 with <i>shDync1h1/EGFP</i> and cultured for 6 days <i>in vitro</i> (DIV) before live imaging (left panels); <i>Nrl.GFP</i>, postnatal day (P) 7.5 (right panels). Rod somal position can then be tracked as described in step 15. Top-down (<i>xy</i>) and orthogonal views (<i>xz;</i> maximum z projection of 5 adjacent slices at 1 μm interval) of the image stacks are shown. For the purpose of this protocol, the presented quantifications are based on the example trajectory data provided in <a href="#tbl1">Table 1</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1226-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. <i>z</i> motility quantification of rod photoreceptors in <i>xyzt</i> image stacks</div>
<p>(A) top-down (<i>xy</i>) and orthogonal views (<i>xz</i>) of live imaged retinae with rod photoreceptors labeled either conditionally by electroporation (left panels; <i>Nrl.Cre</i><sup><i>+/−</i></sup> retina, electroporated at P0.5 with <i>shDync1h1/EGFP</i> and cultured for 6 DIV) or by transgenic reporter labeling (right panels; <i>Nrl.GFP</i>, P7.5).</p>
<p>(B) Example trajectory data <i>z</i> positions.</p>
<p>(C) Example trajectory data AvMSD vs. time lag plot including quadratic function curve fitting. Error bars show SEM.</p>
<p>(D) Example trajectory data instantaneous velocity histogram. Scale bars, 25 μm.</p>
</figcaption>
</figure>
<p>Changes in <i>z</i> position over time were plotted for the example trajectory data (<a href="#fig4">Figure 4</a>B). Average mean squared displacement and standard error values were obtained using the provided Matlab script (step 18.) and plotted versus time lag (<a href="#fig4">Figure 4</a>C). Note the positive curvature in the data points, due to which quadratic function curve fitting (y = ax<sup>2</sup>) should be applied to obtain the quadratic coefficient. Where average MSD data points follow a straight line, linear curve fitting should be applied, in order to obtain the coefficient of movement according to step 22.a.i. Instantaneous velocity values were derived using the provided Matlab script (steps 21, 22.) and plotted as a histogram (<a href="#fig4">Figure 4</a>D).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol described here is inherently low throughput as only one retina can be live imaged at a time. This bottleneck will be especially evident when time-lapse recordings are conducted for ∼ 12 h.</p>
<p><i>In silico</i> trajectory verification presents another potential bottleneck in the protocol as we recommend verifying each individually generated trajectory. Generally, trajectory generation reliability increases with higher signal-to-noise ratios in the <i>xyzt</i> image series. Producing high-quality time-lapse recordings as well as appropriate image processing workflows are thus essential to increasing the data output.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Step 1.a.: Neural retina prolapse from eyeball during its isolation resulting in retinal tissue damage.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>During eyeball dissection to isolate neural retina, the eyeball needs to be carefully opened using dissection tools as described. Small amounts of pressure applied to the eyeball during this step can lead to the neural retina prolapsing from introduced eyeball openings, compromising retinal integrity. This can be avoided by piercing the eyeball at the interface between cornea and sclera using a fine gauge needle (e.g., 30 G BD microlance 3 needle, Cat#304000).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Step 11: tissue drift during time-lapse recording.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>In case <i>xyz</i> tissue drift is detected during time-lapse recording, adjust field of view and/or upper/lower limits of z-stack accordingly in the intervals between time series points (this may require prematurely stopping the acquisition process and initiating a new <i>xyzt</i> acquisition, which can subsequently be concatenated <i>in silico</i> with the initial acquisition, for instance using Fiji/ImageJ). To avoid tissue drift in future experiments, consider the following options during tissue mounting (steps 2. and 3. in <a href="#step-by-step-method-details">step-by-step method details</a>):</p>
<p>Using a heavier platinum harp.</p>
<p>More firmly pressing the edges of the nitrocellulose membrane with mounted retina onto the vaccume grease layer using a round edge of curved forceps.</p>
<p>Securing the edge of nitrocellulose membrane with mounted retina with extra vacuum grease contiguously spread over both the nitrocellulose edge and the underlying petri dish.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Step 11: signs of boiling/bubbling evident during time-lapse recording.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Boiling and bubbling are usually a consequence of pigmented cells (usually black in colour) present within the field of view. These absorb infrared light and rapidly heat up the tissue leading to irreversible tissue damage. To prevent this from occurring, consider imaging a field of view completely devoid of pigmented cells or completely removing all pigmented tissues/cells from neural retina during relevant dissection steps and/or prior to live imaging.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Step 11: loss of retinal integrity during time-lapse recording.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Even the most benign tissue damage introduced during retinal dissection and mounting tends to compound during live imaging, eventually leading to loss of retinal integrity. This can be mitigated by careful tissue handling and/or by avoiding damaged areas of the retina during live imaging.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Step 15: Spot tracking in IMARIS produces too many tracking artifacts, misrepresenting true somal movements.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Using an optimized workflow in Fiji/ImageJ can contribute greatly towards generating higher quality, high signal-to-noise time-lapse recordings post acquisition, which can improve subsequent tracking in IMARIS. While there is no ‘one-size-fits-all’ approach applicable to all time-lapse experiments, we have found the ‘Remove outliers’ tool in Fiji/ImageJ particularly useful to remove noise artifacts that can impede faithful tracking in IMARIS. Set parameters to remove “bright” outliers at a radius and threshold that approximately match the characteristics of noise artifacts.</p>
<p>Furthermore, the spot detection tool in IMARIS offers filter options to remove artifactual trajectories. The ‘Quality’ filter can be utilized to retain only high-quality tracking points, rather than lower-quality tracking points, which are more likely to be associated with imaging noise and/or background signal. Other IMARIS filters that proved to be useful for tracking photoreceptor somata that primarily exhibit radial (along the <i>z</i> axis) motility were <i>x</i> and <i>y</i> dimension filters. These can be used to restrict the maximum permissible amount of tangential (<i>xy</i>) motility in the generated tracks, which further eliminated artifactual tracking in our experiments.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aghaizu, N.D., Warre-Cornish, K.M., Robinson, M.R., Waldron, P.V., Maswood, R.N., Smith, A.J., Ali, R.R., and Pearson, R.A. (2021). Repeated nuclear translocations underlie photoreceptor positioning and lamination of the outer nuclear layer in the mammalian retina. Cell Rep. <i>36</i>, 109461. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2021.109461">https://doi.org/10.1016/j.celrep.2021.109461</a></p>
<p id="bib2">Akimoto, M., Cheng, H., Zhu, D., Brzezinski, J.A., Khanna, R., Filippova, E., Oh, E.C.T., Jing, Y., Linares, J.-L., Brooks, M., et al. (2006). Targeting of GFP to newborn rods by Nrl promoter and temporal expression profiling of flow-sorted photoreceptors. Proc. Natl. Acad. Sci. U S A <i>103</i>, 3890-3895. <a class="external-link" href="https://doi.org/10.1073/pnas.0508214103">https://doi.org/10.1073/pnas.0508214103</a></p>
<p id="bib3">Brightman, D.S., Razafsky, D., Potter, C., Hodzic, D., and Chen, S. (2016). Nrl-Cre transgenic mouse mediates loxP recombination in developing rod photoreceptors. Genesis <i>54</i>, 129-135. <a class="external-link" href="https://doi.org/10.1002/dvg.22918.Nrl-Cre">https://doi.org/10.1002/dvg.22918.Nrl-Cre</a></p>
<p id="bib4">Gong, S., Zheng, C., Doughty, M.L., Losos, K., Didkovsky, N., Schambra, U.B., Nowak, N.J., Joyner, A., Leblanc, G., Hatten, M.E., et al. (2003). A gene expression atlas of the central nervous system based on bacterial artificial chromosomes. Nature <i>425</i>, 917-925. <a class="external-link" href="https://doi.org/10.1038/nature02033">https://doi.org/10.1038/nature02033</a></p>
<p id="bib5">Matsuda, T. and Cepko, C.L. (2004). Electroporation and RNA interference in the rodent retina in vivo and in vitro. Proc. Natl. Acad. Sci. U S A <i>101</i>, 16-22. <a class="external-link" href="https://doi.org/10.1073/pnas.2235688100">https://doi.org/10.1073/pnas.2235688100</a></p>
<p id="bib6">Parslow, A., Cardona, A., and Bryson-Richardson, R.J. (2014). Sample drift correction following 4D confocal time-lapse imaging. J. Vis. Exp. <i>86</i>, 51086. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00714-0/sref6">View at publisher</a></p>
<p id="bib7">Preibisch, S., Saalfeld, S., Schindelin, J., and Tomancak, P. (2010). Software for bead-based registration of selective plane illumination microscopy data partitioning biological data with transitivity clustering. Nat. Publ. Gr. <i>7</i>, 418-419. <a class="external-link" href="https://doi.org/10.1038/nmeth0610-418">https://doi.org/10.1038/nmeth0610-418</a></p>
<p id="bib8">Schindelin, J., Arganda-carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="https://doi.org/10.1038/nmeth.2019">https://doi.org/10.1038/nmeth.2019</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Rachael Pearson (<a href="mailto:rachael.pearson@kcl.ac.uk">rachael.pearson@kcl.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The data sets generated during and/or analysed during the current study are available from the lead author on request. The code generated during this study is available on the project Github repository on <a href="https://github.com/RPearsonLab/Photoreceptor_tracking">https://github.com/RPearsonLab/Photoreceptor_tracking</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank M. Carandini for assistance with data analysis, P.V. Waldron for helpful discussions, L. Abelleira-Hervas and J. Hoke for animal husbandry and the UCL Cell and Developmental Biology Imaging Unit for imaging assistance.</p>
<p>This work was supported by the <a href="https://doi.org/10.13039/501100000288">Royal Society</a> (UF120046; RG080398); <a href="https://doi.org/10.13039/501100012617">Moorfields Eye Charity</a> (E170004A; R150032A, R180005A); <a href="https://doi.org/10.13039/501100000265">Medical Research Council UK</a> (mr/j004553/1; MR/T002735/1). N.D.A. was a UCL Grand Challenge PhD student and Fight For Sight-funded post-doctoral research associate. M.R.R. was an MRC DTP 4 year PhD student.</p>
<h3>Author contributions</h3>
<p>(CRediT taxonomy) Conceptualization, N.D.A., K.M.W-C. and R.A.P.; Methodology, N.D.A. K.M.W-C., and R.A.P. Software, N.D.A. and M.R.R. Validation, N.D.A. and K.M.W-C.; Formal Analysis, N.D.A., M.R.R.; Investigation, N.D.A., K.M.W-C.; Resources, R.R.A. and R.A.P.; Writing – Original Draft, N.D.A. and R.A.P.; Writing – Review &amp; Editing, N.D.A. and R.A.P.; Funding Acquisition, R.R.A. and R.A.P.; Supervision, R.A.P.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>