<article>
<section>
<h1 id="header">Biochemical approach for isolation of polyadenylated RNAs with bound proteins from yeast</h1>
<p><time datetime="2021-11-05">Published: November 5, 2021</time></p>
<p>Ana M. Matia-González,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Ibtissam Jabre,<sup><a href="#aff1">1</a></sup> and André P. Gerber<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Microbial Sciences, School of Biosciences and Medicine, Faculty of Health and Medical Sciences, University of Surrey, Guildford, Surrey GU2 7XH, UK</p>
<p id="fn1"><sup>2</sup>Present address: Department of Biochemistry and Molecular Biology I, Faculty of Sciences, University of Granada, Avda Fuentenueva s/n, 18071 Granada, Spain</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ammatiag@ugr.es">ammatiag@ugr.es</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:a.gerber@surrey.ac.uk">a.gerber@surrey.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100929">10.1016/j.xpro.2021.100929</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p><i>In vivo</i> characterization of RNA-protein interactions is the key for understanding RNA regulatory mechanisms. Herein, we describe a protocol for detection of proteins interacting with polyadenylated RNAs in the yeast <i>Saccharomyces cerevisiae.</i> Proteins are crosslinked to nucleic acids <i>in vivo</i> by ultraviolet (UV) irradiation of cells, and poly(A)-containing RNAs with bound proteins are isolated from cell lysates using oligo[dT]<sub>25</sub> beads. RBPs can be detected by immunoblot analysis or with mass spectrometry to define the mRNA-binding proteome (mRBPome) and its changes under stress.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref8">Matia-González et al. (2021</a>, <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref7">2015)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detection of proteins that interact with polyadenylated RNAs <i>in vivo</i>
</li>
<li>UV irradiation of cells to crosslink RNA-protein interactions</li>
<li>Polyadenylated RNAs are captured with oligo[dT] beads from yeast cell lysate</li>
<li>Suitable for monitoring dynamic changes of the mRNA-binding proteome</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1142-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol describes the specific steps for poly(A) RNA interactome capture (RIC) from UV-irradiated budding yeast cells grown in rich media (Yeast-Peptone-Dextrose (YPD)). The protocol has also been applied to cells treated with 0.5 mM hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>) as a stress reagent for induction of mild-oxidative stress (<a href="#bib8">Matia-González et al., 2021</a>).</p>
<p>The procedure is based on the capture of poly(A) RNA, a major component of it comprising mRNAs. Hence, the protocol presented here is preferred for investigation of mRNA regulation executed by mRNA binding proteins (mRBPs) that mediate the processing of mRNA precursors (pre-mRNAs) in the nucleus, the export and localization of mRNAs to different subcellular locations in the cytoplasm, and the translation and eventual decay of mRNAs (<a href="#bib15">Singh et al., 2015</a>). We wish to note that there are alternative RIC protocols aimed at isolation of proteins associated with total RNA from yeast (including highly abundant ribosomal RNAs, tRNAs) and are based on organic phase extraction (<a href="#bib11">Queiroz et al., 2019</a>) or the use of silica beads (<a href="#bib13">Shchepachev et al., 2019</a>). In any cases, cells are exposed to UV-irradiation to promote covalent crosslinks between the RNA and proteins in close proximity. However, it should be noted that UV-irradiation of cells can also lead to protein-protein and protein-DNA crosslink, a possibility to be considered for interpretation of the data.</p>
<p>Yeast lysates contain a substantial amount of enzymatic activities (e.g., RNases, DNases, and proteases). Cell lysates and samples should therefore be kept on ice, which substantially reduces enzymatic activity. It is also necessary to add RNase and protease inhibitors to buffers/samples immediately before use according to the suggestions given in the protocol. Furthermore, we recommend using sterile filter tips and RNase-free tubes with low RNA/protein binding capacities. Gloves must be worn, and the protocol should be performed in an RNase-free workspace, which can be accomplished by cleaning surfaces with RNase ZAP<sup>TM</sup> or likewise RNase decontamination solutions.</p>
<h3 id="sec1.1">Media preparation and streaking of yeast cells on agar plates</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>Before starting the protocol, sterile media needs to be prepared for the growth of yeast <i>Saccharomyces cerevisiae</i> (<i>S. cerevisiae)</i> cells. In our case, we used autoclaved YPD media, a rich media supplemented with glucose as the carbon source (<a href="#bib14">Sherman, 2002</a>). The desired yeast strain needs then to be streaked on YPD plates so then, a single colony can be picked for growth in liquid culture later on.</p>
<ol>
<li>Create a sterile environment by turning on a Bunsen burner.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The updraft created by the flame for sterility works best within a radius of 30–70 cm.</div>
<ol start="2">
<li>Remove approximately 10–20 μL <i>S. cerevisiae</i> cells (e.g., strain BY4741) from the glycerol stock and streak cells on a YPD agar plate with a sterile inoculation loop.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep the glycerol stock on ice to prevent rapid defrosting. Put the stock vial back to the −80°C freezer immediately after use.</div>
<div class="note">
<span class="note-title">Note:</span> We used the BY4741 strain, but the protocol is also applicable to other <i>S. cerevisiae</i> strains.</div>
<ol start="3">
<li>Incubate the plate upside-down in a 30°C incubator in the dark until individual colonies have approximately 0.2–0.3 cm diameter (usually about 2 days).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The plate can be stored at 4°C for up to 3 months.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse anti-Pab1 (1:5,000)</td>
<td>antibodies-online GmbH</td>
<td>Cat#ABIN1580454</td>
</tr>
<tr>
<td>Mouse anti-Pgk1 - clone 22C5D8 (1:5,000)</td>
<td>AbCam</td>
<td>Cat#ab113687; RRID:AB_10861977</td>
</tr>
<tr>
<td>Mouse anti-Act1 (1:2,500)</td>
<td>MP Biomedicals</td>
<td>Cat#0869100; RRID:AB_2335304</td>
</tr>
<tr>
<td>PAP reagent (1:5,000)</td>
<td>Sigma</td>
<td>Cat#P1291; RRID:AB_1079562</td>
</tr>
<tr>
<td>HRP-conjugated sheep anti-mouse IgG (1:5,000)</td>
<td>Amersham</td>
<td>Cat#NA931; RRID:AB_772210</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>cOmplete EDTA-free protease inhibitors</td>
<td>Roche</td>
<td>11873580001</td>
</tr>
<tr>
<td>RNasin® Ribonuclease inhibitor</td>
<td>Promega</td>
<td>N2511</td>
</tr>
<tr>
<td>Polyadenylic acid potassium salt (Poly(A))</td>
<td>Sigma</td>
<td>P9403</td>
</tr>
<tr>
<td>RNase ONE</td>
<td>Promega</td>
<td>M4265</td>
</tr>
<tr>
<td>peqGREEN RNA/DNA dye</td>
<td>Peqlab</td>
<td>732–3196</td>
</tr>
<tr>
<td>Precision Plus Protein Dual Color Standards</td>
<td>BIO-RAD</td>
<td>1610374</td>
</tr>
<tr>
<td>Tris</td>
<td>Thermo Fisher</td>
<td>75825</td>
</tr>
<tr>
<td>HCl</td>
<td>Sigma</td>
<td>320331</td>
</tr>
<tr>
<td>Lithium chloride</td>
<td>Acros Organics</td>
<td>413275000</td>
</tr>
<tr>
<td>EDTA</td>
<td>Sigma</td>
<td>E5134</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Acros Organics</td>
<td>327372500</td>
</tr>
<tr>
<td>DTT</td>
<td>Sigma</td>
<td>43816</td>
</tr>
<tr>
<td>Tween-20</td>
<td>BIO-RAD</td>
<td>1706531</td>
</tr>
<tr>
<td>Skim milk powder</td>
<td>Sigma</td>
<td>70166</td>
</tr>
<tr>
<td>Yeast Extract</td>
<td>Oxoid</td>
<td>LP0021</td>
</tr>
<tr>
<td>Peptone</td>
<td>Oxoid</td>
<td>LP0037</td>
</tr>
<tr>
<td>Dextrose</td>
<td>Sigma</td>
<td>D9434</td>
</tr>
<tr>
<td>Agar</td>
<td>Oxoid</td>
<td>LP0028</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>ZR RNA MiniPrep kit</td>
<td>Zymo Research</td>
<td>R1065</td>
</tr>
<tr>
<td>Dynabeads™ mRNA DIRECT™ Purification Kit</td>
<td>Life Technologies</td>
<td>61011</td>
</tr>
<tr>
<td>Transcriptor High Fidelity cDNA Synthesis Kit</td>
<td>Roche</td>
<td>05091284001</td>
</tr>
<tr>
<td>ProteoSilver™ Plus Silver Stain Kit</td>
<td>Sigma-Aldrich</td>
<td>PROTSIL2-1KT</td>
</tr>
<tr>
<td colspan="3">Experimental models:Organisms/strains</td>
</tr>
<tr>
<td>
<i>Saccharomyces cerevisiae</i> strain BY4741: MAT<b>a</b> <i>his3</i>Δ<i>1 leu2</i>Δ<i>0 met15</i>Δ<i>0 ura3</i>Δ<i>0</i>
</td>
<td>Euroscarf collection</td>
<td>Y00000</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>ACT1_Fw,5′-GTCTGGATTGGTGGTTCTATC-3′</td>
<td>This lab</td>
<td>N/A</td>
</tr>
<tr>
<td>ACT1_Rev, 5′-GGACCACTTTCGTCGTATTC-3′</td>
<td>This lab</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Polyvinylidene difluoride (PVDF) membranes</td>
<td>Thermo Fisher</td>
<td>88518</td>
</tr>
<tr>
<td>Microcon-10 kD Centrifugal Filter Unit with Ultracel-10 membrane</td>
<td>Millipore</td>
<td>MRCPRT010</td>
</tr>
<tr>
<td>4%–15% Mini-PROTEAN® TGX™ Precast Protein Gels</td>
<td>BIO-RAD</td>
<td>4561083</td>
</tr>
<tr>
<td>0.22 μm EMD Millipore Steriflip</td>
<td>Millipore</td>
<td>SCGP00525</td>
</tr>
<tr>
<td>Disposable Sterile Filter Systems</td>
<td>Corning</td>
<td>431097</td>
</tr>
<tr>
<td>Dyna-Mag-2 Magnet</td>
<td>Invitrogen</td>
<td>12321D</td>
</tr>
<tr>
<td>Eppendorf Protein LoBind microcentrifuge tubes</td>
<td>Sigma</td>
<td>Z666505</td>
</tr>
<tr>
<td>50 mL Corning Centrifuge Tubes</td>
<td>Corning</td>
<td>CLS430291</td>
</tr>
<tr>
<td>15 mL Corning Centrifuge Tubes</td>
<td>Corning</td>
<td>CLS430791</td>
</tr>
<tr>
<td>Disposable Inoculation Loop, 10 μL</td>
<td>Fischer Scientific</td>
<td>12870155</td>
</tr>
<tr>
<td>Quick Start™ Bradford 1<b>×</b> Dye Reagent</td>
<td>BIO-RAD</td>
<td>5000205</td>
</tr>
<tr>
<td>Ponceau-S</td>
<td>Sigma</td>
<td>P3504</td>
</tr>
<tr>
<td>Amicon® Ultra-4 Centrifugal Filter Unit</td>
<td>Millipore</td>
<td>UFC8100</td>
</tr>
<tr>
<td>RNase ZAP<sup>TM</sup> RNase Decontamination Solution</td>
<td>Thermo Fisher</td>
<td>AM9780</td>
</tr>
<tr>
<td>RNase/DNase free Safe-lock 2 mL microcentrifuge tubes</td>
<td>Thermo Fisher</td>
<td>10031282</td>
</tr>
<tr>
<td>15 cm Petri Dish</td>
<td>Thermo Fisher</td>
<td>P5981</td>
</tr>
<tr>
<td>Glass beads 0.5 mm</td>
<td>Biospec</td>
<td>11079105</td>
</tr>
<tr>
<td>Disposable plastic cuvettes for spectrophotometer</td>
<td>Delle industrie</td>
<td>01938–00</td>
</tr>
<tr>
<td>Western blot filter paper</td>
<td>Thermo Fisher</td>
<td>84783</td>
</tr>
<tr>
<td>Immobilon® Western Chemiluminescent HRP Substrate</td>
<td>Immobilon</td>
<td>WBKLS0500</td>
</tr>
<tr>
<td>Analog (SHKA-) MaxQTM Floor Shaker Incubator</td>
<td>Thermo Fisher</td>
<td>4358</td>
</tr>
<tr>
<td>3L Erlenmeyer flasks</td>
<td>Fischer Brand</td>
<td>FB33136</td>
</tr>
<tr>
<td>100 mL Erlenmeyer flasks</td>
<td>Fischer Brand</td>
<td>FB33131</td>
</tr>
<tr>
<td>Spectrophotometer (UV/Vis single holder)</td>
<td>Jenway</td>
<td>6715</td>
</tr>
<tr>
<td>The Stratalinker® UV crosslinker</td>
<td>Stratagene</td>
<td>1800</td>
</tr>
<tr>
<td>TissueLyser II</td>
<td>Qiagen</td>
<td>RETSCH MM200</td>
</tr>
<tr>
<td>TissueLyser Adapter Set 2 <b>×</b> 24</td>
<td>Qiagen</td>
<td>69982</td>
</tr>
<tr>
<td>Refrigerated centrifuge</td>
<td>Eppendorf</td>
<td>5804R</td>
</tr>
<tr>
<td>Swing-bucket rotor</td>
<td>Eppendorf</td>
<td>A-4-44</td>
</tr>
<tr>
<td>Fixed angle rotor</td>
<td>Eppendorf</td>
<td>FA-45-30-11</td>
</tr>
<tr>
<td>NanoDrop2000 Microvolume Spectrophotometer</td>
<td>Thermo Fisher</td>
<td>ND-2000</td>
</tr>
<tr>
<td>Trans-Blot® Turbo™ Transfer System</td>
<td>BIO-RAD</td>
<td>1704150</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Yeast media</h3>
<div class="note">
<span class="note-title">Note:</span> Distilled (d) water (H<sub>2</sub>O) was used to prepare yeast media.</div>
<div class="note">
<span class="note-title">Note:</span> Yeast media must be autoclaved before use for 20 minutes at 121°C.</div>
<table id="undtbl1">
<thead>
<tr>
<th colspan="3">YPD media</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast Extract</td>
<td>1%</td>
<td>10 g</td>
</tr>
<tr>
<td>Peptone</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>Dextrose</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>dH<sub>2</sub>O</td>
<td>n/a</td>
<td>1,000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare the media, yeast extract and peptone (YEP) is dissolved in 900 mL of dH<sub>2</sub>O and autoclaved. To avoid the Maillard reaction (a chemical reaction between amino acid and reducing sugar that will change the medium into a brownish color, dropping the actual sugar concentration (<a href="#bib16">Wang and Hsiao, 1995</a>)), it is recommended to dissolve the 20 g dextrose separately in 100 mL dH<sub>2</sub>O and sterile filter (0.22 μm). 100 mL of the sterile 20% dextrose solution is finally added to 900 mL YEP under sterile conditions, obtaining 1 L of YPD media.</p>
<div class="note">
<span class="note-title">Note:</span> Media and dextrose stock solutions can be stored for up to one year at room temperature (22°C–25°C).</div>
<table id="undtbl2">
<thead>
<tr>
<th colspan="3">YPD agar</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast Extract</td>
<td>1%</td>
<td>10 g</td>
</tr>
<tr>
<td>Peptone</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>Dextrose</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>Agar</td>
<td>2%</td>
<td>20 g</td>
</tr>
<tr>
<td>dH<sub>2</sub>O</td>
<td>n/a</td>
<td>1,000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave ingredients without dextrose in 900 mL dH<sub>2</sub>O in a 1 L glass bottle. After autoclaving, add 100 mL of 20% sterile dextrose solution. Keep the bottle with YPD agar at least at 55°C in a water-bath to avoid media solidification. Pour plates under sterile conditions and prevent the formation of air bubbles (approximately 20 mL of YPD agar per 10 cm standard dish). Solid YPD plates must be stored at 4°C and protected from light.</p>
<div class="note">
<span class="note-title">Note:</span> Pour the plates in a fume hood or in the vicinity of a Bunsen burner flame. The flame can also be used to remove air-bubbles from the surface.</div>
<h3 id="sec2.2">Buffers for poly(A) RIC</h3>
<div class="note">
<span class="note-title">Note:</span> Double-distilled (dd) RNase-free water (H<sub>2</sub>O) was used through the procedure.</div>
<div class="note">
<span class="note-title">Note:</span> Buffers are preferentially prepared freshly from sterile stock solutions and filtered (0.22 μm).</div>
<table id="undtbl3">
<thead>
<tr>
<th colspan="3">Lysis buffer</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl, pH 7.5 (1 M)</td>
<td>100 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>LiCl (3 M)</td>
<td>500 mM</td>
<td>1.66 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>10 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>Triton X-100 (20%)</td>
<td>1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>5 mM</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>RNasin (40 U/μL)</td>
<td>100 U/mL</td>
<td>0.025 mL</td>
</tr>
<tr>
<td>Complete EDTA-free protease-inhibitor cocktail (10×)</td>
<td>1×</td>
<td>1 tablet</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>6.365 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at 4°C for up to 1 day without inhibitors. Inhibitors must be freshly added before use.</p>
<table id="undtbl4">
<thead>
<tr>
<th colspan="3">Wash buffer A</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl, pH 7.5 (1 M)</td>
<td>10 mM</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>LiCl (3 M)</td>
<td>600 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>1 mM</td>
<td>0.02 mL</td>
</tr>
<tr>
<td>Triton X-100 (20%)</td>
<td>0.1%</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>7.83 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at room temperature for up to 1 month.</p>
<div class="note">
<span class="note-title">Note:</span> Wash buffer A can be supplemented with 10 U/mL of RNasin if RNA degradation is of concern (<a href="#sec6.1">troubleshooting problem 1</a>). Add 0.2% lithium dodecyl sulphate (LiDS) to increase stringency if <a href="#sec6.5">troubleshooting problem 3</a> is of concern.</div>
<table id="undtbl5">
<thead>
<tr>
<th colspan="3">Wash buffer B</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl, pH 7.5 (1 M)</td>
<td>10 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>LiCl (3 M)</td>
<td>600 mM</td>
<td>4 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>1 mM</td>
<td>0.04 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>15.76 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at room temperature for up to 1 month.</p>
<table id="undtbl6">
<thead>
<tr>
<th colspan="3">Elution buffer</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl, pH 7.5 (1 M)</td>
<td>10 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>9.8 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at room temperature for up to 1 month.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Cell culture and UV irradiation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 days</div>
<p>In this step, the cells are UV-irradiated to crosslink RNA-protein interactions <i>in vivo</i>, enabling the subsequent capture of both permanent and transient binding proteins. A sample with non-irradiated cells can be included to control the crosslinking. Other controls, such as the poly(A) competition used in this protocol are introduced in the next step of the protocol.</p>
<ol>
<li>Pick 1 yeast colony from the YPD agar plate with a sterile loop and inoculate 20 mL YPD media placed in a sterile 100 mL Erlenmeyer flask. Place the flask in a shaker and grow cells overnight (12–14 h) at 30°C with constant shaking at 220 rounds per minute (r.p.m.).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To ensure good aeration of cells, we recommend using at least 1:5 ratio of culture volume to Erlenmeyer flask volume (i.e., 20 mL of culture in a 100 mL Erlenmeyer flask).</div>
<ol start="2">
<li>Refresh culture in 500 mL YPD placed in a sterile 3 L Erlenmeyer flask, setting OD<sub>600</sub> ∼0.1, and grow at 30°C at 220 r.p.m to OD<sub>600</sub> ∼0.6. This takes about 3 h 30 min (doubling time of cells ∼90 min).
<ol type="a">
<li>Set the spectrophotometer at 600 nm wavelength.</li>
<li>Set up a blank cuvette by placing 1 mL YPD in a disposable plastic cuvette.</li>
<li>Place the blank cuvette in the spectrophotometer and take the blank measurement (absorbance (A) = 0.00).</li>
<li>Make a 1:10 dilution of the 20 mL pre-culture and measure the absorbance.</li>
<li>Calculate the volume of pre-culture required to inoculate 500 mL YPD with a starting OD<sub>600</sub> ∼0.1.</li>
<li>Remove appropriate volume of pre-culture with a pipette and transfer cells to the 500 mL YPD in a sterile 3 L Erlenmeyer flask.</li>
<li>Place the culture in a shaker and grow cells at 30°C at 220 r.p.m. Measure the OD<sub>600</sub> every 2 h until the OD<sub>600</sub> reaches ∼0.4. At this point, cells have passed the lag-phase and reach mid-log phase, so it is advisable to start the measures every 30 min.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cells can also be subjected to stress treatments to eventually monitor changes in RNA-protein associations as compared to untreated cells. For instance, we treated yeast cells at mid-log phase (i.e. OD<sub>600</sub> ∼0.6) with 0.5 mM hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>) for 15 min to induce a mild oxidative stress response (<a href="#bib8">Matia-González et al., 2021</a>). However, it is essential to confirm that an appropriate stress response has been induced, which can be assessed by monitoring the expression of stress-specific markers.</div>
<ol start="3">
<li>Irradiate cells with UV light at 254 nm to crosslink RNA-protein interactions.
<ol type="a">
<li>Harvest cells in 500 mL centrifuge tubes at OD<sub>600</sub> ∼0.6 by centrifugation at 3,000 × <i>g</i> in a swing-bucket rotor for 3 min at room temperature (22°C–25°C).</li>
<li>Wash cells three times at room temperature with 25 mL phosphate-buffered-saline (PBS). Collect cells in between by centrifugation as indicated in the previous step.</li>
<li>Resuspend cells in 15 mL 1× PBS.</li>
<li>Spread the cell-suspension in a 15 cm petri dish.</li>
<li>Place the dish on ice in the UV-crosslinker. Expose cells to 3 × 400 mJ/cm<sup>2</sup> of 254 nm UV-light with two 2-min breaks and gentle mixing between each exposure.</li>
<li>Harvest cells by centrifugation at 3,000 × <i>g</i> for 3 min at 4°C.</li>
</ol>
</li>
<li>Snap freeze pellets in liquid nitrogen.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Cell pellets can be stored at −80°C for up to two months.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is important that cells are evenly spread in the dish, making a thin layer of cells for exposure to UV-light.</div>
<h3 id="sec3.2">Lysate preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h 30 min</div>
<p>Cell lysates are prepared by mechanical disruption of cells with glass beads in a TissueLyser.</p>
<ol start="5">
<li>Resuspend cells in 4 mL lysis buffer by pipetting up and down on ice and split cells in 4 × 2 mL safe-lock microcentrifuge tubes.</li>
<li>Add 2/3 vol. glass beads to each tube.</li>
<li>Place the tubes balanced in a TissueLyser adapter and break cells at 30 Hz for 10 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The TissueLyser is preferentially placed in the cold room to maintain samples at 4°C whilst lysis. Tubes must be balanced within the TissueLyser adapter set to avoid motor failures.</div>
<ol start="8">
<li>Clear lysate by three sequential centrifugations at 3,000 × <i>g</i> for 3 min, and 5,000 × <i>g</i> and 10,000 × <i>g</i> for 5 min each at 4°C. Carefully remove the supernatant with a filter tip and transfer to a fresh microcentrifuge tube after each centrifugation, avoiding any pellet contamination.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Repeated centrifugation at progressively higher-speed removes non-lysed whole cells (3,000 × <i>g</i>), cell debris (5,000 × <i>g</i>) and nuclei and cytoskeletons (10,000 × <i>g</i>).</div>
<ol start="9">
<li>After the last centrifugation, combine all supernatants in a 15 mL Corning centrifuge tube.</li>
<li>Quantify protein concentration of the final extract with a Bradford assay taking bovine serum albumin (BSA) as a reference standard following the <a href="https://www.bio-rad.com/webroot/web/pdf/lsr/literature/4110065A.pdf">manufacturer’s guidelines</a>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Typically, we obtained a concentration of ∼3.5 mg/mL protein.</div>
<ol start="11">
<li>Concentrate protein sample up to 10 mg/mL with an Amicon® Ultra-4 Centrifugal Filter Unit 10 kDa, following the <a href="https://www.merckmillipore.com/ES/es/product/Amicon-Ultra-4-Centrifugal-Filter-Unit,MM_NF-UFC801024?bd=1">manufacturer’s guidelines</a>. Ensure that the final volume does not go below 1.2 mL. Prepare 500 μL aliquots containing ∼5 mg of protein lysate (∼10 mg/mL) in Eppendorf Protein LoBind microcentrifuge tubes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure the membrane of the Centrifugal Unit is not getting damaged with the pipette tip. Quantify protein amount after concentration as explained in step 10 and ensure that proteins have not aberrantly leaked through a damaged membrane.</div>
<ol start="12">
<li>Keep 75 μL of the extract as reference. 50 μL will be used to assess the RNA quality and the remaining can be used for protein analysis (see below).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Lysates can be kept on ice and immediately used for later steps in the protocol, or snap frozen in liquid nitrogen and stored at −80°C for up to one month.</div>
<p><b>Control step</b>: Analysis of RNA integrity before proceeding with poly(A) RIC.</p>
<ol start="13">
<li>RNA quality control.
<ol type="a">
<li>Isolate total RNA from 50 μL of extract (=input) with the ZR RNA MiniPrep kit, following <a href="https://www.bioscience.co.uk/userfiles/pdf/ZR%20RNA%20MiniPrep%E2%84%A2.pdf">manufacturer’s guidelines</a>.</li>
<li>Quantify RNA with a Nanodrop ND-2000 device.</li>
<li>Visualize 1 μg RNA on a 1% agarose gel stained with peqGREEN RNA/DNA dye.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Typically, we obtained 30 μg of total RNA from 50 μL extract.</div>
<div class="critical">
<span class="critical-title">Critical:</span> We highly recommend checking RNA integrity after UV-irradiation by comparing the pattern with a non-crosslinked sample. Ribosomal RNA (rRNA) bands should be clearly visible before proceeding with the poly(A) RNA isolation (<a href="#fig1">Figure 1</a>A, lanes 1 and 2). <a href="#sec6.1">Troubleshooting problem 1</a>.</div>
<p>In addition, when RNase ONE is included as negative control, RNA integrity after digestion must be analyzed to ensure that RNA is completely degraded and rRNA species cannot be distinguished and only a smear can be visualized (<a href="#fig1">Figure 1</a>A, lane 3).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1142-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. RNA analysis after UV-irradiation of cells</div>
<p>(A) Testing UV-irradiation and RNase control digest. One μg of total RNA was electrophoresed on a 1% agarose gel and stained with peqGREEN DNA/RNA dye. Lane 1: RNA from non-irradiated cells; Lane 2: RNA from UV-irradiated cells. Lane 3: RNA from UV-irradiated cells and RNase ONE treated extracts. M: molecular weight marker.</p>
<p>(B) RNA degradation is prevented by UV-irradiation on ice. Lane 1: RNA after exposure of cells to 1,200 mJ without any breaks. Lane 2: one break on ice for 2 min after 600 mJ. Lane 3: 2 breaks on ice (after 400 mJ). M: molecular weight marker.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Total RNA can be stored at −80°C for further analysis.</div>
<h3 id="sec3.3">Poly(A) RNA interactome capture</h3>
<p>This step describes the recovery of poly(A) RNA and interacting proteins with oligo[dT]<sub>25</sub> magnetic beads from cell extracts. Isolation of poly(A) RNA will mainly recover mRNA binding proteins, allowing for analysis of the cellular mRBPome. Stringent washing conditions are used to avoid isolation of unspecific binders.</p>
<div class="note">
<span class="note-title">Note:</span> As negative control for this protocol, we recommend the addition of an excess of polyadenylic acids (poly(A)) to the sample as a competitor. As an alternative, RNA is digested in the extract with an RNase. Optionally, a non-crosslinked sample has been used in a related protocol (<a href="#bib2">Beckmann et al., 2015</a>).</div>
<div class="timing">
<span class="timing-title">Timing:</span> 6–8 h</div>
<ol start="14">
<li>Each experiment requires 500 μL of the yeast cell lysate/extract (10 mg/mL; ∼5 mg of protein lysate).</li>
<li>Control sample with poly(A) competitor: Add 20 μL of poly(A) (10 mg/mL) to 500 μL extract, briefly vortex sample and place on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Optionally, an RNase digested control sample is prepared. Therefore, add 100 U RNase ONE to 500 μL of extract (∼ 5 mg protein lysate), briefly vortex and incubate for 2 h at 37°C.</div>
<ol start="16">
<li>Equilibrate one milligram of oligo[dT]<sub>25</sub> Dynabeads in lysis buffer. Add 500 μL lysis buffer to the beads and vortex. Place tubes on the Dyna-Mag-2 magnet for at least 30 s and remove supernatant with a tip. Repeat three times and finally resuspend beads in 500 μL lysis buffer until use.</li>
<li>Remove lysis buffer from the pre-equilibrated oligo[dT]<sub>25</sub> Dynabeads by placing the tubes on the Dyna-Mag-2 magnet. Add 500 μL of the extract and incubate the tubes on a shaker/rotating wheel for 10 min at room temperature.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend using Eppendorf Protein LoBind microcentrifuge tubes for the poly(A) RIC protocol.</div>
<ol start="18">
<li>Retrieve the beads by placing the tubes in a Dyna-Mag-2 Magnet for at least 30 s and carefully remove the supernatant and transfer to a fresh low-bind tube.</li>
<li>Wash beads once with wash buffer A. Add 500 μL of pre-cooled wash buffer A on ice and gentle vortex for 10 s at room temperature.</li>
<li>Wash beads twice with wash buffer B. Add 500 μL of pre-cooled wash buffer B on ice and gentle vortex for 10 s at room temperature.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Repeated washing is required to remove unspecific binders from RNA, the beads and the tube wall. Wash buffer B also removes traces of Triton X-100, which may interfere with absorbance measurements and compromise elution efficiency (see step 25 below). Ensure complete removal of wash buffer at each step. We also recommend changing the tubes at least once during the procedure (e.g., between washes with buffer B).</div>
<ol start="21">
<li>Elute poly(A) RNA from beads. Add 60 μL of pre-heated elution buffer to the beads and incubate in a Thermoblock for 2 min at 80°C. Place tubes in a Dyna-Mag-2 Magnet and quickly transfer the eluate to a fresh tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Incubation at 80°C denatures the hybrid between poly(A) RNA and oligo[dT]<sub>25</sub> beads, thereby releasing the poly(A) RNA into the solution (=eluate). Therefore, the eluate must be collected as quickly as possible to prevent re-association of eluted poly(A) RNA on beads. Elution from beads at 80°C follows the manufacturer’s suggestions. Other bead suppliers may use different temperatures.</div>
<div class="note">
<span class="note-title">Note:</span> An alternative for the elution is to release proteins via RNase digestion (see <a href="#bib2">Beckmann et al., 2015</a>).</div>
<ol start="22">
<li>Re-equilibrate the ‘eluted’ oligo[dT]<sub>25</sub> Dynabeads in lysis buffer as in step 16.</li>
<li>Repeat step 17–22 twice by reapplying the supernatant to the oligo[dT]<sub>25</sub> beads.</li>
<li>Combine the three sequential eluates (∼180 μL) and concentrate to ∼70 μL in a 0.5 mL Microcon-10 kD Centrifugal Filter Unit with Ultracel-10 membrane in a microfuge (14,000 × <i>g</i>, ∼30 min, 4°C).</li>
<li>Quantify eluted poly(A) RNA with a Nanodrop ND-2000 device.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We usually obtained ∼55 ng/ μL of poly(A) RNA. The 260/280 nm ratio should be at least 1.8:1. Lower ratios and an absorbance peak at 230 nm may relate to residual contamination from Triton X-100 in the eluate (see <a href="#sec6.3">troubleshooting problem 2</a>).</div>
<p><b>Control step</b>: Successful poly(A) RNA/ mRNA isolation can be assessed with RT-PCR.</p>
<ol start="26">
<li>Detection of mRNAs in poly(A) eluates by reverse-transcription (RT) PCR (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1142-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. RNA and protein analysis of poly(A) RIC samples</div>
<p>(A) RT-PCR on total RNA isolated from the extract (input) and RIC-eluates with <i>ACT1</i> specific primers. Products were resolved on a 2% agarose gel and stained with peqGREEN DNA/RNA dye. Poly(A) designates the addition of excess competitor poly(A). A control reaction without RT was included to monitor potential DNA contamination. M: molecular weight marker.</p>
<p>(B) Silver stained PAA gel. Lanes 1–2 correspond to 0.05% of input extract and poly(A) treated control samples; lanes 3–4, 10% of eluates from poly(A) mRNA isolation. A Marker (M) with molecular weights (MW) in kilodaltons (KDa) is indicated to the left.</p>
<p>(C) Immunoblot analysis to monitor the indicated proteins in 0.1% of the inputs (lanes 1–2) and 40% of the RIC eluates (lanes 3–4). Pab1, poly(A) binding protein 1; Puf3:TAP, tandem affinity purification-tagged Pumilio family protein; Pgk1, 3-phosphoglycerate kinase (a non-conventional RBP); Act1, actin (non-RNA binding control protein). Poly(A) designates the addition of excess competitor poly(A). MW is indicated to the left.</p>
</figcaption>
</figure>
<ol type="a">
<li>500 ng of input total RNA (Step 13.a) or 500 ng of eluted RNA (∼ 9.5 μL) are combined with a mixture of oligo[dT]<sub>18</sub> and random hexamer primers for RT with the Transcriptor High Fidelity cDNA Synthesis Kit (Roche) following the <a href="http://www.protocol-online.org/forums/uploads/monthly_01_2011/post-14721-045604100%201294579818.ipb">manufacturer’s guidelines</a>.</li>
<li>PCR is conducted with 1 μL of complementary DNA (cDNA) with the following conditions:
<table id="undtbl7">
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>94°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="3">25–35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>57°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>40 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>8 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>PCR products are visualized on a 1.5% agarose gel stained with peqGREEN RNA/DNA dye.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Poly(A) RNAs captured from negative control samples, either through poly(A) competition or RNase digestion should be included in the analysis to ensure that no mRNA is amplified in these samples (<a href="#fig2">Figure 2</a>A).</div>
<h3 id="sec3.4">Detection of RNA-binding proteins: Silver stained gels and immunoblots</h3>
<p>In this step, proteins bound to poly(A) RNAs are electrophoresed on SDS-polyacrylamide (PAA) gels and stained with silver (<a href="#fig2">Figure 2</a>B); and specific proteins are detected by immunoblot analysis (<a href="#fig2">Figure 2</a>C).</p>
<div class="note">
<span class="note-title">Note:</span> It is important to analyze proteins in the extract and RIC eluates. A silver stained PAA gel can ensure that the protein composition is different in the total extract, RIC eluates and negative control samples. Immunoblot analysis is used to monitor the presence of RBPs in the RIC eluates and not in the negative controls. These RBPs include the poly(A)-binding protein 1 (Pab1), which is known to be involved in regulating mRNAs fate and stability (<a href="#bib1">Amrani et al., 1997</a>; <a href="#bib12">Sachs et al., 1987</a>), and the PUmilio-homology domain Family 3 protein (Puf3), known to be abundantly associated with mRNAs to be involved in their localization and decay (<a href="#bib10">Olivas and Parker, 2000</a>; <a href="#bib4">Gerber et al., 2004</a>). A non-RBP can act as a negative control (e.g., actin1). These tests indicate whether mRBPome isolation has been successful.</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="27">
<li>Analyze the protein complexity with a silver-stained PAA gel (<a href="#fig2">Figure 2</a>B).
<ol type="a">
<li>Resolve 0.05% of the input (=extract/lysate) and at least 10% of the eluates on a 4–15% Mini-PROTEAN® TGX™ Precast SDS-PAA gel.</li>
<li>Run the PAA-gel at 120 V for 90 min in the appropriated electrophoresis chamber. Regularly check the progress of the sample migration.</li>
<li>Stain gel with ProteoSilver™ Plus Silver Stain Kit, following the <a href="https://www.sigmaaldrich.com/deepweb/assets/sigmaaldrich/product/documents/156/682/protsil1bul.pdf">manufacturer’s guidelines</a>.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> One lane of the gel should contain a protein ladder as a molecular weight marker (<i>e.g.</i> Precision Plus Protein Dual Color Standards). We recommend loading up to 3 μL of a 1:10 dilution of the protein ladder to avoid an overstaining.</div>
<ol start="28">
<li>Identify specific proteins with immunoblots (<a href="#fig2">Figure 2</a>C).
<ol type="a">
<li>Resolve 0.1% of the input and at least 10% of the eluate on a 4%–15% Mini-PROTEAN® TGX™ Precast SDS-PAA Protein Gels.</li>
<li>Run the gel for 90 min at 120 V.</li>
<li>Cut a gel-sized polyvinylidene difluoride (PVDF) membrane and activate it with 100% methanol for 2 min.</li>
<li>Soak the gel, filter papers, and PVDF membranes in the appropriate transfer buffer for 2 min at room temperature.</li>
<li>Transfer proteins from the PAA gel to PVDF membranes with a Trans-Blot® Turbo™ Transfer System (or equivalent).
<div class="note">
<span class="note-title">Note:</span> Voltage and length of transfer may vary depending on the molecular weight of the protein of interest. Proteins with high molecular weight (&gt;200 kDa) need longer transfer time.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend checking for transfer of proteins by staining the membrane with Ponceau-S solution according to <a href="https://isg.ku.edu.tr/sites/isg.ku.edu.tr/files/laboratuvar/scib20/Msds/PONCEAUStainingSOLUTION.pdf">manufacturer’s instructions</a>. We also recommend loading a pre-stained protein marker (e.g. Precision Plus Protein Dual Color Standards).</div>
</li>
<li>Block membrane in 1× PBS-0.1% Tween-20 (PTST) containing 5% low fat milk for 1 h at room temperature.</li>
<li>Probe membrane with respective primary antibodies (e.g., anti-Pab1 (1:5,000), anti-Pgk1 (1:5,000), anti-actin (1:2,500), PAP reagent (1:5,000)) diluted in PTST containing 2% low fat milk, for 1 h at room temperature with constant shaking.</li>
<li>Wash the membrane 3 times in PBST for 10 min each at room temperature with constant shaking.</li>
<li>Add the corresponding horseradish peroxidase (HRP)-coupled secondary antibodies (1:5,000) diluted in PTST containing 2% low fat milk for 1 h at room temperature with constant shaking.</li>
<li>Wash the membrane 3 times in PBST for 10 min each at room temperature with constant shaking.</li>
<li>Develop membrane with the Immobilon Western Chemiluminescent HRP Substrate, following the <a href="https://www.sigmaaldrich.com/GB/en/product/mm/wbkls">manufacturer’s guidelines</a>.</li>
<li>Take a picture of the chemiluminescent signals with standard gel imaging system.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Immunoblotting has been described in detail (<a href="#bib6">Litovchick, 2020</a>; <a href="#bib3">Burckhardt et al., 2021</a>). Of note, proteins (i.e. Pab1) may shift in the gel to a higher molecular weight due to crosslinks to RNA.</div>
<div class="note">
<span class="note-title">Note:</span> Use of fluorescently labeled antibodies and detection with a fluorescence scanner may be considered to obtain a more quantitative read-out.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The herein described processing of 500 mL yeast cells yields ∼12 mg of protein in the extract and ∼7.5 μg in poly(A) RIC eluates. Thus, about 0.06% of protein from the total protein extracts is usually recovered with this RIC protocol. The amount of captured proteins is sufficient to proceed with label- free mass spectrometry (MS) analysis, allowing for identification of the yeast mRBPome under different stress conditions (<a href="#bib7">Matia-González et al., 2015</a>, <a href="#bib8">2021</a>). Of note, this protocol is not biased towards the identification of canonical RNA-binding proteins that contain characteristic RNA-binding domains, as our MS data revealed multiple unconventional RBPs, lacking such domains including many metabolic enzymes (<a href="#bib8">Matia-González et al., 2021</a>).</p>
<p>A successful poly(A) RIC enables the monitoring of particular mRNAs, such as <i>ACT1</i> mRNA, in the RIC eluates directly by RT-PCR (<a href="#fig2">Figure 2</a>A), adding the input (total lysate) as reference. Poly(A) RNAs captured from negative control samples, either through poly(A) competition or RNase digestion should be included in the analysis to ensure that no mRNA is amplified in these samples (<a href="#fig2">Figure 2</a>A). In addition, a different protein pattern in the RIC eluate when compared with input extract is observed as result of the poly(A) RIC. Hardly any proteins should be detected in the poly(A) competition control experiment (<a href="#fig2">Figure 2</a>B). Of note, using different type of negative controls than the one used in this protocol, such as non-crosslinked samples, might result in the detection of some protein bands in the eluates (<a href="#sec6.5">troubleshooting problem 3</a>). Well known RBPs, such as Pab1 or Puf3, as well as non-conventional RBPs including the metabolic enzyme Pgk1, can be detected in the RIC eluate by immunoblot (<a href="#sec6.7">troubleshooting problem 4</a>). Conversely, proteins not expected to bind poly(A) RNAs such as actin should not be detectable in the eluate fraction (<a href="#fig2">Figure 2</a>C).</p>
<p>By applying this protocol, we have defined the mRBPome in the budding yeast <i>Saccharomyces cerevisiae</i> (<a href="#bib7">Matia-González et al., 2015</a>) comprised of 765 proteins. Moreover, we applied this protocol to define the mRBPome under oxidative stress conditions, identifying 257 proteins that differentially associate with poly(A) RNA (<a href="#bib8">Matia-González et al., 2021</a>). This protocol also helped in revealing a rearrangement of the poly(A) RNA association upon oxidative stress of several metabolic pathways, specially, the ones implicated in carbon metabolism. Furthermore, we found that the RNA-binding capacity of these enzymes was paralogue specific.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is based on capturing poly(A) RNAs from cells, a major component of it comprising mRNAs. Hence, it is less suitable for identification of RBPs that preferentially interact with other RNA species that lack a poly(A) tail, such as rRNAs and tRNAs. Furthermore, the protocol is based on UV-irradiation of cells to crosslink RNA protein interactions <i>in vivo</i>. The efficiency UV-based crosslinking is relatively low (∼5%) and shows some preference for uridine-rich single-stranded RNA regions, possibly adding a bias towards single-stranded RNA-binding proteins. In addition, we wish to note that UV-irradiation can also induce protein-protein and protein-DNA crosslinks - albeit at lower efficiency – but may lead to false positives through indirect interactors (<a href="#bib5">Gerber, 2021</a>). Other types of crosslinking, such as the photoactivatable ribonucleoside-crosslinking (PAR-CL) (<a href="#bib13">Shchepachev et al., 2019</a>) or the formaldehyde-based crosslinking (FA-CL) (<a href="#bib9">Na et al., 2021</a>) may constitute suitable alternatives.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>RNA degradation</p>
<h3 id="sec6.2">Potential solution</h3>
<p>To avoid RNA degradation during UV-irradiation and lysate preparation, it is very important to keep cells on ice (step 3e). UV-light can directly damage RNA and a careful check of the settings and prior testing of the UV-crosslinker is recommended. Since the suspension absorbs the energy from UV-light and warms-up, we recommend keeping cells on ice for 2 min between three UV exposures (400 mJ each) (<a href="#fig1">Figure 1</a>B). After UV-irradiation, immediately collect the cells at 4°C and snap freeze in liquid nitrogen or proceed immediately with the preparation of the extract. Add sufficient amounts of RNase inhibitors to the lysis buffer. If degradation occurs during poly(A) RIC, supplement the wash buffers with 1 U/mL of RNasin.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>UV-spectra of eluted RNA with a 260/280 nm ratio of less than 1.8 alongside peak absorbance at 230 nm (step 25).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>The eluate may still contain some detergents supplied in lysis and wash buffer A (Triton X-100 absorbs in the range of 230 nm and at 280 nm). This can be solved by a quick spin of tubes in a microfuge (1,000 × <i>g</i>) and careful removal of residual wash buffer B (step 20) with a pipette.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Detection of proteins in the eluate of ne<i>g</i>ative controls (steps 26 and 27).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The detection of non-specific RNA-protein interaction in poly(A) competitor or in non-crosslinked samples (data not shown) is due to non-specific protein-protein interactions or non-covalent binding of protein to the RNA. This has been solved by increasing the stringency of the washes during the poly(A) RNA pull down through addition of 0.2% lithium dodecyl sulfate (LiDS) to wash buffer A (step 19).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Absence of signal for a determined protein in the eluate fraction monitored by immunoblots (step 27).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>The detection of proteins in the poly(A) RNA eluate with immunoblots depends on a variety of instances, ranging from biological factors like protein and mRNA target levels and their affinities to technical variables, such as transfer of protein to membrane and quality of antibodies. Therefore, resolving 10% of the RIC eluate may not be sufficient for reliable detection and requires loading of a higher fraction of the eluate (up to 40%). To monitor the success of the experiment we recommend using a positive control such as Pab1 - a highly expressed protein binding to the poly(A) tail of mRNAs. However, in many cases no suitable antibodies for detection of a specific protein may be available. In that case, strains bearing TAP/GFP-tagged proteins can be used, allowing detection of tagged-proteins with highly sensitive commercially available reagents (e.g., Puf3-TAP, <a href="#fig2">Figure 2</a>C). Finally, we wish to note that besides the input and eluate, samples can also be taken from the supernatant (i.e., extract after incubation with oligo[dT]<sub>25</sub> beads) and wash fractions to follow the protein under consideration during the entire procedure.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Amrani, N., Minet, M., Le Gouar, M., Lacroute, F., and Wyers, F. (1997). Yeast Pab1 interacts with Rna15 and participates in the control of the poly(A) tail length <i>in vitro</i>. Mol. Cell Biol. <i>17</i>, 3694-3701. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref1">View at publisher</a></p>
<p id="bib2">Beckmann, B., Horos, R., Fischer, B., Castello, A., Eichelbaum, K., Alleaume, A.M., Schwarzl, T., Curk, T., Foehr, S., Huber, W., et al. (2015). The RNA-binding proteomes from yeast to man harbour conserved enigmRBPs. Nat. Commun. <i>6</i>, 10127. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref2">View at publisher</a></p>
<p id="bib3">Burckhardt, C.J., Minna, J.D., and Danuser, G. (2021). Co-immunoprecipitation and semi-quantitative immunoblotting for the analysis of protein-protein interactions. STAR Protoc. <i>2</i>, 100644. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref3">View at publisher</a></p>
<p id="bib4">Gerber, A.P., Herschlag, D., and Brown, P.O. (2004). Extensive association of functionally and cytotopically related mRNAs with Puf family RNA-binding proteins in yeast. PLoS Biol. <i>2</i>, e79. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref4">View at publisher</a></p>
<p id="bib5">Gerber, A.P. (2021). RNA-centric approaches to profile the RNA–protein interaction landscape on selected RNAs. Noncoding RNA <i>7</i>, 11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref5">View at publisher</a></p>
<p id="bib6">Litovchick, L. (2020). Immunoblotting. Cold Spring Harb. Protoc. <i>2020</i>, 098392. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref6">View at publisher</a></p>
<p id="bib7">Matia-González, A.M., Laing, E.E., and Gerber, A.P. (2015). Conserved mRNA-binding proteomes in eukaryotic organisms. Nat. Struct. Mol. Biol. <i>22</i>, 1027-1033. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref7">View at publisher</a></p>
<p id="bib8">Matia-González, A.M., Jabre, I., Laing, E.E., and Gerber, A.P. (2021). Oxidative stress induces coordinated remodeling of RNA-enzyme interactions. iScience <i>24</i>, 102753. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref8">View at publisher</a></p>
<p id="bib9">Na, Y., Kim, H., Choi, Y., Shin, S., Jung, J.H., Kwon, S.C., Kim, V.N., and Kim, J.S. (2021). FAX-RIC enables robust profiling of dynamic RNP complex formation in multicellular organisms <i>in vivo</i>. Nucleic Acids Res. <i>49</i>, e28. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref9">View at publisher</a></p>
<p id="bib10">Olivas, W. and Parker, R. (2000). The Puf3 protein is a transcript-specific regulator of mRNA degradation in yeast. EMBO J. <i>19</i>, 6602-6611. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref10">View at publisher</a></p>
<p id="bib11">Queiroz, R.M.L., Smith, T., Villanueva, E., Marti-Solano, M., Monti, M., Pizzinga, M., Mirea, D.M., Ramakrishna, M., Harvey, R.F., Dezi, V., et al. (2019). Comprehensive identification of RNA-protein interactions in any organism using orthogonal organic phase separation (OOPS). Nat. Biotechnol. <i>37</i>, 169-178. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref11">View at publisher</a></p>
<p id="bib12">Sachs, A.B., Davis, R.W., and Kornberg, R.D. (1987). A single domain of yeast poly(A)-binding protein is necessary and sufficient for RNA binding and cell viability. Mol. Cell Biol <i>7</i>, 3268-3276. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref12">View at publisher</a></p>
<p id="bib13">Shchepachev, V., Bresson, S., Spanos, C., Petfalski, E., Fischer, L., Rappsilber, J., and Tollervey, D. (2019). Defining the RNA interactome by total RNA-associated protein purification. Mol. Syst. Biol. <i>15</i>, e8689. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref13">View at publisher</a></p>
<p id="bib14">Sherman, F. (2002). Getting started with yeast. Methods Enzymol. <i>350</i>, 3-41. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref14">View at publisher</a></p>
<p id="bib15">Singh, G., Pratt, G., Yeo, G.W., and Moore, M.J. (2015). The clothes make the mRNA: past and present trends in mRNP fashion. Annu. Rev. Biochem. <i>84</i>, 325-354. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref15">View at publisher</a></p>
<p id="bib16">Wang, X.J. and Hsiao, K.C. (1995). Sugar degradation during autoclaving: Effects of duration and solution volume on breakdown of glucose. Physiologia Plantarum <i>94</i>, 415-418. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00635-3/sref16">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, André P. Gerber (<a href="mailto:a.gerber@surrey.ac.uk">a.gerber@surrey.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze datasets/code<i>.</i></p>
<h3>Acknowledgments</h3>
<p>This study was funded by grants from the <a href="https://doi.org/10.13039/501100000268">Biotechnology and Biological Sciences Research Council</a> (BB/K009303/1, BB/N008820/1, BB/S017747/1) and a <a href="https://doi.org/10.13039/501100000288">Royal Society</a> Wolfson Research Merit Award (WM170036) to A.P.G.</p>
<h3>Author contributions</h3>
<p>A.M.M.-G. conceived and designed the protocol, performed laboratory experiments, and wrote the manuscript. I.J. performed laboratory experiments and wrote the manuscript. A.P.G. conceived and designed the protocol and wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>