<article>
<section>
<h1 id="header">A time-resolved fluorescence resonance energy transfer screening assay for discovery of protein-protein interaction modulators</h1>
<p><time datetime="2021-09-10">Published: September 10, 2021</time></p>
<p>Cong Tang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">5</a></sup> Qiankun Niu,<sup><a href="#aff1">1</a></sup> Danielle Cicka,<sup><a href="#aff1">1</a></sup> Yuhong Du,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a></sup> Xiulei Mo,<sup><a href="#aff1">1</a>,<a href="#fn2">6</a>,<a href="#cor1">*</a></sup> and Haian Fu<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn3">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pharmacology and Chemical Biology, Emory University School of Medicine, Atlanta, GA 30322, USA</p>
<p id="aff2"><sup>2</sup>The First Affiliated Hospital, Medical School of Xi'an Jiaotong University, Xi'an, Shannxi 710061, P.R. China</p>
<p id="aff3"><sup>3</sup>Emory Chemical Biology Discovery Center, Emory University School of Medicine, Atlanta, GA 30322, USA</p>
<p id="aff4"><sup>4</sup>Department of Hematology and Medical Oncology and Winship Cancer Institute, Emory University, Atlanta, GA 30322, USA</p>
<p id="fn1"><sup>5</sup>Present address: Instituto de Medicina Molecular, Faculdade de Medicina, Universidade de Lisboa, Avenida Professor Egas Moniz, Lisboa 1649-028, Portugal</p>
<p id="fn2"><sup>6</sup>Technical contact</p>
<p id="fn3"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:xmo@emory.edu">xmo@emory.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:hfu@emory.edu">hfu@emory.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100804">10.1016/j.xpro.2021.100804</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Protein-protein interactions (PPIs) have emerged as promising yet challenging therapeutic targets. A robust bioassay is required for rapid PPI modulator discovery. Here, we present a time-resolved Förster's (fluorescence) resonance energy transfer assay protocol for PPI modulator screening in a 1536-well plate format. We use hypomorph SMAD4<sup>R361H</sup>-SMAD3 PPI as an example to illustrate the application of the protocol for screening of variant-directed PPI inducers. This platform can be readily adapted for the discovery of both small-molecule PPI inducers and inhibitors.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref30">Tang et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for high-throughput PPI inducer and inhibitor discovery</li>
<li>A TR-FRET assay recapitulates mutated SMAD4 interaction with SMAD3</li>
<li>A TR-FRET uHTS platform for variant-directed PPI inducer discovery</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/989-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Protein-protein interactions (PPI) are emerging therapeutic targets in diverse disease areas, such as cancer (<a href="#bib13">Hahn et al., 2021</a>). The discovery of an increasing number of oncogenic PPIs has significantly expanded the therapeutic target space (<a href="#bib2">Boettcher et al., 2018</a>; <a href="#bib12">Grzeskowiak et al., 2018</a>; <a href="#bib15">Ivanov et al., 2017a</a>, <a href="#bib16">2017b</a>; <a href="#bib20">Li et al., 2017</a>; <a href="#bib22">Mo et al., 2017</a>; <a href="#bib27">Rusnak et al., 2018</a>). This development demands highly efficient experimental approaches for accelerated discovery of PPI modulators. The protocol below describes the specific steps of screening for small molecule inducers, molecular glues, for the target-of-interest PPI, SMAD4<sup>R361H</sup>-SMAD3 (<a href="#bib30">Tang et al., 2020</a>), by using a cell lysate-based TR-FRET HTS assay (<a href="#fig1">Figure 1</a>). This protocol does not require purified protein components for efficient assay development and screening implementation (<a href="#bib20">Li et al., 2017</a>). The “molecular glue” is a promising strategy to enhance protein-protein interactions, and re-activate the function of such protein-protein interactions (<a href="#bib14">Hata and Lagna, 2021</a>; <a href="#bib29">Schreiber, 2021</a>). High-throughput screening technology provides the potential to reveal small molecule modulators for targeted PPIs (<a href="#bib18">Janzen, 2014</a>; <a href="#bib21">Macarron et al., 2011</a>). We have also used a similar strategy to discover other PPI inhibitors or inducers (<a href="#bib7">Du et al., 2013</a>; <a href="#bib23">Mo et al., 2019</a>; <a href="#bib31">Xiong et al., 2018</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/989-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic illustration of the design of the TR-FRET assay for monitoring the SMAD4/SMAD3 PPI to discover small-molecule PPI inducers</div>
<p>Anti-Flag-Tb coupled with Flag-SMAD3 serves as the TR-FRET donor and anti-His-D2 coupled with His-SMAD4<sup>R361H</sup> serves as the acceptor. At the basal level, R361H reduces SMAD4 interaction with SMAD3, yielding low TR-FRET signal. Upon treatment with a PPI inducer, the induced SMAD3/SMAD4<sup>R361H</sup> complex formation brings two fluorophores into close proximity (&lt;10 nm), generating a high TR-FRET signal.</p>
</figcaption>
</figure>
<h3 id="sec1.1">Molecular cloning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<ol>
<li>Generation of mammalian gene expression plasmids for expressing Flag- or His-tagged SMAD4 wild-type (WT), SMAD4-R361H mutant and SMAD3, respectively.</li>
<li>Sequencing verification of plasmids.</li>
<li>DNA plasmid purification.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We generated plasmids to exogenously overexpress proteins with affinity tags for fluorophore-conjugated antibody coupling.</div>
<div class="note">
<span class="note-title">Note:</span> We used Gateway cloning technology for plasmid construction by following manufacturer's protocols (<a href="https://tools.thermofisher.com/content/sfs/manuals/gatewayman.pdf">https://tools.thermofisher.com/content/sfs/manuals/gatewayman.pdf</a>). The SMAD4 point mutation R361H was introduced using QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent Technologies) by following manufacturer's protocols (<a href="https://www.agilent.com/cs/library/usermanuals/public/210518.pdf">https://www.agilent.com/cs/library/usermanuals/public/210518.pdf</a>), and the SMAD4 pDONR221 plasmid as DNA template. Other molecular cloning technologies can also be used.</div>
<div class="note">
<span class="note-title">Note:</span> We highly recommend using sequencing verified plasmid prepared from the same batch throughout the protocol.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Untagged endogenous proteins could also be used to couple authenticated fluorophore-conjugated primary antibodies to the proteins of interest (<a href="#bib5">Cui et al., 2014</a>). In this case, the molecular cloning step could be omitted. Purified proteins can be used for fluorophore coupling. In this case, exogenous expression of the protein of interest in cultured cells is not needed.</div>
<h3 id="sec1.2">Culture cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="4">
<li>Culture human embryonic kidney 293T (HEK293T) cells in Dulbecco's modified Eagle's medium, supplemented with 10% fetal bovine serum, 100 IU/mL penicillin, and 100 μg/mL streptomycin, and keep cells growing in a humid environment with 5% CO<sub>2</sub> at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We highly recommend following standard cell culture guideline to maintain cells within low passages at exponential growth phase without other biological contamination.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Tb cryptate-labeled anti-Flag M2 antibody (1:500~1:1000)</td>
<td>Cisbio Bioassays</td>
<td>Cat# 61FG2TLB</td>
</tr>
<tr>
<td>D2-labeled anti-His antibody (1:200~1:500)</td>
<td>Cisbio Bioassays</td>
<td>Cat# 61HISDLF</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Emory Enriched Bioactive Library (EEBL)</td>
<td>ECBDC</td>
<td>N/A</td>
</tr>
<tr>
<td>Protease Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat# P8340</td>
</tr>
<tr>
<td>PIC2 Phosphatase Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat# P5726</td>
</tr>
<tr>
<td>PIC3 Phosphatase Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat# P0044</td>
</tr>
<tr>
<td>Gateway™ LR Clonase™ II enzyme mix</td>
<td>Thermo Fisher</td>
<td>Cat# 11791100</td>
</tr>
<tr>
<td>Linear polyethylenimine (PEI)</td>
<td>Polysciences</td>
<td>Cat# 23966</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>SMAD3 WT in pDONR221</td>
<td>Human ORFeome Library</td>
<td>Clone# IOH27044</td>
</tr>
<tr>
<td>SMAD4 WT in pDONR221</td>
<td>Human ORFeome Library</td>
<td>Clone# IOH3638</td>
</tr>
<tr>
<td>Gateway™ pDEST™26 Vector for His-tag</td>
<td>Invitrogen</td>
<td>Cat# 11809019</td>
</tr>
<tr>
<td>Modified pcDNA3.2-V5-dest for Flag-tag</td>
<td>ECBDC</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>R361H forward primer (5′-CTTCTGGAGGAGATCACTTTTGTTTGGGTCAAC-3′)</td>
<td>Eurofins Genomics</td>
<td>N/A</td>
</tr>
<tr>
<td>R361H reverse complementary primer (5′-GTTGACCCAAACAAAAGTGATCTCCTCCAGAAG-3′)</td>
<td>Eurofins Genomics</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HEK293T</td>
<td>ATCC</td>
<td>Cat# CRL-3216; RRID:CVCL_0063</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle’s Medium</td>
<td>Corning</td>
<td>Cat# 10-013-CV</td>
</tr>
<tr>
<td>Opti-MEM Reduced Serum Medium</td>
<td>Gibco</td>
<td>Cat# 31985062</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism</td>
<td>GraphPad; v7</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td>Bioassay Software</td>
<td>CambridgeSoft</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Others</td>
</tr>
<tr>
<td>Biomek NXP Automated Workstation</td>
<td>Beckman</td>
<td>N/A</td>
</tr>
<tr>
<td>PHERAstar FS reader</td>
<td>BMG Labtech</td>
<td>N/A</td>
</tr>
<tr>
<td>Assay plate 384 well, black with clear flat bottom tissue culture treated low flange, with lid</td>
<td>Corning</td>
<td>Cat# 3764</td>
</tr>
<tr>
<td>1536-Well black solid bottom microplate, with lid</td>
<td>Corning</td>
<td>Cat# 3724</td>
</tr>
<tr>
<td>MultiDrop Combi Dispenser</td>
<td>Thermo Scientific</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Throughout this protocol, we refer to several specific robotic liquid handling equipment for streamlined operation. Other HTS compatible liquid handlers or multi-channel pipettes can also be used.</div>
<table id="undtbl1">
<caption>FRET buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Add to 1000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH7.0)</td>
<td>20 mM</td>
<td>1M</td>
<td>20 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>50 mM</td>
<td>5M</td>
<td>10 mL</td>
</tr>
<tr>
<td>Nonidet P-40</td>
<td>0.01%</td>
<td>10%</td>
<td>1 mL</td>
</tr>
<tr>
<td>MilliQ water</td>
<td> </td>
<td> </td>
<td>969 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C.</p>
<table id="undtbl2">
<caption>0.5% Triton X-100 Cell lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Add to 400 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH8.0)</td>
<td>20 mM</td>
<td>1M</td>
<td>8 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>137 mM</td>
<td>5M</td>
<td>10.96 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>5%</td>
<td>50%</td>
<td>40 mL</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>0.5%</td>
<td>10%</td>
<td>20 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>2 mM</td>
<td>500 mM</td>
<td>1.6 mL</td>
</tr>
<tr>
<td>MilliQ water</td>
<td> </td>
<td> </td>
<td>319.44 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C. Right before use, add protease inhibitor cocktail and PIC2 and PIC3 phosphatase inhibitor cocktail at 1:100 dilution.</p>
<table id="undtbl3">
<caption>1% NP40</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Add to 200 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH8.0)</td>
<td>20 mM</td>
<td>1M</td>
<td>4 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>137 mM</td>
<td>5M</td>
<td>5.48 mL</td>
</tr>
<tr>
<td>Nonidet P-40 (NP40)</td>
<td>1%</td>
<td>10%</td>
<td>20 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>5%</td>
<td>50%</td>
<td>20 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>2 mM</td>
<td>500 mM</td>
<td>800 μL</td>
</tr>
<tr>
<td>MilliQ water</td>
<td> </td>
<td> </td>
<td>150 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C.</p>
<table id="undtbl4">
<caption>PEI stock solution (1 mg/mL)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Add to 100 mL</th>
<th>Note</th>
</tr>
</thead>
<tbody>
<tr>
<td>Polyethylenimine powder</td>
<td>100 mg</td>
<td rowspan="3">1. Filter through 0.22 uM filter membrane.2. Recommend tested for efficiency by transfecting cells with GFP at a 1:1–1:6 DNA:PEI ratio.</td>
</tr>
<tr>
<td>HCl</td>
<td>adjust to pH 7.0</td>
</tr>
<tr>
<td>MilliQ water</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Aliquot 0.5<b>–</b>1 mL to each 1.5 mL tube, and store at −20°C. Thawed solutions can be stored at 4°C for up to 2 months. Avoid frequent freeze-thaw cycles.</p>
<table id="undtbl5">
<caption>Cell culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Add to 500 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM medium</td>
<td> </td>
<td>440 mL</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>2 mM glutamine</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>100 U/mL penicillin/streptomycin</td>
<td>1%</td>
<td>5 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C. Warm to 37°C right before use.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Protein expression and cell lysate stock preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>This major step describes the specific protocol to prepare cell lysate from HEK293T cells co-overexpressing Flag-SMAD3 and His-SMAD4 WT or R631H. We have also used this protocol for overexpressing proteins with various tag configurations in HEK293T cells or other protein expression systems (<a href="#bib7">Du et al., 2013</a>; <a href="#bib23">Mo et al., 2019</a>; <a href="#bib31">Xiong et al., 2018</a>).</p>
<ol>
<li>Seeding HEK293T cells – Day 1
<ol type="a">
<li>Before seeding cells, check cells visually and under microscope to make sure the cells are healthy indicated by observing 1) pinky orange culture media, 2) majority of cells are attached to the bottom of the flask, 3) cells are in plump or elongated shape, and 4) reached 70%–80% confluency in monolayer.</li>
<li>Pre-warm cell culture medium, PBS and trypsin to 37°C in water bath.</li>
<li>Carefully remove media from one 175 cm<sup>2</sup> flask of the required cells into a waste pot (containing laboratory disinfectant), taking care not to increase contamination risk with any drips.</li>
<li>Wash the cells twice with PBS.</li>
<li>Add 3 mL trypsin, incubate in CO<sub>2</sub> incubator at 37°C until cells are rounded up and detached from the bottom.</li>
<li>Add 10 mL cell culture medium to cease trypsinization.</li>
<li>Detach and resuspend cells by pipetting the medium with cells up and down against the flask bottom.</li>
<li>Count viable cell numbers using TC20<sup>TM</sup> Automated Cell Counter by following manufacturer’s protocols (<a href="https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10024423.pdf">https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10024423.pdf</a>).
<div class="note">
<span class="note-title">Note:</span> We recommend using cells with ≥90% viability.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Viable cell number can be counted automatically using other automated cell counters or manually using a hemocytometer under a microscope.</div>
</li>
<li>Prepare cell suspensions at concentration 0.3–0.5 × 10<sup>6</sup> cells/mL by diluting cells in culture medium.</li>
<li>Plate 2 mL cell suspension per well in 6-well plate and incubate in CO<sub>2</sub> incubator at 37°C for 24 h.
<div class="note">
<span class="note-title">Note:</span> We recommend testing the optimal plating density within the recommended range for the cells to reach 70%–80% confluency on the second day for transfection, and to reach &gt;80% transfection efficiency using GFP plasmid as control (<a href="#bib17">Jäger et al., 2013</a>). <a href="#sec6.1">Troubleshooting 1</a>
</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other cell culture plates or dish formats could also be used with cell numbers adjusted proportionally.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Plasmid transfection – Day 2
<ol type="a">
<li>Thaw PEI stock solution (1 mg/mL) at 25°C and mix by gently inverting tubes before use.
<div class="note">
<span class="note-title">Note:</span> Properly prepared PEI solution should be clear. We do not recommend using PEI solutions that are cloudy or with precipitation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To develop a robust TR-FRET assay for HTS, we recommend testing the combinations of various epitope-tagged constructs with corresponding TR-FRET fluorophore pairs (<a href="#tbl1">Table 1</a>) (<a href="#bib1">Blagg and Workman, 2017</a>).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> We used PEI as a cost-effective transfection reagent for large-scale HTS application and it produces very high transfection efficiencies (&gt;70%) in HEK293T cells (<a href="#bib34">Schirrmann and Büssow, 2010</a>). Other plasmid transfection reagents, such as FuGene® HD, can also be used by following manufacturer’s protocol (<a href="https://www.promega.com/products/luciferase-assays/transfection-reagents/fugene-hd-transfection-reagent/?catNum=E2311&amp;gclid=Cj0KCQiA3NX_BRDQARIsALA3fIKo0C-VsU8ps6GFihjTxZrYOPgzlevuV0g6XAz_O1hUwSvCl67bVIcaAjGrEALw_wcB">https://www.promega.com/products/luciferase-assays/transfection-reagents/fugene-hd-transfection-reagent/?catNum=E2311&amp;gclid=Cj0KCQiA3NX_BRDQARIsALA3fIKo0C-VsU8ps6GFihjTxZrYOPgzlevuV0g6XAz_O1hUwSvCl67bVIcaAjGrEALw_wcB#protocols</a>)</div>
</li>
<li>Add 100 μL transfection mixture (1.5 μg Flag-SMAD3, 1.5 μg His-SMAD4-R361H or WT, and 9 μg PEI in Opti-MEM) per well and mix well by gently rocking the plate. Incubate plates in CO<sub>2</sub> incubator at 37°C for 48 h.
<div class="note">
<span class="note-title">Note:</span> We recommend gently vortexing the transfection mixture and incubating at 25°C for 15–30 min to allow the formation of stable DNA:PEI complex for efficient transfection. Plasmids expressing fluorescence protein can be used to rapidly estimate the transfection efficiency.</div>
<div class="note">
<span class="note-title">Note:</span> Depending on the expression of each plasmid when co-expressed, the ratio of plasmids may need to be changed from 1:1 to ensure optimal expression of each encoded protein from the plasmid that gives desired assay performance. <a href="#sec6.3">Troubleshooting 2</a>.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Transfection efficiency &gt;80% estimated from fluorescence protein are recommended for the following cell lysate preparation. To ensure the desired transfection efficiency specific to this protocol, we recommend i) removing 500 μL growth medium from each well before adding transfection mixture to increase the DNA:PEI complex working concentration, ii) adding transfection mixture in a dropwise manner, and iii) supplementing 1 mL fresh pre-warmed medium per well 24 h after transfection.</div>
</li>
</ol>
</li>
</ol>
<table id="tbl1">
<caption>Table 1. Combinations of constructs and TR-FRET antibody pairs</caption>
<thead>
<tr>
<th>No</th>
<th>DNA construction pairs</th>
<th>TR-FRET antibodies</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>Flag-SMAD4-WT + His-SMAD3</td>
<td rowspan="4">Anti-Flag-Tb (1:500 dilution to FRET buffer, final 1:1000 dilution in mixture with cell lysate) + Anti-His-D2 (1:250 dilution to FRET buffer, final 1:500 dilution in mixture with cell lysate)</td>
</tr>
<tr>
<td>2</td>
<td>Flag-SMAD4-R361H + His-SMAD3</td>
</tr>
<tr>
<td>3</td>
<td>His-SMAD4-WT + Flag-SMAD3</td>
</tr>
<tr>
<td>4</td>
<td>His-SMAD4-R361H + Flag-SMAD3</td>
</tr>
</tbody>
</table>
<ol start="3">
<li>Lysate stock preparation – Day 4
<ol type="a">
<li>Wash monolayer cell culture with 1× PBS three times, followed by adding trypsin (0.25%, 200 μL) and incubate in CO<sub>2</sub> incubator at 37°C for 1 min. Detach transfected HEK293T cells from cell culture plate by adding 1 mL of cell culture medium and pipetting up and down. Transfer cell suspension from each well to a pre-chilled microcentrifuge tube and pellet cells by centrifuge at 200 × <i>g</i> for 5 min. Discard the supernatant and wash cell pellet once with ice-cold PBS.</li>
<li>Add 100 μL lysis buffer per sample and vortex vigorously for 5 s to resuspend cell pellet. Lyse cells with rotation for 30 min at 4°C, followed by a centrifuge at 10,000 × <i>g</i> for 10 min at 4°C. Aspirate the supernatant and place in a fresh tube kept on ice. Combine lysates from wells transfected with the same plasmids to reduce variation.</li>
<li>Measure the protein concentration using the Bradford assay by following manufacturer’s instructions (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2FMAN0011181_Coomassie_Bradford_Protein_Asy_UG.pdf&amp;title=VXNlciBHdWlkZTogIENvb21hc3NpZSAoQnJhZGZvcmQpIFByb3RlaW4gQXNzYXkg">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2FMAN0011181_Coomassie_Bradford_Protein_Asy_UG.pdf&amp;title=VXNlciBHdWlkZTogIENvb21hc3NpZSAoQnJhZGZvcmQpIFByb3RlaW4gQXNzYXkgS2l0</a>).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If using cell lines which adhere well to plate surface, we recommend washing the cells with ice-cold PBS first, then adding lysis buffer directly to the well to detach cells from plates, and transferring them to tube. Cell pellets can be stored at −80°C for short-term or in liquid nitrogen for long-term storage. Both 0.5% Triton X-100 and 1% NP-40 lysis buffer work for cell lines used in this study. Other lysis buffer with different washing stringency could be used.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> For determination of protein concentration, other assays, such as BCA assay, can also be used by following manufacturer’s instructions <a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2FMAN0011430_Pierce_BCA_Protein_Asy_UG.pdf&amp;title=VXNlciBHdWlkZTogUGllcmNlIEJDQSBQcm90ZWluIEFzc2F5IEtpdA==">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2FMAN0011430_Pierce_BCA_Protein_Asy_UG.pdf&amp;title=VXNlciBHdWlkZTogUGllcmNlIEJDQSBQcm90ZWluIEFzc2F5IEtpdA==</a>
</div>
<h3 id="sec3.2">TR-FRET assay development and evaluation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This step describes how cell lysate-based Time-Resolved Fluorescence-Resonance-Energy-Transfer (TR-FRET) assay was designed, evaluated and optimized to monitor protein-protein interactions for high-throughput screening of compounds (<a href="#bib9">Du et al., 2011</a>; <a href="#bib10">Fu, 2004</a>).</p>
<ol start="4">
<li>Dose-dependent TR-FRET assay
<ol type="a">
<li>Perform serial dilution of cell lysate in the FRET buffer.
<div class="note">
<span class="note-title">Note:</span> We recommend 2-fold serial dilution directly in a black 384-well plate with final volume of 15 μL/well to reach 8–16 titrations in total. Empty FRET buffer without lysate will be used as background control.</div>
</li>
<li>Add TR-FRET antibody mixture.
<div class="note">
<span class="note-title">Note:</span> We recommend adding 15 μL/well antibody mixture containing 2× fluorophore-conjugated antibodies (1:500 diluted anti-FLAG M2-Tb and 1:250 diluted anti-6xHIS-D2 antibodies in FRET buffer). The optimal dilution and ratio of conjugated antibodies may vary based on the expression levels of PPI. <a href="#sec6.5">Troubleshooting 3</a>.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Mix the reaction thoroughly by centrifugation of the plate at 200 × <i>g</i> for 5min.</div>
</li>
<li>Measure TR-FRET signal using a BMG Labtech PHERAstar FSx reader by following manufacture’s manual.
<div class="note">
<span class="note-title">Note:</span> We recommend incubating the assay plate for 2 h at 4°C before the measurement. Unless necessary to read plate cold, let assay plate come to 25°C before reading so that there is no variation in signal due to temperature changes while reading. We recommend using the HTRF optic module (excitation at 337 nm, emission A at 665 nm, emission B at 620 nm) with following measurement settings: integration start at 50 μs, integration time for 150 μs and 8 flashes per well.</div>
<div class="critical">
<span class="critical-title">Critical:</span> We noticed that bubbles in the well may lead to inaccurate readings, and thus recommend eliminating bubbles before the measurement.</div>
</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Estimation of an optimal dilution factor of cell lysate
<ol type="a">
<li>The TR-FRET signal is calculated as the ratio of F665 nm/F620 nm × 10<sup>4</sup>, where F665 nm and F620 nm are the fluorescence intensity at 665 and 620 nm, respectively.</li>
<li>Plot the raw FRET signal against cell lysate concentration as shown in <a href="#fig2">Figure 2</a>.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/989-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Cell lysate dose-dependent TR-FRET curves showing different TR-FRET signals from lysate expressing different pairs of SMAD4-WT/R361H and SMAD3 constructs</div>
<p>The TR-FRET signal from cell lysate expressing the plasmid combination indicated in 3&amp;4 exhibit higher S/B and assay window than that in 1&amp;2. The lysate concentration-dependent “bell curve” of 3&amp;4 represents a typical “hook effect” in this antibody-based TR-FRET assay. Selection of optimal lysate concentration should be in the upward-sloping phase, where PPI signal positively correlates with PPI concentration, but not in the downward-sloping phase, where PPI signal negatively correlates with PPI concentration due to the artifact from low antibody occupancy. Data are presented as mean ± SD from triplicate of a representative experiment.</p>
</figcaption>
</figure>
</li>
<li>Estimate an optimal dilution factor of cell lysate with desired assay window. The chosen dilution factor should avoid ramifications of the hook effect (<a href="#bib26">Ross et al., 2020</a>). The selected dilution factor will be used for assay performance evaluation and primary HTS implementation.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To maximize the opportunity to reveal inducers that may enhance mutant SMAD4/SMAD3 interaction, we selected the EC<sub>90</sub> (90% maximal effective concentration) condition of the SMAD4<sup>R361H</sup>/SMAD3 interaction, which corresponded to EC<sub>10</sub> (10% maximal effective concentration) of the SMAD4<sup>WT</sup>/SMAD3 PPI.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Selection of optimal dilution factor should be in the upward-sloping phase, where PPI signal positively correlates with PPI concentration, but not in the downward-sloping phase, where PPI signal negatively correlates with PPI concentration due to the artifact from low antibody occupancy.</div>
<ol start="6">
<li>Assay performance evaluation
<ol type="a">
<li>Prepare the working reaction mixture by diluting the cell lysate stock according to the optimal dilution factor identified in step 5 in FRET buffer (containing 1:1000 anti-FLAG M2-Tb and 1:500 anti-6xHIS-D2 antibodies) to the selected concentration.
<div class="note">
<span class="note-title">Note:</span> We recommend including cell lysate co-expressing SMAD4-WT and SMAD3 PPI as a known biological positive control in parallel for assay development and performance evaluation. Cell lysate expressing empty-vector controls are recommended as a technical negative control.</div>
</li>
<li>Add 5 μL/well working reaction mixture into 1536-well solid bottom plate.
<div class="note">
<span class="note-title">Note:</span> Multidrop™ Combi Reagent Dispenser was used to add the working cell lysate, which was also used for the similar steps during the subsequent HTS.</div>
<div class="note">
<span class="note-title">Note:</span> To cost-effectively scale up the assay for HTS, we recommend miniaturizing the assay by scaling down proportionally the volume from 30 μL in a 384-well plate to 5 μL in a 1536-well plate format. We did not observe any significant changes of the TR-FRET signal between 384- and 1536-well plate format.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend centrifugation of the plate at 200 × <i>g</i> for 5 min and incubation at 4°C for 2 h.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To mitigate the evaporation-induced edge effect, we recommend adding the working cell lysate in the center and dispensing FRET buffer in the empty surrounding wells.</div>
</li>
<li>Measure TR-FRET signal using a BMG Labtech PHERAstar FSx reader by following manufacture’s manual.</li>
<li>Calculate the Signal-to-Background (S/B) ratio and Z’ for SMAD4-WT/SMAD3 and SMAD4-R361H/SMAD3 PPI, respectively (<a href="#fig3">Figure 3</a>), using equations as shown in <a href="#tbl2">Table 2</a>.
<div class="critical">
<span class="critical-title">Critical:</span> S/B ratio &gt;4 and Z’ factor &gt;0.5 suggests a robust assay condition for HTS (<a href="#bib32">Zhang et al., 1999</a>).</div>
</li>
<li>To determine the assay performance in terms of DMSO tolerance, repeat steps 3a–3d by comparing the assay performance with various DMSO added up to 10% (v/v) during step 3a. To determine the assay performance in terms of temporal stability, repeat step 3c–3d after various incubation time ranging from 2–24 h.
<div class="note">
<span class="note-title">Note:</span> Given that DMSO is a common solvent used in chemical libraries ((<a href="#bib33">Wigglesworth et al., 2012</a>); <a href="#bib19">Janzen and Popa-Burke, 2009</a>), we recommend evaluating the assay performance in the presence of DMSO prior to the HTS. Specific to this protocol, the final DMSO concentration is 2% (v/v) during the HTS. Our results show that our reaction system is stable and robust up to 10% DMSO (<a href="#fig4">Figure 4</a>). For confirmation studies in biological assays, DMSO concentration should be adjusted based on DMSO tolerance test results for a particular assay system, usually less than 1%. The stock concentration of re-ordered compounds should be prepared accordingly.</div>
<div class="critical">
<span class="critical-title">Critical:</span> A DMSO tolerant and temporally stable assay that maintains similar robustness (S/B ratio &gt;4 and Z’ factor &gt;0.5) is recommended for HTS application.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/989-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Signal-to-Background (S/B) ratio and Z’ factor for SMAD4-WT/SMAD3 and SMAD4-R361H/SMAD3 PPI respectively in both 384-well format and 1536-well format</div>
<p>S &gt; 4 and Z′ factor &gt;0.5 suggests a robust assay condition for HTS. Data are presented as mean ± SD from triplicate of a representative experiment.</p>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/989-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. DMSO tolerance test</div>
<p>TR-FRET PPI signal were measured using cell lysate containing various amount of DMSO as indicated. Data are presented as mean ± SD from triplicate of a representative experiment.</p>
</figcaption>
</figure>
<table id="tbl2">
<caption>Table 2. Z’ factor and signal-to-background (S/B) equations</caption>
<thead>
<tr>
<th>Equations</th>
<th>Definition</th>
<th>Meaning</th>
</tr>
</thead>
<tbody>
<tr>
<td><math> <mrow> <mrow> <mtext>S</mtext> <mo>/</mo> <mtext>B</mtext> </mrow> <mo>=</mo> <mrow> <msub> <mtext>FRET</mtext> <mtext>PPI</mtext> </msub> <mo>/</mo> <msub> <mtext>FRET</mtext> <mtext>vector</mtext> </msub> </mrow> </mrow> </math></td>
<td>FRET<sub>PPI</sub> and FRET<sub>vector</sub> are the TR-FRET signals from lysate containing His-SMAD4-R361H and Flag-SMAD3 or empty Flag-vector controls respectively.</td>
<td>Signal-to-background ratio suggests the signal window of the assay.</td>
</tr>
<tr>
<td><math> <mrow> <msup> <mtext>Z</mtext> <mtext>′</mtext> </msup> <mo>=</mo> <mn>1</mn> <mo>−</mo> <mfrac> <mrow> <mn>3</mn> <mo>×</mo> <msub> <mtext>SD</mtext> <mtext>PPI</mtext> </msub> <mo>+</mo> <mn>3</mn> <mo>×</mo> <msub> <mtext>SD</mtext> <mtext>vector</mtext> </msub> </mrow> <mrow> <msub> <mtext>FRET</mtext> <mtext>PPI</mtext> </msub> <mo>−</mo> <msub> <mtext>FRET</mtext> <mtext>vector</mtext> </msub> </mrow> </mfrac> </mrow> </math></td>
<td>SD<sub>PPI</sub> and SD<sub>vector</sub> are standard deviations of the TR-FRET signals from lysate containing His-SMAD4-R361H and Flag-SMAD3 or empty Flag-vector controls respectively.</td>
<td>Z’ factor reflects the robustness of the assay for HTS. Z’ factor between 0.5 and 1 indicates a robust assay, suitable for HTS.</td>
</tr>
</tbody>
</table>
<h3 id="sec3.3">uHTS primary screening</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes primary screening process to identify small molecule PPI inducers of SMAD4-R361H/SMAD3 PPI by using the TR-FRET assay we detailed above in an ultrahigh-throughput 1536-well plate format. We have applied similar protocol to identify small molecule PPI inhibitors for other targets as well.</p>
<div class="critical">
<span class="critical-title">Critical:</span> With the established TR-FRET assay, starting with a pilot screening is highly recommended. The positive compounds successfully identified and validated will provide more confidence on the screening platform.</div>
<ol start="7">
<li>Plate the working reaction mixture in black 1536-well solid bottom plates (5μL/well) using MultiDrop Combi reagent dispenser following manufacturer’s protocol (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLPD%2Fmanuals%2FN05616_ver1.7%2520Multidrop%2520Combi%2520User%2520Manual.pdf&amp;title=TXVsdGlkcm9wIENvbWJpIFVzZXIgTWFudWFs">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLPD%2Fmanuals%2FN05616_ver1.7%2520Multidrop%2520Combi%2520User%2520Manual.pdf&amp;title=TXVsdGlkcm9wIENvbWJpIFVzZXIgTWFudWFs</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Taking the dispenser dead volume into account, we recommend preparing 1–10 mL extra reaction mixture.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend adding a known negative biological control, such as cell lysate expressing empty vector control, and positive biological control, such as cell lysate expressing SMAD4-WT/SMAD3 PPI, in each plate for screening quality control.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Even if some wells will not be used, we recommend adding working reaction mixture or control lysates to fill the whole plate to mitigate the evaporation-induced edge effect.</div>
<ol start="8">
<li>Add 0.1 μL compounds from the 384-well compound plate by using a robotic liquid handler, such as Biomek NX<sup>p</sup> Automated Workstation with Pintool (VP Scientific) (<a href="#bib4">Cleveland and Koutz, 2005</a>) by following manufacturer’s protocol (<a href="https://www.beckman.com/techdocs/B20063AC/wsr-145922">https://www.beckman.com/techdocs/B20063AC/wsr-145922</a>)</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We selected our inhouse Emory Enriched Bioactive Library (EEBL), a library of 2036 bioactive compounds. The working stock of 1 mM was used. The final compound concentration is 20 μM with 2% DMSO (v/v). Each compound is tested with single concentration point in four replicates (<a href="#fig5">Figure 5</a>).</div>
<div class="note">
<span class="note-title">Note:</span> We recommend centrifugation of the plates at 200 × <i>g</i> for 5 min and incubation at 4°C for 16 h before measuring the TR-FRET signal.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other chemical libraries with different stock concentrations can be used accordingly depending on the desired hit rate.</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/989-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Schematic illustration of the compound adding from 384-well compound plate to 1536-well reaction buffer plate</div>
<p>Each compound tested at one dose in four replicates.</p>
</figcaption>
</figure>
<ol start="9">
<li>Measure the TR-FRET signal using a BMG Labtech PHERAstar FSx reader by following manufacture’s manual.</li>
<li>Data analysis to calculate the compound’s effect on PPI modulation as the change of TR-FRET signal (ΔTR-FRET) upon compound treatment using the equation as indicated (<a href="#tbl3">Table 3</a>).
<table id="tbl3">
<caption>Table 3. Effect of compound calculation equation</caption>
<thead>
<tr>
<th>Equations</th>
<th>Definition</th>
</tr>
</thead>
<tbody>
<tr>
<td><math> <mrow> <mo>%</mo> <mspace></mspace> <mtext>of</mtext> <mspace></mspace> <mtext>control</mtext> <mo>=</mo> <mn>100</mn> <mo>×</mo> <mfrac> <mrow> <msub> <mtext>FRET</mtext> <mtext>compound</mtext> </msub> <mo>−</mo> <msub> <mtext> FRET</mtext> <mtext>DMSO</mtext> </msub> </mrow> <msub> <mtext>FRET</mtext> <mtext>DMSO</mtext> </msub> </mfrac> </mrow> </math></td>
<td>FRET<sub>compound</sub> and FRET<sub>DMSO</sub> are the TR-FRET signals from PPI in the presence of compounds or DMSO with background FRET<sub>vector</sub> substracted.</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We analyzed the screening data by using Bioassay software from CambridgeSoft (Cambridge, MA). Based on the PPI inducing effect with cutoff of ΔTR-FRET≥2SD<sub>PPI</sub> (40% in this case) compared with the DMSO control, we prioritized the top ranked 23 primary hits. These hits were then cherry-picked from the parental stock for dose-response studies.</div>
<div class="note">
<span class="note-title">Note:</span> To further validate the cherry-picked compounds, a dose-response confirmatory assay is highly recommended to test whether those compounds can reproducibly induce a significant increase of the TR-FRET signal. <a href="#sec6.7">Troubleshooting 4</a>.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If the hit rate is lower than expected, based on the PPI and chemical libraries used, we recommend re-evaluating the whole procedure by <a href="#sec6.9">troubleshooting 5</a>
</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A robust high-throughput screening platform should allow us to identify primary hit compounds which increase or decrease the PPI signal, suggesting the induction or inhibition of the PPI-of-interest, and thus can be further validated and developed into small molecule PPI modulators to interrogate the biological significance and towards therapeutic agents. By following the protocol detailed above, a TR-FRET PPI HTS assay with robust performance with S/B ≥ 4.0, Z’≥0.5, and DMSO tolerance up to 2% is expected. Depending on the nature of PPI and chemical library, hit rate varying from 0.01%–1% is expected. Orthogonal secondary assays with complimentary readouts are expected to confirm the primary hits. For further functional studies, hit compounds, or their derivatives, will be tested in cell or organisms to assess whether the compounds could engage the targeted protein(s) and induce the targeted PPI <i>in vivo</i> through biological experiments.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Ultrahigh-throughput screening is efficient and productive for large scale screening, however it requires advanced technology such as a robotic liquid handling system to ensure the robust assay performance and mitigate variations from human handling, as well as professional techniques or skills to perform such machine. Meanwhile high-throughput suggests a small volume system, which also requires precise and accurate operation. The scale of the screening also depends on the size and number of the compound library.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low transfection efficiency (step 1).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Transfection efficiency is a critical parameter to be optimized before assay development and screen. We recommend authenticating all the reagents and supplies for cell culture, plasmid preparation, and DNA transfection based on the concept of “trust but verify” for utilization. In this protocol, we used PEI as cost-effective transfection reagent for HTS applications. If transfection efficiency is low, we recommend utilizing the cationic lipid transfection reagents, such as Lipofectamine or FuGene HD, which usually has higher transfection efficiency for many cell lines. Also, we recommend optimizing cell density (50%–90%), plasmid amount (1–3 μg/well), DNA:PEI ratio (1:1–1:4), transfection complex formation time (10–40 min) and post-transfection expression time (24–72 h) to achieve desired transfection efficiency.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Low TR-FRET assay window due to sub-optimal protein concentration, stoichiometry and tag-orientation (step 2).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>TR-FRET signals from protein-protein interactions highly depend on the protein concentrations. In a typical two antibody-based TR-FRET assay, a protein concentration-dependent “hook effect” of PPI signal is expected. Therefore, we highly recommend a cell lysate-titration to determine the dilution factors for optimal protein concentrations at the upward-sloping phase of the bell curve. If no bell curve was observed, check transfection efficiency and protein expression using western blot, or further optimize protein stoichiometry and tag-orientation as discussed below.</p>
<p>TR-FRET PPI signal also depends on protein stoichiometry. In this protocol, we recommend 1:1 ratio of donor (Flag-SMAD3) and acceptor (His-SMAD4 WT or R361H) which gives a robust assay window. We also recommend titrating the donor and acceptor plasmid ratio during transfection to adjust protein stoichiometry to further optimize the assay if needed.</p>
<p>Tag-orientation is an important parameter to be optimized during development of a TR-FRET PPI assay. Efficiency energy transfer requires an optimal tag-orientation that brings donor and acceptor fluorophores into close-proximity (<a href="#bib24">Mo and Fu, 2016</a>; <a href="#bib25">Mo et al., 2016</a>). We recommend performing tag-orientation testing as described in <a href="#fig2">Figure 2</a> to determine the optimal tag-orientation.</p>
<p>Comprehensive optimization by combining the test of protein concentration, stoichiometry and tag-orientation is highly recommended to solve the issue of low TR-FRET assay window (<a href="#bib6">Doyle et al., 2021</a>).</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Low TR-FRET assay window due to sub-optimal TR-FRET antibodies selection and combinations (step 4).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>We highly recommend testing combination of available TR-FRET antibodies, since the PPI signal directly depends on the affinity, configuration, and concentration of fluorophore-conjugated antibodies. After determining the optimal protein concentration, stoichiometry and tag-orientation, we recommend optimizing antibody concentration and combination ratios through a two-way titration of donor and acceptor antibody pairs.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>No positive hit compounds shown from pilot screening (step 10).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Re-evaluate the assay performance including S/B ratio, Z’ factor, and DMSO tolerance. We also recommend including a known positive control compound for screening if available. Use freshly prepared TR-FRET buffer.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>False positive hit compounds (step 10).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>It’s inevitable that false positive compounds will show up through the screening due to a series of factors, including the interference of the compound itself, autofluorescence, or the signal interference caused by the non-specific binding of the target protein to the TR-FRET antibody (<a href="#bib3">Busch et al., 2013</a>; <a href="#bib8">Du and Havel, 2012</a>). We recommend validating all the selected compounds through different orthogonal methods, such as dose-response TR-FRET assay and GST-pulldown assay (<a href="#bib10">Fu, 2004</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Blagg, J. and Workman, P. (2017). Choose and use your chemical probe wisely to explore cancer biology. Cancer Cell <i>32</i>, 9-25. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref1">View at publisher</a></p>
<p id="bib2">Boettcher, M., Tian, R., Blau, J.A., Markegard, E., Wagner, R.T., Wu, D., Mo, X., Biton, A., Zaitlen, N., Fu, H., et al. (2018). Dual gene activation and knockout screen reveals directional dependencies in genetic networks. Nat. Biotechnol. <i>36</i>, 170-178. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref2">View at publisher</a></p>
<p id="bib3">Busch, M., Thoma, H.B., and Kober, I. (2013). Does your lab coat fit to your assay?. J. Biomol. Screen. <i>18</i>, 744-747. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref3">View at publisher</a></p>
<p id="bib4">Cleveland, P.H. and Koutz, P.J. (2005). Nanoliter dispensing for uHTS using pin tools. Assay Drug Dev. Technol. <i>3</i>, 213-225. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref4">View at publisher</a></p>
<p id="bib5">Cui, X., Liang, Q., Liang, Y., Lu, M., Ding, Y., and Lu, B. (2014). TR-FRET assays of Huntingtin protein fragments reveal temperature and polyQ length-dependent conformational changes. Sci. Rep. <i>4</i>, 1-8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref5">View at publisher</a></p>
<p id="bib6">Doyle, S.P., Mo, X., Qian, K., Cicka, D.N., Niu, Q., and Fu, H. (2021). CHAPTER 3 high throughput screening methods for PPI inhibitor discovery. In Protein–Protein Interaction Regulators (The Royal Society of Chemistry), pp. 49-86. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref6">View at publisher</a></p>
<p id="bib7">Du, Y., Fu, R.W., Lou, B., Zhao, J., Qui, M., Khuri, F.R., and Fu, H. (2013). A time-resolved fluorescence resonance energy transfer assay for high-throughput screening of 14-3-3 protein-protein interaction inhibitors. Assay Drug Dev. Technol. <i>11</i>, 367-381. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref7">View at publisher</a></p>
<p id="bib8">Du, Y. and Havel, J.J. (2012). Time-resolved fluorescence resonance energy transfer technologies in HTS. Chem. Genomics. 198-214. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref8">View at publisher</a></p>
<p id="bib9">Du, Y., Nikolovska-Coleska, Z., Qui, M., Li, L., Lewis, I., Dingledine, R., Stuckey, J.A., Krajewski, K., Roller, P.P., and Wang, S. (2011). A dual-readout F2 assay that combines fluorescence resonance energy transfer and fluorescence polarization for monitoring bimolecular interactions. Assay Drug Dev. Technol. <i>9</i>, 382-393. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref9">View at publisher</a></p>
<p id="bib10">Fu, H. (2004). Protein-protein Interactions: Methods and Applications (Springer Science &amp; Business Media). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref10">View at publisher</a></p>
<p id="bib12">Grzeskowiak, C.L., Kundu, S.T., Mo, X., Ivanov, A.A., Zagorodna, O., Lu, H., Chapple, R.H., Tsang, Y.H., Moreno, D., Mosqueda, M., et al. (2018). In vivo screening identifies GATAD2B as a metastasis driver in KRAS-driven lung cancer. Nat. Commun. <i>9</i>, 2732. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref12">View at publisher</a></p>
<p id="bib13">Hahn, W.C., Bader, J.S., Braun, T.P., Califano, A., Clemons, P.A., Druker, B.J., Ewald, A.J., Fu, H., Jagu, S., Kemp, C.J., et al. (2021). An expanded universe of cancer targets. Cell <i>184</i>, 1142-1155. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref13">View at publisher</a></p>
<p id="bib14">Hata, A. and Lagna, G. (2021). How do you mend inactive tumor suppressor mutants? You glue them!. Cell Chem. Biol. <i>28</i>, 585-587. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref14">View at publisher</a></p>
<p id="bib15">Ivanov, A.A., Gonzalez-Pecchi, V., Khuri, L.F., Niu, Q., Wang, Y., Xu, Y., Bai, Y., Mo, X., Prochownik, E.V., Johns, M.A., et al. (2017a). OncoPPi-informed discovery of mitogen-activated protein kinase kinase 3 as a novel binding partner of c-Myc. Oncogene <i>36</i>, 5852-5860. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref15">View at publisher</a></p>
<p id="bib16">Ivanov, A.A., Revennaugh, B., Rusnak, L., Gonzalez-Pecchi, V., Mo, X., Johns, M.A., Du, Y., Cooper, L.A.D., Moreno, C.S., Khuri, F.R., et al. (2017b). The OncoPPi Portal: an integrative resource to explore and prioritize protein–protein interactions for cancer target discovery. Bioinformatics <i>34</i>, 1183-1191. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref16">View at publisher</a></p>
<p id="bib17">Jäger, V., Büssow, K., Wagner, A., Weber, S., Hust, M., Frenzel, A., and Schirrmann, T. (2013). High level transient production of recombinant antibodies and antibody fusion proteins in HEK293 cells. BMC Biotechnol. <i>13</i>, 1-20. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref17">View at publisher</a></p>
<p id="bib18">Janzen, W.P. (2014). Screening technologies for small molecule discovery: the state of the art. Chem. Biol. <i>21</i>, 1162-1170. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref18">View at publisher</a></p>
<p id="bib19">Janzen, W.P. and Popa-Burke, I.G. (2009). Advances in improving the quality and flexibility of compound management. J. Biomol. Screen. <i>14</i>, 444-451. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref19">View at publisher</a></p>
<p id="bib20">Li, Z., Ivanov, A.A., Su, R., Gonzalez-Pecchi, V., Qi, Q., Liu, S., Webber, P., McMillan, E., Rusnak, L., Pham, C., et al. (2017). The OncoPPi network of cancer-focused protein–protein interactions to inform biological insights and therapeutic strategies. Nat. Commun. <i>8</i>, 14356. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref20">View at publisher</a></p>
<p id="bib21">Macarron, R., Banks, M.N., Bojanic, D., Burns, D.J., Cirovic, D.A., Garyantes, T., Green, D.V., Hertzberg, R.P., Janzen, W.P., and Paslay, J.W. (2011). Impact of high-throughput screening in biomedical research. Nat. Rev. Drug Discov. <i>10</i>, 188-195. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref21">View at publisher</a></p>
<p id="bib22">Mo, X., Qi, Q., Ivanov, A.A., Niu, Q., Luo, Y., Havel, J., Goetze, R., Bell, S., Moreno, C.S., Cooper, L.A.D., et al. (2017). AKT1, LKB1, and YAP1 revealed as MYC interactors with NanoLuc-based protein-fragment complementation assay. Mol. Pharmacol. <i>91</i>, 339-347. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref22">View at publisher</a></p>
<p id="bib23">Mo, X., Tang, C., Niu, Q., Ma, T., Du, Y., and Fu, H. (2019). HTiP: high-throughput immunomodulator phenotypic screening platform to reveal IAP antagonists as anti-cancer immune enhancers. Cell Chem. Biol. <i>26</i>, 331-339.e3. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref23">View at publisher</a></p>
<p id="bib24">Mo, X.-L. and Fu, H. (2016). BRET: NanoLuc-based bioluminescence resonance energy transfer platform to monitor protein-protein interactions in live cells. In High Throughput Screening: Methods and Protocols, W.P. Janzen, ed. (Springer New York), pp. 263-271. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref24">View at publisher</a></p>
<p id="bib25">Mo, X.-L., Luo, Y., Ivanov, A.A., Su, R., Havel, J.J., Li, Z., Khuri, F.R., Du, Y., and Fu, H. (2016). Enabling systematic interrogation of protein–protein interactions in live cells with a versatile ultra-high-throughput biosensor platform. J. Mol. Cell Biol. <i>8</i>, 271-281. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref25">View at publisher</a></p>
<p id="bib26">Ross, G.M.S., Filippini, D., Nielen, M.W.F., and Salentijn, G.I. (2020). Unraveling the hook effect: a comprehensive study of high antigen concentration effects in sandwich lateral flow immunoassays. Anal Chem. <i>92</i>, 15587-15595. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref26">View at publisher</a></p>
<p id="bib27">Rusnak, L., Tang, C., Qi, Q., Mo, X., and Fu, H. (2018). Large tumor suppressor 2, LATS2, activates JNK in a kinase-independent mechanism through ASK1. J. Mol. Cell Biol. <i>10</i>, 549-558. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref27">View at publisher</a></p>
<p id="bib34">Schirrmann, T. and Büssow, K. (2010). Transient production of scFv-Fc fusion proteins in mammalian cells. In Antibody engineering (Springer), pp. 387-398. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/optK5VcFuL3VP">View at publisher</a></p>
<p id="bib29">Schreiber, S.L. (2021). The rise of molecular glues. Cell <i>184</i>, 3-9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref29">View at publisher</a></p>
<p id="bib30">Tang, C., Mo, X., Niu, Q., Wahafu, A., Yang, X., Qui, M., Ivanov, A.A., Du, Y., and Fu, H. (2020). Hypomorph mutation-directed small-molecule protein-protein interaction inducers to restore mutant SMAD4-suppressed TGF-β signaling. Cell Chem. Biol. <i>28</i>, 636-647.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref30">View at publisher</a></p>
<p id="bib33">Wigglesworth, M., Wood, T., Gray, M., and Bhatt, S. 2012. Solid sample weighing and distribution. In Management of Chemical and Biological Samples for Screening Applications, (Wiley) T.10.1002/ 9783527645251.ch8, <a class="external-link" href="http://onlinelibrary.wiley.com/doi/10.1002/9783527645251.ch8/summary">http://onlinelibrary.wiley.com/doi/10.1002/9783527645251.ch8/summary</a>.</p>
<p id="bib31">Xiong, J., Pecchi, V.G., Qui, M., Ivanov, A.A., Mo, X., Niu, Q., Chen, X., Fu, H., and Du, Y. (2018). Development of a time-resolved fluorescence resonance energy transfer ultrahigh-throughput screening assay for targeting the NSD3 and MYC interaction. Assay Drug Dev. Technol. <i>16</i>, 96-106. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref31">View at publisher</a></p>
<p id="bib32">Zhang, J.-H., Chung, T.D., and Oldenburg, K.R. (1999). A simple statistical parameter for use in evaluation and validation of high throughput screening assays. J. Biomol. Screen. <i>4</i>, 67-73. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00510-4/sref32">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Haian Fu (<a href="mailto:hfu@emory.edu">hfu@emory.edu</a>).</p>
<h4>Materials availability</h4>
<p>The plasmids generated in this study are available to qualified scientists upon request.</p>
<h4>Data and code availability</h4>
<p>The primary screening data is available in Cancer Target Discovery and Development (CTD2) Data Portal.</p>
<h3>Acknowledgments</h3>
<p>We thank members of the Fu Lab for technical support and comments. This work was supported by the <a href="https://doi.org/10.13039/100000054">National Cancer Institute</a>’s Office of Cancer Genomics Cancer Target Discovery and Development (CTD<sup>2</sup>) initiative (U01CA217875 to H.F.), the NCI Emory Lung Cancer SPORE (P50CA217691 to H.F.) Career Enhancement Program (P50CA217691 to X.M.), <a href="https://doi.org/10.13039/100000054">NCI</a> MERIT Award (R37CA255459 to X.M.), Emory URC Award (to X.M.), the Imagine, Innovate and Impact (I<sup>3</sup>) Funds from the <a href="https://doi.org/10.13039/100007623">Emory School of Medicine</a>, and through the Georgia CTSA NIH award (UL1-TR002378) and <a href="https://doi.org/10.13039/100011621">Winship Cancer Institute</a> (NIH 5P30CA138292). C.T. was a visiting student in the Emory University School of Medicine- Xi'an Jiaotong University Health Science Center student exchange program and is a current recipient of the <a href="https://doi.org/10.13039/100010665">Marie Skłodowska-Curie</a> grant (agreement No 101038053) from the European Union's Horizon 2020 research and innovation program.</p>
<h3>Author contributions</h3>
<p>Conceptualization, X.M. and H.F.; assay design and development and chemical screening, C.T., X.M., Q.N., and Y.D.; data analysis, C.T., X.M., Y.D., D.C., and H.F.; writing – original draft, C.T., X.M., and H.F.; writing – review &amp; editing, all authors.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>