<article>
<section>
<h1 id="header">Preparation of monovalent follistatin-like 3-Fc-fusion protein and evaluation of its effects on muscle mass in mice</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Takayuki Ozawa,<sup><a href="#aff1">1</a></sup> Kohei Miyazono,<sup><a href="#aff1">1</a></sup> and Masato Morikawa<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Pathology, Graduate School of Medicine, The University of Tokyo, Bunkyo-ku, Tokyo 113-0033, Japan</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:morikawa-tky@umin.ac.jp">morikawa-tky@umin.ac.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100839">10.1016/j.xpro.2021.100839</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Follistatin-like 3 (FSTL3) is an endogenous antagonist against transforming growth factor-β family ligands. Monovalent FSTL3-Fc fusion protein (mono-FSTL3-Fc) generated with knobs-into-holes technology overcomes limitations of current anti-myostatin therapies. We have developed a facile protocol for affinity purification of the Fc-fused protein from the supernatant of HEK293T cells stably expressing the protein. This protocol is advantageous by only requiring readily accessible equipment. We further outline the steps for validation of mono-FSTL3-Fc increasing systemic muscle mass in mice after intraperitoneal administration.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref6">Ozawa et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Monovalent FSTL3-Fc (mono-FSTL3-Fc) improves anti-myostatin therapy</li>
<li>A protocol for the simple preparation of mono-FSTL3-Fc protein is described</li>
<li>mono-FSTL3-Fc protein is affinity purified from the supernatant of HEK293T cells</li>
<li>Systemic effects of mono-FSTL3-Fc on muscle mass can be confirmed in mice</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1022-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Follistatin-like 3 (FSTL3) is an endogenous antagonist against transforming growth factor-β family ligands, which mainly binds and neutralizes activins, growth differentiation factor 8 (GDF8, also known as myostatin), and GDF11 (<a href="#bib2">Chang, 2016</a>). We have developed a method to generate monovalent FSTL3-Fc (mono-FSTL3-Fc) with knobs-into-holes technology.</p>
<p>In this protocol, we describe the steps required for the expression and purification of recombinant mono-FSTL3-Fc from the supernatant of HEK293T cells stably expressing the protein. We also describe protocols for the <i>in vivo</i> assessment of muscle mass using intraperitoneal injection of mono-FSTL3-Fc, resulting in the induction of muscle fiber hypertrophy.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-human IgG Fc</td>
<td>Bethyl Laboratories</td>
<td>Cat#A80-105A; RRID: AB_67482</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-laminin</td>
<td>Abcam</td>
<td>Cat# ab11575; RRID: AB_298179</td>
</tr>
<tr>
<td>Goat polyclonal anti-rabbit IgG with Alexa Fluor594</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A11012; RRID: AB_141359</td>
</tr>
<tr>
<td>Goat polyclonal anti-mouse IgG light chain specific with HRP</td>
<td>Jackson ImmunoResearch Laboratories</td>
<td>Cat# 115-035-174; RRID: AB_2338512</td>
</tr>
<tr>
<td>Goat polyclonal anti-rabbit IgG with HRP</td>
<td>Cell Signaling Technology</td>
<td>Cat# 7074S; RRID: AB_2099233</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Human IgG1 Fc</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Monovalent human FSTL3-Fc</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Murine ActRIIB-murine Fc</td>
<td>R&amp;D</td>
<td>Cat# 3725-RB</td>
</tr>
<tr>
<td>Puromycin</td>
<td>InvivoGen</td>
<td>Cat# ant-pr</td>
</tr>
<tr>
<td>Hygromycin</td>
<td>Invitrogen/Thermo Fisher Scientific</td>
<td>Cat# 10687010</td>
</tr>
<tr>
<td>Lipofectamine 2000</td>
<td>Invitrogen/Thermo Fisher Scientific</td>
<td>Cat# 11668019</td>
</tr>
<tr>
<td>HiTrap Protein A HP antibody purification columns</td>
<td>GE Healthcare Life Sciences/Cytiva</td>
<td>Cat# 17040201</td>
</tr>
<tr>
<td>FreeStyle 293 Expression Medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12338026</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Anti-human IgG ELISA</td>
<td>Bethyl Laboratories</td>
<td>Cat# E80-114</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Human: HEK293T cells</td>
<td>ATCC</td>
<td>Cat# CRL-3216; RRID: CVCL_0063</td>
</tr>
<tr>
<td>Human: Lenti-X 293T cells</td>
<td>Takara Bio</td>
<td>Cat# 632180; RRID: CVCL_4401</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: CD2F1/slc</td>
<td>Sankyo Labo Service Corporation</td>
<td>Slc:CDF1</td>
</tr>
<tr>
<td>Mouse: C57BL/10ScSn-Dmd<sup>mdx</sup>/J</td>
<td>CLEA Japan</td>
<td>C57BL/10-mdx</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>CSII-EF-RfA</td>
<td>Dr. H. MiyoshiRIKEN BRC</td>
<td>Cat# RDB04387</td>
</tr>
<tr>
<td>CSII-CAG-MCS-IRES-Puro</td>
<td>(<a href="#bib4">Harada et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>CSII-CAG-MCS-IRES-Hyg</td>
<td>(<a href="#bib4">Harada et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pcDNA3-Fc</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pcDNA3-FSTL3-Fc</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pcDNA3-Fc(T366S/L368A/Y407V)</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pcDNA3-FSTL3-Fc(T366W)</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>CSII-CAG-Fc-IRES-Puro</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>CSII-CAG-FSTL3-Fc(T366W)-IRES-Puro</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>CSII-CAG-Fc(T366S/L368A/Y407V)-IRES-Hyg</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: Fc-T366W-rev,ATAGAAGCCTTTGACCAGGCACCACAGGCTGACCTGGT</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: Fc-T366S-L368A-rev,ATAGAAGCCTTTGACGGCGCAGGACAGGCTGACCTGGT</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: Fc-Y407V-FW,GCTCCTTCTTCCTCGTGAGCAAGCTCACCGT</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer: Fc-Y407V-rev,ACGGTGAGCTTGCTCACGAGGAAGAAGGAGC</td>
<td>(<a href="#bib6">Ozawa et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>R software</td>
<td>R-project.org</td>
<td><a href="https://www.R-project.org">https://www.R-project.org</a></td>
</tr>
<tr>
<td>GraphPad Prism 6</td>
<td>GraphPad Software</td>
<td>version 6</td>
</tr>
<tr>
<td>Hybrid Cell Count software</td>
<td>Keyence</td>
<td>Cat# BZ-H3C</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Collagen I-coated T-75 flasks</td>
<td>Iwaki</td>
<td>Cat# 4123-010</td>
</tr>
<tr>
<td>Amicon Ultra-15 Ultracel-50 kDa</td>
<td>Millipore, Merck</td>
<td>Cat# UFC9050</td>
</tr>
<tr>
<td>FSC 22 frozen section media</td>
<td>Leica</td>
<td>Cat# FSC-22</td>
</tr>
<tr>
<td>DAPI Fluoromount-G</td>
<td>SouthernBiotech</td>
<td>Cat# 0100-20</td>
</tr>
<tr>
<td>ImageQuant LAS 4000</td>
<td>Fujifilm</td>
<td>Cat# LAS-4000</td>
</tr>
<tr>
<td>BZ-X700 microscope</td>
<td>Keyence</td>
<td>Cat# BZ-X 700</td>
</tr>
<tr>
<td>Model 680 Microplate Reader</td>
<td>Bio-Rad</td>
<td>Cat# 1681101</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Construction of monovalent FSTL3-Fc</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>To construct mono-FSTL3-Fc, the coding region of the human <i>FSTL3</i> gene (GenBank accession NM_005860.3) and the Fc constant region (Asp 6-Lys 232) of human IgG1 (GenBank: AEV43323.1) are used. The signal peptide of FSTL3 is used in the Fc construct. For introducing the knobs-into-holes mutation T366W or T366S/L368A/Y407V (<a href="#bib1">Atwell et al., 1997</a>; <a href="#bib7">Ridgway et al., 1996</a>), site-directed mutagenesis is performed using PCR with specific primers (<a href="#fig1">Figure 1</a>). Complete amino acid sequences of the mono-FSTL3-Fc heterodimer are available in <a href="#bib6">Ozawa et al. (2021)</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1022-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic presentation of monovalent FSTL3-Fc protein</div>
<p>The mono-FSTL3-Fc heterodimer is generated through co-expression of FSTL3-Fc(T366W, knob) and Fc(T366S/L368A/Y407V, hole) constructs which harbor the knobs-into-holes mutations.</p>
</figcaption>
</figure>
<h3 id="sec2.2">Generation of cells stably expressing mono-FSTL3-Fc with lentivirus</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>Although its biological significance is not well characterized, FSTL3 was reported to be N-glycosylated (<a href="#bib8">Saito et al., 2005</a>). Therefore, mammalian cells should be used for protein expression, since glycosylation of mono-FSTL3-Fc may affect its function or pharmacokinetics profile. Here, we describe the protocol for establishing a stably expressing HEK293T cells, as these cells have high protein expression capacity. Adherent HEK293T cells can be grown in FreeStyle 293 Expression Medium without serum, enabling large-scale culture suitable for protein purification. Other cell types can also be used but will require additional optimization. HEK293T cells and Lenti-X 293T cells are grown in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal bovine serum (FBS) in a 37°C incubator with 5% CO<sub>2</sub>.</p>
<div class="note">
<span class="note-title">Note:</span> We used the CSII-CAG-MCS-IRES-Puro and CSII-CAG-MCS-IRES-Hyg lentiviral vectors (<a href="#bib4">Harada et al., 2019</a>) derived from the CSII-EF-RfA lentiviral vector to generate stably expressing cells. Any other methods to establish stably expressing cells or cell lines can be used for this step.</div>
<ol>
<li>Production and concentration of lentivirus (reverse transfection)
<ol type="a">
<li>For one transfection, seed 6 × 10<sup>6</sup> cells in a collagen I-coated T-75 flask. We normally use Lenti-X 293T cells, as this cell line is optimized for lentivirus production. In order to generate FSTL3-Fc(T366W) and Fc(T366S/L368A/Y407V) lentivirus separately, 2 flasks are required.</li>
<li>Prepare the transient transfection mixture. solution A: Transfer plasmid (CSII-CAG-FSTL3-Fc(T366W)-IRES-Puro, or CSII-CAG-Fc(T366S/L368A/Y407V)-IRES-Hyg), and envelope/packaging vectors, and solution B: Lipofectamine 2000. Incubate for 5 min at 25°C–28°C.
<table id="undtbl1">
<caption>Transfection solution A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Transfer plasmid:1. CSII-CAG-FSTL3-Fc(T366W)-IRES-Puro2. CSII-CAG-Fc(T366S/L368A/Y407V)-IRES-Hyg or3. CSII-CAG-Fc-IRES-Puro (negative control)</td>
<td>(3.67 <b>μ</b>g/mL)</td>
<td>5.51 <b>μ</b>g</td>
</tr>
<tr>
<td>Envelope plasmid: pCMV-VSVG-RSV-Rev</td>
<td>(2.16 <b>μ</b>g/mL)</td>
<td>3.24 <b>μ</b>g</td>
</tr>
<tr>
<td>Packaging plasmid: pCAG-HIVgp</td>
<td>(2.16 <b>μ</b>g/mL)</td>
<td>3.24 <b>μ</b>g</td>
</tr>
<tr>
<td>Opti-MEM I</td>
<td>n/a</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1.5 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>Transfection solution B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Lipofectamine 2000</td>
<td> </td>
<td>36 <b>μ</b>L</td>
</tr>
<tr>
<td>Opti-MEM I</td>
<td>n/a</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1.5 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The original protocol from Dr. Miyoshi/RIKEN recommends a 17:10:10 ratio of transfer plasmid:pCMV-VSVG-RSV-Rev:pCAG-HIVgp.</div>
</li>
<li>Mix solution A and solution B, and incubate for 15 min at 25°C–28°C.</li>
<li>Detach and resuspend the cells in total 10 mL Opti-MEM I supplemented with 10% FBS without antibiotics, and plate the cells in the collagen I-coated T-75 flask.</li>
<li>Add the DNA-Lipofectamine 2000 complex to the cells.</li>
<li>Replace the culture medium with 9 mL of DMEM (10% FBS, P/S) on day 2 (after 12–16 h of transfection, or overnight).</li>
<li>Harvest the supernatant with the lentivirus on day 4.</li>
<li>Pass the supernatant through a 0.22 μm filter.</li>
<li>Transfer the filtered supernatant to a 50-mL tube and add 3 mL of cold Lenti-X Concentrator.</li>
<li>Incubate the supernatant with Lenti-X mixture for 1 h at 4°C.</li>
<li>Centrifuge the mixture at 1500 <b>×</b> <i>g</i> for 45 min at 4°C. After centrifugation, the lentiviral particles appear as a white pellet.</li>
<li>Aspirate the supernatant, and resuspend the lentiviral pellet with 1 mL of DMEM (10% FBS, P/S).</li>
<li>Aliquot 200 μL from the viral solution into a 2-mL cryotube.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The viral solution can be aliquoted, snap frozen, and stored at –80°C as soon as possible to avoid loss of titer.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> HEK293T cells are highly susceptible to lentiviral gene delivery. The filtered supernatant without concentration can be used, although the efficiency might be low.</div>
<ol start="2">
<li>Lentivirus infection in HEK293T cells
<ol type="a">
<li>For one infection, seed 2 × 10<sup>5</sup> cells in a 100-mm dish.</li>
<li>Add the virus stock (200 μL) onto the cells. To generate mono-FSTL3-Fc expressing cells, infect FSTL3-Fc(T366W) and Fc(T366S/L368A/Y407V) lentiviruses together.</li>
<li>Replace the culture medium with 10 mL of DMEM (10% FBS, P/S) on day 2 (after 12–16 h of infection, or overnight).</li>
<li>Refresh the culture medium and supplement with a selection antibiotic (1 μg/mL puromycin and/or 100 μg/mL hygromycin B) after day 4.</li>
</ol>
</li>
<li>Allow the resistant cells to grow as polyclonal populations (∼1 week).</li>
<li>Once the polyclonal populations are growing well and have been sufficiently expanded, prepare cell stocks and/or harvest to test for protein expression by SDS-PAGE.</li>
<li>The polyclonal populations of resistant cells can be used in the next step, which are referred to as HEK293T-mono-FSTL3-Fc cells.</li>
<li>The cells are maintained in DMEM (10% FBS, P/S) and passaged every 2–3 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In general, lentivirally infected cells are safe after one or two passages after infection. However, the experiments should be conducted following national and/or institutional lentivirus safety guidelines.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Monoclonal cell lines can be obtained by limiting dilution, and a high-expressing clone will be used in the next step.</div>
<h3 id="sec2.3">Preparation of the mono-FSTL3-Fc–expressing HEK293T cell supernatant</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<ol start="7">
<li>Split HEK293T-mono-FSTL3-Fc cells at ∼80% confluence in one 150-mm dish into 10–11 new 150-mm dishes with 25 mL of DMEM (10% FBS, P/S).</li>
<li>On day 2–3 (80% confluent after passage), change the medium, wash the cells with phosphate-buffered saline (PBS), and replace with 25 mL of FreeStyle293 without antibiotics.</li>
<li>Three days after the medium change, collect the supernatant of the culture medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the culture medium contains few floating cells, centrifugation is not needed.</div>
<ol start="10">
<li>Filter the collected supernatant using a 0.22 μm filter system.</li>
<li>Store the filtered supernatant in a –20°C freezer before affinity purification.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The filtered supernatant can be stored for several months at −20°C.</div>
<h3 id="sec2.4">Affinity purification of mono-FSTL3-Fc from the supernatant</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<div class="note">
<span class="note-title">Note:</span> From this step forward, all work must be performed at 4°C in the cold room. However, the purification can be carried out on ice at 25°C–28°C, since recombinant mono-FSTL3-Fc protein is relatively stable.</div>
<ol start="12">
<li>Equilibrate a 1-mL HiTrap column with 10 column volumes (CV) of 20 mM sodium phosphate (pH 6.8).</li>
<li>Combine all supernatants from step 11 that contain mono-FSTL3-Fc.</li>
<li>Load the sample into the pre-equilibrated HiTrap Protein A HP column at 1 mL/min using a peristaltic pump.</li>
<li>Wash with over 20 CV of 20 mM sodium phosphate (pH 6.8).</li>
<li>Elute the mono-FSTL3-Fc protein using 5 CV of 0.1 M sodium citrate (pH 3.5) and neutralize it with 1 M Tris-HCl (pH 9.0) to adjust the pH to 7.0–7.4 immediately after the elution. Using 5 mL of 0.1 M sodium citrate (pH 3.5), the protein is manually eluted with a syringe into a new tube containing 100–200 μL of 1 M Tris-HCl (pH 9.0) for neutralization. It is possible to collect the elute as 1mL fraction and evaluate protein existence in each fraction using the absorbance at 280 nm.</li>
<li>Concentrate the pooled fractions from the elution until the volume of the solution is reduced to about 1 mL using ultrafiltration (Amicon Ultra-15 Ultracel-50 kDa).</li>
<li>Replace the solvent with sterile PBS using ultrafiltration as follows. Dilute the recombinant protein solution (about 1 mL) to 15 mL with sterile cold PBS, and centrifuge at 5,000 rpm (4,032 × <i>g</i>) at 4°C for about 20 min until the volume of the solution is reduced to ∼1 mL. Repeat the process at least three times. The final concentration will be about 1–5 mg/mL.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not over concentrate the mono-FSTL3-Fc as it will aggregate at concentration above 5 mg/mL.</div>
<ol start="19">
<li>Aliquot, snap freeze and store the purified recombinant mono-FSTL3-Fc protein in PBS in a –80°C freezer.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The purified recombinant mono-FSTL3-Fc protein in PBS can be stored for several months at −80°C.</div>
<ol start="20">
<li>Assess the quality of the purified proteins by SDS-PAGE (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1022-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. SDS-PAGE of purified recombinant mono-FSTL3-Fc</div>
<p>(A) Schematic presentation of mono-FSTL3-Fc in either non-reduced or reduced conditions. The predicted protein molecular weight (MW) was calculated by the sum of the MW of all amino acids.</p>
<p>(B) Results of Coomassie Brilliant Blue (CBB) staining and immunoblot analysis for human IgG Fc. It is of note that FSTL3 is a target of post-translational modification including glycosylation, which may affect the molecular weights.</p>
</figcaption>
</figure>
</li>
<li>Determine the concentration of the proteins by anti-human IgG enzyme-linked immunosorbent assay (ELISA).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Although the expected molecular weight of the purified mono-FSTL3-Fc is ∼76 kDa, post-translational modification including glycosylation seems to affect the molecular weights (<a href="#fig2">Figure 2</a>).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Amount of the mono-FSTL3-Fc protein can be roughly estimated using SDS-PAGE or BCA assays.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Ligand-neutralizing activities of mono-FSTL3-Fc can be assessed by reporter cell-based ligand-neutralizing assays as described elsewhere (<a href="#bib6">Ozawa et al., 2021</a>).</div>
<h3 id="sec2.5">Intraperitoneal administration of mono-FSTL3-Fc</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>We here describe the protocol used in our recent study to evaluate the effects of mono-FSTL3-FC following intraperitoneal injection (10 mg/kg, twice weekly for 2 weeks) in mice (<a href="#bib6">Ozawa et al., 2021</a>). The administration regimen/dosage was based on previous reports of anti-GDF8/myostatin therapeutic agents using Fc-fused protein, including ActRIIB-Fc (<a href="#bib11">Cadena et al., 2010</a>; <a href="#bib12">Pistilli et al., 2011</a>). In several 4-week protocols for mouse systemic administration, the increase of muscle mass was observed as early as 2 weeks after the initial injection, which was reproduced in our study (<a href="#bib6">Ozawa et al., 2021</a>). To reduce the suffering of animals, we decided to inject the mice twice weekly for 2 weeks rather than for 4 weeks.</p>
<ol start="22">
<li>Weigh the animal and calculate the volume to be administered.</li>
<li>Intraperitoneally inject the mono-FSTL3-Fc protein diluted with sterile PBS (10 mg/kg, 0.1–0.5 mL).</li>
<li>Repeat the process twice weekly for 2 weeks (total 4 injections).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Young male mice of C57BL/6 and CD2F1 (CDF1) strains are preferentially used for the analysis of muscle mass with anti-GDF8/myostatin therapy. One possible reason is to avoid differences in skeletal muscle physiology between the sexes. In addition, male <i>mdx</i> mice, a mouse model of Duchenne muscular dystrophy (DMD), are preferentially analyzed as a disease model. DMD is an X-linked recessive disease caused by mutations in the dystrophin gene and affects mainly boys.</div>
<h3 id="sec2.6">Assessment of mouse muscles following treatment with mono-FSTL3-Fc</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day (depending on the number of mice)</div>
<ol start="25">
<li>Prepare an isopentane and acetone mixture (1:2) cooled to −100°C with a cooling apparatus, such as UT2000F.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is preferable to use a cooling apparatus such as UT2000F, rather than liquid nitrogen or dry ice.</div>
<ol start="26">
<li>Measure mouse body weight.</li>
<li>Euthanize the mouse in accordance with institutional guidelines.</li>
<li>Spray the mouse with 70% ethanol to reduce the risk of the fur sticking to the muscles.</li>
<li>Make a small incision in the lower abdomen and pull back the skin.</li>
<li>Carefully separate the gastrocnemius (GC), tibialis anterior (TA), hamstrings (Ham), and quadriceps femoris (QF) muscles, starting from the tendon.</li>
<li>Handle the muscle by the tendon and excise it (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1022-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative macroscopic images of the mouse hindlimb muscles</div>
<p>(A) Dorsal view of the hindlimb region.</p>
<p>(B) Isolated skeletal muscles.</p>
<p>TA: tibialis anterior, Ham: hamstrings, QF: quadriceps femoris, GC: gastrocnemius. Scale: cm.</p>
</figcaption>
</figure>
</li>
<li>Prepare the sample for cryosectioning
<ol type="a">
<li>Cover the muscle in a small amount of fresh embedding medium such as FSC 22 frozen section medium.</li>
<li>Cut the muscle at the mid-belly region (<a href="#fig4">Figure 4</a>A) and place it with the cut surface down on aluminum foil (<a href="#fig4">Figure 4</a>B).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1022-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Schematic presentation of skeletal muscle sample preparation for cryosectioning</div>
<p>(A) Cover the muscle in a small amount of fresh embedding medium such as FSC 22 frozen section medium, and cut the muscle at the mid-belly region.</p>
<p>(B and C) Place the sample with the cut surface down on the aluminum foil, and dip the aluminum foil with the muscle into the isopentane/acetone bath for 10–20 s</p>
<p>(D and E) Embed the frozen muscle into FSC 22 frozen section medium in a tissue-embedding mold, and dip the mold with the muscle into the isopentane/acetone bath.</p>
</figcaption>
</figure>
</li>
<li>Dip the aluminum foil with the muscle into the isopentane/acetone bath for 10–20 s (<a href="#fig4">Figures 4</a>B and 4C).</li>
<li>Embed the frozen muscle into FSC 22 frozen section medium in a tissue-embedding mold, maintaining the same orientation (cut surface down, <a href="#fig4">Figures 4</a>D and 4E).</li>
<li>Dip the mold with the muscle into the isopentane/acetone bath (1–2 min) (<a href="#fig4">Figure 4</a>E).</li>
</ol>
</li>
<li>Once the specimen is frozen, keep it on dry ice at all times until transferring to a −80°C freezer.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Frozen specimen can be stored for several months at −80°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> When the freezing process is too slow, crystal formation within the myofibers can occur.</div>
<h3 id="sec2.7">Muscle sectioning and quantification of the muscle fiber cross-sectional area (CSA)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="34">
<li>Cut sections 5 μm thick in a cryostat at −20°C.</li>
<li>Transfer the tissue section to a microscope slide by touching the slide to the section at 25°C–28°C.</li>
<li>Air dry the sections using a hair dryer with cool air for 5 min so that the tissue sections tightly adhere to the slides.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Frozen tissue samples and unfixed slides can be stored for several months at −80°C.</div>
<ol start="37">
<li>Fix the sections in 4% PFA/PBS.</li>
<li>Stain the sections with anti-laminin (1:500 dilution) antibody.</li>
<li>Obtain images of the section using a fluorescence microscope, such as BZ-X700 microscope (Keyence).</li>
<li>Quantify the muscle fiber CSA with image analysis using Hybrid Cell Count software (Keyence) (<a href="#fig5">Figure 5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1022-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Quantification of the muscle fiber cross-sectional area (CSA)</div>
<p>(A and B) Representative cross-sectional images of myofibers in the GC muscle excised from mice with 2-week intraperitoneal administration of mono-FSTL3-Fc. A: immunohistochemistry (IHC) for laminin (red) with DAPI nuclear stain (blue), B: quantification using Hybrid Cell Count software. Scale bar: 100 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> There are several open-source software which can automatically quantify muscle fiber CSA, such as ImageJ plug-in MuscleJ (<a href="#bib5">Mayeuf-Louchart et al., 2018</a>).</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>In our system, mono-FSTL3-Fc protein is successfully expressed and purified with a yield of approximately 4 mg per 1 L culture medium.</p>
<p>For <i>in vivo</i> evaluation, each mouse requires approximately 1 mg (250 μg/shot × 4 times) of mono-FSTL3-Fc for the experiment. In our 2-week protocol, intraperitoneal injection of mono-FSTL3-Fc (10 mg/kg, twice weekly for 2 weeks) significantly increased skeletal muscle mass, including the hindlimb muscles; the weight of the hindlimb muscles on both sides, except for the TA muscle, increased by approximately 20–30% compared with that of control-Fc-treated wild-type mice.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The amount/concentration of mono-FSTL3-Fc is quantified with anti-human IgG ELISA.</p>
<p>Muscle weight is normalized to the body weight. Muscles from 10 mice were analyzed for each group/condition. The differences between the conditions were analyzed by analysis of variance (ANOVA) with Tukey-Kramer post hoc test for multiple comparisons.</p>
<p>We used Hybrid Cell Count software (Keyence), which semi-automatically traces and quantifies the muscle fiber CSA. Other software is available for this purpose, such as SMASH, MyoVision, MuscleJ and Open-CSAM (<a href="#bib3">Desgeorges et al., 2019</a>; <a href="#bib5">Mayeuf-Louchart et al., 2018</a>; <a href="#bib9">Smith and Barton, 2014</a>; <a href="#bib10">Wen et al., 2018</a>).</p>
<p>Images of the section were obtained using 5 mice for each group/condition and at least 1 slide from each mouse. Between 3000 and 6000 individual muscle fibers were analyzed, and the average muscle fiber CSA was calculated. The differences in muscle fiber CSA were analyzed using the Wilcoxon rank-sum test, and p values were adjusted with Benjamini-Hochberg correction for multiple comparisons.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Our protocol uses adherent HEK293T cells grown in a standard incubator (5% CO<sub>2</sub>; 37°C). This method requires only commonly accessible equipment to express and affinity-purify recombinant mono-FSTL3-Fc protein, resulting in about 10 mg protein from 2–5 L culture medium. In our 2-week protocol to evaluate the systemic effects of mono-FSTL3-Fc <i>in vivo</i>, each mouse is injected with approximately 1 mg of mono-FSTL3-Fc in total. Thus, our purification protocol is suitable for mouse experiments in a single project. If larger amounts of protein are desired (i.e., 100 mg or more), it is preferable to use cells adapted to grow in large-scale suspension culture, such as HEK293F cells.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>After the antibiotic selection, the resistant cells which express mono-FSTL3-Fc do not grow (step 3).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Make sure that lentiviruses are successfully generated. Usually, Lenti-X 293T cells produce a high-titer lentivirus (up to 1×10<sup>8</sup> IFU/mL). One option is to determine the titer of the lentivirus. Another option is to infect the two viruses separately.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Protein expression level of mono-FSTL3-Fc is low (step 4).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Monoclonal cell lines can be obtained by limiting dilution, and a high-expressing clone will be used in the following experiments.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The purity of mono-FSTL3-Fc protein is low (step 20).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Low purity can result from multiple factors. Make sure that the mono-FSTL3-Fc stably expressing cells efficiently express mono-FSTL3-Fc heterodimer. If the protein expression ratio of FSTL3-Fc(T366W) and Fc(T366S/L368A/Y407V) is not comparable, the purity of the mono-FSTL3-Fc heterodimer will be affected. Therefore, you might need to confirm the DNA sequences of knobs-into-holes mutations in the Fc domain.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The yield of mono-FSTL3-Fc after purification is low (step 21).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Make sure that supernatants contain mono-FSTL3-Fc protein. It is also important to evaluate the flow through of the column. We use protein A for purification, which has been shown to bind to the human IgG Fc domain. We use a pH 3.5 buffer for elution from the protein A column. You can also use protein G or other methods for purification, although this will require additional optimization.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Freezing artifacts in the muscle tissue (step 39).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Formalin fixation and paraffin embedding is a widely adopted method for pathological evaluation. However, this process may cause shrinkage artifacts that affect evaluation of the skeletal muscle. Moreover, freezing of the skeletal muscle is technically challenging since ice crystals can form within myofibers. The ice crystals are considered to be more likely to form when the freezing process is too slow or when the specimen contains excessive water. Therefore, it is critical to keep the isopentane and acetone mixture (1:2) at −100°C. Liquid nitrogen or dry ice can be used to cool refrigerants, but the temperature may fluctuate if the muscle tissue is frozen repeatedly; therefore, it is preferable to use a cooling apparatus such as UT2000F. In addition, it is important to remove any excess water in the specimen by blotting with a paper towel.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Atwell, S., Ridgway, J.B., Wells, J.A., and Carter, P. (1997). Stable heterodimers from remodeling the domain interface of a homodimer using a phage display library. J. Mol. Biol. <i>270</i>, 26-35. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref1">View at publisher</a></p>
<p id="bib11">Cadena, S.M., Tomkinson, K.N., Monnell, T.E., Spaits, M.S., Kumar, R., Underwood, K.W., Pearsall, R.S., and Lachey, J.L. (2010). Administration of a soluble activin type IIB receptor promotes skeletal muscle growth independent of fiber type. J. Appl. Physiol. (1985) <i>109</i>, 635-642. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/optLs5Px8uBmI">View at publisher</a></p>
<p id="bib2">Chang, C. (2016). Agonists and antagonists of TGF-β family ligands. Cold Spring Harb. Perspect. Biol. <i>8</i>, a021923. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref2">View at publisher</a></p>
<p id="bib3">Desgeorges, T., Liot, S., Lyon, S., Bouvière, J., Kemmel, A., Trignol, A., Rousseau, D., Chapuis, B., Gondin, J., Mounier, R., et al. (2019). Open-CSAM, a new tool for semi-automated analysis of myofiber cross-sectional area in regenerating adult skeletal muscle. Skelet. Muscle <i>9</i>, 2. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref3">View at publisher</a></p>
<p id="bib4">Harada, M., Morikawa, M., Ozawa, T., Kobayashi, M., Tamura, Y., Takahashi, K., Tanabe, M., Tada, K., Seto, Y., Miyazono, K., et al. (2019). Palbociclib enhances activin-SMAD-induced cytostasis in estrogen receptor-positive breast cancer. Cancer Sci. <i>110</i>, 209-220. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref4">View at publisher</a></p>
<p id="bib5">Mayeuf-Louchart, A., Hardy, D., Thorel, Q., Roux, P., Gueniot, L., Briand, D., Mazeraud, A., Bouglé, A., Shorte, S.L., Staels, B., et al. (2018). MuscleJ: a high-content analysis method to study skeletal muscle with a new Fiji tool. Skelet. Muscle <i>8</i>, 25. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref5">View at publisher</a></p>
<p id="bib6">Ozawa, T., Morikawa, M., Morishita, Y., Ogikubo, K., Itoh, F., Koinuma, D., Nygren, P.Å., and Miyazono, K. (2021). Systemic administration of monovalent follistatin-like 3-Fc-fusion protein increases muscle mass in mice. iScience <i>24</i>, 102488. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref6">View at publisher</a></p>
<p id="bib12">Pistilli, E.E., Bogdanovich, S., Goncalves, M.D., Ahima, R.S., Lachey, J., Seehra, J., and Khurana, T. (2011). Targeting the activin type IIB receptor to improve muscle mass and function in the mdx mouse model of Duchenne muscular dystrophy. Am. J. Pathol. <i>178</i>, 1287-1297. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/opt9BsocxAGYW">View at publisher</a></p>
<p id="bib7">Ridgway, J.B., Presta, L.G., and Carter, P. (1996). 'Knobs-into-holes' engineering of antibody CH3 domains for heavy chain heterodimerization. Protein Eng. <i>9</i>, 617-621. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref7">View at publisher</a></p>
<p id="bib8">Saito, S., Sidis, Y., Mukherjee, A., Xia, Y., and Schneyer, A. (2005). Differential biosynthesis and intracellular transport of follistatin isoforms and follistatin-like-3. Endocrinology <i>146</i>, 5052-5062. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref8">View at publisher</a></p>
<p id="bib9">Smith, L.R. and Barton, E.R. (2014). Smash - semi-automatic muscle analysis using segmentation of histology: a MATLAB application. Skelet. Muscle <i>4</i>, 21. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref9">View at publisher</a></p>
<p id="bib10">Wen, Y., Murach, K.A., Vechetti, I.J., Fry, C.S., Vickery, C., Peterson, C.A., McCarthy, J.J., and Campbell, K.S. (2018). MyoVision: software for automated high-content analysis of skeletal muscle immunohistochemistry. J. Appl. Physiol. <i>124</i>, 40-51. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00545-1/sref10">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Masato Morikawa (<a href="mailto:morikawa-tky@umin.ac.jp">morikawa-tky@umin.ac.jp</a>).</p>
<h4>Materials availability</h4>
<p>Materials used or generated in this study will be available upon reasonable request, and a material transfer agreement may be required.</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by KAKENHI (Grants-in-Aid for Scientific Research [C] [grant number 19K07683 to M.M.]), Grant-in-Aid for Scientific Research (S) from <a href="https://doi.org/10.13039/501100001691">Japan Society for the Promotion of Science</a> (JSPS) (15H05774 to K.M.), and Grant-in-Aid for Scientific Research on Innovative Area on Integrated Analysis and Regulation of Cellular Diversity (17H06326 to K.M. and M.M.) from the Ministry of Education, Culture, Sports, Science and Technology (<a href="https://doi.org/10.13039/501100001700">MEXT</a>) of Japan. T.O. was supported by <a href="https://doi.org/10.13039/501100001691">JSPS</a> Research Fellowship for Young Scientists (DC2). We thank Dr. H. Miyoshi (deceased, formerly RIKEN, Japan) for the lentivirus vector system and Drs. F. Itoh and P.Å. Nygren for discussion.</p>
<h3>Author contributions</h3>
<p>T.O. and M.M. developed and wrote the method. K.M. supervised the project.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>