<article>
<section>
<h1 id="header">Protocol to prepare functional cellular nanovesicles with PD1 and TRAIL to boost antitumor response</h1>
<p><time datetime="2021-02-12">Published: February 12, 2021</time></p>
<p>Peiwen Yu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Dongye Zheng,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Cuilin Zhang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Ming Wu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> and Xiaolong Liu<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn3">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>The United Innovation of Mengchao Hepatobiliary Technology Key Laboratory of Fujian Province, Mengchao Hepatobiliary Hospital of Fujian Medical University, Fuzhou 350025, P.R. China</p>
<p id="aff2"><sup>2</sup>Mengchao Med-X Center, Fuzhou University, Fuzhou 350116, P.R. China</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:wmmj0419@163.com">wmmj0419@163.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:xiaoloong.liu@gmail.com">xiaoloong.liu@gmail.com</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100324">10.1016/j.xpro.2021.100324</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Immunotherapy has achieved notable success in tumor treatment, but it is restricted to a small number of patients due to multiple immunosuppressive pathways in the tumor microenvironment. Here, we present a step-by-step protocol to prepare functional cellular nanovesicles from HEK293-FT cells displaying PD1 and TRAIL. TRAIL specifically induces immunogenic cancer cell death to initiate an immune response, and ectogenic PD1 blocks the PD1/PDL1 checkpoint signal to reactivate anergic tumor-specific CD8+ T cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib11">Wu et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Established stable HEK293 FT cells expressing PD1 and TRAIL</li>
<li>Prepare functional cellular nanovesicles (FCNVs) from stable HEK293 FT cells</li>
<li>FCNVs can boost antitumor response by the cooperative roles of PD1 and TRAIL</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Design plasmids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol>
<li>Download the target gene sequence from NCBI Gene database. For this protocol, we use house mouse PD1 (Programmed cell death 1, Pdcd1) (<a href="#bib9">Tang et al., 2018</a>), TRAIL (Tumor necrosis factor superfamily member 10, Tnfsf10) (<a href="#bib10">von Karstedt et al., 2017</a>), and their Accession are NC_000067.7 and NC_000069.7, respectively. See <a href="#fig1">Figure 1</a>.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The detailed target gene information from the NCBI database</div>
</figcaption>
</figure>
</li>
</ol>
<ol start="2">
<li>Integrate target gene sequence with Kozak sequence, linker sequence and fluorescent protein sequence (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic illustration of the integration and insertion of the sequence of interest</div>
</figcaption>
</figure>
<ol type="a">
<li>Acquire Kozak sequence on the website (<a href="https://baike.sogou.com/v7771534.htm">https://baike.sogou.com/v7771534.htm</a>)</li>
<li>Determine the Linker sequence according to experimental needs (<a href="#bib2">Chen et al., 2013</a>). For this protocol, we choose the GS linker to connect the target gene sequence and the fluorescent protein sequence.
<div class="note">
<span class="note-title">Note:</span> According to their functions in the construction of recombinant fusion proteins, peptide linkers are generally classified into three categories: flexible linkers, rigid linkers, and <i>in vivo</i> cleavable linkers. Among these linkers, flexible linkers (usually peptides rich in small or hydrophilic amino acids) could provide a certain degree of flexibility between the joined domains to allow interaction or increase spatial separation between the domains, while rigid linkers (such as helical structures and peptides rich in Prolines) could keep the distance between the joined domains to maintain the stability and bioactivity of the domains. And <i>in vivo</i> cleavable linkers are commonly used to generate commensurable proteins, which are physically separated from each other to prevent any potential interaction and steric hindrance. Particularly in this protocol, in order to make sure that the target protein and fluorescent protein are constantly fused to each other to facilitate the imaging, meanwhile to increase spatial separation to minimize steric hindrance, the most commonly used flexible linker consisting of Glycine and Serine (abbreviated as “GS” linker) is chosen as the peptide linker. In addition to act as a flexible linker used in fusion proteins, GS linker also can be used in single-chain variable fragment (scFv) to link the variable regions of heavy and light chain of the complete antibody, or optimized to improve the oral efficacy of recombinant granulocyte colony-stimulating factor and transferrin fusion protein.</div>
</li>
<li>Download the fluorescent protein sequence as step 1 described from the NCBI Gene database. For this protocol, we select red fluorescent protein (mCherry, ADV78248.1) and green fluorescent protein (EGFP, NC_025025.1) as fluorescence tags.</li>
<li>Kozak sequence, Linker sequence, target gene sequence and fluorescent protein sequence are connected in order as shown in <a href="#fig2">Figure 2</a>, and the integrity of the integrated sequences is checked by Snapgene software.</li>
<li>Entrust the company (Sangon Biotech (Shanghai) Co., Ltd. <a href="https://www.sangon.com/">https://www.sangon.com/</a>) to synthesize the integrated sequences and insert them into the lentivirus vector plasmid (pCDH-CMV-MCS-EF1-Puro and pCDH-CMV-MCS-EF1-Blasticidin) through linking at the end of the C-terminal of the plasmid (<a href="#bib12">Zhang et al., 2018a</a>, <a href="#bib13">2018b</a>).
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The pre-synthesized lentivirus vectors can also be homemade by the following steps. First, design the PCR primers to amplify the coding sequence of the target gene (remember to contain Kozak sequence, restriction enzyme cutting site and optimal bases for flanking the restriction site at the 5'end of the forward primer; for the reverse primer, stop codon should be removed to fuse the protein with GS linker and fluorescent protein). Second, the target gene fragment is generated by PCR amplification from cDNA template and purified by electrophoresis. Next, the PCR product and vector plasmid are respectively subjected to restriction enzyme digestion and purified by electrophoresis. The purified linear vector and the PCR fragment are ligated by T4 ligase to construct the interested plasmid, which is further imported into competent cells and dotted onto LB agar plates (plus antibiotic). The monoclonal colonies on the plate are selected and multiplied in LB medium for PCR identification, and the positive clones are sequenced. At last, the correct clone is expanded in large-scale and the plasmid is extracted with EndoFree Maxi Plasmid Kit following the manufacturer’s instructions.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Plasmid amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>After obtaining the above mentioned two plasmids (pCDH-CMV-PD1-mCherry-Puro, pCDH-CMV-TRAIL-EGFP-Blasticidin), high-quality plasmids were obtained by purification with EndoFree Maxi Plasmid Kit (TIANGEN, DP117, <a href="https://www.tiangen.com/?app.html">https://www.tiangen.com/?app.html</a>).</p>
<ol start="3">
<li>To prepare LB medium, dissolve 10 g tryptone, 5 g Yeast extract and 10 g NaCl in 1 L of ultrapure water (use magnetic stirrer, 500 rpm/min), aliquot to 200 mL and autoclave at 121°C for 15 min. After the medium is naturally cooled to 20°C–25°C, add antibiotics (containing ampicillin 100 μg/mL or kanamycin 50 μg/mL).</li>
<li>Take out the Competent Cell (TransGen, CD501-02, <a href="https://www.transgen.com.cn/">https://www.transgen.com.cn/</a>) from the −80°C refrigerator and leave on the ice for 15 min.</li>
<li>Add 20 μL Competent Cell and 5 μL plasmids into a sterilized 1.5 mL Eppendorf (EP) tube, mix well, and place on ice for 0.5 h.</li>
<li>The mixture is then placed on 42°C water bath for 30 s, then immediately placed on ice for 2 min.</li>
<li>500 μL free-antibiotic LB medium is added to EP tube and shaken at 37°C for 40–60 min to restore normal growth of competent cell, using bench type thermostatic oscillator.</li>
<li>The mixture is added to 200 mL LB medium containing the antibiotics and shaken at 37°C for 12–14 h to expand plasmids.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 9.  Unless otherwise indicated, antibiotics are applied as: 100 μg/mL ampicillin and 50 μg/mL kanamycin. Plasmids are extracted from the microbial broth according to the instructions of the EndoFree Maxi Plasmid Kit (TIANGEN, DP117, <a href="https://www.tiangen.com/?app.html">https://www.tiangen.com/?app.html</a>).</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The microbial broth can be stocked in 25% glycerol at −80°C for 1 year.</div>
<ol start="10">
<li>The concentration of the extracted plasmids is detected by NanoDrop 2000/2000c Spectrophotometers (Thermo Fisher Scientific), and the sequence of plasmids is verified by biological company (BioSune (Shanghai) Co., Ltd. <a href="http://www.biosune.com/">http://www.biosune.com/</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A good quality and endotoxin free of plasmids are important for successful transfection in HEK293 FT cells.</div>
<h3 id="sec1.3">Cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<ol start="11">
<li>Prepare cell culture medium. Mix 500 mL of DMEM, 50 mL of FBS, 5 mL of Penicillin-Streptomycin solution (penicillin: 10,000 units/mL, streptomycin 10,000 mg/mL) in a sterilized environment.</li>
</ol>
<ol start="12">
<li>Culture cells for transfection and infection
<ol type="a">
<li>Thaw HEK293FT cells cryo-preserved in a liquid nitrogen freezer at 37°C for 5 min, add 10 mL of PBS to wash and centrifuge at 300 × <i>g</i> for 5 min. Aspirate the PBS.</li>
<li>Resuspend the cell pellet in 1 mL of culture medium as prepared in step 11, transfer the cells into T25 cell culture flask and then supplement with 4 mL of culture medium.</li>
<li>Place the cell culture flask in an incubator with 5% CO<sub>2</sub> at 37°C.
<div class="note">
<span class="note-title">Note:</span> After culture the cells in incubator for about 12 h, there are some dead cells floating in the medium, then the medium should be replaced by fresh one.</div>
</li>
<li>When cells reach 80%–90% confluence, aspirate the culture medium from the cell culture flask and rinse the cells once with 5 mL of PBS using an electronic pipette.</li>
<li>Add 0.5 mL trypsin-EDTA to the cells and place the cells into the incubator with 5% CO<sub>2</sub> at 37°C incubator for 3 min.</li>
<li>Add 10 mL culture medium and detach the cells by slowly pipetting up and down, and count cell number under a microscope using a hemocytometer.</li>
<li>Transfer the culture medium into a T75 cell culture flask using an electronic pipette.</li>
<li>Place the cell culture flask in an incubator with 5% CO<sub>2</sub> at 37°C. When cells have reached 80%–90% confluence, split cells 1:4 into another T75 cell culture flask by repeating steps d–f.</li>
<li>Continue to sub-culture cells, maintaining 80%–90% confluence, until ready for construction of stable cell lines expressing PD1 and TRAIL.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-PD1</td>
<td>Abcam</td>
<td>Cat# ab214421</td>
</tr>
<tr>
<td>Mouse monoclonal anti-TRAIL</td>
<td>Abcam</td>
<td>Cat# ab10516</td>
</tr>
<tr>
<td>Goat anti-rabbit IgG (H+L), HRP conjugate</td>
<td>TransGen Biotech</td>
<td>Cat# HS101-01</td>
</tr>
<tr>
<td>Goat anti-mouse IgG (H+L), HRP conjugate</td>
<td>TransGen Biotech</td>
<td>Cat# HS201-01</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Trans1-T1 phage resistant chemically competent cell</td>
<td>TransGen Biotech</td>
<td>Cat# CD501-02</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM</td>
<td>Gibco</td>
<td>Cat# 11995500</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Gibco</td>
<td>Cat# 31985-LSG</td>
</tr>
<tr>
<td>FBS</td>
<td>PAN</td>
<td>Cat# P30-3302</td>
</tr>
<tr>
<td>PBS</td>
<td>HyClone</td>
<td>Cat# SH30256.01</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>Gibco</td>
<td>Cat# 15140122</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Solarbio</td>
<td>Cat# YZ-1033000 Cas# 69-53-4</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>Solarbio</td>
<td>Cat# YZ-1355006 Cas# 5965-95-7</td>
</tr>
<tr>
<td>Tryptone</td>
<td>OXOID</td>
<td>Cat# LP0042</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>OXOID</td>
<td>Cat# LP0021</td>
</tr>
<tr>
<td>NaCl</td>
<td>Sinopharm</td>
<td>Cat# 73522260Cas# 7647-14-5</td>
</tr>
<tr>
<td>DiO</td>
<td>Beyotime</td>
<td>Cat# C1038</td>
</tr>
<tr>
<td>DiI</td>
<td>Beyotime</td>
<td>Cat# C1036</td>
</tr>
<tr>
<td>DAPI</td>
<td>Sigma-Aldrich</td>
<td>Cat# D9542Cas# 28718-90-3</td>
</tr>
<tr>
<td>PMSF</td>
<td>Beyotime</td>
<td>Cat# ST506</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat# P8340</td>
</tr>
<tr>
<td>Polybrene</td>
<td>Santa Cruz</td>
<td>Cat# SC-134220</td>
</tr>
<tr>
<td>Paraformaldehyde, 4%</td>
<td>Solarbio</td>
<td>Cat# P1110</td>
</tr>
<tr>
<td>RIPA lysis buffer</td>
<td>Servicebio</td>
<td>Cat#G2002</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>EndoFree Maxi Plasmid Kit</td>
<td>TIANGEN BIOTECH (BEIJING)</td>
<td>Cat# DP117</td>
</tr>
<tr>
<td>Easy II Protein Quantitative Kit</td>
<td>TransGen</td>
<td>Cat# DQ111-01</td>
</tr>
<tr>
<td>Membrane and Cytosol Protein Extraction Kit</td>
<td>Beyotime</td>
<td>Cat# P0033</td>
</tr>
<tr>
<td>Lipofectamine 3000 Transfection Kit</td>
<td>Thermo Fisher</td>
<td>Cat# L3000015</td>
</tr>
<tr>
<td>Mouse PD-1 Duo Set ELISA Kit</td>
<td>R&amp;D Systems</td>
<td>Cat# DY1021</td>
</tr>
<tr>
<td>Mouse TRAIL/TNFSF10 Duo Set ELISA Kit</td>
<td>R&amp;D Systems</td>
<td>Cat# DY1121</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Human: HEK293FT</td>
<td>ATCC</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Plasmid: pCDH-CMV-MCS-EF1-Puro</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid: pCDH-CMV-MCS-EF1-Blasticidin</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid: pCDH-CMV-PD1-mCherry-Puro</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid: pCDH-CMV- TRAIL-EGFP-Blasticidin</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Flowjo</td>
<td>Tree Star</td>
<td>10.0</td>
</tr>
<tr>
<td>Zen</td>
<td>Carl Zeiss</td>
<td>2012 SP2(blue edition)</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Confocal dish</td>
<td>NEST</td>
<td>Cat# 801005</td>
</tr>
<tr>
<td>6-well plate</td>
<td>Costar</td>
<td>Cat# 3335</td>
</tr>
<tr>
<td>0.22 μm filter</td>
<td>Merck Millipore</td>
<td>Cat#PRO3683</td>
</tr>
<tr>
<td>15 mL centrifuge tubes</td>
<td>Corning</td>
<td>Cat# 430790</td>
</tr>
<tr>
<td>25 cm<sup>2</sup> cell culture flasks</td>
<td>Corning</td>
<td>Cat# 430639</td>
</tr>
<tr>
<td>75 cm<sup>2</sup> cell culture flasks</td>
<td>Corning</td>
<td>Cat# 430641</td>
</tr>
<tr>
<td>Cell scraper</td>
<td>Thermo Fisher</td>
<td>Cat# 179693PK</td>
</tr>
<tr>
<td>CO<sub>2</sub> incubator</td>
<td>Thermo Scientific</td>
<td>HERAcell150i</td>
</tr>
<tr>
<td>Thermostat water bath</td>
<td>Jinghong</td>
<td>DK-8D</td>
</tr>
<tr>
<td>Clean bench</td>
<td>AIRTECH</td>
<td>SW-CJ-2FD</td>
</tr>
<tr>
<td>Autoclave sterilizer</td>
<td>ZEALWAY</td>
<td>GR85DR</td>
</tr>
<tr>
<td>Milli-Q gradient system</td>
<td>Milli-Q Reference</td>
<td>Milli-Q Integral 3</td>
</tr>
<tr>
<td>NanoDrop2000/2000c spectrophotometer</td>
<td>Thermo Scientific</td>
<td>ND2000LAPTOP</td>
</tr>
<tr>
<td>Ultracentrifuge</td>
<td>Beckman Coulter</td>
<td>Optima XPN-100</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Thermo Fisher</td>
<td>Sorvall ST 16R</td>
</tr>
<tr>
<td>Dry bath</td>
<td>ZHISUN</td>
<td>Mk-20</td>
</tr>
<tr>
<td>Avanti mini-syringe</td>
<td>Avanti</td>
<td>610017-1Ea</td>
</tr>
<tr>
<td>Confocal laser scanning microscope</td>
<td>Zeiss</td>
<td>LSM780</td>
</tr>
<tr>
<td>Flow cytometry</td>
<td>BD</td>
<td>FACSVerse</td>
</tr>
<tr>
<td>Fluorescence microscope</td>
<td>Zeiss</td>
<td>Axio Vert.A1</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Preparation of 2× loading buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl (pH 6.8)</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>3.0 mL</td>
</tr>
<tr>
<td>Bromophenol Blue</td>
<td>1.0 mg</td>
</tr>
<tr>
<td>Glycerin</td>
<td>1 mL</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>0.5 mL</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Construction of stable cell lines expressing PD1 and TRAIL</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 weeks</div>
<p>When the plasmids containing PD1 or TRAIL sequence and HEK293FT cells are ready, the genetic engineered cells are constructed according to the overexpression protocol which involves plasmid transfection, antibiotic selection to construct stable cell lines (<a href="#bib4">Elegheert et al., 2018</a>). The procedure includes two parts. In the first part (steps 1–9), the HEK293FT cell lines are transfected using Lipofectamine 3000 kit (<a href="https://www.thermofisher.com/us/en/home.html?CID=cn-ebz-sem-sgk-gene-TFS-aBrand-pc-hp-082920-abcd-00000009FDC262D8">Thermo Fisher, L3000015</a>), and lentivirus supernatant are collected. In the second part, according to the resistance gene in the target gene plasmid, the corresponding antibiotic is added to screen the infected cells, then large-scale expanded (steps 10–13) (<a href="#bib1">Chaudhary et al., 2012</a>).</p>
<ol>
<li>Inoculate the parent HEK293FT cells in T75 culture flask in advance, and proceed to the next step after the cell density reaches 70%–80%.</li>
<li>Prepare reagents for gene transfection. Prepare a 15 mL centrifuge tube (numbered A) and a 1.5 mL EP tube (numbered B). Transfer 1 mL Opti-MEM medium (Gibco, 31985-LSG, <a href="https://www.thermofisher.com/us/en/home/brands/gibco.html?CID=cn-ebz-sem-sgk-gene-Gibco-aBrand-pc-subb-082920-abcd-00000009FD973428">https://www.thermofisher.com/us/en/home/brands/gibco.html?CID=cn-ebz-sem-sgk-gene-Gibco-aBrand-pc-subb-082920-abcd-00000009FD973428</a>) and 40 μL Lipo 3000 in Tube A, mix them by using gentle pipette motion. Tube B contains target gene plasmid (10 μg), lentivirus packaging systems (plp1 plasmid: 7.5 μg; plp2 plasmid: 3 μg; plp/vsvg plasmid: 4 μg) and P3000 (40 μL), which are dissolved in 1 mL Opti-MEM medium (Gibco, 31985-LSG,<a href="https://www.thermofisher.com/us/en/home/brands/gibco.html?CID=cn-ebz-sem-sgk-gene-Gibco-aBrand-pc-subb-082920-abcd-00000009FD973428">https://www.thermofisher.com/us/en/home/brands/gibco.html?CID=cn-ebz-sem-sgk-gene-Gibco-aBrand-pc-subb-082920-abcd-00000009FD973428</a>). See <a href="#tbl1">Table 1</a> for the description of all reagent for transfection. Then, set tube A and B aside for 5 min.
<table id="tbl1">
<caption>Table 1. The detailed description of transfection reagent</caption>
<thead>
<tr>
<th> </th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="2">Reagent</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>2 mL</td>
</tr>
<tr>
<td>Lipo 3000</td>
<td>40 μL</td>
</tr>
<tr>
<td>P3000</td>
<td>40 μL</td>
</tr>
<tr>
<td colspan="2">Plasmid</td>
</tr>
<tr>
<td>Plp1</td>
<td>7.5 μg</td>
</tr>
<tr>
<td>Plp2</td>
<td>3.0 μg</td>
</tr>
<tr>
<td>Plp/vsvg</td>
<td>4.0 μg</td>
</tr>
<tr>
<td>Target gene plasmid (PD1 or TRAIL )</td>
<td>10.0 μg</td>
</tr>
</tbody>
</table>
</li>
<li>Transfer the solution in Tube B to Tube A, mix them by using gentle pipette motion, and let the mixture stand for 20 min.</li>
<li>Add the above mixture to the cell culture flask, and lentiviral supernatant are collected after 24 h to 72 h of incubation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> mCherry tag and EGFP tag inserted in the target gene plasmid provide the fluorescent signals to evaluate the gene transfection efficiency. Under fluorescence microscope, when the percent of fluorescence cells reach above 60% (<a href="#fig3">Figure 3</a>), and the cell maintain normal morphology, it is suitable to collect the supernatant. If the transfection efficiency is low, See <a href="#troubleshooting">Troubleshooting</a>: <a href="#sec6.1">Problem 1</a>.</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Fluorescence images of HEK293 FT cells after transfection for 24 h</div>
<p>Scale bar, 100 μm.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Collect the supernatant and add fresh culture medium. Try to operate along the side wall of the culture flask, gently and slowly, and avoid direct contact with the cells.</div>
<ol start="5">
<li>Filter the supernatant containing lentivirus by passing it though a 0.22 μm filter to remove large cell fragments.</li>
<li>Transfer the sample to a sterilized ultracentrifugation tube. Place the centrifugation tube into a pre-cooled centrifuge. Counterbalance when needed.</li>
<li>Program the ultracentrifuge run using the following settings:
<ol type="a">
<li>Rotor type: SW 32.1 Ti Rotor</li>
<li>Speed: 76,800 × <i>g</i>
</li>
<li>Time: 90 min</li>
<li>Temperature: 4°C</li>
<li>Acceleration: 0 (MAX)</li>
<li>Deceleration: 0 (MAX)</li>
</ol>
</li>
<li>Start the ultracentrifuge.</li>
<li>After centrifugation, the supernatant is gently discarded, and 1 mL of fresh culture medium is added to resuspend the virus, then aliquot into 3–5 tubes (virus titer: 10<sup>6</sup>–10<sup>7</sup> IFU/mL), and stored at −80°C for future use.</li>
<li>Seed 1 mL of the cell suspension of HEK293FT cells (containing 1 × 10<sup>5</sup> cells) in a 6-well cell culture plate, place the plate in an incubator with 5% CO<sub>2</sub> at 37°C overnight.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Incubation time can be adjusted and terminated when 70%–80% cell confluence reach.</div>
<ol start="11">
<li>The cell culture is replaced by fresh medium supplemented with a tube of virus solution (containing 10<sup>6</sup>–10<sup>7</sup> IFU) and 5 μg/mL polybrene reagent to infect the cells. If the infection efficiency not high, see <a href="#troubleshooting">Troubleshooting</a>: <a href="#sec6.3">Problem 2</a>. After 2 or 3 days of culture, add puromycin or blasticidin to the cell for selecting PD-1 or TRAIL stably expression cells, respectively. The parent cells without lentiviral infection are used as control.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The concentration of puromycin or blasticidin should be adjusted above the lethal doses that kill nearly all the parent cells. If the high dose antibiotics also fail to screen, see <a href="#troubleshooting">Troubleshooting</a>: <a href="#sec6.5">Problem 3</a>.</div>
<ol start="12">
<li>Cells in the infected group are screened for 1–2 weeks until the percent of fluorescence cells reached above 95%, proved that the stably expression cells are successfully screened.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Different cells have different tolerance to antibiotics, so the concentration of drugs needs to be optimized.</div>
<div class="note">
<span class="note-title">Note:</span> Formation of puromycin or blasticidin resistance cells suggests the successful infection of lentivirus.</div>
<ol start="13">
<li>The stably expression cells are large-scale expanded.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When cells have reached 80%–90% confluence, split cells 1:3 onto another 6-well plate with fresh culture medium.</div>
<h3 id="sec3.2">Characterization of stable cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<p>The constructed stable cell lines are characterized by confocal laser scanning microscopy observation (<a href="#bib3">Dong et al., 2019</a>), western blot assay (<a href="#bib8">Ohno et al., 2013</a>) and flow cytometry assay (<a href="#bib7">Koh et al., 2017</a>) to confirm the target protein is well expressed and localized onto the cell membrane.</p>
<ol start="14">
<li>Confocal microscopy observation
<ol type="a">
<li>Cells are seeded in a confocal dish at a density of 1 × 10<sup>5</sup> per well. Place the dish in an incubator with 5% CO<sub>2</sub> at 37°C until they completely adhered.</li>
<li>Wash cells three times with PBS, and fix them with 4% paraformaldehyde for 10–15 min</li>
<li>Cell nucleus is stained with DAPI for 10 min, washed three times with PBS.</li>
<li>Cell membrane is stained with DiO or DiI for 15 min, washed three times with PBS, and observed under a confocal laser scanning microscope (Zeiss LSM780). See <a href="#fig4">Figure 4</a>.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Confocal images of HEK293 FT cells stably expressing mouse PD1</div>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>. Western blot assay
<ol type="a">
<li>Collect cells into a 1.5 mL EP tube by using cell scraper (Thermo Fisher Scientific), when the cells reach 70%–80% confluence in T75 cell culture flask.</li>
<li>The cells are lysed on ice with RIPA lysis buffer for 30 min. After lysis, the cells are centrifuged at 4°C for 30 min (13,000 × <i>g</i>), and the supernatant is collected for BCA protein quantification, using EasyⅡ Protein Quantitative Kit (TransGen, DQ111-01, <a href="https://www.transgen.com.cn/">https://www.transgen.com.cn/</a>).</li>
<li>According to the BCA protein quantification result, the sample is diluted to 4 mg/mL and mixed evenly with 2× loading buffer (volume ratio 1:1), and the mixed sample is boiled at 100°C for 10 min. For details on preparation of 2× loading buffer, see the <a href="#materials-and-equipment">Materials and equipment</a>.</li>
<li>The prepared sample is subjected to polyacrylamide gel electrophoresis for protein separation. Select β-actin protein as the internal reference protein for comparison. And then, transfer to the NC membrane by membrane transfer. Afterwards, the NC membrane is placed in 5% milk and blocked at 20°C–25°C for 2 h, and washed with TBST buffer (100 mM Tris-HCl (pH=8.0), 1.5 mM NaCl, 0.1% Tween 20) for 3 times, 10 min per time. After that, add the primary antibody (dilution rate, 1:1,000), See <a href="#key-resources-table">Key resources table</a>, and incubate for 12–16 h at 4°C.</li>
<li>The next day, recycle the primary antibody and wash the membrane with TBST buffer for 3 times with 15 min per time. Afterwards, add the secondary antibody and incubate for 1 h at 20°C–25°C. Then recycle the secondary antibody (dilution rate, 1:5,000), See <a href="#key-resources-table">Key resources table</a>, and wash the membrane with TBST buffer for 3 times with 10 min per time. Preparation the ECL and imaging on the gel imager (ChemiDoc MP) (<a href="#fig5">Figure 5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Western blot assay to confirm the expression of PD1 (indicated as PD1<sup>+</sup>) and TRAIL (indicated as TRAIL<sup>+</sup>) in 293FT cells</div>
<p>Actin was used as an internal reference.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Flow cytometer assay</li>
</ol>
<p>Above prepared 10<sup>5</sup> ∼5 × 10<sup>5</sup> cells are collected, centrifuged at 20°C–25°C for 5 min (300 × <i>g</i>) and resuspended in PBS. The fluorescent protein signal was analyzed by flow cytometry (BD, FACSVerse) (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. The representative flow cytometric analysis of HEK293 FT cells before and after transfected with PD 1 and TRAIL, respectively</div>
</figcaption>
</figure>
<h3 id="sec3.3">Cell membrane extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2.5 h</div>
<ol start="17">
<li>When cells have reached 80%–90% confluence, remove medium and add 1 mL trypsin to digest cells. Harvest cells from plate and centrifuge cells at 600 × <i>g</i> for 5 min (<a href="#bib6">Kim et al., 2017</a>).</li>
<li>Discard the supernatant, collect the cell precipitation and extract the cell membrane with a cell membrane extraction kit according to the manufacture’s instruction (Beyotime, P0033, <a href="https://www.beyotime.com/index.htm">https://www.beyotime.com/index.htm</a>).</li>
<li>Determine the protein content in the extracted membrane by BCA kit, and then store the membrane in −80°C refrigerator for future use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The storage time of the extracted membrane should be no more than 1 month in −80°C refrigerator.</div>
<div class="critical">
<span class="critical-title">Critical:</span> In order to maintain the biological function, the extracted membrane cannot be lyophilized for storage.</div>
<h3 id="sec3.4">Synthesize the functional vesicles</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>An Avanti mini-syringe extruder (<a href="#fig7">Figures 7</a> and <a href="#fig8">8</a>) is used to prepare cell membrane vesicles using the mechanical force imposed by the extruding process to disrupt the membrane structure and enable it to reform into nano-scale vesicles (<a href="#bib5">Fang et al., 2014</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. The Avanti mini-syringe extruder</div>
</figcaption>
</figure>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Schematic illustration of the parts of the Avanti mini-syringe extruder</div>
</figcaption>
</figure>
<ol start="20">
<li>Assemble the mini-syringe extruder
<ol type="a">
<li>Place the 2 Internal Membrane Supports on a flat test bed with the O-rings vertically upward</li>
<li>Pre-wet 2 Filter Supports with DI water, or buffer, and place over orifice of Internal Membrane Supports. The Filter Supports should adhere to the Teflon orifice inside the O-ring inner scope.</li>
<li>Insert the Internal Membrane Support into the Extruder Outer.</li>
<li>Place 1 Polycarbonate Membrane in the Extruder Outer Casing over the Filter Support and O-ring.
<div class="note">
<span class="note-title">Note:</span> Do not install the blue paper disks which separate the polycarbonate membranes in packing box.</div>
</li>
<li>Pre-wet a second pair of Filter Supports with DI water, or buffer, and place over orifice of remaining Internal Membrane Support.</li>
<li>Carefully place the second Internal Membrane Support into the casing (O-ring facing down).</li>
<li>Place the Retainer Nut on the threaded end of the Extruder Outer Casing and tighten. Tighten the Retainer Nut by hand just until it is finger tight; do not use a wrench.
<div class="note">
<span class="note-title">Note:</span> Autoclaving the Teflon inserts is not recommended as slight distortion may occur (∼0.002 mm). The use of ethylene oxide or UV irradiation is suggested as an alternative. The syringes should be sterilized with ethanol.</div>
<div class="critical">
<span class="critical-title">Critical:</span> See <a href="#fig9">Figure 9</a> and <a href="#mmc1">Methods Video S1</a> for detailed operate instructions of step 20.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Installation process for the Avanti mini-syringe extruder</div>
<p>(A and B) Place two Internal Membrane Support on a flat test bed with the O-rings vertically upward.</p>
<p>(C) Insert one Internal Membrane Support into the Extruder Outer Casing.</p>
<p>(D) Place one pre-wetted Filter Support over orifice of the Internal Membrane Support in the Extruder Outer Casing.</p>
<p>(E) Place one Polycarbonate Membrane over the Filter Support Support and O-ring in the Extruder Outer Casing.</p>
<p>(F) Place the second pair of pre-wetted Filter Support over orifice of the other Internal Membrane Support.</p>
<p>(G) Carefully place the second Internal Membrane Support into the Extruder Outer Casing (O-ring facing down).</p>
<p>(H) Place the Retainer Nut on the threaded end of the Extruder Outer Casing and tighten.</p>
<p>(I) Place the empty gas-tight syringe (with plunger is set to zero) into the one end of the mini-extruder.</p>
</figcaption>
</figure>
<ol start="21">
<li>Add the cell membrane solution into one of the gas-tight syringes and carefully place into one end of the mini-extruder (<a href="#fig10">Figure 10</a>).
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Schematic illustration of drawing the sample by syringe</div>
</figcaption>
</figure>
</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods Video S1. The detailed operate instructions for assemble the mini-syringe extruder, related to step 20</div>
</figcaption>
</figure>
<p><b>Note:</b> To reduce the dead volume and facilitate extrusion, pre-wet the extruder parts by passing a syringe full of buffer through the extruder.</p>
<ol start="22">
<li>Place the empty gas-tight syringe into the other end of the mini-extruder. Make sure the empty syringe plunger is set to zero; the syringe will fill automatically as the cell membrane is extruded through the polycarbonate membranes.</li>
<li>Gently push the plunger of the filled syringe until the cell membrane solution is completely transferred to the alternate syringe (<a href="#fig11">Figure 11</a>).
<figure id="fig11"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Fig11.jpg" alt="Fig11.jpg">
<figcaption>
<div class="figcaption-title">Figure 11. Schematic illustration of extrusion operation</div>
</figcaption>
</figure>
</li>
<li>Gently push the plunger of the alternate syringe to transfer the solution back to the original syringe.</li>
</ol>
<p>Repeat above steps 23 and 24. Total of 11 passes through the 1,000 nm, 400 nm, 200 nm, and 100 nm polycarbonate membranes, respectively. If the FCNVs fail to maintain their biological functions, see <a href="#troubleshooting">Troubleshooting</a>: <a href="#sec6.7">Problem 4</a>.</p>
<div class="note">
<span class="note-title">Note:</span> The final extrusion should fill the alternate syringe. This is to reduce the chances of contamination with larger particles or foreign material.</div>
<div class="critical">
<span class="critical-title">Critical:</span> After being pushed through the syringes for enough times, the diameter of final product will be around 100 nm with a polydispersity index (PDI) less than 0.2. When the diameter and/or PDI are larger than these values, increasing the pass times appropriately.</div>
<ol start="25">
<li>After the final extrusion, inject the obtained solution into a clean sample vial.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> When removing syringes, pull the syringe straight out of the extruder. See <a href="#mmc2">Methods Video S2</a> for detailed operate instructions of steps 21–25.</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/470-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods Video S2. The detailed operate instructions for obtaining the FCNVs, related to steps 21–25</div>
</figcaption>
</figure>
<h4 id="sec3.4.1">The characterization of FCNVs</h4>
<ol start="26">
<li>The size distribution of FCNVs is measured by dynamic light scattering (DLS) method through Zetasizer NanoZS system (Malvern Instruments, Southborough, MA, USA). Therein, the size measurement is performed in Size module with 200 s equilibration time using DTS0012 cuvette. The ultrastructure, integrity and morphology of FCNVs are evaluated by transmission electron microscopy (TEM). Therein, the sample is dropped onto a Cu-grid that covered with carbon film, and dried for 2 h. Subsequently, the sample is stained by 50 μL of tungstophosphoric acid for 5 min, and the redundant solvent was removed blotters. After drying again, the morphology was observed and photographed using JEM-1400Plus transmission electron microscopy (JEOL, Japan). The specific ratio and content of TRAIL and PD1 in FCNVs are determined by ELISA (R&amp;D Systems, DY1121 for TRAIL and DY1021 for PD1, <a href="https://www.rndsystems.com/cn">https://www.rndsystems.com/cn</a>).</li>
</ol>
<h4 id="sec3.4.2">Assessment of immunotherapy of FCNVs</h4>
<p>The TRAIL component on FCNVs could initiate the immune responses by specifically inducing immunogenic cancer cell death, which could be further strengthened by collaborating with presented ectogenic PD1 proteins as the checkpoint blockade to reactivate the anergic tumor-specific CTLs. The corresponding function of FCNVs can be evaluated both <i>in vitro</i> and <i>in vivo</i>.</p>
<ol start="27">
<li>In <i>vitro</i> immune checkpoint inhibition of FCNVs: naive mouse CD8+ T cells (CTLs) are first isolated from the spleen of BALB/c mice, followed by CD3/CD28 antibody stimulation to obtain activated CD8+ T cells. Subsequently, cancer cells with various treatments, such as blocking PDL1 on cell membranes by our FCNVs or PDL1 antibody, are added to the activated CD8+ T cells to check the cytotoxic activity by using Annexin V-FITC and PI cell apoptosis assay.</li>
<li>
<i>In vitro</i> immunogenic cancer cell death (ICD) induced by FCNVs: Anticancer effect of TRAIL proteins on FCNVs can be assessed by Annexin V-FITC and PI cell apoptosis assay, and the danger-associated molecular patterns (DAMPs) like CRT proteins released from dead cancer cells are checked by immunofluorescent staining. Finally, the release of DAMPs together with tumor-associated antigens (TAAs) from dead cells are co-incubated with borrow-derived dendritic cells (BMDCs) to check whether they can promote the maturation of BMDCs though dual staining for CD80 and CD86, the typical markers on the surface of maturated DCs.</li>
<li>
<i>In vivo</i> antitumor effect of FCNVs: The comprehensive immuno-modulating ability and robust antitumor immunity of our FCNVs can be demonstrated through evaluating tumor growth regression, metastasis prevention, distal and immune-memory effects, and long-term survival benefits.</li>
</ol>
<p>All of the above corresponding results can be found in our recent iScience publication (<a href="#bib11">Wu et al., 2020</a>).</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Following this protocol, we prepared functional cellular nanovesicles (FCNVs) with surface presenting PD1 and TRAIL. We advise to perform the particle and protein characterization to qualitatively and quantitatively assess the FCNVs, respectively. ELISA determination of PD1 and TRAIL content on FCNVs is necessary. Immunoprecipitation (IP) assay verifies the orientation of functional proteins on FCNVs. The size distribution is measured by dynamic light scattering (DLS) methods. FCNVs ultrastructure, integrity and morphology are evaluated by transmission electron microscopy (TEM). For detailed instructions on how to perform FCNVs characterization and the corresponding results can be found in our recent iScience publication (<a href="#bib11">Wu et al., 2020</a>).</p>
<p>We demonstrated the ability of this protocol to prepare FCNVs with high purity, which is a prerequisite to obtain reliable data for antitumor immune responses. For full details, we refer to (<a href="#bib11">Wu et al., 2020</a>). Theoretically, in addition to PD1 and TRAIL, other functional proteins decorated cellular nanovesicles can be also obtained by our protocols. Furthermore, the inner core of FCNVs can be used to encapsulated other functional components, for example, catalase is loaded into nanovesicles to overcome tumor hypoxia after producing O<sub>2</sub>, in turn to increase infiltration of effector T cells while deplete immunosuppressive cells in tumor (<a href="#bib11">Wu et al., 2020</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While this protocol was developed to obtain the functional cellular nanovesicles with multiple immuno-modulating ability to elicit robust and systematic antitumor immunity, the procedure is labor intensive and low-throughput, thus keep the same quality FCNVs from batch to batch and large-scale production might be an obstacle for pharmaceutical translation. Moreover, some functional proteins on FCNVs have right orientation (outwards), but some presented inwards which certainly can not play the right function in immunotherapy after co-extrusion, but in our current protocol it is very difficult to precisely control the orientation of functional proteins on the surface of FCNVs.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Why the efficiency of gene transfection is not high?</p>
<h3 id="sec6.2">Potential solution</h3>
<p>There are many factors affecting the transfection efficiency, such as the state of the cells, the quality of the transfection reagent and the purity of the plasmid. Therefore, the cells should be handled in a good condition, the transfection reagent free of quality problems, and the plasmid has high purity and the sequence has been verified before transfection.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Why the efficiency of infected target cells is not high?</p>
<h3 id="sec6.4">Potential solution</h3>
<p>We found that the transfection efficiency and the viral titer may directly influent the infection efficiency. Make sure the transfect efficiency more than 80% after 48 h of transfection. And the viral titer should reach above 10<sup>6</sup> IFU/mL to ensure better infection efficiency. Meanwhile, the storage time of collected virus supernatant at 4°C should not exceed 1 week. Also, if the efficiency of infection is still very low and you may choose secondary infection.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Why the antibiotics fail to screen even at high dose?</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Make sure the resistant drugs are within its warranty and properly stored. Then, determine whether the target cells already have the corresponding resistance gene. Alternatively, flow cytometry can also be used to select stably expression cells instead of drug screening.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>How to obtain homogeneous FCNVs with uniform morphology while maintain their biological functions?</p>
<h3 id="sec6.8">Potential solution</h3>
<p>We advise to mechanically harvest stable cells from cell culture flask by cell scraper rather than trypsin digestion to avoid potential harms to the functional proteins on the cell membranes including PD1 and TRAIL. Another suggestion to maintain the biological function is that the extracted membrane should not be lyophilized for storage. To improve the homogeneity of FCNVs, the repeats of extruding process in steps 23 and 24 can be appropriately increased.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chaudhary, S., Pak, J.E., Gruswitz, F., Sharma, V., and Stroud, R.M. (2012). Overexpressing human membrane proteins in stably transfected and clonal human embryonic kidney 293S cells. Nat. Protoc. <i>7</i>, 453-466. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref1">View at publisher</a></p>
<p id="bib2">Chen, X., Zaro, J.L., and Shen, W. (2013). Fusion protein linkers: property, design and functionality. Adv. Drug Deliver. Rev. <i>65</i>, 1357-1369. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref2">View at publisher</a></p>
<p id="bib3">Dong, X., Gao, J., Zhang, C.Y., Hayworth, C., Frank, M., and Wang, Z. (2019). Neutrophil membrane-derived nanovesicles alleviate inflammation to protect mouse brain injury from ischemic stroke. ACS Nano <i>13</i>, 1272-1283. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref3">View at publisher</a></p>
<p id="bib4">Elegheert, J., Behiels, E., Bishop, B., Scott, S., Woolley, R.E., Griffiths, S.C., Byrne, E.F.X., Chang, V.T., Stuart, D.I., Jones, E.Y., et al. (2018). Lentiviral transduction of mammalian cells for fast, scalable, and high-level production of soluble and membrane proteins. Nat. Protoc. <i>13</i>, 2991-3017. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref4">View at publisher</a></p>
<p id="bib5">Fang, R., Hu, C., Luk, B., Gao, W., Copp, J., Tai, Y., O'Connor, D.E., and Zhang, L. (2014). Cancer cell membrane-coated nanoparticles for anticancer vaccination and drug delivery. Nano Lett <i>14</i>, 2181-2188. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref5">View at publisher</a></p>
<p id="bib6">Kim, J., Han, C., Jo, W., Kang, S., Cho, S., Jeong, D., Gho, Y.S., and Park, J. (2017). Cell-engineered nanovesicle as a surrogate inducer of contact-dependent stimuli. Adv. Healthc. Mater. <i>6</i>, 1700381. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref6">View at publisher</a></p>
<p id="bib7">Koh, E., Lee, E., Nam, G., Hong, Y., Cho, E., Yang, Y., and Kim, I. (2017). Exosome-SIRPα, a CD47 blockade increases cancer cell phagocytosis. Biomaterials <i>121</i>, 121-129. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref7">View at publisher</a></p>
<p id="bib8">Ohno, S., Takanashi, M., Sudo, K., Ueda, S., Ishikawa, A., Matsuyama, N., Fujita, K., Mizutani, T., Ohgi, T., Ochiya, T., et al. (2013). Systemically injected exosomes targeted to EGFR deliver antitumor microRNA to breast cancer cells. Mol. Ther. <i>21</i>, 185-191. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref8">View at publisher</a></p>
<p id="bib9">Tang, J., Yu, J.X., Hubbard-Lucey, V.M., Neftelinov, S.T., Hodge, J.P., and Lin, Y. (2018). The clinical trial landscape for PD1/PDL1 immune checkpoint inhibitors. Nat. Rev. Drug Discov. <i>17</i>, 854-855. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref9">View at publisher</a></p>
<p id="bib10">von Karstedt, S., Montinaro, A., and Walczak, H. (2017). Exploring the TRAILs less travelled: TRAIL in cancer biology and therapy. Nat. Rev. Cancer <i>17</i>, 352-366. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref10">View at publisher</a></p>
<p id="bib11">Wu, M., Zheng, D., Zhang, D., Yu, P., Peng, L., Chen, F., Lin, Z., Cai, Z., Li, J., Wei, Z., et al. (2020). Converting immune cold into hot by biosynthetic functional vesicles to boost systematic antitumor immunity. iScience <i>23</i>, 101341. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref11">View at publisher</a></p>
<p id="bib12">Zhang, P., Zhang, L., Qin, Z., Hua, S., Guo, Z., Chu, C., Lin, H., Zhang, Y., Li, W., Zhang, X., et al. (2018a). Genetically engineered liposome-like nanovesicles as active targeted transport platform. Adv. Mater. <i>30</i>, 1705350. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref12">View at publisher</a></p>
<p id="bib13">Zhang, X., Wang, C., Wang, J., Hu, Q., Langworthy, B., Ye, Y., Sun, W., Lin, J., Wang, T., Fine, J., et al. (2018b). PD-1 blockade cellular vesicles for cancer immunotherapy. Adv. Mater. <i>30</i>, 1707112. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00031-9/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources should be directed to and will be fulfilled by the Lead Contact, Xiaolong Liu (<a href="mailto:xiaoloong.liu@gmail.com">xiaoloong.liu@gmail.com</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique materials.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the <a href="https://doi.org/10.13039/501100001809">National Natural Science Foundation of China</a> (grant no. 61727823), the joint research projects of Health and Education Commission of Fujian Province (grant no. 2019-WJ-20), the Joint Funds for the innovation of science and Technology, Fujian province (grant no. 2019Y9046), and the Scientific Foundation of Fuzhou Municipal Health commission (grant nos. 2020-S-wt7 and 2020-S-wp6).</p>
<h3>Author Contributions</h3>
<p>P.Y. and D.Z. wrote and prepared the manuscript. C.Z., M.W., and X.L. edited and helped in section organization.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>