<article>
<section>
<h1 id="header">Protocol to generate and characterize biofouling transformants of a model marine diatom</h1>
<p><time datetime="2021-08-06">Published: August 6, 2021</time></p>
<p>Weiqi Fu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">7</a>,<a href="#fn2">8</a>,<a href="#cor1">*</a></sup> Bushra Dohai,<sup><a href="#aff2">2</a></sup> Diana Charles El Assal,<sup><a href="#aff2">2</a></sup> Sarah Daakour,<sup><a href="#aff4">4</a></sup> Amnah Alzahmi,<sup><a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#aff6">6</a></sup> David R. Nelson,<sup><a href="#aff4">4</a></sup> Ashish Jaiswal,<sup><a href="#aff2">2</a></sup> Alexandra Mystikou,<sup><a href="#aff4">4</a></sup> Mehar Sultana,<sup><a href="#aff4">4</a></sup> James Weston,<sup><a href="#aff5">5</a></sup> and Kourosh Salehi-Ashtiani<sup><a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Marine Science, Ocean College, Zhejiang University, Zhoushan, Zhejiang 316021, China</p>
<p id="aff2"><sup>2</sup>Laboratory of Algal, Systems, and Synthetic Biology (LASSB), Division of Science and Math, New York University Abu Dhabi, P.O. Box 129188, Abu Dhabi, UAE</p>
<p id="aff3"><sup>3</sup>Center for Systems Biology and Faculty of Industrial Engineering, Mechanical Engineering and Computer Science, School of Engineering and Natural Sciences, University of Iceland, Reykjavik 101, Iceland</p>
<p id="aff4"><sup>4</sup>Center for Genomics and Systems Biology (CGSB), New York University Abu Dhabi, P.O. Box 129188, Abu Dhabi, UAE</p>
<p id="aff5"><sup>5</sup>Core Technology Platforms, New York University Abu Dhabi, Abu Dhabi, UAE</p>
<p id="aff6"><sup>6</sup>Department of Biology, United Arab Emirates University (UAEU), Al Ain, UAE</p>
<p id="fn1"><sup>7</sup>Technical contact</p>
<p id="fn2"><sup>8</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:weiqifu@zju.edu.cn">weiqifu@zju.edu.cn</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:ksa3@nyu.edu">ksa3@nyu.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100716">10.1016/j.xpro.2021.100716</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Diatoms are a major group of microalgae that initiate biofouling by surface colonization of human-made underwater structures; however, the involved regulatory pathways remain uncharacterized. Here, we describe a protocol for identifying and validating regulatory genes involved in the morphology shift of the model diatom species <i>Phaeodactylum tricornutum</i> during surface colonization. We also provide a workflow for characterizing biofouling transformants. By using this protocol, gene targets such as <i>GPCR</i> signaling genes could be identified and manipulated to turn off diatom biofouling.</p>
<p>For complete information on the generation and use of this protocol, please refer to <a href="#bib3">Fu et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for obtaining biofouling transformants of diatoms</li>
<li>Protocol includes characterization of biofouling transformants</li>
<li>We use systems biology approaches to select and validate GPCR signaling genes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/902-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used in a recent publication (<a href="#bib3">Fu et al., 2020</a>) to generate and characterize biofouling transformants of a model marine diatom <i>Phaeodactylum tricornutum</i>. While this protocol should be thoroughly followed and may take several months to complete, there are three major steps in the whole process with three corresponding stopping points that allow the experiment to be suspended if needed.</p>
<p>In addition to the standard lab equipment, including safety cabinets, PCR machines, centrifuges, consumables, etc., it is critical to use best laboratory practices and workspaces to avoid contamination with RNAse and consequent mRNA degradation when working with RNA.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td><i>Phaeodactylum tricornutum</i></td>
<td>CCAP</td>
<td>CCAP 1055/1</td>
</tr>
<tr>
<td>
<i>GPCR1A</i> transformants of <i>P. tricornutum</i>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>GPCR4</i> transformants of <i>P. tricornutum</i>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Solid culture <i>of P. tricornutum</i>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>NaNO<sub>3</sub>
</td>
<td>Sigma-Aldrich or major supplier</td>
<td rowspan="13"><b>f/2 + Si Medium∗ (Guillard’s medium for diatoms) `</b></td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>.2H<sub>2</sub>O</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>Na<sub>2</sub> EDTA</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>FeCl<sub>3</sub>.6H<sub>2</sub>O</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>CuSO<sub>4</sub>.5H<sub>2</sub>O</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>ZnSO<sub>4</sub>.7H<sub>2</sub>O</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>CoCl<sub>2</sub>.6H<sub>2</sub>O</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>MnCl<sub>2</sub>.4H<sub>2</sub>O</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>Na<sub>2</sub>MoO<sub>4</sub>.2H<sub>2</sub>O</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>Cyanocobalamin (vitamin B12)</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>Thiamine HCl (vitamin B1)</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>Biotin</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>Na<sub>2</sub>SiO<sub>3</sub>.9H<sub>2</sub>O</td>
<td>Sigma-Aldrich or major supplier</td>
</tr>
<tr>
<td>Chelex-100</td>
<td>Sigma-Aldrich or major supplier</td>
<td>CAS No. 11139-85-8</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>MagMAX-96 Total RNA Isolation Kit</td>
<td>Thermo Fisher Scientific Inc.</td>
<td>Kit AM1830</td>
</tr>
<tr>
<td>KAPA Library Quantification Kit</td>
<td>Roche</td>
<td>Kapa Biosystems</td>
</tr>
<tr>
<td>DNA Clean &amp; Concentrator<sup>TM</sup>-25 Kit</td>
<td>Zymo Research Corporation</td>
<td>DCC-25, Catalog No. 50444216</td>
</tr>
<tr>
<td>TruSeq V2 RNA Sample Prep Kit</td>
<td>Illumina</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>RNA-seq data with triplicates of biological samples for the liquid culture of <i>Phaeodactylum tricornutum</i>
</td>
<td>GenBank/NCBI database; Dryad</td>
<td>GenBank: PRJNA566271; Dryad with a unique identifier (<a href="https://doi.org/10.5061/dryad.ns1rn8ppx">https://doi.org/10.5061/dryad.ns1rn8ppx</a>).</td>
</tr>
<tr>
<td>RNA-seq data with triplicates of biological samples for the solid culture of <i>P. tricornutum</i>
</td>
<td>GenBank/NCBI database; Dryad</td>
<td>GenBank: PRJNA566271; Dryad with a unique identifier (<a href="https://doi.org/10.5061/dryad.ns1rn8ppx">https://doi.org/10.5061/dryad.ns1rn8ppx</a>).</td>
</tr>
<tr>
<td>RNA-seq data with triplicates of biological samples for the liquid culture of <i>GPCR1A</i> transformants of <i>P. tricornutum</i>
</td>
<td>GenBank/NCBI database; Dryad</td>
<td>GenBank: PRJNA566271; Dryad with a unique identifier (<a href="https://doi.org/10.5061/dryad.ns1rn8ppx">https://doi.org/10.5061/dryad.ns1rn8ppx</a>).</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td><i>Phaeodactylum tricornutum</i></td>
<td>CCAP</td>
<td>CCAP 1055/1</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Primers (forward, 5′- CACAAACCGAACAGCCCTAC-3’; reverse, 5′- TCGAGCTTCACAACCTGTCC-3′)</td>
<td>Integrated DNA Technologies</td>
<td>Primer of GPCR1A</td>
</tr>
<tr>
<td>Primers (forward, 5′-CACAAACCGAACAGCCCTAC-3’; reverse, 5′- CAGTGACGTTGCGACAATCC-3′)</td>
<td>Integrated DNA Technologies</td>
<td>Primer of GPCR4</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pPha-NR</td>
<td>NCBI</td>
<td>GenBank: JN180663.1</td>
</tr>
<tr>
<td>Synthetic genes (DNA constructs) cloned into the vector pPha-NR</td>
<td>Twist Bioscience</td>
<td>Gene names of <i>Phaeodactylum tricornutum</i>: Phatr3_J22677, Phatr3_J41807, Phatr3_Jdraft1756, Phatr3_J10677, Phatr3_J54411, Phatr3_EG02512, Phatr3_J51511, Phatr3_J12877, Phatr3_J2097, Phatr3_Jdraft1000, Phatr3_J54505, Phatr3_J55230, Phatr3_Jdraft1740, Phatr3_J44133</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>BLAST (Basic Local Alignment Search Tool) command-line applications developed at the National Center for Biotechnology Information (NCBI)</td>
<td>NCBI</td>
<td>N/A</td>
</tr>
<tr>
<td>BLAST2GO command line</td>
<td>blast2go_cli_v1.3.3</td>
<td>N/A</td>
</tr>
<tr>
<td>KEGG Automatic Annotation Server</td>
<td>KEGG: Kyoto Encyclopedia of Genes and Genomes</td>
<td>N/A</td>
</tr>
<tr>
<td>Cytoscape</td>
<td>Website: http:// <a href="http://www.cytoscape.org">www.cytoscape.org</a>
</td>
<td>v.3.6.0</td>
</tr>
<tr>
<td>CLUGO: a clustering algorithm for automated functional annotations based on gene ontology</td>
<td>Cytoscape</td>
<td>v2.5.0</td>
</tr>
<tr>
<td>DESeq2</td>
<td>R studio</td>
<td>Bioconductor version: Release (3.11)</td>
</tr>
<tr>
<td>STRING database</td>
<td>Website: <a href="https://string-db.org">https://string-db.org</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>KOD hot-start DNA polymerase</td>
<td>Novagen</td>
<td>Manufacturer Part Number: 710864</td>
</tr>
<tr>
<td>The genome of <i>Phaeodactylum tricornutum</i>
</td>
<td>Genome assembly: ASM15095v2</td>
<td>blastx: 2.6.0+</td>
</tr>
<tr>
<td>AMPK signaling pathway</td>
<td>KEGG</td>
<td><a href="https://www.genome.jp/dbget-bin/www_bget?map04152">https://www.genome.jp/dbget-bin/www_bget?map04152</a></td>
</tr>
<tr>
<td>cAMP signaling pathway</td>
<td>KEGG</td>
<td><a href="https://www.genome.jp/dbget-bin/www_bget?map04024">https://www.genome.jp/dbget-bin/www_bget?map04024</a></td>
</tr>
<tr>
<td>FoxO signaling pathway</td>
<td>KEGG</td>
<td><a href="https://www.genome.jp/dbget-bin/www_bget?map04068">https://www.genome.jp/dbget-bin/www_bget?map04068</a></td>
</tr>
<tr>
<td>MAPK signaling pathway</td>
<td>KEGG</td>
<td><a href="https://www.genome.jp/dbget-bin/www_bget?map04010">https://www.genome.jp/dbget-bin/www_bget?map04010</a></td>
</tr>
<tr>
<td>mTOR signaling pathway</td>
<td>KEGG</td>
<td><a href="https://www.genome.jp/dbget-bin/www_bget?map04150">https://www.genome.jp/dbget-bin/www_bget?map04150</a></td>
</tr>
<tr>
<td>cAMP signaling pathway</td>
<td>KEGG</td>
<td><a href="https://www.kegg.jp/dbget-bin/www_bget?map04024">https://www.kegg.jp/dbget-bin/www_bget?map04024</a></td>
</tr>
<tr>
<td>Fluorescent lights</td>
<td>IKEA</td>
<td>N/A</td>
</tr>
<tr>
<td>Spectrophotometer</td>
<td>Synergy H1</td>
<td>N/A</td>
</tr>
<tr>
<td>Laminar Flow Cabinet</td>
<td>NuAire</td>
<td>N/A</td>
</tr>
<tr>
<td>Autoclave</td>
<td>Prestige</td>
<td>N/A</td>
</tr>
<tr>
<td>pH meter</td>
<td>Mettler Toledo</td>
<td>N/A</td>
</tr>
<tr>
<td>PHYTO-PAM-II</td>
<td>Heinz Walz GmbH</td>
<td>N/A</td>
</tr>
<tr>
<td>Super Electroporator Type II</td>
<td>NEPA21</td>
<td>N/A</td>
</tr>
<tr>
<td>QIAGEN Plasmid Mini Kit</td>
<td>QIAGEN</td>
<td>N/A</td>
</tr>
<tr>
<td>Illumina MiSeq and HiSeq 2500 system</td>
<td>Illumina</td>
<td>N/A</td>
</tr>
<tr>
<td>Light microscope</td>
<td>Olympus BX53</td>
<td>N/A</td>
</tr>
<tr>
<td>Hemocytometer</td>
<td>Nexcelom Bioscience Western Europe</td>
<td>N/A</td>
</tr>
<tr>
<td>Plant growth chamber</td>
<td>CLF Plant Climatics GmbH</td>
<td>N/A</td>
</tr>
<tr>
<td>Glass microscope slides</td>
<td>Thermo Fisher Scientific Inc.</td>
<td>N/A</td>
</tr>
<tr>
<td>Bioanalyser</td>
<td>Agilent Technologies</td>
<td>Agilent 2100</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>For efficient extraction of total RNAs from diatom samples, plastic applicator sticks (with a length of appropriately 14 cm), which fit the 1.5 mL Eppendorf microtubes, are used to grind and break down frozen cell pellets into powders.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparing and cultivating two different morphotype cultures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 <b>weeks</b>
</div>
<ol>
<li>Grow diatoms on solid plates
<ol type="a">
<li>Make up a fresh medium according to the f/2+Si medium recipe (<a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Recipe of f/2 +Si medium<sup>∗</sup>
</caption>
<thead>
<tr>
<th>Components</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaNO<sub>3</sub>
</td>
<td>75 mg/L</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>·H<sub>2</sub>O</td>
<td>5 mg/L</td>
</tr>
<tr>
<td>Na<sub>2</sub>SiO<sub>3</sub>·9H<sub>2</sub>O</td>
<td>30 mg/L</td>
</tr>
<tr>
<td>Na<sub>2</sub>·EDTA</td>
<td>4.36 mg/L</td>
</tr>
<tr>
<td>FeCl<sub>3</sub>·6H<sub>2</sub>O</td>
<td>3.15 mg/L</td>
</tr>
<tr>
<td>CuSO<sub>4</sub>·5H<sub>2</sub>O</td>
<td>0.01 mg/L</td>
</tr>
<tr>
<td>ZnSO<sub>4</sub>·7H<sub>2</sub>O</td>
<td>0.022 mg/L</td>
</tr>
<tr>
<td>CoCl<sub>2</sub>·6H<sub>2</sub>O</td>
<td>0.01 mg/L</td>
</tr>
<tr>
<td>MnCl<sub>2</sub>·4H<sub>2</sub>O</td>
<td>0.18 mg/L</td>
</tr>
<tr>
<td>Na<sub>2</sub>MoO<sub>4</sub>·<sub>2</sub>H<sub>2</sub>O</td>
<td>0.006 mg/L</td>
</tr>
<tr>
<td>Thiamine hydrochloride</td>
<td>0.1 mg/L</td>
</tr>
<tr>
<td>Biotin</td>
<td>0.5 μg/L</td>
</tr>
<tr>
<td>Vitamin B<sub>12</sub>
</td>
<td>0.5 μg/L</td>
</tr>
</tbody>
</table>
<p class="table-legend"><sup>∗</sup>Medium prepared using filtered natural seawater at pH=8.0. For solid plates, agar (1.2%, w/v) will be added to the liquid medium before autoclave.</p>
</li>
<li>Autoclave for 25 min (at 121°C with a pressure of 15 psi or 1.03 × 10<sup>5</sup> Pa) and cool the medium-agar mix to 55°C.</li>
<li>Add any antibiotics if needed (for example, to prepare agar plates with 100 μg/mL zeocin for growing zeocin-resistant diatom transformants) and pour the plates with 20 mL for each plate in a 100 mm diameter plate, and then cool down the plates to room temperature (22 ± 2°C) and dry the plates in the safety cabinet (laminar flow hood) for one hour after the agar solidifies.</li>
<li>Subculture diatoms on the freshly prepared solid plates by distributing 200–400 μL liquid seed cultures, which are cultivated under continuous cool white fluorescent lights of 50 μmol photons m<sup>−2</sup> s<sup>−1</sup> at 22 ± 2°C and maintained in exponential growth phase on each plate.</li>
<li>Grow the solid cultures under continuous cool white fluorescent lights of 50 μmol photons m<sup>−2</sup> s<sup>−1</sup> at 22 ± 2°C until observing single colonies and oval cells as the major morphotype. Freshly prepared samples should be used for cell observation to avoid morphotype changes during storage.</li>
</ol>
</li>
<li>Prepare liquid diatom cultures
<ol type="a">
<li>Make up a fresh medium according to the f/2+Si medium recipe (<a href="#tbl1">Table 1</a>).</li>
<li>Autoclave for 25 min, cool the medium to room temperature (22 ± 2°C), and add any needed antibiotics.</li>
<li>Subculture diatoms by inoculating seed cultures into the freshly prepared liquid medium (10%–20%, v/v) in 250 mL flasks with a total culture volume of 50 mL.</li>
<li>Grow the liquid cultures under continuous cool white fluorescent lights of 50 μmol photons m<sup>−2</sup> s<sup>−1</sup> at 22 ± 2°C to the exponential growth phase and confirm fusiform cells as the major morphotype in the liquid cultures. Freshly prepared samples should be used for cell observation to avoid morphotype changes during storage.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Transcriptome analysis of two different morphotype cultures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> hours to days</div>
<p>High-quality RNA should be extracted from cells for both solid culture (oval &gt; 75%) and liquid culture (fusiform &gt; 90%) to facilitate the downstream RNA sequencing (RNA-seq) and subsequent transcriptome analysis.</p>
<ol start="3">
<li>Extract RNA from two different morphotype cultures for RNA-seq
<ol type="a">
<li>Collect diatom samples from fresh cultures. For both solid culture with oval cells &gt; 75% of the population, collect cells from plates and resuspend them in deionized water; for liquid culture with fusiform cell &gt; 90% of the population, take 30–50 mL cultures for each sample (approximately 10–20 mg dry weight for next steps 1b–1d). (<a href="#fig1">Figure 1</a>)
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/902-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. A workflow to generate liquid culture (fusiform cells in dominance) and solid culture (oval cell in dominance)</div>
<p>There are usually three morphotypes in <i>P. tricornutum</i> cultures, i.e., fusiform, oval and triradiate cells. The fusiform cells are dominant in liquid culture under non-stress conditions, while the oval cells dominate surface colonization in solid culture. The microscopic images are representative to indicate the morphotype difference between samples.</p>
</figcaption>
</figure>
</li>
<li>Pellet cells in 50 mL tube by centrifugation at 4000 × <i>g</i> at 20°C for 5 min, use the pipette to remove the supernatant and wash the cell pellets using deionized water; repeat the step for 2–3 times, and then resuspend cell pellets in 1 mL of deionized water.</li>
<li>Transfer the resuspended cells to a centrifugation tube, pellet cells by centrifugation at 10000 × <i>g</i> at 20°C for 1 min and use the pipette to remove the supernatant.</li>
<li>Freeze cell pellets in liquid nitrogen or dry ice and grind the cell pellets in 1.5 mL microtube into powder with a clean plastic applicator stick by repeating the freeze and thaw cycle five to ten times.</li>
<li>Extract total RNA by using the MagMAX-96 Total RNA Isolation kit (AM1830, Thermo Fisher Scientific Inc.) with all related reagents according to the manufacturer’s instruction, including the steps of lysis and binding, initial nucleic acid purification, DNase treatment, and final RNA cleanup.</li>
<li>Assess the quality of the extracted RNA samples using a Bioanalyser (Agilent 2100; Agilent Technologies, Santa Clara, CA, USA) to measure RNA Integrity Numbers (RIN). Quantify RNA samples with specific concentration, purity, and integrity values by using the Qubit RNA assay kit. A minimum RIN score of 8 is preferred for proceeding to RNA-seq.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The diatom cells must be ground thoroughly to break down cell walls by repeating the freeze and thaw cycle and also kept at low temperature (ice bath) conditions during the whole process to avoid any RNA degradation. In addition, the Lysis/Binding Solution for processing diatom samples must be freshly prepared and used on the same day.</div>
<ol start="4">
<li>Perform RNA-seq on RNA isolated from the WT <i>P. tricornutum</i> strain in liquid and on solid media
<ol type="a">
<li>Take the samples of total RNA with high quality after quality control using a bioanalyzer (Agilent 2100; Agilent Technologies, Santa Clara, CA, USA). The use of two or more technical replicates is recommended for each biological sample.</li>
<li>Prepare mRNA library using Illumina TruSeq V2 RNA sample Prep Kit (San Diego, CA) according to the manufacturer’s protocol (<a href="https://www.illumina.com/products/by-type/sequencing-kits/library-prep-kits/truseq-rna-v2.html">https://www.illumina.com/products/by-type/sequencing-kits/library-prep-kits/truseq-rna-v2.html</a>).</li>
<li>Purify polyadenylate-containing mRNA by oligo(dT) magnetic beads from 1.0 μg of total RNA sample, fragment it, and reversely transcribe the first-strand cDNA using random primers.</li>
<li>Then use the first-strand cDNA to synthesize second-strand cDNA by using DNA polymerase I and ribonuclease H. Treat the fragments after second-strand cDNA synthesis with end-repair, the addition of a single A base, and TruSeq adapter-index ligation to cDNA libraries.</li>
<li>Amplify the fragment samples by PCR for 12 cycles to produce fragments with a size of 350–400 bp, including adapters, to create the final cDNA library for the next step 2f.</li>
<li>Confirm fragment sizes and purity of the libraries with positive control by using a Bioanalyzer 2100 (Agilent 2100; Agilent Technologies, Santa Clara, CA, USA).</li>
<li>Determine the quantities of the libraries required for RNA-seq by real-time qPCR using a KAPA library quantification kit (Kapa Biosystems, Roche) for the Illumina platform.</li>
<li>Sequence the enriched cDNA libraries using the Miseq (Illumina) or HiSeq 2500 system (Illumina) platform with 2 × 150–base pair read length.</li>
<li>Process data using the standard Illumina processing pipeline to segregate each multiplexed sample’s reads.</li>
<li>Take the RNA-seq data for further analysis and deposit the raw data to the National Center for Biotechnology Information (NCBI) database.</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Preparing materials and reagents for diatom transformation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 <b>weeks to months</b>
</div>
<ol start="5">
<li>Identify a group of 61 signaling genes based on a total of 2,468 up-regulated genes in the solid wild type when compared with the liquid wild type cells using the KEGG Automatic Annotation Server (<a href="http://www.genome.jp/tools/kaas/">http://www.genome.jp/tools/kaas/</a>) from 44 potential signaling pathways based on KEGG database with gene annotations of the model diatom <i>P. tricornutum.</i>
</li>
<li>Select 14 cDNAs encoding candidate signaling genes including several GPCR signaling genes from the 61 signaling genes with a focus on GPCR signaling genes as GPCRs are on the top level of signal transduction hierarchy.</li>
<li>Send the design request of the nucleotide sequence to the company to synthesize and clone these signaling genes in the shuttle vector pPha-NR with a nitrate reductase promoter.</li>
<li>Receive the host <i>E. coli</i> strains with the designed plasmids and extract the plasmids for diatom transformation experiments.</li>
</ol>
<h3 id="sec3.4">Transformation and selection of targeted morphotype strains</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 <b>weeks</b>
</div>
<p>For the transformation of different gene constructs to the diatom <i>P. tricornutum</i>, the electroporation protocol (<a href="#bib5">Miyahara et al., 2013</a>) was used with slight modifications for ease of implementation and optimization. A NEPA21 electroporator (Nepa Gene Co. Ltd.) was used for the electroporation experiment.</p>
<ol start="9">
<li>Transform different gene constructs to the diatom <i>P. tricornutum</i>
<ol type="a">
<li>Collect diatom samples from the fresh liquid culture at the exponential growth phase (OD<sub>600</sub>=0.3–0.5). Take 20–30 mL cultures for each sample.</li>
<li>Pellet cells in 50 mL tube by centrifugation at 700 × <i>g</i> at 20°C for 4 min, use the pipette to remove the supernatant and wash the cell pellets using 0.77 M mannitol; repeat the step for 2–3 times, and then resuspend cell pellets in 1 mL of 0.77 M mannitol.</li>
<li>Linearize 3–5 μg of each plasmid construct by <i>Nde</i>I for 4 h at 37°C in a total reaction volume of 50 μL.</li>
<li>Use DNA Clean &amp; Concentrator<sup>TM</sup>-25 to recover linearized plasmids by following the manufacturer’s instructions.</li>
<li>Mix 3–5 μg of linearized plasmids with an aliquot of 0.15 mL cell suspension on ice and then transfer the mixture to an electroporation cuvette with a 0.2 cm gap on ice.</li>
<li>Determine the impedance of the mixture in the electroporation cuvette and adjust its impedance by using 0.77 M mannitol solution to a value between 0.4 and 0.6 kΩ.</li>
<li>Set the electroporator and do the transformation with the following parameters: square electric poring pulses at 300V (pulse duration, 5 ms; 8 pulses; interval 50 ms; 10% decay rate), and transferring pulses at 8V (pulse duration, 50 ms; 5 pulses; interval 50 ms; 40% decay rate). Place the cuvettes with cell suspensions on ice immediately.</li>
<li>Transfer the cell solution from ice bath within 5 min into 4 mL of f/2+Si medium in a 15 mL tube and incubate the sample at 20°C overnight or for 16–20 h under a photon flux density of 30 μmol photons m<sup>−2</sup> s<sup>−1</sup>.</li>
<li>Pellet cells by centrifugation at 700 × <i>g</i> at 20°C for 4 min, use the pipette to remove the supernatant and resuspend the cell pellets using 0.2 mL of f/2+Si medium.</li>
<li>Cultivate the transformed cells on f/2+Si agar plates containing 1.2% agar and 100 μg/mL of zeocin for antibiotic resistance selection under a photon flux density of 30 μmol photons m<sup>−2</sup> s<sup>−1</sup>. For each transformation, a minimum of two replicate plates is used.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The diatom cells must be collected from the exponential growth phase for highly efficient transformation. Moreover, the impedance of mixed cells and plasmid DNA solution in the cuvette for electroporation must be adjusted to get a tradeoff between cell damage and DNA delivery efficiency.</div>
<ol start="10">
<li>Select successful transformants with targeted morphotypes
<ol type="a">
<li>Check the presence of single colonies on the selective agar plates after ten days to 2 weeks.</li>
<li>Pick up single colonies and subculture them into liquid selective medium, respectively, for 1–2 weeks.</li>
<li>Pellet cells from 1–2 mL liquid cultures by centrifugation at 10000 × <i>g</i> for 1 min, use the pipette to remove the supernatant and wash the cell pellets using deionized water; repeat the step two times, and then resuspend cell pellets in 50 μL 5% Chelex-100.</li>
<li>Vortex for 10 s and incubate at 98°C–100°C for 10 min.</li>
<li>Cool down on the ice for 1 min and vortex for 10 s.</li>
<li>Spin at 10000 × <i>g</i> for 1 min and use 1 μL of supernatant for each of 20 μL PCR reaction (<a href="#tbl2">Table 2</a>).
<table id="tbl2">
<caption>Table 2. Conditions for amplification PCR</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Polymerase activation</td>
<td>95°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>20 s</td>
<td>35</td>
</tr>
<tr>
<td>Annealing</td>
<td>59°C</td>
<td>10 s</td>
<td>35</td>
</tr>
<tr>
<td>Extension</td>
<td>70°C</td>
<td>15 s</td>
<td>35</td>
</tr>
<tr>
<td>Final Extension</td>
<td>70°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C–8°C</td>
<td>forever</td>
<td>1</td>
</tr>
</tbody>
</table>
</li>
<li>Screen and verify the incorporation of plasmid DNA into the genome by checking the PCR products with corresponding primers (primers of GPCR1A and GPCR4, as indicated in the Key Resource Table).</li>
<li>Check the morphotypes of selected transformants under light microscopy and confirm the successful transformants are with targeted morphotypes. (<a href="#fig2">Figure 2</a>)
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/902-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. A workflow to create transformants of <i>P. tricornutum</i>
</div>
<p>(A) The major steps involved in using the electroporation approach to create transformants.</p>
<p>(B) The microscopic images of different strains with different morphotypes, i.e., the wild type, GPCR1A transformants, GPCR4 transformants, and transformants with no morphotype change. G-protein coupled receptors (GPCRs), also known as seven-transmembrane domain receptors, mediate vital signaling processes and physiological functions in most living organisms (<a href="#bib6">Shaw et al., 2019</a>). The microscopic images are representative to indicate the morphotype difference among samples.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Preparing samples for characterization of selected transformants</h3>
<div class="timing">
<span class="timing-title">Timing:</span> days to weeks</div>
<ol start="11">
<li>Prepare diatom samples for light microscopy and scanning electron microscopy (SEM) experiments
<ol type="a">
<li>Cultivate and maintain seed cultures of both the wild type and the transformants under continuous cool white fluorescent lights of 50 μmol photons m<sup>−2</sup> s<sup>−1</sup> at 22 ± 2°C.</li>
<li>Subculture diatoms by inoculating seed cultures into the freshly prepared liquid medium (10%–20%, v/v) in 250 mL flasks with a total culture volume of 50 mL.</li>
<li>Grow the liquid cultures under continuous cool white fluorescent lights of 50 μmol photons m<sup>−2</sup> s<sup>−1</sup> at 22 ± 2°C to exponential growth phase (optical density OD<sub>600</sub> at 600 nm is between 0.1 and 0.2 in flasks) and confirm fusiform cells as the major morphotype in the wild type and oval cells as the major morphotype in the transformant liquid cultures.</li>
<li>For the light microscopy experiment, dilute the samples to the desired cell density (between 10<sup>5</sup> and 10<sup>6</sup> cells/mL) and make a glass slide for microscopy observation directly; for the SEM experiment, dilute the samples to the desired cell density (between 10<sup>5</sup> and 10<sup>6</sup> cells/mL) for SEM observation and then remove salts by washing the samples with deionized water.</li>
</ol>
</li>
<li>Prepare diatom samples for light stress, UV stress, and surface colonization experiments
<ol type="a">
<li>Cultivate and maintain seed cultures of both the wild type and the transformants under continuous cool white fluorescent lights of 50 μmol photons m<sup>−2</sup> s<sup>−1</sup> at 22 ± 2°C.</li>
<li>Subculture diatoms by inoculating seed cultures into the freshly prepared liquid medium (10%–20%, v/v) in 250 mL flasks with a total culture volume of 50 mL.</li>
<li>Grow the liquid cultures under continuous cool white fluorescent lights of 50 μmol photons m<sup>−2</sup> s<sup>−1</sup> at 22 ± 2°C to exponential growth phase and confirm fusiform cells as the major morphotype in the wild type and oval cells as the major morphotype in the transformant liquid cultures.</li>
<li>Dilute the samples to the desired cell density (between 1×10<sup>5</sup> and 5×10<sup>5</sup> cells/mL) for each of these experiments. After dilution, samples need to be kept in the dark for one hour for the light stress experiment only.</li>
</ol>
</li>
</ol>
<h3 id="sec3.6">Characterization of selected transformants</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 <b>days</b>
</div>
<p>The morphotypes of selected transformants are determined with SEM imaging. Moreover, the transformants are characterized by the capability to respond to light stress, UV stress, and surface colonization compared to the wild type as a control.</p>
<ol start="13">
<li>SEM imaging
<ol type="a">
<li>Transfer the freshly diluted (using deionized water) samples (1–1.5 mL for each) from the previous preparation step (step 1 in the section of <a href="#sec3.5">preparing samples for characterization of selected transformants</a>) to microcentrifugation tubes.</li>
<li>Dehydrate the samples by centrifuging at 5000 × <i>g</i> for 1 min, removing the supernatant with a pipette, and washing the cell pellets using 20%, 40%, 60%, 80%, and 100% ethanol gradually. A centrifugation step at 5000 × <i>g</i> for 1 min is used between washes.</li>
<li>Vortex the cell samples in 100% ethanol.</li>
<li>Deposit liquid solutions on Whatman track-etched membranes (VWR 514-0049).</li>
<li>Dry the membranes and then sputter deposit approximately 2 nm of Au onto the membranes in a vacuum coater.</li>
<li>Perform SEM imaging using a Thermo/FEI Quanta 450 Scanning Electron Microscope (or equivalent) with 5 kV accelerating voltage and a spot size (beam current) of 2 in high vacuum using an ET (Everhart-Thornley) secondary electron detector.</li>
<li>Observe high-resolution images of individual diatom cells using SEM.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The diatom cells must be dehydrated gradually to keep the natural structure of cells.</div>
<ol start="14">
<li>Light stress experiment
<ol type="a">
<li>Take the freshly prepared samples from the previous preparation step (step 2 in the section of <a href="#sec3.5">preparing samples for characterization of selected transformants</a>).</li>
<li>Dilute the samples using f/2 + Si medium to a cell density of approximately OD<sub>600</sub>=0.05</li>
<li>Keep the samples in the dark for 1 h and vortex for 10 s after dark incubation.</li>
<li>Set the PHYTO-PAM-II (Heinz Walz GmbH) instrument with constant white light stress of 220 μmol photons m<sup>−2</sup> s<sup>−1</sup> for 10–15 min.</li>
<li>Determine the effective quantum yields under different light intensities of 0, 8, 16, 58, 74, 99, 147, 236, 415, and 588 μmol photons m<sup>−2</sup> s<sup>−1</sup> in a consecutive mode.</li>
<li>Compare the difference between the transformants and the wild type in terms of quantum yields.</li>
</ol>
</li>
<li>UV stress experiment
<ol type="a">
<li>Take the freshly prepared samples from the previous preparation step (step 2 in the section of <a href="#sec3.5">preparing samples for characterization of selected transformants</a>).</li>
<li>Dilute the samples to a cell density of 1–3 × 10<sup>6</sup> cells/mL (appropriately at OD<sub>600</sub> of 0.1 to 0.2) using f/2 + Si medium and transfer 3 mL of each sample to a 100 mm diameter plate</li>
<li>Expose the samples in plates to UV light using a UV crosslinker with a UV exposure time of 60 s and energy of 0.1 J/cm<sup>2</sup>
</li>
<li>Incubate samples under a low light intensity of 30 μmol photons m<sup>−2</sup> s<sup>−1</sup> at room temperature (22 ± 2°C) for over two days.</li>
<li>Mix the samples thoroughly by pipetting the solution and count cells to get cell density after UV treatment.</li>
<li>Calculate the survival rate by using the initial cell density and the cell density after UV treatment and compare the difference between the transformants and the wild type</li>
</ol>
</li>
<li>Surface colonization to determine the adhesive difference (<a href="#bib7">Stanley and Callow, 2007</a>) between the transformants and the wild type (<a href="#fig3">Figure 3</a>)
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/902-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. An overview of surface colonization process on glass slides</div>
<p>The microscopic images show the morphotype difference among three different strains: the wild type, GPCR1A transformants, and GPCR4 transformants. The microscopic images are representative to indicate the morphotype difference among samples.</p>
</figcaption>
</figure>
<ol type="a">
<li>Rinse glass microscope slides with deionized water, then soak the slides in 1M HCl for 24 h.</li>
<li>Rinse the slides using deionized water and dry the slides.</li>
<li>Take the freshly prepared samples from the previous preparation step (step 2 in the section of <a href="#sec3.5">preparing samples for characterization of selected transformants</a>) and dilute the samples to a cell density of 500,000 cells/mL (approximately at OD<sub>600</sub> of 0.03).</li>
<li>Place each slide to a 100 mm diameter dish plate.</li>
<li>Add 15 mL of diluted samples to each dish plate.</li>
<li>Incubate the samples for 72 h at 22 ± 2°C in growth chambers.</li>
<li>Shake and rinse the slides gently to remove unattached cells on a shaker at 150 rpm.</li>
<li>Count cells according to microscope images from 9 randomly sampled views.</li>
<li>Compare the adhesive difference between the transformants and the wild type.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The transcriptome analysis protocol will identify differentially expressed genes (DEGs) in the solid WT compared with liquid WT cells and the identification of overrepresented Gene Ontology (GO) terms for the DEGs from WT. The analysis will also allow us to identify the up-regulated genes involved in potential signaling pathways and predict their protein-protein interaction network. The ultimate expected outcome from the transcriptome analysis is to identify candidate GPCR signaling genes for transformation.</p>
<p>The transformation protocol will result in GPCR transformants. The selection of the transformants expressing the GPCR genes will help us sort and select for transformants that can shift cell morphotype from fusiform to oval forms when observed and compared to WT, as shown in <a href="#fig2">Figure 2</a>.</p>
<p>The characterization of the selected GPCR transformants will show whether they can resist light and UV stress compared to WT. The surface colonization protocol will determine these transformants' capability to attach and colonize to the surface of glass slides compared to WT, as shown in <a href="#fig3">Figure 3</a>.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>For the section of Transcriptome analysis of two different morphotype cultures, the CLC Genomics Workbench 11.0.1 (<a href="https://digitalinsights.qiagen.com">https://digitalinsights.qiagen.com</a>) calculates the normalized Reads Per Kilobase per Million (RPKM) of genes expressed in the samples. The tool for obtaining the RPKM reads, RNA-seq Analysis, requires importing the fasta files of the sample-sequenced transcripts. A summary of the major steps is as follows:</p>
<ul>
<li>Import the fasta files using the “Import tracks” tool. Make sure that “Batch mode” is enabled when selecting the files to be imported.</li>
<li>To run RNA-seq Analysis, go to Toolbox, Transcriptomic Analysis, then RNA-seq Analysis. Then there is a dialog box with steps to follow.</li>
<li>Select the sequencing files imported previously and the reference annotation, <i>P. tricornutum</i> (2013-07-EBI-Phatr3), ASM15095v2, downloaded from EnsemblProtists ( <a href="http://protists.ensembl.org/index.html">http://protists.ensembl.org/index.html</a>).</li>
</ul>
<p>The detailed steps are in the manual of the CLC genomics (<a href="http://resources.qiagenbioinformatics.com/manuals/clcgenomicsworkbench">http://resources.qiagenbioinformatics.com/manuals/clcgenomicsworkbench</a>). Using CLC genomics, it is optional to process RNAseq data of different samples to obtain differentially expressed genes (DEGs). Alternatively, you can use the DESeq2 R package (<a href="#bib4">Love et al., 2014</a>) for obtaining the DEGs.</p>
<p>To identify cellular functions enriched in the DEGs, use ClueGo plug-in (V2.5.0) (<a href="#bib1">Bindea et al., 2009</a>) in Cytoscape v3.6.0 ( <a href="https://cytoscape.org/">https://cytoscape.org/</a>). To do this:</p>
<ul>
<li>In a Cytoscape session, install the ClueGO plug-in from App menubar. Select ClueGo App to start functional analysis and Gene Set Enrichment Analysis (GSEA) of the DEGs.</li>
<li>In the control panel window, select the organism, <i>P. tricornutum,</i> and paste the list of the DEGs identifiers.</li>
<li>Select the visual style, <i>significance</i>, the ontologies type, <i>biological processes</i>, and enable “show only pathways with p value less than or equal to 0.05”.</li>
<li>Click “<i>Start</i>” to start the analysis. ClueGO determines the statistically significant Gene Ontology (GO) terms that are overrepresented in the DEGs. ClueGO calculates the p values using a two-sided hypergeometric test and corrects them using the Benjamini-Hochberg false discovery rate (FDR).</li>
</ul>
<p>When ClueGO finishes the analysis, the network pops up, visualizing the enriched GO terms represented in nodes. Each biological process has a specific node-color. The node size is proportional to the number of genes in a particular GO term.</p>
<ul>
<li>Photosynthetic efficiency</li>
</ul>
<p>The WT and transformants’ photosynthetic performance can be evaluated using the chlorophyll fluorometer PHYTO-PAM-II (Heinz Walz GmbH). The effective quantum yields (QY) of photosystem II (PSII) is used to assess the capability of the strain to respond to immediate white light stress (220 μmol photons m<sup>−2</sup> s<sup>−1</sup> over 10 min). The chlorophyll fluorometer uses <a href="#fd1">equation (1)</a> to calculate the maximum photosynthetic efficiency of PSII (<a href="#tbl3">Table 3</a>), where F<sub>0</sub> is the ground fluorescence in the dark- or low-light adapted cells and F<sub>m</sub> represents the maximal fluorescence.</p>
<table id="tbl3">
<caption>Table 3. Example of effective PSII quantum yields measured with a chlorophyll fluorometer (PAM)</caption>
<thead>
<tr>
<th>Light intensity</th>
<th>WT liquid sample 1</th>
<th>WT liquid sample 2</th>
<th>WT liquid sample 3</th>
<th>Average</th>
<th>Standard error</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>0.462</td>
<td>0.487</td>
<td>0.532</td>
<td>0.49</td>
<td>0.02</td>
</tr>
<tr>
<td>8</td>
<td>0.499</td>
<td>0.503</td>
<td>0.517</td>
<td>0.51</td>
<td>0.01</td>
</tr>
<tr>
<td>16</td>
<td>0.476</td>
<td>0.474</td>
<td>0.485</td>
<td>0.48</td>
<td>0.00</td>
</tr>
<tr>
<td>58</td>
<td>0.346</td>
<td>0.341</td>
<td>0.36</td>
<td>0.35</td>
<td>0.01</td>
</tr>
<tr>
<td>74</td>
<td>0.313</td>
<td>0.31</td>
<td>0.343</td>
<td>0.32</td>
<td>0.01</td>
</tr>
<tr>
<td>99</td>
<td>0.26</td>
<td>0.257</td>
<td>0.298</td>
<td>0.27</td>
<td>0.01</td>
</tr>
<tr>
<td>147</td>
<td>0.197</td>
<td>0.192</td>
<td>0.23</td>
<td>0.21</td>
<td>0.01</td>
</tr>
<tr>
<td>236</td>
<td>0.171</td>
<td>0.127</td>
<td>0.155</td>
<td>0.15</td>
<td>0.01</td>
</tr>
<tr>
<td>415</td>
<td>0.078</td>
<td>0.075</td>
<td>0.094</td>
<td>0.08</td>
<td>0.01</td>
</tr>
<tr>
<td>588</td>
<td>0.055</td>
<td>0.054</td>
<td>0.068</td>
<td>0.06</td>
<td>0.00</td>
</tr>
</tbody>
</table>
<p class="chem">Fv/Fm = (Fm − Fo)/Fm     (Equation 1)</p>
<ul>
<li>In Excel, calculate the average quantum yield of the triplicates using the function <i>AVERAGE.</i> Calculate the standard error using the function <i>STDEV(range)/SQRT(3).</i>
</li>
<li>Plot the effective QY of PSII versus the photosynthetically active radiation (PAR, μmol m<sup>−2</sup> s<sup>−1</sup>) and compare light stress response.</li>
</ul>
<p>To identify DEGs in the various samples during surface colonization, use the DESeq2 R package (<a href="#bib4">Love et al., 2014</a>). Using the negative binomial distribution, the DESeq2 method provides a differential analysis of RNA-seq count data. The package and reference manual are available at <a href="http://www.bioconductor.org/packages/release/bioc/html/DESeq2.html">http://www.bioconductor.org/packages/release/bioc/html/DESeq2.html</a>.</p>
<ul>
<li>To reduce the memory size of the data object, pre-filter each sample by removing the raw read counts &lt;10.</li>
<li>Using the single <i>DESeq</i> function implemented in DESeq2, perform differential gene expression analysis after normalizing the total read count. Note that the <i>estimateSizeFactors</i> function occurs within the <i>DESeq</i> function. By default, the <i>geoMeans</i> argument takes the geometric mean of the read counts across all samples, thereby scaling the read counts by a reference sample.</li>
<li>Use the <i>results</i> function to generate a table of the log2 fold changes and p-values.</li>
<li>The <i>plotMA</i> function implemented in DESeq2 can be used to visualize the log2 fold changes against the mean of normalized counts.</li>
</ul>
<p>Unless otherwise stated, a Student t-test should be performed to assess the statistical significance of physiological differences between samples. A p-value of less than 0.05 is used to reject the null hypothesis for comparisons between two groups.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While this protocol aims to identify key genes regulating the surface colonization process in the diatom <i>P. tricornutum</i>, it is possible that some regulatory genes are still not identified due to genome annotation limitations as well as the relatively low expression levels of some genes. Additionally, it is important to note that this protocol is primarily developed to manipulate the morphology shift of the model diatom <i>P. tricornutum</i> and the related surface colonization process. Therefore, we recommend following up this versatile protocol with an optimization workflow involving verifications of genome annotations as well as diatom RNA extraction methods to enable the successful application to other diatom species.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low RNA yield and RNA degradation (referring to the step: Transcriptome analysis of two different morphotype cultures).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>When working with RNA instruments, DNase and RNase-free materials should be used, including pipette tips, tubes, and corresponding solutions. The work area/bench should be cleaned with an RNase inactivating solution before starting with the RNA extraction.</p>
<p>RNase activity is reduced with temperature; thus, unless otherwise stated, work with samples on ice or at 4°C at all times, and set the needed devices/centrifuges at 4°C.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Low transformation efficiency (referring to the step: Transformation and selection of targeted morphotype strains).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>It is crucial that the electroporation is performed on the diatom cells at their exponential growth phase OD<sub>700</sub> =0.2–0.4, 5.9 × 10<sup>6</sup> cells (detailed protocol). A slight modification was introduced to the protocol (<a href="#bib2">Fu et al., 2017</a>), adding a washing step and making sure the cells are active before the electroporation.</p>
<p>Ensure using good quality purified DNA. It is recommended to store plasmids at −20°C for longer periods and minimize freeze-thaw cycles to avoid DNA degradation in the longer-term. Linearize the plasmid with the corresponding restriction enzymes prior to the electroporation. A validation step can be added before the transformation, consisting of running gel electrophoresis for the obtained linear fragment in order to validate the expected fragment size.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Integration (non-targeted integration) of the reporter genes in the diatom genome (referring to the step: Transformation and selection of targeted morphotype strains).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>To examine the integration of the genes in the <i>P. tricornutum</i>, it is essential that the transformed cells (colonies) are sub-cultured 2–3 times in media with 100 μg/mL of zeocin for antibiotic resistance selection. After isolating genomic DNA from transformed cells, PCR verification is conducted on the extracted DNA to detect the inserted fragment using specific primers and validate the cDNA integration in the diatom genome.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Specificity related to oval/fusiform cells (referring to the step: Preparing and cultivating two different morphotype cultures).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Use microscopy to ensure that majority of the cells (&gt;60%) of required form for corresponding analyses (oval <i>vs</i> fusiform)</p>
<p>To analyze the transcription shifts between solid and liquid cultures, RNA sequencing is performed on RNA isolated from the wild type <i>P. tricornutum</i> strain grown in liquid and on solid media. Prior to RNA extraction, cells should be observed under a bright-field microscope to ensure that the expected fusiform and oval morphotypes are the dominant morphotypes in liquid and solid cultures, respectively.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Slow growth of <i>P. tricornutum</i> after transformation on zeocin-selective solid plates (referring to the step: Transformation and selection of targeted morphotype strains).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>To obtain single colonies of transformants on zeocin-selective solid plates as expected, it’s important to see diatom growth on plates within two weeks. The plates should be placed under relatively high humid (appropriately 80%) conditions to avoid drying out if cultivation duration is beyond two weeks. Alternatively, 1 mL of sterile f/2+Si medium could be added to the plates per week.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bindea, G., Mlecnik, B., Hackl, H., Charoentong, P., Tosolini, M., Kirilovsky, A., Fridman, W.H., Pages, F., Trajanoski, Z., and Galon, J. (2009). ClueGO: a Cytoscape plug-in to decipher functionally grouped gene ontology and pathway annotation networks. Bioinformatics <i>25</i>, 1091-1093. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00423-8/sref1">View at publisher</a></p>
<p id="bib2">Fu, W.Q., Chaiboonchoe, A., Khraiwesh, B., Sultana, M., Jaiswal, A., Jijakli, K., Nelson, D.R., Al-Hrout, A., Baig, B., Amin, A., and Salehi-Ashtiani, K. (2017). Intracellular spectral recompositioning of light enhances algal photosynthetic efficiency. Sci. Adv. <i>3</i>, e1603096. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00423-8/sref2">View at publisher</a></p>
<p id="bib3">Fu, W.Q., Chaiboonchoe, A., Dohai, B., Sultana, M., Baffour, K., Alzahmi, A., Weston, J., Al Khairy, D., Daakour, S., Jaiswal, A., Nelson, D.R., Mystikou, A., Brynjolfsson, S., and Salehi-Ashtiani, K. (2020). GPCR genes as activators of surface colonization pathways in a model marine diatom. iScience <i>23</i>, 101424. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00423-8/sref3">View at publisher</a></p>
<p id="bib4">Love, M.I., Huber, W., and Anders, S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. <i>15</i>, 550. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00423-8/sref4">View at publisher</a></p>
<p id="bib5">Miyahara, M., Aoi, M., Inoue-Kashino, N., Kashino, Y., and Ifuku, K. (2013). Highly efficient transformation of the diatom <i>Phaeodactylum tricornutum</i> by multi-pulse electroporation. Biosci. Biotechnol. Biochem. <i>77</i>, 874-876. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00423-8/sref5">View at publisher</a></p>
<p id="bib6">Shaw, W.M., Yamauchi, H., Mead, J., Gowers, G.F., Bell, D.J., Oling, D., Larsson, N., Wigglesworth, M., Ladds, G., and Ellis, T. (2019). Engineering a model cell for rational tuning of GPCR signaling. Cell <i>177</i>, 782-796.e27. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00423-8/sref6">View at publisher</a></p>
<p id="bib7">Stanley, M.S. and Callow, J.A. (2007). Whole cell adhesion strength of morphotypes and isolates of <i>Phaeodactylum tricornutum</i> (Bacillariophyceae). Eur. J. Phycol. <i>42</i>, 191-197. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00423-8/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Weiqi Fu (<a href="mailto:weiqifu@zju.edu.cn">weiqifu@zju.edu.cn</a>).</p>
<h4>Materials availability</h4>
<p>All sequences of synthesized genes can be found in a recent publication (<a href="#bib3">Fu et al., 2020</a>).</p>
<h4>Data and code availability</h4>
<p>RNA-seq data from this article can be found in the GenBank/NCBI data libraries (GenBank: PRJNA566271). The RNA-seq data have also been deposited in Dryad with a unique identifier (<a href="https://doi.org/10.5061/dryad.ns1rn8ppx">https://doi.org/10.5061/dryad.ns1rn8ppx</a>).</p>
<h3>Acknowledgments</h3>
<p>Financial support for this work was provided by <a href="https://doi.org/10.13039/100012025">New York University Abu Dhabi</a> Faculty Research Funds (AD060) and NYUAD Institute grant (73 71210 CGSB9). W.F. was additionally supported by the Icelandic Technology Development Fund (163922-0611) and the <a href="https://doi.org/10.13039/501100012602">Hundred Talents Program of Zhejiang University</a>.</p>
<h3>Author contributions</h3>
<p>W.F. and K.S.-A. conceived the research and wrote the paper. All authors contributed to the drafting and editing of the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>