<article>
<section>
<h1 id="header">Detecting in-solution conformational changes in viral fusogens using tryptophan-induced fluorescence quenching</h1>
<p><time datetime="2021-12-10">Published: December 10, 2021</time></p>
<p>Vitor Hugo B. Serrão<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> and Jeffrey E. Lee<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Laboratory Medicine and Pathobiology, Temerty Faculty of Medicine, University of Toronto, Toronto, ON M5S 1A8, Canada</p>
<p id="fn1"><sup>2</sup>Present address: Department of Chemistry and Biochemistry, University of California, Santa Cruz, Santa Cruz, CA 95060, USA</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:vbalasco@ucsc.edu">vbalasco@ucsc.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:jeff.lee@utoronto.ca">jeff.lee@utoronto.ca</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100994">10.1016/j.xpro.2021.100994</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Dynamic monitoring of protein conformational changes is necessary to fully understand many biological processes. For example, viral entry and membrane fusion require rearrangement of its viral glycoprotein. We present a step-by-step protocol for site-specific bimane labeling of the influenza-C fusogen to map proximity and conformational movements using tryptophan-induced fluorescence quenching. This protocol is adaptable for other proteins and for protein-protein interaction detection.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref17">Serrão et al., 2021</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detailed protocol for purification of ICV HEF<sub>2</sub> glycoprotein fusion subunit</li>
<li>Preparation and optimization of bimane labeling of ICV HEF<sub>2</sub>
</li>
<li>Detailed steps to measure and analyze tryptophan-induced fluorescence quenching</li>
<li>Protocol can be applied to detect protein conformational changes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1215-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used in a recent publication to determine the pH-dependent conformational changes of the influenza C virus (ICV) hemagglutinin-esterase-fusion subunit (HEF<sub>2</sub>), the glycoprotein necessary for fusion of ICV to a host cell (<a href="#bib17">Serrão et al., 2021</a>). Fluorescence spectroscopy of bimane, a heterocyclic chemical probe, has been used to evaluate conformational changes and protein-protein interactions (<a href="#bib12">Jones Brunette and Farrens, 2014</a>; <a href="#bib15">Mansoor et al., 2010</a>). Bimane can be site-specifically incorporated into protein, and it exhibits a change in fluorescence intensity and emission maxima in response to changes in its physiochemical environment when aromatic residues like tryptophans are in proximity; this results in tryptophan-induced fluorescence quenching (TrIQ).</p>
<p>Here, we present the protocol for ICV HEF<sub>2</sub> protein expression and purification, bimane-labeling, fluorescence quenching measurements and data analysis. Prior to the experiment, prepare all buffers and reagents (steps 1–4) and perform site-directed mutagenesis to incorporate cysteine residues at strategic positions of the protein for bimane labeling (steps 5–11).</p>
<h3 id="sec1.1">Preparation of stock solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<ol>
<li>Antibiotic and isopropylthio-β-galactoside (IPTG) stocks
<ol type="a">
<li>
<b>1 M IPTG:</b> Dissolve 4.76 g IPTG in ∼16 mL MilliQ water and bring the final volume to 20 mL in a 50 mL conical centrifuge tube. Sterile filter the solution through a 0.22-μm polyethersulfone (PES) syringe filter. Pipet 1 mL aliquots into 1.5 mL microcentrifuge tubes and store for up to 1 year at −20°C.</li>
<li>
<b>100 mg/mL ampicillin:</b> Dissolve 2 g ampicillin in ∼16 mL Milli-Q water, bring the final volume to 20 mL in a 50 mL conical centrifuge tube. Filter the solution with a 0.22-μm PES syringe filter. Pipet 1 mL aliquots into 1.5 mL microcentrifuge tubes and store for up to 1 year at −20°C.</li>
</ol>
</li>
<li>Liquid and solid media
<ol type="a">
<li>
<b>LB broth:</b> Add 25 g Luria-Bertani (LB) broth (Miller) granulated powder to 1 L MilliQ water in a 2 L Erlenmeyer flask. Stir until powder is fully dissolved and autoclave at 121°C and 20 psi for 30 min.</li>
<li>
<b>LB-agar Amp plates:</b> Add 2.5 g LB broth (Miller) powder and 1.5 g agar to 100 mL MilliQ water in a 250 mL media bottle. Stir until the powder is fully dissolved. Autoclave at 121°C and 20 psi for 30 min and cool the solution to ∼50°C (comfortable to the touch) prior to adding ampicillin to a final concentration of 100 μg/mL. Pour ∼5 mL of the agar solution into each 60 x 15 mm sterile Petri plate. Allow the agar to solidify before sealing the plates with Parafilm. Plates can be stored at 4°C for up to two weeks.</li>
</ol>
</li>
<li>Buffers
<ol type="a">
<li>
<b>500 mM KH</b><sub><b>2</b></sub><b>PO</b><sub><b>4</b></sub><b>/K</b><sub><b>2</b></sub><b>HPO</b><sub><b>4</b></sub> <b>pH 7.5:</b> Add 68.0 g KH<sub>2</sub>PO<sub>4</sub> or 87.1 g K<sub>2</sub>HPO<sub>4</sub> to 800 mL MilliQ water in a 1 L beaker. Dissolve and add MilliQ water to final volumes of 1 L in a 1 L graduated cylinder. Titrate 500 mM KH<sub>2</sub>PO<sub>4</sub> into 500 mM K<sub>2</sub>HPO<sub>4</sub> while monitoring the pH using a pH-meter to obtain a final pH value of 7.5.</li>
<li>
<b>2.5 M imidazole:</b> Add 170.2 g imidazole to 800 mL MilliQ water in a 1 L beaker. Dissolve and add MilliQ water to a final volume of 1 L in a 1 L graduated cylinder.</li>
<li>
<b>5 M NaCl:</b> Add 292.2 g NaCl to 700 mL MilliQ water in a 1 L beaker. Dissolve and add MilliQ water to a final volume of 1 L in a 1 L graduated cylinder.</li>
<li>
<b>Buffer A- 50 mM KH</b><sub><b>2</b></sub><b>PO</b><sub><b>4</b></sub><b>/K</b><sub><b>2</b></sub><b>HPO</b><sub><b>4</b></sub><b>, pH 7.5, 300 mM NaCl, and 20 mM imidazole:</b> Prepare 1 L of Buffer A. Mix 100 mL 500 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 7.5 stock (step 3a), 0.8 mL 2.5 M imidazole stock, and 60 mL 5 M NaCl stock in a 1 L graduated cylinder. Add MilliQ water to a final volume of 1 L.</li>
<li>
<b>Buffer B- 50 mM KH</b><sub><b>2</b></sub><b>PO</b><sub><b>4</b></sub><b>/K</b><sub><b>2</b></sub><b>HPO</b><sub><b>4</b></sub><b>, pH 7.5, 300 mM NaCl, and 100 mM imidazole:</b> Prepare 500 mL of Buffer B. Mix 50 mL 500 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 7.5 stock (step 3a), 20 mL 2.5 M imidazole stock, and 30 mL 5 M NaCl stock in a 1 L graduated cylinder. Add MilliQ water to a final volume of 0.5 L.</li>
<li>
<b>Buffer C- 50 mM KH</b><sub><b>2</b></sub><b>PO</b><sub><b>4</b></sub><b>/K</b><sub><b>2</b></sub><b>HPO</b><sub><b>4</b></sub><b>, pH 7.5, 300 mM NaCl, and 500 mM imidazole:</b> Prepare 1 L of Buffer C. Mix 100 mL 500 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 7.5 stock (step 3a), 200 mL 2.5 M imidazole stock, and 60 mL 5 M NaCl stock in a 1 L graduated cylinder. Add MilliQ water to a final volume of 1 L.</li>
<li>
<b>Buffer D- 10 mM KH</b><sub><b>2</b></sub><b>PO</b><sub><b>4</b></sub><b>/K</b><sub><b>2</b></sub><b>HPO</b><sub><b>4</b></sub><b>, pH 7.5, and 150 mM NaCl:</b> Prepare 2 L of Buffer D containing. Mix 40 mL 500 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 7.5 stock (step 3a) and 60 mL 5 M NaCl stock in a 2 L graduated cylinder. Add MilliQ water to a final volume of 2 L.</li>
<li>
<b>pH-scan buffers- 10 mM KH</b><sub><b>2</b></sub><b>PO</b><sub><b>4</b></sub><b>/K</b><sub><b>2</b></sub><b>HPO</b><sub><b>4</b></sub> <b>at various pH values and 150 mM NaCl:</b> Prepare 50 mL stocks of pH scan buffers at pH values of 4.5, 5.0, 5.5, 6.0, 6.5, 7.0, and 7.5. Titrate 10 mM KH<sub>2</sub>PO<sub>4</sub> into 10 mM K<sub>2</sub>HPO<sub>4</sub> while monitoring the pH using a pH-meter to obtain the appropriate final pH values. Add 1.5 mL 5 M NaCl to the KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> buffer for a final solution concentration of 150 mM NaCl.</li>
</ol>
</li>
<li>Monobromobimane (mBBr) fluorophore stock</li>
</ol>
<p>100 mM mBBr stock. Dissolve 13.56 mg of mBBr in 0.5 mL 100% acetonitrile in a 1.5 mL microcentrifuge tube and store in the dark at 22°C.</p>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to prepare the mBBr stock fresh for each labeling procedure; it should be kept in the dark prior to use to ensure fluorophore stability.</div>
<h3 id="sec1.2">Preparation of ICV HEF<sub>2</sub> expression plasmids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>Cysteine point mutations were made at positions Q558 (Q558C) and H518 (H518C) in ICV HEF<sub>2</sub> to allow for bimane labeling (<a href="#fig1">Figure 1</a>). The position of the cysteine mutations was strategically designed to monitor the conformational changes of ICV HEF<sub>2</sub> that are required during the formation of the extended intermediate and final post-fusion six-helix bundle (discussed further in the <a href="#expected-outcomes">expected outcomes</a> section) (<a href="#fig2">Figure 2</a>). Additionally, native cysteine residues at C583, C591, and C595 were replaced with serine to ensure incorporation of a single label on ICV HEF<sub>2</sub> (<a href="#bib17">Serrão et al., 2021</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1215-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Domain organization of ICV HEF and bimane labeling reaction</div>
<p>(A) Linear segments of ICV: HEF<sub>1</sub> primarily consists of the receptor binding domain (RBD, dark blue), the esterase domain (E, blue), and two small regions belonging to the ICV HEF<sub>2</sub> membrane fusion domain (F, red). The fusion subunit (HEF<sub>2</sub>) consists of a N-terminal hydrophobic fusion peptide (orange), fusion domains (F, red) and a transmembrane anchor (TM, grey) at its C-terminus (<a href="#bib11">Halldorsson et al., 2021</a>). The ICV HEF<sub>2</sub> constructs contained cysteine mutations at either Q558C or H518C (positive control) to allow for bimane labeling. Three additional serine mutations were made to native cysteines in ICV HEF<sub>2</sub> at C583S, C591S, and C595S to ensure site-specific incorporation of the bimane label at either H518C or Q558C.</p>
<p>(B) mBBr reacts with a free cysteine residue to form a covalent bimane-protein adduct.</p>
<p>(C) Plasmid map of the ICV HEF<sub>2</sub> pET46 Ek/LIC vector. The ampicillin-resistance pET46 Ek/LIC vector encodes for a N-terminal 6-His tag. The ICV HEF<sub>2</sub> DNA was gene synthesized with a thrombin cleavage site (LVPRGS) inserted directly before the start of ICV HEF<sub>2</sub> coding sequence (residues 495–586).</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1215-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Strategic design of ICV HEF<sub>2</sub> for TrIQ</div>
<p>Site-directed bimane labels were added to Q558C and H518C of ICV HEF<sub>2</sub>. Care was taken to mutate endogenous cysteine residues to serine (C583S, C591S and C595S) to ensure only one bimane site-specific label was inserted per viral fusion protein protomer. Tryptophan residues that encounter the bimane fluorophore will quench the bimane fluorescence signal. The Q558C-bimane label was designed to probe the formation of the second chain reversal region and the W562-W584 interaction. At pH &gt; 6.0, no fluorescence quenching was observed, indicating no W562-W584 interaction. At pH 5.5–6.0, the formation of the second chain reversal region (shown in purple) allows the interaction of W584 with W562, which was observed in the ICV HEF<sub>2</sub> pH 6.1 extended intermediate crystal structure (<a href="#bib17">Serrão et al., 2021</a>). The H518C-bimane was designed as a positive control to probe the interactions of the C-terminus with the central HR1 core. At pH &lt; 5.5, the C-terminus (orange region) folds back toward the heptad repeat region 1 (HR1), and fluorescence of the bimane fluorophore located at H518C was quenched. The schematic cartoons show the locations of the labeled bimane fluorophore (red stars) and tryptophan residues (orange rectangles) on ICV HEF<sub>2</sub>. The different segments of the ICV HEF<sub>2</sub> are colored in red, yellow, blue, cyan, green, purple, and orange. During fusion, these segments undergo major conformational changes between the pre- and post-fusion states.</p>
</figcaption>
</figure>
<p>DNA for the ICV HEF<sub>2</sub> fusion subunit and its Q558C/C583S/C591S/C595S or H518C/C583S/C591S/C595S variants were codon-optimized for expression in <i>E. coli</i>, commercially synthesized by Thermo Scientific/GeneArt as double-stranded DNA fragments, and subcloned by ligation-independent cloning using the Novagen pET46 Ek/LIC vector kit according to the manufacturer’s protocol (steps 5–11). A thrombin protease site was inserted downstream of the pET46 Ek/LIC vector-encoded N-terminal six-histidine tag (<a href="#fig1">Figure 1</a>C). The constructs used are listed in the key resources table.</p>
<ol start="5">
<li>Resuspend the lyophilized gene fragment to 25 ng/μL in 10 mM Tris-HCl pH 8.0 (Elution buffer from Geneaid High-Speed Plasmid Mini DNA purification kit used in the step 11g).</li>
<li>In a 0.2-mL PCR tube, prepare the following mix using the pET46 Ek/LIC vector kit:</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Gene fragment insert</td>
<td>0.1 pmol</td>
<td>Varies depending on size of insert</td>
</tr>
<tr>
<td>10X T4 DNA polymerase buffer</td>
<td>1X</td>
<td>1 μL</td>
</tr>
<tr>
<td>25 mM dATP</td>
<td>2.5 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>100 mM DTT</td>
<td>5 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>2.5 U/μL LIC-qualified T4 DNA polymerase</td>
<td>0.05 U/μL</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>MilliQ H<sub>2</sub>O (sterile)</td>
<td>n/a</td>
<td>to final volume of 10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>-</td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
<ol start="7">
<li>Mix by finger vortexing or by stirring with a pipette tip for 10 s.</li>
<li>In a PCR thermal cycler, incubate at 22°C for 30 min and then inactivate T4 DNA polymerase at 75°C for 20 min.</li>
<li>To a new 0.2 mL PCR tube, add 0.5 μL pET 46 Ek/LIC vector and 1 μL T4 DNA polymerase-treated gene insert. Stir with a pipette tip.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not vortex or pipet up and down as this may shear the DNA.</div>
<ol start="10">
<li>Incubate at 22°C for 5 min, then add 0.5 μL 25 mM EDTA to the reaction for a 2 μL total volume.</li>
<li>Transform annealed pET46 Ek/LIC vector-insert in <i>E. coli</i> XL10-Gold ultracompetent cells
<ol type="a">
<li>Thaw <i>E. coli</i> XL10-Gold cells on ice. Aliquot 40 μL of cells into a sterile 1.5-mL microcentrifuge tube.</li>
<li>Add 1.5 μL annealed pET46 Ek/LIC vector-insert to the <i>E. coli</i> XL10-Gold cells. Mix gently and incubate on ice for 15 min.</li>
<li>Heat-shock the tubes at 42°C for 45 s and incubate on ice for 3 min.</li>
<li>Add 0.5 mL LB media (without antibiotics) and incubate at 37°C for 1 h in an Eppendorf Excella E24R orbital shaker at 180 rpm.</li>
<li>Plate the cells on a LB-agar Amp plate using a metal bacterial spreader and incubate at 37°C for 16 h.</li>
<li>Using a sterile pipette tip for each colony, pick three or four colonies and inoculate into separate 17 x 100 mm disposable sterile culture tubes containing 5 mL LB broth supplemented with a final concentration of 100 μg/mL ampicillin. Incubate at 37°C for 16 h while shaking at 180 rpm.</li>
<li>Extract the plasmids from the cells using the Geneaid High-Speed Plasmid Mini DNA purification kit or equivalent, following the manufacturer’s protocol.</li>
<li>Verify the insertion of the gene by Sanger sequencing using T7-forward and T7-reverse sequencing primers.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Escherichia coli XL-10 Gold competent cells</td>
<td>Agilent</td>
<td>Cat#200314</td>
</tr>
<tr>
<td>Escherichia coli BL21 (DE3) competent cells</td>
<td>MilliporeSigma</td>
<td>Cat#69450</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>ICV HEF<sub>2</sub> (495–586, Q558C, C583S, C591S, C595S) recombinant protein</td>
<td><a href="#bib17">Serrão et al., 2021</a></td>
<td>Uniprot: A8E060 - C/Johannesburg/1/1966</td>
</tr>
<tr>
<td>ICV HEF<sub>2</sub> (495–586, H518C, C583S, C591S, C595S) recombinant protein</td>
<td><a href="#bib17">Serrão et al., 2021</a></td>
<td>Uniprot: A8E060 - C/Johannesburg/1/1966</td>
</tr>
<tr>
<td>Isopropyl β-D-1-thiogalactopyranoside (IPTG)</td>
<td>BioShop</td>
<td>Cat#IPT001</td>
</tr>
<tr>
<td>Ampicillin, sodium salt</td>
<td>BioShop</td>
<td>Cat#AMP201</td>
</tr>
<tr>
<td>Luria-Bertaini (LB) broth (Miller), granulated</td>
<td>MilliporeSigma</td>
<td>Cat#1.10285</td>
</tr>
<tr>
<td>Agar</td>
<td>BioShop</td>
<td>Cat#AGR003</td>
</tr>
<tr>
<td>Potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>)</td>
<td>BioShop</td>
<td>Cat#PPM666</td>
</tr>
<tr>
<td>Potassium phosphate dibasic (K<sub>2</sub>HPO<sub>4</sub>)</td>
<td>BioShop</td>
<td>Cat#PPD303</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>BioShop</td>
<td>Cat#SOD002</td>
</tr>
<tr>
<td>Imidazole</td>
<td>BioShop</td>
<td>Cat#IMD510</td>
</tr>
<tr>
<td>Monobromobimane (mBBr)</td>
<td>MilliporeSigma</td>
<td>Cat#B4380</td>
</tr>
<tr>
<td>Acetonitrile-190</td>
<td>Caledon Laboratory Chemicals</td>
<td>Cat#1401-7</td>
</tr>
<tr>
<td>Thrombin, restriction grade</td>
<td>MilliporeSigma</td>
<td>Cat#69671</td>
</tr>
<tr>
<td>N,N,N′,N′-tetramethyl ethylenediamine (TMED)</td>
<td>BioShop</td>
<td>Cat#TEM001</td>
</tr>
<tr>
<td>40% acrylamide/bis-acrylamide (37.5:1) solution</td>
<td>BioShop</td>
<td>Cat#ACR005</td>
</tr>
<tr>
<td>Coomassie Brilliant Blue G-250</td>
<td>BioShop</td>
<td>Cat#CBB555</td>
</tr>
<tr>
<td>Ammonium persulfate (APS)</td>
<td>BioShop</td>
<td>Cat#AMP001</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate (SDS)</td>
<td>BioShop</td>
<td>Cat#SDS003</td>
</tr>
<tr>
<td>Acetic acid, glacial</td>
<td>CALEDON LABORATORY CHEMICALS</td>
<td>Cat#1000-1</td>
</tr>
<tr>
<td>Hellmanex III</td>
<td>Helma</td>
<td>Cat#9-307-011-4-507</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>High Speed Plasmid Mini kit</td>
<td>Geneaid Biotech</td>
<td>Cat#PD300</td>
</tr>
<tr>
<td>pET46 Ek/LIC vector kit</td>
<td>MilliporeSigma</td>
<td>Cat#71335</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pET46 Ek/LIC vector</td>
<td>MilliporeSigma</td>
<td>Cat#71335</td>
</tr>
<tr>
<td>dsDNA gene fragments</td>
<td>Thermo Fisher Scientific/GeneArt</td>
<td>-</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Calculations and graphing</td>
<td>Origin Lab</td>
<td>Origin 2017</td>
</tr>
<tr>
<td>One-way ANOVA with a <i>Tukey’s post hoc</i> test</td>
<td>GraphPad Software</td>
<td>Prism v9.0.0 (121)</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>50 mL conical centrifuge tube</td>
<td>VWR</td>
<td>Cat#89039-656</td>
</tr>
<tr>
<td>1 L graduated cylinder</td>
<td>VWR</td>
<td>Cat#65000-012</td>
</tr>
<tr>
<td>1 L beaker</td>
<td>VWR</td>
<td>Cat#10754-960</td>
</tr>
<tr>
<td>250 mL media bottle Kimax GL45</td>
<td>Kimble Chase</td>
<td>Cat#14395-250</td>
</tr>
<tr>
<td>2 L Erlenmeyer flask</td>
<td>VWR</td>
<td>Cat#10545-844</td>
</tr>
<tr>
<td>Parafilm M Bemis</td>
<td>Fisher Scientific</td>
<td>Cat#13-374-12</td>
</tr>
<tr>
<td>PCR 0.2 mL tube</td>
<td>Axygen/Corning</td>
<td>Cat#PCR-02-C</td>
</tr>
<tr>
<td>Petri plate 60 x 15 mm</td>
<td>VWR</td>
<td>Cat#25384-092</td>
</tr>
<tr>
<td>Metal bacterial spreader small</td>
<td>SP Bel-Art</td>
<td>Cat#F377360006</td>
</tr>
<tr>
<td>17 x 100 mm culture tube with snap cap</td>
<td>Fisher Scientific</td>
<td>Cat#149569C</td>
</tr>
<tr>
<td>25 mm 0.22 μm PES syringe filter</td>
<td>Pall Corporation</td>
<td>Cat#4612</td>
</tr>
<tr>
<td>1.5 mL microcentrifuge tube</td>
<td>Fisher Scientific</td>
<td>Cat#02-682-002</td>
</tr>
<tr>
<td>FlexColumn 15 mm x 200 mm column</td>
<td>KONTES</td>
<td>Cat#K420400-1520</td>
</tr>
<tr>
<td>Ni-NTA agarose affinity resin</td>
<td>QIAGEN</td>
<td>Cat#30230</td>
</tr>
<tr>
<td>25 x 16 mm Spectra/Por 2 regenerated cellulose dialysis tubing 12–14 kDa MWCO</td>
<td>Repligen</td>
<td>Cat#132678</td>
</tr>
<tr>
<td>Amicon Ultra-15 Ultracel-10K concentrator</td>
<td>MilliporeSigma</td>
<td>Cat#UFC901096</td>
</tr>
<tr>
<td>Superdex-75 prep grade resin</td>
<td>Cytiva</td>
<td>Cat#17-1044-01</td>
</tr>
<tr>
<td>HiLoad XK 16/40 empty column</td>
<td>Cytiva</td>
<td>Cat#28988938</td>
</tr>
<tr>
<td>HiTrap Desalting column 5 mL</td>
<td>Cytiva</td>
<td>Cat#29048684</td>
</tr>
<tr>
<td>96-well synthetic quartz glass microplate</td>
<td>Helma</td>
<td>Cat#730-009-44</td>
</tr>
<tr>
<td>Deionized water purification system MilliQ Direct 16</td>
<td>MilliporeSigma</td>
<td>Model#ZR0Q01600</td>
</tr>
<tr>
<td>pH meter</td>
<td>Sartorius</td>
<td>Model#PB-11</td>
</tr>
<tr>
<td>SimpliAmp PCR thermal cycler</td>
<td>Thermo Fisher Scientific</td>
<td>Model#A24811</td>
</tr>
<tr>
<td>Isotemp dry block heater</td>
<td>Thermo Fisher Scientific</td>
<td>Model#2050FS</td>
</tr>
<tr>
<td>Excella E24R orbital shaker</td>
<td>Eppendorf</td>
<td>Model#M1352-0004</td>
</tr>
<tr>
<td>I26R high-capacity refrigerated orbital shaker</td>
<td>Eppendorf</td>
<td>Model#M1324-0004</td>
</tr>
<tr>
<td>Microcentrifuge Microfuge 20R</td>
<td>Beckman Coulter</td>
<td>Model#B31612</td>
</tr>
<tr>
<td>Cell disruption system CF1 TS 0.75 series</td>
<td>Constant Systems</td>
<td>Model#BT40/TS2/BA</td>
</tr>
<tr>
<td>High-capacity centrifuge Sorvall RC6+</td>
<td>Thermo Fisher Scientific</td>
<td>Model#46910</td>
</tr>
<tr>
<td>Fiberlite rotor 1000 mL bottles F10S-4x1000 LEX</td>
<td>Thermo Fisher Scientific</td>
<td>Model#096-041075</td>
</tr>
<tr>
<td>Fiberlite rotor 50 mL tubes F21-8x50y</td>
<td>Thermo Fisher Scientific</td>
<td>Model#096-084019</td>
</tr>
<tr>
<td>Benchtop centrifuge Sorvall Legend XTR</td>
<td>Thermo Fisher Scientific</td>
<td>Model#75004521</td>
</tr>
<tr>
<td>TX-750 swinging bucket rotor</td>
<td>Thermo Fisher Scientific</td>
<td>Model#75003607</td>
</tr>
<tr>
<td>FPLC system Akta Pure 25 L1</td>
<td>Cytiva</td>
<td>Model#29018225</td>
</tr>
<tr>
<td>Spectrophotometer NanoDrop 2000c</td>
<td>Thermo Fisher Scientific</td>
<td>Model#ND-2000</td>
</tr>
<tr>
<td>Xcell SureLock Mini-Cell electrophoresis system</td>
<td>Thermo Fisher Scientific</td>
<td>Model#EI0001</td>
</tr>
<tr>
<td>Electrophoresis power source PowerPac HV</td>
<td>Bio-Rad Laboratories</td>
<td>Model#1645056</td>
</tr>
<tr>
<td>Multi-mode plate reader Synergy Neo2</td>
<td>BioTek/Agilent</td>
<td>Model#NEO2SMALPHA</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl2">
<caption>Buffer A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 7.5 (500 mM)</td>
<td>50 mM</td>
<td>100 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>300 mM</td>
<td>60 mL</td>
</tr>
<tr>
<td>Imidazole (2.5 M)</td>
<td>20 mM</td>
<td>0.8 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>839.2 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Storage conditions: store buffer at 4°C for up to 6 months</p>
<table id="undtbl3">
<caption>Buffer B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 7.5 (500 mM)</td>
<td>50 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>300 mM</td>
<td>30 mL</td>
</tr>
<tr>
<td>Imidazole (2.5 M)</td>
<td>100 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>400 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Storage conditions: store buffer at 4°C for up to 6 months</p>
<table id="undtbl4">
<caption>Buffer C</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 7.5 (500 mM)</td>
<td>50 mM</td>
<td>100 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>300 mM</td>
<td>60 mL</td>
</tr>
<tr>
<td>Imidazole (2.5 M)</td>
<td>500 mM</td>
<td>200 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>640 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Storage conditions: store buffer at 4°C for up to 6 months</p>
<table id="undtbl5">
<caption>Buffer D</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> pH 7.5 (500 mM)</td>
<td>10 mM</td>
<td>40 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>150 mM</td>
<td>60 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1900 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Storage conditions: store buffer at 4°C for up to 6 months</p>
<p>Alternative reagents:</p>
<p>Many of the items listed in the Key Resource Table can be purchased from alternate vendors:</p>
<ul>
<li>Common chemicals from Bioshop (e.g., NaCl, KH<sub>2</sub>PO<sub>4</sub>, K<sub>2</sub>HPO<sub>4</sub>, IPTG, ampicillin, agar, imidazole, etc) and Caledon (acetonitrile and acetic acid) can be purchased from Millipore Sigma.</li>
<li>LB broth (Miller) granulated can be substituted with the powdered formulation, available through multiple vendors, such as Thermo Scientific/Invitrogen (Cat#12795027).</li>
<li>mBBr can be purchased from Thermo Scientific/Invitrogen (Cat#M20381).</li>
<li>Ni-NTA resin can be purchased from Cytiva (Cat#17526801) or Thermo Scientific (Cat#88221).</li>
<li>dsDNA gene fragments can be synthesized by Eurofins Genomics, Twist Biosciences, Genscript, or IDT, among others.</li>
<li>Dialysis tubing can be substituted with those purchased from Thermo Scientific (Cat#88243).</li>
<li>Superdex 75 prep grade resin custom packed into a HiLoad XK 16/40 empty column can be replaced by commercially pre-packed FPLC columns, such as Superdex 75 Increase 10/300 (Cat#29148721).</li>
</ul>
<p>Alternative equipment:</p>
<p>Many of the equipment listed in the Key Resource Table can be performed using alternate models. For the more specialized pieces of equipment, alternate equipment is suggested as follows:</p>
<ul>
<li>Constant Systems CF1 TS 0.75 cell disruption system- The lysis of <i>E. coli</i> cells can be performed using alternate systems such as the Avestin Emulsiflex C3, Microfluidics M110P or French Press.</li>
<li>Nanodrop 2000c- Spectrophotometric measurements can be performed using any spectrophotometer capable of measuring in the UV/Vis range.</li>
<li>Biotek Synergy Neo2 Multi-mode plate reader- Fluorescence measurements can be performed using other multi-mode fluorescence plate reader systems, such as those from Tecan, Perkin Elmer, BMG, or Molecular Devices.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">ICV HEF<sub>2</sub> expression and purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<p>An overview of the ICV HEF<sub>2</sub> expression and purification workflow is shown in <a href="#fig3">Figure 3</a>.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1215-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. ICV HEF<sub>2</sub> expression and purification workflow</div>
<p>For expression, the ICV HEF<sub>2</sub> pET46 Ek/LIC vector was transformed into <i>E. coli</i> BL21 (DE3) cells. A single colony was inoculated into a 1-L LB broth culture and induced with IPTG to express the ICV HEF<sub>2</sub> protein. <i>E. coli</i> cells were harvested by centrifugation and lysed using a hydraulic cell disruption system. The protein was then purified by Ni-NTA affinity chromatography followed by size exclusion chromatography. ICV HEF<sub>2</sub> eluted from the gel filtration column as a trimer. Long and short constructs of ICV HEF<sub>2</sub> variants (HEF<sub>495-586</sub> Q558C, HEF<sub>495-586</sub> H518C, HEF<sub>481-621</sub> Q558C and HEF<sub>481-621</sub> H518C) were expressed and purified for the study; for more details, please refer to (<a href="#bib17">Serrão et al., 2021</a>). The figure was generated with <a href="http://BioRender.com">BioRender.com</a>.</p>
</figcaption>
</figure>
<ol>
<li>Transform the pET46 Ek/LIC plasmid encoding N-terminal 6xHis-tagged ICV HEF<sub>2</sub> into <i>E. coli</i> BL21(DE3) cells for expression as described in step 11 in the ‘<a href="#before-you-begin">before you begin</a>’ section. Plate cells on a LB-agar Amp plate and grow at 37°C for at least 12 h.</li>
<li>Inoculate a single colony in 5 mL LB broth with a final concentration of 100 μg/mL ampicillin in a 17 x 100 mm disposable culture tube to produce an <i>E. coli</i> starter culture. Incubate at 37°C for 16 h in an Eppendorf Excella E24R orbital shaker at 180 rpm.</li>
<li>Inoculate 1 L of LB broth containing a final concentration of 100 μg/mL ampicillin with 5 mL of the <i>E. coli</i> starter culture and incubate at 37°C in an Eppendorf Excella E24R orbital shaker at 180 rpm until optical density at 600 nm (O.D.600nm) is ∼0.6.</li>
<li>Add IPTG to a final concentration of 0.5 mM to induce protein expression.</li>
<li>Incubate at 18°C for 20 h in a high-capacity refrigerated Eppendorf I26R orbital shaker at 180 rpm.</li>
<li>Harvest the cells by centrifugation at 4,000 rpm using a Fiberlite F10S-4x1000 LEX rotor (3,000<i>g</i>) in a Sorvall RC6+ high-capacity centrifuge for 45 min at 4°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Pelleted cells can be stored at −80°C for 6 months.</div>
<ol start="7">
<li>Resuspend the harvested cells in 30 mL of pre-chilled Buffer A (recipe in step 3d, <a href="#sec1.1">preparation of stock solutions</a>).</li>
<li>Lyse the cells at 30,000 psi using a Constant Systems TS-series 0.75 cell disruption system.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The cell lysate should be kept cold (either at 4°C or on ice) during cell lysis and purification to prevent protein denaturation.</div>
<ol start="9">
<li>Centrifuge the lysate at 13,500 rpm using a Fiberlite F21-8x50y rotor (21,600<i>g</i>) in a Sorvall RC6+ high-capacity centrifuge for 45 min at 4°C to remove cell debris.</li>
<li>Decant the supernatant and filter using a low-protein binding 0.22-μm PES syringe filter.</li>
<li>Using a 15 x 200 mm gravity flow open column, pack a 3 mL bed volume of Ni-NTA resin and equilibrate with 10 column volumes Buffer A.</li>
<li>Apply the ICV HEF<sub>2</sub>-containing clarified lysate by gravity onto the 3 mL Ni-NTA column at 4°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Once cells are lysed, the lysate should be applied onto a Ni-NTA column as soon as possible.</div>
<ol start="13">
<li>Wash the resin with 15 column volumes Buffer B (recipe in step 3e, <a href="#sec1.1">preparation of stock solutions</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> During step 13, additional Buffer B may be used to wash the resin to eliminate non-specifically bound contaminating proteins.</div>
<ol start="14">
<li>Elute ICV HEF<sub>2</sub> using 5 column volumes Buffer C (recipe in step 3f, <a href="#sec1.1">preparation of stock solutions</a>).</li>
<li>Remove the 6xHis tag using restriction-grade thrombin. Add 1 U thrombin per 1 mg protein to the eluate from step 14. Dialyze the thrombin-protein mixture against 1 L Buffer D (recipe in step 3g, <a href="#sec1.1">preparation of stock solutions</a>) at 4°C for 16 h using 12–14 kDa MWCO dialysis tubing.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> High concentrations of imidazole found in the Ni-NTA elution buffer (Buffer C) are inhibitory to thrombin enzymatic activity. It is important to lower the concentrations of imidazole by dialysis either before or during thrombin digestion. Also, thrombin is effectively inhibited by 1 mM PMSF; do not use serine protease inhibitors during thrombin digestion.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended to use restriction grade quality thrombin for cleavage of the His-tag. Non-restriction grade quality thrombin may contain additional proteases that can non-specifically cleave the protein of interest.</div>
<ol start="16">
<li>Concentrate dialyzed ICV HEF<sub>2</sub> to 1 mL using an Amicon Ultra-15 Centrifugal concentrator (10 kDa MWCO), pre-equilibrated with Buffer D, at a speed of 3,800 rpm at 4°C in a TX-750 swinging bucket rotor (3,150<i>g</i>) in a Sorvall XTR benchtop centrifuge.</li>
<li>Inject sample onto a custom packed Superdex 75 prep grade resin XK 16/40 gel filtration column pre-equilibrated with 2 column volumes Buffer D at a flow rate of 1 mL/min using an Akta Pure FPLC system. Monitor elution fractions by absorbance at 280 nm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The protein can also be purified using a commercially pre-packed Superdex 75 Increase 10/300 gel filtration column.</div>
<ol start="18">
<li>Pool protein-containing fractions and concentrate to ∼1.8 mg/mL using an Amicon Ultra-15 Centrifugal Filter Unit (10 kDa MWCO) at a speed of 3,800 rpm using a TX-750 swinging bucket rotor (3,150<i>g</i>) in a Sorvall XTR benchtop centrifuge at 4°C.</li>
<li>Quantify protein concentration by determining absorbance at 280 nm on a NanoDrop 2000c or another equivalent spectrophotometer. Use the Beer-Lambert law and the theoretical molar extinction coefficient of the protein to calculate the protein concentration.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The theoretical molar extinction coefficient can be calculated from the protein sequence using web servers such as Expasy ProtParam.</div>
<ol start="20">
<li>Analyze protein purity by 16% SDS-polyacrylamide gel electrophoresis. Stain gel for 10 min with Coomassie Brilliant Blue G-250 stain and destain in 10% (v/v) acetic acid.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Purified ICV HEF<sub>2</sub> can be stored at 4°C for up to 10 days without any aggregation or protein precipitation.</div>
<h3 id="sec3.2">Site-directed bimane labeling and fluorescence data acquisition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="21">
<li>Aliquot 0.5 mL of 1.8 mg/mL purified ICV HEF<sub>2</sub> (equivalent to 100 μM) from step 18 into pre-chilled 1.5-mL microcentrifuge tubes.</li>
<li>Add 55 μL of freshly prepared 100 mM mBBr stock (recipe in step 4, <a href="#sec1.1">preparation of stock solutions</a>) to the purified ICV HEF<sub>2</sub> (final concentration 0.5 mM mBBr). The mBBr is added in 5-fold molar excess to the protein sample for labeling.</li>
<li>Incubate the sample for 2 h at 22°C in the dark to avoid fluorophore degradation.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the labeling reaction in the dark using aluminum foil to ensure mBBr stability and to maximize labeling efficiency.</div>
<div class="note">
<span class="note-title">Note:</span> The solution should be slightly orange with a weak green tinge.</div>
<div class="critical">
<span class="critical-title">Critical:</span> mBBr is soluble in acetonitrile; however, acetonitrile may be detrimental to protein stability. It is important to examine the protein sample for precipitation before continuing. The labeling time, reaction temperature, and mBBr:protein ratio may need to be optimized to prevent or reduce protein precipitation. Circular dichroism wavelength scans and thermal melts may be used to determine whether protein structure or stability was affected by bimane labeling (<a href="#bib12">Jones Brunette and Farrens, 2014</a>; <a href="#bib17">Serrão et al., 2021</a>; <a href="#bib19">Taraska et al., 2009</a>).</div>
<ol start="24">
<li>Remove excess, unreacted mBBr on a 5 mL Hi-Trap Desalting column, pre-equilibrated with 20 column volumes of Buffer D (see step 3g, <a href="#sec1.1">preparation of stock solutions</a>), using a flow rate of 1 mL/min.</li>
<li>Measure the A<sub>280</sub> for each eluted fraction from the Desalting column and pool the fractions containing the protein. Concentrate using an Amicon Ultra-15 Centrifugal Filter Unit (10 kDa MWCO) by centrifuging at a speed of 3,800 rpm in a TX-750 swinging bucket rotor (3,150<i>g</i>) at 4°C in a Sorvall XTR benchtop centrifuge.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Bimane-labeled ICV HEF<sub>2</sub> can be stored in the dark using aluminum foil at 4°C for up to 24 h without loss of fluorescence signal.</div>
<ol start="26">
<li>Dilute bimane-labeled ICV HEF<sub>2</sub> to 0.5 mg/mL in the buffer with desired pH, as described in the ‘<a href="#sec1.1">preparation of stock solutions</a>’ section (step 4).</li>
<li>Calculate labeling efficiency as a ratio of moles of protein labeled with bimane over moles of unlabeled protein samples. Measure the UV absorbance at wavelength 295 nm (specific for tryptophan absorption) and 390 nm (specific for bimane absorption) using a NanoDrop 2000c spectrophotometer. Calculate the labeling efficiency using the following equation:</li>
</ol>
<p class="math-ml"><math> <mrow> <mfrac> <mrow> <mi>m</mi> <mi>o</mi> <mi>l</mi> <mi>e</mi> <msub> <mi>s</mi> <mrow> <mi>b</mi> <mi>i</mi> <mi>m</mi> <mi>a</mi> <mi>n</mi> <mi>e</mi> </mrow> </msub> </mrow> <mrow> <mi>m</mi> <mi>o</mi> <mi>l</mi> <mi>e</mi> <msub> <mi>s</mi> <mrow> <mi>p</mi> <mi>r</mi> <mi>o</mi> <mi>t</mi> <mi>e</mi> <mi>i</mi> <mi>n</mi> </mrow> </msub> </mrow> </mfrac> <mo>×</mo> <mn>100</mn> <mo>=</mo> <mspace></mspace> <mi>l</mi> <mi>a</mi> <mi>b</mi> <mi>e</mi> <mi>l</mi> <mi>i</mi> <mi>n</mi> <mi>g</mi> <mspace></mspace> <mi>e</mi> <mi>f</mi> <mi>f</mi> <mi>i</mi> <mi>c</mi> <mi>i</mi> <mi>e</mi> <mi>n</mi> <mi>c</mi> <mi>y</mi> <mspace></mspace> <mrow> <mo>(</mo> <mo>%</mo> <mo>)</mo> </mrow> </mrow> </math></p>
<p class="math-ml"><math> <mrow> <mi>m</mi> <mi>o</mi> <mi>l</mi> <mi>e</mi> <msub> <mi>s</mi> <mrow> <mi>b</mi> <mi>i</mi> <mi>m</mi> <mi>a</mi> <mi>n</mi> <mi>e</mi> </mrow> </msub> <mo>=</mo> <mspace></mspace> <mfrac> <mrow> <mi>A</mi> <mi>b</mi> <msub> <mi>s</mi> <mrow> <mn>390</mn> <mi>n</mi> <mi>m</mi> <mspace></mspace> </mrow> </msub> <mo>×</mo> <mspace></mspace> <mi>v</mi> </mrow> <mrow> <msub> <mi>ε</mi> <mrow> <mi>b</mi> <mi>i</mi> <mi>m</mi> <mi>a</mi> <mi>n</mi> <mi>e</mi> </mrow> </msub> <mo>×</mo> <mi>l</mi> </mrow> </mfrac> </mrow> </math></p>
<p class="math-ml"><math> <mrow> <mi>m</mi> <mi>o</mi> <mi>l</mi> <mi>e</mi> <msub> <mi>s</mi> <mrow> <mi>p</mi> <mi>r</mi> <mi>o</mi> <mi>t</mi> <mi>e</mi> <mi>i</mi> <mi>n</mi> </mrow> </msub> <mo>=</mo> <mspace></mspace> <mfrac> <mrow> <mi>A</mi> <mi>b</mi> <msub> <mi>s</mi> <mrow> <mn>295</mn> <mi>n</mi> <mi>m</mi> </mrow> </msub> <mspace></mspace> <mo>×</mo> <mspace></mspace> <mi>v</mi> </mrow> <mrow> <msub> <mi>ε</mi> <mrow> <mi>p</mi> <mi>r</mi> <mi>o</mi> <mi>t</mi> <mi>e</mi> <mi>i</mi> <mi>n</mi> </mrow> </msub> <mspace></mspace> <mo>×</mo> <mi>l</mi> </mrow> </mfrac> </mrow> </math></p>
<p>where Abs<sub>λ</sub> = absorbance value at particular wavelength; ε<sub>protein</sub> = molar extinction of the protein in M<sup>−1</sup>cm<sup>−1</sup>; ε<sub>bimane</sub> = molar extinction coefficient of bimane = 5,000 M<sup>−1</sup>cm<sup>−1</sup>; l = pathlength in cm; v = volume in liters.</p>
<div class="note">
<span class="note-title">Note:</span> The theoretical protein molar extinction coefficient can be calculated from the protein sequence using web servers such as Expasy ProtParam.</div>
<ol start="28">
<li>Pipet 200 μL bimane-labelled protein sample (0.5 mg/mL) at the various pH values and associated buffers for reference blanking into a 96-well quartz plate. Samples should be prepared in at least independent triplicates (plate map shown in <a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1215-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Representative TrIQ plate setup</div>
<p>ICV HEF<sub>2</sub> Q558C-bimane or H518C-bimane at various pH values from 4.5 to 7.5 are pipet into a 96-well synthetic quartz plate. Technical triplicates are set up for each independently expressed, purified, and labeled batch of protein. A minimum of three independent replicates are measured. Free monobromobimane (mBBr) and buffer are included in technical triplicates as controls. This figure was generated with <a href="http://BioRender.com">BioRender.com</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Care should be taken to ensure that the 96-well quartz plate or the cuvette is properly cleaned before loading each sample. Residual bimane-labeled protein will result in inconsistent readings. Incubate the plate or cuvette in 2% (v/v) Hellmanex III for 20 min, rinse well with doubly distilled H<sub>2</sub>O, and use N<sub>2</sub> gas or compressed air to dry the plate thoroughly between samples.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended to use a synthetic quartz microplate rather than black polystyrene 96-well plates, as the quartz microplate provides the lowest auto-fluorescence/background.</div>
<div class="note">
<span class="note-title">Note:</span> Fluorescence measurements can also be performed using a 10-mm pathlength quartz cuvette on an ISS PC1 photon-counting spectrofluorometer with an “L” configuration or equivalent spectrofluorometer. The use of a 10-mm pathlength quartz cuvette will improve the signal-noise ratio and signal quality but will increase the time required to complete data collection.</div>
<ol start="29">
<li>Measure the fluorescence of bimane-labeled ICV HEF<sub>2</sub> using a Synergy Neo2 multimode plate reader or equivalent fluorescence plate reader at 22°C. Set fluorescence excitation wavelength to 395 (± 5) nm and perform data acquisition in the top-to-top mode with autogain. Collect the emission spectra from 425 to 600 nm with a 1-nm monochromator step and 0.5–1 nm bandwidth, depending on the signal intensity. Set acquisition time to 100 ms and 10 integrations.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> An initial measurement using only Buffer D and the sample at pH 7.5 should be performed to set the autogain function and determine the background profile. The slit and bandwidths can affect the intensity readings. The slit and bandwidths should be optimized for the bimane-labeled protein and kept constant for all experiments.</div>
<div class="note">
<span class="note-title">Note:</span> Experiments should be repeated in at least triplicates with independently expressed and purified batches of protein. Each data point should be measured in technical triplicates. An independent replicate is defined as a protein sample that is independently expressed, purified, and bimane-labeled.</div>
<ol start="30">
<li>Measure free mBBr (final concentration 850 μM) at various pH values as a reference control.</li>
</ol>
<h3 id="sec3.3">Data processing and statistical analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4<b>–8 h</b>
</div>
<div class="note">
<span class="note-title">Note:</span> We recommend using Origin 2017 for initial data processing and GraphPad v9.0 for advanced statistical analysis (<a href="#fig5">Figure 5</a>).</div>
<ol start="31">
<li>Buffer subtract the acquired technical triplicate data using the reference values obtained from the Buffer D fluorescence scan and average the replicates.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1215-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Representative TrIQ data</div>
<p>(A) Experimental fluorescence emission spectra for free mBBr and the buffer control. For free mBBr, data are expressed as the average of four independent buffer-subtracted replicates ± SEM.</p>
<p>(B) Experimental fluorescence emission spectra of ICV HEF<sub>2</sub> Q558C-bimane normalized from 0 to 1. Data from four independent replicates were averaged and buffer subtracted ± SEM. The maximum fluorescence intensity was used in subsequent F/F<sub>0</sub> quenching factor calculations.</p>
<p>(C) The fluorescence quenching factor of ICV HEF<sub>2</sub> Q558C-bimane TrIQ as a function of pH. At pH values &gt; 6.5 and &lt; 5.5, no quenching was observed, suggesting no interactions between W562 and W584. Fluorescence quenching was observed at pH 5.5–6.0, indicating an interaction between W562 and W584 and the formation of the second chain reversal region.</p>
<p>(D) Statistical analysis of the change in TrIQ F/F<sub>0</sub> quenching fraction values as a function of pH are presented as mean ± SEM. One-way ANOVA followed by a <i>Tukey post-hoc</i> comparison test was used for statistical analysis: n.s.= not significant (p &gt; 0.05); ∗ p &lt; 0.0001; ∗∗ p &lt; 0.001. These figure panels are adapted from (<a href="#bib17">Serrão et al., 2021</a>).</p>
</figcaption>
</figure>
</li>
<li>Normalize the data from 0 to 1.</li>
<li>Average the normalized independent replicates and calculate the standard error of the mean (SEM).</li>
<li>Calculate the quenching factor (F/F<sub>0</sub>) for each pH as follows:</li>
</ol>
<p class="math-ml"><math> <mrow> <msub> <mrow> <mo>(</mo> <mfrac> <mi>F</mi> <msub> <mi>F</mi> <mn>0</mn> </msub> </mfrac> <mo>)</mo> </mrow> <mi>i</mi> </msub> <mo>=</mo> <mspace></mspace> <msub> <mrow> <mo>(</mo> <mfrac> <msub> <mi>I</mi> <mrow> <mi>m</mi> <mi>a</mi> <mi>x</mi> <mo>,</mo> <mi>λ</mi> </mrow> </msub> <msub> <mi>I</mi> <mrow> <mn>0</mn> <mo>,</mo> <mspace></mspace> <mi>λ</mi> </mrow> </msub> </mfrac> <mo>)</mo> </mrow> <mi>i</mi> </msub> </mrow> </math></p>
<p>where I<sub>max,λ</sub> is the normalized maximum fluorescence emission value at the wavelength (λ), and I<sub>0,λ</sub> is the normalized maximum fluorescence emission value of the free-mBBr reference at the same wavelength.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Since the maximum fluorescence intensity (I<sub>max λ</sub>) value is arbitrarily defined by the user, it is important to be consistent with the decision for selection of the wavelength value.</div>
<ol start="35">
<li>Perform a one-way ANOVA (<a href="#bib8">Donaldson, 1966</a>; <a href="#bib24">Winkler, 1975</a>) on the F/F<sub>0</sub> quenching fraction to determine whether there are statistically significant differences between pH groups.</li>
<li>Perform a <i>Tukey’s post-hoc</i> test (<a href="#bib20">Tukey, 1949</a>) to determine whether there are statistically different groups.</li>
<li>Estimate the average C<sub>α</sub>-C<sub>α</sub> distance between the probe and tryptophan residues using the calculated ICV HEF<sub>2</sub> quenching fraction (F/F<sub>0</sub>). Based on the obtained calculated quenching fraction, this can be compared with the average C<sub>α</sub>-C<sub>α</sub> distance values as a function of quenching fraction, as previously listed in (<a href="#bib12">Jones Brunette and Farrens, 2014</a>).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>TrIQ represents an easy, sensitive, and inexpensive method to detect and monitor conformational change and protein-protein interactions (<a href="#bib12">Jones Brunette and Farrens, 2014</a>; <a href="#bib15">Mansoor et al., 2010</a>). TrIQ measures the quenching induced by aromatic residues, such as tryptophan. In this technique, bimane, a heterocyclic fluorescence probe, can be site-specifically incorporated into protein via a free cysteine residue. Bimane exhibits a change in fluorescence intensity and emission maxima in response to changes in its physiochemical environment when tryptophans are in proximity; this results in fluorescence quenching of the bimane fluorescence signal. TrIQ is complementary to <i>Förster</i> Resonance Energy Transfer (FRET) but has several advantages over it. TrIQ occurs only over short distances (5–15 Å) and does not require double labeling of the sample with different fluorescent probes. In addition, the incorporation of the label does not need to be quantitative, as the tryptophan quencher is always encoded in the protein. TrIQ is well-suited to detect complex protein conformational change, such as those involved during viral-host cell membrane fusion and entry.</p>
<p>Understanding how viruses fuse with host cells is an area of great interest, especially with the emergence of SARS-CoV-2. The influenza A virus hemagglutinin (HA) glycoprotein has been the prototypical system to study the viral entry and fusion (<a href="#bib18">Skehel and Wiley, 2000</a>; <a href="#bib23">Wilson et al., 1981</a>). Structural studies of IAV HA at neutral and low pH states have captured intimate images of the start and end conformations along the fusion reaction coordinate (<a href="#bib4">Bullough et al., 1994</a>; <a href="#bib23">Wilson et al., 1981</a>). It shows that the IAV HA sits on the surface of the virus in a metastable, pre-fusion conformation. Upon trafficking of the virus into the endosome, the low pH environment triggers a drastic conformational change that leads to HA adopting a low-energy post-fusion six helix bundle (6HB) conformational state (<a href="#bib4">Bullough et al., 1994</a>). The conformational changes of IAV HA was originally thought to occur through a ‘<i>spring-loaded jackknife’</i> mechanism (<a href="#bib6">Carr and Kim, 1993</a>), but more recently data propose that HA forms a transient ensemble of intermediate states during fusion (<a href="#bib2">Benhaim et al., 2020</a>; <a href="#bib5">Calder and Rosenthal, 2016</a>; <a href="#bib7">Chlanda et al., 2016</a>; <a href="#bib10">Gui et al., 2016</a>; <a href="#bib14">Lee, 2010</a>). Regardless, the structural identities of transition intermediates during viral fusion have been poorly defined.</p>
<p>Influenza C virus (ICV) offers a unique opportunity to study the fusion intermediate states. ICV hemagglutinin-esterase-fusion (HEF) is the virally encoded glycoprotein involved in viral-host entry. Similar to IAV, ICV HEF is post-translationally cleaved by host proteases to form the attachment (HEF<sub>1</sub>) and fusion (HEF<sub>2</sub>) subunits (<a href="#fig1">Figure 1</a>) and sits on the virus surface as a metastable pre-fusion trimer (<a href="#bib22">Wang and Veit, 2016</a>). The fusion kinetics of ICV were reported to be different than those of IAV and IBV (<a href="#bib9">Formanowski et al., 1990</a>) with a lag before onset of fusion. Fusion was reported between pH values of 5.6–6.1, and hemolysis occurring at a lower pH of 5.1–5.7. It was hypothesized that the kinetic lag observed during ICV fusion is due to a slow buildup of a stable ICV HEF fusion intermediate (<a href="#bib9">Formanowski et al., 1990</a>). These unique characteristics of ICV HEF<sub>2</sub> allowed Serrão <i>et al.</i> to carry out biophysical and structural studies to detect a stable extended intermediate conformation in the ICV HEF<sub>2</sub> fusion subunit at pH 6.1 (<a href="#bib17">Serrão et al., 2021</a>).</p>
<p>The crystal structure of the ICV HEF<sub>2</sub> pH 6.1 intermediate is different than the previously described pre-fusion, post-fusion and early fusion intermediate conformations. The ICV HEF<sub>2</sub> structure is consistent with a late-stage extended intermediate, as it adopts a structurally flexible C-terminal region that forms a second chain reversal region to allow its C-terminus to be anchored to the viral membrane and its N-terminal fusion peptide to be inserted into the host membrane. This extended intermediate state would exist just before the fold-back of the C-terminal helical region into the central trimeric helical core to form the hemifusion membrane intermediate and final post-fusion 6HB structure. The second chain reversal region appears to be stabilized by hydrophobic forces, including a specific W562 and W584 interaction (<a href="#bib17">Serrão et al., 2021</a>).</p>
<p>W562 and W584 are the only tryptophan residues in ICV HEF<sub>2</sub>, and these residues can be exploited in solution-based biophysical assays to monitor the conformational motions of the intermediate states. To specifically probe the W562-W584 interaction, single cysteine point mutations were introduced into strategic positions of ICV HEF<sub>2</sub> to allow for bimane labeling (<a href="#fig2">Figure 2</a>). All native cysteine residues (C583, C591, and C595) were mutated to serine residues to ensure site-specific labeling with bimane. The cysteine substitution from glutamine at position 558 (Q558C) allowed a direct measurement of the W562-W584 interaction and the formation of the extended intermediate conformation, as Q558 is adjacent to W562. Interaction of W562 and W584 will bring W584 into proximity to the bimane at Q558C, thus leading to increased fluorescence quenching via the bimane-tryptophan interaction. A second cysteine was introduced at position 518 (H518C) as a positive control, as the C-terminal region folds back at low pH and should interact with the central trimeric helix where the H518C-bimane is located.</p>
<p>The protocol described here can be widely applied to characterize conformational changes of other viral glycoproteins. Specifically, the protocol allowed efficient expression of ICV HEF<sub>2</sub> fusion subunit variants (<a href="#fig3">Figure 3</a>). A 1-L bacterial culture yields about 60 mg of purified protein that is highly soluble and stable. Purified ICV HEF<sub>2</sub> protein is stable and does not aggregate for up to 10 days when stored at 4°C. The TrIQ experiments showed that there is substantial fluorescence quenching by tryptophan on the bimane at Q558C at pH values between 5.5 to 6.0 but not above 6.0 or below 5.5 (<a href="#fig2">Figure 2</a> and <a href="#fig5">5</a>). This suggests that there is a second chain reversal region in ICV HEF<sub>2</sub> at the pH of fusion, consistent with the crystal structure of ICV HEF<sub>2</sub> at pH 6.1, which revealed the structure of an extended viral fusion intermediate. The H518C bimane-labeled control mutant, as expected, showed strong quenching only at pH values below 5.5, suggesting a fold-back of the C-terminal ICV HEF<sub>2</sub> region to the HR1 helix (<a href="#fig2">Figure 2</a> and <a href="#fig5">5</a>). Using this protocol, new insights into the gymnastics that viral glycoproteins undergo in solution was possible. For complete details of the biological findings, please refer to (<a href="#bib17">Serrão et al., 2021</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>For the TrIQ biophysical analysis, the fluorescence scans were measured from four independently expressed and purified protein replicates with each data series measured in technical triplicate. A minimum of three independent replicates should be performed. The spectra were processed as normalized averages, baseline-subtracted, and plotted as a function of wavelength (<a href="#fig5">Figure 5</a>). Normalization and standard error of the mean (SEM) were obtained using Origin 2017. One-way ANOVA followed by <i>Tukey’s post-hoc</i> test, calculated using GraphPad Prism v9.0, were used to confirm statistical significance.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>TrIQ is a powerful technique that can be used to monitor the conformational changes of proteins in solution. This technique does have several limitations.</p>
<p><i><u>One limitation is that structural knowledge of the protein target is necessary.</u></i> TrIQ requires the labeling of cysteine residues at strategically located sites in the protein. This requires <i>a priori</i> knowledge of the protein structure in order to smartly select the positions of these mutations so that specific conformational changes can be probed. In the case of ICV HEF<sub>2</sub>, a 2.4-Å resolution crystal structure of an extended intermediate was available (PDB: 6WKO) that guided placement of the cysteines. However, high-resolution experimental structures (e.g., NMR, crystallography, or single-particle analysis cryo-electron microscopy) are not available in the PDB for many proteins. If a high-resolution structure is not available, homology modeling or structural prediction with, for example, AlphaFold Protein Structure Database (<a href="#bib13">Jumper et al., 2021</a>; <a href="#bib21">Tunyasuvunakool et al., 2021</a>), should be employed to gain insights into where to incorporate the bimane label.</p>
<p><i><u>Another challenge is that tryptophan and cysteine must be incorporated into the protein of interest.</u></i> TrIQ requires the strategic positioning of tryptophan and cysteine residues in order for tryptophan to quench the bimane fluorescence signal (covalently linked at the cysteine). TrIQ is highly sensitive to distance and thus the position of both tryptophan and cysteine needs to be carefully selected. The bimane and tryptophan distance must be on average less than 11 Å for it to have a strong quenching effect. In the case of ICV HEF<sub>2</sub>, only two tryptophan residues exist in the protein construct, thus the TrIQ experiments were performed using the endogenous tryptophan probes, and a single cysteine mutation was introduced for bimane labeling. If the protein target does not have a tryptophan or a free cysteine in an ideal location, then mutations to incorporate these will need to be performed. If this is the case, it is important to carefully check the locations of the other tryptophans, as these should not be in proximity to the target tryptophan-bimane pair. Another tryptophan in proximity could complicate quenching and would need to be mutated if possible. Also, given that the bimane must be incorporated at a single site, other free cysteine residues in the protein must be mutated to serine residues.</p>
<p><i><u>Finally, protein expression and solubility can be issues.</u></i> For viral glycoproteins and/or human proteins, expression using prokaryotic systems may not result in sufficient quantities of soluble and stable recombinant protein. Human proteins and viral glycoproteins may require proper post-translational processing (e.g., N-linked glycosylation, disulfide bond formation, phosphorylation, proteolytic cleavage) for proper folding. If this is the case, a mammalian (e.g., HEK293T or CHO) or other eukaryotic (e.g., Sf9, Hi5, S2, <i>Pichia pastoris</i>) system must be used. Moreover, the introduction of a free cysteine into the target protein for bimane labeling may lead to non-specific disulfide formation and protein instability or insolubility (e.g., inclusion bodies). In these cases, changing the location of the cysteine mutation may improve protein behavior.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Partial His-tag cleavage (step 15).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>In some cases, the cleavage site is partially restricted and thrombin treatment leads to incomplete digestion. Consider optimizing the thrombin proteolysis reaction by increasing the time of incubation, increasing the temperature, or increasing the protease concentration. If despite attempts at optimization the His-tag remains resistant to complete cleavage, consider performing a reverse Ni-NTA affinity purification. In this method, the protein collected from the flow-through should consist of protein without the His-tag.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Low levels of protein expression (step 19).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Some viral or human glycoproteins are not expressed at high levels in a bacterial system given the lack of proper post-translational modifications or potential toxicity. Consider testing different expression systems such as eukaryotic S2, Sf9, Sf21, Hi5, or <i>Pichia pastoris</i> systems or mammalian HEK293T or CHO cells to identify a system for expression of functional protein. For proteins that do not have N-linked glycosylation but require native disulfide bond formation, the use of <i>E. coli</i> cells with either a chromosomal copy of the disulfide isomerase DsbC (SHuffle T7) or mutations to glutathione reductase and thioredoxin reductase (Origami B (DE3)) may facilitate proper disulfide bond formation (<a href="#bib1">Anton et al., 2016</a>; <a href="#bib3">Bessette et al., 1999</a>; <a href="#bib16">Ren et al., 2016</a>).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Formation of non-specific disulfide bonds after site-specific incorporation of cysteine (step 17 and 20).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>In some cases, the incorporation of free cysteine residues into the target protein will lead to the formation of non-specific disulfide bonds, which can result in insolubility or the formation of inclusion bodies. Consider changing the location of the cysteine mutation to improve protein behavior.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Bimane labeling is suboptimal or leads to protein precipitation or aggregation (steps 22 and 27).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>The bimane labeling reaction is quite efficient. If the labeling is sub-optimal or labeling leads to protein precipitation or aggregation, ensure that the mBBr solution is freshly prepared. Do not use freeze-thawed mBBr stocks, as mBBr may have degraded. Also consider optimizing the mBBr to protein molar ratios and labeling reaction time.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Protein fluorescence signal is too low after labeling (step 29).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Low quenching efficiency can be due to the use of non-functional or partially functional protein or a low protein concentration in the assay. Always ensure the target protein is functional before and after bimane labeling. Also consider optimizing the acquisition time and protein concentration.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Protein fluorescence signal is too high after labeling (step 29).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>High sample fluorescence signals may fall out of the dynamic range of the plate reader and thus lead to inaccurate readings. Decrease the acquisition time or the labeled protein concentration.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>Free mBBr reference fluorescence signal varies with pH (step 30).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>The fluorescence signal of free mBBr in solution is susceptible to variation as a function of pH. When performing analyses as a function of pH, measure the free mBBr fluorescence profile at each pH value and use this for quenching factor (F/F<sub>0</sub>) calculations.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Anton, B.P., Fomenkov, A., Raleigh, E.A., and Berkmen, M. (2016). Complete genome sequence of the engineered Escherichia coli SHuffle strains and their wild-type parents. Genome Announc. <i>4</i>. e00230-16. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref1">View at publisher</a></p>
<p id="bib2">Benhaim, M.A., Mangala Prasad, V., Garcia, N.K., Guttman, M., and Lee, K.K. (2020). Structural monitoring of a transient intermediate in the hemagglutinin fusion machinery on influenza virions. Sci. Adv. <i>6</i>, eaaz8822. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref2">View at publisher</a></p>
<p id="bib3">Bessette, P.H., Aslund, F., Beckwith, J., and Georgiou, G. (1999). Efficient folding of proteins with multiple disulfide bonds in the Escherichia coli cytoplasm. Proc. Natl. Acad. Sci. U S A <i>96</i>, 13703-13708. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref3">View at publisher</a></p>
<p id="bib4">Bullough, P.A., Hughson, F.M., Skehel, J.J., and Wiley, D.C. (1994). Structure of influenza haemagglutinin at the pH of membrane fusion. Nature <i>371</i>, 37-43. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref4">View at publisher</a></p>
<p id="bib5">Calder, L.J. and Rosenthal, P.B. (2016). Cryomicroscopy provides structural snapshots of influenza virus membrane fusion. Nat. Struct. Mol. Biol. <i>23</i>, 853-858. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref5">View at publisher</a></p>
<p id="bib6">Carr, C.M. and Kim, P.S. (1993). A spring-loaded mechanism for the conformational change of influenza hemagglutinin. Cell <i>73</i>, 823-832. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref6">View at publisher</a></p>
<p id="bib7">Chlanda, P., Mekhedov, E., Waters, H., Schwartz, C.L., Fischer, E.R., Ryham, R.J., Cohen, F.S., Blank, P.S., and Zimmerberg, J. (2016). The hemifusion structure induced by Influenza virus haemagglutinin is determined by physical properties of the target membranes. Nat. Microbiol. <i>1</i>, 16050. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref7">View at publisher</a></p>
<p id="bib8">Donaldson, T.S. (1966). Power of the F-Test for Nonnormal Distributions and Unequal Error Variances (RAND Corporation). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref8">View at publisher</a></p>
<p id="bib9">Formanowski, F., Wharton, S.A., Calder, L.J., Hofbauer, C., and Meier-Ewert, H. (1990). Fusion characteristics of influenza C viruses. J. Gen. Virol. <i>71</i> (<i>Pt 5</i>), 1181-1188. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref9">View at publisher</a></p>
<p id="bib10">Gui, L., Ebner, J.L., Mileant, A., Williams, J.A., and Lee, K.K. (2016). Visualization and sequencing of membrane remodeling leading to influenza virus fusion. J. Virol. <i>90</i>, 6948-6962. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref10">View at publisher</a></p>
<p id="bib11">Halldorsson, S., Sader, K., Turner, J., Calder, L.J., and Rosenthal, P.B. (2021). In situ structure and organization of the influenza C virus surface glycoprotein. Nat. Commun. <i>12</i>, 1694. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref11">View at publisher</a></p>
<p id="bib12">Jones Brunette, A.M. and Farrens, D.L. (2014). Distance mapping in proteins using fluorescence spectroscopy: Tyrosine, like tryptophan, quenches bimane fluorescence in a distance-dependent manner. Biochemistry <i>53</i>, 6290-6301. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref12">View at publisher</a></p>
<p id="bib13">Jumper, J., Evans, R., Pritzel, A., Green, T., Figurnov, M., Ronneberger, O., Tunyasuvunakool, K., Bates, R., Žídek, A., Potapenko, A., et al. (2021). Highly accurate protein structure prediction with AlphaFold. Nature <i>596</i>, 583-589. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref13">View at publisher</a></p>
<p id="bib14">Lee, K.K. (2010). Architecture of a nascent viral fusion pore. EMBO J. <i>29</i>, 1299-1311. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref14">View at publisher</a></p>
<p id="bib15">Mansoor, S.E., DeWitt, M.A., and Farrens, D.L. (2010). Distance mapping in proteins using fluorescence spectroscopy: the tryptophan-induced quenching (TrIQ) method. Biochemistry <i>49</i>, 9722-9731. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref15">View at publisher</a></p>
<p id="bib16">Ren, G., Ke, N., and Berkmen, M. (2016). Use of the SHuffle strains in production of proteins. Curr. Protoc. Protein Sci. <i>85</i>, 5.26.1-5.26.21. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref16">View at publisher</a></p>
<p id="bib17">Serrão, V.H.B., Cook, J.D., and Lee, J.E. (2021). Snapshot of an influenza virus glycoprotein fusion intermediate. Cell Rep. <i>35</i>, 109152. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref17">View at publisher</a></p>
<p id="bib18">Skehel, J.J. and Wiley, D.C. (2000). Receptor binding and membrane fusion in virus entry: the influenza hemagglutinin. Annu. Rev. Biochem. <i>69</i>, 531-569. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref18">View at publisher</a></p>
<p id="bib19">Taraska, J.W., Puljung, M.C., and Zagotta, W.N. (2009). Short-distance probes for protein backbone structure based on energy transfer between bimane and transition metal ions. Proc. Natl. Acad. Sci. U S A <i>106</i>, 16227-16232. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref19">View at publisher</a></p>
<p id="bib20">Tukey, J.W. (1949). Comparing individual means in the analysis of variance. Biometrics <i>5</i>, 99-114. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref20">View at publisher</a></p>
<p id="bib21">Tunyasuvunakool, K., Adler, J., Wu, Z., Green, T., Zielinski, M., Žídek, A., Bridgland, A., Cowie, A., Meyer, C., Laydon, A., et al. (2021). Highly accurate protein structure prediction for the human proteome. Nature <i>596</i>, 590-596. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref21">View at publisher</a></p>
<p id="bib22">Wang, M. and Veit, M. (2016). Hemagglutinin-esterase-fusion (HEF) protein of influenza C virus. Protein Cell <i>7</i>, 28-45. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref22">View at publisher</a></p>
<p id="bib23">Wilson, I.A., Skehel, J.J., and Wiley, D.C. (1981). Structure of the haemagglutinin membrane glycoprotein of influenza virus at 3 A resolution. Nature <i>289</i>, 366-373. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref23">View at publisher</a></p>
<p id="bib24">Winkler, R.L. (1975). Statistics : Probability, Inference, and Decision (Holt, Rinehart and Winston). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00700-0/sref24">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jeffrey E. Lee (<a href="mailto:jeff.lee@utoronto.ca">jeff.lee@utoronto.ca</a>).</p>
<h4>Materials availability</h4>
<p>All expression constructs and reagents generated in this study are available from the Lead Contact upon request.</p>
<h4>Data and code availability</h4>
<p>Data generated in this study are available from the lead contact upon request.</p>
<h3>Acknowledgments</h3>
<p>Funding from a <a href="https://doi.org/10.13039/501100000038">NSERC</a> Discovery Grant (RGPIN-2019-057), <a href="https://doi.org/10.13039/501100000024">CIHR</a> Operating Grant (MOP-115066; PJT-173301), an Ontario Early Researcher Award (ER-13-09-116), and a <a href="https://doi.org/10.13039/501100001804">Canada Research Chair</a> (Tier II) to J.E.L. are gratefully acknowledged. Biophysics and structural biology infrastructure were supported by funding from the <a href="https://doi.org/10.13039/501100000196">Canada Foundation for Innovation</a> John R. Evans Leaders Fund (CFI-JELF). V.H.B.S. is a recipient of a CIHR-affiliated Banting Postdoctoral Fellowship. We thank Dr. Layla Pires from Princess Margaret Cancer Centre – UHN for discussions on statistical analysis. We also thank Diana Gawol, Patrick Yip, Dr. Karen Siu, and other members of the Lee lab for experimental assistance and discussions.</p>
<h3>Author contributions</h3>
<p>All authors contributed to writing the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>