<article>
<section>
<h1 id="header">Isolation of Microglia from Mouse or Human Tissue</h1>
<p><time datetime="2020-04-17">Published: April 17, 2020</time></p>
<p>Evan A. Bordt,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> Carina L. Block,<sup><a href="#aff2">2</a></sup> Tiziana Petrozziello,<sup><a href="#aff3">3</a></sup> Ghazaleh Sadri-Vakili,<sup><a href="#aff3">3</a></sup> Caroline J. Smith,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Andrea G. Edlow,<sup><a href="#aff4">4</a></sup> and Staci D. Bilbo<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pediatrics, Lurie Center for Autism, Massachusetts General Hospital, Harvard Medical School, Boston, MA, USA</p>
<p id="aff2"><sup>2</sup>Department of Psychology and Neuroscience, Duke University, Durham, NC, USA</p>
<p id="aff3"><sup>3</sup>Healey Center for ALS at MassGeneral Hospital, NeuroEpigenetics Laboratory, Massachusetts General Hospital, Boston, MA, USA</p>
<p id="aff4"><sup>4</sup>Obstetrics, Gynecology, and Reproductive Biology, Harvard Medical School. Massachusetts General Hospital, Vincent Center for Reproductive Biology, Boston, MA, USA</p>
<p id="fn1"><sup>5</sup>Technical Contact</p>
<p id="fn2"><sup>6</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ebordt@mgh.harvard.edu">ebordt@mgh.harvard.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:staci.bilbo@duke.edu">staci.bilbo@duke.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100035">10.1016/j.xpro.2020.100035</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Microglia are the innate immune cells of the central nervous system. Although numerous methods have been developed to isolate microglia from the brain, the method of dissociation and isolation can have a profound effect on the function of these highly dynamic cells. Here, we present an optimized protocol to isolate CD11b+ cells (microglia) from mouse or human brain tissue using magnetic bead columns. Isolated microglia can be used to model diseases with neuroinflammatory components for potential therapeutic discoveries.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib7">Hanamsagar et al. (2017)</a>, <a href="#bib12">Rivera et al. (2019)</a>, and <a href="#bib5">Edlow et al. (2019)</a></p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>An optimized protocol for isolation of mouse or human microglia</li>
<li>Use of magnetic beads for CD11b+ cell isolation</li>
<li>Adaptations for isolation from embryonic or adult tissue</li>
<li>Adaptable for different endpoints (e.g., RNA, protein, flow cytometry, cell culture)</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<h3>Can Be Made Prior to Day of Procedure</h3>
<h4>Prepare the MACS Buffer before Starting the Procedure</h4>
<p>(Refer to Materials and Equipment for buffer recipe).</p>
<h4>Preparation of Nylon Filters</h4>
<p>Cut nylon Nitex mesh fabric rolls (Sefar #03-70/33) into approximately 3cm x 3cm squares, placed into a beaker, and autoclaved prior to placement in cell culture hood.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> While our lab uses Sefar nylon mesh filter rolls, any alternative 70 μm filter will suffice (e.g., Miltenyi MACS SmartStrainers).</div>
<h4>Preparation of #1, #2, #3 Pasteur Pipets</h4>
<p>Successively smaller Pasteur pipets are created by flame polishing FisherScientific Fisherbrand<sup>TM</sup> Disposable Cotton-Plugged Borosilicate-Glass Pasteur Pipets. The largest opening pipet is referred to as #1, the second largest as #2, and the smallest as #3. Due to the nature of flame-polishing pipets, every individual pipet will have some variation, however we have determined that on average #1 pipets have an opening of 0.75–0.8 cm cm, #2 have an opening of 0.5–0.55 cm, and #3 have an opening of 0.2–0.25 cm (<a href="#mmc1">Video S1</a>, <a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Representative Images for #1, #2, and #3 Pasteur Pipets</div>
</figcaption>
</figure>
<figure id="mmc1"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Mmc1.mp4" type="video/mp4"></source>
        Your browser does not support HTML5 video.
      </video>
<figcaption>
<div class="figcaption-title">Video S1. Flame polishing of Pasteur pipets</div>
</figcaption>
</figure>
<p>While our lab dissociates tissue by shear force of successively smaller Pasteur pipets, other dissociation methods may prove acceptable. For instance, the Miltenyi gentleMACS Dissociator (Miltenyi #130-093-235) can be used to dissociate tissue prior to subsequent steps.</p>
<p>While the method of force used to dissociate tissues may vary, we do recommend heat and enzyme-mediated digestion be used for this protocol (See Limitations)</p>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>HBSS (without calcium/magnesium)</td>
<td>ThermoFisher Scientific</td>
<td>14175-095</td>
</tr>
<tr>
<td>HBSS (with calcium/magnesium)</td>
<td>ThermoFisher Scientific</td>
<td>14025-092</td>
</tr>
<tr>
<td>HyClone Characterized Fetal Bovine Serum, U.S. Origin</td>
<td>GE Healthcare</td>
<td>SH30071.03</td>
</tr>
<tr>
<td>HEPES (1 mM)</td>
<td>ThermoFisher Scientific</td>
<td>15630-080</td>
</tr>
<tr>
<td>Collagenase A</td>
<td>Millipore Sigma</td>
<td>11088793001</td>
</tr>
<tr>
<td>DNase I, grade II, from bovine pancreas</td>
<td>Millipore Sigma</td>
<td>10104159001</td>
</tr>
<tr>
<td>Percoll</td>
<td>Millipore Sigma</td>
<td>GE17-0891-01</td>
</tr>
<tr>
<td>Debris Removal Solution</td>
<td>Miltenyi Biotec</td>
<td>130-109-398</td>
</tr>
<tr>
<td>PBS (1X), pH 7.4</td>
<td>ThermoFisher Scientific</td>
<td>10010-023</td>
</tr>
<tr>
<td>PBS (10X), pH 7.4</td>
<td>ThermoFisher Scientific</td>
<td>70011-044</td>
</tr>
<tr>
<td>EDTA (0.5 M), pH 8.0, RNase-free</td>
<td>ThermoFisher Scientific</td>
<td>AM9261</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Millipore Sigma</td>
<td>A9647</td>
</tr>
<tr>
<td>Sodium Pyruvate (100 mM)</td>
<td>ThermoFisher Scientific</td>
<td>11360-070</td>
</tr>
<tr>
<td>CD11b (Microglia) Microbeads, human and mouse</td>
<td>Miltenyi Biotec</td>
<td>130-093-634</td>
</tr>
<tr>
<td colspan="3">Experimental Models: Organisms/Strains</td>
</tr>
<tr>
<td>C57BL6J Mice</td>
<td>The Jackson Laboratory</td>
<td>000664</td>
</tr>
<tr>
<td>Postmortem Human Motor Cortex</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>MACS Separation Columns (LS)</td>
<td>Miltenyi</td>
<td>130-042-401</td>
</tr>
<tr>
<td>QuadroMACS Separator</td>
<td>Miltenyi</td>
<td>130-090-976</td>
</tr>
<tr>
<td>MACS MultiStand</td>
<td>Miltenyi</td>
<td>130-042-303</td>
</tr>
<tr>
<td>Refrigerated Centrifuge</td>
<td>Eppendorf</td>
<td>5810R</td>
</tr>
<tr>
<td>Rotor with Buckets</td>
<td>Eppendorf</td>
<td>A-4-81</td>
</tr>
<tr>
<td>Nitex Open Mesh Fabric</td>
<td>Sefar</td>
<td>03-70/33</td>
</tr>
<tr>
<td>Cotton-Plugged Borosilicate-Glass Pasteur Pipet</td>
<td>FisherScientific</td>
<td>13-678-8B</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> While we use Sefar nylon filters that are cut and sterilized, commonly used alternatives such as Miltenyi 70 μm filter should be equivalent.</div>
<p>While we use an Eppendorf 5810R refrigerated centrifuge with A-4-81 rotor with swinging buckets, any refrigerated centrifuge with swinging buckets and an ability to finely control brake and acceleration speeds should suffice. Acceleration and deceleration settings should be validated for alternative equipment</p>
<p>While we use Miltenyi MACS Separation Columns with Miltenyi MACS MultiStand magnet, other positive selection columns and magnets (such as BioLegend MojoSort Magnet and R&amp;D Systems MagCellect Magnet) may be an acceptable alternative. Validation must be performed if using equipment from alternative suppliers.</p>
<div class="note">
<span class="note-title">Note:</span> Validation must be performed if reagents are used from alternative suppliers.</div>
<h3>MACS Buffer</h3>
<h4>Preparation of MACS Buffer</h4>
<p>First, dissolve EDTA and BSA in 500 mL of 1X PBS. Then raise volume to 1 L with 1X PBS. Vacuum filter slowly for 30 min in tissue culture hood. Keep sterile.</p>
<table id="undtbl1">
<thead>
<tr>
<th>Solution</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1X PBS</td>
<td>1 L</td>
</tr>
<tr>
<td>EDTA</td>
<td>1 mM</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>1%</td>
</tr>
</tbody>
</table>
<h4>To Be Made Fresh Day of Procedure</h4>
<div class="note">
<span class="note-title">Note:</span> If performing the Percoll adaptation of this protocol, Percoll, 1X PBS, and 10X PBS should be taken out of the refrigerator and placed at ~20°C–25°C prior to beginning the rest of the protocol.</div>
<div class="note">
<span class="note-title">Note:</span> All procedures following cardiac perfusion with sterile saline and tissue dissection should be performed in a Class II biological hood with standard aseptic technique.</div>
<h3>Enzyme Digestion Mix</h3>
<h4>Preparation of Enzyme Digestion Mix</h4>
<div class="note">
<span class="note-title">Note:</span> Collagenase A powder can be challenging to dissolve in the enzyme digestion mix.</div>
<table id="undtbl2">
<thead>
<tr>
<th>Solution</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HBSS (without calcium/magnesium)</td>
<td> </td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>5%</td>
</tr>
<tr>
<td>HEPES (1 mM stock)</td>
<td>10 μM</td>
</tr>
<tr>
<td>Collagenase A</td>
<td>2 mg/mL</td>
</tr>
<tr>
<td>DNase I</td>
<td>28 U/mL</td>
</tr>
</tbody>
</table>
<h3>Isotonic Percoll</h3>
<h4>Preparation of Isotonic Percoll</h4>
<div class="critical">
<span class="critical-title">Critical:</span> All elements for the Percoll gradients should warm to ~20°C–25°C prior to mixing</div>
<table id="undtbl3">
<thead>
<tr>
<th>Solution</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS (10X)</td>
<td>10%</td>
</tr>
<tr>
<td>Percoll</td>
<td>90%</td>
</tr>
</tbody>
</table>
<h4>70% Percoll Solution</h4>
<p>4 mL of 70% Percoll is required per sample</p>
<div class="critical">
<span class="critical-title">Critical:</span> All elements for the Percoll gradients should come to ~20°C–25°C prior to mixing</div>
<table id="undtbl4">
<thead>
<tr>
<th>Solution</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS (1X)</td>
<td>30%</td>
</tr>
<tr>
<td>Isotonic Percoll</td>
<td>70%</td>
</tr>
</tbody>
</table>
<h4>30% Percoll Solution</h4>
<p>6 mL of 30% Percoll is required per sample</p>
<div class="critical">
<span class="critical-title">Critical:</span> All elements for the Percoll gradients should come to ~20°C–25°C prior to mixing</div>
<table id="undtbl5">
<thead>
<tr>
<th>Solution</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS (1X)</td>
<td>70%</td>
</tr>
<tr>
<td>Isotonic Percoll</td>
<td>30%</td>
</tr>
</tbody>
</table>
<h4>Post-Percoll Solution</h4>
<p>10 mL of Post-Percoll Solution is required per sample</p>
<table id="undtbl6">
<thead>
<tr>
<th>Solution</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HBSS (with calcium/magnesium)</td>
<td> </td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>5%</td>
</tr>
<tr>
<td>HEPES</td>
<td>10 μM</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<div class="timing">
<span class="timing-title">Timing:</span> for Steps 1–7: (~90 min from end of transcardiac perfusions)</div>
<p>This protocol can be adapted depending upon the type of input material as well as the endpoint assay (<a href="#fig2">Figure 2</a>). The protocol can be shortened for time-sensitive endpoints such as RNA extraction. The protocol can also use Percoll or Debris Removal Solution to remove myelinated material and other debris for assays in which cells will be cultured, and to isolate microglia in the interphase layer (Percoll). While our lab more commonly uses the Debris Removal Solution protocol when isolating microglia from adult mouse brain tissue, we use the Percoll method for embryonic mouse tissue, as the Debris Removal Solution did not successfully pellet any cells from embryonic tissue (see <a href="#limitations">Limitations</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Workflow of Isolation Protocol with Corresponding Steps in Parentheses</div>
</figcaption>
</figure>
<ol>
<li>Aliquot 5 mL of Enzyme Digestion Mix into 15 mL tubes and weigh tubes.</li>
<li>Saline transcardiac perfusion (see Note), dissect brain region of interest, chop up with razor blades on ice, and place in cold enzyme digestion mix on ice - for human tissue chop up with razor blades and place in cold enzyme digestion mix on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Not for embryonic tissue</div>
<div class="note">
<span class="note-title">Note:</span> We have tested this protocol using human postmortem fresh frozen motor cortices provided by the Massachusetts Alzheimer’s Disease Research Center (ADRC) with approval from Massachusetts General Hospital IRB (1999p009556). Postmortem tissue was obtained as follows: briefly, the brain hemisphere was removed, coronally sectioned, and fresh frozen on metal plates, after which tissue was stored in a -80°C freezer until Step 2.</div>
<div class="note">
<span class="note-title">Note:</span> We have performed this protocol from mouse tissue using whole brain or dissected regions such as the prefrontal cortex, hippocampus, and nucleus accumbens. While we have not tested this protocol from every region of the mouse brain, we believe that it is capable of isolating CD11b+ cells from all regions. Specific regions should be optimized for use.</div>
<div class="note">
<span class="note-title">Note:</span> Following transcardiac perfusion of mouse with ice cold saline, it is necessary to dissect brain regions of interest/chop of tissue on an ice-cold petri dish using ice-cold dissection tools.</div>
<ol start="3">
<li>Weigh samples (make sure to wipe water off the side of tube with kim wipe). Note the weight of the sample (Measurement in Step 3 vs measurement in Step 1), as this input will influence the volume of beads and buffer added in Step 23.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure to wipe water off the side of tube prior to weighing</div>
<ol start="4">
<li>Place samples in a 37°C water bath for 15 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Agitate every 5 min. Agitate tubes, but do not invert to mix, as tissue has the tendency to become stuck near the lid of 15 mL tubes.</div>
<ol start="5">
<li>Dissociation: Dissociate tissue to a single-cell suspension over the course of three separate flame-polished Pasteur pipets (first use #1, then #2, then #3). Pipet up and down 20 times with each pipet. Incubate at 37°C for 15 min in between each numbered pipet step. (<a href="#mmc2">Video S2</a>, <a href="#fig3">Figure 3</a>)</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Dissociation of Tissue to a Single-Cell Suspension</div>
<p>Appearance of tissue (A) prior to and (B) after dissociation steps</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Agitate every 5 min. For instance, pipet up and down 20 times using #1 pipet, then agitate tubes every 5 min during the 15 min incubation step. Then begin pipetting with #2 pipet.</div>
<figure id="mmc2"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Mmc2.mp4" type="video/mp4"></source>
        Your browser does not support HTML5 video.
      </video>
<figcaption>
<div class="figcaption-title">Video S2. Dissociation Step 1</div>
<p>Pipet aggregated tissue until all tissue moves freely up into the Pasteur pipet. Then pipet up and down 20 times.</p>
</figcaption>
</figure>
<p>During agitation steps, prepare the 30% and 70% percoll solutions (that were previously brought to ~20°C–25°C if using percoll gradients/embryonic tissue.</p>
<ol start="6">
<li>Filter samples into fresh 15 mL tubes. Place 70 μm nylon filters on top of fresh 15 mL tube. Pre-wet each filter as below (<a href="#mmc3">Video S3</a>). Add cell suspension over the pre-wet area on each filter. Rinse old 15 mL tubes with 2 mL of HBSS (with calcium/magnesium. Pipet up and down to mix. Add over the filter. Fill the new tubes to 15 mL with HBSS (with calcium/magnesium).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Pre-wet each filter by adding 1 mL of HBSS (with calcium/magnesium) to the middle of the filter and then drag to side.</div>
<div class="critical">
<span class="critical-title">Critical:</span> After this step, samples should be kept on ice as much as possible, except in the case of the Percoll gradients, which will be at ~20°C–25°C again after Step 7, in preparation for the 23°C spin.</div>
<ol start="7">
<li>Centrifuge: 10 min, 300 × <i>g</i>, 4°C, max acceleration, half brake.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this point, the protocols diverge depending upon input tissue or endpoint methods. Choose one of the methods below to continue the protocol at step 8, step 15 or step 22.</div>
<figure id="mmc3"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Mmc3.mp4" type="video/mp4"></source>
        Your browser does not support HTML5 video.
      </video>
<figcaption>
<div class="figcaption-title">Video S3. Pre-wetting Nylon Filters</div>
</figcaption>
</figure>
<h3>Debris Removal Solution Protocol</h3>
<div class="timing">
<span class="timing-title">Timing:</span> for Steps 8–14: (~30–40 min)</div>
<p>For general use for cell applications. This section will remove debris/myelin from single-cell suspension prior to isolation of CD11b+ cells.</p>
<ol start="8">
<li>Aspirate the supernatant, then resuspend cell suspension carefully with 3,100 μL of ice-cold 1X PBS and transfer cell suspension to a fresh 15 mL tube. DO NOT VORTEX.</li>
<li>Add 900 μL of cold Debris Removal Solution.</li>
<li>Mix well by pipetting 10 times slowly up and down.</li>
<li>SLOWLY layer 4 mL of 1X PBS on top, creating a visible gradient (<a href="#mmc4">Video S4</a>)</li>
</ol>
<ol start="12">
<li>Centrifuge: 10 min, max speed (at least 3,000 × <i>g</i>), 4°C, max acceleration, max brake (<a href="#fig4">Figure 4</a>)</li>
</ol>
<figure id="mmc4"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Mmc4.mp4" type="video/mp4"></source>
        Your browser does not support HTML5 video.
      </video>
<figcaption>
<div class="figcaption-title">Video S4. Generation of Debris Removal Solution Gradients</div>
<p>Begin with tube tilted at 45° angle and slowly layer 1X PBS on top of Debris REMOVAL Solution layer. As more 1X PBS is added, slowly bring tube back to a vertical position.</p>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Debris Removal Solution Gradients</div>
<p>Photographs of example gradients (A) before and (B) after centrifugation.</p>
</figcaption>
</figure>
<ol start="13">
<li>Remove the top two layers. Fill up to 14 mL with ice cold 1X PBS.</li>
<li>Centrifuge: 10 min, 1,000 × <i>g</i>, 4°C, max acceleration, max brake.</li>
</ol>
<p>(To be continued at Step 23)</p>
<h3>Percoll Gradient Protocol</h3>
<div class="timing">
<span class="timing-title">Timing:</span> for Steps 15–21: (~60–75 min)</div>
<p>For general use for cell applications, especially those utilizing embryonic tissues. This section will remove myelin from single-cell suspension prior to isolation of CD11b+ cells.</p>
<div class="note">
<span class="note-title">Note:</span> Fast temp the centrifuge to 23°C as soon as samples from step 7 are removed.</div>
<div class="note">
<span class="note-title">Note:</span> Ensure that 70% Percoll and 30% Percoll solutions are prepared prior to this step.</div>
<ol start="15">
<li>Aspirate the supernatant, then resuspend cell suspension carefully with 1 mL of 30% Percoll. Then add 4 mL of 30% Percoll on top.</li>
<li>Place an unflamed Pasteur pipet in each 15 mL tube (cover the top with a gloved finger while gently placing the tip down to the bottom) and underlay 4 mL of 70% Percoll by pipetting slowly in 1 mL increments into the open top of the Pasteur pipet (<a href="#mmc5">Video S5</a>)</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> When removing the Pasteur pipet, lift tip very slightly off the bottom of the tube to release any last fluid, and then cover the top of pipet with finger and slowly pull up, taking care not to disturb the layers. See Troubleshooting for what to do if the 70% Percoll won’t drain through the tip of the Pasteur pipet.</div>
<div class="note">
<span class="note-title">Note:</span> There is no visible myelin layer in tissue from embryonic mouse brains</div>
<div class="note">
<span class="note-title">Note:</span> The addition of a colored powder (such as phenol red) to the Isotonic Percoll solution will result in a darker 70% Percoll solution compared to 30% Percoll. This could allow for easier visualization of the gradient. However, the effects of any powder added to the cells would have to be taken into account.</div>
<ol start="17">
<li>Centrifuge: 20 min, 340 × <i>g</i>, 23°C, acceleration of 2, no brake.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Note the change in temperature, acceleration, and brake from previous steps.</div>
<div class="note">
<span class="note-title">Note:</span> Given the lack of brake, this step takes approximately 28–30 min.</div>
<div class="note">
<span class="note-title">Note:</span> Remove tubes from centrifuge very gently so as not to disturb the gradients.</div>
<div class="note">
<span class="note-title">Note:</span> Fast temp the centrifuge to 4°C after this step.</div>
<ol start="18">
<li>Bring all tubes back to the cell culture hood at ~20°C–25°C. Aspirate out the top 1 mL of liquid. With adult tissues, this would be the myelin layer- due to the relative lack of myelination in the fetal brain, you will either see nothing in the top 1 mL of fluid (30% percoll) or may see a few bits of debris (<a href="#fig5">Figure 5</a>).</li>
</ol>
<figure id="mmc5"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Mmc5.mp4" type="video/mp4"></source>
        Your browser does not support HTML5 video.
      </video>
<figcaption>
<div class="figcaption-title">Video S5. Generation of Percoll Gradient</div>
<p>Use finger to cover Pasteur pipet as it is placed under the 30% Percoll. Slowly pipet 70% Percoll into the Pasteur pipet and let it release under the 30% Percoll layer. Cover pipet with finger and gently remove.</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Post-centrifugation Percoll Gradients</div>
<p>Note that myelin will separate at the top of the liquid, cells will separate out at the interface of 70% Percoll/30% Percoll, and red blood cells will pellet at the bottom of the conical tube.</p>
</figcaption>
</figure>
<ol start="19">
<li>Pipette out the interphase cells in 1 mL increments four times (total of 4 mLs) and transfer just this 4 mLs containing the cell layer into a new set of labeled 15 mL tubes. Use a circular pipetting method and depress the top of the pipette as you lower the tip down through the 30% percoll to the interphase layer, so you don’t pick up unwanted 30% percoll. See Troubleshooting for tips on visualizing the interphase layer and how to avoid the 70% percoll layer.</li>
<li>Fill 15 mL tube up to 15 mL volume with Post-Percoll Solution</li>
<li>Centrifuge: 10 min, 300 × <i>g</i>, 4°C, max acceleration, max brake.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> After this step, samples should be kept on ice as much as possible</div>
<p>(To be continued at Step 23)</p>
<p>Molecular Protocol - for time-sensitive molecular applications such as RNA/DNA extraction</p>
<ol start="22">
<li>For this protocol, can move from the centrifugation (Step 7) straight to Step 23 for adult tissues.</li>
</ol>
<p>For embryonic mouse tissues, steps 15–21 are still required.</p>
<div class="note">
<span class="note-title">Note:</span> With older animals, we have experienced that not removing debris/myelin with Percoll/Debris Removal Solution can clog the column. If column clogging occurs, either of the first two protocols should be used.</div>
<h3>∗∗∗All Protocols Converge again at This Step∗∗∗</h3>
<div class="timing">
<span class="timing-title">Timing:</span> for steps 23–40: (~90 min)</div>
<ol start="23">
<li>Aspirate the supernatant, then re-suspend pellets with 90 μL MACS buffer and 10 μL CD11b microbeads</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If using a large amount of tissue (i.e. greater than 150 mg), you can increase the amount of MACS buffer/beads. i.e. 135 μL MACS Buffer + 15 μL beads, or 180 μL MACS Buffer + 20 μL beads</div>
<div class="note">
<span class="note-title">Note:</span> 90 μL MACS buffer and 10 μL CD11b microbeads are used when specific brain regions (i.e. hippocampus, nucleus accumbens) of young animals (i.e. PN30-75) are dissected for isolation. Older animals or larger input tissue (i.e. &gt;150 mg) would require more MACS Buffer/beads. We recommend a titration to determine the optimal bead/buffer volume for your input tissue of interest.</div>
<ol start="24">
<li>Incubate on ice for 15 min</li>
<li>Add 1 mL of MACS Buffer to samples.</li>
<li>Centrifuge: 10 min, 300 × <i>g</i>, 4°C</li>
<li>While samples are spinning, set up sorting columns in tissue culture hood (<a href="#fig6">Figure 6</a>)</li>
</ol>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Setup of LS Columns on MACS MultiStand.</div>
<p>(A) LS Columns are placed in the MACS MultiStand over opened 15 mL tubes. The LS Columns are topped with fresh nylon filters. (B) Ensure that LS Columns are placed tightly in the MACS MultiStand.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Place the columns tightly in the magnet (must be fully in place). Then place the “-” tubes that will be used to collect CD11b negative cells under the columns. Top the columns with fresh nylon filters using autoclaved forceps to maintain sterility.</div>
<ol start="28">
<li>Prewash each column with 3 mL of MACS Buffer through nylon filter.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Pre-wet each filter by adding 1 mL of HBSS (with calcium/magnesium) to the middle of the filter and then drag to side.</div>
<ol start="29">
<li>Remove samples from centrifuge and aspirate the supernatant. Resuspend pellets in 500 μL of MACS Buffer.</li>
<li>Add cell suspension to nylon filter, and then rinse the old 15 mL tube with 2 mL of MACS Buffer and add it to the nylon filter.</li>
<li>Add 3 mL of MACS Buffer to column.</li>
<li>Once liquid has finished flowing through the columns, remove the “-” tubes and place on ice.</li>
<li>Remove the columns from the magnet, and place on top of “+” tubes that will be used to collect CD11b positive cells (<a href="#mmc6">Video S6</a>).</li>
</ol>
<ol start="34">
<li>Using a serological pipet, add 5 mL of MACS Buffer to the top of the column, and then quickly plunge column as hard as possible using the supplied plunger. Repeat this (for a total of 10 mL volume) for each column. Then place “+” tubes on ice with the “-” tubes (<a href="#mmc7">Video S7</a>).</li>
</ol>
<figure id="mmc6"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Mmc6.mp4" type="video/mp4"></source>
        Your browser does not support HTML5 video.
      </video>
<figcaption>
<div class="figcaption-title">Video S6. Once Wash Has Flowed through the Columns, Carefully Remove LS Columns from the Magnetic MACS MultiStand and place onto the “+” Tubes</div>
<p>Be sure to dab top of “-” tube on the bottom of the LS column prior to removal, as there will be a drop of liquid containing negative flowthrough cells.</p>
</figcaption>
</figure>
<ol start="35">
<li>Centrifuge: 10 min, 300 × <i>g</i>, 4°C</li>
<li>Aspirate the supernatant (make sure to remove all bubbles, especially those from the “+” tubes)</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> At this point you will only see a robust pellet in the “-” tubes and will not likely see a pellet in the “+” 15 mL falcon tubes. Because you likely will not see the pellet by eye, be sure to only displace the tube once to one side or the other and leave in place while aspirating off the supernatant (see Troubleshooting). Leave a meniscus of liquid covering the bottom of the tube prior to washing the pellet in 1X PBS, to avoid cell loss.</div>
<ol start="37">
<li>Resuspend pellet in 400 μL of ice-cold 1X PBS</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> washing the pellet in 1X PBS does risk some cell loss at this point and is not necessary for certain downstream applications such as quantification of microglial cytokine production in response to stimuli such as lipopolysaccharide (LPS). Short-term culture/incubation of microglial cells in media with and without pro-inflammatory stimuli can be accomplished by resuspending the pellet directly in your desired media at this point.</div>
<figure id="mmc7"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Mmc7.mp4" type="video/mp4"></source>
        Your browser does not support HTML5 video.
      </video>
<figcaption>
<div class="figcaption-title">Video S7. Plunge MACS Buffer through LS Columns into “+” Tube</div>
<p>Plunge column as quickly and with as much force as possible.7</p>
</figcaption>
</figure>
<p>Washing the pellet in 1X PBS is necessary for downstream applications that could be affected by components of the MACS buffer (most notably the EDTA) such as single-cell RNA sequencing or flow cytometry.</p>
<ol start="38">
<li>Transfer the 400 μL of cell resuspension to fresh 1.5 mL centrifuge tubes, rinse the old 15 mL tube with 800 μL of 1X PBS, and add this rinse to the new 1.5 mL tube as well.</li>
<li>Centrifuge: 10 min, 300 × <i>g</i>, 4°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to place all centrifuge tubes in the same orientation, as pellets may be difficult to visualize. See Troubleshooting for more detail on minimizing cell loss here.</div>
<ol start="40">
<li>Carefully remove supernatant and resuspend in media of choice.</li>
</ol>
<p>The media of choice will differ depending upon the intended usage for these isolated cells: (i) we use a DMEM-based media for cell culture and single-cell RNA sequencing applications, (ii) we resuspend cells in Trizol for RNA extraction applications, and (iii) we resuspend in PBS for flow cytometry applications. Final choice of media should be optimized in individual laboratories for their intended applications.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>Using the above method, we have obtained microglial cell counts in the range of 100,000–200,000 cells per embryonic day 17.5 mouse brain, 100,000–300,000 per embryonic day 17.5 mouse placenta, and similar cell counts of 50,000–200,000 cells from bilateral hippocampus and prefrontal cortex and of 200,000–600,000 cells from whole forebrain of postnatal day 30–60 mice (<a href="#fig7">Figure 7</a>). Notably, the Percoll gradient isolation method permits isolation of sufficient microglia for downstream applications from individual embryonic mouse brains - with single cell sequencing</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Cell Yields</div>
<p>Cell counts were obtained manually on a hemocytometer for CD11b+ cells isolated from adult forebrain, adult prefrontal cortex (PFC), adult hippocampus (HP), embryonic mouse forebrain, and embryonic mouse placenta.</p>
</figcaption>
</figure>
<p>This method isolates all CD11b+ cells, which in the healthy brain are almost entirely microglia. We show in adult hippocampus that a high proportion of isolated cells are indeed brain microglia . After magnetic bead isolation, flow cytometry sorting for cells that were positive for either CD11b or Cx3cr1 were assessed for expression of CD45 (<a href="#fig8">Figure 8</a>). This protocol results in ~90% cells that are positive for either Cx3cr1 or CD11b, and under 1% of cells that are CD11b<sup>+</sup>CD45<sup>hi</sup> or Cx3cr1<sup>+</sup>CD45<sup>hi</sup>. CD45<sup>hi</sup> cells are often considered to be infiltrating macrophages, indicating a relative lack of non-microglial CD11b+ cells in our prep. However, it is the case that CD11b cell sorting may lead to non-parenchymal myeloid cell contamination (see ‘LIMITATIONS- Microglia vs other CD11b+ cells’ for further discussion).</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. CD11b+ Magnetic Bead Isolation Pulls down Brain Microglia</div>
<p>(A and B) Gating strategy for (A) CD11b<sup>+</sup> and CD45<sup>hi/lo</sup> and (B) Cx3cr1<sup>+</sup> CD45<sup>hi/lo</sup> cells. Cells were isolated using CD11b+ magnetic bead isolation method presented in this protocol, and assessed for infiltration of non-parenchymal myeloid cells as assessed by increase in CD45 levels.</p>
<p>(C) Quantification of CD11b<sup>+</sup>CD45<sup>lo</sup>, CD11b<sup>+</sup>CD45<sup>hi</sup>, Cx3cr1<sup>+</sup>CD45<sup>lo</sup>, and Cx3cr1<sup>+</sup>CD45<sup>hi</sup> cells obtained using this protocol.</p>
</figcaption>
</figure>
<p>This method can be used to isolate cells at different developmental ages, as we demonstrated in (<a href="#bib7">Hanamsagar et al.,2017</a>). Bulk RNA sequencing analyses of microglia obtained using this method demonstrated that the microglial transcriptome matures with age, and at a different pace in male vs female microglia.</p>
<p>CD11b+ cells are not limited to the brain, and this methodology can be used to isolate CD11b+ cells from other tissues. For example, we have used this method to successfully isolate CD11b+ cells from individual mouse placentas, as well.</p>
<p>We have also successfully used this method to isolate microglia from fresh frozen human brain tissue. Successful isolation was confirmed by immunoblot positivity for Iba1 and negativity for GFAP and NeuN (<a href="#fig9">Figure 9</a>). Importantly, the protocol used to obtain CD11b+ cells from mouse brain tissue was unaltered in order to isolate CD11b+ cells from postmortem human tissue, beginning from Step 2. Step 2 mentions that human tissue was chopped with a razor blade and placed in enzyme digestion mix, after which human tissue underwent the exact same manipulations as mouse tissue. Our laboratory only cultures freshly-isolated cells, and as human postmortem tissue was all previously frozen, we have not attempted to culture CD11b+ cells from this tissue.</p>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Lysates from Human CD11b+ or CD11b- Cells Were Run on Immunoblot and Probed for Iba1 (Microglia), GFAP (Astrocytes), and NeuN (Neurons)</div>
<p>CD11b+ cells were positive for Iba1 and negative for GFAP and NeuN, whereas CD11b- cells were negative for Iba1 and positive for GFAP and NeuN.</p>
</figcaption>
</figure>
<p>When culturing CD11b+ cells, we often treat and collect cells within 24 hr. However, we have been able to successfully culture these isolated cells for up to one week following isolation. We have not attempted to culture these cells longer-term, and this application would need to be optimized prior to use.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<h3>Warm versus Cold Tissue Dissociation</h3>
<p>Although our lab has tried to implement microglia isolations using mechanical dissociation and maintaining cells at 4°C, we have encountered several limitations using this technique. First, despite many attempts, we are unable to attain a comparable yield between cold dissociations and warm enzymatic digestions. Warm enzymatic digestion still results in 2–3 times as many cells compared to cold mechanical dissociation. Second, when using panning methods to isolate microglia, removing microglia cells from the dish is difficult and the force required to remove the cells can negatively impact cell health. Third, cell viability is compromised limiting the ability to perform single cell sequencing. The tools/kits to eliminate dead cells either do not perform well or further reduce yield, thereby limiting an already small output. Lastly, cells maintained at 4°C appear less healthy (<a href="#fig10">Figure 10</a>). When centrifuging cells, each centrifugation from cold microglia yields more cellular debris and results in more clumping when producing single cell suspensions. In sum, microglia isolations using mechanical dissociation at 4°C, produce smaller yields with lower viability, and the cells are less healthy and more likely to clump, reducing the ability to perform state of the art single cell analysis (<a href="#fig10">Figure 10</a>).</p>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Cold Dissociation Results in Lowered Cell Viability and Fewer Single Cells than Does Warm Dissociation</div>
<p>(A and B) Gating strategy for Live-Dead Blue staining from (A) cold dissociation isolated CD11b+ cells and (B) warm dissociation isolated CD11b+ cells.</p>
<p>(C and D) Gating strategy for determining single cells from (C) cold dissociation isolated CD11b+ cells and (D) warm dissociation isolated CD11b+ cells.</p>
</figcaption>
</figure>
<h3>Use of Embryonic Mouse Tissue</h3>
<p>In our hands, the Miltenyi Debris Removal Solution does not work well for embryonic mouse tissue. When the Debris Removal Solution was attempted for embryonic mouse brain tissue, after the spin in Step 12/at Step 13, no cell layer was visualized in the gradients. Although we moved forward with the workflow and resuspended the pellet, ultimately very few to no cells were recovered. If the Debris Removal Solution is primarily targeted toward myelin removal, the relative lack of myelin in embryonic mouse tissue might explain why the solution is not optimal for an embryonic mouse tissue application. We therefore recommend the use of Percoll gradients for embryonic mouse tissue and have reliable cell yield with this method.</p>
<h3>Microglia versus Other CD11b+ Cells</h3>
<p>Although there are numerous methods used to isolate microglia from the brain, the method of dissociation and isolation can have a profound effect upon the function of these highly dynamic cells. For instance, fluorescence activated cell sorting (FACS) can upregulate immediate early gene expression (<a href="#bib1">Ayata et al., 2018</a>; <a href="#bib6">Haimon et al., 2018</a>; <a href="#bib9">Li et al., 2019</a>), and has been found to induce transcription of markers classically associated with the microglial proinflammatory state (<a href="#bib1">Ayata et al., 2018</a>), while methods such as RiboTag that use magnetic bead separation do not have similar gene alterations (<a href="#bib1">Ayata et al., 2018</a>; <a href="#bib6">Haimon et al., 2018</a>). FACS sorting has also been shown to alter the cellular redox state and metabolome in sorted cells (<a href="#bib10">Llufrio et al., 2018</a>). Perhaps the most enticing reason to utilize FACS for microglial isolation is the ability to further distinguish microglia from infiltrating immune cells. Transcardiac saline perfusion will eliminate any circulating CD11b+ cells, but any CD11b+ cells that are not microglia but are residing in the brain will be selected. A potential solution to this limitation could be to utilize the Miltenyi Multisort kits. For instance, the CD11b microbeads used in step 15 (Miltenyi #130-093-634) could be replaced by APC-tagged CD11b beads (Miltenyi #130-109-364), and the protocol for the Anti-APC MultiSort Kit (Miltenyi #130–091-255) could be followed. Central microglia vs peripheral immune cells could then be further sorted by a secondary antibody such as Cx3cr1. For instance, PE-tagged Cx3cr1 antibody (Biolegend #149005) coupled to anti-PE beads (Miltenyi #130-048-801) could be utilized to magnetically sort cells with CD11b and Cx3cr1. Alternatively, other secondary antibodies could be used to selectively isolate cells of interest without the confounds of FACS.</p>
<h3>Protein Mass Spectrometry Using Post-mortem Human Brain Tissue</h3>
<p>We were able to successfully isolate microglia from human post-mortem brain regions using this method, as confirmed by positivity for Iba1 and negativity for GFAP and NeuN by immunoblot (see <a href="#fig9">Figure 9</a>). However, users may struggle to successfully perform proteomic analyses in isolated CD11b+ cells using this method. Two main limitations that we encountered during these studies were (a) the relatively low protein abundance in the isolated CD11b+ cells obtained from 300 mg of human brain tissue and (b) the high levels of bovine serum albumin (BSA) contamination within the sample. The source of the BSA in our samples is the MACS Buffer (1%, see above in described recipe) used during CD11b bead incubation and LS column steps. BSA, a protein widely used for routine experiments is known to be a common contaminant for proteomics due to both its molecular size and its abundance (<a href="#bib8">Hodge et al., 2013</a>; <a href="#bib2">Bell et al., 2009</a>; <a href="#bib11">Pellitteri-Hahn et al., 2006</a>). In general, high levels of protein contamination can cause the loss of low abundance proteins given the large peaks from highly abundant proteins with similar size (<a href="#bib8">Hodge et al., 2013</a>). In our experimental conditions, the presence of large amounts of BSA (up to 90% of the intensity measured in the raw MS data) masked the protein peaks during Mass Spec analysis. Thus, the low protein abundance of CD11b+ cells together with the presence of BSA as a contaminant of the sample negatively affected proteomics analysis. Optimization of this protocol, perhaps with further wash steps to further rid sample of BSA, would be necessary before performing protein mass spectrometry analyses.</p>
<h3>Use of Fetal Bovine Serum</h3>
<p>We use 5% fetal bovine serum (FBS) in our enzyme digestion mix. Serum has been demonstrated to activate microglia <i>in</i> vitro, as serum-exposed microglia exhibit less ramified morphologies and significantly altered gene expressions (<a href="#bib3">Bohlen et al., 2017</a>, <a href="#bib4">2019</a>). Although we do include FBS in our enzyme digestion mix, our previously published data using this method indicate negligible activation of classical proinflammatory cytokines. For instance, isolated microglia treated with vehicle <i>in vitro</i> demonstrated low levels of IL-1b and TNF-a which was significantly stimulated by treatment with LPS (<a href="#bib12">Rivera et al., 2019</a>), and embryonic mouse brain and placental CD11b+ cells showed significant stimulation of TNF-a release following LPS challenge (<a href="#bib5">Edlow et al., 2019</a>). ELISA run on cultured CD11b+ and CD11b- cells isolated from embryonic mouse tissue show low levels of TNF-α release without LPS stimulation, and a significant stimulation of TNF-α production following LPS treatment (<a href="#fig11">Figure 11</a>). Although our data indicates negligible activation of microglia from the FBS contained in our enzyme digestion mix, the presence of FBS must be considered as a potential limitation of this method.</p>
<figure id="fig11"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Fig11.jpg" alt="Fig11.jpg">
<figcaption>
<div class="figcaption-title">Figure 11. Low Level of Basal Cytokine Release by Placental CD11b+ Cells</div>
<p>Protein concentrations for TNF-α released into culture media by CD11b- or CD11b+ cells following 2 hr lipopolysachharide (LPS) or vehicle (Veh) stimulation were determined by ELISA.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem 1</h3>
<p>70% Percoll underlay does not drain out of the unflamed Pasteur pipet tip.</p>
<h3>Potential Solution</h3>
<p>Gently lift the tip of the unflamed Pasteur pipet from the bottom of the falcon tube and place back down. Doing this several times should allow the air bubbles to escape from the bottom of the tip, and the 70% Percoll will subsequently drain.</p>
<h3>Problem 2</h3>
<p>Difficulty visualizing the interphase layer at step 13 while pipetting (critical for pipetting out the microglia, which will be the only cellular material moving forward from this point).</p>
<h3>Potential Solution</h3>
<p>We recommend placing the tube such that the interphase layer is at eye level. The layer is most easily visualized before any cells are removed. Cells should be removed by pipetting in circles just above the 30/70 gradient. If 70% Percoll from below the interphase layer is accidentally obtained, a second gradient after the subsequent spin (step 14) may form. If this happens, see Problem 4 and Solution.</p>
<h3>Problem 3</h3>
<p>Thin or non-robust interphase cell layer in the Percoll gradients.</p>
<h3>Potential Solution</h3>
<p>We have found that the robustness of Percoll gradients and the cell yield following Step 10 is sensitive to specific centrifuges and spin times. A centrifuge with swinging baskets is critical for Step 10, as is the use of zero brake and slow acceleration. The spin, acceleration, and brake settings are optimized for the Eppendorf 5810R centrifuge with A-4-81 rotor and swinging bucket attachments, and may need to be altered for other centrifuge/rotors.</p>
<h3>Problem 4</h3>
<p>A second Percoll gradient is visualized after the spin in step 14.</p>
<h3>Potential Solution</h3>
<p>If this occurs, a band of cells will be visible near the bottom of the tube above the pellet which contains the majority of your cells for subsequent steps. This occurs when 70% Percoll was included in the 4 mLs of the interphase cell layer. This can be avoided by errring on the side of staying above 70% Percoll when pipetting out the interphase layer. To minimize cell loss if this second gradient does occur, we harvest the cells in this second gradient. To do this, aspirate the supernatant to just above the visible cell layer (typically several mm above the pellet), and pipet out the second interphase layer in a circular motion as described in step 12 (only one time). Then aspirate off the remainder of the supernatant above the pellet (may help to tilt tube at this point), and add the cells from the second interphase to the pellet before moving forward. For example of layer removal, see <a href="#mmc8">Video S8</a>.</p>
<figure id="mmc8"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/81-Mmc8.mp4" type="video/mp4"></source>
        Your browser does not support HTML5 video.
      </video>
<figcaption>
<div class="figcaption-title">Video S8. Removal of Second Percoll Gradient</div>
<p>Supernatant is aspirated, cell layer is sucked into pipet and saved, lower supernatant is aspirated, and cells are placed back with other pelleted cells.</p>
</figcaption>
</figure>
<h3>Problem 5</h3>
<p>Lower than expected cell yield.</p>
<h3>Potential Solution</h3>
<p>We have identified the following points at which unexpected cell loss can occur:</p>
<ol>
<li>Step 10 if using Percoll gradients (see above regarding Solution for thin/non-robust interphase layer after the 20 min spin)</li>
<li>Step 6: If the filter is not pre-wet properly and the pre-wet area extends over the entire surface of the filter, the added cell suspension can track around the top of the falcon tube and cell loss can occur.</li>
<li>Steps 19–24: If performing the ‘Molecular Protocol’ that does not utilize Percoll or Debris Removal Solution, it is possible that large amounts of tissue from older animals can cause the column to clog up from excess myelination. While we have not experienced this using tissue from postnatal day 0 to postnatal day 60 mice, using tissue from older animals may cause this issue. If this occurs, we suggest utilization of either Percoll or Debris Removal Solution protocols.</li>
<li>Step 28 and Step 32: At these point s when the pellet can be difficult to visualize, care must be taken when aspirating supernatants to leave sufficient supernatant covering the bottom of the tube if pellet is unclear, and to keep the falcon or eppendorf tube slightly tilted to the same side throughout aspiration to avoid disturbing the pellet.</li>
</ol>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ayata, P., Badimon A, Strasburger HJ, Duff MK, Montgomery SE, Loh YE, Ebert A, Pimenova AA, Ramirez BR, Chan AT, et al. (2018) Epigenetic regulation of brain region-specific microglia clearance activity, Nat. Neurosci., <i>21</i>, pp. 1049-1060.</p>
<p id="bib2">Bell, A. W., , Deutsch EW, Au CE, Kearney RE, Beavis R, Sechi S, Nilsson T, Bergeron JJ. (2009) A HUPO test sample study reveals common problems in mass spectrometry-based proteomics, Nat. Methods, <i>6</i>, pp. 423-430.</p>
<p id="bib3">Bohlen, C. J., Bennett FC, Tucker AF, Collins HY, Mulinyawe SB, Barres BA. (2017) Diverse requirements for microglial survival, specification, and function revealed by defined-medium cultures, Neuron, <i>94</i>, pp. 759-773.e8.</p>
<p id="bib4">Bohlen, C. J., Bennett, F. C. and Bennett, M. L. (2019) Isolation and culture of microglia, Curr. Protoc. Immunol., <i>125</i>, e70.</p>
<p id="bib5">Edlow, A. G., Glass RM, Smith CJ, Tran PK, James K, Bilbo S. (2019) Placental macrophages: a window into fetal microglial function in maternal obesity’, Int. J. Dev. Neurosci.: the official journal of the International Society for Developmental Neuroscience, <i>77</i>, pp. 60-68.</p>
<p id="bib6">Haimon, Z., Volaski A, Orthgiess J, Boura-Halfon S, Varol D, Shemer A, Yona S, Zuckerman B, David E, Chappell-Maor L, et al. (2018) Re-evaluating microglia expression profiles using RiboTag and cell isolation strategies, Nat. Immunol., <i>19</i>, pp. 636-644.</p>
<p id="bib7">Hanamsagar, R., Alter MD, Block CS, Sullivan H, Bolton JL, Bilbo SD. (2017) Generation of a microglial developmental index in mice and in humans reveals a sex difference in maturation and immune reactivity, Glia, <i>65</i>, pp. 1504-1520.</p>
<p id="bib8">Hodge, K., Have ST, Hutton L, Lamond AI. (2013) Cleaning up the masses: exclusion lists to reduce contamination with HPLC-MS/MS, J. Proteomics, <i>88</i>, pp. 92-103.</p>
<p id="bib9">Li, Q., Cheng Z, Zhou L, Darmanis S, Neff NF, Okamoto J, Gulati G, Bennett ML, Sun LO, Clarke LE, et al. (2019) Developmental heterogeneity of microglia and brain myeloid cells revealed by deep single-cell RNA sequencing, Neuron, <i>101</i>, pp. 207-223.e10.</p>
<p id="bib10">Llufrio, E. M., Wang L, Naser FJ, Patti GJ. (2018) Sorting cells alters their redox state and cellular metabolome, Redox Biol., <i>16</i>, pp. 381-387.</p>
<p id="bib11">Pellitteri-Hahn, M. C., Warren MC, Didier DN, Winkler EL, Mirza SP, Greene AS, Olivier M. (2006) Improved mass spectrometric proteomic profiling of the secretome of rat vascular endothelial cells, J. Proteome Res., <i>5</i>, pp. 2861-2864.</p>
<p id="bib12">Rivera, P. D., Hanamsagar R, Kan MJ, Tran PK, Stewart D, Jo YC, Gunn M, Bilbo SD. (2019) Removal of microglial-specific MyD88 signaling alters dentate gyrus doublecortin and enhances opioid addiction-like behaviors, Brain Behav. Immun., <i>76</i>, pp. 104-115.</p>
</section>
<section>
<h2 id="q-and-a">Q&amp;A</h2>
<p class="questions"><b>Q:</b> I've attempted the Percoll gradient version of this protocol a few times and have encountered a lower than expected yield. For example, using 150mg of forebrain tissue, my final cell count has been ~40k CD11b+ cells. The cells obtained are IBA1+, while the CD11b- fraction is IBA1-. I assume microglia are being isolated correctly, but that I am losing a large fraction of them in an earlier step. Compared to Figure 5, it looks like the myelin layer is too thick while the cell interface is much too thin in my samples. I am using a Thermo Sorvall Legend XTR Swinging bucket centrifuge at 340g and I would assume this centrifuge is similar to the Eppendorf 5810R, but with such low yield would you recommend increasing the speed? Alternately, is there another step I might have overlooked? Thanks!</p>
<p class="answer"><b>A:</b> Here are two potential troubleshooting areas for this question:</p>
<ol>
<li>Are you able to confirm that your dissociation steps are resulting in a single cell suspension? Cell clumps may partition differently than a single cell suspension. We have noted that incomplete dissociation can lead to microglia getting ‘stuck’ in another layer – thereby decreasing the cell yield.</li>
<li>We have found that proper placement of the pipet tip to remove the cell interface layer following centrifugation is key. We find this to be the most variable step in the protocol, as it is quite easy not to pull up all of the cells contained in the layer.</li>
</ol>
<p class="questions"><b>Q:</b> Thanks to the authors for getting back to me so quickly. I do have a brief follow up to the first part of your response:</p>
<ol>
<li>For the dissociation step, shouldn’t passing the tissue homogenate through a 70µm cell strainer produce a single cell suspension? Would you recommend checking the suspension with a hemocytometer or something similar before proceeding with the Percoll step?”</li>
</ol>
<p class="answer"><b>A:</b> The tissue homogenate is passed through a cell strainer following a dissociation step (such as our Step #5 using flame-polished Pasteur pipets, or using a tissue dissociator such as the Miltenyi GentleMACS) to reach a single cell level. We would recommend checking the suspension either with a hemocytometer or flow cytometry as you are piloting this protocol to ensure that the dissociation methodology is achieving a single cell suspension in your hands.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Supplemental Information</h3>
<p>Supplemental Information can be found online at <a href="https://doi.org/10.1016/j.xpro.2020.100035">https://doi.org/10.1016/j.xpro.2020.100035</a>.</p>
<h3>Acknowledgements</h3>
<p>We thank Haley Norris for technical assistance. This work was supported by NIEHS R01 ES025549 to S.D.B., NIMH F32 MH116604 to E.A.B., and NICHD R01-HD100022 to A.G.E.</p>
<h3>Author Contributions</h3>
<p>E.A.B. and S.D.B. conceived the manuscript. E.A.B., C.L.B., T.P., G.S-V., C.J.S., A.G.E., and S.D.B. wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>