<article>
<section>
<h1 id="header">Utilizing thermal proteome profiling to identify the molecular targets of anti-leishmanial compounds</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Victoriano Corpas-Lopez<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Susan Wyllie<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Division of Biological Chemistry and Drug Discovery, Wellcome Centre for Anti-infectives Research, School of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, UK</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:v.corpaslopez@dundee.ac.uk">v.corpaslopez@dundee.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:s.wyllie@dundee.ac.uk">s.wyllie@dundee.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100704">10.1016/j.xpro.2021.100704</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we detail our optimized protocol for the identification of drug targets in <i>Leishmania donovani</i> using thermal proteome profiling. This approach is based on the principle that binding of a drug to its protein target can significantly alter the thermal stability of that protein. By monitoring changes in the thermal stability of proteins within drug-treated and untreated cell lysates, using mass spectrometry combined with tandem mass tag labeling, putative targets of the drug can be identified in an unbiased manner.</p>
<p>For further details on the use and application of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00411-1/sref5">Paradela et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Unbiased approach to identify the molecular targets of anti-leishmanial drugs</li>
<li>Targets identified by monitoring changes in the thermal stability of proteins +/- drug</li>
<li>Protocol optimized for the study of <i>Leishmania donovani</i>
</li>
<li>Protocol can be easily adapted to study other pathogens</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/948-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps for TPP analysis of <i>Leishmania donovani</i> LdBOB promastigotes (derived from MHOM/SD/62/1S-CL2D). However, we have also used this protocol to investigate compound interactions in other kinetoplastid parasites, specifically <i>Trypanosoma brucei</i> and <i>Trypanosoma cruzi</i>. Prior to beginning the study, prepare 1 L of LdBOB promastigote media and pre-warm to room temperature (RT; 20°C–25°C).</p>
<p>A rationalized version of this protocol, without compound pre-treatment of parasites or lysate preparation in the presence of detergent, has also been published previously (<a href="#bib3">Corpas-Lopez et al., 2019</a>).</p>
<h3 id="sec1.1">
<i>L. donovani</i> promastigote culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol>
<li>Inoculate 1 L of LdBOB media with <i>L. donovani</i> (LdBOB) promastigotes at 1 × 10<sup>5</sup> cells mL<sup>−1</sup>. Split the culture into two Corning roller bottles (500 mL each). Incubate these bottles in a humidified roller bottle incubator with rotation at 28°C for 72 h or until cultures reach mid-logarithmic (mid-log) growth. Roller bottles are recommended for cultures of this size since the constant rotation of flasks provides optimal aeration.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Monitor cell density in roller bottle cultures daily and do not allow cells to reach stationary phase (∼4 × 10<sup>7</sup> mL<sup>−1</sup>). For LdBOB promastigotes in our media, mid-log growth is considered between 8 × 10<sup>6</sup>–2 × 10<sup>7</sup>cells mL<sup>−1</sup>. It should be noted that mid-log growth may vary for different species of <i>Leishmania</i> and/or culture medium.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Use conventional cell culture flasks instead of roller bottles. In this case, cultures may need to be separated into multiple flasks to ensure sufficient aeration and optimal parasite growth</div>
<h3 id="sec1.2">Prepare and pre-chill material</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="2">
<li>Prepare Lysis Buffer (10 mL; see the detailed recipe in “<a href="#materials-and-equipment">materials and equipment</a>”) and store at 4°C until required.</li>
<li>Pre-chill the ultracentrifuge, rotors, tubes and LoBind tubes to 4°C.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Potassium phosphate monobasic</td>
<td>Sigma</td>
<td>P5655</td>
</tr>
<tr>
<td>Potassium phosphate dibasic</td>
<td>Sigma</td>
<td>P3786</td>
</tr>
<tr>
<td>Ethylenediaminetetraacetic acid (EDTA)</td>
<td>Sigma</td>
<td>E9884</td>
</tr>
<tr>
<td>1,4-Dithiothreitol (DTT)</td>
<td>Sigma</td>
<td>10708984001</td>
</tr>
<tr>
<td>M199 media</td>
<td>Sigma</td>
<td>M0393</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Fisher Scientific</td>
<td>12350273</td>
</tr>
<tr>
<td>Sodium bicarbonate</td>
<td>Sigma</td>
<td>S5761</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>Invitrogen</td>
<td>35050087</td>
</tr>
<tr>
<td>Folic acid</td>
<td>Sigma</td>
<td>F8758</td>
</tr>
<tr>
<td>Adenosine</td>
<td>Sigma</td>
<td>A9251</td>
</tr>
<tr>
<td>Hemin</td>
<td>Sigma</td>
<td>51280</td>
</tr>
<tr>
<td>Lysyl Endopeptidase®, Mass Spectrometry Grade</td>
<td>Alpha Labs (Wako)</td>
<td>125-02543</td>
</tr>
<tr>
<td>Trichloroacetic acid solution 6.1 N</td>
<td>Sigma-Aldrich</td>
<td>T0699</td>
</tr>
<tr>
<td>cOmplete™, Mini, EDTA-free Protease Inhibitor Cocktail</td>
<td>Roche</td>
<td>11836170001</td>
</tr>
<tr>
<td>Octyl β-D-glucopyranoside</td>
<td>Sigma-Aldrich</td>
<td>O8001</td>
</tr>
<tr>
<td>Nα-Tosyl-L-lysine chloromethyl ketone hydrochloride (TLCK)</td>
<td>Sigma-Aldrich</td>
<td>T7254</td>
</tr>
<tr>
<td>Acetone ≥99.8%, AnalaR NORMAPUR® ACS</td>
<td>VWR</td>
<td>20066.330</td>
</tr>
<tr>
<td>Pierce™ Acetonitrile (ACN), LC-MS Grade</td>
<td>Thermo</td>
<td>51101</td>
</tr>
<tr>
<td>Formic acid, LC-MS Grade</td>
<td>Thermo</td>
<td>85178</td>
</tr>
<tr>
<td>Acetonitrile, HPLC LC-MS Grade</td>
<td>VWR</td>
<td>20J141963</td>
</tr>
<tr>
<td>Trifluoroacetic acid, HPLC Grade</td>
<td>Fisher</td>
<td>T/3258/04</td>
</tr>
<tr>
<td>Ammonia solution, OPTIMA Grade</td>
<td>Fisher</td>
<td>A470-250</td>
</tr>
<tr>
<td>Methanol (OPTIMA LC/MS)</td>
<td>Fisher</td>
<td>A456-1</td>
</tr>
<tr>
<td>Ammonium formate</td>
<td>Sigma</td>
<td>70221-100G-F</td>
</tr>
<tr>
<td>pH-indicator strips pH 0–14 Universal indicator</td>
<td>Sigma</td>
<td>1095350001</td>
</tr>
<tr>
<td>Dimethyl sulfoxide, molecular biology grade</td>
<td>Sigma</td>
<td>D8418</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>TMT10plex™ Isobaric Mass Tagging Kit</td>
<td>Thermo</td>
<td>90111</td>
</tr>
<tr>
<td>Tris(2-carboxyethyl)phosphine (TCEP)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Iodoacetamide (IAA)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Triethylamonium bicarbonate (TEAB)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Trypsin</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Quenching reagent (50% hydroxylamine)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Pierce Peptide Desalting Spin Columns</td>
<td>Thermo</td>
<td>89852</td>
</tr>
<tr>
<td>Quick Start™ Bradford Protein Assay</td>
<td>Bio-Rad</td>
<td>5000201</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>Leishmania donovani</i> LdBOB (MHOM/SD/62/1S-CL2D)</td>
<td>(<a href="#bib9">Goyard et al., 2003</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Thermo Xcalibur<sup>TM</sup>
</td>
<td>Thermo</td>
<td>Version 4.0.27.19</td>
</tr>
<tr>
<td>Chromeleon<sup>TM</sup> Software</td>
<td>Thermo</td>
<td>Version 4.0.27.19</td>
</tr>
<tr>
<td>MaxQuant version 1.6.17.0</td>
<td><a href="#bib7">Tyanova et al., 2016</a></td>
<td>Maxquant.org</td>
</tr>
<tr>
<td>Analyze thermal proteome profiling (TPP) experiments. R package version 3.18.0</td>
<td><a href="#bib2">Childs et al., 2020</a></td>
<td><a href="https://bioconductor.org/packages/release/bioc/html/TPP.html">https://bioconductor.org/packages/release/bioc/html/TPP.html</a></td>
</tr>
<tr>
<td>Script to analyze MaxQuant results in TPP</td>
<td>Victoriano Corpas-Lopez</td>
<td><a href="https://github.com/vcorpaslopez/StarProtocolsTPP">https://github.com/vcorpaslopez/StarProtocolsTPP</a></td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>RAW and search files of the project</td>
<td>PRIDE repository</td>
<td><a href="https://www.ebi.ac.uk/pride/archive/projects/PXD023780">https://www.ebi.ac.uk/pride/archive/projects/PXD023780</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Tabletop Optima TLX Ultracentrifuge</td>
<td>Beckman</td>
<td>TLX</td>
</tr>
<tr>
<td>One-Shot Cell Disruptor</td>
<td>Constant Systems</td>
<td>OS</td>
</tr>
<tr>
<td>TLA 100 Fixed-angle rotor</td>
<td>Beckman</td>
<td>343840</td>
</tr>
<tr>
<td>TLA 120.0 Fixed-angle rotor</td>
<td>Beckman</td>
<td>362046</td>
</tr>
<tr>
<td>Sonicator Elmasonic P60H</td>
<td>Elma</td>
<td>101 3761</td>
</tr>
<tr>
<td>Vibrax VXR basic shaker</td>
<td>IKA</td>
<td>0002819002</td>
</tr>
<tr>
<td>Genevac EZ-2 plus evaporator</td>
<td>SP Scientific</td>
<td>EZ3P-23050-HP0</td>
</tr>
<tr>
<td>Dionex UHPLC Ultimate 3000, WPS-3000FC Autosampler</td>
<td>Thermo Scientific</td>
<td>5825.0020</td>
</tr>
<tr>
<td>C18 column XBridge Peptide BEH, 130 Å, 3.5 μm, 2.1 × 150 mm</td>
<td>Waters</td>
<td>186003565</td>
</tr>
<tr>
<td>Guard column XBridge, C18, 3.5 μm, 2.1 × 10 mm</td>
<td>Waters</td>
<td>186003019</td>
</tr>
<tr>
<td>PepMap nanoViper C18 column, 100 μm × 2 cm, 5 μm, 100 Å</td>
<td>Thermo Scientific</td>
<td>164564-CMD</td>
</tr>
<tr>
<td>PepSwift Monolithic Nano resolving column</td>
<td>Thermo Scientific</td>
<td>164584</td>
</tr>
<tr>
<td>Thermo Q Exactive Plus Orbitrap System coupled to a Dionex Ultimate 3000 RS</td>
<td>Thermo Scientific</td>
<td>IQLAAEGAAPFALGMBDK</td>
</tr>
<tr>
<td>Thermo Q Exactive HF Orbitrap System coupled to a Dionex Ultimate 3000 RS</td>
<td>Thermo Scientific</td>
<td>IQLAAEGAAPFALGMBFZ</td>
</tr>
<tr>
<td>Resolving column, 75 μm <b>×</b> 50 cm, PepMap RSLC C18 column, 2 μm, 100 Å</td>
<td>Thermo Scientific</td>
<td>164540</td>
</tr>
<tr>
<td>Corning 1700 cm<sup>2</sup> Roller Bottles, Tissue Culture Treated</td>
<td>Corning</td>
<td>CLS430852</td>
</tr>
<tr>
<td>Protein LoBind® tubes (0.5, 1.5 and 2 mL)</td>
<td>Eppendorf</td>
<td>0030108434, 0030108442, 0030108450</td>
</tr>
<tr>
<td>Open-top 230 μL thickwall polycarbonate centrifuge tubes 7 <b>×</b> 20 mm</td>
<td>Beckman</td>
<td>343775</td>
</tr>
<tr>
<td>Open-top 1 mL thickwall polycarbonate tubes, 11 <b>×</b> 34 mm</td>
<td>Beckman</td>
<td>342778</td>
</tr>
<tr>
<td>1.5 mL Collection tubes</td>
<td>SARSTEDT</td>
<td>72.696</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Suggested equipment and material</h3>
<p>One-Shot cell disruptor (Constant Systems, OS)</p>
<p>Tabletop Optima TLX Ultracentrifuge (Beckman, TLX)</p>
<p>Rotors TLA 100 Fixed angle rotor (Beckman, 343840) and TLA 120.0 Fixed angle rotor (Beckman, 362046).</p>
<p>Sonicator Elmasonic P60H (Elma, 101 3761)</p>
<p>Vibrax VXR basic shaker (IKA, 0002819002)</p>
<p>Genevac EZ-2 plus evaporator (SP Scientific, EZ3P-23050-HP0)</p>
<p>Dionex UHPLC Ultimate 3000, WPS-3000FC autosampler (Thermo Scientific, 5825.0020)</p>
<p>C18 column XBridge peptide BEH, 130 Å, 3.5 μm, 2.1 × 150 mm (Waters, 186003565)</p>
<p>Guard column XBridge, C18, 3.5 μm, 2.1 × 10 mm (Waters, 186003019)</p>
<p>PepMap nanoViper C18 column, 100 μm × 2 cm, 5 μm, 100 Å (Thermo Scientific, 164564-CMD)</p>
<p>Pepswift Monolithic Nano resolving column (Thermo Scientific, 164584)</p>
<p>Thermo Q Exactive Plus Orbitrap System coupled to a Dionex Ultimate 3000 RS (Thermo Scientific, IQLAAEGAAPFALGMBDK)</p>
<p>Thermo Q Exactive HF Orbitrap System coupled to a Dionex Ultimate 3000 RS (Thermo Scientific, IQLAAEGAAPFALGMBFZ)</p>
<p>Resolving column, 75 μm <b>×</b> 50 cm, PepMap RSLC C18 column, 2 μm, 100 Å (Thermo Scientific, 164540)</p>
<p>Corning 1700 cm<sup>2</sup> roller bottles, tissue culture treated (Corning, CLS430852)</p>
<p>Protein LoBind® tubes (0.5, 1.5 and 2 mL) (Eppendorf, 0030108434, 0030108442, 0030108450)</p>
<p>Open-top 230 μL thickwall polycarbonate centrifuge tubes 7 × 20 mm (Beckman, 343775)</p>
<p>Open-top 1 mL thickwall polycarbonate tubes, 11 <b>×</b> 34 mm (Beckman, 342778)</p>
<p>1.5 mL collection tubes (Sarstedt, 72.696)</p>
<p>Choice of mass spectrometer: This TPP protocol relies on TMT quantitation, therefore a successful TPP analysis would then need an instrument compatible with this technology. We use a Q Exactive HF instrument, but other machines can be used as long as their MS/MS resolution is &gt; 50,000 at 150 m/z and can perform higher-energy collision dissociation (HCD). Suggested equipment includes Orbitrap Velos Pro, Q Exactive, Q Exactive Plus, Orbitrap Elite, Orbitrap Fusion and Orbitrap Eclipse (as recommended in the TMT 10plex instructions).</p>
<table id="undtbl1">
<caption>Lysis Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Mono-basic potassium phosphate, 200 mM stock</td>
<td>50 mM</td>
<td>1.0 mL</td>
</tr>
<tr>
<td>Di-basic potassium phosphate, 200 mM stock</td>
<td>50 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>1 mM</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>1 M DTT</td>
<td>1 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>Protease inhibitor cocktail mini tablet, EDTA-free</td>
<td> </td>
<td>1 tablet</td>
</tr>
<tr>
<td>10 mM Tosyl-L-lysyl-chloromethane hydrochloride (TLCK)</td>
<td>100 μM</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>n-octyl-β- d-glucoside</td>
<td>0.8%</td>
<td>80 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>7 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>-</td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> use freshly prepared buffer and keep on ice.</div>
<table id="undtbl2">
<caption>LdBOB Promastigote Media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>M199 media</td>
<td>-</td>
<td>1 sachet</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>100 mL</td>
</tr>
<tr>
<td>Sodium bicarbonate</td>
<td>1 g/L</td>
<td>1 g</td>
</tr>
<tr>
<td>Glutamax (100× stock)</td>
<td>1×</td>
<td>10 mL</td>
</tr>
<tr>
<td>Folic acid</td>
<td>4.4 mg/L</td>
<td>4.4 mg</td>
</tr>
<tr>
<td>Adenosine</td>
<td>27 mg/L</td>
<td>27 mg</td>
</tr>
<tr>
<td>2.5 mg/mL Hemin</td>
<td>10 mg/L</td>
<td>4 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>900 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>-</td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> store at 4°C for a maximum of 1 month.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Incubation of <i>L. donovani</i> promastigotes with test compounds</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5 h</div>
<p>This section details pre-incubation of <i>L. donovani</i> promastigotes with test compound prior to sample processing. Exposing parasites to test compounds for a short period, prior to processing, enables the compound to interact with its target in a native conformation, potentially increasing the chances of thermal stabilization of the target.</p>
<ol>
<li>Add the test compound to one of the roller-bottle cultures (treated sample, 500 mL) at a concentration equivalent to 10× its established EC<sub>50</sub> value. It should be noted that compound stocks must be prepared so that the vehicle (commonly DMSO) does not exceed a final concentration of 0.5% in treated cultures. An equivalent volume of the diluent should be added to the second roller-bottle (control sample, 500 mL).</li>
<li>Treated and untreated cultures should be returned to the incubator at 28°C for a further 3 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The EC<sub>50</sub> value of the test compound should be accurately determined prior to the study. Adding test compound in excess of 10× EC<sub>50</sub> value could result in parasite cell death and lysis with the concomitant release of damaging proteases that will affect the sample quality. Incubation with excessively high compound concentrations may increase the likelihood of off-target compound interactions. Incubation with test compounds at lower concentrations may lead to sub-optimal thermal stabilization of specific targets.</div>
<h3 id="sec3.2">Generation of cell lysate</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This protocol describes how to prepare cell lysates suitable for subsequent TPP studies.</p>
<ol start="3">
<li>Centrifuge (1920 × <i>g</i> for 15 min at 4°C) the 500 mL treated and control cultures of <i>L. donovani.</i> Wash the resulting cell pellets (0.2–0.5 mL expected pellet volume) with the equivalent of 5 pellet volumes of ice-cold PBS and then centrifuge (1920 × <i>g</i> for 15 min at 4°C). It should be noted that the PBS used for cell pellet washes should be spiked with either test compound or vehicle at concentrations equivalent to those used in pre-incubations.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to maintain the presence of the test compound (in the treated sample) or diluent (in the control) throughout cell pellet washes. This ensures that target-ligand interactions made during the initial incubation are maintained.</div>
<ol start="4">
<li>Resuspend washed parasite pellets in 2 mL ice-cold lysis buffer (again supplemented with test compound at a concentration equivalent to 10× its established EC<sub>50</sub> value or diluent) and transfer to LoBind tubes (5 mL).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> At this stage it is vital to use LoBind tubes to prevent loss of protein from the samples due to non-specific binding to plastic. It is also important to maintain samples on ice.</div>
<ol start="5">
<li>Lyse cells under pressure using the Constant Systems cell disruptor. It should be noted that other methods of cell lysis, such as sonication or nitrogen cavitation, may be equally suitable.
<ol type="a">
<li>Pre-chill the cell disruptor lysis chamber to 4°C.</li>
<li>Lyse the control sample at a pressure of 30 kpsi.</li>
<li>Wash the cell disruptor lysis chamber 2× with milliQ H<sub>2</sub>O (5 mL) and 1× with lysis buffer (5 mL) to prevent cross contamination of samples.</li>
<li>Then lyse the treated sample at 30 kpsi.</li>
</ol>
</li>
<li>Transfer cell lysates into 1 mL tubes suitable for ultracentrifugation and centrifuge (100,000 × <i>g</i> for 20 min at 4°C). Recover the supernatant from centrifuged samples and transfer into a fresh 2 mL LoBind tube. Do not disturb the pellet when recovering the supernatants. The pellets can now be discarded.</li>
<li>Accurately determine protein concentration of the supernatants using a standard Bradford or similar protein assay. Adjust the protein concentration of each sample (treated and control) to 1–1.5 mg/mL using ice-cold Lysis Buffer. The protein yield required for a single TPP experiment is 4 mg. Using these culture and lysis conditions, the protein yield should be 6–10 mg per bottle. <a href="#troubleshooting">Troubleshooting 1</a>.</li>
</ol>
<h3 id="sec3.3">Thermal shift assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>This protocol describes the steps required to perform thermal shift assays. Here, treated and control lysates are exposed to a range of temperatures. The proteins within each sample denature at different points of the temperature gradient and ultimately precipitate. Precipitated proteins are then removed by ultracentrifugation and the soluble fraction is assessed.</p>
<ol start="8">
<li>Dispense the treated and control cell lysates into 10 × 0.5 mL LoBind tubes (100 μL aliquots in each tube, representing 100–150 μg of protein) making two series per condition (treated and control) each comprised of 10 samples. Keep these tubes on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Individual aliquots will be incubated at a single temperature point within the following temperature range: 33°C–69°C in 4°C increments.</div>
<div class="note">
<span class="note-title">Note:</span> A thermal cycler or gradient PCR machine can be used to achieve an accurate temperature gradient. If you do not have access to a gradient PCR machine, a standard thermal cycler can be used sequentially.</div>
<ol start="9">
<li>Submit the aliquots to the following temperatures: 15 min at room temperature (RT, 20°C–25°C), 3 min at the designated temperature point, 3 min at room temperature and then place on ice.</li>
<li>Transfer the contents to open-top thickwall 230-μL ultracentrifuge tubes (Beckman) and remove denatured proteins by ultracentrifugation (100,000 × <i>g</i>, 4°C for 20 min). Carefully harvest the supernatants and transfer to fresh 1.5 mL LoBind tubes. Discard the pellets.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is imperative that supernatants are harvested carefully and that they are not contaminated with material from the pellet. Do not attempt to recover all of the supernatant, we suggest that the maximum volume that can be safely recovered is 70 μL.</div>
<ol start="11">
<li>Measure protein concentration in each sample using the Bradford or similar protein concentration assay.</li>
<li>Plot the determined protein concentrations of each sample against the temperature each sample was exposed to. It is expected that as the exposure temperature increases the level of recovered protein should decrease.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The protein concentration of the sample exposed to the highest temperature in the range (in this case 69°C) should be &lt;20% of that in the sample exposed to the lowest temperature. <a href="#troubleshooting">Troubleshooting 2</a><b>.</b>
</div>
<h3 id="sec3.4">Sample processing and labeling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<p>This section describes the process of reducing, alkylating and denaturing samples. In addition, the process of digesting and labeling peptides with TMT 10-plex tags is detailed.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Trifluoroacetic acid (TFA) and formic acid (FA) are harmful when inhaled. They need to be used in an fume hood to minimize the risk of inhalation and disposed following the appropriate health and safety regulations.</div>
<ol start="13">
<li>To reduce each sample add 25–50 mM tris(2-carboxyethyl)phosphine hydrochloride (TCEP, 3.5 μL 1M TCEP) and incubate for 10 min at 37°C. Allow the samples to equilibrate to RT (20°C–25°C).</li>
<li>To alkylate proteins, add 25–50 mM iodoacetamide (IAA, 3.5 μL 1M IAA) to each sample and incubate for 1 h at RT (20°C–25°C) in the dark.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Some precipitation may be seen after incubation with IAA.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use a fresh IAA solution and preserve it from light. Foil can be used to protect alkylated samples and IAA solution from light.</div>
<ol start="15">
<li>To precipitate proteins, add 10.5 μL trichloroacetic acid 6.1 N (TCA, final concentration of 12%) to each sample, vortex briefly (∼5 s) and incubate samples at −20°C for a minimum of 3 h. Samples can be left at −20°C overnight (3–16 h).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Some samples may be frozen following overnight incubation. Allow frozen samples to melt on ice prior to the next step.</div>
<ol start="16">
<li>Centrifuge (20,000 × <i>g</i>, 4°C for 5 min) samples, then carefully discard the supernatants and harvest the pellets. Place the tubes containing the pellets on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Samples must be kept on ice during the acetone washes.</div>
<ol start="17">
<li>Add pre-chilled acetone (500 μL) to each pellet, then vortex (∼10 s) until the pellet is dislodged from the bottom of the tube. Now incubate at 4°C for 10 min with agitation (600 rpm) in a shaker.</li>
<li>Next, centrifuge (20,000 × <i>g</i>, 4°C for 5 min) each sample, carefully remove and dispose of the acetone. Repeat washes 2× with acetone.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During washes be careful not to disturb or aspirate the pellet. It is not essential to remove all the acetone since any excess can be removed by evaporation.</div>
<ol start="19">
<li>Finally, remove and dispose of the acetone and leave pellets to air dry at RT (20°C–25°C).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Acetone evaporation can be accelerated by incubation at 37°C.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this time, protein pellets can be safely stored at −20°C.</div>
<ol start="20">
<li>Resuspend the pellets in 100 mM tetraethylammonium bicarbonate pH 8.5 (TEAB, 500 μL). Vortex each sample vigorously (∼20 s) until pellets are dissolved.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Following resuspension of the pellet, check the pH of the sample is ≥8 using a pH indicator strip.</div>
<div class="note">
<span class="note-title">Note:</span> if the pellets are very stubborn and they do not dissolve, try sonication in a sonicating water bath for 5 min (37 kHz, 100% power, RT (20°C–25°C)) to help dissolve the pellet before adding the digestion enzymes.</div>
<ol start="21">
<li>In order to resuspend the desiccated vial of endoproteinase Lys-C, add 100 mM TEAB (20 μL) and shake gently to make a 1 mg/mL solution.</li>
<li>Add resuspended Lys-C solution to each sample at a 50:1 protein:enzyme ratio (1 μg of enzyme per 50 μg of protein). Incubate with agitation (600 rpm) at 37°C for 4–6 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use protein concentration quantified in step 11 to calculate the amount of Lys-C and Trypsin required (approximately 1.4–2.1 μg of enzyme per tube).</div>
<ol start="23">
<li>In order to resuspend the desiccated vial of trypsin, add 100 mM TEAB (20 μL) to the vial and shake gently to make a 1 mg/mL solution.</li>
<li>Add trypsin solution to the samples at a 50:1 protein:enzyme ratio. Incubate at 37°C with agitation (600 rpm) for between 8–16 h.</li>
<li>Vacuum dry the samples in an evaporator overnight (8–16 h) at 30°C using a program suitable for water-based samples (program “Aqueous” in the recommended evaporator).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> We recommend checking the digestion efficiency of some of the samples via mass spectrometry prior to TMT labelling.</div>
<ol start="26">
<li>To check the digestion efficiency, randomly select a few (2–4) samples representative of different temperatures (for example samples 1, 5 and 9) as they have different protein quantity.
<ol type="a">
<li>Resuspend these samples in 100 μL TEAB, then pick a 1 μL aliquot and dilute it 1:5 in 0.1% FA in a sample tube.</li>
<li>Use the following recommended equipment: Q Exactive Plus mass spectrometer coupled to a Dionex Ultimate 3000 RS (Thermo Scientific) equipped with a trap column (100 μm × 2 cm, PepMap nanoViper C18 column, 5 μm, 100 Å, Thermo Scientific) and a Pepswift Monolithic Nano resolving column.</li>
<li>Place each sample in the sample rack of the HPLC and load 5 μL sample at 10 μL min<sup>−1</sup> onto the trap column pre-equilibrated with Buffer C. Wash the trap column with Buffer C for 3 min at 10 μL min<sup>−1</sup>, then switch the trap column in line with the resolving column. Elute peptides at a constant flow rate of 700 nL min<sup>−1</sup> with a linear gradient from 2%–40% Buffer D over 7 min, and then from 40%–98% Buffer D over 1 min. Wash the column with 98% Buffer D for 1 min and re-equilibrate in 2% Buffer D for 5 min.</li>
<li>Use Q Exactive Plus in data-dependent positive ion mode using the following setup:
<ol type="i">
<li>MS1 scan cycles m/z range from 335–1800, with a maximum ion injection time of 30 ms, a resolution of 70,000 and automatic gain control (AGC) value of 1 × 10<sup>6</sup>.</li>
<li>Followed by 10 sequential dependent MS2 scans with an isolation window set to 1.4 m/z, resolution at 17,500, maximum ion injection time at 100 ms and AGC 2 × 10<sup>5</sup>.</li>
</ol>
</li>
<li>Open the .raw files using Thermo Xcalibur<sup>TM</sup> (Qual browser). This should open two windows: the upper window showing a chromatogram of signal relative abundance over retention time and the lower window showing the relative abundance over m/z ratio.</li>
<li>By selecting peaks on the upper chromatogram, the lower window will display the relative abundance and m/z of that peak.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Check that the majority of peaks have a charge lower than 4 (z &lt;4). High abundance of multiply charged species indicate that digestion has been not accomplished and should be repeated.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Digested peptides can be stored at −20°C.</div>
<ol start="27">
<li>TMT 10plex labels are provided in sachets stored at −20°C. Remove 4 ×TMT 10plex sachets from storage at −20°C and allow to equilibrate to RT (20°C–25°C). Each sachet contains 10 tubes with 0.8 mg of the corresponding TMT 10plex label, enough to label 100 μg of protein.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is important that the 4× sachets should share the same batch number since there can be variability between batches.</div>
<ol start="28">
<li>Resuspend the dried peptide samples in 100 mM TEAB (100 μL) and incubate at RT (20°C–25°C) with agitation (600 rpm).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Check that the pH of samples is ≥8</div>
<ol start="29">
<li>Add acetonitrile (ACN, 41 μL) to each TMT tube and incubate with agitation (600 rpm) for 15 min at RT (20°C–25°C).</li>
<li>Briefly, centrifuge peptide samples and TMT tubes (3,000 × <i>g</i>, 10 s).</li>
<li>Add the content of each TMT tube to the corresponding peptide sample, as follows:</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th>Sample</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
<th>9</th>
<th>10</th>
</tr>
</thead>
<tbody>
<tr>
<td>Temperature (°C)</td>
<td>33</td>
<td>37</td>
<td>41</td>
<td>45</td>
<td>49</td>
<td>53</td>
<td>57</td>
<td>61</td>
<td>65</td>
<td>69</td>
</tr>
<tr>
<td>TMT label</td>
<td>126</td>
<td>127N</td>
<td>127C</td>
<td>128N</td>
<td>128C</td>
<td>129N</td>
<td>129C</td>
<td>130N</td>
<td>130C</td>
<td>131</td>
</tr>
</tbody>
</table>
<ol start="32">
<li>Incubate with agitation for 1 h at RT (20°C–25°C).</li>
<li>Dilute the quenching reagent (50% hydroxylamine solution) 10-fold in 100 mM TEAB, making a final 5% hydroxylamine solution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the quenching reagent appears precipitated, incubate at 37°C until the solution clears.</div>
<ol start="34">
<li>In order to quench the residual, unreacted TMTs, add 5% hydroxylamine solution (8 μL) to each reaction tube and incubate with agitation.</li>
<li>Briefly centrifuge (3,000 × <i>g</i>, 10 s) each reaction tube and then combine the 10× treated samples together in a single LoBind 2 mL tube. Then combine the 10× control samples into another 2 mL LoBind tube.</li>
<li>Vacuum dry the pooled samples in an evaporator overnight (8–16 h) at 30°C using a program suitable for water and organic solvent-based samples (program “HPLC” in the recommended evaporator).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> TMT-labelled peptides can be stored at −20°C.</div>
<ol start="37">
<li>To start the desalting step, resuspend the dried pooled samples in 0.1% TFA (300 μL).</li>
<li>Place the desalting columns (one per sample) in 2 mL LoBind tubes and centrifuge (5,000 <b>×</b> <i>g</i>, 1 min) to remove the equilibration buffer. Discard the flow through.</li>
<li>Wash the desalting columns 2× with 100% ACN (300 μL) and discard the flow-through.</li>
<li>Wash the columns a further 2× with 0.1% TFA (300 μL) and discard the flow-through.</li>
<li>Add the samples to separate desalting columns, centrifuge (3,000 <b>×</b> <i>g</i>, 1 min) and discard the flow-through. The peptides should now be bound to the columns.</li>
<li>Wash the columns 3× with 0.1% TFA and discard the flow-through.</li>
<li>Wash the columns 2× with 5% methanol (diluted in 0.1% TFA) to remove excess TMTs.</li>
<li>Elute peptides from the desalting columns into fresh 2 mL LoBind tubes by centrifugation (3,000 <b>×</b> <i>g</i>, 1 min) using 50% ACN diluted in 0.1% TFA (2× 300 μL).</li>
<li>Vacuum dry the desalted eluates.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Desalted TMT-labelled peptides can be stored at −20°C.</div>
<h3 id="sec3.5">Sample fractionation and mass spectrometry analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 days</div>
<p>This section describes the fractionation of samples by high-pH reverse-phase high performance liquid chromatography (RP-HPLC). Fractionation enables less complex peptide samples to be prepared for subsequent analysis by LC-MS/MS.</p>
<ol start="46">
<li>Set-up of the Ultimate 3000 HPLC (Dionex) using the Chromeleon software.
<ol type="a">
<li>Turn on UV lamp (wavelength 220 nm) at least 15 min prior to a run.</li>
<li>Purge the system with Buffer A (10 mM ammonium formate, pH 9.5) and Buffer B (10 mM ammonium formate in 90% ACN, pH 9.5).</li>
<li>Set the buffer composition to 98% A:2% B and the flow rate to 0.2 mL min<sup>−1</sup>.</li>
<li>Prime the syringe and wash the needle and fluidics.</li>
<li>Run the HPLC under these conditions for 15 min to fully equilibrate the system.</li>
</ol>
</li>
<li>Run a standard containing caffeine and MRFA (Met-Arg-Phe-Ala peptide) to check the HPLC system is working correctly.
<ol type="a">
<li>Place collection tubes in the collection trays.</li>
<li>Prepare the standard sample by diluting 2 μL of caffeine solution (1 mg/mL) and MRFA (1 mg/mL) in 100 μL buffer A in a HPLC sample tube. Place the sample tube in the samples tray.</li>
<li>Run the sample using the following program:
<ol type="i">
<li>Load 25 μL sample in the column.</li>
<li>Elute peptides from the column with a gradient of 2–20% Buffer B over 8 min.</li>
<li>Over the following 37 min increase the % of Buffer B from 20 - 47%.</li>
<li>Wash the column with 100% Buffer B for 15 min.</li>
<li>Fractions (volume) should be collected from 1 – 80 min of the run.</li>
</ol>
</li>
<li>Discard the collected fractions from the standard run.</li>
<li>Check the chromatogram for two peaks in minutes 12 and 14 that correspond to caffeine and MRFA respectively.</li>
</ol>
</li>
<li>Prepare and run the first sample
<ol type="a">
<li>Resuspend the dry sample in 200 μL buffer A, vortex vigorously and then spin down at 20,000 × <i>g</i> 4°C for 15 min. Transfer 190 μL of supernatant to the sample tube and adjust to pH &gt;9 using ammonia (check pH with a pH strip). Place the transfer tube in the sample tray.</li>
<li>Label 80 collection tubes and add 20 μL 10% FA to neutralize the pH of the fraction eluates. Place the collection tubes in the collection trays.</li>
<li>Run the sample using the program detailed in 46c but loading 185 μL sample.</li>
<li>Combine the 80 resulting fractions into 20 fractions by concatenating them. Discard fractions 1 and 2 as they usually contain the TMT excess and other contaminants.</li>
</ol>
</li>
<li>Repeat the run with a standard and continue running the following samples intercalating with standard runs.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Concatenate samples so that each contains eluted fractions collected at different points in the run. For example, fraction 3 would contain fractions eluted at 3, 23, 43 and 63 min.</div>
<ol start="50">
<li>Vacuum dry the concatenated fractions.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Concatenated fractions can be stored at −20°C.</div>
<ol start="51">
<li>Mass spectrometry analysis was performed using a nanoflow LC system (Ultimate 3000, Thermo) coupled to a Q Exactive HF Orbitrap Mass Spectrometer (Thermo).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The HPLC is equipped with a trap column switched in-line with a resolving column in order to protect it from salts and other interfering compounds and to concentrate the peptides before LC-MS/MS.</div>
<ol start="52">
<li>The mass spectrometer must be calibrated prior to analysis of samples.</li>
<li>Resuspend the concatenated samples in 0.1% FA (10 μL) and then load 1 μL aliquots (at 5 μL min<sup>-1</sup>) onto the trap column previously equilibrated with 95% Buffer C (0.1% FA) and 5% Buffer D (0.1% FA in 90% ACN). Once samples are loaded, wash the column for 5 min with 5% Buffer D at 5 μL min<sup>−1</sup>.</li>
<li>Connect the trap column to the resolving column and elute peptides at 300 nL min<sup>−1</sup> with a linear gradient (5% Buffer C for fractions 1–10, 7% for fractions 11–20, up to 35% Buffer C over 130 min, and then to 98% Buffer C at 132 min). Wash the column with 98% Buffer C for 20 min and then re-equilibrate in 5% Buffer C for 17 min.</li>
<li>Use Q Exactive HF in data-dependent mode using the following setup:
<ol type="a">
<li>MS1 scan cycles m/z range from 335–1800, with a maximum ion injection time of 50 ms, a resolution of 120,000 and automatic gain control (AGC) value of 3 × 10<sup>6</sup>
</li>
<li>Followed by 15 sequential dependent MS2 scans with an isolation window set to 0.7 Da, resolution at 60,000, maximum ion injection time at 200 ms and AGC 1 × 10<sup>5</sup>.</li>
</ol>
</li>
</ol>
<h3 id="sec3.6">Protein search, thermal proteome profiling, and hit selection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section describes the bioinformatic steps needed to find and quantify the proteins in the sample from the MS spectra. This section also outlines how to generate melt curves for every identified protein and to confidently identify proteins whose thermal stability has been altered in the presence of the test compound.</p>
<ol start="56">
<li>Place the RAW data files from MS analysis in a folder and open MaxQuant software (version 1.6.17.0).</li>
<li>Download the <i>Leishmania donovani</i> BPK282A1 proteome FASTA file (“annotated proteins”) from <a href="http://TriTrypDB.org">TriTrypDB.org</a>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other software can be used for this step (e.g. Thermo Proteome Discoverer).</div>
<div class="optional">
<span class="optional-title">Optional:</span> TMT labelling efficiency can be checked at this point using the parameters detailed below but by changing TMT reporters from fixed to variable modification. This can be done in the “Configuration” tab in MaxQuant. The “Type” of experiment in the Group-specific parameters tab should also be set to MS1. TMT labelling efficiency can be performed on an aliquot of the sample prior to fractionation to ensure sufficient labelling before further analysis.</div>
<ol start="58">
<li>Use default settings except for the following:
<ol type="a">
<li>In the “Raw data” tab, RAW files are loaded and labeled according to their condition and replicate for example “control_a”, “control_b”, “treated_a”, “treated_b” using the “Set experiment” button. Fraction number can be selected for each of them using the “Set fraction” button and typing 1 to 20. Reference channel should be set to “1” in all samples.</li>
<li>In Group-specific parameters:
<ol type="i">
<li>Set Reporter ion MS2 under “Type” and choose the 10plex TMT option.<br> Change the correction factors in the table that will appear below according to the Reporter ion isotopic distributions table in the TMT10plex product data sheet.</li>
<li>Choose Oxidation (M), Acetyl (Protein N-term), Dioxidation (W), Deamidation (NW) and Gln-&gt;Pyro-Glu as variable modifications and Carbamidomethyl (C) as fixed modification under “Modifications”.</li>
<li>Select Trypsin/P and LysC/P enzymes under “Digestion”</li>
</ol>
</li>
<li>In Global parameters:
<ol type="i">
<li>Add the FASTA file containing the protein sequences in the “Sequences” tab.</li>
<li>Set FTMS MS/MS Match tolerance to 10 ppm and ITMS MS/MS Match tolerance to 0.06 Da in the “MS/MS analyzer” tab.</li>
<li>Activate the “Match between runs” option in the “Identification” tab.</li>
</ol>
</li>
<li>Set the number of processors according to the capabilities of your system.
<div class="note">
<span class="note-title">Note:</span> Analysis time is approximately 20 h using a 28-core computer.</div>
</li>
</ol>
</li>
</ol>
<ol start="59">
<li>Pick the “proteinGroups.txt” file located in the combined/txt/ folder.</li>
<li>Open R studio and create a new project in a new folder.</li>
<li>Copy the “proteinGroups.txt” file, the “TPP_config-txt” file and the script file “StarProtocolsTPP.R” in the project folder. These can be downloaded from <a href="https://github.com/vcorpaslopez/StarProtocolsTPP">https://github.com/vcorpaslopez/StarProtocolsTPP</a>.</li>
<li>If not installed, install Bioconductor and other auxiliary packages by running lines annotated in #step 62 of the script file “StarProtocolsTPP.R”.</li>
<li>Install and load the TPP package (version 3.16.2) by running the lines annotated in #step 63 of the afore mentioned script file.</li>
<li>Extract the relevant columns by running lines annotated in #step 64 of the script file.</li>
<li>Check the quality of the experiment by running lines in #step 65 of the script file. This step will produce box plots of relative protein abundance for both replicas of the two conditions. These plots should show a reduction in median protein abundance following a decreasing sigmoidal pattern as temperature increases and a plateau around 0.15–0.10 protein abundance from 61°C. <a href="#troubleshooting">Troubleshooting 3</a>.</li>
<li>Run lines in #step 66 of the script file to calculate the proteome coverage, a message should appear in the console stating the proteome coverage calculated from the number of protein groups identified. Proteome coverage should be higher than 60% of the theoretical proteome. <a href="#troubleshooting">Troubleshooting 4</a>.</li>
<li>Load the experiment files in the environment using lines in #step 67 of the script file.</li>
<li>Load the temperature setup running lines in #step 68 of the script file, this will load the file “TPP_config.txt”.</li>
<li>Start the workflow running lines in #step 69 of the script file. This will run the entire work-flow, the following modifications can be made:
<ol type="a">
<li>nCores = 1, can be changed to the system capabilities.</li>
<li>normalize = TRUE or FALSE, to apply normalization or not (default is TRUE).</li>
<li>plotCurves=TRUE or FALSE, to plot the melting curves, reducing time if FALSE (default is TRUE).</li>
<li>pValFilter = list(minR2 = 0.8, maxPlateau = 0.4), states the requirements of the curve fitting parameters for p-value calculation. Can be changed to be more restrictive.</li>
</ol>
</li>
<li>Run lines in #step 70 of the script file to create a file containing the Tm targets of the experiment. This will create a .CSV file with the T<sub>m</sub> targets (“T<sub>m</sub>_targets.csv”).</li>
<li>Run lines in #step 71 of the script file to create a file containing the NPARC targets of the experiment. This will create a .CSV file with the NPARC targets (“NPARC_targets.csv”). Adjust the value in line 67 to change the p-value threshold (default 0.01, p_adj_NPARC&lt;0.01).</li>
<li>Full results can be viewed in the file “results_TPP_TR.xlsx” in the “analysis” folder. This excel file contains links to the melting curves (T<sub>m</sub> method) and splines (NPARC method) .PDF files. Melting curve files can also be found in the analysis/Melting_Curves folder, there is a file for each protein plotted. <a href="#troubleshooting">Troubleshooting 5</a>.</li>
<li>In order to obtain high confidence target candidates, two independent experiments should be performed. Any target that is identified as target via T<sub>m</sub> analysis in both experiments (with similar melting curve behavior), should be considered a high confidence target. Similarly, any target considered a NPARC target with a 0.01 confidence level in both experiments (with similar melting curve behavior), should also be considered a high confidence target.</li>
<li>To avoid missing potential targets, we recommend visually cross-checking T<sub>m</sub> and NPARC targets from one experiment with their counterparts in the second experiment. This is particularly important for T<sub>m</sub> targets as some proteins may not match the statistical assumptions made in this data analysis programme.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> It is up to the user to reduce the stringency of the analysis modifying different parameters in step 69 or reducing stringency (increasing <i>p</i>-value threshold) in step 71. Targets found by modifying these parameters should not be considered high confidence though, but this can assist in avoiding false negatives.</div>
<ol start="75">
<li>The file “QCplots.pdf” in the “analysis” folder shows quality control plots of the process. Check that the normalization curves have a R<sup>2</sup> value greater than 0.90.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A clear melting curve should be seen after plotting the protein concentration values of the Bradford assay (Step 12). <a href="#fig1">Figure 1</a> shows a typical protein abundance over temperature plot from one control sample.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/948-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Representative results of Bradford assay to determine protein concentration</div>
<p>The graph shows protein abundance (%) over temperature (°C) of the Bradford assay in step 14.</p>
</figcaption>
</figure>
<p>The chromatogram produced in the HPLC analysis (steps 46–49) is shown in <a href="#fig2">Figure 2</a>. This graph shows the UV intensity at 220 nm wavelength over time (80 min collection time). Good quality peaks are defined by narrow width and high resolution. Presence of wide peaks or loss of resolution (i.e., peaks merge) may indicate the sample is contaminated, with detergent or other reagents, alternative column should be changed. Peaks in min 1–2 correspond to salts and free TMTs and should be discarded. The final peak (min 80) contains potential contaminants and undigested material and should also be discarded.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/948-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative HPLC chromatogram</div>
<p>The graph shows UV absorbance at 220 nm (expressed in milli-arbitrary units) over retention time (min).</p>
</figcaption>
</figure>
<p>Shown in <a href="#fig3">Figure 3</a> is a QC plot of data generated in step 65. It shows the median protein abundance for each temperature, including the distribution of data. Median protein abundance should indicate a clear melt curve (as seen in <a href="#fig3">Figure 3</a>) and the protein abundance in the last temperature should be no higher than 0.2.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/948-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Median abundance box plot of proteins quantified through LC-MS/MS</div>
<p>The graph shows box plots of median protein abundance including confidence intervals (relative to the lowest temperature).</p>
</figcaption>
</figure>
<p><a href="#fig4">Figure 4</a> shows the melting curves of a stabilized protein target (<i>Leishmania donovani</i> LdBPK_060670.1 lanosterol synthase, putative) generated after running the TPP workflow. These melting curves are relevant for the T<sub>m</sub> method. A clear melting temperature difference is evident between the control (dashed lines) and the treated (solid lines) melting curves in both replicates (green and purple).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/948-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Melting curves for a protein target generated in the TPP workflow. The graph shows the non-denatured fraction (protein abundance) over temperature (°C) of control (dashed lines) and treated (solid lines) melting curves, two replicates (green and purple lines).</div>
</figcaption>
</figure>
<p><a href="#fig5">Figure 5</a> shows the melting curves for the same target for the NPARC analysis, displaying the null model of the analysis (that assumes that the treatment has no effect over the melting behavior) and the alternative model (treated, orange; and control, gray) that assumes that the treatment affects the melting behavior. The image shows how the experimental data (circles for the first and triangles for the second replicate) adjust much better to the alternative model curves and the subsequent adjusted p-value (“p.adj” in the upper right corner).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/948-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Melting curves for a protein target generated in the NPARC part of the TPP workflow</div>
<p>The graph shows the non-denatured fraction (protein abundance) over temperature (°C) of a protein, displaying the null model of the analysis that assumes that the treatment has no effect over the melting behavior (black line) and the alternative model (treated, orange; and control, gray) that assumes that the treatment affects the melting behavior. Experimental data is shown as dots (circles for the first and triangles for the second replicate). The adjusted p-value of the analysis is shown as “p.adj” in the upper right corner.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The protocol described above details two different data analysis methods to identify and prioritize protein targets. The original method (<a href="#bib4">Franken et al., 2015</a>), which we call the T<sub>m</sub> method, was published alongside the first TPP publication (<a href="#bib6">Savitski et al., 2014</a>). Here, protein melting curves are fitted to a sigmoidal trend and the melting temperature (T<sub>m,</sub> temperature at which 50% of protein is denatured) is estimated for the treated and control samples (in duplicate) and the melting point shift (ΔT<sub>m</sub>) is calculated for both replicates. Next, a z-test is run in order to determine which ΔT<sub>m</sub>’s are statistically significant. The workflow then picks those proteins with two significant (FDR-adjusted p-value &lt;0.2) T<sub>m</sub> shifts that also fulfill certain assumptions (detailed in the original paper). As shown in step 71d, some filters can be modified by the user to decide which proteins undergo p-value calculation, like R<sup>2</sup> of the curve and plateau.</p>
<p>The alternative NPARC data analysis method was described more recently (<a href="#bib1">Childs et al., 2019</a>) and does not rely on the T<sub>m</sub> of the melting curve. In this case, the workflow estimates the goodness of fit of the data to two different models: a null model that assumes that treatment has no effect on the melting behaviors; and an alternative model that assumes that treatment has an effect. The program then calculates a F statistic based on the difference between experimental data and the two model and produces a FDR-adjusted p-value. This method is very useful in reducing false negatives and to identify proteins with non-canonical melting behavior or a T<sub>m</sub> greater than the specified temperature range. Such targets would not be identified via the standard T<sub>m</sub> method. In the final steps of the protocol, we summarize the strategies we follow to identify potential drug targets after performing two independent experiments each comprising two technical replicates. More information about the statistical analysis can be found in the original TPP and NPARC papers.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>TPP is a very powerful technique that can be used to identify the targets of phenotypically active compounds; however, its power relies on the thermal stabilization of proteins upon drug binding. However, some proteins are not susceptible to thermal stabilization or de-stabilization so would not be identified by this approach. The affinity of a specific drug for its target may also affect the level of thermal-shift observed. Some protein targets may not be abundant enough to be reliably identified and quantified using mass spectrometry, since their abundance would be obscured by other more abundant proteins. It should also be noted that targets within large protein complexes, such as the proteasome, may be more challenging to identify. The same may also be true for targets that are integral membrane proteins. Finally, this approach is not suitable to determine the mechanism of action of compounds that interact with non-protein targets.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Protein yield not sufficient for the assay (step 7)</p>
<h3 id="sec7.2">Potential solution</h3>
<p>If protein yield is lower than 4 mg using the suggested protocol, increase the scale of the initial cell cultures. Do not proceed with the thermal shift assay part of the protocol using samples with a protein concentration &lt; 0.5 mg mL<sup>−1</sup>, since at lower concentrations proteins may fail to aggregate and precipitate. It is important to check the pH of the Lysis Buffer and to use fresh components when preparing the buffer, particularly protease inhibitors.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Unexpected protein concentration results in the Bradford assay (step 12).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>If the protein concentration plot does not resemble that shown in <a href="#fig1">Figure 1</a>, repeat the Bradford assay. If problem persists, validate the set temperatures in the thermal cycler. When harvesting the supernatant after ultracentrifugation steps be careful not to be greedy. Attempting to recover all the supernatant can risk disturbing the pellet and potential affect the quality and/or accuracy of the results.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Unexpected median protein abundance (step 65)</p>
<h3 id="sec7.6">Potential solution</h3>
<p>If median protein abundance does not resemble that shown in <a href="#fig3">Figure 3</a>, check TMT labeling efficiency. Low TMT efficiency (&lt;98%) will affect protein quantitation. If TMT labeling efficiency is low, conditions during labeling should be checked (TMT reagent/protein ratio, pH of the reaction, TMT expiry date and storage conditions).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Low proteome coverage (step 66).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>If the detected proteome coverage is lower than 60% compared to the theoretical proteome, check efficiency of the digestion steps.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>High variability between replicates (step 72).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>If there is a large and consistent difference between the replicates in the control or treated samples (either T<sub>m</sub> between replicates or experimental data points), one solution is to process both replicas at the same time. It is particularly important to fractionate the 4 samples in the HPLC in the same session and run the 96 fractions in the mass spectrometer without interruptions (i.e., running other samples). These will greatly improve the reproducibility of the data.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Childs, D., Bach, K., Franken, H., Anders, S., Kurzawa, N., Bantscheff, M., Savitski, M.M., and Huber, W. (2019). Nonparametric analysis of thermal proteome profiles reveals novel drug-binding proteins. Mol. Cell. Proteomics. <i>18</i>, 2506-2515. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00411-1/sref1">View at publisher</a></p>
<p id="bib3">Corpas-Lopez, V., Moniz, S., Thomas, M., Wall, R.J., Torrie, L.S., Zander-Dinse, D., Tinti, M., Brand, S., Stojanovski, L., Manthri, S., et al. (2019). Pharmacological validation of N-myristoyltransferase as a drug target in leishmania donovani. ACS Infect. Dis. <i>5</i>, 111-122. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00411-1/sref3">View at publisher</a></p>
<p id="bib4">Franken, H., Mathieson, T., Childs, D., Sweetman, G.M., Werner, T., Tögel, I., Doce, C., Gade, S., Bantscheff, M., Drewes, G., et al. (2015). Thermal proteome profiling for unbiased identification of direct and indirect drug targets using multiplexed quantitative mass spectrometry. Nat. Protoc. <i>10</i>, 1567-1593. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00411-1/sref4">View at publisher</a></p>
<p id="bib9">Goyard, S., Segawa, H., Gordon, J., Showalter, M., Duncan, R., Turco, S.J., and Beverley, S.M. (2003). An in vitro system for developmental and genetic studies of Leishmania donovani phosphoglycans. Mol. Biochem. Parasitol. <i>130</i>, 31-42. <a class="external-link" href="https://doi.org/10.1016/s0166-6851(03)00142-7">https://doi.org/10.1016/s0166-6851(03)00142-7</a></p>
<p id="bib5">Paradela, L.S., Wall, R.J., Carvalho, S., Chemi, G., Corpas-Lopez, V., Moynihan, E., Bello, D., Patterson, S., Güther, M.L.S., Fairlamb, et al. (2021). Multiple unbiased approaches identify oxidosqualene cyclase as the molecular target of a promising anti-leishmanial. Cell. Chem. Biol. <i>28</i>, 711-721. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00411-1/sref5">View at publisher</a></p>
<p id="bib6">Savitski, M.M., Reinhard, F.B.M., Franken, H., Werner, T., Savitski, M.F., Eberhard, D., Martinez Molina, D., Jafari, R., Dovega, R.B., Klaeger, et al. (2014). Tracking cancer drugs in living cells by thermal profiling of the proteome. Science <i>346</i>, 6205. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00411-1/sref6">View at publisher</a></p>
<p id="bib7">Tyanova, S., Temu, T., and Cox, J. (2016). The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. Nat. Protoc. <i>11</i>, 2301-2319. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00411-1/sref7">View at publisher</a></p>
<p id="bib8">Vizcaino, J.A., Deutsch, E.W., Wang, R., Csordas, A., Reisinger, F., Rios, D., Dianes, J.A., Sun, Z., Farrah, T., Bandeira, N., et al. (2014). ProteomeXchange provides globally coordinated proteomics data submission and dissemination. Nat. Biotechnol. <i>32</i>, 223-226. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00411-1/sref8">View at publisher</a></p>
<p id="bib2">Childs D, Kurzawa N, Franken H, Doce C, Savitski M, Huber W (2020). TPP: Analyze thermal proteome profiling (TPP) experiments. R package version 3.18.0. Bioconductor. <a class="external-link" href="https://doi.org/10.18129/B9.bioc.TPP">https://doi.org/10.18129/B9.bioc.TPP</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr Susan Wyllie (<a href="mailto:s.wyllie@dundee.ac.uk">s.wyllie@dundee.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>All reasonable requests for materials used in this study will be considered.</p>
<h4>Data and code availability</h4>
<p>The mass spectrometry raw files and their associated MaxQuant output files generated during this study are available at ProteomeXchange Consortium (<a href="#bib8">Vizcaino et al., 2014</a>) via the PRIDE partner repository (<a href="http://www.ebi.ac.uk/pride/archive/">http://www.ebi.ac.uk/pride/archive/</a>), under the identifier PRIDE: PXD023780 as listed in the <a href="#key-resources-table">key resources table</a>.</p>
<p>All the programming functions required to perform this analysis can be found in TPP, an R package (<a href="#bib2">Childs et al., 2020</a>) or available for download at <a href="https://www.bioconductor.org/packages/release/bioc/html/TPP.html">https://www.bioconductor.org/packages/release/bioc/html/TPP.html</a>.</p>
<p>A wrapper script to follow these steps can be downloaded at <a href="https://github.com/vcorpaslopez/StarProtocolsTPP">https://github.com/vcorpaslopez/StarProtocolsTPP</a>.</p>
<h3>Acknowledgments</h3>
<p>We would like to take this opportunity to thank the Fingerprints Proteomics Facility at the University of Dundee for their invaluable support throughout these studies. This work was supported by the following awards from the <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a>: Strategic Award [105021], Innovations Award [218448/Z/19/Z], and a Centre Award [203134/Z/16/Z].</p>
<h3>Author contributions</h3>
<p>Conceptualization, V.C.L. and S.W.; methodology, V.C.L.; experimental, V.C.L.; supervision, S.W.; writing, V.C.L. and S.W.; review and editing, S.W.; funding acquisition, S.W.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>