<article>
<section>
<h1 id="header">Protocol for the determination of intracellular phase separation thresholds</h1>
<p><time datetime="2021-02-05">Published: February 5, 2021</time></p>
<p>Anna K. Lee<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> and Patrick Ryan Potts<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Cell and Molecular Biology, St. Jude Children’s Research Hospital, Memphis, TN 38105, USA</p>
<p id="fn1"><sup>2</sup>Present address: Department of Induced Proximity Platform, Amgen Inc., Thousand Oaks, CA 91320, USA</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:anna.lee@stjude.org">anna.lee@stjude.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:rpotts01@amgen.com">rpotts01@amgen.com</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100308">10.1016/j.xpro.2021.100308</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>To date, phase separation studies have largely been limited to <i>in vitro</i> assays using non-native conditions and aggregation-prone recombinant proteins that are often difficult to purify. This protocol describes the determination of relative protein concentration thresholds for phase separation through fluorescent imaging of GFP-tagged proteins in cells. The commercial availability of various plasmids and antibodies, as well as advances in gene editing, allow this procedure to be modified for the study of various phase-separating proteins in their relevant contexts.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Lee et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Determining relative protein concentration thresholds for phase separation in cells</li>
<li>Fluorescent imaging and analysis of GFP-tagged proteins at single-cell resolution</li>
<li>Immunofluorescent staining and imaging of endogenous proteins</li>
<li>Quantification of endogenous protein levels relative to the threshold concentration</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/445-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was employed in a recent publication to establish the relative threshold concentration of G3BP1 protein that dictates stress granule assembly (<a href="#bib2">Lee et al., 2020</a>). To examine this in cells, we transiently transfected cells lacking stress granule nucleators, G3BP1 and G3BP2 (collectively referred to as G3BP), with GFP-tagged G3BP1 and measured stress granule initiation time as a function of GFP-G3BP1 intensity on a cell-by-cell basis. We then measured the amount of endogenous G3BP1 in wild-type or mutant cells relative to the determined threshold concentration. By doing so, we demonstrated that moderate translational suppression of G3BP could significantly reduce stress granule formation. Others have also utilized this protocol to examine the effects that G3BP1 dimerization or binding partners have on the concentration threshold of G3BP1 necessary to initiate stress granule assembly in cells (<a href="#bib5">Yang et al., 2020</a>).</p>
<p>Prior to starting the experiment, we prepared G3BP knockout U2OS cells, GFP-G3BP1 plasmid, and the appropriate reagents listed below. Investigators should utilize cells lacking their gene of interest to ensure that endogenous protein does not confound the results. Although not necessary, we recommend using U2OS cells because their unique morphology is ideal for imaging.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal anti-G3BP1</td>
<td>BD Biosciences</td>
<td>Cat# 611126; RRID: AB_398437</td>
</tr>
<tr>
<td>Donkey anti-mouse IgG (H+L) secondary antibody, Alexa Fluor 568</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A10037; RRID: AB_2534013</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM</td>
<td>Gibco</td>
<td>11965118</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>HyClone</td>
<td>SH3091003</td>
</tr>
<tr>
<td>L-Glutamine (200 mM)</td>
<td>Gibco</td>
<td>25030149</td>
</tr>
<tr>
<td>Antibiotic-antimycotic (100×)</td>
<td>Gibco</td>
<td>15240062</td>
</tr>
<tr>
<td>Effectene transfection reagent</td>
<td>QIAGEN</td>
<td>301425</td>
</tr>
<tr>
<td>Sodium arsenite</td>
<td>Sigma</td>
<td>35000-1L-R</td>
</tr>
<tr>
<td>16% Paraformaldehyde solution</td>
<td>Electron Microscopy Sciences</td>
<td>15710-S</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>U2OS</td>
<td>ATCC</td>
<td>HTB-96</td>
</tr>
<tr>
<td>U2OS G3BP knockout</td>
<td>Gift from J. Paul Taylor (St. Jude)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Plasmid: EGFP-C3-G3BP1</td>
<td>Gift from J. Paul Taylor (St. Jude)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ</td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
<td>v. 2.0.0-rc-59/1.51n</td>
</tr>
<tr>
<td>Prairie View</td>
<td>Bruker</td>
<td>v. 5.4</td>
</tr>
<tr>
<td>GraphPad Prism</td>
<td>GraphPad.com</td>
<td>v. 8.2.0</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Opterra II swept field confocal microscope</td>
<td>Bruker</td>
<td>N/A</td>
</tr>
<tr>
<td>Lab-Tek II chambered coverglass</td>
<td>Thermo Fisher Scientific</td>
<td>155382</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Experiments were performed using a Bruker Opterra II swept field confocal microscope. Cells were maintained at 37°C and supplied with 5% CO<sub>2</sub> using a Bold Line Cage Incubator (Okolabs) and an objective heater (Bioptechs). Images were acquired using a 60× Plan Apo 1.4NA oil objective with Perfect Focus (Nikon) engaged for the duration of the capture.</p>
<table id="undtbl1">
<thead>
<tr>
<th>Solutions</th>
<th>Formulation</th>
</tr>
</thead>
<tbody>
<tr>
<td>Fixing solution</td>
<td>3% PFA (w/v) in 1× PBS. Filter with 0.45 μm membrane. Make fresh for each experiment</td>
</tr>
<tr>
<td>Wash solution</td>
<td>0.2% Triton X-100 (v/v) in 1× PBS</td>
</tr>
<tr>
<td>Blocking solution</td>
<td>3% BSA (w/v) and 0.2% Triton X-100 (v/v) in 1× PBS. Filter through 0.45 μm membrane. Make fresh for each experiment</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Transfection of cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This section describes how to seed and transfect cells with the GFP-tagged gene of interest.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The assembly of stress granules and other phase separated membrane-less organelles is highly dependent on the concentration of key nucleating factors. Therefore, investigators should utilize cells lacking their gene of interest and optimize the amount of plasmid DNA used during transfection to minimize spontaneous stress granule formation while also providing a wide dynamic range for analysis.</div>
<ol>
<li>The day before transfection, seed cells such that they will be 40% confluent on the day of transfection.
<ol type="a">
<li>Investigators should determine the optimal confluency at the time of transfection.
<ol type="i">
<li>For G3BP knockout U2OS cells, seed 20,000 cells per well in a 4-well chamber slide 16–24 h before transfection.</li>
</ol>
</li>
</ol>
</li>
<li>16–24 h after seeding, dilute 200 ng of plasmid DNA containing GFP-G3BP1 or your GFP-tagged gene of interest in Buffer EC from the Effectene Transfection Reagent kit for a total volume of 60 μL.</li>
<li>Add 1.6 μL of Enhancer from the Effectene Transfection Reagent kit, vortex for 1 s, and briefly spin down the mixture.</li>
<li>Incubate at 20°C–25°C for 2–5 min.</li>
<li>Add 5 μL of Effectene Transfection Reagent, vortex for 10 s, and briefly spin down the mixture.</li>
<li>Incubate at 20°C–25°C for 5–10 min.</li>
<li>Meanwhile, remove cells from the incubator, gently aspirate growth medium, and add 350 μL of fresh growth medium.</li>
<li>Add 350 μL of growth medium to the transfection complex, mix well by pipetting, and add the transfection complex dropwise onto cells.</li>
<li>Gently swirl the slide to distribute the transfection complex and return cells to the incubator.</li>
<li>24 h later, remove cells from the incubator, gently aspirate growth media and add 400 μL of fresh growth medium.</li>
<li>Return cells to the incubator for an additional 24 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Refer to the manufacturer’s instructions for troubleshooting. Alternative transfection methods can be used.</div>
<h3 id="sec3.2">Live-cell imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This section describes how cells should be visualized to measure relative GFP intensity and stress granule initiation time for intracellular threshold determination.</p>
<ol start="12">
<li>Equilibrate the microscope to 37°C and 5% CO<sub>2</sub>.
<ol type="a">
<li>The use of an objective heater (Bioptechs) can help to maintain the temperature at 37°C.</li>
</ol>
</li>
<li>Set the 488 nm laser to 80 power and 100 ms frame exposure time using a 35 μm slit.
<ol type="a">
<li>An average power of 0.37 mW and irradiance at the sample of 2 W/cm<sup>2</sup> was used; however, investigators should optimize the imaging parameters.</li>
</ol>
</li>
<li>Identify cells for imaging.
<ol type="a">
<li>Transient transfection of your gene of interest should result in cell-to-cell variation in expression levels. Select cells with varying intensities of GFP (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/445-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Imaging of GFP-G3BP1 and endogenous G3BP1</div>
<p>(A) Live-cell imaging of transfected cells with varying intensities of GFP. G3BP knockout U2OS cells were transiently transfected with GFP-G3BP1. Scale bar, 10 μm.</p>
<p>(B) Time-lapse imaging of cells treated with sodium arsenite. Cells were transfected as in (A) and were treated with 500 μM sodium arsenite. Images were collected for 2 h; select images depicting stress granule initiation are shown. Yellow arrows indicate stress granules. Scale bar, 10 μm.</p>
<p>(C) Quantification of GFP intensity. The GFP intensity of each cell before sodium arsenite treatment was quantified using ImageJ. Scale bar, 10 μm.</p>
<p>(D) Live-cell and immunofluorescent imaging of transfected cells. Cells were transfected as in (A), subjected to live-cell imaging (left), and subsequently fixed and immunostained for G3BP1 (right).</p>
<p>(E) Live-cell and immunofluorescent imaging of wild-type cells. Wild-type cells (left cell) lack GFP (left panel) but exhibit signal from G3BP antibody staining (right panel), whereas transfected cells (right cell) can be visualized by both GFP and antibody staining.</p>
</figcaption>
</figure>
<ol type="i">
<li>Cells with GFP-G3BP1 intensities ranging from 700 to 5000 RFU were selected.</li>
</ol>
</li>
<li>Avoid cells with spontaneous stress granules. Cells should not exhibit stress granules prior to treatment with sodium arsenite. <a href="#sec6.1">Troubleshooting 1</a>
</li>
<li>Within Prairie View, the selected fields can be stored as stage locations in the “XY-Stage” tab.</li>
</ol>
</li>
<li>Take a multipoint time-lapse, imaging each field every 40 s for two iterations.
<ol type="a">
<li>Within Prairie View, a time-lapse “T-Series” can be set up to “Run at all XYZ stage locations.”</li>
<li>These initial images will be used to quantify GFP intensity for each cell prior to stress granule induction and to ensure proper stage movement and focus between selected stage locations.</li>
</ol>
</li>
<li>Treat cells with sodium arsenite (t = 0) for a final concentration of 500 μM.
<ol type="a">
<li>Dilute the 50 mM stock solution of sodium arsenite to 1 mM with growth medium.</li>
<li>Add 400 μL of the 1 mM sodium arsenite solution to cells (cultured in 400 μL of growth medium) to allow for sufficient mixing without disturbing the slide.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not disturb the slide when treating cells. Slight perturbations can shift cells out of the saved stage locations. <a href="#sec6.3">Troubleshooting 2</a>
</div>
<ol start="17">
<li>Immediately begin a multipoint time-lapse and collect images for 2 h (<a href="#fig1">Figure 1</a>B).
<ol type="a">
<li>Investigators should optimize the duration of the experiment for the specific cell line used.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The next steps can be performed at any time.</div>
<h3 id="sec3.3">Quantification of GFP intensity and stress granule initiation time</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This section describes how to measure GFP intensity and stress granule initiation time on a cell-by-cell basis.</p>
<ol start="18">
<li>Import image files for analysis into ImageJ.</li>
<li>Create an XY table in GraphPad Prism with one row for each cell. GFP intensity will be entered as the X value and stress granule initiation time will be entered as the Y value.</li>
<li>Select a region of interest from the background and use the Analyze &gt; Measure command to measure the mean intensity for background correction.</li>
<li>Quantify the GFP intensity of each cell from the frame before sodium arsenite addition.
<ol type="a">
<li>Within ImageJ, use the “Freehand Selections” tool to outline a cell (<a href="#fig1">Figure 1</a>C).</li>
<li>Use the Analyze &gt; Measure command to determine the total cell integrated density, area, and mean intensities.</li>
<li>To measure the GFP intensity of the cytoplasm only, use the “Freehand Selections” tool to outline the nucleus and use the Analyze &gt; Measure command to determine the integrated density and area of the nucleus. These values can be subtracted from the total cell measurements to determine the average intensity from the cytoplasm.
<ol type="i">
<li>((Total cell integrated density − Nuclear integrated density)/(Total cell area − Nuclear area)) − Mean background intensity = Corrected mean cytoplasmic intensity</li>
</ol>
</li>
<li>Investigators should determine the appropriate region of interest selection method.</li>
</ol>
</li>
<li>To measure the stress granule initiation time of each cell, identify the time after sodium arsenite treatment when two stress granules form (<a href="#fig1">Figure 1</a>B).</li>
<li>The associated graph should have GFP intensity on the X-axis and stress granule initiation time on the Y-axis with each cell plotted as a single point. Using this graph, investigators can determine the threshold for intracellular phase separation (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/445-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. G3BP1 intracellular phase separation threshold</div>
<p>(A) G3BP1 expression correlates with SG initiation time. G3BP knockout U2OS cells were transiently transfected with GFP-G3BP1. GFP-G3BP1 protein levels were determined at single-cell resolution by live-cell imaging before stress. Cells were then treated with 500 μM sodium arsenite to induce stress granule assembly and the time at which stress granule assembly initiated was quantified and correlated to GFP-G3BP1 protein levels (n = 3, total 168 cells analyzed). GFP-G3BP1 expression beyond a threshold (dotted red line) results in enhanced stress granule formation. Figure adapted from <a href="#bib2">Lee et al. (2020)</a>.</p>
<p>(B) Correlation between GFP-G3BP1 fluorescent intensity from GFP and immunofluorescent intensity from G3BP1 antibody staining. G3BP knockout cells were transfected with GFP-G3BP1. 48 h post transfection, GFP intensity was determined at single-cell resolution by live-cell imaging. Cells were then immunostained for G3BP1 and fluorescence from the G3BP1 immunostaining was correlated to the previously measured GFP intensity on a cell-by-cell basis (n = 3, total 76 cells analyzed). Figure adapted from <a href="#bib2">Lee et al. (2020)</a>.</p>
<p>(C) MAGE-B2 knockout increases G3BP1 protein levels beyond the threshold (dotted red line determined from A) for enhanced stress granule formation. The relative amounts of endogenous G3BP1 in either wild-type or MAGE-B2 knockout U2OS cells was determined in relation to GFP-G3BP1 in (A) (n = 3). p value ≤ 0.05 was determined by t test. Figure adapted from <a href="#bib2">Lee et al. (2020)</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The next steps can be performed at any time.</div>
<h3 id="sec3.4">Determination of endogenous protein levels</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 days</div>
<p>This section describes how to seed and immunostain cells in order to measure endogenous protein levels relative to the determined threshold concentration.</p>
<div class="note">
<span class="note-title">Note:</span> In our case, we sought to quantify G3BP1 protein levels in wild-type cells or in cells lacking a translational repressor of G3BP relative to the identified threshold. To do so, we transfected G3BP knockout cells with GFP-G3BP1 as described in steps 1–9 and seeded either wild-type or mutant cells. Then, we used live-cell imaging to measure GFP intensity and subsequently fixed and immunostained cells to measure G3BP1 immunofluorescent intensity in wild-type, mutant, or transfected G3BP knockout cells.</div>
<div class="note">
<span class="note-title">Note:</span> As noted previously, this portion of the experiment can be performed at any time.</div>
<ol start="24">
<li>Use a gridded chamber slide or simply mark X- and Y-axes on the bottom of the slide with an oil-resistant marker.
<ol type="a">
<li>U2OS cells are highly adherent, however, cell types that do not readily adhere to tissue culture plates may benefit from coating plates with a binding agent (e.g., poly-lysine).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is essential that investigators are able to relocate cells because transfected cells will be visualized by live-cell imaging to measure GFP intensity and subsequently immunostained to measure immunofluorescent intensity at single-cell resolution.</div>
<ol start="25">
<li>Transfect knockout cells with your GFP-tagged gene of interest as described in steps 1–9.</li>
<li>24 h after transfection, aspirate the growth media and gently wash cells.</li>
<li>Seed 500 μL of wild-type (or mutant) cells in the same well as the transfected cells.
<ol type="a">
<li>Investigators should determine the optimal number of cells to seed after transfection.
<ol type="i">
<li>For wild-type U2OS cells, seed 10,000 cells per well in a 4-well chamber slide 24 h after transfection.</li>
</ol>
</li>
</ol>
</li>
<li>Return cells to the incubator for an additional 24 h.</li>
<li>The next day, perform live-cell imaging as described in steps 12–14.</li>
<li>Capture images of the selected cells.
<ol type="a">
<li>Save the stage locations of the selected fields to assist in cell relocation after immunostaining.</li>
</ol>
</li>
<li>Prepare fresh Fixing Solution for immunostaining.</li>
<li>Gently wash cells with 1 mL of PBS.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use caution not to lift off cells in subsequent steps, particularly with poorly adherent cell lines. <a href="#sec6.5">Troubleshooting 3</a>
</div>
<ol start="33">
<li>Fix cells with 500 μL of Fixing Solution for 15 min at 20°C–25°C.</li>
<li>Meanwhile, prepare fresh Blocking Solution.</li>
<li>Wash cells 2× with 500 μL of PBS for 5 min.</li>
<li>Permeabilize cells with 500 μL of Blocking Solution for 20 min at 4°C.</li>
<li>Meanwhile, dilute primary antibody in Blocking Solution and spin down at 21,000 × <i>g</i> for 5 min at 4°C.
<ol type="a">
<li>Dilute G3BP1 antibody 1:200. For other cell lines or genes of interest, investigators should optimize the amount of primary antibody used.</li>
</ol>
</li>
<li>Add 250 μL of diluted primary antibody to cells and incubate 12–18 h at 4°C or 1 h at 20°C–25°C.
<ol type="a">
<li>For 12–18 h incubations, place the slide in a container with a wet paper towel as a humidifier. If necessary, increase the volume of diluted primary antibody to prevent cells from drying out.</li>
</ol>
</li>
<li>Wash cells 3× with 500 μL of Wash Solution for 5 min.</li>
<li>Meanwhile, dilute secondary antibody 1:500 in Blocking Solution and spin down at 21,000 × <i>g</i> for 5 min at 4°C.</li>
<li>Add 500 μL of diluted secondary antibody to cells, cover with foil to prevent photobleaching, and incubate for 30 min at 20°C–25°C.</li>
<li>Wash cells 3× with 500 μL of Wash Solution for 5 min.</li>
<li>Add 500 μL of PBS to cells.</li>
<li>Return the slide to the microscope in the same orientation as it was previously imaged.</li>
<li>Set the 488 and 561 nm lasers to 80 power and 100 ms exposure time using a 35 μm slit.
<ol type="a">
<li>Investigators should optimize the imaging parameters.</li>
</ol>
</li>
<li>Relocate cells for immunofluorescent imaging (<a href="#fig1">Figure 1</a>D).
<ol type="a">
<li>Each cell that was previously imaged for GFP should now be imaged for antibody staining.</li>
</ol>
</li>
<li>Identify and image cells that lack GFP but exhibit signal from antibody staining (<a href="#fig1">Figure 1</a>E).
<ol type="a">
<li>These are the wild-type (or mutant) cells that were seeded after transfection.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The next steps describe image analysis and can be performed at any time.</div>
<ol start="48">
<li>Import image files for analysis into ImageJ.</li>
<li>Create an XY table in GraphPad Prism with one row for each transfected cell. GFP intensity from live-cell imaging will be entered as the X value and immunofluorescent intensity from antibody staining will be entered as the Y value.</li>
<li>Select a region of interest from the background and use the Analyze &gt; Measure command to measure the mean intensity for background correction.</li>
<li>Quantify the GFP intensity of each cell from live-cell imaging.
<ol type="a">
<li>Within ImageJ, use the “Freehand Selections” tool to outline a cell (<a href="#fig1">Figure 1</a>C).</li>
<li>Use the Analyze &gt; Measure command to determine the total cell integrated density, area, and mean intensities.</li>
<li>To measure the GFP intensity of the cytoplasm only, use the “Freehand Selections” tool to outline the nucleus and use the Analyze &gt; Measure command to determine the integrated density and area of the nucleus. These values can be subtracted from the total cell measurements to determine the average intensity from the cytoplasm.
<ol type="i">
<li>((Total cell integrated density − Nuclear integrated density)/(Total cell area − Nuclear area)) − Mean background intensity = Corrected mean cytoplasmic intensity</li>
</ol>
</li>
<li>Investigators should determine the appropriate region of interest selection method.</li>
</ol>
</li>
<li>Quantify the immunofluorescent intensity of each cell from antibody staining in the same way.</li>
<li>The associated graph should have GFP intensity on the X-axis and immunofluorescent intensity on the Y-axis with each cell plotted as a single point.</li>
<li>Perform a simple linear regression analysis within GraphPad Prism to model the relationship between GFP intensity and immunofluorescent intensity. <a href="#sec6.7">Troubleshooting 4</a>
</li>
<li>Quantify the immunofluorescent intensity of each wild-type (or mutant) cell from antibody staining.
<ol type="a">
<li>While the transfected cells should exhibit both GFP and immunofluorescent intensities, wild-type (or mutant) cells should not have any GFP signal.</li>
</ol>
</li>
<li>Use the calculated linear regression line to determine the GFP equivalent of the average measured immunofluorescent intensity from wild-type (or mutant) cells.
<ol type="a">
<li>These results can be plotted with the determined threshold concentration included as a reference point (<a href="#fig2">Figure 2</a>C).</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Here, we present a protocol that takes advantage of natural variations in expression that arise from the transient transfection of cells (<a href="#fig1">Figure 1</a>A). Live-cell imaging of transfected cells allows for quantification and correlation of the GFP-tagged protein of interest to stress granule initiation time (<a href="#fig2">Figure 2</a>A). Consistent with <i>in vitro</i> phase separation assays, we found that the relationship between intracellular protein concentration and condensate formation was switch-like, such that phase separation is dictated by a critical threshold concentration (<a href="#fig2">Figure 2</a>A, dotted red line). Moreover, linear regression analysis modeling the relationship between GFP intensity to immunofluorescent intensity will reveal how the determined threshold relates to endogenous protein levels (<a href="#fig2">Figures 2</a>B and 2C).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While this protocol was derived based on the fundamental principles of phase separation, the specific steps presented here have been optimized for studies of G3BP1 and it is possible that other proteins of interest may require additional optimization and modifications. As noted previously, some limitations of this study include the need for cells lacking the gene of interest as well as the availability of specific antibodies. In addition, this protocol relies on the use of a GFP-tag, which could alter protein dynamics. Therefore, care should be taken to demonstrate that the addition of a GFP-tag does not significantly affect protein function. As with any experimental technique, it is important to validate any findings made with this protocol with additional independent methods. For investigators seeking to quantify absolute intracellular protein concentrations, we suggest methods such as mass spectroscopy or fluorescence correlation spectroscopy (<a href="#bib1">Beck et al., 2011</a>; <a href="#bib3">Politi et al., 2018</a>; <a href="#bib4">Unwin, 2010</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Spontaneous stress granule formation.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Check that the microscope is properly equilibrated to 37°C and 5% CO<sub>2</sub>. Avoid exposing cells to extended periods out of the incubator or microscope cage. Reduce the amount of DNA used during the initial transfection or refer to the manufacturer’s instructions.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The slide shifts out of desired stage location.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Proper sample stabilization is critical for obtaining high quality images. Use tight-fitting sample holders. Remove the slide cover before imaging to minimize handling during sodium arsenite treatment.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Too few cells for imaging.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Confirm cell confluency prior to imaging, cells may need to be seeded at a higher confluency prior to transfection. Cell types that do not readily adhere to tissue culture plates may benefit from coating plates with a binding agent (e.g., poly-lysine). During immunostaining, add solutions and aspirate gently to prevent cells from lifting off. In addition, ensure that cells remain hydrated, particularly if incubating cells with primary antibody 12–18 h.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Low R<sup>2</sup> value.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>The R<sup>2</sup> value quantifies the strength of a linear relationship. A low R<sup>2</sup> value indicates a poor linear relationship between the GFP and the immunofluorescent intensities and suggests that the linear regression line should not be used to determine endogenous protein levels. Optimize the protocol such that GFP is expressed at varying levels and detected within a linear dynamic range.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Beck, M., Schmidt, A., Malmstroem, J., Claassen, M., Ori, A., Szymborska, A., Herzog, F., Rinner, O., Ellenberg, J., and Aebersold, R. (2011). The quantitative proteome of a human cell line. Mol. Syst. Biol. <i>7</i>, 549. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00015-0/sref1">View at publisher</a></p>
<p id="bib2">Lee, A.K., Klein, J., Fon Tacer, K., Lord, T., Oatley, M.J., Oatley, J.M., Porter, S.N., Pruett-Miller, S.M., Tikhonova, E.B., Karamyshev, A.L., et al. (2020). Translational repression of G3BP in cancer and germ cells suppresses stress granules and enhances stress tolerance. Mol. Cell <i>79</i>, 645-659.e9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00015-0/sref2">View at publisher</a></p>
<p id="bib3">Politi, A.Z., Cai, Y., Walther, N., Hossain, M.J., Koch, B., Wachsmuth, M., and Ellenberg, J. (2018). Quantitative mapping of fluorescently tagged cellular proteins using FCS-calibrated four-dimensional imaging. Nat. Protoc. <i>13</i>, 1445-1464. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00015-0/sref3">View at publisher</a></p>
<p id="bib4">Unwin, R.D. (2010). Quantification of proteins by iTRAQ. Methods Mol. Biol. <i>658</i>, 205-215. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00015-0/sref4">View at publisher</a></p>
<p id="bib5">Yang, P., Mathieu, C., Kolaitis, R.-M., Zhang, P., Messing, J., Yurtsever, U., Yang, Z., Wu, J., Li, Y., Pan, Q., et al. (2020). G3BP1 is a tunable switch that triggers phase separation to assemble stress granules. Cell <i>181</i>, 325-345.e8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00015-0/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, P. Ryan Potts (<a href="mailto:rpotts01@amgen.com">rpotts01@amgen.com</a>).</p>
<h4>Materials Availability</h4>
<p>All materials generated in this study are available through request but may require a completed Materials Transfer Agreement.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was partially supported by the <a href="https://doi.org/10.13039/100000048">American Cancer Society</a> Research Scholar Award 181691010 (P.R.P.). We thank Dr. J. Paul Taylor for generously providing the G3BP knockout U2OS cell line and the EGFP-C3-G3BP1 plasmid.</p>
<h3>Author Contributions</h3>
<p>P.R.P. and A.K.L. conceptualized the study and designed the experiments. A.K.L. and P.R.P. wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>P.R.P. is an employee and shareholder of Amgen, Inc.</p>
</section>
</article>