<article>
<section>
<h1 id="header">A protocol combining breath testing and <i>ex vivo</i> fermentations to study the human gut microbiome</h1>
<p><time datetime="2020-12-25">Published: December 25, 2020</time></p>
<p>Laura Payling,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">9</a>,<a href="#fn2">10</a>,<a href="#cor1">*</a></sup> Nicole C. Roy,<sup><a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a></sup> Karl Fraser,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff6">6</a></sup> Simon M. Loveday,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Ian M. Sims,<sup><a href="#aff7">7</a></sup> Peter H. Janssen,<sup><a href="#aff3">3</a></sup> Stefan J. Hill,<sup><a href="#aff8">8</a></sup> Laura G. Raymond,<sup><a href="#aff8">8</a></sup> and Warren C. McNabb<sup><a href="#aff2">2</a>,<a href="#aff6">6</a></sup></p>
<p id="aff1"><sup>1</sup>School of Food and Advanced Technology, Massey University, Palmerston North 4410, New Zealand</p>
<p id="aff2"><sup>2</sup>Riddet Institute, Massey University, Palmerston North 4410, New Zealand</p>
<p id="aff3"><sup>3</sup>Food and Bio-Based Products, AgResearch Limited, Tennent Drive, Palmerston North 4410, New Zealand</p>
<p id="aff4"><sup>4</sup>Department of Nutrition, University of Otago, Dunedin 9016, New Zealand</p>
<p id="aff5"><sup>5</sup>Liggins Institute, The University of Auckland, Auckland 1023, New Zealand</p>
<p id="aff6"><sup>6</sup>High-Value Nutrition National Science Challenge, Auckland 1023, New Zealand</p>
<p id="aff7"><sup>7</sup>Ferrier Research Institute, Victoria University of Wellington, Wellington 5010, New Zealand</p>
<p id="aff8"><sup>8</sup>Advanced Chemical Characterisation, Scion, Te Papa Tipu Innovation Park, Rotorua 3010, New Zealand</p>
<p id="fn1"><sup>9</sup>Technical Contact</p>
<p id="fn2"><sup>10</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:l.payling@massey.ac.nz">l.payling@massey.ac.nz</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100227">10.1016/j.xpro.2020.100227</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes the application of breath testing and <i>ex vivo</i> fermentations to study the association between breath methane and the composition and functionality of the gut microbiome. The protocol provides a useful systems biology approach for studying the gut microbiome in humans, which combines standardized methods in human breath testing and fecal sampling. The model described is accessible and easy to repeat, but its relative simplicity means that it can deviate from human physiological conditions.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A systems biology approach for studying the human gut microbiome</li>
<li>Participants are breath tested and fecal samples are collected</li>
<li>Fecal samples are used to inoculate an <i>ex vivo</i> model of the colon</li>
<li>Samples are taken over time and analyzed using a multi-omics approach</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/337-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Prior to commencing the protocol, recruit healthy adult participants using health screening and breath testing. This should only be done after ethics approval from the appropriate institution. Next, prepare fecal collection kits. These will be used to collect fecal samples from participants for inoculating the ex vivo fermentation model. Finally, prepare the solutions and standards for the sulfide assay, and set up the apparatus for the ex vivo fermentation model.</p>
<h3 id="sec1.1">Recruit participants</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 weeks</div>
<ol>
<li>Participant recruitment
<ol type="a">
<li>Distribute a recruitment letter, poster, or email to the locality</li>
<li>On expression of interest, provide participants with detailed study information</li>
<li>Ask interested participants to provide informed consent</li>
</ol>
</li>
<li>Health screening
<ol type="a">
<li>Ask participants to complete an online health screening questionnaire
<ol type="i">
<li>Review study inclusion and exclusion criteria to confirm eligibility</li>
<li>Collect additional metadata including age, ethnicity, gender, height, and bodyweight (to calculate BMI), and frequency and quality of bowel movements according to the Bristol Stool Chart (<a href="#bib18">Lewis and Heaton, 1997</a>)</li>
</ol>
</li>
</ol>
</li>
<li>Breath testing
<ol type="a">
<li>Fast participants for 12 h</li>
<li>Collect a sample of alveolar air using an AlveoSampler kit
<ol type="i">
<li>The participant breathes into the 400 mL discard bag which collects “dead space” air from the airway passages at the end of the previous inhalation</li>
<li>When the discard bag is full, within the same breath, draw open a syringe attached to the mouthpiece to collect 30 mL of alveolar air</li>
</ol>
</li>
<li>Analyze samples within 1 h of collection OR transfer the alveolar air from the collection syringe into a sample holding bag which provides seven days of sample stability
<ol type="i">
<li>Analyze samples for methane, hydrogen, and carbon dioxide on a Quintron Breathtracker</li>
<li>Use the carbon dioxide concentration to check for atmospheric air contamination</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Consider participants excreting ≥ 5 ppm methane as high methane excreters, and those excreting &lt; 5 ppm as low methane excreters (<a href="#bib23">Rezaie et al., 2015</a>).</div>
<ol start="4">
<li>Metadata collection: Provide participants with a three-day food and hydration diary to complete prior to fecal sample collection.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The validated resource guide for dietary assessment by the Food and Agriculture Organization of the United Nations provides a useful template, including details of meals, snacks, and drinks, the time and location of consumption, food or drink type, mass, weight or volume, preparation method and brand, and details of dietary supplements (<a href="#bib8">FAO, 2018</a>). Ask participants to record whether this is representative of their typical diet over the last three months and, if the answer is no, to describe how these three days differ.</div>
<h3 id="sec1.2">Prepare fecal collection kits</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This method is based on the International Human Microbiome Standards Protocol “IHMS_SOP 02 V2” with minor modifications (<a href="#bib7">Dore et al., 2015</a>).</p>
<ol start="5">
<li>Arrange the following items in a cool box (<a href="#fig1">Figure 1</a>)
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/337-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The fecal collection kit provided to participants</div>
</figcaption>
</figure>
<ol type="a">
<li>Fisherbrand Commode Specimen Collection System (<a href="#fig2">Figure 2</a>) lined with a plastic bag
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/337-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. The Fisherbrand Commode Specimen Collection System used for safe and efficient collection of fecal samples (image credit: Fisher Scientific)</div>
</figcaption>
</figure>
</li>
<li>An anaerobic sachet</li>
<li>3 mL water for sachet activation</li>
<li>Gloves</li>
<li>An ice pack</li>
<li>A plastic bag</li>
<li>Kit usage instructions (as follows)
<ol type="i">
<li>The participant provides the sample at home inside the plastic bag lining the Specimen Collection system</li>
<li>Fold the plastic bag closed, but not sealed</li>
<li>Place the anaerobic sachet on top, and activate by adding the water</li>
<li>Seal the Specimen Collection System container immediately</li>
<li>Place the sealed Specimen Collection System container inside the cool box with the ice pack and collect or deliver to the research facility within 1 h of defecation</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Prepare solutions and standards for sulfide assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Prepare solutions and standards for the sulfide assay and plot the calibration curve.</p>
<ol start="6">
<li>Prepare 2% zinc acetate solution with distilled water.</li>
<li>Prepare 0.2% 4-amino-N,-N-dimethylaniline HCl<sub>2</sub> in 20% H<sub>2</sub>SO<sub>4</sub> (diamine solution).</li>
<li>Prepare 0.5% FeNH<sub>4</sub>(SO<sub>4</sub>)<sub>2</sub> in 2% H<sub>2</sub>SO<sub>4</sub> (Fe solution).</li>
<li>To generate sulfide standards:
<ol type="a">
<li>Select sodium sulfide nonahydrate crystals (Na<sub>2</sub>S·9H<sub>2</sub>O) with plastic tweezers</li>
<li>Under nitrogen gas flow, wash with degassed water and blot dry</li>
<li>Immediately transfer to a stoppered weighing bottle containing 5–10 mL degassed reagent water</li>
<li>Repeat until the desired amount of sodium sulfide is prepared</li>
<li>Use this to prepare a known concentration of sulfide stock solution</li>
<li>Analyze stock solutions as detailed from “Sample analysis” 6c</li>
<li>Use data to plot a calibration curve</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 7.5 g Na<sub>2</sub>S·9H<sub>2</sub>O in 1 L water will give a stock solution with a concentration of 1 mg S<sup>2−</sup>/mL.</div>
<div class="note">
<span class="note-title">Note:</span> Sodium sulfide crystals are susceptible to oxidation, hence the washing step is used to remove the oxidized layer. Yellow deposits on the crystals indicate oxidation to elemental sulfur. Colored flecks on the surface indicate metal sulfide contamination.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Sulfide standards can be prepared later in the protocol after the <i>ex vivo</i> fermentations, prior to the analysis of sulfide samples.</div>
<h3 id="sec1.4">Set up <i>ex vivo</i> fermentation model</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<ol start="10">
<li>Weigh 100 mg fermentation substrate into 50 mL sterile serum bottles in duplicate
<ol type="a">
<li>Include positive blanks as well as test substrate treatments (<a href="#fig3">Figure 3</a>)
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/337-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Diagram of the setup of <i>ex vivo</i> fermentation bottles (created with <a href="http://BioRender.com">BioRender.com</a>)</div>
<p>Treatment 1 (negative blank): 5 mL fecal inoculum and 5 mL anaerobic sodium phosphate buffer.</p>
<p>Treatment 2 (positive blank 1): 10 mL anaerobic sodium phosphate buffer and 100 mg substrate 1.</p>
<p>Treatment 3 (positive blank 2): 10 mL anaerobic sodium phosphate buffer and 100 mg substrate 2.</p>
<p>Treatment 4: 5 mL fecal inocula, 5 mL anaerobic sodium phosphate buffer and 100 mg substrate 1.</p>
<p>Treatment 5: 5 mL fecal inocula, 5 mL anaerobic sodium phosphate buffer and 100 mg substrate 2.</p>
</figcaption>
</figure>
</li>
<li>Set aside empty sterile serum bottles for negative blanks</li>
</ol>
</li>
<li>Prepare 1 L sterile sodium phosphate buffer (0.2 M)
<ol start="3" type="a">
<li>Include resazurin (1 mg/L) as a marker of oxygen presence</li>
</ol>
</li>
<li>Place serum bottles and sodium phosphate buffer into the anaerobic workstation at least 24 h before the experiment is due to commence.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Resazurin will appear blue initially; once it begins to reduce it will appear pink. When it is fully de-oxygenated it will be colorless. This is when the buffer is ready to use <a href="#sec6.1">Troubleshooting Problem 1</a>.</div>
<div class="note">
<span class="note-title">Note:</span> Use one set of duplicate 50 mL serum bottles for each sampling time point (six time points). Sampling should not remove more than 2.5% of the total working volume of the fermentation to avoid impacting the physiochemical conditions of the fermentation (<a href="#bib34">Yen et al., 2015</a>). When this is not possible with small scale fermentations and multiple time points, replicates for each time point are employed.</div>
<div class="note">
<span class="note-title">Note:</span> Ensure the temperature of the anaerobic workstation is set to 37°C. Adding buffer to the workstation at least 24 h in advance allows for deoxygenation and pre-warming.</div>
<div class="optional">
<span class="optional-title">Optional:</span> The minimum number of serum bottles required for the <i>ex vivo</i> model is 16 (negative blanks of fecal inocula only). Each substrate requires an additional 24 bottles (including positive blanks at the start and end of the experiment), assuming that all treatments are completed in duplicate (<a href="#tbl1">Table 1</a>). The number of substrates, time points, and replicates can be modified to optimize the scale of the work.</div>
<table id="tbl1">
<caption>Table 1. Setup of fermentation bottles by time point and treatment for two test substrates (64 bottles total)</caption>
<thead>
<tr>
<th rowspan="2">Time point (h)</th>
<th colspan="5">Treatment</th>
</tr>
<tr>
<th>Treatment 1: negative blank (inoculum only)</th>
<th>Treatment 2: substrate 1 positive blank</th>
<th>Treatment 3: substrate 2 positive blank</th>
<th>Treatment 4: substrate 1 + inoculum</th>
<th>Treatment 5: substrate 2 + inoculum</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>XXXX</td>
<td>XXXX</td>
<td>XXXX</td>
<td>XXXX</td>
<td>XXXX</td>
</tr>
<tr>
<td>3</td>
<td>XX</td>
<td> </td>
<td> </td>
<td>XX</td>
<td>XX</td>
</tr>
<tr>
<td>6</td>
<td>XX</td>
<td> </td>
<td> </td>
<td>XX</td>
<td>XX</td>
</tr>
<tr>
<td>12</td>
<td>XX</td>
<td> </td>
<td> </td>
<td>XX</td>
<td>XX</td>
</tr>
<tr>
<td>24</td>
<td>XX</td>
<td> </td>
<td> </td>
<td>XX</td>
<td>XX</td>
</tr>
<tr>
<td>48</td>
<td>XXXX</td>
<td>XXXX</td>
<td>XXXX</td>
<td>XXXX</td>
<td>XXXX</td>
</tr>
</tbody>
</table>
<p class="table-legend">Each X indicates 1 serum bottle. This includes 16 negative blank bottles, plus 24 additional bottles per substrate (positive blank and test treatment including fecal inocula). Two additional replicates are included at 0 h and 48 h for the quantification of dry matter and organic matter.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Sodium phosphate monobasic anhydrous</td>
<td>Sigma</td>
<td>71496</td>
</tr>
<tr>
<td>Sodium phosphate dibasic heptahydrate</td>
<td>Sigma</td>
<td>S9390</td>
</tr>
<tr>
<td>Resazurin</td>
<td>Sigma</td>
<td>R7017</td>
</tr>
<tr>
<td>L-Cysteine</td>
<td>Sigma</td>
<td>168149</td>
</tr>
<tr>
<td>Zinc acetate</td>
<td>Merck</td>
<td>383317</td>
</tr>
<tr>
<td>Sodium sulfide nonahydrate crystals</td>
<td>Merck</td>
<td>208043</td>
</tr>
<tr>
<td>4-Amino-N, N-dimethylaniline HCl<sub>2</sub>
</td>
<td>Sigma/Supelco</td>
<td>07750</td>
</tr>
<tr>
<td>H<sub>2</sub>SO<sub>4</sub> (sulfuric acid)</td>
<td>Sigma</td>
<td>258105</td>
</tr>
<tr>
<td>FeNH<sub>4</sub>(SO<sub>4</sub>)<sub>2</sub> (ammonium iron sulfate)</td>
<td>Merck</td>
<td>221260</td>
</tr>
<tr>
<td>N-Methyl-N-(trimethylsilyl)-trifluoroacetamide (MSTFA)</td>
<td>Sigma/Supelco</td>
<td>69479</td>
</tr>
<tr>
<td>RNAlater stabilization solution</td>
<td>Merck</td>
<td>R0901</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RNeasy Mini Kit</td>
<td>QIAGEN</td>
<td>74104</td>
</tr>
<tr>
<td>NucleoSpin Soil Kit</td>
<td>Fisher Scientific</td>
<td>12718562</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>R and R packages; Edge R, MixOmics, GenoPlotR, Tidyverse, Pheatmap</td>
<td>R Foundation for Statistical Computing, 2013</td>
<td><a href="http://www.r-project.org/">http://www.r-project.org/</a></td>
</tr>
<tr>
<td>MetaboAnalyst</td>
<td><a href="#bib6">Chong et al., 2019</a></td>
<td><a href="https://doi.org/10.1002/cpbi.86">https://doi.org/10.1002/cpbi.86</a></td>
</tr>
<tr>
<td>SIMCA</td>
<td>Sartoruis, Germany, 2020</td>
<td><a href="https://landing.umetrics.com/downloads-simca">https://landing.umetrics.com/downloads-simca</a></td>
</tr>
<tr>
<td>SAS</td>
<td>SAS Institute Inc., 2020</td>
<td><a href="https://www.sas.com/en_nz/software/university-edition.html">https://www.sas.com/en_nz/software/university-edition.html</a></td>
</tr>
<tr>
<td>SortMeRNA</td>
<td>Bonsai bioinformatics</td>
<td><a href="https://bioinfo.lifl.fr/RNA/sortmerna/">https://bioinfo.lifl.fr/RNA/sortmerna/</a></td>
</tr>
<tr>
<td>PEAR</td>
<td>Heidelberg Institute for Theoretical Studies</td>
<td><a href="https://cme.h-its.org/exelixis/web/software/pear/">https://cme.h-its.org/exelixis/web/software/pear/</a></td>
</tr>
<tr>
<td>Megan</td>
<td>Universitat Tubingen</td>
<td><a href="https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics/software/megan6/">https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics/software/megan6/</a></td>
</tr>
<tr>
<td>Diamond</td>
<td>Universitat Tubingen</td>
<td><a href="https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics/software/diamond/">https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics/software/diamond/</a></td>
</tr>
<tr>
<td>Prodigal</td>
<td>University of Tennessee</td>
<td><a href="http://gensoft.pasteur.fr/docs/prodigal/2.50/_README">http://gensoft.pasteur.fr/docs/prodigal/2.50/_README</a></td>
</tr>
<tr>
<td>Kraken</td>
<td>John Hopkins University</td>
<td><a href="https://ccb.jhu.edu/software/kraken/">https://ccb.jhu.edu/software/kraken/</a></td>
</tr>
<tr>
<td>Vizbin</td>
<td>Cedric Laczny</td>
<td><a href="http://claczny.github.io/VizBin/">http://claczny.github.io/VizBin/</a></td>
</tr>
<tr>
<td>MaxBin</td>
<td>Yu-Wei Wu</td>
<td><a href="https://sourceforge.net/projects/maxbin/">https://sourceforge.net/projects/maxbin/</a></td>
</tr>
<tr>
<td>Fast QC</td>
<td>Babraham Bioinformatics</td>
<td><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></td>
</tr>
<tr>
<td>Trimmomatic</td>
<td>The Usadel Lab</td>
<td><a href="http://www.usadellab.org/cms/?page=trimmomatic">http://www.usadellab.org/cms/?page=trimmomatic</a></td>
</tr>
<tr>
<td>IBDA-UD</td>
<td>The University of Hong Kong</td>
<td><a href="https://i.cs.hku.hk/%7Ealse/hkubrg/projects/idba_ud/">https://i.cs.hku.hk/∼alse/hkubrg/projects/idba_ud/</a></td>
</tr>
<tr>
<td>Bowtie2</td>
<td>John Hopkins University</td>
<td><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td>
</tr>
<tr>
<td>SAMSA2</td>
<td>Samuel Westreich</td>
<td><a href="https://github.com/transcript/samsa2">https://github.com/transcript/samsa2</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Anaerobic sachets</td>
<td>Merck</td>
<td>113807</td>
</tr>
<tr>
<td>Fisherbrand Commode specimen collection system</td>
<td>Fisher Scientific</td>
<td>02544208</td>
</tr>
<tr>
<td>AlveoSampler kit</td>
<td>QuinTron</td>
<td>QT00827-P</td>
</tr>
<tr>
<td>Anaerobic workstation</td>
<td>Don Whitley Scientific</td>
<td><a href="https://www.dwscientific.com/whitley-anaerobic-workstations/whitley-a85-workstation">https://www.dwscientific.com/whitley-anaerobic-workstations/whitley-a85-workstation</a></td>
</tr>
<tr>
<td>QuinTron Breathtracker</td>
<td>QuinTron</td>
<td>QT05000-M</td>
</tr>
<tr>
<td>Exetainer vials 5.9 mL</td>
<td>Labco</td>
<td>818W</td>
</tr>
<tr>
<td>QuinTron Sample Holding Bag</td>
<td>QuinTron</td>
<td>QT00842-P</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Prepare fecal inocula</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Receive the fecal sample from the participant, take aliquots for later DNA extraction and combine remaining fecal matter with sodium phosphate buffer to produce a 16% w/v inoculum for <i>ex vivo</i> fermentation experiments.</p>
<ol>
<li>Immediately transfer the sample to the anaerobic workstation
<ol type="a">
<li>Partition five aliquots of 0.5 g feces into 2 mL cryovials and freeze immediately at −80°C (for DNA extraction)</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Transfer 96 g of the remaining sample into a sterile sample collection pottle
<ol type="a">
<li>Add 300 mL sterile pre-reduced sodium phosphate buffer</li>
<li>Seal the pottle with a Teflon-coated magnetic stirrer bar inside and gently mix on a magnetic stirrer plate for 5 min in anaerobic conditions
<div class="critical">
<span class="critical-title">Critical:</span> Treat unscreened fecal samples as potentially infectious. Handle under strict Physical Containment 2 procedures. Wear double-gloves, a face mask, and eye protection always. Clean surfaces thoroughly with 75% ethanol.</div>
<div class="note">
<span class="note-title">Note:</span> This will produce an inoculum of 32% w/v.</div>
<div class="note">
<span class="note-title">Note:</span> Gentle stirring helps to homogenize the inoculum. Avoid vigorous stirring, which can reduce methanogen viability.</div>
</li>
<li>Strain the inoculum through a double layer of cheesecloth with 16 threads per cm in both directions, within the anaerobic workstation
<div class="note">
<span class="note-title">Note:</span> This method of filtration helps to ensure that the inoculum contains both attached and unattached microorganisms (<a href="#bib31">Williams et al., 2005</a>).</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Prepare fermentation bottles</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Add the fecal inocula and the sodium phosphate buffer to the serum bottles to begin the fermentations.</p>
<ol start="3">
<li>In the anaerobic workstation, add 5 mL sodium phosphate buffer to all serum bottles; add an additional 5 mL sodium phosphate buffer to positive blanks.</li>
<li>Add 5 mL fecal inoculum to all serum bottles containing test substrates, and to the negative blanks (final concentration 16% w/v inocula).</li>
<li>Seal the serum bottles with a rubber bung and crimp cap.</li>
<li>Remove from the anaerobic workstation and immediately transfer to a shaking incubator at 37°C and 50 rpm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each fermentation bottle contains 10 mL fluid in total with 40 mL headspace.</div>
<h3 id="sec2.3">Harvest fermentation bottles</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 48 h</div>
<p>At the following time points (hours of fermentation), harvest one set (duplicate) of bottles; 0, 3, 6, 10, 24, and 48 for sampling.</p>
<ol start="7">
<li>Remove the harvested bottles from the incubator and immediately transfer to a water bath at 37°C.</li>
<li>Use a gas pressure probe to measure the gas pressure in the bottle, via a needle through the bung. <a href="#sec6.3">Troubleshooting Problem 2</a>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Before the experiment, calibrate the pressure probe by injecting air into a 50 mL serum bottle with 10 mL water inside. Inject known volumes of air and measure the probe with the pressure to create a calibration curve. Conduct the calibration at 37°C. Use the calibration curve to calculate mL of gas produced.</div>
<ol start="9">
<li>Remove 2 mL headspace gas using a syringe and inject into a 5.9 mL Exetainer vial for later GC analysis of hydrogen and methane.</li>
<li>Remove 0.6 mL fermentation fluid from all bottles except those for dry matter and organic matter quantification
<ol type="a">
<li>Inject 0.5 mL into a five-times volume of RNAlater Stabilization Solution</li>
<li>Inject 0.1 mL into a 1.5 mL Eppendorf tube containing 1 mL zinc acetate solution (2%) (for sulfide determination)</li>
<li>Mix each sample and freeze at −80°C</li>
</ol>
</li>
<li>Plunge samples into an ice water bath for 20 min to slow bacterial activity (<a href="#bib10">Feng et al., 2018</a>; <a href="#bib28">Warren et al., 2018</a>).</li>
<li>Use a pH probe to measure and record the pH of the samples. <a href="#sec6.5">Troubleshooting Problem 3</a>
</li>
<li>Transfer samples to 15 mL centrifuge tubes
<ol type="a">
<li>Centrifuge at 6,750 × <i>g</i> for 15 min at 4°C</li>
<li>Remove two aliquots of 1.5 mL supernatant for later metabolite profiling</li>
<li>Centrifuge the remaining supernatant at 24,000 × <i>g</i> for 10 min at 4°C</li>
<li>Aliquot the remaining supernatant into 1.5 mL fractions</li>
<li>Homogenize the pellet with a spatula and split into two equal aliquots</li>
<li>Freeze all samples at −80°C</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Analyze samples for sulfide determination within 3 months of collection to minimize the loss of sulfide</div>
<h3 id="sec2.4">Sample analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8–12 weeks</div>
<p>Prepare samples for the following analyses; metagenomics, metabolomics, metatranscriptomics, organic acids, headspace gases, sulfide, dry matter, and organic matter.</p>
<ol start="14">
<li>Extract DNA from the fermentation pellet and fecal aliquots and conduct whole-genome sequencing
<ol type="a">
<li>Use the NucleoSpin Soil kit following the <a href="https://www.takarabio.com/assets/documents/User%20Manual/NucleoSpin%20Soil%20Genomic%20DNA%20Purification%20User%20Manual_Rev_05.pdf">manufacturer’s instructions</a> with the addition of a bead-beating step for 4 min to extract DNA from the samples (BioSpec Mini-Beadbeater 96, Bartlesville, OK, United States) (<a href="#bib36">Young et al., 2020</a>)</li>
<li>Prepare libraries using the Illumina TruSeq DNA PCR-free kit</li>
<li>Sequence DNA on an Illumina MiSeq with 24–30 million paired-end reads per run (read length 2 × 150 base pairs), and output of 3–4.5 gigabytes per run</li>
</ol>
</li>
</ol>
<ol start="15">
<li>Analyze one set of supernatant samples by Liquid Chromatography-Mass Spectrometry (LC-MS) and Gas Chromatography-Mass Spectrometry (GC-MS) to determine the relative abundance of all detectable metabolites. For LC-MS (<a href="#bib5">Vanden Bussche et al., 2015</a>):
<ol type="a">
<li>Thaw supernatant samples and centrifuge at 13,300 × <i>g</i> for 5 min</li>
<li>Syringe filter sample through a 13 mm diameter, 0.22 μm pore size Polyvinylidene Fluoride filter</li>
<li>Dilute sample 1:5 with ultrapure water</li>
<li>Run sample through the LC-MS with an HSS-T3 C18 column (1.8 μm, 150 mm × 2.1 mm)
<div class="note">
<span class="note-title">Note:</span> This is a reversed-phase column which will not retain highly polar molecules, allowing semi-polar and non-polar molecules to elute for MS analysis.</div>
</li>
<li>Run the mass spectrometer in polarity switching mode, with sheath, auxiliary and sweep gas at 50, 25, and 5 arbitrary units, respectively
<ol type="i">
<li>Set the heater and capillary temperatures to 350°C and 250°C</li>
<li>Set the tube lens, skimmer, capillary, and spray voltages to 60 V, 20 V, 90 V, and 5 kV (±), respectively</li>
<li>Apply a scan range of <i>m/z</i> 50−800, and a resolution 100 000 FWHM at 1 Hz<br> For GC-MS (<a href="#bib9">Farag et al., 2020</a>):</li>
</ol>
</li>
<li>To 200 μL sample, add 800 μL of acetonitrile/methanol (1:1 v/v) mixture</li>
<li>Incubate at 4°C for 30 min</li>
<li>Centrifuge sample at 12,000 × <i>g</i> for 4 min</li>
<li>Take 100 μL of the sample, and evaporate to dryness under a stream of nitrogen</li>
<li>Add 150 μL N-methyl-N-(trimethylsilyl)-trifluoroacetamide to the residue and incubate at 60°C for 40 min</li>
<li>Run the samples on the GC-MS using an Rtx-5MS column (30 m length, 0.25 mm inner diameter, and 0.25 μm film)</li>
<li>Set injections to a 1:15 split mode</li>
<li>Set the injector at 280°C, column oven at 80°C for 2 min, rate 5°C/minute until 315°C, then maintain for 12 min</li>
<li>Use Helium as the carrier gas at 1 mL/minute</li>
<li>Set transfer line and ion-source temperatures to 280°C and 180°C, respectively</li>
</ol>
</li>
</ol>
<ol start="16">
<li>Extract total RNA from the fermentation fluid stored in RNAlater Stabilization Solution and conduct RNA-seq analyses
<ol type="a">
<li>Use the RNeasy Mini Kit to extract total RNA from samples, according to the <a href="https://www.qiagen.com/es/resources/resourcedetail?id=14e7cf6e-521a-4cf7-8cbc-bf9f6fa33e24&amp;lang=en">manufacturer’s instructions</a>
</li>
<li>Prepare libraries using the Illumina TruSeq Stranded mRNA library preparation kit</li>
<li>Analyze mRNA on an Illumina Miseq generating short-read data, with 44–50 million paired-end reads per run (read length 2 × 75 base pairs), and an output of 3–3.8 Gigabytes per run</li>
</ol>
</li>
<li>Analyze gas samples on a GC fitted with a TCD to measure hydrogen and methane concentration.</li>
<li>Prepare one set of supernatant samples according to the referenced protocol (<a href="#bib24">Richardson et al., 1989</a>) for analysis of organic acids using a GC-FID.</li>
<li>For the sulfide assay:
<ol type="a">
<li>Thaw frozen samples</li>
<li>Add 0.1 mL sample to 1 mL distilled water</li>
<li>Add 0.1 mL diamine solution and 0.1 mL Fe solution and mix</li>
<li>Stand for 10 min at 21°C–27°C</li>
<li>Read and record the optical density (OD<sub>670</sub>) <a href="#sec6.7">Troubleshooting Problem 4</a>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The rate of air-water transfer of sulfide and the final equilibrium are affected by both temperature and pH. If samples differ in either of these factors, a calculation to account for variances in dissociation and emission can be used (<a href="#bib35">Yongsiri et al., 2004</a>).</div>
<ol start="20">
<li>For entire fermentation samples kept for dry matter and organic matter quantification:
<ol type="a">
<li>Dry the total fermentation (supernatant and pellet) to a constant temperature in an oven at 103°C to determine the dry matter content (ISO 6496 1999)</li>
<li>Ash the dry matter samples in a furnace at 550°C to determine organic matter content (ISO 5984 1978)</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Only time points from the start and end of the fermentation need to undergo dry matter and organic matter quantification (0 and 48 h).</div>
<div class="optional">
<span class="optional-title">Optional:</span> Nuclear Magnetic Resonance Spectroscopy (NMR) may be used as an initial screening tool for differences in metabolites occurrence. It can inform the targeting strategy for LC-MS and GC-MS analyses. The fermentation supernatant is suitable for solution-state NMR. After freeze-drying, the fermentation pellet is suitable for solid-state NMR analysis which can indicate changes in the chemical composition of the test substrates used.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Gas pressure, pH, and dry matter and organic matter changes are all measures to validate the occurrence of microbial fermentation in the model.</p>
<p>The starting pH of the fermentation is approximately 6.5. In positive blank treatments, the pH should remain similar throughout the fermentation, and there should be little or no change in gas pressure. This indicates that little or no fermentation is occurring. In the negative blank treatments, there may be some initial fermentation which may be observed by a drop in pH and an increase in gas pressure from 0–6 h but there should be little change from 6–48 h. In the test substrate treatments, the change in pH and gas pressure will depend on the fermentability of the substrate. For fermentable carbohydrates, expect an increase in gas pressure and a drop in pH for the first 24 h, followed by a pH plateau by 48 h. The pH may drop to &lt;6 if the rate and extent of fermentation are large.</p>
<p>In the positive and negative blank treatments, there should be little or no change in the quantities of dry matter or organic matter. However, the substrate test treatments should show a reduction in dry matter and organic matter as the substrates are fermented by the microbe’s present.</p>
<p>The DNA extraction method described yields 5–10 ng/μL DNA for both stool and fermentation pellet samples.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec4.1">Metadata</h3>
<p>Enter all data into a database using a unique ID number for each participant. Handle personal data with strict care to comply with privacy and confidentiality guidelines.</p>
<p>Check food and hydration records for completeness, and approach participants to complete any records where further information is required and available. Records should be considered suitable or unsuitable for analysis based on the completeness and quality of the data present. Check data for outliers and extreme values, and review on a case-by-case basis. Generate compositional data for food and drink using a national Food Composition Database. Analyze data by indices that assess the impact of dietary nutrient content (e.g., dietary fiber, protein, fat) and those that represent dietary pattern analysis (e.g., consumption of whole foods, fruits, and vegetables, red meat, seafood, and diet diversity etc.). Conduct association analyses between the metadata and metagenomic taxonomic data.</p>
<h3 id="sec4.2">Metagenomics</h3>
<h4 id="sec4.2.1">Quality check</h4>
<p>Check the quality of the libraries using Fast QC (<a href="#bib1">Babraham Institute, 2019</a>). Trim reads and remove adapters using Trimmomatic (<a href="#bib3">Bolger et al., 2014</a>).</p>
<h4 id="sec4.2.2">Assembly</h4>
<p>Conduct mini co-assemblies using the De Brujin graph assembly approach in IDBA-UD (<a href="#bib20">Peng et al., 2012</a>). Use Bowtie2 to map unassembled reads against the assembly to generate the differential abundance profiles for each contig (<a href="#bib17">Langmead and Salzberg, 2012</a>).</p>
<h4 id="sec4.2.3">Binning</h4>
<p>Bin the data using MaxBin with a minimum contig size of 1,000 base pairs (<a href="#bib33">Wu et al., 2014</a>). Import data into Vizbin and project a t-SNE plot using coverage per contig (using 20 k contig fragments) (<a href="#bib16">Laczny et al., 2015</a>). Use these plots to re-define the boundaries of bins or identify contigs which should be excluded from the dataset.</p>
<h4 id="sec4.2.4">Taxonomy</h4>
<p>Use the profiling tool Kraken to match the DNA to reference databases using a k-mer search (<a href="#bib32">Wood and Salzberg, 2014</a>).</p>
<h4 id="sec4.2.5">Gene prediction and annotation</h4>
<p>Use the Prodigal tool to predict gene sequences (<a href="#bib13">Hyatt et al., 2010</a>). Run Diamond to annotate protein sequences with gene identities using a BLAST against the UniProt database (<a href="#bib2">Bateman, 2019</a>; <a href="#bib4">Buchfink et al., 2014</a>). Use Megan to extract taxonomy information from the annotations, and build a taxonomic and functional overview using the Lowest Common Ancestor method (<a href="#bib12">Huson et al., 2007</a>).</p>
<h4 id="sec4.2.6">Visualization</h4>
<p>Use the following packages in R to import and visualize the data; GenoPlotR, Tidyverse and Pheatmap (<a href="#bib11">Guy et al., 2011</a>; <a href="#bib14">Kolde, 2019</a>; <a href="#bib22">R Core Team, 2019</a>; <a href="#bib30">Wickham et al., 2019</a>). Compare the differential abundance and potential functionality of operational taxonomic units between substrates and methane phenotype.</p>
<h3 id="sec4.3">Metabolites</h3>
<p>Visualize metabolomic data using PCA plots in MetaboAnalyst (online tool, (<a href="#bib6">Chong et al., 2019</a>)) and/or SIMCA (Sartorius, Germany). Analyze data by differential expression between substrates and methane phenotype.</p>
<p>Analyze gas production data, sulfide assay data, and organic acids data by the PROC MIXED method in SAS.</p>
<h3 id="sec4.4">Metatranscriptomics</h3>
<p>Use the SAMSA2 pipeline to process data (<a href="#bib29">Westreich et al., 2018</a>); merge paired-end reads using PEAR (<a href="#bib37">Zhang et al., 2014</a>). Remove low-quality reads using Trimmomatic (<a href="#bib3">Bolger et al., 2014</a>). Remove rRNA reads using SortMeRNA (<a href="#bib15">Kopylova et al., 2012</a>). Annotate data using Diamond with the NCBI RefSeq database and the CAZy database (<a href="#bib4">Buchfink et al., 2014</a>; <a href="#bib19">Lombard et al., 2014</a>; <a href="#bib21">Pruitt et al., 2007</a>). Conduct differential expression analysis according to substrate and methane phenotype in the package edge R (<a href="#bib25">Robinson et al., 2009</a>). Visualize data using Pheatmap (<a href="#bib14">Kolde, 2019</a>).</p>
<h3 id="sec4.5">Data integration</h3>
<p>Combine the metagenomic, metatranscriptomic, and metabolomic data using an integrative multi-omics approach. Conduct visualization and exploration of the datasets, network analysis, and canonical correlations in the R package MixOmics (<a href="#bib26">Rohart et al., 2017</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The relative simplicity of the model means that it can deviate from human physiological conditions. The mostly unregulated changes in pH, redox potential, and community structure mean that conditions can deviate with regards to metabolite concentrations and pH. Furthermore, the model cannot account for the interactions between the gut microbiota and host cells.</p>
<p>The model provides a good screening tool for the interaction between host and dietary factors, but the generalizations that can be made about human microbiome function are limited without further <i>in vivo</i> trials.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The resazurin in the buffer remains blue or pink and does not turn colorless.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>If the resazurin remains blue, no reduction has occurred. Agitate the buffer in the anaerobic workstation, using a magnetic stirrer plate or by shaking, to encourage the buffer to reduce.</p>
<p>Alternatively, use oxygen-free nitrogen and bubble it through the buffer. Once the color turns pink, add 0.5 g/L L-cysteine to the buffer inside the anaerobic workstation, and mix. It will act as a reducing agent to complete deoxygenation and turn the resazurin colorless.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The gas pressure in the fermentation bottles does not accumulate.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Check that the gas probe was calibrated at 37°C. Temperature change alters the volume of the gas.</p>
<p>Ensure that the samples are transferred from the incubator to the water bath swiftly, and that they are both at the same temperature (the purpose of the water bath is to keep samples at a consistent temperature).</p>
<p>Check that bungs are secure in the serum bottles and the crimp cap is fitted well. Submerge a serum bottle under water and see if any gas bubbles appear.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The pH of the samples drops below 5.5 which can be unrepresentative of human colonic conditions.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Check the pH of the starting buffer. Ensure that it is approximately pH 6.5. If it is too low, add NaOH.</p>
<p>If gas is used to deoxygenate the buffer, re-check the pH afterwards. Use nitrogen rather than carbon dioxide for deoxygenation.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Solutions have high turbidity, which affects the spectrophotometric reading.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Clarify the sample by centrifugation at 12,000 × <i>g</i> for 3 min after color development (<a href="#bib27">Strocchi et al., 1992</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Babraham Institute (2019). FastQC. <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>.</p>
<p id="bib2">Bateman, A. (2019). UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. <i>47</i>, D506-D515.</p>
<p id="bib3">Bolger, A.M., Lohse, M., and Usadel, B. (2014). Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics <i>30</i>, 2114-2120.</p>
<p id="bib4">Buchfink, B., Xie, C., and Huson, D.H. (2014). Fast and sensitive protein alignment using DIAMOND. Nat. Methods <i>12</i>, 59-60.</p>
<p id="bib5">Vanden Bussche, J., Marzorati, M., Laukens, D., and Vanhaecke, L. (2015). Validated high resolution mass spectrometry-based approach for metabolomic fingerprinting of the human gut phenotype. Anal. Chem. <i>87</i>, 10927-10934.</p>
<p id="bib6">Chong, J., Wishart, D.S., and Xia, J. (2019). Using MetaboAnalyst 4.0 for comprehensive and integrative metabolomics data analysis. Curr. Protoc. Bioinformatics <i>68</i>, e86.</p>
<p id="bib7">Dore, J., Ehrlich, S., Levenez, F., Pelletier, E., Alberti, A., Bertrand, L., Bork, P., Costea, P.I., Sunagawa, S., Guarner, F., et al. (2015). IHMS_SOP 02 V1: Standard operating procedure for fecal samples self-collection, laboratory analysis handled within 4 hours (x≤ 4 hours).</p>
<p id="bib8">FAO (2018). Dietary Assessment: A resource guide to method selection and application in low resource settings (Rome) (FAO).</p>
<p id="bib9">Farag, M.A., Abdelwareth, A., Sallam, I.E., el Shorbagi, M., Jehmlich, N., Fritz-Wallace, K., Serena Schäpe, S., Rolle-Kampczyk, U., Ehrlich, A., Wessjohann, L.A., et al. (2020). Metabolomics reveals impact of seven functional foods on metabolic pathways in a gut microbiota model. J. Adv. Res. <i>23</i>, 47-59.</p>
<p id="bib10">Feng, G., Flanagan, B.M., Mikkelsen, D., Williams, B.A., Yu, W., Gilbert, R.G., and Gidley, M.J. (2018). Mechanisms of utilization of arabinoxylans by a porcine fecal inoculum: competition and co-operation. Sci. Rep. <i>8</i>, 4546.</p>
<p id="bib11">Guy, L., Kultima, J.R., Andersson, S.G.E., and Quackenbush, J. (2011). GenoPlotR: comparative gene and genome visualization in R. Bioinformatics <i>27</i>, 2334-2335.</p>
<p id="bib12">Huson, D.H., Auch, A.F., Qi, J., and Schuster, S.C. (2007). MEGAN analysis of metagenomic data. Genome Res. <i>17</i>, 377-386.</p>
<p id="bib13">Hyatt, D., Chen, G., Locascio, P.F., Land, M.L., Larimer, F.W., and Hauser, L.J. (2010). Prodigal : prokaryotic gene recognition and translation initiation site identification. BMC Bioinformatics <i>11</i>, 119.</p>
<p id="bib14">Kolde, R. (2019). pheatmap: Pretty Heatmaps [software]. <a href="https://rdrr.io/cran/pheatmap/">https://rdrr.io/cran/pheatmap/</a>.</p>
<p id="bib15">Kopylova, E., Noé, L., and Touzet, H. (2012). SortMeRNA: Fast and accurate filtering of ribosomal RNAs in metatranscriptomic data. Bioinformatics <i>28</i>, 3211-3217.</p>
<p id="bib16">Laczny, C.C., Sternal, T., Plugaru, V., Gawron, P., Atashpendar, A., Margossian, H.H., Coronado, S., der Maaten, L.V., Vlassis, N., and Wilmes, P. (2015). VizBin - an application for reference-independent visualization and human-augmented binning of metagenomic data. Microbiome <i>3</i>, 1-7.</p>
<p id="bib17">Langmead, B. and Salzberg, S.L. (2012). Fast gapped-read alignment with Bowtie 2. Nat. Methods <i>9</i>, 357-359.</p>
<p id="bib18">Lewis, S.J. and Heaton, K.W. (1997). Stool form scale as a useful guide to intestinal transit time. Scand. J. Gastroenterol. <i>32</i>, 920-924.</p>
<p id="bib19">Lombard, V., Golaconda Ramulu, H., Drula, E., Coutinho, P.M., and Henrissat, B. (2014). The carbohydrate-active enzymes database (CAZy) in 2013. Nucleic Acids Res. <i>42</i>, D490-D495.</p>
<p id="bib20">Peng, Y., Leung, H.C.M., Yiu, S.M., and Chin, F.Y.L. (2012). IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth. Bioinformatics <i>28</i>, 1420-1428.</p>
<p id="bib21">Pruitt, K.D., Tatusova, T., and Maglott, D.R. (2007). NCBI reference sequences (RefSeq): A curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res. <i>35</i>, 61-65.</p>
<p id="bib22">R Core Team (2019). R: A Language and Environment for Statistical Computing.</p>
<p id="bib23">Rezaie, A., Chang, B., Chua, K.S., Lin, E.A., and Pimentel, M. (2015). Accurate identification of excessive methane gas producers by a single fasting measurement of exhaled methane: a large-scale database analysis ACG category award. Am. J. Gastroenterol. <i>110</i>, S759-S760.</p>
<p id="bib24">Richardson, A.J., Calder, A.G., Stewart, C.S., and Smith, A. (1989). Simultaneous determination of volatile and non-volatile acidic fermentation products of anaerobes by capillary gas chromatography. Lett. Appl. Microbiol. <i>9</i>, 5-8.</p>
<p id="bib25">Robinson, M.D., McCarthy, D.J., and Smyth, G.K. (2009). edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics <i>26</i>, 139-140.</p>
<p id="bib26">Rohart, F., Gautier, B., Singh, A., and Lê Cao, K.A. (2017). mixOmics: an R package for ‘omics feature selection and multiple data integration. PLoS Comput. Biol. <i>13</i>, 1-19.</p>
<p id="bib27">Strocchi, A., Furne, J.K., and Levitt, M.D. (1992). A modification of the methylene blue method to measure bacterial sulfide production in feces. J. Microbiol. Methods <i>15</i>, 75-82.</p>
<p id="bib28">Warren, F.J., Fukuma, N.M., Mikkelsen, D., Flanagan, B.M., Williams, B.A., Lisle, A.T., Ó Cuív, P., Morrison, M., and Gidley, M.J. (2018). Food starch structure impacts gut microbiome composition. MSphere <i>3</i>, e00086-18.</p>
<p id="bib29">Westreich, S.T., Treiber, M.L., Mills, D.A., Korf, I., and Lemay, D.G. (2018). SAMSA2: a standalone metatranscriptome analysis pipeline. BMC Bioinformatics <i>19</i>, 1-11.</p>
<p id="bib30">Wickham, H., Averick, M., Bryan, J., Chang, W., McGowan, L., François, R., Grolemund, G., Hayes, A., Henry, L., Hester, J., et al. (2019). Welcome to the Tidyverse. J. Open Source Softw. <i>4</i>, 1686.</p>
<p id="bib31">Williams, B.A., Bosch, M.W., Boer, H., Verstegen, M.W.A., and Tamminga, S. (2005). An in vitro batch culture method to assess potential fermentability of feed ingredients for monogastric diets. Anim. Feed Sci. Technol. <i>123–124 Pa</i>, 445-462.</p>
<p id="bib32">Wood, D.E. and Salzberg, S.L. (2014). Kraken: ultrafast metagenomic sequence classification using exact alignments. Genome Biol. <i>15</i>, R46.</p>
<p id="bib33">Wu, Y.-W., Tang, Y.-H., Tringe, S.G., Simmons, B.A., and Singer, S.W. (2014). MaxBin: an automated binning method to recover individual genomes from metagenomes using. Microbiome <i>2</i>, 4904-4909.</p>
<p id="bib34">Yen, S., McDonald, J.A.K., Schroeter, K., Oliphant, K., Sokolenko, S., Blondeel, E.J.M., Allen-Vercoe, E., and Aucoin, M.G. (2015). Metabolomic analysis of human fecal microbiota: a comparison of feces-derived communities and defined mixed communities. J. Proteome Res. <i>14</i>, 1472-1482.</p>
<p id="bib35">Yongsiri, C., Vollertsen, J., and Hvitved-Jacobsen, T. (2004). Effect of temperature on air-water transfer of hydrogen sulfide. J. Environ. Eng. <i>130</i>, 104-109.</p>
<p id="bib36">Young, W., Arojju, S.K., Mcneill, M.R., Rettedal, E., Gathercole, J., Bell, N., and Payne, P. (2020). Feeding bugs to bugs: edible insects modify the human gut microbiome in an in vitro fermentation model. Front. Microbiol. <i>11</i>.</p>
<p id="bib37">Zhang, J., Kobert, K., Flouri, T., and Stamatakis, A. (2014). PEAR: a fast and accurate Illumina Paired-End reAd mergeR. Bioinformatics <i>30</i>, 614-620.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Laura Payling, <a href="mailto:l.payling@massey.ac.nz">l.payling@massey.ac.nz</a>.</p>
<h4>Materials availability</h4>
<p>This protocol does not require new unique reagents or any other materials that have restricted availability.</p>
<h4>Data and code availability</h4>
<p>No datasets have yet been generated. All code described is publicly available and referenced within the manuscript.</p>
<h3>Acknowledgments</h3>
<p>The authors would like to thank Dr. Stefan Muetzel, Dr. Wayne Young, and Dr. Dulantha Ulluwishewa (AgResearch, New Zealand) for their support in study conceptualization. The graphical abstract was created with <a href="http://BioRender.com">BioRender.com</a>. L.P. is supported by a PhD fellowship and funding from the Riddet Institute, through a Centre of Research Excellence funding provided by the New Zealand Tertiary Education Commission. The research is monetarily and materially supported by the Riddet Institute and AgResearch through the AgResearch Strategic Science Investment Fund, contract number A25773. L.P. is the first author of this manuscript and is a PhD student enrolled at The School of Food and Advanced Technology, Massey University.</p>
<h3>Author Contributions</h3>
<p>Conceptualization and Study Design, L.P., N.C.R., K.F., S.L., I.M.S., P.H.J., S.J.H., L.G.R., and W.C.M.; Participant Recruitment and Management, L.P.; Writing – Original Draft, L.P.; Writing –Review &amp; Editing: L.P., N.C.R., K.F., S.L., I.M.S., P.H.J., S.J.H., L.G.R., and W.C.M. All authors read and approved the final manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>