<article>
<section>
<h1 id="header">Expression and purification of fused kinase from insect cells for <i>in vitro</i> kinase assay</h1>
<p><time datetime="2021-03-05">Published: March 5, 2021</time></p>
<p>Yuhong Han<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Jin Jiang<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Biology, University of Texas Southwestern Medical Center, Dallas, Texas 75390, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yuhong.han@utsouthwestern.edu">yuhong.han@utsouthwestern.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:jin.jiang@utsouthwestern.edu">jin.jiang@utsouthwestern.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100376">10.1016/j.xpro.2021.100376</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The Fused (Fu) kinase is a key transducer of Hedgehog signaling, but its relevant substrates have remained obscured due to the difficulty of obtaining active Fu for <i>in vitro</i> kinase assay. Based on the mechanism of Fu activation <i>in vivo</i>, we engineered a constitutively active Fu and expressed it in Sf9 cells using the baculovirus system. The kinase was affinity purified and applied for <i>in vitro</i> kinase assay using recombinant GST-fusion proteins as substrates to identify Fu-specific phosphorylation sites.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Han et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Purification of constitutively active Fu from insect cells for <i>in vitro</i> kinase assay</li>
<li>Priming phosphorylation by Fu can allow secondary <i>in vitro</i> kinase assay</li>
<li>High-purity protein elution with Flag M2 affinity agarose and 3X Flag peptide</li>
<li>Sensitive phospho-protein detection via pIMAGO-biotin kit or specific antibodies</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/495-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="note">
<span class="note-title">Note:</span> this protocol has been employed to demonstrate the direct phosphorylation of <i>Drosophila</i> Hedgehog pathway components, such as the transcription factor Cubitus interruptus (Ci) and the kinesin like protein Costal2 (Cos2) by Fu (<a href="#bib2">Han et al., 2019</a>). The identified phosphorylation sites fit the consensus sequence of S/T(X)<sub>5</sub>D/E, where X could be any amino acid (<a href="#bib2">Han et al., 2019</a>). The Fu phosphorylation consensus sites are also found in the Gli proteins that are mammalian homologs of Ci and are phosphorylated by the Fu-family kinase Ulk3 and STK36 (<a href="#bib2">Han et al., 2019</a>). This protocol can be applied to other protein kinases and their substrates with respect to construction of active kinases by forced-dimerization, one-step purification with Flag M2 agarose column, and <i>in vitro</i> kinase assay using pIMAGO-biotin phosphoprotein detection kit with the advantage of avoiding radioactive isotope.</div>
<h3 id="sec1.1">Engineering a constitutively active form of Fu for expression in Baculovirus</h3>
<ol>
<li>Fu kinase is activated by Hh-induced dimerization and trans-autophosphorylation (<a href="#bib5">Shi et al., 2011</a>; <a href="#bib6">Zhang et al., 2011</a>; <a href="#bib7">Zhou and Kalderon, 2011</a>). To construct a constitutively active full-length Fu kinase (amino acids (aa) 1–805), a coiled-coil (CC) dimerization motif from yeast GCN4 (aa 2–33) was fused to its N terminus and double phospho-mimetic mutations T151E/T154E were introduced to the activation loop in its kinase domain with original codon usage (<a href="#fig1">Figure 1</a>)(<a href="#bib4">O'Shea et al., 1991</a>; <a href="#bib5">Shi et al., 2011</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/495-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Engineering a constitutively active form of Fu</div>
<p>Schematic drawing of full-length wild-type Fu kinase (top) and its constitutively active form, CC-Fu<sup>EE</sup> (bottom). CC, coiled-coil dimerization motif from GCN4 (aa 2–33).</p>
</figcaption>
</figure>
</li>
<li>The resulting Fu (CC-Fu<sup>EE</sup>) was subcloned into the vector pFastBac1 for expression in baculovirus using EcoRI and Not I sites.</li>
<li>As a negative control, an inactive mutant of Fu (CC-Fu<sup>G13V</sup>) was constructed (<a href="#bib3">Liu et al., 2007</a>).</li>
</ol>
<h3 id="sec1.2">Detection of <i>in vitro</i> phosphorylated proteins by pIMAGO or phospho-specific antibodies</h3>
<ol start="4">
<li>To avoid using <sup>32</sup>P radioactive isotope in a traditional autoradiograph-based assay, the pIMAGO-biotin phosphoprotein detection kit for substrate phosphorylation recognition was employed (<a href="#bib1">Iliuk et al., 2014</a>).</li>
<li>The two Fu phosphorylation sites in Ci/Gli proteins are followed by Casein kinase 1 (CK1) phosphorylation sites (<a href="#bib2">Han et al., 2019</a>). To detect these dual phosphorylation events, two antibodies that recognized these phosphorylation clusters (S218/S220 and S1230/S1233) were generated (<a href="#bib2">Han et al., 2019</a>). An <i>in vitro</i> kinase assay using Fu and CK1 was developed and the phosphorylation was detected by these phospho-specific antibodies.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-Ci pS218/S220</td>
<td>Abmart</td>
<td>N/A</td>
</tr>
<tr>
<td>Anti-Ci pS1230/S1233</td>
<td>Abmart</td>
<td>N/A</td>
</tr>
<tr>
<td>IRDye 680LT donkey anti-rabbit secondary antibody</td>
<td>LI-COR</td>
<td>Cat#926-68023</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>BL21 chemically competent <i>E. coli</i>
</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EC0114</td>
</tr>
<tr>
<td>MAX efficiency DH10Bac chemically competent cells</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10361012</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Gentamycin</td>
<td>Millipore Sigma</td>
<td>Cat#G1397</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Millipore Sigma</td>
<td>Cat#A5354</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>Millipore Sigma</td>
<td>Cat#60615</td>
</tr>
<tr>
<td>Tetracycline</td>
<td>Millipore Sigma</td>
<td>Cat#T7660</td>
</tr>
<tr>
<td>IPTG</td>
<td>Millipore Sigma</td>
<td>Cat#I6758</td>
</tr>
<tr>
<td>Bluo-Gal</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15519028</td>
</tr>
<tr>
<td>L-Glutathione reduced</td>
<td>Millipore Sigma</td>
<td>Cat#G4251</td>
</tr>
<tr>
<td>Sf-900 III SFM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12658027</td>
</tr>
<tr>
<td>3X Flag peptide</td>
<td>Millipore Sigma</td>
<td>Cat#F4799</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>Millipore Sigma</td>
<td>Cat#F4135</td>
</tr>
<tr>
<td>Complete EDTA-free protease inhibitor cocktail</td>
<td>Roche</td>
<td>Cat#04 693 159 001</td>
</tr>
<tr>
<td>PhosSTOP phosphatase inhibitor cocktail</td>
<td>Roche</td>
<td>Cat#04 906 837 001</td>
</tr>
<tr>
<td>Casein kinase I</td>
<td>New ENGLAND BIOLABS</td>
<td>Cat#P6030</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>pIMAGO-biotin Phosphoprotein Detection Kit</td>
<td>Millipore Sigma</td>
<td>Cat#67221</td>
</tr>
<tr>
<td>Anti-Flag M2 affinity gel</td>
<td>Millipore Sigma</td>
<td>Cat#A2220</td>
</tr>
<tr>
<td>Glutathione Sepharose 4B</td>
<td>GE Healthcare</td>
<td>Cat#17075601</td>
</tr>
<tr>
<td>SimplyBlue SafeStain</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#LC6060</td>
</tr>
<tr>
<td>Cellfectin II Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10362100</td>
</tr>
<tr>
<td>Amicon Ultra-0.5 mL centrifugal filters</td>
<td>Millipore Sigma</td>
<td>Cat#UFC501024</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Insect: Sf9</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11496015</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pGEX 4T-1</td>
<td>GE Healthcare</td>
<td>Cat#28954549</td>
</tr>
<tr>
<td>pFastBac1</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10359016</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Baculovirus production and amplification</h3>
<p>Bluo-gal solution: prepared at the concentration of 20 mg/mL in dimethylformamide and store at −20°C with light protection.</p>
<p>LB Agar plates (DH10Bac): containing 50 μg/mL kanamycin, 7 μg/mL gentamycin, 10 μg/mL tetracycline and 40 μg/mL IPTG. 50–100 μL Bluo-gal (for 10 cm plate) was spread over the surface of the LB Agar plates and air dried in a ventilated hood for 5–10 min.</p>
<p>DH10Bac cell growth medium: LB medium containing 50 μg/mL kanamycin, 7 μg/mL gentamycin, 10 μg/mL tetracycline.</p>
<p>P1 buffer: 50 mM Tris-HCl pH 8.0, 10mM EDTA, 100 μg/mL RNase A (4°C, 6 months).</p>
<p>P2 buffer: 200 mM NaOH, 1% SDS (25°C, 6 months).</p>
<p>P3 buffer: 3 M potassium acetate pH 5.5 (25°C, 6 months).</p>
<p>TE buffer: 10 mM Tris-HCl pH 8.0, 1 mM EDTA (25°C, 12 months).</p>
<p>Sf9 cell growth medium: Sf-900 III SFM with 10% FBS and 50 μg/mL Gentamycin.</p>
<h3 id="sec2.2">Fu kinase expression and purification</h3>
<p>Lysis buffer: 20 mM Tris-HCl pH 8.0, 150 mM NaCl, 5 mM EDTA, 1% IGEPAL CA-630, 10% Glycerol, 1× Complete protease inhibitors cocktail, 1× phosSTOP phosphatase inhibitors cocktail (4°C, 1 month).</p>
<p>Storage buffer: 20 mM Tris-HCl pH 7.0, 100 mM NaCl, 1 mM DTT, 1 mM EDTA, 1 mM EGTA, 0.1% Triton X-100 (−20°C, 12 months).</p>
<h3 id="sec2.3">GST-fusion substrate expression and purification</h3>
<p>Extraction/binding buffer: 20 mM Tris-HCl pH 8.0, 100 mM NaCl, 5 mM EDTA (25°C, 6 months).</p>
<p>Elution buffer: 20 mM Tris-HCl pH 8.0, 100 mM NaCl, 5 mM EDTA, 10 mM GSH (4°C, 1 month).</p>
<p>Q700 Sonicator (Qsonica)</p>
<h3 id="sec2.4">
<i>In vitro</i> kinase assay</h3>
<p>Reaction buffer (1×): 15 mM Tris-HCl pH 7.5 (25°C), 0.2 mM Mg<sup>2+</sup>/ATP</p>
<p>(−20°C, 12 months)</p>
<h3 id="sec2.5">Detection of protein phosphorylation by pIMAGO-biotin phosphoprotein detection kit or standard immunoblot with phospho-specific antibodies</h3>
<p>Phospho-specific polyclonal antibodies were made by Abmart (Shanghai, China) using the following phospho-peptides as antigens: RKRALS(p)SS(p)PYSDS for pS218/220 (CiN), and QIIDSS(p)MTS(p)LPEL for pS1230/1233 (CiC), respectively. The corresponding non-phosphorylated peptides were synthesized to serve as a negative control for affinity purification. The crude serum samples from immunized rabbits were loaded onto columns with immobilized non-phosphorylated antigen peptides to absorb antibodies that recognize non-phosphorylated antigens. The flow-throughs were subsequently loaded onto columns with immobilized phosphorylated antigen peptides to enrich the pool of antibodies recognizing the phosphorylated antigens. The resultant antibody fractions were collected and tested for specificity using substrates with phosphorylated sites mutated as negative controls (<a href="#bib2">Han et al., 2019</a>).</p>
<p>Loading buffer (4×): 0.25 M Tris-HCl (pH 6.8), 8% SDS, 20% 2-mercaptoethanol, 40% glycerol, and 0.008% bromophenol blue (−20°C, 12 months) .</p>
<p>Running buffer: 25 mM Tris base, 190 mM glycine, 0.1% SDS (25°C, 3 months).</p>
<p>Transfer buffer: 25 mM Tris base, 190 mM glycine, 20% methanol (25°C, 3 months).</p>
<p>TBST solution: 20 mM Tris-HCl (pH 7.6), 150 mM NaCl, and 0.1% Tween 20 (25°C, 3 months).</p>
<p>Blocking buffer: 5% nonfat dried milk or bovine serum albumin (BSA) in TBST (4°C, 2 weeks).</p>
<p>Odyssey CLx Imaging System (LI-COR).</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">DH10Bac cell transformation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 52–54 h</div>
<ol>
<li>Thaw DH10Bac chemically competent cells on ice.</li>
<li>Add 1–10 ng of pFastBac1 vector containing the coding sequence of N-terminally Flag-tagged CC-Fu<sup>EE</sup> and CC-Fu<sup>GV</sup> (G13V mutant, a kinase dead form) into 100 μL DH10Bac chemically competent cells and incubate on ice for 30 min.</li>
<li>Heat shock the cells at 42°C for 45 s.</li>
<li>Cool down the cells on ice for 2 min.</li>
<li>Add 900 μL SOC medium and incubate the cells at 37°C for 4 h with shaking.</li>
<li>Dilute the mixture with SOC medium at 1:10 and 1:100 (namely, 1 μL mixture with 99 μL SOC medium and 10 μL mixture with 90 μL SOC medium), for each plasmid, spread 100 μL of undiluted mixture along with the above two dilutions (100 μL each) onto one LB Agar plate (DH10Bac).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Different amount of mixture were spread on plates to ensure that at least there would be one plate with appropriate number of colonies.</div>
<ol start="7">
<li>Grow the cells at 37°C protected from light for at least 48 h.</li>
</ol>
<h3 id="sec3.2">Bacmid preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 24 h</div>
<ol start="8">
<li>Pick up a white colony from the above plate and inoculate in 1.5 mL DH10Bac cell growth medium at 37°C for 16 h.</li>
<li>The next day, collect the cells by centrifugation and resuspend in 300 μL P1 buffer.</li>
<li>Mix with 300 μL P2 buffer gently and incubate at 25°C for 2–3 min.</li>
<li>Neutralize pH by adding 300 μL of P3 buffer and gently invert the tubes 4–6 times.</li>
<li>Sediment the precipitate by centrifugation at 14,000 × <i>g</i> for 10 min at 4°C.</li>
<li>Carefully withdraw around 800 μL of supernatant (avoid disturbing the white pellet) and mix gently with 560 μL (0.7 V) of isopropanol. Let the mixture sit on ice for 10–20 min or at −18°C for 12 h.</li>
<li>Collect the Bacmid DNA by centrifugation at 14,000 × <i>g</i> for 10–15 min and wash with 500 μL of 70% ethanol.</li>
<li>Desiccate the Bacmid DNA at 25°C in open air for 10 min (can be done alternatively at a 37°C incubator).</li>
<li>Dissolve the Bacmid DNA in 40 μL TE buffer.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Bacmid DNA could be stored at −20°C for at least 6 months.</div>
<h3 id="sec3.3">Baculovirus production and amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 weeks</div>
<ol start="17">
<li>Determine the DNA concentration of recombinant Bacmid.</li>
<li>Seed 1 million of Sf9 cells into one well of a 6-well plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> At least 90% of cells should settle down within 5–10 min.</div>
<ol start="19">
<li>Allow cells grow at 27°C incubator for at least 1–2 h.</li>
<li>Replace the growth medium by 2 mL SF900-III medium with 1.5% FBS.</li>
<li>For each transfection, dilute 8 μL Cellfectin II reagent (pre-warm to 25°C and gently vortex 1–2 S) in 100 μL SF900-III medium without serum or antibiotics.</li>
<li>For each transfection, dilute 1–3 μg Bacmid DNA (usually 2–3 μL/40 μL Bacmid DNA solution) in 100 μL SF900-III medium without serum or antibiotics.</li>
<li>Combine diluted DNA and Cellfectin II and incubate at 25°C for 15 min.</li>
<li>Add the above mixture to Sf9 cells dropwise.</li>
<li>Let the cells sit at 27°C for 3 h.</li>
<li>Change medium to regular Sf9 growth medium.</li>
<li>Incubate cells at 27°C for 72 h before collecting medium containing P1 virus. Store the P1 virus at 4°C with protection from light.</li>
<li>Seed 1–2 million Sf9 cells into one well of 6-well plate and let the cells grow 1–2 h at 27°C.</li>
<li>Add 1–2 μL P1 virus into each well and let the cells grow for 2–3 days. Collect the medium containing P2 virus and verify the expression of intended protein in the cells with standard western blot procedure.</li>
<li>For large-scale virus production (P3 virus), seed 50–100 mL of Sf9 cells at the density of 1 million cells per mL in 200 mL flask. The next day, add 1/2000 volume of P2 virus and incubate 3 days at 27°C.</li>
<li>Collect P3 virus and store at 4°C for future infection.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Baculovirus (P1, P2 and P3) could be stored at 4°C with light protection at least 3–6 months.</div>
<h3 id="sec3.4">Fu kinase expression and purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–7 days</div>
<ol start="32">
<li>Seed 600 mL Sf9 cells into 2L flask at the density of 1 million cells per mL and infect the cells with 6 mL Flag-CC-Fu<sup>EE</sup> or CC-Fu<sup>GV</sup> P3 virus on the next day.</li>
<li>48 h after infection, collect the cells by centrifugation at 5,000 × <i>g</i> for 20 min.</li>
<li>Resuspend the cell pellet with 40 mL lysis buffer and incubate at 4°C for 10 min with agitation.</li>
<li>Remove the cell debris by centrifugation at 12,000 × <i>g</i> for 20 min.</li>
<li>Incubate the resultant cell lysate with 100 μL (bed volume) Flag M2 affinity agarose at 4°C for 2 h with shaking.</li>
<li>Wash the beads with 3 changes of fresh cold lysis buffer at 4°C for 5 min each.</li>
<li>Elute the bound protein by incubation with 2× Kinase storage buffer containing 200 μg/mL 3X Flag peptide at 4°C for 2 h with shaking.</li>
<li>Load a small aliquot of eluted protein onto SDS-PAGE gel and stain the gel with Coomassie blue to determine the purity and yield of Fu kinase.</li>
<li>Concentrate the kinase to 0.2 μg/μL in 2× kinase storage buffer using Amicon Ultra centrifugal filter with 10K cut-off simultaneously reducing the concentration of Flag peptide in the sample to minimal level.</li>
<li>Dilute the kinase with equal volume of glycerol; aliquot and flash-freeze with liquid nitrogen and store at −80°C.</li>
</ol>
<h3 id="sec3.5">GST-fusion substrate expression and purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<ol start="42">
<li>Transform BL21 chemically competent cells with pGEX 4T-1 and pGEX 4T-1 vectors with expression constructs for Ci/Cos2 polypeptides (CiN: aa 213–234; CiC: aa 1,222–1,240; Cos2: aa 564–585).</li>
<li>Inoculate single colony into 3 mL LB medium with ampicillin and incubate at 37°C on a shaker at 250 rpm for 16 h.</li>
<li>Dilute with the fresh LB medium at the ratio 1:200 (total 600 mL) and continue the incubation until OD<sub>600</sub> reaches around 0.6.</li>
<li>After cooling down on ice, induce the GST/GST-fusion protein expression with 0.1 M IPTG and grow the cells at 30°C for 3–4 h.</li>
<li>Collect the cells by centrifugation at 5,000 × <i>g</i> for 15 min.</li>
<li>Resuspend the bacteria in 40 mL lysis/binding buffer and break the cells with a sonicator on ice in short burst (15 s on, 15 s off, 20 min in total, 30%–40% output).</li>
<li>Centrifuge at 12,000 × <i>g</i> for 15 min at 4°C and collect the supernatant.</li>
<li>Incubate the supernatant with 0.5 mL glutathione Sepharose at 4°C for 16 h with shaking.</li>
<li>Wash the beads thrice with 1 mL of fresh lysis/binding buffer for 5 min at 4°C with agitation.</li>
<li>Elute GST/GST-fusion proteins by addition of 1 mL of elution buffer and incubate 1–2 h at 4°C. Collect the eluted proteins by 2,000 × <i>g</i> centrifugation at 4°C for 2 min.</li>
<li>Determine the protein yield and purity by on a Coomassie Blue stained SDS-PAGE gel against BSA standard applied in the amount of 0.1 μg, 0.2 μg, 0.4 μg, 0.6 μg, 0.8 μg, and 1 μg per lane. Assess the protein amount by scanning of the dry gel using Odyssey system at 700 nm wavelength.</li>
<li>Concentrate the protein by ultracentrifugation to the final concentration to 0.5–1 mg/mL with Amicon Ultra-0.5 mL centrifugal filters with 10K cut-off with simultaneous buffer exchange to 30 mM Tris-HCl pH 7.5 100 mM NaCl. Dilute concentrated proteins with equal volume of glycerol, flash-freeze with liquid nitrogen, and store at −80°C.</li>
</ol>
<h3 id="sec3.6">
<i>In vitro</i> kinase assay with Fu kinase alone</h3>
<ol start="54">
<li>For each experimental point, set up a 25 μL reaction mixture with 1× <i>in vitro</i> kinase buffer, using 0.1 ug of catalytically active/inactive protein of Fu kinase purified from Sf9 cells. and 1 μg of GST Ci/Cos2 fragment fusion protein as substrate.</li>
<li>Incubate the reaction mixture at 30°C for 15–30 min.</li>
<li>Stop the reaction by adding equal volume of 2× SDS loading buffer and boil for 5 min.</li>
</ol>
<h3 id="sec3.7">
<i>In vitro</i> kinase assay with sequential phosphorylation by Fu and CK1</h3>
<ol start="57">
<li>After the first step of <i>in vitro</i> kinase assay with Fu, instead of termination of the reaction by addition of 2× SDS loading buffer, add 20 μL of glutathione beads (previously incubated with 2 mg/mL BSA at 4°C for 1 h with agitation to minimize the non-specific binding) and incubate at 4°C with shaking for at least 1 h to pull down the GST-fusion substrates.</li>
<li>Wash the beads with thrice 1× kinase buffer.</li>
<li>Use washed beads as the substrate in final kinase assay with CK1. All other steps are similar to that described for Fu kinase assay except for using CK1 as a kinase.</li>
<li>Stop the reaction by adding equal volume of 2× SDS loading buffer and boil for 5 min.</li>
</ol>
<h3 id="sec3.8">Detection of protein phosphorylation by pIMAGO-biotin phosphoprotein detection kit</h3>
<ol start="61">
<li>Add 5× IAA solution (10 μL) to the reaction mixture (40 μL) and incubated at dark for 15 min.</li>
<li>Load a small aliquot of reaction mixture (5–10 μL) onto an SDS-PAGE gel and transfer proteins to a nitrocellulose membrane.</li>
<li>Block the membrane with 10 mL 1× blocking buffer at 25°C for 1 h with constant shaking.</li>
<li>Incubate the membrane with 1× pIMAGO buffer supplemented with 1:1,000 pIMAGO reagent at 25°C for 1 h.</li>
<li>Wash the membrane with 1× wash buffer three times for 5 min each.</li>
<li>Wash the membrane with TBST for 5 min.</li>
<li>Incubate the membrane with 1× blocking buffer supplemented with 1:1,000 Avidin-Fluor at 25°C for 1 h.</li>
<li>Wash the membrane with TBST three times for 5 min each.</li>
<li>Scan the membrane with an Odyssey CLx imaging system.</li>
</ol>
<h3 id="sec3.9">Immunoblot with Ci phospho-specific antibodies</h3>
<ol start="70">
<li>Load 1–5 μL of the reaction mixture from an <i>in vitro</i> sequential kinase assay onto an SDS-PAGE and transfer proteins to a nitrocellulose membrane using standard Western blot procedure.</li>
<li>Block the membrane with 5% nonfat milk for 1 h at 25°C.</li>
<li>Incubate the membrane with Ci phospho-specific antibodies (primary, for CiN phospho antibody pS218/S220, 1:1,000; CiC phospho antibody pS1230/S1233, 1:4,000) at 25°C for 2 h with agitation followed by wash with TBST solution for 3 times with agitation (5 min each time) , and incubation with fluorogenic donkey anti-rabbit antibody (secondary, 1:10,000) for 1 h at 25°C with shaking.</li>
<li>Scan the membrane by an Odyssey CLx imaging system at 700 nm wavelength.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Sf9/Baculovirus system was applied to expressing recombinant Fu kinase. Significant amount of Fu protein with high purity could be acquired using Flag M2 affinity agarose/3X Flag peptide purification system as demonstrated in <a href="#fig2">Figure 2</a>.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/495-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Purification of Fu kinase</div>
<p>Coomassie blue staining of Flag-CC-Fu<sup>EE</sup> purified from Sf9 cells.</p>
</figcaption>
</figure>
<p>The constitutively active form of Fu kinase, CC-Fu<sup>EE</sup>, exhibits robust activity toward the GST-fusion protein substrates in <i>in vitro</i> kinase assay. Utilization of pIMAGO-biotin Kit enables convenient and specific detection of phosphorylated substrates as shown in <a href="#fig3">Figure 3</a>, while prevents usage of traditional <sup>32</sup>P radioactive isotope.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/495-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. <i>In vitro</i> kinase assay with purified Fu and detection with pIMAGO</div>
<p><i>In vitro</i> kinase assay using purified CC-Fu<sup>EE</sup> or CC-Fu<sup>GV</sup> as source of kinase and GST-fusion proteins containing either CiN (A) or CiC (B) fragment with the wild-type sequence or the indicated amino acid substitutions. Phosphorylation was detected by the pIMAGO kit. GST-fusion proteins were visualized by Coomassie blue gel staining (CB). The Fu phosphorylation sites are colored in red and the critical acid residues at +6 position are colored in blue. Figure reprinted with permission from <a href="#bib2">Han et al. (2019)</a>.</p>
</figcaption>
</figure>
<p>Both Fu phosphorylation sites in Ci protein have a secondary CK1 site in proximity. To verify the involvement of CK1 in phosphorylation of these two clusters after priming phosphorylation by Fu, we developed two phospho-antibodies against them. The immunoblot data shown in <a href="#fig4">Figure 4</a> provide strong evidence that sequential phosphorylation by Fu and CK1 occurs at these two clusters.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/495-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. <i>In vitro</i> kinase assay with Fu and CK1 followed by detection with phospho-specific antibodies</div>
<p><i>In vitro</i> kinase assay using purified CC-Fu<sup>EE</sup> and/or recombinant CK1 as kinases and GST-fusion substrates containing either CiN (A) or CiC (B) fragment with the indicated amino acid sequence. Phosphorylation was detected by western blot using the indicated phospho-specific antibodies. GST-fusion proteins were visualized by Coomassie blue gel staining (CB). The Fu and CK1 phosphorylation sites are colored in red and green, respectively. Figure reprinted with permission from <a href="#bib2">Han et al. (2019)</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The <i>in vitro</i> kinase assay carried out using current protocol could only demonstrate that Fu kinase alone or combined with CK1 are able to phosphorylate certain sites within the substrates under optimized <i>in vitro</i> conditions. However, other experiments such as cell-based phosphorylation assay and <i>in vivo</i> functional study are needed to demonstrate the physiological significance of these phosphorylation events by the Fu kinase.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>It is difficult to discern white colonies of DH10Bac cells after transformation (step 8).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>DH10Bac cells grow quite slowly on these LB Agar plates containing multiple antibiotics, it is much easier for pick up white colonies after 48 h incubation at 37°C. Consider picking up big white colonies surrounded by multiple blue colonies.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Bacmid transfection is not successful (step 25).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Firstly, do not over-transfect Sf9 cells. Usually 1–2 μg of Bacmid is enough and too much of that will kill Sf9 cells. Secondly, make sure only healthy Sf9 cells are seeded for transfection. Usually, Sf9 cells in suspension culture are seeded onto 6-well plates before transfection and at least 90% cells should attach to the bottom within 5–10 min if Sf9 are healthy enough for further manipulations.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>During the Bacmid transfection, a lot of Sf9 cells died (step 27).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Please be aware that Sf9 is extremely sensitive to desiccation. Without any medium, they will die as short as 40–50 s.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The protein expression level of baculovirus is low (step 29).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Make sure that only healthy Sf9 cells are taken for infection. Keep your baculovirus at 4°C and protect it from light. It is also critical to always amplify baculovirus using P1 or P2 virus, not P3 virus, and keep the MOI as low as 0.01–0.1.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>The yield and purity of the GST-fusion protein are poor (step 52).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Try to induce the expression with low concentrations of IPTG (such as 0.1 mM) and decrease the growth temperature of culture to 18°C–20°C with longer incubation (could be extended to 12 h). For GST-fusion proteins with poor yield, use high volume of cell lysate versus low amount of GST beads that could improve both yield and purity.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>The sonication of bacterial cells is not successful (step 47).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Increase the volume of lysis buffer used in the sonication.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>The phospho-specific antibodies recognize non-phosphorylated substrates (step 73).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>The two-step purification scheme described in <a href="#materials-and-equipment">Materials and equipment</a> section should largely remove antibodies that recognize non-phosphorylated substrates and enrich the content of antibodies that recognize phosphorylated substrates so that the antibodies produce good signal to noise ratio. One approach to further reduce the noise is to dilute the working concentration of the antibodies. If this does not work, try one more round of purification that may reduce non-specific signals.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Iliuk, A., Li, L., Pan, L., and Tao, W.A. (2014). Development of pIMAGO for universal detection of protein phosphorylation. [abstract]. In (AACR; Cancer Res 2014). <a class="external-link" href="https://doi.org/10.1158/1538-7445.AM2014-5307">https://doi.org/10.1158/1538-7445.AM2014-5307</a></p>
<p id="bib2">Han, Y., Wang, B., Cho, Y.S., Zhu, J., Wu, J., Chen, Y., and Jiang, J. (2019). Phosphorylation of Ci/Gli by fused family kinases promotes Hedgehog signaling. Dev. Cell <i>50</i>, 610-626.e614. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00083-6/sref2">View at publisher</a></p>
<p id="bib3">Liu, Y., Cao, X., Jiang, J., and Jia, J. (2007). Fused-Costal2 protein complex regulates Hedgehog-induced Smo phosphorylation and cell-surface accumulation. Genes Dev. <i>21</i>, 1949-1963. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00083-6/sref3">View at publisher</a></p>
<p id="bib4">O'Shea, E.K., Klemm, J.D., Kim, P.S., and Alber, T. (1991). X-ray structure of the GCN4 leucine zipper, a two-stranded, parallel coiled coil. Science <i>254</i>, 539-544. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00083-6/sref4">View at publisher</a></p>
<p id="bib5">Shi, Q., Li, S., Jia, J., and Jiang, J. (2011). The Hedgehog-induced Smoothened conformational switch assembles a signaling complex that activates Fused by promoting its dimerization and phosphorylation. Development <i>138</i>, 4219-4231. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00083-6/sref5">View at publisher</a></p>
<p id="bib6">Zhang, Y., Mao, F., Lu, Y., Wu, W., Zhang, L., and Zhao, Y. (2011). Transduction of the Hedgehog signal through the dimerization of Fused and the nuclear translocation of Cubitus interruptus. Cell Res. <i>21</i>, 1436-1451. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00083-6/sref6">View at publisher</a></p>
<p id="bib7">Zhou, Q. and Kalderon, D. (2011). Hedgehog activates Fused through phosphorylation to elicit a full spectrum of pathway responses. Dev. Cell <i>20</i>, 802-814. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00083-6/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jin Jiang, E-mail: <a href="mailto:Jin.Jiang@UTSouthwestern.edu">jin.jiang@utsouthwestern.edu</a>.</p>
<h4>Materials Availability</h4>
<p>All unique/stable reagents generated in this study are available from the lead contact with a completed Materials Transfer Agreement.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate/analyze datasets and codes.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants from the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (R35GM118063) and <a href="https://doi.org/10.13039/100000928">Welch Foundation</a> (I-1603).</p>
<h3>Author Contributions</h3>
<p>Conceptualization, Y.H. and J.J.; Investigation, Y.H.; Writing – Original Draft, Y.H.; Writing – Review &amp; Editing, J.J.; Funding Acquisition, J.J.; Supervision, J.J.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>