<article>
<section>
<h1 id="header">Agonist and antagonist TRUPATH assays for G protein-coupled receptors</h1>
<p><time datetime="2022-04-01">Published: April 1, 2022</time></p>
<p>Jeffrey F. DiBerto,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Katie Smart,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Reid H.J. Olsen,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> and Bryan L. Roth<sup><a href="#aff1">1</a>,<a href="#fn3">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pharmacology, University of North Carolina at Chapel Hill School of Medicine, Chapel Hill, NC 27599-7365, USA</p>
<p id="aff2"><sup>2</sup>GPCR Pharmacology, Department of Discovery Biology, Exscientia, Oxford OX44GE, UK</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:rolsen@exscientia.co.uk">rolsen@exscientia.co.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:bryan_roth@med.unc.edu">bryan_roth@med.unc.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101259">10.1016/j.xpro.2022.101259</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>TRUPATH is a bioluminescence resonance energy transfer-based platform for quantifying G protein-coupled receptor activity via dissociation of heterotrimeric G protein biosensors. Here, we present protocols for agonist and antagonist TRUPATH assays in the 384-well plate format, thereby providing an opportunity for higher throughput. We also provide both data analysis and quality control analyses for these assays, along with considerations for assay optimization and solutions for troubleshooting needs that may be encountered.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref14">Olsen et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocols for TRUPATH using the 384-well plate format for higher throughput</li>
<li>Agonist and antagonist TRUPATH assays and data analyses</li>
<li>TRUPATH optimization and troubleshooting</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol>
<li>Prepare the buffers listed in the Buffers/Reagents table in advance to ensure that sufficient volumes are available to complete experiments.</li>
</ol>
<h3 id="sec1.1">Background</h3>
<p>TRUPATH is a bioluminescence resonance energy transfer 2 (BRET2)-based G protein-coupled receptor (GPCR) platform that measures the dissociation of heterotrimeric G proteins, the most proximal signaling event downstream of receptor activation (<a href="#bib14">Olsen et al., 2020</a>). It utilizes G<math> <mrow> <mi>α</mi> </mrow> </math> subunits fused to <i>Renilla</i> luciferase (RLuc8) at optimal insertion sites and G<math> <mrow> <mi>γ</mi> </mrow> </math> subunits fused to the green fluorescent protein (GFP) variant GFP2 (this variant, which results from a F64L mutation, is brighter—and therefore more detectable—than wildtype GFP) at their N-termini to produce a BRET signal—a proxy for the associated (i.e., inactive) heterotrimer—that decreases in a concentration-dependent manner upon receptor activation, productive G protein coupling, and consequent subunit dissociation (<a href="#fig1">Figure 1</a>). Compositions of G<math> <mrow> <mi>α</mi> </mrow> </math>-RLuc8, G<math> <mrow> <mi>β</mi> </mrow> </math>, and GFP2-G<math> <mrow> <mi>γ</mi> </mrow> </math> heterotrimers that produce the greatest dynamic range have been empirically determined, and plasmids for these constructs are available from Addgene. Additionally, the “non-optimal” G<math> <mrow> <mi>β</mi> </mrow> </math> and GFP2-G<math> <mrow> <mi>γ</mi> </mrow> </math> plasmids are available for TRUPATH users investigating specific heterotrimeric G protein combinations. To date, numerous groups have successfully employed TRUPATH in their studies to interrogate GPCR structure and function (<a href="#bib13">Nagai et al., 2020</a>; <a href="#bib8">Kim et al., 2020</a>; <a href="#bib9">Knight et al., 2021</a>; <a href="#bib5">Du et al., 2021</a>; <a href="#bib15">Pryce et al., 2021</a>; <a href="#bib6">Gao et al., 2021</a>; <a href="#bib1">Chakraborty et al., 2021a</a>, <a href="#bib2">2021b</a>; <a href="#bib3">Chao et al., 2021</a>; <a href="#bib16">Von Moo et al., 2021</a>; <a href="#bib11">Lu et al., 2021</a>; <a href="#bib10">Lin et al., 2021</a>; <a href="#bib4">Cao et al., 2021</a>; <a href="#bib18">Yang et al., 2021</a>; <a href="#bib17">Yan et al., 2021</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. An overview of the TRUPATH platform</div>
<p>G<math> <mrow> <mi>α</mi> </mrow> </math> subunits fused to RLuc8 and G<math> <mrow> <mi>γ</mi> </mrow> </math> subunits fused to GFP2 produce a BRET signal (calculated as the ratio of GFP2 to RLuc8 counts) in the presence of RLuc8 substrate that is both spatially- and orientationally-dependent. Under basal conditions, when the G protein heterotrimer is associated, this BRET signal is high. Agonist-induced receptor activation catalyzes the exchange of guanosine diphosphate for guanosine triphosphate, resulting in dissociation of the G<i>α</i> subunit from the G<i>βγ</i> heterodimer, and a decrease in the BRET signal that is sensitive to changes in agonist concentration.</p>
</figcaption>
</figure>
<p>TRUPATH relies on recombinant expression of both the receptor and G protein biosensor proteins of interest in live cells. A TRUPATH user should be familiar with the basics of tissue culture and transfection. We have found that one of the most critical aspects for the successful application of TRUPATH as an assay system is the efficiency of transfection to achieve sufficient expression levels of both the receptor and biosensor proteins. Additionally, receptor activation by cell culture media components (e.g., serotonin) or constitutive activity can affect the assay’s sensitivity, and users may benefit from characterizing the expression level of their receptor of interest under alternative culturing and transfection conditions. The protocol presented here describes general steps for a receptor that is already known to be efficiently expressed under the conditions presented herein. We describe the use of TransIT-2020 as a transfection reagent, but users may choose to use their reagent of choice. We have performed this assay with Lipofectamine 3000 as well as polyethylenimine (PEI); however, we find TransIT-2020 to be the most efficient and simple to use. TRUPATH plasmids use the pcDNA5 (G<math> <mrow> <mi>α</mi> </mrow> </math> subunits) and pcDNA3.1 (G<math> <mrow> <mi>β</mi> </mrow> </math> and G<math> <mrow> <mi>γ</mi> </mrow> </math> subunits) backbones to drive transcription of the open reading frame under control of the cytomegalovirus (CMV) promoter. The dopamine receptor D2 (DRD2) expression plasmid utilized here uses the pcDNA3.1 backbone, also under control of the CMV promoter. All tissue culture should be performed in a sterile tissue culture hood with appropriate personal protective equipment.</p>
<div class="note">
<span class="note-title">Note:</span> This protocol describes culture and transfection conditions for HEK293T cells. However, we have also applied this protocol (with minor adaptations to culturing and transfection conditions) to use in Chinese hamster ovary and Neuro2A cells. As a pharmacological assay, TRUPATH provides robust measures of efficacy and potency across these cell lines with marginal differences, if any. Therefore, we encourage the use of HEK293T cells because of their transfectability.</div>
<div class="note">
<span class="note-title">Note:</span> The compositions of cell culture media used here vary across the timeline for the protocol; the constituents of each formulation are described in the <a href="#key-resources-table">key resources table</a> and Buffers/Reagents tables. For clarification, “complete DMEM” refers to Dulbecco’s modified eagle media (DMEM) + 10% heat-inactivated fetal bovine serum (FBS) + 2 mM L-Glutamine + 1% penicillin-streptomycin (pen-strep), while “1% dFBS DMEM” refers to DMEM + 1% dialyzed FBS + 2 mM L-Glutamine + 1% pen-strep.</div>
<div class="note">
<span class="note-title">Note:</span> This protocol also describes the use of a PHERAstar FSX from BMG LABTECH; though we have also performed this assay using a Mithras LB940 Multimodal Plate Reader from Berthold Technologies. Different plate readers can be used provided that the optics are both sensitive enough to detect and resolve the luciferase and GFP2 signals.</div>
<div class="note">
<span class="note-title">Note:</span> Compound preparation for the assay can take a variety of formats depending on user requirements. Here we describe a protocol using drug plates prepared via an Echo 650T Liquid Handler from Beckman Coulter. Finally, we present the assay for use in 384-well plates. Depending on the receptor target and assay requirements, the assay may perform better in a 96-well format.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Different cell lines may require alternative cell media components that should be adapted by the user to meet their specifications.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Bovine Serum Albumin Fraction V, fatty acid free (BSA)</td>
<td>MilliporeSigma</td>
<td>Cat# 10775835001</td>
</tr>
<tr>
<td>Dopamine hydrochloride</td>
<td>Tocris Bioscience</td>
<td>Cat# 3548</td>
</tr>
<tr>
<td>Droperidol</td>
<td>Tocris Bioscience</td>
<td>Cat# 5858</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle Medium (DMEM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11995065</td>
</tr>
<tr>
<td>Ethylenediaminetetraacetic acid (EDTA)</td>
<td>MilliporeSigma</td>
<td>Cat# E5134</td>
</tr>
<tr>
<td>Fetal Bovine Serum, heat inactivated (FBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10082147</td>
</tr>
<tr>
<td>Fetal Bovine Serum, dialyzed (dFBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 26400044</td>
</tr>
<tr>
<td>Hanks’ Balanced Salt Solution (HBSS) (10×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 14065056</td>
</tr>
<tr>
<td>HEPES</td>
<td>MilliporeSigma</td>
<td>Cat# H3375</td>
</tr>
<tr>
<td>L-Ascorbic acid</td>
<td>MilliporeSigma</td>
<td>Cat# 795437</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 25030024</td>
</tr>
<tr>
<td>L741,626</td>
<td>Tocris Bioscience</td>
<td>Cat# 1003</td>
</tr>
<tr>
<td>Methanol</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10141720</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 31985062</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15140122</td>
</tr>
<tr>
<td>Premixed phosphate buffered saline (PBS) (10×)</td>
<td>MilliporeSigma</td>
<td>Cat# P4417</td>
</tr>
<tr>
<td>Poly-L-lysine hydrobromide</td>
<td>MilliporeSigma</td>
<td>Cat# P2636</td>
</tr>
<tr>
<td>Potassium hydroxide (KOH)</td>
<td>MilliporeSigma</td>
<td>Cat# 221473</td>
</tr>
<tr>
<td>Prolume Purple</td>
<td>NanoLight Technology</td>
<td>Cat# 369</td>
</tr>
<tr>
<td>Risperidone</td>
<td>Tocris Bioscience</td>
<td>Cat# 2856</td>
</tr>
<tr>
<td>TransIT-2020</td>
<td>Mirus</td>
<td>Cat# MIR 5400</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.05%)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 25300062</td>
</tr>
<tr>
<td>Ziprasidone hydrlochloride</td>
<td>Tocris Bioscience</td>
<td>Cat# 3085</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Human embryonic kidney 293T cell line</td>
<td>ATCC</td>
<td>Cat# CRL-3216</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td><a href="#bib14">Olsen et al. (2020)</a></td>
<td>Addgene</td>
<td>Cat# 1000000163</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Prism</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/">https://www.graphpad.com/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>12.5 <math> <mrow> <mi>μ</mi> </mrow> </math>LGripTips Pipette Tips for VIAFLO 96/384</td>
<td>Integra</td>
<td>Cat# 6453</td>
</tr>
<tr>
<td>125 <math> <mrow> <mi>μ</mi> </mrow> </math>LGripTips Pipette Tips for VIAFLO 96/384</td>
<td>Integra</td>
<td>Cat# 6463</td>
</tr>
<tr>
<td>384 Channel Pipetting Heads (5–125 μL)</td>
<td>Integra</td>
<td>Cat# 6132</td>
</tr>
<tr>
<td>BackSeal-96/384, White Adhesive Bottom Seal</td>
<td>PerkinElmer</td>
<td>Cat# 6005199</td>
</tr>
<tr>
<td>Cell Culture Microplate, 384 wells</td>
<td>Greiner Bio-One</td>
<td>Cat# 781098</td>
</tr>
<tr>
<td>Centrifuge 5910 Ri – Refrigerated Centrifuge</td>
<td>Eppendorf</td>
<td>Cat# 5943000131</td>
</tr>
<tr>
<td>Echo 650T Liquid Handler</td>
<td>Beckman Coulter Inc.</td>
<td>Cat# 001-16078</td>
</tr>
<tr>
<td>Nunc EasYDish Dishes, 100 mm diameter</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 150464</td>
</tr>
<tr>
<td>Nunc EasYDish Dishes, 150 mm diameter</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 150468</td>
</tr>
<tr>
<td>PHERAstar <i>FSX</i> with BRET2 optic module</td>
<td>BMG LABTECH</td>
<td><a href="http://www.bmglabtech.com">www.bmglabtech.com</a></td>
</tr>
<tr>
<td>VIAFLO 384, 24, 96, and 384 channel handheld pipette</td>
<td>Integra</td>
<td>Cat# 6031</td>
</tr>
<tr>
<td>384 Channel Pipetting Heads (0.5–12.5 μL)</td>
<td>Integra</td>
<td>Cat# 6131</td>
</tr>
</tbody>
</table>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol describes the use of the BMG LABTECH PHERAstar FSX plate reader. Alternate plate readers can be used assuming properly filtered and sufficiently sensitive optics.</div>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Alternate assay and drug buffers can be used depending on user requirements and validation.</div>
<table id="undtbl1">
<thead>
<tr>
<th>Assay buffer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HBSS (10×)</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>HEPES (1 M)</td>
<td>20 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>KOH (1 M)</td>
<td>n/a</td>
<td>To pH 7.4</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>To volume</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterile filter and store at 22°C for up to 1 month</p>
<table id="undtbl2">
<thead>
<tr>
<th>Complete DMEM</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>
<sup>a</sup>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>
<sup>a</sup>100 U/mL-100 <math> <mrow> <mi>μ</mi> </mrow> </math>g/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>5</b><b>60 mL</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend"><sup>a</sup>Approximately; Store at 4°C for up to 1 month.</p>
<table id="undtbl3">
<thead>
<tr>
<th>1% Dialyzed FBS DMEM</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>dFBS</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>100 U/mL-100 <math> <mrow> <mi>μ</mi> </mrow> </math>g/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>51</b><b>5 mL</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend"><sup>a</sup>Approximately; Store at 4°C for up to 1 month.</p>
<table id="undtbl4">
<thead>
<tr>
<th>Drug buffer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ascorbic acid</td>
<td>0.3 mg/mL</td>
<td>15 mg</td>
</tr>
<tr>
<td>BSA</td>
<td>3 mg/mL</td>
<td>150 mg</td>
</tr>
<tr>
<td>KOH (1 M)</td>
<td>n/a</td>
<td>To pH 7.4</td>
</tr>
<tr>
<td>Assay buffer</td>
<td>n/a</td>
<td>To volume</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterile filter and store at 22°C for up to 24 hours</p>
<table id="undtbl5">
<thead>
<tr>
<th>Poly-L-lysine solution</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Poly-L-lysine (50 mg/mL)</td>
<td>100 <math> <mrow> <mi>μ</mi> </mrow> </math>g/mL</td>
<td>2 mL</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>100 U/mL-100 <math> <mrow> <mi>μ</mi> </mrow> </math>g/mL</td>
<td>10 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>n/a</td>
<td>988 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterile filter and store at 4°C for up to 3 months</p>
<table id="undtbl6">
<thead>
<tr>
<th>Versene solution</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.5 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>PBS tablets</td>
<td>1×</td>
<td>5 tablets</td>
</tr>
<tr>
<td>Distilled water</td>
<td>n/a</td>
<td>999 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterile filter and store at 4°C for up to 3 months.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Initial cell culture and transfection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h across 1 day</div>
<p>This section details how to approach cell culture maintenance and transfection of the TRUPATH plasmids.</p>
<ol>
<li>Remove a 10 or 15 cm dish of HEK293T cells from the cell culture incubator (5% CO<sub>2</sub>, 37°C, 85%–95% humidity) (<a href="#fig2">Figure 2</a>A). HEK293T cells should be of low passage number, in log-phase, of high viability (e.g., &gt;90%), and non-confluent at time of harvesting.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Initial cell culture and transfection</div>
<p>(A) Cells are removed from the tissue culture incubator are dislodged from the tissue culture dish with Trypsin-EDTA and manual disruption following aspiration of the cell media.</p>
<p>(B and C) (B) Cells are then gently centrifuged, media aspirated, (C) resuspended, and seeded into a new 10-cm tissue culture dish.</p>
<p>(D and F) (D) DNA are diluted in OptiMEM (pink liquid) and incubated with TransIT-2020 (gray liquid) for 20 min before (E) DNA complexes are added drop-wise to the HEK293T cells from earlier and subsequently (F) returned to the tissue-culture incubator for incubation.</p>
</figcaption>
</figure>
</li>
<li>Using Trypsin-EDTA, harvest HEK293T cells for plating into 10 cm tissue culture dish.
<ol type="a">
<li>Aspirate tissue culture media from initial dish and apply enough Trypsin-EDTA to cover the cells. Let sit for 3–5 min, gently agitating or tipping the dish from side-to-side (<a href="#fig2">Figure 2</a>A).</li>
<li>Gently triturate cells by repeated aspiration and transfer dissociated cells to a 50 mL conical tube containing a greater or equal volume of complete DMEM (<a href="#fig2">Figure 2</a>B).</li>
<li>Using a 22°C table-top centrifuge, pellet by centrifuging cells for 5 min at 250 × <i>g</i> (<a href="#fig2">Figures 2</a>B and <a href="#fig2">2</a>C).</li>
<li>Aspirate the supernatant leaving the gently pelleted cells at the bottom of the conical tube.</li>
<li>Resuspend the cell pellet in 1 mL of complete DMEM by gentle trituration with a 1 mL pipette.</li>
<li>Add 9 mL of complete DMEM and mix by gentle inversion before counting and assessing cell viability.</li>
<li>Gently and evenly distribute cells to a fresh 10 cm tissue culture dish containing 10–15 mL complete DMEM (final volume) at a density of 6.5–7 million cells per dish (<a href="#fig2">Figure 2</a>C).
<ol type="i">
<li>Return dish to the incubator.</li>
</ol>
</li>
</ol>
</li>
<li>5–8 h after plating, prepare DNA reagents for transfection (<a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Reagents and ratios for transfection of the DRD2 and TRUPATH biosensors in this protocol</caption>
<thead>
<tr>
<th>Value</th>
<th>10 mM dish</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cell Density</td>
<td>6.5–7.0 × 10<sup>6</sup>
</td>
</tr>
<tr>
<td>Receptor DNA</td>
<td>1 μg hDRD2</td>
</tr>
<tr>
<td>Gα DNA</td>
<td>1 μg GαZ TRUPATH</td>
</tr>
<tr>
<td>Gβ</td>
<td>1 μg Gβ3</td>
</tr>
<tr>
<td>Gγ-GFP2</td>
<td>1 μg Gγ1-GFP2</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>400 μL</td>
</tr>
<tr>
<td>TransIT-2020</td>
<td>12 μL</td>
</tr>
</tbody>
</table>
<ol type="a">
<li>Bring Opti-MEM and TransIT-2020 (protected from light) to 22°C.
<ol type="i">
<li>Dilute DNA in Opti-MEM for each component (DRD2, G<math> <mrow> <mi>α</mi> </mrow> </math>Z-RLuc8, G<math> <mrow> <mi>β</mi> </mrow> </math>3, and GFP2-G<math> <mrow> <mi>γ</mi> </mrow> </math>1) to 10 ng/μL.</li>
</ol>
</li>
<li>Combine diluted DNA at a ratio of 1:1:1:1, to a total concentration of 4 μg (<a href="#fig2">Figure 2</a>D; <a href="#tbl1">Table 1</a>).
<ol type="i">
<li>Final volume should be 400 μL maintaining DNA concentration at 10 ng/μL.</li>
</ol>
</li>
<li>Add TransIT-2020 to the Opti-MEM/DNA mixture at a ratio of 3:1 TransIT-2020 (μL):DNA (μg) and gently mix/agitate by flicking (<a href="#fig2">Figure 2</a>D).
<ol type="i">
<li>For this protocol this is 12 μL of TransIT-2020.</li>
<li>Let DNA complexes form for 20–30 min (<a href="#fig2">Figure 2</a>D).</li>
</ol>
</li>
<li>Remove the tissue culture dish from step 2 from the incubator and evenly distribute DNA/Transit mixture dropwise into the tissue culture media (<a href="#fig2">Figure 2</a>E).
<ol type="i">
<li>Gently mix by moving the tissue culture dish in a slow figure eight motion.</li>
</ol>
</li>
<li>Return the tissue culture dish to the incubator overnight (<a href="#fig2">Figure 2</a>F).</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Preparation of the assay plate</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This section details how to prepare a 384-well plate of transfected cells to be used in the TRUPATH assay.</p>
<ol start="4">
<li>The following day remove the tissue culture dish prepared the previous day from the incubator and transfer cells to a 384-well white tissue culture plate.
<ol type="a">
<li>The 384-well white tissue culture plate should have been previously coated with Poly-L-Lysine as transfection of many cell types can reduce their adherence.
<ol type="i">
<li>1× Poly-L-Lysine solution should be made up at 100 μg/mL in sterile water + 1% Pen-Strep (v/v). Concentrated Poly-L-Lysine stocks can also be made, aliquoted, and frozen for use. Avoid freeze-thaw cycles.</li>
<li>Dispense 30 μL of 1× Poly-L-Lysine solution via pipette or multidrop into each well of the 384-well plate. We prefer to then incubate the plates with the solution for 30–60 min at 37°C.</li>
<li>Shake out Poly-L-Lysine and add 30 μL of sterile H<sub>2</sub>O + 1% Pen-Strep (v/v) to each well. If wrapped and stored in the dark at 4°C, these plates can be saved and used within 2–3 weeks.</li>
<li>Before using the plate, shake out liquid and let dry in a sterile hood for 1 h.</li>
</ol>
</li>
<li>Retrieve the 10 cm tissue culture dish containing the transfected HEK293T cells from the previous day (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Preparation of the assay plate</div>
<p>(A) Cells are removed from the tissue culture incubator are dislodged from the tissue culture dish with Versene and manual disruption following aspiration of the cell media.</p>
<p>(B–D) (B) Cells are then gently centrifuged, media aspirated, (C) resuspended and diluted to a density between 375 to 625 cells/μL, and (D) seeded into a Poly-L-Lysine coated 384-well tissue culture plate before being returned to the tissue-culture incubator.</p>
</figcaption>
</figure>
<ol type="i">
<li>Aspirate off media and harvest cells by gentle trituration using 4 mL of Versene to detach. More volume may be used as needed. Transfer cells to 50 mL conical tubes containing 6 mL 1% dFBS DMEM to quench the Versene.<br> To aid in detachment, the cells can be washed gently with PBS or Versene, with fresh Versene added after.<br> Using a 22°C table-top centrifuge, spin down the cells at 250× <i>g</i> for 5 min (<a href="#fig3">Figure 3</a>B).</li>
</ol>
</li>
<li>Aspirate off the supernatant from the pellet and add 1 mL of 1% dFBS DMEM to the cell pellet. Resuspend and dilute to 5 mL. Count cells, and adjust cell density between 375 and 625 cells/μL (<a href="#fig3">Figure 3</a>C). Ideal experimental conditions for cell density should be determined by the end user based on their observed growth rate.</li>
<li>Plate 40 μL (15,000–25,000 cells) into each well of the coated 384-well plate using the repeater pipette or multidrop (<a href="#fig3">Figure 3</a>C).</li>
<li>In the table-top centrifuge, spin the plates at 100× <i>g</i> for 15–30 s.</li>
<li>Return plates to the incubator for incubation overnight (<a href="#fig3">Figure 3</a>D).</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Running the assay, agonist mode</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This section details how to perform the TRUPATH assay to determine the activity of agonists.</p>
<div class="note">
<span class="note-title">Note:</span> Drug plates can be prepared by hand or by assisted automation (e.g., an Integra Assist Plus or an Echo acoustic dispenser). In this protocol we have utilized drug plates made by an acoustic dispenser (<a href="#fig4">Figure 4</a>). We have also limited the final DMSO concentrations to 0.33% but tolerance in an assay can be determined by the end-user. We describe a protocol for a 3-fold dilution of compounds to their final concentration, however, these procedural conditions can be modified by the end-user. For this experimental set-up we also perform concentration-response-curves with half-log dilutions, starting at a concentration of 10ˆ-4.5 M. Finally, we describe the use of an Integra VIAFLO for resuspension of compounds, transfer of assay buffer and substrate, and addition of compounds.</div>
<ol start="5">
<li>The following day, resuspend the drug plate by transferring 79.2 μL of drug buffer to each well (this assumes a 384-well drug plate prepared by an Echo acoustic dispenser in the required plate layout is being used, with each well containing 800 nL of compound or DMSO at 100× the desired concentration) (<a href="#fig5">Figure 5</a>A).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Drug plate format for 16 interleaved 10-point concentration-response curves in technical duplicate</div>
<p>Min and max columns represent negative (DMSO, minimum response) and positive (maximal concentration of reference agonist) controls. For this experiment, 300 μM of dopamine in 0.99% DMSO (3×, final concentration of 100 μM, 0.33% DMSO) was used as the maximum control. Full-log dilutions of dopamine starting at 300 μM (100 μM) were used to create the curve in this experiment, but alternate dilutions schemes can be used as well. Drug plates prepared by the Echo acoustic dispenser performs dilutions by seeding low volumes of compound solution and back-filling with DMSO.</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Running the assay</div>
<p>(A) Compounds are resuspended using an Integra VIAFLO to transfer drug buffer to the drug plate from <a href="#fig4">Figure 4</a>.</p>
<p>(B) <i>Renilla</i> Luciferase substrate (e.g., Prolume Purple) is resuspended in methanol to 1 mM and diluted in assay buffer to 7.5 μM.</p>
<p>(C–F) (C) Media is removed by manual ejection from the tissue culture plate from <a href="#fig2">Figure 2</a> and (D) replaced with diluted luciferase substrate (E). Resuspended compounds are added to the tissue culture plate containing transfected cells and substrate before (F) being loaded into the plate reader.</p>
<p>(G) Finally, data are analyzed.</p>
</figcaption>
</figure>
</li>
<li>Dissolve BRET2 substrate such as Prolume Purple or coelenterazine 400a in methanol to a concentration of 1 mM (<a href="#fig5">Figure 5</a>B).
<ol type="a">
<li>Store in the dark on dry ice during use and then wrap in Parafilm and store at −80°C following the experiment. Use within 1–2 weeks and then discard.</li>
</ol>
</li>
<li>Shake out and blot the 384-well tissue culture plate (<a href="#fig5">Figure 5</a>C). Sometimes this will require a flick/shake, blot, then a reversal of the orientation and a repeat. It is important to make sure that as much of the DMEM is out and that this is even.
<ol type="a">
<li>After shaking out the cell culture media, place the PerkinElmer white backing strips (6005199) on the underside of the plate to create a white reflective bottom. If using assay plates which are already white-bottomed the adhesive strip can be skipped. For the 384-well plate format, vacuum aspiration and/or washing may result in detrimental cell loss. We have not experienced a significant interference of residual phenol red-containing media on the BRET signal, but to avoid this phenol-red free DMEM may be used when plating cells.</li>
</ol>
</li>
<li>Dilute 75 μL of substrate (e.g., Prolume Purple) into 10 mL final volume of assay buffer (7.5 μM final concentration) (<a href="#fig5">Figure 5</a>B).
<ol type="a">
<li>Load 20 μL of this solution into each of the wells in the cell plate (<a href="#fig5">Figure 5</a>D). This can be performed manually via pipetting or using a liquid handler such as an Integra VIAFLO.</li>
</ol>
</li>
<li>Using the Integra VIAFLO fitted with a 12.5 μL 384 head, with the drug plate in the left position and the cell plate in the right position, transfer 10 μL of drugs to the cell plate (<a href="#fig5">Figure 5</a>E).</li>
<li>In a table-top centrifuge, spin the plates at 100× <i>g</i> for 10 s.</li>
<li>Load the plate into the PHERAstar FSX (<a href="#fig5">Figure 5</a>F).
<ol type="a">
<li>Reading protocols can be adapted by the end user based on the results and requirements of their target. For this protocol we scan each well with a 3 mm spiral average, reading the plate 8 times over a period of 40 min.
<ol type="i">
<li>We have found that EC<sub>50</sub>s for DRD2 stabilize rapidly in minutes, however these settings should be established by the end-user as unique for their targets.</li>
</ol>
</li>
<li>Gain for GFP2 and RLuc8 channels should be optimized by the end-user as optics for plate readers even of the same build can vary, and signal intensity will differ depending on whether the end-user is using coelenterazine 400a or Prolume Purple. We typically use gain settings of 3800 for channel A (GFP2) and 3000 for channel B (RLuc8).</li>
</ol>
</li>
<li>Export the data from the plate reader for subsequent analysis (<a href="#fig5">Figure 5</a>G).</li>
</ol>
<h3 id="sec3.4">Analysis, agonist mode</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This section details how to analyze the data obtained from TRUPATH agonist assays.</p>
<div class="note">
<span class="note-title">Note:</span> Software for analysis may vary depending on user preference. However, the overall method for curve-fitting is generalizable. Here we describe using GraphPad Prism and a 4-parameter logistic regression.</div>
<ol start="13">
<li>Calculate the ratio of GFP2 signal to RLuc8 signal (i.e., GFP2 counts/RLuc8 counts). As TRUPATH is a G protein dissociation assay, a decrease in the ratio is inversely proportional to the magnitude of activation of the system.
<ol type="a">
<li>BRET values may be variable from experiment-to-experiment. Therefore, these values should be normalized for each experiment if possible.</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Transfer the data to an X/Y plot in GraphPad Prism.
<ol type="a">
<li>For initial experiments with multiple time-points separate data-tables can be generated.
<ol type="i">
<li>Using a 4-parameter logistic fit, calculate the EC<sub>50</sub>s and plot these by time (<a href="#fig6">Figure 6</a>A). Identify the time-point where the EC<sub>50</sub> values have stabilized and use this as the typical time-course for your target system (<a href="#fig6">Figure 6</a>B).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Data acquisition and analysis, agonist mode</div>
<p>(A) BRET2 concentration response curves across 25 min. A decrease in BRET indicates a decrease in the associated (inactive) heteroterimeric G protein complexes (i.e., agonism of the DRD2 receptor). BRET ratio (515/405) is plotted on the Y axis with concentration of dopamine on the X axis. Data represent an average of 32 technical replicates (full plate) with SEM.</p>
<p>(B) Table of Z′ and pEC<sub>50</sub> values showing a robust assay.</p>
<p>(C) Normalized concentration-response-curve from the 20-min time-point from panel A. Intersection of dotted-lines indicates the estimated concentration at the 80% point on the 4-parameter fitted curve. In this experiment the EC<sub>80</sub> was estimated as 60 nM. After multiple internal replicates we identified a typical EC<sub>80</sub> concentration of 30 nM.</p>
</figcaption>
</figure>
Plate statistics such as Z′ or Robust Z′ over time can also be calculated and used to determine when the assay is sufficiently robust, and for how long, as additional constraints on the reliability of the data for analysis.</li>
</ol>
</li>
</ol>
</li>
<li>Using the normalization function, normalize the change in BRET ratio with compound treatment using the average ratio-value from the minimum signal wells (DMSO only) as 0% and maximum signal wells (sufficient concentration of control compound to achieve a maximal response) wells as 100% (<a href="#fig6">Figure 6</a>C).</li>
<li>Using these normalized values, perform a 4-parameter logistic fit regression. This will provide the end-user with estimates of Emin (bottom of curve), Emax (top of curves, and efficacy), EC<sub>50</sub> (potency), and hill-slope.
<ol type="a">
<li>In order to ensure robustness of the assay, independent biological replicates should be carried out over different days to ensure repeatability.</li>
<li>In preparation for running an antagonist-mode assay, the agonist stimulation condition that will be used as a competitor for potential antagonist compounds should be determined.
<ol type="i">
<li>It is general convention to use an EC<sub>80</sub> concentration of the test agonist. Using the “log(Agonist) vs. response – Find EC<sub>anything</sub>” curve, set the EC<sub>F</sub> to 80. This will provide an output of the effective concentration at EC<sub>80</sub> (<a href="#fig6">Figure 6</a>C) rather than the typical EC<sub>50</sub>. This concentration of agonist can be used as a competitor for antagonist assays.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Running the assay, antagonist mode</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This section details how to perform the TRUPATH assay to determine the activity of antagonists.</p>
<div class="note">
<span class="note-title">Note:</span> We generally perform the antagonist assay via co-addition of a mixture of agonist and antagonists.</div>
<ol start="17">
<li>Resuspend the drug plate by transferring 79.2 μL of drug buffer to each well (this assumes a 384-well drug plate with 800 nL of compound dissolved in DMSO). This can be done manually or via an automated liquid handler such as an Integra VIAFLO with a 384-well 125-μL head.
<ol type="a">
<li>The drug plate should contain test compounds at the end user’s desired concentration for either single-point or concentration-response assays (<a href="#fig7">Figure 7</a>). However, each well containing a test antagonist should also contain the test agonist from the previous steps at an EC<sub>80</sub> or other concentration. Control wells should consist of minimum (DMSO), maximum (sufficient concentration of test agonist to achieve a maximal response), and EC<sub>80</sub> concentration of the test agonist.
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Antagonist drug plate format for 16 interleaved 10-point concentration-response curves in technical duplicate</div>
<p>Min and max wells represent negative (DMSO, minimum response) and positive (maximal concentration of reference agonist) controls. For this experiment, 300 μM of dopamine in 0.99% DMSO (3×, final concentration of 100 μM, 0.33% DMSO) was used as the maximum control. EC<sub>80</sub> wells represent 90 nM of dopamine (30 nM final concentration). Test antagonist wells also contained this EC<sub>80</sub> concentration of DMSO. Full-log dilutions of test antagonists starting at 30 μM (10 μM final) were used to create the curve. Droperidol (1–4), Ziprasidone HCl (5–8), L741,626 (9–12), and Risperidone (13–16) were used as test antagonists in this experiment. Drug plates prepared by the Echo acoustic dispenser performs dilutions by seeding low volumes of compound solution and back-filling with DMSO.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Dissolve BRET2 substrate such as Prolume Purple or coelenterazine 400a in methanol to a concentration of 1 mM.
<ol type="a">
<li>Store in the dark on dry ice during use and then wrap in Parafilm and store at −80°C following use. Use within 1–2 weeks and then discard.</li>
</ol>
</li>
<li>Shake out and blot the 384-well tissue culture plate. Sometimes this will require a flick/shake, blot, then a reversal of the orientation and a repeat. It is important to make sure that as much of the DMEM is out and that this is even.
<ol type="a">
<li>After shaking out media, place the white backing strips on the underside of the plate to create a white reflective bottom. If using assay plates which are already white-bottomed the adhesive strip can be skipped.</li>
</ol>
</li>
<li>Dilute 75 μL of substrate (e.g., Prolume Purple) into 10 mL final volume of assay buffer (7.5 μM final concentration).
<ol type="a">
<li>Load 20 μL of this solution into each of the wells in the cell plate. This can be performed manually via pipetting or using a liquid handler such as an Integra VIAFLO.</li>
</ol>
</li>
<li>Using the Integra VIAFLO fitted with a 12.5 μL 384 head, with the drug plate in the left position and the cell plate in the right position, transfer 10 μL of drugs to the cell plate.</li>
<li>In a table-top centrifuge, spin the plates at 100× <i>g</i> for 10 s.</li>
<li>Load the plate into the PHERAstar FSX.
<ol type="a">
<li>Reading protocols can be adapted by the end user based on the results and requirements of their target. For this protocol we utilize a 3 mm spiral average read over a period of 0.52 s per well, reading the plate 8 times over a period of 40 min.
<ol type="i">
<li>We have found that IC<sub>50</sub>s for DRD2 stabilize at about 30 min, but these settings should be established by the end-user as unique for their targets.</li>
</ol>
</li>
<li>Gain for GFP2 and RLuc8 channels should be optimized by the end-user as optics for plate readers even of the same build can vary, and signal intensity will differ depending on whether the end-user is using coelenterazine 400a or Prolume Purple. We typically use gain settings of 3800 for channel A (GFP2) and 3000 for channel B (RLuc8).</li>
</ol>
</li>
<li>Export the data from the plate reader for subsequent analysis.</li>
</ol>
<h3 id="sec3.6">Analysis, antagonist mode</h3>
<p>This section details how to analyze the data obtained from TRUPATH antagonist assays.</p>
<p>Z′ and similar plate statistics should be calculated using the EC<sub>80</sub> response control wells as the “maximum.” The maximum control wells should be used with the minimum control wells to ensure that a response in the EC<sub>80</sub> agonist control wells approximates 80%. We generally tolerate a 10% deflection in this value when approving or failing an assay.</p>
<ol start="25">
<li>Calculate the ratio of GFP2 signal to RLuc8 signal. Because TRUPATH is a G protein dissociation assay, increase in this ratio is inversely proportional to the magnitude of antagonism of the EC<sub>80</sub> stimulation of the test agonist.
<ol type="a">
<li>BRET values may be variable from experiment-to-experiment. Therefore these values should be normalized for each experiment if at all possible.</li>
</ol>
</li>
<li>Transfer the data to an X/Y plot in GraphPad Prism (<a href="#fig8">Figure 8</a>A).
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Data acquisition and analysis, antagonist mode</div>
<p>(A) BRET data showing an increase in association of the heterotrimeric G protein complex with increasing concentration of test antagonists. These data reflect a functional competition with the EC<sub>80</sub> concentration of dopamine. Data represent an average of 8 technical replicates with SEM.</p>
<p>(B) A representation of the activity of dopamine in the test-agonist wells. The dashed line indicates 80% of the maximal response of 10 uM dopamine. At diminishing concentrations of antagonist the activity of the 30 nM of dopamine can be resolved in the upper asymptote. The lower asymptote reflects complete antagonism of the the EC<sub>80</sub> (30 nM dopamine) signal. Z′ for this assay was 0.690.</p>
<p>(C) Concentration-response-curves of the test antagonists normalized.</p>
<p>(D) Test/Retest plot of pIC<sub>50</sub>s of the test antagonists from two temporally separated biological replicates, with an R<sup>2</sup> of 0.824.</p>
</figcaption>
</figure>
</li>
<li>Using the normalization function, calculate the change in BRET ratio with compound treatment using the minimum and EC<sub>80</sub> control wells as 100% and 0% respectively.
<ol type="a">
<li>To visualize the competition of the EC<sub>80</sub> signal (<a href="#fig8">Figure 8</a>B) a graph can be generated normalizing BRET values to max columns (100%) and min columns (0%).</li>
</ol>
</li>
<li>Using these normalized values, perform a 4-parameter logistic fit regression. This will provide the end-user with estimates of Emin (bottom of curve), Emax (top of curves, and efficacy), IC<sub>50</sub> (potency), and hill-slope. Using normalized values Prism will return an “EC<sub>50</sub>” but this reflects the IC<sub>50</sub> in reality (<a href="#fig8">Figure 8</a>C).
<ol type="a">
<li>In order to ensure robustness of the assay, independent biological replicates should be carried out over different days to ensure repeatability (<a href="#fig8">Figure 8</a>D).</li>
</ol>
</li>
</ol>
<h3 id="sec3.7">Running the assay, plate uniformity assessment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Once an assay is fully optimized the robustness of assay performance should be assessed before it is used for high-throughput compound screening. Here we describe a protocol for assessing the plate uniformity of the human DRD2 Gz TRUPATH assay in 384 well format, using 4 assay signals: Max, EC<sub>80</sub>, EC<sub>50</sub> and Min, though these signal conditions can be modified by the user.</p>
<div class="note">
<span class="note-title">Note:</span> Plate uniformity assessments should be repeated with multiple plates and on different days to assess inter-plate and inter-day uniformity.</div>
<p>For more details on the use and setup of plate uniformity assessments, please refer to <a href="#bib7">Iversen et al., 2004</a>.</p>
<div class="note">
<span class="note-title">Note:</span> Assay plates are setup as described in the initial cell culture and transfection, and preparation of assay plate sections above. Drug plates can be prepared by hand or by assisted automation (e.g., an Integra Assist Plus or an Echo acoustic dispenser). In this protocol we have utilized an electronic multichannel pipette to dispense the drugs into the cell plate. We also utilize a final DMSO concentration of 0.33%, as this is the final concentration used for compound screening. For this experimental set-up we use final dopamine concentrations of 100 μM for Max, 30 nM for EC<sub>80</sub> and 4 nM for EC<sub>50</sub> (<a href="#fig9">Figure 9</a>).</div>
<ol start="29">
<li>Prepare 3× final concentrations of dopamine used for Max, EC<sub>80</sub> and EC<sub>50</sub> signal conditions in drug buffer (assuming addition of 10 μL per well for each signal condition as shown in <a href="#fig9">Figure 9</a>).
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Drug plate layout containing 4 assay signals in repeating columns</div>
<p>Max and Min columns represent positive (maximal concentration of reference agonist) and negative (DMSO, minimum response) controls. EC<sub>80</sub> and EC<sub>50</sub> columns represent the concentration of reference agonist that produces 80% and 50% maximal responses, respectively. For this experiment, 300 μM of dopamine in 0.99% DMSO (final concentration of 100 μM, 0.33% DMSO) was used as the Max control, 90 nM of dopamine in 0.99% DMSO (final concentration of 30 nM, 0.33% DMSO) as EC<sub>80</sub> signal, and 12 nM of dopamine in 0.99% DMSO (final concentration of 4 nM, 0.33% DMSO) as EC<sub>50</sub> signal.</p>
</figcaption>
</figure>
</li>
<li>Dissolve BRET2 substrate such as Prolume Purple or coelenterazine 400 in methanol to a concentration of 1 mM.
<ol type="a">
<li>Store in the dark on dry ice during use and then wrapped in Parafilm and store at −80°C following use. Use within 1–2 weeks and then discard.</li>
</ol>
</li>
<li>Shake out and blot the 384-well tissue culture plate. Sometimes this will require a flick/shake, blot, then a reversal of the orientation and a repeat. It is important to make sure that as much of the DMEM is out and that this is even.
<ol type="a">
<li>After shaking out the cell culture media, place the PerkinElmer white backing strips (6005199) on the underside of the plate to create a white reflective bottom. If using assay plates which are already white-bottomed the adhesive strip can be skipped.</li>
</ol>
</li>
<li>Dilute 75 μL of substrate (e.g., Prolume Purple) into 10 mL final volume of assay buffer (7.5 μM final concentration).
<ol type="a">
<li>Load 20 μL of this solution into each of the wells in the cell plate. This can be done manually via pipetting or using a liquid handler such as an Integra VIAFLO.</li>
</ol>
</li>
<li>Immediately using an electronic multichannel pipette transfer 10 μL per well of each signal condition to the appropriate columns on the 384-well cell plate (<a href="#fig9">Figure 9</a>).</li>
<li>In a table-top centrifuge, spin the plate at 100 × <i>g</i> for 10 s.</li>
<li>Load the plate into the PHERAstar FSX.
<ol type="a">
<li>Reading protocols can be adapted by the end-user based on the results and requirements of their target. For this protocol we scan each well with a 3 mm spiral average, reading the plate multiple times over 40 minutes.</li>
<li>Gain for GFP2 and RLuc8 channels should be optimized by the end-user as optics for plate readers even of the same build can vary, and signal intensity will differ depending on whether the end-user is using coelenterazine 400a or Prolume Purple. We typically use gain settings of 3800 for channel A (GFP2) and 3000 for channel B (RLuc8).</li>
</ol>
</li>
<li>Export data from the plate reader for analysis.</li>
</ol>
<h3 id="sec3.8">Analysis, plate uniformity assessment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This section details how to analyze plate uniformity in the TRUPATH assay.</p>
<div class="note">
<span class="note-title">Note:</span> Software for analysis may vary depending on user preference. Here we describe using GraphPad Prism.</div>
<ol start="37">
<li>Calculate the ratio of GFP2 signal to RLuc8 signal. As TRUPATH is a G protein dissociation assay, a decrease in the ratio is inversely proportional to the magnitude of activation of the system.
<ol type="a">
<li>BRET values may be variable from experiment-to-experiment. Therefore these values should be normalized for each experiment if possible.</li>
</ol>
</li>
<li>Organize data by well number (1–384) first by column, then row and also by row, then column.
<ol type="a">
<li>The data from the time-point expected to be analyzed for compound screening should be used for this analysis. The example data shown in <a href="#fig10">Figure 10</a> uses results taken 30 min post drug addition.
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Human DRD2 Gz TRUPATH assay plate uniformity assessment</div>
<p>A decrease in BRET ratio (515/410) indicates dissociation of heterotrimeric G protein complexes (i.e., agonism of the DRD2 receptor).</p>
<p>(A and B) (A) BRET ratios of the 4 signal conditions plotted by well number by column, then row, and (B) well number by row, then column.</p>
<p>(C and D) (C) Normalised BRET ratio (normalised to min and max) plotted by well number by column, then row and (D) by row, then column. The data shown is the response from every well in a 384-well plate 30 min post-drug addition and dopamine was used at 100 μM for Max, 30 nM for EC<sub>80</sub> and 4 nM for EC<sub>50</sub> final concentrations.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Transfer the data to an X/Y plot in GraphPad Prism, separating each signal condition into different groups, and plotting well number along the X-axis.
<ol type="a">
<li>Produce separate graphs for well number plotted by column then row (<a href="#fig10">Figure 10</a>A) and also by row then column (<a href="#fig10">Figure 10</a>B), so any directional effects in the responses (such as drift or edge effects) on the plate can be assessed.</li>
</ol>
</li>
<li>Using the normalization function, normalize the change in BRET ratio using the average BRET ratio of the Min signal wells as 0% and average BRET ratio of the Max signal wells as 100%.</li>
<li>Using these normalized values, create X/Y plots of the normalized data plotted by well number as in step 38 (<a href="#fig10">Figures 10</a>C and <a href="#fig10">10</a>D).</li>
<li>Plate statistics including Z prime, % coefficient of variation (%CV) ((condition mean/condition standard deviation)∗100) should be calculated and used to determine if the assay is sufficiently robust and reliable for high-throughput screening (<a href="#fig11">Figure 11</a>).
<figure id="fig11"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig11.jpg" alt="Fig11.jpg">
<figcaption>
<div class="figcaption-title">Figure 11. Normalized responses from the plate uniformity assessment with signal condition averages and Z′ statistics by row and column</div>
</figcaption>
</figure>
<ol type="a">
<li>These values should be compared to the plate acceptance criteria in <a href="#tbl2">Table 2</a>.
<table id="tbl2">
<caption>Table 2. Summary of acceptance criteria used to assess plate uniformity in intra-plate tests and the determined values for the plate uniformity test carried out on the human DRD2 Gz TRUPATH assay</caption>
<thead>
<tr>
<th>Criterion</th>
<th>Acceptance range</th>
<th>Observed values for DRD2 Gz TRUPATH assay</th>
</tr>
</thead>
<tbody>
<tr>
<td>%CV Max</td>
<td>≤20%</td>
<td>2.49</td>
</tr>
<tr>
<td>%CV EC<sub>80</sub>
</td>
<td>≤20%</td>
<td>3.58</td>
</tr>
<tr>
<td>%CV EC<sub>50</sub>
</td>
<td>≤20%</td>
<td>4.36</td>
</tr>
<tr>
<td>%CV Min</td>
<td>≤20%</td>
<td>4.06</td>
</tr>
<tr>
<td>Normalised SD EC<sub>80</sub>
</td>
<td>≤20</td>
<td>5.39</td>
</tr>
<tr>
<td>Normalised SD EC<sub>50</sub>
</td>
<td>≤20</td>
<td>7.77</td>
</tr>
<tr>
<td>Z prime (max and min)</td>
<td>≥0.4</td>
<td>0.63</td>
</tr>
</tbody>
</table>
<p class="table-legend">% coefficient of variation (%CV) and standard deviation (SD). Criterion and acceptance criteria adapted from <a href="#bib7">Iversen et al., 2004</a>.<br> Data shown in <a href="#fig10">Figure 10</a>.</p>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A successful TRUPATH experiment should show unambiguous signal differences between positive and negative controls, and ideally a concentration-response curve with reproducible measures of potency and efficacy. The absolute signal (measured as the difference in BRET signal) can vary experiment to experiment, as well as target-to-target, which is why it is key to normalize values instead of using raw BRET ratios. To interpret successful experiments, we rely on the use of test statistics such as Z′ and robust Z′, as well as historical comparisons of EC<sub>50</sub> or IC<sub>50</sub> values for reference agonists and antagonists, respectively.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Example data showing values of the acceptor signal (515-530λ, <a href="#fig12">Figure 12</a>A), the donor signal (410-480λ, <a href="#fig12">Figure 12</a>B), and the calculated BRET ratio (Acceptor/Donor, <a href="#fig12">Figure 12</a>C).</p>
<figure id="fig12"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1529-Fig12.jpg" alt="Fig12.jpg">
<figcaption>
<div class="figcaption-title">Figure 12. Example data from an agonist-mode concentration-response curve pairing dopamine and the DRD2 receptor</div>
<p>Data are presented with a heatmap to exemplify both the variability and the magnitude of changing ratio. Layout reflects the design from <a href="#fig3">Figure 3</a>.</p>
<p>(A) Fluorescence counts from the GFP2 acceptor protein as measured using a 515-530 nM filter.</p>
<p>(B) Luminescence counts from the oxidation of Prolume Purple by RLuc8 as measured using a 410/480 nM filter.</p>
<p>(C) The BRET ratio, calculated by taking the ratio of the acceptor counts (A) to the donor counts (B).</p>
</figcaption>
</figure>
<p>The 4-parameter logistic curve for fitting agonist or antagonist concentration response curves is Y=100/(1+10ˆ((LogEC<sub>50</sub>-X)∗HillSlope)) where Y is the response (BRET ratio, normalized value, etc) and X is the logarithm of the test compound at that corresponding datapoint, and is a component of most statistical analysis packages. Normalization is calculated as the percent of activity between either positive and negative controls, or the EC<sub>80</sub> control wells and negative controls.</p>
<p>Z′ is calculated as follows: 1-((3∗(<b>σmin + σ</b>max)))/(abs((<math> <mrow> <mover> <mtext>x</mtext> <mo>¯</mo> </mover> </mrow> </math><b>min –</b> <math> <mrow> <mover> <mtext>x</mtext> <mo>¯</mo> </mover> </mrow> </math>max)), with a robust assay yielding a score between 0.5-1. Individual accept/reject criteria should be established based on experimental needs and historical performance.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Similar to other cell-based experimental systems, TRUPATH depends on successful expression of your receptor of interest at sufficient levels, with proper membrane localization. It is not possible <i>a priori</i> to establish the optimal receptor transfection and expression conditions and so this must be performed by the end-user. Epitope-tagged receptors (e.g., hemagglutinin or flag-tagged N-termini) can be used in conjunction with a cell-surface ELISA to determine successful expression.</p>
<p>Ultimately some receptors may exhibit very limited dynamic range and so the experimental criteria for a successful experiment may differ. Scaling experiments to lower-throughput format (e.g., 96-well vs 384-well) can improve the robustness of the assay at the cost of efficiency.</p>
<p>The dynamic range of each sensor varies and is in part dependent on the receptor system being studied. While TRUPATH is an exhaustively optimized probe set, some pathways for some receptors will perform better than others and this should be taken into account when interpreting results from multiple G protein sensor pathways.</p>
<p>BRET2 substrates such as coelenterazine 400a and Prolume Purple are not as bright as some of their counterparts (e.g., Furimiazine for Nanoluc), however the spectral separation between donor and acceptor provides exceptional signal-to-noise. Nevertheless, expression conditions for each receptor should be optimized by the end-user to ensure sufficient signal.</p>
<p>Since TRUPATH is a luciferase- and GFP2-based assay, compounds that either inhibit RLuc8, quench signal, or have overlapping fluorescent spectra, may interfere with the assay. Appropriate controls should be used to confirm compound-receptor activity, ideally in an orthogonal assay.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<p>The protocol described here and in the original TRUPATH paper should be used as a general guideline; users should tailor protocols to their experimental needs. These include considerations such as cell culture media used, which can be problematic for certain receptors like glutamate and serotonin receptors.</p>
<h3 id="sec7.1">Problem 1</h3>
<p>Low TRUPATH plasmid expression.</p>
<p>TRUPATH plasmids were verified by next-generation sequencing by Addgene and Sanger sequencing by the Roth lab prior to being made available. After at least 16 h post-transfection, cells should be checked for expression of GFP2 by fluorescence microscopy as a proxy for biosensor expression. Ideally, at least 50% of the cells should be positive for GFP2. It should be noted that not all GFP2-positive cells will necessarily have been co-transfected with the receptor, G<math> <mrow> <mi>α</mi> </mrow> </math>-RLuc8, and G<math> <mrow> <mi>β</mi> </mrow> </math> plasmids.</p>
<p>After plasmids have been prepared, users should use a DNA spectrophotometer to quantify the purity and yield of their plasmids, and confirm that the plasmid inserts are intact by Sanger sequencing. Sequencing should use the CMV forward and bovine growth hormone (BGH; the terminator for these plasmid inserts) reverse primers, as well as additional primers (e.g., GFP2, RLuc8) to ensure full and overlapping coverage. Plasmids should be stored at −20°C or 4°C in molecular grade water or TE buffer.</p>
<p>If users find that GFP2 or RLuc8 expression (either by fluorescence microscopy inspection or detection by a plate reader) is low, and the issues are not related to the integrity of the plasmids, the most likely reasons are due to issues with the cells being used (e.g., high passage number) or transfection conditions.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>As described in Initial Cell Culture and Transfection, step 1, cells should be of low passage number and of high viability (e.g., &gt;90%). If these conditions are not met, new cells should be thawed, and allowed to go through several passages prior to experimental use. Prior to transfection, the confluency of cells in the plate to be transfected should be viewed to assess their density and morphology, to ensure that they appear both dense and healthy enough for transfection. Cells should be 70–80% confluent at the time of transfection. An additional consideration is the total amount of each plasmid that is transfected based upon the transfection reagent used. In our experience, twice as much plasmid DNA needs to be added to cells when using PEI versus TransIT-2020.</p>
<p>If GFP2 expression appears robust, but low RLuc8 counts are being detected by a plate reader, this may be due to issues with the substrate being used. Care should be given to prepare substrate stock according to manufacturer’s instructions, to dilute the stock properly while running the assay, and to store the stock at the appropriate temperatures for short-term and long-term use. An additional solution may be to adjust the gain settings on the plate reader.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Cell loss during media decanting.</p>
<p>We recommend shaking out 384-well plates to decant media when ready for experimental use. This may lead to some, but not great, cell loss. However, if it is found that so many cells are being lost that experimental results are affected (e.g., low counts), this may be due to issues with the Poly-L-Lysine coating. Poly-L-Lysine stocks should be properly prepared and stored, and plates coated as described in Preparation of the Assay Plate, step 4. We have had minimal cell loss from coating the plates described in this step; consulting the manufacturer’s instructions, though they should not deviate greatly from those we provide, may be preferred.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>When the plates are shaken out, it should be done gently, but with enough force to minimize residual cell culture media. We have not noticed a significant effect of phenol red on the BRET ratio, but too much media remaining in the plates may lead to noise as a result of unequal volumes across wells.</p>
<p>While we recommend vacuum aspiration of cell culture media for the 96-well plate format, it should not be necessary for the 384-well format. However, this may be an solution to cell loss, so long as care is taken in the aspiration process. Users should determine that cell detachment is occurring during the shaking out, and not due to general poor cell adherence, or vacuum aspiration will likely result in similar cell loss.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Lack of response for known control compounds.</p>
<p>In some circumstances, a response to known control compounds may not be observed. If this is a one-time occurrence, it may be due to an oversight in following the protocol. Repeated attempts and failures to observe an expected response may indicate an underlying issue that should be addressed.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Because TRUPATH relies on robust target expression, users should ensure sufficient receptor expression, and that trafficking to the cell membrane is not perturbed. In addition to performing receptor DNA titration experiments (increasing concentrations of receptor plasmid), the end user can perform a cell-surface ELISA against an N-terminal tag such as FLAG, SNAP, or HA. Identifying conditions that produce robust cell-surface expression should solve this issue.</p>
<p>We have also found that for certain receptors, fusing the TRUPATH G<math> <mrow> <mi>α</mi> </mrow> </math> to the C-termini of the receptor can improve ligand response, as well as provide a rough measure for receptor expression.</p>
<p>A second potential solution may be to explore additional G<math> <mrow> <mi>α</mi> </mrow> </math> subunits. Coupling of receptors to specific G<math> <mrow> <mi>α</mi> </mrow> </math> subunits may be documented in the literature, but this is no guarantee. Traditional cellular assays may not effectively differentiate within G<math> <mrow> <mi>α</mi> </mrow> </math>i/o family subtypes, and so exploring additional G<math> <mrow> <mi>α</mi> </mrow> </math> subunits (a “transducerome screen,”) may identify whether the end user is performing the right assay to begin with.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Low signal window.</p>
<p>The signal window of the TRUPATH assay is dependent upon many factors. A low signal window may be a problem for some users, particularly those investigating receptors that have high basal activity or high sensitivity to cell culture conditions. For example, glutamate and serotonin are present in culture media, and may lead to receptor desensitization and internalization.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>A potential solution to high basal activity is to titrate the ratio of receptor DNA to biosensor DNA transfected. For example, rather than transfecting the receptor:G<math> <mrow> <mi>α</mi> </mrow> </math>-Rluc8:G<math> <mrow> <mi>β</mi> </mrow> </math>:GFP2-G<math> <mrow> <mi>γ</mi> </mrow> </math> at a 1:1:1:1 ratio, a 0.25:1:1:1 or 0.5:1:1:1 ratio may be more favorable. Ideal transfection ratios can only be determined empirically. When performing titration comparisons, an empty vector should be transfected for consistency in transfection amounts.</p>
<p>A potential solution to high sensitivity to cell culture conditions can be addressed by minimizing receptor exposure to such conditions. For example, serotonin is found in high concentrations in FBS. Dialyzed and synthetic FBS has reduced and no serotonin, respectively, and switching from full serum to dialyzed or synthetic serum in a timely manner after transfection (i.e., 6–8 h) may limit receptor exposure following the onset of receptor expression. Transfections in a higher concentration of dialyzed serum may also be beneficial for maintaining cell health while minimizing serotonin exposure, as has been done in other assays, including previous iterations of G protein BRET biosensors (for example, see <a href="#bib12">McCorvy et al. 2018</a>).</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Noisy data.</p>
<p>TRUPATH is a BRET-based platform, and therefore the data it generates are ratiometric. Reasonable variations to the number of cells in a well or receptor expression density should not have a significant effect on the BRET ratio for wells subjected to the same experimental conditions. High noise in TRUPATH data most likely results from variation in well volumes resulting from inconsistencies in pipetting, whether due to equipment or user error.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Users should practice correct pipetting techniques and have their pipettes routinely calibrated.</p>
<p>For the 384-well format in particular, which uses smaller volumes that are more sensitive to fluctuation, we recommend using a liquid handling system (see <a href="#sec3.3">running the assay, agonist mode</a> and <a href="#sec3.5">running the assay, antagonist mode</a>). In our experience, automated liquid handling versus user liquid handling greatly reduces data noise. Users who do not have access to liquid handling systems may be better suited for adapting the TRUPATH platform to the 96-well format, particularly when performing experiments where quantification accuracy matters (as opposed to initial screening, where higher through-put with less quantitative accuracy can be tolerated).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chakraborty, S., Uprety, R., Daibani, A.E., Rouzic, V.L., Hunkele, A., Appourchaux, K., Eans, S.O., Nuthikattu, N., Jilakara, R., Thammavong, L., et al. (2021a). Kratom alkaloids as probes for opioid receptor function: pharmacological characterization of minor indole and oxindole alkaloids from kratom. ACS Chem. Neurosci. <i>12</i>, 2661-2678. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref1">View at publisher</a></p>
<p id="bib2">Chakraborty, S., DiBerto, J.F., Faouzi, A., Bernhard, S.M., Gutridge, A.M., Ramsey, S., Zhou, Y., Provasi, D., Nuthikattu, N., Jilakara, R., et al. (2021b). A novel mitragynine analog with low-efficacy mu opioid receptor agonism displays antinociception with attenuated adverse effects. J. Med. Chem. <i>64</i>, 13873-13892. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref2">View at publisher</a></p>
<p id="bib3">Chao, M.L., Luo, S., Zhang, C., Zhou, X., Zhou, M., Wang, J., Kong, C., Chen, J., Lin, Z., Tang, X., et al. (2021). S-nitrosylation-mediated coupling of G-protein alpha-2 with CXCR5 induces Hippo/YAP-dependent diabetes-accelerated atherosclerosis. Nat. Commun. <i>12</i>, 4452. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref3">View at publisher</a></p>
<p id="bib4">Cao, C., Kang, H.J., Singh, I., Chen, H., Zhang, C., Ye, W., Hayes, B.W., Liu, J., Gumpper, R.H., Bender, B.J., et al. (2021). Structure, function and pharmacology of human itch GPCRs. Nature <i>600</i>, 170-175. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref4">View at publisher</a></p>
<p id="bib5">Du, J., Wang, D., Fan, H., Xu, C., Tai, L., Lin, S., Han, S., Tan, Q., Wang, X., Xu, T., et al. (2021). Structures of human mGlu2 and mGlu7 homo- and heterodimers. Nature <i>594</i>, 589-593. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref5">View at publisher</a></p>
<p id="bib6">Gao, Y., Robertson, M.J., Rahman, S.N., Seven, A.B., Zhang, C., Meyerowitz, J.G., Panova, O., Hannan, F.M., Thakker, R.V., Bräuner-Osborne, H., et al. (2021). Asymmetric activation of the calcium-sensing receptor homodimer. Nature <i>595</i>, 455-459. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref6">View at publisher</a></p>
<p id="bib7">Iversen, P.W., Beck, B., Chen, Y.F., Dere, W., Devanarayan, V., Eastwood, B.J., Farmen, M.W., Iturria, S.J., Montrose, C., Moore, R.A., et al. (2004). Hts assay validation. In Assay Guidance Manual, S. Markossian, A. Grossman, K. Brimacombe, M. Arkin, D. Auld, C.P. Austin, J. Baell, T.D.Y. Chung, N.P. Coussens, J.L. Dahlin, et al, eds. (Eli Lilly &amp; Company and the National Center for Advancing Translational Sciences), pp. 1207-1231. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref7">View at publisher</a></p>
<p id="bib8">Kim, K., Che, T., Panova, O., DiBerto, J.F., Lyu, J., Krumm, B.E., Wacker, D., Robertson, M.J., Seven, A.B., Nichols, D.E., et al. (2020). Structure of a hallucinogen-activated gq-coupled 5-ht2a serotonin receptor. Cell <i>182</i>, 1574-1588.e19. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref8">View at publisher</a></p>
<p id="bib9">Knight, K.M., Ghosh, S., Campbell, S.L., Lefevre, T.J., Olsen, R.H., Smrcka, A.V., Valentin, N.H., Yin, G., Vaidehi, N., and Dohlman, H.G. (2021). A universal allosteric mechanism for G protein activation. Mol. Cell <i>81</i>, 1384-1396.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref9">View at publisher</a></p>
<p id="bib10">Lin, J.Y., Yang, Z., Yang, C., Du, J.X., Yang, F., Cheng, J., Pan, W., Zhang, S.J., Yan, X., Wang, J., et al. (2021). An ionic lock and a hydrophobic zipper mediate the coupling between an insect pheromone receptor BmOR3 and downstream effectors. J. Biol. Chem. <i>297</i>, 101160. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref10">View at publisher</a></p>
<p id="bib11">Lu, S., He, X., Yang, Z., Chai, Z., Zhou, S., Wang, J., Rehman, A.U., Ni, D., Pu, J., Sun, J., et al. (2021). Activation pathway of a G protein-coupled receptor uncovers conformational intermediates as targets for allosteric drug design. Nat. Commun. <i>12</i>, 4721. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref11">View at publisher</a></p>
<p id="bib12">McCorvy, J.D., Wacker, D., Wang, S., Agegnehu, B., Liu, J., Lansu, K., Tribo, A.R., Olsen, R.H., Che, T., Jin, J., et al. (2018). Structural determinants of 5-HT2B receptor activation and biased agonism. Nat. Struct. Mol. Biol. <i>25</i>, 787-796. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref12">View at publisher</a></p>
<p id="bib13">Nagai, Y., Miyakawa, N., Takuwa, H., Hori, Y., Oyama, K., Ji, B., Takahashi, M., Huang, X.P., Slocum, S.T., DiBerto, J.F., et al. (2020). Deschloroclozapine, a potent and selective chemogenetic actuator enables rapid neuronal and behavioral modulations in mice and monkeys. Nat. Neurosci. <i>23</i>, 1157-1167. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref13">View at publisher</a></p>
<p id="bib14">Olsen, R.H.J., DiBerto, J.F., English, J.G., Glaudin, A.M., Krumm, B.E., Slocum, S.T., Che, T., Gavin, A.C., McCorvy, J.D., Roth, B.L., et al. (2020). TRUPATH, an open-source biosensor platform for interrogating the GPCR transducerome. Nat. Chem. Biol. <i>16</i>, 841-849. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref14">View at publisher</a></p>
<p id="bib15">Pryce, K.D., Kang, H.J., Sakloth, F., Liu, Y., Khan, S., Toth, K., Kapoor, A., Nicolais, A., Che, T., Qin, L., et al. (2021). A promising chemical series of positive allosteric modulators of the μ-opioid receptor that enhance the antinociceptive efficacy of opioids but not their adverse effects. Neuropharmacology <i>195</i>, 108673. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref15">View at publisher</a></p>
<p id="bib16">Von Moo, E., Harpsøe, K., Hauser, A.S., Masuho, I., Bräuner-Osborne, H., Gloriam, D.E., and Martemyanov, K.A. (2021). Ligand-directed bias of G protein signaling at the dopamine D2 receptor. Cell Chem. Biol <i>29</i>, 226-238. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref16">View at publisher</a></p>
<p id="bib17">Yan, W., Fan, L., Yu, J., Liu, R., Wang, H., Tan, L., Wang, S., and Cheng, J. (2021). 2-phenylcyclopropylmethylamine derivatives as dopamine d2 receptor partial agonists: design, synthesis, and biological evaluation. J. Med. Chem. <i>64</i>, 17239-17258. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref17">View at publisher</a></p>
<p id="bib18">Yang, F., Guo, L., Li, Y., Wang, G., Wang, J., Zhang, C., Fang, G.X., Chen, X., Liu, L., Yan, X., et al. (2021). Structure, function and pharmacology of human itch receptor complexes. Nature <i>600</i>, 164-169. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00139-3/sref18">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. Bryan Roth (<a href="mailto:bryan_roth@med.unc.edu">bryan_roth@med.unc.edu</a>).</p>
<h4>Materials availability</h4>
<p>TRUPATH plasmids are available from Addgene as a kit (Kit #1000000163; <a href="https://www.addgene.org/kits/roth-trupath/">https://www.addgene.org/kits/roth-trupath/</a>) and individually for use by those associated with academic and non-profit institutions. The kit includes all “optimal” constructs (see <a href="#bib14">Olsen et al., 2020</a>; <a href="#tbl1">Table 1</a>. For heterotrimer compositions). “Non-optimal” constructs are also available under the “Accessory Plasmids” tab on the kit website for users wishing to investigating specific heterotrimeric G protein combinations.</p>
<h4>Data and code availability</h4>
<p>Data displayed in the figures were generated by Exscientia, and are available to non-commercial entities under a confidential disclosure agreement.</p>
<h3>Acknowledgments</h3>
<p>The authors would like to acknowledge Dr. Gabriel Szepesi and his team for assistance with managing and preparing compounds and reagents for this manuscript. We would also like to acknowledge Dr. Patrick Collins and his team for assistance with logistical support in conducting these experiments. Finally, the authors acknowledge the review of the manuscript by Ben Taylor and Dan Ireland and their support in producing this publication. The graphical abstract and <a href="#fig1">Figures 1</a>, <a href="#fig2">2</a>, <a href="#fig3">3</a>, and <a href="#fig5">5</a> were created using <a href="http://BioRender.com">BioRender.com</a>. The TRUPATH platform was developed with support from the Michael Hooker Distinguished Professorship (B.L.R.) and the following grants from the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a>: R37DA035764, U24DK116195, and RO1MH112205 (B.L.R.); KO1MH109943 (Dr. Ryan T. Strachan; not a contributor to this manuscript); and F31NS093917 (R.H.J.O.).</p>
<h3>Author contributions</h3>
<p>J.F.D., K.S., R.H.J.O., and B.L.R. wrote the manuscript. K.S. and R.H.J.O. performed and analyzed the experiments.</p>
<h3>Declaration of interests</h3>
<p>J.F.D., R.H.J.O., and B.L.R. are co-inventors of the TRUPATH platform for which the University of North Carolina at Chapel Hill has filed a patent.</p>
</section>
</article>