<article>
<section>
<h1 id="header">Visualization of naive and learned odor representations using <i>in vivo</i> calcium imaging and immunohistochemical bouton mapping of single <i>Drosophila</i> mushroom body neurons</h1>
<p><time datetime="2020-12-18">Published: December 18, 2020</time></p>
<p>Clare E. Hancock,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> Bart R.H. Geurten,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup> and André Fiala<sup><a href="#aff1">1</a>,<a href="#fn3">5</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Neurobiology of Behaviour, Johann-Friedrich-Blumenbach-Institute for Zoology and Anthropology, University of Goettingen, Julia-Lermontowa-Weg 3, 37077 Goettingen, Germany</p>
<p id="aff2"><sup>2</sup>Department of Cellular Neurobiology, Johann-Friedrich-Blumenbach-Institute for Zoology and Anthropology, University of Goettingen, Julia-Lermontowa-Weg 3, 37077 Goettingen, Germany</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical Contact</p>
<p id="fn3"><sup>5</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:clare.hancock@uni-goettingen.de">clare.hancock@uni-goettingen.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:bgeurte@gwdg.de">bgeurte@gwdg.de</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:afiala@gwdg.de">afiala@gwdg.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100210">10.1016/j.xpro.2020.100210</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol enables the quantification of odor-evoked calcium activity in mushroom body Kenyon cells of the <i>Drosophila melanogaster</i> brain at the single bouton level. We also present subsequent characterization of naive and learned odor representations in the context of olfactory coding. This approach to analyzing the neuronal basis of associative learning provides a substrate for similar studies, perhaps in other animals, to probe the attributes of a neuronal memory trace at the level of synapses distributed across neurons.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Bilz et al. (2020)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Neuronal activity is monitored <i>in vivo</i> at single synaptic bouton level in <i>Drosophila</i>
</li>
<li>Learning-induced modulation of calcium dynamics is visualized</li>
<li>Congruence of odor responses is quantified across distributed synapses</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>To visualize calcium transients in single Kenyon cells, flies must first be generated that express the required transgenes in a limited number of cells. In this study (<a href="#bib2">Bilz et al., 2020</a>), cells were monitored both functionally (<i>in vivo</i>) and anatomically (<i>ex vivo</i>). The former requires the expression of an activity-dependent, genetically encoded calcium indicator (in this case GCaMP3), whilst the latter was aided by the additional inclusion of a strongly expressed fluorescent cell marker (mCherry). Details of the specific fly strains used in this study can be found in the <a href="#key-resources-table">Key resources table</a>. With the exception of the heat-shock protocol required for the induction of gene expression using the MARCM gene expression system, flies were raised under standard laboratory conditions. For a schematic depiction of the preparatory steps described in this section, see <a href="#fig1">Figures 1</a>A–1D.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic depiction of the generation of experimental flies and data acquisition</div>
<p>(A) Parental fly lines that were crossed to generate experimental flies.</p>
<p>(B) F1 generation produced by the parents in (A), collected at the time of egg-laying.</p>
<p>(C) Induction of hs-Flp activity in the larval stage (48 h after egg-laying) by exposure to increased temperature (37°C in a water bath). Induction of hs-Flp-mediated recombination at flippase recognition target (FRT) sites results in stochastic flipping-out of Gal80 in daughter cells.</p>
<p>(D) Potential outcome of the heat-shocking protocol. Daughter cells that express Gal80 (lower panel) do not show transcription of mCherry or GCaMP due to inhibition of the Gal4 transcription factor. Daughter cells in which Gal80 was flipped out (upper panel) show expression of transgenes due to normal functioning of Gal4.</p>
<p>(E) Functional imaging setup for the visualization of odor-evoked calcium transients in the flies generated in (A–D). A single fly is fixed in a custom-built imaging chamber, placed on top of electrical wires and in the path of an airstream such that the fly can be exposed to electric shocks and odor stimuli, respectively.</p>
<p>(F) The head of the fly is opened to expose the brain.</p>
<p>(G) During functional imaging experiments, multiple planes are measured such that odor-evoked Kenyon cell responses can be visualized throughout the neuron(s). Two channels are recorded simultaneously in these experiments – one that detects emission from GCaMP, and another that detects emission from mCherry.</p>
<p>(H) Schematic outline of an example odor delivery protocol (top) in which the fly is exposed to three different odors while recording changes in fluorescence over time (schematically depicted in the lower panel). Use of mCherry as an activity-independent cell marker facilitates clear discrimination between small movement artefacts and changes in intracellular calcium.</p>
<p>(I) Following functional imaging experiments, the brain of the fly is removed and subjected to immunohistochemical staining for the anatomical characterization of measured Kenyon cells.</p>
<p>(J) Bouton that were measured in the previous steps (E–H) can be identified from anatomical confocal scans of the same brain.</p>
<p>(K) Boutons can then be mapped to the different compartments of the mushroom body γ lobe for analysis of odor-evoked responses based on anatomical position along the axon.</p>
</figcaption>
</figure>
<h3 id="sec1.1">Fly strains and husbandry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol>
<li>Raise fruit flies in vials containing standard <i>Drosophila</i> food medium in an incubator at 25°C, relative humidity of 60%, and a controlled 12 h/12 h light/dark cycle.</li>
<li>Collect virgin female flies carrying the following transgenes: neoFRT19A and 5HT1B-Gal4. In parallel, also collect male flies carrying the following transgenes: neoFRT19A, hsFLP, tubP-Gal80, 20xUAS-6xmCherry, 20xUAS-GCaMP3. For the exact genotypes and original sources of the flies used in this study, see <a href="#key-resources-table">Key resources table</a>.</li>
<li>Combine the collected female and male flies in an approximate 2:1 ratio in a fresh food vial and allow for mating for 2–3 days.</li>
</ol>
<h3 id="sec1.2">MARCM technique to generate single Kenyon cell clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼10 days</div>
<p>To achieve the expression of the cell marker (mCherry) and calcium indicator (GCaMP3) in single Kenyon cells, a temporally precise heat-shock must be applied to induce the action of Hsp70-FLP1 on FRT sites. This causes stochastic flipping-out of tubP-Gal80 and allows Gal4-mediated expression of transgenes downstream of UAS sites (<a href="#bib7">Lee and Luo, 2001</a>). To more specifically target γ-type Kenyon cells, here this heat-shock is applied at 48 h after egg laying – the time at which these cells are born (<a href="#bib6">Lee, Lee, and Luo, 1999</a>). For a schematic depiction of this step, see <a href="#fig1">Figure 1</a>.</p>
<ol start="4">
<li>Transfer the fly cross set up in the previous step to a fresh food vial for 4 h to collect eggs.</li>
<li>Allow these offspring to develop for 48 h under normal rearing conditions in an incubator.</li>
<li>48 h after the egg-laying period, transfer the food vial (containing the larval offspring) to a water bath heated to 37°C for 45 min such that the lower 3/4 of the vial is submerged.</li>
<li>Remove the vial from the water bath and return to the incubator for 30 min.</li>
<li>Repeat step 6.</li>
<li>Return the vial to the incubator and allow for development until the adult stage.</li>
<li>Collect the emerging F1 generation daily and keep in standard food vials. Experiments can be conducted on flies aged between 3 and 6 days post-eclosion.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse anti-discs large (DLG)</td>
<td>Developmental Studies Hybridoma Bank</td>
<td>4F3 (RRID:AB_528203)</td>
</tr>
<tr>
<td>Rabbit anti-GFP</td>
<td>Invitrogen</td>
<td>A6455 (RRID:AB_221570)</td>
</tr>
<tr>
<td>Anti-mouse IgG (Alexa Fluor 633)</td>
<td>Invitrogen</td>
<td>A21050 (RRID:AB_2535718)</td>
</tr>
<tr>
<td>Anti-rabbit IgG (Alexa Fluor 488)</td>
<td>Life Technologies</td>
<td>A11034 (RRID:AB_2576217)</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Carl Roth</td>
<td>3051.1</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>Carl Roth</td>
<td>8076.2</td>
</tr>
<tr>
<td>Paraformaldehyde (used at 4% in PBS with 0.1% NaOH)</td>
<td>Carl Roth</td>
<td>03435.1</td>
</tr>
<tr>
<td>Mineral oil</td>
<td>Sigma-Aldrich</td>
<td>M8410</td>
</tr>
<tr>
<td>4-Methylcyclohexanol (MCH)</td>
<td>Sigma-Aldrich</td>
<td>153095</td>
</tr>
<tr>
<td>3-Octanol (3-oct)</td>
<td>Sigma-Aldrich</td>
<td>218405</td>
</tr>
<tr>
<td>1-Octanol (1-oct)</td>
<td>Sigma-Aldrich</td>
<td>297887</td>
</tr>
<tr>
<td>Elastosil RT 601</td>
<td>Wacker Chemie</td>
<td>9102664429</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>All calculated data are deposited in Mendeley Data (<a href="https://data.mendeley.com/datasets/wf7gz3wfr3/1">https://data.mendeley.com/datasets/wf7gz3wfr3/1</a>)</td>
<td>Mendeley Data</td>
<td>https://doi.org/10.17632/wf7gz3wfr3.1</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>Drosophila melanogaster</i> strain y1, w<sup>−</sup>, neoFRT19A; + ; +</td>
<td>Bloomington Stock Centre</td>
<td>1744 (RRID:BDSC_1744)</td>
</tr>
<tr>
<td>
<i>Drosophila melanogaster</i> strain hsFLP, tubP-Gal80, w<sup>−</sup>, neoFRT19A; + ; +</td>
<td>Bloomington Stock Centre</td>
<td>5132 (RRID:BDSC_5132)</td>
</tr>
<tr>
<td>
<i>Drosophila melanogaster</i> strain y, w<sup>−</sup>; 20XUAS-6XmCherry/CyO, Wee-P; Dr[1]/TM6C</td>
<td>Bloomington Stock Centre</td>
<td>52267 (RRID:BDSC_52267)</td>
</tr>
<tr>
<td>
<i>Drosophila melanogaster</i> strain w<sup>−</sup>; + ; 5HT1B-GAL4</td>
<td>Bloomington Stock Centre</td>
<td>27637 (RRID:BDSC_27637)</td>
</tr>
<tr>
<td>
<i>Drosophila melanogaster</i> strain w<sup>−</sup>; + ; 20XUAS-GCaMP3</td>
<td>Bloomington Stock Centre</td>
<td>32237 (RRID:BDSC_32237)</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ/FIJI</td>
<td>National Institutes of Health</td>
<td><a href="https://imagej.net/Fiji">https://imagej.net/Fiji</a></td>
</tr>
<tr>
<td>Zen 2011 SP2</td>
<td>Carl Zeiss AG</td>
<td>N/A</td>
</tr>
<tr>
<td>Leica Application Suite X (LAS)</td>
<td>Leica Microsystems GmbH</td>
<td>N/A</td>
</tr>
<tr>
<td>MATLAB (R2018)</td>
<td>MathWorks</td>
<td>N/A</td>
</tr>
<tr>
<td>Entropy Script (MATLAB File Exchange)</td>
<td>Will Dwinnell</td>
<td><a href="https://de.mathworks.com/matlabcentral/fileexchange/28692-entropy">https://de.mathworks.com/matlabcentral/fileexchange/28692-entropy</a></td>
</tr>
<tr>
<td>ACC Index Scripts (MATLAB)</td>
<td>Bilz et al. (2020; this study)</td>
<td><a href="https://github.com/zerotonin/KCC_KenyonCellCorrelator">https://github.com/zerotonin/KCC_KenyonCellCorrelator</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>LSM 7MP two-photon microscope (functional imaging)</td>
<td>Zeiss (Germany)</td>
<td>N/A</td>
</tr>
<tr>
<td>Two-photon excitation laser</td>
<td>Ti-sapphire laser (Coherent, USA)</td>
<td>N/A</td>
</tr>
<tr>
<td>Confocal laser scanning microscope (anatomical imaging): SP8 confocal laser scanning microscope. equipped with argon laser (488 nm), DPSS laser (561 nm), and HeNe laser (633 nm)</td>
<td>Leica Microsystems (Germany)</td>
<td>N/A</td>
</tr>
<tr>
<td>Microscope slides</td>
<td>Carl Roth GmbH + Co. KG</td>
<td>Cat. No. 0656.1</td>
</tr>
<tr>
<td>Clear adhesive tape</td>
<td>Tesa SE</td>
<td>Cat. No. 56110</td>
</tr>
<tr>
<td>Blue light-curing glue</td>
<td>Kent Express Limited</td>
<td>Cat. No. 953683</td>
</tr>
<tr>
<td>Blue light lamp</td>
<td>mectron Deutschland GmbH</td>
<td>Cat. No. 05100083-001</td>
</tr>
<tr>
<td>Forceps</td>
<td>Fine Science Tools GmbH</td>
<td>Cat. No.11412-11</td>
</tr>
<tr>
<td>Surgical scalpel blade</td>
<td>Swann-Morton</td>
<td>Cat. No. 0303</td>
</tr>
<tr>
<td>Surgical scalpel holder</td>
<td>Swann-Morton</td>
<td>Cat. No. 0907</td>
</tr>
<tr>
<td>Insect pins</td>
<td>Fine Science Tools GmbH</td>
<td>Cat. No. 26002-10</td>
</tr>
<tr>
<td>Concave-convex jaws</td>
<td>Fine Science Tools GmbH</td>
<td>Cat. No. 10053-09</td>
</tr>
<tr>
<td>Custom-built odor delivery system</td>
<td>N/A</td>
<td>N/A (details available on request)</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Ringer’s solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KCl</td>
<td>5 mM</td>
<td>0.3728 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>130 mM</td>
<td>7.597 g</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>·2H<sub>2</sub>O</td>
<td>2 mM</td>
<td>0.406 g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>2 mM</td>
<td>0.294 g</td>
</tr>
<tr>
<td>HEPES</td>
<td>5 mM</td>
<td>1.191 g</td>
</tr>
<tr>
<td>Sucrose</td>
<td>36 mM</td>
<td>12.32 g</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>N/A</td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Ringer’s solution is stored at −20°C before use, then kept refrigerated at 4°C after defrosting for no more than 3 weeks. The solution is adjusted to pH 7.3 using HCl or NaOH.</div>
<table id="undtbl2">
<caption>Phosphate buffered saline (PBS) solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>15 mM</td>
<td>2.07 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>100 mM</td>
<td>5.84 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>85 mM</td>
<td>12.06 g</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>N/A</td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> PBS solution is stored at 4°C for no more than 2 months.</div>
<table id="undtbl3">
<caption>Washing solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton X-100</td>
<td>0.6%</td>
<td>600 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>N/A</td>
<td>99.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Washing solution is stored at 4°C for no more than 4 weeks.</div>
<table id="undtbl4">
<caption>Blocking solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bovine serum albumin</td>
<td>2%</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Washing solution</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Blocking solution is stored at 4°C for no more than 4 weeks.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of fly for imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>In this step, transgenic flies generated in the previous steps are prepared for functional imaging of odor-evoked calcium transients. For an in-depth description of these steps, also see <a href="#bib3">Hancock et al. (2019)</a>. Photographs of the imaging chamber used in this study can be seen in <a href="#fig2">Figure 2</a>.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Custom-built fly chamber used for <i>in vivo</i> functional imaging experiments</div>
<p>(A) The fly chamber is formed of a microscope slide, plastic mesh, and clear adhesive tape and is fitted with two electrical wires that run parallel through the middle.</p>
<p>(B) A small hole is cut in the adhesive tape, leaving a platform to the front on which the head of the fly is later placed.</p>
<p>(C) A fly fixed in the chamber.</p>
<p>(D) The head of the fly glued in position and opened to reveal the brain.</p>
<p>Scale bars in (B) and (C), 1 mm; in (D), 0.5 mm.</p>
</figcaption>
</figure>
<ol>
<li>Select a single transgenic female fly generated in the previous steps and place in an empty plastic vial on ice for 5 min to anaesthetize.</li>
<li>Fix the fly in a custom-built imaging chamber using clear adhesive tape. The chamber should comprise of a narrow channel (with approximate width of 1 mm) into which the fly can be placed, ventral side down, with the head supported and stabilized to reduce movement. Placed in front of the head should be a channel through which an air stream can be delivered during the experiment. Under the fly should be two parallel electrical wires through which electric shocks can be delivered to the thorax during the training phase of the protocol.</li>
<li>Cut a hole in the adhesive tape using a surgical scalpel blade to expose only the head of the fly and keeping the rest of the body covered.</li>
<li>Using blue light-curing glue, fix the head on the back and sides to limit its movement. In this study, an insect pin held by concave-convex jaws was used to manipulate the glue around the head. Leave the dorsal surface of the head free for opening. Ensure there are no gaps through which the Ringer’s solution (applied in the next step) could leak.</li>
<li>Apply a drop of room temperature (19°C–21°C) Ringer’s solution on the exposed head cuticle and open the head capsule using a fine bladed knife.</li>
<li>Using fine forceps, remove the cuticle and any excess tissue that lies on top of the brain.</li>
</ol>
<h3 id="sec3.2">
<i>In vivo</i> imaging of odor-evoked calcium transients</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min to 1 h</div>
<p>In this step, the flies prepared previously are imaged using two-photon microscopy to visualize odor-evoked calcium transients in single Kenyon cells before, during, and after an olfactory associative conditioning protocol conducted <i>in vivo</i> under the microscope. Photographs of the setup used in this study can be seen in <a href="#fig3">Figure 3</a>, and schematic summary of the imaging protocol can be seen in <a href="#fig1">Figures 1</a>E–1H.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Microscope setup for <i>in vivo</i> functional imaging</div>
<p>The fly chamber (A) shown in <a href="#fig2">Figure 2</a> is placed under the microscope objective and is connected to electrical wires (D) and a needle (B) that connects the chamber to the odor delivery system (C). The odor delivery machine holds eight different odor bottles, from which odors are carried by a continuous airstream by the timely controlled opening and closing of valves at their opening.</p>
</figcaption>
</figure>
<ol start="7">
<li>Prepare the odor mixtures. In this study, MCH, 3-octanol, and 1-octanol were used at concentrations of 1:750, 1:500, and 1:400, respectively, in mineral oil. Install odors in the odor delivery system.</li>
<li>Place the prepared fly under a microscope equipped for two-photon imaging. For details of the setup used in this study, see <a href="#fig2">Figures 2</a> and <a href="#fig3">3</a> and the <a href="#key-resources-table">Key resources table</a>.</li>
<li>Illuminate the preparation using a red filter set. Through the eyepieces, observe the fly brain for expression of mCherry in Kenyon cells.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> due to the stochastic nature of the MARCM expression system, a variety of outcomes are possible - ranging from single cell expression to expression in dozens of cells. In order to later identify single cells, only flies in which a maximum of 3 Kenyon cells per hemisphere are labelled can feasibly be utilized in these experiments.</div>
<ol start="10">
<li>If the correct number of cells are labelled, position the fly head in the center of the field of view for imaging, ensuring the odor delivery system is in place and the electrical wires have been connected to a power source.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> due to the sensitivity of detector systems used in this setup, limit external light sources as much as possible.</div>
<ol start="11">
<li>Start the microscope control software and initialize image acquisition settings. To visualize both red and green fluorophores, an excitation wavelength of 940 nm and a customized filter set with a 605 nm beamsplitter and two bandpass filters of ∼620–680 nm for mCherry detection, and ∼500–550 nm for GFP detection is recommended. For specific software and equipment used in this study, see <a href="#key-resources-table">Key Resources Table</a>.</li>
<li>Initiate image acquisition and locate the labelled Kenyon cells. Crop and rotate the acquisition region such that the Kenyon cells are positioned in the center of the field of view.</li>
<li>Through multiple z-planes, run a synchronized image acquisition and odor presentation sequence whereby, in each imaging plane, all of the odors and mineral oil are presented to the fly via air stream. Save these image sequences for later analysis.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this study, the airstream flow rate was 1 mL/s.</div>
<div class="note">
<span class="note-title">Note:</span> Dependent on the number of cells and their anatomical complexity, the number of imaging planes required to capture maximal axonal branches can vary. In this study, generally between 4 and 7 planes were imaged per brain.</div>
<div class="note">
<span class="note-title">Note:</span> In this study, a framerate of 4 Hz was used.</div>
<ol start="14">
<li>Run the associative conditioning protocol. In this protocol, present one of the odors continuously for 60 s with overlapping pulsing electric shocks (this is now the CS+ odor). After a 60 s break, present a second of the odors for 60 s with no electric shocks (now the CS- odor). For control groups, either omit the electric shock (CS-only control) or deliver the electric shock 2 min before the onset of the first odor (unpaired control).</li>
<li>For post-training data, repeat the odor presentation protocol as in step 13. Ensure that measurements are conducted in the same imaging planes as before the conditioning, so that odor-evoked responses can be compared in the same boutons.</li>
<li>When all planes have been measured, capture a z-stack which covers the entire depth of the brain to later refer to when identifying and mapping cells/boutons.</li>
<li>Remove the imaging chamber from the microscope and, with the fly intact, proceed to the next section.</li>
</ol>
<h3 id="sec3.3">Brain explant and immunohistochemistry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼2.5 days</div>
<p>In this step, the brain of the fly imaged in the previous step is removed for immunohistochemical staining. This will facilitate later confocal microscopic scanning of the brain to produce high resolution images from which the functionally imaged Kenyon cells can be fully anatomically characterized.</p>
<ol start="18">
<li>Carefully remove the fly from the imaging chamber and transfer to a dissection dish containing a black-dyed silicon mixture (Elastosil-based, see <a href="#key-resources-table">Key Resources Table</a>).</li>
<li>Fix the fly in place, ventral side up, using insect pins through the thorax and abdomen and cover with 2–3 drops of ice-cold Ringer’s solution.</li>
<li>Using fine forceps, carefully remove the proboscis.</li>
<li>Pull apart the two sides of the head capsule until they are completely removed from the brain. Clear away the residual tissues from the brain.</li>
<li>Detach the brain from the thorax, leaving behind the ventral nerve cord.</li>
<li>Place the brain in fixative solution (4% paraformaldehyde in PBS) for 2 h on a shaker at 4°C.</li>
<li>Wash the brain 3 times for 20 min each in PBS-T washing solution (PBS with 0.6% Triton X-100) at room temperature (19°C–21°C) on a shaker.</li>
<li>Place the brain in blocking solution (2% bovine serum albumin in PBS-T) for 2 h on a shaker at room temperature (19°C–21°C).</li>
<li>Transfer the brain to a dish containing a primary antibody mix containing antibodies against GFP (which will bind GCaMP in Kenyon cells) and discs-large (DLG; which will bind the pan-neuronal synaptic protein, discs-large) and incubate on a shaker at 4°C for 12–16 h. In this study, we used rabbit anti-GFP (Invitrogen) at a concentration of 1:2000 and mouse anti-DLG (Developmental Studies Hybridoma Bank) at a concentration of 1:200 in blocking solution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> mCherry autofluorescence is in most cases robust enough to not require additional fluorophore binding, so no mCherry antibody is used in this study.</div>
<ol start="27">
<li>Wash the brain as in step 24.</li>
<li>Transfer the brains to a dish containing a secondary antibody mix. All secondary antibodies were used at a concentration of 1:300 in blocking solution. In this study, we used anti-rabbit Alexa Fluor 488 and anti-mouse Alexa Fluor 633. For further details about the antibodies used, see <a href="#key-resources-table">Key Resources Table</a>. Incubate on a shaker at 4°C for 12–16 h.</li>
<li>Wash the brain as in step 24.</li>
<li>Mount the brain for confocal scanning. Inside a 13 mm tape ring, pipette approximately 5 μL of VectaShield and carefully place the brain in the liquid, taking care to orient the brain so that it lays flat with the anterior of the brain facing upwards for easier scanning.</li>
<li>Scan the brain using a confocal laser scanning microscope. Laser and detector settings should be chosen according to the secondary antibodies used. To maximize image quality, use an immersion lens and a frame size of at least 1024 × 1024 pixels. Brains were scanned sequentially, such that GFP and DLG channels were captured simultaneously, followed by an additional scan for mCherry. For the exact hardware and software used in this study, see <a href="#key-resources-table">Key Resources Table</a>.</li>
</ol>
<h3 id="sec3.4">Functional imaging analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1–1.5 h</div>
<p>In this step, the images acquired during the functional calcium imaging step are processed and analyzed to quantify odor-evoked activity in individual synaptic boutons.</p>
<ol start="32">
<li>Install and open an image processing program. In this study, ImageJ was used for all image processing steps.</li>
<li>Either individually or using a custom-written batch-processor, correct each image series for any movement in the X and Y dimensions using an image registration plug-in such as TurboReg (<a href="#bib9">Thévenaz et al., 1998</a>).</li>
<li>In each imaging plane, pre- and post-training, mark regions of interest (ROIs) encompassing individual synaptic boutons along the Kenyon cell axons. Boutons can be identified by their spherical shape and their greater size relative to the axonal diameter.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> the number of identifiable boutons for a given Kenyon cell can vary widely, in this study ranging from 29–158 boutons. To investigate the effects of training on odor representations, only boutons that are identifiable in both the pre- and post-training recordings can be used in this analysis.</div>
<ol start="35">
<li>For each bouton, use the “Multi Measure” function in ImageJ to extract fluorescence intensity values over time for both the green (GCaMP) and red (mCherry) channels of each recording.</li>
<li>From the raw fluorescence values, calculate the ΔF/F<sub>0</sub> values for each channel.
<ol type="a">
<li>First, generate the F<sub>0</sub> value by calculating the average fluorescence values from the 5 s preceding odor onset.</li>
<li>Second, generate the ΔF values by subtracting the F<sub>0</sub> value from the raw fluorescence.</li>
<li>Finally, divide the ΔF value at each frame by F<sub>0</sub> to generate a ΔF/F<sub>0</sub> trace over time.</li>
</ol>
</li>
<li>To temporally filter the ΔF/F<sub>0</sub> trace, apply a 3-frame moving average through the time series.</li>
<li>To reduce the presence of ΔF/F<sub>0</sub> fluctuations that are due to slight movements in the preparation rather than calcium fluctuations, subtract the mCherry ΔF/F<sub>0</sub> from the GCaMP ΔF/F<sub>0</sub>.</li>
</ol>
<h3 id="sec3.5">Kenyon cell anatomy and bouton mapping</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>In this final image analysis step, boutons that have been functionally analyzed are allocated to a Kenyon cell (in the case of a brain in which multiple cells were labelled) and assigned to the correct compartment of the mushroom body γ-lobe. This facilitates the analysis of bouton activity in the context of their anatomical position along the Kenyon cell axon. A schematic depiction of this process can be seen in <a href="#fig1">Figures 1</a>J and 1K.</p>
<ol start="39">
<li>In ImageJ, open the confocal image stacks of the immunostained brain.</li>
<li>Using the mCherry and GFP image stacks, track the Kenyon cell(s) through the image stacks and generate a 3-dimensional projection view of the Kenyon cell(s). This projection will aid in the translation of the bouton positions from the functional imaging view (dorsal) to the anatomical imaging view (anterior).</li>
<li>If necessary, separate the distinct Kenyon cells to form separate projections for each cell.</li>
<li>Using the z-stack generated during functional imaging, map the identified boutons onto the anatomical images. Boutons for which positions cannot be translated should not be analyzed.</li>
<li>Using the anti-DLG neuropil staining, identify the 5 compartments of the mushroom body γ-lobe, and note the compartment location of each of the measured boutons.</li>
</ol>
<h3 id="sec3.6">Calculating the amplitude corrected correlation (ACC)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>The overall aim of the analysis described in this step is to quantify single bouton responses before and after learning. This is achieved through two different analysis protocols - the Bouton Response Class analysis (BRC) and the Amplitude Corrected Correlation (ACC) (see <a href="#fig4">Figure 4</a>). Both protocols analyze the bouton responses, with regard to the individualization of each bouton, which is in contrast to the canonical approach, in which the comprehensive cell response is quantified (e.g., in the form of calcium imaging or membrane potential). We developed the ACC to quantify the congruence of Kenyon cell synaptic boutons within the same cell. This congruence reflects both the temporal structure of the calcium responses and their amplitude.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Workflow of the analysis pipeline</div>
<p>Different shapes in the flow diagram are illustrated in the upper right corner, within the dashed gray line. Rectangles with rounded corners denote the start or end of the workflow. Rectangles with sharp corners indicate routine calls. Parallelograms highlight data structures, whereas diamond shapes indicate decisions by the user. The different boxes are color coded for the different programming environments (light blue, ImageJ/Fiji; green, table calculations; orange, MATLAB). The workflow starts with the dynamics of calcium-dependent fluorescence changes in single boutons and their compartment location. These data are then transferred into xlsx or csv files. The KCC loading routines then make those data available for two different analysis options: the Amplitude Corrected Correlation (ACC) index or the Bouton Response Class (BRC) analysis. The analysis pipeline ends in data structures or plots.</p>
</figcaption>
</figure>
<p>The former can be directly quantified by the cross-correlation between both calcium responses. The correlation is further normalized by its auto correlation coefficients, limiting the numerical range of the cross-correlation from 0 (not correlated) to 1 (identical temporal structure).where the time-series of the calcium responses are represented by either Cl<sub>1</sub> or Cl<sub>2</sub>. The time-point is demarcated by t and the phase lag of the correlation by τ.</p>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Si1.jpg" alt="Si1.jpg"></p>
<p>To address the difference in response amplitude, the Michelson contrast of the two amplitudes is calculated as: <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Si2.jpg" alt="Si2.jpg">, where A<sub>x</sub> is the respective maximum amplitude of each calcium response. Thereby, the contrast value becomes 1 if there is no difference between both responses and approaches 0 if they are very dissimilar.</p>
<p>The resulting ACC is sensitive to congruent responses, meaning that it correctly detects similar response pairs. However, traces that lack any response will also render an ACC close to 1. To remedy this, a third term is introduced that calculates the mean response of two boutons, normalized by the maximum response of the neuron:</p>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Si3.jpg" alt="Si3.jpg"></p>
<p>A<sub>1</sub> and A<sub>2</sub> are again the maximum amplitude of the responses and n represents the entirety of all responses. Thereby, flat responses are set to 0, while robust responses will render values close to 1.</p>
<p>Now each of the three measures (cross correlation, amplitude contrast, and normalized mean amplitude) are within the limits of 0 to 1 and can be multiplied to derive a sensitive and specific measure to quantify the congruence in the responses of two synaptic boutons as follows:</p>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Si4.jpg" alt="Si4.jpg"></p>
<p>For the MATLAB (MathWorks, USA) implementation of the aforementioned formulas, as used in this study, see <a href="#key-resources-table">Key Resources Table</a>. In the following steps, an example function call is given that can be tested with the data deposited at <a href="https://data.mendeley.com/datasets/wf7gz3wfr3/1">https://data.mendeley.com/datasets/wf7gz3wfr3/1</a>.</p>
<ol start="44">
<li>Download the scripts from <a href="https://github.com/zerotonin/KCC_KenyonCellCorrelator/">https://github.com/zerotonin/KCC_KenyonCellCorrelator/</a> and the sample data from <a href="https://data.mendeley.com/datasets/wf7gz3wfr3/1">https://data.mendeley.com/datasets/wf7gz3wfr3/1</a>. For these scripts to run smoothly, data files must follow a strict format, as described below.
<ol type="a">
<li>Response data for each cell is saved into one xlsx file. The file consists of two sheets in which sheet 1 contains the pretraining responses and sheet 2 the post-training responses.</li>
</ol>
</li>
</ol>
<p>Each sheet includes 4 tables of m rows and n columns. The columns hold the information for each identified bouton and each row reflects the sample time. The first row of the sheet shows in which γ-lobe compartment the bouton is situated. The first column identifies which odor was used for stimulation.</p>
<ol start="45">
<li>In this step, an xlsx file is read and transcribed it into a 4-dimensional matrix. The input variables are defined as follows: <b>fpos</b> is a string with the file position of your xlsx file. <b>samples</b> describe the length of each recording. <b>gap</b> holds the amount of empty rows between different odor presentations in the xlsx file. The resulting variable is the 4 dimensional <b>data m</b> × <b>n</b> × <b>p</b> × 2 matrix, where <b>m</b> is the number of frames recorded, <b>n</b> the number of boutons, <b>p</b> the number of presented odors, and the fourth dimension holds the pre-/post-training condition. The second return variable is a vector of <b>n</b>-length with the γ-lobe compartment identification (i.e., γ2, γ3, γ4, or γ5).<br> <i>%% load file</i><br> % set path to your xlsx file<br> fPos = '/path2/Controls/A_170320_3003_0304_fly1_cell1_pre_post.xlsx';<br>     samples = 85;<br>    <i>% number of frames acquired during Ca2+ imaging</i>gap = 3;<br>    <i>% empty rows between each of the odor presentations</i><br> <i>% function call to read in xlsx example data file</i><br> [data,ylobesIDX] = KCC_fIO_loadXLSX(fPos,samples,gap);</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> If data are already in MATLAB format, steps 44 and 45 can be omitted. In this case, the data should be arranged in a 4 dimensional matrix (m × n × p × 2) where <b>m</b> is the number of frames recorded during the experiment, <b>n</b> is the number of boutons identified in the cell and <b>p</b> is the number of presented odors. The fourth dimension represents the pretraining recording and then the post-training recording.</div>
<ol start="46">
<li>In the next step, the cross-correlation for all boutons against all other boutons is calculated. To do so, use the data variable created in the last step and define the following inputs: <b>corrWin</b> is a 2 entry integer vector holding the first and the last frame of the odor presentation. <b>fps</b> is the frame rate of the recording. <b>maxPhaseShift</b> is the maximum temporal deviation between two calcium traces allowed during the cross-correlation analysis. The function call “KCC_fbf_xcorr” will result in another 4D matrix (<b>m</b> × <b>m</b> × 4 × 2) holding the correlation coefficients normalized by their auto correlation such that these values will be scaled from 0 (no correlation) to 1 (identity). Here, <b>m</b> is the number of boutons, the third dimension holds odor identity, and the fourth dimension is pre-/post-training. The diagonal of the <b>m</b> × <b>m</b> slides of the cross-correlation matrix are NaN (not a number) values, as these represent identity and are therefore not explicitly calculated. Function call for this step:<br> <i>%% calculate the cross correlation between all boutons%data is the output variable of KCC_fIO_loadXLSX</i>corrWin = [4 13]; <i>% correlation window</i><br> fps = 4; <i>% frames per second</i><br> maxPhaseShift = 5; <i>% maximum of temporal deviation between both boutons</i><br> %function call to calculate cross correlation<br> corrMat = KCC_fbf_xcorr(data,corrWin,fps,maxPhaseShift);</li>
<li>Next, prepare the amplitude data using the <b>KCC_fbf_meanAmps</b> function to then combine everything in Step 48. This function call gets inputs from the previously run functions, and results in 3 new variables: <b>dataM</b> is a <b>m</b> × 4 × 4 × 2 matrix holding the median response for all boutons of each γ-lobe compartment, where <b>m</b> is the number of frames recorded. The second dimension is the four γ-lobe compartments, the third represents the four odors, and the fourth is the pre-/post-training condition. <b>amps</b> is a 4 × 4 × 2 matrix in which the first dimension represents the γ-lobe compartment, the second dimension the odor, and the third the pre-/post-training condition.<br> <i>%% calculate the correction factors</i><br> <i>% calculate the mean response</i><br> [dataM,amps,ampsN] = KCC_fbf_meanAmps(data,ylobesIDX,corrWin,fps);</li>
<li>Finally, to calculate the ACC index, run the <b>KCC_fbf_lobeSimilarityCorr</b> function. The resulting variables are the <b>ACC</b> and <b>lobesCorr. lobesCorr</b> is a <b>m</b> × <b>m</b> × 4 × 2 matrix, where <b>m</b> is the number of γ-lobe compartments. Each entry is the median of all bouton-to-bouton correlations specific for a compartment, an odor, and the pre-/post-training condition. The odors are represented in the third dimension and the pre-/post-training condition is represented in the fourth dimension. The <b>ACC</b> is identical to the <b>lobesCorr</b> variable, but is multiplied with the amplitude contrast and relative amplitude strength, as below:<br> <i>%% calculate ACC index</i><br> [ACC,lobesCorr] = KCC_fbf_lobeSimilarityCorr(corrMat,ylobesIDX,amps,ampsN);</li>
<li>To calculate the pre- to post-training differences in ACC indices, use the function “KCC_fbf_lobeSimilarityDiff”, which will carry out a simple subtraction. The resulting 3D matrix contains the difference in ACC as a m × m × 4 matrix where m is the number of γ-lobe compartments and the third dimension represents the four different odors presented in the experiment.<br> <i>%% calculating the pre post difference of the ACC</i><br> accDiff = KCC_fbf_lobeSimilarityDiff(ACC);</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> If only pre- or post-training data are of interest (and not the comparison between them), step 49 can be omitted.</div>
<ol start="50">
<li>Save the data into a MATLAB data file:<br> <i>%% save result</i><br> <i>%set file position where data should be saved</i><br> savePos = '/folder2/saveDestination.mat';<br> <i>% saving to disk</i><br> save(savePos,'ACC','accDiff','corrMat','dataM','data',...<br> 'ylobesIDX','amps','ampsN','lobesCorr');</li>
<li>Through steps 49 and 50, the primary analysis for the experiment is completed. To plot the results of these steps, use the following functions:<br> <i>%% plotting</i><br> <i>% bouton-to-bouton cross-correlation</i><br> KCC_plot_plotBoutonCorrelation(1,corrMat,ylobesIDX);<br> <i>% mean bouton responses by compartment (see</i> <a href="#bib2">Bilz et al. (2020)</a>, <a href="#fig3">Figure 3</a>A<i>)</i><br> KCC_plot_deltaFbyFmeans([2,3],data,dataM,ylobesIDX);<br> <i>% ACC indices (see</i> <a href="#bib2">Bilz et al. (2020)</a><i>, Figure 6B, left and center plots)</i><br> KCC_plot_plotLobeSimilarityCorrelation([4 5],lobesCorr,ACC);<br> <i>% calculate pre- to post-training ACC differences (see</i> <a href="#bib2">Bilz et al. (2020)</a><i>, Figure 6B rightmost plots)</i><br> KCC_plot_plotLobeSimDiff(6,accDiff);</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The full MATLAB script used in this study is available by following the link in the <a href="#key-resources-table">Key Resources Table</a>.</div>
<h3 id="sec3.7">ACC analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>In the previous step, single synaptic bouton responses were quantified in the form of the ACC. Now, these results are used to acquire information about the responses of all boutons of a single neuron.</p>
<p>In this step, the resulting data from the ACC calculation is combined to draw conclusions about the congruity of bouton responses and, more specifically, how this is modulated through the associative conditioning paradigm applied during the experiment. To this end, it is necessary to build so-called “collections”. That is, compose a collection of a subset of the data grouped according to the different experimental conditions (e.g., the different training conditions used in this study). This collection can then be analyzed further, and the results can be seen in <a href="#bib2">Bilz et al. (2020)</a>, Figure 6.</p>
<ol start="52">
<li>Compose the data collection by choosing the datasets that fit all criteria. The data collection variable should be the compiled <b>dataM</b> variables for the specific odor presentation (e.g., CS+, CS-, CS-only). This would in theory result in a 5 dimensional <b>m</b> × <b>m</b> × 1 × 2 × <b>p</b> matrix, where <b>m</b> represents the number of γ-lobe compartments, the third dimension is the chosen stimulus odor, the fourth dimension represents the pre-/post-training condition, and <b>p</b> represents the different cells. As the third dimension now became a singleton dimension we can eliminate it by using the <b>squeeze</b> function in MATLAB. The 5D matrix (<b>m</b> × <b>m</b> × 1 × 2 × <b>p</b>) thereby becomes 4D (<b>m</b> × <b>m</b> × 2 × <b>p</b>), and here we call this variable “collection”.<br> <i>%% reducing the collection dimensionality</i><br> collection = squeeze(collection5Dl);</li>
<li>Using the collection variable, calculate the median pretraining condition as follows:<br> <i>%% calculating median pretraining ACC indices</i><br> medPre = median(collection(:,:,1,:),4)</li>
<li>Now, the difference between the pre- and post-training conditions can be calculated using the <b>bsxfun</b> function and, again, using the squeeze function, this matrix can be reduced from four dimensions (<b>m</b> × <b>m</b> × 1 × <b>p</b>) to three (<b>m</b> × <b>m</b> × <b>p</b>), where <b>m</b> is the number of γ-lobe compartments and <b>p</b> is the number of cells.<br> <i>%% calculating pre-/post-training difference</i><br> diffData = bsxfun(@minus,collection(:,:,2,:),medPre);<br> diffData = squeeze(diffData);</li>
<li>In this step, the median response difference between pre- and post-training is determined and tested for statistical significance. In this study, we used a non-parametric confidence interval (CI). Using this test, we ask whether there is a statistically significant change (either increase of decrease) in the ACC indices before and after the different training protocols implemented in the experiment. The function definition used in this study is largely based on standard functions of MATLAB and is used as below:<br> <i>%% calculating the median differences and the corresponding 95% confidence interval</i><br> medDiff = median(diffData,3);<br> <i>% lo represents the lower confidence interval and hi the higher confidence interval</i><br> [lo,hi]=confintND(diffData,3);</li>
</ol>
<p>If lo and hi are both below zero, there is 95% confidence that there is a decrease in the bouton response congruity. If both are above zero there is 95% confidence that there is an increase in the bouton response congruity. In both cases, the null hypothesis that the congruity does not change from pre- to post-training condition can be rejected with a 5% reversal probability.</p>
<h3 id="sec3.8">Bouton response classes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>In this step, bouton responses are categorized into distinct classes, so-called Bouton Response Classes (BRCs). Each class then represents a distinguishable response signal. Thereby we can reduce the complexity of the responses from a continuous signal with complex response properties (amplitude, latency, etc.) to a single class. This can then be used to examine which classes are prevalent under different conditions (e.g., pretraining, CS+ post-training, CS- post-training, etc.).</p>
<p>The calculation of Bouton Response Classes (BRC) is an augmenting analysis to the ACC. Where the ACC aims to check if the response of the boutons becomes more or less similar in respect to the training, the BRC approach aims to describe the different response types of boutons. Using a similar collection as in the ACC analysis, we can use hierarchical agglomerative clustering to identify and analyze recurring odor response types (so-called BRCs) observed across Kenyon cell boutons.</p>
<p>The first step in this analysis is feature selection. Features are parameters of the data that can be quantified and are characteristic of the data set to be analyzed. In this study, we used the peak amplitude (ΔF/F<sub>0</sub>) during the stimulus presentation and the latency to the peak (s).</p>
<p>Agglomerative hierarchical clustering initially treats every sample of the data set (i.e., each bouton response) as one cluster. The clustering algorithm then calculates the distances between data samples and merges them into progressively fewer, larger clusters. In this study, we used simple Euclidean distance. We also used Ward’s criterion (<a href="#bib8">Murtagh and Contreras, 2017</a>) to select which samples are merged first, such that minimal within-cluster variance is prioritized.</p>
<p>The result of such a clustering algorithm is the assignment of each sample to one cluster, in this case assignment of each observed bouton response to a BRC. Once the BRCs are calculated, their median response, frequency of occurrence and, in this case, how these parameters change from pre- to post-training condition, can be calculated.</p>
<ol start="56">
<li>Select all cells of your collection by the appropriate criteria (e.g., training condition).</li>
<li>Load the <b>data</b>, <b>ylobesIDX</b>, and <b>fps</b> variable for each of your cells. <b>data</b> is a <b>m</b> × <b>n</b> × <b>p</b> × 2 matrix, where <b>m</b> is the number of frames recorded, <b>n</b> the number of boutons, <b>p</b> the number of presented odors, and the fourth dimension holds the pre-/posttraining condition. <b>ylobesIDX</b> is a vector of <b>n</b>-length with the γ-lobe compartment identification (γ2, γ3, γ4, or γ5). Select the odors that are of interest. <b>fps</b> is the framerate used when acquiring the calcium traces.</li>
<li>For the pretraining condition of each bouton, calculate the maximum amplitude (<b>amp</b>) and the latency (<b>ampT</b>) to that amplitude. Given that a single response is saved in the vector <b>boutResp</b>, this can be done as follows:<br> <i>%% calculate bouton response features</i><br> <i>% find all peaks inside the stimulus presentation</i><br> [ampHeights,frames]=findpeaks(boutResp(startOfStimPresentation:endOfStimPresentation));<br> <i>% find the highest peak</i>[amp,ampT] =max(ampHeights);<br> <i>% find the original frame of the peak and divide by the fps to calculate it in seconds</i><br> ampT = frames(ampT)/fps;</li>
<li>Store the results from step 58 in one m × 2 matrix (<b>clustData</b>) where m is the number of all boutons involved in this collection, and the first column holds the <b>amp</b> values and the second column the <b>ampT</b> values.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> it is important to be able to associate your sample consisting of these features with the cell, γ-lobe compartment, and other critical information for the later analysis.</div>
<ol start="60">
<li>As the peak and the latency are measured in different units and widely differ numerically, in this step they are normalized to a mean of zero and a standard deviation of one – a process called z-scoring (<a href="#bib5">Kreyszig, 1979</a>). Z-score <b>clustData</b> via MATLAB’s built-in zscore function.<br> <i>%% z-scoring</i><br> clustDataZS = zscore(clustData);</li>
<li>This data can now be clustered via MATLAB’s hierarchical clustering algorithms, and a dendrogram and the derivative of the clustering costs over the number of clusters can be plotted (see <a href="#bib2">Bilz et al. (2020)</a>, Figure 7A). The dendrogram shows each data sample on the x axis and the costs of merging on the y axis. The relationship between merger cost and cluster number is directly demonstrated by the second plot generated in this step – for which the derivative of the cost is normalized to the sum of the costs.<br> <i>%% clustering</i><br> <i>% agglomerative hierarchical clustering with Euclidean distance and Ward’s criterion</i><br> Z = linkage(clustDataZS,'ward','euclidean');<br> <i>% plotting a dendrogram with as many leaves as data samples</i><br> dendrogram(Z,size(clustDataZS,1))<br> <i>% calculating the costs relative to number of clusters</i><br> cost = flipud(diff(Z(:,3)));<br> cost = cost./sum(cost);<br> <i>% plotting the costs - from 2 to length+1, as the derivative is n-1</i><br> plot(2:length(cost)+1,cost)<br> set(gca,'XScale','log')</li>
<li>Based on the two graphs generated in step 61, one can identify the optimal number of clusters to utilize based on the increasing cost relative to sample mergers. This increase identifies to the user the number of clusters that can be formed before the cost of merges - due to the sudden merging of very distinct samples – rapidly rises. We refer to this number of clusters as <b>kMax.</b> In this study, the kMax of the dataset was 4. By using the cluster function of MATLAB, determine the assignment of the samples to the different clusters (BRCs) in the <b>IDX</b> variable:<br> <i>%% get cluster assignment</i><br> IDX = cluster(Z,'maxclust',kMax);</li>
<li>Based on the IDX, calculate the mean response and the centroid of each BRC by running the following commands:<br> <i>% get centroids</i><br> for kI =1:kMax<br>    meanResp(kI,:) = mean(boutonResponses(IDX==kI,:),1);<br>    centroid(kI,:) = mean(clustData(IDX==kI,:),1);<br> end</li>
<li>Combine the <b>IDX</b> variable with the <b>ylobesIDX</b> variable to calculate how often a given BRC is found in a specific γ-lobe compartment. As the compartment identifiers in our case range from 2 to 5 and the BRC identifiers range from 1 to 4, this data can be saved in a 4 × 4 matrix (<b>BRC_occ</b>):<br> <i>%% calculating the BRC occurrence per γ-lobe compartment</i><br> <i>% go through compartment identities</i><br> for yI =2:5<br>    <i>% go through BRC classes</i><br>    for brcI =1:4<br>    <i>% sum up the combined logical indices</i><br>       BRC_occ(brcI,yI-1) = sum(ylobesIDX == yI &amp;&amp; IDX==brcI);<br> end<br> end</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to keep the different data descriptors aligned. The assignment of BRCs is only relevant if you can associate it with the correct cell, treatment, γ-lobe compartment, etc.</div>
<h3 id="sec3.9">Calculating bouton entropy</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>In the previous step, odor response data were transformed from numerical to categorical data by assignment into BRCs. This facilitates the application of a more direct calculation of the variance of the neuronal activity that was observed in this study.</p>
<p>To do this, we used the Shannon entropy (Shannon, 1949) as a measure for variance (<a href="#bib1">Allaj, 2018</a>). As entropy is measured in bits, we can directly see how many categories (BRCs, in this study) are needed to describe the responses in a γ-lobe (see <a href="#bib2">Bilz et al. (2020)</a> for more on this principle).</p>
<p>In this study, we used a MATLAB script created by Will Dwinnel (see <a href="#key-resources-table">Key Resources Table</a>) implemented as below.</p>
<ol start="65">
<li>Choose a BRC <b>IDX</b> variable for the training condition and γ-lobe compartment of interest.</li>
<li>Run the Entropy function as below:<br> <i>% calculate entropy for γ-lobe</i><br> entropyRes = Entropy(IDX(ylobesIDX==2));</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Using the method described here, we were able to generate transgenic flies that showed expression of our genetically encoded calcium indicator and cell marker in 1–3 Kenyon cells with a success rate of approximately 50%. The remaining flies generated showed expression in 4 or more cells, or, in some cases, no cells were labelled at all. The protocol shown here was attuned to achieve the optimal cell number for these experiments, and deviations from the heat-shock protocol outlined here would likely cause this success rate to fall. An example of a fly in which two cells were labelled and distinguished can be seen in <a href="#fig5">Figure 5</a>.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/326-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Example images obtained in this study</div>
<p>(A–D) Four example functional imaging planes show the axonal branches of the two labelled Kenyon cells.</p>
<p>(E) After immunohistochemistry, a confocal image series is used to disentangle the branches and identify individual cells, as well as mushroom body compartments. Here, a maximum projection is shown. All scale bars, 20 μm.</p>
</figcaption>
</figure>
<p>Odor representations in the mushroom body are encoded very sparsely, meaning for each given odor only a very small proportion of the cell population is activated. In this study, we found that approximately 8%, 10%, 4%, and 2% of measured cells responded to MCH, 3-octanol, 1-octanol, and the diluent mineral oil, respectively (<a href="#bib2">Bilz et al., 2020</a>). This is in the range to be expected, based on previous studies (<a href="#bib4">Honegger, Campbell, and Turner, 2011</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">Data inclusion/exclusion criteria</h3>
<p>This protocol and analysis are based on cells that exhibit responses to the olfactory stimuli used. Therefore, non-responsive cells are excluded from the analysis. Additionally, boutons must be excluded from the pre- to post-training comparison if they are not clearly identifiable in both recordings or they cannot be assigned accurately to a single cell or a mushroom body γ-lobe compartment.</p>
<p>Similarly, if excessive movement of the fly brain during odor response measurements causes boutons to leave the focal plane such that their fluorescent intensities could not be accurately and reliably captured throughout, those recordings must also be discarded.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>In order to visualize calcium transients at the level of the presynaptic boutons and attribute those boutons to specific cells and mushroom body compartments, only 1–3 Kenyon cells can be imaged per fly. With the approximately 50% success rate of the MARCM technique in generating such flies and the additional limitation of only approximately 7% of Kenyon cells responding to a given odor, many trials are required in order to generate a robust sample size.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No (or few) odor responses are detectable during calcium imaging step (step 13).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>If the percentage of imaged Kenyon cells that show odor-evoked responses is less than approximately 5%–8%, ensure that the fly has not died during the preparation or imaging procedure and that the brain is still intact. Also make sure that no glue or Ringer’s solution is covering the antennae and that odor delivery is not otherwise obstructed.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Flies die during functional imaging experiments (steps 1–17).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>If the imaging chamber is not constructed with the correct dimensions, it is possible that the body of the fly is compressed and the fly may die either through the process of the preparation or during the experiment. Ensure the fit of the chamber is adequate to restrain the fly and limit movement, without causing injury.</p>
<p>Excessive humidity and/or the presence of liquid in the imaging chamber or on the thorax of the fly (e.g., due to leaks of Ringer’s solution during preparation, or condensation carried over from the chilled plastic vials used for anesthesia) can lead to a burning of the fly during the electric shock presentation. Ensure the seal around the head of the fly is water-tight (using the blue light-curing glue) to avoid leaks, and take care to not transfer any liquid on the body of the fly when placing it in the imaging chamber (e.g., by ensuring the plastic vial is dry before placing the fly inside).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>No cells are labelled (step 9).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Ensure that all transgenes are present in the flies generated. Also, during the heat-shock step, make sure that the vials are adequately submerged in the water bath so that larvae cannot crawl up the vial walls and partially escape the heat-shock.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Allaj, E. (2018). Two simple measures of variability for categorical data. J. Appl. Stat. <i>45</i>, 1497-1516.</p>
<p id="bib2">Bilz, F., Geurten, B.R.H., Hancock, C.E., Widmann, A., and Fiala, A. (2020). Visualization of a distributed synaptic memory code in the <i>Drosophila</i> brain. Neuron <i>106</i>, 963-976.</p>
<p id="bib3">Hancock, C.E., Bilz, F., and Fiala, A. (2019). In vivo optical calcium imaging of learning-induced synaptic plasticity in <i>Drosophila melanogaster</i>. J. Vis. Exp. <i>152</i>, e60288.</p>
<p id="bib4">Honegger, K.S., Campbell, R.A., and Turner, G.C. (2011). Cellular-resolution population imaging reveals robust sparse coding in the <i>Drosophila</i> mushroom body. J. Neurosci. <i>31</i>, 11772-11785.</p>
<p id="bib5">Kreyszig, E. (1979). Advanced Engineering Mathematics.</p>
<p id="bib6">Lee, T., Lee, A., and Luo, L. (1999). Development of the <i>Drosophila</i> mushroom bodies: sequential generation of three distinct types of neurons from a neuroblast. Development <i>126</i>, 4065-4076.</p>
<p id="bib7">Lee, T. and Luo, L. (2001). Mosaic analysis with a repressible cell marker (MARCM) for <i>Drosophila</i> neural development. Trends Neurosci <i>24</i>, 251-254.</p>
<p id="bib8">Murtagh, F. and Contreras, P. (2017). Algorithms for hierarchical clustering: an overview, II. Wiley Interdiscip. Rev. Data Min. Knowl. Discov. <i>7</i>, e1219.</p>
<p id="bib9">Thévanaz, P., Ruttiman, U.E., and Unser, M. (1998). A pyramid approach to subpixel registration based on intensity. IEEE Trans. Image Process. <i>7</i>, 27-41.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, André Fiala (<a href="mailto:afiala@gwdg.de">afiala@gwdg.de</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The dataset generated during this study are available on Mendeley (<a href="https://doi.org/10.17632/wf7gz3wfr3.1">https://doi.org/10.17632/wf7gz3wfr3.1</a>) and code is deposited on GitHub (<a href="https://github.com/zerotonin/KCC_KenyonCellCorrelator">https://github.com/zerotonin/KCC_KenyonCellCorrelator</a>).</p>
<h3>Acknowledgments</h3>
<p>We are grateful to the Bloomington Drosophila Stock Center for providing fly strains. We thank Florian Bilz for his contribution in establishing many of the procedures used in this study. We thank the Department of Applied Mathematics (University of Goettingen) for providing MATLAB licenses. This work was supported by the <a href="https://doi.org/10.13039/501100001659">German Research Council</a> (FOR 2705 and SFB 889/B4), the <a href="https://doi.org/10.13039/501100001663">Volkswagen Foundation</a>, and the <a href="https://doi.org/10.13039/501100010570">Ministry for Science and Culture of Lower Saxony</a> (ZN 3014).</p>
<h3>Author Contributions</h3>
<p>A.F. conceived and supervised the study. C.E.H. and B.R.H.G. wrote the manuscript. C.E.H. performed imaging experiments and analysis. B.R.H.G. designed and performed the computational analyses.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>