<article>
<section>
<h1 id="header">Affinity enrichment and identification of inositol poly- and pyrophosphate interactomes</h1>
<p><time datetime="2021-01-15">Published: January 15, 2021</time></p>
<p>David Furkert,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> Michal Nadler-Holly,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> and Dorothea Fiedler<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn3">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, 13125 Berlin, Germany</p>
<p id="aff2"><sup>2</sup>Institut für Chemie, Humboldt-Universität zu Berlin, Brook-Taylor-Straße 2, 12489 Berlin, Germany</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:dfurkert@fmp-berlin.de">dfurkert@fmp-berlin.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:fiedler@fmp-berlin.de">fiedler@fmp-berlin.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100277">10.1016/j.xpro.2020.100277</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes an affinity enrichment approach from mammalian cell extracts to identify protein binding partners of inositol hexakisphosphate (InsP<sub>6</sub>) and 5-diphosphoinositol pentakisphosphate (5PP-InsP<sub>5</sub>), two important eukaryotic metabolites. The interactomes are annotated using mass spectrometry-based proteomics, and comparison against a control resin can uncover hundreds of protein targets. Quantitative analysis of InsP<sub>6</sub>- versus 5PP-InsP<sub>5</sub>-binding proteins highlights specific protein-ligand interactions. The approach is applicable to different cells and organisms and will contribute to a mechanistic understanding of inositol poly- and pyrophosphate signaling.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib3">Furkert et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for generation of Inositol pyrophosphate affinity matrices</li>
<li>Step-by-step procedure for affinity enrichment from mammalian cell lysates</li>
<li>LC-MS/MS proteomic analysis using label-free quantification</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/410-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used in a recent publication from <a href="#bib3">Furkert et al. (2020)</a> to annotate the interactomes of the metabolites 5PP-InsP<sub>5</sub> and InsP<sub>6</sub> in mammalian cell lysates from HCT116 and HEK293T cells. To employ a classical non-covalent affinity enrichment approach, InsP<sub>6</sub> was immobilized onto agarose beads <i>via</i> attachment of a polyethyleneglycol linker at three alternative positions (the 1-, 2-, and 3-position, see <a href="#fig1">Figure 1</a>). Similarly, a non-hydrolysable bisphosphonate analog of 5PP-InsP<sub>5</sub>, 5PCP-InsP<sub>5</sub>, was derivatized at three different positions. The synthesis of the amino-functionalized compounds is described in full detail in <a href="#bib3">Furkert et al. (2020)</a> and <a href="#bib6">Wu et al. (2016)</a>. A detailed discussion of the subsequent immobilization, affinity enrichment, and mass spectrometry analysis is provided below.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/410-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overview of different amino-functionalized InsP<sub>6</sub>- and 5PCP-InsP<sub>5</sub>-molecules used in this protocol</div>
<p>Amino-1/3<b>L</b>-InsP<sub>6</sub> (<b>1</b>) and amino-1/3<b>L</b>-PCP-InsP<sub>5</sub> (<b>2</b>) (derivatized at the 1- or the 3-position). The alternative linker (<b>L</b>) attachment is indicated by dashed lines. For synthesis see <a href="#bib3">Furkert et al. (2020)</a> Amino-2<b>L</b>-InsP<sub>6</sub> and amino-2<b>L</b>-PCP-InsP<sub>5</sub> reagents (derivatized at the 2-position) and the amino-control reagent (<b>C</b>) were described in <a href="#bib6">Wu et al. (2016)</a>. The numbering of the positions in <i>myo</i>-inositol backbone is indicated in red in InsP<sub>6</sub>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>OCRL</td>
<td>Cell Signaling Technology</td>
<td>Cat# 8797, RRID:AB_2797669</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Bio-Safe Coomassie stain</td>
<td>Bio-Rad</td>
<td>Cat# 1610786</td>
</tr>
<tr>
<td>Trifluoroacetic acid</td>
<td>Merck</td>
<td>Cat# 1.08178</td>
</tr>
<tr>
<td>2-Chloroacetamide</td>
<td>Sigma-Aldrich</td>
<td>Cat# C0267</td>
</tr>
<tr>
<td>Formic acid 98%–100%</td>
<td>Merck</td>
<td>Cat# 5.43804</td>
</tr>
<tr>
<td>Sequencing grade modified trypsin</td>
<td>Promega</td>
<td>Cat# V5111</td>
</tr>
<tr>
<td>Triethylammonium bicarbonate buffer</td>
<td>Sigma-Aldrich</td>
<td>Cat# T7408</td>
</tr>
<tr>
<td>Amino-1/3<b>L</b>-InsP<sub>6</sub> (<b>1</b>)</td>
<td><a href="#bib3">Furkert et al., 2020</a></td>
<td>n/a</td>
</tr>
<tr>
<td>Amino-1/3<b>L</b>-PCP-InsP<sub>5</sub> (<b>2</b>)</td>
<td><a href="#bib3">Furkert et al., 2020</a></td>
<td>n/a</td>
</tr>
<tr>
<td>Amino-2<b>L</b>-InsP<sub>6</sub> (<b>3</b>)</td>
<td><a href="#bib6">Wu et al., 2016</a></td>
<td>n/a</td>
</tr>
<tr>
<td>Amino-2<b>L</b>-PCP-InsP<sub>5</sub> (<b>4</b>)</td>
<td><a href="#bib6">Wu et al., 2016</a></td>
<td>n/a</td>
</tr>
<tr>
<td>Amino-control reagent (<b>C</b>)</td>
<td><a href="#bib6">Wu et al., 2016</a></td>
<td>n/a</td>
</tr>
<tr>
<td>InsP<sub>6</sub>
</td>
<td>SciChem</td>
<td>Cat# 6-0-123456-Na</td>
</tr>
<tr>
<td>5PCP-InsP<sub>5</sub>
</td>
<td><a href="#bib7">Wu et al., 2013</a></td>
<td>n/a</td>
</tr>
<tr>
<td>Dulbecco's modified Eagle’s medium (DMEM)</td>
<td>Gibco</td>
<td>Cat# 11960-044</td>
</tr>
<tr>
<td>Penicillin/streptomycin</td>
<td>Gibco</td>
<td>Cat# 15140-122</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Gibco</td>
<td>Cat# 25030-024</td>
</tr>
<tr>
<td>DPBS</td>
<td>Gibco</td>
<td>Cat# 14190-094</td>
</tr>
<tr>
<td>Pierce IP lysis buffer</td>
<td>Thermo Scientific</td>
<td>Cat# 87787</td>
</tr>
<tr>
<td>PhosSTOP phosphatase inhibitor</td>
<td>Sigma-Aldrich</td>
<td>Cat# 4906845001</td>
</tr>
<tr>
<td>cOmplete, Mini, EDTA-free Protease Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat# 11836170001</td>
</tr>
<tr>
<td>Affi-Gel 15 gel</td>
<td>Bio-Rad</td>
<td>Cat# 1536051</td>
</tr>
<tr>
<td>Deuteriumoxide</td>
<td>Eurisotop</td>
<td>Cat# DLM-4-99.8-PK</td>
</tr>
<tr>
<td>4× Laemmli sample buffer</td>
<td>Bio-Rad</td>
<td>Cat# 1610747</td>
</tr>
<tr>
<td>Mini-PROTEAN TGX Precast Protein Gels, 10-well, 50 μL</td>
<td>Bio-Rad</td>
<td>Cat# 4561094</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Sigma-Aldrich</td>
<td>Cat# M3148</td>
</tr>
<tr>
<td>1,4-Dithiothreitol (DTT)</td>
<td>Merck</td>
<td>Cat# 1114740025</td>
</tr>
<tr>
<td>Phosphonoacetic acid Trace CERT</td>
<td>Sigma-Aldrich</td>
<td>Cat# 96708</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Pierce BCA Protein assay</td>
<td>Thermo Scientific</td>
<td>Cat# 24612</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Perseus analysis and raw data</td>
<td>This study and <a href="#bib3">Furkert et al., 2020</a>
</td>
<td><a href="https://data.mendeley.com/datasets/5382d2n8nm/draft?preview=1">https://data.mendeley.com/datasets/5382d2n8nm/draft?preview=1</a></td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>HEK293T</td>
<td>ATCC</td>
<td>CRL-3216</td>
</tr>
<tr>
<td>HCT116</td>
<td>ATCC</td>
<td>CCL-247</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Xcalibur software</td>
<td>Thermo Fisher Scientific</td>
<td>version 4.2</td>
</tr>
<tr>
<td>Tune</td>
<td>Thermo Fisher Scientific</td>
<td>version 3.1</td>
</tr>
<tr>
<td>MaxQuant software</td>
<td><a href="#bib1">Cox and Mann, 2008</a></td>
<td>version 1.6.2.6a</td>
</tr>
<tr>
<td>Perseus software</td>
<td><a href="#bib4">Tyanova et al., 2016</a></td>
<td>version 1.6.7.0.</td>
</tr>
<tr>
<td>MestReNova</td>
<td>n/a</td>
<td>version 10.0.2</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Acclaim PepMap 100 C18 LC columns</td>
<td>Thermo Fisher Scientific</td>
<td>164567</td>
</tr>
<tr>
<td>200 cm μPAC column</td>
<td>PharmaFluidics, Ghent, Belgium</td>
<td>Cat# 5525031518200B</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>IPm lysis buffer</td>
<td>25 mM Tris-HCl pH 7.4, 150 mM NaCl, IGEPAL 1%, glycerol 5%, MgCl<sub>2</sub> 1 mM</td>
<td>500 mL</td>
</tr>
<tr>
<td>Compound solution</td>
<td>3 mM of compound <b>1</b>, <b>2</b>, <b>3</b>, <b>4</b>, or <b>C</b> in 200 mM HEPES pH 7.4, 0.01% Triton X</td>
<td>500 μL</td>
</tr>
<tr>
<td>Ethanolamine solution</td>
<td>1 M ethanolamine pH 8 in milliQ</td>
<td>10 mL</td>
</tr>
<tr>
<td>Sodium azide solution</td>
<td>2% NaN<sub>3</sub> (v/v)</td>
<td>10 mL</td>
</tr>
<tr>
<td>Growth medium</td>
<td>Dulbecco’s modified Eagle’s medium (DMEM), complemented with 10% FBS, 100 U/mL penicillin-streptomycin, 2 mM glutamine</td>
<td>500 mL</td>
</tr>
<tr>
<td>InsP<sub>6</sub> elution solution</td>
<td>10 mM InsP<sub>6</sub> in IPm lysis buffer pH adjusted to 7.4</td>
<td>1 mL</td>
</tr>
<tr>
<td>5PCP-InsP<sub>5</sub> elution solution</td>
<td>10 mM 5PCP-InsP<sub>5</sub> in IPm lysis buffer pH adjusted to 7.4</td>
<td>1 mL</td>
</tr>
<tr>
<td>Laemmli buffer 4×</td>
<td>4× Laemmli sample buffer (Bio-Rad) and 10% (v/v) 2-mercaptoethanol</td>
<td>1 mL</td>
</tr>
<tr>
<td>Washing buffer</td>
<td>50 mM TEAB(triethylammonium bicarbonate) solution mixed with acetonitrile 1:1 (v/v)</td>
<td>100 mL</td>
</tr>
<tr>
<td>Equilibration/digestion buffer</td>
<td>50 mM TEAB</td>
<td>100 mL</td>
</tr>
<tr>
<td>Stop solution</td>
<td>0.5%(v/v) trifluoroacetic acid (TFA) in acetonitrile</td>
<td>20 mL</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Generation of beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>In the first step, amine functionalized reagents <b>1</b>,<b>2</b>,<b>3</b>,<b>4</b> or <b>C</b> (<a href="#fig1">Figure 1</a>) are coupled to commercially available Affi-Gel 15, which contains an already activated NHS ester (<a href="#fig2">Figure 2</a>). Make sure that you conduct all steps at 4°C. Centrifugation of the beads is always conducted at 2,000 × <i>g</i>. All buffers can be prepared in advanced and stored at 4°C for at least 4 weeks.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/410-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative scheme to illustrate coupling of amino-functionalized 5PCP-InsP<sub>5</sub> to Affi-gel 15</div>
</figcaption>
</figure>
<ol>
<li>Pipette ∼650 μL of Affi-Gel 15 beads into a 1.5 mL Eppendorf tube and centrifuge for 1 min. For pipetting, use a 1 mL tip and cut off the top of the tip. Make sure that, after centrifugation, you have more than 500 μL of bead suspension inside the tube.</li>
<li>After centrifugation remove the supernatant carefully and add 800 μL of a 0.01% Triton X-100 solution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For removing the supernatant, it is easiest to use a micro tip or a gel loading tip stuck onto a 1 mL tip.</div>
<ol start="3">
<li>Centrifuge again for 1 min and remove the supernatant. Repeat this washing cycle with 800 μL of 0.01% Triton X-100 one more time so that three washing steps total are carried out.</li>
<li>Make sure you have 500 μL of bead suspension in every Eppendorf tube. If you have more than 500 μL, remove excess beads.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Eppendorf tubes have a 500 μL measure.</div>
<ol start="5">
<li>Add 500 μL of freshly prepared compound solution and incubate <i>via</i> orbital rotation for 12–14 h at 4°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Incubate for 12–14 h</div>
<ol start="6">
<li>Centrifuge bead solution for 1 min and collect the supernatant carefully in an extra 15 mL tube. Wash beads as described above three times and collect the supernatant every time in the dedicated tube. The combined supernatants will be used for determining coupling efficiency.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to pipette very carefully, do not remove additional beads.</div>
<ol start="7">
<li>Add 750 μL of milliQ water and 50 μL of a 1 M Ethanolamine solution (pH 8). Incubate for 1 h at 4°C via orbital rotation.</li>
<li>Centrifuge for 1 min and collect the supernatant. Wash the beads with 800 μL of a 0.01% Triton X-100 solution and store the beads after the last washing step by adding 500 μL of a 2% (w/v) sodium azide solution.</li>
<li>For determination of the coupling yield, lyophilize the collected supernatants from step 6. Redissolve the remaining solid in 500 μL D<sub>2</sub>O. Add 37.5 μL of a 40 mM phosphonoacetic acid standard.</li>
<li>Measure a quantitative <sup>31</sup>P-NMR spectrum and compare the intensity of the signal of the standard at ∼15.7 ppm (1,500 nmol) with the intensity of the signals corresponding to the unbound compound (see <a href="#expected-outcomes">Expected outcomes</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For quantification by <sup>31</sup>P-NMR spectroscopy you need to use an experiment with a long delay time between the scans, that corresponds to about 5 ∗ T1 relaxation time. The coupling efficiency is usually between 30% and 50%.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If you do not have access to an NMR facility, another way of determining the success of coupling is by using recombinant protein. This protein should be known to interact with the modified resin. We recommend using the SPX domain of Vtc4 or the phosphohydrolase NUDT3. The plasmids can be obtained from Addgene. For expression details, see <a href="#bib5">Wild and Hothorn (2017</a>) or <a href="#bib6">Wu et al. (2016)</a>. For assay conditions please refer to <a href="#bib6">Wu et al. (2016)</a> or <a href="#bib3">Furkert et al. (2020)</a>. It is important to note, however, that this approach will not provide a quantitative assessment of the coupling efficiency.</div>
<h3 id="sec3.2">Cell lysis, preparation of cell lysate</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>In this step, the cell lysate is prepared for subsequent affinity enrichment. After aspirating the media all steps should be conducted on ice.</p>
<ol start="11">
<li>Culture HEK293T (female origin) or HCT116 (male origin) cells to 70%–80% confluency in 15 cm dishes in cell culture medium (see <a href="#materials-and-equipment">Materials and equipment</a>). Alternatively, any other mammalian cell line can be cultured, given the appropriate growth medium.</li>
<li>Wash the cells twice with 10 mL ice-cold DPBS per 15 cm dish and lyse with 2.7 mL Pierce IP Lysis buffer, supplemented with phosphatase and protease inhibitors (Roche PhosStop and cOmplete EDTA-free protease inhibitor cocktail).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Protease and phosphatase inhibitors should be completely dissolved in the lysis buffer before use.</div>
<ol start="13">
<li>Scrape the cells off with a cell scraper, transfer them with a pipette to protein low binding microcentrifuge tubes, and incubated on ice for 10 min.</li>
<li>Centrifuge the lysate at 4°C for 10 min at 17,900 × <i>g</i>.</li>
<li>Combine the supernatants in a 15 mL falcon tube and determine the protein concentration using the Pierce BCA Protein Assay kit. Store the lysate in the meantime at 4°C.</li>
<li>Dilute the cell lysate with IP lysis buffer to reach a final concentration of 3 mg/mL.</li>
</ol>
<h3 id="sec3.3">Affinity enrichment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>At this stage, the lysate from the previous step is incubated with the affinity reagents to enrich proteins that interact with the inositol (pyro)phosphates. IPm lysis buffer can be prepared in advance and stored for at least 4 weeks at 4°C.</p>
<div class="critical">
<span class="critical-title">Critical:</span> All steps should be conducted at 4°C.</div>
<ol start="17">
<li>Transfer 200 μL of the prepared bead suspension (stored in 2% sodium azide solution) to a protein low bind tube.</li>
<li>Wash the beads three times with 1 mL of IPm lysis buffer.</li>
<li>After removing the supernatant, add 1 mL of cell lysate (3 mg/mL) to the beads and incubate with constant rotation for 1 h.</li>
<li>Centrifuge the tubes for 1 min and remove the supernatant.</li>
<li>Wash the beads three times with 1 mL of IPm lysis buffer.</li>
<li>After removal of the supernatant, add 150 μL of an InsP<sub>6</sub>- or 5PCP-InsP<sub>5</sub>-elution solution to the beads and incubate for 60 min under orbital rotation.</li>
<li>Collect the supernatant and freeze it in liquid nitrogen. Store the samples at −80°C. For proteomic analysis and label-free quantification, a minimum of four technical replicates should be performed. Before preparing the samples for mass spectrometry, it is advisable to test success of enrichment <i>via</i> western blot analysis of a small aliquot for known binding partners. We recommend blotting for OCRL (primary antibody is commercially available).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Four technical replicates were used to improve the statistical significance of the measured LFQ values. At least three replicates should be recorded.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The samples can be stored at −80°C for several months.</div>
<h3 id="sec3.4">Sample preparation for proteomic analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>In this section, the enriched protein samples are prepared <i>via</i> in-gel digestion for proteomic analysis.</p>
<div class="note">
<span class="note-title">Note:</span> In-solution digestion can also be used at this step.</div>
<ol start="24">
<li>Add 5 μL of a 4 mM DTT solution to 90 μL of enriched protein lysate. Incubate for 30 min at 55°C.</li>
<li>After the samples reach 22°C–26°C, add 5 μL of a freshly prepared 15 mM chloroacetamide solution and incubate for 30 min at 22°C–26°C.</li>
<li>Quench by adding 33 μL of Laemmli buffer 4× (see <a href="#key-resources-table">Key resources table</a>) and incubate the solution at 95°C for 10 min.</li>
<li>Load 50 μL on a 10 well 4%–20% SDS-PAGE gel and let the sample run until the whole sample has transferred from the loading well into the agarose gel and no sample is left in the loading pockets.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend to leave a free lane between each sample.</div>
<ol start="28">
<li>Wash the gel three times with water.</li>
<li>Stain the gel with Bio-Rad Bio-safe Coomassie G-250 stain for 12–14 h.</li>
<li>Cut out the stained gel bands and transfer each of them into a 0.5 mL Eppendorf tube.</li>
<li>Wash the gel bands with 200 μL wash solution, shake for 10 min at 30°C, and remove the wash solution.</li>
<li>Equilibrate the gel bands with 200 μL equilibration buffer and shake for 10 min at 30°C, then remove the equilibration solution.</li>
<li>Shrink the gel pieces by adding 200 μL acetonitrile and incubate for 1 min and remove the acetonitrile solution. Repeat the shrinking step once more.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The color of the gel pieces should turn white at this step, indicating complete dehydration of the gel pieces.</div>
<ol start="34">
<li>Digest the proteins with 0.1 μg trypsin solution for 12–14 h at 37°C on a shaker.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Digest protein until next morning</div>
<ol start="35">
<li>Spin down the samples and afterwards stop digestion using 30 μL of the stop solution to extract the peptides.</li>
<li>Transfer the supernatant to a glass vial for HPLC use.</li>
<li>Shrink the gel again by dehydration with 20 μL acetonitrile, vortex gently for a few seconds.</li>
<li>Transfer the supernatant to the same glass vial from step 36.</li>
<li>Dry the sample in a speedVac vacuum and concentrate the sample until the solvent is completely removed/evaporated. It takes approximately 30–50 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not leave the samples in the speedVac vacuum for too long. Over-drying the samples might cause difficulties in recovering the peptides.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dried peptides can be stored at −80°C until subjected to the LC-MS/MS analysis.</div>
<h3 id="sec3.5">LC-MS/MS analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> days to weeks</div>
<p>In this protocol, a Dionex UltiMate 3000 system coupled with an Orbitrap Fusion mass spectrometer is used. A PepMap C-18 trap-column (Thermo Fischer Scientific) of 0.075 mm ID × 50 mm length is utilized; 3 μm particle size and 100 Å pore size for sample trapping. Mobile phase A contains 1% acetonitrile and 0.05% TFA acid in water, and mobile phase B 0.05% TFA acid in acetonitrile. For sample separation, a 200 cm μPAC column is employed (PharmaFluidics, Ghent, Belgium), with mobile phase A containing 0.1% formic acid in water, and mobile phase B 0.1% formic acid in acetonitrile.</p>
<ol start="40">
<li>Dissolve the samples in 6 μL of 1% acetonitrile + 0.05% TFA in water.</li>
<li>Sonicate the samples in an ultrasonic bath for 5 min and place them in the HPLC.</li>
<li>Inject 3 μL of the sample (equivalent to 0.5–1 μg) into the HPLC system.</li>
<li>Trapping of the sample is carried out with a trap C-18 column at a flow rate of 5.6 μL/min for 5 min, using mobile phase A.</li>
<li>Samples are separated with alternating flow rate of 750 nL/min and 350 nL/min, see <a href="#tbl1">Table 1</a>.
<table id="tbl1">
<caption>Table 1. Parameters for reverse phase chromatography separation for LC-MS/MS</caption>
<thead>
<tr>
<th>Time (min)</th>
<th>Mobile phase B (%)</th>
<th>Flow rate (nL/min)</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>4</td>
<td>750</td>
</tr>
<tr>
<td>75</td>
<td>26</td>
<td>750</td>
</tr>
<tr>
<td>75.1</td>
<td>26</td>
<td>350</td>
</tr>
<tr>
<td>80</td>
<td>28</td>
<td>350</td>
</tr>
<tr>
<td>92</td>
<td>40</td>
<td>350</td>
</tr>
<tr>
<td>95</td>
<td>50</td>
<td>350</td>
</tr>
<tr>
<td>97</td>
<td>80</td>
<td>350</td>
</tr>
<tr>
<td>104</td>
<td>80</td>
<td>350</td>
</tr>
<tr>
<td>104.1</td>
<td>4</td>
<td>750</td>
</tr>
<tr>
<td>117</td>
<td>0</td>
<td>750</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The μPAC column enables high flow rate separation because of its low back pressure, the lower flow rate is used to get optimized spray in the middle and at the end of the gradient.</div>
<ol start="45">
<li>MS scans are acquired in DDA mode. MS1 scan are acquired in the Orbitrap with a range of 350–1,500 m/z, mass resolution of 120,000, AGC target value of 4e5 and 50 ms injection time. MS2 scans are acquired in the ion trap with an AGC target value of 1e4 and 35 ms injection time. Precursor ions with charge states 2–4 are isolated with an isolation window of 1.6 m/z and 40 s dynamic exclusion. Precursor ions are fragmented using higher-energy collisional dissociation (HCD) with 30% normalized collision energy. Cycle time is set to 1 s. Data acquisition is done with Xcalibur software 4.2 and tune 3.1.</li>
<li>Analysis and processing of the raw data is conducted with MaxQuant software version 1.6.2.6a, keeping the MaxQuant standard settings. In the search parameters two missed cleavage sites are included, the fixed modification is set to cysteine carbamidomethyl modification, and variable modifications is set to methionine oxidation and N-terminal protein acetylation. The peptide mass tolerance is set to 4.5 ppm for MS scans and 20 ppm for MS/MS scans. Enable the match between runs option. The Human UniProt/Swiss-Prot database version 2016-10 (<a href="https://www.uniprot.org/)is">https://</a><a href="http://www.uniprot.org/">www.uniprot.org/</a>)is searched using Andromeda. The false discovery rate (FDR) is set to 1% for both peptide and protein level. Protein quantification is based on at least 2 razor and unique peptides. Enable Label-free quantification and iBAQ calculation.</li>
<li>Bioinformatic analysis is carried out using Perseus software (<a href="#bib4">Tyanova et al., 2016</a>). The ProteinGroup file, which is an output file of MaxQuant (<a href="#bib1">Cox and Mann, 2008</a>), is imported into Perseus software version 1.6.7.0. Proteins are filtered to exclude reverse database hits, potential contaminants, and proteins only identified by site. Proteins are filtered by rows, requiring a valid value for at least two proteins out of four technical replicates. The date is Log2(x) transformed and impute using Perseus default parameters, 0.3 width and 1.8 down shift. Volcano plots are generated using a t test (number of randomizations: 250) and FDR = 0.01 and S0 = 2. (<a href="#fig4">Figure 4</a>)
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/410-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Exemplary Volcano plot depicting LFQ values of 5PCP-InsP<sub>5</sub> versus control reagent (Ctrl) after a t test (LFQ = 0.01, s0 = 2)</div>
<p>The X axis displays the difference of LFQ values on a log2 scale and the Y axis shows the −log10 p value. Inositol pyrophosphate metabolizing enzymes IP6K1, NUDT3, NUDT4, NUDT10, NUDT11, PPIP5K2, and the characterized 5PP-InsP<sub>5</sub> binding protein SYT1 are highlighted in red. Figure reprinted with permission from <a href="#bib3">Furkert et al. (2020)</a>.</p>
</figcaption>
</figure>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">Coupling to the beads</h3>
<p>After subjecting 1,500 nmol of the amino-functionalized compounds (<b>1</b>, <b>2</b>, <b>3</b>, <b>4</b>, or <b>C</b>) to the NHS ester coupling, the combined supernatants are analyzed via <sup>31</sup>P-NMR spectroscopy, to determine the amount of remaining, uncoupled compounds. For quantitative analysis of the coupling efficiency, 1,500 nmol phosphonoacetic acid is used as a <sup>31</sup>P-NMR standard. In the example in <a href="#fig3">Figure 3</a>, the integration of the standard at 15.7 ppm is compared to the sum of the three peaks around 0 ppm, corresponding to the five phosphoryl groups on compound <b>3</b>. For compound <b>3</b>, a ratio of 5:1 (compound:standard) would correspond to 100% unreacted compound. Thus, the coupling efficiency in this example amounts to 46% (690 nmol of immobilized Amino-2<b>L</b>-5PCP-InsP<sub>5</sub>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/410-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. <sup>31</sup>P NMR spectrum of collected supernatants after coupling of amino-2<b>L</b>-5PCP-InsP<sub>5</sub> to Affi-gel 15 with 1,500 nmol of phosphonoacetic acid at 15.7 ppm</div>
</figcaption>
</figure>
<h3 id="sec4.2">Proteomics</h3>
<p>In previous experiments, up to 2,000 different proteins were detected, and bioinformatic analysis as described in steps 46 and 47 revealed enrichment of ca. 400 proteins by the inositol polyphosphate affinity reagents compared to the control reagent.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>With this protocol the interactome of the metabolites 5PP-InsP<sub>5</sub> and InsP<sub>6</sub> can be analyzed and interacting proteins can subsequently be validated. An inherent limitation for this type of non-covalent chemoproteomic analysis is the affinity enrichment of whole protein complexes in which not all components directly interact with the inositol phosphate. This challenge can be addressed in the future by developing covalent probes, for example by incorporating a photoreactive diazirine group into the linker, which will allow for much more stringent washing conditions. Consequently, only the direct binding proteins could be identified.</p>
<p>Because the pyrophosphate group of 5PP-InsP<sub>5</sub> is prone to hydrolysis, it was replaced by a non-hydrolysable methylene bisphosphonate (PCP) moiety. These mimics have shown to imitate the properties and functions of the natural molecules sufficiently in many cases. However, a shortcoming of the PCP bioisosters are the differences in pK<sub>a</sub> values of the bisphosphonate moiety. (<a href="#bib2">Elliott et al., 2012</a>) Therefore, the development of stabilized analogs that can most closely resemble the properties of the natural molecules, for example difluoromethylene bisphosphonates (PCF<sub>2</sub>P), may enable the identification of additional interacting proteins.</p>
<p>In all our datasets a substantial overlap between the interactomes of InsP<sub>6</sub> and 5PP-InsP<sub>5</sub> was observed. This overlap was anticipated to some extent, because of the similarity in size and charge of these small molecules. However, to identify proteins that preferentially interact with 5PP-InsP<sub>5</sub> over InsP<sub>6</sub> or <i>vice versa</i>, systematic competition experiments should be carried out. Adding increasing amounts of different soluble inositol (pyro)phosphates into the lysate - which will compete for binding to the affinity matrix – should enable a more quantitative and granular analysis of the binding specificities in the future.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Coupling yield to the beads is very low (&lt;10%).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Control the pH of your HEPES buffer solution and use a fresh batch of Affi-gel 15.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The coupling efficiency of the reagents to Affi-Gel 15 cannot be determined because of the high signal to noise ratio in the <sup>31</sup>P-NMR spectrum</p>
<h3 id="sec6.4">Potential solution</h3>
<p>The signal to noise ratio can be improved by increasing the number of scans. Additionally, make sure to not accidently add beads into the sample for NMR analysis.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Protein concentration of the cell lysate is too low.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Use a larger number of cell dishes and combine the material following cell dissociation using trypsin.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Control lane in the SDS-PAGE is colored as much as the other lanes after staining.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Add an additional washing step during affinity enrichment and make sure to remove all the supernatants each time.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>The amount of peptides detected during MS is low.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The peptide samples may have dried out too much in the speedVac. Monitor the time in the speedVac vacuum carefully and make sure to stop the speedVac, when the samples are dry.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cox, J. and Mann, M. (2008). MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol. <i>26</i>, 1367-1372.</p>
<p id="bib2">Elliott, T.S., Slowey, A., Ye, Y., and Conway, S.J. (2012). The use of phosphate bioisosteres in medicinal chemistry and chemical biology. MedChemComm <i>3</i>, 735-751.</p>
<p id="bib3">Furkert, D., Hostachy, S., Nadler-Holly, M., and Fiedler, D. (2020). Triplexed affinity reagents to sample the mammalian inositol pyrophosphate interactome. Cell Chem. Biol. <i>27</i>, 1097-1108.e4.</p>
<p id="bib4">Tyanova, S., Temu, T., Sinitcyn, P., Carlson, A., Hein, M.Y., Geiger, T., Mann, M., and Cox, J. (2016). The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods <i>13</i>, 731-740.</p>
<p id="bib5">Wild, R. and Hothorn, M. (2017). The macro domain as fusion tag for carrier-driven crystallization. Protein Sci. <i>26</i>, 365-374.</p>
<p id="bib6">Wu, M., Chong, L.S., Perlman, D.H., Resnick, A.C., and Fiedler, D. (2016). Inositol polyphosphates intersect with signaling and metabolic networks via two distinct mechanisms. Proc. Natl. Acad. Sci. U S A <i>113</i>, E6757-E6765.</p>
<p id="bib7">Wu, M., Dul, B.E., Trevisan, A.J., and Fiedler, D. (2013). Synthesis and characterization of non-hydrolysable diphosphoinositol polyphosphate messengers. Chem. Sci. <i>4</i>, 405-410.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Dorothea Fiedler (<a href="mailto:fiedler@fmp-berlin.de">fiedler@fmp-berlin.de</a>)</p>
<h4>Materials availability</h4>
<p>Requests for resources and reagents should be directed to Dorothea Fiedler (<a href="mailto:fiedler@fmp-berlin.de">fiedler@fmp-berlin.de</a>). Availability of the reagents may be limited due to the requirement for multi-step syntheses.</p>
<h4>Data and code availability</h4>
<p>The datasets and corresponding files generated are available as supporting files of this manuscript and the raw data can be found at: <a href="https://data.mendeley.com/datasets/5382d2n8nm/draft?preview=1">https://data.mendeley.com/datasets/5382d2n8nm/draft?preview=1</a></p>
<h3>Acknowledgments</h3>
<p>We thank Heike Stephanowitz for assisting with sample preparation for mass spectrometry and Dr. Fan Liu for access to the mass spectrometers. We also thank Katy Franke for conducting the syntheses of certain precursors and Lena von Oertzen for supporting the cell culture experiments. We thank all members of the Fiedler group for careful reading of the protocol. D. Furkert is supported by funding from the Swiss National Foundation Sinergia Grant CRSII5_170925.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, D. Furket and D. Fiedler; methodology, D. Furket, M.N.-H., and D. Fiedler; formal analysis, D. Furket, M.N.-H., and D. Fiedler; investigation, D. Furket and M.N.-H.; writing – original draft, D. Fiedler, M.N.-H., and D. Furket; writing – review and editing, D. Furket and D. Fiedler; visualization, D. Furket; funding acquisition, D. Fiedler.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>