<article>
<section>
<h1 id="header">Chromatin-Bound Proteome Profiling by Genome Capture</h1>
<p><time datetime="2020-03-06">Published: March 6, 2020</time></p>
<p>Sergi Aranda,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Eva Borràs,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Eduard Sabidó,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Luciano Di Croce<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona 08003, Spain</p>
<p id="aff2"><sup>2</sup>Universitat Pompeu Fabra (UPF), Barcelona, Spain</p>
<p id="aff3"><sup>3</sup>Institució Catalana de Recerca i Estudis Avançats (ICREA), Pg. Lluis Companys 23, Barcelona 08010, Spain</p>
<p id="fn1"><sup>4</sup>Technical Contact</p>
<p id="fn2"><sup>5</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:sergi.aranda@crg.eu">sergi.aranda@crg.eu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:luciano.dicroce@crg.eu">luciano.dicroce@crg.eu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100014">10.1016/j.xpro.2020.100014</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p><i>De novo</i> identification of chromatin interactors can reveal unexpected pathways relevant to physiology and human disease. Inspired by the DNA mediated chromatin pull-down (Dm-ChP) technology (also known as iPOND [isolation of proteins on nascent DNA]) for the proteomic characterization of nascent DNA, we have recently reported a new experimental protocol that allows for the identification of proteins on total DNA (iPOTD) for bulk chromatome profiling and <i>de novo</i> identification of chromatin-bound proteins. Here, we detail a step-by-step protocol to survey the cellular chromatin-bound proteome in a simple, robust, and unbiased manner.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib1">Aranda et al. (2019)</a>.</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for the identification of proteins on total DNA</li>
<li>Method for global and unbiased chromatin-bound proteome profiling</li>
<li>Method for <i>de novo</i> identification of chromatin binders</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/61-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<h3>Setting up the EdU Labelling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>Before starting to use the strategy described herein, the initial setup of the concentration of EdU and the incubation time is required to evaluate the complete EdU-labelling of cells as well as the EdU toxicity. We recommend evaluating the possible cytotoxicity of EdU incubation at three different levels (<a href="#bib1">Aranda et al., 2019</a>): i) at the transcriptional level, either globally (e.g. by massive parallel RNA sequencing) or on specific targets-genes (e.g. Quantitative reverse transcription PCR (RT-qPCR) on DNA damage response genes); ii) at the proliferation level; and iii) at the DNA damage response activation.</p>
<p>Use an appropriate number of cells and culture conditions so that the cells are under exponential growth during the entire incubation time. In order to perform a complete EdU labelling of the genome of all the cells in culture, one should take into consideration the doubling time of the cells. Typically, chromatin capture uses about 2–4x10<sup>7</sup> exponentially growing cells as starting point. However, we have efficiently purified chromatin from less than 1x10<sup>7</sup> cells. Further, a typical study would include two conditions (e.g., EdU-incubated cells as well as non-incubated control cells) and at least three biological replicates. The time and concentration indicated in this protocol refer to those used with mouse embryonic stem cells (ESCs) (<a href="#bib1">Aranda et al., 2019</a>).</p>
<ol>
<li>Expand the cells in twelve independent dishes for Western blot analysis, and twelve independent dishes for immunofluorescence analysis. Include a positive control for DNA damage, using a DNA damage agent reported in the literature ( <i>e.g.</i> Doxorubicin 10 μM 6h). Expand the cell cultures 1 d before EdU Incubation.</li>
<li>Replace the media with fresh media containing a serial dilution of EdU, as indicated in the table below.</li>
</ol>
<table>
<thead>
<tr>
<th>Incubation Time:</th>
<th>24 h</th>
<th>48 h</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="6">EdU concentration (μM)</td>
<td>0</td>
<td>0</td>
</tr>
<tr>
<td>0,02</td>
<td>0,02</td>
</tr>
<tr>
<td>0,1</td>
<td>0,1</td>
</tr>
<tr>
<td>0,5</td>
<td>0,5</td>
</tr>
<tr>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>5</td>
<td>5</td>
</tr>
<tr>
<td colspan="3">DNA damage agent</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> This table is given as a reference; conditions may need to be adjusted for specific cell types.</div>
<h3>For Cell Cycle Analysis</h3>
<ol start="3">
<li>After EdU incubation, rinse the cells twice with 1× PBS (pH 7.4).</li>
<li>Trypsinise cells and count them.</li>
<li>Collect 1 × 10<sup>6</sup> cells by centrifugation in a swing-bucket rotor at 150<i>g</i> for 5 min at 4°C.</li>
<li>Suspend the cells in 1 mL of 1× PBS (pH 7.4).</li>
<li>Add 3 mL of cold ethanol dropwise, while maintaining the cells suspended by agitation with low vortexing.</li>
<li>Place the cell suspension at –20°C for at least 1 h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The cell suspension can be stored at –20°C for up to 1 year.</div>
<ol start="9">
<li>Add 10 mL of 1× PBS (pH 7.4) to the cell suspension and incubate for 20 min at room temperature.</li>
<li>Collect the cells by centrifugation in a swing-bucket rotor at 500<i>g</i> for 10 min at 4°C.</li>
<li>Discard the supernatant and wash the cells with an additional 10 mL of 1× PBS (pH 7.4).</li>
<li>Collect the cells by centrifugation in a swing-bucket rotor at 500<i>g</i> for 10 min at 4°C.</li>
<li>Resuspend the cells in 1 mL of 1× PBS (pH 7.4) containing 15 μg/mL of propidium iodide, 1 mM sodium citrate and 0.3 mg/mL of ribonuclease A.</li>
<li>Incubate the cells for 1 h at 37°C, or alternatively overnight at 4°C.</li>
<li>Analyse the cell cycle profile with an appropriate flow cytometer.</li>
</ol>
<h3>For Western Blot Analysis of Whole Cell Lysates</h3>
<ol start="16">
<li>After EdU incubation, place the dishes on ice and rinse the cells with 1× PBS (pH 7.4).</li>
<li>Lyse the cells with 1× SDS-lysis buffer by adding it directly to the dish, using at ratio of 1:15 to 1:20 (vol/vol) of SDS-lysis buffer to initial culture media.</li>
<li>Harvest cells with a cell lifter and collect them in an Eppendorf tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The addition of the SDS-lysis buffer results in viscous solution. Cut about 2 mm off the end of the pipette tip to be used prior to transferring the solution into the Eppendorf tube.</div>
<ol start="19">
<li>Boil the samples for 15 min at 95°C, and centrifuge for clarification, and collect the the supernatant into a clean 1.5-mL Eppendorf tube.</li>
<li>Quantify the protein concentration using a BCA Protein Assay Kit with an appropriate standard.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This kit is based on a Biuret reaction and is compatible with the SDS-buffer lysis used. Other assays based on Lowry or Bradford reactions must be checked for possible interference.</div>
<ol start="21">
<li>Evaluate the levels of DNA damage markers (e.g. γH2AX) on 20-40 μg of total protein by conventional Western blot as in (<a href="#bib2">Aranda et al., 2014</a>).</li>
</ol>
<h3>For Immunofluorescence Analysis of EdU Labelling</h3>
<ol start="22">
<li>Fix the cells by adding an equal volume of 8% (wt/vol) formaldehyde fixation solution directly into the media.</li>
</ol>
<p><b><i>Caution:</i></b> Add the solution under a fume hood using gloves.</p>
<ol start="23">
<li>Mix gently to dilute the formaldehyde in the media, and incubate the cells for 10 min at room temperature.</li>
<li>Remove the fixation media and rinse the cells three times with 1× PBS (pH 7.4).</li>
</ol>
<p><b><i>Caution:</i></b> Fixation solution is harmful for the researcher and the environment. Dispense of it appropriately.</p>
<ol start="25">
<li>Stain the cells using an antibody against a DNA damage marker (e.g. γH2AX) using conventional immunofluorescence protocols.</li>
<li>Prepare the Click reaction mix with Alexa Fluor <sup>TM</sup> 647 Azide.</li>
<li>After antibody staining, and before placing the samples into mounting media, rinse the cells twice with 1× PBS (pH 7.4).</li>
<li>Incubate the cells with the Click reaction for 30 min at room temperature, covering the sample with aluminium foil.</li>
<li>Evaluate the intensity and distribution of the DNA damage marker and EdU staining, using conventional florescent microscopy, similar to that performed in (<a href="#bib1">Aranda et al., 2019</a>).</li>
</ol>
<h3>Monitoring DNA Fragmentation and Labelling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>The efficient purification of chromatin by streptavidin beads required an average size from fragmentation of 200–500 bp. At larger fragment sizes, the efficiency of chromatin purification drops. The current protocol does not exclude the use of different fixative conditions. Typically (<a href="#bib1">Aranda et al., 2019</a>), the cells are crosslinked with a 1% methanol-free formaldehyde solution. The use of different fixative solutions might modify the chromatin fragmentation efficiencies. Using a simple and direct dot blot assay from fragmented DNA, researchers can monitor and quantify the efficiency of biotin incorporation during the Click reaction, using a biotinylated oligonucleotide as a standard.</p>
<h4>Fragmentation</h4>
<ol start="30">
<li>Collect 200-μL fragmented sample from Step 38 (of the main procedure) and add 10-μL NaCl (5 M) and 2-μL proteinase K. Incubate the mixture overnight at 65°C with vigorous shaking (1,000 rpm) in a Thermomixer.</li>
<li>Purify the DNA from the mixture using the QIAquick PCR purification kit, following the manufacture’s recommendations.</li>
<li>Separate 500 ng to 800 ng of purified genomic DNA (gDNA) over a 1.5% agarose gel.</li>
</ol>
<p>See <a href="#troubleshooting">Troubleshooting</a></p>
<div class="critical">
<span class="critical-title">Critical:</span> Optimal fragmentation is around 200–500 bp. If fragmentation is not optimal at this step, include additional sonication cycles.</div>
<h4>EdU Labelling</h4>
<ol start="33">
<li>Spot 1-μL dots of gDNA onto a positively-charged nylon membrane in triplicate.</li>
<li>Similarly, spot a serially-diluted 5′-biotinylated oligonucleotide as a standard.</li>
<li>Air-dry the membrane, and crosslink with 0.4 J/cm<sup>2</sup>.</li>
<li>Wet the membrane with TBS-T and bend into a 50-mL polypropylene tube.</li>
<li>Block the membrane with 5% (wt/vol) of skimmed milk in TBS-T overnight at 4°C.</li>
<li>Incubate the membrane with HRP-Avidin (1:1000 dilution) for 30 min at room temperature.</li>
<li>Develop the membranes using the ECL prime Western blotting detection system, according to manufacturer’s instruction.</li>
</ol>
<p>See <a href="#troubleshooting">Troubleshooting</a></p>
<h3>Monitoring Chromatin Capture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Similar to the method for monitoring EdU incorporation, the efficiency and specific chromatin capture can be evaluated by a dot blot assay with an anti-histone H3 (chromatin marker) or anti-vinculin (cytoskeleton marker) antibodies. Additionally, the intensity of H3 signal per sample can be used to normalize the amount of chromatin-to-be-analysed between different conditions (e.g. drug-treated versus non-treated cells).</p>
<ol start="40">
<li>Spot 1-μL dots from Input sample from Step 22, and eluted capture samples from Step 38 (from the main procedure), onto a nitrocellulose membrane in triplicate. Prepare the same membrane twice for anti-H3 and anti-vinculin blotting.</li>
<li>Air-dry the membranes, wet them with TBS-T and then bend each into a 50-mL polypropylene tube.</li>
<li>Block the membranes for 1 h at room temperature with a 10% (wt/vol) of skimmed milk in TBS-T in a roller mixer.</li>
<li>Incubate the membranes with a 5% (wt/vol) of skimmed milk in TBS-T, with either an anti-H3 (1:1000; chromatin marker) or anti-vinculin (1:1000; cytoskeleton marker) antibody overnight at 4°C in a roller mixer.</li>
<li>Wash the membranes twice with TBS-T for 10 min at room temperature in a roller mixer.</li>
<li>Incubate the membranes with a 5% (wt/vol) of skimmed milk in TBS-T, with an anti-mouse HRP antibody during 1h at RT in a roller mixer.</li>
<li>Develop the membranes using the ECL prime Western blotting detection system according to manufacturer’s instruction.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-histone H3 antibody</td>
<td>Abcam</td>
<td>Cat# ab1791</td>
</tr>
<tr>
<td>Anti-vinculin antibody</td>
<td>Merck</td>
<td>Cat# V9131</td>
</tr>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 31350010</td>
</tr>
<tr>
<td>Alexa FluorTM 647 Azide, triethylammonium salt</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A10277</td>
</tr>
<tr>
<td>Acetonitrile LCMS grade</td>
<td>Sigma-Aldrich</td>
<td>Cat# 34967</td>
</tr>
<tr>
<td>Acetonitrile + 0.1% formic acid LCMS grade</td>
<td>Fisher Chemical</td>
<td>Cat# LS120-1</td>
</tr>
<tr>
<td>Ammonium bicarbonate, &gt;99.5%</td>
<td>Sigma-Aldrich</td>
<td>Cat# 09830-500G</td>
</tr>
<tr>
<td>Avidin, HRP conjugate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 434423</td>
</tr>
<tr>
<td>B27 supplement, serum-free</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 17504044</td>
</tr>
<tr>
<td>Biotin azide (PEG4 carboxamide-6-azidohexanyl biotin)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# B10184</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>New England biolabs</td>
<td>Cat# P8108S</td>
</tr>
<tr>
<td>Bovine albumin fraction V, BSA V</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15260-037</td>
</tr>
<tr>
<td>β-Glycerol phosphate disodium salt pentahydrate, ≥98.0%</td>
<td>Merck</td>
<td>Cat# 50020</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>Selleck</td>
<td>Cat# S1263-25mg</td>
</tr>
<tr>
<td>Copper(II) sulfate pentahydrate</td>
<td>Merck</td>
<td>Cat# C8027</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11320074</td>
</tr>
<tr>
<td>Dimethyl sulfoxide, DMSO</td>
<td>Merck</td>
<td>Cat# D2650</td>
</tr>
<tr>
<td>DTT, DL-dithiothreitol solution, BioUltra, for molecular biology, ∼1 M in H2O</td>
<td>Merck</td>
<td>Cat# 43816</td>
</tr>
<tr>
<td>DTT, DL-dithiothreitol &gt;99%</td>
<td>Sigma-Aldrich</td>
<td>Cat# D9163-25G</td>
</tr>
<tr>
<td>EDTA disodium salt dihydrate</td>
<td>PanReac</td>
<td>Cat# 131669</td>
</tr>
<tr>
<td>EdU, 5-ethynyl-2’-deoxyuridine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A10044</td>
</tr>
<tr>
<td>EmbryoMax® 0.1% gelatin solution</td>
<td>Merck</td>
<td>Cat# ES-006</td>
</tr>
<tr>
<td>Formic acid &gt;98%</td>
<td>Merck</td>
<td>Cat# 1.00264.0100</td>
</tr>
<tr>
<td>GlutaMAX™ supplement</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 35050038</td>
</tr>
<tr>
<td>Glycerol, for molecular biology, ≥99%</td>
<td>Merck</td>
<td>Cat# G5516</td>
</tr>
<tr>
<td>Glycine, BioUltra, for molecular biology, ≥99.0%</td>
<td>Merck</td>
<td>Cat# 50046</td>
</tr>
<tr>
<td>HCl, 36.5–38.0%</td>
<td>Merck</td>
<td>Cat# H1758</td>
</tr>
<tr>
<td>HEPES, BioPerformance certified, ≥99.5% (titration), cell culture tested</td>
<td>Merck</td>
<td>Cat# H4034</td>
</tr>
<tr>
<td>HEPES buffer solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15630080</td>
</tr>
<tr>
<td>Iodoacetamide, &gt;99%</td>
<td>Sigma-Aldrich</td>
<td>Cat# I1149-5G</td>
</tr>
<tr>
<td>KnockOut™ DMEM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10829018</td>
</tr>
<tr>
<td>KnockOut™ Serum Replacement</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10828028</td>
</tr>
<tr>
<td>L-Ascorbic acid</td>
<td>Merck</td>
<td>Cat# A7506</td>
</tr>
<tr>
<td>LIF, ESGRO® Recombinant Mouse LIF Protein</td>
<td>Merck</td>
<td>Cat# ESG1107</td>
</tr>
<tr>
<td>Lysyl Endopeptidase, LysC</td>
<td>Wako Chemicals</td>
<td>Cat# 129-02541</td>
</tr>
<tr>
<td>MEM Non-Essential Amino Acids Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11140050</td>
</tr>
<tr>
<td>Methanol LCMS grade</td>
<td>Sigma-Aldrich</td>
<td>Cat# 14262-2L</td>
</tr>
<tr>
<td>Magnesium chloride hexahydrate, MgCl2</td>
<td>Merck</td>
<td>Cat# M2670</td>
</tr>
<tr>
<td>N-2 supplement</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 17502048</td>
</tr>
<tr>
<td>Sodium orthovanadate, ≥90%; Na3VO4</td>
<td>Merck</td>
<td>Cat# S6508</td>
</tr>
<tr>
<td>Sodium chloride, NaCl</td>
<td>Merck</td>
<td>Cat# 106404</td>
</tr>
<tr>
<td>Neurobasal</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 21103-049</td>
</tr>
<tr>
<td>Paraformaldehyde, reagent grade</td>
<td>Merck</td>
<td>Cat# P6148</td>
</tr>
<tr>
<td>PD0325901</td>
<td>Selleck</td>
<td>Cat# S1036-5mg</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15140122</td>
</tr>
<tr>
<td>PIC, cOmplete™, EDTA-free protease inhibitor cocktail</td>
<td>Merck</td>
<td>Cat# 5056489001</td>
</tr>
<tr>
<td>Potassium chloride; KCl</td>
<td>Merck</td>
<td>Cat# 104936</td>
</tr>
<tr>
<td>Potassium phosphate monobasic, KH2PO4</td>
<td>Merck</td>
<td>Cat# P5655</td>
</tr>
<tr>
<td>Sodium carbonate anhydrous, Na2CO3</td>
<td>PanReac</td>
<td>Cat# A3900</td>
</tr>
<tr>
<td>Sodium phosphate dibasic dehydrate, Na2HPO4·2H2O</td>
<td>Merck</td>
<td>Cat# 71643</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate, ultrapure, SDS</td>
<td>PanReac</td>
<td>Cat# A1112</td>
</tr>
<tr>
<td>Sodium deoxycholate ≥97%</td>
<td>Merck</td>
<td>Cat# D6750</td>
</tr>
<tr>
<td>ssDNA, deoxyribonucleic acid, low molecular weight from salmon sperm</td>
<td>Merck</td>
<td>Cat# 31149</td>
</tr>
<tr>
<td>Sucrose, for molecular biology, ≥99.5%</td>
<td>Merck</td>
<td>Cat# S0389</td>
</tr>
<tr>
<td>Sequencing-grade modified trypsin</td>
<td>Promega</td>
<td>Cat# V5111</td>
</tr>
<tr>
<td>Triton® X-100 for analysis</td>
<td>Merck</td>
<td>Cat# 108603</td>
</tr>
<tr>
<td>Trizma® base, ≥99.9%, (titration), crystalline</td>
<td>Merck</td>
<td>Cat# T4661</td>
</tr>
<tr>
<td>TWEEN® 20</td>
<td>Merck</td>
<td>Cat# P1379</td>
</tr>
<tr>
<td>Urea</td>
<td>Sigma-Aldrich</td>
<td>Cat# 17-1319-01</td>
</tr>
<tr>
<td>Water, LCMS grade</td>
<td>Fisher Chemical</td>
<td>Cat# 10434902</td>
</tr>
<tr>
<td>Water + 0.1% formic acid, LCMS grade</td>
<td>Fisher Chemical</td>
<td>Cat# LS118-212</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>BCA Protein Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 23225</td>
</tr>
<tr>
<td>ECL Prime Western blotting detection system</td>
<td>GE Healthcare Life Sciences</td>
<td>Cat# RPN2232</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>Quiagen</td>
<td>Cat# 28104</td>
</tr>
<tr>
<td colspan="3">Experimental Models: Cell Lines</td>
</tr>
<tr>
<td>Embryonic stem cells (ESCs), ES-E14TG2a, mouse (strain 129/Ola)</td>
<td>European Collection of Authenticated Cell Cultures (ECACC)</td>
<td>Cat# 8021401</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>5-Biotinylated oligonucleotide (, 5’-CTCATAGCTCACGCTGTA <br>GGTATCTCAGTTCGG-3’)</td>
<td>Merck</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>Proteomics database search engine (Proteome Discoverer 1.4 and Mascot v2.5 or newer)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>SAINT (Significance Analysis of INTeractome, SAINTexpress-spc)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>LC-MSMS acquisition software (Xcalibur v2.2 or newer)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Deposited Data</td>
</tr>
<tr>
<td>Dataset</td>
<td>PRIDE: PXD011670</td>
<td>Accesion number or file name</td>
</tr>
<tr>
<td>Chromatome ; EdU minus #1</td>
<td>PRIDE: PXD011670</td>
<td>141223_S_SALD_01_01_45pto.raw</td>
</tr>
<tr>
<td>Chromatome ; EdU minus #2</td>
<td>PRIDE: PXD011670</td>
<td>141223_S_SALD_05_01_45pto.raw</td>
</tr>
<tr>
<td>Chromatome ; EdU minus #3</td>
<td>PRIDE: PXD011670</td>
<td>141223_S_SALD_10_01_45pto.raw</td>
</tr>
<tr>
<td>Chromatome ; EdU minus #4</td>
<td>PRIDE: PXD011670</td>
<td>141223_S_SALD_13_01_45pto.raw</td>
</tr>
<tr>
<td>Chromatome ; EdU plus #1</td>
<td>PRIDE: PXD011670</td>
<td>141223_S_SALD_02_01_45pto.raw</td>
</tr>
<tr>
<td>Chromatome ; EdU plus #2</td>
<td>PRIDE: PXD011670</td>
<td>141223_S_SALD_06_01_45pto.raw</td>
</tr>
<tr>
<td>Chromatome ; EdU plus #3</td>
<td>PRIDE: PXD011670</td>
<td>141223_S_SALD_11_01_45pto.raw</td>
</tr>
<tr>
<td>Chromatome ; EdU plus #4</td>
<td>PRIDE: PXD011670</td>
<td>150727_S_SALD_14_01_45pt.raw</td>
</tr>
<tr>
<td colspan="3">Others</td>
</tr>
<tr>
<td>Acid-resistant centrifugal vacuum concentrator, CentriVap Benchtop with glass lid</td>
<td>LabConco</td>
<td>Cat# 7810036</td>
</tr>
<tr>
<td>Bioruptor</td>
<td>Diagenode</td>
<td>N/A</td>
</tr>
<tr>
<td>Cell lifter, 18-cm</td>
<td>Corning</td>
<td>Cat# 3008</td>
</tr>
<tr>
<td>Dynabeads™ M-280 streptavidin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11205D</td>
</tr>
<tr>
<td>DynaMag™-2 Magnet</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12321D</td>
</tr>
<tr>
<td>Eppendorf® Safe-Lock microcentrifuge tubes, 1.5 mL</td>
<td>Eppendorf</td>
<td>Cat# 0030120086</td>
</tr>
<tr>
<td>Eppendorf® Safe-Lock microcentrifuge tubes, 2-mL</td>
<td>Eppendorf</td>
<td>Cat# 0030120094</td>
</tr>
<tr>
<td>Eppendorf® Thermomixer Compact</td>
<td>Merck</td>
<td>N/A</td>
</tr>
<tr>
<td>Eppendorf™ 5424R Microcentrifuges</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 5424R</td>
</tr>
<tr>
<td>Eppendorf™ 5810R Centrifuge</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 5810R</td>
</tr>
<tr>
<td>LC system for online LC-MS analysis, Easy Nano liquid chromatography instrument EASY-nLC 1000</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Microcon-30 kDa Centrifugal Filter Unit with Ultracel-30 membrane</td>
<td>Merk Millipore</td>
<td>Cat# MRCF0R030</td>
</tr>
<tr>
<td>MS system for online LC-MS analysis, LTQ Orbitrap Velos Pro</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>NanoDrop 2000</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Nitrocellulose blotting membranes, pore size 0.2 μm</td>
<td>GE Healthcare Life Science</td>
<td>Cat# 10600006</td>
</tr>
<tr>
<td>Nylon hybond -N+ membranes</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 45-000-850</td>
</tr>
<tr>
<td>Polypropylene centrifuge tube, 50 mL</td>
<td>Sarstedt</td>
<td>Cat# 62547004</td>
</tr>
<tr>
<td>Polystyrene centrifuge tube, 15 mL</td>
<td>Corning</td>
<td>Cat# 352095</td>
</tr>
<tr>
<td>Roller mixer</td>
<td>Stuart</td>
<td>N/A</td>
</tr>
<tr>
<td>Rotator</td>
<td>Stuart</td>
<td>N/A</td>
</tr>
<tr>
<td>Standard equipment for mammalian cell culture</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Stirrer</td>
<td>Stuart</td>
<td>N/A</td>
</tr>
<tr>
<td>Stratagene Stratalinker 1800</td>
<td>Stratagene</td>
<td>N/A</td>
</tr>
<tr>
<td>UltraMicrospin C18, 300 A silica</td>
<td>The Nest Group Inc.</td>
<td>Cat# SUM SS18V</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<h3>Reagent Setup</h3>
<ul>
<li>
<b>Alexa FlourTM 647 azide</b>: Dilute 0.5 mg of Alexa FlourTM 647 azide with 58.1-mL DMSO. Freeze aliquots and store at –20°C for up to 1 year. Avoid freeze/thaw cycles.</li>
<li>
<b>50 mM Ammonium bicarbonate:</b> Dissolve 39.6 mg of NH4HCO3 in 10 mL of LCMS grade water.</li>
<li>
<b>0.5 M Ascorbic acid</b>: Dissolve 8.8 g L-ascorbic acid in 100-mL dH2O. Store in aliquots at −20°C for up to 1 year.</li>
<li>
<b>10 mM Biotin azide</b>: Dissolve 1 mg Biotin Azide in 162,5 uL DMSO. Store in aliquots at −20°C for up to 1 year.</li>
<li>
<b>1 M CuSO</b><sub><b>4</b></sub>: Dissolve 24.9 g copper(II) sulfate pentahydrate in 100 mL dH<sub>2</sub>O. Store at 4°C for up to 6 months.</li>
</ul>
<p><b><i>Caution:</i></b> <b>CuSO</b><sub><b>4</b></sub> can cause skin, eye and respiratory irritation. Harmful for aquatic environment. Prepare the solution under a fume hood using gloves.</p>
<ul>
<li>
<b>0.5 M EDTA (pH 8.0)</b>: Dissolve 186.1 g EDTA disodium salt dihydrate with 900 mL dH<sub>2</sub>O. Adjust pH with NaOH. Store at room temp for up to 6 months.</li>
</ul>
<p><b><i>Caution:</i></b> This solution can cause eye irritation upon contact.</p>
<ul>
<li>
<b>25 mM EdU</b>: Dilute 50 mg of EdU in 7.9-mL PBS (pH 7.4). Dilution is achieved by heating at 37°C. Freeze aliquots and store at –20°C for up to 1 year. Avoid freeze/thaw cycles.</li>
</ul>
<p><b><i>Caution:</i></b> Prepare the solution under a fume hood using gloves. Dilute the entire amount of EdU, and cover your face with a safety mask.</p>
<ul>
<li>
<b>5% Formic acid</b>: Mix 5% formic acid in LCMS-grade water. Store at 25°C for up to 1 month.</li>
</ul>
<p><b><i>Caution:</i></b> Formic acid is highly corrosive to eyes, skin and the respiratory system. Prepare the solution under a fume hood using gloves.</p>
<ul>
<li>
<b>5% Formic acid in 50% acetonitrile</b>: Mix 5 % formic acid in a 1:1 mix of LC-MS-grade acetonitrile and water solution. Store at 25°C for up to 1 month</li>
</ul>
<p><b><i>Caution:</i></b> Formic acid is highly corrosive to eyes, skin and the respiratory system. Prepare the solution under a fume hood using gloves. Acetonitrile is flammable and toxic.</p>
<ul>
<li>
<b>16% Formaldehyde fixation solution (wt/vol)</b>: Dilute 0.17 g of Na<sub>2</sub>CO<sub>3</sub> in 80-mL dH<sub>2</sub>O at 60°C. While stirring at 60°C, add 16 g of paraformaldehyde. Cover with aluminium foil and continue stirring at 60°C until clarification. Remove from heat, cool-down on ice and filter. Use immediately, or freeze aliquots at –80°C and store for up to 1 year. Avoid repeated freeze/thaw cycles.</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> Do not heat solution above 65°C. Paraformaldehyde breaks down above this temperature.</div>
<ul>
<li>
<b>8% (wt/vol) Formaldehyde fixation solution</b>: For 10 mL, dilute 5-mL formaldehyde fixation solution 16% (wt/vol) into 4 mL dH<sub>2</sub>O and 1 mL of 10×PBS (pH 6.7). Prepare fresh every time.</li>
</ul>
<p><b><i>Caution:</i></b> Formaldehyde is toxic if swallowed, or upon contact to skin, eyes or respiratory system. Prepare the solution under a fume hood using gloves. To weight the paraformaldehyde, cover your nose with safety mask.</p>
<ul>
<li>
<b>80% (vol/vol) Glycerol</b>: Dilute glycerol to 80% (vol/vol) in dH<sub>2</sub>O. Store at 4°C for up to 1 month.</li>
<li>
<b>1.25 M Glycine</b>: Dilute 9.4 g of glycine in 100-mL 1× PBS (pH 7.4). Store the solution at 4°C for up to 1 month.</li>
<li>
<b>1 M HEPES (pH 7.4)</b>: Dilute 238.3 g HEPES in 900 mL dH<sub>2</sub>O. Adjust the pH and bring the volume up to 1 L with dH<sub>2</sub>O. Store at 4°C for up to 6 months.</li>
<li>
<b>1 M KCl</b>: Dissolve 74.55 g of KCl in 1 L of H<sub>2</sub>O. Store at 25°C for up to 1 year.</li>
<li>
<b>1 M MgCl</b><sub><b>2</b></sub>: Dissolve 203.3 g of magnesium chloride hexahydrate in 1 L H<sub>2</sub>O. Store at 25°C for up to 1 month.</li>
<li>
<b>5 M NaCl</b>: Dissolve 292.2 g NaCl in 1 L dH<sub>2</sub>O. Store at 25°C for up to 6 months.</li>
<li>
<b>10× PBS (pH 6.7)</b>: Dissolve 80 g NaCl, 2 g KCl, 2.4 g KH<sub>2</sub>PO<sub>4</sub> and 14.4 g Na<sub>2</sub>HPO<sub>4</sub>·2H<sub>2</sub>O. Store at 4°C for up to 4 month.</li>
<li>
<b>1× PBS (pH 7.4)</b>: Dissolve 8 g NaCl, 0.2 g KCl, 0.24 g KH2PO4, and 1.44 g Na<sub>2</sub>HPO<sub>4</sub>·2H<sub>2</sub>O. Store at 4°C for up to 1 month.</li>
<li>
<b>25× Protease cocktail inhibitors (PIC)</b>: Dissolve one tablet of PIC in 2 mL dH<sub>2</sub>O. Store in aliquots at −20°C for up to 3 months. Avoid freeze/thaw cycles.</li>
<li>
<b>Salmon-sperm DNA (ssDNA) stock solution</b>: Dissolve 75 mg salmon sperm DNA with 1 mL dH2O. Sonicate for 10 minutes in a Bioruptor at 4°C at high intensity (30 seconds ON/30 seconds OFF). Prepare freshly every time.</li>
<li>
<b>10% (wt/vol) SDS:</b> Dissolve 100 g SDS in 1 L dH<sub>2</sub>O.</li>
</ul>
<p><b><i>Caution:</i></b> SDS is inflammable and toxic if swallowed, and causes skin, eye and respiratory irritation. Harmful for aquatic environment. Prepare the solution under a fume hood using gloves. To weigh the SDS, wear a safety mask.</p>
<ul>
<li>
<b>10% (wt/vol) Sodium deoxycholate:</b> Dissolve 10 g sodium deoxycholate in 100 mL dH<sub>2</sub>O. Store at 4°C for up to 6 months.</li>
</ul>
<p><b><i>Caution:</i></b> Harmful if swallowed. Prepare the solution under a fume hood using gloves.</p>
<ul>
<li>
<b>200 mM Sodium orthovanadate (Na</b><sub><b>3</b></sub><b>VO</b><sub><b>4</b></sub><b>)</b>: Dissolve 3.6 g Na<sub>3</sub>VO<sub>4</sub> in 90-mL dH<sub>2</sub>O. Adjust the 200 mM stock solution to pH 10, at which point the solution should be yellow. Boil for 10 min to clarify. Cool to 25°C and readjust to pH 10. If required, repeat cycles of boiling and pH adjustment until the solution remains clear at pH 10. Store in aliquots at −20°C for up to 1 year.</li>
</ul>
<p><b><i>Caution:</i></b> Harmful if swallowed and by skin contact. Prepare the solution under a fume hood using gloves.</p>
<ul>
<li>
<b>1 M Sucrose</b>: Dissolve 34.2 g sucrose in 80-mL dH<sub>2</sub>O at 40°C. Cool down and then bring the volume up to 100 mL with dH<sub>2</sub>O. Store at 4°C for up to 1 month.</li>
<li>
<b>1 μg/μL Trypsin</b>: Dissolve 20 μg trypsin in 20 μL of ammonium bicarbonate (50 mM). Prepare fresh every time.</li>
<li>
<b>1 M Tris-HCl</b>: Dissolve 121 g Trizma base in 900-mL dH<sub>2</sub>O. Adjust pH to 8.0 or to 7.4. Adjust the volume to 1 L using dH<sub>2</sub>O. Store at 4°C for up to 6 months.</li>
</ul>
<p><b><i>Caution:</i></b> HCl is highly corrosive to the eyes, skin and the respiratory system. Prepare the solution under a fume hood using gloves.</p>
<ul>
<li>
<b>0.1 M Tris-HCl</b>: Dissolve 12.1 g Trizma base into 900 mL dH<sub>2</sub>O. Adjust to pH 8.8. Adjust the volume to 1 L using dH<sub>2</sub>O. Store at 4°C for up to 6 months.</li>
</ul>
<p><b><i>Caution:</i></b> HCl is highly corrosive to the eyes, skin and the respiratory system. Prepare the solution under a fume hood using gloves.</p>
<ul>
<li>
<b>10% (vol/vol) Triton X-100:</b> Dilute Triton X-100 in dH<sub>2</sub>O. Store at 4°C for up to 6 month.</li>
</ul>
<p><b><i>Caution:</i></b> Harmful in case of eye contact, ingestion or inhalation. Harmful for aquatic environment.</p>
<ul>
<li>
<b>1 M β-Glycerol phosphate stock solution</b>: Dissolve 30.6 g β-glycerol phosphate disodium salt pentahydrate in H<sub>2</sub>O, adjusting the volume to 100 mL. Store at 4°C for up to 1 month.</li>
<li>
<b>Alkylation buffer:</b> 18.5 mg of iodoacetamide in 2 mL of 0.1M Tris-HCl.</li>
</ul>
<p><b><i>Caution:</i></b> Iodoacetamide is toxic.</p>
<ul>
<li>
<b>Blocking beads solution</b>: 10 mg/mL ssDNA in PBS.</li>
<li>
<b>Click reaction mix with Alexa-azide</b>: 100 mM Tris-HCl (pH 8.0), 2 mM CuSO<sub>4</sub>, 0.01 mM Alexa FlourTM 647 azide and (added immediately before use) 100 mM ascorbic acid.</li>
</ul>
<p><b><i>Caution:</i></b> CuSO<sub>4</sub> is toxic by ingestion and causes skin and eye irritation.</p>
<ul>
<li>
<b>Click reaction mix with biotin-azide</b>: 100 mM Tris-HCl (pH 8.0), 2 mM CuSO<sub>4</sub>, 0.2 mM biotin-azide and (added immediately before use) 100 mM L-ascorbic acid.</li>
<li>
<b>Dilution buffer</b>: 1% Triton X-100, 2 mM EDTA (pH 8), 150 mM NaCl, 20 mM Tris-HCl (pH 8), 20 mM β-glycerol phosphate, 2 mM sodium orthovanadate, 1× PIC and 5 mg/mL salmon-sperm DNA.</li>
<li>
<b>Elution buffer</b>: 2% SDS, 0.06 M Tris-HCl (pH 6.5) and 0.1 M DTT.</li>
<li>
<b>Fragmentation buffer</b>: 20 mM Tris-HCl (pH 7.4), 150 mM NaCl, 1 mM EDTA, 0.1% SDS, 0.5% sodium deoxycolate, 1% Triton X-100, 10 mM β-glycerol phosphate, 1 mM sodium orthovanadate and 1× PIC.</li>
<li>
<b>High-salt washing buffer</b>: 1% Triton X-100, 2 mM EDTA, 500 mM NaCl, 20 mM Tris-HCl (pH 8).</li>
<li>
<b>PBS-PIC</b>: Dissolve one PIC tablet in 50 mL PBS. Store at 4°C for up to 3 days.</li>
<li>
<b>Permeabilization buffer</b>: 10 mM HEPES pH 7.9, 10 mM KCl, 1.5 mM MgCl2, 0.34 M sucrose, 10% [v/v] glycerol, 1 mM DTT, 10 mM β-glycerol phosphate, 1 mM sodium orthovanadate, 1× PIC and 0.1% (vol/vol) Triton X-100. Prepare fresh every time.</li>
<li>
<b>Reduction buffer:</b> 15.4 mg DTT in 1 mL of 0.1 M Tris-HCl.</li>
<li>
<b>1× SDS-lysis buffer</b>: For 1 mL, mix 0.5 mL 2× SDS-lysis buffer, 0.02 mL of 1 M β-glycerol phosphate, 0.01 mL of 200 mM sodium orthovanadate and 0.02 mL of 25× PIC; adjust the volume to 1 mL with dH<sub>2</sub>O and use immediately.</li>
<li>
<b>2× SDS-lysis buffer</b>: For 100 mL, 5 mL of 1 M Tris-HCl (pH 7.4), 20 mL of 10% (wt/vol) SDS and 0.4 mL of 0.5 M EDTA; adjust the volume to 100 mL with dH<sub>2</sub>O. Keep the solution at 25°C for up to 6 months.</li>
<li>
<b>Serum-containing ESC-media (SR-LIF)</b>: Mix knockout DMEM, 20% knockout serum replacement, 1% MEM non-essential amino acids, 1% glutamax, 1% HEPES and 0.1% 2-mercaptoethanol, supplemented with 1000 U/mL LIF. Keep the solution at 4°C for up to 2 weeks.</li>
<li>
<b>Serum-free ESC-media (2iLIF)</b>: Mix DMEM/F12:Neurobasal 1:1, 0.5% N-2 supplement, 1% B-27 serum-free supplement, 0.1% 2-mercaptoethanol, 0.033% BSA V, 1% glutamax, 1% MEM non-essential amino acids and 1% penicillin-streptomycin. Supplement with 1 μM PD0325901, 3 mM CHIR99021 and 1000 U/mL LIF. Keep the solution at 4°C for up to 2 weeks.</li>
<li>
<b>2 M Urea-T:</b> 1.2 g urea in 10 mL of 0.1 M Tris-HCl. Prepare fresh every time.</li>
<li>
<b>6 M Urea-T:</b> 3.6 g urea in 10 mL of 0.1 M Tris-HCl. Prepare fresh every time.</li>
<li>
<b>8 M Urea-T</b>: 4.8 urea in 10 mL of 0.1 M Tris-HCl. Prepare fresh every time.</li>
<li>
<b>TBS-T buffer</b>: 10 mM Tris-HCl (pH 7.5), 100 mM NaCl, 0.1% Tween 20. Store at 25°C for up to 1 month.</li>
<li>
<b>1× TE</b>: 10 mM Tris-HCl (pH 7.5), 1 mM EDTA (pH 8.0). Store at 25°C for up to 1 month.</li>
<li>
<b>Washing buffer</b>: 1% Triton X-100, 2 mM EDTA, 150 mM NaCl and 20 mM Tris-HCl (pH 8). Prepare fresh every time.</li>
</ul>
<h4>LC-MS/MS Setup</h4>
<ul>
<li>This protocol uses an LC-MS/MS set up with a UPLC Proxeon EASY-nLC 1000 coupled to an LTQ Orbitrap Velos Pro mass spectrometer, though other LC-MS systems with similar capabilities can be used.</li>
<li>We use a 25-cm reverse-phase chromatographic column with an inner diameter of 75 μm, packed with 1.9 μm C18 particles (Nikkyo Technos Co., Ltd. Japan) for peptide analysis.</li>
<li>Chromatographic gradients start at 97% buffer A / 3% buffer B and increases to 65% buffer A / 35% buffer B over 120 min at a flow rate of 250 nL/min; buffer A is water + 0.1% formic acid (LCMS grade), and buffer B is acetonitrile + 0.1% formic acid (LCMS grade) (see <a href="#tbl1">Table 1</a>).</li>
</ul>
<table id="tbl1">
<caption>Table 1. UPLC-MS Reverse Phase Chromatography Gradient</caption>
<thead>
<tr>
<th>Time Interval (min)</th>
<th>Gradient (% B)</th>
<th>Flow Rate (nL/min)</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>3</td>
<td>250</td>
</tr>
<tr>
<td>5</td>
<td>3</td>
<td>250</td>
</tr>
<tr>
<td>125</td>
<td>35</td>
<td>250</td>
</tr>
<tr>
<td>126</td>
<td>90</td>
<td>250</td>
</tr>
<tr>
<td>141</td>
<td>90</td>
<td>250</td>
</tr>
</tbody>
</table>
<ul>
<li>Eluted peptides are directly introduced into the mass spectrometer by nano electro-spray ionization.</li>
<li>The mass spectrometer is operated in DDA mode, and full MS scans with 1 microscan at a resolution of 60,000 are used over a mass range of m/z 350–2,000, with detection in the Orbitrap. Following each survey scan, the twenty most intense ions with multiple charged ions above a threshold ion count of 5,000 are selected for fragmentation at a normalized collision energy of 35%. Fragment ion spectra produced via collision-induced dissociation (CID) are acquired in the linear ion trap. Data are acquired with Xcalibur software v2.2 (see <a href="#tbl2">Table 2</a>).</li>
</ul>
<table id="tbl2">
<caption>Table 2. LTQ Orbitrap Velos Pro Acquisition Method</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>Polarity</td>
<td>Positive</td>
</tr>
<tr>
<td colspan="2">Full MS Scans</td>
</tr>
<tr>
<td>Mass analyser</td>
<td>Orbitrap</td>
</tr>
<tr>
<td>Orbitrap resolution</td>
<td>60,000</td>
</tr>
<tr>
<td>Scan range (m/z)</td>
<td>350–2,000</td>
</tr>
<tr>
<td>Automatic Gain Control (AGC) target</td>
<td>1 × 10<sup>6</sup>
</td>
</tr>
<tr>
<td>Microscans</td>
<td>1</td>
</tr>
<tr>
<td colspan="2">Dependent MS/MS Scans</td>
</tr>
<tr>
<td>Number of dependent scans</td>
<td>20 (from most intense precursors)</td>
</tr>
<tr>
<td>Mass analyzer</td>
<td>Ion trap</td>
</tr>
<tr>
<td>Scan rate</td>
<td>Normal</td>
</tr>
<tr>
<td>Mass range</td>
<td>Normal</td>
</tr>
<tr>
<td>Automatic Gain Control (AGC) target</td>
<td>1 × 10<sup>4</sup>
</td>
</tr>
<tr>
<td>Microscans</td>
<td>1</td>
</tr>
<tr>
<td>Activation type</td>
<td>Collision-induced dissociation</td>
</tr>
<tr>
<td>Normalized collision energy (%)</td>
<td>35</td>
</tr>
<tr>
<td>Charge exclusion</td>
<td>Exclude +1 charge state</td>
</tr>
<tr>
<td>Dynamic exclusion</td>
<td>60 s</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Instrument performance is assessed by acquiring a quality control (digested bovine serum albumin) after every sample injection.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<p>The main procedure is divided into seven steps (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/61-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Formaldehyde Cross-Link</div>
</figcaption>
</figure>
<h3>EdU Labeling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This step describes EdU incorporation into the genome of the cells.</p>
<ul>
<li>Prepare an intermediate dilution of EdU at 100 μM in ESC media from the EdU stock solution. This is a 1000× EdU solution for ESCs.</li>
</ul>
<p><b><i>Caution:</i></b> Prepare the solution in the sterile tissue culture hood and use gloves.</p>
<ul>
<li>Change media from exponentially growing ESCs cultures that have been split one day in advance (2 x 10<sup>5</sup> cells/mL at initial concentration), by adding 25 mL (or the corresponding volume of media) of fresh media with or without 0.1-μM EdU. Incubate the cells for 20 h.</li>
</ul>
<h3>Cell Fixation and Permeabilization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–1.5 h</div>
<p>This step describes the chemical crosslinking of chromatin interacting molecules and permeabilization of cellular membrane.</p>
<ol start="3">
<li>Add 1.5-mL formaldehyde fixation solution 16% (wt/vol) to the growing cells, by adding it directly to the media. Mix gently and then incubate for 10 min at 25°C. Note that the ESC media will turn slightly yellow (<a href="#fig1">Figure 1</a>).</li>
</ol>
<p><b><i>Caution:</i></b> Add the solution under a fume hood and use gloves.</p>
<ol start="4">
<li>Add 2.5-mL glycine solution (1.25 M) directly to the fixed cells to quench the formaldehyde. Mix gently and then incubate for 5 min at 25°C. Note that the ESC media will turn even more yellow (<a href="#fig1">Figure 1</a>).</li>
<li>Discard the media from the dishes.</li>
</ol>
<p><b><i>Caution:</i></b> Fixation solution is harmful for both researchers and the environment. Dispense the fixation solution in an appropriated recipient.</p>
<ol start="6">
<li>Wash the attached cells three times with PBS 1× (pH 7.4).</li>
<li>Add 8 mL of ice-cold PBS-PIC solution, harvest the cells with a cell lifter and collect the cells into a 15-mL polystyrene centrifuge tube. Add 4 mL of additional ice-cold PBS-PIC to the dish to recover the remaining cells.</li>
<li>Pellet the cells by centrifugation in a swing-bucket rotor at 1,300<i>g</i> for 4 min at 4°C.</li>
<li>Suspend the cells with 2-mL ice-cold PBS-PIC and count them.</li>
</ol>
<p>See <a href="#troubleshooting">Troubleshooting</a></p>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The cell pellets can be stored at this point at –80°C up to 1 year.</div>
<ol>
<li>Dilute 2-4 × 10<sup>7</sup> cells with 5-mL permeabilization buffer. Incubate cells for 30 min at 4°C with gentle rotation.</li>
<li>Collect the permeabilized cells by centrifugation in a swing-bucket rotor at 1,300<i>g</i> for 4 min at 4°C.</li>
</ol>
<h3>Click Reaction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60 min</div>
<p>This step describes the copper-catalyzed azide-alkyne cyclo-addition with Biotin azide and EdU via a click reaction.</p>
<ol>
<li>Prepare the Click reaction mix with biotin azide.</li>
<li>Wash the cells with 2-mL ice-cold PBS-PIC and then collect them by centrifugation in a swing-bucket rotor at 1,300<i>g</i> for 4 min at 4°C.</li>
<li>Suspend the cells in 1-mL ice-cold PBS-PIC and transfer the suspension into a 1.5-mL Eppendorf tube. Place the Eppendorf tube on top of an opened 15-mL centrifuging tube, and collect the cells by centrifugation in a swing-bucket rotor at 1,300<i>g</i> for 4 min at 4°C.</li>
<li>Suspend the cells in 1 mL of the Click reaction mix, and incubate them for 30 min-1h at 25°C with gentle rotation.</li>
</ol>
<h3>Sonication</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5–2 h</div>
<p>This step describes ultrasound-mediated cell lysis and chromatin shearing.</p>
<ol start="16">
<li>Place the Eppendorf tube on top of an opened 15-mL centrifuging tube, and collect the cells by centrifugation in a swing-bucket rotor at 1,300<i>g</i> for 4 min at 4°C.</li>
<li>Suspend the cells in 1-mL ice-cold PBS-PIC, and transfer the cell suspension to a 15-mL polystyrene centrifuge tube. Collect the cells by centrifugation in a swing-bucket rotor at 1,300<i>g</i> for 4 min at 4°C.</li>
<li>Wash the cell pellet with 2-mL additional of ice-cold PBS-PIC, and collect the cells by centrifugation in a swing-bucket rotor at 1,300<i>g</i> for 4 min at 4°C.</li>
<li>Suspend the cells in 4-mL fragmentation buffer and then incubate them 10–30 min on ice.</li>
<li>Fragment the cells with a Bioruptor sonicator using 40 cycles of 30 seconds ON and 30 seconds OFF at high intensity.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Mix well the cell suspension right before the first sonication cycle. In order to maintain the temperature of the water bath, follow the manufacturer’s recommendation if your sonicator is not coupled to a refrigeration system.</div>
<ol start="21">
<li>Transfer each sample into two 1.5-mL Eppendorf tubes, and centrifuge them at 15,000<i>g</i> for 10 min at 4°C in a tabletop centrifuge.</li>
<li>Collect the cleared samples into a single 15-mL tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Chromatin fragmentation is critical for optimal streptavidin capturing. DNA fragmentation and EdU labelling must be monitored, as indicated in the section on Monitoring DNA Fragmentation before moving on to the next step. Note that the timing to monitor of DNA fragmentation is 1 day.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Fragmented samples can be stored at –80°C.</div>
<h3>Streptavidin Affinity Purification and Elution</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4.5–5 h</div>
<p>This step describes genomic capture by streptavidin-biotin affinity purification, reverse cross-linking and chromatin elution from streptavidin beads.</p>
<p>The beads must be pre-blocked in advance, as follows:</p>
<div class="critical">
<span class="critical-title">Critical:</span> Do not let the beads dry.</div>
<ol start="23">
<li>Pipette 0.5-mL streptavidin magnetic beads M280 into one 1.5-mL Eppendorf tube for each chromatin purification.</li>
<li>Collect beads using the DynaMag™-2 Magnet for 1 minute. Discard the supernatant, and add 1-mL PBS 1× (pH 7.4).</li>
<li>Repeat the previous washing step (Step 24).</li>
<li>Add 1-mL PBS containing 10 mg/mL ssDNA.</li>
<li>Incubate the samples for 1 h at 25°C with gentle agitation in a rotator.</li>
<li>Collect the beads using the DynaMag™-2 Magnet. Discard the supernatant, and dilute them in 0.5-mL dilution buffer. The beads can be prepared 1 day in advance and stored at 4°C.</li>
<li>Dilute the fragmented samples by adding four times the volume with dilution buffer, and add the blocked beads.</li>
<li>Incubate samples for 2 h at 25°C with gentle agitation in a rotator.</li>
<li>Aliquot suspension into 1.5-mL Eppendorf tubes, collect the beads with the DynaMag™-2 Magnet, and discard the flow-through.</li>
<li>Collect the aliquoted beads from the same captured sample by pooling them with 1-mL washing buffer into a single 1.5-mL Eppendorf tube.</li>
<li>Wash the beads twice by adding 1-mL washing buffer, mixing the tube upside-down, and collect the beads with the DynaMag™-2 Magnet.</li>
<li>Repeat Step 33 once, only using 1-mL high-salt washing buffer.</li>
<li>Repeat Step 33 once, only using 1-mL TE1× buffer.</li>
<li>Suspend the beads in 200-μL TE1× buffer, and collect the beads with the DynaMag™-2 Magnet.</li>
<li>Suspend the beads in 120-μL elution buffer. Boil the bead suspension for 20 min at 95°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> After boiling for 10 min, mix the bead suspension by pipetting up and down five times.</div>
<ol start="38">
<li>Collect the supernatant after precipitating the beads with the DynaMag™-2 Magnet.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples can be stored at –20°C for up to 1 year.</div>
<h3>Sample Preparation for Mass Spectrometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This step describes protein digestion with endopeptidase Lys-C and trypsin, and sample clean-up for mass spectrometric analysis.</p>
<ol start="39">
<li>Reduce the supernatants by adding 13-μL reduction buffer and incubating for 30 min at 60°C with constant agitation.</li>
<li>Add 284-μL urea-T (8M) to reach a 5.5 M final concentration of urea.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples can be stored 12–16 h at 4°C and equilibrate at 25°C before the use.</div>
<ol start="41">
<li>Pre-soak the 30-kDa centrifugal filters with 200-μL LCMS-grade water and centrifuge at 15,000<i>g</i> for 10 min. Repeat twice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not let the membranes of the centrifugal filters dry out during this step or the upcoming washing steps.</div>
<ol start="42">
<li>Add the reduced supernatant to the centrifugal filters.</li>
<li>Add 100-μL 6M urea-T to the centrifugal filters and centrifuge at 15,000<i>g</i> for 10 min. Discard the flow-through. Repeat twice.</li>
<li>Alkylate the samples by adding 100-μL alkylation buffer to the centrifugal filters, and agitate (600 rpm) in the dark at 25 °C for 30 min.</li>
<li>Centrifuge the filters at 15,000<i>g</i> for 10 min, and discard the flow-through.</li>
<li>Add 100-μL 2M urea-T to the centrifugal filters and centrifuge at 15,000<i>g</i> for 10 min. Discard the flow-through. Repeat twice.</li>
<li>Add 50-μL 50 mM ammonium bicarbonate to the centrifugal filters.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure the filter membrane is completely covered.</div>
<ol start="48">
<li>Add 1-μL Lys-C (1 μg/μL)–TEAB (50 mM), mix for 1 min at 650 rpm, place the centrifugal filters into a recipient with high humidity and digest 12-16 h at 37°C.</li>
<li>Add 1 μl of trypsin (1 μg/μL), mix for 1 min at 650 rpm, place the centrifugal filters into a recipient with high humidity and digest for 8 h at 37 °C.</li>
<li>Elute the digested peptides in clean collecting tubes from the filters by centrifuging at 15,000<i>g </i> for 10 min. Keep the flow-through.</li>
<li>Add 50-μL 50 mM ammonium bicarbonate to the filters and centrifuge at 15,000<i>g</i> for 10 min. Keep the flow-through in the same collector tubes.</li>
<li>Stop the reaction by adding ∼20-μL 5% formic acid (final pH ∼2.5).</li>
</ol>
<p><b><i>Caution:</i></b> Formic acid is highly corrosive to the eyes, skin and the respiratory system. Use a fume hood using gloves.</p>
<ol start="53">
<li>Wash the UltraMicrospin C18 columns by adding 200-μL LCMS-grade methanol and centrifuge at 200<i>g</i> for 5 min. Use one C18 column per sample. Discard the flow-through. Repeat twice.</li>
</ol>
<p><b><i>Caution:</i></b> Methanol is toxic and flammable.</p>
<ol start="54">
<li>Equilibrate the C18 columns by adding 200-μL 5% formic acid and centrifuge at 200<i>g</i> for 5 min. Discard the flow-through. Repeat twice.</li>
<li>Load the samples into the C18 columns and centrifuge at 200<i>g</i> for 5 min.</li>
<li>Re-load the samples into the C18 columns (by adding the flow-through of the previous step into the C18 column). Centrifuge at 200<i>g</i> for 5 min. Discard the flow-through.</li>
<li>Wash the C18 columns by adding 200-μL 5% formic acid, and centrifuge at 200<i>g</i> for 5 min. Use one C18 column per sample. Discard the flow-through. Repeat twice.</li>
</ol>
<p><b><i>Caution:</i></b> Formic acid is highly corrosive to the eyes, skin and the respiratory system. Use a fume hood using gloves.</p>
<ol start="58">
<li>Elute the peptides from the C18 columns by adding 200-μL 5% formic acid in 50% acetonitrile, and centrifuge at 200<i>g</i> for 5 min. Keep the flow-through. Repeat twice and combine the peptides of the previous elution.</li>
<li>Evaporate the solvent to dryness with an acid-resistant centrifugal vacuum concentrator.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples can be stored at –20°C before analysis.</div>
<ol start="60">
<li>Resuspend the samples in 10-μL 0.1% formic acid.</li>
</ol>
<h3>Mass Spectrometry Analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3-4 h</div>
<p>This step describes sample data acquisition by LC-MS and computational analysis to identify bona fide chromatin-associated proteins.</p>
<ol start="61">
<li>Analyse 4.5 μL of each sample in the LC-MS/MS system using the parameters described in <a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a> from the Equipment setup section.</li>
<li>Analyse the acquired MS data with the proteomics database search engine (e.g. Proteome Discoverer 1.4 with Mascot v2.5) using the parameters described in the table.</li>
</ol>
<table>
<thead>
<tr>
<th>Parameter</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">Variable modifications</td>
<td>Oxidation (M)</td>
</tr>
<tr>
<td>Acetyl (protein N-terminal)</td>
</tr>
<tr>
<td>Fixed modifications</td>
<td>Carbamidomethyl (C)</td>
</tr>
<tr>
<td>Enzyme</td>
<td>Trypsin</td>
</tr>
<tr>
<td>Maximum missed cleavages</td>
<td>3</td>
</tr>
<tr>
<td>Precursor mass tolerance (MS1)</td>
<td>7 ppm</td>
</tr>
<tr>
<td>Product ion mass tolerance (MS2)</td>
<td>0.5 Da</td>
</tr>
<tr>
<td>FDR Control strategy</td>
<td>Decoy database</td>
</tr>
<tr>
<td>FDR cut-off</td>
<td>5% (peptide level)</td>
</tr>
</tbody>
</table>
<ol start="63">
<li>Export the results in text format or as a spreadsheet to prepare the input tables for SAINT (Significance Analysis of INTeractome) analysis as described in the Experimental Setup section.</li>
<li>Prepare the <code>interaction.txt</code>, <code>prey.txt</code> and <code>bait.txt</code> input files for SAINT analysis.</li>
<li>Run the statistical assessment with SAINT analysis using the command: <code>SAINTexpress-spc interaction.txt
                prey.txt bait.txt.</code>
</li>
<li>Open the SAINT results file (<code>list.txt</code>) with a text editor or a spreadsheet software, and filter the list by BFDR &lt;1% to retrieve the <i>bona fide</i> chromatin-associated proteins.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>The protocol described here enables the user to profile the chromatin composition in a simple and robust manner. Our iPOTD procedure is compatible with conventional ChIP analysis, thereby allowing follow-up analyses of the genomic occupancy to be performed to identify with the chromatin binders identified. In the supporting publication, we used this technology to profile the chromatin composition of ESCs. We characterized the one-carbon metabolic enzyme AHCY as a chromatin binder (<a href="#bib1">Aranda et al., 2019</a>). We have shown that the biochemical procedure to capture the chromatin can be used in adherent growing cells as well as in suspension, indicating that the iPOTD procedure can be applied efficiently to other cell types (<a href="#bib1">Aranda et al., 2019</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and Statistical Analysis</h2>
<p>The statistical evaluation of specific chromatin protein interactors is carried out using the SAINT software and, more specifically, using the SAINTexpress implementation (<a href="#bib3">Choi et al., 2011</a>; <a href="#bib4">Teo et al., 2014</a>). This analysis requires SAINTexpress-spc to be installed on a computer running Linux or Windows. Both the source code and pre-compiled binaries are available for download (at <a href="https://sourceforge.net/projects/saint-apms/files/">https://sourceforge.net/projects/saint-apms/files/</a>). Other flavours of the SAINT software can be found at <a href="http://saint-apms.sourceforge.net/Main.html">http://saint-apms.sourceforge.net/Main.html</a>.</p>
<p>To run the SAINT software, the user will need to prepare three input text files from the LC-MSMS identification results: the <code>bait.txt</code>, the <code>prey.txt</code> and the <code>interaction.txt</code> files.</p>
<p>The <code>bait.txt</code> file should have three tab-separated columns as shown below (but without the header): IP name (your favourite name), bait name or equivalent, and the indicator for test and negative control purifications (T, test; C, negative control).</p>
<table>
<thead>
<tr>
<th>IP Name</th>
<th>Bait</th>
<th>Type</th>
</tr>
</thead>
<tbody>
<tr>
<td>EXP1</td>
<td>CHROM</td>
<td>T</td>
</tr>
<tr>
<td>EXP2</td>
<td>CHROM</td>
<td>T</td>
</tr>
<tr>
<td>EXP3</td>
<td>CHROM</td>
<td>T</td>
</tr>
<tr>
<td>CTL1</td>
<td>CHROM</td>
<td>C</td>
</tr>
<tr>
<td>CTL2</td>
<td>CHROM</td>
<td>C</td>
</tr>
<tr>
<td>CTL3</td>
<td>CHROM</td>
<td>C</td>
</tr>
</tbody>
</table>
<p>The <code>prey.txt</code> file should contain three tab-separated columns as shown below (but without the header): i) prey (protein) accession; ii) prey protein length, indicated by the number of amino acids; and iii) a prey gene or protein name. The file should contain the information for all proteins identified in the experiment.</p>
<table>
<thead>
<tr>
<th>Prey Accession</th>
<th>Length (aa)</th>
<th>Name</th>
</tr>
</thead>
<tbody>
<tr>
<td>P50247</td>
<td>432</td>
<td>AHCY</td>
</tr>
<tr>
<td>P23198</td>
<td>183</td>
<td>CBX3</td>
</tr>
<tr>
<td>P20263</td>
<td>352</td>
<td>POU5F1</td>
</tr>
</tbody>
</table>
<p>The <code>interaction.txt</code> file should contain four tab-separated columns as shown below (but without the header): i) IP name, ii) bait name or equivalent, ii) prey accession and iv) total number of spectral counts per prey. The prey accession type should coincide with the first column of the prey file. Interactions with zero spectral counts should be removed from the file.</p>
<table>
<thead>
<tr>
<th>IP Name</th>
<th>Bait</th>
<th>Prey Accession</th>
<th>Spectral Counts</th>
</tr>
</thead>
<tbody>
<tr>
<td>CTL1</td>
<td>CHROM</td>
<td>P50247</td>
<td>1</td>
</tr>
<tr>
<td>CTL1</td>
<td>CHROM</td>
<td>P23198</td>
<td>3</td>
</tr>
<tr>
<td>CTL1</td>
<td>CHROM</td>
<td>P20263</td>
<td>1</td>
</tr>
<tr>
<td>CTL2</td>
<td>CHROM</td>
<td>P50247</td>
<td>1</td>
</tr>
<tr>
<td>CTL2</td>
<td>CHROM</td>
<td>P23198</td>
<td>2</td>
</tr>
<tr>
<td>CTL2</td>
<td>CHROM</td>
<td>P20263</td>
<td>2</td>
</tr>
<tr>
<td>CTL3</td>
<td>CHROM</td>
<td>P50247</td>
<td>2</td>
</tr>
<tr>
<td>CTL3</td>
<td>CHROM</td>
<td>P23198</td>
<td>1</td>
</tr>
<tr>
<td>CTL3</td>
<td>CHROM</td>
<td>P20263</td>
<td>1</td>
</tr>
<tr>
<td>EXP1</td>
<td>CHROM</td>
<td>P50247</td>
<td>17</td>
</tr>
<tr>
<td>EXP1</td>
<td>CHROM</td>
<td>P23198</td>
<td>28</td>
</tr>
<tr>
<td>EXP1</td>
<td>CHROM</td>
<td>P20263</td>
<td>15</td>
</tr>
<tr>
<td>EXP2</td>
<td>CHROM</td>
<td>P50247</td>
<td>20</td>
</tr>
<tr>
<td>EXP2</td>
<td>CHROM</td>
<td>P23198</td>
<td>27</td>
</tr>
<tr>
<td>EXP2</td>
<td>CHROM</td>
<td>P20263</td>
<td>15</td>
</tr>
<tr>
<td>EXP3</td>
<td>CHROM</td>
<td>P50247</td>
<td>19</td>
</tr>
<tr>
<td>EXP3</td>
<td>CHROM</td>
<td>P23198</td>
<td>29</td>
</tr>
<tr>
<td>EXP3</td>
<td>CHROM</td>
<td>P20263</td>
<td>14</td>
</tr>
</tbody>
</table>
<p>In the <a href="#krt">Key Resources Table</a> we provide the code from MS proteomics sample data-set used in (<a href="#bib1">Aranda et al., 2019</a>), which is deposited in the ProteomeXchange Consortium via the PRIDE partner repository with the data set identifier PXD011670</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The iPOTD methodology is in its essence a global chromatome surveyor method, which provides an average composition of the chromatome from a cell population. Yet, the experimental procedure is highly compatible with additional labelling approaches (e.g. immunostaining), thereby allowing iPOTD combination with fluorescence-activated cell sorter (FACS) analysis. For example, this would enable to profile the chromatin-bound proteome from metabolically distinct cells, or from cells in different phases of the cell cycle.</p>
<p>As mentioned above, our methodology enables bulky chromatin to be analyzed. However, chromatin composition changes associated to a reduced portion of the genome (e.g. single locus), or re-localization of chromatin proteins, might not be detectable by analysing bulk chromatin. We have previously coupled antibody-based affinity purification (AP) techniques with EdU-labelled DNA purification methods to capture the interactome of a protein of interest within nascent chromatin (<a href="#bib2">Aranda et al., 2014</a>). Thus, combining iPOTD with AP (e.g. using antibodies against a histone modifications) could provide a powerful tool for unveiling chromatin-bound proteome of particular genomic regions, while avoiding sample contamination from antibodies.</p>
<p>Our strategy relies on the incorporation of EdU during replication, which limits its use to proliferating cells. Yet, this can be however highly advantageous for purifying chromatin from specific sets of proliferating cells, as highly proliferative tumor cells or proliferating adult stem cells) within a mitotically inactive stroma or niche, thereby highlighting important regulators in cancer and stem cell biology.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Procedure</h3>
<p>Cell recovery after fixation; Step 9</p>
<h3>Problem</h3>
<p>Fewer cells recovered from EdU-treated culture than expected.</p>
<h3>Solution</h3>
<ul>
<li>Ensure that all cells have been lifted from the dish.</li>
<li>If EdU incubation is toxic for cells, re-adjust EdU concentration and incubation time for your cells.</li>
</ul>
<h3>Procedure</h3>
<p>Chromatin fragmentation; Step 20</p>
<h3>Problem</h3>
<p>Cell lysate is cloudy after sonication.</p>
<h3>Solution</h3>
<ul>
<li>Ensure that the proper volume of fragmentation buffer is used.</li>
<li>Increase the sonication time.</li>
</ul>
<h3>Procedure</h3>
<p>MS spectrometry data, Step 61</p>
<h3>Problem</h3>
<p>Low LC-MS/MS signal</p>
<h3>Solution</h3>
<ul>
<li>The presence of SDS in the sample can be reduced by increasing the washing steps at step 43.</li>
<li>Check the quality control samples during equipment setup.</li>
</ul>
<h3>Procedure</h3>
<p>Monitoring DNA fragmentation</p>
<h3>Problem</h3>
<p>Low DNA recovery</p>
<h3>Solution</h3>
<ul>
<li>If DNA dilution is too high, reduce the volume of the elution buffer used or concentrate the material using a DNA vacuum concentrator.</li>
</ul>
<h3>Procedure</h3>
<p>Monitoring DNA fragmentation</p>
<h3>Problem</h3>
<p>Chromatin fragmentation gives DNA fragments larger than 500 bp.</p>
<h3>Solution</h3>
<p>Increase the sonication time.</p>
<h3>Procedure</h3>
<p>Monitoring DNA labeling</p>
<h3>Problem</h3>
<p>Low signal-to-noise ratio in the dot blot.</p>
<h3>Solution</h3>
<p>Reduce the time of HRP-avidin incubation.</p>
<p>Adjust EdU concentration and incubation time for your cells.</p>
<p>Concentrate the material using a DNA vacuum concentrator.</p>
<h3>Procedure</h3>
<p>Monitoring chromatin capture</p>
<h3>Problem</h3>
<p>Low signal-to-noise ratio in the dot blot</p>
<h3>Solution</h3>
<p>Optimize the concentration and timing of incubation with the antibodies used.</p>
<p>If the sample is too diluted, adjust the volume of the fragmentation buffer to increase the protein concentration.</p>
<h3>Procedure</h3>
<p>Monitoring chromatin capture</p>
<h3>Problem</h3>
<p>Low H3 recovery after elution</p>
<h3>Solution</h3>
<p>Increase the boiling time to effectively elute the chromatin sample from streptavidin beads.</p>
<h3>Procedure</h3>
<p>Monitoring chromatin capture</p>
<h3>Problem</h3>
<p>High recovery of vinculin after elution.</p>
<h3>Solution</h3>
<p>Increase the time for blocking the beads.</p>
<p>Increase the stringency of washes.</p>
<p>Ensure the proper dilution of the fragmented samples with the dilution buffer</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aranda, S., Alcaine-Colet, A., Blanco, E., Borras, E., Caillot, C., Sabido, E., and Di Croce, L. (2019). Chromatin capture links the metabolic enzyme AHCY to stem cell proliferation. Sci. Adv. <i>5</i>, eaav2448.</p>
<p id="bib2">Aranda, S., Rutishauser, D., and Ernfors, P. (2014). Identification of a large protein network involved in epigenetic transmission in replicating DNA of embryonic stem cells. Nucleic Acids Res. <i>42</i>, 6972-6986.</p>
<p id="bib3">Choi, H., Larsen, B., Lin, Z.Y., Breitkreutz, A., Mellacheruvu, D., Fermin, D., Qin, Z.S., Tyers, M., Gingras, A.C., and Nesvizhskii, A.I. (2011). SAINT: probabilistic scoring of affinity purification-mass spectrometry data. Nat. Methods <i>8</i>, 70-73.</p>
<p id="bib4">Teo, G., Liu, G., Zhang, J., Nesvizhskii, A.I., Gingras, A.C., and Choi, H. (2014). SAINTexpress: improvements and additional features in Significance Analysis of INTeractome software. J. Proteomics <i>100</i>, 37-43.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>We thank Dr. Anna Alcaine-Colet and the members of the Di Croce laboratory for critical reading of the manuscript and insightful discussions, V.A. Raker for scientific editing, and the CRG Genomics Unit, CRG/UPF Flow Cytometry Unit, and the CRG Advanced Light Microscopy Unit for assistance with sequencing, FACS, and microscopy services, respectively. We acknowledge support from the <a href="https://doi.org/10.13039/501100010198">Spanish Ministry of Economy, Industry and Competitiveness</a> to the EMBL partnership, Centro de Excelencia Severo Ochoa, the CERCA Programme/<a href="https://doi.org/10.13039/501100002809">Generalitat de Catalunya</a>, the Secretary for Universities and Research of the Ministry of Economy and Knowledge of the Government of Catalonia, and the <a href="https://doi.org/10.13039/501100004101">Lady Tata Memorial Trust</a> (to S.A.). The CRG/UPF Proteomics Unit is part of the Spanish Infrastructure for Omics Technologies (ICTS OmicsTech), and it is a member of the ProteoRed PRB3 consortium, which is supported by grant PT17/0019 of the PE I+D+i 2013-2016 from the <a href="https://doi.org/10.13039/501100004587">Instituto de Salud Carlos III</a> (ISCIII), ERDF, and “Secretaria d’Universitats i Recerca del Departament d’Economia i Coneixement de la Generalitat de Catalunya” (2017SGR595). The Proteomics Unit is supported by EPIC-XS, project number 823839, funded by the <a href="https://doi.org/10.13039/501100007601">Horizon 2020</a> programme of the European Union. The Di Croce Laboratory is supported by grants from the Spanish Ministerio de Educación y Ciencia (BFU2016-75008-P) and by <a href="https://doi.org/10.13039/501100003030">AGAUR </a>.</p>
<h3>Author Contributions</h3>
<p>S.A. conceived and planned this project, conducted, analyzed, and interpreted data, and wrote the manuscript with the input from coauthors. E.B. and E.S. analyzed the proteomic data and contributed to the writing of the manuscript. L.D.C. conceived and planned this project and contributed to the writing of the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>