<article>
<section>
<h1 id="header">fSHAPE, fSHAPE-eCLIP, and SHAPE-eCLIP probe transcript regions that interact with specific proteins</h1>
<p><time datetime="2021-08-27">Published: August 27, 2021</time></p>
<p>Meredith Corley,<sup><a href="#aff1">1</a>,<a href="#fn1">6</a>,<a href="#cor1">*</a></sup> Ryan A. Flynn,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Steven M. Blue,<sup><a href="#aff1">1</a></sup> Brian A. Yee,<sup><a href="#aff1">1</a></sup> Howard Y. Chang,<sup><a href="#aff4">4</a>,<a href="#aff5">5</a></sup> and Gene W. Yeo<sup><a href="#aff1">1</a>,<a href="#fn2">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Cellular and Molecular Medicine, Institute for Genomic Medicine, UCSD Stem Cell Program, University of California San Diego, La Jolla, CA 92093, USA</p>
<p id="aff2"><sup>2</sup>Stem Cell Program, Boston Children’s Hospital, Boston, MA 02115, USA</p>
<p id="aff3"><sup>3</sup>Department of Stem Cell and Regenerative Biology, Harvard University, Cambridge, MA 02138, USA</p>
<p id="aff4"><sup>4</sup>Center for Personal Dynamic Regulomes, Stanford University School of Medicine, Stanford, CA 94305, USA</p>
<p id="aff5"><sup>5</sup>Howard Hughes Medical Institute, Stanford University, Stanford, CA 94305, USA</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:meracorley@gmail.com">meracorley@gmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:geneyeo@ucsd.edu">geneyeo@ucsd.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100762">10.1016/j.xpro.2021.100762</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Selective 2′-hydroxyl acylation analyzed by primer extension (SHAPE) structure probing techniques characterize the secondary structure of RNA molecules, which influence their functions and interactions. A variation of SHAPE, footprinting SHAPE (fSHAPE), probes RNA in the presence and absence of protein to identify RNA bases that hydrogen-bond with protein. SHAPE or fSHAPE coupled with enhanced crosslinking and immunoprecipitation (SHAPE-eCLIP or fSHAPE-eCLIP) pulls down RNAs bound by any protein of interest and returns their structure or protein interaction information, respectively. Here, we describe detailed protocols for SHAPE-eCLIP and fSHAPE-eCLIP and an analysis protocol for fSHAPE.</p>
<p>For complete details on the use and execution of these protocols, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref2">Corley et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocols for probing protein-RNA hydrogen bonds (fSHAPE) and RNA structure (SHAPE)</li>
<li>fSHAPE-eCLIP and SHAPE-eCLIP probe RNA regions bound by immunoprecipitated protein</li>
<li>Samples probed in parallel under different conditions is the basis of each protocol</li>
<li>Probed regions yield fSHAPE or SHAPE reactivities at nucleotide resolution</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/962-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>fSHAPE, fSHAPE-eCLIP, and SHAPE-eCLIP are all methods that probe RNA in cells with small RNA-reactive reagents and prepare sequencing libraries from the probed RNA, whose analysis generates “reactivities” at nucleotide resolution. Pairwise analysis comparing the reactivities from samples probed under different conditions produces values that describe the propensity of each nucleobase to hydrogen bond with protein (fSHAPE) or its propensity to be base-paired (SHAPE).</p>
<p>fSHAPE:</p>
<p>To perform fSHAPE, follow the published icSHAPE protocol (<a href="#bib6">Flynn et al., 2016</a>) for both “in vitro” and “in vivo” icSHAPE, at least two replicates each, ignoring the steps for the DMSO-treated controls. Analyze fSHAPE sequencing data according to instructions under the <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a> section of this manuscript.</p>
<p>fSHAPE-eCLIP and SHAPE-eCLIP:</p>
<p>fSHAPE-eCLIP and SHAPE-eCLIP are very similar in execution, having only a few important differences. They will be presented here together with specific instructions where they differ and are referred to jointly as (f)SHAPE-eCLIP.</p>
<p>fSHAPE has been tested in K562, HepG2, HeLa, and HEK293T cells. (f)SHAPE-eCLIP has been tested in K562 cells.</p>
<p>The RNA probing reagent NAI-N<sub>3</sub> has been tested with fSHAPE. fSHAPE-eCLIP has been tested with the reagent NAI. SHAPE-eCLIP has been tested with NAI and dimethyl sulfate (DMS); NAI is preferred because DMS does not react with all RNA bases.</p>
<p>(f)SHAPE-eCLIP has been performed with antibodies for SLBP (described here) and RBFOX2. These methods are assumed to be compatible with any of the antibodies successfully employed for eCLIP, of which there are over 300 (<a href="#bib12">Van Nostrand et al., 2020</a>). It is highly recommended to verify your antibody’s immunoprecipitation efficiency before performing this protocol for a full set of samples.</p>
<p><i>It is recommended to store aliquots of primers at −80</i>°<i>C. Master mixes of enzymatic reactions, minus the enzymes, may be pre-mixed and aliquoted and stored at −80</i>°C <i>for later use.</i></p>
<h3 id="sec1.1">Prepare primers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol>
<li>Dilute primers to specified concentrations.
<ol type="a">
<li>RNA oligo “InvRiL19” for RNA linker ligation: Order 100 nmole, standard desalting. Resuspend at 200 μM; dilute to 40 μM for working stock. Store 5 μL aliquots at <i>−</i>80°C.</li>
<li>DNA oligo “InvRand3Tr3” for cDNA ligation: Order 100 nmole, standard desalting. Resuspend at 200 μM; dilute to 80 μM for working stock. Store 5 μL aliquots at <i>−</i>80°C.</li>
<li>DNA oligo “InvAR17” for reverse transcription: Order 25 nmole, standard desalting. Resuspend at 200 μM; dilute to 5 μM for working stock. Store 5 μL aliquots at <i>−</i>80°C.</li>
<li>PCR_F_D501, PCR_F_D502, PCR_R_D701,PCR_R_D702: Order PAGE purified. Resuspend at 100 μM; dilute to 20 μM for working stock. Store 5 μL aliquots at <i>−</i>80°C.
<ol type="i">
<li>Any Illumina standard high-throughput RNA-sequencing primer, such as D503-508, D703-712, can substitute for the above.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> days, depending on cell type</div>
<ol start="2">
<li>Grow K562 cells to 90% confluency. Aim for 40 million cells per sample condition. For 2 sample conditions with 2 replicates each at a minimum, this equates to 160 million cells in total.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-stem loop binding protein</td>
<td>MBLI</td>
<td>RRID:AB_10606342; RN045P Lot# 001</td>
</tr>
<tr>
<td>Anti-rabbit TrueBlot HRP</td>
<td>Rockland</td>
<td>RRID:AB_2610848; Cat# 18-8816-33</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Dimethylsulfoxide (DMSO)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 276855</td>
</tr>
<tr>
<td>2-Methylnicotinic acid imidazolide (NAI) in DMSO</td>
<td>EMD Millipore</td>
<td>Cat# 03-310</td>
</tr>
<tr>
<td>Buffer RLT</td>
<td>QIAGEN</td>
<td>cat # 79216</td>
</tr>
<tr>
<td>T4 polynucleotide kinase (PNK), 10 U/μL</td>
<td>New England Biolabs</td>
<td>Cat# M0201L</td>
</tr>
<tr>
<td>5X PNK 6.5 buffer</td>
<td>New England Biolabs</td>
<td>Cat# M0201L</td>
</tr>
<tr>
<td>1 M DTT</td>
<td>Thermo Scientific</td>
<td>Cat# P2325</td>
</tr>
<tr>
<td>100% Ethanol</td>
<td>Thermo Scientific</td>
<td>Cat# T038181000</td>
</tr>
<tr>
<td>100% Methanol</td>
<td>Thermo Scientific</td>
<td>Cat# T001021000</td>
</tr>
<tr>
<td>FastAP Thermosensitive Alkaline Phosphatase, 1 U/μL</td>
<td>Thermo Scientific</td>
<td>Ct# EF0651</td>
</tr>
<tr>
<td>Proteinase K, 0.8 U/μL</td>
<td>New England Biolabs</td>
<td>Cat# P8107S</td>
</tr>
<tr>
<td>SuperScript II Reverse Transcriptase, 200 U/μL</td>
<td>Invitrogen</td>
<td>Cat# 18064014</td>
</tr>
<tr>
<td>Turbo DNase, 2 U/μL</td>
<td>Life Technologies</td>
<td>Cat# AM2239</td>
</tr>
<tr>
<td>T4 RNA ligase 1, high concentration</td>
<td>New England Biolabs</td>
<td>Cat# M0437M</td>
</tr>
<tr>
<td>10 X ligase buffer (with DTT)</td>
<td>New England Biolabs</td>
<td>Cat# M0437M</td>
</tr>
<tr>
<td>100 mM ATP</td>
<td>New England Biolabs</td>
<td>Cat# M0437M</td>
</tr>
<tr>
<td>50% PEG 8000</td>
<td>New England Biolabs</td>
<td>Cat# M0437M</td>
</tr>
<tr>
<td>Murine RNase Inhibitor, 40 U/μL</td>
<td>New England Biolabs</td>
<td>Cat# M0314L</td>
</tr>
<tr>
<td>RNase I, 100 U/μL</td>
<td>Life Technologies</td>
<td>Cat# AM2295</td>
</tr>
<tr>
<td>Q5 PCR Master Mix</td>
<td>New England Biolabs</td>
<td>Cat# M0492L</td>
</tr>
<tr>
<td>Protease Inhibitor Cocktail III</td>
<td>EMD Millipore</td>
<td>Cat# 539134-1SET</td>
</tr>
<tr>
<td>ExoSAP-IT</td>
<td>Affymetrix</td>
<td>Cat# 78201</td>
</tr>
<tr>
<td>Tris-HCI buffer, 1 M pH 7.5</td>
<td>Invitrogen</td>
<td>Cat# 15567-027</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Sigma-Aldrich</td>
<td>Cat# P1379</td>
</tr>
<tr>
<td>HEPES, 1 M pH 8.0</td>
<td>Life Technologies</td>
<td>Cat# 15630-080</td>
</tr>
<tr>
<td>Magnesium chloride, 1 M</td>
<td>Ambion</td>
<td>Cat# AM9530G</td>
</tr>
<tr>
<td>Sodium chloride, 5 M</td>
<td>Ambion</td>
<td>Cat# AM9759</td>
</tr>
<tr>
<td>Potassium chloride, 3M</td>
<td>Millipore Sigma</td>
<td>Cat# 60135</td>
</tr>
<tr>
<td>Nonidet P40 (NP-40)</td>
<td>Roche</td>
<td>Cat# 11332473001</td>
</tr>
<tr>
<td>EDTA, 500 mM</td>
<td>Millipore</td>
<td>Cat# 324504</td>
</tr>
<tr>
<td>HCl, 1.0 N</td>
<td>Sigma-Aldrich</td>
<td>Cat# H9892</td>
</tr>
<tr>
<td>NaOH, 1.0 N</td>
<td>Sigma-Aldrich</td>
<td>Cat# S2770</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate (SDS) 10%</td>
<td>Sigma-Aldrich</td>
<td>Cat# 71736</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>Sigma-Aldrich</td>
<td>Cat# 30970</td>
</tr>
<tr>
<td>20X MOPS Running Buffer</td>
<td>Thermo Fisher</td>
<td>Cat# NP0001</td>
</tr>
<tr>
<td>20X NuPAGE Transfer Buffer</td>
<td>Thermo Fisher</td>
<td>Cat# NP00061</td>
</tr>
<tr>
<td>10X TBST Buffer</td>
<td>Sigma-Aldrich</td>
<td>Cat# 91414</td>
</tr>
<tr>
<td>10X TBE Buffer</td>
<td>Millipore Sigma</td>
<td>Cat# SRE0062</td>
</tr>
<tr>
<td>Manganese (II) chloride solution 1M</td>
<td>Sigma-Aldrich</td>
<td>Cat# M1787</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Thermo Scientific</td>
<td>Cat# 15514011</td>
</tr>
<tr>
<td>Power SYBR Green PCR Master Mix</td>
<td>Applied Biosystems</td>
<td>Cat# 4368577</td>
</tr>
<tr>
<td>Dynabeads M-280 Sheep Anti-Rabbit, or anti-mouse, 10 mg/mL</td>
<td>Life Technologies</td>
<td>Cat# 11204D</td>
</tr>
<tr>
<td>Dynabeads MyOne Silane, 40 mg/mL</td>
<td>Life Technologies</td>
<td>Cat# 37002D</td>
</tr>
<tr>
<td>Agencourt AMPure XP beads</td>
<td>Beckman Coulter</td>
<td>Cat# A63881</td>
</tr>
<tr>
<td>ECL Western Blotting Substrate</td>
<td>Thermo Scientific</td>
<td>Cat# 32106</td>
</tr>
<tr>
<td>Broad Range Protein Ladder</td>
<td>Thermo Scientific</td>
<td>Cat # 26634</td>
</tr>
<tr>
<td>Milk powder</td>
<td>Sigma-Aldrich</td>
<td>Cat# M7409</td>
</tr>
<tr>
<td>SYBR Safe DNA Gel Stain</td>
<td>Invitrogen</td>
<td>Cat# S33102</td>
</tr>
<tr>
<td>6X Orange G Buffer</td>
<td>New England Biolabs</td>
<td>Cat# B7022S</td>
</tr>
<tr>
<td>50 bp DNA Ladder</td>
<td>Thermo Scientific</td>
<td>Cat# SM0373</td>
</tr>
<tr>
<td>NuSieve GTG Agarose (low melting temperature)</td>
<td>Lonza</td>
<td>Cat# 50081</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RNA Clean and Concentrator Kit</td>
<td>Zymo Research</td>
<td>Cat# R1019</td>
</tr>
<tr>
<td>MinElute Gel Extraction Kit</td>
<td>QIAGEN</td>
<td>Cat# 28604</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Sequencing data and processed fSHAPE and SHAPE reactivities from fSHAPE, fSHAPE-eCLIP, and SHAPE-eCLIP libraries</td>
<td>(<a href="#bib2">Corley et al., 2020</a>)</td>
<td>GEO: GSE149767</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>K562</td>
<td>ATCC</td>
<td>CCL-243</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>“InvRiL19” (RNA oligo): /5Phos/rArGrArUrCrGrGrArArGrArGrCrArCrArCrGrUrC/3SpC3/, standard desalting</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>“InvRand3Tr3”: /5Phos/NNNNNNNNNNAGATCGGAAGAGCGTCGTGT/3SpC3/, standard desalting</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>“InvAR17”: CAGACGTGTGCTCTTCCGA, standard desalting</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>“PCR_F_D501”: AATGATACGGCGACCACCGAGATCTACACTATAGCCTACACTCTTTCCCTACACGACGCTCTTCCGATCT, PAGE purified</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>“PCR_F_D502”: AATGATACGGCGACCACCGAGATCTACACATAGAGGCACACTCTTTCCCTACACGACGCTCTTCCGATCT, PAGE purified</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>“PCR_R_D701”: CAAGCAGAAGACGGCATACGAGATCGAGTAATGTGACTGGAGTTCAGACGTGTGCTCTTCCGATC, PAGE purified</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>“PCR_R_D702”: CAAGCAGAAGACGGCATACGAGATTCTCCGGAGTGACTGGAGTTCAGACGTGTGCTCTTCCGATC, PAGE purified</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Cutadapt 1.14</td>
<td>(<a href="#bib9">Martin, 2011</a>)</td>
<td><a href="https://cutadapt.readthedocs.io/en/stable/">https://cutadapt.readthedocs.io/en/stable/</a></td>
</tr>
<tr>
<td>UMItools &gt;= 0.5.0</td>
<td>(<a href="#bib11">Smith et al., 2017</a>)</td>
<td><a href="https://github.com/CGATOxford/UMI-tools">https://github.com/CGATOxford/UMI-tools</a></td>
</tr>
<tr>
<td>STAR &gt;= 2.4.0i</td>
<td>(<a href="#bib4">Dobin et al., 2013</a>)</td>
<td><a href="https://github.com/alexdobin/STAR">https://github.com/alexdobin/STAR</a></td>
</tr>
<tr>
<td>Bedtools &gt;= 2.25.0</td>
<td>(<a href="#bib10">Quinlan and Hall, 2010</a>)</td>
<td><a href="https://bedtools.readthedocs.io/en/latest/">https://bedtools.readthedocs.io/en/latest/</a></td>
</tr>
<tr>
<td>Samtools &gt;= 1.9</td>
<td>(<a href="#bib7">Li et al., 2009</a>)</td>
<td><a href="https://samtools.github.io">https://samtools.github.io</a></td>
</tr>
<tr>
<td>Bamtools 2.3.0</td>
<td>(<a href="#bib1">Barnett et al., 2011</a>)</td>
<td><a href="https://github.com/pezmaster31/bamtools">https://github.com/pezmaster31/bamtools</a></td>
</tr>
<tr>
<td>fSHAPE analysis pipeline</td>
<td>This manuscript</td>
<td><a href="https://github.com/meracorley/fSHAPE">https://github.com/meracorley/fSHAPE</a></td>
</tr>
<tr>
<td>fSHAPE- and SHAPE-eCLIP analysis pipelines</td>
<td>This manuscript</td>
<td><a href="https://github.com/meracorley/f-SHAPE-eCLIP/tree/pipeline">https://github.com/meracorley/f-SHAPE-eCLIP/tree/pipeline</a></td>
</tr>
<tr>
<td>Python &gt;= 3.5.4</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Hmmlearn 0.2.1</td>
<td>Scikit-learn</td>
<td><a href="https://hmmlearn.readthedocs.io/en/latest/">https://hmmlearn.readthedocs.io/en/latest/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Laminar flow hood</td>
<td>Standard lab equipment</td>
<td>N/A</td>
</tr>
<tr>
<td>Cell culture incubator</td>
<td>Thermo Scientific</td>
<td>Cat# 51026331</td>
</tr>
<tr>
<td>Large centrifuge</td>
<td>Thermo Scientific</td>
<td>Cat# 1189M64</td>
</tr>
<tr>
<td>UV crosslinker</td>
<td>Thermo Scientific</td>
<td>Cat# 95034</td>
</tr>
<tr>
<td>Small centrifuge</td>
<td>Eppendorf</td>
<td>Cat# 5406000046</td>
</tr>
<tr>
<td>Bioruptor sonicator</td>
<td>Bioruptor</td>
<td>Cat# B01020001</td>
</tr>
<tr>
<td>1.5 mL Tube magnet</td>
<td>Thermo Scientific</td>
<td>Cat# CS15000</td>
</tr>
<tr>
<td>0.2 mL Tube magnet</td>
<td>EpiCypher</td>
<td>Cat# 10-0008</td>
</tr>
<tr>
<td>Tube rotator</td>
<td>Thermo Scientific</td>
<td>Cat# 88881001</td>
</tr>
<tr>
<td>Thermomixer</td>
<td>Eppendorf</td>
<td>Cat# 5382000023</td>
</tr>
<tr>
<td>PAGE gel tank</td>
<td>Invitrogen</td>
<td>Cat# EI0001</td>
</tr>
<tr>
<td>4%–12% Bis-Tris gel, 1.5 mm, 10 well</td>
<td>Invitrogen</td>
<td>Cat# NP0335BOX</td>
</tr>
<tr>
<td>Transfer tank</td>
<td>Thermo Scientific</td>
<td>Cat# VEP-2</td>
</tr>
<tr>
<td>Power supply for running gels/transfers</td>
<td>Invitrogen</td>
<td>Cat# PS0350</td>
</tr>
<tr>
<td>PVDF membrane</td>
<td>Invitrogen</td>
<td>Cat# PB9320</td>
</tr>
<tr>
<td>Nitrocellulose membrane</td>
<td>Invitrogen</td>
<td>Cat# PB7320</td>
</tr>
<tr>
<td>Agarose gel electrophoresis equipment</td>
<td>Standard lab equipment</td>
<td>N/A</td>
</tr>
<tr>
<td>Thermocycler</td>
<td>Standard lab equipment</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>eCLIP Lysis Buffer (step 5), store at 4°C for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH 7.4</td>
<td>50mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>100 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>NP-40</td>
<td>1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.5%</td>
<td>0.5 g</td>
</tr>
<tr>
<td>diH2O</td>
<td>n/a</td>
<td>91 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>eCLIP High Salt Wash Buffer (step 10), store at 4°C for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH 7.4</td>
<td>50mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>1 M</td>
<td>20 mL</td>
</tr>
<tr>
<td>500 mM EDTA</td>
<td>1 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>NP-40</td>
<td>1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>SDS 10%</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.5%</td>
<td>0.5 g</td>
</tr>
<tr>
<td>diH2O</td>
<td>n/a</td>
<td>72.8 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>eCLIP Wash Buffer (step 10), store at 4°C for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH 7.4</td>
<td>20 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>5 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1% Tween-20</td>
<td>0.2%</td>
<td>20 mL</td>
</tr>
<tr>
<td>diH2O</td>
<td>n/a</td>
<td>76.9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>RLTW Buffer (step 38), store at RT for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RLT Buffer (1X)</td>
<td>1X</td>
<td>9.75 mL</td>
</tr>
<tr>
<td>1% Tween-20</td>
<td>0.025%</td>
<td>250 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>PKS Buffer (step 28), store at RT for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH 7.4</td>
<td>100 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>50 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>500 mM EDTA</td>
<td>10 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>0.2%</td>
<td>200 μL</td>
</tr>
<tr>
<td>diH2O</td>
<td>n/a</td>
<td>8.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>1X RNA Ligase Buffer (step 15), store at RT for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH <b>7.5</b>
</td>
<td>50 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>10 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>diH2O</td>
<td>n/a</td>
<td>9.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl7">
<caption>3X SHAPE Folding Buffer (step 31), store at RT for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M HEPES pH 8.0</td>
<td>333 mM</td>
<td>333 μL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>20 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>333 mM</td>
<td>66.6 μL</td>
</tr>
<tr>
<td>diH2O</td>
<td>n/a</td>
<td>580.4 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>1 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl8">
<caption>SHAPE FS Buffer (step 41), store at RT for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH <b>8.0</b>
</td>
<td>500 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>3 M KCl</td>
<td>750 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>250 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>1 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl9">
<caption>TT Elution Buffer (step 38), store at RT for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH 7.5</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>500 mM EDTA</td>
<td>0.1 mM</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>1% Tween-20</td>
<td>0.01% Tween-20</td>
<td>10 μL</td>
</tr>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>979.8 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>1 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl10">
<caption>PCR Elution Buffer (step 48), store at RT for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH 7.5</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>20 mM</td>
<td>4 μL</td>
</tr>
<tr>
<td>500 mM EDTA</td>
<td>0.1 mM</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>985.8 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>1 mL</td>
</tr>
</tbody>
</table>
<p>Enzymatic reactions:</p>
<table id="undtbl11">
<caption>“Early” FastAP treatment (on-bead) (step 13)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>38 μL</td>
</tr>
<tr>
<td>10X FastAP buffer</td>
<td>1X</td>
<td>5 μL</td>
</tr>
<tr>
<td>Murine RNase Inhibitor, 40 U/μL</td>
<td>1.6 U/μL</td>
<td>2 μL</td>
</tr>
<tr>
<td>Turbo DNase, 2 U/μL</td>
<td>0.08 U/μL</td>
<td>2 μL</td>
</tr>
<tr>
<td>FastAP enzyme, 1 U/μL</td>
<td>0.06 U/μL</td>
<td>3 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>50 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl12">
<caption>“Early” PNK treatment (on-bead) (step14)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>116 μL</td>
</tr>
<tr>
<td>5X PNK Buffer pH 6.5</td>
<td>1X</td>
<td>30 μL</td>
</tr>
<tr>
<td>T4 PNK enzyme, 10 U/μL</td>
<td>0.27 U/μL</td>
<td>4 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>150 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl13">
<caption>“Early” 3′ RNA linker ligation (on-bead) (step 16)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>8.4 μL</td>
</tr>
<tr>
<td>10X Ligase buffer (no DTT; self-made, see buffers above)</td>
<td>1X</td>
<td>3 μL</td>
</tr>
<tr>
<td>0.1 M ATP</td>
<td>1 mM</td>
<td>0.3 μL</td>
</tr>
<tr>
<td>100% DMSO</td>
<td>3%</td>
<td>0.9 μL</td>
</tr>
<tr>
<td>1% Tween-20</td>
<td>0.02%</td>
<td>0.6 μL</td>
</tr>
<tr>
<td>50% PEG 8000</td>
<td>15%</td>
<td>9 μL</td>
</tr>
<tr>
<td>Murine RNase Inhibitor, 40 U/μL</td>
<td>0.53 U/μL</td>
<td>0.4 μL</td>
</tr>
<tr>
<td>RNA ligase high conc., 30 U/μL</td>
<td>2.4 U/μL</td>
<td>2.4 μL</td>
</tr>
<tr>
<td><b>Combine above</b></td>
<td><b>n/a</b></td>
<td><b>25 μL</b></td>
</tr>
<tr>
<td>Beads binding RNA</td>
<td>n/a</td>
<td>~2.5 μL</td>
</tr>
<tr>
<td>InvRiL19 oligo, 40 μM</td>
<td>3.33 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>~30 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl14">
<caption>
<i>In vitro</i> structure probing (step 31)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RNA sample, heat denatured and flash-cooled</td>
<td>n/a</td>
<td>~10 μL</td>
</tr>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>2.4 μL</td>
</tr>
<tr>
<td>3X SHAPE Folding Buffer</td>
<td>1X</td>
<td>6.6 μL</td>
</tr>
<tr>
<td>Murine RNase Inhibitor, 40 U/μL</td>
<td>2 U/μL</td>
<td>1.0 μL</td>
</tr>
<tr>
<td><b>Combine above</b></td>
<td>n/a</td>
<td>20 μL</td>
</tr>
<tr>
<td>2 M NAI in DMSO</td>
<td>100 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>21 μL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> The reagent NAI should be handled with caution, avoiding contact with skin.</div>
<table id="undtbl15">
<caption>“Late” FastAP treatment (step 34)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>6 μL</td>
</tr>
<tr>
<td>10X FastAP Buffer</td>
<td>1X</td>
<td>2 μL</td>
</tr>
<tr>
<td>Murine RNase Inhibitor, 40 U/μL</td>
<td>2 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td>FastAP enzyme, 1 U/μL</td>
<td>0.1 U/μL</td>
<td>2 μL</td>
</tr>
<tr>
<td><b>Combine above</b></td>
<td>n/a</td>
<td>11 μL</td>
</tr>
<tr>
<td>RNA sample</td>
<td>n/a</td>
<td>~10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>21 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl16">
<caption>“Late” PNK treatment (step 35)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>45 μL</td>
</tr>
<tr>
<td>5X PNK Buffer pH 6.5</td>
<td>1X</td>
<td>20 μL</td>
</tr>
<tr>
<td>1 M DTT</td>
<td>5 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Turbo DNase, 2 U/μL</td>
<td>0.02 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td>T4 PNK enzyme, 10 U/μL</td>
<td>0.4 U/μL</td>
<td>4 μL</td>
</tr>
<tr>
<td><b>Combine above</b></td>
<td>n/a</td>
<td>75 μL</td>
</tr>
<tr>
<td>“Late” FastAP reaction</td>
<td>n/a</td>
<td>~21 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>~100 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl17">
<caption>“Late” 3′ RNA linker ligation for Input samples (step 37)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>2.8 μL</td>
</tr>
<tr>
<td>10X NEB Ligase Buffer (with DTT)</td>
<td>1X</td>
<td>2.0 μL</td>
</tr>
<tr>
<td>0.1 M ATP</td>
<td>1 mM</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>100% DMSO</td>
<td>3%</td>
<td>0.6 μL</td>
</tr>
<tr>
<td>1% Tween-20</td>
<td>0.02%</td>
<td>0.4 μL</td>
</tr>
<tr>
<td>50% PEG 8000</td>
<td>15%</td>
<td>6.0 μL</td>
</tr>
<tr>
<td>Murine RNase Inhibitor, 40 U/μL</td>
<td>0.6 U/μL</td>
<td>0.3 μL</td>
</tr>
<tr>
<td>RNA ligase high conc., 30 U/μL</td>
<td>1.8 U/μL</td>
<td>1.2 μL</td>
</tr>
<tr>
<td><b>Combine above</b></td>
<td><b>n/a</b></td>
<td><b>13.5 μL</b></td>
</tr>
<tr>
<td>RNA sample</td>
<td>n/a</td>
<td>5 μL</td>
</tr>
<tr>
<td>100% DMSO</td>
<td>n/a</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>InvRiL19 oligo, 40 μM</td>
<td>1 μM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>20 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl18">
<caption>“Late” 3′ RNA linker ligation for IP samples (step 37)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>5.6 μL</td>
</tr>
<tr>
<td>10X NEB Ligase buffer (with DTT)</td>
<td>1X</td>
<td>4 μL</td>
</tr>
<tr>
<td>0.1 M ATP</td>
<td>1 mM</td>
<td>0.4 μL</td>
</tr>
<tr>
<td>100% DMSO</td>
<td>3%</td>
<td>1.2 μL</td>
</tr>
<tr>
<td>1% Tween-20</td>
<td>0.02%</td>
<td>0.8 μL</td>
</tr>
<tr>
<td>50% PEG 8000</td>
<td>15%</td>
<td>12.0 μL</td>
</tr>
<tr>
<td>Murine RNase Inhibitor, 40 U/μL</td>
<td>0.6 U/μL</td>
<td>0.6 μL</td>
</tr>
<tr>
<td>RNA ligase high conc., 30 U/μL</td>
<td>1.8 U/μL</td>
<td>2.4 μL</td>
</tr>
<tr>
<td><b>Combine above</b></td>
<td><b>n/a</b></td>
<td><b>27 μL</b></td>
</tr>
<tr>
<td>RNA sample</td>
<td>n/a</td>
<td>~10 μL</td>
</tr>
<tr>
<td>100% DMSO</td>
<td>n/a</td>
<td>2.4 μL</td>
</tr>
<tr>
<td>InvRiL19 oligo, 40 μM</td>
<td>1 μM</td>
<td>1.0 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>~40 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl19">
<caption>SHAPE reverse transcription (step 41)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10X SHAPE FS Buffer</td>
<td>1X</td>
<td>2.0 μL</td>
</tr>
<tr>
<td>Murine RNase Inhibitor, 40 U/μL</td>
<td>0.4 U/μL</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>0.1 M DTT</td>
<td>5 mM</td>
<td>1.0 μL</td>
</tr>
<tr>
<td>500 mN MnCl<sub>2</sub>
</td>
<td>6 mM</td>
<td>0.24 μL</td>
</tr>
<tr>
<td>Superscript II reverse transcriptase, 200 U/μL</td>
<td>10 U/μL</td>
<td>1.0 μL</td>
</tr>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>5.36 μL</td>
</tr>
<tr>
<td><b>Combine above</b></td>
<td>n/a</td>
<td>10 μL</td>
</tr>
<tr>
<td>RNA sample</td>
<td>n/a</td>
<td>~9 μL</td>
</tr>
<tr>
<td>5 μM InvAR17 primer</td>
<td>0.25 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>10 mM dNTPs</td>
<td>0.5 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>~20 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl20">
<caption>5′ cDNA linker ligation (step 44)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>1.4 μL</td>
</tr>
<tr>
<td>10X NEB Ligase buffer (with DTT)</td>
<td>1X</td>
<td>4 μL</td>
</tr>
<tr>
<td>0.1 M DTT</td>
<td>2 mM</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>0.1 M ATP</td>
<td>1 mM</td>
<td>0.1 μL</td>
</tr>
<tr>
<td>1% Tween-20</td>
<td>0.02%</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>50% PEG 8000</td>
<td>18%</td>
<td>3.6 μL</td>
</tr>
<tr>
<td>RNA ligase high conc., 30 U/μL</td>
<td>3.0 U/μL</td>
<td>1.0 μL</td>
</tr>
<tr>
<td><b>Combine above</b></td>
<td><b>n/a</b></td>
<td><b>7.8 μL</b></td>
</tr>
<tr>
<td>RNA sample in dried beads</td>
<td>n/a</td>
<td>0 μL</td>
</tr>
<tr>
<td>100% DMSO</td>
<td>8%</td>
<td>0.8 μL</td>
</tr>
<tr>
<td>InvRand3Tr3 oligo, 80 μM</td>
<td>4.8 μM</td>
<td>0.6 μL</td>
</tr>
<tr>
<td>TT Elution Buffer</td>
<td>n/a</td>
<td>1.1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>10.3 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl21">
<caption>cDNA qPCR quantification (step 46)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>diH<sub>2</sub>O</td>
<td>n/a</td>
<td>3.6 μL</td>
</tr>
<tr>
<td>2X PowerSybr master mix</td>
<td>1X</td>
<td>5.0 μL</td>
</tr>
<tr>
<td>20 μM D50x primer mixed in equal parts with 20 μM D70x primer</td>
<td>0.4 μM</td>
<td>0.4 μL</td>
</tr>
<tr>
<td>1:10 dilution cDNA sample</td>
<td>1:100</td>
<td>1.0 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl22">
<caption>cDNA PCR amplification (step 47)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2X Q5 PCR master mix</td>
<td>1X</td>
<td>20 μL</td>
</tr>
<tr>
<td>20 μM right primer (D50x)</td>
<td>1 μM</td>
<td>2 μL</td>
</tr>
<tr>
<td>20 μM left primer (D70x)</td>
<td>1 μM</td>
<td>2 μL</td>
</tr>
<tr>
<td>cDNA sample</td>
<td>n/a</td>
<td>16 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>40 μL</td>
</tr>
</tbody>
</table>
<p>Equipment:</p>
<p>Cell culture equipment: laminar flow hood, cell culture incubator</p>
<p>UV crosslinker (254 nm, UV-C)</p>
<p>Bioruptor sonicator, for RNA fragmentation</p>
<p>Centrifuge for 15 mL conicals; Centrifuge for 1.5 mL tubes; minifuge</p>
<p>Magnet for 1.5 mL tubes, 0.2 mL tubes</p>
<p>Tube rotator</p>
<p>Thermomixer</p>
<p>Western blot and RNA transfer equipment: SDS-PAGE gels 4–12% Bis-Tris, 10-well, 1.5 mm (1.0 mm if 1.5 mm not available), buffer tanks for PAGE and for western blots/RNA transfers, power supply for gel electrophoresis, PVDF and nitrocellulose membranes, sponges, roller</p>
<p>Thermocycler</p>
<p>Real time qPCR equipment</p>
<p>Agarose gel electrophoresis equipment</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Treat cells with NAI or DMSO and UV crosslink</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>Treat cells with the structure probing reagent NAI or mock treat with DMSO, followed by UV crosslinking to covalently link RNA binding proteins to bound RNA molecules. Complete these steps for all protocol types: fSHAPE-eCLIP, SHAPE-eCLIP, or “in vitro” SHAPE-eCLIP.</p>
<ol>
<li>Cell sample requirements
<ol type="a">
<li>Grow cells to ∼30–40 million cells per sample. With two treatment conditions each needing at least two replicates, this adds up to at least four samples, 120–160 million cells total.</li>
<li>fSHAPE-eCLIP and SHAPE-eCLIP both consist of two RNA treatment conditions performed in parallel. The different conditions are referred to here as “A”, “B”, or “C” (see Graphical Abstract):
<ol type="i">
<li>fSHAPE-eCLIP consists of one set of cells treated with NAI (“A”), i.e., RNA is treated <i>in vivo.</i> The other set is mock-treated with DMSO, but later treated with NAI once its RNA is <i>in vitro</i> (“B”).</li>
<li>SHAPE-eCLIP consists of one set of cells treated with NAI (“A”) and the other mock-treated with DMSO (“C“). An “in vitro” variant of SHAPE-eCLIP consists of two sets of samples that are both mock-treated with DMSO at the cell stage, but one set later treats RNA with NAI <i>in vitro</i> (“B“).</li>
</ol>
</li>
</ol>
</li>
<li>Setup
<ol type="a">
<li>Pre-chill on ice 100 mL 1X PBS.</li>
<li>Set an incubator to 37°C.</li>
<li>Pre-chill on ice metal plate that fits inside crosslinker. Set crosslinker to 400 mJ/cm2 and 2 min. Crosslinker should emit UV-C light at ∼254 nm wavelengths.</li>
<li>Fetch some dry ice.</li>
</ol>
</li>
<li>Treat cells with the structure probing reagent (NAI) or mock-treat with DMSO
<ol type="a">
<li>Split cells into 30–40 million cells per sample, two samples for each treatment condition.Instructions are for suspension cells; see note below on adherent cells.</li>
<li>Reduce each cell sample volume to 2 mL in plain media (no FBS) in 10 cm plates.</li>
<li>Per sample pair: For “A” conditions (in fSHAPE-eCLIP or SHAPE-eCLIP), add 170 uL 2 M NAI structure probing reagent to each sample. For “B” or “C” conditions (in fSHAPE-eCLIP or the “in vitro” variant of SHAPE-eCLIP), add 170 uL DMSO to each sample. Swirl to mix.</li>
<li>Incubate plates at 37°C for 10 min.</li>
</ol>
</li>
<li>UV crosslink cells and wash
<ol type="a">
<li>Place pre-chilled metal plate into crosslinker and place plates on top. The chilled plate prevents cells from heating up during crosslinking.</li>
<li>Remove lids from plates. Run the pre-set crosslinking program.</li>
<li>Move cells from plates to 15 mL conicals. Spin down at 500 <i>g</i> for 3 min.</li>
<li>Remove supernatant (NAI-treated supernatant should go into its own waste container).</li>
<li>Resuspend cells in 5 mL cold PBS. Spin down and remove supernatant as before. Resuspend again with cold PBS.</li>
<li>Spin down cells one more time, remove PBS supernatant, and flash freeze cells in dry ice. Store at <i>−</i>80°C.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Removing lids from plates before crosslinking is critical to cells being properly crosslinked.</div>
<div class="note">
<span class="note-title">Note:</span> Adherent cells should have media removed and replaced with 2 mL plain media (no FBS) and subsequently treated with 170 uL NAI or DMSO, swirling to mix. Cells should then be crosslinked as instructed, then scraped off plates with a cell scraper and transferred to 15 mL conicals, following the same washing and centrifugation steps outlined for suspension cells.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Cell pellets can be stored at <i>−</i>80°C for months.</div>
<h3 id="sec3.2">Lyse cells and immunoprecipitate RNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–16 h</div>
<p>Lyse protein-bound RNA from cells. Pull down RNA bound to the protein of interest with a monoclonal antibody coupled to magnetic beads. Complete these steps for all protocol types.</p>
<ol start="5">
<li>Setup
<ol type="a">
<li>Pre-chill eCLIP Lysis Buffer.</li>
<li>Thaw Protease Inhibitor.</li>
<li>Bioruptor sonicator water tank should be at 4°C (in a cold room).</li>
<li>Pre-heat Thermomixer to 37°C.</li>
<li>Pre-set 1.5 mL tube centrifuge to 4°C.</li>
</ol>
</li>
<li>Couple antibody to Dynabeads M-280
<ol type="a">
<li>Prepare beads.
<ol type="i">
<li>Use 125 μL beads per sample, 500 μL for two conditions each with two replicates.</li>
<li>Apply bead solution to magnet; remove supernatant.</li>
<li>Wash beads twice in 500 μL cold eCLIP Lysis Buffer.</li>
<li>Resuspend beads in 100 μL cold eCLIP Lysis Buffer for every sample, typically 400 μL.</li>
</ol>
</li>
<li>Bind antibody.
<ol type="i">
<li>Add 10 μg antibody per sample to prepared beads.</li>
<li>Rotate at 15°C–25°C for 45 min.</li>
</ol>
</li>
</ol>
</li>
<li>Lyse cells, while beads are rotating
<ol type="a">
<li>For two replicates—four samples total—combine 4 mL eCLIP Lysis Buffer with 22 μL Protease Inhibitor Cocktail III and 44 μL Murine RNase Inhibitor.</li>
<li>Retrieve cell pellets from <i>−</i>80°C, keeping on dry ice. Immediately add 1 mL master mix from 7a to each pellet, pipetting to resuspend. Move each sample to a 1.5 mL tube.</li>
<li>Place on ice (not dry ice) and allow to lyse for 5 min.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Fragment RNA
<ol type="a">
<li>Sonicate lysed samples in Bioruptor on “low”, 4°C, 5 cycles of 30 s on and 30 s off. This assists cell lysis and shears chromatin, aiding release of RNA-protein complexes.
<div class="note">
<span class="note-title">Note:</span> Alternatively, sonication may be done on ice with a probe sonicator, sonicating each sample twice at 5 decibels in 10 second bursts. Avoid touching the side of tube with sonicator.</div>
</li>
<li>Dilute 1 μL RNase I in PBS at 1:50 on ice.</li>
<li>Add 10 μL Turbo DNase to each lysed sonicated sample.</li>
<li>Add 10 μL diluted RNase I (20 U total) to each sample. Immediately incubate in a Thermomixer at 37°C, 1200 rpm, 5 min. Place on ice.</li>
<li>Centrifuge samples at 15,000 <i>g</i>, 4°C, for 3 min.</li>
<li>Meanwhile, proceed with preparing antibody-bound magnetic beads (below).</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Immunoprecipitate RNA bound by protein of interest
<ol type="a">
<li>Wash antibody beads from 6b two times with 500 μL in cold eCLIP Lysis Buffer.</li>
<li>Remove as much supernatant as possible from beads and resuspend once more in 100 μL cold eCLIP Lysis Buffer per sample, typically 400 μL.</li>
<li>Distribute beads very well by pipetting up and down (taking tube off of magnet) and divide bead solution evenly between separate 1.5 mL tubes, one tube per sample.</li>
<li>Only once lysates from 8e are finished centrifuging, apply each bead sample to the magnet and remove supernatant. Add cleared lysate for each sample from 8e to its own antibody bead sample.</li>
<li>Rotate at 4°C for 2 h, or up to 15 h in cold room.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If doing fSHAPE-eCLIP or “in vitro” variant of SHAPE-eCLIP, opt for 2 hours of rotation and skip to “Isolate RNA” section.</div>
<h3 id="sec3.3">Ligate 3′ RNA adapter</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>Complete this section only for SHAPE-eCLIP samples at this time in the protocol. Treat RNA and ligate an adapter, later used for reverse transcription.</p>
<ol start="10">
<li>Setup
<ol type="a">
<li>Verify High Salt Wash Buffer, Wash Buffer, and 1x FastAP Buffers are cold.</li>
<li>Pre-heat Thermomixer to 37°C.</li>
</ol>
</li>
<li>Set aside non-immunoprecipitated sample controls, referred to as “Input” samples
<ol type="a">
<li>Of the sample conditions (“A”,”B”, or “C”; see 1b for naming convention), pick one condition to take sample Input controls from. Typically “B” or “C” conditions. One can carry forward Input controls for all conditions if one chooses, but this creates more controls than is necessary.</li>
<li>Gently mix samples from 9e well. Take <b>two</b> 20 μL aliquots from each replicate of the chosen condition, four <b>aliquots</b> total, adding each to separate tubes labeled “Transfer-Input” and “Western-Input” and their replicate number. <b>Set Inputs aside on ice until step 20.</b>
</li>
<li>The remainder of the samples are referred to as immunoprecipitated (IP) samples.</li>
</ol>
</li>
<li>Wash antibody-bound beads
<ol type="a">
<li>Perform the following washes on ice if possible.</li>
<li>Apply all IP samples to magnet. Wait for beads to pellet on the magnet, removing the supernatant when it is clear.</li>
<li>Wash bead pellets with 900 μL <b>cold</b> High Salt Wash Buffer by pipetting the buffer onto the bead pellets. Wait until supernatant is clear (∼1 min) and remove supernatant by pipetting. Repeat.</li>
<li>Wash beads 2 times as above with 500 μL cold Wash Buffer. During the second wash, move tubes back and forth on magnet (<a href="#mmc1">Methods Video S1</a>). One time is sufficient, such that the bead pellet is forced to re-form. Wait until beads have reformed pellet until removing supernatant, ∼1 min.</li>
<li>Resuspend beads with 500 μL 1X FastAP Buffer, pipetting to completely resuspend beads in buffer.</li>
<li>Allow supernatant to clear and remove supernatant. Add 500 μL 1X FastAP Buffer and place tubes on ice.</li>
</ol>
</li>
<li>FastAP treat RNA on beads
<ol type="a">
<li>Prepare “Early” FastAP master mix on ice (see <a href="#materials-and-equipment">materials and equipment</a>). Mix well.</li>
<li>Place IP bead samples on magnet, removing supernatant when clear. Add 50 μL FastAP master mix to each IP bead sample.</li>
<li>Incubate samples in Thermomixer at 1200 rpm at 37°C for 10 min.</li>
</ol>
</li>
<li>PNK treat RNA on beads
<ol type="a">
<li>While FastAP reaction is incubating, prepare “Early” PNK master mix.</li>
<li>Mix well and add 150 μL master mix to each sample.</li>
<li>Incubate samples in Thermomixer at 1200 rpm at 37°C for 20 min.</li>
</ol>
</li>
<li>Wash beads
<ol type="a">
<li>Apply all IP samples to magnet, removing supernatant when clear.</li>
<li>Wash bead pellets with 500 μL cold Wash Buffer. Move tubes back and forth on magnet, allow supernatant to clear and remove. Repeat wash, moving tubes back and forth on magnet, followed by removal of clear supernatant.</li>
<li>Wash beads 2 times as above with 500 μL cold High Salt Wash Buffer.</li>
<li>Wash beads 2 times as above with 500 μL cold Wash Buffer.</li>
<li>Wash beads with 300 μL 1X Ligase Buffer (no DTT), moving back and forth on magnet and removing cleared supernatant.</li>
<li>Wash beads 2 times with 300 μL 1X Ligase Buffer (no DTT). Keep beads in buffer on ice until next step.</li>
</ol>
</li>
<li>Ligate 3′ RNA adapter
<ol type="a">
<li>Prepare “Early” 3′ RNA Ligation master mix. Mix well by pipetting or flicking.</li>
<li>Remove supernatant from IP bead samples. Briefly spin samples in minifuge, apply to magnet, and use a small pipette tip to remove all residual liquid in tubes.</li>
<li>Add 25 μL of master mix to each IP bead sample.</li>
<li>Add 2.5 μL 40 μM InvRiL19 to each sample.</li>
<li>Incubate at 15°C–25°C for 75 min, flicking tubes every 10 min to mix.</li>
<li>Wash beads 2 times with 500 μL cold Wash Buffer.</li>
</ol>
</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/962-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods Video S1. Example for how tubes can be moved back and forth on the magnet during bead washes, such as in step 12d (tube in the example video does not actually contain beads)</div>
</figcaption>
</figure>
<h3 id="sec3.4">Isolate RNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–24 h</div>
<p>Complete this section for all IP and Input samples and for all protocol types: fSHAPE-eCLIP, SHAPE-eCLIP, or “in vitro” SHAPE-eCLIP. Verify efficacy of the antibody with a Western blot and isolate IP and Input RNA with a gel transfer.</p>
<ol start="17">
<li>Setup
<ol type="a">
<li>Pre-heat thermomixer to 70°C.</li>
<li>Retrieve gel tank(s) and power supply.</li>
<li>Dilute 20X MOPS Running Buffer to 2 L of 1X buffer.</li>
<li>Prepare 2 L of <b>cold</b> 1X NuPAGE Transfer Buffer with 10% methanol. Keep at 4°C.</li>
</ol>
</li>
<li>For only fSHAPE-eCLIP or “in vitro” SHAPE-eCLIP samples from <b>9</b><b>e</b>: set aside non-immunoprecipitated samples as controls, referred to as “Input” samples
<ol type="a">
<li>Of the sample conditions (“A”,”B”, or “C”; see 1b for naming convention) from step 9e, pick one condition to take sample Input controls from. Typically “B” or “C” conditions. One can carry forward Input controls for all conditions if one chooses, but this creates more controls than is necessary.</li>
<li>Gently mix samples from 9e well. Take <b>two</b> 20 μL aliquots from each replicate of the chosen condition, <b>four</b> aliquots total, adding each to separate tubes labeled “Transfer-Input” or “Western-Input” and their replicate number. <b>Set Inputs aside on ice until step 20.</b>
</li>
<li>The remainder of the samples is referred to as immunoprecipitated (IP) samples.</li>
</ol>
</li>
<li>Wash IP bead samples, from step 16f or 18c
<ol type="a">
<li>Apply IP samples to magnet, remove supernatant, and wash two times with 500 μL High Salt Wash Buffer. Remove cleared supernatant.</li>
<li>Resuspend beads in 500 μL cold High Salt Wash Buffer. Add 500 μL cold Wash Buffer. Allow bead pellets to reform on magnet and remove cleared supernatant.</li>
<li>Wash three times with cold Wash Buffer.</li>
</ol>
</li>
<li>Prepare IP and Input samples for gel separation
<ol type="a">
<li>Prepare two 4%–12% Bis-Tris SDS-PAGE gels, 12-well, 1.5 mm. One will be used for a Western Blot (“Western”) and the other for a gel transfer of RNA (“Transfer”).</li>
<li>Remove all supernatant from IP bead samples. Take beads off magnet and resuspend beads well in 100 μL Wash Buffer.</li>
<li>For each resuspended IP bead sample, move 20 μL to a new tube. Label these aliquots with their treatment type, replicate number, and “Western.” These samples will be used in the Western Blot.</li>
<li>For the remaining 80 μL of IP bead samples, apply to magnet, remove all supernatant, and resuspend beads in 20 μL Wash Buffer. These samples will be used for the RNA transfer.</li>
<li>Prepare a loading buffer master mix of 7.5 μL 4X NuPAGE LDS Buffer with 3 μL 1 M DTT for every sample (IPs and Inputs for the Western Blot and RNA Transfer; typically 12 samples total).</li>
<li>Add 10 μL DTT-Buffer mix to each sample and denature all samples in Thermomixer for 10 min at 70°C, 1200 rpm. Samples are now decoupled from beads.</li>
<li>Cool samples on ice for 1 min followed by a brief spin in a minifuge.</li>
<li>Place all tubes (including Inputs) on magnet on ice. Desired material is contained in the supernatant.</li>
</ol>
</li>
<li>Gel separate IP and Input samples
<ol type="a">
<li>Set up the two Bis-Tris gels. Label wells according to <a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a>. Fill tanks with 1X MOPS Running Buffer.
<table id="tbl1" class="dense-table">
<caption>Table 1. Gel set up for Transfer samples</caption>
<thead>
<tr>
<th>Column</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
<th>9</th>
<th>10</th>
<th>11</th>
<th>12</th>
</tr>
<tr>
<th>Sample</th>
<th>M</th>
<th>Input rep1</th>
<th>M</th>
<th>IP, treatment A/B rep1</th>
<th>M</th>
<th>IP, treatment B/C rep1</th>
<th>M</th>
<th>Input rep2</th>
<th>M</th>
<th>IP, treatment A/B rep2</th>
<th>M</th>
<th>IP, treatment B/C rep2</th>
</tr>
</thead>
<tbody>
<tr>
<td>Volume to Load</td>
<td>5 μL</td>
<td>30 μL</td>
<td>5 μL</td>
<td>30 μL</td>
<td>5 μL</td>
<td>30 μL</td>
<td>5 μL</td>
<td>30 μL</td>
<td>5 μL</td>
<td>30 μL</td>
<td>5 μL</td>
<td>30 μL</td>
</tr>
<tr>
<td>% of Sample Represented</td>
<td> </td>
<td>2%</td>
<td> </td>
<td>80%</td>
<td> </td>
<td>80%</td>
<td> </td>
<td>2%</td>
<td> </td>
<td>80%</td>
<td> </td>
<td>80%</td>
</tr>
</tbody>
</table>
<p class="table-legend">M = protein marker (3 uL) in loading buffer (2 uL).</p>
<table id="tbl2" class="dense-table">
<caption>Table 2. Gel set up for Western Blot</caption>
<thead>
<tr>
<th>Column</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>9</th>
<th>10</th>
</tr>
<tr>
<th>Sample</th>
<th>M</th>
<th>Input rep1</th>
<th>IP, treatment A/B rep1</th>
<th>IP, treatment B/C rep1</th>
<th>M</th>
<th>Input rep2</th>
<th>IP, treatment A/B rep2</th>
<th>IP, treatment B/C rep2</th>
<th>M</th>
</tr>
</thead>
<tbody>
<tr>
<td>Volume to Load</td>
<td>5 μL</td>
<td>15 μL</td>
<td>15 μL</td>
<td>15 μL</td>
<td>5 μL</td>
<td>15 μL</td>
<td>15 μL</td>
<td>15 μL</td>
<td>5 μL</td>
</tr>
<tr>
<td>% of Sample Represented</td>
<td> </td>
<td>1%</td>
<td>10%</td>
<td>10%</td>
<td> </td>
<td>1%</td>
<td>10%</td>
<td>10%</td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">M = protein marker in loading buffer.</p>
</li>
<li>Load IP and Input RNA Transfer samples into gel according to map in <a href="#tbl1">Table 1</a>. Run at 150 V for 75 min, or until dye front is at bottom.</li>
<li>Load half of IP and Input Western Blot samples (15 μL) into gel according to map in <a href="#tbl2">Table 2</a>. Save the remainder of Western Blot samples at <i>−</i>20°C for <a href="#troubleshooting">troubleshooting</a>. Run at 150 V for 75 min, or until dye front is at bottom.</li>
</ol>
</li>
<li>Set up membranes while gels are running
<ol type="a">
<li>For the Transfer gel: cut out four Whatman papers, one nitrocellulose membrane, and two sponges, all sized to fit the gel.</li>
<li>For the Western gel: cut out four Whatman papers, one PVDF membrane, and two sponges, all sized to fit the gel.</li>
<li>Fill two separate shallow trays half-way with cold 1X Transfer buffer with 10% methanol. Place cassettes for Western Blot/RNA Transfer open in trays.</li>
<li>Place sponges, Whatman papers, and nitrocellulose/PVDF membranes in trays to soak. Use rollers to roll bubbles out of sponges if necessary.</li>
</ol>
</li>
<li>Western Blot and RNA Transfer
<ol type="a">
<li>Once gels are finished running, carefully remove one side of their casing. Using a straight edge, remove extraneous portions of the gels: the bottom below the dye front, the wells. Make sure to note the samples’ orientation in the gels.</li>
<li>Set up the RNA Transfer inside open cassette in the shallow tray. Carefully stack layers as follows, using a small roller carefully roll out bubbles between additions, such that each layer is flush to the one below.
<ol type="i">
<li>Sponge</li>
<li>2 Whatman papers</li>
<li>Gel from 21b</li>
<li>Nitrocellulose membrane</li>
<li>2 Whatman papers</li>
<li>Sponge</li>
</ol>
</li>
<li>Set up the Western Blot inside open cassette in the shallow tray. Carefully stack layers as follows, using a small roller carefully roll out bubbles between additions, such that each layer is flush to the one below.
<ol type="i">
<li>Sponge</li>
<li>2 Whatman papers</li>
<li>Gel from 21c</li>
<li>PVDF membrane</li>
<li>2 Whatman papers</li>
<li>Sponge</li>
</ol>
</li>
<li>Close each cassette. Cassettes should snugly fit the RNA Transfer/Western Blot, holding everything in place. If loose, add another sponge on top. <b>Note the side of the cassette that the nitrocellulose/PVDF membrane is closest to.</b>
</li>
<li>Place cassettes in a wet transfer tank. <b>The side of the cassette that is closest to the nitrocellulose/PVDF membrane should face the positive electrode in the tank</b>, such that material flows from the gel into the membrane.</li>
<li>Fill transfer tank with cold 1X Transfer Buffer with 10% methanol.</li>
<li>Run at 30V 10–20 h in cold room (preferred for Transfer), or at 200 mA for 2 h.</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Develop the western blot</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>Develop the Western Blot to confirm that immunoprecipitation was successful.</p>
<ol start="24">
<li>Setup
<ol type="a">
<li>Preheat Thermomixer to 37°C.</li>
<li>In the meantime, continue to run the RNA Transfer (recommended). Or, remove the RNA Transfer nitrocellulose membrane, rinse briefly with sterile 1X PBS, wrap in plastic wrap, and store at <i>−</i>20°C.</li>
</ol>
</li>
<li>Develop western blot
<ol type="a">
<li>Stop Western Blot transfer and remove PVDF membrane from stack.</li>
<li>Place a small cut on the membrane for noting its orientation.</li>
<li>Incubate membrane in Blocking Buffer (5% milk in 1X TBST Buffer) at 15°C–25°C for 30 min with gentle mixing.</li>
<li>Dilute primary antibody in Blocking Buffer to a final antibody concentration of 0.2–0.5 ug/mL. Incubate membrane with antibody at 15°C–25°C for 1 h with gentle mixing.</li>
<li>Move membrane to fresh container in 1X TBST Buffer. Let incubate for 5 min with gentle mixing. Repeat 2 times.</li>
<li>Dilute secondary antibody, rabbit or mouse TruBlot HRP (depending on whether primary antibody is derived from rabbit or mouse etc), to a final dilution of 1:2000 in Blocking Buffer. Incubate at 15°C–25°C for 1–3 h with gentle mixing.</li>
<li>Move membrane to fresh container in 1X TBST Buffer. Let incubate for 5 min with gentle mixing. Repeat 2 times.</li>
<li>Mix equal parts ECL Reagent 1 and 2, 0.5 mL each. Apply to membrane and incubate in place for 1 min.</li>
<li>Remove membrane from solution, touching edges of membrane to paper towels to absorb excess solution. Encase membrane in protective clear plastic.</li>
<li>Expose membrane to film in dark room for 30 s and 5 min time spans. Adjust exposure times as necessary to render a clear visualization of bands.</li>
<li>Verify that protein band appears for IP samples and that it indicates the expected size for the given protein being immunoprecipitated. See <a href="#fig1">Figure 1</a>A and <a href="#sec7.1">troubleshooting 1</a>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/962-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. RNA isolation steps</div>
<p>(A) Western blot of immunoprecipitated (IP) and Input control (f)SHAPE-eCLIP samples for stem loop binding protein (SLBP).</p>
<p>(B) (f)SHAPE-eCLIP RNA samples transferred to nitrocellulose membrane, with excised samples.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Bands in the Western Blot may be much darker for the IP samples that were not treated with NAI <i>in vivo</i>, indicating greater yields for these samples. This is normal and likely due to <i>in vivo</i> NAI treatment unavoidably resulting in some cellular stress/death.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> RNA transfer may continue to run at 4°C for 10–20 hours total. Alternatively, the RNA Transfer nitrocellulose membrane may be rinsed briefly with sterile 1X PBS, wrapped in plastic wrap, and stored at <i>−</i>20°C for ∼1 day.</div>
<h3 id="sec3.6">Isolate RNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Extract RNA from nitrocellulose membrane for all IP and Input samples.</p>
<ol start="26">
<li>Setup
<ol type="a">
<li>Preheat Thermomixer to 37°C.</li>
<li>Fetch sterile razor blades, one per sample (six blades).</li>
<li>Clean a glass plate (or other hard surface to cut against) with ethanol and/or RNase-removing solution.</li>
<li>Fetch 1.5 mL tubes, one per sample.</li>
</ol>
</li>
<li>Cut RNA Transfer nitrocellulose membrane
<ol type="a">
<li>Fetch nitrocellulose membrane from RNA transfer. Note expected positioning of protein band within each lane based on the western blot. See <a href="#fig1">Figure 1</a>A.</li>
<li>Cut the IP and Input sample lanes using a fresh razor blade for each lane.
<ol type="i">
<li>Cuts should start at the expected size of the immunoprecipitated protein (40 kDa for SLBP) and extend to 75 kDa above (<a href="#fig1">Figure 1</a>B). (75 kDa correspond to the mass of an RNA molecule that is ∼230 bases long).</li>
<li>Further cut each excised lane into 1–2 mm pieces. Place pieces into 1.5 mL tubes and label with sample name (<a href="#mmc2">Methods Video S2</a>).</li>
</ol>
</li>
</ol>
</li>
<li>Extract RNA from membrane
<ol type="a">
<li>Prepare 150 μL Proteinase K SDS mix per sample, consisting of 130 μL PKS Buffer + 20 μL Proteinase K.</li>
<li>Add Proteinase K mixture to membrane pieces, ensuring all pieces are submerged.</li>
<li>Incubate in Thermomixer 37°C for 20 min, 1200 rpm.</li>
<li>Further incubate in Thermomixer at 50°C for 20 min, 1200 rpm.</li>
<li>Transfer solution in each tube to new 1.5 mL tube.</li>
<li>Add 55 μL water to leftover membrane pieces. Transfer water to supernatant above.</li>
</ol>
</li>
<li>Purify RNA with RNA Clean and Concentrator Kit column cleanup
<ol type="a">
<li>To each sample add 400 μL (2x starting volume) RNA binding buffer to supernatant from membrane (step 28 f), pipetting to mix.</li>
<li>Add 700 μL (3.5x starting volume) of 100% ethanol and mix well.</li>
<li>Transfer 650 μL of each sample to a spin column (the max volume column can accommodate at once).</li>
<li>Centrifuge columns for 30 s at 5000 <i>g</i> at 15°C–25°C. Add flow-through back into columns and centrifuge again. Discard flow-through.</li>
<li>Load remaining sample to the same column and centrifuge, re-load, centrifuge as before. Discard flow-through.</li>
<li>Wash columns with the following buffers, centrifuging for 30 s at 5000 g, discarding flow-through each time:
<ol type="i">
<li>400 μL RNA Prep Buffer</li>
<li>500 μL RNA Wash Buffer</li>
<li>500 μL RNA Wash Buffer</li>
<li>200 μL RNA Wash Buffer. Centrifuge for 1 min at 9000 <i>g</i>.</li>
</ol>
</li>
<li>Centrifuge for 2 min at max speed to dry columns.</li>
<li>Transfer columns to new 1.5 mL tubes.</li>
<li>Add 10 μL water to columns, sit for 1 min, and centrifuge for 30 s at 9000 <i>g</i>.</li>
<li>Add flow-through back onto columns and let sit and centrifuge at 9000 <i>g</i> as before. Flow-through contains purified RNA.</li>
<li>Samples may be stored at <i>−</i>80°C.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples may be kept at <i>−</i>80°C for a few days.</div>
<div class="note">
<span class="note-title">Note:</span> If following SHAPE-eCLIP protocol skip to “Ligate RNA (late)”. fSHAPE-eCLIP and the “in vitro” variant of SHAPE-eCLIP proceed to “In vitro RNA treament.”</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/962-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods Video S2. Technique for cutting transferred RNA sample in nitrocellulose membrane into 1–2 mm pieces, step 27b</div>
</figcaption>
</figure>
<h3 id="sec3.7">In vitro RNA treatment with NAI</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Complete this section only for treatment condition “B” IP samples, either for fSHAPE-eCLIP or the “in vitro” variant of SHAPE-eCLIP (see step 1). Refold IP RNA samples and treat with structure probing reagent NAI.</p>
<ol start="30">
<li>Setup
<ol type="a">
<li>Samples not undergoing this step should be set aside on ice.</li>
<li>Preheat thermocycler or heat block to 95°C.</li>
</ol>
</li>
<li>Probe RNA with NAI
<ol type="a">
<li>Heat ∼10 μL “B” IP samples to 95°C for 2 min, then place on ice for 1 min.</li>
<li>Prepare “in vitro structure probing” reaction with heated samples, without NAI (see <a href="#materials-and-equipment">materials and equipment</a>).</li>
<li>Heat to 37°C for 5 min to re-fold RNA.</li>
<li>Add 1 μL 2 M NAI to each sample and continue heating at 37°C for 10 min.</li>
</ol>
</li>
<li>Purify RNA with RNA Clean and Concentrator Kit column cleanup
<ol type="a">
<li>To each sample add 42 μL (2x starting volume) RNA binding buffer, pipetting to mix.</li>
<li>Add 73.5 μL (3.5x starting volume) of 100% ethanol and mix well.</li>
<li>Transfer the entirety of each sample to a spin column.</li>
<li>Centrifuge columns for 30 s at 5000 <i>g</i> at 15°C–25°C . Add flow-through back into columns and centrifuge again. Discard flow-through.</li>
<li>Load remaining sample to column and centrifuge, re-load, centrifuge as before. Discard flow-through.</li>
<li>Wash columns with the following buffers, centrifuging for 30 s at 5000 <i>g</i>, discarding flow-through each time:
<ol type="i">
<li>400 μL RNA Prep Buffer</li>
<li>500 μL RNA Wash Buffer</li>
<li>500 μL RNA Wash Buffer</li>
<li>200 μL RNA Wash Buffer. Centrifuge for 1 min at 9000 <i>g</i>.</li>
</ol>
</li>
<li>Centrifuge for 2 min at max speed to dry columns.</li>
<li>Transfer columns to new 1.5 mL tubes.</li>
<li>Add 10 μL water to columns, sit for 1 min, and centrifuge for 30 s at 9000 <i>g</i>.</li>
<li>Add flow-through back onto columns and let sit and centrifuge as before. Flow-through contains purified RNA.</li>
<li>Samples may be stored at <i>−</i>80°C.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples may be kept at <i>−</i>80°C for a few days.</div>
<h3 id="sec3.8">Ligate 3′ RNA adapter (late)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>For the SHAPE-eCLIP protocol complete this section for Input samples only. For fSHAPE-eCLIP or “in vitro” SHAPE-eCLIP protocols complete this section for all IP and Input samples. Ligate an adapter, later used for reverse transcription.</p>
<ol start="33">
<li>Setup
<ol type="a">
<li>Preheat Thermomixer to 37°C.</li>
<li>If following SHAPE-eCLIP protocol, set IP samples aside on ice until “Reverse Transcribe RNA” section. These sample will have already undergone RNA adapter ligation.</li>
</ol>
</li>
<li>FastAP treat RNA
<ol type="a">
<li>Prepare “Late” FastAP master mix on ice (see <a href="#materials-and-equipment">materials and equipment</a>). Mix well.</li>
<li>Add 11 μL FastAP master mix to each sample.</li>
<li>Incubate samples in Thermomixer at 1200 rpm at 37°C for 10 min.</li>
</ol>
</li>
<li>PNK treat RNA
<ol type="a">
<li>While FastAP reaction is incubating, prepare “Late” PNK master mix.</li>
<li>Mix well and add 75 μL master mix to each sample.</li>
<li>Incubate samples in Thermomixer at 1200 rpm at 37°C for 20 min.</li>
</ol>
</li>
<li>Purify RNA with RNA Clean and Concentrator Kit column cleanup
<ol type="a">
<li>To each sample add 200 μL (2x starting volume) RNA binding buffer, pipetting to mix.</li>
<li>Add 300 μL (3x starting volume) of 100% ethanol and mix well.</li>
<li>Transfer the entirety of each sample to a spin column.</li>
<li>Centrifuge columns for 30 s at 5000 <i>g</i> at 15°C–25°C. Add flow-through back into columns and centrifuge again. Discard flow-through.</li>
<li>Load remaining sample to column and centrifuge, re-load, centrifuge as before. Discard flow-through.</li>
<li>Wash columns with the following buffers, centrifuging for 30 s at 5000 <i>g</i>, discarding flow-through each time:
<ol type="i">
<li>400 μL RNA Prep Buffer</li>
<li>500 μL RNA Wash Buffer</li>
<li>500 μL RNA Wash Buffer</li>
<li>200 μL RNA Wash Buffer. Centrifuge for 1 min at 9000 <i>g</i>.</li>
</ol>
</li>
<li>Centrifuge for 2 min at max speed to dry columns.</li>
<li>Transfer columns to new 1.5 mL tubes.</li>
<li>Add 10 μL water to columns, sit for 1 min, and centrifuge for 30 s at 9000 <i>g</i>.</li>
<li>Add flow-through back onto columns and let sit and centrifuge as before. Flow-through contains purified RNA.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Treated RNA may be stored at <i>−</i>80°C.</div>
<ol start="37">
<li>Ligate 3′ RNA adapter
<ol type="a">
<li>Preheat Thermomixer or heat block to 65°C.</li>
<li>Prepare “Late” 3′ RNA Ligation master mix for Inputs and “Late” 3′ RNA Ligation mix for IP samples if treating IP samples at this stage (i.e., for fSHAPE-eCLIP or “in vitro” SHAPE-eCLIP).</li>
<li>For Input samples: remove half of samples (5 μL) and store at <i>−</i>80°C as backups.</li>
<li>For IP samples: use entire sample.</li>
<li>Incubate samples with InvRiL19 adapter:
<ol type="i">
<li>Add 2 μL of 40 μM InvRiL19 adapter and 1.5 μL DMSO to Input samples.</li>
<li>Add 4 μL of 40 μM InvRiL19 adapter and 2.4 μL DMSO to IP samples (for fSHAPE-eCLIP or “in vitro” SHAPE-eCLIP).</li>
<li>Incubate at 65°C for 2 min.</li>
<li>Snap cool on ice for 1 min.</li>
</ol>
</li>
<li>Add 13.5 μL of master mix to Input samples and 27 μL to IP samples. Mix well.</li>
<li>Incubate at 15°C–25°C for 60 min, flicking tubes every 15 min to mix.</li>
</ol>
</li>
<li>Cleanup samples with Silane magnetic beads
<ol type="a">
<li>Prepare beads:
<ol type="i">
<li>Take 10 μL MyONE Silane Beads per Input sample and 20 μL per sample.</li>
<li>Add 5x volumes RLT Buffer to beads. (500 μL RLT Buffer to 100 μL of beads). Mix well.</li>
<li>Apply to magnet and remove cleared supernatant.</li>
<li>Add 6.3x volumes RLT<b>W</b> Buffer to beads. (630 μL RTLW Buffer to originally 100 μL of beads). Mix well.</li>
</ol>
</li>
<li>Bind RNA samples to beads:
<ol type="i">
<li>Add 61 μL beads in RLTW Buffer to each Input sample, 122 μL to each IP sample. Mix well.</li>
<li>Add 65 μL 100% ethanol to each Input sample, 130 μL to each IP sample.</li>
<li>Pipette mix each sample 10 times and leave pipette tip in tube.</li>
<li>Incubate with beads for 10 min, pipetting to mix every 3–5 min.</li>
</ol>
</li>
<li>Wash beads:
<ol type="i">
<li>Prepare 20 mL 80% ethanol.</li>
<li>Apply samples to magnet and discard cleared supernatant.</li>
<li>Add 300 μL 80% ethanol to samples if in 0.2 mL tubes, or 1 mL if in 1.5 mL tubes. Mix well.</li>
<li>Wait 30 s, then apply samples to magnet and discard cleared supernatant.</li>
<li>Repeat wash with 80% ethanol. Discard supernatant.</li>
<li>Repeat wash with 100 μL or 750 μL 80% ethanol. Discard supernatant.</li>
<li>Briefly spin tubes, apply to magnet, and remove residual supernatant with a fine tip.</li>
<li>Leave tubes open to let beads dry, ∼5 min. Beads are dry when they no longer shine or look wet. (Beads are over-dried if they change color from brown to burnt orange.)</li>
</ol>
</li>
<li>Elute RNA from beads:
<ol type="i">
<li>Resuspend beads in 9.5 μL TT Elution Buffer. Sit for 5 min.</li>
<li>Apply samples to magnet and capture supernatant, which contains RNA samples. Move supernatant to new 0.2 mL tubes.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples may be stored at <i>−</i>80°C for up to two weeks.</div>
<h3 id="sec3.9">Reverse transcribe RNA and ligate 5′ cDNA adapter</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h + 10–20 h incubation</div>
<p>Complete this section for all samples and all protocol types. Reverse transcribe RNA with manganese to perform “mutational profiling,” followed by 5′ adapter ligation to the cDNA.</p>
<ol start="39">
<li>Setup
<ol type="a">
<li>Preheat thermocycler or heat block to 65°C.</li>
</ol>
</li>
<li>Anneal primer
<ol type="a">
<li>To each ∼9 μL sample add 1 μL 5 μM InvAR17 primer and 1 μL 10 mM dNTPs.</li>
<li>Heat to 65°C for 2 min. Snap cool on ice for one minute. (Do not let cool down on heat block).</li>
</ol>
</li>
<li>Reverse transcribe
<ol type="a">
<li>Set thermocycler or heat block to 45°C.</li>
<li>Prepare SHAPE reverse transcription master mix (see <a href="#materials-and-equipment">materials and equipment</a>).</li>
<li>Add 10 μL master mix to each sample. Mix well.</li>
<li>Incubate at 45°C for 3 h.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The reverse transcription here requires all 3 hours, since manganese-induced errors slow down the RT.</div>
<div class="note">
<span class="note-title">Note:</span> Errors are induced in the cDNA by manganese when the RT encounters NAI adducts in the RNA. This “mutational profiling” records the location of NAI reactions with RNA bases and is a crucial step of RNA secondary structure probing.</div>
<ol start="42">
<li>Isolate cDNA
<ol type="a">
<li>Treat with ExoSAP-IT
<ol type="i">
<li>Set heat block to 37°C.</li>
<li>Add 3.5 μL ExoSAP-IT to each sample. Vortex and briefly spin down.</li>
<li>Incubate at 37°C for 15 min.</li>
<li>Add 1 μL 0.5 M EDTA to each sample and mix.</li>
</ol>
</li>
<li>Remove RNA
<ol type="i">
<li>Set heat block to 70°C.</li>
<li>Add 3 μL 1 M NaOH to each sample and mix.</li>
<li>Incubate at 70°C for 10 min.</li>
<li>Add 3 μL 1 M HCl and mix, to neutralize NaOH.</li>
</ol>
</li>
</ol>
</li>
<li>Cleanup cDNA with Silane magnetic beads
<ol type="a">
<li>Prepare beads:
<ol type="i">
<li>Take 5 μL MyONE Silane Beads per sample.</li>
<li>Add 5x volumes RLT Buffer to beads. (150 μL RLT Buffer to 30 μL of beads). Mix well.</li>
<li>Apply to magnet and remove cleared supernatant.</li>
<li>Add 18.6x volumes RLT<b>W</b> Buffer to beads. (558 μL RTLW Buffer to originally 30 μL of beads). Mix well.</li>
</ol>
</li>
<li>Bind cDNA samples to beads:
<ol type="i">
<li>Add 90 μL beads in RLTW Buffer to each sample. Mix well.</li>
<li>Add 108 μL 100% ethanol to each sample.</li>
<li>Pipette mix each sample 10 times and leave pipette tip in tube.</li>
<li>Incubate with beads for 10 min, pipetting to mix every 3–5 min.</li>
</ol>
</li>
<li>Wash beads:
<ol type="i">
<li>Prepare 20 mL 80% ethanol.</li>
<li>Apply samples to magnet and discard cleared supernatant.</li>
<li>Add 300 μL 80% ethanol to samples if in 0.2 mL tubes, or 1 mL if in 1.5 mL tubes. Mix well.</li>
<li>Wait 30 s, then apply samples to magnet and discard cleared supernatant.</li>
<li>Repeat wash with 80% ethanol. Discard supernatant.</li>
<li>Repeat wash with 100 μL or 750 μL 80% ethanol. Discard supernatant.</li>
<li>Briefly spin tubes, apply to magnet, and remove residual supernatant with a fine tip.</li>
<li>Leave tubes open to let beads dry, ∼5 min. Beads are dry when they no longer shine or look wet. (Beads are over-dried if they change color from brown to burnt orange.)</li>
</ol>
</li>
</ol>
</li>
<li>Ligate 5′ cDNA linker on beads
<ol type="a">
<li>Set heat block to 70°C.</li>
<li>Make cDNA adapter mix, per sample:
<ol type="i">
<li>1.1 μL TT Elution Buffer</li>
<li>0.6 μL InvRand3Tr3 adapter</li>
<li>0.8 μL 100% DMSO</li>
<li>Heat at 70°C for 2 min.</li>
<li>Snap cool on ice for 1 min.</li>
</ol>
</li>
<li>Add 3 μL cDNA adapter mix to each dried Silane bead sample.</li>
<li>Prepare 5′ cDNA linker ligation master mix (See <a href="#materials-and-equipment">materials and equipment</a>).</li>
<li>Add 7.8 master mix to each bead sample by first stirring beads with pipette tip, then slowly pipetting the viscous master mix while stirring until homogenous.</li>
<li>Incubate at 15°C–25°C 10–20 h on a rotator.</li>
</ol>
</li>
<li>Cleanup ligated cDNA with Silane magnetic beads
<ol type="a">
<li>Add 5 μL TT Elution Buffer to each sample to total to 15 μL.</li>
<li>Prepare beads:
<ol type="i">
<li>Prepare 2.5 μL MyONE Silane Beads per sample.</li>
<li>Add 5x volumes RLT Buffer to beads. (75 μL RLT Buffer to 15 μL of beads). Mix well.</li>
<li>Apply to magnet and remove cleared supernatant.</li>
<li>Add 18.8x volumes RLT<b>W</b> Buffer to beads. (282 μL RTLW Buffer to originally 15 μL of beads). Mix well.</li>
</ol>
</li>
<li>Bind cDNA samples to beads:
<ol type="i">
<li>Add 45 μL beads in RLTW Buffer to each sample. Mix well.</li>
<li>Add 45 μL 100% ethanol to each sample.</li>
<li>Pipette mix each sample 10 times and leave pipette tip in tube.</li>
<li>Incubate with beads for 10 min, pipetting to mix every 3–5 min.</li>
</ol>
</li>
<li>Wash beads:
<ol type="i">
<li>Prepare 20 mL 80% ethanol.</li>
<li>Apply samples to magnet and discard cleared supernatant.</li>
<li>Add 300 μL 80% ethanol to samples if in 0.2 mL tubes, or 1 mL if in 1.5 mL tubes. Mix well.</li>
<li>Wait 30 s, then apply samples to magnet and discard cleared supernatant.</li>
<li>Repeat wash with 80% ethanol. Discard supernatant.</li>
<li>Repeat wash with 100 μL or 750 μL 80% ethanol. Discard supernatant.</li>
<li>Briefly spin tubes, apply to magnet, and remove residual supernatant with a fine tip.</li>
<li>Leave tubes open to let beads dry, ∼5 min. Beads are dry when they no longer shine or look wet. (Beads are over-dried if they change color from brown to burnt orange.)</li>
</ol>
</li>
<li>Elute cDNA from beads:
<ol type="i">
<li>Resuspend beads in 25 μL TT Elution Buffer. Sit for 5 min.</li>
<li>Apply samples to magnet and capture supernatant, which contains cDNA samples. Move supernatant to new 1.5 mL tubes.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store samples at <i>−</i>80°C.</div>
<h3 id="sec3.10">Quantify and amplify cDNA library</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8 h</div>
<p>Complete this section for all samples and all protocol types. Quantify cDNA with quantitative PCR (qPCR), also known as “real time PCR”, followed by amplification and gel purification of libraries.</p>
<ol start="46">
<li>qPCR
<ol type="a">
<li>Prepare cDNA qPCR master mix (see <a href="#materials-and-equipment">materials and equipment</a>).</li>
<li>For each sample, add 9 μL master mix to a well in a 384 well plate, or other plate compatible with qPCR instrument.</li>
<li>Add 1 μL of a 1:10 dilution of each cDNA sample to the wells in the 384 well plate.</li>
<li>Seal the plate with clear film.</li>
<li>Execute qPCR according to the preset protocol in <a href="#tbl3">Table 3</a>.
<table id="tbl3">
<caption>Table 3. Quantify libraries with qPCR</caption>
<thead>
<tr>
<th colspan="4">qPCR protocol</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95 C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95 C</td>
<td>15 s</td>
<td rowspan="3">30</td>
</tr>
<tr>
<td>Annealing &amp; extension</td>
<td>60 C</td>
<td>1 min</td>
</tr>
<tr>
<td>Take image</td>
<td>NA</td>
<td>NA</td>
</tr>
</tbody>
</table>
</li>
<li>Record Ct calculated for each sample. Typical Input sample Ct: 12. IP sample Ct: 19. See <a href="#expected-outcomes">expected outcomes</a>.</li>
</ol>
</li>
<li>PCR amplify cDNA library
<ol type="a">
<li>Prepare cDNA PCR master mix, selecting a unique combination of forward and reverse (D50x/D70x) primers for each sample.</li>
<li>Add PCR master mix to 16 μL of each cDNA sample for 40 μL total.</li>
<li>The PCR cycle number for each sample should be 3 cycles less than the qPCR-calculated Ct for its 1:10 dilution. Example:
<ol type="i">
<li>Typical Input: 12−3 = 9 total PCR cycles</li>
<li>Typical IP: 19<i>−</i>3 = 16 total PCR cycles</li>
</ol>
</li>
<li>The PCR consists of two stages, the first of which always consists of 6 cycles. The second stage should be repeated such that the total number of cycles attains the preferred total cycles calculated above. Example:
<ol type="i">
<li>Typical Input: 6 cycles + 3 cycles = 9 total PCR cycles</li>
<li>Typical IP: 6 cycles + 10 cycles = 16 total PCR cycles</li>
</ol>
</li>
<li>Amplify libraries according to PCR protocol in <a href="#tbl4">Table 4</a>.
<table id="tbl4">
<caption>Table 4. Amplify libraries with PCR</caption>
<thead>
<tr>
<th colspan="4">PCR protocol</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98 C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98 C</td>
<td>15 s</td>
<td rowspan="3">6</td>
</tr>
<tr>
<td>Annealing</td>
<td>68 C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72 C</td>
<td>40 s</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98 C</td>
<td>15 s</td>
<td rowspan="2">Ct# - 3 - 6</td>
</tr>
<tr>
<td>Annealing &amp; extension</td>
<td>72 C</td>
<td>60 s</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72 C</td>
<td>60 s</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4 C</td>
<td>Forever</td>
<td>1</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>Cleanup library with AmpureXP beads
<ol type="a">
<li>Prepare 10 mL 80% ethanol.</li>
<li>Leave beads out at 15°C–25°C 15 min before use.</li>
<li>Vortex beads well.</li>
<li>Add 72 μL bead suspension to each PCR reaction. Mix well.</li>
<li>Incubate at 15°C–25°C for 10 min, mixing 2–3 times during this time.</li>
<li>Apply samples to magnet and remove cleared supernatant.</li>
<li>Wash beads on magnet 3 times with 100 μL 80 ethanol (if in 0.2 mL tubes) or 500 μL (if in 1.5 mL tubes).</li>
<li>Air dry beads for 5 min on magnet. Avoid over-drying (pellet will appear cracked).</li>
<li>Resuspend beads in 20 μL PCR Elution Buffer, incubating for 5 min at 15°C–25°C .</li>
<li>Apply samples to magnet and move supernatant containing cleaned cDNA library to new tubes.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store samples at <i>−</i>80°C.</div>
<ol start="49">
<li>Gel purify cDNA library
<ol type="a">
<li>Prepare 3% low melting temperature agarose gel in 1X TBE Buffer.
<ol type="i">
<li>Prepare 50 mL total for 7 × 10 cm tray, 80 mL for 15 × 7 cm tray, 120 mL for 15 × 10 cm tray</li>
<li>Heat TBE Buffer and gradually add agarose powder while stirring.</li>
<li>Completely dissolve agarose by intermittently heating solution in microwave in 30 s intervals followed by stirring/swirling.</li>
<li>Add 1:10000 SYBR Safe gel stain. Mix well and pour agarose solution into tray.</li>
</ol>
</li>
<li>Separate library samples on gel
<ol type="i">
<li>Add 6 μL 6X OrangeG Buffer to each cleaned sample from step 48j (∼24 μL total). Mix well.</li>
<li>Prepare 50 base-pair ladder in OrangeG Buffer, enough for two wells.</li>
<li>Move agarose gel into gel tank in 1X TBE Buffer. Load samples into wells with ladder on each end.</li>
<li>Run gel at ∼95 V for 50 min. More time will result in finer separation, but larger gel cuts. See example in <a href="#expected-outcomes">expected outcomes</a>.</li>
</ol>
</li>
<li>Cut gel
<ol type="i">
<li>Visualizing with blue light, cut each lane from 175–350 base-pairs in size with a fresh razor blade for each sample. Place cut gel pieces into 15 mL conical tube.
<div class="note">
<span class="note-title">Note:</span> primer-dimers run at 142 base-pairs. Avoid cutting too close to this size.</div>
</li>
<li>Weigh the gel slice in each conical, taring with an empty 15 mL conical tube.</li>
</ol>
</li>
<li>Extract libraries from gel with Qiagen MinElute gel extraction kit
<ol type="i">
<li>Add 6x μL of Buffer QG for every 100 mg of gel. Ex: 600 μL Buffer QG for 100 mg gel.</li>
<li>Incubate gel in buffer at 15°C–25°C . Can shake or swirl to help dissolve; no vortexing.</li>
<li>Add 1x μL of isopropanol and mix very well. Ex: 100 μL isopropanol for 100 mg original weight of gel slice.</li>
<li>Load a maximum of 750 μL dissolved gel solution onto column and spin in centrifuge at max speed for 1 min. Discard flow-through. Repeat until all solution has been spun through.</li>
<li>For gel weights &gt;400 mg, spin column with 500 μL Buffer QG after every 4 spins.</li>
<li>After all sample has been spun through column, wash column once with 500 μL Buffer QG at max speed for 1 min.</li>
<li>Add 750 μL Buffer PE to column and spin at max speed for 1 min. Discard flow-through and repeat wash, spinning for 2 min.</li>
<li>Move column to a new 1.5 mL tube. If column has any residual Buffer PE on the outside, repeat 2 min spin to dry completely.</li>
<li>With a fine pipette tip, move tip around the rim lining the filter inside the column taking up any residual liquid around rim. (No residual Buffer PE should be present in final libraries).</li>
<li>Air dry columns for 2 min.</li>
<li>Add 12.5 μL Buffer EB to the center of column. Incubate for 2 min. Spin at max speed for 1 min. Add eluate back into column and repeat spin for improved yield.</li>
</ol>
</li>
<li>Quantitate libraries
<ol type="i">
<li>Run a sample of each library on a D1000 DNA TapeStation, Bioanalyzer, or equivalent.</li>
<li>Quantify library peaks between ∼150–600 base-pairs. See example in <a href="#expected-outcomes">expected outcomes</a> and <a href="#sec7.3">troubleshooting 2</a> and <a href="#sec7.5">3</a>. Sufficient library concentration is &gt;= 5 nM per library.</li>
</ol>
</li>
<li>Sequence
<ol type="i">
<li>Pool libraries according to a final molarity specified by your sequencing provider. Each Input sample library may be pooled at half the concentration that IP sample libraries are pooled at. For example, if pooling samples to a final concentration of 10 nM, four IP libraries could each comprise 2 nM and two Input libraries could each comprise 1 nM.</li>
<li>Sequence single-end libraries aiming for a depth of at least 50 million reads per library.</li>
</ol>
</li>
<li>Process sequencing reads with the (f)SHAPE-eCLIP pipeline. See <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Success of the (f)SHAPE-eCLIP libraries will be first apparent after qPCR. The library preparation has likely been successful if the Ct number of its 1:10 dilution is in the expected range (<a href="#tbl5">Table 5</a>). Libraries are size-selected on an agarose gel (below).</p>
<table id="tbl5">
<caption>Table 5. Expected PCR cycle numbers for (un)successful libraries</caption>
<thead>
<tr>
<th>Sample library type</th>
<th>Good qPCR Ct</th>
<th>Bad qPCR Ct</th>
</tr>
</thead>
<tbody>
<tr>
<td>Input</td>
<td>9-15</td>
<td>&gt;15</td>
</tr>
<tr>
<td>IP</td>
<td>&lt;21</td>
<td>&gt;=21</td>
</tr>
</tbody>
</table>
<p>Successful cDNA libraries after PCR amplification and bead purification are visualized on a gel as a smear in the 150–350 bp range (<a href="#fig2">Figure 2</a>). Dotted outlines in the figure above indicate the approximate region boundaries for excision of libraries (175–350 bp). Star indicates adapter dimer and cross indicates unincorporated PCR primers remaining after bead purification. With conservative gel excision, both of these should be separated from the libraries with high confidence.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/962-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. (f)SHAPE-eCLIP libraries purified on agarose gel</div>
</figcaption>
</figure>
<p>Successful final libraries as visualized and quantified on a TapeStation instrument are expected to resemble <a href="#fig3">Figure 3</a>. The “region molarity” value is the concentration used to determine the pooling strategy for individual samples prior to sequencing and will ideally be &gt;5 nM for any given library.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/962-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Visualized final (f)SHAPE-eCLIP library</div>
</figcaption>
</figure>
<p>fSHAPE Expected Outcomes:</p>
<p>The (f)SHAPE-eCLIP wet lab protocol, not fSHAPE, is described in this manuscript. For an fSHAPE protocol, follow the published icSHAPE protocol (<a href="#bib6">Flynn et al., 2016</a>) for both “in vitro” and “in vivo” icSHAPE, ignoring steps for the DMSO-treated control. While most of the fSHAPE library preparation does not have visualization steps, after PCR and gel purification it will be clear if the protocol was successful. Successful fSHAPE libraries are indicated by ample dsDNA products &gt;150 bp in size. See <a href="#sec7.7">troubleshooting 4</a>.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>fSHAPE-eCLIP and SHAPE-eCLIP analysis:</p>
<p>The pipeline for (f)SHAPE-eCLIP is written in common workflow language (CWL) and run with cwltool and pre-built Docker containers holding the software necessary for each step of the pipeline. The pipeline requires at least 8 cores and 60 Gb of memory. If opting for an AWS EC2 instance we recommend a Centos7 AMI (ami-02676464a065c9c05, located on us-west-1) with either c3.8xlarge or r4.2xlarge machines and 500 Gb of attached storage.</p>
<p>Dependencies:</p>
<p>Docker</p>
<p>cwltool==3.0.20200706173533</p>
<p>pip==20.1.1</p>
<p>Logging into an AWS EC2 instance will likely require a pem key. SSH instructions should be given on the last page of setup for the instance. Log into your instance with the ip address given when you set up the instance:</p>
<p>$ ssh -i "my.pem" centos@&lt;ip address&gt;</p>
<p>Setup up your remote compute instance as follows:</p>
<p>$ sudo yum install -y install screen wget; sudo yum -y group install "Development Tools"</p>
<p>$ wget <a href="https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh">https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh</a> ; sh Miniconda3-latest-Linux-x86_64.sh</p>
<p>$ source ∼/.bashrc</p>
<p>$ pip install pip==20.1.1</p>
<p>$ sudo yum install -y yum-utils ; sudo yum-config-manager --add-repo <a href="https://download.docker.com/linux/centos/docker-ce.repo">https://download.docker.com/linux/centos/docker-ce.repo</a> ; sudo yum install -y docker-ce docker-ce-cli containerd.io</p>
<p>$ sudo systemctl start docker</p>
<p>$ sudo groupadd docker ; sudo usermod -aG docker ${USER}</p>
<p>Log out and then back into the machine. Use command “groups” and verify that “docker” is one of the groups. Test that Docker is working properly with:</p>
<p>$ docker run hello-world</p>
<p>$ pip install schema-salad==7.0.20210124093443</p>
<p>$ pip install cwltool==3.0.20200706173533</p>
<p>$ git clone <a href="https://github.com/YeoLab/eclip">https://github.com/YeoLab/eclip</a></p>
<p>$ git clone <a href="https://github.com/meracorley/f-SHAPE-eCLIP">https://github.com/meracorley/f-SHAPE-eCLIP</a></p>
<p>$ cd f-SHAPE-eCLIP</p>
<p>$ git checkout pipeline</p>
<p>Softlink the CWL definition files from eclip into f-SHAPE-eCLIP:</p>
<p>$ cd /home/centos/f-SHAPE-eCLIP/cwl</p>
<p>$ ln -s /home/centos/eclip/cwl/∗.cwl ./</p>
<p>Add the paths to the f-shape-eclip executables to PATH variable. The following is the command if <a href="https://github.com/yeolab/f-shape-eclip">https://github.com/yeolab/f-shape-eclip</a> is cloned to the home directory of a centos machine. If on your system they were instead cloned to /home/user/mypath/, for example, then /home/centos/eclip/bin would become /home/user/mypath/eclip/bin, et cetera.</p>
<p>$ echo export PATH=/home/centos/eclip/bin:/home/centos/f-SHAPE-eCLIP:/home/centos/f-SHAPE-eCLIP/cwl:/home/centos/f-SHAPE-eCLIP/wf:∖$PATH &gt;&gt; ∼/.bashrc</p>
<p>$ source ∼/.bashrc</p>
<p>Build reference genome and repBase reference. For human (GRCh38/Hg38) as follows:</p>
<p>$ mkdir refs; cd refs</p>
<p>$ wget <a href="https://www.encodeproject.org/files/GRCh38_no_alt_analysis_set_GCA_000001405.15/@@download/GRCh38_no_alt_analysis_set_GCA_000001405.15.fasta.gz">https://www.encodeproject.org/files/GRCh38_no_alt_analysis_set_GCA_000001405.15/@@download/GRCh38_no_alt_analysis_set_GCA_000001405.15.fasta.gz</a></p>
<p>$ wget <a href="https://external-collaborator-data.s3-us-west-1.amazonaws.com/reference-data/gencode.v29e.annotation.gtf.db">https://external-collaborator-data.s3-us-west-1.amazonaws.com/reference-data/gencode.v29e.annotation.gtf.db</a></p>
<p>$ gunzip GRCh38_no_alt_analysis_set_GCA_000001405.15.fasta.gz</p>
<p>$ mkdir STAR</p>
<p>$ docker pull brianyee/star:2.7.6a</p>
<p>docker run -v /home/centos/refs:/refs brianyee/star:2.7.6a STAR ∖</p>
<p>--runThreadN 8 ∖</p>
<p>--runMode genomeGenerate ∖</p>
<p>--genomeDir refs/STAR ∖</p>
<p>--genomeFastaFiles refs/GRCh38_no_alt_analysis_set_GCA_000001405.15.fasta</p>
<p>$ wget <a href="https://external-collaborator-data.s3-us-west-1.amazonaws.com/reference-data/homo_sapiens_repbase_fixed_v2.fasta">https://external-collaborator-data.s3-us-west-1.amazonaws.com/reference-data/homo_sapiens_repbase_fixed_v2.fasta</a></p>
<p>$ mkdir STAR_repeat</p>
<p>$ docker run -v /home/centos/refs:/refs brianyee/star:2.7.6a STAR ∖</p>
<p>--genomeSAindexNbases 9 ∖</p>
<p>--runThreadN 8 ∖</p>
<p>--runMode genomeGenerate ∖</p>
<p>--genomeDir refs/STAR_repeat ∖</p>
<p>--genomeFastaFiles refs/homo_sapiens_repbase_fixed_v2.fasta</p>
<p>Download blacklisted regions for your species, GRCh38 (Hg38) as follows:</p>
<p>$ wget <a href="https://www.encodeproject.org/files/ENCFF269URO/@@download/ENCFF269URO.bed.gz">https://www.encodeproject.org/files/ENCFF269URO/@@download/ENCFF269URO.bed.gz</a></p>
<p>$ gunzip ENCFF269URO.bed.gz</p>
<p>$ cd ../</p>
<p>Fetch your data. Use scp or AWS CLI to copy your fastq files onto your machine.</p>
<p>Or use the fSHAPE-eCLIP example dataset stored in GEO: GSE149767, samples SRR11682378, SRR11682379, SRR11682382, SRR11682383, SRR11682384, SRR11682385:</p>
<p>$ cd ∼/</p>
<p>$ wget <a href="http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/current/sratoolkit.current-centos_linux64.tar.gz">http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/current/sratoolkit.current-centos_linux64.tar.gz</a> ; tar xzf sratoolkit.current-centos_linux64.tar.gz</p>
<p>$ echo export PATH=/home/centos/sratoolkit.2.11.0-centos_linux64/bin:∖$PATH &gt;&gt; ∼/.bashrc</p>
<p>$ source ∼/.bashrc</p>
<p>$ mkdir data; cd data/</p>
<p>$ for f in {SRR11682378,SRR11682379,SRR11682382,SRR11682383,SRR11682384,SRR11682385}; do fastq-dump $f ; gzip $f; done</p>
<p>Format a yaml metadata file to tell the pipeline which files represent which sample types. An example yaml file can be found in <a href="https://github.com/meracorley/f-SHAPE-eCLIP/tree/pipeline/examples/wf_fSHAPE.yaml">https://github.com/meracorley/f-SHAPE-eCLIP/tree/pipeline/examples/wf_fSHAPE.yaml</a></p>
<p>The yaml file requires two “treated” fastq files, two “untreated” fastq files, and two Input fastq files. For SHAPE-eCLIP, the “treated” samples will be treatment conditions “A” or “B” and “untreated” samples will be condition “C.” See step 1b of this protocol for letter names behind treatment conditions. For fSHAPE-eCLIP, “treated” samples will be condition “B” and “untreated” will be condition “A” (<a href="#tbl6">Table 6</a>). For the example files from GEO above, SRR11682382.fastq and SRR11682383.fastq are the “treated” files, SRR11682378.fastq and SRR11682379.fastq are the “untreated”, and SRR11682384.fastq and SRR11682385.fastq are the Input sample files.</p>
<table id="tbl6">
<caption>Table 6. How to label "treated" and "untreated" samples in yaml file</caption>
<thead>
<tr>
<th>Protocol</th>
<th>Conditions (see step 1b)</th>
<th>"Treated" condition in yaml</th>
<th>"Untreated" condition in yaml</th>
<th>Example "treated" files</th>
<th>Example "untreated" files</th>
</tr>
</thead>
<tbody>
<tr>
<td>fSHAPE-eCLIP</td>
<td>A,B</td>
<td>B</td>
<td>A</td>
<td>SRR11682382.fastq, SRR11682383.fastq</td>
<td>SRR11682378.fastq, SRR11682379.fastq</td>
</tr>
<tr>
<td>SHAPE-eCLIP</td>
<td>A,C</td>
<td>A</td>
<td>C</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>“in vitro” SHAPE-eCLIP</td>
<td>B,C</td>
<td>B</td>
<td>C</td>
<td>NA</td>
<td>NA</td>
</tr>
</tbody>
</table>
<p>Order very much matters in the yaml file. The yaml file also specifies paths to the STAR index, chromosome sizes of the STAR index, blacklisted regions, the STAR_repeat index, and a GTF file for the given reference genome. Note that “GRCh38_v29e” is a Clipper parameter and is the recognized species name for hg38 in the “species” field of the yaml file (“_v29e” refers to “Gencode V29 Encode” as we recommend matching Clipper references to the assembly.</p>
<p>Make executable:</p>
<p>$ chmod +x wf_fSHAPE.yaml</p>
<p>The pipeline is ready to be run. Start a Screen session to run the pipeline:</p>
<p>$ screen -S pipeline</p>
<p>Execute the pipeline:</p>
<p>$ ./wf_fSHAPE.yaml</p>
<p>The pipeline puts output files within the ./wf_fSHAPE/results/ directory. Binding sites for the protein that was immunoprecipitated are output to BED files, one for each replicate. fSHAPE/SHAPE reactivities are output for genes that have at least on annotated protein binding site in .map file format, one file per gene. Reactivities across genes are also combined in bedgraph and bigWig format (for + and – strands separately) for the purpose of viewing reactivities on a genome browser.</p>
<p>The map file format is a tab-delimited file with four columns and one row per base. Columns are described in <a href="#tbl7">Table 7</a>. Traditionally, reactivity values of "-999.0" are understood to indicate "no data." High fSHAPE reactivities (typically &gt; 2.0) indicate bases that hydrogen bond with protein. High SHAPE reactivities (typically &gt; 0.85) indicate bases that are most likely unpaired. Reactivity measurements are most certain for bases having higher coverage (&gt;200 reads) and good agreement between replicates. For SHAPE reactivities, map files contain everything the user needs to predict structures informed by structure probing data, such as with RNAstructure (<a href="#bib3">DiChiacchio and Mathews, 2016</a>) or RNAfold (<a href="#bib8">Lorenz et al., 2011</a>).</p>
<table id="tbl7">
<caption>Table 7. Example of .map output format</caption>
<thead>
<tr>
<th>Relative position in region</th>
<th>Reactivity value, averaged across replicates</th>
<th>Standard error</th>
<th>Base</th>
</tr>
</thead>
<tbody>
<tr>
<td>30</td>
<td>-999.0</td>
<td>0.02499</td>
<td>T</td>
</tr>
<tr>
<td>31</td>
<td>0.7903</td>
<td>0.00035</td>
<td>T</td>
</tr>
<tr>
<td>32</td>
<td>2.65775</td>
<td>0.09731</td>
<td>G</td>
</tr>
<tr>
<td>33</td>
<td>1.54583</td>
<td>0.2568</td>
<td>C</td>
</tr>
<tr>
<td>34</td>
<td>4.80619</td>
<td>0.96297</td>
<td>T</td>
</tr>
<tr>
<td>35</td>
<td>-0.4285</td>
<td>0.13546</td>
<td>T</td>
</tr>
</tbody>
</table>
<p>fSHAPE analysis:</p>
<p>fSHAPE requires a very minimal set of bioinformatics tools and python libraries. Open a linux terminal instance and install the following software:</p>
<p>Python &gt;= 3.5.4</p>
<p>Python libraries: sci-kit learn &gt;= 0.20.3, joblib &gt;= 0.16.0, hmmlearn 0.2.1</p>
<p>pybedtools &gt;= 0.8.0</p>
<p>Cutadapt 1.14</p>
<p>UMItools &gt;= 0.5.0</p>
<p>STAR &gt;= 2.4.0i</p>
<p>Bedtools &gt;= 2.25.0</p>
<p>Samtools &gt;= 1.9</p>
<p>Bamtools 2.3.0</p>
<p>If this software cannot be successfully installed for some reason, it may work to build a linux-based Docker image that includes all the required software, or to launch an AWS EC2 instance or equivalent cloud computing instance and install the software there.</p>
<p>Download scripts and example data for fSHAPE analysis with:</p>
<p>git clone <a href="https://github.com/meracorley/fSHAPE">https://github.com/meracorley/fSHAPE</a></p>
<p>Locate example fastq files for each sample condition. There should be “in vitro/-protein” condition samples and “in vivo/+protein” condition samples, at least two replicates each for a total of at least four fastq files. Example fastq files to try out the pipeline are included in the fSHAPE directory: K562_vitro_N1.ftl.fastq.gz, K562_vitro_N2.ftl.fastq.gz, K562_vivo_N1.ftl.fastq.gz,K562_vivo_N2.ftl.fastq.gz</p>
<p>Create a directory for storing analyzed data and cd into it. Move fastq files into this directory.</p>
<p>Download reference fasta file, hg38 in the case of the example files:</p>
<p>$ wget <a href="https://hgdownload.cse.ucsc.edu/goldenpath/hg38/bigZips/hg38.fa.gz">https://hgdownload.cse.ucsc.edu/goldenpath/hg38/bigZips/hg38.fa.gz</a></p>
<p>$ gunzip hg38.fa.gz</p>
<div class="note">
<span class="note-title">Note:</span> All the following commands are time and resource intensive, and for shared compute environments these commands should be submitted as jobs with qsub or equivalent. On private setups it is recommended to run the commands flanked by “nohup &lt;command&gt; &amp;” or in a Screen session.</div>
<p>Build a STAR reference for the reference (requires &gt; 32 Gb of memory for human genome):</p>
<p>$ STAR --runThreadN 8 --runMode genomeGenerate --genomeDir STAR_hg38 --genomeFastaFiles hg38.fa</p>
<p>Raw reads for each sample must first be trimmed, mapped, de-duplicated, and coverage quantifiedover each chromosome in the reference genome. This is achieved by running the processReadsPipeline.sh script as follows (use “/bin/bash processReadsPipeline.sh” -h for help message):</p>
<p>$ /bin/bash processReadsPipeline.sh -i sample1.fastq,sample2.fast2,…,sampleN.fastq -r STAR_reference -a adapterSequence -u UMIpattern -p threads(optional) -l logfile.txt</p>
<p>See UMItools documentation for how to specify the UMI specific to your reads.</p>
<p>With the example fastq files the processReadsPipeline command would be as follows:</p>
<p>$ /bin/bash processReadsPipeline.sh -i K562_vitro_N1.ftl.fastq,K562_vitro_N2.ftl.fastq,K562_vivo_N2.ftl.fastq,K562_vivo_N1.ftl.fastq -r STAR_hg38/ -a AGATCGGAAGAGCGGTTCAGCAGGAATGCCGAGACCGATCTCGTATGCCGTCTTCTGCTTG -u NNNCCCCNNNNN -p 8 -l LOGFILE</p>
<p>This calculates coverages across each chromosome for each input fastq file, which are output to ./sampleN/coverage/. For example, for K562_vitro_N1.ftl its coverage files are output to ./K562_vitro_N1.ftl/coverage/</p>
<p>For very large input fastq files, running a separate instance of processReadsPipeline.sh per sample will complete in a more reasonable amount of time, i.e., 24 h.</p>
<p>Generate reactivities for regions or full transcripts with bedReactivities.py:</p>
<p>$ python bedReactivities.py -i input_bed_regions.bed -g reference_genome.fa -v invivo_sample1/coverage/,invivo_sample2/coverage/,...,invivo_sampleNcoverage/ -t invitro_sample1/coverage/,...invitro_sampleN/coverage/</p>
<p>The unless the "-s" option is used, the input bed regions are grouped together by name (transcript identifier in column 4 of the bed region) such that reactivities are grouped, calculated, and normalized by transcript.</p>
<p>With the provided example files, generating reactivities for the mRNA <i>FTL</i> would be done as follows:</p>
<p>$ python bedReactivities.py -i ../fSHAPE/ftl.bed -g hg38.fa -v K562_vivo_N1.ftl/coverage/,K562_vivo_N2.ftl/coverage/ -t K562_vitro_N1.ftl/coverage/,K562_vitro_N2.ftl/coverage/</p>
<p>This outputs .map reactivity files for each transcript with regions in the input bed file. If a transcript has no coverage, however, no files are output for it. The map file format is a tab-delimited file with four columns and one row per base. Columns are described in <a href="#tbl7">Table 7</a>.</p>
<p>Traditionally, reactivity values of "-999.0" are understood to indicate "no data." High fSHAPE reactivities (typically &gt; 2.0) indicate bases that hydrogen bond with protein. fSHAPE reactivities are most certain for bases with higher coverage (&gt;200 reads) and good agreement between replicates.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>(f)SHAPE-eCLIP will yield unreliable results for samples that begin with an inadequate number of cells (&lt;∼40 million). Some cell types, such as motor neurons, are intrinsically sparse in RNA content, and thus will be difficult to generate (f)SHAPE-eCLIP libraries for. Antibody quality is also key, where the antibody for the desired protein being studied should be verified beforehand. Low quality antibodies may exhibit non-specific binding, such that the RNA molecules they immunoprecipitate are not in fact bound by the protein of interest.</p>
<p>The fSHAPE library preparation protocol uses cDNA circularization to generate the initial DNA template for PCR, however CircLigase has known sequence biases (poorly ligates terminal cytosines). This impacts both the overall yield of the ligation reaction and may also bias RT-stops against cytosines. fSHAPE produces the most reliable data for transcripts with high expression, although the sensitivity of fSHAPE can be increased with higher sequencing depths. Our fSHAPE libraries were sequenced to depths of 200 million per sample condition, and transcripts with expression above ∼200 transcripts per million (TPM) typically produced at least partial fSHAPE profiles.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No or unexpected band on (f)SHAPE-eCLIP western blot, step 25k.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>If no bands appear on the western blot, this indicates that the western blot was done incorrectly, or the antibody is ineffective, or the protein being targeted by the antibody is simply not expressed in the given cell type. The Input samples are generally expected to resemble the IP samples in the Western blot, but will occasionally show additional band(s) that are due to non-specific recognition of abundant proteins in the lysate, such as for the SLBP Input (<a href="#fig1">Figure 1</a>A). This is not concerning. Multiple bands in the IP sample could indicate multiple isoforms of the desired protein, but could also indicate non-specific immunoprecipitation of other proteins. We highly recommend IP validating antibodies prior to performing (f)SHAPE-eCLIP and comparing the Western to the vendor’s data on the given antibody. If an antibody fails validation, it is best to find an alternative antibody that can both withstand the increased stringency and varied environments of (f)SHAPE-eCLIP and deliver a clean pull-down. Alternatively, epitope-tagging of proteins with V5 or Halo tags could be a viable option to circumvent antibody viability issues (<a href="#bib5">England et al., 2015</a>). Note that this may introduce unforeseen binding effects due to potential conformational changes to the protein of interest.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Excessive adapter dimer in (f)SHAPE-eCLIP library, step 49e.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Incomplete removal of adapter dimer will appear in the final library trace as a sharp peak ∼164 bp (<a href="#fig4">Figure 4</a>). Adapter dimers will consume sequencing resources at the expense of the desired libraries and should be removed. We generally tolerate adapter dimers that make up &lt; 10% of the integrated area for any given sample. To remove primer dimers repeat the gel extraction, adjusting the lower boundary of the excision window higher to more confidently remove the adapter dimers. Gel extraction causes an estimated ∼50% sample loss, so samples with low library yield (5–10 nM) may require an additional PCR with a few cycles prior to the secondary gel extraction, following times and temperatures of the latter Denaturation and Annealing/Extension steps in the PCR instructions in <a href="#tbl4">Table 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/962-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Example (f)SHAPE-eCLIP library contaminated with primer dimers</div>
</figcaption>
</figure>
<h3 id="sec7.5">Problem 3</h3>
<p>Final (f)SHAPE-eCLIP libraries are low-yield, quantified at &lt; 5 nM, step 49e.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Low library yield could indicate that the protein of interest either does not bind RNA or binds very few specific targets. If the original PCR cycle number was relatively low, 2 or 3 additional PCR cycles can be done to the existing amplified material to increase yield. However, if the original PCR cycle number was relatively high (12+ for input, 18+ for IP), additional amplification will likely only create PCR duplicates and it is best to pool as many femtomoles of library as your pooling strategy allows and to concentrate the pool by bead purification, if necessary to meet sequencing requirements. It may be best to restart the experiment, scaling up the starting material, to increase the probability of sufficiently high yield for sequencing.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Low uniquely mapped yield (&lt;50%) after sequencing and data analysis.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Having low uniquely mapped read yields after mapping and analyzing reads is an indicator that the library had a large number of PCR duplicates. This means that the library underwent overamplification during PCR. Unfortunately, the best way to solve this problem is to re-make the library, scaling up the starting material and taking care not to use too many cycles during library PCR amplification. Sequencing the sample much more deeply, up to 100 million reads per library, will also produce more reads on unique molecules, but at the cost of millions of reads wasted on more PCR duplicates.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>fSHAPE, Excessive short reads in library</p>
<p>Excessive short products in fSHAPE library, refer to <a href="#bib6">Flynn et al., 2016</a>.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>fSHAPE library reads less than 150 bp in length will prove difficult to map and process in the analysis steps. Too large a fraction of short reads in the final libraries may result from a poorly executed gel-cut in the post-PCR purification PAGE gel. Short cDNAs may also come from too-high levels of SHAPE modifications, causing early truncation and reducing the cDNA length that is possible. Less time with the SHAPE reagent could improve this. Short cDNAs may also come from generating short RNA fragments during the fragmentation step after mRNA enrichment. Reducing the time for RNA fragmentation and lengthening the average fragment could help improve this.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Barnett, D.W., Garrison, E.K., Quinlan, A.R., Strömberg, M.P., and Marth, G.T. (2011). BamTools. Bioinformatics <i>27</i>, 1691-1692. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref1">View at publisher</a></p>
<p id="bib2">Corley, M., Flynn, R.A., Lee, B., Blue, S.M., Chang, H.Y., and Yeo, G.W. (2020). Footprinting SHAPE-eCLIP Reveals Transcriptome-wide Hydrogen Bonds at RNA-Protein Interfaces. Mol. Cell <i>80</i>, 903-914. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref2">View at publisher</a></p>
<p id="bib3">DiChiacchio, L. and Mathews, D.H. (2016). Predicting RNA–RNA Interactions Using RNAstructure. In RNA Structure Determination (Humana Press), pp. 51-62. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref3">View at publisher</a></p>
<p id="bib4">Dobin, A., Davis, C.A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., and Gingeras, T.R. (2013). STAR: ultrafast universal RNA-seq aligner. Bioinformatics <i>29</i>, 15-21. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref4">View at publisher</a></p>
<p id="bib5">England, C.G., Luo, H., and Cai, W. (2015). HaloTag technology: a versatile platform for biomedical applications. Bioconj. Chem. <i>26</i>, 975-986. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref5">View at publisher</a></p>
<p id="bib6">Flynn, R.A., Zhang, Q.C., Spitale, R.C., Lee, B., Mumbach, M.R., and Chang, H.Y. (2016). Transcriptome-wide interrogation of RNA secondary structure in living cells with icSHAPE. Nat. Protoc. <i>11</i>, 273-290. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref6">View at publisher</a></p>
<p id="bib7">Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R., and Genome Project Data Processing, S. (2009). The Sequence Alignment/Map format and SAMtools. Bioinformatics <i>25</i>, 2078-2079. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref7">View at publisher</a></p>
<p id="bib8">Lorenz, R., Bernhart, S.H., Zu Siederdissen, C.H., Tafer, H., Flamm, C., Stadler, P.F., and Hofacker, I.L. (2011). ViennaRNA Package 2.0. Algorithms Mol. Biol. <i>6</i>, 1-14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref8">View at publisher</a></p>
<p id="bib9">Martin, M. (2011). Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet. J. <i>17</i>, 10-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref9">View at publisher</a></p>
<p id="bib10">Quinlan, A.R. and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics <i>26</i>, 841-842. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref10">View at publisher</a></p>
<p id="bib11">Smith, T., Heger, A., and Sudbery, I. (2017). UMI-tools: modeling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy. Genome Res. <i>27</i>, 491-499. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref11">View at publisher</a></p>
<p id="bib12">Van Nostrand, E.L., Freese, P., Pratt, G.A., Wang, X., Wei, X., Xiao, R., Blue, S.M., Chen, J.Y., Cody, N.A., Dominguez, D., et al. (2020). A large-scale binding and functional map of human RNA-binding proteins. Nature <i>583</i>, 711-719. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00468-8/sref12">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Gene Yeo (<a href="mailto:geneyeo@ucsd.edu">geneyeo@ucsd.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>All fSHAPE, fSHAPE-eCLIP, and SHAPE-eCLIP raw sequencing reads and final processed data generated during this study are available for download in the Gene Expression Omnibus (GEO), accession number GEO: GSE149767. All custom software used to process this data is available at github.com/meracorley/fshape, github.com/meracorley/f-shape-eclip/tree/pipeline/, and github.com/YeoLab/f-shape-eclip, and upon request.</p>
<h3>Acknowledgments</h3>
<p>This work was partially supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grants R01-HG004659, U41-HG009889, R01-HL137219, R01-HL137223, and R01-HD085902 to G.W.Y. and R01-HG004361 R35-CA209919 to H.Y.C. M.C. was supported by the <a href="https://doi.org/10.13039/100000971">ALS Association</a> Milton Safenowitz Post-Doctoral Fellowship.</p>
<h3>Author contributions</h3>
<p>Conceptualization, M.C., R.A.F., H.Y.C., and G.W.Y.; experimentation, M.C. and S.M.B.; analysis, M.C. and B.A.Y.; writing, M.C., R.A.F., and G.W.Y.; supervision, G.W.Y.</p>
<h3>Declaration of interests</h3>
<p>G.W.Y. is co-founder, member of the Board of Directors, equity holder, and paid consultant for Locana and Eclipse Bioinnovations. G.W.Y. is a Distinguished Visiting Professor at the National University of Singapore. The terms of this arrangement have been reviewed and approved by the University of California, San Diego, in accordance with its conflict of interest policies. M.C. and G.W.Y. are inventors on a patent disclosure to UCSD on technologies described in this manuscript. H.Y.C. is an inventor on a patent owned by Stanford University on in vivo SHAPE. H.Y.C. is a co-founder of Accent Therapeutics and Boundless Bio and an advisor for 10× Genomics, Arsenal Biosciences, and Spring Discovery. H.Y.C. is an Investigator of the Howard Hughes Medical Institute.</p>
</section>
</article>