<article>
<section>
<h1 id="header">Gene-specific quantification of nascent transcription following targeted degradation of endogenous proteins in cultured cells</h1>
<p><time datetime="2021-12-10">Published: December 10, 2021</time></p>
<p>Alissa D. Guarnaccia,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">6</a>,<a href="#cor1">*</a></sup> April M. Weissmiller,<sup><a href="#aff3">3</a>,<a href="#fn1">6</a>,<a href="#cor2">**</a></sup> and William P. Tansey<sup><a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#fn2">7</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Discovery Oncology, Genentech Inc., 1 DNA Way, South San Francisco, CA 94080, USA</p>
<p id="aff2"><sup>2</sup>Department of Microchemistry, Proteomics, Lipidomics, and Next Generation Sequencing, Genentech Inc., 1 DNA Way, South San Francisco, CA 94080, USA</p>
<p id="aff3"><sup>3</sup>Department of Biology, Middle Tennessee State University, Murfreesboro, TN 32132, USA</p>
<p id="aff4"><sup>4</sup>Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, TN 37232, USA</p>
<p id="aff5"><sup>5</sup>Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN 37232, USA</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:guarnaccia.alissa@gene.com">guarnaccia.alissa@gene.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:april.weissmiller@mtsu.edu">april.weissmiller@mtsu.edu</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:william.p.tansey@vanderbilt.edu">william.p.tansey@vanderbilt.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101000">10.1016/j.xpro.2021.101000</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Genome-wide nuclear run-ons are a powerful way to determine the impact of a perturbation such as transcription factor degradation on transcriptional patterns. But often investigators are interested in monitoring transcriptional effects at specific sets of genes, rather than the entire genome. Here we describe an approach that couples genome engineering to tag endogenous proteins for degradation with a streamlined nuclear run-on assay to yield gene-specific information on primary transcriptional changes elicited by factor depletion.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref3">Guarnaccia et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Strategy for degrading a protein and analyzing gene-specific transcriptional effects</li>
<li>CRISPR tagging enables rapid degradation of a target protein in cultured cells</li>
<li>Gene-specific nuclear run-on (NRO) monitors changes in transcripts of interest</li>
<li>Streamlined workflow from cell line engineering through to NRO data analysis</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1222-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The complexity of gene expression processes can make it challenging to identify primary transcriptional targets of any given factor. Localization of a factor on a relevant piece of chromatin can give insight into direct regulation, but does not always provide functional information connecting a factor to transcriptional output of the linked gene. Similarly, the common method of knocking down the expression of a factor and examining transcript changes by RNA-Seq lacks temporal resolution to separate primary from secondary transcriptional changes. And because it measures transcripts—not transcription—this approach is susceptible to processes that affect the steps between transcription and steady state mRNA levels. An elegant solution to this problem was recently developed by Stengel et al. (<a href="#bib11">Stengel et al., 2021</a>) who coupled targeted protein degradation with PRO-Seq, a global nuclear run-on (<a href="#bib5">Mahat et al., 2016</a>), to reveal direct transcriptional changes associated with acute loss of the AML1-ETO oncofusion transcription factor. The information available from PRO-Seq experiments provides a broad genome-wide view of transcriptional changes, but the technique itself is technically demanding, and can be excessive if the goal of the experiment is to monitor transcription at specific genes. For instances where gene-specific transcriptional information is quickly required, as we experienced in our recent study of WDR5 (<a href="#bib3">Guarnaccia et al., 2021</a>), we developed a streamlined workflow that couples targeted factor degradation with a robust gene-specific nuclear run-on approach.</p>
<p>The workflow can be divided into two main sections. The first section involves creating and validating an engineered cell line in which the factor of interest can be rapidly and selectively degraded via the dTAG system (<a href="#bib7">Nabet et al., 2018</a>). The second section involves triggering factor degradation and performing a nuclear run-on (NRO) to label nascent transcripts with biotin-CTP, followed by capture of these RNAs on streptavidin beads, reverse transcription (RT), and quantification of gene-specific cDNAs via quantitative PCR (qPCR). Total RNA is also collected and similarly processed by RT-qPCR. Together, these steps enable detection of early transcriptional changes resulting from targeted depletion of the protein of interest and allows for comparison of transcriptional changes with steady-state changes in mRNA transcript levels.</p>
<p>In this workflow, CRISPR-mediated genome engineering is used to tag the endogenous protein of interest at the carboxy-terminus with a double HA-epitope tag and F36V mutant form of FKBP12 [FKBP(F36V)-2HA]. This process involves electroporation of cells to deliver gene-specific targeting vectors, as well as ribonucleoprotein complexes of Cas9 and a target gene-specific sgRNA. After enrichment and validation, factor degradation is triggered by addition of a bifunctional small molecule degrader—such as dTAG-13 (<a href="#bib7">Nabet et al., 2018</a>) or dTAG-47 (<a href="#bib3">Guarnaccia et al., 2021</a>; <a href="#bib4">Huang et al., 2017</a>; <a href="#bib8">Popay et al., 2021</a>; <a href="#bib14">Weissmiller et al., 2019</a>; <a href="#bib15">Woodley et al., 2021</a>)—that couples the 2HA-FKBP(F36V)-tagged protein to the cereblon (CRBN) ubiquitin protein ligase, triggering its rapid ubiquitin-mediated proteolysis. Following degradation, immediate early transcriptional consequences are measured by NRO-RT-qPCR, using a protocol that is based on the NRO-RT-qPCR assay described by <a href="#bib10">Roberts et al. (2015)</a>, modified to incorporate aspects of our PRO-Seq pipeline (<a href="#bib12">Thomas et al., 2019</a>; <a href="#bib14">Weissmiller et al., 2019</a>; <a href="#bib15">Woodley et al., 2021</a>).</p>
<p>Development of the dTAG system is described in <a href="#bib7">Nabet et al. (2018)</a>. The dTAG-47 degrader is described in <a href="#bib4">Huang et al. (2017)</a>. Targeting vectors for enrichment of multi-allelic tagging events are described in <a href="#bib13">Weintraub et al. (2017)</a>. The nuclear run-on protocol is adapted from <a href="#bib10">Roberts et al. (2015)</a>.</p>
<h3 id="sec1.1">Preparation: Design and order single-guide RNAs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h (excluding lead time on ordering custom sgRNAs)</div>
<p>Here we describe selection of gene-specific single-guide RNAs (sgRNAs) that direct Cas9 to cut within 20 bp of the stop codon of the target gene. We test at least two different sgRNAs for each gene, as in silico design tools do not always reliably predict which sgRNAs will perform best in cells. And we transfect recombinant Cas9-sgRNA ribonucleoprotein complexes into cells, which is efficient and minimizes erroneous genomic modifications.</p>
<ol>
<li>Choose two or three sequences for sgRNAs.
<ol type="a">
<li>Obtain the genomic sequence of the C-terminal coding region of your gene of interest from the NCBI Reference Genome. Select a ∼100 bp sequence centered on the stop codon.</li>
<li>Use a CRISPR/Cas9 guide RNA search program such as CRISPOR (<a href="#bib2">Concordet and Haeussler, 2018</a>) to derive on-target potential sgRNA sequences.
<div class="note">
<span class="note-title">Note:</span> It is important to select sgRNA sequences that score as highly specific for the gene of interest. Most programs rank guide sequences by their predicted specificity to the genome and provide accompanying information to interpret these rankings. The CRISPOR Manual is useful in this regard.</div>
</li>
<li>Select two or three sgRNA sequences that meet specificity/selectivity criteria. Choose sgRNA sequences that cut within 20 bp of the stop codon, as this region will be rendered resistant to cutting by Cas9 after the targeting cassette is integrated. See <a href="#fig1">Figure 1</a>A for an example of a sgRNA targeting WD repeat-containing protein 5 (WDR5) (<a href="#bib3">Guarnaccia et al., 2021</a>). Troubleshooting 1 (No specific sgRNA sequence available within 20 bp of the stop codon).</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Using these sequences, order custom chemically modified CRISPRevolution sgRNAs from Synthego. Also obtain nuclear-targeted Cas9-2xNLS recombinant protein from Synthego or an alternative vendor.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Chemically modified sgRNAs from Synthego contain 2′ O- methyl analogs and 3′ phosphorothioate internucleotide linkages at the three 5′ and 3′ terminal RNA residues. These modifications impart stability and make the sgRNA less sensitive to degradation by exonucleases and innate immune responses.</div>
<div class="note">
<span class="note-title">Note:</span> When deciding on scale, note that 1.5 nmol of sgRNA will be enough for ∼150 electroporation reactions (10 pmol per reaction). Similarly, 300 pmol of Cas9-2xNLS will be enough for ∼100 electroporation reactions (3 pmol per reaction).</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1222-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Design of CRISPR/Cas9 genome editing of the <i>WDR5</i> locus for 3′ insertion of a FKBP(F36V)-2xHA-P2A-fluorophore cassette</div>
<p>(A) Cartoon outlining the Cas9 cut site for sgRNA-1 (target sequence CTCTCGCGGGCAGGAGCAAA). Stop codon is colored red, sgRNA and its annealing sequence are colored blue, and the PAM sequence is green. Cut sites are indicated by scissors.</p>
<p>(B) Schematic of genomic integration events at the 3′ end of the coding region for <i>WDR5</i>. After cutting, the targeting vectors carrying mCherry or mTagBFP2 are incorporated by homologous recombination into the locus resulting in the in-frame expression of the tag sequence and the fluorescent markers. “P2A” is a ribosome skipping peptide sequence.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Preparation: Design and generate targeting vectors</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 1 week</div>
<p>Here we describe the design and construction of plasmids that carry the integration cassettes for the gene of interest. We use asymmetrical homology arms—typically 200 bp 5′ of the stop codon and 800 bp 3′ of the stop codon—as these have been described to improve the efficiency of CRISPR-induced homology directed repair (<a href="#bib9">Richardson et al., 2016</a>). Symmetrical homology arms of 400–800 bp each can also be used.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Long single-stranded DNA templates can be synthesized and used instead of plasmid to deliver the template sequence. Such single-stranded templates can be obtained through vendors including GenScript. This approach, however, is more costly and could take longer than the traditional cloning approach described below.</div>
<ol start="3">
<li>Design homology arms to the gene of interest.
<ol type="a">
<li>Design homology arms for the C-terminal coding region of your gene of interest based on the genomic sequence from the NCBI Reference Genome. These homology arms should flank—but exclude—the stop codon. The 5′ homology region should stretch approximately 180–200 bp upstream of the stop codon and the 3′ region should stretch approximately 700–1000 bp downstream of the stop codon. See <a href="#fig1">Figure 1</a>B for a schematic of homology and targeting vector design used for WDR5.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Be sure to exclude the native stop codon in the 5′ homology arm so that the FKBP(F36V)-2HA tag will be incorporated in-frame as a fusion to your target protein.</div>
<div class="note">
<span class="note-title">Note:</span> Three sgRNA sequences were successfully used for C-terminal tagging of WDR5, and two were successfully used for C-terminal tagging of PDPK1, are listed in the Key Resources Table.</div>
<div class="note">
<span class="note-title">Note:</span> For each gene to be tagged, you will create two vectors that carry the same homology arms but differ only by the fluorescent marker they express: mCherry (Addgene #104370) or mTagBFP2 (Addgene #104371).</div>
<ol start="4">
<li>Design targeting vectors in silico.
<ol type="a">
<li>Use sequences of pAW62.YY1.FKBP.knock-in.mCherry (Addgene #104370) and pAW62.YY1.FKBP.knock-in.mTagBFP2 (Addgene #104371) vectors (<a href="#bib13">Weintraub et al., 2017</a>) to generate vector maps using SnapGene (or a similar program). Double check that the FKBP(F36V)-2HA tag in the vector will be in-frame with the coding region of the 5′ homology arm you have designed.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The vectors #104370 and #104371 from Addgene contain homology arms to YY1 which will need to be replaced with homology arms to your gene of interest via Gibson Assembly.</div>
<ol start="5">
<li>Clone targeting vectors by Gibson Assembly.
<ol type="a">
<li>Design primers for Gibson cloning of your homology arms into Addgene plasmids #104370 and #104371 using the NEBuilder online tool from NEB. The Gibson primers to amplify the homology regions will contain sequences complementary to the vector. For the downstream homology region two different forward primers will need to be acquired, one for mCherry and one for mTagBFP2. A total of five fragments will be amplified with these primers (see <a href="#fig2">Figure 2</a>A).</li>
<li>Order primers as 25 nmole of each oligonucleotide with standard desalting; further purification is not needed. Resuspend primers to 100 μM stock in nuclease-free water.</li>
<li>Isolate genomic DNA from the cell line you plan to modify using PureLink™ Genomic DNA Mini Kit (Invitrogen) following the manufacturer’s protocol.</li>
<li>Amplify homology arms using Gibson primers from genomic DNA using 50 μL PCR reactions and Q5 DNA polymerase (NEB) and 100–300 ng of genomic DNA. The resulting DNA products will be gel purified and used in Gibson assembly, below.
<table id="undtbl1">
<thead>
<tr>
<th colspan="3">PCR reaction setup</th>
</tr>
<tr>
<th>Component</th>
<th>50 μL reaction</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>5× Q5 reaction buffer</td>
<td>10 μL</td>
<td>1×</td>
</tr>
<tr>
<td>10 mM dNTPs</td>
<td>1 μL</td>
<td>200 μM</td>
</tr>
<tr>
<td>10 μM Forward primer</td>
<td>2.5 μL</td>
<td>0.5 μM</td>
</tr>
<tr>
<td>10 μM Reverse primer</td>
<td>2.5 μL</td>
<td>0.5 μM</td>
</tr>
<tr>
<td>Template DNA</td>
<td>Variable</td>
<td>Variable</td>
</tr>
<tr>
<td>Q5 polymerase</td>
<td>0.5 μL</td>
<td>0.02 U/μL</td>
</tr>
<tr>
<td>5× Q5 High GC enhancer (optional)</td>
<td>(10 μL)</td>
<td>(1×)</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>To 50 μL</td>
<td> </td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<thead>
<tr>
<th colspan="4">PCR cycling conditions for homology arms</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>50°C–72°C∗</td>
<td>15 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗ Determine for your primer pairs using NEBTm Calculator</p>
<div class="note">
<span class="note-title">Note:</span> Varying the amount of genomic DNA used in the PCR reaction can influence the efficiency of amplification. Also, for DNA regions that are difficult to amplify, we have had success using the buffer optimization protocol provided with the Advantage GC 2 PCR Kit from Takara Bio (Cat. # 639120).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Homology arms can also be generated using synthetic DNA segments, such as Gene Fragments and gBlocks from IDT, using the reference genome as a guide. However, amplifying homology arms from genomic DNA isolated from the cell line to be modified is advantageous because it retains any potential cell type-specific differences in sequence compared to the reference genome.</div>
</li>
<li>Order primers described in <a href="#tbl1">Table 1</a> to amplify the vector backbone and the FKBP(F36V)-2xHA-P2A-mTagBFP2/mCherry inserts from Addgene plasmids #104370 and #104371 (<a href="#fig1">Figure 1</a>A).</li>
<li>Using the primers in <a href="#tbl1">Table 1</a> and 1 ng plasmid template, amplify the vector fragments using Q5 DNA polymerase (NEB) in 50 μL PCR reactions following the manufacturer’s protocol.
<table id="undtbl3">
<thead>
<tr>
<th colspan="3">PCR reaction setup</th>
</tr>
<tr>
<th>Component</th>
<th>50 μL reaction</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>5× Q5 reaction buffer</td>
<td>10 μL</td>
<td>1×</td>
</tr>
<tr>
<td>10 mM dNTPs</td>
<td>1 μL</td>
<td>200 μM</td>
</tr>
<tr>
<td>10 μM Forward primer</td>
<td>2.5 μL</td>
<td>0.5 μM</td>
</tr>
<tr>
<td>10 μM Reverse primer</td>
<td>2.5 μL</td>
<td>0.5 μM</td>
</tr>
<tr>
<td>Template DNA</td>
<td>1 ng</td>
<td>0.02 ng/μL</td>
</tr>
<tr>
<td>Q5 polymerase</td>
<td>0.5 μL</td>
<td>0.02 U/μL</td>
</tr>
<tr>
<td>5× Q5 High GC enhancer (optional)</td>
<td>(10 μL)</td>
<td>(1×)</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>To 50 μL</td>
<td> </td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<thead>
<tr>
<th colspan="4">PCR cycling conditions for targeting vector backbone fragment (primers used here are ‘backbone for’ with ‘backbone Rev.’ see <a href="#tbl1">Table 1</a>.)</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>66°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>2 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<thead>
<tr>
<th colspan="4">PCR cycling conditions for mTagBFP2 or mCherry tag fragments [primers used here are ‘FKBP(F36V) cassette for’ with ‘FKBP(F36V) cassette Rev BFP’ (BFP fragment), and ‘FKBP(F36V) cassette for’ with ‘FKBP(F36V) cassette Rev mCherry’ (mCherry fragment). See <a href="#tbl1">Table 1</a>.]</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>68°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>Using a QIAGEN gel purification kit, gel purify all five DNA fragments produced in steps 5.d. and 5.f. (two homology arms, vector backbone, FKBP(F36V)-mCherry, and FKBP(F36V)-mTAGBFP2). Follow the manufacturer’s protocol and elute each fragment in 25 μL EB. Quantify products by Nanodrop.</li>
<li>Combine 50 ng of vector backbone fragment with a two to five-fold molar excess each of the appropriate DNA fragments (homology arm fragments and the mTagBFP2/mCherry tag fragment). Bring volume to 10 μL with nuclease-free H<sub>2</sub>O. Add 10 μL of 2× Gibson Assembly Master Mix. Pipet to mix.</li>
<li>Incubate at 50°C for one hour.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Completed Gibson Assembly can be stored at −20°C.</div>
<div class="note">
<span class="note-title">Note:</span> Gibson Assembly for both homology arms can be done in a single Gibson reaction, but cloning can also be done sequentially, one homology arm at a time.</div>
</li>
<li>Transform 2 μL of the reaction into NEB competent cells included in the NEB Gibson Assembly Cloning Kit.</li>
<li>Select at least three colonies to screen for proper cloning and purify plasmid DNA by QIAGEN QIAprep Spin Miniprep Kit following the manufacturer’s protocol.
<div class="optional">
<span class="optional-title">Optional:</span> Screen by restriction enzyme digestion with enzymes diagnostic for the desired targeting vectors.</div>
</li>
<li>Sequence-verify purified plasmids by Sanger sequencing. Suggested sequencing primers are listed in <a href="#tbl2">Table 2</a>.</li>
<li>Purify plasmids by QIAGEN Plasmid MIDI Kit, or other high quality plasmid purification method following the manufacturer’s protocol.
<div class="optional">
<span class="optional-title">Optional:</span> Also purify a GFP plasmid to use as an electroporation control, such as pAcGFP1-C1 from Takara Bio.</div>
</li>
<li>Bring the concentration of each plasmid to 2.5 mg/mL in nuclease-free water and combine mTagBFP2 and mCherry plasmids 1:1. This 1:1 mTagBFP2:mCherry plasmid prep will be used for transfection of target cells. See <a href="#fig2">Figure 2</a> for plasmid maps of the mTagBFP2 and mCherry targeting vectors for WDR5.
<div class="critical">
<span class="critical-title">Critical:</span> It is crucial to use highly purified DNA for plasmids that will be delivered by Neon electroporation as lower quality DNA can negatively impact the transfection outcome.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1222-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Cloning strategy detail for homology targeting vectors</div>
<p>(A) Schematic of Gibson Assembly strategy for making the two targeting vectors with different fluorescent markers. Note that five total fragments are involved.</p>
<p>(B) Vector map of the plasmid designed for C-terminal tagging of WDR5 with FKBP(F36V)-2xHA-P2A-mCherry.</p>
<p>(C) Vector map of the plasmid designed for C-terminal tagging of WDR5 with FKBP(F36V)-2xHA-P2A-mTagBFP2. These vectors are adapted from Addgene plasmids #104370 and #104371.</p>
</figcaption>
</figure>
<table id="tbl1">
<caption>Table 1. Primers to amplify plasmids #104370 and #104371 for targeting vector construction</caption>
<thead>
<tr>
<th>Primer name</th>
<th>Sequence</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>Backbone For</td>
<td>GGATCCCCGGGTACCGAG</td>
<td>Forward primer to amplify pUC19 vector backbone</td>
</tr>
<tr>
<td>Backbone Rev</td>
<td>TCAAGGAAAAACCAGACATCAACC</td>
<td>Reverse primer to amplify pUC19 vector backbone</td>
</tr>
<tr>
<td>FKBP(F36V) cassette For</td>
<td>GGATCCGGAGGAGTGCAG</td>
<td>Forward primer to amplify BFP and mCherry tagging cassettes</td>
</tr>
<tr>
<td>FKBP(F36V) cassette Rev mCherry</td>
<td>TTACTTGTACAGCTCGTCCATGC</td>
<td>Reverse primer to amplify BFP tagging cassette</td>
</tr>
<tr>
<td>FKBP(F36V) cassette Rev BFP</td>
<td>TTAATTAAGCTTGTGCCCCAGTTTG</td>
<td>Reverse primer to amplify mCherry tagging cassette</td>
</tr>
</tbody>
</table>
<table id="tbl2">
<caption>Table 2. Suggested sequencing primers for FKBP(F36V) targeting vectors</caption>
<thead>
<tr>
<th>Primer</th>
<th>Sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>Backbone_For</td>
<td>GAGGAAGGAGACACACTC</td>
</tr>
<tr>
<td>FKBP_Rev</td>
<td>TGTCCCGGGAGGAATCAAC</td>
</tr>
<tr>
<td>pUC19dTAG_Rev</td>
<td>CTTCGCTATTACGCCAGCTG</td>
</tr>
<tr>
<td>mCherry_For</td>
<td>GGCGCCTACAACGTCAAC</td>
</tr>
<tr>
<td>mTagBFP2_For</td>
<td>CTATGTGGACTACAGACTGG</td>
</tr>
</tbody>
</table>
<h3 id="sec1.3">Preparation: Optimize electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>Efficient transfection is key to producing a sufficient number of genetically modified cells to maintain a diverse population. In this step we describe optimization of electroporation conditions, using a GFP-expressing plasmid such as pAcGFP1-C1 from Takara Bio to monitor transfection efficiency. In our experiments we use the Invitrogen Neon Transfection System because it is an electroporation system capable of delivering sgRNA/Cas9 ribonucleoprotein into cells and is readily optimizable to a variety of cell types. Other methods of delivery can be used, with the proper optimization. If you already know effective electroporation parameters for your cell type, you may omit this optimization step.</p>
<div class="note">
<span class="note-title">Note:</span> PBS used for Neon transfections should be without Ca<sup>2+</sup> and Mg<sup>2+</sup> (e.g., Gibco Cat# 10010023).</div>
<ol start="6">
<li>Prepare cells and media for Neon electroporation.
<ol type="a">
<li>Choose one or more Neon Electroporation transfection parameters to test for your cell type, with reference to the Invitrogen Neon Transfection System user guide. Invitrogen provides a table of validated transfection parameters which is a good starting point to verify parameters or to choose parameters based on a similar cell type.</li>
<li>Warm PBS, trypsin, and recovery media (media supplemented with 10% FBS without antibiotics) in a 37°C heat bath.</li>
<li>Set up a 6-well plate with 2 mL of recovery media in each well and pre-incubate the plate in a humidified 37°C/5% CO<sub>2</sub> incubator.</li>
<li>Trypsinize cells (if adherent) and collect cells in media. Count viable cells using trypan blue exclusion.</li>
<li>Aliquot ∼300,000 live cells for each reaction. Pellet cells by centrifugation at 100×<i>g</i> for 5 min at 18°C–22°C . Remove liquid.</li>
<li>Rinse cell pellets in warm PBS. Pellet cells by centrifugation at 100×<i>g</i> for 5 min at 18°C–22°C . Remove liquid.</li>
<li>Resuspend cells in 10 μL Neon Buffer R (Invitrogen).</li>
<li>Add 0.5 μg purified GFP plasmid to cells and pipet up and down gently to mix.</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Perform Neon Transfections.
<ol type="a">
<li>Using 10 μL Neon tips, set up a Neon Tube with 3 mL Buffer E in the Neon apparatus.</li>
<li>Program the desired conditions for your first test.</li>
<li>Gently mix cells and DNA by pipetting up and down with regular pipette.</li>
<li>Using the Neon 10 μL pipette, carefully aspirate 10 μL of cell/DNA mixture, place the pipette in the Neon apparatus, and apply electrical pulse by pressing ‘Start’ on the touchscreen.</li>
<li>When transfection is complete, carefully transfer cells from the Neon tip into the prepared 6-well dish.</li>
<li>Repeat for all desired test parameters.
<div class="optional">
<span class="optional-title">Optional:</span> Cell number and amount of DNA transfected can also be varied to optimize if needed.</div>
<div class="note">
<span class="note-title">Note:</span> For U2OS cells, we tested six different conditions, including conditions recommended by Invitrogen for this cell type. The condition that worked the best in our hands for U2OS cells is: 1230 V, 10 ms pulse width, and 4 pulses.</div>
</li>
<li>Place cells in a humidified 37°C/5% CO2 incubator.</li>
<li>24–48 h post transfection, examine cell viability and transfection efficiency using a fluorescent microscope, aiming for high viability and high GFP expression.
<div class="note">
<span class="note-title">Note:</span> Invitrogen reports 82% transfection efficiency and 72% viability for U2OS cells with the parameters of 1230V, 10 ms pulse width, 4 pulses, and we observed similar results.</div>
<div class="critical">
<span class="critical-title">Critical:</span> For transfection by Neon electroporation, it is important to have all reagents warmed to 37°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Buffer R reduces viability and time in this buffer should be minimized. Do not let cells stay in Buffer R for more than 30 min.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.4">Preparation: Design primers for nascent transcript and mRNA quantification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> variable</div>
<p>For each gene of interest two sets of primers are needed, one for detection of mRNAs and another for detection of nascent transcripts. For detection of mRNAs by standard RT-qPCR, we recommend 18–22 nucleotide primers that anneal within two exons spanning a large intron, thus minimizing signal from contaminating genomic DNA in the mRNA preparations. These primers should produce an amplicon of 80–200 bp. For detection of nascent transcripts by NRO-RT-qPCR, we recommend primers located within an intron—or spanning an intron-exon boundary—ideally centered within 500–1000 bp of the transcription start site. Primer and amplicon length are the same as for RT-qPCR primers. Using intron-directed primers focuses the assay on primary transcripts, but renders the signal susceptible to contaminating genomic DNA, making it necessary to remove DNA by DNase I treatment and to control for the presence of genomic DNA in the qPCR reactions. As well as primer sets specific to the genes of interest, primer sets for normalization to control genes are also needed. A validated list of reference genes for NRO-RT-qPCR has been described (<a href="#bib10">Roberts et al., 2015</a>). We have had good experience with using ACTIN B (<i>ACTB</i>) and GAPDH (<i>GAPDH</i>) as controls. Sequences of primers we have used for NRO-RT-qPCR assays are listed in <a href="#tbl3">Table 3</a>.</p>
<table id="tbl3">
<caption>Table 3. Sequences of validated primer sequences for NRO transcript detection in human cells</caption>
<thead>
<tr>
<th>Target gene</th>
<th>Sequence forward</th>
<th>Sequence reverse</th>
</tr>
</thead>
<tbody>
<tr>
<td><i>GAPDH</i></td>
<td>AATCCCATCACCATCTTCCAG</td>
<td>GAGCCACACCATCCTAGTTG</td>
</tr>
<tr>
<td><i>ACTB</i></td>
<td>AGCTCATTGTAGAAGGTGTGG</td>
<td>GGCATGGGTCAGAAGGATTC</td>
</tr>
<tr>
<td><i>CENPE</i></td>
<td>TGCGTATGTGTGTTTTGTTT</td>
<td>TGATCTTCTGAACCCATCAT</td>
</tr>
<tr>
<td><i>CENPF</i></td>
<td>ACTGGTTTTAGCAGCCAAACT</td>
<td>ATCTTTGGCCAGACACACCC</td>
</tr>
<tr>
<td><i>ASPM</i></td>
<td>ATAATGTATTGTTTTGATTATAGCC</td>
<td>ATCTCTCTTACTCGGCCTTC</td>
</tr>
<tr>
<td><i>KIF18A</i></td>
<td>GGTGAGAAGTCATTGGAGAC</td>
<td>TGATACGTTCATCAAAAGCA</td>
</tr>
<tr>
<td><i>TOP2A</i></td>
<td>GGTTAACTGCCTTTGATGAGCTT</td>
<td>ACATATTTTGCTCCGCCCAG</td>
</tr>
<tr>
<td><i>KIF20B</i></td>
<td>AGGGAAGTAGTGGGCTAGACT</td>
<td>GTCGAGGTACTCCCTCTTGAT</td>
</tr>
<tr>
<td><i>SGO2</i></td>
<td>TTTCTTCGCCTAAAGCTAAA</td>
<td>GCTTCTATAATAATGCAGCTAAAA</td>
</tr>
<tr>
<td><i>RPL35</i></td>
<td>CTGAGGCACACTCTCTCTTG</td>
<td>GTCGTCCAGCTGTTTCAG</td>
</tr>
<tr>
<td><i>RPS24</i></td>
<td>CCTGGATGTACTCTTTTCTCA</td>
<td>ATTCTGTTCTTGCGTTCCT</td>
</tr>
</tbody>
</table>
<ol start="8">
<li>Design primers for qPCR detection of nascent transcripts
<ol type="a">
<li>Design target gene-specific qPCR primer pairs for your genes of interest using NCBI primer BLAST. Input a segment of genomic sequence that includes at least one exon-intron junction. Important primer search parameters include melting temperature (Tm) of 60 ± 3°C (max Tm difference of 3), PCR product size of 80–200 bp, and selecting the option that primers may not span an exon-exon junction. Be sure to select the “Refseq representative genomes” option as a search database.
<div class="note">
<span class="note-title">Note:</span> Searching a smaller genomic region focused on one exon-intron junction may more readily result in the output of exon-intron-spanning primers by the search tool.</div>
</li>
<li>Order primers as 25 nmole of each oligonucleotide with standard desalting; further purification is not needed.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-PDPK1 antibody</td>
<td>Abcam</td>
<td>Cat# ab52893; RRID:AB_881962</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-WDR5</td>
<td>Cell Signaling Technology</td>
<td>Cat# 13105; RRID:AB_2620133</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-GAPDH XP (HRP conjugate)</td>
<td>Cell Signaling Technology</td>
<td>Cat# 8884; RRID:AB_11129865</td>
</tr>
<tr>
<td>Rat monoclonal anti-HA (clone 3F10)</td>
<td>Roche</td>
<td>Cat# 12013819001; RRID:AB_390917</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-HA-Tag</td>
<td>Cell Signaling Technology</td>
<td>Cat# 3724; RRID:AB_1549585</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>dTAG-47</td>
<td>Vanderbilt University Chemical Synthesis Core</td>
<td>(<a href="#bib4">Huang et al., 2017</a>)</td>
</tr>
<tr>
<td>Q5 High-Fidelity DNA Polymerase</td>
<td>New England Biolabs</td>
<td>Cat# M0491L</td>
</tr>
<tr>
<td>Deoxynucleotide (dNTP) Solution Set</td>
<td>New England Biolabs</td>
<td>Cat# N0446S</td>
</tr>
<tr>
<td>spCas9 2NLS Nuclease</td>
<td>Synthego</td>
<td>Add-On Product</td>
</tr>
<tr>
<td>Protein Assay Dye Reagent Concentrate</td>
<td>Bio-Rad Laboratories</td>
<td>Cat# 5000006</td>
</tr>
<tr>
<td>cOmplete, EDTA-free Protease Inhibitor Cocktail</td>
<td>Roche</td>
<td>Cat# 05056489001</td>
</tr>
<tr>
<td>Phosphate buffered saline (no magnesium or calcium)</td>
<td>Gibco</td>
<td>Cat# 10010023</td>
</tr>
<tr>
<td>RNaseZAP</td>
<td>Sigma-Aldrich</td>
<td>Cat# R2020</td>
</tr>
<tr>
<td>
<i>N</i>-Dodecanoyl-<i>N</i>-methylglycine sodium salt (Sarkosyl)</td>
<td>Sigma-Aldrich</td>
<td>Cat# L5125</td>
</tr>
<tr>
<td>TRIzol Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15596026</td>
</tr>
<tr>
<td>TRIzol LS Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10296010</td>
</tr>
<tr>
<td>Dynabeads MyOne Streptavidin T1</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 65601</td>
</tr>
<tr>
<td>ATP solution (100 mM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R0441</td>
</tr>
<tr>
<td>GTP solution (100 mM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R0461</td>
</tr>
<tr>
<td>CTP solution (100 mM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R0451</td>
</tr>
<tr>
<td>UTP solution (100 mM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R0471</td>
</tr>
<tr>
<td>Biotin-11-CTP</td>
<td>PerkinElmer</td>
<td>Cat# NEL54200</td>
</tr>
<tr>
<td>SUPERase-In RNase Inhibitor</td>
<td>Invitrogen</td>
<td>Cat# AM2694</td>
</tr>
<tr>
<td>GlycoBlue Coprecipitant</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM9515</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>PureLink Genomic DNA Mini Kit</td>
<td>Invitrogen</td>
<td>Cat# K182001</td>
</tr>
<tr>
<td>Advantage GC 2 PCR Kit</td>
<td>Takara Bio Inc.</td>
<td>Cat# 639120</td>
</tr>
<tr>
<td>Zombie NIR™ Fixable Viability Kit</td>
<td>BioLegend</td>
<td>Cat# 423105</td>
</tr>
<tr>
<td>Gibson Assembly Cloning Kit (with competent cells)</td>
<td>New England Biolabs</td>
<td>Cat# E5510S</td>
</tr>
<tr>
<td>QIAGEN Plasmid Midi Kit</td>
<td>QIAGEN</td>
<td>Cat# 12143</td>
</tr>
<tr>
<td>QIAprep Spin Miniprep Kit</td>
<td>QIAGEN</td>
<td>Cat# 27104</td>
</tr>
<tr>
<td>QIAquick Gel Extraction Kit</td>
<td>QIAGEN</td>
<td>Cat# 28115</td>
</tr>
<tr>
<td>Neon Transfection System 10 μL Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# MPK1025</td>
</tr>
<tr>
<td>Direct-zol RNA Miniprep Kits</td>
<td>Zymo Research</td>
<td>Cat# R2050</td>
</tr>
<tr>
<td>LunaScript RT SuperMix Kit</td>
<td>New England Biolabs</td>
<td>Cat# E3010S</td>
</tr>
<tr>
<td>KAPA SYBR FAST qPCR Master Mix (2×) Universal</td>
<td>KAPA Biosystems</td>
<td>Cat# KK4601</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>U2OS (<i>Homo sapiens</i>, female)</td>
<td>ATCC</td>
<td>Cat# HTB-96, RRID:CVCL_0042</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Chemically modified WDR5 sgRNA-1 (target: CTCTCGCGGGCAGGAGCAAA)</td>
<td>Synthego</td>
<td>Custom order</td>
</tr>
<tr>
<td>Chemically modified WDR5 sgRNA-2 (target: CCCGACAGTCTCTCGCGGGC)</td>
<td>Synthego</td>
<td>Custom order</td>
</tr>
<tr>
<td>Chemically modified WDR5 sgRNA-3 (target: ACTTCCCGACAGTCTCTCGC)</td>
<td>Synthego</td>
<td>Custom order</td>
</tr>
<tr>
<td>Chemically modified PDPK1 sgRNA-1 (target: CAGGCCACGTCACTGCACAG)</td>
<td>Synthego</td>
<td>Custom order</td>
</tr>
<tr>
<td>Chemically modified PDPK1 sgRNA-2 (target: GACGCCGCTGTGCAGTGACG)</td>
<td>Synthego</td>
<td>Custom order</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pAcGFP1-C1</td>
<td>Takara Bio Inc.</td>
<td>Cat# 632470</td>
</tr>
<tr>
<td>pAW62.YY1.FKBP.knock-in.mCherry</td>
<td>(<a href="#bib13">Weintraub et al., 2017</a>)</td>
<td>Addgene plasmid Cat# 104370</td>
</tr>
<tr>
<td>pAW63.YY1.FKBP.knock-in.BFP</td>
<td>(<a href="#bib13">Weintraub et al., 2017</a>)</td>
<td>Addgene plasmid Cat# 104371</td>
</tr>
<tr>
<td>pAG.WDR5.FKBP.knock-in.mCherry</td>
<td>(<a href="#bib3">Guarnaccia et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pAG.WDR5.FKBP.knock-in.BFP</td>
<td>(<a href="#bib3">Guarnaccia et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>CRISPOR</td>
<td>(<a href="#bib2">Concordet and Haeussler, 2018</a>) <a href="http://crispor.tefor.net">http://crispor.tefor.net</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td>SnapGene Viewer 3.2.1</td>
<td><a href="https://www.snapgene.com">https://www.snapgene.com</a></td>
<td>N/A</td>
</tr>
<tr>
<td>NEBuilder</td>
<td>NEB <a href="http://nebuilder.neb.com">http://nebuilder.neb.com</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td>NEBTm Calculator</td>
<td>NEB <a href="http://tmcalculator.neb.com/#!/main">http://tmcalculator.neb.com/#!/main</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td>Primer BLAST</td>
<td>NCBI <a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Neon Transfection System</td>
<td>Invitrogen</td>
<td>Cat# MPK5000</td>
</tr>
<tr>
<td>5 mL Round Bottom Polystyrene Test Tube, with Cell Strainer Snap Cap (35 μm nylon mesh)</td>
<td>BD Falcon</td>
<td>Cat# 352235</td>
</tr>
<tr>
<td>DynaMag-2 Magnet</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12321D</td>
</tr>
<tr>
<td>SuperPlate PCR Plate, 96-well</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AB2800W</td>
</tr>
<tr>
<td>ABsolute qPCR Plate Seals</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AB1170</td>
</tr>
<tr>
<td>CFX96 Touch Deep Well Real-Time PCR Detection System</td>
<td>Bio-Rad Laboratories</td>
<td>Cat# 1854095</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Buffers for nuclear run-on assay</h3>
<div class="note">
<span class="note-title">Note:</span> All stock solutions for nuclear run-on assay should be made in nuclease-free water and sterile filtered. These buffers should be made fresh. In our experience, nuclease-free water works well, and DEPC-treated water is not required.</div>
<table id="undtbl6">
<caption>Nuclei Extraction Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris, pH 7.4</td>
<td>10 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>1 M sucrose</td>
<td>300 mM</td>
<td>6 mL</td>
</tr>
<tr>
<td>3 M KCl</td>
<td>10 mM</td>
<td>66.5 μL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>5 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>10% NP-40</td>
<td>0.5%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>12.6 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for each experiment and store at 18°C–22°C. Store for a maximum of one week. Immediately prior to use, supplement with cOmplete Protease Inhibitor (Roche) to 1× and RNAse-OUT (Invitrogen) to 40 U/mL.</p>
<table id="undtbl7">
<caption>Nuclei Storage Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris, pH 8.0</td>
<td>50 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>25%</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>5 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.1 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>3.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for each experiment and store at 18°C–22°C. Store for a maximum of one week. Immediately prior to use, supplement with cOmplete Protease Inhibitor (Roche) to 1× and RNAse-OUT (Invitrogen) to 40 U/mL.</p>
<table id="undtbl8">
<caption>2× Transcription Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final (2×) concentration</th>
<th>Amount for 100 μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris, pH 8.0</td>
<td>20 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>1 M MgCl2</td>
<td>5 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>100 mM DTT</td>
<td>5 mM</td>
<td>5 μL</td>
</tr>
<tr>
<td>3M KCl</td>
<td>300 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>RNaseOUT (40 U/μL)</td>
<td>1 U/μL</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Bio-11-CTP (10 mM)</td>
<td>0.5 mM</td>
<td>5 μL</td>
</tr>
<tr>
<td>ATP (50 mM)</td>
<td>1 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>GTP (50 mM)</td>
<td>1 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>CTP (10 mM)</td>
<td>0.5 mM</td>
<td>5 μL</td>
</tr>
<tr>
<td>UTP (50 mM)</td>
<td>1 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>2% Sarkosyl</td>
<td>1%</td>
<td>50 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>14 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>100</b> μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh immediately before use and store for a maximum of two hours. If placed on ice, this solution may precipitate, so store at 16°C–22°C for as little time as is feasible. Each reaction sample will take 60 μL of this 2× mix.</p>
<table id="undtbl9">
<caption>Bead Wash Buffer 1</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10 M NaOH</td>
<td>0.1 M</td>
<td>100 μL</td>
</tr>
<tr>
<td>5M NaCl</td>
<td>50 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>n/a</td>
<td>9.8 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for each experiment and store at 18°C–22°C. Store for a maximum of one week.</p>
<table id="undtbl10">
<caption>Bead Wash Buffer 2</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5M NaCl</td>
<td>100 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>n/a</td>
<td>9.8 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for each experiment and store at 18°C–22°C. Store for a maximum of one week.</p>
<table id="undtbl11">
<caption>Binding Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris pH 7.4</td>
<td>10 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>300 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td>10% Triton X-100</td>
<td>0.1%</td>
<td>100 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>n/a</td>
<td>9.2 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for each experiment and store at 18°C–22°C. Store for a maximum of one week.</p>
<table id="undtbl12">
<caption>Low Salt Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris pH 7.4</td>
<td>5 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>10% Triton X-100</td>
<td>0.1%</td>
<td>100 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>n/a</td>
<td>9.85 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for each experiment and store at 18°C–22°C. Store for a maximum of one week.</p>
<table id="undtbl13">
<caption>High Salt Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris pH 7.4</td>
<td>50 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>2 M</td>
<td>4 mL</td>
</tr>
<tr>
<td>10% Triton X-100</td>
<td>0.5%</td>
<td>500 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>n/a</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for each experiment and store at 18°C–22°C. Store for a maximum of one week.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Delivery of CRISPR genome editing reagents by Neon electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>In this step, the targeting vector for homology-mediated repair and sgRNA-Cas9 ribonucleoprotein (RNP) complexes are delivered into cells by electroporation. When comparing different sgRNAs, each sgRNA should be tested separately to determine the most efficient targeting sequence. To increase the yield of transfected cells, we transfect in triplicate and pool the three rounds of transfected cells.</p>
<ol>
<li>Prepare mTagBFP2 and mCherry targeting plasmid mixture.
<ol type="a">
<li>Thaw plasmid prep (1:1 mixture of mTagBFP2:mCherry at 2.5 mg/mL concentration) purified in step 5 above. 5 μL of this mixture will be 6.25 μg of each plasmid and will provide sufficient DNA for a triplicate of transfections.</li>
</ol>
</li>
<li>Prepare ribonucleoprotein complexes using a 3:1 molar ratio of sgRNA:Cas9-NLS.
<ol type="a">
<li>Resuspend Synthego chemically modified sgRNAs in TE, following the manufacturer’s recommendations. Using the extinction coefficient and molecular weight of the sgRNA provided by Synthego, measure sgRNA concentration by Nanodrop, and dilute sgRNA to 15 μM in TE.</li>
<li>For each sgRNA sequence, assemble a triple scale of RNP complexes in Neon Buffer R in a microcentrifuge tube. Assemble one tube for each triple reaction: Three reactions of CRISPR RNP complexes (triple scale)
<table id="undtbl14">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cas9 (20 <b>μ</b>M)</td>
<td>10 pmol</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>sgRNA (15 <b>μ</b>M)</td>
<td>30 pmol</td>
<td>2 μL</td>
</tr>
<tr>
<td>Neon Buffer R</td>
<td>n/a</td>
<td>10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>12.5</b> μL</td>
</tr>
</tbody>
</table>
Store at 18°C–22°C for one hour or less.</li>
<li>Incubate Cas9 and sgRNA mixture at 18°C–22°C for 10 min. Place on ice.
<div class="note">
<span class="note-title">Note:</span> According to Synthego, assembled CRISPR ribonucleoprotein complexes are stable at 18°C–22°C for 1 hour, at 4°C for up to a week, and at −20°C for up to one month.</div>
</li>
</ol>
</li>
<li>Set up Neon Transfection System apparatus
<ol type="a">
<li>Using 10 μL Neon tips, set up a Neon Tube with 3 mL Buffer E in the Neon apparatus.</li>
<li>Program the desired conditions for electroporation transfection, as determined by optimization with GFP transfection, above. The setting we use for U2OS cells is: 1230 V, 10 ms pulse width, and 4 pulses.</li>
</ol>
</li>
<li>Prepare cells for electroporation.
<ol type="a">
<li>Warm PBS, trypsin, and recovery media in a 37°C heat bath.</li>
<li>Based on the number of transfections you plan to perform, calculate how many cells you need: In our case we use 300,000 U2OS cells per reaction, with 3 reactions per sgRNA (i.e., 900,000 cells for each sgRNA being used). Factor in one or two extra reactions to account for pipetting errors etc.</li>
<li>Set up a 6-well plate with the appropriate antibiotic-free complete media and pre-incubate the plate in a humidified 37°C/5% CO<sub>2</sub> incubator.</li>
<li>Trypsinize cells (if adherent) and collect cells in media. Count cells and aliquot the number of cells determined above in step b. Pellet cells by centrifugation at 100×<i>g</i> for 5 min at 18°C–22°C . Remove liquid.</li>
<li>Rinse the cell pellet in warm PBS. Pellet cells by centrifugation at 100×<i>g</i> for 5 min at 18°C–22°C . Remove liquid.</li>
<li>Resuspend the cell pellet in Neon Buffer R (Invitrogen), using 5 μL Neon Buffer R per 300,000 cells.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Buffer R reduces viability and time in this buffer should be minimized. Proceed directly to the next step. Do not let cells stay in Buffer R for more than 30 min.</div>
<ol start="5">
<li>Assemble triplicate reactions for Neon Transfection.
<ol type="a">
<li>Remove the tubes containing 12.5 μL assembled triplicate of ribonucleoprotein complexes from ice (step 2.c.) and add 5 μL of the mTagBFP2 and mCherry plasmid mixture (from step 1.a.) to each of the tubes.</li>
<li>Add 15μL cell suspension (∼900,000 cells) to the mixture of RNP complexes and DNA (above). Pipette to mix. The total volume should now be 32.5 μL.</li>
<li>Using the Neon 10 μL pipette, aspirate 10 μL of cell/RNP/DNA mixture into a 10μL Neon tip (avoid bubbles). Place pipette in Neon apparatus and apply electrical pulse by pressing ‘Start’ on the touchscreen.</li>
<li>When electroporation has completed, immediately transfer cells from the Neon tip into the prepared, pre-warmed 6-well dish of antibiotic-free media.</li>
<li>Repeat steps 5.c. and 5.d. twice more for a total of three transfections with the same tip and the same reaction mixture.
<div class="note">
<span class="note-title">Note:</span> To conserve disposable items, a single Neon tip is used for each of the three triplicate reactions described above. After three reactions the tip is changed.</div>
</li>
<li>Change to a new tip.</li>
<li>Add 15μL cell suspension to the next mixture of RNP complexes and DNA. Pipette to mix.</li>
<li>Perform steps 5.c.–5.e. for this second reaction.</li>
<li>Repeat this process—mixing cells with CRISPR reagents, electroporating, and plating cells—for all sgRNA mixtures being tested.</li>
<li>After all transfections are complete, incubate cells in a humidified 37°C/5% CO<sub>2</sub> incubator. It is important that cells are not disturbed and remain in antibiotic-free media for at least two days before analyzing, expanding, or adding antibiotics to the media.</li>
<li>Monitor cells over the next four to seven days as they recover. Split and expand as needed.</li>
<li>Freeze down a sample of the population of transfected cells as a backup.
<div class="critical">
<span class="critical-title">Critical:</span> Avoid air bubbles in the Neon tip, which will lower transfection efficiency.</div>
<div class="note">
<span class="note-title">Note:</span> At this stage, it is useful to freeze down a population of transfected cells as a backup (step l.), lest anything happens to the main culture of transfected cells.</div>
<div class="note">
<span class="note-title">Note:</span> In our experience, different cell types can display differential sensitivity to the amount of DNA transfected. We use ∼4 μg per 10 μL electroporation with ∼300,000 U2OS cells, but for other cell types the total amount of DNA used may need to be adjusted to maximize recovery.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Analyze transfected cells for incorporation of fluorescent markers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>The next step is to confirm that cassette integration at the target locus has occurred. As the mTagBFP2 and mCherry cassettes can only be expressed after successful genomic tagging, we use mTagBFP2 and mCherry expression—detected by flow cytometry—as a first indicator of proper integration. Double-positive cells indicate that two alleles are tagged, which for diploid cells corresponds to complete tagging. Untransfected cells are required at this step as a gating control, and a far-red viability dye is included to gate for live cells.</p>
<ol start="6">
<li>Collect cells and stain with viability dye.
<ol type="a">
<li>Trypsinize (if adherent) transfected cells and collect in media. Be sure to replate some of the transfected cells to keep them growing. Count cells and collect ∼1 × 10<sup>6</sup> cells from each sample.</li>
<li>Trypsinize (if adherent) untransfected cells and collect in media. Count cells and collect two samples of untransfected cells, one to stain with viability dye and one to remain unstained, ∼1 × 10<sup>6</sup> cells per sample.</li>
<li>Pellet cells by centrifugation at 100×<i>g</i> for 5 min at 18°C–22°C . Remove liquid.</li>
<li>Rinse cell pellets in warm PBS. Pellet cells by centrifugation at 100×<i>g</i> for 5 min at 18°C–22°C . Remove liquid.</li>
<li>Dilute Zombie NIR™ dye (BioLegend) at 1:2000 on PBS.</li>
<li>Resuspend cells in diluted 100 μL Zombie NIR™ solution (100 μL is sufficient for 1–10 × 10<sup>6</sup> cells). For the unstained untransfected sample, simply resuspend it in PBS.</li>
<li>Incubate the cells at 18°C–22°C , in the dark, for 15–30 min.</li>
<li>Wash one time with PBS containing 0.5% bovine serum albumin (BSA).</li>
<li>Resuspend cells in PBS with 0.5% BSA to a concentration of 2 × 10<sup>6</sup> cells/mL. Pipet through strainer caps into 5 mL polystyrene tubes with 35 μm nylon mesh cell strainer caps. Place tubes on ice.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use a far-red viability dye (Zombie NIR from BioLegend) since the mTagBFP2 signal overlaps with that of common DNA-based stains such as DAPI and Hoechst. If performing this process with different fluorescent markers in the tagging vectors, be sure to select a compatible viability dye.</div>
<ol start="7">
<li>Analyze cells by flow cytometry using a 4-laser Fortessa (BD Biosciences), or equivalent instrument equipped with 405 nM and 561 nM lasers to detect mTagBFP2 and mCherry markers respectively and a 633 nm laser to detect the Zombie NIR viability stain.
<ol type="a">
<li>Use the untransfected, unstained, cell sample to gate for single cells using forward scatter (FSC) and side scatter (SSC).</li>
<li>Switch to the untransfected, viability dye-stained, sample and gate for live cells by excluding cells that are positive for the viability dye, based on the unstained sample.</li>
<li>Switch to the transfected cell sample and, with reference to the untransfected samples, gate for cells that are positive for the fluorescent markers. If locus-specific integration of the targeting vectors occurred, you should see mTagBFP2-positive, mCherry-positive cells, and double positive cells. See <a href="#fig3">Figure 3</a> for examples of flow cytometry data of untransfected and transfected cells. Troubleshooting 2 (No fluorescently positive cells are apparent by flow cytometry analysis) and Troubleshooting 3 (mCherry positive cells are present, but not mTagBFP2 positive cells).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1222-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Flow cytometry analysis of genomic integration of fluorescent markers</div>
<p>(A and B) Untransfected and (B) CRISPR-transfected U2OS cells were counterstained with Zombie-NIR viability dye (BioLegend) and then analyzed by flow cytometry to detect mTagBFP2 and mCherry expression. Forward scatter and side scatter are used to gate for single cells. Next, dead cells are excluded by gating against the Zombie NIR viability dye signal. Viable single cells are then analyzed for mCherry and mTagBFP2 signals using appropriate laser lines. In (B), distinct populations of mTagBFP2-positive, mCherry-positive, and double-positive cells are observed. The mTagBFP2-positive cells are gated in blue. The mCherry-positive cells are gated in red. The population of double positive cells are gated in black and purple. The purple box indicates the more stringent gating strategy which is used to select a population of double-tagged cells more confidently. Troubleshooting 4 (Sorted population of cells is not pure).</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As shown in <a href="#fig3">Figure 3</a>, double positive cells can be fairly rare at this stage (0.1%–2%). In addition to doubly positive cells, there should be clear populations of singly-positive cells for mTagBFP2 and for mCherry.</div>
<h3 id="sec3.3">Sort transfected cells to isolate doubly-positive cells</h3>
<p>After confirming that the pool of transfected cells contains cells that express mCherry and mTagBFP2, the next step is to enrich for fluorescent-positive cells by fluorescence activated cells sorting (FACS). We prefer to work with populations of modified cells, rather than clones, as this process is more rapid and avoids potential artifacts associated with cell cloning. But as a backup, we also sort for single cell cloning. Both processes are described here.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 7–10 days</div>
<ol start="8">
<li>For population sorting, place 2 mL of appropriate complete media (including antibiotics) in each well of a 6-well dish. For single cell sorting, place 150 μL media into each well of a 96-well dish.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Some cell types require conditioned media for survival after single cell sorting.</div>
<ol start="9">
<li>Collect transfected and untransfected populations and counterstain with a viability dye for flow cytometry as described in step 6.a.–6.i. For sorting cells, at step 6.i., reconstitute the cells to 10 × 10<sup>6</sup> cells/mL.</li>
<li>Sort double positive cells by FACS using a 4-laser FACSAria lll, or equivalent instrument, housed inside a biosafety cabinet. The instrument should be equipped with lasers to detect the fluorescent markers you are using, for example 405 nM and 561 nM lasers to detect mTagBFP2 and mCherry markers respectively, and 566 nm for the Zombie NIR viability dye.
<ol type="a">
<li>Use the untransfected, unstained, cell sample to gate for single cells using forward scatter (FSC) and side scatter (SSC).</li>
<li>Switch to the untransfected, viability dye-stained, sample and gate for live cells by excluding cells that are positive for the viability dye, based on the unstained sample.</li>
<li>Switch to the transfected cell samples and carefully gate for cells that are doubly positive for mTagBFP2 and mCherry. Be sure to gate in a way that will largely avoid collecting background, untagged cells (see <a href="#fig3">Figure 3</a>B).</li>
<li>Sort cells that meet the set gating criteria into the prepared plates with media and antibiotics. Collect populations in the 6-well dishes and single cells in the 96-well plates. In collecting populations, we typically collect 5,000–10,000 cells per transfected sample at this stage.</li>
<li>After sorting, immediately place cells in a humidified 37°C/5% CO<sub>2</sub> incubator.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The gating for FACS should be stringent enough to minimize non-positive cells being erroneously collected. We gate towards the top fraction of cells that give signal for the fluorescent markers (see <a href="#fig3">Figure 3</a>B). In our experience, more stringent gating yields a purer population of efficiently tagged cells.</div>
<div class="note">
<span class="note-title">Note:</span> The number of cells collected in this process is typically limited by instrument time access. With a 90-min window of sorting, we typically recover one 96-well plate sorted for single cell clones and a 6-well dish with a total of 5,000–10,000 sorted doubly positive cells into a single well of a 6-well dish.</div>
<ol start="11">
<li>For single cell clones, incubate 96-well plates in a humidified 37°C/5% CO<sub>2</sub> incubator, refreshing media as necessary, until clonal populations have formed visible colonies and are ready to move into larger dishes.</li>
<li>For the sorted population, one day after sorting, carefully change the media on cells to refresh antibiotics and to remove residual sheath fluid from sorting. Incubate cells in a humidified 37°C/5% CO<sub>2</sub> incubator, refreshing media as necessary, until cells reach confluency.</li>
<li>Expand the population of sorted cells for validation and analysis by Western blotting.</li>
<li>Freeze down a subset of the population of sorted cells and any clonal populations as desired.</li>
</ol>
<h3 id="sec3.4">Validate tagging by Western blotting</h3>
<p>Next, validate tagging of the target protein by Western blotting. At this step, also verify degradation of the protein of interest by analyzing a time course of cells treated with a degrader molecule. We have observed efficient target protein degradation with 500 nM dTAG-47 without need for optimization in multiple cell types. The concentration of degrader may, however, need to be optimized for other cell types or degrader molecules.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 days</div>
<ol start="15">
<li>Prepare lysates of enriched cells using your preferred method and analyze by Western blotting with target-specific antibody and a loading control antibody. Troubleshooting 4 (Sorted population of cells is not pure).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Efficient tagging and sorting will cause an increase in the apparent molecular weight of the target protein of ∼15 kDa; little if any untagged protein should remain. See <a href="#fig4">Figure 4</a>A for an example of Western blot analysis of targeted populations of cells (tagging PDPK1) before and after sorting.</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1222-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Western blot analyses of C-terminal tagging with FKBP(F36V)-2xHA.</div>
<p>(A) <i>PDPK1</i> was targeted for C-terminal FKBP(F36V)-2xHA tagging in U2OS cells. Two sgRNA sequences were analyzed (sg1 and sg2). Both sgRNAs result in a shifted band for PDPK1, even prior to flow cytometry sorting, indicative of efficient tagging. The sgRNA1 resulted in a more efficiently tagged population and this pool was taken forward for downstream analysis.</p>
<p>(B) <i>WDR5</i> was targeted for C-terminal tagging with FKBP(F36V)-2xHA in U2OS cells. A sorted population of cells was treated with 500 nM dTAG-47 degrader molecule or DMSO control for the indicated times and analyzed by Western blotting with comparison to an untargeted (untagged) population of cells.</p>
</figcaption>
</figure>
<ol start="16">
<li>Validate small molecule-mediated degradation of the tagged protein using dTAG-47 (<a href="#bib3">Guarnaccia et al., 2021</a>; <a href="#bib4">Huang et al., 2017</a>; <a href="#bib8">Popay et al., 2021</a>; <a href="#bib14">Weissmiller et al., 2019</a>; <a href="#bib15">Woodley et al., 2021</a>) or dTAG-13 (<a href="#bib7">Nabet et al., 2018</a>, and available from Tocris Bioscience).
<ol type="a">
<li>Plate 1–2 million cells in 60 mm dishes, one plate for each treatment or time point (for example DMSO, 2 h dTAG, 6 h dTAG, and 24 h dTAG). In our experience, depletion of the target protein is noticeable within 1 h, and completes around 6 h of treatment with dTAG-47 molecule.</li>
<li>The next day, change media to media containing DMSO or 500 nM dTAG degrader molecule.</li>
<li>At the appropriate time points, collect cells, wash in PBS, and make lysates according to the preferred method for your cell line.</li>
<li>Measure protein concentrations with Bio-Rad Protein Assay Dye Reagent and resuspend lysates with SDS sample buffer supplemented with reducing agent.</li>
<li>Run protein lysates on SDS-PAGE and transfer to a membrane for Western blotting with target-specific antibody, HA-specific antibody, and a loading control antibody.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Treatment with the degrader molecule, but not DMSO, should induce rapid depletion of the target protein. Most proteins we have targeted using this method show a considerable reduction at two hours and are &gt;90% depleted by six hours. See <a href="#fig4">Figure 4</a>B for an example of a time course Western blot analysis of a tagged protein (WDR5).</div>
<div class="note">
<span class="note-title">Note:</span> The dTAG-47 molecule is not currently commercially available and is only available by custom chemical synthesis. We had dTAG-47 synthesized by the Vanderbilt Chemical Synthesis Core. dTAG-13 is commercially available, but we have no experience with this degrader. Recently, Nabet et al., (<a href="#bib6">Nabet et al., 2020</a>) described a degrader, dTAG<sup>V</sup>-1, which is commercially available. dTAG<sup>V</sup>-1 connects FKBP(F36V) to the VHL ubiquitin protein ligase and may be advantageous under specific circumstances; dTAG<sup>V</sup>-1 may also be used in combination with CRBN-linked degraders to accelerate target protein destruction.</div>
<h3 id="sec3.5">Nuclear run-on RT-qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Once the required cells are obtained and validated, you can now assess the early transcriptional consequences of target protein depletion. This is done by treating cells with degrader molecules to induce protein depletion, performing a nuclear run-on (NRO), and purifying and quantifying the nascent transcripts (see <a href="#fig5">Figure 5</a>A). The appropriate timing of dTAG treatment in this experiment is determined from the Western blot validation in step 16 above, but note that earlier time points increase confidence that direct transcriptional changes are being monitored. Total RNA is also harvested in parallel to allow comparison of nascent transcripts. Total protein also can be harvested in parallel to confirm target degradation.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1222-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Nuclear run-on assay following by RT-qPCR</div>
<p>(A) Steps for analyzing nascent transcription and total transcript levels are outlined. CRISPR-engineered cells are cultured and the degradation of the target protein is induced by dTAG treatment. Eighty percent of the cells are used for the NRO; the remaining 20% are used for total RNA isolation and analysis. After purification, NRO-RNA and total RNA is reverse transcribed (with random primers) into cDNA that can then be detected by qPCR. Data for NRO RNA and total RNA are then compared, allowing an examination of transcription that may precede, or at least coincide, with changes in total transcript levels that are due to loss of the protein that was degraded.</p>
<p>(B) Example data from NRO RNA isolated from cells treated with DMSO control or 500 nM dTAG-47 for 6 h. Signal is normalized to nascent <i>ACTB</i> transcripts. Data are represented as mean ± SEM; n = 3 independent biological replicates. ∗∗∗p &lt; 0.001, ∗∗p &lt; 0.01, ∗p &lt; 0.05 by unpaired two-tailed t test. These data were originally published in <a href="#bib3">Guarnaccia et al. (2021)</a> and are reprinted with permission.</p>
<p>(C) Example data from total RNA isolated from cells treated with DMSO control or 500 nM dTAG-47 for 6 h. Signal is normalized to <i>ACTB</i> transcript levels. Data are represented as mean ± SEM; n = 3 independent biological replicates.</p>
<p>∗∗p &lt; 0.01, ∗p &lt; 0.05 by unpaired two-tailed t test.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> For this portion of the protocol, you will need a water bath set to 30°C.</div>
<div class="note">
<span class="note-title">Note:</span> Buffers for this assay are described in the <a href="#materials-and-equipment">materials and equipment</a> section, above.</div>
<div class="note">
<span class="note-title">Note:</span> Prior to making any solutions, or performing the experiment itself, it is advisable that you take all steps to minimize RNase contamination (e.g., clean bench, pipettes, etc. with RNaseZAP or equivalent).</div>
<p>Day 1:</p>
<ol start="17">
<li>One day prior to performing the NRO, plate FKBP(F36V)-tagged cells. Plate one 10 cm plate per sample with ∼5 × 10<sup>6</sup> cells. You will need a DMSO-treated control sample, and at least one sample treated with dTAG degrader molecule for the appropriate time (as determined by the time-course Western blot in step 16). We suggest testing two different time points for the first experiment to optimize for a timepoint.</li>
<li>Place cells in a humidified 37°C/5% CO<sub>2</sub> incubator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our experience, the NRO-RT-qPCR assay can be performed with as few as 2 million cells per sample, but this number may need to be increased depending on the cells and rates of target gene transcription. Four million cells have worked well in our hands in multiple cell lines without need for further optimization.</div>
<p>Day 2:</p>
<ol start="19">
<li>Treat cells with dTAG degrader molecules to induce target protein depletion.
<ol type="a">
<li>Remove media from cells and replace with media containing DMSO or 500 nM dTAG degrader molecules.</li>
<li>Incubate cells in a humidified 37°C/5% CO<sub>2</sub> incubator for the selected time point(s).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The final DMSO concentration we use is 0.1% as some cells can be sensitive to higher DMSO levels.</div>
<ol start="20">
<li>Extract nuclei from cells.
<ol type="a">
<li>Trypsinize cells (if adherent) and collect cells in their normal media. Collect cells by centrifugation at 400×<i>g</i> for 4 min at 4°C.</li>
<li>Resuspend cells in 1 mL ice-cold PBS.</li>
<li>Count viable cells using trypan blue exclusion. Aliquot 4 × 10<sup>6</sup> viable cells per treatment sample into labeled 1.5 mL tubes and place on ice (<a href="#fig5">Figure 5</a>A).</li>
<li>From remaining cells, collect approximately 1 × 10<sup>6</sup> cells from each sample for total RNA extraction.</li>
<li>Pellet cells by centrifugation at 400×<i>g</i> for 4 min at 4°C and remove PBS.</li>
<li>For mRNA analysis, if you are using the Zymo Direct-zol RNA Miniprep Kit, lyse cell pellets in 600 μL of TRIzol for at least 20 min on a rotator at 18°C–22°C . For protein analysis, prepare cell pellets for protein lysates according to your standard protocol.</li>
<li>For the NRO samples, pellet all cells by centrifugation at 400×<i>g</i> for 4 min at 4°C and remove PBS.</li>
<li>Resuspend the cell pellet in 1 mL of ice-cold Nuclei Extraction Buffer supplemented with RNAse inhibitor and protease inhibitors as described in the <a href="#materials-and-equipment">materials and equipment</a> section.</li>
<li>Incubate samples on ice for 5 min.</li>
<li>Pellet nuclei at 300×<i>g</i> for 4 min at 4°C.</li>
<li>Remove the supernatant carefully by pipetting.</li>
<li>Gently wash the nuclei with 1 mL of Nuclei Extraction Buffer.</li>
<li>Immediately pellet again at 300×<i>g</i> for 4 min at 4°C.</li>
<li>Resuspend the nuclei in 40 μL of Nuclei Storage Buffer. Mix well by pipetting and place on ice.</li>
<li>Proceed to the Nuclear Run-On reaction.</li>
</ol>
</li>
<li>Perform the Nuclear Run-On Reaction.
<ol type="a">
<li>Prepare a 2× Transcription Buffer according to the table in the <a href="#materials-and-equipment">materials and equipment</a> section. Each reaction requires 60 μL of the 2× mix and you should have this buffer made fresh immediately prior to starting nuclear extraction.</li>
<li>Add 60 μL of 2× Transcription Buffer to each nuclei sample. Mix gently by pipetting (200 μL tip) until the nuclei are homogeneously dispersed.</li>
<li>Incubate the samples for 30 min in a 30°C water bath to allow transcription to occur and nascent transcripts to be labeled by the incorporation of biotin-CTP.</li>
</ol>
</li>
<li>Stop nuclear run-on reaction and purify RNA.
<ol type="a">
<li>Remove samples from the water bath and add 300 μL of TRIzol LS to each sample. Pipette up and down to mix until homogeneous. Typically, this mixture will be viscous and it can be helpful to cut the end of the pipette tip off with a sterile razor blade in order to increase its diameter to facilitate mixing.</li>
<li>Vortex each sample for 10 s and then rotate samples at 18°C–22°C for 5 min to complete lysis.</li>
<li>Perform an RNA cleanup using a Zymo Direct-zol RNA Miniprep Kit, ensuring that you incorporate the on-column DNase-treatment step. At this time, you should also prepare the samples you collected for mRNA analysis (step 20.f.).
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> We routinely use the Zymo Direct-zol RNA Miniprep Kit and find it produces reliable and consistent results, however, other RNA preparation protocols that include a DNase I-treatment step will likely work as an alternative.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Because NRO-RT-qPCR primers are directed against introns or intron-exon boundaries, they have the potential to detect contaminating genomic DNA. As a result, efficient DNase I treatment of samples is crucial to minimize carry-over of genomic DNA.</div>
</li>
<li>Elute each sample (NRO and total RNA) in 25 μL of nuclease-free water into a fresh microcentrifuge tube.</li>
<li>Quantify samples by Nanodrop, and record yield.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point you can store the NRO RNA samples at −80°C for up to a week, but we recommend proceeding forward with the biotin enrichment as soon as possible, ideally within 24 hours. Total RNA can be stored at −80°C for at least a year.</div>
Day 3:</li>
</ol>
</li>
<li>Purify biotinylated NRO-RNA.
<ol type="a">
<li>For each reaction, place 30 μL of Dynabeads MyOne Streptavidin T1 bead slurry in a single 1.5 mL microfuge tube (e.g., if six NRO reactions are being processed, dispense 180 μL of bead slurry at this step).</li>
<li>Add 1 mL of Bead Wash Buffer 1 and rotate for 2 min at 18°C–22°C to wash beads.</li>
<li>Place the sample on a magnetic rack for 1 min. Aspirate wash liquid.</li>
<li>Repeat washing steps using 1 mL of Bead Wash Buffer 2 for a total of two times. Aspirate wash liquid each wash.</li>
<li>Resuspend beads with Binding Buffer, using 50 μL binding buffer per sample (e.g., if six NRO reactions are being processed, resuspend beads in 300 μL binding buffer).</li>
<li>Thaw NRO samples (if frozen) from step 22.d. and add 50 μL of resuspended bead slurry to each sample. Note that the entire NRO reaction (25 μL eluted RNA) is processed at this stage.</li>
<li>Incubate RNA/beads on a rotator for 20 min at 18°C–22°C .</li>
<li>Place samples on a magnetic rack for 1 min. Remove liquid.
<div class="note">
<span class="note-title">Note:</span> To avoid sample loss, make sure beads are not present in the tube tops during bead recovery. If so, spin tubes for 5–10 s in a mini tabletop centrifuge to collect beads and liquid at the bottom of the tubes.</div>
</li>
<li>Wash beads once with 500 μL High Salt Buffer by resuspending beads in the buffer and inverting several times.</li>
<li>Place samples on a magnetic rack for 1 min. Remove liquid by aspiration.</li>
<li>Wash beads once with 500 μL Binding Buffer by resuspending beads in the buffer and inverting several times.</li>
<li>Place samples on a magnetic rack for 1 min. Remove liquid.</li>
<li>Wash bead samples once with 500 μL Low Salt Buffer by resuspending beads in the buffer and inverting several times.</li>
<li>Place samples on a magnetic rack for 1 min. Remove liquid.</li>
<li>Add 300 μL TRIzol directly to beads to extract biotinylated RNAs from the magnetic beads. Pipette up and down vigorously to resuspend the beads in TRIzol and then vortex on high speed for 10 s.</li>
<li>Rotate samples at 18°C–22°C for 5 min.</li>
<li>Add 60 μL chloroform to each sample and vortex for 1 min at 18°C–22°C on high speed.</li>
<li>Centrifuge samples at 16,000×<i>g</i> for 5 min at 18°C–22°C .</li>
<li>Carefully collect the upper aqueous layer (containing the purified biotinylated RNA) and transfer to a fresh tube. Store this fresh tube on ice.</li>
<li>Remove remaining organic liquid from the extraction tube using a pipette until only the magnetic beads remain.</li>
<li>Perform a second TRIzol extraction on the beads by repeating steps 23.o.–r.</li>
<li>Collect aqueous layer as in 23.s. and combine with the first extraction for each respective sample.</li>
</ol>
</li>
<li>Harvest purified biotinylated RNA.
<ol type="a">
<li>Add 1 μL Invitrogen Ambion GlycoBlue Coprecipitant to each sample.</li>
<li>Add a 3× volume of ice-cold 100% ethanol to each extracted sample. Vortex to mix well.</li>
<li>Incubate samples at −20°C for 10 min to precipitate the biotinylated-RNA.</li>
<li>Centrifuge samples at 16,000×<i>g</i> at 4°C for 20 min. Troubleshooting 5 (No RNA pellet or altered appearance of RNA pellet following extraction).</li>
<li>Carefully remove the liquid from around the small blue RNA/GlycoBlue pellet using a pipette. These pellets do not usually adhere to the side of the tube, making aspiration of the liquid difficult. If this happens, gently move the RNA containing pellet to the side of the tube with your pipette so that you can efficiently remove remaining liquid from the bottom of the tube.</li>
<li>Wash the RNA pellets by adding 500 μL of ice-cold 75% ethanol directly over the pellet. Invert to mix.</li>
<li>Centrifuge samples at 16,000×<i>g</i> at 4°C for 5 min.</li>
<li>Carefully remove liquid by pipetting as in 24e., and let the pellet dry by leaving tubes on the bench with their caps open. This should only take 5–10 min. Do not over-dry the pellet at this stage; the objective is for all visible ethanol to be gone.</li>
<li>Resuspend each RNA pellet in 25 μL of nuclease-free water. Incubate samples for 5 min at 18°C–22°C to dissolve RNA. Mix by pipetting gently if necessary.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We do not quantify the yield at this step, as purified RNA levels are usually too low to measure accurately. The entire purified RNA sample is used in the next step.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point, the purified NRO RNA can be stored at −80°C, but we recommend proceeding to the reverse transcription reaction as soon as possible.</div>
<ol start="25">
<li>Reverse-transcribe the purified NRO-RNA and purified total RNA.
<ol type="a">
<li>Assemble a 20 μL reverse transcription reaction by combining one half of the purified NRO-RNA (12 μL) with 4 μL NEB Luna LunaScript RT SuperMix and 4 μL of nuclease-free water.</li>
<li>Use the other half of the purified NRO-RNA to prepare a -RT control reaction by combining 12 μL of purified RNA with 4 μL NEB Luna LunaScript No-RT Control Mix and 4 μL of nuclease-free water.</li>
<li>At this time, also reverse-transcribe 1 μg of each total mRNA sample using LunaScript RT and LunaScript No-RT mixes. Be sure to include a -RT control for these reactions as well.</li>
<li>Incubate reactions in a thermocycler with the following steps:
<table id="undtbl15">
<thead>
<tr>
<th colspan="4">PCR cycling conditions for reverse transcription</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Primer Annealing</td>
<td>25°C</td>
<td>2 min</td>
<td rowspan="3">1</td>
</tr>
<tr>
<td>cDNA synthesis</td>
<td>55°C</td>
<td>10 min</td>
</tr>
<tr>
<td>Heat Inactivation</td>
<td>95°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>Bring all cDNA samples to 80 μL by adding 60 μL of nuclease-free water.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Any other preferred reverse transcription protocol can be used here at this step. Note, however, that for the NRO-RNA samples, random primers (as opposed to oligo-dT primers) must be used for the RT reaction.</div>
<div class="critical">
<span class="critical-title">Critical:</span> No-RT controls for both the NRO and total RNA samples are critical to ensure that signal in the qPCR assay is not coming from genomic DNA contamination. Assuming the DNase I digestion in step 22.c. was successful, there should be little if any signal from the -RT reactions in the qPCR analysis. Monitor at least one primer pair by qPCR of the No-RT cDNA samples to assess for any background signal from DNA. We routinely assess our control primers (<i>GAPDH</i> and <i>ACTB</i>) in No-RT qPCR.</div>
<div class="note">
<span class="note-title">Note:</span> cDNA reactions can be stored for up to a month at −20°C or a year at −80°C.</div>
</li>
</ol>
</li>
</ol>
<ol start="26">
<li>Perform qPCR for the target genes of interest from the NRO cDNA samples.
<ol type="a">
<li>Dilute NRO qPCR primers to a 10 μM working stock using nuclease-free water.</li>
<li>Prepare a master mix for each primer. Each master mix should be sufficient for triplicate reactions for the RT samples, triplicate reactions for the No-RT samples, plus one extra reaction to account for loss. For example, for two NRO samples analyzed with the same primer sets (such as DMSO- and dTAG-treated), including both RT and No-RT reactions, prepare a master mix sufficient for 13 reactions. The table below outlines the recipe for a single reaction of this master mix. Scale up according to your sample number.
<table id="undtbl16">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (in 15 μL reaction)</th>
<th>Amount per reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>2× SYBR FAST qPCR Master Mix</td>
<td>1×</td>
<td>7.5 μL</td>
</tr>
<tr>
<td>Forward primer (10 μM)</td>
<td>200 nM</td>
<td>0.3 μL</td>
</tr>
<tr>
<td>Reverse primer (10 μM)</td>
<td>200 nM</td>
<td>0.3 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>5.9 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>14</b> μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The final concentrations listed in this table refer to the final concentration after addition of the 1 μL cDNA, not to the final concentration of the master mix alone.</div>
</li>
<li>Pipette 14 μL of the appropriate master mix into each well of a qPCR plate.</li>
<li>Add 1 μL of cDNA reaction (prepared in step 25) to each well, as appropriate. Each well will now have a 15 μL volume.</li>
<li>Seal qPCR plate with qPCR Plate Seals.</li>
<li>Spin plate at 300×<i>g</i> for 1 min to collect liquid at the bottom of the wells.</li>
<li>Place the plate in a real-time PCR analyzer, such as the BioRad CFX96 Real-Time PCR Detection System, and perform the following cycling conditions. Note that a melting curve step can be performed as a final step to ensure that the primer pair produces a single product.
<table id="undtbl17">
<thead>
<tr>
<th colspan="4">qPCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>2 s</td>
<td rowspan="2">2–40 cycles</td>
</tr>
<tr>
<td>Annealing/Extension</td>
<td>60°C</td>
<td>20 s</td>
</tr>
<tr>
<td>
<b>Optional:</b> Melt curve</td>
<td>70°C–95°C</td>
<td colspan="2">10 s/0.2°C increment</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
</ol>
<ol start="27">
<li>Perform qPCR for the target genes of interest from the total RNA cDNA samples.
<ol type="a">
<li>Dilute total RNA qPCR primers to a 10 μM working stock using nuclease-free water.</li>
<li>Prepare a master mix for each primer, as in 26.b above. Each master mix should be sufficient for triplicate reactions for the RT samples, triplicate reactions for the No-RT samples, plus one extra reaction to account for loss.</li>
<li>Pipette 14 μL of the appropriate master mix into each well of a qPCR plate.</li>
<li>Add 1 μL of cDNA prepared from the total RNA (step 25) to each well, as appropriate. Each well will now have a 15 μL volume.</li>
<li>Seal qPCR plate with qPCR Plate Seals.</li>
<li>Spin plate at 300×<i>g</i> for 1 min to collect liquid at the bottom of the wells.</li>
<li>Place the plate in a real-time PCR analyzer, such as the BioRad CFX96 Real-Time PCR Detection System, and perform the following cycling conditions. Note that a melting curve step can be performed as a final step to ensure that the primer pair produces a single product.</li>
</ol>
</li>
</ol>
<table id="undtbl18">
<thead>
<tr>
<th colspan="4">qPCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>2 s</td>
<td rowspan="2">2–40 cycles</td>
</tr>
<tr>
<td>Annealing/Extension</td>
<td>60°C</td>
<td>20 s</td>
</tr>
<tr>
<td>
<b>Optional:</b> Melt curve</td>
<td>70°C–95°C</td>
<td colspan="2">10 s/0.2°C increment</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<ol start="28">
<li>Analyze qPCR data by the ΔΔCq method (for both NRO qPCR data and total RNA qPCR data).
<ol type="a">
<li>Calculate the average of the technical triplicates for each primer-sample reaction.</li>
<li>Calculate ΔCq by subtracting the average Cq of the reference gene from the average Cq of the target gene: e.g., ΔCq = (Cq<sub>target</sub> – Cq<sub>reference</sub>)</li>
<li>Calculate ΔΔCq of by subtracting the ΔCq of the control (DMSO) from the ΔCq of the treatment (dTAG): e.g., ΔΔCq = (ΔCq<sub>dTAG</sub> – ΔCq<sub>DMSO</sub>)</li>
<li>Calculate the fold change between conditions by exponentially transforming the ΔΔCq values: calculate two raised to the power of negative ΔΔCq (e.g., Fold Change = 2<sup>−ΔΔCq</sup>)</li>
<li>Graph the Fold Change results. See example graph of results in <a href="#fig5">Figures 5</a>B and 5C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Suitable normalization genes are transcripts that do not change upon depletion of the tagged target. In our experience, both <i>ACTB</i> and <i>GAPDH</i> are suitable normalization controls across assays and cell lines.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The ΔΔCq method reports relative gene expression. Identifying and using an appropriate reference gene(s) that is not affected by degradation of the target protein is thus crucial. If a suitable normalization gene cannot be determined, an alternative quantification method should be used, for example using spike-in biotinylated RNA (see Alternatives below).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> As an alternative to the ΔΔCq method, a biotinylated spike-in control can be used. A protocol for producing BrdU-labeled luciferase transcript spike-in controls is described by Roberts et al. (<a href="#bib10">Roberts et al., 2015</a>), and this protocol could be adapted to produce biotinylated luciferase spike in controls. Such spike in controls should be introduced during the nuclear run-on (at step 21.b.) and prior to RNA purification. For quantification, gene expression can be normalized to qPCR detection of the spike-in RNA(s).</div>
<ol start="29">
<li>Compare the output NRO data to that from the total RNA samples. Changes that are evident in the NRO sample indicate a change in transcription. These changes are not always evident at the total RNA level, depending on the timing of the assay and processing of the transcript. See <a href="#fig5">Figures 5</a>B and 5C for an example comparison of NRO and total RNA levels from the same sample.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>If successful, the expected outcomes of this workflow will be 1. generation of a population of cultured cells in which a protein factor can be selectively and rapidly degraded via the dTAG system, and 2. an understanding of how loss of the factor alters both primary transcriptional events, as well as steady-state transcript levels, of specific target genes.</p>
<p>The process of tagging genes using sgRNA/Cas9 ribonucleoprotein complexes, together with dual fluorescent reporters, is efficient. In the initial validation by flow cytometry described in step 7., expect to see that between 0.1 and 2% of cells are positive for both mCherry and mTagBFP2 markers. After sorting these double-positive cells by FACS (steps 8–14) expect populations in which &gt;90% of endogenous genes are modified to express the FKBP(F36V)-2HA tag. Populations of these modified cells can be maintained for extended analyses, but as there are no selectable markers carried on the integration cassettes it is advisable to periodically check the level of tagged proteins in the population by Western blotting. If necessary, tagged cells can be enriched by additional rounds of FACS, or cloned out for stable long-term propagation.</p>
<p>The dTAG degradation system is also very efficient. Changes in steady-state levels of the target protein are usually evident within two hours, and for PDPK1 (<a href="#bib3">Guarnaccia et al., 2021</a>), WDR5 (<a href="#bib1">Betz et al., 2002</a>; <a href="#bib3">Guarnaccia et al., 2021</a>), and HCFC1 (<a href="#bib8">Popay et al., 2021</a>) greater than 90% of these proteins are depleted within six hours. There are reports of some proteins being recalcitrant to dTAG-mediated destruction, at least with CRBN-based degraders (<a href="#bib6">Nabet et al., 2020</a>). If, therefore, you have successfully tagged your protein and it resists degradation in response to dTAG-47 or dTAG-13, it may be useful to use the dTAG<sup>V</sup> degrader, either alone or in combination with a CRBN-based degrader (<a href="#bib6">Nabet et al., 2020</a>).</p>
<p>Through scale-down and adaptation of PRO-Seq technologies, the protocol presented here provides a simple and robust way to monitor primary transcriptional changes at specific genes after factor destruction. The expected outcome of NRO experiments is very much dependent on the nature of the question being asked. But minimally, the results will determine whether or not degradation of a specific protein results in rapid transcriptional changes at genes of interest. Changes observed at early timepoints after factor destruction are highly indicative of a primary mode of transcriptional regulation. By varying degradation timepoints and comparing with parallel analyses of steady state mRNA levels, it will be possible to know whether changes in transcription lead to changes in the abundance of mature mRNAs. Alternatively, if primary transcription is unaffected in response to factor degradation but mRNA levels do change, it is possible to infer that mRNA levels are regulated at a post-transcriptional level.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Success in this workflow depends on the ability to deliver DNA and sgRNA-Cas9 RNP complexes into cells with high efficiency; cells refractory to such methods will not be suitable for this approach. Tagging the protein of interest with the FKBP(F36V)-2HA tag may impact its function in ways that could be either obvious or opaque. If there are known functional consequences of C-terminal tagging, N-terminal tagging is also an option. We begin with a C-terminal tagging approach because it maintains continuity between the gene promoter and its coding sequences, but N-terminal tagging is an option, and one we used successfully for HCFC1 (<a href="#bib8">Popay et al., 2021</a>). Additionally, the use of fluorescent reporters as indicators of gene tagging could limit downstream analyses that require fluorescent-based imaging, although this impact can be minimized by careful experimental design and utilization of different or inducible fluorophores. Finally, the most obvious limitation of the NRO-RT-qPCR assay is that it is candidate gene-selective and requires upfront knowledge of which genes to interrogate. PRO-Seq is clearly a better option if an unbiased survey is required. But because of its simplicity and rapid turnaround, the NRO-RT-qPCR assay is a viable option when transcriptional events at a defined set of genes are the subject of investigation, or as a feasibility study for a PRO-Seq analysis.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>No specific gRNA sequence available within 20 bp of the stop codon. See <a href="#before-you-begin">before you begin</a> step 1.c.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Selecting an sgRNA sequence within 20 bp of the stop codon is crucial because it ensures that integration of the FKBP(F36V)-2HA tag will destroy the sgRNA target sequence and prevent cutting by Cas9 after integration. If an appropriate sgRNA cannot be identified, an alternative approach is to use an sgRNA that is as close as possible to the stop codon and modify the appropriate homology arm with mutations to block Cas9 cutting after integration. The simplest way to do this is to introduce a silent mutation that disables the PAM sequence.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>No fluorescent cells apparent by flow cytometry analysis. See step 7.c.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This may occur for several reasons. The transfection may have failed. The sgRNA may have degraded. Check the absorbance of the stock at 260 nm, comparing the expected concentration of sgRNA to the printed value provided with the QC sheet from Synthego. Depending on expression levels of the target protein, expression of the fluorescent proteins may also be too low to detect. Use an orthologous method, such as Western blotting, to probe for the presence of modified target protein in the population (e.g., <a href="#fig4">Figure 4</a>A). If addressing these issues does not lead to a successful outcome, try additional sgRNAs or switch to an N-terminal tagging strategy. For N-terminal tagging, additional vector cloning is required to position the fluorescent marker-P2A ahead of the 2HA-FKBP(F36V) tag and in frame with the gene of interest. We have been successful in tagging HCFC1 at the N-terminus (<a href="#bib8">Popay et al., 2021</a>).</p>
<h3 id="sec6.5">Problem 3</h3>
<p>mCherry positive cells are present, but not mTagBFP2 positive cells. See step 7.c.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>We have found that mTagBFP2 is not as bright as mCherry. This lower fluorescence intensity can be problematic for detection by flow cytometry—this was our experience when tagging <i>PDPK1</i>. There are two possible solutions to this problem. First, ignore mTagBFP2 and try gating on the brightest fraction of mCherry-positive cells; this can isolate a population of multi-allelically tagged cells, presumably by selecting cells where mCherry has integrated at multiple alleles. Second, you could use a different monomeric fluorescent protein, for example mNeon instead of mTagBFP2, modifying one of the targeting vectors as required.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Sorted population of cells is not pure; both tagged and untagged proteins are evident by Western blotting. See step 15.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Incomplete tagging of proteins in the sorted population is not uncommon. The simplest solution is to perform multiple rounds of enrichment by FACS. Cloning individual modified cells is also a solution, although if this route is taken it is advisable to work in parallel with multiple clones to avoid clone-specific artifacts. If neither of these approaches yields suitable cells for analysis, it is possible that the gene of interest is present in more than two copies in the cell line, or undergoes some kind of regulatory event (e.g., alternative splicing) that makes it impossible to tag all forms of the protein via this strategy. In this case, one option is to change cell types, or express the required FKBP(F36V) fusion protein exogenously, such as via lentiviral transduction, and then knock out the endogenous protein by traditional CRISPR/Cas9 genome editing. An example of this latter strategy can be found in (<a href="#bib14">Weissmiller et al., 2019</a>).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>No RNA pellet or altered appearance of RNA pellet following bead binding/RNA extraction from magnet beads. See step 24.d.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The NRO-RNA pellet formed after centrifugation should be blue colored and flat. The pellet may be against the side of the tube, but in most cases it will float to the bottom of the tube as a small disc. Occasionally, the pellet at this step may be rounded, resembling a small bubble. This outcome is indicative of carry-over of material from the organic phase during the extraction. Do not proceed if this occurs, as subsequent reactions will be inhibited. Instead, perform the following steps to remove contaminating material:</p>
<p>Redissolve round pellet in 100 μL of nuclease-free water</p>
<p>Add 300 μL of TRIzol LS to the redissolved RNA. Vortex for 10 s at high speed and place on a rotator for 5 min at 18°C–22°C.</p>
<p>Add 80 μL of chloroform and vortex for 1 min.</p>
<p>Spin tube for 5 min at 16,000×<i>g</i>.</p>
<p>Transfer aqueous phase to a fresh tube. To ensure you do not carry over any organic phase leave a thin line of the aqueous layer behind. It is better to obtain a pure aqueous phase rather than run the risk of pipetting any organic layer on accident.</p>
<p>Repeat steps 24.a.-i. If the RNA pellet appears flat and blue, you can proceed. But if you still see a small round ball floating at the bottom, repeat this re-extraction until you obtain an RNA pellet that contains no organic solvents.</p>
<p>It is also possible that you will see no pellet at all after the final centrifugation step. This can be due to issues with precipitation. Some tips that we have found that work are:</p>
<p>Make sure to mix well by vortexing vigorously once you add the Ambion GlycoBlue Coprecipitant and the ethanol.</p>
<p>Add 0.4× isopropanol, in addition to the Ambion GlycoBlue Coprecipitant and ethanol, to assist in precipitation.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Betz, B.L., Strobeck, M.W., Reisman, D.N., Knudsen, E.S., and Weissman, B.E. (2002). Re-expression of hSNF5/INI1/BAF47 in pediatric tumor cells leads to G1 arrest associated with induction of p16ink4a and activation of RB. Oncogene <i>21</i>, 5193-5203. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref1">View at publisher</a></p>
<p id="bib2">Concordet, J.P. and Haeussler, M. (2018). CRISPOR: intuitive guide selection for CRISPR/Cas9 genome editing experiments and screens. Nucleic Acids Res. <i>46</i>, W242-W245. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref2">View at publisher</a></p>
<p id="bib3">Guarnaccia, A.D., Rose, K.L., Wang, J., Zhao, B., Popay, T.M., Wang, C.E., Guerrazzi, K., Hill, S., Woodley, C.M., Hansen, T.J., et al. (2021). Impact of WIN site inhibitor on the WDR5 interactome. Cell Rep <i>34</i>, 108636. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref3">View at publisher</a></p>
<p id="bib4">Huang, H.T., Seo, H.S., Zhang, T., Wang, Y., Jiang, B., Li, Q., Buckley, D.L., Nabet, B., Roberts, J.M., Paulk, J., et al. (2017). MELK is not necessary for the proliferation of basal-like breast cancer cells. Elife <i>6</i>, e26693. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref4">View at publisher</a></p>
<p id="bib5">Mahat, D.B., Kwak, H., Booth, G.T., Jonkers, I.H., Danko, C.G., Patel, R.K., Waters, C.T., Munson, K., Core, L.J., and Lis, J.T. (2016). Base-pair-resolution genome-wide mapping of active RNA polymerases using precision nuclear run-on (PRO-seq). Nat. Protoc. <i>11</i>, 1455-1476. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref5">View at publisher</a></p>
<p id="bib6">Nabet, B., Ferguson, F.M., Seong, B.K.A., Kuljanin, M., Leggett, A.L., Mohardt, M.L., Robichaud, A., Conway, A.S., Buckley, D.L., Mancias, J.D., et al. (2020). Rapid and direct control of target protein levels with VHL-recruiting dTAG molecules. Nat. Commun. <i>11</i>, 4687. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref6">View at publisher</a></p>
<p id="bib7">Nabet, B., Roberts, J.M., Buckley, D.L., Paulk, J., Dastjerdi, S., Yang, A., Leggett, A.L., Erb, M.A., Lawlor, M.A., Souza, A., et al. (2018). The dTAG system for immediate and target-specific protein degradation. Nat. Chem. Biol. <i>14</i>, 431-441. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref7">View at publisher</a></p>
<p id="bib8">Popay, T.M., Wang, J., Adams, C.M., Howard, G.C., Codreanu, S.G., Sherrod, S.D., McLean, J.A., Thomas, L.R., Lorey, S.L., Machida, Y.J., et al. (2021). MYC regulates ribosome biogenesis and mitochondrial gene expression programs through its interaction with host cell factor-1. Elife <i>10</i>, e60191. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref8">View at publisher</a></p>
<p id="bib9">Richardson, C.D., Ray, G.J., DeWitt, M.A., Curie, G.L., and Corn, J.E. (2016). Enhancing homology-directed genome editing by catalytically active and inactive CRISPR-Cas9 using asymmetric donor DNA. Nat. Biotechnol. <i>34</i>, 339-344. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref9">View at publisher</a></p>
<p id="bib10">Roberts, T.C., Hart, J.R., Kaikkonen, M.U., Weinberg, M.S., Vogt, P.K., and Morris, K.V. (2015). Quantification of nascent transcription by bromouridine immunocapture nuclear run-on RT-qPCR. Nat. Protoc. <i>10</i>, 1198-1211. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref10">View at publisher</a></p>
<p id="bib11">Stengel, K.R., Ellis, J.D., Spielman, C.L., Bomber, M.L., and Hiebert, S.W. (2021). Definition of a small core transcriptional circuit regulated by AML1-ETO. Mol. Cell <i>81</i>, 530-545.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref11">View at publisher</a></p>
<p id="bib12">Thomas, L.R., Adams, C.M., Wang, J., Weissmiller, A.M., Creighton, J., Lorey, S.L., Liu, Q., Fesik, S.W., Eischen, C.M., and Tansey, W.P. (2019). Interaction of the oncoprotein transcription factor MYC with its chromatin cofactor WDR5 is essential for tumor maintenance. Proc. Natl. Acad. Sci. U S A <i>116</i>, 25260-25268. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref12">View at publisher</a></p>
<p id="bib13">Weintraub, A.S., Li, C.H., Zamudio, A.V., Sigova, A.A., Hannett, N.M., Day, D.S., Abraham, B.J., Cohen, M.A., Nabet, B., Buckley, D.L., et al. (2017). YY1 is a structural regulator of enhancer-promoter loops. Cell <i>171</i>, 1573-1588.e28. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref13">View at publisher</a></p>
<p id="bib14">Weissmiller, A.M., Wang, J., Lorey, S.L., Howard, G.C., Martinez, E., Liu, Q., and Tansey, W.P. (2019). Inhibition of MYC by the SMARCB1 tumor suppressor. Nat. Commun. <i>10</i>, 2014. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref14">View at publisher</a></p>
<p id="bib15">Woodley, C.M., Romer, A.S., Wang, J., Guarnaccia, A.D., Elion, D.L., Maxwell, J.N., Guerrazzi, K., McCann, T.S., Popay, T.M., Matlock, B.K., et al. (2021). Multiple interactions of the oncoprotein transcription factor MYC with the SWI/SNF chromatin remodeler. Oncogene <i>40</i>, 3593-3609. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00706-1/sref15">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, William P. Tansey (<a href="mailto:william.p.tansey@vanderbilt.edu">william.p.tansey@vanderbilt.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<p>The dTAG-47 degrader (<a href="#bib4">Huang et al., 2017</a>) is not commercially available, but can be chemically synthesized. The dTAG13 degrader (<a href="#bib7">Nabet et al., 2018</a>) is a similar degrader molecule and is available from Tocris Bioscience (Cat# 6605).</p>
<h4>Data and code availability</h4>
<p>This study did not generate datasets or code.</p>
<h3>Acknowledgments</h3>
<p>Graphical abstract was created in part with <a href="http://BioRender.com">BioRender.com</a>. For reagents we thank Richard Young. For collaboration, assistance, and advice we thank David Flaherty, Brittany Matlock, Kristy Stengel, Monica Bomber, Scott Hiebert, and Vivian Gama. The dTAG-47 degrader molecule was synthesized by the Vanderbilt Chemical Synthesis Core. The VUMC Flow Cytometry Shared Resource is supported by the Vanderbilt Ingram Cancer Center (P30CA68485) and the Vanderbilt Digestive Disease Research Center (DK058404). This work was supported by awards from the NIH/NCI—under Chemical Biology Consortium Contract No. HHSN261200800001E (W.P.T.), CA247833 (W.P.T.), CA200709 (W.P.T.), F31CA225065 (A.D.G), T32CA009582 (A.D.G.), T32CA119925 (A.M.W. and W.P.T.), and by the Rally Foundation for Childhood Cancer Research Young Investigator Award (A.M.W.).</p>
<h3>Author contributions</h3>
<p>Conceptualization, A.D.G., A.M.W., and W.P.T.; methodology, A.D.G., A.M.W., and W.P.T.; investigation, A.D.G.; data curation, A.D.G. and A.M.W.; writing – original draft, A.D.G., A.M.W., and W.P.T.; writing – review &amp; editing, A.D.G., A.M.W., and W.P.T.; visualization, A.D.G.; supervision, A.M.W. and W.P.T.; funding acquisition, A.D.G., A.M.W., and W.P.T.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests. A.D.G. is currently an employee of Genentech, Inc.</p>
</section>
</article>