<article>
<section>
<h1 id="header">A simple protocol to produce mature human-induced pluripotent stem cell-derived cardiomyocytes</h1>
<p><time datetime="2021-10-29">Published: October 29, 2021</time></p>
<p>Walter E. Knight,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">5</a>,<a href="#cor2">**</a></sup> Yingqiong Cao,<sup><a href="#aff1">1</a></sup> Phoebe Dillon,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff4">4</a></sup> and Kunhua Song<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">6</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Division of Cardiology, University of Colorado Anschutz Medical Campus, Aurora, CO 80045, USA</p>
<p id="aff2"><sup>2</sup>Gates Center for Regenerative Medicine and Stem Cell Biology, University of Colorado Anschutz Medical Campus, Aurora, CO 80045, USA</p>
<p id="aff3"><sup>3</sup>Consortium for Fibrosis Research and Translation, University of Colorado Anschutz Medical Campus, Aurora, CO 80045, USA</p>
<p id="aff4"><sup>4</sup>The University of Denver, Denver, CO 80210, USA</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kunhua.song@cuanschutz.edu">kunhua.song@cuanschutz.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:walter.knight@cuanschutz.edu">walter.knight@cuanschutz.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100912">10.1016/j.xpro.2021.100912</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>When cultured under typical conditions, human-induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs) are structurally and functionally immature. We have previously demonstrated that culture of hiPSC-CMs in maturation medium containing fatty acids, in combination with culture on micropatterned surfaces, produces cells that demonstrate a more mature phenotype compared to standard approaches. Here, we show in detail the steps needed to produce mature hiPSC-CMs. Compared with many approaches, our protocol is relatively simple and can be easily adapted to new laboratories.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref6">Knight et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Two-step protocol to increase maturity of stem cell-derived cardiomyocytes</li>
<li>Improves sarcomeric organization, contractile function, and metabolic behavior</li>
<li>Improved cardiomyocyte maturation suppresses baseline hypertrophy</li>
<li>Simple protocol which most labs should be able to implement</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1130-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes a method for maturing cardiomyocytes derived from human induced pluripotent stem cells (hiPSCs). While hiPSC-derived cardiomyocytes (hiPSC-CMs) represent a powerful tool for characterizing cardiovascular disease <i>in vitro</i>, when cultured under standard conditions, hiPSC-CMs are structurally and functionally immature; in other words, they resemble fetal, rather than adult cardiomyocytes. This fetal phenotype is associated with primarily glycolytic, rather than fatty acid-based, metabolic activity, weak cell and myofibril contractile force, poorly organized sarcomeres, and relatively high cell cycle activity (<a href="#bib2">Birket et al., 2015</a>; <a href="#bib5">Correia et al., 2017</a>; <a href="#bib9">Mills et al., 2017</a>; <a href="#bib10">Pioner et al., 2016</a>; <a href="#bib13">Yang et al., 2014</a>). Additionally, prolonged glucose-based culture induces significant hypertrophy and B-type natriuretic peptide (BNP) expression (<a href="#bib6">Knight et al., 2021</a>). Therefore, using maturation-inducing methods for hiPSC-CM based disease models is crucial. Here, we present a relatively simple method to produce mature hiPSC-CMs, which structurally and functionally recapitulate phenotypes of adult cardiomyocytes to some degree. These methods are relatively simple and require only commercially available reagents and materials. While we briefly describe a protocol to differentiate and select for hiPSC-CMs below, different differentiation protocols may be optimal for different hiPSC lines. We have successfully used this protocol for multiple hiPSC lines; however it should presumably be applicable to any hiPSC line, commercially available hiPSC-CMs, embryonic stem cell line, and possibly for cardiomyocytes derived from non-human iPSC lines as well. However, we will also briefly summarize our hiPSC-CM differentiation and selection protocol below. We prepare hiPSC-CMs via modulation of Wnt signaling, using the method described by Lian et al. (<a href="#bib7">Lian et al., 2012</a>, <a href="#bib8">2013</a>) and select for cardiomyocytes using the lactate method, as described previously (<a href="#bib11">Tohyama et al., 2013</a>). Recipes for all types of cell media used in this study are also listed in <a href="#tbl1">Table 1</a>.</p>
<table id="tbl1">
<caption>Table 1. Reference of constituents of various types of cell media used for this study</caption>
<thead>
<tr>
<th>Medium type</th>
<th>Basal medium type</th>
<th>Additive 1 /Dilution or concentration</th>
<th>Additive 2 /Dilution or concentration</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI-20 (for replating hiPSC-CMs)</td>
<td>RPMI-1640 with glucose</td>
<td>Fetal bovine serum – 20% (V:V)</td>
<td>Y-27632 (5 μM) – add immediately before use</td>
<td>Store at 4°C for up to 1 month. Y-2 is only necessary when plating cells (not for halting digestion)</td>
</tr>
<tr>
<td>RPMI-1640 (for hiPSC-CM culture)</td>
<td>RPMI-1640 with glucose</td>
<td>B-27™ supplement (with or without insulin as specified in the protocol) 1:50 dilution (V:V)</td>
<td> </td>
<td>Store at 4°C for up to 1 month.</td>
</tr>
<tr>
<td>DMEM with lactate (for hiPSC-CM selection)</td>
<td>DMEM, glucose-free</td>
<td>1M (L+)-lactic acid solution - 1:250 dilution (V:V) – final [] = 4mM</td>
<td> </td>
<td>Store at 4°C for up to 1 month.</td>
</tr>
<tr>
<td>50<b>×</b> (500 mM) galactose solution</td>
<td>RPMI-1640 <b>without</b> glucose</td>
<td>Galactose = to final concentration of 500 mM: 45.1 g/500mL</td>
<td> </td>
<td>See “<a href="#step-by-step-method-details">step-by-step method details</a>” for solution preparation. Store at −20°C for up to 3 months</td>
</tr>
<tr>
<td>Na<sub>2</sub> EGTA (10mL) - used for myofibril lysis buffer</td>
<td>MQ H<sub>2</sub>O</td>
<td>0.380 g EGTA powder</td>
<td>0.08 g NaOH powder</td>
<td>pH to 7 with KOH. filter with 0.4 μm filter. Store at −20°C for up to 1 year.</td>
</tr>
<tr>
<td>1 M K-prop solution(10mL) - used for myofibril lysis buffer</td>
<td>10 mL 1M propionic acid solution</td>
<td>0.561 KOH powder</td>
<td> </td>
<td>Mix KOH into propionic acid and filter with 0.4 μm filter. Store at −20°C for up to 1 year.</td>
</tr>
</tbody>
</table>
<table id="tbl2">
<caption>Table 2. 50× fatty acid solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (for 10 mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI-1640 <b>without</b> glucose</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
<tr>
<td>BSA Fraction V (Fatty acid free)</td>
<td>10% (W:V)</td>
<td>1 g</td>
</tr>
<tr>
<td>250 mM palmitic acid</td>
<td>2.5 mM (1:100 dilution)</td>
<td>100 μL</td>
</tr>
<tr>
<td>500 mM oleic acid</td>
<td>5 mM (1:100 dilution)</td>
<td>100 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">See “<a href="#step-by-step-method-details">step-by-step method details</a>” for fatty acid conjugation. Store at −20°C for up to 3 months.</p>
<table id="tbl3">
<caption>Table 3. Maturation medium (for hiPSC-CM maturation)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (for 500 mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI-1640 <b>without</b> glucose</td>
<td>N/A</td>
<td>470 mL</td>
</tr>
<tr>
<td>50<b>×</b> fatty acid solution</td>
<td>1<b>×</b> (100 μM oleic acid, 50 μM palmitic acid)</td>
<td>10 mL</td>
</tr>
<tr>
<td>50<b>×</b> galactose solution</td>
<td>1<b>×</b> (10 mM galactose)</td>
<td>10 mL</td>
</tr>
<tr>
<td>50<b>×</b> B-27™ supplement <b>with</b> insulin</td>
<td>1<b>×</b>
</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">See “<a href="#step-by-step-method-details">step-by-step method details</a>” for solution preparation. Filter w/ 0.22 μm filter before use. Good for 2–3 weeks if stored at 4°C</p>
<h3 id="sec1.1">Stem cell derived cardiomyocyte differentiation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 weeks before cells will be sufficiently differentiated for maturation methods.</div>
<div class="note">
<span class="note-title">Note:</span> We prepare hiPSC-CMs via modulation of Wnt signaling, using the method described by Lian et al. (<a href="#bib7">Lian et al., 2012</a>, <a href="#bib8">2013</a>).</div>
<ol>
<li>Prepare Matrigel-coated plates:
<ol type="a">
<li>When Matrigel is received, aliquot it such that each individual aliquot is sufficient to produce 6.25 mL of 1<b>×</b> Matrigel solution (the exact volume will depend on the lot of Matrigel, and a dilution factor will be provided by the manufacturer). Store Matrigel aliquots at −80°C.</li>
<li>At least two days before plating hiPSCs, thaw out an aliquot of Matrigel overnight (12–18 h) at 4°C.</li>
<li>Working in the sterile hood and keeping Matrigel on ice, dilute it into 6.25 mL cold DMEM/F12 medium. Take care to keep everything cold to prevent Matrigel solution from solidifying.</li>
<li>Add 0.3 mL Matrigel solution per 24 well plate well to be used for induction. Add 6.25 mL solution per 10 cm dish used to grow up hiPSCs.</li>
<li>Wrap plate with paraffin, cover in foil, and store overnight (12–18 h) at 4°C. Plates may be stored in the refrigerator for up to 10 days before use.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Plate hiPSCs for induction
<div class="note">
<span class="note-title">Note:</span> hiPSC-CM induction, all steps are conducted with 1mL of cell medium per 24 well plate well unless otherwise specified.</div>
<ol type="a">
<li>Culture hiPSCs of passage 10–30 to 80%–90% confluence on 10 cm Matrigel-coated plates, in mTeSR1™ medium. One confluent 10 cm dish will produce approximately 10 million hiPSCs.</li>
<li>At least one hour before replating cells, remove Matrigel coated 24 well plates from the refrigerator and warm up to room temperature (∼22°C).</li>
<li>Dissociate hiPSCs using 2.5 mL Accumax™ dissociation medium. 3–5 min incubation at 37°C is typically sufficient.</li>
<li>Count cells using hematocytometer and calculate cell yield and density.</li>
<li>Centrifuge cells at 228 <i>g</i> and resuspend to a density of 1 million cells per mL in mTeSR1™ supplemented with 5 μM Y-27632.</li>
<li>Plate 375,000 hiPSCs (375 μL) per well onto Matrigel-coated 24 well plate wells (24WPW) and add an additional 0.5 mL medium to each well.
<div class="note">
<span class="note-title">Note:</span> This cell number has been found by our lab to be optimal for induction efficiency. Using different hiPSC lines, it may need to be altered to some degree. For the purposes of the induction timeline, this is considered Day -4 (4 days prior to initiating induction).</div>
</li>
<li>Culture hiPSCs for 4 days with daily media changes (1 mL mTeSR1™ per well).
<div class="critical">
<span class="critical-title">Critical:</span> When removing old medium by suction, take care not to scratch off cells from the culture dish.</div>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Day 0 (D0): Initiate induction:
<ol type="a">
<li>Initiate cardiomyocyte induction by changing medium to RPMI-1640 with B-27™ supplement without insulin, supplemented with 6–14 μM CHIR99021 (GSK3β inhibitor).
<div class="critical">
<span class="critical-title">Critical:</span> For most hiPSC lines in use in our lab, we have identified 8 μM CHIR99021 as optimal for inducing cardiomyocyte induction. However, the optimal concentration will vary by cell line (<a href="#bib8">Lian et al., 2013</a>). We highly recommend optimizing CHIR99021 concentration when inducing cardiomyocytes from a new cell line. Additionally, record the time at which this step was performed, as the next step should be performed exactly 24 hours later.</div>
</li>
<li>Exactly 24 h later (D1), change medium to RPMI-1640 with glucose, with B-27™ supplement without insulin.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>D3: Add combined medium:
<ol type="a">
<li>Prepare combined medium: from each well of cells, carefully decant 0.5 mL of old medium into 50 mL conical tube. Combine this with an equal volume of fresh RPMI-1640 with glucose, with B-27™ supplement without insulin, and add 5 μM IWP2 (Wnt inhibitor).</li>
<li>Remove the remaining medium from each well, and replace with combined medium.</li>
<li>Repeat this process for each hiPSC line being induced (do not add medium harvested from one cell genotype to cells from another genotype).</li>
</ol>
</li>
<li>Continue cell culture:
<ol type="a">
<li>On D5, change medium to RPMI-1640 with glucose, with B-27™ supplement without insulin.</li>
<li>On D7, and every 3 days thereafter, RPMI-1640 with glucose, with B-27™ supplement with insulin.</li>
</ol>
</li>
<li>D15-18: Replate cells: induced hiPSCs typically begin forming beating clusters on D12. Once robust beating is established (usually Day 15–18, large clusters of cardiomyocytes should be visible beating using a low magnification microscope) cells are replated from 24 well plates onto 6 well plates or 10 cm dishes:
<ol type="a">
<li>The night before replating, coat new cell culture dishes with sterile 0.1% gelatin dissolved in water, overnight (12–18 h) in 37°C cell culture incubator.</li>
<li>Wash cells once with Dulbecco’s phosphate buffered saline (DPBS), and incubate them with 0.25% Trypsin-EDTA for 7 min at 37°C (0.5 mL per well).</li>
<li>Halt digestion via addition of RPMI-1640 medium with glucose, supplemented with 20% fetal bovine serum (FBS) (RPMI-20, 1 mL per well). Pipette cells robustly (approximately 10 times) to detach them, and pipette into a secondary container.</li>
<li>Pellet cells: centrifuge at 228 <i>g</i> for 3 min, and resuspend pellets in RPMI-20 supplemented with 5 μM Y-27632.</li>
<li>Plate cells onto gelatin-coated plates (remove gelatin solution first by suction). We typically plate 1∗24WPW onto 1∗6WPW or 4∗24WPW onto 1∗10 cm dish. Use standard media volumes for subsequent culture steps.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If culturing cells under serum-free conditions, RPMI-20 may be replaced with STEMdiff™ Cardiomyocyte Support Medium from Stemcell Technologies. As an alternative to gelatin, induced cardiomyocytes may be plated on fibronectin-coated surfaces (see step 20 for information on coating cell culture dishes with fibronectin) although this will make further replating more difficult, as we have found it is harder to detach cells from fibronectin-coated than from gelatin-coated surfaces. Potentially, other surface coating materials, such as Matrigel® or laminin, may be used, although we have not tried this ourselves.</div>
<ol start="7">
<li>∼D18-20: Purify cardiomyocytes: 2 days after replating cells. When inducing stem cells into cardiomyocytes, not all cells will assume cardiomyocyte identity. Therefore, it is beneficial to purify cardiomyocytes. This can drastically increase the percentage of cells in culture demonstrating cardiomyocyte identity, producing cultures with greater than 95% Troponin-T positive cells as assessed by flow cytometry (<a href="#bib4">Chi et al., 2018</a>).
<ol type="a">
<li>Before manipulating cells, prepare 1M lactate solution by diluting 10M L(+)-Lactic acid 1:10 (V:V) in sterile double distilled water.</li>
<li>Prepare lactate medium by adding 1M lactate solution to glucose-free DMEM to final concentration of 4 mM (1:250 dilution by volume)</li>
<li>Filter medium using 0.22 μm filter. Warm medium to 37°C before using.</li>
<li>Wash cells 1<b>×</b> with DPBS.</li>
<li>Remove PBS and add lactate medium to cells.</li>
<li>Culture cells in lactate-supplemented DMEM for 5 days, with 1 medium change 2 or 3 days after initiating selection.</li>
<li>After 5 days of lactate selection, replace medium with RPMI-1640 supplemented with B-27™ supplement with insulin. Culture cells in this medium for at least 48 h.</li>
</ol>
</li>
<li>∼D25-27: Continue to culture cells prior to use of maturation methods. At this point, cells may continue to be cultured in RPMI-1640 supplemented with B-27™ supplement with insulin with medium changes twice weekly, or changed to maturation medium, and/or replated onto patterned surfaces. We recommend replating and switching to maturation medium within 40 days of inducing cells to avoid prolonged culture in glucose medium.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-NT-proBNP antibody [15F11] (Use at 1:200 dilution)</td>
<td>Abcam</td>
<td>Ab13115</td>
</tr>
<tr>
<td>Anti-sarcomeric alpha actinin (Use at 1:200 dilution)</td>
<td>Millipore Sigma</td>
<td>A7811</td>
</tr>
<tr>
<td>Alexafluor 488 goat anti-mouse (Use at 1:400 dilution)</td>
<td>Thermo Fisher Scientific</td>
<td>A11034</td>
</tr>
<tr>
<td>Alexafluor 555 goat anti-mouse (Use at 1:400 dilution)</td>
<td>Thermo Fisher Scientific</td>
<td>A21422</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Hoescht 33342 (Use at 1:5000 dilution)</td>
<td>Life Technologies</td>
<td>H1399</td>
</tr>
<tr>
<td>Y-27632 dihydrochloride (Rock Inhibitor)</td>
<td>ApexBio</td>
<td>Cat # A3008</td>
</tr>
<tr>
<td>CHIR99021 (GSK3 inhibitor)</td>
<td>Cayman Chemical</td>
<td>Cat # 13122</td>
</tr>
<tr>
<td>IWP2 (Wnt inhibitor)</td>
<td>Tocris</td>
<td>Cat # 3533</td>
</tr>
<tr>
<td>L(+)-Lactic Acid, 90% solution in water (10M)</td>
<td>Acros Organics</td>
<td>Cat # AC 189872500</td>
</tr>
<tr>
<td>B-27™ supplement (50<b>×</b>) with insulin</td>
<td>Gibco</td>
<td>Cat # 17–504-044</td>
</tr>
<tr>
<td>B-27™ supplement (50<b>×</b>) without insulin</td>
<td>Gibco</td>
<td>Cat # A18956-01</td>
</tr>
<tr>
<td>BSA Fraction V</td>
<td>Goldbio</td>
<td>Cat # A-421-50, Cas 9048-46-8, <a href="https://www.goldbio.com/product/6281/bovine-serum-albumin-bsa-fraction-v-fatty-acid-free-for-tissue-culture">https://www.goldbio.com/product/6281/bovine-serum-albumin-bsa-fraction-v-fatty-acid-free-for-tissue-culture</a>
</td>
</tr>
<tr>
<td>D-Galactose</td>
<td>Sigma</td>
<td>Cat # G0750, Cas 59-23-4, <a href="https://www.sigmaaldrich.com/US/en/product/sial/g0750?gclid=Cj0KCQjw0K-HBhDDARIsAFJ6UGiwW31tjdcebXhqXgtKOuPUpVlMPXRB8qh1QMPQdRtMfzL9_nuWK7oaAvnKEALw_wcB">https://www.sigmaaldrich.com/US/en/product/sial/g0750?gclid=Cj0KCQjw0K-HBhDDARIsAFJ6UGiwW31tjdcebXhqXgtKOuPUpVlMPXRB8qh1QMPQdRtMfzL9_nuWK7oaAvnKEALw_wcB</a>
</td>
</tr>
<tr>
<td>Oleic Acid</td>
<td>MP Biomedicals</td>
<td>Cat # 151781, Cas 112-80-1, <a href="https://www.mpbio.com/us/oleic-acid-99">https://www.mpbio.com/us/oleic-acid-99</a>
</td>
</tr>
<tr>
<td>D-Galactose</td>
<td>Sigma</td>
<td>Cat # G0750, Cas 59-23-4, <a href="https://www.sigmaaldrich.com/US/en/product/sial/g0750?gclid=Cj0KCQjw0K-HBhDDARIsAFJ6UGiwW31tjdcebXhqXgtKOuPUpVlMPXRB8qh1QMPQdRtMfzL9_nuWK7oaAvnKEALw_wcB">https://www.sigmaaldrich.com/US/en/product/sial/g0750?gclid=Cj0KCQjw0K-HBhDDARIsAFJ6UGiwW31tjdcebXhqXgtKOuPUpVlMPXRB8qh1QMPQdRtMfzL9_nuWK7oaAvnKEALw_wcB</a>
</td>
</tr>
<tr>
<td>Palmitic Acid</td>
<td>Sigma</td>
<td>Cat # P0500, Cas 57-10-3, <a href="https://www.sigmaaldrich.com/US/en/product/sigma/p0500">https://www.sigmaaldrich.com/US/en/product/sigma/p0500</a>
</td>
</tr>
<tr>
<td>Cellmask™ Orange</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # H32713</td>
</tr>
<tr>
<td>Fibronectin</td>
<td>Corning</td>
<td>Cat # 356008</td>
</tr>
<tr>
<td>Dry Milk Powder</td>
<td>Research Products International</td>
<td>Cat # M17200</td>
</tr>
<tr>
<td>Dulbecco’s Phosphate Buffered Saline (DPBS)</td>
<td>Corning</td>
<td>Cat # 21-031-CV</td>
</tr>
<tr>
<td>Triton™-X 100</td>
<td>Sigma</td>
<td>Cat # X100</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Gibco</td>
<td>Cat # 10437-028</td>
</tr>
<tr>
<td>Glucose free DMEM</td>
<td>Gibco</td>
<td>Cat # 11966-025</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>Gibco</td>
<td>Cat # 10565-018</td>
</tr>
<tr>
<td>RPMI 1640 media glucose free (for maturation medium)</td>
<td>Gibco</td>
<td>Cat # 11879-020</td>
</tr>
<tr>
<td>RPMI 1640 media with glucose</td>
<td>Gibco</td>
<td>Cat # 11875-093</td>
</tr>
<tr>
<td>STEMdiff™ Cardiomyocyte Support Medium (for replating cells)</td>
<td>STEMCELL Technologies</td>
<td>Cat # 05027</td>
</tr>
<tr>
<td>Accumax™ Cell Detachment Solution</td>
<td>STEMCELL Technologies</td>
<td>Cat # 07921</td>
</tr>
<tr>
<td>mTESR1™ Medium</td>
<td>STEMCELL Technologies</td>
<td>Cat # 85850</td>
</tr>
<tr>
<td>Matrigel</td>
<td>Corning</td>
<td>Cat # 354277</td>
</tr>
<tr>
<td>Gelatin</td>
<td>Sigma</td>
<td>Cat # G2500</td>
</tr>
<tr>
<td>32% paraformaldehyde</td>
<td>Electron Microscopy Services</td>
<td>Cat # 15714S</td>
</tr>
<tr>
<td>EGTA</td>
<td>Millipore Sigma</td>
<td>Cat # E4378</td>
</tr>
<tr>
<td>KOH</td>
<td>Millipore Sigma</td>
<td>Cat # P5958</td>
</tr>
<tr>
<td>Propionic acid</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # A258-500</td>
</tr>
<tr>
<td>Na<sub>2</sub>SO<sub>4</sub>
</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # S379</td>
</tr>
<tr>
<td>MOPS</td>
<td>Millipore Sigma</td>
<td>Cat # M1254</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>·6H<sub>2</sub>O</td>
<td>Millipore Sigma</td>
<td>Cat # 1374248</td>
</tr>
<tr>
<td>ATP</td>
<td>Millipore Sigma</td>
<td>Cat # 3377</td>
</tr>
<tr>
<td>Creatine phosphate</td>
<td>ACROS Organics</td>
<td>Cat # AC226790250</td>
</tr>
<tr>
<td>Potassium hydroxide (KOH)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # P250</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Millipore Sigma</td>
<td>Cat # S0389</td>
</tr>
<tr>
<td>Leupeptin</td>
<td>Tocris</td>
<td>Cat # 1167</td>
</tr>
<tr>
<td>Pepistatin A</td>
<td>Tocris</td>
<td>Cat # 1190</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>Millipore Sigma</td>
<td>Cat # 3860-OP</td>
</tr>
<tr>
<td>Phenylmethylsulfonyl (PMSF)</td>
<td>Tocris</td>
<td>Cat # 4486</td>
</tr>
<tr>
<td>E-64d</td>
<td>Tocris</td>
<td>Cat # 4545</td>
</tr>
<tr>
<td>NaN<sub>3</sub> (sodium azide)</td>
<td>Millipore Sigma</td>
<td>Cat # 247-852-1</td>
</tr>
<tr>
<td>Urea</td>
<td>Millipore Sigma</td>
<td>Cat # U5128</td>
</tr>
<tr>
<td>Thiourea</td>
<td>Millipore Sigma</td>
<td>Cat # T8656</td>
</tr>
<tr>
<td>CHAPS detergent</td>
<td>Millipore Sigma</td>
<td>Cat # CHAPS-RO</td>
</tr>
<tr>
<td>EDTA (500 mM solution)</td>
<td>Millipore Sigma</td>
<td>Cat # 324504</td>
</tr>
<tr>
<td>Halt™ Phosphatase Inhibitor</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # 78420</td>
</tr>
<tr>
<td>Protease Inhibitor Cocktail</td>
<td>Millipore Sigma</td>
<td>Cat # P8340</td>
</tr>
<tr>
<td>Tributylphosphine (TBP, 200 mM)</td>
<td>Bio-Rad</td>
<td>Cat # 163–2101</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>CUSO-1 hiPSC line</td>
<td>Derived by our lab; see <a href="#bib4">Chi et al., (2018)</a>
</td>
<td>CUSO-1 hiPSC line. We recommend deriving hiPSC-CMs from passages 10–40.</td>
</tr>
<tr>
<td>CUSO-2 hiPSC line</td>
<td>Derived by our lab; see <a href="#bib4">Chi et al., (2018)</a>
</td>
<td>CUSO-2 hiPSC line. We recommend deriving hiPSC-CMs from passages 10–40.</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Coverslips for patterned surfaces (22 mm squares)</td>
<td>VWR</td>
<td>Cat # 48376-049, <a href="https://us.vwr.com/store/product/4646038/vwr-micro-slides-and-coverslips-plastic">https://us.vwr.com/store/product/4646038/vwr-micro-slides-and-coverslips-plastic</a>
</td>
</tr>
<tr>
<td>Lapping paper for patterning coverslips</td>
<td>Norton Abrasives</td>
<td>Cat # L12F3, 20 micron, <a href="https://www.mcmaster.com/4837A111/">https://www.mcmaster.com/4837A111/</a>
</td>
</tr>
<tr>
<td>Sonicator</td>
<td>Thermo Fisher Scientific</td>
<td>Model FB120 with CL-18 Probe</td>
</tr>
<tr>
<td>Bright-Line™ Hematocytometer</td>
<td>Millipore Sigma</td>
<td>Cat # Z359629</td>
</tr>
<tr>
<td>6 well plates</td>
<td>Greiner Bio-One</td>
<td>Cat # 657160</td>
</tr>
<tr>
<td>24 well plates</td>
<td>Corning</td>
<td>Cat # 353935</td>
</tr>
<tr>
<td>10 cm dish</td>
<td>Corning</td>
<td>Cat # 353003</td>
</tr>
<tr>
<td>0.2 μm filter</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # FB12566504</td>
</tr>
<tr>
<td>0.45 μm filter</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # FB12566503</td>
</tr>
<tr>
<td>Tissue-Tearor Homogenizer</td>
<td>Biospec</td>
<td>Model # 985370</td>
</tr>
<tr>
<td>Force probes for myofibrils</td>
<td>Self-built; see Woulfe et al. (<a href="#bib12">Woulfe et al., 2019</a>)</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Preparation of 50<b>×</b> fatty acid solution and 50<b>×</b> galactose solution</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This procedure will describe how to prepare stocks of BSA-conjugated fatty acids and galactose solutions for later preparation of maturation medium. It is essential that fatty acids are conjugated to BSA prior to their use in maturation medium, as otherwise fatty acids will form insoluble droplets or globules which cells will largely be unable to absorb. See <a href="#tbl1">Table 1</a> for the recipe for the recipe for 50X galactose solution, and <a href="#tbl2">Table 2</a> for the recipe for 50X fatty acid solution.</p>
<ol>
<li>Prepare fatty acid stocks:
<ol type="a">
<li>Prepare a 250 mM palmitic acid solution by dissolving palmitic acid crystals in 100% ethanol, and vortexing until completely dissolved.</li>
<li>Prepare a 500 mM oleic acid solution by diluting oleic acid (oleic acid is liquid at room temperature: (∼22°C) in 100% ethanol.</li>
<li>Fatty acid stocks can be stored, paraffin wrapped, at -20°C for up to three months.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Prepare 10% BSA solution:
<ol type="a">
<li>In a 50 mL conical tube, weigh out 1–4 g of Fatty Acid Free BSA Fraction V (depending on the final volume of 50<b>×</b> solution desired).
<div class="note">
<span class="note-title">Note:</span> this can be scaled up to produce a larger volume of fatty acid solution, but we do not recommend creating more than 40mL of solution in a single 50 mL conical tube due to BSA frothing that may occur.</div>
</li>
<li>In the sterile hood, add glucose-free RPMI to create a 10% BSA solution: for example 10 mL RPMI to 1 g BSA, 20 mL to 2 g, etc.</li>
<li>Vigorously vortex solution until BSA is completely dissolved (<a href="#fig1">Figure 1</a>A). This may require several minutes.</li>
<li>Heat solution to 37°C in water bath for 15 min.</li>
</ol>
</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1130-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Conjugation of fatty acids with BSA</div>
<p>(A) Image of 10% fatty-acid free BSA/glucose-free RPMI solution, with BSA completely dissolved.</p>
<p>(B) Image of 10% BSA solution immediately after addition of oleic acid (palmitic acid has already been added and conjugated). Note that the mixture has become cloudy, indicating fatty acid precipitation.</p>
<p>(C) Image of the same fatty acid solution after 30 min incubation at 37°C. The solution has cleared, indicating that both palmitic and oleic acid have successfully conjugated to BSA. This solution is ready to be used for preparation of maturation medium (or aliquoted and frozen).</p>
</figcaption>
</figure>
<ol start="3">
<li>Add palmitic acid to BSA:
<ol type="a">
<li>Add 250 mM palmitic acid solution at a 1:100 dilution, such that the final concentration will be 2.5 mM.
<div class="note">
<span class="note-title">Note:</span> Add palmitic acid slowly while agitating solution, as this may reduce aggregation and precipitation of palmitic acid .</div>
</li>
<li>Vortex solution vigorously. It is likely that solution will appear somewhat cloudy with fine particles at this point.</li>
<li>Incubate solution for 30 min at 37°C, occasionally vortexing. Solution should shift from cloudy to clear during this process.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Add oleic acid solution:
<ol type="a">
<li>Add 500 mM oleic acid at a 1:100 dilution and vigorously vortex such that the final concentration will be 5 mM. Once again, solution will become somewhat turgid (<a href="#fig1">Figure 1</a>B).</li>
<li>Incubate solution for 30 min at 37°C, occasionally vortexing. Solution should again clear up during this process, as fatty acids conjugate onto BSA (<a href="#fig1">Figure 1</a>C).</li>
</ol>
</li>
<li>At this point, the 50<b>×</b> fatty acid solution is ready to use.</li>
<li>Prepare 50<b>×</b> galactose solution:
<ol type="a">
<li>Weigh out galactose, and resuspend in glucose-free RPMI to a concentration of 500 mM (for example, resuspend 4.51 g galactose in 50 mL RPMI).</li>
<li>Vortex solution vigorously, until galactose is totally dissolved.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point, both 50<b>×</b> galactose and 50<b>×</b> fatty acid solutions are ready to be used, and may be stored at −20°C for up to three months.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Using Goldbio BSA Fraction V with palmitic and oleic acid, we have always had success conjugating fatty acids to BSA at 37°C. However, if using a different source of BSA, different concentrations or different types of fatty acids, it is possible that a more rigorous conjugation protocol may required. Although this protocol describes specifically conjugating oleic and palamitic acid to BSA for maturation medium, other fatty acids can be used as well – for example, we have also successfully conjugated linoleic acid to BSA. However, depending on the particular fatty acids used, the protocol may need to be modified, in terms of incubation times, temperatures, etc.</div>
<h3 id="sec2.2">Preparation of maturation medium</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="7">
<li>See <a href="#tbl3">Table 3</a> for a complete recipe for maturation medium. Thaw out aliquots of 50<b>×</b> galactose solution, 50<b>×</b> fatty acid solution, and B-27™ supplement to room temperature (∼22°C).</li>
<li>Working in a sterile hood, add galactose solution, fatty acid solution, and B-27™ supplement to glucose-free RPMI medium, each to a 1:50 dilution.
<ol type="a">
<li>For example, if creating 250 mL maturation medium, add 5 mL 50<b>×</b> galactose solution, 5 mL 50<b>×</b> fatty acid solution, and 5 mL 50<b>×</b> B-27™ supplement each to 235 mL RPMI. This will produce a final concentration of 10 mM galactose, 50 μM palmitic acid, and 100 μM oleic acid.</li>
<li>Mix solution via repeated pipetting or via use of a stir bar until it is completely homogenized.</li>
</ol>
</li>
<li>Filter medium using a 0.22 μm filter, into a sterilized/autoclaved container.
<ol type="a">
<li>Maturation medium preparation is now complete. Medium will be good for 2–3 weeks if stored at 4°C. Medium may also be aliquoted into smaller vessels (under sterile conditions) to avoid repeated warming/chilling cycles.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> As fatty acid and galactose solutions are not prepared under fully sterile conditions, it is absolutely essential to filter maturation medium prior to storage or use.</div>
<h3 id="sec2.3">Prepare patterned coverslips</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 90 min on day 1; overnight incubation; 2 h on day 2</div>
<p>This protocol describes the process of preparing and sterilizing micropatterned coverslips for plating of hiPSC-CMs. This entire process is also documented in <a href="#mmc1">Methods video S1</a> of the supplement.</p>
<ol start="10">
<li>Cut 20 micron lapping paper into ∼2 cm squares.</li>
<li>Holding coverslip firmly, draw lapping paper square along half of coverslip 15–20 times. Move lapping paper unidirectionally, in as straight of a line as possible to ensure linear patterning (<a href="#fig2">Figures 2</a>A and 2B).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1130-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Process of patterning+ sterilizing coverslip</div>
<p>(A) Lapping paper being drawn across coverslip to create micropatterns. This is repeated 15–20 times.</p>
<p>(B) Closeup image of a patterned coverslip.</p>
<p>(C) Photograph of coverslips being sonicated in soapy water to clean.</p>
<p>(D) Image of patterned coverslip in a well of a 6 well plate after ethanol evaporation and UV treatment. This coverslip is sterilized and ready for fibronectin coating.</p>
<p>(E) Image of patterned coverslip under the microscope.</p>
</figcaption>
</figure>
</li>
<li>Repeat this process on the other half of the coverslip.
<ol type="a">
<li>To help ensure linear patterning, it may be useful to use a ruler or straight piece of tape to coordinate coverslip.</li>
<li>After patterning the coverslip, it may also be useful to bend up one corner of coverslip to indicate which side has patterning and allow for later manipulation (<a href="#fig2">Figure 2</a>B). Alternatively, one may consider making a small mark or scratch (such as an X) on the unpatterned side of the coverslip, again to make it easier to identify. This will help prevent mistakenly plating cells on the unpatterned side of the coverslip.</li>
</ol>
</li>
<li>Place coverslips in a 50 mL conical tube, and wash twice, with vigorous vortexing, using distilled water. Take care when decanting water between washes to avoid losing coverslips (it may be useful to use mesh or a strainer when decanting water).</li>
<li>Incubate coverslips with soapy water. Sonicate them (we use 70% strength, 5 s pulse on, 10 s pulse off, for 3 min total), keeping conical tube on ice to prevent overheating (<a href="#fig2">Figure 2</a>C).</li>
<li>Rinse coverslips repeatedly, with vigorous vortexing, with distilled water until soap is completely removed.</li>
<li>Incubate coverslips overnight (for 12–18 h) in 100% ethanol on rocker.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The remaining steps are to be conducted in a sterile hood.</div>
<ol start="17">
<li>Working in a sterile hood, use forceps to place coverslips in proper size cell culture dishes (we use 6 well plates for full sized coverslips, 24 well plates for quarter sized coverslips) (<a href="#fig2">Figures 2</a>D, <a href="#fig3">3</a>A, and 3B).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1130-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Plating of hiPSC-CMs on patterned surfaces</div>
<p>(A) Photograph of hiPSC-CMs being pipetted onto sterilized, fibronectin-coated patterned surfaces.</p>
<p>(B) Photograph of patterned surfaces with cell solution for initial cell attachment (prior to filling wells with medium).</p>
<p>(C) Microscope images of hiPSC-CMs progressively attaching to surfaces over the first 5 days post plating. In later pictures, cells are difficult to see as they elongate, but can be seen to contract (Methods <a href="#mmc2">video S2</a>).</p>
</figcaption>
</figure>
</li>
<li>Allow ethanol to evaporate from coverslip by leaving them uncovered in sterile hood (this usually takes about 15 min).</li>
<li>Incubate coverslips (again, uncovered) under UV light from the sterile hood for 15 min. Coverslips should now be sterile. Make sure that each patterned surface, as well as the remainder of the cell culture containers they are in, are completely dry before proceeding to the next step (<a href="#fig2">Figure 2</a>D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If one has concerns about whether coverslips have successfully been sterilized, incubate one of them, after sterilization, in antibiotic-free media, without cells, to confirm lack of growth of any microbes.</div>
<ol start="20">
<li>Prepare fibronectin solution by diluting fibronectin stock in DPBS, to final concentration of 12 μg/mL. Carefully pipette an equal volume of fibronectin solution onto each patterned surface. The volume chosen should be sufficient to adequately cover the surface, typically about 500 μL per large patterned surface, or 100–125 μL per small patterned surface.
<ol type="a">
<li>Incubate coverslips with fibronectin solution at room temperature (∼22°C) for at least one hour.</li>
<li>Aspirate all fibronectin solution from coverslips and wash once with DPBS.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Completely aspirate DPBS. If cell culture plates are wrapped in wrapping film, coverslips may now be stored at 4°C for up to one week. Prior to sterilization and fibronectin coating, patterned coverslips may also be stored for several months at 4°C in ultrapure water.</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1130-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. A short video demonstrating the construction of patterned surfaces, related to steps 10–12</div>
</figcaption>
</figure>
<h3 id="sec2.4">Plate cardiomyocytes on patterned surfaces</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h on day 1; subsequent cell culture can last for weeks to months as desired.</div>
<p>This protocol describes the process of plating hiPSC-CMs on patterned surfaces and long-term cell culture.</p>
<ol start="21">
<li>Warm up replating and cell dissociation medium to 37°C:
<ol type="a">
<li>We have used the following medium types for hiPSC-CM replating:
<ol type="i">
<li>RPMI-20 + Y-2: RPMI medium mixed with 20% (v/v) fetal bovine serum, supplemented with 5 μM Y-27632 (a ROCK inhibitor).</li>
<li>STEMdiff™ Cardiomyocyte Support Medium (Stem Cell Technologies).</li>
</ol>
</li>
<li>For cell dissociation, we use Accumax™ (Stem Cell Technologies).</li>
<li>Fibronectin-coated patterned surfaces in appropriate cell culture containers should also already be prepared and on hand.</li>
</ol>
</li>
<li>We recommend replating hiPSC-CMs between day 35 and 45 post induction. Cells should be plated on sterile 0.1% gelatin-coated dishes and grown to a high degree of confluency prior to replating, as detailed earlier in this protocol.</li>
<li>Wash cells once with room temperature (∼22°C) DPBS.</li>
<li>Add Accumax™ or other dissociation medium to cells (typically 2.5 mL for a 10 cm dish, 0.5 mL for a 6WPW).
<ol type="a">
<li>Incubate cells at 37°C for 5–9 min.</li>
<li>Observe cells during dissociation until cells begin to detach. You may agitate cell culture dishes and tap the bottom of the cell culture dish lightly to help promote cell detachment.</li>
</ol>
</li>
<li>Once cells have detached, halt dissociation by adding an equal volume of cell replating medium (RPMI-20 + 5 μM Y-27632 or STEMdiff™ Cardiomyocyte Support Medium) to dissociated cell mixture.
<ol type="a">
<li>Pipette medium up and down several times to help detach cells and create single cell suspension.</li>
<li>Cell detachment should be visually apparent but it may be helpful to confirm by observing cell culture dishes under a microscope.</li>
<li>Pipette cells into a 10 or 50 mL conical tube (depending on volume of cell solution). It may be beneficial to wash plates with additional medium to ensure all cells are transferred.</li>
</ol>
</li>
<li>Count cells using hematocytometer, and calculate cell concentration and total yield.</li>
<li>Carefully aspirate medium from cell pellet, and resuspend in replating medium, to density of 500,000 to 1,000,000 cells/mL. Gently pipette cells up and down repeatedly to further homogenize cell suspension.</li>
<li>Carefully add cell suspension onto patterned surfaces, dropwise. Try to slowly cover the entire surface with cell solution (<a href="#fig3">Figure 3</a>A).
<ol type="a">
<li>For large surfaces (6WP), add 300k–500 k cells/surface. For small (24WP) surfaces, add 30k–60 k cells per surface.
<ol type="i">
<li>This plating density will generally yield a confluent hiPSC-CM monolayer. Lower densities may be preferable for microscopy of isolated cells.</li>
</ol>
</li>
<li>If cell suspension added doesn’t adequately cover patterned surface, slowly add additional media across surface (again, dropwise) but do so very slowly to avoid causing cell solution to run off of surface.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not add excess media to patterned coverslips, or cells and solution will run off of coverslip into well. Be extremely careful handling cell culture dishes containing coverslips. If this occurs, it may be possible to salvage the well by pipetting up excess solution, then using vacuum suction to ∗thoroughly∗ dry the area around the coverslip, then adding cell solution back onto coverslip. Plating cells at a high density reduces the volume of solution which must be added per coverslip, thus reducing the likelihood of runoff.</div>
<ol start="29">
<li>Allow cell solution to sit on patterned surface for at least 30 min before adding additional medium, to allow hiPSC-CMs to attach to the coverslip (<a href="#fig3">Figure 3</a>B).
<ol type="a">
<li>After this, bring medium volume up to typical amount using replating medium (aka 2 mL for 6WPW, 0.5 mL for 24WPW).</li>
<li>Over the next several hours, cells should begin to elongate as they spread out along surface (<a href="#fig3">Figure 3</a>C). As cells attach and elongate, they will become more difficult to see.</li>
<li>When plated at a high density, hiPSC-CMs will generally form a monolayer exhibiting synchronous contraction along the direction of patterning (<a href="#mmc2">Methods video S2</a>).</li>
</ol>
</li>
<li>48 h after replating cells, switch cells to maturation medium. Cells may be cultured in maturation medium indefinitely depending on desired cell age at time of experimentation.
<ol type="a">
<li>When changing medium, remove old medium via suction from an area of the cell culture well outside of the patterned surface to avoid scratching off cells. As with any cell culture, work quickly when changing medium to avoid drying out cells.</li>
</ol>
</li>
<li>When cells have been cultured for the desired duration, they may be fixed and stained (see below), harvested for RNA or protein studies, or other protocols in a similar fashion as hiPSC-CMs cultured via other approaches.
<ol type="a">
<li>When lysing cells, it will be necessary to use vigorous scraping to detach cells from surfaces.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We typically culture cells on patterned surfaces until day 55–75 post induction: this duration of culture is typically sufficient to produce elongated cardiomyocytes with relatively organized sarcomeres and low BNP expression. We have cultured cells up to day 120 post induction, but observed relatively few changes in terms of gene expression or myofibril mechanics compared to day 75 (unpublished observations).</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1130-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. A synchronously contracting hiPSC-CM monolayer, cultured on a patterned surface in maturation medium, related to steps 30 and 31</div>
</figcaption>
</figure>
<h3 id="sec2.5">Fix and stain hiPSC-CMs plated on patterned surfaces for BNP</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days: approximately 2 h of work on each day.</div>
<p>This protocol describes the process of fixing hiPSC-CMs and staining them for the hypertrophic marker BNP. Cells may then be stained for other markers and imaged. The protocol for staining for BNP is immunofluorescence-based and is adapted from a previous protocol (<a href="#bib3">Carlson et al., 2013</a>).</p>
<ol start="32">
<li>Prepare permeabilization buffer: 3% (w/v) milk and 0.1% (v/v) Triton-X 100 in DPBS. Use this solution within 5 days of preparation and store at 4°C. This step can be delayed if storing fixed cells before staining.</li>
</ol>
<ol start="33">
<li>Fix cells:
<ol type="a">
<li>Wash cells to be fixed once with DPBS.</li>
<li>Fix cells in 4% paraformaldehyde/PBS for 15 min at room temperature (∼22°C).
<div class="note">
<span class="note-title">Note:</span> Although lab-prepared paraformaldehyde solution is acceptable for staining cells, we have had best results when using commercially available, electron microscopy-grade paraformaldehyde (see <a href="#key-resources-table">key resources table</a>). For other stains, other fixatives may also be used, but methanol-acetone may change the color of (bleach) the plastic of the coverslips.</div>
</li>
<li>Wash cells once with DPBS.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> After fixing cells and removing paraformaldehyde solution, fixed cells may be immersed in DPBS and stored in wrapping film sealed containers at 4°C for 1–2 weeks.</div>
</li>
</ol>
</li>
</ol>
<ol start="34">
<li>Permeabilize cells: incubate patterned surfaces (coverslips) with permeabilization buffer at room temperature (∼22°C) for 30 min.</li>
<li>Incubate cells with primary antibody: dilute BNP antibody (in our case, Abcam ab13115, 1:200 dilution) in permeabilization buffer, and add over coverslips.
<ol type="a">
<li>Incubate coverslips with primary antibody solution overnight (12–18 h) at 4°C. We recommend performing this incubation in a wrapping film or Saran wrap-sealed container, to prevent evaporation and minimize volume of antibody solution required.</li>
</ol>
</li>
<li>The following day: wash cells;
<ol type="a">
<li>Wash coverslips 3 times with DPBS supplemented with 0.05% (v/v) Triton-X 100. Incubate cells at 5 min at room temperature (∼22°C).</li>
</ol>
</li>
<li>Add secondary antibody solution, diluted in permeabilization buffer.
<ol type="a">
<li>We typically use Alexa Fluor 488 secondary goat anti-mouse antibody, diluted 1:400, for BNP staining, but other antibody types should work as well.</li>
<li>Incubate coverslips in secondary antibody solution at room temperature (∼22°C) for 1 h.</li>
</ol>
</li>
<li>Wash cells:
<ol type="a">
<li>Wash coverslips 3 times with DPBS supplemented with 0.05% (v/v) Triton-X 100. Incubate cells at 5 min at room temperature (∼22°C).</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Counterstain cells for other markers as desired. While we have found that optimal BNP staining can be achieved using milk-based permeabilization buffer as indicated, most other markers, including sarcomeric proteins, stain well using more typical methods, such as blocking and antibody dilution solutions based on horse or goat serum in DPBS, or using DAKO reagents. These stains can be performed after staining for BNP, without any noticeable loss of BNP signal.</div>
<ol start="39">
<li>Stain nuclei and/or mount coverslips:
<div class="note">
<span class="note-title">Note:</span> If using mounting medium containing DAPI, it is unnecessary to conduct nuclear staining.</div>
<ol type="a">
<li>If staining cells directly, dilute a nuclear dye such as DAPI or Hoescht 33342 1:5000 in DPBS, and incubate coverslips with this solution for 5 min at room temperature (∼22°C).</li>
<li>Wash cells twice with DPBS.</li>
<li>Mount coverslips: on a glass slide, place a small drop of mounting medium such as Vectashield® or Prolong™ Gold, and place coverslip with the patterned side facing down into mounting medium.
<div class="note">
<span class="note-title">Note:</span> Gently press onto coverslip to remove any bubbles. Remove excess mounting reagent by vacuuming or pipetting.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make absolutely sure that the cell covered/patterned side of the surface is in mounting medium here (facing down) as otherwise cells will rapidly dry out and signal will be lost.</div>
</li>
<li>Seal coverslip to slide using nail gloss. Cells are now ready to image.</li>
<li>If staining is successful, BNP is primarily observed to be localized proximal to cell nuclei. Please note that maturation medium suppresses BNP expression, so relatively few BNP positive cells will be observed at baseline, in the absence of hypertrophic agonists (<a href="#fig4">Figure 4</a>C). Some degree of autofluorescence will likely be observed on patterned surfaces, especially in the green channel.</li>
</ol>
</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1130-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Morphological changes induced in hiPSC-CMs with maturation-inducing culture</div>
<p>(A) Cellmask™ Orange-stained (CMO) hiPSC-CMs cultured in standard glucose-based RPMI (GLUC), or in maturation medium on patterned surfaces (MPAT).</p>
<p>(B) α-actinin staining in hiPSC-CMs. The patterned hiPSC-CMs demonstrate significant elongation, with a greater proportion of α-actinin localized perpendicular to the long axis of the cells.</p>
<p>(C) BNP/pro-NT-BNP staining in hiPSC-CMs. Culture with maturation medium and on patterned surfaces suppresses BNP expression.</p>
</figcaption>
</figure>
<h3 id="sec2.6">Isolate myofibrils from hiPSC-CMs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h to isolate myofibrils, several additional hours if measuring myofibril mechanics. Extensive solution preparation required before beginning.</div>
<div class="note">
<span class="note-title">Note:</span> This protocol describes ONLY how to isolate myofibrils from hiPSC-CMs. These myofibrils can then be used for protein studies, or for measuring myofibril mechanics. The protocols and equipment needed to measure mechanics are quite sophisticated and specialized and therefore will not be described here – for further information on these techniques, please refer to Woulfe et al. (<a href="#bib12">Woulfe et al., 2019</a>). For examples of isolated myofibrils from cells cultured in maturation medium and on patterned surfaces (MPAT) and an activation/relaxation trace from a myofibril, see <a href="#fig5">Figure 5</a>. Preparing bath and myofibril lysis buffer takes some time. Prepare these before beginning this procedure.</div>
<ol start="40">
<li>For characterizing myofibril mechanics, we recommend using 2 full size (22 mm by 22 mm) patterned surfaces with a confluent layer of hiPSC-CMs (typically 350k–500 k cells per patterned surface) per experiment.</li>
<li>Prepare myofibril lysis buffer as described in <a href="#tbl4">Tables 4</a> and <a href="#tbl5">5</a>. Cool this buffer on ice.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1130-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Myofibrils and a representative myofibril activation/relaxation trace</div>
<p>(A and B) (A) Myofibrils isolated from hiPSC-CMs cultured under MPAT conditions, and mounted between glass microtools in relaxing solution. Different sizes of stretchers, the microtool on the left, may be used, as seen in (A and B). When Ca<sup>2+</sup> is added to this solution, the myofibril will contract, pulling the force probe (the dark microtool) to the left. When Ca<sup>2+</sup> is removed from solution, the myofibril will relax. By tracking the deflection of the force probe, which is displayed on the y-axis, an activation/relaxation trace as show in (B), can be prepared. This trace was also derived from a myofibril from an MPAT hiPSC-CM. From such a trace, speed of activation and relaxation, as well as force generated by the myofibril, can be calculated. Please note that significant equipment and protocols which are not covered by this manuscript will be needed to fully conduct myofibril mechanics experiments.</p>
</figcaption>
</figure>
<table id="tbl4">
<caption>Table 4. Myofibril bath buffer (Stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (for 100 mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>EGTA (100 mM, pH 7)</td>
<td>10 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>K-prop (1 M)</td>
<td>54 mM</td>
<td>5.4 mL</td>
</tr>
<tr>
<td>Na<sub>2</sub>SO<sub>4</sub> (100 mM)</td>
<td>17.69 mM</td>
<td>17.69 mL</td>
</tr>
<tr>
<td>MOPS (1M)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1M)</td>
<td>6.08 mM</td>
<td>608 μL</td>
</tr>
<tr>
<td>ATP∗</td>
<td>6.70 mM</td>
<td>0.3396 g</td>
</tr>
<tr>
<td>Creatine phosphate∗</td>
<td>15.5 mM</td>
<td>0.327 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare, mix the first 5 ingredients, pH solution to 6.8, add final two starred components (ATP and creatine), and pH solution to 7, and filter with 0.45 μm filter. Store at −20°C for up to 1 year.</p>
<table id="tbl5">
<caption>Table 5. Myofibril lysis buffer (Stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (for 5 mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Myofibril bath</td>
<td>N/A</td>
<td>5 mL</td>
</tr>
<tr>
<td>Sucrose</td>
<td>0.584 M/ 20 % W:V</td>
<td>1 g</td>
</tr>
<tr>
<td>Leupeptin</td>
<td>10 μM</td>
<td>5 μL 10 mM solution</td>
</tr>
<tr>
<td>Pepistatin A</td>
<td>5 μM</td>
<td>12.5 μL 2 mM solution</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>1 mM</td>
<td>10 μL 500 mM solution</td>
</tr>
<tr>
<td>Phenylmethylsulfonyl (PMSF)</td>
<td>200 μM</td>
<td>10 μL 100 mM solution</td>
</tr>
<tr>
<td>E-64</td>
<td>10 μM</td>
<td>5 μL 10 mM solution</td>
</tr>
<tr>
<td>NaN<sub>3</sub> (sodium azide)</td>
<td>500 μM</td>
<td>2.5 μL 1 M solution</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare, mix all ingredients immediately before use. Store once ice until used to lyse cells. Do not store long term. In steps 46–47, to wash myofibrils, use myofibril lysis buffer with all components <b>except</b> sucrose.</p>
</li>
<li>Wash hiPSC-CMs once with DPBS. Remove PBS using vacuum.</li>
<li>Add chilled myofibril lysis buffer to each patterned surface to be lysed.
<ol type="a">
<li>Using a plastic scraper, lift cells from patterned surfaces. Vigorously scrape cells, passing scraper over patterned surface 20 or more times. Cells detaching from surface should be visible (lysis buffer will become increasingly cloudly).</li>
<li>Pipette cell lysate into 1.5 mL microfuge tube. Vortex solution vigorously.</li>
</ol>
</li>
<li>Incubate lysate on ice for 10 min, periodically vortexing.</li>
<li>Centrifuge lysate at 1500 <i>g</i> for 5 min, at 4°C.
<ol type="a">
<li>After centrifugation, a small myofibril-rich pellet should be visible.</li>
</ol>
</li>
<li>Remove supernatant, and wash pellet with fresh myofibril bath with protease inhibitors (same as in step 42, but without sucrose; see <a href="#tbl5">Table 5</a>).</li>
<li>Repeat steps 45 and 46 twice more, to remove any residual sucrose.</li>
<li>Homogenize lysate using a brief (∼5 s) pulse of a Tissue Tearor Homogenizer at 30–50% of maximum speed. Avoid introducing bubbles into solution. Under the microscope, elongated myofibrils should now be visible, along with cell debris. The lysate is now ready for use in mechanical studies.</li>
<li>If using myofibrils for protein studies, centrifuge solution at 1500 g for 15 min to pellet myofibrils. Remove relaxing solution, and resuspend myofibrils in isoelectric focusing buffer (IEF, see <a href="#tbl6">Table 6</a>). Lysate may be frozen at −80°C until use.
<table id="tbl6">
<caption>Table 6. Isoelectric focusing solution (IEF)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (for 5 mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Urea</td>
<td>8.07 M</td>
<td>2.425 g</td>
</tr>
<tr>
<td>Thiourea</td>
<td>2.5 M</td>
<td>0.95 g</td>
</tr>
<tr>
<td>CHAPS</td>
<td>4 % (W:V)</td>
<td>200 mg</td>
</tr>
<tr>
<td>EDTA</td>
<td>2 mM</td>
<td>20 μL 500 mM solution</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)∗</td>
<td>10 mM</td>
<td>100 μL 500 mM solution</td>
</tr>
<tr>
<td>Tributylphosphine (TBP)∗</td>
<td>2 mM</td>
<td>50 μL 200 mM solution</td>
</tr>
<tr>
<td>Sigma P8340 Protease Inhibitor Cocktail∗</td>
<td>1<b>×</b>
</td>
<td>50 μL 100 solution</td>
</tr>
<tr>
<td>Halt™ Phosphatase inhibitor∗</td>
<td>1<b>×</b>
</td>
<td>50 μL 100<b>×</b> solution</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare, mix first four ingredients and filter with 0.45 μm filter. Aliquot and store for up to 1 year at −20°C. Immediately before use, thaw aliquot of solution, vortex to clear out crystals and add final four starred ingredients. Usually only a small volume (1 mL or less) of IEF will be used for a single experiment, so scale down these ingredients accordingly. Keep IEF with inhibitors on ice until use, but do not store long term.</p>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>hiPSC-CM culture with patterned surfaces and using maturation medium will produce hiPSC-CMs which morphologically (<a href="#fig4">Figures 4</a>A and 4B) and functionally demonstrate an increased degree of maturity, compared to hiPSC-CMs cultured in standard glucose-based medium. Typically, this involves a greater degree of cell elongation and sarcomere organization (<a href="#fig4">Figures 4</a>A and 4B), reduced expression of the hypertrophic marker B-type natriuretic peptide (BNP, <a href="#fig4">Figure 4</a>C), and increased sarcomere organization, contractile force, and reduced cell area with prolonged culture (<a href="#bib6">Knight et al., 2021</a>), shifting these cells towards a more adult-like phenotype.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Limitations: Compared to standard culture, cell yields from patterned surfaces are relatively low (typically approximately 50% of that of standard culture, on a well per well basis). This may be due to the smaller area of patterned surfaces, cells not attaching to the entire area of surfaces, and that hiPSC-CMs cultured in fatty acid-based medium (such as maturation medium) essentially cease dividing (<a href="#bib9">Mills et al., 2017</a>). Accordingly, we recommend that experiments be planned accordingly, scaling up patterned surface culture as necessary to ensure that sufficient numbers of cells are attained. While the cells produced by this method will be much more mature than hiPSC-CMs derived via standard glucose-based culture, they do not fully resemble or recapitulate the function of adult cardiomyocytes.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Occasionally, we have noticed that cells attach poorly to patterned surfaces, resulting in low yields (step 28).</p>
<h3 id="sec5.2">Potential solution</h3>
<p>To some degree, this appears to depend on the batch of hiPSC-CMs used: it may be useful to repeat using a different batch of cells. Otherwise, consider increasing the amount of time hiPSC-CMs are incubated on surfaces before adding additional medium (step 30), as this may wash cells off of surfaces. Switching cells to maturation medium too quickly after replating also appears to stress cells and may cause cell detachment or death. In this case, it may be beneficial to culture cells for 1–2 days in RPMI-1640 supplemented with B-27™ (with glucose) prior to switching cells to maturation medium.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>When staining hiPSC-CMs for cardiac markers, many cells stain negative for these markers, or similarly, when culturing cells on patterned surfaces, cells on large areas of the surface do not beat (steps 30 and 31).</p>
<h3 id="sec5.4">Potential solution</h3>
<p>It is likely that insufficiently stringent selection for cardiomyocytes was used. If many nonmyocytes survive selection and are plated on patterned surfaces, they will attach to the surfaces and continue to divide, eventually outcompeting/replacing most cardiomyocytes. This is exacerbated by the tendency of maturation medium to induce cell cycle arrest. If this occurs, it will be necessary to conduct a new hiPSC-CM induction, and it may be beneficial to culture cells for more than 5 days in lactate medium, although this will eventually begin to kill cardiomyocytes as well.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>hiPSC-CMs become contaminated after plating on patterned surfaces (step 28, patterned surface sterilization detailed in step 16–19).</p>
<h3 id="sec5.6">Potential solution</h3>
<p>It is very likely that surfaces are being insufficiently well sterilized. Consider increasing duration of incubation in 100% ethanol, or duration of exposure to UV light. Confirm that ethanol is completely evaporated before UV light sterilization. Consider flipping patterned surfaces over during UV sterilization to sterilize bottom surface of surfaces (although we have never found this to be necessary).</p>
<h3 id="sec5.7">Problem 4</h3>
<p>hiPSC-CMs become contaminated after switching to maturation medium (step 30).</p>
<h3 id="sec5.8">Potential solution</h3>
<p>It is likely that contaminants are present in maturation medium. Filter media through sterile, 0.22 μm filter prior to use of media, and conduct filtration in a sterile hood. Consider double filtering media before use. If nothing else works, one may consider supplementing maturation medium with antibiotics, although this could potentially have deleterious effects on cardiomyocyte function (<a href="#bib1">Belus and White, 2001</a>).</p>
<h3 id="sec5.9">Problem 5</h3>
<p>Fatty acids precipitate out of 50<b>×</b> fatty acid solution during preparation (typically the precipitate will take the form of fine white dots or possibly crystals) (steps 3 and 4).</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Fatty acids are being insufficiently well conjugated to BSA. We would recommend initially increasing incubation time at 37°C. If this does not solve the problem, consider increasing the incubation temperature, up to 50°C. It was also be beneficial or necessary to sonicate the BSA/fatty acid mixture.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Belus, A. and White, E. (2001). Effects of antibiotics on the contractility and Ca2+ transients of rat cardiac myocytes. Eur. J. Pharmacol. <i>412</i>, 121-126. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref1">View at publisher</a></p>
<p id="bib2">Birket, M.J., Ribeiro, M.C., Kosmidis, G., Ward, D., Leitoguinho, A.R., van de Pol, V., Dambrot, C., Devalla, H.D., Davis, R.P., Mastroberardino, P.G., et al. (2015). Contractile defect caused by mutation in MYBPC3 revealed under conditions optimized for human PSC-cardiomyocyte function. Cell Rep. <i>13</i>, 733-745. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref2">View at publisher</a></p>
<p id="bib3">Carlson, C., Koonce, C., Aoyama, N., Einhorn, S., Fiene, S., Thompson, A., Swanson, B., Anson, B., and Kattman, S. (2013). Phenotypic screening with human iPS cell-derived cardiomyocytes: HTS-compatible assays for interrogating cardiac hypertrophy. J. Biomol. Screen. <i>18</i>, 1203-1211. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref3">View at publisher</a></p>
<p id="bib4">Chi, C., Leonard, A., Knight, W.E., Beussman, K.M., Zhao, Y., Cao, Y., Londono, P., Aune, E., Trembley, M.A., Small, E.M., et al. (2018). LAMP-2B regulates human cardiomyocyte function by mediating autophagosome-lysosome fusion. Proc. Natl. Acad. Sci. U S A <i>116</i>, 556-565. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref4">View at publisher</a></p>
<p id="bib5">Correia, C., Koshkin, A., Duarte, P., Hu, D., Teixeira, A., Domian, I., Serra, M., and Alves, P.M. (2017). Distinct carbon sources affect structural and functional maturation of cardiomyocytes derived from human pluripotent stem cells. Sci. Rep. <i>7</i>, 8590. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref5">View at publisher</a></p>
<p id="bib6">Knight, W.E., Cao, Y., Lin, Y.H., Chi, C., Bai, B., Sparagna, G.C., Zhao, Y., Du, Y., Londono, P., Reisz, J.A., et al. (2021). Maturation of pluripotent stem cell-derived cardiomyocytes enables modeling of human hypertrophic cardiomyopathy. Stem Cell Rep. <i>16</i>, 519-533. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref6">View at publisher</a></p>
<p id="bib7">Lian, X., Hsiao, C., Wilson, G., Zhu, K., Hazeltine, L.B., Azarin, S.M., Raval, K.K., Zhang, J., Kamp, T.J., and Palecek, S.P. (2012). Robust cardiomyocyte differentiation from human pluripotent stem cells via temporal modulation of canonical Wnt signaling. Proc. Natl. Acad. Sci. U S A <i>109</i>, E1848-E1857. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref7">View at publisher</a></p>
<p id="bib8">Lian, X., Zhang, J., Azarin, S.M., Zhu, K., Hazeltine, L.B., Bao, X., Hsiao, C., Kamp, T.J., and Palecek, S.P. (2013). Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/beta-catenin signaling under fully defined conditions. Nat. Protoc. <i>8</i>, 162-175. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref8">View at publisher</a></p>
<p id="bib9">Mills, R.J., Titmarsh, D.M., Koenig, X., Parker, B.L., Ryall, J.G., Quaife-Ryan, G.A., Voges, H.K., Hodson, M.P., Ferguson, C., Drowley, L., et al. (2017). Functional screening in human cardiac organoids reveals a metabolic mechanism for cardiomyocyte cell cycle arrest. Proc. Natl. Acad. Sci. U S A <i>114</i>, E8372-E8381. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref9">View at publisher</a></p>
<p id="bib10">Pioner, J.M., Racca, A.W., Klaiman, J.M., Yang, K.C., Guan, X., Pabon, L., Muskheli, V., Zaunbrecher, R., Macadangdang, J., Jeong, M.Y., et al. (2016). Isolation and mechanical measurements of myofibrils from human induced pluripotent stem cell-derived cardiomyocytes. Stem Cell Rep. <i>6</i>, 885-896. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref10">View at publisher</a></p>
<p id="bib11">Tohyama, S., Hattori, F., Sano, M., Hishiki, T., Nagahata, Y., Matsuura, T., Hashimoto, H., Suzuki, T., Yamashita, H., Satoh, Y., et al. (2013). Distinct metabolic flow enables large-scale purification of mouse and human pluripotent stem cell-derived cardiomyocytes. Cell Stem Cell <i>12</i>, 127-137. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref11">View at publisher</a></p>
<p id="bib12">Woulfe, K.C., Ferrara, C., Pioner, J.M., Mahaffey, J.H., Coppini, R., Scellini, B., Ferrantini, C., Piroddi, N., Tesi, C., Poggesi, C., et al. (2019). A novel method of isolating myofibrils from primary cardiomyocyte culture suitable for myofibril mechanical study. Front. Cardiovasc. Med. <i>6</i>, 12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref12">View at publisher</a></p>
<p id="bib13">Yang, X., Rodriguez, M., Pabon, L., Fischer, K.A., Reinecke, H., Regnier, M., Sniadecki, N.J., Ruohola-Baker, H., and Murry, C.E. (2014). Tri-iodo-l-thyronine promotes the maturation of human cardiomyocytes-derived from induced pluripotent stem cells. J. Mol. Cell. Cardiol. <i>72</i>, 296-304. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00618-3/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>For further information or to request reagents, please direct requests to Dr. Kunhua Song, <a href="mailto:kunhua.song@cuanschutz.edu">kunhua.song@cuanschutz.edu</a>.</p>
<h4>Materials availability</h4>
<p>The materials used for this study can be created from commercially available materials, as is indicated in <a href="#tbl1">Table 1</a>.</p>
<h4>Data and code availability</h4>
<p>No data sets or code were generated in the process of conducting this study.</p>
<h3>Acknowledgments</h3>
<p>WEK was supported by a postdoctoral fellowship from the <a href="https://doi.org/10.13039/100010175">University of Colorado</a> Consortium for Fibrosis Research &amp; Translation, <a href="https://doi.org/10.13039/100000002">NIH</a> Cardiology training grant T32HL007822, an <a href="https://doi.org/10.13039/100000968">American Heart Association</a> postdoctoral fellowship (19POST34380250), and an <a href="https://doi.org/10.13039/100000002">NIH</a>/CCTSI CO-Pilot Mentored Faculty Award. P.D. was a fellow of the Gates Summer Internship Program. K.S. was supported by funds from the <a href="https://doi.org/10.13039/100005508">Boettcher Foundation</a>, <a href="https://doi.org/10.13039/100000968">American Heart Association</a>, <a href="https://doi.org/10.13039/100007493">University of Colorado</a> Department of Medicine Outstanding Early Career Scholar Program, Gates Frontiers Fund, and <a href="https://doi.org/10.13039/100000002">NIH</a> HL133230. The authors would also like to thank Dr. Congwu Chi, Dr. Kathleen Woulfe, and Dr. Lori Walker for extremely helpful and detailed technical discussions and feedback while preparing this manuscript.</p>
<h3>Author contributions</h3>
<p>W.E.K. and K.S. conceived of this project and prepared the manuscript. W.E.K. conducted experiments. P.D. participated in method development and manuscript preparation. Y.C. assisted with method development, hiPSC-CM inductions, and replating cells onto patterned surfaces.</p>
<h3>Declaration of interests</h3>
<p>An application for a patent has been filed.</p>
</section>
</article>