<article>
<section>
<h1 id="header">Acquisition of murine splenic myeloid cells for protein and gene expression profiling by advanced flow cytometry and CITE-seq</h1>
<p><time datetime="2021-09-24">Published: September 24, 2021</time></p>
<p>Inga Rødahl,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a></sup> James Gotley,<sup><a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Stacey B. Andersen,<sup><a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Meihua Yu,<sup><a href="#aff2">2</a></sup> Ahmed M. Mehdi,<sup><a href="#aff2">2</a></sup> Angelika N. Christ,<sup><a href="#aff3">3</a></sup> Emma E. Hamilton-Williams,<sup><a href="#aff2">2</a></sup> Ian H. Frazer,<sup><a href="#aff2">2</a></sup> Samuel W. Lukowski,<sup><a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor2">**</a></sup> and Janin Chandra<sup><a href="#aff2">2</a>,<a href="#fn2">5</a>,<a href="#fn4">7</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Center for Infectious Medicine, Department of Medicine, Karolinska Institute, Karolinska University Hospital Huddinge, 141 86 Stockholm, Sweden</p>
<p id="aff2"><sup>2</sup>The University of Queensland Diamantina Institute, The University of Queensland, Woolloongabba, QLD 4102, Australia</p>
<p id="aff3"><sup>3</sup>The Institute for Molecular Bioscience, The University of Queensland, Woolloongabba, QLD 4067, Australia</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>These authors contributed equally</p>
<p id="fn3"><sup>6</sup>Technical contact</p>
<p id="fn4"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:s.andersen2@uq.edu.au">s.andersen2@uq.edu.au</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:s.lukowski@imb.uq.edu.au">s.lukowski@imb.uq.edu.au</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:j.chandra@uq.edu.au">j.chandra@uq.edu.au</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100842">10.1016/j.xpro.2021.100842</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we outline detailed protocols to isolate and profile murine splenic dendritic cells (DCs) through advanced flow cytometry of the myeloid compartment and single-cell transcriptomic profiling with integrated cell surface protein expression through CITE-seq. This protocol provides a general transferrable road map for different tissues and species.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref14">Lukowski et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol to obtain integrated single-cell gene and protein expression data</li>
<li>Optimized flow cytometry panel for confident delineation of six main myeloid lineages</li>
<li>Gating strategy identifies large cell state heterogeneity within each lineage</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1032-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol consists of two main workflows; (i) to acquire advanced flow cytometry data, and (ii) to acquire single cell transcriptomic data with integrated epitope detection (CITE-seq). For both workflows, the preparation of cell suspensions is a starting point. The workflow is exemplified using murine splenocytes with C57BL/6 background to delineate and isolate myeloid cells and dendritic cells (DCs). Experiments were carried out using female mice aged 6–12 weeks. All animal procedures and experiments were performed in compliance with the ethical guidelines of the National Health and Medical Research Council of Australia, with approval from the IMVS Animal Ethics Committee and the University of Queensland Animal Ethics Committee (UQDI/452/1).</p>
<p>Other tissue types (e.g., lymph node, skin), genetic backgrounds and species, as well as human clinical samples, may require species- or tissue-specific optimization of dissociation and choice of antibodies. A protocol that outlines the preparation of single cells from other murine tissue can be found here (<a href="#bib12">Liu et al., 2020</a>).</p>
<p>High-dimensional flow cytometry assays require expert knowledge that lie beyond the scope of this protocol, and principles for design and execution of advanced flow cytometry have been discussed elsewhere (<a href="#bib5">Cossarizza et al., 2019</a>; <a href="#bib6">Dipiazza et al., 2019</a>; <a href="#bib8">Kalina, 2020</a>; <a href="#bib24">Liechti et al., 2021</a>; <a href="#bib15">Maciorowski et al., 2017</a>; <a href="#bib18">Mair and Tyznik, 2019</a>; <a href="#bib22">Wang and Hoffman, 2017</a>).</p>
<p>CITE-seq is a rather young technology and resources to optimized protocols currently remain sparse. Useful resources regarding determining the optimal concentration of oligo-tagged antibodies can be found here (<a href="#bib17">Mair et al., 2020</a>; <a href="#bib3">Buus et al., 2021</a>).</p>
<h3 id="sec1.1">Specific considerations</h3>
<ol>
<li>Due to differences in antibody batches and flow cytometer hardware components, the flow cytometry antibody panel presented here can only serve as guide, and should be optimized at each user lab. Re-titration of all antibodies is highly recommended to attain the best stain index for each antibody.</li>
<li>To allow confident discrimination of positive and negative cell populations in the flow cytometry workflow, we strongly encourage preparation of fluorescence minus one (FMO) controls.</li>
<li>Ensure antibodies used for cell sorting bind to different epitopes than CITE-seq oligo-tagged antibodies.</li>
<li>To validate and reduce background from oligo-tagged antibodies, we highly recommend titrating oligo-tagged antibodies using a dT-oligonucleotide tagged fluorescence dye and flow cytometry to determine optimal concentration.</li>
<li>Determine a target cell number before performing the CITE-seq protocol, and perform a mock sorting and antibody incubation, to inform yourself of required numbers of animals or samples, estimation of time from tissue collection to library preparation and protocol-associated cell loss.</li>
<li>Decide on a sequencing depth that is appropriate for your aim. The minimum recommendation from 10<b>×</b> Genomics is &gt;20,000 reads per cell for gene expression libraries and 5,000 reads per cell for barcoded antibody libraries.</li>
</ol>
<h3 id="sec1.2">Preparation of solutions</h3>
<div class="note">
<span class="note-title">Note:</span> Recipes for all required solutions are outlined under <a href="#materials-and-equipment">materials and equipment</a>
</div>
<ol start="7">
<li>Prepare buffers required for preparation of single cell suspensions: PBS, digestion buffer, ACK lysis buffer.</li>
<li>Prepare solutions for flow cytometry: FACS buffer, compensation bead solution, Fc-block/FVS solution, antibody solution.</li>
<li>Prepare solutions for 10<b>×</b> Genomics Chromium workflow as described in the appropriate kit protocol or on the 10<b>×</b> Genomics website.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-mouse CD11c PE-Dazzle (dilution 1:400)</td>
<td>BioLegend</td>
<td>Cat#117348; RRID: AB_2563655; Clone: N418</td>
</tr>
<tr>
<td>Anti-mouse CD11c PE-Cy7 (dilution 1:100)</td>
<td>BD Pharmingen</td>
<td>Cat#558079; RRID: AB_647251; Clone: HL3</td>
</tr>
<tr>
<td>Anti-mouse CD16/CD32 (Fc block) (dilution 1:100)</td>
<td>BioLegend</td>
<td>Cat# 101301; RRID: AB_312800; Clone: 93</td>
</tr>
<tr>
<td>Anti-mouse I-A/I-E APC-Cy7 (dilution 1:400)</td>
<td>BioLegend</td>
<td>Cat#107628; RRID: AB_2069377; Clone: M5/114.15.2</td>
</tr>
<tr>
<td>Anti-mouse TCRb Biotin (dilution 1:200)</td>
<td>BioLegend</td>
<td>Cat#109204; RRID: AB_313427; Clone: H57-597</td>
</tr>
<tr>
<td>Anti-mouse CD19 Biotin (dilution 1:200)</td>
<td>BioLegend</td>
<td>Cat#115504; RRID: AB_313639; Clone: 6D5</td>
</tr>
<tr>
<td>Anti-mouse CD45 BUV615 (dilution 1:1600)</td>
<td>BD Biosciences</td>
<td>Cat#751170; RRID: AB_2875194; Clone: 30-F11</td>
</tr>
<tr>
<td>Anti-mouse CD3 BV750 (dilution 1:100)</td>
<td>BioLegend</td>
<td>Cat# 100225; RRID: AB_10900444; Clone: 17A2</td>
</tr>
<tr>
<td>Anti-mouse CD19 BV570 (dilution 1:200)</td>
<td>BioLegend</td>
<td>Cat# 115535; RRID: AB_10933260; Clone: B4</td>
</tr>
<tr>
<td>Anti-mouse Ly6G BUV737 (dilution 1:100)</td>
<td>BD Biosciences</td>
<td>Cat# 741813; RRID: AB_2871151; Clone: 1A8</td>
</tr>
<tr>
<td>Anti-mouse PDCA1 APC (dilution 1:800)</td>
<td>BioLegend</td>
<td>Cat# 127015; RRID: AB_1967101; Clone:927</td>
</tr>
<tr>
<td>Anti-mouse CD11b APC-R700 (dilution 1:800)</td>
<td>BD Biosciences</td>
<td>Cat# 564985; RRID: AB_2739033; Clone: M1/70</td>
</tr>
<tr>
<td>Anti-mouse F4/80 PE-Cy5 (dilution 1:800)</td>
<td>BioLegend</td>
<td>Cat#123111; RRID: AB_893494; Clone: BM8</td>
</tr>
<tr>
<td>Anti-mouse Ly6C FITC (dilution 1:800)</td>
<td>BioLegend</td>
<td>Cat# 128005; RRID: AB_1186134; Clone: HK1.4</td>
</tr>
<tr>
<td>Anti-mouse I-A/I-E BV711 (dilution 1:800)</td>
<td>BioLegend</td>
<td>Cat# 107643; RRID: AB_2565976; Clone: M5/114.15.2</td>
</tr>
<tr>
<td>Anti-mouse CD11c BV421 (dilution 1:200)</td>
<td>BioLegend</td>
<td>Cat# 117329; RRID: AB_10897814; Clone: N418</td>
</tr>
<tr>
<td>Anti-mouse CD24 BV605 (dilution 1:100)</td>
<td>BioLegend</td>
<td>Cat# 101827; RRID: AB_2563464; Clone: M1/69</td>
</tr>
<tr>
<td>Anti-mouse XCR1 BV650 (dilution 1:100)</td>
<td>BioLegend</td>
<td>Cat# 148220; RRID: AB_2566410; Clone: ZET</td>
</tr>
<tr>
<td>Anti-mouse CD172A PerCP-eFluor710 (dilution 1:200)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 46-1721-82; RRID: AB_10804639; Clone: P84</td>
</tr>
<tr>
<td>Anti-mouse CD4 BUV496 (dilution 1:200)</td>
<td>BD Biosciences</td>
<td>Cat# 612952; RRID: AB_2813886; Clone: GK1.5</td>
</tr>
<tr>
<td>Anti-mouse CD326 (Epcam) BUV394 (dilution 1:400)</td>
<td>BD Biosciences</td>
<td>Cat# 740281; RRID: AB_2740020; Clone:G8.8</td>
</tr>
<tr>
<td>Anti-mouse CD11c (Totalseq-A0106, GTTATGGACGCTTGC)</td>
<td>BioLegend</td>
<td>Cat#: 117355; RRID: AB_2750352; Clone: N418</td>
</tr>
<tr>
<td>Anti-mouse/ human CD11b (Totalseq-A0014, TGAAGGCTCATTTGT)</td>
<td>BioLegend</td>
<td>Cat#: 101265; RRID: AB_2734152; Clone: M1/70</td>
</tr>
<tr>
<td>Anti-mouse CD172a (Totalseq-A0422, GATTCCCTTGTAGCA)</td>
<td>BioLegend</td>
<td>Cat#: 144033; RRID: AB_2800670; Clone: P84</td>
</tr>
<tr>
<td>Anti-mouse CD370 (Totalseq-A0556, AACTCAGTTGTGCCG)</td>
<td>BioLegend</td>
<td>Custom-built; Part #: 96461; Clone: 7H11</td>
</tr>
<tr>
<td>Anti-mouse CD4 (Totalseq-A001, AACAAGACCCTTGAG)</td>
<td>BioLegend</td>
<td>Cat#: 100569; RRID: AB_2749956; Clone: RM4-5</td>
</tr>
<tr>
<td>Anti-mouse CD8a (Totalseq-A002, TACCCGTAATAGCGT)</td>
<td>BioLegend</td>
<td>Cat#: 100773; RRID: AB_2734151; Clone: 53-6.7</td>
</tr>
<tr>
<td>Anti-mouse F4/80 (Totalseq-A0114, TTAACTTCAGCCCGT)</td>
<td>BioLegend</td>
<td>Cat#: 123153; RRID: AB_2749986; Clone: BM8</td>
</tr>
<tr>
<td>Anti-mouse Siglec H (Totalseq-A0119, CCGCACCTACATTAG)</td>
<td>BioLegend</td>
<td>Cat#: 129615; RRID: AB_2750537; Clone: 551</td>
</tr>
<tr>
<td>Anti-mouse, rat XCR1 (Totalseq-A0568, TCCATTACCCACGTT)</td>
<td>BioLegend</td>
<td>Cat#: 148227; RRID: AB_2783120; Clone: ZET</td>
</tr>
<tr>
<td>Anti-mouse CD24 (Totalseq-A0212, TATATCTTTGCCGCA)</td>
<td>BioLegend</td>
<td>Cat#: 101841; RRID: AB_2750380; Clone: M1/69</td>
</tr>
<tr>
<td>Anti-mouse CD117 (Totalseq-A0012, TGCATGTCATCGGTG)</td>
<td>BioLegend</td>
<td>Cat#: 105843; RRID: AB_2749960; Clone: 2B8</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Collagenase D</td>
<td>Roche</td>
<td>11088858001</td>
</tr>
<tr>
<td>DNase</td>
<td>Roche</td>
<td>11284932001</td>
</tr>
<tr>
<td>Reagents according to Chromium protocol</td>
<td><a href="https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf">https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf</a></td>
<td>Various</td>
</tr>
<tr>
<td>Reagents according to CITE-seq protocol</td>
<td>
<a href="https://www.biolegend.com/en-us/protocols/totalseq-a-antibodies-and-cell-hashing-with-10x-single-cell-3-reagent-kit-v3-3-1-protocol">Protocol - TotalSeq™-A Antibodies and Cell Hashing with 10x Single Cell 3' Reagent Kit v3 3.1 Protocol (BioLegend.com</a>)</td>
<td>Various</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>EasySep™ Mouse Streptavidin RapidSpheres™ Isolation Kit</td>
<td>STEMCELL<sup>TM</sup> Technologies</td>
<td>Cat#19860</td>
</tr>
<tr>
<td>LIVE/DEAD™ Fixable Aqua Dead Cell Stain Kit</td>
<td>Invitrogen</td>
<td>Cat#L34965</td>
</tr>
<tr>
<td>Fixable Viability Stain (FVS) 440UV (dilution 1:250)</td>
<td>BD Biosciences</td>
<td>Cat#566332</td>
</tr>
<tr>
<td>Brilliant Stain Buffer Plus</td>
<td>BD Biosciences</td>
<td>Cat#566385</td>
</tr>
<tr>
<td>Cell Staining Buffer</td>
<td>BioLegend</td>
<td>Cat#420201</td>
</tr>
<tr>
<td>Chromium Next GEM Single Cell 3ʹ Kit v3.1</td>
<td>10<b>×</b> Genomics</td>
<td>PN-1000268 or PN-1000269</td>
</tr>
<tr>
<td>Chromium Next GEM Chip G Single Cell Kit</td>
<td>10<b>×</b> Genomics</td>
<td>PN-1000120 or PN-1000127</td>
</tr>
<tr>
<td>Dual Index Kit TT Set A</td>
<td>10<b>×</b> Genomics</td>
<td>PN-1000215</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Murine CITE-seq data</td>
<td>(<a href="#bib14">Lukowski et al., 2021</a>)</td>
<td>NCBI GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149544">GSE149544</a>
</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mus musculus C57BL/6J-Tg(HLA-A2/H2-K)6Scr/HsdArc</td>
<td>Animal Resources Center <a href="http://www.arc.wa.gov.au/">www.arc.wa.gov.au</a>
</td>
<td>Product code: A2KB</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>AF647-dT [5'-5Alex647N-T<sub>30</sub>-3’]</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>Oligonucleotides according to CITE-seq protocol</td>
<td>BioLegend: <a href="https://www.biolegend.com/en-us/protocols/totalseq-a-antibodies-and-cell-hashing-with-10x-single-cell-3-reagent-kit-v3-3-1-protocol">https://www.BioLegend.com/en-us/protocols/totalseq-a-antibodies-and-cell-hashing-with-10x-single-cell-3-reagent-kit-v3-3-1-protocol</a>
</td>
<td>Various</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Kaluza</td>
<td>Beckman Coulter</td>
<td>
<a href="https://www.beckman.com.au/flow-cytometry/software/kaluza">https://www.beckman.com.au/flow-cytometry/software/kaluza</a><u>;</u> RRID:SCR_016182;</td>
</tr>
<tr>
<td>bcl2fastq</td>
<td>Illumina</td>
<td>v2.20</td>
</tr>
<tr>
<td>Cellranger</td>
<td>10<b>×</b> Genomics</td>
<td>v6.0</td>
</tr>
<tr>
<td>Seurat</td>
<td><a href="https://satijalab.org/seurat/">https://satijalab.org/seurat/</a></td>
<td>v3/v4</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Disposable transfer pipette</td>
<td>Copan</td>
<td>Cat#201CS01</td>
</tr>
<tr>
<td>FACS tube: Falcon® Round-Bottom Tubes with Cell Strainer Cap, 5 mL</td>
<td>Corning</td>
<td>Cat#352235</td>
</tr>
<tr>
<td>1.5 mL Eppendorf Tubes® 3810<b>×</b>
</td>
<td>Eppendorf</td>
<td>Cat#0030125150</td>
</tr>
<tr>
<td>50 mL Falcon Tube</td>
<td>Corning</td>
<td>Cat#430290</td>
</tr>
<tr>
<td>BD 300800 Microlance Hypodermic Needle 23G <b>×</b> 1" Blue</td>
<td>GP Supplies</td>
<td>Cat#300800</td>
</tr>
<tr>
<td>BD 1 mL insulin syringe</td>
<td>BD</td>
<td>Cat#326719</td>
</tr>
<tr>
<td>Falcon® 70 μm Cell Strainer</td>
<td>Corning</td>
<td>Cat#352350</td>
</tr>
<tr>
<td>Corning® Costar® Stripette® serological pipettes</td>
<td>Corning</td>
<td>SKU: CLS4492-200EA</td>
</tr>
<tr>
<td>Bio-Rad TC20 Cell counter</td>
<td>Bio-Rad</td>
<td>Cat#1450102</td>
</tr>
<tr>
<td>Eppendorf 5810R centrifuge or centrifuge capable of holding FACS tube inserts</td>
<td>Eppendorf</td>
<td>5810000084</td>
</tr>
<tr>
<td>Pipettes and and pipette tips capable of 0.25 μL–1000 μL volumes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>EasySep<sup>TM</sup> Magnet</td>
<td>STEMCELL<sup>TM</sup> Technologies</td>
<td>Cat#18000</td>
</tr>
<tr>
<td>BD FACSymphony A5 Flow Cytometer</td>
<td>BD</td>
<td>N/A</td>
</tr>
<tr>
<td>MoFlo Astrios Cell Sorter</td>
<td>Beckman Coulter</td>
<td>N/A</td>
</tr>
<tr>
<td>10<b>×</b> Genomics Chromium Controller</td>
<td>10<b>×</b> Genomics</td>
<td>N/A</td>
</tr>
<tr>
<td>Sequencing instrument (NextSeq/HiSeq/NovaSeq; single cell libraries not compatible with MiSeq)</td>
<td>Illumina</td>
<td>N/A</td>
</tr>
<tr>
<td>Fine scissors</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Forceps</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Equipment/consumables according to Chromium NextGEM single cell 3′ gene expression protocol, version 3.1 rev. B</td>
<td>
<a href="https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf">10x</a> Genomics: <a href="https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf">https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf</a>
</td>
<td>Various</td>
</tr>
<tr>
<td>Equipment/consumables according to BioLegend CITE-seq protocol for TotalSeq-A antibodies and 10<b>×</b> Genomics Single Cell 3′ Reagent Kit v3.1</td>
<td>BioLegend: <a href="https://www.biolegend.com/en-us/protocols/totalseq-a-antibodies-and-cell-hashing-with-10x-single-cell-3-reagent-kit-v3-3-1-protocol">Protocol - TotalSeq™-A Antibodies and Cell Hashing with 10x Single Cell 3' Reagent Kit v3 3.1 Protocol (BioLegend.com</a>)</td>
<td>Various</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Phosphate-buffered saline (PBS)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Final volume/amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>137 mM</td>
<td>8g</td>
</tr>
<tr>
<td>KCl</td>
<td>2.7 mM</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>10 mM</td>
<td>1.44 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1.8 mM</td>
<td>0.24 g</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>N/A</td>
<td>1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>N/A</td>
<td>1 L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Adjust PBS to pH 7.4. PBS can be stored at room temperature for 6 months, but should be prepared fresh for highly sensitive sequencing assays.</div>
<table id="undtbl2">
<caption>Digestion buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Final volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Collagenase D</td>
<td>1 mg/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>DNase I</td>
<td>0.1 mg/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>N/A</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>N/A</td>
<td>1 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This solution should be prepared fresh.</div>
<table id="undtbl3">
<caption>Ammonium-chloride-potassium (ACK) lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Final volume/amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NH<sub>4</sub>Cl</td>
<td>0.15 M</td>
<td>8.29 g</td>
</tr>
<tr>
<td>KHCO3</td>
<td>0.01 M</td>
<td>1 g</td>
</tr>
<tr>
<td>EDTA (0.5M pH 8.0)</td>
<td>0.1 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>Milli-Q purified water</td>
<td>N/A</td>
<td>1L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>N/A</td>
<td>1L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Adjust ACK buffer to pH 7.2–7.4. ACK buffer can be stored at 4°C for up to 6 months, but should be prepared fresh for highly sensitive sequencing assays.</div>
<table id="undtbl4">
<caption>Compensation bead solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Final volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ultra eComp beads</td>
<td>30% (v/v)</td>
<td>120 μL (6 drops)</td>
</tr>
<tr>
<td>PBS</td>
<td>N/A</td>
<td>240 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>N/A</td>
<td>360 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This solution should be prepared fresh.</div>
<table id="undtbl5">
<caption>FACS buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Final volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>N/A</td>
<td>960 mL</td>
</tr>
<tr>
<td>EDTA (0.5M pH 8.0)</td>
<td>2 mM</td>
<td>4 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>2%</td>
<td>20 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>N/A</td>
<td>1L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> FACS buffer can be stored at 4°C for up to 3 months for flow cytometry assays,but should be prepared fresh for highly sensitive sequencing assays.</div>
<table id="undtbl6">
<caption>Fc-block/FVS solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Final volume/amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Fc block</td>
<td>5 μg/mL</td>
<td>0.25 μg</td>
</tr>
<tr>
<td>FVS440UV</td>
<td>40 μg/mL</td>
<td>2 μg</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>N/A</td>
<td>50 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Fc-block/FVS solution should be prepared fresh on the day of experiment. Upscale for samples &gt;1. The amount of reagents has been determined based on a cell number of 2–5 <b>×</b> 10<sup>6</sup> cells in a total staining volume of 50 μL. The volume of PBS depends on the required volume of antibody, and should lead to a total volume of 50 μl.</div>
<table id="undtbl7">
<caption>Antibody solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Final volume/amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FACS buffer</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Epcam-BUV395</td>
<td>0.5 μg/mL</td>
<td>0.025 μg</td>
</tr>
<tr>
<td>CD4-BUV496</td>
<td>1 μg/mL</td>
<td>0.05 μg</td>
</tr>
<tr>
<td>CD45-BUV615</td>
<td>0.125 μg/mL</td>
<td>0.00625 μg</td>
</tr>
<tr>
<td>Ly6G-BUV737</td>
<td>2 μg/mL</td>
<td>0.1 μg</td>
</tr>
<tr>
<td>CD11c-BV421</td>
<td>1 μg/mL</td>
<td>0.05 μg</td>
</tr>
<tr>
<td>CD19-BV570</td>
<td>0.5 μg/mL</td>
<td>0.025 μg</td>
</tr>
<tr>
<td>CD24-BV605</td>
<td>2 μg/mL</td>
<td>0.1 μg</td>
</tr>
<tr>
<td>XCR1-BV650</td>
<td>2 μg/mL</td>
<td>0.1 μg</td>
</tr>
<tr>
<td>MHCII∗-BV711</td>
<td>0.25 μg/mL</td>
<td>0.0125 μg</td>
</tr>
<tr>
<td>CD3-BV750</td>
<td>2 μg/mL</td>
<td>0.1 μg</td>
</tr>
<tr>
<td>CD64-BV786</td>
<td>2 μg/mL</td>
<td>0.1 μg</td>
</tr>
<tr>
<td>Ly6C-FITC</td>
<td>0.625 μg/mL</td>
<td>0.03125 μg</td>
</tr>
<tr>
<td>CD712a-PerCP-eFluor710</td>
<td>1 μg/mL</td>
<td>0.05 μg</td>
</tr>
<tr>
<td>F4/80-PE/Cy5</td>
<td>0.25 μg/mL</td>
<td>0.0125 μg</td>
</tr>
<tr>
<td>PDCA1-APC</td>
<td>0.25 μg/mL</td>
<td>0.0125 μg</td>
</tr>
<tr>
<td>CD11b-APC-R700</td>
<td>0.25 μg/mL</td>
<td>0.0125 μg</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>50 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Antibody solution should be prepared fresh on the day of experiment. Upscale for samples &gt;1. The antibody amount has been determined based on a cell number of 2–5 <b>×</b> 10<sup>6</sup> cells in a total staining volume of 50 μL. The volume of FACS buffer depends on the required volume of antibody, and should lead to a total volume of 50 μl. ∗The MHCII antibody depends on the murine genetic background. We here used an I-A/I-E-specific antibody that is compatible with C57BL/6J mice.</div>
<table id="undtbl8">
<caption>Computer requirements</caption>
<thead></thead>
<tbody>
<tr>
<td><b>For 10× Genomics cell ranger</b></td>
</tr>
<tr>
<td>8-core Intel or AMD processor (16 cores recommended)</td>
</tr>
<tr>
<td>64GB RAM (128GB recommended)</td>
</tr>
<tr>
<td>1TB free disk space</td>
</tr>
<tr>
<td>64-bit CentOS/RedHat 6.0 or Ubuntu 12.04</td>
</tr>
<tr>
<td><b>For high performance computing (HPC) cluster mode</b></td>
</tr>
<tr>
<td>8-core Intel or AMD processor per node</td>
</tr>
<tr>
<td>6GB RAM per core</td>
</tr>
<tr>
<td>Shared filesystem (e.g. NFS)</td>
</tr>
<tr>
<td>SGE or LSF batch scheduling system</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Spleen collection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This section describes euthanasia and collection of spleens from mouse.</p>
<ol>
<li>Use healthy, 6–12 weeks old mice.</li>
<li>Euthanize mice using an institutionally approved method (e.g., CO<sub>2</sub> asphyxiation).</li>
<li>Place the mouse right side down on the dissection mat, and spray fur with 80% ethanol to disinfect.</li>
<li>Using blunt forceps and scissors, make a 1.5 cm horizontal incision in the skin on the left thorax just below the ribcage.</li>
<li>Dissect further through peritoneum until the spleen is visible (dark red bean-shaped organ of approximately 1 cm length).</li>
<li>To avoid tearing and potential cell loss/ damage, carefully grasp the spleen with the forceps, lift it out of the abdomen and ensure complete detachment from the pancreas.</li>
<li>Dislodge spleen by cutting the splenic blood vessels with fine scissors. Some bleeding is normal.</li>
<li>Using forceps, place spleen into 1.5 mL Eppendorf tube containing 1 mL of cold PBS, and store on ice.</li>
<li>Dispose of the mouse cadaver.</li>
</ol>
<h3 id="sec3.2">Preparation of single cell splenocytes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 90 min</div>
<p>This section describes the preparation of murine splenic single cell suspensions.</p>
<ol start="10">
<li>Remove the spleen from PBS using blunt forceps and place into a fresh Eppendorf tube.</li>
<li>Using a 22-gauge needle and 1 mL syringe, inject 1 mL of collagenase D (Col/D) Digestion Buffer at 18°C–23°C into each spleen, using multiple puncture locations to ensure Col/D reaches all parts of the tissue. The tissue typically inflates in this process, and excess digestion buffer will leak into the tube. Spleen will then be incubated immersed in digestion buffer.</li>
<li>Incubate spleen immersed in digestion buffer in Eppendorf tube in a 37°C water bath for 30 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The choice of digestion enzyme can impact the antigen density on the cell surface of cells, and can therefore interfere with ability to detect antigens using flow cytometry and CITE-seq. Col/D works well in regard to detection of the antigens described in this protocol, and users should test the effects of digestion on the detection of antigens should other antibodies be used.</div>
<div class="note">
<span class="note-title">Note:</span> During cell digestion, complete the bead-based single stain compensation controls (see next section).</div>
<ol start="13">
<li>Transfer contents of the splenic digestion onto a 70 μm cell strainer over a 50 mL Falcon tube and gently mash the spleen through the strainer using a rubber plunger of a 5 mL syringe.</li>
<li>Wash strainer with 10 mL of PBS at 18°C–23°C using a 10 mL serological pipette.</li>
<li>Centrifuge splenocytes at 300 <b>×</b> <i>g</i> for 10 min at 4°C.</li>
<li>Decant the supernatant and resuspend cells of 1 spleen in 1 mL of Ammonium-Chloride-Potassium (ACK) lysis buffer and incubate on ice for 2 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Only one spleen should be processed through one cell strainer to ensure maximal cell yield. If multiple spleens need to be combined, cells should be pooled after Step 19.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not let cells incubate in ACK lysis buffer for longer than 2 min as this can negatively impact cell yield and viability.</div>
<ol start="17">
<li>Add 10 mL PBS and centrifuge at 300 <b>×</b> <i>g</i> for 10 min at 4°C.</li>
<li>Decant supernatant and resuspend cells of one spleen (approximately 50 <b>×</b> 10<sup>6</sup> cells) in 1 mL of PBS.</li>
<li>If absolute cell counts are required, resuspend 20 μL of unstained cells in 20 μL trypan blue, transfer to counting slide, and count.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If cell viability of splenocytes is below 80%, PBS in digestion buffer and washes can be replaced by plain RPMI1640 media.</div>
<ol start="20">
<li>For flow cytometry workflow proceed to step 21, for CITE-seq workflow proceed to step 59.</li>
</ol>
<h3 id="sec3.3">Flow cytometry compensation controls</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This section describes the preparation of compensation controls for flow cytometry.</p>
<div class="note">
<span class="note-title">Note:</span> Due to overlapping emission spectra of diverse fluorochromes, a spill over of fluorescence from the primary to the secondary detector is common for certain combinations of fluorochromes, and needs to be corrected in a process termed compensation. The accuracy of compensation is paramount to distinguish positive and negative signals, and therefore requires special attention. The topic of compensation can be explored in detail elsewhere (<a href="#bib5">Cossarizza et al., 2019</a>). The protocol here uses beads to acquire compensation controls, while other protocols might suggest using cells as single-color controls. Each of these methods has disadvantages which should be considered on the basis of the experiment.</div>
<ol start="21">
<li>Vortex compensation beads for 30 s to break up aggregates.</li>
<li>Pipette 20 μL of bead solution to empty FACS tubes, followed by 80 μL of FACS buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The manufacturer recommends to use one drop (approx. 20 μL) of beads per single stain control, however we have found that beads can be diluted in FACS buffer in up to a 1:3 dilution.</div>
<ol start="23">
<li>Pipette 1 μL of the corresponding antibody to each tube.</li>
<li>Incubate on ice or at 4°C in the dark for 15 min.</li>
<li>Add 1 mL of FACS buffer.</li>
<li>Centrifuge at 300 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Carefully decant supernatant.</li>
<li>Resuspend in 200 μL of FACS buffer.</li>
<li>Keep compensation controls on ice or at 4°C in the dark until acquisition.</li>
</ol>
<h3 id="sec3.4">Flow cytometry sample antibody staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This section describes the incubation of single cells with fluorescence-tagged antibodies.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Advanced flow cytometry require a deep understanding of the technology including panel design based on antigen-density, spectral overlap, spreading errors, compensation and downstream data analysis that lie beyond the scope of this protocol. New users to flow cytometry are strongly encouraged to seek expert advice, mentoring and training by their flow cytometry facility before commencing experiments. Useful additional resources can be found here: (<a href="#bib5">Cossarizza et al., 2019</a>; <a href="#bib1">Brummelman et al., 2019</a>; <a href="#bib18">Mair and Tyznik, 2019</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> All antibodies and buffers should be kept on ice while preparing antibody solutions. Antibody solutions should be prepared fresh within the hour of sample incubation to minimise chances for antibody interaction, and kept shielded from light.</div>
<div class="note">
<span class="note-title">Note:</span> A complete list of antibodies to immunophenotype splenic myeloid cells is provided in <a href="#tbl1">Table 1</a>. This panel was developed and optimised based on a similar panel developed by (<a href="#bib6">Dipiazza et al., 2019</a>). The panel can be extended, and channels that are suitable for the addition of antibodies which detect functional features are indicated. Antibody concentrations can be found under <a href="#materials-and-equipment">materials and equipment</a>.</div>
<table id="tbl1">
<caption>Table 1. Configuration table of murine myeloid antibody panel for BD FACSymphony A5</caption>
<thead>
<tr>
<th>Laser</th>
<th>Filter</th>
<th>Feature</th>
<th>Fluorochrome</th>
</tr>
</thead>
<tbody>
<tr>
<td>355 nm (UV)</td>
<td>UV_379/28</td>
<td>Epcam (optional)</td>
<td>BUV395</td>
</tr>
<tr>
<td> </td>
<td>UV_450/50</td>
<td>FVS</td>
<td>440UV</td>
</tr>
<tr>
<td> </td>
<td>UV_515/30</td>
<td>CD4</td>
<td>BUV496</td>
</tr>
<tr>
<td> </td>
<td>UV_586/15</td>
<td><b>Functional</b></td>
<td>BUV563</td>
</tr>
<tr>
<td> </td>
<td>UV_610/20</td>
<td>CD45</td>
<td>BUV615</td>
</tr>
<tr>
<td> </td>
<td>UV_740/35</td>
<td>Ly6G</td>
<td>BUV737</td>
</tr>
<tr>
<td>405 nm (Violet)</td>
<td>V_450/50</td>
<td>CD11c</td>
<td>BV421</td>
</tr>
<tr>
<td> </td>
<td>V_586/15</td>
<td>CD19</td>
<td>BV570</td>
</tr>
<tr>
<td> </td>
<td>V_610/20</td>
<td>CD24</td>
<td>BV605</td>
</tr>
<tr>
<td> </td>
<td>V_670/30</td>
<td>XCR1</td>
<td>BV650</td>
</tr>
<tr>
<td> </td>
<td>V_710/50</td>
<td>MHC II</td>
<td>BV711</td>
</tr>
<tr>
<td> </td>
<td>V_740/35</td>
<td>CD3</td>
<td>BV750</td>
</tr>
<tr>
<td> </td>
<td>V_780/60</td>
<td>CD64</td>
<td>BV786</td>
</tr>
<tr>
<td>488 nm (Blue)</td>
<td>B_515/20</td>
<td>Ly6C</td>
<td>FITC</td>
</tr>
<tr>
<td> </td>
<td>B_610/20</td>
<td><b>Functional</b></td>
<td>PE/Dazzle</td>
</tr>
<tr>
<td> </td>
<td>B_710/50</td>
<td>CD172A</td>
<td>PerCP eFluor710</td>
</tr>
<tr>
<td> </td>
<td>B_780/60</td>
<td><b>Functional</b></td>
<td>PE/Cy7</td>
</tr>
<tr>
<td>561 nm (Yellow/Green)</td>
<td>YG_670/30</td>
<td>F4/80</td>
<td>PE/Cy5</td>
</tr>
<tr>
<td>642 nm (Red)</td>
<td>R_670/30</td>
<td>PDCA-1</td>
<td>APC</td>
</tr>
<tr>
<td> </td>
<td>R_710/50</td>
<td>CD11b</td>
<td>APC-R700</td>
</tr>
</tbody>
</table>
<ol start="30">
<li>Pipette approximately 5 <b>×</b> 10<sup>6</sup> splenocytes into FACS tubes for the unstained sample and each experimental sample tube, and 2 <b>×</b> 10<sup>6</sup> splenocytes into a live/dead single stain control tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Falcon® 5 mL Round Bottom Polystyrene tubes work well to retain myeloid cells, and allow the addition of sufficient volumes of FACS buffer for washing steps.</div>
<div class="note">
<span class="note-title">Note:</span> Alternative to a cell-based single cell control for live-dead stain, amine-reactive compensation beads can be used here. The stain index of these beads can be improved by dilution in PBS and incubation at 18°C–23°C to encourage the covalent binding reaction.</div>
<ol start="31">
<li>Keep unstained sample on ice or at 4°C until acquisition.</li>
<li>Transfer half of live/dead single stain cells into a fresh Eppendorf tube and put on a heat plate set at 50°C for 10 min to initiate cell death.</li>
<li>During incubation time, prepare a Fc block/FVS solution as outlined in <a href="#materials-and-equipment">materials and equipment</a>.</li>
<li>After 10 min, place dead cells on ice for 2 min, then transfer back to original live/dead Eppendorf tube containing live cells.</li>
<li>Centrifuge live/dead single stain and experimental sample tubes at 300 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Decant supernatant.</li>
<li>Add 50 μL of live/dead/Fc-block cocktail to experimental samples and live/dead single stain.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Live/dead labelling must be performed in PBS only. The presence of FCS in FACS buffer will bind live/dead reagent and interfere with live/dead staining of splenocytes.</div>
<ol start="38">
<li>Incubate on ice or at 4°C in the dark for 15 min.</li>
<li>During incubation time prepare antibody cocktail using the desired concentration of each antibody in 40 μL FACS buffer + 10 μL Brilliant Stain buffer per experimental sample (upscale according to number of experimental samples plus 10%, see antibody cocktail constituent table in <a href="#materials-and-equipment">materials and equipment</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The antibody concentrations provided in this protocol are optimized for a cell number of 5 <b>×</b> 10<sup>6</sup> cells which are stained in a final volume of 50 μL. Because antibody binding depends on the cell number, cell density and antibody density, both volume and antibody need to be adjusted with any change in cell numbers.</div>
<div class="note">
<span class="note-title">Note:</span> Brilliant Stain Buffer should be used if more than one BD Brilliant dye is used to prevent non-specific polymer interactions. 10 μL of Brilliant Stain Buffer should be added to the antibody cocktail for each experimental sample.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To distinguish specific staining from abnormalities derived from fluorescence spillover, spreading error and cell-type specific autofluorescence, fluorescence minus one (FMO) controls should be included (<a href="#bib18">Mair and Tyznik, 2019</a>). These controls are especially critical when measuring features with low to medium antigen density and a clear bimodal expression pattern (expressed and not expressed) is not expected.</div>
<ol start="40">
<li>Add 1 mL of PBS to all tubes incubated with live/dead/Fc-block cocktail, and centrifuge at 300 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Decant supernatant.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Using a smooth ‘invert-pour-tab-revert’ movement, pour supernatant followed by a brief and gentle tab to paper towel to absorb remaining liquid. Cell pellets are routinely not disturbed by this action.</div>
<ol start="42">
<li>Resuspend live/dead single stain control in 400 μL of FACS buffer and leave on ice or at 4°C in the dark until acquisition.</li>
<li>Resuspend experimental samples each in 50 μL of antibody cocktail.</li>
<li>Incubate samples on ice or at 4°C in the dark for 20 min.</li>
<li>Add 1 mL FACS buffer and centrifuge at 300 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Decant supernatants.</li>
<li>Resuspend cell pellets in 400 μL of FACS buffer store on ice or at 4°C in the dark until acquisition.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Once stained, unfixed experimental samples should be acquired within 2 h to ensure viability, and avoid fluorochrome degradation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> This protocol has not been tested with fixation reagents. Any method of fixation, should be tested prior as fixation can negatively affect tandem fluorophores. If fixation is required, compensation controls also need to be treated with fixation agents, and the user should refer to fixation protocols (<a href="#bib5">Cossarizza et al., 2019</a>).</div>
<h3 id="sec3.5">Flow cytometry data acquisition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼2 h</div>
<p>This section describes a basic workflow to acquire cells using a BD FACSymphony A5 flow cytometer.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Flow cytometers should only be operated by trained users. Detailed guidelines for optimal instrument set up have been discussed elsewhere (<a href="#bib18">Mair and Tyznik, 2019</a>), and users of this protocol will need to be familiar with these processes, or have their samples acquired by professional instrument operators.</div>
<ol start="48">
<li>Vortex and strain cells through a FACS tube microstrainer using a disposable transfer pipette immediately before loading onto cytometer.</li>
<li>Using unstained cells, manipulate PMT voltages to adjust forward and side scatter (FSC/SSC), and reflect 3 times of the robust standard deviation (rSD) for each channel in order to ensure comparable fluorophore sensitivity across all channels (<a href="#bib22">Wang and Hoffman, 2017</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Using 3 <b>×</b> rSD to set PMT voltages reliably approximates appropriate signal detection while reducing electronic noise. However, for dim signals, a range of up to 10 <b>×</b> rSD is recommended (<a href="#bib5">Cossarizza et al., 2019</a>). Manipulated rSD values should be derived from pre-gated lymphocytes using the forward and side scatter gate.</div>
<ol start="50">
<li>After setting voltages, begin acquisition of single stain compensation controls, beginning with the unstained sample according to the automatic compensation workflow instructions of your specific cytometer.</li>
<li>Vortex compensation tubes for 3–5 s to resuspend beads before acquisition.</li>
<li>Acquire 5000 total events for each compensation control.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Successful automatic compensation depends on a sufficiently high stain index of the compensation controls, that optimally is larger than the stain index of the antibody in the sample. Bead-based compensation controls typically provide a high stain index, but can at times for yet unknown reasons lead to overcompensation (<a href="#bib1">Brummelman et al., 2019</a>).</div>
<ol start="53">
<li>Following compensation control acquisition, perform automatic compensation calculation.</li>
<li>Vortex experimental samples before acquisition to ensure cells are in solution. Acquire at least 2 <b>×</b> 10<sup>6</sup> events of FSC/SSC lymphocyte gate of each experimental sample for adequate resolution.</li>
<li>Export flow cytometry standard (FCS) files.</li>
</ol>
<h3 id="sec3.6">Flow cytometry data analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼4–8 h</div>
<p>This section describes a basic workflow to carry out quality control of the acquired flow cytometry data and the delineation of the myeloid cells in murine spleen. The steps below give a guideline to analyze flow cytometric data using Kaluza software (Beckman Coulter, version 3.1). Other software are available and the principles for data analysis remain the same.</p>
<ol start="56">
<li>Verify and correct automatic compensation if required.
<div class="critical">
<span class="critical-title">Critical:</span> The success of the automatic compensation algorithms depends on multiple factors, and can under circumstances lead to over- and undercompensation. A very careful quality control of the compensation matrix is essential before commencing any data analysis. This compensation quality control should only be carried out by highly experienced flow cytometry experts, who have an in-depth understanding of fluorochrome spill over, spreading errors and the effects of cell-type specific autofluorescence. If a manual modification to the algorithm-based compensation is required, this too should only be attempted by experts in the field, and is prone to subjectivity-derived error.</div>
Achieving accurate algorithm-based compensation can be challenging and might require comparison of bead- and cell-based compensation controls (<a href="#bib11">Liechti and Roederer, 2019</a>), and if differences are observed, cell-based compensation controls should be considered as biologically relevant gold standard (<a href="#bib5">Cossarizza et al., 2019</a>). Working forward with either a falsely algorithm-based or a falsely manually corrected compensation has major implications to the data analysis and interpretation of data. The steps outlined below provide a basic NxN (every parameter against every parameter) approach how to quality check the compensation and attempt correction.
<ol type="a">
<li>Import FCS files to the analysis software of choice.</li>
<li>To omit debris and isolate lymphocytes, generate a FSC-A/SCC-A bivariate plot, and gate populations of interest (<a href="#fig2">Figure 2</a>A).</li>
<li>To omit doublets, generate a FSC-A/FSC-H bivariate plot, and gate events in a linear, 45 degree line (<a href="#fig2">Figure 2</a>A).</li>
<li>Create a bivariate plot for each used channel, with the channel on the X axis.</li>
<li>Highlight all plots and change the Y axes to the first channel.</li>
<li>If typical patterns that suggest overcompensation (“super-negative” events) or undercompensation (“leaning” triangular populations) are observed, correct compensation as required (see example provided in <a href="#fig1">Figure 1</a>).</li>
<li>Change Y axes to the next channel and repeat compensation correction.</li>
<li>Repeat for all channels.
<div class="note">
<span class="note-title">Note:</span> We find that when a manual compensation is required, the complete NxN approach should be carried out at least twice, or until a full round of NxN can be completed without further modifications are needed. This can be laborious but is critical to the quality of data.</div>
</li>
<li>Apply corrected compensation to all experimental samples.</li>
</ol>
</li>
</ol>
<ol start="57">
<li>Identify myeloid cells based on gating strategy demonstrated in <a href="#fig2">Figures 2</a> and <a href="#fig3">3</a>. Begin by isolating live, single, leukocytes, and then excluding non-target populations such as T-cells (CD3+), and B-cells (CD19+).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1032-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Compensation of fluorochrome spill over</div>
<p>Compensation using bead-based single-color controls can lead to over- and undercompensation in a cell-based sample, which requires manual correction.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1032-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Flow cytometry gating strategy for a 16-color murine myeloid panel</div>
<p>Splenocytes of C57BL/6 mice were incubated with fluorochrome-conjugated antibodies and analyzed by flow cytometry. Red boxes indicate final target populations.</p>
<p>(A) CD45+ leukocytes were identified after exclusion of debris, doublet discrimination and exclusion of dead cells.</p>
<p>(B) From CD45+ cells, B cells and NKT/T cells were excluded, and Ly6G+ neutrophils were validated based on CD11b expression. pDCs were identified as PDCA1+. From PDCA1- cells, macrophages were delineated based on expression of F4/80 and absence of Ly6C- and CD11c-.</p>
<p>(C) Monocytes were identified from CD3-CD19-Ly6G-PDCA1- cells as Ly6C+, and monocyte diversity was further identified through differential expression of CD11b and CD172a.</p>
<p>(D) cDCs were identified from CD3-CD19-Ly6G-PDCA1-Ly6C- cells based on expression of MHC-II+ and CD11c+. cDCs were subdivided into cDC1 (XCR1+CD172a-) and cDC2 (CD172a+XCR1-) cells, and their identity was verified by expression of CD24 and CD11b, respectively. cDC1 and cDC2 cells were further subdivided into CD4+ and CD4- cells.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1032-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative flow cytometry gating tree for delineating murine myeloid cells in spleen</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Identical gating strategy should be applied to all experimental samples. However, when analyzing samples of different origins (eg. unstimulated versus stimulated, or spleen versus lymph node), the expression level of features can differ, and gate limits should be chosen to include these differences.</div>
<ol start="58">
<li>Export statistics (cell numbers, percentages, median fluorescence intensity (MFI) data) for significance testing.</li>
</ol>
<h3 id="sec3.7">Enrichment of myeloid cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>This section describes magnetic bead enrichment of myeloid cells using a negative isolation of non-T and non-B cells, with the aim to reduce sorting time and thus increase cell viability post sorting. The following steps are in accordance with the manufacturer’s protocol (EasySep™ Mouse Streptavidin RapidSpheres™ Isolation Kit).</p>
<div class="note">
<span class="note-title">Note:</span> We pooled splenocytes of 6 C57BL/6 mice, and used 2 <b>×</b> 10<sup>8</sup> splenocytes for myeloid cell enrichment. This allowed us to subsequently sort ∼50,000 DCs.</div>
<ol start="59">
<li>Resuspend splenocytes at 1 <b>×</b> 10<sup>8</sup> cells/mL in FACS buffer in a round bottomed 5 mL tube.</li>
<li>Add 5 μL/mL of biotin conjugated anti-TCRb and biotin conjugated 5 μL/mL anti-CD19 to the cell solution.</li>
<li>Incubate for 10 min at 18°C–23°C.</li>
<li>Add 75 μL of RapidSpheres<sup>TM</sup> magnetic beads and incubate for 2.5 min at 18°C–23°C.</li>
<li>Place the tube in an EasySep<sup>TM</sup> magnet and incubate for 2.5 min.</li>
<li>Carefully decant the supernatant with the myeloid cells into a new FACS tube.</li>
</ol>
<h3 id="sec3.8">Fluorescent activated cell sorting (FACS)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼2 h</div>
<p>This section is to isolate DCs to high purity using FACS. This reduces the levels of debris, doublets and unwanted cells for downstream sequencing.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Cell sorting requires highly trained staff and should be carried out optimally by a flow cytometry core facility.</div>
<ol start="65">
<li>Use all enriched cells from the previous step, set aside a small quantity of cells for unstained and single stain controls for compensation. Single stain controls can alternatively be bead-based as described in the section <a href="#sec3.3">flow cytometry compensation controls</a>.</li>
<li>Stain cells with Fc-block (1:100) and live/dead cell stain (1:250) for 20 min on ice.</li>
<li>Wash with 1 mL PBS and centrifuge at 300 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Resuspend pellet with anti-MHCII-APC-Cy7 (1:400) and anti-CD11c-PE-Cy7 (1:200) in 500 μL FACS buffer and incubate for 20 min on ice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If oligo-tagged antibodies for CITE-seq overlap with antibodies used for sorting, a different antibody clone needs to used. For antibody staining of CD11c, we used the clone HL3 for sorting, and the clone N418 for CITE-seq.</div>
<ol start="69">
<li>Wash cells with 3 mL FACS buffer, centrifuge at 300 <b>×</b> <i>g</i> for 5 min at 4°C and resuspend in 1 mL FACS buffer</li>
<li>Set up cell sorter PMT voltages and compensation using unstained and single stained cells.
<ol type="a">
<li>Determine cells and eliminate debris with FCS-H vs SSC-H.</li>
<li>Determine single cells with FSC-A vs FSC-H and SSC-A vs SSC-H.</li>
<li>Determine live cells as live/dead stain negative.</li>
<li>Determine cDCs as MHCII+CD11c+.</li>
</ol>
</li>
<li>Sort cells into FACS tube containing 500 μL 100% FCS.</li>
<li>Dilute with 2 mL FACS buffer and centrifuge at 300 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Record a small fraction of the sorted cells to check post sort purity and viability (<a href="#fig4">Figure 4</a>).</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1032-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. DC sorting</div>
<p>Myeloid enriched splenocytes were incubated with live/dead stain, Fc-block and antibodies against CD11c and MHCII. DCs were sorted as alive single CD11c+MHCII+ cells to a purity &gt;90%.</p>
</figcaption>
</figure>
<h3 id="sec3.9">Incubation of DCs with nucleotide barcoded antibodies</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Here, cells are stained with oligo-tagged antibodies that are conjugated with nucleotide sequences specific for each antibody. The steps listed below are in accordance with BioLegend CITE-seq protocol for TotalSeq-A antibodies and 10<b>×</b> Genomics Single Cell 3′ Reagent Kit v3.1 (<a href="https://www.biolegend.com/en-us/protocols/totalseq-a-antibodies-and-cell-hashing-with-10x-single-cell-3-reagent-kit-v3-3-1-protocol">https://www.biolegend.com/en-us/protocols/totalseq-a-antibodies-and-cell-hashing-with-10x-single-cell-3-reagent-kit-v3-3-1-protocol</a>).</p>
<ol start="73">
<li>Resuspend cell pellet from the sort in Cell Staining Buffer (1 <b>×</b>10ˆ<sup>6</sup>mouse cells in 49.5 μL) in a FACS tube.</li>
<li>Add 0.5 μL of TruStain FcX™ PLUS (anti-mouse CD16/32) antibody per 49.5 μL cell suspension and incubate for 10 min at 4°C.</li>
<li>Prepare antibody pool using titrated amount (up to 1 μg of each TotalSeq-A antibody). Add Cell Staining Buffer to make the volume of the antibody pool 50 μL for each 50 μL cell suspension.</li>
<li>To remove antibody aggregates, centrifuge the antibody pool at 1,400 <b>×</b> <i>g</i> for 10 min at 4°C and carefully pipette out the antibody pool without touching the bottom of the tube.</li>
<li>Add the 50 μL of antibody pool to each 50 μL of cell suspension.</li>
<li>Incubate for 30 min on ice.</li>
<li>Wash with 3 mL Cell Staining Buffer and centrifuge at 400 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Decant supernatant.</li>
<li>Repeat wash twice.</li>
<li>Resuspend cell pellet in 250 <u>μ</u>L Cell Staining Buffer and immediately move to the next step.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Antibody validation and titration should be performed prior to carrying out this protocol. It is possible to titrate the oligo-tagged antibodies using either single cell sequencing read outs (<a href="https://www.biolegend.com/en-us/blog/titrating-totalseq-antibodies">https://www.biolegend.com/en-us/blog/titrating-totalseq-antibodies</a>), which is costly, or by flow cytometry using the same clone with a fluorophore or dT-oligonucleotide tagged fluorescent dye. An example of oligo-tagged antibody validation at a concentration of 0.5 <u>μ</u>g and using a dT-oligonucleotide tagged fluorescent dye is shown in <a href="#fig5">Figure 5</a>. However, because sequencing is more sensitive than flow cytometry, the optimal antibody concentration for flow cytometry might not be the optimal antibody concentration for sequencing. The accurate identification of the optimal antibody concentration without the need for sequencing represents a current limitation to the technology.</div>
<div class="optional">
<span class="optional-title">Optional:</span> According to the 10<b>×</b> Genomics protocol for Cell Surface Protein Labelling, cells can be stained with oligo-tagged antibodies simultaneously with fluorochrome conjugated antibodies prior to sorting (<a href="https://support.10xgenomics.com/single-cell-gene-expression/overview/doc/demonstrated-protocol-cell-surface-protein-labeling-for-single-cell-rna-sequencing-protocols">https://support.10xgenomics.com/single-cell-gene-expression/overview/doc/demonstrated-protocol-cell-surface-protein-labeling-for-single-cell-rna-sequencing-protocols</a>). Staining prior to sorting can have advantages: it may reduce the number of unbound oligo-tagged antibodies as the cells will be washed multiple times in the sorting process; and reduce the time between cell isolation and downstream steps, thus reducing the time until cell lysis. However, this requires staining a higher number of cells and the amount of oligo-tagged antibodies needs to be adjusted accordingly.</div>
<div class="optional">
<span class="optional-title">Optional:</span> To distinguish oligo-tagged antibody specific binding and background noise, users can consider a spike-in of cells that are negative for the antibody-specific epitopes. For example, for CITE-seq experiments of murine cells, users can spike in human cells. This provides an internal control for antibody background noise. Alternatively, if a diverse population of cells is processed, cells that are considered negative for epitope expression can be used to determine background (<a href="#bib14">Lukowski et al., 2021</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Wash thoroughly after oligo-tagged antibody staining to reduce background noise from unbound antibodies.</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1032-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Total-Seq-A antibody validation</div>
<p>Splenocytes were incubated with flow cytometry live-dead stain, Fc-block and antibodies against MHCII and CD11c, and each Total-Seq-A antibody which was conjugated to a dT-oligonucleotide tagged fluorescence dye. Flow cytometry and Total-Seq-A antibodies against CD11c were specific to different epitopes.</p>
</figcaption>
</figure>
<h3 id="sec3.10">10<b>×</b> chromium platform and library preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 days</div>
<p>This step includes generating gel beads in emulsions (GEMs), barcoding, cDNA amplification and library construction.</p>
<div class="note">
<span class="note-title">Note:</span> Due to Chromium kit updates that have occurred since the original study (<a href="#bib14">Lukowski et al., 2021</a>), the instructions and resources outlined below differ slightly from what was performed at the time. We describe below the current best-practice workflow, using the most updated kits available, as a recommendation for how we would conduct the experiment today.</div>
<div class="note">
<span class="note-title">Note:</span> Single cell sequencing experiments are highly technical and expensive to perform. If you are inexperienced with single cell library preparation and sequencing, or there are aspects that will be performed by a core facility, we advise discussing your project in advance with the facility or with 10<b>×</b> Genomics technical support. Understanding the requirements and key points at each step of the protocol will give the best chance for a successful outcome.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The Chromium library preparation protocol is required to be performed in a timely manner once cells are ready. It is imperative that the relevant protocols and user guides have been read thoroughly and are familiar to the user before beginning, and all necessary reagents and consumables have been acquired and prepared in advance.</div>
<ol start="83">
<li>Prepare all necessary reagents for GEM generation as outlined in the 10<b>×</b> Genomics Chromium User Guide (<a href="https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf">https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf</a>).</li>
<li>Manually count cells and ensure that cell viability is above 80%</li>
<li>For super-loading of the Chromium reaction (targeting more than 10,000 cells per reaction), consult the table on page 33 of the Cell Multiplexing User Guide (<a href="https://assets.ctfassets.net/an68im79xiti/Ic17bUxh3yu95BmBNGsE3/3d333cebd76782ed1ccf8a032bb3e294/CG000388_ChromiumNextGEMSingleCell3-v3.1_CellMultiplexing_Rev_A.pdf">https://assets.ctfassets.net/an68im79xiti/Ic17bUxh3yu95BmBNGsE3/3d333cebd76782ed1ccf8a032bb3e294/CG000388_ChromiumNextGEMSingleCell3-v3.1_CellMultiplexing_Rev_A.pdf</a>) to determine the appropriate volume of cells to add for a particular concentration. For example, to target 15,000 single cells would require ∼27.5uL of a cell suspension at a concentration of 1 <b>×</b> 10<sup>6</sup> cells/mL.</li>
<li>Prepare gene expression and CITE-seq libraries by following the steps outlined in the BioLegend TotalSeq-A protocol (<a href="https://www.biolegend.com/en-us/protocols/totalseq-a-antibodies-and-cell-hashing-with-10x-single-cell-3-reagent-kit-v3-3-1-protocol">https://www.BioLegend.com/en-us/protocols/totalseq-a-antibodies-and-cell-hashing-with-10x-single-cell-3-reagent-kit-v3-3-1-protocol</a>) and the 10x Genomics Chromium User Guide (<a href="https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf">https://assets.ctfassets.net/an68im79xiti/1W6c5yj7FAiBQdOKabc6sc/06e962ecdff018fb20b6979859cf1f5e/CG000317_ChromiumNextGEMSingleCell3-v3.1_CellSurfaceProtein_RevB.pdf</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Start GEM generation immediately after oligo-tagged antibody staining to minimize cell death. It is recommended that the viability of the samples at this point is &gt;80%. Using low-viability samples can impact data quality due to increased ambient RNA from dead and dying cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that you are following the TotalSeq-A protocol to include ADT additive primer in the cDNA amplification mix, and save the low molecular weight fragments in the supernatant of the post-amplification clean-up, otherwise a CITE-seq library will not be able to be generated for your sample.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Purified full-length amplified cDNA (including CITE-seq cDNA) and final libraries can both be stored at −20<sup>o</sup>C until moving forward to library preparation or sequencing, respectively.</div>
<div class="note">
<span class="note-title">Note:</span> To reduce the possibility of index cross-talk, if possible avoid using the following combinations of CITE-seq indexes with samples intended for the same sequencing run:</div>
<p>- RPI01 and RPI09</p>
<p>- RPI08 and RPI12</p>
<p>The above index combinations have a Sequence-Levenshtein edit distance of 2. All other combinations of CITE-seq (RPI01 to RPI12) and Chromium (SI-TT-A1 to SI-TT-H12) indexes have an edit distance of &gt;=3, allowing for accurate correction of a single base mismatch caused by substitution, insertion or deletion. If the pairs noted above must be used together, it would be preferable to adjust the default parameters during fastq generation to allow no mismatches.</p>
<div class="note">
<span class="note-title">Note:</span> Both 10<b>×</b> Genomics and BioLegend will periodically provide version updates for the protocols cited above. The documents in this protocol are the latest versions at time of writing; you may want to check to confirm you are using the most up-to-date protocol available before starting your experiment, as subsequent updates may improve performance and outcomes.</div>
<h3 id="sec3.11">Illumina sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This section generates sequencing data from the prepared gene expression and CITE-seq libraries.</p>
<ol start="87">
<li>Pool libraries and sequence according to desired sequencing depth. For the antibody pool used here, we recommend pooling the CITE-seq and gene expression libraries for each sample at a 10% and 90% molar ratio respectively. When sequencing two samples targeting 15,000 cells each with this pooling, you can expect to generate &gt;20,000 gene expression reads per cell (the minimum recommended by 10<b>×</b> Genomics) using a NovaSeq SP 100 cycle sequencing run. Sequencing kits can be scaled up as needed for projects with greater sample numbers. For projects utilizing higher numbers of antibodies, it will be necessary to increase the pooling proportion of the CITE-seq libraries accordingly to ensure adequate read numbers for each antibody used. The minimum recommended sequencing depth for 10<b>×</b> Genomics gene expression libraries is 20,000 reads per cell and 5,000 reads per cell for barcoded antibody (CITE-seq) libraries. It is important to note that if the number of barcoded antibodies increases significantly or includes highly abundant proteins such as CD45, the read depth per cell may require adjustment.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Pooled CITE-seq/gene expression libraries can be run using the recommended configuration of Read1 – 28bases, Index1 – 10bases, Index2 – 10bases, Read2 – 90 bases.</div>
<h3 id="sec3.12">Data pre-processing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This section includes initial data pre-processing using Cell Ranger software to generate fastq files for each sample and count tables containing both gene expression and CITE-seq feature information.</p>
<ol start="88">
<li>Raw sequencing data from the Illumina platform is in the BCL format. These raw files must be demultiplexed into individual samples and subsequently converted to fastq files. This can be achieved using the 10<b>×</b> Genomics Cell Ranger mkfastq pipeline in conjunction with the sample-specific index barcodes. Follow the instructions for running the Cell Ranger mkfastq pipeline found on the 10<b>×</b> Genomics support site: <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/mkfastq">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/mkfastq</a>.</li>
<li>Cell Ranger mkfastq requires a comma-separated table with columns specifying the flowcell lane, sample names and indexes used. It is possible to process multiple library types at once, which avoids the need to run mkfastq once for each library type. However some modifications to the recommended SimpleCSV samplesheet are required. In this modification, the CITE-seq library indexes can be presented in the SimpleCSV samplesheet in a dual-index format. To do this, add ‘ATCT’ (the four bases immediately following the 6 bp index in the adaptor sequence) to the end of each index sequence, then provide the sequence ‘GTGTAGATCT’ in an additional ‘Index2’ column. This will be the sequence generated from single-indexed CITE-seq library fragments by the i5 index primer. The 10<b>×</b> dual index TT Set A ID can be replicated in both ‘Index’ and ‘Index2’ columns for the gene expression libraries. Please see example below for generating fastq files for samples using RPI01 (ATCACG) and RPI02 (CGATGT) CITE-seq indexes, for samples run on a NovaSeq6000 platform with v1.5 kits.</li>
</ol>
<div class="textbox">
<p><code>Lane,Sample,Index,Index2</code></p>
<p><code>∗,CDC_WT,SI-TT-A1,SI-TT-A1</code></p>
<p><code>∗,CS_CDC_WT,ATCACGATCT,GTGTAGATCT</code></p>
<p><code>∗,CDC_BATF3_KO,SI-TT-A2,SI-TT-A2</code></p>
<p><code>∗,CS_CDC_BATF3_KO,CGATGTATCT,GTGTAGATCT</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> The exemplary samples CDC_WT and CDC_BATF3_KO were sorted cDCs from wildtype and <i>Batf3</i>−/− mice published in (<a href="#bib14">Lukowski et al., 2021</a>).</div>
<ol start="90">
<li>Using the fastq files generated in Step 1, follow the instructions for running the Cell Ranger count pipeline with feature barcoding, which utilizes Cell Ranger Feature Barcoding capabilities to incorporate the CITE-seq information in the output gene-count matrix: <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/feature-bc-analysis">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/feature-bc-analysis</a>.</li>
<li>The reference genomes and annotations for mouse (GRCm38) and human (GRCh38) required for Cell Ranger count can be obtained from the 10<b>×</b> Genomics website, or can be provided by the user.</li>
<li>Following the generation of sample-level count matrices by Cell Ranger count, multiple samples can be combined into a single large matrix using Cell Ranger aggr (aggregate). It is also possible to analyze each sample individually without the use of the aggregate function.</li>
<li>After count matrix aggregation, the dataset comprising gene and protein expression values is ready for analysis in Seurat or other software.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The strand read by the i5 sequencing primer depends on the sequencing platform used. The index2 sequence presented above is for sequencers using a ‘reverse complement’ workflow for dual-indexing, as for NovaSeq6000 with v1.5 kits. The index2 sequence should be modified to use the reverse complement if a ‘forward strand’ workflow is used, for example on a NovaSeq6000 with v1.0 kits.</div>
<div class="note">
<span class="note-title">Note:</span> If single-indexed gene expression libraries are generated (with SI-GA-xx index IDs), a SimpleCSV with a single index column is sufficient. In this case, to accommodate an 8bp i7 index read, the suffix ‘AT’ can be added to the CITE-seq indexes. The SimpleCSV for the same example as above would then look like this:</div>
<div class="textbox">
<p><code>Lane,Sample,Index</code></p>
<p><code>∗,CDC_WT,SI-GA-A1</code></p>
<p><code>∗,CS_CDC_WT,ATCACGAT</code></p>
<p><code>∗,CDC_BATF3_KO,SI-GA-A2</code></p>
<p><code>∗,CS_CDC_BATF3_KO,CGATGTAT</code></p>
</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol outlines a general workflow and critical considerations for phenotyping of cells using advanced flow cytometry, cell isolation using fluorescence-activated cell sorting and single cell sequencing of epitopes and transcripts. Specifically, we demonstrate the step-by-step methodology using murine splenocytes and antibodies to delineate, isolate and sequence myeloid cells and DCs.</p>
<p>The expected outcomes of the flow cytometric delineation of myeloid cells are shown in detail in <a href="#fig2">Figures 2</a> and <a href="#fig3">3</a>. The 16-color flow cytometry panel allows delineation of 5 main myeloid lineages, including monocytes, macrophages, neutrophils, pDCs and cDC cells, and a large cell subset and cell state heterogeneity within these main lineages. Our manual analysis allowed identification of 16 discrete populations, and additional cell state diversity is revealed when using unbiased bioinformatic approaches to identify heterogeneity based on differential co-expression of phenotypic features (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1032-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Expected outcome of a bioinformatic unbiased analysis of the 16-color murine myeloid flow cytometry panel</div>
<p>(A) Single alive CD45+ CD3- CD19- cells were subjected to bioinformatic FlowSOM analysis based on expression of Ly6G, Ly6C, PDCA1, CD11b, CD11c, F4/80, MHCII, CD172a, CD24, XCR1, and CD4 with a metacluster number of 40. User-determined thresholds for positive signal were applied to all features. The analysis identified 40 distinct clusters of cells.</p>
<p>(B) Clusters were annotated based on key myeloid lineage features.</p>
</figcaption>
</figure>
<p>Expected outcomes of the CITE-seq workflow is a single cell sequencing dataset of approximately 25,000 DCs with a sequencing depth of 20,000 reads per cells (gene expression) and 5,000 reads per cell (barcoded antibodies).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>For the flow cytometric analysis of myeloid cells, absolute numbers and relative abundances of myeloid cell lineages and cell states can be compared between different genotypes and treatment regimes. Differences in the mean fluorescence intensities of features with functional implications such as MHCII, CD64 and XCR1 can also be compared. For comparisons of functional feature expression, a fluorescence minus one (FMO) control should be included and the MFI of FMO should be subtracted to control for population-specific spreading errors and autofluorescence. In addition to traditional manual population gating, flow cytometry data can be analyzed using unbiased bioinformatic platforms (<a href="#bib19">Melsen et al., 2020</a>; <a href="#bib1">Brummelman et al., 2019</a>), and both should be used hand-in-hand to uncover cell lineage and cell state heterogeneity.</p>
<p>CITE-seq data can be analyzed using standard single cell bioinformatic workflows, such as Seurat (<a href="#bib2">Butler et al., 2018</a>) or Scanpy (<a href="#bib23">Wolf et al., 2018</a>). Examples of integrated analysis platforms for multi-omic data are weighted-nearest neighbor analysis (<a href="#bib7">Hao et al., 2021</a>), One-SENSE (<a href="#bib4">Cheng et al., 2016</a>; <a href="#bib17">Mair et al., 2020</a>), CiteFuse (<a href="#bib9">Kim et al., 2020</a>) and CITE-sort (<a href="#bib10">Lian et al., 2020</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Flow cytometry data is especially difficult to standardize between different laboratories due to differences in the hardware components of the cytometer as well as differences in batches of antibodies. The user will therefore have to re-titrate antibodies and establish optimal cytometer settings to achieve good data outcomes. Manual correction of the compensation is prone to user subjectivity and should be handled with extreme caution. A very detailed quality control of the algorithm-based compensation performed by the flow cytometer is absolutely essential. If the algorithm-based compensation does not successfully apply to the experimental samples, the user should aim to optimize the preparation and acquisition of the compensation controls to circumvent the need for a manual correction. Key principles to achieve this are outlined here (<a href="#bib5">Cossarizza et al., 2019</a>). Furthermore, spreading error needs to be investigated on individual cytometers to avoid false positive cell annotation. Acquisition of a complete set of FMO controls is laborious but will aid confident discrimination of positive and negative signals. We urge new flow cytometry users to study the current literature regarding advanced flow cytometry methodology guidelines (<a href="#bib18">Mair and Tyznik, 2019</a>) and recommended guidelines for flow cytometry reporting in detail (<a href="#bib13">Lucas et al., 2020</a>; <a href="#bib5">Cossarizza et al., 2019</a>), and advise that this is not a trivial methodology.</p>
<p>CITE-seq data may contain high levels of background noise from the barcoded antibody staining. This may be due to non-specific binding or high staining concentration, and antibody concentration optimization may help to reduce this noise. Of note, the current version of the protocol is optimized for cell surface proteins only. Concerning per-cell read depth, antibody panels designed to capture highly abundant proteins such as CD45 are likely to require deeper sequencing than the suggested depth. Since highly expressed proteins will account for a large proportion of sequencing reads in a given run, it is crucial to ensure the sequencing depth is sufficient to capture the real expression profile of lower expressed proteins. Overcoming this may require resequencing of the same CITE-seq library.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Lack of positive flow cytometry bead staining</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Ensure the isotypes of the antibodies match those of rat, mouse, or hamster (steps 21–23). While eComp beads (Thermofisher) contain beads of all three isotypes, other commercial beads such as those provided by BD may not contain the isotype compatible with your antibodies.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Unclear discrimination of positive and negative signals.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>FMO controls where the marker of interest is omitted from the antibody cocktail, and isotype controls can provide more accurate discrimination of positive staining (<a href="#bib15">Maciorowski et al., 2017</a>; <a href="#bib16">Maecker and Trotter, 2006</a>). Optimally FMO, controls should be acquired for every antibody, but this can be laborious (step 39). These controls however are essential for detection of antigens that are expressed at low to medium levels and a bimodal expression profile does not apply. The use of isotype controls which control only for binding of antibodies via Fc receptors is considered non-essential when Fc receptors are efficiently blocked before antibody staining. Internal negative controls (cells within the sample that don’t express the feature) can help with confident discrimination of expression, although these should have a comparable autofluorescence.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Low antibody staining</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Absence of signal may be due to either absence of protein expression or of low photomultiplier tube (PMT) voltages (step 49). Low PMT voltages may cause insufficient filter sensitivity for receiving positive signals. Accurate setting of PMT voltages using an unstained sample combined with using antibody concentrations determined from titrations typically overcomes this problem. The PMT voltage can be increased to up to 10<b>×</b> rSD to detect dim signals, and should be accompanied by an FMO control. Furthermore, use of rainbow beads (BD) to set PMT voltages offers a quick and accurate method for consistent voltage setting over time (<a href="#bib8">Kalina, 2020</a>; <a href="#bib20">Perfetto et al., 2006</a>; <a href="#bib22">Wang and Hoffman, 2017</a>).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Inaccuracy in automatic compensation</p>
<h3 id="sec7.8">Potential solution</h3>
<p>High parameter compensation matrices are complex and require great care in creation. All downstream data analysis depends on the quality of compensation, and accurate compensation therefore is absolute paramount to any flow cytometry experiment (step 56). Optimally, all compensation controls, including the control of the viability stain, are acquired based on both cells and beads, and the performance of the algorithm-based compensation is compared. If differences in the spill-over values are observed between cells and beads, then cells are to be used as biologically relevant gold standard single color controls (<a href="#bib5">Cossarizza et al., 2019</a>). Of note, beads provided by different suppliers can vary in their capacity to achieve accurate algorithm-based compensation. Further, the stain index of all compensation controls should be higher than that of the antibody staining in the experimental sample, although this is not absolute. If still experiencing problems that the algorithm-based compensation leads to over- or under-compensation, only a highly trained and experienced user should attempt a manual correction using considerations of spill over, spreading error and co-expression. Compensation strategies have been described in detail by others (<a href="#bib21">Szalóki and Goda, 2015</a>; <a href="#bib15">Maciorowski et al., 2017</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Brummelman, J., Haftmann, C., Nunez, N.G., Alvisi, G., Mazza, E.M.C., Becher, B., and Lugli, E. (2019). Development, application and computational analysis of high-dimensional fluorescent antibody panels for single-cell flow cytometry. Nat. Protoc. <i>14</i>, 1946-1969. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref1">View at publisher</a></p>
<p id="bib2">Butler, A., Hoffman, P., Smibert, P., Papalexi, E., and Satija, R. (2018). Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat. Biotechnol. <i>36</i>, 411-420. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref2">View at publisher</a></p>
<p id="bib3">Buus, T.B., Herrera, A., Ivanova, E., Mimitou, E., Cheng, A., Herati, R.S., Papagiannakopoulos, T., Smibert, P., Odum, N., and Koralov, S.B. (2021). Improving oligo-conjugated antibody signal in multimodal single-cell analysis. Elife <i>10</i>, e61973. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref3">View at publisher</a></p>
<p id="bib4">Cheng, Y., Wong, M.T., Van Der Maaten, L., and Newell, E.W. (2016). Categorical analysis of human T cell heterogeneity with one-dimensional soli-expression by nonlinear stochastic embedding. J. Immunol. <i>196</i>, 924-932. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref4">View at publisher</a></p>
<p id="bib5">Cossarizza, A., Chang, H.D., Radbruch, A., Acs, A., Adam, D., Adam-Klages, S., Agace, W.W., Aghaeepour, N., Akdis, M., Allez, M., et al. (2019). Guidelines for the use of flow cytometry and cell sorting in immunological studies (second edition). Eur. J. Immunol. <i>49</i>, 1457-1973. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref5">View at publisher</a></p>
<p id="bib6">Dipiazza, A.T., Hill, J.P., Graham, B.S., and Ruckwardt, T.J. (2019). OMIP-061: 20-color flow cytometry panel for high-dimensional characterization of murine antigen-presenting cells. Cytometry A <i>95</i>, 1226-1230. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref6">View at publisher</a></p>
<p id="bib7">Hao, Y., Hao, S., Andersen-Nissen, E., Mauck, W.M., Zheng, S., Butler, A., Lee, M.J., Wilk, A.J., Darby, C., Zager, M., et al. (2021). Integrated analysis of multimodal single-cell data. Cell <i>184</i>, 3573-3587 e29. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref7">View at publisher</a></p>
<p id="bib8">Kalina, T. (2020). Reproducibility of flow cytometry through standardization: Opportunities and challenges. Cytometry A <i>97</i>, 137-147. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref8">View at publisher</a></p>
<p id="bib9">Kim, H.J., Lin, Y., Geddes, T.A., Yang, J.Y.H., and Yang, P. (2020). CiteFuse enables multi-modal analysis of CITE-seq data. Bioinformatics <i>36</i>, 4137-4143. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref9">View at publisher</a></p>
<p id="bib10">Lian, Q., Xin, H., Ma, J., Konnikova, L., Chen, W., Gu, J., and Chen, K. (2020). Artificial-cell-type aware cell-type classification in CITE-seq. Bioinformatics <i>36</i>, i542-i550. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref10">View at publisher</a></p>
<p id="bib11">Liechti, T. and Roederer, M. (2019). OMIP-060: 30-parameter flow cytometry panel to assess T cell effector functions and regulatory T cells. Cytometry A <i>95</i>, 1129-1134. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref11">View at publisher</a></p>
<p id="bib24">Liechti, T., Weber, L.M., Ashhurst, T.M., Stanley, N., Prlic, M., Van Gassen, S., and Mair, F. (2021). An updated guide for the perplexed: cytometry in the high-dimensional era. Nat Immunol. <a class="external-link" href="https://doi.org/10.1038/s41590-021-01006-z">https://doi.org/10.1038/s41590-021-01006-z</a></p>
<p id="bib12">Liu, Z., Gu, Y., Shin, A., Zhang, S., and Ginhoux, F. (2020). Analysis of myeloid cells in mouse tissues with flow cytometry. STAR Protoc. <i>1</i>, 100029. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref12">View at publisher</a></p>
<p id="bib13">Lucas, F., Gil-Pulido, J., Lamacchia, J., Preffer, F., Wallace, P.K., and Lopez, P. (2020). MiSet RFC standards: defining a universal minimum set of standards required for reproducibility and rigor in research flow cytometry experiments. Cytometry A <i>97</i>, 148-155. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref13">View at publisher</a></p>
<p id="bib14">Lukowski, S.W., Rodahl, I., Kelly, S., Yu, M., Gotley, J., Zhou, C., Millard, S., Andersen, S.B., Christ, A.N., Belz, G., et al. (2021). Absence of Batf3 reveals a new dimension of cell state heterogeneity within conventional dendritic cells. iScience <i>24</i>, 102402. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref14">View at publisher</a></p>
<p id="bib15">Maciorowski, Z., Chattopadhyay, P.K., and Jain, P. (2017). Basic multicolor flow cytometry. Curr. Protoc. Immunol. <i>117</i>, 5.4.1-5.4.38. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref15">View at publisher</a></p>
<p id="bib16">Maecker, H.T. and Trotter, J. (2006). Flow cytometry controls, instrument setup, and the determination of positivity. Cytometry A <i>69</i>, 1037-1042. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref16">View at publisher</a></p>
<p id="bib17">Mair, F., Erickson, J.R., Voillet, V., Simoni, Y., Bi, T., Tyznik, A.J., Martin, J., Gottardo, R., Newell, E.W., and Prlic, M. (2020). A targeted multi-omic analysis approach measures protein expression and low-abundance transcripts on the single-cell level. Cell Rep. <i>31</i>, 107499. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref17">View at publisher</a></p>
<p id="bib18">Mair, F. and Tyznik, A.J. (2019). High-dimensional immunophenotyping with fluorescence-based cytometry: a practical guidebook. Methods Mol. Biol. <i>2032</i>, 1-29. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref18">View at publisher</a></p>
<p id="bib19">Melsen, J.E., Van Ostaijen-Ten Dam, M.M., Lankester, A.C., Schilham, M.W., and Van Den Akker, E.B. (2020). A comprehensive workflow for applying single-cell clustering and pseudotime analysis to flow cytometry data. J. Immunol. <i>205</i>, 864-871. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref19">View at publisher</a></p>
<p id="bib20">Perfetto, S.P., Ambrozak, D., Nguyen, R., Chattopadhyay, P., and Roederer, M. (2006). Quality assurance for polychromatic flow cytometry. Nat. Protoc. <i>1</i>, 1522-1530. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref20">View at publisher</a></p>
<p id="bib21">Szalóki, G. and Goda, K. (2015). Compensation in multicolor flow cytometry. Cytometry A <i>87</i>, 982-985. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref21">View at publisher</a></p>
<p id="bib22">Wang, L. and Hoffman, R.A. (2017). Standardization, calibration, and control in flow cytometry. Curr. Protoc. Cytom. <i>79</i>, 1.3.1-1.3.27. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref22">View at publisher</a></p>
<p id="bib23">Wolf, F.A., Angerer, P., and Theis, F.J. (2018). SCANPY: large-scale single-cell gene expression data analysis. Genome Biol. <i>19</i>, 15. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00548-7/sref23">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact Janin Chandra, <a href="mailto:j.chandra@uq.edu.au">j.chandra@uq.edu.au</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The flow cytometry data sets supporting this protocol, and used in Figures, have not been deposited in a public repository but are available from the corresponding author upon request. The Gene Expression Omnibus (GEO) accession number for the CITE sequencing data reported in this article is GEO: <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149544">GSE149544</a> (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149544">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149544</a>).</p>
<h3>Acknowledgments</h3>
<p>This research was carried out at the Translational Research Institute, Woolloongabba, QLD 4102, Australia. The Translational Research Institute is supported by a grant from the <a href="https://doi.org/10.13039/100007622">Australian Government</a>. The authors acknowledge the Translational Research Institute for providing the excellent research environment and core facilities that enabled this research. In particular, we acknowledge staff of the TRI Biological Resource Facility for the excellent care of animals and assistance with animal experimentation, and staff of the TRI Flow Core Facility for their technical assistance, with special thanks to David Sester for flow cytometry support, as well as staff of the UQ-IMB Sequencing Facility for support with the single-cell RNA sequencing. I.H.F. holds an <a href="https://doi.org/10.13039/501100000925">NHMRC</a> Senior Investigator Grant (APP1173927) and an <a href="https://doi.org/10.13039/501100000925">NHMRC</a> Development Grant (APP2000135), and funding from the Merchant Foundation and the Metal Manufacturing Industries. The graphical abstract was created with <a href="http://BioRender.com">BioRender.com</a>.</p>
<h3>Author contributions</h3>
<p>Conceptualization and supervision: S.L., J.C.; data curation: S.L., I.R., S.B.A., J.G., J.C.; formal analysis: S.L., I.R., S.B.A., M.Y., A.M., J.G., J.C.; funding acquisition: I.H.F.; investigation and methodology: S.L., I.R., S.B.A., A.M., M.Y., J.G., and J.C.; project administration: S.L., J.C., and E.E.H.W.; resources: I.H.F. and E.E.H.W.; software: S.L., I.R., J.C., and A.M.; visualization: S.L., I.R., M.Y., A.M., J.G., A.M., and J.C.; writing – original draft: I.R., J.G., S.B.A., and J.C.; writing – review &amp; editing: S.L., I.R., M.Y., J.G., E.E.H.W., I.H.F., and J.C.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests. S.L. is currently an employee of Boehringer-Ingelheim.</p>
</section>
</article>