<article>
<section>
<h1 id="header">An optimized NGS sample preparation protocol for <i>in vitro</i> CRISPR screens</h1>
<p><time datetime="2021-03-19">Published: March 19, 2021</time></p>
<p>Corrin A. Wohlhieter,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a></sup> Fathema Uddin,<sup><a href="#aff2">2</a>,<a href="#fn1">5</a></sup> Àlvaro Quintanal-Villalonga,<sup><a href="#aff2">2</a></sup> John T. Poirier,<sup><a href="#aff3">3</a></sup> Triparna Sen,<sup><a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#cor1">*</a></sup> and Charles M. Rudin<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#fn2">6</a>,<a href="#fn3">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Graduate Program in Pharmacology, Weill Cornell Medicine, New York, NY 10021, USA</p>
<p id="aff2"><sup>2</sup>Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</p>
<p id="aff3"><sup>3</sup>Perlmutter Cancer Center, New York University Langone Health, New York, NY 10016, USA</p>
<p id="aff4"><sup>4</sup>Molecular Pharmacology Program, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</p>
<p id="fn1"><sup>5</sup>These authors contributed equally</p>
<p id="fn2"><sup>6</sup>Technical contact</p>
<p id="fn3"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:sent@mskcc.org">sent@mskcc.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:rudinc@mskcc.org">rudinc@mskcc.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100390">10.1016/j.xpro.2021.100390</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This standardized protocol describes the preparation of PCR amplified and purified samples from human cell lines passaged and collected from CRISPR screening. High-quality samples can be used to perform next-generation sequencing (NGS) to uncover changes in sgRNA abundance from the timepoint at which library-transduced cells are selected to the timepoint when the screen is ended. Here, we describe proper calculation methods for library representation and show how to overcome potential issues often encountered by researchers.</p>
<p>For complete information on the use and execution of this protocol, please refer to <a href="#bib3">Wohlhieter et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Calculate sgRNA representation to achieve adequate depth of coverage</li>
<li>Amplify sequencing regions using optimized PCR primers</li>
<li>Purify PCR samples to pass quality controls for next-generation sequencing</li>
<li>Include proper controls to help troubleshoot issues with sample preparation</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/524-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol assumes that the user has already performed a CRISPR-Cas9 screen <i>in vitro</i> and is prepared to collect cells to process for next-generation sequencing. As an additional resource, our lab has previously published a review article describing the design, execution, and analysis of pooled <i>in vitro</i> CRISPR-Cas9 screens (<a href="#bib1">Miles et al., 2016</a>).</p>
<p>This protocol describes specific steps for preparation of samples for next-generation sequencing (NGS) from human cell lines transduced with a lentiviral sgRNA library. As an example for this protocol, we use the Saturn V CRISPR library containing the lentiGuide-PuroV2 backbone. The Saturn V library is designed as five independent pools (as outlined in <a href="#tbl1">Table 1</a>) with each pool containing gene targets characterized by their “druggability” (<a href="#bib2">Oprea et al., 2018</a>). This protocol can be used for any library containing the lentiGuide-PuroV2 or lentiGuide-Puro backbone. Alternatively, this protocol can be adapted for use with other libraries by designing appropriate forward and reverse primers and calculating guide representation as outlined in <a href="#fig1">Figure 1</a> to ensure that the library of sgRNAs are adequately covered. The genomic DNA (gDNA) yield represented in the calculations (<a href="#fig1">Figure 1</a>) was established and standardized by our laboratory in the human cell line A549 and would need to be established for use in a different model system. Most human cell lines have a similar yield to A549 cells; this protocol was successfully used for NCI-H358, NCI-H292, NCI-H82, NCI-H69, NCI-H526, and DMS114 cell lines.</p>
<table id="tbl1">
<caption>Table 1. Library coverage for Saturn V pools</caption>
<thead>
<tr>
<th>Saturn V Pool #</th>
<th>Number of guides</th>
<th>Library representation</th>
<th>Minimum no. cells for gDNA extraction</th>
<th>Total input genomic DNA required (μg)</th>
<th>Parallel PCR reactions (4 μg gDNA/reaction)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3">1</td>
<td rowspan="3">3,427</td>
<td>177X</td>
<td>760,000</td>
<td>4</td>
<td>1</td>
</tr>
<tr>
<td>530X</td>
<td>2,300,000</td>
<td>12</td>
<td>3</td>
</tr>
<tr>
<td>1061X</td>
<td>4,600,000</td>
<td>24</td>
<td>6</td>
</tr>
<tr>
<td rowspan="3">2</td>
<td rowspan="3">3,208</td>
<td>189X</td>
<td>760,000</td>
<td>4</td>
<td>1</td>
</tr>
<tr>
<td>567X</td>
<td>2,300,000</td>
<td>12</td>
<td>3</td>
</tr>
<tr>
<td>945X</td>
<td>3,800,000</td>
<td>20</td>
<td>5</td>
</tr>
<tr>
<td rowspan="3">3</td>
<td rowspan="3">3,184</td>
<td>190X</td>
<td>760,000</td>
<td>4</td>
<td>1</td>
</tr>
<tr>
<td>571X</td>
<td>2,300,000</td>
<td>12</td>
<td>3</td>
</tr>
<tr>
<td>952X</td>
<td>3,800,000</td>
<td>20</td>
<td>5</td>
</tr>
<tr>
<td rowspan="3">4</td>
<td rowspan="3">1,999</td>
<td>303X</td>
<td>760,000</td>
<td>4</td>
<td>1</td>
</tr>
<tr>
<td>606X</td>
<td>1,500,000</td>
<td>8</td>
<td>2</td>
</tr>
<tr>
<td>910X</td>
<td>2,300,000</td>
<td>12</td>
<td>3</td>
</tr>
<tr>
<td rowspan="3">5</td>
<td rowspan="3">2,168</td>
<td>280X</td>
<td>760,000</td>
<td>4</td>
<td>1</td>
</tr>
<tr>
<td>559X</td>
<td>1,500,000</td>
<td>8</td>
<td>2</td>
</tr>
<tr>
<td>1118X</td>
<td>3,000,000</td>
<td>16</td>
<td>4</td>
</tr>
</tbody>
</table>
<p class="table-legend">The following table lists the library representation covered by one PCR reaction per pool and gives 2 additional examples for the number of PCR reactions required to cover a certain library representation. These values can be calculated as described in <a href="#fig1">Figure 1</a> and can be adopted accordingly for any library. We recommend processing samples to cover a library representation of at least 300X or higher for high-quality NGS products (<a href="#bib1">Miles et al., 2016</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/524-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Example calculations for Saturn V library pool 1</div>
<p>This calculation is used to obtain the library representation present in one PCR sample. The color-coded variables describe the purpose of each value. Numbers in this example assume the use of human cell lines and Saturn V library pool 1 but can be adapted to other experimental conditions.</p>
</figcaption>
</figure>
<h3 id="sec1.1">Decontaminate PCR workstation</h3>
<p>PCR preparation should be performed inside a decontaminated PCR workstation to avoid cross-contamination between unique samples.</p>
<ol>
<li>Spray the PCR workstation area with RNase AWAY, which eliminates DNA in addition to RNases, and thoroughly wipe.</li>
<li>Autoclave microcentrifuge and PCR tubes and pipette tips before placing them in the PCR workstation.</li>
<li>Place all needed microcentrifuge tubes and/or PCR tubes on racks inside the PCR workstation with the caps open.</li>
<li>Turn on UV light for at least 20 min to decontaminate workstation.</li>
<li>Turn off UV light and proceed to gDNA extraction.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend performing gDNA extraction, PCR setup, and template addition in the decontaminated workstation using separate pipettes to avoid contamination.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>PureLink Genomic DNA Mini Kit</td>
<td>Invitrogen</td>
<td>K1820-01</td>
</tr>
<tr>
<td>Qubit dsDNA BR Assay Kit</td>
<td>Invitrogen</td>
<td>Q32853</td>
</tr>
<tr>
<td>Qubit dsDNA HS Assay Kit</td>
<td>Invitrogen</td>
<td>Q32854</td>
</tr>
<tr>
<td>Qubit Assay Tubes</td>
<td>Invitrogen</td>
<td>32856</td>
</tr>
<tr>
<td>PCR tubes</td>
<td>Thermo Scientific</td>
<td>AB-0266</td>
</tr>
<tr>
<td>RNase <i>AWAY</i>
</td>
<td>Molecular BioProducts</td>
<td>7002</td>
</tr>
<tr>
<td>Qubit dsDNA BR Assay Kit</td>
<td>Invitrogen</td>
<td>Q32850</td>
</tr>
<tr>
<td>GeneJET PCR Purification Kit</td>
<td>Thermo Scientific</td>
<td>K0702</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>NGS-adapted forward and reverse primers with barcodes</td>
<td>In house</td>
<td>N/A</td>
</tr>
<tr>
<td>Exonuclease I</td>
<td>New England Biolabs</td>
<td>M0293L</td>
</tr>
<tr>
<td>Herculase reagents</td>
<td>Agilent Technologies</td>
<td>600675</td>
</tr>
<tr>
<td>Saturn V Library (or other sgRNA lentiviral library)</td>
<td>In house</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>PCR workstation</td>
<td>AirClean Systems</td>
<td>AC624LFUV</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>This protocol begins with the completion of cell passaging and collection from an <i>in vitro</i> CRISPR screen using human cell lines. Be sure to collect enough cells from each sample to meet the intended library representation (<a href="#fig1">Figure 1</a>). At least 16 cell doublings is recommended to ensure changes in sgRNA representation will be captured. These changes may show sgRNA depletion (negative selection screen) or sgRNA enrichment (positive selection screen). The number of doublings may change if the screen uses additional variables like pharmacological agents.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">gDNA extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–4 h</div>
<p>The following procedure is used to harvest gDNA from cells that have undergone CRISPR screening. Cells that have been transduced with a lentiviral library have integrated genomic DNA containing an sgRNA sequence and adaptor sequences complimentary to the forward and reverse primers (<a href="#fig2">Figure 2</a>). The harvested gDNA will be used later for one-step PCR sample preparation for NGS. Refer to <a href="#tbl1">Table 1</a> to obtain the minimum number of cells required for gDNA extraction for your selected library coverage.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/524-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Forward and reverse primer design</div>
<p>Forward PCR primers were designed to create a stagger sequence between the priming site and the Illumina sequence to facilitate diversity during NGS reads. Ten forward primers are pooled in each reaction with a stagger length of 1-10 bases. Reverse primers are designed to introduce a unique barcode to each sample. This barcode is used during analysis to identify individual samples within a pool. Details on primer design can be found in <a href="#tbl2">Tables 2</a> and <a href="#tbl3">3</a>.</p>
</figcaption>
</figure>
<ol>
<li>Harvest and centrifuge the selected number of cells (<a href="#tbl1">Table 1</a>) in 1.5 mL microcentrifuge tubes at 300 × <i>g</i> for 3 min at 20°C. Do not pellet more than 5 million cells per microcentrifuge tube. Use an automated cell counter or equivalent to count your cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The maximum capacity for PureLink Genomic DNA spin columns is ∼5 million cells for gDNA extraction.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> You may store dry cell pellets at −80°C for up to 1 month or proceed to gDNA extraction.</div>
<ol start="2">
<li>Use PureLink Genomic DNA extraction kit following manufacturer’s protocol to extract gDNA from cell pellets. Elute in Molecular Grade Water in final step. Please refer to the manufacturer’s instructions for gDNA extraction using the following link: (<a href="http://tools.thermofisher.com/content/sfs/manuals/purelink_genomic_man.pdf">http://tools.thermofisher.com/content/sfs/manuals/purelink_genomic_man.pdf</a>)</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not process more than 5 million cells per column as this may cause the spin columns to clog and will decrease your yield. If more than 5 million cells is required to meet the depth of library coverage, extract gDNA in multiple spin columns and pool the gDNA after extraction.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that there is no remaining volume of wash buffer above spin-column filters after centrifugation during washing steps. Repeat centrifuge step if there is a remaining volume of wash buffer above the spin column. Ethanol contamination from wash buffers will decrease your gDNA yield during the elution step.</div>
<div class="note">
<span class="note-title">Note:</span> Aim for a final concentration of at least 190 ng/μL (this is the minimum concentration needed to input 4 μg of gDNA into a single 50 μL PCR reaction). Generally, eluting in 50 μL Molecular Grade Water for 5 million processed cells yields concentrations &gt;200 ng/μL.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Perform a second elution with 20–30 μL additional Molecular Grade Water to recover more gDNA.</div>
<ol start="3">
<li>Obtain concentrations of extracted gDNA with Qubit dsDNA BR Assay Kit following manufacturer’s protocol (<a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/Qubit_dsDNA_BR_Assay_UG.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/Qubit_dsDNA_BR_Assay_UG.pdf</a>)</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> You may store gDNA samples at −20°C for the duration of your experiment, or proceed to one-step PCR sample preparation. Your gDNA samples can be stored at −20°C for over 10 years.</div>
<h3 id="sec3.2">One-step PCR sample preparation for NGS</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 h</div>
<p>The following procedure is for one-step PCR NGS sample preparation. Forward primers are designed to include a priming site adjacent to the guide spacer sequence and introduces the P5 Illumina adapter, as well as stagger sequences to allow for diversity during NGS reads. Reverse primers are designed to include a priming site adjacent to the UMI sequence and introduces the P7 Illumina adaptor as well as a unique barcode to allow for multiplexed NGS (<a href="#fig2">Figure 2</a>).</p>
<ol start="4">
<li>Thaw frozen PCR reagents.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Thaw 5× Herculase buffer and dNTP Mix on top of ice and keep on ice after they are thawed. All other PCR components (other than Herculase) can be thawed at 20°C.</div>
<div class="note">
<span class="note-title">Note:</span> Keep Herculase on ice or at −20°C until needed.</div>
<div class="note">
<span class="note-title">Note:</span> DMSO takes roughly 30–40 min to thaw at 20°C. You may aliquot DMSO into smaller volumes to save time thawing.</div>
<ol start="5">
<li>Mix equal volumes of each 10 μM forward primer 1–10 to make a pool of all 10 forward primers and use this during PCR setup.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each of the forward primers is one base shorter than the next (<a href="#tbl2">Tables 2</a> and <a href="#tbl3">3</a>). The different length primers improves NGS reads by adding diversity to the reads.</div>
<table id="tbl2">
<caption>Table 2. Forward PCR primer design</caption>
<thead>
<tr>
<th>Primer</th>
<th>Stagger</th>
<th>Full primer sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>NGS-Fwd-1</td>
<td>T</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>T</b>AAGTAGAGGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-2</td>
<td>AT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>AT</b>CATGCTTAGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-3</td>
<td>GAT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>GAT</b>GCACATCTGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-4</td>
<td>CGAT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>CGAT</b>TGCTCGACGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-5</td>
<td>TCGAT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>TCGAT</b>AGCAATTCGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-6</td>
<td>ATCGAT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>ATCGAT</b>AGTTGCTTGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-7</td>
<td>GATCGAT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>GATCGAT</b>CCAGTTAGGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-8</td>
<td>CGATCGAT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>CGATCGAT</b>TTGAGCCTGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-9</td>
<td>ACGATCGAT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>ACGATCGAT</b>ACACGATCGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
<tr>
<td>NGS-Fwd-10</td>
<td>TACGATCGAT</td>
<td>AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT<b>TACGATCGAT</b>GGTCCAGAGCTTTATATATCTTGTGGAAAGGACGAAACACC</td>
</tr>
</tbody>
</table>
<table id="tbl3">
<caption>Table 3. Reverse PCR primer design</caption>
<thead>
<tr>
<th>Primer</th>
<th>Barcode</th>
<th>Full primer sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>KO-Rev-1</td>
<td>TCGCCTTG</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>TCGCCTTG</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-2</td>
<td>ATAGCGTC</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>ATAGCGTC</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-3</td>
<td>GAAGAAGT</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>GAAGAAGT</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-4</td>
<td>ATTCTAGG</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>ATTCTAGG</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-5</td>
<td>CGTTACCA</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>CGTTACCA</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-6</td>
<td>GTCTGATG</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>GTCTGATG</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-7</td>
<td>TTACGCAC</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>TTACGCAC</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-8</td>
<td>TTGAATAG</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>TTGAATAG</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-9</td>
<td>TATAGCCT</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>TATAGCCT</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
<tr>
<td>KO-Rev-10</td>
<td>ATAGAGGC</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>ATAGAGGC</b>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTAAGATCTAGTTACGCCAAGCTTAAA</td>
</tr>
</tbody>
</table>
<ol start="6">
<li>Set Up PCR master mixes and negative control reactions for each sample (<a href="#tbl4">Table 4</a>).
<table id="tbl4">
<caption>Table 4. PCR setup</caption>
<thead>
<tr>
<th>PCR component</th>
<th>Single PCR reaction</th>
<th>Master Mix example (10 parallel PCR reactions)</th>
<th>Master Mix example for negative control (10 PCR reactions)</th>
</tr>
</thead>
<tbody>
<tr>
<td>gDNA</td>
<td>4 μg</td>
<td>40 μg</td>
<td>-</td>
</tr>
<tr>
<td>5× Herculase Buffer</td>
<td>10 μL</td>
<td>100 μL</td>
<td>100 μL</td>
</tr>
<tr>
<td>2.5 mM dNTP Mix</td>
<td>4 μL</td>
<td>40 μL</td>
<td>40 μL</td>
</tr>
<tr>
<td>Pooled Forward Primers (10 μM)</td>
<td>4.5 μL</td>
<td>45 μL</td>
<td>45 μL</td>
</tr>
<tr>
<td>Unique Reverse Primer (10 μM)</td>
<td>4.5 μL</td>
<td>45 μL</td>
<td>-</td>
</tr>
<tr>
<td>DMSO</td>
<td>3.75 μL</td>
<td>37.5 μL</td>
<td>37.5 μL</td>
</tr>
<tr>
<td>Herculase</td>
<td>2 μL</td>
<td>20 μL</td>
<td>20 μL</td>
</tr>
<tr>
<td>Molecular Grade Water</td>
<td>Bring volume up to 50 μL</td>
<td>Bring volume up to 500 μL</td>
<td>212.5 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">For the negative control master mix, unique reverse primers and template gDNA are not included. Instead, reverse primers are added to each reaction separately for each unique reverse primer used for the negative control PCR reactions. gDNA template is not included in the negative control reactions to ensure that PCR components are not contaminated.</p>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> There should be a negative control for every reverse primer used (i.e., A PCR setup for 10 samples with 10 unique reverse primers should also have 10 negative controls, with at least one negative control for each unique reverse primer). gDNA template is not included in the negative control reactions to ensure that PCR components are not contaminated.</div>
<ol start="7">
<li>Vortex master mixes for 15–20 s and spin down.</li>
<li>Aliquot 50 μL of the master mix to respectively labeled tubes.
<ol type="a">
<li>For negative controls, aliquot 45.5 μL of negative control master mix and add 4.5 μL of 10 μM unique reverse primer to respectively labeled tubes and vortex.</li>
<li>Spin down tubes at maximum speed for 10 s to ensure that all the liquid is at the bottom of the tube.</li>
</ol>
</li>
<li>Proceed to thermocycling (see <a href="#tbl5">Table 5</a>). The Denature, annealing, and extension steps should occur in order for 24 total cycles.
<table id="tbl5">
<caption>Table 5. Thermocycling conditions for one-step PCR</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>98°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>24</td>
</tr>
<tr>
<td>Annealing</td>
<td>52°C</td>
<td>30 s</td>
<td>24</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>45 s</td>
<td>24</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>infinite</td>
<td> </td>
</tr>
</tbody>
</table>
</li>
<li>Pool parallel PCR reactions and vortex to thoroughly mix (i.e., pool all PCR reactions with Sample A + Rev 1 primer and all reactions with Sample B + Rev 2 primer).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> You can store pooled PCR products at −20°C or proceed.</div>
<ol start="11">
<li>Run 5 μL of pooled PCR products and negative controls on a 1.5% TAE gel to confirm successful PCR. Expected band size should be approximately 350 bp.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step can be performed in parallel with step 3 in purification of amplicons and gel quality control procedure.</div>
<div class="note">
<span class="note-title">Note:</span> Unused primers are usually visible on the gel at this stage (approximately 100 bp). These primers will be digested with Exonuclease I treatment during purification steps (see <a href="#sec3.3">purification of amplicons and gel quality control test</a> section).</div>
<div class="note">
<span class="note-title">Note:</span> A primer dimer band may appear on the gel in negative control lanes (approximately 200 bp). This is acceptable only if primer dimers are not present in reactions that receive DNA template.</div>
<ol start="12">
<li>Transfer half or less of your pooled reaction in a separate labeled microcentrifuge tube to process for purification. Store remainder of the unpurified products at −20°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The quantity of PCR products far exceeds the amount needed to submit for NGS. We recommend taking a fraction of this product to process for purification steps so that there is plenty of unpurified samples to work from in case errors are made during purification.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> You can store the aliquoted pooled PCR products at −20°C or proceed to purification steps.</div>
<h3 id="sec3.3">Purification of amplicons and gel quality control test</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 h</div>
<p>The following procedure describes how to purify the PCR products before submission for NGS. Free single-stranded primers, which carry P5/P7 adapter sequences and can bind the Illumina flow cell during NGS, are removed through Exonuclease I digestion. Exonuclease I will degrade any single-stranded oligonucleotides, including free primers remaining after the PCR. The PCR products are then further purified to remove PCR components through spin-column purification using the GeneJET PCR Purification Kit.</p>
<ol start="13">
<li>Exonuclease I digest - Add 1 μL Exonuclease I for every 10 μL of reaction (i.e., Add 10 μL Exonuclease I for 100 μL of reaction).</li>
<li>Incubate reactions at 37°C for 1 h followed by a 20 min incubation at 80°C (see <a href="#tbl6">Table 6</a>).
<table id="tbl6">
<caption>Table 6. Thermocycling conditions for Exonuclease I digestion</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Enzyme activation</td>
<td>37°C</td>
<td>60 min</td>
<td>1</td>
</tr>
<tr>
<td>Enzyme deactivation</td>
<td>80°C</td>
<td>20 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>infinite</td>
<td> </td>
</tr>
</tbody>
</table>
</li>
<li>Spin-column purify digested PCR products using the GeneJET PCR Purification Kit following manufacturer’s protocol (<a href="https://assets.fishersci.com/TFS-Assets/LSG/manuals/MAN0012662_GeneJET_PCR_Purification_UG.pdf">https://assets.fishersci.com/TFS-Assets/LSG/manuals/MAN0012662_GeneJET_PCR_Purification_UG.pdf</a>). Elute in 50 μL Molecular Grade Water in the final step.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> There is an optional step that recommends adding 100% isopropanol to increase yield of products ≤500 bp. Add 100% isopropanol as suggested in the manufacturer’s protocol (size of amplicon of interest is ∼350 bp).</div>
<ol start="16">
<li>Prepare 5 μL of purified samples to run on a 1.5% Tris-Acetate-EDTA (TAE) gel. There should be no visible primers on the gel at this stage (see <a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/524-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Gel quality control test of unpurified and purified PCR products</div>
<p>One negative control for each independent sample (6 total) was run side by side with the unpurified and purified samples obtained in this protocol. Expected PCR band size is approximately 350 bp. Exonuclease I digest removes free primers from the PCR product. Negative control lanes should not have any visible bands other than free primers.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A faint band higher than the PCR product often appears on the gel after spin-column purification. This should not cause any sequencing problems (samples with this band have been submitted for NGS with no issues in quality). We speculate that the higher band is degraded gDNA that is carried over after purification. However, this DNA will not have the Illumina adapter sequences that were introduced via PCR and therefore will not influence NGS reads.</div>
<div class="note">
<span class="note-title">Note:</span> The expected band after purification may fall slightly higher than prior to purification on the agarose gel. This is likely due to slight changes in ionic concentration after purification affecting apparent molecular weight and should not provoke any concerns.</div>
<ol start="17">
<li>Quantify the purified samples with the Qubit dsDNA HS Assay Kit using manufacturer’s protocol (<a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/Qubit_dsDNA_HS_Assay_UG.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/Qubit_dsDNA_HS_Assay_UG.pdf</a>).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples can be stored at −20°C for no more than 5 months until ready to submit for NGS.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p><a href="#fig3">Figure 3</a> shows a representative gel with unpurified and purified PCR amplicons. If the purified samples pass this quality control step, they are ready to submit for NGS. Our lab uses Genewiz for NGS, but it is up to the researcher to choose an NGS provider. Typically, a provider of NGS services will perform one or more of the following quality control tests on samples: Tapestation, Qubit concentration reading, or quantitative PCR. This protocol produces 50 μL of purified sample and Genewiz requests 20 μL of purified sample to run quality control tests and to load onto the flow cell. Contact the NGS provider of your choice for details on sample submission requirements.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is only applicable and successful for cells that have been effectively transduced and selected for expression of the sgRNA library of interest. Cells that have not been transduced properly will not produce PCR products. This protocol is designed to be used for libraries containing the LentiGuide-Puro or LentiGuide-PuroV2 backbone, but may be adapted for use with other libraries by designing appropriate forward and reverse primers and calculating guide representation as outlined.</p>
<p>The gDNA yield represented in the calculations was established by our group in human cell lines but would need to be standardized separately in a different model system.</p>
<p>Variability in thermocycler efficiency may affect the product obtained using this protocol. This protocol was intended to be completed in a decontaminated PCR workstation. Performing this protocol on a standard benchtop may increase the likelihood of contamination and decrease the purity of the samples.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low gDNA yield during gDNA extraction (step 3 in <a href="#sec3.1">gDNA extraction</a>).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Perform a second elution step to recover more gDNA from spin column (see “<a href="#disp0045">Optional</a>” step in step 2 of <a href="#sec3.1">gDNA extraction</a>). Potential causes of low gDNA yields include clogged filters from processing too many cells or ethanol contamination during washing steps. Ensure that there is no remaining wash buffer above spin column after washes. You may perform an additional 1-min spin centrifuge at maximum speed prior to eluting in water to remove any residual wash buffer.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Concentration of gDNA is &lt;190 ng/μL and 4 μg of template cannot fit a 50 μL PCR reaction set up (step 3 in one-step PCR sample preparation for NGS).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>You may input 2 μg of template instead of 4 μg per reaction. However, you will need to double the number of PCR reactions per sample to capture the same library coverage.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>PCR fails and there is no visible band of expected size on the agarose gel (step 8 in one-step PCR sample preparation for NGS).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Ethanol contamination in gDNA solution can inhibit the PCR reaction. One indication of ethanol contamination is if the gDNA in solution does not freeze at −20°C. gDNA in pure water will freeze at this temperature. You may repeat PCR reaction using 2 μg input template instead of 4 μg to dilute the contaminating ethanol in the PCR set up and reduce its inhibitory effects in the reaction.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Free primers remain in the purified PCR products and is visible on the agarose gel (step 4 in purification of amplicons and gel quality control test).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>You may repeat Exonuclease I digestion and re-purify samples using spin-column purification.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Unexpected PCR products in negative control lanes.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>PCR components, pipettes, or consumables are contaminated with plasmid or PCR products. Make fresh aliquots of reagents and decontaminate work areas.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Miles, L., Garippa, R., and Poirier, J. (2016). Design, execution, and analysis of pooled in vitro CRISPR/Cas9 screens. FEBS J. <i>283</i>, 3170-3180. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00097-6/sref1">View at publisher</a></p>
<p id="bib2">Oprea, T.I., Bologa, C.G., Brunak, S., Campbell, A., Gan, G.N., Gaulton, A., Gomez, S.M., Guha, R., Hersey, A., Holmes, J., et al. (2018). Unexplored therapeutic opportunities in the human genome. Nat. Rev. Drug Discov. <i>17</i>, 317-332. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00097-6/sref2">View at publisher</a></p>
<p id="bib3">Wohlhieter, C.A., Richards, A.L., Uddin, F., Hulton, C.H., Quintanal-Villalonga, À., Martin, A., de Stanchina, E., Bhanot, U., Asher, M., Shah, N.S., et al. (2020). Concurrent mutations in STK11 and KEAP1 promote ferroptosis protection and SCD1 dependence in lung cancer. Cell Rep. <i>33</i>, 108444. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00097-6/sref3">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Charles M. Rudin (<a href="mailto:rudinc@mskcc.org">rudinc@mskcc.org</a>).</p>
<h4>Materials Availability</h4>
<p>Unique reverse barcoding primers were adapted for use in this protocol. These primers can be requested by contacting the lead contact, Charles M. Rudin (refer to the primer sequence <a href="#tbl3">Table 3</a>). The Saturn V library referenced in this protocol can be requested by contacting the lead contact, Charles M. Rudin.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work from this project was supported by an <a href="https://doi.org/10.13039/100000002">NIH</a> T32 training grant (C.A.W.), the Druckenmiller Center for Lung Cancer Research (T.S.), <a href="https://doi.org/10.13039/100014547">Parker Institute for Cancer Immunotherapy</a> grant (T.S.), <a href="https://doi.org/10.13039/100005923">International Association for the Study of Lung Cancer</a> grant (T.S.), and <a href="https://doi.org/10.13039/100000054">NCI</a> U01 CA199215 (C.M.R.).</p>
<h3>Author Contributions</h3>
<p>Conceptualization, C.A.W., F.U., J.T.P., and C.M.R.; Methodology, C.A.W., F.U., and A.Q.V.; Resources, J.T.P.; Writing – Original Draft, F.U., C.A.W., T.S., and C.M.R.; Visualization, C.A.W., F.U., and J.T.P.; Supervision, T.S. and C.M.R.; Funding Acquisition, C.M.R., T.S., and C.A.W.</p>
<h3>Declaration of Interests</h3>
<p>C.M.R. serves on the Scientific Advisory Boards of Bridge Medicines and Harpoon Therapeutics.</p>
</section>
</article>