<article>
<section>
<h1 id="header">Protocol to profile the bioenergetics of organoids using Seahorse</h1>
<p><time datetime="2021-03-19">Published: March 19, 2021</time></p>
<p>Marlies C. Ludikhuize,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Maaike Meerlo,<sup><a href="#aff1">1</a></sup> Boudewijn M.T. Burgering,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Maria J. Rodríguez Colman<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Molecular Cancer Research, Center for Molecular Medicine, University Medical Center Utrecht, Heidelberglaan 100, 3584 CG Utrecht, the Netherlands</p>
<p id="aff2"><sup>2</sup>Oncode Institute</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:m.c.ludikhuize-2@umcutrecht.nl">m.c.ludikhuize-2@umcutrecht.nl</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:m.j.rodriguezcolman@umcutrecht.nl">m.j.rodriguezcolman@umcutrecht.nl</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100386">10.1016/j.xpro.2021.100386</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Addressing bioenergetics is key to evaluate the impact of metabolism on the regulation of biological processes and its alteration in disease. Organoids are <i>in vitro</i> grown self-organizing structures derived from healthy and diseased tissue that recapitulate with high fidelity the tissue of origin. Bioenergetics is commonly analyzed by Seahorse XF analysis. However, its application to organoid studies is technically challenging. Here, we share our in-house optimized protocols to examine organoid bioenergetics in response to drugs, gene knockdown, or to characterize the metabolism of specific cell types.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib3">Ludikhuize et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Analysis of bioenergetics in organoids is technically challenging</li>
<li>We share a detailed protocol to overcome the pitfalls of Seahorse analysis in organoids</li>
<li>Accurate timing of organoid re-plating improves the robustness of Seahorse analysis</li>
<li>DNA-based normalization overcomes organoid-related limitations for normalization</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/521-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Organoids closely recapitulate the <i>in vivo</i> mechanisms of epithelial homeostasis and harbor all different cell types of the tissue of origin (<a href="#bib11">Schutgens and Clevers, 2020</a>). Importantly, organoids are rapidly gaining ground as research tools and are used in a wide range of scientific disciplines including fundamental biology, hereditary diseases, and cancer (<a href="#bib12">Kim et al., 2020</a>). The analysis of bioenergetics is key to better understand, and potentially target, the supporting role of metabolism in such processes. Glycolysis in the cytosol and oxidative phosphorylation in the mitochondria produce energy in the form of ATP, at different rates and with different requirements and side-products. While in the absence of oxygen cells primarily convert the glycolytic product pyruvate into lactate, under aerobic conditions, pyruvate can fuel the TCA cycle and mitochondrial respiration. Interestingly, most actively proliferating cells produce and secrete lactate independently of oxygen availability, which is known as aerobic glycolysis and defines the “Warburg effect” in cancer cells (<a href="#bib13">Vander Heiden et al., 2009</a>). Medium acidification by lactate secretion and oxygen consumption by mitochondrial respiration can be used to monitor the activity of these metabolic pathways. In brief, Seahorse-based bioenergetics analysis is based on the simultaneous and over time measurements of the oxygen consumption rate (OCR) and extracellular acidification rate (ECAR) as read-outs of mitochondrial respiration and glycolysis, respectively. Changes in OCR and ECAR are measured in response to the presence of metabolic substrates and inhibitors and serve to calculate several bioenergetic parameters. In this manuscript, we share our protocol to measure bioenergetics in organoids (<a href="#fig1">Figure 1</a>A). Here we focus on the most frequently used tests, the mitochondrial and glycolysis stress tests (<a href="#fig1">Figures 1</a>B and 1C). We provide detailed information on how to successfully perform these analyses in organoids derived from mouse small intestine, human colon, and colorectal tumors. Our optimized protocols include how to prepare organoids for analysis, how to adjust the assay medium composition and the concentrations of substrates and inhibitors, the specific settings for the XF Analyzer, and how to normalize the measurements. These protocols have broad applications, such as metabolic profiling of specific cell types, determination of the metabolic changes upon stem cell differentiation or addressing metabolic programs resulting from genetic mutations (<a href="#fig1">Figure 1</a>A). We have listed a number of interesting applications developed by others and us in <a href="#tbl1">Table 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/521-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Experimental setup and mitochondrial and glycolysis stress test parameters</div>
<p>(A) Experimental setup for profiling bioenergetics in intestine-derived organoids. This setup can be used to analyze organoid bioenergetics upon various treatments. Examples are (i) short treatment with metabolic inhibitors that are injected during the assay, (ii) incubation of organoids in the presence of different metabolic substrates such as glucose vs palmitate in the assay medium, (iii) induction or gene knockdown after re-plating organoids for analysis, or (iv) induction of organoid differentiation or cell-type enrichment which is already started prior to re-plating the organoids for analysis. ∗For other organoid types or specific differentiation protocols, culturing for longer than 10 days might be required (B and C). Setup of and respiratory parameters measured during (B) a mitochondrial stress test or (C) a glycolysis stress test.</p>
</figcaption>
</figure>
<table id="tbl1">
<caption>Table 1. Different applications for profiling bioenergetics in organoids</caption>
<thead>
<tr>
<th>Application</th>
<th>Example</th>
<th>Treatment</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr>
<td>Metabolic characterization of specific cell types</td>
<td>Determination of stem cell or enterocyte metabolism of mSIOs</td>
<td>Adjustment of medium composition to enrich organoids with stem cells or with enterocytes</td>
<td>(<a href="#bib2">Lindeboom et al., 2018</a>), (<a href="#bib6">Okkelman et al., 2020</a>)</td>
</tr>
<tr>
<td rowspan="2">Metabolic profiling during differentiation.</td>
<td>Metabolic profiling of proliferative vs. differentiating mSIOs</td>
<td>Adjustment of medium composition to prevent or enhance differentiation of organoids</td>
<td>(<a href="#bib7">Rodríguez-Colman et al., 2017</a>)</td>
</tr>
<tr>
<td>Metabolic profiling during maturation of human cardiac organoids</td>
<td>Adjustment of the medium to induce human heart organoid maturation</td>
<td>(<a href="#bib4">Mills et al., 2017</a>)</td>
</tr>
<tr>
<td rowspan="6">Determining the effect or effectiveness of metabolic drugs on bioenergetics.</td>
<td>Determination of the contribution of specific metabolic pathways to energy metabolism of mSIOs</td>
<td>Treatment with DCA, Piercidin, Azide, UK5099, Phloretin, PCMB, or Antimycin A</td>
<td>(<a href="#bib7">Rodríguez-Colman et al., 2017</a>)</td>
</tr>
<tr>
<td>Determination of contribution of Mdivi-1 on energy metabolism in mSIOs</td>
<td>Mdivi-1 treatment</td>
<td>(<a href="#bib3">Ludikhuize et al., 2020</a>)</td>
</tr>
<tr>
<td>Determination of effectiveness complex I inhibitor rotenone in mSIOs</td>
<td>Rotenone treatment</td>
<td>(<a href="#bib3">Ludikhuize et al., 2020</a>)</td>
</tr>
<tr>
<td>Determination of FA oxidation in human heart organoids</td>
<td>Etomoxir treatment</td>
<td>(<a href="#bib4">Mills et al., 2017</a>)</td>
</tr>
<tr>
<td>Determination of FA oxidation mSIOs</td>
<td>Etomoxir treatment</td>
<td>(<a href="#bib8">Schell et al., 2017</a>)</td>
</tr>
<tr>
<td>Determination of bioenergetics upon metformin treatment in CRC PDX-derived organoids</td>
<td>Metformin treatment</td>
<td>(<a href="#bib5">Mohamed Suhaimi et al., 2017</a>)</td>
</tr>
<tr>
<td rowspan="2">Assessment of substrates for mitochondrial respiration.</td>
<td>Determination of mitochondrial respiration upon lactate supplementation in mSIOs</td>
<td>Glucose vs lactate supplementation</td>
<td>(<a href="#bib7">Rodríguez-Colman et al., 2017</a>)</td>
</tr>
<tr>
<td>Determination of FA oxidation with palmitate as substrate in mSIOs</td>
<td>Palmitate supplementation</td>
<td>(<a href="#bib8">Schell et al., 2017</a>)</td>
</tr>
<tr>
<td rowspan="2">Profiling bioenergetics upon gene knockdown.</td>
<td>Examining the role of FOXO on mSIO bioenergetics</td>
<td>Doxycycline-induced FOXO1/3 KD</td>
<td>(<a href="#bib3">Ludikhuize et al., 2020</a>)</td>
</tr>
<tr>
<td>Determining the effect of mitochondrial fission on mSIO bioenergetics</td>
<td>Doxycycline-induced Drp1 KD</td>
<td>(<a href="#bib3">Ludikhuize et al., 2020</a>)</td>
</tr>
<tr>
<td>Profiling bioenergetics upon gene knockout.</td>
<td>Examining the role of MPC1 on mSIO bioenergetics</td>
<td>Stem cell-specific MPC1 KO</td>
<td>(<a href="#bib8">Schell et al., 2017</a>)</td>
</tr>
<tr>
<td>Determining activity of O<sub>2</sub>-producing or consuming enzymes.</td>
<td>Examining the oxygen consumption by the enzymes LbNOX or TpNOX in HeLa cells</td>
<td>Doxycycline-induced expression of LbNOX or TpNOX</td>
<td>(<a href="#bib1">Cracan et al., 2017</a>)</td>
</tr>
</tbody>
</table>
<p class="table-legend">Abbreviations: mSIO: mouse small intestinal organoid, CRC: colorectal cancer, DCA: dichloroacetate, Drp1: Dynamin-Related Protein 1, FOXO: Forkhead Box O transcription factor, LbNOX: <i>Lactobacillus brevis</i> H<sub>2</sub>O-forming NADH oxidase, Mdivi-1: mitochondrial division inhibitor 1, MPC1: Mitochondrial pyruvate carrier 1, PCMB: p-chloromercuribenzoate, PDX: patient-derived xenograft, TpNOX: triphosphopyridine nucleotide oxidase.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>2-Deoxyglucose</td>
<td>Sigma-Aldrich</td>
<td>Cat# D8375, CAS 154-17-6</td>
</tr>
<tr>
<td>Advanced DMEM/F-12</td>
<td>Life Technologies</td>
<td>Cat# 12634-010</td>
</tr>
<tr>
<td>Antimycin A</td>
<td>Sigma-Aldrich</td>
<td>Cat# A8674, CAS 1397-94-0</td>
</tr>
<tr>
<td>Cell recovery solution</td>
<td>Corning</td>
<td>Cat# 354253</td>
</tr>
<tr>
<td>DMSO</td>
<td>VWRC</td>
<td>Cat# 23488.294 CAS 67-68-5</td>
</tr>
<tr>
<td>FCCP</td>
<td>Sigma-Aldrich</td>
<td>Cat# C2920, CAS 370-86-5</td>
</tr>
<tr>
<td>Glucose</td>
<td>Merck Millipore</td>
<td>Cat# 1.08337.1000, CAS 50-99-7</td>
</tr>
<tr>
<td>Glutamax</td>
<td>Life Technologies</td>
<td>Cat# 35050087</td>
</tr>
<tr>
<td>HEPES</td>
<td>Life Technologies</td>
<td>Cat# 15630-056, CAS 7365-45-9</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Lonza</td>
<td>Cat# 17-605E</td>
</tr>
<tr>
<td>Matrigel growth factor matrix</td>
<td>Corning</td>
<td>Cat# 356231</td>
</tr>
<tr>
<td>NaOH</td>
<td>Merck Millipore</td>
<td>Cat# 1.06498.1000, CAS #: 1310-73-2</td>
</tr>
<tr>
<td>Oligomycin</td>
<td>Sigma-Aldrich</td>
<td>Cat# 75351, CAS 579-13-5</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>Lonza</td>
<td>Cat# DE17602E</td>
</tr>
<tr>
<td>Pyruvate</td>
<td>Sigma-Aldrich</td>
<td>Cat# S8636, CAS 113-24-6</td>
</tr>
<tr>
<td>Rotenone</td>
<td>Sigma-Aldrich</td>
<td>Cat# R8875, CAS 83-79-4</td>
</tr>
<tr>
<td>Seahorse XF base medium</td>
<td>Agilent</td>
<td>Cat# 102353-100</td>
</tr>
<tr>
<td>Seahorse XF Calibrant</td>
<td>Agilent</td>
<td>Cat# 102353-100</td>
</tr>
<tr>
<td>TrypLE Express Enzyme (1×) phenol red</td>
<td>Sigma-Aldrich</td>
<td>Cat# 12605010</td>
</tr>
<tr>
<td>Trypsin solution</td>
<td>Sigma-Aldrich</td>
<td>Cat# T3924</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>QIAamp DNA Micro Kit</td>
<td>Qiagen</td>
<td>Cat# 56304</td>
</tr>
<tr>
<td>Seahorse XF24 V28 PS cell culture microplates</td>
<td>Agilent</td>
<td>Cat# 100882-004</td>
</tr>
<tr>
<td>Seahorse XFe24 analyzer</td>
<td>Agilent</td>
<td>420017</td>
</tr>
<tr>
<td>Seahorse XFe24 FluxPak</td>
<td>Agilent</td>
<td>Cat# 102340-100</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>sh<i>Foxo1/3</i> mouse small intestinal organoids</td>
<td>(<a href="#bib3">Ludikhuize et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>P9 CRC-derived organoids</td>
<td>(<a href="#bib9">van de Wetering et al., 2015</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>P16 CRC-derived organoids</td>
<td>(<a href="#bib9">van de Wetering et al., 2015</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>P19b CRC-derived organoids</td>
<td>(<a href="#bib9">van de Wetering et al., 2015</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Human WT colon organoids</td>
<td><a href="#bib10">(Drost et al., 2015)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism 8</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Wave Desktop and Controller 2.6 Software</td>
<td><a href="https://www.agilent.com/en/product/cell-analysis/real-time-cell-metabolic-analysis/xf-software/seahorse-wave-desktop-software-740897">https://www.agilent.com/en/product/cell-analysis/real-time-cell-metabolic-analysis/xf-software/seahorse-wave-desktop-software-740897</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>EVOS M5000 imaging system</td>
<td>Invitrogen</td>
<td>AMF5000</td>
</tr>
<tr>
<td>Nanodrop 2000 spectrophotometer</td>
<td>Thermo Scientific</td>
<td>ND-2000</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Thermo Scientific</td>
<td>SL40R</td>
</tr>
<tr>
<td>Rotor centrifuge</td>
<td>Thermo Scientific</td>
<td>75003607</td>
</tr>
<tr>
<td>Eppendorf centrifuge</td>
<td>Eppendorf</td>
<td>5417R</td>
</tr>
<tr>
<td>Rotor Eppendorf centrifuge</td>
<td>Eppendorf</td>
<td>F45-30-11</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Seahorse XF analyzer</h3>
<p>Agilent provides multiple Seahorse XF Analyzers (8 well, 24 well, or 96 well systems). Our protocol is optimized for the Seahorse XFe24 system (24 wells). General manufacturer’s instructions for performing a mitochondrial or glycolysis stress test can be respectively found at <a href="https://www.agilent.com/cs/library/usermanuals/public/XF_Cell_Mito_Stress_Test_Kit_User_Guide.pdf">https://www.agilent.com/cs/library/usermanuals/public/XF_Cell_Mito_Stress_Test_Kit_User_Guide.pdf</a> and <a href="https://www.agilent.com/cs/library/usermanuals/public/XF_Glycolysis_Stress_Test_Kit_User_Guide.pdf">https://www.agilent.com/cs/library/usermanuals/public/XF_Glycolysis_Stress_Test_Kit_User_Guide.pdf</a>.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Seahorse XFe96 (96 wells) or Seahorse XF HS mini (8 wells) Analyzers can be alternatively used.</div>
<h3 id="sec2.2">Preparation of assay medium</h3>
<p>In this protocol, we prepare assay medium with base medium with phenol red (Agilent, #102353-100) and we buffer it to pH 7.4 with NaOH. This medium can be stored for 24 h at 4°C.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Assay medium can be prepared based on different types of media. For Agilent’s recommendation on alternative media visit: <a href="https://www.agilent.com/cs/library/selectionguide/public/5991-7878EN.pdf">https://www.agilent.com/cs/library/selectionguide/public/5991-7878EN.pdf</a>
</div>
<h3 id="sec2.3">Preparation of stocks of substrates and inhibitors</h3>
<p>Stocks of substrates and inhibitors can be prepared in advance by dissolving them as depicted below. Glucose can be stored at 4°C. All other stocks should be stored at −20°C. All stocks can be stored for at least six months.</p>
<p>For mitochondrial stress tests, prepare the following stocks:</p>
<ul>
<li>2.5 mM Oligomycin in DMSO</li>
<li>2.5 mM FCCP in DMSO (prepare in fume hood)</li>
<li>2.5 mM Rotenone in DMSO</li>
<li>2.5 mM Antimycin A in DMSO</li>
</ul>
<p>For glycolysis stress tests, prepare the following stocks:</p>
<ul>
<li>2.5 mM Oligomycin in DMSO</li>
<li>1 M glucose in H<sub>2</sub>O</li>
</ul>
<h3 id="sec2.4">DNA-based normalization</h3>
<p>For normalization, we isolate and quantify organoid DNA by using the QIAamp DNA Micro Kit. Although alternative DNA extraction methods could be applied, the expected amounts of purified DNA are low in concentration, hence the method needs to be able to account for that.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>This step-by-step protocol consists of the following sections: culturing organoids, re-plating organoids for analysis, cartridge hydration, set up of mitochondrial and glycolysis stress tests and normalization by DNA content (<a href="#fig1">Figure 1</a>A). All steps have been optimized for examining bioenergetics in organoids derived from mouse intestine, human colon, and from colorectal tumors.</p>
<h3 id="sec3.1">Culturing organoids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>Organoid growth rates vary depending on the tissue of origin, differentiation rates, and genetic mutations. To achieve an accurate and reliable assay, the number of organoids per well needs to be comparable within and across conditions. To better control for differences in growth rates, we first expand our organoids in normal culture plates and re-plate the organoids in Seahorse Cell Culture microplates plates 24–72 h prior to analysis. This re-plating step is meant to better control for the differences in number and size of the organoids between the conditions to be compared on the day of the Seahorse analysis. Thus, this timing should be adjusted depending on the growth rate of the organoid type. For mouse small intestine-, human colon- or colorectal tumor-derived organoids, 36–48 h prior to analysis is an adequate re-plating time.</p>
<p>When long treatments are needed, for instance for organoid differentiation protocols, the treatments can be started during normal culture and continued after re-plating into Seahorse microplates. Here, we introduce our culturing methods for organoids derived from mouse small intestine, human colon, and colorectal tumors.</p>
<p>For mouse intestinal organoids:</p>
<ol>
<li>Disrupt organoids by mechanical shearing and plate them in a 24-well plate.
<ol type="a">
<li>Collect organoids in 1 mL of ice cold DMEM F12 advanced medium supplemented with penicillin/streptomycin, 10 mM HEPES, and 20 mM glutamax (+++ medium).
<div class="note">
<span class="note-title">Note:</span> Collecting 100 μL of organoids in Matrigel at the density depicted in <a href="#fig2">Figure 2</a>Ci is sufficient.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/521-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Organoid culturing</div>
<p>(A) 1,000 μL tip on a 1,000 μL pipet.</p>
<p>(B) 200 μL tip on top of a 1,000 μL tip.</p>
<p>(C) Mouse small intestinal organoids before splitting, after splitting, and before re-plating.</p>
<p>(D) P16t colorectal tumor-derived organoids before trypsinization, after trypsinization, and before re-plating.</p>
</figcaption>
</figure>
</li>
<li>Disrupt organoids by pipetting up and down with a 1000 μL pipet.
<ol type="i">
<li>First pipet up and down with a single 1,000 μL tip (<a href="#fig2">Figure 2</a>A).</li>
<li>When organoids are not disrupted yet, pipet up and down with a 200 μL pipet tip on top of a 1,000 μL pipet tip (<a href="#fig2">Figure 2</a>B) until organoids are broken into small crypts. The grade of mechanical shearing of the organoids should be monitored and visualized in between steps using a microscope.
<div class="note">
<span class="note-title">Note:</span> To prevent organoids from sticking to the pipet tips, pre-wet the pipet tips by pipetting op and down with cold +++ medium before starting the mechanical shearing.</div>
</li>
<li>Add 5 mL of ice cold +++ medium to the organoids.</li>
</ol>
</li>
<li>Centrifuge at 140–220 × <i>g</i> at 4°C for 5 min (Thermo Scientific SL40R centrifuge with Thermo Scientific 75003607 rotor, 800–1,000 rpm).</li>
<li>Re-suspend the pellet in 200–300 μL of Matrigel to reach the following density (<a href="#fig2">Figure 2</a>Cii)
<ol type="i">
<li>200 μL of organoids in Matrigel is sufficient for the later re-plating step in the Seahorse cell culture microplate.</li>
</ol>
</li>
<li>Plate organoids in a 24-well plate and culture for 36–48 h.
<ol type="i">
<li>Plate 50 μL Matrigel per well divided in 3–4 droplets.</li>
</ol>
</li>
<li>Incubate the plate upside down at 37°C for 30 min.</li>
<li>Add culture medium.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Culture organoids for 36–48 h prior to re-plating them in a Seahorse Cell Culture microplate (<a href="#fig2">Figure 2</a>Ciii).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The time of culturing before re-plating organoids in Seahorse microplate largely relies on the growth rate of the organoids. As an example of this, medium composition favoring organoid expansion by preventing differentiation, such as Wnt addition to mouse small intestinal organoids (<a href="#bib7">Rodríguez-Colman et al., 2017</a>), requires a shorter culture time from splitting to re-plating. In addition to that, the selection of the culture timing should be adjusted to properly address the research question.</div>
<p>For colorectal cancer or colon organoids:</p>
<ol start="3">
<li>Disrupt organoids by trypsinization and plate in a 24-well plate.
<ol type="a">
<li>Collect organoids in 10 mL of ice cold +++ medium.
<div class="note">
<span class="note-title">Note:</span> Collecting 50 μL of organoids in Matrigel in the following density (<a href="#fig2">Figure 2</a>Di) will be sufficient.</div>
</li>
<li>Centrifuge at 140–220 × <i>g</i> at 4°C for 5 min (Thermo Scientific SL40R centrifuge with Thermo Scientific 75003607 rotor, 800–1,000 rpm).</li>
<li>Trypsinize the organoids in 2 mL of trypsin or trypLE solution until organoids are disrupted (single cells or small clumps of cells depending on the organoid line).</li>
<li>Add 10 mL of ice cold +++ medium.</li>
<li>Centrifuge at 220–320 × <i>g</i> at 4°C for 5 min (1,000–1,200 rpm).</li>
<li>Remove supernatant and add 10 mL of ice cold +++ medium.</li>
<li>Centrifuge at 220–320 × <i>g</i> at 4°C for 5 min (1,000–1,200 rpm).</li>
<li>Re-suspend the pellet in 150–300 μL Matrigel to reach the following density (<a href="#fig2">Figure 2</a>Dii).
<ol type="i">
<li>200 μL of organoids in Matrigel is sufficient for the later re-plating step in the Seahorse cell culture microplate.</li>
</ol>
</li>
<li>Plate organoids in a 24-well plate
<ol type="i">
<li>Plate 50 μL Matrigel per well, divided in 3–4 droplets.</li>
</ol>
</li>
<li>Incubate the plate upside down at 37°C for 30 min.</li>
<li>Add culture medium.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Culture organoids until the desired size prior to re-plating them in a Seahorse Cell Culture microplate (<a href="#fig2">Figure 2</a>Diii). Check Note in step 2 for a comment on this.</li>
</ol>
<h3 id="sec3.2">Plate organoids for analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 days</div>
<p>Seahorse analysis has to be performed by plating organoids in Seahorse XF24 V28 PS cell culture microplates (Agilent, #100882-004) (<a href="#fig3">Figure 3</a>A). Below we describe step-by-step how to accurately re-plate organoids to guarantee reliable measurements.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/521-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Organoid plating in Seahorse cell culture microplate and preparation of sensor cartridge</div>
<p>(A) Seahorse XF24 V28 PS cell culture microplate.</p>
<p>(B and C) Plating organoids in a Seahorse cell culture microplate placed on top of a pre-warmed water-filled flask.</p>
<p>(D) Seahorse Sensor Cartridge on top of a Hydro Booster (black arrow) and Utility plate (dashed arrow) with injection ports A–D and O<sub>2</sub>- and pH-sensitive probe tips (black circle).</p>
</figcaption>
</figure>
<ol start="5">
<li>At least 24 h before plating, incubate the Seahorse XF24 V28 PS cell culture microplate to be used and a flask filled with water (<a href="#fig3">Figure 3</a>B) at 37°C. This step enhances the adhesion of the Matrigel drop to the bottom of the plate.</li>
<li>Collect the previously cultured organoids in 1 mL of ice cold +++ medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is important not to disrupt the integrity of the organoids while collecting. For mouse small intestinal organoids cultured for 36 h on medium supplemented with EGF, Noggin, and R-spondin, it is recommended to collect 4 wells of a 24-well plate (50 μL Matrigel each) to fill 20 wells of a Seahorse microplate. For colon- or colorectal tumor-derived organoids cultured for 7 days after trypsinization, it is recommended to collect 1 well of a 24-well plate to fill 20 wells of a Seahorse microplate.</div>
<ol start="7">
<li>Centrifuge at 140–220 × <i>g</i> at 4°C for 5 min (800–1,000 rpm).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the organoid type, centrifugation can affect organoid integrity. Choose the gravitational force that efficiently pulls down the organoids, but does not cause organoid disruption.</div>
<ol start="8">
<li>Re-suspend the organoid pellet in a volume of Matrigel that leads to the appropriate organoid density (steps b–d).
<ol type="a">
<li>It is important to reduce the variability in organoid density across and within experiments.</li>
<li>For mouse small intestine- and colon- or colorectal tumor-derived organoids, 20–40 organoids per well should be plated to measure OCR and ECAR within the range of sensitivity of the method (<a href="#fig4">Figures 4</a>A and 4B). Organoid density can be further adjusted to the specific requirements of the organoid type, under the consideration that the obtained measurements are within the range of sensitivity of the method (see CRITICAL).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/521-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Intestine-derived organoids in a Seahorse cell culture microplate and DNA quantification</div>
<p>(A and B) Organoids plated in a Seahorse cell culture plate. Organoid densities have been adjusted to fit the requirements of the organoid type. Mouse intestinal organoids undergo crypt formation-based morphological changes and are plated at a lower density than wild-type colon and colorectal tumor organoids. The depicted densities result in OCR and ECAR measurements within the sensitive range. (A) Mouse small intestinal sh<i>Foxo1/3</i> organoids treated with doxycycline or Mdivi-1 for 36 h. (B) Wild-type colon and colorectal tumor-derived organoids P9t, P16t, and P19bt. In (A) and (B), yellow crosses indicate control wells.</p>
<p>(C) DNA quantification of mouse small intestine- and human colon- and colorectal tumor-derived organoids in (A) and (B).</p>
</figcaption>
</figure>
</li>
<li>As a guideline for mouse small intestinal organoids, collect 4 wells of a 24-plate, start by re-suspending the organoids in 100 μL of Matrigel and dilute further when necessary.</li>
<li>As a guideline for colon- or colorectal tumor-derived organoids, collect 1 well of a 24-plate, start by re-suspending the organoids in 150 μL of Matrigel and dilute further when necessary.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Small volumes of Matrigel rapidly polymerize at room temperature. Prevent this by keeping the organoid-containing tubes on ice at all possible times. The density of organoids in Matrigel droplet needs attention: if the density of the organoids is too low or too high, the sensitivity of the method may limit the reliability of the measurements. ECAR can be reliably measured between 20 and 200 mpH/min. OCR can be reliably measured between 100 and 1,000 pmol O<sub>2</sub>/min (higher OCR measurements could be considered upon validation).</div>
<ol start="9">
<li>Plate organoids in a Seahorse Cell Culture Microplate [<a href="#sec7.1">Troubleshooting 1</a>] [<a href="#sec7.3">Troubleshooting 2</a>].
<ol type="a">
<li>Place the Seahorse microplate on top of the pre-warmed water-filled flask (<a href="#fig3">Figure 3</a>B).
<div class="note">
<span class="note-title">Note:</span> Placing the microplate on top of a warm surface facilitates plating 3 μL Matrigel droplets in the middle of Seahorse Cell Culture as the warmer temperature enhances the stability of the droplet, as such, and accelerates its polymerization.</div>
</li>
<li>Plate 3 μL of Matrigel (without organoids) in 4 control wells (<a href="#fig4">Figures 4</a>A and 4B).
<div class="note">
<span class="note-title">Note:</span> OCR and ECAR measurements are sensitive to the presence of Matrigel. Therefore, in control wells Matrigel should also be plated in the same volume.</div>
</li>
<li>Plate droplets of 3 μL Matrigel with organoids in the other wells (<a href="#fig3">Figures 3</a>C, <a href="#fig4">4</a>A, and 4B).
<div class="note">
<span class="note-title">Note:</span> For an accurate assessment of the bioenergetic parameters, we suggest to have four to five wells per condition.</div>
</li>
<li>After plating, place the Seahorse microplate plate back in the incubator for 10–15 min at 37°C.
<div class="critical">
<span class="critical-title">Critical:</span> Normal plating of organoids involves turning the plate upside down. Here that step should be avoided to prevent disruption of the Matrigel droplets. As small volumes of Matrigel polymerize fast, do not leave the plate for longer than the stated time to prevent drying out of the Matrigel droplet.</div>
</li>
<li>Add 200 μL of culture medium in each well.
<div class="critical">
<span class="critical-title">Critical:</span> Due to the limited plating area, organoids should be plated in the middle of the well and in between the three nodes (<a href="#fig3">Figure 3</a>C). Inaccurate plating leads to its adherence to the walls of the well and consequently to inaccurate measurements.</div>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>When required add the treatment of interest to the selected wells.</li>
<li>Culture organoids in the Seahorse microplate for 1–3 days until a mitochondrial or glycolysis stress test is performed.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As described in the “culturing organoids” section, this time frame is dependent on the growth rates of the organoids and the desired effect of the optional treatment.</div>
<h3 id="sec3.3">Cartridge hydration</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–72 h</div>
<p>Each probe tip of the sensor cartridge contains sensor material that detects changes in O<sub>2</sub> concentration and pH (<a href="#fig3">Figure 3</a>D). For correct functioning, the probe tips of the sensor cartridge have to be hydrated in XF Calibrant for a minimum of 4 h and a maximum of 72 h. For practical reasons we recommend to hydrate the cartridge for 12 to 24 h.</p>
<ol start="12">
<li>Lift the sensor cartridge from the Utility plate, and pipet 1 mL of XF Calibrant into each well of the Utility plate (<a href="#fig3">Figure 3</a>D).</li>
<li>Place the sensor cartridge back on the Utility plate with the Hydro Booster in between, in such a way that the probe tips of the Sensor Cartridge are submerged in XF Calibrant.</li>
<li>Place the Sensor Cartridge in a non-CO<sub>2</sub> 37°C humidified incubator for at least 4 h, but at maximum for 72 h.</li>
</ol>
<h3 id="sec3.4">XF mitochondrial stress test or glycolysis stress test</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>Agilent offers multiple assays to analyze different bioenergetic parameters. We have optimized the assay medium composition, the concentrations of substrates and inhibitors and the measurement settings for the XF Analyzer for performing mitochondrial and glycolysis stress tests in mouse small intestine, colon, and colorectal tumor-derived organoids.</p>
<ol start="15">
<li>Take pictures of the organoids in all wells of the Seahorse microplate (<a href="#fig3">Figures 3</a>C, <a href="#fig4">4</a>A, and 4B) by a bright-field microscope.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Imaging the organoids helps to control for viability and to compare differences in organoid growth and morphology across conditions and among experiments.</div>
<ol start="16">
<li>Prepare 50 mL assay medium.
<ol type="a">
<li>In this protocol, we prepare assay medium with base medium with phenol red (Agilent, #102353-100) and buffer it to pH 7.4 with NaOH. Assay medium needs to be supplemented with different metabolic substrates as indicated in <a href="#tbl2">Tables 2</a> and <a href="#tbl3">3</a>, and can be stored for 24 h at 4°C.
<table id="tbl2">
<caption>Table 2. Assay medium mitochondrial stress test</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Default assay medium composition<sup><a href="#tblfn1">a</a></sup>
</th>
<th>Optimized assay medium composition for intestine-derived organoids<sup><a href="#tblfn2">b</a></sup>
</th>
<th>Volumes to prepare 50 mL optimized assay medium</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glucose (1 M)</td>
<td>10 mM</td>
<td>10 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Pyruvate (100 mM)</td>
<td>1 mM</td>
<td>5 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Glutamine (200 mM)</td>
<td>2 mM</td>
<td>2 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>NaOH (1 M)</td>
<td>Adjust to pH 7.4</td>
<td>Adjust to pH 7.4</td>
<td>25 μL</td>
</tr>
<tr>
<td>Base medium</td>
<td>n/a</td>
<td>n/a</td>
<td>46.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p id="tblfn1" class="table-footnote"><sup>a</sup>Default assay medium composition suggested by Agilent.</p>
<p id="tblfn2" class="table-footnote"><sup>b</sup>Assay medium composition optimized for mouse small intestine- and human colon- or colorectal tumor-derived organoids.</p>
<table id="tbl3">
<caption>Table 3. Assay medium glycolysis stress test</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Default assay medium composition<sup><a href="#tblfn3">a</a></sup>
</th>
<th>Optimized assay medium composition for intestine-derived organoids<sup><a href="#tblfn4">b</a></sup>
</th>
<th>Volumes to prepare 50 mL optimized assay medium</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glutamine (200 mM)</td>
<td>2 mM</td>
<td>2 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>NaOH (1 M)</td>
<td>Adjust to pH 7.4</td>
<td>Adjust to pH 7.4</td>
<td>25 μL</td>
</tr>
<tr>
<td>Base medium</td>
<td>n/a</td>
<td>n/a</td>
<td>49.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p id="tblfn3" class="table-footnote"><sup>a</sup>Default assay medium composition suggested by Agilent.</p>
<p id="tblfn4" class="table-footnote"><sup>b</sup>Assay medium composition optimized for mouse small intestine- and human colon- or colorectal tumor-derived organoids.</p>
</li>
<li>For a mitochondrial stress test, prepare assay medium according to <a href="#tbl2">Table 2</a>.</li>
<li>For a glycolysis stress test, prepare assay medium according to <a href="#tbl3">Table 3</a>.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Following Agilent’s recommendations, substrates should be provided in saturating concentrations to ensure that they will not be limiting OCR and ECAR measurements. In <a href="#tbl2">Tables 2</a> and <a href="#tbl3">3</a> we have listed the saturating substrate concentrations adjusted for both, mouse small intestine- and human colon- or colorectal tumor-derived organoid analysis. Note that these concentrations are organoid type-specific and need to be determined empirically when applied to other organoid types.</div>
<ol start="17">
<li>Prepare organoids for the XF analysis.
<ol type="a">
<li>Remove the culture medium from the organoids.</li>
<li>Carefully wash organoids twice with 500 μL assay medium.</li>
<li>Add 525 μL assay medium into each well.</li>
<li>Incubate organoids in an incubator without CO<sub>2</sub> control at 37°C for at least 30 min, but at maximum 1 h.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> When analyzing the effect of drugs/treatment on bioenergetics, it could be necessary to maintain the treatment during the assay. This needs to be decided depending on the mechanism of action of the drug and the research question. Note that the assay medium is not buffered. Therefore, depending on the compound type, it can be required to readjust the pH of the compound’s stock.</div>
<ol start="18">
<li>During the incubation time of organoids in assay medium, prepare the injection solutions. Injections should be prepared in assay medium and 75 μL of these injections should be pipetted in the injection ports (<a href="#fig3">Figure 3</a>D). The device will inject the drugs in the order of port A to port D.
<ol type="a">
<li>For a standard mitochondrial stress test, prepare Oligomycin, FCCP and Rotenone &amp; Antimycin A and inject 75 μL of each in port A, B, and C, respectively (<a href="#tbl4">Table 4</a> and <a href="#fig3">Figure 3</a>D).
<table id="tbl4">
<caption>Table 4. Mitochondrial stress test injections optimized for mouse small intestine-, human colon- and colorectal tumor-derived organoids</caption>
<thead>
<tr>
<th>Injection port</th>
<th>Drug</th>
<th>Final concentration in Seahorse plate</th>
<th>Injection concentration</th>
<th>To add from 2.5 mM stock (μL)</th>
<th>XF assay medium (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Port A</td>
<td>Oligomycin</td>
<td>5 μM</td>
<td>40 μM</td>
<td>32</td>
<td>1,968</td>
</tr>
<tr>
<td>Port B</td>
<td>FCCP</td>
<td>2 μM</td>
<td>18 μM</td>
<td>14.4</td>
<td>1,985.6</td>
</tr>
<tr>
<td>Port C</td>
<td>Rotenone and Antimycin A</td>
<td>2 μM + 2 μM</td>
<td>20 μM</td>
<td>16 (each)</td>
<td>1.68</td>
</tr>
<tr>
<td>Port D</td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table>
</li>
<li>For a standard glycolysis stress test, prepare glucose, Oligomycin and 2-deoxyglucose (2-DG) and inject 75 μL of each in port A, B, and C, respectively (<a href="#tbl5">Table 5</a> and <a href="#fig3">Figure 3</a>D).
<table id="tbl5">
<caption>Table 5. Glycolysis stress test injections optimized for mouse small intestine-, human colon- and colorectal tumor-derived organoids</caption>
<thead>
<tr>
<th>Injection port</th>
<th>Drug</th>
<th>Final concentration in Seahorse plate</th>
<th>Injection concentration</th>
<th>To add from stock</th>
<th>XF assay medium (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Port A</td>
<td>Glucose</td>
<td>10 mM</td>
<td>80 mM</td>
<td>160 μL</td>
<td>1,840</td>
</tr>
<tr>
<td>Port B</td>
<td>Oligomycin</td>
<td>5 μM</td>
<td>45 μM</td>
<td>36 μL</td>
<td>1,964</td>
</tr>
<tr>
<td>Port C</td>
<td>2-DG<sup><a href="#tblfn5">a</a></sup>
</td>
<td>100 mM</td>
<td>1 M</td>
<td>333 mg</td>
<td>2,000</td>
</tr>
<tr>
<td>Port D</td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
<td> </td>
</tr>
</tbody>
</table>
<p id="tblfn5" class="table-footnote"><sup>a</sup>2-Deoxyglucose solutions are not stable over time and should be dissolved on the day of performing the glycolysis stress test</p>
</li>
<li>Upon completion of preparation of the injection ports, place the sensor cartridge hold in the Utility plate back into the incubator.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When other organoid types are used, optimal substrate and inhibitor concentrations have to be determined. We recommend to start with titrating Oligomycin, followed by respectively FCCP and Rotenone and Antimycin A. The lowest concentration that causes maximal inhibition of every drug should be selected. For an indication of inhibitor concentrations used for cell lines, check the Seahorse XF Analyzer guides: <a href="https://www.agilent.com/en/support/cell-analysis/seahorse-assay-guides-templates">https://www.agilent.com/en/support/cell-analysis/seahorse-assay-guides-templates</a>
</div>
<div class="optional">
<span class="optional-title">Optional:</span> For analysis of the immediate metabolic response to a certain drug, drugs can be injected during the mitochondrial or glycolysis stress test via the injection ports (<a href="#fig1">Figures 1</a>B and 1C). For such a bioenergetics assay, use injection port A for the drug of interest and move the injections corresponding to the mitochondrial or glycolysis stress test to port B, C and D respectively (<a href="#fig3">Figure 3</a>D). An example of such an application in a mitochondrial stress test, can be found in our recent study (<a href="#bib3">Ludikhuize et al., 2020</a>). Note that the concentrations of the inhibitors in injection ports need to be adjusted according to the change in the volume resulting from adding preceding injection step in A. Based on the research question, the order of the substrates and inhibitors can also be re-arranged. Then, the experimental settings can be updated in the Wave software accordingly.</div>
<ol start="19">
<li>Prepare settings of the XF Analyzer.
<ol type="a">
<li>Open the Wave software.</li>
<li>Open a mitochondrial or glycolysis stress test template, and fill in the assay details. A detailed explanation on how to create and customize assay template files using the Wave software can be found in section 2.4 in: <a href="https://www.agilent.com/en/product/cell-analysis/how-to-run-an-assay">https://www.agilent.com/en/product/cell-analysis/how-to-run-an-assay</a>.</li>
<li>Make sure you adjust the XF analysis settings in the “Instrument Protocol” section.
<ol type="i">
<li>For optimized mitochondrial stress test settings for mouse small intestine- and colon- or colorectal tumor-derived organoids, see <a href="#tbl6">Table 6</a>.
<table id="tbl6">
<caption>Table 6. Settings mitochondrial stress test</caption>
<thead>
<tr>
<th>Settings</th>
<th> </th>
<th> </th>
<th>Default<sup><a href="#tblfn6">a</a></sup>
</th>
<th>Intestine-derived organoids<sup><a href="#tblfn7">b</a></sup>
</th>
<th>Your organoids?</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3">Basal</td>
<td rowspan="3">3 cycles</td>
<td>mix</td>
<td>3 min</td>
<td>4 min</td>
<td> </td>
</tr>
<tr>
<td>wait</td>
<td>2 min</td>
<td>0 min</td>
<td> </td>
</tr>
<tr>
<td>measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
<tr>
<td rowspan="6">Oligomycin</td>
<td rowspan="3">1 cycle</td>
<td>mix</td>
<td>3 min</td>
<td>5 min</td>
<td> </td>
</tr>
<tr>
<td>wait</td>
<td>2 min</td>
<td>10 min</td>
<td> </td>
</tr>
<tr>
<td>measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
<tr>
<td rowspan="3">2 cycles</td>
<td>mix</td>
<td>3 min</td>
<td>4 min</td>
<td> </td>
</tr>
<tr>
<td>wait</td>
<td>2 min</td>
<td>0 min</td>
<td> </td>
</tr>
<tr>
<td>measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
<tr>
<td rowspan="3">FCCP</td>
<td rowspan="3">3 cycles</td>
<td>mix</td>
<td>3 min</td>
<td>4 min</td>
<td> </td>
</tr>
<tr>
<td>wait</td>
<td>2 min</td>
<td>0 min</td>
<td> </td>
</tr>
<tr>
<td>measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
<tr>
<td>Rotenone + Antimycin A</td>
<td>3 cycles</td>
<td>mix</td>
<td>3 min</td>
<td>4 min</td>
<td> </td>
</tr>
<tr>
<td> </td>
<td> </td>
<td>wait</td>
<td>2 min</td>
<td>0 min</td>
<td> </td>
</tr>
<tr>
<td> </td>
<td> </td>
<td>measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
</tbody>
</table>
<p id="tblfn6" class="table-footnote"><sup>a</sup>Suggested by Agilent.</p>
<p id="tblfn7" class="table-footnote"><sup>b</sup>These settings can be applied to mouse small intestine- and human colon- or colorectal tumor-derived organoids.</p>
</li>
<li>For optimized glycolysis stress test settings for mouse small intestine- and colon- or colorectal tumor-derived organoids, see <a href="#tbl7">Table 7</a>.
<table id="tbl7">
<caption>Table 7. Settings glycolysis stress test</caption>
<thead>
<tr>
<th>Settings</th>
<th> </th>
<th> </th>
<th>Default<sup><a href="#tblfn8">a</a></sup>
</th>
<th>Intestine-derived organoids<sup><a href="#tblfn9">b</a></sup>
</th>
<th>Your organoids?</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3">Basal</td>
<td rowspan="3">3 cycles</td>
<td>Mix</td>
<td>3 min</td>
<td>4 min</td>
<td> </td>
</tr>
<tr>
<td>Wait</td>
<td>2 min</td>
<td>0 min</td>
<td> </td>
</tr>
<tr>
<td>Measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
<tr>
<td rowspan="3">Glucose</td>
<td rowspan="3">3 cycles</td>
<td>Mix</td>
<td>3 min</td>
<td>4 min</td>
<td> </td>
</tr>
<tr>
<td>Wait</td>
<td>2 min</td>
<td>1 min</td>
<td> </td>
</tr>
<tr>
<td>Measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
<tr>
<td rowspan="6">Oligomycin</td>
<td rowspan="3">1 cycles</td>
<td>mix</td>
<td>3 min</td>
<td>5 min</td>
<td> </td>
</tr>
<tr>
<td>wait</td>
<td>2 min</td>
<td>10 min</td>
<td> </td>
</tr>
<tr>
<td>measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
<tr>
<td rowspan="3">2 cycles</td>
<td>mix</td>
<td>3 min</td>
<td>4 min</td>
<td> </td>
</tr>
<tr>
<td>wait</td>
<td>2 min</td>
<td>0 min</td>
<td> </td>
</tr>
<tr>
<td>measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
<tr>
<td rowspan="3">2-DG</td>
<td rowspan="3">3 cycles</td>
<td>mix</td>
<td>3 min</td>
<td>4 min</td>
<td> </td>
</tr>
<tr>
<td>wait</td>
<td>2 min</td>
<td>0 min</td>
<td> </td>
</tr>
<tr>
<td>measure</td>
<td>3 min</td>
<td>2 min</td>
<td> </td>
</tr>
</tbody>
</table>
<p id="tblfn8" class="table-footnote"><sup>a</sup>Suggested by Agilent.</p>
<p id="tblfn9" class="table-footnote"><sup>b</sup>These settings can be applied to mouse small intestine- and human colon- or colorectal tumor-derived organoids.</p>
</li>
</ol>
</li>
<li>Save your template.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> compared to cell lines, organoids show a slower response to Oligomycin, possibly due to the presence of Matrigel. Therefore, we adjusted the settings by applying a 10-min waiting step right after the injection of the inhibitor in both, the mitochondrial stress test and the glycolysis stress test (<a href="#tbl6">Tables 6</a> and <a href="#tbl7">7</a>).</div>
<ol start="20">
<li>Run the mitochondrial or glycolysis stress test [<a href="#sec7.3">Troubleshooting 3</a>].
<ol type="a">
<li>Upon clicking on “Run,” the Wave software will ask to insert the sensor cartridge in the XF Analyzer.</li>
<li>Remove the Hydro Booster and the lid (<a href="#fig3">Figure 3</a>D), and place the sensor cartridge hold in the Utility plate in the XF Analyzer.</li>
<li>After the calibration step which takes approximately 15 min, the Wave software will request to replace the Utility plate by the Seahorse Cell Culture microplate containing the organoids.</li>
<li>The XF analysis will automatically begin.
<div class="note">
<span class="note-title">Note:</span> Once the experiment has finished, the OCR and ECAR measurements will be displayed on the XF Analyzer screen. A first inspection of the results can be done at this point. When high standard deviations within one condition are observed, switching from “group” view to “well” view, allows to detect possible outliers (<a href="#fig5">Figure 5</a>), which can result, for instance, from inaccurate plating or differential organoid viability. Pictures taken at step 15 can be used to judge the cause of large standard deviations. Excluding outliers from the analysis can be considered (<a href="#fig5">Figures 5</a>A–5C) [<a href="#sec7.7">Troubleshooting 4</a>].</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/521-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Detection of possible outliers in Seahorse XF Wave software</div>
<p>OCR measurements of an exemplary mitochondrial stress test. In (A), the measurements are displayed as “group view” and in (B) as “well view.” In (B), the line in bold indicates the outlier measurement. (C) “Group view” of the OCR measurements displayed in (A) upon exclusion of the outlier depicted in (B).</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<ol start="21">
<li>After the experiment, carefully remove medium from the wells.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The plate should be removed immediately after the experiment has finished in order to prevent loosing organoid viability as a result of the presence of the inhibitors used in the assay.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> When required, after medium aspiration, the Seahorse Cell Culture Microplate can be stored at −20°C for at least 2 weeks to later proceed with the DNA quantification step.</div>
<h3 id="sec3.5">DNA quantification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>Calculation of bioenergetic parameters requires normalization to correct for differences in numbers of cells, organoids, or amount of biological material. Standard normalization methods are based on the determination of cell numbers or protein content. Due to the presence of Matrigel, normalization by protein content is not adequate when using organoids. Quantification of cell numbers in organoids is technically challenging due to their 3D structure and the limited number of cells. A reliable alternative for normalization of OCR and ECAR measurements from organoids is normalization by the DNA content. Below we describe how to collect and isolate DNA from organoids cultured on Seahorse Cell Culture microplates.</p>
<ol start="22">
<li>Add 100 μL ice cold Cell Recovery Solution (Corning, #354253) to each well of the Seahorse Cell Culture microplate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Upon decision to remove a certain outlier from analysis in step 20, the organoids from this well should not be collected for DNA quantification.</div>
<ol start="23">
<li>Incubate the plate at 4°C for 30–60 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Incubation in Cell Recovery Solution facilitates organoid collection and removal of the Matrigel.</div>
<ol start="24">
<li>Collect the organoids and pool the organoids belonging to one condition in an Eppendorf tube.</li>
<li>Centrifuge at 10,620 × <i>g</i> at 4°C for 5 min (Eppendorf 5417R centrifuge with an Eppendorf F45-30-11 rotor, 10 000 rpm).</li>
<li>Remove supernatant while leaving the pellet undisrupted.</li>
<li>Isolate genomic DNA by using the QIAamp DNA micro kit according to the manufacturer’s instructions.
<ol type="a">
<li>In order to maximize the DNA yield, add 30 μL elution buffer let it stand for 2–3 min to maximize DNA elution.</li>
</ol>
</li>
<li>Measure genomic DNA by Nanodrop.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Aiming for similar organoid densities across conditions improves the robustness of the analysis. To be noticed, normalization by pooled DNA corrects for variability across conditions but does not correct for differences within conditions. Therefore, the variation in organoid density within conditions should be kept low.</div>
<div class="note">
<span class="note-title">Note:</span> In these protocols, we apply DNA normalization; this does not exclude the application of other normalization methods. Importantly, RNA quantification could be considered when differences in DNA amount are expected across compared conditions. Importantly, some organoid types could have a high rate of shedding dead cells. This requires attention, as damaged cells can be metabolically low/inactive but they still contribute to total DNA quantification.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Our protocol describes how to perform mitochondrial stress tests and glycolysis stress tests with mouse intestine-, human colon-, or colorectal tumor-derived organoids. Here we provide an example of a mitochondrial stress test performed with doxycycline-inducible sh<i>Foxo1/3</i> mouse small intestinal organoids in order to examine the respiratory state upon <i>Foxo1/3</i> knockdown in the small intestine (<a href="#bib3">Ludikhuize et al., 2020</a>). This mitochondrial stress test shows that <i>Foxo1/3</i> knockdown results in decreased basal and maximal respiration (<a href="#fig1">Figures 1</a>B and <a href="#fig6">6</a>A). Although a mitochondrial stress test does not enable the analysis of specific glycolytic parameters, ECAR measurements during a mitochondrial stress test do provide information about the glycolytic rates. For instance, the first three time points show that <i>Foxo1/3</i> knockdown also reduces glycolysis (<a href="#fig6">Figure 6</a>B). For a detailed analysis of bioenergetics upon <i>Foxo1/3</i> knockdown in mouse small intestinal organoids, please refer to (<a href="#bib3">Ludikhuize et al., 2020</a>). We also provide an example of a glycolysis stress test performed with the colorectal tumor-derived organoids P16t and P19bt (<a href="#bib9">van de Wetering et al., 2015</a>). This glycolysis stress test shows that P16t organoids have a lower glycolytic rate and glycolytic capacity compared to P19bt organoids (<a href="#fig1">Figures 1</a>C and <a href="#fig6">6</a>C). We also provide examples of quantification of DNA isolated from mouse small intestinal organoids and human colon or colorectal tumor-derived organoids. DNA isolation from 5 Seahorse microplate wells mouse small intestinal organoids results in 10–30 ng DNA/μL in 30 μL elution buffer (<a href="#fig4">Figures 4</a>A and 4C). DNA isolation from 5 Seahorse microplate wells colon- or colorectal tumor-derived organoids results in 20–60 ng DNA/μL in 30 μL elution buffer (<a href="#fig4">Figures 4</a>B and 4C).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/521-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Profiling bioenergetics of intestine-derived organoids</div>
<p>(A and B) OCR (A) and ECAR (B) measurements of a mitochondrial stress test of doxycycline-inducible sh<i>Foxo1/3</i> mouse small intestinal organoids treated with or without doxycycline for 36 h (mean of 5 technical replicates ± SEM). Figure reprinted with permission from <a href="#bib3">Ludikhuize et al. (2020)</a>.</p>
<p>(C and D) ECAR (A) and OCR (B) measurements of a glycolysis stress test of two different patient-derived colorectal cancer organoids (P16t and P19bt) (mean of 5 technical replicates ± SEM).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>After normalization, the mitochondrial stress tests and glycolysis stress tests can be analyzed by using the Seahorse Wave Desktop software (free to download from the Agilent website: <a href="https://www.agilent.com/en/product/cell-analysis/real-time-cell-metabolic-analysis/xf-software/seahorse-wave-desktop-software-740897">https://www.agilent.com/en/product/cell-analysis/real-time-cell-metabolic-analysis/xf-software/seahorse-wave-desktop-software-740897</a>).</p>
<ol>
<li>Open your experiment in the software.</li>
<li>Normalize your data for the DNA content, measured during step 28.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Adjust the DNA concentration to the number of the wells you have pooled for DNA isolation.</div>
<ol start="3">
<li>Exporting data.
<ol type="a">
<li>For a mitochondrial stress test, export your data as Seahorse XF Cell Mito Stress Test Report Generator.</li>
<li>For a glycolysis stress test, export your data as Seahorse XF Glycolysis Stress Test Report Generator.</li>
<li>Raw data can also be exported to Excel or GraphPad Prism.</li>
</ol>
</li>
<li>Analyze parameters of interest, such as <i>basal respiration</i>, <i>maximal respiration</i>, <i>glycolysis</i>, and <i>glycolytic capacity</i> (<a href="#fig1">Figures 1</a>B and 1C). The values used for each Seahorse XF Report Generator calculation are outlined in the specific Seahorse XF Report Generator assay user guides.</li>
<li>Statistical analysis can be applied to the technical replicates, which reflects the technical accuracy of the experiment. To address whether the observed differences are reproducible and biologically relevant, repeat the experiments at least three times [<a href="#sec7.9">Troubleshooting 5</a>]. After experiments are performed multiple times, statistics can be performed on independent experiments. Statistical analysis needs to be adjusted to the experimental set up and conditions in the assay. For applied examples see (<a href="#bib3">Ludikhuize et al., 2020</a>).</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The bioenergetic parameters measured by Seahorse analysis are tightly coupled to the <i>in vitro</i> conditions of the analysis. The setup of the assay conditions include the use of saturating concentrations of substrates and drug concentrations. This should be taken into consideration when aiming at drawing conclusions referring to the <i>in vitro</i> culture conditions or the <i>in vivo</i> situation, where substrates can be present at different concentrations.</p>
<p>Due to the low amount of organoid material per well, we indicate to pool the technical replicates of each condition prior DNA extraction and quantification. This strategy increases the accuracy of the DNA measurements and serves to correct for the variability across conditions but not within conditions. Nevertheless, other steps in this protocol such as the re-plating step and guidelines to improve the accuracy of plating are meant to minimize the variability across the wells of the same condition.</p>
<p>Organoids are heterogeneous as they consist of multiple cell types. Thus, bioenergetics analysis of organoids will always reflect the average metabolic phenotype of the different cell types that comprise the organoid. Hence, when analyzing the metabolic consequence of for instance the knockdown of a gene, the effect of this knockdown on differentiation should be taken into consideration. When the cellular heterogeneity is not of interest for the research question, organoid cultures could be modified to prevent differentiation and increase the cellular homogeneity. Examples of such treatments can be found in <a href="#tbl1">Table 1</a>. Importantly, organoid cultures can also be modified to enrich organoid cultures for a specific cell type to enable its metabolic characterization (see <a href="#tbl1">Table 1</a>). Combining differentiation protocols and Seahorse technology therefore provides a great opportunity to determine the metabolic phenotype of specific cell types, and even the metabolic changes associated to the differentiation process.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The Matrigel droplets do not stay in the middle of the Seahorse microplate well after re-plating (step 9).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Incubate the Seahorse microplate at 37°C for at least 24 h and put the plate on top of a pre-warmed water-filled flask. When this is not sufficient, reduce the amount of medium that is mixed with Matrigel. Diluted Matrigel will hardly form droplets. Alternatively, reducing the volume of the Matrigel droplets from 3 to 2.5 μL can help.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>The organoids are disrupted or broken or their appearance indicates low fitness or viability in the Seahorse cell culture microplate (step 9).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Some organoid lines can be particularly sensitive to mechanical shearing compromising their fitness and viability. To minimize this, try to reduce the pipetting movements while re-plating the organoids. Alternatively, try to adjust the timing of the re-plating to when the organoids have a size that is less sensitive to mechanical shearing. Depending of the organoid type, this size can be either larger or smaller. For intestinal organoids, re-plating the organoids when they are rather small (less than 120 μm diameter, <a href="#fig2">Figure 2</a>C), protects them from mechanical disruption.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The organoids do not or only mildly respond to the injections (step 20).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>First of all, it is essential to control the stocks of the inhibitors and replace by new fresh stocks when the organoid response to the drugs decreases over time.</p>
<p>When the low response occurs with fresh and correctly prepared drug stocks, it should be checked whether the unresponsiveness of the organoids is a technical artifact or the actual phenotype of the organoids. For instance, when Oligomycin does not increase or only mildly increases the ECAR (<a href="#fig6">Figure 6</a>C), it should be analyzed whether Oligomycin was functional. In this experiment, Oligomycin did reduce the OCR (<a href="#fig6">Figure 6</a>D), indicating that the amount of Oligomycin that was used was correct. Furthermore, it should be examined whether the setup of the experiment was optimal. For instance, it should be checked whether increasing glucose concentrations will enhance the glycolytic capacity, in order to check that glucose availability was not limited in the experiment.</p>
<p>When the ineffectiveness of the drugs cannot be attributed to technical problems, the density of the organoids should be checked. When the density of the organoids is high, the drugs can possibly be less effective. In that case, try to reduce the number of organoids plated in the Seahorse Cell Culture microplate wells. When the organoids are plated in the proper density, titrate the concentrations of the injection reagents. When this does not work, this can be repeated in presence of more or less substrates in the XF assay medium.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>There is high variability in OCR and ECAR measurement within technical replicates of an experiment (step 20).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>This observation can result from uneven organoid numbers in the different wells. To avoid that, carefully re-suspend the organoids in the Matrigel and plate them immediately in the Seahorse cell culture microplate. Alternatively, inaccurate plating can lead to Matrigel drops fusing to the wall of the plate instead of being centered in the well (<a href="#fig3">Figure 3</a>C). This also leads to variable measurements of OCR and ECAR. Follow the steps concerning microplate incubation at 37°C and the use of a warm water-filled flask (step 9). Plating accuracy can be checked by microscope inspection after plating. Consider to exclude “failed wells” from the analysis. When doing so, restrain to collect that well for DNA extraction.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>There is high variability in OCR and ECAR measurement between different experiments (Section Quantification &amp; statistical analysis, step 5).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Make sure that the timing of the experiments is always similar. Reduce variability in culture time after splitting or trypsinizing the organoids and plating organoids in the Seahorse cell culture microplate. Furthermore, always plate the same number of organoids per well in every experiment.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cracan, V., Titov, D.V., Shen, H., Grabarek, Z., and Mootha, V.K. (2017). A genetically encoded tool for manipulation of NADP+/NADPH in living cells. Nat. Chem. Biol. <i>13</i>, 1088-1095. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref1">View at publisher</a></p>
<p id="bib10">Drost, J., Jaarsveld, R.H., and Ponsioen, B.; an Jaarsveld, R.H., Ponsioen, B., Zimberlin, C., van Boxtel, R., Buijs, A., Sachs, N., Overmeer, R.M., Offerhaus, G.J., Begthel, H., et al. (2015). Sequential cancer mutations in cultured human intestinal stem cells. Nature <i>521</i>, 43-47. <a class="external-link" href="https://doi.org/10.1038/nature14415">https://doi.org/10.1038/nature14415</a></p>
<p id="bib12">Kim, J., Koo, B.K., and Knoblich, J.A. (2020). Human organoids: model systems for human biology and medicine. Nat Rev Mol Cell Biol <i>21</i>, 571-584. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/optAUk2qt6HV8">View at publisher</a></p>
<p id="bib2">Lindeboom, R.G., van Voorthuijsen, L., Oost, K.C., Rodríguez-Colman, M.J., Luna-Velez, M.V., Furlan, C., Baraille, F., Jansen, P.W., Ribeiro, A., Burgering, B.M., et al. (2018). Integrative multi-omics analysis of intestinal organoid differentiation. Mol. Syst. Biol. <i>14</i>, e8227. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref2">View at publisher</a></p>
<p id="bib3">Ludikhuize, M.C., Meerlo, M., Gallego, M.P., Xanthakis, D., Burgaya Julià, M., Nguyen, N.T.B., Brombacher, E.C., Liv, N., Maurice, M.M., Paik, J.-H., et al. (2020). Mitochondria define intestinal stem cell differentiation downstream of a foxo/notch axis. Cell Metab. <i>32</i>, 889-900.e7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref3">View at publisher</a></p>
<p id="bib4">Mills, R.J., Titmarsh, D.M., Koenig, X., Parker, B.L., Ryall, J.G., Quaife-Ryan, G.A., Voges, H.K., Hodson, M.P., Ferguson, C., Drowley, L., et al. (2017). Functional screening in human cardiac organoids reveals a metabolic mechanism for cardiomyocyte cell cycle arrest. Proc. Natl. Acad. Sci. U S A <i>114</i>, E8372-E8381. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref4">View at publisher</a></p>
<p id="bib5">Mohamed Suhaimi, N.-A., Phyo, W.M., Yap, H.Y., Choy, S.H.Y., Wei, X., Choudhury, Y., Tan, W.J., Tan, L.A.P.Y., Foo, R.S.Y., Tan, S.H.S., et al. (2017). Metformin inhibits cellular proliferation and bioenergetics in colorectal cancer patient-derived xenografts. Mol. Cancer Ther. <i>16</i>, 2035-2044. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref5">View at publisher</a></p>
<p id="bib6">Okkelman, I.A., Neto, N., Papkovsky, D.B., Monaghan, M.G., and Dmitriev, R.I. (2020). A deeper understanding of intestinal organoid metabolism revealed by combining fluorescence lifetime imaging microscopy (FLIM) and extracellular flux analyses. Redox Biol. <i>30</i>, 101420. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref6">View at publisher</a></p>
<p id="bib7">Rodríguez-Colman, M.J., Schewe, M., Meerlo, M., Stigter, E., Gerrits, J., Pras-Raves, M., Sacchetti, A., Hornsveld, M., Oost, K.C., Snippert, H.J., et al. (2017). Interplay between metabolic identities in the intestinal crypt supports stem cell function. Nature <i>543</i>, 424-427. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref7">View at publisher</a></p>
<p id="bib8">Schell, J.C., Wisidagama, D.R., Bensard, C., Zhao, H., Wei, P., Tanner, J., Flores, A., Mohlman, J., Sorensen, L.K., Earl, C.S., et al. (2017). Control of intestinal stem cell function and proliferation by mitochondrial pyruvate metabolism. Nat. Cell Biol. <i>19</i>, 1027-1036. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref8">View at publisher</a></p>
<p id="bib11">Schutgens, F. and Clevers, H. (2020). Human Organoids: Tools for Understanding Biology and Treating Diseases. Annual Review of Pathology: Mechanisms of Disease 2020 <i>15:1</i>, 211-234. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/optheDiJvoXaY">View at publisher</a></p>
<p id="bib9">van de Wetering, M., Francies, H.E., Francis, J.M., Bounova, G., Iorio, F., Pronk, A., van Houdt, W., van Gorp, J., Taylor-Weiner, A., Kester, L., et al. (2015). Prospective derivation of a living organoid biobank of colorectal cancer patients. Cell <i>161</i>, 933-945. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/sref9">View at publisher</a></p>
<p id="bib13">Vander Heiden, M.G., Cantley, L.C., and Thompson, C.B. (2009). Understanding the Warburg Effect: The Metabolic Requirements of Cell Proliferation. Science <i>324</i>, 1029-1033. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00093-9/optmVUJmT9AzU">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Maria J. Rodríguez Colman (<a href="mailto:M.J.RodriguezColman@umcutrecht.nl">M.J.RodriguezColman@umcutrecht.nl</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents, plasmids, or organoid lines.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate new datasets or codes.</p>
<h3>Acknowledgments</h3>
<p>This work was financially supported by <a href="https://doi.org/10.13039/501100004622">Dutch Cancer Society</a> (KWF 2016-I 10471, KWF 2017-II 11315).</p>
<h3>Author Contributions</h3>
<p>Methodology and Investigation, M.J.R.C., M.C.L., and M.M. Writing – Original Draft, M.J.R.C. and M.C.L. Writing – Review &amp; Editing, M.J.R.C.; Funding Acquisition, M.J.R.C. and B.M.T.B.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>