<article>
<section>
<h1 id="header">Protocol for multimodal analysis of human kidney tissue by imaging mass spectrometry and CODEX multiplexed immunofluorescence</h1>
<p><time datetime="2021-08-13">Published: August 13, 2021</time></p>
<p>Elizabeth K. Neumann,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">9</a></sup> Nathan Heath Patterson,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Jamie L. Allen,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Lukasz G. Migas,<sup><a href="#aff3">3</a></sup> Haichun Yang,<sup><a href="#aff4">4</a></sup> Maya Brewer,<sup><a href="#aff4">4</a></sup> David M. Anderson,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Jennifer Harvey,<sup><a href="#aff2">2</a></sup> Danielle B. Gutierrez,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Raymond C. Harris,<sup><a href="#aff4">4</a></sup> Mark P. deCaestecker,<sup><a href="#aff4">4</a></sup> Agnes B. Fogo,<sup><a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a></sup> Raf Van de Plas,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Richard M. Caprioli,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff7">7</a></sup> and Jeffrey M. Spraggins<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff7">7</a>,<a href="#aff8">8</a>,<a href="#fn2">10</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA</p>
<p id="aff2"><sup>2</sup>Mass Spectrometry Research Center, Vanderbilt University, Nashville, TN 37232, USA</p>
<p id="aff3"><sup>3</sup>Delft Center for Systems and Control (DCSC), Delft University of Technology, 2628 CD Delft, the Netherlands</p>
<p id="aff4"><sup>4</sup>Division of Nephrology and Hypertension, Department of Medicine, Vanderbilt University Medical Center, Nashville, TN 37232, USA</p>
<p id="aff5"><sup>5</sup>Department of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center, Nashville, TN 37232, USA</p>
<p id="aff6"><sup>6</sup>Departments of Medicine and Pediatrics, Vanderbilt University Medical Center, Nashville, TN 37232, USA</p>
<p id="aff7"><sup>7</sup>Department of Chemistry, Vanderbilt University, Nashville, TN 37232, USA</p>
<p id="aff8"><sup>8</sup>Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, TN 37232, USA</p>
<p id="fn1"><sup>9</sup>Technical contact</p>
<p id="fn2"><sup>10</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:jeff.spraggins@vanderbilt.edu">jeff.spraggins@vanderbilt.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100747">10.1016/j.xpro.2021.100747</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we describe the preservation and preparation of human kidney tissue for interrogation by histopathology, imaging mass spectrometry, and multiplexed immunofluorescence. Custom image registration and integration techniques are used to create cellular and molecular atlases of this organ system. Through careful optimization, we ensure high-quality and reproducible datasets suitable for cross-patient comparisons that are essential to understanding human health and disease. Moreover, each of these steps can be adapted to other organ systems or diseases, enabling additional atlas efforts.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Combines histology, imaging mass spectrometry, and multiplexed immunofluorescence</li>
<li>Describes workflows for producing high-quality data sets of the human kidney</li>
<li>Can be adapted to other organ systems or diseases</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This specific protocol describes how to perform multimodal analysis on healthy, human tissue with the goal of constructing a cellular and molecular atlas of normal human tissues as part of the National Institutes of Health Human Biomolecular Atlas Program (HuBMAP) (<a href="#bib6">Hu, 2019</a>). These atlases require both co-registration and integration of many molecular and imaging assays so spatial comparisons can be made between samples. All the reagents you will need are in the <a href="#key-resources-table">key resources table</a>. We have also successfully performed similar analyses on other human tissues, such as spleen and heart. Additional or different antibodies may be substituted depending on the application and organ. Moreover, these protocols have been developed using the specific instrumentation listed below and although they are applicable to similar instrumentation; certain parameters, such as spatial resolution and number of immunofluorescence cycles, will need to be adjusted based on individual platform limitations.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>CD7 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>4150022</td>
</tr>
<tr>
<td>Aquaporin 1 (1/200 dilution)</td>
<td>Abcam</td>
<td>ab9566</td>
</tr>
<tr>
<td>Aquaporin 2 (1/200 dilution)</td>
<td>Abcam</td>
<td>ab230170</td>
</tr>
<tr>
<td>CD90 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>4150021</td>
</tr>
<tr>
<td>CD38 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>4150007</td>
</tr>
<tr>
<td>Ki67 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>4250019</td>
</tr>
<tr>
<td>Nestin (1/200 dilution)</td>
<td>Abcam</td>
<td>ab221660</td>
</tr>
<tr>
<td>CD45 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>4150003</td>
</tr>
<tr>
<td>Uromodulin (1/200 dilution)</td>
<td>Abcam</td>
<td>ab207170</td>
</tr>
<tr>
<td>ß-Catenin (1/200 dilution)</td>
<td>Abcam</td>
<td>ab32572</td>
</tr>
<tr>
<td>Vimentin (1/200 dilution)</td>
<td>Abcam</td>
<td>ab92547</td>
</tr>
<tr>
<td>PARP1 (1/200 dilution)</td>
<td>Abcam</td>
<td>ab191217</td>
</tr>
<tr>
<td>CD31 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>4250009</td>
</tr>
<tr>
<td>CD93 (1/200 dilution)</td>
<td>Abcam</td>
<td>ab134079</td>
</tr>
<tr>
<td>E-Cadherin (1/200 dilution)</td>
<td>Cell Signaling Technology</td>
<td>3195S</td>
</tr>
<tr>
<td>Laminin (1/200 dilution)</td>
<td>DSHB</td>
<td>2E8</td>
</tr>
<tr>
<td>KDR/Vegf 2 (1/200 dilution)</td>
<td>Abcam</td>
<td>ab237634</td>
</tr>
<tr>
<td>Calbindin (1/200 dilution)</td>
<td>Abcam</td>
<td>ab108404</td>
</tr>
<tr>
<td>Cytokeratin 7 (1/200 dilution)</td>
<td>Abcam</td>
<td>ab68459</td>
</tr>
<tr>
<td>α-Smooth Muscle Actin (1/200 dilution)</td>
<td>Abcam</td>
<td>ab7817</td>
</tr>
<tr>
<td>Renin (1/200 dilution)</td>
<td>Abcam</td>
<td>ab212197</td>
</tr>
<tr>
<td>Tryptase (1/200 dilution)</td>
<td>Abcam</td>
<td>ab151757</td>
</tr>
<tr>
<td>MARCKS (1/200 dilution)</td>
<td>Abcam</td>
<td>ab184546</td>
</tr>
<tr>
<td>CODEX Barcodes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>RX025 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6450009</td>
</tr>
<tr>
<td>RX002 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6250001</td>
</tr>
<tr>
<td>RX015 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6350003</td>
</tr>
<tr>
<td>RX022 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6150008</td>
</tr>
<tr>
<td>RX035 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6250010</td>
</tr>
<tr>
<td>RX033 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6350008</td>
</tr>
<tr>
<td>RX007 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6150003</td>
</tr>
<tr>
<td>RX026 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6250007</td>
</tr>
<tr>
<td>RX024 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6350005</td>
</tr>
<tr>
<td>RX001 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6150001</td>
</tr>
<tr>
<td>RX047 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6250012</td>
</tr>
<tr>
<td>RX042 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6350010</td>
</tr>
<tr>
<td>RX013 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6150005</td>
</tr>
<tr>
<td>RX032 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6250009</td>
</tr>
<tr>
<td>RX006 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6350002</td>
</tr>
<tr>
<td>RX016 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6150006</td>
</tr>
<tr>
<td>RX023 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6250006</td>
</tr>
<tr>
<td>RX004 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6150002</td>
</tr>
<tr>
<td>RX005 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6250002</td>
</tr>
<tr>
<td>RX021 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6350004</td>
</tr>
<tr>
<td>RX010 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6150004</td>
</tr>
<tr>
<td>RX014 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6250003</td>
</tr>
<tr>
<td>RX027 (1/200 dilution)</td>
<td>Akoya Biosciences</td>
<td>6350006</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human kidney</td>
<td>Vanderbilt Medical Center CHTN</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Dimethyl sulfoxide, ReagentPlus, ≥99.5%</td>
<td>Sigma-Aldrich</td>
<td>D5879-4L</td>
</tr>
<tr>
<td>Poly-L-lysine solution, 0.1% (w/v) in H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>P8920-500ML</td>
</tr>
<tr>
<td>10× CODEX Buffer</td>
<td>Akoya Biosciences</td>
<td>7000001</td>
</tr>
<tr>
<td>CODEX Assay Reagent</td>
<td>Akoya Biosciences</td>
<td>7000002</td>
</tr>
<tr>
<td>Dumont Cover Slip Forceps Dumoxel</td>
<td>Fine Science Tools</td>
<td>11251-33</td>
</tr>
<tr>
<td>4% Formaldehyde, methanol-free</td>
<td>Thermo Fisher Scientific</td>
<td>R37814</td>
</tr>
<tr>
<td>Methanol-free Formaldehyde Ampules</td>
<td>Thermo Fisher Scientific</td>
<td>28908</td>
</tr>
<tr>
<td>Ambion Nuclease-Free Water</td>
<td>Thermo Fisher Scientific</td>
<td>43–879-36</td>
</tr>
<tr>
<td>96-Well plates</td>
<td>Akoya Biosciences</td>
<td>7000006</td>
</tr>
<tr>
<td>Molecular probes DAPI (4′,6 Diamidino 2 Phenylindole, Dihydrochloride)</td>
<td>Thermo Fisher Scientific</td>
<td>D1306</td>
</tr>
<tr>
<td>CODEX Gaskets v2</td>
<td>Akoya Biosciences</td>
<td>7000010</td>
</tr>
<tr>
<td>Carboxymethylcellulose sodium salt, medium viscosity</td>
<td>Sigma-Aldrich</td>
<td>C4888-500G</td>
</tr>
<tr>
<td>2-Methylbutane, Reagent Grade, ≥99%</td>
<td>Fisher Scientific</td>
<td>60048070</td>
</tr>
<tr>
<td>Disposable aluminum dishes</td>
<td>Fisher Scientific</td>
<td>S40284</td>
</tr>
<tr>
<td>2′,5′-Dihydroxyacetophenone, 97%</td>
<td>Sigma-Aldrich</td>
<td>D107603-25G</td>
</tr>
<tr>
<td>Methanol, for HPLC, ≥99.9%, CAS 67-56-1, Sigma-Aldrich 34860-4X4L-R</td>
<td>Sigma-Aldrich</td>
<td>34860-4X4L-R</td>
</tr>
<tr>
<td>Acetone, HPLC Plus, for HPLC, GC, and residue analysis, ≥99.9%</td>
<td>Sigma-Aldrich</td>
<td>650501-4X4L</td>
</tr>
<tr>
<td>Tetrahydrofuran, inhibitor-free, for HPLC, ≥99.9%,</td>
<td>Sigma-Aldrich</td>
<td>34865</td>
</tr>
<tr>
<td>16% Paraformaldehyde</td>
<td>Electron Microscopy Sciences</td>
<td>15710</td>
</tr>
<tr>
<td>Ethanol, 99.5%</td>
<td>Fisher Scientific</td>
<td>AC615095000</td>
</tr>
<tr>
<td>Xylenes</td>
<td>Fisher Scientific</td>
<td>X5-500</td>
</tr>
<tr>
<td>Ammonium hydroxide</td>
<td>Fisher Scientific</td>
<td>A669S-500</td>
</tr>
<tr>
<td>TRIzol Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>15596026</td>
</tr>
<tr>
<td>1× Phosphate-Buffered Saline</td>
<td>Life Technologies</td>
<td>14190144</td>
</tr>
<tr>
<td>RLT Buffer Plus</td>
<td>QIAGEN</td>
<td>1053393</td>
</tr>
<tr>
<td>RNeasy MinElute Cleanup Kit</td>
<td>QIAGEN</td>
<td>74204</td>
</tr>
<tr>
<td>RPE Buffer</td>
<td>QIAGEN</td>
<td>1018013</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Fisher Scientific</td>
<td>21985023</td>
</tr>
<tr>
<td>Cryoseal</td>
<td>Fisher Scientific</td>
<td>22050262</td>
</tr>
<tr>
<td>Poly-L-lysine 0.1%</td>
<td>Sigma-Aldrich</td>
<td>P8920</td>
</tr>
<tr>
<td>Paraffin</td>
<td>Fisher Scientific</td>
<td>23021750</td>
</tr>
<tr>
<td>Periodic Acid and Schiff Reagent</td>
<td>Abcam</td>
<td>Ab150680</td>
</tr>
<tr>
<td>Hematoxylin</td>
<td>Fisher Scientific</td>
<td>22050198</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td>Thermo Fisher Scientific</td>
<td>AM9938</td>
</tr>
<tr>
<td>DEPC-Treated Water</td>
<td>Thermo Fisher Scientific</td>
<td>4387937</td>
</tr>
<tr>
<td>10% Buffered Formalin Phosphate</td>
<td>Fisher Scientific</td>
<td>SF100-4</td>
</tr>
<tr>
<td>Glycogen</td>
<td>Thermo Fisher Scientific</td>
<td>AM9510</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Indium tin oxide-coated glass slides</td>
<td>Delta Technologies</td>
<td>CG-81IN-S115</td>
</tr>
<tr>
<td>Anti-roll Bar</td>
<td>Leica</td>
<td>14047742497</td>
</tr>
<tr>
<td>Bent-tip tweezers</td>
<td>Fine Science Tools</td>
<td>11251-33</td>
</tr>
<tr>
<td>6-Well TC Plates</td>
<td>VWR</td>
<td>10861-554</td>
</tr>
<tr>
<td>Phase Lock Gel - Heavy</td>
<td>Quantabio</td>
<td>2302830</td>
</tr>
<tr>
<td>Cryostat Blade</td>
<td>Electron Microscopy Systems</td>
<td>63059-25</td>
</tr>
<tr>
<td>27G Precisionglide needles</td>
<td>BD</td>
<td>305109</td>
</tr>
<tr>
<td>Embedding molds</td>
<td>Thermo Fisher Scientific</td>
<td>1841</td>
</tr>
<tr>
<td>50 kDa MWCO filter</td>
<td>EMD Millipore</td>
<td>UFC505096</td>
</tr>
<tr>
<td>TM Sprayer M3</td>
<td>HTX Imaging</td>
<td>N/A</td>
</tr>
<tr>
<td>Bruker timsTOF Pro</td>
<td>Bruker</td>
<td>N/A</td>
</tr>
<tr>
<td>Axio Scanner Slide Scanner</td>
<td>Carl Zeiss AG</td>
<td>N/A</td>
</tr>
<tr>
<td>Axio Observer</td>
<td>Carl Zeiss AG</td>
<td>N/A</td>
</tr>
<tr>
<td>CODEX System</td>
<td>Akoya Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>Flatbed Scanner</td>
<td>Epson</td>
<td>B11B175012</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>CODEX Conjugation Kit</td>
<td>Akoya Biosciences</td>
<td>7000009</td>
</tr>
<tr>
<td>CODEX Staining Kit</td>
<td>Akoya Biosciences</td>
<td>7000008</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Highly Multiplexed Immunofluorescence of the Human Kidney using Co-Detection by Indexing (CODEX)</td>
<td>N/A</td>
<td><a href="http://10.1101/2020.12.04.412429">10.1101/2020.12.04.412429</a></td>
</tr>
<tr>
<td>Next Generation Histology-Directed Imaging Mass Spectrometry Driven by Autofluorescence Microscopy</td>
<td>N/A</td>
<td><a href="http://10.1021/acs.analchem.8b02885">10.1021/acs.analchem.8b02885</a></td>
</tr>
<tr>
<td>Advanced Registration and Analysis of MALDI Imaging Mass Spectrometry Measurements through Autofluorescence Microscopy</td>
<td>N/A</td>
<td><a href="http://10.1021/acs.analchem.8b02884">10.1021/acs.analchem.8b02884</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FIJI ImageJ</td>
<td><a href="https://fiji.sc">https://fiji.sc</a></td>
<td>N/A</td>
</tr>
<tr>
<td>regToolboxMSRC</td>
<td><a href="https://github.com/nhpatterson/regtoolboxmsrc">https://github.com/nhpatterson/regtoolboxmsrc</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Cardinal</td>
<td><a href="https://cardinalmsi.org">https://cardinalmsi.org</a></td>
<td>N/A</td>
</tr>
<tr>
<td>LipidMAPS</td>
<td><a href="https://www.lipidmaps.org/">https://www.lipidmaps.org/</a></td>
<td>N/A</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Freeze thaw cycles quickly degrade many reagents, such as antibodies. In general, we recommend aliquoting out single use volumes of these reagents. If this is not possible, we recommend leaving these reagents within the refrigerator once they are thawed.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Collection and stabilization of human kidney tissue after surgical excision</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This portion of the protocol enables collection and stabilization of human kidney tissue after key clinical diagnostic workup. Assessments are made for experimental pathological assessment and multimodal analysis. Human kidney tissue was surgically removed during a full nephrectomy and remnant tissue was processed for research purposes by the Cooperative Human Tissue Network at Vanderbilt University Medical Center. Remnant biospecimens were collected in compliance with the Cooperative Human Tissue Network standard protocols and National Cancer Institute’s Best Practices for the procurement of remnant surgical research material. Participants were consented for remnant tissue collection in accordance to institutional IRB policies.</p>
<ol>
<li>Prepare the kidney tissue for freezing.
<ol type="a">
<li>De-identified human kidney tissues are collected from surgical resections selected based on patient and clinical metadata (e.g., specific age range). All patients must have consented to allow their tissue to be used for research purposes.</li>
<li>Cut the kidney in half coronally, generally the longest plane (<a href="#fig1">Figure 1</a>A).</li>
<li>Cut out a portion of the kidney (∼80 × 60 × 4 mm) that is farthest away from tumor or diseased areas using a scalpel or other blade (<a href="#fig1">Figure 1</a>B).
<ol type="i">
<li>Photograph the smaller piece of kidney in reference to its original location within the entire kidney (<a href="#fig1">Figure 1</a>B).</li>
<li>Smaller or larger tissue pieces can be extracted depending on the original size of the organ, tumor location, and desired number of samples. The size of the pieces that require embedding are essential for proper freezing.</li>
</ol>
</li>
<li>Further cut the piece of kidney into 3–5 smaller pieces (∼2 × 15 × 4 mm, <a href="#fig1">Figure 1</a>C).
<div class="critical">
<span class="critical-title">Critical:</span> Larger kidney resections result in freezing artifacts from uneven freezing between the outside and the center of the tissue. To prevent this, we recommend freezing tissues that are ∼2 <b>×</b> 15 <b>×</b> 4 mm or smaller.</div>
<ol type="i">
<li>Photograph the smaller pieces to reference how they were cut so that the smaller pieces that are frozen and embedded can be traced back to the larger organ (<a href="#fig1">Figure 1</a>C).</li>
<li>Each of these pieces will later be cut in half, where one half will be fresh frozen (Step 2) and the other formalin fixed and paraffin embedded (FFPE) (Step 3).</li>
</ol>
</li>
</ol>
</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Kidney resections</div>
<p>Section of resected kidney with the superior and inferior poles labeled.</p>
<p>(A) Whole kidney cut in half, coronally with superior and inferior poles labeled (black dotted line).</p>
<p>(B) A portion of the kidney (∼80 × 60 × 4 mm) that is farthest away from the tumor (green dotted line).</p>
<p>(C) The extracted portion is divided into thirds or fourths to ensure event freezing (blue dotted line).</p>
</figcaption>
</figure>
<ol start="2">
<li>The stabilization and freezing procedure for fresh frozen tissue blocks.
<ol type="a">
<li>Place a layer of dry ice into a freezer box and add enough isopentane to make a slurry.</li>
<li>Remove kidney tissue blocks from plastic encasing and measure the weight, length, width, and height (<a href="#fig2">Figure 2</a>).</li>
<li>Place a metal weigh boat on top of the isopentane slurry and let cool.</li>
<li>Write the sample number and orientation onto mold edges, indicating inner/outer and superior/inferior edges (<a href="#fig3">Figure 3</a>).</li>
<li>Add tissue to the metal boat and wait until the tissue is completely frozen.</li>
<li>Add enough 2.6% carboxymethylcellulose (CMC) to cover the bottom of the embedding mold.
<div class="note">
<span class="note-title">Note:</span> CMC is used as an embedding material rather than more common embedding materials, such as optimal cutting temperature (OCT) compound, to avoid introducing chemical interferences during MS analysis (<a href="#bib17">Stoeckli et al., 2007</a>). While other embedding materials are compatible for IMS (<a href="#bib4">Gill et al., 2017</a>, <a href="#bib18">Strohalm et al., 2011</a>), they require optimization and testing compatibility with different methodologies.</div>
</li>
<li>Place the embedding mold with CMC on top of the isopentane slurry.
<ol type="i">
<li>Wait until CMC starts to freeze and begins to turn opaque (<a href="#fig4">Figure 4</a>, center).</li>
</ol>
</li>
<li>Place tissue onto freezing CMC layer and slowly add more CMC until the tissue is completely covered (<a href="#fig4">Figure 4</a>, left).</li>
<li>When completely frozen, wrap in foil, label, and store at −80°C.
<div class="note">
<span class="note-title">Note:</span> Foil has a lower specific heat than other storage containers (e.g., plastic) so this prevents freeze/thaw artifacts from uneven cooling/freezing. This is most important right after freezing.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Measurement guide</div>
<p>Measure and record the length, width, and thickness of the kidney tissue pieces using gridded paper (as shown above) or other instruments.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Kidney embedding and freezing</div>
<p>(A–D) (A) Label mold such that orientation of the kidney can be maintained. Images of unfrozen tissue (B), partially frozen tissue and CMC (C), and CMC embedded tissue ready for storage (D).</p>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Schiff’s reagent stain example</div>
<p>(A) Tissue after incubation with periodic acid. The tissue is clear and uncolored.</p>
<p>(B) Tissue appearance after incubation with Schiff’s reagent. The tissue is light pink.</p>
<p>(C) Tissue appearance after exposure to warm water. The tissue turns bright pink when exposed to warm water.</p>
</figcaption>
</figure>
<ol start="3">
<li>Fixing and embedding procedure for FFPE tissue.
<ol type="a">
<li>Carefully place a non-frozen piece of tissue into an embedding cassette with the cut side facing down and submerge in a 10% neutral-buffered formalin solution for at least 12 h but typically 18 h.
<ol type="i">
<li>The tissue can be left in fixative for up to 24 h, although this will vary based on tissue type and thicknesses.</li>
</ol>
</li>
<li>Move the tissue cassettes into 70% ethanol for 35 min.</li>
<li>Submerge the tissue into 90% ethanol for 35 min.</li>
<li>Submerge the tissue in 95% ethanol for 35 min.</li>
<li>Submerge the tissue into three sequential containers of 100% ethanol, each for 35 min.</li>
<li>Submerge the tissue in 100% xylenes for 35 min, three times (total of 105 min). Fresh xylene solutions are required for each separate submersion.</li>
<li>Submerge the tissue in paraffin (wax) for 60 min, three times (total of 3 h).</li>
<li>The tissue is then embedded with additional paraffin at ∼20°C (room temperature) to make a block such that the tissue is entirely encased.</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Pathological assessment of PAS-stained FFPE tissues</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>By the end of this major step, a representative histological stain of a human kidney tissue section is assessed for normalcy by a pathologist prior to multimodal imaging. After pathological assessment, each tissue can be classified by proportion of medulla and/or cortex, disease, or artifacts.</p>
<ol start="4">
<li>Procedure for periodic acid-Schiff (PAS) staining of FFPE tissues for histological assessment (adapted from <a href="#bib9">Munro, 1971</a>).
<div class="note">
<span class="note-title">Note:</span> Hematoxylin and eosin or other histological staining procedures can be used instead of a PAS stain and may be preferred for other tissue types. PAS is optimal for kidney assessment, as it allows assessment of proximal tubular brush borders and other kidney structures. Specific histological stains may be optimal for specific tissues and an expert pathologist should be consulted before choosing a staining procedure.</div>
<ol type="a">
<li>At the beginning of the staining process, staining reagents should be equilibrated to ∼20°C (room temperature) from refrigerator storage.</li>
<li>Deparaffinize tissue by submerging in xylenes for 3 min, twice. The wax should be removed, and the tissue should look transparent. Additional washes can be incorporated as needed for complete paraffin removal.</li>
<li>Rehydrate the tissue using graded alcohol solutions and distilled water.
<ol type="i">
<li>Begin by submerging in 100% ethanol for 1 min, twice.</li>
<li>Submerge the tissue in 95% ethanol for 1 min.</li>
<li>Submerge the tissue in 70% ethanol for 1 min.</li>
<li>Submerge the tissue in 100% water for 1 min.</li>
</ol>
</li>
<li>Rinse the tissues with generous amounts of distilled water.</li>
<li>Incubate the tissue in 0.5% periodic acid in distilled water for 10 min for tissue oxidation.
<div class="note">
<span class="note-title">Note:</span> The periodic acid solution can be reused for ∼1 month before needing to be replaced.</div>
</li>
<li>Rinse the tissues generously with distilled water.</li>
<li>Incubate the tissues in Schiff’s reagent for 30 min.
<div class="note">
<span class="note-title">Note:</span> Different tissue types require different incubation times, generally ranging from 15–30 min.</div>
<ol type="i">
<li>The Schiff’s reagent can be reused for ∼1 month before needing to be replaced.</li>
</ol>
</li>
<li>Rinse the tissues in generous amounts of warm (∼40°C) water.
<ol type="i">
<li>While distilled water is preferred, heated tap water is sufficient for this step.</li>
<li>The tissue should appear pink after rinsing with warm water (<a href="#fig4">Figure 4</a>C).</li>
</ol>
</li>
<li>Counterstain the tissue in hematoxylin for 1 min and rinse generously with distilled water.</li>
<li>Dip the tissue section into 0.5% ammonium hydroxide a single time and rinse with generous amounts of distilled water.
<ol type="i">
<li>The tissue should be rinsed until the runoff water is colorless.</li>
</ol>
</li>
<li>Dehydrate the tissue using graded alcohol solutions.
<ol type="i">
<li>Dip the tissue 10 times for ∼ 1 s into 95%, 95%, 100%, and 100% ethanol solutions, sequentially, with ten dips for each different solution.</li>
<li>If the last 100% ethanol rinse is colored by residual stain, additional washing in new 100% ethanol solution should be performed.</li>
</ol>
</li>
<li>Submerge the tissue in xylenes for 1 min, twice.</li>
<li>Coverslip the tissues for long term storage and higher quality microscopy.
<ol type="i">
<li>Dip the tissue in xylenes, remove slide and hold horizontally and add 2–3 drops of cytoseal to the tissue surface.</li>
<li>Line up the slide and cover slip before slowly placing the coverslip on the slide.</li>
<li>If needed, use a dissecting tool or other pointed object to remove/displace bubbles from the tissue by gently pressing down on the cover slip.
<div class="critical">
<span class="critical-title">Critical:</span> Any bubbles between the tissue and coverslip will prevent acquisition of high-quality microscopy images.</div>
</li>
</ol>
</li>
<li>Image slides using a Leica brightfield scanner (or equivalent) at 20× (∼1 μm spatial resolution) and save the file.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Histological assessment of FFPE tissue for determining tissue architecture, disease, and normalcy.
<ol type="a">
<li>Determine the percentage of cortex, medulla, renal pyramids, and renal pelvis within the tissue section.
<ol type="i">
<li>Determine the presence or absence of kidney diseases or sampling degradation, such as autolysis or non-renal diseases (e.g., cancer).</li>
<li>Assess the severity of common renal diseases (0–3, None-Severe), such as glomerular or tubulointerstitial diseases.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.3">Sample preparation for multimodal workflows</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 days</div>
<p>This protocol describes cryosectioning, microscopy, and RNA assessment in preparation for and prior to samples entering the multimodal workflow.</p>
<ol start="6">
<li>Cryosection fresh frozen tissue onto a glass slide or coverslip for different analytical workflows.
<ol type="a">
<li>Set the internal temperature of the cryostat to −21°C and object temperature to −20°C.</li>
<li>Clean the blade with ethanol and a lint free wipe to remove potential chemical contaminants before placing the blade into the cryostat.</li>
<li>Mount the CMC embedded tissue onto a cold chuck using OCT. Take care to avoid OCT touching the tissue directly, and only contact the CMC shell.
<ol type="i">
<li>A flat surface can be shaved onto the mounting surface using a razor blade to ensure correct orientation and enhanced tissue block adherence.
<div class="critical">
<span class="critical-title">Critical:</span> OCT causes significant ion suppression during mass spectrometry analysis. Tissue that directly touches OCT should be discarded.</div>
</li>
</ol>
</li>
<li>Adjust angle of sectioning stage (<a href="#fig5">Figure 5</a>) to ensure quality sectioning.
<ol type="i">
<li>Poor angle adjustment causes CMC embedding to fragment and lose structural integrity during sectioning. The angle adjustment depends on the flatness and orientation of the tissue as well as the blade that is used. This will need to be optimized for procedure.</li>
</ol>
</li>
<li>Trim the CMC shell using 30–50 μm step increments until the tissue becomes visible.
<ol type="i">
<li>The blade often dulls during this trimming step and may need to be replaced for high-quality and/or serial sections.</li>
<li>Excess CMC surrounding the tissue may be removed with a cooled razor blade to reduce section size and improve sectioning.</li>
</ol>
</li>
<li>Adjust the section thickness setting to 10 μm and section until the portion of the tissue for multimodal analysis is reached, this often is when the entire tissue face is cut, rather than part or portion of the tissue face.</li>
<li>Position the anti-roll bar, such that it is ahead of the blade, but does not scrape the tissue.
<ol type="i">
<li>If the anti-roll bar is too far forward it scrapes the tissue, but if it is pulled back too far, the tissue section will roll and fold during sectioning.</li>
</ol>
</li>
<li>Section until the desired number of tissue sections are acquired (at least one for each modality although multiple sections for modality may be beneficial for technical replicates). Manipulate sections onto slides with a fine tipped paint brush and thaw mount using a gloved hand and then place mounted, cut tissue on a 20°C hot plate for 30 s.
<ol type="i">
<li>Imaging mass spectrometry analysis requires mounting onto indium tin oxide coated slides or another conductive substrate.</li>
<li>CODEX immunofluorescence analysis requires mounting onto poly-lysine coated coverslips.</li>
</ol>
</li>
<li>Tissue sections should be stored in a desiccator if undergoing immediate analysis within 24 h or −80°C within a vacuum sealed slide holder for longer term storage.
<div class="critical">
<span class="critical-title">Critical:</span> If serial sections are required for multimodal analysis, it is imperative that the tissues are high-quality and devoid of wrinkles, folds, or damage.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Cryostat configuration</div>
<p>(A and B) (A) Unadjusted angle and (B) adjusted angle of the cryostat blade holder.</p>
</figcaption>
</figure>
<ol start="7">
<li>Perform autofluorescence microscopy on each tissue section to acquire a nondestructive image of the tissue that can be used for later registration.
<ol type="a">
<li>Sections stored in a desiccator can be immediately used, otherwise slides should return to 20°C within a desiccator to prevent artifacts.</li>
<li>Place the slide within the microscope adapter and insert into the Zeiss AxioScan Slide Scanner or equivalent.</li>
<li>Perform coarse focusing on the tissue using the DAPI filter set (ex. 335–383 nm; em. 420–470 nm, blue), 90% lamp power, and 150 ms exposure times.</li>
<li>Perform fine focusing on the tissue using the DAPI, GFP (ex. 450–490 nm; em. 500–550), and DsRed (ex. 538–562 nm; em. 570–640) filter sets to build a focus map using 90% lamp power and 150 ms exposure times.</li>
<li>Define an imaging region that includes the entire tissue section and acquire an autofluorescence image.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> While we describe performing autofluorescence microscopy on a Zeiss AxioScan Slide Scanner, other fluorescence microscopy systems can be used so long as they can perform whole tissue imaging with the described filters. Differences in lamp power and excitation power are expected and depend on the employed system and light source. These parameters should be optimized by the individual user for each sample type.</div>
<ol start="8">
<li>Determine the RNA quality of the tissue block (adapted from Masato Hoshi; <a href="#bib7">Lake et al., 2019</a>).
<ol type="a">
<li>Submerge the tissue section into 750 μL of TRIzoILS and homogenize the sample by passaging through a 23G then a 27G syringe needle.</li>
<li>Freeze the homogenate solution on dry ice, thaw, and mix by vortexing. Then, dilute the homogenate solution with RNase free water to a final volume of 960 μL.</li>
<li>Add 40 μL of a 5 mg/mL glycogen solution to the homogenate, vortex, and incubate for 5 min at 20°C.</li>
<li>Add 250 μL of chloroform and vortex for 15 s before centrifuging at 12,000 × <i>g</i> for 10 min at 4°C. Once complete, pipette the supernatant in new container.</li>
<li>Add 600 μL of isopropanol, mix, and incubate at –20°C for 20 min.</li>
<li>Centrifuge at 20,000 × <i>g</i> for 20 min at 4°C and discard the supernatant.</li>
<li>Add 600 μL of 80% ethanol diluted in RNase free water and vortex.</li>
<li>Centrifuge the solution at 20,000 × <i>g</i> for 5 min at 4°C, discard the supernatant, and dry down the sample using nitrogen or another inert gas.</li>
<li>Add 300 μL of RLT plus buffer containing 1% 2-ME and mix before adding 450 μL 100% ethanol.</li>
<li>Transfer 750 μL of the solution to a RNeasy MinElute spin column and centrifuge for 1 min at 12,000 × <i>g</i> and 24°C.</li>
<li>Centrifuge the phase lock gel at 12,000 × <i>g</i> for 5 min before adding the TRIzol mixture and 500 μL of RPE buffer. Then centrifuge at 12,000 × <i>g</i> for 1 min at 24°C.</li>
<li>Discard the flow through, add an additional 500 μL of RPE buffer to the column, and centrifuge at 12,000 × <i>g</i> for 2 min at 24°C. Discard the flow through again.</li>
<li>Centrifuge the mixture again for 1 min at 12,000 × <i>g</i> and 24°C. Discard the flow through.</li>
<li>Open the column lid and centrifuge at 12,000 × <i>g</i> for 5 min at 24°C to dry the column membrane.</li>
<li>Place the column in a new container and add 10 μL of RNase free water in the center of the column membrane. Incubate for 2 min at 20°C before centrifuging at 12,000 × <i>g</i> and 24°C for 3 min.</li>
<li>Add 10 μL of RNase free water in the center of the column membrane and incubate for 2 min at 20°C before centrifuging at 12,000 × <i>g</i> and 24°C for 3 min.</li>
<li>Obtain the RNA integrity number (RIN) and percentage of RNA fragments &gt;200 nucleotides in size (DV 200) analysis to assess the RNA integrity.</li>
</ol>
</li>
</ol>
<h3 id="sec2.4">Imaging mass spectrometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>This protocol describes imaging mass spectrometry (IMS) specific sample preparation and IMS analysis on fresh frozen tissue sections for 10 μm spatial resolutions for 200 lipid species.(<a href="#bib10">Neumann et al., 2020a</a>, <a href="#bib1">Caprioli et al., 1997</a>, <a href="#bib8">McDonnell and Heeren, 2007</a>)</p>
<ol start="9">
<li>Matrix application onto the tissue surface.
<ol type="a">
<li>Apply 6 psi of nitrogen to the sprayer nozzle and turn on the TM sprayer.</li>
<li>Open the software and set the nozzle temperature to 40°C.</li>
<li>Change the LC solvent composition to 100% acetonitrile and flow rate to 0.05 mL/min.</li>
<li>Load the sample loop with 5 mL of pure acetonitrile and spray from the sample loop for 2 min to ensure cleanliness of the instrument.</li>
<li>Load 6 mL of matrix solution (5 mg/mL 1,5-diaminonaphthalene in tetrahydrofuran) into the sample loop.</li>
<li>Switch sprayer to “Inject” and spray for ∼1 min. Use a blank slide to check that matrix is spraying from the nozzle tip.</li>
<li>Tape slide(s) onto a 75°C heated block on top of stage (<a href="#fig6">Figure 6</a>) and adjust software scanning area to encompass the entire sample.
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Matrix application configuration</div>
<p>TM Sprayer set up for high resolution matrix coating.</p>
</figcaption>
</figure>
</li>
<li>Adjust settings to 1350 mm/min nozzle velocity, 1.5 mm track spacing, 0.05 mL/min flow rate, CC Pattern, 2 L/min flow rate, 5 passes, 40 mm nozzle height, and no drying time within the software.</li>
<li>Under the “Cycle” tab, click “Start,” then click “Continue” to start the coating process.</li>
<li>Once the coating process is finished, inject 20 mL of acetonitrile and set the LC flow rate to 1 mL/min for 10 min to clean the instrument.</li>
<li>Clean tip of nozzle with 5 mL of methanol.</li>
<li>Change the HLPC flow rate to 0.05 mL/min of 100% acetonitrile.</li>
<li>Shut down the software, turn off the TM Sprayer, and shut the nitrogen off.</li>
<li>Check matrix coating on slide for uniform application (<a href="#fig7">Figure 7</a>).
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Example of a matrix coated slide</div>
<p>An example of uniform matrix coating of sample of a sample outlined using sharpie on the back of the slide.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> While we describe performing matrix application with a commercial TM Sprayer, there are many ways to apply matrix (e.g. airbrush, sublimation, and sieving) (<a href="#bib11">Neumann et al., 2019</a>) as well as many different types of matrices (e.g. 2,5-dihydroxybenzoic acid, α-cyano-4-hydroxycinnamic acid, and 2′,6′-dihydroxyacetophenone). Each of these matrix application methods and matrices require different optimizations, particularly if 10 μm spatial resolution is an important attribute of the experiment. If matrix is not applied as described here, we cannot guarantee 10 μm spatial resolution is possible.</div>
<ol start="10">
<li>MALDI imaging mass spectrometry analysis of fresh frozen tissue (<a href="#bib1">Caprioli et al., 1997</a>).
<div class="note">
<span class="note-title">Note:</span> While we describe performing imaging mass spectrometry on a Bruker MALDI timsTOF platform, (<a href="#bib16">Spraggins et al., 2019</a>) a multitude of imaging mass spectrometry systems can be used. Differences in laser power and methods parameters are expected and should be optimized by the individual user for each sample type.</div>
<ol type="a">
<li>Using a flatbed scanner, scan a 3200-dpi image as a jpg of the matrix coated tissue section held in the MTP-slide adapter II with sufficient contrast to visualize the tissue boundaries.
<div class="note">
<span class="note-title">Note:</span> Tracing the tissue with a sharpie on the back of the slide may help in tissue visualization and region selection (<a href="#fig7">Figure 7</a>). This can be performed before or after matrix application.</div>
</li>
<li>Insert the slide adapter plate into the instrument and perform height detection procedures.</li>
<li>Open the FlexImaging software and follow the prompts to designate sample file name, method, pitch, scanned image of the slide, etc.</li>
<li>Train the target position with three teaching points.</li>
<li>Create a measurement region around the tissue for IMS analysis.</li>
<li>Go to an area off the tissue and ensure the focus of the laser is adequate for 10 μm spatial resolution imaging.
<div class="critical">
<span class="critical-title">Critical:</span> An improperly focused laser will not allow for 10 μm imaging and cause artifacts from oversampling.</div>
</li>
<li>Start the MS acquisition of the tissue section.
<div class="note">
<span class="note-title">Note:</span> Subsections b-e describe tissue acquisition using Bruker software and will vary on other instrument platforms. Generally, the process on other instruments will consist of inserting your sample into the source of the mass spectrometer, designate an imaging area, and train the mass spectrometry stage to the image of the sample.</div>
</li>
</ol>
</li>
</ol>
<ol start="11">
<li>Collect post-IMS autofluorescence microscopy of the MALDI ablated sample.
<ol type="a">
<li>Perform coarse focusing on the tissue using the GFP filter set (ex. 450–490 nm; em. 500–550, green) with 90% lamp power and 150 ms exposure times.</li>
<li>Perform fine focusing on the tissue using the GFP filter set and brightfield setting.</li>
<li>Define the imaging region that includes the entire tissue.</li>
<li>Acquire brightfield and autofluorescence images of the MALDI ablated sample.</li>
</ol>
</li>
<li>Post-IMS PAS staining of tissue.
<ol type="a">
<li>Unlike above, the PAS staining is performed on the same tissue section as MALDI IMS analysis. First, remove matrix by submerging the slide in 100% ethanol for ∼2–3 min or until matrix is gone.</li>
<li>Submerge the slides in 70% ethanol for 3 min.</li>
<li>The PAS protocol described above can be performed after the MALDI matrix is removed.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Leftover matrix will change the pH of the solutions and prevent adequate staining.</div>
<h3 id="sec2.5">CODEX multiplexed immunofluorescence</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>This protocol describes antibody conjugation, tissue preparation, and microscopy for CODEX multiplexed immunofluorescence analysis (<a href="#bib12">Neumann et al., 2020b</a>, <a href="#bib5">Goltsev et al., 2018</a>). CODEX multiplexed immunofluorescence can be used for labeling cell types or states. In brief, a primary antibody is conjugated to a unique oligonucleotide barcode. All primary antibodies (&lt;50) are added to a tissue section at once. After incubation, these antibodies are fixed to the tissue, enhancing tissue integrity during multiple staining rounds. Secondary (complementary) oligonucleotide barcodes are serially added and removed from the tissue, enabling staining of &lt;50 targets within a single tissue section.</p>
<ol start="13">
<li>Antibody conjugation for custom antibody markers (a list of which is provided in the <a href="#key-resources-table">key resources table</a>).
<ol type="a">
<li>Block 50 kDa molecular weight cut off filters with 500 μL of filter blocking solution for 10 min at ∼20°C.</li>
<li>Dilute 50 μg of each antibody into 100 μL in PBS and filter with 50 MW cut off filters.</li>
<li>Reduce antibodies with the reduction mixture for 25 min at ∼20°C.
<div class="note">
<span class="note-title">Note:</span> 25 minutes of reduction is suitable for many antibodies but may need to be optimized (15–40 minutes) depending on the primary antibody used. In general, shorter reduction times are more appropriate than longer times.</div>
</li>
<li>Centrifuge the reduced antibodies at 12,000 X g and add buffer solution.</li>
<li>Rehydrate oligonucleotide barcodes in 10 μL nuclease free water and further dilute with 210 μL of conjugation solution.
<div class="critical">
<span class="critical-title">Critical:</span> Nuclease free water must be used; otherwise the oligonucleotide barcodes will be degraded.</div>
</li>
<li>Respective barcode solutions are added to the reduced primary antibodies and incubated for 2 h at ∼20°C.</li>
<li>The solution is then purified by buffer exchanging with purification solution and stored in storage solution at ∼4°C.
<div class="critical">
<span class="critical-title">Critical:</span> Primary antibodies used for conjugation must be purchased without any additives or preservatives, many of which prevent successful antibody conjugation. Some additives can be removed but this can be difficult and inefficient.</div>
<div class="note">
<span class="note-title">Note:</span> Freshly conjugated antibodies produce excess background and should not be used for 3 days post-conjugation.</div>
<div class="note">
<span class="note-title">Note:</span> Conjugated antibodies can be kept within a refrigerator for at least 6 months after conjugation. This varies on the antibody, but we have successfully used custom conjugated antibodies over a year from initial conjugation with good results.</div>
</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Tissue preparation for CODEX (<a href="#bib5">Goltsev et al., 2018</a>, <a href="#bib15">Schürch et al., 2020</a>) multiplexed immunofluorescence analysis.
<ol type="a">
<li>Remove fresh frozen tissue from −80°C and place inside a desiccator for at least 20 min. Coverslips can be stored using a plastic petri dish.</li>
<li>Submerge tissues completely in acetone (∼5 mL) for 5 min using a glass petri dish.</li>
<li>Rehydrate tissues in hydration buffer (∼5 mL) for 3 min inside a humidity chamber.</li>
<li>Fix the tissues for 10 min in 1.6% paraformaldehyde in hydration buffer (∼5 mL).</li>
<li>Submerge tissues in excess staining buffer (∼5 mL) for 30 min using a plastic petri dish or other container.</li>
<li>Incubate tissues with a blocking buffer and primary antibody cocktail (1:200 antibody dilution, ∼200 μL) for 3 h within a humidity chamber (plastic tupperware with wet paper towel as an example) at ∼20°C.
<div class="note">
<span class="note-title">Note:</span> Tissues can be incubated in primary antibody solutions overnight if placed inside a refrigerator. Background may increase if longer incubation times are used but may be necessary for some antigen targets.</div>
<div class="note">
<span class="note-title">Note:</span> The volume of these reagents will depend on the size of the container that the coverslip is placed in. The reagents need to completely cover the coverslip and be in excess.</div>
</li>
<li>Rinse samples with staining buffer and fix the primary antibodies to the tissue with 1.6% paraformaldehyde in storage buffer (∼5 mL) for 10 min.</li>
<li>Rinse with excess PBS (∼5 mL) and incubate in cold methanol (∼5 mL) for 5 min. Rinse samples with excess PBS again (∼5 mL).</li>
<li>Fix the samples with fixative solution (∼5 mL) for 10 min before being stored in storage buffer at ∼4°C (∼5 mL).</li>
</ol>
</li>
</ol>
<ol start="15">
<li>Perform fluorescence microscopy and CODEX multiplexed immunofluorescence.
<ol type="a">
<li>Dilute antibodies in reporter solution to a 1:200 dilution with 1 μL of a 1 mg/mL solution of DAPI.
<div class="note">
<span class="note-title">Note:</span> Most secondary oligonucleotide barcode reports function at a 1:200 dilution; however, this dilution may need to be optimized in select cases or with custom reagents.</div>
</li>
<li>Fill respective bottles with nuclease free water, 1× CODEX buffer, and dimethyl sulfoxide.</li>
<li>Mount a blank coverslip into the coverslip microfluidic chamber of the CODEX instrument and flush the fluidic lines.</li>
<li>Mount the prepared tissue sample and incubate with 1× CODEX buffer and 1 μL of a 1 mg/mL of DAPI for 5 min and wash with the automated CODEX system.</li>
<li>Set up tiled regions covering the tissue and focus at the tissue surface.</li>
<li>Perform automated microscopy using a Zeiss Axio equipped with a Colibri 7 LED light source and C13440 camera or other equivalent equipment.
<ol type="i">
<li>Important parameters include a 10% tile overlap and a z-stack ranging from 11 to 20 tiles (depending on the tissue size).</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.6">Data processing and image registration</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 days</div>
<p>This protocol describes image registration, data processing, and analysis that integrates the different modalities produced by this workflow.</p>
<ol start="16">
<li>Data processing
<ol type="a">
<li>Convert Bruker (.d) raw file (or other raw data format) into a custom binary data format in profile mode at native spectral resolution.</li>
<li>Perform mass alignment and calibration of the dataset.
<ol type="i">
<li>Spectral misalignment is estimated for each pixel and corrected using PCHIP interpolation or linear shift. Alignment is carried out using a Python <i>msalign</i> package (<a href="https://github.com/lukasz-migas/msalign">https://github.com/lukasz-migas/msalign</a>). All spectra are first aligned based on automatically selected peaks and then recalibrated using at least three of the following well-characterized lipids:
<ul>
<li>Positive mode: <i>m/z</i> 703.5748, 734.5694, 760.5851, 810.6001.</li>
<li>Negative mode: <i>m/z</i> 673.4814, 744.5549, 834.5291, 885.5499.</li>
</ul>
</li>
</ol>
</li>
<li>Create a mean mass spectrum by taking the mean of each mass bin in the dataset.</li>
<li>Calculate the ppm error for common lipids, such as [PC(32:0)+H]<sup>+</sup> and [PC(34:1)+H]<sup>+</sup>.
<ol type="i">
<li>If the mass error is &gt;3 ppm, re-calibrate the entire dataset with additional well-characterized ions.</li>
</ol>
</li>
<li>Compute total ion current (TIC)-based normalization factors for each pixel after alignment and calibration had been performed.
<div class="note">
<span class="note-title">Note:</span> We employ TIC normalization to avoid outlier intensities skewing the normalization by imposing intensity quantile bounds on the area considered (i.e. only intensity values between the 0.05 and 0.95 intensity quantiles of each mass spectrum are considered). Other normalization procedures can be used and will depend on the mass spectrometer on which the data is acquired.</div>
</li>
<li>Perform peak picking on the mean mass spectrum and use the LIPIDMAPS database (<a href="#bib2">Fahy et al., 2009</a>, <a href="#bib3">Fahy et al., 2007</a>, <a href="#bib19">Sud et al., 2006</a>, <a href="#bib20">Sud et al., 2012</a>) to annotate each <i>m/z</i> value with a putative annotation.
<ol type="i">
<li>Use [M+H]<sup>+</sup>, [M+H-H2O]<sup>+</sup>, [M+Na]<sup>+</sup>, [M+K]<sup>+</sup> for positive mode adducts and [M+H]<sup>-</sup>, [M+Cl]<sup>-</sup>, [M+HCOO]<sup>-</sup>, [M+OAc]<sup>-</sup> for negative mode adducts.</li>
<li>Specify a narrow 0.005 <i>m/z</i> mass tolerance window.</li>
<li>Select “even chains only”.
<div class="note">
<span class="note-title">Note:</span> While we suggest these LIPIDMAPS search criteria, many of the adducts will depend on sample preparation methods and the mass tolerance window will depend on the mass accuracy and resolution of the mass spectrometer employed.</div>
</li>
</ol>
</li>
<li>Calculate the ppm error associated with each assignment and remove assignments with errors &gt;5 ppm, although most have errors &lt;3 ppm.</li>
<li>Extract peak intensity from the calibrated dataset using one of the available options:
<ol type="i">
<li>Extract ion images based on an untargeted feature list that has not been narrowed down by the previous step.</li>
<li>Extract ion images based on the identified feature list generated using the previous step.
<div class="note">
<span class="note-title">Note:</span> In either case, narrow 3–5 <i>m/z</i> bin extraction windows are created for each selected peak and the ion images are generated. The width of the extraction window is important because if it is too wide, several poorly resolved ions can be captured within one ion image.</div>
</li>
</ol>
</li>
<li>Export the extracted peak intensities to a columnar CSV file by concatenating the x and y-coordinates from the registration step with the preselected ions in the form: <i>x, y, m/z 1, m/z 2 … m/z N.</i>
</li>
<li>Read-in the table into Python and export the peak intensities to an imzML file using the Python <b>pyimzML</b> library (<a href="https://github.com/lukasz-migas/pyimzML">https://github.com/lukasz-migas/pyimzML</a>).</li>
</ol>
</li>
</ol>
<ol start="17">
<li>Image alignment and registration of each modality (<a href="#bib14">Patterson et al., 2018b</a>, <a href="#bib13">Patterson et al., 2018a</a>).
<ol type="a">
<li>Generate a MALDI imaging mass spectrometry pixel map using regToolboxMSRC (<a href="https://github.com/nhpatterson/regtoolboxmsrc">https://github.com/nhpatterson/regtoolboxmsrc</a>).</li>
<li>Using FIJI ImageJ, select corresponding laser ablation marks and IMS pixels in the post-acquisition autofluorescence image map and IMS pixel map, respectively.</li>
<li>Use a "Landmark Correspondences" FIJI plugin to find an affine transformation between the two images, resampling the post-AF image to the IMS pixel map by selecting post-AF image as source image, and IMS pixel map as the template image within the plugin.</li>
<li>Save the transformed and registered post-AF image.</li>
<li>Using <i>wsireg</i> (<a href="https://github.com/nhpatterson/wsireg">https://github.com/nhpatterson/wsireg</a>) to align other microscopy modalities, such as CODEX multiplexed IF, preIMS AF, PAS or any other associated microscopy image, to the newly saved post-AF image. <i>Wsireg</i> will generate pyramidal OME-TIFF images for all images after registration.</li>
<li>When complete, all images will be sampled in the same coordinates as the IMS pixel map and IMS can be overlaid onto microscopy by scaling the IMS data to the same resolution as the microscopy data.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol results in several different results and outcomes, because of its many procedures. In brief, mm<sup>3</sup> sized blocks of frozen tissue embedded in CMC should be obtained after collection and stabilization of human kidney tissue. The fresh frozen blocks can be stored for months without significant chemical degradation or loss of structure. In addition to a fresh frozen block, a portion of the kidney should be formalin-fixed and paraffin-embedded (FFPE) to obtain an FFPE tissue block that can be used for histological analysis and assessment. The FFPE block can be stored indefinitely at room temperature and within a humidity-controlled environment. PAS staining of sections (<a href="#fig8">Figure 8</a>A) from this FFPE block can be pathologically assessed and deemed normal or diseased prior to progression into the overall multimodal imaging workflow. 10 μm thick serial sections from the fresh frozen block are placed onto indium tin-oxide coated glass slides for MALDI IMS or glass coverslips for CODEX. Serial sections without tears, folds or wrinkles will then enable later image alignment and registration. Autofluorescence images at appropriate exposure times (<a href="#fig8">Figure 8</a>B) can be used to assess tissue integrity as well as label several cellular structures. MALDI IMS results in ∼100 lipids from positive ion mode analysis and an additional 100 lipids in negative ion mode. With proper sample preparation and laser focusing, 10 μm spatial resolution images without artifacts can be acquired (<a href="#fig8">Figure 8</a>C). CODEX IF sample preparation and imaging results in high signal-to-noise fluorescence images of up to 60 antibodies (<a href="#fig8">Figure 8</a>D). Datasets can be aligned and registered together for obtaining lipid profiles of defined cell types within the healthy human kidney.</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/928-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Multimodal images from the same kidney tissue section</div>
<p>Example images of a human kidney after following each step within the protocol: (A) PAS histological stain (B) autofluorescence microscopy image, (C) MALDI IMS of [SM(d34:1)+Na]<sup>+</sup> (green), [PC(32:0)+H]<sup>+</sup> (pink), and [PC(36:4)+H]<sup>+</sup> (blue), (D) and CODEX IF of aquaporin 1 (teal), aquaporin 2 (yellow), uromodulin (red), and vimentin (purple). Scale bars are 500 nm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocols described herein were developed for human kidney specimens acquired from a hospital within short walking distance of the research laboratory. While these protocols have been adapted for other organs, they may not work for all organs or for partially degraded tissue. Instrumental substitutions may also affect results and may not give outcomes and results we have achieved (e.g., 10 μm spatial resolutions for IMS or multiple staining cycles for CODEX IF). Tissues for both fresh frozen and FFPE protocols must be high quality and with minimalized times left unstabilized (an hour unstabilized maximum, although shorter is better).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Imaging mass spectrometry: During acquisition of mass spectrometry datasets, if there is little or no mass spectrometry signal within the lipid range this may be a result of sample preparation or instrumental issues.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Make sure matrix coating is uniform and appropriate thickness under a light microscope. If the matrix coating is patchy or too thin, additional matrix can be added. If the matrix coating is too thick, a new tissue will need to be used.</p>
<p>Verify instrument parameters and test instrument function on standards. If the instrument is working on standards, then the instrument is performing appropriately. If the standards are low intensity, clean the instrument or optimize instrument parameters, such as laser shots or laser intensity.</p>
<p>Verify the integrity of the sample by noting the time left unstabilized. If the sample is left unstabilized for too long, a new sample will need to be used.</p>
<p>Try a fresh tissue section or fresh matrix solution if nothing else explains the poor signal.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>Pathological Assessment of PAS stained FFPE Tissues: PAS stain does not result in appropriate staining of the human kidney, defined as either not staining or poor contrast.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Check the freshness of the solutions and make sure the solutions were stored appropriately. If the solutions are older than a month or not stored in a refrigerator, remake the solutions and try staining again.</p>
<p>Verify that the tissue is de-paraffinized (if appropriate) and the matrix is completely removed (if appropriate). Paraffin is completely removed if the tissue appears transparent and the wax is completely removed. Matrix is completely removed if the tissue appears transparent and the matrix layer is completely removed (not cloudy).</p>
<h3 id="sec5.5">Problem 3</h3>
<p>CODEX Multiplexed Immunofluorescence: Tissue does not fluoresce after CODEX immunofluorescence protocol. This is most notable when the image appears like the autofluorescence images acquired at the beginning or end of the procedure.</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Make sure microfluidics are working and not leaking. This is most easily seen with liquid pooling around the microfluidic system or microscope. Additionally, if the sample appears dry, the microfluidic is not dispensing sufficient volumes of solution.</p>
<p>Verify that correct oligonucleotide barcodes were incubated with the tissue and they match what is conjugated on the primary antibody.</p>
<p>Ensure that the microscope exposure times are appropriate (&gt;50 ms) and that the light path is not blocked.</p>
<p>Make sure that the tissue is adequately fixed and permeabilized. Mostly commonly a result from expired reagents or forgetting the step entirely.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>CODEX Multiplexed Immunofluorescence: One or two of the antibodies do not work after the CODEX IF protocol but other antibodies appear to stain appropriately.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Ensure that the correct antibody, oligonucleotide pair was added. If the wrong pair was added, the antibody cannot be visualized.</p>
<p>Try the antibody within a single-plex experiment without the microfluidic chamber to see if it stains. In brief, follow the CODEX IF staining protocol until insertion into the instrument. Instead, incubate with appropriate secondary barcodes, rinse in 5 mL of CODEX buffer three times, coverslip, and image on a microscope.</p>
<p>Try a different primary antibody clone or barcode. Sometimes the epitope of an antibody is damaged or altered upon the addition of a barcode. This can sometimes be circumvented by changing antibody clone which often binds a different epitope.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>Sample Preparation for Multimodal Workflows: Poor or low intensity fluorescence microscopy for either autofluorescence microscopy or CODEX multiplexed IF.</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Increase exposure times for each channel.</p>
<p>Try increasing the lamp/LED intensity for each channel if increasing the exposure times does not improve the image quality.</p>
<p>Verify the light path is not blocked and that each of the shutters are operating appropriately.</p>
<p>If using a lamp, check the age of the bulb on the lamp and verify it is not old and/or burnt out by referring to the manufacturer. If the bulb is old, replace the bulb.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Caprioli, R.M., Farmer, T.B., and Gile, J. (1997). Molecular imaging of biological samples: localization of peptides and proteins using maldi-tof ms. Anal. Chem. <i>69</i>, 4751-4760. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref1">View at publisher</a></p>
<p id="bib2">Fahy, E., Subramaniam, S., Murphy, R.C., Nishijima, M., Raetz, C.R.H., Shimizu, T., Spener, F., Van Meer, G., Wakelam, M.J.O., and Dennis, E.A. (2009). Update of the lipid maps comprehensive classification system for lipids. J. Lipid Res. <i>50</i>, s9-s14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref2">View at publisher</a></p>
<p id="bib3">Fahy, E., Sud, M., Cotter, D., and Subramaniam, S. (2007). Lipid maps online tools for lipid research. Nucleic Acids Res. <i>35</i>, w606-w612. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref3">View at publisher</a></p>
<p id="bib4">Gill, E.L., Yost, R.A., Vedam-Mai, V., and Garrett, T.J. (2017). Precast gelatin-based molds for tissue embedding compatible with mass spectrometry imaging. Anal. Chem. <i>89</i>, 576-580. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref4">View at publisher</a></p>
<p id="bib5">Goltsev, Y., Samusik, N., Kennedy-Darling, J., Bhate, S., Hale, M., Vazquez, G., Black, S., and Nolan, G.P. (2018). Deep profiling of mouse splenic architecture with codex multiplexed imaging. Cell <i>174</i>, 968-981.e15. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref5">View at publisher</a></p>
<p id="bib6">Hu, B.C. (2019). The human body at cellular resolution: the nih human biomolecular atlas program. Nature <i>574</i>, 187-192. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref6">View at publisher</a></p>
<p id="bib7">Lake, B.B., Chen, S., Hoshi, M., Plongthongkum, N., Salamon, D., Knoten, A., Vijayan, A., Venkatesh, R., Kim, E.H., Gao, D., et al. (2019). A single-nucleus rna-sequencing pipeline to decipher the molecular anatomy and pathophysiology of human kidneys. Nat. Commun. <i>10</i>, 2832. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref7">View at publisher</a></p>
<p id="bib8">McDonnell, L.A. and Heeren, R.M.A. (2007). Imaging mass spectrometry. Mass Spectrom. Rev. <i>26</i>, 606-643. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref8">View at publisher</a></p>
<p id="bib9">Munro, B. (1971). Manual of histologic staining methods of the armed forces institute of pathology. Pathology <i>3</i>, 249. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref9">View at publisher</a></p>
<p id="bib10">Neumann, E.K., Djambazova, K.V., Caprioli, R.M., and Spraggins, J.M. (2020a). Multimodal imaging mass spectrometry: next generation molecular mapping in biology and medicine. J. Am. Soc. Mass Spectrom. <i>31</i>, 2401-2415. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref10">View at publisher</a></p>
<p id="bib11">Neumann, E.K., Do, T.D., Comi, T.J., and Sweedler, J.V. (2019). Exploring the fundamental structures of life: non-targeted, chemical analysis of single cells and subcellular structures. Angew. Chem. Int. Ed. <i>58</i>, 9348-9364. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref11">View at publisher</a></p>
<p id="bib12">Neumann, E.K., Rivera, E.S., Patterson, N.H., Allen, J.L., Decaestecker, M.P., Fogo, A.B., Caprioli, R.M., and Spraggins, J.M. (2020b). Highly multiplexed immunofluorescence of the human kidney using co-detection by indexing (codex). biorxiv. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref12">View at publisher</a></p>
<p id="bib13">Patterson, N.H., Tuck, M., Lewis, A., Kaushansky, A., Norris, J.L., Van De Plas, R., and Caprioli, R.M. (2018a). Next generation histology-directed imaging mass spectrometry driven by autofluorescence microscopy. Anal. Chem. <i>90</i>, 12404-12413. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref13">View at publisher</a></p>
<p id="bib14">Patterson, N.H., Tuck, M., Van De Plas, R., and Caprioli, R.M. (2018b). Advanced registration and analysis of maldi imaging mass spectrometry measurements through autofluorescence microscopy. Anal. Chem. <i>90</i>, 12395-12403. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref14">View at publisher</a></p>
<p id="bib15">Schürch, C.M., Bhate, S.S., Barlow, G.L., Phillips, D.J., Noti, L., Zlobec, I., Chu, P., Black, S., Demeter, J., Mcilwain, D.R., et al. (2020). Coordinated cellular neighborhoods orchestrate antitumoral immunity at the colorectal cancer invasive front. Cell <i>182</i>, 1341-1359.e19. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref15">View at publisher</a></p>
<p id="bib16">Spraggins, J.M., Djambazova, K.V., Rivera, E.S., Migas, L.G., Neumann, E.K., Fuetterer, A., Suetering, J., Goedecke, N., Ly, A., Van De Plas, R., et al. (2019). High-performance molecular imaging with maldi trapped ion-mobility time-of-flight (timstof) mass spectrometry. Anal. Chem. <i>91</i>, 14552-14560. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref16">View at publisher</a></p>
<p id="bib17">Stoeckli, M., Staab, D., and Schweitzer, A. (2007). Compound and metabolite distribution measured by maldi mass spectrometric imaging in whole-body tissue sections. Int. J. Mass Spectrom. <i>260</i>, 195-202. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref17">View at publisher</a></p>
<p id="bib18">Strohalm, M., Strohalm, J., Kaftan, F., Krásný, L., Volný, M., Novák, P., Ulbrich, K., and Havlíček, V. (2011). Poly[n-(2-hydroxypropyl)methacrylamide]-based tissue-embedding medium compatible with maldi mass spectrometry imaging experiments. Anal. Chem. <i>83</i>, 5458-5462. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref18">View at publisher</a></p>
<p id="bib19">Sud, M., Fahy, E., Cotter, D., Brown, A., Dennis, E.A., Glass, C.K., Merrill, A.H., Murphy, R.C., Raetz, C.R.H., Russell, D.W., et al. (2006). Lmsd: lipid maps structure database. Nucleic Acids Res. <i>35</i>, d527-d532. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref19">View at publisher</a></p>
<p id="bib20">Sud, M., Fahy, E., Cotter, D., Dennis, E.A., and Subramaniam, S. (2012). Lipid maps-nature lipidomics gateway: an online resource for students and educators interested in lipids. J. Chem. Ed. <i>89</i>, 291-292. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00454-8/sref20">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jeffrey M. Spraggins (<a href="mailto:Jeff.Spraggins@Vanderbilt.edu">Jeff.Spraggins@Vanderbilt.edu</a>).</p>
<h4>Materials availability</h4>
<p>There are restrictions to the availability of custom conjugated antibodies due to the limited amount of product and resources required to produce them. No other unique reagents are generated in this protocol.</p>
<h4>Data and code availability</h4>
<p><i>The published articles include all code generated or analyzed during this study (</i>10.1021/acs.analchem.8b02885 and 10.1021/acs.analchem.8b02884)<i>. The datasets generated during this study are available at the Human BioMolecular Atlas Program Data Portal [</i><a href="https://portal.hubmapconsortium.org/">https://portal.hubmapconsortium.org/</a><i>].</i></p>
<h3>Acknowledgments</h3>
<p>Support was provided by the <a href="https://doi.org/10.13039/100000002">NIH</a> Common Fund and <a href="https://doi.org/10.13039/100000062">National Institute of Diabetes and Digestive and Kidney Diseases</a> (U54DK120058 awarded to J.M.S. and R.M.C.), <a href="https://doi.org/10.13039/100000002">NIH</a> National Institute of Allergy and Infectious Disease (R01 AI138581 and R01 AI145992 awarded to J.M.S.), the <a href="https://doi.org/10.13039/100000001">National Science Foundation</a> Major Research Instrument Program (CBET–1828299 awarded to J.M.S. and R.M.C.), and the <a href="https://doi.org/10.13039/100000002">NIH</a> National Institute of General Medical Sciences (2P41GM103391 awarded to R.M.C.). E.K.N. is supported by a <a href="https://doi.org/10.13039/100000066">National Institute of Environmental Health Sciences</a> training grant (T32ES007028). The Cooperative Human Tissue Network is supported by the <a href="https://doi.org/10.13039/100000002">NIH</a> National Cancer Institute (5 UM1 CA183727-08).</p>
<h3>Author contributions</h3>
<p>E.K.N., N.H.P., R.C.H., M.P.d., A.B.F., R.M.C., R.V.d.P., and J.M.S. conceived the study and designed the experiments. J.M.S., R.V.d.P., D.B.G., M.P.d., and A.B.F. supervised this work. Imaging MS, histological staining, microscopy, and CODEX immunostaining were performed, and protocols were written by E.K.N. Sample preservation protocols were performed and written by J.L.A. and M.B. The cryosectioning work was performed and written by J.L.A. and D.M.A. The RNA assessment protocol was performed by E.K.N. and M.B. Methods for histological assessment were developed and written by H.Y. and A.B.F. Image registration and analysis protocols were developed and written by N.H.P. Data processing was written by L.G.M. All authors have edited and approved the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>