<article>
<section>
<h1 id="header">Glioblastoma patient-derived cell-based phenotypic drug screening and identification of possible action mechanisms through proteomic analysis</h1>
<p><time datetime="2021-10-01">Published: October 1, 2021</time></p>
<p>Young Eun Kim,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Hyun Young Kim,<sup><a href="#aff1">1</a></sup> Daeyoung Jung,<sup><a href="#aff1">1</a></sup> Dukjin Kang,<sup><a href="#aff2">2</a></sup> Do-Hyun Nam,<sup><a href="#aff3">3</a></sup> Hye Jin Nam,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> and Heeyeong Cho<sup><a href="#aff1">1</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Drug Discovery Platform Research Center, Therapeutics and Biotechnology Division, Korea Research Institute of Chemical Technology, Daejeon 34114, Republic of Korea</p>
<p id="aff2"><sup>2</sup>Center for Bioanalysis, Division of Chemical and Medical Metrology, Korea Research Institute of Standards and Science, Daejeon 34113, Republic of Korea</p>
<p id="aff3"><sup>3</sup>Department of Neurosurgery, Samsung Medical Center (SMC), Sungkyunkwan University School of Medicine, Seoul 06351, Republic of Korea</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:hjnam@krict.re.kr">hjnam@krict.re.kr</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:hycho@krict.re.kr">hycho@krict.re.kr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100849">10.1016/j.xpro.2021.100849</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Because glioblastoma (GBM) exhibits high heterogeneity, it is desirable to use patient-derived cells from the first stage of screening for GBM drug discovery. Here, we describe a protocol to culture patient-derived GBM cells on the extracellular matrix-coated plates to allow high-throughput screening. Further, we detail approaches to identify the mechanism of action (MOA) of the selected effective drug through proteomics. This protocol will be useful for researchers interested in drug screening and the MOA of drugs.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.isci.2021.102238">Nam et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Methods for patient-derived GBM cell culture</li>
<li>Automated high-throughput screening methods using chemical libraries</li>
<li>Quantitative mass spectrometry to elucidate mechanism of action of drugs</li>
<li>Methods to perform data analysis from mass spectrometry</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Preparation of patient-derived GBM cells</h3>
<p>The Primary GBM cells used in this protocol were kindly gifted by Dr. Do-Hyun Nam (Samsung Medical Center, Seoul, Korea), and the patients provided written informed consent in accordance with the Institutional Review Boards (IRB No. 2005-04-001, 2010-04-004). Pathologists classified tumors as GBM according to the WHO criteria (<a href="#bib3">Louis et al., 2007</a>). Surgically isolated tumor tissues were mechanically dissected, followed by enzymatic dissociation and single-cell purification (<a href="#bib1">Joo et al., 2008</a>). Human GBM subtypes were assigned based on their distinct gene signatures by the experiments using RNA expression arrays (<a href="#bib2">Joo et al., 2013</a>).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Formamide</td>
<td>Sigma-Aldrich</td>
<td>Cat# F9037</td>
</tr>
<tr>
<td>Potassium silicate solution</td>
<td>RS Chem</td>
<td>Cat# PS-820S</td>
</tr>
<tr>
<td>Urea</td>
<td>Sigma-Aldrich</td>
<td>Cat# U5378</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Sigma-Aldrich</td>
<td>Cat# S7653</td>
</tr>
<tr>
<td>cOmplete, Mini Protease Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat# 11836153001</td>
</tr>
<tr>
<td>1 M HEPES</td>
<td>Gibco</td>
<td>Cat# 15630-056</td>
</tr>
<tr>
<td>Dithiothreitol</td>
<td>Sigma-Aldrich</td>
<td>Cat# D0632</td>
</tr>
<tr>
<td>Iodoacetamide</td>
<td>Sigma-Aldrich</td>
<td>Cat# I1149</td>
</tr>
<tr>
<td>L-Cysteine</td>
<td>Sigma-Aldrich</td>
<td>Cat# 168149</td>
</tr>
<tr>
<td>Trypsin, sequencing grade</td>
<td>Promega</td>
<td>Cat# V5111</td>
</tr>
<tr>
<td>1 M Triethylammonium bicarbonate, pH 8.5</td>
<td>Sigma-Aldrich</td>
<td>Cat# T7408</td>
</tr>
<tr>
<td>Water, HPLC grade</td>
<td>Honeywell</td>
<td>Cat# AH365-4</td>
</tr>
<tr>
<td>Acetonitrile, HPLC grade</td>
<td>Honeywell</td>
<td>Cat# AH015-4</td>
</tr>
<tr>
<td>Methanol, HPLC grade</td>
<td>Fisher Scientific</td>
<td>Cat# A-452-4</td>
</tr>
<tr>
<td>FDA-approved drug (640 drugs)</td>
<td>Enzo Life Sciences</td>
<td>Cat# BML-2841</td>
</tr>
<tr>
<td>NIH clinical collection (446 drugs)</td>
<td>Evotec (BioFocus)</td>
<td>Cat# NCC library</td>
</tr>
<tr>
<td>Neurobasal-A medium</td>
<td>Gibco</td>
<td>Cat# 10888-022</td>
</tr>
<tr>
<td>N-2 supplement</td>
<td>Gibco</td>
<td>Cat# 17502-048</td>
</tr>
<tr>
<td>B-27 supplement, minus vitamin A</td>
<td>Gibco</td>
<td>Cat# 12587-010</td>
</tr>
<tr>
<td>GlutaMAX-I</td>
<td>Gibco</td>
<td>Cat# 35050-061</td>
</tr>
<tr>
<td>Penicillin-streptomycin (10,000 U/mL)</td>
<td>Gibco</td>
<td>Cat# 15140-148</td>
</tr>
<tr>
<td>Recombinant human EGF</td>
<td>R&amp;D Systems</td>
<td>Cat# 236-EG</td>
</tr>
<tr>
<td>Recombinant hyman FGF basic (bFGF)</td>
<td>R&amp;D Systems</td>
<td>Cat# 3718-FB</td>
</tr>
<tr>
<td>Accumax</td>
<td>Innovative Cell Technologies</td>
<td>Cat# AM105</td>
</tr>
<tr>
<td>Laminin</td>
<td>Sigma-Aldrich</td>
<td>Cat# L2020</td>
</tr>
<tr>
<td>Fibronectin</td>
<td>Gibco</td>
<td>Cat# 33016-015</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>iTRAQ Reagents Multiplex Kit (4plex)</td>
<td>AB Sciex</td>
<td>Cat# 4352135</td>
</tr>
<tr>
<td>BCA Protein Assay Kit</td>
<td>Thermo Scientific</td>
<td>Cat# 23225</td>
</tr>
<tr>
<td>Cyto X</td>
<td>LPS Solution</td>
<td>Cat# CYT3000</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>559T</td>
<td>Dr. Do-Hyun Nam</td>
<td>N/A</td>
</tr>
<tr>
<td>592T</td>
<td>Dr. Do-Hyun Nam</td>
<td>N/A</td>
</tr>
<tr>
<td>626T</td>
<td>Dr. Do-Hyun Nam</td>
<td>N/A</td>
</tr>
<tr>
<td>A549</td>
<td>ATCC</td>
<td>Cat# CCL-185;RRID:CVCL_0023</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>MaxQuant</td>
<td><a href="#bib6">(Cox and Mann, 2008)</a></td>
<td>
<a href="http://www.maxquant.org">http://www.maxquant.org</a>,</td>
</tr>
<tr>
<td>Perseus</td>
<td><a href="#bib7">(Tyanova et al., 2016)</a></td>
<td><a href="http://www.maxquant.org/perseus/">http://www.maxquant.org/perseus/</a></td>
</tr>
<tr>
<td>GraphPad Prism</td>
<td>GraphPad Software</td>
<td>N/A</td>
</tr>
<tr>
<td>KEGG pathway</td>
<td><a href="#bib8">(Kanehisa et al., 2012)</a></td>
<td><a href="https://www.genome.jp/kegg/">https://www.genome.jp/kegg/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Fused silica capillary tubing (I.D. 200 μm and O.D. 350 μm)</td>
<td>Polymicro</td>
<td>Cat# 1068150204</td>
</tr>
<tr>
<td>Fused silica capillary tubing (I.D. 75 μm and O.D. 350-μm)</td>
<td>Polymicro</td>
<td>Cat# 1068150019</td>
</tr>
<tr>
<td>Fused Silica capillary tubing (I.D. 25 μm and O.D. 350-μm)</td>
<td>Polymicro</td>
<td>Cat#1068150011</td>
</tr>
<tr>
<td>ProntoSIL C18 AQ, 5 μm-200 Å</td>
<td>ProntoSIL</td>
<td>Cat# 2502H184PS050</td>
</tr>
<tr>
<td>Unisol C18, 3 μm-100 Å</td>
<td>Bonna-Agela Technologies</td>
<td>Cat# UO921005-0</td>
</tr>
<tr>
<td>PolySULFOETHYL A, 5 μm-200 Å</td>
<td>PolyLC</td>
<td>Cat# 204SE0502</td>
</tr>
<tr>
<td>Pressure bomb</td>
<td>Next Advance</td>
<td>Cat# PC77</td>
</tr>
<tr>
<td>MicroCross PEEK 360 μm w/Fittings</td>
<td>IDEX Health &amp; Science</td>
<td>Cat# P-889</td>
</tr>
<tr>
<td>MicroTee PEEK 360 μm w/Fittings</td>
<td>IDEX Health &amp; Science</td>
<td>Cat# P-888</td>
</tr>
<tr>
<td>Stereoscopic microscope</td>
<td>Nikon</td>
<td>Cat# SMZ745</td>
</tr>
<tr>
<td>Ultrasonic homogenizer</td>
<td>Omni</td>
<td>Cat# Sonic Ruptor 400</td>
</tr>
<tr>
<td>Thermomixer</td>
<td>Eppendorf</td>
<td>Cat# 5355000.011</td>
</tr>
<tr>
<td>SpeedVac concentrator</td>
<td>Eppendorf</td>
<td>N/A</td>
</tr>
<tr>
<td>Oasis HLB extraction cartridges</td>
<td>Waters</td>
<td>Cat# 186000383</td>
</tr>
<tr>
<td>384-Well cell culture plate</td>
<td>SPL</td>
<td>Cat# 37384</td>
</tr>
<tr>
<td>384-Well, PP, v-bottom plate</td>
<td>Greiner</td>
<td>Cat# 781280</td>
</tr>
<tr>
<td>384-Well P30XL tips, sterile</td>
<td>Beckman Coulter</td>
<td>Cat# A22288</td>
</tr>
<tr>
<td>EnVision Multilabel Reader</td>
<td>PerkinElmer</td>
<td>Cat# 2103-0010</td>
</tr>
<tr>
<td>Robot system for HTS</td>
<td>Beckman Coulter</td>
<td>Cat# Scara Robot</td>
</tr>
<tr>
<td>Liquid handling system for HTS</td>
<td>Beckman Coulter</td>
<td>Cat# Biomek NX<sup>p</sup>
</td>
</tr>
<tr>
<td>Microplate dispenser for HTS</td>
<td>BioTek</td>
<td>Cat# MultiFlo FX</td>
</tr>
<tr>
<td>Microplate washer for HTS</td>
<td>BioTek</td>
<td>Cat# EL405</td>
</tr>
<tr>
<td>CO2 incubator with plate shuttle system for HTS</td>
<td>Thermo Fisher</td>
<td>Cat# CytoMAT 2C</td>
</tr>
<tr>
<td>De-lidder for HTS</td>
<td>Beckman Coulter</td>
<td>Cat# A70963</td>
</tr>
<tr>
<td>Hotel Carousel for HTS</td>
<td>Thermo Fisher</td>
<td>Cat# CytoMAT 51021435</td>
</tr>
<tr>
<td>SAMI EX, version 4.1</td>
<td>Beckman Coulter</td>
<td>N/A</td>
</tr>
<tr>
<td>Orbital shaker for HTS</td>
<td>Beckman Coulter</td>
<td>Cat# BioShaker 3000</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Patient-derived GBM cell culture media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neurobasal-A medium</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
<tr>
<td>N2 (100<b>×</b>)</td>
<td>0.5<b>×</b>
</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>B27 (50<b>×</b>)</td>
<td>0.5<b>×</b>
</td>
<td>5 mL</td>
</tr>
<tr>
<td>Glutamax (100<b>×</b>)</td>
<td>1<b>×</b>
</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin/streptomycin (100<b>×</b>)</td>
<td>1<b>×</b>
</td>
<td>5 mL</td>
</tr>
<tr>
<td>EGF (100 μg/mL)</td>
<td>100 ng/mL</td>
<td>500 μL</td>
</tr>
<tr>
<td>bFGF (100 μg/mL)</td>
<td>100 ng/mL</td>
<td>500 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, and do not store more than 3 months.</p>
<table id="undtbl2">
<caption>Plate coating solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Laminin (1 mg/mL)</td>
<td>15 μg/mL</td>
<td>15 μL</td>
</tr>
<tr>
<td>Fibronectin (500 μg/mL)</td>
<td>3 μg/mL</td>
<td>6 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>N/A</td>
<td>979 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Make the required amount fresh just before use.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Laminin should be thawed slowly at 4°C and diluted in a gel state (less melted), as quick thawing reduces the coating ability. Fibronectin should be reconstituted in distilled water (DW). Vortexing or excessive agitation of the fibronectin solutions should be avoided.</div>
<table id="undtbl3">
<caption>Amount of plate coating solution required depending on the type of well plate</caption>
<thead>
<tr>
<th>Plate type</th>
<th>Coating solution volume/well</th>
</tr>
</thead>
<tbody>
<tr>
<td>384-well</td>
<td>20 μL</td>
</tr>
<tr>
<td>96-well</td>
<td>50 μL</td>
</tr>
<tr>
<td>24-well</td>
<td>250 μL</td>
</tr>
<tr>
<td>6-well</td>
<td>1 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The amount of coating solution depends on the type of cell culture plate used.</div>
<p><b>Cell lysis buffer for LC-MS/MS analysis (8 M urea, 75 mM sodium chloride, 50 mM hydroxyethyl piperazine ethane sulfonic acid (HEPES) (pH 7.4), and protease inhibitor cocktail):</b> Dissolve 4.8 g urea, 0.044 g sodium chloride, and one protease inhibitor cocktail tablet in high-performance liquid chromatography (HPLC)-grade water. Add 0.5 mL of 1M HEPES solution (pH 7.4) and add HPLC-grade water to a total volume of 10 mL. Prepare the solution fresh before each use.</p>
<p><b>100 mM dithiothreitol (DTT) in 500 mM triethylammonium bicarbonate</b>: Dissolve 15.42 mg DTT in 1 mL of 500 mM triethylammonium bicarbonate. Prepare the solution fresh before each use.</p>
<p><b>270 mM iodoacetamide (IAA) in 50 mM triethylammonium bicarbonate</b>: Dissolve 50 mg IAA in 1 mL of 50 mM triethylammonium bicarbonate. Prepare the solution fresh before each use.</p>
<div class="note">
<span class="note-title">Note:</span> IAA is sensitive to light. Therefore, this solution should be kept in the dark to avoid light exposure.</div>
<p><b>400 mM L-cysteine in 50 mM triethylammonium bicarbonate</b>: Dissolve 48.64 mg L-cysteine in 1 mL of 50 mM triethylammonium bicarbonate. Prepare the solution fresh before each use.</p>
<p><b>Liquid chromatography (LC) mobile phase A: 0.1% (v/v) formic acid in water</b>: Add 1 mL formic acid to 999 mL HPLC-grade water. Mix by inversion. The solution can be stored at room temperature (15°C–25°C) for up to 3 months.</p>
<p><b>LC mobile phase B: 0.1% (v/v) formic acid in 98% (v/v) acetonitrile</b>: Add 1 mL formic acid and 20 mL HPLC-grade water to 979 mL acetonitrile. Mix by inversion. The solution can be stored at room temperature (15°C–25°C) for up to 3 months.</p>
<p><b>SCX elution buffers (20, 22, 24, 26, 28, 30, 35, 40, 60, 100, and 1000 mM ammonium bicarbonate)</b> Dissolve 3.95 g ammonium bicarbonate in 50 mL of HPLC-grade water with 0.1% (v/v) formic acid to prepare 1000 M ammonium bicarbonate. Dilute 1000 mM ammonium bicarbonate to 20, 22, 24, 26, 28, 30, 35, 40, 60, and 100 mM by adding HPLC-grade water with 0.1% (v/v) formic acid. The solution can be stored at room temperature (15°C–25°C) for up to 3 months.</p>
<p><b>LC-mass spectrometry (MS)/MS System setup</b></p>
<p>In this protocol, nanoflow LC-ESI-MS/MS experiments were carried out using a 1260 capillary LC system (Agilent Technologies, Waldbronn, Germany) interfaced with a Q Exactive™ Hybrid-Quadrupol-Orbitrap mass spectrometer (Thermo Fisher Scientific, Bremen, Germany). The C18/SCX biphasic trap column was connected to an RP analytical column via a PEEK micro-cross. The open end of the trap column was connected to the PEEK microtee to split the flow from the HPLC pump. The sample or salt solution was injected from the autosampler onto the trap column at a flow rate of 4 μL/min (valve position A). After injection, the 6-port valve was switched to position B, and peptides were loaded onto the RP analytical column. The column flow rate of 200 nL/min was adjusted with a splitter (long length of capillary tubing, I.D. 25 μm). The LC gradient conditions are listed in <a href="#tbl1">Table 1</a>. The MS instrument was operated in data-dependent acquisition (DDA) mode. The MS parameters are listed in <a href="#tbl2">Table 2</a>.</p>
<table id="tbl1">
<caption>Table 1. Liquid chromatography gradient conditions</caption>
<thead>
<tr>
<th>Time (min)</th>
<th>Mobile phase B (%)</th>
<th>Column flow rate (nL/min)</th>
<th>Valve configuration</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>2</td>
<td>Sample loading</td>
<td>A</td>
</tr>
<tr>
<td>10</td>
<td>2</td>
<td>Sample loading</td>
<td>A</td>
</tr>
<tr>
<td>10.50</td>
<td>8</td>
<td>200</td>
<td>B</td>
</tr>
<tr>
<td>15</td>
<td>15</td>
<td>200</td>
<td>B</td>
</tr>
<tr>
<td>85</td>
<td>30</td>
<td>200</td>
<td>B</td>
</tr>
<tr>
<td>88</td>
<td>90</td>
<td>200</td>
<td>B</td>
</tr>
<tr>
<td>103</td>
<td>90</td>
<td>200</td>
<td>B</td>
</tr>
<tr>
<td>105</td>
<td>2</td>
<td>200</td>
<td>B</td>
</tr>
<tr>
<td>120</td>
<td>2</td>
<td>200</td>
<td>B</td>
</tr>
</tbody>
</table>
<table id="tbl2">
<caption>Table 2. MS/MS parameters</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>Polarity</td>
<td>Positive</td>
</tr>
<tr>
<td colspan="2">Full MS</td>
</tr>
<tr>
<td>Resolution</td>
<td>70,000</td>
</tr>
<tr>
<td>Automatic Gain Control (AGC) target</td>
<td>3 × 10<sup>6</sup>
</td>
</tr>
<tr>
<td>Maximum IT</td>
<td>80 ms</td>
</tr>
<tr>
<td>Scan range</td>
<td>300–1800 <i>m/z</i>
</td>
</tr>
<tr>
<td colspan="2">Data dependent MS/MS</td>
</tr>
<tr>
<td>Resolution</td>
<td>35,000</td>
</tr>
<tr>
<td>Automatic Gain Control (AGC) target</td>
<td>1 × 10<sup>6</sup>
</td>
</tr>
<tr>
<td>Maximum IT</td>
<td>100 ms</td>
</tr>
<tr>
<td>Loop count</td>
<td>12</td>
</tr>
<tr>
<td>Isolation window</td>
<td>2.0 <i>m/z</i>
</td>
</tr>
<tr>
<td>Fixed first mass</td>
<td>100 <i>m/z</i>
</td>
</tr>
<tr>
<td>Normalization collision energy (%)</td>
<td>27</td>
</tr>
<tr>
<td>Charge exclusion</td>
<td>1</td>
</tr>
<tr>
<td>Dynamic exclusion</td>
<td>30 s</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Patient-derived GBM cell culture</h3>
<p>This protocol describes a method to passage and store patient-derived GBM cell culture. This method consists of the following steps: (1) thawing, (2) passaging, and (3) cryo-freezing for preservation.</p>
<h4 id="sec3.1.1">Thawing</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 10–20 min</div>
<ol>
<li>Transfer quickly cryo-frozen vails to a 37°C water bath and thaw cryo-frozen cells in a 37°C water bath until the freezing medium is approximately 70% thawed.</li>
<li>Transfer the contents of the cryovial into a sterile 15 mL conical tube containing 9 mL of culture media.</li>
<li>Centrifuge the cells at 200 <b>×</b> <i>g</i> for 3 min at room temperature (15°C–25°C).</li>
<li>Aspirate and remove the supernatant and resuspend the cell pellet in 1 mL of culture medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Excessive pipetting can damage cells. Gently pipette the cells up and down in the tube a few times.</div>
<ol start="5">
<li>Add the cells to a T-75 flask containing 15 mL of culture media.</li>
<li>Gently move the flask back and forth and side to side to evenly distribute the cells and place the T flask in a CO<sub>2</sub> incubator (37°C, 5% CO<sub>2</sub>).</li>
<li>Add 5 mL of culture media every 3–4 days until the GBM cells are ready for passaging.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Patient-derived GBM cells (559T, 592T, and 626T) grow in suspension. Over time, the GBM cells aggregate and grow into a spherical shape.</div>
<div class="note">
<span class="note-title">Note:</span> The first passaging can be performed approximately 10–14 days after thawing. However, the length of time may vary depending on the condition of the cells or the number of frozen cells. The proper passaging time should be checked by continuous microscopic observations. It is recommended that cells be passaged when they reach a sphere size of about 200 μm (<a href="#fig1">Figure 1</a>).</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Representative images of GBM cells</div>
<p>559T and 592T cell images captured using 4<b>×</b> objective. Scale bars, 200 μm.</p>
</figcaption>
</figure>
<h4 id="sec3.1.2">Passaging</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="8">
<li>Transfer the entire culture medium from a T-75 flask containing the suspended cells to a 50 mL conical tube.</li>
<li>Centrifuge the conical tube at 200 <b>×</b> <i>g</i> for 3 min at room temperature (15°C–25°C).</li>
<li>Remove the supernatant by aspiration and resuspend the cells in 500 μL of accumax.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Accutase (Invitrogen, Cat# 00-4555-56) can be used instead of accumax.</div>
<ol start="11">
<li>Incubate for 3 min at room temperature (15°C–25°C).</li>
<li>For single cell dissociation, dissociate spheres by gentle pipetting up and down about 20 times.</li>
<li>Add 5 mL of media and centrifuge at 200 <b>×</b> <i>g</i> for 3 min at room temperature (15°C–25°C).</li>
<li>Aspirate and resuspend cells in 3–5 mL of culture media and add the cell suspension to a T-75 flask containing 20–25 mL media in a 1:5- 1:15 ratio.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If there are too many cells, they can stick to the bottom and differentiate. It is important to maintain adequate confluence to avoid losing the original GBM characteristics of the cells. For us, the split ratio was variable. Occasionally, cells grow at different rates, and the split ratio will need to be adjusted.</div>
<ol start="15">
<li>GBM cells are ready for the next passaging after 7–14 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Patient-derived GBM cells (559T, 592T, and 626T) grow at different rates. 592T cells grew the fastest and 626T cells grew the slowest. The proper passaging time was confirmed through continuous microscopic observations. It is recommended to passage the cells when they reach a sphere size of approximately 200 μm (<a href="#fig1">Figure 1</a>).</div>
<ol start="16">
<li>Add 5 mL of culture media every 3–4 days until the GBM cells are ready for passaging.</li>
</ol>
<h4 id="sec3.1.3">Cryo-freezing</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 30–45 min</div>
<ol start="17">
<li>Prepare the GBM cells dissociated into single cells by accumax using the same method as mentioned above (step 8–13).</li>
<li>Add freezing media instead of the culture media depending on the amount of the cells, and gently pipette to evenly mix the suspension.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Freezing media are prepared by mixing GBM culture media with 5% dimethyl sulfoxide (DMSO). The entire culture and storage protocol for GBM does not contain fetal bovine serum (FBS), as this may cause the loss of GBM stemness.</div>
<ol start="19">
<li>Aliquot 1 mL of cell suspension into each cryovial and place into an isopropanol-containing freezing container.</li>
<li>Place the freezing container in a −80°C deep freezer for 24 h.</li>
<li>Transfer the cryovials to a liquid nitrogen tank (−135°C) for long-term storage.</li>
</ol>
<h3 id="sec3.2">HTS using patient-derived GBM cells</h3>
<p>A fully automated HTS system was used to quickly and efficiently measure the viability of GBM cells against drug treatment. The entire experiment took a total of 5 days. It is possible to implement a fully automated connection from day 1 to day 5 of the test, but this has the disadvantage that the system cannot be used for other purposes during the long-term incubation time. Therefore, to increase system usability, suitable protocols were prepared and operated every day. One test set contains different types of cells with various concentrations of test compound libraries. It is important to determine the reproducibility of the test by repeating 2–3 test sets on different days.</p>
<p>Total experimental timing: 5 days</p>
<p>The main process and/or brief purpose of the entire HTS assay divided by date are listed below.</p>
<p>Day 0 (Day before the experiment): Plate coating</p>
<p>Day 1: Cell seeding</p>
<p>Day 2: Compound dilution and treatment</p>
<p>Days 3–4: Incubation</p>
<p>Day 5: Read out</p>
<h4 id="sec3.2.1">Day 0 (the day before the experiment): Plate coating</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>The day before the experiment, Multiflo FX drop-dispenser and EL405 plate washer were used to coat plates. The plate coating protocol was created and scheduled with SAMI EX 4.1 software and performed (<a href="#fig2">Figure 2</a>). The summary of each step is as follows.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Plate coating protocol for automated HTS systems</div>
<p>The plate coating protocol was created and scheduled using the SAMI EX 4.1 software. The entire process of coating on 30 plates, including the 2 h incubation, takes 3 h and 8 min.</p>
</figcaption>
</figure>
<ol start="22">
<li>Prepare a coating solution according to the number of plates to be used.</li>
<li>Load 20 μL/well of coating solution into 384-well plates.</li>
<li>Incubate the plates at 37°C for 2 h.</li>
<li>Following incubation, remove the coating solution by aspiration, and wash the wells three times with phosphate-buffered saline (PBS).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The MultiFlow FX drop-dispenser has various types of cassettes that can be used (e.g., 1, 5, and 10 μL). The 10 μL cassette was most suitable for dispensing 20 μL of the coating solution. The coating solution was removed with aspiration pins during the EL405 plate washing step. The aspiration pins should be adjusted to an appropriate height so as not to scratch the bottom of the plates. After the washing step is completed, the plates are kept at room temperature (15°C–25°C) and used the following day.</div>
<h4 id="sec3.2.2">Day 1: Cell seeding</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>A sufficient amount of cells should be prepared by predicting the number of cells required for the experiment in advance. Cells were maintained and cultured as mentioned above (step 8–16). A Multiflo FX drop-dispenser was used to seed cells into 384-well-plates. The seeding protocol was created and scheduled using the SAMI EX software and was performed (<a href="#fig3">Figure 3</a>). Two different 10 μL cassettes were used to simultaneously separate and dispense wells with and without cells. After dispensing one type of cells into plates, the cassettes were replaced to prevent intercellular contamination. Each step is summarized as follows.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Cell seeding protocol for automated HTS systems</div>
<p>The seeding protocol was created and scheduled using SAMI EX software. It took 1 h and 4 min to seed the three types of cells into 30 plates. When replacing the tip cassette for new cell seeding, the entire system is temporarily paused. The process resumes after the tip cassette is changed.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> All cassettes must be autoclaved prior to use.</div>
<div class="note">
<span class="note-title">Note:</span> The standard treatment for GBM patients involves surgical removal of the tumor, radiotherapy, and adjuvant chemotherapy with temozolomide (TMZ). 559T and 592T GBM cells are resistant to TMZ, whereas 626T cells are sensitive. As we aimed to find effective drugs against TMZ resistance, 559T and 592T cells were used for HTS.</div>
<ol start="26">
<li>Seed GBM cells in 384-well laminin-fibronectin-coated plates at the following density in 50 μL media/well. A549 cells were seeded into 384-well tissue culture plates. The A549 cell line was used to determine the specificity of the drug to GBM cells. To measure absorption value of the blank solution, cells were not seeded in the first column of each plate.
<ol type="a">
<li>559T: 4.5 <b>×</b> 10<sup>3</sup> cells/well</li>
<li>592T: 4 <b>×</b> 10<sup>3</sup> cells/well</li>
<li>A549: 1.5 <b>×</b> 10<sup>3</sup> cells/well</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each cells proliferate at different growth rate. The optimal seeding number of each cells was determined based on absorbance value (0.5–1) by Cyto X treatment after 3 days of culture.</div>
<div class="note">
<span class="note-title">Note:</span> A549 cells were maintained in RPMI 1640 supplemented with 10% FBS and 1% antibiotics.</div>
<ol start="27">
<li>Incubate the plates at 37°C and 5% CO<sub>2</sub> for 24 h.</li>
<li>Use the plates on which cells are seeded as the assay plates.</li>
</ol>
<h4 id="sec3.2.3">Day 2: Compound dilution and treatment</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>The main process for compound dilution and treatment on the second day of the experiment is divided into three steps (<a href="#fig4">Figure 4</a>A): (Step A) compound aspiration, (Step B) two-step dilution, and (Step C) treatment. Before starting the dilution process, four 384-well plates were prepared: ① compound plate, ② control plate, ③ dilution plate-1, and ④ dilution plate-2. 384-well, PP, and v-bottom plates were used for the compound and control plates, and compounds were prepared at 10 μL/well. It is recommended to use v-bottom plates for compound dilutions because small amounts of chemicals can be accurately aspirated during the dilution step.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Compound dilution and treatment</div>
<p>(A) Step A: Double aspiration of compounds and control compounds. Step B: Two-step dilution. Step C: Treatment of assay plate. All procedures were carried out automatically.</p>
<p>(B) Final map of the assay plates. The first column of each assay plate was left blank, with wells containing vehicles (0.1% DMSO in cell-free culture media). The second column of the assay plates contained a lethal dose of doxorubicin and was used as a negative control. The 3<sup>rd</sup> and 4<sup>th</sup> columns were used for vehicle/positive controls (0.1% DMSO) and the reference compound (4 wells in bottom, approximately 50% viable by 40 nM doxorubicin), respectively. Therefore, only the remaining 360 wells of each plates were used for compound test.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Control plate contains DMSO, and doxorubicin (reference compound). Since DMSO and doxorubicin are treated at the same location on all assay plates, preparing a control plate separately from the compound plate makes repeat process easier. The control plate is used repeatedly until the end of the entire dilution process. See step A in <a href="#fig4">Figure 4</a>A for detail.</div>
<div class="note">
<span class="note-title">Note:</span> For dilution, fill the Plate ③ with 32.3 μL of culture medium and fill the Plate ④ with 81 μL of culture medium as the diluent. MultiFlow FX dispenser is used to fill the diluent. See step B in <a href="#fig4">Figure 4</a>A for detail.</div>
<ol start="29">
<li>Step A: Aspirate 1 μL from Plates ① and ②.</li>
<li>Step B: Two-step dilution
<ol type="a">
<li>Transfer the aspirated chemicals into Plate ③ and mix thoroughly by pipetting up and down several times at a 20 μL volume setting (33.3-fold dilution).</li>
<li>Transfer 9 μL from Plate ③ to Plate ④ and mix thoroughly by pipetting up and down several times at a 50 μL volume setting (333-fold dilution).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Beckman Coulter's Biomek NX 384 liquid handler and 30 μL 384 tips were used for aspiration and two-step dilution procedures. Replace the tips before the Step B-b (step 30) to improve the accuracy of the results.</div>
<ol start="31">
<li>Step C: Compound treatment.
<ol type="a">
<li>Before treatment, remove 40 μL of the media from 50 μL of the assay plate using an EL405 plate washer.
<div class="critical">
<span class="critical-title">Critical:</span> In the media removal process, it is important to retain 10 μL of media in the assay plate to prevent cell loss or damage.</div>
</li>
<li>Re-add 40 μL of fresh media to the assay plate using a MultiFlo dispenser.</li>
<li>Aspirate diluted chemicals from Plate ④ and sequentially dispense 25 μL to the assay plates seeded with three different cell types. Beckman Coulter's Biomek NX 384 liquid handler and 30 μL 384 tips were used.</li>
<li>Incubate at 37°C and 5% CO<sub>2</sub> for 72 h.</li>
</ol>
</li>
</ol>
<p>After all dilution and treatment processes, each assay plate has a map and concentration, as shown in <a href="#fig4">Figure 4</a>B. In addition, the reference compound, doxorubicin, is added to four wells at a final concentration of 40 nM in all assay plates and used to observe the accuracy of the experiment. It is recommended to treat all assay plates with a reference compound that can determine the accuracy of the experiment.</p>
<h4 id="sec3.2.4">Description of fully automated HTS system operation for day 2 of the experiment</h4>
<p>The entire process was divided each day, and the experiment was conducted automatically. A more detailed process of the second-day course is as follows:</p>
<p><b>Configuration of HTS system components</b></p>
<p>The fully automated HTS system is customized with a structure in which various kinds of devices are arranged around the Scara Robot, and two plate shuttles connect the Scara Robot and liquid handlers. The components and arrangements of the HTS system are shown in <a href="#fig5">Figure 5</a>.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Configuration of HTS system devices</div>
<p>The fully automated HTS system is customized with a structure in which various types of devices are arranged around the Scara Robot, and two plate shuttles connect the Scara Robot and liquid handlers.</p>
</figcaption>
</figure>
<p><b>Configuration of the system software</b></p>
<p>The following software was used to operate each of the devices.</p>
<p>Biomek FX &amp; NX 384 Liquid handler - Biomek software Version: 4.1.(Build: 31.0), Beckman Coulter®</p>
<p>EL405 plate washer &amp; MultiFlo FX drop dispenser – LHC software Version: 2.18 (build 1), BioTek®</p>
<p>Full system control &amp; schedule – SAMI Workstation EX Version 4.1 (Build 11), Beckman Coulter®</p>
<p>In addition to the devices listed above, Scara Robot, Cytomat 2C, Delidder, Hotel Carosel, Plate shaker, Barcode reader, etc., are controlled by the SAMI EX 4.1 software.</p>
<p><b>Fully automated assay protocol</b></p>
<p>The day 2 protocol was edited using Beckman coulter's SAMI EX version 4.1 schedule software for a fully automated assay process (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Fully automated assay protocol for experimentation on day 2</div>
<p>(A) Whole step schedule. One family described here is scheduled to run automatically without human intervention. One family represents the automatically diluted compounds and treatments in assay plates seeded with three types of cells. If 10 families are repeated, the process takes about 2 h and 40 min, indicating that a total of 30 assay plates can be tested at once.</p>
<p>(B) Protocols for the compound dilution and treatment process. The protocol was edited using Beckman coulter's SAMI EX version 4.1 schedule software for fully automated assay process.</p>
</figcaption>
</figure>
<h4 id="sec3.2.5">Day 5: End of incubation, read out, finish assay</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 7–8 h</div>
<p>After incubation on the last day of the assay, the following process was performed to detect the experimental results.</p>
<ol start="32">
<li>Add 5 μL/well of Cyto X using MultiFlo FX dispenser to all plates.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cyto X is a product that measures the amount of viable cells using WST-1. WST-1 is reduced by dehydrogenase activity in living cells to give an orange-colored formazan dye, which is soluble in culture media or water. The amount of the formazan dye is directly proportional to the number of living cells. In the case of MTT, it forms crystalline formazan that is not soluble in water. A complicated process of dissolving it using an organic solvent such as DMSO is required for colorimetric detection.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Cell Counting Kit-8 (WST-8 based, Dojindo Molecular Technologies, Cat. No. CK04-01) can be used instead of Cyto X.</div>
<ol start="33">
<li>Incubate at 37°C and 5% CO<sub>2</sub> for 6 h.</li>
<li>Centrifuge at 130 <b>×</b> <i>g</i> for 1 min.</li>
<li>Measure the absorbance at 450 nm using a PerkinElmer Envision plate reader</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Because Cyto X is light sensitive, it is recommended to block it from light to prevent data fluctuations between plates.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Before measuring optical density (OD), be sure to centrifuge to remove bubbles.</div>
<h4 id="sec3.2.6">Cytotoxicity data analysis</h4>
<p>Using Cyto X data (absorbance value at 450 nm), cytotoxicity (%) was calculated as follows:</p>
<p class="math-ml"><math> <mrow> <mtext>Cytotoxicity</mtext> <mrow> <mo>(</mo> <mtext>%</mtext> <mo>)</mo> </mrow> <mo>=</mo> <mfrac> <mrow> <mrow> <mo>(</mo> <mrow> <msub> <mtext>Abs</mtext> <mtext>control</mtext> </msub> <mo>−</mo> <msub> <mtext>Abs</mtext> <mtext>blank</mtext> </msub> </mrow> <mo>)</mo> </mrow> <mo>−</mo> <mrow> <mo>(</mo> <mrow> <msub> <mtext>Abs</mtext> <mtext>sample</mtext> </msub> <mo>−</mo> <msub> <mtext>Abs</mtext> <mtext>blank</mtext> </msub> </mrow> <mo>)</mo> </mrow> </mrow> <mrow> <msub> <mtext>Abs</mtext> <mtext>control</mtext> </msub> <mo>−</mo> <msub> <mtext>Abs</mtext> <mtext>blank</mtext> </msub> </mrow> </mfrac> <mtext>×</mtext> <mn>100</mn> </mrow> </math></p>
<p>For dose-dependent assay, compounds were serially diluted on a semi-logarithmic scale. Experiment was performed independently at least three times. In order to calculate IC<sub>50</sub> values, dose-response curve fit was analyzed using Prism software package (GraphPad Software, San Diego, CA, USA). The input data (concentration vs. triplicates of percentage cytotoxicity) were analyzed by nonlinear regression options with log [inhibitor] vs normalized response – variable slope (four parameters). Differences with P values ≤ 0.05 were considered significant.</p>
<h4 id="sec3.2.7">Hit identification</h4>
<p>Through three independent HTS and dose-dependent assays, azathioprine was selected as a hit compound to induce GBM cell death (<a href="#bib4">Nam et al., 2021</a>). A549 lung cancer cells were used as a comparative group to identify drugs that have a specific effect on GBM. As an effective drug against GBM was found through phenotypic screening, various experiments were conducted to identify the mechanism of action of the selected drug. In the next section of this protocol, we describe how to detect and analyze proteomes that are globally altered by azathioprine.</p>
<h3 id="sec3.3">Sample preparation for proteomic analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days for step<b>s</b> 36–43</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days for step<b>s</b> 44–69</div>
<p>In this section, we describe the sample preparation method for quantitative bottom-up proteomic analysis of GBM cells, which is generally applicable to other cell types. This method consists of the following steps: (1) protein extraction, (2) tryptic digestion of proteins, (3) cleanup of peptides, and (4) iTRAQ labeling of peptides.</p>
<ol start="36">
<li>Coat 100 mm TC-treated dishes with laminin-fibronectin for at least 2 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As the coating process is carried out manually, there is no need to prepare coated dishes the day before cell seeding.</div>
<ol start="37">
<li>Seed GBM cells at 2×10<sup>6</sup> cells/dish, and place in a CO<sub>2</sub> incubator for 24 h to ensure stable attachment and cell stabilization.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Multiple cell dishes may be needed to obtain sufficient protein for proteomic analysis. We prepared 2–3 dishes for each experimental group.</div>
<ol start="38">
<li>Treat with appropriately concentrated drug or vehicle control. For azathioprine, we treated cells with 10 μM azathioprine for 48 h in 37°C CO<sub>2</sub> incubator.</li>
<li>Wash the vehicle- or drug-treated cells with cold PBS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Carefully add 3–6 mL of PBS to the cell culture dish to avoid detaching the cells and aspirate the PBS.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ice-cold PBS should be used. Warm PBS may cause protein degradation. In addition, sterile-filtered PBS should be used for subsequent mass analysis.</div>
<ol start="40">
<li>Add 1 mL of cold PBS, manually dissociate cells from dish by scraping, while keeping dish on ice.</li>
<li>Transfer to a 1.5 mL tube</li>
<li>Centrifuge at 3000 <b>×</b> <i>g</i> for 3 min at 4°C.</li>
<li>Aspirate the supernatant and save the cell pellet.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be stored at −80°C.</div>
<ol start="44">
<li>Resuspend the cell pellet in 300 μL of cell lysis buffer (see <a href="#materials-and-equipment">materials and equipment</a> section).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cell lysis methods based on common detergents (e.g., sodium dodecyl sulfate, Triton X-100, or Nonidet P-40) should be avoided, because these detergents interfere with subsequent MS analysis.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If the use of detergents is required, an alternative sample preparation method to remove detergents should be performed (e.g., filter-aided sample preparation) (<a href="#bib5">Wiśniewski et al., 2009</a>). Otherwise, some MS-compatible detergents such as acid-labile detergents (e.g., RapiGest SF or PPS Silent Surfactant) and a subset of non-ionic detergents (e.g., n-Dodecyl β-D-maltoside) can be applied.</div>
<ol start="45">
<li>Sonicate the cell suspension at an amplitude of 25% on ice (Program: Time- 2 min, Pulser- 20%).</li>
<li>Centrifuge the lysate at 14000 <b>×</b> <i>g</i> for 15 min at 4°C.</li>
<li>Transfer the supernatants into a 1.5 mL tube.</li>
<li>Measure the protein concentration of the supernatants using the BCA Protein Assay Kit.</li>
<li>Transfer 100 μg of proteins into a 1.5 mL tube.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be stored at −80°C.</div>
<ol start="50">
<li>Add 10 μL of 100 mM DTT in 500 mM triethylammonium bicarbonate to the protein sample to reduce disulfide bonds between the thiol groups of cysteine residues in proteins and adjust the total reaction volume to 100 μL by adding HPLC-grade water. Incubate the mixture at 37°C and 850 rpm for 2 h in a thermomixer.</li>
<li>Add 8 μL of 270 mM IAA in 50 mM triethylammonium bicarbonate to alkylate the free thiol groups of cysteine residues. Incubate the mixture at 25°C and 850 rpm for 30 min in a thermomixer under dark conditions.</li>
<li>Quench the remaining IAA by adding 12 μL of 400 mM L-cysteine in 50 mM triethylammonium bicarbonate to the protein mixture. Incubate the mixture at 25°C and 850 rpm for 30 min in a thermomixer.</li>
<li>Dilute the protein samples with 50 mM triethylammonium bicarbonate to reduce the final urea concentration to less than 2 M.</li>
<li>Add 4 μL of 0.25 μg/μL trypsin and incubate at 37°C, 850 rpm for 18 h in a thermomixer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Trypsin is the most commonly used protease in bottom-up proteomics that cleaves peptide bonds at the C-terminal of lysine or arginine residues, except for lysine-proline and arginine-proline bonds. Lys-C, which cleaves lysine-proline bonds, is often used with trypsin to reduce missed cleavages.</div>
<ol start="55">
<li>Add formic acid to stop tryptic digestion to a final concentration of 1% (v/v).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be stored at −80°C.</div>
<ol start="56">
<li>For the cleanup of peptides, condition the Oasis HLB cartridge with 1 mL of 100% acetonitrile to wet the Oasis HLB sorbents.</li>
<li>Equilibrate with 2 mL of 0.1% formic acid in HPLC-grade water.</li>
<li>Load the peptide mixtures onto the cartridge.</li>
<li>Wash with 2 mL of 0.1% formic acid in HPLC-grade water.</li>
<li>Elute with 0.5 mL of 70% acetonitrile/30% HPLC-grade water</li>
<li>Dry eluted peptide mixtures using a SpeedVac concentrator.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be stored at −80°C.</div>
<ol start="62">
<li>For iTRAQ labeling, dissolve each dried peptide in 30 μL of 500 mM triethylammonium bicarbonate in an ultrasonic bath for 5 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> iTRAQ reagents react with amine groups of the N-terminus and lysine on peptides. iTRAQ quantification is based on specific reporter ions produced upon MS/MS fragmentation, which allows multiplex analysis of up to 4plex or 8plex.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> TMT kits (6, 10, 11, or 16plex) are also available for isobaric labeling.</div>
<ol start="63">
<li>Equilibrate the iTRAQ reagents to room temperature (15°C–25°C) and briefly spin down iTRAQ reagent vials.</li>
<li>Add ethanol (70 μL) to each iTRAQ reagent vial. Vortex the iTRAQ reagent vials and briefly spin down.</li>
<li>Transfer entire contents of iTRAQ reagents to each peptide solution. Vortex and briefly spin down. Incubate the mixture at 25°C for 2 h in a thermomixer.</li>
<li>Acidify by adding 1 μL of formic acid to each sample tube to stop the labeling reaction.</li>
<li>Combine the contents of all iTRAQ-labeled peptides into a new 1.5 mL tube. Vortex and briefly spin down.</li>
<li>Prepare dry pooled peptide mixtures using a SpeedVac concentrator to remove the ethanol. If necessary, HPLC-grade water can be added to reduce the ethanol concentration to ∼ 5%.</li>
<li>Clean-up pooled peptide mixtures with Oasis HLB cartridge and dry completely using a SpeedVac concentrator (repeat steps 56–61).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be stored at −80°C.</div>
<h3 id="sec3.4">Preparation of C18/SCX biphasic trap column</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–6 h</div>
<p>This protocol describes an in-house packing method for a biphasic trap column (I.D. 200 μm × 4 cm long) packed with reverse phase (RP) and strong cation exchange (SCX) resin (<a href="#fig7">Figure 7</a>A).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Fabrication of C18/SCX biphasic trap column</div>
<p>(A and B) Schematic representation of (A) C18/SCX biphasic trap column and (B) capillary column packing system using pressure bomb.</p>
<p>(C) An image showing the setup of pressure bomb with nitrogen gas and magnetic stir.</p>
</figcaption>
</figure>
<ol start="70">
<li>Freshly prepare the frit solution by combining 80 μL potassium silicate solution with 20 μL formamide in a 1.5 mL tube.</li>
<li>Vortex and centrifuge at 400 <b>×</b> <i>g</i> for 5 min.</li>
<li>Cut silica capillary tubing (I.D. 200 μm and O.D. 350 μm) to approximately 20 cm.</li>
<li>Dip the end of the capillary tubing into the frit solution for a few seconds until the solution fills 3–5 cm in the capillary tubing.</li>
<li>Incubate the capillary tubing at 90°C for 3 h to polymerize silicate.</li>
<li>Cut the frit part of capillary tubing to approximately 0.2 cm long.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The fritted capillary can be stored at room temperature (15°C–25°C) before use.</div>
<ol start="76">
<li>Prepare a C18 slurry by adding 3–5 mg of C18 resin (5 μm–200 Å) with 1 mL of 100% methanol in a 1.5 mL tube. Add a stir bar to the tube, and vortex the mixture.</li>
<li>Cut the cap off the tube and place the C18 slurry in a pressure bomb on a magnetic stirrer (<a href="#fig7">Figures 7</a>B and 7C).</li>
<li>Dip the open end of the fritted capillary into a C18 slurry.</li>
<li>Fill the fritted capillary with C18 resin in N<sub>2</sub> gas (600 psi) to 1 cm.</li>
<li>Remove the C18 slurry from the pressure bomb, and dry the C18-packed capillary with N<sub>2</sub> gas to avoid mixing with the SCX resin.</li>
<li>Prepare an SCX slurry by adding 3–5 mg SCX resin (5 μm–200 Å) with 1 mL of 100% methanol in a 1.5 mL tube. Add a stir bar to the tube, and vortex the mixture.</li>
<li>Cut off the cap and place the SCX slurry in the pressure bomb on the magnetic stirrer.</li>
<li>Fill the C18-packed capillary with SCX resin at N<sub>2</sub> gas (600 psi) to another 1.5 cm.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The pressure should be released slowly from the pressure bomb to avoid backflow of resin in the column and consequently mix C18 and SCX resin.</div>
<ol start="84">
<li>Remove the SCX slurry from the pressure bomb and dry the C18/SCX-packed capillary with N<sub>2</sub> gas.</li>
<li>Cut the C18/SCX-packed capillary column to the desired length (approximately 4 cm) and equilibrate with the appropriate buffer (e.g., mobile phase A).</li>
</ol>
<h3 id="sec3.5">Preparation of reverse-phase nanoflow liquid chromatography column</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–6 h</div>
<p>This protocol describes an in-house packing method for an RP nanoLC column (I.D. 75 μm <b>×</b> 15 cm long) packed with C18 resin (3 μm–100 Å), which can also be applied to prepare other types of nanoLC columns (e.g., different lengths of column and sizes of packing material). NanoLC columns are commercially available.</p>
<ol start="86">
<li>Cut silica capillary tubing (I.D. 75 μm and O.D. 350 μm) to approximately 40 cm.</li>
<li>Attach the capillary with tape to the glass plate that is assembled to the stand with a clamp (<a href="#fig8">Figure 8</a>).
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Schematic representation of the tapered tip emitter fabrication</div>
</figcaption>
</figure>
</li>
<li>Heat a portion of the capillary with a torch until it melts and pulls down the melted capillary to form a tapered tip emitter.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A laser-based micropipette puller (e.g., SUTTER INSTRUMENT CO) can be used to pull the tip.</div>
<ol start="89">
<li>Carefully cut the end of the tapered tip into a blunt shape with a diamond cutter, which is visible under a stereoscopic microscope.</li>
<li>Prepare a slurry by adding 2–3 mg C18 resin (3 μm–100 Å) with 1 mL of 100% methanol in a 1.5 mL tube. Add a stir bar to the tube, and vortex the mixture.</li>
<li>Cut the cap off the tube and place the C18 slurry in a pressure bomb on a magnetic stirrer (<a href="#fig7">Figures 7</a>B and 7C).</li>
<li>Fill the tapered tip capillary with C18 resin at N<sub>2</sub> gas (800 psi) to approximately 18 cm.</li>
<li>Equilibrate with the appropriate buffer (e.g., mobile phase A) using an LC system and cut the capillary column to a length of 15 cm.</li>
</ol>
<h3 id="sec3.6">Online two-dimensional nanoflow liquid chromatography-tandem mass spectrometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 days (total analysis time of about 23 h per replicate)</div>
<p>Multidimensional LC coupled with tandem MS/MS is a powerful tool for large-scale proteomic analysis. Two-dimensional (2D) LC combined with SCX and RP is the most commonly used approach. In this protocol, a 2D LC system was employed automatically (online) using a C18/SCX biphasic trap column.</p>
<ol start="94">
<li>Resuspend the sample in 50 μL of 0.1% formic acid in HPLC-grade water in an ultrasonic bath for 5 min.</li>
<li>Inject the sample (10 μL) from the autosampler onto the C18/SCX trap column (valve position A in <a href="#fig9">Figure 9</a>) at a flow rate of 4 μL/min for 10 min. The peptides bind to the SCX part of the trap column.
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Instrumental setup of online 2D nanoLC system</div>
</figcaption>
</figure>
</li>
<li>Switch the 6-prot valve to position B (<a href="#fig9">Figure 9</a>) and perform a breakthrough run on the RP analytical column at a column flow rate of 200 nL/min for 50 min. The LC gradient should be as follows: 0–1 min (2%–8% mobile phase B), 1–10 min (8%–15% mobile phase B), 10–20 min (15%–30% mobile phase B), 20–23 min (30%–90% mobile phase B), 23–30 min (90% mobile phase B), 30–32 min (90%–2% mobile phase B), and 32–60 min (2% mobile phase B).</li>
<li>Inject 10 μL of SCX elution buffer from the autosampler onto the C18/SCX trap column (valve position A in <a href="#fig9">Figure 9</a>) at a flow rate of 4 μL/min for 10 min. The peptides were eluted from the SCX and re-bound to the C18 part on the trap column.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> SCX elution buffers were prepared as described in the <a href="#materials-and-equipment">materials and equipment</a> section. In the initial experimental setup of 2D-LC-MS/MS analysis, we checked the distribution of unique peptides identified in each fraction. If necessary, optimize the elution conditions of SCX (e.g., salt concentration or pH).</div>
<ol start="98">
<li>Switch the 6-prot valve to position B (<a href="#fig9">Figure 9</a>), and load the peptides onto the RP analytical column. Perform the LC-MS/MS analysis with the parameters described in <a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a> from the equipment setup section.</li>
<li>Repeat 97–98 steps for each step of SCX elution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Two or three biological replicates were prepared for the proteomic analysis. Each biological replicate was analyzed with two technical replicates in 2D-LC-MS/MS experiments.</div>
<h3 id="sec3.7">Proteomic data analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> days to weeks</div>
<ol start="100">
<li>Import the acquired MS/MS raw files (.raw) into MaxQuant software (version 1.6.6.0).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> MaxQuant software can be downloaded at <a href="http://www.maxquant.org">http://www.maxquant.org</a>, which is freely available.</div>
<ol start="101">
<li>Search raw data using the Andromeda search engine integrated into the MaxQuant against the UniProt <i>Homo sapiens</i> proteome database (release Apr 21, 2019). The search parameters are as follows:
<ol type="a">
<li>Enzyme: Trypsin</li>
<li>Missed cleavages: 2</li>
<li>FTMS MS/MS match tolerance: 20 ppm</li>
<li>FTMS MS/MS de novo tolerance: 10 ppm</li>
<li>Fixed modifications: carbamidomethylation (C) and iTRAQ 4plex (N-term and K).</li>
<li>Variable modifications: oxidation (M) and acetylation (N-term)</li>
<li>False discovery rate (FDR) of peptide and protein: 0.01</li>
<li>Minimum number of unique peptides: 2</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> UniProt <i>Homo sapiens</i> proteome database (UP000005640) is available for download at <a href="http://www.uniprot.org/proteomes/">http://www.uniprot.org/proteomes/</a>.</div>
<ol start="102">
<li>Obtain the intensity values for each iTRAQ reporter ion from the MaxQuant result file (named ‘ProteinGroup.txt’). Calculate the iTRAQ ratio (115/114) of drug treatment (115 tag) versus control (114 tag).</li>
<li>Import the resulting file (.txt) into Perseus software (version 1.5.8.5) for data processing and statistical analysis.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Perseus software can be freely download at <a href="http://www.maxquant.org/perseus/">http://www.maxquant.org/perseus/</a>.</div>
<ol start="104">
<li>Filter out the potential contaminants, reverse hits, and proteins identified only by site from the data.</li>
<li>Transform the iTRAQ ratio to log<sub>2</sub> scale, and normalize by subtracting the median.</li>
<li>Filter proteins with valid values for statistical analyses. The minimum number of values is three out of the total replicates.</li>
<li>Determine the statistical significance of differentially expressed proteins (DEPs) by performing Student’s t-test, and create a volcano plot based on -log<sub>10</sub> (t-test p-value) against log<sub>2</sub> (fold change).</li>
<li>Analyze the DEPs using various bioinformatics databases and resources.
<ol type="a">
<li>Gene Ontology (GO) enrichment: PANTHER (<a href="http://pantherdb.org">http://pantherdb.org</a>) and DAVID (<a href="https://david.ncifcrf.gov">https://david.ncifcrf.gov</a>).</li>
<li>Protein–Protein Interaction (PPI) network: STRING database (<a href="https://string-db.org">https://string-db.org</a>)</li>
<li>Pathway enrichment: KEGG (<a href="https://www.genome.jp/kegg">https://www.genome.jp/kegg</a>) and Reactome (<a href="https://reactome.org">https://reactome.org</a>).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> These databases and software are freely accessible on the websites. Alternatively, bioinformatics analysis (e.g., GO enrichment, KEGG, and Reactome pathway) and data visualization can be performed using R (<a href="http://www.r-project.org">www.r-project.org</a>) and Bioconductor package (<a href="https://www.bioconductor.org">https://www.bioconductor.org</a> ).</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">HTS</h3>
<p>In a previous study (<a href="#bib4">Nam et al., 2021</a>), azathioprine was found to be a promising therapeutic agent for GBM treatment through HTS using clinically relevant drug libraries. A total of 1,086 drugs and three types of cells were used for HTS. Detailed HTS results are available in the Supplemental Information from a previous publication (<a href="#bib4">Nam et al., 2021</a>).</p>
<h3 id="sec4.2">Proteomics</h3>
<p>In a previous study (<a href="#bib4">Nam et al., 2021</a>), this procedure was applied to profile the azathioprine-induced proteome in GBM cells. A total of ∼40,000 unique peptides corresponding to ∼6000 proteins were identified in a single 2D LC-run. When the filter was applied according to the method described in the <a href="#sec3.7">proteomic data analysis</a> section, ∼5000 proteins were quantified. An example of a volcano plot representing DEPs between azathioprine treatment and control is shown in <a href="#fig10">Figure 10</a>.</p>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1038-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Expected outcome of proteomic analysis</div>
<p>Figure reprinted with permission from <a href="#bib4">Nam et al. (2021)</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>With this protocol, the global drug-induced changes in protein expression can be analyzed, thereby providing insight into the possible targets and mechanisms of drug action. However, in the case of certain drugs (e.g., kinase inhibitors), a global survey of changes in protein abundance is often insufficient to understand the underlying mechanisms of the drug, owing to a lack of information regarding post-translational modifications (PTMs) on proteins (e.g., phosphorylation and methylation). As PTMs on proteins play important roles in cellular signaling processes, proteomic approaches for mapping PTMs can expand the knowledge of drug action.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Poor iTRAQ labeling (step 62–69)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Check whether the primary amine-containing buffer (e.g., Tris, ammonium bicarbonate) was used in the sample preparation process. Substances containing primary amines might interfere with the iTRAQ labeling reaction.</p>
<p>Check the pH of the sample. The pH of the samples should be higher than 7.5.</p>
<p>Organic solvent concentration should be maintained at least at 60%.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Low number of identified proteins in 2D-LC-MS/MS analysis (step 100–108)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Check the number of identified proteins in a breakthrough LC run after sample injection (step 96). If a high number of proteins were identified at this stage, the pH of the sample was lower than 3.0, and the salts were completely removed from the sample. Ensure that the C18 and SCX parts are completely divided in the biphasic C18/SCX trap column.</p>
<p>Check the distribution of unique peptides identified in each SCX elution fraction (step 98).</p>
<p>Check the LC-MS/MS instrument setup using quality control samples.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>High fluctuation in percentage cytotoxicity (step 32–35)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Check the single cell dissociation in cell preparation for seeding. If cell aggregates are observed, incubate accumax-treated GBM at 37°C instead of room temperature (15°C–25°C) for 3 min (step 10–11). Then, gently pipette the cells up and down at least 20 times (step 12). Single cell dissociation is critical for uniform cell distribution during cell seeding.</p>
<p>During cell dispensing step, occasionally swirl the cell suspension bottle of the Multiflo FX drop-dispenser to ensure even distribution of cells in the medium (step 26). Over time, cells may settle down due to gravity.</p>
<p>If particular rows of plate have lower absorbance value compared to other rows, check the cassette nozzle on the Multiflo FX drop-dispenser and replace with a new cassette. The cassette nozzle may be partially clogged.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Poor GBM cell attachment (step 22–28, 36–37)</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Ensure that the coating solution covers the entire bottom. Sufficient coating solution is required for cells to adhere evenly (<a href="#materials-and-equipment">materials and equipment</a> section). In addition, quick thawing of Laminin reduces its coating ability. Laminin should be thawed slowly at 4°C.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Discrepancy in absorbance values between positive controls (related to cytotoxicity data analysis)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Check the air bubbles in Cyto X-treated sample. Centrifugation should be performed to remove air bubbles before measuring OD (step 34–35). To maintain the functionality of Cyto X, always keep Cyto X in the dark. Cyto X-treated plates should be incubated in the dark (step 33). Cyto X is sensitive to light.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Joo, K.M., Kim, S.Y., Jin, X., Song, S.Y., Kong, D.-S., Lee, J.-I., Jeon, J.W., Kim, M.H., Kang, B.G., Jung, Y., et al. (2008). Clinical and biological implications of CD133-positive and CD133-negative cells in glioblastomas. Lab. Invest. <i>88</i>, 808-815. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00555-4/sref1">View at publisher</a></p>
<p id="bib6">Cox, J. and Mann, M. (2008). MaxQuant enables high peptide identification rates, individualized ppb-range mass accuracies and proteome-wide protein quantification. Nature biotechnology <i>26</i>, 1367-1372. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00555-4/sref6">View at publisher</a></p>
<p id="bib2">Joo, K.M., Kim, J., Jin, J., Kim, M., Seol, H.J., Muradov, J., Yang, H., Choi, Y.L., Park, W.Y., Kong, D.S., et al. (2013). Patient-specific orthotopic glioblastoma xenograft models recapitulate the histopathology and biology of human glioblastomas in situ. Cell Rep. <i>3</i>, 260-273. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2012.12.013">https://doi.org/10.1016/j.celrep.2012.12.013</a></p>
<p id="bib8">Kanehisa, M., Goto, S., Sato, Y., Furumichi, M., and Tanabe, M. (2012). KEGG for integration and interpretation of large-scale molecular data sets. Nucleic acids research <i>40</i>, D109-D114. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00555-4/sref8">View at publisher</a></p>
<p id="bib3">Louis, D.N., Ohgaki, H., Wiestler, O.D., Cavenee, W.K., Burger, P.C., Jouvet, A., Scheithauer, B.W., and Kleihues, P. (2007). The 2007 WHO classification of tumours of the central nervous system. Acta Neuropathol. <i>114</i>, 97-109. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00555-4/sref3">View at publisher</a></p>
<p id="bib4">Nam, H.J., Kim, Y.E., Moon, B., Kim, H.Y., Jung, D., Choi, S., Jang, J.W., Nam, Do., and Cho, H. (2021). Azathioprine antagonizes aberrantly elevated lipid metabolism and induces apoptosis in glioblastoma. iScience <i>24</i>, 102238. <a class="external-link" href="https://doi.org/10.1016/j.isci.2021.102238">https://doi.org/10.1016/j.isci.2021.102238</a></p>
<p id="bib7">Tyanova, S., Temu, T., Sinitcyn, P., Carlson, A., Hein, M.Y., Geiger, T., Mann, M., and Cox, J. (2016). The Perseus computational platform for comprehensive analysis of (prote)omics data. Nature methods <i>13</i>, 731-740. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00555-4/sref7">View at publisher</a></p>
<p id="bib5">Wiśniewski, J.R., Zougman, A., Nagaraj, N., and Mann, M. (2009). Universal sample preparation method for proteome analysis. Nat. Methods <i>6</i>, 359-362. <a class="external-link" href="https://doi.org/10.1038/nmeth.1322">https://doi.org/10.1038/nmeth.1322</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Heeyeong Cho (<a href="mailto:hycho@krict.re.kr">hycho@krict.re.kr</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The study did not generate new datasets.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by a grant from the <a href="https://doi.org/10.13039/501100003704">Korea Research Institute of Chemical Technology</a> (project number SI-2131-50) and 3D-TissueChip Based Drug Discovery Platform Technology Development Program (20009774, High-Throughput 3D Multifunctional Tissue-based Screening Service of Efficacy and Safety for Drug Discovery) funded by the <a href="https://doi.org/10.13039/501100003052">Ministry of Trade, Industry &amp; Energy</a> (MOTIE, Korea) and the Bio &amp; Medical Technology Development Program of the National Research Foundation (NRF) funded by the <a href="https://doi.org/10.13039/501100003621">Ministry of Science &amp; ICT</a> (2020M3A9I4036072).</p>
<h3>Author contributions</h3>
<p>H.C. and H.J.N. conceived and designed the study and analysis. Y.E.K., H.Y.K., D.J., and H.J.N. performed the experiments and collected the data. D.-H.N. provided patient-derived GBM cells. Y.E.K. and D.J. established the LC-MS/MS system. Y.E.K., H.Y.K., D.J., H.J.N., and H.C. wrote the manuscript. The authors read and approved the final manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>