<article>
<section>
<h1 id="header">Single-Cell Transcriptomics of Human Lymph Node Stroma</h1>
<p><time datetime="2020-03-13">Published: March 13, 2020</time></p>
<p>Akira Takeda<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Sirpa Jalkanen<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>MediCity Research Laboratory and Institute of Biomedicine, University of Turku, Turku, Finland</p>
<p id="fn1"><sup>2</sup>Technical Contact</p>
<p id="fn2"><sup>3</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:sirjal@utu.fi">sirjal@utu.fi</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100021">10.1016/j.xpro.2020.100021</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Accumulating evidence indicates that the immune system is regulated not only by immune cells but also by stromal cells in the tissue microenvironment. Characterization of non-hematopoietic cells has not been performed in depth, since markers of the subsets are limited. Recent advances of single-cell technology allow researchers to characterize comprehensively the heterogeneity of stromal cells in an unbiased manner. In this article, we provide step-by-step protocols for cell preparation for single-cell RNA sequencing to characterize the heterogeneity of stroma in human lymph nodes.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib4">Takeda et al. (2019)</a>.</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol to prepare human stromal and endothelial cells for single-cell RNA sequencing</li>
<li>Gentle cell dissociation of human lymph nodes is critical for successful isolation</li>
<li>Protocol can be used to characterize the heterogeneity of stroma in lymph nodes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/62-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol>
<li>Prepare 15 ml enzyme mix consisted of RPMI, 0.8 mg/ml Dispase, 0.2 mg/ml collagenase P and 0.1 mg/ml DNase for each tissue.</li>
<li>Prepare PBS supplemented with 2% FCS and 2 mM EDTA.</li>
<li>Prepare phenol-red free DMEM supplemented with 2% FCS and 10 mM HEPES.</li>
<li>Set the temperature of the centrifuge to 4°C,</li>
<li>Set the temperature of a water bath to 37°C.</li>
<li>Coat a 1.5 ml tube for one sample with FCS by adding 1 ml of FCS and incubating for more than 12 hrs to prevent adhesion of sorted cells to tubes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Proper isolation of human stromal cells is critical for generation of comprehensive single-cell data since insufficient tissue digestion lose certain cell subsets and excessive dissociation damage cells and affect cell viability. Currently available protocols regarding single-cell studies mainly address different single-cell RNA-seq (scRNA-seq) platforms, overall experimental design and data analysis and processing (<a href="#bib1">Kulkarni et al., 2019</a>; <a href="#bib2">Lafzi et al., 2018</a>; <a href="#bib3">See et al., 2018</a>). Here we mainly focus on cell isolation which is critical to comprehensively characterize stromal cell subsets.</div>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>AF488-anti-human CD45</td>
<td>Biolegend</td>
<td>Cat# 304019; RRID:AB_493033</td>
</tr>
<tr>
<td>PE-anti-human podoplanin</td>
<td>Biolegend</td>
<td>Cat# 337004; RRID:AB_1595457</td>
</tr>
<tr>
<td>APC-anti-human CD31</td>
<td>Biolegend</td>
<td>Cat# 303115; RRID:AB_1877152</td>
</tr>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>RPMI</td>
<td>Sigma</td>
<td>Cat# R5886</td>
</tr>
<tr>
<td>Phenol-red free DMEM</td>
<td>Sigma</td>
<td>Cat# D1145</td>
</tr>
<tr>
<td>FCS</td>
<td>Sigma</td>
<td>Cat# F7524</td>
</tr>
<tr>
<td>HEPES</td>
<td>Lonza</td>
<td>Cat# BE17-737E</td>
</tr>
<tr>
<td>Dispase</td>
<td>Gibco</td>
<td>Cat# 17105-041</td>
</tr>
<tr>
<td>Collagenase P</td>
<td>Roche</td>
<td>Cat# 11213857001</td>
</tr>
<tr>
<td>DNase</td>
<td>Roche</td>
<td>Cat# 10104159001</td>
</tr>
<tr>
<td>DMEM</td>
<td>Sigma</td>
<td>Cat# D6429</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>EasySep human CD45 depletion kit</td>
<td>Stem Cell Technology</td>
<td>Cat# 17898</td>
</tr>
<tr>
<td>LIVE/DEAD Fixable Near-IR Dead Cell Stain kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# L34975</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>100 μm cell strainer</td>
<td>BD</td>
<td>Cat# 352360</td>
</tr>
<tr>
<td>Water bath</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Scalpels</td>
<td>Swann-Morton</td>
<td>Cat# 0503</td>
</tr>
<tr>
<td>Wide bore tips</td>
<td>Sartorius</td>
<td>Cat# 791021</td>
</tr>
<tr>
<td>Cell sorter</td>
<td>Sony</td>
<td>Cat# SH800S</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<p>Enzyme Mix- Make 15 ml for each tissue.</p>
<table>
<thead>
<tr>
<th>RPMI</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.8 mg/ml Dispase</td>
</tr>
<tr>
<td>0.2 mg/ml collagenase-P</td>
</tr>
<tr>
<td>0.1 mg/ml DNase</td>
</tr>
</tbody>
</table>
<p>Can be stored at 4 °C for up to 1 day.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Preparation of Single-Cell Suspension from Human Lymph Nodes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>This step enzymatically dissociates cells from fresh human lymph nodes.</p>
<ol>
<li>Immediately after surgery, transfer human lymph nodes (LNs) into 35 mm dishes filled with 5 ml RPMI on ice and cut into small pieces (around 0.5 mm<sup>3</sup> cubes) using fine scalpels.</li>
<li>Add LN pieces into 5 ml of the enzyme mix in 15 ml tube and incubate at 37°C in a water bath for 20 min in total and gently invert the tube 2-3 times every 5 min.</li>
<li>After incubation, LN pieces are gently passed through a wide bore tip of a 1 ml pipette. Let fragments settle to the bottom for 30 sec, after which the enzyme mix was removed and added into the 15 ml tubes containing the cold PBS supplemented with 2% FCS and 2 mM EDTA (supernatant 1).</li>
<li>Add 5 ml of fresh enzyme mix into the tubes with LN fragments and incubate for 10 min at 37°C. Invert the tubes every 5 min.</li>
<li>After the incubation, mix the fragments strongly for 30 sec using a 1 ml pipette equipped with a wide bore tip. Let fragments settle down and remove the supernatant and add it to the new tube containing the cold PBS including 2% FCS and 2 mM EDTA (supernatant 2).</li>
<li>Add 5 ml of fresh enzyme mix to the tubes with LN fragments and incubate it at 37°C until the fragments were almost completely digested. Mix fragments vigorously every 5 min (usually 5 min x 2-4 times). Combine the dissociated single cell suspension with the supernatant 1 and 2, and centrifuge at 300 g, 4 °C and filter the cells with 100 μm cell strainer and centrifuge it.</li>
<li>Add phenol red-free DMEM supplemented with 2% FCS and 10 mM HEPES. Count the cell number. Typical viability at this point is 95%.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use phenol red-free DMEM containing 2% FCS and 10 mM HEPES instead of PBS containing FCS until cell sorting to keep the cell viability high.</div>
<h3>Deplete CD45<sup>+</sup> Cells and Sort the Target Stromal Cells with a Cell-Sorter</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>This step enriches stromal cells via a depletion of hematopoietic cells.</p>
<ol start="8">
<li>Deplete CD45<sup>+</sup> cells with CD45<sup>-</sup> negative cell isolation kit by following manufacture’s protocols. Use phenol red-free DMEM during isolation and incubate the cells at 4<sup>°</sup>C.</li>
<li>Incubate the CD45<sup>-</sup> cells with fluorescence-conjugated antibodies including AF488-CD45, PE-podoplanin, APC-CD31 and LIVE/DEAD-Near IR for 30 min at 4<sup>o</sup>C.</li>
<li>Sort the cell-type of interest using a cell-sorter (e.g. SH800 from Sony) equipped with 100 μm chip. Sort PDPN<sup>+</sup>CD31<sup>+</sup>, PDPN<sup>-</sup>CD31<sup>+</sup> or PDPN<sup>+</sup>CD31<sup>-</sup> cells for isolation of lymphatic endothelial cells (LECs), blood endothelial cells (BECs) or non-endothelial stromal cells (SCs), respectively. Sort more than 20000 cells if possible into pre-coated 1.5 ml tubes containing 500 μl of DMEM supplemented with 10% FCS. Gate single cells with the FSC-H versus FCS-W plot and the SSC-H versus SSC-W plot to avoid doublets, and exclude dead cells based on LIVE-DEAD dead cell stain (<a href="#fig1">Figure 1</a>).</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/62-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. LEC Sorting from Human LNs for Single-Cell Analysis</div>
<p>Gating strategy for isolation of LECs from human LNs. CD45<sup>+</sup> cells and dead cells are excluded based on CD45 and Live/Dead dead cell stain, respectively. Live single CD45<sup>-</sup>PDPN<sup>+</sup>CD31<sup>+</sup> are sorted for the single-cell analysis of human LECs. Adopt from Figure S1A of <a href="#bib4">Takeda et al., 2019</a>.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> LIVE-DEAD dead cell stain-negative cells should be more than around 80 % of all the cells. Cell viability should be recorded pre and post-sorting (when counting cell number for library preparation) for proper metadata collection (<a href="#fig2">Figure 2</a>A). If the cell viability is low, final cell number will be low after quality check of scRNA-seq data and minor cell populations will not be detected.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/62-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Viability of Dissociated Human LN Stromal Cell Subsets after Cell Dissociation and Cell Sorting</div>
<p>(A and B) Viability of CD45<sup>+</sup> cells, SCs, BECs and LECs dissociated from human LNs.</p>
<p>(A) FACS plots. The images are representative of six biological replicates.</p>
<p>(B) Comparison of viability of dissociated stromal cells and post-sort LECs. Viability of cells after dissociation were counted by FACS and viability of post-sort cells were counted with Trypan blue. The circles in the Violin plot indicate the number of biological replicates (n=6). ∗p &lt; 0.05, ∗∗∗∗p &lt; 0.0001.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Viability differs between different subsets (<a href="#fig2">Figures 2</a>A and 2B). Stromal cells especially LECs (viability ~80%) are more prone to cell death than CD45<sup>+</sup> leukocytes (viability ~95%). Even though live LECs are gated and sorted for scRNA-seq of LECs, viability of post-sort LECs is comparable to that of LECs after cell dissociation (<a href="#fig2">Figure 2</a>B). This indicates that FACS sorting damages stromal cells.</div>
<h3>Library Preparation for Single-Cell RNA-Sequencing</h3>
<p>Count cell number and prepare single-cell RNA-sequencing libraries.</p>
<ol start="11">
<li>Count sorted human stromal cells manually at least two times. Check the cell-viability with trypan blue and recode it as metadata.</li>
<li>Process the samples according to 10X Genomics User Guide.</li>
<li>Prepare single-cell RNA-sequencing libraries according to the manufacturer’s instruction from 10X Genomics (v2 or v3 chemistry).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>Viability differs between different subsets in human LNs (<a href="#fig2">Figures 2</a>A and 2B). Stromal cells especially LECs are more prone to cell death than CD45<sup>+</sup> leukocytes. Among LEC subtypes, there might be some more sensitive LEC subsets than others. Thus, frequency of subpopulations in scRNA-seq may not be equal to that in <i>in vivo</i> (<a href="#bib4">Takeda et al., 2019 </a>). Even though live LECs are gated and sorted to enrich live LECs, viability of post-sort LECs is comparable to that of LECs after dissociation (<a href="#fig2">Figure 2</a>B). This may imply that FACS sorting damages subtypes of stromal cells. Thus, recording of viability as metadata is important and it should be taken into account when analyzing single-cell data.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Using 10X Genomics Chromium kit, we can analyze up to 10,000 cells with one reagent. If you want to comprehensively characterize non-hematopoietic cells in human LNs, you need to deplete hematopoietic cells since majority of cells (more than 90%) in LNs are lymphocytes. Subpopulations of stromal cells may be more prone to dissociation-associated cell death than other subsets. Thus, it is important to keep in mind that you might not detect rare and sensitive cell types depending on viability.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem</h3>
<p>Single-cell RNA-seq is a powerful tool to investigate the cellular heterogeneity, developmental pathway and molecular interactions between cell subsets (<a href="#bib4">Takeda et al., 2019</a>; <a href="#bib6">Velten et al., 2017</a>; <a href="#bib7">Vento-Tormo et al., 2018</a>). However, interpretation of unbiased computational analysis is sometimes tricky. Cell stress caused by the enzymatic dissociation may induce transcriptome changes at certain populations and produce cell clusters that are unlikely to exist in the physiological conditions (<a href="#bib5">van den Brink et al., 2017</a>).</p>
<h3>Potential Solutions</h3>
<p>It has been shown that dissociation of the tissue induces transcriptional changes in certain genes such as genes coding heat-shock proteins (<a href="#bib5">van den Brink et al., 2017</a>). Cell clusters highly expressing those genes can be removed from further analysis. Moreover, clusters that are not observed constantly between different samples may be removed. In this context, biological replicates (LNs from multiple patients) are crucial. To confirm the presence of heterogeneity detected by single-cell RNA-seq, other supporting information such as visualization of the subsets via immunostaining or in-situ hybridization is needed.</p>
<h3>Problem</h3>
<p>The use of specific stromal cell markers may lead to loss of cell heterogeneity. It has been widely believed that LECs can be distinguished from BECs by expression of multiple molecules including LYVE1 and PROX1. However, the scRNA-seq of LECs sorted with anti-LYVE1 will show limited heterogeneity since LYVE1 is not expressed on all LEC subsets (<a href="#bib4">Takeda et al., 2019</a>).</p>
<h3>Potential Solutions</h3>
<p>To avoid loss of heterogeneity by using selective markers, the use of general cell markers is recommended. For instance, PDPN and PROX1 are more general lymphatic markers than LYVE1. Our scRNA-seq of human LEC analysis also showed the difference of PDPN expression in six human LEC subsets. Thus, to comprehensively analyze the LEC population, PDPN low or intermediate cells should also be included for the analysis. Contaminated cells such as BECs and SCs can be removed in scRNA-seq analysis by use of BEC and SC markers (e.g. JAM2 and CCL19, respectively).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Kulkarni, A., Anderson, A.G., Merullo, D.P., and Konopka, G. (2019). Beyond bulk: a review of single cell transcriptomics methodologies and applications. Curr. Opin. Biotechnol. <i>58</i>, 129-136.</p>
<p id="bib2">Lafzi, A., Moutinho, C., Picelli, S., and Heyn, H. (2018). Tutorial: guidelines for the experimental design of single-cell RNA sequencing studies. Nat. Protoc. <i>13</i>, 2742-2757.</p>
<p id="bib3">See, P., Lum, J., Chen, J., and Ginhoux, F. (2018). A single-cell sequencing guide for immunologists. Front. Immunol. <i>9</i>, 2425.</p>
<p id="bib4">Takeda, A., Hollmen, M., Dermadi, D., Pan, J., Brulois, K.F., Kaukonen, R., Lonnberg, T., Bostrom, P., Koskivuo, I., Irjala, H., et al. (2019). Single-cell survey of human lymphatics unveils marked endothelial cell heterogeneity and mechanisms of homing for neutrophils. Immunity <i>51</i>, 561-572.e5.</p>
<p id="bib5">van den Brink, S.C., Sage, F., Vertesy, A., Spanjaard, B., Peterson-Maduro, J., Baron, C.S., Robin, C., and van Oudenaarden, A. (2017). Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations. Nat. Methods <i>14</i>, 935-936.</p>
<p id="bib6">Velten, L., Haas, S.F., Raffel, S., Blaszkiewicz, S., Islam, S., Hennig, B.P., Hirche, C., Lutz, C., Buss, E.C., Nowak, D., et al. (2017). Human haematopoietic stem cell lineage commitment is a continuous process. Nat. Cell Biol. <i>19</i>, 271-281.</p>
<p id="bib7">Vento-Tormo, R., Efremova, M., Botting, R.A., Turco, M.Y., Vento-Tormo, M., Meyer, K.B., Park, J.E., Stephenson, E., Polanski, K., Goncalves, A., et al. (2018). Single-cell reconstruction of the early maternal-fetal interface in humans. Nature <i>563</i>, 347-353.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>This study was supported by the <a href="https://doi.org/10.13039/501100002341">Academy of Finland</a> (A.T and S.J.). Some graphical illustrations were created with <a href="http://Biorender.com">Biorender.com</a>.</p>
<h3>Author Contributions</h3>
<p>A.T. and S.J. conceived the idea and wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>