<article>
<section>
<h1 id="header">Visualizing murine breast and melanoma tumor microenvironment using intravital multiphoton microscopy</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Rohit Jain,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn3">5</a>,<a href="#fn4">6</a>,<a href="#cor1">*</a></sup> Shweta Tikoo,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Kathy On,<sup><a href="#aff1">1</a></sup> Brendon Martinez,<sup><a href="#aff1">1</a></sup> Suat Dervish,<sup><a href="#aff1">1</a>,<a href="#fn2">4</a></sup> Lois L. Cavanagh,<sup><a href="#aff1">1</a></sup> and Wolfgang Weninger<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Centenary Institute, Faculty of Medicine and Health, The University of Sydney, Camperdown, NSW 2050, Australia</p>
<p id="aff2"><sup>2</sup>Department of Dermatology, Medical University of Vienna, Vienna 1090, Austria</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Present address: Westmead Research Hub, Westmead Institute for Medical Research, Sydney, NSW, Australia</p>
<p id="fn3"><sup>5</sup>Technical contact</p>
<p id="fn4"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:R.Jain@centenary.org.au">R.Jain@centenary.org.au</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:W.Weninger@centenary.org.au">W.Weninger@centenary.org.au</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100722">10.1016/j.xpro.2021.100722</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Intravital multiphoton imaging of the tumor milieu allows for the dissection of intricate and dynamic biological processes <i>in situ</i>. Herein, we present a step-by-step protocol for setting up an experimental cancer imaging model that has been optimized for solid tumors such as breast cancer and melanoma implanted in the flanks of mice. This protocol can be utilized for dissecting tumor-immune cell dynamics <i>in vivo</i> or other tumor-specific biological questions.</p>
<p>For complete details on the use of this protocol for intravital imaging of breast cancer, please refer to <a href="https://doi.org/10.1101/2021.07.24.453629">Tikoo et al. (2021a)</a>, and for intravital imaging of melanoma, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref56">Tikoo et al. (2021b)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detailed protocol for setting up high-resolution intravital imaging of murine tumors</li>
<li>3D printing of custom stage inserts for tumor stabilization</li>
<li>Procedures for cannulation of blood vessels</li>
<li>Surgical preparation and tissue stabilization for imaging tumor milieu <i>in vivo</i>
</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Experimental design considerations</h3>
<p>For the successful implementation of this intravital imaging protocol, several factors associated with the experimental design and setup need to be considered. These factors, either individually or in combination, can dramatically alter a study’s outcome or adversely impact the quality and reproducibility of the data generated. We have highlighted some of these factors below. Researchers with adequate experience in setting up intravital imaging models can directly proceed to the section on the <a href="#sec1.2">generation of the stage inserts</a>.</p>
<ol>
<li>Choice of tumor model
<ol type="a">
<li>
<b>Breast Cancer:</b> The choice of tumor model relies on the scientific question being pursued (<a href="#bib12">Fantozzi and Christofori, 2006</a>). In the case of mammary tumors, employing genetically modified mouse models such as MMTV-PyMT or MMTV-Neu allows for the spontaneous generation of tumors akin to the human condition. In these model systems, unique oncogenes are overexpressed under tissue-specific promoters (<a href="#bib12">Fantozzi and Christofori, 2006</a>). To visualize immune cells, these mice strains need to be crossed to specific immune cell reporter mice on an identical genetic background. Conversely, fluorescent immune cells can be purified and adoptively transferred before imaging. Please note that MMTV-PyMT mice develop multifocal mammary adenocarcinomas (<a href="#bib17">Guy et al., 1992</a>), wherein the presence of multiple tumors within the ipsilateral mammary fat pads can lead to complications during surgical preparation for imaging, especially the increased risk of damaging patent vasculature. Therefore, the potential risk posed by these complications should be taken into consideration before commencing imaging.<br> Alternatively, implantation of mammary tumors can be performed using syngeneic or xenograft cells. While xenograft models allow for the implantation of well-characterized human mammary tumor cell lines such as MDA-MB-231, the need for implantation into immunocompromised mice, for example, NOD/SCID mice, curtails analysis of the role of adaptive immune cells. Nonetheless, xenograft models are useful for studying innate immune cells as well as the tumor cell behavior and physiology during cancer progression or drug therapy.<br> To gain a detailed understanding of anti-tumor immune responses in breast cancer, syngeneic tumor cell lines such as E0771 mammary adenocarcinoma (compatible with mice on C57BL/6 background) or 4T1 (compatible with mice on BALB/c background) can be utilized. Sub-clones of E0771 and 4T1 with varying metastatic dissemination characteristics are also available for deciphering the mechanisms of tumor cell metastasis (<a href="#bib21">Johnstone et al., 2015</a>). Syngeneic tumor cells can be conveniently implanted into the mammary fat pad to generate tumors with reproducible size and growth dynamics. In our laboratory, the transplantation of E0771 mammary adenocarcinoma cells into the inguinal mammary fat pad of young, virgin female mice is the method of choice (detailed in-depth below) (<a href="#bib64">Tikoo et al., 2021a</a>). Using this methodology, we regularly obtain a &gt;98% success rate in implantation with similar tumor immune profiles and tumor growth rates obtained between replicate experiments.</li>
<li>
<b>Melanoma:</b> For visualizing cellular dynamics within melanoma, researchers can employ various models, including genetically engineered mouse (GEM) melanoma models with or without ultraviolet radiation (UVR) exposure, syngeneic models, or patient-derived xenograft models (<a href="#bib40">Patton et al., 2021</a>, <a href="#bib44">Pérez-Guijarro et al., 2017</a>). GEM melanoma models harbor one or more driver mutations and have been instrumental in deciphering various facets of melanoma cell biology. GEM melanoma models also have an advantage in that they mimic physiological growth rates of melanoma development. Recent and emerging advances in genetic engineering will hopefully lead to the development of GEM melanoma models with inducible and temporally controlled inactivation of genes leading to the generation of more physiological models of melanoma (<a href="#bib40">Patton et al., 2021</a>, <a href="#bib44">Pérez-Guijarro et al., 2017</a>). Conversely, syngeneic mouse models have long been used to understand melanoma cell biology mainly due to their low cost, ease of usage, and fast turnaround times. One of the most commonly used syngeneic cell lines for generating melanoma tumors are the B16 lines (<a href="#bib37">Overwijk and Restifo, 2001</a>), particularly the B16-F10 cells (<a href="#bib16">Guo et al., 2018</a>, <a href="#bib15">Guo et al., 2020</a>, <a href="#bib56">Tikoo et al., 2021b</a>). Our cell line of choice is the recently generated amel-B16-F10 melanoma cell line which is compatible with the intravital imaging modality due to its lack of melanin pigmentation (<a href="#bib56">Tikoo et al., 2021b</a>). Additionally, patient-derived xenografts are an excellent choice for understanding melanoma cell biology as they capture the tumor heterogeneity and provide a spectrum of genetic diversity. However, patient-derived tumors have to be implanted into an immunocompromised mouse, therefore, limiting the understanding generated of the role of immune system (<a href="#bib40">Patton et al., 2021</a>).</li>
</ol>
</li>
</ol>
<ol start="2">
<li>
<b>Choice of microscopic equipment:</b> Compared to confocal microscopy, multiphoton microscopy permits deeper tissue imaging with less phototoxicity (<a href="#bib19">Jain et al., 2016</a>). We employ multiphoton imaging modality using the upright LaVision BioTec TriMScope equipped with a 20× (NA 0.95) Olympus water immersion objective (<a href="#fig1">Figure 1</a>), but several other commercial multiphoton systems are also readily available. Our setup has been configured to include up to eight external, non-descanned, dual-channel reflection/fluorescence detectors, and three diode-pumped, wideband, mode-locked Ti-Sapphire femtosecond lasers (Mai Tai®, 730–850 nm; Mai Tai Deep See™, 850–1010 nm, and Insight Deep See™, 1010–1300 nm; supplied by Spectra-Physics, Newport, USA) (<a href="#fig1">Figure 1</a>). For most imaging studies, we use 4–5 detectors to visualize spectrally distinct fluorophores with minimal overlap.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic representation of the intravital tumor imaging setup</div>
<p>Layout of the intravital imaging setup used for imaging tumors.</p>
</figcaption>
</figure>
</li>
<li>
<b>Visualization of tumor vasculature:</b> Most tumor imaging studies employ visualization of tumor vasculature (<a href="#tbl1">Table 1</a>). The advent of transgenic reporter mice strains, such as mT/mG and Tie2-GFP, which feature fluorescently labeled blood vessels, has enhanced the quality of data that can be obtained without having to inject a plasma dye during the imaging session. Briefly, mT/mG is a double fluorescent, Cre reporter strain wherein all cells constitutively express membrane-targeted, tandem dimer tomato under the control of ROSA26 promoter. In these mice, Cre-mediated excision of mTomato results in expression of the membrane-targeted GFP within the Cre expressing cells (<a href="#bib34">Muzumdar et al., 2007</a>). However, in the absence of Cre activity, blood vessels can be easily distinguished from the surrounding tissue due to the brighter mTomato fluorescent signal. Tie2-GFP transgenic mice were developed by Motoike and co-workers wherein the Tie2 promoter and enhancer elements were utilized to drive the expression of GFP in vascular endothelial cells (<a href="#bib31">Motoike et al., 2000</a>). In the event where these transgenic reporter mice strains are unavailable or the transgenic fluorophores conflict with a given experimental setup, various vascular dyes can be used to visualize blood vessels (<a href="#tbl1">Table 1</a>). A good vascular dye is crucial for imaging the behavior of tumor and immune cells at the endothelial interface. Our dye of choice is Evans blue conjugated to bovine serum albumin, referred to as Evans blue-BSA (preparation detailed in <a href="#materials-and-equipments">Materials</a>). The conjugation of Evans blue to BSA enables the vascular dye to be retained within the vasculature for extended periods of time when imaging inflamed tissues. Experimental designs requiring the analysis of vascular leakage may prefer to use unconjugated Evans blue for imaging. The single-photon emission peak of Evans blue is 680 nm, and the excitation peaks are at 470 and 540 nm. These excitation and emission properties allow for simultaneous visualization of Evans blue-BSA with other commonly used fluorophores such as CFP, GFP, RFP, and mCherry. We regularly use two-photon excitation wavelength range from 890–940 nm to excite Evans blue-BSA, a wavelength range suitable for the excitation of CFP, GFP and YFP. In most cases, we employ the Insight Deep See™ for the excitation of RFP and mCherry (<a href="#fig7">Figure 7</a>B).
<table id="tbl1">
<caption>Table 1. List of various methodologies utilized for visualizing blood vessels</caption>
<thead>
<tr>
<th>Labeling methodology</th>
<th>Advantages</th>
<th>Disadvantages</th>
</tr>
</thead>
<tbody>
<tr>
<td>mT/mG transgenic reporter mice</td>
<td>Endogenous labeling of all cells (<a href="#bib34">Muzumdar et al., 2007</a>). Blood vessels are marked by high fluorescence intensity of the mTomato signal (<a href="#bib25">Li et al., 2012</a>). If crossed to a Cre reporter strain for example, CD11c-Cre, both blood vessels (red) and dendritic cells (green) could be visualized (<a href="#bib53">Stoler-Barak et al., 2014</a>). It can also be used in conjunction with inducible Cre reporters to temporally regulate transgene expression.</td>
<td>Not useful for studying the vascular and perivascular structures at high resolutions as mTom<sup>+</sup> tumor-infiltrating cells could lead to the loss of endothelial cell resolution.</td>
</tr>
<tr>
<td>Tie2-GFP transgenic reporter mice</td>
<td>Excellent model for labeling blood vessel networks (<a href="#bib11">De Palma et al., 2005</a>).</td>
<td>Highlights other cell types such as Tie2<sup>+</sup> macrophages (<a href="#bib43">Pucci et al., 2009</a>, <a href="#bib11">De Palma et al., 2005</a>). GFP expression in blood vessels may interfere with the use of other transgenic reporter strains.</td>
</tr>
<tr>
<td>Evans blue-BSA</td>
<td>Cheap, readily excited at several wavelengths with emission fluorescence in the far-red spectral range (<a href="#bib56">Tikoo et al., 2021</a>).</td>
<td>Vascular leakage can result in the loss of resolution.</td>
</tr>
<tr>
<td>Quantum dots</td>
<td>Can be used for a variety of purposes including highlighting blood vasculature and vascular leakage (<a href="#bib58">Vennin et al., 2017</a>).</td>
<td>Expensive.</td>
</tr>
<tr>
<td>Fluorescently labeled dextrans</td>
<td>Available in a range of different molecular weights. Fluorescently labeled dextrans of varying molecular weights can be employed to visualize vascular permeability. (10,000-2,000,000 Da) (<a href="#bib38">Pai et al., 2014</a>).</td>
<td>Expensive, vascular leakage compromises resolution.</td>
</tr>
<tr>
<td>Fluorescently labeled lectins</td>
<td>Binds to blood vessels (<a href="#bib45">Robertson et al., 2015</a>).</td>
<td>Toxic at high doses.</td>
</tr>
<tr>
<td>Fluorescently labeled anti-CD31 antibody</td>
<td>Specifically labels endothelial cells (<a href="#bib45">Robertson et al., 2015</a>).</td>
<td>Expensive.</td>
</tr>
</tbody>
</table>
</li>
</ol>
<p>An additional benefit of using Evans blue-BSA is that it acts as an internal control for the quality of the skin flap preparation. Extensive leakage of the vascular dye indicates tissue damage inflicted during the surgical preparation. Caution should be observed during the surgical preparation of the tumor to minimize potential injury to the patent vasculature.</p>
<ol start="4">
<li>
<b>Choice of fluorophores:</b> Prior to setting up an imaging model, care should be taken to choose a set of fluorophores with minimal spectral overlap. This ensures relative ease in identifying various cell subsets during imaging and analysis. Availability of fluorescent transgenic reporter strains (<a href="#tbl2">Table 2</a>) and their intercrosses should be considered for the experimental design. Another important consideration before choosing the experimental fluorophores is the filter setup available for the multiphoton microscope. Nonetheless, in scenarios wherein spectral overlap cannot be avoided, for example, between fluorophores such as GFP and YFP or CFP and GFP, the extent of bleed-through between channels can be minimized at the time of acquisition by fine-tuning the excitation wavelengths (in the case of tunable lasers) and optimizing the filter setup. Various imaging software, for example, FIJI (<a href="#bib49">Schindelin et al., 2012</a>) and ImageJ (<a href="#bib50">Schneider et al., 2012</a>, <a href="#bib10">Collins, 2007</a>, <a href="#bib46">Rueden et al., 2017</a>), also allow for correcting spectral overlaps post-acquisition.
<table id="tbl2">
<caption>Table 2. List of various fluorescent transgenic reporter mice</caption>
<thead>
<tr>
<th>Transgenic reporter mice</th>
<th>Abbreviation</th>
<th>Commonly used for</th>
</tr>
</thead>
<tbody>
<tr>
<td>cKit-GFP (<a href="#bib3">Berrozpe et al., 2006</a>)</td>
<td>cKit <sup>BAC-GFP</sup>
</td>
<td>GFP expressed under the control of c-kit locus. Can be utilized for imaging mast cells <i>in vivo</i> (<a href="#bib57">Tong et al., 2015</a>, <a href="#bib55">Tikoo et al., 2018</a>).</td>
</tr>
<tr>
<td>mT/mG (<a href="#bib34">Muzumdar et al., 2007</a>)</td>
<td>mT/mG</td>
<td>mTomato is expressed in all cells (<a href="#bib25">Li et al., 2012</a>). Cre mediated recombination switches expression from mTom to mGFP.</td>
</tr>
<tr>
<td>Tie2-GFP (<a href="#bib31">Motoike et al., 2000</a>)</td>
<td>Tie2<sup>GFP</sup>
</td>
<td>GFP expression on vascular endothelial cells. Aids in visualizing blood vessels (<a href="#bib11">De Palma et al., 2005</a>).</td>
</tr>
<tr>
<td>CD11c-YFP (<a href="#bib26">Lindquist et al., 2004</a>)</td>
<td>CD11c<sup>YFP</sup>
</td>
<td>DCs can be visualized by virtue of their expression of eYFP (<a href="#bib61">Ng et al., 2008</a>, <a href="#bib5">Boissonnas et al., 2013</a>, <a href="#bib6">Boldajipour et al., 2016</a>).</td>
</tr>
<tr>
<td>CXCR6-GFP (<a href="#bib60">Unutmaz et al., 2000</a>)</td>
<td>CXCR6<sup>GFP</sup>
</td>
<td>GFP is knocked into the CXCR6 gene locus. Can be utilized for imaging dermal γδ T cells (<a href="#bib54">Sumaria et al., 2011</a>, <a href="#bib14">Gray et al., 2011</a>), dendritic epidermal T cells (<a href="#bib14">Gray et al., 2011</a>), dermal ILC-2 (<a href="#bib69">Roediger et al., 2013</a>), invariant NKT cells (iNKT cells) (<a href="#bib70">Wong et al., 2011</a>) and tumor infiltrating T cells (<a href="#bib29">Matsumura et al., 2008</a>).</td>
</tr>
<tr>
<td>MacBlue (<a href="#bib48">Sauter et al., 2014</a>)</td>
<td>Csf1r<sup>CFP</sup>
</td>
<td>
<i>Csf1r</i> promoter drives the expression of gal4-VP16, which in turn drives a co-integrated gal4-responsive UAS-ECFP cassette. Monocytes, macrophages, and other CD11b cells express CFP in these mice with highest transgene expression observed in monocytes (<a href="#bib56">Tikoo et al., 2021</a>, <a href="#bib23">Laviron et al., 2019</a>).</td>
</tr>
<tr>
<td>MacGreen (<a href="#bib47">Sasmono et al., 2003</a>)</td>
<td>Csf1r<sup>GFP</sup>
</td>
<td>Monocytes, macrophages, and other CD11b cells express GFP (<a href="#bib20">Jain and Weninger, 2013</a>, <a href="#bib23">Laviron et al., 2019</a>).</td>
</tr>
<tr>
<td>CX3CR1-GFP (<a href="#bib22">Jung et al., 2000</a>)</td>
<td>CX3CR1<sup>EGFP</sup>
</td>
<td>Monocytes and macrophages express GFP. Used for <i>in vivo</i> imaging of tumor infiltrating myeloid cells (<a href="#bib23">Laviron et al., 2019</a>, <a href="#bib24">Lee et al., 2018</a>)</td>
</tr>
<tr>
<td>LyzM-GFP (<a href="#bib13">Faust et al., 2000</a>)</td>
<td>
<i>Lyz2</i><sup>EGFP</sup>
</td>
<td>GFP knock-in in the lysozyme M locus. Neutrophils, monocytes and macrophages express GFP. Care should be taken while breeding so as to use only heterozygous transgenic mice in experiments as the strain is a knock-in (<a href="#bib56">Tikoo et al., 2021</a>, <a href="#bib30">McDonald et al., 2010</a>, <a href="#bib23">Laviron et al., 2019</a>, <a href="#bib24">Lee et al., 2018</a>) .</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This protocol can also be employed to carry out label-free intravital imaging using Simultaneous Label-free Autofluorescence Multiharmonic (SLAM) microscopy. SLAM microscopy is an optical imaging platform that enables the detection of Nicotinamide adenine dinucleotide (NADH) from 3-photon autofluorescence imaging (3PAF), Flavin adenine dinucleotide (FAD) from 2-photon autofluorescence imaging (2PAF), detection of non-centrosymmetric structures from second-harmonic generation (SHG), and interfacial features from third-harmonic generation (THG) (<a href="#bib59">You et al., 2018</a>). SLAM microscopy has recently been shown to be capable of simultaneously visualizing leukocytes, red blood cells, tumor cells, adipocytes, collagen fibers, and microvasculature with sufficient resolution owing to the simultaneous excitation of autofluorescence and multi-harmonic processes (<a href="#bib59">You et al., 2018</a>).</div>
<ol start="5">
<li>
<b>Choice of anesthesia:</b> Before surgical preparation of mice, we administer via intraperitoneal injection a combined dose of injectable anesthetics, ketamine/xylazine (100/10 mg/kg) to provide deep surgical plane anesthesia. Anesthetized mice are monitored every 20–40 min for adequate depth of anesthesia. Repeated half doses of the anesthetics are administered post-monitoring leading to successful imaging of the tumor tissue for up to a period of five hours. Alternatively, inhalational anesthetics such as isoflurane can be employed to maintain mice under surgical plane anesthesia. Care should be taken to ensure that mice are carefully monitored to avoid discomfort, hypoxemia, and hypercapnia due to the incorrect dosing of anesthetics. In case of excessive movement observed for tissues being imaged, additional drugs such as acepromazine can be used in combination with the anesthetics. Care should be taken when using these drugs as they can potentiate the effects of anesthetics.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Protocols utilizing injectable as well as inhalational anesthetics must be approved by the institutional safety committee to avoid any risks to the operator. The use of anesthetics raises concerns regarding the health and safety of research staff. These concerns should be properly addressed before commencing the experiments.</div>
<div class="optional">
<span class="optional-title">Optional:</span> In scenarios wherein researchers would like to image continuously without several pauses required for monitoring mice or minimize movement artefacts due to repeated monitoring, and anesthetic injections, automated physiological monitoring systems such as the PhysioSuite from Kent Scientific Corporation can be employed. Additionally, venous cannulation can be used to administer anesthetic top-ups. Care should be taken with the dosing of anesthetics when administering directly into the venous circulation.</div>
<p><i>Ketamine</i>- Ketamine is a controlled substance (S8, Australia; Schedule III, USA; Class B, UK) and may require relevant licenses to hold and use. Therefore, it is crucial that large volumes of the drug are stored in a safe and secure place with limited access and rigorous monitoring of the usage. As an example, we store large volumes in fireproof safety lockers with a single individual holding access to the drug and regular monitoring of the usage performed by an independent safety and compliance officer. The drug is dispensed to individual (approved) researchers depending on their experimental usage. Similarly, possession and use of xylazine, a prescription drug (S4, Australia), might require approval by relevant authorities.</p>
<p><i>Isoflurane</i>- The use of an inhalational anesthetic is dependent on the availability of equipment capable of providing a constant flow of the vaporized anesthetic. As tumor imaging is performed over an extended period of time, it is essential that the excess vented anesthetic is sequestered using chemical scrubbers. As most imaging setups employ a chamber for maintaining humidity and preventing interference from extraneous light sources, even small leaks in the anesthetic setup can result in the build-up of anesthetic vapor. Similarly, the use of inhalational anesthetics requires a constant supply of oxygen gas which is highly flammable. Therefore, it is crucial that before proceeding with experiments, the institutional safety officers approve the use of inhalational anesthetics within the facility. Alternatively, an advanced system can be set up wherein the operation of the microscope and monitoring of the mice can be performed remotely (usually in an antechamber).</p>
<ol start="6">
<li>
<b>Duration of imaging:</b> Employing this methodology, we have been able to image continuously for durations of more than three hours. Imaging sessions lasting several hours are also feasible with this protocol. However, we do pause the imaging at 20–30 min intervals to check for the depth of anesthesia via palpebral reflex and hindlimb withdrawal. Repeated half doses of ketamine/xylazine are administered intraperitoneally when required. These pauses should be taken into consideration when time-lapse videos are merged, particularly during image analysis. The duration of imaging and the time interval between individual imaging frames depends on the cellular or molecular processes under investigation. Usually, analysis of slower cellular processes such as tumor cell migration or cell cycle dynamics requires imaging over extended periods of time, albeit with large time intervals between individual acquisitions. For example, tumor cell migration dynamics can be studied by imaging for more than 2 h, with data acquisition occurring every 2 min. To study the dynamic behavior of immune cells, for example, intravascular crawling of monocytes, data is acquired for 1 h at 30-s intervals or less. The precise time setup depends on several parameters, including the resolution of the image, the volume of the z stack, the intensity of fluorophores, and the biological process being studied. For most data acquisition, it is crucial that the frame rate is kept constant during the imaging period. Moreover, when comparing datasets from multiple experiments, the frame rate should be kept constant between experiments. To ascertain the frame rate for acquisition, it is recommended that there be a partial overlap between the cell body of a cell being tracked in subsequent frames. In scenarios wherein, cells are migrating at very high speeds, for example, rolling cells within the vascular lumen, the cell body should be clearly distinguishable between the acquisition frames to generate reliable data. If sudden jumps (sudden disappearance followed by reappearance) are observed for the cells being tracked, the dataset should be excluded. Please note that once the parameters for a given study are optimized, they should be kept constant between experiments.</li>
<li>
<b>Image Analysis:</b> The data generated from multiphoton microscopy can either be analyzed employing commercial software packages, for example, Volocity (<a href="https://quorumtechnologies.com/index.php/component/content/category/31-volocity-software">Quorum Technologies</a>) and Imaris (<a href="https://imaris.oxinst.com/">Oxford Instruments</a>), or open-source image analysis platforms such as <a href="http://imagej.nih.gov/ij/">Image J</a> (<a href="#bib50">Schneider et al., 2012</a>, <a href="#bib10">Collins, 2007</a>, <a href="#bib46">Rueden et al., 2017</a>) or <a href="https://fiji.sc/">FIJI</a> (<a href="#bib49">Schindelin et al., 2012</a>).</li>
</ol>
<h3 id="sec1.2">Generation of stage inserts</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8–12 h</div>
<p>Tissue movement introduces artefacts during high-resolution imaging thus interfering with post-acquisition image analysis. Therefore, time-lapse imaging of tumors, similar to other tissues, requires external stabilization. Imaging of melanoma and breast tumors entails surgical exposure of the tumor, thereby providing a large vascularized surface for imaging. Nonetheless, the tumor shape and size can make immobilization of the sample challenging. To achieve stable imaging of the tumor tissue, we have modified our previously described ear skin-imaging model (<a href="#bib25">Li et al., 2012</a>) (<a href="#fig2">Figure 2</a>A). To stabilize and superfuse the tumor tissue, we have employed 3D printing technology to generate stage inserts of variable heights to accommodate varying tumor volumes (<a href="#fig2">Figure 2</a>B), as detailed below.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Setup of imaging stage</div>
<p>(A) Schematic representation of the tumor imaging stage highlighting the organization and dimensions of various elements.</p>
<p>(B) Side view of the tumor imaging stage.</p>
<p>(C) Images of stage inserts used for stabilizing and perfusing the tumor.</p>
<p>Please note: The height of the stage appears shorter than the equivalent length; this effect is due to the angle of the object within the image.</p>
</figcaption>
</figure>
<ol start="8">
<li>We designed the 3D printed stage inserts compatible with our imaging stage setup (<a href="#fig2">Figures 2</a>A and 2B) using Google Sketchup v8 and exported as STL files for G-code generation. Due to the localization of the tumors in the flank, a large number of angiogenic blood vessels arise from the cranial or caudal superficial epigastric artery and vein. Therefore, the stage inserts need to be printed with a small notch to accommodate the neoangiogenic vasculature (<a href="#fig2">Figure 2</a>C). A template of the design is available as <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Mmc1.zip">Data S1</a> and <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Mmc2.zip">S2</a>.</li>
<li>MakerBot Desktop was used to generate printer-specific G-code for 3D printing on a MakerBot Replicator 2× using Acrylonitrile-Butadiene Styrene (ABS).</li>
<li>To stabilize the inserts, we embedded 4 mm<sup>3</sup> neodymium magnets in the insert, which engages with magnets embedded underneath the stage (<a href="#fig2">Figure 2</a>B and 2C).</li>
</ol>
<p>Using our methodology, the stage platform along with the tumor is maintained at 37°C throughout the imaging period utilizing the feedback system (Temperature controller TC-344C) (<a href="#bib25">Li et al., 2012</a>). In a scenario wherein, the stage cannot be constructed, researchers can 3D-print a stage with a raised platform (with embedded magnets) and superfuse the inserts with warm saline to maintain the tissue temperature at 37°C.</p>
<div class="note">
<span class="note-title">Note:</span> The use of 3D printed inserts detailed above provides researchers with a unique opportunity for utilizing tumor models wherein the localization of tumor cannot be pre-emptively determined; for example, in the case of GEM models of melanoma and MMTV-PyMT mice. These inserts can be printed in less than an hour and can be generated to varying heights and shapes. Notches for maintaining patent vasculature can be created by gently filing the base of the insert to accomodate the angiogenic vasculature. Although we prefer to retain a large chamber cavity in the inserts (inserts compatible with most tumor models), other researchers may prefer to use inserts with limited space around the tumors to enhance stability. The use of 3D printed stage inserts also allows researchers to utilize their pre-existing imaging setups, thereby reducing the setup cost. These stage inserts also provide a unique access point for adding drugs, dyes, and small molecule inhibitors within the tumor milieu, bypassing the systemic circulation.</div>
<h3 id="sec1.3">Preparation of tumor cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min</div>
<p>For syngeneic or xenograft models, follow the protocol detailed below. For imaging of spontaneous tumors generated in mice, for example, in the MMTV-PyMT model, proceed to the section on <a href="#step-by-step-method-details">step by step method details</a><i>.</i></p>
<p>This protocol has been specifically used for:</p>
<ul>
<li>B16-F10 and amel-B16-F10 cells and their fluorescent-tagged versions for implanting tumors in the flanks of C57Bl/6 mice.</li>
<li>E0771 cells and their fluorescent-tagged versions for implanting tumors in the mammary fat pads of C57Bl/6 mice.</li>
<li>4T1 cells and their fluorescent-tagged versions for implanting tumors in the mammary fat pads of BALB/c mice.</li>
</ul>
<p>For xenograft models, we recommend rendering the tumor cells fluorescent for imaging purposes. Fluorescent tumor cells can be generated by standard retroviral or lentiviral transduction methods.</p>
<ol start="11">
<li>Maintain the cells in DMEM, high glucose, GlutaMAX™ + 10% FCS + sodium pyruvate, and routinely passage at optimal confluency of 60%–80%.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Routinely test the tumor cell lines for mycoplasma and other microbial contaminants in accordance with institutional policies.</div>
<ol start="12">
<li>On the day of the injection, wash the tumor cells growing at 70%–80% confluency three times with 1× PBS. We regularly use one TC150 cm<sup>2</sup> tissue culture flask at the above-mentioned confluency to prepare cells for injecting up to 15 mice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is crucial to use cells that are metabolically healthy and not overgrown or stressed.</div>
<ol start="13">
<li>Detach the tumor cells by incubating with 0.05% Trypsin EDTA for 30 s–1 min followed by gently tapping the cell culture flask to dislodge the adherent cells.</li>
<li>Arrest trypsinization by adding 10–15 mL of FBS containing growth media.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Care should be taken not to incubate the cells with Trypsin for extended periods of time as this reduces the viability of tumor cells.</div>
<ol start="15">
<li>Collect the cells via centrifugation at 524×<i>g</i> for 5 min.</li>
<li>Resuspend the resulting pellet in 1× PBS. Count and subsequently adjust the cell suspension to a concentration of 2.5–5.0 million viable tumor cells/mL.
<ol type="a">
<li>Assess the cell viability by trypan blue exclusion. Estimate the cell numbers using the Neubauer chamber.</li>
<li>Cell viability of &gt;95% is deemed acceptable for most experiments.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When estimating cell numbers, only viable cell count needs to be taken into consideration.</div>
<ol start="17">
<li>Filter the cell suspension through a 100 μm cell strainer to remove large cell clumps, which can result in variability in the cell number injected. Transfer the cell suspension into an Eppendorf tube and place on ice until injection. Troubleshooting <a href="#sec6.1">problem 1</a>
</li>
</ol>
<h3 id="sec1.4">Tumor cell implantation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20–30 min</div>
<ol start="18">
<li>Using a 1 mL syringe equipped with a 25G needle, inject 0.1 mL of the tumor cell suspension into the left flank of the anesthetized mice (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Methodology for implantation of tumors</div>
<p>(A) Schematic representation of the methodology used to orthotopically implant melanoma in mice.</p>
<p>(B) Illustration of the methodology used to transplant mammary tumors in mice.</p>
<p>Please note, the changes in the direction of tumor injection.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> We inject tumor cells only within the left flank of the mice due to the design of the stage inserts. In case the researchers require tumor injection in the right flank, please ensure that the notch for blood vessels is etched based on the new orientation for tumor imaging. In brief, for successfully imaging the tumors implanted within the right flank, the mouse needs to be rotated 180 degrees on the stage (<a href="#fig6">Figure 6</a>, step 2); therefore, the notch for caudal blood vessels needs to be placed on the cis-end of the insert (<a href="#fig2">Figure 2</a>C, left cis side of the insert rather the presently depicted right).</div>
<ol type="a">
<li>
<b>Melanoma cell implantation</b>: Inject melanoma cells subcutaneously into the flank of male or female anesthetized mice (<a href="#fig3">Figure 3</a>A, step 2 or 3). We typically inject 0.25–0.5 million B16-F10 melanoma cells in mice using the methodology detailed below. This number of tumor cells generates palpable tumors within ten days, with 0.9 cm<sup>3</sup> tumors (end-point criteria) obtained within approximately two to three weeks.
<div class="critical">
<span class="critical-title">Critical:</span> During tumor cell implantation, it is crucial to ensure that the needle stays within the subcutaneous tissue and does not enter the peritoneum. We recommend keeping the angle of injection very shallow to achieve this. For successful injection, the syringe should be parallel to the abdominal skin (<a href="#fig3">Figure 3</a>A, step 2), with the cells being injected as soon as the skin is punctured (marked by the loss of resistance to needle penetration). Alternatively, the mice can be positioned on their side, and the syringe can be placed parallel to the chest with the needle facing the forelimb. The injection is performed distal from the hind limb (<a href="#fig3">Figure 3</a>A, step 3). Post-injection, a slight bulge in the skin signifies successful implantation of tumor cells (<a href="#fig3">Figure 3</a>A, step 4). Care should be taken to inject the tumor cells distal to the mammary fat pad at a shallow depth to prevent cell deposition in the mammary fat pad.</div>
<div class="note">
<span class="note-title">Note:</span> For injecting melanoma cells in male mice, the general guidelines stated above should be followed, wherein the melanoma cells are injected away from the hind limbs as soon as the skin is punctured. The absence of nipples in male mice creates difficulty in identifying the key landmarks. Therefore, we recommend using female mice for setting up the model. Once sufficiently experienced, researchers can easily perform this procedure on male mice. Also, please note that the male mice are prone to fighting; therefore, adequate observation or separation of individual mice may be required to ensure that the growing tumor is not impacted.</div>
</li>
<li>
<b>Breast cancer cell implantation</b>: For generating mammary tumors, inject the cells into the mammary fat pad of a virgin female mouse (8–12 weeks of age) (<a href="#fig3">Figure 3</a>B, step 2 or 3). We typically inject 0.25–0.5 million E0771 tumor cells in the inguinal mammary fat pad of mice using the methodology detailed below. This number of tumor cells generates palpable tumors within ten days, with 0.9 cm<sup>3</sup> tumors (end-point criteria) obtained within approximately two to three weeks.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> During tumor cell implantation, it is crucial to ensure that the needle stays within the mammary fat pad and does not enter the peritoneum. We recommend keeping the angle of injection very shallow to achieve this. For successful injection into the mammary fat pad, the syringe should be parallel to the abdominal skin (<a href="#fig3">Figure 3</a>, step 2), with the cells being injected as soon as the skin is punctured (marked by the loss of resistance to needle penetration). Alternatively, the mice can be positioned on their side, and the syringe can be placed parallel to the chest with the needle facing the hind limb. The injection is performed just above the hind limb under the skin (<a href="#fig3">Figure 3</a>, step 3). Post-injection, a slight bulge in the skin signifies successful implantation of tumor cells in the mammary fat pad (<a href="#fig3">Figure 3</a>, step 2).</div>
<ol start="19">
<li>Rotate the syringe while withdrawing the needle to seal the puncture wound and to prevent any loss of cell suspension.</li>
<li>Allow the tumor to grow for 2–4 weeks with continuous monitoring.</li>
<li>Image the tumor once it reaches a volume of 0.5–0.9 cm<sup>3</sup>. In our experience, imaging tumors at this particular size is optimal for studying immune responses. However, larger or smaller tumors can also be imaged successfully depending upon the biological question being pursued. Troubleshooting <a href="#sec6.3">problem 2</a>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Animal monitoring should be carried out in accordance with the approved institutional ethics protocols. Caution should be observed while maintaining tumor-bearing mice. Animals with tumors that begin to show signs of ulceration should be removed and not included in the imaging experiments. Relevant ethics guidelines should always be followed during tumor growth in mice. Undue stress or inflammation can alter the tumor milieu and can result in variability in the data. Special consideration should be given to the well-being of the tumor-bearing mice in accordance with the approved ethics guidelines.</div>
<h3 id="sec1.5">Immune cell preparation and labeling</h3>
<p>For experiments wherein labeled immune cells need to be tracked <i>in vivo</i>, several purification and labeling methodologies can be employed depending on the immune cell population being evaluated. The exogenous administration of immune cells can also be used when comparing two immune cell populations under identical conditions. For example, utilizing <i>ex</i> <i>vivo</i> expanded antigen-specific effector CD8<sup>+</sup> T cells, we were able to highlight the ability of effector T cells to adhere to inflamed vasculature in the absence of G<sub>i</sub> signaling (<a href="#bib52">Shulman et al., 2011</a>). Similarly, we employed this strategy to highlight the importance of Granzyme B in the transmigration of effector T cells during sterile injury (<a href="#bib42">Prakash et al., 2014</a>).</p>
<ol start="22">
<li>
<b>Immune cell preparation</b>- The precise methodology used for immune cell preparation is dependent on the immune cell population being studied and the scientific question being pursued. For example, neutrophils can be purified from the bone marrow of mice utilizing the magnetic sorting methodology (Miltenyi Biotec, <a href="https://www.miltenyibiotec.com/AU-en/products/neutrophil-isolation-kit-mouse.html">Neutrophil isolation kit</a>) (<a href="#bib35">Ng et al., 2011</a>) or Percoll gradient (<a href="#bib27">Lämmermann et al., 2013</a>). The number of highly purified neutrophils obtained using the magnetic sorting methodology (2<b>×</b>10<sup>7</sup> purified neutrophils) corresponds to 2% of total blood neutrophils when administered intravenously and would represent a small fraction of events being studied within the inflamed tissues (<a href="#bib35">Ng et al., 2011</a>). Nonetheless, the purification methodologies mentioned above would be adequate to study migration dynamics of neutrophils when injected directly into the tissue such as the dermis (<a href="#bib35">Ng et al., 2011</a>). In scenarios wherein neutrophil recruitment needs to be studied, it is best to utilize either the <i>Lyz2</i><sup>gfp/+</sup> mice (<a href="#bib27">Lämmermann et al., 2013</a>, <a href="#bib30">McDonald et al., 2010</a>, <a href="#bib41">Peters et al., 2008</a>, <a href="#bib8">Casanova-Acebes et al., 2018</a>) or their bone marrow for adoptive transfer (<a href="#bib1">Abtin et al., 2014</a>).</li>
</ol>
<p>Similarly, as described above, <i>ex vivo</i> antigen primed and expanded effector T cells can be utilized for studies (<a href="#bib52">Shulman et al., 2011</a>, <a href="#bib42">Prakash et al., 2014</a>). However, the ability of T cells to respond to tumor-derived antigens provides an opportunity wherein endogenously fluorescent T cells can be transferred into mice bearing tumor cells expressing cognate antigen (<a href="#bib33">Mrass et al., 2006</a>, <a href="#bib32">Mrass et al., 2008</a>, <a href="#bib7">Breart et al., 2008</a>, <a href="#bib9">Cazaux et al., 2019</a>, <a href="#bib39">Park et al., 2019</a>). This methodology allows for effector T cells to infiltrate tumor tissue over extended period of time thereby permitting robust tracking <i>in vivo</i>.</p>
<ol start="23">
<li>
<b>Immune cell labeling</b>- Several commercially available dyes can be utilized for labeling immune cells with minimal cytotoxicity. However, due to the possibility of dilution during cell division, transfer of dyes between cell populations, or photobleaching during longitudinal imaging, it is highly recommended that endogenously fluorescent cells are utilized wherever possible. A brief list of commonly used fluorescent reporter mice strains is presented in <a href="#tbl2">Table 2</a>. Alternatively, labeling can be performed utilizing DNA transfection (<a href="#bib33">Mrass et al., 2006</a>) or retroviral transduction (<a href="#bib36">Obeidy et al., 2020</a>).</li>
</ol>
<p>Nonetheless, commonly used labeling dyes such as <a href="https://www.thermofisher.com/order/catalog/product/C34554#/C34554">CFSE</a> and <a href="https://www.thermofisher.com/order/catalog/product/C2927?SID=srch-hj-CMTMR#/C2927?SID=srch-hj-CMTMR">CMTMR</a> can be utilized for labeling immune cells as per the manufacturer’s protocol (please see the detailed staining protocol for <a href="https://www.thermofisher.com/order/catalog/product/C34554#/C34554">CFSE</a> and <a href="https://www.thermofisher.com/order/catalog/product/C2927?SID=srch-hj-CMTMR#/C2927?SID=srch-hj-CMTMR">CMTMR</a>) prior to adoptive transfer. To avoid labeling-induced artifacts confounding the study, we recommend that wherever possible, researchers should swap the label between the populations (<a href="#bib27">Lämmermann et al., 2013</a>).</p>
<p>Therefore, we strongly advise that prior to tumor imaging, researchers optimize the immune cell preparation and labeling. An optimization experiment is essential, wherein all steps required for the proposed experiment are performed, but the tumor is harvested, fixed, sectioned and imaged using a confocal microscope to ascertain the abundance and distribution of cells being tracked, their labeling intensity, and spectral separation between fluorophores (when two or more fluorophores are being employed). As intravital imaging focuses on smaller regions (usually less than 500×500 μm), the use of confocal microscopy, especially the tiling function, can help researchers assess the parameters that need to be further optimized before intravital imaging can be successfully undertaken.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM</td>
<td>GIBCO/Life Technologies</td>
<td>Cat. No. 11960-044</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>Sigma Aldrich</td>
<td>Cat. No. P5280-25G</td>
</tr>
<tr>
<td>Trypsin EDTA</td>
<td>GIBCO/Life Technologies</td>
<td>Cat. No. 25300-054</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>GIBCO/Life Technologies</td>
<td>Cat. No. 15140-122</td>
</tr>
<tr>
<td>HBSS</td>
<td>GIBCO/Life Technologies</td>
<td>Cat. No. 14175-095</td>
</tr>
<tr>
<td>PBS</td>
<td>GIBCO/Life Technologies</td>
<td>Cat. No. 10010-023</td>
</tr>
<tr>
<td>Evans Blue</td>
<td>Sigma Aldrich</td>
<td>E2129-10G</td>
</tr>
<tr>
<td>Bovine Serum Albumin, Fraction V</td>
<td>Sigma Aldrich</td>
<td>Cat. no.10735078001</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>mCherry expressing E0771 breast adenocarcinoma cells</td>
<td>Available from authors</td>
<td>Available from authors</td>
</tr>
<tr>
<td>B16-F10 amelanotic melanoma cells</td>
<td>Available from authors</td>
<td><a href="#bib56">Tikoo et al., 2021</a></td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: Female MacBlue mice on C57Bl/6 background (Age: 8–12-weeks)</td>
<td><a href="#bib48">Sauter et al. 2014</a></td>
<td>JAX Stock no-026051 <a href="https://elsevier.proofcentral.com/en/5587942">5587942</a>
</td>
</tr>
<tr>
<td>Mouse: Male or female <i>Lyz2</i><sup><i>gfp</i></sup> on C57Bl/6 background (Age: 8–12 weeks)</td>
<td><a href="#bib13">Faust et al., 2000</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Volocity</td>
<td>Quorum Technologies</td>
<td><a href="https://quorumtechnologies.com/index.php/component/content/category/31-volocity-software">Quorum Technologies</a></td>
</tr>
<tr>
<td>Imaris</td>
<td>Oxford Instruments</td>
<td><a href="https://imaris.oxinst.com/">Oxford Instruments</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>RightTemp® Temperature Monitor and Homeothermic Warming Control Module.</td>
<td>Kent Scientific Corporation</td>
<td><a href="https://www.kentscientific.com/products/righttemp-jr/">RightTemp</a></td>
</tr>
<tr>
<td>Interface Cable CC-28 with DIN connector</td>
<td>Warner Instruments</td>
<td>Cat. No. 640106</td>
</tr>
<tr>
<td>Resistive heater blocks RH-2</td>
<td>Warner Instruments</td>
<td>Cat. No. 640274</td>
</tr>
<tr>
<td>Temperature Controller TC-344C</td>
<td>Warner Instruments</td>
<td>Order No. 64-2401</td>
</tr>
<tr>
<td>Dumont #7 – Fine Forceps</td>
<td>Fine Science Tools</td>
<td>Cat. No. 11274-20</td>
</tr>
<tr>
<td>Student Vannas Spring Scissors</td>
<td>Fine Science Tools</td>
<td>Cat. No. 91500-09</td>
</tr>
<tr>
<td>Fine Scissors</td>
<td>Fine Science Tools</td>
<td>Cat. No. 14040-10</td>
</tr>
<tr>
<td>Adson Forceps</td>
<td>Fine Science Tools</td>
<td>Cat. No. 11006-12</td>
</tr>
<tr>
<td>S&amp;T Vascular Clamp- 7mm</td>
<td>Fine Science Tools</td>
<td>Cat. No. 00396-01</td>
</tr>
<tr>
<td>S&amp;T Clamp Applying Forceps</td>
<td>Fine Science Tools</td>
<td>Cat. No. 00071-14</td>
</tr>
<tr>
<td>Sutures (Dysilk, cat. No. SA4040)</td>
<td>Dysilk</td>
<td>Cat. No. SA4040</td>
</tr>
<tr>
<td>Surgical Microscope</td>
<td>Leica</td>
<td>Model no. KL1500 LCD</td>
</tr>
<tr>
<td>PE-10 tubing</td>
<td>Becton Dickinson (BD)</td>
<td>Cat. No- BD427400</td>
</tr>
<tr>
<td>3-way stopcock</td>
<td>BD</td>
<td>Cat. No. 04301433200</td>
</tr>
<tr>
<td>P10 Pipette</td>
<td>Rainin</td>
<td>Cat. No. 17008649</td>
</tr>
<tr>
<td>P20 Pipette</td>
<td>Rainin</td>
<td>Cat. No. 17008650</td>
</tr>
<tr>
<td>P200 Pipette</td>
<td>Rainin</td>
<td>Cat. No. 17008652</td>
</tr>
<tr>
<td>P1000 Pipette</td>
<td>Rainin</td>
<td>Cat. No. 17008653</td>
</tr>
<tr>
<td>Pipette gun</td>
<td>Drummond Scientific</td>
<td>Cat. No. 4-000-220</td>
</tr>
<tr>
<td>Kimwipes</td>
<td>Liv-Wipe</td>
<td>Cat. No. LIVWIPRFL</td>
</tr>
<tr>
<td>Kleenex</td>
<td>Kleenex</td>
<td>Cat. No. 4720</td>
</tr>
<tr>
<td>Cover glass - 22 × 22 mm</td>
<td>ProSciTech</td>
<td>Cat. No. G422</td>
</tr>
<tr>
<td>Cover glass - 22 × 50 mm</td>
<td>ProSciTech</td>
<td>Cat. No. G417</td>
</tr>
<tr>
<td>Water bath</td>
<td>Thermomix 1442D</td>
<td>Cat. No. 6003231</td>
</tr>
<tr>
<td>TC25 Tissue Culture Flask</td>
<td>Corning</td>
<td>Cat. No. COR430639</td>
</tr>
<tr>
<td>TC75 Tissue Culture Flask</td>
<td>Corning</td>
<td>Cat. No. COR430641U</td>
</tr>
<tr>
<td>TC150 Tissue Culture Flask</td>
<td>Falcon</td>
<td>Cat. No. FAL355001</td>
</tr>
<tr>
<td>Falcon Tubes (15 mL)</td>
<td>Falcon</td>
<td>Cat. No. 430791</td>
</tr>
<tr>
<td>Falcon Tubes (50 mL)</td>
<td>Falcon</td>
<td>Cat. No. 352070</td>
</tr>
<tr>
<td>Eppendorf tubes</td>
<td>Sarstedt</td>
<td>Cat. No. 72-690-001</td>
</tr>
<tr>
<td>10 μL pipette tips</td>
<td>Interpath Services</td>
<td>Cat. No. 24100</td>
</tr>
<tr>
<td>20 μL pipette tips</td>
<td>Interpath Services</td>
<td>Cat. No. 24500</td>
</tr>
<tr>
<td>200 μL pipette tips</td>
<td>Interpath Services</td>
<td>Cat. No. 24700</td>
</tr>
<tr>
<td>1000 μL pipette tips</td>
<td>Interpath Services</td>
<td>Cat. No. 24800</td>
</tr>
<tr>
<td>1 mL Disposable pipettes</td>
<td>Copan</td>
<td>Cat. No. 201C</td>
</tr>
<tr>
<td>3 mL Disposable pipettes</td>
<td>Copan</td>
<td>Cat. No. 200C</td>
</tr>
<tr>
<td>25G needles</td>
<td>BD</td>
<td>Cat. No. 301807</td>
</tr>
<tr>
<td>1 mL Syringes</td>
<td>BD</td>
<td>Cat. No. 302100</td>
</tr>
<tr>
<td>Surgical tape</td>
<td>Various</td>
<td>N/A</td>
</tr>
<tr>
<td>Surgical Glue</td>
<td>Leukosan</td>
<td>N/A</td>
</tr>
<tr>
<td>Vacuum grease</td>
<td>Dow Corning; Sigma Aldrich</td>
<td>Cat. No. Z273554</td>
</tr>
<tr>
<td>Veet hair removal cream</td>
<td>Reckitt Benckiser</td>
<td>N/A</td>
</tr>
<tr>
<td>Dialysis cassettes</td>
<td>ThermoFischer Scientific</td>
<td>Cat. no. 66807</td>
</tr>
<tr>
<td>0.22 μm Syringe Filter</td>
<td>Sigma Aldrich</td>
<td>Cat. No. SLGV033RS</td>
</tr>
<tr>
<td>0.45 μm Syringe Filter</td>
<td>Sigma Aldrich</td>
<td>Cat. No. SLHV033RS</td>
</tr>
<tr>
<td>100 μm Cell Strainer</td>
<td>Corning</td>
<td>Cat No. 352360</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipments">Materials and equipments</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 6–8 h</div>
<p>Preparation of Evans blue conjugated to Bovine serum albumin (BSA).</p>
<ul>
<li>Prepare the BSA solution by solubilizing 1 gm of BSA in 10 mL of sterile saline. Filter the resultant solution using a 0.22 μm syringe filter.</li>
<li>To conjugate Evans blue to BSA, add 1 mg of Evans blue to 10 mL of 100 mg/mL BSA solution in a 15 mL falcon tube.</li>
<li>Wrap the tube in aluminum foil, and incubate the mixture at 22°C–24°C for 2 h.</li>
<li>Post-incubation, filter the solution using a 0.45 μm syringe filter followed by two rounds of filtration through a 0.22 μm syringe filter.</li>
<li>Place the filtered solution in a dialysis cassette and dialyze against 2 liters of sterile, refrigerated saline for 2–4hrs at 4°C to remove any unconjugated Evans blue.</li>
<li>Transfer the cassette containing the labeled Evans blue solution to a cell culture hood. Using a sterile syringe, aseptically withdraw the Evans blue-BSA solution from the dialysis cassette.</li>
<li>Filter the solution through a 0.22 μm syringe filter, aseptically aliquot into 100 μL aliquots, and store at −20°C.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Wherever possible, all steps should be performed in cell culture hood (BSL-2). Dialysis can be set up in the cell culture hood using sterile refrigerated saline and subsequently moved to a cold room.</div>
<div class="critical">
<span class="critical-title">Critical:</span> As Evans blue-BSA solution is used for intravenous administration during imaging, one should take utmost care to avoid any microbial contaminations. Wherever possible, aseptic methodology should be followed during the preparation of this solution.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A large array of fluorescently labeled dextran dyes are available for use from commercial vendors; see <a href="#tbl1">Table 1</a>.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Animal housing and ethics statements</h3>
<p>All mice were housed in individually ventilated cages maintained at 22°C–26°C, 30%–70% relative humidity, and a 12 h light-dark cycle. Sterilized corncob was used as bedding, with sterilized toilet rolls, tissues, and parts of cardboard boxes being provided as environmental enrichment. All mice used were within the age range of 8–12 weeks. For breast cancer studies, only female mice were used, while mice from both sexes were used for imaging melanoma tumors. All experimental techniques, methodology, and experiments were approved by the relevant ethics committee.</p>
<h3 id="sec3.2">Monitoring mice under anesthesia</h3>
<p>Post-administration of anesthetics, the mice need to be monitored throughout the duration of imaging. For this purpose, physiological monitoring using PhysioSuite (Kent Scientific Corporation) can be used. Alternatively, the depth of anesthesia can also be monitored by checking for hind limb and palpebral reflex. For testing the palpebral reflex, a sterile cotton tip is used to gently touch the periocular skin to generate a reflex. The absence of reflex indicates deep anesthesia. Similarly, the hind limb can also be gently pinched; if no reflex (hindlimb jerk or withdrawal) is observed, the mouse is under anesthesia.</p>
<p>During the anesthetic period, special attention should be paid to the breathing of the mouse, which should neither be too fast nor labored. Breathing under anesthesia should be deep, rhythmic, and distress-free. Moreover, the testing for palpebral and hindlimb reflexes should not affect the breathing rate. Wherever possible, we suggest that researchers utilize the help of more experienced researchers or veterinary staff to ensure that proper monitoring of anesthesia is being performed and the mice are not in any kind of distress.</p>
<h3 id="sec3.3">Preparation prior to surgery for intravital imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>The below-detailed steps allow the researchers to prepare mice for cannulation, surgery, and imaging.</p>
<ol>
<li>Assess the tumor-bearing mouse for any visible signs of distress.</li>
<li>Anesthetize the mouse according to its body weight.
<ol type="a">
<li>We administer injectable anesthesia, ketamine/xylazine (100/10 mg/kg) combined dose to provide deep surgical-plane anesthesia.</li>
<li>Administer repeated half doses upon monitoring during imaging session to enable extended periods of anesthesia. See detailed above.</li>
</ol>
</li>
<li>Once the mouse is completely under anesthesia (as assessed via palpebral reflex and hindlimb withdrawal), carefully remove the abdominal hair using an electric shaver. We remove 80%–90% of hair by shaving until very little hair remains around the poorly accessible regions such as the hindlimbs.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> While using the electric shaver, take care to avoid damaging the skin overlying the tumor to avoid any accidental incision or injury. The shape of the underlying tumor may necessitate multiple attempts to remove the hair.</div>
<ol start="4">
<li>Post-shaving, apply a depilatory cream such as Nair (Church and Dwight) or Veet (Reckitt Benckiser) to the whole abdomen to remove the remaining hair.
<ol type="a">
<li>After about 1–2 min (depending on the depilatory cream), clean the hair removal cream with a wet tissue. Thoroughly clean the skin and tap-dry with dry tissues. We recommend using soft tissues so as to not cause any abrasions on the skin. Lastly, swab the tissue with moist, lint-free paper. This step prevents the transfer of auto-fluorescent hair to the tumor region during surgical preparation.</li>
<li>Post-depilation, quickly place the mouse on its back on a thermal heating pad to maintain its core body temperature.</li>
</ol>
</li>
<li>Next, proceed with cannulation as described below. <i>If no cannulation is required, please proceed directly to the section on</i> <a href="#sec3.4.4">surgical preparation for intravital imaging</a><i>.</i>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not leave the hair removal creams in contact with the skin for too long as this may irritate and inflame the skin. These steps should be carried out as quickly as possible to minimize the reduction in core body temperature. Alternatively, perform the procedure on a temperature-regulated heat pad or 24 h prior to surgery.</div>
<h3 id="sec3.4">Cannulation</h3>
<p>Employ this procedure when repeat injections need to be performed to administer drugs, dyes, or cells. If cannulation is not required, proceed to the surgical preparation for intravital imaging.</p>
<p>For cannulation, we use tail vein, jugular vein, or carotid artery cannulation, depending on the experimental requirements. Both tail vein and jugular vein cannulations transfer the administered agents into the venous circulation and can be used in most scenarios. Carotid artery cannulation is useful for administering cells into the aorta, a methodology utilized when the pulmonary vascular bed needs to be bypassed to avoid sequestration of the adoptively transferred cells in the lungs immediately after administration. As cells are injected into the aorta, they pass through organs (including the tumor mass) before entering the pulmonary vascular bed.</p>
<h4 id="sec3.4.1">Tail vein cannulation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>This procedure can be performed by a researcher with routine mouse-handling experience. It is the method of choice for administering dyes and drugs.</p>
<ol start="6">
<li>Using an artery/toothed-forceps, move the needle of an insulin syringe side-to-side till it breaks from the syringe.
<ol type="a">
<li>Insert the broken/blunt end of the needle into PE10 tubing to create a cannula.</li>
<li>Insert a needle connected to a 3-way stopcock on the other end of the PE10 tubing for administering drugs, dyes, or cells (<a href="#fig4">Figure 4</a>A). Commercially available intravenous cannula can also be used for this purpose.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Cannulation of blood vessels</div>
<p>(A) Illustration of the methodology used to generate the tail vein cannula.</p>
<p>(B) Schematic representation highlighting the methodology for cannulation of tail vein, adapted from The Laboratory Mouse (<a href="#bib51">Shimizu, 2004</a>).</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Anesthetize the mouse (if not previously anesthetized).
<ol type="a">
<li>Disinfect the tail by swabbing with 70% w/v ethanol before placing it on its side.</li>
<li>Warm the tail under a heat lamp for 1–2 min. Lateral veins are the best for cannulation (<a href="#fig4">Figure 4</a>B, Panel 1).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Care should be taken while heating the tail of the mouse. Radiant heat should be applied as per the best working practices within the facility. Wherever possible, the mouse should be covered with paper towels (<a href="https://store.independenceaustralia.com/halyard-teri-wiper-small-100-wipes-small-24230101?gclid=EAIaIQobChMI6tmRzvC-8QIViQVyCh2k_QRtEAQYASABEgK_7PD_BwE">Teri Wipers, Halyard</a>), and only the tail should be exposed to the radiant heat directly. This methodology prevents damage to sensitive tissues such as the eyes and ears. After 1–2 min, researchers can clearly observe the lateral veins within the tail.</div>
<ol start="8">
<li>Once the blood vessel appears to be dilated (increased diameter and redness), perform the following steps:
<ol type="a">
<li>Lift the tail firmly, reorient and position the needle of the cannula on top of the distal end of the tail parallel to the vessel.</li>
<li>Gently tilt the needle to an angle of 10°–15° and insert the bevel (facing up) into the blood vessel (<a href="#fig4">Figure 4</a>B, Panel 2).</li>
<li>Draw the syringe attached to the cannula to see if blood enters the tubing. If yes, then the needle is placed correctly; if not, then repeat the process carefully in another region of the tail.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Hold the needle attached to the cannula with artery/toothed forceps using the dominant hand.</div>
<ol start="9">
<li>Immobilize the needle and tubing on the tail using surgical tape. Flush 50–100 μL of saline to prevent blood coagulation within the needle.
<ol type="a">
<li>If required, inject drugs and cells now via the cannula using the 3-way stopcock.</li>
<li>Take care so that no more than 10% of the blood volume (approximately 100 μL) is injected at any given time. We usually wait 30–40 min between any two intravenous administrations to avoid fluid overload.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In all of our cannulation procedures, we prefer to use a 3-way stopcock to allow for the availability of multiple injection ports for simultaneous injection of dyes and cells. Additionally, this methodology provides surrogate ports in case a port is blocked.</div>
<h4 id="sec3.4.2">Jugular vein cannulation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 20–30 min</div>
<p>This procedure requires expertise and is best suited for researchers with experience in small animal surgery. It is most useful for injecting large number of cells in a relatively small volume (to avoid fluid overload/hypervolemia). It is best used for the adoptive transfer of immune cell populations such as neutrophils, monocytes, and effector T cells.</p>
<ol start="10">
<li>Remove hair from approximately 1–2 cm region of skin on the ventral side of the neck using depilatory creams. Perform all the subsequent steps under a surgical microscope.</li>
<li>Make a superficial skin incision (approximately 1 cm) using a pair of fine-tipped surgical scissors right of the midline of the neck (left to the trachea).
<ol type="a">
<li>Gently tease the skin apart to highlight the underlying tissue (<a href="#fig5">Figure 5</a>A, step1).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Cannulation of blood vessels</div>
<p>(A) Schematic representation depicting the stepwise methodology used for cannulating jugular vein.</p>
<p>(B) Illustration depicting the stepwise methodology used for cannulating carotid artery.</p>
</figcaption>
</figure>
</li>
<li>Using fine forceps, gently tease the underlying fascia and the surrounding tissue whilst avoiding injury. At this point, the jugular vein becomes visible as a large blood vessel, and care should be taken to avoid any injury.</li>
<li>Immobilize the skin tissue using a surgical suture (<a href="#fig5">Figure 5</a>A, step 1).</li>
<li>Tape the other end of the suture onto the heating pad or the metal base plate of the imaging stage.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is crucial at this stage to make sure that the fascia is well separated to provide access to the jugular vein, but precaution should be taken to avoid any injury to the fascia and the surrounding tissue which have deep-seated veins and may lead to bleeding.</div>
<ol start="12">
<li>Gently tease the tissue between the common carotid artery, vagus nerve, and jugular vein.
<ol type="a">
<li>Using ultrafine forceps (preferably Dumont #7 forceps), separate the jugular vein and the vagus nerve from the underlying tissue.</li>
<li>Gently lift the vein using forceps. At this point, no underlying tissue should be displaced, indicating a proper surgical preparation (<a href="#fig5">Figure 5</a>A, step 2).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is crucial that the vagus nerve and the carotid artery are well separated to avoid any nerve damage when the vein is ligated, but it should be done with utmost care to avoid any injury to the vein, which could lead to extensive bleeding.</div>
<ol start="13">
<li>Ligate the anterior region of the vein using the <a href="https://www.animatedknots.com/surgical-tie-knot-instrument-technique">instrument tie methodology</a> (with two opposing throws) using a surgical silk suture (Dynek Dysilk 4/0 suture) (<a href="#fig5">Figure 5</a>A, step 3, Ligation 1).</li>
<li>Clamp the distal end using a microvascular clamp attached to the S&amp;T clamp applying forceps (<a href="#fig5">Figure 5</a>A, step 3). Leave a single suture untied at the distal end.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To ligate the vein, cut appropriately sized sutures (approximately 10 cm in length) and pass under the vein which has been gently lifted by inserting forceps underneath (<a href="#fig5">Figure 5</a>A, step 3). Take care to avoid excessive abrasion with the sutures or forceps to prevent any tear or bleeding.</div>
<ol start="15">
<li>Using a pair of Vannas scissors, make a small incision on the vein. A small amount of blood retained in the lumen of the clamped vein will leak from the incision site (<a href="#fig5">Figure 5</a>A, step 4).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To make a precise incision on the vein, we recommend lifting the vein from the top using Dumont#7 forceps, positioning the Vannas scissors 30° to 45° to the forceps so that just a small part of the vein comes in contact with the scissors. Make a single, sharp cut to avoid excessive damage to the vein.</div>
<ol start="16">
<li>Insert a cannula (without the needle) (PE10®, BD) connected to a 3-way stopcock containing saline into the vein via the incision (<a href="#fig5">Figure 5</a>A, step 5).
<ol type="a">
<li>Immobilize the cannula in the vein by ligating with a surgical suture employing instrument tie methodology with two opposing throws (<a href="#fig5">Figure 5</a>A, step 6, Ligation 2).</li>
<li>If needed, use a second suture at the distal end of the blood vessel to stabilize the cannula further (<a href="#fig5">Figure 5</a>A, step 6, Ligation 3) Troubleshooting <a href="#sec6.5">problem 3</a>.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This part of the procedure requires gentle manipulations and dexterity. The tip of the cannula should be cut with a scalpel blade to provide a sharp 45° bevel. Before inserting the cannula, confirm that the cannula or the connecting stopcock has no air entrapped and the stopcock is in a locked position.</div>
<ol start="17">
<li>Remove the microvascular clamp using the S&amp;T clamp forceps.
<ol type="a">
<li>Trim the excess suture thread.</li>
<li>Placing the skin flap on the underlying tissue, seal the incision site using surgical glue.</li>
<li>The researchers can also attach the cannula onto the skin surface using surgical glue followed by an adhesive bandage. These additional steps prevent the accidental removal of cannula either during the surgical preparation of the tumor or during the transfer of the stage to the microscope.</li>
<li>Flush 50–100 μL of saline to prevent blood coagulation within the cannula.</li>
<li>Inject drugs and cells as required via the cannula using the 3-way stopcock, taking care that no more than 10% of the blood volume is injected at any given time to avoid any changes in the hemodynamic parameters (<a href="#fig5">Figure 5</a>A, step 7).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We usually lightly tape the syringe, 3-way stopcock, and the tubing to the stage or heat pad so as to prevent any accidental damage, bending, or removal. This further helps during the surgical preparation of the tumor as the tubing is securely placed away from the surgical setup (<a href="#fig6">Figure 6</a>).</div>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Surgical preparation of tumors for intravital imaging</div>
<p>Schematic representation of the methodology used to surgically prepare the tumors in mice for intravital imaging. The secure placement of cannula and tubing is also been depicted.</p>
</figcaption>
</figure>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Take brief pauses (&lt;30 sec) between steps 10 to 17 to prepare for the upcoming step, change surgical instruments, and check ligations or cannula.</div>
<h4 id="sec3.4.3">Carotid artery cannulation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 30–45 min</div>
<p>This procedure is best suited for researchers with extensive experience in small animal surgery. It is very useful for injecting large number of cells in experiments wherein the pulmonary vascular bed needs to be bypassed to avoid sequestration of cells immediately post-transfer.</p>
<ol start="18">
<li>Perform steps 10 and 11 as described for jugular vein cannulation (<a href="#fig5">Figure 5</a>B, step 1).</li>
<li>Gently tease the tissue between the trachea and the jugular vein to reveal the presence of the common carotid artery which is easily detected by rhythmic pulsation and the vagus nerve which runs parallel to the common carotid artery.
<ol type="a">
<li>Using ultrafine forceps (preferably Dumont #7 forceps), separate the common carotid artery and the vagus nerve. Isolate the carotid artery from the underlying tissue.</li>
<li>Gently lift the artery using the forceps. At this point, no underlying tissue should be displaced, thus indicating a good surgical preparation (<a href="#fig5">Figure 5</a>B, step 2).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is crucial that the vagus nerve and carotid artery are well separated to avoid any nerve damage when the artery is ligated, but it should be done with utmost care to avoid any injury to the artery, which would lead to excessive bleeding.</div>
<ol start="20">
<li>Ligate the anterior region of the common carotid artery prior to the partition into internal and external carotid arteries using the instrument tie methodology (with two opposing throws) employing a silk surgical suture (Dynek Dysilk 4/0 suture) (<a href="#fig5">Figure 5</a>B, step 3, Ligation 1).</li>
<li>Clamp the distal end using microvascular clamps attached to the S&amp;T clamp applying forceps.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To ligate the artery, cut appropriately sized sutures (approximately 10 cm in length) and pass under the artery which has been gently lifted by inserting forceps underneath. Care should be taken to avoid excessive abrasion with the sutures or forceps that may lead to bleeding.</div>
<ol start="22">
<li>Using Vannas scissors, make a small incision in the carotid artery. Small amounts of blood retained in the lumen of the clamped artery will leak from the incision site, and the artery will collapse (<a href="#fig5">Figure 5</a>B, step 4). Troubleshooting <a href="#sec6.7">problem 4</a>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To make a precise incision on the artery, we recommend lifting the artery from the top using Dumont#7 forceps, positioning the Vannas scissors 45° to the forceps so that just a small part of the artery comes in contact with the scissors. Make a single, sharp cut to avoid excessive damage to the artery.</div>
<ol start="23">
<li>Insert a cannula (without the needle) (PE10®, BD) connected to a 3-way stopcock containing saline into the artery via the incision (<a href="#fig5">Figure 5</a>B, step 5).
<ol type="a">
<li>Immobilize the cannula in the artery by ligation with a surgical suture (<a href="#fig5">Figure 5</a>B, step 6, Ligation 2).</li>
<li>If required, use a second suture at the distal end of the blood vessel to stabilize the cannula further (<a href="#fig5">Figure 5</a>B, step 6, Ligation 3).Troubleshooting <a href="#sec6.9">problem 5</a> and <a href="#sec6.11">6</a>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This part of the procedure requires gentle manipulation and dexterity. We advise pulling the cannula tube at the tip to make it as thin as possible without generating any kinks in the cannula. The tip of the cannula should then be cut with a scalpel blade to provide a sharp 30° to 45° bevel. Before inserting the cannula, confirm that the cannula or the connecting stopcock has no air entrapped and the stopcock is in a locked position.</div>
<ol start="24">
<li>Remove the microvascular clamp using the S&amp;T clamp forceps.
<ol type="a">
<li>Trim the excess suture thread.</li>
<li>Placing the skin flap on the underlying tissue, seal the incision site using a surgical glue.</li>
<li>The researchers can also attach the cannula onto the skin surface using surgical glue followed by an adhesive bandage. These additional steps prevent accidental removal of the cannula during either tumor surgery or transfer of the stage to the microscope.</li>
<li>Flush 50–100 μL of saline to prevent blood coagulation within the cannula.</li>
<li>Inject drugs and cells as required via the cannula using the 3-way stopcock, taking care that no more than 10% of the blood volume is injected at any given time to avoid any changes in the hemodynamic parameters (<a href="#fig5">Figure 5</a>B, step 7).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We usually lightly tape the syringe, 3-way stopcock, and the tubing to the stage or heat pad so as to prevent any accidental damage, bending, or removal. This further helps during the tumor surgery as the tubing is securely placed away from the surgical setup (<a href="#fig6">Figure 6</a>).</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Take brief pauses (&lt;30 sec) between steps 18–24 to prepare for the upcoming step, change surgical instruments, and check the ligations or cannula.</div>
<h4 id="sec3.4.4">Surgical preparation for intravital imaging</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This procedure requires expertise and is best suited for researchers with experience in small animal surgery. These steps allow for access to the underlying mammary or melanoma tumor for intravital imaging.</p>
<ol start="25">
<li>Place the mouse on its back (<a href="#fig6">Figure 6</a>, step 2), on the thermal-controlled surgical board. Gently restrain the limbs using surgical tape to avoid excessive movement during surgery.</li>
<li>Insert a rectal probe into the mouse to continuously monitor the core body temperature using a thermal regulator set at maintaining the core body temperature at 37°C (<a href="#fig6">Figure 6</a>, step 2).</li>
<li>Make a lateral incision from the lower abdomen to the neck area. Exercise appropriate caution so as to not puncture the peritoneum (<a href="#fig6">Figure 6</a>, step 3).</li>
<li>Using blunt dissection, lift the skin from the underlying peritoneal wall (<a href="#fig6">Figure 6</a>, step 3).</li>
<li>Make angled incisions pointing towards the limbs (<a href="#fig6">Figure 6</a>, step 4).</li>
<li>Gently pull the skin back. Using blunt dissection, separate the skin from the underlying peritoneal wall to create a skin flap, thus exposing the tumor.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The tumor is vascularized by newly formed angiogenic blood vessels originating from both cranial and caudal superficial epigastric artery and vein. Take utmost care to avoid injury to these blood vessels.</div>
<ol start="31">
<li>Remove the fascia connecting the skin to the peritoneal wall very carefully and with a lot of precision using the curved edge of the Dumont #7 forceps and a pair of surgical scissors if required (<a href="#fig6">Figure 6</a>, step 4).
<ol type="a">
<li>Gently stretch the ends of the skin flap and immobilize them by suturing (<a href="#fig6">Figure 6</a>, step 5). Cut the ends of the suture and hold in place by surgical tape. It is crucial to use good quality surgical tape to ensure the stability of the skin flap (<a href="#fig6">Figure 6</a>, step 5).</li>
<li>Furthermore, remove any non-vascularized fascia, connective or adipose tissue from the surface of the tumor to improve the quality of imaging.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If the fascia connecting the skin flap to the peritoneal wall is not adequately removed, these connections may result in lateral movement observed during tumor imaging caused due to breathing, with the inhalation and exhalation of the breath changing the tension within the skin flap.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Remove any residual fat (for mammary tumor) or fascia (for melanoma) from the tumor surface as this may hamper the imaging. Gently lift the non-vascularized fascia or adipose tissue and cut from the underlying tumor. The presence of excess fascia, connective tissue, or mammary fat is occasionally observed, especially in smaller tumors wherein the surrounding tissue has not yet been replaced by the tumor mass.</div>
<ol start="32">
<li>Once the tumor has been surgically prepared, apply hydrophobic ointment (such as vacuum grease) to the base of an appropriately sized stage insert and gently place it onto the stage to generate a watertight seal.
<ol type="a">
<li>Superfuse the tumor with saline solution at 37°C.</li>
<li>Place a coverslip on top of the stage insert. Alternatively, image the tumor in the superfused chamber using a water-dipping objective. Troubleshooting <a href="#sec6.13">problem 7</a>
</li>
</ol>
</li>
<li>Cover the exposed tissue with a sterile cotton-gauze sheet. Keep the tissue warm and perfused with saline at 37°C.</li>
<li>To prevent the drying of the animal’s eyes during the imaging session, apply a lubricating ophthalmic ointment before imaging is commenced.</li>
<li>Remove or relax the temporary restraint on the limbs to allow for a normal posture.</li>
<li>Scan the large exposed tumor to find a suitable region for imaging.</li>
<li>While the tumor is being imaged, inject vascular dyes, drugs, or cells to study the real-time dynamics of immune cells <i>in vivo.</i>
</li>
<li>Clean the microscope stage by wiping it down with 70% ethanol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All steps should be carried out using sterile surgical instruments. Sterile, surgical-grade cotton-gauze should be used in this procedure. Consider administration of analgesics during the surgical procedure as per the institutional ethics protocols and experimental design.</div>
<h3 id="sec3.5">Intravital imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–5 h</div>
<p>For operating the intravital microscope, please follow the specific guidelines of the manufacturer. We suggest turning the microscope on, including all the laser components, and detectors to allow sufficient time for the lasers and optics to warm up and stabilize. This should be done before the researcher starts preparing the mouse for surgery.</p>
<ol start="39">
<li>Clean the microscope stage by wiping it down with 70% ethanol.</li>
<li>Following the surgical preparation of the tumor, transfer the imaging stage (<a href="#fig6">Figure 6</a>, step 6) onto the microscope stage.</li>
<li>Once the mouse is placed under the microscope, scan the tissue using the eyepiece for the region of interest.
<ol type="a">
<li>Use the endogenous fluorescence of transgenic immune cells or tumor cells to determine an appropriate region of interest (ROI).</li>
<li>Usually, regions with blood vessels are chosen to study leukocyte trafficking. However, the choice of the region to be imaged depends on the scientific query being pursued.</li>
<li>Take care to image regions that reflect the overall biology of the tissue and to avoid any biases being introduced based on selective imaging.</li>
</ol>
</li>
<li>Once an ROI is selected, switch the imaging to the multiphoton mode.</li>
<li>At this point, imaging is controlled by computer-assisted software. The authors of this article employ IMSpector (LaVision). The imaging is performed as described previously (<a href="#bib25">Li et al., 2012</a>). Troubleshooting <a href="#sec6.15">Problem 8</a>
</li>
<li>Post-completion of the imaging session, euthanize the mice in accordance with the approved ethical guidelines.</li>
<li>Save and export the data.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Using the methodology detailed above, we regularly generate longitudinal imaging datasets as detailed in <a href="#fig7">Figure 7</a>. We have been successfully able to use this methodology for imaging mammary tumors and fatpad (<a href="#bib64">Tikoo et al., 2021a</a>), and melanoma (<a href="#bib56">Tikoo et al., 2021b</a>). We have detailed the most common steps for data processing and analysis below.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<ol>
<li>Store and export the data collected by the IMSpector software (LaVision) as ome.tif files containing the metadata for the imaging sessions.
<ol type="a">
<li>The files can be imported into various image processing software programs such as Image J, Volocity, and Imaris. Depending on the experimental query, we employ both Volocity and Imaris software for data processing and analyses.
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Intravital imaging of tumors</div>
<p>(A) Time lapse intravital imaging of amelanotic melanoma tumors in <i>lyz2</i><sup>gfp/+</sup> mice highlighting the behavior of GFP<sup>+</sup> neutrophils.</p>
<p>(B) Time lapse intravital imaging of mCherry-expressing E0771 mammary tumors in Macblue mice highlighting the intraluminal crawling behavior of CFP<sup>+</sup> leukocytes (pseudo-colored green).</p>
<p>(C) Leukocyte crawling (white tracks) can be observed in the blood vessels.</p>
<p>(D) Extravasation of CFP<sup>+</sup> leukocyte (yellow tracks) can be visualized.</p>
<p>Blood vessels are highlighted by injection of Evans blue-BSA. 00:00:00 represents hr:min:sec. Scale bars have been provided in individual images.</p>
</figcaption>
</figure>
</li>
<li>The caveat of using commercial software is the prohibitive licensing costs associated with them; therefore, non-commercial image analysis software, for example, Image J or FIJI, can also be utilized for post-acquisition image analysis. Nonetheless, the incorporation of a large number of features, ease of performing multiple tasks, and the ability to process multiple datasets provided by commercial software are of immense value to researchers.</li>
<li>To identify the best software for analysis, we often consider the ease of use, list of features, stability of the analysis platform, batch processing capabilities, compatibility with acquired data files from various imaging instruments, ease of employing server-based licenses, and the cost of licensing.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>To highlight some of the most commonly used features, we will utilize Volocity and Imaris software as an example. A detailed list of all the features and capabilities of the software can be found at Volocity and Imaris websites. Outlined below is a brief overview of the processes performed utilizing these software:
<ol type="a">
<li>
<b>Visualizing the data set:</b> Post-acquisition, import the data files into either Volocity or Imaris. Files imported into Volocity and Imaris can be viewed in different modes.
<ol type="i">
<li>While using Volocity, we prefer to use the extended focus mode to visualize the tumor tissue (<a href="#fig7">Figure 7</a>A and <a href="#mmc3">Methods video S1</a>). This projection allows for rapid screening of the imaging data, optimization of the signal intensity for individual channels, and identification of any imaging artefacts; for example, a distortion caused by muscle twitch can be observed at timepoint 00:27:28 (hr:min:sec) in the lower half of the <a href="#mmc3">Methods video S1</a>.</li>
<li>Similarly, in Imaris, under ‘3D view’, images are opened as a maximum intensity projection (MIP) interactive 3D image that can be rotated and zoomed in and out (<a href="#fig7">Figure 7</a>B and Methods <a href="#mmc4">video S2</a>). The ‘Slice’ mode allows images of a z stack to be viewed as individual slices within the z plane, whilst, ‘Section’ mode is chosen to view the xyz dimensions of an image stack. For simplicity, we have outlined a few features of the Imaris software below.
<div class="note">
<span class="note-title">Note:</span> When utilizing automated analysis, it is highly recommended to normalize/negate the effects of transient movements from the final data sets.</div>
<figure id="mmc3"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Mmc3.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Intravital imaging<b>of amelanotic melanoma tumours highlighting the dynamic behaviour of GFP+ leukocytes</b>, related to Quantification and statistical analysis, Step 3 a,i</div>
</figcaption>
</figure>
<figure id="mmc4"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Mmc4.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. <b>Intravital imaging of E0771 tumours highlighting the dynamic behaviour of monocytes and tumour cells</b>, related to Quantification and statistical analysis, Step 3 a,ii</div>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>
<b>Editing the data set:</b> Under the ‘Edit’ menu, various functions are located that can be used to edit the original uploaded data set.
<ol type="i">
<li>Time points, slices, and channels can be added or deleted by choosing the corresponding option and uploading the additional files.</li>
<li>Images can also be cropped to a specific region of interest (ROI) by choosing the ‘Crop 3D’ option and defining the ROI (<a href="#fig7">Figures 7</a>C and 7D).</li>
</ol>
</li>
<li>
<b>Adjusting brightness and contrast:</b> Using the ‘Display Adjustment’ window, the brightness, contrast, and color of each individual channel can be adjusted based on an intensity color scale.</li>
<li>
<b>Correcting drift:</b> Correct the translational drift in the XY plane and rotational drift in relation to stationary reference objects within the same image frame.
<ol type="i">
<li>First, create tracks for the reference objects for example, non- migratory cells or blood vessels using the ‘Spots’ or ‘Surface’ function.</li>
<li>Once tracks are created, select the ‘Correct Drift’ function, found in the ‘Edit Tracks’ tab, to correct the drift of the whole image frame based on the tracks of the stationary objects created.</li>
</ol>
</li>
<li>
<b>3D rendering:</b> Perform the 3D rendering of the objects of interest, such as blood vessels or cells, by creating or reconstructing a surface over the selected objects using the ‘Surface’ function.
<ol type="i">
<li>Create surfaces on objects within a chosen channel by following a series of steps.</li>
<li>Once the final surface is generated, a properties panel appears that can be used to edit various aspects of the surface such as color and transparency and calculate statistical metrics including surface volume, intensity.</li>
</ol>
</li>
<li>
<b>Cell tracking:</b> Using the ‘Spots’ function, track the individual cells and statistically analyze over time.
<ol type="i">
<li>Create a spot on each individual cell within a chosen channel by following the series of steps outlined.</li>
<li>Create tracks from these spots automatically using an algorithm. Alternatively, draw and track individual spots manually for each time frame of the data set (<a href="#fig7">Figures 7</a>B and 7C).</li>
<li>Similar to the ‘Surface’ function, a properties panel enables tracks to be edited and statistically analyzed for parameters such as track distance and velocity.</li>
<li>Export all statistical data as an excel spreadsheet file.</li>
<li>Other than the spot function, researchers can also employ the surface function for tracking cell populations.</li>
<li>The choice of the methodology used for tracking cells relies on the quality of the data, the physiology and behavior of the cell population being tracked, and the experimental query being answered. We recommend utilizing the resources provided within the Learning center, IMARIS.</li>
</ol>
</li>
<li>
<b>Exporting the data set:</b> Once all the image processing is completed, export all data sets from Imaris as an mp4 movie file. Alternatively, a ‘Snapshot’ function is also available to capture a single image at any given time point.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>One of the limitations of this protocol is its surgical invasiveness. Therefore, it is carried out as an end-point approach, thus limiting its potential in repeated longitudinal imaging (<a href="#bib2">Alexander et al., 2008</a>, <a href="#bib18">Ilina et al., 2018</a>). In a scenario wherein longitudinal imaging would be beneficial, tumors can be implanted utilizing the dorsal skin-fold window chamber methodology and imaged longitudinally (<a href="#bib28">Maeda and DaCosta, 2014</a>, <a href="#bib2">Alexander et al., 2008</a>, <a href="#bib4">Biel et al., 2014</a>). Similarly, poor surgical preparation of the tumor might affect the inflammatory response. However, if performed correctly, the surgery causes minimal change to the inflammatory status of the tissue being imaged. Moreover, the protocol allows for reproducible growth and development of tumors within the mammary fat pad and skin. The surgical preparation provides accessibility to a large region of the growing tumor mass, permitting increased flexibility in selecting the regions of interest.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Rapid growth of tumors with minimal vascularization.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>The growth rate of tumor <i>in vivo</i> is determined by the number of cells injected. While injection of 1 million cells results in the development of tumors within two weeks, injection of 0.25–0.5 million cells results in highly vascularized tumors within approximately three weeks. The number of tumor cells injected influences the growth rate of the tumors, the degree of vascularization, infiltration of immune cells, and the localization of immune infiltrate. Therefore, it is prudent that researchers optimize these parameters before proceeding with the intravital imaging studies.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Variable rate of tumor growth within a cohort of injected mice.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Once the cell suspension is aspirated into the syringe, injections should be performed within 30 s. Due to their size and density, tumor cells can readily settle within the syringe leading to inaccurate injections. We prefer to aspirate tumor cell suspensions for a single mouse each time to avoid variability in tumor cell numbers injected between mice. The Eppendorf tube with the tumor cell suspensions is thoroughly mixed before each aspiration. Also, the variability introduced due to the characteristics of the tumor cell line being employed should be taken into consideration when ascertaining the experimental group size.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Difficulty in cannulating the vein.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The ligation should be performed such that it provides a large accessible region for cannulating the blood vessel. Similarly, the clamping should be done as far as possible from the ligation site to provide accessibility to the vein for cannulation. Care should be taken that during clamping, no tissue is caught between the clamps as this will lead to imperfect clamping.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Rapid bleeding after the incision is made on the artery</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Rapid bleeding is probably caused by the improper use of a vascular clamp. To alleviate the issue, apply a second clamp on the artery to stop excessive bleeding. Readjust the first clamp taking care that during clamping, no tissue is caught between the clamps as this will lead to imperfect clamping. Only the artery should be present within the clamp.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Difficulty in cannulating the artery.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The ligation should be performed just before the artery bifurcates to provide accessibility to as much region as possible for cannulation. Similarly, the clamping should be performed as far away as possible from the ligation site to provide accessibility to the artery for cannulation. Care should be taken such that during clamping, no tissue is caught between the clamps as this will lead to imperfect clamping. Only the artery should be present in the clamp.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Inability to insert the cannula.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>To efficiently insert the cannula, make sure that it is well-positioned, preferably 45° to the incision on the artery. It may also be helpful to insert the tip of the forceps into the incision and then insert the cannula in the artery using the lower side of the forceps as a guide. Excessive force will disrupt the artery, resulting in blockage or a tear.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Poor perfusion of the tumor tissue as evidenced by lack of blood flow or non-adherent stationary cells within the vascular lumen.</p>
<h3 id="sec6.14">Potential solution</h3>
<p>To ensure good vascular perfusion, ensure that the notch on the insert aligns with the region of the cranial and caudal superficial epigastric artery and vein. Any compression of this vasculature will lead to poor perfusion of the tumor tissue. Within a well perfused tumor tissue, most of the labeled cells within the vascular lumen can be visualized to rapidly flow through with a small number of cells depicting the behavior of rolling or vascular adhesion (see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Mmc1.zip">Methods videos S1</a> and S2). Please ensure that the insert height is 2–3 mm higher than the tumor being imaged and no overt pressure is being applied onto the tumor by either the insert or the coverslip.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>Tissue drifts observed during the imaging sessions.</p>
<h3 id="sec6.16">Potential solution</h3>
<p>It is not uncommon to observe tissue drifts for the first 20–40 min of the imaging sessions. These drifts are caused by the difference in temperature between the tumor tissue, heated platform, buffer, and the objective. These differences normalize over time and the drifts are stabilized. However, if tissue drifts are observed over extended periods, these may be caused by improper stabilization of the skin flap, fluctuation in stage temperature, or tissue movement due to breathing. To correct for these issues, check that all the sutures are adequately secured, ensure proper connection of electronics and ensure that the fascia between the skin flap has been separated from the peritoneal wall.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Abtin, A., Jain, R., Mitchell, A.J., Roediger, B., Brzoska, A.J., Tikoo, S., Cheng, Q., Ng, L.G., Cavanagh, L.L., Von Andrian, U.H., et al. (2014). Perivascular macrophages mediate neutrophil recruitment during bacterial skin infection. Nat. Immunol. <i>15</i>, 45-53. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref1">View at publisher</a></p>
<p id="bib2">Alexander, S., Koehl, G.E., Hirschberg, M., Geissler, E.K., and Friedl, P. (2008). Dynamic imaging of cancer growth and invasion: a modified skin-fold chamber model. Histochem. Cell Biol. <i>130</i>, 1147-1154. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref2">View at publisher</a></p>
<p id="bib3">Berrozpe, G., Agosti, V., Tucker, C., Blanpain, C., Manova, K., and Besmer, P. (2006). A distant upstream locus control region is critical for expression of the Kit receptor gene in mast cells. Mol. Cell. Biol. <i>26</i>, 5850-5860. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref3">View at publisher</a></p>
<p id="bib4">Biel, N.M., Lee, J.A., Sorg, B.S., and Siemann, D.W. (2014). Limitations of the dorsal skinfold window chamber model in evaluating anti-angiogenic therapy during early phase of angiogenesis. Vasc. Cell <i>6</i>, 17. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref4">View at publisher</a></p>
<p id="bib5">Boissonnas, A., Licata, F., Poupel, L., Jacquelin, S., Fetler, L., Krumeich, S., Théry, C., Amigorena, S., and Combadière, C. (2013). CD8+ tumor-infiltrating T cells are trapped in the tumor-dendritic cell network. Neoplasia <i>15</i>, 85-94. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref5">View at publisher</a></p>
<p id="bib6">Boldajipour, B., Nelson, A., and Krummel, M.F. (2016). Tumor-infiltrating lymphocytes are dynamically desensitized to antigen but are maintained by homeostatic cytokine. JCI Insight <i>1</i>, e89289. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref6">View at publisher</a></p>
<p id="bib7">Breart, B., Lemaître, F., Celli, S., and Bousso, P. (2008). Two-photon imaging of intratumoral CD8+ T cell cytotoxic activity during adoptive T cell therapy in mice. J. Clin. Invest. <i>118</i>, 1390-1397. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref7">View at publisher</a></p>
<p id="bib8">Casanova-Acebes, M., Nicolás-Ávila, J.A., Li, J.L., García-Silva, S., Balachander, A., Rubio-Ponce, A., Weiss, L.A., Adrover, J.M., Burrows, K., A-González, N., et al. (2018). Neutrophils instruct homeostatic and pathological states in naive tissues. J. Exp. Med. <i>215</i>, 2778-2795. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref8">View at publisher</a></p>
<p id="bib9">Cazaux, M., Grandjean, C.L., Lemaître, F., Garcia, Z., Beck, R.J., Milo, I., Postat, J., Beltman, J.B., Cheadle, E.J., and Bousso, P. (2019). Single-cell imaging of CAR T cell activity in vivo reveals extensive functional and anatomical heterogeneity. J. Exp. Med. <i>216</i>, 1038-1049. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref9">View at publisher</a></p>
<p id="bib10">Collins, T.J. (2007). ImageJ for microscopy. Biotechniques <i>43</i>, 25-30. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref10">View at publisher</a></p>
<p id="bib11">De Palma, M., Venneri, M.A., Galli, R., Sergi Sergi, L., Politi, L.S., Sampaolesi, M., and Naldini, L. (2005). Tie2 identifies a hematopoietic lineage of proangiogenic monocytes required for tumor vessel formation and a mesenchymal population of pericyte progenitors. Cancer Cell <i>8</i>, 211-226. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref11">View at publisher</a></p>
<p id="bib12">Fantozzi, A. and Christofori, G. (2006). Mouse models of breast cancer metastasis. Breast Cancer Res. <i>8</i>, 212. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref12">View at publisher</a></p>
<p id="bib13">Faust, N., Varas, F., Kelly, L.M., Heck, S., and Graf, T. (2000). Insertion of enhanced green fluorescent protein into the lysozyme gene creates mice with green fluorescent granulocytes and macrophages. Blood <i>96</i>, 719-726. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref13">View at publisher</a></p>
<p id="bib14">Gray, E.E., Suzuki, K., and Cyster, J.G. (2011). Cutting edge: Identification of a motile IL-17-producing gammadelta T cell population in the dermis. J. Immunol. <i>186</i>, 6091-6095. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref14">View at publisher</a></p>
<p id="bib15">Guo, D., Beaumont, K.A., Sharp, D.M., Lui, G.Y.L., Weninger, W., Haass, N.K., and Tikoo, S. (2020). Abrogation of RAB27A expression transiently affects melanoma cell proliferation. Pigment Cell Melanoma Res. <i>33</i>, 889-894. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref15">View at publisher</a></p>
<p id="bib16">Guo, D., Lui, G.Y.L., Lai, S.L., Wilmott, J.S., Tikoo, S., Jackett, L.A., Quek, C., Brown, D.L., Sharp, D.M., Kwan, R.Y.Q., et al. (2018). RAB27A promotes melanoma cell invasion and metastasis via regulation of pro-invasive exosomes. Int. J. Cancer <i>144</i>, 3070-3085. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref16">View at publisher</a></p>
<p id="bib17">Guy, C.T., Cardiff, R.D., and Muller, W.J. (1992). Induction of mammary tumors by expression of polyomavirus middle T oncogene: a transgenic mouse model for metastatic disease. Mol. Cell. Biol. <i>12</i>, 954-961. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref17">View at publisher</a></p>
<p id="bib18">Ilina, O., Campanello, L., Gritsenko, P.G., Vullings, M., Wang, C., Bult, P., Losert, W., and Friedl, P. (2018). Intravital microscopy of collective invasion plasticity in breast cancer. Dis. Model. Mech. <i>11</i>, dmm034330. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref18">View at publisher</a></p>
<p id="bib19">Jain, R., Tikoo, S., and Weninger, W. (2016). Recent advances in microscopic techniques for visualizing leukocytes in vivo. F1000Research <i>5</i>, 915. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref19">View at publisher</a></p>
<p id="bib20">Jain, R. and Weninger, W. (2013). Shedding light on cutaneous innate immune responses: the intravital microscopy approach. Immunol. Cell Biol. <i>91</i>, 263-270. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref20">View at publisher</a></p>
<p id="bib21">Johnstone, C.N., Smith, Y.E., Cao, Y., Burrows, A.D., Cross, R.S., Ling, X., Redvers, R.P., Doherty, J.P., Eckhardt, B.L., Natoli, A.L., et al. (2015). Functional and molecular characterisation of EO771.LMB tumours, a new C57BL/6-mouse-derived model of spontaneously metastatic mammary cancer. Dis. Model. Mech. <i>8</i>, 237-251. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref21">View at publisher</a></p>
<p id="bib22">Jung, S., Aliberti, J., Graemmel, P., Sunshine, M.J., Kreutzberg, G.W., Sher, A., and Littman, D.R. (2000). Analysis of fractalkine receptor CX(3)CR1 function by targeted deletion and green fluorescent protein reporter gene insertion. Mol. Cell Biol. <i>20</i>, 4106-4114. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref22">View at publisher</a></p>
<p id="bib23">Laviron, M., Combadière, C., and Boissonnas, A. (2019). Tracking Monocytes and Macrophages in Tumors With Live Imaging. Front. Immunol. <i>10</i>, 1201. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref23">View at publisher</a></p>
<p id="bib24">Lee, S.H., Park, S.A., Zou, Y., Seo, S.U., Jun, C.D., Lee, W.J., Hyun, Y.M., and Cho, N.H. (2018). Real-Time Monitoring of Cancer Cells in Live Mouse Bone Marrow. Front. Immunol. <i>9</i>, 1681. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref24">View at publisher</a></p>
<p id="bib25">Li, J.L., Goh, C.C., Keeble, J.L., Qin, J.S., Roediger, B., Jain, R., Wang, Y., Chew, W.K., Weninger, W., and Ng, L.G. (2012). Intravital multiphoton imaging of immune responses in the mouse ear skin. Nat. Protoc. <i>7</i>, 221-234. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref25">View at publisher</a></p>
<p id="bib26">Lindquist, R.L., Shakhar, G., Dudziak, D., Wardemann, H., Eisenreich, T., Dustin, M.L., and Nussenzweig, M.C. (2004). Visualizing dendritic cell networks in vivo. Nat. Immunol. <i>5</i>, 1243-1250. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref26">View at publisher</a></p>
<p id="bib27">Lämmermann, T., Afonso, P.V., Angermann, B.R., Wang, J.M., Kastenmüller, W., Parent, C.A., and Germain, R.N. (2013). Neutrophil swarms require LTB4 and integrins at sites of cell death in vivo. Nature <i>498</i>, 371-375. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref27">View at publisher</a></p>
<p id="bib28">Maeda, A. and DaCosta, R.S. (2014). Optimization of the dorsal skinfold window chamber model and multi-parametric characterization of tumor-associated vasculature. Intravital <i>3</i>, e27935. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref28">View at publisher</a></p>
<p id="bib29">Matsumura, S., Wang, B., Kawashima, N., Braunstein, S., Badura, M., Cameron, T.O., Babb, J.S., Schneider, R.J., Formenti, S.C., Dustin, M.L., et al. (2008). Radiation-induced CXCL16 release by breast cancer cells attracts effector T cells. J. Immunol. <i>181</i>, 3099-3107. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref29">View at publisher</a></p>
<p id="bib30">McDonald, B., Pittman, K., Menezes, G.B., Hirota, S.A., Slaba, I., Waterhouse, C.C., Beck, P.L., Muruve, D.A., and Kubes, P. (2010). Intravascular danger signals guide neutrophils to sites of sterile inflammation. Science <i>330</i>, 362-366. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref30">View at publisher</a></p>
<p id="bib31">Motoike, T., Loughna, S., Perens, E., Roman, B.L., Liao, W., Chau, T.C., Richardson, C.D., Kawate, T., Kuno, J., Weinstein, B.M., et al. (2000). Universal GFP reporter for the study of vascular development. Genesis <i>28</i>, 75-81. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref31">View at publisher</a></p>
<p id="bib32">Mrass, P., Kinjyo, I., Ng, L.G., Reiner, S.L., Puré, E., and Weninger, W. (2008). CD44 mediates successful interstitial navigation by killer T cells and enables efficient antitumor immunity. Immunity <i>29</i>, 971-985. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref32">View at publisher</a></p>
<p id="bib33">Mrass, P., Takano, H., Ng, L.G., Daxini, S., Lasaro, M.O., Iparraguirre, A., Cavanagh, L.L., Von Andrian, U.H., Ertl, H.C., Haydon, P.G., et al. (2006). Random migration precedes stable target cell interactions of tumor-infiltrating T cells. J. Exp. Med. <i>203</i>, 2749-2761. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref33">View at publisher</a></p>
<p id="bib34">Muzumdar, M.D., Tasic, B., Miyamichi, K., Li, L., and Luo, L. (2007). A global double-fluorescent Cre reporter mouse. Genesis <i>45</i>, 593-605. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref34">View at publisher</a></p>
<p id="bib61">Ng, L.G., Hsu, A., Mandell, M.A., Roediger, B., Hoeller, C., Mrass, P., Iparraguirre, A., Cavanagh, L.L., Triccas, J.A., Beverley, S.M., Scott, P., and Weninger, W. (2008). Migratory dermal dendritic cells act as rapid sensors of protozoan parasites. PLoS Pathogen <i>4</i>, e1000222. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref61">View at publisher</a></p>
<p id="bib35">Ng, L.G., Qin, J.S., Roediger, B., Wang, Y., Jain, R., Cavanagh, L.L., Smith, A.L., Jones, C.A., De Veer, M., Grimbaldeston, M.A., et al. (2011). Visualizing the neutrophil response to sterile tissue injury in mouse dermis reveals a three-phase cascade of events. J. Invest. Dermatol. <i>131</i>, 2058-2068. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref35">View at publisher</a></p>
<p id="bib36">Obeidy, P., Ju, L.A., Oehlers, S.H., Zulkhernain, N.S., Lee, Q., Galeano Niño, J.L., Kwan, R.Y., Tikoo, S., Cavanagh, L.L., Mrass, P., et al. (2020). Partial loss of actin nucleator actin-related protein 2/3 activity triggers blebbing in primary T lymphocytes. Immunol. Cell Biol. <i>98</i>, 93-113. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref36">View at publisher</a></p>
<p id="bib37">Overwijk, W.W. and Restifo, N.P. (2001). B16 as a mouse model for human melanoma. Curr. Protoc. Immunol. <i>Chapter 20</i>, Unit 20.1. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref37">View at publisher</a></p>
<p id="bib38">Pai, S., Qin, J., Cavanagh, L., Mitchell, A., El-Assaad, F., Jain, R., Combes, V., Hunt, N.H., Grau, G.E., and Weninger, W. (2014). Real-time imaging reveals the dynamics of leukocyte behaviour during experimental cerebral malaria pathogenesis. PLoS Pathog. <i>10</i>, e1004236. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref38">View at publisher</a></p>
<p id="bib39">Park, S.L., Buzzai, A., Rautela, J., Hor, J.L., Hochheiser, K., Effern, M., Mcbain, N., Wagner, T., Edwards, J., Mcconville, R., et al. (2019). Tissue-resident memory CD8 promote melanoma-immune equilibrium in skin. Nature <i>565</i>, 366-371. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref39">View at publisher</a></p>
<p id="bib40">Patton, E.E., Mueller, K.L., Adams, D.J., Anandasabapathy, N., Aplin, A.E., Bertolotto, C., Bosenberg, M., Ceol, C.J., Burd, C.E., Chi, P., et al. (2021). Melanoma models for the next generation of therapies. Cancer Cell <i>39</i>, 610-631. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref40">View at publisher</a></p>
<p id="bib41">Peters, N.C., Egen, J.G., Secundino, N., Debrabant, A., Kimblin, N., Kamhawi, S., Lawyer, P., Fay, M.P., Germain, R.N., and Sacks, D. (2008). In vivo imaging reveals an essential role for neutrophils in leishmaniasis transmitted by sand flies. Science <i>321</i>, 970-974. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref41">View at publisher</a></p>
<p id="bib42">Prakash, M.D., Munoz, M.A., Jain, R., Tong, P.L., Koskinen, A., Regner, M., Kleifeld, O., Ho, B., Olson, M., Turner, S.J., et al. (2014). Granzyme B promotes cytotoxic lymphocyte transmigration via basement membrane remodeling. Immunity <i>41</i>, 960-972. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref42">View at publisher</a></p>
<p id="bib43">Pucci, F., Venneri, M.A., Biziato, D., Nonis, A., Moi, D., Sica, A., Di Serio, C., Naldini, L., and De Palma, M. (2009). A distinguishing gene signature shared by tumor-infiltrating Tie2-expressing monocytes, blood "resident" monocytes, and embryonic macrophages suggests common functions and developmental relationships. Blood <i>114</i>, 901-914. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref43">View at publisher</a></p>
<p id="bib44">Pérez-Guijarro, E., Day, C.P., Merlino, G., and Zaidi, M.R. (2017). Genetically engineered mouse models of melanoma. Cancer <i>123</i>, 2089-2103. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref44">View at publisher</a></p>
<p id="bib45">Robertson, R.T., Levine, S.T., Haynes, S.M., Gutierrez, P., Baratta, J.L., Tan, Z., and Longmuir, K.J. (2015). Use of labeled tomato lectin for imaging vasculature structures. Histochem. Cell Biol. <i>143</i>, 225-234. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref45">View at publisher</a></p>
<p id="bib69">Roediger, B., Kyle, R., Yip, K.H., Sumaria, N., Guy, T.V., Kim, B.S., Mitchell, A.J., Tay, S.S., Jain, R., Forbes-Blom, E., et al. (2013). Cutaneous immunosurveillance and regulation of inflammation by group 2 innate lymphoid cells. Nat Immunol <i>14</i>, 564-573. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref60">View at publisher</a></p>
<p id="bib46">Rueden, C.T., Schindelin, J., Hiner, M.C., Dezonia, B.E., Walter, A.E., Arena, E.T., and Eliceiri, K.W. (2017). Imagej2: Imagej For The next generation of scientific image data. BMC Bioinformatics <i>18</i>, 529. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref46">View at publisher</a></p>
<p id="bib47">Sasmono, R.T., Oceandy, D., Pollard, J.W., Tong, W., Pavli, P., Wainwright, B.J., Ostrowski, M.C., Himes, S.R., and Hume, D.A. (2003). A macrophage colony-stimulating factor receptor-green fluorescent protein transgene is expressed throughout the mononuclear phagocyte system of the mouse. Blood <i>101</i>, 1155-1163. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref47">View at publisher</a></p>
<p id="bib48">Sauter, K.A., Pridans, C., Sehgal, A., Bain, C.C., Scott, C., Moffat, L., Rojo, R., Stutchfield, B.M., Davies, C.L., Donaldson, D.S., et al. (2014). The MacBlue binary transgene (csf1r-gal4VP16/UAS-ECFP) provides a novel marker for visualisation of subsets of monocytes, macrophages and dendritic cells and responsiveness to CSF1 administration. PLoS One <i>9</i>, e105429. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref48">View at publisher</a></p>
<p id="bib49">Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref49">View at publisher</a></p>
<p id="bib50">Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to ImageJ: 25 years of image analysis. Nat. Methods <i>9</i>, 671-675. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref50">View at publisher</a></p>
<p id="bib51">Shimizu, S. (2004). Chapter 32- Routes of Administration. The Laboratory Mouse, 1st Edition (Elsevier Ltd.). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref51">View at publisher</a></p>
<p id="bib52">Shulman, Z., Cohen, S.J., Roediger, B., Kalchenko, V., Jain, R., Grabovsky, V., Klein, E., Shinder, V., Stoler-Barak, L., Feigelson, S.W., et al. (2011). Transendothelial migration of lymphocytes mediated by intraendothelial vesicle stores rather than by extracellular chemokine depots. Nat Immunol <i>13</i>, 67-76. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref52">View at publisher</a></p>
<p id="bib53">Stoler-Barak, L., Moussion, C., Shezen, E., Hatzav, M., Sixt, M., and Alon, R. (2014). Blood vessels pattern heparan sulfate gradients between their apical and basolateral aspects. PLoS One <i>9</i>, e85699. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref53">View at publisher</a></p>
<p id="bib54">Sumaria, N., Roediger, B., Ng, L.G., Qin, J., Pinto, R., Cavanagh, L.L., Shklovskaya, E., Fazekas De St Groth, B., Triccas, J.A., and Weninger, W. (2011). Cutaneous immunosurveillance by self-renewing dermal gammadelta T cells. J. Exp. Med. <i>208</i>, 505-518. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref54">View at publisher</a></p>
<p id="bib55">Tikoo, S., Barki, N., Jain, R., Zulkhernain, N.S., Buhner, S., Schemann, M., and Weninger, W. (2018). Imaging of mast cells. Immunol. Rev. <i>282</i>, 58-72. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref55">View at publisher</a></p>
<p id="bib64">Tikoo, S., Jain, R., Martinez, B., Song, R., Wielscher, M., Rizzetto, S., Shaw, L.E., Mitchell, A.J., Torres-Pacheco, M.E., Luciani, F., et al. (2021a). A distinct CD115<sup>-</sup> erythro-myeloid precursor present at the maternal-embryonic interface and in the bone marrow of adult mice. bioRxiv. <a class="external-link" href="https://doi.org/10.1101/2021.07.24.453629">https://doi.org/10.1101/2021.07.24.453629</a></p>
<p id="bib56">Tikoo, S., Jain, R., Tomasetig, F., On, K., Martinez, B., Heu, C., Stehle, D., Obeidy, P., Guo, D., Vincent, J.N., et al. (2021b). Amelanotic B16-F10 melanoma compatible with advanced 3-D imaging modalities. J. Invest. Dermatol. <i>141</i>, 2090-2094. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref56">View at publisher</a></p>
<p id="bib57">Tong, P.L., Roediger, B., Kolesnikoff, N., Biro, M., Tay, S.S., Jain, R., Shaw, L.E., Grimbaldeston, M.A., and Weninger, W. (2015). The skin immune atlas: three-dimensional analysis of cutaneous leukocyte subsets by multiphoton microscopy. J. Invest. Dermatol. <i>135</i>, 84-93. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref57">View at publisher</a></p>
<p id="bib60">Unutmaz, D., Xiang, W., Sunshine, M.J., Campbell, J., Butcher, E., and Littman, D.R. (2000). The primate lentiviral receptor Bonzo/STRL33 is coordinately regulated with CCR5 and its expression pattern is conserved between human and mouse. Journal of immunology <i>165</i>, 3284. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/optiKkgz9QGQR">View at publisher</a></p>
<p id="bib58">Vennin, C., Chin, V.T., Warren, S.C., Lucas, M.C., Herrmann, D., Magenau, A., Melenec, P., Walters, S.N., Del Monte-Nieto, G., Conway, J.R., et al. (2017). Transient tissue priming via ROCK inhibition uncouples pancreatic cancer progression, sensitivity to chemotherapy, and metastasis. Sci. Transl. Med. <i>9</i>, eaai8504. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref58">View at publisher</a></p>
<p id="bib70">Wong, C.H., Jenne, C.N., Lee, W.Y., Léger, C., and Kubes, P. (2011). Functional innervation of hepatic iNKT cells is immunosuppressive following stroke. Science <i>334</i>, 101-105. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref70">View at publisher</a></p>
<p id="bib59">You, S., Tu, H., Chaney, E.J., Sun, Y., Zhao, Y., Bower, A.J., Liu, Y.Z., Marjanovic, M., Sinha, S., Pu, Y., and Boppart, S.A. (2018). Intravital imaging by simultaneous label-free autofluorescence-multiharmonic microscopy. Nat. Commun. <i>9</i>, 2125. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00429-9/sref59">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Rohit Jain (<a href="mailto:R.Jain@centenary.org.au">R.Jain@centenary.org.au</a>).</p>
<p>Request for reagents and resources can also be directed to Prof. Wolfgang Weninger (<a href="mailto:W.Weninger@centenary.org.au">W.Weninger@centenary.org.au</a>; <a href="mailto:w.weninger@meduniwien.ac.at">w.weninger@meduniwien.ac.at</a>).</p>
<h4>Materials availability</h4>
<p>The breast cancer and melanoma cell lines are available from the laboratory upon request. Amelanotic B16-F10 melanoma and mCherry expressing E0771 breast adenocarcinoma cell lines require material transfer agreement (MTA), as per institutional guidelines.</p>
<h4>Data and code availability</h4>
<p>The sample imaging dataset utilized in this study can be obtained from the authors upon request. Analysis was performed utilizing commercially available software, Volocity (<a href="https://quorumtechnologies.com/index.php/component/content/category/31-volocity-software">Quorum Technologies</a>) and Imaris (<a href="https://imaris.oxinst.com/">Oxford Instruments</a>).</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Mmc1.zip">Data S1. CAD files for 3D printing stage insert and magnet holders for the stage, related to Step 8, Generation of stage insert</a></p>
<p><a id="mmc2" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/952-Mmc2.zip">Data S2. CAD files for 3D printing stage insert of four different heights, related to Step 8, Generation of stage insert</a></p>
<h3>Acknowledgments</h3>
<p>The authors would like to acknowledge the core flow cytometry and imaging facility at the Centenary Institute of Cancer Medicine and Cell Biology (Drs. Adrian Smith and Kristina Jahn). We would like to acknowledge Sydney cytometry for providing the resources needed for generating the 3-D printed stage inserts and thank Dr. Frank Kao and Mr. Steven Allen for their help in generating 3D printed imaging stage inserts. We would also like to acknowledge various funding bodies especially, <a href="https://doi.org/10.13039/501100000925">National Health and Medical Research Council</a>, PdCCRS research scheme from <a href="https://doi.org/10.13039/501100001111">Cancer Australia</a> (CanToo), and The <a href="https://doi.org/10.13039/501100001774">University of Sydney</a>’s internal funding schemes. Schematics have been created using <a href="https://biorender.com/">BioRender</a>.</p>
<h3>Author contributions</h3>
<p>R.J. and S.T. conceptualized and designed the study, standardized the protocol, and carried out the experiments. K.O. assisted with data analysis. S.D. assisted with designing and standardizing of the 3D inserts. L.C. and B.M. assisted with animal ethics. R.J., S.T., and W.W. supervised the study, wrote, and edited the manuscript. All authors have read and edited the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>