<article>
<section>
<h1 id="header">Measuring <i>in vivo</i> protein turnover and exchange in yeast macromolecular assemblies</h1>
<p><time datetime="2021-09-10">Published: September 10, 2021</time></p>
<p>Zhanna Hakhverdyan,<sup><a href="#aff1">1</a></sup> Kelly R. Molloy,<sup><a href="#aff2">2</a></sup> Roman I. Subbotin,<sup><a href="#aff2">2</a></sup> Javier Fernandez-Martinez,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Brian T. Chait,<sup><a href="#aff1">1</a>,<a href="#cor2">**</a></sup> and Michael P. Rout<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Laboratory of Cellular and Structural Biology, The Rockefeller University, New York, NY 10065, USA</p>
<p id="aff2"><sup>2</sup>Laboratory of Mass Spectrometry and Gaseous Ion Chemistry, The Rockefeller University, New York, NY 10065, USA</p>
<p id="fn1"><sup>3</sup>Present address: Instituto Biofisika (CSIC-UPV/EHU), University of the Basque Country (UPV/EHU), 48940, Leioa, Spain; Ikerbasque, Basque Foundation for Science, 48013 Bilbao, Spain</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:jfernandez@rockefeller.edu">jfernandez@rockefeller.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:chait@rockefeller.edu">chait@rockefeller.edu</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:rout@rockefeller.edu">rout@rockefeller.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100800">10.1016/j.xpro.2021.100800</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We present a comprehensive and robust protocol to track the dynamics of all proteins in a complex in yeast cells. A single member of the protein assembly is tagged and conditionally expressed, minimizing the perturbations to the protein complex. Then, SILAC labeling and affinity purification are used for the assessment of the whole protein complex dynamics. This method can determine and distinguish both subunit turnover and exchange specifically in an assembly to provide a comprehensive picture of assembly dynamics.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref4">Hakhverdyan et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>General proteomics protocol to add a dynamic dimension to macromolecular assemblies</li>
<li>Quantitative MS of SILAC-labeled affinity purified complexes</li>
<li>Can determine both subunit turnover and exchange specifically in an assembly</li>
<li>Provides a comprehensive picture of protein assembly dynamics</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/982-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>We will describe the protein complex dynamics measurement procedure on the yeast nuclear pore complex (NPC). Metabolic labeling with isotopically heavy lysine is employed to track “old” and “new” proteins. The old proteins are labeled with heavy lysine and the new proteins are labeled with light lysine, thus allowing us to measure the fraction of the remaining old protein over time by mass spectrometry. To a first approximation both turnover and exchange are first order reactions. Thus, both processes can be estimated by fitting an exponential model to the data. For the turnover experiments (<a href="#fig1">Figure 1</a>), firstly, the cells are grown in appropriate medium such that they are completely heavy labeled. Next, the cells are switched to a light medium. The cells are then grown for several hours. Each hour a sample is harvested and flash-frozen. The frozen samples are processed and NPCs affinity captured from them through a GFP tag. The purified samples are then analyzed by quantitative MS, which yields a measure of the appearance of “new” protein in the growing cells’ population of NPCs. As diagrammed in <a href="#fig2">Figure 2</a>, the design of the exchange experiments is a straightforward adaptation of that for the turnover experiments described above, with an additional capacity to also track “old” NPCs over time, allowing us to assess if any Nups have been replaced with new ones. For this, we conditionally express the tagged Nup with which we affinity capture the NPCs. During the first, heavy labeling step, the tagged Nup is expressed normally. When cells are then switched to light medium, we stop translation of the tagged Nup from its transcript to ensure extremely rapid cessation of its synthesis. During affinity capture, post-lysis exchange of proteins is blocked by mild chemical crosslinking. The affinity capture step thus generates only mature i.e., “old”, initially heavy-labeled, NPCs; the subsequent appearance of light-labeled Nups in this fraction from cells collected over time must then result from the exchange between soluble Nups and those bound to mature NPCs.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/982-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Diagram of nucleoporin turnover experiment</div>
<p>(A) Initially, the cells are completely heavy labeled. Next, the cells are switched to a light medium. The cells are grown for 5 h. Each hour a sample is harvested and flash-frozen. (B) The frozen samples are disrupted and the NPCs are affinity captured through a GFP tag. The purified samples are analyzed by quantitative MS and the fraction of remaining old protein is plotted against time for each Nup. (C) An exponential curve is fitted to the data to determine the decay rate and half-life.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/982-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Diagram of the nucleoporin exchange experiment</div>
<p>(A) Initially, the cells are completely heavy labeled. Next, the cells are switched to light medium. Simultaneously, the expression of the GFP-tagged Nup is repressed. The cells are grown for 5 h. Each hour a sample is harvested and flash-frozen. (B) The frozen samples are disrupted and the NPCs are affinity captured through the GFP tag, which is still present in the “old” NPCs. The purified protein samples are analyzed by quantitative MS and the remaining fraction of old protein is plotted against time for each Nup. (C) An exponential curve is fit to the data to determine the decay rate and half-life.</p>
</figcaption>
</figure>
<div class="critical"><span class="critical-title">Critical:</span></div>
<ul>
<li>identify the member of the complex with the slowest/negligible dynamics using prior knowledge and literature (Nup157 and Nup84 for the yeast NPC)</li>
<li>Use prior knowledge or experimentally confirm that N-terminal tagging does not disrupt the function of the selected protein</li>
<li>Identify the affinity capture condition that yields all/most proteins in the complex</li>
<li>Pick a yeast background that is lysine auxotrophic or knock out an essential gene in lysine biosynthesis pathway</li>
</ul>
<p>In the first step we prepare the baker’s yeast (<i>Saccharomyces cerevisiae</i>) strain and analytical procedures for measuring protein turnover and exchange rates (<a href="#fig1">Figures 1</a> and <a href="#fig2">2</a>) in a complex that can be affinity captured from lysed cells. While in principle this method should be readily adaptable for other model organisms, we have not done so here. It is imperative that the protein chosen for affinity tagging is stable and has slow dynamics in the protein complex of interest; use prior knowledge and published literature to make the best choice. Our choice of the N-terminal vs C-terminal tagging is based on the conditional expression system: translational repression with a tetracycline-binding aptamer in the 5′-UTR (untranslated region) of the target gene (details of which are provided in (<a href="#bib5">Kotter et al., 2009</a>; <a href="#bib13">Suess et al., 2012</a>)). Note, since the experiments are performed in diploid cells, the tagged protein (repressed during the exchange time course) can be essential, so long as it does not produce haploinsufficiency. Next, it is important to become acquainted with sample preparation, cryomilling and affinity capture techniques. Since protein complexes have their physicochemical idiosyncrasies, the optimal purification conditions have to be determined empirically and it is best to do this on mock/unlabeled samples, since the actual samples labeled with stable isotopes are more expensive to produce. Finally, we have observed best heavy lysine labeling results with a lysine auxotrophic strain after 12–16 h growth. Hence, we recommend either using an auxotrophic strain or knocking out an essential gene in the lysine biosynthetic pathway, e.g., LYS2 / YBR115C.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-HA (Y-11) rabbit polyclonal</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat.# sc-805</td>
</tr>
<tr>
<td>Anti-GFP llama polyclonal IgG</td>
<td>(<a href="#bib2">Fridy et al., 2014</a>)</td>
<td>anti-GFP antibody</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, recombinant proteins, and media</td>
</tr>
<tr>
<td>Chlortetracycline hydrochloride</td>
<td>Sigma-Aldrich</td>
<td>Cat.# 26430</td>
</tr>
<tr>
<td>Calf Intestinal Alkaline Phosphatase (CIP)</td>
<td>New England Biolabs</td>
<td>Cat.# M0290</td>
</tr>
<tr>
<td>cOmplete™, EDTA-free Protease Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat.# 11873580001</td>
</tr>
<tr>
<td>Coomassie Blue R250</td>
<td>MP Biomedicals</td>
<td>Cat.# 190682</td>
</tr>
<tr>
<td>Glutaraldehyde</td>
<td>Sigma-Aldrich</td>
<td>Cat.# 340855</td>
</tr>
<tr>
<td>Trypsin Sequencing Grade, modified</td>
<td>Roche</td>
<td>Cat.# 11418033001</td>
</tr>
<tr>
<td>Iodoacetamide</td>
<td>Sigma-Aldrich</td>
<td>Cat.# I6125-10g</td>
</tr>
<tr>
<td>L-Lysine:2HCl 13C6</td>
<td>Cambridge Isotope Laboratories Inc.</td>
<td>Cat.# CNLM-291-H</td>
</tr>
<tr>
<td>NuPAGE LDS Sample Buffer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# NP0007</td>
</tr>
<tr>
<td>Pepstatin A</td>
<td>Sigma-Aldrich</td>
<td>Cat.# P5318</td>
</tr>
<tr>
<td>Polyvinylpyrrolidone, ave mol. Wt. ~40,000 Da</td>
<td>Sigma-Aldrich</td>
<td>Cat.# PVP40-500G</td>
</tr>
<tr>
<td>Protease Inhibitor Cocktail (PIC)</td>
<td>Sigma-Aldrich</td>
<td>Cat.# P8340</td>
</tr>
<tr>
<td>4-(2-Aminoethyl)benzenesulfonyl fluoride hydrochloride</td>
<td>Sigma-Aldrich</td>
<td>Cat.# A8456</td>
</tr>
<tr>
<td>Ultrapure Salmon Sperm DNA Solution</td>
<td>Invitrogen</td>
<td>Cat.# 15632011</td>
</tr>
<tr>
<td>Dextrose</td>
<td>Fisher Scientific</td>
<td>Cat.# OXLP0071B</td>
</tr>
<tr>
<td>Bacto Agar</td>
<td>Fisher Scientific</td>
<td>DF0140-01-0</td>
</tr>
<tr>
<td>SC (Synthetic complete) amino acid mixtures</td>
<td>Sunrise Science Products</td>
<td>Cat.# #: 1303-030, 1329-030</td>
</tr>
<tr>
<td>Yeast Nitrogen Base (YNB) w/o Amino Acids</td>
<td>Fisher Scientific</td>
<td>Cat.# DF0919-15-3</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Dynabeads M270 Epoxy</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # 143.02D</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Mass spectrometry raw datasets</td>
<td>Zenodo (<a href="https://zenodo.org/">https://zenodo.org/</a> )</td>
<td>DOI #: 10.5281/zenodo.4062150</td>
</tr>
<tr>
<td colspan="3">Experimental model organisms: <i>Saccharomyces cerevisiae</i> strains</td>
</tr>
<tr>
<td>MATa/α his3-Δ200/his3-Δ200 leu2-3,112/leu2-3,112 lys2-801/lys2-801 trp1-1(am)/trp1-1(am) ura3-52ura3-52</td>
<td>(<a href="#bib10">Rout et al., 2000</a>)</td>
<td>DF5</td>
</tr>
<tr>
<td><i>MATa/α ade2-1::ADE2/ade2-1::ADE2 trp1-1/TRP1 LYS2/lys2 ura3-1/ura3-1 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 can1-100/can1-100 myc-LoxP-nup145ΔGLFG/myc-LoxP-nup145ΔGLFG</i></td>
<td>(<a href="#bib11">Strawn et al., 2004</a>)</td>
<td>SWY2870</td>
</tr>
<tr>
<td>DF5, NUP84<i>/pTDH3-tc3-GFP-Nup84::SpHIS5</i>
</td>
<td>This study</td>
<td>Tc3-GFP-Nup84</td>
</tr>
<tr>
<td>DF5, NUP157<i>/pTDH3-tc3-GFP-Nup157::SpHIS5</i>
</td>
<td>This study</td>
<td>Tc3-GFP-Nup157</td>
</tr>
<tr>
<td>DF5, NUP84/<i>pTDH3-tc3-GFP-Nup84::SpHIS5, Nup145N-LoxP-6xHA-Nup145C/Nup145N-LoxP-6xHA-Nup145C</i>
</td>
<td>This study</td>
<td>Nup145NC-fusion</td>
</tr>
<tr>
<td>SWY2870, <i>pTDH3-tc3-GFP-Nup84::SpHIS5, lys2::KanR/lys2</i>
</td>
<td>This study</td>
<td>nup145NΔFG</td>
</tr>
<tr>
<td>DF5, <i>nup120::URA3/nup120::URA3</i> NUP157/ <i>pTDH3-tc3-GFP-Nup157::SpHIS5</i>
</td>
<td>This study</td>
<td>nup120null</td>
</tr>
<tr>
<td>DF5, NUP84/<i>pTDH3-tc3-GFP-Nup84::SpHIS5,</i> NUP170/<i>Nup170-AID::KanR,</i> OsTIR::URA3/<i>ura3-52</i>
</td>
<td>This study</td>
<td>Nup170-AID</td>
</tr>
<tr>
<td>
<i>DF5,</i> pTDH3-tc3-GFP-Nup84::SpHIS5, NUP116/ Nup116-AID::KanR, OsTIR::URA3/<i>ura3-52</i>
</td>
<td>This study</td>
<td>Nup116-AID</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA: List of Plasmids Used in this Study</td>
</tr>
<tr>
<td>TDH3 promoter replacement and N-terminal tagging with tetracycline binding aptamers and 3HA tag, Amp, KanR</td>
<td>Euroscarf</td>
<td>pTDH3-tc3-3xHA</td>
</tr>
<tr>
<td>Gene deletion, Amp, HIS5</td>
<td>Euroscarf</td>
<td>pUG27</td>
</tr>
<tr>
<td>KanR marker in pTDH3-tc3-3xHA replaced with HIS5 in pTDH3-tc3-3xHA, Amp</td>
<td>This study</td>
<td>Ttc3HAHis5</td>
</tr>
<tr>
<td>GFP encoding sequence inserted in Ttc3HAHis5, Amp</td>
<td>This study</td>
<td>Ttc3GFP-His5</td>
</tr>
<tr>
<td>N-terminal or internal 6xHA tagging, Amp, KanR</td>
<td>Euroscarf</td>
<td>pOM10</td>
</tr>
<tr>
<td>Transient, conditional expression of Cre recombinase under GAL1 promoter, Amp, URA3</td>
<td>Euroscarf</td>
<td>pSH47</td>
</tr>
<tr>
<td>Gene deletion, Amp, KlLEU2</td>
<td>Euroscarf</td>
<td>pUG73</td>
</tr>
<tr>
<td>Integratable plasmid, Amp, TRP1</td>
<td>Rout Lab</td>
<td>p404GALL-GFP</td>
</tr>
<tr>
<td>C-terminal AID tagging, KanMX, Amp</td>
<td>Yeast NIG</td>
<td>BYP6740</td>
</tr>
<tr>
<td>osTIR expression, integrative URA3, Amp</td>
<td>Yeast NIG</td>
<td>BYP6744</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>GAAATGAACTATATATCCTATAATCCCTTTGGCGGGACTTGGACTTTCAAAGTCAATCCTTGCAGGTCGACAACCCTTAAT</td>
<td>This study</td>
<td>ZH127</td>
</tr>
<tr>
<td>ACTCAAATCGTCTTCATCTATTTCCGCATCTTCTTCATTGACTAACCCCCAAATGCTAGAGCTAGAAGCGTAATCTGGAAC</td>
<td>This study</td>
<td>ZH128</td>
</tr>
<tr>
<td>CCGCATCAGACAAATCAG</td>
<td>This study</td>
<td>ZH131</td>
</tr>
<tr>
<td>AAACGAGCTCTCGAGAAC</td>
<td>This study</td>
<td>ZH132</td>
</tr>
<tr>
<td>ATGACTAACGAAAAGGTCTGGATAGAGAAGTTGGATAATCCAACTCTTTCAGTGTTACCAGATCCGCAGGCTAACCGGAAC</td>
<td>This study</td>
<td>ZH150</td>
</tr>
<tr>
<td>TTAAGCTGCTGCGGAGCTTCCACGAGCACCAGCACCTGAAGCAACTAGACTTATTTGCGCAGCTCGCTGTGAAGATCCCAG</td>
<td>This study</td>
<td>ZH151</td>
</tr>
<tr>
<td>TATGAAGGATCCGGCCTGGAAGTTCTGTTCCA</td>
<td>This study</td>
<td>ZH90</td>
</tr>
<tr>
<td>GCACCGAAGCTTTTTGTACAATTCATCCATAC</td>
<td>This study</td>
<td>ZH94</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>MaxQuant (version 1.2.2.5)</td>
<td>(<a href="#bib1">Cox and Mann, 2008</a>)</td>
<td><a href="http://www.coxdocs.org/doku.php?id=maxquant:start">http://www.coxdocs.org/doku.php?id=maxquant:start</a></td>
</tr>
<tr>
<td>UCSF Chimera, version 1.13</td>
<td>(<a href="#bib9">Pettersen et al., 2004</a>)</td>
<td><a href="https://www.cgl.ucsf.edu/chimera/">https://www.cgl.ucsf.edu/chimera/</a></td>
</tr>
<tr>
<td>Custom Python scripts</td>
<td>This study</td>
<td><a href="https://github.com/zhakhverdyan/protein-dynamics-pipeline">https://github.com/zhakhverdyan/protein-dynamics-pipeline</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Mass spectrometer</td>
<td>Thermo Fisher Scientific</td>
<td>Orbitrap Fusion</td>
</tr>
<tr>
<td>Mass spectrometer</td>
<td>Thermo Fisher Scientific</td>
<td>Q Exactive Plus</td>
</tr>
<tr>
<td>Liquid chromatograph</td>
<td>Thermo Fisher Scientific</td>
<td>Easy-nLC 1000</td>
</tr>
<tr>
<td>Easy-spray column</td>
<td>Thermo Fisher Scientific</td>
<td>ES800</td>
</tr>
<tr>
<td>NuPAGE 4%–12% Bis-Tris Gel 1.0 mm <b>×</b> 10 well</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# NP0321Box</td>
</tr>
<tr>
<td>NuPAGE 4<b>×</b> LDS Sample Buffer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# NP0007</td>
</tr>
<tr>
<td>Poros R2 reversed phase resin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat.# 1112906</td>
</tr>
<tr>
<td>OMIX Tips</td>
<td>Agilent</td>
<td>Cat.# A57003100</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Endogenous N-terminal tagging of the stable subunit of the protein complex of interest</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 week]</div>
<ol>
<li>Prepare synthetic defined agar plates lacking Histidine (SD-His) from SC-His (synthetic complete lacking Histidine) amino acid mixture following manufacturer's instructions (Sunrize Sciences).</li>
<li>PCR amplify the N-terminal tagging cassette from Ttc3GFP-His5 plasmid with 45–60 bp overhangs homologous to the insertion locus (i.e., right at the ATG start codon), using standard approaches and as described (<a href="#bib4">Hakhverdyan et al., 2021</a>).</li>
</ol>
<p>Ttc3GFP-His5 contains the pTDH3 yeast promoter, 3 tetracycline-binding aptamers in the 5′-UTR and GFP - all parts necessary for conditional expression of the tagged gene of interest in the absence of tetracycline/ chlortetracycline (ClTc) and repression in the presence of the antibiotic (<a href="#bib4">Hakhverdyan et al., 2021</a>; <a href="#bib5">Kotter et al., 2009</a>; <a href="#bib13">Suess et al., 2012</a>).</p>
<ol start="3">
<li>Use the standard <i>S. cerevisiae</i> lithium acetate transformation procedure with the diploid strain of interest.</li>
<li>Plate the transformants on SD-His plates.</li>
<li>Confirm the correct insertion of the cassette as well as the remaining wild type locus by PCR with oligos specific for the insertion locus and tag. See reference (<a href="#bib7">Longtine et al., 1998</a>) for details.</li>
<li>Confirm proper expression and localization of the construct in positive clones by western-blot using antibodies specific against the tag and fluorescence / immunofluorescence microscopy.
<ol type="a">
<li>Confirmation of the functionality of the N-terminal tagged protein by standard yeast genetic methods is advisable.</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Harvesting cells and making noodles</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 days]</div>
<p>Prepare Resuspension Buffer:</p>
<p>1.2% w/v PVP-40 (Polyvinylpyrrolidone, ave mol. Wt. ∼40,000 Da), 20 mM HEPES / KOH pH 7.4 (keep at 4°C for months)</p>
<div class="note">
<span class="note-title">Note:</span> Before using the Resuspension Buffer, add the following solutions to the volume of buffer you intend to use.</div>
<ul>
<li><i>1:100 PIC (protease inhibitors cocktail).</i></li>
<li><i>1:100 Solution P (Dissolve 90 mg 4-(2-aminoethyl)benzenesulfonyl fluoride and 2 mg pepstatin A in 5 mL dry absolute ethanol).</i></li>
<li><i>1:1000 of 1 M DTT (dithiothreitol) in water.</i></li>
</ul>
<p>This protocol is designed to harvest a yeast cell culture and prepare it for milling.</p>
<ol start="7">
<li>Prepare synthetic defined medium lacking Histidine (SD-His) from SC-His (synthetic complete lacking Histidine) amino acid mixture following manufacturer's instructions (Sunrize Sciences).</li>
<li>Grow cell culture in SC-His to at least 3.0<b>×</b>10<sup>7</sup> cell/mL total. Growth is in an orbital shaker / incubator at 200 RPM, 30°C.</li>
<li>Spin cultures down at 4000<b>×</b><i>g</i>, for 10 min, 4°C.</li>
<li>Wash the cell pellet, by resuspending the pellet with 50 mL of ice-cold water. Put the resuspended solution into a 50 mL Falcon tube(s) and spin down at 2600<b>×</b><i>g</i>, for 5 min, at 4°C. Repeat this step once.</li>
<li>Resuspend pellet, with a volume of Resuspension Buffer at 4°C equal to the volume of the pellet (estimated by eye). Spin down at 2600<b>×</b><i>g</i>, for 15 min, 4°C. Aspirate all liquid from the pellet.</li>
<li>Spin down again (just the pellet), at 2600<b>×</b><i>g</i> for 15 min to ensure all of the buffer is removed.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> use an aspirator to remove as much liquid as possible from the pellet as excessive liquid interferes with successful cryomilling (next step).</div>
<ol start="13">
<li>The pellets should be free of excess liquid and resemble a thick paste.</li>
<li>Place liquid nitrogen in a styrofoam container, cover with double ply aluminum foil and place a 50 mL Falcon tube through a hole in the foil ensuring that the bottom of the tube is submerged in the liquid nitrogen. Allow the tube to cool.</li>
</ol>
<p>Caution: exercise care when handling liquid nitrogen, always wear cryo gloves and perform the procedure in a well-ventilated area or a chemical fume hood.</p>
<ol start="15">
<li>Fill the cooled 50 mL Falcon tube, to the very top, with liquid nitrogen.</li>
<li>With a spatula, scoop out the cell paste from step 6 and place into a 10 mL or 20 mL syringe. Squirt the cell paste into the liquid nitrogen in the Falcon tube; this will form “noodles” of frozen cell paste.</li>
<li>When all cell paste is frozen as noodles, decant liquid nitrogen from the tube (try not to lose any noodles; can pour off liquid nitrogen by poking holes into the cap of the Falcon tube, screw on the cap and then turn the tube upside down to pour out the liquid nitrogen).</li>
<li>Do not tighten the tube cap completely, in order to allow liquid nitrogen to evaporate. Store tubes at - 80°C. Close the cap after 24 h.</li>
</ol>
<h3 id="sec2.3">Cryogenic disruption of yeast cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 day]</div>
<p>We use a Retsch PM100 Planetary Ball Mill, with either the 50 mL or 125 mL stainless steel jars. Alternatively, other models such as the Retsch Cryomill or Retsch MM 400 can also be used. For visualization of a very similar protocol, see reference (<a href="#bib6">LaCava et al., 2016</a>).</p>
<ol start="19">
<li>Fill a rectangular ice bucket about 1⁄4 full with liquid nitrogen.</li>
<li>Determine the required volume of the milling jar and number of 20 mm steel grinding beads based on the noodle volume:
<ol type="a">
<li>For 20 mL–50 mL of noodles, use a 125 mL jar, for less than 20 mL of noodles, use a 50 mL jar.</li>
<li>For 50 mL noodles use 7–9 balls, 20–50 mL - 9–11 balls, 15–20 mL - 2–3 balls</li>
</ol>
</li>
<li>Pre-chill everything in the liquid nitrogen: immerse the stainless steel milling jars, the stainless steel lid, the milling balls and the storage tube, with the frozen yeast noodles, in the liquid nitrogen.</li>
<li>Pre-cooling is finished when the nitrogen bath is no longer bubbling vigorously.</li>
<li>Once everything is chilled, tip the appropriate amount of packed noodles into the milling jar. Place the required number of balls inside the jar.</li>
<li>Weigh the milling jar (with noodles and beads) and adjust the counterbalance weight.</li>
<li>Be sure no liquid nitrogen is in the milling jar prior to milling to avoid a buildup of pressure in the jars during milling.</li>
<li>Milling is completed in 8 cycles:
<ol type="a">
<li>when using the 125 mL jar each cycle is set to 400 RPM, 3 min plus 1 min reverse rotation with no breaks between rotations.</li>
<li>when using the 50 mL jar the milling settings are: 500 RPM, 3 min plus 1 min reverse rotation with no breaks between rotations.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> You must hear the balls rattling around in the jar. If there is no rattling then add/remove balls to the jar until you hear it rattle.</div>
<div class="critical">
<span class="critical-title">Critical:</span> A rattle should be audible throughout the grinding cycle, otherwize the beads and powder are stuck and the powder is not being milled.</div>
<ol start="27">
<li>Between each cycle the jars are removed and re-cooled in liquid nitrogen.
<ol type="a">
<li>Do not remove the lid (removal of lid may result in cell material loss).</li>
<li>To ensure the lid is chilled use an empty Falcon tube to pour liquid nitrogen over the top of the milling jar while the bowl of the milling jar cools in the liquid nitrogen bath.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not submerge the jar completely as this will allow liquid nitrogen into the milling bowl and may also result in cell loss.</div>
<ol start="28">
<li>When 8 cycles are complete check inside the jar – if there is powder stuck to the side of the jar repeat 1 milling cycle at 350 RPM, 2 min plus 1 min reverse rotation no breaks between rotations.</li>
<li>Use a pre-chilled metallic spatula to move the milled powder to falcon tubes pre-chilled in liquid nitrogen. Store at −80°C until use.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> the frozen powder can be stored at −80°C for up to a year.</div>
<h3 id="sec2.4">Antibody conjugation of magnetic beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [3 days]</div>
<div class="note">
<span class="note-title">Note:</span> antibody-conjugated magnetic beads can be stored at −20°C for months.</div>
<p>Prepare:</p>
<p>Dynabeads (M270 Epoxy). Keep dry powder at 4°C, stable for months.</p>
<p>0.1M sodium phosphate, pH 7.4. Stable for months at 20°C–24°C.</p>
<p>3M ammonium sulfate in 0.1M sodium phosphate, pH 7.4. Stable for months at 20°C–24°C.</p>
<p>100 mM Glycine pH 2.5. Stable at 4°C for months.</p>
<p>10 mM Tris, pH 8.8. Stable at 20°C–24°C for months.</p>
<p>Fresh 100 mM Triethylamine (56uL Triethylamine in 4 mL water). Prepare immediately before use.</p>
<p>10<b>×</b> PBS (1.37 M NaCl, 27 mM KCl, 100 mM Na<sub>2</sub>HPO<sub>4</sub>, 18 mM KH<sub>2</sub>PO<sub>4</sub>, pH 7.4). Stable at 20°C–24°C for months.</p>
<p>1<b>×</b> PBS. Stable at 20°C–24°C for months.</p>
<p>1<b>×</b> PBS + 0.5% Triton X-100. Stable at 20°C–24°C for months.</p>
<p>50% glycerol, 1x PBS. Stable at 20°C–24°C for months.</p>
<ol start="30">
<li>Measure out 100 mg of Dynabeads (M270 Epoxy).</li>
<li>Add 6 mL 0.1 M sodium phosphate, pH 7.4.</li>
<li>Vortex for 30 s, nutate 10 min, 20°C–24°C. For all subsequent magnetic bead recovery steps, use a manufacturer’s recommended magnet to pellet the beads.</li>
<li>Repeat the sodium phosphate buffer washing step, rock until ready to add antibody solution.</li>
<li>In a round bottom 5 mL falcon tube combine 1<b>×</b> PBS with 1 mg of purified antibody specific for the tag up to 1 mL (in our case, we used either rabbit or llama anti-GFP polyclonal antibodies).</li>
<li>Add 1 mL of 3 M ammonium sulfate in 0.1 M sodium phosphate, pH 7.4, drop wise while gently shaking the tube. Filter the resulting antibody solution through a 0.22 micron filter.</li>
<li>Combine the antibody solution with beads in a 15 mL round bottom f tube, seal the cap with parafilm and incubate in a slowly rotating wheel (or equivalent) at 30°C for 2 days.</li>
</ol>
<ol start="37">
<li>After the incubation, transfer to a 15 mL falcon tube, and perform the following washes using the magnet from step 32:
<div class="critical">
<span class="critical-title">Critical:</span> glycine, tris and triethylamine washes should be performed quickly (no incubation) to avoid excessive denaturation of the conjugated antibodies.</div>
<ol type="a">
<li>4 mL 100 mM Glycine pH 2.5.</li>
<li>4 mL 10 mM Tris, pH 8.8.</li>
<li>4 mL fresh 100 mM Triethylamine.</li>
<li>4 mL 1<b>×</b> PBS and place the tube on a rocker for 5 min, next pellet the beads on the magnet and remove the supernatant</li>
<li>Repeat the wash in 37 d 3 more times.</li>
<li>Finally, incubate the beads with 4 mL 1<b>×</b> PBS + 0.5% Triton X-100 for 15 min on a rocker.</li>
</ol>
</li>
</ol>
<ol start="38">
<li>Remove the final wash and resuspend the beads in 670 uL 50 % glycerol, 1<b>×</b> PBS and store at −20°C. The conjugated beads are stable up to a year</li>
</ol>
<h3 id="sec2.5">Affinity capture optimization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1–5 days]</div>
<div class="note">
<span class="note-title">Note:</span> this analysis assumes the composition of the protein complex is known, e.g. the number of subunits and stoichiometry. The goal of optimizing the extraction condition is to purify the majority of the complex, observed as sharp, roughly stoichiometric bands on a Coomassie-stained protein gel. Please see (<a href="#bib3">Hakhverdyan et al., 2015</a>) for optimal purification profiles.</div>
<div class="critical">
<span class="critical-title">Critical:</span> avoid primary amines in the extraction buffers, as this will interfere with the crosslinking step (crosslinking is applied to labeled samples only, see below).</div>
<div class="critical">
<span class="critical-title">Critical:</span> do not proceed with the protocol unless an optimal extraction buffer is selected.</div>
<p>Prepare a 50 <b>×</b> protease inhibitor cocktail solution by dissolving a complete EDTA-free protease inhibitor tablet (Sigma-Aldrich) in 1 mL of milliQ filtered water. The solution is stable at −20°C for months. Thaw out the stock immediately prior to use. Note, protease inhibitors are only added to the extraction buffer prior to cell powder resuspension and should be omitted for other steps. Additionally, the extraction buffer added to the frozen cell material should be at 20°C–24°C to prevent the buffer from freezing and aid in the rapid resuspension of cell material.</p>
<p>Prepare extraction buffers to test. Once prepared the buffers are stable at 20°C–24°C for several months (without protease inhibitors). Some exemplar extraction buffers are listed as follows, but the exact composition to maximally preserve any given macromolecular assembly must be empirically determined. A full description of empirical parameterization of optimal buffer conditions can be found in (<a href="#bib3">Hakhverdyan et al., 2015</a>):</p>
<ul>
<li>20 mM Na-HEPES (pH 7.4), 0.15–1 M sodium chloride, and 0.1% (v/v) Tween 20.</li>
<li>Between 0.25 M to 1.5 M ammonium acetate (pH close to 7.0) and 1% (v/v) Triton X-100.</li>
<li>250 mM sodium phosphate (try between pH 7 to pH 8) and 1% (v/v) Triton X-100.</li>
<li>40 mM Tris-Cl (pH 8.0), between 50 mM to 250 mM sodium citrate, 150 mM sodium chloride, and 1% (v/v) Triton X-100.</li>
</ul>
<p>Prepare 1<b>×</b> LDS buffer (stable at 20°C–24°C for months), by diluting 4<b>×</b> LDS buffer (Thermo Fisher) 4-fold with Milli Q water.</p>
<p>Pre-chill a benchtop centrifuge to 4°C.</p>
<ol start="39">
<li>Using a liquid nitrogen cold metallic spatula, weigh ∼100 mg of cell powder into liquid nitrogen pre-chilled 1.5 mL tubes.</li>
<li>Remove these tubes containing the ∼100 mg aliquots of yeast cell powder from liquid nitrogen and let them stand with open (or loosened) caps for 30 s (per 100 mg) at 20°C–24°C.</li>
<li>Add four volumes of extraction buffer at 20°C–24°C supplemented with 1<b>×</b>protease inhibitors to the yeast cell powder (e.g., 400 uL extraction buffer + protease inhibitors per 100 mg of powder). Briefly vortex (30 s) to completely resuspend the cell powder. Once the cell material is resuspended the subsequent steps should be performed at 4 ̊C.</li>
<li>Clarify the crude extracts by spinning in a benchtop microcentrifuge at 23,000 <i>g</i> for 7 min at 4 ̊C.</li>
<li>While the cell extracts are in the centrifuge, aliquot the antibody-conjugated magnetic affinity medium into microcentrifuge tubes (5 uL bead slurry per 100 mg of powder).</li>
<li>Pre-equilibrate the beads by washing them three times with 0.5 mL of extraction buffer:
<ol type="a">
<li>Add 500 uL extraction buffer to microcentrifuge tubes, add 5 uL of bead slurry per tube</li>
<li>Use a vortex mixer to completely resuspend the beads</li>
<li>Place on the magnet and wait for the beads to pellet</li>
<li>Use an aspirator to remove the supernatant</li>
<li>Repeat the wash 2 more times, but leave the supernatant for the final wash</li>
<li>Remove the final wash only when ready to proceed (to avoid drying the beads)</li>
</ol>
</li>
<li>Transfer the clarified extracts to the tubes containing the washed magnetic beads (after removing the final wash). Incubate the mixture on a rotating wheel (∼20 RPM) mixer for 30 min at 4 ̊C.</li>
<li>After the batch binding is complete, place the tubes on a magnet to collect the beads and remove the supernatant. Wash the beads three times with 500 uL of extraction buffer as in step 44. During the second wash, transfer the beads to a fresh microcentrifuge tube, and then perform the final wash.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the transfer to a fresh tube reduces the contaminants in the final elution.</div>
<ol start="47">
<li>Remove as much liquid as possible from the beads and elute the protein complexes by incubating the beads with 20 uL of 1<b>×</b> LDS sample buffer (without a reducing agent to prevent IgG leakage) at 70°C. After magnetic removal of the beads, add 50 mM of DTT and incubate the sample at 70°C for 10 more minutes. .</li>
<li>Perform standard SDS-PAGE to evaluate the affinity capture results (<a href="#bib3">Hakhverdyan et al., 2015</a>).</li>
</ol>
<h4 id="sec2.5.1">Exchange and turnover measurement time course</h4>
<div class="timing">
<span class="timing-title">Timing:</span> [2 days]</div>
<div class="note">
<span class="note-title">Note:</span> all yeast growth is at 200 RPM, 30°C in an orbital shaker / incubator.</div>
<p>Prepare stock solutions and growth media:</p>
<ul>
<li>50 mg/mL <sup>13</sup>C<sub>6</sub> L-lysine (K6, 1000<b>×</b>). Dissolve in water, filter-sterilize and freeze aliquots at −20°C until ready for use (stable for months).</li>
<li>50 mg/mL <sup>12</sup>C<sub>6</sub> L-lysine (K0, 1000<b>×</b>). Dissolve in water, filter-sterilize and freeze aliquots at −20°C until ready for use (stable for months).</li>
<li>&gt; 120 mL of 8 mg/mL Chlortetracycline (ClTc) dissolved in water. Filter sterilize and freeze / store aliquots at −20°C (stable for months). Thaw right before use.</li>
<li>11 L of synthetic complete yeast medium, prepare a day prior to experiment :
<ul>
<li>1<b>×</b> SC-His-Lys (synthetic complete lacking histidine and lysine, Sunrise Sciences), autoclave in foil-covered erlenmeyer flasks (or 2 L bottles) the day before the experiment.</li>
<li>10<b>×</b> YNB (yeast nitrogen base, Fisher Scientific) , filter-sterilize and store at 4 ̊C until ready for use (stable for months).</li>
<li>20% Glucose, autoclave and store at 20°C–24°C until ready for use (stable for months).</li>
<li>combine the ingredients to make synthetic dropout medium lacking histidine and lysine (SCD-His-Lys) as follows:
<ul>
<li>1 part 10x YNB (1.1 L), 1 part 20% Glucose (1.1 L), 8 parts 1<b>×</b> SC-His-Lys (8.8 L).</li>
</ul>
</li>
</ul>
</li>
<li>2 L Heavy labeling medium:
<ul>
<li>2 L SCD-His-Lys + 2 mL K6</li>
</ul>
</li>
<li>4.2 L Turnover medium:
<ul>
<li>4.2 L SCD-His-Lys + 4.2 mL K0 + 105 mL sterile water</li>
</ul>
</li>
<li>4.2 L Exchange medium:
<ul>
<li>4.2 L SCD-His-Lys + 4.2 mL K0 + 105 mL 8 mg/mL ClTc</li>
</ul>
</li>
</ul>
<p>Prechill a centrifuge for spinning large cultures to 4°C.</p>
<p>Prepare Resuspension buffer:</p>
<p>1.2% w/v PVP-40 (Polyvinylpyrrolidone, ave mol. Wt. ∼40,000 Da), 20 mM HEPES / KOH pH 7.4 (keep at 4°C for months).</p>
<p>Prepare a 50 <b>×</b> PIC (protease inhibitor cocktail) solution by dissolving a complete EDTA-free protease inhibitor tablet (Sigma-Aldrich) in 1 mL of milliQ filtered water. The solution is stable at −20°C for months. Thaw out the stock immediately prior. Keep frozen until use.</p>
<ol start="49">
<li>Inoculate the tagged yeast strain into 40 mL of Heavy labeling medium and grow 12–16 h.The following morning, inoculate the culture into remaining ∼2 L Heavy labeling medium. Grow until OD<sub>600</sub> = 1.</li>
<li>Spin yeast culture down in 500 mL or 1 L centrifuge bottles for 5 min at 2000<b>×</b> <i>g</i>. Discard the supernatant.</li>
<li>Re-suspend one half of the resulting yeast pellet into 4.2 L of Turnover medium (turnover culture <a href="#fig1">Figure 1</a>). Re-suspend the remaining half into 4.2 L of Exchange medium (exchange culture <a href="#fig2">Figure 2</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The time course starts as soon as the cells are re-suspended.</div>
<ol start="52">
<li>Follow the table below to harvest samples at given time points, e.g., collect 1.5 L of each Turnover and Exchange cultures after 1 h of growth, collect 1 L after 2 h, etc.</li>
</ol>
<table id="undtbl1">
<thead></thead>
<tbody>
<tr>
<td>Time (h)</td>
<td>1</td>
<td>2</td>
<td>3</td>
<td>4</td>
<td>5</td>
</tr>
<tr>
<td>Volume(L)</td>
<td>1.5</td>
<td>1</td>
<td>0.7</td>
<td>0.5</td>
<td>0.3</td>
</tr>
</tbody>
</table>
<p class="table-legend">Note, the presented harvesting times work for cells doubling roughly every 2 h. For slower/faster cells, the growth interval between harvests needs to be adjusted.</p>
<ol start="53">
<li>For each sample perform the (modified) harvesting procedure:
<div class="note">
<span class="note-title">Note:</span> the protocol essentially follows steps 9–18, except it is streamlined and adapted for less cell material.</div>
<ol type="a">
<li>The cell material must be kept cold immediately after it is taken out of the incubator. Harvest the cells at 4000<b>×</b> <i>g</i> for 5 min in a 4°C pre-chilled centrifuge.</li>
<li>No washing steps are necessary. Re-suspend the cell material in 25 mL Resuspension Buffer with 1 <b>×</b> PIC and put in a capped 60 mL syringe. Save the syringe plungers for later use.</li>
<li>Place syringes in 50 mL Falcon tubes and spin at 4000<b>×</b> <i>g</i> for 5 min, 4°C. Discard the supernatant and repeat the spin. Remove the remaining supernatant with an aspirator. Try to remove as much liquid as possible.
<div class="note">
<span class="note-title">Note:</span> the drier the cell paste, the less cell material will stick to the walls of the milling jar, which is important when the cell material is limited.</div>
</li>
<li>Sit labeled falcon tubes in liquid nitrogen (liquid nitrogen filled inside). Remove the caps from syringes and push the cell material into the corresponding tube with the plunger. Carefully remove the liquid nitrogen from the tubes, loosely cap the tubes (to allow nitrogen to escape) and place −80°C. The tubes can be tightened the day after.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> the frozen powder is stable at −80°C for many months</div>
<ol start="54">
<li>Cryogenic disruption of the frozen noodles is performed exactly as in steps 19–29.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> the frozen powder is stable at −80°C for many months</div>
<h3 id="sec2.6">Affinity capture (modified)</h3>
<p>The following steps are minor modifications of the procedure as described in steps 39–48.</p>
<p>Prepare the optimal extraction buffer, determined in steps 39–48. Here, the optimal buffer is supplemented with a low concentration of glutaraldehyde, which introduces substoichiometric stabilizing crosslinks in the complex to inhibit in vitro protein exchange (<a href="#bib12">Subbotin and Chait, 2014</a>).</p>
<p>Prepare a 50 <b>×</b> protease inhibitor cocktail solution by dissolving a complete EDTA-free protease inhibitor tablet (Sigma-Aldrich) in 1 mL of milliQ filtered water. The solution is stable at −20°C for months. Thaw out the stock immediately prior to use. Note, protease inhibitors are only added to the extraction buffer prior to cell powder resuspension and should be omitted for other steps. Additionally, the extraction buffer added to the frozen cell material should be at 20°C–24°C to prevent the buffer from freezing and aid in the rapid resuspension of cell material.</p>
<p>Prepare the optimal extraction buffers.</p>
<p>EM-grade glutaraldehyde 50% w/v solution in water, aliquot, store at −20°C. Discard after 3 freeze-thaw cycles.</p>
<p>Label 2 mL round-bottom tubes.</p>
<p>Prepare ice and liquid nitrogen.</p>
<ol start="55">
<li>Pre-chill 2-mL round bottom tubes in liquid nitrogen.</li>
<li>Weigh out 200 mg of cryomilled powder.</li>
<li>Add 800uL of 20°C–24°C extraction buffer with 1<b>×</b> protease inhibitors and 10 mM EM-grade glutaraldehyde (keep 50% w/v in water, ∼5M aliquoted at −20C).
<ol type="a">
<li>Note, 20°C–24°C buffer aids in rapid powder resuspension.</li>
<li>Glutaraldehyde is very reactive, add immediately prior to use.</li>
</ol>
</li>
<li>Vortex for 30 s, check if the powder is completely resuspended, otherwise, ice for 30 s and repeat the vortexing step. Incubate on ice for 5 min.</li>
<li>Quench the cross linking reaction by adding 100 mM Tris, pH 8.0.</li>
<li>Clarify the lysate by spinning 10 min at &gt;20,000 <i>g</i>, 4°C.</li>
<li>While the lysate is spinning wash 10 uL of the packed antibody conjugated dynabeads with 500uL of extraction buffer 2<b>×</b> as in step 44.</li>
<li>Apply the supernatant to the beads, incubate for 1 h at 4°C with agitation, e.g., orbital shaker or a wheel.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is better to use flat-bottomed 2mL tubes for the binding step, since conical tubes do not permit complete mixing of the sample.</div>
<ol start="63">
<li>Remove the flow-through and wash the beads 2<b>×</b> with 500uL of cold extraction buffer as in step 44 (no glutaraldehyde or protease inhibitors necessary at this stage) .</li>
<li>Transfer the beads to a fresh tube (with 200uL of fresh extraction buffer).</li>
<li>Remove the buffer and incubate the beads with 25uL of 1<b>×</b> NuPage LDS Sample Buffer (10 min, 70°C).</li>
<li>Move the eluate to a fresh tube, add 50 mM (final concentration) DTT, incubate for a further 10 min at 70°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> the eluates should be stored at −20°C until ready for the next step</div>
<h3 id="sec2.7">Sample preparation for mass spectrometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [3 days]</div>
<p>There are numerous procedures for sample cleanup for mass spectrometry. We prefer the gel plug approach as it enables efficient detergent removal in a concentrated sample and a convenient means for handling and digesting the entire mixture of the denatured proteins of interest. Note, starting from step 11, destaining of gel bands, all reagents used must be HPLC/LC-MS grade.</p>
<p>Prepare:</p>
<div class="note">
<span class="note-title">Note:</span> Bicarbonate containing solutions should be stored at 4°C for no longer than a month. Remaining solutions are stable at 20°C–24°C for months, except Iodoacetamide, stored at −20°C.</div>
<p>50 mM Ammonium Bicarbonate in milliQ water.</p>
<p>50 mM ammonium bicarbonate, 50% acetonitrile (v/v).</p>
<p>1M Iodoacetamide.</p>
<p>Destaining solution (16% methanol, 10% acetic acid in milliQ water).</p>
<p>HPLC/LC-MS grade acetonitrile.</p>
<p>HPLC/LC-MS grade water.</p>
<p>5% trifluoroacetic acid stock, sequencing grade.</p>
<p>0.1% trifluoroacetic acid in HPLC grade water.</p>
<p>40% acetonitrile, 0.1% trifluoroacetic acid in HPLC grade water.</p>
<p>50% acetonitrile, 0.1% trifluoroacetic acid in HPLC grade water.</p>
<p>80% acetonitrile, 0.1% trifluoroacetic acid in HPLC grade water.</p>
<ol start="67">
<li>Alkylate the gel sample (above) with 100 mM iodoacetamide for 30 min in the dark at 20°C–24°C.</li>
<li>Load the sample into the well of a NuPage 4%–12% Bis-Tris Gel 1.0 mm <b>×</b> 10 well</li>
<li>Run the gel for 5 min at 200 V, until the sample completely runs into the stacking gel as a tight band</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> time and voltage need to be adjusted depending on the gel system.</div>
<ol start="70">
<li>Place the gel into a fresh container allowing ample movement during agitation.</li>
<li>Wash the gel once for 5 min with milliQ water.</li>
<li>Fix for 5 min with destaining solution.</li>
<li>Wash 2<b>×</b> 5 min with milliQ water.</li>
<li>Stain the gel for 1 h with commercial colloidal Coomassie stain according to manufacturer’s instructions, e.g., Thermo Fisher GelCode™ Blue Safe Protein Stain.</li>
<li>Destain the gel in milliQ water until clear background</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> this step takes (3 h–12 h, frequent water changes expedite the procedure).</div>
<ol start="76">
<li>Excise protein bands from the gel with a disposable razor and put in low binding Eppendorf tubes.</li>
<li>De-stain remaining Coomassie from the resulting gel plug with 500uL 50 mM ammonium bicarbonate, 50% acetonitrile at 37°C with shaking.</li>
<li>Replace the destaining solution every 20–30 min until the coomassie is fully gone (keep the tubes at 4°C if destaining for &gt; 6 h).</li>
<li>Remove destain and add 100uL 100% acetonitrile, vortex 10–20 s and spin down. Pipette out acetonitrile and leave the tubes open to air dry for 5 min (cover with a Kimwipe to prevent dust from falling in) or speedvac to remove residual acetonitrile (30°C, 5 min).</li>
<li>While the tubes are drying prepare the digestion solution: 12.5 ng/uL trypsin in 50 mM ammonium bicarbonate.</li>
<li>Add 30uL of digestion solution on dry gel pieces and place the tube on ice for 30 min −1 h to allow the gel pieces to swell. Check if the gel pieces are sticking out of the liquid, add 10–20uL 50 mM ammonium bicarbonate as necessary to cover the pieces.</li>
<li>Put the tubes at 37°C, check after 1 h, sometimes because of evaporation or further swelling gel pieces stick out, add 50 mM ammonium bicarbonate as necessary. Incubate for 4–12 h..</li>
<li>Transfer the Trypsin digest above gel pieces to a fresh Eppendorf LoBind tube (all extracts will be combined in this tube).</li>
<li>Add 50uL of 0.1% trifluoroacetic acid and incubate at 20°C–24°C on a shaker for 20–30 min. Combine the extract with trypsin digest (from step 17).</li>
<li>Add 40% acetonitrile, 0.1% trifluoroacetic acid and incubate at 20°C–24°C on a shaker for 20–30 min. Combine the extract with trypsin digest (from step 17).</li>
<li>Add 80% acetonitrile, 0.1% trifluoroacetic acid and incubate at 20°C–24°C on a shaker for 20–30 min. Combine the extract with trypsin digest (from step 17).</li>
<li>Add 100% acetonitrile and incubate at 20°C–24°C on a shaker for 20–30 min. Combine the extract with trypsin digest (from step 17).</li>
<li>Evaporate off combined fractions in a speedvac.</li>
<li>Resuspend the peptides in 0.1% trifluoroacetic acid.</li>
<li>Follow manufacturer’s instructions for binding the protein digest to C18 tips.</li>
<li>Wash the tips 4<b>×</b> with 0.1% trifluoroacetic acid.</li>
<li>Elute once 40% acetonitrile, 0.1% trifluoroacetic acid and once with 80% acetonitrile, 0.1% trifluoroacetic acid, combine eluates.</li>
<li>Evaporate off combined fractions in a speedvac.</li>
</ol>
<h3 id="sec2.8">Mass spectrometry procedure</h3>
<p>Before you begin prepare:</p>
<p>5% (v/v) methanol, 0.2% (v/v) formic acid.</p>
<p>Solvent B: 0.1% (v/v) formic acid in acetonitrile.</p>
<p>Solvent A: 0.1% (v/v) formic acid in water.</p>
<div class="note">
<span class="note-title">Note:</span> the solutions are stable at 20°C–24°C for months.</div>
<div class="note">
<span class="note-title">Note:</span> the MS run parameters below have been optimized for Q Exactive Plus and Orbitrap Fusion and would need to be adjusted for a different system.</div>
<ol start="94">
<li>Resuspend peptides in 5% (v/v) methanol, 0.2% (v/v) formic acid.</li>
<li>Load onto an EASY-Spray column (Thermo Fisher Scientific, ES800, 15 cm <b>×</b> 75 mm ID, PepMap C18, 3 μm) via an EASY-nLC 1000 (Thermo Fisher Scientific).</li>
<li>Set the column temperature to 35°C.</li>
<li>Set the flow rate to 300 nl/min.</li>
<li>Elute the peptides with a gradient of 4%–30% solvent B in 35 min, followed by 30%–80% B in 5 min.</li>
<li>Apply a voltage of 1.7–2.5 kV to obtain stable spray of the peptide solution.</li>
<li>Use either a Q Exactive Plus or Orbitrap Fusion (both Thermo Fisher Scientific) or equivalent to perform online mass spectrometric analyses.</li>
</ol>
<ol start="101">
<li>For analyses performed on the Q Exactive Plus:
<ol type="a">
<li>Fragment the top 10 most intense ions in each full scan by higher-energy collisional dissociation.</li>
<li>Detailed parameters of the method are as follows:
<table id="undtbl2">
<thead></thead>
<tbody>
<tr>
<td colspan="2">The parameters for full scans were</td>
</tr>
<tr>
<td>Microscans</td>
<td>1</td>
</tr>
<tr>
<td>Resolution</td>
<td>70,000</td>
</tr>
<tr>
<td>AGC target</td>
<td>3e6</td>
</tr>
<tr>
<td>Maximum injection time</td>
<td>500 msec</td>
</tr>
<tr>
<td>m/z scan range</td>
<td>350–1500</td>
</tr>
<tr>
<td>Spectrum data type</td>
<td>Profile</td>
</tr>
<tr>
<td colspan="2">Parameters relevant to the selection of precursors for MS2</td>
</tr>
<tr>
<td>Exclude charge 1 and unassigned</td>
<td>True</td>
</tr>
<tr>
<td>Peptide match</td>
<td>Preferred</td>
</tr>
<tr>
<td>Exclude isotopes</td>
<td>True</td>
</tr>
<tr>
<td>Dynamic exclusion</td>
<td>Exclude after 1 MS2 scan for 15 s, with mass tolerance 10 ppm</td>
</tr>
<tr>
<td colspan="2">The parameters for MS2 scans were</td>
</tr>
<tr>
<td>Microscans</td>
<td>1</td>
</tr>
<tr>
<td>Resolution</td>
<td>17,500</td>
</tr>
<tr>
<td>AGC target</td>
<td>1e5</td>
</tr>
<tr>
<td>Maximum injection time</td>
<td>100 msec</td>
</tr>
<tr>
<td>Spectrum data type</td>
<td>Centroid</td>
</tr>
<tr>
<td>Normalized collision energy</td>
<td>30%</td>
</tr>
<tr>
<td>Isolation window</td>
<td>2 m/z</td>
</tr>
</tbody>
</table>
</li>
<li>For analyses performed on the Orbitrap Fusion:
<ol type="i">
<li>Perform full scans every 5 s.</li>
<li>As time between full scans allows, fragment ions by collision-induced dissociation in descending intensity order.</li>
<li>Precursors are detected in the Orbitrap and fragments are detected in the ion trap.</li>
</ol>
</li>
</ol>
</li>
</ol>
<p>Detailed parameters of the method are as follows:</p>
<table id="undtbl3">
<thead></thead>
<tbody>
<tr>
<td colspan="2">The parameters for full scans were</td>
</tr>
<tr>
<td>Microscans</td>
<td>1</td>
</tr>
<tr>
<td>Resolution</td>
<td>60,000</td>
</tr>
<tr>
<td>AGC target</td>
<td>2e5</td>
</tr>
<tr>
<td>Maximum injection time</td>
<td>100 msec</td>
</tr>
<tr>
<td>m/z scan range</td>
<td>300–1500</td>
</tr>
<tr>
<td>Use quadrupole isolation</td>
<td>True</td>
</tr>
<tr>
<td>Spectrum data type</td>
<td>Profile</td>
</tr>
<tr>
<td colspan="2">Parameters relevant to the selection of precursors for MS2</td>
</tr>
<tr>
<td>Monoisotopic precursor selection</td>
<td>True</td>
</tr>
<tr>
<td>Intensity threshold</td>
<td>1e3</td>
</tr>
<tr>
<td>Include charge</td>
<td>2–6</td>
</tr>
<tr>
<td>Dynamic exclusion</td>
<td>Exclude after 1 MS2 scan for 15 s, with mass tolerance 10 ppm</td>
</tr>
<tr>
<td colspan="2">The parameters for MS2 scans were</td>
</tr>
<tr>
<td>Microscans</td>
<td>1</td>
</tr>
<tr>
<td>Isolation mode</td>
<td>Quadrupole</td>
</tr>
<tr>
<td>AGC target</td>
<td>7e3</td>
</tr>
<tr>
<td>Maximum injection time</td>
<td>100 msec</td>
</tr>
<tr>
<td>Spectrum data type</td>
<td>Centroid</td>
</tr>
<tr>
<td>Isolation window</td>
<td>2 m/z</td>
</tr>
<tr>
<td>Collision energy</td>
<td>35%</td>
</tr>
<tr>
<td>Activation Q</td>
<td>0.25</td>
</tr>
<tr>
<td>Ion trap scan rate</td>
<td>Rapid</td>
</tr>
</tbody>
</table>
<h3 id="sec2.9">Use of MaxQuant for relative heavy-light evaluation</h3>
<ol start="102">
<li>Search raw data against forward and reversed yeast (or organism of choice) translated ORF sequences, as well as common contaminants.</li>
<li>Use the following parameters:</li>
</ol>
<table id="undtbl4">
<thead></thead>
<tbody>
<tr>
<td>Multiplicity</td>
<td>2</td>
</tr>
<tr>
<td>Maximum labeled amino acids</td>
<td>3</td>
</tr>
<tr>
<td>Fixed modification</td>
<td>carbamidomethylation at cysteine</td>
</tr>
<tr>
<td>Variable modifications</td>
<td>oxidation at methionine, acetylation of protein N-terminus</td>
</tr>
<tr>
<td>Maximum modifications per peptide</td>
<td>5</td>
</tr>
<tr>
<td>Digestion enzyme</td>
<td>Trypsin</td>
</tr>
<tr>
<td>Maximum missed cleavages</td>
<td>2</td>
</tr>
<tr>
<td>Requantify</td>
<td>Enabled</td>
</tr>
<tr>
<td>Match between runs</td>
<td>enabled with match time window 0.7 min and alignment time window 20 min</td>
</tr>
<tr>
<td>Minimum label ratio count for protein quantitation</td>
<td>2</td>
</tr>
<tr>
<td>Peptides for quantitation</td>
<td>unique and razor, including unmodified peptides and peptides with aforementioned modifications</td>
</tr>
<tr>
<td>MS/MS FTMS top peaks per 100 Da</td>
<td>12, with match tolerance 20 ppm</td>
</tr>
<tr>
<td>MS/MS ITMS top peaks per 100 Da</td>
<td>8, with match tolerance 0.5 Da</td>
</tr>
</tbody>
</table>
<ol start="104">
<li>Additionally, make sure that the time point samples for one experiment are sequentially (alphanumerically) labeled, e.g., a1, a2, a3, ....</li>
</ol>
<h3 id="sec2.10">Curve fitting, parameter extraction</h3>
<div class="note">
<span class="note-title">Note:</span> the following procedure is intended as a foundation for the mass spec analysis. Depending on the project needs and the particular protein complex of interest, further regrouping/analysis of data might be necessary. The following analysis is provided as a python script here:</div>
<p><a href="https://github.com/zhakhverdyan/protein-dynamics-pipeline">https://github.com/zhakhverdyan/protein-dynamics-pipeline</a></p>
<ol start="105">
<li>Please follow the instructions in the README.md file for running the script. The final analysis is written in the data_out/Assigned/evidence_poi_grouped.txt file.<br> Following is the description of data cleaning and analysis steps performed by the script:
<ol type="a">
<li>Filter out contaminants, decoys, and peptides with no lysine.</li>
<li>Compute the heavy labeled fraction (HLF) for all remaining peptides using the MaxQuant h/l column:
<p class="math-ml"><math> <mrow> <mi>H</mi> <mi>L</mi> <mi>F</mi> <mo>=</mo> <mspace></mspace> <mfrac> <mi>h</mi> <mrow> <mi>h</mi> <mo>+</mo> <mi>l</mi> </mrow> </mfrac> <mo>=</mo> <mfrac> <mfrac> <mi>h</mi> <mi>l</mi> </mfrac> <mfrac> <mrow> <mi>h</mi> <mo>+</mo> <mi>l</mi> </mrow> <mi>l</mi> </mfrac> </mfrac> <mo>=</mo> <mfrac> <mfrac> <mi>h</mi> <mi>l</mi> </mfrac> <mrow> <mfrac> <mi>h</mi> <mi>l</mi> </mfrac> <mo>+</mo> <mn>1</mn> </mrow> </mfrac> </mrow> </math></p>
</li>
<li>Average the HLF of the peptides for the same protein.</li>
<li>Log-transform the HLF and fit a linear regression model with time as independent variable and HLF for time points as dependent variable. Some parameters and statistics extracted from fitting are:
<ul>
<li>k (slope) - hourly rate of heavy labeled fraction reduction or the “old” protein reduction, which is taken as the apparent rate of protein turnover or exchange depending on the experiment</li>
<li>r - correlation coefficient</li>
<li>p value - for the null hypothesis that the slope is 0</li>
</ul>
</li>
<li>The final analysis file is split the data into two groups: “proteins of interest” and “other”. This is accomplished by searching Saccharomyces Genome Database gene annotations for keywords provided as an argument by the user; all matching proteins are put into “proteins of interest” group, and non-matching proteins into the “other” group.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the “other” category usually corresponds to abundant cytosolic proteins that commonly co-isolate as contaminants in affinity capture experiments (<a href="#bib8">Mellacheruvu et al., 2013</a>). This data is useful as an empirical null distribution. For example, one can use the average and standard deviation of exchange parameters estimated for “other” proteins to conduct a 1-tailed Z-test for the proteins in the “proteins of interest” category, where z score &gt;2 is significantly slower exchange rate than expected by chance.</div>
<h3 id="sec2.11">Parameter interpretation</h3>
<p>For ease of interpretation, convert the slopes (hourly rate of reduction) of fitted curves into half-lives (the time it takes for half the heavy labeled fraction to disappear):</p>
<p>T<sub>half</sub> = −ln(2)/k [h], where k[h<sup>−1</sup>] is the slope from linear fits in k4.</p>
<p>There are 2 regimes for turnover:</p>
<ol>
<li>Fast turnover: T<sub>half</sub> &lt; cell doubling time - regulatory proteins, e.g., cyclines.</li>
<li>Slow turnover: T<sub>half</sub> ≅ cell doubling time - the majority of yeast proteome.</li>
</ol>
<p>We observe 3 regimes for exchange:</p>
<ol>
<li>Fast exchangers: T<sub>half</sub> &lt;= 2.5 h (or ∼ the cell division time), examples are proteins involved in dynamic processes, such as nuclear transport.</li>
<li>Intermediate exchangers: residence time 2.5–7 h, examples are flexible connectors in the nuclear pore complex.</li>
<li>Slow exchangers: residence time &gt; 7 h, structural proteins, e.g., proteins comprising nuclear pore complex scaffold.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Turnover - most proteins in yeast will turn over at the rate of cell division (through new protein synthesis and dilution to daughter cells). A rate faster than cell division implies additional disappearance of the protein through degradation.</p>
<p>Exchange - the residence time of protein complex members will range from 2 h to &gt; 7 h. Please see the limitation for fast exchangers. Longer residence times imply a structural/ scaffold function for a protein.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This method is suitable for studying the dynamics of protein complexes that remain assembled for longer than one cell generation. Consequently, assemblies such as the yeast spindle pole body, that assemble and disassemble with each cell division, cannot be studied with this method. Moreover, since amino acid labeling, old label depletion and free protein pull switching take time, precise exchange rates below cell division rate cannot be determined.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>The cell powder is compacted and stuck to the wall after a grinding cycle (step 28)</p>
<h3 id="sec5.2">Potential solution</h3>
<p>This is usually caused by excessive moisture in the cell paste. Make sure to remove as much liquid as possible from the cell pase before producing the noodles. During grinding dislodge any balls stuck to the walls/powder with a liquid nitrogen cold spatula and repeat 1 milling cycle at 350 RPM, 2 min plus 1 min reverse rotation no breaks between rotations</p>
<h3 id="sec5.3">Problem 2</h3>
<p>After weighing the frozen cell powder into microcentrifuge tubes and taking them out to 20°C–24°C, the caps pop open, resulting in cell material loss. Or when opening the tube some powder is expelled out (steps 39–40, 57).</p>
<h3 id="sec5.4">Potential solution</h3>
<p>This is likely because liquid nitrogen seeps into tubes and expands rapidly when warmed to 20°C–24°C. One simple solution is to simply leave the tubes open after weighing the powder or open the tubes before taking them out to 20°C–24°C.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>During the exchange experiment a light - labeled version of the tagged protein is observed (after tagged protein expression is stopped no light labeled copies should be produced), step 106.</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Shorten the duration of the time course. In our experience after ∼ 5 h the ClTc repression loosens and some light labeled tagged protein is produced. Perform a western analysis on time course samples to determine the best stopping point (when the tagged protein is still repressed).</p>
<h3 id="sec5.7">Problem 4</h3>
<p>All residence times in the exchange experiments are ∼ 2 h (division rate), step 106.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>If there is sufficient evidence to support intermediate/slow dynamics for protein complex members, then the most likely explanation for quick apparent exchange is that the tagged protein itself exchanges rapidly, e.g., if the tag hinders the binding site. Switching the tag to an alternate member will likely solve the issue.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>After processing the data for the exchange experiment, some proteins have unfavorable correlation coefficient (r &lt;&lt; 1) and p value (&gt;0.05), step 106.</p>
<h3 id="sec5.10">Potential solution</h3>
<p>If the bad fits are associated with a large slope magnitude, then more time points or deeper MS scan will help improve the fit and get a more accurate slope. However, if the poor fits are associated with a small slope, this is because the slope is close to 0, hence the null hypothesis that the slope is 0 cannot be rejected. Hence, in the latter case the poor fit is not an issue (and the corresponding protein is a slow exchanger).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cox, J. and Mann, M. (2008). MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol. <i>26</i>, 1367-1372. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref1">View at publisher</a></p>
<p id="bib2">Fridy, P.C., Li, Y., Keegan, S., Thompson, M.K., Nudelman, I., Scheid, J.F., Oeffinger, M., Nussenzweig, M.C., Fenyo, D., Chait, B.T., et al. (2014). A robust pipeline for rapid production of versatile nanobody repertoires. Nat. Methods <i>11</i>, 1253-1260. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref2">View at publisher</a></p>
<p id="bib3">Hakhverdyan, Z., Domanski, M., Hough, L.E., Oroskar, A.A., Oroskar, A.R., Keegan, S., Dilworth, D.J., Molloy, K.R., Sherman, V., Aitchison, J.D., et al. (2015). Rapid, optimized interactomic screening. Nat. Methods <i>12</i>, 553-560. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref3">View at publisher</a></p>
<p id="bib4">Hakhverdyan, Z., Molloy, K.R., Keegan, S., Herricks, T., Lepore, D.M., Munson, M., Subbotin, R.I., Fenyo, D., Aitchison, J.D., Fernandez-Martinez, J., et al. (2021). Dissecting the structural dynamics of the nuclear pore complex. Mol. Cell <i>81</i>, 153-165 e157. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref4">View at publisher</a></p>
<p id="bib5">Kotter, P., Weigand, J.E., Meyer, B., Entian, K.D., and Suess, B. (2009). A fast and efficient translational control system for conditional expression of yeast genes. Nucleic Acids Res. <i>37</i>, e120. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref5">View at publisher</a></p>
<p id="bib6">LaCava, J., Jiang, H., and Rout, M.P. (2016). Protein complex affinity capture from cryomilled mammalian cells. J. Vis. Exp. <i>118</i>, 54518. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref6">View at publisher</a></p>
<p id="bib7">Longtine, M.S., McKenzie, A., Demarini, D.J., Shah, N.G., Wach, A., Brachat, A., Philippsen, P., and Pringle, J.R. (1998). Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae. Yeast <i>14</i>, 953-961. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref7">View at publisher</a></p>
<p id="bib8">Mellacheruvu, D., Wright, Z., Couzens, A.L., Lambert, J.P., St-Denis, N.A., Li, T., Miteva, Y.V., Hauri, S., Sardiu, M.E., Low, T.Y., et al. (2013). The CRAPome: a contaminant repository for affinity purification-mass spectrometry data. Nat. Methods <i>10</i>, 730-736. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref8">View at publisher</a></p>
<p id="bib9">Pettersen, E.F., Goddard, T.D., Huang, C.C., Couch, G.S., Greenblatt, D.M., Meng, E.C., and Ferrin, T.E. (2004). UCSF Chimera--a visualization system for exploratory research and analysis. J. Comput. Chem. <i>25</i>, 1605-1612. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref9">View at publisher</a></p>
<p id="bib10">Rout, M.P., Aitchison, J.D., Suprapto, A., Hjertaas, K., Zhao, Y., and Chait, B.T. (2000). The yeast nuclear pore complex: composition, architecture, and transport mechanism. J. Cell Biol. <i>148</i>, 635-651. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref10">View at publisher</a></p>
<p id="bib11">Strawn, L.A., Shen, T., Shulga, N., Goldfarb, D.S., and Wente, S.R. (2004). Minimal nuclear pore complexes define FG repeat domains essential for transport. Nat. Cell Biol. <i>6</i>, 197-206. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref11">View at publisher</a></p>
<p id="bib12">Subbotin, R.I. and Chait, B.T. (2014). A pipeline for determining protein-protein interactions and proximities in the cellular milieu. Mol. Cell. Proteomics <i>13</i>, 2824-2835. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref12">View at publisher</a></p>
<p id="bib13">Suess, B., Entian, K.D., Kotter, P., and Weigand, J.E. (2012). Aptamer-regulated expression of essential genes in yeast. Methods Mol. Biol. <i>824</i>, 381-391. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00506-2/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact author Michael P. Rout (<a href="mailto:rout@rockefeller.edu">rout@rockefeller.edu</a>).</p>
<h4>Materials availability</h4>
<p>All unique/stable reagents generated in this study are available from the lead contact with a completed Materials Transfer Agreement.</p>
<h4>Data and code availability</h4>
<p>Mass spectrometry raw datasets used in this study are publicly available in Zenodo (<a href="https://zenodo.org/">https://zenodo.org/</a>) under DOI #: 10.5281/zenodo.4062150.</p>
<p>The code is available on github: <a href="https://github.com/zhakhverdyan/protein-dynamics-pipeline">https://github.com/zhakhverdyan/protein-dynamics-pipeline</a></p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grants U54 GM103511 (to B.T.C. and M.P.R.), R01 GM112108 (to M.P.R.), P41 GM109824 (to M.P.R. and B.T.C.), and P41 GM103314 (to B.T.C.).</p>
<h3>Author contributions</h3>
<p>Conceptualization, Z.H., J.F.-M., B.T.C., and M.P.R.; investigation, Z.H., K.R.M., and R.I.S.; formal analysis, Z.H., B.T.C., and M.P.R.; writing, Z.H., J.F.-M., B.T.C., and M.P.R.; funding acquisition, B.T.C. and M.P.R.; supervision, J.F.-M., B.T.C., and M.P.R.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>