<article>
<section>
<h1 id="header">CapZyme-Seq: A 5′-RNA-Seq Method for Differential Detection and Quantitation of NAD-Capped and Uncapped 5′-Triphosphate RNA</h1>
<p><time datetime="2019-12-20">Published: December 20, 2019</time></p>
<p>Irina O. Vvedenskaya<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> and Bryce E. Nickels<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Genetics and Waksman Institute, Rutgers University, Piscataway, NJ 08854, USA</p>
<p id="fn1"><sup>2</sup>Lead Contact</p>
<p id="fn2"><sup>3</sup>Technical Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:irinavv@waksman.rutgers.edu">irinavv@waksman.rutgers.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:bnickels@waksman.rutgers.edu">bnickels@waksman.rutgers.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2019.100002">10.1016/j.xpro.2019.100002</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Nucleoside-containing metabolites such as the oxidized and reduced forms of nicotinamide adenine dinucleotide (NAD<sup>+</sup> and NADH), 3′-desphospho-coenzyme A (dpCoA), and flavin adenine dinucleotide (FAD) can be incorporated as RNA 5′ end caps by serving as non-canonical initiating nucleotides (NCINs) for transcription initiation by RNA polymerase. We recently reported “CapZyme-seq,” a 5′-RNA-seq method that enables the differential detection and quantitation of relative yields of NCIN-capped RNA and uncapped 5′-triphosphate RNA. Here we provide the protocol for constructing cDNA libraries for CapZyme-seq.</p>
<p>For complete information on the generation and use of this protocol, please refer to <a href="#bib8">Vvedenskaya et al. (2018a)</a>.</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for high-throughput detection and quantitation of NAD<sup>+</sup>-capped RNA</li>
<li>Method enables analysis of capping for RNA generated both <i>in vitro</i> and <i>in vitro</i>
</li>
<li>Analysis of relative efficiency of NAD<sup>+</sup>- versus NTP-mediated transcription initiation</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/16-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<div class="critical">
<span class="critical-title">Critical:</span> Use nuclease-free H<sub>2</sub>O at all steps whenever H<sub>2</sub>O is needed.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Perform all steps in a nuclease-free environment.</div>
<h3>Source of Input RNA</h3>
<p>Libraries can be prepared using RNA generated <i>in vitro</i> (part A) or <i>in vivo</i> (part B) as input.</p>
<h3>A: Isolation of RNA Products Generated <i>In Vitro</i>
</h3>
<p>For CapZyme-seq analysis of RNA products generated <i>in vitro</i>, the experimenter controls the identity of the NCIN added to transcription assays, and thus controls the identity of the NCIN-capped RNA products used as input. Transcription assays are performed in the presence and absence of an NCIN; results obtained for assays performed in the absence of an NCIN are used as a negative control, or background correction (<a href="#bib8">Vvedenskaya et al., 2018a</a>). We recommend performing transcription assays in triplicate. If necessary, <i>E. coli</i> RNA polymerase can be replaced by another RNAP (e.g., T7 RNAP).</p>
<h4>Transcription Assay</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h</div>
<ol>
<li>Mix reagents for <i>in vitro</i> transcription assays as indicated below.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>Volume, μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>100 nM template DNA</td>
<td>10</td>
</tr>
<tr>
<td>500 nM RNAP holoenzyme</td>
<td>10</td>
</tr>
<tr>
<td>500 mM Tris-HCl pH 8.0</td>
<td>8</td>
</tr>
<tr>
<td>1M MgCl<sub>2</sub>
</td>
<td>1</td>
</tr>
<tr>
<td>0.1 mg/mL BSA</td>
<td>10</td>
</tr>
<tr>
<td>1M KCl</td>
<td>10</td>
</tr>
<tr>
<td>25% glycerol</td>
<td>20</td>
</tr>
<tr>
<td>100 mM DTT</td>
<td>10</td>
</tr>
<tr>
<td>RNaseOUT, 40 U/μL</td>
<td>1</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>80</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Incubate the mixture at 37°C for 10 min to form RNAP-promoter open complexes.</li>
<li>Prepare NTP mixtures with or without an NCIN (e.g., NAD<sup>+</sup>) as indicated below and incubate at 37°C for 10 min.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>Assay with NTPs and NAD<sup>+</sup> Volume, μL</th>
<th>Assay with NTPs Only Volume, μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>ATP, 5 mM</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>CTP, 5 mM</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>GTP, 5 mM</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>UTP, 5 mM</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>NAD<sup>+</sup>, 100 mM</td>
<td>2</td>
<td>-</td>
</tr>
<tr>
<td>Heparin, 10 μg/μL</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>500 mM Tris-HCl pH 8.0</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>7</td>
<td>9</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>20</td>
<td>20</td>
</tr>
</tbody>
</table>
<ol start="4">
<li>Initiate a single round of transcription by adding an NTP mixture (prepared in 3) to the solution containing RNAP-promoter open complexes (prepared in 2). Incubate at 37°C for 15 min and stop the reactions by adding 0.5 M EDTA (pH 8) to a final concentration of 50 mM.</li>
<li>Add 10 μL of 3 M NaOAc, mix by vortexing for 30 s, add 330 μL of 100% EtOH, and precipitate nucleic acids at −80°C for 16 h.</li>
<li>Recover the nucleic acids by centrifugation for 30 min at 21,000 x <i>g</i> at 4°C, wash pellets 3 times with cold 80% EtOH, resuspend in 30 μL of nuclease-free water, and add 30 μL of 2x RNA loading dye.</li>
</ol>
<h4>Isolation of RNA Products (<a href="#fig1">Figure 1</a>)</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼4 h</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/16-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Isolation of Nucleic Acids by Gel Extraction</div>
</figcaption>
</figure>
<ol>
<li>Load sample (from step 6 above) on a 10% TBE-urea slab gel along with a ssRNA size ladder. Run gel in 1x TBE buffer until bromophenol blue dye front reaches ∼3/4 of the length of the gel. Samples for distinct transcription reactions should be run using separate gels.</li>
<li>Incubate gels with SYBR Gold nucleic acid gel stain and visualize bands with UV transillumination. Excise RNA products of the expected size from the gel using a scalpel. Gels for distinct transcription reactions should be stained in separate containers.</li>
<li>Assemble two tubes to crush the gel pieces. Pierce one to three holes in the bottom of a 0.5 mL low binding microfuge tube with a 23-gauge needle. Place this tube into a 1.7 mL low binding microfuge tube. Place gel slices into the upper tube and centrifuge the tube-in-tube assembly at 14000 <i>x g</i> for 2 min or until all gel fragments have passed through the holes in the upper 0.5 mL tube into the lower 1.7 mL tube.</li>
<li>Add 400 μL of gel elution buffer (0.3 M NaCl, 10 mM Tris-HCl pH 8, 0.1 mM EDTA) to the crushed gel and incubate the slurry at 70°C for 10 min. Mix by vortexing for 30 s, transfer the slurry to a Spin-X tube filter, centrifuge at 14,000 <i>x g</i> for 1 min, and transfer the eluate into a 1.7 mL tube.</li>
<li>Remove the crushed gel from the Spin-X tube, mix with 400 μL of gel elution buffer, and repeat steps described in step 4 above.</li>
<li>Combine the eluates from steps 4 and 5, add 80 μL of 3M NaOAc and 8.8 μL of glycogen (10 mg/mL), mix by vortexing for 30 s, add 650 μL of isopropanol, mix by vortexing for 30 s, and precipitate RNA at −80°C for 16 h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> RNA can be stored at −80°C for ∼6 months.</div>
<ol start="7">
<li>Centrifuge tubes at 4°C for 30 min at 21,000 x <i>g</i> to pellet RNA. Wash pellets 3 times with 1 mL of 80% ice-cold EtOH.</li>
<li>Allow the pellet to air-dry for ∼5 min. Resuspend RNA in 10 μL of nuclease-free water. Pipette the suspension and incubate on ice for at least 20 min to fully resuspend the RNA.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of RNA generated in four 100 μL reactions is sufficient for use in two separate enzymatic treatments (Rpp and NudC or Rai1) and two distinct control reactions (mock Rpp treatment and mock NudC or mock Rai1 treatment; <a href="#bib8">Vvedenskaya et al., 2018a</a>).</div>
<h3>B. Isolation of RNA Products Generated <i>In Vivo</i> (in <i>E. coli</i>)</h3>
<p>We recommend use of TRI Reagent solution to isolate RNA from <i>E. coli</i> cultures as described in <a href="#bib9">Vvedenskaya et al. (2018b)</a>.</p>
<h4>Cell Culture and Harvest</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼2 days</div>
<ol>
<li>Place 5 mL of LB media in a 15 mL culture tube and inoculate with <i>E. coli</i>. Grow culture overnight at 37°C on a roller drum set to 210 RPM.</li>
<li>Add 0.5 mL of the overnight culture to 50 mL of LB media in a 250 mL DeLong flask. Grow culture at 37°C on a platform shaker at 210 RPM to mid-exponential phase (OD600 ∼0.5).</li>
<li>Harvest 2 mL of cell suspension into 2-mL tubes and collect cells by centrifugation for 1 min at 21,000 x <i>g</i> at 25°C. Remove supernatant and freeze cell pellets rapidly on dry ice. Store cell pellets at −80°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Cell pellets can be stored at −80°C for ∼6 months.</div>
<div class="note">
<span class="note-title">Note:</span> Use one cell pellet for RNA isolation, store the remaining pellets at −80°C as a back up.</div>
<h4>RNA Isolation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h</div>
<ol>
<li>Resuspend cell pellet in 600 μL of TRI Reagent solution. Incubate at 70°C for 10 min and centrifuge for 10 min at 21,000 x <i>g</i> at 4°C to remove insoluble material. Transfer supernatant to a fresh tube.</li>
<li>Add 100% EtOH to a final concentration of 60.5% and divide the mixture equally between two Direct-zol spin columns (Zymo Research). Perform DNase I treatment on-column according to the manufacturer’s recommendations.</li>
<li>Elute RNA from the column using 30 μL of nuclease-free water heated to 70°C. Repeat the elution two more times. Pool the eluates.</li>
<li>Add 10 μL of TURBO DNase 10x buffer and 1 μL of TURBO DNase to the recovered RNA. Incubate the mixture at 37°C for 30 min to remove residual DNA.</li>
<li>Purify RNA using acid phenol:chloroform pH 4.5.</li>
</ol>
<ol type="a">
<li>Add nuclease-free H<sub>2</sub>O to TURBO DNase-treated RNA to a total volume of 200 μL, mix with 200 μL of acid phenol/chloroform pH 4.5. Mix by vortexing for 10 s.</li>
<li>Centrifuge for 30 s at 21,000 x <i>g</i> at 25°C and recover the upper aqueous phase containing RNA.</li>
<li>Add 200 μL of chloroform to remove residual phenol, mix by vortexing for 10 s. Centrifuge for 30 s at 21,000 x <i>g</i> at 25°C and remove the majority of the lower organic phase. Centrifuge for 1 min at 21,000 x <i>g</i> at 25°C and remove the residual organic phase.</li>
<li>Add 20 μL of 3M NaOAc and 2.2 μL of glycogen (10 mg/mL), mix by vortexing for 30 s, add 670 μL of 100% EtOH, mix by vortexing for 30 s, and precipitate RNA at −80°C for 16 h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> RNA can be stored at −80°C for ∼6 months.</div>
<ol start="5" type="a">
<li>Centrifuge at 4°C for 30 min at 21,000 x <i>g</i> to pellet RNA. Wash pellets 3 times with 1 mL of 80% ice-cold EtOH.</li>
<li>Air-dry the pellet for ∼5 min. Resuspend RNA by addition of 50 μL of nuclease-free water. Pipette the suspension and incubate on ice for at least 20 min to permit complete resuspension of the RNA. Measure the concentration using a NanoDrop.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The typical yield of RNA isolated from 2 mL of an <i>E. coli</i> cell suspension is ∼60 μg.</div>
<h4>rRNA Depletion</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h</div>
<ol>
<li>Use MICROBExpress Kit to deplete rRNAs from 9 μg of recovered RNA according to the manufacturer’s recommendations.</li>
<li>Measure the volume of rRNA-depleted RNA separated from beads. Add 10% volume of 3M NaOAc and 1% volume of glycogen (10 mg/mL), mix by vortexing for 30 s, add 3 volumes of 100% EtOH, and precipitate RNAs at −80°C for 16 h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> RNA can be stored at −80°C for ∼6 months.</div>
<ol start="3">
<li>Recover RNA as in (<b>B</b>, RNA isolation, step 5e-f). Resuspend RNA in 30 μL of nuclease-free water. Measure the concentration with a NanoDrop.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend using 2 μg of rRNA-depleted RNAs as a minimum starting amount for each enzymatic processing reaction described below.</div>
<h4>CIP Treatment of RNA Prior to NudC or Rai1 Treatments</h4>
<div class="critical">
<span class="critical-title">Critical:</span> For analysis of NCIN-capped RNA, we recommend treating RNA with Calf Intestinal Phosphatase (CIP) to remove 5′ end phosphates prior to performing the NudC or Rai1 treatments described below.</div>
<ol>
<li>Assemble the reaction as indicated below.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>rRNA-depleted RNA, 2 μg</td>
<td>10</td>
</tr>
<tr>
<td>10x CutSmart Buffer</td>
<td>3</td>
</tr>
<tr>
<td>RNaseOUT, 40U/μL</td>
<td>1</td>
</tr>
<tr>
<td>CIP, 10U/μL</td>
<td>0.2</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>15.8</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>30</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Incubate the reaction at 37°C for 1 h.</li>
<li>Add nuclease-free H<sub>2</sub>O to 200 μL, mix with 200 μL of acid phenol/chloroform (pH 4.5), and purify RNA using the procedure described in (<b>B</b>, RNA isolation, step 5a-f).</li>
<li>Resuspend the RNA pellet in 10 μL of nuclease-free water. Pipette the suspension and incubate on ice for at least 20 min to ensure the complete resuspension of RNA. Samples can be stored at −80°C prior to use in subsequent enzymatic reactions.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>Nuclease-free water (not DEPC-treated)</td>
<td>ThermoFisher</td>
<td>Cat#AM9932</td>
</tr>
<tr>
<td>NAD<sup>+</sup>
</td>
<td>Roche</td>
<td>Cat#10127965001</td>
</tr>
<tr>
<td>NADH</td>
<td>Roche</td>
<td>Cat#10107735001</td>
</tr>
<tr>
<td>desphospho-CoA</td>
<td>Sigma-Aldrich</td>
<td>Cat#D3385</td>
</tr>
<tr>
<td>FAD</td>
<td>Sigma-Aldrich</td>
<td>Cat#F6625</td>
</tr>
<tr>
<td>3M Sodium Acetate, pH 5.5</td>
<td>ThermoFisher</td>
<td>Cat#AM9740</td>
</tr>
<tr>
<td>Glycogen from Oyster (type II)</td>
<td>Sigma-Aldrich</td>
<td>Cat#G8751</td>
</tr>
<tr>
<td>Ethyl Alcohol</td>
<td>Pharmco-AAPER</td>
<td>Cat#111000200</td>
</tr>
<tr>
<td>Isopropyl Alcohol</td>
<td>BDH</td>
<td>Cat#BDH1133-1LP</td>
</tr>
<tr>
<td>Low Range ssRNA Ladder</td>
<td>NEB</td>
<td>Cat#N0364S</td>
</tr>
<tr>
<td>SYBR Gold nucleic acid gel stain</td>
<td>ThermoFisher</td>
<td>Cat#S11494</td>
</tr>
<tr>
<td>Acid phenol:chloroform (CHCl<sub>3</sub>) pH 4.5</td>
<td>Fisher</td>
<td>Cat#BP17541</td>
</tr>
<tr>
<td>5X Detergent-free Phusion HF Buffer Pack</td>
<td>NEB</td>
<td>Cat#B0520S</td>
</tr>
<tr>
<td>RNaseOUT, Recombinant Ribonuclease Inhibitor</td>
<td>ThermoFisher</td>
<td>Cat#10777-019</td>
</tr>
<tr>
<td>Calf Intestine Alkaline Phosphatase</td>
<td>NEB</td>
<td>Cat#M0290S</td>
</tr>
<tr>
<td>RNA 5′ Polyphosphatase</td>
<td>Lucigen (Epicenter)</td>
<td>Cat#RP8092H</td>
</tr>
<tr>
<td>NudC</td>
<td>(<a href="#bib2">Cahová et al., 2015</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Rai1</td>
<td>(<a href="#bib10">Xiang et al., 2009</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>T4 RNA Ligase 1 (ssRNA Ligase)</td>
<td>NEB</td>
<td>Cat#M0204L</td>
</tr>
<tr>
<td>Superscript III Reverse Transcriptase</td>
<td>ThermoFisher</td>
<td>Cat#18080-044</td>
</tr>
<tr>
<td>RNase H</td>
<td>ThermoFisher</td>
<td>Cat#AM2293</td>
</tr>
<tr>
<td>Phusion HF DNA Polymerase</td>
<td>ThermoFisher</td>
<td>Cat#F-530L</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Illumina PCR primer, RP1:<br>AATGATACGGCGACCACCGAGATCTACACGTTCAGAGTTCTACAGTCCGA</td>
<td>Illumina</td>
<td>N/A</td>
</tr>
<tr>
<td>Illumina indexing PCR primer, RPI1 (index sequence is in bold): CAAGCAGAAGACGGCATACGAGAT<b>CGTGAT</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCCA</td>
<td>Illumina</td>
<td>N/A</td>
</tr>
<tr>
<td>i105, 5′ adaptor with CUGA barcode and 11N extension (barcode is underlined): GUUCAGAGUUCUACAGUCCGACGAUC<u>CUGA</u>NNNNNNNNNNN</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>i106, 5′ adaptor with GACU barcode and 11N extension (barcode is underlined): GUUCAGAGUUCUACAGUCCGACGAUC<u>GACU</u>NNNNNNNNNNN</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>i107, 5′ adaptor with AGUC barcode and 11N extension (barcode is underlined): GUUCAGAGUUCUACAGUCCGACGAUC<u>AGUC</u>NNNNNNNNNNN</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>i108, 5′ adaptor with UCAG barcode and 11N extension (barcode sequence is underlined): GUUCAGAGUUCUACAGUCCGACGAUC<u>UCAG</u>NNNNNNNNNNN</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>RT primer: CCTTGGCACCCGAGAATTCCANNNNNNNNN</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>s1115, Custom Illumina Sequencing Primer: CTACACGTTCAGAGTTCTACAGTCCGACGATC</td>
<td>Illumina</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Micellula DNA Emulsion PCR Kit</td>
<td>Chimerx</td>
<td>Cat#3600-02</td>
</tr>
<tr>
<td>Spin-X centrifuge tube filter, 0.45 μm, RNase/DNase free</td>
<td>Costar</td>
<td>Cat#8162</td>
</tr>
<tr>
<td>10% TBE-Urea gels, 1mm x 10 wells</td>
<td>ThermoFisher</td>
<td>Cat#EC6875Box</td>
</tr>
<tr>
<td>10% TBE gels, 1mm x 10 wells</td>
<td>ThermoFisher</td>
<td>Cat#EC6275Box</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<div class="note">
<span class="note-title">Note:</span> The protocol for construction of cDNA libraries in CapZyme-seq enables single-nucleotide-resolution analysis of RNA 5′ ends. In particular, single-stranded oligonucleotide adapters are ligated to RNA 5′ ends and the 5′-adapter-ligated products are used as templates for generation of cDNAs that are analyzed by high-throughput sequencing. For each sequencing read, the precise identification of the RNA 5′ end sequence is accomplished by using the sequence of the 3′ end of the 5′ adapter as a point of reference.</div>
<h3>Enzymatic Treatments of RNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1-2 h</div>
<p>CapZyme-seq relies on ligation of single-stranded oligonucleotide adapters to RNA 5′ ends prior to cDNA synthesis. This ligation step requires that the RNA have a 5′-monophosphate. Thus, analysis of RNAs without a 5′-monophosphate requires enzymatic processing of the RNAs to yield RNAs having a 5′-monophosphate. In CapZyme-seq the processing enzymes NudC and Rai1, which are specific for NCIN-capped RNA, along with RNA 5′ polyphosphatase (Rpp), a processing enzyme specific for uncapped, 5′-triphosphate RNA, are employed to enable differential detection and quantitation of NCIN-capped RNA and uncapped, 5′-triphosphate RNA.</p>
<p>For selective processing of NCIN-capped RNA to 5′-monophosphate RNA, we use the bacterial NCIN-decapping enzyme NudC or the fungal RNA-decapping enzyme Rai1. NudC processes NCIN-capped RNA to 5′-monophosphate RNA by cleaving the diphosphate group of the NCIN cap, yielding products comprising 5′-pNp- (where N is the 3′ nucleoside moiety of the NCIN) followed by the remainder of the RNA (<a href="#bib2">Cahová et al., 2015</a>; <a href="#bib4">Höfer et al., 2016</a>). Rai1 processes NCIN-capped RNAs to 5′-monophosphate RNA by cleaving the phosphodiester bond connecting the NCIN cap to the remainder of the RNA, yielding products comprising 5′-p- followed by the remainder of the RNA (<a href="#bib5">Jiao et al., 2017</a>; <a href="#fig2">Figure 2</a>). NudC and Rai1 process RNA capped with at least four of the nucleoside-containing metabolites that can serve as NCINs: NAD<sup>+</sup>, NADH, dpCoA, and FAD (<a href="#bib1">Bird et al., 2016</a>; <a href="#bib2">Cahová et al., 2015</a>; <a href="#bib5">Jiao et al., 2017</a>; <a href="#bib8">Vvedenskaya et al., 2018a</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/16-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Enzymatic Processing of RNA 5′ Ends by Rpp, NudC, and Rai1</div>
</figcaption>
</figure>
<ol>
<li>Set heat block to 37°C.</li>
<li>Aliquot RNA into 1.7 mL nuclease-free tubes.</li>
</ol>
<h4>Rpp Treatment</h4>
<p>For selective processing of uncapped, 5′-triphosphate RNA to 5′-monophosphate RNA we use the RNA processing enzyme RNA 5′ polyphosphatase (Rpp), which cleaves the phosphodiester bond between the triphosphate β and α phosphates, yielding products comprising 5′-p- followed by the remainder of the RNA (<a href="#fig2">Figure 2</a>). Reaction mixtures performed without addition of Rpp serve as a control.</p>
<ol>
<li>Assemble the reactions as indicated below.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>Rpp Treatment Volume (μL)</th>
<th>Mock Rpp Treatment Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>RNA (generated <i>in vitro</i> or <i>in vivo</i>)</td>
<td>10</td>
<td>10</td>
</tr>
<tr>
<td>10x Rpp buffer</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>RNaseOUT, 40U/μL</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>Rpp, 20U/μL</td>
<td>1</td>
<td>-</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>6</td>
<td>7</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>20</td>
<td>20</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Incubate at 37°C for 30 min.</li>
</ol>
<h4>NudC Treatment</h4>
<p>NudC processes NCIN-capped RNA to 5′-monophosphate RNA by cleaving the diphosphate group of the NCIN cap, yielding products comprising 5′-pNp- (where N is the 3′-nucleoside moiety of the NCIN, for example the adenosine moiety of NAD<sup>+</sup>) followed by the remainder of the RNA (<a href="#fig2">Figure 2</a>). Reaction mixtures performed without addition of NudC serve as a control.</p>
<ol>
<li>Assemble the reactions as indicated below.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>NudC Treatment Volume (μL)</th>
<th>Mock NudC Treatment Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>CIP-treated RNA (generated<i>in vitro</i> or <i>in vivo</i>)</td>
<td>10</td>
<td>10</td>
</tr>
<tr>
<td>10x NEB Buffer 2</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>RNaseOUT, 40U/μL</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>NudC, 55 μM</td>
<td>1.3</td>
<td>-</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>5.7</td>
<td>7</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>20</td>
<td>20</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Incubate reactions at 37°C for 30 min.</li>
</ol>
<h4>Rai1 Treatment</h4>
<p>Rai1 processes NCIN-capped RNAs to 5′-monophosphate RNA by cleaving the phosphodiester bond connecting the NCIN cap to the remainder of the RNA, yielding products comprising 5′-p- followed by the remainder of the RNA (<a href="#fig2">Figure 2</a>). Reaction mixtures performed without addition of Rai1 serve as a control.</p>
<ol start="3">
<li>Assemble the reactions as indicated below.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>Rai1 Treatment Volume (μL)</th>
<th>Mock Rai1 Treatment Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>CIP-treated RNA (generated <i>in vitro</i> or <i>in vivo</i>)</td>
<td>10</td>
<td>10</td>
</tr>
<tr>
<td>10x Rai1 buffer</td>
<td>2</td>
<td>2</td>
</tr>
<tr>
<td>100 mM MnCl<sub>2</sub>, prepared fresh</td>
<td>0.2</td>
<td>0.2</td>
</tr>
<tr>
<td>RNaseOUT, 40U/μL</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>Rai1, 3.4 μM</td>
<td>2</td>
<td>-</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>4.8</td>
<td>6.8</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>20</td>
<td>20</td>
</tr>
</tbody>
</table>
<ol start="4">
<li>Incubate reactions at 37°C for 30 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As mentioned above, NudC and Rai1 process RNA capped with at least four of the nucleoside-containing metabolites that can serve as NCINs: NAD<sup>+</sup>, NADH, dpCoA, and FAD. Therefore, use of NudC or Rai1 enables detection of NAD<sup>+</sup>-, NADH-, dpCoA-, and FAD-capped RNA by CapZyme-seq.</div>
<h3>Recovery of RNA Products after Enzymatic Processing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼4 h</div>
<h4>RNA Products Generated <i>In Vitro</i>
</h4>
<ol>
<li>Add 20 μL of 2x RNA loading dye to the sample and load along with a ssRNA ladder on a 10% TBE-urea slab gels. Run gel in 1x TBE buffer until bromophenol blue dye front reaches ∼1/2 of the length of the gel.</li>
<li>Incubate gel with SYBR Gold nucleic acid gel stain and visualize bands with UV transillumination. Excise RNA products of the expected size from the gel using a scalpel. Gels for distinct reactions should be stained in separate containers.</li>
<li>Elute RNAs from gel using the procedure described in (<b>A</b>, Isolation of RNA products, step 1-7).</li>
<li>Resuspend RNAs in 10 μL of nuclease-free water. Pipette the suspension and incubate on ice for at least 20 min to permit full resuspension of the RNA.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Considering the relatively low amount of RNA generated by <i>in vitro</i> transcription, we do not recommend acid phenol/chloroform purification after enzymatic treatments as this may result in significant RNA loss. Instead, we recommend gel purification as described above.</div>
<div class="critical">
<span class="critical-title">Critical:</span> We do not recommend heating samples prior to gel loading as it may result in degradation of the 5′ end NCIN cap.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If desired, GlycoBlue can be used to assist in the visualization of the RNA pellet. However, use of GlycoBlue interferes with quantification of cDNA by Qubit ssDNA and dsDNA Assay kits.</div>
<h4>RNA Products Generated <i>In Vivo</i>
</h4>
<ol start="5">
<li>Add nuclease-free H<sub>2</sub>O to 200 μL, mix with 200 μL of acid phenol/chloroform pH 4.5 and purify RNAs using the procedure described in (<b>B</b>, RNA isolation, step 5a-f).</li>
<li>Resuspend RNA in 10 μL of nuclease-free water. Pipette the suspension and incubate on ice for 20 min to permit solubilization of RNA.</li>
</ol>
<h3>5′ Adaptor Ligation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼18-20 h</div>
<p>CapZyme-seq<sup>NudC</sup> involves quantitative comparisons between RNA samples treated with Rpp or NudC, along with two “mock” treated samples. CapZyme-seq<sup>Rai1</sup> involves quantitative comparisons between RNA samples treated with Rpp or Rai1, along with two “mock” treated samples. Thus, to enable quantitative comparisons between the four samples in CapZyme-seq<sup>NudC</sup> or CapZyme-seq<sup>Rai1</sup>, a unique, barcoded 5′-adaptor oligonucleotide is used in ligation reactions performed with each of the four individual samples (see <a href="#fig3">Figure 3</a> and table below).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/16-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. CapZyme-Seq Library Construction Workflow</div>
</figcaption>
</figure>
<p>After the ligation reactions have been performed, the four samples of adaptor-ligated products are isolated by gel electrophoresis, combined together, and used for the remaining steps of library construction. After high-throughput sequencing, the identity of the 5′-adaptor barcode is used to associate each sequencing read with the sample from which it was derived.</p>
<p>The barcoded 5′-adaptor oligonucleotides also carry an extended randomized region that provides each sequencing read with a unique molecular identifier, “UMI,” enabling quantitative comparisons between cDNA libraries generated from RNA subjected to distinct enzymatic treatments. A typical CapZyme-seq experiment involves analysis of RNA subjected to four distinct treatments, and thus requires use of four barcoded 5′-adapters. Each barcoded 5′-adaptor contains a four nucleotide barcode followed by an 11-nucleotide randomized sequence. The barcode comprises the first four bases of each sequencing read while the 11-nucleotide randomized sequence comprises bases 5 to 15 of each sequencing read. Thus, the identity of the first four bases of each read is used to associate each sequencing read with a corresponding enzymatic processing reaction while bases 5 to 15 of each sequencing read are used as a UMI to reduce effects of PCR amplification bias.</p>
<table>
<thead>
<tr>
<th>Sample</th>
<th>Barcoded 5′-adaptor oligonucleotide</th>
</tr>
</thead>
<tbody>
<tr>
<td>Rpp-treated RNA</td>
<td>i105</td>
</tr>
<tr>
<td>mock Rpp-treated RNA</td>
<td>i106</td>
</tr>
<tr>
<td>NudC- or Rai1- treated RNA</td>
<td>i107</td>
</tr>
<tr>
<td>mock NudC- or Rai1-treated RNA</td>
<td>i108</td>
</tr>
</tbody>
</table>
<ol>
<li>Assemble ligation reaction as indicated below.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>
<i>in vitro</i> RNAs Volume (μL)</th>
<th>
<i>in vivo</i> RNAs Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>RNA (Rpp-, NudC-, Rai1-, or mock-treated)</td>
<td>10</td>
<td>10</td>
</tr>
<tr>
<td>5′ RNA adaptor (i105, i106, i107, or i108), 10 μM</td>
<td>1</td>
<td>3</td>
</tr>
<tr>
<td>10x T4 RNA ligase buffer</td>
<td>3</td>
<td>3</td>
</tr>
<tr>
<td>10 mM ATP</td>
<td>3</td>
<td>3</td>
</tr>
<tr>
<td>PEG8000, 50%</td>
<td>6</td>
<td>6</td>
</tr>
<tr>
<td>RNaseOUT, 40U/μL</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>T4 RNA Ligase 1, 10U/μL</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>5</td>
<td>3</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>30</td>
<td>30</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The concentration of the 5′ adapter in reactions performed using RNA generated <i>in vivo</i> is three times greater than in reactions performed using RNA generated <i>in vitro</i>.</div>
<ol start="2">
<li>Incubate reactions at 16°C for 16 h.</li>
<li>Stop each reaction by adding 30 μL of 2x RNA loading dye. Load the four ligation reactions along with a ssRNA ladder on a 10% 7M urea slab gel. Run gel in 1x TBE buffer until the bromophenol blue dye front reaches ∼1/2 of the length of the gel.</li>
<li>Incubate gels with SYBR Gold nucleic acid gel stain and visualize bands with UV transillumination (<a href="#fig4">Figure 4</a>). Gels for individual ligation mixture should be stained in their own container.</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/16-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Representative Gel Stained with SYBR Gold Nucleic Acid Gel Stain</div>
</figcaption>
</figure>
<ol start="5">
<li>Excise 5′-adaptor-ligated products of the desired length from the gel using a scalpel.</li>
<li>Place gel slices into 4 to 8 tube-in-tube assemblies and recover nucleic acids using the procedure described in (<b>A</b>, Isolation of RNA products, step 3-8).</li>
<li>Resuspend adaptor-ligated products in nuclease-free water. (We recommend a volume of 15 μL for RNA generated <i>in vitro</i> and 30 μL for RNA generated <i>in vivo</i>). Pipette the suspension and incubate on ice for at least 20 min to enable full resuspension of the 5′-adaptor-ligated products.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For analysis of RNA products having a defined range of lengths, excise the portion of the gel slightly above and slightly below the desired size range. For analysis of all RNA products, excise the portion of the gel slighlty above the 5′ adapter to the top.</div>
<h3>First-Strand cDNA Synthesis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼6-7 h</div>
<p>Mix 15 μL of 5′-adaptor-ligated products with 0.3 μL of 100 μM of the reverse transcription (RT) primer (or a mixture of several RT primers to analyze several promoter sequences that are each upstream of a distinct transcribed region).</p>
<ol>
<li>Incubate the mixture at 65°C for 5 min, then cool to 4°C.</li>
<li>Prepare the RT reaction mix as indicated below.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>5x First-Strand buffer</td>
<td>6</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td>1.5</td>
</tr>
<tr>
<td>100 mM DTT</td>
<td>1.5</td>
</tr>
<tr>
<td>RNaseOUT, 40U/μL</td>
<td>1.5</td>
</tr>
<tr>
<td>SuperScript III Reverse Transcriptase, 200U/μL</td>
<td>1.5</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>2.7</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>14.7</td>
</tr>
</tbody>
</table>
<ol start="3">
<li>Add mixture prepared in step 1 to mixture prepared in step 2. Pipette to mix. Incubate at 25°C for 5 min, 55°C for 60 min, 70°C for 15 min, then cool to 25°C.</li>
<li>Add 10U of RNase H and pipette to mix.</li>
<li>Incubate reaction at 37°C for 20 min then add 30 μL of 2x RNA loading dye.</li>
<li>Load the combined samples on a 10% 7M TBE-urea slab gel along with a ssRNA size ladder. Run gel in 1x TBE buffer until bromophenol blue dye front reaches ∼1/2 of the length of the gel. Incubate gels with SYBR Gold nucleic acid gel stain and visualize bands with UV transillumination. Gels for separate reactions should be stained in separate containers. Excise cDNAs of the desired length from the gel using a scalpel.</li>
<li>Place gel slices into 2 to 4 tube-in-tube assemblies and recover nucleic acids using the procedure described in (A, Isolation of RNA products, step 3–8).</li>
<li>Resuspend cDNAs in 20 μL of nuclease-free water. Pipette the suspension and incubate on ice for at least 20 min to enable the complete resuspension of cDNA. Measure the concentration of nucleic acids in each solution using Qubit ssDNA Assay kit (ThermoFisher).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> cDNA can be stored at −80°C for at least one week.</div>
<div class="note">
<span class="note-title">Note:</span> Ensure that any primers used for RT contain sequence complementary to the Illumina 3′ PCR primer.</div>
<h3>cDNA Amplification and Recovery</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼7-8 h</div>
<ol>
<li>Dilute the cDNA with nuclease-free water to a concentration of ∼10<sup>9</sup> molecules/μL.</li>
<li>Prepare emulsion PCR (ePCR) reaction mix as indicated below.</li>
</ol>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>cDNA, ∼10<sup>9</sup> molecules/μL</td>
<td>2</td>
</tr>
<tr>
<td>5x Detergent-free HF Phusion Buffer with MgCl<sub>2</sub>
</td>
<td>10</td>
</tr>
<tr>
<td>0.1 mg/mL BSA</td>
<td>2.5</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td>2</td>
</tr>
<tr>
<td>10 μM Primer RP1</td>
<td>2.5</td>
</tr>
<tr>
<td>10 μM Primer RPI1-48</td>
<td>2.5</td>
</tr>
<tr>
<td>HF Phusion Polymerase, 2U/μL</td>
<td>1</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>27.5</td>
</tr>
<tr>
<td><b>Total:</b></td>
<td>50</td>
</tr>
</tbody>
</table>
<ol start="3">
<li>Perform ePCR reactions using the recommended cycling conditions: 10 s at 90°C, 20 cycles of 5 s at 90°C, 5 s at 60°C, and 15 s at 72°C, followed by 5 min at 72°C.</li>
<li>Isolate cDNA from ePCR reactions using Micellula DNA Emulsion and Purification Kit. Elute cDNA from the column filter with 150 μL of the elution buffer provided with the kit. Add 15 μL of 3M NaOAc, 1.5 μL of glycogen (10 mg/mL) and 500 μL of 100% EtOH. Mix solution by vortexing and precipitate cDNA at −80°C for 16 h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> cDNA can be stored at −80°C for at least one week.</div>
<ol start="5">
<li>Centrifuge tubes at 4°C for 30 min to pellet cDNAs. Wash pellets 3 times with 1 mL of 80% ice-cold EtOH and resuspend cDNAs in 20 μL of nuclease-free water.</li>
<li>Mix 1 μL of the resuspended cDNA with 6x Orange DNA load dye and run a non-denaturing 10% TBE gel to check the library quality. Load O’Gene Ruler Ultra Low Range DNA ladder as size standard.</li>
<li>If residual primers are present, mix the rest of cDNA with 6x Orange DNA load dye and isolate amplified cDNAs by gel electrophoresis using a non-denaturing 10% TBE gel. Isolate products by gel excision using the procedure described in (<b>A</b>, Isolation of RNA products).</li>
<li>Elute cDNAs from gel with 600 μL of gel elution buffer at 37°C for 3 h, vortex the slurry every 30 min. Collect the eluate.</li>
<li>Add 60 μL of 3M NaOAc, 6.6 μL of glycogen (10 mg/mL), and an equal volume of isopropanol. Precipitate cDNAs at −80°C for 16 h.</li>
<li>Centrifuge samples at 4°C for 30 min to pellet cDNAs. Wash pellets 3 times with 1 mL of 80% ice-cold EtOH and resuspend cDNAs in 20 μL of nuclease-free water.</li>
<li>Measure the concentration of the amplified libraries using Qubit dsDNA HS Assay kit. Submit libraries for sequencing.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use of ePCR minimizes the generation of amplicons derived from “template switching.”</div>
<div class="note">
<span class="note-title">Note:</span> Optimization of the amplification of cDNAs by ePCR is critical for successful library sequencing. In our experience, use of a Qubit ssDNA Assay kit to prepare ssDNA dilution of ∼10<sup>9</sup> molecules/μl provides an imprecise measure of ssDNA concentration, likely because of chemicals in the solution that interfere with quantitation.</div>
<div class="note">
<span class="note-title">Note:</span> Barcoded libraries generated by CapZyme-seq are suitable for sequencing on Illumina NextSeq and Illumina HiSeq platforms in high-output mode. Sequencing primers provided by Illumina typically contain a mixture of several oligos. To avoid potential complications that may arise due to the presence of additional primers in the Illumina reagents we use a custom sequencing primer (s1115).</div>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>In published work, CapZyme-seq has been used to quantitate relative yields of NAD<sup>+</sup>-capped and uncapped RNA and to define preferred transcription start-site positions and consensus promoter sequences for NAD<sup>+</sup> capping by bacterial RNAP <i>in vitro</i> and <i>in vivo</i> (<a href="#bib8">Vvedenskaya et al., 2018a</a>). The method can be readily adapted to define the promoter-sequence determinants for NCIN capping by bacterial RNAP holoenzymes carrying alternative σ factors, archaeal RNAP, eukaryotic RNAP I, II, and III, mitochondrial RNAP, and chloroplast RNAP.</p>
<p>The prevalence of nucleoside-containing metabolites that can function as NCINs (e.g., NAD<sup>+</sup>, NADH, dpCoA, FAD, UDP-glucose, and UDP-GlcNAc) underscores the need to determine, for each NCIN, the promoter-sequence determinants for NCIN capping. The CapZyme-seq method, either using the same decapping enzymes described here or using alternative decapping enzymes with alternative decapping specificities, should enable analysis of determination of NCIN-capping efficiencies and promoter sequence determinants thereof for each nucleoside-containing metabolite that can serve as an NCIN.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>CapZyme-seq, in its current version that uses NudC and RaiI as decapping enzymes, identifies NCIN-capped RNA but does not identify the NCIN cap. This is not an issue for experiments <i>in vitro</i>, where the experimenter controls the identity of the NCIN. However, this can be an issue for experiments <i>in vivo</i>, where NCIN capping may occur with any of nucleoside-containing metabolites. Accordingly, for experiments <i>in vivo</i>, identification of the NCIN cap may require additional analysis (e.g., metabolic labeling, electrophoretic mobility, boronic-acid reactivity, immunoassay, or mass spectrometry [<a href="#bib1">Bird et al., 2016</a>; <a href="#bib2">Cahová et al., 2015</a>; <a href="#bib3">Chen et al., 2009</a>; <a href="#bib6">Kowtoniuk et al., 2009</a>; <a href="#bib7">Nübel et al., 2017</a>]). A goal of future work will be to develop alternate versions of CapZyme-seq that employ additional RNA 5′ end processing enzymes that selectively report on specific 5′ end modifications.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem</h3>
<p>Multiple bands are visible on the gel below the 5′ adaptor during size selection of 5′-adaptor-ligated products.</p>
<h3>Potential Solution</h3>
<p>Have adaptor resynthesized or perform gel purification of adaptor prior to ligation (load adaptor on a 10%-urea TBE slab gel, excise the major band from gel, elute and purify as described in A, Isolation of RNA products, steps 1–8).</p>
<h3>Problem</h3>
<p>Insufficient amount of 5′-adaptor-ligated products.</p>
<h3>Potential Solution</h3>
<p>Increase amount of starting material; collect all of the upper aqueous phase during the acid phenol/chloroform purification step.</p>
<h3>Problem</h3>
<p>Insufficient yield of ePCR amplicons.</p>
<h3>Potential Solution</h3>
<p>If the library after ePCR amplification is &lt; 2.5 ng/μl as detected by Qubit dsDNA Assay kit, (i) perform two ePCR reactions per library; (ii) increase the volume of diluted ssDNA used in ePCR reaction by two-fold; (iii) increase the number of amplification cycles up to 30.</p>
<h3>Problem</h3>
<p>A large portion of the library is comprised of a ∼150 bp product.</p>
<h3>Potential Solution</h3>
<p>After 5′ adapter-ligated products or ssDNA have been subjected to gel electrophoresis, place the gel in SYBR Gold nucleic acid gel stain, incubate for 5 min with low agitation, and carefully excise products of the desired size. Alternatively, prior to incubation with SYBR Gold nucleic acid gel stain, remove the lower part of the gel containing an excess of 5′ adapters or an excess of RT primer(s), stain the remaining part of the gel, and carefully excise products of the desired size. For the final cDNA library, also perform gel electrophoresis, gel staining, and careful excision of products of the desired size.</p>
</section>
<section>
<h2 id="q-and-a">Q&amp;A</h2>
<p class="question"><b>Q:</b> How stable is the RNAP-promoter open complexes (1.2)? Should the user keep the product on ice or 4 degrees when preparing the NTP mixtures?</p>
<p class="answer"><b>A:</b> RNAP-open complex stability depends on the sequence of the promoter and the identity of the RNAP used— for the purposes of this protocol if the issue is what to do with the RNAP-open complexes when preparing the NTP mixtures, the answer is that they should be stored at 37 degrees, not 4 degrees.</p>
<p class="question"><b>Q:</b> Several steps in the protocol requires isolating RNA or cDNA using gel electrophoresis. Could the authors specify the expected size for each product?</p>
<p class="answer"><b>A:</b> Initial size selection of in vitro generated templates: 150 nt.(For our MASTER templates).<br> 5’ Adapter-ligated in vitro generated template size selection: 150 nt plus the size of 5’ adapter (41 nt).<br> RT product size selection for in vitro generated templates: calculated based on the position of sequence complementary to the RT primer.<br> Final library size selection: calculated based on the size of RT products.<br> In vivo generated libraries: 5’ Adapter-ligated template size selection: above the 5’ adapter to the gel top.<br> RT product size selection: in case of gene-specific primers, calculated based on the position of sequence complementary to the RT primer; in case of whole-genome analysis and the use of RT primer with randomized sequence (s1082-N9 as an example), products above the RT primer and up to 300-500 nt are selected.<br> Final library size selection: calculated based on the size of RT products.</p>
<p class="question"><b>Q:</b> What is the composition of the elution solution?</p>
<p class="answer"><b>A:</b> 0.3 M NaCl, 10 mM Tris-HCl pH 8, 0.1 mM EDTA.</p>
<p class="question"><b>Q:</b> What is the recommended instrument(s) for performing ePCR?</p>
<p class="answer"><b>A:</b> We used the following:<br> Nutating Mixer from Labnet<br> Vortex mixer, VWR<br> Microcentrifuge, eppendorf<br> Themocycler, Biorad</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bird,J.G.,Zhang,Y.,Tian,Y.,Panova,N.,Barvík,I., Greene, L., Liu, M., Buckley, B., Krásný, L., Lee, J.K., et al. (2016). The mechanism of RNA 5′ capping with NAD<sup>+</sup>, NADH and desphospho-CoA Nature <i>535</i>, 444–447.</p>
<p id="bib2">Cahová, H., Winz, M.L., Höfer, K., Nübel, G., and Jäschke, A. (2015). NAD captureSeq indicates NAD as a bacterial cap for a subset of regulatory RNAs. Nature <i>519</i>, 374–377.</p>
<p id="bib3">Chen, Y.G., Kowtoniuk, W.E., Agarwal, I., Shen, Y., and Liu, D.R. (2009). LC/MS analysis of cellular RNA reveals NAD-linked RNA. Nat. Chem. Biol. <i>5</i>, 879–881.</p>
<p id="bib4">Höfer, K., Li, S., Abele, F., Frindert, J., Schlotthauer, J., Grawenhoff, J., Du, J., Patel, D.J., and Jäschke, A. (2016). Structure and function of the bacterial decapping enzyme NudC. Nat. Chem. Biol. <i>12</i>, 730–734.</p>
<p id="bib5">Jiao, X., Doamekpor, S.K., Bird, J.G., Nickels, B.E., Tong, L., Hart, R.P., and Kiledjian, M. (2017). 5′ end nicotinamide adenine dinucleotide cap in human cells promotes RNA decay through DXO-mediated deNADding. Cell <i>168</i>, 1015-1027.e10.</p>
<p id="bib6">Kowtoniuk, W.E., Shen, Y., Heemstra, J.M., Agarwal, I., and Liu, D.R. (2009). A chemical screen for biological small molecule-RNA conjugates reveals CoA-linked RNA. Proc. Natl. Acad. Sci. USA <i>106</i>, 7768-7773.</p>
<p id="bib7">Nübel, G., Sorgenfrei, F.A., and Jäschke, A. (2017). Boronate affinity electrophoresis for the purification and analysis of cofactor-modified RNAs. Methods <i>117</i>, 14-20.</p>
<p id="bib8">Vvedenskaya, I.O., Bird, J.G., Zhang, Y., Zhang, Y., Jiao, X., Barvík, I., Krásný, L., Kiledjian, M., Taylor, D.M., Ebright, R.H., and Nickels, B.E. (2018a). CapZyme-seq comprehensively defines promoter-sequence determinants for RNA 5′ capping with NAD+. Mol. Cell <i>70</i>, 553-564.e9.</p>
<p id="bib9">Vvedenskaya, I.O., Goldman, S.R., and Nickels, B.E. (2018b). Analysis of bacterial transcription by “massively systematic transcript end readout,” MASTER. Methods Enzymol. <i>612</i>, 269-302.</p>
<p id="bib10">Xiang, S., Cooper-Morgan, A., Jiao, X., Kiledjian, M., Manley, J.L., and Tong, L. (2009). Structure and function of the 5′--&gt;3′ exoribonuclease Rat1 and its activating partner Rai1. Nature <i>458</i>, 784-788.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>Work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grant GM118059 (B.E.N).</p>
<h3>Author Contributions</h3>
<p>Writing, I.O.V. and B.E.N.; Funding Acquisition, B.E.N.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>