<article>
<section>
<h1 id="header">
<i>In vitro</i> and <i>ex vivo</i> evaluation of tumor-derived exosome-induced dendritic cell dysfunction in mouse</h1>
<p><time datetime="2021-03-05">Published: March 5, 2021</time></p>
<p>Wenfeng Zeng,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a></sup> Xiaozhe Yin,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Yunhan Jiang,<sup><a href="#aff3">3</a></sup> Lingtao Jin,<sup><a href="#aff3">3</a>,<a href="#fn3">6</a>,<a href="#cor1">*</a></sup> and Wei Liang<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Protein and Peptide Pharmaceutical Laboratory, Institute of Biophysics, Chinese Academy of Sciences, 15 Datun Road Chaoyang District, Beijing 100101, China</p>
<p id="aff2"><sup>2</sup>University of Chinese Academy of Sciences, Beijing 100864, China</p>
<p id="aff3"><sup>3</sup>Department of Anatomy and Cell Biology, College of Medicine, University of Florida, 2033 Mowry Road, Gainesville, FL 32610-3033, USA</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:LJIN1@ufl.edu">LJIN1@ufl.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:weixx@ibp.ac.cn">weixx@ibp.ac.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100361">10.1016/j.xpro.2021.100361</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Exosomes that contain various signaling molecules, such as proteins, nucleotides, metabolites, and lipids, are important for intercellular communication. Dendritic cells (DC) are central regulators of anti-tumor immunity but can be suppressed by tumor-derived exosomes (TDEs) in the tumor microenvironment. Here, we describe a step-by-step protocol for TDE isolation and evaluation of TDEs on DCs both <i>in vitro</i> and <i>ex vivo</i> with high repeatability. This approach is useful for the interrogating TDE-DC interactions and identification of novel immune regulators.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib6">Yin et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Separation of tumor-derived exosomes (TDEs) from tumor cell supernatant</li>
<li>Generation of TDE-treated DCs <i>in vitro</i> and <i>ex vivo</i>
</li>
<li>Magnetic purification of CD8<sup>+</sup> T cell from OT I transgenic mice spleen</li>
<li>Analysis of T cell proliferation via flow cytometry</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/503-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Exosome-depleted fetal bovine serum preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>Exosome-depleted FBS is prepared by centrifuging at 100,000 × <i>g</i> for 2 h at 4°C, and supernatant are collected for future RPMI 1640 medium preparation.</p>
<div class="note">
<span class="note-title">Note:</span> To exclude the potential effect of FBS-related exosome, exosome-depleted FBS is required for all exosome-isolating experiments.</div>
<h3 id="sec1.2">Anesthesia preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Prepare 20 mL tribromo-ethanol anesthesia.</p>
<ol>
<li>Weigh 0.25 g 2,2,2-tribromo-ethanol and dissolve in 0.5 mL 2-methyl-2-butanol, by vortex.</li>
<li>Heat 19.5 mL ddH<sub>2</sub>O to 50°C.</li>
<li>Mix 1 and 2 thoroughly, and heat at 50°C for 20 min.</li>
<li>Sterilize the solution with 0.22 μm filter and store at 4°C for up to 6 months.</li>
</ol>
<h3 id="sec1.3">Antibody mix preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Prepare the antibody mix before staining the sample.</p>
<ol>
<li>Antibody mix for BMDC as the flow panel below.</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Fluorophore</th>
<th>Marker</th>
<th>Clone</th>
<th>Final dilution</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>APC</td>
<td>CD11c</td>
<td>N418</td>
<td>1/100</td>
<td>1 μL</td>
</tr>
<tr>
<td>FITC</td>
<td>MHC II</td>
<td>M5/114.15.2</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Antibody mix for OT I CD8<sup>+</sup> T cell as the flow panel below.</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th>Fluorophore</th>
<th>Marker</th>
<th>Clone</th>
<th>Final dilution</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>PE</td>
<td>TCRVa2</td>
<td>B20.1</td>
<td>1/100</td>
<td>1 μL</td>
</tr>
<tr>
<td>APC</td>
<td>CD8a</td>
<td>53–6.7</td>
<td>1/100</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
<ol start="3">
<li>Antibody mix for DCs from lymph node as the flow panel below.</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th>Fluorophore</th>
<th>Marker</th>
<th>Clone</th>
<th>Final dilution</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>BV605</td>
<td>CD45</td>
<td>30-F11</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>PE</td>
<td>CD11b</td>
<td>M1/70</td>
<td>1/400</td>
<td>0.25 μL</td>
</tr>
<tr>
<td>PerCP/Cyanine5.5</td>
<td>Ly6C</td>
<td>HK1.4</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>PE/Cy7</td>
<td>MHC II</td>
<td>M5/114.15.2</td>
<td>1/400</td>
<td>0.25 μL</td>
</tr>
<tr>
<td>BV785</td>
<td>F4/80</td>
<td>BM8</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>APC/Cy7</td>
<td>CD11c</td>
<td>N418</td>
<td>1/100</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
<ol start="4">
<li>Antibody mix for proliferation of OT I CD8<sup>+</sup> T cell as the flow panel below.</li>
</ol>
<table id="undtbl4">
<thead>
<tr>
<th>Fluorophore</th>
<th>Marker</th>
<th>Clone</th>
<th>Final dilution</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>PE</td>
<td>CD11c</td>
<td>N418</td>
<td>1/100</td>
<td>1 μL</td>
</tr>
<tr>
<td>APC</td>
<td>CD8a</td>
<td>53–6.7</td>
<td>1/100</td>
<td>1 μL</td>
</tr>
<tr>
<td>PE/Cy7</td>
<td>CD3</td>
<td>17A2</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th colspan="2">IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="4">Antibodies</td>
</tr>
<tr>
<td>Anti-mouse CD8a APC</td>
<td>BioLegend</td>
<td colspan="2">Cat # 100712; RRID: AB_312751</td>
</tr>
<tr>
<td>Anti-mouse CD11b PE</td>
<td>eBioscience</td>
<td colspan="2">Cat # 12-0112-82; RRID: AB_2734869</td>
</tr>
<tr>
<td>Anti-mouse CD11c APC/Cy7</td>
<td>BioLegend</td>
<td colspan="2">Cat # 117323; RRID: AB_830646</td>
</tr>
<tr>
<td>Anti-mouse CD11c APC</td>
<td>BioLegend</td>
<td colspan="2">Cat # 117310; RRID: AB_313779</td>
</tr>
<tr>
<td>Anti-mouse CD11c PE</td>
<td>BioLegend</td>
<td colspan="2">Cat # 117307; RRID: AB_313776</td>
</tr>
<tr>
<td>Anti-mouse CD3 PE/Cy7</td>
<td>BioLegend</td>
<td colspan="2">Cat # 100219; RRID: AB_1732068</td>
</tr>
<tr>
<td>Anti-mouse F4/80 BV785</td>
<td>BioLegend</td>
<td colspan="2">Cat # 123141; RRID: AB_2563667</td>
</tr>
<tr>
<td>Anti-mouse MHC II PE/Cy7</td>
<td>BioLegend</td>
<td colspan="2">Cat # 107630; RRID: AB_2069376</td>
</tr>
<tr>
<td>Anti-mouse MHC II FITC</td>
<td>BioLegend</td>
<td colspan="2">Cat # 107615; RRID: AB_493524</td>
</tr>
<tr>
<td>Anti-mouse CD45 BV605</td>
<td>BioLegend</td>
<td colspan="2">Cat # 103139; RRID: AB_2562341</td>
</tr>
<tr>
<td>Anti-mouse Ly6C PerCP/Cyanine5.5</td>
<td>BioLegend</td>
<td colspan="2">Cat # 128012; RRID: AB_1659241</td>
</tr>
<tr>
<td>Anti-mouse-TCRVα2 PE</td>
<td>BioLegend</td>
<td colspan="2">Cat # 127809; RRID: AB_1089251</td>
</tr>
<tr>
<td>TruStain FcXTM PLUS (anti-mouse CD16/32) Antibody</td>
<td>BioLegend</td>
<td colspan="2">Cat # 156603; RRID: AB_2783137</td>
</tr>
<tr>
<td colspan="4">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Ovalbumin from Egg White</td>
<td>BBI</td>
<td colspan="2">Cat # A003056-0100</td>
</tr>
<tr>
<td>PKH67 Green Fluorescent Cell Linker Midi Kit</td>
<td>Sigma</td>
<td colspan="2">Cat # MIDI67-1KT</td>
</tr>
<tr>
<td>CFSE</td>
<td>eBioscience</td>
<td colspan="2">Cat # 65-0850-84</td>
</tr>
<tr>
<td>Recombinant murine GM-CSF</td>
<td>Peprotech</td>
<td colspan="2">Cat # 315-03</td>
</tr>
<tr>
<td>Recombinant murine IL-4</td>
<td>Peprotech</td>
<td colspan="2">Cat # 214-14</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Millipore</td>
<td colspan="2">Cat # A3733</td>
</tr>
<tr>
<td>b-Mercaptoethanol</td>
<td>Gibco</td>
<td colspan="2">Cat # 21985032</td>
</tr>
<tr>
<td>2,2,2-Tribromo-ethanol</td>
<td>Sigma</td>
<td colspan="2">Cat # T48402</td>
</tr>
<tr>
<td>2-Methyl-2-butanol</td>
<td>Sigma</td>
<td colspan="2">Cat # 152463</td>
</tr>
<tr>
<td>Pierce ECL western blotting substrate</td>
<td>Thermo Fisher</td>
<td colspan="2">Cat # 32106</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td>Sigma</td>
<td colspan="2">Cat # P8340-1ML</td>
</tr>
<tr>
<td colspan="4">Critical commercial assays</td>
</tr>
<tr>
<td>MojoSort Mouse CD8 T cell Isolation Kit</td>
<td>BioLegend</td>
<td colspan="2">Cat # 480035</td>
</tr>
<tr>
<td>Mojo buffer</td>
<td>BioLegend</td>
<td colspan="2">Cat # 480017</td>
</tr>
<tr>
<td>10× RBC lysis buffer</td>
<td>eBioscience</td>
<td colspan="2">Cat # 00-4300-54</td>
</tr>
<tr>
<td>ExoAb Antibody Kit</td>
<td>SBI</td>
<td colspan="2">Cat # EXOAB-KIT-1</td>
</tr>
<tr>
<td>ExoQuick-TC</td>
<td>SBI</td>
<td colspan="2">Cat # EXOTC50A-1</td>
</tr>
<tr>
<td>EXOCET Exosome Quantitation Kit</td>
<td>SBI</td>
<td colspan="2">Cat # EXOCET96A-1</td>
</tr>
<tr>
<td>BCA Protein Assay Kit</td>
<td>Pierce</td>
<td colspan="2">Cat # 23225</td>
</tr>
<tr>
<td colspan="4">Experimental models: cell lines</td>
</tr>
<tr>
<td>MC38 lines</td>
<td>Laboratory of Yangxin Fu</td>
<td colspan="2">N/A</td>
</tr>
<tr>
<td colspan="4">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>C57BL/6</td>
<td>Vital River Laboratory Animal Technology</td>
<td colspan="2">Cat # 213</td>
</tr>
<tr>
<td>C57BL/6-Tg (TcraTcrb)1100Mjb/J</td>
<td>The Jackson Laboratory</td>
<td colspan="2">Cat # N000208</td>
</tr>
<tr>
<td colspan="4">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo</td>
<td>BD</td>
<td colspan="2">
<a href="https://www.flowjo.com/solutions/flowjo">https://www.flowjo.com/solutions/flowjo</a>;RRID: SCR_008520</td>
</tr>
<tr>
<td>GraphPad Prism 8</td>
<td>GraphPad Software</td>
<td colspan="2">
<a href="https://www.graphpad.com:443">https://www.graphpad.com:443</a>;RRID: SCR_002798</td>
</tr>
<tr>
<td>ImageJ</td>
<td>ImageJ public freeware</td>
<td colspan="2">
<a href="https://imagej.nih.gov/nih-image/index.html">https://imagej.nih.gov/nih-image/index.html</a>;RRID: SCR_003073</td>
</tr>
<tr>
<td colspan="3">Other</td>
<td> </td>
</tr>
<tr>
<td>Amicon Ultra-2 (100 kDa MWCO)</td>
<td>Merck Millipore</td>
<td colspan="2">Cat # UFC910024</td>
</tr>
<tr>
<td>70 μm cell strainers</td>
<td>Falcon</td>
<td colspan="2">Cat # 352350</td>
</tr>
<tr>
<td>Optima XPN-100 ultracentrifuge</td>
<td>Backman Coulter</td>
<td colspan="2">Cat # A99846</td>
</tr>
<tr>
<td>Allegra X-22R centrifuge</td>
<td>Backman Coulter</td>
<td colspan="2">Cat # 392188</td>
</tr>
<tr>
<td>Zetasizer Nano ZS</td>
<td>Malvern Instruments</td>
<td colspan="2">N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-subject-details">Materials and subject details</h2>
<h3 id="sec2.1">1× RIPA buffer</h3>
<p>For 100 mL 1×RIPA buffer:</p>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Add to 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tri-HCl pH 7.6</td>
<td>25 mM</td>
<td>302.85 mg</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>876.6 mg</td>
</tr>
<tr>
<td>NP-40</td>
<td>1%</td>
<td>1 mg</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>1%</td>
<td>1 mg</td>
</tr>
<tr>
<td>SDS</td>
<td>1%</td>
<td>1 mg</td>
</tr>
</tbody>
</table>
<p class="table-legend">Stored at −20°C for up to 3 months. Add protease inhibitors before use.</p>
<h3 id="sec2.2">1× TBS buffer</h3>
<p>For 1,000 mL 1×TBS buffer:</p>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Add to 1,000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tri-HCl pH 7.4</td>
<td>20 mM</td>
<td>2.4 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>8.7 g</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L, pH adjusts to 7.6</td>
</tr>
</tbody>
</table>
<p class="table-legend">Stored at 25°C for up to 3 months.</p>
<h3 id="sec2.3">ACK lysis buffer</h3>
<p>For 1 L 10×ACK lysis buffer:</p>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Add to 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>NH<sub>4</sub>Cl</td>
<td>1.55 M</td>
<td>82.9 g</td>
</tr>
<tr>
<td>KHCO<sub>3</sub>
</td>
<td>100 mM</td>
<td>10 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>-EDTA</td>
<td>0.5 M</td>
<td>0.372 g</td>
</tr>
<tr>
<td>NaOH 1 M</td>
<td>n/a</td>
<td>Adjust pH to 7.2</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<p class="table-legend">Stored at 4°C for up to 3 months. Dilute 10 times with ddH<sub>2</sub>O before use.</p>
<h3 id="sec2.4">CFSE solution</h3>
<p>To make 10 mM CFSE stock, dissolve 500 μg CFSE powder in 90 μL DMSO, store at −20°C for up to 6 months. For 10 μM CFSE working solution, add 1 μL CFSE stock into 1 mL PBS, store at 25°C, avoid light, and use immediately.</p>
<h3 id="sec2.5">BSA solution</h3>
<p>To make 10% BSA solution, dissolve 2 mg BSA powder in 20 mL ddH<sub>2</sub>O, Sterilize the solution with 0.22 μm filter and store at 4°C for up to 6 months.</p>
<h3 id="sec2.6">FACS buffer</h3>
<p>To prepare 100 mL FACS buffer, dilute 2 mL FBS (final concentration 2%, v/v) in 100 mL 1× PBS, mix well and store at 4°C for up to 1 month.</p>
<h3 id="sec2.7">Fc blocker buffer</h3>
<p>To prepare Fc blocking buffer for each sample, add 0.5 μL of anti-mouse CD16/32 antibody (<a href="#key-resources-table">Key resources table</a>) in 50 μL FACS buffer. Always prepare before use.</p>
<h3 id="sec2.8">Animals</h3>
<p>Female C57BL/6 mice (6–8 weeks old) were purchased from Vital River Laboratory Animal Technology (Beijing, China). OT-I T cell receptor-transgenic mice (6–8 weeks old, male or female) (C57BL/6-Tg (TcraTcrb)1100mjb) whose T cell receptors recognize ovalbumin (OVA) residues 257–264 in the context of H2Kb were obtained from the Jackson Laboratory (Bar Harbor, ME, USA). All animal experiments were performed according to the institutional ethical guidelines on animal care and the protocols used for this study were approved by the Animal Care and Use Committee at the Institute of Biophysics, Chinese Academy of Sciences.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Isolation and characterization of TDEs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 16 h</div>
<p>This step details how to isolate TDEs from tumor culture medium (TCM) (<a href="#fig1">Figure 1</a>A).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/503-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Isolation of exosomes from tumor culture medium</div>
<p>(A) Illustrative overview of exosome separation.</p>
<p>(B) 100 kDa MWCO to separate &gt; 100 kDa fraction from tumor culture medium.</p>
<p>(C) The exosome pellet is brown in appearance.</p>
<p>(D) Dynamic light scattering analysis of exosomes. PdI, polydispersity index.</p>
<p>(E) Western blot workflow.</p>
</figcaption>
</figure>
<ol>
<li>Culture tumor cells in RPMI 1640 media supplemented with 10% exosome-depleted FBS. Collect supernatant of tumor cells after 48 h when cells reach 80% confluency.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this protocol, MC38 mouse colon cancer cell line is used.</div>
<ol start="2">
<li>Remove cell debris from tumor culture medium (typically 20 mL) by centrifuging at 3,000 × <i>g</i> for 2 h at 4°C. Then transfer supernatant to a sterile 100 kDa MWCO (Molecular Weight Cut Off) (<a href="#fig1">Figure 1</a>B), and centrifuge at 3,000 × <i>g</i> for 30 min at 4°C to discard the faction &lt; 100 kDa, and finally make the concentrated-supernatant (usually 1.5 mL).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Concentrated-supernatant can be stored at −80°C for up to 6 months until future use.</div>
<div class="note">
<span class="note-title">Note:</span> An additional 10,000 × <i>g</i> centrifugation after this step would further increase the purity of exosomes isolated.</div>
<ol start="3">
<li>Isolation of exosomes by ExoQuick-TC (EXOTC50A-1, SBI).
<div class="note">
<span class="note-title">Note:</span> ExoQuick-TC from SBI is a proprietary polymer (PEG, polyethylene glycol) that gently precipitates exosomes between 30 and 200 nm in size from tissue culture media, urine, or spinal fluid. The detailed mechanism of this method can be found in SBI user manual: <a href="https://systembio.com/wp-content/uploads/MANUAL_EXOTCXXA-1-1.pdf">https://systembio.com/wp-content/uploads/MANUAL_EXOTCXXA-1-1.pdf</a>.</div>
<ol type="a">
<li>Transfer concentrated tumor culture supernatant to a sterile vessel and add 1/5 volume of ExoQuick-TC. Mix well by inverting the tube.</li>
<li>Incubation the ExoQuick-TC/supernatant mixture 16–20 h at 4°C. The tubes should not be rotated or mixed during the incubation period and must remain upright.
<div class="note">
<span class="note-title">Note:</span> Do not vortex or rotate the ExoQuick-TC/supernatant mixture, as it is recommended for PEG-based precipitation of exosomes.</div>
</li>
<li>Centrifuge the ExoQuick-TC/supernatant mixture at 3,000 × <i>g</i> for 30 min. Centrifugation may be performed at either 25°C or 4°C with similar results. After centrifugation, exosomes may appear as a brown pellet at the bottom of vessel (<a href="#fig1">Figure 1</a>C).</li>
<li>Discard the supernatant and centrifuge again at 3,000 × <i>g</i> for 5 min to remove the residual fluid.</li>
<li>Resuspend the pellet with about 200 μL 1×PBS to make exosome solution.</li>
<li>Exosome may be quantified by total protein quantification using BCA protein assay kit, or by EXOCET Exosome Quantitation Kit.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Exosomes can be characterized as below and used immediately or stored at −80°C for 3 months.</div>
<div class="note">
<span class="note-title">Note:</span> Do not vortex but use pipette to handle exosome solution. Avoid repeatedly freeze and thaw exosomes. The diameter of exosomes ranges 40–60 nm accessed by dynamic light scattering (DLS) (<a href="#fig1">Figure 1</a>D) and the yield typically ranges 0.5–1 mg exosomes per 1×10<sup>6</sup> cells based on BCA protein quantification method. Quality control assays of exosomes are routinely performed by DLS and BCA methods during −80°C storage. Within 3 months, the integrity, mean diameter, particle distribution, and concentration of exosomes remain nearly the same as fresh isolation.</div>
<ol start="4">
<li>Staining and imaging of exosome by transmission electron microscopy (TEM)
<ol type="a">
<li>Fix purified exosomes with 1 mL of 2% paraformaldehyde for 5 min.</li>
<li>Glow-discharge the thin formvar/carbon film coated grids for 1 min.</li>
<li>Load 5–7 μL exosome suspension (containing 50 μg exosomes) solution on the grid and incubate for 1 min.
<div class="note">
<span class="note-title">Note:</span> If the concentration of exosome is too high, dilute it to 1/10–1/20.</div>
</li>
<li>Stain with filtered 1% uranyl acetate solution on the surface of grid.</li>
<li>Remove the excess uranyl acetate with filter paper.</li>
<li>Rinse the grid with drop of water to remove the excess staining solution.</li>
<li>Dry for 10 min at 25°C.</li>
<li>Store grid in a box for future observation by TEM at 80 kV (<a href="#fig5">Figure 5</a>A).</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Detection of markers of exosome by western blot (<a href="#fig1">Figure 1</a>E)
<ol type="a">
<li>Resuspend 0.2 mg exosome pellet in 200 μL RIPA buffer (with protease inhibitor added) and vortex 15 s.</li>
<li>Place at ice for 5 min to allow complete lysis.</li>
<li>Add 2×SDS buffer and heat at 95°C for 5 min. Chilled on ice before loading onto gel.</li>
<li>Perform standard SDS-PAGE electrophoresis and western transfer onto PVDF.</li>
<li>Block with 5% milk in Tris buffered saline + 0.05% Tween (TBST) for 1 h.</li>
<li>Incubate blot 16–20 h at 4°C with exosome specific primary antibody at 1:1,000 dilution (5% milk in TBST).</li>
<li>Wash 3 times with TBST.</li>
<li>Incubate 1 h at 25°C with secondary antibody at 1:1,500 dilution (5% milk in TBST).</li>
<li>Wash 3 times with TBST.</li>
<li>Incubate blot with western blotting substrate and visualize on film.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is important to note that not all cells produce exosomes with the same composition of tetraspanin markers, therefore not all the exosomal marker proteins (e.g., ALIX, TSG101, CD63, CD9, CD81, etc) could be definitely detected in your samples. However, it is strongly recommended to use some intracellular organelle markers to exclude contaminations of Golgi bodies (GM130), mitochondria (Cytochrome C) or nucleus (Histone H3). In our study, GM130, Cytochrome C, and Histone H3 was excluded to make sure the purity of exosomes(<a href="#bib6">Yin et al., 2020</a>).</div>
<h3 id="sec3.2">Generation of TDE-treated DCs <i>in vitro</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days</div>
<p>This step details how to generate bone marrow-derived dendritic cells (BMDCs) (<a href="#fig2">Figure 2</a>A).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/503-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Generation of BMDC</div>
<p>(A) Overview of BMDC generation.</p>
<p>(B) Flush out the bone marrow into a 10 cm cell culture dish with a syringe plunge.</p>
<p>(C) Morphology of BMDC on day 6 (a), and firmly adherent macrophages are remained (b).</p>
<p>(D) Flow cytometry analysis of phenotype of BMDC (MHC II<sup>+</sup>CD11c<sup>+</sup>).</p>
</figcaption>
</figure>
<ol start="6">
<li>Euthanize the C57BL/6 mice (about 8–10 weeks of age), spray the mice with 75% ethanol and fix the mice in a supine position.</li>
<li>Isolate the entire leg bones, carefully remove the connective tissue and muscle.</li>
<li>Gently cut down the condyles of leg bones.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Be careful not to break the bones.</div>
<ol start="9">
<li>Infuse the bone cavity with 1 mL RPMI medium (without FBS), and flush out the bone marrow into a 10 cm cell culture dish (<a href="#fig2">Figure 2</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Bone cavity can be flushed multiple times until all bone marrow is collected and the bones turn to white.</div>
<ol start="10">
<li>The cells from bone marrow are pass through a 70-μm cell strainer to prepare single-cell suspension, and centrifuge at 500–600 × <i>g</i> for 5 min.</li>
<li>Discard the supernatant, resuspend the cells in 2 mL of 1×ACK lysis buffer and incubate for 1 min at 25°C to eliminate red blood cells.</li>
<li>Add 20 mL of RPMI with 10% FBS to the cells and re-centrifuge at 500–600 × <i>g</i> for 5 min to terminate ACK lysis process.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After this step, use of lymphocyte gradient separation buffer is recommended to further ensure the purity of DCs.</div>
<ol start="13">
<li>After wash, resuspend and count the cells, then adjust the cell suspension to 3 × 10<sup>5</sup> cells/mL with RPMI medium (10% FBS, 55 μM β-mercaptoethanol and 20 ng/mL rmGM-CSF plus 5 ng/mL rmIL-4).</li>
<li>Add 10 mL of cell suspension to a 10 cm cell culture dish and put in cell culture incubator.</li>
<li>Three days later, replenish 10 mL RPMI medium (10% FBS, 55 μM β-mercaptoethanol and 20 ng/mL rmGM-CSF plus 5 ng/mL rmIL-4) to the cells.</li>
<li>6 days after differentiation, collect cells for further use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When harvesting, only cells in suspension and loosely adherent cells should be isolated for further use and this protocol can render about 6–8 × 10<sup>7</sup> DCs in sum.</div>
<ol start="17">
<li>Analyze the phenotype of DC by microscope and flow cytometry (MHC II<sup>+</sup>CD11c<sup>+</sup>) (<a href="#fig2">Figures 2</a>C and 2D).
<ol type="a">
<li>Discard the supernatant, and resuspend the cells with 50 μL Fc blocker buffer and mix well.</li>
<li>Incubate the cells at 25°C for 5–10 min.</li>
<li>Add 1/100 anti-mouse CD11c APC antibody and 1/200 anti-mouse MHC II FITC to each tube and mix well.</li>
<li>Incubate the cells at 4°C for 20 min.</li>
<li>After antibody incubation, neutralize the reaction by adding extra 1 mL FACS (1×PBS with 1% FBS) buffer in each tube.</li>
<li>Centrifuge the cells at 500–600 × <i>g</i> for 5 min at 4°C, discard the supernatant, and resuspend the cells for FACS analysis with 1/1000 DAPI for separating live cells.</li>
</ol>
</li>
<li>Treat BMDCs with 400 μg/mL TDEs for 48 h for further experiments.</li>
</ol>
<h3 id="sec3.3">Generation of TDE-treated DCs <i>ex vivo</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 48 h</div>
<p>This step details how to label TDEs and isolate of TDE-treated DCs in lymph nodes (LNs).</p>
<ol start="19">
<li>Exosome labeling.
<ol type="a">
<li>Start with freshly isolated exosome pellets.</li>
<li>Add 500 μL Diluent C from PKH67 kit to resuspend exosome pellets.</li>
<li>Add 3 μL PKH67 dye into another 500 μL Diluent C tube.</li>
<li>Mix b and c continuously at 25°C for 5 min.</li>
<li>Neutralize the mixture by adding 1 mL 10% sterile BSA solution.</li>
<li>Wash the PKH67 labeled exosomes 2 times with 1× sterile PBS.</li>
<li>Collect the PKH67-labeled TDEs by ultracentrifugation (100,000 × <i>g</i>, 30 min, 25°C), and resuspend the exosomes in 1× sterile PBS.</li>
<li>Treat the BMDCs with 400 μg/mL PKH67-labeled TDEs at 37°C for 6 h.</li>
<li>Uptake of PKH67-labeled TDEs by BMDCs can be analyzed by Confocal microscope (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/503-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Isolation of TDE-treated DC <i>ex vivo</i>
</div>
<p>(A) Uptake of PKH67 labeled exosomes by BMDC.</p>
<p>(B) Injection of PKH67 labeled exosomes into the mice footpad.</p>
<p>(C) Location of popliteal lymph nodes and inguinal lymph nodes of mice.</p>
<p>(D) Flow cytometry and gating strategy of PKH67-uptaken DCs.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> PKH67 labeled TDEs can be used immediately or stored at 4°C for 1 week.</div>
<ol start="20">
<li>Inject 1,000 μg PKH67 labeled TDEs (in 50 μL 1×PBS) into the footpad of C57BL/6 mice under tribromo-ethanol anesthesia (<a href="#fig3">Figure 3</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 0.5 mL tribromo-ethanol per mouse by intraperitoneal injection is used for anesthetization.</div>
<ol start="21">
<li>Isolate DCs in popliteal lymph nodes and inguinal lymph nodes at 48 h post injection (<a href="#fig3">Figure 3</a>C).
<ol type="a">
<li>Place a 70-μm cell strainer on top of a 50 mL centrifuge tube.</li>
<li>Isolate the lymph nodes of mice, and put the lymph nodes in the cell strainer and grind with a syringe plunger, then flush the cells with 10 mL RPMI1640 (without FBS).</li>
<li>Centrifuge the cell suspension at 500–600 × <i>g</i> for 5 min, and discard the supernatant.</li>
<li>Resuspend the cell pellet with PBS (1% FBS) for staining.</li>
<li>Discard the supernatant, and resuspend the cells with 50 μL Fc blocker buffer and mix well.</li>
<li>Incubate the cells at 25°C for 5–10 min.</li>
<li>Add antibody as “Antibody Mix Preparation-3” to each tube and mix well.</li>
<li>Incubate the cells at 4°C for 20 min.</li>
<li>After antibody incubation, neutralize the reaction by adding extra 1 mL FACS buffer in each tube.</li>
<li>Centrifuge the cells at 500–600 × <i>g</i> for 5 min at 4°C, discard the supernatant, and resuspend the cells for FACS analysis with 1/1000 DAPI (0.1 μg/mL) for separating live cells.</li>
<li>Collect the DCs (CD45<sup>+</sup>Ly6C<sup>−</sup>MHC II<sup>+</sup>F4/80<sup>−</sup>CD11c<sup>+</sup>PKH67<sup>+</sup>) by flow cytometer (<a href="#fig3">Figure 3</a>D) for further experiments.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> B220 and CD14 is highly recommended to be included in the ex vivo isolation of DCs to further eliminate B220<sup>+</sup> B cells and CD14<sup>high</sup> monocytes.</div>
<h3 id="sec3.4">Isolation CD8<sup>+</sup> T cells and CFSE labeling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 70–90 min</div>
<p>This step details how to anesthetize and sacrifice the mice, and prepare CFSE-labeled OTI CD8<sup>+</sup> T cells.</p>
<ol start="22">
<li>Terminally anesthetize OTI transgenic mice by cervical dislocation.</li>
<li>Take out the spleen using tissue scissors and tissue forceps.
<ol type="a">
<li>Put the spleen on a 70 μm cell strainer on the top of a 50 mL centrifuge tube.</li>
<li>Grind the spleen with a syringe plunger while flushing the strainer with 10 mL RPMI 1640 medium (without FBS) (<a href="#fig4">Figure 4</a>A).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/503-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Isolation of CD8<sup>+</sup> T cell</div>
<p>(A) Grind the spleen on the 70 μm cell strainer with a syringe plunger.</p>
<p>(B) Flow cytometry analysis of phenotype of OT I CD8<sup>+</sup> T cell (TCR Vα 2<sup>+</sup>CD8<sup>+</sup>).</p>
</figcaption>
</figure>
</li>
<li>Centrifuge the cell suspension at 500–600 × <i>g</i> for 5 min at 4°C once, and discard the supernatant.</li>
<li>Resuspend the cells with 3 mL 1×RBC lysis buffer and let the process last for 3–5 min with intermittent swirling.</li>
<li>Neutralize the RBC lysis process by adding 30 mL RPMI 1640 medium (or HBSS) (10 times volume of RBC lysis buffer added in the above step) and mix the cells well followed by passing through a 70 μm cell strainer again to exclude the cell aggregates or clots.</li>
<li>Repeat step c and discard the supernatant to get splenocytes.</li>
</ol>
</li>
<li>To isolate OT I CD8<sup>+</sup> T cells
<ol type="a">
<li>Resuspend the splenocytes with 1×Mojo buffer to make sure the cell density is between 1×10<sup>7</sup>–1×10<sup>8</sup> cells per mL (usually 0.4 mL buffer per spleen is sufficient based on our experience), and mix well by pipetting several times followed by transferring the cells into a new sterile 5 mL round bottom tube (BD Falcon).</li>
<li>Add 10 μL Biotin-conjugated antibody cocktail per 100 μL (0.1 mL) cells (from spleen) and mix well. Then incubate the cells with antibodies on ice for 15 min.</li>
<li>Add 10 μL nano-beads per 100 μL (0.1 mL) cells (the same volume used as the antibody cocktail). Then incubate the cells with antibodies on ice for 15 min.</li>
<li>Add another 2 mL 1×Mojo buffer in the tube and put the tube in the magnate to separate CD8<sup>+</sup> T cells from other cell compartments by standing on the bench for 5 min at 25°C.</li>
<li>Hold the magnate to pour out the un-attached cells and collect them in a 15 mL conical centrifuge tube.</li>
<li>Centrifuge the cell suspension at 500–600 × <i>g</i> for 5 min at 4°C and discard the supernatant. Generally, 1–2×10<sup>7</sup> OT I CD8<sup>+</sup> T cells can be isolated from a single mouse.</li>
<li>FACS analyze the phenotype of isolated CD8<sup>+</sup> OT I T cells (TCR Vα 2<sup>+</sup>CD8<sup>+</sup>) (<a href="#fig4">Figure 4</a>B).</li>
<li>Discard the supernatant, and resuspend the cells with 50 μL Fc blocker buffer and mix well.</li>
<li>Incubate the cells at 25°C for 5–10 min.</li>
<li>Add 1/100 anti-mouse CD8a APC antibody and 1/100 anti-mouse TCRVa2 PE to each tube and mix well.</li>
<li>Incubate the cells at 4°C for 20 min.</li>
<li>After antibody incubation, neutralize the reaction by adding extra 1 mL FACS buffer in each tube.</li>
<li>Centrifuge the cells at 500–600 × <i>g</i> for 5 min at 4°C, discard the supernatant, and resuspend the cells for FACS analysis with 1/1000 DAPI (0.1 μg/mL) for separating live cells.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> it is recommended to leave 5%–10% of the cells as non-CFSE-labeled cells control for further FACS application.</div>
<ol start="25">
<li>CFSE labeling
<ol type="a">
<li>Resuspend the cells with 10 μM CFSE solution (diluted in 1×PBS) and mix well followed by incubation at 25°C for 10 min in the dark.</li>
<li>After incubation, neutralize the CFSE labeling process by adding 0.2–0.5 mL FBS in the tube.</li>
<li>Centrifuge the cells at 500–600 × <i>g</i> for 5 min at 4°C, and discard the supernatant.</li>
<li>Resuspend the cells by RPMI 1640 medium added 10% FBS (BI) and 55 μM β-mercaptoethanol and count the cells.</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Co-culture to analyze T cell proliferation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 days</div>
<p>This step details the co-culture of TDE-treated DCs with CFSE-labeled OT I CD8<sup>+</sup> T cells and how to analyze T cell proliferation.</p>
<ol start="26">
<li>Collect TDE-treated DCs as described in step 18 and step 21, and treat the 1×10<sup>6</sup> DCs with 2 mg/mL OVA antigen for 48 h (applicable for TDE-treated DCs both from <i>in vitro</i> or <i>ex vivo</i>).</li>
<li>After treatment, collect and wash the cells by centrifuging at 500–600 × <i>g</i> for 5 min at 4°C to eliminate residual OVA antigen.</li>
<li>Discard the supernatant and resuspend the cells with proper volume (200~400 μL) of 1640 medium added 10% FBS (BI) and 55 μM β-mercaptoethanol (Gibco) and count the DCs.</li>
<li>Co-culture the DCs (step 27) and CFSE-labeled OTI CD8<sup>+</sup> T cells (step 24) at the ratio between 1:5 to 1:8.</li>
<li>4 days later collect and centrifuge the cells at 500–600 × <i>g</i> for 5 min at 4°C.</li>
</ol>
<ol start="31">
<li>FACS analyze the proliferation of OT I CD8<sup>+</sup> T cells (<a href="#fig5">Figure 5</a>C).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/503-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Immunosuppressive effect of TDE on DCs</div>
<p>(A) Morphology of TDEs were analyzed by TEM. Scale bar, 100 nm.</p>
<p>(B) Quantify exosomes by EXOCET Exosome Quantitation Kit and BCA method.</p>
<p>(C) TDE-treated BMDCs or lymph node DCs, loaded with 2 mg/mL OVA, followed by antigen removal, and co-culture with OT I CD8<sup>+</sup> T cells for 3 days. Proliferation was analyzed by flow cytometry. Neg CTR, undivided OT I CD8<sup>+</sup> T cells. Pos CTR, OT I CD8<sup>+</sup> T cell were treated with OVA<sub>250-264</sub>, as a positive control. Untreated, DCs without TDE treatment were co-cultured with OT I CD8<sup>+</sup> T cells. TDE-treated, DCs with TDE treatment were co-cultured with OT I CD8<sup>+</sup> T cells.</p>
</figcaption>
</figure>
<ol type="a">
<li>Discard the supernatant, and resuspend the cells with 50 μL Fc blocker buffer and mix well.</li>
<li>Incubate the cells at 25°C for 5–10 min.</li>
<li>Add 1/100 anti-mouse CD8 APC antibody, 1/100 anti-mouse CD11c PE and 1/200 anti-mouse CD3 PE/Cy7 to each tube and mix well.
<div class="note">
<span class="note-title">Note:</span> Staining with CD11c can exclude CD8<sup>+</sup> DCs from CD8<sup>+</sup> T cells.</div>
</li>
<li>Incubate the cells at 4°C for 20 min.</li>
<li>After antibody incubation, neutralize the reaction by adding extra 1 mL FACS buffer in each tube.</li>
<li>Centrifuge the cells at 500–600 × <i>g</i> for 5 min at 4°C, discard the supernatant, and resuspend the cells for FACS analysis.</li>
<li>CFSE<sup>low</sup> CD11c<sup>−</sup>CD3<sup>+</sup>CD8<sup>+</sup> population is considered as proliferating OT I CD8<sup>+</sup> T cells.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Following this protocol, we isolated exosomes from tumor culture medium, and evaluate the immunosuppressive function of tumor-derived exosomes on DC <i>in vitro</i> and <i>ex vivo</i>.</p>
<p>Morphologies of the purified exosomes were characterized by transmission electron microscopy (TEM; <a href="#fig5">Figure 5</a>A), showing a mean diameter of ~40 nm, consistent with the size of previous reports (<a href="#bib4">Kalluri, 2016</a>; <a href="#bib5">Kalluri and LeBleu, 2020</a>). Western blot analysis revealed the presence of exosome-enriched proteins (e.g., ALIX, HSP70, TSG101, CD9, and CD81)(<a href="#bib2">Hessvik and Llorente, 2017</a>; <a href="#bib3">Huang-Doran et al., 2017</a>)(Data as shown in (<a href="#bib6">Yin et al., 2020</a>)). In our experience, we also calculated the amount of exosomes by EXOCET Exosome Quantitation Kit and found that the number of exosomes were highly correlated with the protein content by BCA (<a href="#fig5">Figure 5</a>B). Thus, these two methods may be used to quantify exosome.</p>
<p>Isolated TDEs were used to treat BMDCs <i>in vitro</i> or lymph node DCs <i>ex vivo</i>, then we analyzed proliferation of OT I CD8<sup>+</sup> T cell to identify the function of TDE-treated DCs. As the data shown, compared with untreated DCs, TDE-treated DCs had a defective T cell priming capability (<a href="#fig5">Figure 5</a>C). The dysfunctional DCs was the result of the uptake of TDE-derived FAs. For full details, we refer to (<a href="#bib6">Yin et al., 2020</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Low yield and low purity of BMDCs.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Increase the number of leg bones used to differentiate BMDCs according to your experiment needs. And the purity of GM-CSF differentiated BMDC is strongly recommended to be tested every time before use, since low purity of BMDCs may lead to inaccurate experiment outcomes.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>The effect of tumor-derived exosomes (TDEs) on BMDCs is unreproducible.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>If you find (in some rare cases) TDEs-treated BMDCs failed to or are less able to induce evident T cell suppression. You could</p>
<ol>
<li>Re-extract exosomes using fresh extraction kit/reagent.</li>
<li>Check the status of parent tumor cells.</li>
<li>Make sure exosome-depleted FBS is used during exosomes collection.</li>
<li>Do the quality control of TDEs every time before use to make sure the isolation of TDEs is successful by performing TEM or WB assays described in steps 4 and 5.</li>
</ol>
<h3 id="sec5.5">Possible reasons</h3>
<p>Based on our experience, exosomes from different parent cells may have different impact on DC function. Exosomes can be taken as an important FFAs carrier. Thus, the quality and species of FFAs encapsulated in exosome may vary due to the type and status of parent cells (<a href="#bib1">Haraszti et al., 2016</a>). For example, FFAs content in non-cancerous cells (NIH-3T3) derived exosomes is far less than that in tumor cells derived exosomes (<a href="#bib6">Yin et al., 2020</a>).</p>
<h3 id="sec5.6">Problem 3</h3>
<p>CFSE labeling associated toxicity</p>
<h3 id="sec5.7">Potential solution</h3>
<p>Resuspend purified CD8<sup>+</sup> T cells with 10 μM ready-to-use CFSE solution (in 1× PBS), rather than adding 10 mM CFSE stock solution in suspended CD8<sup>+</sup> T cells, can avoid CFSE labeling associated toxicity to a great extent. And 10 min incubation time at 25°C is enough, while exceeding the incubation time may increase T cell death.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Haraszti, R.A., Didiot, M.C., Sapp, E., Leszyk, J., Shaffer, S.A., Rockwell, H.E., Gao, F., Narain, N.R., DiFiglia, M., Kiebish, M.A., et al. (2016). High-resolution proteomic and lipidomic analysis of exosomes and microvesicles from different cell sources. J Extracell Vesicles <i>5</i>, 32570. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00068-X/sref1">View at publisher</a></p>
<p id="bib2">Hessvik, N.P. and Llorente, A. (2017). Current knowledge on exosome biogenesis and release. Cell Mol Life Sci <i>75</i>, 193-208. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00068-X/sref2">View at publisher</a></p>
<p id="bib3">Huang-Doran, I., Zhang, C.Y., and Vidal-Puig, A. (2017). Extracellular vesicles: novel mediators of cell communication in metabolic disease. Trends Endocrinol Metab <i>28</i>, 3-18. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00068-X/sref3">View at publisher</a></p>
<p id="bib4">Kalluri, R. (2016). The biology and function of exosomes in cancer. J Clin Invest <i>126</i>, 1208-1215. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00068-X/sref4">View at publisher</a></p>
<p id="bib5">Kalluri, R. and LeBleu, V.S. (2020). The biology, function, and biomedical applications of exosomes. Science <i>367</i>, eaau6977. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00068-X/sref5">View at publisher</a></p>
<p id="bib6">Yin, X., Zeng, W., Wu, B., Wang, L., Wang, Z., Tian, H., Wang, L., Jiang, Y., Clay, R., Wei, X., et al. (2020). PPARalpha inhibition overcomes tumor-derived exosomal lipid-induced dendritic cell dysfunction. Cell Rep <i>33</i>, 108278. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00068-X/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Lingtao Jin (<a href="mailto:LJIN1@ufl.edu">LJIN1@ufl.edu</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate any unique materials or reagents.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate new unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work is supported by the <a href="https://doi.org/10.13039/501100001809">National Natural Sciences Foundation of China</a> (81702823 to W.Z.), NIH/<a href="https://doi.org/10.13039/100000054">NCI</a> R37 award CA249305 (to L.J.), <a href="https://doi.org/10.13039/100000005">DoD</a> LCRP Career Development Award W81XWH2010309 (to L.J.), 2019 <a href="https://doi.org/10.13039/100001006">Breast Cancer Research Foundation</a>-AACR Career Development Award for Translational Breast Cancer Research 19-20-26-JIN (to L.J.), <a href="https://doi.org/10.13039/100001287">Elsa U. Pardee Foundation</a> pilot grant (to L.J.), <a href="https://doi.org/10.13039/100007038">Lung Cancer Research Foundation</a> pilot grant (to L.J.), and UF Health Cancer Center pilot grant (to L.J.).</p>
<h3>Author Contributions</h3>
<p>W.Z., X.Y., and Y.J. conducted the experiments, interpreted the data, and wrote the paper; L.J. and W.L. supervised the study and polished the paper.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>