<article>
<section>
<h1 id="header">A genotype-phenotype screening system using conditionally immortalized immature dendritic cells</h1>
<p><time datetime="2021-08-13">Published: August 13, 2021</time></p>
<p>Liwei Zhao,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">11</a></sup> Peng Liu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">11</a>,<a href="#fn2">12</a>,<a href="#cor1">*</a></sup> Wei Xie,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Shuai Zhang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Sebastian Thieme,<sup><a href="#aff4">4</a></sup> Laurence Zitvogel,<sup><a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#aff7">7</a>,<a href="#aff8">8</a></sup> Guido Kroemer,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff8">8</a>,<a href="#aff9">9</a>,<a href="#aff10">10</a>,<a href="#cor2">**</a></sup> and Oliver Kepp<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn3">13</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Equipe labellisée par la Ligue contre le cancer, Université de Paris, Sorbonne Université, INSERM UMR1138, Centre de Recherche des Cordeliers, Paris, France</p>
<p id="aff2"><sup>2</sup>Metabolomics and Cell Biology Platforms, Gustave Roussy, Villejuif, France</p>
<p id="aff3"><sup>3</sup>Université Paris Sud, Paris Saclay, Faculty of Medicine, Kremlin Bicêtre, France</p>
<p id="aff4"><sup>4</sup>Department of Pediatrics, University Clinic ‘Carl Gustav Carus’ Dresden, Dresden, Germany</p>
<p id="aff5"><sup>5</sup>Gustave Roussy Cancer Campus, Villejuif, France</p>
<p id="aff6"><sup>6</sup>Institut National de la Santé et de la Recherche Médicale, U1015, Gustave Roussy, Villejuif, France</p>
<p id="aff7"><sup>7</sup>Center of clinical investigations BIOTHERIS, Gustave Roussy, Villejuif, France</p>
<p id="aff8"><sup>8</sup>Suzhou Institute for Systems Medicine, Chinese Academy of Medical Sciences, Suzhou, China</p>
<p id="aff9"><sup>9</sup>Pôle de Biologie, Hôpital Européen Georges Pompidou, AP-HP, Paris, France</p>
<p id="aff10"><sup>10</sup>Karolinska Institute, Department of Women's and Children's Health, Karolinska University Hospital, Stockholm, Sweden</p>
<p id="fn1"><sup>11</sup>These authors contributed equally</p>
<p id="fn2"><sup>12</sup>Technical contact</p>
<p id="fn3"><sup>13</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kunzhiling@gmail.com">kunzhiling@gmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:kroemer@orange.fr">kroemer@orange.fr</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:captain.olsen@gmail.com">captain.olsen@gmail.com</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100732">10.1016/j.xpro.2021.100732</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we describe a protocol for CRISPR/Cas9-mediated gene knockout in conditionally immortalized immature dendritic cells (DCs), which can be limitlessly expanded before differentiation. This facilitates the genetic screening of DC functions <i>in vitro</i> including assessment of phagocytosis, cytokine production, expression of co-stimulatory or co-inhibitory molecules, and antigen presentation, as well as evaluation of the capacity to elicit anticancer immune responses <i>in vivo</i>. Altogether, these approaches described in this protocol allow investigators to link the genotype of DCs to their phenotype.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib18">Le Naour et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Conditionally immortalized immature dendritic cells (DCs) can be expanded without limits</li>
<li>A CRISPR/Cas9 system allows for genetic screening of DC functions</li>
<li>Different DC functions are assessed in vitro</li>
<li>DC genotype-dependent anticancer immunity can be determined in mice</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps for using synthetic CRISPR RNAs (crRNAs) from Horizon Discovery. However, we have also used this protocol with synthetic crRNAs and lentiviral single guide RNAs (sgRNAs) from other providers. Refer to the <a href="#key-resources-table">key resources table</a> for a complete list of reagents and tools, and to <a href="#materials-and-equipment">materials and equipment</a> for details on reagent preparation. Before you begin, review the key steps of the protocol.</p>
<div class="note">
<span class="note-title">Note:</span> The current protocol can be realized in a standard cell biology laboratory equipped with apparatuses for cell culture (bench-top-centrifuge, laminar flowhood, cell culture incubator, phase contrast microscope), and assessment (ELISA reader, cytofluorometer), with access to a biosafety level 2 laboratory and a standard animal facility. All experiments can be done by a single experienced experimenter.</div>
<h3 id="sec1.1">Preparation of cell cultures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [10 days]</div>
<ol>
<li>Prepare the cell culture media for each cell line as described in the “<a href="#materials-and-equipment">materials and equipment</a>” section.
<ol type="a">
<li>For the inducible immortalized dendritic cells (iniDC), use basic DC culture medium (BC medium) prewarmed to 37°C. For each T175 flask, aliquot 50 mL BC medium in a falcon tube, add 50 μL β-mercaptoethanol (1000×), 50 μL doxycycline / dexamethasone (Dex/Dox, 1000×), and 10 μL recombinant GM-CSF (50 μg/mL) to obtain complete DC growth medium (CG medium) ready to use.</li>
<li>For B3Z cells, aliquot 50 mL prewarmed BC medium in a falcon tube and add 50 μL β-mercaptoethanol to each T175 flask.</li>
<li>For MCA205 cells, use 30 mL prewarmed DMEM-based complete growth medium (D-BC medium) for each T175 flask.</li>
</ol>
</li>
<li>Thaw frozen cells in cryotubes in a 37°C water bath, add thawed cell suspension to a new tube of prewarmed media (according to the cell line) and immediately spin for 5 min at 500 <i>g</i> at room temperature (20°C–22°C). Aspirate the supernatant and resuspend the cell pellet in media (prepared in the previous step).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The iniDC cell line can be provided by S.T. upon request. MCA205 cells are commercially available, detailed information can be found in the <a href="#key-resources-table">key resources table</a>. All frozen cell lines are initially prepared with 1.5-2<b>×</b>10<sup>7</sup> viable cells/cryotube.</div>
<ol start="3">
<li>Homogenize the cellular suspension by pipetting up-and-down several times, and transfer to T175 flasks to be incubated in a cell culture incubator that maintains 37°C, 5% CO<sub>2</sub>, and 95% humidity.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Viability and proliferation can vary between cell lines and might be influenced by storage duration. Typically, fresh iniDC and B3Z cell cultures exhibit apparently reduced viability and proliferation and it is important to monitor cell cultures daily.</div>
<ol start="4">
<li>When initial cultures are established (about 3–4 days for the iniDCs, 1–3 days for B3Z, and 1–2 days for MCA205) at 70–80% confluency, sub-culturing can be performed, but should not exceed 10–15 passages:
<ol type="a">
<li>For iniDCs, viable cells can be both adherent and in suspension. Collect 40 mL of supernatant into a 50 mL falcon tube, remove remaining medium and add 10 mL/flask of prewarmed TrypLE™ Express for 5 min to detach adherent cells (no need to wash). Gently agitate the flask to confirm visually that all cells are detached, then stop the trypsinization using the original supernatant, transfer all content into the 50 mL falcon tube, spin at 500 <i>g</i> at room temperature (20°C–22°C) to obtain a cell pellet. Resuspend the pellet with 5 mL prewarmed CG medium, and use 1 mL cell suspension with 50 mL of CG medium for each sub-culture in one T175 flask, i.e., splitting ratio 1:5. The sub-culture will typically reach 70%–80% confluency in 3–4 days, yielding 3–4 × 10<sup>7</sup> viable cells/flask.</li>
<li>B3Z is a suspension cell line and proliferates rapidly, monitor the change in color of the phenol red-containing cell culture media daily and split the culture immediately when the medium acidifies and becomes yellow, which indicates that cellular confluency has reached about 80%. Gently agitate the flask to detach loosely adherent cells, transfer the cell suspension into a 50 mL falcon tube, spin at 500 <i>g</i> at room temperature to obtain a cell pellet. Resuspend the pellet with 5 mL of prewarmed CG medium, and use 0.5 mL cell suspension with 50 mL of β-mercaptoethanol complemented BC medium for each sub-culture in T175 flask, i.e., a ratio 1:10. The sub-culture will typically reach 70%–80% confluency within 2 days, yielding 4–5 × 10<sup>7</sup> viable cells/flask.</li>
<li>All viable MCA205 cells are adherent. Aspirate the medium with a vacuum pump, wash the cell culture layer with 20 mL prewarmed PBS, and add 10 mL/flask of prewarmed TrypLE™ Express for 1 min to detach adherent cells. Gently agitate the flask to confirm cellular detachment, then stop the trypsinization with 10 mL prewarmed D-BC medium, transfer all content into a 50 mL falcon tube and spin at 500 <i>g</i> at room temperature to obtain a cell pellet. Resuspend the pellet with 5 mL prewarmed D-BC medium, and use 0.5 mL cell suspension with 30 mL of D-BC medium for each sub-culture in one T175 flask, i.e., splitting ratio 1:10. The sub-culture will typically reach 70%–80% confluency in 2–3 days, yielding 5–6 × 10<sup>7</sup> viable cells/flask.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If initial cell cultures do not reach 70%–80% confluency within 3 days, sub-culturing can be done at a lower dilution, nevertheless cells should be transferred into a new flask; sub cultured cells typically start regaining viability and reach normal proliferation rates, so it is important to monitor them daily, especially for the B3Z cells and MCA205 cells, to avoid over confluency.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Here we used the TrypLE™ Express reagent to detach adherent cells, which can be replaced by other common trypsin products, but the time for detachment might need to be optimized.</div>
<ol start="5">
<li>Repeat step 4 to have third-generation sub-cultures that are ready to use or freeze.</li>
</ol>
<p>To prepare cryopreserved cells, collect cultured cells at 70%–80% confluency as described above. After centrifugation and removal of the supernatant, resuspend the cell pellet with 1.8 mL FBS, homogenize the suspension by pipetting up-and-down several times, and transfer 0.9 mL aliquots into each 2.5 mL cryotubes, i.e., cells from one T175 flask are aliquoted into two cryotubes. Add 0.1 mL of DMSO into each cryotube, tighten the caps, mix by turning the cryotubes up-and-down several times and immediately transfer the cryotubes to −80°C.</p>
<div class="note">
<span class="note-title">Note:</span> In this protocol, we freeze all cells by direct transfer to −80°C instead of a gradient cooling process, which has proven to maintain cellular viability and can be used for the establishment of initial culture as described in step 1–4.</div>
<ol start="6">
<li>To de-immortalize (de) the iniDCs and let them differentiate into de-iniDCs with the characteristic immunophenotype of primary dendritic cells, collect established iniDC culture as described in Step 4a, wash them with 20 mL prewarmed BC medium, and resuspend with 4 mL DC differentiating medium (DF medium, refer to the “<a href="#materials-and-equipment">materials and equipment</a>” section). Depending on the quantity needed, use 1–2 mL cell suspension, equal to 1–2 × 10<sup>7</sup> viable cells with 50 mL DF medium per T 175 tissue culture flask and maintain in the incubator for 3–4 days.</li>
<li>To collect the de-iniDCs for phenotypical and functional analysis, gently shake the flask horizontally to resuspend non-adherent cells, remove the supernatant and add 10 mL/flask of prewarmed TrypLE™ Express for 5 min to detach adherent cells. Gently agitate the flask to confirm that all cells have detached, then stop the trypsinization with 10 mL of prewarmed BC medium, transfer all content into a 50 mL falcon tube, spin at 500 <i>g</i> at room temperature to obtain a cell pellet. Resuspend the pellet with 5 mL of prewarmed BC medium for counting, and dilute with DF medium to a density as needed for experiments.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In the DF medium that does not contain Dex/Dox, proliferation of de-iniDCs will stop about 48 h after withdrawl, and viability will start decreasing shortly after. Typically, 2 × 10<sup>7</sup> iniDCs in 50 mL DF medium / flask will yield ∼ 1.25–1.5 × 10<sup>7</sup> viable de-iniDCs at day 3–4. Extended de-immortalization will result in a significant decrease in viability.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>CRISPR CAS9 (1/2,000 for immunoblot)</td>
<td>Cell Signaling Technology</td>
<td>Cat# 14697</td>
</tr>
<tr>
<td>β-Actin HRP (1/20,000 for immunoblot)</td>
<td>Abcam</td>
<td>Cat# ab49900</td>
</tr>
<tr>
<td>HMGB1 (1/2,000 for immunoblot)</td>
<td>Abcam</td>
<td>Cat# ab18256</td>
</tr>
<tr>
<td>FPR1 (1/500 for immunoblot)</td>
<td>Invitrogen</td>
<td>Cat# PA1-41398</td>
</tr>
<tr>
<td>CD16/32 (1/1000 for blocking)</td>
<td>BioLegend</td>
<td>Cat# 101302</td>
</tr>
<tr>
<td>CD40 eFluor 450 (1/100 for flow cytometry)</td>
<td>eBioscience</td>
<td>Cat# 48-0402-82</td>
</tr>
<tr>
<td>CD11c APC (1/100 for flow cytometry)</td>
<td>BioLegend</td>
<td>Cat# 117310</td>
</tr>
<tr>
<td>MHC-II FITC (1/200 for flow cytometry)</td>
<td>BioLegend</td>
<td>Cat# 107606</td>
</tr>
<tr>
<td>CD80 PerCP-Cy5.5 (1/200 for flow cytometry)</td>
<td>BioLegend</td>
<td>Cat# 104722</td>
</tr>
<tr>
<td>CD69 E-Cy5 (1/100 for flow cytometry)</td>
<td>eBioscience</td>
<td>Cat# 15-0691-82</td>
</tr>
<tr>
<td>CD83 PE-Cy7 (1/100 for flow cytometry)</td>
<td>eBioscience</td>
<td>Cat# 25-0839-42</td>
</tr>
<tr>
<td>CD86 APC-Fire750 (1/100 for flow cytometry)</td>
<td>BioLegend</td>
<td>Cat# 105046</td>
</tr>
<tr>
<td>IL-1β (1/500 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 503502</td>
</tr>
<tr>
<td>IL-1β biotin (1/500 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 515801</td>
</tr>
<tr>
<td>IL-2 (1/500 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 503702</td>
</tr>
<tr>
<td>IL-2 biotin (1/500 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 503804</td>
</tr>
<tr>
<td>IL-6 (1/500 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 504502</td>
</tr>
<tr>
<td>IL-6 biotin (1/500 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 504602</td>
</tr>
<tr>
<td>IL-12 (p70) (1/100 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 511802</td>
</tr>
<tr>
<td>IL-12 (p40) biotin (1/1,000 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 505302</td>
</tr>
<tr>
<td>TNFα (1/200 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 506102</td>
</tr>
<tr>
<td>TNFα biotin (1/1,000 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 516003</td>
</tr>
<tr>
<td>TGFβ (1/500 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 361202</td>
</tr>
<tr>
<td>TGFβ biotin (1/500 for ELISA)</td>
<td>BioLegend</td>
<td>Cat# 141412</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Edit-R Lentiviral CAG-Blast-Cas9 Nuclease Particles</td>
<td>Horizon Discovery</td>
<td>Cat# VCAS10129</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>EUROMEDEX</td>
<td>Cat# 04-100-812-E</td>
</tr>
<tr>
<td>Bovine Serum Albumin (BSA), Low Endotoxin &amp; IgG, Protease</td>
<td>US Biological</td>
<td>Cat# A1315</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Sigma</td>
<td>Cat# M3148</td>
</tr>
<tr>
<td>Recombinant Murine GM-CSF</td>
<td>PeproTech</td>
<td>Cat# 315-03</td>
</tr>
<tr>
<td>Dexamethasone</td>
<td>Sigma</td>
<td>Cat# D0700000</td>
</tr>
<tr>
<td>Doxycycline hyclate</td>
<td>Sigma</td>
<td>Cat# D3000000</td>
</tr>
<tr>
<td>Blasticidin</td>
<td>InvivoGen</td>
<td>Cat# ant-bl-1</td>
</tr>
<tr>
<td>CellTracker™ Blue CMAC dye</td>
<td>Invitrogen</td>
<td>Cat# C2110</td>
</tr>
<tr>
<td>Crizotinib</td>
<td>Sigma</td>
<td>Cat# PZ0191</td>
</tr>
<tr>
<td>Lipopolysaccharides (LPSs)</td>
<td>Sigma</td>
<td>Cat# L2654</td>
</tr>
<tr>
<td>Recombinant murine TNFα</td>
<td>BioLegend</td>
<td>Cat# 575206</td>
</tr>
<tr>
<td>Albumin from chicken egg white</td>
<td>Sigma</td>
<td>Cat# A5503</td>
</tr>
<tr>
<td>OVA peptide 257–264 (SIINFEKL, SL8)</td>
<td>GenScript</td>
<td>Cat# RP10611</td>
</tr>
<tr>
<td>Recombinant murine IL-1β</td>
<td>BioLegend</td>
<td>Cat# 575109</td>
</tr>
<tr>
<td>Recombinant murine IL-2</td>
<td>BioLegend</td>
<td>Cat# 575409</td>
</tr>
<tr>
<td>Recombinant murine IL-6</td>
<td>BioLegend</td>
<td>Cat# 575709</td>
</tr>
<tr>
<td>Recombinant murine IL-12(p70)</td>
<td>BioLegend</td>
<td>Cat# 577009</td>
</tr>
<tr>
<td>Recombinant murine TGFβ</td>
<td>BioLegend</td>
<td>Cat# 594509</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Zombie UV™ Fixable Viability Kit</td>
<td>BioLegend</td>
<td>Cat# 423108</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Inducible immortalized murine dendritic cell line (iniDC)</td>
<td>(<a href="#bib26">Richter et al., 2013</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>B3Z T cell line</td>
<td>(<a href="#bib17">Kurts et al., 1996</a>)</td>
<td>RRID: CVCL 6277</td>
</tr>
<tr>
<td>MCA205 Mouse Fibrosarcoma Cell Line</td>
<td>Sigma</td>
<td>Cat# SCC173 RRID: CVCL VR90</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse C57BL/6JOlaHsd (six- to ten-week-old females)</td>
<td>ENVIGO France</td>
<td>5704F</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Predesigned synthetic guide RNAs (crRNA)</td>
<td>Please see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Mmc1.extension">Table S1</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td>Edit-R synthetic crRNA non-targeting control#1</td>
<td>Horizon Discovery</td>
<td>Cat# U-007501-01-2</td>
</tr>
<tr>
<td>trans-activating CRISPR RNA (tracrRNA)</td>
<td>Horizon Discovery</td>
<td>Cat# U-002005-5000</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Microsoft office Excel</td>
<td>Microsoft</td>
<td><a href="https://www.microsoft.com/en-us/microsoft-365/excel">https://www.microsoft.com/en-us/microsoft-365/excel</a></td>
</tr>
<tr>
<td>FlowJo</td>
<td>FlowJo LLC</td>
<td><a href="https://www.flowjo.com/">https://www.flowjo.com/</a></td>
</tr>
<tr>
<td>Graphpad Prism</td>
<td>GraphPad Software</td>
<td><a href="https://www.graphpad.com/">https://www.graphpad.com/</a></td>
</tr>
<tr>
<td>TumGrowth</td>
<td>(<a href="#bib9">Enot et al., 2018</a>)</td>
<td><a href="https://kroemerlab.shinyapps.io/TumGrowth/">https://kroemerlab.shinyapps.io/TumGrowth/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>RPMI 1640 medium, GlutaMAX™</td>
<td>Gibco</td>
<td>CAT# 61870010</td>
</tr>
<tr>
<td>DMEM, high glucose, GlutaMAX™</td>
<td>Gibco</td>
<td>CAT# 10566016</td>
</tr>
<tr>
<td>HEPES (1 M)</td>
<td>Gibco</td>
<td>CAT# 15630056</td>
</tr>
<tr>
<td>Sodium Pyruvate (100 mM)</td>
<td>Gibco</td>
<td>CAT# 11360070</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (Pen/Strep, 10,000 U/mL)</td>
<td>Gibco</td>
<td>CAT# 15140122</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Sigma</td>
<td>CAT# F7524</td>
</tr>
<tr>
<td>Phosphate-Buffered Saline (PBS)</td>
<td>Gibco</td>
<td>CAT# 20012027</td>
</tr>
<tr>
<td>TrypLE™ Express</td>
<td>Gibco</td>
<td>CAT# 12604013</td>
</tr>
<tr>
<td>Trypan Blue solution, 0.4%, liquid</td>
<td>Sigma</td>
<td>CAT# T8154</td>
</tr>
<tr>
<td>DAPI Solution (1 mg/mL)</td>
<td>Life Technologies</td>
<td>CAT# 62248</td>
</tr>
<tr>
<td>RNase AWAY™ surface decontaminant</td>
<td>Molecular BioProducts</td>
<td>CAT# 7002</td>
</tr>
<tr>
<td>DharmaFECT 1 transfection reagent</td>
<td>Horizon Discovery</td>
<td>CAT# T-2001-04</td>
</tr>
<tr>
<td>10 mM Tris-HCl Buffer pH 7.4</td>
<td>Horizon Discovery</td>
<td>CAT# B-006000-100</td>
</tr>
<tr>
<td>NuPAGE® LDS Sample Buffer (4×)</td>
<td>Life Technologies</td>
<td>CAT# NP0008</td>
</tr>
<tr>
<td>NuPAGE® Sample Reducing Agent (10×)</td>
<td>Life Technologies</td>
<td>CAT# NP0009</td>
</tr>
<tr>
<td>Pierce Protease&amp; Phosphatase Inhibitor Tablets</td>
<td>Life Technologies</td>
<td>CAT# A32961</td>
</tr>
<tr>
<td>RIPA Extraction and Lysis Buffer</td>
<td>Life Technologies</td>
<td>CAT# 89901</td>
</tr>
<tr>
<td>10× TBS buffer</td>
<td>EUROMEDEX</td>
<td>CAT# ET220-B</td>
</tr>
<tr>
<td>Tween® 20</td>
<td>Sigma</td>
<td>CAT# P1379</td>
</tr>
<tr>
<td>ELISA Coating Buffer (5×)</td>
<td>BioLegend</td>
<td>CAT# 421701</td>
</tr>
<tr>
<td>HRP-Avidin</td>
<td>BioLegend</td>
<td>CAT# 405103</td>
</tr>
<tr>
<td>1-Step™ Ultra TMB-ELISA Substrate Solution</td>
<td>Life Technologies</td>
<td>CAT# 34028</td>
</tr>
<tr>
<td>Sulfuric acid</td>
<td>Sigma</td>
<td>CAT# 258105-1L-PC</td>
</tr>
<tr>
<td>Titramax 100R shaker</td>
<td>Heidolph</td>
<td>CAT# 544–11200-00</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Preparation of stock reagents/solutions</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Preparation</th>
</tr>
</thead>
<tbody>
<tr>
<td>Predesigned synthetic guide RNAs (crRNA)</td>
<td rowspan="3">10 μM</td>
<td rowspan="3">Briefly centrifuge tubes or plates containing the RNAs to ensure that the RNA is located at the bottom of the tubes or wells. Resuspend in 10 mM Tris pH 7.4 to obtain a 10 μM stock concentration with appropriate volumes, e.g., for 10 nmol of crRNA, add 1 mL of 10 mM Tris pH 7.4. Securely seal plates or tubes and solubilize on a plate shaker at maximum speed for 30 min at room temperature. In case of large volumes of RNA solution, especially regarding the tracrRNA, aliquot to smaller quantities. All RNA solutions should be stored at −80°C and not subjected to multiple freeze-thaw cycles.</td>
</tr>
<tr>
<td>Trans-activating CRISPR RNA (tracrRNA)</td>
</tr>
<tr>
<td>Edit-R synthetic crRNA non-targeting control#1</td>
</tr>
<tr>
<td>Decomplemented FBS</td>
<td>N/A</td>
<td>Thaw the frozen FBS in a 37°C water bath and decomplement by heating at 56°C for 30 min. Store at 4°C for a up to one month.</td>
</tr>
<tr>
<td>β-mercaptoethanol</td>
<td>50 mM (1,000×)</td>
<td>Dilute 200 μL β-mercaptoethanol (14.3 M) with 56.8 mL PBS and pass through a 0.22 μm filter. Store at 4°C for up to 6 months</td>
</tr>
<tr>
<td>Recombinant murine GM-CSF</td>
<td>50 μg/mL</td>
<td>Reconstitute the lyophilized powder at 50 μg/mL in 0.1% BSA (Low-Endotoxin)/PBS in a sterile environment, aliquot and store at −80°C. Stable for more than one year, avoid freeze-thaw cycles, store at 4°C for a maximum of 14 days.</td>
</tr>
<tr>
<td>Dexamethasone</td>
<td>25 mM</td>
<td>Reconstitute at 25 mM in ethanol, stable at −20°C for more than a year.</td>
</tr>
<tr>
<td>Doxycycline</td>
<td>25 mg/mL</td>
<td>Reconstitute at 25 mg/mL in water, stable at −20°C for more than a year.</td>
</tr>
<tr>
<td>CellTracker™ Blue CMAC Dye</td>
<td>10 mM</td>
<td>Reconstitute the 5 mg powder with 2.39 mL DMSO in a sterile environment, aliquot to smaller volume and store at −20°C for up to 2 years.</td>
</tr>
<tr>
<td>Crizotinib</td>
<td>10 mM</td>
<td>Reconstitute the 5 mg powder with 1.1 mL DMSO in a sterile environment, aliquot and store at −20°C for up to 1 year.</td>
</tr>
<tr>
<td>LPS</td>
<td>1 mg/mL</td>
<td>Reconstitute lyophilized LPS with sterile Milli-Q water in a sterile environment, aliquot store at −20°C for up to 1 year.</td>
</tr>
<tr>
<td>Recombinant murine TNFα</td>
<td>50 μg/mL</td>
<td>Recombinant murine TNFα from Biolegend is bottled at lot-specific concentrations is diluted at 50 μg/mL with 0.1% BSA (Low-Endotoxin)/PBS, aliquoted and stored at −80°C. Stable for more than one year, avoid freeze-thaw cycles.</td>
</tr>
<tr>
<td>Ovalbumin (OVA)</td>
<td>100 mg/mL</td>
<td>Reconstitute the powder with DC medium, pass through a 0.22 μm filter, and aliquot and store at −20°C for up to 1 year.</td>
</tr>
<tr>
<td>OVA peptide (SL8)</td>
<td>100 mg/mL</td>
<td>Reconstitute the powder at 1 mg/mL stock concentration with PBS, pass through a 0.22 μm filter, aliquot and store at −80°C for up to 1 year. Once thawed, dilute to 1 μg/mL with RPMI 1640 medium and store at −80°C for up to 3 months.</td>
</tr>
<tr>
<td>FACS buffer (1% BSA)</td>
<td>1 mg/mL</td>
<td>PBS containing 1% (w/v) BSA. Dissolve 5 g of BSA powder in 500 mL of PBS, pass through a 0.22 μm filter and store at 4°C for up to 1 month.</td>
</tr>
<tr>
<td>ELISA coating buffer (1×)</td>
<td>N/A</td>
<td>For each 96-well plate, dilute 2 mL of 5× concentrated coating buffer with 8 mL ddH<sub>2</sub>O and use the diluted buffer within 48 h, or pass through a 0.22 μm filter and store at 4°C for maximum 3 months.</td>
</tr>
<tr>
<td>ELISA washing buffer</td>
<td>N/A</td>
<td>ELISA washing buffer is 1× TBS containing 0.05% Tween 20 (v/v). Store at room temperature for a maximum of one month</td>
</tr>
<tr>
<td>ELISA diluent</td>
<td>N/A</td>
<td>ELISA diluent is PBS with 1% (w/v) BSA and 5% FBS (v/v). Dissolve 5 g of BSA powder in 500 mL of PBS, gently shake to dissolve the powder completely, then add 50 mL FBS, mix, and pass through a 0.22 μm filter. The sterile diluent can be stored at 4°C for up to 1 month.</td>
</tr>
<tr>
<td>ELISA stopping buffer (2 M sulfuric acid)</td>
<td>2 M</td>
<td>Prepare 845 mL ddH<sub>2</sub>O in a 1 L glass bottle, then use a 25 mL serological pipette to slowly add 100 mL pure sulfuric acid (18.9 M), do not mix the solution but let the bottle at room temperature for about 2 h. Gently mix and store at room temperature, for up to 6 months.</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> β-mercaptoethanol is a hazardous reagent and must be handled in a chemical foam hood. Once diluted the working stock can be handled in ordinary cell culture conditions.</div>
<div class="critical">
<span class="critical-title">Critical:</span> When diluting sulfuric acid (18.9 M), it is important to take precaution and always add acid to water, and never the other way around!</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The ELISA coating buffer in this protocol is commercially available from Biolegend, and it can be replaced with a customized carbonate-bicarbonate buffer (0.1 M, pH 9.6) containing 0.0686 M sodium bicarbonate and 0.0314 M sodium carbonate.</div>
<table id="undtbl2">
<caption>Basic DC culture medium (BC medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI 1640 medium</td>
<td>1×</td>
<td>500 mL</td>
</tr>
<tr>
<td>Decomplemented FBS</td>
<td>10%</td>
<td>57 mL</td>
</tr>
<tr>
<td>Sodium pyruvate (100 mM)</td>
<td>1 mM</td>
<td>5.7 mL</td>
</tr>
<tr>
<td>HEPES (1M)</td>
<td>10 mM</td>
<td>5.7 mL</td>
</tr>
<tr>
<td>Pen/Strep, 10,000 U/mL</td>
<td>100 U/mL:</td>
<td>5.7 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td><b>575 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 4 weeks</p>
<table id="undtbl3">
<caption>Antibiotics-free basic DC culture medium (AF-BC medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI 1640 medium</td>
<td>1×</td>
<td>500 mL</td>
</tr>
<tr>
<td>Decomplemented FBS</td>
<td>10%</td>
<td>57 mL</td>
</tr>
<tr>
<td>Sodium pyruvate (100 mM)</td>
<td>1 mM</td>
<td>5.7 mL</td>
</tr>
<tr>
<td>HEPES (1M)</td>
<td>10 mM</td>
<td>5.7 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td><b>570 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 4 weeks</p>
<table id="undtbl4">
<caption>Dex/Dox (1000×)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dexamethasone (25 mM)</td>
<td>100 μM</td>
<td>40 μL</td>
</tr>
<tr>
<td>Doxycycline (25 mg/mL)</td>
<td>1 mg/mL</td>
<td>400 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>1×</td>
<td>9.56 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1000×</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for a maximum of 3 months</p>
<table id="undtbl5">
<caption>DC complete growth medium (CG medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BC medium</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>β-Mercaptoethanol (50 mM)</td>
<td>50 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>recombinant GM-CSF (50 μg/mL)</td>
<td>10 ng/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>DEX/DOX (1000×)</td>
<td>100 nM DEX &amp; 1 μg/mL DOX</td>
<td>50 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use within 24 h</p>
<table id="undtbl6">
<caption>DC differentiating medium (DF medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BC</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>β-Mercaptoethanol (50 mM)</td>
<td>50 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Recombinant GM-CSF (50 μg/mL)</td>
<td>20 ng/mL</td>
<td>20 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use within 24 h</p>
<table id="undtbl7">
<caption>DC transduction &amp; transfection medium (TT medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>AF-BC medium</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>β-Mercaptoethanol (50 mM)</td>
<td>50 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>DEX/DOX (1000×)</td>
<td>100 nM DEX &amp; 1 μg/mL DOX</td>
<td>50 μL</td>
</tr>
<tr>
<td>Recombinant GM-CSF (50 μg/mL)</td>
<td>10 ng/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use within 24 h</p>
<table id="undtbl8">
<caption>DC transduction &amp; transfection and differentiation medium (TT-DF medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>AF-BC medium</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>β-Mercaptoethanol (50 mM)</td>
<td>50 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Recombinant GM-CSF (50 μg/mL)</td>
<td>20 ng/mL</td>
<td>20 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use within 24 h</p>
<table id="undtbl9">
<caption>DMEM based culture medium (D-BC medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM medium</td>
<td>1×</td>
<td>500 mL</td>
</tr>
<tr>
<td>Decomplemented FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Pen/Strep, 10,000 U/mL</td>
<td>100 U/mL:</td>
<td>5.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td><b>555 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 4 weeks</p>
<table id="undtbl10">
<caption>Rapid western blotting protein preparation buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>4× LDS sample buffer</td>
<td>2×</td>
<td>5 mL</td>
</tr>
<tr>
<td>10× sample reducing agent</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>RIPA buffer</td>
<td>1×</td>
<td>4 mL</td>
</tr>
<tr>
<td>Protease &amp; phosphatase inhibitors</td>
<td>1×</td>
<td>1 tablet</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">The homogenized buffer can be aliquoted and stored at −20°C for up to 6 months</p>
<div class="critical">
<span class="critical-title">Critical:</span> Contains SDS and DTT, should be handled in a chemical foam hood. The buffer is for rapid preparation of denatured protein samples for WB only when protein quantification is not necessary.</div>
<h3 id="sec2.1">Predesigned synthetic crRNAs</h3>
<p>Briefly centrifuge tubes or plates containing synthetic crRNAs to ensure that the RNA is pelleted at the bottom of the vial, then add appropriate volume of nuclease-free 10 mM Tris-HCl Buffer (pH 7.4) to reach a final concentration of 10 μM. After mixing, the solutions are aliquoted to limit the number of freeze-thaw cycles, and can be stored at −80°C for more than a year.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Synthetic CRISPR RNAs (crRNAs) used in this protocol are predesigned sequences from Horizon discovery. Alternative crRNAs by other providers can be used but KO efficacy and target specificity might be different.</div>
<h3 id="sec2.2">Tumor cell lysis</h3>
<p>Collect MCA205 cells and wash twice with cold PBS before reconstituting to desired cell density in PBS. Prepare cell lysate by 5 repeated freezing-thawing process in liquid nitrogen before sonication to ensure disruption of the cells. Centrifuge the lysate at 1,000 <i>g</i> for 5 min to remove membranes and debris and use the supernatant for further experimentation. Aliquot the prepared lysate and store at −80°C or use immediately.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Edit-R lentiviral CAS9 nuclease expression particle transduction and single cell cloning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 weeks</div>
<p>This step describes the construction of CAS9-expressing inducible immortalized dendritic cells (iniDCs), containing the culture of immortal cells, de-immortalization to obtain immature DCs (de-iniDCs), lentiviral transduction, and clone validation regarding the expression of CAS9 and gene-editing efficacy (<a href="#fig1">Figure 1</a>A).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overview of the genotype-phenotype screening procedure and hit validation steps</div>
<p>(A) Generation of inducible immortalized dendritic cells (iniDC) that stably express the CRISPR-CAS9 nuclease.</p>
<p>(B) CRISPR RNA (crRNA) library screening for the identification of genes involved in the iniDC functions.</p>
<p>(C) Generation of gene-edited iniDC clones and validation of DC function alteration.</p>
<p>(D) Principle of DC immunotherapy based on <i>in vitro</i> differentiated iniDC (de-iniDC).</p>
</figcaption>
</figure>
<ol>
<li>Transduction of iniDCs with CRISPR CAS9 lentiviral particles
<ol type="a">
<li>Collect 70–80% confluent cultured iniDCs by trypsinizing with TrypLE™ Express for 5 min at 37°C. Collect cell suspension and centrifugate at 500 <i>g</i> for 5 min at room temperature (RT) and resuspend the cell pellet with freshly prepared TT medium for cell counting.</li>
<li>Adjust the cell density to 2×10<sup>5</sup> viable cells/mL, seed 1 mL cell suspension in TT medium (equal to 2×10<sup>5</sup> cells) into a 12-well plate and let the cells adapt overnight (&gt;12 h) in a cell culture incubator at 37°C and 5% CO<sub>2</sub>.</li>
<li>The next day, remove the medium and replace with 0.5 mL RPMI 1640. Calculate the volume of lentiviral particles to be used for transduction at a MOI of 0.3 as detailed below:<br> Volume = 0.3 MOI × cell number (2×10<sup>5</sup>)/lentiviral titer (10<sup>7</sup>TU/mL) = 0.006 mL</li>
<li>Thaw the lentiviral particles on ice, gently mix the solution by pipetting up and down several times, and drop 6 μL into the 0.5 mL iniDC culture. Gently shake the plate to homogenize lentiviral particles in the iniDC culture, and put the plate back to the cell culture incubator.</li>
<li>Six hours after adding the lentiviral particles, complement the medium with additional 1.5 mL TT medium, gently shake to mix, and put the plate back to the CO<sub>2</sub> incubator.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is recommended to stain the cells with trypan blue before cell counting, which is helpful to obtain a precise number of viable cells. Once the lentiviral particles are thawed and mixed, aliquot the solution into single use quantities and store them at −80°C. Repeated freeze-thaw cycles should be avoided. For all transduction and transfection procedures, it is recommended to use medium without Pen/Strep or blasticidin which potentially affects cellular viability and transduction efficacy according to the manufactures’ instructions.</div>
<ol start="2">
<li>Antibiotic selection of transfected cells and single cell sorting for cloning
<ol type="a">
<li>At 48 h post transduction, replace the medium with fresh CG medium containing 10 μg/mL blasticidin and maintain the selection. Once the blasticidin resistant cells recover to normal proliferation (in about 3 days) and reach confluency in the 12-well plate (in 5–7 days, yield 2–5 × 10<sup>5</sup> cells), aspirate old media and add 500 μL prewarmed TrypLE™ Express reagent for 5 min to trypsinize the cells. Gently flush the cell layer with a P1000 pipette to confirm detachment, stop the trypsinization with 500 μL prewarmed CG medium and transfer all content into a T25 tissue culture flask containing 10 mL complete DC growth medium (supplemented with 10 μg/mL blasticidin) and keep the cells in the cell culture incubator.</li>
<li>When blasticidin resistant cells reach 70%–80% confluency in the T25 flask, aspirate used media and add 3 mL prewarmed TrypLE™ Express reagent for 5 min to detach adherent cells. Gently agitate the T25 flask to confirm cellular detachment, stop the trypsinization with 5 mL of prewarmed CG medium and transfer all content into a 15 mL falcon tube. Spin at 500 <i>g</i> at room temperature, remove the supernatant, and resuspend the pellet with 5 mL CG medium. To sort single cell clones:
<ol type="i">
<li>Pass the cell suspension through a 70 μm cell strainer to make a single cell suspension, count cellular density and adjust it to 1×10<sup>6</sup> cells/mL with CG medium containing 10 μg/mL blasticidin.</li>
<li>Prepare 96-well flat bottom tissue culture-treated microplates by adding 150 μL/well of the CG medium containing 10 μg/mL blasticidin.</li>
<li>Transfer 1 mL single cell suspension to a FACS tube, add 1 μL DAPI solution (1 mg/mL) as a viability dye, and immediately proceed to flowcytometric sorting.</li>
<li>Single cell sorting can be performed with a BD FACSAria™ Fusion Cell Sorter with the help of an experienced operator. Debris and dead cells are excluded based on forward / side scatter characteristics and DAPI staining, while healthy cells are seeded in 96-well flat-bottom tissue-culture plates prepared in step (ii) at one cell/well. Keep the cells in a cell culture incubator and observe plates regularly until visible clones arise.</li>
</ol>
</li>
<li>Once clones have grown in the 96-well plates, discard the culture supernatant and add 50 μL/well TrypLE™ Express and incubate for 5 min at 37°C. Gently flush the cell layer with a P200 pipette to confirm cellular detachment, stop the trypsinization by adding 100 μL CG medium containing 10 μg/mL blasticidin, resuspend cells by pipetting up-and-down several times, transfer clones to a new 96-well plate and incubate further in a cell culture incubator until cells reach confluency. This step allows transfering clones to a single 96 well plate which facilitates the handling in subsequent steps.</li>
<li>When cells reach confluency, trypsinize them as described in step (c), and stop with 150 μL CG medium containing 10 μg/mL blasticidin. Resuspend cells by pipetting up-and-down several times, transfer 180 μL cell suspension into a new 96-well plate that will be used as the assay plate; transfer the remaining cellular suspension into another 96-well plate and supply 180 μL/well CG medium (containing 10 μg/mL blasticidin) to obtain the seed plate for preserving clones. The assay plate is cultured overnight (&gt;12 h) before protein extraction and WB detection of CAS9 nuclease expression.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The procedure in step 1 will likely yield 30%–50% transduction efficiency, which means more than 50% of cell death is expected at day 3 after blasticidin treatment, and resistant cells will recover normal proliferation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to maintain blasticidin throughout the selection of single clones to minimize the number of CAS9 nuclease-negative clones in further verification steps. Due to the risk of contamination during the sorting process, it is recommended to use double amount of Pen/Strep in the CG medium for this step. Visually detectable clones form 2–3 weeks after sorting. Evaporation should be controlled and supplementation of fresh CG medium might be necessary.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If the western blotting verification will take longer, the seed plates can be frozen at −80°C. Collect clones as described above and transfer cell suspensions into 96-well V-shape plate. Centrifuge the V-shape plates, discard supernatant, then resuspend the cell pellets with 100 μL FBS containing 10% DMSO and immediately put them into −80°C.</div>
<ol start="3">
<li>Detection of CAS9 expression levels in different clones.
<ol type="a">
<li>Centrifugate the assay plate at 500 <i>g</i> for 5 min at RT, discard the supernatant and invert the plate on a paper tissue to remove remaining liquid. Add 100 μL rapid WB protein preparation buffer per well, shake the plate on a plate shaker at maximum speed (approximately 1300 rpm, on a Heidolph TITRAMAX 100R shaker) for 5 min, transfer the solution into a 96-well PCR plate, and heat the samples to 100°C for 5 min.</li>
<li>Subject the denatured proteins to standard western blotting for the detection of CAS9 nuclease expression level. A loading control such as β-actin should be detected on the same blot.</li>
<li>Clones that express a high level of CAS9 can be selected for further testing.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All western blotting data exemplified here was obtained by using the Invitrogen NuPAGE® electrophoresis system, including all equipment and reagents as listed in the NuPAGE® technical guidelines (Manual part no. IM-1001/MAN0003188). Alternative products might be used, but methodology might need to be adjusted. Representative data provided in this protocol was obtained using a mouse monoclonal antibody to CAS9 (dilution 1:2,000, incubation overnight (&gt;12 h) at 4°C) and an HRP mouse monoclonal antibody to β-actin (dilution 1:20,000, incubation for 2 h at room temperature).</div>
<p><a href="#sec7.1">Troubleshooting 1</a></p>
<ol start="4">
<li>Validation of gene editing efficacy of selected CAS9-expressing iniDC clones
<ol type="a">
<li>Based on CAS9-expression detected by WB, label selected clones that express high level of CAS9 in the seed plates. Trypsinize those clones as described in step 2c and stop trypsin with 150 μL CG medium, transfer all content to one well of a 6-well plate, supply 2 mL of CG medium (containing 10 μg/mL blasticidin), and maintain in an incubator until cells reach 50%–70% confluency.</li>
<li>Trypsinize the clones in 6-well plates with 0.5 mL/well prewarmed TrypLE™ Express for 5 min at room temperature. Gently flush the cell layer with a P1000 pipette to confirm cellular detachment, and stop trypsin with 1.5 mL TT-DF medium. For each clone, transfer 0.4 mL/well of cell suspension into a new 6-well plate, supply 1.6 mL/well CG medium (containing 10 μg/mL blasticidin) to obtain the seed plate for clone preservation. The remaining cell suspension is transferred to another 6-well plate (assay plate), supplied with 1 mL TT-DF medium, and let adapt in the incubator overnight (&gt;12 h) for crRNA transfection.</li>
<li>The next day, remove used medium and replace with 2 mL fresh TT-DF medium.</li>
<li>Prepare the transfection mixture:
<ol type="i">
<li>add 5.5 μL of previously prepared 10 μM crRNA targeting the murine high mobility group box 1 (<i>Hmgb1</i>) gene or the same quantity of non-targeting control gRNA #1, plus 5.5 μL of 10 μM tracrRNA to 100 μL of serum-free RPMI 1640 medium, mix by pipetting up-and-down to get the guidance RNA transfection complex;</li>
<li>add 10 μL of DharmaFECT 1 transfection reagent to 100 μL of serum-free RPMI 1640 medium, mix gently by slowly pipetting up-and-down;</li>
<li>5 min later, gently mix the solutions i and ii and incubate for another 20 min at room temperature to obtain transfection complexes.</li>
</ol>
</li>
<li>Add 200 μL of transfection mix dropwise into 2 mL CAS9-expressing iniDC culture, and gently agitate the plate. Incubate transfected cells in a cell culture incubator for 72 h to achieve gene editing.</li>
<li>As described in step 3, prepare homogenate from post-transfected cells with 250 μL/well of the rapid WB protein preparation buffer. Subject the denatured proteins to WB detection of HMGB1 expression with β-actin as a loading control. Clones that show the highest <i>Hmgb1</i> knockout efficiency should be selected and expanded for the following screening campaign while maintaining blasticidin (10 μg/mL) selection pressure.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure that all plasticware is RNase-free verified, and treat the bench as well as instruments with the RNase away spray or other surface RNase decontaminants before handling RNA to avoid degradation of crRNA and tracrRNA oligos.</div>
<div class="note">
<span class="note-title">Note:</span> This is a general description for the preparation of a transfection mixture, which is designed for the transfection of one single well of a 6-well plate. The conditions (final concentration of crRNA and tracrRNA are 25 nM used with 10 μL of DharmaFECT 1 per well) are optimized for an efficient transfection of 1<b>×</b>10<sup>6</sup> iniDC CAS9 in 2 mL of TT-DF medium. For the test of gene-editing efficacy in iniDC CAS9 clones, we suggest using gRNA targeting the murine <i>Hmgb1</i> gene (positive control) together with non-targeting control gRNA #1 as a negative control (please see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Mmc1.extension">Table S1</a> for details). For the detection of HMGB1 expression, the rabbit polyclonal antibody to HMGB1 can be diluted at 1:2,000 for an overnight (&gt;12 h) incubation at 4°C.</div>
<div class="note">
<span class="note-title">Note:</span> CRISPR crRNA transfection can also be performed with freshly prepared CAS9-expressing iniDCs (iniDC CAS9). In such case, collect cultured iniDC CAS9 by trypsinization and adjust cell suspension to a concentration of 5×10<sup>5</sup>/mL with TT-DF medium. Seed 2 mL of the suspension to 6-well tissue culture plates (equal to 1×10<sup>6</sup> cells/well) and proceed to the described transfection steps. We recommend to use the TT-DF medium for transfection which allows for simultaneous differentiation and facilitates downstream functional assays with the gene-modified de-iniDCs. However, transfection can be performed with TT medium if clones need to be sorted.</div>
<p>The protocol established for HMGB1 (positive control) and non-targeting control crRNAs (negative controls) can be applied for the genetic screening of genes of interest using the clone that we selected based on the high CAS9 activity, as detailed below.</p>
<div class="note">
<span class="note-title">Note:</span> The knockout of genes of interest needs to be confirmed by western blot or sequencing of crRNA-targeted region in the genomic DNA when cell lines are being established for further evaluation.</div>
<h3 id="sec3.2">Genetic screening with non-pooled guidance RNAs by means of an <i>in vitro</i> antigen cross-presentation assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days.</div>
<p>This major step describes a typical small to medium-scale genetic screening based on the iniDC CAS9 cells, with non-pooled synthetic crRNA + tracrRNA transfection to obtain specific gene knockout (<a href="#fig1">Figure 1</a>B). The exemplified screening uses the OVA antigen cross-presentation assay as functional readouts.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This step focuses on identifying genes that are involved in the antigen cross presentation ability of DCs, but it can be adapted to other DC phenotypical or functional assays as described below, after the ini-DC CAS9 cells are transfected with the crRNA library.</div>
<ol start="5">
<li>Prepare the verified iniDC CAS9 clones and expand cells according to the number of gRNAs that need to be transfected. For example, a 96-well crRNA library plate typically holds 80 distinct crRNAs which together with 8 non-targeting controls (<a href="#fig2">Figure 2</a>) results in 88 transfections. Considering the dead volume, a total number of iniDC CAS9 cells for seeding &gt;90 wells should be prepared, i.e., a minimum of 90 × 5×10<sup>5</sup>/mL × 2 mL = 9 ×10<sup>7</sup> viable iniDC CAS9 cells for each crRNA plate. Seed 1×10<sup>6</sup> de-iniDC CAS9 cells/well in 2 mL DF medium into 6-well tissue-culture plates. Label plates and wells according to the gRNA to be transfected and incubate the cells in a cell culture incubator
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Scheme of synthetic CRISPR RNA (crRNA) transfection in 96-well plate format</div>
<p>A typical gRNA library plate containing 80 synthetic crRNAs of interest (as described in Steps 5 to 9) is employed for screening. It is suggested to use 8 wells of non-targeting control gRNA #1 for screening one plate to obtain a sufficient number of unmodified cells as controls. Key steps include 1) preparation of the trans-activating CRISPR RNA (tracrRNA) and distribution into a 96-well plate at 100 μL/well to be complemented with equal quantities of specific gRNAs or non-targeting control gRNA; 2) preparation of DharmaFECT #1 transfection reagent; 3) addition of 100 μL/well transfection reagent to the tracrRNA/gRNA complexes, followed by incubation to obtain transfection-ready solutions; 4) transfer of 200 μL transfection mix to 1×10<sup>6</sup> iniDC CSA9 cells.</p>
</figcaption>
</figure>
</li>
<li>Prepare the crRNA-tracrRNA mixture:</li>
</ol>
<p>Calculate the total number of crRNAs to be transfected, and prepare a pool of 10 μM tracrRNA (100 μL/well) in serum-free RPMI 1640 medium to cover all the crRNA to be transfected. For example, to transfect a crRNA library from one 96-well plate, which usually covers 80 genes (columns 1 and 12 are commonly reserved for controls, please refer to <a href="#fig2">Figure 2</a> for a typical plate layout), tracrRNA sufficient for 90 transfections should be prepared. Dilute 495 μL of tracrRNA working stock in 9 mL serum-free RPMI 1640 medium in a RNase-free falcon tube, mix by pipetting up-and-down, and distribute 100 μL/well to the corresponding 80 wells in a new 96-well plate. Additional tracrRNA solution will be loaded to the rest wells for the non-targeting control gRNA. Then add 5.5 μL of each specific crRNAs (from library plates) or the non-targeting control crRNA to their corresponding wells in the tracrRNA plate, mix by pipetting up-and-down with a multichannel, and incubate 5 min at room temperature.</p>
<ol start="7">
<li>Prepare DharmaFECT 1 transfection reagent in equal amounts i.e., 900 μL of DharmaFECT 1 in 9 mL serum-free RPMI1640 medium. Incubate 5 min at room temperature.</li>
<li>Add 100 μL of DharmaFECT 1 solution to each well contains 100 μL of the crRNA-tracrRNA mixture, mix by pipetting up-and-down and incubate 20 min at room temperature to obtain transfection complexes.</li>
<li>Directly drop the 200 μL transfection mix to the designated 6-well plates. Gently shake the plates to homogenize transfection efficacy across the well and incubate the plates in a cell culture incubator for 3 days.</li>
<li>Remove supernatant and replace with 2 mL of fresh DF medium, incubate at 37°C for 24 h to let the transfected cells recover from transfection.</li>
<li>On the 4th day post transfection, remove 1 mL of the medium from each well, detach cells by scraping with dispensable cell lifters and transfer the cellular suspension to a 96-deep-well plate. Centrifuge the plate at 500 <i>g</i> for 5 min, remove supernatant and resuspended cell pellets with 500 μL fresh DF medium for cell counting. There is a typical yield of 5–7 ×10<sup>5</sup> cells from each well, which is sufficient for functional assays and/or frozen stocks.</li>
</ol>
<p>For genetic screening the concentration of viable cells (with the average of non-target crRNA transfected wells as a reference) is adjusted to 5×10<sup>5</sup>/mL, and cells are seeded in 96-well U-bottom tissue culture plates in 100 μL/well, equals to 5×10<sup>4</sup> cells/well, in replicate wells. The <i>in vitro</i> antigen cross-presentation assay can be proceeded as detailed from step 19 to 24. The ELISA quantification of IL-2 in the supernatant of B3Z-DC coculture can be performed as detailed in step 16 and IL-2 concentration will be used as a proxy of DC antigen cross-presentation capability.</p>
<ol start="12">
<li>For data analysis, IL-2 values from crRNA transfected samples are normalized to the average of non-target control crRNA-transfected cells.</li>
<li>When performing transfection in 6 well plates, it is likely to obtain more than enough cells for functional assays. The rest of the cells can be frozen in a 96-well V-shape plate at −80°C, which can be used for further functional screening, or for sorting to select clones that are KO of the gene of interest (<a href="#fig3">Figure 3</a>). To freeze cells, centrifuge plates for 5 min at 500 <i>g</i>, remove supernatant and rapidly resuspend cell pellets with FBS containing 10% DMSO (100 μL/well), seal the plates and directly transfer to −80°C where the cells can be kept for a maximum of 12 months.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Validation of Fpr1 or Clec4a2 KO</div>
<p>(A) Detection of Fpr1 expression in Fpr1 KO clones by immunoblot; β-actin was used as loading control.</p>
<p>(B) Schematic diagrams of the target sequence of gRNA in the Clec4a2 gene and representative sequence electropherogram for the validation of target region deletion.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.3">Detection of DC surface costimulatory markers and typical cytokine production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days cell differentiation + 6 h handling time.</div>
<p>This major step contains the phenotypic analysis of de-iniDCs expressing CRISPR-CAS9, including the detection of surface co-stimulatory and maturation markers via flow cytometric staining and quantification of major cytokine production via customized ELISAs (<a href="#fig1">Figure 1</a>C). The analyzed cytokine panel includes pro-inflammatory markers such as IL-1β, IL-12, TNFα, and IL-6 (<a href="#bib5">Cavaillon, 2001</a>; <a href="#bib30">Scheller et al.,2011</a>) as well as latent TGFβ, that is linked with immunosuppressive functions (<a href="#bib12">Ghiringhelli et al.,2005</a>), which are inhibited in the presence of DEX (<a href="#bib14">Jones et al., 2010</a>) yet are released by de-iniDC in response to stimulation with LPS (<a href="#bib26">Richter et al., 2013</a>). DC maturation and activation gain of function or loss of function phenotypes are indicated in gene edited de-iniDC by changes in the abundance of the tested cytokines in the cell culture supernatant or the expression of surface co-stimulatory and maturation markers such as CD69, CD80, CD86 as well as CD83 and MHCII, respectively.</p>
<ol start="14">
<li>De-immortalization and stimulation of DC maturation and activation.
<ol type="a">
<li>Collect iniDC CAS9 cells and dilute to 5×10<sup>5</sup> cells/mL with DF medium. Seed 1 mL of cell suspension/well in 12-well tissue culture plates (equal to 5×10<sup>5</sup> cells/well) and incubate at 37°C for 3 days to obtain differentiated de-iniDC CAS9. Determine the necessary well number needed according to the experimental design including all controls and treatment conditions. A typical experiment includes triplicates for each condition.</li>
<li>Prepare a working solution of LPS by diluting the 1 mg/mL stock at a factor of 1/500 (final concentration 2 μg/mL), recombinant murine TNFα by diluting the 50 μg/mL stock at a factor of 1/1,000 (final concentration 50 ng/mL), and MCA205 lysate to a concentration equal to 5×10<sup>5</sup> cells/mL, all with the DF medium. Remove used medium and replace with 1 mL of fresh DF medium, working solution of LPS, recombinant TNFα, or the MAC205 lysate as stimuli for DC maturation and activation. Incubate the plates for 16 h at 37°C</li>
<li>Collect all content of each well by scrapping cells from the well bottom using a plastic cell lifter, transfer the suspension into a 96-deep well round-bottom plate (which holds a maximum of 1.2 mL liquid in each well). Spin the plate at 500 <i>g</i> for 5 min at 4°C.</li>
<li>Carefully aspirate 500 μL of supernatant for the quantification of cytokines by means of ELISA or directly store samples at −20°C for a short period (&lt;1 month) or −80°C for longer period (up to 12 month). Remove the remaining supernatant, resuspend the cell pellet with 200 μL cold PBS, mix thoroughly by pipetting and transfer the cell solution to a 96-well V-shape plate for surface staining with the fluorescently-labeled antibody panel (<a href="#tbl1">Table 1</a>), followed by flow cytometric analysis.
<table id="tbl1">
<caption>Table 1. Antibody staining panel for flowcytometric analysis on the BD LSR Fortessa cytometer with appropriate compensation setup</caption>
<thead>
<tr>
<th>Laser</th>
<th>Excitation (nm)</th>
<th>Emission Filter (nm)</th>
<th>Target label</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>UV</td>
<td>355</td>
<td>450/50</td>
<td>Zombie UV</td>
<td>1/500</td>
</tr>
<tr>
<td>Violet</td>
<td>405</td>
<td>450/40</td>
<td>CD40 eFluor450</td>
<td>1/100</td>
</tr>
<tr>
<td rowspan="2">Blue</td>
<td rowspan="2">488</td>
<td>530/30</td>
<td>MHC-II FITC</td>
<td>1/200</td>
</tr>
<tr>
<td>695/40</td>
<td>CD80 PerCP-Cy5.5</td>
<td>1/200</td>
</tr>
<tr>
<td rowspan="2">Yellow-Green</td>
<td rowspan="2">561</td>
<td>710/50</td>
<td>CD69 PE-Cy5</td>
<td>1/100</td>
</tr>
<tr>
<td>LP750</td>
<td>CD83 PE-Cy7</td>
<td>1/100</td>
</tr>
<tr>
<td rowspan="2">Red</td>
<td rowspan="2">633</td>
<td>670/14</td>
<td>CD11C APC</td>
<td>1/100</td>
</tr>
<tr>
<td>780/60</td>
<td>CD86 APC-Cy7</td>
<td>1/100</td>
</tr>
</tbody>
</table>
<p class="table-legend">The gating strategy is exemplified in <a href="#fig4">Figure 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Gating strategy for the flow cytometric analysis of DC surface markers</div>
<p>Cells were first gated in FSC and SSC, followed by gating on live cells (Zombie UV<sup>-</sup>) and exclusion of doublets. Live CD11C<sup>+</sup> DCs were selected to generate histograms of the indicated activation and costimulatory markers.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Sample dilution factors for different ELISA kits have to be optimized to be compatible with the detection rage of the selected ELISA kit. Supernatant collected according to the aforementioned steps and to be quantified with the ELISA method described in this protocol, we recommend to dilute the sample as listed in <a href="#tbl2">Table 2</a>.</div>
<table id="tbl2">
<caption>Table 2. Key information for the customized ELISA</caption>
<thead>
<tr>
<th>Target</th>
<th>Sample dilution</th>
<th>Top standard concentration</th>
<th>Capture antibody</th>
<th>Detection antibody</th>
<th>Avidin-HRP</th>
</tr>
</thead>
<tbody>
<tr>
<td>IL-1β</td>
<td>1:1</td>
<td>2,000 pg/mL</td>
<td>1:500</td>
<td>1:500</td>
<td>1:1,000</td>
</tr>
<tr>
<td>IL-2</td>
<td>1:5<sup><a href="#tblfn1">a</a></sup>
</td>
<td>125 pg/mL</td>
<td>1:500</td>
<td>1:500</td>
<td>1:1,000</td>
</tr>
<tr>
<td>IL-6</td>
<td>1:50–100</td>
<td>500 pg/mL</td>
<td>1:500</td>
<td>1:500</td>
<td>1:1,000</td>
</tr>
<tr>
<td>IL-12(p70)</td>
<td>1:1</td>
<td>500 pg/mL</td>
<td>1:100</td>
<td>1:1,000</td>
<td>1:1,000</td>
</tr>
<tr>
<td>TGFβ</td>
<td>1:5</td>
<td>500 pg/mL</td>
<td>1:500</td>
<td>1:500</td>
<td>1:1,000</td>
</tr>
<tr>
<td>TNFα</td>
<td>1:5</td>
<td>4,000 pg/mL</td>
<td>1:200</td>
<td>1:1,000</td>
<td>1:1,000</td>
</tr>
</tbody>
</table>
<p id="tblfn1" class="table-footnote"><sup>a</sup>Under certain treatment conditions, the production of IL-2 by B3Z (in case of co-culture with SL10 charged DCs) can be very abundant, thus the dilution factors of those samples may need to be adapted. For example, IL-2 sample can be diluted at 1/25–1/50.</p>
<ol start="15">
<li>Flow cytometric staining of de-iniDCs:
<ol type="a">
<li>Centrifuge cells at 500 <i>g</i> for 5 min at 4°C, discard the supernatant by inverting and gently flicking down the plate and remove all remaining liquid by touching a paper tissue.</li>
<li>Resuspend the cell pellet in 100 μL cold PBS containing 1/500 diluted Zombie UV viability dye, mix by pipetting up-and-down several times and incubate at 4°C in the dark for 25 min.</li>
<li>Directly add 100 μL FACS buffer containing 1/500 diluted Fc block antibody (anti-CD16/CD32) and incubate for 10 min at 4°C in the dark.</li>
<li>Repeat step a, then resuspend the pellet with 100 μL antibody mixture diluted (in FACS buffer) according to the information in <a href="#tbl1">Table 1</a>, mix by pipetting up-and-down several times and incubate for 30 min at 4°C in the dark.</li>
<li>Add 150 μL FACS buffer per well and repeat step a, then resuspend the pellet with 100 μL FACS buffer containing 10% PFA, mix by pipetting up-and-down several times and incubate for 20 min at 4°C in the dark. Directly add 150 μL FACS buffer per well and repeat step a.</li>
<li>Resuspend the pellet with 200 μL FACS buffer, centrifuge at 500 <i>g</i> for 5 min at 4°C, discard the supernatant by inverting and gently flicking down the plate and remove all remaining liquid by touching a paper tissue.</li>
<li>Repeat step f.</li>
<li>Resuspend the cells in 100 μL FACS buffer and conduct flow cytometry analysis.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All steps involving the manipulation of PFA (from step e to h) should be handled in a chemical foam hood.</div>
<div class="note">
<span class="note-title">Note:</span> When fixing the cells, it is important to mix and resuspend the pellet immediately when cells are in contact with PFA (e).</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Fixed samples can be stored at 4°C for up to 2 weeks before flowcytometric analysis.</div>
<ol start="16">
<li>Customized ELISA reagents and procedures:
<ol type="a">
<li>Dilute an appropriate amount of ELISA capture antibody in 1× ELISA coating buffer according to the suggested dilution factor in <a href="#tbl2">Table 2</a>. Add 100 μL/well of the capture antibody solution into 96-well high-binding assay plates. Incubate at 4°C overnight (&gt;12 h), or at room temperature for a minimum of 4 h.</li>
<li>Discard the liquid by inverting and gently flicking down the plate and remove all residual liquid by touching a paper tissue, then add &gt; 250 μL of ELISA washing buffer and incubate for at least 30 s before discarding the liquid. Altogether this is one out of several ‘washing’ steps. Repeat the washing step 4 times.</li>
<li>Discard the liquid and add 200 μL/well of ELISA diluent to block unspecific binding sites. Incubate for a minimum of an hour at room temperature. The plates are now ready for sample loading.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If not use immediately, discard the diluent and remove all residual liquid by touching a paper tissue, and freeze the assay-ready plates at −20°C. The plates can be stored at −20°C for up to 3 months.</div>
</li>
<li>Reconstitute the lyophilized standard to the highest concentration (please refer to <a href="#tbl2">Table 2</a>) with ELISA diluent, and prepare a serial dilution with ELISA diluent in separate tubes, the standards, together with a blank (ELISA diluent alone), can be transferred to a 96-well V-shape plate or a deep well plate to facility sample handling. The mother solution can be aliquoted to smaller volumes and stored at −20°C for up to 6 months. Dilute samples (supernatant) with ELISA diluent as suggested in <a href="#tbl2">Table 2</a>.</li>
<li>Discard the ELISA diluent, remove residual liquid by tapping the plate upside down on a paper tissue, and transfer 100 μL/well of blank, standards, and appropriately diluted samples to the designated wells. Precoated plates from −20°C should be brought to room temperature prior to sample loading. Incubate the plates at room temperature on a plate shaker (e.g., benchtop shakers for western blotting at a speed of 40–80 rpm) for 2 h.</li>
<li>Discard the liquid and wash the plates 4 times. Dilute an appropriate amount of biotinylated detection antibody in ELISA diluent according to the suggested dilution factor in <a href="#tbl2">Table 2</a>. Discard the washing buffer and remove residual liquid by tapping the plate upside down on a paper tissue, and add 100 μL of diluted detection antibody solution to each well. Incubate at room temperature for 1 h on a plate shaker at a speed of 40–80 rpm.</li>
<li>Discard the liquid and wash the plates 4 times. Dilute an appropriate volume of avidin-HRP at 1/1,000 with the ELISA diluent. Discard the washing buffer, remove residual liquid by tapping the plate upside down on a paper tissue, and add 100 μL/well avidin-HRP. Seal the plates with adhesive aluminum foil and incubate at room temperature for 30 min on a plate shaker.</li>
<li>During step g, bring the 1-Step™ Ultra TMB-ELISA substrate solution to room temperature. Discard the avidin-HRP solution and wash the plates 5 times. After the last wash, remove residual liquid by tapping plate upside down on a paper tissue, and add 100 μL/well substrate. Incubate in the dark, and observe the colorization, the positive controls (e.g., the maximum concentration used for the standard curve) should turn blue. Typical substrate incubation time is 5–20 min, when the most concentrated standards will become dark but clear blue.</li>
<li>Add 50 μL/well of stop solution to terminate the reaction. The color of the solution should turn from blue to orange/yellow. Read the absorbance at 450 nm as soon as possible.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is important to bring the ELISA diluent, standards, samples, and TMB substrate to room temperature before use. The colorization step should not exceed 20 min, and must be continuously observed. Formation of crystals after adding the stop solution indicates over-colorization, which will interfere the interpretation of the results.</div>
<p><a href="#sec7.3">Troubleshooting 2–5</a></p>
<h3 id="sec3.4">Evaluation of DC phagocytosis of dying cancer cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days cell differentiation + 6 h handling time.</div>
<p>The uptake of (parts of the) dying cancer cells by phagocytic DC provides the basis for tumor-associated antigen transfer from malignant cells to the immune system, thus constitutes an essential step in the generation of anticancer immunity (<a href="#bib11">Galluzzi et al., 2017</a>). This major step describes the protocol for the quantification of de-iniDC mediated phagocytosis of dying cancer cells by regular flowcytometry (<a href="#fig1">Figure 1</a>C), which allows for the identification of essential genes implicated in the generation of anticancer immunity.</p>
<ol start="17">
<li>Prepare de-iniDC and MCA205 cells for co-culture
<ol type="a">
<li>Collect cultured iniDC cells and dilute to 5×10<sup>5</sup> cells/mL with DF medium. Seed 1 mL cell suspension/well in 12-well tissue culture plates (equal to 1×10<sup>5</sup> cells/well) and incubate at 37°C for 3 days to obtain differentiated de-iniDC CAS9. A typical experiment includes triplicates of each condition, determine well numbers according to the experimental design including controls and treatments.</li>
<li>Prepare MCA205 cells in 75 cm<sup>2</sup> tissue culture flasks at 1×10<sup>7</sup> cells/20 mL D-BC medium for crizotinib treatment, and 0.5×10<sup>7</sup> cells/20 mL D-BC medium for the DMSO control (untreated cells), let adapt overnight (&gt;12 h).</li>
<li>Stain the MCA205 cells with CellTracker blue dye (10 μM diluted in 20 mL serum-free DMEM) for 60 min, then replace with 20 mL D-BC containing 15 μM crizotinib or 0.15% DMSO. Treatment of 18–24 h should yield &gt; 50% apoptotic cells.</li>
</ol>
</li>
</ol>
<ol start="18">
<li>Collect de-iniDCs and MCA205 cells for coculture.
<ol type="a">
<li>For de-iniDCs, remove the culture supernatant by aspirating with a vacuum pump, replace with 1 mL fresh BC medium, and detach adherent cells with a cell lifter. Flush the well bottom with a P1000 pipette, transfer all content into 1.5 mL centrifuge tubes (or 96-deep well round-bottom plates), and centrifuge at 500 <i>g</i> for 5 min at room temperature. Remove supernatant, resuspended each cell pellet with 0.5 mL DF medium for cell counting, and adjust the de-iniDC density to 1×10<sup>6</sup> viable cells/mL.</li>
<li>For MCA205 cells, do not change medium, but directly lift the cell culture layer with a cell scrapper (Corning). Flush the flask bottom with a 10 mL serological pipet, transfer all content into 50 mL falcon tubes, and centrifuge at 500 <i>g</i> for 5 min at room temperature. Remove supernatant, resuspended each cell pellet with 5 mL DF medium for cell counting, and adjust the MCA205 density to 1×10<sup>6</sup> total cells/mL.
<div class="note">
<span class="note-title">Note:</span> In these steps we recommend to detach cells by scrapping instead of trypsinization for two reasons: 1) this allows to maintain the phenotypes of differentiated de-iniDCs which will be used immediately for co-incubation with tumor cells; 2) MCA205 cells are treated with crizotinib to induce immunogenic cell death (ICD) (<a href="#bib20">Liu et al., 2019</a>) which promotes their uptake by de-iniDCs, this allows to collect all viable, dying, and dead cells without affecting the exposed ICD hallmarks; this note also explains the reason for reconstituting densities of de-iniDCs based on viable cells and MCA205 on total cells.</div>
</li>
<li>Mix 0.2 mL of de-iniDCs and 0.8 mL of MCA205 cells (1 to 4 ratio of DC to MCA205) in tissue culture-treated 12-well plates and co-culture for 2 h in a cell culture incubator. Detach adherent cells with a cell lifter. Flush the well bottom with a P1000 pipette, transfer all content into a 1.5 mL centrifuge tube (or 96-deep well round-bottom plate), and centrifuge at 500 <i>g</i> for 5 min at 4°C to collect the cells and proceed to surface staining with fluorescently labeled anti-mouse CD11c antibody:
<ol type="i">
<li>Centrifuge cells at 500 <i>g</i> for 5 min at 4°C, discard the supernatant by vacuum aspiration or by inverting and gently flicking down the plate and remove residual liquid by touching a paper tissue.</li>
<li>Resuspend the cell pellet with 200 μL FACS buffer containing 1/1000 diluted Fc block antibody (anti-CD16/CD32), transfer the cell suspension to a 96-well V-shape plate, and incubate for 10 min at 4°C in the dark.</li>
<li>Spin the plate at 500 <i>g</i> for 5 min at 4°C, discard the supernatant by inverting and gently flicking down the plate and remove residual liquid by touching a paper tissue.</li>
<li>Resuspend the pellet with 50 μL of APC-conjugated anti-CD11c antibody (1/100 diluted in FACS buffer), mix by pipetting up-and-down several times and incubate for 30 min at 4°C in the dark.</li>
<li>Add 150 μL of FACS buffer per well and repeat step iii), then resuspend the pellet with 50 μL FACS buffer containing 10% PFA, mix by pipetting up-and-down several times and incubate for 20 min at 4°C in the dark.</li>
<li>Add 200 μL FACS buffer per well and repeat step iii). Repeat this step twice more to wash out residential PFA.</li>
<li>Resuspend with 100 μL FACS buffer and assess the sample by flow cytometry to quantify the proportion of CD11c and CellTracker double positive events as a proxy for DC mediated phagocytosis.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is important to mix and resuspend the cell pellet immediately upon addition of PFA.</div>
<div class="critical">
<span class="critical-title">Critical:</span> All steps involving the manipulation of PFA (from sub steps v to vii) should be handled in a chemical foam hood.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Fixed samples can be stored at 4°C for up to 2 weeks for flowcytometric analysis.</div>
<h3 id="sec3.5">Evaluation of DC antigen-cross presentation capability <i>in vitro</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days cell differentiation + 48 h handling time.</div>
<p>Strong evidence suggest that DC-mediated antigen cross-presentation is paramount for induction of anticancer immunity (<a href="#bib11">Galluzzi et al., 2017</a>). This major step describes the protocol for the evaluation of antigen cross-presentation <i>in vitro</i> using soluble ovalbumin (OVA) as a model antigen (<a href="#fig1">Figure 1</a>C). Cross-presentation efficiency is measured by the readout of B3Z hybridoma T cell line in terms of interleukin 2 (IL-2) production, which allows for the quantitative comparisons between de-iniDCs with different genotypes</p>
<ol start="19">
<li>Collect well-differentiated de-iniDCs and dilute to 5×10<sup>5</sup> cells/mL with DF medium. Seed 100 μL cell suspension/well in 96-well tissue culture U-bottom plates (equal to 5×10<sup>4</sup> cells/well) and let adapt in the 37°C incubator. A typical experiment includes triplicate wells for each condition, determine well numbers according to designed control or treatments.</li>
<li>Prepare the 2×OVA working solution by adding 1 mL of the 100 mg/mL stock solution to 50 mL DF medium, and prepare the 2×OVA peptide SIINFEKL working solution by serially diluting the stock solution to 4 ng/mL. Add 100 μL of 2× OVA, 2× SIINFEKL, or non-antigen supplemented DF medium to each well containing the de-iniDCs, gently shake the plate to mix and then incubate the cells for 4 h in the cell culture incubator. Please refer to <a href="#fig5">Figure 5</a> for a brief scheme.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Typical plate design for the <i>in vitro</i> antigen cross-presentation assay including necessary control wells</div>
<p>De-iniDCs are seeded in 96-well plates at 5×10<sup>4</sup> cells/well in 100 μL DF medium, while ovalbumin (OVA) and OVA257-264 (SIINFEKL, SL8) peptide are prepared at 2× concentration for the exemplified plate layout (as described in Step 19). Wells for the ELISA standard curve should be reserved.</p>
</figcaption>
</figure>
</li>
<li>Spin the plate at 500 <i>g</i> for 5 min, discard the supernatant containing OVA, and add 200 μL/well RPMI 1640 medium (without any additives). NO NEED TO MIX at this stage. Spin the plate again at 500 <i>g</i> for 5 min and discard the supernatant. Repeat this procedure once, i.e., 2 washes in total.</li>
<li>Prepare B3Z cells in DF medium at 5×10<sup>5</sup> cells/mL.</li>
<li>After the last centrifugation, discard the DC supernatant and add 200 μL B3Z cell suspension (in total 1×10<sup>5</sup> cells) to the DCs. NO NEED TO MIX at this stage.</li>
<li>Incubate for 18 h at 37°C. Spin the plates at 500 <i>g</i> for 5 min and gently aspirate 100 μL supernatant for the quantification of IL-2 secretion as described in step 16, or with a selected commercial ELISA assay.</li>
</ol>
<p><a href="#sec7.11">Troubleshooting 6</a> and <a href="#sec7.13">7</a></p>
<h3 id="sec3.6">DC <i>in vivo</i> administration for tumor growth control</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 days.</div>
<p>This major step describes the establishment of the MCA205 subcutaneous tumors in C57BL/6J mice and the adaptive transfer of <i>ex vivo</i> primed de-iniDCs (<a href="#fig1">Figure 1</a>D). This allows for the evaluation of antitumor efficacy of intratumorally delivered gene-edited de-iniDCs, thus establish a direct link between a genotype and the corresponding antitumor phenotype of DCs.</p>
<ol start="25">
<li>Brief technical procedures for the establishment of MCA205 fibrosarcoma are:
<ol type="a">
<li>Prepare MCA205 cells in T175 tissue culture flasks at a confluency of 50%–75%, (approximately 3-5×10<sup>7</sup>/flask). Detach cells as described in the “<a href="#before-you-begin">before you begin</a>-culture of cells” section, and pass the cell suspension through a 70 μm cell strainer into a 50 mL falcon tube to obtain single cell suspension. Centrifuge at 500 <i>g</i> for 5 min at room temperature, discard supernatant and resuspend with 10 mL cold PBS, mix well to obtain a homogenous cell suspension for counting, and fill the falcon tube with cold PBS to 50 mL. Centrifuge at 500 <i>g</i> for 5 min at room temperature, discard supernatant, and resuspend the pellet with an appropriate volume of cold PBS to adjust the cell concentration to 5×10<sup>6</sup> cells/mL</li>
<li>Shave the right flank of C57Bl/6 mice (preferably six- to ten-week-old female animals) and subcutaneously inject 100 μL cell suspension under isoflurane anesthesia (day 0). Generally palpable tumors form around day 7.</li>
<li>Prepare de-iniDCs for intratumoral injection. At day 3, collect iniDC CAS9 cells or any gene-modified iniDC CAS9 cells as described in the previous sections, adjust cell density to 1×10<sup>6</sup>/mL in DF medium, put 40 mL cell suspension/150 mm tissue-culture dishes, which equals to 4×10<sup>7</sup> cells/dish. At day 7, remove used medium and replace with 40 mL of fresh DF medium contains lysate yield from 4×10<sup>7</sup> MCA205 cells, and incubate for 2 h. Collected de-iniDCs by scrapping the cell culture layer with a plastic cell lifter. Centrifuge at 500 <i>g</i> for 5 min, resuspend with 10 mL cold PBS, mix to homogeneous suspension and pass through a 70 μm for cell counting, and fill the falcon tube with PBS to 50 mL. Centrifuge at 500 <i>g</i> for 5 min and resuspend pellet with appropriate volume of cold PBS to adjust cell concentration to 2×10<sup>7</sup> cells/mL. Keep the cells on ice until injection.</li>
<li>Measure the longest dimension and perpendicular dimension of established tumors with a digital caliper, calculate the tumor surface following the formula:<br> longest dimension (mm) × perpendicular dimension (mm) × π/4.</li>
<li>Randomize mice according to their tumor sizes and distribute into groups, containing a vehicle control group (injection of PBS), a parental DC groups (injection of CAS9-expressing de-iniDCs), and other groups for injection of iniDCs with different genotypes. For intratumoral injection, the mouse is subjected to isoflurane anesthesia (2.5% with a flow rate of 1.5 L/min) in an anesthesia box, and is then maintained in an anesthesia mask with the same flow rate of isoflurane, on a heating pad warmed at 37°C. Disinfect the tumor area with 70% ethanol followed by iodine to maintain sterility, and inject 50 μL of de-iniDC suspension or cold PBS (as vehicle control) into the tumor bed. Repeat the de-iniDC administration every 4 days until day 19 (i.e., 4 intratumoral injections per animal in total), and measure the tumor size regularly to generate the growth curve. Mice with tumors exceeding 250 mm<sup>2</sup> or with apparent necrotic malignancies need to be euthanized.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Mycoplasma contamination will negatively affect tumor formation in immunocompetent mice and need to be avoided. The number of animals in each group and experimental endpoints need to be defined according to the advice and regulations of the local ethics committee.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Dendritic cells (DCs) are professional antigen presenting cells that bridge innate and acquired immunity in the sense that they can be alerted by non-antigenic (adjuvant) signals to migrate into tissues in which infection or other types of damage occur, locally capture antigenic material in the form of proteins, and then present peptide fragments from such proteins to T lymphocytes (<a href="#bib3">Anderson et al., 2020</a>; <a href="#bib33">Steinman and Banchereau, 2007</a>; <a href="#bib35">Wculek et al., 2020</a>). Cross-presentation involves the presentation of engulfed proteins after their digestion into short peptides (usually 8–10 amino acids) that bind to major histocompatibility (MHC) Class I molecules on the surface of DCs to interact with specific T cell receptors (TCRs) expressed on CD8+ cytotoxic T lymphocytes (<a href="#bib6">Cruz et al., 2017</a>; <a href="#bib13">Gros and Amigorena, 2019</a>). Since cross-presentation plays a decisive role in the cellular immune response against viral and tumor-associated antigens, DCs are believed to dictate whether a T cell response is initiated or not, and even to determine which polarity such a T cell response will acquire (<a href="#bib1">Alloatti et al., 2017</a>; <a href="#bib22">Ma et al., 2011</a>; <a href="#bib27">Roberts et al., 2016</a>). In a way, DCs function as semiotic units (<a href="#bib21">Lopez-Otin and Kroemer, 2021</a>) that obtain information from their microenvironment by multiple receptors including pattern recognition receptors (<a href="#bib15">Kawai and Akira, 2011</a>), cytokine receptors (<a href="#bib7">Dwarshuis et al., 2017</a>), hormone receptors (<a href="#bib36">Yang et al., 2019</a>) and others, to take ‘decisions’ whether to migrate (chemotaxis), to approach stressed cells (to form a synapse), to engulf antigenic material (by phagocytosis or pinocytosis), to mature (to achieve competence in antigenic presentation) and eventually to present the antigen in the context of a diversified set of costimulatory and coinhibitory signals (including immunostimulatory and immunosuppressive cytokines, such as IL-1β, IL-6, IL-12 (p70), TNFα and TGFβ) to T lymphocytes (<a href="#bib8">Eisenbarth, 2019</a>; <a href="#bib29">Ruhland et al., 2020</a>). Immunotherapies are being developed based on the concept that the provision of indispensable signals or the removal of inhibitory circuitries may enhance the immunostimulatory DC function (<a href="#bib19">Littman, 2015</a>; <a href="#bib24">Palucka and Coussens, 2016</a>; <a href="#bib31">Sharma and Allison, 2020</a>). For this reason, genetic screens that explore DC function and conveniently relate genotypes to phenotypes are useful.</p>
<p>Usually, murine DCs must be isolated ex vivo (for instance from circulating myeloid cells or from the spleen, where they only constitute a minority of cells) or must be differentiated from precursors <i>in vitro</i> (for instance from bone marrow cells) for their subsequent functional characterization (<a href="#bib3">Anderson et al., 2020</a>; <a href="#bib33">Steinman and Banchereau, 2007</a>; <a href="#bib35">Wculek et al., 2020</a>). Human DCs, for research purposes, are typically isolated from circulating peripheral blood mononuclear cells (PBMCs) (<a href="#bib23">Nair et al., 2012</a>). As DCs are terminally differentiated cells, there are no cell lines that exhibit the full spectrum of mature DC function, rendering difficult their genetic manipulation and subsequent expansion for <i>in vitro</i> characterization and <i>in vivo</i> experimentation. To circumvent this problem, we took advantage of an immature DC cell line in which the Simian Virus (SV40) large T cell antigen (SV40LgT) is expressed under the control of a Tet-on (doxycycline-inducible) promoter and the reverse tetracycline transactivator (irtTA) fused to the ligand-binding domain of a mutated glucocorticoid receptor (<a href="#bib2">Anastassiadis et al., 2010</a>; <a href="#bib26">Richter et al., 2013</a>). Due to the capacity of SV40LgT to block retinoblastoma (RB) and tumor protein 53 (TP53) such cells are immortalized in the presence of doxycycline (DOX) and the synthetic glucocorticoid dexamethasone (DEX), yet can be de-immortalized by the simultaneous removal of both factors (<a href="#bib26">Richter et al., 2013</a>). Hence, it is possible to genetically modify immortal (and infinitely expandable) cells in the presence of DEX/DOX and then to generate de-induced or de-immortalized DCs (de-iniDCs) by the withdrawal of DEX/DOX and to study their phenotype (<a href="#bib26">Richter et al., 2013</a>).</p>
<p>In our protocol, we introduced the CRISPR/Cas9 system into conditionally immortalized DCs and selected clones that expressed Cas9 enzyme but are otherwise functionally identical to the parental cells, once they are de-immortalized and hence converted into de-iniDCs (<a href="#fig1">Figure 1</a>A). In the immortal stage, the Cas9 expressing DC precursors can be transfected with synthetic CRISPR RNAs (crRNAs) to obtain the knockout of target genes and then de-immortalized for their functional exploration <i>in vitro</i> and <i>in vivo</i> (<a href="#bib18">Le Naour et al., 2020</a>).</p>
<p>Being conditionally immortalized, the knockout cells generated by transfection with crRNAs can be expanded without limits and then de-immortalized to generate functional de-iniDCs (<a href="#bib2">Anastassiadis et al., 2010</a>; <a href="#bib26">Richter et al., 2013</a>). Such cells can be characterized <i>in vitro</i> for their functional properties such as phagocytosis of dying cells, the acquisition of maturation/activation markers (like expression of costimulatory molecules on the cell surface or the release of cytokines into the supernatant), as well as cross-presentation of peptides derived from protein to T cells (<a href="#bib18">Le Naour et al., 2020</a>). For this latter aspect, we took advantage of a protein antigen that is widely used in immunology, namely, chicken ovalbumin (OVA), which can be presented by C57Bl/6 mouse-derived DCs (which express MHC class I H2-Kb molecules) in form of the octapeptide SIINFEKL (also abbreviated SL8) (<a href="#bib28">Rotzschke et al., 1991</a>). The H2-Kb–SL8 complex is recognized by the TCR of B3Z hybridoma cell line, which produce interleukin-2 (IL-2) upon its stimulation (<a href="#bib32">Shastri and Gonzalez, 1993</a>). Hence, the measurement of the amount of IL-2 produced by B3Z cells cocultured with OVA-pulsed de-iniDCs can be used as a proxy for antigen presentation (<a href="#fig1">Figure 1</a>B). In addition, it is possible to expand gene-edited iniDC-Cas9 cells (<a href="#fig1">Figure 1</a>C), to pulse them with tumor antigens and to inject them into mice to determine their capacity to induced a therapeutic anticancer immune response. In this case, loss-of-function due to the knockout of immunostimulatory genes/proteins will compromise the capacity of DCs to reduce tumor growth (as compared to wild type DCs), while gain-of function due to the knockout of immunosuppressive genes/proteins will enhance tumor growth reduction (<a href="#fig1">Figure 1</a>D).</p>
<p>The current protocol allows for the identification of genes that affect DC functions including distinct steps of the process of antigen presentation (phagocytosis, differentiation, maturation, expression of co-inhibitory or co-stimulatory molecules, cytokine secretion…). Immature DCs expressing Cas9 validated by immunoblot (<a href="#fig6">Figure 6</a>A) can be subjected to the knockout of the genes of interest by suitable guidance RNAs (<a href="#fig6">Figure 6</a>B). Most DC functions remain detectable in DCs upon stable expression of Cas9, as documented for the induction of cytokines (<a href="#fig6">Figures 6</a>C–6G) or activation/maturation markers (<a href="#fig6">Figures 6</a>H–7N) by bacterial lipopolysaccharide (LPS) or tumor necrosis factor α, as well as MHC Class I-restricted antigen presentation to CD8<sup>+</sup> T cells (<a href="#fig6">Figures 6</a>N and 6O). However, the augmentation of CD83, CD86, and MHCII expression in response to stimulation with LPS or TNFα is attenuated. Cas9 expression did not interfere neither with phagocytosis of dying tumor cells (<a href="#fig7">Figures 7</a>A and 7B), the induction of cytokines (<a href="#fig7">Figures 7</a>C–7G) or activation/maturation markers (<a href="#fig7">Figures 7</a>H–7M) by tumor cell lysates, or the inhibition of <i>in vivo</i> tumor growth by DCs (<a href="#fig7">Figure 7</a>N). In contrast, transfection with guidance RNAs can induce major functional changes in Cas9-expressing DCs (<a href="#fig8">Figure 8</a>A), as exemplified for the knockout of <i>Fpr1</i>, which reduces the production of cytokines by LPS or tumor lysate (<a href="#fig8">Figures 8</a>B–8D), compromises antigen presentation (<a href="#fig8">Figure 8</a>E), reduces phagocytosis of dying tumor cells (<a href="#fig8">Figure 8</a>F) and compromises the reduction of fibrosarcoma growth by locally injected DCs (<a href="#fig8">Figure 8</a>G). Conversely, other guidance RNAs can improve DC function in antigen presentation assays (<a href="#fig9">Figure 9</a>A), as exemplified for the knockout of Clec4a2, which enhances the presentation of protein or peptide to T cells (<a href="#fig9">Figure 9</a>B), enhances the phagocytosis of dying cancer cells (<a href="#fig9">Figure 9</a>C) and improves cancer growth control by intratumorally injected DCs (<a href="#fig9">Figure 9</a>D). Thus, the effects of gene knockouts may be inhibitory, suggesting the implication of the corresponding gene product in antigen presentation, or stimulatory, suggesting that the gene/protein has a checkpoint function. As usually in screening experiments, hits must be carefully validated in repeat experiments on a small scale, as well as by orthogonal approaches (such as parallel <i>in vitro</i> and <i>in vivo</i> experiments) to make sure that the incriminated gene/protein truly impacts DC function. This experimental strategy allows for the exclusion of false-positive hits. False-negative results may also be further investigated. Thus, failure of a gene-specific gRNA to induce a functional phenotype must be followed up by control experiments evaluating the effective removal of the gene/protein that has been targeted, for example by genomic sequencing, RT-PCRs or immunoblot.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Characterization of the CAS9-expressing inducible immortalized dendritic cells (iniDC CAS9)</div>
<p>(A) Validation of CAS9 expression in iniDC CAS9 clones by immunoblot.</p>
<p>(B) Hmgb1 expression in iniDC CAS9 cells that were transfected with crRNA targeting murine <i>Hmgb1</i> gene (two gRNA sequence labeled as H1 and H2), or the non-target control gRNA (NT). Different transfection conditions (0.25 or 0.5×10<sup>6</sup> cells/well in 12-well plate, 1 or 2×10<sup>6</sup> cells/well in 6-well plate) were tested.</p>
<p>(C–M) Maturation and activation of DCs. BMDC, de-iniDC and de-iniDC CAS9 were treated with LPS or recombinant TNFα for 16 h. Supernatants were collected for ELISA quantification and cytokine release is reported as bar charts (C–G). Cells were subjected to antibody staining of surface markers and analyzed by flow cytometry. Representative histograms indicating the increase in the expression of surface marker and normalized median fluorescence intensity (MFI) are shown in (H–M).</p>
<p>Comparison of IL-2 production (N) as a proxy of OVA antigen cross-presentation to B3Z hybridoma by DCs was evaluated as schematically depicted in (O). Nonantigen loaded- and SIINFEKL peptide (SL8)-incubated DCs were used as negative and positive controls, respectively. Bar charts express means ± SD of triplicate assessments, ND, nondetectable. Statistical differences were calculated by means of an ANOVA test (Dunnett's multiple comparisons test), with ∗p &lt; 0.05, ∗∗p &lt; 0.01, ∗∗∗p &lt; 0.001 as compared to the untreated condition (UT).</p>
</figcaption>
</figure>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Activation of the CAS9 expressing-inducible immortalized dendritic cells (iniDC CAS9) in response to synergistic tumor cells</div>
<p>(A and B) <i>In vitro</i> phagocytosis of untreated (UT) or crizotinib-treated (CRIZO) MCA205 cells by bone marrow-derived DC (BMDC), as well as iniDC and iniDC CAS9 that were differentiated for 4 days into de-iniDC and de-iniDC CAS9, respectively. MCA205 cells were pre-stained with CellTracker blue. Representative dot plots (A) and the percentage of CellTracker<sup>+</sup>/CD11C<sup>+</sup> cells (B) are reported.</p>
<p>(C–M) BMDC, de-iniDC and de-iniDC CAS9 were treated with the lysate of MCA205 cells for 16 h. Supernatant was collected for cytokine quantification by ELISA (C-G), while cells were subjected to an immunostaining of surface markers and analyzed by flow cytometry (H–M). Cytokine concentration in cell culture supernatants and median fluorescence intensity (MFI) of DC surface markers are reported as bar charts (means ± SD of triplicates). Statistical differences were calculated by means of ANOVA test (Dunnett's multiple comparisons test), with ∗p &lt; 0.05, ∗∗p &lt; 0.01, ∗∗∗p &lt; 0.001 as compared to the untreated condition (UT). Representative histograms of flowcytometric analysis are shown in (H). Differentiated iniDC and iniDC CAS9 cells (de-iniDC) were pulsed with tumor cell lysates and intratumorally <i>(i.t.</i>) injected into subcutaneous MCA205 tumors grafted on immunocompetent mice to test their immunotherapeutic potential.</p>
<p>Tumor growth curves are reported in (N) as mean ± SEM (n = 6). Statistical significance was calculated by means of the ANOVA Type 2 (Wald test) and significant results are indicated as ∗∗∗p&lt;0.001 as compared with the PBS injection group.</p>
</figcaption>
</figure>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Screen for genes that modulate dendritic cell phenotype and function</div>
<p>CAS9 expressing-inducible immortalized dendritic cells (iniDC CAS9) were transfected with specific CRISPR RNAs (crRNA) to perform gene knockouts. Dexamethasone and doxycycline (DEX/DOX) were removed from the culture system during transfection and recovery of cells to differentiate iniDC CAS9 cells into primary de-iniDC CAS9 which were then used for assessing the phagocytosis of dying cancer cells (Phago), quantification of IL-1β, IL-6, and TNFα production upon stimulation with LPS, as well as the assessment of antigen cross-presentation capacity upon ovalbumin (OVA) treatment by interleukin-2 (IL-2) quantification in B3Z hybridoma co-culture supernatants. For each parameter, the readouts of crRNA-transfected wells were normalized to non-target control crRNA. Averaged ratios are summarized as a heatmap (A), in which Fpr1 is highlighted as a candidate gene whose KO leads to a loss of function phenotype in DC. Fpr1 KO de-iniDC CAS9 cells (Fpr1<sup>−/−</sup>) were analyzed for cytokine production (B–D) upon stimulation with LPS or MCA205 cells lysate, cross-presentation of OVA antigen or SINFKEL peptide (SL8) (E), and phagocytosis of MCA205 cells that were either untreated (UT) or crizotinib (CRIZO)-treated (F) Cytokine concentration and phagocytic events (CD11C<sup>+</sup> CellTracker<sup>+</sup> cells) are reported as bar charts (means ± SD of triplicates). Statistical differences were calculated by means of the ANOVA test (Dunnett's multiple comparisons test), with ∗∗p &lt; 0.01, ∗∗∗p &lt; 0.001 as compared to UT, <sup>###</sup>p&lt;0.001 or <sup>$$$</sup>p&lt;0.001 as compared between Fpr1<sup>−/−</sup> and wild type (wt) de-iniDC CAS9 cells with the same stimuli. Differentiated iniDC and iniDC CAS9 cells (de-iniDC) were pulsed with tumor cell lysate and intratumorally (<i>i.t.</i>) injected into subcutaneous MCA205 tumors grafted on immunocompetent mice to test their (immuno)therapeutic potential. Tumor growth curves are reported in (G) as mean ± SEM (n = 6). Statistical significance was calculated by means of the ANOVA Type 2 (Wald test) and significant results are indicated as ∗P&lt;0.05 compared to the PBS group.</p>
</figcaption>
</figure>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Screen for genes that modulate <i>in vitro</i> antigen cross-presentation by DC</div>
<p>CAS9 expressing-inducible immortalized dendritic cells (iniDC CAS9) were transfected with specific CRISPR RNAs (crRNA) to perform gene knockout during differentiation into de-iniDC CAS9. Gene edited de-iniDC CAS9 were incubated with ovalbumin (OVA) and cocultured with B3Z hybridoma cells to quantify interleukin-2 (IL-2) secretion as an indicator for antigen cross-presentation capacity. A violin plot is used to depict the IL-2 concentrations (mean value of duplicates) (A). The nonantigen loaded- and SIINFEKL peptide (SL8) loaded-DCs were used as negative and positive controls, respectively. Candidate gene whose KO leads to defective (blue color) or increased (red color) antigen cross-presentation are highlighted. De-iniDC CAS9 cells that are KO for Clec4a2 (Clec4a2<sup>−/−</sup>) were tested for cross-presentation of OVA antigen or the SINFKEL peptide (SL8) (B), and phagocytosis of MCA205 cells that were untreated (UT) or crizotinib (CRIZO)-treated (C). Cytokine concentration and phagocytic events (CD11C<sup>+</sup> CellTracker<sup>+</sup> cells) are reported as bar charts (means ± SD of triplicates). Statistical differences were calculated by means of the ANOVA test (Dunnett's multiple comparisons test), with ∗∗∗p &lt; 0.001 as compared to UT, <sup>###</sup>p&lt;0.001 or <sup>$$$</sup>p&lt;0.001 as compared between Fpr1<sup>−/−</sup> and wild type (wt) de-iniDC CAS9 cells with the same stimuli. Differentiated iniDC and iniDC CAS9 cells (de-iniDC) were pulsed with tumor cell lysates and intratumorally (<i>i.t.</i>) injected into subcutaneous MCA205 tumors grafted on immunocompetent mice to test their immunotherapeutic potential. Tumor growth curves are reported in (D) as mean ± SEM (n = 6). Statistical significance was calculated by means of the ANOVA Type 2 (Wald test) and significant results are indicated as ∗p&lt;0.05 and ∗∗∗p&lt;0.001 compared to the PBS injection group.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantitative-and-statistical-analysis">Quantitative and statistical analysis</h2>
<p>Unless otherwise mentioned, data are reported as means ± SD of triplicate determinations, and experiments were repeated several times yielding similar results. Statistical significance was assessed by means of an ANOVA test (Dunnett's multiple comparisons test) or ANOVA Type 2 (Wald test). TumGrowth was used to analyze in vivo data (Enot et al.). TumGrowth is free available at Github/Kroemerlab. p values of 0.05 or less were considered to denote significance.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>A published study used mice expressing Cas9 in a constitutive fashion in all cell types to generate bone marrow-derived DCs (BM-DCs) that were then subjected to transduction with a lentivirus-encoded, bar-coded crRNA library, followed by the stimulation of the BM-DCs with bacterial lipopolysaccharide (LPS) and the purification of cells producing tumor necrosis factor-α (TNFα). This protocol required permeabilization, fixation and immunostaining of the DCs to detect TNFα. Subsequent deep sequencing of the DNA recovered from the cells allowed for the identification of novel regulators of LPS-induced TNFα induction (<a href="#bib25">Parnas et al., 2015</a>). This approach has the advantage of using primary cells, yet is overshadowed by limitations in the quantity of the cells used for the screening, as well as the necessity to kill the cells with the desired phenotype, hence imposing several rounds of validation experiments with new batches of short-lived primary BM-DCs.</p>
<p>The limitation of using primary BM-DCs has been overcome by using immortal cell lines. So-called MutuDCs originate from splenic tumors obtained from CD11c:SV40LgT-transgenic C57BL/6 mice. Such cells have the phenotype and the functional characteristics of conventional type 1 DCs (cDC1) and can be used <i>in vitro</i>, in cross-presentation assays (<a href="#bib10">Fuertes Marraco et al., 2012</a>). However, they present the disadvantage that they are tumor-derived (and hence likely present unexpected genetic alterations) and cannot be de-immortalized on command. As a result, they potentially form tumors if inoculated into histocompatible mice. Nonetheless, such cells are amenable to genetic or pharmacological screens <i>in vitro</i> (<a href="#bib16">Kozik et al., 2020</a>).</p>
<p>There a several subpopulations of DCs that differ in their phenotype and functional characteristics (<a href="#bib4">Balan et al., 2019</a>; <a href="#bib34">Villar and Segura, 2020</a>). The CRISPR/Cas9-based screen that we describe here uses de-iniDCs that adopt the phenotype of monocyte-derived DCs (moDCs) since they are cultured in the continuous presence of granulocyte monocyte-colony stimulating factor (GM-CSF), before and after DEX/DOX withdrawal. Hence, the insights obtained with these cells may be limited to this particular DC subtype, pending further exploration.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Sticky cell lysates after boiling; diffuse bands during western blot (step 3).</p>
<p>Cause: Insufficient volume of buffer per each sample; insufficient denaturation of proteins; crystallization of SDS</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Increase the volume of rapid western blotting protein preparation buffer for each sample, vigorously shake the plate on a microplate shaker after adding the rapid western blotting protein preparation buffer, and prolong the incubation at 100°C. Reboil the samples at 100°C for 5 min and cool to room temperature before loading the gels.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Cause: Insufficient washing or blocking; Contamination of HRP in plastics or instruments.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Make sure the washing buffer stays long enough (&gt;30 s) in the wells. If manipulating several plates simultaneously, process the plates in the same order to avoid variation due to uneven washing; Increase the protein concentration in blocking buffer, e.g., 10% FBS, 2% BSA.</p>
<p>Use new plastics for each step, thoroughly wash tubing systems when automated liquid dispensers are used; carefully clean heads of multichannel pipettes.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>High variability between replicates in customized ELISA (step 16)</p>
<p>Cause: Contamination of cells when collecting supernatant; insufficient mixing of samples; uneven adding of liquids; bubble formation in wells before assessment</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Be careful when collecting supernatant from cell cultures, keep a safe distance between tips and the well bottom; carefully mix samples when applying dilution and before loading to coated plates.</p>
<p>If feasible, use an automated liquid dispenser or verified multichannel pipettes.</p>
<p>Centrifuge plates before measuring the absorbance</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Variation between plates in customized ELISA (step 16)</p>
<p>Cause: Distinct incubation periods.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Precisely control the incubation time of each plate</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Edge effects in customized ELISA (step 16)</p>
<p>Cause: Solution not at room temperature. Tips not well inserted in the multichannel pipettes.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Bring solutions to room temperature in advance; use verified multichannel pipettes; ensure all tips are loaded correctly before and during all pipetting procedures.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Highly uniform values between conditions for the antigen cross presentation assay. (step 24)</p>
<p>Cause: Insufficient washout of OVA antigen before co-culture with B3Z cells. Wrong ratio of DC to B3Z cells</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Ensure complete washout of non-bound OVA before co-culture with B3Z, make sure no residual solution remains in wells between washing steps. Adjust the ratio of DC to B3Z cells per well.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>Very low concentration of IL-2 secretion. (step 24)</p>
<p>Cause: Extended co-culture period leads to increased cell death of B3Z</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Control the co-culture period with B3Z cells.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Alloatti, A., Rookhuizen, D.C., Joannas, L., Carpier, J.M., Iborra, S., Magalhaes, J.G., Yatim, N., Kozik, P., Sancho, D., Albert, M.L., et al. (2017). Critical role for Sec22b-dependent antigen cross-presentation in antitumor immunity. J. Exp. Med. <i>214</i>, 2231-2241. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref1">View at publisher</a></p>
<p id="bib2">Anastassiadis, K., Rostovskaya, M., Lubitz, S., Weidlich, S., and Stewart, A.F. (2010). Precise conditional immortalization of mouse cells using tetracycline-regulated SV40 large T-antigen. Genesis <i>48</i>, 220-232. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref2">View at publisher</a></p>
<p id="bib3">Anderson, D.A., Dutertre, C.A., Ginhoux, F., and Murphy, K.M. (2020). Genetic models of human and mouse dendritic cell development and function. Nat. Rev. Immunol. <i>21</i>, 101-115. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref3">View at publisher</a></p>
<p id="bib4">Balan, S., Saxena, M., and Bhardwaj, N. (2019). Dendritic cell subsets and locations. Int. Rev. Cell Mol. Biol. <i>348</i>, 1-68. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref4">View at publisher</a></p>
<p id="bib5">Cavaillon, J.M. (2001). Pro- versus anti-inflammatory cytokines: myth or reality. Cell. Mol. Biol. <i>47</i>, 695-702. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref5">View at publisher</a></p>
<p id="bib6">Cruz, F.M., Colbert, J.D., Merino, E., Kriegsman, B.A., and Rock, K.L. (2017). The biology and underlying mechanisms of cross-presentation of exogenous antigens on MHC-I molecules. Annu. Rev. Immunol. <i>35</i>, 149-176. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref6">View at publisher</a></p>
<p id="bib7">Dwarshuis, N.J., Parratt, K., Santiago-Miranda, A., and Roy, K. (2017). Cells as advanced therapeutics: State-of-the-art, challenges, and opportunities in large scale biomanufacturing of high-quality cells for adoptive immunotherapies. Adv. Drug Deliv. Rev. <i>114</i>, 222-239. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref7">View at publisher</a></p>
<p id="bib8">Eisenbarth, S.C. (2019). Dendritic cell subsets in T cell programming: location dictates function. Nat. Rev. Immunol. <i>19</i>, 89-103. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref8">View at publisher</a></p>
<p id="bib9">Enot, D.P., Vacchelli, E., Jacquelot, N., Zitvogel, L., and Kroemer, G. (2018). TumGrowth: An open-access web tool for the statistical analysis of tumor growth curves. Oncoimmunology <i>7</i>, e1462431. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref9">View at publisher</a></p>
<p id="bib10">Fuertes Marraco, S.A., Grosjean, F., Duval, A., Rosa, M., Lavanchy, C., Ashok, D., Haller, S., Otten, L.A., Steiner, Q.G., Descombes, P., et al. (2012). Novel murine dendritic cell lines: a powerful auxiliary tool for dendritic cell research. Front. Immunol. <i>3</i>, 331. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref10">View at publisher</a></p>
<p id="bib11">Galluzzi, L., Buque, A., Kepp, O., Zitvogel, L., and Kroemer, G. (2017). Immunogenic cell death in cancer and infectious disease. Nat. Rev. Immunol. <i>17</i>, 97-111. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref11">View at publisher</a></p>
<p id="bib12">Ghiringhelli, F., Puig, P.E., Roux, S., Parcellier, A., Schmitt, E., Solary, E., Kroemer, G., Martin, F., Chauffert, B., and Zitvogel, L. (2005). Tumor cells convert immature myeloid dendritic cells into TGF-beta-secreting cells inducing CD4+CD25+ regulatory T cell proliferation. J. Exp. Med. <i>202</i>, 919-929. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref12">View at publisher</a></p>
<p id="bib13">Gros, M. and Amigorena, S. (2019). Regulation of antigen export to the cytosol during cross-presentation. Front. Immunol. <i>10</i>, 41. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref13">View at publisher</a></p>
<p id="bib14">Jones, L.A., Kreem, S., Shweash, M., Paul, A., Alexander, J., and Roberts, C.W. (2010). Differential modulation of TLR3- and TLR4-mediated dendritic cell maturation and function by progesterone. J. Immunol. <i>185</i>, 4525-4534. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref14">View at publisher</a></p>
<p id="bib15">Kawai, T. and Akira, S. (2011). Toll-like receptors and their crosstalk with other innate receptors in infection and immunity. Immunity <i>34</i>, 637-650. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref15">View at publisher</a></p>
<p id="bib16">Kozik, P., Gros, M., Itzhak, D.N., Joannas, L., Heurtebise-Chretien, S., Krawczyk, P.A., Rodriguez-Silvestre, P., Alloatti, A., Magalhaes, J.G., Del Nery, E., et al. (2020). Small molecule enhancers of endosome-to-cytosol import augment anti-tumor immunity. Cell Rep. <i>32</i>, 107905. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref16">View at publisher</a></p>
<p id="bib17">Kurts, C., Heath, W.R., Carbone, F.R., Allison, J., Miller, J.F., and Kosaka, H. (1996). Constitutive class I-restricted exogenous presentation of self antigens in vivo. J. Exp. Med. <i>184</i>, 923-930. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref17">View at publisher</a></p>
<p id="bib18">Le Naour, J., Liu, P., Zhao, L., Adjemian, S., Sztupinszki, Z., Taieb, J., Mulot, C., Silvin, A., Dutertre, C.A., Ginhoux, F., et al. (2020). A TLR3 ligand reestablishes chemotherapeutic responses in the context of FPR1 deficiency. Cancer Discov. <i>11</i>, 408-423. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref18">View at publisher</a></p>
<p id="bib19">Littman, D.R. (2015). Releasing the brakes on cancer immunotherapy. Cell <i>162</i>, 1186-1190. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref19">View at publisher</a></p>
<p id="bib20">Liu, P., Zhao, L., Pol, J., Levesque, S., Petrazzuolo, A., Pfirschke, C., Engblom, C., Rickelt, S., Yamazaki, T., Iribarren, K., et al. (2019). Crizotinib-induced immunogenic cell death in non-small cell lung cancer. Nat. Commun. <i>10</i>, 1486. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref20">View at publisher</a></p>
<p id="bib21">Lopez-Otin, C. and Kroemer, G. (2021). Hallmarks of health. Cell <i>184</i>, 33-63. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref21">View at publisher</a></p>
<p id="bib22">Ma, Y., Aymeric, L., Locher, C., Kroemer, G., and Zitvogel, L. (2011). The dendritic cell-tumor cross-talk in cancer. Curr. Opin. Immunol. <i>23</i>, 146-152. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref22">View at publisher</a></p>
<p id="bib23">Nair, S., Archer, G.E., and Tedder, T.F. (2012). Isolation and generation of human dendritic cells. Curr. Protoc. Immunol. <i>7</i>, Unit7 32. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref23">View at publisher</a></p>
<p id="bib24">Palucka, A.K. and Coussens, L.M. (2016). The basis of oncoimmunology. Cell <i>164</i>, 1233-1247. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref24">View at publisher</a></p>
<p id="bib25">Parnas, O., Jovanovic, M., Eisenhaure, T.M., Herbst, R.H., Dixit, A., Ye, C.J., Przybylski, D., Platt, R.J., Tirosh, I., Sanjana, N.E., et al. (2015). A genome-wide CRISPR screen in primary immune cells to dissect regulatory networks. Cell <i>162</i>, 675-686. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref25">View at publisher</a></p>
<p id="bib26">Richter, C., Thieme, S., Bandola, J., Laugsch, M., Anastassiadis, K., and Brenner, S. (2013). Generation of inducible immortalized dendritic cells with proper immune function in vitro and in vivo. PLoS One <i>8</i>, e62621. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref26">View at publisher</a></p>
<p id="bib27">Roberts, E.W., Broz, M.L., Binnewies, M., Headley, M.B., Nelson, A.E., Wolf, D.M., Kaisho, T., Bogunovic, D., Bhardwaj, N., and Krummel, M.F. (2016). Critical role for CD103(+)/CD141(+) dendritic cells bearing CCR7 for tumor antigen trafficking and priming of T cell immunity in melanoma. Cancer Cell <i>30</i>, 324-336. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref27">View at publisher</a></p>
<p id="bib28">Rotzschke, O., Falk, K., Stevanovic, S., Jung, G., Walden, P., and Rammensee, H.G. (1991). Exact prediction of a natural T cell epitope. Eur. J. Immunol. <i>21</i>, 2891-2894. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref28">View at publisher</a></p>
<p id="bib29">Ruhland, M.K., Roberts, E.W., Cai, E., Mujal, A.M., Marchuk, K., Beppler, C., Nam, D., Serwas, N.K., Binnewies, M., and Krummel, M.F. (2020). Visualizing synaptic transfer of tumor antigens among dendritic Cells. Cancer Cell <i>37</i>, 786-799 e785. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref29">View at publisher</a></p>
<p id="bib30">Scheller, J., Chalaris, A., Schmidt-Arras, D., and Rose-John, S. (2011). The pro- and anti-inflammatory properties of the cytokine interleukin-6. Biochim. Biophys. Acta <i>1813</i>, 878-888. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref30">View at publisher</a></p>
<p id="bib31">Sharma, P. and Allison, J.P. (2020). Dissecting the mechanisms of immune checkpoint therapy. Nat. Rev. Immunol. <i>20</i>, 75-76. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref31">View at publisher</a></p>
<p id="bib32">Shastri, N. and Gonzalez, F. (1993). Endogenous generation and presentation of the ovalbumin peptide/Kb complex to T cells. J. Immunol. <i>150</i>, 2724-2736. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref32">View at publisher</a></p>
<p id="bib33">Steinman, R.M. and Banchereau, J. (2007). Taking dendritic cells into medicine. Nature <i>449</i>, 419-426. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref33">View at publisher</a></p>
<p id="bib34">Villar, J. and Segura, E. (2020). Decoding the heterogeneity of human dendritic cell Subsets. Trends Immunol. <i>41</i>, 1062-1071. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref34">View at publisher</a></p>
<p id="bib35">Wculek, S.K., Cueto, F.J., Mujal, A.M., Melero, I., Krummel, M.F., and Sancho, D. (2020). Dendritic cells in cancer immunology and immunotherapy. Nat. Rev. Immunol. <i>20</i>, 7-24. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref35">View at publisher</a></p>
<p id="bib36">Yang, H., Xia, L., Chen, J., Zhang, S., Martin, V., Li, Q., Lin, S., Chen, J., Calmette, J., Lu, M., et al. (2019). Stress-glucocorticoid-TSC22D3 axis compromises therapy-induced antitumor immunity. Nat. Med. <i>25</i>, 1428-1441. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00439-1/sref36">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Oliver Kepp (<a href="mailto:captain.olsen@gmail.com">captain.olsen@gmail.com</a>).</p>
<h4>Materials availability</h4>
<p>The iniDC cell line can be provided by S.T. upon reasonable request.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze [data sets/code].</p>
<h3>Supplemental Information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/921-Mmc1.xlsx">Table S1. Predesigned synthetic guide RNAs, related to step 4</a></p>
<p>Sequences of gRNA target regions employed for CRISPR-CAS9 screening.</p>
<h3>Acknowledgments</h3>
<p>G.K. is supported by the <a href="https://doi.org/10.13039/501100004099">Ligue contre le Cancer</a> (équipe labellisée); <a href="https://doi.org/10.13039/501100001665">Agence National de la Recherche</a> (ANR) – Projets blancs; AMMICa US23/CNRS UMS3655; <a href="https://doi.org/10.13039/100007391">Association pour la recherche sur le cancer</a> (ARC); Association “Ruban Rose”; <a href="https://doi.org/10.13039/501100006431">Cancéropôle Ile-de-France</a>; <a href="https://doi.org/10.13039/501100002915">Fondation pour la Recherche Médicale</a> (FRM); a donation by Elior; Equipex Onco-Pheno-Screen; European Joint Programme on Rare Diseases (EJPRD); Gustave Roussy Odyssea, the European Union <a href="https://doi.org/10.13039/100010661">Horizon 2020</a> Projects Oncobiome and Crimson; Fondation Carrefour; High-end Foreign Expert Program in China (GDW20171100085), <a href="https://doi.org/10.13039/501100006364">Institut National du Cancer</a> (INCa); <a href="https://doi.org/10.13039/501100001677">Inserm</a> (HTE); <a href="https://doi.org/10.13039/501100004795">Institut Universitaire de France</a>; <a href="https://doi.org/10.13039/501100004100">LabEx</a> Immuno-Oncology (ANR-18-IDEX-0001); the RHU Torino Lumière; Seerave Foundation; SIRIC Stratified Oncology Cell DNA Repair and Tumor Immune Elimination (SOCRATE); and SIRIC Cancer Research and Personalized Medicine (CARPEM). This study contributes to the IdEx Université de Paris ANR-18-IDEX-0001. O.K. is supported by the DIM elicit of the Ile de France. The work was realized with the help of the Plate-form Imagerie et Cytométrie (PFIC), Gustave Roussy, Université Paris-Saclay, UMS AMMICa, Villejuif, F-94805, France</p>
<h3>Author contributions</h3>
<p>L.Zhao and P.L. constructed the CRISPR-CAS9 expressing iniDCs, designed the customized ELISA procedures, set up the genetic screening procedures, and established the procedures for animal experimentation; W.X. optimized the <i>in vitro</i> antigen cross presentation assay; S.Z. optimized the multicolor flowcytometric analysis protocols; S.T. generated the parental iniDC cell line; L.Zitvogel, O.K. and G.K. conceived the study.</p>
<h3>Declaration of interests</h3>
<p>L.Zitvogel receives consulting/advisory honoraria from Transgene, EpiVax, and Lytix; G.K. reports consulting/advisory honoraria from The Longevity Labs and Lytix and membership of the Executive Board of Bristol Myers Squibb Foundation France; L.Zitvogel and G.K. are founders of everImmune. G.K. and O.K. are founders of Samsara Therapeutics. G.K. is a founder of TheraFast. The other authors declare no competing interests.</p>
</section>
</article>