<article>
<section>
<h1 id="header">Identification of the interacting partners of a lysosomal membrane protein in living cells by BioID technique</h1>
<p><time datetime="2022-04-01">Published: April 1, 2022</time></p>
<p>Dat Nguyen-Tien,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Takehiro Suzuki,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a></sup> Toshihiko Kobayashi,<sup><a href="#aff1">1</a></sup> Noriko Toyama-Sorimachi,<sup><a href="#aff1">1</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup> and Naoshi Dohmae<sup><a href="#aff2">2</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Immunology and Inflammation, Research Institute, National Center for Global Health and Medicine, 1-21-1 Toyama, Shinjuku, Tokyo 162-8655, Japan</p>
<p id="aff2"><sup>2</sup>Biomolecular Characterization Unit, RIKEN Center for Sustainable Resource Science, Saitama 351-0198, Japan</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:nsorimachi@ri.ncgm.go.jp">nsorimachi@ri.ncgm.go.jp</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:dohmae@riken.jp">dohmae@riken.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101263">10.1016/j.xpro.2022.101263</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The purpose of this protocol is to screen and identify the physiologically relevant interactors of a lysosomal protein in living cells. Here, we describe how to identify solute carrier family 15 member 4 (SLC15A4)-interacting proteins by BioID and mass spectrometry analysis. This protocol utilizes fusion of SLC15A4 with a mutant form of biotin ligase, BirA. The fusion protein can promiscuously biotinylate the proteins proximal to SLC15A4. The biotinylated endogenous proteins are pulled down by magnetic streptavidin beads and detected by mass spectrometry analysis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1073/pnas.2100295118">Kobayashi et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Exploring the relevant interactors of a lysosomal protein in its native environment</li>
<li>Biotinylation and pulldown method of the proximal proteins with BioID fusion protein</li>
<li>Detailed steps of sample preparation for mass spectrometer and quantification analysis</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1528-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used to identify the interacting proteins of solute carrier family 15 member 4 (SLC15A4), a lysosome-resident amino acid/oligopeptide transporter, by BioID and mass spectrometry analysis (<a href="#bib1">Kobayashi et al., 2021</a>). The co-immunoprecipitation (Co-IP) is a conventional technique to identify protein-protein interactions. However, it has limitations in membrane proteins like SLC15A4 because of protein solubility or aggregation/filament formation in Co-IP lysis buffer. BioID, a new method to screen for physiologically relevant protein interactions based on a promiscuous biotin ligase (BirA) in a natural cellular setting (<a href="#bib2">Roux et al., 2013</a>), has advantages to overcome the Co-IP limitations because all of the cellular proteins are denatured and solubilized, and only those that are labeled with biotin are selectively captured. Here, we generated Control-BirA or BirA-SLC15A4-expressing HEK293T cells. The fusion ligases biotinylate adjacent endogenous proteins, which can be pulled down by magnetic streptavidin beads. These biotinylated proteins were identified by mass spectrometry. After identifying biotinylated proteins in each sample, we compared the proteins detected in the control samples (BirA only) with the BirA-SLC15A4 samples. The proteins enriched in the BirA-SLC15A4 sample represent the potentially SLC15A4-interacting candidates.</p>
<h3 id="sec1.1">Generation of cells expressing a BirA fusion protein</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 weeks</div>
<p>As an example, we constructed the cMyc-BirA-SLC15A4 (hereafter BirA-SLC15A4) expression vector by subcloning full-length human SLC15A4 cDNA into the pcDNA3.1 mycBioID vector (Addgene) and further subcloned into pEBMulti-Bsd (FUJIFILM Wako Chemicals) to obtain the blasticidin S-resistance. BirA protein was fused at the N-terminal of SLC15A4. BirA-SLC15A4 expression vector was transfected into HEK293T cells using Lipofectamine 3000 Transfection Reagent (Thermo Fisher Scientific) to generate stably expressing cells; stable transfectants were selected in 10 μg/mL blasticidin S (FUJIFILM Wako Chemicals). We created a control cell line by transfecting HEK293T cells with a pEBMulti-Met-Gly-Cys-cMyc-BirA vector (Cont-BirA). The MGC motif helps to anchor cMyc-BirA to the cell membrane, which is related subcellular localization with BirA-SLC15A4.</p>
<h3 id="sec1.2">Confirm the expression of BirA fusion protein</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<p>We performed flow cytometric analysis with c-Myc staining to confirm the expression of BioID constructs and western blotting with streptavidin-HRP (Thermo Fisher Scientific) to confirm the activity of BioID before scaling up for BioID pull down assays. If necessary, single cell cloning by limited dilution should be performed to choose the cell lines with comparable expression level. If necessary, cell viability and/or cytotoxicity assays should be performed because some recombinant fusion proteins have cytotoxic effects.</p>
<p>In addition, it is important to confirm the subcellular localization of the BirA fusion protein using fluorescence imaging whether it shows the same localization as the endogenous protein. In our case, we confirmed that both endogenous SLC15A4 and BirA-SLC15A4 showed the localization in LAMP1<sup>+</sup> compartment.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>c-Myc Monoclonal Antibody (9E10), Alexa Fluor 555 (dilute 1:100)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# MA1-980-A555</td>
</tr>
<tr>
<td>Pierce™ High Sensitivity Streptavidin-HRP (dilute 1:10000)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 21130</td>
</tr>
<tr>
<td>PE Rat anti-Mouse CD107a (LAMP1) (dilute 1:100)</td>
<td>BD Biosciences</td>
<td>Cat# 558661</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Blasticidin S</td>
<td>Nacalai Tesque</td>
<td>Cat# ant-bl-05</td>
</tr>
<tr>
<td>(+)-Biotin</td>
<td>FUJIFILM Wako Chemicals</td>
<td>Cat# 023-08716</td>
</tr>
<tr>
<td>D-MEM (high-glucose)</td>
<td>Nacalai Tesque</td>
<td>Cat# 08458-16</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Biowest</td>
<td>Cat# S1820-500</td>
</tr>
<tr>
<td>Penicillin-Streptomycin Mixed Solution (PC/SM)</td>
<td>Nacalai Tesque</td>
<td>Cat# 09367-34</td>
</tr>
<tr>
<td>Lipofectamine™ 3000 Transfection Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# L3000001</td>
</tr>
<tr>
<td>0.22-μM syringe-driven filter unit</td>
<td>Merck Millipore</td>
<td>Cat# SLGVR33RS</td>
</tr>
<tr>
<td>NaCl</td>
<td>Nacalai Tesque</td>
<td>Cat# 31320-05</td>
</tr>
<tr>
<td>Sodium Lauryl Sulfate (SDS)</td>
<td>Nacalai Tesque</td>
<td>Cat# 30400-85</td>
</tr>
<tr>
<td>Halt™ Protease Inhibitor Cocktail (100×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 78429</td>
</tr>
<tr>
<td>DC protein assay kit</td>
<td>Bio-Rad Laboratories</td>
<td>Cat# 5000111JA</td>
</tr>
<tr>
<td>Bovine Serum Albumin (BSA) Standard Ampules, 2 mg/mL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 23209</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>FUJIFILM Wako Chemicals</td>
<td>Cat# 192-08312</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat# T9284</td>
</tr>
<tr>
<td>Dynabeads MyOne Streptavidin C1 (magnetic streptavidin beads)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 65002</td>
</tr>
<tr>
<td>HEPES</td>
<td>Nacalai Tesque</td>
<td>Cat# 17514-15</td>
</tr>
<tr>
<td>NP-40 (Nonidet P40)</td>
<td>Nacalai Tesque</td>
<td>Cat# 25223-04</td>
</tr>
<tr>
<td>LiCl</td>
<td>Nacalai Tesque</td>
<td>Cat# 20624-62</td>
</tr>
<tr>
<td>Sample Buffer, Laemmli 2× Concentrate</td>
<td>Sigma-Aldrich</td>
<td>Cat# S3401-10VL</td>
</tr>
<tr>
<td>Pierce Silver Stain for Mass Spectrometry</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 24600</td>
</tr>
<tr>
<td>Acetonitrile</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A995-212</td>
</tr>
<tr>
<td>Formic acid, 99.0+%, Optima LC/MS grade</td>
<td>Fisher Scientific</td>
<td>Cat# A117-50</td>
</tr>
<tr>
<td>n-Dodecyl-β-D-maltoside (DM)</td>
<td>Dojindo</td>
<td>Cat# D316</td>
</tr>
<tr>
<td>Citric Acid Monohydrate</td>
<td>Nacalai Tesque</td>
<td>Cat# 09106-02</td>
</tr>
<tr>
<td>CBB G-250</td>
<td>Nacalai Tesque</td>
<td>Cat# 094-09</td>
</tr>
<tr>
<td>Polyvinylpyrrolidone (PVP-40)</td>
<td>Sigma-Aldrich</td>
<td>Cat# PVP-40-500G</td>
</tr>
<tr>
<td>Trypsin (N-TosylL-phenylalanyl chloromethyl ketone treated</td>
<td>Worthington Biochemical Corporation</td>
<td>Cat# TRTPCK</td>
</tr>
<tr>
<td>Acrylamide</td>
<td>FUJIFILM Wako Chemicals</td>
<td>Cat# 011-08015</td>
</tr>
<tr>
<td>(+/-)-Dithiothreitol (DTT)</td>
<td>FUJIFILM Wako Chemicals</td>
<td>Cat# 048-29224</td>
</tr>
<tr>
<td>Methanol</td>
<td>Junsei Chemical, Co.</td>
<td>Cat# 73125-0330</td>
</tr>
<tr>
<td>Guanidine Hydrochloride</td>
<td>Kanto Chemical</td>
<td>Cat# 17051-08</td>
</tr>
<tr>
<td>EDTA・2NA</td>
<td>Dojindo</td>
<td>Cat# 343-01861</td>
</tr>
<tr>
<td>Trizma (R) base</td>
<td>Sigma-Aldrich</td>
<td>Cat# T1503</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Raw and analyzed data</td>
<td>ProteomeXchange Consortium</td>
<td><a href="https://www.ebi.ac.uk/pride/archive/projects/PXD020370">https://www.ebi.ac.uk/pride/archive/projects/PXD020370</a></td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Cont-BirA HEK293T</td>
<td><a href="#bib1">Kobayashi et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>BirA-SLC15A4 HEK293T</td>
<td><a href="#bib1">Kobayashi et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pcDNA3.1 mycBioID</td>
<td><a href="#bib3">Roux et al., 2012</a></td>
<td>Addgene plasmid # 35700</td>
</tr>
<tr>
<td>pEBMulti-Bsd</td>
<td>FUJIFILM Wako Chemicals</td>
<td>Cat# 055-08311</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Mascot Server 2.6</td>
<td>Matrix Science</td>
<td><a href="https://www.matrixscience.com/">https://www.matrixscience.com/</a></td>
</tr>
<tr>
<td>Proteome Discoverer 2.2</td>
<td>Thermo Fisher Scientific</td>
<td><a href="https://www.thermofisher.com/order/catalog/product/OPTON-31040">https://www.thermofisher.com/order/catalog/product/OPTON-31040</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Q Exactive HF-X Hybrid Quadrupole-Orbitrap Mass Spectrometer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 0726042</td>
</tr>
<tr>
<td>Easy nLC-1200</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# LC140</td>
</tr>
<tr>
<td>Nano spray column NTCC-360/75–3-105,</td>
<td>Nikkyo Technos, Co.</td>
<td>Cat#NTCC-360/75-3-105</td>
</tr>
<tr>
<td>SpeedVac vacuum concentrator</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Magnetic Stand (for 1.5 mL tube)</td>
<td>Tamagawa Seiki Co.</td>
<td>Cat# TA4899N12</td>
</tr>
<tr>
<td>Ultrasonic Homogenizer UH-50</td>
<td>SMT Corp.</td>
<td>UH-50</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Complete medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>D-MEM</td>
<td>–</td>
<td>500</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>PC/SM</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>555 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C.</p>
<table id="undtbl2">
<caption>1 mM (20×) biotin buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Biotin</td>
<td>1 mM</td>
<td>12.2 mg</td>
</tr>
<tr>
<td>D-MEM</td>
<td>–</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterilize by passing through a 0.22-μM syringe-driven filter unit. Aliquot and store at −20°C.</p>
<table id="undtbl3">
<caption>PBS/EDTA</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× PBS</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>0.5 M EDTA pH 8</td>
<td>1.8% (54 mM)</td>
<td>54 mL</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>–</td>
<td>Fill up to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterilize by autoclave and store at 4°C.</p>
<table id="undtbl4">
<caption>Cell-suspension buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl, pH 7.5</td>
<td>50 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>500 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>100× Protease inhibitor cocktail</td>
<td>1×</td>
<td>50 μL</td>
</tr>
<tr>
<td>0.5 M EDTA pH 8</td>
<td>5 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>–</td>
<td>Fill up to 5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use freshly prepared solution.</p>
<table id="undtbl5">
<caption>Lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl, pH7.5</td>
<td>50 mM</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>500 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>2% SDS (w/v)</td>
<td>0.4% (w/v)</td>
<td>1 mL</td>
</tr>
<tr>
<td>100× Protease inhibitor cocktail</td>
<td>1×</td>
<td>50 μL</td>
</tr>
<tr>
<td>0.5 M EDTA pH 8</td>
<td>5 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>–</td>
<td>Fill up to 5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use freshly prepared solution.</p>
<table id="undtbl6">
<caption>Mix buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cell-suspension buffer</td>
<td>25% (v/v)</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>Lysis buffer</td>
<td>25% (v/v)</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>50 mM Tris-HCl, pH 7.5</td>
<td>50% (v/v)</td>
<td>2.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use freshly prepared solution.</p>
<table id="undtbl7">
<caption>Wash buffer 1</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>SDS</td>
<td>2% (w/v)</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>–</td>
<td>Fill up to 25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store up 2 weeks at room temperature (20°C–25°C); also use this buffer as a SDS stock solution for lysis buffer.</p>
<table id="undtbl8">
<caption>Wash buffer 2</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10% Sodium deoxycholate (w/v)</td>
<td>0.1%</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>20% Triton X-100</td>
<td>1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>0.5 M EDTA pH 8</td>
<td>1 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>500 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M HEPES pH 7.5</td>
<td>50 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>–</td>
<td>Fill up to 10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store up to 2 weeks at room temperature (20°C–25°C).<br> Sodium deoxycholate stock solution must be protected from light.</p>
<table id="undtbl9">
<caption>Wash buffer 3</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10% Sodium deoxycholate (w/v)</td>
<td>0.5%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>20% NP-40</td>
<td>0.5%</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>0.5 M EDTA pH 8</td>
<td>1 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>2.5 M LiCl</td>
<td>500 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl, pH 7.5</td>
<td>10 mM</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>–</td>
<td>Fill up to 10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store up to 2 weeks at room temperature (20°C–25°C).<br> Sodium deoxycholate stock solution must be protected from light.</p>
<table id="undtbl10">
<caption>CGP staining solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CBB G-250</td>
<td>0.05%</td>
<td>0.025 g</td>
</tr>
<tr>
<td>PVP-40</td>
<td>2%</td>
<td>1 g</td>
</tr>
<tr>
<td>Citric acid</td>
<td>20%</td>
<td>10 g</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>–</td>
<td>Fill up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use freshly prepared solution.</p>
<table id="undtbl11">
<caption>De-staining buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>100% Acetonitrile</td>
<td>30%</td>
<td>3 mL</td>
</tr>
<tr>
<td>2 M Tris-HCl (pH 8.0)</td>
<td>10 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>–</td>
<td>6.95 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use freshly prepared buffer.</p>
<table id="undtbl12">
<caption>Reducing buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>8 M guanidine-HCl / 1 M Tris-HCl (pH8.5) / 10 mM EDTA Stock solution</td>
<td>4 M guanidine-HCl / 0.5 M Tris-HCl (pH8.5) / 5 mM EDTA</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>100 mM DTT</td>
<td>50 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use freshly prepared buffer.</p>
<table id="undtbl13">
<caption>Alkylating buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>8 M guanidine-HCl / 1 M Tris-HCl (pH8.5) / 10 mM EDTA Stock solution</td>
<td>4 M guanidine-HCl / 0.5 M Tris-HCl (pH8.5) / 5 mM EDTA</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>200 mM Acrylamide</td>
<td>100 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Use freshly prepared buffer.</p>
<table id="undtbl14">
<caption>Digestion buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2 M Tris-HCl (pH 8.0)</td>
<td>20 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>0.5% DM</td>
<td>0.025%</td>
<td>50 μL</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td><b>–</b></td>
<td>0.985 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>–</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Can be stored for at least a month at 4°C.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Collection of biotinylated proteins</h3>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1528-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Outline of collecting biotinylated proteins</div>
<p>The following step-by-step outline from BirA fusion protein expressing HEK293T cells to biotinylated precipitates.</p>
</figcaption>
</figure>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>Preparation of samples with sufficient amount of protein biotinylated by the BioID fusion protein is identified by mass spectrometry (<a href="#fig1">Figure 1</a>). At the starting point, the amount of total protein per sample is about 20 mg.</p>
<div class="note">
<span class="note-title">Note:</span> Prepare biological triplicate samples to see reproducibility in the next mass spectrometric analysis. All steps hereafter are carried out using DNase/RNase-free tubes that have not previously opened and a tube cap opener, and wearing gloves, face mask, safety goggle and head net, to avoid contamination of the samples, particularly with keratin. Should perform the experiment inside a clean bench.</div>
<h4 id="sec3.1.1">Perform cell lysis</h4>
<ol>
<li>Begin with one 15-cm dish (or three 10-cm dishes) for each sample (HEK293T cells expressing BioID fusion protein or BioID constructs).</li>
<li>When cells reach 80% confluency, change medium to fresh complete medium containing 50 μM biotin (1×).</li>
<li>Incubate cells for 16 h–18 h.</li>
<li>Collect the cells.
<ol type="a">
<li>Remove medium completely by aspiration.</li>
<li>Collect the cells by pipetting with 10 mL/dish of PBS/EDTA into 15 mL tube.</li>
<li>Wash the dish one more time with 5 mL PBS/EDTA to collect all residue cells.</li>
<li>Spin down for 5 min at 400 × g at room temperature (20°C–25°C).</li>
</ol>
</li>
<li>Discard the supernatant and wash the cells twice by spin down for 5 min at 400 × g at room temperature (20°C–25°C) with 10 mL/tube of PBS to remove all residue biotin.</li>
<li>Lyse the cells. This step is to completely disrupt all protein interactions and denature proteins.
<ol type="a">
<li>Resuspend the cells in 900 μL cell-suspension buffer (the cells need to be completely suspended for sufficient lysis).</li>
<li>Add 900 μL lysis buffer and pipette strongly the solution but avoid making the bubble. Perform this step at room temperature (20°C–25°C).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If it is hard to pipette the solution due to DNA genome releasing, apply a short sonication at 30% amplitude for 15–30 s at room temperature (20°C–25°C).</div>
<ol start="7">
<li>Add 180 μL of 20% Triton X-100 and mix by pipetting (to prevent SDS precipitation on ice). From now keep the tubes on ice for subsequent sonication.</li>
<li>Position the sonicator probe tip in the sample just above the tube bottom to avoid making the bubble.</li>
<li>Sonicate samples.
<ol type="a">
<li>Apply sonication for two sessions for 30 s using an ultrasonic homogenizer at 30% amplitude.</li>
<li>Then let the tube rest on ice for 2 min between each session to prevent overheating.</li>
<li>During the resting time, apply the sonication for other samples after washing the probe tip with 70% ethanol and ultra-pure water.</li>
</ol>
</li>
<li>Add 1.62 mL of prechilled 50 mM Tris-HCl, pH 7.5, and mix well.</li>
<li>Repeat one more session of sonication as mentioned in steps 8 and 9.</li>
<li>Aliquot the sample evenly into three prechilled 1.5-mL tubes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step can be excluded if the rotor for 5 ml tube is available.</div>
<ol start="13">
<li>Spin down for 10 min at 16,500 × g, 4°C. Carefully transfer all supernatant of the same sample to a prechilled 5 mL tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not disturb the small insoluble pellet on the tube wall when transferring the supernatant.</div>
<ol start="14">
<li>Measure protein concentration by using the DC protein assay kit (microplate assay) with BSA as a standard. Adjust the protein concentration of samples to the lowest concentration samples with the mix buffer. Keep samples on ice until use.</li>
</ol>
<h4 id="sec3.1.2">Perform affinity purification of biotinylated proteins</h4>
<p>(Steps 15–17 are to equilibrate the magnetic streptavidin beads with the mix buffer).</p>
<ol start="15">
<li>During steps 13 and 14, place three new 1.5-mL tubes in the Tamagawa magnetic stand (TA4899N12). Add 1 mL of the mix buffer to each tube.</li>
<li>Mix the stock of magnetic streptavidin beads well with gentle tapping to resuspend and add 150 μL of beads to each tube prepared in step 15. Wait for 3 min at room temperature (20°C–25°C).</li>
<li>Once the beads accumulate at one side of tube wall, remove the supernatant gently by 1-mL pipette.</li>
</ol>
<p>(Continue with the samples from step 14).</p>
<ol start="18">
<li>Divide 1.2 mL of sample from step 14 to the three tubes containing equilibrated beads.</li>
<li>Resuspend the samples and beads with gentle pipetting.</li>
<li>Incubate the tube on a rotator at 4°C for overnight (at least 12 h).</li>
<li>Place the tubes on the magnetic stand and wait for 3 min to collect beads at room temperature (20°C–25°C).</li>
<li>Remove the supernatant gently by pipetting. Try not to disturb beads on the wall of the tube.</li>
<li>Add 1.2 mL of wash buffer 1 to each tube and resuspend beads gently by pipetting.</li>
<li>Rinse the beads with rotation in a rotator for 8 min at room temperature (20°C–25°C).</li>
<li>Remove the supernatant as described in steps 21 and 22.</li>
</ol>
<p>(Pool beads from same samples into one tube).</p>
<ol start="26">
<li>Add 1.2 mL of wash buffer 1 to one of the three tubes and resuspend gently.</li>
<li>Transfer resuspended sample from step 26 to the second of the three tubes, and resuspend gently. Repeat this step to pool all beads from same samples into one tube in a final volume of 1.2 mL wash buffer 1.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> From this step, it is easy to lose beads, be careful with pipetting (<a href="#fig2">Figure 2</a>).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1528-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. The tip to pipetting for beads washing steps (steps 22–31)</div>
<p>The tip should be held at a 30°–40° angle and immersed about 1–2 mm. Move down the tip when aspirating the washing buffer.</p>
</figcaption>
</figure>
<ol start="28">
<li>Place the combined tube on a rotator for 8 min at room temperature (20°C–25°C).</li>
<li>Wash once with 1.2 mL of wash buffer 2 as described in steps 21 through 25.</li>
<li>Wash once with 1.2 mL of wash buffer 3 as described in steps 21 through 25.</li>
<li>Wash once with 1.2 mL of 50 mM Tris-HCl, pH 7.4 as described in steps 21 through 25. (<a href="#troubleshooting">troubleshooting 1</a>).</li>
<li>Remove the supernatant completely. Do not disrupt beads at the tube wall.</li>
<li>Add 110 μL Laemmli SDS sample buffer containing 2 mM biotin.</li>
<li>Save 10 μL (10% of total) of sample for further analysis by silver stain or western blot (<a href="#fig3">Figure 3</a>). 100 μL of sample will be used for mass spectrometric analysis.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1528-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Precipitation of BirA fusion protein Control-BirA and BirA-SLC15A4</div>
<p>Biotinylated precipitates were visualized by silver staining (left panel) or by Western blot (right panel) (at step 34).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If necessary, freeze the sample quickly in liquid nitrogen. Frozen samples can be stored at −80°C until performing mass spectrometric analysis.</div>
<h3 id="sec3.2">Preparation of samples for mass spectrometric analysis of BioID pull-down proteins</h3>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1528-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Outline of mass spectrometric sample preparation</div>
<p>The following step-by-step outline from biotinylated precipitates to mass-spec-ready samples.</p>
</figcaption>
</figure>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>Single-band focused SDS-PAGE containing whole proteins and its in-gel digestion with trypsin (<a href="#fig4">Figure 4</a>).</p>
<div class="note">
<span class="note-title">Note:</span> Prepare biological triplicate samples to see reproducibility in the next mass spectrometric analysis. To avoid modification by unreacted acrylamide, the one-day-old gel is used. All steps after are carried out not to contaminate the samples with keratin.</div>
<ol start="35">
<li>Elute the streptavidin bound proteins with 100 μL Laemmli SDS sample buffer containing 2 mM biotin at 98°C.</li>
<li>Concentrate the samples to the volume of 20 μL with a SpeedVac vacuum concentrator for 2 h at 25°C.</li>
<li>Load the concentrated samples onto a polyacrylamide mini gel.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If there are multiple samples, load each sample onto a gel at a distance to avoid cross-contamination.</div>
<ol start="38">
<li>Stop the electrophoresis when the dye front migrated from the stacking gel to the separation gel.</li>
<li>Wash the gel three times with 200 mL of ultra-pure water using a shaker and discard the washing solution.</li>
<li>Stain the gel according to the CGP stain method (<a href="#bib5">Yasumitsu et al., 2010</a>). Pour 50 mL of the CGP staining solution into the gel, and incubate for at least 1 h at 25°C.</li>
<li>Discard the staining solution and wash the gel three times with 200 mL ultra-pure water for 1 h to visualize the protein bands. In each lane, a single band is observed as a mixture of all the proteins.</li>
<li>Cut the visualized band into the four pieces using a clean razor blade and transfer them to a 600 μL microcentrifuge tube.</li>
<li>Soak the gel pieces in 300 μL of de-staining buffer to de-stain CBB G-250. Incubate for 30 min at 37°C and exchange the de-staining buffer until the gel pieces turn clear.</li>
<li>Dehydrate the gel pieces by adding 100% acetonitrile. Exchange 100% acetonitrile until the gel pieces turn white. Discard the acetonitrile and dry the gel pieces completely with a SpeedVac vacuum concentrator for 10 min at 25°C.</li>
<li>Add 50 μL of reducing buffer onto the dried gel pieces and incubate for 2 h at 37°C.</li>
<li>Add 50 μL of alkylating buffer onto the reduced gel pieces and incubate for 30 min at 25°C. Remove the buffer after alkylation by pipetting.</li>
<li>Wash gel pieces three times with 200 μL of ultra-pure water for 10 min at 25°C. Exchange washing solution with 200 μL of 100% acetonitrile and reswell with 200 μL of ultra-pure water. This step is repeated three times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is advisable to wash over three times to remove any reagents that may affect the subsequent mass spectrometric analysis.</div>
<ol start="48">
<li>Remove the washing solution and dehydrate the gel pieces with 100% acetonitrile, Dry the gel pieces with a centrifugal vacuum concentrator for 10 min at 25°C.</li>
<li>Reswell the dried gels completely with 2 μL of 10 ng/μL TPCK-treated trypsin. Add 30 μL of digestion buffer to cover the top of gel pieces.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The standard protocol is for about 1 μg protein, and the amount of enzyme and buffer are changed according to the amount of protein.</div>
<ol start="50">
<li>Incubate samples for 12 h at 37°C to digest with trypsin. The resulting peptides can be stored at 4°C for 1 week.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this time, the peptide mixture can be safely stored at −20°C for one month.</div>
<h3 id="sec3.3">LC-MS/MS analysis for label-free proteomics of BioID pull-down proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Peptide mixture generated by carrying out steps 35–50 is analyzed using LC-MS/MS. The obtained data are used for the next label-free quantitative analysis.</p>
<div class="note">
<span class="note-title">Note:</span> Prepare biological triplicate samples to see reproducibility and all samples are analyzed in the same volume for use in the next step of comparative quantification.</div>
<ol start="51">
<li>Transfer the digested peptides mixture to the vials in the LC autosampler and acidify the peptides solution by adding 1% formic acid. Adjust all sample volumes to equal. (<a href="#troubleshooting">troubleshooting 2</a>).</li>
<li>Analyze the digestion mixtures with a nanospray column (NTCC-360/75–3-105, Nikkyo Technos, Co., Ltd.) using EASY-nLC 1200 (Thermo Fisher Scientific) coupled with a Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific). LC-MS/MS data of three independent biological replicates are available through the ProteomeXchange: <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD020370">PXD020370</a>) (<a href="#troubleshooting">troubleshooting 3</a>).
<ol type="a">
<li>A flow rate of 300 nL/min was used and peptides were separated using a 50 min linear gradient of 0%–64% over B for 50 min (solvent A: 0.1% formic acid in water, solvent B: 0.1% formic acid in 80% acetonitrile).</li>
<li>Carry out LC-MS/MS analysis according to the following parameters in <a href="#tbl1">Table 1</a>. LC-MS/MS parameters are listed in <a href="#tbl1">Table 1</a>.
<table id="tbl1">
<caption>Table 1. LC-MS/MS parameters for a label-free proteomics</caption>
<thead>
<tr>
<th>Instrument</th>
<th>Q exactive HF-X</th>
</tr>
</thead>
<tbody>
<tr>
<td>Spray Voltage</td>
<td>2.00 kV</td>
</tr>
<tr>
<td>Capillary Temp</td>
<td>250°C</td>
</tr>
<tr>
<td colspan="2">Full MS</td>
</tr>
<tr>
<td>Resolution</td>
<td>60,000</td>
</tr>
<tr>
<td>AGC target</td>
<td>3e6</td>
</tr>
<tr>
<td>Maximum IT</td>
<td>60 ms</td>
</tr>
<tr>
<td>Number of scan ranges</td>
<td>1</td>
</tr>
<tr>
<td>Scan range</td>
<td>200–2,000 m/z</td>
</tr>
<tr>
<td>Spectrum data type</td>
<td>Profile</td>
</tr>
<tr>
<td colspan="2">dd-MS2 /dd-SIM</td>
</tr>
<tr>
<td>Microscans</td>
<td>1</td>
</tr>
<tr>
<td>Resolution</td>
<td>15,000</td>
</tr>
<tr>
<td>AGC target</td>
<td>1e5</td>
</tr>
<tr>
<td>Maximum IT</td>
<td>85 ms</td>
</tr>
<tr>
<td>Loop count</td>
<td>10</td>
</tr>
<tr>
<td>MSX count</td>
<td>1</td>
</tr>
<tr>
<td>TopN</td>
<td>10</td>
</tr>
<tr>
<td>Isolation window</td>
<td>1.6 m/z</td>
</tr>
<tr>
<td>NCE/stepped NCE</td>
<td>27</td>
</tr>
<tr>
<td>Spectrum data type</td>
<td>Centroid</td>
</tr>
<tr>
<td colspan="2">dd Settings</td>
</tr>
<tr>
<td>Minimum AGC target</td>
<td>8.30e3</td>
</tr>
<tr>
<td>Intensity threshold</td>
<td>9.8e4</td>
</tr>
<tr>
<td>Charge exclusion</td>
<td>unassigned, 5–8, &gt;8</td>
</tr>
<tr>
<td>Peptide match</td>
<td>Preferred</td>
</tr>
<tr>
<td>Exclude isotopes On Dynamic exclusion</td>
<td>20.0 s</td>
</tr>
</tbody>
</table>
<p class="table-legend">All other settings use the default settings.</p>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>To check our protocol works well, an intensity correlation between BioID experimental samples and control samples was plotted in <a href="#fig5">Figure 5</a>. Pyruvate carboxylase (PC), an endogenous biotin-binding protein, and SLC15A4, a BirA fusion protein, are shown on the plot. SLC15A4 was detected with the highest intensity only in the BioID experimental sample, while PC was strongly detected in both BioID experimental samples and control samples. The results showed that BirA-SLC15A4 was specifically and strongly biotinylated by BioID experiments. Also, proteins with a higher abundance ratio than the endogenous PC are likely to have been specifically biotinylated by BioID experiments. Therefore, the proteins in the region between the dashed lines were treated as candidates for biotinylated proteins in the BioID experiment. As a result, 629 proteins could be identified as biotinylated proteins by BioID experiments. Furthermore, as indicated in the section on quantitative and statistical analysis and troubleshooting, it is possible to verify the validity of the results by creating several constructs and conducting the same experiment. Among the identified proteins, we could confirm the interaction between SLC15A4 and the lysosomal proteins involved in mTOR pathway by Co-IP assay (<a href="#bib1">Kobayashi et al. 2021</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1528-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Intensity correlation plot between BioID experimental samples and control samples</div>
<p>The ion intensity of proteins identified by BioID experiments is plotted. The dashed line connects each of the endogenous PC and SLC15A4 intensities to the zero point.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>The LC-MS/MS data is processed by label-free proteomic analysis and the list of proteins that have been specifically increased by the BioID experiment is obtained. (<a href="#troubleshooting">troubleshooting 4, 5, 6</a>).</p>
<div class="note">
<span class="note-title">Note:</span> The use of data from biological triplicate samples increases the reliability of the results.</div>
<ol>
<li>Analyze LC-MS/MS data using Proteome Discoverer 2.2 (Thermo Scientific) combined with Mascot 2.6 (Matrix Science) for protein identification and label-free quantification.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The analysis software should be used Proteome Discoverer version 2.2 or higher. Also, Other MS analysis software like SEQUEST (<a href="#bib6">Yates et al. 1995</a>) and MaxQuant (<a href="#bib4">Tyanova et al., 2016</a>) can be used.</div>
<ol start="2">
<li>Read the resulting raw data of LC-MS/MS by Proteome Discoverer and set the names in “Study Factors” for each condition. e.g., ‘‘Control 1’’, ‘‘BioID sample 1’’.</li>
<li>Set up a “processing workflow” and a “consensus workflow”. Select the method for label-free quantification in the “processing workflow” of the Proteome Discoverer.
<ol type="a">
<li>Select “PWF_QE_Precursor_Quan_and_LFQ_SequestHT_Percolator” in the “ProcessingWF_QExactive” folder, which is the default workflow, and set Mascot node with the following parameters instead of Sequest. Database: NCBI-nr protein database (20160711, Taxonomy: H. sapiens 326,427 sequences), Parameters: enzyme = trypsin; maximum missed cleavages = 3; Fixed modifications = Propionamide (C); variable modifications = Acetyl (Protein N-term), Gln- &gt; pyro-Glu (N-term Q), Oxidation (M); product mass tolerance = ± 15 ppm; product mass tolerance = ± 30 milli mass unit; instrument type = ESI-TRAP.</li>
<li>Select “CWF_Comprehensive_Enhanced Annotation_LFQ_and_Precursor_Quan” in the “ConsensusWF” folder, which is the default consensus workflow. Use the default settings.</li>
</ol>
</li>
<li>In the “Grouping &amp; Quantification” section, set the protein abundance ratio. e.g., “BioID sample 1 / control 1”.</li>
<li>Run the program when the various settings are finished.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure hard disk space is enough.</div>
<ol start="6">
<li>Once the analysis is complete, you can also view the results in Proteome Discoverer. It is also possible to output the data in Excel or other formats. Statistical analysis like PCA plot can be performed on Proteome Discoverer.</li>
<li>In this setting, proteins with a high protein abundance ratio can be considered specifically enriched by the BioID experiment. We set the threshold for the protein abundance ratio (BioID-SLC15A4 / BioID-control) below the value of endogenous PC, resulting in 629 proteins as candidates. As shown in a Note, the results were validated by focusing on those that showed a common increase between several experiments.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is advisable to perform BioID experiments on other proteins like the target protein, or constructs with BirA fused to the N- and C-termini respectively. Proteins that are commonly increased in several experiments can be considered as interacting candidates, which helps to validate the results.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Limitations of BioID include the expression of an exogenous fusion protein. The addition of biotin ligase may impair normal targeting, stability, or function. Another limitation is that the identity of a candidate protein does not immediately imply an interaction with the bait. Biotinylated proteins resulting from the use of BioID fall under three categories: (i) direct interactions, (ii) indirect interactions, or (iii) proximal proteins that do not interact directly or indirectly. It is possible that false negatives may occur if candidates lack accessible primary amines for biotinylation. The permanent addition of biotin to primary amines may alter protein function by modifying charge and removing accessibility of those sites for alternative modifications. Negative consequences from expression of a BioID fusion protein in our case have not been observed, but must be considered possible.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Lose the bead in the washing steps (<a href="#step-by-step-method-details">step-by-step method details</a> steps 22–31).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Pipetting the wash buffer containing the beads to a new tube, re-collect by magnetic stand and pool together with its sample in the next step.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Difficult to de-stain CGP stain (<a href="#step-by-step-method-details">step-by-step method details</a> step 51).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Proceed to the alkylation step and then carry out the de-staining step.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Low detection sensitivity of peptides in LC-MS (<a href="#step-by-step-method-details">step-by-step method details</a> step 52).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>If the protein amounts are too low, start by re-preparing BioID samples to increase amounts of samples.</p>
<h3 id="sec7.7">Potential solution</h3>
<p>Confirmation of in-gel digestion steps by peptide mass fingerprinting using MALDI-TOF MS.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>If it is an LC-MS problem, use a standard sample (e.g., tryptic Hela cell lysate) to check the condition of the instrument. For example, we analyzed 0.5 μg of a proteome sample of HL60 cells using our instruments and settings. As a result, 940 proteins and 2,967 peptides were identified with high confidence.</p>
<h3 id="sec7.9">Problem 4</h3>
<p>Problems of quantification accuracy due to the accuracy of gel excision (<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>If further accuracy is required, digestion in a solution system should be considered.</p>
<h3 id="sec7.11">Problem 5</h3>
<p>Differences in the efficiency of in-gel digestion between samples (<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Examine the number of missed cleavages per sample and monitor the variability.</p>
<h3 id="sec7.13">Problem 6</h3>
<p>Many false-positive candidates (<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Perform BioID experiments with other proteins like the target protein, or proteins with BirA fused to the N- and C-termini respectively. The candidate proteins can be validated by comparing proteins that show similar behavior in these multiple experiments.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Kobayashi, T., Nguyen-Tien, D., Sorimachi, Y., Sugiura, Y., Suzuki, T., Karyu, H., Shimabukuro-Demoto, S., Uemura, T., Okamura, T., Taguchi, T., et al. (2021). SLC15A4 mediates M1-prone metabolic shifts in macrophages and guards immune cells from metabolic stress. PNAS <i>118</i>, e2100295118. <a class="external-link" href="https://doi.org/10.1073/pnas.2100295118">https://doi.org/10.1073/pnas.2100295118</a></p>
<p id="bib2">Roux, K.J., Kim, D.I., and Burke, B. (2013). BioID: a screen for protein-protein interactions. Curr. Protoc. Protein Sci. <i>74</i>, 19.23.1-19.23.14. <a class="external-link" href="https://doi.org/10.1002/0471140864.ps1923s74">https://doi.org/10.1002/0471140864.ps1923s74</a></p>
<p id="bib3">Roux, K.J., Kim, D.I., Raida, M., and Burke, B. (2012). A promiscuous biotin ligase fusion protein identifies proximal and interacting proteins in mammalian cells. J. Cell Biol <i>196</i>, 801-810. <a class="external-link" href="https://doi.org/10.1083/jcb.201112098">https://doi.org/10.1083/jcb.201112098</a></p>
<p id="bib4">Tyanova, S., Temu, T., and Cox, J. (2016). The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. Nat. Protoc. <i>11</i>, 2301-2319. <a class="external-link" href="https://doi.org/10.1038/nprot.2016.136">https://doi.org/10.1038/nprot.2016.136</a></p>
<p id="bib5">Yasumitsu, H., Ozeki, Y., Kawsar, S.M., Toda, T., and Kanaly, R. (2010). CGP stain: an inexpensive, odorless, rapid, sensitive, and in principle in vitro methylation-free Coomassie Brilliant Blue stain. Anal. Biochem. <i>406</i>, 86-88. <a class="external-link" href="https://doi.org/10.1016/j.ab.2010.06.035">https://doi.org/10.1016/j.ab.2010.06.035</a></p>
<p id="bib6">Yates, J.R., Eng, J.K., McCormack, A.L., and Schieltz, D. (1995). Method to correlate tandem mass spectra of modified peptides to amino acid sequences in the protein database. Anal. Chem. <i>67</i>, 1426-1436. <a class="external-link" href="https://doi.org/10.1021/ac00104a020">https://doi.org/10.1021/ac00104a020</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Noriko Toyama-Sorimachi (<a href="mailto:nsorimachi@ri.ncgm.go.jp">nsorimachi@ri.ncgm.go.jp</a>).</p>
<h4>Materials availability</h4>
<p>All reagents generated in this study are available from the lead contacts upon completing a Materials Transfer Agreement.</p>
<h4>Data and code availability</h4>
<p>LC-MS/MS raw files and Proteome Discoverer output files are available at ProteomeXchange: <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD020370">PXD020370</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank Mses. Hitomi Karyu and Shiho Shimabukuro-Demoto for technical supports. This work was supported by grants-in-aid for Scientific Research from the Ministry of Education, Science, Sports and Culture of Japan (for N.T.-S., 17H04070 and 20H05354; for T.K., 18K07056 and 20H04959; for D.N.-T., 18K16162 and 21K16155), a grant from <a href="https://doi.org/10.13039/100009619">AMED</a> (AMED-CREST for N.D.), a grant from <a href="https://doi.org/10.13039/501100013236">MSD Life Science Foundation</a> (for T.K.) and a grant from the <a href="https://doi.org/10.13039/100012319">National Center for Global Health and Medicine</a> (for N.T.-S., 23S001).</p>
<h3>Author contributions</h3>
<p>T.S., D.N.-T., and T.K. performed experiments and wrote the manuscript. N.D. conducted mass spectrometry experiments, and N.T.-S. designed and supervised all experiments.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>