<article>
<section>
<h1 id="header">Live imaging <i>Arabidopsis thaliana</i> embryos under different hydration conditions</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Yanniv Dorone,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup> Steven Boeynaems,<sup><a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#cor2">**</a></sup> and Seung Y. Rhee<sup><a href="#aff1">1</a>,<a href="#fn3">6</a>,<a href="#fn1">4</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Plant Biology, Carnegie Institution for Science, Stanford, CA 94305, USA</p>
<p id="aff2"><sup>2</sup>Department of Biology, Stanford University, Stanford, CA 94305, USA</p>
<p id="aff3"><sup>3</sup>Department of Genetics, Stanford University School of Medicine, Stanford, CA 94305, USA</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ydorone@alumni.stanford.edu">ydorone@alumni.stanford.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sboeynae@stanford.edu">sboeynae@stanford.edu</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:srhee@carnegiescience.edu">srhee@carnegiescience.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101025">10.1016/j.xpro.2021.101025</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Despite the ecological and agronomical importance of seed germination, how seeds integrate environmental signals to trigger germination remains enigmatic. Recently we reported that a protein called FLOE1 is involved in sensing and responding to water availability during germination. Here, we present a live-imaging protocol to assess the subcellular localization of a protein of interest during imbibition of desiccated <i>Arabidopsis thaliana</i> seeds with the goal of understanding protein dynamics during the early stages of water uptake.</p>
<p>For complete details on the use and execution of this profile, please refer to <a class="external-link" href="https://doi.org/10.1016/j.cell.2021.06.009">Dorone et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Steps for <i>Arabidopsis thaliana</i> embryo dissection</li>
<li>Protocol for confocal imaging of embryos</li>
<li>Analysis of protein localization under different levels of water potential</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The process of seed germination spans the initial uptake of water and ends with the protrusion of the radicle (<a href="#bib10">Steinbrecher and Leubner-Metzger, 2018</a>). Water uptake can be broken into three main phases based on the water content of the seed (<a href="#bib4">Gallardo et al., 2001</a>). Phase I is characterized by rapid water uptake, and is quickly followed by a plateau phase (Phase II) during which water content stays stable. In Phase III, water uptake increases again and enables radicle emergence through cellular growth. Germination <i>sensu stricto</i> refers to the first two phases, during which seeds maintain their desiccation tolerance (<a href="#bib4">Gallardo et al., 2001</a>). Up until Phase III, seeds may undergo numerous cycles of hydration-dehydration while maintaining their germination potential (<a href="#bib1">Bai et al., 2012</a>). However, the molecular details behind whether a seed will enter Phase III remain unresolved.</p>
<p>As illustrated by the example of the germination regulator FLOE1 which changes its biophysical state and subcellular localization in response to water (<a href="#bib3">Dorone et al., 2021</a>), a key element of this puzzle is understanding protein dynamics such as liquid-liquid phase separation and protein localization <i>in vivo</i> in response to water potential. Assessing protein dynamics in embryos <i>in vivo</i> is complicated by multiple confounding factors. First, embryos are protected by a seed coat that serves as a physical barrier that slows down water entry (<a href="#bib2">Debeaujon et al., 2007</a>). Therefore, the water potential (Ψ<sub>w</sub>) of the environment surrounding the seed may not reflect the water potential perceived by the embryo at a given point. In this protocol, we describe two complementary approaches to assess protein localization in response to varying levels of water potential: one that entails removing the seed coat and directly exposing naked embryos, and another that captures the state of the embryo prior to removal of the seed coat. Secondly, embryo cells contain highly autofluorescent protein storage vacuoles, which may compromise interpretations of subcellular localization of fluorescent proteins that are not expressed at high levels. This drawback is compounded by the fact that the level of autofluorescence is affected by water potential. We indeed found in <a href="#bib3">Dorone et al. (2021</a>) that protein storage vacuoles appear more autofluorescent when water potential is low. Therefore, the fluorescence needs to be normalized to compare fluorescent protein localization under different levels of water potential. By using the example of embryos expressing FLOE1p::FLOE1-GFP , we describe such a normalization method.</p>
<h3 id="sec1.1">Obtaining <i>Arabidopsis</i> seeds</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 months</div>
<p>Because environmental maternal effects can significantly impact seed characteristics such as weight, germination ability, and dormancy (<a href="#bib6">Jaganathan, 2016</a>; <a href="#bib11">Wulff, 2017</a>), it is important to grow plants in a consistent and controlled manner to ensure reproducibility of the results. This section is intended to specify best practices when growing mother plants to reduce environmental effects on the seed progeny. For more general protocols on growing <i>A. thaliana</i>, see (<a href="#bib9">Rivero et al., 2014</a>).</p>
<ol>
<li>To ensure synchronous germination, stratify seeds from the fluorescently tagged lines and a wild-type control (e.g., Col-0) as follows:
<ol type="a">
<li>Submerge seeds in double-distilled water in 1.5 mL Eppendorf tubes. Properly submerged seeds should sink to the bottom. If they float, invert the tubes multiple times until all seeds sink to the bottom.</li>
<li>Cover tubes with aluminum foil.</li>
<li>Place tubes at 4°C for 3–5 days.</li>
</ol>
</li>
<li>Sow seeds in individual pots of soil (e.g., PRO-MIX® HP Mycorrhizae) and cover with a plastic dome. We recommend growing at least six plants per genotype.</li>
<li>Transfer pots to a controlled environment, preferably a growth cabinet programmed to the desired growth settings (e.g., 16 h light/8 h dark at 22°C, 130 μmol m<sup>−2</sup> s<sup>−1</sup> Photosynthetic Photon Flux Density (PPFD), 50%–60% relative humidity). For optimal conditions, see the Arabidopsis Biological Resource Center (ABRC) recommendations at ftp://ftp.arabidopsis.org/ABRC/abrc_plant_growth.pdf.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Since seed characteristics can be dramatically altered by the local microenvironment during maturation on the mother plants (<a href="#bib5">Gutterman, 2000</a>), it is important to use a growth system (e.g., growth cabinet) to precisely control growth conditions. We recommend placing a data logger (e.g., HOBO temp/RH/light data logger) in the chamber, which can measure environmental parameters—including temperature, light and humidity—for the entire duration of the experiment.</div>
<ol start="4">
<li>To avoid any position effects within the growth chambers and to ensure population homogeneity (<a href="#bib8">Measures et al., 1973</a>), pots should be systematically randomized and rotated daily. The dome should be removed 5–7 days after germination.</li>
<li>When siliques begin to mature (indicated by their yellowing), relative humidity should be decreased to less than 50% as recommended by ABRC (ftp://ftp.arabidopsis.org/ABRC/abrc_plant_growth.pdf).</li>
</ol>
<ol start="6">
<li>To ensure that seeds from the same time windows of fertilization are compared, harvest brown siliques in 3 batches:
<ol type="a">
<li>Batch 1 should be harvested two weeks after the first brown siliques appeared.</li>
<li>Batch 2 should be harvested two weeks later.</li>
<li>Batch 3 should be harvested after the plants are completely senesced.
<div class="note">
<span class="note-title">Note:</span> We recommend harvesting seeds from each individual plant separately so that they can each serve as biological replicates.</div>
</li>
<li>After air-drying seeds in open 1.5 mL Eppendorf tubes for a week at 20% relative humidity (RH) in a controlled cabinet, seeds should be transferred to cold storage (4°C) with RH ≤ 20%. When properly stored, seeds should last for years (<a href="#bib9">Rivero et al., 2014</a>).
<div class="critical">
<span class="critical-title">Critical:</span> <i>Arabidopsis</i> seeds may quickly rehydrate if exposed to high humidity (<a href="#bib9">Rivero et al., 2014</a>), which will affect the results of this protocol. When taking seeds out of cold storage, tubes should be equilibrated at room temperature (20°C–23°C) before opening to prevent condensation from occurring around the seeds. When putting seeds back in storage, tubes should be left open in a 20% RH environment before closing to avoid trapping any moisture in. If a walk-in seed storage room (4°C and 20% RH) is available, it is preferable to work with aliquots taken directly inside the room to avoid subjecting the seed stocks to environmental fluctuations.</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Glycerin</td>
<td>MilliporeSigma</td>
<td>Cat. # G2289</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>MilliporeSigma</td>
<td>Cat. # S3014</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>Arabidopsis thaliana</i>: Col-0</td>
<td>Arabidopsis Biological Resource Center (ABRC)</td>
<td>CS70000</td>
</tr>
<tr>
<td>
<i>Arabidopsis thaliana</i>: FLOE1p::FLOE1-GFP;<i>floe1-1/floe1-1</i> (Col-0)</td>
<td>(<a href="#bib3">Dorone et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>Arabidopsis thaliana</i>: 35S::FLOE1-GFP;<i>floe1-1/floe1-1</i> (Col-0)</td>
<td>(<a href="#bib3">Dorone et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Leica Application Suite X (LAS X)</td>
<td>Leica Microsystems</td>
<td>RRID: SCR_013673 (<a href="https://www.leica-microsystems.com/products/microscope-software/p/leica-las-x-ls/">https://www.leica-microsystems.com/products/microscope-software/p/leica-las-x-ls/</a>)</td>
</tr>
<tr>
<td>GraphPad Prism (v8.4.1)</td>
<td>GraphPad Software</td>
<td>RRID: SCR_002798 (<a href="https://www.graphpad.com/">https://www.graphpad.com/</a>)</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Nickel Plated Pin Holder</td>
<td>Fine Science Tools</td>
<td>Cat. # 26018-17</td>
</tr>
<tr>
<td>Tungsten Needles (rod diameter: 0.125 mm)</td>
<td>Fine Science Tools</td>
<td>Cat. # 10130-05</td>
</tr>
<tr>
<td>Leica TCS SP8 Confocal Microscope with HC PL APO CS2 63x/1.20 WATER and 63/1.30 GLYCERIN objectives.</td>
<td>Leica Microsystems</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Direct imbibition of embryos</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>In this step, naked embryos are directly exposed to solutions of varying levels of water potential.</p>
<ol>
<li>Prepare solutions of varying levels of water potential.
<ol type="a">
<li>Prepare a stock solution containing a high concentration of an osmolyte of your choice that will be used to modify the water potential. For example, in <a href="#bib3">Dorone et al., (2021)</a>, we used NaCl, mannitol and sorbitol. In the case of NaCl, a solution of up to 6 M can be prepared by mixing 17.532 g NaCl (Sigma-Aldrich) in double-distilled water and adjusting the volume to 50 mL.
<div class="note">
<span class="note-title">Note:</span> It is best to first prepare a high concentration of the osmolyte (without surpassing its maximum solubility) as well as a high volume to minimize variabilities between experiments. In the example above, we chose 6 M and 50 mL NaCl which will be used as a stock solution in the next step to generate 2 mL solutions of 0.2–1.8 M.</div>
</li>
<li>Prepare dilutions of the stock solutions. In the case of NaCl, prepare solutions in increments of 200 mM from 0 M to 2 M. To mitigate pipetting inaccuracies, we recommend preparing first a 10 mL secondary stock of 2 M, and then preparing solutions of 2 mL using the following table:
<table id="undtbl1">
<caption>Dilutions of NaCl stock solution</caption>
<thead>
<tr>
<th>Concentration</th>
<th>0.2 M</th>
<th>0.4 M</th>
<th>0.6 M</th>
<th>0.8 M</th>
<th>1 M</th>
<th>1.2 M</th>
<th>1.4 M</th>
<th>1.6 M</th>
<th>1.8 M</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>2 M stock</b></td>
<td>0.2 mL</td>
<td>0.4 mL</td>
<td>0.6 mL</td>
<td>0.8 mL</td>
<td>1 mL</td>
<td>1.2 mL</td>
<td>1.4 mL</td>
<td>1.6 mL</td>
<td>1.8 mL</td>
</tr>
<tr>
<td><b>Water</b></td>
<td>1.8 mL</td>
<td>1.6 mL</td>
<td>1.4 mL</td>
<td>1.2 mL</td>
<td>1 mL</td>
<td>0.8 mL</td>
<td>0.6 mL</td>
<td>0.4 mL</td>
<td>0.2 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> We found in (<a href="#bib3">Dorone et al., 2021</a>) that exposure of embryos to 2 M NaCl induces a similar subcellular localization of FLOE1 as in the dry state, and therefore chose 2 M NaCl as the upper bound for the serial dilutions. However, depending on the protein of interest, the upper bound may be different and this parameter will need to be determined empirically.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Seed treatment.
<ol type="a">
<li>Seed imbibition:
<ol type="i">
<li>Submerge 2–3 seeds in 0.5–1 mL in one of the osmolyte solutions prepared in step 1. Ensure that the seeds sink to the bottom of the 1.5 mL Eppendorf tube by inverting it a few times.</li>
<li>Incubate for 15–30 min at room temperature (20°C–23°C).</li>
</ol>
</li>
<li>To examine the dry state of the embryo, submerge 2–3 seeds in 0.5–1 mL glycerin. Glycerin is a non-aqueous solution that, upon seed coat removal, will maintain the hydration state of the embryo as it was in the intact seed (<a href="#bib3">Dorone et al., 2021</a>). It also serves as an immersion medium for the embryo during imaging.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Seed dissection.
<ol type="a">
<li>Pipette one seed with approximately 5–10 μL of the solution in which it was submerged. Place on a cover slip or slide.</li>
<li>Under a dissecting microscope (3× magnification), hold the seed between tweezer tips (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Dissection of an <i>Arabidopsis thaliana</i> seed</div>
<p>(A) An <i>Arabidopsis</i> seed is held between tweezer tips while its seed coat and endosperm are being scraped off with a tungsten needle.</p>
<p>(B–F) The same seed is imaged at different stages of dissection. A cavity is first created by superficially scraping the seed coat and endosperm off. Then, the embryo is gently pushed through it.</p>
<p>Scale bar for (B–F) is shown in F.</p>
</figcaption>
</figure>
</li>
<li>Using a tungsten needle held with a pin holder (see <a href="#key-resources-table">key resources table</a>), superficially scrape the seed coat and endosperm without damaging the embryo to create a large enough cavity for the embryo to pass through (<a href="#fig1">Figures 1</a>B and 1C).</li>
<li>Gently push the embryo out of the cavity to remove the seed coat and endosperm (<a href="#fig1">Figures 1</a>D–1F).
<div class="note">
<span class="note-title">Note:</span> This step can be particularly challenging when dissecting seeds in non-aqueous solutions or under very low levels of water potential. In these cases, we recommend gently scraping the surface across the entire embryo until all the seed coat and endosperm pieces come off. Damaged embryos (see <a href="#fig8">Figures 8</a> and <a href="#fig9">9</a>) should be removed from further analysis.</div>
</li>
<li>Pipette the seed coat and endosperm debris out.</li>
<li>Add 20–50 μL of the osmolyte solution or glycerin depending on the condition.</li>
<li>Place a cover slip on top and proceed immediately to <i>Imaging.</i>
<div class="note">
<span class="note-title">Note:</span> Rinse the tweezers and needle with water and wipe between each dissection to avoid the formation of osmolyte crystals upon evaporation.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Physiologically relevant imbibition of the embryos</h3>
<div class="timing">
<span class="timing-title">Timing:</span> variable (&gt; 1 day)</div>
<p>In this step, intact seeds are imbibed at physiologically relevant osmolyte concentrations. After this step, seeds are dissected in glycerin to preserve the hydration state of the embryos as it was prior to seed coat removal. As a result, the subcellular localization of the protein of interest can be assessed at the hydration state that is naturally achieved within intact seeds.</p>
<ol start="4">
<li>Fresh media preparation.
<ol type="a">
<li>Prepare standard growth medium:
<ol type="i">
<li>Prepare Murashige and Skoog (“MS”) medium using the following recipe:
<table id="undtbl2">
<caption>Murashige and Skoog medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount for 1L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Murashige &amp; Skoog Basal Medium (PhytoTechnologies Laboratory)</td>
<td>0.5×</td>
<td>2.15 g</td>
</tr>
<tr>
<td>MES (Research Products International)</td>
<td>0.05%</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Sucrose (Sigma-Aldrich)</td>
<td>1%</td>
<td>10 g</td>
</tr>
<tr>
<td>Agar (Difco)</td>
<td>0.8%</td>
<td>8 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Before adding the agar, adjust the pH to 5.7 using potassium hydroxide.</p>
</li>
<li>Autoclave using a liquid cycle (20–30 min, 121°C).
<div class="note">
<span class="note-title">Note:</span> Other growth media can be chosen depending on the study. For example, medium containing only water and agar, or MS medium without sucrose can be used.</div>
</li>
</ol>
</li>
<li>Prepare the osmolyte:
<ol type="i">
<li>Weigh the osmolyte of choice needed to generate a 5–10× stock medium. For NaCl, weigh 29.22 g (which will be used to make a 1M stock solution in step 4c).</li>
<li>Autoclave a bottle containing the weighed osmolyte using a dry cycle (20–30 min, 121°C).</li>
</ol>
</li>
<li>Prepare a stock medium by filling the autoclaved bottle from step 4b. with the media prepared in step 4a. to the required volume. In the case of NaCl, add MS medium to the 29.22 g from step 4b for a final volume of 500 mL to obtain a 1M NaCl solution.
<div class="note">
<span class="note-title">Note:</span> While the osmolyte can be introduced directly in step 4a., we recommend separating these two steps as evaporation during autoclaving will concentrate the osmolyte and affect reproducibility.</div>
</li>
<li>Use the stock medium to prepare diluted media at the desired concentrations. In the case of NaCl, the following range of concentrations can be prepared:
<table id="undtbl3">
<caption>Dilutions of MS medium supplemented with NaCl</caption>
<thead>
<tr>
<th>Concentration</th>
<th>80 mM</th>
<th>120 mM</th>
<th>160 mM</th>
<th>200 mM</th>
<th>240 mM</th>
<th>280 mM</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>1 M NaCl MS medium</b></td>
<td>12 mL</td>
<td>18 mL</td>
<td>24 mL</td>
<td>30 mL</td>
<td>36 mL</td>
<td>42 mL</td>
</tr>
<tr>
<td><b>MS medium</b></td>
<td>138 mL</td>
<td>132 mL</td>
<td>126 mL</td>
<td>120 mL</td>
<td>124 mL</td>
<td>108 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> We recommend using this approach of diluting a stock medium instead of preparing each medium individually from scratch. In our experience, even small variabilities during preparation affect the results, which makes the experiment less reproducible.</div>
<div class="note">
<span class="note-title">Note:</span> Here, 280 mM NaCl was chosen as the maximum concentration since Col-0 (WT) seeds will stop germinating at concentrations above 240 mM (<a href="#bib3">Dorone et al., 2021</a>). However, for other ecotypes or genotypes, this upper bound may differ.</div>
</li>
<li>Plate the media in a sterile, laminar flow hood:
<ol type="i">
<li>To avoid evaporation after plating that may alter the final osmolyte concentration, cool the bottles to 50°C in a water bath set at 50°C.</li>
<li>Gently shake the bottle to mix its content, and pipette 40 mL into square petri dishes (120 × 120 wide × 15 mm high (VWR)).</li>
<li>Close the plates as soon as they dry (∼10 min) to avoid evaporation.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Seed sterilization. Work in a sterile, laminar flow hood
<ol type="a">
<li>Place 20–100 seeds in 1.5 mL Eppendorf tubes and add 1–1.5 mL of freshly prepared 70% ethanol (70:30 v/v ethanol:double-distilled water).</li>
<li>Vortex for 5 min.</li>
<li>Discard the solution and add 1 mL of 100% ethanol.</li>
<li>Pipette the seeds onto pre-sterilized filter papers (e.g., Grade 410, VWR)</li>
<li>Let the seeds dry on the filter papers for at least 20 min.</li>
</ol>
</li>
<li>Seed imbibition.
<ol type="a">
<li>Place the seeds onto the agar plates prepared in step 4e by gently tapping them off the filter papers. Ensure that seeds are spaced out. Seeds that touch one another can be separated with a sterile toothpick.</li>
<li>Seal the plates with micropore surgical tape (3M).</li>
<li>Place the plates at the desired conditions. For example, seeds can be first stratified by placing the plates in the dark at 4°C for 3–5 days, and then placed in a growth cabinet at 22°C–23°C with 16 h light/8 h dark with 130 μmol m<sup>−2</sup> s<sup>−1</sup> PPFD and 50%–60% relative humidity.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The purpose of this protocol is to assess protein localization before Phase III of water uptake (<i>i.e.,</i> during germination <i>sensu stricto</i>). It is important to ensure that seeds have not yet entered Phase III when imaging. For instance, under low osmolyte concentrations (e.g., &lt; 80 mM NaCl), Col-0 seeds will typically start germinating within 24 hours when placed in otherwise optimal conditions (e.g., 22°C with illumination at 130 μmol m<sup>−2</sup> s<sup>−1</sup> PPFD). At that point, cells will have begun dramatic structural changes that will make comparisons with seeds placed under higher osmolyte concentrations less relevant. See Troubleshooting, <a href="#sec6.5">problem 3</a> for an example of such changes.</div>
<ol start="7">
<li>To capture the hydration state of the seeds at a given time point, pick 2–3 seeds from the plates using a sterile toothpick or equivalent, and submerge in 0.5–1 mL glycerin. If the experiment involves a time series, work in sterile conditions and place the plate back in its experimental conditions until the next time point.</li>
<li>Dissect seeds as described in step 3 and proceed immediately to <a href="#sec2.3">Imaging</a>.</li>
</ol>
<h3 id="sec2.3">Imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<p>In this step, we describe live embryo imaging using a Leica TCS SP8 laser scanning confocal microscope. While many of the steps described are specific to the Leica system, they can be easily adapted to use other confocal microscopes and software.</p>
<ol start="9">
<li>Software set up.
<ol type="a">
<li>Open the Leica Application Suite X software.</li>
<li>Select resonant mode when prompted.</li>
<li>“Configuration” settings:
<ol type="i">
<li>In the “Laser Config” tab, turn on the white light laser (WLL) and set at 70%.</li>
<li>In the “Hardware” tab, enable “Line Average during Live Acquisition” and set the “Bit Depth Resolution” at 12 bit.</li>
<li>In the “USB Panel” tab, select the “Fine” option for the “Phase” setting.</li>
</ol>
</li>
<li>“Acquire” settings:
<ol type="i">
<li>In the “Acquisition” tab, enable “Bidirectional X”, set the “Line Average” at 32, and set the “Zoom Factor” at 1.25 (<a href="#fig2">Figures 2</a>A, 2C, and 2D).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Initial imaging configurations on the Leica Application Suite X software</div>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> the line average can be set at lower or higher than 32 depending on the laser intensity, the fluorescence level of the protein of interest, and the desired resolution.</div>
</li>
<li>Set the excitation wavelength at 488 nm if using GFP (<a href="#fig2">Figure 2</a>F).</li>
<li>Select the HC PL APO CS2 63×/1.20 WATER objective if using samples mounted in aqueous solutions and 63/1.30 GLYCERIN if mounted in glycerin (<a href="#fig2">Figure 2</a>G).</li>
<li>Turn on one of the HyD SMD (Hybrid Detector Single Molecule Detection) detectors and set a detection interval of 50 nm starting at a wavelength at least 10 nm higher than the emission wavelength. In the case of GFP, set it at 500–550 nm (<a href="#fig2">Figure 2</a>H).</li>
<li>Set the lifetime gate filter at 1.00–6.00 ns (<a href="#fig2">Figure 2</a>H).
<div class="note">
<span class="note-title">Note:</span> Since most fluorescent proteins have fluorescence lifetimes within the 2.3–3.5 ns range (<a href="#bib7">Mamontova et al., 2018</a>), a lifetime gate filter of 1.50–6.00 ns can also be used.</div>
</li>
<li>Under TLD (Transmitted Light Detection) Turn on PMT Trans (transmission photomultiplier tube) (<a href="#fig2">Figure 2</a>I).</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Image acquisition
<ol type="a">
<li>Load sample using the appropriate immersion media (water when using the HC PL APO CS2 63×/1.20 WATER objective and glycerin when using the 63/1.30 GLYCERIN objective).
<div class="note">
<span class="note-title">Note:</span> We recommend starting with the wild-type ecotype (e.g., Col-0) and getting familiar with the protein storage vacuoles’ autofluorescence to avoid confusing these signals with those of the fluorescently tagged protein of interest. See <a href="#fig3">Figure 3</a>.</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Protein storage vacuoles are autofluorescent</div>
<p>A section of an embryonic radicle of Col-0 (WT) was imaged with different settings. Protein storage vacuoles are autofluorescent at imaging settings typically used for GFP imaging (top panels), RFP imaging (bottom left panel) as well as at the lower end of the visible spectrum (bottom right panel). Fluorescence lifetime gating of 1–6 ns (top middle panel) and of 2–6 ns (top right panel) can reduce autofluorescence. Single optical section images are shown.</p>
</figcaption>
</figure>
</li>
<li>Find and focus on the sample. For example, in <a href="#fig2">Figure 2</a>K we centered on an embryonic radicle.</li>
<li>Adjust the gain on the transmitted light channel (<a href="#fig2">Figure 2</a>I).</li>
<li>Starting with a low laser intensity (e.g., 1% of the laser intensity), adjust intensity until the HyD SMD channel detects the fluorescence signal without reaching saturation which could damage the detector. One way of assessing saturation is to view the acquisition by its intensity values by clicking on the “Over-/underexposure” button (<a href="#fig2">Figure 2</a>J). The laser intensity can then be adjusted until only a minimal number of pixels appear blue.
<div class="critical">
<span class="critical-title">Critical:</span> When gating is turned on, the detector is exposed to a higher intensity than it appears on the software. To avoid damaging the detector, gating can be turned off while setting up the laser intensity.</div>
</li>
<li>Adjust the phase so that the image appears clear (<a href="#fig2">Figure 2</a>B).</li>
<li>Set the Z-Stack (<a href="#fig2">Figure 2</a>E).</li>
<li>If needed, adjust the Line Average number (<a href="#fig2">Figure 2</a>D) so that the signal appears less grainy.</li>
<li>Start image acquisition.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol aims to study the subcellular localization of a fluorescently tagged protein of interest under varying levels of water potential in <i>Arabidopsis thaliana</i> embryos.</p>
<p>In <a href="#fig5">Figure 5</a>, we give the example of how FLOE1’s localization is affected by different NaCl concentrations using the two methods described in this protocol. In <a href="#fig5">Figure 5</a>A, we show the results of the first approach that directly exposes naked embryos to varying levels of water potential (see the <a href="#sec2.1">direct imbibition of embryos</a> section<i>)</i>. FLOE1 forms granules across embryo radicles when exposed to water (<a href="#fig5">Figure 5</a>A, left panel). However, when salt is added at a concentration of 1 M, granules are less prominent and fewer cells exhibit them (<a href="#fig5">Figure 5</a>A, middle panel). At 2 M, granules can no longer be observed (<a href="#fig5">Figure 5</a>A, right panel). In <a href="#fig5">Figure 5</a>C, we employed the second approach (see the <a href="#sec2.2">physiologically relevant imbibition of embryos</a> section<i>)</i>, in which intact seeds (i.e., having their seed coats) were first imbibed on media supplemented with different osmolyte concentrations, after which they were dissected in glycerin to prevent further hydration and “capture” the hydration state of the embryos prior to seed coat and endosperm removal. Since the seed coat—which acts as a physical barrier—is present, FLOE1 granules are barely visible at concentrations of salt higher than 280 mM even after a total of six days (five days of stratification, one day of incubation), compared to 80 mM where they are more prominent. These differences can be quantified using the fluorescence heterogeneity score of each cell across three independent embryos for each condition. As shown in <a href="#fig5">Figures 5</a>B and 5D, the less cells exhibit FLOE1 granules the lower their heterogeneity scores.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec4.1">FLOE1 case example</h3>
<p>In this step, we use embryos expressing FLOE1p::FLOE1-GFP (<a href="#bib3">Dorone et al., 2021</a>) as an example of how to analyze cell-to-cell variations in protein localization. FLOE1 forms cytoplasmic granules whose intensity and number vary in response to water potential (<a href="#bib3">Dorone et al., 2021</a>). We anticipate that similar normalization methods could be used to study the partitioning of other proteins of interest into different compartments (e.g., nucleocytoplasmic partitioning).</p>
<ol>
<li>Quantification
<ol type="a">
<li>Open the Quantification tab in the Leica Application Suite X software and open the saved Z-stack.</li>
<li>Select the Channel used to acquire the fluorescence signal.</li>
<li>Generate a maximum projection image by clicking on the MAX icon (<a href="#fig4">Figure 4</a>A).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Image analysis using the Leica application suite X software</div>
</figcaption>
</figure>
</li>
<li>Draw ROIs (Regions Of Interest) around each cell (<a href="#fig4">Figure 4</a>B). This will automatically generate data in the Statistics tab (<a href="#fig4">Figure 4</a>C).</li>
<li>Export and save the Statistics table (<a href="#fig4">Figure 4</a>C).</li>
</ol>
</li>
<li>Analysis
<ol type="a">
<li>Open the saved table in Excel.</li>
<li>Divide the Standard Deviation (RMS) value by the Mean Value to obtain heterogeneity scores for each embryonic cell. These scores measure the normalized variability of fluorescence within each cell and, depending on the case, can serve as a proxy for assessing partitioning between different organelles. For example, in <a href="#bib3">Dorone et al. (2021)</a>, we used this score as a proxy for the number of granules and their intensity (<a href="#bib3">Dorone et al., 2021</a>).</li>
<li>Datasets for at least three embryos can then be used for comparison across different conditions (see Expected Outcomes). For example, these values can be visualized and analyzed on GraphPad Prism as shown in the examples provided in <a href="#fig5">Figures 5</a>B and 5D.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Imaging and quantification examples in embryos expressing FLOE1p::FLOE1-GFP</div>
<p>(A) Col-0 embryos (top panels) and embryos expressing FLOE1::FLOE1-GFP (bottom panels) were imbibed and dissected as described in the <a href="#sec2.1">direct imbibition of embryos</a> section in water supplemented with NaCl at the following concentrations: 0 M (left panel), 1 M (middle panel) and 2 M (right panel). FLOE1 forms granules in a hydration-dependent manner. The vast majority of cells exhibits granules at 0 M NaCl, less so at 1 M NaCl, and no visible granules at 2 M NaCl. Images are maximum projections of embryo radicles.</p>
<p>(B) Quantification results for (A) (see <a href="#quantification-and-statistical-analysis">Quantification and statistical analysis</a><i>: FLOE1 case example</i>) for three embryos at each condition are plotted in GraphPad Prism (v. 8.4.1). Each dot represents the fluorescence heterogeneity score of an individual radicle cell. One-way ANOVA Kruskal Wallis. ∗∗∗∗ p-value &lt; 0.0001.</p>
<p>(C) Embryos expressing FLOE1::FLOE1-GFP were stratified for 5d (4°C, dark) and incubated for 24h (22°C, 130 μmol.m<sup>−2</sup> s<sup>−1</sup> PPFD) on MS medium supplemented with NaCl at 80 mM (left panel) or 280 mM (right panel). They were then dissected in glycerin as described in the <a href="#sec2.2">physiologically relevant imbibition of embryos</a> section. Unlike in (A), imbibition occurs through the seed coat, which alters the environmental water potential required for FLOE1 granules to appear. Images are maximum projections of embryo radicles.</p>
<p>(D) Quantification results for (C) (see <a href="#quantification-and-statistical-analysis">Quantification and statistical analysis</a><i>: FLOE1 case example</i>) for three embryos at each condition are plotted in GraphPad Prism (v. 8.4.1). Each dot represents the fluorescence heterogeneity score of an individual radicle cell. Mann-Whitney. ∗∗∗∗ p-value &lt; 0.0001.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Autofluorescence of the protein storage vacuoles is a major obstacle when a protein is expressed below a low threshold. We recommend always comparing the images to the background ecotype (e.g., Col-0), and getting familiar with what constitutes autofluorescence versus real signals emitted by the fluorescent tag.</p>
<p>Additionally, as this procedure is invasive, the seeds cannot be reused for subsequent germination experiments. Therefore, it complicates experiments that aim to understand whether specific subcellular localization patterns are directly linked to the germination phenotype of a given seed.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>No fluorescence signal can be detected other than the autofluorescent protein storage vacuoles.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>First, ensure that the protein is expressed in the seeds using a Western Blot or ELISA test (see (<a href="#bib3">Dorone et al., 2021</a>) for protein extraction and GFP ELISA using seeds). In our experience, certain promoters such as the 35S Cauliflower Mosaic Virus (CaMV) promoter may have sporadic expression levels in the embryo (see <a href="#sec6.3">problem 2</a>), with some transgenic lines not expressing the transgene at all (step 1).</p>
<p>Second, if the protein is indeed expressed, the signal may be drowned out by autofluorescence. In this case, we recommend trying different promoters that are highly expressed in the seed, but avoiding certain overexpression promoters like 35S CaMV (see the comment above as well as <a href="#sec6.3">problem 2</a>).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Protein expression is patchy in the embryos.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>When using the 35S CaMV promoter, expression of certain proteins may be patchy in the embryos (<a href="#fig6">Figure 6</a>). We recommend using the native promoter or trying different promoters (e.g., known seed-specific promoters).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. A line carrying a 35S::FLOE1-GFP transgene exhibits a patchy expression in <i>Arabidopsis</i> embryos</div>
<p>An embryo carrying a 35S::FLOE1-GFP construct was imbibed and dissected in water as described in the <a href="#sec2.1">direct imbibition of embryos</a> section. Maximum projection image of a cotyledon (embryonic leaf). The GFP channel was merged with that of the transmitted light detector.</p>
</figcaption>
</figure>
<h3 id="sec6.5">Problem 3</h3>
<p>The protein storage vacuoles seem abnormal or not well-defined.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>This is likely due to the embryos having entered Phase III of germination (step 6c). In the example shown in <a href="#fig7">Figure 7</a>A, seeds expressing FLOE1p::FLOE1-GFP were first sowed on 40 mM NaCl media as described in the <a href="#sec2.2">physiologically-relevant imbibition of the embryos</a> section. They were first stratified for 5 days at 4°C in the dark and then placed in light conditions for 24 h at 22°C with 130 μmol.m<sup>−2</sup>.s<sup>−1</sup> PPFD. While seeds exposed to concentrations higher than 80 mM have not yet germinated at that point and do not appear to have reached Phase III yet, those at 40 mM are further along the process, as evidenced by seed cracks in the seed population (<a href="#fig7">Figure 7</a>B).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Protein storage vacuoles have a different appearance in embryos that have initiated germination</div>
<p>(A) After 5 days of stratification (4°C, dark) on MS medium supplemented with 40 mM NaCl, seeds expressing FLOE1p::FLOE1-GFP were transferred to continuous light (130 μmol m<sup>−2</sup> s<sup>−1</sup> PPFD) for 24 h at 22°C. Embryos were then dissected out in glycerin as described in the <a href="#sec2.2">physiologically-relevant imbibition of the embryos</a> section. Image is a maximum projection of an embryo radicle.</p>
<p>(B) Example of a seed crack that indicates that the radicle is about to emerge.</p>
</figcaption>
</figure>
<h3 id="sec6.7">Problem 4</h3>
<p>Some cells appear dark.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This often occurs upon accidental scratching or poking of the embryo during dissection (see <a href="#fig8">Figure 8</a>) (step 3c). To avoid this, we recommend making the incision in the seed coat along a tissue that will not be imaged (e.g., incision along the cotyledons if the radicle is to be imaged). Additionally, tungsten needles of a higher diameter can be used. While dissection will require more patience, it will also reduce the risk of damage.</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Example of a damaged embryo</div>
<p>(A) A maximum projection image capturing the autofluorescence of a Col-0 (WT) embryonic radicle exhibits a “dark cell” (indicated by the arrowhead).</p>
<p>(B) A single optical section of the dark region shown in (A) reveals tissue damage.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> we recommend discarding damaged samples from further analysis.</div>
<h3 id="sec6.9">Problem 5</h3>
<p>Embryos split during imaging (<a href="#fig9">Figure 9</a>).</p>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1279-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Example of an embryo that splits</div>
<p>A Col-0 (WT) embryonic radicle imbibed in water splits longitudinally along the middle of the radicle. The same embryo (left) was imaged 8 min later (right) to show the progression of the damage. Single optical sections are shown.</p>
</figcaption>
</figure>
<h3 id="sec6.10">Potential solution</h3>
<p>In our experience, this happens when even minimal pressure is applied on the cover slip (step 3g). To avoid this, we recommend delicately placing the cover slip on top of the sample right before mounting it on the microscope instead of preparing the slides in advance and carrying them to the instrument.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bai, B., Sikron, N., Gendler, T., Kazachkova, Y., Barak, S., Grafi, G., Khozin-Goldberg, I., and Fait, A. (2012). Ecotypic variability in the metabolic response of seeds to diurnal hydration-dehydration cycles and its relationship to seed vigor. Plant Cell Physiol <i>53</i>, 38-52. <a class="external-link" href="https://doi.org/10.1093/pcp/pcr169">https://doi.org/10.1093/pcp/pcr169</a></p>
<p id="bib2">Debeaujon, I., Lepiniec, L., Pourcel, L., and Routaboul, J.M. (2007). Seed coat development and dormancy. In Seed Development and Germination. <a class="external-link" href="https://doi.org/10.1002/9780470988848.ch2">https://doi.org/10.1002/9780470988848.ch2</a></p>
<p id="bib3">Dorone, Y., Boeynaems, S., Flores, E., Jin, B., Hateley, S., Bossi, F., Lazarus, E., Pennington, J.G., Michiels, E., De Decker, M., et al. (2021). A prion-like protein regulator of seed germination undergoes hydration-dependent phase separation. Cell. <a class="external-link" href="https://doi.org/10.1016/j.cell.2021.06.009">https://doi.org/10.1016/j.cell.2021.06.009</a></p>
<p id="bib4">Gallardo, K., Job, C., Groot, S.P., Puype, M., Demol, H., Vandekerckhove, J., and Job, D. (2001). Proteomic analysis of arabidopsis seed germination and priming. Plant Physiol. <i>126</i>, 835-848. <a class="external-link" href="https://doi.org/10.1104/pp.126.2.835">https://doi.org/10.1104/pp.126.2.835</a></p>
<p id="bib5">Gutterman, Y. (2000). Maternal effects on seeds during development. In Seeds: The Ecology of Regeneration in Plant Communities (CAB International). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00731-0/sref5">View at publisher</a></p>
<p id="bib6">Jaganathan, G.K. (2016). Influence of maternal environment in developing different levels of physical dormancy and its ecological significance. Plant Ecol. <i>217</i>, 71-79. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00731-0/sref6">View at publisher</a></p>
<p id="bib7">Mamontova, A.V., Solovyev, I.D., Savitsky, A.P., Shakhov, A.М., Lukyanov, K.A., and Bogdanov, A.M. (2018). Bright GFP with subnanosecond fluorescence lifetime. Scientific Rep. <i>8</i>, 1-5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00731-0/sref7">View at publisher</a></p>
<p id="bib8">Measures, M., Weinberger, P., and Baer, H. (1973). Variability of plant growth within controlled-environment chambers as related to temperature and light distribution. Can. J. Plant Sci. <i>53</i>, 215-220. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00731-0/sref8">View at publisher</a></p>
<p id="bib9">Rivero, L., Scholl, R., Holomuzki, N., Crist, D., Grotewold, E., and Brkljacic, J. (2014). Handling Arabidopsis plants: growth, preservation of seeds, transformation, and genetic crosses. Methods Mol. Biol. <i>1062</i>, 3-25. <a class="external-link" href="https://doi.org/10.1007/978-1-62703-580-4_1">https://doi.org/10.1007/978-1-62703-580-4_1</a></p>
<p id="bib10">Steinbrecher, T. and Leubner-Metzger, G. (2018). Tissue and cellular mechanics of seeds. Curr. Opin. Genet. Dev. <i>51</i>, 1-10. <a class="external-link" href="https://doi.org/10.1016/j.gde.2018.03.001">https://doi.org/10.1016/j.gde.2018.03.001</a></p>
<p id="bib11">Wulff, R.D. (2017). Environmental maternal effects on seed quality and germination. In Seed Development and Germination (Routledge), pp. 491-505. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00731-0/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Seung Rhee (<a href="mailto:srhee@carnegiescience.edu">srhee@carnegiescience.edu</a>).</p>
<h4>Materials availability</h4>
<p>Plant lines used in this study will be made available on request from the lead contact upon completion of a Materials Transfer Agreement.</p>
<h4>Data and code availability</h4>
<p>This study did not generate new datasets or codes.</p>
<h3>Acknowledgments</h3>
<p>This research was supported, in part, by the <a href="https://doi.org/10.13039/100000015">U.S. Department of Energy</a>, Office of Science, Office of Biological and Environmental Research, Genomic Science Program grant nos. DE-SC0018277, DE-SC0008769, DE-SC0020366, and DE-SC0021286 and the U.S. <a href="https://doi.org/10.13039/100000001">National Science Foundation</a> grants MCB-1617020 and IOS-1546838 to S.Y.R. Y.D. was supported, in part, by the Stanford Graduate Fellowship in Science and Engineering, Carnegie Institution for Science and Brigitte Berthelemot. S.B. acknowledges an EMBO Long Term Fellowship. We are grateful to Drs. A. Gitler, F. Bossi, S. Sukenik, M.B. Mudgett, D. Jarosz, and H. Meyer for helpful discussion and suggestions. We thank H. Cartwright and A. Malkovskiy at the Carnegie’s Department of Plant Biology Advanced Imaging Facility, G. Materassi-Shultz for growth facilities management, and D. Irizarry for graphical assistance.</p>
<h3>Author contributions</h3>
<p>Y.D.: Conceptualization, Methodology, Software, Validation, Formal Analysis, Investigation, Writing-Original Draft, Visualization. S.B.: Conceptualization, Formal Analysis, Writing – Review &amp; Editing. S.Y.R.: Conceptualization, Supervision, Funding acquisition, Writing – Review &amp; Editing.</p>
<h3>Declaration of interests</h3>
<p>Y.D., S.B., and S.Y.R. are inventors on a PCT Patent Application named “FLOE1-MEDIATED MODULATION OF SEED LONGEVITY AND GERMINATION RATES” filed on August 6th 2021 by The Board of Trustees of the Leland Stanford Junior University and Carnegie Institution of Washington (application number: PCT/US2021/045103).</p>
</section>
</article>