<article>
<section>
<h1 id="header">Protocol for cell type-specific labeling, enrichment, and proteomic profiling of plasma proteins in mice</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Wei Wei,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#fn1">6</a>,<a href="#cor1">*</a></sup> Nicholas M. Riley,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#fn1">6</a>,<a href="#cor2">**</a></sup> Xuchao Lyu,<sup><a href="#aff1">1</a>,<a href="#aff4">4</a></sup> Carolyn R. Bertozzi,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a></sup> and Jonathan Z. Long<sup><a href="#aff1">1</a>,<a href="#aff4">4</a>,<a href="#fn2">7</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pathology, Stanford University School of Medicine, Stanford, CA 94305, USA</p>
<p id="aff2"><sup>2</sup>Department of Biology, Stanford University, Stanford, CA 94305, USA</p>
<p id="aff3"><sup>3</sup>Department of Chemistry, Stanford University, Stanford, CA 94305, USA</p>
<p id="aff4"><sup>4</sup>Stanford ChEM-H, Stanford University, Stanford, CA 94305, USA</p>
<p id="aff5"><sup>5</sup>Howard Hughes Medical Institute, Stanford University, Stanford, CA 94305, USA</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:wwbiomed@stanford.edu">wwbiomed@stanford.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:nmriley@stanford.edu">nmriley@stanford.edu</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:jzlong@stanford.edu">jzlong@stanford.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101014">10.1016/j.xpro.2021.101014</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Secreted polypeptides represent a fundamental axis of intercellular communication. Here, we present a protocol for the cell type-specific biotinylation, enrichment, and proteomic profiling of secreted plasma proteins directly in mice. This protocol uses conditional “turn-on” adeno-associated viruses expressing an endoplasmic reticulum-targeted biotin ligase to globally biotinylate proteins of the secretory pathway in a cell type-specific manner. Biotinylated secreted proteins can be directly purified from blood plasma and analyzed by SDS-PAGE gel or shotgun proteomics.</p>
<p>For complete information on the generation and use of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00720-6/sref6">Wei et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for cell type-specific biotinylation of secreted proteins directly in mice</li>
<li>Secreted proteins can be directly enriched from blood and profiled by proteomics</li>
<li>This protocol is applicable to any cell type for which a Cre mouse line is available</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1235-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol below describes the specific steps for proteomic profiling of the classical secretome (e.g., proteins conventionally secreted via the ER/Golgi pathway) from hepatocytes. This protocol can also be applied to profile in vivo secretomes from other cell types by transduction of the appropriate mouse Cre driver lines with the same conditional adeno-associated virus (AAV).</p>
<h3 id="sec1.1">Plasmid preparation and AAV production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6–8 weeks</div>
<ol>
<li>Acquire plasmid pAAV-FLEX-ER-TurboID (Addgene ID 160857) or send reagent requests to the lead contact.</li>
<li>Prepare AAV Plasmid
<ol type="a">
<li>Transform pAAV-FLEX-ER-TurboID plasmid into competent Top10 bacteria cells.</li>
<li>Incubate transformed bacteria on a shaker at 37°C overnight (12–16 h) in 500 mL LB culture supplemented with ampicillin.</li>
<li>Maxiprep plasmid using an endotoxin-free Qiagen Maxiprep kit (Qiagen 12362)</li>
<li>Confirm the coding sequence by sanger sequencing using primers pENN forward and reverse (primer sequences provided <a href="#key-resources-table">key resources table</a>).</li>
<li>To confirm intact AAV plasmid, perform a double restriction digest with EcoRI/HindIII to excise the coding insert. Perform a SmaI digest to confirm intact ITRs. Also load uncut plasmid to show that more than 80% of plasmid is in supercoil form (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1235-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Agarose gel of pAAV-FLEX-ER-TurboID plasmid diagnostic digests using the indicated restriction enzymes</div>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Produce AAV expressing FLEX-ER-TurboID
<ol type="a">
<li>Send endotoxin-free Maxiprep of pAAV-FLEX-ER-TurboID to the Penn Vector Core for production of AAV (serotype AAV8, small scale). This scale should be sufficient to produce a virus for ∼100 mice.</li>
<li>Upon receipt of the virus, freeze purified AAVs in 100 μL aliquots to prevent multiple freeze-and-thaw cycles. Purified AAV can be stably stored in −80°C for a few years.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend choosing an AAV serotype according to the specific cell type to be transduced. AAV8 was selected here because of robust and broad transduction of hepatocytes that can be observed with this serotype.</div>
<ol start="4">
<li>Acquire hepatocyte-specific homozygous Cre mouse driver line (Jackson Stock 003574) and breed with female C57BL/6J mice (Jackson Stock 000664) to generate hemizygous <i>Albumin</i>-Cre mice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To profile in vivo secretomes from additional cell types, choose an appropriate and validated Cre mouse driver that expresses Cre recombinase in your cell type of interest.</div>
<h3 id="sec1.2">Animal protocols and maintenance</h3>
<p>Perform all animal experiments according to procedures approved by academic institutions. All procedures described here were approved by the Stanford University Administrative Panel on Laboratory Animal Care (APLAC). Maintain mice in 12 h light-dark cycles at 22°C and ∼50% relative humidity. Feed mice a standard irradiated rodent chow diet.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-V5 antibody (1:1000 dilution)</td>
<td>Invitrogen</td>
<td>R960-25</td>
</tr>
<tr>
<td>Anti-beta Actin (1:1000 dilution)</td>
<td>Abcam</td>
<td>ab8227</td>
</tr>
<tr>
<td>Anti-mouse IgG IRDye 680RD (1:10000 dilution)</td>
<td>LI-COR Biosciences</td>
<td>925-68070</td>
</tr>
<tr>
<td>Anti-rabbit IgG IRDye 800RD (1:10000 dilution)</td>
<td>LI-COR Biosciences</td>
<td>926-32211</td>
</tr>
<tr>
<td>Anti-biotin Streptavidin AlexaFluor680 (1:1000 dilution)</td>
<td>Thermo Fisher Scientific</td>
<td>S32358</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pAAV-FLEx-ER-TurboID plasmid</td>
<td>Addgene</td>
<td>160857</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Biotin</td>
<td>Sigma-Aldrich</td>
<td>B4501-1G</td>
</tr>
<tr>
<td>DTT</td>
<td>Sigma-Aldrich</td>
<td>D0632-1G</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>Sigma-Aldrich</td>
<td>S2002-25G</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Sigma-Aldrich</td>
<td>S9888-500G</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>Sigma-Aldrich</td>
<td>D6750-100G</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate</td>
<td>Thermo Fisher Scientific</td>
<td>BP166-500</td>
</tr>
<tr>
<td>Sodium carbonate</td>
<td>Thermo Fisher Scientific</td>
<td>S263-500</td>
</tr>
<tr>
<td>Sodium bicarbonate</td>
<td>Thermo Fisher Scientific</td>
<td>187508</td>
</tr>
<tr>
<td>Sodium ascorbate</td>
<td>Thermo Fisher Scientific</td>
<td>A0539500G</td>
</tr>
<tr>
<td>Dimethyl sulfoxide</td>
<td>Sigma-Aldrich</td>
<td>D8418-100ML</td>
</tr>
<tr>
<td>Potassium chloride</td>
<td>Sigma-Aldrich</td>
<td>P3911-500G</td>
</tr>
<tr>
<td>EDTA</td>
<td>Sigma-Aldrich</td>
<td>E9884-100G</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>Sigma-Aldrich</td>
<td>1185-53-1</td>
</tr>
<tr>
<td>Acetonitrile</td>
<td>Thermo Fisher Scientific</td>
<td>A998-4</td>
</tr>
<tr>
<td>HALT protease inhibitor</td>
<td>Thermo Fisher Scientific</td>
<td>78429</td>
</tr>
<tr>
<td>NP-40 alternative</td>
<td>Merck Millipore</td>
<td>492016-100ML</td>
</tr>
<tr>
<td>Chloroform</td>
<td>Thermo Fisher Scientific</td>
<td>C607-4</td>
</tr>
<tr>
<td>Ponceau S solution</td>
<td>Sigma-Aldrich</td>
<td>P7170-1L</td>
</tr>
<tr>
<td>NuPAGE MOPS SDS Running Buffer (20×)</td>
<td>Invitrogen</td>
<td>NP0001-02</td>
</tr>
<tr>
<td>NuPAGE 4-12% Bis-Tris Protein Gels, 1.0 mm, 15-well</td>
<td>Thermo Fisher Scientific</td>
<td>NP0323BOX</td>
</tr>
<tr>
<td>Odyssey blocking buffer</td>
<td>LI-COR Biosciences</td>
<td>927-50000</td>
</tr>
<tr>
<td>Seeblue plus 2 protein ladder</td>
<td>Thermo Fisher Scientific</td>
<td>LC5925</td>
</tr>
<tr>
<td>NuPAGE™ LDS Sample Buffer (4×)</td>
<td>Thermo Fisher Scientific</td>
<td>NP0008</td>
</tr>
<tr>
<td>Urea</td>
<td>Sigma-Aldrich</td>
<td>31K0038</td>
</tr>
<tr>
<td>Acetone</td>
<td>Thermo Fisher Scientific</td>
<td>A184</td>
</tr>
<tr>
<td>Formic acid</td>
<td>Thermo Fisher Scientific</td>
<td>A117-50</td>
</tr>
<tr>
<td>Phosphoric acid</td>
<td>Sigma-Aldrich</td>
<td>345245-100ML</td>
</tr>
<tr>
<td>Trolox</td>
<td>Sigma-Aldrich</td>
<td>648471-500MG</td>
</tr>
<tr>
<td>0.9% saline solution</td>
<td>Teknova</td>
<td>S5825</td>
</tr>
<tr>
<td>TEAB solution (1M)</td>
<td>Sigma-Aldrich</td>
<td>T7408-100ML</td>
</tr>
<tr>
<td>Kolliphor EL</td>
<td>Sigma-Aldrich</td>
<td>C5135-500G</td>
</tr>
<tr>
<td>Trypsin</td>
<td>Promega</td>
<td>V5113</td>
</tr>
<tr>
<td>Iodoacetamide</td>
<td>Sigma-Aldrich</td>
<td>A3221</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>SilverQuest™ Silver Staining Kit</td>
<td>Thermo Fisher Scientific</td>
<td>LC6070</td>
</tr>
<tr>
<td>Trans-blot turbo RTA transfer kit, nitrocellulose</td>
<td>Bio-Rad Laboratories</td>
<td>1704271</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>C57BL/6J (M. musculus)</td>
<td>The Jackson Laboratory</td>
<td>000664</td>
</tr>
<tr>
<td>
<i>Albumin</i>-Cre mice</td>
<td>The Jackson Laboratory</td>
<td>003574</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>One Shot™ TOP10 Chemically Competent E. coli</td>
<td>Invitrogen</td>
<td>C404010</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>pENN forward: gcctctgctaaccatgttcatg</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>pENN reverse: gcagcgtatccacatag</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Proteomic data</td>
<td>This study</td>
<td>ProteomeXchange: PXD021602</td>
</tr>
<tr>
<td>Processed proteomic datasets</td>
<td><a href="#bib6">Wei et al. (2021)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>3 kDa Molecular Weight Cutoff Filter</td>
<td>Amicon</td>
<td>UFC9003</td>
</tr>
<tr>
<td>EcoRI</td>
<td>New England Biolabs</td>
<td>R0101S</td>
</tr>
<tr>
<td>HindIII</td>
<td>New England Biolabs</td>
<td>R0104S</td>
</tr>
<tr>
<td>SmaI</td>
<td>New England Biolabs</td>
<td>R0141S</td>
</tr>
<tr>
<td>S-Trap™ micro columns</td>
<td>ProtiFi</td>
<td>C02-micro-40</td>
</tr>
<tr>
<td>Exel International Insulin Syringes</td>
<td>Thermo Fisher Scientific</td>
<td>14-841-32</td>
</tr>
<tr>
<td>lithium heparin tube</td>
<td>BD</td>
<td>365985</td>
</tr>
<tr>
<td>PrecisionGlide Needle 21G</td>
<td>BD</td>
<td>305129</td>
</tr>
<tr>
<td>Exel International Insulin Syringes 29G</td>
<td>Exel International</td>
<td>14-841-32</td>
</tr>
<tr>
<td>Tailveiner Restrainer for Mice</td>
<td>BRAINTREE SCIENTIFIC, INC.</td>
<td>TV-150 STD</td>
</tr>
<tr>
<td>Streptavidin magnetic beads</td>
<td>Thermo Fisher Scientific</td>
<td>65601</td>
</tr>
<tr>
<td>Bulk tubes</td>
<td>Thermo Fisher Scientific</td>
<td>15340162</td>
</tr>
<tr>
<td>Metal Bulk Beads</td>
<td>Thermo Fisher Scientific</td>
<td>15340158</td>
</tr>
<tr>
<td>Benchmark BeadBlaster Homogenizer</td>
<td>Thermo Fisher Scientific</td>
<td>15-340-163</td>
</tr>
<tr>
<td>Orbitrap Fusion Tribrid Mass Spectrometer</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>NanoDrop Spectrophotometer</td>
<td>Thermo Fisher Scientific</td>
<td>ND-ONE-W</td>
</tr>
<tr>
<td>Dionex UltiMate 3000 RPLCnano system</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Acclaim PepMap 100 C18, 5-μm particles</td>
<td>Thermo Fisher Scientific</td>
<td>164213</td>
</tr>
<tr>
<td>Acclaim™ PepMap™ 100 C18 HPLC Columns (75 μm, packed with 2-μm, 100-Å)</td>
<td>Thermo Fisher Scientific</td>
<td>164942</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>RIPA buffer (store at 4°C for up to one year)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NP40</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate</td>
<td>0.1% (w/v)</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.5% (w/v)</td>
<td>2.5 g</td>
</tr>
<tr>
<td>PBS</td>
<td>/</td>
<td>Fill up to 500 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>Washing buffer (store at 4°C for up to 6 months)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl</td>
<td>50 mM</td>
<td>6 g</td>
</tr>
<tr>
<td>NP40</td>
<td>1%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate</td>
<td>0.1% (w/v)</td>
<td>1 g</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>150 mM</td>
<td>8.77 g</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.5% (w/v)</td>
<td>2.5 g</td>
</tr>
<tr>
<td>Sodium ascorbate</td>
<td>10 mM</td>
<td>1.98 g</td>
</tr>
<tr>
<td>EDTA</td>
<td>1 mM</td>
<td>0.29 g</td>
</tr>
<tr>
<td>HALT protease inhibitor (add upon use)</td>
<td>1×</td>
<td>10 mL (100×)</td>
</tr>
<tr>
<td>Trolox</td>
<td>5 mM</td>
<td>1.25 g</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>10 mM</td>
<td>0.65 g</td>
</tr>
<tr>
<td>PBS</td>
<td>/</td>
<td>Fill up to 1000 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>KCl solution (store at 22°C–25°C for up to one year)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KCl</td>
<td>1M</td>
<td>37.28 g</td>
</tr>
<tr>
<td>PBS</td>
<td>/</td>
<td>Fill up to 500 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>Na<sub>2</sub>CO<sub>3</sub> solution (store at 22°C–25°C for up to one year)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>CO<sub>3</sub>
</td>
<td>0.1M</td>
<td>5.3 g</td>
</tr>
<tr>
<td>PBS</td>
<td>/</td>
<td>Fill up to 500 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>Urea solution (PH=8.0, store at 22°C–25°C for up to one year)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Urea</td>
<td>2M</td>
<td>60 g</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>10 mM</td>
<td>0.6 g</td>
</tr>
<tr>
<td>PBS</td>
<td>/</td>
<td>Fill up to 500 mL and adjust PH to 8.0</td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>Elution buffer (store at −80°C for up to 6 months)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Biotin</td>
<td>4 mM</td>
<td>48.9 mg</td>
</tr>
<tr>
<td>DTT</td>
<td>20 mM</td>
<td>3.08 g</td>
</tr>
<tr>
<td>NuPAGE™ LDS Sample Buffer (4×)</td>
<td>2×</td>
<td>50 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>/</td>
<td>Fill up to 100 mL</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">AAV transduction and biotinylation of secretomes in mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 weeks</div>
<p>The following steps describe the specific transduction of <i>Albumin</i>-Cre mice with pAAV-FLEX-ER-TurboID followed by biotinylation of the hepatocyte secretome. We recommend using wild-type C57BL/6J mice as negative control for background labeling. 3 to 5 mice per genotype should be used for viral transduction.</p>
<ol>
<li>Transduce pAAV-FLEX-ER-TurboID into adult <i>Albumin</i>-Cre mice
<ol type="a">
<li>Thaw aliquoted AAV on ice. Prepare AAV injection solution by diluting 10<sup>11</sup> genome copies (GC) in saline in a total volume of 100 μL per mouse. We recommend making an extra solution for each mouse.
<div class="note">
<span class="note-title">Note:</span> Higher dose, i.e., 10<sup>12</sup> GC, of AAV may be used to achieve better viral transduction efficiency. We have found 10<sup>11</sup> GC to be sufficient to label hepatocyte secretome in mice and higher dose to be more efficient for other cell types that are less frequently transduced by AAV.</div>
</li>
<li>Load 100 μL of the saline injection solution containing AAV into a 29G needle.
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to empty air from the needle and not to spill AAV solution onto skin or eyes.</div>
</li>
<li>Restrain 8–12 week old <i>Albumin</i>-Cre male mice using a Tailveiner Restrainer. The tail vein is exposed by pulling it by the tail through the slotted tube. Orient the mouse so that the head is away from you and the tail is pointing towards you. Use 70% ethanol to disinfect the tail.
<div class="optional">
<span class="optional-title">Optional:</span> Use a mouse warmer lamp to help dilate the tail vein.</div>
</li>
<li>An ideal injection site is ∼2 cm away from the base of the tail (<a href="#mmc1">Methods video S1</a>). Hold the tail taut at the approximate injection site with thumb above and forefinger below the tail. Visually identify the tail vein, which appears slightly darker in color relative to the tail. Insert the needle bevel up into the vein at an ∼15 degree angle to the skin. Inject the AAV solution into the vein over a period of 1–5 s (<a href="#fig2">Figure 2</a>).
<div class="note">
<span class="note-title">Note:</span> Other routes of AAV administration may also be used here. For example, direct viral injections into the muscle or adipose has been successfully employed by others (<a href="#bib3">Long et al., 2016</a>; <a href="#bib7">Zeng et al., 2019</a>) to drive expression in these tissues. In this protocol we only describe systemic transduction via the tail vein.</div>
<div class="critical">
<span class="critical-title">Critical:</span> No resistance should be felt during the injection if the needle is appropriately inserted into the tail vein. If resistance is felt, remove the needle and repeat the injection at a slightly different location ∼0.5 cm away from the initial injection site.</div>
</li>
<li>After injection, return mice to home cages for 2 weeks to allow for expression of the AAV.
<div class="note">
<span class="note-title">Note:</span> Some conditional Cre-ER mouse drivers require administration of tamoxifen after viral transduction to turn on the expression of the ER-TurboID cassette. In this case, recombination can be induced two weeks after AAV injection by intraperitoneally injecting 2 mg tamoxifen powder dissolved in 100 μL corn oil for each mouse for 5 consecutive days. Return mice to home cages for 2 weeks to allow for expression of the AAV after the final tamoxifen injection.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> AAV expression time can vary between 2 weeks to 6 weeks according to the experimental need.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Two weeks after AAV transduction, deliver biotin by drinking water using the following procedure: add 100 mg biotin to a mixture of 294 mL water and 6 mL DMSO. Heat the solution to 95°C for 20min to completely dissolve biotin. Cool down the biotin-supplemented water to room temperature (20°C–25°C) and provide it to micer in place of their normal drinking water for three consecutive days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The biotin water labeling period can be adjusted longer or shorter depending on experimental needs. We have not observed any differences in total water consumed between regular water and biotin-supplemented water. Biotin may form long crystals and precipitate in drinking water. If this happens, prepare fresh biotin-supplemented water every day.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Secretome labeling can also be accomplished by intraperitoneal injection of biotin. Prepare the biotin injection solution by dissolving biotin in a solution of 18:1:1 saline:Kolliphor EL:DMSO (by volume), 2.5 mg/mL. Heat solution at 95°C for 10 min to fully dissolve biotin. Once cool to room temperature, inject 200 μL per mouse per day using a 29G syringe for a final dose of 24 mg/kg biotin. Fresh intraperitoneal biotin injection solution should be delivered daily for three consecutive days but may vary according to the experimental needs. We have found strong biotinylation signals in the blood a few hours after a single intraperitoneal injection of biotin.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1235-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Injection of AAV via tail vein</div>
<p>(A) Mouse restrainer and 29G syringe.</p>
<p>(B) Mouse tail vein position.</p>
<p>(C) Tail vein injection position.</p>
</figcaption>
</figure>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1235-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Tail vein injection of AAV saline solution into mice, related to step 1d</div>
</figcaption>
</figure>
<h3 id="sec3.2">Characterization of secretome labeling by SDS-PAGE gel</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>The following steps describe the protocol for gel-based characterization of biotinylated secretome.</p>
<ol start="3">
<li>Harvest blood plasma and liver tissue from <i>Albumin</i>-Cre mice and control wild-type C57BL/6J mice.
<ol type="a">
<li>Preparation for liver and blood collection.
<ol type="i">
<li>Prepare eppendorf tubes to collect tissues and lithium heparin tubes (BD, 365985) to collect mouse blood.</li>
<li>Prepare dry ice to store liver tissues and wet ice to store blood.</li>
<li>Prepare dissection scissors, tweezers, diapers and 70% ethanol.</li>
</ol>
</li>
<li>Perform a submandibular bleed to collect blood (<a href="#bib2">JoVE Science Education Database and Lab Animal Research, 2021</a>). Restrain the mouse by the scruff of the neck. Using a 21G needle, puncture the submandibular vein by insertion of the needle 1–2 mm into the cheek. The puncture site should be slightly behind the mandible, but in front of the ear canal (<a href="#fig3">Figure 3</a>). Use a lithium heparin tube to collect blood. When the flow of blood slows, repeat the same submandibular bleed procedure on the other cheek.</li>
<li>Centrifuge total blood (2,500×<i>g</i>, 5 min, 4°C). The supernatant is the blood plasma fraction. Freeze blood plasma on dry ice.
<div class="note">
<span class="note-title">Note:</span> The plasma fraction should be pale yellow in color. If it becomes red, be cautious that the plasma has been contaminated with the red blood cell contents. Whole blood can also be stored on wet ice for up to 30 min prior to centrifugation. Approximately 100–500 μL plasma can be obtained per mouse.</div>
</li>
<li>Euthanize the mouse by CO<sub>2</sub>. Use ethanol to spray down the mouse abdomen and use dissection scissors and tweezers to open the intraperitoneal cavity. Excise the liver without the gallbladder. Freeze down the liver immediately in dry ice.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Plasma and liver samples can be stored in −80°C for a few years.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1235-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Blood plasma collection</div>
<p>(A) Puncture site for submandibular bleed.</p>
<p>(B) Example of successful blood collection into a lithium tube.</p>
</figcaption>
</figure>
<ol start="4">
<li>Prepare blood plasma and liver tissue for Western blot analysis.
<ol type="a">
<li>For blood plasma, add 4 μL of blood plasma to a 20 μL 4× loading buffer. Then add PBS to 80 uL final volume.</li>
<li>For the liver, add 5 metal bulk beads and 500 μL RIPA buffer into a bulk tube. Add ∼100 mg liver tissue. Homogenize liver samples using a Benchmark BeadBlaster Homogenizer (40 s on 20 s off, 5 min, 4°C). Centrifuge (16,000×<i>g</i>, 10 min, 4°C), remove supernatant containing liver proteins, and quantify protein concentration using Nanodrop One. Add 400 μg proteins into a 20 μL 4× loading buffer and add PBS to the 80ul final volume.</li>
<li>Boil the blood and liver samples at 95°C for 10min and cool down to room temperature prior to loading on gel.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The 4× sample loading buffer here may be replaced by home-made buffer using the following recipe: 2.0 mL 1M Tris-HCl pH 6.8, 0.8 g SDS, 4.0 mL 100% glycerol, 0.4 mL 14.7 M β-mercaptoethanol, 1.0 mL 0.5 M EDTA, 8 mg bromophenol Blue for making 10 mL buffer.</div>
<ol start="5">
<li>Perform Western blotting analysis to detect biotinylated proteins from blood plasma and liver.
<ol type="a">
<li>Load 20 μL plasma or tissue samples into each well of a 4%–12% SDS-PAGE gel (approximately 100 μg protein/lane). Run at 120 V for 1.5 h.</li>
<li>Transfer the gel to the nitrocellulose membrane.</li>
<li>Block the membrane with a 5 mL Odyssey blocking buffer at room temperature for 30 min.</li>
<li>For detecting biotin labeling signals, incubate the membrane with 1:1000 streptavidin-Alexa Fluor 680 antibody at room temperature for 1 h or in the cold room overnight (12–24 h).</li>
<li>For detecting the expression of ER-TurboID, incubate the membrane with 1:1000 mouse anti-V5 in the cold room overnight (12–24 h), followed by 1:10000 anti-mouse IgG as the secondary antibody for 30 min.</li>
<li>For loading control of blood plasma, use POnceau S staining reagent to stain the nitrocellulose membrane and choose the region around 50kd across the blot for comparison. For loading control of the liver, use 1:1000 rabbit anti-mouse actin as the primary antibody and 1:10000 anti-rabbit IgG as the secondary antibody.</li>
<li>Image the blots at LI-COR gel imager (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1235-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Western blotting of biotinylated proteins from liver lysates and blood plasma using the indicated antibodies. Wild-type C57BL/6J mouse was injected with pAAV-FLEX-ER-TurboID and supplemented with biotin as negative control for background labeling.</div>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1235-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Silver stain of elution from streptavidin-enriched plasma samples from one <i>Albumin</i>-Cre mouse (right) and one wild-type C57BL/6J mouse that was injected with pAAV-FLEX-ER-TurboID and supplemented with biotin as negative control for background labeling (left).</div>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> NuPAGE MOPS SDS Running Buffer (20×) may be replace by home-made buffer using the following recipe: 209.2 g MOPS, 121.2 g Tris base, 20 g SDS, 6 g EDTA (free acid) or 7.44 g disodium EDTA for making 1 L buffer.</div>
<div class="note">
<span class="note-title">Note:</span> The intensity of biotin signals on a gel can be variable and dependent on the cell types transduced. For parenchymal cells of an organ, e.g., hepatocytes in the liver, biotin signals are very strong in both tissue and plasma levels by western blotting. However, for some other cell types, e.g., endothelial cells and immune cells, biotin signals are generally weak in both tissue and plasma levels by western blotting. To obtain higher sensitivity by gel-based detection, biotinylated protein samples can be immunoaffinity purified by streptavidin beads, separated by SDS-PAGE, and analyzed by Silver Stain (<a href="#fig5">Figure 5</a>).</div>
<h3 id="sec3.3">Enrichment of biotinylated plasma proteins</h3>
<p>The following steps describe the protocol of streptavidin-based enrichment of biotinylated plasma proteins for proteomic analysis.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 20 h</div>
<ol start="6">
<li>Remove excess biotin from plasma.
<ol type="a">
<li>Thaw frozen plasma on ice (200 μL). To remove excess biotin in plasma, dilute 200 μL plasma from a single mouse into 15 mL PBS and concentrate 30-fold using 3 kDa filter tubes by centrifuging (4,000 x g, 1.5 h, 4°C).</li>
<li>Discard the flow through and repeat the concentration step.</li>
<li>Discard the flow through, add another 5 mL PBS into the tube and concentrate to a final volume of 500 μL. This represents 9,000-fold final dilution.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Pale yellow sediments may be present in the plasma and may block the filter during the centrifugation. Remove the sediments using a pipette tip.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Further dilution of biotin in plasma can be achieved by repeating the centrifugation step 6(a).</div>
<ol start="7">
<li>Wash streptavidin beads.
<ol type="a">
<li>Vortex the bottle containing Dynabeads MyOne Streptavidin T1 magnetic beads for 1 min.</li>
<li>Using a 1 mL pipette tip, transfer 200 μL bead slurry into eppendorf tubes placed on a magnetic rack. Wait 10s until all the beads are fully attached to the wall.
<div class="critical">
<span class="critical-title">Critical:</span> Take out the beads immediately after vortex to ensure the same amount of beads are used for each sample.</div>
</li>
<li>Remove supernatant with 200 μL pipette tips. Add 1 mL washing buffer (50 mM Tris-HCl, 150 mM NaCl, 0.1% SDS, 0.5% sodium deoxycholate, 1% NP-40, 1 mM EDTA, 1× HALT protease inhibitor, 5 mM trolox, 10 mM sodium azide and 10 mM sodium ascorbate) and vortex for 3s. Aspirate washing buffer. Repeat the washing step twice.
<div class="note">
<span class="note-title">Note:</span> When making washing buffer stock, add every component excluding 1× HALT protease inhibitor. Add protease inhibitor fresh. The washing buffer stock can be stored in a cold room (4°C) for a few months.</div>
</li>
<li>Remove supernatant and resuspend beads in a 100 μL washing buffer.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Add 500 μL diluted plasma to the washed beads in a 100 μL washing buffer (600 μL volume final) and rotate in a cold room overnight (12–24 h). Biotinylated plasma proteins will bind to streptavidin beads.</li>
</ol>
<ol start="9">
<li>Wash streptavidin beads containing bound biotinylated plasma proteins.
<ol type="a">
<li>Place the eppendorf tubes containing streptavidin beads on a magnetic rack and wait 10s until the beads are fully attached to the wall.</li>
<li>Remove the supernatant using 200 μL pipette tips and save as flow through. Add 1 mL washing buffer. Vortex for 3s and put back on a magnetic rack. Repeat this step.</li>
<li>Add 1 mL 1M KCl solution. Vortex for 3 s and put back on a magnetic rack.</li>
<li>Add 1 mL 0.1M Na<sub>2</sub>CO<sub>3</sub> solution. Vortex for 3 s and put back on a magnetic rack.</li>
<li>Add 1 mL 2M urea in 10 mM Tris-HCl (pH 8.0). Vortex for 3 s and place back on a magnetic rack.
<div class="note">
<span class="note-title">Note:</span> This step should not exceed 1min.</div>
</li>
<li>Add 1 mL washing buffer. Vortex for 3s and put back on a magnetic rack. Repeat the wash step once.</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Elute biotinylated plasma proteins from beads
<ol type="a">
<li>Following the second wash, aspirate wash supernatant completely. Add 60 μL elution buffer (2× NuPAGE™ LDS Sample Buffer supplemented with 20 mM DTT and 2 mM biotin). Vortex and spin down the beads.
<div class="note">
<span class="note-title">Note:</span> To make an elution buffer, dissolve 4 mM biotin into a 4× loading buffer by boiling for 10 min and add PBS to make 2× loading buffer with 2 mM biotin. Then dissolve DTT in water to make 500 M stock and Add DTT into the elution buffer to reach 20 mM final DTT concentration. Aliquot elution buffer into 500ul aliquots and store at -80°C for future use (up to six months).</div>
</li>
<li>Boil the beads at  95°C for 10min. Put the tubes back on a magnetic rack and transfer the supernatant into a new tube. In total ∼ 85 μL final eluant containing biotinylated proteins should be recovered.</li>
<li>Take 5 μL of final eluant and analyze by SDS-PAGE using either a silver stain (ThermoFisher LC6070) or anti-streptavidin blotting. As a control, use 5 μL elution from wild-type C57BL/6J mice (e.g., no cre-mediated recombination) injected with pAAV-FLEX-ER-TurboID and supplemented with biotin water. Successfully enrichment should show much more intense bands in <i>Albumin</i>-Cre mice samples compared to samples from control mice (<a href="#fig5">Figure 5</a>).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Once successful streptavidin isolation has been confirmed by SDS-PAGE, samples can be frozen at -80°C for long-term storage. Alternatively, proceed with S-trap-based sample processing for proteomic analysis.</div>
<div class="note">
<span class="note-title">Note:</span> This streptavidin enrichment protocol is adapted from <a href="#bib1">Branon et al. (2018</a>).</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">S-trap-based proteomic sample processing for biotinylated plasma proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>The following steps describe the protocol of proteomic analysis of the final eluant obtained in Step 10 using an S-trap spin column.</p>
<ol start="11">
<li>Alkylate biotinylated plasma proteins
<ol type="a">
<li>Make 0.25M iodoacetamide (IAA) solution by dissolving 23.1 mg IAA into 500 μL water.</li>
<li>Add 9.6 μL of 0.25 M IAA into 80 μL final eluant obtained in Step 10 and vortex to mix. Place the tube in the dark for 30min.</li>
</ol>
</li>
<li>Bind and wash biotinylated plasma proteins on S-trap columns
<ol type="a">
<li>Add 1.1 μL 85% phosphoric acid and vortex to mix. The color should turn from blue to green.</li>
<li>Add 560 μL bind/wash buffer (add 3 mL 1M TEAB solution into 27 mL methanol and mix) to each tube and vortex to mix.</li>
<li>Add 150 μL sample into each micro-S-Trap column and spin (4,000×<i>g</i>, 20 s). Discard flow through.</li>
<li>Rotate the top of the column 180 degrees and repeat step (c) until the whole sample has passed through.</li>
<li>Add 150 μL bind/wash buffer and spin (4,000×<i>g</i>, 30 s). Discard flow through.</li>
<li>Rotate the top of the column 180 degrees and repeat step (e) 3 times.</li>
<li>Transfer the upper part of the column into a new 2 mL eppendorf tube.</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Digest biotinylated proteins on S-trap column
<ol type="a">
<li>Add 1 μg trypsin into 25 μL of 50mM TEAB solution. Add 25 μL trypsin solution into the S-Trap column. Spin the column briefly and return the solution that passes through the top of the column.</li>
<li>Close the cap loosely to prevent evaporative loss.
<div class="critical">
<span class="critical-title">Critical:</span> Do not shake! Shaking will greatly impede the digestion performance.</div>
</li>
<li>Digest at 47°C for 90 min.</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Elute tryptic peptides
<ol type="a">
<li>Add 40 μL of 50mM TEAB and spin (1,000×<i>g</i>, 60 s). Save the flow through.</li>
<li>Add 40 μL of 0.2% formic acid and spin (1,000×<i>g</i>, 60 s). Save the flow through.</li>
<li>Add 40 μL of 50% acetonitrile in water + 0.2% formic acid and spin (1,000×<i>g</i>, 60 s). Save the flow through.</li>
<li>Add 40 μL of 0.2% formic acid and spin (4,000×<i>g</i>, 60 s). Save the flow through.</li>
<li>Pool eluted peptides by combining the flow through samples. Lyophilize samples and resuspend in 20 μL 0.2% formic acid.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Proteomic samples can be stored in -80°C for up to 2 years.</div>
<div class="note">
<span class="note-title">Note:</span> This S-Trap protocol is adjusted from Protifi, S-Trap™ micro spin column digestion protocol (<a href="http://mass-spec.wp.st-andrews.ac.uk/files/2019/10/S-Trap-method.pdf">http://mass-spec.wp.st-andrews.ac.uk/files/2019/10/S-Trap-method.pdf</a>).</div>
<h3 id="sec3.5">Shotgun liquid chromatography mass spectrometry (LC-MS/MS) analysis of biotinylated plasma proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h per sample</div>
<p>The following steps describe the protocol and parameters of LC-MS/MS runs.</p>
<ol start="15">
<li>Set up the nanoflow liquid chromatography conditions.
<ol type="a">
<li>Estimate peptide concentration using a NanoDrop Spectrophotometer (Thermo Fisher) with absorbance measurements at 205 nm (A205), or using quantitative peptide colorimetric or fluorometric assays (Pierce).</li>
<li>Inject ∼1 μg of resuspended peptides on column using a Dionex UltiMate 3000 RPLCnano system (Thermo Fisher) to control and drive the mobile phases (A, water with 0.2% formic acid and B, 100% acetonitrile with 0.2% formic acid).</li>
<li>Use an integrated loading pump to load peptides onto a trap column (Acclaim PepMap 100 C18, 5-μm particles, 20 mm in length, Thermo Fisher) during the first 6 min of the acquisition. Hold the gradient at an isocratic flow at 0% B during this time.</li>
<li>At 6.1 min, use the switching pump to move the trap column in line with the analytical column, which is a 50-cm EASY-Spray reversed-phase LC column (inner diameter of 75 μm, packed with 2-μm, 100-Å, PepMap C18 particles, Thermo Fisher).</li>
<li>Perform gradient elution at 300 nl/min and increase the gradient from 0% to 5% B from 6 to 6.1 min, followed by an increase from 5% to 25% B from 6 to 86 min, an increase from 25% to 90% B from 86 to 94 min, isocratic flow at 90% B from 94 to 102 min and a re-equilibration at 0% for 18 min, for a total analysis time of 120 min.</li>
</ol>
</li>
<li>Set up the mass spectrometer conditions. The instrument described in these steps is an Orbitrap Fusion Tribrid Mass Spectrometer (Thermo Fisher).
<ol type="a">
<li>Ionize eluting peptide with an EASY-Spray ionization source (Thermo Fisher) held at +2.2 kV compared to ground. Use the integrated column heater on the EASY-Spray source to hold the column temperature at 45°C.</li>
<li>Hold the inlet capillary temperature at 275°C.</li>
<li>Collect survey scans of peptide precursors in the Orbitrap from 350–1500 m/z with an AGC target of 1,000,000, a maximum injection time of 50 ms, and a resolution of 120,000 at 200 m/z.</li>
<li>Enable Monoisotopic precursor selection for peptide isotopic distributions. Select precursors of Z = 2–5 for data-dependent MS/MS scans for 2s of cycle time.</li>
<li>Set the dynamic exclusion to 45 s, with a ±10 ppm window set around the precursor monoisotope.</li>
<li>Use an isolation window of 1 m/z to select precursor ions with the quadrupole.</li>
<li>Collect MS/MS scans using HCD at a normalized collision energy of 30, with an AGC target of 50,000 and a maximum injection time of 54 ms.</li>
<li>Perform mass analysis in the Orbitrap with a resolution of 30,000 at 200 m/z and an automatically determined mass range.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A successful in vivo secretome labeling experiment should show a clear enrichment of cell type-specific biotinylated proteins from blood plasma versus controls that lack viral transduction or biotin administration. As a representative result, an anti-streptavidin Western blot of liver lysates and blood plasma from a hepatocyte secretome labeling experiment is shown in <a href="#fig4">Figure 4</a>, and the heatmap of biotinylated plasma proteins by shotgun proteomics analysis is shown in <a href="#fig6">Figure 6</a>.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1235-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Hierarchical clustering of z-score intensities for biotinylated secretome samples from three Alb-Cre mice (left) and three wild-type C57BL/6J mice that were injected with pAAV-FLEX-ER-TurboID and supplemented with biotin as negative control for background labeling (right). Some of known hepatocyte-derived plasma proteins are listed on the right as positive control. Scale bar: normalized Z-score intensities.</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">Proteomic data analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days, depending on number of samples</div>
<ol>
<li>Download MaxQuant (<a href="#bib4">Tyanova et al., 2016a</a>) and Perseus software platforms (<a href="#bib5">Tyanova et al., 2016b</a>) at <a href="https://maxquant.org/">https://maxquant.org/</a>. This website has excellent resources and documents to aid with setting up MaxQuant searches and using Perseus for data analysis beyond what is described here.</li>
<li>Set up the MaxQuant search to identify and quantify proteins from LC-MS/MS data.
<ol type="a">
<li>Load .RAW MS data files into MaxQuant and label each with the appropriate Experiment name.</li>
<li>Set group specific parameters to include:
<ol type="i">
<li>Type set to “Standard” with a multiplicity of 1.</li>
<li>Variable modifications of oxidation of methionine, biotinylation of lysine and protein N-terminal acetylation.</li>
<li>Fixed modification of carbamidomethylation of cysteine.</li>
<li>Digestion mode as “Specific” with “Trypsin/P” as Enzyme selection and a maximum missed cleavage setting of 2.</li>
<li>Label-free quantification set to “LFQ” with a minimum ratio count of 2, fast LFQ enabled, minimum number of neighbors set to 3, and average number of neighbors set to 6.</li>
<li>Instrument parameters set to 20 ppm and 4.5 ppm for first search MS1 tolerance and main search MS1 tolerance. All other parameters can be left as default settings.</li>
</ol>
</li>
<li>Set global parameters to include:
<ol type="i">
<li>A fasta file of the mouse reference proteome database downloaded from <a href="http://uniprot.org">uniprot.org</a> (canonical only). MaxQuant will append decoy sequences for false discovery rate (FDR) calculations automatically, as is set by default as “Revert” under the “Advanced” tab. Otherwise use default settings for other parameters in the “Sequences” tab.</li>
<li>Enable the “Match between runs” feature under the “Advanced Identification” tab.</li>
<li>Set peptide spectral match (PSM) FDR and Protein FDR to 0.01 under the “Identification” tab to ensure filtering of peptide identifications at 1% FDR and to apply a 1% protein FDR.</li>
<li>Under the “Protein quantification” tab, specify the minimum ratio count as 2, and enable the requirement for MS/MS for LFQ comparisons under the “Label free quantification” tab.</li>
<li>Set the FTMS MS/MS match tolerance to 20 ppm under the “MS/MS - FTMS” tab.</li>
</ol>
</li>
<li>Start MaxQuant processing using the number of total cores available on your processing system minus 1. The more cores used, the faster the processing time.</li>
<li>After MaxQuant processing finishes, inspect data using the output tables and the viewing features within the MaxQuant environment under the “Visualization” tab.</li>
</ol>
</li>
<li>After MaxQuant processing finishes, use Perseus to analyze data.
<ol type="a">
<li>Use a generic matrix upload to import the proteinGroups.txt results file from MaxQuant. Each column from the proteinGroups.txt file will appear, which can be sorted into appropriate variable types.
<ol type="i">
<li>Set LFQ intensity columns for each sample (will be the names you provided under “Experiment” in the MaxQuant search) as Main variables.</li>
<li>Numerical variables should include Peptides, Sequence coverage [%], Mol. weight [kDa], Q-value, Score, and Intensity (a summary value of intensity of a given protein group across all conditions).</li>
<li>Categorical variables should include Reverse and Potential contaminants.</li>
<li>Text variables should include Protein IDs, Majority Protein IDs, and Fasta headers.</li>
</ol>
</li>
<li>Filter rows based on categorical variables to remove rows (i.e., protein group IDs) that are indicated as potential contaminants and reverse hits. To do this select values that have “+” for the indicated categorical variable and select the “Remove matching rows” mode. This is done individually for each categorical variable, so filtering must be done twice to remove both potential contaminants and reverse hits.
<ol type="i">
<li>Depending on your experiment, it is worth checking the potential contaminants for protein groups of interest. This can be done in Perseus prior to filtering or with spreadsheet software (e.g., Microsoft Excel).</li>
</ol>
</li>
<li>Use “Categorical annotation rows” to name replicates of each experimental group with the same name. This enables further statistical processing.</li>
<li>Under the “Basic” tab, transform main column data (i.e., LFQ intensity values) from base 10 numbers into log2 space.</li>
<li>Check distributions using the histogram visualization tool to ensure log2 LFQ intensity distributions approximate normal distributions.</li>
<li>Filter rows based on valid values. This can depend on the experiment and expected outcomes. Here, proteins are expected to be detected in more than one replicate of a given experimental condition, but are not necessarily expected to be detected across multiple experimental conditions. In this case, require 2 or more valid values in at least one group designated in the groups set in the Categorical annotation rows step (3c above).</li>
<li>Replace missing values using imputation from a normal distribution. Default settings of a distribution width of 0.3 and a downshift of 1.8 are usually acceptable.</li>
<li>Perform ANOVA using the “Multiple-sample tests” window under the “Tests” tab. Recommended settings use an S0 value of 1, a permutation-based FDR set at 5% (0.05) with 250 randomizations for multiple testing corrections.</li>
<li>Filter rows based on the categorial column of “ANOVA Significant” to keep matching rows with the “+” indicator.</li>
<li>Under the “Normalization” tab, perform Z-score processing for protein groups calculated to be significant by ANOVA.</li>
<li>Perform hierarchical clustering (under the “Clustering/PCA” tab) using default settings (<a href="#fig6">Figure 6</a>). Ensure replicates from experimental conditions clustered together.</li>
<li>If desired, Perseus can also be used to perform pairwise comparisons using two-sample t-tests and volcano plots, perform principal component analyses if desired, and to calculate Fisher exact tests to determine enriched features (e.g., gene ontology terms).</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Although the protocol described here is the only methodology reported to date that enables cell type-specific secretome profiling in a mouse, there remain limitations. First, the proximity biotinylation occurs on lysine residues. Secreted proteins or peptide fragments lacking lysines may not be identified in the data. Second, we estimate this method has sensitivity of ∼100 pg/mL from blood plasma based on the identification of several very low abundance hormones and ectodomain shed receptors (e.g., hepatic LIFR ectodomain, ∼100 pg/mL). While this is comparable to that of other state-of-the-art technologies currently available for multiplexed plasma proteomics, there may be potentially important, very low abundance proteins that are not captured with this approach. Third, a cell’s secreted proteins may not enter the bloodstream but stay within the interstitial fluid to function locally as paracrine or autocrine signals. We have not yet benchmarked the performance of this methodology to identify these types of secretory proteins in mice. Finally, we use a three-day biotin supplementation protocol. While this labeling time can be reduced to one day or even several hours, rapid secretome changes that occur on the minutes timescale would not be detectable.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low biotinylation signal by Western blot (step 5)</p>
<h3 id="sec7.2">Potential solution</h3>
<p>There may be multiple causes of low biotinylation signal, including: (1) Poor viral transduction. Increase viral dose (e.g., 10<sup>12</sup>GC/mL) or transduction time (e.g., 2 months), or for other cell types beyond hepatocytes consider different AAV serotypes suitable for the cell type of interest; (2) poor biotin delivery. Increase biotin availability by combining both water supplementation and intraperitoneal injection as described above, or consider longer labeling times; (3) Low resolution of streptavidin antibody. Consider using a more sensitive detection method. We have observed the streptavidin Western blotting to be the least sensitive method and the shotgun proteomics to be the most sensitive method for detection of biotinylated secretomes, with silver staining of streptavidin-purified blood plasma of intermediate sensitivity.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>High variability in secretome biotinylation across mice (step 5)</p>
<h3 id="sec7.4">Potential solution</h3>
<p>We have found that the differences in labeling efficiency between mice often results from variable transduction efficiencies and TurboID expression across animals. We recommend preparing extra animals for each experiment and choose animals that have similar TurboID expression for final analysis.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Low yield of biotinylated proteins after streptavidin enrichment (step 10)</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Non-specific binding of plasma proteins onto streptavidin beads can sometimes impede the enrichment efficiency. Increasing the number of wash steps or the wash stringency to reduce background binding. We have also observed batch effect in streptavidin beads; consider using a new batch of streptavidin beads when strong non-specific binding is observed.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Low intensity of the mass spectrometry signal (steps 15 and 16)</p>
<h3 id="sec7.8">Potential solution</h3>
<p>We recommend semi-quantifying the eluted peptides after streptavidin enrichment by running a silver stain and comparing with a known amount of BSA standard. Also, we recommend semi-quantifying peptide amounts following S-trap digestion using a NanoDrop spectrophotometer (Thermo Fisher) or colorimetric or fluorometric peptide assays (Pierce). Inject ∼1 ug peptides, or all available material if total amount is less, for a single LC-MS/MS acquisition.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Enrichment of proteins in wild-type C57BL/6J mice (Proteomic data analysis step 3)</p>
<h3 id="sec7.10">Potential solution</h3>
<p>We have found that sometimes a subset of proteins are enriched in wild-type C57BL/6J mice. This may be due to leaky expression of ER-TurboID in the liver or plasma protein contamination during proteomic sample preparation. We recommend removing these proteins from the list of cell type-specific secreted proteins.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Branon, T.C., Bosch, J.A., Sanchez, A.D., Udeshi, N.D., Svinkina, T., Carr, S.A., Feldman, J.L., Perrimon, N., and Ting, A.Y. (2018). Efficient proximity labeling in living cells and organisms with TurboID. Nat. Biotechnol. <i>36</i>, 880-887. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00720-6/sref1">View at publisher</a></p>
<p id="bib2">JoVE Science Education Database; Lab Animal Research (2021). Blood Withdrawal II. JoVE. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00720-6/sref2">View at publisher</a></p>
<p id="bib3">Long, C., Amoasii, L., Mireault, A.A., McAnally, J.R., Li, H., Sanchez-Ortiz, E., Bhattacharyya, S., Shelton, J.M., Bassel-Duby, R., and Olson, E.N. (2016). Postnatal genome editing partially restores dystrophin expression in a mouse model of muscular dystrophy. Science <i>351</i>, 400-403. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00720-6/sref3">View at publisher</a></p>
<p id="bib4">Tyanova, S., Temu, T., and Cox, J. (2016a). The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. Nat. Protoc. <i>11</i>, 2301-2319. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00720-6/sref4">View at publisher</a></p>
<p id="bib5">Tyanova, S., Temu, T., Sinitcyn, P., Carlson, A., Hein, M.Y., Geiger, T., Mann, M., and Cox, J. (2016b). The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods <i>13</i>, 731-740. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00720-6/sref5">View at publisher</a></p>
<p id="bib6">Wei, W., Riley, N.M., Yang, A.C., Kim, J.T., Terrell, S.M., Li, V.L., Garcia-Contreras, M., Bertozzi, C.R., and Long, J.Z. (2021). Cell type-selective secretome profiling in vivo. Nat. Chem. Biol. <i>17</i>, 326-334. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00720-6/sref6">View at publisher</a></p>
<p id="bib7">Zeng, X., Ye, M., Resch, J.M., Jedrychowski, M.P., Hu, B., Lowell, B.B., Ginty, D.D., and Spiegelman, B.M. (2019). Innervation of thermogenic adipose tissue via a calsyntenin 3beta-S100b axis. Nature <i>569</i>, 229-235. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00720-6/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jonathan Z. Long (<a href="mailto:jzlong@stanford.edu">jzlong@stanford.edu</a>)</p>
<h4>Materials availability</h4>
<p>Plasmids generated in this study have been deposited to Addgene, pAAV-FLEX-ER-TurboID, 160857. All unique/stable reagents generated in this study are available from the lead contact with a completed Materials Transfer Agreement.</p>
<h4>Data and code availability</h4>
<p>All proteomic data generated here are publicly available on ProteomeXchange under project accession no. PXD021602. Processed proteomic datasets are provided in <a href="#bib6">Wei et al. (2021)</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank members of the Long and Bertozzi labs for helpful discussions. We gratefully acknowledge the staff at the Penn Vector Core for the production of AAVs. This work was supported by the <a href="https://doi.org/10.13039/100000098">US National Institutes of Health</a> (DK124265 and DK130641 to J.Z.L. and K00CA21245403 to N.M.R.) and the <a href="https://doi.org/10.13039/100014499">Stanford Diabetes Research Center</a> (P30DK116074).</p>
<h3>Author contributions</h3>
<p>L.X.C. and W.W. prepared the samples and performed the experiments; W.W. and N.M.R. analyzed the data; W.W, N.M.R., and J.Z.L. wrote the manuscript; and J.Z.L. and C.R.B. supervised the project.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>