<article>
<section>
<h1 id="header">Integrated functional neuronal network analysis of 3D silk-collagen scaffold-based mouse cortical culture</h1>
<p><time datetime="2021-01-29">Published: January 29, 2021</time></p>
<p>Yu-Ting L. Dingle,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Mattia Bonzanni,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Volha Liaudanskaya,<sup><a href="#aff1">1</a></sup> Thomas J.F. Nieland,<sup><a href="#aff1">1</a></sup> and David L. Kaplan<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biomedical Engineering, Tufts University, Medford, MA 02155, USA</p>
<p id="fn1"><sup>2</sup>These authors contributed equally</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yu-ting.dingle@tufts.edu">yu-ting.dingle@tufts.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:david.kaplan@tufts.edu">david.kaplan@tufts.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100292">10.1016/j.xpro.2020.100292</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Bioengineered 3D tunable neuronal constructs are a versatile platform for studying neuronal network functions, offering numerous advantages over existing technologies and providing for the discovery of new biological insights. Functional neural networks can be evaluated using calcium imaging and quantitatively described using network science. This protocol includes instructions for fabricating protein-based composite scaffolds, 3D <i>in vitro</i> culture of embryonic mouse cortical neurons, virally induced expression of GCaMP6f, wide-field calcium imaging, and computational analysis with open-source software and custom MATLAB code.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib6">Dingle et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Customizable 3D scaffold system to recapitulate <i>in vitro</i> neuronal networks</li>
<li>Optical mesoscale detection of neuronal activity</li>
<li>Structural, molecular, and functional characterization of the 3D <i>in vitro</i> system</li>
<li>Network science approach reveals millimeter-scale topologies</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Extraction of silk fibroin fibers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>Silk fibroin is a natural polymer and a suitable material for both <i>in vivo</i> and <i>in vitro</i> biomedical applications because of its biocompatibility, tunable chemical and mechanical properties, and long-term stability <i>in vitro</i> yet with 100% degradability over time <i>in vivo</i> (<a href="#bib16">Rockwood et al., 2011</a>; <a href="#bib17">Rouleau et al., 2020</a>). We have previously reported several <i>in vitro</i> brain models, with both primary rodent neurons and human induced pluripotent stem cells (iPSCs), using porous scaffolds fabricated with silk fibroin and extracellular matrix composites. Such models recapitulated many key features of the brain tissue (<a href="#bib3">Cairns et al., 2020</a>; <a href="#bib4">Cantley et al., 2018</a>; <a href="#bib6">Dingle et al., 2020</a>; <a href="#bib10">Liaudanskaya et al., 2020</a>; <a href="#bib17">Rouleau et al., 2020</a>; <a href="#bib18">Tang-Schomer et al., 2014</a>).</p>
<p>Native <i>Bombyx mori</i> (silkworm) silk is composed of the core silk fibroin protein and the adhesive sericin proteins. This section describes the step-by-step instructions on the removal (degumming) of sericin and the extraction of fibroin fibers. See <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Preparation of silk fibroin solution</div>
<p>See also <a href="#mmc1">Methods Videos S1</a>, <a href="#mmc2">S2</a>, <a href="#mmc3">S3</a>, <a href="#mmc4">S4</a>, and <a href="#mmc5">S5</a>.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Each batch of 6.25 g cut cocoons yield approximately 1.5 pieces of 10 cm diameter × 3–4 mm height silk sponge scaffold.</div>
<ol>
<li>Cut <i>Bombyx mori</i> cocoons open. Dispose pupa and cut cocoons into small pieces that are no larger than 1.5 cm in any dimension. Remove any inner layers that are stained brown from the silkworm pupa. See <a href="#mmc1">Methods Video S1</a>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Approximately 15 whole cocoons are needed for a batch of 6.25 g cut cocoons.</div>
<ol start="2">
<li>Weigh 6.25 g of cut cocoon pieces.</li>
<li>Boil 2.5 L research grade (RG) or deionized (DI) water in a 3 L metal beaker on a hot plate. Boil one extra beaker of RG or DI water for refilling evaporative water loss during boiling.</li>
<li>Add 5.3 g of Na<sub>2</sub>CO<sub>3</sub> slowly to the boiling water (final concentration = 0.02 M).</li>
<li>Boil cocoon pieces for 30 min to degum the cocoons, which removes the sericin component from the silk fibroin fibers. Use serological pipettes to pull the fibers apart every 3–5 min while boiling. See <a href="#mmc2">Methods Video S2</a>.</li>
</ol>
<figure id="mmc1"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc1.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S1. Bombyx mori cocoons are cut open and pupas are removed (extraction of silk fibroin fibers step 1)</div>
<p>See also Figure 1.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Glass beakers are used in the video for better visualization.</div>
<ol start="6">
<li>Transfer degummed silk fibroin fibers to 4 L of cold RG or DI water in a plastic beaker. See <a href="#mmc3">Methods Video S3</a>.</li>
</ol>
<figure id="mmc2"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc2.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S2. Degumming cocoons by boiling</div>
<p>Cut cocoon pieces are placed in boiling 0.02 M Na2CO3 solution. Serological pipettes are used to periodically submerge and pull apart silk fibroin fibers. (Extraction of Silk Fibroin Fibers step 5). See also Figure 1.</p>
</figcaption>
</figure>
<ol start="7">
<li>Wash by manually pulling the fibers apart in the water followed by squeezing to dry. See <a href="#mmc4">Methods Video S4</a>. Repeat this process 3–5 times.</li>
</ol>
<figure id="mmc3"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc3.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S3. Cooling silk fibroin fibers</div>
<p>Degummed silk fibroin fibers are transferred to cold water at the end of 30-min boiling (Extraction of Silk Fibroin Fibers step 6). See also Figure 1.</p>
</figcaption>
</figure>
<ol start="8">
<li>Place the squeezed-dry silk fibroin fibers on aluminum foil and allow to dry completely at 20°C–25°C at places with high air flow, such as the fume hood.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A minimum of 12 h in the fume hood is usually sufficient.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Dry silk fibers should feel stiff and crisp. If the silk fibers feel soft, continue drying process. Fibers that contain water will affect the solubilization in the subsequent step.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dry silk fibroin fibers can be stored at 20°C–25°C for at least a year.</div>
<figure id="mmc4"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc4.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S4. Washing silk fibroin fibers</div>
<p>Silk fibroin fibers are washed by manually pulling apart the fibers in water followed by squeezing to dry (Extraction of Silk Fibroin Fibers step 7). See also Figure 1.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Solubilization of silk fibroin</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<p>Silk fibroin is a natural polymer rich in hydrophobic β-sheet-forming blocks linked by small hydrophilic linkers (<a href="#bib16">Rockwood et al., 2011</a>). The degummed silk fibroin fibers extracted from the previous step retain their largely β-sheet secondary protein structure and thus remain water-insoluble. This section describes the use of lithium bromide (LiBr) to denature the β-sheets to create water-based fibroin solution. See <a href="#fig1">Figure 1</a>.</p>
<ol start="9">
<li>Weigh dry silk fibroin fibers.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Silk fibers must be completely dry.</div>
<div class="note">
<span class="note-title">Note:</span> 6.25 g cocoons typically yield around 4 g dry silk fibroin fibers.</div>
<ol start="10">
<li>Prepare 9.3 M LiBr solution.
<div class="critical">
<span class="critical-title">Critical:</span> Stock LiBr should be dry and not clumpy. Keep the LiBr container closed and sealed using parafilm as much as possible to prevent LiBr from absorbing moisture from room air.</div>
<ol type="a">
<li>Total LiBr solution (mL) needed = 4 × silk fibroin weight (g)</li>
<li>Amount of LiBr (g) = (86.85 g/mol) × (9.3 mol/L) × (vol of total solution in step 10a) × (1 L/1000 mL)</li>
<li>Starting volume of RG or DI water = 0.7 × volume from step 10a.</li>
<li>Add RG or DI water from step 10c to a beaker. Slowly add LiBr to dissolve.</li>
<li>Measure volume from step 10d with a graduated cylinder and bring to the final volume (calculated in step 10a) with RG or DI water.</li>
</ol>
</li>
</ol>
<ol start="11">
<li>Place silk fibroin fibers in a 100–200 mL glass beaker. Pour 9.3 M LiBr solution onto the silk fibroin fibers. Use a serological pipette to move fibers around to maximize exposure to LiBr solution. See <a href="#mmc5">Methods Video S5</a>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is normal that the volume of LiBr solution appears very low in comparison to the amount of fibers.</div>
<ol start="12">
<li>Cover the beaker with aluminum foil and place in a 60°C oven for 4 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#troubleshooting">Troubleshooting 1</a>.</div>
<ol start="13">
<li>Prepare 4 L RG or DI water in a plastic beaker. Cut 20 cm of dialysis tubing (MWCO = 3,500 Da) and wet with RG or DI water. Tie and clip one end.</li>
<li>Transfer dissolved silk fibroin into dialysis tubing using a serological pipette.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Dissolved silk fibroin solution should appear slightly yellow in color and transparent. It should be viscous but not too viscous to pipette.</div>
<ol start="15">
<li>Leave approximately an additional 1/3 of tubing length, while minimizing the amount of air inside the tubing, for volume expansion of the silk fibroin solution during dialysis, as water will cross the dialysis membrane into the tubing. Remove air by squeezing and tie the open end. Clip both ends and secure to a float buoy with rubber bands.</li>
<li>Dialyze to remove LiBr by changing 4 L RG or DI water six times in the next 48 h on a stir plate. First two changes should be 1–2 h apart. Remaining four changes should be at least 4 h apart. Check conductivity of the final water with a handheld conductivity meter (see <a href="#key-resources-table">Key resources table</a>), which should match fresh RG or DI water.</li>
<li>Collect silk fibroin solution in 50 mL conical tubes (does not need to be sterile).</li>
<li>Centrifuge silk fibroin solution at 4°C at 12,700 × <i>g</i> for 20 min. Pour silk fibroin solution into new conical tubes. Repeat centrifugation and pass silk fibroin solution through a 100-μm strainer into new conical tubes.</li>
<li>Measure concentration by drying 500 μL of silk fibroin solution in a weigh boat (smaller boats with diameter &lt; 4 cm typically work better) in a 60°C oven for a minimum of 8 h.
<ol type="a">
<li>Silk fibroin concentration % (w/v) = 100 × dry weight of silk fibroin (g) ÷ 0.5 mL.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the silk concentration is &lt;6% (w/v), place silk fibroin solution back in new dialysis tubing and hang it in a fume hood for 8–12 h while exposed to air to allow evaporation.</div>
<ol start="20">
<li>Dilute silk fibroin solution with RG or DI water to 6% (w/v).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> This solution can be used immediately or stored at 4°C for up to 2 weeks.</div>
<div class="note">
<span class="note-title">Note:</span> See <a href="#troubleshooting">Troubleshooting 2</a>.</div>
<figure id="mmc5"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc5.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S5. Dissolving silk fibroin fibers</div>
<p>Dry silk fibroin fibers are placed in a glass beaker and LiBr solution (9.3 M) is poured onto the fibers. The fibers are dispersed in the solution using a serological pipette to maximize exposure to LiBr (Solubilization of Silk Fibroin step 11). See also Figure 1.</p>
</figcaption>
</figure>
<h3 id="sec1.3">Preparation of silk scaffolds</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<p>This section provides step-by-step instructions for fabricating porous silk fibroin scaffolds with pore sizes of 425–500 μm. These scaffolds will be used for 3D cortical culture. We had previously determined that 6% (w/v, in water) silk fibroin solution resulted in scaffolds with desirable mechanical properties for neuronal culture (<a href="#bib18">Tang-Schomer et al., 2014</a>). To initiate the self-assembly of aqueous silk fibroin into a scaffold composed of insoluble β-sheet crystalline structure, sodium chloride (NaCl) particles are incorporated in the silk solution, incubated, and later washed out to create pores. The resulting sponge material is manually trimmed to the final dimensions. See <a href="#fig2">Figure 2</a>.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Preparation of silk scaffolds</div>
<p>See also <a href="#mmc6">Methods Videos S6</a>, <a href="#mmc7">S7</a>, <a href="#mmc8">S8</a>, and <a href="#mmc9">S9</a>.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Each 10 cm diameter × 3–4 mm height sponge requires 30 mL 6% (w/v) silk fibroin solution and 60 g sieved NaCl and yields at least 300 3-mm diameter scaffolds.</div>
<ol start="21">
<li>Stack stainless steel sieves in the top to bottom order of lid, 500-μm pore size sieve, 425-μm pore size sieve, and receiver tray (see <a href="#key-resources-table">Key resources table</a>). Pour a small amount of NaCl at a time and sieve NaCl particles to size range of 425–500 μm. Shake until very little NaCl passes through the bottom 425-μm sieve. See <a href="#mmc6">Methods Video S6</a>.</li>
</ol>
<ol start="22">
<li>Prepare 60 g sieved NaCl particles in a large weigh boat (e.g., Fisherbrand 02-202-102, 110-mm diameter).</li>
<li>In a 10 cm plastic petri dish, first add 30 mL 6% (w/v) silk solution, and then slowly and evenly pour 60 g NaCl from the large weigh boat into silk solution. See <a href="#mmc7">Methods Video S7</a>.</li>
</ol>
<figure id="mmc6"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc6.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S6. Sieving salt particles</div>
<p>Sieves are stacked in the following order (top to bottom): lid, 500-μm sieve, 425-μm sieve, and receiver tray. A small amount of sodium chloride (NaCl) is poured on the 500-μm sieve. The stack is manually shaken to separate NaCl particle sizes of &gt;500 μm (in the top 500-μm sieve). 425–500 μm (in the middle 425-μm sieve), and &lt;425 μm (in the bottom receiver tray) (Preparation of Silk Scaffolds step 21). See also Figure 2.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Achieving homogenous sponge formation, even distribution of NaCl particles is important. Slowly pour NaCl in a back and forth motion to add thin layers and rotate the dish after each layer.</div>
<ol start="24">
<li>Place the cover on the petri dish and place in a fume hood for 2 days, followed by placing in 60°C oven for 1 h. This allows silk fibroin to form water-insoluble β-sheet crystalline structure.</li>
<li>Fill a 4 L beaker with RG or DI water and rinse the cured sponge. Carefully break the edge of the plastic petri dish and slide a microspatula underneath the sponge to release the sponge from the dish. Soak and gently squeeze the sponge with DI water multiple times until the sponge is soft. See <a href="#mmc8">Methods Video S8</a>.</li>
</ol>
<figure id="mmc7"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc7.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S7. Fabricating porous silk scaffolds</div>
<p>In a 10-cm dish, first add 30 mL 6% (w/v) silk fibroin solution. Then add 60 g of NaCl particles from a large weigh boat by slowly pouring the salt into the solution in a back and forth motion to add thin layers. The dish is rotated after each layer (Preparation of Silk Scaffolds step 23). See also Figure 2.</p>
</figcaption>
</figure>
<ol start="26">
<li>Place the sponge in a clean petri dish lid and secure it to a float buoy using rubber bands. Wash the sponge in 4 L RG or DI water on a stir plate and change water every few hours over 1–2 days. Check conductivity of the final water, which should match fresh RG or DI water.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The sponge can be stored in water in a petri dish, parafilm-sealed, at 4°C for up to 3 months.</div>
<div class="note">
<span class="note-title">Note:</span> See <a href="#troubleshooting">Troubleshooting 3</a>.</div>
<ol start="27">
<li>Punch out 3-mm diameter cylindrical scaffolds with a biopsy punch. See <a href="#mmc9">Methods Video S9</a>.</li>
</ol>
<figure id="mmc8"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc8.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S8. Releasing and leaching cured silk fibroin sponge from the 10-cm dish</div>
<p>The dish is carefully broken manually, and the cured silk fibroin sponge (containing NaCl particles) is separated from the plastic with a microspatula. To first remove the majority of NaCl particles, the sponge is soaked and squeezed in 4 L water until it is soft (Preparation of Silk Scaffolds step 25). See also Figure 2.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Always keep sponges wet in RG or DI water as dry sponges are fragile. See <a href="#troubleshooting">Troubleshooting 4</a>.</div>
<ol start="28">
<li>Trim scaffold height to 1.5 mm with a razor blade. A useful method to keep height consistent is a customized, 3D printed scaffold trimmer. The SDL file of the scaffold trimmer can be downloaded from <a href="https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture">https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture</a>.</li>
<li>Place the scaffolds in an autoclavable container filled with RG or DI water. Autoclave-sterilize the scaffolds using a 120°C, 20-min liquid cycle.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Store the sterile scaffolds at 4°C for up to 3 months.</div>
<figure id="mmc9"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc9.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S9. Trimming scaffolds</div>
<p>The salt-leached sponge is cut into 3-mm diameter cylindrical scaffolds using a biopsy punch (Preparation of Silk Scaffolds step 27). See also Figure 2.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>AAV-hSyn1-GCaMP6f-P2A-nls-dTomato</td>
<td>Addgene</td>
<td>51085-AAV1<a href="https://www.addgene.org/51085/">https://www.addgene.org/51085/</a>
</td>
</tr>
<tr>
<td>pENN.AAV.hSyn.TurboRFP.WPRE.RBG</td>
<td>Addgene</td>
<td>105552-AAV1<a href="https://www.addgene.org/105552/">https://www.addgene.org/105552/</a>
</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Sodium carbonate (Na<sub>2</sub>CO<sub>3</sub>)</td>
<td>MilliporeSigma</td>
<td>57795</td>
</tr>
<tr>
<td>Lithium bromide (LiBr)</td>
<td>MilliporeSigma</td>
<td>213225</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>MilliporeSigma</td>
<td>S7653</td>
</tr>
<tr>
<td>Poly-D-lysine (PDL)</td>
<td>MilliporeSigma</td>
<td>P6407</td>
</tr>
<tr>
<td>Laminin (0.5 mg/mL)</td>
<td>MilliporeSigma</td>
<td>MFCD00081739</td>
</tr>
<tr>
<td>Collagen type I solution</td>
<td>Corning</td>
<td>354236</td>
</tr>
<tr>
<td>Sodium hydroxide (NaOH)</td>
<td>MilliporeSigma</td>
<td>S2770</td>
</tr>
<tr>
<td>Calcein-AM</td>
<td>Thermo Fisher</td>
<td>C3100MP</td>
</tr>
<tr>
<td>Picrotoxin</td>
<td>Abcam</td>
<td>ab120315</td>
</tr>
<tr>
<td>Bicuculline</td>
<td>Abcam</td>
<td>ab120107</td>
</tr>
<tr>
<td>2,3-Dioxo-6-nitro-7-sulfamoyl-benzo[f]quinoxaline (NBQX)</td>
<td>Tocris</td>
<td>1044</td>
</tr>
<tr>
<td>(2R)-Amino-5-phosphonopentanoate (AP5)</td>
<td>MilliporeSigma</td>
<td>A8054</td>
</tr>
<tr>
<td>Tetrodotoxin citrate</td>
<td>Abcam</td>
<td>ab120055</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>MilliporeSigma</td>
<td>M3148</td>
</tr>
<tr>
<td>Dimethyl sulfoxide</td>
<td>MilliporeSigma</td>
<td>D2650</td>
</tr>
<tr>
<td>0.25% Trypsin-EDTA</td>
<td>Thermo Fisher</td>
<td>25200</td>
</tr>
<tr>
<td>DNase</td>
<td>MilliporeSigma</td>
<td>10104159001</td>
</tr>
<tr>
<td>Hank's balanced salt solution (HBSS)</td>
<td>Thermo Fisher</td>
<td>14025</td>
</tr>
<tr>
<td>Phosphate-buffered saline (PBS), sterile</td>
<td>Thermo Fisher</td>
<td>10010</td>
</tr>
<tr>
<td>Neurobasal medium minus phenol red</td>
<td>Thermo Fisher</td>
<td>12348</td>
</tr>
<tr>
<td>B27 supplement</td>
<td>Thermo Fisher</td>
<td>15240</td>
</tr>
<tr>
<td>Antibiotic-antimycotic</td>
<td>Thermo Fisher</td>
<td>15240</td>
</tr>
<tr>
<td>GlutaMax</td>
<td>Thermo Fisher</td>
<td>35050</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Thermo Fisher</td>
<td>10437</td>
</tr>
<tr>
<td>10× PBS</td>
<td>Fisher Scientific</td>
<td>BP399-4</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Gabra1 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00439046_m1</td>
</tr>
<tr>
<td>Gad2 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00484623_m1</td>
</tr>
<tr>
<td>Gria1 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00433753_m1</td>
</tr>
<tr>
<td>Grin1 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00433790_m1</td>
</tr>
<tr>
<td>Pax6 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00443081_m1</td>
</tr>
<tr>
<td>Rn18s (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm03928990_g1</td>
</tr>
<tr>
<td>Shank3 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00498775_m1</td>
</tr>
<tr>
<td>Slc17a7 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00812886_m1</td>
</tr>
<tr>
<td>Slc1a3 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00600697_m1</td>
</tr>
<tr>
<td>Syn1 (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00449772_m1</td>
</tr>
<tr>
<td>Dcx (Taqman Assay)</td>
<td>Thermo Fisher</td>
<td>Mm00438400_m1</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FluoroSNNAP</td>
<td><a href="https://www.seas.upenn.edu/%7Emolneuro/software.html">https://www.seas.upenn.edu/∼molneuro/software.html</a></td>
<td>n/a</td>
</tr>
<tr>
<td>Photoshop</td>
<td>Adobe</td>
<td>n/a</td>
</tr>
<tr>
<td>Illustrator</td>
<td>Adobe</td>
<td>n/a</td>
</tr>
<tr>
<td>NIS-Elements AR</td>
<td>Nikon</td>
<td>n/a</td>
</tr>
<tr>
<td>MATLAB</td>
<td>MathWorks</td>
<td>n/a</td>
</tr>
<tr>
<td>MATLAB Bioinformatics Toolbox Add-on</td>
<td><a href="https://www.mathworks.com/products/bioinfo.html">https://www.mathworks.com/products/bioinfo.html</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>RunNetworkAnalysis MATLAB Code</td>
<td><a href="https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture">https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Weigh dishes (small, medium, large)</td>
<td>Fisher Scientific</td>
<td>02-202-100, 02-202-101, 02-202-102</td>
</tr>
<tr>
<td>Tissue culture dishes (35-mm, 60-mm, 100-mm)</td>
<td>Corning</td>
<td>430165, 430166, 430167</td>
</tr>
<tr>
<td>Syringe filters (0.22-μm, sterile)</td>
<td>MilliporeSigma</td>
<td>SLGV033RB</td>
</tr>
<tr>
<td>Syringes (5 mL)</td>
<td>Fisher Scientific</td>
<td>14-817-53</td>
</tr>
<tr>
<td>Cell strainers, 100-μm</td>
<td>Corning</td>
<td>431752</td>
</tr>
<tr>
<td>
<i>Bombyx mori</i> cocoons</td>
<td>Tajima Shoji Co., Ltd, Japan</td>
<td>n/a</td>
</tr>
<tr>
<td>Dialysis tubing, molecular weight cutoff 3,500 Da</td>
<td>Fisher Scientific</td>
<td>21-152-9</td>
</tr>
<tr>
<td>Dialysis clips</td>
<td>Thermo Fisher</td>
<td>68011</td>
</tr>
<tr>
<td>Stainless steel spatula</td>
<td>Fisher Scientific</td>
<td>21-401-10</td>
</tr>
<tr>
<td>Biopsy punch, 3-mm diameter</td>
<td>Integra</td>
<td>12-460-406</td>
</tr>
<tr>
<td>Razor blades</td>
<td>Fisher Scientific</td>
<td>12-640</td>
</tr>
<tr>
<td>Scalpel blades</td>
<td>Fisher Scientific</td>
<td>NC9855170</td>
</tr>
<tr>
<td>Surgical scissors</td>
<td>Fine Science Tools</td>
<td>14060-09</td>
</tr>
<tr>
<td>Tissue forceps</td>
<td>Fine Science Tools</td>
<td>11052-10</td>
</tr>
<tr>
<td>Tissue culture plates (12-, 24-, 48-, 96-wells)</td>
<td>Fisher Scientific</td>
<td>08-772-29, 08-772-1, 08-772-1C, 08-7722c</td>
</tr>
<tr>
<td>Conical tubes (15 mL, 50 mL)</td>
<td>Fisher Scientific</td>
<td>14-959-53a, 14-432-22</td>
</tr>
<tr>
<td>Tissue culture plates, glass-bottom, 24-well</td>
<td>Cellvis</td>
<td>P24-1.5P</td>
</tr>
<tr>
<td>Nuclease-free 1.5 mL microcentrifuge tubes</td>
<td>Fisher Scientific</td>
<td>05-408-129</td>
</tr>
<tr>
<td>Homogenizer pestle</td>
<td>Fisher Scientific</td>
<td>12-141-364</td>
</tr>
<tr>
<td>Motorized tissue grinder</td>
<td>Fisher Scientific</td>
<td>12-1413-61</td>
</tr>
<tr>
<td>QIAshredder column</td>
<td>Qiagen</td>
<td>79656</td>
</tr>
<tr>
<td>Nuclease-free 0.2 mL PCR tubes or strips</td>
<td>Costar</td>
<td>07-200-681</td>
</tr>
<tr>
<td>Dumont #5 forceps</td>
<td>Fine Science Tools</td>
<td>11251-35 or11252-50</td>
</tr>
<tr>
<td>Angled forceps</td>
<td>Fine Science Tools</td>
<td>11251-35</td>
</tr>
<tr>
<td>3 L stainless steel beakers</td>
<td>Polar Ware</td>
<td>3W</td>
</tr>
<tr>
<td>4 L plastic beakers</td>
<td>Thermo Fisher</td>
<td>12014000</td>
</tr>
<tr>
<td>Float Buoys</td>
<td>Thermo Fisher</td>
<td>66432</td>
</tr>
<tr>
<td>Conductivity meter</td>
<td>Fisher Scientific</td>
<td>15-078-201</td>
</tr>
<tr>
<td>Stainless steel sieves (425 μm, 500 μm, lid/receiver)</td>
<td>Fisher Scientific</td>
<td>04-884-1AP, 04-884-1AN, 04-888A</td>
</tr>
<tr>
<td>Sterile, nuclease-free water</td>
<td>Thermo Fisher</td>
<td>10977015</td>
</tr>
<tr>
<td>Glass beaker (small)</td>
<td>Fisher Scientific</td>
<td>FB100100</td>
</tr>
<tr>
<td>Glass graduated cylinder</td>
<td>Fisher Scientific</td>
<td>08-550D</td>
</tr>
<tr>
<td>Dissection microscope</td>
<td>Nikon or equivalent</td>
<td>SMZ-754T or equivalent</td>
</tr>
<tr>
<td>Epifluorescence microscope equipped with a 4× objective, green fluorescence filter cube, and time-lapse acquisition function</td>
<td>Nikon or equivalent</td>
<td>Nikon Eclipse Ti-2 withObjective: CFI Plan Fluor DL 4× na 0.13 wd 16.5 mm Objective, filter #49002, and NIS-Elements AR software, or equivalent</td>
</tr>
<tr>
<td>Microscope stage-top incubator with temperature and CO<sub>2</sub> controls</td>
<td>Bioscience Tools or equivalent</td>
<td>TC-MWP or equivalent</td>
</tr>
<tr>
<td>Confocal microscope with excitation laser and detector for red fluorescence</td>
<td>Leica or equivalent</td>
<td>SP8 or equivalent</td>
</tr>
<tr>
<td>NanoDrop UV-vis spectrophotometer</td>
<td>Thermo Fisher</td>
<td>2000 or equivalent</td>
</tr>
<tr>
<td>Thermocycler</td>
<td>Bio-Rad or equivalent</td>
<td>T100 or equivalent</td>
</tr>
<tr>
<td>qRT-PCR machine</td>
<td>Thermo Fisher</td>
<td>QuantStudio 5 or equivalent</td>
</tr>
<tr>
<td>3D Printer</td>
<td>Formlabs or equivalent</td>
<td>Form2 or equivalent<b>Alternative:</b> Order 3D printed parts from <a href="http://ShapeWays.com">ShapeWays.com</a>
</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Qiagen RNeasy Mini Kit</td>
<td>Qiagen</td>
<td>79656</td>
</tr>
<tr>
<td>iScript gDNA Clear cDNA Synthesis Kit</td>
<td>Bio-Rad</td>
<td>1725035</td>
</tr>
<tr>
<td>Taqman Gene Expression Master Mix</td>
<td>Thermo Fisher</td>
<td>4369016</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Timed-pregnant (E16.5) C57BL/6 mouse</td>
<td>Charles River Laboratories</td>
<td>027</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Stock solutions and aliquots preparation</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock solution concentration</th>
<th>Preparation procedures</th>
</tr>
</thead>
<tbody>
<tr>
<td>Poly-D-lysine (PDL)</td>
<td>0.1 mg/mL</td>
<td>Dissolve 5 mg stock powder in 5 mL sterile water. Warm up the solution in 37°C bath, vortex, filter with a 0.22 μm syringe filter, and dilute with sterile water to a final concentration of 0.1 mg/mL. Store at 4°C for up to 1 month</td>
</tr>
<tr>
<td>DNase</td>
<td>10 mg/mL</td>
<td>Dissolve in water. Prepare 1 mL aliquots and store at −20°C for up to 1 year</td>
</tr>
<tr>
<td>Picrotoxin</td>
<td>100 mM</td>
<td>Add DMSO directly to stock vial. Prepare 4 μL aliquots and store at −20°C for up to 1 month</td>
</tr>
<tr>
<td>Bicuculline</td>
<td>20 mM</td>
<td>Add DMSO directly to stock vial. Prepare 4 μL aliquots and store at −20°C for up to 1 month</td>
</tr>
<tr>
<td>NBQX</td>
<td>10 mM</td>
<td>Add DMSO directly to stock vial. Prepare 4 μL aliquots and store at −20°C for up to 1 month</td>
</tr>
<tr>
<td>AP5</td>
<td>5 mM</td>
<td>Add water to stock vial. Transfer solution to conical tube and store at 4°C for up to 6 months</td>
</tr>
<tr>
<td>Tetrodotoxin</td>
<td>2 mM</td>
<td>Add water directly to stock vial. Prepare 20 μL aliquots and store at −20°C for up to 1 month</td>
</tr>
<tr>
<td>Laminin</td>
<td>(0.5 mg/mL)</td>
<td>Thaw stock vial on ice. Prepare 200–300 μL aliquots on ice and store at −20°C for up to 3 months</td>
</tr>
<tr>
<td>Adeno-associated virus (AAV)</td>
<td>N/A</td>
<td>Record titer information. Thaw stock vial on ice. Prepare 10–15 μL aliquots on ice and store at −80°C for up to 1 year</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Picrotoxin, bicuculline, NBQX, AP5, and TTX are neurotoxins. Prepare stock solutions in chemical hood with proper PPE and safety practice. Add solvent directly to the stock vials.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare PDL, DNase, Laminin, and AAV with sterile techniques. Use standard biosafety practice when handling AAV.</div>
<table id="undtbl2">
<thead>
<tr>
<th>PDL/Laminin coating solution</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PDL (0.1 mg/mL)</td>
<td>0.1 mg/mL</td>
<td>10 mL</td>
</tr>
<tr>
<td>Laminin stock (0.5 mg/mL)</td>
<td>10 μg/mL</td>
<td>200 μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Thaw laminin stock on ice or at 4°C. Laminin should never be thawed at 20°C–25°C or at 37°C. Make PDL/laminin coating solution fresh on the day of coating.</div>
<table id="undtbl3">
<thead>
<tr>
<th>Neuro medium</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neurobasal medium minus phenol red</td>
<td>n/a</td>
<td>480 mL</td>
</tr>
<tr>
<td>B27 supplement</td>
<td>2%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Antibiotic-antimycotic</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>GlutaMax</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 4°C for up to 1 month.</div>
<table id="undtbl4">
<thead>
<tr>
<th>Neuro medium Plus 5% FBS</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neuro medium</td>
<td>n/a</td>
<td>38 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>5%</td>
<td>2 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 4°C for up to 2 weeks.</div>
<table id="undtbl5">
<thead>
<tr>
<th>Neuro medium Plus 10% FBS</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neuro medium</td>
<td>n/a</td>
<td>36 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 4°C for up to 1 month.</div>
<table id="undtbl6">
<thead>
<tr>
<th>Trypsin/DNase</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.25% Trypsin-EDTA</td>
<td>n/a</td>
<td>12 mL</td>
</tr>
<tr>
<td>DNase</td>
<td>0.3 mg/mL</td>
<td>360 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>12 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare 2 mL aliquots and store at −20°C for up to 6 months.</div>
</section>
<section>
<h2 id="step-by-step-methods">Step-by-step methods</h2>
<h3 id="sec3.1">Scaffold coating</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h each on day −1 and day 0</div>
<p>This step coats the silk scaffolds with PDL and laminin, which is essential for cell adhesion to the silk scaffolds. This procedure should be done in a biosafety hood under sterile conditions. See <a href="#fig3">Figure 3</a>.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. 3D GCaMP<sup>+</sup> neural culture setup</div>
</figcaption>
</figure>
<ol>
<li>Prepare fresh PDL/Laminin Coating Solution (See <a href="#materials-and-equipment">Materials and equipment</a>).</li>
<li>Place up to 30 scaffolds per well in a 12-well plate and add 2 mL PDL/laminin coating solution per well.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> See additional steps in Quality Control #1 - 2D Culture (steps 84–89).</div>
<ol start="3">
<li>Incubate at 37°C for 12–24 h.</li>
<li>On the following day, aspirate the PDL/laminin coating solution from the scaffolds. Wash the scaffolds with PBS (4 mL/well) twice with 10 min incubation. Soak once with Neuro Medium at 37°C for at least 1 h, and soak with Neuro Medium Plus 5% FBS at 37°C for at least 1 h until cells are ready for seeding.</li>
</ol>
<h3 id="sec3.2">Cortical neuron isolation and cell seeding</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–6 h on day 0</div>
<p>This section describes the isolation of primary neural cells from mouse cerebral cortices and the subsequent seeding of cells onto 3D silk scaffolds. See <a href="#fig3">Figure 3</a>.</p>
<div class="note">
<span class="note-title">Note:</span> Proper institutional approval per animal protocols must be attained prior to embarking on this method.</div>
<div class="note">
<span class="note-title">Note:</span> Animal dissection should be done in a biosafety hood under sterile conditions. If the lab is not equipped with a dissection microscope inside a biosafety hood, it is acceptable to perform dissection on the bench. Tissue dissociation and cell culture should be done in a biosafety hood under sterile conditions.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Refer to “Quality control assay #1, steps 84–89” and “Quality control assay #3, steps 94–97” to examine the viability of the neurons.</div>
<ol start="5">
<li>Chill 150–200 mL HBSS on ice.</li>
<li>Fill three 10 cm dishes with ice-cold HBSS. Fill several 60 mm dishes with ice-cold HBSS and keep on ice.</li>
<li>Sacrifice timed-pregnant (E16.5) mouse. Shave abdomen and clean with 70% ethanol. Hold abdomen with tissue forceps and cut open skin with surgical scissors.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To avoid contamination from the skin, avoid reusing these tools.</div>
<ol start="8">
<li>With separate scissors and forceps, remove and place the uterus in the first 10 cm dish of ice-cold HBSS to remove excess blood.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Blood will adversely affect the health of the cells.</div>
<ol start="9">
<li>Transfer the uterus to the second dish of HBSS, extract embryos, and place embryos in the third dish of HBSS.</li>
<li>Decapitate the embryos with scissors and place the heads in the third 60 mm dish of HBSS on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep tissues on ice as much as possible to maintain viability.</div>
<ol start="11">
<li>Under a dissection microscope, stabilize the head by inserting angled forceps into the eye sockets and hold down. Gently make a midline cut with a scalpel blade and peel back the skin and skull with Dumont #5 forceps. Carefully remove the brain with a spatula and place the brain in ice-cold HBSS in a 60 mm dish.</li>
<li>Separate the cerebellum and the hemispheres with forceps.</li>
<li>Peel and remove the meninges with forceps.</li>
<li>With the medial side facing up, remove the midbrain and separate the striatum and cortex with forceps. Collect cortices in a 50 mL conical tube with HBSS on ice.</li>
<li>Remove HBSS from the conical tube containing cortices. Rinse the cortices with 30 mL PBS.</li>
<li>Carefully remove the PBS and add 2 mL of 0.25% Trypsin + 0.3 mg/mL DNase solution.</li>
<li>Transfer tissues/Trypsin/DNase to a 35-mm dish and incubate at 37°C for 20 min.</li>
<li>Add 2 mL Neuro Medium Plus 10% FBS to the tissue/Trypsin/DNase and transfer the 4 mL mixture to a 15 mL conical tube.</li>
<li>Add an additional 4 mL of Neuro Medium Plus 10% FBS. Triturate with a 5 mL pipette tip 20–25 times to mechanically dissociate the tissues.</li>
<li>Pass the solution through a 100-μm cell strainer into a new 50 mL conical tube. Rinse the strainer with an additional 4 mL Neuro Medium Plus 10% FBS.</li>
<li>Count the cells (recommended at 1:10 dilution) using Trypan Blue dye.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Expected viability &gt;80% and 6–10 × 10<sup>6</sup> live cells per pup.</div>
<ol start="22">
<li>Centrifuge at 210 × <i>g</i> for 5 min.</li>
<li>Remove supernatant.</li>
<li>The final seeding cell number and volume per scaffold is 2 × 10<sup>6</sup> cells in 5 μL (i.e., 400 × 10<sup>6</sup> cells/mL) of Neuro Medium Plus 5% PBS. Because the cell pellet accounts for a significant volume at this density, only add 3.5 μL per 2 × 10<sup>6</sup> cells of Neuro Medium Plus 5% PBS to resuspend the cell pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We have determined empirically that the cell pellet accounts for approximately 30% of the final volume.</div>
<ol start="25">
<li>Aspirate dry the PDL/laminin-coated scaffolds and transfer each scaffold to the center of a well in a 96-well plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not dry the scaffolds until the cell suspension is ready for seeding and it is recommended to dry no more than 30 scaffolds at a time. Dry scaffolds quickly become fragile, so cell suspension needs to be added as soon as possible.</div>
<ol start="26">
<li>Resuspend cells by triturating 2–5 times. Avoid air bubbles.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Shear force can affect cell viability. Once the suspension seems homogeneous, stop pipetting.</div>
<ol start="27">
<li>Add 5 μL of cell suspension directly onto the scaffold.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> A repeater pipette can be used to speed up the seeding process.</div>
<div class="note">
<span class="note-title">Note:</span> If scaffold gets stuck on the pipette tip, use another pipette tip or forceps to gently dislodge.</div>
<ol start="28">
<li>Incubate seeded scaffolds for 30 min at 37°C, 5% CO<sub>2</sub>, 95% relative humidity (RH) for initial cell attachment.</li>
<li>Gently add 200 μL Neuro Medium Plus 5% FBS to each well and incubate at 37°C, 5% CO<sub>2</sub>, 95% RH for 18–24 h.</li>
</ol>
<h3 id="sec3.3">Adeno-associated virus (AAV) infection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h on day 1</div>
<p>This section describes the protocol for infecting 3D cultures with AAV-Syn1-GCaMP6f (<a href="#bib5">Chen et al., 2013</a>). Cells adhere onto the 3D silk scaffold surfaces overnight before neurons are infected with AAV-Syn1-GCaMP6f, an adenoviral system for expressing the genetically encoded calcium indicator (GECI) under the human synapsin 1 promotor. This procedure should be done in a biosafety hood under sterile conditions. See <a href="#fig3">Figure 3</a>.</p>
<ol start="30">
<li>Warm up Neuro Medium, 500 μL per scaffold, to 37<sup>o</sup>C.</li>
<li>Thaw AAV1-hSyn1-GCaMP6f-P2A-nls-dTomato (referred as AAV hereafter) stock solution on ice.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> See additional steps in Quality Control #3 -Viability Staining (steps 94–97), Quality Control #4 – Visualizing Structural Network (steps 98–100), and Quality Control #5 – Gene Expression (steps 101–109).</div>
<ol start="32">
<li>Dilute AAV in pre-warmed Neuro Medium (serum-free) to a final concentration of 2 × 10<sup>10</sup> GC/mL.</li>
<li>Add 500 μL AAV/Neuro Medium to individual wells of 48-well plate for infection (1 well for each scaffold to be infected).</li>
<li>Using forceps, transfer each scaffold into one of these wells containing AAV/Neuro Medium. Incubate at 37°C for 24 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is normal to have many unadhered cells on the bottom of the 96-well plate from day 0. Seeding efficiency is estimated to be around 60%–70%.</div>
<h3 id="sec3.4">Collagen I gel casting and culture maintenance</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h on day 2 and 30 min every 3–4 days</div>
<p>This section describes casting of Collagen Type I onto the seeded 3D scaffolds. By this day, cells have adhered to the surfaces throughout the 3D scaffold. Collagen gel will fill the pores of the scaffold to provide an additional substrate for 3D neurite extension. This procedure should be done in a biosafety hood under sterile conditions. See <a href="#fig3">Figure 3</a>.</p>
<ol start="35">
<li>Warm up Neuro Medium, 2 mL per scaffold, to 37<sup>o</sup>C.</li>
</ol>
<ol start="36">
<li>Prepare 3 mg/mL Collagen Type I solution by mixing Collagen stock solution, 1 M NaOH, 10× PBS, and cold media according to the following calculations. Keep solution on ice.
<div class="critical">
<span class="critical-title">Critical:</span> To account for pipetting loss, prepare at least 20 μL per scaffold and at least 500 μL total.</div>
<div class="note">
<span class="note-title">Note:</span> Use Collagen Type I stock solution with concentration &gt;3.5 mg/mL if possible. Otherwise, there will not be enough room for 10× PBS and/or medium.</div>
<ol type="a">
<li>Collagen Type l I stock volume = [final volume] × [final concentration 3 mg/mL] ÷ [stock concentration]</li>
<li>NaOH = 0.023 × [Col I stock volume]</li>
<li>10× PBS = 0.1 × [final volume]</li>
<li>Cold media = [final volume] − [Col I stock volume] − [NaOH] − [10× PBS]</li>
<li>Check pH, which should be in the neutral range.</li>
</ol>
</li>
</ol>
<ol start="37">
<li>Transfer each scaffold to an individual well of a 96-well plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not remove excess medium from the scaffold as it may result in cell detachment.</div>
<ol start="38">
<li>Add 10 μL Collagen Type I solution directly onto each scaffold.</li>
<li>Incubate at 37°C, 5% CO<sub>2</sub>, 95% RH for 30–40 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not incubate longer as cell health is affected by the lack of medium.</div>
<ol start="40">
<li>In 24-well glass-bottom plates, add 2 mL Neuro Medium to each well and avoid edge wells. Fill empty wells and space between wells with PBS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The PBS helps with maintaining humidity during the culture period and with maintaining temperature during Ca<sup>2+</sup> imaging in the subsequent step.</div>
<ol start="41">
<li>Transfer the now complete 3D cultures to the 24-well glass-bottom plates (one scaffold/well).</li>
<li>Maintain cultures at 37°C, 5% CO<sub>2</sub>, 95% RH with 50% medium change every 3–4 days.</li>
<li>Change 50% medium 1–2 days prior to Ca<sup>2+</sup> imaging.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid changing or adding new medium on the day of imaging because sudden change in medium composition may cause temporary alternation in baseline activities.</div>
<h3 id="sec3.5">Ca<sup>2+</sup> imaging on a wide-field epifluorescence microscope</h3>
<div class="timing">
<span class="timing-title">Timing:</span> variable (approximately 1 h preparation plus 10–30 min per sample)</div>
<p>The functional measurement of the 3D system is achieved using the genetically encoded calcium indicator (GCaMP) to detect changes in the cytoplasmic Ca<sup>2+</sup> concentrations in the neural culture (<a href="#bib5">Chen et al., 2013</a>). A large field of view is needed to investigate functional networks at mesoscale, i.e., activity of populations of neurons across hundreds of μm (<a href="#bib1">Betzel and Bassett, 2017</a>). This section describes time-lapse recordings of Ca<sup>2+</sup> transients in neurons expressing GCaMP at a low magnification/large field of view to capture the entire 3D culture projection. The image data are used for network analysis in the subsequent sections. See <a href="#fig4">Figure 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Wide-field Ca<sup>2+</sup> imaging of 3D cultures</div>
<p>See also <a href="#mmc10">Methods Video S10</a>.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Ca<sup>2+</sup> imaging is recommended at 3 weeks of culture. GCaMP expression is sufficient at 2 weeks of culture for Ca<sup>2+</sup> imaging but higher sample-to-sample variability has been observed at an earlier time point.</div>
<div class="note">
<span class="note-title">Note:</span> For terminal time points, this procedure does not need to be done under sterile conditions. For longitudinal studies, conduct this procedure under sterile conditions as much as possible.</div>
<ol start="44">
<li>Set the microscope stage-top incubator to 37°C and 5% CO<sub>2</sub>.</li>
<li>Remove enough medium (approximately 1.5 mL per well) so that the 3D culture is not floating, but still submerged in medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> 3D cultures must not be moving during time-lapse imaging.</div>
<div class="note">
<span class="note-title">Note:</span> For longitudinal studies of the same set of 3D cultures, this medium should be removed under sterile conditions, saved and placed back into the original wells/plates after Ca<sup>2+</sup> imaging is complete. Use sterile tools and tips during imaging for longitudinal study samples.</div>
<ol start="46">
<li>Place the plate in the microscope stage-top incubator.</li>
<li>Use the red fluorescence channel to visually check the density of dTomato-positive cells. Flip the 3D culture with forceps (or microspatula) to determine which side has the higher number of dTomato<sup>+</sup> cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is not unusual if one side has significantly more dTomato<sup>+</sup> cells than the other.</div>
<ol start="48">
<li>Leave 3D cultures undisrupted in the microscope stage-top incubator for at least 30 min to reach steady state.</li>
<li>Choose the microscope objective based on the required field size and resolution. In this study, a 4× objective, with 4.15 mm × 3.51 mm field size was used to capture the full 3D culture projection.</li>
<li>Set the acquisition rate, time-lapse duration, FITC channel exposure time, gain, and binning. A minimum 1 Hz is required (higher frequency recommended) to capture Ca<sup>2+</sup> events, and the acquisition parameters should be optimized according to user’s microscope and camera setup. <a href="#tbl1">Table 1</a> describes the factors to consider for each acquisition parameter.
<table id="tbl1">
<caption>Table 1. Time-lapse image acquisition setup for Ca<sup>2+</sup> imaging</caption>
<thead>
<tr>
<th>Image acquisition factors</th>
<th>Options/Explanations</th>
<th>Used in this study with Nikon Eclipse Ti-2 Microscope and Sola SE-Illuminator</th>
</tr>
</thead>
<tbody>
<tr>
<td>Objective</td>
<td>
<ul>
<li>Low (4×) – larger field of view. Enables millimeter-sized network analysis. Lower or no cellular resolution</li>
<li>High (10× or higher) – smaller field of view. Higher and possible cellular resolution</li>
</ul>
</td>
<td>4×</td>
</tr>
<tr>
<td>Fluorescence output</td>
<td>
<ul>
<li>Low – less photobleaching. Requires longer exposure</li>
<li>High – more photobleaching. Requires shorter exposure</li>
</ul>
</td>
<td>Low (2%)</td>
</tr>
<tr>
<td>Gain</td>
<td>
<ul>
<li>Low gain – less noise. Requires longer exposure</li>
<li>High gain – more noise. Requires shorter exposure</li>
</ul>
</td>
<td>High gain (gain-4)</td>
</tr>
<tr>
<td>Binning</td>
<td>
<ul>
<li>No binning – high resolution. Low signal and requires longer exposure</li>
<li>Binning – reduced resolution. Higher signal and requires shorter exposure</li>
</ul>
</td>
<td>3 × 3 binning</td>
</tr>
<tr>
<td>Exposure</td>
<td>Must be short enough to achieve the required acquisition speed</td>
<td>50 ms</td>
</tr>
<tr>
<td>Speed (interval)</td>
<td>Must be ≥1 Hz to capture Ca<sup>2+</sup> events. Higher frequency provides better resolution of the peaks</td>
<td>5 Hz (200 ms)</td>
</tr>
<tr>
<td>Duration</td>
<td>
<ul>
<li>Short – smaller file size and requires less computational power for analysis</li>
<li>Long – captures better activity patterns but larger file size</li>
</ul>
</td>
<td>1–3 min</td>
</tr>
</tbody>
</table>
</li>
<li>Move the stage so that the 3D culture is centered in the image field of view.</li>
<li>Record time-lapse series of green fluorescence signal.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If there is no GCaMP signal or change in fluorescence intensity, see <a href="#troubleshooting">Troubleshooting 5</a> and <a href="#troubleshooting">Troubleshooting 6</a>.</div>
<div class="optional">
<span class="optional-title">Optional:</span> See additional steps in Quality Control #2 – Drug Response (steps 90–93).</div>
<ol start="53">
<li>Export the time-lapse files as OEM .tif image stacks.</li>
</ol>
<h3 id="sec3.6">Generating region-of-interest (ROI) mask for network analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;1 h</div>
<p>This section describes how to design and convert to binary mask using Adobe Illustrator and Adobe Photoshop. The goal is to create a mask containing 37 hexagonal ROIs, with 500 μm center-to-center distancing (<a href="#fig5">Figure 5</a>). The mask must be binary (black/white) in Bitmap format in order to be used in the subsequent Image Processing Step.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. ROI masks composed of 37 500-μm center-to-center distanced hexagons</div>
</figcaption>
</figure>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Most microscope software has built-in ROI mask functions and can be utilized if it generates a mask in black/white binary and bitmap format. We also provide a pre-made mask with FOV size 4.15 mm × 3.51 mm (852 px × 720 px), which can be downloaded at <a href="https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture">https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture</a>.</div>
<ol start="54">
<li>Obtain μm-to-pixel scale information from the microscopy software.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All the following steps are described in μm, but converted values (in pixels) should be used.</div>
<ol start="55">
<li>In Adobe Illustrator, <i>File → Create new → Custom →</i> enter the width and height (pixels) of the time-lapse image and in <i>Advanced Settings</i> <i>→</i> <i>Color Mode</i>, select RGB.</li>
<li>Double click on the Fill color and change it to black (R=0, G=0, B=0)</li>
<li>From the Tools window, select Rectangle Tool.</li>
<li>Drag and draw a black rectangle to completely cover the artboard.</li>
<li>Double click on the Fill Color and change it to white (R=255, G=255, B=255)</li>
<li>Switch to Polygon Tool (click and hold Rectangle Tool to open Polygon Tool). Single click and enter Radius and Sides (6). A white hexagon should appear.</li>
<li>Use Selection Tool to select the hexagon. Use Rotate Tool if necessary.</li>
<li>From the top menu, <i>Object → Transform → Move.</i> Enter Distance (500 μm) and Angle, Check Preview, and Copy.</li>
<li>Repeat Move/Copy with the correct angles until the mask pattern is generated.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Multiple selected hexagons can be grouped by <i>Right Click → Group</i>. The Move/Copy comment will apply to all the hexagons in the group.</div>
<ol start="64">
<li>
<i>File → Export → Export As</i> and save as TIF, Color Mode: Grayscale, Anti-aliasing: None, Resolution: 72 dpi (LZW compression is ok).</li>
</ol>
<ol start="65">
<li>Open the TIF file in Adobe Photoshop. Image <i>→</i> Mode, switch to Bitmap.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Open the TIF file in NIS-elements. Select the white hexagons as ROI and save ROI as separate TIF.</div>
<ol start="66">
<li>Save as TIF.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> LZW compression is fine. This generates a binary mask that can be read by FluoroSNNAP in the subsequent steps. An RGB or grayscale format mask is incompatible.</div>
<h3 id="sec3.7">Time-lapse image processing on FluoroSNNAP</h3>
<div class="timing">
<span class="timing-title">Timing:</span> variable (typically 20–30 min per file)</div>
<p>This section describes using open-source software (FluoroSNNAP) to extract fluorescence intensity <i>versus</i> time traces and identifying Ca<sup>2+</sup> events in each ROI (<a href="#bib15">Patel et al., 2015</a>). A global synchronization index is calculated in this step. Cross-correlation of Ca<sup>2+</sup> traces from each ROI pair is calculated and a matrix of cross-correlation coefficients of all the paired ROIs is generated.</p>
<ol start="67">
<li>Download and install FluoroSNNAP.</li>
<li>Run FluoroSNNAP as an administrator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If FluoroSNNAP closes immediately, see <a href="#troubleshooting">Troubleshooting 7</a>.</div>
<ol start="69">
<li>Select <i>File → Add folders containing .tif stacks.</i> Select the folder containing the stack files to analyze. After the path appears in the left window, click on the path.</li>
<li>Select the .tif stack file in the right window, select <i>Pre-Processing → Segmentation GUI → select a frame →#1.</i>
</li>
<li>The Segmentation GUI window appears. Select <i>File → Open Segmentation.</i>In the File Type dropdown menu, select <i>All Files</i>. Select the ROI mask created from the previous section.</li>
<li>Select <i>File → Save segmentation.</i> A .tif file and a .mat (Microsoft Access Table Shortcut) file will be generated.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> These files are subsequently used by the program for further analysis in the subsequent section.</div>
<ol start="73">
<li>Select <i>Analysis → Preferences → Analysis Modules</i> and leave only the following modules checked
<ol type="a">
<li>Convert raw fluorescence</li>
<li>Detect onset of calcium event</li>
<li>Perform synchronization analysis</li>
<li>Estimate functional connectivity</li>
</ol>
</li>
<li>
<i>Select Analysis → Preferences →</i>
<ol type="a">
<li>Acquisition frame rate: Enter the acquisition rate used for the recording, i.e., 5 Hz</li>
<li>Baseline fluorescence: F<sub>0</sub> = 10 s of previous and 20th percentile</li>
<li>Ca<sup>2+</sup> event detection: template-based, threshold = 0.85, and minimum dF/F amplitude = 0.01.</li>
<li>Synchronization analysis: method = phase (1), number of times to perform surrogate resampling = 20, minimum size of synchronization cluster = 3</li>
</ol>
</li>
<li>Select <i>Analysis → Preferences → Functional Connectivity.</i> In this study, we use Cross-Correlation for functional connectivity analysis. See <a href="#troubleshooting">Troubleshooting 8</a> for recommendations on the choices of the functional connectivity. Save preference.</li>
<li>Select <i>Analysis→ Process single file.</i> At the end of the analysis run, a <i>processed_analysis.mat</i> file is generated for each time-lapse and will be used for further analysis.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If FluoroSNNAP freezes, see <a href="#troubleshooting">Troubleshooting 9</a>.</div>
<ol start="77">
<li>Rename the <i>processed_analysis.mat</i> file to <i>processed_analysis_(your_file_name).mat</i> file in the folder, but do not rename it in MATLAB (the <i>RunNetworkAnalysis.m</i> script described in the next section works under the assumption that the name of the file loaded in the Workspace is <i>processed_analysis.mat</i>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Rename the processed-analysis.mat file in the folder after it is generated, or it will get overwritten by FluoroSNNAP when the next time-lapse stack is processed.</div>
<div class="optional">
<span class="optional-title">Optional:</span> To check the output file is in the correct format, open the <i>processed_analysis_(your_file_name).mat</i> in MATLAB. It should load a 1 × 1 processed_analysis.mat structure, containing 21 fields. To view the cross-correlation coefficient matrix, open <i>FC→CC→C</i>.</div>
<h3 id="sec3.8">Network analysis with <i>RunNetworkAnalysis.m</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> variable (typically 1–2 min per file)</div>
<p>This section describes a step-by-step walkthrough of using <i>RunNetworkAnalysis.m</i> which can be accessed at <a href="https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture">https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture</a>. This custom MATLAB code loads the <i>processed_analysis.mat</i> file generated from FluoroSNNAP in the previous step, extracts and organizes the variables and computes parameters that describe the network properties (described in detail after the steps). This code runs on MATLAB (version 2019 or later), and Bioinformatics Toolbox Add-on is required.</p>
<p>In brief, the analysis is based on graph-theory descriptors. A graph (network) is composed of nodes interconnected by edges (<a href="#bib2">Bullmore and Sporns, 2009</a>). In our analysis, each ROI is a node and the statistical dependence (assessed using the cross-correlation method in this manuscript) between two ROI activities is an edge. The computed parameters describe several features of the reconstructed network topology.</p>
<p>The protocol here focuses on mesoscale activity, namely the statistical dependence of the activities of clusters of neurons, to capture the population, rather than single-cell activities, of the entire 3D culture. The user can however customize the size and shape of the ROIs to address different questions. It follows that different spatial parcellations lead to different interpretations of the resulting network (<a href="#bib11">Messé, 2020</a>).</p>
<div class="note">
<span class="note-title">Note:</span> Descriptions of Network Parameters - Node. Each ROI is represented in the network as a node. Overall, a node can be defined from a single neuron to entire circuits; considering the multiscale organization of neuronal networks, there is no single, privileged scale (<a href="#bib9">Hallquist and Hillary, 2019</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Descriptions of Network Parameters - Edge. The edge between two nodes represents their functional connectivity, e.g., the statistical dependence between their activities. The value of functional connectivity between two nodes must not be necessarily interpreted as an indication of a physical connection (structural connectivity).</div>
<div class="note">
<span class="note-title">Note:</span> Descriptions of Network Parameters - Binary versus Weighted networks. A weighted network is a network whose edges have values between 0 and 1. An edge weight represents the weighted statistically dependence of the activity of the two nodes. On the other hand, such estimates are associated with measurement noise (small nonzero and/or negative weights). A binary network could be thus derived through a cutoff procedure with a user-defined threshold to the edge weights. An edge is preserved if its weight is higher than the threshold and removed otherwise. The preserved edges assume a value of 1 in the matrix; the eliminated edges assume a value of 0 instead.</div>
<ol start="78">
<li>Download and unzip Matlab Code-Integrated Functional Neuronal Network Analysis of 3D Silk-Collagen Scaffold-based Cortical.zip. This folder contains the following files:
<ol type="a">
<li>avg_clus_matrix.m</li>
<li>avg_path_matrix.m</li>
<li>circularGraph.m</li>
<li>clustering_coef_matrix.m</li>
<li>matrixExtraction.m</li>
<li>NetworkAnalysis.m</li>
<li>node.m</li>
<li>processed_analysis_sample.mat</li>
<li>RunNetworkAnalysis.m</li>
<li>singleROIAnalyis.m</li>
</ol>
</li>
<li>In MATLAB, change directory to the <i>RunNetworkAnalysis.m</i> folder.</li>
<li>Run <i>RunNetworkAnalysis.m</i> file.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#troubleshooting">Troubleshooting 10</a>.</div>
<ol start="81">
<li>When prompted, select the <i>processed_analysis_(your_file_name).mat</i> of interest.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Run the sample test file <i>processed_analysis_sample.mat</i> to ensure the program operates smoothly.</div>
<ol start="82">
<li>The code will ask:<br> “Do you want to binarize the matrix?”<br> The user can either run the weighted or binary matrix.
<ol type="a">
<li>Weighted graph analysis: Select No.</li>
<li>Binary graph analysis: Select Yes and enter the threshold value.</li>
<li>Savefileas:Enterafilenamefortheexportedexcelspreadsheetcontainingalloutputs.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This threshold value is user-defined. See <a href="#troubleshooting">Troubleshooting 11</a>. To view the raw cross-correlation coefficient values, see <a href="#sec3.7">Time-lapse Image Processing on FluoroSNNAP</a> section, step 79 or the <i>Matrix.mat</i> output from the code.</div>
<ol start="83">
<li>The code outputs the following variables that can be viewed in MATLAB and in the exported Excel spreadsheet. These variables are summarized in <a href="#tbl2">Table 2</a> and <a href="#fig6">Figure 6</a> and described in greater detail below:
<table id="tbl2">
<caption>Table 2. List of output parameters from <i>RunNetworkAnalysis.m</i> script and brief descriptions</caption>
<thead>
<tr>
<th>Data</th>
<th colspan="2">Variable name or location</th>
<th>Brief description</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="7">ROI/node-based</td>
<td colspan="2"><i>CalciumEventsTable</i></td>
<td>timing of calcium peaks in each ROI (s)</td>
</tr>
<tr>
<td colspan="2"><i>ISITable</i></td>
<td>inter-spike intervals (ISI) per ROI (s)</td>
</tr>
<tr>
<td rowspan="3"><i>SingleParameterTable</i></td>
<td><i>AverageFrequency</i></td>
<td>average calcium event frequency (events/min) of the sample</td>
</tr>
<tr>
<td><i>PercentageActiveROI</i></td>
<td>percentage of ROIs with at least one Ca<sup>2+</sup> event</td>
</tr>
<tr>
<td><i>SyncIndex</i></td>
<td>global synchronization index</td>
</tr>
<tr>
<td rowspan="2"><i>NodeDegree</i></td>
<td><i>NodeIndex</i></td>
<td>index of each ROI</td>
</tr>
<tr>
<td><i>Degree</i></td>
<td>node degree of each ROI</td>
</tr>
<tr>
<td rowspan="11">Network-based</td>
<td rowspan="6"><i>FinalTableNetworkData</i></td>
<td><i>N</i></td>
<td>number of nodes (ROIs)</td>
</tr>
<tr>
<td><i>Modularity</i></td>
<td>modularity value</td>
</tr>
<tr>
<td><i>NumberModules</i></td>
<td>number of modules</td>
</tr>
<tr>
<td><i>AverageEdgeWeight</i></td>
<td>average edge weight</td>
</tr>
<tr>
<td><i>AverageClusteringCoef</i></td>
<td>average clustering coefficient</td>
</tr>
<tr>
<td><i>AveragePathLength</i></td>
<td>average path length</td>
</tr>
<tr>
<td rowspan="3"><i>ModulesComposition</i></td>
<td><i>(First column)</i></td>
<td>module index</td>
</tr>
<tr>
<td><i>(Second Column)</i></td>
<td>number of ROIs in the module</td>
</tr>
<tr>
<td><i>(Third Column)</i></td>
<td>index of the ROIs per module</td>
</tr>
<tr>
<td colspan="2"><i>Node’s degree distribution</i></td>
<td>kernel density distribution of the node degree</td>
</tr>
<tr>
<td colspan="2"><i>Circular Graph</i></td>
<td>functional connectivity graph</td>
</tr>
</tbody>
</table>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Summary of single node and network analysis outputs</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this analysis, each ROI is defined as a node, and the terms ROI and node therefore can be used interchangeably. See <a href="#troubleshooting">Troubleshooting 12</a> for error message regarding negative weighted edge.</div>
<div class="note">
<span class="note-title">Note:</span> CalciumEventsTable (Calcium Events Table) – A calcium event is a transient and synchronous change in fluorescence of a cluster of neurons in response to their intracellular elevation of Ca<sup>2+</sup> associated with an action potential. In the Calcium Events Table, each column represents a ROI, and the data represents the time of peak (s) of the Ca<sup>2+</sup> events per ROI (a peak is defined as the time at which a calcium event reaches its maximum value).</div>
<div class="note">
<span class="note-title">Note:</span> ISITable (Inter-spike Interval Table) - The inter-spike interval (ISI) is the average time (s) between two consecutive Ca<sup>2+</sup> events (<a href="#fig6">Figure 6</a>A). It is computed for each ROI and organized in a tabular format. The ISI is calculated as:</div>
<p><i>ISI<sub>n</sub></i> = <i>Peak<sub>n</sub></i> - <i>Peak<sub>n-1</sub></i></p>
<p>where <i>Peak</i> is the time of peak of the <i>n</i><sup>th</sup> event. <b>Interpretation:</b> A greater ISI indicates slower Ca<sup>2+</sup> activity.</p>
<div class="note">
<span class="note-title">Note:</span> AverageFrequency -The average frequency of Ca<sup>2+</sup> events of all ROIs (events/min). <b>Interpretation:</b> A greater average frequency indicates faster neuronal activity.</div>
<div class="note">
<span class="note-title">Note:</span> PercentageActiveROI - The percentage of ROIs with at least one recorded calcium event. It represents the fraction of the entire sample active during the measurement (<a href="#fig6">Figure 6</a>B – Green hexagon: Active ROI; Gray Hexagon: Inactive ROI). <b>Interpretation:</b> A greater value suggests a larger proportion of active surface area of the sample.</div>
<div class="note">
<span class="note-title">Note:</span> SyncIndex (Synchronization Index) - The global synchronization index of the sample. It quantifies the global degree of synchronization of multiple time series recorded simultaneously from multiple ROIs. It is important to notice that the entire traces and not the firing patterns are used to compute the synchronization index. <b>Interpretation:</b> A greater global synchronization index suggests that the sample activity is more synchronous (<a href="#fig6">Figure 6</a>B).</div>
<div class="note">
<span class="note-title">Note:</span> N- The number of ROIs/nodes. It is customized by the user while creating the ROI mask (ex. <a href="#fig6">Figure 6</a> – seven ROIs).</div>
<div class="note">
<span class="note-title">Note:</span> NodeDegree (Node Degree) - The node degree (<a href="#fig6">Figure 6</a>C - the numbers refer to the central green ROI), normalized to the maximum number of edges (N-1), calculated as:</div>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Si2.jpg" alt="Si2.jpg"></p>
<p>In a weighted graph, <i>degree<sub>i</sub></i> is the sum of the edge weights for edges directly connected to node<sub>i</sub>. In a binary graph, <i>degree<sub>i</sub></i> is the number of edges directly connected to node<sub>i</sub>. <b>Interpretation:</b> A greater node degree suggests a stronger connectivity of the i<sup>th</sup> node to the other nodes.</p>
<div class="note">
<span class="note-title">Note:</span> Node degree distribution - A Kernel density estimation graph showing probability distribution <i>versus.</i> node degree value. <b>Interpretation</b>: A distribution curve with a long tail may suggest the existence of hub nodes – a small number of nodes with node degree greatly exceeding the average. Real world networks (i.e., brain, internet, air traffic network) tend to possess a long tail distribution, reflecting the presence of hub nodes (<a href="#bib2">Bullmore and Sporns, 2009</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Modules and Modularity - A module is a group of nodes that are densely/strongly connected within modules and weakly/sparsely between nodes belonging to other modules (<a href="#fig6">Figure 6</a>D – green, red, and blue modules). Modularity is a value that measures the strength by which the network can be partitioned into modules. Number of modules and module assignment of nodes in a network are identified by maximizing the network’s modularity during computation. <b>Interpretation:</b> Real world networks tend to divide naturally into communities or modules. A greater modularity indicates stronger community structure of the network (<a href="#bib13">Newman, 2006</a>).</div>
<div class="note">
<span class="note-title">Note:</span> NumberofModules (Number of Modules) - The number of modules in which the network has been partitioned (<a href="#fig6">Figure 6</a>D – green, red, and blue modules).</div>
<div class="note">
<span class="note-title">Note:</span> ModulesComposition (Modules Composition) - A table summarizing the module index (first column), the number of ROIs in each module (second column) and the node indices in each module (third column).</div>
<div class="note">
<span class="note-title">Note:</span> AverageEdgeWeight (Average Edge Weight) - The average value of the normalized node degree of the sample. <b>Interpretation:</b> In a weighted graph, it represents the average edge weight of a node. In a binary graph, it represents the average probability that a node is connected to another one. Larger values correspond to greater average connectivity of the network.</div>
<div class="note">
<span class="note-title">Note:</span> AverageClustCoeff (Average Clustering Coefficient) - The average clustering coefficient of a network (<a href="#fig6">Figure 6</a>E) is calculated as the average of the local clustering coefficients <i>CC</i><sub>i</sub> of all the nodes as follows:</div>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Si4.jpg" alt="Si4.jpg"></p>
<p>In a weighted network, <i>CC</i><sub>i</sub> represents the strength of triangle networks (<a href="#fig6">Figure 6</a>E, left) formed by three neighboring nodes and is calculated as follows (<a href="#bib14">Onnela et al., 2005</a>):</p>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Si5.jpg" alt="Si5.jpg"></p>
<p>In a binary graph, <i>CC</i><sub>i</sub> represents the probability that two nodes <i>j</i> and <i>k</i> are connected to each other while they are both connected to a node <i>i</i> (<a href="#fig6">Figure 6</a>E, right). <b>Interpretation:</b> A greater average clustering coefficient suggests stronger tendency of the network to form local clusters. Real world networks tend to have a high clustering coefficient (<a href="#bib12">Muldoon et al., 2016</a>).</p>
<div class="note">
<span class="note-title">Note:</span> AveragePathLength (Average Path Length) – The average path length of a network (<a href="#fig6">Figure 6</a>F) is calculated as:</div>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Si6.jpg" alt="Si6.jpg"></p>
<p>where <i>N</i> is the total number of nodes and <i>d(i,j)</i> is the shortest path between node <i>i</i> and <i>j</i> (<a href="#fig6">Figure 6</a>F – shortest path is indicated in red). For a weighted network, the shortest path is calculated as <i>d</i>(<i>i</i>,<i>j</i>) = 1/<i>w<sub>ij</sub></i>, where <i>w<sub>ij</sub></i> is the edge weight between the node <i>i</i> and <i>j</i>. For a binary network, the shortest path represents the smallest number of edges to connect node <i>i</i> and <i>j</i>. <b>Interpretation:</b> A greater average path length suggests that a longer distance or more nodes must be crossed to transmit the information between two random nodes. Average path length is also inversely correlated to the probability of shortcuts in the network. Real world networks tend to have a small value of the path length (<a href="#bib12">Muldoon et al., 2016</a>). It is also important to mention the debate on its interpretation in correlation-based functional networks. In short, paths in those networks represents sequences of statistical association but may not necessarily characterize the presence of information routes (<a href="#bib7">Fornito et al., 2010</a>).</p>
<h3 id="sec3.9">Quality control assays (optional)</h3>
<p>The quality control assays provide additional procedures to examine the viability, development, and functionality of the neurons and 3D cultures. We recommend conducting the quality control assays at the same time as the experiments using the same batch of materials and cells, even though they are not required for the functional analysis.</p>
<h3 id="sec3.10">Quality control assay #1 – 2D</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Add 0.5–1 h to each day of the 3D culture steps</div>
<p>This step describes conventional 2D culture of cortical neurons. We recommend conducting it using the same batch of neurons and for the same duration as in the 3D culture to ensure the viability of the neurons from the isolation procedure.</p>
<ol start="84">
<li>On day −1, coat tissue culture plates with PDL (0.1 mg/mL) and laminin (10 μg/mL) for 12–24 h.</li>
<li>On day 0, wash the coating with PBS three times (each with 5 min incubation) and once with Neuro Medium + 5% FBS (incubate until cell seeding).</li>
<li>Seed cells at 100–250k/cm<sup>2</sup> in Neuro Medium + 5% FBS.</li>
<li>On day 1, remove all Neuro Medium + 5% FBS and replace with serum-free Neuro Medium.</li>
<li>Change 50% medium every 3–4 days.</li>
<li>Examine cell morphology frequently.</li>
</ol>
<h3 id="sec3.11">Quality control assay #2 – drug response</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Add 15–30 min per sample on Ca <sup>2+</sup> <b>imaging day</b>
</div>
<p>This section describes using neurotransmitter receptor antagonists, namely bicuculline (GABA<sub>A</sub> receptor blocker), picrotoxin (GABA<sub>A</sub> and GABAρ receptor blocker), NBQX (AMPA receptor blocker), AP5 (NMDA receptor blocker), and tetrodotoxin (sodium channel blocker) on the 3D cultures to elicit known responses. We recommend conducting it after 2–3 weeks of culture and at the same time point as the control 3D cultures.</p>
<ol start="90">
<li>Dilute the 4 μL stock solution of bicuculline, picrotoxin, and NBQX with 76 μL Neuro Medium to make 100× working solution.</li>
<li>After acquiring a baseline fluorescence time-lapse, pipet 5 μL drug stock solution (100×) into the 500 μL medium. (Final concentrations: bicuculline 10 μM, picrotoxin 50 μM, NBQX 10 μM, AP5 25 μM, tetrodotoxin 20–40 μM).</li>
<li>Incubate for the following time: 10 min for picrotoxin and bicuculline, 20 min for NBQX, AP5, and tetrodotoxin.</li>
<li>Acquire post-treatment fluorescence time-lapse.</li>
</ol>
<h3 id="sec3.12">Quality control assay #3 – viability staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This section describes viability staining of 3D cultures with Calcein-AM, which is taken up only by viable cells and becomes fluorescent in the green spectrum. This step can be done at any time point.</p>
<ol start="94">
<li>Add 500 μL Calcein-AM working solution to a well in a 48-well plate.</li>
<li>Transfer 3D culture into the well containing the Calcein-AM working solution. Incubate at 37°C for 30 min.</li>
<li>Aspirate the Calcein-AM working solution and wash once with PBS with 5 min incubation.</li>
<li>Image with green fluorescence filter.</li>
</ol>
<h3 id="sec3.13">Quality control assay #4 – visualizing structural network</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Add 15 min to day 1 (AAV infection) and 2–3 h per sample to image on confocal microscope</div>
<p>This section describes infecting the 3D culture with AAV to express RFP in the neurons, in order to visualize the structural neuronal network in the 3D culture using confocal microscopy.</p>
<ol start="98">
<li>Follow Cortical Neuron Isolation and Cell Seeding section to prepare 3D cultures. In the AAV Infection section, steps 31–33, infect the seeded scaffolds with AAV1-hSyn1-TurboRFP at 1–3 × 10<sup>6</sup> GC/mL.</li>
<li>Between day 14 and 21, fix the 3D cultures with 4% paraformaldehyde/4% sucrose for 1 h followed by three 30-min PBS washes.</li>
<li>Acquire z-stack images on a confocal microscope (20× to 40× objectives recommended).</li>
</ol>
<h3 id="sec3.14">Quality control assay #5 – gene expression</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Variable (5–6 h per 24 samples of RNA isolation, 3 h for cDNA synthesis, 3 h per 96-well plate of qRT-PCR)</div>
<p>This section describes qRT-PCR analysis of gene expression of proteins associated with neuronal maturation and synaptic neurotransmission (See <a href="#key-resources-table">Key resources table</a> and <a href="#tbl3">Table 3</a>) of the 3D cultures shortly after cells seeding and as the neuronal networks develop.</p>
<table id="tbl3">
<caption>Table 3. List of genes to detect in the 3D culture and their expected dCt values (normalized to 18s)</caption>
<thead>
<tr>
<th rowspan="2">Gene</th>
<th rowspan="2">Encoded protein</th>
<th rowspan="2">Association</th>
<th rowspan="2">Day</th>
<th colspan="2">dCt (Target gene – 18s)</th>
</tr>
<tr>
<th>Min</th>
<th>Max</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3"><i>Dcx</i></td>
<td rowspan="3">Doublecortin</td>
<td rowspan="3">immature neurons</td>
<td>1</td>
<td>9.57</td>
<td>13.97</td>
</tr>
<tr>
<td>14</td>
<td>12.13</td>
<td>16.15</td>
</tr>
<tr>
<td>21</td>
<td>13.58</td>
<td>18.27</td>
</tr>
<tr>
<td rowspan="3"><i>Pax6</i></td>
<td rowspan="3">Paired Box Protein Pax-6</td>
<td rowspan="3">immature neurons</td>
<td>1</td>
<td>14.08</td>
<td>21.20</td>
</tr>
<tr>
<td>14</td>
<td>17.06</td>
<td>23.34</td>
</tr>
<tr>
<td>21</td>
<td>17.01</td>
<td>17.83</td>
</tr>
<tr>
<td rowspan="3"><i>Syn1</i></td>
<td rowspan="3">Synapsin I</td>
<td rowspan="3">presynaptic terminal</td>
<td>1</td>
<td>14.39</td>
<td>19.00</td>
</tr>
<tr>
<td>14</td>
<td>14.05</td>
<td>17.57</td>
</tr>
<tr>
<td>21</td>
<td>12.59</td>
<td>17.84</td>
</tr>
<tr>
<td rowspan="3"><i>Shank3</i></td>
<td rowspan="3">SH3/ankyrin domain gene 3</td>
<td rowspan="3">postsynaptic terminal</td>
<td>1</td>
<td>15.62</td>
<td>16.58</td>
</tr>
<tr>
<td>14</td>
<td>15.37</td>
<td>15.73</td>
</tr>
<tr>
<td>21</td>
<td>14.04</td>
<td>15.28</td>
</tr>
<tr>
<td rowspan="3"><i>Slc17a7</i></td>
<td rowspan="3">Vesicular glutamate transporter 1</td>
<td rowspan="3">glutamate neurotransmission (presynaptic)</td>
<td>1</td>
<td>16.31</td>
<td>19.97</td>
</tr>
<tr>
<td>14</td>
<td>13.96</td>
<td>16.22</td>
</tr>
<tr>
<td>21</td>
<td>11.64</td>
<td>15.83</td>
</tr>
<tr>
<td rowspan="3"><i>Gria1</i></td>
<td rowspan="3">Glutamate Ionotropic Receptor AMPA Type Subunit 1</td>
<td rowspan="3">glutamate neurotransmission (postsynaptic)</td>
<td>1</td>
<td>14.28</td>
<td>18.44</td>
</tr>
<tr>
<td>14</td>
<td>14.32</td>
<td>17.13</td>
</tr>
<tr>
<td>21</td>
<td>13.60</td>
<td>17.28</td>
</tr>
<tr>
<td rowspan="3"><i>Grin1</i></td>
<td rowspan="3">Glutamate Ionotropic Receptor NMDA Type Subunit 1</td>
<td rowspan="3">glutamate neurotransmission (postsynaptic)</td>
<td>1</td>
<td>14.81</td>
<td>19.64</td>
</tr>
<tr>
<td>14</td>
<td>14.27</td>
<td>18.14</td>
</tr>
<tr>
<td>21</td>
<td>13.16</td>
<td>16.40</td>
</tr>
<tr>
<td rowspan="3"><i>Slc1a3</i></td>
<td rowspan="3">Excitatory amino acid transporter 1</td>
<td rowspan="3">GABA neurotransmission (presynaptic)</td>
<td>1</td>
<td>14.27</td>
<td>18.95</td>
</tr>
<tr>
<td>14</td>
<td>13.13</td>
<td>16.09</td>
</tr>
<tr>
<td>21</td>
<td>12.12</td>
<td>16.90</td>
</tr>
<tr>
<td rowspan="3"><i>Gabra1</i></td>
<td rowspan="3">Gamma-aminobutyric acid type A receptor alpha1 subunit</td>
<td rowspan="3">GABA neurotransmission (postsynaptic)</td>
<td>1</td>
<td>16.67</td>
<td>19.16</td>
</tr>
<tr>
<td>14</td>
<td>14.62</td>
<td>16.46</td>
</tr>
<tr>
<td>21</td>
<td>12.32</td>
<td>15.15</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Use nuclease-free supplies. Keep mRNA samples on ice as much as possible.</div>
<ol start="101">
<li>Collect uninfected 3D cultures on day 1, 14, and 21 each in 600 μL RLT/βME buffer in a 1.5 mL nuclease-free microcentrifuge tube. Store at −80°C.Thaw samples on ice. Use a homogenizer pestle to manually grind 3D culture into fine pieces.</li>
<li>Use a motorized tissue grinder to further grind 3D culture for 20 s.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Homogenized samples can be stored at −80°C.</div>
<div class="note">
<span class="note-title">Note:</span> Long-term storage at this step can significantly affect mRNA concentration. Perform mRNA isolation as soon as possible.</div>
<ol start="103">
<li>Transfer sample to a QIAshredder column and centrifuge at 21,000 × <i>g</i> for 2 min.</li>
<li>Isolate mRNA from the flow through using a Qiagen RNeasy Mini Kit according to the manufacturer’s protocol (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/rna-purification/total-rna/rneasy-mini-kit/%23resources">https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/rna-purification/total-rna/rneasy-mini-kit/#resources</a>) with the exception to elute the mRNA using only 22–24 μL of nuclease-free H<sub>2</sub>O.</li>
<li>Measure mRNA concentrations using a NanoDrop spectrophotometer.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> mRNA can be temporarily stored at −80°C, but it is recommended to reverse transcribe to cDNA as soon as possible.</div>
<ol start="106">
<li>Synthesize cDNA using Bio-Rad iScript gDNA clear cDNA synthesis kit according to the manufacturer’s protocol (<a href="https://www.bio-rad.com/en-us/product/iscript-gdna-clear-cdna-synthesis-kit?ID=NUU8WD15">https://www.bio-rad.com/en-us/product/iscript-gdna-clear-cdna-synthesis-kit?ID=NUU8WD15</a>).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> cDNA can be stored long-term at −80°C.</div>
<ol start="107">
<li>Assume 100% reverse transcription efficiency (1 ng mRNA = 1 ng cDNA), dilute cDNA with nuclease-free water to 5 ng/μL.</li>
<li>Run quantitative real-time PCR (qRT-PCR) to detect mRNA expression using Taqman Gene Expression Assays according to the manufacturer’s protocol (<a href="http://tools.thermofisher.com/content/sfs/manuals/cms_041280.pdf">http://tools.thermofisher.com/content/sfs/manuals/cms_041280.pdf</a>). Each reaction contains 10 ng of cDNA (i.e., 2 μL), 7 μL nuclease-free water, 1 μL Taqman assay, and 10 μL Taqman Gene Expression Assay Master Mix.</li>
<li>Calculate relative mRNA expression fold change using Rn18s as the reference gene and normalizing to the day 1 sample.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">AAV infection and Ca<sup>2+</sup> imaging</h3>
<p>By two weeks, a large number of neurons in 3D cultures infected with AAV-hSyn1-GCaMP6f-P2A-nls-dTomato should express strong nuclear dTomato signal (<a href="#fig7">Figure 7</a>A). GCaMP6f has less prominent fluorescence signal (<a href="#fig7">Figure 7</a>B) and is not as clearly visible as dTomato. Spontaneous neuronal activities, observable as changes in GCaMP6f signal intensity, should be seen in clusters (&gt; 30 μm) of neurons (See <a href="#mmc10">Methods Video S10</a>). It should be noted that silk scaffolds are autofluorescent and do produce background in both the red and green channels. Non-homogenous distribution of cells is not unexpected.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Fluorescence images of 3D culture</div>
<p>(A) Nuclear dTomato and (B) GCaMP6f at 2 weeks of culture. Background fluorescence from silk scaffolds. Scale bar, 500 μm. See also <a href="#mmc10">Methods Video S10</a>.</p>
</figcaption>
</figure>
<figure id="mmc10"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Mmc10.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S10. Spontaneous neuronal activities in the 3D cortical culture at 3 weeks</div>
<p>Time-lapse video of changes in GCaMP6f fluorescence intensity (related to the Expected outcomes section). Scaffold diameter = 3 mm. Video reused with permission from <a href="#bib6">Dingle et al. (2020)</a>. See also Figures 4 and 7.</p>
</figcaption>
</figure>
<h3 id="sec4.2">Network activity and analysis</h3>
<p>At 3 weeks, the 3D cultures are expected to show spontaneous and sustained activity in most of the regions (<a href="#fig8">Figure 8</a>A). Small clusters of neurons should have locally synchronized activities and Ca<sup>2+</sup> transients should be visible under the microscope.</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Examples of expected results of single ROI and network analyses</div>
<p>(A) ROI-based analysis. Left: Raster plot of Ca<sup>2+</sup> events per ROI versus time. Each dot represents a Ca<sup>2+</sup> event. Right: Average Ca<sup>2+</sup> event frequency and global synchronization index. Each dot represents a 3D culture sample.</p>
<p>(B) Network-based analysis. Top left: Probability density function (pdf) versus normalized node degree of a representative culture. Top right: Average clustering coefficient and average path length. Bottom left: Module compositions of a representative 3D culture sample. Bottom right: Number of modules and modularity. Each dot in the scatter graphs represents a 3D culture sample.</p>
<p>(A) and (B) All bar graphs are mean, first and third quartiles, and minimum and maximum. n = 7.</p>
</figcaption>
</figure>
<p>When FluoroSNNAP finishes, it should generate 6 files:</p>
<ul>
<li>original_filename.xlsx</li>
<li>original_filename_summary.txt</li>
<li>analysis.mat</li>
<li>analysis-original_filename.mat</li>
<li>CaSignal-original_filename.mat</li>
<li>processed_analysis.mat</li>
</ul>
<p>When <i>RunNetworkAnalysis.m</i> finishes running without errors, it should generate the following results in the workspace (additional items are also visible in the workspace). See <a href="#sec3.8">Network analysis with RunNetworkAnalysis.m</a> Section for descriptions of each variable</p>
<ul>
<li>CalciumEventsTable</li>
<li>FinalTableNetworkData</li>
<li>ISITable</li>
<li>Matrix: This is the cross-correlation coefficient matrix extracted from the processed_analysis.mat</li>
<li>Modules</li>
<li>ModuleComposition</li>
<li>NodeDegree</li>
<li>SingleParameterTable: This contains AverageFrequency, PercentageActiveROI, and SyncIndex.</li>
</ul>
<p>Two figures will be generated:</p>
<p>The first figure is Kernel Density Distribution of the node degree, with the average degree indicated as well.</p>
<p>The second figure is a circular graph showing the functional connectivity graph. For weighted graphs, the edge weight is directly proportional to the thickness of the lines.</p>
<p>In addition, an Excel file is automatically saved containing the outputs divided in different sheets (as detailed in the code comments section).</p>
<p>Example results from real samples are provided in <a href="#fig8">Figure 8</a>, using weighted network analysis.</p>
<p>A representative raster plot (each dot is the time of peak of a single calcium event) is shown in <a href="#fig8">Figure 8</a>A (left). The number of events/min and synchronization index are then graphed for 7 samples (<a href="#fig8">Figure 8</a>A - right). Upon network reconstruction (weighted network), the probability density function (pdf) of the normalized node degree is generated, as shown in the representative distribution of <a href="#fig8">Figure 8</a>B (top, left). The curve had a Gaussian-like profile suggesting the absence of hub nodes, namely nodes with a high node degree. Two topological descriptors, the average clustering coefficient and average path length, are computed from the network (<a href="#fig8">Figure 8</a>B – top, right). Finally, the network is partitioned into modules (<a href="#fig8">Figure 8</a>B – bottom, left; red nodes and edges) based on the computation of the modularity (<a href="#fig8">Figure 8</a>B – right, bottom).</p>
<h3 id="sec4.3">Quality controls</h3>
<p>QC #1 - In 2D cultures, short neurites emerged on day 1. Visual networks with increasing density form between day 3 and 7 (<a href="#fig9">Figure 9</a>A).</p>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/437-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Representative images of expected results from quality control assays</div>
<p>(A) Phase contrast image of 2D morphology of cortical neurons at 3 weeks.</p>
<p>(B) Fluorescence image of live cells in 3D culture at 2 weeks, indicated by calcein (green).</p>
<p>(C) Reconstructed confocal z-stack image of TurboRFP+ neurons in the 3D culture.</p>
<p>Scale bars, (A) 100 μm (B) 200 μm (C) 50 μm.</p>
</figcaption>
</figure>
<p>QC #2 - When 3D cultures are treated with picrotoxin or bicuculline, Ca2+ event frequency increases. When 3D cultures are treated with NBQX or AP5, Ca2+ event frequency decreases or ceases. When 3D cultures are treated with tetrodotoxin, all or almost all activities are eliminated.</p>
<p>QC #3 - High density of live calcein-positive cells throughout the 3D culture. 3D cultures are expected to have lower density of viable cells at 2–3 weeks of culture than at &lt;1 week (<a href="#fig9">Figure 9</a>B).</p>
<p>QC #4 - In AAV-syn1-TurboRFP infected 3D cultures, strong TurboRFP expression in neurons at 2–3 weeks of culture is observed, demonstrating an extensive neuronal network with long neurites (hundreds of μm). When visualized using confocal microscopy with z-stack imaging, observations of many small and large round objects are expected (<a href="#fig9">Figure 9</a>C).</p>
<p>QC #5 - A 400–1,200 ng mRNA yield per 3D culture. Significant decreases of <i>Dcx</i> and <i>Pax6</i> and increases of <i>Syn1, Shank3, Slc17a7, Gria1, Grin1, Slc1a3,</i> and <i>Gabra1</i> from day 1 to day 14 and day 21. See <a href="#tbl3">Table 3</a> for dCt (Ct<sub>target gene</sub> − Ct<sub>18s</sub>) value ranges obtained from real samples.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Network activity depends on the health of the primary neurons and the distribution of cells within the scaffold. Therefore, the cultures are prone to sample-to-sample, batch-to-batch, and/or user-to-user variability. The network analysis is based on the projection imaging of the 3D culture, and therefore the z-direction network information flow cannot be inferred from this protocol. Ultra-speed confocal microscopy, such as light sheet confocal microscopy, will be required to obtain x,y,z information at the speed necessary to detect Ca<sup>2+</sup> transients. This protocol focuses on wide-field, mesoscale network analysis among populations of neurons. It relies on synchronized Ca<sup>2+</sup> transients in local clusters of neurons for a Ca<sup>2+</sup> event to be detectable per ROI. If the neuronal activity is mainly asynchronous, Ca<sup>2+</sup> event detection becomes unreliable and cannot be distinguished from the background signal. When 3D cultures are under conditions where there will be low or no activity, global synchronization index may become artificially high due to the similarly of background noise, and network analysis cannot be reliably performed.</p>
<p>This protocol has the potential to be adapted to analyze neural networks of human iPSC-derived or neural stem cell-derived neurons. Key issues to consider include: 1) the AAV-GCaMP6f infection step needs to be optimized for human neuronal cultures and may require repeated AAV infections to maintain adequate GCaMP levels, 2) human neurons mature at a slower rate than rodent neurons and synapsin I expression is required for GCaMP6f expression, and 3) functional neural network formation may take longer in human cultures than in rodent cultures and therefore Ca<sup>2+</sup> imaging time points may need to be adjusted accordingly.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Silk fibroin not fully dissolved after 4 h in 60°C (<a href="#before-you-begin">Before you begin</a> step 12).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Inaccurate LiBr solution concentration due to the stock LiBr powder absorbing too much moisture from air. Use new LiBr and keep the container closed and parafilm-sealed as much as possible.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Silk solution became opaque and hardens (<a href="#before-you-begin">Before you begin</a> step 20).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Silk solution is in storage for too long and starts to form insoluble β-sheet structure. Prepare fresh silk solution and use it within 2 weeks.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Sponge is not homogenous with visible large pores and cracks (<a href="#before-you-begin">Before you begin</a> step 26).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>NaCl must be dry with individual particles well-separated and not in clumps. NaCl must be poured evenly when preparing the silk sponge. Practice pouring NaCl into an empty petri dish.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Sponges/scaffolds appear fragile and are breaking apart (<a href="#before-you-begin">Before you begin</a> step 27).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Sponges/scaffolds are likely too dry at some point. Make new sponges and always keep them hydrated.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>No or faint GCaMP signal (major step 52).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Check dTomato signal. Image the side with higher dTomato expression, which also indicates higher GCaMP expression. Perform Ca<sup>2+</sup> imaging no earlier than 2 weeks. Use lowest fluorescence light output to reduce photobleaching. Make sure exposure time is not too long. Check the viability of the 3D cultures (See Quality Control #3). Check AAV quality and infection efficiency with 2D culture.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>No Ca<sup>2+</sup> transients (major step 52).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Incubate the samples on the stage at 37°C for a longer duration after placing on the stage and before imaging. Check the actual temperature of the culture on the stage to ensure that it is within the range of 36°C–38°C. Overheating can quickly and irreversibly damage the culture. Check multiple 3D cultures. Treat 3D cultures with picrotoxin or bicuculline to induce activity (See Quality Control #2).</p>
<h3 id="sec6.13">Problem 7</h3>
<p>FluoroSNNAP closes immediately after opening (major step 68).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>Run as an administrator. In PC, right click the software, select Run as Administrator.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>How to choose the functional connectivity method (major step 75).</p>
<h3 id="sec6.16">Potential solution</h3>
<p>Given a dataset, different functional connectivity approaches result in different networks. It is important to notice that there is no privileged technique. Each method comes with pros/cons that must be considered by the user, whom should choose based on the type of question to answer and the mathematical meaning of each approach. For further discussion, please refer to “Fundamentals of brain network analysis” (<a href="#bib8">Fornito et al., 2016</a>).</p>
<h3 id="sec6.17">Problem 9</h3>
<p>FluoroSNNAP freezes at the end of functional analysis (major step 76).</p>
<h3 id="sec6.18">Potential solution</h3>
<p>FluoroSNNAP can crash if there is little or no activity to perform network analysis. Check the original time-lapse images to visually determine whether there is enough activity. Run only the “Calcium Events Detection” module in FluoroSNNAP and check the Ca<sup>2+</sup> peaks in the resulting .txt file. Restart the software and/or restart the computer.</p>
<h3 id="sec6.19">Problem 10</h3>
<p>Error while running <i>RunNetworkAnalysis.m.</i> (major step 80).</p>
<h3 id="sec6.20">Potential solution</h3>
<p>Make sure Bioinformatics Toolbox is installed. Check folder directory.</p>
<h3 id="sec6.21">Problem 11</h3>
<p>How to choose a threshold for binarization (major step 82).</p>
<h3 id="sec6.22">Potential solution</h3>
<p>The choice of the binarization threshold is a crucial step since it shapes the resulting network. We can use prior knowledges of the system to determine the noise level of the weight edges between two nodes, thus filtering out irrelevant connections. If unsure about the choice of a threshold, analyze the weighted matrix.</p>
<h3 id="sec6.23">Problem 12</h3>
<p>Negative edges in the functional connectivity matrix. The computation of path length does not admit negative values. In addition, the <i>circularGraph.m</i> will not accept negative entries (major step 83).</p>
<h3 id="sec6.24">Potential solution</h3>
<p>The meaning of negative edge values in the computation of the path length is under debate. The first option is to choose an alternative functional connectivity method to avoid negative edges. In addition, the user can rescale the functional matrix between zero and one. If the code detects negative edges, it will display a dialog box reporting how many there were; moreover, it automatically substitutes them with zero entries (to allow the program to run). Finally, we can take the absolute value of each edge. For a recent survey, please refer to (<a href="#bib9">Hallquist and Hillary, 2019</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Betzel, R.F. and Bassett, D.S. (2017). Multi-scale brain networks. Neuroimage <i>160</i>, 73-83. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref1">View at publisher</a></p>
<p id="bib2">Bullmore, E. and Sporns, O. (2009). Complex brain networks: graph theoretical analysis of structural and functional systems. Nat. Rev. Neurosci. <i>10</i>, 186-198. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref2">View at publisher</a></p>
<p id="bib3">Cairns, D.M., Rouleau, N., Parker, R.N., Walsh, K.G., Gehrke, L., and Kaplan, D.L. (2020). A 3D human brain–like tissue model of herpes-induced Alzheimer’s disease. Sci. Adv. <i>6</i>, eaay8828. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref3">View at publisher</a></p>
<p id="bib4">Cantley, W.L., Du, C., Lomoio, S., DePalma, T., Peirent, E., Kleinknecht, D., Hunter, M., Tang-Schomer, M.D., Tesco, G., and Kaplan, D.L. (2018). Functional and sustainable 3D human neural network models from pluripotent stem cells. ACS Biomater. Sci. Eng. <i>4</i>, 4278-4288. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref4">View at publisher</a></p>
<p id="bib5">Chen, T.W., Wardill, T.J., Sun, Y., Pulver, S.R., Renninger, S.L., Baohan, A., Schreiter, E.R., Kerr, R.A., Orger, M.B., Jayaraman, V., et al. (2013). Ultrasensitive fluorescent proteins for imaging neuronal activity. Nature <i>499</i>, 295-300. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref5">View at publisher</a></p>
<p id="bib6">Dingle, Y.-T.L., Liaudanskaya, V., Finnegan, L.T., Berlind, K.C., Mizzoni, C., Georgakoudi, I., Nieland, T.J.F., and Kaplan, D.L. (2020). Functional characterization of three-dimensional cortical cultures for in vitro modeling of brain networks. iScience <i>23</i>, 101434. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref6">View at publisher</a></p>
<p id="bib7">Fornito, A., Zalesky, A., and Bullmore, E. (2010). Network scaling effects in graph analytic studies of human resting-state fMRI data. Front. Syst. Neurosci. <i>4</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref7">View at publisher</a></p>
<p id="bib8">Fornito, A., Zalesky, A., and Bullmore, E.T. (2016). Fundamentals of Brain Network Analysis (Academic Press). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref8">View at publisher</a></p>
<p id="bib9">Hallquist, M.N. and Hillary, F.G. (2019). Graph theory approaches to functional network organization in brain disorders: A critique for a brave new small-world. Netw Neurosci. <i>3</i>, 1-26. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref9">View at publisher</a></p>
<p id="bib10">Liaudanskaya, V., Chung, J.Y., Mizzoni, C., Rouleau, N., Berk, A.N., Wu, L., Turner, J.A., Georgakoudi, I., Whalen, M.J., Nieland, T.J.F., et al. (2020). Modeling controlled cortical impact injury in 3D brain-like tissue cultures. Adv. Healthc. Mater. <i>9</i>, 2000122. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref10">View at publisher</a></p>
<p id="bib11">Messé, A. (2020). Parcellation influence on the connectivity-based structure–function relationship in the human brain. Hum. Brain Mapp. <i>41</i>, 1167-1180. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref11">View at publisher</a></p>
<p id="bib12">Muldoon, S.F., Bridgeford, E.W., and Bassett, D.S. (2016). Small-world propensity and weighted brain networks. Sci. Rep. <i>6</i>, 22057. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref12">View at publisher</a></p>
<p id="bib13">Newman, M.E. (2006). Modularity and community structure in networks. Proc. Natl. Acad. Sci. U S A <i>103</i>, 8577-8582. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref13">View at publisher</a></p>
<p id="bib14">Onnela, J.-P., Saramäki, J., Kertész, J., and Kaski, K. (2005). Intensity and coherence of motifs in weighted complex networks. Phys. Rev. E <i>71</i>, 065103. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref14">View at publisher</a></p>
<p id="bib15">Patel, T.P., Man, K., Firestein, B.L., and Meaney, D.F. (2015). Automated quantification of neuronal networks and single-cell calcium dynamics using calcium imaging. J. Neurosci. Methods <i>243</i>, 26-38. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref15">View at publisher</a></p>
<p id="bib16">Rockwood, D.N., Preda, R.C., Yucel, T., Wang, X., Lovett, M.L., and Kaplan, D.L. (2011). Materials fabrication from <i>Bombyx mori</i> silk fibroin. Nat. Protoc. <i>6</i>, 1612-1631. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref16">View at publisher</a></p>
<p id="bib17">Rouleau, N., Cantley, W.L., Liaudanskaya, V., Berk, A., Du, C., Rusk, W., Peirent, E., Koester, C., Nieland, T.J.F., and Kaplan, D.L. (2020). A long-living bioengineered neural tissue platform to study neurodegeneration. Macromol. Biosci. <i>20</i>, 2000004. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref17">View at publisher</a></p>
<p id="bib18">Tang-Schomer, M.D., White, J.D., Tien, L.W., Schmitt, L.I., Valentin, T.M., Graziano, D.J., Hopkins, A.M., Omenetto, F.G., Haydon, P.G., and Kaplan, D.L. (2014). Bioengineered functional brain-like cortical tissue. Proc. Natl. Acad. Sci. U S A <i>111</i>, 13811-13816. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30279-3/sref18">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, David L. Kaplan (<a href="mailto:david.kaplan@tufts.edu">david.kaplan@tufts.edu</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and Code Availability</h4>
<p>The MATLAB code RunNetworkAnalysis can be downloaded at <a href="https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture">https://github.com/mattiabonzanni/Integrated-Functional-Neuronal-Network-Analysis-of-3D-Silk-Collagen-Scaffold-based-Cortical-Culture</a></p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (NIH) research grants R01NS092847 and P41EB027062 to D.L.K., and NIH S10 OD021624 and <a href="https://doi.org/10.13039/100000001">National Science Foundation</a> MRI 1531683 research infrastructure grants to Tufts University. The authors would like to thank Michael Lovett, PhD, for help with proofreading and editing and Liam T. Finnegan and Kyler C. Berlind for data analysis.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, Y.-T.L.D., M.B., and T.J.F.N.; methodology, Y.-T.L.D., M.B., and V.L.; software, M.B.; formal analysis, Y.-T.L.D. and M.B.; investigation, Y.-T.L.D. and M.B.; writing, Y.-T.L.D. and M.B.; visualization, Y.-T.L.D., M.B., V.L., and T.J.F.N; supervision, T.J.F.N. and D.L.K; funding acquisition, D.L.K.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>