<article>
<section>
<h1 id="header">An optimized protocol for rapid, sensitive and robust on-bead ChIP-seq from primary cells</h1>
<p><time datetime="2021-02-26">Published: February 26, 2021</time></p>
<p>Lorane Texari,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn3">5</a>,<a href="#fn4">6</a>,<a href="#cor1">*</a></sup> Nathanael J. Spann,<sup><a href="#aff2">2</a>,<a href="#fn3">5</a></sup> Ty D. Troutman,<sup><a href="#aff1">1</a>,<a href="#fn3">5</a></sup> Mashito Sakai,<sup><a href="#aff2">2</a>,<a href="#fn2">4</a></sup> Jason S. Seidman,<sup><a href="#aff2">2</a></sup> and Sven Heinz<sup><a href="#aff1">1</a>,<a href="#fn5">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Medicine, University of California, San Diego, La Jolla, CA 92093, USA</p>
<p id="aff2"><sup>2</sup>Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla, CA 92093, USA</p>
<p id="fn1"><sup>3</sup>Present address: Department of Analytical Sciences, Nestlé Institute of Food Safety and Analytical Sciences, Nestlé Research, Société des Produits Nestlé SA, Lausanne, Switzerland</p>
<p id="fn2"><sup>4</sup>Present address: Department of Biochemistry &amp; Molecular Biology, Nippon Medical School, 1-1-5 Sendagi, Bunkyo-ku, Tokyo 113-8602, Japan</p>
<p id="fn3"><sup>5</sup>These authors contributed equally</p>
<p id="fn4"><sup>6</sup>Technical contact</p>
<p id="fn5"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:lorane.texari@gmail.com">lorane.texari@gmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sheinz@health.ucsd.edu">sheinz@health.ucsd.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100358">10.1016/j.xpro.2021.100358</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Integrative analysis of next-generation sequencing data can help understand disease mechanisms. Specifically, ChIP-seq can illuminate where transcription regulators bind to regulate transcription. A major obstacle to performing this assay on primary cells is the low numbers obtained from tissues. The extensively validated ChIP-seq protocol presented here uses small volumes and single-pot on-bead library preparation to generate diverse high-quality ChIP-seq data. This protocol allows for medium-to-high-throughput ChIP-seq of low-abundance cells and can also be applied to other mammalian cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib3">Brigidi et al. (2019)</a>, <a href="#bib4">Carlin et al. (2018)</a>, <a href="#bib10">Heinz et al. (2018)</a>, <a href="#bib16">Nott et al. (2019)</a>, <a href="#bib18">Sakai et al. (2019)</a>, and <a href="#bib20">Seidman et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Cost-effective rapid, high-quality ChIP-seq prep (∼$40/library)</li>
<li>8-well tube strip format for medium- to high-throughput ChIP-seq</li>
<li>Wide input range (0.1–5 million cells per IP)</li>
<li>Recommendations for antibody choice, assay optimization, and troubleshooting</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/491-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<ol>
<li>The presented protocol is applicable for a wide range of input materials, including sorted cells and nuclei from primary tissues, and cell lines from cell culture.</li>
<li>An important requirement for successful ChIP-seq from tissue samples is preservation of protein integrity. This can be achieved by either snap-freezing tissues immediately after resection, by perfusing organs with fixative solution prior to freezing, or disrupting the tissue and sorting nuclei, as detailed in the protocol by (<a href="#bib22">Troutman et al., 2021</a>).</li>
</ol>
<h3 id="sec1.1">Experimental considerations</h3>
<p>This protocol details an improved method for generating high-quality ChIP-seq libraries from a wide range of input material, including primary cell populations or nuclei purified by fluorescence-activated cell sorting and cell lines. ChIP-seq libraries are generated in a one-pot reaction while the immunoprecipitated DNA is attached to magnetic Dynabeads via antibodies binding to their DNA-crosslinked protein targets. This avoids having to isolate the low amounts of ChIPped DNA before library prep, which can be a source of sample loss. In this regard, the protocol resembles ChIPmentation, where the library is produced on-beads with Tn5 transposase (<a href="#bib19">Schmidl et al., 2015</a>). While the Tn5 approach allows for very low input amounts, the DNA:Tn5 ratio has to be carefully controlled to yield reproducible results. In contrast, the protocol presented here uses the traditional blunt/A-tail/adapter ligation approach to robustly and reliably generate high-quality DNA libraries from a wide range of cell numbers (100,000–10 × 10<sup>6</sup> cells).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>ChIP-grade antibody (either confirmed by ChIP or IHC-P, PFA-fixed FACS, CyTOF); this antibody against histone H3 lysis 27 acetylation (H3K27ac, active enhancer/promoter mark) is a robust positive control</td>
<td>Active Motif</td>
<td>Cat# 39133; RRID: AB_2561016</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Agarose</td>
<td>Fisher Scientific</td>
<td>BP160-500</td>
</tr>
<tr>
<td>0.5 M EGTA, pH 8.0, DNase and RNase-free (ethyleneglycol-bis(2-aminoethylether)-N,N,N′,N′-tetraacetic acid)</td>
<td>bioWORLD</td>
<td>40121266-3</td>
</tr>
<tr>
<td>0.5 M UltraPure EDTA, pH 8.0 (ethylenediaminetetraacetic acid)</td>
<td>Thermo Fisher</td>
<td>15575020</td>
</tr>
<tr>
<td>10% Tween 20</td>
<td>Teknova</td>
<td>T0027; CAS: 900564-5</td>
</tr>
<tr>
<td>10% UltraPure sodium dodecyl sulfate (SDS)</td>
<td>Invitrogen</td>
<td>24730-020</td>
</tr>
<tr>
<td>1 M 4-(2-Hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES)</td>
<td>Teknova</td>
<td>H1090</td>
</tr>
<tr>
<td>0.1 M Dithiothreitol (DTT)</td>
<td>Promega</td>
<td>P1171; CAS: 3483-12-3</td>
</tr>
<tr>
<td>1 M Magnesium chloride (MgCl<sub>2</sub>)</td>
<td>Millipore Sigma</td>
<td>63069</td>
</tr>
<tr>
<td>1 M UltraPure Tris-HCl pH 7.5</td>
<td>Invitrogen</td>
<td>15567-027</td>
</tr>
<tr>
<td>1 M UltraPure Tris-HCl pH 8.0</td>
<td>Invitrogen</td>
<td>15568-025</td>
</tr>
<tr>
<td>2 M Potassium chloride (KCl)</td>
<td>Teknova</td>
<td>P0330</td>
</tr>
<tr>
<td>5 M Sodium chloride (NaCl)</td>
<td>Thermo Fisher</td>
<td>AM9759</td>
</tr>
<tr>
<td>8 M Lithium chloride</td>
<td>Millipore Sigma</td>
<td>L7026</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Fisher Scientific</td>
<td>BP9706-100</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Millipore Sigma</td>
<td>D8418</td>
</tr>
<tr>
<td>Disuccinimidyl glutarate (DSG)</td>
<td>ProteoChem</td>
<td>c1104-100mg; CAS: 79642-50-5</td>
</tr>
<tr>
<td>Dynabeads Protein A</td>
<td>Thermo Fisher Scientific</td>
<td>10002D</td>
</tr>
<tr>
<td>Dynabeads Protein G</td>
<td>Thermo Fisher Scientific</td>
<td>10004D</td>
</tr>
<tr>
<td>Ethanol, 100%</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Formaldehyde</td>
<td>Thermo Fisher Scientific</td>
<td>BP531-500; CAS: 7732-18-5, 50-00-0, 67-56-1</td>
</tr>
<tr>
<td>GelGreen (10,000×)</td>
<td>Biotium</td>
<td>41004</td>
</tr>
<tr>
<td>Glycine</td>
<td>Fisher Scientific</td>
<td>BP381-5</td>
</tr>
<tr>
<td>IGEPAL CA-630</td>
<td>Millipore Sigma</td>
<td>I8896</td>
</tr>
<tr>
<td>Isopropanol</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Phenylmethylsulfonyl fluoride (PMSF)</td>
<td>Millipore Sigma</td>
<td>P7626</td>
</tr>
<tr>
<td>Phosphate-buffered saline (PBS)</td>
<td>Millipore Sigma</td>
<td>806552-500ML</td>
</tr>
<tr>
<td>Polyethylene glycol average mol. Wt. 8,000</td>
<td>Millipore Sigma</td>
<td>P5413</td>
</tr>
<tr>
<td>Protease inhibitor cocktail (PIC)</td>
<td>Sigma-Aldrich</td>
<td>P8340</td>
</tr>
<tr>
<td>Proteinase K (20 mg/mL)</td>
<td>New England Biolabs</td>
<td>P8107S</td>
</tr>
<tr>
<td>PureLink RNase A (20 mg/mL)</td>
<td>Invitrogen</td>
<td>12091021</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>Millipore Sigma</td>
<td>30970</td>
</tr>
<tr>
<td>SpeedBeads magnetic carboxylate-modified particles</td>
<td>GE Healthcare</td>
<td>65152105050250</td>
</tr>
<tr>
<td>Tris-EDTA (TE) buffer, pH 8.0</td>
<td>Invitrogen</td>
<td>AM9849</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Millipore Sigma</td>
<td>T8787</td>
</tr>
<tr>
<td>UltraPure DNase/RNase-free distilled water</td>
<td>Invitrogen</td>
<td>10977023</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>NEBNext Ultra II DNA library preparation kit</td>
<td>NEB</td>
<td>E7645L</td>
</tr>
<tr>
<td>NEXTflex DNA barcodes – 48 (or similar, see <a href="#sec2.4">sequencing adaptors</a> section below)</td>
<td>Bioo Scientific</td>
<td>NOVA-514104</td>
</tr>
<tr>
<td>Qubit dsDNA HS assay kit</td>
<td>Invitrogen</td>
<td>Q32851</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Solexa 1GA (PCR primer, typically provided with the adapters; if more needed, order as desalted oligonucleotide:) AATGATACGGCGACCACCGA</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>Solexa 1GB (PCR primer, typically provided with the adapters; if more needed, order as desalted oligonucleotide:) CAAGCAGAAGACGGCATACGA</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Bowtie 2</td>
<td><a href="#bib13">Langmead and Salzberg, 2012</a></td>
<td><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td>
</tr>
<tr>
<td>HOMER (and associated programs, see website)</td>
<td><a href="#bib9">Heinz et al., 2010</a></td>
<td><a href="http://homer.ucsd.edu/homer">http://homer.ucsd.edu/homer</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>0.2 mL PCR strip</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>0.45 μm pore size syringe filter, PES</td>
<td>Millipore Sigma</td>
<td>SLHP033RS</td>
</tr>
<tr>
<td>1.5 mL DNA LoBind tubes</td>
<td>Eppendorf</td>
<td>022431021</td>
</tr>
<tr>
<td>15 mL conical tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>50 mL conical tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>10 mL disposable syringe</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>50 mL disposable syringe</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>DynaMag-2 magnet (or similar)</td>
<td>Invitrogen</td>
<td>12321D</td>
</tr>
<tr>
<td>DynaMag-96 side magnet (or similar)</td>
<td>Invitrogen</td>
<td>12331D</td>
</tr>
<tr>
<td>Gel electrophoresis, Enduro Gel XL (or similar)</td>
<td>Labnet</td>
<td>E0160</td>
</tr>
<tr>
<td>microTUBE AFA fiber crimp-cap 6 ×16 mm</td>
<td>Covaris</td>
<td>520052</td>
</tr>
<tr>
<td>Micropipettes and sterile DNase/RNase-free filter tips</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Pipette aid</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Qubit dsDNA HS assay kit</td>
<td>Invitrogen</td>
<td>Q32851</td>
</tr>
<tr>
<td>Qubit fluorometer (or similar)</td>
<td>Invitrogen</td>
<td>N/A</td>
</tr>
<tr>
<td>Refrigerated centrifuges (or similar)</td>
<td>Eppendorf</td>
<td>5427R and 5810R</td>
</tr>
<tr>
<td>Serological pipettes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>E220 focused-ultrasonicator (or alternative, see <a href="#sec2.5">sonicator</a> section below)</td>
<td>Covaris</td>
<td>500239</td>
</tr>
<tr>
<td>Vortex Genie 2 (or similar)</td>
<td>Scientific Industries</td>
<td>SI-025</td>
</tr>
<tr>
<td>Optional: 96 well plate rack (fits DynaMag-96 magnet)</td>
<td>Axygen</td>
<td>R-96-PCR-FSP</td>
</tr>
<tr>
<td>Optional: TapeStation 2200</td>
<td>Agilent</td>
<td>G2991AA</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">ChIP-grade antibodies</h3>
<p>ChIP-seq crucially depends on the antibody used for the immunoprecipitation: it must recognize its epitope under conditions where fixatives have chemically modified primary amines. Therefore, antibodies that have previously worked in ChIP-seq are most likely to generate favorable outcomes. If no such antibodies are available, antibodies that work in other techniques that use similar fixation conditions, i.e., immunohistochemistry of paraffin-embedded sections (IHC-P) or immunocytochemistry, flow cytometry, or CyTOF of paraformaldehyde-fixed cells are likely to work. When choosing between different candidates, antibodies raised against immunogenic peptides that do not contain lysines are preferable. A simple way to screen multiple antibodies is to compare their immunoprecipitation efficiency on fixed versus unfixed samples by Western blot. This can be accomplished by using the ChIP protocol presented below on the same number of fixed and unfixed cells/nuclei and boiling the Dynabeads in Laemmli buffer for 10 min for SDS-PAGE and Western blot, instead of preparing ChIP-seq libraries for sequencing.</p>
<h3 id="sec2.2">Fixation of proteins to DNA</h3>
<p>The linked protocol “Purification of Hepatic Non-Parenchymal Cells or Nuclei for Use in ChIP-Seq and other Next-Generation Sequencing Approaches” provides fixed cells for ChIP-seq according to the protocol below, starting at Step 12. Protocols for single and double crosslinking of tissue culture cells are provided in steps 1–11. Generally, the zero-distance crosslinker formaldehyde is used to crosslink proteins to their DNA binding sites, as well as to directly interacting proteins. Additional fixation of protein-protein interactions with protein-protein crosslinkers with a longer spacer arm, such as disuccinimidyl glutarate (DSG) or ethylene glycol bis (succinimidyl succinate) (EGS), can help increase ChIP yield of dimeric transcription factors (e.g., AP-1) and non-DNA-binding proteins (e.g., BRG1) (<a href="#bib15">Nowak et al., 2005</a>; <a href="#bib21">Zeng et al., 2006</a>). Therefore, it is advisable to compare formaldehyde fixation and “double crosslinking” when testing new antibodies against dimeric or non-DNA-binding proteins using cell lines that express the proteins of interest.</p>
<h3 id="sec2.3">Next-generation sequencing platform</h3>
<p>The protocol presented here generates libraries suitable for Illumina sequencers. Consult with your sequencing provider or core facility about their sample requirements before you start.</p>
<h3 id="sec2.4">Sequencing adapters</h3>
<p>This protocol uses T-tailed, Y-shaped double-stranded DNA sequencing adapters that require A-tailing of the double-stranded DNA prior to their ligation. The adapter amount is optimized for 30 ng of insert DNA with an average size of 250 bp. It will generate libraries from up to 150 ng DNA, however with reduced yields due to lack of adapters to ligate to both ends of the DNA fragments. Suitable adapters can be purchased from multiple vendors, including Bioo/Perkin Elmer (see <a href="#materials-and-equipment">materials and equipment</a>), Illumina, NEB, and Roche. We prefer adapters that already contain index sequences that allow to multiplex several samples on the same sequencing lane. If other adapters such as hairpin or stubby adapters are used, the protocol will require additional PCR primers that contain the index sequences, which can be purchased from the supplier of the adapters or IDT. It is most cost-effective to purchase sequencing adapters for genomic DNA sequencing, which generally starts with higher DNA and adapter amounts, and to dilute the adapters to 0.5 μM final concentration with a buffer that preserves their double-stranded nature. We have found that 1× T4 DNA ligase buffer (50 mM Tris-HCl, 10 mM MgCl<sub>2</sub>, 1 mM ATP, 10 mM DTT, pH 7.5 at 25°C) serves this purpose well and maintains adapter function for at least 10 freeze-thaw cycles.</p>
<h3 id="sec2.5">Sonicator</h3>
<p>Many suitable sonication platforms can be used for ChIP-seq (examples: small-volume probe sonicators, e.g., Branson 450 or QSonica Q125 [both with 1/8 inch tip]), Diagenode Bioruptor Pico, Active Motif PIXUL, Covaris M220). DNA fragmentation conditions should be optimized in an analogous manner as described for the Covaris E220 for each platform. In our hands, the Covaris and PIXUL sonicators yield the most reproducible results. The sonication procedure described herein uses a Covaris E220. For optimization purposes, after 6, 10, 14, 18, 22, 24 cycles of Covaris sonication the sonicator can be paused and 5 μL aliquots taken out of the sonication tube. The extracted DNA can be isolated (as described for the “ChIP input” samples, steps 26–36) and run on a 2% agarose gel (use GelGreen or SYBR Gold for sensitive DNA detection) or TapeStation to determine the fragment size profile. The cycle number yielding a majority of fragments around 250–300 bp should be used to sonicate the ChIP samples (see <a href="#fig1">Figure 1</a>). Further optimization can be performed with more fine-grained cycle intervals if desired.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/491-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Effect of increasing chromatin sonication time on DNA fragment size</div>
<p>DSG/formaldehyde-crosslinked Kupffer cell nuclei were sonicated in Lysis Buffer in a Covaris E220 focused-ultrasonicator for the indicated numbers of cycles as detailed in Step 19a, DNA isolated as described for “ChIP inputs” and analyzed on an Agilent TapeStation 2200.</p>
</figcaption>
</figure>
<p>Probe (or “horn”) sonicators can work equally well, however this requires proper mounting of the sample tube and sonicator probe, an uncorroded sonication probe (replace sonication probe if probe tip is not smooth and shiny) and permanent cooling of the sample on wet ice. Because the sonicator probe needs to be immersed deeply enough in buffer to prevent foaming, the sample is typically sonicated in 400–500 μL Lysis Buffer. Accurate placement of the probe tip at about 3 mm from the bottom of the 1.5 mL microcentrifuge tube without touching the tube wall is essential for reproducible sonication and ChIP-seq results. Sonication power of probe sonicators needs to be adjusted such that power output is high enough to efficiently sonicate chromatin/DNA to around 250–300 bp average DNA fragment size but low enough to preventing foaming. As an example, with a Misonix XL-2000 probe sonicator (similar to a QSonica Q125), this is achieved by sonicating at 11–12 W power output for 8–16 cycles of 10 s on, 30 s off, with cells suspended in 500 μL lysis buffer in a 1.5 mL Eppendorf LoBind tube. For reproducible sonication, the tube is fixed in position in a clamp underneath the sonicator and immersed to half of the tube’s length in a 200 mL beaker filled to the top with wet ice slush.</p>
<p>The number of cycles required to achieve proper sonication depends on the fixation conditions and the detergents in the lysis buffer: DSG/formaldehyde-double-crosslinked cells need several additional sonication cycles compared to formaldehyde-fixed cells to properly shear the DNA to the right size range. Lysis buffers with more non-ionic than ionic detergent, such as the RIPA buffer described in this protocol, will require up to double the number of cycles compared to buffers containing higher amounts of ionic detergents, such as SDS or lauroyl sarcosine. For the latter, it is possible to shear chromatin where almost all DNA fragments smaller than 500 bp. In contrast, lysis buffers containing high concentrations of non-ionic detergents such as Triton X-100 will lead to a high proportion of larger (&gt;1 kb) DNA fragments, regardless of how long the sample is sonicated. This is not a problem as long as there is also a significant amount of smaller DNA generated in the sonication process (at a minimum 30%–50% of the total DNA amount).</p>
<h3 id="sec2.6">Buffers and solutions</h3>
<div class="critical">
<span class="critical-title">Critical:</span> Many chemicals used in this protocol are hazardous and/or toxic. Institutional safety guidelines should be followed for safe use and disposal.</div>
<h3 id="sec2.7">BSA (10%)</h3>
<p>Place 5 grams of BSA in a 50 mL conical tube and add 20°C–25°C UltraPure DNase/RNase-Free Distilled Water to the 50 mL mark. Mix (e.g., rocking, rotation) at 20°C–25°C until dissolved and adjust the final volume to 50 mL with 20°C–25°C UltraPure DNase/RNase-Free Distilled Water. Sterile filter into a fresh 50 mL conical with a 50 mL disposable syringe and attached 0.45 μm sterile filter. Store 10 mL aliquots at −20°C for up to 1 year, keep a working aliquot at 4°C for up to 3 months.</p>
<h3 id="sec2.8">Ethanol (80%)</h3>
<p>Prepare 40 mL of 80% ethanol by mixing 8 mL UltraPure water with 32 mL ethanol. Store at 20°C–25°C.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Ethanol at this concentration is flammable and should be handled and discarded appropriately.</div>
<h3 id="sec2.9">Glycine (2.625 M)</h3>
<p>Place 9.85 g glycine in a 50 mL conical and add 20°C–25°C UltraPure DNase/RNase-Free Distilled Water to the 50 mL mark. Mix (e.g., rocking, rotation) at 20°C–25°C until dissolved and adjust the final volume to 50 mL with 20°C–25°C UltraPure DNase/RNase-Free Distilled Water. Store at 20°C–25°C.</p>
<h3 id="sec2.10">IGEPAL CA-630 (10%)</h3>
<p>Prepare 40 mL 10% IGEPAL CA-630 by adding 10 mL (using a 10 mL syringe) of the detergent to 35 mL UltraPure water in a 50 mL conical. Close the tube, seal with parafilm and rock 12–16 h at 20°C–25°C. Adjust to a final 50 mL volume using UltraPure water and store at 4°C for up to 1 year.</p>
<h3 id="sec2.11">Low-EDTA TE</h3>
<table id="undtbl1">
<caption>Prepare 10 mL Low-EDTA TE using reagents at 20°C–25°C store at 4°C for up to 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-Free distilled water</td>
<td> </td>
<td>9.9 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 8.0</td>
<td>10 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.1 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.12">Lysis buffer</h3>
<table id="undtbl2">
<caption>Prepare 39.2 mL Lysis Buffer stock using reagents at 20°C–25°C (omit DTT, PIC, PMSF), store buffer stock at 4°C for up to 6 months.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-free distilled water</td>
<td> </td>
<td>31.1 mL</td>
</tr>
<tr>
<td>1 M Tris-HCL pH 7.5</td>
<td>20 mM</td>
<td>800 μL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>150 mM</td>
<td>1,200 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>1 mM</td>
<td>80 μL</td>
</tr>
<tr>
<td>0.5 M EGTA</td>
<td>0.5 mM</td>
<td>40 μL</td>
</tr>
<tr>
<td>10% Sodium deoxycholate</td>
<td>0.4%</td>
<td>1,600 μL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>0.1%</td>
<td>400 μL</td>
</tr>
<tr>
<td>10% IGEPAL CA-630</td>
<td>1%</td>
<td>4,000 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>39.2 mL</b></td>
</tr>
<tr>
<td colspan="3"><i>Add fresh to each 97.5 μL Lysis Buffer stock per sample (details see step 2):</i></td>
</tr>
<tr>
<td>0.1 M DTT</td>
<td>0.5 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>100× PIC</td>
<td>1×</td>
<td>1 μL</td>
</tr>
<tr>
<td>100× PMSF</td>
<td>1×</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare complete Lysis Buffer immediately before use by adding PMSF (100 mM), PIC (100×) and DTT (0.1 M) to the appropriate volume of Lysis Buffer (1 mL final/sample), see Step 2 for details.</div>
<h3 id="sec2.13">PMSF (100 mM, 100×)</h3>
<p>Prepare 100 mM PMSF (100×) by dissolving 0.5 g PMSF in 28.7 mL of isopropanol with heat. Immediately prepare 1 mL aliquots and store at −20°C.</p>
<div class="note">
<span class="note-title">Note:</span> PMSF has a short half-life in aqueous solutions and should be added to buffers &lt; 30 min before use.</div>
<h3 id="sec2.14">Polyethylene glycol 8,000 (40% PEG8000)</h3>
<p>Place 20 grams of polyethylene glycol in a 50 mL conical tube and add 20°C–25°C UltraPure DNase/RNase-Free Distilled Water to the 50 mL mark. Mix (e.g., rocking, rotation) at 20°C–25°C until dissolved and adjust the final volume to 50 mL with 20°C–25°C UltraPure DNase/RNase-Free Distilled Water. Store in the dark at 4°C for 6 months, or in 10 mL aliquots at −20°C long-term.</p>
<div class="note">
<span class="note-title">Note:</span> As all polyethers, PEG8000 is prone to chain shortening by peroxidation, caused by a combination of atmospheric oxygen, light, and temperature. To ensure long-term stable results, ensure storage at or below 4°C.</div>
<h3 id="sec2.15">Polyethylene glycol 8,000/1.5 M NaCl (20% PEG8000/1.5% NaCl)</h3>
<p>In a 50 mL conical tube, mix 25 mL PEG8000 (40%), 15 mL 5 M NaCl and 10 mL UltraPure DNase/RNase-Free Distilled Water. Store in the dark at 4°C for 6 months, or in 10 mL aliquots at −20°C long-term.</p>
<h3 id="sec2.16">Polyethylene glycol 8,000/2.5 M NaCl (20% PEG8000/2.5% NaCl)</h3>
<p>In a 50 mL conical tube, mix 25 mL PEG8000 (40%) and 25 mL 5 M NaCl. Store in the dark at 4°C for 6 months, or in 10 mL aliquots at −20°C long-term.</p>
<h3 id="sec2.17">Resuspension buffer</h3>
<table id="undtbl3">
<caption>Prepare 49 mL Resuspension buffer stock in 50 mL conical at 20°C–25°C (omit PIC, PMSF), store buffer stock at 4°C for up to 6 months.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-free distilled water</td>
<td> </td>
<td>44.05 mL</td>
</tr>
<tr>
<td>1 M HEPES-KOH pH 7.9</td>
<td>9 mM</td>
<td>450 μL</td>
</tr>
<tr>
<td>2 M KCl</td>
<td>76.4 mM</td>
<td>1.91 mL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.9 mM</td>
<td>90 μL</td>
</tr>
<tr>
<td>10% IGEPAL CA-630</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>49 mL</b></td>
</tr>
<tr>
<td colspan="3"><i>Add fresh to 980 μL Resuspension Buffer stock per sample (details see step 1):</i></td>
</tr>
<tr>
<td>100× PIC</td>
<td>1×</td>
<td>10 μL</td>
</tr>
<tr>
<td>100× PMSF</td>
<td>1×</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<p>Freshly prepare complete Resuspension Buffer directly before use by adding PMSF (100 mM) and PIC (100×) to the appropriate volume of Resuspension Buffer (1 mL final /sample), see step 1 for details.</p>
<div class="critical">
<span class="critical-title">Critical:</span> IGEPAL CA-630 is toxic and should be disposed of following approve institutional guidelines.</div>
<h3 id="sec2.18">Sodium deoxycholate (10%)</h3>
<p>Prepare 40 mL 10% sodium deoxycholate by adding 4 g of the powder to 30 mL UltraPure water in a 50 mL conical. Close the tube, seal with parafilm and rock in the dark for 12–16 h at 20°C–25°C. Adjust to a final 40 mL volume using UltraPure water, sterile filter into a fresh 50 mL conical with a 50 mL disposable syringe and attached 0.45 μm sterile filter and store in the dark at 20°C–25°C for up to 1 year.</p>
<h3 id="sec2.19">Triton X-100 (10%)</h3>
<p>Prepare 50 mL 10% Triton X-100 by adding 10 mL of the detergent (use a 10 mL syringe) to 35 mL UltraPure water in a 50 mL conical. Close the tube, seal with parafilm, and rock for 12–16 h at 20°C–25°C. Adjust to a final 50 mL volume using UltraPure water and store at 4°C for up to 1 year.</p>
<h3 id="sec2.20">TE + 0.5% BSA + 0.1% Tween</h3>
<table id="undtbl4">
<caption>Prepare 100 mL TE (Tris-EDTA) + 0.5% BSA (bovine serum albumin) + 0.1% Tween 20 and store at 4°C for 2–3 weeks.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TE Buffer (pH 8.0)</td>
<td> </td>
<td>Add to 100 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>0.5%</td>
<td>0.5 g</td>
</tr>
<tr>
<td>10% Tween 20</td>
<td>0.1%</td>
<td>400 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.21">TET (10 mM Tris, 0.2% Tween 20, 1 mM EDTA)</h3>
<table id="undtbl5">
<caption>Prepare 20 mL Tris-EDTA-Tween (TET) using reagents at 20°C–25°C, then store at 4°C.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-free distilled water</td>
<td> </td>
<td>19.36 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 8.0</td>
<td>10 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>1 mM</td>
<td>40 μL</td>
</tr>
<tr>
<td>10% Tween 20</td>
<td>0.2%</td>
<td>400 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.22">TT (10 mM Tris, 0.05% Tween 20)</h3>
<table id="undtbl6">
<caption>Prepare 10 mL Tris-Tween (TT) using reagents at 20°C–25°C, store at 4°C.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-free distilled water</td>
<td> </td>
<td>9.85 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 8.0</td>
<td>10 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>10% Tween 20</td>
<td>0.05%</td>
<td>50 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td>10 <b>mL</b>
</td>
</tr>
</tbody>
</table>
<h3 id="sec2.23">Wash Buffer 1</h3>
<table id="undtbl7">
<caption>Prepare 49.75 mL Wash Buffer 1 in a 50 mL conical at 20°C–25°C, store buffer stock at 4°C for up to 6 months.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-free distilled water</td>
<td> </td>
<td>41.59 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 7.5</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>8 M LiCl</td>
<td>250 mM</td>
<td>1.56 mL</td>
</tr>
<tr>
<td>10% IGEPAL CA-630</td>
<td>1%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>10% Sodium deoxycholate</td>
<td>0.7%</td>
<td>3.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>49.75 mL</b></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>This protocol was developed specifically for robust ChIP-seq of small cell population such as isolated hepatic macrophages. This framework can easily be extended for other liver non-parenchymal cells (NPCs) including sinusoidal endothelial cells, stellate cells, alternative lineages of immune cells or other low-abundance cell populations such as virus-infected cells and non-infected bystander cells or primary neurons.</p>
<h3 id="sec3.1">Optimized on-bead ChIP-seq</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>Herein is described an improved ChIP-seq library generation protocol used successfully in our labs to generate high-quality libraries for diverse chromatin immunoprecipitation products (<a href="#bib4">Carlin et al., 2018</a>; <a href="#bib10">Heinz et al., 2018</a>; <a href="#bib16">Nott et al., 2019</a>; <a href="#bib18">Sakai et al., 2019</a>; <a href="#bib20">Seidman et al., 2020</a>).</p>
<div class="optional">
<span class="optional-title">Optional:</span> Fix cells</div>
<p>The linked protocol, “Purification of Hepatic Non-Parenchymal Cells or Nuclei,” provides fixed material that can directly be used for ChIP as detailed from step 12 onward. If starting with unfixed cells from tissue culture, first perform steps 1–11 to fix protein-DNA interactions.</p>
<ol>
<li>Harvest cells:
<ol type="a">
<li>If suspension cells: transfer cell suspension to 15 mL or 50 mL conical, depending on volume.</li>
<li>If adherent cells: harvest cells by trypsinizing and neutralize trypsin with 4 volumes of FBS-containing media, transfer cell suspension to 15 mL or 50 mL conical, depending on volume.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Pellet cells by centrifuging for 8 min at 300 × <i>g</i> at 20°C. Aspirate or decant supernatant.
<div class="optional">
<span class="optional-title">Optional:</span> Dual crosslinking with DSG and formaldehyde:</div>
<ol type="a">
<li>While cells are in the centrifuge, for each 1 million cells, freshly prepare a minimum of 1 mL of a 2 mM DSG solution in PBS. To do so, first warm up the glass vial of DSG to 20°C–25°C before opening (to avoid water condensation into the vial). On an analytical scale, pour at least 0.654 mg DSG for each 1 million cells onto a fresh weighing paper, transfer the DSG to a 1.5 mL microcentrifuge tube and dissolve in 3.92 μL anhydrous DMSO per 0.654 mg DSG (adjust the amount of DMSO to the amount of DSG in the tube). Place 1 mL PBS per 1 million cells in a microcentrifuge tube or conical (according to the PBS volume). Using a micropipette, swiftly “shoot” 3.92 μL of the DSG/DMSO solution into the PBS, immediately close the tube, and invert multiple times to rapidly disperse the DSG. If this is done too slowly, the DSG will form a white precipitate, and the DSG/PBS preparation procedure will need to be repeated.</li>
<li>Aspirate or decant supernatant, and wash cells once by resuspending in 10 mL PBS and centrifugation for 8 min at 300 × <i>g</i> at 20°C–25°C.</li>
<li>Aspirate supernatant and resuspend the cell pellet in at least 1 mL DSG/PBS solution per 1 million cells.</li>
<li>Incubate at 20°C–25°C for 30 min while rotating the tube overhead at 8 rpm.</li>
<li>Per 1 million cells, add 27.78 μL 37% formaldehyde to each 1 ml cell suspension in DSG/PBS, invert tube 5–6 times to mix, and incubate for an additional 10 min at 20°C–25°C while rotating the tube overhead at 8 rpm.</li>
<li>Skip to step 5.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>While cells are in the centrifuge, for each 1 million cells, freshly prepare a minimum of 1 mL of a 1% formaldehyde solution in PBS: to each 1 mL PBS, add 27.78 μL 37% formaldehyde.</li>
<li>Resuspend the cell pellet in the 1% formaldehyde/PBS solution, and incubate at 20°C–25°C for 10 min while rotating the tube overhead at 8 rpm.</li>
<li>Quench the crosslinking reaction by adding 1/20<sup>th</sup> volume of 2.625 M glycine and 1/20<sup>th</sup> volume 10% BSA (50 μL each per each 1 mL crosslinking reaction), and mix by inverting 3–4 times.</li>
<li>Pellet cells by centrifuging for 5 min at 1,000–1,500 × <i>g</i> at 4°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Fixed cells are more buoyant than unfixed cells and refractory to the higher centrifugal force needed to pellet them. The added BSA helps prevent the more hydrophobic fixed cells from sticking to the tube wall, which otherwise would lead to sample loss when aspirating the supernatant. If possible, perform the centrifugations in steps 6–11 in a swing bucket rotor to further minimize cell loss, especially when working with less than 1 million cells.</div>
<ol start="7">
<li>Aspirate and discard supernatant, then resuspend fixed cells in 1 mL ice-cold 0.5% BSA/PBS per 1 million cells.</li>
<li>Pellet cells by centrifuging for 5 min at 1,000–1,500 × <i>g</i> at 4°C.</li>
<li>Aspirate and discard supernatant, resuspend fixed cells in 1 mL ice-cold 0.5% BSA/PBS and transfer the cell suspension to a 1.5 mL microcentrifuge tube (if not already in one).</li>
<li>Pellet cells by centrifuging for 5 min at 1,000–1,500 × <i>g</i> at 4°C.</li>
<li>Aspirate and discard supernatant, and place the cell pellet on ice.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> snap-freeze the cell pellet in dry ice/methanol bath or liquid nitrogen and store indefinitely at −80°C.</div>
<h3 id="sec3.2">Prepare necessary buffers for chromatin fragmentation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="12">
<li>Freshly prepare 1 mL complete Resuspension Buffer per sample by adding 10 μL protease inhibitor cocktail (PIC, 100×) and 10 μL PMSF (100×) to 980 μL Resuspension Buffer stock per sample (1× PIC, 1× PMSF final). Mix by vortexing and keep on ice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Add PMSF immediately prior to use!</div>
<ol start="13">
<li>For sonication using a Covaris sonicator, prepare 100 μL complete Lysis Buffer per sample by adding 1 μL PIC (100×) and 1 μL PMSF (100 mM) and 0.5 μL DTT (100 mM) to 97.5 μL Lysis Buffer stock per sample (1× PIC, 1× PMSF, 0.5 mM DTT final). Mix by vortexing and keep on ice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Add PMSF immediately prior to use!</div>
<h3 id="sec3.3">Fragment fixed chromatin by sonication</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h to 1 day, depending on number of samples and method of chromatin fragmentation</div>
<div class="note">
<span class="note-title">Note:</span> This step requires optimization depending on the user’s access to equipment. Here we describe fragmentation using a Covaris E220, which is available at many core facilities. Alternatively, a probe sonicator may be used effectively with proper experience (see <a href="#sec2.5">sonicator</a> section for more details). Many additional platforms are available, and the goal is to sufficiently fragment chromatin to an appropriate size without overheating the sample.</div>
<ol start="14">
<li>Suspend 0.1–5 million fixed cells or nuclei in 1 mL ice-cold Resuspension Buffer.</li>
<li>Pellet at 1,000 × <i>g</i> for 5 min at 4°C.</li>
<li>Carefully discard the supernatant, taking care to not lose material from the cell or nuclei pellet.</li>
<li>Suspend the pellet in 80 μL ice-cold Lysis Buffer and transfer the lysate to Covaris sonication tubes on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If using a probe sonicator, cells will need to be resuspended in a larger volume of Lysis Buffer (400 μL - 500 μL, see <a href="#sec2.5">sonicator</a> section).</div>
<ol start="18">
<li>Load the sonication tubes into the Covaris 96 place microtube holder chromatin by sonication using the Covaris E220.</li>
<li>Load and then run the desired user-optimized fragmentation protocol at 4°C.
<ol type="a">
<li>We have found success for sonication of macrophage nuclei in Lysis Buffer using the following settings: 18 cycles at 60 s, duty factor at 5.0%, peak power of 140 W, cycles per burst 200.</li>
<li>We have found success for sonication of macrophage, microglia and neuron nuclei in alternative lysis and washing buffers described in <a href="#bib3">Brigidi et al., 2019</a>, <a href="#bib6">Eichenfield et al., 2016</a>, <a href="#bib8">Gosselin et al., 2014</a>, and <a href="#bib10">Heinz et al., 2018</a>. Changes in sonication buffers require optimization of sonication conditions.</li>
</ol>
</li>
</ol>
<ol start="20">
<li>While the sonication protocol is running, prepare the antibody-bead complexes.
<ol type="a">
<li>Transfer a total of 10–20 μL protein A and/or protein G Dynabeads per sample into a microcentrifuge tube.
<div class="note">
<span class="note-title">Note:</span> The choice of protein A and/or protein G Dynabeads, and the volume to use, is dictated by the antibody used for ChIP. Protein A binds rabbit antibodies with highest affinity, while protein G has higher affinity for mouse, rat, goat, and sheep antibodies. We have found success in coupling 2 μg of a validated ChIP antibody with 20 μL of magnetic beads. If the ChIP method utilizes a cocktail of antibodies, we have found success using 1–2 μg of each antibody with a total volume of 20 μL of magnetic beads.</div>
<div class="note">
<span class="note-title">Note:</span> For many difficult or low complexity ChIP-seqs, we have found that optimization and utilization of antibody cocktails (mixture of multiple antibodies recognizing different epitopes for an individual ChIP target) have provided both optimal signal:noise and peak complexity. Sometimes this approach has proved to be the difference between many difficult ChIP-seqs working or not. Refer to the <a href="#troubleshooting">troubleshooting</a> section for more details.</div>
</li>
<li>Wash the required bead volume twice with 1 mL of TE + 0.5% BSA + 0.1% Tween 20 by collection on an Dynamag-2 Magnet (or similar).</li>
<li>Suspend the washed magnetic beads in 200 μL TE + 0.5% BSA + 0.1% Tween 20 and add the necessary antibodies. Incubate with rotation for about 30 min at 20°C–25°C, then rotate at 4°C until the sonications are complete.</li>
<li>Collect the antibody-bound magnetic beads on the magnet and discard the supernatant. Add 100 μL Lysis Buffer per sample, mix well by pipetting 20 times.</li>
</ol>
</li>
</ol>
<ol start="21">
<li>Recover sonicated chromatin on ice and transfer to a 1.5 mL DNA LoBind tubes.</li>
<li>Centrifuge for 10 min at more than 16,000 × <i>g</i> at 4°C.</li>
<li>Transfer the supernatant into a 0.2-mL PCR tube strip.</li>
<li>Set aside 1% of the sonication product (for 0.5 to 1 million cells, or 1 – 2 μL for larger cell numbers) for each sample to use as the “ChIP input.” Store at 4°C for up to 2 days.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not forget to set aside sample prior to immunoprecipitation for the “ChIP input,” which is used to identify ChIP-seq signal enriched over background.</div>
<div class="note">
<span class="note-title">Note:</span> Different cell lines may have copy number variants, and sonication efficiency of hetero- and euchromatin does change slightly with different sonication conditions and lysis buffers. A minimum of one “ChIP input” sample per cell type and sonication condition is required to control for differences between cell types, and sonication efficiency with different fixation and lysis conditions. “ChIP input” samples generated from the same cells stimulated with various stimuli typically do not differ significantly from each other. Therefore, a single “ChIP input” sample is usually sufficient to serve as control for multiple ChIP experiments done in the same cell type and with the same fixation/lysis/sonication conditions.</div>
<ol start="25">
<li>Distribute antibody cocktails to the tube strips and rotate overnight (up to 16 h) at 4°C with gentle rotation.</li>
</ol>
<h3 id="sec3.4">Prepare input DNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5–18 h</div>
<ol start="26">
<li>Bring the “ChIP inputs” set aside in step 24 to 46.5 μL final volume with UltraPure water at 4°C.</li>
<li>Freshly prepare the following reaction mix at 20°C–25°C. Add 29 μL to the 46.5 μL “ChIP inputs.”</li>
</ol>
<table id="undtbl8">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-Free Distilled Water</td>
<td> </td>
<td>18.9 μL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>0.5%</td>
<td>4 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>18.75 mM</td>
<td>3 μL</td>
</tr>
<tr>
<td>0.5 M EGTA</td>
<td>10</td>
<td>1.6 μL</td>
</tr>
<tr>
<td>20 mg/mL Proteinase K</td>
<td>250 μg/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>20 mg/mL RNase A</td>
<td>125 μg/mL</td>
<td>0.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>29 μL</b></td>
</tr>
</tbody>
</table>
<ol start="28">
<li>Add 4.5 μL 5 M NaCl to each “ChIP input,” mix by gentle flicking, then collect the reaction contents to the tube bottom by quickly centrifuging.</li>
<li>Incubate at 55°C for 1 h (in a thermal cycler with the heated lid set to 75°C) to digest proteins and RNA, then shift the incubation temperature to 65°C for at least 2 h (up to 16 h) to reverse the crosslinks.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure the thermal cycler’s heated lid is set to 75°C to prevent evaporation and drying out of the samples.</div>
<ol start="30">
<li>Per sample, combine 2 μL SpeedBeads with 123 μL 20% PEG8000/1.5 M NaCl to make 125 μL SpeedBead/PEG mix per sample in the form of a master mix, make 10% excess.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Completely resuspend the SpeedBeads by thorough vortexing before use.</div>
<ol start="31">
<li>Add 125 μL of SpeedBead/PEG master mix to each 80 μL ChIP input sample. The final concentration of PEG will be 12% and the final concentration of NaCl will be 1 M.</li>
<li>Mix the samples to homogeneity by vortexing or repetitive pipetting. Incubate at 20°C–25°C for 10 min.</li>
<li>Apply the tube strips to a magnet and aspirate the cleared supernatant. Be careful to not lose the magnetic beads which have bound the “ChIP input” DNA.</li>
<li>Wash the beads by adding 180 μL of 80% ethanol at 20°C–25°C and moving the strips to either side of the magnet six times. Collect the beads on the magnet and discard the supernatant once cleared. Repeat this wash one additional time.</li>
<li>After removing the second ethanol wash, cap then pulse-centrifuge the tube strip, attract the beads to the magnet, then aspirate the residual ethanol volume using a P10 pipettor and tip. Air-dry the beads for ∼5 min, or until “cracks” just begin to appear in the packed beads.</li>
<li>Elute the “ChIP input” DNA by adding 25 μL TT Buffer. Vortex then incubate at 20°C–25°C for 5–10 min. Pulse-centrifuge, then apply the tube strips to the magnet. Transfer the supernatant to a new PCR strip and discard the SpeedBeads.</li>
<li>Store the “ChIP input” DNA at 4°C until ChIP-seq and ChIP Input End Repair and Adapter Ligation (Step 54).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Immunoprecipitation can be carried out for up to 16 h with “ChIP input” samples safely kept at 4°C.</div>
<h3 id="sec3.5">Prepare ChIP wash buffers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="38">
<li>Freshly prepare 1.2 mL complete ChIP Wash Buffer 1 per sample by adding 6 μL PIC (100×) to 1,194 μL ChIP Wash Buffer 1 stock per sample (0.5× PIC final). Mix by vortexing and keep on ice.</li>
<li>Freshly prepare 600 μL complete TET Buffer per sample by adding 3 μL PIC (100×) to 557 μL TET Buffer per sample (0.5× PIC final). Mix by vortexing and keep on ice.</li>
<li>Freshly prepare 200 μL complete Low-EDTA TE per sample by adding 1 μL PIC (100×) to 199 μL Low-EDTA TE per sample (0.5× PIC). Mix by vortexing and keep on ice.</li>
<li>Freshly prepare 0.6 mL complete Lysis Buffer per sample by adding 6 μL PIC (100×) and 6 μL PMSF (100 mM) and 3 μL DTT (100 mM) to 585 μL Lysis Buffer stock per sample (1× PIC, 1× PMSF, 0.5 mM DTT final). Mix by vortexing and keep on ice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Add PMSF immediately prior to use!</div>
<h3 id="sec3.6">Wash ChIP samples</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<div class="note">
<span class="note-title">Note:</span> Store wash buffers on ice during use. Samples should be stored on ice in between steps.</div>
<ol start="42">
<li>Briefly (1–3 s) centrifuge (in a bench centrifuge) ChIP samples in PCR strip to collect material to the bottom of the tube.</li>
<li>Collect immunoprecipitation product using a DynaMag-96 Side Magnet (or similar).</li>
<li>Carefully aspirate and discard the supernatant.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Be very careful to not aspirate any beads.</div>
<ol start="45">
<li>Remove the tube strip from the magnet and resuspend beads 180 μL Lysis Buffer from Step 41.</li>
<li>Transfer the sample to a new tube to reduce background.</li>
<li>Place the new tube strip next to the magnet. Once the beads are collected, aspirate the supernatant.</li>
<li>Resuspend beads in another 180 μL Lysis Buffer by shifting the tube strip back and forth 5 times between magnet strips to move beads from one side to the other.</li>
<li>Collect beads on magnet, aspirate supernatant, and repeat this wash (as in Step 48) once more.</li>
<li>Wash the beads 6× with 180 μL ChIP Wash Buffer 1 from Step 38 (as in Steps 48–49).</li>
<li>Wash the beads 3× with 180 μL TET prepared in Step 39. Transfer the beads to a new tube strip after resuspending the beads for the 3<sup>rd</sup> wash, which reduces lithium carryover.</li>
<li>Wash the beads once with 180 μL Low-EDTA TE prepared in Step 40.</li>
<li>Resuspend the beads in 25 μL TT. The samples are now ready for on-bead library preparation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is not recommended to pause at this point as we have not tested it.</div>
<h3 id="sec3.7">ChIP-seq and ChIP input end repair and adapter ligation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<div class="note">
<span class="note-title">Note:</span> All buffers should be stored on ice during use. Samples should be stored on ice between steps.</div>
<ol start="54">
<li>Freshly prepare End Prep Mix on ice. Each reaction requires 5 μL and it is recommended to prepare 1.1 reaction volumes per sample. Carefully mix by repeated pipetting &gt;50% of the total reaction volume. Collect the contents by brief centrifugation.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not forget to also prepare libraries from the “ChIP input” DNA samples from Step 37!</div>
<table id="undtbl9">
<thead>
<tr>
<th>Reagent</th>
<th>Amount for each reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>NEBNext Ultra II End Prep Enzyme Mix</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>NEBNext Ultra II End Prep Reaction Buffer</td>
<td>3.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>5 μL</b></td>
</tr>
</tbody>
</table>
<ol start="55">
<li>On ice, add 5 μL End Prep Mix to each sample. Set a P20 micropipette to 20 μL and gently pipette the entire volume up and down at least 10 times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is important to mix well. The presence of a small amount of bubbles will not interfere with performance.</div>
<ol start="56">
<li>Incubate for 30 min at 20°C, then 30 min at 65°C, then hold at 4°C in a thermal cycler with the lid set to 75°C.</li>
<li>Freshly prepare the Ligation Mix on ice. Each reaction requires 15.5 μL. Carefully mix by repeated pipetting &gt;50% of the total reaction volume. Collect the contents by quickly centrifuging. This Ligation Mix is viscous, and it is recommended to make a waste volume of 15%–20%.</li>
</ol>
<table id="undtbl10">
<thead>
<tr>
<th>Reagent</th>
<th>Amount for each reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>NEBNext Ultra II Ligation Master Mix</td>
<td>15 μL</td>
</tr>
<tr>
<td>NEBNext Ligation Enhancer</td>
<td>0.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>15.5 μL</b></td>
</tr>
</tbody>
</table>
<ol start="58">
<li>Recover the ChIP and input samples from the thermal cycler and place on ice.</li>
<li>Add 15.5 μL of the Ligation Mix to each sample.</li>
<li>Add 1 μL of 0.5 μM sequencing platform-compatible barcode adapter to each reaction.</li>
<li>Set a pipette to 40 μL and pipette the reaction volume up and down 12 times.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to mix the contents well for proper ligation efficiency. A small amount of bubbles will not interfere with performance.</div>
<ol start="62">
<li>Incubate at 20°C for 15 min with the heated lid set to “off.”</li>
<li>Stop the “ChIP input” library ligations by adding 2.5 μL of 0.5 M EDTA and store them on ice.</li>
</ol>
<h3 id="sec3.8">Elute barcoded ChIP DNA, reverse crosslinks, and prepare sequencing libraries for PCR amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5–18 h</div>
<ol start="64">
<li>To the 46.5 μL ChIP library samples, add 29 μL of the following reaction mix (freshly prepared as a master mix at 20°C–25°C):</li>
</ol>
<table id="undtbl11">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount per reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-Free Distilled Water</td>
<td> </td>
<td>18.9 μL</td>
</tr>
<tr>
<td>10% SDS (%)</td>
<td>0.5%</td>
<td>4 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>18.75 mM</td>
<td>3 μL</td>
</tr>
<tr>
<td>0.5 M EGTA</td>
<td>10</td>
<td>1.6 μL</td>
</tr>
<tr>
<td>20 mg/mL Proteinase K</td>
<td>250 μg/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>20 mg/mL RNase A</td>
<td>125 μg/mL</td>
<td>0.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>29 μL</b></td>
</tr>
</tbody>
</table>
<ol start="65">
<li>Add 4.5 μL 5 M NaCl to each ChIP sample (not the ChIP inputs), mix by gentle flicking, then collect the reaction contents at the tube bottom by brief centrifugation.</li>
<li>Incubate at 55°C for 1 h (in a PCR cycler with the heated lid set to 75°C) to digest RNA and proteins, then shift the incubation temperature to 65°C for at least 2 h (up to 16 h) to reverse the crosslinks.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure the heated lid of the thermal cycler is set to 75°C to prevent evaporation and drying out of the samples.</div>
<ol start="67">
<li>Collect the reaction contents at the tube bottom by brief centrifugation, then place the samples on a magnet to collect the magnetic beads and transfer the ∼79 μL supernatant containing the library to a new tube strip.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is essential to separate the ChIP library-containing supernatant from the Dynabeads.</div>
<ol start="68">
<li>Bring each “ChIP input” library from step 63 to 79 μL by adding 25.5 μL UltraPure water and 4.5 μL 5 M NaCl.</li>
<li>Per ChIP or “ChIP input” sample, combine 2 μL SpeedBeads with 61 μL 20% PEG8000/1.5 M NaCl to make 63 μL SpeedBeads/PEG Mix per sample in the form of a master mix, make 10% excess.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Thoroughly resuspend the SpeedBeads by vortexing before use.</div>
<ol start="70">
<li>Add 63 μL of SpeedBeads/PEG mix to each 79 μL sample. The final concentration of PEG will be 8.6% and the overall concentration of NaCl will be 0.8 M.</li>
<li>Mix the samples by light vortexing or repetitive pipetting. Incubate at 20°C–25°C for 10 min.</li>
<li>Apply the tube strips to a magnet and aspirate the cleared supernatant. Be careful to not lose the magnetic beads which have bound the “ChIP input.”</li>
<li>Wash the beads by adding 180 μL of 80% ethanol at 20°C–25°C and moving the strips to either side of the magnet six times. Collect the beads on the magnet and discard the supernatant once cleared. Repeat this wash one additional time.</li>
<li>After removing the second ethanol wash, cap then pulse-centrifuge the tube strip, attract the beads to the magnet, then aspirate the residual ethanol volume using a P10 pipettor and tip. Air-dry the beads for ∼5 min, or until “cracks” just begin to appear in the packed beads.</li>
<li>Elute the “ChIP input” DNA by adding 25 μL TT Buffer. Vortex then incubate at 20°C–25°C for 5–10 min. Pulse-centrifuge then apply the tube strips to the magnet. Transfer the supernatant to a new PCR strip and discard the SpeedBeads.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples may now be safely stored in the freezer until convenient.</div>
<h3 id="sec3.9">PCR amplify the ChIP and input libraries</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="76">
<li>Prepare a PCR master mix by combining 25 μL NEBNext Ultra II Q5 2× master mix with 0.25 μL each of 100 μM sequencing platform-compatible forward and reverse primers. Mix well by pipetting or 2–3 s vortexing, followed by brief centrifugation.</li>
<li>Add 25.5 μL of the PCR master mix to each sample and amplify using a thermal cycler with a heated lid set to 105°C:</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th colspan="4">PCR Cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">9–16 cycles (empirically determined, 12 cycles is a good default)</td>
</tr>
<tr>
<td>Annealing</td>
<td>60°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Total amplification cycles should be empirically balanced to yield sufficient amplification product for sequencing while maximizing library complexity.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The PCR product may be safely stored in the freezer until convenient.</div>
<h3 id="sec3.10">DNA cleanup</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<div class="note">
<span class="note-title">Note:</span> The low (8.5%) PEG8000 amount size selects against short adapter dimers (∼125 bp size) that would lead to uninformative insert-less reads if sequenced and could negatively affect sequencer performance if present in high amounts. While longer (&gt;500 bp) PCR fragments that are potentially present in the library will also be isolated, we found them to not significantly affect the final results, which can be explained by the fact that Illumina sequencers strongly favor sequencing shorter DNAs over longer ones (<a href="#bib7">Gohl et al., 2019</a>).</div>
<ol start="78">
<li>Per PCR reaction combine 2 μL SpeedBeads and 38.5 μL PEG8000/2.5 M NaCl to make 40.5 μL SpeedBeads/PEG Mix per sample in the form of a master mix, make 10% excess.</li>
<li>To each 50 μL PCR reaction, add 40.5 μL SpeedBeads/PEG master mix, for a final concentration of 8.5% PEG8000 and 1 M NaCl. Mix to homogeneity by vortexing or repetitive pipetting. Incubate at 20°C–25°C for 10 min.</li>
<li>Apply the tube strips to a magnet and aspirate the cleared supernatant. Be careful to not lose the magnetic beads which have bound the ChIP-seq libraries.</li>
<li>Wash the beads by adding 180 μL of 80% ethanol at 20°C–25°C and moving the strips to either side of the magnet six times. Collect the beads on the magnet and discard the supernatant once cleared. Repeat this wash one additional time.</li>
<li>After removing the second ethanol wash, cap then pulse-centrifuge the tube strip, attract the beads to the magnet, then aspirate the residual ethanol volume using a P10 pipette and tip. Air-dry the beads for ∼5 min, or until “cracks” just begin to appear in the packed beads.</li>
<li>Elute the ChIP-seq DNA libraries by adding 20 μL TT Buffer. Vortex, then incubate at 20°C–25°C for 5–10 min. Pulse-centrifuge then apply the tube strips to the magnet. Transfer the supernatant to a new PCR strip and discard the SpeedBeads.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The libraries may be safely stored in the freezer until convenient.</div>
<h3 id="sec3.11">DNA quantification</h3>
<ol start="84">
<li>Determine the concentration of the libraries in ng/μL using a Qubit fluorometer following the manufacturer’s protocol. The purified DNA library is now ready for sequencing and downstream analysis.</li>
<li>Estimate the mean library size in base pair (bp) by running a small amount (∼2 μL) of the library on a TapeStation, Bioanalyzer, or a 2% agarose/TBE gel pre-stained with GelGreen.</li>
<li>Calculate the molarity of each library in nM as:</li>
</ol>
<p class="math-ml"><math> <mi>c</mi> <mspace></mspace> <mfenced> <mtext>nM</mtext> </mfenced> <mo>=</mo> <mfrac> <mrow> <mtext>fragment concentration </mtext> <mfenced> <mfrac> <mi>ng</mi> <mi>μL</mi> </mfrac> </mfenced> </mrow> <mrow> <mtext>fragment size </mtext> <mfenced> <mtext>bp</mtext> </mfenced> <mtext> × </mtext> <mi>molar</mi> <mspace></mspace> <mi>mass</mi> <mspace></mspace> <mi>of</mi> <mspace></mspace> <mi>a</mi> <mspace></mspace> <mi>base</mi> <mspace></mspace> <mi>pair</mi> <mspace></mspace> <mo>(</mo> <mtext>660 [</mtext> <mstyle> <mfrac> <mi>g</mi> <mrow> <mi>mol</mi> <mo>∗</mo> <mi>bp</mi> </mrow> </mfrac> </mstyle> <mtext>] </mtext> <mo>)</mo> </mrow> </mfrac> <mo>×</mo> <msup> <mn>10</mn> <mn>6</mn> </msup> <mo>=</mo> <mfrac> <mrow> <mtext>fragment concentration </mtext> <mfenced> <mfrac> <mtext>ng</mtext> <mtext>μL</mtext> </mfrac> </mfenced> </mrow> <mrow> <mtext>fragment size </mtext> <mfenced> <mtext>bp</mtext> </mfenced> </mrow> </mfrac> <mo>×</mo> <mn>1,515.15</mn> <mtext> nM</mtext> </math></p>
<div class="note">
<span class="note-title">Note:</span> For example, a library with a concentration of 10 ng/μL and an estimated mean fragment size of 300 bp has a molarity of:</div>
<p class="math-ml"><math> <mrow> <mi>c</mi> <mo>=</mo> <mfrac> <mn>10</mn> <mn>300</mn> </mfrac> <mo>×</mo> <mn>1,515.15</mn> <mtext> nM</mtext> <mo>=</mo> <mspace></mspace> <mn>50.5</mn> <mtext> nM</mtext> </mrow> </math></p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The ChIP-seq libraries generated with this protocol are ready for sequencing. Optimal loading concentration and overall amount to submit for sequencing depends on the sequencer, contact your sequencing facility for details. For ChIP-seq, we recommend sequencing 15–25 million unique reads per sample. After acquisition of sequencing data, standard software for mapping and analysis can be used. Our lab typically maps ChIP-seq data using Bowtie2 (<a href="#bib13">Langmead and Salzberg, 2012</a>) and analyzes the processed files using the HOMER software suite (<a href="#bib9">Heinz et al., 2010</a>). The HOMER website (<a href="http://homer.ucsd.edu/homer">http://homer.ucsd.edu/homer</a>) also contains a detailed tutorial on how to analyze ChIP-seq data. It is recommended to assess ChIP quality through several combined strategies, including comparison and visualization of ChIP-seq data to input sequencing data on the UCSC Genome Browser (<a href="#bib11">Kent et al., 2002</a>; ) or a preferred alternative (e.g., WashU Genome Browser (<a href="#bib14">Li et al., 2019</a>) - <a href="https://epigenomegateway.wustl.edu">https://epigenomegateway.wustl.edu</a>; IGV (<a href="#bib17">Robinson et al., 2011</a>) - <a href="https://software.broadinstitute.org/software/igv/">https://software.broadinstitute.org/software/igv/</a>). It is also recommended to compare the percentage of sequencing reads found within “peaks,” i.e., the “Fraction of Reads in Peaks” (FRiP, or “Approximate IP efficiency” in the HOMER output of the findPeaks.pl command), which for a good ChIP can reach 20% (<a href="#bib12">Landt et al., 2012</a>). These two metrics can inform on the quality of the ChIP with high FRiP/signal to noise and uniform fragment distribution in the ChIP input indicating high ChIP enrichment. Additionally, metrics such as read duplication rate and GC content distribution generated by FastQC (<a href="#bib2">Andrews, 2010</a>) or HOMER provide information about the diversity of the library and whether the procedure yielded a faithful representation of the genome (typically &gt;80% of a ChIP library represents genomic background).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Reproducible ChIP-seq data depends on antibody quality and specificity. Refer to current ENCODE Experimental Guidelines (<a href="https://www.encodeproject.org/about/experiment-guidelines/">https://www.encodeproject.org/about/experiment-guidelines/</a>) as an accepted framework for antibody validation.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>ChIP-seq libraries have low signal over ChIP input background compared to positive control ChIP-seqs.</p>
<h3 id="sec6.2">Potential solution</h3>
<ol>
<li>Generating quality ChIP-seq data can take several rounds of optimization and strongly depend on both the quality of the antibody and preservation of the target epitope. It is recommended to try several different antibodies, and even combinations of antibodies.</li>
<li>Sample integrity, especially when using primary tissues, is essential. Ensure that tissues are preserved as quickly as possible, by perfusion, snap-freezing, or fixation.</li>
<li>Crosslinking can affect the efficacy of the ChIP-seq. As a general rule, proteins that do not have a DNA binding domain and are present in big complexes should be double-crosslinked (see section <a href="#sec2.2">fixation of proteins to DNA</a>), while DNA binding domain-containing proteins such as transcription factors frequently can be ChIPped when only fixed with formaldehyde. An exception to this rule is dimeric transcription factors such as AP-1, which often benefit from double crosslinking as well.</li>
<li>Also, consider use of alternative ChIP lysis and wash buffers. The buffer conditions described above have worked very well for several challenging transcription factors, however, we have also found success using lysis buffers and wash buffers described elsewhere (<a href="#bib10">Heinz et al., 2018</a>).</li>
</ol>
<h3 id="sec6.3">Problem 2</h3>
<p>During the final size determination on an agarose gel or TapeStation, in addition to the expected library DNA smear from ∼200–500 bp, there is a visible adapter band at ∼125 bp that makes up a substantial amount (&gt;5%–10%) of the overall DNA signal.</p>
<p>Possible explanations: Adapter dimers typically occur when there is little ChIP material for the adapters to ligate to, which often goes hand in hand with a low amount of correctly sized ChIP library. This can be due to various reasons, including the cell number being too low, poor affinity of the used antibody for the ChIP target protein, low expression of the ChIP target protein/low abundance in the nucleus, or a problem with the overall ChIP experiment. Additionally, the size selection efficiency of the final PCR cleanup step may have been variable, leading to increased amounts of adapter dimers being isolated. Adapter dimers represent the shortest DNA fragments that can still be sequenced on an Illumina sequencer. Given the fact that short DNA fragments are preferentially sequenced (<a href="#bib7">Gohl et al., 2019</a>) and that high amounts of duplicate sequences can negatively affect sequencer operation, it is important that the adapter dimers in a library and in the entire pool of samples to be sequenced are kept to a minimum.</p>
<h3 id="sec6.4">Potential solution</h3>
<ol>
<li>If there is still a well-visible smear of correctly sized library, re-purifying and size-selecting large DNA fragments by bringing the volume of the library to 50 μL with TT buffer and repeating the library cleanup and size selection as in steps 78–83 is usually sufficient to remove most adapter dimers.</li>
<li>If multiple libraries exhibit low levels of adapter dimers (less than 30% of the total DNA amount), the libraries can be pooled for sequencing according to the guidelines provided by the sequencing provider or sequencing core, and the entire library pool subjected to library cleanup and size selection as in steps 78–83 to simultaneously remove adapter dimers from all libraries in the pool.</li>
<li>As an alternative to SpeedBeads/PEG size selection, adapter dimers and oversized products can be removed by size-selecting the amplified libraries to 200–500 bp by gel extraction. Run the libraries on a 2% agarose/TBE or 10% polyacrylamide/TBE gel next to 150 ng of 1 kb Plus DNA ladder or similar and stain the DNA with SYBR Gold. On a blue light illuminator, excise a gel slice containing library fragments ranging from 200–500 bp and isolate DNA from the slice. For agarose gel extraction use a commercial kit, e.g., Zymo Research cat# D4001; for PAGE gel extraction we found a procedure most effective that involves shredding the gel slice and diffusion (see <a href="#bib5">Duttke et al., 2019</a> for details).</li>
<li>If adapter dimers pose a problem more frequently and across a range of antibodies, the overall amount of recovered DNA may be generally too low, which points to problems with the reagents. Under these circumstances, it may help to remake the lysis and wash buffers used, and confirm increased library yield by including a positive control using a cell line, higher cell numbers (&gt;1 million) and a positive control antibody such as anti-H3K27ac. In the short term, or for cases where the antigen is lowly expressed or only in a subset of cells, lowering the adapter concentration in the ligation reaction in step 60 may help, or alternatively increasing the cell number used for ChIP, if possible.</li>
</ol>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib2">Andrews, S. (2010). FastQC: a quality control tool for high throughput sequence data [Online]. <a class="external-link" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></p>
<p id="bib3">Brigidi, G.S., Hayes, M.G.B., Delos Santos, N.P., Hartzell, A.L., Texari, L., Lin, P.-A., Bartlett, A., Ecker, J.R., Benner, C., Heinz, S., et al. (2019). Genomic decoding of neuronal depolarization by stimulus-specific NPAS4 heterodimers. Cell <i>179</i>, 373-391.e27. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref3">View at publisher</a></p>
<p id="bib4">Carlin, A.F., Vizcarra, E.A., Branche, E., Viramontes, K.M., Suarez-Amaran, L., Ley, K., Heinz, S., Benner, C., Shresta, S., and Glass, C.K. (2018). Deconvolution of pro- and antiviral genomic responses in Zika virus-infected and bystander macrophages. Proc. Natl. Acad. Sci. U S A <i>115</i>, E9172-E9181. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref4">View at publisher</a></p>
<p id="bib5">Duttke, S.H., Chang, M.W., Heinz, S., and Benner, C. (2019). Identification and dynamic quantification of regulatory elements using total RNA. Genome Res. <i>29</i>, 1836-1846. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref5">View at publisher</a></p>
<p id="bib6">Eichenfield, D.Z., Troutman, T.D., Link, V.M., Lam, M.T., Cho, H., Gosselin, D., Spann, N.J., Lesch, H.P., Tao, J., Muto, J., et al. (2016). Tissue damage drives co-localization of NF-κB, Smad3, and Nrf2 to direct Rev-erb sensitive wound repair in mouse macrophages. eLife <i>5</i>, e13024. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref6">View at publisher</a></p>
<p id="bib7">Gohl, D.M., Magli, A., Garbe, J., Becker, A., Johnson, D.M., Anderson, S., Auch, B., Billstein, B., Froehling, E., McDevitt, S.L., et al. (2019). Measuring sequencer size bias using REcount: a novel method for highly accurate Illumina sequencing-based quantification. Genome Biol. <i>20</i>, 85. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref7">View at publisher</a></p>
<p id="bib8">Gosselin, D., Link, V.M., Romanoski, C.E., Fonseca, G.J., Eichenfield, D.Z., Spann, N.J., Stender, J.D., Chun, H.B., Garner, H., Geissmann, F., et al. (2014). environment drives selection and function of enhancers controlling tissue-specific macrophage identities. Cell <i>159</i>, 1327-1340. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref8">View at publisher</a></p>
<p id="bib9">Heinz, S., Benner, C., Spann, N., Bertolino, E., Lin, Y.C., Laslo, P., Cheng, J.X., Murre, C., Singh, H., and Glass, C.K. (2010). Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Mol. Cell <i>38</i>, 576-589. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref9">View at publisher</a></p>
<p id="bib10">Heinz, S., Texari, L., Hayes, M.G.B., Urbanowski, M., Chang, M.W., Givarkes, N., Rialdi, A., White, K.M., Albrecht, R.A., Pache, L., et al. (2018). Transcription elongation can affect genome 3D structure. Cell <i>174</i>, 1522-1536.e22. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref10">View at publisher</a></p>
<p id="bib11">Kent, W.J., Sugnet, C.W., Furey, T.S., Roskin, K.M., Pringle, T.H., Zahler, A.M., and Haussler, D. (2002). The human genome browser at UCSC. Genome Res. <i>12</i>, 996-1006. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref11">View at publisher</a></p>
<p id="bib12">Landt, S.G., Marinov, G.K., Kundaje, A., Kheradpour, P., Pauli, F., Batzoglou, S., Bernstein, B.E., Bickel, P., Brown, J.B., Cayting, P., et al. (2012). ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia. Genome Res. <i>22</i>, 1813-1831. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref12">View at publisher</a></p>
<p id="bib13">Langmead, B. and Salzberg, S.L. (2012). Fast gapped-read alignment with Bowtie 2. Nat. Methods <i>9</i>, 357-359. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref13">View at publisher</a></p>
<p id="bib14">Li, D., Hsu, S., Purushotham, D., Sears, R.L., and Wang, T. (2019). WashU epigenome browser update 2019. Nucleic Acids Res <i>47</i>, W158-W165. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref14">View at publisher</a></p>
<p id="bib15">Nowak, D.E., Tian, B., and Brasier, A.R. (2005). Two-step cross-linking method for identification of NF-kappaB gene network by chromatin immunoprecipitation. Biotechniques <i>39</i>, 715-725. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref15">View at publisher</a></p>
<p id="bib16">Nott, A., Holtman, I.R., Coufal, N.G., Schlachetzki, J.C.M., Yu, M., Hu, R., Han, C.Z., Pena, M., Xiao, J., Wu, Y., et al. (2019). Brain cell type–specific enhancer–promoter interactome maps and disease - risk association. Science <i>366</i>, 1134-1139. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref16">View at publisher</a></p>
<p id="bib17">Robinson, J.T., Thorvaldsdóttir, H., Winckler, W., Guttman, M., Lander, E.S., Getz, G., and Mesirov, J.P. (2011). Integrative genomics viewer. Nat. Biotechnol. <i>29</i>, 24-26. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref17">View at publisher</a></p>
<p id="bib18">Sakai, M., Troutman, T.D., Seidman, J.S., Ouyang, Z., Spann, N.J., Abe, Y., Ego, K.M., Bruni, C.M., Deng, Z., Schlachetzki, J.C.M., et al. (2019). Liver-derived signals sequentially reprogram myeloid enhancers to initiate and maintain Kupffer cell identity. Immunity <i>51</i>, 655-670.e8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref18">View at publisher</a></p>
<p id="bib19">Schmidl, C., Rendeiro, A.F., Sheffield, N.C., and Bock, C. (2015). ChIPmentation: fast, robust, low-input ChIP-seq for histones and transcription factors. Nat. Methods <i>12</i>, 963-965. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref19">View at publisher</a></p>
<p id="bib20">Seidman, J.S., Troutman, T.D., Sakai, M., Gola, A., Spann, N.J., Bennett, H., Bruni, C.M., Ouyang, Z., Li, R.Z., Sun, X., et al. (2020). Niche-specific reprogramming of epigenetic landscapes drives myeloid cell diversity in nonalcoholic steatohepatitis. Immunity <i>52</i>, 1057-1074.e7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref20">View at publisher</a></p>
<p id="bib22">Troutman, T.D., Bennett, H., Sakai, M., Seidman, J.S., Heinz, S., and Glass, C.K. (2021). Purification of mouse hepatic non-parenchymal cells or nuclei for use in ChIP-seq and other next-generation sequencing approaches. STAR Protocols <i>2</i>. <a class="external-link" href="https://doi.org/10.1016/j.xpro.2021.100363">https://doi.org/10.1016/j.xpro.2021.100363</a></p>
<p id="bib21">Zeng, P.-Y., Vakoc, C.R., Chen, Z.-C., Blobel, G.A., and Berger, S.L. (2006). In vivo dual cross-linking for identification of indirect DNA-associated proteins by chromatin immunoprecipitation. Biotechniques <i>41</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00065-4/sref21">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information requests should be directed to and will be fulfilled by the lead contact, Sven Heinz (<a href="mailto:sheinz@health.ucsd.edu">sheinz@health.ucsd.edu</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new materials.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate new data or code.</p>
<h3>Acknowledgments</h3>
<p>These studies were supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grant DK063491. L.T. was supported by <a href="https://doi.org/10.13039/501100001711">SNSF</a> Advanced Postdoc Mobility fellowship P300PA_164714. T.D.T. was supported by P30 DK063491, T32DK007044, and <a href="https://doi.org/10.13039/100009429">NRSA</a> T32CA009523. M.S. was supported by the <a href="https://doi.org/10.13039/501100007664">Manpei Suzuki Diabetes Foundation</a> of Tokyo, Japan, and the Osamu Hayaishi Memorial Scholarship for Study Abroad, Japan. J.S.S. was supported by <a href="https://doi.org/10.13039/100000968">American Heart Association</a> Fellowship 16PRE30980030 and <a href="https://doi.org/10.13039/100000002">NIH</a> Predoctoral Training grant 5T32DK007541. S.H. received additional support from NIH grants R01GM129523, P30DK063491, and P30DK120515. The graphical abstract contains icons from BioRender.com.</p>
<h3>Author Contributions</h3>
<p>L.T., protocol development, optimization and data analysis, draft editing; N.J.S., protocol development and optimization, draft editing; T.D.T., protocol optimization, manuscript conceptualization and writing, experimentation and data analysis; M.S., protocol optimization, manuscript editing, experimentation; J.S.S., protocol optimization, draft writing; S.H., protocol development and optimization, manuscript editing, and manuscript final approval.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>