<article>
<section>
<h1 id="header">Mass production of human airway-like neutrophils via transmigration in an organotypic model of human airways</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Brian Dobosh,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Vincent D. Giacalone,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Camilla Margaroli,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a></sup> and Rabindra Tirouvanziam<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pediatrics, Emory University School of Medicine, Atlanta, GA 30322, USA</p>
<p id="aff2"><sup>2</sup>Center for CF &amp; Airways Disease Research, Children’s Healthcare of Atlanta, Atlanta, GA 30322, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:bdobosh@emory.edu">bdobosh@emory.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:tirouvanziam@emory.edu">tirouvanziam@emory.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100892">10.1016/j.xpro.2021.100892</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Neutrophils are difficult to study, particularly in tissues, due to their short half-life and propensity for activation. We describe an organotypic airway model that uses patient airway fluid to enable the transmigration of blood neutrophils to acquire an airway-like phenotype in order to better understand their contribution to airway diseases. In particular, we showcase how conditioned neutrophils modulate their bacteria-killing abilities.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.xcrm.2021.100239">Margaroli et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A lung organotypic model allows epithelial cells to be grown at an air-liquid interface</li>
<li>Primary blood neutrophils (and other leukocytes) can transmigrate across the epithelium</li>
<li>Patient airway fluid added apically conditions transmigrating cells, which mirrors the <i>in vivo</i> state</li>
<li>Drugs can be used to understand underlying mechanisms driving disease</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1116-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol details the use of primary patient material. Confirm that you are allowed, and able, to process primary patient material with your local Medical Ethics Committee and Internal Review Board (IRB) and obtain informed consent from the donors for the use of 1) blood and 2) sputum.</p>
<p>The objective of the below protocol is to condition primary blood neutrophils to an airway-like phenotype via transmigration in an organotypic model of human airways with apically applied sputum supernatant from patients with cystic fibrosis (CFASN). The procedure is divided into five main steps (Graphical Abstract). 1) Growing epithelial cells on filters for two weeks at air-liquid interface (ALI) for full differentiation; 2) Isolating primary neutrophils from human blood; 3) Flipping the epithelialized filters to enable transmigration; 4) Transmigration of the neutrophils and subsequent conditioning into airway-like neutrophils; and 5) Use of transmigrated neutrophils in a bacteria-killing assay.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> We have also used this model with sputum from patients with asthma, chronic obstructive pulmonary disease (COPD) and tracheal aspirates from pediatric patients with acute respiratory distress syndrome (ARDS) (<a href="#bib4">Grunwell et al., 2019</a>). In addition, there are multiple methods to isolate neutrophils from the blood of human donors. We highlight the protocol using Polymorphprep (PMP) due to its affordability. In addition, for the bacteria-killing assay, we use <i>Pseudomonas aeruginosa</i> (PA01), but we have also used <i>Staphylococcus aureus</i> and <i>Haemophilus influenzae.</i> Other bacterial strains or isolates could be used, which should be grown according to their optimal conditions. Furthermore, although we use the model here to highlight a particular functional characteristic of airway-like neutrophils (i.e., bacteria-killing ability), other downstream assays such as flow cytometry, RNA-seq, proteomics, etc., can be used to generate a more complete profile of airway neutrophils.</div>
<h3 id="sec1.1">Preparation of 1<b>×</b> PBS + 2.5 mM EDTA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol>
<li>Add 2.5 mL 0.5 M EDTA pH 8.0 and 50 mL 10<b>×</b> PBS to a graduated cylinder.
<ol type="a">
<li>Although 0.5 M EDTA pH 8.0 solution can be purchased directly from a vendor, the solution can also be prepared by dissolving the solid disodium-EDTA or tetrasodium EDTA in double-distilled water. EDTA will not dissolve until the pH of the solution is alkaline. Use a pH-electrode and a 1 M NaOH solution to increase the pH to 8.0 at which point the EDTA will dissolve. The solution can be stored at 25°C.</li>
</ol>
</li>
<li>Adjust volume to 500 mL with double-distilled water.</li>
<li>In a sterile tissue culture hood, add the 500 mL solution to the top compartment of the 0.22 μm filter bottle.</li>
<li>Apply vacuum to filter bottle.
<ol type="a">
<li>Leave vacuum on until the solution has completely passed from the top compartment of the filter bottle to the bottom compartment.</li>
<li>Turn off vacuum.</li>
</ol>
</li>
<li>Unscrew top compartment and replace with the screw cap that came with the filter bottle.</li>
<li>Store at 4°C for up to 6 months.
<ol type="a">
<li>The solution should remain usable until the EDTA precipitates out of solution. Every time this solution is used, check for a white precipitate. If a precipitate forms, discard the solution and make a fresh batch.</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Preparation of airway supernatant</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<div class="note">
<span class="note-title">Note:</span> excluding the time it takes to collect sputum from patient</div>
<ol start="7">
<li>Collect sputum in the clinic by having the patient expectorate into a sample collection cup. <i>(Step 7 is excluded in the total time of the protocol to prepare the airway supernatant due to the variability of patient visits and the time it takes to go from clinic to lab. Also, sputum collection can be achieved through either spontaneous expectoration or induction by inhalation of hypertonic saline. Both methods yield usable airway supernatant for the transmigration model).</i>
<ol type="a">
<li>Anonymize all relevant patient details as described in your IRB or Medical Ethics protocol.</li>
<li>Keep sputum on ice until ready to proceed with steps 8–15 (preferably within 2–3 h of collection).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to strictly adhere to all relevant safety procedures when working with human samples.</div>
<ol start="8">
<li>Weigh sputum and record mass. Add 3 mL of 1<b>×</b> PBS + 2.5 mM EDTA per gram of sputum. If mass is less than 1 g, add 3 mL 1<b>×</b> PBS + 2.5 mM EDTA.</li>
<li>Keeping cup on ice, dissociate by 12 cycles of repeated aspiration using a 10 mL syringe with 18G <b>×</b> 1.5” needle with a blunted tip.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to wear eye goggles as there is an increased risk of a splash during this step. Make sure to keep the cup and syringe angled away from any people in the lab.</div>
<ol start="10">
<li>Transfer sputum to a 15 mL or 50 mL conical tube.</li>
<li>Spin at 800 <b>×</b> <i>g</i> for 10 min at 4°C.
<ol type="a">
<li>This step pellets any cells and large debris or aggregates that may be present in the sputum. Although the pellet is not used in this protocol, it may be useful for other analyses.
<ol type="i">
<li>See (<a href="#bib7">Margaroli et al., 2021</a>) for a use of the cell pellet to study drug effects in ex vivo cells isolated from the lungs of patients with CF.</li>
</ol>
</li>
</ol>
</li>
<li>Transfer supernatant to a clean 15 mL conical tube.</li>
<li>Spin at 3000 <b>×</b> <i>g</i> for 10 min at 4°C.
<ol type="a">
<li>This step pellets any bacteria and larger debris or aggregates that may be present in the sputum. Although the pellet is not used in this protocol, it may be useful for other analyses.</li>
</ol>
</li>
<li>Transfer supernatant to a clean 15 mL conical tube.
<ol type="a">
<li>This supernatant will be hereafter referred to as CF airway supernatant (CFASN)</li>
</ol>
</li>
<li>Store at −20°C for up to 5 years.
<ol type="a">
<li>We have frozen and thawed the CFASN at least 3 times with no problems. However, due to the inherent variability of patient samples multiple freeze thaws should be avoided. We now make 8 mL (or less) aliquots of CFASN to minimize unforeseen consequences.</li>
</ol>
</li>
<li>Dilute CFASN 1:3 in plain RPMI (1 part CFASN: 3 parts total) for use in the transmigration model.
<ol type="a">
<li>For example, combine 2 mL of CFASN with 4 mL plain RPMI for a total of 6 mL.</li>
<li>An important contrast to other protocols is the exclusion of any denaturing chemicals when processing the sputum, such as dithiothreitol (DTT) which influences cell phenotype and transcriptome (<a href="#bib3">Goldmann et al., 2013</a>; <a href="#bib5">Loppow et al., 2000</a>; <a href="#bib6">Makam et al., 2009</a>; <a href="#bib10">Tirouvanziam et al., 2008</a>).</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Preparation of 50<b>×</b> ultroser G stock solution</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<ol start="17">
<li>Add 20 mL sterile double-distilled water to a bottle of Ultroser G (USG).
<ol type="a">
<li>In the United States, access to USG is controlled by the USDA. When we order USG, we pay a fee to have the vendor fill out a permit for ‘IMPORTATION AND TRANSPORTATION OF CONTROLLED MATERIALS’.</li>
</ol>
</li>
<li>Incubate 10 min at 37°C or until fully dissolved.</li>
<li>Make 1 mL aliquots in 1.7 mL tubes or cryovials.
<ol type="a">
<li>USG has limited stability at 4°C and multiple freeze-thaws should be avoided.</li>
</ol>
</li>
<li>Store at −20°C for up to 6 months.</li>
</ol>
<h3 id="sec1.4">Preparation of 70% ethanol</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<ol start="21">
<li>Mix 35 mL 200 proof ethanol with 15 mL double-distilled water.</li>
<li>Store at 25°C indefinitely.</li>
</ol>
<h3 id="sec1.5">Preparation of airway interface media (AIM)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="23">
<li>Thaw 100<b>×</b> Penicillin (10,000 Units/mL)/Streptomycin (10,000 μg/mL) at 25°C.</li>
<li>In a sterile tissue culture hood mix 5 mL of 100<b>×</b> Penicillin/Streptomycin with 500 mL 50/50 DMEM/F12 media. We will refer to this solution as Airway Interface Medium (AIM).</li>
<li>Store at 4°C for up to 2 months.
<ol type="a">
<li>Discard once the color of the media becomes closer to dark red/purple indicating the pH has increased.</li>
</ol>
</li>
</ol>
<h3 id="sec1.6">Preparation of DMEM-F12 + 10% FBS</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="26">
<li>Thaw Fetal Bovine serum (FBS) and 100<b>×</b> Penicillin/Streptomycin at 25°C.
<ol type="a">
<li>Although there is lot-to-lot and vendor-to-vendor variation in the components of FBS, we have not noticed any effect on our culture systems. However, we tend to buy our FBS from Corning (Cat#35-011-CV/<a href="#key-resources-table">key resources table</a>).</li>
</ol>
</li>
<li>In a sterile tissue culture hood, mix 5 mL of 100<b>×</b> Penicillin/Streptomycin with 50 mL FBS and 500 mL 50/50 DMEM/F12 media. We will refer to this solution as DMEM-F12 + 10% FBS.</li>
<li>Add the 500 mL solution to the top compartment of the 0.22 μm filter bottle.</li>
<li>Apply 0.22 μm filter bottle to vacuum to sterilize the solution.</li>
<li>Store at 4°C for up to 2 months.
<ol type="a">
<li>Discard once the color of the media becomes closer to dark red/purple indicating the pH has increased.</li>
</ol>
</li>
</ol>
<h3 id="sec1.7">Preparation of RPMI + 10% FBS</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="31">
<li>Thaw FBS at 25°C.</li>
<li>In a sterile tissue culture hood, mix 5 mL FBS with 45 mL 50/50 DMEM/F12 media. We will refer to this solution as RPMI + 10% FBS.</li>
<li>Use a 0.22 μm syringe filter to sterilize the solution.</li>
<li>Store at 4°C for up to 2 months.
<ol type="a">
<li>Discard once the color of the media becomes closer to dark red/purple indicating the pH has increased.</li>
</ol>
</li>
</ol>
<h3 id="sec1.8">Preparation of NaCl solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="35">
<li>0.45% w/v NaCl:
<ol type="a">
<li>Dissolve 2.25 g of NaCl in 450 mL double-distilled water. Adjust volume to 500 mL with double-distilled water and then filter using a 0.22 μm filter bottle. Store at 25°C.
<ol type="i">
<li>This solution can be stored indefinitely as long as a precipitate does not form.</li>
</ol>
</li>
</ol>
</li>
<li>1.8% w/v NaCl:
<ol type="a">
<li>Dissolve 9.00 g of NaCl in 450 mL double-distilled water. Adjust volume to 500 mL with double-distilled water and then filter using a 0.22 μm filter bottle. Store at 4°C.
<ol type="i">
<li>This solution can be stored indefinitely as long as a precipitate does not form.</li>
<li>Although this solution can be stored at room temperature (∼25°C), it is used cold in the protocol. As a result, we store this solution at 4°C. Similarly, the water that will be used for hypotonic lysis of red blood cells should also be stored at 4°C.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.9">Preparation of luria-bertani (LB) media</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="37">
<li>Weigh 10 g of LB media powder and add 500 mL water. Allow to dissolve.</li>
<li>Autoclave at 121°C for 30 min.</li>
<li>Let cool to 25°C.</li>
<li>Store at room temperature for 1 year.
<ol type="a">
<li>Always check for growth of contaminants in the media. If any appear, add concentrated bleach to a final concentration of 10% v/v and discard the solution.</li>
</ol>
</li>
</ol>
<h3 id="sec1.10">Preparation of luria-bertani (LB) agar plates</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="41">
<li>Weigh 16 g of LB agar powder and add 500 mL water. Autoclave at 121°C for 15 min.</li>
<li>Let agar cool to 55°C (Able to touch the bottle somewhat comfortably).</li>
<li>Using a serological pipette, dispense 20 mL into a 10 cm Petri dish.</li>
<li>Let the plates cool until solid. Leave the plates at room temperature for 1–2 days out of direct sunlight to confirm that no contaminants are present. Then seal the plates in a plastic sleeve and store at 4°C for up to 4 months.</li>
</ol>
<h3 id="sec1.11">Calibration of colony forming units (CFU) to OD<sub>600</sub>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<ol start="45">
<li>This step is done in advance of the bacteria-killing assay so that the multiplicity of infection (ratio of number of bacteria to number of immune cells) can be determined from the OD<sub>600</sub>. This calibration should be repeated every time there is a large change in the growth conditions of your bacteria including different media or atmospheric conditions of the lab, i.e., change of seasons, humidity, temperature, or the shaking incubator was repaired or changed, etc.</li>
<li>Streak out a glycerol stock of <i>Pseudomonas aeruginosa</i> (PA01) on an LB plate. Grow at 30°C for 16–18 h.
<ol type="a">
<li>Although many recommend growing PA01 at 37°C, we have found that growing <i>Pseudomonas aeruginosa</i> at a lower temperature helps with the morphology of the colonies. At 37°C, we observe more mucoid-like colonies that may blend together, potentially due to overgrowing (See <a href="#sec6.15">troubleshooting 8</a>).</li>
</ol>
</li>
<li>Inoculate a single colony in 3 mL LB media, and grow at 37°C at 200 rpm for 18 h.
<ol type="a">
<li>Since there are no antibiotics or other selective-agent present, it is good practice to have a second tube of LB media that is mock-inoculated to check for contaminants.</li>
</ol>
</li>
<li>Subculture the grown bacteria in LB media by pipetting 10 μL of overnight culture into 3 mL LB media. Conduct in triplicate.</li>
<li>Shake at 37°C at 200 rpm.</li>
<li>After 1 h, every 30 min transfer 400 μL from each tube to a cuvette and record the OD<sub>600</sub>.
<ol type="a">
<li>Replace tube into the shaking incubator to continue growth.</li>
<li>Serially dilute the bacteria, and plate onto an LB agar plate. Let dry for 15 min, then place upside down at 30°C for 12–18 h.</li>
</ol>
</li>
<li>Continue measurements until OD<sub>600</sub>=0.4 is reached. Then conduct spectrophotometer measurements and plating every 15 min. Continue until OD<sub>600</sub>=0.6 is reached.</li>
<li>Count colonies and plot against the OD<sub>600</sub> (See representative standard curve; <a href="#fig3">Figure 3</a>A).</li>
</ol>
<h3 id="sec1.12">Preparation of EtBr/AO live/dead stain</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="53">
<li>Ethidium bromide (EtBr) is considered a mutagen and should be handled with care. Refer to the MSDS and your EHSO department for all safety precautions.</li>
<li>Add 150 μL of 5 mg/mL Ethidium bromide (EtBr) and 150 μL of 3 mg/mL acridine orange (AO) to 7.5 mL of 1<b>×</b> PBS.
<ol type="a">
<li>Store at 4°C out of direct light and wrapped in aluminum foil for up to 1 year. We have observed decreased fluorescence of ethidium bromide after this time.</li>
</ol>
</li>
<li>When quantifying cells, mix 10 μL of stain with 10 μL of cells.
<ol type="a">
<li>If a higher dilution is needed, dilute cells in 1<b>×</b> PBS so that the final mixture is no more than 50% EtBr/AO. The stain is inherently toxic at high concentrations.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td><i>P. aeruginosa</i></td>
<td>PerkinElmer</td>
<td>Cat#119229</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>CFASN</td>
<td>Sputum acquired from patients diagnosed with cystic fibrosis via CF@tlanta CFBR</td>
<td>NA</td>
</tr>
<tr>
<td>Blood</td>
<td>Blood was drawn by a trained phlebotomist from healty donors</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>10<b>×</b> PBS</td>
<td>Corning</td>
<td>Cat#46-013-CM</td>
</tr>
<tr>
<td>0.5 M EDTA, pH 8.0</td>
<td>VWR</td>
<td>Cat#351-027-101</td>
</tr>
<tr>
<td>Cell culture-grade water</td>
<td>VWR</td>
<td>Cat#17–724f</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Corning</td>
<td>Cat#35-011-CV</td>
</tr>
<tr>
<td>Ultroser G (USG)</td>
<td>Crescent Chemical Company</td>
<td>Cat#67042</td>
</tr>
<tr>
<td>50/50 DMEM-F12</td>
<td>VWR</td>
<td>Cat#10-090-CV</td>
</tr>
<tr>
<td>RPMI</td>
<td>VWR</td>
<td>Cat#10-040-CV</td>
</tr>
<tr>
<td>200 Proof Ethanol</td>
<td>Fisher Chemical</td>
<td>Cat#BP2818-500</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Millipore Sigma</td>
<td>Cat#TMS-AB2-C</td>
</tr>
<tr>
<td>NaCl</td>
<td>Fisher Scientific</td>
<td>Cat#S271-500</td>
</tr>
<tr>
<td>LB Media Powder</td>
<td>Research Products International</td>
<td>Cat#L24060-500</td>
</tr>
<tr>
<td>LB Agar Powder</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#22700-025</td>
</tr>
<tr>
<td>Leukotriene B<sub>4</sub> (LTB4)</td>
<td>Millipore Sigma</td>
<td>Cat#L0517-10UG</td>
</tr>
<tr>
<td>PolymorphPrep (PMP)</td>
<td>Progen</td>
<td>Cat#AN1114683</td>
</tr>
<tr>
<td>Collagen (Rat Tail Collagen) Type I</td>
<td>Sigma-Aldrich</td>
<td>Cat#C3867-1VL</td>
</tr>
<tr>
<td>Tryspin 0.25%</td>
<td>HyClone</td>
<td>Cat#SH30042.01</td>
</tr>
<tr>
<td>Accutase</td>
<td>BioLegend</td>
<td>423201</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>NCI-H441</td>
<td>ATCC</td>
<td>HTB-174</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>0.22 μm Filter bottle</td>
<td>Millipore Sigma</td>
<td>Cat#Z760900-10EA</td>
</tr>
<tr>
<td>10 mL Syringe</td>
<td>Fisher Scientific</td>
<td>Cat#14955459</td>
</tr>
<tr>
<td>18G <b>×</b> 1.5 " Needle</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat#sc-360794</td>
</tr>
<tr>
<td>15 mL Conical tube</td>
<td>Millipore Sigma</td>
<td>Cat#188271</td>
</tr>
<tr>
<td>10 cm Petri dishes</td>
<td>Millipore Sigma</td>
<td>Cat#P5731-500EA</td>
</tr>
<tr>
<td>0.22 μm Nylon syringe filter</td>
<td>Fisher Scientific</td>
<td>Cat#09–719C</td>
</tr>
<tr>
<td>60 mL Syringe</td>
<td>Fisher Scientific</td>
<td>Cat#14955461</td>
</tr>
<tr>
<td>Alvetex filters</td>
<td>Reprocell</td>
<td>Cat#AVP005-96</td>
</tr>
<tr>
<td>Stainless steel forceps, curved end</td>
<td>Millipore Sigma</td>
<td>Cat#Z168696-1EA</td>
</tr>
<tr>
<td>Pasteur pipette</td>
<td>Fisher Scientific</td>
<td>Cat#13–711-20</td>
</tr>
<tr>
<td>Shaking incubator</td>
<td>Thermo Scientific</td>
<td>MaxQ 5000</td>
</tr>
<tr>
<td>Spectrophotometer</td>
<td>Molecular Devices</td>
<td>SpectraMaxM2</td>
</tr>
<tr>
<td>Cuvette</td>
<td>Fisher Scientific</td>
<td>Cat#14–955-128</td>
</tr>
<tr>
<td>Rotator</td>
<td>Gene Express</td>
<td>Ref: H-6800</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Preparation of air-liquid interface (ALI) cultures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>H441 cells (2.5 ∗ 10<sup>5</sup> cells) are seeded on an Alvetex scaffold to achieve ALI as observed in the lung. The porosity of the Alvetex scaffold was chosen to utilize neutrophils for transmigration assays with optimum yield (<a href="#bib2">Forrest et al., 2018</a>). This scaffold does not constrain transmigrating neutrophils in prefabricated channels unlike conventional Transwells / Snapwells. Instead, transmigrating neutrophils gather under the epithelium and transmigrate <i>en masse</i> while allowing the epithelial to reseal, as evidenced by a temporary increase in dextran permeability (<a href="#bib2">Forrest et al., 2018</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1116-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Preparation of Alvetex filters for use in the organotypic model of human airways</div>
<p>(A) In order to fit into the 12-well plate, the scaffolds need to have the outermost tabs of plastic removed. They are already perforated. Be careful not to crack the scaffold.</p>
<p>(B) 70% ethanol is added dropwise to filters to rehydrate them and allow for them to be coated with rat tail collagen Type I. The filter will be white if not enough ethanol was added, and very shiny, with a visible pool of ethanol if it is too wet. When the ring of ethanol is barely visible the filters are ready for coating.</p>
<p>(C) Uneven coating of collagen on the filters may result in holes on the filters that cells cannot grow on. Make sure that the collagen is evenly distributed across the entire filter.</p>
<p>(D) Too much acidity in the media may lower the pH too far leading to non-optimal conditions for cell growth. Make sure the media is pink before growing cells on the scaffolds.</p>
</figcaption>
</figure>
<p>The USG is used to obtain long-term, confluent, and polarized H441 cultures at ALI. USG-treated cultures develop as monolayers with apical microvilli and exclude any basally added medium from their surface. Exposure to air is a major physiological constraint faced by the pulmonary epithelium. These conditions lead to extensive differentiation of H441 cells with morphological and electrophysiological properties comparable to human small airway epithelial cells.</p>
<ol>
<li>Grow H441 cells in T-75 flasks or equivalent using DMEM-F12 + 10% FBS at 37°C + 5% CO<sub>2</sub>.
<ol type="a">
<li>H441 cells grow slowly when first thawed and should be passaged at least twice before use in the model. Discard cells after passage 21.</li>
<li>H441 cells should not be allowed to grow more than 90% confluent. Typically, we split our cells every 2–3 days.</li>
</ol>
</li>
<li>Leave collagen at 25°C for ∼20 min before use (Day -1; <a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Schedule for differentiation of H441 cells at air-liquid interface</caption>
<thead>
<tr>
<th>ID</th>
<th>Day</th>
<th>Instructions</th>
</tr>
</thead>
<tbody>
<tr>
<td>Day -1</td>
<td>Wed</td>
<td>Coat the wells with 200 μL collagen using a P1000</td>
</tr>
<tr>
<td>Day 0</td>
<td>Thurs</td>
<td>Add 1.5 mL DMEM-F12 + 10% FBS basally and 1 mL/250,000 cells apically.</td>
</tr>
<tr>
<td>Day 1</td>
<td>Fri</td>
<td>1) Aspirate basal and apical fluids2) Replace media with AIM+USG.3) Leave apical side exposed to air.</td>
</tr>
<tr>
<td>Day 4</td>
<td>Mon</td>
<td>Aspirate basal media, replace with 2 mL AIM+USGTransfer filters to a clean plate</td>
</tr>
<tr>
<td>Day 6</td>
<td>Wed</td>
<td>Aspirate basal media, replace with 2 mL AIM+USG</td>
</tr>
<tr>
<td>Day 8</td>
<td>Fri</td>
<td>Aspirate basal media, replace with 2 mL AIM+USG</td>
</tr>
<tr>
<td>Day 11</td>
<td>Mon</td>
<td>Aspirate basal media, replace with 2 mL AIM+USG</td>
</tr>
<tr>
<td>Day 13</td>
<td>Wed</td>
<td>Aspirate basal media, replace with 2 mL AIM+USG</td>
</tr>
<tr>
<td>Day 14</td>
<td>Thurs</td>
<td>Filters ready to use if H441 growth seems appropriate-if cells too dense, may see small bumps on the filter-if media is pooling on the filter, cells may be dying off</td>
</tr>
<tr>
<td>Day 19</td>
<td>Tues</td>
<td>Use filters by this date.</td>
</tr>
</tbody>
</table>
</li>
<li>Remove the Alvetex inserts from the package and gently snap off the outermost tab such that the scaffolds fit into a non-tissue-culture-treated 12 well-plate (<a href="#fig1">Figure 1</a>A).
<ol type="a">
<li>Make sure to only snap off the outermost tab. If the inner plastic is broken, then the filter and scaffold will not sit flat in the 12-well plate (See <a href="#sec6.1">troubleshooting 1</a>).</li>
</ol>
</li>
<li>Use 2–3 drops of 70% sterile filtered ethanol to render the Alvetex membrane hydrophilic. Dry until the filter is only slightly opaque (if becomes white again it is too dry, add more ethanol) (<a href="#fig1">Figure 1</a>B).
<ol type="a">
<li>If the filter is too dry, the collagen will be unable to bind (See <a href="#sec6.3">troubleshooting 2</a>).</li>
</ol>
</li>
<li>Coat the Alvetex membranes with 200 μL Rat Tail Collagen I (at 3 mg/mL). Spread evenly over filter as it is dispensed from tip (without touching the filter) (<a href="#fig1">Figure 1</a>C).
<ol type="a">
<li>We have found that using a P1000 tip and utilizing reverse-pipetting techniques is sufficient for handling the viscous collagen, particularly at room temperature (See <a href="#sec6.3">troubleshooting 2</a>).</li>
<li>We have found that there may be brand-to-brand variability in the quality of collagen. We highly recommend using the collagen from Sigma-Aldrich (See <a href="#key-resources-table">key resources table</a>).</li>
</ol>
</li>
<li>Store at 37°C + 5% CO<sub>2</sub> between 3-24 h.
<ol type="a">
<li>At least 3 h is needed for the collagen to adhere to the hydrophilic membrane. Any longer than a day and the filter may dry out. Within this timeframe we have not noticed any differences in quality of the filter or subsequent ALI cultures.</li>
</ol>
</li>
<li>Add 2 mL of trypsin to the 90% confluent T-75 flask that contains H441 cells (Day 0; <a href="#tbl1">Table 1</a>).</li>
<li>Incubate at 37°C + 5% CO<sub>2</sub> with gentle agitation for 5 min or until cells have completely detached from the bottom of the flask.</li>
<li>Neutralize trypsin by adding 8 mL of DMEM-F12 + 10% FBS to the flask.</li>
<li>Transfer trypsinized-cells to a 15 mL conical tube, remove a 10 μL aliquot, and spin the conical tube at 800 <b>×</b> <i>g</i> for 10 min to pellet the H441 cells.</li>
<li>While the cells are centrifuging mix the 10 μL aliquot of cells with 10 μL of EB/AO live/dead stain and quantify the number of cells using a hemacytometer or similar cell-counting method.
<ol type="a">
<li>Calculate the total number of cells in the pellet by the following calculation:<math> <mrow> <mo>#</mo> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mi>s</mi> <mo>∗</mo> <mn>2</mn> <mo>∗</mo> <msup> <mn>10</mn> <mn>4</mn> </msup> <mo>∗</mo> <mn>10</mn> <mo>=</mo> <mi>t</mi> <mi>o</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> <mspace></mspace> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mi>s</mi> </mrow> </math>Where #cells is the average number of cells counted in each quadrant of the hemacytometer, “2” is the dilution factor, 10<sup>4</sup> is the distance to volume conversion for a hemacytometer and “10” is the total volume of cells that were originally centrifuged (10 mL). Adjust the dilution factor to have an adequate number of cells to count.</li>
</ol>
</li>
<li>When the cells are done centrifuging, discard the supernatant and resuspend the cells in X mL of DMEM-F12 + 10% FBS such that the cells are at a final concentration of 2.5 ∗ 10<sup>5</sup> cells/mL.
<ol type="a">
<li>Calculate the volume (X) to resuspend the pellet in by the following calculation:</li>
</ol>
</li>
</ol>
<p class="math-ml"><math> <mrow> <mfrac> <mrow> <mi>t</mi> <mi>o</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> <mspace></mspace> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mi>s</mi> </mrow> <mrow> <mi>X</mi> <mspace></mspace> <mi>m</mi> <mi>L</mi> </mrow> </mfrac> <mo>=</mo> <mn>2.5</mn> <mo>∗</mo> <mspace></mspace> <msup> <mn>10</mn> <mn>5</mn> </msup> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mi>s</mi> <mo>/</mo> <mi>m</mi> <mi>L</mi> </mrow> </math></p>
<ol start="13">
<li>Add 1.5 mL of DMEM-F12 + 10% FBS basally to the bottom of the well.
<ol type="a">
<li>Collagen is dissolved in a sodium acetate buffer that is around pH 5.5. Since phenol red turns yellow below pH 6.8, if there is too much residual collagen the media will change color (<a href="#fig1">Figure 1</a>D). If this occurs, rinse the bottom of the filter with additional DMEM-F12 + 10% FBS or RPMI until the media remains pink.</li>
</ol>
</li>
<li>Gently, add 1 mL of resuspended H441 cells (2.5 ∗ 10<sup>5</sup> cells/mL) apically (top of the filter).
<ol type="a">
<li>Being too rough at this stage will result in cells overflowing the filter and seeding the bottom of the well (not the filter) leading to a variable number of cells on the filter. Inevitably, some cells will be found on the bottom of the culture dish, but this should be minimized by slowly dispensing the cells on top of the filter.</li>
</ol>
</li>
<li>Grow at 37°C + 5% CO<sub>2</sub> for 24 h.</li>
<li>The next day (Day 1; <a href="#tbl1">Table 1</a>), aspirate all media from both the basal and apical sides and replace with 1.5 mL of AIM + USG basally (bottom of the filter). Leave the apical side exposed to air.</li>
<li>Change the media basally on a Monday/Wednesday/Friday schedule by aspirating the media from the opening between the wall of the well and the filter without removing the filter from the well.
<ol type="a">
<li>It is important to use freshly prepared AIM + USG for each media change as USG is unstable in solution at 4°C.</li>
<li>Avoid bubbles under the filter, as it will prevent even distribution of nutrients.</li>
<li>On Day 3 or 4 (<a href="#tbl1">Table 1</a>) transfer the filters to a clean 12-well plate. Over time, the cells that were washed off the filter during the initial seeding of cells on the filter (step 14) would start to consume nutrients from the media.</li>
</ol>
</li>
<li>After 2 weeks (Day 14; <a href="#tbl1">Table 1</a>) from the seeding date (Day 0) the filters are ready to be flipped and used. For best results, use the plate no later than Day 19 from seeding.</li>
</ol>
<h3 id="sec2.2">Purification of neutrophils</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This step describes a method for the purification of primary neutrophils from human blood using a gradient density centrifugation. Refer to the manufacturer’s instructions for complete details (<a href="https://www.proteogenix.science/product/polymorphprep/">https://www.proteogenix.science/product/polymorphprep/</a>). Other methods can be used, but cell activation and death need to be assessed prior to use for downstream assays. These neutrophils will be used to transmigrate across the epithelium grown at ALI to induce acquisition of an airway-like phenotype. The purification of neutrophils should be conducted on the day that the filters are to be used (Day 14–19; <a href="#tbl1">Table 1</a>). We have also used this model to transmigrate and induce an airway-like phenotype in primary human blood monocytes (<a href="#bib1">Ford et al., 2021</a>).</p>
<ol start="19">
<li>Collect blood in an EDTA tube (final concentration 1.5–2.0 mM) in the clinic or appropriate location for phlebotomy (<i>step 19 is excluded in the total time of the protocol to prepare the airway supernatant due to the variability of patient visits and the time it takes to go from clinic to lab.)</i>
<ol type="a">
<li>Anonymize all relevant donor details as described in your IRB or Medical Ethics protocol.</li>
<li>Process blood as soon as possible, typically within 2–3 h.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to strictly adhere to all relevant safety procedures when working with human samples.</div>
<ol start="20">
<li>Add 6 mL of Polymorphprep™ (PMP) to a 15 mL conical tube.</li>
<li>Slowly layer 6 mL of whole blood on top of the PMP solution.</li>
<li>Centrifuge at 400 <b>×</b> <i>g</i> for 42 min at room temperature with slow acceleration and minimal brake.
<ol type="a">
<li>For density gradient centrifugations a swinging-bucket style centrifuge is best.</li>
</ol>
</li>
<li>Remove the plasma and top mononuclear cell band (upper band) (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1116-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Density gradient separation of human blood neutrophils and flipping of the filters</div>
<p>(A) Blood is loaded on top of the Polymorphprep (PMP) solution and then centrifuged at 400 <b>×</b> <i>g</i> for 42 min with minimal brake. The neutrophils of some donors sediment with a large number of red blood cells, while others have very few red blood cells observed with the neutrophil band (3). For the blood from donor 1 it would be advisable to find an alternative protocol to PMP purification of neutrophils. Donor 2 may be usable but would require a more extensive hypotonic lysis (See <a href="#sec6.5">troubleshooting 3</a>). The peripheral blood mononuclear (PBMCs) cells make up the top band, while the bottom band contains the granulocytes, which are overwhelmingly comprised of neutrophils.</p>
<p>(B) The Alvetex scaffolds consist of two plastic parts: Components 1 &amp; 2. The filter is sandwiched between the two components.</p>
<p>(C) In order to flip the membrane to have the correct orientation of the epithelium for transmigration of neutrophils, components 1 &amp; 2 must be separated and the filter flipped. Upon separation of the components, the filter can end up in one of two locations. If the filter is attached to component 1, <a href="#mmc1">Methods video S1</a> – Flipping Component 1 shows how to flip the filter. If the filter is attached to component 2, <a href="#mmc2">Methods video S2</a> – Flipping Component 2 shows how to flip the filter. After flipping, the components are pieced back together and put into a 12-well plate containing transmigration media such as CFASN or LTB4. When placing the filter into the well, tile the plate to avoid the introduction of air bubbles.</p>
</figcaption>
</figure>
</li>
<li>Collect the neutrophil band (lower band) using a Pasteur pipette and transfer to a clean 15- or 50-mL conical tube depending upon volume.
<ol type="a">
<li>We have observed that some donors have a large amount of red blood cells that sediment with the blood neutrophils. We define large amount roughly as RBC contamination that is ∼3+ times thicker than the neutrophil band (<a href="#fig2">Figure 2</a>A). These donors should generally be avoided (See <a href="#sec6.5">troubleshooting 3</a>). Make a note of donors whose blood separates adequately or not on PMP for future reference in case of repeat visits. For donors with blood that does not cleanly separate out the neutrophils use an alternative protocol such as EasySep (<a href="https://www.stemcell.com/easysep-direct-human-neutrophil-isolation-kit.html">https://www.stemcell.com/easysep-direct-human-neutrophil-isolation-kit.html</a>).</li>
</ol>
</li>
<li>Add 1 volume of 0.45% w/v NaCl to the neutrophils. Gently mix by inverting the tube 3–4 times.</li>
<li>Centrifuge at 800 <b>×</b> <i>g</i> for 5–10 min until the solution is clean and all cells are pelleted. Use maximum acceleration and braking for this step and all subsequent steps.
<ol type="a">
<li>We have found that 800 <b>×</b> <i>g</i> for 5–10 min is a good compromise for efficient pelleting of the neutrophils while still avoiding cell death.</li>
</ol>
</li>
<li>After the centrifugation step is complete, cool the centrifuge to 4°C.</li>
<li>Discard the supernatant.
<ol type="a">
<li>To avoid unwanted activation or cell death of the neutrophils keep neutrophils on ice at all times from this step forward.</li>
</ol>
</li>
<li>Quickly resuspend the pellet in 5 mL of ice-cold, sterile cell-culture water. Incubate on ice for 30 s.
<ol type="a">
<li>This step lyses red blood cells that may be contaminating the neutrophils. They will be clearly visible. There is donor to donor variation (as well as variation on a day-to-day basis) for the amount of red blood cells that may copurify with the neutrophils.</li>
<li>Do not allow this incubation to go past 45 s to avoid undesired cell death of the neutrophils.</li>
<li>We have not tried other lysis solutions such as ACK lysis buffer, but these may be acceptable. As always, when working with neutrophils make sure to determine that your protocol does not result in errant activation of the cells as a result of your purification protocol.</li>
</ol>
</li>
<li>Quickly restore osmolarity by adding 5 mL of ice-cold 1.8% NaCl to the tube. Gently mix by inverting the tube 3–4 times.
<ol type="a">
<li>Other solutions such as 2<b>×</b> HEPES may also be used.</li>
</ol>
</li>
<li>Centrifuge at 800 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Repeat the red blood cell lysis for a total of 2–3 times until no red is visible in the cell pellet. More than three lysis steps should be avoided to prevent activation of the neutrophils.</li>
<li>Resuspend the neutrophils in cold RPMI at a concentration of 1∗10<sup>7</sup> cells/1 mL and keep on ice until needed. Remove a small aliquot to quantify the number of cells and take note of live and dead cells.
<ol type="a">
<li>Viability should be greater than 95%. Refer to <a href="#sec6.7">troubleshooting 4</a> for details if low viability is observed.</li>
<li>Use the neutrophils within 1.5 h from purification.</li>
</ol>
</li>
</ol>
<h3 id="sec2.3">Flipping of filters</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼20 min (about 45 seconds/filter) + 5 minutes of preparation time</div>
<p>This step reverses the orientation of the H441 epithelial cells grown at ALI on Alvetex filters and allows purified blood neutrophils to be added on top (basal aspect of the epithelium) and transmigrated towards the fluid bathing the apical aspect of the epithelium. Transmigrated neutrophils will take on an airway-like phenotype that changes based on the media they are transmigrated towards.</p>
<p>This step should be conducted immediately after the neutrophils are purified. It may be helpful to have one person process the blood and purify the neutrophils. Then, when the hypotonic lysis starts (step 28), the second person can begin preparing the transmigration media and flipping of the filters. There are two plastic components to the filters, which we have termed Component 1 (Larger) and Component 2 (Smaller) in addition to the filter (<a href="#fig2">Figure 2</a>B).</p>
<ol start="34">
<li>Dilute CFASN 1:3 (6.67 mL of CFASN + 13.33 mL RPMI) for a 12-well plate.</li>
<li>Aliquot 2 mL of diluted CFASN into each well of a non-tissue culture treated plate.
<ol type="a">
<li>A non-tissue culture treated plate should be used as tissue-culture plates may activate the neutrophils (<a href="#bib8">Nathan, 1987</a>).</li>
</ol>
</li>
<li>Separate components 1 and 2 to allow flipping of the membrane. The membrane will remain attached to either of the components. Different steps must be taken in either scenario to ensure correct inversion of the membrane for the epithelium to be facing the apical compartment. Take care not to drop or fling the filter by applying too much force (<a href="#fig2">Figure 2</a>C; <a href="#mmc1">Methods video S1</a> and <a href="#mmc2">S2</a>; <a href="#sec6.9">troubleshooting 5</a>).</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1116-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Flipping component 1, related to step 2C</div>
</figcaption>
</figure>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1116-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. Flipping component 2, related to step 2C</div>
</figcaption>
</figure>
<p><b><i>Scenario 1 –</i></b> <i>The membrane sticks to component 1 (Larger piece) –</i> <a href="#mmc1">Methods video S1</a></p>
<p>This is usually the easier scenario to correctly flip the membrane.</p>
<ol start="37">
<li>After separating components 1 and 2, place both components upside down in the sterile packaging that contained the new plate to use for the transmigration assay.</li>
<li>Using forceps, gently nudge the filter then pick up the opposite end, invert it, and place it back on the center of the bottom side of component 1.</li>
<li>Align the clips of component 2 with the tabs on component 1. Gently press component 2 back onto component 1 so that all the clips click into place.
<ol type="a">
<li>Take care not to touch the filter when doing this.</li>
</ol>
</li>
<li>Revert the reconstructed filter apparatus and place it into the appropriate well containing the transmigration fluid.
<ol type="a">
<li>To avoid the introduction of air bubbles, slightly tilt the plate when adding the filter to the well. See <a href="#sec6.11">troubleshooting 6</a> for steps to take to deal with air bubbles.</li>
</ol>
</li>
</ol>
<p><b><i>Scenario 2 –</i></b> <i>The membrane sticks to component 2 (Smaller piece) –</i> <a href="#mmc2">Methods video S2</a></p>
<p>In this case removal of the membrane is usually more difficult, but it will not be physically inverted before being placed back into the filter apparatus.</p>
<ol start="41">
<li>After separating components 1 and 2, place component 1 upside down in the sterile packaging and use the forceps to pick up the filter by aligning the curve of the forceps with the curve of the membrane.</li>
<li>Pick up the membrane and place it back down in the center of the bottom side of component 1 <b>without</b> inversion.</li>
<li>Align the clips of component 2 with the tabs on component 1. Gently press component 2 back onto component 1 so that all the clips click into place.
<ol type="a">
<li>Take care not to touch the filter when doing this.</li>
</ol>
</li>
<li>Place the reconstructed filter apparatus into the appropriate well containing the transmigration fluid.
<ol type="a">
<li>To avoid the introduction of air bubbles, slightly tilt the plate when adding the filter to the well. See <a href="#sec6.11">troubleshooting 6</a> for steps to take to deal with air bubbles.</li>
<li>Once all of the filters are placed into the appropriate wells containing the transmigration fluid proceed to step 45, which details the application of the purified neutrophils to the flipped filters.</li>
<li>Steps 44 to 45 should be done as quickly as possible. Having a person prepare and flip the filters while another person is preparing the neutrophils may help with the transition between steps.</li>
</ol>
</li>
</ol>
<h3 id="sec2.4">Transmigration of neutrophils</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h–1 day depending upon transmigration time</div>
<p>These steps describe the addition of neutrophils to the epithelialized Alvetex filter and subsequent collection of airway-like neutrophils that have transmigrated across it. The time for transmigration is a balance between maintaining viability of the neutrophils and obtaining the desired phenotype. See these papers for details (<a href="#bib2">Forrest et al., 2018</a>; <a href="#bib4">Grunwell et al., 2019</a>; <a href="#bib7">Margaroli et al., 2021</a>). Again, note that we have also successfully used this model to transmigrate and induce an airway-like phenotype in primary human blood monocytes (<a href="#bib1">Ford et al., 2021</a>).</p>
<ol start="45">
<li>Add 100 μL of purified blood neutrophils (<b>1∗10</b><sup><b>7</b></sup> <b>cells/1 mL from step 32)</b> to the top of the filter.
<ol type="a">
<li>Take care not to touch the filter or pierce it with the pipette tip (<a href="#sec6.13">troubleshooting 7</a>).</li>
<li>Make sure that the plate is flat and that neutrophils are evenly dispersed across the filter.</li>
</ol>
</li>
<li>Gently move the plate into the incubator at 37°C + 5% CO<sub>2</sub>. Allow the transmigration to occur for the desired amount of time.
<ol type="a">
<li>We typically use a 4-h transmigration time to study “early” emigrated neutrophils and 10–18 h (overnight) for late-stage phenotypes. However, a time-course may also be used to study the induction of a particular phenotype (<a href="#bib7">Margaroli et al., 2021</a>).</li>
</ol>
</li>
<li>Remove the filters.
<ol type="a">
<li>We have found that epithelial cells and neutrophils that did not transmigrate can be released from the Alvetex filter using Accutase. Using trypsin is also possible, but we observed dramatically reduced viability of neutrophils that did not transmigrate.</li>
</ol>
</li>
<li>Gently pipette up and down 2 times with a P1000 to resuspend the transmigrated neutrophils and transfer to a 2 mL tube. Spin cells at 800 <b>×</b> <i>g</i> for 5 min at room temperature to pellet.</li>
<li>Discard or save the supernatant for other assays.
<ol type="a">
<li>Comparison of the components of the fluid prior to, and after, transmigration may give insight into the function of both the epithelial cells and transmigrating leukocytes.</li>
</ol>
</li>
<li>Wash the pellet by adding 1 mL of 1<b>×</b> PBS + 2.5 mM EDTA to the cells and spinning at 800 <b>×</b> <i>g</i> for 5 min. Discard the supernatant.</li>
<li>Resuspend the neutrophils 250 μL RPMI and make a 10 μL aliquot to quickly count the cells using a hemacytometer.
<ol type="a">
<li>The counting of transmigrated cells should be done quickly and efficiently so that the transmigrated neutrophils do not alter their phenotype or lose their phenotype.</li>
</ol>
</li>
<li>Adjust the concentration of neutrophils to 1.2∗10<sup>5</sup> cells/120 μL for the bacteria-killing assay.
<ol type="a">
<li>Other assays may require the cell concentration to be different.</li>
<li>With a transmigration efficiency of 60% (6∗10<sup>5</sup> cells; the low-end of what we typically observe), at least 5 bacteria-killing reactions can be conducted. This may be helpful for replicates or when multiple conditions (drugs added, strains used, etc.) are needed.</li>
</ol>
</li>
</ol>
<h3 id="sec2.5">Bacteria-killing assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes the use of <i>P. aeruginosa</i> (PA01), although other strains and bacteria can be used, to be combined with transmigrated neutrophils in a bacteria-killing assay. It may be helpful to have one person begin preparing the bacteria and subculturing while another handles transmigrated cells.</p>
<ol start="53">
<li>The day before the transmigration, inoculate <i>P. aeruginosa</i> (PA01) in 3 mL of LB media.
<ol type="a">
<li>If doing an 18-h transmigration the cells can be grown on the day that the transmigration is initiated.</li>
</ol>
</li>
<li>Grow overnight at 37°C at 200 rpm.
<ol type="a">
<li>To make sure that there is no contamination in the LB media it may be prudent to also have a mock inoculated tube.</li>
</ol>
</li>
<li>About 1.5 h before the finish of the transmigration, subculture the bacteria by pipetting 10 μL of overnight growth in 3 mL of fresh LB media. Grow at 37°C with shaking until OD<sub>600</sub> = 0.4–0.6.</li>
<li>Use the standard curve generated from the original preparation of the bacteria to convert the OD<sub>600</sub> to a concentration of cells (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1116-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Use of <i>P. aeruginosa</i> (PA01) in a bacteria-killing assay with neutrophils</div>
<p>(A) In order to grow bacteria to the correct density, a standard curve must be generated to relate the OD<sub>600</sub> to the colony forming units (CFU).</p>
<p>(B) When the bacteria and neutrophils are mixed together, they must be rotated end over end to prevent settling of the cells during this assay. The rotator is placed in a 37°C + 5% CO<sub>2</sub> incubator.</p>
<p>(C) PA01 grown at 37°C forms mucoid colonies that have poorly defined edges and are difficult to count. Grow the bacteria at a lower temperature, such as 30°C, to have better defined colonies.</p>
<p>(D) Neutrophils that were transmigrated towards LTB4, a control chemoattractant, had a higher bacteria-killing capacity than neutrophils that were transmigrated towards the airway supernatant from the sputum of patients with CF (CFASN). Statistics were calculated with a Mann-Whitney U test. Plotted is the median with interquartile range.</p>
</figcaption>
</figure>
</li>
<li>Centrifuge 2∗10<sup>8</sup> CFU at 16,000 <b>×</b> <i>g</i> for 2 min at 25°C.</li>
<li>Resuspend the bacteria pellet in 500 μL of warm RPMI + 10% FBS.</li>
<li>Incubate at 37°C + 5% CO<sub>2</sub> on a rotator end-over-end for 30 min.
<ol type="a">
<li>In our experience around the time that the 30-min incubation is over, the neutrophils have been counted and are ready for the bacteria-killing assay. Try to synchronize the end of this 30-min incubation with the completion of the post-transmigration neutrophil count (step 51).</li>
<li>The bacteria are opsonized and are also allowed to equilibrate to a medium that is more similar to that of the killing assay.</li>
</ol>
</li>
<li>Dilute 15 μL of bacteria in 3 mL of RPMI + 10% FBS warmed to 37°C (1.2 ∗ 10<sup>5</sup> CFU/60 μL).</li>
<li>Add 60 μL of bacteria (1.2∗10<sup>5</sup> CFU to 120 μL of neutrophils, equivalent to 1.2∗10<sup>5</sup> cells) for a multiplicity of infection of 1.
<ol type="a">
<li>For different MOIs, adjust the ratio of bacteria and/or cells.</li>
<li>As a control, make bacteria only (without neutrophils) tube. Add 60 μL of bacteria to 120 μL of RPMI.</li>
<li>The use of standard 1.7 mL snap-cap tubes is appropriate for this. Make sure that the tubes will fit into the rotator and will not open during the incubation period.</li>
</ol>
</li>
<li>Rotate end-over-end at 37°C + 5% CO<sub>2</sub> for 1 h (<a href="#fig3">Figure 3</a>B).</li>
<li>While the incubation is proceeding, prepare a 96-well plate for serial dilutions.
<ol type="a">
<li>Pipette 10 μL of 1% w/v Triton X-100 (1 g/100 mL) into Row A.
<ol type="i">
<li>The Triton X-100 will be at a final concentration of 0.1% w/v and will be used to lyse the neutrophils, but <i>P. aeruginosa</i> is resistant. Other bacteria or conditions may require different lysing conditions.</li>
</ol>
</li>
<li>Pipette 180 μL of 1<b>×</b> PBS into rows B through G.</li>
<li>Label LB agar plates.</li>
</ol>
</li>
</ol>
<ol start="64">
<li>Pipette 90 μL from the bacteria-killing reaction into a well of row A. Pipette up and down to mix and lyse the neutrophils to release intracellular bacteria.
<div class="optional">
<span class="optional-title">Optional:</span> (1) To quantify the number of bacteria in the extracellular fluid compared to bacteria that are now intracellular spin the cells at 100 <b>×</b> <i>g</i> for 7 min to pellet the neutrophils, but not the bacteria.</div>
<div class="optional">
<span class="optional-title">Optional:</span> (2) After centrifuging, to quantify the number of bacteria that are only associated with the neutrophils but not intracellular, incubate the pelleted neutrophils (from optional 1) in 100 μL of Accutase and incubate at 37°C + 5% CO<sub>2</sub> for 5 min. Neutralize by adding 400 μL of DMEM + 10% FBS. Spin at 100 <b>×</b> <i>g</i> for 7 min to pellet the neutrophils, but not the bacteria. We have observed that using trypsin results in significant neutrophil cell death.</div>
</li>
</ol>
<ol start="65">
<li>Using a multichannel pipette transfer 20 μL of sample from Row A into Row B. Mix up and down to evenly distribute the bacteria. Repeat 10<b>×</b> serial dilutions.
<ol type="a">
<li>We have found that using an electronic pipette that can automate some of the serial dilutions or multiple pipetting steps reduces hand strain and increases efficiency.</li>
</ol>
</li>
<li>Using a multichannel pipette, spot 5 μL of each dilution onto a LB plate.
<ol type="a">
<li>We have observed that 5–6 replicates of 6 dilutions can fit on an entire 10 cm LB plate (<a href="#fig3">Figure 3</a>C).</li>
<li>For more information on spotting of bacteria for quantification, refer to (<a href="#bib9">Thomas et al., 2015</a>).</li>
</ol>
</li>
<li>Leave the plates exposed to air, but covered or in a sterile hood, for 10 min to allow the spots to dry. Make sure the plates are flat so that multiple spots do not mix.
<ol type="a">
<li>It may be helpful to first dry the plates in an oven to further minimize the spreading.</li>
</ol>
</li>
<li>Place the lid on the plates and grow inverted at 30°C for 12–16 h.
<ol type="a">
<li>We have observed odd colony morphology when growing PA01 at 37°C. The lower temperature for growth seems to give a more defined morphology that is easier to count (See <a href="#sec6.15">troubleshooting 8</a>).</li>
</ol>
</li>
<li>When colonies are easily observable, but are not overgrown, count and record the number of colony-forming units (CFUs) in each spot. Refer to <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a> for how to quantify the total number of bacteria as well as calculate bacteria killing efficiency (<a href="#fig3">Figure 3</a>D).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Two potential outcome measures are the number and viability of cells that transmigrated through the filter and bacteria-killing efficiency.</p>
<p><b>Transmigration efficiency</b>: For CFASN conditions we typically observe that 60–100% of the input neutrophils will make it to the other side. The efficiency of monocyte transmigration is much lower (<a href="#bib1">Ford et al., 2021</a>). Furthermore, viability of the neutrophils after a 4-h transmigration is typically about 70–90% viable and after an 18-h transmigration may have ∼50% viability.</p>
<p><b>Bacteria Killing:</b> We typically observe that CFASN-conditioned neutrophils have a killing capacity of about 40%, while LTB<sub>4</sub>-conditioned neutrophils have a killing capacity of around 60%. The addition of drugs may modulate this ability [<a href="#fig3">Figure 3</a>D (<a href="#bib7">Margaroli et al., 2021</a>)].</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Quantification of bacteria-killing capacity:</p>
<ol>
<li>Count the number of CFUs in each 5 μL spot for which distinct colonies are viewable.</li>
<li>Average the number of colonies together for each dilution factor.</li>
</ol>
<ol start="3">
<li>Use the following formula to quantify the total number of live bacteria:
<p class="math-ml"><math> <mrow> <mfrac> <mrow> <mi>a</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mo>#</mo> <mi>s</mi> <mi>p</mi> <mi>o</mi> <mi>t</mi> <mi>s</mi> </mrow> <mrow> <mn>0.005</mn> <mspace></mspace> <mi>m</mi> <mi>L</mi> </mrow> </mfrac> <mo>∗</mo> <msup> <mn>10</mn> <mi>d</mi> </msup> <mo>∗</mo> <mn>1.111</mn> <mo>=</mo> <mfrac> <mrow> <mi>C</mi> <mi>F</mi> <mi>U</mi> </mrow> <mrow> <mi>m</mi> <mi>L</mi> </mrow> </mfrac> <mi>i</mi> <mi>n</mi> <mspace></mspace> <mi>t</mi> <mi>h</mi> <mi>e</mi> <mspace></mspace> <mi>o</mi> <mi>r</mi> <mi>i</mi> <mi>g</mi> <mi>i</mi> <mi>n</mi> <mi>a</mi> <mi>l</mi> <mspace></mspace> <mi>t</mi> <mi>u</mi> <mi>b</mi> <mi>e</mi> <mo>.</mo> </mrow> </math></p>
<ol type="a">
<li>Where <i>d</i> is the dilution factor. 1.111 comes from the Triton X-100 lysis. For the bacteria only control, this number should be relatively close to the expected input of 1.2 ∗ 10<sup>5</sup> CFU.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Use the following formula to quantify the bacteria-killing efficiency:
<p class="math-ml"><math> <mrow> <mfrac> <mrow> <mi>a</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mo>#</mo> <mi>s</mi> <mi>p</mi> <mi>o</mi> <mi>t</mi> <mi>s</mi> <mspace></mspace> <mi>i</mi> <mi>n</mi> <mspace></mspace> <mi>e</mi> <mi>x</mi> <mi>p</mi> <mi>e</mi> <mi>r</mi> <mi>i</mi> <mi>m</mi> <mi>e</mi> <mi>n</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>n</mi> <mi>d</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> <mo>∗</mo> <msup> <mn>10</mn> <mi>d</mi> </msup> </mrow> <mrow> <mi>a</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mo>#</mo> <mi>s</mi> <mi>p</mi> <mi>o</mi> <mi>t</mi> <mi>s</mi> <mspace></mspace> <mi>i</mi> <mi>n</mi> <mspace></mspace> <mi>b</mi> <mi>a</mi> <mi>c</mi> <mi>t</mi> <mi>e</mi> <mi>r</mi> <mi>i</mi> <mi>a</mi> <mspace></mspace> <mi>o</mi> <mi>n</mi> <mi>l</mi> <mi>y</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>n</mi> <mi>d</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> <mo>∗</mo> <mspace></mspace> <msup> <mn>10</mn> <mi>d</mi> </msup> </mrow> </mfrac> <mo>∗</mo> <mn>100</mn> <mo>=</mo> <mo>%</mo> <mi>b</mi> <mi>a</mi> <mi>c</mi> <mi>t</mi> <mi>e</mi> <mi>r</mi> <mi>i</mi> <mi>a</mi> <mspace></mspace> <mi>s</mi> <mi>u</mi> <mi>r</mi> <mi>v</mi> <mi>i</mi> <mi>v</mi> <mi>e</mi> <mi>d</mi> </mrow> </math></p>
<p class="math-ml"><math> <mrow> <mn>1</mn> <mo>−</mo> <mfrac> <mrow> <mi>a</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mo>#</mo> <mi>s</mi> <mi>p</mi> <mi>o</mi> <mi>t</mi> <mi>s</mi> <mspace></mspace> <mi>i</mi> <mi>n</mi> <mspace></mspace> <mi>e</mi> <mi>x</mi> <mi>p</mi> <mi>e</mi> <mi>r</mi> <mi>i</mi> <mi>m</mi> <mi>e</mi> <mi>n</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>n</mi> <mi>d</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> <mo>∗</mo> <msup> <mn>10</mn> <mi>d</mi> </msup> </mrow> <mrow> <mi>a</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mo>#</mo> <mi>s</mi> <mi>p</mi> <mi>o</mi> <mi>t</mi> <mi>s</mi> <mspace></mspace> <mi>i</mi> <mi>n</mi> <mspace></mspace> <mi>b</mi> <mi>a</mi> <mi>c</mi> <mi>t</mi> <mi>e</mi> <mi>r</mi> <mi>i</mi> <mi>a</mi> <mspace></mspace> <mi>o</mi> <mi>n</mi> <mi>l</mi> <mi>y</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>n</mi> <mi>d</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> <mo>∗</mo> <mspace></mspace> <msup> <mn>10</mn> <mi>d</mi> </msup> </mrow> </mfrac> <mo>∗</mo> <mn>100</mn> <mo>=</mo> <mo>%</mo> <mi>b</mi> <mi>a</mi> <mi>c</mi> <mi>t</mi> <mi>e</mi> <mi>r</mi> <mi>i</mi> <mi>a</mi> <mspace></mspace> <mi>k</mi> <mi>i</mi> <mi>l</mi> <mi>l</mi> <mi>e</mi> <mi>d</mi> <mspace></mspace> </mrow> </math></p>
<ol type="a">
<li>Where <i>d</i> is the dilution factor.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol allows for medium to low-throughput conditioning of neutrophils. At this time, more than 48 conditions can be very difficult to work with at once and certainly requires at least 2 or 3 researchers to participate for each experiment. In addition, since the transmigration fluid requires the airway supernatant from patient samples there is an inherent variability of the input conditions. Pooling multiple airway supernatant samples may result in averaging of conditions. Consideration should be taken into what conditions should be standardized.</p>
<p>We have been able to draw various parallels between airway leukocytes in-vivo and the metabolic and functional phenotypes of the transmigrating cells in our model including that CF airway neutrophils have transcriptional plasticity and that this directly correlates with their bacteria-killing capacity. (<a href="#bib2">Forrest et al., 2018</a>; <a href="#bib7">Margaroli et al., 2021</a>). Furthermore, we have also used tracheal aspirates from non-CF patients diagnosed with ARDS to show that ARDS-like neutrophils have varied killing capacity as determined by the fluid the neutrophils transmigrated towards (<a href="#bib4">Grunwell et al., 2019</a>). We have also used monocytes in lieu of neutrophils and shown that transmigrating monocytes adapt to their environment (<a href="#bib1">Ford et al., 2021</a>). Furthermore, H441 cells used as the human airway epithelial layer in the model may be swapped by a different cell line or primary cells.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The filter scaffold does not sit flat in the well.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Make sure that you are using a 12-well plate. There are also perforated tabs on the outside of each filter when they are being removed from their packaging. In order to fit into each well, the outermost tab, and <i>only</i> the outermost tab, should be broken. Do not break the scaffold (<a href="#fig1">Figure 1</a>A).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The collagen will not spread evenly across the filter.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Make sure that the collagen is at room temperature. Collagen at 4°C is too viscous and cannot be use to coat the filters. When pipetting collagen, it may be helpful to use a P1000 tip to pipette 200 μL and to use a reverse-pipetting technique. In addition, firmly tapping the side of the plate to agitate the collagen on the filter may also fill in any gaps. If the uneven coating of collagen does not resolve, try adding an additional 50 μL on the spot where the collagen is not present. Lastly, check that the brand of collagen is from Sigma-Aldrich (Catalog number: C3867-1VL). We have observed that other brands of collagen (even those labeled as Rat tail, Type I) have different viscosities or may come at a different stock concentration. In the latter case, diluting the collagen to a more workable concentration may be difficult requiring even mixing to generate a homogenous solution and having a non-precise pH may also result in the collagen crashing out of solution (<a href="#bib11">Vasilev et al., 1973</a>).</p>
<h3 id="sec6.5">Problem 3</h3>
<p>After separation on PMP, there is a large amount of red blood cells that sediment with the neutrophils.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The blood of some donors certainly can separate better than others. In the case of there being two distinct bands, but the neutrophil band has a large number of red blood cells (<a href="#fig2">Figure 2</a>A) spin the gradient again at 400 <b>×</b> <i>g</i> for 42 min at 25°C with minimal brake to see if separation improves. If there is still no improvement, collect the neutrophil band with all of the red blood cells and proceed to the hypotonic lysis step. Additional lysis steps for longer periods of time may have to be undertaken. Furthermore, if that particular donor is to be used again in the future increase the ratio of PMP:blood from 1:1 to 1.2–1.4:1. The greater amount of PMP may facilitate better separation and removal of the red blood cells. Lastly, there are some donors for which PMP does not work well. Consider using other purification kits such as EasySep (<a href="https://www.stemcell.com/easysep-direct-human-neutrophil-isolation-kit.html">https://www.stemcell.com/easysep-direct-human-neutrophil-isolation-kit.html</a>).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Low viability of blood neutrophils after purification from blood.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>After restoring osmolarity, make sure that the cells are constantly kept on ice or in the cold (step 26). Furthermore, the hypotonic lysis (step 28) should be done on ice for no more than 30 s, <i>total.</i> The 30 s is the time from the water being used to resuspend the pellet to the addition and mixing with 1.8% w/v NaCl. Make sure to account for pipetting time when counting. Furthermore, work efficiently, but gently with neutrophils. Do not vortex and do not pipette vigorously these cells – conduct the next step in the protocol as soon as the previous step is completed. For larger experiments, having multiple people conduct the purification or do preparation work can help to improve efficiency and thereby neutrophil viability.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Filter is dropped or falls off the plastic scaffold during the separation of component 1 and component 2.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Handle the filters gently and do not apply excessive force to separate the plastic parts. The process of flipping the filters requires more finesse than it does force. Furthermore, for flipping the filters it is helpful to constantly think about what side of the filters the cells are on and what direction they are facing (<a href="#mmc1">Methods videos S1</a> and <a href="#mmc2">S2</a>). The process of flipping filters gets easier with practice. If a filter gets dropped there is not a way to salvage it.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>There are air bubbles underneath the filter either at the media changes with AIM+USG or with the addition of transmigration fluid. If filters are dropped that filter cannot be salvaged. Practice with unused filters can also be done until the flipping process can be conducted with few lost filters.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>When placing the filter apparatuses back into the original wells, holding the plate at a slight angle and slowly lowering the filter apparatus into the well can minimize the chance of a large air bubble forming under the filter apparatus. This may be due to the tendency of the membrane to bend during the two-week culturing, resulting in a dome shape following inversion of the membrane. In the event an air bubble forms, follow the below steps in the written order to remove it:</p>
<p>Remove the filter apparatus and return it to the well several times</p>
<p>Remove the filter apparatus, pop large bubbles in the well with a sterile pipette tip</p>
<p>Add additional media under the well</p>
<h3 id="sec6.13">Problem 7</h3>
<p>The filter has a hole in it either from piercing with a pipette tip or from ripping it with tweezers during the flipping/transmigration process.</p>
<h3 id="sec6.14">Potential solution</h3>
<p>Unfortunately, that filter is no longer usable. Discard the filter.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>PA01 colonies have a mucoid-like morphology that makes counting colonies difficult.</p>
<h3 id="sec6.16">Potential solution</h3>
<p>Regrow the PA01 at a lower temperature. We have found that 30°C works well for us, but this should be determined empirically for each lab as there are different medias, incubators, and atmospheric conditions between locations. Furthermore, avoid growing the PA01 in liquid culture for more than 16–18 h.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ford, B.D., Moncada Giraldo, D., Margaroli, C., Giacalone, V.D., Brown, M.R., Peng, L., and Tirouvanziam, R. (2021). Functional and transcriptional adaptations of blood monocytes recruited to the cystic fibrosis airway microenvironment in vitro. Int. J. Mol. Sci. <i>22</i>, 2530. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00598-0/sref1">View at publisher</a></p>
<p id="bib2">Forrest, O.A., Ingersoll, S.A., Preininger, M.K., Laval, J., Limoli, D.H., Brown, M.R., Lee, F.E., Bedi, B., Sadikot, R.T., Goldberg, J.B., et al. (2018). Frontline Science: pathological conditioning of human neutrophils recruited to the airway milieu in cystic fibrosis. J. Leukoc. Biol. <i>104</i>, 665-675. <a class="external-link" href="https://doi.org/10.1002/JLB.5HI1117-454RR">https://doi.org/10.1002/JLB.5HI1117-454RR</a></p>
<p id="bib3">Goldmann, T., Pedersen, F., Seehase, S., Marwitz, S., Lang, D.S., Kirsten, A.M., Zabel, P., Vollmer, E., Magnussen, H., Rabe, K.F., and Watz, H. (2013). The effect of dithiothreitol on the transcriptome of induced sputum cells. Respiration <i>86</i>, 262-263. <a class="external-link" href="https://doi.org/10.1159/000348392">https://doi.org/10.1159/000348392</a></p>
<p id="bib4">Grunwell, J.R., Giacalone, V.D., Stephenson, S., Margaroli, C., Dobosh, B.S., Brown, M.R., Fitzpatrick, A.M., and Tirouvanziam, R. (2019). Neutrophil dysfunction in the airways of children with acute respiratory failure due to lower respiratory tract viral and bacterial coinfections. Sci. Rep. <i>9</i>, 2874. <a class="external-link" href="https://doi.org/10.1038/s41598-019-39726-w">https://doi.org/10.1038/s41598-019-39726-w</a></p>
<p id="bib5">Loppow, D., Böttcher, M., Gercken, G., Magnussen, H., and Jörres, R.A. (2000). Flow cytometric analysis of the effect of dithiothreitol on leukocyte surface markers. Eur. Respir. J. <i>16</i>, 324-329. <a class="external-link" href="https://doi.org/10.1034/j.1399-3003.2000.16b22.x">https://doi.org/10.1034/j.1399-3003.2000.16b22.x</a></p>
<p id="bib6">Makam, M., Diaz, D., Laval, J., Gernez, Y., Conrad, C.K., Dunn, C.E., Davies, Z.A., Moss, R.B., Herzenberg, L.A., Herzenberg, L.A., and Tirouvanziam, R. (2009). Activation of critical, host-induced, metabolic and stress pathways marks neutrophil entry into cystic fibrosis lungs. Proc. Natl. Acad. Sci. U S A <i>106</i>, 5779-5783. <a class="external-link" href="https://doi.org/10.1073/pnas.0813410106">https://doi.org/10.1073/pnas.0813410106</a></p>
<p id="bib7">Margaroli, C., Moncada-Giraldo, D., Gulick, D.A., Dobosh, B., Giacalone, V.D., Forrest, O.A., Sun, F., Gu, C., Gaggar, A., Kissick, H., et al. (2021). Transcriptional firing represses bactericidal activity in cystic fibrosis airway neutrophils. Cell Rep. Med. <i>2</i>, 100239. <a class="external-link" href="https://doi.org/10.1016/j.xcrm.2021.100239">https://doi.org/10.1016/j.xcrm.2021.100239</a></p>
<p id="bib8">Nathan, C.F. (1987). Neutrophil activation on biological surfaces. Massive secretion of hydrogen peroxide in response to products of macrophages and lymphocytes. J. Clin. Invest. <i>80</i>, 1550-1560. <a class="external-link" href="https://doi.org/10.1172/jci113241">https://doi.org/10.1172/jci113241</a></p>
<p id="bib9">Thomas, P., Sekhar, A.C., Upreti, R., Mujawar, M.M., and Pasha, S.S. (2015). Optimization of single plate-serial dilution spotting (SP-SDS) with sample anchoring as an assured method for bacterial and yeast cfu enumeration and single colony isolation from diverse samples. Biotechnol. Rep. <i>8</i>, 45-55. <a class="external-link" href="https://doi.org/10.1016/j.btre.2015.08.003">https://doi.org/10.1016/j.btre.2015.08.003</a></p>
<p id="bib10">Tirouvanziam, R., Gernez, Y., Conrad, C.K., Moss, R.B., Schrijver, I., Dunn, C.E., Davies, Z.A., Herzenberg, L.A., and Herzenberg, L.A. (2008). Profound functional and signaling changes in viable inflammatory neutrophils homing to cystic fibrosis airways. Proc. Natl. Acad. Sci. U S A <i>105</i>, 4335-4339. <a class="external-link" href="https://doi.org/10.1073/pnas.0712386105">https://doi.org/10.1073/pnas.0712386105</a></p>
<p id="bib11">Vasilev, M.P., Kotetskii, V.V., Pukhova, Z.I., Volf, L.A., and Istranov, L.P. (1973). Collagen precipitation from solutions. Fibre Chem. <i>4</i>, 486-488. <a class="external-link" href="https://doi.org/10.1007/BF00548873">https://doi.org/10.1007/BF00548873</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Rabindra Tirouvanziam (<a href="mailto:tirouvanziam@emory.edu">tirouvanziam@emory.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This protocol paper did not generate or analyze any datasets or code.</p>
<h3>Acknowledgments</h3>
<p>The authors would like to acknowledge funding by the <a href="https://doi.org/10.13039/100000002">NIH</a> (R01HL126603 to R.T.), the <a href="https://doi.org/10.13039/100006501">Cystic Fibrosis Foundation</a> (TIROUV17G0 and TIROUV19G0 to R.T.), and the <a href="https://doi.org/10.13039/100012423">Children's Healthcare of Atlanta</a> and <a href="https://doi.org/10.13039/100007623">Emory University</a>'s CF Discovery Core for access to blood and airway samples.</p>
<h3>Author contributions</h3>
<p>Methodology and writing- original draft preparation, B.D.; methodology and writing- reviewing and editing, V.D.G. and C.M.; conceptualization, writing-reviewing and editing, and supervision, R.T.</p>
<h3>Declaration of interests</h3>
<p>Contents within this manuscript are included in patent USPTO 10670594.</p>
</section>
</article>