<article>
<section>
<h1 id="header">Analysis of histone modifications in mouse neocortical neural progenitor-stem cells at various developmental stages</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Masafumi Tsuboi<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup> and Yukiko Gotoh<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup></p>
<p id="aff1"><sup>1</sup>Graduate School of Engineering, The University of Tokyo, Tokyo 113-0033, Japan</p>
<p id="aff2"><sup>2</sup>Graduate School of Pharmaceutical Sciences, The University of Tokyo, Tokyo 113-0033, Japan</p>
<p id="aff3"><sup>3</sup>International Research Center for Neurointelligence (WPI-IRCN), The University of Tokyo, Tokyo 113-0033, Japan</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:tsuboi@chembio.t.u-tokyo.ac.jp">tsuboi@chembio.t.u-tokyo.ac.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100763">10.1016/j.xpro.2021.100763</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Dynamic changes in histone modifications mediated by Polycomb group proteins can be indicative of the transition of gene repression mode during development. Here, we present methods for the isolation of mouse neocortical neural progenitor-stem cells (NPCs) and their culture, followed by chromatin immunoprecipitation quantitative PCR (ChIP-qPCR) techniques to examine changes in histone H2A ubiquitination patterns at various developmental stages. This protocol can be applied for both in vitro NPCs and NPCs directly isolated from mouse neocortices.</p>
<p>For complete details on the use and execution of this protocol, please refer to (<a href="https://doi.org/10.1016/j.devcel.2018.11.018">Tsuboi et al., 2018</a>).</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>
<i>In vitro</i> culture system for obtaining neocortical neurogenic and astrogliogenic NPCs</li>
<li>Analysis for histone modification profiles in mouse neocortical NPCs by ChIP-qPCR</li>
<li>This ChIP protocol is optimized to yield a high signal-to-noise ratio for H2Aub analysis</li>
<li>Applicable for both in vitro NPCs and NPCs directly isolated from mouse neocortices</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/951-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>All developmental genes are maintained in a poised state for future transcriptional activation in pluripotent stem cells, whereas a subset of genes becomes persistently repressed after fate restriction during commitment to a specific lineage. PcG proteins mediate both temporary and persistent repression of the developmental genes during development. PcG comprises two main multisubunit complexes referred to as Polycomb repressive complex (PRC) 1 and PRC2. PRC2 mediates trimethylation of histone 3 at lysine-27 (H3K27me3), which is recognized by chromobox proteins (CBX) -containing PRC1 (<a href="#bib4">Cao et al., 2002</a>; <a href="#bib9">Entrevan et al., 2016</a>; <a href="#bib16">Min et al., 2003</a>; <a href="#bib23">Wang et al., 2004b</a>; <a href="#bib24">Yu et al., 2019</a>). PRC1 contains the E3 ubiquitin ligases Ring1A and Ring1B, which catalyze H2A ubiquitination (H2Aub) at K118 and K119 (<a href="#bib6">de Napoles et al., 2004</a>; <a href="#bib7">Elderkin et al., 2007</a>; <a href="#bib14">McGinty et al., 2014</a>; <a href="#bib22">Wang et al., 2004a</a>). H2Aub deposition is enriched at developmental gene loci in mouse embryonic stem cells (ESCs) (<a href="#bib2">Blackledge et al., 2020</a>; <a href="#bib8">Endoh et al., 2012</a>; <a href="#bib20">Tamburri et al., 2020</a>) and dynamically regulated at a subset of neurogenic gene loci during neuronal fate restriction in NPCs (<a href="#bib21">Tsuboi et al., 2018</a>). For instance, the level of H2Aub significantly increases at neurogenic genes such as <i>Neurog1</i>, <i>Fezf2</i>, <i>Tcfap2</i> and, <i>Lef1</i> when neurogenic potential of NPCs becomes restricted at the onset of the astrogliogenic phase (<a href="#bib21">Tsuboi et al., 2018</a>). By optimization of some conditions including antibody concentration, this protocol enables us to efficiently perform conventional chromatin immunoprecipitation (ChIP) analysis of H2Aub at K118 and K119 from neocortical NPCs. This ChIP protocol can be applied to both <i>in vitro</i> NPC cultures and NPCs directly isolated from mouse neocortices (<a href="#bib21">Tsuboi et al., 2018</a>). The following protocol is for the former, but the same is applicable to the latter after isolation of NPCs according to the method described in (<a href="#bib13">Kishi and Gotoh, 2021</a>).</p>
<h3 id="sec1.1">Prepare solutions for NPC culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol>
<li>Prepare solutions listed below (see detailed recipes in the “<a href="#materials-and-equipment">Materials and equipment</a>” section). Make sure that the volume of all solutions is sufficient for culture of mouse neocortical NPCs. The approximate volume required for 1.0 × 10<sup>6</sup> of NPCs is indicated.
<ol type="a">
<li>DMEM-F12 containing 1% penicillin-streptomycin (100 mL, 4°C)</li>
<li>Trypsin solution (20 mL, −20°C)</li>
<li>DNase I solution (500 μL, −20°C)</li>
<li>Ovomucoid solution (200 μL, −20°C)</li>
<li>Artificial cerebrospinal fluid (aCSF) (20 mL, 4°C)</li>
<li>Low Ca<sup>2+</sup>-aCSF (lc-aCSF) (4°C)</li>
<li>Phosphate-buffered saline (PBS) (Room temperature; i.e., 18°C–24°C)</li>
<li>Fibroblast growth factor 2 (FGF2) solution (50 μL, −80°C)</li>
<li>Epidermal growth factor (EGF) solution (50 μL, −80°C)</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare all the solutions excluding PBS aseptically under laminar flow bench. Ovomucoid, aCSF and lc-aCSF solutions should be filtrated with a 0.22 μm filter (Millipore, SLGSR33SB for Ovomucoid and SCGPS05RE for aCSF and lc-aCSF). PBS is dissolved and then should be autoclaved.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Dissolve the reagents with ice-cold PBS for preparing FGF2 and EGF solutions. Aliquot 100 μl of each solution into 0.6 ml tubes and flash-freeze them rapidly with liquid nitrogen and store it at −80°C.</div>
<ol start="2">
<li>Before beginning dissection of the mouse neocortex, trypsin solution, DNase I solution, and ovomucoid solution (stored at −20°C), FGF2 and EGF solution (stored at −80°C) should be thawed on ice.</li>
<li>Incubate trypsin solution and aCSF in a water bath at 37°C for ~10 min before beginning dissociation of the dissected neocortex.</li>
</ol>
<h3 id="sec1.2">Prepare solutions for ChIP</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="4">
<li>Prepare solutions listed below (Tables in Materials and Equipment). Make sure that the volume of all solutions is sufficient for ChIP and that the solutions a, d–h have been adjusted to the indicated temperature before use. The approximate volume required for 1.0 × 10<sup>6</sup> of NPCs is indicated.
<ol type="a">
<li>PBS containing 0.5% bovine serum albumin (BSA) (500 μL, 4°C)</li>
<li>11% formaldehyde solution (150 μL)</li>
<li>2.5 M glycine (100 μL)</li>
<li>Swelling buffer (1.5 mL, 4°C)</li>
<li>Cell lysis buffer (10 mL, 4°C)</li>
<li>Wash buffer A (2 mL, 4°C)</li>
<li>Wash buffer B (2 mL, 4°C)</li>
<li>TE buffer (2 mL, 4°C)</li>
<li>Elution buffer (0.5 mL)</li>
<li>Proteinase K solution (20 μL)</li>
<li>3 M sodium acetate, pH 5.2 (50 μL)</li>
<li>80% ethanol (1 mL)</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Adjust pH of 3 M sodium acetate with acetic acid while stirring.</div>
<h3 id="sec1.3">Design primers for qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<ol start="5">
<li>Obtain genomic DNA sequences for the target gene loci from the mouse genomic DNA database with the UCSC genome browser (Mouse GRCm39/mm39: <a href="https://genome.ucsc.edu/">https://genome.ucsc.edu/</a>). ChIP-seq databases for Ring1B and H3K27me3 in the developing mouse neocortices (<a href="#bib1">Albert et al., 2017</a>; <a href="#bib3">Bonev et al., 2017</a>) are informative to identify the sites of H2Aub deposition.</li>
<li>Design quantitative polymerase chain reaction (qPCR) primer sets with the use of LightCycler Probe Design Software 2.0 (Roche). Design primers as follows; Tm value of primer, 60.0°C; primer concentration, 0.5 μM; primer size, 19~ bp; amplicon size, 80–200 bp. Confirm that designed primer sets show no non-specific binding with the use of the BLAST tool.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other qPCR design tools can be adopted for design of qPCR primer sets.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-H2Aub (E6C5) (1:62.5)</td>
<td>Merck Millipore</td>
<td>Cat#05–678; RRID: AB_309899</td>
</tr>
<tr>
<td>Rabbit anti-mouse IgMμ (1:125)</td>
<td>Merck Millipore</td>
<td>Cat#12–488; RRID: AB_390193</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Dulbecco’s modified Eagle’s medium (DMEM)–F12 (1:1, v/v)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#1130-032</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>B-27</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#17504-44</td>
</tr>
<tr>
<td>Human FGF2</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#PHG0023</td>
</tr>
<tr>
<td>Human EGF</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#PHG0314</td>
</tr>
<tr>
<td>Neuron Dissociation Solutions</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>Cat#291-78001</td>
</tr>
<tr>
<td>Trypsin (2.5%), no phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15090046</td>
</tr>
<tr>
<td>Ovomucoid (trypsin inhibitor from chicken egg white Type II-O)</td>
<td>Sigma</td>
<td>Cat#T9253</td>
</tr>
<tr>
<td>Bovine serum albumin solution 7.5% in DPBS</td>
<td>Sigma</td>
<td>Cat#A8412</td>
</tr>
<tr>
<td>DNase I</td>
<td>Roche</td>
<td>Cat#11284932001</td>
</tr>
<tr>
<td>PBS</td>
<td>NISSUI</td>
<td>Cat#05913</td>
</tr>
<tr>
<td>0.4w/v% Trypan blue solution</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>Cat#207-17081</td>
</tr>
<tr>
<td>Bovine serum albumin lyophilized powder, ≥96%</td>
<td>Sigma</td>
<td>Cat#A2153</td>
</tr>
<tr>
<td>Pierce™ 16% formaldehyde (w/v), methanol-free</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#28908</td>
</tr>
<tr>
<td>Aprotinin from bovine lung</td>
<td>Sigma</td>
<td>Cat#A6279</td>
</tr>
<tr>
<td>Hepes</td>
<td>Nacalai</td>
<td>Cat#17546-05</td>
</tr>
<tr>
<td>Nonidet P-40</td>
<td>Sigma</td>
<td>Cat#21-3277</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>Cat# 042-29222</td>
</tr>
<tr>
<td>EDTA</td>
<td>Nacalai</td>
<td>Cat#15105-35</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Nacalai</td>
<td>Cat#12967-45</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate (SDS)</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>Cat#194-13985</td>
</tr>
<tr>
<td>Lithium chloride (LiCl)</td>
<td>Nacalai</td>
<td>Cat#20645-92</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Nacalai</td>
<td>Cat#29442-14</td>
</tr>
<tr>
<td>Protein A/G beads</td>
<td>Pierce</td>
<td>Cat#88803</td>
</tr>
<tr>
<td>Dynabeads protein A</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10002D</td>
</tr>
<tr>
<td>Phenol-chloroform-isoamyl alcohol</td>
<td>Nacalai</td>
<td>Cat#25970-56</td>
</tr>
<tr>
<td>Sodium acetate trihydrate</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>Cat#198-01055</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>Cat#017-00256</td>
</tr>
<tr>
<td>Ethanol</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>Cat#054-07225</td>
</tr>
<tr>
<td>Glycogen solution</td>
<td>FUJIFILM Wako Pure Chemical Corporation</td>
<td>Cat#076-06621</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>THUNDERBIRD® SYBR® qPCR Mix</td>
<td>Toyobo</td>
<td>Cat#QPS-201</td>
</tr>
<tr>
<td>Neural Tissue Dissociation Kit</td>
<td>Sumitomo Bakelite</td>
<td>Cat#MB-X9901</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: Jcl:ICR</td>
<td>CLEA Japan</td>
<td>MGI Cat# 5652875; RRID:MGI:5652875</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>
<i>∗Neurog1</i> mRNA F: 5′-ATCACCACTCTCTGACCC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Neurog1</i> mRNA R: 5′-GAGGAAGAAAGTATTGATGTTGCCTTA-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Fezf2</i> mRNA F: 5′-CTCTACTGACAGCAAACCCA-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Fezf2</i> mRNA R: 5′-CTTTGCACACAAACGGTCT-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Lef1</i> mRNA F: 5′-CCGGTGGCTCTTTATTGT-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Lef1</i> mRNA R: 5′-CAAGAGGCTCCCTTTGC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Tcfap2c</i> mRNA F: 5′-TTGTCTCATTTCAGCCTGATAAC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Tcfap2c</i> mRNA R: 5′-CGATTAGAGCCTCCTTGATATAATTC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Gfap</i> mRNA F: 5′-TCCCTAGAGCGGCAAAT-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Gfap</i> mRNA R: 5′-TCCTCCTTGAGGCTTTGG-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Gapdh</i> mRNA F: 5′-TGGGTGTGAACCACGAG-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗Gapdh</i> mRNA R: 5′-AAGTTGTCATGGATGACCTT-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Fezf2</i> F: 5′-TTCCTTGGGTGATTCACAAC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Fezf2</i> R: 5′-CTTTGAGGTAGCGCGTC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Lef1</i> F: 5′-GGCAGCTCTTTGCTTTGAC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Lef1</i> R: 5′-CCCGATCCCTCTTTGTTC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Tcfap2c</i> F: 5′-CTTCCCATTAAGGCGTTCG-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Tcfap2c</i> R: 5′-GCCGTGGGTGTGTAGAG-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Cdkn2d</i> F: 5′-CGGAGACCCAGGACAG-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Cdkn2d</i> R: 5′-CCAGGATTCCGGTGC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Gapdh</i> F: 5′-TGCAGTCCGTATTTATAGGAACC-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Gapdh</i> R: 5′-CTTGAGCTAGGACTGGATAAGCA-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Cdkn1a</i> F: 5′-CACCTCTAAGGCCAGCTA-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Cdkn1a</i> R: 5′-AGCAATGTCAAGAGTCGG-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Cdkn1c</i> F: 5′-CTGGCTGATTGGTGATGG-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Cdkn1c</i> R: 5′-GACTGAGAGCAAGCGAACA-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Dlk1</i> F: 5′-ACGTTCTAAGGTGCAACT-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td>
<i>∗∗Dlk1</i> R: 5′-GGAAATTAGGGACTACGACTG-3′</td>
<td>(<a href="#bib21">Tsuboi et al., 2018</a>)</td>
<td><a href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>LightCycler Probe Design Software 2.0</td>
<td>Roche</td>
<td>Cat#04342054001</td>
</tr>
<tr>
<td>LightCycler 480 Software, version 1.5.1</td>
<td>Roche</td>
<td><a href="https://lifescience.roche.com/global_en/products/lightcycler14301-480-software-version-15.html">https://lifescience.roche.com/global_en/products/lightcycler14301-480-software-version-15.html</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>100-mm Plastic culture dish</td>
<td>VIOLAMO</td>
<td>Cat#VTC-D100</td>
</tr>
<tr>
<td>Millex-GS Syringe Filter Unit, 0.22 μm</td>
<td>Merck Millipore</td>
<td>Cat#SLGSR33SB</td>
</tr>
<tr>
<td>Steritop Threaded Bottle Top Filter</td>
<td>Merck Millipore</td>
<td>Cat#SCGPS05RE</td>
</tr>
<tr>
<td>Cell counting chamber according to Neubauer</td>
<td>Hirschmann</td>
<td>Cat#8100104</td>
</tr>
<tr>
<td>Inverted microscope Eclipse Ts2-FL</td>
<td>Nikon</td>
<td>N/A</td>
</tr>
<tr>
<td>10-mL Disposable pipette</td>
<td>VIOLAMO</td>
<td>Cat#4733</td>
</tr>
<tr>
<td>Heat block</td>
<td>Astec</td>
<td>Cat#BI-516S</td>
</tr>
<tr>
<td>Vortex mixer</td>
<td>LMS</td>
<td>Cat#VTX-3000L</td>
</tr>
<tr>
<td>Tube rotator</td>
<td>TAITEC</td>
<td>Cat#AT-5</td>
</tr>
<tr>
<td>27G Injection needle</td>
<td>Terumo Corporation</td>
<td>Cat#NN-2719S</td>
</tr>
<tr>
<td>Aspirator</td>
<td>AS ONE</td>
<td>Cat#1-689-04</td>
</tr>
<tr>
<td>LightCycler 480 System II</td>
<td>Roche</td>
<td>N/A</td>
</tr>
<tr>
<td>DynaMag-2 Magnet</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12321D</td>
</tr>
<tr>
<td>Prelubricated microcentrifuge tubes (1.7 mL)</td>
<td>Costar</td>
<td>Cat#3207</td>
</tr>
<tr>
<td>Cool rack for 96-well plate</td>
<td>AS ONE</td>
<td>Cat#IO-2</td>
</tr>
<tr>
<td>Sapphire PCR microplate, 96 Well</td>
<td>Greiner Bio-One</td>
<td>Cat#669285</td>
</tr>
<tr>
<td>LightCycler® 480 Sealing Foil</td>
<td>Roche</td>
<td>Cat#04729757001</td>
</tr>
<tr>
<td>Handy Sonic UR-20P</td>
<td>Tomy</td>
<td>Cat#UR-20P</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗ Primers for qPCR of mRNA expression analysis<br> ∗∗ Primers for ChIP-qPCR analysis</p>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Buffers for neocortical NPC culture</h3>
<table id="undtbl1">
<caption>DMEM-F12 containing 1% penicillin-streptomycin</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM-F12</td>
<td> </td>
<td>495 mL</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>1% (v/v)</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, and do not store more than 3 months.</p>
<table id="undtbl2">
<caption>Trypsin</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Trypsin (2.5%), no phenol red</td>
<td>0.0625% (v/v)</td>
<td>2.5%</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>0.01875% (v/v)</td>
<td>7.5%</td>
<td>0.125 mL</td>
</tr>
<tr>
<td>lc-aCSF</td>
<td> </td>
<td> </td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at –20°C, and do not store more than 6 months.</p>
<table id="undtbl3">
<caption>Ovomucoid</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ovomucoid</td>
<td>70 mg/mL</td>
<td>1 g</td>
</tr>
<tr>
<td>aCSF</td>
<td> </td>
<td>14.3 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>14.3 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at –20°C, and do not store more than 6 months.</p>
<table id="undtbl4">
<caption>DNase I</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNase I</td>
<td>5 mg/mL</td>
<td>100 mg</td>
</tr>
<tr>
<td>lc-aCSF</td>
<td> </td>
<td>20 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at –20°C, and do not store more than 6 months.</p>
<table id="undtbl5">
<caption>aCSF</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>124 mM</td>
<td>7.25 g</td>
</tr>
<tr>
<td>KCl</td>
<td>5 mM</td>
<td>0.37 g</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>/6H<sub>2</sub>O</td>
<td>1.3 mM</td>
<td>0.26 g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>/2H<sub>2</sub>O</td>
<td>2 mM</td>
<td>0.29 g</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>26 mM</td>
<td>2.18 g</td>
</tr>
<tr>
<td>D-Glucose</td>
<td>10 mM</td>
<td>1.80 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td>Up to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td>
<b>1</b> L</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, and do not store more than 3 months.</p>
<table id="undtbl6">
<caption>lc-aCSF</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>124 mM</td>
<td>7.25 g</td>
</tr>
<tr>
<td>KCl</td>
<td>5 mM</td>
<td>0.37 g</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>/6H<sub>2</sub>O</td>
<td>3.2 mM</td>
<td>0.65 g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>/2H<sub>2</sub>O</td>
<td>0.1 mM</td>
<td>0.015 g</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>26 mM</td>
<td>2.18 g</td>
</tr>
<tr>
<td>D-Glucose</td>
<td>10 mM</td>
<td>1.80 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td>Up to 1 Ll</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td>
<b>1</b> Ll</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, and do not store more than 3 months.</p>
<table id="undtbl7">
<caption>PBS</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>4.8 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–24°C, and do not store more than 1 year.</p>
<table id="undtbl8">
<caption>FGF2 solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FGF2</td>
<td>20 μg/mL</td>
<td>25 μg</td>
</tr>
<tr>
<td>PBS</td>
<td> </td>
<td>1.25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>1.25 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at –80°C, and do not store more than 1 year.</p>
<table id="undtbl9">
<caption>EGF solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EGF</td>
<td>20 μg/mL</td>
<td>100 μg</td>
</tr>
<tr>
<td>PBS</td>
<td> </td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at –80°C, and do not store more than 1 year.</p>
<table id="undtbl10">
<caption>DMEM-F12 medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM-F12</td>
<td> </td>
<td>24.45 mL</td>
</tr>
<tr>
<td>B-27</td>
<td>2% (v/v)</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>FGF2 solution</td>
<td>20 ng/mL</td>
<td>0.025 mL</td>
</tr>
<tr>
<td>EGF solution</td>
<td>20 ng/mL</td>
<td>0.025 mL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>25 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Please prepare the required volume of fresh medium just before use.</p>
<h3 id="sec2.2">Buffers for H2Aub ChIP</h3>
<table id="undtbl11">
<caption>PBS containing 0.5% bovine serum albumin (BSA)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bovine serum albumin lyophilized powder</td>
<td>0.5% (w/v)</td>
<td>0.05 g</td>
</tr>
<tr>
<td>PBS</td>
<td> </td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, and do not store more than 1 month.</p>
<table id="undtbl12">
<caption>11% formaldehyde solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>16% formaldehyde</td>
<td>11% (v/v)</td>
<td>6.875 mL</td>
</tr>
<tr>
<td>PBS</td>
<td> </td>
<td>3.125 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Please prepare the required volume of fresh solution just before use.</p>
<table id="undtbl13">
<caption>2.5 M Glycine</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glycine</td>
<td>2.5 M</td>
<td>0.94 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at –20°C, and do not store more than 6 months.</p>
<table id="undtbl14">
<caption>Swelling buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes (pH 7.9)</td>
<td>20 mM</td>
<td>1 M</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>1.5 mM</td>
<td>1 M</td>
<td>0.0075 mL</td>
</tr>
<tr>
<td>KCl</td>
<td>10 mM</td>
<td>1 M</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>NP-40</td>
<td>0.1% (v/v)</td>
<td>10% (v/v)</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>Aprotinin<sup><a href="#dtbl15fna">a</a></sup>
</td>
<td>0.2% (v/v)</td>
<td> </td>
<td>0.01 mL</td>
</tr>
<tr>
<td>DTT<sup><a href="#dtbl15fna">a</a></sup>
</td>
<td>1 mM</td>
<td>1 M</td>
<td>0.005 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Up to 5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–24°C, and do not store more than 6 months.</p>
<p id="dtbl15fna" class="table-footnote"><sup>a</sup>Add fresh before use</p>
<table id="undtbl15">
<caption>Cell lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (pH 8.0)</td>
<td>10 mM</td>
<td>1 M</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>140 mM</td>
<td>5 M</td>
<td>1.4 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>1 mM</td>
<td>0.5 M</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>1% (v/v)</td>
<td>10% (v/v)</td>
<td>5 mL</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.1% (w/v)</td>
<td> </td>
<td>0.05 g</td>
</tr>
<tr>
<td>SDS</td>
<td>0.1% (w/v)</td>
<td>10% (w/v)</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, and do not store more than 6 months.</p>
<table id="undtbl16">
<caption>Wash buffer A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (pH 8.0)</td>
<td>10 mM</td>
<td>1 M</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>500 mM</td>
<td>5 M</td>
<td>5 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>1 mM</td>
<td>0.5 M</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>1% (v/v)</td>
<td>10% (v/v)</td>
<td>5 mL</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.1% (w/v)</td>
<td> </td>
<td>0.05 g</td>
</tr>
<tr>
<td>SDS</td>
<td>0.1% (w/v)</td>
<td>10% (w/v)</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, and do not store more than 6 months.</p>
<table id="undtbl17">
<caption>Wash buffer B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (pH 8.0)</td>
<td>10 mM</td>
<td>1 M</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>1 mM</td>
<td>0.5 M</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>LiCl</td>
<td>250 mM</td>
<td>5 M</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>NP-40</td>
<td>0.5% (v/v)</td>
<td>10% (v/v)</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.5% (w/v)</td>
<td> </td>
<td>0.25 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, and do not store more than 6 months.</p>
<table id="undtbl18">
<caption>TE buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (pH 8.0)</td>
<td>10 mM</td>
<td>1 M</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>1 mM</td>
<td>0.5 M</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–24°C, and do not store more than 6 months.</p>
<table id="undtbl19">
<caption>Elution buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (pH 8.0)</td>
<td>10 mM</td>
<td>1 M</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>5 mM</td>
<td>0.5 M</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>300 mM</td>
<td>5 M</td>
<td>3 mL</td>
</tr>
<tr>
<td>SDS</td>
<td>0.5% (w/v)</td>
<td>10% (w/v)</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–24°C, and do not store more than 6 months.</p>
<table id="undtbl20">
<caption>Proteinase K solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Proteinase K</td>
<td>10 mg/mL</td>
<td>100 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td>Up to 10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at –20°C, and do not store more than 1 year.</p>
<table id="undtbl21">
<caption>3M sodium acetate (Adjust to pH 5.2)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sodium acetate trihydrate</td>
<td>3 M</td>
<td>40.8 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–24°C, and do not store more than 1 year.</p>
<table id="undtbl22">
<caption>80% Ethanol</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ethanol</td>
<td>80% (v/v)</td>
<td>40 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td> </td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at –20°C, and do not store more than 1 year.</p>
<h3 id="sec2.3">Equipment</h3>
<ul>
<li>Handy Sonic UR-20P (Tomy)</li>
</ul>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol was developed with the use of a Handy Sonic UR-20P device to shear chromatin DNA. Picoruptor (diagenode, P-150504) can also be used for this purpose.</div>
<ul>
<li>LightCycler 480 System II (Roche) + THUNDERBIRD® SYBR® qPCR Mix (Toyobo)</li>
</ul>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol is based on a qPCR procedure performed with a LightCycler 480 System II (Roche). Any other qPCR machine can also be used.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol describes a qPCR procedure performed with THUNDERBIRD® SYBR® qPCR Mix in a reaction volume of 16 μl. SYBR Premix Ex Taq (Takara), LightCycler 480 SYBR Green I Master (Roche), and KAPA SYBR FAST qPCR Master Mix (Kapa Biosystems) are potential alternatives.</div>
<ul>
<li>DynaMag-2 magnet</li>
</ul>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Any other magnetic stand for 1.5-ml tubes can also be used.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Isolation and <i>in vitro</i> culture of neocortical (E11-12) NPCs</h3>
<h4 id="sec3.1.1">Cross-linking of the chromatin—Days 1–10</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 9 days</div>
<p>Neocortical NPCs can be propagated in suspension culture with FGF2 and EGF, during which they form cell aggregates known as neurospheres. The cells cultured for 0 or 3 days in vitro (DIV) correspond to the neurogenic phase of development, whereas those cultured for 9 DIV correspond to the astrogliogenic phase (<a href="#bib10">Hirabayashi and Gotoh, 2005</a>; <a href="#bib17">Qian et al., 2000</a>).</p>
<h4 id="sec3.1.2">Day 1</h4>
<ol>
<li>Kill pregnant Institute of Cancer Research (ICR) mice by cervical dislocation at embryonic day (E) 11 to 12, with E1 being defined as 12 h after detection of the vaginal plug.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Approximately 5 × 10<sup>5</sup> NPCs can be obtained from a single mouse embryo at E11-12.</div>
<div class="note">
<span class="note-title">Note:</span> C57BL/6 mice also can serve as a source of NPCs for culture.</div>
<div class="note">
<span class="note-title">Note:</span> The procedure for sacrifice of mice should be in accordance with the institutional animal care and use committee.</div>
<ol start="2">
<li>Remove the uterus and immerse it in ice-cold DMEM-F12 containing 1% penicillin-streptomycin in a 100-mm plastic culture dish.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After isolation of embryos from the uterus, all manipulations are performed in ice-cold DMEM-F12 containing 1% penicillin-streptomycin unless indicated otherwise.</div>
<ol start="3">
<li>Remove embryos one by one from the embryonic sacs and transfer then to another 100-mm dish filled with ice-cold medium.</li>
<li>Hold down the head of each embryo with forceps and cut off the telencephalon with a scalpel.</li>
<li>Pull and tease apart the skin from the telencephalon and separate the telencephalon from the brainstem.</li>
<li>Remove the vessel membrane (meninges) from the telencephalon.</li>
<li>Isolate the neocortex from the ventral telencephalon.
<ol type="a">
<li>Collect the dissected neocortices into a 1.7-mL prelubricated microcentrifuge tube containing 500 μL of DMEM-F12 containing 1% penicillin-streptomycin.</li>
<li>Maintain the tube on ice until the neocortices settle to the bottom of the tube and then remove the medium.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Digest the neocortical tissue with Neuron Dissociation Solutions (FUJIFILM Wako Pure Chemical Corporation) consisting of Enzyme solution, Dispersion solution and Isolation solution.
<ol type="a">
<li>Add 300 μL of Enzyme solution to the tissue and incubate for 20 min at 37°C.</li>
<li>Centrifuge the digest at 160 × <i>g</i> for 2 min at room temperature and remove the resulting supernatant.</li>
<li>Add 300 μL of Dispersion solution to the pellet and triturate gently with the use of a 1-mL micropipette.
<div class="critical">
<span class="critical-title">Critical:</span> Take care during trituration not to damage the cell suspension by pipetting too vigorously.</div>
</li>
<li>Add 300 μL of Isolation solution as a layer below the cell suspension.
<div class="note">
<span class="note-title">Note:</span> The specific gravity of Isolation solution is greater than that of Dispersion solution. Therefore, by gently adding Isolation solution to the bottom of the tube, these solutions become separated into two layers— Dispersion solution in the top layer and Isolation solution in the bottom layer. Be careful not to disturb the interface between the layered solutions. If the solutions become mixed, it will not be possible to remove the papain.</div>
</li>
<li>Centrifuge the tube at 160 × <i>g</i> for 5 min at room temperature and then remove the supernatant.</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Prepare the DMEM-F12 supplemented with B-27 (final B-27 concentration of 2%), recombinant human FGF2 (final FGF2 concentration of 20 ng/mL), and recombinant human EGF (final EGF concentration of 20 ng/mL).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prewarm the DMEM-F12 (without any supplements) to 37°C in a water bath for at least 20 min before starting dissociation of neocortices.</div>
<ol start="10">
<li>Gently suspend the cell pellet in 4 mL of DMEM-F12 supplemented with B-27, FGF2, and EGF.</li>
<li>Count the number of isolated cells with the use of a cell counting chamber (Neubauer haemocytometry, Electron Microscopy Sciences)</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When counting the number of cells, it is recommended to mix 10 μl of cell suspension with an equal volume of Trypan blue solution and evaluate the cell viability under a microscope. The percentage of cells stained with Trypan blue should be less than 2–3%.</div>
<ol start="12">
<li>Collect 0.5 × 10<sup>6</sup> to 1.0 × 10<sup>6</sup> dissociated cells into a 1.7-mL prelubricated microcentrifuge tube as neurogenic NPCs.</li>
<li>Add 0.1 volume of 11% formaldehyde solution to the cell suspension (final formaldehyde concentration of 1%) and rotate the tube for 10 min on a tube rotator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The formaldehyde solution should be added dropwise to the cell suspension in order to avoid a localized high formaldehyde concentration.</div>
<ol start="14">
<li>Add 2.5 M glycine to the cell suspension to achieve a final concentration of 125 mM.</li>
<li>Centrifuge the mixture at 2,700 × <i>g</i> for 5 min at 4°C.</li>
<li>Remove the resulting supernatant and then freeze the pellet in liquid nitrogen and store it at –80°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The pellet can be stored at –80°C for at least several months.</div>
<ol start="17">
<li>The remaining dissociated cells are cultured at a density of 3.0 × 10<sup>5</sup> cells/mL in DMEM-F12 medium supplemented with B-27, FGF2, and EGF, resulting in neurosphere formation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Culture the dissociated cells in a dish at least 60 mm in diameter (e.g., 100-mm dish, 3.0 × 10<sup>6</sup> cells in 10 mL medium; 60-mm dish, 1.2 × 10<sup>6</sup> cells in 4 mL medium). Use of a smaller culture dish can result in aggregation of neurospheres, with such large aggregates being difficult to dissociate completely.</div>
<div class="note">
<span class="note-title">Note:</span> Do not move the culture dish after its transfer to the incubator. Even a slight movement can also result in the aggregation of neurospheres.</div>
<h4 id="sec3.1.3">Day 4</h4>
<ol start="18">
<li>After culture for 3 DIV, transfer the neurospheres to a 15-mL Falcon tube. Centrifuge the tube at 200 × <i>g</i> for 2 min at room temperature and remove the supernatant.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Remove as much of the supernatant as possible after centrifugation. Residual medium in the tube can inhibit the enzymatic activity of trypsin.</div>
<ol start="19">
<li>Add 100 μL of DNase I solution into 4 mL of trypsin solution (final DNase I concentration of 0.1 mg/mL).</li>
<li>Add 4 mL of trypsin solution containing DNase I to the neurosphere pellet.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Take care not to prewarm trypsin at 37°C (in a water bath) for &gt;10 min. Incubation for longer can result in self-digestion of trypsin and a consequent reduction in dissociation efficiency.</div>
<div class="note">
<span class="note-title">Note:</span> DNase I solution should be added to the trypsin solution immediately before the addition of trypsin to the neurosphere pellet. Otherwise, trypsin digests the DNase I and reduces the efficiency of genomic DNA digestion.</div>
<ol start="21">
<li>Incubate the mixture for 10 min at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Gently mix the suspension every 2–3 min with the use of a 10-mL disposable pipette in order to promote dissociation of the neurospheres.</div>
<ol start="22">
<li>Add 40 μL of ovomucoid solution into 4 mL of aCSF (final ovomucoid concentration of 0.7 mg/mL).</li>
<li>Add 4 mL of aCSF containing ovomucoid to the cell suspension from step 21.</li>
<li>Centrifuge the suspension at 200 × <i>g</i> for 2 min at room temperature and then remove the supernatant.</li>
<li>Resuspend the cell pellet in 5 mL of DMEM-F12 medium supplemented with B-27, FGF2, and EGF and then mechanically dissociate the NPCs with the use of a 10-mL disposable pipette.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Lightly press the tip of the pipette against the bottom of the tube and pipette the cell suspension at least 10 times. This step is critical for complete dissociation of the cells. Be careful not to press the tip of the pipette too hard against the bottom of the tube, however, as this can result in cell damage or death.</div>
<ol start="26">
<li>Determine the cell number with a cell counting chamber and set aside 0.5 × 10<sup>6</sup> to 1.0 × 10<sup>6</sup> of the cells for ChIP analysis of NPCs in the neurogenic phase at 3 DIV.</li>
<li>Culture the remaining dissociated cells at a density of 3.0 × 10<sup>5</sup> cells/mL in DMEM-F12 medium supplemented with B-27, FGF2, and EGF.</li>
<li>Repeat steps 18–27 after successive 3-day intervals to obtain NPCs at 6 or 9 DIV.</li>
</ol>
<h4 id="sec3.1.4">Day 10</h4>
<ol start="29">
<li>Transfer 0.5 × 10<sup>6</sup> to 1.0 × 10<sup>6</sup> dissociated cells after culture for 9 DIV into a 1.7-mL prelubricated microcentrifuge tube for analysis as astrogliogenic NPCs.</li>
<li>Add 0.1 volume of 11% formaldehyde solution to the cell suspension (final formaldehyde concentration of 1%) from steps 26 or 29 and rotate the tube for 10 min on a tube rotator.</li>
<li>Add 2.5 M glycine to the cell suspension to achieve a final concentration of 125 mM.</li>
<li>Centrifuge the mixture at 2,700 × <i>g</i> for 5 min at 4°C.</li>
<li>Remove the supernatant and then freeze the pellet in liquid nitrogen and store it at –80°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The pellet can be stored at –80°C for at least one year.</div>
<h3 id="sec3.2">Preparation of antibody-conjugated magnetic beads—Days 11 and 12</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Antibodies that bind the protein of interest are linked to magnetic beads. The antibody to H2Aub from Merck-Millipore (#05–678) is mouse immunoglobulin M (IgM) and shows low affinity for both protein A and G beads. The antibody to the μ light chain of mouse IgM (Merck-Millipore, #12–488) shows high affinity for the beads, and is therefore used to link the antibody to H2Aub and the protein A beads. The specificity of this H2Aub antibody has been confirmed by the reduction of signals in Ring1A/B double knockout NPCs compared to contorl NPCs in Western blotting and ChIP-qPCR analyses (<a href="#bib21">Tsuboi et al., 2018</a>).</p>
<div class="note">
<span class="note-title">Note:</span> The following H2Aub ChIP protocol is optimized from one previously described (<a href="#bib8">Endoh et al., 2012</a>; <a href="#bib19">Stock et al., 2007</a>).</div>
<h4 id="sec3.2.1">Day 1</h4>
<ol start="34">
<li>Resuspend protein A magnetic beads (Dynabeads) by vortex-mixing for &gt;30 s.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Protein A/G beads (Pierce) can be used as an alternative to protein A beads (Dynabeads).</div>
<ol start="35">
<li>Transfer a volume of beads corresponding to 20 μL (600 μg) of beads per ChIP reaction and 20 μL of beads per sample (for preclearing) to a 1.7-mL prelubricated microcentrifuge tube.</li>
<li>Place the tube on the magnet (DynaMag-2) to separate the beads from the solution and then remove the supernatant by aspiration.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Complete separation of the beads from the solution can be facilitated by swirling the beads gently until the solution becomes clear.</div>
<div class="note">
<span class="note-title">Note:</span> When removing the supernatant by aspiration, attach a 27G injection needle to the tip of the aspirator. This will reduce the suction force and the risk of aspirating the magnetic beads by accident.</div>
<ol start="37">
<li>Wash the beads once with 250 μL of PBS containing 0.5% BSA by vortex-mixing or gentle tapping.</li>
<li>Place the tube on the magnet to separate the beads from the solution and again remove the supernatant.</li>
<li>Add 250 μL of PBS containing 0.5% BSA to the tube containing the beads. Subsequent nonspecific binding of protein to the magnetic beads is minimized by this beads-blocking step.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step is for blocking beads with BSA, and a volume of 250 μl is required for the solution to be stirred by the rotator.</div>
<ol start="40">
<li>Incubate the tube on a tube rotator at approximately 5 rotations per minute for 1 h at 4°C (in a cold room).</li>
<li>Place the tube on the magnet to separate the beads from the solution and remove the supernatant.</li>
<li>Add a volume of PBS containing 0.5% BSA corresponding to that of the beads transferred to the tube in step 35 (for example, if 100 μL of beads were used initially, then add 100 μL of PBS containing 0.5% BSA to the tube).</li>
<li>Transfer beads for ChIP reaction into a new 1.7-mL prelubricated microcentrifuge tube and proceed to step 44. Keep rotating the tube containing beads used for preclearing at 4°C (in a cold room).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is frequently observed that magnetic beads precipitate into the bottom of the tube after the rotation. Gently resuspend the beads-containing solution by pipetting.</div>
<ol start="44">
<li>Add PBS containing 0.5% BSA to the tube containing beads up to 250 μL and then add 2 μg of antibodies to mouse IgMμ per ChIP reaction to the tube containing the blocked magnetic beads.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Given that the optimal amounts of antibodies vary from lot to lot, it is recommended that ChIP be performed in advance with various amounts of antibody by using 9 DIV NPC culture in order to determine the optimal amount of antibody that yields a high signal-to-noise ratio [e.g., signal value for the <i>HoxA1</i> locus (positive control) in NPCs at 9 DIV should be &gt;10 times than that for the <i>Gapdh</i> locus (negative control)].</div>
<ol start="45">
<li>Incubate the tube on a tube rotator for &gt;2 h at 4°C (cold room).</li>
<li>Place the tube on the magnet to separate the beads from the solution and then remove the supernatant.</li>
<li>Wash the anti-IgMμ–conjugated beads five times with 500 μL of Cell lysis buffer.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Washing of the beads at least five times at this stage is important to reduce nonspecific binding of chromatin.</div>
<div class="note">
<span class="note-title">Note:</span> When washing the beads, place the tube on the magnet to separate the beads from the solution and then remove the supernatant, add the washing buffer, and resuspend the beads by gentle rotation of the magnetic stand.</div>
<ol start="48">
<li>Resuspend the anti-IgMμ–conjugated beads in 250 μL of Cell lysis buffer.</li>
<li>Add 4 μg of antibodies to H2Aub per ChIP reaction to the beads.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step is for binding of anti-H2Aub antibody to the anti-IgMμ–conjugated beads, and a volume of 250 μl is required for the solution to be stirred by the rotator.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to determine the optimal amount of anti-H2Aub antibody that yields a high signal-to-noise ratio by using 9 DIV NPC culture, given that the efficiency of immunoprecipitation varies depending on the antibody lot [e.g., signal value for the <i>HoxA1</i> locus (positive control) in NPCs at 9 DIV should be &gt;10 times than that for the <i>Gapdh</i> locus (negative control)].</div>
<ol start="50">
<li>Incubate the tube on a tube rotator overnight (10<b>–</b>16 h) at 4°C (cold room).</li>
</ol>
<h4 id="sec3.2.2">Day 2</h4>
<ol start="51">
<li>Place the tube on the magnet to separate the beads from the solution and then remove the supernatant.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Complete separation of the beads from the solution can be facilitated by swirling the beads gently until the solution becomes clear.</div>
<div class="note">
<span class="note-title">Note:</span> When removing the supernatant by aspiration, attach a 27G injection needle to the tip of the aspirator. This will reduce the suction force and the risk of aspirating the magnetic beads by accident.</div>
<ol start="52">
<li>Wash the beads three times with 500 μL of Cell lysis buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When washing the beads, place the tube on the magnet to separate the beads from the solution and then remove the supernatant, add the washing buffer, and resuspend the beads by gentle rotation of the magnetic stand.</div>
<ol start="53">
<li>Resuspend the antibody–conjugated beads in 20 μL of Cell lysis buffer per ChIP reaction.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Given that the protein A beads tend to form pellets under the influence of gravity, pipette the bead suspension gently and thoroughly before transfer to tubes containing chromatin.</div>
<h3 id="sec3.3">Lysis of the cell pellet and shearing of chromatin—Day 12</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>NPCs are lysed to extract chromatin from the nucleus. Chromatin is then fragmented by ultrasonic treatment, which is necessary to increase the resolution of ChIP.</p>
<ol start="54">
<li>Resuspend the cell pellet with 1 mL of swelling buffer per sample and incubate for 20 min on ice.</li>
<li>Centrifuge the tube at 5,800 × <i>g</i> for 5 min at 4°C.</li>
<li>Remove the resulting supernatant and resuspend the pellet in 150 μL of Cell lysis buffer.</li>
<li>Shear the chromatin with the use of a Hand Sonic UR-20P device (Tomy). The power setting should be “3” for 8 cycles of ON for 10 s and OFF for 20 s and then “4” for 2 cycles of ON for 10 s and OFF for 20 s.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The sonication setting is essential for the appropriate shearing of chromatin. The power of sonication can vary depending on the equipment. After shearing of chromatin, it is recommended that its length be 200–1000 base pairs. It is also recommended to run the samples on an agarose gel electrophoresis and confirm that the size of chromatin is appropriate. If shearing is insufficient, increase the power setting step by step.</div>
<div class="note">
<span class="note-title">Note:</span> While sonicating the cell suspension, place the tube on ice in order to prevent an increase in temperature. Be careful not to cause bubbling of the suspension. If bubbling occurs, centrifuge the tube at 20,380 × <i>g</i> for 5 min at 4°C. This operation will remove bubbling that have been generated.</div>
<ol start="58">
<li>Centrifuge the tube at 20,380 × <i>g</i> for 15 min at 4°C. Cell debris such as insoluble membranes should be removed by this step.</li>
<li>Transfer all supernatant (~150 μL) to a new 1.7-mL prelubricated microcentrifuge tube.</li>
<li>Add 100 μL of Cell lysis buffer and 20 μL of blocked Dynabeads for preclearing from step 43 to the tube containing the sheared chromatin. Incubate the mixture on a tube rotator for 1 h at 4°C (cold room). Subsequent nonspecific binding of protein to the magnetic beads is minimized by this preclearing step.</li>
<li>Place the tube on the magnet to separate the beads from the solution and then transfer 100 μL of the supernatant to a new 1.7-mL prelubricated microcentrifuge tube for ChIP as well as 50 μL to another new tube for determination of input (the amount of chromatin DNA used for the immunoprecipitation). Store the latter tube at –20°C until reversal of cross-linking.</li>
<li>Add 150 μL of Cell lysis buffer to the tube containing the supernatant for ChIP reaction.</li>
</ol>
<h3 id="sec3.4">Immunoprecipitation, washing, elution, and reversal of cross-linking—Days 12–14</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>In the following steps, H2Aub is specifically captured by antibodies conjugated to magnetic beads. Nonspecific binding of nonmodified histone and nonbound DNA are largely eliminated by washing of the beads with various buffers. Immunoprecipitated chromatin is then eluted from the magnetic beads, and histone-DNA cross-linking is reversed to release DNA fragments.</p>
<h4 id="sec3.4.1">Day 1</h4>
<ol start="63">
<li>Add 20 μL of anti-H2Aub–conjugated magnetic beads per ChIP to the chromatin extract from step 62.</li>
<li>Incubate the tube on a tube rotator overnight (10<b>–</b>16 h) at 4°C (cold room).</li>
</ol>
<h4 id="sec3.4.2">Day 2</h4>
<ol start="65">
<li>Place the tube on the magnet to separate the beads from the solution and then remove the supernatant by aspiration.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Complete separation of the beads from the solution can be facilitated by swirling the beads gently until the solution becomes clear.</div>
<div class="note">
<span class="note-title">Note:</span> When removing the supernatant by aspiration, attach a 27G injection needle to the tip of the aspirator. This will reduce the suction force and the risk of aspirating the magnetic beads by accident.</div>
<ol start="66">
<li>Wash the beads six times with 500 μL of Cell lysis buffer.</li>
<li>Wash the beads twice with 500 μL of Wash buffer A.</li>
<li>Wash the beads twice with 500 μL of Wash buffer B.</li>
<li>Wash the beads twice with 500 μL of TE buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After the addition of each wash buffer, separate the beads on the magnetic stand, remove the supernatant, and resuspend the beads by agitation.</div>
<ol start="70">
<li>Resuspend the beads in 200 μL of Elution buffer and proceed to step 71. At the same time, thaw the tube containing the chromatin solution for determination of total input (from step 61), add 150 μL of Elution buffer and proceed to step 72.</li>
<li>Incubate the former tube at 65°C for 15 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For efficient elution of chromatin from the magnetic beads, tap and agitate the tube gently every 2–3 min during the incubation.</div>
<ol start="72">
<li>Add 5 μL of proteinase K solution to a final enzyme concentration of 0.25 mg/mL.</li>
<li>Incubate overnight (10<b>–</b>16 h) at 37°C.</li>
</ol>
<h4 id="sec3.4.3">Day 3</h4>
<ol start="74">
<li>Transfer the tubes to a 65°C incubator for reversal of cross-linking and incubate for at least 6 h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The chromatin can be stored at –20°C for at least several days at this point.</div>
<ol start="75">
<li>Add 200 μL of phenol–chloroform–isoamyl alcohol to each tube and shake vigorously.</li>
<li>Centrifuge at 20,380 × <i>g</i> for 15 min at room temperature.</li>
<li>Collect the resulting aqueous upper phase (~200 μL) and transfer it to a new 1.7-mL prelubricated microcentrifuge tube.</li>
<li>Add 2.5 volumes of ethanol, 0.1 volume of 3 M sodium acetate, and 1 μL of glycogen (final glycogen concentration of 0.1 mg/mL) and apply the tube to a vortex mixer.</li>
<li>Incubate at –20°C for 30 min.</li>
<li>Centrifuge at 20,380 × <i>g</i> for 15 min at room temperature.</li>
<li>Remove the resulting supernatant and wash the DNA pellet with 300 μL of 80% (v/v) ethanol.</li>
<li>Allow the DNA pellet to dry in air and then dissolve it in 50 μL of ddH<sub>2</sub>O.</li>
</ol>
<h3 id="sec3.5">Quantification of binding by qPCR—Day 15</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>The amount of H2Aub at target genes is quantified by qPCR. The relative binding frequency is estimated by calculating the percentage of total input that is recovered in the immunoprecipitate.</p>
<div class="note">
<span class="note-title">Note:</span> Neurogenic gene loci such as <i>Neurog1</i>, <i>Fezf2</i>, <i>Lef1</i>, and <i>Tcfap2c</i> and housekeeping gene loci such as <i>Actb</i> and <i>Gapdh</i> can be used as positive and negative controls, respectively, to verify the fidelity of the H2Aub ChIP assay. The binding ratio of the signals at target gene loci relative to those at negative genes loci can then be calculated. However, if the background is very low, it can be difficult to quantify. If there is a lot of variation in the background signal, the enrichment over background will also have a lot of variation. To confirm that neuronal differentiation potential is indeed restricted in NPC cultures at 9 DIV, the level of H2Aub at neurogenic gene loci should be examined.</div>
<div class="note">
<span class="note-title">Note:</span> This qPCR protocol was designed for a 16 μL reaction mixture in the 96-well format and for the use of THUNDERBIRD® SYBR® qPCR Mix (Toyobo) and a LightCycler 480 System II (Roche). The precise procedure will differ depending on the reagents and machine used, but the general design principles should be the same as described here.</div>
<table id="undtbl23">
<caption>Reaction mixture for a single 16 μL qPCR reaction with THUNDERBIRD® SYBR® qPCR Mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>THUNDERBIRD® SYBR® qPCR Mix</td>
<td>8 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>4.4 μL</td>
</tr>
<tr>
<td>Forward + reverse primer mix (10 μM each)</td>
<td>1.6 μL</td>
</tr>
<tr>
<td>DNA template</td>
<td>2 μL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> When designing primers for qPCR, ensure that the amplicons are not too large (optimal size of ~100 bp).</div>
<ol start="83">
<li>Prepare a stock solution of both forward and reverse primers in ddH<sub>2</sub>O at a final concentration of 10 μM.</li>
<li>Combine 4.4 μL of ddH<sub>2</sub>O, 8 μL of THUNDERBIRD® SYBR® qPCR Mix, and 1.6 μL of primer stock solution per reaction for 1.1 times the desired number of reactions as a premixture. Mix well by pipetting or tapping.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Assay samples at least in duplicate and ensure that there is no substantial difference between the duplicate values.</div>
<div class="note">
<span class="note-title">Note:</span> When performing qPCR, always generate a standard curve for the level of PCR amplification by the use of each primer pair on each plate with the use of purified genomic DNA of the input sample and with the same premixture as used for the immunoprecipitated samples. We only used the range of PCR amplification cycles for each primer pair that shows the level of amplification greater than 1.5 at each cycle.</div>
<ol start="85">
<li>Cool a 96-well plate on a plate cooler and add 14 μL of the premixture followed by 2 μL of the purified DNA to each well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep the plate cold during preparation of the reaction mixture with the use of a plate cooler so as to avoid evaporation.</div>
<div class="note">
<span class="note-title">Note:</span> When adding DNA to the wells, be careful not to lose track of those to which it has been added. Use of a printed plate layout, on which the wells to which the DNA has been added can be checked off, is recommended.</div>
<ol start="86">
<li>Seal the plate with a seal compatible with qPCR.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Before transferring the plate to the qPCR machine, wipe the surface of the seal to remove dirty spots such as hand grease. Given that the qPCR instrument measures fluorescence through the seal, any material on the seal that is in the light path can affect the fluorescence intensity.</div>
<ol start="87">
<li>Vortex the plate briefly and centrifuge at 100 × <i>g</i> for a few seconds at room temperature.</li>
<li>Perform the qPCR reaction with an appropriate program (see examples below).</li>
</ol>
<table id="undtbl24">
<caption>Example of qPCR reaction program for THUNDERBIRD® SYBR® qPCR Mix</caption>
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>5 s</td>
<td rowspan="2">40</td>
</tr>
<tr>
<td>Annealing + Extension</td>
<td>60°C</td>
<td>30 s</td>
</tr>
</tbody>
</table>
<table id="undtbl25">
<caption>Example of melting curve analysis</caption>
<thead>
<tr>
<th>Step</th>
<th>Target temperature</th>
<th>Incubation time</th>
<th>Temperature transition rate (°C/s)</th>
<th>Acquisition mode</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>95°C</td>
<td>15 s</td>
<td>20</td>
<td>NONE</td>
</tr>
<tr>
<td>2</td>
<td>65°C</td>
<td>15 s</td>
<td>20</td>
<td>NONE</td>
</tr>
<tr>
<td>3</td>
<td>95°C</td>
<td>0 s</td>
<td>0.20</td>
<td>CONT</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Given that the precise qPCR program will differ depending on the reagents used, check the protocol recommended by the product manufacturer.</div>
<div class="note">
<span class="note-title">Note:</span> A melting curve analysis at the end of the program is essential to confirm that only a single specific product was formed. If two or more peaks are obtained in the melting curve analysis, the reaction should be performed again or the qPCR primer set should be redesigned.</div>
<ol start="89">
<li>Calculate the relative concentration of the target DNA for all reactions for each primer pair with the use of the corresponding standard curve (Roche software will do this automatically).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend use of the second derivative max method for calculation of the Cp (Ct) value.</div>
<ol start="90">
<li>Export the data to the “.txt” format and calculate the average of relative concentration of DNA for each sample from two or more replicates.</li>
</ol>
<p><i>Q</i>, the average value of relative concentration of each sample; <i>q</i>, the value of relative concentration of input sample; <i>q’</i>, the value of relative concentration of immunoprecipitate with H2Aub antibody, <i>n</i>: the number of qPCR replicates</p>
<p class="math-ml"><math> <mrow> <msub> <mi>Q</mi> <mrow> <mi>t</mi> <mi>o</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> </mrow> </msub> <mo>=</mo> <mfrac> <mrow> <mo>(</mo> <msub> <mi>q</mi> <mn>1</mn> </msub> <mo>+</mo> <msub> <mi>q</mi> <mn>2</mn> </msub> <mo>+</mo> <mo>⋯</mo> <mo>+</mo> <msub> <mi>q</mi> <mi>n</mi> </msub> <mo>)</mo> </mrow> <mi>n</mi> </mfrac> <mo>,</mo> <mspace></mspace> <msub> <mi>Q</mi> <mrow> <mi>H</mi> <mn>2</mn> <mi>A</mi> <mi>u</mi> <mi>b</mi> </mrow> </msub> <mo>=</mo> <mfrac> <mrow> <mo>(</mo> <msub> <mrow> <msup> <mi>q</mi> <mo>′</mo> </msup> </mrow> <mn>1</mn> </msub> <mo>+</mo> <msub> <mrow> <msup> <mi>q</mi> <mo>′</mo> </msup> </mrow> <mn>2</mn> </msub> <mo>+</mo> <mo>⋯</mo> <mo>+</mo> <msub> <mrow> <msup> <mi>q</mi> <mo>′</mo> </msup> </mrow> <mi>n</mi> </msub> <mo>)</mo> </mrow> <mi>n</mi> </mfrac> </mrow> </math></p>
<ol start="91">
<li>Given that the total input sample was diluted, correct the relative concentration value of each reaction by multiplying by the dilution factor.</li>
</ol>
<p><i>I</i>, corrected value of qPCR result of input sample; <i>D</i>, dilution factor (e.g., If the equal volume of ddH2O is added to the input sample, the dilution factor is calculated as “2”.)</p>
<p class="math-ml"><math> <mrow> <msub> <mi>I</mi> <mrow> <mi>t</mi> <mi>o</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> </mrow> </msub> <mo>=</mo> <msub> <mi>Q</mi> <mrow> <mi>t</mi> <mi>o</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> </mrow> </msub> <mo>×</mo> <mi>D</mi> </mrow> </math></p>
<div class="note">
<span class="note-title">Note:</span> If ChIP is performed with several antibodies in addition to anti-H2Aub antibody, the input sample will be used for qPCR reaction many times. In this case, it is recommended the input samples be diluted to 2–5 times with ddH2O.</div>
<ol start="92">
<li>Calculate the percent of input by dividing the value for each immunoprecipitate by that for its corresponding input.</li>
</ol>
<p><i>r</i>, The ratio of the volume of input samples to that of immunoprecipitated samples</p>
<p class="math-ml"><math> <mrow> <mrow> <mo>[</mo> <mo>%</mo> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>i</mi> <mi>n</mi> <mi>p</mi> <mi>u</mi> <mi>t</mi> <mo>]</mo> </mrow> <mo>=</mo> <mfrac> <mrow> <mo>(</mo> <msub> <mi>Q</mi> <mrow> <mi>H</mi> <mn>2</mn> <mi>A</mi> <mi>u</mi> <mi>b</mi> </mrow> </msub> <mo>÷</mo> <mspace></mspace> <mi>r</mi> <mo>)</mo> </mrow> <msub> <mi>I</mi> <mrow> <mi>t</mi> <mi>o</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> </mrow> </msub> </mfrac> <mo>×</mo> <mn>100</mn> </mrow> </math></p>
<div class="note">
<span class="note-title">Note:</span> <i>Q</i><sub><i>H2Aub</i></sub> must be divided by “<i>r</i>” when <i>r</i>-fold amount of the solution to input sample is used for immunoprecipitation of H2Aub antibody at step 61.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">Progressive neuronal fate restriction of neocortical NPC cultures</h3>
<p>Neocortical cells isolated from mouse embryos at E11.5 and cultured in suspension with FGF2 and EGF for 3 or 9 DIV form spherical aggregates known as neurospheres (<a href="#fig1">Figure 1</a>). Under these culture conditions, NPCs undergo a progressive neuronal fate restriction. Reverse transcription (RT) and qPCR analysis (<a href="#bib13">Kishi and Gotoh, 2021</a>) should show that the expression of neurogenic genes <i>Neurog1</i>, <i>Fezf2</i>, <i>Lef1</i> and <i>Tcfap2c</i> is down-regulated, whereas that of the astrogliogenic gene for glial fibrillary acidic protein (<i>Gfap</i>) is up-regulated, at 9 DIV compared with 0 or 3 DIV (<a href="#bib21">Tsuboi et al., 2018</a>) (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/951-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Mouse neocortical NPCs cultured as neurospheres for ChIP-qPCR analysis</div>
<p>NPCs isolated from the mouse neocortex at E11.5 were cultured in the presence of FGF2 and EGF for 3 or 9 DIV and then imaged by phase-contrast microscopy. Scale bars, 50 μm.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/951-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Confirmation of the neurogenic to astrogliogenic fate switch in mouse neocortical NPCs by RT-qPCR analysis</div>
<p>NPCs isolated from the mouse neocortex at E11.5 were cultured with FGF2 and EGF for 0, 3, or 9 DIV, after which the amounts of <i>Neurog1</i> (A), <i>Fezf2</i> (B), <i>Lef1</i> (C), <i>Tcfap2c</i> (D), and <i>Gfap</i> (E) mRNAs were measured by RT-qPCR analysis (<a href="#bib21">Tsuboi et al., 2018</a>). Data are means ± SEM from three independent experiments. ∗p &lt; 0.05, ∗∗p &lt; 0.01, ∗∗∗p &lt; 0.001 (Tukey-Kramer's test).</p>
</figcaption>
</figure>
<h3 id="sec4.2">Specific enrichment of H2Aub at PRC1 target genes</h3>
<p>A successful H2Aub ChIP assay should reveal clear enrichment of the modified histone at PRC1 target gene loci relative to non–PRC1 target gene loci and intergenic regions. Obtained signal values (% of input) for H2Aub vary depending on immunoprecipitation efficiency (which is affected by culture conditions, antibody titer, fixation conditions, and the extent of DNA shearing, among other factors). Comparison of the signal values for PRC1 target gene loci with those for non–PRC1 target gene loci provides an indication of whether H2Aub ChIP was successful or not (for example, signal values for the <i>Fezf2</i> locus in NPCs at 9 DIV should be ~10 times those for the <i>Gapdh</i> locus). An example of H2Aub levels at four PRC1 targets (<i>Fezf2</i>, <i>Lef1</i>, <i>Tcfap2c</i> and <i>Cdkn2d</i>) and a negative control gene (<i>Gapdh</i> and <i>Cdkn1a</i>) in NPCs at 9 DIV is shown in <a href="#fig3">Figure 3</a>.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/951-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Confirmation of the quality of H2Aub ChIP</div>
<p>NPCs isolated from the mouse neocortex at E11.5 were cultured with FGF2 and EGF for 9 DIV, after which ChIP-qPCR analysis of H2Aub was performed with primers specific for <i>Fezf2</i>, <i>Lef1</i>, <i>Tcfap2c</i>, <i>Cdkn2d</i>, <i>Gapdh</i> and <i>Cdkn1a</i> loci. Data are means ± SEM from three independent experiments.</p>
</figcaption>
</figure>
<h3 id="sec4.3">Increase in H2Aub at neurogenic gene loci during development</h3>
<p>Successful NPC culture and H2Aub ChIP should show an increase in the level of H2Aub specifically at neurogenic gene loci during neuronal fate restriction. Such an increase should not be observed at negative control loci (such as <i>Gapdh</i> and <i>Cdkn1a</i>) or other PRC1 target gene loci (such as <i>Cdkn2d</i>, <i>Cdkn1c</i> and <i>Dlk1</i>). An example of developmental changes in the amount of H2Aub at neurogenic gene loci (<i>Fezf2</i>, <i>Lef1</i> and <i>Tcfap2c</i>) as well as at other PRC1 targets (<i>Cdkn2d, Cdkn1c</i> and <i>Dlk1</i>) and non–PRC1 targets (<i>Gapdh</i> and <i>Cdkn1a</i>) gene loci in NPCs is shown in <a href="#fig4">Figure 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/951-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Increase in the level of H2Aub at neurogenic gene loci in astrogliogenic NPCs</div>
<p>NPCs isolated from the mouse neocortex at E11.5 were cultured with FGF2 and EGF for 0, 3, or 9 DIV, after which ChIP-qPCR analysis of H2Aub was performed with primers specific for <i>Fezf2</i>, <i>Lef1</i>, <i>Tcfap2c</i>, <i>Cdkn2d</i>, <i>Cdkn1c</i>, <i>Dlk1</i>, <i>Gapdh</i> and <i>Cdkn1a</i> loci. Data are means ± SEM from three independent experiments. ∗p &lt; 0.05, ∗∗p &lt; 0.01, ∗∗∗p &lt; 0.001 (Tukey-Kramer's test).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol reveals changes in histone modification patterns in cortical NPCs at various developmental time points with a highly favorable signal-to-noise ratio. However, there are several technical limitations to the protocol.</p>
<p>In addition to H2Aub, we tried to apply this protocol to ChIP-qPCR analysis of trimethylated (H3K27me3) or acetylated (H3K27ac) lysine-27 of histone H3 mediated by PRC2 and histone acetyltransferase complexes, respectively (<a href="#bib21">Tsuboi et al., 2018</a>). However, in those cases, the signal-to-noise ratios were not as high as that for H2Aub. Optimization of the protocol may thus be required for ChIP-qPCR analysis of certain other histone modifications.</p>
<p>We optimized the ratio between the number of cells and the amount of antibodies to H2Aub and were able to minimize both in the described protocol. However, use of fewer than 1 × 10<sup>5</sup> cells per immunoprecipitation may result in a loss of signal intensity and a consequent poor signal-to-noise ratio.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Neurospheres are not well dissociated by the mechanical dissociation with a disposable pipette at step 25.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>There are several potential causes of this problem. First, the residual medium in the tube at step 18 may inhibit the enzymatic activity of trypsin. Therefore, remove as much of the supernatant as possible after centrifugation. Second, large aggregates formed by neurospheres are difficult to be dissociated. Increase the number of times of pipetting with disposable pipette during the incubation of trypsin at step 21, or in the mechanical dissociation at step 25.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Neocortical NPCs do not form neurospheres in suspension culture at step 18 and at step 28.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>There are several potential causes of this problem. First, it may be a result of the dissociation of previously formed neurospheres with trypsin. NPCs are sensitive to mechanical dissociation with a disposable pipette. If the pipette is pushed too forcefully to the bottom of the 15-mL tube, the NPCs may experience membrane damage. Conversely, if the pipette is not pushed with enough force, the cells will not dissociate sufficiently. Appropriate mechanical force should be determined by checking for membrane-damaged cells by staining with trypan blue as well as for remaining small cell aggregates in the cell counting chamber when counting cell number. Second, variability between lots of B-27, FGF2, and EGF can affect the growth or survival rate of NPCs, which should be examined for two or three lots by quantification of cell number at each dissociation step.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>NPCs do not manifest the neurogenic to astrogliogenic fate switch during <i>in vitro</i> culture at steps 29 and 92.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Several growth factor or cytokine signaling pathways—including those triggered by EGF, FGF, leukemia inhibitory factor (LIF), and bone morphogenetic protein (BMP)—as well as cell-cell signal transduction (such as that mediated by Notch-Delta) have been implicated in the fate switch of neocortical NPCs (<a href="#bib11">Hirabayashi and Gotoh, 2010</a>; <a href="#bib15">Miller and Gauthier, 2007</a>). Dissociation of NPCs into single cells and expansion of neurospheres from single-cell clones are thought to be important for the proper fate switch of NPC cultures (<a href="#bib5">Coles-Takabe et al., 2008</a>; <a href="#bib12">Jessberger et al., 2007</a>; <a href="#bib18">Singec et al., 2006</a>). It should therefore be examined whether the neocortical tissue and neurospheres are dissociated into single cells after each dissociation step (at steps 8c and 25).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>No obvious cell pellet of NPCs is observed after centrifugation at steps 15 and 32.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Even if prelubricated microcentrifuge tubes are used, it occasionally occurs that dissociated NPCs stick to the wall of tubes, resulting in no obvious cell pellet at the bottom of the tube. In this case, coat the inner surface of prelubricated microcentrifuge tubes with PBS containing 0.5% BSA before centrifugation to ameliorate this problem.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Significant amplification is observed in the negative control sample (only ddH2O is added) in the qPCR reaction at step 92.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Such amplification can be caused by the contamination with mouse genomic DNA. Therefore, it is recommended to renew the water used for the preparation of qPCR reaction mixtures. Moreover, if possible, it is recommended to use dedicated micropipettes, reagents, and a bench for performing qPCR (and related experiments). If this is difficult, clean the bench and equipments thoroughly before preparing the reagents.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>The ratio of the signal intensity for H2Aub at neurogenic gene loci to that at non–PRC1 targets is not sufficiently high at step 92.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>The degree of chromatin shearing is likely not appropriate. If chromatin fragments are oversheared and too small (&lt;100 bp), they may not be detected by qPCR primer pairs that flank targeted genomic regions of &gt;100 bp. If background enrichment at negative control regions is high, it is possible that chromatin shearing is insufficient and chromatin fragments are too large. The extent of chromatin shearing should be checked by agarose gel electrophoresis or with the use of a bioanalyzer. It is also possible that the ratio between the number of cells and the antibody amount is not appropriate. If the amount of H2Aub relative to that of anti-H2Aub is too high, the antibody amount will be rate limiting and the signal-to-noise ratio will be adversely affected. Optimization of the ratio of cell number to antibody amount is thus required to obtain a high signal-to-noise ratio for ChIP-qPCR.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Albert, M., Kalebic, N., Florio, M., Lakshmanaperumal, N., Haffner, C., Brandl, H., Henry, I., and Huttner, W. (2017). Epigenome profiling and editing of neocortical progenitor cells during development. EMBO J. <i>36</i>, 2642-2658. <a class="external-link" href="https://doi.org/10.15252/embj.201796764">https://doi.org/10.15252/embj.201796764</a></p>
<p id="bib2">Blackledge, N.P., Fursova, N.A., Kelley, J.R., Huseyin, M.K., Feldmann, A., and Klose, R.J. (2020). PRC1 catalytic activity is central to polycomb system function. Mol. Cell <i>77</i>, 857-874.e9. <a class="external-link" href="https://doi.org/10.1016/j.molcel.2019.12.001">https://doi.org/10.1016/j.molcel.2019.12.001</a></p>
<p id="bib3">Bonev, B., Cohen, N., Szabo, Q., Fritsch, L., Papadopoulos, G., Lubling, Y., Xu, X., Lv, X., Hugnot, J., Tanay, A., and Cavalli, G. (2017). Multiscale 3D genome rewiring during mouse neural development. Cell <i>171</i>, 557. <a class="external-link" href="https://doi.org/10.1016/j.cell.2017.09.043">https://doi.org/10.1016/j.cell.2017.09.043</a></p>
<p id="bib4">Cao, R., Wang, L., Wang, H., Xia, L., Erdjument-Bromage, H., Tempst, P., Jones, R.S., and Zhang, Y. (2002). Role of histone H3 lysine 27 methylation in Polycomb-group silencing. Science <i>298</i>, 1039-1043. <a class="external-link" href="https://doi.org/10.1126/science.1076997">https://doi.org/10.1126/science.1076997</a></p>
<p id="bib5">Coles-Takabe, B.L., Brain, I., Purpura, K.A., Karpowicz, P., Zandstra, P.W., Morshead, C.M., and van der Kooy, D. (2008). Don't look: growing clonal versus nonclonal neural stem cell colonies. Stem Cells <i>26</i>, 2938-2944. <a class="external-link" href="https://doi.org/10.1634/stemcells.2008-0558">https://doi.org/10.1634/stemcells.2008-0558</a></p>
<p id="bib6">de Napoles, M., Mermoud, J.E., Wakao, R., Tang, Y.A., Endoh, M., Appanah, R., Nesterova, T.B., Silva, J., Otte, A.P., Vidal, M., et al. (2004). Polycomb group proteins Ring1A/B link ubiquitylation of histone H2A to heritable gene silencing and X inactivation. Dev. Cell <i>7</i>, 663-676. <a class="external-link" href="https://doi.org/10.1016/j.devcel.2004.10.005">https://doi.org/10.1016/j.devcel.2004.10.005</a></p>
<p id="bib7">Elderkin, S., Maertens, G.N., Endoh, M., Mallery, D.L., Morrice, N., Koseki, H., Peters, G., Brockdorff, N., and Hiom, K. (2007). A phosphorylated form of Mel-18 targets the Ring1B histone H2A ubiquitin ligase to chromatin. Mol. Cell <i>28</i>, 107-120. <a class="external-link" href="https://doi.org/10.1016/j.molcel.2007.08.009">https://doi.org/10.1016/j.molcel.2007.08.009</a></p>
<p id="bib8">Endoh, M., Endo, T.A., Endoh, T., Isono, K., Sharif, J., Ohara, O., Toyoda, T., Ito, T., Eskeland, R., Bickmore, W.A., et al. (2012). Histone H2A mono-ubiquitination is a crucial step to mediate PRC1-dependent repression of developmental genes to maintain ES cell identity. PLoS Genet. <i>8</i>, e1002774. <a class="external-link" href="https://doi.org/10.1371/journal.pgen.1002774">https://doi.org/10.1371/journal.pgen.1002774</a></p>
<p id="bib9">Entrevan, M., Schuettengruber, B., and Cavalli, G. (2016). Regulation of genome architecture and function by polycomb proteins. Trends Cell Biol. <i>26</i>, 511-525. <a class="external-link" href="https://doi.org/10.1016/j.tcb.2016.04.009">https://doi.org/10.1016/j.tcb.2016.04.009</a></p>
<p id="bib10">Hirabayashi, Y. and Gotoh, Y. (2005). Stage-dependent fate determination of neural precursor cells in mouse forebrain. Neurosci. Res. <i>51</i>, 331-336. <a class="external-link" href="https://doi.org/10.1016/j.neures.2005.01.004">https://doi.org/10.1016/j.neures.2005.01.004</a></p>
<p id="bib11">Hirabayashi, Y. and Gotoh, Y. (2010). Epigenetic control of neural precursor cell fate during development. Nat. Rev. Neurosci. <i>11</i>, 377-388. <a class="external-link" href="https://doi.org/10.1038/nrn2810">https://doi.org/10.1038/nrn2810</a></p>
<p id="bib12">Jessberger, S., Clemenson, G.D., and Gage, F.H. (2007). Spontaneous fusion and nonclonal growth of adult neural stem cells. Stem Cells <i>25</i>, 871-874. <a class="external-link" href="https://doi.org/10.1634/stemcells.2006-0620">https://doi.org/10.1634/stemcells.2006-0620</a></p>
<p id="bib13">Kishi, Y. and Gotoh, Y. (2021). Isolation of genetically manipulated neural progenitors and immature neurons from embryonic mouse neocortex by FACS. STAR Protoc. <i>2</i>, 100540. <a class="external-link" href="https://doi.org/10.1016/j.xpro.2021.100540">https://doi.org/10.1016/j.xpro.2021.100540</a></p>
<p id="bib14">McGinty, R.K., Henrici, R.C., and Tan, S. (2014). Crystal structure of the PRC1 ubiquitylation module bound to the nucleosome. Nature <i>514</i>, 591-596. <a class="external-link" href="https://doi.org/10.1038/nature13890">https://doi.org/10.1038/nature13890</a></p>
<p id="bib15">Miller, F.D. and Gauthier, A.S. (2007). Timing is everything: making neurons versus glia in the developing cortex. Neuron <i>54</i>, 357-369. <a class="external-link" href="https://doi.org/10.1016/j.neuron.2007.04.019">https://doi.org/10.1016/j.neuron.2007.04.019</a></p>
<p id="bib16">Min, J., Zhang, Y., and Xu, R.M. (2003). Structural basis for specific binding of Polycomb chromodomain to histone H3 methylated at Lys 27. Genes Dev. <i>17</i>, 1823-1828. <a class="external-link" href="https://doi.org/10.1101/gad.269603">https://doi.org/10.1101/gad.269603</a></p>
<p id="bib17">Qian, X., Shen, Q., Goderie, S.K., He, W., Capela, A., Davis, A.A., and Temple, S. (2000). Timing of CNS cell generation: a programmed sequence of neuron and glial cell production from isolated murine cortical stem cells. Neuron <i>28</i>, 69-80. <a class="external-link" href="https://doi.org/10.1016/s0896-6273(00)00086-6">https://doi.org/10.1016/s0896-6273(00)00086-6</a></p>
<p id="bib18">Singec, I., Knoth, R., Meyer, R.P., Maciaczyk, J., Volk, B., Nikkhah, G., Frotscher, M., and Snyder, E.Y. (2006). Defining the actual sensitivity and specificity of the neurosphere assay in stem cell biology. Nat Methods <i>3</i>, 801-806. <a class="external-link" href="https://doi.org/10.1038/nmeth926">https://doi.org/10.1038/nmeth926</a></p>
<p id="bib19">Stock, J.K., Giadrossi, S., Casanova, M., Brookes, E., Vidal, M., Koseki, H., Brockdorff, N., Fisher, A.G., and Pombo, A. (2007). Ring1-mediated ubiquitination of H2A restrains poised RNA polymerase II at bivalent genes in mouse ES cells. Nat. Cell Biol. <i>9</i>, 1428-1435. <a class="external-link" href="https://doi.org/10.1038/ncb1663">https://doi.org/10.1038/ncb1663</a></p>
<p id="bib20">Tamburri, S., Lavarone, E., Fernández-Pérez, D., Conway, E., Zanotti, M., Manganaro, D., and Pasini, D. (2020). Histone H2AK119 mono-ubiquitination is essential for polycomb-mediated transcriptional repression. Mol. Cell <i>77</i>, 840-856.e5. <a class="external-link" href="https://doi.org/10.1016/j.molcel.2019.11.021">https://doi.org/10.1016/j.molcel.2019.11.021</a></p>
<p id="bib21">Tsuboi, M., Kishi, Y., Yokozeki, W., Koseki, H., Hirabayashi, Y., and Gotoh, Y. (2018). Ubiquitination-independent repression of PRC1 targets during neuronal fate restriction in the developing mouse neocortex. Dev. Cell <i>47</i>, 758. <a class="external-link" href="https://doi.org/10.1016/j.devcel.2018.11.018">https://doi.org/10.1016/j.devcel.2018.11.018</a></p>
<p id="bib22">Wang, H., Wang, L., Erdjument-Bromage, H., Vidal, M., Tempst, P., Jones, R.S., and Zhang, Y. (2004a). Role of histone H2A ubiquitination in Polycomb silencing. Nature <i>431</i>, 873-878. <a class="external-link" href="https://doi.org/10.1038/nature02985">https://doi.org/10.1038/nature02985</a></p>
<p id="bib23">Wang, L., Brown, J.L., Cao, R., Zhang, Y., Kassis, J.A., and Jones, R.S. (2004b). Hierarchical recruitment of polycomb group silencing complexes. Mol. Cell <i>14</i>, 637-646. <a class="external-link" href="https://doi.org/10.1016/j.molcel.2004.05.009">https://doi.org/10.1016/j.molcel.2004.05.009</a></p>
<p id="bib24">Yu, J.R., Lee, C.H., Oksuz, O., Stafford, J.M., and Reinberg, D. (2019). PRC2 is high maintenance. Genes Dev. <i>33</i>, 903-935. <a class="external-link" href="https://doi.org/10.1101/gad.325050.119">https://doi.org/10.1101/gad.325050.119</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Masafumi Tsuboi (<a href="mailto:tsuboi@chembio.t.u-tokyo.ac.jp">tsuboi@chembio.t.u-tokyo.ac.jp</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate any new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique data sets or code.</p>
<h3>Acknowledgments</h3>
<p>We thank H. Sugishita for critical reading of the manuscript and members of the Gotoh Laboratory for discussion. This research was supported by <a href="https://doi.org/10.13039/100009619">AMED</a>-CREST (JP19gm0610013 and JP21gm1310004 to Y.G.) and <a href="https://doi.org/10.13039/501100001700">MEXT</a>/JSPS KAKENHI (JP11J10035 to M.T. and JP15H05773, JP16H06481, JP16H06479, and JP16H06279 to Y.G.) as well as the International Research Center for Neurointelligence (WPI-IRCN) at The University of Tokyo Institutes for Advanced Study.</p>
<h3>Author contributions</h3>
<p>Conceptualization, M.T. and Y.G.; investigation, M.T.; writing—original draft, M.T.; writing—review and editing, Y.G.; funding acquisition, M.T. and Y.G.; supervision, Y.G.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>