<article>
<section>
<h1 id="header">Medium throughput protocol for genome-based quantification of intracellular mycobacterial loads and macrophage survival during <i>in vitro</i> infection</h1>
<p><time datetime="2022-03-18">Published: March 18, 2022</time></p>
<p>Shivraj M. Yabaji,<sup><a href="#aff1">1</a>,<a href="#fn2">4</a></sup> Sujoy Chatterjee,<sup><a href="#aff1">1</a></sup> Emily Waligursky,<sup><a href="#aff1">1</a></sup> Alexander Gimelbrant,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a></sup> and Igor Kramnik<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>National Emerging Infectious Diseases Laboratories (NEIDL), Pulmonary Center, Department of Medicine, Boston University School of Medicine, Boston University, Boston, MA 02118, USA</p>
<p id="aff2"><sup>2</sup>Department of Cancer Biology and Center for Cancer Systems Biology, Dana-Farber Cancer Institute, Boston, MA 02115, USA</p>
<p id="fn1"><sup>3</sup>Present address: Altius Institute for Biomedical Sciences, 2211 Elliott Ave, Seattle, WA 98121, USA</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ikramnik@bu.edu">ikramnik@bu.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101241">10.1016/j.xpro.2022.101241</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we present a streamlined protocol for assessing intracellular Mycobacterium tuberculosis (Mtb) loads in macrophages. This protocol describes the simultaneous assessment of macrophage viability using automated microscopy. Further, we detail the quantification of mycobacterial loads using a rapid, inexpensive, and accurate approach for mycobacterial DNA isolation from paraformaldehyde-fixed macrophages. Simultaneous assessment of the bacterial loads using internal standard and macrophage viability allows for precise quantification of the effects of perturbations on Mtb and host cells while accounting for technical artifacts.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref5">Chatterjee et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Rapid protocol for mycobacterial DNA isolation from fixed macrophages in 96-well format</li>
<li>Use of BCG spike and duplex qPCR for accurate quantification of Mtb loads</li>
<li>Simultaneous assessment of macrophage viability using automated microscopy in BSL2</li>
<li>Compatibility with advanced imaging and mycobacterial DNA-based applications</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Rationale for the method development</h3>
<p><i>Mycobacterium tuberculosis</i> (Mtb) is a specialized human pathogen whose existence is entirely dependent on interactions with the immune system of human hosts. Macrophages are an important innate immune cells that confronts Mtb at different stages of its life cycle, but also provides a niche for the pathogen persistence and replication (<a href="#bib7">Cohen et al., 2018</a>; <a href="#bib16">Lerner et al., 2017</a>). Activation of T cell immunity, essential for Mtb control, is also dependent on the Mtb-macrophages interactions. In macrophages Mtb may enter a drug tolerant state and thus avoid the eradication by antibiotics (<a href="#bib17">Liu et al., 2016</a>). Therefore, studying Mtb in context of macrophage infection is key in testing new drug regimens and host-directed therapies.</p>
<p>We have developed a method that allows for a simultaneous determination of Mtb loads, bacterial replication and macrophage viability in a 96 well format. A number of techniques are currently used to assess Mtb survival and growth in macrophages. Several recently developed methods are based on imaging using flow cytometry of Mtb-infected cells in suspension or automated microscopy in monolayers. Microscopy-based quantification methods include modified Ziehl-Neelsen staining, labeling with antibodies, use of reporter strains, FITC-biotin-avidin (FBA) staining or prestaining of bacteria (<a href="#bib1">Agerer et al., 2004</a>; <a href="#bib3">Barczak et al., 2017</a>; <a href="#bib4">Brodin et al., 2010</a>; <a href="#bib6">Chen et al., 2012</a>; <a href="#bib12">Heesemann and Laufs, 1985</a>; <a href="#bib14">Jepras et al., 1997</a>; <a href="#bib20">Queval et al., 2014</a>; <a href="#bib22">Stanley et al., 2014</a>). Keller et al. developed a method for tracking intracellular Mtb vacuole rupture using CCF4-AM (a β-lactamase- sensitive FRET reporter) loaded infected cells (<a href="#bib15">Keller et al., 2013</a>). Brodin et al. used automated confocal imaging to quantify Mtb and its mutants for intracellular trafficking into acidified compartments of infected macrophages using CypHer5 mono ester dye (red)-labeling (<a href="#bib4">Brodin et al., 2010</a>). Flow cytometry combined with monoclonal antibodies, fluorescent dyes, and reporter strains can be used to estimate bacterial load and viability (<a href="#bib2">Ambriz-Avina et al., 2014</a>; <a href="#bib9">Emerson et al., 2017</a>; <a href="#bib10">Frossard et al., 2016</a>; <a href="#bib18">Mejuto et al., 2017</a>; <a href="#bib21">Rubio et al., 2019</a>; <a href="#bib23">Tracy et al., 2010</a>). A cytometric method using live/dead staining of mycobacteria has been adopted for rapid drug susceptibility testing (<a href="#bib13">Hendon-Dunn et al., 2016</a>).</p>
<p>The major challenges for assays based on the quantification of bacterial growth in macrophages are associated with the slow growth of pathogenic mycobacteria (Mtb duplication time is 18–24 h), bacterial clumping, presence of viable but not culturable bacteria, as well as macrophage killing by Mtb - all of which can produce artifacts. In addition, the requirement for BSL3 containment limits access to downstream applications. To address these challenges, we developed a method for simultaneous assessment of macrophage viability and mycobacterial loads using automated microscopy and subsequent duplex qPCR of the extracted Mtb genomic DNA. Importantly, fixation with paraformaldehyde (PFA) inactivates the bacteria and allows for the removal of cultures from BSL3 biocontainment for downstream imaging analyses, which are often unavailable inside BSL3 containment facilities. We applied this method for testing the combinatorial effects of host-directed therapies (HDTs), T cell populations and antibiotics on outcomes of mycobacteria – macrophage interactions <i>in vitro</i>. This method can also be used for other DNA-based assays, such as replication clock or screening of pooled Mtb mutant libraries, in a medium or high throughput format, as well as using cells isolated from infected tissues using flow sorting or other separation methods.</p>
<h3 id="sec1.2">Mice</h3>
<p>C57BL/6 J mice were obtained from the Jackson Laboratory and B6J.C3-<i>Sst1</i><sup><i>C3HeB/Fej</i></sup> Krmn mice were developed in our laboratory (available from MMRRC stock # 043908-UNC). Adult mice of 6–12 weeks old of both sexes were used in the experiments. No sex-dependent variation was observed in the described assay. All experiments were performed with the full knowledge and approval of the Standing Committee on Animals at Boston University (IACUC protocol number PROTO201800218).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT OR RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM/Ham's F-12 50/50 Mix [+] L-glutamine</td>
<td>Corning®</td>
<td>Cat# 45000-344</td>
</tr>
<tr>
<td>DMEM, 1× (Dulbecco’s Modified Eagle’s Medium)</td>
<td>Corning®</td>
<td>Cat# 10-013-CV</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Hyclone<sup>TM</sup>,</td>
<td>Cat# SH30071.03</td>
</tr>
<tr>
<td>Hoechst 33342</td>
<td>Fisher Scientific</td>
<td>Cat# H3570</td>
</tr>
<tr>
<td>Paraformaldehyde Solution 4% in PBS</td>
<td>Fisher Scientific</td>
<td>Cat# J19943-K2</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Corning®</td>
<td>Cat# 25-005-CI</td>
</tr>
<tr>
<td>Penicillin Streptomycin solution</td>
<td>Corning®</td>
<td>Cat# 30-002-CI</td>
</tr>
<tr>
<td>HEPES buffer</td>
<td>Corning®</td>
<td>Cat# 25-060-CI</td>
</tr>
<tr>
<td>L929 Cell Conditioned Media (LCCM)</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Murine IFN-gamma</td>
<td>Fisher Scientific</td>
<td>Cat# 50813665</td>
</tr>
<tr>
<td>Murine Interleukin -3</td>
<td>Fisher Scientific</td>
<td>Cat# 50813291</td>
</tr>
<tr>
<td>Murine Interleukin-4</td>
<td>Fisher Scientific</td>
<td>Cat# 50399557</td>
</tr>
<tr>
<td>Murine TNF-alpha</td>
<td>Fisher Scientific</td>
<td>Cat# 50813656</td>
</tr>
<tr>
<td>Lymphoprep<sup>TM</sup> (1.077A)</td>
<td>STEMCELL Technologies</td>
<td>Cat#07801</td>
</tr>
<tr>
<td>Poly Ethylene Glycol (PEG), Bioultra-8000</td>
<td>Sigma-Aldrich</td>
<td>Cat#89510</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>Invitrogen</td>
<td>Cat#AM9759</td>
</tr>
<tr>
<td>Tris Hydrochloride, 1 M solutions (pH 8.0)</td>
<td>Fisher Scientific</td>
<td>Cat#77-86-1</td>
</tr>
<tr>
<td>Ultrapure 0.5 M EDTA pH 8.0</td>
<td>Invitrogen</td>
<td>Cat#15575-038</td>
</tr>
<tr>
<td>Ambion<sup>TM</sup> Nuclease-free Water</td>
<td>Invitrogen</td>
<td>Cat#AM9932</td>
</tr>
<tr>
<td>SpeedBead Magnetic Carboxylate Modified Particles</td>
<td>GE Healthcare</td>
<td>Cat#65152105050250</td>
</tr>
<tr>
<td>DynaMag<sup>TM</sup>-96 side</td>
<td>Life Technologies<sup>TM</sup>
</td>
<td>Cat#12331D</td>
</tr>
<tr>
<td>Glycine</td>
<td>Sigma-Aldrich</td>
<td>Cat#50046</td>
</tr>
<tr>
<td>NaOH Solution</td>
<td>Sigma-Aldrich</td>
<td>Cat#72068</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Ambion</td>
<td>Cat#AM2546</td>
</tr>
<tr>
<td>Middlebrook 7H9 Broth</td>
<td>BD Biosciences</td>
<td>Cat# 271310</td>
</tr>
<tr>
<td>Middlebrook 7H10 Agar</td>
<td>BD Biosciences</td>
<td>Cat# 262710</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: C57BL/6J</td>
<td>The Jackson Laboratory</td>
<td>Stock No.: 000664</td>
</tr>
<tr>
<td>Mouse: B6J.C3-<i>Sst1</i><sup><i>C3HeB/Fej</i></sup>Krmn</td>
<td><a href="#bib19">Pichugin et al. (2009)</a></td>
<td>Stock No: 043908-UNC <a href="https://www.mmrrc.org">https://www.mmrrc.org</a>
</td>
</tr>
<tr>
<td>
<i>Mycobacterium tuberculosis</i> H37Rv</td>
<td>ATCC</td>
<td>Cat# 27294</td>
</tr>
<tr>
<td>
<i>Mycobacterium bovis</i> BCG</td>
<td>ATCC</td>
<td>Cat# 35737</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Live-or-Dye™ 594/614 Fixable Viability Staining Kits</td>
<td>Biotium</td>
<td>Cat# 32006</td>
</tr>
<tr>
<td>TaqMan™ Environmental Master Mix 2.0</td>
<td>Fisher Scientific</td>
<td>Cat#4396838-5 mL</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Mtb and BCG specific primers and probes</td>
<td>This paper</td>
<td>See <a href="#tbl1">Table 1</a>.</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Corning™ Costar™ Ultra-Low Attachment Microplates</td>
<td>Corning®</td>
<td>Cat# 3471</td>
</tr>
<tr>
<td>Cell strainer, 70 mm</td>
<td>Falcon</td>
<td>Cat#352350</td>
</tr>
<tr>
<td>Thermo Scientific™ ART™ Wide Bore Filtered Pipette Tips</td>
<td>Thermo Scientific</td>
<td>Cat# 21-236-2C</td>
</tr>
<tr>
<td>25 cm<sup>2</sup> Cell Culture Flask</td>
<td>Falcon®</td>
<td>Cat#353109</td>
</tr>
<tr>
<td>75 cm<sup>2</sup> Cell Culture Flask</td>
<td>Falcon®</td>
<td>Cat#353136</td>
</tr>
<tr>
<td>96-well cell culture plate</td>
<td>Corning®</td>
<td>Cat#3599</td>
</tr>
<tr>
<td>30 mL Nalgene™ Sterile Square PETG Bottle with 20/415 Cap</td>
<td>Nalgene</td>
<td>Cat# 79053</td>
</tr>
<tr>
<td>5 μm filter unit</td>
<td>Millex</td>
<td>Cat#SLSV025LS</td>
</tr>
<tr>
<td>10 mL control syringe</td>
<td>BD</td>
<td>Cat#309695</td>
</tr>
<tr>
<td>Plastic Laboratory 500 mL Module Stackable Storage Container Tray</td>
<td>ROSTI MEPAL</td>
<td>N/A</td>
</tr>
<tr>
<td>CFX96TM Real-Time System</td>
<td>Bio-Rad</td>
<td>N/A</td>
</tr>
<tr>
<td>PTC-100TM Programmable Thermal Controller</td>
<td>MJ Research. Inc.</td>
<td>N/A</td>
</tr>
<tr>
<td>Nexcelom Celigo imaging cytometer</td>
<td>Nexcelom Biosciences</td>
<td>N/A</td>
</tr>
</tbody>
</table>
<table id="tbl1">
<caption>Table 1. Mtb and BCG specific primers and probes</caption>
<thead>
<tr>
<th>Sr.No.</th>
<th>Name of primer</th>
<th>Sequence (5′3′)</th>
<th>Primer length</th>
<th>Amplicon size</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>Mtb specific FP</td>
<td>GGAAATGTCACGTCCATTCATTC</td>
<td>23</td>
<td rowspan="2">146</td>
</tr>
<tr>
<td>2</td>
<td>Mtb specific RP</td>
<td>GCGTTGTTCAGCTCGGTA</td>
<td>18</td>
</tr>
<tr>
<td>3</td>
<td>Probe</td>
<td>56-FAM/AGCTTGGTCAGGGACTGCTTCC/36-TAMSp/</td>
<td>22</td>
<td>–</td>
</tr>
<tr>
<td>4</td>
<td>BCG specific FP</td>
<td>GTGGTGGAGCGGATTTGA</td>
<td>18</td>
<td rowspan="2">162</td>
</tr>
<tr>
<td>5</td>
<td>BCG specific RP</td>
<td>CAACCGGACGGTGATCC</td>
<td>17</td>
</tr>
<tr>
<td>6</td>
<td>Probe</td>
<td>/5Cy5/TTCTGGTCG/TAO/ACGATTGGCACATCC/3IAbRQSp/</td>
<td>24</td>
<td>–</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">One step single-plex or multiplex assay</h3>
<p>CFX96<sup>TM</sup> Real-Time System, 96 well (Bio-Rad, Software version Bio-Rad CFX Manager 3.1).</p>
<h3 id="sec2.2">Heating samples</h3>
<p>Programmable heating block for 96 well plates for heating samples (PTC-100TM Programmable Thermal Controller or another model) in (step 52 b vii).</p>
<h3 id="sec2.3">Imager for live/dead/total macrophages</h3>
<p>The Nexcelom Celigo imaging cytometer (Nexcelom Biosciences) for cell counting and the analysis of macrophage viability.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> any automated fluorescent cell imager that supports live/dead analysis, for example, MuviCyte<sup>TM</sup> Imaging System (Perkin Elmer).</div>
<h3 id="sec2.4">Media composition</h3>
<ul>
<li>Media 0</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM 1×</td>
<td>–</td>
<td>490 mL</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>1%</td>
<td>5 mL</td>
</tr>
</tbody>
</table>
<ul>
<li>Media 1</li>
</ul>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM/F12</td>
<td>–</td>
<td>430 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>1 M HEPES buffer</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>200 mM L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare Media 1, add 10 mL of L929 Cell Conditioned Media (LCCM) to every 90 mL above media composition to a final concentration 10%.</p>
<ul>
<li>Media 2</li>
</ul>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM/F12</td>
<td>–</td>
<td>435 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>1 M HEPES buffer</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>200 mM L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare Media 2, add 10 mL of LCCM to every 90 mL above media composition.</p>
<ul>
<li>Media 3</li>
</ul>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM/F12</td>
<td>–</td>
<td>430 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>1 M HEPES buffer</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>200 mM L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare Media 3, add 20 mL of LCCM to every 80 mL above media composition to a final concentration 20%.</p>
<ul>
<li>Media 4</li>
</ul>
<p>DMEM 1× (Corning, cat. #10-013-CV) + 1% FBS.</p>
<div class="note">
<span class="note-title">Note:</span> Filter-sterilize all the media and store at 4°C up to 14 days.</div>
<table id="undtbl5">
<caption>Washing Medium</caption>
<thead>
<tr>
<th>Component</th>
<th>Volume for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>1× PBS</td>
<td>99 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>1 mL (1% final concentration)</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Filter-sterilize the media and store at 4°C up to one month.</div>
<table id="undtbl6">
<caption>PEG Buffer</caption>
<thead>
<tr>
<th>Component</th>
<th>Volume for 150 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>PEG-8000 (Sigma, Cat#89510)</td>
<td>27 gm</td>
</tr>
<tr>
<td>NaCl (Autoclaved) (5.0 M)</td>
<td>30 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl (pH 8.0)</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>0.5 EDTA (pH 8.0)</td>
<td>0.3 mL</td>
</tr>
<tr>
<td>Nuclease free Water</td>
<td>Up to 140 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Add magnetic beads prepared in <b>step 52a (10 mL)</b> to a final volume of 150 mL, cover with aluminum foil and store at 20°C–25°C up to one month.</div>
<table id="undtbl7">
<caption>1× TE Buffer</caption>
<thead>
<tr>
<th>Component</th>
<th>Working concentration</th>
<th>Volume for 50 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl (pH 8.0)</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>0.5 EDTA (pH 8.0)</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Water</td>
<td>–</td>
<td>Up to 50 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave and store at 20°C–25°C up to 6 months.</div>
<table id="undtbl8">
<caption>Live/dead staining solution:</caption>
<thead>
<tr>
<th>Component</th>
<th>Working concentration</th>
<th>Volume for 10 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>100% Live-or-Dye™ 594/614 Fixable Viability dye</td>
<td>0.1%</td>
<td>10 μL</td>
</tr>
<tr>
<td>FBS</td>
<td>1%</td>
<td>100 μL (from 100% stock)</td>
</tr>
<tr>
<td>1× PBS</td>
<td>–</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Always prepare fresh staining solution and use immediately.</div>
<table id="undtbl9">
<caption>Lysis buffer:</caption>
<thead>
<tr>
<th>Component</th>
<th>Working concentration</th>
<th>Volume for 10 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M NaOH</td>
<td>25 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.2 mM</td>
<td>4 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>–</td>
<td>Up to 10 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave the buffer and store at 20°C–25°Cup to 6 months.</div>
<table id="undtbl10">
<caption>Lysis buffer for fixed cells:</caption>
<thead>
<tr>
<th>Component</th>
<th>Working concentration</th>
<th>Volume for 10 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M NaOH</td>
<td>25 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.2 mM</td>
<td>4 μL</td>
</tr>
<tr>
<td>Proteinase K (20 mg/mL)</td>
<td>0.2 mg/mL</td>
<td>100 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>–</td>
<td>Up to 10 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave the buffer and store at 20°C–25°C up to 6 months<i>.</i> Add Proteinase K to autoclaved buffer just before the lysis.</div>
<table id="undtbl11">
<caption>Neutralization buffer:</caption>
<thead>
<tr>
<th>Component</th>
<th>Working concentration</th>
<th>Volume for 10 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>1M Tris-HCl</td>
<td>40 mM</td>
<td>400 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>–</td>
<td>Up to 10 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave the buffer and store at 20°C–25°C up to 6 months.</div>
<p>To prepare the stock solution use Tris-HCl powder, <b>not Tris base;</b> final pH should be ∼ 6, do not need to adjust pH, it will be adjusted by mixing with an equal volume of 25 mM NaOH during DNA isolation.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Workflow overview (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Workflow overview</div>
</figcaption>
</figure>
<h3 id="sec3.1">Preparation of bone marrow derived macrophages</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–12 days</div>
<p>Monolayers of bone marrow derived macrophages are prepared as described in (<a href="#bib19">Pichugin et al., 2009</a>), with modifications described in the following steps.</p>
<p><b>Day 1:</b></p>
<p><i>Procedure:</i></p>
<ol>
<li>Euthanize the mice using an approved procedure and remove tibia and femurs (some muscle tissue may remain attached, as shown in <a href="#fig2">Figure 2</a>A). Store the removed bones in a 50 mL conical tube containing 10 mL Media 0 on ice
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Preparation of bone marrow-derived macrophages</div>
<p>(A and B) (A) Mice tibia and femur with some attached muscle tissue.</p>
<p>(B) Separation of interphase by density gradient centrifugation.</p>
</figcaption>
</figure>
</li>
<li>Transfer the removed bones of each mouse in 10 mL ice-cold Media 0 to mortar and gently crush the bones with a pestle.</li>
<li>Remove the media to a clean tube placed on ice, add 10 mL of fresh ice-cold Media 0 to the mortar and repeat the bone crushing. Combine cell suspensions and store on ice.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Bone marrow can be isolated using traditional method - flushing the bones with medium using a syringe and a 25G needle. The crushing method is simpler and produces higher cell yields. Care should be taken to avoid bacterial contamination.</div>
<ol start="4">
<li>Pour fluid over a cell strainer (Φ70 μm) placed on top of a clean 50 mL conical tube and dilute the greasy tissues by adding Media 0 up to 50 mL.</li>
<li>Spin cells at 200 × g (e.g., in Sorvall RT7, 1,000 rpm) for 10 min at 20°C–25°C, then resuspend cell pellets in 5mL of Media 0 per each mouse.</li>
<li>Gently break clumps by pipetting up and down several times using 1mL wide-bore tips (Thermo Scientific, Cat. #21-236-2C). Avoid air bubbles. Allow remaining clumps to settle by placing the tube in a vertical position for 10 min</li>
<li>Add 5 mL of Lymphoprep<sup>TM</sup> (1.077A) (StemCell, Cat. #07801) to a clean 15 mL centrifuge tube and carefully overlay with 5 mL of the bone marrow suspension using a 5 mL pipette.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For proper density gradient separation: (i) do not transfer the cell clumps from the bottom; (ii) overlay the cells slowly, do not disturb the gradient interface; (iii) subsequent steps are performed at 20°C–25°C.</div>
<ol start="8">
<li>Perform gradient centrifugation at 600 × g for 20 min at 20°C–25°C with brakes off.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Switching the centrifuge brakes off is important for preserving the gradient interface.</div>
<ol start="9">
<li>Gently remove cells from the interface using a 5 mL serological pipette (Fisherbrand, Cat. #13-678-11D). Place the pipette tip at the interface and collect the cells while slowly moving the tip over/at the top of the interface for taking up the cells (<a href="#fig2">Figure 2</a>B).</li>
<li>Transfer the collected cells from each gradient (from one mouse) into a 50 mL conical tube. Add Media 0 to a total volume of 50 mL and centrifuge 200 × g (1,000 rpm) for 10 min at 20°C–25°C.</li>
<li>Slowly remove the supernatant and resuspend cells in 10 mL Media 0 at 20°C–25°C and centrifuge at 200 × g (1,000 rpm) for 10 min at 20°C–25°C.</li>
<li>Resuspend cells in 5 mL Media 1, count and dilute to a final concentration of 1–3×10<sup>6</sup> cells/mL, and add a to a 25 cm<sup>2</sup> polystyrene tissue culture-treated flask.</li>
<li>Incubate cells for 24 h in a humidified incubator at 37°C, 5% CO<sub>2</sub> to remove adherent cells.</li>
</ol>
<p><b>Day 2:</b></p>
<ol start="14">
<li>Transfer all <u>non-adherent</u> cells to a new 75 cm<sup>2</sup> polystyrene tissue culture-treated flask.</li>
<li>Wash the 25 cm<sup>2</sup> flask with 10 mL Media 1 to recover any remaining cell in suspension and transfer to the same 75 cm<sup>2</sup> flask (Approximate final cell concentration is 5–7×10<sup>5</sup> cells/mL).</li>
<li>Incubate cells for two days in a humidified incubator at 37°C with 5% CO<sub>2</sub>.</li>
</ol>
<p><b>Day 4:</b></p>
<ol start="17">
<li>Collect <u>non-adherent</u> cells and centrifuge at 200 × g (1,000 rpm) for 10 min at 20°C–25°C, and resuspend cells in 10 mL Media 3.</li>
<li>Count the cells. Expect 3–5 × 10<sup>6</sup> cells per mouse. Dilute the cells in Media 3 to a final concentration of 2.5×10<sup>5</sup> cells/mL</li>
<li>Plate 1.5×10<sup>6</sup> cells per well in 6 well ultra-low attachment plate (Costar/Corning Cat. #3471).</li>
<li>Incubate cells 2 days in humidified 37°C, 5% CO<sub>2</sub> incubator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this step you may (i) proceed with the macrophage monolayer preparation; (ii) freeze extra cells at 2×10<sup>6</sup> cells per vial in 1 ml FBS with 10% DMSO; (iii) expand the cells by an additional passage on ultra-low attachment plate as in step 17.</div>
<p>Day 6: Preparation of macrophage monolayer</p>
<ol start="21">
<li>Collect all cells (non-adherent and partially attached) enriched after step 20 from 6 well ultra-low attachment plate by pipetting up and down several times avoiding bubbles, and centrifuge at 200 × g for 10 min at 20°C–25°C and resuspend in antibiotic-free media (Media 2). Alternatively use gentle scrapping.</li>
<li>Count and dilute the cells in Media 2 and plate 15×10<sup>3</sup> cells per well (in 200 μL volume) in a 96 well plate.</li>
<li>Incubate cells for 2 days in a humidified incubator at 37°C with 5% CO<sub>2</sub>.</li>
<li>After 2 days replace 90% of old media with the same volume of fresh Media 2 pre-warmed at 37°C.</li>
<li>Incubate cells for additional 2 days in a humidified incubator at 37°C with 5% CO<sub>2</sub>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this step, the cells form monolayer. If required, they can be maintained for additional 2 days after replacing the medium as in step 24. At this stage the cells are ready either for infection only or for treatment with compounds to assess a specific effect, followed by infection.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Macrophage monolayers for Mtb infection are prepared in antibiotic free medium for at least 4 days prior to the infection, including two media changes.</div>
<p>Medium is replaced 1 day before infection. After that, cells can be treated with cytokines or small molecules, i.e., the day before the infection.</p>
<p>The method described in this protocol produces high macrophage yields, homogeneous macrophage populations and uniform monolayers, critical for reproducible results of Mtb infection experiments. Other methods to prepare macrophage monolayers are also suitable.</p>
<h3 id="sec3.2">Preparation of Mtb for infection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<ol start="26">
<li>This step is performed inside the Biosafety level 3 facility (BSL3) using standard operating procedures, protocols and disinfectants approved by the Institutional Biosafety Committee (IBC). All work is performed inside Biosafety Cabinets (BSC).</li>
<li>Dilute 0.5–1.0 mL of a frozen Mtb stock in 5 mL of liquid culture medium (we use Middlebrook 7H9 Broth with OADC), add to a 30 mL square sterile media bottle (Nalgene, Cat. #79053), close the cap tightly to prevent opening and place the bottle(s) and adsorbent paper in a secondary container with a leakage proof lid to avoid aerosols.</li>
<li>Incubate at 37°C for 2–3 days in a shaking incubator at 100 rpm (to OD<sub>600</sub> = 0.4–0.5). To avoid clumping, do not overgrow the bacteria.</li>
<li>After 2–3 days transfer the Mtb suspension from the media bottle to a sterile 15 mL conical tube. Centrifuge for 10 min at 2,100 × g (∼3,000 rpm) at 20°C–25°C.</li>
<li>Resuspend the pellet in 5 mL of Media 2 pre-warmed to 37°C (avoid bubbles or foam).</li>
<li>Sonicate twice for 5 s each using a sonicating water-bath with a 5–10 s interval at 20°C–25°C.</li>
<li>Add 5 mL of pre-warmed Media 2, mix and leave the tube in a vertical position for 30–60 min at 20°C–25°C to let the residual clumps settle down. Use the upper 8 mL for subsequent steps without disturbing the clumps at the bottom.</li>
<li>Remove plunger from a 10mL “control syringe” (BD, Cat. #309695). Attach the syringe barrel to a 5 μm filter (Micron Separation) and place the filter on top of a 15 mL conical tube secured in a rack. Transfer the top 8 mL of the Mtb suspension, insert the plunger and press to filter the bacterial suspension.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Hold the syringe, filter, and the reception tube with one hand and rapidly insert the plunger and gently press with the other hand. Slow filtration at gravity force will result in a loss of Mtb.</div>
<ol start="34">
<li>Measure OD of the liquid Mtb bacterial culture at A<sub>600</sub> using the dilution medium as a blank control. Expected OD<sub>600</sub> after filtration is between 0.1 and 0.2. Calculate the bacterial concentration assuming that after filtration single-cell Mtb suspension at OD<sub>600</sub> = 1 is approx. 3×10<sup>8</sup> bacteria per mL.</li>
<li>Dilute the Mtb suspension with Media 2 to the required concentration for macrophage infection. For example, to infect macrophages in 96 well format at multiplicity of infection (MOI) 1:1, prepare 0.5×10<sup>6</sup> bacteria per mL and add 0.1 mL per well. At OD<sub>600</sub> = 0.1 after filtration, the bacterial suspension is estimated at 3×10<sup>7</sup> bacteria per mL and should be diluted 60-fold. Plate an aliquot of Mtb on solid media and count colonies 3 weeks later to determine the precise infectious dose retrospectively.</li>
</ol>
<h3 id="sec3.3">Preparation of BCG spike for normalization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="36">
<li>Avirulent vaccine strain of <i>M. bovis</i> BCG is prepared in the BSL2 lab using the same procedure as described for Mtb in steps 26–35.</li>
<li>After filtration, dilute BCG 10-fold in PBS (to approx. 3–5×10<sup>6</sup> cells per mL) to avoid clumping and store at 4°C up to 21 days.</li>
</ol>
<h3 id="sec3.4">Infection of macrophages and maintenance of infected macrophage cultures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0–10 days</div>
<ol start="38">
<li>Transfer the 96 well plates containing macrophage cultures into the BSL3 facility.</li>
<li>Remove 50% volume (100 μL for 96 well plate) of media prior to infection.</li>
<li>Prepare a bacterial suspension for each multiplicity of infection (MOI) to be used by diluting the Mtb suspensions (prepared in <b>step 35</b>) in Media 2. Place 10 mL of the diluted Mtb in a 100 mm petri dish.</li>
<li>To infect macrophage monolayers, add 100 μL of the bacterial suspension to each well of the 96 well plate using a multichannel pipette.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Save filtered bacterial suspensions in a 2.0 mL screw-cap tube for plating on solid medium (Middlebrook 7H10 Agar) to determine inoculum by colony forming units (CFU) enumeration.</div>
<ol start="42">
<li>To avoid spills or danger of leakage, use centrifuge platforms with biocontainment lids and centrifuge the plates at 200 × g for 5 min at 22°C–25°C to facilitate infection and incubate at 37°C, 5% CO<sub>2</sub> and 95% humidity for 2 h.</li>
<li>To eliminate extracellular bacteria, add amikacin (to a final concentration of 200 μg/mL) and incubate at 37°C, 5% CO<sub>2</sub> and 95% humidity for 30–45 min.</li>
<li>Wash the cells 3 times with Media 4 at 20°C–25°C. To avoid drying of monolayers, replace 90% of the media and cover the plate with a lid after removing the media.</li>
<li>Incubate cells in a final volume of 200 μL Media 2 per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If cells need to be treated with any drug, dilute the compound at 2× concentration in Media 2 and then add 100 μL to each well containing macrophage monolayers in 100 μL of media.</div>
<ol start="46">
<li>Place the plates inside secondary containers (see <a href="#key-resources-table">key resources table</a>) with covers slightly opened to allow for gas exchange and incubate at 37°C, 5% CO<sub>2</sub> and 95% humidity, changing medium every two days.</li>
</ol>
<h3 id="sec3.5">Washing of infected monolayers in BSL3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="47">
<li>Carefully remove 90% of supernatant (180 μL for 96 well plate) from the infected plate and dispose into liquid waste container containing an undiluted approved disinfectant.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure the effective concentration of the disinfectant in final volume after all washing steps. For example, use 5 mL of Vesphene II (100× concentrate) per 500 mL container.</div>
<ol start="48">
<li>Immediately add Media 2 pre-warmed to 37°C (200 μL for 96 well plate).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Infected cells can be maintained by replacing 90% of the old media with fresh antibiotic-free media (Media 2) with or without experimental treatment every 48 h.</div>
<p>It is recommended to work with 1–3 columns/rows at a time, covering the plate with a lid between steps to prevent the monolayer drying.</p>
<h3 id="sec3.6">Staining and fixation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h (step 49)</div>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h (step 50)</div>
<ol start="49">
<li>Staining with Live/Dead Stain
<ol type="a">
<li>Carefully remove media from infected plates and dispose into liquid waste containers.</li>
<li>Wash cells once with washing media.</li>
<li>Add 100 μL of staining solution.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Any fixable live/dead dye can be used to analyze the viability of cells.</div>
</li>
<li>Incubate cells at 37°C, 5% CO<sub>2</sub> and 95% humidity for 30 min in the dark.</li>
<li>Remove 90% (90 μL) staining solution and wash the cells 2 times with 200 μL washing media.</li>
</ol>
</li>
</ol>
<ol start="50">
<li>Fixation with 4% PFA
<ol type="a">
<li>Carefully remove washing media and add 100 μL 4% paraformaldehyde (PFA).</li>
<li>Incubate cells at 22°C–25°C for 30 min in dark.
<div class="note">
<span class="note-title">Note:</span> Fixation time is extended to kill pathogenic Mycobacterium tuberculosis.</div>
</li>
<li>Wash cells twice with washing media and cover the macrophage monolayer with washing media by adding a final volume of 200 μL per well.</li>
<li>Seal the plate with parafilm, wipe with an approved disinfectant, place in a sealed container and safely remove it from BSL3 following standard operating procedures.</li>
</ol>
</li>
</ol>
<h3 id="sec3.7">Outside BSL3 containment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h (step 51)</div>
<div class="timing">
<span class="timing-title">Timing:</span> 6–7 h (step 52); 1 h (step 52a); 3 h (step 52b); 2 h (step 52c)</div>
<ol start="51">
<li>Imaging for analysis of cell numbers and dead cells
<ol type="a">
<li>Remove 90% of washing media from the infected plate (approx. 180 μL) and add 100 μL/well Hoechst 33342 nuclear dye (final concentration 50 μg/mL) suspended in 1× PBS.</li>
<li>Incubate the plate at 20°C–25°C for 15 min in the dark.</li>
<li>Remove the Hoechst dye and add 200 μL/well of 1× PBS. Cover the plate with aluminum foil.</li>
<li>Image and analyze the total and dead cell numbers using the Nexcelom Celigo imaging cytometer (select Live-or-Dye™ 594/614 Fixable Viability dye fluorescence for dead cells and Hoechst stain for total cells).
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other automated fluorescence microscopy systems can be used.</div>
</li>
</ol>
</li>
</ol>
<ol start="52">
<li>Cell lysis and DNA isolation using magnetic beads
<ol type="a">
<li>Preparation of Solid Phase Reversible Immobilization (SPRI) magnetic beads
<div class="note">
<span class="note-title">Note:</span> The SPRI beads are prepared according to (<a href="#bib8">DeAngelis et al., 1995</a>).</div>
<ol type="i">
<li>Resuspend stock solution of Sera-Mag SpeedBeads by vortexing.
<div class="note">
<span class="note-title">Note:</span> Vortex magnetic beads until the solution becomes uniformly distributed (vortex 4–5 times for 5 s each).</div>
</li>
<li>Aliquot 3 mL beads into a 50 mL conical tube.</li>
<li>Add 20 mL 1× TE buffer and pellet beads in a magnetic rack for 20–30 min at 20°C–25°C (<a href="#fig3">Figure 3</a>A).</li>
<li>Remove 1× TE buffer without disturbing magnetic beads and repeat step iii.</li>
<li>Remove 1× TE buffer and resuspend the beads in 5 mL of nuclease free water.</li>
<li>Add resuspended beads to the PEG buffer (the volume of PEG buffer is around 140 mL)
<div class="note">
<span class="note-title">Note:</span> PEG buffer recipe is provided in the <a href="#materials-and-equipment">materials and equipment</a> section.</div>
</li>
<li>Wash the 50 mL conical tube with another 5 mL nuclease free water and add to PEG buffer.</li>
<li>Store the PEG buffer containing magnetic beads at 4°C in dark.
<div class="note">
<span class="note-title">Note:</span> Use the prepared SPRI magnetic bead suspension within one month.</div>
</li>
</ol>
</li>
<li>Cell lysis
<ol type="i">
<li>Remove 1× PBS from the plate and add 200 μL/well of 30 mM glycine solution in 1× PBS for 5 min to quench PFA.</li>
<li>Remove glycine solution and wash cells 2 times with 1× PBS.</li>
<li>Aliquot sufficient volume of the lysis buffer containing Proteinase K (described in the <a href="#materials-and-equipment">materials and equipment</a> section) in a separate tube (50 μL per well, i.e., 5 mL/plate).</li>
<li>Add BCG to the lysis buffer and mix. The number of BCG per well should approximately match the expected Mtb load. For example, add 15×10<sup>3</sup> CFU of BCG per well if monolayers were infected with Mtb at MOI =1, i.e., add 1.5×10<sup>6</sup> BCG to the 5 mL of the lysis buffer to prepare the stock solution.</li>
<li>Remove 1× PBS and add 50 μL of the lysis buffer stock solution containing BCG to each well.
<div class="note">
<span class="note-title">Note:</span> Avoid drying the fixed monolayers.</div>
</li>
<li>Incubate the plate at 60<sup>0</sup> C for 45 min <u>to lyse macrophages</u> and release the bacteria.</li>
<li>Transfer the lysate to a PCR plate and heat at 95°C for 45 min <u>to lyse the bacteria</u> (Mtb and BCG).</li>
<li>Centrifuge the plate for 1 min at 200 × g to collect liquid at the bottom and add an equal volume (50 μL) of neutralization buffer (40 mM Tris-HCl, pH ∼ 6, see note in the <a href="#materials-and-equipment">materials and equipment</a> section). By adding an equal volume of the neutralization buffer to the lysate, the final pH will be adjusted to 7.5–8.0).
<div class="note">
<span class="note-title">Note:</span> At this stage the lysates can be used immediately for DNA isolation, stored 16–18 h at 4°C, or stored frozen at −20°C for up to 1 month.</div>
</li>
</ol>
</li>
<li>DNA isolation and enrichment
<ol type="i">
<li>
<b><u>For DNA isolation</u></b> add 70 μL SPRI magnetic bead suspension (prepared in step 52 a) per well and mix by pipetting up and down, until the bead suspension is homogeneous.</li>
<li>Incubate the PCR plate at 22°C–25°C for 15–20 min.</li>
<li>Place the PCR plate on a magnetic plate (Life Technologies, Cat. #12331D) and allow precipitation of magnetic beads for 15–20 min.
<div class="note">
<span class="note-title">Note:</span> an extended time of 15–20 min is necessary because the lysate is viscous.</div>
</li>
<li>Keeping the plate on the magnetic plate, carefully remove flow-through without disturbing or removing any magnetic beads (approx. volume would be ∼170 μL) (<a href="#fig3">Figure 3</a>B).</li>
<li>Add 180 μL of 70% ethanol per well to wash the SPRI magnetic beads.</li>
<li>Incubate the plate for 2 min at 22°C–25°C and remove 70% ethanol keeping the PCR plate on the magnetic plate.</li>
<li>Repeat the wash step two more times, and remove ethanol.</li>
<li>Remove PCR plate from the magnetic plate and incubate at 37°C for 15 min to evaporate residual ethanol.</li>
<li>Add 40 μL nuclease free water, mix by pipetting up and down, incubate at 22°C–25°C for 15–20 min to elute DNA from the beads.</li>
<li>Place the PCR plate back on the magnetic plate at 22°C–25°C for 2 min.</li>
<li>Keeping the PCR plate on the magnetic plate, carefully transfer 35 μL (If 40 μL was the elution volume used) of isolated and enriched DNA to a new PCR plate.</li>
<li>Cap the plate and store it at −20°C for long-term storage or continue to the qPCR step.</li>
</ol>
</li>
</ol>
</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. DNA isolation using magnetic beads</div>
<p>(A and B) Washing magnetic beads with 1× TE buffer (B) Separation of flow through from DNA containing magnetic beads using magnetic plate.</p>
</figcaption>
</figure>
<h3 id="sec3.8">Mtb- and BCG-specific primers and probes</h3>
<p>Mtb-specific primers amplify a 146 bp fragment of the <i>esat6</i> (Rv3875) gene, which is encoded within the region of difference 1 (RD1) which is absent in BCG. BCG-specific primers anneal immediately upstream and downstream of the RD1 region and amplify a 162 bp fragment covering the junction region absent in Mtb.</p>
<p>For Mtb probe the 5′ reporter dye used is 6-FAM<sup>TM</sup> (code 56-FAM) and the 3′ quencher dye is TAMRA<sup>TM</sup>-SP (36-TAMsp), while for the BCG probe the 5′ reporter dye used is Cy5<sup>TM</sup> (5Cy5) and the 3′ quencher dye is Iowa black® RQ-Sp (3IAbRQSP).</p>
<p>Details of primer design are shown in <a href="#fig4">Figure 4</a> and primers and TaqMan probes used are listed in the <a href="#materials-and-equipment">materials and equipment</a> section.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Mtb- and BCG-specific primers and probes design</div>
<p>The Mtb and BCG specific primers were designed considering absence of RD1 in BCG.</p>
</figcaption>
</figure>
<h3 id="sec3.9">qPCR and normalization for one step multiplex qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h post DNA isolation to results</div>
<p>The one step multiplex qPCR is standardized for Mtb and BCG genome specific primer-probes pairs. The BCG is used as an internal standard for the normalization of every step of the sample processing.</p>
<ol start="53">
<li>Reaction Master Mix and Plate Setup for qRT-PCR
<ol type="a">
<li>Thaw TaqMan Environmental Master Mix 2.0, Mtb and BCG primers/probes on ice, protected from light.</li>
<li>Mix the master mix by pipetting up and down, and mix the primers and probes by inverting 3–4 times, then briefly spin to collect liquid at the bottom.</li>
<li>Prepare the Mtb and BCG specific 2× primers and TaqMan probes mix as follows:
<table id="undtbl12">
<thead>
<tr>
<th colspan="4">Primer - probe master mix preparation</th>
</tr>
<tr>
<th>Component</th>
<th>Stock concentration (picomole/μL)</th>
<th>Working concentration (picomole/μL)</th>
<th>Volume for 100 μL mix (2×)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Forward primer</td>
<td>100</td>
<td>10</td>
<td>20 μL</td>
</tr>
<tr>
<td>Reverse primer</td>
<td>100</td>
<td>10</td>
<td>20 μL</td>
</tr>
<tr>
<td>Probe</td>
<td>100</td>
<td>10</td>
<td>20 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>–</td>
<td>–</td>
<td>40 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Determine the number of reactions based on the number of wells you have in 96-well plate.</li>
<li>To prepare the one-step multiplex qPCR master mix, see the table below:
<table id="undtbl13">
<thead>
<tr>
<th colspan="2">One-step multiplex qPCR master mix recipe (for 20 μL)</th>
</tr>
<tr>
<th>Reagent</th>
<th>Volume per reaction (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>TaqMan Environmental Master Mix 2.0</td>
<td>10</td>
</tr>
<tr>
<td>Mtb primer/probe mix</td>
<td>0.5</td>
</tr>
<tr>
<td>BCG primer/probe mix</td>
<td>0.5</td>
</tr>
<tr>
<td>Isolated DNA template</td>
<td>9</td>
</tr>
</tbody>
</table>
</li>
<li>Mix by gently pipetting up and down. Centrifuge briefly at 4°C for 15 s at 1,000 × g.</li>
<li>Dispense 11 μL of the master mix reaction into a new 96 well PCR plate (Bio-Rad Cat. #MLL9601).</li>
<li>Add 9 μL template DNA (Isolated using magnetic beads protocol, step 52 c).</li>
<li>Seal the plate and centrifuge briefly at 4°C for 1 min at 200 × g.</li>
</ol>
</li>
</ol>
<ol start="54">
<li>qPCR
<ol type="a">
<li>Turn on the machine and allow the block to equilibrate.</li>
<li>Set up the following program: Total run time: ∼ 120 min
<table id="undtbl14">
<thead>
<tr>
<th colspan="3">One-step multiplex qPCR program</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature (<sup>o</sup>C)</th>
<th>Time</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>50</td>
<td>2 min</td>
</tr>
<tr>
<td>2</td>
<td>95</td>
<td>10 min</td>
</tr>
<tr>
<td>3</td>
<td>95</td>
<td>15 s</td>
</tr>
<tr>
<td>4</td>
<td>60</td>
<td>30 s</td>
</tr>
<tr>
<td>5 (repeat steps 3 and 4, 39×)</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>6</td>
<td>40</td>
<td>5 min</td>
</tr>
<tr>
<td>7</td>
<td>4</td>
<td><b>∞</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Before starting the run, designate each target gene with the appropriate fluorophore: Mtb=FAM/6-TAMsp and BCG=Cy5/3IAbRQSP (as described in section 8).</div>
</li>
<li>Load the 96-well plate and start the run.</li>
</ol>
</li>
</ol>
<ol start="55">
<li>Standard curves of Mtb and BCG specific primers and TaqMan probes:</li>
</ol>
<p>A known amount of Mtb and BCG genomic DNA (0.1–1,000 pg with 10-fold dilutions) is run in triplicate together with the samples and used as a standard quantity (SQ) for quantification of DNA from samples. <a href="#fig5">Figure 5</a> shows the standard curves of Mtb and BCG specific primers and TaqMan probes in multiplex qPCR reaction.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Standard curves of Mtb and BCG DNA</div>
<p>(A and B) Standard curves of (A) Mtb and (B) BCG DNA were generated by qPCR using specific primers/probes and serial dilutions of known concentrations of Mtb and BCG genomic DNA. The goodness of fit R square (0.9847 for Mtb and 0.997 for BCG) was calculated by simple linear regression.</p>
</figcaption>
</figure>
<ol start="56">
<li>Quantification of genome equivalent<br> BCG and Mtb have similar genome sizes of approximately 4.37 and 4.4 MB. Based on the genome sizes, we calculated the weight of BCG and Mtb genome (which was 2.11 and 2.06 genomes in 1 femtogram for BCG and Mtb subsequently) and used it to calculate genome equivalent after qPCR as follows:<br> <b>Calculation of weight of Mtb genome:</b><br> Genome size: 4.4 × 106 bp<br> Mass of 1 bp = 660 Dalton<br> Weight of Mtb genome  = genome size × mass of 1 bp<br>                       = 4.4 × 106 × 660<br>                       = 2.9 × 108<br> 1 genome in grams    = 1.67 × 10-24 (grams per Dalton) × 2.9 × 108<br>                       = 4.84 x10-16 or 4.84 × 10-10 μg or 4.84 × 10-7 ng<br> Genomes in 1 ng      = 1/4.84 × 10-7<br>                       = 2.06 × 106<br> Genomes in 1 pg      = 2.06 × 103 Genomes in 1 fg        = 2.06
<ol type="a">
<li>Quantify standard quantity (SQ) of DNA in 9 μL template using standard concentration of BCG or Mtb DNA.
<div class="note">
<span class="note-title">Note:</span> Load standards also in 9 μL total volume.</div>
</li>
<li>Multiply the SQ by the amount of DNAse-free water used for DNA elution from magnetic beads (we isolate DNA in 40 μL nuclease free MQ water).</li>
<li>Calculate the SQ in femtograms.</li>
<li>To quantify total genome equivalents per well, multiply the values from step 56 c (in femtograms) by 2.06.<br> The example of genome equivalent calculation is provided in <a href="#tbl2">Table 2</a>.</li>
</ol>
</li>
</ol>
<ol start="57">
<li>Quantification of Mtb fold change<br> The changes in bacterial loads in each well are calculated using ΔΔCt method with BCG Ct values serving as the housekeeping standard and Mtb Ct as a gene of interest. The calculations are presented in <a href="#tbl3">Tables 3</a> and <a href="#tbl4">4</a> and described below:
<ol type="a">
<li>Subtract the Mtb Ct values from BCG Ct values for corresponding well:
<p class="chem">[Ct <sub>Mtb</sub> - Ct <sub>BCG</sub> = x]</p>
</li>
<li>Calculate the 2<sup>-x</sup> (fold change as compared to the BCG standard)</li>
<li>Normalize Mtb load per well adjusting for the total cell number in that well (<a href="#tbl4">Table 4</a>).
<div class="note">
<span class="note-title">Note:</span> Use the total number of macrophages (live and dead) (step 51) per well to normalize the Mtb load to account for cell loss during Mtb infection. The number of dead cells provide additional information regarding macrophage killing by Mtb under specific treatment conditions<b>.</b>
</div>
</li>
<li>Calculate the average Mtb load for the <u>untreated Mtb-infected control (UTC)</u> group.</li>
<li>Divide the values in each well (calculated in 9.5c) by the UTC average to determine fold Mtb change per well and perform appropriate statistical analysis (unpaired t-test in this example)</li>
</ol>
</li>
</ol>
<table id="tbl2">
<caption>Table 2. Calculation of genome equivalent</caption>
<thead>
<tr>
<th>Group</th>
<th>Ct Mtb</th>
<th>Standard quality in pg (SQ)</th>
<th>SQ in total eluted volume (T) (SQ × 4.44)</th>
<th>SQ in fg (T × 10ˆ3)</th>
<th>Genome equivalent (SQ in fg X 2.06)</th>
<th>p <i>value</i>
</th>
</tr>
</thead>
<tbody>
<tr>
<td>Std-1</td>
<td>26.7</td>
<td>1,000.0</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-1</td>
<td>26.9</td>
<td>1,000.0</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-2</td>
<td>29.8</td>
<td>100.0</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-2</td>
<td>29.7</td>
<td>100.0</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-3</td>
<td>33.1</td>
<td>10.0</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-3</td>
<td>33.0</td>
<td>10.0</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-4</td>
<td>35.6</td>
<td>1.0</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-4</td>
<td>35.5</td>
<td>1.0</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-5</td>
<td>38.3</td>
<td>0.1</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Std-5</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>UT</td>
<td>31.3</td>
<td>31.5</td>
<td>139.8</td>
<td>139,788.1</td>
<td>287,963.5</td>
<td rowspan="10">0.0004</td>
</tr>
<tr>
<td>UT</td>
<td>30.9</td>
<td>43.2</td>
<td>192.0</td>
<td>191,980.5</td>
<td>395,479.9</td>
</tr>
<tr>
<td>UT</td>
<td>31.3</td>
<td>31.7</td>
<td>140.6</td>
<td>140,642.2</td>
<td>289,723.0</td>
</tr>
<tr>
<td>UT</td>
<td>31.5</td>
<td>27.2</td>
<td>120.7</td>
<td>120,737.3</td>
<td>248,718.7</td>
</tr>
<tr>
<td>UT</td>
<td>31.4</td>
<td>29.3</td>
<td>130.0</td>
<td>130,044.8</td>
<td>267,892.3</td>
</tr>
<tr>
<td>IFNγ</td>
<td>34.0</td>
<td>3.6</td>
<td>16.1</td>
<td>16,075.0</td>
<td>33,114.4</td>
</tr>
<tr>
<td>IFNγ</td>
<td>33.2</td>
<td>6.8</td>
<td>30.2</td>
<td>30,217.5</td>
<td>62,248.0</td>
</tr>
<tr>
<td>IFNγ</td>
<td>32.2</td>
<td>14.8</td>
<td>65.8</td>
<td>65,822.8</td>
<td>135,594.9</td>
</tr>
<tr>
<td>IFNγ</td>
<td>32.6</td>
<td>10.8</td>
<td>47.8</td>
<td>47,844.9</td>
<td>98,560.5</td>
</tr>
<tr>
<td>IFNγ</td>
<td>32.0</td>
<td>17.5</td>
<td>77.7</td>
<td>77,748.5</td>
<td>160,161.8</td>
</tr>
</tbody>
</table>
<table id="tbl3">
<caption>Table 3. Example of Mtb load calculation without normalization for the macrophage number</caption>
<thead>
<tr>
<th>Group</th>
<th>Ct Mtb</th>
<th>Ct BCG</th>
<th>a. dCt (Mtb-BCG)</th>
<th>b. 2ˆ-X</th>
<th>d. Average UTC</th>
<th>e. Mtb fold change</th>
<th>p <i>value</i>
</th>
</tr>
</thead>
<tbody>
<tr>
<td>UT</td>
<td>31.73</td>
<td>29.74</td>
<td>1.99</td>
<td>0.25</td>
<td rowspan="3">0.33</td>
<td>0.76</td>
<td rowspan="6">0.05</td>
</tr>
<tr>
<td>UT</td>
<td>30.80</td>
<td>29.75</td>
<td>1.05</td>
<td>0.48</td>
<td>1.46</td>
</tr>
<tr>
<td>UT</td>
<td>32.04</td>
<td>30.02</td>
<td>2.01</td>
<td>0.25</td>
<td>0.75</td>
</tr>
<tr>
<td>IFNγ</td>
<td>33.24</td>
<td>29.84</td>
<td>3.40</td>
<td>0.09</td>
<td> </td>
<td>0.29</td>
</tr>
<tr>
<td>IFNγ</td>
<td>34.00</td>
<td>30.44</td>
<td>3.56</td>
<td>0.09</td>
<td> </td>
<td>0.26</td>
</tr>
<tr>
<td>IFNγ</td>
<td>31.52</td>
<td>28.73</td>
<td>2.79</td>
<td>0.14</td>
<td> </td>
<td>0.44</td>
</tr>
</tbody>
</table>
<p class="table-legend">Data for <a href="#fig7">Figure 7</a>B.</p>
<table id="tbl4">
<caption>Table 4. Example of Mtb load calculation after normalization for the macrophage number</caption>
<thead>
<tr>
<th>Group</th>
<th>Ct Mtb</th>
<th>Ct BCG</th>
<th>dCt (Mtb-BCG)</th>
<th>2ˆ-X</th>
<th>Cells per well</th>
<th>NormalizedMtb per cell</th>
<th>Average UTC</th>
<th>Mtb fold change</th>
<th>p <i>value</i>
</th>
</tr>
</thead>
<tbody>
<tr>
<td>UT</td>
<td>31.73</td>
<td>29.74</td>
<td>1.99</td>
<td>0.25</td>
<td>28,803</td>
<td>8.7593E-06</td>
<td rowspan="3">1.00001E-05</td>
<td>0.88</td>
<td rowspan="6">0.03</td>
</tr>
<tr>
<td>UT</td>
<td>30.80</td>
<td>29.75</td>
<td>1.05</td>
<td>0.48</td>
<td>35,503</td>
<td>1.357E-05</td>
<td>1.36</td>
</tr>
<tr>
<td>UT</td>
<td>32.04</td>
<td>30.02</td>
<td>2.01</td>
<td>0.25</td>
<td>32,303</td>
<td>7.6712E-06</td>
<td>0.77</td>
</tr>
<tr>
<td>IFNγ</td>
<td>33.24</td>
<td>29.84</td>
<td>3.40</td>
<td>0.09</td>
<td>22,990</td>
<td>4.1135E-06</td>
<td> </td>
<td>0.41</td>
</tr>
<tr>
<td>IFNγ</td>
<td>34.00</td>
<td>30.44</td>
<td>3.56</td>
<td>0.09</td>
<td>23,514</td>
<td>3.6158E-06</td>
<td> </td>
<td>0.36</td>
</tr>
<tr>
<td>IFNγ</td>
<td>31.52</td>
<td>28.73</td>
<td>2.79</td>
<td>0.14</td>
<td>27,945</td>
<td>5.1627E-06</td>
<td> </td>
<td>0.52</td>
</tr>
</tbody>
</table>
<p class="table-legend">Data for <a href="#fig7">Figure 7</a>C.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">Time-course for macrophage survival and mycobacterial loads</h3>
<p>The method provides simultaneous assessment of macrophage cell numbers, percentage of dead cells and Mtb loads in individual samples (wells). Use of high-quality macrophage monolayers, single-cell Mtb suspensions, low MOI and change of culture medium every 2 days enables us to perform extended time-course experiments. Usually, individual plates are fixed at two-day intervals (for example, days 1, 3, 5 and 7 post infection). All plates are lysed using the same lysis and therefore, equal amount of the BCG spike is added across samples and plates. The use of equal BCG spike allows for accurate comparison of Mtb loads across multiple samples fixed on different days. This allows us to reliably distinguish patterns of the intracellular bacterial growth, persistence and killing. In parallel, we assess macrophage survival, proliferation or death. Additional fluorescent probes and fluorescent multiplexed imaging can be utilized to characterize macrophage status in greater detail. <a href="#fig6">Figure 6</a> shows Mtb replication through a time-course and macrophage survival/death experiment. Mtb loads and macrophage death rates show an increase in a time-dependent manner. Thus, this method is equally suitable for assessing effects of interventions targeting the pathogen and/or the host cells. We used this method in our recent study to show that rocaglate (CMLD010538) in combination with the cytokine IL-4 increases macrophage survival and improves macrophage effector functions to control Mtb replication (<a href="#bib5">Chatterjee et al., 2021</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Time-course of BMDM infection with Mtb</div>
<p>(A and B) The BMDMs were infected with Mtb at MOI 1 for 2 h in 96 well plate, extracellular Mtb was killed using 200 μg/mL amikacin for 1 h. The cells were infected for different time points. At day of harvest the cells were washed with 1× PBS, stained with Biotium red, fixed using 4% PFA and took out from BSL3. (A) Total cell number (left Y-axis) and percentage cell death (Right Y-axis) was observed using Celigo cytometer and plotted. (B) Mtb load was calculated using BCG spike and cell number as normalization. The data are represented as mean ± SEM and p value ≤0.05 was considered statistically significant.</p>
</figcaption>
</figure>
<h3 id="sec4.2">Cell number</h3>
<p>Virulent Mtb can kill infected macrophages. We also observed that some compounds become toxic in the presence of virulent Mtb, even at concentrations that were not toxic for non-infected macrophages. In addition, host genetic background, co-infections, cytokines and other biological variables can modify the macrophage survival. The detachment and loss of the infected cells may generate an illusion of the bacterial control. If excessive macrophage death and cell loss during the time course of Mtb infection are observed in some samples, they should be removed from subsequent analyses. The normalization of Mtb loads to the total cell numbers in individual wells accounts for experimental variation and some unavoidable loss of cells due to multiple washes. Inspecting fixed macrophage monolayers allows for the detection and removal of technical outliers (as illustrated in <a href="#fig7">Figure 7</a> and <a href="#tbl3">Tables 3</a> and <a href="#tbl4">4</a>). Although this method can be adopted for screening of compound libraries using robotics, we have developed this approach for testing candidate compounds and specific pathway inhibitors in a medium throughput format. First, we determine non-toxic concentrations for each compound using non-infected macrophages. We also use a range of the compound concentrations during Mtb infections, since virulent Mtb can synergistically increase macrophage death.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Effect of treatment with IFNγ on macrophage survival and Mtb load</div>
<p>(A–C) The BMDMs were pre-treated with 100 U/mL IFNγ for 16 h and subsequently infected with Mtb at MOI 1 for 2 h, extracellular Mtb was killed using 200 μg/mL amikacin for 1 h. After phagocytosis cells were treated with IFNγ for 24 h, cells were washed with 1× PBS, stained with staining solution, fixed using 4% PFA and took out from BSL3. (A) Total cell number was observed using Celigo cytometer and plotted. (B) Mtb load without normalization with cell number. (C) Mtb load after normalization with cell number. The data are represented as mean ± SEM and p value ≤0.05 was considered statistically significant.</p>
</figcaption>
</figure>
<p>The use of fixed macrophage monolayers allows for in-depth analyses of macrophages and comparison of multiple timepoints outside BSL3. Overall, this technique complements a commonly used CFU enumeration method by increasing throughput, while simultaneously eliminating the need of biocontainment for all method-specific steps.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>In absence of bacterial DNA degradation induced by the macrophages, quantification techniques based on DNA detection cannot distinguish between DNA from alive, viable but not growing, or dead bacteria. Consequently, we see no drop in bacterial load over the time-course of treatment of liquid Mtb cultures with rifampicin (bactericidal) and isoniazid (bacteriostatic) antibiotics. Therefore, this method should not be applied to liquid Mtb cultures.</p>
<p>However, in IFNγ-activated macrophages, we observed the decrease in the bacterial loads within 24 h post infection. This indicates that activated macrophage can kill intracellular Mtb and eliminate the dead Mtb DNA, perhaps via enzymatic digestion. Although we have not determined the half-life of dead Mtb DNA in macrophages, our experiments proceed for 3–7 days – a period sufficient for the elimination of the dead Mtb DNA. Importantly, the described method of Mtb DNA isolation can be adopted for use in combination with DNA-based approaches, other than Mtb genome quantification, such as the replication clock (<a href="#bib11">Gill et al., 2009</a>), transposon insertion site sequencing, competition of pools of the bar-coded Mtb strains and Mtb mutants, as well as for the DNA-based TB diagnostics. However, each novel application of this technique will require additional optimization and validation steps.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<h4 id="sec6.1.1">Incomplete lysis of PFA fixed macrophages (step 52 b).</h4>
<p>Lysis buffer (25 mM NaOH and 0.2 mM EDTA) is quick and efficient when used on unfixed cells. Upon the 4% PFA fixation, the macrophage monolayer is less efficiently lysed.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Incubate the monolayer with lysis buffer containing 0.2 mg/mL proteinase K at 60°C for 45 min. The addition of proteinase K to lysis buffer achieves complete lysis of macrophages (<b>Table below),</b> does not alter mycobacterial DNA quality and improves qPCR yields.</p>
<table id="undtbl15">
<thead>
<tr>
<th colspan="4">Lysis conditions for 4% PFA fixed BMDMs</th>
</tr>
<tr>
<th>Buffer</th>
<th>Temperature</th>
<th>Time (min)</th>
<th>Observation</th>
</tr>
</thead>
<tbody>
<tr>
<td>Lysis Buffer</td>
<td>20°C–25°C</td>
<td>120</td>
<td>Partial lysis</td>
</tr>
<tr>
<td>Lysis Buffer + Proteinase K</td>
<td>20°C–25°C</td>
<td>120</td>
<td>Complete lysis</td>
</tr>
<tr>
<td>Lysis Buffer</td>
<td>37°C</td>
<td>60</td>
<td>Partial lysis</td>
</tr>
<tr>
<td>Lysis Buffer + Proteinase K</td>
<td>37°C</td>
<td>60</td>
<td>Complete lysis</td>
</tr>
<tr>
<td>Lysis Buffer</td>
<td>60°C</td>
<td>45</td>
<td>Partial lysis</td>
</tr>
<tr>
<td>Lysis Buffer + Proteinase K</td>
<td>60°C</td>
<td>45</td>
<td>Complete lysis</td>
</tr>
</tbody>
</table>
<h3 id="sec6.3">Problem 2</h3>
<h4 id="sec6.3.1">Problems with 4% PFA fixation (steps 50 and 52 b)</h4>
<p>The fixation of monolayers with 4% PFA for short duration (10 min) was not sufficient to kill Mtb while long fixation (over 60 min) makes lysis of macrophages difficult. DNA isolated from PFA fixed macrophages also shows low signal in qPCR.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>The proper duration of PFA fixation is necessary to kill Mtb and to allow lysis. We find that incubation of monolayers with 4% PFA for 30 min is more than sufficient to kill Mtb.</p>
<p>PFA fixation inhibits qPCR. To quench PFA effects, incubate fixed monolayers with 30 mM glycine in 1× PBS for 5 min and wash cells with 1× PBS before the lysis.</p>
<h3 id="sec6.5">Problem 3</h3>
<h4 id="sec6.5.1">Low recovery of Mtb DNA (steps 52 b and 52 c)</h4>
<p>NaOH concentration and incubation temperatures have to achieve complete mycobacterial lysis on the one hand, and to avoid DNA hydrolysis on the other hand. Also, DNA fragment size and salt concentration affect the efficiency of DNA isolation using SPRI magnetic beads.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Mtb and BCG bacteria were incubated in different concentration of NaOH at 65, 75 and 95°C. We found that treatment with 12.5, 25 and 50 mM NaOH at 95<sup>0</sup>C for 20 min produced a partially sheared mycobacterial DNA of desirable fragment size, while further increase of the NaOH concentration resulted in DNA hydrolysis and decrease of the qPCR signal (<a href="#fig8">Figure 8</a>).</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1506-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Standardization of mycobacterial lysis and qPCR</div>
<p>(A) <i>M. bovis</i> BCG grown in liquid culture was diluted to 1 × 10<sup>6</sup> per mL and treated with NaOH at the indicated final concentrations and temperatures for 20 min, neutralized with 40 mM Tris-HCl (pH ∼ 6) and run on 0.8% agarose gel.</p>
<p>(B) The lysate (9 μL) was used in qPCR reaction with BCG specific primers.</p>
<p>The data are represented as average Ct values.</p>
</figcaption>
</figure>
<h3 id="sec6.7">Problem 4</h3>
<h4 id="sec6.7.1">Inconsistent amplification of macrophage DNA as internal control (step 54)</h4>
<p>We observed that using our Mtb DNA isolation and purification protocol, macrophage DNA could not be used for the normalization, because amplification of mammalian genes produced variable results.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This protocol was optimized for efficient mycobacterial lysis and DNA isolation, we added mycobacterial spike to provide a better internal control. This provided control for the efficiency of mycobacterial lysis, DNA isolation and qPCR amplification. Using the same lysis buffer allows for comparison between different plates, i.e., samples prepared at different timepoints for the accurate time course analysis.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Macrophage death or loss over a time course of infection (step 51)</p>
<p>Mtb is an intracellular pathogen and loss or death of infected macrophages can lead to misinterpretation of the bacterial load.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Simultaneous control of macrophage monolayers quality and cell viability. Normalization of Mtb load to the macrophage number reduces variation within experimental groups due to cell loss during long experiments. If excessive macrophage death and cell loss during the time course of Mtb infection are observed in some samples or treatment groups, they should be removed from subsequent analyses of Mtb loads.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Agerer, F., Waeckerle, S., and Hauck, C.R. (2004). Microscopic quantification of bacterial invasion by a novel antibody-independent staining method. J. Microbiol. Methods <i>59</i>, 23-32. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref1">View at publisher</a></p>
<p id="bib2">Ambriz-Avina, V., Contreras-Garduno, J.A., and Pedraza-Reyes, M. (2014). Applications of flow cytometry to characterize bacterial physiological responses. Biomed. Res. Int. <i>2014</i>, 461941. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref2">View at publisher</a></p>
<p id="bib3">Barczak, A.K., Avraham, R., Singh, S., Luo, S.S., Zhang, W.R., Bray, M.A., Hinman, A.E., Thompson, M., Nietupski, R.M., Golas, A., et al. (2017). Systematic, multiparametric analysis of Mycobacterium tuberculosis intracellular infection offers insight into coordinated virulence. PLoS Pathog. <i>13</i>, e1006363. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref3">View at publisher</a></p>
<p id="bib4">Brodin, P., Poquet, Y., Levillain, F., Peguillet, I., Larrouy-Maumus, G., Gilleron, M., Ewann, F., Christophe, T., Fenistein, D., Jang, J., et al. (2010). High content phenotypic cell-based visual screen identifies Mycobacterium tuberculosis acyltrehalose-containing glycolipids involved in phagosome remodeling. PLoS Pathog. <i>6</i>, e1001100. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref4">View at publisher</a></p>
<p id="bib5">Chatterjee, S., Yabaji, S.M., Rukhlenko, O.S., Bhattacharya, B., Waligurski, E., Vallavoju, N., Ray, S., Kholodenko, B.N., Brown, L.E., Beeler, A.B., et al. (2021). Channeling macrophage polarization by rocaglates increases macrophage resistance to Mycobacterium tuberculosis. iScience <i>24</i>, 102845. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref5">View at publisher</a></p>
<p id="bib6">Chen, P., Shi, M., Feng, G.D., Liu, J.Y., Wang, B.J., Shi, X.D., Ma, L., Liu, X.D., Yang, Y.N., Dai, W., et al. (2012). A highly efficient Ziehl-Neelsen stain: identifying de novo intracellular Mycobacterium tuberculosis and improving detection of extracellular M. tuberculosis in cerebrospinal fluid. J. Clin. Microbiol. <i>50</i>, 1166-1170. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref6">View at publisher</a></p>
<p id="bib7">Cohen, S.B., Gern, B.H., Delahaye, J.L., Adams, K.N., Plumlee, C.R., Winkler, J.K., Sherman, D.R., Gerner, M.Y., and Urdahl, K.B. (2018). Alveolar macrophages provide an early Mycobacterium tuberculosis niche and initiate dissemination. Cell Host Microbe <i>24</i>, 439-446 e434. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref7">View at publisher</a></p>
<p id="bib8">DeAngelis, M.M., Wang, D.G., and Hawkins, T.L. (1995). Solid-phase reversible immobilization for the isolation of PCR products. Nucleic Acids Res. <i>23</i>, 4742-4743. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref8">View at publisher</a></p>
<p id="bib9">Emerson, J.B., Adams, R.I., Roman, C.M.B., Brooks, B., Coil, D.A., Dahlhausen, K., Ganz, H.H., Hartmann, E.M., Hsu, T., Justice, N.B., et al. (2017). Schrodinger's microbes: tools for distinguishing the living from the dead in microbial ecosystems. Microbiome <i>5</i>, 86. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref9">View at publisher</a></p>
<p id="bib10">Frossard, A., Hammes, F., and Gessner, M.O. (2016). Flow cytometric assessment of bacterial abundance in soils, sediments and sludge. Front. Microbiol. <i>7</i>, 903. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref10">View at publisher</a></p>
<p id="bib11">Gill, W.P., Harik, N.S., Whiddon, M.R., Liao, R.P., Mittler, J.E., and Sherman, D.R. (2009). A replication clock for Mycobacterium tuberculosis. Nat. Med. <i>15</i>, 211-214. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref11">View at publisher</a></p>
<p id="bib12">Heesemann, J. and Laufs, R. (1985). Double immunofluorescence microscopic technique for accurate differentiation of extracellularly and intracellularly located bacteria in cell culture. J. Clin. Microbiol. <i>22</i>, 168-175. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref12">View at publisher</a></p>
<p id="bib13">Hendon-Dunn, C.L., Doris, K.S., Thomas, S.R., Allnutt, J.C., Marriott, A.A., Hatch, K.A., Watson, R.J., Bottley, G., Marsh, P.D., Taylor, S.C., et al. (2016). A flow cytometry method for rapidly assessing Mycobacterium tuberculosis responses to antibiotics with different modes of action. Antimicrob. Agents Chemother. <i>60</i>, 3869-3883. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref13">View at publisher</a></p>
<p id="bib14">Jepras, R.I., Paul, F.E., Pearson, S.C., and Wilkinson, M.J. (1997). Rapid assessment of antibiotic effects on Escherichia coli by bis-(1,3-dibutylbarbituric acid) trimethine oxonol and flow cytometry. Antimicrob. Agents Chemother. <i>41</i>, 2001-2005. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref14">View at publisher</a></p>
<p id="bib15">Keller, C., Mellouk, N., Danckaert, A., Simeone, R., Brosch, R., Enninga, J., and Bobard, A. (2013). Single cell measurements of vacuolar rupture caused by intracellular pathogens. J. Vis. Exp. <i>76</i>, e50116. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref15">View at publisher</a></p>
<p id="bib16">Lerner, T.R., Borel, S., Greenwood, D.J., Repnik, U., Russell, M.R., Herbst, S., Jones, M.L., Collinson, L.M., Griffiths, G., and Gutierrez, M.G. (2017). Mycobacterium tuberculosis replicates within necrotic human macrophages. J. Cell Biol <i>216</i>, 583-594. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref16">View at publisher</a></p>
<p id="bib17">Liu, Y., Tan, S., Huang, L., Abramovitch, R.B., Rohde, K.H., Zimmerman, M.D., Chen, C., Dartois, V., VanderVen, B.C., and Russell, D.G. (2016). Immune activation of the host cell induces drug tolerance in Mycobacterium tuberculosis both in vitro and in vivo. J. Exp. Med. <i>213</i>, 809-825. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref17">View at publisher</a></p>
<p id="bib18">Mejuto, P., Luengo, M., and Diaz-Gigante, J. (2017). Automated flow cytometry: an alternative to urine culture in a routine clinical microbiology laboratory?. Int. J. Microbiol. <i>2017</i>, 8532736. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref18">View at publisher</a></p>
<p id="bib19">Pichugin, A.V., Yan, B.S., Sloutsky, A., Kobzik, L., and Kramnik, I. (2009). Dominant role of the sst1 locus in pathogenesis of necrotizing lung granulomas during chronic tuberculosis infection and reactivation in genetically resistant hosts. Am. J. Pathol. <i>174</i>, 2190-2201. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref19">View at publisher</a></p>
<p id="bib20">Queval, C.J., Song, O.R., Delorme, V., Iantomasi, R., Veyron-Churlet, R., Deboosere, N., Landry, V., Baulard, A., and Brodin, P. (2014). A microscopic phenotypic assay for the quantification of intracellular mycobacteria adapted for high-throughput/high-content screening. J. Vis. Exp. <i>83</i>, e51114. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref20">View at publisher</a></p>
<p id="bib21">Rubio, E., Zboromyrska, Y., Bosch, J., Fernandez-Pittol, M.J., Fidalgo, B.I., Fasanella, A., Mons, A., Roman, A., Casals-Pascual, C., and Vila, J. (2019). Evaluation of flow cytometry for the detection of bacteria in biological fluids. PLoS One <i>14</i>, e0220307. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref21">View at publisher</a></p>
<p id="bib22">Stanley, S.A., Barczak, A.K., Silvis, M.R., Luo, S.S., Sogi, K., Vokes, M., Bray, M.A., Carpenter, A.E., Moore, C.B., Siddiqi, N., et al. (2014). Identification of host-targeted small molecules that restrict intracellular Mycobacterium tuberculosis growth. Plos Pathog. <i>10</i>, e1003946. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref22">View at publisher</a></p>
<p id="bib23">Tracy, B.P., Gaida, S.M., and Papoutsakis, E.T. (2010). Flow cytometry for bacteria: enabling metabolic engineering, synthetic biology and the elucidation of complex phenotypes. Curr. Opin. Biotechnol. <i>21</i>, 85-99. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00121-6/sref23">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should contact Dr. Igor Kramnik (<a href="mailto:ikramnik@bu.edu">ikramnik@bu.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate datasets/code and any additional information will be available from the lead contact upon request.</p>
<h3>Acknowledgments</h3>
<p>The work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> R01 HL133190-01 (I.K.). The authors are grateful to Dr. Michael T. Kirber, the Director of Cellular Imaging Core at Boston University Medical Campus, for advice and help with imaging analysis.</p>
<h3>Author contributions</h3>
<p>Conceptualization, I.K. and A.G.; writing – original draft, S.M.Y.; writing – review &amp; editing, S.M.Y., A.G., and I.K.; methodology, I.K., S.M.Y., S.C., and A.G.; funding acquisition, I.K.; resources, I.K. and A.G.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>