<article>
<section>
<h1 id="header">Protocol for using MULTILAYER to reveal molecular tissue substructures from digitized spatial transcriptomes</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Julien Moehlin,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a></sup> Aysis Koshy,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> François Stüder,<sup><a href="#aff1">1</a></sup> and Marco Antonio Mendoza-Parra<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Génomique Métabolique, Genoscope, Institut François Jacob, CEA, CNRS, University of Evry, University Paris-Saclay, 91057 Évry, France</p>
<p id="fn1"><sup>2</sup>These authors contributed equally</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:mmendoza@genoscope.cns.fr">mmendoza@genoscope.cns.fr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100823">10.1016/j.xpro.2021.100823</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Spatially resolved transcriptomics (SrT) allow researchers to explore organ/tissue architecture from the angle of the gene programs involved in their molecular complexity. Here, we describe the use of MULTILAYER to reveal molecular tissue substructures from the analysis of localized transcriptomes (defined as gexels). MULTILAYER can process low- and high-resolution SrT data but also perform comparative analyses within multiple SrT readouts.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.cels.2021.04.008">Moehlin et al., 2021</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>MULTILAYER reveals spatial transcriptomes (ST) into biologically relevant substructures</li>
<li>MULTILAYER provides a user-friendly environment for ST processing</li>
<li>MULTILAYER has been used on a variety of data, including high-resolution ST</li>
<li>A “Visium data convertor” and a “high-resolution data compressor” are also available</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span>  &lt; 30 min</div>
<p>This section includes the minimal hardware requirements, the installation procedures, as well as the format of the files to be processed by MULTILAYER.</p>
<h3 id="sec1.1">Hardware</h3>
<p>Local-Memory: a minimum of 8GB required</p>
<h3 id="sec1.2">Downloading MULTILAYER toolkit</h3>
<ol>
<li>The Multilayer toolkit can be downloaded from <a href="https://github.com/SysFate/MULTILAYER">https://github.com/SysFate/MULTILAYER</a> (<a href="#fig1">Figure 1</a>A). This is done by clicking in the green ‘Code’ tab and downloading the zip folder. It contains the data set used as examples in the paper, the gene ontology databases, the tutorial in a pdf format, the MULTILAYER tool, MULTILAYER compressor tool, “Visium Converter” tool and a ReadMe file.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Download and Launching MULTILAYER</div>
<p>(A) The MULTILAYER toolkit is available at <a href="https://github.com/SysFate/MULTILAYER">https://github.com/SysFate/MULTILAYER</a>.</p>
<p>(B) The PowerShell prompt console with the written function to launch MULTILAYER.</p>
<p>(C) The MULTILAYER application home page providing either an “explore mode” for the analysis of a single SrT dataset, or the “batch mode” dedicated for processing multiple files.</p>
<p>(D) The MULTILAYER interface allowing to upload either raw, normalized or differentially-expressed SrT matrix data. In addition, users can transpose and/or round off the grid coordinates, as well as select the grid size if required.</p>
</figcaption>
</figure>
</li>
<li>After downloading the zip folder, extract all files.</li>
</ol>
<h3 id="sec1.3">Anaconda Python Platform</h3>
<p>MULTILAYER is functional on all operating systems (Windows, Linux and Mac OSX) with Python 3. Python version 3.8 is recommended. A simplified strategy to run Python on any operating systems is to use Anaconda.</p>
<ol start="3">
<li>Anaconda can be downloaded from <a href="https://www.anaconda.com/products/individual">https://www.anaconda.com/products/individual</a> according to individual computer specifications.</li>
<li>Once Anaconda is installed, open up the Anaconda Navigator to launch the PowerShell Prompt console.</li>
<li>Before using MULTILAYER for the first time, it is necessary to install the following dependencies as following:</li>
</ol>
<div class="textbox">
<p><code>&gt; pip install numpy</code></p>
<p><code>&gt; pip install matplotlib</code></p>
<p><code>&gt; pip install pandas</code></p>
<p><code>&gt; pip install scipy</code></p>
<p><code>&gt; pip install scikit-learn</code></p>
<p><code>&gt; pip install seaborn</code></p>
<p><code>&gt; pip install networkx</code></p>
<p><code>&gt; pip install python-louvain</code></p>
<p><code>&gt; pip install pillow</code></p>
</div>
<h3 id="sec1.4">Launching MULTILAYER</h3>
<ol start="6">
<li>To launch MULTILAYER, you have to first access the files by tracing the correct path from the PowerShell Prompt Console (<a href="#fig1">Figure 1</a>B). For e.g., If the Multilayer.py is located in the Multilayer-master folder, which is located in the downloads folder, type:</li>
</ol>
<div class="textbox">
<p><code>&gt; cd downloads∖Multilayer-master</code></p>
</div>
<ol start="7">
<li>To launch the Multilayer tool, type</li>
</ol>
<div class="textbox">
<p><code>&gt; python3 Multilayer.py</code></p>
</div>
<p>or</p>
<div class="textbox">
<p><code>&gt; python Multilayer.py</code></p>
</div>
<p>(This will depend on which version of python has been installed.)</p>
<h3 id="sec1.5">Data collection</h3>
<p>Spatially resolved transcriptomics (SrT) promotes the transition of histological studies towards a data science context, notably by the generation of digitized maps of tissue architecture. MULTILAYER takes advantage of such a digitized view to reveal molecularly-defined spatial signatures, with the use of agglomerative clustering over contiguous gene expression elements, herein defined as gexels (in analogy to pixels retrieved on digital images). Considering the recent heterogeneity of platforms available for generating SrT data (e.g., use of DNA arrays (<a href="#bib4">Rodriques et al., 2019</a>; <a href="#bib5">Ståhl et al., 2016</a>) or microfluidic channels (<a href="#bib2">Liu et al., 2020</a>)), users might refer to the corresponding platforms for the preprocessing steps (spatial barcodes demultiplexing, read counts alignment, gene/transcript read counts association, etc). In all cases, the spatial transcriptome output requires to be in the format of a matrix composed of columns associated to spatial coordinates and rows to gene identifiers.</p>
<ol start="8">
<li>Depending on the SrT platform in use, such a spatial matrix is generated within the corresponding analytical pipeline or it can be obtained as following:
<ol type="a">
<li>In cases in which the structure of the matrix is transposed (i.e., columns are associated to gene identifiers and rows to spatial coordinates) or where the provided coordinates are not as round numbers; MULTILAYER provides options to handle these types of situations. Within the “Select dataset(s)” panel, user can opt to transpose and/or round off the entry matrix (<a href="#fig1">Figure 1</a>D). At this stage MULTILAYER is not able to recognize the format of the dataset in use.</li>
<li>In cases in which the data is available as a three column dataframe (spatial coordinates, gene identifiers and read counts), the ad-hoc module called “MULTILAYER compressor”, allows to convert it into the required matrix. In addition, this module is able to decrease the resolution of the available data by agglomerating contiguous gexels defined by a user-provided compression factor (refer to section “<a href="#sec2.6">processing high-resolution SrT data</a>”).</li>
<li>In cases in which the data is issued from the commercial solution “Visium”, the ad-hoc module, called “VisiumConverter”, allows to generate the required matrix. For this, users need to download the 10<b>×</b>Genomics proprietary tool spaceranger from here: <a href="https://support.10xgenomics.com/spatial-gene-expression/software/downloads/latest">https://support.10xgenomics.com/spatial-gene-expression/software/downloads/latest</a> and proceed with the following Visium generated files:
<ol type="i">
<li>a matrix in h5 format.</li>
<li>the spatial imaging data containing the tissue_positions_list.csv file.</li>
<li>the feature barcodes matrix files containing the corresponding features.tsv.gz file.</li>
</ol>
</li>
</ol>
</li>
</ol>
<p>On the PowerShell Prompt Console, users can covert h5 matrix files generated by Visium to csv format as following:</p>
<div class="textbox">
<p><code>&gt; spaceranger mat2csv cellmatrix_HDF5.h5 out_file_matrix.csv</code></p>
</div>
<p>Where cellmatrix_HDF5.h5 is the h5 matrix files to convert to the csv format. For a detailed description of these steps, users can follow the spaceranger dedicated tutorial here: <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices">https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices</a></p>
<p>The obtained csv matrix can be processed by our ad-hoc module, called “VisiumConverter” (available at <a href="https://github.com/SysFate/MULTILAYER">https://github.com/SysFate/MULTILAYER</a><u>)</u> module as follows:</p>
<div class="textbox">
<p><code>&gt; python visiumConverter.py -m out_file_matrix.csv -p spatial/tissue_positions_list.csv -g raw_feature_bc_matrix/features.tsv.gz -o matrix_multilayer.tsv --compressor</code></p>
</div>
<p>Where -m defines the matrix to be converted; -p: the tissue positions information, -g: the features and -o: the matrix compatible with MULTILAYER. Additionally, the “--compressor" option generates a three-column format (spatial coordinates / Gene ID / read counts) file, which can be processed by “MULTILAYER compressor” in case users need to decrease gexels density prior to MULTILAYER processing. Since new Visium DNA arrays present an interstitial printed spot, the matrix generated with our “VisiumConverter” tool tends to stretch the maps on the y-axis, without a real impact on the spatial information, as illustrated on <a href="#fig2">Figure 2</a>.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. MULTILAYER display for datasets generated by the commercial platform Visium</div>
<p>A dataset corresponding to the analysis of a mouse brain section (Coronal) has been downloaded from 10<b>×</b>Genomics <a href="https://www.10xgenomics.com/resources/datasets/mouse-brain-section-coronal-1-standard">https://www.10xgenomics.com/resources/datasets/mouse-brain-section-coronal-1-standard</a>. As indicated on “Data Collection” section, we have converted the corresponding cell matrix HDF5 (filtered) to csv with spaceranger; then we have used our “visiumConverter” module to generate a MULTILAYER compatible matrix.</p>
<p>(A) Raw count matrix corresponding to the coronal mouse brain section, displayed within the MULTILAYER platform. Notice that the image seems stretched on the y-axis. This is due to the fact that Visium DNA arrays present interstitial printed spots.</p>
<p>(B and C) By applying an image rotation of 90°; a flipping conversion and a stretch of the image on the new x-axis we can obtain a view identical to that provided by the commercial platform (displayed in C).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Whole_mouse_embryo raw matrix (DBiT-Seq)</td>
<td><a href="#bib2">Liu et al., 2020</a></td>
<td><a href="https://github.com/SysFate/MULTILAYER/tree/master/Data/Whole_mouse_embryo">https://github.com/SysFate/MULTILAYER/tree/master/Data/Whole_mouse_embryo</a></td>
</tr>
<tr>
<td>High_resolution_brain raw matrices (Slide-seq)</td>
<td><a href="#bib4">Rodriques et al., 2019</a></td>
<td><a href="https://github.com/SysFate/MULTILAYER/tree/master/Data/High_resolution_brain">https://github.com/SysFate/MULTILAYER/tree/master/Data/High_resolution_brain</a></td>
</tr>
<tr>
<td>Prostate_cancer raw matrices (ST)</td>
<td><a href="#bib1">Berglund et al., 2018</a></td>
<td><a href="https://github.com/SysFate/MULTILAYER/tree/master/Data/Prostate_cancer">https://github.com/SysFate/MULTILAYER/tree/master/Data/Prostate_cancer</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>MULTILAYER</td>
<td><a href="#bib3">Moehlin et al., 2021</a></td>
<td><a href="https://github.com/SysFate/MULTILAYER">https://github.com/SysFate/MULTILAYER</a></td>
</tr>
<tr>
<td>MULTILAYER compressor</td>
<td><a href="#bib3">Moehlin et al., 2021</a></td>
<td><a href="https://github.com/SysFate/MULTILAYER">https://github.com/SysFate/MULTILAYER</a></td>
</tr>
<tr>
<td>Visium Converter</td>
<td>This article</td>
<td><a href="https://github.com/SysFate/MULTILAYER">https://github.com/SysFate/MULTILAYER</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Herein we describe Step-by-step methods for analyzing spatial read counts, from the loading of raw data, to its normalization, differential gene expression detection, gene co-expression pattern mapping and finally revealing the spatial communities corresponding to biologically relevant tissue substructures. To illustrate these various steps, we use as an example, the analysis of the Whole Mouse Embryo data generated by Liu and colleagues (<a href="#bib2">Liu et al., 2020</a>).</p>
<h3 id="sec2.1">Open data on MULTILAYER</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol>
<li>Use Explore mode for this data. This is used for the analysis of one sample. (<a href="#fig1">Figure 1</a>C).</li>
<li>Provide your dataset by clicking Raw matrix and select the appropriate data; in this case Folder ‘Multilayer-master’, Folder ‘Data’, Folder ‘Whole_mouse_embryo’, File ‘GSM4096262_0725cL.tsv’. (<a href="#fig1">Figure 1</a>D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A .tsv file (table separated values) should be provided as raw matrix. The correct format is a matrix with gene names as rows, coordinates as columns (coordinates should be in the format ‘XxY’) (see <a href="#sec1.5">data collection</a>). It is necessary to enter a raw matrix. Optionally, if you provide your own normalized matrix and / or differential expressed matrix MULTILAYER will use them. If you provide only raw matrix, you can save those generated by MULTILAYER by checking the option for ‘Save Matrix (norm, diff)’.</div>
<ol start="3">
<li>Select option to transpose matrix.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Transposing the dataset is useful when you have a matrix with gene names as columns and coordinates as rows (see <a href="#sec1.5">data collection</a>). When it is not performed (but required), the following error comes up – AttributeError: 'NoneType' object has no attribute 'columns'.</div>
<ol start="4">
<li>Optional setting to round off the matrix; default is unchecked.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> MULTILAYER requires to round out spatial coordinates for the analysis. For this, the tool needs an integer as X &amp; Y (coordinates) which will get rounded off. For eg: a given coordinate ‘3.36<b>×</b>13.94’ will be converted to ‘3<b>×</b>14’.</div>
<ol start="5">
<li>Indicate maximum matrix size or leave as default option 32 <b>×</b> 32.</li>
<li>Click Next.</li>
<li>Indicate threshold for up and down regulated differential gene expression; default range is 1 to −1 (in log2).</li>
<li>Indicate minimum number of contiguous Gexels; default is 10.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In Pattern detection, the tool uses agglomerative clustering on gexels (<a href="https://scikit-learn.org/stable/modules/gen%20erated/sklearn.cluster.AgglomerativeCluste%20ring.html">https://scikit-learn.org/stable/modules/gen erated/sklearn.cluster.AgglomerativeCluste ring.html</a>) to determine a contiguous pattern. It considers a pattern if the number of gexels is equal to or higher than this value. Default value is 10. This value is to be adapted to the size of the tissue, and the resolution of the data (number of total gexels within the matrix). An empirical evaluation of the minimal number of contiguous gexels is required on the basis of the pattern’s detection performance (as illustrated in <a href="#fig10">Figure 10</a>).</div>
<ol start="9">
<li>Indicate similarity methods- Tanimoto or dice; default is Tanimoto.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Similarity methods calculate the similarity between patterns.</div>
<p>- Tanimoto: the similarity is calculated as A Union B / A Intersection B.</p>
<p>- Dice- the similarity is calculated as 2∗(A Union B) / A Intersection B.</p>
<ol start="10">
<li>Indicate multiprocessing i.e., number of threads; default is 1.</li>
<li>Click next.</li>
<li>If the indicated matrix size is inaccurate and/or data is detected in points outside the matrix size suggested, a dialog box will appear indicating a size warning and ask if you want to open the file in a suggested size; here for e.g., 46 <b>×</b> 47. Click yes (go with MULTILAYER’s recommendation) (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Processing whole_mouse_embryo data on MULTILAYER</div>
<p>(A) When the size matrix defined by the user doesn’t comply with the grid size retrieved within the data, a size warning error is displayed. User can decide whether or not to accept the proposed grid size modification. This option does not impact the computing steps, but only the matrix display.</p>
<p>(B) As MULTILAYER processes the data, the Anaconda prompt interface displays a listing of the different steps taking place.</p>
<p>(C) Once it is finished, the processed data is available via the MULTILAYER interface.</p>
</figcaption>
</figure>
</li>
<li>The MULTILAYER interface allows at this stage to access to the various types of processed data (Raw counts, Normalized counts, Gene expression levels, Gene co-expression patterns, gexel communities) (<a href="#fig3">Figures 3</a>B and 3C).</li>
</ol>
<h3 id="sec2.2">Visualize raw and normalized data on MULTILAYER</h3>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;10 min</div>
<ol start="14">
<li>On the MULTILAYER interface, select the type of data you want to view (Raw counts, or Normalized counts).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In contrast to Raw counts, the Normalized matrix provides a view in which all read counts were corrected by using a quantile normalization strategy (as described in (<a href="#bib3">Moehlin et al., 2021</a>)). Briefly, global read counts normalization is essential to correct technical bias affecting local read count levels across the tissue section of interest. Hence, all downstream steps (differential gene expression, gene co-expression patterns detection, spatial communities’ detection) are issued from the normalized data. In cases in which the user might prefer to use their own normalization strategy, they can upload the normalized and corresponding raw matrix at the beginning of the analysis. In such a situation, MULTILAYER will not apply the quantile normalization correction, but instead it will use the provided normalized data for all downstream steps.</div>
<ol start="15">
<li>In both cases, the Menu button can be used to search for specific genes.</li>
<li>When a gene is selected, users can indicate the minimum or maximum read count threshold to enhance the display; as well as to convert it to log2 (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Visualizing a Whole Mouse Embryo data with MULTILAYER</div>
<p>(A) MULTILAYER interface providing general information, including the dataset sampel ID, matrix size, and the fraction of gexels presenting read counts.</p>
<p>(B) Raw count matrix displayed when selecting “raw counts” on the MULTILAYER interface. Each read count matrix (raw, Normalized, etc.), is accompanied by a menu bar dedicated to query for genes of interest. Read count levels per gexel are displayed as a heatmap. Hovering over a gexel on the image (indicated by white square and the pointer arrow) gives information of that particular area on the MULTILAYER interface (coordinate, number of counts or level of expression when visualizing the gene expression level matrix). The heatmap refers to the total read counts retrieved per gexel.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec2.3">Visualize differential gene expression data on MULTILAYER</h3>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;10 min</div>
<ol start="17">
<li>On the MULTILAYER interface, select “Gene Expression levels” (<a href="#fig4">Figure 4</a>A).</li>
<li>By clicking the “Menu” button, a panel displaying a list of the upregulated genes ranked by the number of associated gexels is displayed (<a href="#fig5">Figure 5</a>A).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Revealing the differential gene expression within digitized tissue sections</div>
<p>(A) By selecting “Gene Expression levels” on the MULTILAYER interface, users can access a matrix view in which all precomputed differentially expressed genes are ranked on the basis of the number of their associated gexels. A double-click on the gene FABP7, retrieved on the ranked list, allows to visualize the over-expression (in red) or repressed (in green) signature of this gene over the whole tissue. The number at the left side of the Gene name (in this case “108”) indicates the number of gexels within the whole tissue presenting a differential expression for that corresponding gene. The heatmap refers to the differential expression levels (log2) retrieved per gexel.</p>
<p>(B) when selecting “Gene co-expression patterns” on the MULTILAYER interface, a dialog box appears, allowing to select the minimal number of gexels per gene expression pattern. Like above, users can access a list of ranked genes, this time on the basis of the number of gexels per pattern. A double-click on the gene FABP7 reveals this time a cleaner view relative to (A), in which two distinct patterns (green and blue) complying with the minimal number of gexels per pattern criteria (at least 10) is displayed. For downstream analyses, users have to select a pattern of interest by clicking on one of their associated colored gexels.</p>
</figcaption>
</figure>
</li>
<li>Similarly, the list of ranked downregulated genes is available by clicking on “Show repressed genes”.</li>
<li>To visualize the gene expression location, users can either enter the gene name in the search panel, or double-click on the ranked list.</li>
<li>To enhance the display, users can modify the heatmap intervals (within the “threshold panels”), and also modify the differential expression threshold (threshold expression panels).</li>
</ol>
<h3 id="sec2.4">Visualize gene co-expression patterns</h3>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;10 min</div>
<ol start="22">
<li>On the MULTILAYER interface, select “Gene co-expression patterns” (<a href="#fig4">Figure 4</a>A).</li>
<li>A dialog box will appear requesting the minimum number of gexels to consider as part of a gene expression pattern (10 gexels by default) (<a href="#fig5">Figure 5</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The minimum number of contiguous gexels for defining a gene expression pattern depends on the size of the tissue under study, its complexity as well as the resolution of the SrT data.</div>
<ol start="24">
<li>After clicking on “Run patterns”, a matrix will be displayed. Like in the previous cases, the Menu button displays a panel in which gene patterns are ranked by the number of occupied gexels (<a href="#fig5">Figure 5</a>B).</li>
<li>To visualize the gene pattern location, users can either enter the gene name in the search panel, or double-click on the ranked list.</li>
<li>In some cases, multiple patterns (represented by different colors) can be displayed for a given gene (<a href="#fig5">Figure 5</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Multiple patterns per gene potentially correspond to biologically relevant events, notably if they are separated by several gexels (excluding a technical discontinuity).</div>
<ol start="27">
<li>Select a pattern by clicking on one of their associated colored gexels.</li>
<li>Select the minimal gene co-expression similarity (in percent).</li>
<li>By clicking on “similarity”, MULTILAYER will compute spatial gene co-expression signatures for the selected pattern. Gene co-expression is displayed by a spread of gexels from the selected pattern (corresponding to other gexels associated to co-expressed genes), as well as a heatmap corresponding to the gene co-expression similarity; i.e., from red meaning 100% of co-expression similarity (defining the location of the initial gene query), till dark blue corresponding to the least co-expression similarity level. (<a href="#fig6">Figure 6</a>A). A comprehensive list of the co-expressed genes and their corresponding similarity with the query gene is displayed within the Powershell prompt console (<a href="#fig6">Figure 6</a>B).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. A functional analysis of co-expressed genes revealed by MULTILAYER</div>
<p>(A) Gene co-expression analysis for the spatial pattern associated to the gene FABP7. After selecting one of the FABP7 patterns shown in 5b (by clicking on one of the colored gexels associated to the pattern of interest), a gene co-expression similarity (minimal threshold: 10%) is computed. The illustrated matrix, displays gexels occupancy by spatial co-expressed genes with FABP7 (heatmap: similarity level with the query gene).</p>
<p>(B) A comprehensive list of the identified co-expressed genes is displayed within the PowerShell prompt.</p>
<p>(C) By clicking on the “Gene Ontology” button in (A), a new dialog panel allows to select for a pertinent GO terms database (here Jensen TISSUES). By pressing the Run button, MULTILAYER assesses the enrichment for the pertinent GO terms and displays their ranking on the basis of their confidence.</p>
<p>(D) Enriched GO terms associated to the co-expressed genes associated to FABP7 displayed as a Barplot.</p>
</figcaption>
</figure>
</li>
<li>Click on “Gene Ontology” to access a dialog panel to select a GO terms database.</li>
<li>Select the adequate GO terms, then press “Run” to access to the Gene Ontology outcome, available either as a Barplot format or a gene vs GO terms heatmap (<a href="#fig6">Figure 6</a>C and 6D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In case users might require to interrogate a particular GO terms database; users can add the corresponding information to the folder “GO_DB” folder provided together with the MULTILAYER tool. For this purpose, users might adjust their GO terms database of interest to the format associated to the already available collections.</div>
<h3 id="sec2.5">Visualize spatial communities defining tissue substructures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;10 min</div>
<p>While with the “Gene co-expression patterns” function we can query for a gene of interest and visualize the list of other genes that are spatially co-expressed, the “Gexel Communities” function allows to screen for all gene co-expression patterns over the whole tissue, and to then classify them within spatial communities. This is performed as follows:</p>
<ol start="32">
<li>When selecting “Gexel Communities” on the MULTILAYER interface a dialog box allows users to select the gene co-expression similarity threshold (in percentage), and also the possibility to perform 15 iterative runs and include the gene co-expression similarity levels as a “weight” parameter within the gene regulatory networks used for their spatial communities partitioning (<a href="#fig7">Figure 7</a>A).
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Revealing spatial tissue substructures by spatial communities partitioning</div>
<p>(A) Dialog panel defining the minimal gene co-expression similarity to detect gene co-expression Communities. By running this process, a matrix view in which all inferred gene co-expression communities within the tissue section is displayed. Note the presence of a scrolling bar (right side) by which users can access each of the communities.</p>
<p>(B) When selecting community “1”, the list of the co-expressed genes and their corresponding spatial location is displayed.</p>
<p>(C) By clicking on Gene Ontology button in (B), a dialog panel allows to select for a pertinent GO terms database (here “Jensen_TISSUES”).</p>
<p>(D) Barplot display of the enriched GO terms associated to the community “1” displayed in (B).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Louvain algorithm is performed over multiple iterations (default value: 15 times) and the most frequent outcomes are retained. This can be seen in the PowerShell prompt console.</div>
<ol start="33">
<li>By clicking the “Run Communities” button, MULTILAYER displays a matrix where the spatial communities are highlighted (color-coded gexels) (<a href="#fig7">Figure 7</a>A).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When all communities are displayed, it’s possible for communities to overlap. If there are overlaps between communities, the tool will color the gexel as part of the community with the highest similarity.</div>
<ol start="34">
<li>By choosing one community, the over-expressed genes colocalized in that region are listed (<a href="#fig7">Figure 7</a>B).</li>
<li>Similar to the gene expression analysis described on step 4, by clicking on the “Gene Ontology” button users can access the Gene ontology analysis (<a href="#fig7">Figure 7</a>C and 7D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A video illustrating the aforementioned steps is available here: <a href="https://www.youtube.com/watch?v=zByIdsUyJPg">https://www.youtube.com/watch?v=zByIdsUyJPg</a>
</div>
<h3 id="sec2.6">Processing high-resolution SrT data</h3>
<p>Step-by-step method of analyzing high resolution data. In this particular case, datasets issued from the technology Slide-seq (<a href="#bib4">Rodriques et al., 2019</a>) is used (Hippocampus Brain section). High resolution data are, in general, stored as a three-column format, instead of a gene/coordinate matrix. Prior to MULTILAYER processing, high-resolution files need to be condensed into larger gexels, thus reducing the number of gexels to display, but simultaneously gain on read counts per gexel.</p>
<h4 id="sec2.6.1">Compress data with MULTILAYER compressor</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="36">
<li>In order to compress the file, it is necessary to open the MULTILAYER compressor tool and also indicate the correct directory path to the data.</li>
<li>In order to access the compressor tool, type in the PowerShell prompt console:</li>
</ol>
<div class="textbox">
<p><code>&gt; python3 Multilayer_compressor.py -i input.tsv -o output.tsv -cx 60 -cy 60</code></p>
</div>
<p>or</p>
<div class="textbox">
<p><code>&gt; python Multilayer_compressor.py -i input.tsv -o output.tsv -cx 60 -cy 60</code></p>
</div>
<p>For e.g.,</p>
<div class="textbox">
<p><code>&gt; python Multilayer_compressor.py -i data/high_resolution_brain/Hippocampus_matrix.tsv -o data/high_resolution_brain/Hippocampus_matrix60X.tsv -cx 60</code></p>
</div>
<p>(<a href="#fig8">Figure 8</a>A)</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Processing high-resolution hippocampus SrT with MULTILAYER</div>
<p>(A) The PowerShell prompt console displaying the function to use the MULTILAYER compressor tool. The “-cx” parameter defined to “60”, indicates a compression factor of 60 folds; i.e., the aggregation of proximal gexels such that the size of the matrix is reduced by 60 folds.</p>
<p>(B) High-resolution hippocampus spatial transcriptomics data compressed by a factor of 60<b>×</b>; 100<b>×</b> and 175<b>×</b> respectively and analyzed by MULTILAYER. Display of the spatial expression of the gene PPP3CA for these three compression factors demonstrates a conservation of its spatial signature. The heatmap refers to the total read counts retrieved per gexel.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> The format of the input data has to be in 3 columns, where the first column correspond to barcode coordinates (XxY) with header ‘bc’; the second column to genes with header ‘gene’; and the third column to the gene counts with header ‘count’.</div>
<p>The Multilayer_Compressor.py has several arguments: The function -i indicates the input matrix, -o indicates the output matrix, -cx and -cy indicates the compression factor (if -cx and -cy are identical only -cx needs to be defined). By typing</p>
<div class="textbox">
<p><code>&gt; python Multilayer_compressor.py -h</code></p>
</div>
<ul>
<li>into the PowerShell console, you get access to the help section of the compressor.</li>
</ul>
<ol start="38">
<li>The data will be stored in your output directory and can now be opened using the MULTILAYER tool.</li>
</ol>
<h4 id="sec2.6.2">Visualize compressed data on MULTILAYER</h4>
<div class="timing">
<span class="timing-title">Timing:</span> minutes</div>
<ol start="39">
<li>After being compressed, datasets can be visualized on MULTILAYER tool as described above in the example of Whole Mouse Embryo (<a href="#fig8">Figure 8</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A video illustrating the aforementioned steps is available here: <a href="https://www.youtube.com/watch?v=ww4x2aP6ENA">https://www.youtube.com/watch?v=ww4x2aP6ENA</a>
</div>
<h3 id="sec2.7">Processing multiple SrT data at once</h3>
<p>Step-by-step method of using the batch-mode option. To illustrate this module, we have used twelve prostate cancer datasets generated by <a href="#bib1">Berglund et al. (2018)</a>.</p>
<h4 id="sec2.7.1">Open data on MULTILAYER</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="40">
<li>Use Batch mode for this data (<a href="#fig9">Figure 9</a>A).
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Use of the “batch-mode” available on MULTILAYER for processing several datasets at once</div>
<p>(A) MULTILAYER main interface allowing the selection of a folder containing all datasets to process.</p>
<p>(B) Once all datasets are processed (in this case a set of twelve prostate sections), MULTILAYER provides a similar interface than that used for the “explore mode”, giving access to all different analytical panels when one dataset is selected. In addition, a “Comparison datasets” button is displayed.</p>
<p>(C) This “Comparison datasets” module, allows us to select the samples of interest for processing. Furthermore, it allows us to define the parameters for the spatial community’s detection.</p>
<p>(D) For selected datasets, users can also define the minimal number of gexels for patterns detection.</p>
<p>(E) Panel illustrating the classification of all spatial communities detected per analyzed datasets. The interactive panel allows us to visualize the spatial tissue substructure ID and its corresponding class. This information can also be downloaded as a table for further analyses (e.g., graphs display). Furthermore, this panel allows us to access the gene ontology analysis as described for the “explore” mode.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Batch mode is useful for analyzing multiple datasets from the same sample.</div>
<ol start="41">
<li>Provide your dataset by clicking Raw matrix and select the appropriate folder where all datasets to process are available (in this case the “Prostate Cancer” datasets). All .tsv files in the selected folder will be considered as the input matrix.</li>
<li>All values are kept as default values.</li>
<li>Click next.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Since multiple files are being processed, this step takes ∼6 min.</div>
<ol start="44">
<li>You are now able to visualize digitally, the raw counts, normalized counts etc. for individual samples as in the example for Whole Mouse Embryo or a comparison dataset for several samples at once on the basis of their associated spatial communities (<a href="#fig9">Figure 9</a>B).</li>
</ol>
<h4 id="sec2.7.2">Compare multiple spatial communities on MULTILAYER</h4>
<div class="timing">
<span class="timing-title">Timing:</span> minutes</div>
<ol start="45">
<li>On the Multilayer interface, select “comparison datasets” and select all the files to be compared (“Select All” button) or make your own selection (<a href="#fig9">Figure 9</a>C).</li>
<li>Default values for community’s detection are kept constant for all datasets (weight is checked, multiple iterations (15 times) is checked, threshold is 0%). Furthermore, users can adjust the minimal number of gexels for patterns detection (<a href="#fig9">Figure 9</a>D). This option is of major interest in case users have previously identified the minimal number of gexels (<a href="#fig10">Figure 10</a>).
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/998-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Spatial communities revealed over prostate sections as computed with different minimal number of gexels for pattern detection</div>
<p>Three of the twelve prostate sections analyzed in <a href="#fig9">Figure 9</a>, are displayed (P3.1, P4.3 and P3.2). When using a minimal number of gexels of 10 (i.e., &gt;9) for pattern detection, several gexels within the tissues are not associated to any of the inferred spatial communities (black-colored gexels; Top panels). By decreasing the minimal number of gexels criteria, it is possible to maximize the number of gexels associated to any of the inferred spatial communities (Bottom panels).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Users can empirically evaluate the minimal number of contiguous gexels for the analysis, by evaluating the fraction of gexels that are associated to at least one of the inferred spatial communities. As illustrated in <a href="#fig10">Figure 10</a>, by decreasing the minimal number of gexels from 10 to 5, the spatial communities retrieved within the displayed prostate tissue sections managed to incorporate most of the gexels of the tissue within the inferred patterns.</div>
<ol start="47">
<li>Run communities and save the .tsv file.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When saving the files for the comparison analysis, the tool saves 2 files (ex. save.tsv, save_filter.tsv).</div>
<ol start="48">
<li>An Interactive heatmap comes up with all the dataset communities. The x-axis represents all the dataset communities, and the y-axis represents all the classes. A class is a group of 1 or several communities. (<a href="#fig9">Figure 9</a>E).</li>
<li>For gene ontology analysis, click GO terms.</li>
<li>Select the number of GO terms and database. In this example, top GO terms =5, database= DisGeNET.</li>
<li>This gives a heatmap of the GO terms that can be saved as a plot or matrix (<a href="#fig9">Figure 9</a>E).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>MULTILAYER provides an intuitive platform for processing SrT data issued from a variety of technological platforms (e.g., DNA-array based methodologies (<a href="#bib4">Rodriques et al., 2019</a>; <a href="#bib5">Ståhl et al., 2016</a>), microfluidic channels (<a href="#bib2">Liu et al., 2020</a>)). Furthermore, thanks to additional modules, MULTILAYER can also analyze high-resolution data (MULTILAYER compressor module), as well as those issued from the commercial platform “Visium” (Visium converter module).</p>
<p>As outcome, MULTILAYER processes SrT maps as a digital image, in which users can visualize gene expression signatures, reveal gexel patterns resulting from agglomerative clustering, and even infer biologically relevant tissue substructures.</p>
<p>Considering the future needs for processing multiple datasets on a comparative manner, MULTILAYER also provides a “batch-mode”, allowing to process multiple samples at once, as well as compare them, notably by classifying the assessed tissue substructures per dataset into groups that might reveal common biological functions.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The current version of MULTILAYER reveals spatial gene co-expression signatures, but it is not able to infer their related co-regulatory relationships. Furthermore, it is not yet adapted for integrating single-cell transcriptome data. MULTILAYER is currently in progress to be updated with these two aspects. From a technical angle, MULTILAYER is limited with its visual functionalities for datasets represented within matrices &gt; 250 gexels per side. In such a situation, the use of the MULTILAYER compressor module is strongly recommended.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<p>The most common problem when running MULTILAYER is the format of the input dataset.</p>
<h3 id="sec5.1">Problem 1</h3>
<p>MULTILAYER fails to process the input dataset because the matrix has spatial coordinates as rows and genes as columns.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Use the “Transpose” option at the time of the data upload.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>MULTILAYER fails to process the input dataset because the matrix has decimal values within the gene counts.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Use the “Round” option at the time of the data upload.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>MULTILAYER fails to process the input dataset despite taking care of transposing the matrix or rounding off the read counts.</p>
<h3 id="sec5.6">Potential solution</h3>
<p>The provided data might miss values for the read counts. Make sure to fill empty cells or replace “NAs” by a value (“0” by default).</p>
<h3 id="sec5.7">Problem 4</h3>
<p>MULTILAYER fails to process the input dataset, notably because it has utilized all the available RAM of the computer.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>High-resolution data might require large amounts of RAM for processing. Either use the “MULTILAYER Compressor” module to reduce the matrix size, or increase the RAM of your system.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>Multilayer fails to process when the gene identifiers are not unique within the provided dataset.</p>
<h3 id="sec5.10">Potential solution</h3>
<p>To avoid this issue, we recommend using unique identifiers, for instance corresponding to transcripts (e.g., EnsemblID). Since the gene ontology analyses requires access to the gene symbol ID, an optimal solution is to concatenate both gene symbol and transcript ID into a single identifier separated by a “_” (e.g., ‘geneSymbol_ensemblID’). MULTILAYER is able to separate such a gene ID structure to recognize the first part of the identifier as the corresponding gene symbol to be used for the gene ontology processing when required.</p>
<h3 id="sec5.11">Problem 6</h3>
<p>MULTILAYER processing does not provide gene patterns and/or spatial communities.</p>
<h3 id="sec5.12">Potential solution</h3>
<p>The processed data does not have sufficient differential expression data, or the minimal number of contiguous gexels is not satisfied. To evaluate such potential reasons, decrease the minimal number of contiguous gexels and/or decrease the differential gene expression threshold.</p>
<h3 id="sec5.13">Problem 7</h3>
<p>The predicted spatial communities do not cover the whole surface of the tissue (i.e., a significant part of the tissue displays black gexels, not associated with any of the predicted communities).</p>
<h3 id="sec5.14">Potential solution</h3>
<p>The minimal number of contiguous gexels for patterns detection is too high. Evaluate multiple lower thresholds to identify empirically the least number of contiguous gexels that allows to predict spatial communities within the whole (maximum) surface of the tissue.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Berglund, E., Maaskola, J., Schultz, N., Friedrich, S., Marklund, M., Bergenstråhle, J., Tarish, F., Tanoglidi, A., Vickovic, S., Larsson, L., et al. (2018). Spatial maps of prostate cancer transcriptomes reveal an unexplored landscape of heterogeneity. Nat. Commun. <i>9</i>, 2419. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00529-3/sref1">View at publisher</a></p>
<p id="bib2">Liu, Y., Yang, M., Deng, Y., Su, G., Enninful, A., Guo, C.C., Tebaldi, T., Zhang, D., Kim, D., Bai, Z., et al. (2020). High-spatial-resolution multi-omics sequencing via deterministic barcoding in tissue. Cell <i>183</i>, 1665-1681.e18. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00529-3/sref2">View at publisher</a></p>
<p id="bib3">Moehlin, J., Mollet, B., Colombo, B.M., and Mendoza-Parra, M.A. (2021). Inferring biologically relevant molecular tissue substructures by agglomerative clustering of digitized spatial transcriptomes with multilayer. Cell Systems <i>21</i>, 694-705. <a class="external-link" href="https://doi.org/10.1016/j.cels.2021.04.008">https://doi.org/10.1016/j.cels.2021.04.008</a></p>
<p id="bib4">Rodriques, S.G., Stickels, R.R., Goeva, A., Martin, C.A., Murray, E., Vanderburg, C.R., Welch, J., Chen, L.M., Chen, F., and Macosko, E.Z. (2019). Slide-seq: a scalable technology for measuring genome-wide expression at high spatial resolution. Science <i>363</i>, 1463. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00529-3/sref4">View at publisher</a></p>
<p id="bib5">Ståhl, P.L., Salmén, F., Vickovic, S., Lundmark, A., Navarro, J.F., Magnusson, J., Giacomello, S., Asp, M., Westholm, J.O., Huss, M., et al. (2016). Visualization and analysis of gene expression in tissue sections by spatial transcriptomics. Science <i>353</i>, 78-82. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00529-3/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Marco Antonio Mendoza-Parra (<a href="mailto:mmendoza@genoscope.cns.fr">mmendoza@genoscope.cns.fr</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents<i>.</i></p>
<h4>Data and code availability</h4>
<p>The datasets and code used during this study are available at <a href="https://github.com/SysFate/MULTILAYER">https://github.com/SysFate/MULTILAYER</a></p>
<h3>Acknowledgments</h3>
<p>This work was supported by the institutional bodies <a href="https://doi.org/10.13039/501100006489">CEA</a>, <a href="https://doi.org/10.13039/501100004794">CNRS</a>, and Université d’Evry-Val d’Essonne. J.M. was supported by <a href="https://doi.org/10.13039/501100007149">Genopole</a> Thematic Incentive Actions funding (ATIGE-2017); A.K. by the “<a href="https://doi.org/10.13039/501100014259">Fondation pour la Recherche Medicale</a>” (FRM; funding ALZ°201912009904); and F.S. by the funding 2OI9-L22 from the <a href="https://doi.org/10.13039/501100006364">Institut National du Cancer</a> (INCa).</p>
<h3>Author contributions</h3>
<p>Conceptualization, J.M. and M.A.M.-P.; protocol elaboration, A.K., J.M., and M.A.M.-P.; software development, J.M.; “Visium converter” development, F.S.; scientific evaluation, M.A.M.-P.; writing, review, and editing, J.M., A.K., and M.A.M.-P.; funding acquisition, M.A.M.-P.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>