<article>
<section>
<h1 id="header">Protocol for labeling and fixation of intact lysosomes with esterified amino acid analogs to assess lysosomal expansion in living eukaryotic cells</h1>
<p><time datetime="2021-10-29">Published: October 29, 2021</time></p>
<p>Gianluca Scerra,<sup><a href="#aff1">1</a></sup> Maria Gabriella Caporaso,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Maurizio Renna,<sup><a href="#aff1">1</a></sup> and Massimo D’Agostino<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Medicine and Medical Biotechnologies, University of Naples ‘‘Federico II’’, 80131 Naples, Italy</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:massimo.dagostino@unina.it">massimo.dagostino@unina.it</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100916">10.1016/j.xpro.2021.100916</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The lysosomal compartment is a key hub for cell metabolism, and morphological alterations have been described in several pathological conditions. Here, we describe the use of amino acid analogs modified by the presence of a methyl ester group that accumulates within lysosomes. This generates an intraluminal osmotic effect able to trigger a rapid and selective expansion of the lysosomal compartment within 2 h of treatment. We also present protocols to preserve lysosomal morphology, which yields a more accurate size measurement.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref10">Scerra et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Methyl-group-modified amino acid analogs (LEU-ME) can localize to lysosomes</li>
<li>LEU-ME triggers a rapid expansion of the lysosomal compartment</li>
<li>Fixation procedure is a key step for preserving lysosomal morphology</li>
<li>Quantification and analysis of lysosomal dimensions</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1131-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the experimental steps required for using HeLa cells stably expressing the lysosomal protein marker Lamp1 tagged with a monomeric form of the green fluorescent protein (mGFP), which allows one to perform the confocal microscopy analysis in living cells. In this regard, the in vivo analysis does represent a key aspect, as it is instrumental in preserving the fragile architecture of the expanded lysosomes. Additionally, here we describe cell fixation methods, which are still able to preserve lysosomal morphology and allow the visualization, as well as the morphometric analysis of the enlarged lysosomes. Importantly, the latter procedure can be deployed even in non-expressing Lamp1-mGFP cell lines, by combining it with a standardized immunofluorescence protocol, which is included into this technical report. Finally, to gain the best and most reproducible effect, it is highly recommended to prepare the amino acid analogs solution at time just before usage.</p>
<h3 id="sec1.1">Overview</h3>
<p>Before starting – covers 1–3 steps: Preparation</p>
<p>Step 1 – covers 1–3 steps: Plate Lamp1-mGFP-expressing HeLa cells on clean, sterile coverslips</p>
<p>Step 2 – covers 4–8 steps: Treat cells with amino acid analogs</p>
<p>Step 3 – covers 9–10 steps: Confocal microscopy acquisition</p>
<p>Step 4 – Quantification and statistical analysis</p>
<h3 id="sec1.2">Preparation</h3>
<h4 id="sec1.2.1">Thaw and culture Lamp1/mGFP-expressing HeLa cell line</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>The aim of this part is to prepare culture cell lines starting from frozen stock aliquots.</p>
<ol>
<li>Thaw cell line from liquid nitrogen or ultra-low temperature fridge.
<ol type="a">
<li>Thaw cells by putting cryovial in ice and wait 5 min before proceeding.</li>
<li>Transfer cryovial content into a tube containing 5–10 mL of culture medium.</li>
<li>Centrifuge at 1670 rcf for 5 min.</li>
<li>Remove cryopreserving agent-containing supernatant by gentle aspiration.</li>
<li>Gently resuspend cell pellet in an appropriate volume of culture medium and transfer into a cell-culture dish.</li>
<li>Grow cells in a humidified incubator at 37°C and 5% CO<sub>2</sub>.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not add any kind of selection agent (i.e.: G-418, Neomycin, or Blasticidin) immediately after thawing.</div>
<ol start="2">
<li>Culture cells until passage 2–3. Split cells every 2–3 days.
<ol type="a">
<li>Aspirate medium.</li>
<li>Wash cells three times with sterile 1<b>×</b> PBS.</li>
<li>Add trypsin and put cells in the incubator for 5 min or up to when cells detach.</li>
<li>Add culture medium and centrifuge cells at 168rcf for 5 min.</li>
<li>Aspirate trypsin-containing medium, resuspend cell pellet in fresh medium and transfer it into a new cell-culture dish.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> You can even use non-expressing Lamp1-mGFP cell lines. In such case, you would need to fix cells before staining lysosomes with an anti-Lamp1 antibody (immunofluorescence protocol is described in the <a href="#step-by-step-method-details">step-by-step method details</a> section).</div>
<h4 id="sec1.2.2">Prepare amino acid analog stock solution</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼30 min</div>
<ol start="3">
<li>Prepare 100 mM stock solutions.
<ol type="a">
<li>Weigh an appropriate amount of powder and transfer it into a 15 mL tube.</li>
<li>Dissolve powder in deionized water by vortexing.
<div class="note">
<span class="note-title">Note:</span> these compounds are readily dissolvable in water.</div>
</li>
<li>Filter solution with 0.22 μM syringe filters.</li>
<li>Aliquot the stock solution in 1.5 mL tubes and store at −20°C.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Dispose of aliquots after 1–2 freeze/thaw cycles since the biological activity of amino acid analogs is stronger and more reproducible when they are prepared and used freshly.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal anti-LAMP1</td>
<td>Sigma-Aldrich</td>
<td>Cat#SAB4700416; RRID:<a href="http://antibodyregistry.org/AB_10932380">AB_10932380</a>
</td>
</tr>
<tr>
<td>Mouse Alexa-Fluor (488 and 546) secondary antibodies</td>
<td>Thermo Fisher Scientific-Invitrogen</td>
<td>Cat#A-11029;RRID: <a href="http://antibodyregistry.org/AB_138404">AB_138404</a>Cat#A-11030;RRID: <a href="http://antibodyregistry.org/AB_144695">AB_144695</a>
</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A4503</td>
</tr>
<tr>
<td>Formaldehyde solution 37%</td>
<td>Sigma-Aldrich</td>
<td>Cat#F15587</td>
</tr>
<tr>
<td>Glutaraldehyde solution 70%</td>
<td>Sigma-Aldrich</td>
<td>Cat#G7776</td>
</tr>
<tr>
<td>HEPES</td>
<td>Sigma-Aldrich</td>
<td>Cat#H3375</td>
</tr>
<tr>
<td>L-Leucine methyl-ester hydrochloride</td>
<td>Sigma-Aldrich</td>
<td>Cat#L1002</td>
</tr>
<tr>
<td>L-Alanine methyl-ester hydrochloride</td>
<td>Sigma-Aldrich</td>
<td>Cat#330639</td>
</tr>
<tr>
<td>L-Histidine methyl-ester hydrochloride</td>
<td>Sigma-Aldrich</td>
<td>Cat#H15403</td>
</tr>
<tr>
<td>Methanol</td>
<td>Sigma-Aldrich</td>
<td>Cat#34860</td>
</tr>
<tr>
<td>Saponin</td>
<td>Sigma-Aldrich</td>
<td>Cat#47036</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HeLa cells stably expressing Lamp1-mGFP</td>
<td>Kindly provided by prof. Andrea Ballabio (<a href="#bib9">Sbano et al., 2017</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Fiji ImageJ</td>
<td>National Institute of Health</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>10 mm-diameter coverslips</td>
<td>VWR</td>
<td>Cat#630-2115</td>
</tr>
<tr>
<td>13 mm-diameter coverslips</td>
<td>VWR</td>
<td>Cat#630-2118</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Step 1: Plate Lamp1-mGFP-expressing HeLa cells on clean, sterile coverslips</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1 h</div>
<p>Lamp1-mGFP-expressing HeLa cells are seeded on coverslips to allow cell handling in the following steps of the protocol and to allow visualization by confocal microscopy.</p>
<ol>
<li>Trypsinize cells in exponential growth phase, between passage 3 and 20.
<ol type="a">
<li>Gently remove culture medium.</li>
<li>Wash cells 1 time with pre-warmed PBS (rapid wash).</li>
<li>Add trypsin (or any detaching solution) and put cells in the incubator.
<div class="critical">
<span class="critical-title">Critical:</span> at the end of trypsinization, make sure cells are at a single-cell level by looking at them under the microscope. Computational analysis becomes difficult if cells form clusters on the coverslip.</div>
</li>
<li>Resuspend trypsinized cells in culture medium and remove trypsin by gentle centrifugation and subsequent aspiration of medium.</li>
<li>Resuspend cell pellet in an appropriate volume of culture medium, or at a desired cell concentration.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Prepare coverslips.
<ol type="a">
<li>By using appropriate forceps, put sterile coverslips at the bottom of the well(s).
<div class="critical">
<span class="critical-title">Critical:</span> We recommend using round 10mm coverslips for 24-multiwell plates or 13mm for 12-multiwell plates, which will minimize any issue (i.e.: breakage, overlay) during their handling. Coverslips must be clean; if they are not, clean them with 1% Sodium Dodecyl Sulfate (SDS), followed by extensive washes with deionized water. Moreover, coverslips must be sterile; if they are not, sterilize them at 200°C for 2h in a heater.</div>
</li>
<li>Wash the coverslips 1 time with 1<b>×</b> PBS. Make sure they adhere to the bottom of the well(s) by gently pushing them with sterile forceps after cell seeding.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This step is crucial to ensure cells do not grow below the coverslips.</div>
<ol start="3">
<li>Plate cells on coverslips.
<ol type="a">
<li>Seed cells to have about 50% confluency.
<div class="critical">
<span class="critical-title">Critical:</span> Do not seed too many cells, if characterized by a high growing rate, as the over-confluency will hamper the consequent computational analysis.</div>
On the same line, spread cells properly by extensively moving the multi-well (or whichever type of plastic support you are using) in multiple directions, to avoid the formation of cell clusters.</li>
<li>Coverslips can be coated with specific adhesion agents depending on the cell type used.</li>
<li>Using a sterile tip, push gently the coverslips to the bottom of the well since you want cells to adhere on top of the coverslip.</li>
<li>Put cells in the humidified incubator at 37°C and 5% CO<sub>2</sub> for 16 h before proceeding.</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Step 2: Treat cells with amino acid analogs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h</div>
<p>Amino acid analogs are methyl ester group-bearing amino acids, which cause the rapid, osmosis-driven, expansion of lysosomes. We tested three types of such compounds on cells (individually): L-Leucine methyl ester hydrochloride, L-Alanine methyl ester hydrochloride, L-Histidine methyl ester hydrochloride. L-Leucine methyl ester hydrochloride caused the strongest effect in our cell models.</p>
<ol start="4">
<li>Treat cells with amino acid analog.
<ol type="a">
<li>Dilute amino acid analog stock solution in culture medium at the desired concentration.
<div class="note">
<span class="note-title">Note:</span> The expansion effect is dose and time-dependent. Use a working concentration between 1mM (mild expansion) and 10mM (maximum expansion) up to at least 2h of incubation to reach the maximum effect. Moreover, if you are using phenol red-containing medium, you will notice a change in color towards yellow. Indeed, amino acid analogs are slightly acidic compounds.</div>
</li>
<li>Take cells from point 6d and gently remove culture medium.</li>
<li>Add culture medium supplemented with amino acid analogs and put back cells in the incubator for 2 h.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After having removed culture medium (7b) add the one containing amino acids analogs (7c) without introducing any further cell starvation step.</div>
<h3 id="sec2.3">Cell fixation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼30 min</div>
<p>Fixation is the most critical step of the protocol. The membrane of expanded lysosomes is extremely fragile, and their morphology can be strongly affected by the fixation method. Hence, the aim of this section is to describe a detailed protocol based on three different fixation procedures, which allows to preserve at its best the lysosomal morphology.</p>
<ol start="5">
<li>Methanol-based fixation procedure:
<ol type="a">
<li>Pour pure methanol in a multi-well or in any kind of plastic support you need to use.</li>
<li>Put methanol-containing multi-well at −20°C in a fridge.</li>
<li>Wait approximately for 15 min.</li>
<li>Put multi-well from 6d on ice.</li>
<li>Using the appropriate forceps, transfer coverslips-containing cells into cold methanol.</li>
<li>Wait 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> Do not extend fixation time over 5 min, as the methanol fixation also promotes membranes permeabilization and, as such, a longer exposure could eventually affect <i>per se</i> the lysosomal morphology.</div>
</li>
<li>Aspirate methanol and wash with 1<b>×</b> PBS by 4 or 5 rapid washes before proceeding with immunostaining step or confocal microscopy analysis.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Glutaraldehyde-based fixation procedure:
<ol type="a">
<li>Aspirate culture medium and wash the cells one time with 1<b>×</b> PBS at room temperature (<a href="#bib1">Alcantara-Ortigoza et al., 2010</a>).</li>
<li>Fix the cells by adding a mixture of 4% paraformaldehyde (PFA) and 0.05% glutaraldehyde (GA) prepared in 0.2M HEPES buffer for 10 min (<a href="#bib1">Alcantara-Ortigoza et al., 2010</a>).</li>
<li>Wash the cells once with 4% PFA (prepared in 0.2M HEPES buffer) to remove the residual GA and fix them again by adding 4% PFA and incubate them at RT for 30 min.</li>
<li>Wash the cells three time with 1<b>×</b> PBS solution before proceeding with immunostaining step or microscopy analysis.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Upon fixation, cells can be stored at 4°C for few days. If a longer period is needed, it is recommended to seal up the multi-well plate with Parafilm to avoid any evaporation of the PBS solution.</div>
<ol start="7">
<li>Formaldehyde-based fixation procedure:
<ol type="a">
<li>Aspirate culture medium and wash the cells one time with 1<b>×</b> PBS at RT.</li>
<li>Fix the cells by adding formaldehyde solution (3.7% in 1<b>×</b> PBS) and let it to act for 30 min at RT.</li>
<li>During incubation, prepare the glycine-based blocking solution (0.1M Gly in 1<b>×</b> PBS). This solution must be freshly prepared and cooled in ice before use.</li>
<li>Aspirate fixation solution and replace it with glycine-based blocking solution. Incubate on ice for 5 min.</li>
<li>Wash the cells two times with 1<b>×</b> PBS before proceeding with the immunostaining protocol.</li>
</ol>
</li>
</ol>
<h3 id="sec2.4">Cell immunostaining of lysosomal endogenous proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h</div>
<p>This section describes the immunostaining method for the endogenous lysosomal marker Lamp1 protein. This is a critical step for both the visualization of enlarged lysosomes by means of confocal microscopy and for the quantitative analysis of their morphological changes (i.e.,: number, size, diameter).</p>
<ol start="8">
<li>In our study, we used anti-Lamp1 antibody to visualize the lysosomal compartment.
<ol type="a">
<li>After having treated cells with amino acid analogs and proceeded with fixation protocol, cells were permeabilized by using blocking buffer (0.05% saponin, 1% bovine serum albumin in 1<b>×</b> PBS) for 15 min at RT.
<div class="note">
<span class="note-title">Note:</span> Methanol fixation method already promotes cell permeabilization. Thus, step 6a can be omitted.</div>
</li>
<li>During the blocking step, dilute at a 1:500 ratio the anti-Lamp1 antibody in blocking buffer.</li>
<li>Wash cells once with 1× PBS at 25°C.</li>
<li>Mix the diluted antibody by inverting or vortexing, and spin down for a second.</li>
<li>Prepare a rectangle of parafilm on the working bench by spreading a few drops of water between the parafilm and the bench to let it to adhere.</li>
<li>Add 50 μL of the diluted primary antibody solution on the parafilm. This volume is appropriate for 10 mm and 13 mm coverslips. Add more volume for larger coverslips. The right volume is the one that allows the coverslip to float on the drop.</li>
<li>Gently put the coverslip upside-down with the cells in contact with the drops of antibody-containing solution and incubate for 1 h at RT.
<div class="note">
<span class="note-title">Note:</span> This drop-based protocol allows one to reduce the consumption of the antibody for each experiment, yet without influencing the efficacy and/or reproducibility of the procedure.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Before positioning the coverslip on the drops, remove the excess PBS solution that is still adsorbed on the coverslip by letting it touch the surface of a piece of adsorbing paper. This step is fundamental to avoid any further and undesired dilution of the antibody.</div>
</li>
<li>Afte the incubation with primary antibody, add 150 μL of 1<b>×</b> PBS between parafilm and coverslip to let it float on the drops.
<div class="critical">
<span class="critical-title">Critical:</span> This step is instrumental for transferring back coverslips from Parafilm to multi-well without damaging or leaving cells on the Parafilm bedding.</div>
</li>
<li>Wash cells three times with 1<b>×</b> PBS for 5 min each.</li>
<li>Dilute the Alexa Fluor 488-conjugated secondary antibodies at a ratio of 1:500 using blocking buffer.</li>
<li>Mix the diluted secondary antibodies by inverting or vortexing, and spin down for a second.</li>
<li>Add 50 μL drops of the diluted secondary antibodies solution.
<div class="critical">
<span class="critical-title">Critical:</span> Incubation with secondary antibodies must be performed in the dark. Cover the coverslips with a cover to protect them from light and prevent photobleaching.</div>
</li>
<li>Incubate for 45 min at RT.</li>
<li>Wash cells three times with 1<b>×</b> PBS 5 min each.</li>
<li>Add 10 μL of mounting solution on the slide.</li>
<li>Dip coverslips in sterile water before putting them on the drop of mounting solution to remove any trace of salts.</li>
<li>Let them dry in a dark chamber before proceeding with microscopy analysis.</li>
</ol>
</li>
</ol>
<h3 id="sec2.5">Step 3: Confocal microscopy acquisition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>In this section we describe in detail the setting and acquisition mode applicable to either living or fixed cells by using the Ultraview VoX spinning disk system (PerkinElmer) (<a href="#bib4">D'Agostino et al., 2018</a>; <a href="#bib5">D'Agostino et al., 2017</a>; <a href="#bib6">D'Agostino et al., 2016</a>) or the LSM700 confocal system (Carl Zeiss) (<a href="#bib2">D'Agostino et al., 2014</a>; <a href="#bib3">D'Agostino et al., 2013</a>), respectively. The protocol described here is applicable for any fluorescent molecule excitable at 488 nm (such as a GFP, FITC, Alexa Fluor 488, etc.), as well as for other fluorescent molecules excitable at any other different wavelengths.</p>
<ol start="9">
<li>Live-imaging acquisition procedure:<br> This section of the protocol describes the visualization of the exogenous lysosomal marker Lamp1-mGFP protein. For the live-based visualization, cells must be grown in multi-well with glass-bottom and the microscope must be equipped with a dedicated chamber necessary to maintain temperature and CO<sub>2</sub> at 37°C and 5%, respectively.
<ol type="a">
<li>Switch on the microscope and set up parameters to the thermostatic chamber to let it reach the selected temperature and CO<sub>2</sub>.</li>
<li>All samples are observed by using an oil immersion objective (63× 1.4 NA).</li>
<li>The channel is set to 488-nm laser by using the Ultraview VoX spinning disk software.</li>
<li>Adjust the laser power and time exposure. For this study, the variables were adjusted as follows: laser power at 50% and time exposure between 200 and 500 ms.</li>
<li>Researchers should optimize the settings for optimal image intensity.</li>
<li>After capturing, images can be processed using the Ultraview VoX spinning disk software to adjust their intensity.</li>
<li>Capture images from at least 20 different fields for the quantification of lysosomal diameter.</li>
<li>Acquired images are saved as .Tiff, .JPG or .PNG formats and can be visualized for quantitative analysis by using the ImageJ Fiji software.</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Acquisition procedure from fixed cells: This section describes the visualization of stained cells to analyze the endogenous lysosomal marker Lamp1 protein. All images are acquired by confocal microscopy. To use LSM700 software, researchers need to refer to the manufacturer’s manual for the Zeiss LSM700 instrument.
<ol type="a">
<li>All samples are observed using a confocal microscope equipped with a 63<b>×</b> or 40<b>×</b> 1.4 NA oil immersion objective. The observation can be made even with a non-confocal microscope.
<div class="critical">
<span class="critical-title">Critical:</span> Do not use non-oil objectives. The oil is necessary to improve the resolving power of the microscope which is crucial to get high-resolution images that allow the best estimation of the dimension from the smallest to the largest lysosomal structures.</div>
</li>
<li>The channel is set to 488-nm laser by using the LSM700 Zeiss software.</li>
<li>Adjust the pinhole, laser power, PMT gain, and offset. For this study, the variables were adjusted as follows: pinhole at 0,42 A.U., laser power at 2.0, PMT gain between 500 and 700, and the offset at −20.</li>
<li>Researchers should optimize the settings for optimal image intensity varying and keeping laser power or PMT gain always under the maximum before bleaching. Anyway, the instructions we have provided on the Lamp1-mGFP expressing cells, the antibody against Lamp1, and the fixation procedures guarantee the optimal outcome which can be easily reached also with other acquisition systems including the non-confocal ones.</li>
<li>After capturing, images can be processed using LSM700 Zeiss software to adjust their intensity.</li>
<li>Capture images from at least 20 different fields of three independent experiments. For a good statistically significant quantification, one need to analyze at least 50 lysosomal diameters for 20 cells/experiment for a total of 3000 lysosomal diameters.</li>
<li>Acquired images are saved as “.lsm5” format and can be opened using LSM700 or ImageJ Fiji software. Images can then be exported to the .Tiff, .JPG, .PNG, and many others for the post-processing analysis.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The analysis of lysosomal dimension is important to understand how cells respond to the substrate accumulation into the lysosomal compartment. In this regard, this protocol provides a useful method for monitoring lysosomal dimension in either living cells, or upon fixation. Amino acid analogs are characterized by the presence of a methyl ester group, which renders them able to diffuse through any membrane-enclosed compartment, without causing any osmotic effect due to their accumulation. However, the lysosomal compartment possesses a hydrolase that removes the methyl ester group causing the selective accumulation of the amino acid analog within the lysosomal lumen (<a href="#bib8">Reeves, 1979</a>). As such, the aa-analog accumulation causes an osmotic effect, which ultimately triggers lysosome expansion via a membrane fusion-dependent process (<a href="#bib7">Desfougeres et al., 2016</a>; <a href="#bib10">Scerra et al., 2021</a>). Consistent with this, we observed that after 2 h of amino acid analogs accumulation, lysosomes expand appearing like large and perfect spheres (<a href="#fig1">Figure 1</a>). The usage of a transmembrane protein to label the limited membrane of the compartment is instrumental to better define the boundary of a single lysosomal structure and to allow a more accurate measurement of its diameter.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1131-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Morphological comparison between enlarged lysosomes observed in living condition or after different fixation procedures</div>
<p>HeLa cells, stably expressing the lysosomal marker Lamp1-mGFP, were treated or not (mock cells were treated with only the buffer used for the preparation of the stock solution of the amino acids analogs) for 2 h with amino acid analogs (10 mM LEU-ME) and differently handled (live versus methanol or glutaraldehyde/paraformaldehyde fixation), before being analyzed by confocal microscopy.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h</div>
<p>All images exported in JPEG, Tiff, PNG, and .lsm5 or .lsm7 format can be directly opened in Fiji ImageJ software for the consequent quantitative analysis.</p>
<ol>
<li>Open the image file in Fiji program.</li>
<li>Click on “straight” in the tools bar.</li>
<li>Draw a line corresponding to the diameter of a selected enlarged lysosome by using the Lamp1 fluorescent signal as a reference of the limiting membranes of each selected structure (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1131-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Measurement of lysosomal diameter</div>
<p>Living HeLa cells stably expressing Lamp1-mGFP were grown in 12-multiwell plate with glass-bottom, treated or not with LEU-ME 10 mM for 2 h, and imaged by using a Ultraview VoX spinning disk microscope. Some representative lysosomal diameters are indicated by magenta lines and their lengths have been estimated by ImageJ Fiji software and shown in μm scale (A). The histogram shows the average length of lysosomal diameters from 3000 measurements from 20 cells for each of the three independent experiments (B). Statistical analysis was performed by One-way ANOVA. Scale bar: 20 μm.</p>
</figcaption>
</figure>
</li>
<li>Go to “analyze” and click on “measure”.</li>
<li>A table in a new window will appear. In the table are reported different measurements referring to the lines dawn on the picture including area, mean fluorescence intensity, and length (this number corresponds to the number of pixels contained in the drawn line).</li>
<li>For the statistical significance, we measure the diameter of 50 lysosomes from 20 cells for each of the three experiments performed for a total of 3000 lysosomal diameters.</li>
<li>Before proceeding with the estimation of the real length expressed in μm, repeat the same measurement on the scale bar (for this protocol a 20 μm scale bar was used).</li>
<li>To convert the length of each diameter, expressed in number of pixels, into μm scale, the following formula was used: (n pixels diameter/n pixels scale bar) <b>×</b> 20 μm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Since the smaller lysosomal size of the mock-treated cells, we suggest performing the measurement of the lysosomal diameters on the zoomed images already opened in Fiji software. Indeed, zoomed images conserve the pixel dimension of the original images and it allows better visualization of the florescent signal extension necessary to correctly draw the diameter line.</div>
<ol start="9">
<li>Once converted, mean values were plotted on the histogram (<a href="#fig2">Figure 2</a>B). Standard deviations were determined, and significance assessed by using One-way ANOVA .</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Amino acid analogs accumulate within lysosomes because of intra-luminal cleavage of methyl ester groups which render them membrane permeable, thereby limiting their export from the lysosomes back to the cytoplasm by free diffusion. In this protocol, we analyzed lysosomal size after 2 h of treatment with LEU-ME. This timing was sufficient to promote the lysosomal expansion whose enlarged size was stable up to 3 h. However, from this time point forward, lysosomes start to rescue their initial size. Indeed, once accumulated, cleaved amino acid analogs can be exported by lysosomal transporters into the cytosol, thereby dissipating the osmotic effect and the enlarged size of the organelle. Thus, this protocol is applicable only for a short time treatment whereas, for a longer time, a microfluidic chamber needs to be employed to constantly replenish the extracellular solution and keep constant the high concentration of the amino acid analogs into the lumen of lysosomes.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The fixation procedure affects the morphology of the enlarged lysosomes (<a href="#fig3">Figure 3</a>. Please compare the FA panel on the right respect to the Met-OH and GA/PFA panels) (referred to the sections from 5 to 7).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1131-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Comparison between good versus bad outcome of the fixation procedure on the lysosomal morphology</div>
<p>HeLa cells were grown on coverslips and fixed with different chemical reagents indicated in each panel and lysosomes were labeled by using a specific antibody against the lysosomal protein marker Lamp1. Met-OH: methanol; PFA: para-formaldehyde; FA: formaldehyde. Scale bar: 20 μm.</p>
</figcaption>
</figure>
<h3 id="sec6.2">Potential solution</h3>
<p>Enlarged lysosomes are highly sensitive to the fixation procedure and formaldehyde solution (3,7% formaldehyde dissolved in 1<b>×</b> PBS) determines a severe alteration of the lysosomal morphology (<a href="#fig3">Figure 3</a>, right panel). Such a morphological alteration is evident either in Lamp1-mGFP stably-expressing cells or in cells expressing the endogenous levels of Lamp1 protein and stained by using a specific antibody. We therefore strongly recommend avoiding using formaldehyde and to opt (also in consideration of the eventual combination of antibodies to be used along with the anti-Lamp1) for one of two alternative procedures we have established, which have proved successful in preserving at its best the lysosomal morphology.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Problem to take coverslips from wells (referred to the section 8 g).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>We recommend using a 10 mm or 13 mm coverslip for 12 well plates. 10 mm coverslip can be used also for 24 but not 48 well plates. As a rule, we suggest using coverslips with an area about half that of one of the corresponding wells.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Before putting coverslips on the antibody-containing drops, by distraction, one could forget on which side of the coverslip the cells are (referred to the section 8 g).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Keeping the coverslip with forceps, gently scrape one time a side of the coverslip by using a tip. The appearance of a trace indicates the side of the coverslip where the cells were seeded.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>After the incubation with the primary antibody, the drop was dry, and the coverslip remains attached to the parafilm (referred to the section 8 h).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This problem should not arise if the lab is equipped with an automatic system that guarantees constant control of the temperature over the entire year. However, many labs do not possess these systems and the room temperature could vary depending on the seasons. For this reason, during summer, we suggest improving the volume of the drops used. Alternatively, we suggest building a humid chamber that prevents drops to dry and allows even longer incubation times. Moreover, we suggest adding 100–150 μL of PBS 1<b>×</b> to let the coverslip float on the drop. This will render the handling of the coverslip much easier (the last suggestion is shown in the figure).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Once the mounting medium was dry, many bubbles appear, and the microscopy analysis is largely impaired (referred to the section 8 q).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Before letting the mounting medium becomes dry, we recommend pushing with a tip on the coverslips while aspirating the mounting excess. In this way, all the bubbles should be removed.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Alcantara-Ortigoza, M.A., Gonzalez-del Angel, A., Barrientos-Rios, R., Cupples, C., Garrido-Garcia, L.M., de Leon-Bojorge, B., and Alva-Chaire Adel, C. (2010). Screening of late-onset Pompe disease in a sample of Mexican patients with myopathies of unknown etiology: identification of a novel mutation in the acid alpha-glucosidase gene. J. Child Neurol. <i>25</i>, 1034-1037. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref1">View at publisher</a></p>
<p id="bib2">D'Agostino, M., Crespi, A., Polishchuk, E., Generoso, S., Martire, G., Colombo, S.F., and Bonatti, S. (2014). ER reorganization is remarkably induced in COS-7 cells accumulating transmembrane protein receptors not competent for export from the endoplasmic reticulum. J. Membr. Biol. <i>247</i>, 1149-1159. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref2">View at publisher</a></p>
<p id="bib3">D'Agostino, M., Lemma, V., Chesi, G., Stornaiuolo, M., Cannata Serio, M., D'Ambrosio, C., Scaloni, A., Polishchuk, R., and Bonatti, S. (2013). The cytosolic chaperone alpha-crystallin B rescues folding and compartmentalization of misfolded multispan transmembrane proteins. J. Cell Sci. <i>126</i>, 4160-4172. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref3">View at publisher</a></p>
<p id="bib4">D'Agostino, M., Risselada, H.J., Endter, L.J., Comte-Miserez, V., and Mayer, A. (2018). SNARE-mediated membrane fusion arrests at pore expansion to regulate the volume of an organelle. EMBO J. <i>37</i>, e99193. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref4">View at publisher</a></p>
<p id="bib5">D'Agostino, M., Risselada, H.J., Lurick, A., Ungermann, C., and Mayer, A. (2017). A tethering complex drives the terminal stage of SNARE-dependent membrane fusion. Nature <i>551</i>, 634-638. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref5">View at publisher</a></p>
<p id="bib6">D'Agostino, M., Risselada, H.J., and Mayer, A. (2016). Steric hindrance of SNARE transmembrane domain organization impairs the hemifusion-to-fusion transition. EMBO Rep. <i>17</i>, 1590-1608. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref6">View at publisher</a></p>
<p id="bib7">Desfougeres, Y., Neumann, H., and Mayer, A. (2016). Organelle size control - increasing vacuole content activates SNAREs to augment organelle volume through homotypic fusion. J. Cell Sci. <i>129</i>, 2817-2828. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref7">View at publisher</a></p>
<p id="bib8">Reeves, J.P. (1979). Accumulation of amino acids by lysosomes incubated with amino acid methyl esters. J. Biol. Chem. <i>254</i>, 8914-8921. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref8">View at publisher</a></p>
<p id="bib9">Sbano, L., Bonora, M., Marchi, S., Baldassari, F., Medina, D.L., Ballabio, A., Giorgi, C., and Pinton, P. (2017). TFEB-mediated increase in peripheral lysosomes regulates store-operated calcium entry. Sci. Rep. <i>7</i>, 40797. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref9">View at publisher</a></p>
<p id="bib10">Scerra, G., De Pasquale, V., Pavone, L.M., Caporaso, M.G., Mayer, A., Renna, M., and D'Agostino, M. (2021). Early onset effects of single substrate accumulation recapitulate major features of LSD in patient-derived lysosomes. iScience <i>24</i>, 102707. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00622-5/sref10">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Massimo D’Agostino (<a href="mailto:massimo.dagostino@unina.it">massimo.dagostino@unina.it</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was partially funded by the Italian Minister for Research and University (PRIN 20177XJCHX) to M.R., and departmental funding to M.D.A.</p>
<h3>Author contributions</h3>
<p>Conceptualization, M.D.A.; investigation, G.S. and M.G.C.; writing, M.D.A., G.S., and M.R.; supervision, M.D.A.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>