<article>
<section>
<h1 id="header">Protocol for isolation of spermatids from mouse testes</h1>
<p><time datetime="2021-01-15">Published: January 15, 2021</time></p>
<p>Chang Rok Kim,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> Taichi Noda,<sup><a href="#aff2">2</a></sup> Yuki Okada,<sup><a href="#aff3">3</a></sup> Masahito Ikawa,<sup><a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#cor2">**</a></sup> and Sung Hee Baek<sup><a href="#aff1">1</a>,<a href="#fn2">6</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Research Center for Epigenetic Code and Diseases, Department of Biological Sciences, Seoul National University, Seoul 08826, South Korea</p>
<p id="aff2"><sup>2</sup>Research Institute for Microbial Diseases, Osaka University, Osaka 565-0871, Japan</p>
<p id="aff3"><sup>3</sup>Laboratory of Pathology and Development, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo 113-0032, Japan</p>
<p id="aff4"><sup>4</sup>The Institute of Medical Science, The University of Tokyo, Tokyo 108-8639, Japan</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ryung1113@snu.ac.kr">ryung1113@snu.ac.kr</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:ikawa@biken.osaka-u.ac.jp">ikawa@biken.osaka-u.ac.jp</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:sbaek@snu.ac.kr">sbaek@snu.ac.kr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100254">10.1016/j.xpro.2020.100254</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Post-meiotic spermatids become spermatozoa through developmental stages during spermiogenesis. Isolation of spermatid fractions is required to examine the change of protein expression during spermiogenesis. Here, we present a simple method to isolate spermatid fractions from mouse testes using unit gravity sedimentation in a BSA density gradient. Isolation of spermatid fractions can be used to analyze changes of transcript or protein during spermiogenesis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib4">Kim et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol for isolation of post-meiotic spermatids from mouse testes</li>
<li>Preparation of single-cell suspension through serial digestion of testes</li>
<li>Analysis of cell fractions from unit gravity sedimentation in BSA density gradient</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/362-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Spermatogenesis includes the dramatic alternation of morphology of spermatogenic cells, leading to difference in size and density of each developmental stages. Several methods such as STA-PUT and elutriation have been developed to separate different stages of spermatogenic cells by their size and density in the BSA density gradient (<a href="#bib1">Bellvé et al., 1977</a>; <a href="#bib2">Bryant et al., 2013</a>; <a href="#bib3">Chang et al., 2011</a>; <a href="#bib8">Zhang et al., 2017</a>). Also, FACS has been used for the same goal using antibody against surface antigen and difference in ploidy (<a href="#bib5">Lima et al., 2017</a>; <a href="#bib6">Okada et al., 2007</a>). However, previous methods are difficult to perform due to complicated steps of experiment. Here, we modify classical STA-PUT and provide a simpler protocol.</p>
<p>Before the preparation of testis single-cell suspension from mice, prepare the BSA density gradient.</p>
<h3 id="sec1.1">BSA density gradient</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~1 h</div>
<ol>
<li>Prepare 4% BSA, 2% BSA, and 0.5% BSA solution in DMEM: F12 media</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To check the stacking of solution with different concentrations of BSA, we used DMEM: F12 media with/without phenol red. 4% BSA and 0.5% BSA were solved in red DMEM: F12 media and 2% BSA was solved in colorless DMEM: F12 media. As the color of BSA was yellow, the color of the BSA solutions changed according to the concentration of BSA: 4% BSA (orange), 2% BSA (yellow), and 0.5% BSA (red). We used 100 mL disposable syringe as sedimentation column.</div>
<ol start="2">
<li>Pour 40 mL of 4% BSA into sedimentation column slowly using serological pipette.</li>
<li>Pour 40 mL of 2% BSA into the tube above the sedimentation column. Flow 2% BSA to the wall of column at 2 mL/min to stack over 4% BSA using connecting tube (<a href="#fig1">Figures 1</a>A and 1B).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/362-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic representation of BSA density gradient sedimentation</div>
<p>(A) Schematic representation of unit gravity sedimentation.</p>
<p>(B) The image of apparatus for unit gradient sedimentation.</p>
<p>(C) The image of incorporation of 0.5% BSA over 2% BSA using a serological pipette.</p>
<p>(D) View of BSA density gradient. 4% BSA (bottom, orange), 2% BSA (middle, yellow), and 0.5% BSA (top, red) are stacked in a sedimentation column.</p>
</figcaption>
</figure>
</li>
<li>Flow 3 mL of 0.5% BSA to the wall of column using serological pipette over 2% BSA (<a href="#fig1">Figure 1</a>C).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To prevent the incorporation of foreign matter into BSA solution, the top of column is covered by clear wrap. Bubble should not be incorporated into BSA solution. Do not touch the sedimentation column to maintain BSA density gradient.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM: F12 (with phenol red)</td>
<td>Welgene</td>
<td>LM002-04</td>
</tr>
<tr>
<td>DMEM: F12 (without phenol red)</td>
<td>Welgene</td>
<td>LM002-05</td>
</tr>
<tr>
<td>Collagenase A</td>
<td>Sigma-Aldrich</td>
<td>10103578001</td>
</tr>
<tr>
<td>DNase I, grade II, from bovine pancreas</td>
<td>Sigma-Aldrich</td>
<td>10104159001</td>
</tr>
<tr>
<td>Trypsin (1×, 0.25%)</td>
<td>Welgene</td>
<td>LS015-03</td>
</tr>
<tr>
<td>Percoll</td>
<td>Sigma-Aldrich</td>
<td>P1644</td>
</tr>
<tr>
<td>FBS</td>
<td>HyClone</td>
<td>SH30084.03</td>
</tr>
<tr>
<td>HBSS (10×) (with calcium/magnesium)</td>
<td>Welgene</td>
<td>LB203-04</td>
</tr>
<tr>
<td>HBSS (1×) (with calcium/magnesium)</td>
<td>Welgene</td>
<td>LB003-02</td>
</tr>
<tr>
<td>DPBS (1×)</td>
<td>Welgene</td>
<td>LB001-02</td>
</tr>
<tr>
<td>RBC lysis buffer (10×)</td>
<td>Invitrogen</td>
<td>00-4300-54</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>Bovogen</td>
<td>BSA100</td>
</tr>
<tr>
<td>TOPreal qPCR 2× PreMIX</td>
<td>Enzynomics</td>
<td>RT500</td>
</tr>
<tr>
<td>Ifran solution (isoflurane 100 mL)</td>
<td>Hana Pharm</td>
<td>657801261</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Dead Cell Removal Kit (<a href="https://www.miltenyibiotec.com/US-en/products/dead-cell-removal-kit.html">https://www.miltenyibiotec.com/US-en/products/dead-cell-removal-kit.html</a>)</td>
<td>Miltenyi Biotec</td>
<td>130-090-101</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse : C57BL/6J male (10–12 weeks)</td>
<td>KOATECH</td>
<td>B601-110</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>mouse <i>Amh</i> forward:5′-CCCGCTATTTGGTGCTAACC-3′</td>
<td>This paper</td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Amh</i> reverse:5′-CGTGAAACAGCGGGAATCA-3′</td>
<td>This paper</td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Sox9</i> forward:5′-CGTCAACGGCTCCAGCA-3′</td>
<td>This paper</td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Sox9</i> reverse:5′-TGCGCCCACACCATGA-3′</td>
<td>This paper</td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Cyp11a1</i> forward:5′-CCAGTGTCCCCATGCTCAAC-3′</td>
<td><a href="#bib3">Chang et al., 2011</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Cyp11a1</i> reverse:5′-TGCATGGTCCTTCCAGGTCT-3′</td>
<td><a href="#bib3">Chang et al., 2011</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Plzf</i> forward:5′-GAGCAGTGCAGCGTGTGT-3′</td>
<td><a href="#bib7">Weinreich et al., 2010</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Plzf</i> reverse:5′-AACCGTTTTCCGCAGAGTT-3′</td>
<td><a href="#bib7">Weinreich et al., 2010</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Sycp2</i> forward:5′-GACACTGAAACCGAATGTGGA-3′</td>
<td>This paper</td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Sycp2</i> reverse:5′-TGTGGGTCTTGGTTGTCCTTT-3′</td>
<td>This paper</td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Acrv1</i> forward:5′-TCAGCAACTTTCAAGCGAGTAT-3′</td>
<td><a href="#bib3">Chang et al., 2011</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Acrv1</i> reverse:5′-CTCCTGAAGAGTGCTCACCTG-3′</td>
<td><a href="#bib3">Chang et al., 2011</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Spaca1</i> forward:5′-CCCGAGTCCGAAACCACAG-3′</td>
<td>This paper</td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Spaca1</i> reverse:5′-ACAACACACTTGGATTCACCTC-3′</td>
<td>This paper</td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Tnp2</i> forward:5′-GAAGGGAAAGTGAGCAAGAGAA-3′</td>
<td><a href="#bib6">Okada et al., 2007</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Tnp2</i> forward:5′-GCATAGAAATTGCTGCAGTGAC-3′</td>
<td><a href="#bib6">Okada et al., 2007</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Prm2</i> forward:5′-GCTGCTCTCGTAAGAGGCTACA-3′</td>
<td><a href="#bib6">Okada et al., 2007</a></td>
<td>n/a</td>
</tr>
<tr>
<td>mouse <i>Prm2</i> reverse:5′-AGTGATGGTGCCTCCTACATTT -3′</td>
<td><a href="#bib6">Okada et al., 2007</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Cell strainer, 70 μm</td>
<td>SPL</td>
<td>93070</td>
</tr>
<tr>
<td>Cell strainer, 40 μm</td>
<td>SPL</td>
<td>93040</td>
</tr>
<tr>
<td>Disposable syringe 100 mL</td>
<td>Zibo Gufeng Medical Products</td>
<td>n/a</td>
</tr>
<tr>
<td>Serological pipette 10 mL</td>
<td>SARSTEDT</td>
<td>86.1254.001</td>
</tr>
<tr>
<td>Iris scissors</td>
<td>HEBU</td>
<td>HB7454</td>
</tr>
<tr>
<td>Operating scissors</td>
<td>HEBU</td>
<td>HB1704</td>
</tr>
<tr>
<td>Iris forceps</td>
<td>HEBU</td>
<td>HB425</td>
</tr>
<tr>
<td>Centrifuge 5415 R</td>
<td>Eppendorf</td>
<td>n/a</td>
</tr>
<tr>
<td>Centrifuge 5810 R</td>
<td>Eppendorf</td>
<td>n/a</td>
</tr>
<tr>
<td>Countess II</td>
<td>Life Technologies</td>
<td>n/a</td>
</tr>
<tr>
<td>Confocal laser scanning microscope LSM 700</td>
<td>Carl Zeiss</td>
<td>n/a</td>
</tr>
<tr>
<td>CFX384 real-time system</td>
<td>Bio-Rad</td>
<td>n/a</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Dissolved collagenase A (50 mg/mL) and DNase I (20 mg/mL) into HBSS (1×) are stored at 4°C before experiments and −80°C after experiments. Stock solution of collagenase A and DNase I should be aliquoted to minimize freeze/thaw cycles. All solution/medium should be made right before experiments. DMEM: F12 with/without phenol red is available for digestion media.</p>
<table id="undtbl1">
<caption>Digestion media I</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM: F12</td>
<td>n/a</td>
<td>9.8 mL</td>
</tr>
<tr>
<td>Collagenase A</td>
<td>0.5 mg/mL</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>DNase I</td>
<td>0.2 mg/mL</td>
<td>0.1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>5% Percoll solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HBSS (10×)</td>
<td>n/a</td>
<td>4 mL</td>
</tr>
<tr>
<td>Percoll</td>
<td>5%</td>
<td>2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>34 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>40 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>Digestion media II</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM: F12</td>
<td>n/a</td>
<td>7.9 mL</td>
</tr>
<tr>
<td>Trypsin</td>
<td>0.05%</td>
<td>2 mL</td>
</tr>
<tr>
<td>DNase I</td>
<td>0.2 mg/mL</td>
<td>0.1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Second digestion can be performed without DNase I. After digestion with Trypsin, add DNase I to the digested cells and shake the cells by hand for 30 s.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Sacrifice mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~30 min</div>
<p>The steps include preparation of testes from mice.</p>
<ol>
<li>Sacrifice mice using isoflurane inhalation in the closed chamber and open the lower abdomen using surgical scissors (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/362-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Preparation of mouse testes</div>
<p>(A) View of mouse with opening at the lower abdomen on the dissection tray. Arrows indicate epididymal fat.</p>
<p>(B) Pulling epididymal fat can expose testes and epididymis. Arrow indicates testis.</p>
<p>(C) Removal of epididymal fat and epididymis around testes.</p>
<p>(D) Removal of tunica albuginea. Arrow indicates tunica albuginea of testis.</p>
</figcaption>
</figure>
</li>
<li>Expose testes by pulling epididymal fat and remove fat and epididymis around testes (<a href="#fig2">Figures 2</a>B and 2C).</li>
<li>Transfer testes to cold DPBS, and wash blood. Then, remove the tunica albuginea of testes (<a href="#fig2">Figure 2</a>D).</li>
<li>Transfer the tunica albuginea-removed testes to digestion media I (10 mL).</li>
</ol>
<h3 id="sec3.2">Preparation of single-cell suspension of testes (testicular cells)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~100 min</div>
<p>The steps exhibit dissociation of testes using enzymes to get single-cell suspension of testes (testicular cells).</p>
<ol start="5">
<li>Primary digestion is performed at 37°C (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/362-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Digestion of mouse testes</div>
<p>(A) The left image shows decapitulated testes in digestion media I. In 15 min, seminiferous tubules of testes are dissociated (right).</p>
<p>(B) Digested product poured over 5% Percoll solution is separated by gravity (left). In 20 min, high density seminiferous tubules sink to the bottom (right).</p>
<p>(C) Secondary digestion results in single-cell suspension from seminiferous tubules.</p>
</figcaption>
</figure>
<ol type="a">
<li>Following incubation for 5 min, dissociate testes thorough pipetting.</li>
<li>Incubate for 15 min further. Then, gently pipette until seminiferous tubules are dissociated.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Seminiferous tubules should be maintained intact. Too much pipetting can result in fragmentation of tubules.</div>
<ol start="6">
<li>Load the digested product over the top of 5% Percoll solution (40 mL) at 20°C–25°C. In 20 min, while the majority of interstitial cells are distributed at the top of the solution, highly dense seminiferous tubules sink at the bottom (<a href="#fig3">Figure 3</a>B).</li>
<li>Remove 35 mL of supernatant and transfer 5 mL of bottom including seminiferous tubules to digestion media II (10 mL).</li>
<li>Secondary digestion is performed for 20 min at 37°C. Carefully pipette until seminiferous tubules disappear (<a href="#fig3">Figure 3</a>C).</li>
<li>Add FBS 3 mL to neutralize trypsin.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Digestion for a long time can cause cell aggregation (see <a href="#troubleshooting">Troubleshooting 1</a>).</div>
<ol start="10">
<li>The digested product is filtered through a 70 μm cell strainer and 40 μm cell strainer sequentially.</li>
<li>Centrifuge filtered cell suspension with 500 × <i>g</i> at 4°C for 10 min.</li>
<li>Pipette cell pellet with HBSS (1×) 1 mL and centrifuge it with 500 × <i>g</i> at 4°C for 2 min. Pipette cell pellet with RBC lysis buffer (1×, dilution with distilled water) 1 mL and incubate at 4°C for 1 min. Then, centrifuge it with 500 × <i>g</i> at 4°C for 2 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the time for this step. Too long time for RBC lysis can damage testicular cells.</div>
<ol start="13">
<li>Pipette cells with HBSS (1×) 1 mL and then centrifuge with 500 × <i>g</i> at 4°C for 2 min.</li>
<li>Pipette cells with 1 mL of 0.5% BSA.</li>
</ol>
<h3 id="sec3.3">Unit gravity sedimentation in BSA density gradient</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ~4 h</div>
<p>Through these steps, 20 fractions of testicular cells are collected.</p>
<ol start="15">
<li>Load testicular cells in 1 mL of 0.5% BSA to the wall of column slowly over 0.5% BSA using serological pipettes (<a href="#fig1">Figure 1</a>D).</li>
<li>Stand for 150 min at 20°C–25°C.</li>
<li>Open the valve at the bottom of column so that the flow rate is 2 mL/min.</li>
<li>Collect fractions per 4 mL.</li>
<li>Centrifuge all fractions with 1,000 × <i>g</i> at 4°C for 10 min.</li>
<li>Discard supernatant and pipette cell pellets with HBSS (1×) 1 mL.</li>
<li>Observe cells in each fraction with a microscope.</li>
<li>Following centrifugation with 1,000 × <i>g</i> at 4°C for 3 min, store cell pellet at −80°C.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Following sedimentation, we analyzed cell number, viability, and composition in each fraction, and performed qRT-PCR to measure the expression of marker genes at spermatogenic stages.</p>
<h3 id="sec4.1">Cell number, viability, and composition</h3>
<p>We collected 3 × 10<sup>7</sup> cells from the testes of 2 mice. Following BSA density sedimentation, early fractions (from 2 to 7) showed relatively low cell number and viability compared to later fractions (<a href="#fig4">Figure 4</a>A). Late fractions (from 16 to 20) exhibited relatively high cell number and viability. However, fractions 18–20 contain many dead cells. In fractions 19 and 20, residual bodies, released from spermatids through spermiogenesis, were observed and seemed to contribute to low cell viability.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/362-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Analysis of cells from each fraction</div>
<p>(A) The cell number and viability of fractions.</p>
<p>(B) Representative confocal images of cells from fractions with DAPI (blue). Scale bar, 10 μm.</p>
<p>(C) The percentage of cell types from fractions 14–20.</p>
</figcaption>
</figure>
<p>When we counted the percentage of spermatids in fractions (<a href="#bib8">Zhang et al., 2017</a>), round spermatids were the most enriched in fraction 16 and elongating/condensing spermatids (ES/CS) were dominantly enriched in fractions 18 and 19 (<a href="#fig4">Figures 4</a>B and 4C). Fractions earlier than fraction 16 contained other cell types including spermatocytes (SC) and spermatogonia (SG) more.</p>
<h3 id="sec4.2">Expression of marker genes at spermatogenic stages</h3>
<p>We used primers to probe Amh, Sox9, Cyp11a1, Zbtb16, Sycp2, Spaca1, Acrv1, Tnp2, and Prm2 for qRT-PCR. The expression of Amh and Sox9, marker genes of Sertoli cells, and Cyp11a1, a marker gene of Leydig cells, were high in fractions 1–6 and decreased after fractions 7 and 8, indicating that Sertoli cells and Leydig cells were enriched in fractions 1–6 (<a href="#fig5">Figure 5</a>A). Although Leydig cells were removed at steps 6 and 7, some still remained. The expression of Zbtb16, a marker of spermatogonia, was high in fractions 13–15, indicating that spermatogonia were enriched in these fractions. Sycp2, a marker of spermatocytes, showed high mRNA levels in fractions 7–12. Spaca1 and Acrv1, marker genes of round spermatids, were expressed similarly throughout all fractions beside fractions 17–20 (<a href="#fig5">Figure 5</a>B). Tnp2 and Prm2, marker genes of elongating and condensing spermatids, were detected in high levels in fractions 18 and 19, representing that elongating and condensing spermatids were enriched in fractions 18 and 19 (<a href="#fig5">Figure 5</a>C). However, as the expression of Amh and Sox9 increased in fractions 19 and 20, we excluded these fractions from the collection of elongating and condensing spermatids. Together, we selected fraction 16 for round spermatids and fraction 18 for elongating and condensing spermatids.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/362-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. The expression of marker genes at the spermatogenic stage from fractions</div>
<p>(A) The relative mRNA levels of marker genes of Sertoli cells (Amh and Sox9) and Leydig cells (Cyp11a1) from fractions 1–20.</p>
<p>(B) The relative mRNA levels of marker genes of spermatogonia (Zbtb16), spermatocytes (Sycp2), and early spermatids (Spaca1 and Acrv1) from fractions 1–20.</p>
<p>(C) The relative mRNA levels of marker genes of late spermatids (Tnp2 and Prm2) from fractions 1–20.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol has issues on the purity of the cell population. Each fraction contains a mixture of several cell populations. However, we could select spermatid fractions based on microscopic images and marker gene expression: fraction 16 for round / elongating spermatids (RS/ES) and fraction 18 for elongating/condensing spermatids (ES/CS). Dividing fraction more to increase the volume of the sedimentation column may improve the purity of the cell population. As we used a 100 mL column for sedimentation column with testicular cells from 2 mice, a bigger column can be also used to separate more testicular cells (see <a href="#troubleshooting">Troubleshooting 2</a>).</p>
<p>Fractions 17–20 containing elongating and condensing spermatids show low cell viability. To increase viability, we recommend using a kit to remove dead cells (see <a href="#troubleshooting">Troubleshooting 3</a>). Moreover, since fractions 19 and 20 exhibit increased expression of Amh and Sox9, marker genes of Sertoli cells, we do not recommend using fractions 19 and 20.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Inefficient dissociation of testes or the aggregation of cells (steps 4–9).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>DNase I is critical to digest testes. Low concentration or poor quality of DNase I does not work for the digestion of testes. And, extended exposure of trypsin can cause the aggregation of cells. Keep the time for digestion step.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Failure of separation of spermatogenic cells (<a href="#before-you-begin">Before you begin</a>, steps 14 and 15).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>BSA density gradient is important for the separation of spermatogenic cells. When building the BSA density gradient in the sedimentation column, fast loading of BSA media can lead to failure in building the BSA density gradient. We recommend less than 2 mL/min as loading velocity of BSA media. Physical stress can disturb the BSA density gradient, leading to inefficient separation of spermatogenic cells. And, loading too many testicular cells can decrease the efficiency of separation. To separate more testicular cells, the bigger volume of the sedimentation column/chamber is required.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Low viability of spermatids (step 21).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Above stated, cell viability of fractions 17–20 that include elongating and condensing spermatids is low due to dead cells and residual body. To increase cell viability, we tested a kit to remove dead cells from fraction 18 containing mainly elongating/condensing spermatids. Using the kit which remove dead cells through magnetic beads and magnetic-activated cell sorting (MACS) improved the viability of cells from fraction 18 (<a href="#fig6">Figure 6</a>A). Moreover, sedimentation in the cold room at 4°C showed higher viability in late fractions than sedimentation at 25°C (<a href="#fig6">Figure 6</a>B). Experiment at low temperature may be helpful to collect better quality of spermatids.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/362-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Improvement of the spermatids viability in late fractions</div>
<p>(A) The improvement of the spermatids viability by dead cell removal.</p>
<p>(B) The difference of the cell viability following unit gravity sedimentation performed at 25°C and 4°C.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bellvé, A.R., Cavicchia, J.C., Millette, C.F., O'Brien, D.A., Bhatnagar, Y.M., and Dym, M. (1977). Spermatogenic cells of the prepuberal mouse. Isolation and morphological characterization. J. Cell Biol. <i>74</i>, 68-85.</p>
<p id="bib2">Bryant, J.M., Meyer-Ficca, M.L., Dang, V.M., Berger, S.L., and Meyer, R.G. (2013). Separation of spermatogenic cell types using STA-PUT velocity sedimentation. J. Vis. Exp. 50648.</p>
<p id="bib3">Chang, Y.F., Lee-Chang, J.S., Panneerdoss, S., MacLean, J.A., 2nd, and Rao, M.K. (2011). Isolation of Sertoli, Leydig, and spermatogenic cells from the mouse testis. Biotechniques <i>51</i>, 341-342.</p>
<p id="bib4">Kim, C.R., Noda, T., Kim, H., Kim, G., Park, S., Na, Y., Oura, S., Shimada, K., Bang, I., Ahn, J.Y., et al. (2020). PHF7 modulates BRDT stability and histone-to-protamine exchange during spermiogenesis. Cell Rep. <i>32</i>, 107950.</p>
<p id="bib5">Lima, A.C., Jung, M., Rusch, J., Usmani, A., Lopes, A.M., and Conrad, D.F. (2017). A standardized approach for multispecies purification of mammalian male germ cells by mechanical tissue dissociation and flow cytometry. J. Vis. Exp. 55913.</p>
<p id="bib6">Okada, Y., Scott, G., Ray, M.K., Mishina, Y., and Zhang, Y. (2007). Histone demethylase JHDM2A is critical for Tnp1 and Prm1 transcription and spermatogenesis. Nature <i>450</i>, 119-123.</p>
<p id="bib7">Weinreich, M.A., Odumade, O.A., Jameson, S.C., and Hogquist, K.A. (2010). T cells expressing the transcription factor PLZF regulate the development of memory-like CD8+ T cells. Nat. Immunol. <i>11</i>, 709-716.</p>
<p id="bib8">Zhang, Y., Tang, C., Yu, T., Zhang, R., Zheng, H., and Yan, W. (2017). MicroRNAs control mRNA fate by compartmentalization based on 3’ UTR length in male germ cells. Genome Biol. <i>18</i>, 105.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Sung Hee Baek (<a href="mailto:sbaek@snu.ac.kr">sbaek@snu.ac.kr</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the Creative Research Initiatives Program (Research Center for Epigenetics Code and Diseases, NRF-2017R1A3B1023387) and the Science Research Center program (NRF-2016R1A5A1010764) to S.H.B.; the <a href="https://doi.org/10.13039/501100001700">Ministry of Education, Culture, Sports, Science, and Technology</a> (MEXT); and <a href="https://doi.org/10.13039/501100001691">Japan Society for the Promotion of Science</a> (JSPS) KAKENHI grants (JP18K14612 to T.N. and 19H05750 to M.I.).</p>
<h3>Author Contributions</h3>
<p>C.R.K. conducted the experiments. C.R.K., T.N., Y.O., M.I., and S.H.B. designed the experiments and wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>