<article>
<section>
<h1 id="header">A cell-sorting-based protocol for cell cycle small-scale ChIP sequencing</h1>
<p><time datetime="2022-03-18">Published: March 18, 2022</time></p>
<p>Johnathan R. Whetstine<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#cor1">*</a></sup> and Capucine Van Rechem<sup><a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Cancer Signaling and Epigenetics Program, Fox Chase Cancer Center, Philadelphia, PA 19111, USA</p>
<p id="aff2"><sup>2</sup>Cancer Epigenetics Institute, Fox Chase Cancer Center, Philadelphia, PA 19111, USA</p>
<p id="aff3"><sup>3</sup>Stanford Medicine Department of Pathology, Stanford, CA 94305, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:johnathan.whetstine@fccc.edu">johnathan.whetstine@fccc.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:cvrechem@stanford.edu">cvrechem@stanford.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101243">10.1016/j.xpro.2022.101243</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Classic approaches to characterizing cell cycle leverage chemicals or altered nucleotide pools, which could impact chromatin states at specific phases of the cell cycle. Such approaches could induce metabolic alterations and/or DNA damage, which could reshape protein recruitment and histone modifications. In this protocol, we describe ways to fix and sort cells across the cell cycle based on their DNA content. We further detail immunoprecipitation and library preparation, allowing analysis of the epigenome by chromatin immunoprecipitation sequencing (ChIP-seq) for small numbers of cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00123-X/sref1">Van Rechem et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Drugs are not used to arrest cells at different cell cycle phases</li>
<li>This protocol is adapted to small amounts of chromatin for ChIP-seq</li>
<li>This protocol can be adapted to separate additional cell cycle phases</li>
<li>This protocol can be adapted to any cycling cells and other histone modifications</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1498-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Experimental conditions described in this protocol apply to hTERT RPE-1 cells. However, we have also used this protocol successfully in HEK293T cells.</p>
<p>All buffers in the tables (<a href="#sec2.2">buffers</a> section) need to be prepared in advance and stored at the reported temperature.</p>
<h3 id="sec1.1">Optimize sonication</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days to few weeks</div>
<p>This optimization step can take longer for two main reasons: (1) cells growth rate and (2) difficulty to sonicate certain cell types. One might realize that even the longest tested sonication time is not enough and restart over with adjusted sonication time to test.</p>
<div class="note">
<span class="note-title">Note:</span> while we optimize the sonication with unsorted cells, sorted cells can be easier to sonicate and may need less sonication time (∼5 min less). Because of the time and price of sorting (especially if using a facility) we do not optimize with sorted cells, but one could decide to do so.</div>
<ol>
<li>Plate cells in one 15 cm<sup>2</sup> plate in order to obtain 70% confluency at 48 h (∼1 × 10<sup>7</sup> cells per plate).</li>
<li>Add formaldehyde to the medium at a final concentration of 1%, swirl to mix and incubate at 37°C for 13 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> formaldehyde should be freshly opened and is toxic, use gloves and dispose properly.</div>
<ol start="3">
<li>Add 1.25 M glycine pH 2.5 at a final concentration of 0.125 M to stop the crosslink, swirl to mix and incubate for 2 min minimum.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> during the quenching the medium should turn bright yellow.</div>
<div class="note">
<span class="note-title">Note:</span> the efficacy of glycine is improved by low pH.</div>
<ol start="4">
<li>Remove the medium and wash the cells with cold 1× PBS.</li>
<li>Collect the cells with a lifter with 1× PBS and transfer in a conical tube. Centrifuge for 2 min at 100 g at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> use a cell lifter instead of the scraper which damage less the cells.</div>
<ol start="6">
<li>Resuspend the cells in 1 mL cellular lysis buffer freshly complemented with protease inhibitors (cocktail tablets EDTA free). Incubate 5 min on ice and centrifuge for 2 min at 100 g at 4°C.</li>
<li>Discard the supernatant by aspirating or pipetting and resuspend the cells in 300 μL of nuclear lysis buffer freshly complemented with protease inhibitors (cocktail tablets EDTA free). This corresponds to ∼1 × 10<sup>7</sup> cells in total.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the amount of nuclear lysis buffer can be adjusted according to the number of cells.</div>
<ol start="8">
<li>Sonication: we use the QSonica Q800R system with the chiller turned on. Transfer the 300 μL of chromatin in 0.5 mL thin wall PCR tubes. Sonicate at amplitude 70% 15 s ON 45 s OFF for 20 min of <u>total time ON</u>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> we used 0.5 ml thin wall PCR tubes (BRAND 781312) but any equivalent (thin wall) will work.</div>
<div class="note">
<span class="note-title">Note:</span> we found this sonicator optimal for DNA sonication, however equivalent bath sonicators could be used and will need adapting for the sonication time and amplitude.</div>
<div class="note">
<span class="note-title">Note:</span> if you do not know how much sonication your cells will need, sonicate for 10 min, remove few microliters from the tube, add 5 extra min, remove few microliters, and so on up to 40 min of sonication time, and test all these times following the next steps.</div>
<div class="note">
<span class="note-title">Note:</span> do not over sonicate, the size will not decrease further but the chromatin will start to be altered and the ChIP will be less efficient.</div>
<div class="note">
<span class="note-title">Note:</span> the bath should never get hot.</div>
<ol start="9">
<li>Add 3 μL of chromatin and 1 μL of proteinase K to 65 μL of elution buffer, incubate at 65°C for 4–16 h in a benchtop thermomixer shaking at 1,000 rpm.</li>
<li>Add 65 μL of phenol/chloroform/isoamyl alcohol 25:24:1, vortex, centrifuge for 1 min at 20,000 g at 18°C–25°C.</li>
<li>Take 20 μL of the upper phase, mix with appropriate dye and run on a 1% agarose gel. The chromatin should be under 300 bp. If there is a big tail, the chromatin is not sonicated enough (see <a href="#fig1">Figure 1</a>). <a href="#troubleshooting">Troubleshooting 1</a>.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1498-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Example of agarose gel profiles after sonication</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> use a DNA loading dye with only one tracking color that runs at a high molecular to avoid using a dye running at the same size as your expected chromatin as it would hide it.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-H3K4me1, 0.2 μg</td>
<td>Abcam</td>
<td>Cat# ab8895, RRID:AB_306847</td>
</tr>
<tr>
<td>Anti-H3K4me2, 0.1 μg</td>
<td>Abcam</td>
<td>Cat# ab32356, RRID:AB_732924</td>
</tr>
<tr>
<td>Anti-H3K4me3, 0.2 μg</td>
<td>Millipore</td>
<td>Cat# 07-473, RRID:AB_1977252</td>
</tr>
<tr>
<td>Anti-H3K27Ac, 0.2 μg</td>
<td>Active Motif</td>
<td>Cat# 39133, RRID:AB_2561016</td>
</tr>
<tr>
<td>Anti-H3K9Ac, 0.15 μg</td>
<td>Abcam</td>
<td>Cat# ab4441, RRID:AB_2118292</td>
</tr>
<tr>
<td>Anti-H3K27me3, 0.15 μg</td>
<td>Millipore</td>
<td>Cat# 07-449, RRID:AB_310624</td>
</tr>
<tr>
<td>Anti-H3K9me3, 0.2 μg</td>
<td>Abcam</td>
<td>Cat# ab8898, RRID:AB_306848</td>
</tr>
<tr>
<td>Anti-H3K36me3, 0.2 μg</td>
<td>Abcam</td>
<td>Cat# ab9050, RRID:AB_306966</td>
</tr>
<tr>
<td>Anti-H3K9me1, 0.2 μg</td>
<td>Abcam</td>
<td>Cat# ab8896, RRID:AB_732929</td>
</tr>
<tr>
<td>Anti-H3K9me2, 0.2 μg</td>
<td>Abcam</td>
<td>Cat# ab1220, RRID:AB_449854</td>
</tr>
<tr>
<td>Anti-H3K36me1, 0.2 μg</td>
<td>Cell Signaling Technology</td>
<td>Cat# 14111, RRID:AB_2798395</td>
</tr>
<tr>
<td>Anti-H3K36me2, 0.2 μg</td>
<td>Abcam</td>
<td>Cat# ab9049, RRID:AB_1280939</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle’s Medium - High Glucose</td>
<td>Sigma-Aldrich</td>
<td>Cat# D5648</td>
</tr>
<tr>
<td>FBS</td>
<td>GIBCO</td>
<td>Cat# 26140-079</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Life Technologies</td>
<td>Cat# 15140122</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Life Technologies</td>
<td>Cat# 25030-081</td>
</tr>
<tr>
<td>Hoechst 33342</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# H3570</td>
</tr>
<tr>
<td>Trypsin-0.25% EDTA</td>
<td>Life Technologies</td>
<td>Cat# 2520056</td>
</tr>
<tr>
<td>Formaldehyde</td>
<td>Electron Microscopy Sciences</td>
<td>Cat# 15686</td>
</tr>
<tr>
<td>Ribonuclease A</td>
<td>Sigma-Aldrich</td>
<td>Cat# R4875</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Sigma-Aldrich</td>
<td>Cat# P6556</td>
</tr>
<tr>
<td>AMPure XP</td>
<td>Beckman Coulter</td>
<td>Cat# A63881</td>
</tr>
<tr>
<td>Dynabeads Protein A</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10001D</td>
</tr>
<tr>
<td>Dynabeads Protein G</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # 10003D</td>
</tr>
<tr>
<td>Protein A–Agarose Fast Flow</td>
<td>Sigma-Aldrich</td>
<td>Cat # P3476-5ML</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>TruSeq ChIP Library Preparation Kit Set A</td>
<td>Illumina</td>
<td>Cat# IP-202-1012</td>
</tr>
<tr>
<td>TruSeq ChIP Library Preparation Kit Set B</td>
<td>Illumina</td>
<td>Cat# IP-202-1024</td>
</tr>
<tr>
<td>Qubit dsDNA HS Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # Q32851</td>
</tr>
<tr>
<td>High Sensitivity D1000 ScreenTape</td>
<td>Agilent</td>
<td>Cat # 5067-5584</td>
</tr>
<tr>
<td>High Sensitivity D1000 Reagents</td>
<td>Agilent</td>
<td>Cat # 5067-5585</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Raw data</td>
<td>(<a href="#bib1">Van Rechem et al., 2021</a>)</td>
<td>GEO: GSE175752</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>hTERT RPE-1</td>
<td>Nicholas Dyson’s lab</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>FACS sorter</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Magnetic stand for 1.5 mL tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Magnetic stand for 96 wells plate</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Sonicator</td>
<td>QSonica</td>
<td>Q800R</td>
</tr>
<tr>
<td>Qubit</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Nanodrop</td>
<td>Thermo Fisher Scientific</td>
<td>2000c</td>
</tr>
<tr>
<td>TapeStation or Bioanalyser</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Thermocycler</td>
<td>N/A</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Antibodies</h3>
<p>Here is a list of antibodies we have optimized for this protocol. We specify which dilution IP buffer worked best for ChIP Sequencing and the amount of antibody to use. To get good sequencing results, some antibodies needed several IPs to be pooled together and some more chromatin in the IP. This could be due to the amount of histone modifications present in the genome and/or to the efficacy of these antibodies. When nothing is specified, we used our standard conditions (0.5 μg of chromatin and 0.2 μg of antibody per IP).</p>
<p>H3K4me1 ab8895 lot GR193882-1, Dilution IP buffer High Triton.</p>
<p>H3K4me2 ab32356 lot GR209821-1, Dilution IP buffer High Triton, 0.1 μg of antibody per IP.</p>
<p>H3K4me3 Millipore 07-473 lot 2648189 Dilution IP buffer Low Triton.</p>
<p>H3K27Ac Active Motif 39133 lot 31814008, Dilution IP buffer High Triton.</p>
<p>H3K9Ac ab4441 lot GR224698-1, Dilution IP buffer High Triton, 0.15 μg of antibody per IP.</p>
<p>H3K27me3 Millipore 07-449 lot 26532203, Dilution IP buffer High Triton, 0.15 μg of antibody per IP.</p>
<p>H3K9me3 ab8898 lot GR30928-1, Dilution IP buffer High Triton.</p>
<p>H3K36me3 ab9050 lot GR10860-1, Dilution IP buffer High Triton.</p>
<p>H3K9me1 ab8896-100 lot 815309, Dilution IP buffer High Triton. Pool two IPs.</p>
<p>H3K9me2 ab1220 lot GR32351-2, Dilution IP buffer High Triton. Pool three IPs using 1 μg of chromatin per IP.</p>
<p>H3K36me1 Cell Signaling 14111S ref 03/2017 lot 1, Dilution IP buffer High Triton.</p>
<p>H3K36me2 ab9049 lot GR316128-1, Dilution IP buffer High Triton. Pool two IPs.</p>
<h3 id="sec2.2">Buffers</h3>
<p>All buffers need to be filtered.</p>
<table id="undtbl1">
<caption>Cellular Lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PIPES pH 8 (500 mM)</td>
<td>5 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>KCl</td>
<td>85 mM</td>
<td>1.27 g</td>
</tr>
<tr>
<td>NP-40</td>
<td>0.5%</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>up to total</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to a year</p>
<table id="undtbl2">
<caption>Nuclear Lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 8 (1 M)</td>
<td>50 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>EDTA pH 8 (100 mM)</td>
<td>10 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>1%</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–25°C for up to a year</p>
<table id="undtbl3">
<caption>Dilution IP buffer Low Triton</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 8 (1 M)</td>
<td>16.7 mM</td>
<td>8.35 mL</td>
</tr>
<tr>
<td>EDTA pH 8 (100 mM)</td>
<td>1.2 mM</td>
<td>6 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>167 mM</td>
<td>16.7 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Triton X100</td>
<td>0.24%</td>
<td>1.2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>462.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to a year</p>
<table id="undtbl4">
<caption>Dilution IP buffer High Triton</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 8 (1 M)</td>
<td>16.7 mM</td>
<td>8.35 mL</td>
</tr>
<tr>
<td>EDTA pH 8 (100 mM)</td>
<td>1.2 mM</td>
<td>6 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>167 mM</td>
<td>16.7 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Triton X100</td>
<td>1.84%</td>
<td>9.2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>454.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to a year</p>
<table id="undtbl5">
<caption>TSE buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 8 (1 M)</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>EDTA pH 8 (100 mM)</td>
<td>2 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>500 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>Triton X100</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>0.1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>430 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–25°C for up to a year</p>
<table id="undtbl6">
<caption>LiCl buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 8 (1 M)</td>
<td>100 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>LiCl</td>
<td>500 mM</td>
<td>10.6 g</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>1%</td>
<td>5 g</td>
</tr>
<tr>
<td>NP-40</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>up to total</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–25°C for up to a year</p>
<table id="undtbl7">
<caption>TE buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 8 (1 M)</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>EDTA pH 8 (100 mM)</td>
<td>1 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>490 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–25°C for up to a year</p>
<table id="undtbl8">
<caption>Elution buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>50 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td>SDS (10%)</td>
<td>1%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>140 mM</td>
<td>700 <b>μ</b>L</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>up to total</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–25°C for up to 1 month</p>
<table id="undtbl9">
<caption>Glycine</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glycine</td>
<td>1.25 M</td>
<td>23.46 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>up to 250 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>250 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 18°C–25°C for up to a year</p>
<div class="note">
<span class="note-title">Note:</span> add ddH<sub>2</sub>O to ∼200 mL, adjust the pH to 2.5 with HCl, add the remaining ddH<sub>2</sub>O to 250 mL.</div>
<div class="critical">
<span class="critical-title">Critical:</span> LiCl and Sodium deoxycholate are toxic, especially in a volatile powder form. Handle with gloves in a fume hood.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Cell sorting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days to few weeks</div>
<p>During this step cells are fixed and sorted based on DNA content. Depending on the number of cells, their cell cycle distribution, and how much chromatin you will need, this step can be repeated as needed and samples pooled together (this explains the broad range in timing).</p>
<ol>
<li>Seed 2.5 × 10<sup>6</sup> RPE in four 15 cm<sup>2</sup> plates for 48 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> cells need to be at ∼70% confluence when harvested.</div>
<ol start="2">
<li>Treat the cells with Hoechst 33342 adding 1/1,000 directly on the medium, swirl to mix and incubate the cells at 37°C for 1 h. <a href="#troubleshooting">Troubleshooting 2</a>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> incubation time will need to be adjusted depending on the growth rate of the cells (30 min for HEK293T for example).</div>
<div class="note">
<span class="note-title">Note:</span> Hoechst can be toxic for certain cells, after incubation if cells look different (<i>i.e.,</i> rounded) less Hoechst should be used.</div>
<ol start="3">
<li>Shake off the loosely attached mitotic cells by tapping the side of the plates. Collect the medium in 50 mL conical and centrifuge in order to pellet the cells in mitosis that could be less adherent. Transfer the spun medium in another tube as it will be used in steps 5 and 9, leave enough medium behind to make sure not to aspirate the pelleted mitotic cells.</li>
<li>Rinse the cells with 1× PBS and add 2.5 mL of trypsin per 15 cm<sup>2</sup> plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> a single cell suspension is critical for the sorting.</div>
<ol start="5">
<li>Resuspend the cells in the spun medium from step 3, for four 15 cm<sup>2</sup> plates use 36 mL of medium, transfer in the 50 mL conical tube containing the pellet of mitotic cells.</li>
<li>Crosslink the cells by adding 1 mL of 37% formaldehyde to the medium (1% final), incubate at 37°C for 13 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> formaldehyde should be freshly opened and is toxic, use gloves and dispose properly.</div>
<ol start="7">
<li>Stop the fixation by adding 3.7 mL of glycine 1.25 M pH 2.5 (0.125 M final), incubate for at least 2 min at 18°C–25°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the medium should turn bright yellow.</div>
<ol start="8">
<li>Centrifuge the cells, wash the cell pellet with 1× PBS and centrifuge again.</li>
<li>Resuspend the cell pellet in 2 mL of spun medium from step 3.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the sorting step is more efficient when the cells have been resuspended in their original medium containing Hoechst, especially if the sorting step takes several hours.</div>
<ol start="10">
<li>Place the cells on ice until sorting.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> fixed cells can be kept on ice for few hours before sorting.</div>
<ol start="11">
<li>Sort the cells with a BD FACS Fusion using the BV421-A laser. A typical sort is represented in <a href="#fig2">Figure 2</a>.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1498-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Gating example for cell sorting</div>
</figcaption>
</figure>
</li>
</ol>
<p>Gating: place G1 (from the base of the large peak (1N) to 50% of the slope on the right), then G2/M (from the base of the second peak (2N) and encompassing the entire peak), and finally early and late S phase (the S fractions are equally divided in the middle). <a href="#troubleshooting">Troubleshooting 3</a> and <a href="#troubleshooting">4</a>.</p>
<div class="note">
<span class="note-title">Note:</span> any sorter with a laser that can read Hoechst can be used. The containers to hold and collect the cells will depend on the sorter.</div>
<div class="note">
<span class="note-title">Note:</span> all cells are sorted in four fractions and the lysis volume adjusted on the number of cells in step 14.</div>
<div class="note">
<span class="note-title">Note:</span> The amount of cells needed will depend on the yield of chromatin obtained and on the number of IPs the experimentalist wants to do with the chromatin. Several sorts can be pooled together is necessary.</div>
<ol start="12">
<li>Transfer the cells to 15 mL conical tubes and centrifuge for 15 min at 1,000 g at 4°C .</li>
<li>Lyse the cells in 200 μL cellular lysis buffer complemented with fresh proteinase inhibitors (cocktail tablets EDTA free), incubate for 5 min on ice, centrifuge for 5 min at 2,000 g at 4°C .</li>
<li>Remove the supernatant and resuspend the pellet in nuclear lysis buffer complemented with fresh proteinase inhibitors (cocktail tablets EDTA free), adjusting for the size of the pellet. For an estimated volume we use 10 μL for 300,000 cells, 25 μL for 1 million, 60 μL for 4 million.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> keep at −80°C (the unsonicated chromatin will be stable for many years).</div>
<h3 id="sec3.2">Sonication</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>During this step the collected chromatin will be sheared at an expected size of &lt; 300 pb, roughly one to two nucleosomes and some linker DNA.</p>
<ol start="15">
<li>Turn on the QSonica Q800R chiller. Transfer the chromatin in 0.5 mL thin wall PCR tubes. Sonicate according to the time determined in the optimization step.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> sorted cells can be easier to sonicate, therefore, they may need less sonication time. Do not over sonicate as the ChIP will be less efficient. The size will not decrease further but the chromatin will start to be altered.</div>
<div class="note">
<span class="note-title">Note:</span> the sonication bath should never get hot.</div>
<div class="note">
<span class="note-title">Note:</span> Steps 16–18 are optional but recommended. While the sonication efficiency should always be the same, we advise to check the sonication every time to make sure that this step worked properly before proceeding to the immunoprecipitation step. During these steps store the remaining chromatin at −80°C.</div>
<ol start="16">
<li>Add 3 μL of chromatin and 1 μL of proteinase K to 65 μL of elution buffer, incubate at 65°C for 4–16 h in a benchtop thermomixer shaking at 1,000 rpm.</li>
<li>Add 65 μL of phenol/chloroform/isoamyl alcohol 25:24:1, vortex, centrifuge for 1 min at full speed at 18°C–25°C.</li>
<li>Take 20 μL of the upper phase, mix with appropriate dye and run on a 1% agarose gel. The chromatin should be under 300 bp. If there is a big tail, the chromatin is not sonicated enough (see <a href="#fig1">Figure 1</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> use DNA loading dye with only one tracking color that runs at a high molecular to avoid using a dye running at the same size as your expected chromatin as it hides the chromatin.</div>
<div class="note">
<span class="note-title">Note:</span> the efficiency of sonication for cells within different cell cycle phases is identical.</div>
<ol start="19">
<li>Transfer the sonicated chromatin to 1.5 mL tubes and centrifuge for 10 min at 20,000 g at 4°C to clear the debris. Transfer the supernatant in a clean tube leaving any potential pellet behind.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> cold chromatin (just thaw or kept on ice) can present some SDS precipitates which will be pelleted during the centrifugation. Before proceeding with step 19, leave the chromatin at 18°C–25°C for few minutes to allow SDS to go back into solution.</div>
<div class="note">
<span class="note-title">Note:</span> we recommend keeping the chromatin in low bind DNA tubes, however regular tubes will work as well.</div>
<div class="note">
<span class="note-title">Note:</span> the expected chromatin concentration (DNA concentration from Nanodrop) is 300–1,000 ng/μL. <a href="#troubleshooting">Troubleshooting 5</a>.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> keep the chromatin at −80°C (the chromatin will be stable for few years, aliquot to avoid too many freezing/thawing cycles which will deteriorate the chromatin).</div>
<h3 id="sec3.3">Immunoprecipitation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>During this step, the chromatin will be immunoprecipitated with your antibodies of choice. Some antibodies require specific conditions to work their best. When in doubt, we recommend testing two different dilution IP buffers (low and high triton). We have incorporated a section with the antibodies we tested, and their optimal conditions in the material section.</p>
<ol start="20">
<li>Prebind the beads with the antibody. In a 200 μL tube: add 100 μL dilution IP buffer complemented with fresh protease inhibitors (cocktail tablets EDTA free), 2.5 μL protein A or G magnetic beads (prewashed and resuspended in dilution IP buffer, protein A for rabbit polyclonal and G for mouse monoclonal antibodies), and 0.2 μg of antibody. Incubate at 4°C for 6 h under rotation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> PCR strip tubes are useful at this step as they are easier to handle and allow to vortex several tubes at once in this and subsequent steps.</div>
<div class="note">
<span class="note-title">Note:</span> the incubation time can be increased.</div>
<div class="note">
<span class="note-title">Note:</span> the amount of antibody can vary depending on efficacy.</div>
<ol start="21">
<li>Pool the chromatin in a 1.5 mL tube: per IP include 0.5 μg of chromatin in 10 μL of nuclear lysis buffer with fresh protease inhibitors added (cocktail tablets EDTA free), also include 0.5 μg of chromatin for input (if using this to generate baseline input sequencing profiles) and 0.5 μg for pipetting error.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> example for 5 IPs + input + pipetting error: 5 × 0.5 + 0.5 + 0.5 = 2.5 μg of chromatin in 5 × 10 +10 +10 = 70 μL of nuclear lysis buffer. Note that the volume of chromatin will be deducted to the volume of nuclear lysis buffer: if 2.5 μg of chromatin represent 5 μL then 65 μL of nuclear lysis buffer will be needed.</div>
<div class="note">
<span class="note-title">Note:</span> the amount of chromatin per IP can vary, we have used 1 μg for some antibodies (see material section).</div>
<div class="note">
<span class="note-title">Note:</span> the chromatin concentration is based on DNA concentration taken on a Nanodrop.</div>
<ol start="22">
<li>Take 10 μL of the pooled chromatin (diluted in nuclear lysis buffer in the previous step). This will serve as the input, store at 4°C until further use.</li>
<li>Preclear step. Bring the remaining of the pooled chromatin to 100 μL per IP with dilution IP buffer complemented with fresh protease inhibitors (cocktail tablets EDTA free) (low or high triton depending on the IP condition).</li>
</ol>
<p>Add 2.5 μL of protein A agarose beads per IP (prewashed and resuspended in dilution IP buffer), incubate at 4°C for 2 h under rotation.</p>
<p>Centrifuge and transfer the chromatin in another tube leaving the beads behind.</p>
<p>Add 2.5 μL of protein A (or G) magnetic beads per IP (prewashed and resuspended in dilution IP buffer), incubate at 4°C for 2 h under rotation.</p>
<div class="note">
<span class="note-title">Note:</span> while protein A or G magnetic beads are used accordingly to the antibody that will be used for the IP, protein A agarose are used regardless of the IP. If using protein G for the IP, one could choose to use protein G agarose but this will not improve the preclear.</div>
<ol start="24">
<li>Quickly spin down the tubes with beads/antibodies (from step 20) and place them on a magnetic rack before removing the supernatant. Add 100 μL of precleared chromatin per tube, incubate at 4°C for 12–16 h with rotation.</li>
<li>Proceed to the washes. After a quick spin place the tubes on a magnet and remove the supernatant. For each wash, add 100 μL of buffer, vortex thoroughly, spin quickly, place on the magnetic rack and remove the buffer. Perform 6 washes: twice dilution IP buffer, once TSE buffer, once LiCl buffer, twice TE buffer. All washes are performed at 18°C–25°C. <a href="#troubleshooting">Troubleshooting 6</a>.</li>
<li>Elution. After the last wash, add 50 μL of elution buffer to the beads, transfer in a 1.5 mL tube, add 1 μL of RNAseA 100 μg/μL and incubate at 37°C for 30 min in a benchtop thermomixer shaking at 1,000 rpm. Add 1 μL of proteinase K (10 mg/mL) and incubate at 37°C for 1 h in a benchtop thermomixer shaking at 1,000 rpm. If pooled chromatin has been kept at 4°C for an input preparation, add 40 μL of elution buffer and proceed as for the IPs from this step.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> if pooling IPs together (see material section) it should be done at that step using 50 μL of elution buffer for all IPs.</div>
<div class="note">
<span class="note-title">Note:</span> the elution buffer should be less than a month old.</div>
<ol start="27">
<li>Decrosslinking. Remove the samples from the beads (quick spin and magnetic rack) and incubate at 65°C for 4 h in a benchtop thermomixer shaking at 1,000 rpm.</li>
<li>DNA purification. Add 125 μL of Agencourt AMPure XP to both the de-crosslinked IPs and input, incubate them for 10 min at 18°C–25°C, then place them on the magnet and remove the supernatant.</li>
</ol>
<p>The tubes are left on the magnet and 450 μL of freshly prepped 70% EtOH is added, then remove and repeat for a second time.</p>
<p>The last step is to wait for the beads to dry on the magnet before resuspending them in 40 μL of H<sub>2</sub>O. They are then incubated for 1 min at 18°C–25°C before being placed on the magnet. IP or input material is then removed from the beads.</p>
<div class="note">
<span class="note-title">Note:</span> In order to know when the beads are dry, we recommend looking under the light. The beads are dry when they are no longer shiny. However, do not wait longer than needed. If the beads dry too much (the smear against the magnet looks like it is “cracking”) the recovery will be less efficient (see example in <a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1498-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Examples of AMPure beads ready or not to be resuspended in water for DNA purification</div>
</figcaption>
</figure>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> keep the DNA at −20°C. The DNA is ready for library preparation or qPCR.</div>
<h3 id="sec3.4">Libraries preparation for sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>During this step the DNA will be prepped to be sequenced with the Illumina technology.</p>
<p>To avoid any exogenous DNA contamination work on a clean bench with clean pipetmans and sterile filter tips.</p>
<p>We used the TruSeq ChIP Sample Preparation kits and implemented few protocol modifications annotated below. The Illumina protocol can be found here: <a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/samplepreps_truseq/truseqchip/truseq-chip-sample-prep-guide-15023092-b.pdf">https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/samplepreps_truseq/truseqchip/truseq-chip-sample-prep-guide-15023092-b.pdf</a>.</p>
<p>We are annotating the main steps below as an overview, please refer to the guide except when noted otherwise.</p>
<ol start="29">
<li>Quantify each ChIP DNA using a Qubit dsDNA HS Assay Kit. We used 1 ng of DNA per library prep (or the entire IP if the concentration is below the threshold of detection). Place each DNA in a well of a 96 well plate and bring the volume up to 50 μL with H<sub>2</sub>O.</li>
<li>Perform end repair (page 11 of the Illumina PDF listed above).</li>
<li>Adenylate 3′ ends (page 14 of the Illumina PDF listed above).</li>
<li>Ligate adapters (page 16 of the Illumina PDF listed above).</li>
<li>Go directly to Enrich DNA fragments step (page 25 of the Illumina PDF listed above) <u>omitting</u> Purify ligation products (we purify using a different method after the PCR step) and <u>stopping at step 5</u> (page 27 of the Illumina PDF listed above).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> we typically perform 13 cycles of PCR.</div>
<div class="note">
<span class="note-title">Note:</span> we do not perform the washes step after the PCR, this step is replaced by a double-sided SPRI purification with AMPure beads as noted thereafter.</div>
<ol start="34">
<li>Add 30 μL of AMPure beads to the 50 μL PCR reaction, incubate at 18°C–25°C for 10 min.</li>
<li>Place the tubes on the magnetic stand. <u>Keep</u> the sample: transfer 77 μL of the sample in new wells. Discard the beads.</li>
<li>Add 12 μL of AMPure beads, incubate at 18°C–25°C for 10 min.</li>
<li>Place the plate on the magnetic stand. <u>Discard</u> the sample: take off 80 μL of the sample and discard. Keep the beads.</li>
<li>Perform two washes leaving the plate on the magnetic stand: add 200 μL of fresh prepped 80% EtOH, remove, repeat and dry the beads (as before, not too much).</li>
<li>Resuspend the beads in 17.5 μL of resuspension buffer, incubate at 18°C–25°C for 1 min.</li>
<li>Place the plate on the magnetic stand. Transfer 15 μL of each library in the final plate.</li>
<li>Validate the libraries using a TapeStation or a Bioanalyzer and proceed to sequencing.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> a typical TapeStation result is presented in <a href="#fig4">Figure 4</a>.</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1498-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. TapeStation analyses for a typical ChIP-seq library performed with this protocol</div>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> libraries should yield at a minimum 1,000 ng/mL. <a href="#troubleshooting">Troubleshooting 7</a>.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Expected outcomes were annotated in the protocol. The final libraries should yield at a minimum 1,000 ng/mL (Qubit concentration).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The main limitation to this protocol is the availability of antibodies that work for ChIP-seq. Some antibodies will be more effective than others with the lower amounts of chromatin being used. We recommend purchasing consistent lots that work when possible. Modifying the Dilution IP buffer as annotated in this protocol can lead to successful sequencing results. Another option for a better outcome is to modify the amount of SDS in the Nuclear Lysis buffer, bringing it down to as low as 0.2%. Such chromatin will require further sonication and the Dilution IP buffers will need to be adapted in order for the IP to be done in 0.2% SDS (bring the SDS at 0.2% in the Dilution IP buffers). Of note, this protocol has only been successfully tested for histone and histone marks, additional adaptations might be necessary for other proteins. <a href="#troubleshooting">Troubleshooting 8</a>.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p><a href="#before-you-begin">Before you begin</a> step 11. Clearly visible DNA fragments above 300 bp after sonication.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Sonicate more (see <a href="#fig1">Figure 1</a>).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Step 2. Cells change shape after Hoechst treatment.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Hoechst is toxic to the cells, reduce the concentration.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Step 11. Cells not really visible with the laser exciting Hoechst.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Increase the time/concentration of Hoechst treatment.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Step 11. Cells are clumpy during sorting.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Make sure to resuspend the cells in a single cell suspension and dilute more the cells before sort (sort in a higher volume). A cell strainer can be used at this step to remove any non-single cell.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Step 19. Chromatin concentration too low.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Decrease the volume of Nuclear Lysis buffer when resuspending the cells.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Step 25. The magnetic beads lose their magnetism (slide down the tube when on the magnetic stand).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Increase the speed of the washes / reduce to one wash with TE instead of two.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Step 41. Libraries with very low amount of DNA.</p>
<h3 id="sec6.14">Potential solution</h3>
<p>Multiply the number of IPs and pool them together at the elution step (#26).</p>
<p>Try another antibody or another Dilution IP buffer (see <a href="#limitations">limitations</a> section).</p>
<p>The numbers of PCR cycles in step 33 can be increased slightly (∼2 cycles), however too many cycles will create PCR duplicates that will need to be removed from the sequencing results. We therefore recommend to multiple the number of IPs instead.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>Limitations. Too much background / no peaks when assessing the ChIP-seq on IGV or equivalent.</p>
<h3 id="sec6.16">Potential solution</h3>
<p>Increase the number of washes and vortex more.</p>
<p>Try another antibody or another Dilution IP buffer (see <a href="#limitations">limitations</a> section).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Van Rechem, C., Ji, F., Chakraborty, D., Black, J.C., Sadreyev, R.I., and Whetstine, J.R. (2021). Collective regulation of chromatin modifications predicts replication timing during cell cycle. Cell Rep. <i>37</i>, 109799. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00123-X/sref1">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Capucine Van Rechem (<a href="mailto:cvrechem@stanford.edu">cvrechem@stanford.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The published article includes all datasets and codes generated during this study.</p>
<h3>Acknowledgments</h3>
<p>This study is supported by R01GM097360 and R35GM144131 (J.R.W.), <a href="https://doi.org/10.13039/100000002">NIH</a>/<a href="https://doi.org/10.13039/100000054">NCI</a> Cancer Center Support Grant P30 CA006927 (J.R.W.), and the <a href="https://doi.org/10.13039/100002590">American Lung Association</a> Lung Cancer Discovery Award.</p>
<h3>Author contributions</h3>
<p>C.V.R. and J.R.W. worked together to optimize the conditions that are described in this methods article. C.V.R. conducted the methods.</p>
<h3>Declaration of interests</h3>
<p>In the past year, J.R.W. has consulted for Qsonica (manufacturer of Q800R system). J.R.W. has served as a consultant or advisor for Salarius Pharmaceuticals, Daiichi Sankyo, Inc., and VYNE Therapeutics. J.R.W. has also received sponsored research from Salarius Pharmaceuticals. C.V.R. declares no competing interests.</p>
</section>
</article>