<article>
<section>
<h1 id="header">Protocol for isolation and functional validation of label-retaining quiescent colorectal cancer stem cells from patient-derived organoids for RNA-seq</h1>
<p><time datetime="2022-03-11">Published: March 11, 2022</time></p>
<p>Joseph L. Regan<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Charité Comprehensive Cancer Center, Charité - Universitätsmedizin Berlin, 10117 Berlin, Germany</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:joseph.regan@charite.de">joseph.regan@charite.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101225">10.1016/j.xpro.2022.101225</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Quiescent cancer stem cells (qCSCs) are a major source of posttreatment relapse, but methods to identify molecular targets for qCSC elimination are limited. Here, we present a protocol using the fluorescent dye PKH26 to isolate label-retaining qCSCs from colorectal cancer (CRC) patient-derived organoids (PDOs). We describe processing of organoids to single cells, followed by PKH26 labeling and FACS-based cell isolation. We then detail steps for functional assays and RNA sequencing. This protocol can also be applied to normal tissue-derived organoids.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.isci.2021.102618">Regan et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for the isolation of PKH26 label-retaining cells from CRC organoids</li>
<li>Processing organoids to single cells, PKH26 labeling, and FACS</li>
<li>Functional validation and RNA sequencing of quiescent cancer stem cells</li>
<li>Applicable to normal tissue-derived and patient-derived organoids</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1492-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol employs a pulse-chase label retention approach, wherein dividing cells lose the label, and quiescent or slow cycling cells retain the label for an extended period. In contrast to histone 2B-GFP (H2B-GFP) dilution approaches, that can identify transient quiescent cells, label retention strategies allow for the identification of cells that remain quiescent from the early stages of tumorigenesis (<a href="#bib4">Blanpain and Simons, 2013</a>; <a href="#bib18">Puig et al., 2018</a>). This is important since cells selectively surviving chemotherapy have been shown to be the same cells that are quiescent/slow cycling in untreated tumors and not cells that become quiescent upon treatment (<a href="#bib10">Francescangeli et al., 2020</a>). Label retention strategies therefore enable the identification of long-term qCSCs with the potential to cause post-treatment disease relapse (<a href="#bib7">Dembinski and Krauss, 2009</a>; <a href="#bib23">Santos-de-Frutos and Djouder, 2021</a>).</p>
<p>The protocol below describes the specific steps for isolating label-retaining qCSCs from CRC PDOs using the lipophilic fluorescent dye PKH26. PKH26 labelling can be used to identify long-term label-retaining cells (LRCs) for up to six months (<i>in vitro</i> and <i>in vivo</i>) and has lower cytotoxicity than cytoplasmic and nuclear dyes (<a href="#bib12">Horan and Slezak, 1989</a>; <a href="#bib6">Cicalese et al., 2009</a>). In addition, this protocol may also be applied to organoid models derived from different cancer and tissue types and used for tracking of quiescent cells <i>in vivo</i>.</p>
<p>We have successfully used this protocol to isolate PKH26 LRCs and reported the first whole-transcriptome analyses of functionally validated qCSCs in a panel of colon cancer PDOs (<a href="#bib20">Regan et al., 2021</a>). These cells maintain a large proliferative capacity, persist long term <i>in vivo,</i> and display the molecular hallmarks of quiescent tissue stem cells (<a href="#bib5">Cheung and Rando, 2013</a>). In addition, we generated “the colon cancer quiescent cancer stem cell signature” and showed that qCSCs are enriched for p53 interacting negative regulators of cell cycle, which we propose may be targeted for cell cycle activation and qCSC elimination.</p>
<h3 id="sec1.1">Organoid establishment and culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–14 days</div>
<p>Establish and maintain PDOs in Matrigel culture as described in (<a href="#bib24">Sato et al., 2011</a>; <a href="#bib19">Regan et al., 2017</a>; <a href="#bib27">Schütte et al., 2017</a>). For detailed step-by-step protocols see (<a href="#bib17">Pleguezuelos-Manzano et al., 2020</a>) and (<a href="#bib26">Schumacher et al., 2021</a>). For the establishment of PDO cultures directly from patient material, informed patient consent and approval of the relevant local/national ethics committee/review board must first be obtained. Tumor material used for this protocol was obtained with informed consent from CRC patients under approval from the local Institutional Review Board of Charité University Medicine (Charité Ethics Committee, Berlin, Germany) (EA 1/069/11), the ethics committee of the Medical University of Graz (Graz, Austria) and by the ethics committee of the St John of God Hospital Graz (23-015 ex 10/11). For additional information see (<a href="#bib20">Regan et al., 2021</a>).</p>
<p>Culture PDOs as Matrigel domes (droplets) in multi-well cell culture plates and overlay with Complete PDO Culture Medium (formulated as described below). The size of the Matrigel dome, multi-well plate, and volume of media used should be altered depending on the downstream application for which the organoids will be used.</p>
<p>Label-retaining qCSCs are rare. Therefore, in order to isolate sufficient cell numbers for functional analyses and RNA-sequencing, this protocol uses 120 μL Matrigel domes in 6-well plate format overlaid with 2 mL of Complete PDO Culture Medium.</p>
<h3 id="sec1.2">Preparation for processing of PDOs for labeling and functional assays</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 h</div>
<ol>
<li>Thaw Matrigel overnight at 4°C.</li>
<li>Store sterile micropipette tip boxes at −20°C prior to use. This will minimize solidification of Matrigel on the tip surface during pipetting.</li>
<li>Warm multi-well plates in a 37°C incubator.</li>
</ol>
<h3 id="sec1.3">Preparation for extraction of RNA from low numbers of PDO cells isolated by fluorescence-activated cell sorting (FACS)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 30 min</div>
<ol start="4">
<li>Prior to performing RNA isolation from low numbers of rare label-retaining qCSCs, wipe all work surfaces and equipment with RNAseZAP RNase Decontamination Solution.</li>
<li>Ensure all reagents and materials used for RNA extraction are certified as DNase- and RNase-free.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>EpCAM-PE (1:100 dilution)</td>
<td>Abcam</td>
<td>12-9326-42</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>CRC PDOs</td>
<td>Charité Universitätsmedizin Berlin, Germany</td>
<td>OncoTrack</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Matrigel, Phenol Red-Free, Growth Factor Reduced</td>
<td>Corning</td>
<td>CLS356231</td>
</tr>
<tr>
<td>RNAseZAP RNase Decontamination Solution</td>
<td>Thermo Fisher Scientific</td>
<td>AM9780</td>
</tr>
<tr>
<td>Advanced DMEM/F12</td>
<td>Gibco</td>
<td>12634-010</td>
</tr>
<tr>
<td>Penicillin/Streptomycin (100×)</td>
<td>Invitrogen</td>
<td>15140-122</td>
</tr>
<tr>
<td>GlutaMAX (100×)</td>
<td>Thermo Fisher Scientific</td>
<td>35050061</td>
</tr>
<tr>
<td>N2 Supplement (100×)</td>
<td>Thermo Fisher Scientific</td>
<td>17502-048</td>
</tr>
<tr>
<td>B27 Supplement (50×)</td>
<td>Thermo Fisher Scientific</td>
<td>17504-044</td>
</tr>
<tr>
<td>N-acetyl-L-cysteine (500 mM)</td>
<td>Sigma-Aldrich</td>
<td>A9165</td>
</tr>
<tr>
<td>HEPES buffer (1 M)</td>
<td>Thermo Fisher Scientific</td>
<td>15630049</td>
</tr>
<tr>
<td>EGF (50 μg/mL)</td>
<td>Sigma-Aldrich</td>
<td>E9644</td>
</tr>
<tr>
<td>bFGF (100 μg/mL)</td>
<td>Sigma-Aldrich</td>
<td>F0291</td>
</tr>
<tr>
<td>4′,6-diamidino-2-phenylindole (DAPI)</td>
<td>Thermo Fisher Scientific</td>
<td>D1306</td>
</tr>
<tr>
<td>TrypLE Express</td>
<td>Thermo Fisher Scientific</td>
<td>12604039</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>PKH26 Red Fluorescent Cell Linker Kit</td>
<td>Sigma-Aldrich</td>
<td>PKH26GL</td>
</tr>
<tr>
<td>RNeasy Plus Micro Kit</td>
<td>QIAGEN</td>
<td>74034</td>
</tr>
<tr>
<td>CellTiter-Glo Luminescent Cell Viability Assay</td>
<td>Promega</td>
<td>G7572</td>
</tr>
<tr>
<td>Ovation RNA-Seq System V2</td>
<td>Tecan</td>
<td>7102-A01</td>
</tr>
<tr>
<td>Universal Plus Total RNA-Seq</td>
<td>Tecan</td>
<td>0344, 0344NB</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Array data</td>
<td>(<a href="#bib20">Regan et al., 2021</a>); <a href="http://www.ebi.ac.uk/arrayexpress">www.ebi.ac.uk/arrayexpress</a>
</td>
<td>ArrayExpress: E-MTAB-8924</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>BD FACSDiva</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>ELDA software</td>
<td>(<a href="#bib13">Hu and Smyth, 2009</a>)</td>
<td><a href="http://bioinf.wehi.edu.au/software/elda/index.html">http://bioinf.wehi.edu.au/software/elda/index.html</a></td>
</tr>
<tr>
<td>ImageJ</td>
<td>(<a href="#bib25">Schneider et al., 2012</a>)</td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td>DESeq2</td>
<td>(<a href="#bib15">Love et al., 2014</a>)</td>
<td><a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html">https://bioconductor.org/packages/release/bioc/html/DESeq2.html</a></td>
</tr>
<tr>
<td>STAR aligner (version 2.4.2a).</td>
<td>(<a href="#bib8">Dobin et al., 2013</a>)</td>
<td><a href="https://github.com/alexdobin/STAR">https://github.com/alexdobin/STAR</a></td>
</tr>
<tr>
<td>Gene Ontology Resource</td>
<td>(<a href="#bib1">Ashburner et al., 2000</a>)</td>
<td><a href="http://www.geneontology.org">www.geneontology.org</a></td>
</tr>
<tr>
<td>GSEA software</td>
<td>(<a href="#bib28">Subramanian et al., 2005</a>; <a href="#bib14">Liberzon et al., 2015</a>)</td>
<td><a href="https://www.gsea-msigdb.org/gsea/index.jsp">https://www.gsea-msigdb.org/gsea/index.jsp</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>6-well cell culture multi-well plates</td>
<td>VWR</td>
<td>82050-842</td>
</tr>
<tr>
<td>96-well cell culture multi-well plates</td>
<td>Corning</td>
<td>3595</td>
</tr>
<tr>
<td>384-well ultra-low attachment spheroid plates</td>
<td>Corning</td>
<td>3830</td>
</tr>
<tr>
<td>FACS Aria II</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>Microscope</td>
<td>Leica Microsystems</td>
<td>DMi8</td>
</tr>
<tr>
<td>70 μm cell strainer</td>
<td>Corning</td>
<td>352350</td>
</tr>
<tr>
<td>FACS tubes</td>
<td>Stellar Scientific</td>
<td>FSC-9010</td>
</tr>
<tr>
<td>Illumina HiSeq 2500</td>
<td>Illumina</td>
<td>SY–401–2501</td>
</tr>
<tr>
<td>Qubit</td>
<td>Thermo Fisher Scientific</td>
<td>Q33238</td>
</tr>
<tr>
<td>Fragment analyzer</td>
<td>Agilent</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Basic PDO Culture Medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Advanced DMEM/F12</td>
<td>n/a</td>
<td>495 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin (100×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C until expiration date.</p>
<table id="undtbl2">
<caption>Complete PDO Culture Medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Basic PDO Culture Medium</td>
<td>n/a</td>
<td>94.69 mL</td>
</tr>
<tr>
<td>GlutaMAX (100×)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>N2 Supplement (100×)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>B27 Supplement (50×)</td>
<td>1×</td>
<td>2 mL</td>
</tr>
<tr>
<td>N-acetyl-L-cysteine (500 mM)</td>
<td>1 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>HEPES buffer (1 M)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>EGF (50 μg/mL)</td>
<td>50 ng/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>bFGF (100 μg/mL)</td>
<td>20 ng/mL</td>
<td>100 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to two weeks.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Process PDOs to single cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼30 min</div>
<p>Organoid size and growth rate can vary greatly between different patient derived cultures. Therefore, the culture size and density prior to processing is dependent on the growth dynamics of the individual PDO model. For examples of PDOs that are ready for processing, see <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1492-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Example of PDOs that are ready for processing</div>
<p>(A) Low magnification image of PDOs in culture, showing numerous organoids at different depths (at the focal plane and below) in the Matrigel dome, illustrating the 3D culture environment and providing an example of confluency and organoid size (Scale bar 400 μm).</p>
<p>(B) Examples of different PDOs that are ready for processing (Scale bar 200 μm). Images in B used with permission under a CC-BY 4.0 license from (<a href="#bib19">Regan et al., 2017</a>).</p>
</figcaption>
</figure>
<ol>
<li>Use a P1000 micropipette tip to detach and break up Matrigel dome organoid cultures and create a Matrigel-organoid suspension within the Complete PDO Culture Medium.</li>
<li>Transfer the Matrigel-organoid suspension(s) to a 15 mL Falcon tube and rinse each well with an additional 1 mL of Basic PDO Culture Media to collect any residual Matrigel-organoid fragments.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Multiple wells of the same PDO model may be combined in a single 15 mL Falcon tube.</div>
<ol start="3">
<li>Add the residual Matrigel-organoid suspension to the 15 mL Falcon tube and triturate (pipette up and down) with a P1000 tip for 30 s to mechanically dissociate the Matrigel and liberate organoids into suspension.</li>
<li>Centrifuge at 300×<i>g</i> for 3 min and carefully aspirate the supernatant, including Matrigel.</li>
<li>Resuspend the organoid pellet in 200 μL of TrypLE Express, incubate at 37°C for 15 min and triturate with a P200 tip every 5 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Adjust the volume of TrypLE Express depending on the number of wells added to the 15 mL Falcon tube.</div>
<div class="note">
<span class="note-title">Note:</span> Some PDOs may require longer incubation times to achieve single cell suspensions. In such cases 100 μg/mL DNase I may be added to TrypLE Express to prevent cell clumping, caused by the release of “sticky” DNA molecules from dying cells. However, DNase I should not be used prior to downstream processes involving DNA or RNA extraction.</div>
<ol start="6">
<li>Add 2 mL of Basic PDO Culture Medium to stop the effect of TrypLE Express and centrifuge at 300×<i>g</i> for 3 min.</li>
<li>Carefully remove supernatant and resuspend cells in 1 mL Basic PDO Culture Media.</li>
</ol>
<h3 id="sec3.2">Label single cells with PKH26 dye</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 1 h</div>
<p>The following protocol has been optimized for PKH26 labeling of CRC PDOs and uses a final concentration of 2 × 10<sup>-6</sup> M of PKH26 and 1 × 10<sup>7</sup> cells per mL suspended in a final volume of 2 mL. This method may also be used for <i>in vitro</i> and <i>ex vivo</i> labeling of other cell types from a wide variety of systems. PKH26 fluorescence is extremely stable and compatible for downstream use with green, violet, red, or far-red viability probes, DNA dyes, fluorescent proteins, and antibodies. For further details on PKH26 Red Fluorescent Cell Linker Kits for General Cell Membrane Labeling see manufacturer’s protocol (<a href="https://www.sigmaaldrich.com/deepweb/assets/sigmaaldrich/product/documents/271/475/mini26bul.pdf">https://www.sigmaaldrich.com/deepweb/assets/sigmaaldrich/product/documents/271/475/mini26bul.pdf</a>)</p>
<div class="critical">
<span class="critical-title">Critical:</span> Over-labeling of cells will cause loss of membrane integrity and reduced cell recovery. It is therefore essential that the amount of lipophilic PKH26 dye incorporated into cell membranes be limited.</div>
<ol start="8">
<li>Dilute 10 μL of cell suspension with 10 μL of trypan blue and place 10 μL into a hemocytometer to count live cells.</li>
<li>Prepare a 2 × 10<sup>7</sup> per mL single cell suspension in Basic PDO Culture Media in a 1.5 mL Eppendorf tube and centrifuge at 300×<i>g</i> for 3 min.</li>
<li>Carefully remove the supernatant with a micropipette leaving no more than 25 μL of supernatant in the tube.</li>
<li>Resuspend cells in 1 mL of Diluent C (provided with the PKH26 Red Fluorescent Cell Linker Kit) with gentle trituration with a P1000 tip to ensure complete dispersion of cells.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not disperse cells in Diluent C by vortexing and do not keep cells in Diluent C for extended periods, i.e., no more than 5–10 min, including incubation time in step 14 below.</div>
<ol start="12">
<li>Add 4 μL of PKH26 dye to 1 mL of Diluent C in a 2 mL Eppendorf tube and mix well.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare the dye solution in Diluent C immediately prior to staining.</div>
<ol start="13">
<li>Immediately after preparation of the PKH26 dye solution, add the 1 mL Diluent C cell suspension to the 1 mL dye solution and instantly triturate with a P1000 tip.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Staining is instantaneous. Therefore, for uniform and reproducible cell labeling ensure rapid and uniform dispersion of cells in dye solution.</div>
<ol start="14">
<li>Incubate cell-dye suspension for 2–3 min and triturate with a P1000 tip every 30 s.</li>
<li>Stop the staining by adding the cell-dye solution to a 15 mL Falcon tube containing 8 mL of 10% FCS DMEM/F12 and incubate at room temperature (20°C–22°C) for 1 min.</li>
<li>Centrifuge at 400×<i>g</i> for 5 min, carefully remove supernatant, resuspend in 10 mL of Basic PDO Culture Media and transfer to a fresh 15 mL Falcon tube.</li>
<li>Centrifuge at 300×<i>g</i> for 5 min, remove supernatant and wash cells in 10 mL Basic PDO Culture Media. Repeat two more times to ensure removal of unbound dye.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Successfully PKH26 labeled cell pellets will appear visibly pink.</div>
<div class="optional">
<span class="optional-title">Optional:</span> For flow cytometric evaluation of PKH26 labeling and cell viability, prior to the final wash step remove 500 μL of the cell suspension, transfer to a 1.5 mL Eppendorf tube and centrifuge at 300×<i>g</i> for 3 min. Resuspend cells in 500 μL Basic PDO Culture Media containing a 1:10,000 DAPI dilution, pass through a 70 μm cell strainer into FACS tubes, place on ice (for up to 3 h) and proceed with flow cytometric analysis.</div>
<ol start="18">
<li>Centrifuge at 300×<i>g</i> for 3 min and place the 15 mL Falcon containing the stained cell pellet on ice.</li>
<li>Resuspend cells on ice in the appropriate volume of Matrigel, i.e., 120 μL Matrigel per well of a 6-well plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The volume of Matrigel used for resuspension and the number of wells used for plating is dependent on the number of cells isolated and their growth dynamics as organoids in Matrigel. Organoid proliferation rates and sizes can vary greatly between patient models and so the number of cells resuspended per Matrigel dome should be individually determined based on the previously observed growth dynamics of each PDO model.</div>
<ol start="20">
<li>Remove pre-warmed 6-well plates from the incubator and add 120 μL of Matrigel-cell suspension to the center of each well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For immunofluorescence staining and confocal analysis of PKH26 labeled PDOs (<a href="#fig3">Figure 3</a>E), increase the volume of Matrigel added to resuspend the cell pellet in step 19 and add 12 μL of Matrigel-cell suspension per well of a 48-well plate, allow to set at 37°C, overlay with Complete PDO Culture Media and culture for 12 days. After 12 days, formalin fix cells and proceed with immunofluorescence staining as described in (<a href="#bib21">Regan, 2022</a>).</div>
<ol start="21">
<li>Allow Matrigel to set at 37°C for 10 min in the incubator and overlay solidified Matrigel domes with 2 mL of Complete PDO Culture Media.</li>
<li>Visually inspect and record PKH26 labeling under a fluorescence microscope (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1492-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. PKH26 stained PDO cells in Matrigel directly after labeling (LHS) and after 12 days (RHS)</div>
<p>Scale bar 100 μm.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1492-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Expected outcomes: Non-cycling label-retaining PDO cells are quiescent CSCs that can re-enter cell cycle</div>
<p>(A) Representative FACS plots of PKH26-labeled PDO cells after 12 h (middle panel) and 12 days (right side panel) compared with non-labeled control (left side panel).</p>
<p>(B) Frequency of PKH26<sup>Positive</sup> LRCs in PDO models after 12 days (±SD; data from 5 independent experiments).</p>
<p>(C) FACS histograms demonstrating frequency of PKH26<sup>Positive</sup> cells in PDOs at 12 h (middle left side panel) and 12 days (middle right side panel) after staining and 24 days (right side panel) after FACS isolation and serial replating of PKH26<sup>Positive</sup> cells from 12-day cultures.</p>
<p>(D and E) Phase contrast of unlabeled PDOs (negative control) (scale bar, 100 μm) and (E) immunofluorescence images of PKH26-labeled PDOs at 12 h and 12 days (middle panels) and 24 days after FACS isolation and serial re-plating of PKH26<sup>Positive</sup> LRCs from 12-day cultures (right side panel). Cells are stained for F-ACTIN (green), and nuclei are counterstained with DAPI (blue) (scale bars, 20 μm; scale bar for zoomed image, 5 μm). Images used with permission under a CC-BY 4.0 license from (<a href="#bib20">Regan et al., 2021</a>).</p>
</figcaption>
</figure>
</li>
<li>Culture plates at 37°C for 12 days, changing media every 2 days.</li>
</ol>
<h3 id="sec3.3">Isolation of PKH26 label-retaining cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 days</div>
<p>As the PKH26 labeled single cells proliferate and self-organize into PDOs they will dilute the dye between successive daughter cells. Therefore, after 12 days in culture the majority of PDOs will be PKH26<sup>Negative</sup> and a rare minority of non-proliferating quiescent cells will remain PKH26<sup>Positive</sup>.</p>
<ol start="24">
<li>After 12 days in culture, visually inspect Matrigel domes with a fluorescence microscope. The majority of PDO cells will be PKH26<sup>Negative</sup>. Rare PKH26<sup>Positive</sup> cells will be observed both inside and outside PDOs (<a href="#fig2">Figure 2</a>).</li>
<li>To isolate PKH26 label retaining cells for functional testing and RNA-sequencing, process PDOs to single cells as described in steps 1–7.</li>
<li>Carefully remove supernatant and resuspend pellet in 5 mL Basic PDO Culture Media containing a 1:10,000 DAPI dilution, pass through a 70 μm cell strainer into FACS tubes, place on ice and proceed with FACS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare the appropriate FACS compensation controls of unstained cells and single stained cells (freshly labeled with PKH26 or PE, e.g., EpCAM-PE only, and DAPI only).</div>
<ol start="27">
<li>Load the sample tube and create the appropriate gating strategy to isolate and sort single live DAPI<sup>Negative</sup> PKH26<sup>Negative</sup> cycling cells and DAPI<sup>Negative</sup> PKH26<sup>Positive</sup> non-cycling LRCs.
<ol type="a">
<li>Create a Forward Scatter (FSC) versus Side Scatter (SSC) plot and adjust the individual FSC and SSC photomultiplier tube settings to ensure that all cells are visible.</li>
<li>Set and adjust the FSC Threshold to remove FSC-low cellular debris.</li>
<li>Create a gate to select “all cells”.</li>
<li>Create a FSC-H vs. FSC-A (or FSC-W) plot to exclude doublets and multiplets (cells stuck together that may be counted as a single event when passing through the laser) and apply the “all cells” gate onto it.</li>
<li>Create a gate to select “singlets”.</li>
<li>Create a FSC vs. DAPI plot and apply the “singlets” gate onto it. Create a gate around the DAPI<sup>Negative</sup> cells and label it “live cells”.</li>
<li>Create a FSC vs PKH26 (PE) plot and apply the “live cells” gate.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use the negative control to set the positive and negative gates and discern the positive signal from background fluorescence.</div>
<ol start="28">
<li>Perform a purity check and make any necessary technical adjustments before proceeding with collection of cell subpopulations.</li>
<li>Proceed with FACS collection of cellular subpopulations for functional testing and RNA-sequencing as described below.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The number of PKH26<sup>Positive</sup> cells and their fluorescence intensity will vary between PDO models. Subpopulations of slow-cycling cells that have not had time to completely dilute the PKH26 dye may have low-to-intermediate levels of label retention (PKH26<sup>Positive/Low</sup>). Such cells can be excluded from FACS isolation by setting the gate for PKH26<sup>Positive/High</sup> cells at the far end of the PKH26<sup>Positive</sup> cell dot plot. Alternatively, if a large number of PKH26<sup>Positive/Low</sup> cells are observed, consider extending the culture period to allow further dye dilution before proceeding with FACS isolation steps.</div>
<h3 id="sec3.4">Functional testing of PKH26<sup>Positive</sup> LRCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 Days</div>
<p>Functional analyses of PKH26<sup>Positive</sup> LRCs are carried out to determine their self-renewal and proliferative capacity as non-adherent spheroids and as organoids in adherent Matrigel culture, respectively.</p>
<ol start="30">
<li>For spheroid formation assays, prepare 384-well non-adherent ultra-low attachment plates by adding 100 μL Complete PDO Culture Media per well.</li>
<li>Separately sort single live PKH26<sup>Negative</sup> cells and PKH26<sup>Positive</sup> LRCs directly into 384-well plates at a frequency of one cell per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Adjust the frequency of cells plated per well and the number of wells used per cell dilution depending on the clonogenicity of the cell population, e.g., for less clonogenic cells, set up a limiting dilution assay in 96-well non-adherent ultra-low attachment plates with cell frequencies ranging from 1 – 100 cells per well.</div>
<ol start="32">
<li>Visually inspect each well with a microscope to confirm the presence of a single cell per well, incubate plates at 37°C and culture for up to 20 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The time required for qCSCs from different PDOs to recover from sorting, exit quiescence and initiate spheroid formation may vary between models. Extend the culture period if no spheroids have formed by day 20.</div>
<ol start="33">
<li>Count the number of spheroids formed by PKH26<sup>Negative</sup> cells and PKH26<sup>Positive</sup> LRCs and calculate CSC frequency using ELDA software (<a href="http://bioinf.wehi.edu.au/software/elda/index.html">http://bioinf.wehi.edu.au/software/elda/index.html</a>) (<a href="#bib13">Hu and Smyth, 2009</a>).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Image each well using a microscope to measure spheroid size using ImageJ software.</div>
<ol start="34">
<li>For adherent colony forming proliferation assays, sort live PKH26<sup>Negative</sup> cells and PKH26<sup>Positive</sup> LRCs separately into chilled FACS collections tubes containing 500 μL Complete PDO Culture Media.</li>
<li>Dilute 10 μL of each cell suspension with 10 μL of trypan blue and place 10 μL into a hemocytometer to count live cells.</li>
<li>Transfer the sorted cell suspensions to 1.5 mL Eppendorf tubes and centrifuge at 4°C at 300×<i>g</i> for 3 min.</li>
<li>Resuspend cells at a concentration of 100 live cells per 5 μL Matrigel.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Include an additional 10% cells and Matrigel in the overall volume to account for loss during pipetting.</div>
<ol start="38">
<li>Remove pre-warmed 96-well plates from the incubator and add 5 μL of Matrigel-cell suspension to the center of each well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When pipetting small volumes of Matrigel into large multi-well plate formats it is especially important to keep pipette tips chilled. This will help to minimize solidification of Matrigel on the pipette surface and reduce loss of Matrigel-cell suspension during pipetting.</div>
<ol start="39">
<li>Overlay each well with 100 μL Complete PDO Culture Media and incubate for 12 days. Wait for organoids to begin forming before changing media. Thereafter, change media every 2 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Some PDO cells may require more time to recover and form organoids after sorting.</div>
<ol start="40">
<li>Image each well with a microscope, measure organoid size using ImageJ software and use CellTiter-Glo viability assay (<a href="https://www.promega.de/en/products/cell-health-assays/cell-viability-and-cytotoxicity-assays/celltiter_glo-luminescent-cell-viability-assay/">https://www.promega.de/en/products/cell-health-assays/cell-viability-and-cytotoxicity-assays/celltiter_glo-luminescent-cell-viability-assay/</a>) to determine differences in cell proliferation between the PKH26<sup>Negative</sup> cell-derived PDOs and PKH26<sup>Positive</sup> LRC-derived PDOs.</li>
</ol>
<h3 id="sec3.5">Library preparation and RNA-sequencing of PKH26<sup>Positive</sup> label-retaining qCSCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<p>The RNA-seq experiments in (<a href="#bib20">Regan et al., 2021</a>) used the Ovation RNA-Seq System V2 and Ultralow V2 Library System, which are optimized for purified total RNA in the range of 5–100 ng. However, other suitable library construction methods, such as the Universal Plus Total RNA-Seq library preparation kit with NuQaunt (Tecan) may also be used. Detailed instructions for whole transcriptome RNA amplification for preparing amplified cDNA from total RNA (<a href="https://lifesciences.tecan.com/ovation-low-input-rna-seq-kit-v2?p=tab--1">https://lifesciences.tecan.com/ovation-low-input-rna-seq-kit-v2?p=tab--1</a>) and library preparation (<a href="https://lifesciences.tecan.com/universal-rna-seq-library-prep-kit">https://lifesciences.tecan.com/universal-rna-seq-library-prep-kit</a>) can be found on the manufacturer’s website.</p>
<ol start="41">
<li>Collect DAPI<sup>Negative</sup> PKH26<sup>Negative</sup> cells and DAPI<sup>Negative</sup> PKH26<sup>Positive</sup> LRCs by FACS from four different biological replicates into collection tubes containing RLT lysis buffer at 4°C.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> For single cell RNA-sequencing (scRNA-seq), collect cells in Complete PDO Culture Media and load single cell suspensions onto a 10× Chromium platform. For detailed instruction see the manufacturer’s website (<a href="https://support.10xgenomics.com/single-cell-gene-expression/library-prep/doc/user-guide-chromium-single-cell-3-reagent-kits-user-guide-v31-chemistry-dual-index">https://support.10xgenomics.com/single-cell-gene-expression/library-prep/doc/user-guide-chromium-single-cell-3-reagent-kits-user-guide-v31-chemistry-dual-index</a>).</div>
<ol start="42">
<li>Process samples for RNA using an RNeasy Plus RNA extraction kit according to the manufacturer’s instructions (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/rna-purification/total-rna/rneasy-plus-kits/">https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/rna-purification/total-rna/rneasy-plus-kits/</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the number of PKH26<sup>Positive</sup> LRCs collected, an RNeasy Micro Plus or Mini Plus extraction kit may be used. For lower cell numbers (&lt; 5 × 10<sup>5</sup> cells, which is likely for qCSCs) it is recommended to use a Micro Plus kit.</div>
<ol start="43">
<li>Quantify the amount of RNA recovered using a Qubit instrument or Nanodrop.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A Qubit Fluorometer instrument is more sensitive and preferable for quantifying low amounts of RNA. Alternatively, use a Fragment Analyzer System to assess both quality and quantity.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> RNA can be stored in RLT buffer at −80°C for long-term storage (up to three months).</div>
<ol start="44">
<li>Construct cDNA libraries and perform sequencing using a next-generation sequencer (e.g., Illumina HiSeq 2500) according to the manufacturer’s (<a href="http://www.illumina.com/">http://www.illumina.com/</a>) and the sequence core facility’s instructions.</li>
<li>Check raw data in Fastq format for sample quality and map reads to the human reference genome using the STAR aligner (version 2.4.2a) (<a href="#bib8">Dobin et al., 2013</a>).</li>
<li>Compute the total read counts per gene, e.g., using the Subread package program featureCounts.</li>
<li>Perform variance-stabilizing transformation, e.g., using the Bioconductor package DESeq2 (<a href="#bib15">Love et al., 2014</a>), for normalization and differential-expression analysis comparing PKH26<sup>Negative</sup> cycling cells to PKH26<sup>Positive</sup> label-retaining qCSCs.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>PDOs act as patient-specific avatars echoing the cellular heterogeneity, morphological, mutational, and drug sensitivity status of the original tumor (<a href="#bib22">Roerink et al., 2018</a>; <a href="#bib31">Vlachogiannis et al., 2018</a>; <a href="#bib11">Fujii and Sato, 2021</a>) and thus provide an excellent model for the prospective isolation and profiling of qCSCs.</p>
<p>This protocol provides a straightforward method to identify, isolate and characterize qCSCs from PDOs. Cells isolated using this protocol can be used for functional testing and for whole transcriptome analyses by RNA-sequencing (<a href="#bib20">Regan et al., 2021</a>). The example illustrated in <a href="#fig3">Figure 3</a> shows the frequency of qCSCs that may be expected to be found in a panel of CRC PDOs.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>Perform gene set enrichment analysis using pre-ranked feature of the Broad Institute GSEA software version 2 using MSigDB gene sets (<a href="#bib28">Subramanian et al., 2005</a>; <a href="#bib14">Liberzon et al., 2015</a>). Derive the ranking list from the fold changes calculated from the differential gene expression calculation and nominal p-values. P-values &lt;0.05 are considered as statistically significant. Compute the False Discovery Rate (FDR) for significantly expressed genes using the Benjamini-Hochberg method (<a href="#bib3">Benjamini and Hochberg, 1995</a>). Perform gene ontology enrichment analysis using the Gene Ontology Resource (<a href="http://www.geneontology.org">www.geneontology.org</a>) (<a href="#bib1">Ashburner et al., 2000</a>; <a href="#bib30">The Gene Ontology Consortium, 2018</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>In (<a href="#bib20">Regan et al., 2021</a>), we also generated PDX models from PKH26 labeled cells and identified LRCs in vivo. However, due to their extreme rarity in vivo, we did not perform isolation, re-transplantation or RNA-sequencing of in vivo LRCs. Such analyses would require additional optimization and would be more suitable for scRNA-seq approaches.</p>
<p>The number of label-retaining qCSCs recoverable from PDOs is dependent on individual PDO growth dynamics, the expansion of PDO cultures prior to and after PKH26 labeling, efficient cell staining and the ability to successfully obtain single cell suspensions from Matrigel cultures. In addition, isolating rare cell types by FACS can be time consuming, expensive and result in recovery of insufficient cell numbers for downstream analyses.</p>
<p>qCSCs in untreated tumors have been shown to survive conventional treatments (<a href="#bib10">Francescangeli et al., 2020</a>). However, the protocol presented here does not include steps to induce quiescence through irradiation or chemotherapy, to test the ability of label-retaining qCSCs to survive therapy, or to analyze the effects of therapy on qCSC transcription or other effects, e.g., phenotypic, epigenetic, cell plasticity. For studies examining the effects of therapy on quiescence in organoids see (<a href="#bib16">Pece et al., 2010</a>; <a href="#bib2">Basak et al., 2017</a>; <a href="#bib32">Wu et al., 2017</a>; <a href="#bib18">Puig et al., 2018</a>; <a href="#bib29">Tejero et al., 2019</a>; <a href="#bib9">Endo et al., 2020</a>; <a href="#bib10">Francescangeli et al., 2020</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<p>For troubleshooting problems related to PKH26 dye, see the manufacturer’s Troubleshooting Guide (<a href="https://www.sigmaaldrich.com/deepweb/assets/sigmaaldrich/product/documents/752/321/pkh26glpis.pdf">https://www.sigmaaldrich.com/deepweb/assets/sigmaaldrich/product/documents/752/321/pkh26glpis.pdf</a>).</p>
<h3 id="sec7.1">Problem 1</h3>
<p>Difficulty processing PDO to single cells. (Related to step 5).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Extend TrypLE Express incubation time and volume and increase trituration frequency and intensity.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Variable PKH26 staining between different PDO models and cell types. (Related to steps 8–18).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Variability in staining between PDO models and cell types may result from inter- and intra-tumor heterogeneity in cell adhesion properties, respectively. More adhesive and cell dense PDOs may thus require longer processing times to achieve single cell suspension. In addition, PKH26 staining occurs instantly upon mixing with cells in the salt-free Diluent C vehicle, which is provided to maximize dye solubility and staining efficiency. Therefore, to obtain uniform and reproducible staining it is crucial to 1) ensure that each PDO model has been processed to single cells (extend TrypLE Express incubation time where required), 2) limit any proteins and salts present during the staining step by minimizing any residual cell culture supernatant in step 10, and 3) simultaneously expose all cells to an equal concentration of PKH26 dye by ensuring rapid and homogeneous mixing of cells and dye in step 13.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Uneven distribution and/or clumping of cells in Matrigel domes after plating. (Related to steps 19–21).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>When creating the Matrigel-cell suspension prior to plating, ensure 1) cells are well mixed before adding Matrigel, 2) Matrigel is maintained at 4°C until plated, and 3) mix cells and Matrigel with gentle trituration and avoid formation of air bubbles.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Cell clumping during FACS. (Related to steps 27–29).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Maintain sort samples and collection tubes at 4°C during FACS. Triturate cell suspensions with a P1000 tip and/or pass through a new 70 μm cell strainer to remove clumps and cell debris that cannot be resuspended.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Media evaporating in the wells at the edge of multi-well plates used in limiting dilution colony forming and spheroid assays. (Related to steps 30–40).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>The long culture period required to assess colony and spheroid formation at limiting dilution may result in media loss due to evaporation in the wells closest to the edge of the plates. Such an “edge effect” may lead to differences in colony formation and growth rates between cells in the outer wells and cells in the middle of the plate. To counter this, 1) do not seed cells into edge wells and instead fill them with PBS, 2) maintain humidity of the incubator by keeping the water pan full, 3) regularly replenish media, and 4) use plates with a low evaporation lid and condensation rings.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ashburner, M., Ball, C.A., Blake, J.A., Botstein, D., Butler, H., Cherry, J.M., Davis, A.P., Dolinski, K., Dwight, S.S., Eppig, J.T., et al. (2000). Gene Ontology: tool for the unification of biology. Nat. Genet. <i>25</i>, 25-29. <a class="external-link" href="https://doi.org/10.1038/75556">https://doi.org/10.1038/75556</a></p>
<p id="bib2">Basak, O., Beumer, J., Wiebrands, K., Seno, H., van Oudenaarden, A., and Clevers, H. (2017). Induced quiescence of Lgr5+ stem cells in intestinal organoids enables differentiation of hormone-producing enteroendocrine cells. Cell Stem Cell <i>20</i>, 177-190.e4. <a class="external-link" href="https://doi.org/10.1016/j.stem.2016.11.001">https://doi.org/10.1016/j.stem.2016.11.001</a></p>
<p id="bib3">Benjamini, Y. and Hochberg, Y. (1995). Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Statist Soc. B <i>57</i>, 289-300. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00105-8/sref3">View at publisher</a></p>
<p id="bib4">Blanpain, C. and Simons, B.D. (2013). Unravelling stem cell dynamics by lineage tracing. Nat. Rev. Mol. Cell Biol. <i>14</i>, 489-502. <a class="external-link" href="https://doi.org/10.1038/nrm3625">https://doi.org/10.1038/nrm3625</a></p>
<p id="bib5">Cheung, T.H. and Rando, T.A. (2013). Molecular regulation of stem cell quiescence. Nat. Rev. Mol. Cel. Biol. <i>14</i>, 329-340. <a class="external-link" href="https://doi.org/10.1038/nrm3591">https://doi.org/10.1038/nrm3591</a></p>
<p id="bib6">Cicalese, A., Bonizzi, G., Pasi, C.E., Faretta, M., Ronzoni, S., Giulini, B., Brisken, C., Minucci, S., Di Fiore, P.P., and Pelicci, P.G. (2009). The tumor suppressor p53 regulates polarity of self-renewing divisions in mammary stem cells. Cell <i>138</i>, 1083-1095. <a class="external-link" href="https://doi.org/10.1016/j.cell.2009.06.048">https://doi.org/10.1016/j.cell.2009.06.048</a></p>
<p id="bib7">Dembinski, J.L. and Krauss, S. (2009). Characterization and functional analysis of a slow cycling stem cell-like subpopulation in pancreas adenocarcinoma. Clin. Exp. Metastasis <i>26</i>, 611. <a class="external-link" href="https://doi.org/10.1007/s10585-009-9260-0">https://doi.org/10.1007/s10585-009-9260-0</a></p>
<p id="bib8">Dobin, A., Davis, C.A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., and Gingeras, T.R. (2013). STAR: ultrafast universal RNA-seq aligner. Bioinformatics <i>29</i>, 15-21. <a class="external-link" href="https://doi.org/10.1093/bioinformatics/bts635">https://doi.org/10.1093/bioinformatics/bts635</a></p>
<p id="bib9">Endo, H., Kondo, J., Onuma, K., Ohue, M., and Inoue, M. (2020). Small subset of Wnt-activated cells is an initiator of regrowth in colorectal cancer organoids after irradiation. Cancer Sci. <i>111</i>, 4429-4441. <a class="external-link" href="https://doi.org/10.1111/cas.14683">https://doi.org/10.1111/cas.14683</a></p>
<p id="bib10">Francescangeli, F., Contavalli, P., De Angelis, M.L., Careccia, S., Signore, M., Haas, T.L., Salaris, F., Baiocchi, M., Boe, A., Giuliani, A., et al. (2020). A pre-existing population of ZEB2+ quiescent cells with stemness and mesenchymal features dictate chemoresistance in colorectal cancer. J. Exp. Clin. Cancer Res. <i>39</i>, 2. <a class="external-link" href="https://doi.org/10.1186/s13046-019-1505-4">https://doi.org/10.1186/s13046-019-1505-4</a></p>
<p id="bib11">Fujii, M. and Sato, T. (2021). Somatic cell-derived organoids as prototypes of human epithelial tissues and diseases. Nat. Mater. <i>20</i>, 156-169. <a class="external-link" href="https://doi.org/10.1038/s41563-020-0754-0">https://doi.org/10.1038/s41563-020-0754-0</a></p>
<p id="bib12">Horan, P.K. and Slezak, S.E. (1989). Stable cell membrane labelling. Nature <i>340</i>, 167-168. <a class="external-link" href="https://doi.org/10.1038/340167a0">https://doi.org/10.1038/340167a0</a></p>
<p id="bib13">Hu, Y. and Smyth, G.K. (2009). ELDA: extreme limiting dilution analysis for comparing depleted and enriched populations in stem cell and other assays. J. Immunol. Methods <i>347</i>, 70-78. <a class="external-link" href="https://doi.org/10.1016/j.jim.2009.06.008">https://doi.org/10.1016/j.jim.2009.06.008</a></p>
<p id="bib14">Liberzon, A., Birger, C., Thorvaldsdóttir, H., Ghandi, M., Mesirov, J.P., and Tamayo, P. (2015). The molecular signatures database hallmark gene set collection. Cell Syst. <i>1</i>, 417-425. <a class="external-link" href="https://doi.org/10.1016/j.cels.2015.12.004">https://doi.org/10.1016/j.cels.2015.12.004</a></p>
<p id="bib15">Love, M.I., Huber, W., and Anders, S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. <i>15</i>, 550. <a class="external-link" href="https://doi.org/10.1186/s13059-014-0550-8">https://doi.org/10.1186/s13059-014-0550-8</a></p>
<p id="bib16">Pece, S., Tosoni, D., Confalonieri, S., Mazzarol, G., Vecchi, M., Ronzoni, S., Bernard, L., Viale, G., Pelicci, P.G., and Di Fiore, P.P. (2010). Biological and molecular heterogeneity of breast cancers correlates with their cancer stem cell content. Cell <i>140</i>, 62-73. <a class="external-link" href="https://doi.org/10.1016/j.cell.2009.12.007">https://doi.org/10.1016/j.cell.2009.12.007</a></p>
<p id="bib17">Pleguezuelos-Manzano, C., Puschhof, J., van den Brink, S., Geurts, V., Beumer, J., and Clevers, H. (2020). Establishment and culture of human intestinal organoids derived from adult stem cells. Curr. Protoc. Immunol. <i>130</i>, e106. <a class="external-link" href="https://doi.org/10.1002/cpim.106">https://doi.org/10.1002/cpim.106</a></p>
<p id="bib18">Puig, I., Tenbaum, S.P., Chicote, I., Arqués, O., Martínez-Quintanilla, J., Cuesta-Borrás, E., Ramírez, L., Gonzalo, P., Soto, A., Aguilar, S., et al. (2018). TET2 controls chemoresistant slow-cycling cancer cell survival and tumor recurrence. J. Clin. Invest. <i>128</i>, 3887-3905. <a class="external-link" href="https://doi.org/10.1172/JCI96393">https://doi.org/10.1172/JCI96393</a></p>
<p id="bib19">Regan, J.L., Schumacher, D., Staudte, S., Steffen, A., Haybaeck, J., Keilholz, U., Schweiger, C., Golob-Schwarzl, N., Mumberg, D., Henderson, D., et al. (2017). Non-canonical hedgehog signaling is a positive regulator of the WNT pathway and is required for the survival of colon cancer stem cells. Cell Rep. <i>21</i>, 2813-2828. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2017.11.025">https://doi.org/10.1016/j.celrep.2017.11.025</a></p>
<p id="bib20">Regan, J.L., Schumacher, D., Staudte, S., Steffen, A., Lesche, R., Toedling, J., Jourdan, T., Haybaeck, J., Mumberg, D., Henderson, D., et al. (2021). RNA sequencing of long-term label-retaining colon cancer stem cells identifies novel regulators of quiescence. iScience <i>24</i>, 102618. <a class="external-link" href="https://doi.org/10.1016/j.isci.2021.102618">https://doi.org/10.1016/j.isci.2021.102618</a></p>
<p id="bib21">Regan, J.L. (2022). Immunofluorescence staining of colorectal cancer patient-derived organoids. In Methods in Cancer Stem Cell Biology - Part B. Methods in Cell Biology, I. Vitale, G. Manic, and L. Galluzzi, eds. (Academic Press). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00105-8/sref21">View at publisher</a></p>
<p id="bib22">Roerink, S.F., Sasaki, N., Lee-Six, H., Young, M.D., Alexandrov, L.B., Behjati, S., Mitchell, T.J., Grossmann, S., Lightfoot, H., Egan, D.A., et al. (2018). Intra-tumour diversification in colorectal cancer at the single-cell level. Nature <i>556</i>, 457-462. <a class="external-link" href="https://doi.org/10.1038/s41586-018-0024-3">https://doi.org/10.1038/s41586-018-0024-3</a></p>
<p id="bib23">Santos-de-Frutos, K. and Djouder, N. (2021). When dormancy fuels tumour relapse. Commun. Biol. <i>4</i>, 747. <a class="external-link" href="https://doi.org/10.1038/s42003-021-02257-0">https://doi.org/10.1038/s42003-021-02257-0</a></p>
<p id="bib24">Sato, T., Stange, D.E., Ferrante, M., Vries, R.G., Van Es, J.H., Van den Brink, S., Van Houdt, W.J., Pronk, A., Van Gorp, J., Siersema, P.D., et al. (2011). Long-term expansion of epithelial organoids from human colon, adenoma, adenocarcinoma, and Barrett’s epithelium. Gastroenterology <i>141</i>, 1762-1772. <a class="external-link" href="https://doi.org/10.1053/j.gastro.2011.07.050">https://doi.org/10.1053/j.gastro.2011.07.050</a></p>
<p id="bib25">Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to ImageJ: 25 years of image analysis. Nat. Methods <i>9</i>, 671-675. <a class="external-link" href="https://doi.org/10.1038/nmeth.2089">https://doi.org/10.1038/nmeth.2089</a></p>
<p id="bib26">Schumacher, D., Regan, J.L., Przybilla, D., and Schäfer, R. (2021). Generation of patient-derived colorectal cancer organoids for RAS studies. In Ras Activity and Signaling. Methods in Molecular Biology, I. Rubio and I. Prior, eds. (Humana), pp. 349-360. <a class="external-link" href="https://doi.org/10.1007/978-1-0716-1190-6_22">https://doi.org/10.1007/978-1-0716-1190-6_22</a></p>
<p id="bib27">Schütte, M., Risch, T., Abdavi-Azar, N., Boehnke, K., Schumacher, D., Keil, M., Yildiriman, R., Jandrasits, C., Borodina, T., Amstislavskiy, V., et al. (2017). Molecular dissection of colorectal cancer in pre-clinical models identifies biomarkers predicting sensitivity to EGFR inhibitors. Nat. Commun. <i>8</i>, 14262. <a class="external-link" href="https://doi.org/10.1038/ncomms14262">https://doi.org/10.1038/ncomms14262</a></p>
<p id="bib28">Subramanian, A., Tamayo, P., Mootha, V.K., Mukherjee, S., Ebert, B.L., Gillette, M.A., Paulovich, A., Pomeroy, S.L., Golub, T.R., Lander, E.S., et al. (2005). Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. U S A <i>102</i>, 15545-15550. <a class="external-link" href="https://doi.org/10.1073/pnas.0506580102">https://doi.org/10.1073/pnas.0506580102</a></p>
<p id="bib29">Tejero, R., Huang, Y., Katsyv, I., Kluge, M., Lin, J.Y., Tome-Garcia, J., Daviaud, N., Wang, Y., Zhang, B., Tsankova, N.M., et al. (2019). Gene signatures of quiescent glioblastoma cells reveal mesenchymal shift and interactions with niche microenvironment. EBioMedicine <i>42</i>, 252-269. <a class="external-link" href="https://doi.org/10.1016/j.ebiom.2019.03.064">https://doi.org/10.1016/j.ebiom.2019.03.064</a></p>
<p id="bib30">The Gene Ontology Consortium (2018). The gene ontology resource: 20 years and still GOing strong. Nucleic Acids Res. <i>47</i>, D330-D338. <a class="external-link" href="https://doi.org/10.1093/nar/gky1055">https://doi.org/10.1093/nar/gky1055</a></p>
<p id="bib31">Vlachogiannis, G., Hedayat, S., Vatsiou, A., Jamin, Y., Fernández-Mateos, J., Khan, K., Lampis, A., Eason, K., Huntingford, I., Burke, R., et al. (2018). Patient-derived organoids model treatment response of metastatic gastrointestinal cancers. Science <i>359</i>, 920-926. <a class="external-link" href="https://doi.org/10.1126/science.aao2774">https://doi.org/10.1126/science.aao2774</a></p>
<p id="bib32">Wu, F.-H., Mu, L., Li, X.L., Hu, Y.B., Liu, H., Han, L.T., and Gong, J.P. (2017). Characterization and functional analysis of a slow-cycling subpopulation in colorectal cancer enriched by cell cycle inducer combined chemotherapy. Oncotarget <i>8</i>, 78466-78479. <a class="external-link" href="https://doi.org/10.18632/oncotarget.19638">https://doi.org/10.18632/oncotarget.19638</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Joseph Regan (<a href="mailto:joseph.regan@charite.de">joseph.regan@charite.de</a>)</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This protocol did not in itself generate any data. A recent example of RNA-seq data generated from this technique is publicly available in (<a href="#bib20">Regan et al., 2021</a>) and deposited on ArrayExpress: E-MTAB-8924.</p>
<h3>Acknowledgments</h3>
<p>The author thanks all the coauthors of the original research paper (<a href="#bib20">Regan et al., 2021</a>) for their contributions. The research leading to the development of this protocol received support from the <a href="https://doi.org/10.13039/501100010767">Innovative Medicines Initiative</a> Joint Undertaking under Grant Agreement 115234 (OncoTrack), the resources of which are composed of financial contribution from the European Union Seventh Framework Programme (FP7/2007-2013) and <a href="https://doi.org/10.13039/100013322">EFPIA</a> companies’ in-kind contribution.</p>
<h3>Author contributions</h3>
<p>J.L.R. conceptualized and developed the methodology and wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The author declares no competing interests.</p>
</section>
</article>