<article>
<section>
<h1 id="header">Protocol for transgenerational learned pathogen avoidance behavior assays in <i>Caenorhabditis elegans</i>
</h1>
<p><time datetime="2021-03-12">Published: March 12, 2021</time></p>
<p>Rebecca S. Moore,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Rachel Kaletsky,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a></sup> and Coleen T. Murphy<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular Biology, Princeton University, Princeton, NJ 08544, USA</p>
<p id="aff2"><sup>2</sup>LSI Genomics, Princeton University, Princeton, NJ 08544, USA</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ctmurphy@princeton.edu">ctmurphy@princeton.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100384">10.1016/j.xpro.2021.100384</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Animal experiences, including learned behaviors, can be passed down to several generations of progeny in a phenomenon known as transgenerational epigenetic inheritance. Yet, little is known regarding the molecular mechanisms regulating physiologically relevant transgenerational memories. Here, we present a method for <i>Caenorhabditis elegans</i> in which worms learn to avoid the pathogen <i>Pseudomonas aeruginosa</i> (PA14). Unlike previous protocols, this training paradigm, either using PA14 lawns or through exposure to a PA14 small RNA (P11), induces memory in four generations of progeny.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib8">Moore et al. (2019)</a> and <a href="#bib5">Kaletsky et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for <i>C. elegans</i> transgenerational pathogen avoidance memory training</li>
<li>Trained worms are tested for pathogen avoidance learning using choice assays</li>
<li>Several generations of progeny from trained mothers are propagated and tested</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/515-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the steps for performing a transgenerational aversive learning assay using <i>C. elegans</i>. We have used this protocol to train young adult <i>C. elegans</i> on several bacterial species, including <i>P. aeruginosa</i> (PA14), which induces transgenerational memory, and <i>S. marcescens</i>, which does not. For PA14, the bacterial P11 small RNA induces transgenerational learned avoidance (<a href="#bib5">Kaletsky et al., 2020</a>), and this effect is recapitulated in this assay using transgenic <i>E. coli</i> that express PA14 P11. For this type of training, <i>E. coli</i> carrying an empty vector is used as a control. The desired type of training (PA14 lawns vs. <i>E. coli</i> + P11) should be considered before starting. Regardless of the type of training, the behavior test (choice assay) is performed using OP50 and PA14 lawns.</p>
<ol>
<li>Prepare stocks of unseeded <i>C. elegans</i> maintenance, training, and choice assay plates at least 2 days before beginning the protocol. Nematode Growth Media (NGM) (both 10 cm and 6 cm) and High Growth (HG) (10 cm) plates may be kept at 4°C for up to 2 months before use. Plates that contain antibiotics may be kept for 1 month at 4°C before use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Plates must be dry and at room temperature (20°C–23°C) in order for bacteria to grow and for consistent worm behavior.</div>
<ol start="2">
<li>Inoculate an overnight culture of <i>E. coli</i> OP50 in Luria Broth (LB), as per standard worm maintenance protocols. This stock can be stored at room temperature for several days, or at 4°C for up to 1 week.</li>
<li>Obtain stocks of the remaining bacteria and pathogen strains needed for training and/or choice assays. Bacteria are maintained as glycerol stocks at −80°C. Fresh stocks of bacteria are streaked out on plates and grown overnight (16–22 h) at 37°C. Plates containing single colonies are wrapped with parafilm and kept at 4°C for up to 2 weeks before use (If plates begin to crack, discard earlier). Single colonies are picked from plates to inoculate overnight liquid bacterial cultures (in LB) used throughout the protocol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This protocol is suitable for studying <i>C. elegans</i> learned avoidance and transgenerational inheritance using bacteria other than PA14 (<a href="#bib4">Kaletsky et al., 2019</a>). Adjust bacteria growth conditions as necessary for the strain used.</div>
<div class="note">
<span class="note-title">Note:</span> Healthy stocks of bacteria are critical for optimal worm behavior. The bacteria growth conditions at every step, particularly temperature, can greatly affect the efficacy of worm training.</div>
<ol start="4">
<li>Worms must be properly maintained for several generations before starting this protocol, and throughout the transgenerational experiment. Stressful conditions, including starvation or overcrowding, may affect naïve worm bacteria preference, learned behavior (<a href="#bib6">Kauffman et al., 2011</a>), and transgenerational memory (<a href="#bib3">Houri-Ze’evi et al., 2019</a>). If worms experience stress, thaw fresh stocks and/or maintain worms under standard conditions before resuming this protocol.</li>
<li>Approximately 3 HG maintenance plates containing gravid worms are used on day 2 of this protocol. Plan accordingly using general <i>C. elegans</i> maintenance protocols so that gravid worms are healthy and available at this time. The starting population of synchronized progeny (approximately 3,000 embryos, at minimum) is sufficient for training on both control and experimental bacteria, testing of P0 animals for behavior, and propagation of progeny for testing subsequent generations. Adjust the number starting animals as needed for testing additional training conditions, or when using worm genotypes with reduced fertility.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> <i>Pseudomonas aeruginosa</i> is a Biosafety Level 2 (BSL2) pathogen. Follow your institutional guidelines for handling and safety of PA14 and other pathogenic bacteria.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>E. coli:</i> OP50</td>
<td>Caenorhabditis Genetics Center</td>
<td>CGC: OP50</td>
</tr>
<tr>
<td>
<i>P. aeruginosa</i>: PA14</td>
<td>Z. Gitai</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>E. coli</i> expressing P11</td>
<td>C.T. Murphy</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>E. coli</i> expressing empty vector</td>
<td>C.T. Murphy</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>Bioworld</td>
<td>Cat #41920044-3</td>
</tr>
<tr>
<td>Potassium hydroxide</td>
<td>Fisher Scientific</td>
<td>Cat #P258-212</td>
</tr>
<tr>
<td>Sodium hypochlorite</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #SS290-1</td>
</tr>
<tr>
<td>Peptone</td>
<td>Fisher Scientific</td>
<td>Cat #DF0118-07-2</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Sigma-Aldrich</td>
<td>Cat #S9888</td>
</tr>
<tr>
<td>Bacto-Agar</td>
<td>Fisher Scientific</td>
<td>Cat #DF0140-07-4</td>
</tr>
<tr>
<td>L-Arabinose</td>
<td>Bioworld</td>
<td>Cat #40100301-3</td>
</tr>
<tr>
<td>Potassium phosphate monobasic</td>
<td>Fisher Scientific</td>
<td>Cat #7778-77-0</td>
</tr>
<tr>
<td>Potassium phosphate dibasic</td>
<td>Millipore Sigma</td>
<td>Cat #7758-11-4</td>
</tr>
<tr>
<td>Magnesium sulfate anhydrous</td>
<td>Fisher Scientific</td>
<td>Cat #7487-88-9</td>
</tr>
<tr>
<td>Calcium chloride</td>
<td>Fisher Scientific</td>
<td>Cat #AC349615000</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>Fisher Scientific</td>
<td>Cat# DF0127-17-9</td>
</tr>
<tr>
<td>Tryptone</td>
<td>Biosciences</td>
<td>Cat #211705</td>
</tr>
<tr>
<td>Sodium phosphate dibasic anhydrous</td>
<td>Fisher Scientific</td>
<td>Cat #7558-79-4</td>
</tr>
<tr>
<td>Cholesterol</td>
<td>Fisher Scientific</td>
<td>Cat #C314-500</td>
</tr>
<tr>
<td>Carbenicillin</td>
<td>Sigma-Aldrich</td>
<td>Cat #C4316</td>
</tr>
<tr>
<td>Isopropyl β-d-1-thiogalactopyranoside (IPTG)</td>
<td>Fisher Scientific</td>
<td>Cat #BP1755</td>
</tr>
<tr>
<td>Tetracycline</td>
<td>Sigma-Aldrich</td>
<td>Cat #87128</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain N2 var. Bristol: wild-type</td>
<td>Caenorhabditis Genetics Center</td>
<td>CGC: N2</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Prism 8</td>
<td>GraphPad Prism</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p><b>Equipment</b>: This protocol uses a 37°C temperature-controlled shaker to grow bacteria overnight, as well as both a 20°C incubator (non-shaking) and 25°C incubator (non-shaking) to maintain <i>C. elegans</i> and incubate bacteria, respectively. A dissecting light microscope is used to monitor worm growth and to count worms following the choice assay. Additionally, an electronic repeat pipettor is used for preparing choice assay plates.</p>
<p><b>Data Analysis Program</b>: All data is recorded in Microsoft Excel and graphed using GraphPad Prism.</p>
<table id="undtbl1">
<caption>Bleach solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M KOH pH 6</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Sodium hypochlorite</td>
<td>6 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>41.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p>Bleach solution can be kept at 4°C or at room temperature (20°C–23°C), and can be kept indefinitely, but its potency tends to decrease over time. It is recommended to replace the bleach solution after 1 month.</p>
<table id="undtbl2">
<caption>Nematode growth media plate</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Peptone</td>
<td>0.025%</td>
<td>2.5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>0.03%</td>
<td>3 g</td>
</tr>
<tr>
<td>Bacto-agar</td>
<td>0.17%</td>
<td>17 g</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td> </td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<p>Add a stir bar prior to autoclaving Nematode Growth Media (NGM). Following sterilization, media is left until cool enough to touch. Right before pouring plates, 1 mL/L cholesterol (5 mg/mL in ethanol), 1 mL/L 1 M CaCl<sub>2</sub>, 1 mL/L 1 M MgSO<sub>4</sub>, and 25 mL/L 1 M potassium phosphate (pH 6.0) are added to molten agar. For NGM + Arabinose plates, supplement media with 100 μg/mL carbenicillin, and 0.2% w/v arabinose. Poured and solidified plates are stored at 4°C until use. Plates without antibiotics can be maintained at 4°C for up to 2 months, while plates containing antibiotics can be maintained for up to 1 month at 4°C before use.</p>
<table id="undtbl3">
<caption>High Growth Media/High Growth Media RNAi plates</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Peptone</td>
<td>0.20%</td>
<td>20 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>0.03%</td>
<td>3 g</td>
</tr>
<tr>
<td>Bacto-agar</td>
<td>0.30%</td>
<td>30 g</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td> </td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<p>Add a stir bar prior to autoclaving High Growth Media (HG). Following sterilization, media is left until cool enough to touch. Right before pouring plates, 4 mL/L cholesterol (5 mg/mL in ethanol), 1 mL/L 1 M CaCl<sub>2</sub>, 1 mL/L 1 M MgSO<sub>4</sub>, and 25 mL/L 1 M potassium phosphate (pH 6.0) are added to molten agar. Poured and solidified plates are stored at 4°C until use. For HG RNAi plates, also supplement media with 100 μg/mL carbenicillin and 1 mL/L 1 M IPTG. HG plates without antibiotics can be maintained at 4°C for up to 2 months before use, while plates containing antibiotics can be maintained for up to 1 month at 4°C before use.</p>
<table id="undtbl4">
<caption>Other solutions</caption>
<thead>
<tr>
<th>Name</th>
<th>Reagents</th>
</tr>
</thead>
<tbody>
<tr>
<td>Luria Broth (LB)</td>
<td>0.1% w/v tryptone, 0.05% w/v yeast extract, 0.1% w/v NaCl in Milli-Q water, autoclave, store at room temperature</td>
</tr>
<tr>
<td>M9</td>
<td>0.03% w/v KH<sub>2</sub>PO<sub>4,</sub> 0.06% w/v Na<sub>2</sub>HPO<sub>4</sub>, 0.05% w/v NaCl, 1 mL/L 1 M MgSO<sub>4</sub> in Milli-Q water, filter sterilize, store at room temperature</td>
</tr>
<tr>
<td>Calcium chloride</td>
<td>1 M in Milli-Q water, filter sterilize, store at room temperature</td>
</tr>
<tr>
<td>Magnesium sulfate</td>
<td>1 M in Milli-Q water, filter sterilize, store at room temperature</td>
</tr>
<tr>
<td>Cholesterol</td>
<td>5 mg/mL in ethanol, filter sterilize, store at room temperature</td>
</tr>
<tr>
<td>Potassium phosphate</td>
<td>1 M in Milli-Q water, filter sterilize, store at room temperature</td>
</tr>
<tr>
<td>Carbenicillin</td>
<td>100 μL/mL in Milli-Q water, filter sterilize, store in −20°C</td>
</tr>
<tr>
<td>Arabinose</td>
<td>15% w/v in Milli-Q water, filter sterilize, store at room temperature</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>400 mM in water, store at room temperature. (TOXIC)</td>
</tr>
<tr>
<td>Isopropyl β-d-1-thiogalactopyranoside (IPTG)</td>
<td>1 M in Milli-Q water, filter sterilize, store in −20°C0.1 M in Milli-Q water, filter sterilize, store in −20°C</td>
</tr>
<tr>
<td>Tetracycline</td>
<td>12.5 μg/mL in Milli-Q water, filter sterilize, store in −20°C</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Sodium azide is toxic both as a powder and in solution. Sodium azide exists as an odorless white solid. Once mixed with water, sodium azide changes to a toxic gas. Be careful when preparing this solution, and always prepare sodium azide solutions in a fume food.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Bacteria can be spotted onto choice assay plates using a non-repeating pipette.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Inoculate bacteria in liquid culture for training plates and seed worm maintenance plates: day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>Bacterial colonies from freshly streaked plates are used to inoculate liquid culture. These overnight cultures will be used in the next step to seed plates used to train worms. Additionally, worm growth plates will be seeded using a healthy stock of liquid OP50.</p>
<ol>
<li>Inoculate 6 mL of LB media with bacteria from single colonies using a sterile inoculating loop.
<ol type="a">
<li>Adjust the amount of LB inoculated based on the number of training plates required. 1 mL will be used for each training plate. To test P0 worms and subsequent generations, prepare enough bacteria for 6 training plates per bacteria and condition (i.e., worm genotype) tested.</li>
<li>LB is suitable for growth of most bacteria. Use appropriate growth media for strains, as needed.</li>
<li>
<i>E. coli</i> expressing P11/empty vector are grown in LB + 100 ug/mL carbenicillin + 0.2% (w/v) arabinose.</li>
<li>Use an appropriately sized tube to ensure proper aeration during growth. A 14 mL round bottom conical is suitable for 6 mL of LB.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Worms can be raised on RNA interference (RNAi) to knock down individual gene expression. To adapt this protocol for RNAi use, inoculate LB + 100 ug/mL carbenicillin + 12.5 ug/mL tetracycline with a colony or glycerol stock from a sequence-verified RNAi clone. This step is performed on day 0 (1 day prior to the start of the experiment).</div>
<ol start="2">
<li>Grow bacteria at 37°C overnight (16–22 h), shaking at 250 rpm.
<ol type="a">
<li>Bacteria should be grown for a minimum of 8 h, but not to exceed more than 16 h. If a biofilm forms (PA14), do not use the culture.</li>
</ol>
</li>
<li>Seed HG 10 cm plates with 1 mL of OP50 grown from “<a href="#before-you-begin">before you begin</a>” (or RNAi bacteria on HG RNAi plates, if using) per plate. These will be used to bleach worms onto for day 2 of the experiment.
<ol type="a">
<li>Seed three 10 cm HG plates per genotype.</li>
<li>OP50 plates need to be seeded with sufficient time to ensure they are completely dry, but not so old that these plates crack (at minimum one day before use).</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> For RNAi use, seed HG RNAi plates with RNAi cultures prepared on day 0.</div>
<div class="note">
<span class="note-title">Note:</span> A stock of pre-prepared LB + 100 ug/mL carbenicillin + 0.2% (w/v) arabinose can be maintained at 4°C for up to 1 week, or prepared fresh each time.</div>
<h3 id="sec3.2">Synchronize worms, prepare training plates, and inoculate bacteria for choice assay plates: day 2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>In this step, healthy adult worms are bleached to obtain a synchronized population of eggs. These animals will be used on day 4 for training. The plates required for training will also be prepared. Finally, liquid cultures of bacteria will be inoculated (separate from the cultures on day 1) that will used to seed choice assays plates on day 3.</p>
<ol start="4">
<li>Synchronize worm population: Bleach worms to be used for experiments onto HG OP50-seeded plates (or HG RNAi-seeded plates, if using) (∼250–350 eggs per plate).
<ol type="a">
<li>Wash gravid worms off 3 HG plates with M9 and pour (or use wide-orifice pipette to collect) M9 + worms into a 15 mL conical.</li>
<li>Allow worms to settle (do not centrifuge). To obtain enough eggs for the experiment, the resulting worm pellet should be roughly 300 μL.</li>
<li>Remove supernatant using a vacuum, making sure not to disturb the worm pellet.</li>
<li>Add 5 mL of bleach solution.</li>
<li>Place tube onto nutator to shake.</li>
<li>Check on worms every 5 min for the first 15 min. Once worms start to dissolve, check every 2 min until the majority of worm bodies have dissolved (about 99% of the worms should be dissolved).</li>
<li>Using a table top centrifuge, spin bleached worms for 1 min at 160 × <i>g</i> (1,000 rpm in a clinical centrifuge). There will be a small white pellet at the bottom of the tube containing embryos.</li>
<li>Use a vacuum to remove all of the bleach solution.</li>
<li>Add 2 mL of M9 and spin embryos again for 1 min at 1,000 rpm.</li>
<li>Remove supernatant and repeat wash 2 times.</li>
<li>Plate eggs (250–350 eggs) onto each of the 3 OP50-seeded HG plates prepared on day 1. This is performed by estimating the density of eggs in M9 by counting 2 μL of the egg suspension on a slide using a dissecting microscope.</li>
<li>Place synchronized embryos on HG plates into a slightly opened container and in a 20°C incubator.</li>
<li>Incubate worms for 2 days at 20°C for use on day 4.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> For RNAi use, HG RNAi plates seeded on day 1 are induced with IPTG before use. Using sterile conditions, place a total of 200 μL of 0.1 M IPTG in small spots all over the HG RNAi lawn. Allow IPTG to dry before plating bleached eggs as in step 4 (∼1 h).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Bleaching is usually performed on the morning of day 2 <u>(48–52 h before training) to allow sufficient time for worm development</u>. Worms that are too young during training (pre-L4 stage) on PA14 will arrest and are not suitable for the standard assay described here. Additionally, arrested animals will not make progeny, interfering with the transgenerational portion of this assay. Bleaching in the morning is recommended so that worms are the appropriate age on the morning of day 4 of the experiment, and allows the steps on day 5 to be performed during convenient daytime hours.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Worms should never be centrifuged prior to the addition of bleach. Allow worms to pellet by gravity on the bench. Centrifugation can cause physical damage/induce stress.</div>
<ol start="5">
<li>Prepare training plates: For each genotype, obtain the following numbers of plates.
<ol type="a">
<li>OP50 = at least four 10 cm NGM plates per genotype of worms being tested.</li>
<li>PA14 = at least four 10 cm NGM plates per genotype of worms being tested.</li>
<li>
<i>E. coli</i> expressing P11 or empty vector = at least four 10 cm plates (10 cm NGM + 100 ug/mL carbenicillin + 0.2% w/v arabinose) per genotype of worms being tested.</li>
<li>Seed OP50 plates with 1 mL of liquid OP50 culture (prepared from “<a href="#before-you-begin">before you begin</a>” section) per plate.</li>
<li>Seed <i>E. coli</i> expressing P11 or empty vector plates with 1 mL of overnight culture (prepared from step 1) per plate.</li>
<li>Seed PA14 plates with 1 mL of diluted overnight culture PA14 per plate (prepared from step 1), as follows.
<ol type="i">
<li>Use a nanodrop or other spectrophotometer to determine OD of PA14 liquid culture.</li>
<li>Dilute the overnight culture of PA14 to an OD<sub>600 nm</sub> = 1 using LB.</li>
<li>For a healthy liquid PA14 culture (<a href="#fig1">Figure 1</a>), this dilution corresponds to about 1 mL of PA14 overnight culture into 3 mL of LB.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/515-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Appearance of liquid bacteria cultures</div>
<p>Overnight cultures of OP50 (left) and PA14 (undiluted, right). PA14 grown in the proper condition appears blue when aerated.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> If the PA14 bacteria are too concentrated, motile spots and biofilms will form. These conditions will kill worms very rapidly.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Training plates need to be fully seeded with bacteria so that there are no unseeded areas of the plate. Worms on pathogenic bacteria may move to areas without bacteria if there are empty spots, which could affect training.</div>
</li>
</ol>
</li>
<li>Put all plates right side up into a large plastic container and into a 25°C incubator for 2 days. Do not seal the box containing the plates. These plates must fully dry, otherwise worms will swim and/or arrest. Allowing the plates to come to room temperature (20°C–23°C) before seeding with bacteria helps ensure drying.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>To prepare bacteria for choice assay plates, inoculate bacteria in liquid culture as in step 1.
<ol type="a">
<li>Calculate the amount of liquid culture needed to seed choice assay plates. 25 μL of each bacteria (OP50 and diluted PA14) is used for each spot per plate.</li>
<li>Maintain stock of OP50 as needed.</li>
<li>Inoculate pathogenic bacteria (from single colonies on streaked out plates) in LB or appropriate growth media (i.e., for <i>E. coli</i> expressing P11/Empty vector).</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Prepare choice assay plates: day 3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="7">
<li>Take the appropriate number of 60 mm NGM plates out of fridge and let warm to room temperature (20°C–23°C) before use.
<ol type="a">
<li>Use at least 10 choice assay plates per condition tested.</li>
<li>Prepare several extra choice assay plates to replace any that need to be discarded due to contamination or irregular spread of bacterial spots.</li>
</ol>
</li>
<li>Arrange the 60 mm plates in stacks of 5 upside down (bottoms up).</li>
<li>Using a black permanent marker, dot the bottom center of each plate (for OP50), and dot the top center of the plate with another color (for PA14/pathogen).
<ol type="a">
<li>Dots should be as far apart as possible on the plates. See <a href="#fig2">Figure 2</a>.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/515-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Choice assay plate setup</div>
<p>Left, illustration of choice assay plate. X = origin, yellow = OP50 spot, Blue = pathogen spot (PA14). Right, plate seeded with PA14 and OP50 and incubated for 2 days at 25°C. PA14 spot appears darker than OP50, but may not appear blue.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Turn plates right side up.</li>
<li>For PA14, dilute bacteria as on day 2 (step 5f).</li>
<li>Using a repeating pipette with a sterile tip, carefully pipette 25 μL of OP50 onto the agar where the black dot is marked.</li>
<li>Using a new sterile tip, pipette 25 μL of other bacteria (for PA14, used diluted sample) onto the other dot.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When preparing choice assay plates, bacteria spots should not be plated so close to the edge of the plate that worms will be forced onto the vertical walls of the plate.</div>
<div class="note">
<span class="note-title">Note:</span> For experiments in which worms are raised on RNAi bacteria, the choice assay remains the same (OP50 vs. PA14).</div>
<ol start="14">
<li>Allow plates to dry at room temperature (20°C–23°C) before moving them (∼30 min).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Spots of bacteria may shift if the plates are moved before they are dry, causing greater and/or uneven distribution of bacteria on the choice assay plates. Discard any plates in which the bacteria are present outside of the designated spot.</div>
<ol start="15">
<li>Move dry plates face up into a plastic container and place in a 25°C incubator for 2 days.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Choice assay plates must be moved to an incubator with a constant temperature of 25°C. It is important that both the training plate and choice assay plates are prepared in this manner.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If transgenerational inheritance of learned behavior will be tested in the F1 generation, use 1 mL of OP50 to seed each of two 10 cm HG plates per strain and condition being tested. Incubate plates at room temperature for use on day 5 of the protocol. Seed additional plates if more F1 progeny are required.</div>
<h3 id="sec3.4">Train worms for aversive learning: day 4</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>L4 worms from day 2 (48–52 h after bleaching) are washed from the growth plates and transferred to training plates. If worms were bleached on the morning of day 2, the following steps for day 4 are performed during the morning or early afternoon. Worms are then incubated for 24 h for training.</p>
<ol start="16">
<li>Take training plates out of 25°C incubator and let come to room temperature (20°C–23°C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If training worms on <i>E. coli</i> expressing P11, both empty vector (control) and P11 training plates need to be induced with 0.1% arabinose prior to use. Place a total of 200 μL of 0.1% (w/v) arabinose in small spots all over training plates. Allow arabinose to fully dry at room temperature prior to continuing to step 17 (∼ 1 h).</div>
<ol start="17">
<li>Observe the worms intended for training—<b>approximately 90% of the population must be at least at the L4 stage.</b>
</li>
<li>Wash worms using M9 into a 15 mL conical tube, and let them settle by gravity.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> <u>NEVER</u> centrifuge worms to pellet them (<a href="#bib7">Kauffman et al., 2010</a>). Centrifugation can interfere with behavior.</div>
<ol start="19">
<li>Remove supernatant, leaving some M9 behind (about 50 μL of extra M9 per 100 μL of worms).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not leave worms in M9 for extended periods of times, since this can lead to starvation/stress. Worms should be washed and plated quickly.</div>
<ol start="20">
<li>Plate worms onto training plates.
<ol type="a">
<li>Use your index finger to lightly tap the worms to resuspend them.</li>
<li>Using wide-orifice tips, and making sure not to puncture the agar, pipette 10 μL of worms onto OP50 or transgenic <i>E. coli</i> plates (any more than this and the worms will starve by the end of the 24 h training period).</li>
<li>Using wide-orifice tips, pipette 10 μL of worms onto pathogenic bacteria plates – (worms will not run out of PA14 by 24 h. Also, the recovery from PA14 plates is lower since the lawns are very thick).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Results do not differ when more worms are trained on each PA14 training plate. Up to 40 μL of worms can be used per PA14 plate, without affecting the results compared to training equal numbers of worms on OP50 and PA14 plates. Increasing the number of worms per PA14 plate improves worm recovery for choice assays and bleaching following training.</div>
<div class="note">
<span class="note-title">Note:</span> When pipetting worms onto training plates, pipette from the middle of the worm suspension. This will help ensure an equal number of worms on each plate. Worms settle rapidly. Tap tube as needed to keep worms resuspended.</div>
<ol start="21">
<li>Allow the M9 spot of worms to dry, and put plates into a 20°C incubator with the lid of the container cracked.</li>
<li>Do not put PA14 plates in the same box as OP50, since volatile odorants from PA14 may interfere with worm training.</li>
<li>Incubate plates at 20°C for 24 h. The training time must be precise.</li>
</ol>
<h3 id="sec3.5">Aversive learning behavior assay: day 5</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–6 h</div>
<p>In this step, a subset of the trained worms is washed from the training plates and used to test the behavior of the population using the prepared choice assays. The remaining trained worms are used to collect the next generation of progeny.</p>
<ol start="24">
<li>Take choice assay plates out of the 25°C incubator at least 1 h before use.</li>
<li>Set aside a subset of worms on training plates to be used for collection of the next generation (F1). Half of the worms (2 of the 4 training plates) are used during step 37 to obtain F1 synchronized embryos via bleaching. These plates are left at 20°C until step 37, which should be performed as close to the end of the 24 h training period as possible.
<ol type="a">
<li>The worms used for behavior testing on choice assay plates are <i>not</i> the same worms used to bleach for F1 collection, but they do derive from the same batch of trained animals.</li>
</ol>
</li>
<li>For the choice assay, use M9 to wash worms off of the remaining training plates and into a 15 mL conical, being careful not to carry over large chunks of bacteria. Wash worms 2 more times to ensure no food from the training plates is carried over. If there is food in the washed worm pellet, worms will not move from the origin.</li>
<li>Allow worms to pellet by gravity, do not disrupt worm pellet.</li>
<li>Lay out choice assay plates and remove lids.</li>
<li>While worms are settling, place 1 μL of sodium azide (used at 400 mM, diluted in water) onto the center of each bacterial spot.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not spot choice assay plates with sodium azide until immediately before use. Sodium azide should be put onto the bacteria spots right before trained worms are washed off the plate in preparation for the assay. Sodium azide is used to paralyze worms so that the worm’s first choice is recorded. Left too long on plates, sodium azide will diffuse broadly, and this will cause worms to be paralyzed far away from the bacterial spots. Worms are counted if they are within a few millimeters of the bacterial spot. The remaining worms are censored. If the majority of the worms are paralyzed outside of a few millimeters of the bacterial spots, the entire plate is censored. (If the experiment is performed as described, censoring occurs in &lt;1% of all choice assay plates, and &lt;1% of the worms on those rare plates are censored. The number of censored animals should be recorded during data collection.)</div>
<ol start="30">
<li>Label bottoms of plates or place a piece of tape by the plates to label the genotype and condition being testing.</li>
<li>Carefully remove supernatant from washed worms.
<ol type="a">
<li>Leave some M9 on top of pellet (∼50 μL of M9 per 100 μL of worms).</li>
</ol>
</li>
<li>Using your index finger, gently tap the worm pellet into suspension immediate before pipetting worms.</li>
<li>Using a wide-orifice pipette tip, place 5 μL of the resuspended worm pellet on the bottom of the plate (origin), midway between the two bacteria spots (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/515-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Initial appearance of the choice assay plate</div>
<p>Choice assay plate shortly after worms are placed at origin.</p>
</figcaption>
</figure>
<ol type="a">
<li>When sticking your pipette tip into the worm pellet, take worms from the top of the resuspension. This will help ensure that a reasonable number of worms are being measured in the choice assay.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For the choice assay plates, aim for 75–200 worms per plate. Overcrowding on choice assay plates may affect behavior, and makes counting more challenging.</div>
<ol start="34">
<li>Close lids and wait 1 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The 1 h incubation for the choice assay is the preferred time to proceed to step 37, where worms on the remaining training plates are bleached to obtain the F1 progeny.</div>
<ol start="35">
<li>Count number of worms on each spot (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/515-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Choice assay plate 1 h following the addition of worms</div>
<p>All of the worms are paralyzed on either the PA14 or OP50 spot, which measures the worms’ first choice. This is in contrast to other pathogen learning assays that allow worms to continually move on the plate (<a href="#bib9">Ooi and Prahlad, 2017</a>).</p>
</figcaption>
</figure>
<ol type="a">
<li>Remove lid and place plate upside down (with bottom of plate facing up) on a dissecting microscope. Manually count each worm, marking off each worm that is already counted using a fine-tip marker.</li>
<li>Record the number of worms at each spot.</li>
<li>Do not count worms that are burrowed into the agar.</li>
<li>Do not count worms that remain at the origin.</li>
<li>If the bacterial spots are too close to the sides of the plate, plates may need to be tilted to observe worms that are paralyzed near the wall.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Plates should be counted at the end of the 1 h behavior assay. Worms left too long on plates in the presence of azide will explode, which will interfere with counting.</div>
<ol start="36">
<li>Calculate Choice Index.</li>
<li>To collect the progeny of trained worms, bleach the remaining trained animals (as described for day 2), and plate eggs onto OP50-seeded HG plates.</li>
<li>Place plates with eggs in a 20°C incubator. Incubate 72 h until worms develop to day 1 of adulthood.</li>
<li>To test for transgenerational memory in F1 animals, repeat protocol starting at day 2, omitting the step for training worms.</li>
<li>P0 worms trained on PA14 or P11 small RNA pass on the transgenerational memory through the F4 generation compared to controls (<a href="#bib5">Kaletsky et al., 2020</a>; <a href="#bib8">Moore et al., 2019</a>). The F1 through F4 generation are each grown on OP50-seeded HG plates until day 1 of adulthood, at which time a subset of worms is tested for behavior and a separate subset is bleached to obtain the next generation. Since N2 animals produce more progeny than needed for behavior assays and worm propagation, the remaining worms are discarded. Repeat protocol for additional generations, as needed.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A successful aversive learning assay will show a significant change in <i>C. elegans</i> food choice following training on PA14 or <i>E. coli</i> + P11 compared to controls. The actual choice index will vary between biological and technical replicates. The choice index is dependent on the initial PA14 attraction of the worm (naïve preference), and the number of worms that ultimately change their preference after training. The naïve preference and magnitude of learning depend on the strain of bacteria used, genotype of the worm used, and the generation being tested.</p>
<p>The naïve preference (choice index) of wild-type worms (N2) ranges between (-0.2 – 0.1), depending of the health of the worms and environmental/laboratory conditions. The choice index of P0 worms trained on PA14 usually ranges from (0.4 – 1), and transgenerational memory is typically lower (range of 0.2 – 0.4) from F1 – F4 (<a href="#bib5">Kaletsky et al., 2020</a>; <a href="#bib8">Moore et al., 2019</a>). The magnitude of learning from P11 training in P0 worms is (0.1 – 0.4), and this level is maintained through F4 (<a href="#bib5">Kaletsky et al., 2020</a>).</p>
<p>In the experiment shown below (<a href="#fig5">Figure 5</a>), wild-type (N2) worms were trained on <i>E. coli</i> OP50 (control) or <i>P. aeruginosa</i> (PA14) for 24 h. Following exposure to PA14, <i>C. elegans</i> exhibit PA14 avoidance compared to control trained worms. Avoidance is maintained in the F1 generation. Similar results are obtained when worms are trained on <i>E. coli</i> expressing P11 compared to a control (<a href="#fig5">Figure 5</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/515-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Pathogen avoidance learning in mothers and F1 progeny</div>
<p>Mothers (P0) exposed to PA14 or P11 learn to avoid PA14 after PA14 (left), or P11 training (right). Progeny (F1) of both PA14 and P11 trained mothers inherit learned avoidance of PA14. This avoidance persists through the F4 generation. Box-and-Whiskers Plot. The box extends from the 25<sup>th</sup> to 75<sup>th</sup> percentiles, with whiskers from the minimum to the maximum values. Two-way ANOVA, Tukey’s multiple comparison test. ∗p ≤ 0.05, ∗∗∗∗p ≤ 0.0001.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Formulas:</p>
<p class="math-ml"><math> <mrow> <mtext>Choice Index</mtext> <mo>=</mo> <mfrac> <mrow> <mo>(</mo> <mtext>Number of worms on OP</mtext> <mn>50</mn> <mo>−</mo> <mtext>Number of worms on PA</mtext> <mn>14</mn> <mo>)</mo> </mrow> <mrow> <mo>(</mo> <mrow> <mtext>Total number of worms on OP</mtext> <mn>50</mn> <mo>+</mo> <mtext>PA</mtext> <mn>14</mn> </mrow> <mo>)</mo> </mrow> </mfrac> </mrow> </math></p>
<p class="math-ml"><math> <mrow> <mtext>Learning Index</mtext> <mo>=</mo> <mtext> naive choice index</mtext> <mo>−</mo> <mtext>trained choice index</mtext> </mrow> </math></p>
<p>Criteria for plate inclusion/exclusion: Individual aversive learning assay plates were excluded if there was any contamination on the plate, sodium azide was not placed on bacterial lawn spots, or the total number of worms on plates is less than 20 worms.</p>
<p>Each experiment is performed at least 3 times. Data from all replicates is pooled together for statistical analysis. Data from 1 replicate is shown in <a href="#fig5">Figure 5</a>.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is highly effective at measuring transgenerational changes in <i>C. elegans</i> food preference following exposure to pathogenic bacteria or bacteria-derived small RNAs. However, food choice in this assay is determined exclusively in populations of animals, and not on individuals. The ability of individual animals to learn, and the transgenerational duration of memory as set by parental experiences are interesting avenues of research (<a href="#bib2">Houri-Zeevi et al., 2020</a>) and may require alternative assays for measuring individual worm behavior (<a href="#bib1">Ha et al., 2010</a>).</p>
<p>The number of worms that can be analyzed for learned pathogen avoidance is limited by worm counting on choice assay plates, as this step of the protocol is performed manually. The choice assays performed here differ from odorant-based chemotaxis assays in <i>C. elegans</i>. Automated counting of population chemotaxis assays is possible, since worms are paralyzed at spots on plates containing only odorant and azide, and not thick bacterial lawns (<a href="#bib6">Kauffman et al., 2011</a>). In the food choice assay presented here, worms are paralyzed upon reaching either the OP50 or PA14 spots. The PA14 spots are slightly thicker and darker in color than OP50, and the contrast between the worms and the lawn is reduced. These factors make automated detection of worms on PA14 more challenging.</p>
<p>The protocol is highly amenable to testing different mutant <i>C. elegans</i> strains, either by genetic lesion or RNAi knockdown of gene expression. However, the ability to measure pathogen avoidance learning and transgenerational memory in mutant animals depends on the naïve food preference of the mutant (<a href="#bib5">Kaletsky et al., 2020</a>; <a href="#bib8">Moore et al., 2019</a>). Compared to wild-type animals, mutants with a significantly high naïve avoidance for PA14 may not exhibit learning after training, and this may reflect the role of a particular gene in neuronal development and/or health that is not directly related to acquired pathogen avoidance behavior. Results from mutants with high naïve PA14 avoidance should be interpreted with caution.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Worms are paralyzed too far from the bacterial spot during choice assay (day 5, step 35), as demonstrated in <a href="#fig6">Figure 6</a>.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/515-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Example of choice assay plate in which sodium azide was added too early before the addition of worms</div>
<p>Worms are paralyzed far from the OP50 and PA14 spots, as shown by the presence of worms between the black and red outlines.</p>
</figcaption>
</figure>
<h3 id="sec7.2">Potential solution</h3>
<p>Discard choice assay plate. Do not include in data collection. Repeat choice assay by placing azide directly on bacterial spot immediately before use.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Naïve preference for wild-type worms (N2) is high (day 5, step 35).</p>
<h3 id="sec7.4">Potential solution</h3>
<p><i>C. elegans</i> behavior can be sensitive to environmental and laboratory conditions, including heat, high humidity, and odors. Exposure to any of these conditions during the choice assay can lead to aberrant results, including high naïve PA14 avoidance.</p>
<p>Every 6 months, it is recommended that worm stocks are replaced with a freshly thawed stock. Over time, the naïve preference of worm populations may drift consistently higher, which may interfere with behavior performance and/or the magnitude of learning.</p>
<p>Worms may be injured during handling and pipetting. Whenever handling worms, use a wide-orifice pipette tip and do not roughly handle tubes containing worms. In order to resuspend worms, flick the tube VERY LIGHTLY with your finger. Do not ever pipette up and down to resuspend worms.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Worms burrow into training plates or choice assay plates (day 5, step 33).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Be careful not to puncture the agar when seeding plates or when placing worms on plates.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Worms remain at the origin during the choice assay (day 5, step 26, 35).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Bacteria was transferred from training plates onto the choice assay plates during worm collection and plating. Be careful not to transfer chunks of bacteria from training or hold plates as the worms are washed off. Wash worms thoroughly with M9 to remove bacteria carried over from training plates.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>PA14-exposed worms have few or no eggs when bleached on day 5 following training (day 4, step 17).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Worms were too young (pre-L4 stage) when placed on PA14 training plates, causing worms to arrest. Repeat experiment and wait until the majority of worms are at the L4 stage before transferring to training plates.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Worms are thrashing in liquid on plates at the end of the 24 h training period (day 4, step 21).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Discard training plate. The training plates were too wet when worms were plated, or too much M9 was transferred with the worms during plating. Repeat using training plates that are fully dry, and/or transfer less liquid during worm plating.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>Embryos do not hatch or worms are asynchronous following bleaching (day 2, step 4).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Worms have been over bleached (left in the bleach solution too long). Repeat experiment with a new starting population of worms and reduce the time that worms spend in the bleach solution before washing eggs.</p>
<h3 id="sec7.15">Problem 8</h3>
<p>Bacterial or fungal contamination is present on worm maintenance plates, training plates, or choice assay plates.</p>
<h3 id="sec7.16">Potential solution</h3>
<p>Any type of contamination can affect bacteria choice preference. If contamination is observed, do not use worms from these plates for your experiment. Worms can either be bleached to remove contamination, or a new starting population of worms can be obtained.</p>
<h3 id="sec7.17">Problem 9</h3>
<p>Synchronized worms growing on maintenance plates from step 4 have crawled onto the side or lid of Petri dishes.</p>
<h3 id="sec7.18">Potential solution</h3>
<p>Worms are starved and should not be used for an experiment. Worms can be bleached and maintained for several generations before use, or a new population of worms should be obtained.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ha, H., Hendricks, M., Shen, Y., Gabel, C.V., Fang-Yen, C., Qin, Y., Colón-Ramos, D., Shen, K., Samuel, A.D.T., and Zhang, Y. (2010). Functional organization of a neural network for aversive olfactory learning in <i>Caenorhabditis elegans</i>. Neuron <i>68</i>, 1173-1186. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref1">View at publisher</a></p>
<p id="bib2">Houri-Zeevi, L., Korem Kohanim, Y., Antonova, O., and Rechavi, O. (2020). Three rules explain transgenerational small RNA inheritance in <i>C. elegans</i>. Cell <i>182</i>, 1186-1197.e12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref2">View at publisher</a></p>
<p id="bib3">Houri-Ze’evi, L., Teichman, G., Gingold, H., and Rechavi, O. (2019). Stress resets transgenerational small RNA inheritance. bioRxiv. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref3">View at publisher</a></p>
<p id="bib4">Kaletsky, R., Moore, R.S., Parsons, L.L., and Murphy, C.T. (2019). Cross-kingdom recognition of bacterial small RNAs induces transgenerational pathogenic avoidance (preprint). Genetics. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref4">View at publisher</a></p>
<p id="bib5">Kaletsky, R., Moore, R.S., Vrla, G.D., Parsons, L.R., Gitai, Z., and Murphy, C.T. (2020). <i>C. elegans</i> interprets bacterial non-coding RNAs to learn pathogenic avoidance. Nature <i>586</i>, 445-451. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref5">View at publisher</a></p>
<p id="bib6">Kauffman, A., Parsons, L., Stein, G., Wills, A., Kaletsky, R., and Murphy, C. (2011). <i>C. elegans</i> positive butanone learning, short-term, and long-term associative memory assays. J. Vis. Exp. 2490. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref6">View at publisher</a></p>
<p id="bib7">Kauffman, A.L., Ashraf, J.M., Corces-Zimmerman, M.R., Landis, J.N., and Murphy, C.T. (2010). Insulin signaling and dietary restriction differentially influence the decline of learning and memory with age. PLoS Biol. <i>8</i>, e1000372. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref7">View at publisher</a></p>
<p id="bib8">Moore, R.S., Kaletsky, R., and Murphy, C.T. (2019). Piwi/PRG-1 Argonaute and TGF-β mediate transgenerational learned pathogenic avoidance. Cell <i>177</i>, 1827-1841.e12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref8">View at publisher</a></p>
<p id="bib9">Ooi, F.K. and Prahlad, V. (2017). Olfactory experience primes the heat shock transcription factor HSF-1 to enhance the expression of molecular chaperones in <i>C. elegans</i>. Sci. Signal. <i>10</i>, eaan4893. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00091-5/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Coleen T. Murphy (<a href="mailto:ctmurphy@princeton.edu">ctmurphy@princeton.edu</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new reagents.</p>
<h4>Data and Code Availability</h4>
<p>Original data for figures in the paper is available (<a href="https://doi.org/10.1038/s41586-020-2699-5">https://doi.org/10.1038/s41586-020-2699-5</a>).</p>
<h3>Acknowledgments</h3>
<p>We thank the <i>C. elegans</i> Genetics Center for strains, Z. Gitai for bacteria strains, and BioRender for figure-generating software. C.T.M. is the Director of the Glenn Center for Aging Research at Princeton and an HHMI-Simons Faculty Scholar. This work was supported by a Pioneer Award to C.T.M. (<a href="https://doi.org/10.13039/100000057">NIGMS</a> DP1GM119167), the <a href="https://doi.org/10.13039/100001642">Glenn Foundation for Medical Research</a> (GMFR CNV1001899), the HHMI-Simons Faculty Scholar Program (AWD1005048), and T32GM007388 (NIGMS) support of R.S.M.</p>
<h3>Author Contributions</h3>
<p>R.K., R.S.M., and C.T.M. designed experiments. R.K. and R.S.M. performed experiments and analyzed data. R.K., R.S.M., and C.T.M. wrote the manuscript. C.T.M. supervised the project and acquired funding.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>