<article>
<section>
<h1 id="header">Isolation and culturing of primary mouse astrocytes for the analysis of focal adhesion dynamics</h1>
<p><time datetime="2021-12-10">Published: December 10, 2021</time></p>
<p>Baran E. Güler,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Jacek Krzysko,<sup><a href="#aff1">1</a></sup> and Uwe Wolfrum<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Molecular Physiology, Molecular Cell Biology, Johannes Gutenberg University of Mainz, Hanns-Dieter-Hüsch-Weg 17, 55128 Mainz, Germany</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:wolfrum@uni-mainz.de">wolfrum@uni-mainz.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100954">10.1016/j.xpro.2021.100954</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Primary astrocytes have gained attention as an important model for <i>in vitro</i> biological and biochemical research in the last decades. In this protocol, we describe a fast and cost-effective technique for isolating, culturing, and maintaining primary mouse astrocytes at ∼ 80% purity levels, which can be used in <i>in vitro</i> studies for migration and focal adhesion dynamics. In addition, we present an optimized transfection and manual quantification approach for focal adhesion analysis in fixed and living cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref12">Kusuluri et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>High purity of primary mouse astrocyte isolation without commercial kits</li>
<li>Isolated mouse primary astrocytes are functional for downstream applications</li>
<li>Quantitative analysis of focal adhesion properties in fixed and living astrocytes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Dissection of mice and preparation of brain culture should be performed under sterile conditions. All equipment (Biosafety cabinet, forceps, scissors and plates, etc.) should be sterile to prevent any contamination. In this protocol, the isolation of primary astrocytes cells depends on the differential binding method which separates primary astrocytes, oligodendrocyte progenitor cells (OPCs) and microglia cells based on their different affinities for bacterial grade plastic dishes. Do not use plates that have been treated for animal cell culture because all cell types will attach to these plates with equal affinities. Primary astrocytes were isolated from brains of P0 C57BL/6J (wild type). This protocol was also successfully applied for two mouse models for Usher syndrome 2C namely <i>Vlgr1</i>/del7TM (<a href="#bib14">McMillan and White, 2004</a>) and <i>Drum B</i> mutated mice (<a href="#bib18">Potter et al., 2016</a>), bred on a C57BL/6J background and we did not observe any difference in protocol efficiency. All experiments described herein were performed in accordance with the guidelines provided by Association for Research in Vision and Ophthalmology.</p>
<p>Before starting the experiment several chemicals and surgical instruments should be prepared (For detailed information please see <a href="#key-resources-table">key resources table</a> and <a href="#materials-and-equipment">materials and equipment</a> section).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-GFAP (Glial fibrillary acidic protein), working dilution 1 in 1000</td>
<td>DAKO-Agilent</td>
<td>Cat#ZO334</td>
</tr>
<tr>
<td>Mouse monoclonal anti-GFAP (Glial fibrillary acidic protein), working dilution 1 in 1000</td>
<td>Sigma-Aldrich</td>
<td>Cat#G3839</td>
</tr>
<tr>
<td>Guineapig monoclonal anti-MAP2 (Microtubule associated protein), working dilution 1 in 1000</td>
<td>Synaptic systems</td>
<td>Cat#188004</td>
</tr>
<tr>
<td>Mouse monoclonal anti-Vinculin, working dilution 1 in 100</td>
<td>Sigma-Aldrich</td>
<td>Cat#V9131</td>
</tr>
<tr>
<td>Rabbit polyclonal anti- PDGFR-α, working dilution 1 in 200</td>
<td>Santa Cruz Biotechnology, Inc.</td>
<td>Cat#sc-338</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD68, working dilution 1 in 200</td>
<td>Bio-Rad Laboratories</td>
<td>Cat#MCA1957</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Alexa Fluor 488-conjugated goat anti-rabbit IgG, working dilution 1 in 400</td>
<td>Molecular Probes</td>
<td>Cat#A-11034</td>
</tr>
<tr>
<td>Alexa Fluor 568-conjugated goat anti-rat IgG, working dilution 1 in 400</td>
<td>Biotrend</td>
<td>Cat#20092-1</td>
</tr>
<tr>
<td>Alexa Fluor 640-conjugated goat anti-mouse IgG, working dilution 1 in 200</td>
<td>Biotrend</td>
<td>Cat#20177</td>
</tr>
<tr>
<td>Phalloidin-TRITC, working dilution 1 in 400</td>
<td>Sigma-Aldrich</td>
<td>Cat#P1951</td>
</tr>
<tr>
<td>Dulbecco`s Modified Eagle Medium (DMEM)</td>
<td>Gibco<sup>TM</sup>
</td>
<td>Cat#31966-021</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Gibco<sup>TM</sup>
</td>
<td>Cat#16000044</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>Gibco<sup>TM</sup>
</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>10× Hanks’ Balanced Salt Solution (HBSS)</td>
<td>Gibco<sup>TM</sup>
</td>
<td>Cat#14065-056</td>
</tr>
<tr>
<td>1 M HEPES solution</td>
<td>Gibco<sup>TM</sup>
</td>
<td>Cat#15630106</td>
</tr>
<tr>
<td>DNase I, recombinant, Grade II, from bovine pancreas</td>
<td>Merck</td>
<td>Cat#10104159001</td>
</tr>
<tr>
<td>0.01% Poly-L-Lysine solution</td>
<td>Sigma-Aldrich</td>
<td>Cat#25988-63-0</td>
</tr>
<tr>
<td>0.05% Trypsin-EDTA (1×)</td>
<td>Gibco<sup>TM</sup>
</td>
<td>Cat#25300-054</td>
</tr>
<tr>
<td>1×PBS</td>
<td>Gibco<sup>TM</sup>
</td>
<td>Cat#14190-094</td>
</tr>
<tr>
<td>Opti-MEM™ I Reduced Serum Medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#31985062</td>
</tr>
<tr>
<td>GeneJuice® Transfection Reagent</td>
<td>Merck</td>
<td>Cat#70967-6</td>
</tr>
<tr>
<td>Fibronectin bovine plasma</td>
<td>Merck</td>
<td>Cat#F4759-5MG</td>
</tr>
<tr>
<td>Albumin crude from chicken egg = Ovalbumin</td>
<td>PanReac AppliChem</td>
<td>Cat#A4344</td>
</tr>
<tr>
<td>Gelatin From Cold Water Fish Skin, 40–50% in Water</td>
<td>Sigma-Aldrich</td>
<td>Cat#G7765-250ML</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Raw and analyzed data</td>
<td>This paper and <a href="#bib12">Kusuluri et al. (2021)</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: C57BL/6J (P0 age)</td>
<td>The Jackson Laboratory</td>
<td>Stock no: 000664</td>
</tr>
<tr>
<td>Mouse: <i>Vlgr1</i>/del7TM (P0 age)</td>
<td>Breed on a C57BL/6 background</td>
<td><a href="#bib14">McMillan and White (2004)</a></td>
</tr>
<tr>
<td>Mouse: <i>Drum</i> B (P0 age)</td>
<td>Breed on a C57BL/6 background</td>
<td>
<a href="#bib18">Potter et al. (2016)</a> and <a href="#bib12">Kusuluri et al. (2021)</a>
</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>RFP-Paxillin</td>
<td>Rudolf E. Leube, Rick Horwitz</td>
<td>Rudolf E. Leube, Rick Horwitz</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ software</td>
<td>National Institutes of Health (NIH)</td>
<td><a href="http://imagej.nih.gov/ij/download.html">http://imagej.nih.gov/ij/download.html</a></td>
</tr>
<tr>
<td>Focal adhesion analysis server</td>
<td><a href="https://faas.bme.unc.edu">https://faas.bme.unc.edu</a></td>
<td><a href="#bib2">Berginski and Gomez (2013)</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Stereomicroscope</td>
<td>Leica Microsystems</td>
<td>Leica WILD M3B</td>
</tr>
<tr>
<td>Cold light source</td>
<td>Leica Microsystems</td>
<td>Schott KL750</td>
</tr>
<tr>
<td>Inverted microscope</td>
<td>Nikon Instruments Inc.</td>
<td>Nikon Eclipse Ti2-E</td>
</tr>
<tr>
<td>Spinning disc unit</td>
<td>Yokogawa Electric Corporation</td>
<td>CSU-W1</td>
</tr>
<tr>
<td>Leica confocal microscope</td>
<td>Leica Microsystems</td>
<td>DM6000B</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Eppendorf</td>
<td>Centrifuge 5430 R</td>
</tr>
<tr>
<td>Medium-sized scissors</td>
<td>A. Dumont &amp; Fils</td>
<td>Cat#T5074</td>
</tr>
<tr>
<td>Dumont #3 curved forceps</td>
<td>A. Dumont &amp; Fils</td>
<td>Cat#T504</td>
</tr>
<tr>
<td>Dumont #4 standard tip forceps</td>
<td>A. Dumont &amp; Fils</td>
<td>Cat#T505</td>
</tr>
<tr>
<td>Dumont #7 standard tip forceps</td>
<td>A. Dumont &amp; Fils</td>
<td>Cat#T508</td>
</tr>
<tr>
<td>Vannas spring scissors</td>
<td>A. Dumont &amp; Fils</td>
<td>Cat#T5322</td>
</tr>
<tr>
<td>100 mm petri dish (cell culture)</td>
<td>Greiner Bio-One</td>
<td>Cat#664160</td>
</tr>
<tr>
<td>100 mm bacterial grade petri dish</td>
<td>SARSTEDT AG &amp; Co. KG</td>
<td>Cat#82.1472</td>
</tr>
<tr>
<td>T-75 culture flasks</td>
<td>Greiner Bio-One</td>
<td>Cat#658170</td>
</tr>
<tr>
<td>6-well culture plates</td>
<td>Greiner Bio-One</td>
<td>Cat#657160</td>
</tr>
<tr>
<td>15 mL Centrifuge Tubes</td>
<td>SARSTEDT AG &amp; Co. KG</td>
<td>Cat#62.554.502</td>
</tr>
<tr>
<td>10 mm glass coverslips</td>
<td>Carl Roth GmbH +Co. KG</td>
<td>Cat#YX02.1</td>
</tr>
<tr>
<td>0.22 μm filter</td>
<td>Carl Roth GmbH +Co. KG</td>
<td>Cat#KH541</td>
</tr>
<tr>
<td>0.45 μm filter</td>
<td>Carl Roth GmbH +Co. KG</td>
<td>Cat#CCX9.1</td>
</tr>
<tr>
<td>μ-Slide 4 Well chamber</td>
<td>Ibidi GmbH</td>
<td>Cat#80426</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Growth medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dulbecco‘s Modified Eagle Medium (DMEM)</td>
<td>n/a</td>
<td>450 mL</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>2%</td>
<td>11 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>511 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Final antibiotic concentration may be adjusted, since different antibiotic concentrations can affect cell proliferation and differentiation.</div>
<div class="note">
<span class="note-title">Note:</span> Growth medium should be kept at +4°C for a maximum of 1 month. If there is a pH level change during storage which can be distinguished by a color change, new medium should be prepared.</div>
<table id="undtbl2">
<caption>1× HBSS for brain tissue storage during dissection</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× HBSS</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>1 M HEPES</td>
<td>0.01 M</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>890 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Adjust pH 7.0 of buffer using 1 N HCl or 1 N NaOH and sterile medium by passing through a 0.22 μm filter.</div>
<div class="note">
<span class="note-title">Note:</span> Medium should be stored at +4°C for a maximum of 1 month. If there is a pH level change during storage which can be distinguished by the color change in medium, the new medium should be prepared.</div>
<table id="undtbl3">
<caption>DNase I solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNase I</td>
<td>0.05%</td>
<td>100 mg</td>
</tr>
<tr>
<td>1× HBSS</td>
<td>n/a</td>
<td>200 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Do not dissolve the DNase I by using vortex as mechanical force may denaturate the enzyme which can cause decrease in the activity. The solution should be prepared by pipetting and sterilized by passing through a 0.22 μm filter. To prevent any freeze/thaw cycles which may reduce activity, 1–5 mL aliquots can be prepared and kept at –20°C or –80°C for a maximum of 6 months.</div>
<table id="undtbl4">
<caption>Blocking solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ovalbumin</td>
<td>0.1%</td>
<td>0.1 g</td>
</tr>
<tr>
<td>40–50% Fish gelatin</td>
<td>0.5%</td>
<td>1.1 g</td>
</tr>
<tr>
<td>1× PBS</td>
<td>n/a</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Mix the Ovalbumin and 40%–50% Fish gelatin in 100 mL PBS for 12–16 h. The day after, bring the temperature of the solution to 50°C. Centrifuge for 10 min at 3000 RCF to remove undissolved particles. Sterile medium by passing through a 0.45 μm filter. In order to prevent freeze/thaw cycles, prepare 1–2 mL aliquots and keep at −20°C for a maximum of 12 months.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of primary brain culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–15 days</div>
<p>Before starting dissection of mice pups, place 10 mL of 1× HBSS (in a 15 mL Falcon tube), 1 mL of 0.05% DNase I and 1 mL of 0.05% trypsin on ice and arrange dissection tools, microscope, a cold light source and 70% ethanol under the dissection hood. This step includes dissection and preparation of primary brain cultures. To obtain a confluent layer of primary astrocytes cells, collect 3–4 pups’ brains in a 15 mL tube containing 1× HBSS solution.</p>
<h3 id="sec3.2">Preparation of brain tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15–30 min</div>
<ol>
<li>Before starting the dissection, gently hold the mice pup and spray the neck with 70% of ethanol to prevent contamination.</li>
<li>Decapitation of mice can be done by using sharp scissors in one cut.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Brain isolation can take time due to inexperience. In order to achieve high efficiency after the procedure and healthy brain culture, only one decapitation should be applied at a time and the remaining decapitation procedures should be carried out after step 14 is finished from the first mouse.</div>
<ol start="3">
<li>To keep the head fixed and stabile use anchoring forceps at orbital cavities (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Dissection of the brain from PN0 mouse</div>
<p>(A and B) (A) Cut should be performed to remove the skin and (B) skull layers covering the brain. Dashed lines indicate the cuts using Vannas spring scissors. The cut starts from the neck where the vertebral foramen is located and extends anteriorly to the nose.</p>
<p>(C) Exposed brain. Scale bar: 5 mm.</p>
</figcaption>
</figure>
</li>
<li>Make an incision of the skin at the hindbrain and follow along the midline to eye level where transverse cuts to the eye cavity will be made (<a href="#fig1">Figure 1</a>B).</li>
<li>Grasp the skin from both sides by using Dumont #5 forceps and pull it aside to reveal the skull (<a href="#fig1">Figure 1</a>C).</li>
<li>Carefully cut the cranium using a small scissors. The cut should start from the neck where the vertebral foramen is located and extended anteriorly to the nose. Make additional transverse cuts to the eye cavity.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> While cutting the cranium be sure that no excess pressure is applied to the brain. This may cause deformation in brain tissue and bleeding, which can lower culture purity because of contamination with red blood cells and endothelial cells.</div>
<ol start="7">
<li>Carefully pull apart the cranium using forceps to reveal the brain.</li>
<li>Take out the brain with the help of a small spatula. First, place the spatula under the cerebellum and gently push it towards the olfactory bulbs and scoop out by lifting. This will disconnect the brain from the skull base.</li>
<li>Transfer the brain to a 10 cm petri dish containing ice-cold 1× HBSS under sterile conditions. The brain should submerge completely in the liquid.</li>
<li>Remove olfactory bulbs in order to free the meninges. Meninges can be distinguished by their reddish color resulting from blood vessels (<a href="#fig2">Figures 2</a>A and 2B).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Dissection of mouse brain at P0</div>
<p>(A and B) Removing the meninges from mouse brain. (A) Dorsal view (B) and ventral view of isolated mouse brain. Meninges can be distinguished by reddish color in the cortex of the mouse brain. (A′ and B′) Meninges removal from the cortex of the mouse brain. Arrows indicate removed meninges from the cortex of the mouse brain. (A″ and B″) Dorsal and ventral view of completely meninges-free brain.</p>
<p>(C) Dissection of brain parts with a sharp razor blade or fine forceps. Scale bars: A and B: 10 mm; C: 5 mm.</p>
</figcaption>
</figure>
</li>
<li>Reversing the brain and starting from the bottom side can help to remove meninges easily. Dissect the meninges from the surface of the cortical layer by using fine forceps.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We observed that keeping brain integrity during the process may reduce the risk of meningeal cells and fibroblast contamination and ensure full removal.</div>
<ol start="12">
<li>Flip the brain and continue meninges removal.</li>
<li>Carefully check whether there is any meninges residue and if not, cut the brain into 6–10 pieces using forceps or sharp blades in order to increase disassociation efficiency (<a href="#fig2">Figure 2</a>C).</li>
<li>Place the brain pieces into a 15 mL tubes that contain 10 mL 1× HBSS and keep on ice (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Dissociation of mouse brain tissue for primary cell cultivation</div>
<p>(A) Brain tissue dissected from wild type mice in 1× HBSS.</p>
<p>(B) Homogenized mouse brain tissue (MBT).</p>
<p>(C) MBT pellet after centrifugation.</p>
<p>(D) MBT pellet after removing supernatant.</p>
<p>(E) Resuspended MBT pellet in complete growth medium.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Only 3–4 brains should be collected in 10 mL 1× HBSS. If more brain tissue is needed, use a 50 mL Falcon and increase the volume of 1× HBSS for 5 brains to 11 mL and for 12 brains 20 mL. If the brain pieces are not submerged completely, tap the bottom of the Falcon tube. This may help to submerge brain pieces that float on top.</div>
<div class="note">
<span class="note-title">Note:</span> After obtaining meninges-free brain, repeat the steps 1–14 for the rest of the mice. Doing only one brain at a time will ensure healthy cultures.</div>
<ol start="15">
<li>Under the sterile condition, add ice cold 0.05% DNase I into the tube and gently triturate brain tissue 10–15 times with a 10 mL and 2 mL pipette, respectively.</li>
<li>Afterwards, add 0.05% trypsin, mix by using 10 mL pipette 20–30 times and incubate at room temperature (22°C–25°C) for 20 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The amount of DNase I and trypsin can be adjusted depending on the number of brains in one tube. For 3–4 brains; 1 mL of DNase I and 1 mL of trypsin, for 5 brains; 1.5 mL of DNase I and 1.5 mL of trypsin, for 12 brains; 3 mL of DNase I and 3 mL of trypsin should be added.</div>
<ol start="17">
<li>During incubation time, prepare one T-75 flask for 3–4 brains and coat with 3 mL of 0.01% Poly-L-Lysine (PLL) for 20 min. Place the T-75 flask in the incubator.</li>
<li>20 min after incubation of brain pieces mix the Falcon tube thoroughly (<a href="#fig3">Figure 3</a>B).</li>
<li>Centrifuge for 10 min at 150 RCF to the pellet brain tissue (<a href="#fig3">Figure 3</a>C).</li>
<li>After centrifugation, carefully aspirate the supernatant using a Pasteur pipette connected to a pump (<a href="#fig3">Figure 3</a>D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If there is any meninges residue or blood vessel contamination, which can be distinguished by a reddish color on top of the pellet, also aspirate and discard this layer. Resuspend pellet with 2 mL of complete growth medium and repeat centrifugation step.</div>
<ol start="21">
<li>Resuspend cell pellet with 2–4 mL of DMEM with a 10 mL pipette and mix thoroughly 10–20 times in order to obtain a single-cell suspension (<a href="#fig3">Figure 3</a>E).</li>
<li>Aspirate PLL from the T-75 flask and rinse with 10 mL of DMEM to completely wash out any residue. Add 8–10 mL of pre-warmed 37°C DMEM supplemented with 10% FBS and 2% pen/strep.</li>
<li>Plate the resuspended cell pellet and incubate at 37°C with 5% CO<sub>2</sub> in an incubator.</li>
<li>Change the medium on day 1, day 2 and day 7 after plating the primary cells (<a href="#fig7">Figure 7</a>A).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In order to remove unattached and death cells wash the plate by using 1× PBS, before adding fresh complete growth medium<b>.</b>
</div>
<h3 id="sec3.3">Deriving pure primary mouse astrocyte culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7–15 days</div>
<p>This step will be used for the separation of primary astrocytes cells from oligodendrocyte progenitor cells (OPCs), microglia and neurons. In confluence, their tight arrangement at the bottom of the culture flask can distinguish primary astrocytes cells from other cell types. Widefield microcopy distinguishes microglia cells from other brain-derived cells revealing a bright and small rounded cell morphology.</p>
<h3 id="sec3.4">Removal of oligodendrocyte progenitor cells (OPCs)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–20 min</div>
<ol start="25">
<li>Mixed primary cell cultures which were obtained in steps 1 to 24 reach full confluency on days 10–15. When mixed cell culture reach confluency, gently tap the flask from both sides several times in order to detach the OPCs layer. After tapping the flask, check the cells under the microscope. Detach OPCs can be distinguished as floating cells in the plate (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Detached oligodendrocyte progenitor cells after tapping the flask</div>
<p>OPCs are distinguished as floating cells above the monolayer of astrocytes and microglia. Arrow heads indicate round shaped floating OPCs. Scale bars: 100 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In case of incomplete detachment repeat tapping several times. Tapping should be done gently so as not detach the astrocyte layer.</div>
<ol start="26">
<li>After the OPCs are released aspirate the medium completely to remove detached cells and rinse remaining cells carefully with 10 mL of complete growth medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To avoid any loosening or loss of astrocyte layer, medium aspiration and rinsing should be done carefully. A 10 mL pipette should be placed at the corner of the flask and the medium should be slowly added against the wall of the flask and not directly onto the attached cells.</div>
<ol start="27">
<li>After removing OPCs, add 10 mL of complete growth medium and ice cold 0.5 mL DNase I to loosen the remaining cell layer and incubate 5 min at 37°C with 5% CO<sub>2</sub> in an incubator.</li>
<li>After DNase I treatment, the remaining OPCs are released by horizontal shaking the T-75 Flask 20–30 times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> While too gentle shaking may cause insufficient OPCs removal, too intense shaking can damage the astrocyte layer. Shaking should be performed at a steady speed by hand.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Shaking can cause foaming; therefore, the shaken flask should be held vertically for 30 s to allow the foam to float on the top.</div>
<h3 id="sec3.5">Removal of microglia</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–2.30 h</div>
<ol start="29">
<li>Aspirate the medium which may still contains OPCs, and wash the astrocytes and microglia layer with 3–4 mL PBS.</li>
<li>Add 3–4 mL of 0.05% trypsin and incubate 3–5 min at 37°C with 5% CO<sub>2</sub> in an incubator.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cells should not be incubated with trypsin for more than 5 min. Longer incubation time may harm primary astrocytes and results in an unhealthy culture.</div>
<ol start="31">
<li>After incubation, check the cells under the microscope. If there is incomplete detachment, tap the slide of the flask several times and check under the microscope again.</li>
<li>After trypsinization, add 6–7 mL of complete growth medium and resuspend the cells.</li>
<li>Transfer the suspension to a new 15 mL Falcon tube and pellet detached cells by centrifugation for 10 min at 150 rcf.</li>
<li>During centrifugation, prepare 4 sterile 10-cm bacterial grade plates. Add 2 mL of complete growth medium in each of 4 Petri dishes.</li>
<li>Aspirate the supernatant which you obtain after step 33 and resuspend cells with 4 mL of complete growth medium.</li>
<li>Add 1 mL of the cell suspension to each petri dish (<a href="#fig5">Figure 5</a>A) and incubate at 37°C with 5% CO<sub>2</sub> in the incubator for 20 min (<a href="#fig5">Figure 5</a>B).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Differential surface binding of primary astrocytes from mouse brain</div>
<p>(A) Whole brain cell suspension which obtained in step 35.</p>
<p>(B) Red arrow heads indicate settled microglia after 20 min of incubation.</p>
<p>(C) Transferring the supernatant which obtained in step 37 to a new bacterial grade plate.</p>
<p>(D) Red arrows indicate settled microglia after 90 min of incubation. Scale bar: 50 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this step, remaining microglia cells should adhere to the bacterial grade plate while primary astrocyte cells should not.</div>
<ol start="37">
<li>After incubation, collect all supernatants containing astrocytes of all 4 Petri dishes and pool them to a new 10-cm bacterial grade plate (<a href="#fig5">Figure 5</a>C).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The transfer of supernatant should be performed carefully as shaking of the plate may result in loosening bound microglia. Gently tilt the plate and collect the supernatant from the side of the plate.</div>
<ol start="38">
<li>Incubate the supernatant (obtained in step 37) contains ∼10–12 mL medium at 37°C with 5% CO<sub>2</sub> in the incubator for 90 min (<a href="#fig5">Figure 5</a>D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After incubation, check the cells under the microscope. Microglia cells should completely adhere after this step and primary astrocytes can be distinguished as floating cells in the culture medium.</div>
<ol start="39">
<li>Transfer the supernatant containing primary astrocytes into a new T-75 flask.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> According to volume of the obtained supernatant, add additional amount of pre-warmed complete growth medium to the T-75 flask to fill up to 20 mL.</div>
<ol start="40">
<li>Incubate the primary astrocytes for 7–15 days at 37°C with 5% CO<sub>2</sub> in the incubator. Primary astrocytes are reaching confluency in between day 7–15 after isolation. After culture reach confluency, primary astrocytes can be used for downstream experiments (<a href="#fig7">Figure 7</a>B).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Change the complete growth medium by aspirating half of the old medium (approx. 10 mL) and replacing it with a fresh, pre-warmed complete growth medium (approx. 10 mL) every 4 days during cultivation. This will provide you with a conditioned medium that will help maintain a healthy culture.</div>
<h3 id="sec3.6">Transfection of isolated primary astrocytes and live-cell imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>In these steps, we describe how to transfect primary mouse astrocytes with a focal adhesion (FA) construct namely RFP-Paxillin and how to perform live imaging to investigate FA dynamics. For live-cell imaging, we use a μ-Slide 4 Well chamber (Ibidi, Munich, Germany) and for transfection, we use GeneJuice® transfection reagent (Merck, Darmstadt, Germany).</p>
<ol start="41">
<li>First, coat the μ-Slide 4 Well chamber with 300 μL of fibronectin 5 μg/mL dissolved in dH<sub>2</sub>O for 1 h at 37°C.</li>
<li>Next, remove the fibronectin from the wells and wash the wells once with 500 μL of sterile 1× PBS or water. After the washing, place the chamber in the incubator in order to dry it.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Fibronectin solution can be used up to 3 times, therefore discarded fibronectin can be store at −20°C for further usage.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If the purpose of the experiment is to study migration or focal adhesion dynamics, we strongly recommend the usage of fibronectin instead of poly-L-Lysine coating, since fibronectin may enhance cell adhesion and spreading (<a href="#bib3">Blau 2013</a>).</div>
<ol start="43">
<li>Wash the primary astrocytes when reach confluency which are cultured in T-75 flask with 1× PBS and add 3 mL of 0.05% trypsin for 5 min at 37°C to detach cells.</li>
<li>Collect the detached cells in a 15 mL tube and add 1–5 mL of growth media for preparing cell suspension. Count the number of cells and prepare 5 × 10<sup>4</sup> cell/mL suspension.</li>
<li>Add 700 μL (∼35,000 cells) of cell suspension into each well and incubate 1 day at 37°C with 5% CO<sub>2</sub> in the incubator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cell number can be adjusted based on chamber size used. Cells should be at least 50% confluent before transfection.</div>
<ol start="46">
<li>The day after incubation, change the medium to fresh 700 μL of complete growth medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Transfection reagent volumes and DNA amount can be scaled up if more than 4 wells are needed.</div>
<ol start="47">
<li>For the single transfection of focal adhesion construct, prepare 1 Eppendorf tube and add 100 μL of Opti-MEM<sup>TM</sup> and add 6 μL of GeneJuice®. Vortex the solution thoroughly to mix it and incubate at room temperature for 5 min.</li>
<li>Add 2 μL of focal adhesion construct at a 1 μg/mL concentration into the serum-free/ GeneJuice® mixture and mix it by tapping the tubes and incubate at room temperature for 15 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not vortex the solution after adding the DNA construct, vigorous mixing can cause low transfection efficiency.</div>
<ol start="49">
<li>Drop-wisely add 27 μL of reagent/DNA mixture to the cells and rock the chamber to homogenize the solution in the complete growth medium.</li>
<li>After 24 h of transfection remove the old medium which contains transfection reagent and add pre-warmed complete growth medium and incubate the cells for additional 24 h at 37°C with 5% CO<sub>2</sub> in the incubator.</li>
<li>Before starting live-cell imaging, turn on the temperature and CO<sub>2</sub> controller of Nikon Eclipse Ti2-E/Yokogawa CSU-W1 Spinning disk microscope and wait till the condition stabilizes for usage.</li>
<li>Acquire the movies at 1 frame/3–5 min with 20 z-stacks at 0.5 μm step size for 1.25 h to 3-h time course (<a href="#mmc1">Methods video S1</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Time interval between frames and z-stacks size and steps can be adjusted based on your experimental setup. After determining the parameter, all movies should be acquired using the same settings in order to provide experimental consistence.</div>
<ol start="53">
<li>Apply maximum projections to the movies and analyze focal adhesion assembly/disassembly rates by using Focal Adhesion Analysis Server (FAAS) (<a href="#fig10">Figure 10</a>) (<a href="#bib2">Berginski and Gomez 2013</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Microtubule induced focal adhesion disassembly experiment can be used as a control experiment in order to establish the protocol (<a href="#bib8">Ezratty et al., 2005</a>).</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Time-lapse images of RFP-Paxillin transfected BL/6 mouse primary astrocyte cell, related to step 52</div>
<p>Images were acquired for 85 min with 3 min time intervals. Live-cell imaging performed under 5% CO<sub>2</sub> and at 37°C using Nikon Eclipse Ti2-E equipped with a spinning disc microscope. Scale bar: 20 μm.</p>
</figcaption>
</figure>
<h3 id="sec3.7">Immunostaining of focal adhesion molecules in fixed primary astrocytes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>We used a Leica DM6000B microscope (Leica, Bensheim, Germany) for imaging of specimens.</p>
<ol start="54">
<li>Prepare 10 mm glass coverslips in 6-well plates and coat with fibronectin as described in step 41–42.</li>
<li>Seed 1–1.5 × 10<sup>5</sup> cells in each well and incubate the primary astrocytes cells for 48 h at 37°C with 5% CO<sub>2</sub> in the incubator.</li>
<li>After the 48 h incubation, carefully remove the growth medium and wash one time rapidly with 1× PBS and do second time washing using 1× PBS for 10 min in room temperature.</li>
<li>Prepare a parafilm sheet and place it a wet chamber for keeping humidity during incubation times. Take the coverslips out from wells, place onto parafilm sheet and fix the cells by using 50 μL of 2% of PFA in 1× PBS for each coverslip for 10 min at room temperature.</li>
<li>Wash the cells 2 times for 10 min with 1× PBS to remove any remaining PFA.</li>
<li>Permeabilize the cells with 50 μL of 0.2% Triton-X-100 in PBS solution.</li>
<li>Remove the permeabilization solution and wash cells with 50 μL of 50 mM NH<sub>4</sub>Cl in PBS for 5 min in room temperature.</li>
<li>Incubate the cells with 50 μL of blocking solution (0.1% Ovalbumin and 0.5% Fish gelatin in 1× PBS) for 60 min in room temperature.</li>
<li>After incubation, treat the cells with 50 μL of primary antibodies 12–16 h at 4°C. We used rabbit anti-GFAP (1:400 dilution) and mouse anti-vinculin (1:200 dilution) prepared in blocking solution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Primary antibodies can be incubated at room temperature for 2–3 h.</div>
<ol start="63">
<li>After primary antibody incubation, wash the cells with 1× PBS one time rapidly and two additional times for 10 min.</li>
<li>Apply the 50 μL of secondary antibodies for each coverslip. Alexa Flour 488-conjugated goat anti-rabbit 488 IgG antibody (1:400 diluted in blocking solution), Alexa Fluor 640-conjugated goat anti-mouse IgG antibody (1:400 diluted in blocking solution) and the dyes TRITC-phalloidin (1:400 diluted in blocking solution) for labeling actin filaments (F-actin) and 4′,6-Diamidino-2-Phenylindole (DAPI) (1:400 diluted in blocking solution) for counterstaining of nucleus were prepared in a 1.5 mL tube. Incubate the cells with antibodies at room temperature for 60–75 min in room temperature.</li>
<li>Wash the cells with 1× PBS one time rapidly and three times for 10 min.</li>
<li>Mount the coverslips with Mowiol.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Step 63–66 should be conducted in subdued lighting to prevent bleaching of the fluorescent probes.</div>
<h3 id="sec3.8">Quantification of number and length of focal adhesion molecules in the primary astrocyte cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>Images were acquired on Leica DM6000B microscope (Leica, Bensheim, Germany) with a 63× objective and 20 cells for each condition were used for quantification. The number and length of focal adhesion molecules were quantified by using Fiji (<a href="#fig6">Figure 6</a>). For this, we modified a protocol which previously described (<a href="#bib9">Horzum et al., 2014</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Step by step quantification of focal adhesion (FA) numbers and lengths</div>
<p><b>Step 77:</b> Representative image shows merged image of primary mouse astrocytes which co-labeled with vinculin marker (magenta), actin filaments (red), GFAP (green) and for nuclear staining DAPI (blue). <b>Step 78:</b> After splitting channels, only vinculin staining is shown. <b>Step 79:</b> 8-bit image. <b>Step 80</b>: FFT Bandpass filter applied image. <b>Step 81–83:</b> Subtracting background and detecting FAs in the cell. Scale bar: 10 μm.</p>
</figcaption>
</figure>
<ol start="67">
<li>Open the images using ImageJ software.</li>
<li>If Z-projection is not created
<ol type="a">
<li>Go to Image-Stacks-Z Project-Maximum Projection.</li>
<li>Save the processed image in the folder.</li>
</ol>
</li>
<li>Pick an exported and processed image from the images folder.</li>
<li>To set the scale: First, draw a straight line on the image scale bar.
<ol type="a">
<li>Go to Analyze-Set scale</li>
<li>And mark as global.</li>
</ol>
</li>
<li>Verify the scale by drawing a line on the scale bar.</li>
<li>Make a duplicate image named DUP1 for the RGB image (for comparison).</li>
<li>Draw a scale bar in the image near to the cell to be cropped.
<ol type="a">
<li>Analyze-Tools-Scale bar</li>
</ol>
</li>
<li>Now, select the Rectangle box symbol from the imageJ toolbar and draw around the cell.</li>
<li>Then right-click within the drawn box and press duplicate.
<ol type="a">
<li>Go to Image-Adjust-Brightness and contrast for adjustments and then save the image in the folder.</li>
</ol>
</li>
<li>Again, go to 1<sup>st</sup> duplicate RGB image and again right-click within yellow box for the FFT image.</li>
<li>Separate the merged imaged
<ol type="a">
<li>Go to Image-Color-Split channels</li>
<li>And pick the channels which contain focal adhesion staining</li>
</ol>
</li>
<li>To acquire the FFT image, change the RGB image to an 8-bit image.
<ol type="a">
<li>Image-Type 8-bit</li>
</ol>
</li>
<li>In order to subtract background
<ol type="a">
<li>Process-FFT Bandpass filter</li>
<li>Set pixel size for large structure to 20 pixels and small 1 pixel.</li>
<li>Adjust tolerance of direction to 5%</li>
<li>Select Autoscale after filtering</li>
<li>Select Saturate image when autoscaling</li>
<li>And press OK</li>
</ol>
</li>
<li>Go to Image-Adjust-Brightness
<ol type="a">
<li>Adjust brightness till getting black background and save the Min/Max values for applying to other images.</li>
<li>Adjust maximum &amp; then minimum.</li>
</ol>
</li>
<li>Save the FFT filtered image.
<ol type="a">
<li>Duplicate image.</li>
<li>Image-Type-RGB color as in this step you cannot save 8-bit image.</li>
<li>Close RGB image.</li>
</ol>
</li>
<li>Go to Image-Adjust-Threshold
<ol type="a">
<li>Select Dark Background.</li>
<li>In first drop-down box select-Huang.</li>
<li>In the second drop-down box select-B&amp;W<i>.</i>
</li>
<li>Then apply.</li>
</ol>
</li>
<li>Run analyze particles.
<ol type="a">
<li>Set the parameter for size: 40-Infinity and for circularity 0.00–0.99.</li>
<li>Select pixel units and show the outlines option.</li>
<li>Select display results, clear results, summarize &amp; include holes.</li>
</ol>
</li>
<li>Quantification of focal adhesion numbers per cell area
<ol type="a">
<li>Open the image with F-actin staining.</li>
<li>Draw a line around the cell body using the freehand tool in imageJ tool bar.</li>
<li>Press control+M, this will give the area of the cell body.</li>
<li>Save the results.</li>
<li>Transfer the data to an Excel file.</li>
<li>Count the number of focal adhesions and divide it by the total cell area.</li>
</ol>
</li>
<li>Use Mann-Whitney-Wilcoxon Test for the significant test.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>In this protocol, we provide a cost-efficient, reproducible isolation of primary astrocytes from P0 BL/6 mice without the requirement of commercial kits or special equipment. Applying our protocol, a healthy and easy to maintain primary astrocyte culture can be obtained from wild type or mutant mice strains (<a href="#fig7">Figure 7</a>). Furthermore, and most importantly, the purity of the primary astrocyte cell culture is suitable for studying several astrocytes dependent or related physiological processes. Isolated primary astrocytes can be identified by indirect immunofluorescence using antibodies against the glial fibrillary acidic protein (GFAP). In addition, we applied immunofluorescence staining of microtubule-associated protein (MAP2), PDGRF-alpha and CD68 as marker, respectively, for the identification primary neurons, OPC and microglia, to confirm the purity of astrocyte culture (<a href="#fig8">Figure 8</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Growth of BL/6 mouse primary astrocyte cells in mixed brain culture and pure culture</div>
<p>(A) Representative differential interference contrast (DIC) images show growth of different cell types during culturing days before applying the differential binding method.</p>
<p>(B) Primary astrocytes in culture after applying the differential binding method. After differential binding method primary astrocytes reached the confluency. Red asterisks indicate microglia cells, arrows indicate fibroblast cells and green arrowhead indicates a neuron. Scale bars: 100 μm.</p>
</figcaption>
</figure>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Representative indirect immunostaining of BL/6 mouse brain primary cell cultures</div>
<p>(A) Indirect immunofluorescence double staining of MAP2 (green) and primary astrocyte marker GFAP (magenta), as common markers for primary neurons and primary astrocytes, counterstained by DAPI (blue) for nuclear DNA, before and after the purification of astrocytes.</p>
<p>(B) Immunofluorescence staining of a primary astrocyte culture after primary astrocyte purification. OPC, microglia and primary astrocytes cells were stained by antibodies against PDGRF-α, CD68 and GFAP, respectively.</p>
<p>(C) Quantification of the number of GFAP, PDGRF-α and CD68-stained positive cells after astrocyte purifications demonstrates that primary astrocytes are the most prominent cell type (∼80%). N = 360–490 cells per condition, 3 independent experiments. Scale bar: 10 μm.</p>
</figcaption>
</figure>
<p>We applied our isolation protocol described above for the analysis of focal adhesion dynamics in murine isolated primary astrocytes (<a href="#fig9">Figure 9</a>). Focal adhesions (FAs) are highly dynamic molecules which serve as bi-directional signaling hub in the cell by sensing environmental (outside-in) and intracellular cues (inside-out) (<a href="#bib21">Shen et al., 2012</a>; <a href="#bib22">Sun et al., 2016</a>). With this unique bi-directional signaling feature FA molecules play essential roles in the migration and spreading (<a href="#bib11">Kim et al., 2012</a>; <a href="#bib10">Kim and Wirtz 2013</a>). Namely, size and turnover of FAs are highly important for the migration capacity of the cell. We recently showed that the adhesion G protein coupled receptor (GPCR) ADGRV1/VLGR1 (very large G protein coupled-receptor-1) is a component of FAs and mutations in <i>VLGR1</i> cause defects in FA growth and maturation processes (<a href="#bib12">Kusuluri et al., 2021</a>). In the latter study, we used isolated primary BL/6 wild-type astrocytes, <i>Vlgr1</i><i>/</i>del7TM mice lacking the seven transmembrane domains (7TM) and the intracellular domain (ICD) of <i>Vlgr1</i> (<a href="#bib14">McMillan and White, 2004</a>), and mutant Drum B mice in which only a small portion of the N-terminal fragment is present and the entire C-terminal fragment is missing (<a href="#bib18">Potter et al., 2016</a>; <a href="#bib12">Kusuluri et al., 2021</a>). A significant decrease in the number and length of FAs was observed in both <i>Vlgr1</i> mutant mice compared with primary BL/6 wild-type astrocytes (<a href="#fig10">Figure 10</a>) (<a href="#bib12">Kusuluri et al., 2021</a>). For the quantifications, we applied the ImageJ protocol described above in step 67 to 85. Moreover, our study demonstrated that VLGR1 is metabotropic mechanosensor at FAs and provided novel insights into the pathomechanisms underlying VLGR1-associated diseases, such as human Usher syndrome (<a href="#bib20">Reiners and Wolfrum 2006</a>) and epilepsy (<a href="#bib23">Wang et al., 2015</a>; <a href="#bib16">Myers et al., 2018</a>).</p>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. VLGR1 deficiency alters FA morphology</div>
<p>Representative images of FAs in <i>VLGR1</i>-deficient mouse primary astrocytes.</p>
<p>(A and D) FAs were stained for vinculin (magenta), F-actin with TRITC-phalloidin (red), and nuclei with DAPI (blue); GFAP (green) was used as an astrocyte marker. FA number and length were reduced after <i>VLGR1</i>-depletion in primary astrocytes of both <i>Vlgr1</i> mutant mouse lines.</p>
<p>(B and C) (B) 20 cells per condition, (C) ∼1300–1700 FAs per category for control and ∼200–1000 FAs for mutant astrocytes.</p>
<p>(E and F) (E) 20 cells per condition, (F) ∼950–1250 FAs per condition for control and ∼410–510 FAs for mutant astrocytes. Scale bars: (A and D) 10 μm. Data are represented in B and E are shown as box plot and statistical analyses were done using two-tailed Mann-Whitney U test. Data in C and F are represented as mean±SD. Statistical analyses were done using Sidak’s multiple comparison test; ∗p ≤ 0.05, ∗∗p ≤ 0.01, ∗∗∗p ≤ 0.001.</p>
<p>(A–F). Figure is acquired and adapted from <a href="#fig3">Figures 3</a>D–3H in <a href="#bib12">Kusuluri et al. (2021)</a> with the permission of Elsevier.</p>
</figcaption>
</figure>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1220-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Analysis of assembly and disassembly of focal adhesions by life cell imaging in RFP-Paxillin transfected BL/6 mouse primary astrocytes</div>
<p>(A) Tracking of RFP-Paxillin molecules in focal adhesions (FAs) by live cell imaging. Images were acquired for 85 min with 3 min time intervals. Arrows point to an individual FA which assembles and asterisk indicated a FA which disassembles during the time course of imaging. Lower panel shows color coded individual FAs identified and analyzed via the Focal adhesion analysis server (FAAS) (<a href="#bib2">Berginski and Gomez 2013</a>).</p>
<p>(B) Assembly and disassembly rates of FAs in a single BL/6 primary mouse astrocyte.</p>
<p>(C) Automated intensity analysis of RFP-paxillin in an individual FAs during assembly by FAAS. Green line in the intensity plot indicates the trend line of assembling FA. Scale bar: 10 μm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is designed for isolation of primary astrocytes from P0 mice and all isolation steps are optimized for this age. Nevertheless, our protocol can be adjusted for different aged mice brain culture. It might be taken under consideration that precursor and adult primary astrocyte can show different transcriptional profile which may affect experimental outcome (<a href="#bib25">Zhang et al., 2016</a>; <a href="#bib6">Clarke et al., 2018</a>; <a href="#bib4">Bronzuoli et al., 2019</a>). In addition, there might be species-specific differences between species that primary astrocyte derived may show differences in transcriptomic and protein interaction level. In human and mice astrocytes, it has been reported that metabolism and mitochondria related protein network was higher in mice astrocyte whereas defensive response and extracellular space related proteins were higher in human astrocytes (<a href="#bib13">Li et al., 2021</a>). In rat and mouse astrocytes, significant differences in GFAP expression level, cell proliferation and morphology has been also shown (<a href="#bib19">Puschmann et al., 2010</a>). We observed that primary astrocytes can only be sub-passaged up to 3 times. As more than 3 sub passaging cause astrocyte marker (GFAP) expression loss and unhealthy cell phenotype such as poor cell growth, reduce in cell migration capacity and changes in cell morphology, we highly recommend that experiments should be planned in advance and passage 1 and passage 2 should be used. Astrocytes are one of the main parts of active information transport in central nervous system by interacting with neurons. Neuron-astrocyte interaction through messenger and signaling molecules such as Ca<sup>2+</sup> and glutamate, is highly important in the CNS for function, development and pathology of the brain (<a href="#bib24">Wilhelm et al., 2004</a>; <a href="#bib1">Benarroch 2005</a>; <a href="#bib17">Nimmerjahn 2009</a>). Therefore, it should be taken under consideration that pure primary astrocyte culture may not always reflect in-vivo physiology in the absence of different cell types.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Contamination of fibroblast and endothelial cells in the primary cultures (step 10).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Fibroblast and endothelial cells are the two major cell types present in meninges. Poor handling in meningeal removing increases the contamination level in culture. These two cell types can reach confluency very rapidly which may cause a reduced number of astrocytes and purity of cultures. Meninges should peel away using fine forceps. Be sure that meninges are removed in one piece, if not there can be patches of meninges left which may cause contamination.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Insufficient dissociation of brain tissue (step 21).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>When performing dissociation of brain tissue, be sure to use freshly thawed ice-cold DNase I solution as DNase I is enzymatically helping dissociation of brain tissue. Due to environmental and mechanical stress during the dissociation process DNA can leak as a result of cell damage. Released DNA is often the cause of cell clumps that affects efficiency of dissociation. Start the resuspension of cell pellet with a 10 mL pipette and continue with 5 mL and 2 mL pipettes, respectively. Using a smaller pipette size can increase dissociation efficiency and reduce aggregation levels.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Cell viability is not sufficient after plating (step 24).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>This problem may occur because of excessive Poly-L-Lysine (PLL) residue in flask. Therefore, wash the plates coated with PLL thoroughly before plating single-cell suspension. After plating single-cell suspension, cellular growth should be observed on day 1, day 2, day 7 and day 14. Differential binding should be performed when cultures reach full confluency, as insufficient cell numbers may affect the cell viability.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>OPCs contamination in isolated astrocyte cultures (step 25).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>After 7–15 days of culturing, primary astrocytes reach confluency. It is important to keep the astrocyte layer attached to the flask surface during the OPCs. While too gentle shaking of flask may cause lower yield of OPCs removal, too intense shanking may damage the astrocyte layer. Therefore, shaking should be done carefully.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Microglia contamination after differential binding (step 29).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>After trypsinization, transfer medium containing astrocytes and potential microglia cells to the bacterial grade plates. Microglia cells are attached to the surface of bacterial grade plates whereas primary astrocytes do not. After 20 min of transferring to bacterial grade plate, check the cells, primary astrocytes can be distinguished easily as they float in the medium. Collect the medium containing primary astrocytes and repeat the step carefully for another 90 min. When collecting medium, be careful not to shake the plate or the microglia may become detached.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Low transfection rate (step 41).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>An efficient transfection rate is very important for successful experiment. We have observed that transfection efficiency in primary astrocyte was the range of 15%–30%. To optimize transfection efficiency for your gene/protein of interest, you may try different transfection methods, such as transfection by electroporation or transfection by liposomes. Transfection efficiency may depend on several factors, such as DNA:transfection reagent ratio, purity and composition of the DNA, the degree of confluency of the culture and transfection method. Therefore, to achieve sufficient number of transfected cells and transfection protocol must be optimized. Plasmid DNA should be prepared by using Endotoxin-free isolation kit, as residue of bacterial toxin may lower the efficiency. After obtaining endotoxin-free DNA, different transfection reagent:DNA ratios should be tested to optimize transfection.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Poor migration speed during live-cell imaging (step 52).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>Extracellular matrix is vital for migration of cells in vitro and in vivo. Primary astrocytes produce and secrete extracellular matrix (ECM) proteins such as laminin in vitro condition to provide growth and spreading condition (<a href="#bib5">Chiu et al., 1991</a>). However, in low cell density such as single cell migration study, migration speed can reduce dramatically due to lack of sufficient ECM. A potential solution to this problem is to coating the surface using fibronectin at a 5 μg/mL. Since, different ECM concentrations have an effect on migration speed and migrating cell numbers (<a href="#bib7">Desban and Duband 1997</a>; <a href="#bib15">Millon-Fremillon et al., 2008</a>), effective fibronectin concentration from 1 μg/mL to 10 μg/mL may be tested before starting the experiment. Also, providing conditioned astrocyte medium can enhance the migration speed of primary astrocyte cells as it contains ECM molecules.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Benarroch, E.E. (2005). Neuron-astrocyte interactions: partnership for normal function and disease in the central nervous system. Mayo Clin. Proc. <i>80</i>, 1326-1338. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref1">View at publisher</a></p>
<p id="bib2">Berginski, M.E. and Gomez, S.M. (2013). The Focal Adhesion Analysis Server: a web tool for analyzing focal adhesion dynamics. F1000Res <i>2</i>, 68. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref2">View at publisher</a></p>
<p id="bib3">Blau, A. (2013). Cell adhesion promotion strategies for signal transduction enhancement in microelectrode array in vitro electrophysiology: an introductory overview and critical discussion. Curr. Opin. Colloid Interface Sci. <i>18</i>, 481-492. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref3">View at publisher</a></p>
<p id="bib4">Bronzuoli, M.R., Facchinetti, R., Valenza, M., Cassano, T., Steardo, L., and Scuderi, C. (2019). Astrocyte function is affected by aging and not Alzheimer's disease: a preliminary investigation in hippocampi of 3xTg-AD mice. Front. Pharmacol. <i>10</i>, 644. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref4">View at publisher</a></p>
<p id="bib5">Chiu, A.Y., Espinosa De Los Monteros, A., Cole, R.A., Loera, S., and De Vellis, J. (1991). Laminin and s-laminin are produced and released by astrocytes, Schwann cells, and schwannomas in culture. Glia <i>4</i>, 11-24. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref5">View at publisher</a></p>
<p id="bib6">Clarke, L.E., Liddelow, S.A., Chakraborty, C., Munch, A.E., Heiman, M., and Barres, B.A. (2018). Normal aging induces A1-like astrocyte reactivity. Proc. Natl. Acad. Sci. U S A <i>115</i>, E1896-E1905. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref6">View at publisher</a></p>
<p id="bib7">Desban, N. and Duband, J.L. (1997). Avian neural crest cell migration on laminin: interaction of the alpha1beta1 integrin with distinct laminin-1 domains mediates different adhesive responses. J. Cell Sci. <i>110</i>, 2729-2744. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref7">View at publisher</a></p>
<p id="bib8">Ezratty, E.J., Partridge, M.A., and Gundersen, G.G. (2005). Microtubule-induced focal adhesion disassembly is mediated by dynamin and focal adhesion kinase. Nat. Cell Biol. <i>7</i>, 581-590. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref8">View at publisher</a></p>
<p id="bib9">Horzum, U., Ozdil, B., and Pesen-Okvur, D. (2014). Step-by-step quantitative analysis of focal adhesions. MethodsX <i>1</i>, 56-59. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref9">View at publisher</a></p>
<p id="bib10">Kim, D.H. and Wirtz, D. (2013). Focal adhesion size uniquely predicts cell migration. FASEB J. <i>27</i>, 1351-1361. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref10">View at publisher</a></p>
<p id="bib11">Kim, M.C., Kim, C., Wood, L., Neal, D., Kamm, R.D., and Asada, H.H. (2012). Integrating focal adhesion dynamics, cytoskeleton remodeling, and actin motor activity for predicting cell migration on 3D curved surfaces of the extracellular matrix. Integr. Biol. <i>4</i>, 1386-1397. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref11">View at publisher</a></p>
<p id="bib12">Kusuluri, D.K., Guler, B.E., Knapp, B., Horn, N., Boldt, K., Ueffing, M., Aust, G., and Wolfrum, U. (2021). Adhesion G protein-coupled receptor VLGR1/ADGRV1 regulates cell spreading and migration by mechanosensing at focal adhesions. iScience <i>24</i>, 102283. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref12">View at publisher</a></p>
<p id="bib13">Li, J., Pan, L., Pembroke, W.G., Rexach, J.E., Godoy, M.I., Condro, M.C., Alvarado, A.G., Harteni, M., Chen, Y.W., Stiles, L., et al. (2021). Conservation and divergence of vulnerability and responses to stressors between human and mouse astrocytes. Nat. Commun. <i>12</i>, 3958. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref13">View at publisher</a></p>
<p id="bib14">McMillan, D.R. and White, P.C. (2004). Loss of the transmembrane and cytoplasmic domains of the very large G-protein-coupled receptor-1 (VLGR1 or Mass1) causes audiogenic seizures in mice. Mol. Cell Neurosci. <i>26</i>, 322-329. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref14">View at publisher</a></p>
<p id="bib15">Millon-Fremillon, A., Bouvard, D., Grichine, A., Manet-Dupe, S., Block, M.R., and Albiges-Rizo, C. (2008). Cell adaptive response to extracellular matrix density is controlled by ICAP-1-dependent beta1-integrin affinity. J. Cell Biol. <i>180</i>, 427-441. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref15">View at publisher</a></p>
<p id="bib16">Myers, K.A., Nasioulas, S., Boys, A., Mcmahon, J.M., Slater, H., Lockhart, P., Sart, D.D., and Scheffer, I.E. (2018). ADGRV1 is implicated in myoclonic epilepsy. Epilepsia <i>59</i>, 381-388. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref16">View at publisher</a></p>
<p id="bib17">Nimmerjahn, A. (2009). Astrocytes going live: advances and challenges. J. Physiol. <i>587</i>, 1639-1647. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref17">View at publisher</a></p>
<p id="bib18">Potter, P.K., Bowl, M.R., Jeyarajan, P., Wisby, L., Blease, A., Goldsworthy, M.E., Simon, M.M., Greenaway, S., Michel, V., Barnard, A., et al. (2016). Novel gene function revealed by mouse mutagenesis screens for models of age-related disease. Nat. Commun. <i>7</i>, 12444. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref18">View at publisher</a></p>
<p id="bib19">Puschmann, T.B., Dixon, K.J., and Turnley, A.M. (2010). Species differences in reactivity of mouse and rat astrocytes in vitro. Neurosignals <i>18</i>, 152-163. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref19">View at publisher</a></p>
<p id="bib20">Reiners, J. and Wolfrum, U. (2006). Molecular analysis of the supramolecular usher protein complex in the retina. Harmonin as the key protein of the Usher syndrome. Adv. Exp. Med. Biol. <i>572</i>, 349-353. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref20">View at publisher</a></p>
<p id="bib21">Shen, B., Delaney, M.K., and Du, X. (2012). Inside-out, outside-in, and inside-outside-in: G protein signaling in integrin-mediated cell adhesion, spreading, and retraction. Curr. Opin. Cell Biol. <i>24</i>, 600-606. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref21">View at publisher</a></p>
<p id="bib22">Sun, Z., Guo, S.S., and Fassler, R. (2016). Integrin-mediated mechanotransduction. J. Cell Biol. <i>215</i>, 445-456. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref22">View at publisher</a></p>
<p id="bib23">Wang, Y., Fan, X., Zhang, W., Zhang, C., Wang, J., Jiang, T., and Wang, L. (2015). Deficiency of very large G-protein-coupled receptor-1 is a risk factor of tumor-related epilepsy: a whole transcriptome sequencing analysis. J. Neurooncol. <i>121</i>, 609-616. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref23">View at publisher</a></p>
<p id="bib24">Wilhelm, A., Volknandt, W., Langer, D., Nolte, C., Kettenmann, H., and Zimmermann, H. (2004). Localization of SNARE proteins and secretory organelle proteins in astrocytes in vitro and in situ. Neurosci. Res. <i>48</i>, 249-257. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref24">View at publisher</a></p>
<p id="bib25">Zhang, Y., Sloan, S.A., Clarke, L.E., Caneda, C., Plaza, C.A., Blumenthal, P.D., Vogel, H., Steinberg, G.K., Edwards, M.S., Li, G., et al. (2016). Purification and characterization of progenitor and mature human astrocytes reveals transcriptional and functional differences with mouse. Neuron <i>89</i>, 37-53. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00660-2/sref25">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Uwe Wolfrum (<a href="mailto:wolfrum@uni-mainz.de">wolfrum@uni-mainz.de</a>).</p>
<h4>Materials availability</h4>
<p>Reagents and resources used in this study are commercially available with the exception of RFP-paxillin plasmid which was kindly provided by Drs. Leube and Hortwitz. Nevertheless, requests for resources and reagents can be directed to and will be fulfilled by the lead contact.</p>
<h4>Data and code availability</h4>
<p>This study did not generate new data sets.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the German Research Council DFG FOR 2149 “Elucidation of Adhesion-GPCR signaling” WO 548/8 (UW). We thank Drs Rudolf E. Leube and Rick Horwitz for kindly sharing reagents and Dr Karl R. Fath for critical reading and language corrections.</p>
<h3>Author contributions</h3>
<p>Conceptualization, B.E.G. and U.W.; investigation, B.E.G. and J.K.; writing – original draft, B.E.G. and U.W.; writing – review &amp; editing, B.E.G., J.K. and U.W.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>