<article>
<section>
<h1 id="header">A protocol to differentiate nociceptors, mechanoreceptors, and proprioceptors from human pluripotent stem cells</h1>
<p><time datetime="2022-03-25">Published: March 25, 2022</time></p>
<p>Kenyi Saito-Diaz<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> and Nadja Zeltner<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Center for Molecular Medicine, University of Georgia, Athens, GA 30602, USA</p>
<p id="aff2"><sup>2</sup>Department of Biochemistry and Molecular Biology, Franklin College of Arts and Sciences, University of Georgia, Athens, GA 30602, USA</p>
<p id="aff3"><sup>3</sup>Department of Cellular Biology, Franklin College of Arts and Sciences, University of Georgia, Athens, GA 30602, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:saitod@uga.edu">saitod@uga.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:nadja.zeltner@uga.edu">nadja.zeltner@uga.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101187">10.1016/j.xpro.2022.101187</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Human pluripotent stem cells (hPSCs) show promise for studying diseases affecting cell populations that are not easily available, including sensory neurons (SNs). Here, we present a differentiation protocol in chemically defined conditions to generate peripheral SNs from hPSCs. We describe four main steps: expansion of hPSCs, neural crest cell (NCC) differentiation, NCC dissociation and replating, and sensory neuron (SN) differentiation. This protocol enables generation of a mechanoreceptor-enriched culture or a population containing all three SN subtypes (nociceptors, mechanoreceptors, and proprioceptors).</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2021.01.001">Saito-Diaz et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for differentiation of hPSCs into different types of SNs from one culture</li>
<li>Step-by-step protocol for <i>in vitro</i> differentiation of NCCs</li>
<li>
<i>In vitro</i> differentiation into mechanoreceptor-enriched culture</li>
<li>Differentiation of three main SN subtypes that mimic <i>in vivo</i> composition</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1517-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes a method to differentiate sensory neurons (SNs) from human embryonic stem cells (hESCs, H9, WA09) (<a href="#fig1">Figure 1</a>). The differentiated SNs undergo all developmental stages and can be used to obtain a mechanoreceptor-enriched culture, or DRG-like composition (nociceptors, mechanoreceptors, and proprioceptors). SNs obtained in this protocol can be replated to maintain the neurons for long periods of time (&gt;70 days) if necessary. This replated step can be also used as an axotomy model (<a href="#bib2">Saito-Diaz et al., 2021</a>). Moreover, this protocol has also been successfully tested in patient-derived induced pluripotent stem cells (iPSCs). All the steps described here are carried out in a laminar flow Class II biological hood using proper aseptic technique to avoid contamination.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1517-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Differentiation strategy</div>
<p>(A) Differentiation timeline of SNs replated on day 6 (to obtain mainly a population of mechanoreceptors). Developmental steps (top), growth factors used at each step of differentiation (middle) and representative brightfield images of each step (bottom) are shown. Red arrows indicated typical NCC aggregations (“ridges”).</p>
<p>(B) Differentiation timeline of SNs replated on day 12 (to obtain a DRG-like SN composition). Developmental steps (top) and representative brightfield images (bottom) are shown. Red arrows indicated typical “ridges” of NCCs.</p>
<p>(C) Effects of different coats on day 20 SNs. Day 12 NCCs were replated on day 12 on plates coated with PO/LM/FN or PO/FN and differentiated until day 20. SNs were then fixed and stained using the indicated antibodies. Note that the number of BRN3A<sup>-</sup> nuclei (arrows) decreases in PO/FN coated plates, indicating low number of contaminant cells.</p>
<p>(D) Phenotypes of hPSC colonies not suitable for differentiations. We observed that the following colonies will not yield a robust differentiation (arrows): small colonies very close to each other (left image), colonies with no defined edges (middle-left image), and colonies showing differentiated cells on the edges (middle-right and right images).</p>
</figcaption>
</figure>
<p>The main steps consist of: 1) Expansion of hPSCs, 2) Neural crest cell (NCC) differentiation stage, 3) NCC dissociation and replating stage, and 4) sensory neuron (SN) differentiation. NCCs can be dissociated and replated on day 6 to generate mainly mechanoreceptor SNs, or on day 12 to generate a DRG-like (with higher percentage of nociceptors) SN composition (<a href="#bib2">Saito-Diaz et al., 2021</a>) (<a href="#fig1">Figures 1</a>A and 1B).</p>
<p>We start the culture from 10 cm dishes with H9 cells. Each dish yields ∼12 × 10<sup>6</sup> cells. Thus, it is possible to differentiate one 6-well plate (10.8 × 10<sup>6</sup> cells) and one 4-well plate (1.52 × 10<sup>6</sup> cells) of NCCs from a single 10 cm dish. Similarly, one 6-well plate of NCCs yields ∼30 × 10<sup>6</sup> cells, which can be used to differentiate two 24-well plates of SNs (12 × 10<sup>6</sup> cells each). We recommend differentiating H9 cells in 6-well plates (for the NCC stage) for further replating and SN differentiation, and doing a differentiation in parallel in a 4-well plate to measure expression of NCC markers (such as SOX10) as a quality control (QC) check.</p>
<p>This protocol is detailed for one 6-well plate of NCCs. Three wells will be replated on day 6 to grow one 24-well plate of mechanoreceptors and the remaining wells will be replated on day 12 to grow a pool of nociceptors, mechanoreceptors, and proprioceptors. You will need to scale up the volumes for multiple plates or a different format.</p>
<h3 id="sec1.1">Preparation of vitronectin (VTN) 10 cm dishes for hPSC expansion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>To prepare one 10 cm dish:</p>
<ol>
<li>Thaw a vial of VTN (0.5 mg/mL) in a water bath</li>
<li>Dilute 7 μL VTN into 7 mL DPBS.</li>
<li>Mix vigorously.</li>
<li>Add mixture to a 10 cm dish.</li>
<li>Incubate for 1 h at room temperature (RT; 20°C–22°C).</li>
<li>Aspirate the mix.</li>
<li>Use the plate immediately or add 8 mL DPBS and store at 37°C for up to 7 days. Make sure the dish does not dry out during this time by adding additional DPBS if necessary.</li>
</ol>
<h3 id="sec1.2">Preparation of VTN plates for NCC differentiation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="8">
<li>Prepare one 6-well plate for differentiation.</li>
<li>Thaw an aliquot of VTN (0.5 mg/mL) in a 37°C water bath.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Geltrex (ThermoFisher, Cat# A1413301) can be used instead of VTN. However, differentiation efficiency decreases (<a href="#bib2">Saito-Diaz et al., 2021</a>).</div>
<ol start="10">
<li>Add 12 μL VTN to 12 mL DPBS.</li>
<li>Mix and add 2 mL of the mixture to each well.</li>
<li>Incubate for 1 h at RT.</li>
<li>Use the plate immediately or add 2 mL DPBS to each well and store at 37°C for up to 7 days. Make sure the dish does not dry out during this time by adding additional DPBS if necessary.</li>
</ol>
<h3 id="sec1.3">Preparation of poly-L-ornithine/laminin/fibronectin (PO/LM/FN) plates for SN differentiation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="14">
<li>Prepare one 24-well plate for differentiation.</li>
<li>Add 6 μL of poly-L-ornithine hydrobromide (PO, 7.5 μg/mL) to 12 mL DPBS.</li>
<li>Add 0.5 mL of the mixture to each well.</li>
<li>Incubate at 37°C for 16–20 h.</li>
<li>The next day wash each well twice with 0.5 mL DPBS.</li>
<li>Add 12 μL Laminin (LM, 1 μg/mL) and 12 μL Fibronectin (FN, 1 μg /mL) to 12 mL DPBS.</li>
<li>Mix and add 0.5 mL of the mixture to each well.</li>
<li>Incubate at 37°C for ≥ 16 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Coated plates can be stored for ∼7 days at 37°C. For longer storage (&gt;1 week), seal them with Parafilm and store at 4°C. Make sure that the wells do not dry out by adding DPBS. These plates can be used on step 34 and step 52.</div>
<div class="critical">
<span class="critical-title">Critical:</span> SNs can grow in only PO/FN-coated plates which will reduce the presence of contaminant cell types (<a href="#fig1">Figure 1</a>C). Please note that SNs do not attach strongly on the PO/FN plates, thus should be handled with care to avoid detachment.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>SOX10 (dilution: 1:100)</td>
<td>Santa Cruz</td>
<td>Cat# sc-365692, RRID:AB_10844002</td>
</tr>
<tr>
<td>BRN3A (dilution: 1:100)</td>
<td>Merck Millipore</td>
<td>Cat# MAB1585, RRID:AB_94166</td>
</tr>
<tr>
<td>TUJ1 (dilution: 1:1500)</td>
<td>BioLegend</td>
<td>Cat# 802001, RRID:AB_2564645</td>
</tr>
<tr>
<td>TUJ1 (dilution: 1:1500)</td>
<td>BioLegend</td>
<td>Cat# 801201, RRID:AB_2313773</td>
</tr>
<tr>
<td>TFAP2A (dilution: 1:500)</td>
<td>Abcam</td>
<td>Cat# ab108311, RRID:AB_10861200</td>
</tr>
<tr>
<td>TRKB (dilution: 1:100)</td>
<td>R&amp;D Systems</td>
<td>Cat# MAB3971, RRID:AB_2155144</td>
</tr>
<tr>
<td>RET (dilution: 1:100)</td>
<td>Sigma-Aldrich</td>
<td>Cat# SAB1409600</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Essential 8 Medium</td>
<td>Gibco</td>
<td>Cat# A1517001</td>
</tr>
<tr>
<td>Essential 8 Medium supplement</td>
<td>Gibco</td>
<td>Cat# A1517001</td>
</tr>
<tr>
<td>Essential 6 Medium</td>
<td>Gibco</td>
<td>Cat# A1516401</td>
</tr>
<tr>
<td>Neurobasal Medium</td>
<td>Gibco</td>
<td>Cat# 21103-049</td>
</tr>
<tr>
<td>Recombinant human BMP4</td>
<td>R&amp;D Systems</td>
<td>Cat# 314-BP</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>R&amp;D Systems</td>
<td>Cat# 4423</td>
</tr>
<tr>
<td>SB431542</td>
<td>R&amp;D Systems</td>
<td>Cat# 1614</td>
</tr>
<tr>
<td>DAPT</td>
<td>R&amp;D Systems</td>
<td>Cat# 2634</td>
</tr>
<tr>
<td>SU5402</td>
<td>Biogems</td>
<td>Cat## 2159233</td>
</tr>
<tr>
<td>Y-27632</td>
<td>Biogems</td>
<td>Cat# #1293823</td>
</tr>
<tr>
<td>N2 Supplement</td>
<td>Gibco</td>
<td>Cat# 17502-048</td>
</tr>
<tr>
<td>B-27 Supplement</td>
<td>Gibco</td>
<td>Cat# 12587-010</td>
</tr>
<tr>
<td>L-glutamine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 25030-081</td>
</tr>
<tr>
<td>BDNF</td>
<td>R&amp;D Systems</td>
<td>Cat# 248-BD</td>
</tr>
<tr>
<td>GDNF</td>
<td>PeproTech</td>
<td>Cat# 450-10</td>
</tr>
<tr>
<td>NGF</td>
<td>PeproTech</td>
<td>Cat# 450-01</td>
</tr>
<tr>
<td>Retinoic acid</td>
<td>Sigma-Aldrich</td>
<td>Cat# R2625</td>
</tr>
<tr>
<td>Antibiotic-Antimycotic</td>
<td>Gibco</td>
<td>Cat# 15240096</td>
</tr>
<tr>
<td>Poly-L-ornithine hydrobromide</td>
<td>Sigma-Aldrich</td>
<td>Cat# P3655</td>
</tr>
<tr>
<td>EDTA (0.5 M, pH 8.0, RNase-free)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM9262</td>
</tr>
<tr>
<td>NaCl (BioReagent, suitable for cell culture, ≥99%)</td>
<td>Sigma-Aldrich</td>
<td>Cat# S5886-500G</td>
</tr>
<tr>
<td>Dulbecco’s Phosphate-Buffered Saline (DPBS)</td>
<td>Corning</td>
<td>Cat# 21-031-CM</td>
</tr>
<tr>
<td>Recombinant human Vitronectin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A31804</td>
</tr>
<tr>
<td>Laminin-1</td>
<td>Cultrex</td>
<td>Cat# 3401-010-02</td>
</tr>
<tr>
<td>Fibronectin</td>
<td>Corning</td>
<td>Cat# 47743-654</td>
</tr>
<tr>
<td>Trypan Blue Solution (0.4% w/v in PBS, pH 7.5)</td>
<td>Corning</td>
<td>Cat# 45000-717</td>
</tr>
<tr>
<td>TRIZOL</td>
<td>Invitrogen</td>
<td>Cat# 15-596-018</td>
</tr>
<tr>
<td>Accutase</td>
<td>Innovative Cell Technologies</td>
<td>Cat# NC9464543</td>
</tr>
<tr>
<td>DNase I</td>
<td>Roche</td>
<td>Cat# 10104159001</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10977015</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Sso Advanced Universal SYBR Green Supermix</td>
<td>Bio-Rad</td>
<td>Cat# 1725272</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Human embryonic stem cells (H9)</td>
<td>WiCell</td>
<td>WA-09</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>RT-qPCR Primer: <i>SOX10</i> F</td>
<td>Sigma-Aldrich; <a href="#bib2">Saito-Diaz et al., 2021</a>
</td>
<td>5′-CCAGGCCCACTACAAGAGC-3′</td>
</tr>
<tr>
<td>RT-qPCR Primer: <i>SOX10</i> R</td>
<td>Sigma-Aldrich; <a href="#bib2">Saito-Diaz et al., 2021</a>
</td>
<td>5′-CTCTGGCCTGAGGGGTGC-3′</td>
</tr>
<tr>
<td>RT-qPCR Primer: <i>BRN3A</i> F</td>
<td>Sigma-Aldrich; <a href="#bib2">Saito-Diaz et al., 2021</a>
</td>
<td>5′-AGTACCCGTCGCTGCACTCCA-3′</td>
</tr>
<tr>
<td>RT-qPCR Primer: <i>BRN3A</i> R</td>
<td>Sigma-Aldrich; <a href="#bib2">Saito-Diaz et al., 2021</a>
</td>
<td>5′-TTGCCCTGGGACACGGCGATG-3′</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Prism 9.2.0</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/">https://www.graphpad.com/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Tissue culture dishes (10 cm)</td>
<td>Corning</td>
<td>Cat# 353003</td>
</tr>
<tr>
<td>Tissue culture plates (6-well)</td>
<td>Fisher Scientific</td>
<td>Cat# FB012927</td>
</tr>
<tr>
<td>Tissue culture plates (24-well)</td>
<td>Corning</td>
<td>Cat# 353047</td>
</tr>
<tr>
<td>Vacuum filtration system (500 mL, 0.22 μm filter)</td>
<td>Corning</td>
<td>Cat# 28199-778</td>
</tr>
<tr>
<td>CFX96 Touch Real-Time PCR Detection System</td>
<td>Bio-Rad</td>
<td>Cat# 1855195</td>
</tr>
<tr>
<td>Hard-Shell Thin-Wall 96-Well Skirted PCR plates</td>
<td>Bio-Rad</td>
<td>Cat# HSP9601</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>EDTA Dissociation solution (500 mL stock):</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Storage</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>EDTA</td>
<td>RT</td>
<td>0.5 M</td>
<td>0.5 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>NaCl</td>
<td>RT</td>
<td>3.0801 M</td>
<td>30 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>DPBS</td>
<td>RT</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td> </td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Filter the solution using a 0.22 μm filter, aliquot, and store at RT. It is stable for &gt; 3 months.</p>
<table id="undtbl2">
<caption>NCC Differentiation media 1:</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Storage</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>SB431542</td>
<td>−20°C</td>
<td>10 mM</td>
<td>10 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>BMP4</td>
<td>−80°C</td>
<td>10 μg/mL</td>
<td>1 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>−20°C</td>
<td>6 mM</td>
<td>300 nM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Y-27632</td>
<td>−20°C</td>
<td>10 mM</td>
<td>10 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>E6 Essential Medium</td>
<td>4°C</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C. It is recommended to prepare fresh but can be stored for up to 7 days.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Y-27632 can also be purchased from R&amp;D Systems (cat# 1254).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> E6 Essential Medium can be made in house using stock components (<a href="#bib1">Chen et al., 2011</a>). All components need to be as fresh as possible when the media is made to ensure high quality. We recommend making single-use aliquots and store them at their recommended storage temperature.</div>
<div class="note">
<span class="note-title">Note:</span> BMP4 concentrations might need to be adjusted depending on the cell line used (See troubleshooting <a href="#sec6.1">problem 1</a>).</div>
<table id="undtbl3">
<caption>NCC Differentiation media 2:</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Storage</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>SB431542</td>
<td>−20°C</td>
<td>10 mM</td>
<td>10 μM</td>
<td>100 μL</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>−20°C</td>
<td>6 mM</td>
<td>0.75 μM</td>
<td>12.5 μL</td>
</tr>
<tr>
<td>DAPT</td>
<td>−80°C</td>
<td>40 mM</td>
<td>2.5 μM</td>
<td>6.25 μL</td>
</tr>
<tr>
<td>SU5402</td>
<td>−80°C</td>
<td>40 mM</td>
<td>2.5 μM</td>
<td>6.25 μL</td>
</tr>
<tr>
<td>E6 Essential Medium</td>
<td>4°C</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td> </td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 14 days. It is recommended to prepare the exact volume required for the duration of the experiment (10 days).</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> SU5402 can also be purchased from Biovision (cat# 1645-1)</div>
<table id="undtbl4">
<caption>SN Differentiation media:</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Storage</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>N2 Supplement</td>
<td>−20°C</td>
<td>100X</td>
<td>1X</td>
<td>1 mL</td>
</tr>
<tr>
<td>B-27 Supplement</td>
<td>−20°C</td>
<td>50X</td>
<td>1X</td>
<td>2 mL</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>−20°C</td>
<td>200 mM</td>
<td>2 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>GDNF</td>
<td>−80°C</td>
<td>10 μg/mL</td>
<td>20 ng/mL</td>
<td>200 μL</td>
</tr>
<tr>
<td>BDNF</td>
<td>−80°C</td>
<td>10 μg/mL</td>
<td>20 ng/mL</td>
<td>200 μL</td>
</tr>
<tr>
<td>NGF</td>
<td>−80°C</td>
<td>25 μg/mL</td>
<td>25 ng/mL</td>
<td>100 μL</td>
</tr>
<tr>
<td>Laminin-1</td>
<td>−80°C</td>
<td>1 mg/mL</td>
<td>0.6 μg/mL</td>
<td>60 μL</td>
</tr>
<tr>
<td>Fibronectin</td>
<td>−80°C</td>
<td>1 mg/mL</td>
<td>0.6 μg/mL</td>
<td>60 μL</td>
</tr>
<tr>
<td>Retinoic acid</td>
<td>−80°C</td>
<td>1 mM</td>
<td>0.125μM</td>
<td>12.5 μL</td>
</tr>
<tr>
<td>Antibiotic-Antimycotic</td>
<td>−20°C</td>
<td>100X</td>
<td>1X</td>
<td>1 mL</td>
</tr>
<tr>
<td>DAPT (see note)</td>
<td>−80°C</td>
<td>40 mM</td>
<td>1 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Neurobasal medium</td>
<td>4°C</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td> </td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C. It is recommended to prepare fresh with each differentiation. It can be stored for up to 14 days.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Retinoic acid is light sensitive and unstable. Aliquot in small volumes (25 μL) and protect it from light. Add fresh in every feeding. Do not use aliquots that are older than 1 year and avoid using the same aliquot more than 5 times.</div>
<div class="note">
<span class="note-title">Note:</span> DAPT promotes differentiation of NCCs into SNs. By day 20, we do not see SOX10<sup>+</sup> proliferating cells (which represent NCCs) in culture so DAPT is not necessary after that.</div>
<div class="note">
<span class="note-title">Note:</span> The media do not need to be warmed up prior to using.</div>
<table id="undtbl5">
<caption>Antibodies to be used:</caption>
<thead>
<tr>
<th>Antibody</th>
<th>Storage</th>
<th>Species</th>
<th>Working dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>SOX10</td>
<td>4°C</td>
<td>Mouse</td>
<td>1:100</td>
</tr>
<tr>
<td>BRN3A</td>
<td>−20°C</td>
<td>Mouse</td>
<td>1:100</td>
</tr>
<tr>
<td>TUJ1</td>
<td>4°C</td>
<td>Mouse</td>
<td>1:1500</td>
</tr>
<tr>
<td>TUJ1</td>
<td>4°C</td>
<td>Rabbit</td>
<td>1:1500</td>
</tr>
<tr>
<td>TFAP2A</td>
<td>−20°C</td>
<td>Rabbit</td>
<td>1:500</td>
</tr>
<tr>
<td>TRKB</td>
<td>−20°C</td>
<td>Mouse</td>
<td>1:100</td>
</tr>
<tr>
<td>RET</td>
<td>−20°C</td>
<td>Mouse</td>
<td>1:100</td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>RT-qPCR: Prepare the RT-qPCR master mix by adding the following components (except cDNA) for a 10 μL reaction:</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>SYBR mix</td>
<td>2X</td>
<td>1X</td>
<td>5 μL</td>
</tr>
<tr>
<td>Primer F</td>
<td>10 μM</td>
<td>500 nM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Primer R</td>
<td>10 μM</td>
<td>500 nM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>cDNA</td>
<td>1 μg</td>
<td>10 ng</td>
<td>1 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td> </td>
<td> </td>
<td>3 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">The master mix should be prepared at 4°C.<br> Mix the reaction thoroughly, add into the wells of a qPCR plate and add the cDNA to each of the wells containing the RT-qPCR master mix.</p>
<table id="undtbl7">
<caption>Program the following protocol on a real-time PCR thermocycler:</caption>
<thead>
<tr>
<th>Step</th>
<th>Cycles</th>
<th>Temperature</th>
<th>Time</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>1</td>
<td>95°C</td>
<td>30 s</td>
</tr>
<tr>
<td>2</td>
<td>40</td>
<td>95°C</td>
<td>30 s</td>
</tr>
<tr>
<td>3</td>
<td>40</td>
<td>60°C</td>
<td>10 s</td>
</tr>
<tr>
<td>4</td>
<td>Melting curve</td>
<td>variable</td>
<td> </td>
</tr>
<tr>
<td>5</td>
<td>1</td>
<td>4°C</td>
<td>keep</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">hPSCs expansion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼5 days</div>
<ol>
<li>Grow undifferentiated H9 hESCs (WiCell Research Institute) in 10 mL E8 Essential Medium (according to Thermo Fisher instructions: <a href="https://www.thermofisher.com/order/catalog/product/A1517001">https://www.thermofisher.com/order/catalog/product/A1517001</a>) in a VTN-coated 10 cm dish. To differentiate one 6-well plate, you will need 10.8 × 10<sup>6</sup> cells. Each 10 cm dish yields ∼12 × 10<sup>6</sup> undifferentiated hPSCs.</li>
<li>Expand the hPSCs to as many dishes as needed. To split cells:
<ol type="a">
<li>Aspirate media.</li>
<li>Add 6 mL DPBS to the cells.</li>
<li>Aspirate DBPS.</li>
<li>Add 4 mL EDTA Dissociation solution. Incubate for 2 min at 37°C.</li>
<li>Aspirate EDTA Dissociation solution.</li>
<li>Resuspend cells by adding 10 mL E8 Essential Medium directly to the cells.</li>
<li>Pipet mixture of media and cells relatively harsh (no more than five times) while rotating the dish to remove all the cells.</li>
<li>Add cells to a 15 mL conical tube.</li>
<li>Split the cells at a 1:20 ratio using E8 Essential Medium and seed them into a new VTN-coated 10 cm dish.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cell colonies must have clearly defined bright edges with minimal spontaneous differentiation, visualized under a brightfield microscope (<a href="#fig1">Figures 1</a>A, 1B, and 1D, day 0). Differentiating efficiency might decrease if cells are cultured for &gt;5 consecutive passages (See troubleshooting <a href="#sec6.1">problem 1</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Check the cultures at least every two weeks to rule out presence of mycoplasma and other contaminants. We recommend using MycoAlert Mycoplasma Detection Kit (Lonza, Cat# LT07-318).</div>
<div class="note">
<span class="note-title">Note:</span> Research involving hPSCs must be done in accordance with legal and ethical guidelines.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> E8 Essential Medium can be made in house using stock components (<a href="#bib1">Chen et al., 2011</a>). All components need to be as fresh as possible when the media is made to ensure high quality. We recommend making single-use aliquots and store them at their recommended storage temperature.</div>
<h3 id="sec3.2">Differentiation into neural crest cells (NCCs)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> up to 12 days</div>
<p>In this section, we describe in detail the steps to differentiate hPSCs into NCCs with a SN fate. NCCs start to aggregate on day 4, and by day 8 dark aggregates (“ridges”) are visible under a brightfield microscope (<a href="#fig1">Figures 1</a>A, 1B, and <a href="#fig2">2</a>A). Additionally, NCCs start expressing SN markers as early as day 4 (<a href="#fig2">Figure 2</a>A).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1517-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Characterization of NCC stage</div>
<p>(A) Expected phenotypes of properly differentiated NCCs. Brightfield images of NCCs were taken on the indicated days (top panel). Arrows show growth and progression of NCC “ridges”, which start to form on day 4. Expression of NCC markers. NCCs were fixed on the indicated days and stained for SOX10 (green), TFAP2A (red), and DAPI (blue). Images were taken at two different magnifications (middle and bottom panels).</p>
<p>(B) Appropriate seeding density. H9 cells were seeded at a density of 200,000 cell/cm<sup>2</sup> (day 0). The next day (day 1) brightfield images were taken to show the expected density (&gt;80% confluency).</p>
<p>(C) Expression of <i>SOX10</i> mRNA by RT-qPCR. n=4 (biological replicates), one-way ANOVA followed by Dunnett’s multiple comparisons. Graph show mean ± S.D. ∗p &lt; 0.05, ∗∗∗∗p &lt; 0.0001.</p>
<p>(D) Expected phenotypes of improper NCC differentiation. Brightfield images were taken on day 12. Note small size of the NCC “ridges” (red arrows left and middle images) or the formation of a single large NCC cluster (red arrow, right image), which will cause a very low differentiation efficiency. Also, note the presence of large blister-like cells/aggregates (black arrows, left and right images), the identity of which we do not know at this point.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> It is important to carry out a parallel differentiation at the NCC stage for quality control (QC) purposes in a small format (4-well plate). This allows isolation of RNA and/or perform immunofluorescence (on day 6 or 8) from a plate that will be treated in the same way as the main plate.</div>
<ol start="3">
<li>Aspirate E8 media from the 10 cm dish of undifferentiated, ready to split hPSCs.</li>
<li>Wash once with ∼10 mL DPBS.</li>
<li>Add 4 mL EDTA Dissociation solution to a 10-cm dish, incubate for 15 min at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not aspirate the EDTA Dissociation solution. The hPSCs colonies will be dissociated and resuspended into single cells and the EDTA Dissociation solution will be cloudy.</div>
<ol start="6">
<li>Use 10 mL DPBS to wash off the cells. Most or all cells should be resuspended or detach from the plate rather easily at this point.</li>
<li>Transfer the cells into a 50 mL conical tube and add an additional 10 mL DPBS.</li>
<li>Spin down at 200×<i>g</i> for 4 min.</li>
<li>Resuspend in 5 mL NCC Differentiation media 1.</li>
<li>Count cells using a Countess II or hematocytometer:
<ol type="a">
<li>Mix 10 μL Trypan blue (Corning) with 10 μL of cell suspension.</li>
<li>Mix by pipetting and put 10 μL in a counting slide (ThermoFisher, Cat# C10312).</li>
<li>Count the cells.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Resuspend 0.5 − 1 × 10<sup>6</sup> cells in 500 μL TRIZOL for RNA for day 0.</div>
<ol start="11">
<li>Aspirate VTN from the 6-well plate (NCC differentiation plate).</li>
<li>Transfer 10.8 × 10<sup>6</sup> cells (to seed one 6-well plate at a density of 200,000 cells/cm<sup>2</sup>) to a 15 mL conical tube and fill with NCC Differentiation media 1 up to 12 mL.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A total of 10.8 × 10<sup>6</sup> cells are needed for a one 6-well plate, thus we recommend growing one 10 cm dish of H9 cells per 6-well plate.</div>
<ol start="13">
<li>Plate 2 mL of the cell suspension directly onto each of the VTN-coated wells (do not let it dry before seeding the cells).</li>
<li>Incubate at 37°C for 16–20 h.</li>
<li>On the next day (day 1), replace media with 3 mL per well of NCC Differentiation media 1.</li>
<li>Take a picture for your records of cell density. High density is desired (&gt;80%) (<a href="#fig2">Figure 2</a>B).</li>
<li>Incubate at 37°C for 16–20 h.</li>
<li>The following day (day 2), replace media (3 mL/well) of NCC Differentiation media 2.</li>
<li>Change media every 2 days until day 6 for inducing mechanoreceptors or day 12 for obtaining all three neuronal types:
<ol type="a">
<li>To generate mainly mechanoreceptors, on day 6 follow the steps in the section “Differentiation into SNs (mechanoreceptor enrichment)”. Step 22.</li>
<li>To generate nociceptors, mechanoreceptors, and proprioceptors, on day 12 follow the steps in the section “Differentiation into SNs (nociceptors, mechanoreceptors, and proprioceptors)”. Step 40.</li>
</ol>
</li>
<li>On day 8, harvest RNA from 4-well quality control plate:
<ol type="a">
<li>Aspirate the media from one well.</li>
<li>Lyse the cells in 500 μL TRIZOL.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> do not wash cells with PBS as this could cause mRNA degradation.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The quality control plate can be fixed for immunofluorescence staining.</div>
<ol start="21">
<li>Take pictures on every feeding day to follow proper progression (<a href="#fig1">Figures 1</a>A and 1B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is normal to see some cell aggregates (“ridges”) starting on day 4 (<a href="#fig2">Figure 2</a>A), which overlaps with an increase of SOX10 expression at the mRNA and protein levels (<a href="#fig2">Figures 2</a>A and 2C) (See troubleshooting <a href="#sec6.3">problem 2</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> If there is no clear formation of NCC ridges (<a href="#fig2">Figure 2</a>D), it is best to stop the differentiation because the SN yield will be too low or the presence of contaminant cells will be too high to perform any subsequent analysis (<a href="#fig3">Figure 3</a>D). See troubleshooting <a href="#sec6.1">problem 1</a>.</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1517-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Characterization of SNs</div>
<p>(A) Expected phenotypes of SNs obtained from day-6 NCCs. Cells were fixed 8 days post-replated and stained for TUJ1, the panSN-marker BRN3A, and the mechanoreceptor-specific markers RET and TRKB.</p>
<p>(B) Expected phenotypes of SNs obtained from day-12 NCCs. Brightfield images (top) and immunofluorescent (bottom) images of SNs on day 16 and 20. For immunofluorescence, SNs were fixed on the indicated days and stained for TUJ1 (green), BRN3A (red), and DAPI (blue).</p>
<p>(C) Expression of the pan-SN marker <i>BRN3A</i> mRNA by RT-qPCR. n=4 (biological replicates), one-way ANOVA followed by Dunnett’s multiple comparisons. Graph show mean ± S.D. ∗p &lt; 0.05, ∗∗∗p &lt; 0.001.</p>
<p>(D) Representative images of failed differentiations. On day 12, NCCs showing suboptimal phenotypes (low number or complete lack of ridges) were replated. On the indicated days, brightfield images were taken (left) or fixed and stained (right). Fixed SNs were stained for αSMA (green), TUJ1 (red), and DAPI (blue). Note the low number of dendrites and the presence of a high number of non-neural ectoderm cells (arrows).</p>
</figcaption>
</figure>
<h3 id="sec3.3">Differentiation into SNs (mechanoreceptor enrichment)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ≥ 8 days</div>
<p>Here we describe the steps to differentiate NCCs into sensory neurons with an enrichment of mechanoreceptors. By replating on day 6, you will obtain a majority of mechanoreceptors which aggregate in ganglia-like structures. SNs express pan-SN marker BRN3A as well as the mechanoreceptor markers RET and TRKB (<a href="#fig3">Figure 3</a>A). In these conditions, SNs can be maintained for at least 30 days.</p>
<div class="note">
<span class="note-title">Note:</span> This protocol details the differentiation of one 24-well plate of SNs from three wells of the 6-well plate of differentiated NCCs. The volumes will need to be scaled up if more plates are needed.</div>
<ol start="22">
<li>On day 6 of the NC differentiation, wash each of the three designated wells of the 6-well plate with 1 mL DPBS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The remaining three wells will be processed on step 40.</div>
<ol start="23">
<li>Aspirate DPBS.</li>
<li>Add 1 mL/well Accutase and incubate at 37°C for 20 min (or until the cells start to detach).</li>
<li>Add 1 mL DPBS to each well to resuspend the cells and transfer them to a 50 mL conical tube using a p1000.</li>
<li>Fill up with DPBS.</li>
<li>Centrifuge, 200×<i>g</i>, 5 min, RT.</li>
<li>Resuspend the cells in 30 mL DPBS by pipetting up to six times using a 5 mL pipette, trying to gently break up the clumps.</li>
<li>Add 150 μL DNaseI (from a 10 mg/mL stock) and mix by inversion.</li>
<li>Centrifuge, 200×<i>g</i>, 5 min, RT.</li>
<li>Resuspend in 5 mL of SN Differentiation media + 1 μM DAPT.</li>
<li>Gently break all the clumps by pipetting with a p1000 (&lt;6 times).</li>
<li>Count cells.</li>
<li>Aspirate LM/FN mixture from the PO/LM/FN-coated 24-well plate.</li>
<li>Transfer 12 × 10<sup>6</sup> cells (to seed one 24-well plate at a density of 250,000 cells/cm<sup>2</sup>) and fill up to 12 mL with SN Differentiation media + 1 μM DAPT.</li>
<li>Add 0.5 mL of the cell suspension to each well of the 24-well plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> replating is necessary to increase the differentiation efficiency.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Resuspended NCCs can be cryopreserved at a concentration of 5 × 10<sup>6</sup> cell in 1 mL NCC Differentiation media 2 containing 10% DMSO. The viability using this method is ∼15%. Commercially available freezing reagent (such as STEM-CELLBANKER, amsbio, cat# 11890) provide a better viability (∼70%).</div>
<ol start="37">
<li>Incubate cells at 37°C for 16–20 h.</li>
<li>On the next day, aspirate the media and add 1 mL SN Differentiation media + 1 μM DAPT per well of the 24-well plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Media should be added to the walls of the wells slowly. Avoid adding the media directly to the SNs as this can cause detachment of the newly plated cells (See troubleshooting <a href="#sec6.3">problem 2</a>).</div>
<ol start="39">
<li>Replace the media every 2–3 days until day ∼30.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After day 14 (eight days post-replating), the number of SOX10<sup>+</sup> progenitor cells will decrease so DAPT can be removed from the SN Differentiation media (See troubleshooting <a href="#sec6.5">problem 3</a>). If a high number of non-neural ectoderm cells arise, please see troubleshooting <a href="#sec6.7">problem 4</a> and troubleshooting <a href="#sec6.9">problem 5</a>.</div>
<h3 id="sec3.4">Differentiation into SNs (nociceptors, mechanoreceptors, and proprioceptors)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ≥ 8 days</div>
<p>Here, we describe the steps to differentiate NCCs into SNs at a DRG-like ratio. By replating on day 12, you will obtain a mixed peripheral SN population of nociceptors, mechanoreceptors, and proprioceptors. SNs aggregate in ganglia-like structures (<a href="#fig3">Figure 3</a>B). In these conditions, SNs can be maintained for over 90 days. SNs can also be replated a second time after day 16 for additional experiments or to maintain them for a longer time period (<a href="#bib2">Saito-Diaz et al., 2021</a>).</p>
<div class="note">
<span class="note-title">Note:</span> This protocol details the differentiation of one 24-well plate of SNs from the three remaining wells of one 6-well plate of differentiated NCCs.</div>
<div class="note">
<span class="note-title">Note:</span> these steps need to be carried out on day 12 and continue from step 21</div>
<ol start="40">
<li>On day 12 of the NC differentiation, wash each well with 1 mL DPBS.</li>
<li>Add 1 mL/well Accutase and incubate at 37°C for 20 min (or until the cells start to detach).</li>
<li>Add 1 mL DPBS to each well to resuspend the cells and transfer them to a 50 mL conical tube using a p1000.</li>
<li>Fill up with DPBS.</li>
<li>Centrifuge, 200×<i>g,</i> 5 min, RT.</li>
<li>Aspirate the supernatant and add 30 mL DPBS.</li>
<li>Resuspend the cells by pipetting up to six times using a 5 mL pipette, trying to gently break up the clumps.</li>
<li>Add 150 μL DNaseI (from a 10 mg/mL stock) and mix by inversion.</li>
<li>Centrifuge, 200×<i>g</i> for 5 min at RT.</li>
<li>Resuspend in 5 mL of SN Differentiation media + 1 μM DAPT using a 5 mL pipette.</li>
<li>Gently break all the clumps by pipetting with a p1000 (&lt;6 times).</li>
<li>Count cells.</li>
<li>Aspirate LM/FN mix from the PO/LM/FN-coated 24-well plate.</li>
<li>Transfer 12 × 10<sup>6</sup> cells (for one 24-well plate at a density of 250,000 cells/cm<sup>2</sup>) to a 15 mL conical tube and fill up to 12 mL with SN Differentiation media + 1 μM DAPT.</li>
<li>Add 0.5 mL to the cell suspension (from step 49) to each well of the 24-well plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> replating is necessary to increase the differentiation efficiency.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Resuspended NCCs can be cryopreserved using NCC Differentiation media 2 (see above for details on how to prepare the media) containing 10% DMSO or commercially available freezing reagents (see step 36). We recommend freezing at a concentration of 5 × 10<sup>6</sup> cells/mL.</div>
<ol start="55">
<li>Incubate the cells at 37°C for 16–20 h.</li>
<li>Next day, aspirate the media and add 1 mL SN Differentiation media + 1 μM DAPT to each well.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Media should be added to the walls of the wells slowly. Avoid adding the media directly to the SNs as this can cause detachment of the newly plated cells (See troubleshooting <a href="#sec6.3">problem 2</a>).</div>
<ol start="57">
<li>Replace the media every 2–3 days until day 30. After 30 days, media can be changed every five days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After day 20 (eight days post-replating), DAPT is not necessary in the SN Differentiation media (see above for details). See troubleshooting <a href="#sec6.5">problem 3</a>. If a high number of non-neural ectoderm cells arise, please see troubleshooting <a href="#sec6.7">problem 4</a> and troubleshooting <a href="#sec6.9">problem 5</a>.</div>
<div class="note">
<span class="note-title">Note:</span> SNs can be replated a second time after day 16. This can be used as an axotomy model (<a href="#bib2">Saito-Diaz et al., 2021</a>). To do so: 1) Wash SNs with DPBS, followed by incubation at 37°C with Accutase for at least 1 h. 2) Transfer the cells to a 50 mL conical tube with 30 mL DPBS and centrifuge it at 200x g for 4 min at RT. 3) Resuspend the pellet in 1 mL of SN Differentiation media using a p1000 micropipette to break all the clumps. 4) Add 4 mL of SN Differentiation media. 5) Count the cells and replate at a density of 250,000 cells/cm<sup>2</sup> on a PO/LM/FN plate (see above). 6) Change the media the next day.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>In the NC stage of the differentiation, multiple circular or irregular-shaped ridges of different sizes appear starting on day 4 (<a href="#fig2">Figure 2</a>A). These ridges are SOX10 positive as shown by immunofluorescence (<a href="#fig2">Figure 2</a>A). Furthermore, SOX10 expression measured by RT-qPCR should be approximately 100-fold higher compared to hPSCs (<a href="#fig2">Figure 2</a>C). Additionally, the number of NCCs peaks on day 8 (∼90% of viable cells) as measured by the number of cells expressing the surface marker CD49d (which correlates with SOX10 expression (<a href="#bib4">Zeltner et al., 2016</a>)) by flow cytometry. Finally, by day 12 we observe expression of the pan-neuronal marker TUJ1 (<a href="#bib2">Saito-Diaz et al., 2021</a>). It is important to note that a lack of NCC ridges is an indicative of a failed differentiation (<a href="#fig2">Figure 2</a>D) (Troubleshooting 1).</p>
<p>On day 6, NCCs can be replated to generate SNs that by day 14 express the pan-sensory neuronal marker BRN3A as well as the mechanoreceptor-specific markers RET and TRKB (<a href="#fig3">Figure 3</a>A). Furthermore, approximately ∼80% of all cells are mechanoreceptors (TRKB<sup>+</sup>) (<a href="#bib2">Saito-Diaz et al., 2021</a>).</p>
<p>Alternatively, NCCs can be replated on day 12 to generate SNs that also BRN3A measured by immunofluorescence and RT-qPCR (<a href="#fig3">Figures 3</a>B and 3C). By day 20, the earliest SNs appear, at approximately 70% of the total cell population (Saito-Diaz et al<i>.</i>, 2021). Furthermore, all SN subtypes are differentiated at a ratio of roughly 70% nociceptors (TRKA<sup>+</sup>), 30% mechanoreceptors (TRKB<sup>+</sup>), and 30% proprioceptors (TRKC<sup>+</sup>) (<a href="#bib2">Saito-Diaz et al., 2021</a>). It is important to note that in this protocol some SNs (i.e., mechanoreceptors) are double labeled for TRKB<sup>+</sup>/C<sup>+</sup>, which explains why the total proportion is higher than 100%. Additional characterization could include analysis of expression of additional SN markers such as ISL1, PRPH, and RET. A failed differentiation will be characterized by the low number of SNs and presence of non-neural cells, mostly expressing smooth muscle actin (αSMA) (<a href="#fig3">Figure 3</a>D). We have observed the same outcomes using healthy iPSC-derived SNs (∼70% from total population). In contrast, SNs differentiated from iPSCs derived from patients with the peripheral neuropathy familial Dysautonomia comprise ∼20% of the total population (Saito-Diaz et al<i>.</i>, 2021). This correlates with the phenotypes observed in patients which demonstrates that this protocol is a powerful tool to model diseases of the PNS.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>We have observed that approximately one out of every five differentiations does not yield high NCC ridge formation and SOX10 expression, resulting in a low number of SNs (Troubleshooting 1). This number (below the projected 70%), may result in an increased number of non-neural ectoderm cells which express αSMA (<a href="#fig3">Figure 3</a>D). Also, due to the very similar developmental pathway, it is possible that the cultures contain trigeminal neurons, which we haven’t been able to completely rule out due to the lack of specifically differentiating markers between the two lineages. Finally, a limited number of Schwann cells, (expressing the markers myelin basic protein (MBP), myelin protein zero (MPZ), and SOX10) sometimes can be obtained in this protocol and can be found along the axons of the SNs.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>No formation of NCCs ridges or appearance of undetermined cell types and blister-like aggregates (step 21, <a href="#fig2">Figure 2</a>D).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Some hPSCs (including iPSCs) have high levels of intrinsic BMP4 expression, which can cause a failure to generate NCCs. We recommend titrating BMP4, starting at 0 ng/mL up to 1 ng/mL, to identify the concentration that results in the highest SOX10 expression and formation of NCC ridges and might remove blister-like aggregates. Also, different sources of BMP4 have different activity, which introduce variability to the efficiency.</p>
<p>Use a lower passage number of hPSCs. We have found that maintaining the cells in culture for over 4–5 consecutive passages reduces the rate of successful differentiations. Additionally, using the CryoPause method drastically improved differentiation efficiency (<a href="#bib3">Wong et al., 2017</a>).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>NCCs and SNs in culture are lifting off the plate (steps 19, 38, and 56).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>At the NCC stage we would recommend performing the differentiation in Geltrex instead of VTN. For SNs cultured for longer periods of time, we recommend replating them on a later date following the steps described on step 57. We have successfully replated SNs from day 16 up to day 50 of differentiation.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Presence of SOX10<sup>+</sup> cells in SN culture even after adding DAPT to SN Differentiation media after eight days post-replating (steps 39 and 57).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>SOX10<sup>+</sup> might indicate presence of proliferating undifferentiated NCCs. This can be confirmed by co-staining of SOX10 and a proliferation marker (e.g., Ki67). If you identify multiple co-stained cells, titrate DAPT concentration (up to 10 μM) in the SN Differentiation media. It would also be advisable to continue the treatment with DAPT beyond eight days post-replating. Additionally, you can use antimitotic agents, such as mitomycin C or Cytosine β-D arabinofuranoside after replating (on day 6 or day 12) to remove proliferating cells. If there is a population of SOX10<sup>+</sup> Ki67<sup>-</sup> cells, this might suggest the presence of Schwann cells. RT-qPCR of additional Schwann cell markers such MBP and MPZ can confirm the presence of Schwann cells.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Presence of a high number of non-neural ectoderm cells and few SNs (steps 39 and 57, <a href="#fig3">Figure 3</a>D).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Replating NCCs at a low density (&lt;100,000 cells/cm<sup>2</sup>) causes differentiation into αSMA<sup>+</sup> non-neural ectoderm cells. Titrate the number of cells per cm<sup>2</sup> that best suits your applications. We recommend testing a range between 100,000 – 400,000 cells/cm<sup>2</sup>.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>There is a large variability between differentiations (steps 39 and 57).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>We have observed that hPSCs at different passage numbers tend to cause variable differentiation efficiencies. If this is the case, we recommend using the CryoPause method (<a href="#bib3">Wong et al., 2017</a>), which will allow the differentiation from a well characterized hPSC population. This will decrease variability between differentiations.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chen, G., Gulbranson, D.R., Hou, Z., Bolin, J.M., Ruotti, V., Probasco, M.D., Smuga-Otto, K., Howden, S.E., Diol, N.R., Propson, N.E., et al. (2011). Chemically defined conditions for human iPSC derivation and culture. Nat. Methods <i>8</i>, 424-429. <a class="external-link" href="https://doi.org/10.1038/nmeth.1593">https://doi.org/10.1038/nmeth.1593</a></p>
<p id="bib2">Saito-Diaz, K., Street, J.R., Ulrichs, H., and Zeltner, N. (2021). Derivation of peripheral nociceptive, mechanoreceptive, and proprioceptive sensory neurons from the same culture of human pluripotent stem cells. Stem Cell Rep. <i>16</i>, 446-457. <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2021.01.001">https://doi.org/10.1016/j.stemcr.2021.01.001</a></p>
<p id="bib3">Wong, K.G., Ryan, S.D., Ramnarine, K., Rosen, S.A., Mann, S.E., Kulick, A., De Stanchina, E., Muller, F.J., Kacmarczyk, T.J., Zhang, C., et al. (2017). CryoPause: a new method to immediately initiate experiments after cryopreservation of pluripotent stem cells. Stem Cell Rep. <i>9</i>, 355-365. <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2017.05.010">https://doi.org/10.1016/j.stemcr.2017.05.010</a></p>
<p id="bib4">Zeltner, N., Fattahi, F., Dubois, N.C., Saurat, N., Lafaille, F., Shang, L., Zimmer, B., Tchieu, J., Soliman, M.A., Lee, G., et al. (2016). Capturing the biology of disease severity in a PSC-based model of familial dysautonomia. Nat. Med. <i>22</i>, 1421-1427. <a class="external-link" href="https://doi.org/10.1038/nm.4220">https://doi.org/10.1038/nm.4220</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. Nadja Zeltner (<a href="mailto:nadja.zeltner@uga.edu">nadja.zeltner@uga.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any datasets or code.</p>
<h3>Acknowledgments</h3>
<p>We want to thank Dr. Lorenz Studer for his support at the initial stages of this work and Jessica McAlpine for critical reading of the manuscript. This work was funded by faculty start-up funds from the <a href="https://doi.org/10.13039/100007699">University of Georgia</a> to N.Z. and NIH/<a href="https://doi.org/10.13039/100000065">NINDS</a> 1R01NS114567-01A1 to N.Z. Schematics were done using <a href="http://Biorender.com">Biorender.com</a>.</p>
<h3>Author contributions</h3>
<p>N.Z. and K.S-D. conceived and designed the experiments; K.S-D. conducted the experiments; K.S-D. and N.Z. analyzed and interpreted the data; K.S-D. and N.Z. wrote the manuscript; N.Z. provided mentoring, financial, and administrative support and approved the final version of manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests. This work is linked to the patent ‘Compositions and methods for making sensory neurons’, Serial number is 63/127,9, held by the University of Georgia.</p>
</section>
</article>