<article>
<section>
<h1 id="header">XACT-seq: A photocrosslinking-based technique for detection of the RNA polymerase active-center position relative to DNA in <i>Escherichia coli</i>
</h1>
<p><time datetime="2021-10-08">Published: October 8, 2021</time></p>
<p>Chirangini Pukhrambam,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Irina O. Vvedenskaya,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Bryce E. Nickels<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Genetics and Waksman Institute, Rutgers University, Piscataway, NJ 08854, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:bnickels@waksman.rutgers.edu">bnickels@waksman.rutgers.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100858">10.1016/j.xpro.2021.100858</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>XACT-seq (“crosslink between active-center and template sequencing”) is a technique for high-throughput, single-nucleotide resolution mapping of RNA polymerase (RNAP) active-center positions relative to the DNA template. XACT-seq overcomes limitations of approaches that rely on analysis of the RNA 3′ end (e.g., native elongating transcript sequencing) or that report RNAP positions with low resolution (e.g., ChIP-seq and ChIP-exo). XACT-seq can be used to map RNAP active-center positions in transcription initiation complexes, initially transcribing complexes, and transcription elongation complexes.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref10">Winkelman et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for mapping of RNA polymerase (RNAP) active-center position relative to DNA</li>
<li>Can be applied to transcription initiation, initial transcription, and elongation</li>
<li>RNAP active center positions for a library of 4<sup>11</sup> (∼4,000,000) sequences in vivo</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1063-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Introduction</h3>
<p>XACT-seq involves formation of transcription complexes using an RNAP derivative that has the photo-activatable crosslinking amino acid p-benzoyl-L-phenylalanine (Bpa) (<a href="#bib3">Chin et al., 2002</a>) incorporated at RNAP β′ subunit residue R1148 (RNAP-β′<sup>R1148Bpa</sup>) which, upon photo activation, forms covalent crosslinks with DNA at a position exactly 5 nt downstream of the RNAP active-center A site (<a href="#fig1">Figures 1</a>A and 1B) (<a href="#bib11">Yu et al., 2017</a>). XACT-seq is performed using merodiploid <i>Escherichia coli</i> cells containing a plasmid-encoded, decahistidine-tagged RNA polymerase (RNAP) derivative and a chromosomally encoded, untagged RNAP (<a href="#fig1">Figure 1</a>C). In this case, plasmid, pIA900-β′<sup>R1148Bpa</sup>, contains a gene for the RNAP β′ subunit with a nonsense codon (TAG) at position 1148 and a decahistidine coding sequence (<a href="#fig1">Figure 1</a>C). A second plasmid, pEVOL-pBpF, contains a gene for an engineered Bpa specific UAG suppressor tRNA and a gene for an engineered Bpa specific aminoacyl tRNA synthetase (<a href="#fig1">Figure 1</a>C). Growth of cells containing both plasmids in media supplemented with Bpa results in production of β′<sup>R1148Bpa</sup> and corresponding formation of transcription complexes containing RNAP-β′<sup>R1148Bpa</sup> (<a href="#fig1">Figures 1</a>B and 1C).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1063-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. XACT-seq approach to map RNAP active-center A-site positions</div>
<p>(A) RNAP derivatives containing photoactivatable agent (green) that crosslinks 5 bp downstream of the RNAP active-center A site position. Light gray, RNAP; dark gray, decahistidine tag; brown, initiation factor σ<sup>70</sup>; σR2 and σR4, σ<sup>70</sup> promoter binding domains.</p>
<p>(B) Transcription complexes containing RNAP-β′<sup>R1148Bpa</sup>. Black, DNA (nontemplate strand above template strand; position of transcription bubble indicated by separated strands); red, RNA; blue, promoter -35 and -10 elements; black bracket, RNAP active-center P site; purple bracket, RNAP active-center A site; green line, position of crosslinking; bulged out transcription-bubble DNA indicates expansion of the bubble (“DNA scrunching”).</p>
<p>(C) Merodiploid system for co-production of decahistidine-tagged RNAP-β′<sup>R1148Bpa</sup> in the presence of untagged wild-type RNAP. Plasmid pEVOL-pBpF contains genes for engineered Bpa-specific nonsense-suppressor tRNA and aminoacyl-tRNA synthetase (white boxes), plasmid pIA900-β′<sup>R1148Bpa</sup> contains a gene for RNAP βʹ subunit (gray box) with nonsense codon (green) at residue βʹR1148; and chromosome carries genes for wild-type RNAP core subunits (<i>rpoA</i>, <i>rpoB</i>, and <i>rpoC</i>; gray boxes) and σ<sup>70</sup> (<i>rpoD</i> brown box).</p>
</figcaption>
</figure>
<p>To initiate covalent crosslinking of RNAP to DNA, cells are UV irradiated (<a href="#fig2">Figure 2</a>, step 1). After UV irradiation, cells are lysed, RNAP-DNA complexes are recovered by immobilized metal ion affinity chromatography (IMAC) targeting the decahistidine tag on RNAP-β′<sup>R1148Bpa</sup> (<a href="#fig2">Figure 2</a>, step 2), and crosslink positions and crosslink yields are defined by high-throughput sequencing of primer extension products (<a href="#fig2">Figure 2</a>, steps 3–6). The procedure enables the position of the RNAP active-center relative to DNA to be mapped, <i>in vivo</i>, with single-nucleotide resolution.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1063-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Steps in XACT-seq</div>
</figcaption>
</figure>
<p>Below we describe use of XACT-seq to analyze transcription complexes associated with up to at least ∼4 million promoter sequences in <i>E. coli</i> (<a href="#fig3">Figure 3</a>). In this case, XACT-seq is performed with cells containing a plasmid-borne library of barcoded template sequences (e.g., p<i>lac</i>CONS-N11, see <a href="#fig3">Figure 3</a>). For the details on construction of p<i>lac</i>CONS-N11 library see (<a href="#bib9">Vvedenskaya et al., 2015</a>, <a href="#bib7">2018</a>; <a href="#bib10">Winkelman et al., 2020</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1063-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Three plasmid system to apply XACT-seq to a promoter library of up to at least ∼4 million sequences</div>
<p>Top shows cells containing decahistidine-tagged RNAP-β′<sup>R1148Bpa</sup> and a promoter library of ∼4 million sequences (plasmid p<i>lac</i>CONS-N11). Bottom shows sequence of the p<i>lac</i>CONS-N11 promoter library. Blue, promoter -35 and -10 elements; dark yellow, randomized sequences from position +3 to +13; light yellow, transcribed region barcode sequence. Other symbols and colors as in <a href="#fig1">Figure 1</a>.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Prepare growth media and buffers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol>
<li>
<u>Lysogeny Broth (LB)</u>: dissolve components in 1 L of ddH<sub>2</sub>O and autoclave. Store at 25°C.</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bacto tryptone</td>
<td>10 g</td>
</tr>
<tr>
<td>Bacto yeast extract</td>
<td>5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>10 g</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>maximum time for storage:</b> 2 months</p>
<ol start="2">
<li>
<u>LB Agar</u>: dissolve components in 1 L of ddH<sub>2</sub>O and autoclave. Pour ∼25 mL of media into each 100 <b>×</b> 15 mm Petri dish. Store at 4°C.</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bacto tryptone</td>
<td>10 g</td>
</tr>
<tr>
<td>Bacto yeast extract</td>
<td>5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>10 g</td>
</tr>
<tr>
<td>Agar</td>
<td>15 g</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>maximum time for storage:</b> 2 months</p>
<ol start="3">
<li>
<u>Resuspension Buffer</u>: combine components in 0.9 L of ddH<sub>2</sub>O, adjust pH to 8 and add ddH<sub>2</sub>O to 1 L. Filter using a sterile vacuum filter unit (1 L, 0.2 μm pore size). Store at 25°C.</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount/volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub> • 7H<sub>2</sub>O</td>
<td>0.05 M</td>
<td>13.4 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>1.4 M</td>
<td>81.9 g</td>
</tr>
<tr>
<td>Imidazole (C<sub>3</sub>H<sub>4</sub>N<sub>2</sub>)</td>
<td>0.02 M</td>
<td>1.36 g</td>
</tr>
<tr>
<td>Tween 20</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Ethanol (100%)</td>
<td>5%</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>maximum time for storage:</b> 6 months</p>
<div class="note">
<span class="note-title">Note:</span> immediately before use, add β-mercaptoethanol (BME) to a concentration of 15 mM.</div>
<ol start="4">
<li>
<u>Ni-NTA Wash Buffer</u>: combine components in 0.9 L of ddH<sub>2</sub>O, adjust pH to 8 and add ddH<sub>2</sub>O to 1 L. Filter using a sterile vacuum filter unit (1 L, 0.2 μm pore size). Store at 4°C.</li>
</ol>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount/volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub> • 7H<sub>2</sub>O</td>
<td>50 mM</td>
<td>13.4 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>300 mM</td>
<td>17.5 g</td>
</tr>
<tr>
<td>Imidazole (C<sub>3</sub>H<sub>4</sub>N<sub>2</sub>)</td>
<td>30 mM</td>
<td>2.0 g</td>
</tr>
<tr>
<td>Tween 20</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Ethanol (100%)</td>
<td>5%</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>maximum time for storage:</b> 6 months</p>
<div class="note">
<span class="note-title">Note:</span> immediately before use, add BME to a concentration of 15 mM.</div>
<ol start="5">
<li>
<u>Ni-NTA Elution Buffer</u>: combine components in 0.9 L of ddH<sub>2</sub>O, adjust pH to 8 and add ddH<sub>2</sub>O to 1 L. Filter using a sterile vacuum filter unit (1 L, 0.2 μm pore size). Store at 4°C.</li>
</ol>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount/volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub> • 7H<sub>2</sub>O</td>
<td>50 mM</td>
<td>13.4 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>300 mM</td>
<td>17.5 g</td>
</tr>
<tr>
<td>Imidazole (C<sub>3</sub>H<sub>4</sub>N<sub>2</sub>)</td>
<td>300 mM</td>
<td>20.4 g</td>
</tr>
<tr>
<td>Tween 20</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>100% Ethanol</td>
<td>5%</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>maximum time for storage:</b> 6 months</p>
<div class="note">
<span class="note-title">Note:</span> immediately before use, add BME to a concentration of 15 mM.</div>
<ol start="6">
<li>
<u>Storage Buffer (2</u><b>×</b><u>)</u>: combine components in 80 mL of ddH<sub>2</sub>O, adjust pH to 8 and add ddH<sub>2</sub>O to 100 mL. Filter using a sterile vacuum filter unit (0.2 μm pore size). Store at 4°C.</li>
</ol>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-Cl pH 8.0 (1 M)</td>
<td>20 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>KCl (2 M)</td>
<td>200 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>20 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.2 mM</td>
<td>0.04 mL</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>1 mM</td>
<td>0.1 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>maximum time for storage:</b> 6 months</p>
<ol start="7">
<li>
<u>TBE (10</u><b>×</b><u>)</u>: dissolve components in 1 L of ddH<sub>2</sub>O. Store at 25°C.</li>
</ol>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris base</td>
<td>20 mM</td>
<td>108 g</td>
</tr>
<tr>
<td>Boric Acid</td>
<td>200 mM</td>
<td>55 g</td>
</tr>
<tr>
<td>EDTA, disodium salt, dihydrate</td>
<td>20 mM</td>
<td>7.4 g</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>maximum time for storage:</b> 4 months</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial strains</td>
</tr>
<tr>
<td>NiCo21 (DE3)</td>
<td>NEB</td>
<td>Cat#C2529H</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Nuclease-Free Water (not DEPC-treated)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9932</td>
</tr>
<tr>
<td>Bacto agar</td>
<td>VWR</td>
<td>Cat#90000-760</td>
</tr>
<tr>
<td>Bacto tryptone</td>
<td>VWR</td>
<td>Cat#90000-286</td>
</tr>
<tr>
<td>Bacto yeast extract</td>
<td>VWR</td>
<td>Cat#90000-726</td>
</tr>
<tr>
<td>Chloramphenicol</td>
<td>Gold Biotech</td>
<td>Cat#C-105-25</td>
</tr>
<tr>
<td>Spectinomycin</td>
<td>Duchefa Biochemie</td>
<td>Cat#S0188.0025</td>
</tr>
<tr>
<td>Streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>Carbenicillin</td>
<td>Gold Biotech</td>
<td>Cat#C-103-25</td>
</tr>
<tr>
<td>Rifampicin</td>
<td>Gold Biotech</td>
<td>Cat#R-120-25</td>
</tr>
<tr>
<td>H-Bpa-OH</td>
<td>Bachem</td>
<td>Cat#F-2800</td>
</tr>
<tr>
<td>IPTG Gold</td>
<td>Biotech</td>
<td>Cat#I2481C50</td>
</tr>
<tr>
<td>SOC Outgrowth Medium</td>
<td>NEB</td>
<td>Cat#B9020S</td>
</tr>
<tr>
<td>Lysozyme Egg White</td>
<td>Gold Biotech</td>
<td>Cat#L-040-10</td>
</tr>
<tr>
<td>Glycerol, nuclease free</td>
<td>VWR</td>
<td>Cat#EM 4750</td>
</tr>
<tr>
<td>Tris base (Amresco)</td>
<td>VWR</td>
<td>Cat#97061-800</td>
</tr>
<tr>
<td>Boric acid (ACS grade)</td>
<td>VWR</td>
<td>Cat#97061-980</td>
</tr>
<tr>
<td>EDTA disodium salt dihydrate</td>
<td>VWR</td>
<td>Cat#97061-018</td>
</tr>
<tr>
<td>Imidazole</td>
<td>VWR</td>
<td>Cat#EM5720</td>
</tr>
<tr>
<td>Formamide, deionized</td>
<td>VWR</td>
<td>Cat#EM-4610</td>
</tr>
<tr>
<td>Sodium dodecylsulfate (SDS)</td>
<td>VWR</td>
<td>Cat#97064-470</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>VWR</td>
<td>Cat#EM-BX1410-7</td>
</tr>
<tr>
<td>Xylene Cyanol</td>
<td>Sigma-Aldrich</td>
<td>Cat#X4126-10G</td>
</tr>
<tr>
<td>0.5 M EDTA pH 8</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9260G</td>
</tr>
<tr>
<td>3 M Sodium Acetate pH 5.5</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9740</td>
</tr>
<tr>
<td>BSA, Molecular Biology Grade</td>
<td>NEB</td>
<td>Cat#B9000S</td>
</tr>
<tr>
<td>5 M Betaine solution</td>
<td>VWR</td>
<td>Cat#101375-612</td>
</tr>
<tr>
<td>Dimethyl sulfoxide</td>
<td>VWR</td>
<td>Cat#BDH1115-1LP</td>
</tr>
<tr>
<td>Heparin sulfate</td>
<td>Sigma-Aldrich</td>
<td>Cat#H-3393</td>
</tr>
<tr>
<td>Glycogen from oyster (type II)</td>
<td>Sigma-Aldrich</td>
<td>Cat#G8751</td>
</tr>
<tr>
<td>Ethyl alcohol</td>
<td>Pharmco-AAPER</td>
<td>Cat#111000200</td>
</tr>
<tr>
<td>Isopropyl alcohol</td>
<td>VWR</td>
<td>Cat#BDH1133-1LP</td>
</tr>
<tr>
<td>Low Range ssRNA Ladder</td>
<td>NEB</td>
<td>Cat#N0364S</td>
</tr>
<tr>
<td>O’Gene Ruler Ultra Low Range DNA Ladder</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#SM1223</td>
</tr>
<tr>
<td>6<b>×</b> Orange DNA Loading Dye</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#R0631</td>
</tr>
<tr>
<td>SYBR Gold Nucleic Acid Gel Stain</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#S11494</td>
</tr>
<tr>
<td>Phenol:Chloroform:IAA pH 8</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9732</td>
</tr>
<tr>
<td>Taq DNA polymerase</td>
<td>NEB</td>
<td>Cat#M0273</td>
</tr>
<tr>
<td>5′ App DNA/RNA ligase</td>
<td>NEB</td>
<td>Cat#M0319S</td>
</tr>
<tr>
<td>T4 RNA Ligase 1 (ssRNA Ligase)</td>
<td>NEB</td>
<td>Cat#M0204L</td>
</tr>
<tr>
<td>Phusion HF DNA polymerase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#F-530L</td>
</tr>
<tr>
<td>Phusion HF Buffer Pack, detergent-free</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#F520L</td>
</tr>
<tr>
<td>dNTP Solution Mix</td>
<td>NEB</td>
<td>Cat#N0447S</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Qubit dsDNA HS Assay Kit</td>
<td>Thermo Fisher Scientific, <a href="https://www.thermofisher.com/invitrogen/qubit-assays">https://www.thermofisher.com/invitrogen/qubit-assays</a>
</td>
<td>Cat#Q32851</td>
</tr>
<tr>
<td>Micellula DNA Emulsion PCR Kit</td>
<td>Chimerx, <a href="http://www.chimerx.com">http://www.chimerx.com</a>
</td>
<td>Cat#3600</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>s128A, oligo used for primer extension of library templates: ccttggcacccgagaattcca</td>
<td><a href="#bib10">Winkelman et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td>s1248, 3′ adapter with 10N at 5′ end (HPLC purified): /5′Phos/nnnnnnnnnngatcgtcggactgtagaactctgaac/3ddC/</td>
<td><a href="#bib10">Winkelman et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td>RP1, Illumina PCR primer (HPLC purified): aatgatacggcgaccaccgagatctacacgttcagagttctacagtccga</td>
<td><a href="#bib10">Winkelman et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td>RPI1, Illumina indexing PCR primer 1 (HPLC purified): caagcagaagacggcatacgagatcgtgatgtgactg gagttccttggcacccgagaattcca</td>
<td><a href="#bib10">Winkelman et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td>s1115, custom Illumina sequencing primer: ctacacgttcagagttctacagtccgacgatc</td>
<td><a href="#bib10">Winkelman et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pIA900-RNAP-β′<sup>R1148Bpa</sup>
</td>
<td><a href="#bib12">Winkelman et al., 2015</a></td>
<td>N/A</td>
</tr>
<tr>
<td>pEVOL-pBpF</td>
<td>Gift from Peter Schultz (<a href="#bib3">Chin et al., 2002</a>)</td>
<td>Addgene plasmid # 31190, <a href="https://www.addgene.org">https://www.addgene.org</a>
</td>
</tr>
<tr>
<td>p<i>lac</i>CONS-N11</td>
<td><a href="#bib10">Winkelman et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>XACT-Seq</td>
<td><a href="#bib10">Winkelman et al. (2020)</a></td>
<td><a href="https://github.com/NickelsLabRutgers/XACT-seq">https://github.com/NickelsLabRutgers/XACT-seq</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>MicroPulser Electroporator</td>
<td>Bio-Rad</td>
<td>Cat#1652662</td>
</tr>
<tr>
<td>Electroporation cuvettes 1 mm gap</td>
<td>VWR</td>
<td>Cat#58017-890</td>
</tr>
<tr>
<td>Cel-Gro Tissue Culture Rotator</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Rayonet RPR-100 photochemical reactor</td>
<td>Southern New England Ultraviolet</td>
<td>N/A</td>
</tr>
<tr>
<td>Sonics Vibra Cell VCX 130</td>
<td>Sonics</td>
<td>N/A</td>
</tr>
<tr>
<td>Belly Dancer, Stovall</td>
<td>Life Sciences</td>
<td>N/A</td>
</tr>
<tr>
<td>2200 TapeStation</td>
<td>Agilent Technologies, Inc.</td>
<td>N/A</td>
</tr>
<tr>
<td>Nalgene Sterile Filter Unit, 1 L</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#567-0020</td>
</tr>
<tr>
<td>Disposable Borosilicate Glass Culture Tubes</td>
<td>VWR</td>
<td>Cat#47729-572</td>
</tr>
<tr>
<td>Ni-NTA Agarose</td>
<td>QIAGEN</td>
<td>Cat#30230</td>
</tr>
<tr>
<td>Amicon Ultra-4 100K Centrifugal Filters</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#UFC810024</td>
</tr>
<tr>
<td>MagneHis Ni Particles</td>
<td>Promega</td>
<td>Cat#V8560</td>
</tr>
<tr>
<td>Promega MagneSphere Magnetic Separation Stand</td>
<td>Promega</td>
<td>Cat#Z5343</td>
</tr>
<tr>
<td>Spin-X centrifuge tube filter, 0.45 μm, RNase/DNase free</td>
<td>Costar</td>
<td>Cat#8162</td>
</tr>
<tr>
<td>10% TBE-Urea gels, 1 mm <b>×</b> 10 wells</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EC6875Box</td>
</tr>
<tr>
<td>10% TBE gels, 1 mm <b>×</b> 10 wells</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EC6275Box</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Introduction of plasmids into <i>E. coli</i> cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼18 h for step 1</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼5 h for step 2</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼18 h for step 3</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼5 h for step 4</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼18 h for step 5</div>
<div class="note">
<span class="note-title">Note:</span> We have found that the sequential introduction of plasmid library p<i>lac</i>CONS-N11, followed by plasmid pIA900-β′<sup>R1148Bpa</sup>, followed by plasmid pEVOL-pBpF into <i>E. coli</i> cells provides consistent results.</div>
<ol>
<li>First step to generate three-plasmid merodiploid system to analyze up to ∼4 million promoter sequences: <u>introduction of placCONS-N11 library into E. coli.</u>
<div class="note">
<span class="note-title">Note:</span> Because of the total number of sequences present in the p<i>la</i>cCONS-N11 library (up to ∼4 million promoter sequences) our lab uses electrocompetent cells for each transformation step. In general, use of electrocompetent cells results in a higher transformation efficiency compared with use of chemically competent cells.</div>
<ol type="a">
<li>Combine 50 μL of NiCo21 (DE3) electrocompetent cells with 1 μL (90–100 ng) of p<i>lac</i>CONS-N11 in a microcentrifuge tube.</li>
<li>Transfer the mixture to a 1 mm electroporation cuvette and apply current using an electroporator. (Our lab uses a MicroPulser electroporator and a pulse time of 3.5–4 ms.)</li>
<li>Add 950 μL of SOC (prewarmed to 37°C), mix by pipetting, and transfer to a 17 mm <b>×</b> 100 mm round-bottom culture tube.</li>
<li>Incubate for 1 h at 37°C with gentle mixing. (Our lab uses a Cel-Gro Tissue Culture Rotator Drum set to ∼55 rpm.)</li>
<li>Remove cells from culture tube and mix with 50 mL of LB containing 50 μg/mL streptomycin and 50 μg/mL spectinomycin in a 250 mL flask.</li>
<li>Shake flask in orbital shaker at 220 rpm for 16 h at 37°C.
<div class="note">
<span class="note-title">Note:</span> we recommend performing steps 1.a-b in a cold room.</div>
<div class="note">
<span class="note-title">Note:</span> The desired number of transformants is ∼5 <b>×</b> 10<sup>6</sup>. To estimate the number of transformants, a serial dilution is prepared using the cell suspension prior to step 1.e. Each dilution is plated on LB agar containing 50 μg/mL streptomycin and 50 μg/mL spectinomycin and is grown for 16 h at 37°C. The number of colonies on each plate is counted. The number of transformants is calculated assuming each colony is derived from an independent transformation event. For additional details see: Troubleshooting, Problem 1.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Next, electrocompetent cells of NiCo21 (DE3) containing the placCONS-N11 library are generated prior to introduction of pIA900-β′<sup>R1148Bpa</sup>.
<ol type="a">
<li>In a 500 mL flask inoculate 100 mL of LB medium containing 50 μg/mL streptomycin and 50 μg/mL spectinomycin with 1 mL of the cell suspension from step 1.f.</li>
<li>Place flask on orbital shaker set to 220 rpm and incubate at 37°C until the cell suspension reaches an OD<sub>600</sub> of 0.5 (∼2 h).</li>
<li>Transfer cell suspensions into two 50-mL centrifuge tubes submerged in ice.</li>
<li>Collect cells by centrifugation (3000 <b>×</b> <i>g</i>; 4°C; 10 min), remove the supernatant and resuspend cells in 50 mL of ddH<sub>2</sub>O kept at 4°C.</li>
<li>Repeat step 2.d two more times.</li>
<li>Resuspend cells in 10 mL of 10% glycerol kept at 4°C. Collect cells by centrifugation (3000 <b>×</b> <i>g</i>; 4°C; 10 min) and remove the supernatant.</li>
<li>Repeat step 2.f two more times and resuspend cells in 0.5 mL of 10% glycerol kept at 4°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not vortex tubes while resuspending cell pellets at wash steps 2.d-g. Instead, swirl the tubes until cells are resuspended. In addition, if you decide to transfer cell suspensions into new tubes at step 2.g, use wide bore pipette tips.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend using freshly prepared competent cells for the next transformation step, otherwise cells can be stored at −80°C for up to 6 months.</div>
<ol start="3">
<li>Second step to generate three-plasmid merodiploid system to analyze up to ∼4 million promoter sequences: <u>introduction of plasmid pIA900-</u><u>β′</u><sup>R1148Bpa</sup> <u>into NiCo21 (DE3) cells containing the placCONS-N11 library</u>.
<ol type="a">
<li>Combine 50 μL of NiCo21 (DE3) cells containing the p<i>lac</i>CONS-N11 library (prepared in step 2.g above) with 1 μL (100–120 ng) of pIA900-β′<sup>R1148Bpa</sup> in a microcentrifuge tube. Transfer the mixture to a 1 mm electroporation cuvette and apply current using an electroporator. (Our lab uses a MicroPulser electroporator and a pulse time of 3.5–4 ms.)</li>
<li>Add 950 μL of SOC (prewarmed to 37°C), mix by pipetting, and transfer to a 17 mm <b>×</b> 100 mm round-bottom culture tube.</li>
<li>Incubate for 1 h at 37°C in Cel-Gro Tissue Culture Rotator Drum set to ∼55 rpm.</li>
<li>Remove cells from culture tube and mix with 50 mL of LB containing 50 μg/mL streptomycin, 50 μg/mL spectinomycin and 100 μg/mL carbenicillin in a 250 mL flask.</li>
<li>Place flask on orbital shaker set to 220 rpm for 16 h at 37°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend performing step 3.a-b in a cold room.</div>
<div class="note">
<span class="note-title">Note:</span> The desired number of transformants is ∼5 <b>×</b> 10<sup>6</sup>. To estimate the number of transformants, prepare a serial dilution of the cell suspension prior to step 3.d. Plate each dilution on LB agar containing 50 μg/mL streptomycin, 50 μg/mL spectinomycin and 100 μg/mL carbenicillin. Incubate plates for 16 h at 37°C. Count the number of colonies on each plate. Calculate the number of transformants assuming each colony is derived from an independent transformation event. For additional details see: Troubleshooting, Problem 1.</div>
<ol start="4">
<li>Next, electrocompetent cells of NiCo21 (DE3) containing pIA900-β′<sup>R1148Bpa</sup> and the placCONS-N11 library are prepared prior to introduction of pEVOL-pBpF.
<ol type="a">
<li>Inoculate 100 mL of LB medium containing 50 μg/mL streptomycin, 50 μg/mL spectinomycin and 100 μg/mL carbenicillin in 500 mL flask with 1 mL of the cell suspension from step 3.e.</li>
<li>Place flask in orbital shaker set to 220 rpm and incubate at 37°C until the cell suspension reaches an OD<sub>600</sub> of 0.5 (∼2 h).</li>
<li>Transfer cell suspensions into two 50 mL centrifuge tubes submerged in ice.</li>
<li>Collect cells by centrifugation (3000 <b>×</b> <i>g</i>; 4°C; 10 min) and remove the supernatant. Resuspend cells in 50 mL of ddH<sub>2</sub>O kept at 4°C.</li>
<li>Perform step 4.d two more times.</li>
<li>Resuspend cells in 10 mL of 10% glycerol kept at 4°C by pipetting. Collect cells by centrifugation (3000 <b>×</b> <i>g</i>; 4°C; 10 min) and remove the supernatant.</li>
<li>Repeat step 4.f two more times and resuspend cells in 0.5 mL of 10% glycerol kept at 4°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not vortex tubes while resuspending cell pellets at wash steps 4.d-g. Instead, swirl the tubes until cells are resuspended. In addition, if you decide to transfer cell suspensions into new tubes at step 4.g, use wide bore pipette tips.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend using freshly prepared competent cells for the next transformation step, otherwise cells can be stored at −80°C for up to 6 months.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Cells can be stored at −80°C for up to 6 months.</div>
<ol start="5">
<li>Third step to generate three-plasmid merodiploid system to analyze up to ∼4 million promoter sequences: <u>introduction of plasmid pEVOL-pBpF into NiCo21 (DE3) cells containing pIA900-</u>β′<sup>R1148Bpa</sup> <u>and the placCONS-N11 library.</u>
<ol type="a">
<li>Combine 50 μL of NiCo21 (DE3) cells containing pIA900-β′<sup>R1148Bpa</sup> and the p<i>lac</i>CONS-N11 library (prepared in step 4.g) with 1 μL (∼8 ng) of pEVOL-pBpF in a microcentrifuge tube. Transfer the mixture to a 1 mm electroporation cuvette and apply current using an electroporator. (Our lab uses a MicroPulser electroporator and a pulse time of 3.5–4 ms.)</li>
<li>Add 950 μL of SOC (prewarmed to 37°C), mix by pipetting, and transfer to a 17 mm <b>×</b> 100 mm round-bottom culture tube.</li>
<li>Incubate for 1 h at 37°C with gentle mixing. (Our lab uses a Cel-Gro Tissue Culture Rotator Drum set to ∼55 rpm.)</li>
<li>Centrifuge cell suspension to pellet cells (21,000 <b>×</b> <i>g</i>; 25°C; 1 min) and remove 800 μL media. Resuspend cells in the remaining 200 μL media. Plate 100 μL of the cell suspension on two LB agar 100 <b>×</b> 15 mm Petri dishes containing 50 μg/mL spectinomycin, 50 μg/mL streptomycin, 100 μg/mL carbenicillin, and 25 μg/mL chloramphenicol.</li>
<li>Incubate plates at 37°C for 16 h.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Perform step 5.a-b in a cold room.</div>
<div class="note">
<span class="note-title">Note:</span> Each LB agar plate will contain a lawn of transformants. We estimate the number of transformants as 11–16 <b>×</b> 10<sup>6</sup> per plate.</div>
<h3 id="sec2.2">Growth and UV irradiation of plasmid-containing cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼6–8 h</div>
<ol start="6">
<li>Expression of Bpa-labeled RNAP in NiCo21 (DE3) cells containing pEVOL-pBpF, pIA900-β′<sup>R1148Bpa</sup>, and the placCONS-N11 library followed by UV-irradiating cells to induce crosslinking between Bpa-labeled RNAP and DNA.
<ol type="a">
<li>Prepare 2.5 mL of 100 mM Bpa by dissolving 67.5 mg Bpa in 2.5 mL of 1 M NaOH.</li>
<li>Prepare 250 mL LB media containing 1 mM Bpa by adding the 100 mM Bpa solution prepared in step 6.a dropwise into 250 mL LB media while stirring. Adjust the pH to 7.2 with 1 M HCl (add ∼ 0.8 mL of 1 M HCl per 100 mL LB media).</li>
<li>Add 100 μg/mL carbenicillin, 50 μg/mL spectinomycin, 50 μg/mL streptomycin and 25 μg/mL chloramphenicol to the media prepared in step 6.b.
<div class="note">
<span class="note-title">Note:</span> Bpa is a photoreactive compound. Therefore, steps 6.a-b should be performed in a room with minimal light and performed in as short a time period as possible.</div>
</li>
<li>To recover transformants from step 5.e, add 7 mL of LB to the surface of each agar plate and resuspend cells using a 6” sterilized wooden applicator. Combine the cell suspensions from each plate in a 50 mL Falcon tube and measure the OD<sub>600</sub> of the mixture.
<div class="note">
<span class="note-title">Note:</span> We recommend making a 1:10 dilution of cell suspension for accurate measurement of OD<sub>600</sub>.</div>
</li>
<li>Using the mixture prepared in step 6.d, inoculate 250 mL LB containing 1 mM Bpa, 100 μg/mL carbenicillin, 50 μg/mL spectinomycin, 50 μg/mL streptomycin and 25 μg/mL chloramphenicol to an OD<sub>600</sub> of ∼0.3. (For example, if the OD<sub>600</sub> of the undiluted cell suspension is 7.5, then adding ∼10 mL of this suspension to 250 mL media will yield an OD<sub>600</sub> of ∼0.3).</li>
<li>Place the mixture in a 1 L flask in an orbital shaker set to 220 rpm and incubate for 1 h at 37°C.
<div class="critical">
<span class="critical-title">Critical:</span> Bpa is a photoreactive compound. Therefore, the exposure of the cultures to light during growth should be minimized by covering culture flasks with aluminum foil and incubating cells in an unlit room or growth chamber.</div>
</li>
<li>Add IPTG to a final concentration of 1 mM. Incubate cultures for additional 3 h at 37°C (220 rpm).</li>
<li>Transfer 9 mL of the cell suspension into a 13 mm <b>×</b> 100 mm borosilicate glass test tubes (total of 27 tubes per sample).</li>
<li>UV-irradiate cell suspensions. To ensure reproducibility, we recommend use of a commercially available photoreactor. Our lab uses a Rayonet RPR-100 photochemical reactor (350 nm wavelength, 20 min, 25°C).</li>
<li>Transfer cell suspensions from tubes to 250-mL centrifugal bottles. Collect cells by centrifugation (3000 × <i>g</i>; 4°C; 10 min), remove supernatant, and store cell pellets at −20°C.
<div class="note">
<span class="note-title">Note:</span> To trap static initial-transcribing complexes containing 2- to 3-nt RNA products <i>in vivo</i>, despite the presence of all NTP substrates <i>in vivo</i>, the RNAP inhibitor rifampin (Rif), which blocks extension of RNA products beyond a length of 2–3 nt, can be added to a final concentration of 200 μg/mL prior to UV irradiation. After adding rifampin, we recommend incubating cells for 10 min at 37°C (220 rpm).</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Cell pellets can be stored at −20°C for 2–3 weeks.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.3">Isolation of crosslinked complexes from cell lysates</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼4–6 h for step 7</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1 h for step 8</div>
<ol start="7">
<li>Use of immobilized metal-ion affinity chromatography (IMAC) to isolate Bpa-labeled RNAP from cell lysate
<ol type="a">
<li>Thaw frozen cell pellets prepared in step 6.j at 4°C for 30 min and resuspend in 40 mL of Resuspension Buffer containing 15 mM BME and 2 mg lysozyme.</li>
<li>Sonicate cells at 4°C (Our lab uses a Sonics Vibra Cell instrument set at 10 s pulse/10 s pause for 5 min).</li>
<li>Remove cell debris by centrifugation (23,000 × <i>g</i>; 30 min; 4°C) to form compact pellet of debris.</li>
<li>Aliquot 1 mL of Ni-NTA agarose in 50-mL Falcon tube. Wash the resin with 1 mL of nuclease-free water and equilibrate in 1 mL of Resuspension Buffer. Repeat the equilibration step two more times.</li>
<li>Add supernatant from step 7.c to the resin. Incubate for 30 min at 4°C with gentle rocking on rotator (Our lab normally uses a Stovall Belly Dancer Shaker). Keep tube on ice and cover with foil to protect from light.</li>
<li>Load the slurry into a 15 mL polyprep column to collect the Ni-NTA-agarose resin.</li>
<li>Wash resin with 10 mL of 1 X Ni-NTA Wash Buffer.</li>
<li>Add 3 mL of 1 X Ni-NTA Elution Buffer containing 300 mM imidazole, incubate 1 min and elute His-tagged proteins from the resin.</li>
<li>Concentrate the eluate to ∼0.1–0.2 mL by centrifugation (3000 × <i>g</i>; 7–12 min, 4°C) using 100K MWCO Amicon Ultra-4 centrifugal filter.</li>
<li>To perform buffer exchange, add 2 mL of 2 X Storage Buffer to the concentrated eluate. Reduce the volume of the mixture to ∼0.1–0.2 mL by centrifugation (3000 × <i>g</i>; 7–12 min, 4°C).</li>
<li>Repeat step 7.j.</li>
<li>Add an equal volume of 100% glycerol (∼0.1–0.2 mL) to the concentrated eluate. Mix thoroughly by pipetting and store sample at −80°C.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Perform steps 7.a-l at 4°C.</div>
<div class="note">
<span class="note-title">Note:</span> The eluate at step 7.l contains Bpa-labeled RNAP that is crosslinked to DNA and Bpa-labeled RNAP that is not crosslinked to DNA.</div>
<div class="note">
<span class="note-title">Note:</span> After step 7.l, the concentration of protein in the eluate is typically ∼3.5–5 μM.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The eluate can be stored at −80°C for at least 3 months.</div>
<ol start="8">
<li>Heat denaturation of crosslinked complexes and binding to MagneHis Ni-particles The next step of the protocol involves heat denaturation of crosslinked complexes followed by binding to MagneHis Ni-particles. This step removes non-crosslinked DNA from the reaction mixture.
<ol type="a">
<li>Aliquot 20 μL MagneHis Ni-particles in two 1.5 mL tubes. Remove storage solution and equilibrate the particles in 50 μL of 1 X <i>Taq</i> DNA polymerase buffer using MagneSphere magnetic separation stand (Promega). Repeat equilibration step. Keep tubes on ice while preparing the crosslinked complexes for binding with MagneHis Ni-particles.</li>
<li>Set up two tubes containing 25 μL of the eluate prepared in step 7.l. Add 25 μL of nuclease-free H<sub>2</sub>O, 12 μL of 5 M NaCl, and 0.5 μL of 1 mg/mL heparin to each tube.</li>
<li>Incubate the mixtures at 95°C for 5 min to denature dsDNA. Cool to 4°C and incubate for 5 min.</li>
<li>Add the denatured mixture to MagneHis Ni-particles from step 8.a.</li>
<li>To bind the crosslinked complexes to beads, incubate for 10 min at 25°C. Gently flick the tube 2–3 times every 2 min to mix the sample.</li>
<li>Collect the beads using a magnetic separation stand. Discard the supernatant.</li>
<li>Add 50 μL of wash buffer containing 10 mM Tris-Cl pH 8.0, 1.2 M NaCl, 10 mM MgCl<sub>2</sub> and 10 μg/mL heparin. Incubate 10 min at 25°C. Gently flick the tube 2–3 times every 2 min to mix the sample.</li>
<li>Collect the beads using a magnetic separation stand. Discard the supernatant.</li>
<li>Add 50 μL of 1<b>×</b> <i>Taq</i> DNA polymerase buffer (NEB) to wash the beads and to prepare samples for the next step. Gently flick the tube 2–3 times to mix the sample.</li>
<li>Collect the beads using a magnetic separation stand. Discard the supernatant.</li>
<li>Repeat steps 8.i-j.</li>
<li>Resuspend the beads in 10 μL of 1 X <i>Taq</i> DNA polymerase buffer.</li>
<li>Pool the beads in one tube (total volume ∼20 μL). Store at 4°C while preparing primer extension reactions.
<div class="critical">
<span class="critical-title">Critical:</span> Proceed immediately to primer extension step.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.4">Primer extension reactions and purification of primer extension products</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼4 h for step 9</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼18 h for step 10</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼20 h for step 11</div>
<p>The next step of the protocol involves primer extension reactions using oligonucleotide complementary to sequences downstream of the N11 region of the p<i>lac</i>CONS-N11 library.</p>
<ol start="9">
<li>Primer extension reaction
<ol type="a">
<li>Prepare primer extension reaction:
<table id="undtbl8">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10 X Taq DNA Polymerase Buffer</td>
<td>1 X</td>
<td>10 μL</td>
</tr>
<tr>
<td>s128A, Primer Extension oligo (10 μM)</td>
<td>0.2 μM</td>
<td>2 μL</td>
</tr>
<tr>
<td>Taq DNA Polymerase (5 U/μL)</td>
<td>0.1 U/μL</td>
<td>2 μL</td>
</tr>
<tr>
<td>10 X dNTPs (2.5 mM dATP, 2.5 mM dCTP, 2.5 mM dGTP, 2.5 mM dTTP)</td>
<td>0.25 mM each</td>
<td>10 μL</td>
</tr>
<tr>
<td>5 M Betaine</td>
<td>2 M</td>
<td>40 μL</td>
</tr>
<tr>
<td>DMSO (100%)</td>
<td>5%</td>
<td>5 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>15 μL</td>
</tr>
<tr>
<td>Crosslinked complexes on MagneHis beads</td>
<td>n/a</td>
<td>16 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Split each reaction in two 0.2 mL PCR tubes (50 μL reaction mixture in each tube) and place in a thermocycler.</li>
<li>Perform primer extension reactions using the following thermocycler conditions:
<table id="undtbl9">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">40 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
</li>
<li>Pool 2 reactions into a single 1.7 mL tube.
<div class="note">
<span class="note-title">Note:</span> We recommend optimizing the number of cycles in primer extension reaction (i) to generate enough products for downstream steps in the protocol and (ii) to avoid non-specific background products.</div>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Purification of primer extension products using phenol/chloroform/IAA mixture<br> Next, purification of primer extension products is performed using phenol/chloroform/IAA mixture followed by ethanol precipitation.
<ol type="a">
<li>Add 100 μL phenol:chloroform:IAA (pH 8.0), mix by vortexing for 30 s, centrifuge (21,000 × <i>g</i>; 1 min; 25°C), and recover the upper aqueous phase (∼100 μL).</li>
<li>Add 10 μL of 3M NaOAc, 1.1 μL of 10 mg/mL glycogen, and 330 μL of 100% EtOH. Mix by vortexing for 30 s. Incubate the mixture at −80°C for 16 h.</li>
<li>Centrifuge the mixture to pellet the DNA (21,000 × <i>g</i>; 30 min; 4°C) and remove supernatant.</li>
<li>Wash pellet in 1 mL of 80% EtOH kept at 4°C. Centrifuge (21,000 × <i>g</i>; 5 min; 4°C) and remove supernatant.</li>
<li>Perform step 10.d two more times.</li>
<li>Air dry pellets for 5 min at 25°C.</li>
<li>Add 20 μL nuclease-free H<sub>2</sub>O. Incubate at 25°C for 5 min and pipette the sample 10–15 times to ensure the full resuspension of the extension products.</li>
</ol>
</li>
</ol>
<ol start="11">
<li>Gel purification of primer extension products Next, we perform gel purification of primer extension products to remove excess primer followed by ethanol precipitation.
<ol type="a">
<li>Add 20 μL of 2 X loading dye (95% deionized formamide, 18 mM EDTA, 0.02% SDS, xylene cyanol, bromophenol blue).</li>
<li>Load 40 μL of the sample in one well of 10% 7M urea slab gel (equilibrated and run in 1 X TBE) alongside Low Range ssRNA Ladder. Run gel at 80 V for ∼25 min to allow the sample to enter the gel, then increase the voltage to 200 V and run until the bromophenol blue dye front reaches approximately one half of the total length of the gel (∼4 cm on an 8 cm gel).</li>
<li>Visualize nucleic acids by using SYBR Gold nucleic acid gel stain followed by UV transillumination (<a href="#fig4">Figure 4</a>A).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1063-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Primer extension and 3′-adapter ligation: results</div>
<p>PAGE analysis of primer extension reactions (panel A; step 11, section “<a href="#sec2.4">Primer extension reactions and purification of primer extension products</a>”) and 3′-adapter ligation reactions (panel B; step 13, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”). Brackets indicate the positions of products isolated from the gel and excess primers. M, Low Range ssRNA ladder.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> The amounts of templates of particular library can vary and in some cases the products are not seen on stained gel until the amplification step.</div>
</li>
<li>Excise ssDNA products ∼40–80 nt in size from the gel using a sterile scalpel.</li>
<li>Elute nucleic acids from gel by crushing gel slices as described in (<a href="#bib8">Vvedenskaya and Nickels, 2020</a>), adding 350 μL of 0.3 M NaCl in 1 X TE Buffer (10 mM Tris-HCl pH 8, 0.1 mM EDTA) to the crushed gel, and incubating the slurry for 10 min at 70°C.</li>
<li>Transfer slurry to a Spin-X centrifuge tube filter, centrifuge (14,000 × <i>g</i>; 1 min; 25°C), and transfer eluate into a 1.7 mL tube.</li>
<li>Remove the crushed gel from the Spin-X tube and perform steps 11.e-f one more time. Combine eluates (total volume ∼700 μL).</li>
<li>Add 70 μL of 3 M NaOAc, 7.7 μL of glycogen (10 mg/mL) and 650 mL of isopropanol, mix by vortexing for 30 s, and incubate at −80°C for 16 h.</li>
<li>Centrifuge (21,000 × <i>g</i>; 30 min; 4°C) to pellet the DNA and remove supernatant.</li>
<li>Wash DNA pellet in 1 mL of 80% EtOH kept at 4°C. Centrifuge (21,000 × <i>g</i>; 5 min; 4°C) and remove supernatant.</li>
<li>Perform step 11.j two more times.</li>
<li>Air dry pellet for 5 min at 25°C.</li>
<li>Add 5 μL nuclease-free H<sub>2</sub>O. Incubate at 25°C for 5 min and pipette the sample 10–15 times to ensure the full resuspension of the nucleic acids.
<div class="note">
<span class="note-title">Note:</span> Our protocol is aimed at preparation of libraries suitable for high throughput sequencing on Illumina sequencing platform. The ssDNA size selection at step 11.d provides template size range that is suitable for 75-cycle sequencing run.</div>
<div class="note">
<span class="note-title">Note:</span> Gels for distinct primer extension reactions should be stained in separate containers to avoid cross contamination of samples.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> DNA can be stored at −80°C for ∼6 months.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼18 h for step 12</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼19 h for step 13</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h for step 14</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼20 h for step 15</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼19 h for step 16</div>
<p>The next step of the protocol involves ligation of an oligonucleotide (the “3′ adapter”) to 3′ end of primer extension products prepared in the previous section. After ligation of the 3′ adapter, emulsion PCR is performed, reaction products are separated by gel electrophoresis, and amplicons of the desired lengths are purified by gel extraction. The gel-purified products are then sequenced using an Illumina NextSeq platform.</p>
<ol start="12">
<li>DNA adapter ligation to the 3′ end of the extension products isolated from gel
<div class="note">
<span class="note-title">Note:</span> We have found that use of a two-step ligation procedure improves yields of adapter-ligated products. The first step uses 5′-AppDNA/RNA Ligase, which can join 3′-OH end of DNA substrate and 5′ end of adenylated DNA adapter (AppDNA). The second step uses T4 RNA Ligase 1, which can join 3′-OH end of DNA substrate and 5′ end of DNA adapter that has 5′-phosphoryl group (5′P-DNA). We speculate that the 5′P-DNA, the original substrate in the adenylation reaction by the Mth enzyme (NEB) is a potential co-reagent in inefficient adapter adenylation reaction.</div>
<ol type="a">
<li>Prepare the first ligation reaction as follows:
<table id="undtbl10">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Extension products from step 11.m</td>
<td>n/a</td>
<td>5 μL</td>
</tr>
<tr>
<td>App s1248 (16.9 μM)</td>
<td>0.85 μM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>10 X NEB Buffer 1</td>
<td>1 X</td>
<td>1 μL</td>
</tr>
<tr>
<td>50 mM MnCl<sub>2</sub>
</td>
<td>5 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>5′-AppDNA/RNA Ligase (20 μM)</td>
<td>1 μM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>2 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Incubate for 1 h at 65°C, followed by 3 min at 90°C, cool to 4°C and incubate for 5 min. Use the entire mixture for the second ligation reaction.</li>
<li>Prepare the second ligation reaction as follows:
<table id="undtbl11">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>First ligation reaction from step 12.b</td>
<td>n/a</td>
<td>10 μL</td>
</tr>
<tr>
<td>10 X T4 RNA Ligase Reaction Buffer</td>
<td>1 X</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>50% PEG8000</td>
<td>12%</td>
<td>6 μL</td>
</tr>
<tr>
<td>100 mM DTT</td>
<td>10 mM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>1 mg/mL BSA</td>
<td>60 μg/mL</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>T4 RNA Ligase 1 (10 U/μL)</td>
<td>10 U</td>
<td>1 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>1.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>25 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Pipette the mixture 10 times, briefly spin and incubate at 16°C for 16 h.</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Isolation of adapter-ligated products by gel size selection Next, adapter-ligated products are separated from excess adapter using gel size selection and purified by ethanol precipitation.
<ol type="a">
<li>Add 20 μL of 2 X loading dye (95% deionized formamide, 18 mM EDTA, 0.02% SDS, xylene cyanol, bromophenol blue).</li>
<li>Load the sample in one well on a 10% 7M urea slab gel (equilibrated and run in 1 X TBE) alongside Low Range ssRNA Ladder. Run gel at 80 V for ∼25 min to allow the sample to enter the gel, then increase the voltage to 200 V and run until the bromophenol blue dye front reaches approximately one half of the total length of the gel (∼4 cm on an 8 cm gel).</li>
<li>Visualize nucleic acids using SYBR Gold nucleic acid gel stain followed by UV transillumination (<a href="#fig4">Figure 4</a>B).
<div class="note">
<span class="note-title">Note:</span> Gels for individual ligation mixture should be stained in separate containers.</div>
<div class="note">
<span class="note-title">Note:</span> For alternative options of adapter ligation see: Troubleshooting, Problem 2.</div>
</li>
<li>Using a sterile scalpel excise 3′-adapter-ligated products. These products will be longer than the primer extension products by 37 nt (∼75–120 nt).
<div class="note">
<span class="note-title">Note:</span> The yields of products in the desired size range may be too low to directly visualize on the gel at step 13.c (<a href="#fig4">Figure 4</a>B) for several reasons, including low template yields and co-migration of the templates with the bromophenol blue dye in the loading buffer (see also step 11.c, section “<a href="#sec2.4">Primer extension reactions and purification of primer extension products</a>”).</div>
</li>
<li>Elute nucleic acids from gel by crushing gel slices as described in (<a href="#bib8">Vvedenskaya and Nickels, 2020</a>), adding 350 μL of 0.3 M NaCl in 1 X TE Buffer (10 mM Tris-HCl pH 8, 0.1 mM EDTA) to the crushed gel, and incubating the slurry for 10 min at 70°C.</li>
<li>Transfer the slurry to a Spin-X centrifuge tube filter, centrifuge (14,000 × <i>g</i>; 1 min; 25°C) and transfer the eluate into a 1.7 mL tube.</li>
<li>Remove the crushed gel from the Spin-X tube and perform steps 13.e-f one more time. Combine eluates (total volume ∼700 μL).</li>
<li>Add 70 μL of 3 M NaOAc, 7.7 μL of glycogen (10 mg/mL) and 650 mL of isopropanol, mix by vortexing for 30 s, and incubate at −80°C for 16 h.</li>
<li>Centrifuge the mixture to pellet DNA (21,000 × <i>g</i>; 30 min; 4°C) and remove supernatant.</li>
<li>Wash pellet in 1 mL of 80% EtOH kept at 4°C. Centrifuge (21,000 × <i>g</i>; 5 min; 4°C) and remove supernatant.</li>
<li>Perform steps 13.j two more times.</li>
<li>Air dry pellets for 5 min at 25°C.</li>
<li>Add 15 μL nuclease-free H<sub>2</sub>O to DNA pellet. Incubate at 25°C for 5 min and pipette the sample 10–15 times to ensure the full resuspension of the nucleic acids.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Purified adapter-ligated products can be stored at −80°C for ∼6 months.</div>
</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Control PCR amplification of adapter-ligated products Next, a control PCR amplification is performed to estimate the yield and quality of a library generated from the adapter-ligated products.
<ol type="a">
<li>Prepare control PCR reaction mix as indicated below:
<table id="undtbl12">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Adapter-ligated products from step 13.m</td>
<td>n/a</td>
<td>2 μL</td>
</tr>
<tr>
<td>5 X HF Phusion Buffer with MgCl<sub>2</sub>
</td>
<td>1 X</td>
<td>2 μL</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td>0.2 mM</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>2.5 μM Illumina RP1 Primer</td>
<td>0.25 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>2.5 μM Illumina Index Primer (RPI1-48)</td>
<td>0.25 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>HF Phusion DNA Polymerase, 2 U/μL</td>
<td>0.2 U</td>
<td>0.1 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>3.7 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Perform reactions using cycling conditions below:
<table id="undtbl13">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">12 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>62°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>10 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
</li>
<li>Mix 10 μL of amplified DNA with 3 μL of 6<b>×</b> Orange DNA load dye and run a non-denaturing 10% TBE gel at the conditions described at step 13.b. Load O’Gene Ruler Ultra Low Range DNA ladder as size standard. Run gel until yellow dye reaches gel bottom.</li>
<li>Visualize nucleic acids using SYBR Gold nucleic acid gel stain followed by UV transillumination. Verify library quality by size range and band(s) intensity.
<div class="note">
<span class="note-title">Note:</span> Library quality verification can also be performed using an automated electrophoresis system.</div>
</li>
</ol>
</li>
</ol>
<ol start="15">
<li>Amplification of library using emulsion PCR Next, the library is amplified and purified by use of Micellula DNA Emulsion and Purification Kit (emulsion PCR or ePCR) followed by ethanol precipitation.
<div class="note">
<span class="note-title">Note:</span> Use of ePCR reduces the generation of amplicons derived from template switching (<a href="#bib6">Odelberg et al., 1995</a>).</div>
<ol type="a">
<li>Use Micellula DNA Emulsion and Purification Kit to prepare ePCR reaction using manufacturer recommendations.
<table id="undtbl14">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Adapter-ligated products from step 13.m</td>
<td>n/a</td>
<td>1–2 μL</td>
</tr>
<tr>
<td>5 X Detergent-Free HF Phusion Buffer with MgCl<sub>2</sub>
</td>
<td>1 X</td>
<td>10 μL</td>
</tr>
<tr>
<td>0.1 mg/mL BSA</td>
<td>5 μg/mL</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>10 mM dNTP mix</td>
<td>400 μM</td>
<td>2 μL</td>
</tr>
<tr>
<td>10 μM Illumina RP1 Primer</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>10 μM Illumina Index Primer (RPI1-48)</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>HF Phusion DNA Polymerase, 2 U/μL</td>
<td>0.04 U</td>
<td>1 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>27.5–28.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Perform ePCR reactions using the following thermocycler conditions:
<table id="undtbl15">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>10 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>5 s</td>
<td rowspan="3">20 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>60°C</td>
<td>5 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
</li>
<li>Recover ePCR amplicons by breaking the emulsion and purifying DNA according to the manufacturer’s recommendations. Elute DNA from the filter with 150 μL of elution buffer, add 15 μL 3 M NaOAc, 1.5 μL glycogen (10 mg/mL) and 500 μL 100% EtOH.</li>
<li>Precipitate the eluate at −80°C for 16 h.</li>
<li>Centrifuge the mixture to pellet DNA (21,000 <b>×</b> <i>g</i>; 30 min; 4°C) and remove the supernatant.</li>
<li>Wash pellet in 1 mL of 80% EtOH kept at 4°C. Centrifuge (21,000 <b>×</b> <i>g</i>; 5 min; 4°C) and remove supernatant.</li>
<li>Perform step 15.f two more times.</li>
<li>Air dry pellets for 10 min at 25°C.</li>
<li>Add 20 μL nuclease-free H<sub>2</sub>O to DNA pellet. Incubate at 25°C for 5 min and pipette the sample 10–15 times to ensure the full resuspension of the nucleic acids.</li>
</ol>
</li>
</ol>
<ol start="16">
<li>Isolation of libraries by gel size selection Next, amplicons are isolated by gel size selection to remove non-specific products generated at amplification.
<ol type="a">
<li>Add 4 μL of 6 X Orange DNA load dye to sample from step 15.i.</li>
<li>Load sample in 2–3 wells on a 10% TBE slab gel (equilibrated and run in 1 X TBE) alongside O’Gene Ruler Ultra Low Range DNA ladder. Run gel until yellow dye reaches gel bottom.</li>
<li>Visualize nucleic acids by using SYBR Gold nucleic acid gel stain at gentle agitation followed by UV transillumination (<a href="#fig5">Figure 5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1063-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. ePCR results: analysis by standard polyacrylamide gel electrophoresis (PAGE)</div>
<p>PAGE analysis of amplicons generated in ePCR reactions before (step 15.i) and after gel purification (step 16.n, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”). DNA products of 160–180 bp in size are the amplicons of interest. DNA band of ∼130 bp in size are amplicons of unknown origin. M, O’Gene Ruler Ultra Low Range DNA ladder.</p>
</figcaption>
</figure>
</li>
<li>Excise amplicons in the 160–180 bp size range using a sterile scalpel. Take special care to avoid the non-specific amplicons in the ∼130-bp size range.</li>
<li>Elute nucleic acids from gel by crushing gel slices as described in (<a href="#bib8">Vvedenskaya and Nickels, 2020</a>), adding 350 μL of 0.3 M NaCl in 1 X TE Buffer (10 mM Tris-HCl pH 8, 0.1 mM EDTA) to the crushed gel, and incubating the slurry for 2 h at 37°C.</li>
<li>Transfer the slurry to a Spin-X centrifuge tube filter, centrifuge (14,000 <b>×</b> <i>g</i>; 1 min; 25°C) and transfer the eluate into a 1.7 mL tube.</li>
<li>Remove the crushed gel from the Spin-X centrifuge tube filter and repeat steps 16.e-f.</li>
<li>Combine eluates from steps 16.f-g (total volume ∼700 μL).</li>
<li>Add 70 μL of 3 M NaOAc, 7.7 μL of glycogen (10 mg/mL) and 650 mL of isopropanol, mix by vortexing for 30 s, and incubate at −80°C for 16 h.</li>
<li>Centrifuge the mixture to pellet DNA (21,000 <b>×</b> <i>g</i>; 30 min; 4°C) and remove supernatant.</li>
<li>Wash DNA pellet 3 times in 1 mL of 80% EtOH kept at 4°C. Centrifuge (21,000 <b>×</b> <i>g</i>; 5 min; 4°C) and remove supernatant after each wash step.</li>
<li>Air dry pellets for 10 min at 25°C.</li>
<li>Add 20 μL nuclease-free H<sub>2</sub>O. Incubate at 25°C for 5 min and pipette the sample 10–15 times to ensure the full resuspension of the nucleic acids.</li>
<li>Measure the concentration of the libraries using Qubit dsDNA HS Assay kit prior to sequencing. Run control 10% TBE gel to check the quality of final library. To ensure accurate quantification of each library prior to sequencing we recommend use of an automated electrophoresis system. We routinely use an Agilent TapeStation system to assess the quality of our libraries (<a href="#fig6">Figure 6</a>).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1063-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. ePCR results: analysis by Agilent TapeStation system</div>
<p>Electropherogram of sample intensity (in fluorescence units, FU) for amplicons (step 16.n, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”) analyzed by Agilent TapeStation High Sensitivity DNA assay. (Lower: lower marker; Upper: upper marker; peak centered at 141 bp, amplicons of unknown origin; peak centered at 172 bp, amplicons of interest).</p>
</figcaption>
</figure>
</li>
<li>Sequence library. See (<a href="#bib10">Winkelman et al., 2020</a>) for a detailed description of the analysis of the sequencing data.
<div class="note">
<span class="note-title">Note:</span> Any automated electrophoresis system, Bioanalyzer Systems, Fragment Analyzer Systems or TapeStation Systems, can be used for efficient and accurate analysis of libraries prepared for high throughput sequencing.</div>
<div class="note">
<span class="note-title">Note:</span> For additional details on how to improve the yield and the quality of the final library see: Troubleshooting, Problems 3–5.</div>
<div class="note">
<span class="note-title">Note:</span> Barcoded libraries generated by XACT-seq are suitable for sequencing on Illumina NextSeq platform in high-output mode. Illumina PCR forward and index primers from TruSeq Small RNA Sample Prep Kits are used. Sequencing primers provided by Illumina contain a mixture of several oligos. To avoid potential complications due to the presence of a mixture of primer sequences in the Illumina sequencing reagents we use a custom sequencing primer, s1115.</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Representative results of primer extension and 3′-adapter ligation reactions are shown in <a href="#fig4">Figure 4</a>. Representative results of ePCR reactions are shown in <a href="#fig5">Figures 5</a> and <a href="#fig6">6</a>.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol provided above describes use of XACT-seq to analyze transcription complexes in the context of plasmid-borne sequences in <i>E. coli</i>. Application of XACT-seq to analyze transcription complexes in the context of the bacterial chromosome would require modifications to the protocol (e.g., inclusion of a DNA fragmentation step prior to IMAC, modification of the adapter ligation steps, and modification of the ePCR step).</p>
<p>The transcription complex can sample pre-translocated, post-translocated, reverse-translocated, and hyper-translocated states (<a href="#bib1">Belogurov and Artsimovitch, 2019</a>; <a href="#bib5">Larson et al., 2011</a>). XACT-seq, when used on its own, is unable to define the translocation state of a transcription complex. This limitation of XACT-seq can be overcome by combining XACT-seq and NET-seq for parallel analysis of RNAP active-center A-site positions and RNA 3ʹ end positions, respectively, each with single-nucleotide resolution.</p>
<p>We used the model bacterium <i>E. coli</i> to develop and first apply XACT-seq, but XACT-seq could be applied to organisms with more complex genomes. Bpa has previously been incorporated into RNAP subunits in yeast (<a href="#bib2">Chen et al., 2007</a>) and can be incorporated into proteins in mammalian cells (<a href="#bib4">Hino et al., 2005</a>). Methods for UV-irradiation, DNA fragmentation, RNAP purification, and sequencing library preparation would be similar for each organism. Application of XACT-seq to other organisms requires development and validation of a Bpa-containing RNAP derivative that, upon UV-irradiation, crosslinks to DNA at a precise position relative to the RNAP active center in a manner analogous to the reagent used in this work (<i>E. coli</i> RNAP-β′<sup>R1148Bpa</sup>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Number of individual transformants obtained prior to steps 1.e and 3.d in section “<a href="#sec2.1">Introduction of plasmids into E. coli cells</a>” is low.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Check the quality of plasmid DNA used in transformation and perform additional purification to remove contaminating salts/organics. Optimize the concentration of plasmid DNA used in the transformation. Use larger volume of competent cells for transformation. Perform additional or multiple transformations using the same stock of plasmid and combine transformants. Repeat transformation with freshly generated competent cells.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>Low yield of ePCR products (steps 15, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”) because of low adapter ligation efficiency.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Other ligases can be tested in steps 12.a-d in section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>.” The CircLigase ssDNA ligase (Lucigen) joins 3′-OH end of DNA substrate and 5′ end of DNA adapter that has 5′-phosphoryl group (5′P-DNA). The T4 DNA ligase HC (Thermo Fisher Scientific) joins 3′-OH end of DNA substrate and 5′-P end of double stranded adapter that has overhang of a randomized sequence to provide an efficient annealing to the substrate. Each ligase requires an optimization of reaction conditions.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>Low yield of ePCR products (step 15, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”) because of low input DNA in ePCR reaction.</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Increase amounts of adapter-ligated products used in the ePCR reaction. Perform multiple ePCR reactions. Increase the number of amplification cycles to 30.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>Presence of non-specific products in the final library (step 16.n, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”, <a href="#fig5">Figures 5</a> and <a href="#fig6">6</a>).</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Perform a second gel extraction step of amplified templates as described in step 16, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>Presence of non-specific ∼130-bp amplicon in the final library (step 16.d, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”, <a href="#fig5">Figures 5</a> and <a href="#fig6">6</a>).</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Carry over of excess primer extension oligo (step 11, section “<a href="#sec2.4">Primer extension reactions and purification of primer extension products</a>”) or excess 3′ adapter oligo (step 13, section “<a href="#sec2.5">3′-Adapter ligation, amplification, and gel purification of libraries</a>”) as a result of gel staining at high agitation or prolonged staining time can result in non-specific ∼130-bp amplicon in the final library. Hence, care should be taken to avoid an oligo carry over at library gel selection step. One of the potential solutions is loading an oligo on gel in a well separated from the library by one empty well. Run the gel as described in the sections above and separate the part containing an oligo. Stain with SYBER GOLD staining solution and define the position of oligo. Use this as a guide to separate part of the gel containing the library from the part of the gel containing excess oligo. Stain two gel pieces with SYBER GOLD staining solution in separate containers. Use a gel part containing the library for elution of library templates, and a gel part containing excess oligo to control that no traces of oligo left on the top piece. The procedure requires optimization but overall is very efficient and at standardized conditions can be done only one time. Also, suboptimal PCR reaction conditions can result in non-specific products. To avoid this, optimize PCR reaction components and amplification conditions.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Belogurov, G.A. and Artsimovitch, I. (2019). The mechanisms of substrate selection, catalysis, and translocation by the elongating RNA polymerase. J. Mol. Biol. <i>431</i>, 3975-4006. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref1">View at publisher</a></p>
<p id="bib2">Chen, H.T., Warfield, L., and Hahn, S. (2007). The positions of TFIIF and TFIIE in the RNA polymerase II transcription preinitiation complex. Nat. Struct. Mol. Biol. <i>14</i>, 696-703. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref2">View at publisher</a></p>
<p id="bib3">Chin, J.W., Martin, A.B., King, D.S., Wang, L., and Schultz, P.G. (2002). Addition of a photocrosslinking amino acid to the genetic code of <i>Escherichia coli</i>. Proc. Natl. Acad. Sci. U. S. A. <i>99</i>, 11020-11024. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref3">View at publisher</a></p>
<p id="bib4">Hino, N., Okazaki, Y., Kobayashi, T., Hayashi, A., Sakamoto, K., and Yokoyama, S. (2005). Protein photo-cross-linking in mammalian cells by site-specific incorporation of a photoreactive amino acid. Nat. Methods <i>2</i>, 201-206. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref4">View at publisher</a></p>
<p id="bib5">Larson, M.H., Landick, R., and Block, S.M. (2011). Single-molecule studies of RNA polymerase: one singular sensation, every little step it takes. Mol. Cell <i>41</i>, 249-262. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref5">View at publisher</a></p>
<p id="bib6">Odelberg, S.J., Weiss, R.B., Hata, A., and White, R. (1995). Template-switching during DNA synthesis by <i>Thermus aquaticus</i> DNA polymerase I. Nucleic Acids Res. <i>23</i>, 2049-2057. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref6">View at publisher</a></p>
<p id="bib7">Vvedenskaya, I.O., Goldman, S.R., and Nickels, B.E. (2018). Analysis of bacterial transcription by "massively systematic transcript end readout," MASTER. Methods Enzymol. <i>612</i>, 269-302. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref7">View at publisher</a></p>
<p id="bib8">Vvedenskaya, I.O. and Nickels, B.E. (2020). CapZyme-seq: a 5’-RNA-seq method for differential detection and quantitation of NAD-capped and uncapped 5’-triphosphate RNA. STAR Protoc. <i>1</i>, 100002. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref8">View at publisher</a></p>
<p id="bib9">Vvedenskaya, I.O., Zhang, Y., Goldman, S.R., Valenti, A., Visone, V., Taylor, D.M., Ebright, R.H., and Nickels, B.E. (2015). Massively systematic transcript end readout, "MASTER": transcription start site selection, transcriptional slippage, and transcript yields. Mol. Cell <i>60</i>, 953-965. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref9">View at publisher</a></p>
<p id="bib10">Winkelman, J.T., Pukhrambam, C., Vvedenskaya, I.O., Zhang, Y., Taylor, D.M., Shah, P., Ebright, R.H., and Nickels, B.E. (2020). XACT-seq comprehensively defines the promoter-position and promoter-sequence determinants for initial-transcription pausing. Mol. Cell <i>79</i>, 797-811 e798. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref10">View at publisher</a></p>
<p id="bib12">Winkelman, J.T., Winkelman, B.T., Boyce, J., Maloney, M.F., Chen, A.Y., Ross, W., and Gourse, R.L. (2015). Crosslink Mapping at Amino Acid-Base Resolution Reveals the Path of Scrunched DNA in Initial Transcribing Complexes. Mol. Cell <i>59</i>, 768-780. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/optFFGmTosrEw">View at publisher</a></p>
<p id="bib11">Yu, L., Winkelman, J.T., Pukhrambam, C., Strick, T.R., Nickels, B.E., and Ebright, R.H. (2017). The mechanism of variability in transcription start site selection. eLife <i>6</i>, e32038. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00564-5/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Additional information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Bryce Nickels (<a href="mailto:bnickels@waksman.rutgers.edu">bnickels@waksman.rutgers.edu</a>).</p>
<h4>Materials availability</h4>
<p>Plasmids and strains are available upon request.</p>
<h4>Data and code availability</h4>
<p>Unprocessed sequencing reads have been deposited in the NIH/NCBI Sequence Read Archive under the study accession number PRJNA615362. Source code and documentation for analysis of sequencing data are provided at <a href="https://github.com/NickelsLabRutgers/XACT-seq">https://github.com/NickelsLabRutgers/XACT-seq</a>.</p>
<h3>Acknowledgments</h3>
<p>Work was supported by the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> grant GM118059 (B.E.N.).</p>
<h3>Author contributions</h3>
<p>C.P. designed research, performed research, and wrote the manuscript. I.O.V. designed research, performed research, and wrote the manuscript. B.E.N designed research, acquired funding, supervised the project, and wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>