<article>
<section>
<h1 id="header">HIV reservoir quantification by five-target multiplex droplet digital PCR</h1>
<p><time datetime="2021-10-15">Published: October 15, 2021</time></p>
<p>Claire N. Levy,<sup><a href="#aff1">1</a></sup> Sean M. Hughes,<sup><a href="#aff1">1</a></sup> Pavitra Roychoudhury,<sup><a href="#aff2">2</a></sup> Chelsea Amstuz,<sup><a href="#aff2">2</a></sup> Haiying Zhu,<sup><a href="#aff3">3</a></sup> Meei-Li Huang,<sup><a href="#aff3">3</a></sup> Dara A. Lehman,<sup><a href="#aff4">4</a>,<a href="#aff5">5</a></sup> Keith R. Jerome,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff5">5</a>,<a href="#fn1">7</a>,<a href="#cor1">*</a></sup> and Florian Hladik<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff6">6</a>,<a href="#fn1">7</a>,<a href="#fn2">8</a>,<a href="#fn3">9</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Obstetrics &amp; Gynecology, University of Washington, Seattle, WA 98109, USA</p>
<p id="aff2"><sup>2</sup>Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p>
<p id="aff3"><sup>3</sup>Department of Laboratory Medicine and Pathology, University of Washington, Seattle, WA 98109, USA</p>
<p id="aff4"><sup>4</sup>Division of Human Biology, Fred Hutchinson Cancer Research Center, Seattle, WA 98109, USA</p>
<p id="aff5"><sup>5</sup>Department of Global Health, University of Washington, Seattle, WA 98109, USA</p>
<p id="aff6"><sup>6</sup>Department of Medicine, University of Washington, Seattle, WA 98109, USA</p>
<p id="fn1"><sup>7</sup>These authors contributed equally</p>
<p id="fn2"><sup>8</sup>Technical contact</p>
<p id="fn3"><sup>9</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kjerome@fredhutch.org">kjerome@fredhutch.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:florian@uw.edu">florian@uw.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100885">10.1016/j.xpro.2021.100885</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Most latent human immunodeficiency virus (HIV) proviruses are defective and cannot produce infectious virions. Thus, the number of HIV proviruses with intact genomes is a relevant clinical parameter to assess therapies for HIV cure. We describe high-molecular-weight DNA isolation, followed by restriction enzyme fragmentation that limits cutting within the HIV genome. Multiplexed droplet digital PCR quantifies five targets spanning the HIV genome to estimate potentially intact proviral copies. A reference assay counts the number of T lymphocytes and assesses the level of DNA shearing.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00591-8/sref4">Levy et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Extracting genomic DNA is necessary to determine intactness of HIV-1 provirus</li>
<li>Selective DNA fragmentation allows droplet formation and maintains proviral intactness</li>
<li>Results from QuantaSoft™ are analyzed using R scripts or a spreadsheet program</li>
<li>Final result is intact proviruses per 10<sup>6</sup> T cells, with shearing correction</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1097-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the steps for using total peripheral blood mononuclear cells (PBMC), isolated T cells and mucosal tissues. We recommend using samples that are fresh or have either been cryopreserved (tissue or cells) or dry frozen (tissues). This protocol takes 5–8 h of hands-on time spread over 3 days for cell samples or 4 days for tissue samples, due to multiple overnight incubation steps. The instructions for tissues have been tested on mucosal tissues only (specifically, ectocervix, vagina, rectum). Optimization is likely required for other tissue types. The Jurkat and J-Lat 8.4 cell lines may be used for negative and positive controls respectively. Details about these cell lines can be found in the <a href="#key-resources-table">key resources table</a>. In addition to assays targeting regions in the HIV genome, this protocol employs a reference assay to count total cells (using targets in the <i>RPP30</i> gene) and all non-T cells (the “deltaD” assay). Primer and probe sequences are listed in <a href="#tbl1">Table 1</a> (HIV targets), <a href="#tbl2">Table 2</a> (target in <i>TRD</i>) and <a href="#tbl3">Table 3</a> (<i>RPP30</i> targets).</p>
<p>We recommend that the HIV assays be run in triplicate and the reference assay be run in duplicate wells.</p>
<p>Institutional Review Board (IRB) approval to work with human tissues may be needed prior to this experiment. Refer to your institution’s regulations and ensure that all documents are complete before collecting samples or data.</p>
<h3 id="sec1.1">Prepare HIV and reference gene droplet digital polymerase chain reaction (ddPCR) reagents</h3>
<ol>
<li>Prepare lyophilized HIV and deltaD primers and probes.
<ol type="a">
<li>Resuspend lyophilized primers to 332 μM in Tris-EDTA (10 mM Tris, 0.1 mM EDTA pH 8.0).</li>
<li>Bring all probes to 100 μM in Tris-EDTA (10 mM Tris, 0.1 mM EDTA pH 8.0).</li>
</ol>
</li>
<li>Prepare working stocks of HIV gene and deltaD primers/probes (separately for each primer and probe).</li>
</ol>
<table id="undtbl1">
<caption>Primer and probe working stock</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>85 μL</td>
</tr>
<tr>
<td>Fwd primer</td>
<td>16.6 μM</td>
<td>5 μL</td>
</tr>
<tr>
<td>Rev primer</td>
<td>16.6 μM</td>
<td>5 μL</td>
</tr>
<tr>
<td>Probe</td>
<td>5 μM</td>
<td>5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for two years</p>
<ol start="3">
<li>Prepare 20<b>×</b> stock of the two <i>RPP30</i> PrimeTime® Std qPCR Assays by adding Tris-EDTA (10 mM Tris, 0.1 mM EDTA pH 8.0) according to the documentation provided by IDT.</li>
<li>Prepare HIV triplex <b>assay1</b> pool from the working stocks prepared in step 2 above.
<ol type="a">
<li>Prepare aliquots and store at −20°C for future use.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Consider the number of aliquots you will need to complete a series of experiments to avoid needing to make a new batch partway through a project.</div>
<table id="undtbl2">
<caption>Assay1 pool, amount is for one reaction</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<i>tat</i> FAM working stock</td>
<td>2.25 μM</td>
<td>1.35 μL</td>
</tr>
<tr>
<td>3′<i>pol</i> FAM working stock</td>
<td>2.25 μM</td>
<td>0.95 μL</td>
</tr>
<tr>
<td>
<i>env</i> HEX working stock</td>
<td>2.25 μM</td>
<td>0.7 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>3 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for two years</p>
<ol start="5">
<li>Prepare HIV triplex <b>assay2</b> pool from working stocks prepared in step 2 above
<ol type="a">
<li>Note that water is added to this master mix simply to make the volume the same as assay1 so that the same volume of master mix can be used for both assays.</li>
<li>Aliquot and store for future use.</li>
</ol>
</li>
</ol>
<table id="undtbl3">
<caption>Assay2 pool, amount is for one reaction</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5′<i>pol</i> FAM working stock</td>
<td>1.83 μM</td>
<td>1.1 μL</td>
</tr>
<tr>
<td>
<i>gag</i> HEX working stock</td>
<td>0.97 μM</td>
<td>0.55</td>
</tr>
<tr>
<td>
<i>env</i> HEX working stock</td>
<td>1.58 μM</td>
<td>0.95</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>0.4</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>3 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for two years</p>
<ol start="6">
<li>Prepare RPP30_deltaD pool from working stock prepared in step 2 above.
<ol type="a">
<li>Aliquot and store for future use.</li>
</ol>
</li>
</ol>
<table id="undtbl4">
<caption>RPP30_deltaD pool, amount is for one reaction</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5′<i>RPP30</i> FAM</td>
<td>2.25 μM</td>
<td>1.35 μL</td>
</tr>
<tr>
<td>deltaD FAM</td>
<td>2.25 μM</td>
<td>0.95 μL</td>
</tr>
<tr>
<td>3′ <i>RPP30</i> HEX</td>
<td>2.25 μM</td>
<td>0.7 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>3 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for two years</p>
<h3 id="sec1.2">Prepare plasmid controls</h3>
<div class="note">
<span class="note-title">Note:</span> Please see the <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1097-Mmc1.pdf">Figure S1</a> for an illustration of how these controls are prepared.</div>
<ol start="7">
<li>Order agar stabs of plasmids from Addgene (see Catalog numbers in <a href="#key-resources-table">key resources table</a>). Grow cultures and extract DNA.</li>
<li>Linearize plasmid controls with restriction enzyme <i>Aat</i>II. This will result in better droplet formation than circular plasmid. Dilute plasmid such that there will be about 10,000 copies per ddPCR reaction. Calculate the number of copies using the following formula, where <i>ngSample</i> is the mass in ng of plasmid that you have and <i>bpSample</i> is the length in base pairs of the plasmid:</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>p</mi> <mi>l</mi> <mi>a</mi> <mi>s</mi> <mi>m</mi> <mi>i</mi> <mi>d</mi> <mspace></mspace> <mi>m</mi> <mi>o</mi> <mi>l</mi> <mi>e</mi> <mi>c</mi> <mi>u</mi> <mi>l</mi> <mi>e</mi> <mi>s</mi> <mo>=</mo> <mrow> <mo>(</mo> <mrow> <mi>n</mi> <mi>g</mi> <mi>S</mi> <mi>a</mi> <mi>m</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mspace></mspace> <mo>×</mo> <mfrac> <mrow> <mn>6.022</mn> <mo>×</mo> <msup> <mn>10</mn> <mn>23</mn> </msup> <mspace></mspace> <mi>m</mi> <mi>o</mi> <mi>l</mi> <mi>e</mi> <mi>c</mi> <mi>u</mi> <mi>l</mi> <mi>e</mi> <mi>s</mi> </mrow> <mrow> <mi>m</mi> <mi>o</mi> <mi>l</mi> <mi>e</mi> </mrow> </mfrac> </mrow> <mo>)</mo> </mrow> <mo>÷</mo> <mrow> <mo>(</mo> <mi>b</mi> <mi>p</mi> <mi>S</mi> <mi>a</mi> <mi>m</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mspace></mspace> <mo>×</mo> <mfrac> <mrow> <mn>660</mn> <mi>g</mi> <mspace></mspace> <mtext>DNA</mtext> </mrow> <mrow> <mi>m</mi> <mi>o</mi> <mi>l</mi> <mi>e</mi> </mrow> </mfrac> <mspace></mspace> <mo>×</mo> <mfrac> <mrow> <msup> <mn>10</mn> <mrow> <mn>9</mn> <mspace></mspace> </mrow> </msup> <mi>n</mi> <mi>g</mi> </mrow> <mi>g</mi> </mfrac> <mo>)</mo> </mrow> </mrow> </math></p>
<p>Combine the linearized plasmids by adding an equal mass of each to a tube.</p>
<ol start="9">
<li>Dilute the mixture with water to 0.01 pg/mL by serial dilution. Save the dilutions at −20°C for future use.
<ol type="a">
<li>The exact serial dilution plan will depend on the starting stock concentration. We started with a concentration of about 19 ng/μL and performed 3 serial dilutions of 1:100 and a final 1:2 dilution to bring the concentration to approximately 0.1 pg/μL.</li>
</ol>
</li>
<li>Prepare plasmid control “P1”: Plasmid control “P1” is a mixture of 7 plasmids and HIV negative human gDNA. Please see the Addgene entries for additional information about the plasmids. Combine the diluted plasmid with <i>Bgl</i>I-digested HIV negative human gDNA (see <a href="#sec1.4">prepare negative control template</a> section below) in a ratio of 4 parts plasmid to 1 part human DNA.
<ol type="a">
<li>Example: Linearized diluted plasmid is at about 2,500 copies per μL. Combine 4 μL of the plasmid (= 10,000 copies) with 1 μL of <i>Bgl</i>I-digested HIV-negative gDNA at a concentration between 100–200 ng/ μL.</li>
<li>Prepare 12 μL aliquots of the P1 control and store at −20°C.</li>
</ol>
</li>
<li>For plasmid control “P2”, repeat the steps above but instead of combining the 7 plasmids, <b>only</b> use the plasmid that contains all HIV targets: 167347, Seattle_IPDA_control_1_001 (see <a href="#key-resources-table">key resources table</a>).</li>
<li>In ddPCR experiments, use 5 μL of P1 or P2 control per control well in assay1 and assay 2. Use 5 ul of 1:2 diluted plasmid P1 or P2 per control well for the RPP30_deltaD reference assay.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Plasmid control signal may vary between preparations and therefore require some optimization. The authors suggest testing a few dilutions of the controls when a new dilution is prepared (for example, neat, 1:5, 1:10).</div>
<h3 id="sec1.3">Prepare workspace and materials</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<ol start="13">
<li>Designate a workspace free of plasmid or PCR product contamination. For thawing cells and tissues, work in a biosafety cabinet following your institution’s environmental health and safety protocols for working with HIV-infected human cells or tissues.</li>
<li>Set heat block to 56°C</li>
<li>Guanidine HCl and guanidine isothiocyanate waste may require special chemical waste disposal depending on your facility’s regulations. Prepare labeled waste containers for liquid and solid waste.</li>
<li>If digesting tissues: dissolve any precipitates in buffer ATL by heating to 70°C with gentle agitation.</li>
<li>Guanidine HCl, guanidine isothiocyanate and proteinase K should be at room temperature, about 25°C.</li>
</ol>
<h3 id="sec1.4">Prepare negative control template</h3>
<p>Commercially prepared HIV-negative gDNA can be used as a negative control. Prior to use, this DNA should be digested in <i>Bgl</i>I and precipitated with ethanol for later use.</p>
<ol start="18">
<li>Make the following master mix to digest ∼ 20 ug of gDNA:</li>
</ol>
<table id="undtbl5">
<caption>Restriction enzyme digestion of HIV-negative gDNA</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NEBuffer 3.1</td>
<td>1<b>×</b>
</td>
<td>20 μL</td>
</tr>
<tr>
<td>NEB <i>Bgl</i>I 10,000 IU/mL</td>
<td>750 IU/mL</td>
<td>15 μL</td>
</tr>
<tr>
<td>Human HIV-negative gDNA</td>
<td>0.1 μg/mL</td>
<td>20 μg</td>
</tr>
<tr>
<td>Molecular grade H<sub>2</sub>O</td>
<td>n/a</td>
<td>To 200 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Do not store, begin digestion after preparation</p>
<ol start="19">
<li>Digest at 37°C for 1 h.</li>
<li>After digestion, add 100 μL molecular grade H<sub>2</sub>O to bring the total volume to 300 μL, then proceed with EtOH precipitation (<a href="#bib1">Green and Sambrook, 2012</a>) as described below.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human HIV-negative genomic DNA</td>
<td>Promega</td>
<td>Cat#G3041</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Buffer ATL</td>
<td>QIAGEN</td>
<td>Cat#19076</td>
</tr>
<tr>
<td>Buffer EB</td>
<td>QIAGEN</td>
<td>Cat#19086</td>
</tr>
<tr>
<td>Guanidine HCl</td>
<td>Research Products International Corp.</td>
<td>Cat#G49000–100.0CAS No. 50-01-1</td>
</tr>
<tr>
<td>3M Sodium acetate</td>
<td>Invitrogen</td>
<td>Cat#AM9740CAS No. 127-09-3</td>
</tr>
<tr>
<td>Guanidine isothiocyanate</td>
<td>Chem Cruz: Santa Cruz Biotechnology Inc.</td>
<td>Cat#SC-202638CAS No. 593-84-0</td>
</tr>
<tr>
<td>
<i>Bgl</i>I</td>
<td>New England Biolabs</td>
<td>Cat#R0143S</td>
</tr>
<tr>
<td>Proteinase K, 20 mg/mL, &gt;318 mAu/mL at 30°C</td>
<td>QIAGEN</td>
<td>Cat#19131</td>
</tr>
<tr>
<td>
<i>Aat</i>II</td>
<td>New England Biolabs</td>
<td>Cat#R0117S</td>
</tr>
<tr>
<td>Pen Strep</td>
<td>Gibco</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>RPMI 1640 Medium, HEPES</td>
<td>Gibco</td>
<td>Cat#22400089</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Nucleus Biologics</td>
<td>Cat#FBS1824-001</td>
</tr>
<tr>
<td>L-Glutamine (200 mM)</td>
<td>Gibco</td>
<td>Cat#A2916801</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>ddPCR Supermix for Probes (no dUTP)</td>
<td>Bio-Rad</td>
<td>Cat#1863024</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>J-Lat 8.4 cell line</td>
<td>NIH HIV Reagent Program</td>
<td>Cat#9847-427RRID:CVCL_8284</td>
</tr>
<tr>
<td>Jurkat E6-1 cell line</td>
<td>ATCC</td>
<td>Cat#TIB-152RRID:CVCL_0367</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>HIV-1 primers/probes</td>
<td>Integrated DNA Technologies</td>
<td><a href="#tbl1">Table 1</a></td>
</tr>
<tr>
<td>
<i>TRD</i> gene primers/probes (<a href="#bib9">Zoutman et al., 2017</a>)</td>
<td>Integrated DNA Technologies</td>
<td><a href="#tbl2">Table 2</a></td>
</tr>
<tr>
<td>
<i>RPP30</i> gene primers/probes</td>
<td>Integrated DNA Technologies</td>
<td><a href="#tbl3">Table 3</a></td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Seattle_IPDA_control_1_001</td>
<td>Addgene</td>
<td>Cat#167347</td>
</tr>
<tr>
<td>Seattle_IPDA_control_6_002</td>
<td>Addgene</td>
<td>Cat#167348</td>
</tr>
<tr>
<td>Seattle_IPDA_control_9_003</td>
<td>Addgene</td>
<td>Cat#167349</td>
</tr>
<tr>
<td>Seattle_IPDA_control_14_004</td>
<td>Addgene</td>
<td>Cat#167350</td>
</tr>
<tr>
<td>Seattle_IPDA_control_17_005</td>
<td>Addgene</td>
<td>Cat#167351</td>
</tr>
<tr>
<td>Seattle_IPDA_control_29_006</td>
<td>Addgene</td>
<td>Cat#167352</td>
</tr>
<tr>
<td>Seattle_IPDA_control_32_007</td>
<td>Addgene</td>
<td>Cat#167353</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Microsoft Office</td>
<td><a href="https://www.microsoft.com/en-us/microsoft-365">https://www.microsoft.com/en-us/microsoft-365</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Bioconductor</td>
<td><a href="http://www.bioconductor.org">http://www.bioconductor.org</a></td>
<td>RRID:SCR_006442</td>
</tr>
<tr>
<td>Biorender</td>
<td><a href="http://biorender.io/">http://biorender.io/</a></td>
<td>N/A</td>
</tr>
<tr>
<td>R software 4.0</td>
<td><a href="http://www.r-project.org/">http://www.r-project.org/</a></td>
<td>RRID:SCR_001905</td>
</tr>
<tr>
<td>QuantaSoft AP 1.0.596</td>
<td>Bio-Rad</td>
<td><a href="http://www.bio-rad.com/en-us/product/qx200-droplet-digital-pcr-system">http://www.bio-rad.com/en-us/product/qx200-droplet-digital-pcr-system</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>twin.tec PCR plate 96, semi-skirted</td>
<td>Eppendorf</td>
<td>Cat#951020362</td>
</tr>
<tr>
<td>PCR Plate Heat Seal, foil, pierceable</td>
<td>Bio-Rad</td>
<td>Cat#1814040</td>
</tr>
<tr>
<td>Bio-Rad PX1 PCR Plate Sealer</td>
<td>Bio-Rad</td>
<td>Cat#1814000</td>
</tr>
<tr>
<td>Wide bore p1000 pipette tips</td>
<td>Rainin</td>
<td>Cat#30389195</td>
</tr>
<tr>
<td>Wide bore p200 pipette tips</td>
<td>Rainin</td>
<td>Cat#30389217v</td>
</tr>
<tr>
<td>Automated Droplet Generation Oil</td>
<td>Bio-Rad</td>
<td>Cat#1864110</td>
</tr>
<tr>
<td>Droplet Reader Oil</td>
<td>Bio-Rad</td>
<td>Cat#1863004</td>
</tr>
<tr>
<td>ddPCR<sup>TM</sup> Buffer Control for Probes</td>
<td>Bio-Rad</td>
<td>Cat#1863052</td>
</tr>
<tr>
<td>DG32 Automated Droplet Generator Cartridges</td>
<td>Bio-Rad</td>
<td>Cat#1864109</td>
</tr>
<tr>
<td>QX200 Droplet Digital PCR System</td>
<td>Bio-Rad</td>
<td>Cat# 1864001</td>
</tr>
<tr>
<td>Eppendorf Thermomixer C</td>
<td>Eppendorf</td>
<td>Cat#5382000023</td>
</tr>
<tr>
<td>Eppendorf Thermomixer SmartBlock 1.5 mL</td>
<td>Eppendorf</td>
<td>Cat#13687712</td>
</tr>
<tr>
<td>EasySep™ Human CD4+ T Cell Isolation Kit</td>
<td>STEMCELL</td>
<td>Cat#17952</td>
</tr>
<tr>
<td>EasyEights™ EasySep™ Magnet</td>
<td>STEMCELL</td>
<td>Cat#18103</td>
</tr>
</tbody>
</table>
<table id="tbl1">
<caption>Table 1. HIV-1 multiplex primers and probes</caption>
<thead>
<tr>
<th>Multiplex assay</th>
<th>Primer/probe</th>
<th>Dye, quencher</th>
<th>Sequence 5′ to 3′</th>
<th>NC_001802 coordinates downloaded 27June19</th>
</tr>
</thead>
<tbody>
<tr>
<td>assay1</td>
<td>3′<i>pol</i> F</td>
<td> </td>
<td>AGAGATCCACTTTGGAAAGGACCAGCAAA</td>
<td>4466–4494</td>
</tr>
<tr>
<td>assay1</td>
<td>3′<i>pol</i> probe</td>
<td>FAM, BHQ</td>
<td>T<b>G</b>GAA<b>AG</b>G<b>T</b>GA<b>A</b>GG</td>
<td>4502–4515</td>
</tr>
<tr>
<td>assay1</td>
<td>3′<i>pol</i> R</td>
<td> </td>
<td>CACTACTTTTATGTCACTATTATCTTGTATTACTACTGC</td>
<td>4517–4555</td>
</tr>
<tr>
<td>assay1</td>
<td>
<i>tat</i> F</td>
<td> </td>
<td>TTTGTTTCATAACAAAAGGCTTAGGCATCTCC</td>
<td>5483–5514</td>
</tr>
<tr>
<td>assay1</td>
<td>
<i>tat</i> probe</td>
<td>FAM, BHQ</td>
<td>
<b>ATG</b>GCA<b>G</b>GA<b>A</b>GAAG</td>
<td>5516–5529</td>
</tr>
<tr>
<td>assay1</td>
<td>
<i>tat</i> R</td>
<td> </td>
<td>TGAGGAGCTCTTCGTCGCTGTCTCC</td>
<td>5531–5555</td>
</tr>
<tr>
<td> </td>
<td>5′ <i>pol</i> F</td>
<td> </td>
<td>TCCTTTAGCTTCCCTCAGATCACTCT</td>
<td>1787–1812</td>
</tr>
<tr>
<td>assay2</td>
<td>5′ <i>pol</i> probe</td>
<td>FAM, BHQ</td>
<td>TT<b>GG</b>CA<b>AC</b>G<b>A</b>CC</td>
<td>1813–1824</td>
</tr>
<tr>
<td> </td>
<td>5′ <i>pol</i> R</td>
<td> </td>
<td>TACTGTATCATCTGCTCCTGTATCTAATAGAGCTTC</td>
<td>1859–1894</td>
</tr>
<tr>
<td> </td>
<td>
<i>gag</i> F</td>
<td> </td>
<td>GACTAGCGGAGGCTAGAAGGAGAGA</td>
<td>310–334</td>
</tr>
<tr>
<td> </td>
<td>
<i>gag</i> probe</td>
<td>HEX, BHQ</td>
<td>A<b>TG</b>GG<b>T</b>G<b>C</b>GAGA</td>
<td>336–347</td>
</tr>
<tr>
<td> </td>
<td>
<i>gag</i> R</td>
<td> </td>
<td>CTAATTCTCCCCCGCTTAATAYTGACG</td>
<td>345–375</td>
</tr>
<tr>
<td>assay1&amp;2</td>
<td>
<i>env</i> F</td>
<td> </td>
<td>TGTTCCTTGGGTTCTTGGGAGCAGCAGG</td>
<td>7323–7350</td>
</tr>
<tr>
<td>assay1&amp;2</td>
<td>
<i>env</i> probe</td>
<td>HEX, BHQ</td>
<td>
<b>AG</b>C<b>A</b>CT<b>ATG</b>GG</td>
<td>7352–7362</td>
</tr>
<tr>
<td>assay1&amp;2</td>
<td>
<i>env</i> R</td>
<td> </td>
<td>GCACTATACCAGACAATAATTGTCTGGCCTGTACC</td>
<td>7384–7418</td>
</tr>
</tbody>
</table>
<p class="table-legend">Bold text indicates locked nucleic acids.</p>
<table id="tbl2">
<caption>Table 2. Primers and probes for the deltaD target in <i>TRD</i>
</caption>
<thead>
<tr>
<th>Primer/probe</th>
<th>Multiplex assay</th>
<th>Dye, quencher</th>
<th>Sequence 5′ to 3′</th>
<th>GenBank gene ID 6964 (TRD) coordinates downloaded 27June19</th>
</tr>
</thead>
<tbody>
<tr>
<td>deltaD F</td>
<td>RPP30_deltaD</td>
<td> </td>
<td>GCTGGCTGTAATGGGAATGT</td>
<td>18144–18163</td>
</tr>
<tr>
<td>deltaD probe</td>
<td>RPP30_deltaD</td>
<td>FAM, BHQ</td>
<td>TGTGAAGATGTCTGTAGCCATCTTAT</td>
<td>18171–18196</td>
</tr>
<tr>
<td>deltaD R</td>
<td>RPP30_deltaD</td>
<td> </td>
<td>TAATGGCTTGATAAAGATAAGTGATCAT</td>
<td>18201–18228</td>
</tr>
</tbody>
</table>
<table id="tbl3">
<caption>Table 3. Primers and probes for the two targets in <i>RPP30</i>
</caption>
<thead>
<tr>
<th>Primer/probe</th>
<th>Multiplex assay</th>
<th>Dye, quencher</th>
<th>Sequence 5′ to 3′</th>
<th>GenBank gene ID 10556 (<i>RPP30</i>) coordinates downloaded 24June19</th>
</tr>
</thead>
<tbody>
<tr>
<td>5′<i>RPP30</i> F</td>
<td>RPP30_deltaD</td>
<td> </td>
<td>GCATATCAGGGTACAGCATAGG</td>
<td>16599–16620</td>
</tr>
<tr>
<td>5′<i>RPP30</i> probe</td>
<td>RPP30_deltaD</td>
<td>FAM, ZEN/IBFQ</td>
<td>TCTGCTCGTTGTTAGTCACCAGCT</td>
<td>16635–16658</td>
</tr>
<tr>
<td>5′<i>RPP30</i> R</td>
<td>RPP30_deltaD</td>
<td> </td>
<td>CTTCCCTCACGGCATATACTTC</td>
<td>16678–16699</td>
</tr>
<tr>
<td>3<i>′RPP30</i> F</td>
<td>RPP30_deltaD</td>
<td> </td>
<td>GCTGTGTTTGCTCTCTTGATTT</td>
<td>28266–28287</td>
</tr>
<tr>
<td>3′<i>RPP30</i> probe</td>
<td>RPP30_deltaD</td>
<td>HEX, ZEN/IBFQ</td>
<td>AATGTCTGTGACTGGGTTCTGGCT</td>
<td>28322–28345</td>
</tr>
<tr>
<td>3′<i>RPP30</i> R</td>
<td>RPP30_deltaD</td>
<td> </td>
<td>GGTCTGTCCATGGCATCTTAT</td>
<td>28348–28368</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec3.1">Equipment</h3>
<p>We recommend using a shaking heat block such as the Eppendorf Thermomixer C (Eppendorf Cat#5382000023) during the restriction enzyme step. An alternative is to use a tube rotator inside an incubator.</p>
<p>QX200 Droplet Digital PCR System (Bio-Rad):</p>
<p>This protocol assumes knowledge of the QX200 digital droplet PCR system (droplet generator and reader). Reference manuals are available from the Bio-Rad website: <a href="https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10031906.pdf">https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10031906.pdf</a></p>
<h3 id="sec3.2">Reagents</h3>
<p>Commercially prepared HIV negative human genomic DNA is convenient to use for preparing controls (ex. Promega Cat#G3041), but it is not necessary if an equivalent is available.</p>
<h3 id="sec3.3">Kits</h3>
<p>If T cell isolation is to be performed using the EasySep™ Human CD4+ T Cell Isolation Kit (StemCell Cat#17952) prior to DNA extraction, you must also have access to the EasySep™ magnets. The EasyEights™ EasySep™ Magnet is useful for processing multiple samples at a time.</p>
<h3 id="sec3.4">Data analysis</h3>
<p>In order to be able to quantify data about multi-positive populations, .qlp files exported from the Bio-Rad QX200 software must be analyzed in Bio-Rad’s Quanta-Soft Analysis Pro software, which is available for free download (with registration) from <a href="https://www.bio-rad.com/en-us/product/qx200-droplet-digital-pcr-system?ID=MPOQQE4VY">https://www.bio-rad.com/en-us/product/qx200-droplet-digital-pcr-system?ID=MPOQQE4VY</a></p>
<p>See also: Bio-Rad’s Droplet Digital <sup>TM</sup> PCR Applications Guide for mathematical details. Page 35 derives the equation for calculating concentration from droplet counts and volume.</p>
<p><a href="https://www.bio-rad.com/webroot/web/pdf/lsr/literature/Bulletin_6407.pdf">https://www.bio-rad.com/webroot/web/pdf/lsr/literature/Bulletin_6407.pdf</a></p>
<p>All steps in the analysis workflow subsequent to initial analysis in QuantaSoft Analysis Pro can be carried out using MS Excel or a similar spreadsheet program. However, we recommend the use of a script-based software/GUI such as R/Rstudio to make the analysis more reproducible and potentially faster.</p>
<h4 id="sec3.4.1">Example data and analyses</h4>
<p><b>Excel</b>: The excel workbook “V3_annotated_example_data.xlsx” contains example data and examples of a spreadsheet-based workflow. The first tab shows an example plate layout following the requirements of the plater package (<a href="#bib2">Hughes, 2016</a>). This format is not strictly required for excel-based analysis but is required if the instructions for analysis in R are followed and may be useful for other data manipulations. The second tab is an orientation to the cluster data that is exported from QuantaSoft. Subsequent tabs depict further steps of the workflow as described below.</p>
<p><b>R</b>: An .RProject directory is included with this protocol with example data and scripts. See the README file in the directory for a description of the contents and how to use it.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec4.1">Tissue lysis: If starting material is cells, not tissue, skip this and proceed to next section</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h hands-on, then 18 h incubation. Begin the protocol in the late afternoon so the 18 h incubation can b<b>e</b> <b>scheduled overnight.</b>
</div>
<div class="note">
<span class="note-title">Note:</span> If working with tissues, proceed through this section, if working with cells, skip to the “cell lysis and thawing” section below.</div>
<ol>
<li>When working with tissues, clean forceps by soaking in 70% EtOH for 5 min followed by a rinse in 1<b>×</b> PBS or use disposable sterile forceps.</li>
<li>Warm complete cell culture medium to 37°C, 5 mL per sample and put 5 mL of warmed medium in a petri dish or well of a 6-well plate. For vaginal, cervical and rectal tissues, we recommend complete RPMI medium: RPMI medium with 10% fetal bovine serum, 1% L-glutamine and 1% pen-strep. Please see the <a href="#key-resources-table">key resources table</a> for catalog numbers for these items. If you are attempting this protocol with a different type of tissue, we a recommend using the same medium as would be used for culture or short-term transport of the tissues.</li>
</ol>
<ol start="3">
<li>For each sample, label and weigh an empty 1.5–2 mL tube and record the weights.</li>
<li>If the tissues are dry frozen:
<ol type="a">
<li>Get dry ice to store replicate samples if you are working with &gt;1</li>
<li>Using forceps, remove the tissue from the tube and place in the medium to thaw</li>
</ol>
</li>
<li>If the tissues are cryopreserved:
<ol type="a">
<li>Get dry ice to store replicate samples if you are working with &gt;1 at a time.</li>
<li>Place the cryovial in a 37°C water bath and thaw until a small (e.g., 3<b>×</b>3<b>×</b>3 mm) piece of ice is left.</li>
<li>Using forceps, remove the tissue and place into the warmed medium.</li>
<li>Leave the tissue in the warm medium for 5 min to remove the cryo-preservative.</li>
</ol>
</li>
<li>Blot the tissue on a lint-free laboratory wipe to remove excess liquid clinging to the surface.</li>
<li>Put the tissue in the pre-weighed tube and weigh the tube again. Calculate the tissue weight by subtracting the weight of the empty tube.
<ol type="a">
<li>If the tissue is ≥ 7 mg, trim until it is &lt; 7 mg using a sterile scalpel or razor blade. Pieces &gt; 7 mg will likely not digest well with this protocol. Individual samples can be split into multiple tubes and recombined later.</li>
</ol>
</li>
<li>Add 180 μL buffer ATL to each tube, making sure that the tissue is submerged in the buffer.</li>
<li>Add 20 μL 20 mg/mL proteinase K to the tube.</li>
<li>Mix by pulse-vortexing for 15 s. It is important that the proteinase K is well mixed into the solution.</li>
<li>Place the tube in the heat block and incubate without shaking at 56°C for <b>at least 18 h</b> until the sample is completely lysed.
<ol type="a">
<li>If possible, check the digestion after a few hours to see how it has progressed. As the tissue digests the original piece will dissolve and no longer be recognizable as a solid chunk. Depending on tissue type, a shorter incubation may be possible for complete digestion/lysis.</li>
</ol>
</li>
<li>After 18 h of incubation, gently invert the tube and visually inspect to see if tissue has fully digested.
<ol type="a">
<li>It is useful to hold the tube up to a dark surface to see if there are remaining light-colored undigested tissue pieces floating in the tube.</li>
<li>If sample is completely dissolved, proceed to genomic DNA (gDNA) isolation.</li>
<li>If sample has not fully lysed (e.g., tissue is still intact, or there is visible material in suspension), repeat proteinase K addition as described above and return samples to heat block. Check after a few hours or up to 18 h (this long incubation can be performed overnight) until the tissue is digested completely. If the tissue does not digest, the piece of tissue may have been too large.</li>
</ol>
</li>
<li>Once the tissue is lysed, proceed to gDNA extraction</li>
</ol>
<h3 id="sec4.2">Cell thawing and lysis (if starting material is cells, not tissue)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h depending on number of samples and if isolating CD4<sup><b>+</b></sup> <b>cells</b>
</div>
<ol start="14">
<li>Thaw cryopreserved peripheral blood mononuclear cells ( PBMCs) or CD4<sup>+</sup> T cells into 37°C complete culture medium.</li>
<li>Spin cells for 10 min at 500 <b>×</b> <i>g</i> to pellet the thawed cells so that the cryopreservative can be removed.</li>
<li>Discard supernatant and resuspend cells in an appropriate volume for counting.</li>
<li>Count total cells, or, if not already done, isolate CD4<sup>+</sup> T cells using the EasySep™ Human CD4+ T Cell Isolation Kit (StemCell 17952) or your preferred method, then count the cells.
<ol type="a">
<li>Counting may be performed manually with a hemocytometer or with an automated cell counter. Once a method is chosen, it should be used for all samples going forward to avoid batch to batch differences.
<ol type="i">
<li>If cells are counted manually, a minimum of 200 cells should be counted on each side of the hemocytometer to ensure an accurate count.</li>
<li>If an automated cell counter is used, make sure that the instrument has been calibrated to the company’s specifications and any necessary quality control and cleaning steps have been performed prior to counting the experimental sample.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> that while we recommend isolating CD4+ T cells when possible, this method can work on samples in which CD4+ T cell separation is not ideal such as cells frozen without cryopreservation.</div>
<ol start="18">
<li>Transfer up to 2 <b>×</b> 10<sup>6</sup> cells to a 2 mL conical tube (ex. “Eppendorf”). Samples with more cells should be split and processed separately.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Check the centrifuge rating for the tubes and do not spin above this speed as the tubes may crack.</div>
<ol start="19">
<li>Pellet cells by spinning at 2,400 <b>×</b> <i>g</i> in a benchtop centrifuge for 5 min.</li>
<li>Remove supernatant and discard into 10% bleach by pipetting.</li>
<li>Using wide-bore filter tips, resuspend the pellet in 100 μL of 3M guanidine HCl
<ol type="a">
<li>If wide-bore tips are not available, use sterile scissors or a sterile razor blade to cut 2–3mm from the narrow end of a regular bore tip. This will leave a wider orifice like in a commercially sourced wide-bore tip.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Pipette gently using wide-bore filter tips to avoid shearing the DNA.</div>
<ol start="22">
<li>Add 20 μL of 20 mg/mL proteinase K.</li>
<li>Invert gently five times to mix thoroughly. It is important to mix gently to avoid shearing the DNA.</li>
<li>Incubate at 56°C for 1 h. During this time, put the 6M guanidine isothiocyanate in 37°C water bath or incubator to dissolve any precipitates. Vortex occasionally to help the crystals dissolve.</li>
<li>Proceed to gDNA extraction.</li>
</ol>
<h3 id="sec4.3">gDNA extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min for one sample, more time is needed with additional samples</div>
<div class="critical">
<span class="critical-title">Critical:</span> Dispose of guanidinium salts as chemical waste according to your facility’s rules. Do not mix with bleach, do not pour down drain.</div>
<div class="note">
<span class="note-title">Note:</span> This step includes the initial extraction of gDNA using guanidine salts, precipitation and washing with 70% ethanol. At this step, cell samples will contain 20 μL of Guanidine HCl (GuHcl) and 20 μL of 20 mg/mL proteinase K (<a href="#bib8">Wiegand et al., 2017</a>). Tissue samples will contain 180 μL of buffer ATL and 20 μL of 20 mg/mL proteinase K.</div>
<ol start="26">
<li>Add 400 μL 6M guanidine isothiocyanate to the sample tube.</li>
<li>Mix :
<ol type="a">
<li>For a <b>cell</b> sample, mix by pipetting with wide-bore filter tips.</li>
<li>For a <b>tissue</b> sample, invert the tube several times (pipette tips may clog).</li>
</ol>
</li>
<li>Incubate at 56°C for 10 min.</li>
<li>Add 1 mL of 100% molecular grade isopropanol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Isopropanol should be at about 25°C, cold temperatures will cause more contaminating salts to precipitate out with the DNA.</div>
<ol start="30">
<li>Invert the tube 10 times to mix.</li>
<li>Centrifuge for 10 min at the maximum speed (e.g., 21,000 <b>×</b> <i>g</i>) your tubes are rated for.</li>
<li>Pipet off most of the supernatant, leaving ∼100 μL behind.
<ol type="a">
<li>Dispose of the supernatant according to your facility’s procedures for guanidine.</li>
</ol>
</li>
<li>Add 1 mL of 70% ethanol to wash the pellet.</li>
<li>Invert the tube 10 times to mix.</li>
<li>Centrifuge 1 min at maximum speed.</li>
<li>Completely remove the supernatant with a pipette.</li>
<li>Spin down briefly with a micro benchtop centrifuge and remove any remaining liquid with a 20 μL pipette tip.</li>
</ol>
<p>Air dry 5–10 min (lid open) on bench, in fume hood or biosafety cabinet. The pellet may be mostly clear or an opaque white color when it is still saturated with ethanol. Due to the viscosity of the high molecular weight DNA, the precipitated salts may spread into a “smear” on the side of the tube rather than a tight pellet. This is not a problem.</p>
<ol start="38">
<li>Check that the precipitate appears dry and crystalline and that there is no odor of ethanol. If it appears wet and/or there is an odor, continue drying until it meets the criteria above.</li>
<li>Add 300 μL buffer EB and vortex 5 s.</li>
<li>Incubate at 60°C for 1 h to dissolve the pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The extracted nucleic acid will be high molecular weight at this point in the protocol and will therefore be much more viscous than nucleic acid extracted over a column or with intentional shearing. Avoid pipetting, vortexing, or freezing the sample at this point in the protocol: the goal is to maintain as much intactness as possible. All required fragmentation is achieved through restriction enzyme digestion at the next step. It is not necessary to attempt to quantify the DNA at this point in the protocol, it will likely be inaccurate due to the viscosity of the material.</div>
<h3 id="sec4.4">Restriction enzyme digestion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min hands-on, 19 h incubation</div>
<p>In this step, the restriction enzyme <i>Bgl</i>I (New England Biolabs, Cat# R0143S) is used to digest the high molecular weight nucleic acid isolated in the previous step. This enzyme was specifically chosen because it has cut sites throughout the human genome whereas most proviral genomes do not contain any <i>Bgl</i>I cut sites. The high molecular weight extraction followed by this specific enzyme digestion allows controlled fragmentation of the genomic DNA. Instructions for preparing an HIV negative control sample from commercially sourced human genomic DNA are in the preparation section above.</p>
<div class="note">
<span class="note-title">Note:</span> If there are multiple samples to be digested, make a master mix of 10<b>×</b> NEBuffer 3.1 and NEB <i>Bgl</i>I with the volumes described below and aliquot 48.2 ul into each sample.</div>
<ol start="41">
<li>Bring 10<b>×</b> NEBuffer 3.1 to ambient temperature.
<ol type="a">
<li>If salts have precipitated in the buffer, place in a 37°C incubator or water bath for about 5 min, then vortex, to dissolve the precipitates.</li>
</ol>
</li>
<li>Add 35 μL 10<b>×</b> NEBuffer 3.1 to the sample (which is already suspended in 300 μL buffer EB).</li>
<li>Add 13.2 μL NEB <i>Bgl</i>I enzyme (132 units).</li>
<li>Invert a few times to mix.</li>
<li>Put samples in the Thermomixer C using SmartBlock 1.5 mL with the lowest shaking speed and incubate overnight (16–18 h) at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 300 RPM is the lowest shaking speed for all blocks that fit in the Thermomixer C.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Overnight incubation 16–18 h.</div>
<ol start="46">
<li>After 16–18 h incubation, add 5 μL more <i>Bgl</i>I and incubate at 37°C for 1 h in the thermomixer at the lowest speed, which is approximately 300 RPM.</li>
</ol>
<h3 id="sec4.5">EtOH precipitation of <i>Bgl</i>I-digested samples</h3>
<ol start="47">
<li>Get wet ice in a suitably sized ice bucket for the number of samples you have.</li>
<li>Put 70% and 100% ethanol on dry ice or in a freezer to cool. These reagents do not need to reach a certain temperature; the cooling is only for making the precipitation more efficient. Salts precipitate more at colder temperatures.</li>
<li>Set a heat block for 65°C</li>
<li>Add 35 μL of 3M sodium acetate (bringing the solution to 0.3M sodium acetate) to the tubes containing the <i>Bgl</i>I-digested samples.</li>
<li>Add 766 μL of ice-cold 100% ethanol and mix the solution by inverting.</li>
<li>Place the tube on wet ice for 30 min.</li>
<li>Centrifuge at maximum speed (e.g., 21,000 <b>×</b> <i>g</i>) your tubes are rated for 10 min.</li>
<li>Carefully pipette off the supernatant down to 50–100 μL residual volume.</li>
<li>Add 1 mL of 70% ethanol and centrifuge at maximum speed for 2 min.</li>
<li>Spin down briefly with a micro benchtop centrifuge and remove any remaining liquid with a 20 μL pipette tip.</li>
<li>Air dry 5–10 min (lid open)</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The pellet of precipitated nucleic acid must be free of any remaining alcohol.</div>
<ol start="58">
<li>Add 50 μL of buffer EB to the pellet and allow it to dissolve 1 h at 65°C, then leave up to overnight at room temperature, about 25°C.
<ol type="a">
<li>If the gDNA is still viscous, repeat the restriction enzyme digestion and EtOH precipitation.</li>
</ol>
</li>
</ol>
<h3 id="sec4.6">Quantify concentration and dilute</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Depends on number of samples, at least 15 min</div>
<ol start="59">
<li>Measure the concentration of the gDNA with a NanoDrop or other spectrophotometer.</li>
<li>If the gDNA is &gt;200 ng/μL and &lt;300 ng/ul after the EtOH precipitation, dilute with Qiagen Buffer EB to a maximum of 200 ng/μL for the HIV assays. If the gDNA is &gt;300 ng/ul, repeat <i>Bgl</i>I digestion.</li>
<li>Prepare at least 20 μL of 1:100 diluted gDNA in Qiagen Buffer EB or molecular grade water. This will be used for the two replicate wells of the reference assay.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The values from the reference assay are used to make inferences about the concentrations in the HIV assay wells. Therefore, it is important to prepare the 1:100 dilution accurately. To minimize error, use a minimum of 5 μL stock template and 495 μL buffer EB.</div>
<ol start="62">
<li>Store the gDNA at 4°C until ready to proceed with ddPCR.</li>
</ol>
<h3 id="sec4.7">Add reagents to plates and generate droplets</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Depends on number of samples, at least 20 min</div>
<ol start="63">
<li>Get out two new twin-tec ddPCR 96–well plates, one for the master mix and DNA and one to receive the droplets once they are formed in the droplet generator.</li>
<li>Each well will receive a total of 22 μL before droplet generation: 17 μL of master mix and 5 μL of template. First prepare the master mix:
<ol type="a">
<li>For a single well, the master mix consists of 11 μL of 2<b>×</b> supermix and 3 μL of the assay pool (either the assay 1, assay 2 or RPP30_deltaD assay pools described above) and 3 μL of molecular grade water. Prepare enough master mix for the number of wells you are running plus 10% overage.</li>
</ol>
</li>
<li>Add 17 μL of master mix (supermix + assay pool + water) per well
<ol type="a">
<li>Fill the plate column-wise to save on droplet generator consumables</li>
<li>If there are any columns that don’t have master mix in all 8 wells, add 22 μL of 2<b>×</b> ddPCR Buffer Control for Probes to the empty wells in those columns.</li>
<li>
<i>Gently</i> tap the plate on the bench to bring reagents to the bottom of the wells.</li>
</ol>
</li>
<li>Seal the plate with an adhesive plate seal to prevent contamination if you are not immediately adding template DNA or need to move to the plate to another area.</li>
</ol>
<h3 id="sec4.8">Add template DNA and plasmid control samples to plate</h3>
<ol start="67">
<li>Add 5 μL template to the wells according to the plate layout: the HIV assays should be run in triplicate and the reference assays run in duplicate (see NOTE). We suggest that for each column, wells A, B and C are used for HIV assay1, D, E and F are used for HIV assay2 and wells G and H are used for the reference assay. An example plate layout is available in spreadsheet form in the supplemental folder <b>“</b>Data S1<b>”.</b> The file is called <b>“05162019_PTID1031_plate_layout_CL.csv</b>”
<div class="note">
<span class="note-title">Note:</span> If the sample has very low levels of integrated provirus, it may be necessary to survey a larger volume of sample to pick up this rare signal. If this is suspected, we suggest increasing the number of replicates of the HIV assay wells so that a larger amount of DNA can be assayed.</div>
<ol type="a">
<li>Add <b>undiluted</b> gDNA to the HIV assay wells.
<div class="critical">
<span class="critical-title">Critical:</span> DNA used for the ddPCR template should have a maximum concentration of 200 ng/μL. A higher concentration will interfere with droplet formation.</div>
</li>
<li>Add <b>diluted template</b> to the RRP30_deltaD assay wells: undiluted template will likely lead to an oversaturated signal (all droplets positive) and an underestimate of DNA shearing.
<ol type="i">
<li>For experimental (derived from a study participant as opposed to a plasmid control or other artificially prepared sample) samples, dilute the gDNA 1:100 with Qiagen Buffer EB.</li>
<li>For the plasmid/HIV-negative gating controls, dilute 1:2 with Qiagen Buffer EB.
<div class="note">
<span class="note-title">Note:</span> This dilution may need to be adjusted due to batch to batch variation in plasmid controls or degradation of the controls over time. The goal is to have enough positive droplets for clear populations and this will vary from batch to batch of controls.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="68">
<li>Apply the foil seal to the plate with the auto sealer and invert the plate several times to thoroughly mix the master mix and DNA in the plate.</li>
<li>Spin down the plate to bring all liquid to the bottom of the wells.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Check for bubbles on the bottom of the wells. Spin again if you see any. Bubbles on top are ok. Bubbles on the bottom will interfere with droplet generation by the autoDG.</div>
<h3 id="sec4.9">Generate droplets</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Up to 35 min, or less if entire plate is not used</div>
<div class="note">
<span class="note-title">Note:</span> See the autoDG manual for details on use: <a href="https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10043138.pdf">https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10043138.pdf</a>
</div>
<ol start="70">
<li>Assemble the autoDG (droplet generator) with cartridge, tips, template plate and empty plate (where the newly generated droplets will go) in their appropriate locations.</li>
<li>Configure the wells on the autoDG and initiate droplet generation.</li>
<li>When the droplets are done, heat seal the droplet plate with a pierce-able foil seal with auto sealer and dispose of the plate that held the master mix and DNA.</li>
<li>Observe wells for any abnormalities in droplet formation: there should be a uniform opaque layer at the top of each well.</li>
</ol>
<h3 id="sec4.10">Thermal cycling (PCR)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> about 2.5 h</div>
<ol start="74">
<li>Transfer the droplets plate to the thermal-cycler and run the ddPCR protocol. The ramp rate should be 2°C for all steps. The run volume is 40 μL per well.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The plate must be completely sealed around all the edges to avoid evaporation during PCR cycling.</div>
<table id="undtbl7">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial enzyme activation</td>
<td>95°C</td>
<td>10 m</td>
<td>1</td>
</tr>
<tr>
<td>Denature</td>
<td>94°C</td>
<td>30 s</td>
<td rowspan="2">60 cycles</td>
</tr>
<tr>
<td>Anneal and extend</td>
<td>60°C</td>
<td>60 s</td>
</tr>
<tr>
<td>Enzyme deactivation</td>
<td>98°C</td>
<td>10 m</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> After the cycling is complete, the plate can be stored overnight (12–18 h) at 4°C before reading the droplets.</div>
<h3 id="sec4.11">Read droplets</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 min per well, 2 h for an entire 96-well plate.</div>
<ol start="75">
<li>When the PCR reaction is done, take the plate to the droplet reader, put the plate in the holder, and clamp it down.</li>
<li>Fill out the plate info in the QuantaSoft software.
<ol type="a">
<li>Highlight the wells that you are going to read.</li>
<li>Supermix is ddPCR supermix for probes (no dUTP).</li>
<li>Assign channel 1 and channel 2 to unknowns.</li>
</ol>
</li>
<li>Click run plate, then save the template file.
<ol type="a">
<li>Choose the FAM/HEX dye set.</li>
</ol>
</li>
<li>Wait until the first well starts to make sure there are no issues initiating the reading.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>If the initial input is about 2<b>×</b>10<sup>6</sup> PBMC or T cells, the maximum expected total output of extracted nucleic acid is approximately 10 ug (50 μL of 200 ng/μL). The expected droplet count is between 15,000 and 20,000 droplets per well. We recommend excluding any wells with fewer than 10,000 droplets due to reduced accuracy.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec6.1">Overview</h3>
<p>The main goal of this analysis is to convert droplet counts, which can be exported directly from QuantaSoft AP, to the number of copies of template that was positive for all three HIV targets (i.e., potentially intact HIV provirus), normalized to total cells or total T cells (e.g., X copies per 1E6 T cells).</p>
<p>Definitions:</p>
<p><b>Target</b>: The sequence of interest that is targeted by a set of primers and a corresponding probe (e.g., gag or env).</p>
<p><b>Cluster</b>: A group of droplets positive for a particular combination of targets. For example, the “triple positive” cluster is positive for all three targets in an assay.</p>
<p><b>Sample</b>: The source of template DNA. Each sample is measured in two HIV assays (three replicate wells each) and a reference assay (two replicate wells).</p>
<p><b>Merged counts</b>: Summed droplet counts across the replicate wells for a single sample, accounting for cluster membership.</p>
<p><b>Total merged count:</b> The total number of droplets in a merged well: all the droplets, regardless of cluster membership.</p>
<p><b>Negative droplets:</b> Droplets that are negative for all targets OR droplets that are negative for the cluster of interest.</p>
<p>Set-up: For each assay (the two HIV assays and the reference assay):</p>
<ul>
<li>“Merge” the replicate wells by adding up the droplet counts from replicate wells as if they were in a single big well.</li>
</ul>
<p>For the reference assay only:</p>
<ul>
<li>Correct the concentration based on the dilution of the template.</li>
<li>Correct the concentration for diploidy.</li>
</ul>
<p>Relate the reference targets to the HIV targets and normalize.</p>
<ul>
<li>Divide the concentration of each HIV cluster by the concentration of total cells or T cells.</li>
<li>Normalize to millions of total cells or total T cells.</li>
</ul>
<p>Calculate the five-target estimate (“5-TE”)</p>
<ul>
<li>Equations that will be used:</li>
</ul>
<p><b>Equation A</b>: Convert droplet counts to concentration in copies per μL</p>
<p class="math-ml"><math> <mrow> <mi>C</mi> <mi>o</mi> <mi>p</mi> <mi>i</mi> <mi>e</mi> <mi>s</mi> <mspace></mspace> <mi>p</mi> <mi>e</mi> <mi>r</mi> <mspace></mspace> <mtext>μ</mtext> <mi>L</mi> <mo>=</mo> <mo>−</mo> <mi>ln</mi> <mspace></mspace> <mrow> <mo>(</mo> <mfrac> <mrow> <mi>n</mi> <mi>e</mi> <mi>g</mi> <mi>a</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <mi>e</mi> <mspace></mspace> <mi>d</mi> <mi>r</mi> <mi>o</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mi>t</mi> <mi>s</mi> </mrow> <mrow> <mi>t</mi> <mi>o</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> <mspace></mspace> <mi>d</mi> <mi>r</mi> <mi>o</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mi>t</mi> <mi>s</mi> </mrow> </mfrac> <mo>)</mo> </mrow> <mo>÷</mo> <mi>d</mi> <mi>r</mi> <mi>o</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mi>t</mi> <mspace></mspace> <mi>v</mi> <mi>o</mi> <mi>l</mi> <mi>u</mi> <mi>m</mi> <mi>e</mi> </mrow> </math></p>
<p><b>Equation B</b>: Calculate the DNA shearing index (DSI):</p>
<p class="math-ml"><math> <mrow> <mi>D</mi> <mi>S</mi> <mi>I</mi> <mo>=</mo> <mspace></mspace> <mfrac> <mrow> <mi>a</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mi>s</mi> <mi>i</mi> <mi>n</mi> <mi>g</mi> <mi>l</mi> <mi>e</mi> </mrow> <mrow> <mi>a</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mi>s</mi> <mi>i</mi> <mi>n</mi> <mi>g</mi> <mi>l</mi> <mi>e</mi> <mo>+</mo> <mi>R</mi> <mi>P</mi> <mi>P</mi> <mn>30</mn> <mspace></mspace> <mi>d</mi> <mi>o</mi> <mi>u</mi> <mi>b</mi> <mi>l</mi> <mi>e</mi> <mspace></mspace> <mi>p</mi> <mi>o</mi> <mi>s</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <mi>e</mi> </mrow> </mfrac> </mrow> </math></p>
<p><b>Equation C:</b> Correct observed triple positive copies for DNA shearing</p>
<p class="math-ml"><math> <mrow> <mi>C</mi> <mi>o</mi> <mi>r</mi> <mi>r</mi> <mi>e</mi> <mi>c</mi> <mi>t</mi> <mi>e</mi> <mi>d</mi> <mo>=</mo> <mspace></mspace> <mfrac> <mrow> <mi>o</mi> <mi>b</mi> <mi>s</mi> <mi>e</mi> <mi>r</mi> <mi>v</mi> <mi>e</mi> <mi>d</mi> </mrow> <mrow> <mn>1</mn> <mo>−</mo> <mi>D</mi> <mi>S</mi> <mi>I</mi> </mrow> </mfrac> </mrow> </math></p>
<p><b>Equation D</b>: Calculate the probability that a droplet is positive for all three targets</p>
<p class="math-ml"><math> <mrow> <mi>p</mi> <mrow> <mo>(</mo> <mi>t</mi> <mi>r</mi> <mi>i</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mspace></mspace> <mi>p</mi> <mi>o</mi> <mi>s</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <mi>e</mi> <mo>)</mo> </mrow> <mo>=</mo> <mspace></mspace> <mfrac> <mrow> <mi>t</mi> <mi>r</mi> <mi>i</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mspace></mspace> <mi>p</mi> <mi>o</mi> <mi>s</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <mi>e</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>n</mi> <mi>c</mi> <mi>e</mi> <mi>n</mi> <mi>t</mi> <mi>r</mi> <mi>a</mi> <mi>t</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> </mrow> <mrow> <mi>t</mi> <mi>o</mi> <mi>t</mi> <mi>a</mi> <mi>l</mi> <mspace></mspace> <mi>H</mi> <mi>I</mi> <mi>V</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>n</mi> <mi>c</mi> <mi>e</mi> <mi>n</mi> <mi>t</mi> <mi>r</mi> <mi>a</mi> <mi>t</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> </mrow> </mfrac> </mrow> </math></p>
<p><b>Equation E</b>: Calculate the probability that a droplet is positive for all targets in assay1 <i>and</i> assay2:</p>
<p class="math-ml"><math> <mrow> <mi>p</mi> <mrow> <mo>(</mo> <mrow> <mn>5</mn> <mspace></mspace> <mi>t</mi> <mi>a</mi> <mi>r</mi> <mi>g</mi> <mi>e</mi> <mi>t</mi> <mspace></mspace> <mi>p</mi> <mi>o</mi> <mi>s</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <mi>e</mi> </mrow> <mo>)</mo> </mrow> <mo>=</mo> <mi>p</mi> <mrow> <mo>(</mo> <mrow> <mi>t</mi> <mi>r</mi> <mi>i</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mspace></mspace> <mi>p</mi> <mi>o</mi> <mi>s</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <msub> <mi>e</mi> <mrow> <mi>a</mi> <mi>s</mi> <mi>s</mi> <mi>a</mi> <mi>y</mi> <mn>1</mn> </mrow> </msub> </mrow> <mo>)</mo> </mrow> <mo>×</mo> <mi>p</mi> <mrow> <mo>(</mo> <mi>t</mi> <mi>r</mi> <mi>i</mi> <mi>p</mi> <mi>l</mi> <mi>e</mi> <mspace></mspace> <mi>p</mi> <mi>o</mi> <mi>s</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <msub> <mi>e</mi> <mrow> <mi>a</mi> <mi>s</mi> <mi>s</mi> <mi>a</mi> <mi>y</mi> <mn>2</mn> </mrow> </msub> <mo>)</mo> </mrow> </mrow> </math></p>
<p><b>Equation F:</b> Determine the five target estimate (the estimated number of droplets containing template with all 5 targets)</p>
<p class="math-ml"><math> <mrow> <mi>p</mi> <mrow> <mo>(</mo> <mrow> <mn>5</mn> <mspace></mspace> <mi>t</mi> <mi>a</mi> <mi>r</mi> <mi>g</mi> <mi>e</mi> <mi>t</mi> <mspace></mspace> <mi>p</mi> <mi>o</mi> <mi>s</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <mi>e</mi> </mrow> <mo>)</mo> </mrow> <mo>=</mo> <mi>p</mi> <mrow> <mo>(</mo> <mrow> <mn>5</mn> <mspace></mspace> <mi>t</mi> <mi>a</mi> <mi>r</mi> <mi>g</mi> <mi>e</mi> <mi>t</mi> <mspace></mspace> <mi>p</mi> <mi>o</mi> <mi>s</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>v</mi> <mi>e</mi> </mrow> <mo>)</mo> </mrow> <mo>×</mo> <mfrac> <mrow> <mi>H</mi> <mn>1</mn> <mo>+</mo> <mi>H</mi> <mn>2</mn> </mrow> <mn>2</mn> </mfrac> </mrow> </math></p>
<p>Where:</p>
<p><i>H1</i> = total HIV copies per 10<sup>6</sup> T cells by assay1</p>
<p><i>H2 =</i> total HIV copies per 10<sup>6</sup> T cells by assay2</p>
<p><b>Example data: Excel users,</b> please see the Excel file “V3_annotated_example_data.xlsx” in supplemental folder Data S2 for further description of analyzing the data in a spreadsheet program. <b>R users</b>, please see the .RProject directory in supplemental Data S2 for examples of using scripts to analyze the data.</p>
<h3 id="sec6.2">Export cluster data from QuantaSoft AP</h3>
<ol>
<li>Open the .qlp file in QuantaSoft AP (“QSAP”) and assign targets and assays:
<ol type="a">
<li>Experiment type: “Direct Quantification (DQ)”</li>
<li>Highlight all the HIV assay1 wells and in the plate editor tab, under “Assay Information” set these wells as “Amplitude multiplex” and assign the targets and dyes specific to assay1</li>
<li>Name: &lt;Enter sample name&gt; (optional)</li>
<li>Type: “Unknown”</li>
<li>Click the “ – “ sign to the left of the row with the Target Name “Target 4” to remove this row. There are only 3 targets per assay so the fourth slot is not needed.</li>
</ol>
</li>
</ol>
<table id="undtbl8">
<caption>Make the following assignments for assay1 wells:</caption>
<thead>
<tr>
<th>Target name</th>
<th>Target type</th>
<th>Signal Ch1</th>
<th>Signal Ch2</th>
</tr>
</thead>
<tbody>
<tr>
<td>3′pol</td>
<td>Unkn</td>
<td>FAM Lo</td>
<td>None</td>
</tr>
<tr>
<td>Tat</td>
<td>Unkn</td>
<td>FAM Hi</td>
<td>None</td>
</tr>
<tr>
<td>Env</td>
<td>Unkn</td>
<td>None</td>
<td>HEX Hi</td>
</tr>
<tr>
<td><i>Target 4</i></td>
<td><i>Unkn</i></td>
<td><i>None</i></td>
<td><i>HEX Lo</i></td>
</tr>
</tbody>
</table>
<table id="undtbl9">
<caption>assay2 wells:</caption>
<thead>
<tr>
<th>Target name</th>
<th>Target type</th>
<th>Signal Ch1</th>
<th>Signal Ch2</th>
</tr>
</thead>
<tbody>
<tr>
<td>5′pol</td>
<td>Unkn</td>
<td>FAM Hi</td>
<td>None</td>
</tr>
<tr>
<td>Gag</td>
<td>Unkn</td>
<td>None</td>
<td>HEX Lo</td>
</tr>
<tr>
<td>Env</td>
<td>Unkn</td>
<td>None</td>
<td>HEX Hi</td>
</tr>
<tr>
<td><i>Target 4</i></td>
<td><i>Unkn</i></td>
<td><i>FAM Lo</i></td>
<td><i>none</i></td>
</tr>
</tbody>
</table>
<table id="undtbl10">
<caption>RPP30_deltaD assay wells:</caption>
<thead>
<tr>
<th>Target name</th>
<th>Target type</th>
<th>Signal Ch1</th>
<th>Signal Ch2</th>
</tr>
</thead>
<tbody>
<tr>
<td>deltaD</td>
<td>Unkn</td>
<td>FAM Lo</td>
<td>None</td>
</tr>
<tr>
<td>5′<i>RPP30</i>
</td>
<td>Unkn</td>
<td>FAM Hi</td>
<td>None</td>
</tr>
<tr>
<td>3′<i>RPP30</i>
</td>
<td>Unkn</td>
<td>None</td>
<td>HEX Hi</td>
</tr>
<tr>
<td><i>Target 4</i></td>
<td><i>Unkn</i></td>
<td><i>None</i></td>
<td><i>HEX Lo</i></td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Repeat previous step for assay2 wells and the reference assay wells</li>
<li>In the Droplets tab, check if any wells had &lt;10k droplets. These should be excluded from the analysis later on, but it is good to check now in case there are obvious patterns such as all wells in one row having a low droplet number. This could indicate a problem with the droplet generator or a pipetting problem.</li>
<li>Apply auto-gates to the plasmid control wells</li>
<li>Apply the plasmid-based gates to the sample wells</li>
<li>Highlight all wells and export the Well Data in csv format.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> .csv format is required if you plan to use R for analyses. Excel is ok if analysis will be carried out in a spreadsheet program.</div>
<ol start="7">
<li>Go to “Analysis Tools” and “Export Cluster Data”
<ol type="a">
<li>Target names assigned in the Plate Editor will <i>not</i> appear in the cluster data, there will only be “Target 1”, “Target 2″and “Target 3” as column names. These names are assigned to targets in the following order: FAM Lo, FAM Hi, HEX Lo, HEX Hi.
<table id="undtbl11">
<thead>
<tr>
<th>Assay</th>
<th>Target 1</th>
<th>Target 2</th>
<th>Target 3</th>
</tr>
</thead>
<tbody>
<tr>
<td>assay1</td>
<td>3′pol (FAM Lo)</td>
<td>tat (FAM Hi)</td>
<td>env (HEX Hi)</td>
</tr>
<tr>
<td>assay2</td>
<td>5′pol (FAM Hi)</td>
<td>gag (HEX Lo)</td>
<td>env (HEX Hi)</td>
</tr>
<tr>
<td>RPP30_deltaD</td>
<td>deltaD (FAM Lo)</td>
<td>5′<i>RPP30</i> (FAM Hi)</td>
<td>3′<i>RPP30</i> (HEX Hi)</td>
</tr>
</tbody>
</table>
</li>
<li>Sample names assigned in the Plate Editor will <i>not</i> appear in the cluster data, samples can be identified by plate location (ex. “A01”). The R-based analysis example uses the package “plater” to merge meta-data with the output from QuantaSoft AP, which is one way to add Sample IDs to the results. For use with the R scripts, meta-data must be prepared in the format shown in the “example_plate_layout” tab of the “V3_annotated_example_data.xlsx” file in supplmental folder Data S1.</li>
</ol>
</li>
</ol>
<h3 id="sec6.3">Analysis of data after export from QSAP</h3>
<div class="note">
<span class="note-title">Note:</span> The following instructions refer to examples in the Excel example data. In the example spreadsheet, plain text represents data exported from QuantaSoft or carried over from a previous tab. Yellow highlighted data indicates new calculations that have been done in the worksheet. There are 12 tabs in <b>“V3_annotated_example_data.xlsx”</b>. The protocol below will reference the tab where an example can be found. Formulas are included in the spreadsheet cells to guide the calculation.</div>
<p>R users should review the README file provided with the supplemental R project file (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1097-Mmc3.zip">Data S2</a>) (<a href="#bib7">Wickham, 2017</a>; <a href="#bib2">Hughes, 2016</a>; <a href="#bib5">Muller, 2020</a>; <a href="#bib3">Lai, 2020</a>; <a href="#bib6">RStudio, 2015</a>). The scripts also contain comments detailing the steps and functions. The steps are generally the same as those described below for the Excel workflow but modified for efficiency (steps are performed on all samples at once) and data objects are formatted for manipulation using tidyverse (<a href="#bib7">Wickham, 2017</a>) functions.</p>
<h3 id="sec6.4">Set up cluster data</h3>
<ol start="8">
<li>Confirm plate layout (tab 1)</li>
<li>Open exported cluster data. (tabs 2–4).</li>
<li>Make a fake data set that includes all possible clusters for every sample. (Excel example data, tab 5) This step is necessary because QSAP only includes rows for clusters that were present for that sample (i.e., it will not list a cluster and indicate that there were zero droplets, there will simply be a row missing, tab 3, but this information is necessary).</li>
<li>Use the Query tool in Excel to perform an anti-join to see which rows in the “all possible clusters” created in the previous step data are missing from the real data. The anti-join merges the data so that the resulting table shows all rows present in one data set and absent in the other data set. (tab 6) The exact steps required to perform the anti-join will depend on your version of excel. See the following link for a tutorial on anti-joins with the Query tool: <a href="https://docs.microsoft.com/en-us/power-query/merge-queries-left-anti">https://docs.microsoft.com/en-us/power-query/merge-queries-left-anti</a>
</li>
<li>Add the missing clusters (with zero as the count) to the exported data (tab 7) by copying the results of the anti-join and pasting them to the bottom of the data set that was exported from QuantaSoft AP (in this example, we use the data from tab 4). After pasting the “missing” values, sort the results by Well to put all the rows belonging to the same Well in consecutive rows of the data set (tab 7).</li>
<li>Add meta data including Assay and unique sample identifiers that will be needed for sorting data (tab 8).</li>
</ol>
<p>Calculate HIV triple positive copies per μL from droplet counts</p>
<ol start="14">
<li>Tab 9: This example will use assay1 results from a single sample. The same steps should be performed for the rest of the samples and for assay2.
<ol type="a">
<li>Section A shows the three replicate wells of assay1 for sample_01. There are 8 rows per well, each representing a different cluster of targets.
<ol type="i">
<li>Rename the column headers “Target 1”, “Target 2” and “Target 3” with the actual HIV target names for assay1.</li>
</ol>
</li>
<li>Section B: Calculate the total droplets in each well, the total triple positive droplets and NON-triple positive droplets in each replicate well.
<ol type="i">
<li>Confirm that each well meets the required threshold of at least 10,000 droplets total. If not, exclude that well.</li>
</ol>
</li>
<li>Section C:
<ol type="i">
<li>“Merge” the counts from the 3 replicate wells by adding the counts together from section B.</li>
<li>Use the total merged and non-triple positive counts and the known droplet volume to calculate copies per μL of triple positive droplets (<b>Equation A).</b><br> <b>This is the concentration of droplets containing potentially intact provirus. NOTE:</b> See Bio-Rad’s Droplet Digital <sup>TM</sup> PCR Applications Guide, page 35 for mathematical details.</li>
</ol>
</li>
<li>Repeat the steps described in Sections A-C for assay2.</li>
</ol>
</li>
</ol>
<h3 id="sec6.5">Calculate copies per μL for reference assay clusters</h3>
<ol start="15">
<li>Section D shows the reference assay wells for sample_01. These wells received the same template as the wells in Section A but at a 1:100 dilution.
<ol type="a">
<li>Add a name for each cluster (section D column H). The cluster name references the targets that make up that cluster, which are indicated in columns B, C and D.</li>
</ol>
</li>
<li>Section E: Calculate the merged counts of the clusters across the replicate wells</li>
<li>Section F and G: Calculate copies per μL for droplets containing deltaD (alone or with another target) and for droplets containing 5′<i>RPP30</i> (<b>Equation A).</b>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> These concentrations are also available from the “well data” from QuantaSoft AP. “Well data” results show the concentration of each target regardless of whether it was in a cluster with any other target.</div>
<h3 id="sec6.6">Calculate the total amount of HIV present in any type of cluster</h3>
<ol start="18">
<li>Add Cluster names as was done for the reference assay in tab 9 section D. This is shown for assay1 data in tab 10 section H.</li>
<li>Sum the counts for each cluster and for all droplets to get merged counts (tab 10 section I).</li>
<li>Subtract the merged count for each cluster from the total merged count to get the number of droplets NOT positive for your cluster of interest (tab 10 section I column N).</li>
<li>Using the known droplet volume and <b>Equation A</b>, calculate the concentration in copies per μL for each cluster (tab 10 section I column P).</li>
<li>Calculate the concentration of HIV in any form (i.e., a target alone or in combination with any other target) by summing the all cluster concentrations <i>except the negative cluster</i>). See tab 10 section J. <b>This is the total concentration of any HIV.</b> This value can be compared to the triple positive results obtained in tab 9 section C.</li>
</ol>
<h3 id="sec6.7">Normalize to cell counts</h3>
<p>After following the steps above, we have concentrations in copies per μL for all clusters, including the triple positive cluster. Next, we will correct the reference target concentrations for dilution and diploidy, then use those corrected concentrations to normalize the HIV concentrations to copies per million T cells.</p>
<ol start="23">
<li>Multiply the reference gene concentrations by the dilution factor (100). In the reference gene assay, the template is 100<b>×</b> more dilute than the template used in the corresponding HIV assay wells. The concentrations therefore need to be multiplied by this dilution factor (100) to allow us to estimate how many cells-worth of DNA was used in the HIV assay wells (tab 11 section K).</li>
<li>Divide the reference gene concentrations by 2 to account for diploidy (every cell has two copies of the reference genes; tab 11 section K).</li>
<li>Tab 11 section K columns E and F show the values after making the dilution and diploidy corrections. <b>These are the concentrations of cells per μL and non-T cell per μL used in the HIV assays.</b>
</li>
<li>Calculate the number of T cells per μL. 5′<i>RPP30</i> represents the number of cells and deltaD represents the number of non-T cells. To get number of T cells, we subtract: total cells – nonT cells = T cells (tab 11 section K column G).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All T cell/μL counts should be &gt;0, otherwise it means that there were more non-T cells that total cells, which could indicate an error. If T cells were isolated prior to DNA extraction, there will be little (or no) DNA from non-T cells (therefore a low concentration of deltaD).</div>
<h3 id="sec6.8">Correct for DNA shearing</h3>
<p>The next step is to correct the observed concentration of triple positive droplets per million T cells for incidental DNA shearing with the DNA Shearing Index (DSI). The DSI is the probability that DNA template was sheared. The reference assay contains two targets in the <i>RPP30</i> gene: 5′<i>RPP30</i> and 3′<i>RPP30.</i> The distance between these targets is approximately the same length as the HIV genome and the <i>Bgl</i>I enzyme does not have cut sites in this region. When both <i>RPP30</i> targets occur in the same droplet, we can assume that they were on the same piece of template, i.e., the template was not sheared. If one of these targets occurs in a droplet without the other, we know that the DNA was sheared. We can then use the amount of shearing in this known fragment of genomic DNA as a proxy measurement for shearing in general, and apply this probability to the normalized triple positive HIV concentrations (<a href="#bib4">Levy et al., 2021</a>).</p>
<ol start="27">
<li>Use the counts of <i>RPP30</i> double and single positive droplets to calculate the DSI using Equation B (tab 11 section M, column F):</li>
<li>Using the observed triple positive copies per million cells calculated previously, we divide by 1-DSI to estimate a corrected value that accounts for shearing that reduced the number of triple positive droplets, thus artificially lowering the concentration <b>(Equation C).</b> Note that it is only possible to do this correction on the triple positive cluster.</li>
</ol>
<h3 id="sec6.9">Calculate 5-target estimate (“5TE”)</h3>
<p>Using the values we have calculated above for the total HIV copies per 1E6 T cells and the triple positive copies per 1E6 T cells for each of the two HIV triplex assays, we can calculate the probability that a template is positive for all three targets in the assay using <b>Equation D,</b> and the probability that a template is positive for all three targets in assay1 AND all three targets in assay2 with <b>Equation E.</b></p>
<p>Finally, we estimate the actual number of copies that are likely positive for all 5 targets by multiplying the probability by the average total copies per 1E6 cells calculated for the two HIV assays using <b>Equation F</b><b>.</b></p>
<ol start="29">
<li>Calculate the probability of triple positive for assay1 and assay2 using <b>Equation D</b> (tab 12 section N columns F and G).</li>
<li>Calculate the average total HIV detected for assay1 and assay2 (tab 12 section N column H)</li>
<li>Calculate the “five target estimate” (“5TE”) using <b>Equation F</b> (tab 12 section N column I).</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The degree of mechanical shearing of the DNA will depend on initial sample quality and preservation. In our experience, tissue samples have more shearing than cell samples.</p>
<p>In samples from volunteers on anti-retroviral treatment, the amount of integrated virus may be very low. A higher number of replicate wells may be needed to measure enough DNA to get a positive signal.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec8.1">Problem 1</h3>
<p>DNA is viscous and difficult to pipette after extraction or precipitation (Step-by-Step Method Details step 58).</p>
<h3 id="sec8.2">Potential solution</h3>
<p>Use fewer cells or bring the volume of extracted DNA to 300 μL with Qiagen buffer EB and repeat the 18hr <i>Bgl</i>I digestion, followed by precipitation.</p>
<h3 id="sec8.3">Problem 2</h3>
<p>Very few or zero droplets form during droplet generation (Step-by-Step Method Details after step 98).</p>
<h3 id="sec8.4">Potential solution</h3>
<p>The template DNA may be too viscous for droplet formation. Repeat the restriction digestion and precipitation.</p>
<h3 id="sec8.5">Problem 3</h3>
<p>Well volume is reduced after thermal-cycling (Step-by-Step Method Details step 78).</p>
<h3 id="sec8.6">Potential solution</h3>
<p>Adjust heat-sealing parameters to ensure a complete seal, check the plate sealer for malfunction.</p>
<h3 id="sec8.7">Problem 4</h3>
<p>There are no triple positive droplets present before shearing correction (Quantification and Statistical Analysis step 26).</p>
<h3 id="sec8.8">Potential solution</h3>
<p>If intact HIV proviral DNA is expected from the sample, there may be no triple positive droplets if the DNA was sheared too much. Check the DSI and the double positive droplet counts for the <i>RPP30</i> targets. Another possibility is that the HIV reservoir from which the sample was derived is very small and more DNA needs to be surveyed to detect intact provirus.</p>
<h3 id="sec8.9">Problem 5</h3>
<p>All droplets are positive for a particular target so it is not possible to use the provided equations to calculate target concentrations (Step-by-Step Method Details, after step 78).</p>
<p>This is not likely to occur with HIV targets in human samples but could occur with plasmid or other control samples and could occur when the reference assay is used on improperly diluted human samples.</p>
<h3 id="sec8.10">Potential solution</h3>
<p>All positive droplets are an indication that the target concentration of the input sample is too high. This could occur if the input was not diluted properly or if there is contamination of the template sample. If contamination (for example, of template sample with plasmid) is suspected, first test the PCR reagents, including water, for contamination by running samples without template added to the PCR mixture. If the reagents are contaminated, these “no template” samples will still amplify. If this occurs, replace the contaminated reagents. If the PCR reagents are not contaminated, test several dilutions of the template in case it was not diluted enough. If the dilution factor is not the problem, review lab protocols for template and plasmid containment and prepare new template.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Green, M.R. and Sambrook, J. (2012). Precipitation of DNA with ethanol. Cold Spring Harb. Protoc. <i>2012</i>, 21-25. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00591-8/sref1">View at publisher</a></p>
<p id="bib2">Hughes, S.M. (2016). Plater: read, tidy, and display data from microtiter plates. J. Open Source Softw. . <a class="external-link" href="https://doi.org/10.21105/joss.00106">https://doi.org/10.21105/joss.00106</a></p>
<p id="bib3">Lai, R. (2020). Arrangements: Fast Generators and Iterators for Permutations, Combinations, Integer Partitions and Compositions. R Package Version 1.1.9. The Comprehensive R Archive Network. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00591-8/sref3">View at publisher</a></p>
<p id="bib4">Levy, C.N., Hughes, S.M., Roychoudhury, P., Reeves, D.B., Amstuz, C., Zhu, H., Huang, M.-L., Wei, Y., Bull, M.E., Cassidy, N.A.J., et al. (2021). A highly multiplexed droplet digital PCR assay to measure the intact HIV-1 proviral reservoir. Cell Rep. Med. <i>2</i>, 100243. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00591-8/sref4">View at publisher</a></p>
<p id="bib5">Muller, K. (2020). Here: A Simpler Way to Find Your Files. R Package Version 1.0.1. ed. The Comprehensive R Archive Network. . <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00591-8/sref5">View at publisher</a></p>
<p id="bib6">RStudio. (2015). RStudio: integrated development for R. RStudio, inc. <a class="external-link" href="http://rstudio.com/">http://rstudio.com/</a></p>
<p id="bib7">Wickham, H. (2017). Tidyverse: easily install and load the 'tidyverse'. <a class="external-link" href="https://CRAN.R-project.org/package=tidyverse">https://CRAN.R-project.org/package=tidyverse</a></p>
<p id="bib8">Wiegand, A., Spindler, J., Hong, F.F., Shao, W., Cyktor, J.C., Cillo, A.R., Halvas, E.K., Coffin, J.M., Mellors, J.W., and Kearney, M.F. (2017). Single-cell analysis of HIV-1 transcriptional activity reveals expression of proviruses in expanded clones during ART. Proc. Natl. Acad. Sci. U S A <i>114</i>, E3659-E3668. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00591-8/sref8">View at publisher</a></p>
<p id="bib9">Zoutman, W.H., Nell, R.J., Versluis, M., Van Steenderen, D., Lalai, R.N., Out-Luiting, J.J., De Lange, M.J., Vermeer, M.H., Langerak, A.W., and Van Der Velden, P.A. (2017). Accurate quantification of T cells by measuring loss of germline T-cell receptor loci with generic single duplex droplet digital PCR assays. J. Mol. Diagn. <i>19</i>, 236-243. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00591-8/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Florian Hladik (<a href="mailto:florian@uw.edu">florian@uw.edu</a>).</p>
<h4>Materials availability</h4>
<p>The plasmid sequences used for gating controls are available from Addgene with the following identifiers: 167347, Seattle_IPDA_control_1_001; 167348, Seattle_IPDA_control_6_002; 167349, Seattle_IPDA_control_9_003; 167350, Seattle_IPDA_control_14_004; 167351, Seattle_IPDA_control_17_005; 167352, Seattle_IPDA_control_29_006; 167353, Seattle_IPDA_control_32_007.</p>
<h4>Data and code availability</h4>
<p>An annotated example data set accompanies this protocol. These data are a subset of the UW_CFAR_KINETICS cohort that was analyzed for our related publication: Levy et al., A highly multiplexed droplet digital PCR assay to measure the intact HIV-1 proviral reservoir, Cell Reports Medicine (2021), <a href="https://doi.org/10.1016/j.xcrm.2021.100243">https://doi.org/10.1016/j.xcrm.2021.100243</a>.</p>
<p>These data are from participant samples that were obtained with approval from the University of Washington Institutional Review Board.</p>
<p>In addition to the example data, we provide R code and a step-by-step spreadsheet-based workflow for data analysis in the supplementary documents to this manuscript.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1097-Mmc1.pdf">Document S1. Figure S1</a></p>
<p><a id="mmc2" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1097-Mmc2.zip">Data S1. Data analysis example (spreadsheet method), related to quantification and statistical analysis steps 8–31</a></p>
<p><a id="mmc3" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1097-Mmc3.zip">Data S2. Data analysis example (R method), related to quantification and statistical analysis steps 8–31</a></p>
<h3>Acknowledgments</h3>
<p>Research reported in this publication was supported by the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (NIH) grants UM1 AI126623 (to K.R.J.), R01 AI116292 and R01 DA040386 (to F.H.), R01 HD094718 (to D.A.L.), and the <a href="https://doi.org/10.13039/100000002">NIH</a>-funded Centers for AIDS Research P30 AI027757 and P30 AI064518. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. We thank Alex Taber for advice on plasmid cloning methods. The <a href="https://doi.org/10.13039/100010296">James B. Pendleton Trust</a> provided funds to purchase the ddPCR system used for much of the assay development. The graphical abstract was created using Biorender. We thank study participants for their generous and essential support of this work.</p>
<h3>Author contributions</h3>
<p>C.N.L., S.M.H., M.-L.H., and H.Z. planned assay development experiments. C.N.L., S.M.H., and H.Z. performed assay development experiments. S.M.H. and P.R. performed <i>in vitro</i> and <i>in silico</i> restriction enzyme digestion optimization. All authors contributed to assay design planning. P.R. assisted with calculations related to the DNA shearing index (DSI) and provided R expertise. C.N.L. developed the analysis pipeline and wrote this protocol.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>