<article>
<section>
<h1 id="header">Assessing Budding Yeast Phosphoproteome Dynamics in a Time-Resolved Manner using TMT10plex Mass Tag Labeling</h1>
<p><time datetime="2020-03-20">Published: March 20, 2020</time></p>
<p>Andrew W. Jones,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Helen R. Flynn,<sup><a href="#aff1">1</a></sup> Frank Uhlmann,<sup><a href="#aff3">3</a></sup> Ambrosius P. Snijders,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> and Sandra A. Touati<sup><a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Mass Spectrometry Proteomics Science Technology Platform, The Francis Crick Institute, London NW1 1AT, UK</p>
<p id="aff2"><sup>2</sup>Cell Cycle Laboratory, The Francis Crick Institute, London NW1 1AT, UK</p>
<p id="aff3"><sup>3</sup>Chromosome Segregation Laboratory, The Francis Crick Institute, London NW1 1AT, UK</p>
<p id="aff4"><sup>4</sup>Institut de Biologie Paris Seine, CNRS UMR7622, Sorbonne Université, Paris, France</p>
<p id="fn1"><sup>5</sup>Technical Contact</p>
<p id="fn2"><sup>6</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:bram.snijders@crick.ac.uk">bram.snijders@crick.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sandra.touati@sorbonne-universite.fr">sandra.touati@sorbonne-universite.fr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100022">10.1016/j.xpro.2020.100022</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Amine-reactive Tandem Mass Tag 10plex (TMT10plex) labeling permits multiplexed protein identification and quantitative analysis by tandem mass spectrometry (MS/MS). We have used this technology to label 20 <i>Saccharomyces cerevisiae</i> samples collected in a time-resolved manner from a wild-type and phosphatase mutant background to characterize phosphoproteome dynamics. Here, we provide a detailed protocol for biological and mass spectrometry sample preparation and analysis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib9">Touati et al. (2019)</a>.</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Characterization of the phosphoproteome in a time-resolved manner</li>
<li>TMT10plex Mass Tag labelling of budding yeast extracts</li>
<li>Mass spectrometry sample preparation and analysis</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/66-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<h3>Cell Culture and Synchronization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol>
<li>Grow control and mutant cells for 16 hours in 400 ml of YP media (<a href="#bib6">Rose et al., 1990</a>) containing 2 % galactose and 2 % raffinose at 25°C.</li>
<li>In the morning, back dilute the cells to OD <sub>600</sub>=0.1 (around 1.5∗10<sup>6</sup> cells/ml) and grow them in YP medium containing 2 % galactose and 2 % raffinose at 25°C. At OD<sub>600</sub>=0.2 (around 3∗10<sup>6</sup> cells/ml), filter the cells, wash with 3 l of YP galactose-free medium and grow them for 3 h in YP medium containing 2 % raffinose at 25°C. This results in a metaphase cell cycle arrest due to Cdc20 depletion.</li>
<li>Cells should have reached OD<sub>600</sub>=0.3-0.4 (around 4.5-6∗10<sup>6</sup> cells/ml). Collect 25 ml for the first time point (t=0) just before galactose re-addition and process the sample immediately (see 2. Extract preparation).</li>
<li>Add 2 % of galactose to both control and mutant cells to re-induce Cdc20 expression and synchronous progression through the mitotic stages before returning to G1. Add also 0.8 μg/ml of α-factor to re-arrest cells in G1 at the end of the experiment.</li>
<li>Collect time-point samples at t=5, 10, 15, 20, 25, 30, 40, 60 and 90 min in control and mutant strains and process the samples immediately (<a href="#fig1">Figure 1</a>).</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/66-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Cell Synchronization and Collection</div>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> A stock solution of 20% galactose and 20% raffinose is used.</div>
<div class="critical">
<span class="critical-title">Critical:</span> A high level of cell cycle synchrony is essential when performing global phosphoproteome analysis in a time-resolved manner. We used <i> Saccharomyces cerevisiae</i> cells expressing the anaphase promoting complex/cyclosome (APC/C) co-activator Cdc20 under control of the galactose-inducible <i>GAL1</i> promoter. Both control and phosphatase mutant cells harbor this modified version of the <i>CDC20</i> gene, allowing cell cycle arrest in metaphase and synchronous release into progression through anaphase and mitotic exit (<a href="#bib10">Uhlmann et al., 1999</a>).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> To synchronize the cells in metaphase, the gene Cdc20 can be placed under control of the methionine-repressible MET3 promoter. In this case, cells have to be grown for 16 hours in minimal media without methionine.</div>
<h3>Extract Preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<ol start="6">
<li>Centrifuge samples at 2000 g for 3 min and discard the supernatant.</li>
<li>Re-suspended the pellet in 10 ml of ice-cold 20 %TCA, vortex and keep on ice for at least 1 h. The samples from throughout the timecourse analysis will be collected at this stage. Then all samples are further processed together.</li>
<li>Remove the TCA by aspiration and wash the pellets twice with 10 ml of cold acetone (kept at -20 ˚C). Following the final acetone wash, ensure that any remaining acetone has evaporated before continuing to the next step.</li>
<li>Resuspend the samples in 400 μl of lysis buffer (8 M urea, 50 mM HEPES pH 8.2, 5 mM EDTA, 1 mM dithiothreitol, 50 mM sodium fluoride, 1 mM sodium vanadate, 1 mM PMSF, cOmplete EDTA-Free Protease Inhibitor Cocktail) and transfer the samples into 2 ml Eppendorf tubes suitable for use in a cell breaker.</li>
<li>Add 500 μl acid-washed glass beads and beat on a Fast-Prep cell breaker in the cold room. Program: 6.0 m/s - 4 x 40 s. Ensure that lids remain safely closed between the breakage rounds. Confirm efficient cell breakage using a phase contrast microscope where intact cells appear bright while broken cells appear dark.</li>
<li>Punch a hole with a heated hypodermic needle to the bottom of the cell breaker tubes, place into a 15 ml centrifuge tube and collect the extracts by brief centrifugation.</li>
<li>Transfer the extracts into 1.5 ml test tubes and centrifuge at 16000 g for 10 min.</li>
<li>Retrieve the supernatants and measure the protein concentration using Protein Assay Dye. The protein concentration should be &gt;500 μg/ml.</li>
<li>Transfer a volume corresponding to 200 μg of protein into LoBind microcentrifuge tubes and freeze at -80 °C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples can be stored at -80 °C indefinitely.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The liquid culture can be precipitated directly in TCA 100% with a ratio 1 per 5. Simplified lysis buffer containing 8 M urea, 5 mM EDTA pH 7.5 and 50 mM ammonium bicarbonate has also been used.</div>
<div class="note">
<span class="note-title">Note:</span> Primary amines interfere with the TMT labeling, so the lysis buffer has to be free of reactive amines (<i>e.g.</i> Tris), hence it is recommended to use buffers such as HEPES or triethylammonium bicarbonate.</div>
<div class="note">
<span class="note-title">Note:</span> Detergents can interfere with LC-MS/MS-based analysis, so the lysis buffer should be free of detergents (<i>e.g.</i> sodium dodecyl sulfate). If this is not possible, then alternative sample cleaning methods, <i>e.g.</i> SP3 (<a href="#bib3">Hughes et al., 2019</a>) or FASP (<a href="#bib11">Wiśniewski et al., 2009</a>), may be employed.</div>
<div class="note">
<span class="note-title">Note:</span> The remainder of the material can be retained to perform additional experiments, e.g. western blotting, as applicable.</div>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>α-factor</td>
<td>Peptide Chemistry Science Technology Platform, The Francis Crick institute <br>Sequence: NH<sub>2</sub> -WHWLQLKPGQPMY-COOH</td>
<td>N/A</td>
</tr>
<tr>
<td>cOmplete EDTA-Free Protease Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>Cat# 04693132001</td>
</tr>
<tr>
<td>Protein Assay Dye</td>
<td>Bio-Rad</td>
<td>Cat# 5000006</td>
</tr>
<tr>
<td>Trichloroacetic acid</td>
<td>Sigma-Aldrich</td>
<td>Cat# T0699</td>
</tr>
<tr>
<td>Urea</td>
<td>Fisher Scientific</td>
<td>Cat# BP169</td>
</tr>
<tr>
<td>HEPES pH 8.2</td>
<td>Sigma-Aldrich</td>
<td>Cat# H3375</td>
</tr>
<tr>
<td>EDTA</td>
<td>Sigma-Aldrich</td>
<td>Cat# E9884</td>
</tr>
<tr>
<td>Sodium fluoride</td>
<td>Sigma-Aldrich</td>
<td>Cat# S1504</td>
</tr>
<tr>
<td>Sodium vanadate</td>
<td>New England BioLabs</td>
<td>Cat# P0758</td>
</tr>
<tr>
<td>PMSF</td>
<td>Sigma-Aldrich</td>
<td>Cat# P7626</td>
</tr>
<tr>
<td>Pierce Trypsin Protease</td>
<td>ThermoFisher</td>
<td>Cat# 90058</td>
</tr>
<tr>
<td>Dithiothreitol</td>
<td>Sigma-Aldrich</td>
<td>Cat# D5545</td>
</tr>
<tr>
<td>Iodoacetamide</td>
<td>Sigma-Aldrich</td>
<td>Cat# A3221</td>
</tr>
<tr>
<td>HEPES pH 8.5</td>
<td>Fisher Scientific</td>
<td>Cat# 15433969</td>
</tr>
<tr>
<td>Trifluoroacetic acid</td>
<td>Fisher Scientific</td>
<td>Cat# A116</td>
</tr>
<tr>
<td>Acetonitrile</td>
<td>Fisher Scientific</td>
<td>Cat# A955</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>Fisher Scientific</td>
<td>Cat# A11350</td>
</tr>
<tr>
<td>50 % Hydroxylamine</td>
<td>ThermoFisher</td>
<td>Cat# 90115</td>
</tr>
<tr>
<td>Formic acid</td>
<td>Fisher Scientific</td>
<td>Cat# A117</td>
</tr>
<tr>
<td>DMSO</td>
<td>Honeywell</td>
<td>Cat# 41640</td>
</tr>
<tr>
<td>Water</td>
<td>Fisher Scientific</td>
<td>Cat# W6</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>TMT10plex Isobaric Label Reagent Set 1 x 0.8 mg</td>
<td>ThermoFisher</td>
<td>Cat# 90110</td>
</tr>
<tr>
<td>Pierce TiO2 Phosphopeptide Enrichment Spin Kits</td>
<td>ThermoFisher</td>
<td>Cat# 88303</td>
</tr>
<tr>
<td>High-Select Fe-NTA Phosphopeptide Enrichment Kit</td>
<td>ThermoFisher</td>
<td>Cat# A32992</td>
</tr>
<tr>
<td colspan="3">Experimental Models: Organisms/Strains</td>
</tr>
<tr>
<td>
<i>Saccharomyces cerevisiae</i> strains W303</td>
<td><a href="https://doi.org/10.17632/tspgywx7g3.1"> https://doi.org/10.17632/tspgywx7g3.1 </a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>Perseus v1.4.0.2</td>
<td>Perseus</td>
<td><a href="https://maxquant.net/perseus/"> https://maxquant.net/perseus/</a></td>
</tr>
<tr>
<td>MaxQuant v1.5.0.13</td>
<td>MaxQuant</td>
<td><a href="https://www.maxquant.org"> https://www.maxquant.org </a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Fast-Prep cell breaker</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Refrigerated centrifuge capable of achieving 18,000 ×<i>g</i>
</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Benchtop centrifuge</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Spectrophotometer</td>
<td>Ultrospec 7000</td>
<td>Cat# 29003605</td>
</tr>
<tr>
<td>Heat block (37°C and 56°C)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Thermomixer</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>SpeedVac vacuum concentrator</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Ultrasonic water bath</td>
<td>Ultrawave</td>
<td>Cat# U300</td>
</tr>
<tr>
<td>Sep-Pak C<sub>18</sub> Vac Cartridges</td>
<td>Waters</td>
<td>Cat# WAT054955</td>
</tr>
<tr>
<td>Vacuum manifold</td>
<td>Phenomenex</td>
<td>Cat# AH0-6024</td>
</tr>
<tr>
<td>Empore C<sub>18</sub> membrane</td>
<td>Supelco</td>
<td>Cat# 66883-U</td>
</tr>
<tr>
<td>UltiMate 3000 HPLC System</td>
<td>Thermo Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>EASY-Spray C<sub>18</sub> column 75 μm × 50 cm</td>
<td>ThermoFisher</td>
<td>Cat# ES803</td>
</tr>
<tr>
<td>Orbitrap Fusion Lumos Tribrid Mass Spectrometer</td>
<td>Thermo Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Total recovery glass autosampler vials</td>
<td>Waters</td>
<td>Cat# 186005663CV</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Protein Digestion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 18–24 h</div>
<p>As this protocol is interested in identifying and quantifying individual phosphosites, a bottom-up approach is being undertaken. The initial step of the vast majority of bottom-up proteomic approaches requires the digestion of proteins into peptides, in this case, we are using the widely used enzyme trypsin which cleaves C-terminal to lysine and arginine residues.</p>
<ol>
<li>Thaw the samples and treat each lysate with 1 μl of 1 M dithiothreitol for 25 min at 56 °C to reduce disulfide bonds between cysteine residues. Cool samples to ∼22 °C before proceeding to the next step.</li>
<li>Alkylate the cysteine sulfhydryls using 4 μl of 500 mM iodoacetamide for 30 min at ∼22 °C, protected from light. Iodoacetamide has to be in excess compared to the dithiothreitol. This step prevents the reformation of disulfide bonds between cysteine residues.</li>
<li>Quench the reaction with 1.5 μl 1 M dithiothreitol to stop the alkylation reaction.</li>
<li>Before preforming the proteolytic digestion, the concentration of urea must be reduced to &lt;2 M. Add four sample volumes of 50 mM HEPES pH 8.5. (e.g. 200 μl sample:800 μl 50 mM HEPES pH 8.5).</li>
<li>Add sequencing grade trypsin at 2 % of total protein (4 μg/sample) and incubate the reaction at 37 °C for 6-18 h. Trypsin cleaves peptides on the C-terminal side of lysine and arginine amino acid residues (except when proline is the next residue).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Trypsin digestion results in peptides with an average of 14 amino acids in length, which is favorable for LC-MS/MS analysis. Other enzymes, <i>e.g.</i> lysyl endopeptidase (LysC), may be used in place of, or in parallel to, trypsin. LysC cleaves C-terminal to lysine residues which ensures all peptides contain two primary amine groups which are the acceptors for TMT labelling.</div>
<h3>C<sub>18</sub> Spin Column Cleanup of Proteolytic Digests</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>Before performing TMT labeling, components of the lysis and digestion buffer need to be removed using solid phase extraction. To concentrate and clean up the samples a C<sub>18</sub> SepPak 130 mg bed volume is used for each sample in conjunction with a vacuum manifold set at -2 psi, to slowly pass each solution below through the solid phase extraction device.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> C<sub>18</sub> MacroSpin column (Nest Group) in conjunction with a centrifuge can be used to replace the C<sub>18</sub> SepPak column and vacuum.</div>
<ol start="6">
<li>Centrifuge the samples briefly to collect any condensate liquids on the lid.</li>
<li>To prepare the samples for desalting, acidify each digested sample with trifluoroacetic acid (TFA) to a concentration of 0.4 % (v/v) and ensure that the pH is ≤2.</li>
<li>Wash the columns with 1.25 ml of acetonitrile and condition them with 450 μl of 50 % acetonitrile / 0.5 % acetic acid. For all the steps, make sure that the entire solution passes through the device, but without drying the column.</li>
<li>Equilibrate the columns with 1.25 ml of 0.1 % TFA. Discard the flow through.</li>
<li>Load the samples onto the columns. Save flow-through.</li>
<li>Wash the columns with 125 μl of 0.5% acetic acid. Combine the flow through with the flow through from Step 5, retain the SepPak and store at -20 °C, in case of loss of sample due to unforeseen technical or user error.</li>
<li>Elute the peptides from the columns by adding 750 μl of 50 % acetonitrile with 0.5 % acetic acid.</li>
<li>Dry the eluted peptides by vacuum centrifugation (∼2 h, ambient). Store the dried samples at -80 °C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dried peptides can be stored at -80 °C indefinitely.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to store the flow through fractions until peptide recovery has been confirmed in subsequent steps.</div>
<h3>TMT10plex Mass Tag Labelling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>The TMT10plex reagent set contains ten different isobaric compounds with the same nominal mass and chemical structure. A mass reporter is linked via a spacer arm to an amine-reactive NHS-ester group. By using tandem mass spectrometry, TMT10plex allows the identification and relative quantification of ten different samples simultaneously. Two sets of TMT10plex (0.8 mg per tag) are required to label the 20 samples (i.e. 10 controls and 10 phosphatase mutants) as shown in <a href="#fig2">Figure 2</a>. The protocol follows the steps of the manufacturer but is adapted to label 200 μg of peptide material per channel.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/66-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Labeling with TMT10plex Isobaric Mass Tag Reagents Sets</div>
</figcaption>
</figure>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The TMT kit is also available in 6-, 11- or 16- plexing capacity. Alternative isobaric labeling reagents include iTRAQ.</div>
<ol start="14">
<li>Solubilize each dried sample in 200 μl of 50 mM HEPES pH 8.5 by sonication in an ultrasonic bath for 10 min. Check that the pH is ∼8.5 by spotting 0.2 μl on a pH indicator strip. If necessary adjust the pH by adding 1 M HEPES pH 8.5, and add equivalent volumes of 50 mM HEPES to the other samples to ensure that the peptide concentration is equal.</li>
<li>Equilibrate the TMT10plex Isobaric Label Reagent at ∼22°C for 15 min and briefly centrifuge the reagent tubes.</li>
<li>Solubilize each vial of isobaric labels in 40 μl anhydrous acetonitrile carefully and keep the tubes closed as much as possible. Briefly centrifuge and use the TMT reagents immediately.</li>
<li>Add the sample solution to the reagent tubes according to the scheme in <a href="#fig2">Figure 2</a>. Vortex and briefly centrifuge before incubating for 1 h at ∼22 °C on a Thermomixer set at 750 rpm. If Thermomixer is not available, vortex the samples occasionally during the 1 h incorporation</li>
<li>Centrifuge the tubes briefly and retrieve 2 μl of each of the 20 samples and dilute in 18 μl of 0.1% TFA (i.e. 20 individual diluted samples). Use these samples to determine the labeling efficiency as described in section 4.</li>
<li>Make two groups of 10 samples following the scheme in <a href="#fig2">Figure 2</a>. Then, combine 5 μl each of the 10 samples in both sets, resulting in two mixed samples. Vortex and centrifuge briefly. Dilute 2 μl of each the two mixes in 18 μl of 0.1% TFA and use these samples in the mixing check as described in section 4. Store the remainder of each labelled sample immediately at -80 °C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To allow a relative comparison of the phosphoproteome in the wild type and mutant context, it is essential that one half of the TMT10plex set labels the control samples and the other half labels the phosphatase mutant samples.</div>
<div class="note">
<span class="note-title">Note:</span> Every TMT lot has a different correction error, which can be found in the "reporter ion isotopic distribution" section of the product data sheet. It is important to keep a note of the correction error for the later data analysis.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not quench the reaction until labeling has been confirmed by LC-MS/MS.</div>
<h3>Evaluation of TMT10plex Isobaric Mass Tag Labeling Efficiency and Sample Mixing by LC-MS/MS</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 h</div>
<p>The following steps are illustrative of our specific LC-MS/MS protocol and instrument; however, any mass spectrometer with beam-type CID and MS2 resolution ≥35,000 coupled to liquid chromatography and any database search algorithm capable of processing TMT labeled data should be suitable.</p>
<ol start="20">
<li>Transfer the 20 individual diluted samples and the two mixed samples to total recovery glass autosampler vials to check the enrichment efficiency. The liquid needs to be in the bottom of the vial and must not contain any air bubbles.</li>
</ol>
<p>Inject 10 μl of each diluted TMT labelled sample or mixture on an UltiMate 3000 HPLC system coupled to an Orbitrap Fusion<sup>TM</sup> Lumos<sup>TM</sup> Tribrid<sup>TM</sup> Mass Spectrometer. Set the nanoLC flow rate to 250 nl/min. Separate the peptides using the following gradient: 0-5 min (2 % mobile phase B), 33 min (40 % mobile phase B), 34-44 min (95 % mobile phase B), 45-60 min (2 % mobile phase B). (Phase A: 95 % H<sub>2</sub>O, 5 % DMSO, 0.1 % Formic Acid, Phase B: 75 % Acetonitrile, 5 % DMSO, 20 % H<sub>2</sub>O, 0.1 % Formic Acid).</p>
<ol start="21">
<li>Use the following instrument method for the mixtures. MS settings: Orbitrap mass analyser (120000 resolution), scan range 400-1400 <i>m</i>/<i>z</i>, AGC target 4.0×10<sup>5</sup>, maximum injection time 100 ms. Data dependent mode cycle time of 3 s between MS scans. MS2 settings: Orbitrap mass analyser 60000), activation type HCD, HCD collision energy 38 %, isolation window 0.7 <i>m</i>/<i>z</i>, first mass 100<i>m</i>/<i>z</i>, AGC target 1.0×10<sup>5</sup>, maximum injection time 105 ms, include charge states 2-6, dynamic exclusion 30 s after 1 time.</li>
<li>A more detailed discussion regarding data acquisition strategies for phospho-TMT experiments is provided by <a href="#bib4"> Jiang et al, 2017</a>.</li>
<li>Perform a MaxQuant database search using an up to date <i>S. cerevisiae</i> proteome FASTA database (freely available from the UniProt website; <a href="http://www.uniprot.org/proteomes/">http://www.uniprot.org/proteomes/</a>) for the 20 label check samples (<a href="http://www.maxquant.org">http://www.maxquant.org</a> and guidelines to search process). Ensure that TMT10plex labels (+229.1629 Da on peptide N-termini and Lysine side chains) are selected as variable modifications.</li>
<li>To ensure that peptides have been optimally labelled, fewer than 5 % of the identified peptides within the modificationSpecificPeptides.txt output should not have a TMT10plex-Nter and/or TMT10plex-Lys variable modification. If this criterion is not reached for a specific sample, then a second round of labelling is recommended.</li>
<li>Perform a MaxQuant database search using an up to date <i>S. cerevisiae</i> proteome database for the two mixing checks. Ensure that Reporter ion MS2 with TMT10plex for quantification is selected. During the MaxQuant analysis, the correction numbers of the TMT lot need to be taken into consideration. At this stage, difference in the quantity of proteins between each sample can be observed and will be adjusted during the mixing of the original samples (see point 6).</li>
<li>To ensure that samples are correctly mixed, the ratios of the summed reporter ion intensities (peptides.txt) of each individual sample within a mixture should ideally be within a range of 0.75-1.33</li>
<li>When the criteria for mixing and labelling efficiency are satisfied, thaw the samples and quench the reaction with 16 μl of 5 % hydroxylamine (solution in 50 mM HEPES) for 15 min at ∼22 °C.</li>
<li>Acidify with 10 % formic acid to ∼pH 2 (∼40 μl).</li>
<li>Combine individually the two groups of the 10 TMT labelled samples according to <a href="#fig2">Figure 2</a>. If the samples do not have the same concentration of proteins (see point 4.6), adjust proportionally the volume of each sample before mixing.</li>
<li>Dry, or partially dry to remove the organic solvent (∼30 %), by vacuum centrifugation before Sep Pak clean up. After drying for 2 h the organic solvent should have been removed. Store the dried samples at -80 °C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dried peptides can be stored at -80 °C indefinitely.</div>
<h3>C18 SepPak Lite Cleanup of the Pooled Samples</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h + drying</div>
<p>As in Major Step ‘C<sub>18</sub> Spin Column Cleanup of Proteolytic Digests’, a C<sub>18</sub> SepPak is used for each sample in conjunction with a vacuum manifold set at -2 psi, to slowly pass each solution below through the solid phase extraction device.</p>
<ol start="9">
<li>Thaw the dried, pooled TMT samples, resuspend in 0.1 % TFA and centrifuge briefly.</li>
<li>Check that the pH is ≤2, acidify with TFA, if necessary.</li>
<li>Wash the columns with 3.2 ml of acetonitrile and condition them with 1.1 ml of 50 % acetonitrile with 0.5 % acetic acid. For the following steps, make sure that the entire solution passes through the device without drying the column.</li>
<li>Equilibrate the columns with 3.2 ml 0.1 % TFA. Discard the flow through.</li>
<li>Load the samples onto the columns. Retain the flow-through.</li>
<li>Desalt with 3.2 ml of 0.1 % TFA and wash the columns with 325 μl of 0.5% acetic acid. Combine the flow through with the flow through from Step 5 and store at -80 °C in case of technical or user error during procedure and loss of sample.</li>
<li>Elute the peptides from the column by adding 1.8 ml of 50 % acetonitrile with 0.5 % acetic acid.</li>
<li>Perform a final check of the mixed sample by LC-MS/MS. In theory, 2 mg of proteins are present in each combined sample (200 μg from the 10 original samples). Dilute 10 μl of the elution in 90 μl 0.1% TFA and perform the final check by injecting 10 μl for 60 min gradient elution on the Orbitrap Fusion Lumos Tribrid mass spectrometer with an HCD MS2 fragmentation method, as described in Section 4.</li>
<li>Dry the eluted peptides by vacuum centrifugation (∼4 h, ambient). Store the dried samples at -80 °C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dried peptides can be stored at -80 °C indefinitely.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> As in step 2, C<sub>18</sub> MacroSpin column in conjunction with a centrifuge can be used to replace the C<sub>18</sub> SepPak column and vacuum.</div>
<div class="note">
<span class="note-title">Note:</span> As in step 2, flow-through fractions of 5. and 6. are unlikely to contain peptides, they are retained in case of fault during the procedure.</div>
<div class="note">
<span class="note-title">Note:</span> Ideally the dried samples will remain frozen under vacuum centrifugation and have a yellow/white powder appearance. Sometimes, sample might have an oily appearance.</div>
<h3>Phosphopeptide Enrichment Using Sequential Enrichment of Metal Oxide Affinity Chromatography (SMOAC)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Sequential enrichment by Metal Oxide Affinity Chromatography (SMOAC) is used to increase the phosphopeptide enrichment. It combines titanium dioxide (Thermo Scientific High-Select<sup>TM</sup> TiO<sub>2</sub> phosphopeptide enrichment kit) and ferric nitrilotriacetate (Thermo Scientific High-Select<sup>TM</sup> Fe-NTA phosphopeptide enrichment kit) phosphopeptide enrichment procedures (<a href="#fig3">Figure 3</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/66-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. SMOAC Strategy—Combination of TiO<sub>2</sub> and Fe-NTA Phosphopeptide Enrichment Kits</div>
</figcaption>
</figure>
<ol start="40">
<li>Initially, enrichment is performed using the hermo Scientific High-Select<sup>TM</sup> TiO<sub>2</sub> phosphopeptide enrichment kit. Suspend the lyophilized peptide samples in 150 μl of Binding/Equilibration Buffer. Sonicate the sample tubes for 10-15 min in an ultrasonic bath until complete dissolution and centrifuge at 18000 × g for 5 min. Only soluble material should be used.</li>
<li>The manufacturer supplied protocol is followed with the exception that all unbound flow-through, wash and elution fractions are retained to be used in the next step of the protocol (<a href="#fig3">Figure 3</a>).</li>
<li>Combine the flow–through and wash fractions containing unbound peptides, this will be used during the Fe-NTA phosphopeptide enrichment step. Dry by vacuum centrifugation (∼3 h). It is normal to observe a rose-tinted jelly-like material at the end of the drying.</li>
<li>Secondary enrichment of the dried TiO<sub>2</sub> flow-through fractions is completed using Thermo Scientific High-Select<sup>TM</sup> Fe-NTA phosphopeptide enrichment kit. Solubilize the dried TiO<sub>2</sub> flow-through fractions in 200 μl of binding buffer using an ultrasonic bath for 15 minutes. The manufacturer supplied protocol is then followed.</li>
<li>Retain and combine the flow-through and wash fractions (which can be analyzed for protein level quantification, if required. Dry the combined flow through and eluate using vacuum centrifugation (2 h).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Peptides should be stored as lyophilized pellets at -80 °C. Keeping them in solution at elevated pH leads to phosphate loss from phosphopeptides.</div>
<h3>Phosphopeptide C<sub>18</sub> StageTip Concentration and Cleanup</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="45">
<li>Thaw the samples, re-suspended in 100 μl of 1 % TFA and sonicate for 15 min.</li>
<li>Prepare the Stage Tips (<a href="#bib5">Rappsilber et al., 2007</a>) by cutting and packing an Empore C<sub>18</sub> membrane into a 200 μl pipette tip.</li>
<li>Wash the Stage Tips with 100 μl of methanol and centrifuge at 2500 rpm for 2 min. Discard the flow through.</li>
<li>Equilibrate the Stage Tips with 200 μl of 1 % TFA and centrifuge at 2500 rpm for 3 min. Discard the flow through.</li>
<li>Load the sample onto a Stage Tip and centrifuge at 2500 rpm for 3 min. Retain the flow through.</li>
<li>Wash the bound peptides with 300 μl of 1 % TFA and centrifuge at 3000 rpm for 3 min. Retain the flow through.</li>
<li>Into a fresh tube elute the peptides with 50 μl of 80 % acetonitrile and 5 % TFA at 2000 rpm for 2 min.</li>
<li>Dry the peptides by vacuum centrifugation (∼30 min).</li>
<li>Combine and store the flow-through fractions in case the peptides were not retained on the C<sub>18</sub> membrane.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> C<sub>18</sub> ZipTip Pipette Tips can be used to replace the use of C<sub>18</sub> Stage Tips.</div>
<h3>Mass Spectrometry Analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="54">
<li>Re-suspend the samples in 35 μl of 1 % TFA, sonicate for 15 min and centrifuge at 14000 rpm at 4 °C for 5 min.</li>
<li>Transfer the samples to Total Recovery glass autosampler vials.</li>
<li>Perform triplicate injections (10 μl per injection) of each mixed sample. Set the nanoLC flow rate to 250 nl/min. Separate the peptides using the following gradient: 0-5 min (2 % mobile phase B), 120 min (35 % mobile phase B), 145 min (45 % mobile phase B), 150-160 min (95 % mobile phase B), 161-180 min (2 % mobile phase B). (Phase A: 95% H2O, 5% DMSO, 0.1% Formic Acid, Phase B: 75% Acetonitrile, 5% DMSO, 20% H2O, 0.1% Formic Acid).</li>
<li>MS settings: Orbitrap mass analyser (120000 resolution), scan range 350-1500 <i>m</i>/<i>z</i>, AGC target 4.0×10<sup>5</sup>, maximum injection time 50 ms. Data dependent mode cycle time of 3 s between MS scans. MS2 settings: Orbitrap mass analyser 60000), activation type HCD, HCD collision energy 38 %, isolation window 0.7 <i>m</i>/<i>z</i>, first mass 100 <i>m</i>/<i>z</i>, AGC target 1.0×10<sup>5</sup>, maximum injection time 105 ms, include charge states 2-7, dynamic exclusion 45 s after 1 time.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>Quantitative mass-spectrometry based proteomics can be performed using a range of quantitative techniques including, but not limited to, SILAC (stable isotope labeling by amino acids in cell culture) and LFQ (label free quantification). Due to its inherent ability to quantify ten samples simultaneously, ideal for a time course, we decided to employ isobaric labelling for quantification, specifically Tandem Mass Tags (TMT10plex) in this study. As ten samples are analyzed concurrently, this results in fewer, if any, missing values between individual time points compared to SILAC and LFQ where each time point is analyzed independently, as well as lower instrument resource requirements. We have previously published similar work using SILAC for quantification (<a href="#bib2">Godfrey et al., 2017</a> and <a href="#bib8">Touati et al., 2018</a>) (<i>S. cerevisiae</i>), (<a href="#bib1">Carpy et al., 2014</a> and <a href="#bib7">Swaffer et al., 2016</a>) (<i>S. pombe</i>). In budding yeast, a recent study has also taken advantage of the TMT10plex technology to study phosphoproteome variations in a time-resolved manner (<a href="#bib12">Zhang et al., 2019</a>). If all steps are followed correctly it is expected that &gt;95 % of all peptides will be TMT labelled (this is often &gt;99 %), and a phosphopeptide enrichment yield of &gt;80 % will be reached (often &gt;90 %). Optimal conditions will typically result in 5000-6000 quantified phosphosites.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and Statistical Analysis</h2>
<p>All data was analyzed using MaxQuant software which utilizes the embedded Andromeda database searching algorithm. MaxQuant is a freely available software (<a href="http://www.maxquant.org">http://www.maxquant.org</a>), is frequently updated and functions on a range of Microsoft Windows OS. A current <i>S. cerevisiae</i> proteome database can be freely downloaded from the UniProt website (<a href="http://www.uniprot.org/proteomes/">http://www.uniprot.org/proteomes/</a>). Define the two mixes as separate experiment groups in MaxQuant. Once the MaxQuant analysis has been completed, the phosphoSTY.txt output file (which contains all pertinent data for phosphosite analysis) can be imported into Perseus software for further statistical analysis and data visualization. Import the TMT reporter ion intensities as expression values into Perseus (<a href="http://www.maxquant.org/perseus/">http://www.maxquant.org/perseus/</a>). Peptides may be phosphorylated on multiple residues. Hence, select the desired multiplicity. Remove phosphosites that were identified from the decoy and contaminant databases. Log2(x) transform all reporter ion intensity values. Normalize the data by subtracting the median value from each column (sample) to correct for mixing errors. Add categorical annotation to group samples from the two multiplexed pools, add a separate category denoting the yeast strain. Filter valid values (at least ten valid values) in at least one multiplexed pool.</p>
<p>Subtract the median value for each phosphorylation site separately for each multiplexed pool or use an equivalent method. For imputation, any missing values can be replaced by the mean of the two adjacent time points (t<sub>-1</sub> and t<sub>+1</sub> values). Data smoothing can be applied by averaging each data point with the adjacent t<sub>-1</sub> and t<sub>+1</sub> values. The data from the associated study have been deposited with the ProteomeXchange Consortium via the PRIDE partner repository, Project ID: PXD012860.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The specific protocol presented will allow the quantification of two sets of 10 samples, and thus requires imputation between TMT sets, this could be partly overcome by using 16 samples and utilizing TMT-16plex.</p>
<p>No primary amine containing buffers can be used prior to TMT labelling, this can be overcome by using other digestion and cleaning techniques, e.g. SP3 (<a href="#bib3">Hughes et al., 2019</a>) or FASP (<a href="#bib11">Wiśniewski et al., 2009</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem</h3>
<p>Loss of sample/low sample intensity</p>
<h3>Potential Solution</h3>
<p>Return to collected flow-through fractions to identify where sample may have been lost, repeat steps ensuring correct buffers are used.</p>
<h3>Problem</h3>
<p>Poor TMT labelling</p>
<h3>Potential Solution</h3>
<p>If this is experienced in all samples then a primary amine-containing buffer may have been present, ensure non-primary amine-containing buffers were used, e.g. HEPES. If this is experienced in individual samples then the specific label may have become hydrolyzed, repeat labelling ensuring that buffer pH and organic:aqueous ratio is correct.</p>
<h3>Problem</h3>
<p>Poor phosphopeptide enrichment/yield</p>
<h3>Potential Solution</h3>
<p>Ensure that initial sample and all buffers used were at the correct pH. Ensure that the samples were desalted prior to phosphopeptide enrichment.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Carpy A., Krug K., Graf S., Koch A., Popic S., Hauf S., Macek B. (2014). Absolute proteome and phosphoproteome dynamics during the cell cycle of Schizosaccaromyces pombe (fission yeast). Mol. Cell. Proteom. <i>13</i>, 1925-1936</p>
<p id="bib2">Godfrey M., Touati S.A., Kataria M., Jones A.W., Snijders A.P., Uhlmann F. (2017). PP2A<sub>Cdc55</sub> phosphatase imposes ordered cell-cycle phosphorylation by opposing threonine phosphorylation. Mol. Cell <i>65</i>, 393-402</p>
<p id="bib3">Hughes C.S., Moggridge S., Müller T., Sorensen P.H., Morin G.B., Krijgsveld J. Single-pot, solid-phase-enhanced sample preparation for proteomics experiments. (2019). Nat. Protoc. <i>14</i>, 68-85</p>
<p id="bib4">Jiang X, Bomgarden R., Brown J., Drew D.L., Robitaille A.M., Viner R., Huhmer A.R. (2017). Sensitive and accurate quantification of phosphopeptides using TMT isobaric labeling technique. J. Proteome Res. <i>16</i>, 4244-4252</p>
<p id="bib5">Rappsilber J., Mann M., Ishihama Y. (2007). Protocol for micro-purification, enrichment, pre-fractionation and storage of peptides for proteomics using StageTips. Nat. Protoc. <i>2</i>, 1896-1906</p>
<p id="bib6">Rose MD., Winston F., Hieter P. (1990). Methods in Yeast Genetics: A Laboratory Course Manual. (New York: Cold Spring Harbor Laboratory Press)</p>
<p id="bib7">Swaffer M.P., Jones A.W., Flynn H.R., Snijders A.P., Nurse P. (2016). CDK substrate phosphorylation and ordering the cell cycle. Cell <i>167</i>, 1750-1761</p>
<p id="bib8">Touati, S.A., Kataria, M., Jones, A.W., Snijders, A.P., and Uhlmann, F. (2018). Phosphoproteome dynamics during mitotic exit in budding yeast. EMBO J. <i>37</i></p>
<p id="bib9">Touati S.A., Hofbauer L., Jones A.W., Snijders A.P., Kelly G., Uhlmann F. (2019). Cdc14 and PP2A phosphatases cooperate to shape phosphoproteome dynamics during mitotic exit. Cell Rep. <i>29</i>, 2105-2119</p>
<p id="bib10">Uhlmann F., Lottspeich F., Nasmyth K. (1999). Sister-chromatid separation at anaphase onset is promoted by cleavage of the cohesin subunit Scc1. Nature <i>400</i>, 37-42</p>
<p id="bib11">Wiśniewski J.R., Zougman A., Nagaraj N., Mann M. (2009). Universal sample preparation method for proteome analysis. Nat. Methods <i>6</i>, 359-362</p>
<p id="bib12">Zhang L., Winkler S., Schlottmann FP., Kohlbacher O., Elias JE., Skotheim JM., Ewald JC. (2019). Multiple layers of phospho-regulation coordinate metabolism and the cell cycle in budding yeast. Front. Cell. Dev. Biol. <i>7</i>, 338</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>This work was supported by The <a href="https://doi.org/10.13039/100010438">Francis Crick Institute</a>, which receives its core funding from <a href="https://doi.org/10.13039/501100000289">Cancer Research UK</a> (FC001198), the UK <a href="https://doi.org/10.13039/501100000265">Medical Research Council</a> (FC001198), and the <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> (FC001198). S.T. was supported by the ARC (Association de la Recherche Contre le Cancer) postdoctoral fellowship from June to September 2019.</p>
<h3>Author Contributions</h3>
<p>S.A.T. and F.U. conceived the research question; A.W.J, H.R.F., and A.P.S. developed the mass spectrometry analysis workflow. S.A.T prepared the figures. S.A.T. and A.W.J. wrote the manuscript with input from all co-authors.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>