<article>
<section>
<h1 id="header">Characterization of backbone dynamics using solution NMR spectroscopy to discern the functional plasticity of structurally analogous proteins</h1>
<p><time datetime="2021-11-05">Published: November 5, 2021</time></p>
<p>Ashish A. Kawale<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> and Björn M. Burmann<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Wallenberg Centre for Molecular and Translational Medicine, University of Gothenburg, 405 30 Gothenburg, Sweden</p>
<p id="aff2"><sup>2</sup>Department of Chemistry and Molecular Biology, University of Gothenburg, 405 30 Gothenburg, Sweden</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ashish.kawale@gu.se">ashish.kawale@gu.se</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:bjorn.marcus.burmann@gu.se">bjorn.marcus.burmann@gu.se</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100919">10.1016/j.xpro.2021.100919</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The comprehensive delineation of inherent dynamic motions embedded in proteins, which can be crucial for their functional repertoire, is often essential yet remains poorly understood in the majority of cases. In this protocol, we outline detailed descriptions of the necessary steps for employing solution NMR spectroscopy for the in-depth amino acid level understanding of backbone dynamics of proteins. We describe the application of the protocol on the structurally analogous Tudor domains with disparate functionalities as a model system.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.str.2021.06.007">Kawale and Burmann (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Backbone dynamics characterization of Tudor domains by solution NMR spectroscopy</li>
<li>Step-by-step setup and analysis scheme for <sup>15</sup>N NMR relaxation experiments</li>
<li>Practical guide for discerning dynamics of structurally analogous proteins</li>
<li>Protocol applicable for small-to-medium sized proteins and protein-ligand complexes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1138-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Proteins are intrinsically dynamic biomolecules. Although a determination of protein structures provides an excellent <i>ab-initio</i> information about the structural arrangements of proteins and thereby provides also insight into its function, it predominantly represents merely the time-averaged conformation out of several dynamic conformations adopted by the protein molecule critical for its functionality (<a href="#bib37">Schiro et al., 2020</a>). Therefore, a comprehensive depiction of biomolecular dynamics in solution is of high importance not only to complement the structural information but also for the thorough understanding of the biomolecular functional spectrum (<a href="#bib38">Sekhar and Kay, 2019</a>).</p>
<p>The protein molecule undergoes a variety of structural transitions at timescales such as the fast (ps–ns) as well as the slow (μs–s) timescale displaying physical motions ranging from bond vibrations, loop motions as well as backbone torsion angle and side-chain rotations with implications in enzyme catalysis, conformational changes, ligand binding and protein folding/unfolding etc. (<a href="#fig1">Figure 1</a>) (<a href="#bib20">Kovermann et al., 2016</a>; <a href="#bib29">Marsh and Teichmann, 2015</a>). Probing protein dynamics involves the systematic characterization of these embedded complex motions occurring at various timescales governed by the conformations accompanied by distinct energy barriers (<a href="#bib20">Kovermann et al., 2016</a>; <a href="#bib28">Markwick et al., 2008</a>). Solution NMR spectroscopy is one of the most powerful as well as most versatile techniques commonly used for the structural and dynamical characterization of biomolecules. The <sup>15</sup>N NMR relaxation experiments for the characterization of protein backbone dynamics are based on the 2D [<sup>15</sup>N,<sup>1</sup>H] heteronuclear single quantum coherence (HSQC) experiment which provides information about all <sup>15</sup>N atoms bound to <sup>1</sup>H atoms (<a href="#bib6">Bodenhausen and Ruben, 1980</a>). In the case of uniformly <sup>15</sup>N ([<i>U</i>-<sup>15</sup>N] or [<i>U</i>-<sup>13</sup>C,<sup>15</sup>N]) labelled proteins this experimental approach yields one backbone amide NH peak for each amino acid residue except proline. Thus, this experiment provides atomic resolution information for each backbone amide moiety of the protein molecule. The <sup>15</sup>N relaxation experiments described in this protocol rely on the intensity changes of the backbone amide peaks providing site-specific information crucial to deciphering timescales and amplitudes of these dynamic fluctuations with atomic precision (<a href="#bib19">Kleckner and Foster, 2011</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1138-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. NMR methods to discern protein dynamics at different timescales</div>
<p>Overview of the protein dynamics, timescale and available NMR methods to discern atomic-resolution information for the thorough understanding of the biomolecular dynamics. PRE: paramagnetic relaxation enhancement, CPMG: Carr-Purcell-Meiboom-Gill relaxation dispersion; CEST: Chemical exchange saturation transfer.</p>
</figcaption>
</figure>
<p>This protocol consists of a step-by-step setup and analysis scheme for the measurements of <sup>15</sup>N protein NMR backbone relaxation experiments using Tudor domain proteins as a model system (<a href="#fig2">Figure 2</a>). We then describe the data evaluation procedure followed by the description of the steps for the HYDRONMR (<a href="#bib12">Garcia de la Torre et al., 2000</a>) and Tensor2 (<a href="#bib11">Dosset et al., 2000</a>) analysis. Lastly, we outline the procedure for the Model-free analysis (<a href="#bib25">Lipari and Szabo, 1982a</a>, <a href="#bib26">1982b</a>) using relaxation data obtained at two magnetic field strengths quintessential to draw meaningful conclusions about the dynamical aspects of the protein functions by using the approach developed by d’Auvergne and Gooley applicable <i>via</i> the program relaxGUI (<a href="#bib4">Bieri et al., 2011</a>). The main advantage of the outlined protocol is that the data can be collected in aqueous buffers under near-physiological conditions. Although the protocol presented is used for well-folded proteins around ∼8 kDa in size, we also outline experimental adaptations so that this protocol can serve as a template to obtain and interpret relaxation data to discern relaxation properties of the protein NH groups also for medium-sized (up to 25–30 kDa) proteins as well as protein-ligand complexes.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1138-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic representation of the steps involved in the backbone dynamics characterization</div>
<p>(A) Schematic diagram for the interpretation of hetNOE spectra.</p>
<p>(B) T<sub>1</sub> (inversion recovery) experiment pulse sequence diagram, representative spectra at each delay point and the graph of the peak intensity versus the relaxation delay for each residue.</p>
<p>(C) T<sub>1ρ</sub> experiment pulse sequence diagram, representative spectral changes observed at each delay point and the graph of the peak intensity versus the relaxation delay observed for each residue.</p>
<p>(D) Validation using HYDRONMR computed R1, R2 rates.</p>
<p>(E) Model-free anaysis step to provide physically meaningful dynamic motional analysis from the experimental relaxation data.</p>
</figcaption>
</figure>
<p>The protocol requires the following things to be set up before beginning. The individual steps are summarized to provide the necessary details for protein production. For a more detailed general descriptions about protein expression refer to the Sambrook protocol (<a href="#bib36">Sambrook and Russell, 2001</a>).</p>
<h3 id="sec1.1">NMR sample preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 weeks</div>
<ol>
<li>Obtain the following constructs for recombinant production of SMN, NusG and RfaH Tudor domain proteins from <i>E. coli.</i>
<ol type="a">
<li>The SMN Tudor domain construct (82–147), cloned into a pET24d vector with an amino-terminal hexahistidine-GST tag followed by the TEV protease cleavage site (<a href="#bib40">Sprangers et al., 2003b</a>) (a kind gift by M. Sattler, TU Munich).</li>
<li>The NusG Tudor construct (123–181) cloned into a pET28b(+) vector containing an amino-terminal hexahistidine tagged Sumo protein solubility tag (<a href="#bib17">Kawale and Burmann, 2021</a>)(available upon request from the Burmann Lab, Gothenburg).</li>
<li>The RfaH Tudor construct (101–162) cloned into a pET28b(+) vector backbone harboring an amino-terminal hexahistidine tagged Sumo fusion protein (<a href="#bib7">Burmann et al., 2012</a>) (purchased from GenScript; available upon request from the Burmann Lab, Gothenburg).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Inconsistent design of domain boundaries can lead to disparities between protein stability and structure and can cause a mismatch with published NMR data. Perform DNA sequencing to confirm the DNA sequence.</div>
<ol start="2">
<li>Transformation of Tudor domain constructs in chemically competent bacteria.
<ol type="a">
<li>Take chemical competent <i>E. coli</i> BL21 Star<sup>TM</sup> (λDE3) cells out of −80°C and thaw on ice (∼ 30 min).</li>
<li>Take LB agar plates containing Kanamycin from the fridge and incubate at 37°C in an incubator.</li>
<li>Pipette 1–2 μL of plasmid (∼50 ng/μL) into 25–50 μL competent cells and mix gently.</li>
<li>Incubate the cells on ice for 20–30 min.</li>
<li>Heat shock at 42°C for 45 s on a heating block.</li>
<li>Incubate the cells on ice for 2 min.</li>
<li>Add 400 μL of LB medium and incubate at 37°C for 1 h on a shaking incubator.</li>
<li>Centrifuge the cells at 3400 <b>×</b> <i>g</i> for 2 min and remove the 400 μL supernatant solution.</li>
<li>Gently mix the cells and streak out evenly on the LB agar plates from the incubator.</li>
<li>Incubate plates for 16–20 h at 37°C.</li>
<li>Seal the agar plates with parafilm and transfer to the fridge (4°C) to prevent the growth of satellite colonies.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Agar plates can be kept at the fridge (4°C) for about 2 weeks before proceeding to the next step, but for the best results proceed to the next step within a week.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Maintain a sterile environment while working with bacterial cultures. Clean all the equipment with 70% Ethanol and work under a flame or Laminar airflow to prevent contamination.</div>
<ol start="3">
<li>Expression of [<i>U</i>-<sup>15</sup>N]-labelled Tudor domain proteins.
<ol type="a">
<li>Pick a single colony from the transformed plate by using a sterile loop to inoculate the 5 mL LB medium containing Kanamycin (50 μg/mL final concentration) and incubate at 37°C, 200 rpm for over-day i.e., 6–8 h.</li>
<li>Inoculate 50 mL of M9 medium in 250 mL Erlenmeyer flask with 1 mL of over-day grown LB preculture and incubate at 37°C, 200 rpm for 16–20 h.</li>
<li>Transfer the M9 preculture to the main M9 culture (1 L) in a 5 L Erlenmeyer flask and grow at 37°C, 200 rpm. Monitor the optical density (OD<sub>600 nm</sub>) using UV/Vis-spectrophotometer until it reaches 0.6–0.8.</li>
<li>Add 0.5 mM Isopropyl β-D-thiogalactoside (IPTG) for the induction of the RfaH Tudor domain and 1 mM IPTG for the induction of SMN and RfaH Tudor protein.</li>
<li>Incubate cells at 25°C, 200 rpm for 16–20 h for NusG and RfaH Tudor domain constructs whereas for the SMN Tudor domain culture incubate cells for 4 h.</li>
<li>Harvest bacterial cultures by centrifugation at 4000 <b>×</b> <i>g</i> for 20 min at 4°C.</li>
<li>Discard the supernatant and retain the cell pellets.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store cell pellets at −80°C until further use. Pellets can be stored at −80°C for up to 1–2 months. Prior to proceeding to the next step, thaw them at 20°C–22°C and later keep on them ice.</div>
<ol start="4">
<li>Purification of [<i>U</i>-<sup>15</sup>N]-labelled Tudor domain proteins.
<ol type="a">
<li>Add one cOmplete, EDTA-free Protease Inhibitor Cocktail tablet (Roche), 5 μL HL-SAN DNase I (ArticZymes) and 10 mM MgSO<sub>4</sub> to the lysis buffer (50 mL buffer per 10 g of wet cell-pellet weight).</li>
<li>Use the above-mentioned buffer solution to resuspend the resultant cell pellets harvested after centrifugation by gently pipetting up and down until a homogenous solution is achieved.</li>
<li>Turn on the Emulsiflex C3 (Avestin) homogenizer and keep it at 4°C.</li>
<li>Pass the resuspended cell mass at least three times through an Emulsiflex C3 (Avestin) homogenizer at 4°C to achieve a clear cell lysate indicative of near-complete cell lysis.</li>
<li>Centrifuge at 19000 <b>×</b> <i>g</i> for 45 min at 4°C to separate the cell debris from the lysate.</li>
<li>Transfer the clear supernatant into 50 mL tubes and keep it on ice and discard the pellet containing debris.</li>
<li>Equilibrate a manually packed Ni<sup>2+</sup>–NTA (HisPur<sup>TM</sup> resin, Thermo Fisher Scientific) gravity flow column with lysis buffer and load the clear supernatant at least twice onto it at 20°C–22°C. Collect the flow-through for SDS-PAGE analysis.</li>
<li>Wash the column by passing 10 column volumes (CV) of lysis buffer and collect the flow-through for SDS-PAGE analysis.</li>
<li>Perform an additional washing step by passing lysis buffer supplemented with 25 mM Imidazole to remove non-specifically bound proteins.</li>
<li>Pass 5 CV elution buffer for the elution of bound proteins and collect flow-through in fresh 50 mL conical tubes and place them on ice.</li>
<li>Confirm the presence and degree of purification of the desired proteins by running an SDS-PAGE.</li>
<li>Add human Sumo protease/TEV protease (His-tagged SenP1: Addgene #16356 (<a href="#bib44">Mikolajczyk et al., 2007</a>); His-tagged TEV-protease: Addgene #8827 (<a href="#bib15">Kapust et al., 2001</a>)) and put the proteins for dialysis for 16–20 h in a dialysis buffer for the subsequent Sumo-tag/GST-tag cleavage.</li>
<li>Confirm the Sumo-tag/GST-tag cleavage by running an SDS-PAGE.</li>
<li>If the cleavage is not complete, add additional protease and incubate at RT for 2–3 h.</li>
<li>Perform a second Ni<sup>2+</sup>–NTA gravity column purification step to remove the cleaved Sumo-tag/GST- tag or SenP1/TEV protease.</li>
<li>Retain the flow-through fractions containing the protein of interest for subsequent purification, keep on ice.</li>
<li>Perform further washing and elution steps with the respective buffers for the SDS-PAGE analysis together with the collected flow-through fractions.</li>
<li>Run SDS-PAGE to confirm the purification process and the presence of desired protein band.</li>
<li>Combine all the fractions containing the protein of interest and concentrate up to 1–2 mL using Vivaspin 15R centrifugal concentrators (Sartorius) with 3 kDa MWCO (Molecular weight cut-off) at 4°C.</li>
<li>Equilibrate a HiLoad 16/600 Superdex75 prep grade column (GE Healthcare) with size-exclusion buffer on an ÄKTA FPLC (GE Healthcare) at 4°C for the size exclusion chromatography and subsequently apply the concentrated protein fractions for the final purification step.</li>
<li>Monitor UV absorbance to identify the protein of interest and confirm the presence and purity by SDS-PAGE.</li>
<li>Combine all the fractions containing the protein of interest and buffer exchange to the respective NMR buffers using Vivaspin 15R centrifugal concentrators (Sartorius) with 3 kDa MWCO (Molecular weight cut-off).</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Flash-freeze protein aliquots with the help of liquid nitrogen, and store at −80°C until further use for several months. Before proceeding to the next step, thaw the proteins at 20°C–22°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Always store buffers in the fridge or the cold room. It is recommended to use fresh Ni<sup>2+</sup>–NTA resin for each protein purification to avoid contamination. Use clean chromatographic columns and regularly clean all parts of the ÄKTA system for obtaining the best results. Be careful not to pass air into the size exclusion column. Monitor the DNA contamination by checking the ratio of the absorbance at 260 and 280 nm (ratio of 0.5–0.6 reflects pure protein) using a nanodrop or alternate a UV/VIS spectrophotometer.</div>
<h3 id="sec1.2">Sample quality check and BMRB assignment transfer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 day<b>s</b>
</div>
<ol start="5">
<li>Quality check by 1D (&amp;/or 2D) NMR spectroscopy
<ol type="a">
<li>Record <sup>1</sup>H 1D NMR spectrum for each Tudor domain in the respective buffer condition at increasing sample concentrations (typically in the range of 0.1–0.5 mM).</li>
<li>Process and analyze the 1D spectra using TopSpin software.</li>
<li>Monitor the consistency in the protein amide region of the <sup>1</sup>H 1D spectra to find out if concentration-dependent aggregation effects occur.</li>
<li>Select the highest sample concentration (≤ 0.5 mM) for the backbone data measurement, where the protein <sup>1</sup>H 1D spectra is consistent with the lower protein concentrations.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 2D NMR spectroscopy can also be applied to perform the above-mentioned procedure, though it is more time-consuming than 1D NMR spectroscopy.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Always use clean and dry high-quality sample tubes. Wipe the sample tube using clean tissue paper. Use the correct spinner type, which can hold the sample tightly. Adjust the sample depth using a sample depth gauge and turn on lift air prior to inserting the sample into the NMR instrument. Ensure that the NMR spectrometer is set to the correct temperature before inserting the sample.</div>
<ol start="6">
<li>BMRB assignment transfer
<ol type="a">
<li>Record [<sup>15</sup>N,<sup>1</sup>H] heteronuclear single quantum coherence (HSQC) spectra for each Tudor domain in the respective buffer conditions.</li>
<li>Use TopSpin4 (Bruker BioSpin) to process and analyze the spectra.</li>
<li>Download the sequence-specific resonance assignment for each Tudor domain from the Biological Magnetic Resonance Data Bank website (<a href="https://bmrb.io">https://bmrb.io</a>) in the NMR_STAR v3 file format. The respective BMRB Entry codes are 15490 (NusG Tudor), 17615 (RfaH Tudor), 18005 (SMN-Tudor) (<a href="#bib7">Burmann et al., 2012</a>; <a href="#bib16">Kawale and Burmann, 2020</a>; <a href="#bib33">Mooney et al., 2009</a>; <a href="#bib42">Tripsianes et al., 2011</a>).</li>
<li>Open the spectrum in NMRFAM-sparky (<a href="#bib23">Lee et al., 2015</a>). More information regarding sparky commands and usage of the program can be found under (<a href="https://nmrfam.wisc.edu/nmrfam-sparky-distribution">https://nmrfam.wisc.edu/nmrfam-sparky-distribution</a>) and (<a href="https://www.cgl.ucsf.edu/home/sparky/manual/indx.html">https://www.cgl.ucsf.edu/home/sparky/manual/indx.html</a>).</li>
<li>From the NMRFAM-SPARKY <i>Menubar</i> go to the NMRFAM, click on <i>Utilities</i>, then to the <i>NMRSTAR 3.1 to SPARKY</i> option or alternatively type <b><i>ns</i></b> in the NMRFAM-sparky command line.</li>
<li>Select the downloaded NMRstar file and type the file name to convert it to the required sparky resonance format (.list)</li>
<li>Type <b><i>rl</i></b> to open the resonance list and click on the <i>Load</i> option at the bottom and select the recently converted resonance .list file to load the sequence-specific resonance assignments of the protein.</li>
<li>From the main NMRFAM-SPARKY Menu bar go to the <i>NMRFAM</i> option, click on the <i>Superfast</i> assignments, then to the Transfer and simulate assignments option or alternatively type <b><i>ta</i></b> in the NMRFAM-sparky command line.</li>
<li>Select <i>N-HSQC</i> as the type of the spectrum and click on the transfer option, which will transfer the BMRB downloaded assignments onto the [<sup>15</sup>N,<sup>1</sup>H] HSQC spectrum.</li>
<li>Perform the outlined steps above for all the Tudor domains datasets.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> In case of a mismatch between the BMRB assignments and your spectrum peaks, perform chemical shift referencing by using the DSS (sodium trimethylsilylpropanesulfonate) standard. The following website pages describe a quick guide for the process.</div>
<p><a href="http://www.iiserpune.ac.in/%7Ecjeet/wordpress/2008/07/14/chemical-shift-referencing-calculator/">http://www.iiserpune.ac.in/∼cjeet/wordpress/2008/07/14/chemical-shift-referencing-calculator/</a> and</p>
<p><a href="https://lsom.uthscsa.edu/biochemistry/core-facilities/biomolecular-nmr-core/technical-resources/chemical-shift-referencing-and-temperature-calibration/">https://lsom.uthscsa.edu/biochemistry/core-facilities/biomolecular-nmr-core/technical-resources/chemical-shift-referencing-and-temperature-calibration/</a></p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>E. coli</i> BL21 Star<sup>TM</sup> (λDE3)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat. #C601003</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>VWR</td>
<td>Cat. #26936.293</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Merck</td>
<td>Cat. #P3786</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Sigma-Aldrich</td>
<td>Cat. #S9763</td>
</tr>
<tr>
<td>Glucose</td>
<td>VWR</td>
<td>Cat. #101176K</td>
</tr>
<tr>
<td>FeCl<sub>2</sub>∗4H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>Cat. #103861</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>∗2H<sub>2</sub>O</td>
<td>Merck</td>
<td>Cat. #102382</td>
</tr>
<tr>
<td>H<sub>3</sub>BO<sub>3</sub>
</td>
<td>Sigma-Aldrich</td>
<td>Cat. #B-6768</td>
</tr>
<tr>
<td>CoCl<sub>2</sub>∗6H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>Cat. #C8661</td>
</tr>
<tr>
<td>CuCl<sub>2</sub>∗2H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>Cat. #22178-3</td>
</tr>
<tr>
<td>ZnCl<sub>2</sub>
</td>
<td>VWR</td>
<td>Cat. #29157.234</td>
</tr>
<tr>
<td>Na<sub>2</sub>MoO<sub>4</sub>∗2H<sub>2</sub>O</td>
<td>ACROS ORGANICS</td>
<td>Cat. #206371000</td>
</tr>
<tr>
<td>MnCl<sub>2</sub>∗4H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>Cat. #105927</td>
</tr>
<tr>
<td>D-Biotin</td>
<td>Sigma-Aldrich</td>
<td>Cat. #B4501</td>
</tr>
<tr>
<td>Choline chloride</td>
<td>Sigma-Aldrich</td>
<td>Cat. #C7017</td>
</tr>
<tr>
<td>Folic acid</td>
<td>Sigma-Aldrich</td>
<td>Cat. #F7876</td>
</tr>
<tr>
<td>Myo-Inositiol</td>
<td>Sigma-Aldrich</td>
<td>Cat. #I5125</td>
</tr>
<tr>
<td>Nicotinamide</td>
<td>Sigma-Aldrich</td>
<td>Cat. #72340</td>
</tr>
<tr>
<td>D-pantothenic acid ½ Ca</td>
<td>Sigma-Aldrich</td>
<td>Cat. #21210</td>
</tr>
<tr>
<td>Pyridoxal-HCl</td>
<td>Sigma-Aldrich</td>
<td>Cat. #93759</td>
</tr>
<tr>
<td>Riboflavin</td>
<td>Sigma-Aldrich</td>
<td>Cat. #R4500</td>
</tr>
<tr>
<td>Thiamine HCl</td>
<td>Sigma-Aldrich</td>
<td>Cat. #T4625</td>
</tr>
<tr>
<td>KCl</td>
<td>Merck</td>
<td>Cat. #P3911</td>
</tr>
<tr>
<td>NaCl</td>
<td>VWR</td>
<td>Cat. #27800.360</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>
</td>
<td>Merck</td>
<td>Cat. #230391</td>
</tr>
<tr>
<td>Kanamycin Sulfate</td>
<td>Applichem GmbH</td>
<td>Cat. #A4789,0025</td>
</tr>
<tr>
<td>(<sup>15</sup>NH<sub>4</sub>)Cl</td>
<td>Merck</td>
<td>Cat. #299251</td>
</tr>
<tr>
<td>Isopropyl β-D-thiogalactoside (IPTG)</td>
<td>Fisher Scientific</td>
<td>Cat. #10021793</td>
</tr>
<tr>
<td>cOmplete, EDTA-free Protease Inhibitor Cocktail tablet</td>
<td>Roche</td>
<td>Cat. #5056489001</td>
</tr>
<tr>
<td>HL-SAN DNase I</td>
<td>ArticZymes</td>
<td>Cat. #70910-202</td>
</tr>
<tr>
<td>HisPur<sup>Tm</sup> resin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat. #88223</td>
</tr>
<tr>
<td>Imidazole</td>
<td>Alfa Aesar</td>
<td>Cat. #47274.0E</td>
</tr>
<tr>
<td>Tris(hydroxymethyl)aminomethane</td>
<td>VWR</td>
<td>Cat. #103157P</td>
</tr>
<tr>
<td>HEPES</td>
<td>VWR</td>
<td>Cat. #441487M</td>
</tr>
<tr>
<td>DTT</td>
<td>Sigma-Aldrich</td>
<td>Cat. #D0632</td>
</tr>
<tr>
<td>D<sub>2</sub>O</td>
<td>Merck</td>
<td>Cat. #151882</td>
</tr>
<tr>
<td>Tryptone</td>
<td>VWR</td>
<td>Cat. # 97063-388</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>VWR</td>
<td>Cat. #97064-370</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>SMN Tudor</td>
<td>(<a href="#bib39">Sprangers et al., 2003a</a>)</td>
<td>BMRB: 18005</td>
</tr>
<tr>
<td>NusG Tudor</td>
<td>(<a href="#bib33">Mooney et al., 2009</a>)</td>
<td>BMRB: 15490</td>
</tr>
<tr>
<td>RfaH Tudor</td>
<td>(<a href="#bib7">Burmann et al., 2012</a>)</td>
<td>BMRB: 17615</td>
</tr>
<tr>
<td>SMN Tudor</td>
<td>(<a href="#bib39">Sprangers et al., 2003a</a>)</td>
<td>PDB: 1MHN</td>
</tr>
<tr>
<td>NusG Tudor</td>
<td>(<a href="#bib33">Mooney et al., 2009</a>)</td>
<td>PDB: 2JVV</td>
</tr>
<tr>
<td>RfaH Tudor</td>
<td>(<a href="#bib7">Burmann et al., 2012</a>)</td>
<td>PDB: 2LCL</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Plasmid: pET24b_SMN_Tudor</td>
<td>(<a href="#bib40">Sprangers et al., 2003b</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid: pET28b_Sumo_NusG_Tudor</td>
<td>(<a href="#bib17">Kawale and Burmann, 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid: pET28b_Sumo_RfaH_Tudor</td>
<td>(<a href="#bib17">Kawale and Burmann, 2021</a>)</td>
<td>GenScript</td>
</tr>
<tr>
<td>Plasmid: His-tagged TEV-protease</td>
<td>(<a href="#bib15">Kapust et al., 2001</a>)</td>
<td>Addgene: Cat. #8827</td>
</tr>
<tr>
<td>Plasmid: His-tagged SenP1</td>
<td>(Mikolajczyk et al., 2007)</td>
<td>Addgene: Cat. #16356</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>TopSpin4.0.4</td>
<td>Bruker BioSpin</td>
<td><a href="https://www.bruker.com/en/products-and-solutions/mr/nmr-software/topspin.html">https://www.bruker.com/en/products-and-solutions/mr/nmr-software/topspin.html</a></td>
</tr>
<tr>
<td>NMRFAM-sparky</td>
<td>(<a href="#bib23">Lee et al., 2015</a>)</td>
<td><a href="https://nmrfam.wisc.edu/nmrfam-sparky-distribution">https://nmrfam.wisc.edu/nmrfam-sparky-distribution</a></td>
</tr>
<tr>
<td>relaxGUI</td>
<td>(<a href="#bib4">Bieri et al., 2011</a>)</td>
<td><a href="https://www.nmr-relax.com">https://www.nmr-relax.com</a></td>
</tr>
<tr>
<td>HYDRONMR</td>
<td>(<a href="#bib12">Garcia de la Torre et al., 2000</a>)</td>
<td><a href="http://leonardo.inf.um.es/macromol/programs/hydronmr/hydronmr.htm">http://leonardo.inf.um.es/macromol/programs/hydronmr/hydronmr.htm</a></td>
</tr>
<tr>
<td>Tensor2</td>
<td>(<a href="#bib11">Dosset et al., 2000</a>)</td>
<td><a href="https://www.ibs.fr/research/scientific-output/software/tensor/?lang=fr">https://www.ibs.fr/research/scientific-output/software/tensor/?lang=fr</a></td>
</tr>
<tr>
<td>NMRBox</td>
<td>(<a href="#bib27">Maciejewski et al., 2017</a>)</td>
<td><a href="https://nmrbox.org">https://nmrbox.org</a></td>
</tr>
<tr>
<td>Microsoft Excel (or similar spreadsheet application for the calculations)</td>
<td>Microsoft</td>
<td><a href="https://www.microsoft.com/en-us/microsoft-365/excel">https://www.microsoft.com/en-us/microsoft-365/excel</a></td>
</tr>
<tr>
<td>GraphPrism (or similar data analysis software)</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td>NMRPipe/NMRDraw (optional)</td>
<td>(<a href="#bib10">Delaglio et al., 1995</a>)</td>
<td><a href="https://www.ibbr.umd.edu/nmrpipe/index.html">https://www.ibbr.umd.edu/nmrpipe/index.html</a></td>
</tr>
<tr>
<td>PyMol (optional)</td>
<td>Schrödinger</td>
<td><a href="https://pymol.org/2/">https://pymol.org/2/</a></td>
</tr>
<tr>
<td>FAST ModelFree (optional)</td>
<td>(<a href="#bib9">Cole and Loria, 2003</a>)</td>
<td><a href="http://ursula.chem.yale.edu/%7Elorialab/software.php">http://ursula.chem.yale.edu/∼lorialab/software.php</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Vivaspin 3 (MWCO) centrifugal concentrators</td>
<td>Sartorius</td>
<td>Cat. #VS0692</td>
</tr>
<tr>
<td>HiLoad 16/600 Superdex75 prep grade column</td>
<td>GE Healthcare</td>
<td>Cat. #28989333</td>
</tr>
<tr>
<td>5 mm NMR tubes</td>
<td>ARMAR AG, Switzerland</td>
<td>Cat. #032100.5045</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>The protocol requires two high magnetic field strength NMR instruments (e.g., Bruker BioSpin; 500–1,200 MHz <sup>1</sup>H frequency) running respective data acquisition software (e.g., TopSpin, Bruker BioSpin).</p>
<div class="critical">
<span class="critical-title">Critical:</span> Depending on the availability of high-field NMR spectrometers, a larger spread of the magnetic field strength is advantageous, e.g. 500 and 900 MHz.</div>
<div class="optional">
<span class="optional-title">Optional:</span> NMR instruments equipped with cryogenic probes (e.g. Prodigy- or TCI-probes, Bruker BioSpin) for increased sensitivity.</div>
<div class="note">
<span class="note-title">Note:</span> This protocol makes use of the NMR instruments, accessories, and associated software from Bruker BioSpin, but it can be easily adapted to any other NMR platforms (e.g. Jeol or Agilent/Varian).</div>
<p>As temperature affects the kinetic properties of the protein samples; constant temperature needs to be set during all the experiments and continuous checking ensures stability of the analysis of the dynamic properties.</p>
<p>High-resolution structures and backbone NMR assignments of the proteins under study are required. It is essential to use uniformly <sup>15</sup>N ([<i>U</i>-<sup>15</sup>N] or [<i>U</i>-<sup>13</sup>C,<sup>15</sup>N]) labelled protein samples (Tudor domain) with a &gt;95% purity in the respective buffer solutions for the NMR data measurements. In addition, 10% D<sub>2</sub>O needs to be supplemented to the samples for the NMR lock signal. The protein sample concentrations of ≥ 0.1 mM with sample volumes ranging between 180–500 μL are required depending on the type of NMR tube used.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Choose an appropriate protein concentration. For Tudor domains, no aggregation/oligomerization was observed for samples ∼500 μM, whereas for larger proteins often the concentration has to be reduced to 100–200 μM to avoid these effects (See <a href="#before-you-begin">before you begin</a>, 5. Quality check by 1D (&amp;/or 2D) NMR spectroscopy).</div>
<p>The media and buffer recipes used in this protocol are listed below.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Read the Material Safety Data Sheet (MSDS), pay attention to the safety concerns and handle any hazardous chemicals with care while preparing the following recipes.</div>
<table id="undtbl1">
<caption>LB media (1 L)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tryptone</td>
<td>140.7 mM</td>
<td>10 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>171.1 mM</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast Extract</td>
<td>15.7 mM</td>
<td>5 g</td>
</tr>
<tr>
<td>Water</td>
<td>n/a</td>
<td>950 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 1 L LB media, weigh and mix the above reagents in 950 mL Milli-Q water. Mix the reagents until the solutes have dissolved completely. Adjust final volume to 1 L using Milli-Q water. Autoclave to sterilize the final medium. The media can be stored at 25°C for up to 1 week.</p>
<table id="undtbl2">
<caption>M9 minimal media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>33.7 mM</td>
<td>6 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>22 mM</td>
<td>3 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>8.55 mM</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Solution Q∗</td>
<td>n/a</td>
<td>1mL</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> powder</td>
<td>2 mM</td>
<td>240.7 mg</td>
</tr>
<tr>
<td>Vitamins mix∗∗</td>
<td>n/a</td>
<td>15 mL</td>
</tr>
<tr>
<td>Glucose</td>
<td>11.1 mM</td>
<td>2 g</td>
</tr>
<tr>
<td>[<sup>15</sup>N] NH<sub>4</sub>Cl</td>
<td>18..4 mM</td>
<td>1 g</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>50 μg/mL</td>
<td>50 mg/mL</td>
</tr>
<tr>
<td>Water (autoclaved)</td>
<td>n/a</td>
<td>900 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">M9 minimal media (<a href="#bib36">Sambrook and Russell, 2001</a>) is used to generate uniformly <sup>15</sup>N-labelled [<i>U</i>-<sup>15</sup>N] proteins. To prepare 1L M9 minimal media, weigh and dissolve the above reagents in 900 mL autoclaved Milli-Q water. Use Milli-Q water to adjust the final volume to 1 L. Sterilize by filtration using a 0.2 μm filter. The media can be stored at 25°C for up to 1 week.</p>
<table id="undtbl3">
<caption>∗Solution Q (1000<b>×</b> stock)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FeCl<sub>2</sub>∗4H<sub>2</sub>O</td>
<td>25.2 mM</td>
<td>5 g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>∗2H<sub>2</sub>O</td>
<td>1.3 mM</td>
<td>184 mg</td>
</tr>
<tr>
<td>H<sub>3</sub>BO<sub>3</sub>
</td>
<td>1 mM</td>
<td>64 mg</td>
</tr>
<tr>
<td>CoCl<sub>2</sub>∗6H<sub>2</sub>O</td>
<td>0.1 mM</td>
<td>18 mg</td>
</tr>
<tr>
<td>CuCl<sub>2</sub>∗2H<sub>2</sub>O</td>
<td>0.02 mM</td>
<td>4 mg</td>
</tr>
<tr>
<td>ZnCl<sub>2</sub>
</td>
<td>2.5 mM</td>
<td>340 mg</td>
</tr>
<tr>
<td>Na<sub>2</sub>MoO<sub>4</sub>∗2H<sub>2</sub>O</td>
<td>2.9 mM</td>
<td>605 mg</td>
</tr>
<tr>
<td>MnCl<sub>2</sub>∗4H<sub>2</sub>O</td>
<td>0.2 mM</td>
<td>40 mg</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 1 L Solution Q, weigh and dissolve the above reagents in 900 mL autoclaved<br> Milli-Q water. Sterilize by filtering the solution by a 0.2 μm filter and store at 4°C. The solution can be stored at 4°C for up to several months.</p>
<table id="undtbl4">
<caption>∗∗Vitamins mix (100<b>×</b>)</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>D-Biotin</td>
<td>0.4 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Choline chloride</td>
<td>0.7 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Folic acid</td>
<td>0.2 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Myo-Inositiol</td>
<td>1.1 mM</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Nicotinamide</td>
<td>0.8 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td>D-pantothenic acid ½ Ca</td>
<td>0.2 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Pyridoxal-HCl</td>
<td>0.5 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Riboflavin</td>
<td>0.03 mM</td>
<td>0.01 g</td>
</tr>
<tr>
<td>Thiamine HCl</td>
<td>0.3 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>145.4 mM</td>
<td>8.5 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 1 L Vitamins mix solution, dissolve the above reagents in 700 mL Milli-Q water. Adjust pH between 6.8 to 7.2 at 25°C. Adjust final volume to 1 L using Milli-Q water. Filter using a 0.2 μm filter. Aliquot to 10 mL fractions. Store at −20°C. The solution can be stored at −20°C for up to several months.</p>
<p>Lysis/Load buffer</p>
<table id="undtbl5">
<caption>50 mM Tris pH 8.0, 500 mM NaCl, 5 mM Imidazole</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-base</td>
<td>50 mM</td>
<td>6.06 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>500 mM</td>
<td>29.22 g</td>
</tr>
<tr>
<td>Imiazole</td>
<td>5 mM</td>
<td>0.34 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 1 L Lysis/Load buffer, dissolve the above reagents in 700 mL Milli-Q water. Adjust pH of the buffer to 8.0 at 25°C. Adjust final volume to 1 L using Milli-Q water. Filter using a 0.2 μm filter. The buffer can be stored at 4°C for 1–2 weeks.</p>
<p>Elution buffer</p>
<table id="undtbl6">
<caption>50 mM Tris pH 8.0, 500 mM NaCl, 250 mM Imidazole</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-base</td>
<td>50 mM</td>
<td>6.06 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>500 mM</td>
<td>29.22 g</td>
</tr>
<tr>
<td>Imidazole</td>
<td>250 mM</td>
<td>17.02 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 1 L Elution buffer, dissolve the above reagents in 700 mL Milli-Q water. Adjust pH of the buffer to 8.0 at 25°C. Adjust final volume to 1 L using Milli-Q water. Filter using a 0.2 μm filter. The buffer can be stored at 4°C for 1–2 weeks.</p>
<p>Dialysis buffer</p>
<table id="undtbl7">
<caption>50 mM Tris pH 8.0, 150 mM NaCl, 1 mM DTT</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-base</td>
<td>50 mM</td>
<td>30.30 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>43.85 g</td>
</tr>
<tr>
<td>DTT</td>
<td>1 mM</td>
<td>0.75 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 5 L Lysis/Load buffer, dissolve the above reagents in 4000 mL Milli-Q water. Adjust pH of the buffer to 8.0 at 25°C. Adjust final volume to 5 L using Milli-Q water. The buffer can be stored at 4°C for 1–2 weeks.</p>
<p>Size-exclusion buffer</p>
<table id="undtbl8">
<caption>20 mM Tris pH 8.0, 150 mM NaCl, 1 mM DTT</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-base</td>
<td>20 mM</td>
<td>2.42 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>8.77 g</td>
</tr>
<tr>
<td>DTT</td>
<td>1 mM</td>
<td>0.15 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 1 L Size exclusion buffer, dissolve the above reagents in 700 mL Milli-Q water. Adjust pH of the buffer to 8.0 at 25°C. Adjust final volume to 1 L using Milli-Q water. Filter the buffer using a 0.2 μm filter. The buffer can be stored at 4°C for 1–2 weeks.</p>
<p>NusG/RfaH Tudor NMR Buffer</p>
<table id="undtbl9">
<caption>25 mM HEPES pH 7.5, 50 mM NaCl</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES</td>
<td>25 mM</td>
<td>0.60 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>50 mM</td>
<td>0.29 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 100 mL NusG/RfaH NMR Buffer, dissolve the above reagents in 70 mL Milli-Q water. Adjust pH of the buffer to 7.5 at 25°C. Adjust final volume to 100 mL using Milli-Q water. Filter using a 0.2 μm filter. The buffer can be stored at 4°C for 1–2 weeks.</p>
<p>SMN Tudor NMR Buffer</p>
<table id="undtbl10">
<caption>25 mM potassium phosphate pH 6.8, 20 mM NaCl, 2 mM DTT</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>11.4 mM</td>
<td>0.20 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>13.6 mM</td>
<td>0.18 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>20 mM</td>
<td>0.12 g</td>
</tr>
<tr>
<td>DTT</td>
<td>2 mM</td>
<td>0.03 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 100 mL SMN Tudor NMR Buffer, dissolve the above reagents in 70 mL Milli-Q water. Adjust pH of the buffer to 6.8 at 25°C. Adjust final volume to 100 mL using Milli-Q water. Filter using a 0.2 μm filter. The buffer can be stored at 4°C for 1–2 weeks.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>The protocol involves six major steps which are repeated on each protein structure under consideration (<a href="#fig2">Figure 2</a>).</p>
<h3 id="sec3.1">Step 1: Determination of hetNOE values at two magnetic fields</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>This step illustrates the procedure for the determination of hetNOE values by measuring <sup>15</sup>N{<sup>1</sup>H}-NOE experiment (<a href="#bib34">Renner et al., 2002</a>) (<a href="#fig2">Figure 2</a>A). This experiment incorporates through-space magnetization transfer <i>via</i> the dipolar coupling from the <sup>1</sup>H to the directly attached <sup>15</sup>N nucleus providing information regarding the motions of the individual N-H Bond vectors of the protein molecules. A considerable decrease in the NOE intensity value in comparison to the average NOE intensity is observed for the residues undergoing fast timescale motions (ps–ns timescale). The <sup>15</sup>N{<sup>1</sup>H}-NOE experiment typically employs measurements of two different spectra with (termed as <i>Saturated</i>) and without (termed as <i>Reference</i>) proton saturation in an interleaved fashion. The ratio of the intensities in the <i>Reference</i> and <i>Saturated</i> spectra corresponds to the value of the steady-state heteronuclear NOEs for each residue.</p>
<ol>
<li>Measurement on the NMR instrument with magnetic field strength B<sub>1</sub> (for e.g., 11.7 Tesla (T) with a <sup>1</sup>H-frequency of 500 MHz).
<ol type="a">
<li>Data acquisition
<ol type="i">
<li>Create a new dataset and experiment.</li>
<li>Load the Bruker pulse sequence for the 2D <sup>15</sup>N{<sup>1</sup>H}-NOE experiment (e.g., <i>hsqcnoef3gpsi</i> from the standard Bruker pulse sequence library).</li>
<li>Adjust the sample temperature on the NMR instrument to the desired value based on the stability of the sample and the quality of the spectrum.</li>
<li>Insert the sample into the magnet.</li>
<li>Perform steps for the lock, tune and match and shim.</li>
<li>Calibrate the <sup>1</sup>H pulse length (90° hard pulse) for the sample using a standard calibration experiment.</li>
<li>Calibrate the water position to determine the <sup>1</sup>H radio frequency carrier position.</li>
<li>Calibrate the <sup>15</sup>N pulse length (90° hard pulse) for the sample.</li>
<li>Adjust the desired <sup>15</sup>N radio frequency carrier length from the previous experiments (typically in the range of 116–122) and the sweep width of the spectrum to observe all resonances. If you have no idea from previous experiments, use 119 ppm as a carrier position and 36 ppm as a sweep width.
<div class="optional">
<span class="optional-title">Optional:</span> Calibrate the <sup>13</sup>C pulse length (90° hard pulse) for the sample if using an [<i>U</i>-<sup>13</sup>C,<sup>15</sup>N] labeled sample and adjust the desired <sup>13</sup>C radio frequency carrier to 110 ppm. This approach ensures that the <sup>15</sup>N is decoupled from neighboring Cα and CO resonances.</div>
</li>
<li>Adjust the sweep widths for <sup>1</sup>H and <sup>15</sup>N and acquisition times.</li>
<li>Set 2048 × 256 complex data points.</li>
<li>Set the number of scans (<b>ns</b>) to the desired value, which is the compromise between obtained signal to noise and the availability of measurement time. In the example spectra, <b>ns</b> of 8 was used for 500 μM sample concentration. Increase the number of scans value, using the <b>ns</b> parameter for dilute samples accordingly.</li>
<li>Adjust the recycling delay, <b>d1</b>, to at least 3s. It is important to have a longer <i>d1</i> delay value to improve the accuracy of the measurement (<a href="#bib34">Renner et al., 2002</a>).</li>
<li>Check the receiver gain by typing <b>rga</b> in the topspin command line.</li>
<li>Type <b>zg</b> to start the acquisition.
<div class="critical">
<span class="critical-title">Critical:</span> Check the pulse sequence for the phase cycling value and determine the <i>ns</i> value accordingly to allow the selection of the proper coherence pathway. Typically, it is a multiple of 4, 8, or 16.</div>
</li>
</ol>
</li>
<li>Data processing
<ol type="i">
<li>After the experiment is finished, type <b>split</b> in the Topspin command line to separate the interleaved <i>Reference</i> and <i>Saturated</i> spectra.</li>
<li>Enter <b>2</b> as the rows to split and desired experiment number to store the new dataset, which will create two new separate datasets. The first will be the <i>Reference</i> and the second will be the <i>Saturated</i> experiment dataset.</li>
<li>First process the <i>Reference</i> spectrum by using standard processing parameters, phase correction values and the 2D Fourier transformation command <b>xfb.</b>
</li>
<li>Process the <i>Saturated</i> experiment using the same processing parameters as the reference spectrum. Most importantly the intensity scaling factor needs to be the same. To check the intensity scaling factor of the <i>Reference</i> spectrum, open the reference spectrum and type <b>nc_proc.</b> Note this value and open the <i>Saturated</i> experiment and type in the command line <b>xfb nc_proc x</b>, where x is the intensity scaling factor for the reference spectrum in an integer value.</li>
<li>Convert Bruker files into the .ucsf file using the <i>bruk2ucsf</i> macro by NMRFAM-SPARKY.</li>
<li>Open .ucsf files for both <i>Reference</i> and <i>Saturated</i> spectra in NMRFAM-SPARKY. Transfer resonance assignments from the BMRB (see <a href="#before-you-begin">before you begin</a>, 6. BMRB assignment transfer).</li>
<li>To save the project file, click on the <i>File</i> option from the NMRFAM-Sparky main menu bar followed by <i>Project</i> and <i>Save As…</i> and provide a suitable file name for the project.</li>
<li>Type <b>lt</b> to open the peak list of the reference spectrum. Click on the <i>options</i> and check the <i>Data height</i> box. Click <i>Apply</i> and then <i>Close.</i> Now you can see the additional column displaying data height for each peak.</li>
<li>Click <i>save</i> to save the list for the reference spectrum.</li>
<li>For the noise estimation. Type <b>st</b> in the command line and note the value for the estimated noise.</li>
<li>Perform the above-mentioned steps for the processing of the <i>Saturated</i> spectrum.</li>
</ol>
</li>
<li>Data analysis using relaxGUI
<ol type="i">
<li>Open relaxGUI. The detailed information regarding the use of relaxGUI can be found at (<a href="https://www.nmr-relax.com/manual/The_GUI.html">https://www.nmr-relax.com/manual/The_GUI.html</a>).</li>
<li>Start new <i>Steady-state NOE analysis</i> by clicking on <i>File</i>, <i>New analysis</i>, <i>Steady-state NOE.</i>
</li>
<li>Enter NMR frequency and define the Results directory path.</li>
<li>Load N and H spins by clicking on the Spin editor function by using a PDB structure file or file containing sequence data.</li>
<li>In the main relax window, click on the <i>Add</i> button under the Spectra list option to open the Peak intensity loading wizard.</li>
<li>Next step involves specifying the data file names by selecting correct file paths for both <i>Reference</i> and <i>Saturated</i> peak lists exported from the NMRFAM-Sparky followed by specifying the spectrum ID Strings (1, 2 etc.). Click <i>Apply</i> and <i>Next.</i>
</li>
<li>Choose <i>Baseplane RMSD</i> for the analysis of the peak intensity error and in the next step provide noise associated with each <i>Reference</i> and <i>Saturated</i> spectrum from NMRFAM-SPARKY as outlined in 1b. x.</li>
<li>In the next step specify the spectrum type (i.e., <i>Reference</i> or <i>Saturated</i>) to the correct spectrum ID string (1, 2 etc.) and click <i>Apply</i> and <i>Finish.</i>
</li>
<li>Click <i>execute</i> from the main relax window to start the hetNOE analysis.</li>
<li>Once the calculations are finished, open the specified results directory. The hetNOE output file will be stored as <b>noe.700.out.</b> Change the extension of the file from .out to .txt and import it into Excel to check the residue-specific hetNOE values and to plot graphs of hetNOE values against the residue number for the respective data.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Use Graphpad Prism to plot graphs.</div>
</li>
<li>Determine the average hetNOE value by taking the median and standard deviation of the hetNOE values obtained for the residues.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Measurement on the NMR instrument with magnetic field strength B<sub>2</sub> (for e.g., 21.1 T with a <sup>1</sup>H-frequency 900 MHz)
<ol type="a">
<li>Repeat all the steps outlined under 1. at the magnetic field strength B<sub>2</sub>.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Step 2: Determination of longitudinal relaxation rates (<i>R</i><sub>1</sub>) at two magnetic fields</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>Longitudinal relaxation (<i>T</i><sub>1</sub>), also called spin-lattice relaxation, is the relaxation process by which the net magnetization returns to the equilibrium as a function of time. Determination of <i>T</i><sub>1</sub> relaxation involves quantifying the recovery rate of net magnetization aligned with the applied magnetic field (B<sub>0</sub>) using time-dependent recovery of intensity (<a href="#bib18">Kay et al., 1989</a>). Longitudinal relaxation rate (<i>R</i><sub>1</sub>), the simple inverse of the <i>T</i><sub>1</sub>, is most-commonly determined by using the inversion recovery experiment (<a href="#fig2">Figure 2</a>B). This experiment involves the application of 180° pulse to perturb the magnetization from <i>z</i> to -<i>z</i>-axis, followed by a variable delay (to allow magnetization to relax to <i>z</i>), and subsequent 90 ° pulse to flip the magnetization from <i>z</i>-axis to the <i>xy</i> plane for the detection.</p>
<ol start="3">
<li>Measurement of longitudinal (<i>R</i><sub>1</sub>) relaxation on the NMR instrument with magnetic field strength B<sub>1</sub>
<ol type="a">
<li>Data acquisition
<ol type="i">
<li>Create a new dataset and experiment.</li>
<li>Load the Bruker pulse sequence for the <i>R</i><sub>1</sub> relaxation measurement (e.g., <i>hsqct1etf3gpsi</i>).</li>
<li>Perform further steps as described in 1a. iii–xii.</li>
<li>Adjust the relaxation delay (<i>d7</i>) to the desired relaxation delay for e.g., 0.4 s.</li>
<li>Type zg to start the acquisition.</li>
<li>Perform the above-mentioned steps to record experiments with effective relaxation delays such as 0.6, 0.8, 1.2, 1.6, 2.0 and 2.4 s.</li>
</ol>
</li>
<li>Data processing
<ol type="i">
<li>After all the experiments are finished, process the first spectrum (delay 0.4 s) using standard processing parameters and the 2D Fourier transformation command xfb. Note that this spectrum will have peaks with negative intensity. Determine the phase correction values and process with the 2D Fourier transformation command xfb. Note the intensity scaling factor for this spectrum by typing nc_proc in the topspin command line.</li>
<li>Process all the remaining spectra using the same processing parameters as the first spectrum. Type in the command line xfb nc_proc x, where x intensity scaling factor (integer value) for the first spectrum.</li>
<li>Before opening spectra in the NMRFAM-Sparky, convert all the Topspin processed Bruker files into the .ucsf files using the <i>bruk2ucsf</i> macro from the NMRFAM-Sparky.</li>
<li>Open NMRFAM-Sparky. Click on <i>File</i>, <i>Open</i> and load all the processed .ucsf spectra files with variable delays from the <i>R</i><sub>1</sub> relaxation experiment.</li>
<li>Transfer resonance assignments from the BMRB (see <a href="#before-you-begin">before you begin</a>, 6. BMRB assignment transfer).</li>
<li>Save the project by clicking on <i>File</i> followed by <i>Project</i> and <i>Save As…</i> and provide a suitable File name for the project.</li>
<li>Type lt to open the peak list. To display the column displaying data height for each peak, click on the <i>Options</i> and check the <i>Data height</i> box. Click <i>Apply</i> and then <i>Close.</i>
</li>
<li>Click <i>save</i> to save the list.</li>
<li>Perform the above-mentioned steps for each spectrum.</li>
</ol>
</li>
<li>Data analysis using relax GUI
<ol type="i">
<li>Open <i>relax-GUI</i> and start new <i>R</i><sub>1</sub> <i>relaxation analysis</i> by clicking on <i>File</i>, <i>New analysis</i> and <i>R</i><sub>1</sub> <i>relaxation</i> to fit the obtained data heights for each residue <i>via</i> NMRFAM-Sparky to determine the <i>R</i><sub>1</sub> relaxation rates<i>.</i>
</li>
<li>Enter <i>NMR frequency</i>, <i>Results directory path</i> and load <i>spin system</i> as outlined in the hetNOE analysis.</li>
<li>In the main relax window, click on the <i>Add</i> button under the <i>Spectra list</i> option to open the <i>Peak intensity loading wizard</i> and load the data file names by loading sequentially peak list files (associated with each relaxation delay such as 0.4, 0.6, 0.8, 1.2, 1.6, 2.0 and 2.4 i.e., in total seven) exported from the NMRFAM-Sparky followed by specifying the relevant spectrum ID Strings (1, 2, 3 etc.). Click <i>Apply</i> and <i>Next.</i>
</li>
<li>The next step is required for the error analysis. The <i>Baseplane RMSD</i> using the noise obtained from NMRFAM-Sparky for each spectrum took a very long time for the calculations as well as resulted in the program crash. To bypass this problem, set the <i>Baseplane RMSD</i> value of 10 for each spectrum and use a statistical bootstrapping scheme to determine the associated errors for the <i>R</i><sub>1</sub> relaxation rates (see following steps x and xi).</li>
<li>In the following step assign the relaxation time (in seconds) such as 0.4, 0.6, 0.8, 1.2, 1.6, 2.0 and 2.4 to the respective spectrum ID string and click <i>Apply</i> and <i>Finish.</i>
</li>
<li>Set <i>Exponential curve model</i> to the inversion recovery. It is the three-parameter fit model which employs the parameters such as <math> <mrow> <msub> <mi>R</mi> <mi>x</mi> </msub> </mrow> </math>, <i>I</i><sub>0</sub> and <i>I</i><sub>∞</sub> and the following equation
<p class="math-ml"><math> <mrow> <msub> <mi>I</mi> <mrow> <mo>(</mo> <mi>t</mi> <mo>)</mo> </mrow> </msub> <mo>=</mo> <msub> <mi>I</mi> <mi>∞</mi> </msub> <mo>−</mo> <msub> <mi>I</mi> <mn>0</mn> </msub> <mo>∗</mo> <msup> <mtext>e</mtext> <mrow> <mo>(</mo> <mo>−</mo> <msub> <mi>R</mi> <mi>x</mi> </msub> <mo>∗</mo> <mi>t</mi> <mo>)</mo> </mrow> </msup> </mrow> </math></p>
<br> This equation takes into account that the magnetization proceeds from the negative value at <i>-I</i><sub>0</sub> to finally relax as a positive <i>I</i><sub><i>∞</i></sub> value.</li>
<li>Click execute from the main relax window to start the <i>R</i><sub>1</sub> <i>relaxation analysis.</i>
</li>
<li>Once the calculations are finished, open the specified results directory. The <i>R</i><sub>1</sub> relaxation output file will be stored as <b>r1.700.out.</b> Change the extension of the file from .out to .txt and import it into Excel to check the residue-specific <i>R</i><sub>1</sub> relaxation rates and to plot residue-wise graphs of <i>R</i><sub>1</sub> relaxation rates.</li>
<li>Determine the average <i>R</i><sub>1</sub> relaxation rate by taking the median and standard deviation for the rates obtained for all the residues.</li>
<li>To employ a statistical bootstrapping scheme for determining the associated uncertainty, iterate above mentioned (i–viii) steps (at least) three times by alternatively omitting two peak lists (i.e., use five peak lists instead of seven peak lists) for determining the <i>R</i><sub>1</sub> relaxation rate.</li>
<li>Determine the standard deviation for the <i>R</i><sub>1</sub> relaxation rates from these four datasets (one original + three from bootstrapping) for each residue to calculate the associated uncertainty.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Measurement of longitudinal (<i>R</i><sub>1</sub>) relaxation on the NMR instrument with magnetic field strength B<sub>2</sub>
<ol type="a">
<li>Repeat all the steps outlined under 3.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> An alternative to the inversion recovery method is the saturation recovery method, which allows quicker <i>T</i><sub>1</sub> measurement, though inversion recovery is generally preferred for better sensitivity and its robustness with respect to the radiofrequency inhomogeneity as well pulse imperfections. For high molecular weight proteins TROSY-based experiments (<a href="#bib43">Zhu et al., 2000</a>), which offer better sensitivity than the related HSQC-based experiments, would be an ideal choice. We recommend the optimized pulse sequences from (<a href="#bib21">Lakomek et al., 2012</a>) which are implemented in the Bruker pulse sequence library (<i>trnoeetf3gpsi3d.3</i>, <i>trt1etf3gpsitc3d.3</i>, <i>trtretf3gpsitc3d.3</i>).</div>
<h3 id="sec3.3">Step 3: Determination of transverse relaxation rates (<i>R</i><sub>2</sub>) at two magnetic fields</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>Transverse relaxation (<i>T</i><sub>2</sub>), also called as spin-spin relaxation, is the relaxation process for the decay of excited net magnetization perpendicular to the applied magnetic field (B<sub>0</sub>) with respect to time (<a href="#bib18">Kay et al., 1989</a>). One of the most-commonly used procedures to determine the <i>T</i><sub>2</sub> is to measure transverse relaxation in the rotating frame i.e., <i>T</i><sub>1</sub><sub>ρ</sub>. Transverse relaxation rate in the rotating frame (<i>R</i><sub>1</sub><sub>ρ</sub>), the simple inverse of the <i>T</i><sub>1</sub><sub>ρ</sub>, is determined by measuring the decay of a signal influenced by the spin-lock conditions to generate a rotating magnetic field near to the <i>xy</i> plane perpendicular to the static magnetic field (<a href="#fig2">Figure 2</a>C) (<a href="#bib30">Massi et al., 2004</a>). In this experiment, spins are excited by the application of a 90° pulse followed by a variable spin-lock period (Δ). A series of experiments with varying spin-lock periods are recorded to quantify the <i>R</i><sub>1</sub><sub>ρ</sub> rate. The <i>R</i><sub>1</sub><sub>ρ</sub> rate is then used to derive the <i>R</i><sub>2</sub> relaxation rate.</p>
<ol start="5">
<li>Measurement of transverse (<i>R</i><sub>1</sub><sub>ρ</sub>) relaxation on the NMR instrument with magnetic field strength B<sub>1</sub>
<ol type="a">
<li>Data acquisition
<ol type="i">
<li>Create a new dataset and experiment.</li>
<li>Load the Bruker pulse sequence for the transverse (<i>R</i><sub>1</sub><sub>ρ</sub>) relaxation measurement (e.g., <i>hsqctretf3gpsi</i>).</li>
<li>Perform further steps as described in 1a. iii–xii.</li>
<li>Adjust the relaxation delay (<i>d31</i>) to the desired relaxation delay (e.g., 0 ms) to record the first dataset.</li>
<li>Start acquisition by typing zg in the command line.</li>
<li>Perform the above-mentioned steps to record experiments with effective relaxation delays such as 15, 25, 35, 45, 55, and 65 ms.</li>
</ol>
</li>
<li>Data processing
<ol type="i">
<li>Determine the phase correction values for the first spectrum (delay 0 ms) and process it using standard processing parameters and the 2D Fourier transformation command xfb.</li>
<li>Perform further data processing steps same as <i>R</i><sub>1</sub> relaxation data processing (as described in 3b. ii–ix).</li>
</ol>
</li>
<li>Data analysis using relax GUI
<ol type="i">
<li>To determine the <i>R</i><sub>1</sub><sub>ρ</sub> relaxation rates from the obtained data heights for each residue with variable delay via NMRFAM-Sparky, open relax-GUI and start new <i>R</i><sub>2</sub> relaxation analysis by clicking on File, New analysis and <i>R</i><sub>2</sub> relaxation.</li>
<li>Perform steps to load NMR frequency, Results directory path, spin system and Spectra list as outlined in the R<sub>1</sub> relaxation analysis (see 3c. ii–iv).</li>
<li>In the following step assign the relaxation time (in seconds) such as 0, 0.015, 0.025, 0.035, 0.045, 0.055 and 0.065 to the respective spectrum ID string and click Apply and Finish.</li>
<li>Set Exponential curve model to the Two parameter exponential fit model. In this model, the magnetization begins at <i>I</i><sub>0</sub> decaying towards zero. It employs parameters such as <math> <mrow> <msub> <mi>R</mi> <mi>x</mi> </msub> </mrow> </math> and <i>I</i><sub>0</sub> and uses the following equation
<p class="math-ml"><math> <mrow> <msub> <mi>I</mi> <mrow> <mo>(</mo> <mtext>t</mtext> <mo>)</mo> </mrow> </msub> <mo>=</mo> <msub> <mi>I</mi> <mn>0</mn> </msub> <mtext>∗</mtext> <msup> <mtext>e</mtext> <mrow> <mo>(</mo> <mo>−</mo> <msub> <mi>R</mi> <mi>x</mi> </msub> <mtext>∗t</mtext> <mo>)</mo> </mrow> </msup> </mrow> </math></p>
</li>
<li>Next click <i>Execute</i> button from the main relax window to start the analysis.</li>
<li>The output file for the <i>R</i><sub>1</sub><sub>ρ</sub> relaxation rates will be stored as <b>r2.700.out</b> in the results directory. Import it into i.e., Microsoft Excel after changing file extension to .txt. Subsequently analyze and plot <i>R</i><sub>1</sub><sub>ρ</sub> relaxation rates with respect to the residue numbers. Determine the associated uncertainity using statistical bootstraping scheme (see 3c. x,xi).</li>
<li>Calculate <i>R</i><sub>2</sub> relaxation rate for each residue from the <i>R</i><sub>1</sub><sub>ρ</sub> relaxation rate using the following equation
<p class="math-ml"><math> <mrow> <msub> <mi>R</mi> <mrow> <mn>1</mn> <mtext>ρ</mtext> </mrow> </msub> <mo>=</mo> <msub> <mi>R</mi> <mn>1</mn> </msub> <msup> <mtext> cos</mtext> <mn>2</mn> </msup> <mtext>θ</mtext> <mo>+</mo> <msub> <mi>R</mi> <mn>2</mn> </msub> <msup> <mtext> sin</mtext> <mn>2</mn> </msup> <mtext>θ,</mtext> </mrow> </math></p>
where <math> <mrow> <mtext>θ</mtext> <mo>=</mo> <msup> <mtext>tan</mtext> <mrow> <mo>−</mo> <mn>1</mn> </mrow> </msup> <mrow> <mo>(</mo> <msub> <mtext>ν</mtext> <mn>1</mn> </msub> <mo>/</mo> <mtext>Δν</mtext> <mo>)</mo> </mrow> </mrow> </math> and <math> <mrow> <mtext>Δν</mtext> </mrow> </math> is the offset of the rf field to the resonance (<a href="#bib30">Massi et al., 2004</a>).</li>
<li>Determine the average <i>R</i><sub>1</sub><sub>ρ</sub> and <i>R</i><sub>2</sub> relaxation rates by taking the median and standard deviation for the values obtained for all the residues.</li>
<li>Perform error propagation to determine the errors for <i>R</i><sub>2</sub> relaxation rates by using <i>R</i><sub>1</sub> and <i>R</i><sub>1ρ</sub> rates and associated errors.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Measurement of transverse (<i>R</i><sub>1</sub><sub>ρ</sub>) relaxation on the NMR instrument with magnetic field strength B<sub>2</sub>
<ol type="a">
<li>Repeat all the steps outlined under 5.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> An alternative to the <i>T</i><sub>1</sub><sub>ρ</sub> measurements is the direct measurement of <i>T</i><sub>2</sub> involving magnetization decay. However, <i>T</i><sub>2</sub> measurements are often complicated by the phase inhomogeneity and hence, generally considered unreliable. <i>T</i><sub>1</sub><sub>ρ</sub> measurements are more suitable for determining <i>R</i><sub>2</sub> relaxation rates in the low-microsecond timescale. A very good alternative to determine <i>R</i><sub>2</sub> relaxation rates higher than this time window, is to employ TRACT (TROSY for rotational correlation times)-based approach (<a href="#bib22">Lee et al., 2006</a>).</div>
<h3 id="sec3.4">Step 4: Validation using theoretical (HYDRONMR computed) relaxation rates</h3>
<p>This step is performed to crosscheck the experimentally determined <sup>15</sup>N relaxation rates against the theoretically predicted relaxation rates using HYDRONMR program (<a href="#bib12">Garcia de la Torre et al., 2000</a>). Refer to the program manual for the complete description of the program and example files: <a href="http://leonardo.inf.um.es/macromol/programs/hydronmr/hydronmr.htm">http://leonardo.inf.um.es/macromol/programs/hydronmr/hydronmr.htm</a></p>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="7">
<li>HYDRONMR Calculation
<ol type="a">
<li>Create a directory for the HYDRONMR calculation and copy the protein <i>pdb</i> file in the directory.</li>
<li>Create an HYDRONMR input file <i>hydronmr.dat.</i>
</li>
<li>Prepare an input file <i>hydronmr.dat</i> file by providing information about parameters such as the name of the molecule, name for the output file, name of the input PDB file, AER value (3.3 Å), which is the average value of the atomic element radius for most proteins which ranges in between 2.8 – 3.3 Å (<a href="#bib3">Bernado et al., 2002</a>), temperature (298 K), Magnetic fields (16.4, 18.8 T etc.) and interatomic distance for the N-H bond (1.02 Å), Chemical shift anisotropy (−173 ppm etc.).</li>
<li>The values for <i>NSIG</i>, <i>SIGMIN</i> and <i>SIGMAX</i>, <i>ETA</i>, <i>IFLAG</i>, <i>Gyromagnetic ratio</i>, <i>no. of values</i> of the magnetic field can be left unaltered.</li>
<li>Upon successful completion of the run, the output will be stored in the same directory.</li>
<li>Open the output file containing main results file with extension <i>.res</i> in a text editor.</li>
<li>Analyze the results and subsequently open it in Microsoft Excel or Graphpad Prism to plot graphs with respect to the residue numbers.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> HYDRONMR output file (.res) also contains the information about theoretically calculated diffusion properties such as translational diffusion coefficient, rotational diffusion coefficient, rotational diffusion anisotropy and mean rotational correlation time.</div>
<h3 id="sec3.5">Step 5: Determination of rotational correlation time and diffusion tensor.</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–8 h</div>
<p>The rotational correlation time (τ<sub>c</sub>) is the time taken by a protein molecule to rotate by one radian in solution thus reflecting its overall size (<a href="#bib13">Gaspari and Perczel, 2010</a>). Longitudinal (<i>T</i><sub>1</sub>) and transverse (<i>T</i><sub>2</sub>) <sup>15</sup>N relaxation times derived from the experimentally derived relaxation rates such as (<i>T</i><sub>1</sub> = 1/<i>R</i><sub>1</sub>) and (<i>T</i><sub>2</sub> = 1/<i>R</i><sub>2</sub>) are used to determine the rotational correlation time (τ<sub>c</sub>) (<a href="#bib18">Kay et al., 1989</a>; <a href="#bib35">Rossi et al., 2010</a>) by using the following empirical equation:</p>
<p class="math-ml"><math> <mrow> <msub> <mi>τ</mi> <mi>c</mi> </msub> <mo>≈</mo> <mfrac> <mn>1</mn> <mrow> <mn>4</mn> <mi>π</mi> <msub> <mi>ν</mi> <mi>N</mi> </msub> </mrow> </mfrac> <msqrt> <mrow> <mn>6</mn> <mfrac> <msub> <mi>T</mi> <mn>1</mn> </msub> <msub> <mi>T</mi> <mn>2</mn> </msub> </mfrac> <mo>−</mo> <mn>7</mn> </mrow> </msqrt> </mrow> </math></p>
<p>where ν<sub>N</sub> is the resonance frequency of <sup>15</sup>N in Hz.</p>
<p>The rotational diffusion tensor is determined using TENSOR2 (<a href="#bib11">Dosset et al., 2000</a>) program which uses the experimentally determined backbone relaxation data and protein three-dimensional structural coordinates as an input. The complete description of the program is found at</p>
<p><a href="http://rmni.iqfr.csic.es/HTML-manuals/TENSORV2_DOC/Practice.html">http://rmni.iqfr.csic.es/HTML-manuals/TENSORV2_DOC/Practice.html</a></p>
<ol start="8">
<li>Determination of rotational correlation time
<ol type="a">
<li>Open Excel sheet and import residue-wise <i>R</i><sub>1</sub> and <i>R</i><sub>2</sub> relaxation rates and associated uncertainties determined from step 2 and step 3.</li>
<li>Take the simple inverse of <i>R</i><sub>1</sub> and <i>R</i><sub>2</sub> relaxation rates to determine <i>T</i><sub>1</sub> and <i>T</i><sub>2</sub> relaxation times, respectively.</li>
<li>Calculate τ<sub>c</sub> for each residue from <i>T</i><sub>1</sub> and <i>T</i><sub>2</sub> relaxation times using the above equation for τ<sub>c</sub> as a function of the <i>T</i><sub>1</sub> and <i>T</i><sub>2</sub> relaxation times.</li>
<li>Determine the average τ<sub>c</sub> by taking the median and standard deviation for the values obtained for all the residues.</li>
<li>Perform error propagation to determine the errors for τ<sub>c</sub> by using <i>R</i><sub>1</sub> and <i>R</i><sub>2</sub> rates and associated errors.</li>
<li>Repeat the above steps to determine the τ<sub>c</sub> from the second magnetic field strength.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The TENSOR2 program can also be used to determine the rotational correlation time.</div>
<ol start="9">
<li>Determination of rotational diffusion tensor
<ol type="a">
<li>Create a directory and launch TENSOR2 program in the terminal with <i>tensor2.</i>
</li>
<li>Click on <i>File</i>, <i>Open data file</i> and load the input <sup>15</sup>N relaxation data file containing <i>R</i><sub>1</sub>, R<sub>2</sub>, hetNOE relaxation rates along with the respective uncertainty values.</li>
<li>Next, in the TENSOR2 main window, click on <i>File</i>, <i>Open PDB file</i> and load the pdb file for the protein of interest.</li>
<li>Click on <i>Visualization</i>, <i>Definition to view</i>, select or deselect the residues used for the calculations.</li>
<li>Click on <i>Setup, Spectro</i> to enter the spectrometer frequency.
<div class="optional">
<span class="optional-title">Optional:</span> Go to the lower menu and select <i>Isotropy</i>, <i>Fit</i> to determine the τ<sub>c</sub>. Click on <i>Visualization</i>, <i>R1/R2 Tc</i> to check the results.</div>
</li>
<li>From the lower menu, Select <i>Anisotropy</i> and click on <i>Fit</i> to calculate the diffusion tensor. This calculates the principal components of the diffusion tensor for axially symmetric (<i>D</i><sub>||</sub> and <i>D</i><sub>⊥</sub>) and fully anisotropic (<i>D</i><sub>xx</sub>, <i>D</i><sub>yy</sub>, <i>D</i><sub>zz</sub>) models.</li>
<li>Set <i>Nb Cycle</i> to 500 and click on <i>Start Monte-Carlo</i> to determine the associated uncertainty.</li>
<li>Text files <i>resaniso.0</i> and <i>resaniso.1</i> contain the results of fits with Monte-Carlo simulations, respectively.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> relaxGUI can be used instead of TENSOR2 to determine diffusion parameters.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.6">Step 6: Model-free analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>Model-free analysis allows to extract physically meaningful dynamic motional analyses in a quantitative form on multiple timescales from the experimental steady state NOE, <i>R</i><sub>1</sub> and <i>R</i><sub>2</sub> relaxation data using the formalism initially developed by (<a href="#bib25">Lipari and Szabo, 1982a</a>, <a href="#bib26">b</a>). The following text contains steps to perform model-free analysis using the fully automated d’Auvergne protocol in the relaxGUI mode. For the detailed description refer to the relax manual (<a href="https://www.nmr-relax.com/manual/">https://www.nmr-relax.com/manual/</a>).</p>
<ol start="10">
<li>Model-free calculation using relaxGUI
<ol type="a">
<li>Open relaxGUI and start new model-free analysis by clicking on the File, new analysis, Model-free analysis button followed by the <i>Next</i> and the <i>Start</i> buttons.</li>
<li>Start new Steady-state NOE analysis by clicking on <i>File</i>, <i>New analysis</i>, <i>Steady-state NOE.</i>
</li>
<li>First define the directory path for the results by clicking on the <i>change</i> button in front of the <i>Results directory.</i>
</li>
<li>Next define the spin systems by clicking on the <i>spin editor</i> button. Spin editor window will be launched. Load the spin system for protein backbone spins @N and @H as described above.</li>
<li>Next step is to load the <sup>15</sup>N relaxation data by clicking on the <i>Add</i> button in the <i>relaxation data</i> list.</li>
<li>Add relaxation data from the hetNOE, <i>R</i><sub>1</sub> and <i>R</i><sub>2</sub> analyses using <i>relaxation data loading wizard.</i> The steps include specifying the relaxation data identification string (for e.g., R1_700, R2_700 etc.), the relaxation data type (R1, R2 etc.), and the frequency in Hertz and the relaxation data file. Specify the file format using <i>Free format file settings</i> option. Click <i>Apply</i> after loading each data type (in total 6 data sets) and click on <i>Next</i> option to load the data.</li>
<li>Next load the metadata, which contains information regarding the measurements of <i>peak intensities</i>, <i>temperature control</i> and <i>calibration method.</i> If there was no temperature control and calibration method use, select the option for <i>no control</i> applied.</li>
<li>In the next step click on the <i>dipolar relaxation</i> option in the main relax window to define the Dipole-dipole interaction. Click on the <i>Next</i> button and confirm the preloaded average distance of 1.02 Å. Click <i>Next</i> and then the <i>Finish</i> option to finalize the setup.</li>
<li>Next, define the Chemical Shift Anisotropy by clicking the <i>OK</i> option to the preloaded an averaged CSA value of −172 ppm.</li>
<li>Further, confirm the spin isotopes selections by opening X isotope and H isotope wizards and simply clicking on the <i>OK</i> options.</li>
<li>Next, return to the main relax window and leave the settings for the local τ<sub>m</sub>, Model-free models, Grid search increments, Monte Carlo simulation number, Maximum interactions as well as Protocol mode as it is and click on the <i>Execute</i> relax option to start the model-free analysis.</li>
</ol>
</li>
<li>Data interpretation
<ol type="a">
<li>Relax output will be stored in the specified directory with separate folders for each model name.</li>
<li>Open the final folder which will contain the results files such as <i>S2.out</i>, <i>rex.out</i> and <i>te.out</i> which contains the output for <i>S</i><sup>2</sup>, <i>R</i><sub>ex</sub> and τ<sub>e</sub> respectively.</li>
<li>Change the extension of these files from .out to .txt and import it into Microsoft Excel to check the residue-specific values and plot graphs (using Micrososft Excel or Graphpad Prism) against the residue number for the respective data.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> FAST Modelfree (<a href="#bib9">Cole and Loria, 2003</a>) and TENSOR2 (<a href="#bib11">Dosset et al., 2000</a>) can be used instead of relaxGUI to perform Model-free analysis. A major caveat for TENSOR2 is that it uses data from only a single magnetic field for the model-free analysis which could lead to inaccurate findings.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol was used to characterize the functional plasticity of the bacterial Tudor domains but with small changes might also be applied to other structurally homologous protein domains for obtaining a thorough understanding of the functional variations associated with the protein backbone dynamics. The described experimental procedure to determine <sup>15</sup>N{<sup>1</sup>H}-NOE, longitudinal (<i>R</i><sub>1</sub>) and transverse (<i>R</i><sub>2</sub>) relaxation rates combined with the model-free analysis provide a useful understanding of the motions displayed by the protein NH groups on the fast to slow timescale regimes, which can be applied on the uncharacterized protein to understand its motional spectrum or can be used as a compare and contrast approach as described in this study.</p>
<p>Our comprehensive backbone relaxation data analysis revealed a drastically different relaxation behavior for the RfaH Tudor domain contrasting the relaxation properties of the SMN and NusG Tudor domains. The presence of extended motions on the fast timescale (ps–ns) was evident by the fluctuating hetNOE values (<a href="#fig3">Figure 3</a>) as well as <i>R</i><sub>1</sub> rates (<a href="#fig4">Figure 4</a>) across the RfaH Tudor domain. In contrast, we observed nearly planar hetNOE and <i>R</i><sub>1</sub> relaxation profiles for the SMN and NusG Tudor domains. Similarly, we also observed enhanced <i>R</i><sub>2</sub> relaxation rates predominantly for the loop residues of the RfaH Tudor domain in comparison to the SMN and NusG Tudor domain (<a href="#fig5">Figure 5</a>). The analysis of experimental versus HYDRONMR computed relaxation rates and rotation correlation times for each Tudor domain is presented in <a href="#fig6">Figure 6</a>.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1138-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Comparative hetNOE analysis</div>
<p>(A) Representative example of the <sup>15</sup>N{<sup>1</sup>H}–heteronuclear NOE experimental data shown by the overlay of reference and saturated spectrum for the SMN Tudor protein. Single-letter residue name and the respective residue number are used to annotate the assignments and the positions of the backbone amide resonances.</p>
<p>(B) The steady-state <sup>15</sup>N{<sup>1</sup>H}–heteronuclear NOE data at 700 and 800 MHz magnetic fields plotted against the residue numbers for the SMN, NusG and RfaH Tudor domains. Secondary structure elements are indicated by the grey background.</p>
<p>Adapted from (<a href="#bib17">Kawale and Burmann, 2021</a>) with permission from Elsevier. Error margins were obtained via the relaxGUI program using estimated root-mean-square base plane noise of the spectra.</p>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1138-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Comparison of longitudinal relaxation rates (<i>R</i><sub>1</sub>)</div>
<p>(A) Representative examples of the longitudinal relaxation (<i>R</i><sub>1</sub>) measurement data (inversion recovery method) depicted by the intensity changes.</p>
<p>(B) The associated relaxation data profiles for the residues T122 and D140 of the SMN Tudor protein at 700 MHz. Negative and positive intensity peaks are shown in orange and red, respectively.</p>
<p>(C) Residue-wise plots of the <sup>15</sup>N longitudinal relaxation rates for the Tudor domains under study. Secondary structure elements are indicated by the grey background.</p>
<p>Adapted from (<a href="#bib17">Kawale and Burmann, 2021</a>) with permission from Elsevier. Error margins for the <i>R</i><sub>1</sub> relaxation rates were determined using the statistical bootstrapping scheme.</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1138-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Comparison of transverse relaxation rates (<i>R</i><sub>2</sub>)</div>
<p>(A) Examples of the Transverse relaxation (<i>R</i><sub>2</sub>) measurement data represented by the intensity decay.</p>
<p>(B) The associated relaxation data profiles for the residues T122 and D140 of the SMN Tudor protein at 700 MHz.</p>
<p>(C) <sup>15</sup>N transverse relaxation rates plotted against residue numbers for the Tudor domains under study. Secondary structure elements are indicated by the grey background.</p>
<p>Adapted from (<a href="#bib17">Kawale and Burmann, 2021</a>) with permission from Elsevier. Error margins for the <i>R</i><sub>1ρ</sub> relaxation rates were determined using the statistical bootstrapping scheme. <i>R</i><sub>2</sub> error margins were determined using the error propagation from the <i>R</i><sub>1</sub> and <i>R</i><sub>1ρ</sub> error margins.</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1138-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Validation against HYDRONMR computed values and rotational correlation times of Tudor domains</div>
<p>(A) Comparison between the theoretical (HYDRONMR predicted) versus the experimental values of the <i>T</i><sub>1</sub>/<i>T</i><sub>2</sub> ratio.</p>
<p>(B) Backbone relaxation NMR data based rotational correlation time (τ<sub>c</sub>) plotted against the residue numbers of the Tudor domains with secondary structure elements highlighted by the grey background.</p>
<p>Adapted from (<a href="#bib17">Kawale and Burmann, 2021</a>) with permission from Elsevier. Error margins for the rotational correlation times were determined using the error propagation from the <i>R</i><sub>1</sub> and <i>R</i><sub>1ρ</sub> error margins.</p>
</figcaption>
</figure>
<p>The differential relaxation behavior of the RfaH Tudor domain was further ascertained by the Model-free analysis revealing a strong fluctuation (ranging from 0.37 to 0.91) in the generalized order parameter, (<i>S</i><sup>2</sup>), revealing the presence of extensive amplitudes of motions (<a href="#fig7">Figure 7</a>). The strong presence of τ<sub>e</sub> terms in the carboxy-terminal part of the protein reflected the motions on the ps–ns timescale whereas the presence and the stronger magnitudes of <i>R</i><sub>ex</sub> terms corroborated the presence of slow timescale motions predominantly by the loop residues. In contrast, the rigidity of the SMN and NusG Tudor proteins is reflected by the higher <i>S</i><sup>2</sup> values along with the fewer τ<sub>e</sub> and <i>R</i><sub>ex</sub> terms.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1138-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Results from the Model-free analysis</div>
<p>Residue-wise plots of (A) the generalized order parameters (<i>S</i><sup>2</sup>), (B) the calculated τ<sub>e</sub> values representing internal motions faster than the rotational correlation time, and (C) conformational exchange (<i>R</i><sub>ex</sub>) values for each Tudor domain under study. The broken lines in panel A indicate an <i>S</i><sup>2</sup> of 0.5 and 0.8 distinguishing highly flexible (<i>S</i><sup>2</sup> &lt; 0.5) and structurally rigid (<i>S</i><sup>2</sup> &gt; 0.8) regions. Adapted from (<a href="#bib17">Kawale and Burmann, 2021</a>) with permission from Elsevier. Error margins were obtained from the relaxGUI program.</p>
</figcaption>
</figure>
<p>Thus, our in-depth backbone relaxation data analysis discerns the crucial differences between the dynamic properties of the structurally similar Tudor domains corroborating the conformational plasticity and the metamorphic behavior of the RfaH Tudor.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>NMR data were analyzed using NMRFAM-sparky (<a href="#bib23">Lee et al., 2015</a>). Error margins for the hetNOE data were obtained from the spectral noise as outlined in detail above. For the <i>R</i><sub>1</sub> and <i>R</i><sub>1</sub><sub>ρ</sub> rates a statistical bootstrapping scheme was used to determine the error margin of the relaxation rates as in more detail described in the respective sections. These error margins were propagated for the determination of the margins of the <i>R</i><sub>2</sub> rates as outlined above. The used programs, relaxGUI (<a href="#bib4">Bieri et al., 2011</a>) and Tensor2 (<a href="#bib11">Dosset et al., 2000</a>), use a Monte-Carlo based scheme directly embedded in the respective software; details can be obtained from the respective software documentation as indicated above.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is applicable to a wide array of protein domains as well as protein-ligand complexes ranging from small to medium-size proteins. But there are limitations regarding the protein size. Higher molecular weight proteins display strong signal overlaps and in addition tumble more slowly in solution and thus, leads to the line broadening of the resonances and reduced signal sensitivity. Although deuteration and TROSY-based relaxation experiments (<a href="#bib21">Lakomek et al., 2012</a>; <a href="#bib43">Zhu et al., 2000</a>) could help in increasing the signal sensitivity, protein size pose limitation to the application of this protocol.</p>
<p>Unavailability of high field spectrometers with two magnetic fields and long data measurement time could also limit the application of this protocol. The symmetry requirement approximation in the Model-free approach, also limits the use of this protocol to the proteins with definite protein shapes, and hence for intrinsically disorders proteins application of this protocol is not valid and the reader is referred to more specialized approaches such as i.e., developed by the Blackledge group (<a href="#bib1">Adamski et al., 2019</a>; <a href="#bib31">Milles et al., 2018</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Sample deteriorates over time.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Use 0.02% sodium azide solution to prevent bacterial or fungal growth in the samples. When not measuring data, remove the sample from the NMR tube and flash freeze it in liquid nitrogen and subsequently store it at −80°C. If possible, use lower protein concentrations 0.1–0.2 mM for the measurement to avoid concentration dependent aggregation which might increase over time. To prevent degradation of the sample, add a few μl of protease inhibitor cocktail solution into the sample. Typically, ∼5 μL of one cOmplete, EDTA-free Protease Inhibitor Cocktail tablet (Roche) dissolved in the respective buffer is sufficient. If the sample requires DTT, add fresh DTT from the stock as DTT is prone to degradation over time. If not limited with the material, aliquot the sample in several batches and store at −80°C and use a fresh sample for each data measurement to ensure consistent data for unstable samples.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Low signal to noise for the NMR signals.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Increase the sample concentration used for the measurement unless protein is aggregation-prone or limited by the material availability otherwise increase the number of scans.</p>
<h3 id="sec7.5">Problem 3</h3>
<p><i>R</i><sub>1</sub> and <i>R</i><sub>2</sub> relaxation data curves are not ideal.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Increase the range of delays used for the data measurement to obtain ideal data curves for <i>R</i><sub>1</sub> and <i>R</i><sub>2</sub> relaxation data. A good approximation of a suitable time-range is that the signal intensity at the last measurement point is at least <math> <mrow> <mo>∼</mo> <mfrac> <msub> <mi>I</mi> <mn>0</mn> </msub> <mi>e</mi> </mfrac> </mrow> </math> to ensure good sampling of the relaxation data curve.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Calculations are computationally too demanding and time consuming.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Use NMRbox (<a href="#bib27">Maciejewski et al., 2017</a>), which provides all the required software mentioned in the protocol on cloud based virtual machines, offering faster calculations for computationally demanding steps. In addition, using NMRbox has the advantage, that the required software does not need to be installed locally and eventually needs to be compiled for the local computer used.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Overlapping peaks complicates the estimation of individual peak height or volume.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Use peak integration programs, e.g., PINT (<a href="#bib2">Ahlner et al., 2013</a>), specifically designed for this purpose. If the residues causing overlapping peaks are not crucial for the analysis (for e.g., terminal residues) exclude them from the analysis.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Limited NMR spectrometer time and/or limited sample stability preventing the full dynamical analysis required.</p>
<h3 id="sec7.12">Potential solution</h3>
<p>In recent years developments of nonuniform sampling (NUS) methods have found broad applicability in the NMR community as a time-saving method in NMR spectroscopy (<a href="#bib14">Jaravine et al., 2006</a>; <a href="#bib32">Mobli and Hoch, 2014</a>). However, quantitative analysis of such spectra has been complicated by the non-linearity of the signal intensities preventing its broad usage (<a href="#bib24">Linnet and Teilum, 2016</a>; <a href="#bib41">Stetz and Wand, 2016</a>). Nevertheless, latest developments employing accordion spectroscopy (<a href="#bib5">Bodenhausen and Ernst, 1982</a>) with nonuniform sampling providing comparable data quality as conventional approaches offers a promising route under time-limited circumstances (<a href="#bib8">Carlstrom et al., 2019</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Adamski, W., Salvi, N., Maurin, D., Magnat, J., Milles, S., Jensen, M.R., Abyzov, A., Moreau, C.J., and Blackledge, M. (2019). A unified description of intrinsically disordered protein dynamics under physiological conditions using NMR spectroscopy. J. Am. Chem. Soc. <i>141</i>, 17817-17829. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref1">View at publisher</a></p>
<p id="bib2">Ahlner, A., Carlsson, M., Jonsson, B.H., and Lundström, P. (2013). PINT: a software for integration of peak volumes and extraction of relaxation rates. J. Biomol. NMR <i>56</i>, 191-202. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref2">View at publisher</a></p>
<p id="bib3">Bernado, P., Garcia de la Torre, J., and Pons, M. (2002). Interpretation of <sup>15</sup>N NMR relaxation data of globular proteins using hydrodynamic calculations with HYDRONMR. J. Biomol. NMR <i>23</i>, 139-150. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref3">View at publisher</a></p>
<p id="bib4">Bieri, M., d'Auvergne, E.J., and Gooley, P.R. (2011). relaxGUI: a new software for fast and simple NMR relaxation data analysis and calculation of ps-ns and μs motion of proteins. J. Biomol. NMR <i>50</i>, 147-155. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref4">View at publisher</a></p>
<p id="bib5">Bodenhausen, G. and Ernst, R.R. (1982). Direct determination of rate constants of slow dynamic Processes by two-dimensional accordion spectroscopy in nuclear magnetic-resonance. J. Am. Chem. Soc. <i>104</i>, 1304-1309. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref5">View at publisher</a></p>
<p id="bib6">Bodenhausen, G. and Ruben, D.J. (1980). Natural abundance nitrogen-15 NMR by enhanced heteronuclear spectroscopy. Chem. Phys. Lett. <i>69</i>, 185-189. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref6">View at publisher</a></p>
<p id="bib7">Burmann, B.M., Knauer, S.H., Sevostyanova, A., Schweimer, K., Mooney, R.A., Landick, R., Artsimovitch, I., and Rösch, P. (2012). An α-helix to β-barrel domain switch transforms the transcription factor RfaH into a translation factor. Cell <i>150</i>, 291-303. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref7">View at publisher</a></p>
<p id="bib8">Carlström, G., Elvander, F., Sward, J., Jakobsson, A., and Akke, M. (2019). Rapid NMR relaxation measurements using optimal nonuniform sampling of multidimensional accordion data analyzed by a sparse reconstruction method. J. Phys. Chem. A <i>123</i>, 5718-5723. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref8">View at publisher</a></p>
<p id="bib9">Cole, R. and Loria, J.P. (2003). FAST-Modelfree: a program for rapid automated analysis of solution NMR spin-relaxation data. J. Biomol. NMR <i>26</i>, 203-213. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref9">View at publisher</a></p>
<p id="bib10">Delaglio, F., Grzesiek, S., Vuister, G.W., Zhu, G., Pfeifer, J., and Bax, A. (1995). NMRPipe: a multidimensional spectral processing system based on UNIX pipes. J. Biomol. NMR <i>6</i>, 277-293. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref10">View at publisher</a></p>
<p id="bib11">Dosset, P., Hus, J.C., Blackledge, M., and Marion, D. (2000). Efficient analysis of macromolecular rotational diffusion from heteronuclear relaxation data. J. Biomol. NMR <i>16</i>, 23-28. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref11">View at publisher</a></p>
<p id="bib12">Garcia de la Torre, J., Huertas, M.L., and Carrasco, B. (2000). HYDRONMR: prediction of NMR relaxation of globular proteins from atomic-level structures and hydrodynamic calculations. J. Magn. Reson. <i>147</i>, 138-146. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref12">View at publisher</a></p>
<p id="bib13">Gaspari, Z. and Perczel, A. (2010). Protein dynamics as reported by NMR. Annu. Rep. NMR Spectrosc. <i>71</i>, 35-75. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref13">View at publisher</a></p>
<p id="bib14">Jaravine, V., Ibraghimov, I., and Orekhov, V.Y. (2006). Removal of a time barrier for high-resolution multidimensional NMR spectroscopy. Nat. Methods <i>3</i>, 605-607. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref14">View at publisher</a></p>
<p id="bib15">Kapust, R.B., Tozser, J., Fox, J.D., Anderson, D.E., Cherry, S., Copeland, T.D., and Waugh, D.S. (2001). Tobacco etch virus protease: mechanism of autolysis and rational design of stable mutants with wild-type catalytic proficiency. Protein Eng. <i>14</i>, 993-1000. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref15">View at publisher</a></p>
<p id="bib16">Kawale, A.A. and Burmann, B.M. (2020). UvrD helicase-RNA polymerase interactions are governed by UvrD's carboxy-terminal Tudor domain. Commun. Biol. <i>3</i>, 607. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref16">View at publisher</a></p>
<p id="bib17">Kawale, A.A. and Burmann, B.M. (2021). Inherent backbone dynamics fine-tune the functional plasticity of Tudor domains. Structure. . <a class="external-link" href="https://doi.org/10.1016/j.str.2021.06.007">https://doi.org/10.1016/j.str.2021.06.007</a></p>
<p id="bib18">Kay, L.E., Torchia, D.A., and Bax, A. (1989). Backbone dynamics of proteins as studied by <sup>15</sup>N inverse detected heteronuclear NMR spectroscopy: application to staphylococcal nuclease. Biochemistry <i>28</i>, 8972-8979. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref18">View at publisher</a></p>
<p id="bib19">Kleckner, I.R. and Foster, M.P. (2011). An introduction to NMR-based approaches for measuring protein dynamics. Biochim. Biophys. Acta <i>1814</i>, 942-968. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref19">View at publisher</a></p>
<p id="bib20">Kovermann, M., Rogne, P., and Wolf-Watz, M. (2016). Protein dynamics and function from solution state NMR spectroscopy. Q. Rev. Biophys. <i>49</i>, e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref20">View at publisher</a></p>
<p id="bib21">Lakomek, N.A., Ying, J., and Bax, A. (2012). Measurement of <sup>15</sup>N relaxation rates in perdeuterated proteins by TROSY-based methods. J. Biomol. NMR <i>53</i>, 209-221. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref21">View at publisher</a></p>
<p id="bib22">Lee, D., Hilty, C., Wider, G., and Wüthrich, K. (2006). Effective rotational correlation times of proteins from NMR relaxation interference. J. Magn. Reson. <i>178</i>, 72-76. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref22">View at publisher</a></p>
<p id="bib23">Lee, W., Tonelli, M., and Markley, J.L. (2015). NMRFAM-SPARKY: enhanced software for biomolecular NMR spectroscopy. Bioinformatics <i>31</i>, 1325-1327. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref23">View at publisher</a></p>
<p id="bib24">Linnet, T.E. and Teilum, K. (2016). Non-uniform sampling of NMR relaxation data. J. Biomol. NMR <i>64</i>, 165-173. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref24">View at publisher</a></p>
<p id="bib25">Lipari, G. and Szabo, A. (1982a). Model-Free approach to the interpretation of nuclear magnetic-resonance relaxation in macromolecules: 1. Theory and range of validity. J. Am. Chem. Soc. <i>104</i>, 4546-4559. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref25">View at publisher</a></p>
<p id="bib26">Lipari, G. and Szabo, A. (1982b). Model-Free approach to the interpretation of nuclear magnetic-resonance relaxation in macromolecules: 2. Analysis of experimental results. J. Am. Chem. Soc. <i>104</i>, 4559-4570. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref26">View at publisher</a></p>
<p id="bib27">Maciejewski, M.W., Schuyler, A.D., Gryk, M.R., Moraru, I.I., Romero, P.R., Ulrich, E.L., Eghbalnia, H.R., Livny, M., Delaglio, F., and Hoch, J.C. (2017). NMRbox: a resource for biomolecular NMR computation. Biophys. J. <i>112</i>, 1529-1534. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref27">View at publisher</a></p>
<p id="bib28">Markwick, P.R., Malliavin, T., and Nilges, M. (2008). Structural biology by NMR: structure, dynamics, and interactions. PLoS Comput. Biol. <i>4</i>, e1000168. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref28">View at publisher</a></p>
<p id="bib29">Marsh, J.A. and Teichmann, S.A. (2015). Structure, dynamics, assembly, and evolution of protein complexes. Annu. Rev. Biochem. <i>84</i>, 551-575. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref29">View at publisher</a></p>
<p id="bib30">Massi, F., Johnson, E., Wang, C., Rance, M., and Palmer, A.G. (2004). NMR <i>R</i><sub>1</sub>ρ rotating-frame relaxation with weak radio frequency fields. J. Am. Chem. Soc. <i>126</i>, 2247-2256. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref30">View at publisher</a></p>
<p id="bib44">Mikolajczyk, J., Drag, M., Bekes, M., Cao, J.T., Ronai, Z., and Salvesen, G.S. (2007). Small ubiquitin-related modifier (SUMO)-specific proteases: profiling the specificities and activities of human SENPs. J. Biol. Chem. <i>282</i>, 26217-26224. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/opt92fwUIaLqo">View at publisher</a></p>
<p id="bib31">Milles, S., Salvi, N., Blackledge, M., and Jensen, M.R. (2018). Characterization of intrinsically disordered proteins and their dynamic complexes: from <i>in vitro</i> to cell-like environments. Prog. Nucl. Magn. Reson. Spectrosc. <i>109</i>, 79-100. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref31">View at publisher</a></p>
<p id="bib32">Mobli, M. and Hoch, J.C. (2014). Nonuniform sampling and non-Fourier signal processing methods in multidimensional NMR. Prog. Nucl. Magn. Reson. Spectrosc. <i>83</i>, 21-41. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref32">View at publisher</a></p>
<p id="bib33">Mooney, R.A., Schweimer, K., Rösch, P., Gottesman, M., and Landick, R. (2009). Two structurally independent domains of <i>E. coli</i> NusG create regulatory plasticity <i>via</i> distinct interactions with RNA polymerase and regulators. J. Mol. Biol. <i>391</i>, 341-358. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref33">View at publisher</a></p>
<p id="bib34">Renner, C., Schleicher, M., Moroder, L., and Holak, T.A. (2002). Practical aspects of the 2D <sup>15</sup>N-[<sup>1</sup>H]-NOE experiment. J. Biomol. NMR <i>23</i>, 23-33. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref34">View at publisher</a></p>
<p id="bib35">Rossi, P., Swapna, G.V.T., Huang, Y.J., Aramini, J.M., Anklin, C., Conover, K., Hamilton, K., Xiao, R., Acton, T.B., Ertekin, A., et al. (2010). A microscale protein NMR sample screening pipeline. J. Biomol. NMR <i>46</i>, 11-22. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref35">View at publisher</a></p>
<p id="bib36">Sambrook, J. and Russell, D.W. (2001). , Third Edition (Cold Spring Harbor Laboratory Press). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref36">View at publisher</a></p>
<p id="bib37">Schiro, A., Carlon, A., Parigi, G., Murshudov, G., Calderone, V., Ravera, E., and Luchinat, C. (2020). On the complementarity of X-ray and NMR data. J. Struct. Biol. X <i>4</i>, 100019. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref37">View at publisher</a></p>
<p id="bib38">Sekhar, A. and Kay, L.E. (2019). An NMR view of protein dynamics in Health and Disease. Annu. Rev. Biophys. <i>48</i>, 297-319. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref38">View at publisher</a></p>
<p id="bib39">Sprangers, R., Groves, M.R., Sinning, I., and Sattler, M. (2003a). High-resolution X-ray and NMR structures of the SMN Tudor domain: conformational variation in the binding site for symmetrically dimethylated arginine residues. J. Mol. Biol. <i>327</i>, 507-520. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref39">View at publisher</a></p>
<p id="bib40">Sprangers, R., Selenko, P., Sattler, M., Sinning, I., and Groves, M.R. (2003b). Definition of domain boundaries and crystallization of the SMN Tudor domain. Acta Crystallogr. D <i>59</i>, 366-368. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref40">View at publisher</a></p>
<p id="bib41">Stetz, M.A. and Wand, A.J. (2016). Accurate determination of rates from non-uniformly sampled relaxation data. J. Biomol. NMR <i>65</i>, 157-170. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref41">View at publisher</a></p>
<p id="bib42">Tripsianes, K., Madl, T., Machyna, M., Fessas, D., Englbrecht, C., Fischer, U., Neugebauer, K.M., and Sattler, M. (2011). Structural basis for dimethylarginine recognition by the Tudor domains of human SMN and SPF30 proteins. Nat. Struct. Mol. Biol. <i>18</i>, 1414-1420. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref42">View at publisher</a></p>
<p id="bib43">Zhu, G., Xia, Y., Nicholson, L.K., and Sze, K.H. (2000). Protein dynamics measurements by TROSY-based NMR experiments. J. Magn. Reson. <i>143</i>, 423-426. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00625-0/sref43">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Björn M. Burmann (<a href="mailto:bjorn.marcus.burmann@gu.se">bjorn.marcus.burmann@gu.se</a>).</p>
<h4>Materials availability</h4>
<p>This study did not create new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The NMR data used for the relaxation analysis have been tabulated and are available on Mendeley data: <a href="https://doi.org/10.17632/3thwpmz88s.1">https://doi.org/10.17632/3thwpmz88s.1</a></p>
<h3>Acknowledgments</h3>
<p>We thank Michael Sattler (Munich) for kindly providing the SMN Tudor domain construct. The Swedish NMR Centre of the University of Gothenburg is acknowledged for spectrometer time. B.M.B. gratefully acknowledges funding from the <a href="https://doi.org/10.13039/501100004359">Swedish Research Council</a> (Starting Grant 2016-04721; Consolidator Grant 2020-00466) and the <a href="https://doi.org/10.13039/501100004063">Knut och Alice Wallenberg Foundation</a> through a Wallenberg Academy Fellowship (2016.0163), as well as through the Wallenberg Centre for Molecular and Translational Medicine, <a href="https://doi.org/10.13039/501100005760">University of Gothenburg</a>, Sweden. This study made use of NMRbox: National Center for Biomolecular NMR Data Processing and Analysis, a Biomedical Technology Research Resource (BTRR), which is supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grant P41GM111135 (NIGMS).</p>
<h3>Author contributions</h3>
<p>The study was conceived by B.M.B. and experimentally planned together with A.A.K. All the experimental work was performed by A.A.K. Data analysis and interpretation was performed by A.A.K. and B.M.B. Both A.A.K. and B.M.B. jointly wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>