<article>
  <section>
    <h1 id="header">Protocol for assaying chromatin accessibility using ATAC-seq in plants</h1>
    <p><time datetime="2021-01-22">Published: January 22, 2021</time></p>
    <p>Fu-Xiang Wang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Guan-Dong Shang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Lian-Yu Wu,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a></sup> Yan-Xia Mai,<sup><a href="#aff1">1</a></sup> Jian Gao,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Zhou-Geng Xu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Jia-Wei
      Wang<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup></p>
    <p id="aff1"><sup>1</sup>National Key Laboratory of Plant Molecular Genetics (NKLPMG), CAS center for Excellence in
      Molecular Plant Science, Institute of Plant Physiology and Ecology (SIPPE), Chinese Academy of Sciences (CAS),
      Shanghai 200032, China</p>
    <p id="aff2"><sup>2</sup>University of Chinese Academy of Sciences (UCAS), Shanghai 200032, China</p>
    <p id="aff3"><sup>3</sup>ShanghaiTech University, Shanghai 200031, China</p>
    <p id="fn1"><sup>4</sup>Technical contact</p>
    <p id="fn2"><sup>5</sup>Lead contact</p>
    <p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:jwwang@sippe.ac.cn">jwwang@sippe.ac.cn</a></p>
    <p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100289">10.1016/j.xpro.2020.100289</a></p>
  </section>
  <section>
    <h2 id="summary">Summary</h2>
    <p>Open or accessible regions of the genome are the primary positions of binding sites for transcription factors and
      chromatin regulators. Transposase-accessible chromatin sequencing (ATAC-seq) can probe chromatin accessibility in
      the intact nucleus. Here, we describe a protocol to generate ATAC-seq libraries from fresh <i>Arabidopsis
        thaliana</i> tissues and establish an easy-to-use bioinformatic analysis pipeline. Our method could be applied
      to other plants and other tissues and allows for the reliable detection of changes in chromatin accessibility
      throughout plant growth and development.</p>
    <p>For complete details on the use and execution of this protocol, please refer to <a href="#bib15">Wang et al.
        (2020)</a>.</p>
    
    
    
    
  <div class="highlights">
<h3>Highlights</h3>
<ul>
      <li>A protocol to generate ATAC-seq libraries from fresh plant tissues</li>
      <li>An easy-to-use bioinformatic analysis pipeline for ATAC-seq</li>
      <li>Identification of differentially accessible peaks by ATAC-seq</li>
    </ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure>
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/418-GA.jpg" alt="GraphicalAbstract.jpg">
    </figure>
</div></section>
  <section>
    <h2 id="before-you-begin">Before you begin</h2>
    <h3 id="sec1.1">Experimental considerations</h3>
    <ol>
      <li>Here, we describe a detailed ATAC-seq protocol as applied to embryos and seedlings in <i>Arabidopsis
          thaliana.</i> This protocol can also be used for other plants and other tissues. Before beginning this
        protocol, please ensure that your core facility has the instruments compatible for flow sorting as the sorting
        step is extremely important for acquiring high-quality data.</li>
      <li>To ensure success, please perform pilot experiments to detect the position of nuclei with different ploidy
        levels in the peak diagram and to set appropriate gates for FACS sorting.</li>
    </ol>
  </section>
  <section>
    <h2 id="key-resources-table">Key resources table</h2>
    <table id="krt">
      <thead>
        <tr>
          <th>REAGENT or RESOURCE</th>
          <th>SOURCE</th>
          <th>IDENTIFIER</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td colspan="3">Chemicals, peptides, and recombinant proteins</td>
        </tr>
        <tr>
          <td>Gamborg B-5 basal medium</td>
          <td>Phyto Technology</td>
          <td>Cat#G398</td>
        </tr>
        <tr>
          <td>Sucrose</td>
          <td>ABCONE</td>
          <td>Cat#57501</td>
        </tr>
        <tr>
          <td>2-(N-morpholino) ethanesulfonic acid (MES)</td>
          <td>BBI Life Sciences</td>
          <td>Cat#145224948</td>
        </tr>
        <tr>
          <td>Phytagel</td>
          <td>Sigma-Aldrich</td>
          <td>Cat#7101052-1</td>
        </tr>
        <tr>
          <td>Agar Bacteriological Grade</td>
          <td>Shanghai Jiafeng</td>
          <td>Cat#H8145</td>
        </tr>
        <tr>
          <td>Triton X-100</td>
          <td>Sigma-Aldrich</td>
          <td>Cat#9002931</td>
        </tr>
        <tr>
          <td>KCl</td>
          <td>Sigma-Aldrich</td>
          <td>Cat#7447407</td>
        </tr>
        <tr>
          <td>NaCl</td>
          <td>Sigma-Aldrich</td>
          <td>Cat#7647145</td>
        </tr>
        <tr>
          <td>MgCl<sub>2</sub>·6H<sub>2</sub>O</td>
          <td>Sigma-Aldrich</td>
          <td>Cat#7791186</td>
        </tr>
        <tr>
          <td>Tris base</td>
          <td>Sigma-Aldrich</td>
          <td>Cat#77861</td>
        </tr>
        <tr>
          <td>DAPI</td>
          <td>AAT Bioquest</td>
          <td>Cat#28718903</td>
        </tr>
        <tr>
          <td>2-Mercaptoethanol</td>
          <td>Ruibio</td>
          <td>Cat#60242</td>
        </tr>
        <tr>
          <td>Spermine</td>
          <td>Sigma-Aldrich</td>
          <td>Cat#85590</td>
        </tr>
        <tr>
          <td>Eva green dye</td>
          <td>Biotium</td>
          <td>Cat#31000</td>
        </tr>
        <tr>
          <td>Phusion high-fidelity DNA polymerase</td>
          <td>Thermo Fisher Scientific</td>
          <td>Cat#022021</td>
        </tr>
        <tr>
          <td>Complete protease inhibitor cocktail</td>
          <td>Merck</td>
          <td>Cat#04693132001</td>
        </tr>
        <tr>
          <td colspan="3">Critical commercial assays</td>
        </tr>
        <tr>
          <td>MinElute Reaction Cleanup Kit</td>
          <td>QIAGEN</td>
          <td>Cat#28204</td>
        </tr>
        <tr>
          <td>TruePrep DNA Library Prep Kit v2</td>
          <td>Vazyme Biotech</td>
          <td>Cat#TD50102</td>
        </tr>
        <tr>
          <td>TruePrep Index Kit v2</td>
          <td>Vazyme Biotech</td>
          <td>Cat#TD202</td>
        </tr>
        <tr>
          <td>2× NEBNext high-fidelity PCR mix</td>
          <td>New England Biolabs</td>
          <td>Cat#M0541L</td>
        </tr>
        <tr>
          <td>AMPure beads</td>
          <td>Beckman</td>
          <td>Cat#A63880</td>
        </tr>
        <tr>
          <td colspan="3">Deposited data</td>
        </tr>
        <tr>
          <td>ATAC-seq, ChIP-seq, and RNA-seq experiment data</td>
          <td>This paper</td>
          <td>BioProject PRJCA002620,Beijing Institute of Genomics Data Center (<a href="http://bigd.big.ac.cn">http://bigd.big.ac.cn</a>)</td>
        </tr>
        <tr>
          <td colspan="3">Experimental models: organisms/strains</td>
        </tr>
        <tr>
          <td>
<i>A. thaliana</i>: Col-0</td>
          <td>N/A</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td colspan="3">Software and algorithms</td>
        </tr>
        <tr>
          <td>R version 3.6</td>
          <td>The R Foundation</td>
          <td>RRID: SCR_001905 <a href="https://www.r-project.orgl">https://www.r-project.orgl</a>
</td>
        </tr>
        <tr>
          <td>Adobe Photoshop CC 2018</td>
          <td>Adobe Acrobat</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>Adobe Illustrator CC 2018</td>
          <td>Adobe Acrobat</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>Fastp</td>
          <td>(<a href="#bib1">Chen et al., 2018</a>)</td>
          <td>RRID: SCR_016962 <a href="https://github.com/OpenGene/fastp">https://github.com/OpenGene/fastp</a>
</td>
        </tr>
        <tr>
          <td>FastQC v0.11.7</td>
          <td>FastQC</td>
          <td>RRID:SCR_014583 <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>
          </td>
        </tr>
        <tr>
          <td>MulitQC v1.6</td>
          <td>(<a href="#bib2">Ewels et al., 2016</a>)</td>
          <td><a href="https://multiqc.info/">https://multiqc.info/</a></td>
        </tr>
        <tr>
          <td>Bowtie2 v2.3.4.3</td>
          <td>(<a href="#bib5">Langmead and Salzberg, 2012</a>)</td>
          <td><a href="http://bowtie-bio.sourceforge.net/bowtie2/">http://bowtie-bio.sourceforge.net/bowtie2/</a></td>
        </tr>
        <tr>
          <td>Samtools v1.9</td>
          <td>(<a href="#bib6">Li et al., 2009</a>)</td>
          <td>RRID: SCR_002105 <a href="http://www.htslib.org/">http://www.htslib.org/</a>
</td>
        </tr>
        <tr>
          <td>sambamba v0.6.7</td>
          <td>(<a href="#bib14">Tarasov et al., 2015</a>)</td>
          <td><a href="https://lomereiter.github.io/sambamba/">https://lomereiter.github.io/sambamba/</a></td>
        </tr>
        <tr>
          <td>bedtools v2.25.0</td>
          <td>(<a href="#bib9">Quinlan and Hall, 2010</a>)</td>
          <td>RRID: SCR_006646 <a href="https://bedtools.readthedocs.io/en/latest/">https://bedtools.readthedocs.io/en/latest/</a>
</td>
        </tr>
        <tr>
          <td>deepTools v3.1.2</td>
          <td>(<a href="#bib10">Ramirez et al., 2014</a>)</td>
          <td>RRID: SCR_016366 <a href="https://deeptools.readthedocs.io/en/develop/">https://deeptools.readthedocs.io/en/develop/</a>
</td>
        </tr>
        <tr>
          <td>featureCounts v1.6.2</td>
          <td>(<a href="#bib7">Liao et al., 2014</a>)</td>
          <td>RRID: SCR_012919 <a href="http://bioinf.wehi.edu.au/featureCounts/">http://bioinf.wehi.edu.au/featureCounts/</a>
</td>
        </tr>
        <tr>
          <td>DESeq2 v1.26.0</td>
          <td>(<a href="#bib8">Love et al., 2014</a>)</td>
          <td>RRID: SCR_015687 <a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html">https://bioconductor.org/packages/release/bioc/html/DESeq2.html</a>
          </td>
        </tr>
        <tr>
          <td>MACS2 v2.1.2</td>
          <td>(<a href="#bib17">Zhang et al., 2008</a>)</td>
          <td><a href="https://github.com/taoliu/MACS">https://github.com/taoliu/MACS</a></td>
        </tr>
        <tr>
          <td>DiffBind v2.14.0</td>
          <td>(<a href="#bib13">Ross-Innes et al., 2012</a>)</td>
          <td>RRID: SCR_012918 <a href="https://bioconductor.org/packages/release/bioc/html/DiffBind.html">https://bioconductor.org/packages/release/bioc/html/DiffBind.html</a>
          </td>
        </tr>
        <tr>
          <td>ChIPseeker v1.22.0</td>
          <td>(<a href="#bib16">Yu et al., 2015</a>)</td>
          <td>
<a href="https://bioconductor.org/packages/release/bioc/html/ChIPseeker.html">https://bioconductor.org/packages/release/bioc/html/ChIPseeker.html</a>
          </td>
        </tr>
        <tr>
          <td>HOMER v4.10</td>
          <td>(<a href="#bib3">Heinz et al., 2010</a>)</td>
          <td>RRID: SCR_010881<a href="http://homer.ucsd.edu/homer/">http://homer.ucsd.edu/homer/</a>
</td>
        </tr>
        <tr>
          <td>Intervene v0.6.1</td>
          <td>(<a href="#bib4">Khan and Mathelier, 2017</a>)</td>
          <td><a href="https://intervene.readthedocs.io/en/latest/">https://intervene.readthedocs.io/en/latest/</a></td>
        </tr>
        <tr>
          <td>Integrative Genomics Viewer</td>
          <td>(<a href="#bib12">Robinson et al., 2011</a>)</td>
          <td>RRID: SCR_011793 <a href="http://software.broadinstitute.org/software/igv/">http://software.broadinstitute.org/software/igv/</a>
          </td>
        </tr>
        <tr>
          <td colspan="3">Other</td>
        </tr>
        <tr>
          <td>Eppendorf realplex2</td>
          <td>Eppendorf</td>
          <td>Cat#A248709R</td>
        </tr>
        <tr>
          <td>The BD FACSAria II</td>
          <td>BD</td>
          <td>N/A</td>
        </tr>
        <tr>
          <td>Percival chamber</td>
          <td>Percival</td>
          <td>N/A</td>
        </tr>
      </tbody>
    </table>
  </section>
  <section>
    <h2 id="materials-and-equipment">Materials and equipment</h2>
    <p>The lysis buffer and wash buffer can be stored at 4°C for several months. Please add spermine and
      2-mercaptoethanol before experiments.</p>
    <table id="undtbl1">
      <caption>Lysis buffer</caption>
      <thead>
        <tr>
          <th>Reagent</th>
          <th>Final concentration</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Tris-HCl pH 7.5 (1 M)</td>
          <td>15 mM</td>
          <td>0.75 mL</td>
        </tr>
        <tr>
          <td>NaCl (5 M)</td>
          <td>20 mM</td>
          <td>0.2 mL</td>
        </tr>
        <tr>
          <td>KCl (2.5 M)</td>
          <td>80 mM</td>
          <td>1.6 mL</td>
        </tr>
        <tr>
          <td>2-Mercaptoethanol (14.3 M)</td>
          <td>5 mM</td>
          <td>17.5 μL</td>
        </tr>
        <tr>
          <td>Spermine (1 M)</td>
          <td>0.5 mM</td>
          <td>25 μL</td>
        </tr>
        <tr>
          <td>Cocktail (100×)</td>
          <td>1×</td>
          <td>0.5 mL</td>
        </tr>
        <tr>
          <td>Triton X-100 (100%)</td>
          <td>0.2%</td>
          <td>0.1 mL</td>
        </tr>
        <tr>
          <td>ddH<sub>2</sub>O</td>
          <td></td>
          <td>47.3 mL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>n/a</b></td>
          <td><b>50 mL</b></td>
        </tr>
      </tbody>
    </table>
    <table id="undtbl2">
      <caption>Wash buffer</caption>
      <thead>
        <tr>
          <th>Reagent</th>
          <th>Final concentration</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Tris-HCl pH 8.0 (1 M)</td>
          <td>10 mM</td>
          <td>0.5 mL</td>
        </tr>
        <tr>
          <td>MgCl<sub>2</sub> (1 M)</td>
          <td>5 mM</td>
          <td>0.25 mL</td>
        </tr>
        <tr>
          <td>Cocktail (100×)</td>
          <td>1×</td>
          <td>0.5 mL</td>
        </tr>
        <tr>
          <td>ddH<sub>2</sub>O</td>
          <td></td>
          <td>49.25 mL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>n/a</b></td>
          <td><b>50 mL</b></td>
        </tr>
      </tbody>
    </table>
    <div class="critical">
      <span class="critical-title">Critical:</span> Both 2-mercaptoethanol and spermine have pungent odors, please add
      them in a fume hood.
    </div>
    <div class="alternatives">
      <span class="alternatives-title">Alternatives:</span> This protocol describes cell nuclear FACS using a BD
      FACSAria II machine, any other FACS machine can be used as well. The FACS parameters used here are: nozzle size
      (70 μm), psi (70), sorting speed (1), sheath fluid (1× PBS).
    </div>
  </section>
  <section>
    <h2 id="step-by-step-method-details">Step-by-step method details</h2>
    <h3 id="sec3.1">Extraction of cell nuclei</h3>
    <div class="timing">
      <span class="timing-title">Timing:</span> 1–2 h
    </div>
    <p>This step describes the procedure for preparation of the nuclei suspension. Steps 1 and 2 are used for
      preparation of cell nuclei from embryos, while steps 3 and 4 are used for seedlings.</p>
    <ol>
      <li>Approximately 200 embryos at the late-cotyledon stage were collected with a 1 mL injection syringe under a
        dissecting microscope at 25°C. The collected embryos were placed on a 3.5 cm diameter round dish supplemented
        with B5 agar media (<a href="#fig1">Figure 1</a>A). The B5 agar will keep the embryos alive. The dissected
        embryos were transferred into a 12 cm long square dish with 500 μL pre-chilled lysis buffer on ice.
        <figure id="fig1">
          <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/418-Fig1.jpg" alt="Fig1.jpg">
          <figcaption>
            <div class="figcaption-title">Figure 1. Generation of nuclei suspension</div>
            <p>(A) The <i>Arabidopsis</i> embryos at the late-cotyledon stage were collected on the B5 agar.</p>
            <p>(B) The nuclei lysis suspension of embryos after filtering.</p>
            <p>(C) The 3-day-old <i>Arabidopsis</i> seedlings were collected on the B5 agar.</p>
            <p>(D) The nuclei lysis suspension of seedlings after filtering.</p>
          </figcaption>
        </figure>
      </li>
      <li>Finely chop embryos with a double-sided blade to release nuclei on ice in fume hood.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> The embryos naturally settle at the bottom of the lysis buffer, so we can
      collect the materials together and repeatedly cut them. Due to the small volume of embryos, please cut them
      repeatedly to release more nucleus until the color of the lysis buffer is yellow-green (<a href="#fig1">Figure 1</a>B), which should take approximately 10 min. 200 embryos should provide a sufficient
      number of nuclei.
    </div>
    <ol start="3">
      <li>Approximately 3-day-old 150 - 200 seedlings grown on B5 agar were collected with forceps at RT. The seedlings
        were transferred to a 12 cm long square dish with 500 μL pre-chilled lysis buffer on ice.</li>
      <li>Finely chop seedlings with a double-sided blade to release nuclei on ice in fume hood.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> For seedlings, 5 min of chopping is enough. The color of the lysis buffer
      will be bright green after chopping.
    </div>
    <ol start="5">
      <li>Filter the slurry through a 40 μm filter into 15 mL collection tubes on ice. Rinse the petri dish with 1.5 mL
        lysis buffer and filter into the same 15 mL collection tube. The final nuclei suspension is about 2 mL in volume
        and the color should appear yellow-green.</li>
    </ol>
    <div class="critical">
      <span class="critical-title">Critical:</span> For acquiring more intact nuclei, we recommend extensively chopping
      plant materials in lysis buffer rather than grinding them in liquid nitrogen which leads to more tissue fragments
      that will disturb FACS.
    </div>
    <h3 id="sec3.2">FACS of cell nuclei</h3>
    <div class="timing">
      <span class="timing-title">Timing:</span> 1–5 h
    </div>
    <p>This step describes the procedure for sorting nuclei by FACS.</p>
    <ol start="6">
      <li>The nuclei suspensions are stained with DAPI for flow cytometric analysis. Set up an unstained control without
        DAPI.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> It is not necessary to estimate the density of nuclei suspension in this
      step because we will sort ∼50,000 nuclei per replicate to ensure repeatability. The amount of nuclei suspension
      required for control samples is flexible. The purpose of control is to determine the appropriate voltage needed to
      exclude the interference of other particles.
    </div>
    <ol start="7">
      <li>Add 2 μL of 1 mg/mL DAPI to the nuclei suspension in a sample tube, flick gently to mix.</li>
      <li>Select parameters including FSC-A (forward scatter-A, particle size), SSC-A (side scatter-A, internal
        complexity), DAPI-A (intensity of DAPI staining), and DAPI-W (voltage pulse width).</li>
      <li>Use the unstained control and stained samples to set appropriate photomultiplier tube (PMT) voltages and
        adjust compensations on a BD FACSAria II cell analyzer.</li>
      <li>Create gates for the samples according to the position of 2C or 4C nuclei (<a href="#fig2">Figure 2</a>).
        <figure id="fig2">
          <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/418-Fig2.jpg" alt="Fig2.jpg">
          <figcaption>
            <div class="figcaption-title">Figure 2. Gate set for FACS</div>
            <p>(A) FACS histogram of embryo nuclei suspension. The DAPI-A-2C and 4C represent cell ploidy with 2 times
              and 4 times, respectively. Only DAPI-A-2C cell nuclei was sorted.</p>
            <p>(B) FACS histogram of seedling nuclei suspension. The DAPI-A-2C, -4C, and -8C represent cell ploidy with
              2 times, 4 times, and 8 times, respectively. All DAPI-A-2C, -4C, and -8C cell nuclei were sorted.</p>
          </figcaption>
        </figure>
      </li>
      <li>Add 500 μL of lysis buffer to a 10 mL BD collect tube and collect a total of 50,000 events per sample (each
        sample per tube). The nozzle size, psi, and sorting speed of FACS are 70 μm, 70, and 1, respectively.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> 200 embryos and 150–200 3-day-old seedlings are enough material for getting
      50,000 nuclei.
    </div>
    <ol start="12">
      <li>Centrifuge at 1,000 × <i>g</i> for 10 min at 4°C.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> Since we cannot see the pellet at the bottom of the tube after
      centrifugation, please leave ∼50 μL of liquid to avoid nuclei loss when discarding the supernatant.
    </div>
    <ol start="13">
      <li>Pipette 1 mL of pre-chilled wash buffer into the centrifuge tube.</li>
      <li>Centrifuge at 1,000 × <i>g</i> for 10 min at 4°C and discard the supernatant as much as possible.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> Immature embryo cells at late-cotyledon developmental stage rarely divide.
      DAPI-A detected nuclei almost with 2C from FACS (<a href="#fig2">Figure 2</a>A). The nuclei were sorted into lysis
      buffer as plant nuclei do not remain intact in PBS buffer that is used for FACS.
    </div>
    <h3 id="sec3.3">ATAC-seq library constructing</h3>
    <div class="timing">
      <span class="timing-title">Timing:</span> 5–8 h
    </div>
    <p>This step describes the procedure for generation of ATAC-seq library.</p>
    <p>We use the TruePrep DNA Library Prep Kit V2 for Illumina including 5× TTBL and TTE Mix V50 (Vazyme, TD50102; see
      manual at <a href="http://www.vazymebiotech.com/products_detail/productId=70.html">http://www.vazymebiotech.com/products_detail/productId=70.html</a>)
      to construct ATAC-seq libraries. We use the MinElute Reaction Cleanup Kit including Buffer PB, PE and EB (Qiagen)
      to purify transposed DNA.</p>
    <ol start="15">
      <li>Prepare the transposition master mix in a 200 μL Lobind tube (Eppendorf) on ice. Add reagents in the order
        shown below.</li>
    </ol>
    <table id="undtbl3">
      <thead>
        <tr>
          <th>Reagent</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>5× TTBL</td>
          <td>10 μL</td>
        </tr>
        <tr>
          <td>TTE Mix V50</td>
          <td>5 μL</td>
        </tr>
        <tr>
          <td>Cocktail (100×)</td>
          <td>0.5 μL</td>
        </tr>
        <tr>
          <td>Nuclei suspension</td>
          <td>x μL</td>
        </tr>
        <tr>
          <td>ddH<sub>2</sub>O</td>
          <td>34.5–x μL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td>
<b>50</b> μL</td>
        </tr>
      </tbody>
    </table>
    <div class="note">
      <span class="note-title">Note:</span> The volume of nuclei suspension after step 14 is variable. Please sure that
      the remaining nuclear suspension is less than 34.5 μL since the total reaction volume for transposition is 50 μL.
      It is not necessary to optimize reaction conditions (e.g., transposition time) when working with different initial
      volumes or numbers of nuclei.
    </div>
    <ol start="16">
      <li>Dispense master mix into a PCR tube containing nuclear suspension. Pipette to mix.</li>
      <li>Load the tube into a thermal cycler pre-warmed to 37°C and incubate for 30 min with occasional gentle mixing
        to keep the nuclei in suspension.</li>
      <li>Add 3–5 volumes of Buffer PB to 1 volume of the PCR sample.</li>
      <li>Transfer the mixture to a spin column that is situated in a provided 2 mL collection tube.</li>
      <li>Centrifuge at 13,523 × <i>g</i> for 1 min at 4°C.</li>
      <li>Discard the flow-through and add 750 μL of Buffer PE to the spin column.</li>
      <li>Centrifuge at 13,523 × <i>g</i> for 1 min at 4°C.</li>
      <li>Discard the flow-through and centrifuge the column for an additional 2 min.</li>
      <li>Place column in a clean 1.5 mL DNA Lobind microcentrifuge tube and add 11 μL of buffer EB to the center of the
        column for 2 min.</li>
      <li>Centrifuge at 13,523 × <i>g</i> for 1 min at 4°C.</li>
    </ol>
    <div class="pause-point">
      <span class="pause-point-title">Pause Point:</span> Samples can be safely stored overnight at 4°C or −20°C.
    </div>
    <p>We use the TruePrep DNA Index Kit V2 for Illumina sequencing (Vazyme) and NEBNext high-fidelity PCR mix to
      amplify transposed DNA.</p>
    <ol start="26">
      <li>Prepare the PCR amplification mix in a 0.2 mL PCR tube on ice as follows:</li>
    </ol>
    <table id="undtbl4">
      <thead>
        <tr>
          <th>Reagent</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Transposed DNA</td>
          <td>10 μL</td>
        </tr>
        <tr>
          <td>N5 Primer</td>
          <td>2.5 μL</td>
        </tr>
        <tr>
          <td>N7 Primer</td>
          <td>2.5 μL</td>
        </tr>
        <tr>
          <td>2× NEBNext high-fidelity PCR mix</td>
          <td>25 μL</td>
        </tr>
        <tr>
          <td>ddH<sub>2</sub>O</td>
          <td>10 μL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>50 μL</b></td>
        </tr>
      </tbody>
    </table>
    <ol start="27">
      <li>Incubate the reaction in a thermo cycler with the following PCR program.</li>
    </ol>
    <table id="undtbl5">
      <thead>
        <tr>
          <th colspan="4">PCR cycling conditions</th>
        </tr>
        <tr>
          <th>Steps</th>
          <th>Temperature</th>
          <th>Time</th>
          <th>Cycles</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Pre-incubation</td>
          <td>72°C</td>
          <td>5 min</td>
          <td>1</td>
        </tr>
        <tr>
          <td>Initial denaturation</td>
          <td>98°C</td>
          <td>30 s</td>
          <td>1</td>
        </tr>
        <tr>
          <td>Denaturation</td>
          <td>98°C</td>
          <td>10 s</td>
          <td rowspan="3">5 cycles</td>
        </tr>
        <tr>
          <td>Annealing</td>
          <td>63°C</td>
          <td>30 s</td>
        </tr>
        <tr>
          <td>Extension</td>
          <td>72°C</td>
          <td>1 min</td>
        </tr>
        <tr>
          <td>Hold</td>
          <td>4°C</td>
          <td colspan="2">Forever</td>
        </tr>
      </tbody>
    </table>
    <ol start="28">
      <li>To determine the number of additional PCR cycles needed to adequately amplify the DNA library, prepare the
        quantitative PCR (qPCR) Library Amplification Mix as follows:</li>
    </ol>
    <table id="undtbl6">
      <thead>
        <tr>
          <th>Reagent</th>
          <th>Amount</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Amplified library</td>
          <td>5 μL</td>
        </tr>
        <tr>
          <td>N5 Primer</td>
          <td>0.5 μL</td>
        </tr>
        <tr>
          <td>N7 Primer</td>
          <td>0.5 μL</td>
        </tr>
        <tr>
          <td>2× NEBNext high-fidelity PCR mix</td>
          <td>7.5 μL</td>
        </tr>
        <tr>
          <td>20× Eva green dye</td>
          <td>0.75 μL</td>
        </tr>
        <tr>
          <td>ddH<sub>2</sub>O</td>
          <td>0.75 μL</td>
        </tr>
        <tr>
          <td><b>Total</b></td>
          <td><b>15 μL</b></td>
        </tr>
      </tbody>
    </table>
    <ol start="29">
      <li>Incubate the reaction in a qPCR thermocycler with the following PCR program.</li>
    </ol>
    <table id="undtbl7">
      <thead>
        <tr>
          <th colspan="4">PCR cycling conditions</th>
        </tr>
        <tr>
          <th>Steps</th>
          <th>Temperature</th>
          <th>Time</th>
          <th>Cycles</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Initial denaturation</td>
          <td>98°C</td>
          <td>30 s</td>
          <td>1</td>
        </tr>
        <tr>
          <td>Denaturation</td>
          <td>98°C</td>
          <td>10 s</td>
          <td rowspan="3">20 cycles</td>
        </tr>
        <tr>
          <td>Annealing</td>
          <td>63°C</td>
          <td>30 s</td>
        </tr>
        <tr>
          <td>Extension</td>
          <td>72°C</td>
          <td>1 min</td>
        </tr>
      </tbody>
    </table>
    <ol start="30">
      <li>To determine the optimal number of cycles needed to amplify the remaining 45 μL of each library, view the
        linear fluorescence versus cycle number plot on the qPCR machine once the reaction is finished. The cycle number
        at which the fluorescence for a given reaction is at 1/3 of its maximum is the number of additional cycles (N)
        that each library requires for adequate amplification (<a href="#fig3">Figure 3</a>).
        <figure id="fig3">
          <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/418-Fig3.jpg" alt="Fig3.jpg">
          <figcaption>
            <div class="figcaption-title">Figure 3. Schematic diagram of qPCR</div>
            <p>The maximum of fluorescence is 2,000. As such, one-third of the maximum is ~666, and the number of PCR
              cycles should be 9 (red line).</p>
          </figcaption>
        </figure>
      </li>
      <li>Incubate the remaining reaction in a thermocycler with the following PCR program.</li>
    </ol>
    <table id="undtbl8">
      <thead>
        <tr>
          <th colspan="4">PCR cycling conditions</th>
        </tr>
        <tr>
          <th>Steps</th>
          <th>Temperature</th>
          <th>Time</th>
          <th>Cycles</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Initial denaturation</td>
          <td>98°C</td>
          <td>30 s</td>
          <td>1</td>
        </tr>
        <tr>
          <td>Denaturation</td>
          <td>98°C</td>
          <td>10 s</td>
          <td rowspan="3">N cycles</td>
        </tr>
        <tr>
          <td>Annealing</td>
          <td>63°C</td>
          <td>30 s</td>
        </tr>
        <tr>
          <td>Extension</td>
          <td>72°C</td>
          <td>1 min</td>
        </tr>
        <tr>
          <td>Hold</td>
          <td>4°C</td>
          <td colspan="2">Forever</td>
        </tr>
      </tbody>
    </table>
    <ol start="32">
      <li>Add 67.5 μL (1.5× of sample volume) resuspended Ampure XP beads to the PCR reaction. Mix well by pipetting up
        and down at least 10 times. Alternatively, samples can be mixed by vortexing for 3–5 s.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> If Ampure XP beads are stored at 4°C, please pre-warm to 25°C for at least
      30 min and vortex before use.
    </div>
    <ol start="33">
      <li>Incubate samples on the bench for at least 5 min at 25°C.</li>
      <li>Place the tube on an appropriate magnetic stand to separate the beads from the supernatant. Quickly centrifuge
        the sample to collect the liquid from the sides of the tube before placing it on the magnetic stand if
        necessary.</li>
      <li>After 5 min, carefully remove and discard the supernatant. Be careful not to disturb the beads that contain
        bound DNA.</li>
      <li>Add 200 μL of freshly prepared 80% ethanol to the tube while in the magnetic stand.</li>
      <li>Incubate at 25°C for 30 s, and carefully remove and discard the supernatant.</li>
      <li>Repeat steps 34 and 35 once for a total of two washes.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> Be sure to remove all visible liquid after the second wash. Briefly
      centrifuge the tube on a benchtop centrifuge if necessary.
    </div>
    <ol start="39">
      <li>Air-dry the beads for up to 5 min while the tube is on the magnetic stand with the lid open.</li>
    </ol>
    <div class="note">
      <span class="note-title">Note:</span> Do not over-dry the beads. This may result in a lower recovery of DNA. Elute
      the samples when the beads are still dark-brown and glossy looking, but when all visible liquid has evaporated.
      When the beads turn lighter brown and start to crack, it shows beads are too dry.
    </div>
    <ol start="40">
      <li>Remove the tube from the magnetic stand. Elute the DNA from the beads by adding 30 μL EB buffer provided by
        Qiagen.</li>
      <li>Mix well by pipetting up and down 10 times, or on a vortex mixer. Incubate for at least 2 min at room
        temperature. Quickly spin the sample to collect the liquid from the sides of the tube if necessary.</li>
      <li>Place the tube on the magnetic stand for 5 min. Transfer 28 μL to a new 1.5 mL tube. Libraries can be stored
        at −20°C. The concentration of the library can be checked by NanoDrop.</li>
    </ol>
    <h3 id="sec3.4">ATAC-seq library quality assessment</h3>
    <p>This step describes the procedure for assessment of ATAC-seq library.</p>
    <p>We pipette 20 μL of library samples and send them to Novogene for library quality determination and sequencing
      (<a href="#fig4">Figure 4</a>). The library quality platform: NGS3K; Sequencing platform: HiseqPE150, paired-end.
    </p>
    <figure id="fig4">
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/418-Fig4.jpg" alt="Fig4.jpg">
      <figcaption>
        <div class="figcaption-title">Figure 4. Assessment of ATAC-seq library</div>
        <p>(A–D) The purified ATAC-seq DNA libraries of 4 samples on NGS3K platform. Em, immature embryo; G3, 3-day-old
          seedlings grown on the B5 agar plate.</p>
      </figcaption>
    </figure>
    <div class="note">
      <span class="note-title">Note:</span> If we started with 50,000 sorted nuclei, 20 μL of library sample is enough
      for quality assessment and sequencing. Most sequencing companies perform library quality analysis before
      sequencing.
    </div>
  </section>
  <section>
    <h2 id="expected-outcomes">Expected outcomes</h2>
    <p>We performed ATAC-seq on four samples following this protocol. The resultant datasets inform us of the dynamics
      of accessible chromatin regions during seed germination. First, our result shows the distribution of open
      chromatin regions along the genome (<a href="#fig5">Figure 5</a>A). Second, the analysis of differences between
      samples suggest the major variable regions and the principal biological processes encoded therein by gene ontology
      (GO) enrichment annotation (<a href="#fig5">Figures 5</a>C and 5D). Finally, we inferred putative transcription
      factor binding sites which are enriched in open regions by HOMER-motif analysis (<a href="#fig5">Figures 5</a>B
      and 5E).</p>
    <figure id="fig5">
      <img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/418-Fig5.jpg" alt="Fig5.jpg">
      <figcaption>
        <div class="figcaption-title">Figure 5. Results of ATAC-seq analysis</div>
        <p>(A) The genome-wide distribution of ATAC-seq peaks. Window size: gene body ± 3.0 kb.</p>
        <p>(B) HOMER DNA-motif enrichment analyses of accessible peaks. The enrichment of binding motifs of 8
          transcription factors per sample are shown.</p>
        <p>(C) Volcano plot showing the genes associated with decreased (blue) or increased (red) accessible peaks
          between immature embryos (Em) and 3-day-old seedlings (G3). The known genes related to seed or embryo
          development are shown. Ns (gray in color), no difference between two samples.</p>
        <p>(D) GO term analyses showing distinct gene ontologies of target genes linked to differentially accessible
          peaks (G3 versus Em). The selected 20 enriched GO biological processes are indicated. The −log10(p.adj) is
          given.</p>
        <p>(E) HOMER DNA-motif enrichment analyses of differentially accessible peaks (G3 versus Em). The enrichment of
          binding motifs of 5 transcription factors per sample are shown.</p>
      </figcaption>
    </figure>
  </section>
  <section>
    <h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
    <h3 id="sec5.1">ATAC-seq analysis pipeline</h3>
    <p>This step describes the bioinformatic pipeline used for the analysis of ATAC-seq dataset and identification of
      differentially accessible peaks.</p>
    <p>We will describe the basic analysis pipeline of processing and analyzing ATAC-seq including data filtering,
      quality control, matching, difference analysis, motif prediction, and GO set enrichment analysis as shown below.
      For a detailed analysis process and related code refer to the following link: <a href="https://github.com/WangLab-SIPPE/2020STARProtocols_pipeline">https://github.com/WangLab-SIPPE/2020STARProtocols_pipeline</a>.
    </p>
    <ol>
      <li>For each library, raw.fastq was trimmed by fastp v0.20.0 with the parameters "-a
        CTGTCTCTTATACACATCT."</li>
      <li>After trimming, FastQC v0.11.7 and MulitQC v1.6 were performed as quality control to obtain clean fq files.
      </li>
      <li>Reads were aligned using Bowtie2 v2.3.4.3 to the <i>Arabidopsis</i> genome (TAIR10). Alternative align tools
        such as Bwa mean can be chosen.</li>
      <li>The resulting SAM file containing mapped reads were converted to BAM format, sorted, and indexed using
        Samtools v1.9.</li>
      <li>The biological replicates were merged by Samtools v1.9.</li>
      <li>The sorted BAM were processed to remove duplicated and organellar reads by sambamba v0.6.7 and bedtools
        v2.25.0. An alternative tool to mark and remove duplicated reads is Picard.</li>
      <li>To normalize and visualize the individual and merged replicate datasets, the BAM files were converted to
        bigwig using bamCoverage provided by deepTools v3.1.2 with a bin size of 10 bp and normalized by Bin Per Million
        mapped reads.</li>
      <li>MACS2 v2.1.2 was used to call peaks with default parameters "-t ATAC.bam -f BAMPE -g 1.1e8."</li>
      <li>For differentially accessible region analyses, we used "DiffBind" with the parameter
        "minOverlap = 1." "DiffBind" was also used to calculate merged peak locations, based on the
        outer boundaries of overlapping peaks from all the analyzed samples.</li>
      <li>The peaks called by MACS2 and differential peaks identified by DiffBind v2.14.0 were annotated by
        TxDb.Athaliana.BioMart.plantsmart28 and ChIPseeker v1.22.0 package with the "annotatePeak" function.
        The gene promoters are defined as ± 3.0 kb from the transcription start site (TSS).</li>
      <li>The differential regions were scanned for the enrichment of motifs using the "findMotifsGenome.pl"
        function provided by HOMER v4.10 with default parameters.</li>
      <li>The GO and Gene Set Enrichment Analysis of differentially accessible regions were performed using
        clusterProfiler v3.14.0 and org.At.tair.db v3.10.0.</li>
    </ol>
  </section>
  <section>
    <h2 id="limitations">Limitations</h2>
    <p>This protocol is designed for fresh plant materials. If the materials are stored at −80°C or in liquid nitrogen,
      it will lead to a high amount of tissue debris, which can interfere with the flow sorting of intact nuclei.</p>
  </section>
  <section>
    <h2 id="troubleshooting">Troubleshooting</h2>
    <h3 id="sec7.1">Problem 1</h3>
    <p>Sequenced reads have a low mapping rate.</p>
    <h3 id="sec7.2">Potential solution</h3>
    <p>There are several reasons that can cause low mapping rates. Please ensure that all solutions, equipment, and work
      areas are free of DNA. Wear a mask and gloves when chopping samples and constructing libraries. Avoid excessive
      PCR cycles during amplification. In general, 12–17 cycles are sufficient.</p>
    <h3 id="sec7.3">Problem 2</h3>
    <p>Poor data quality.</p>
    <h3 id="sec7.4">Potential solution</h3>
    <p>This might be caused by low quality or a low amount of cell nuclei. Please make sure that fresh samples are used,
      and an appropriate sorting speed and sheath pressure are selected.</p>
    <h3 id="sec7.5">Problem 3</h3>
    <p>Sequence reads do not map primarily to non-coding regions.</p>
    <h3 id="sec7.6">Potential solution</h3>
    <p>When you find that mapped sequences were mostly aligned to exonic regions in the reference genome, this might be
      caused by a low quality of nuclei or exogenous DNA pollution. Low-quality cell nuclei might cause nucleosome
      instability, which leads to an increase in the mapping rate at non-signal regions. To fix this problem, please
      make sure that only high-quality cell nuclei are used for the Tn5 tagmentation. As such, the examination of the
      sorted nuclei under a microscope is strongly recommended. Otherwise, 75% ethanol or DNA cleaner can be used to
      remove exogenous DNAs, and thereby improving your data quality.</p>
    <h3 id="sec7.7">Problem 4</h3>
    <p>The differentially accessible regions are far less as expected.</p>
    <h3 id="sec7.8">Potential solution</h3>
    <p>This problem can also be caused by poor data quality. Please check peak numbers, plot profiles, and the
      correlation between replicates. You can achieve these using the "plotCorrelation" function in deeptools,
      "plotPCA" function in DESeq2, or the "dba.plotPCA" function in DESeq2. Meanwhile, batch
      effects should also be considered. You can introduce the batch in the design effect with block parameter
      "dba.contrast" in DiffBind. Finally, you can try different ways to probe differentially accessible
      regions. According to a recent report (<a href="#bib11">Reske et al., 2020</a>), different bioinformatic tools may
      lead to different results.</p>
  </section>
  <section>
  <h2 id="q-and-a">Q&amp;A</h2>
  <p class="questions"><b>Q:</b> Can I use the following materials instead of the recommended ones in the kit? 1) Vazyme beads or SPRIselect beads instead of AmPure beads;  2) NEBNEXT Ultra II DNA library kit for Illumina instead of TruePrep?</p>
  <p class="answer"><b>A:</b> 1) Vazyme beads or SPRIselect beads instead of AmPure beads?<br>The AmPure bead can be replaced by VAHTS DNA Clean Beads(Vazyme #N411), but we don’t recommend the SPRIselect beads because it is used for size selection. The PCR products of ATAC-seq should be purified totally rather than being selected by size.<br>2) NEBNEXT Ultra II DNA library kit for Illumina instead of TruePrep?<br>The NEBNEXT Ultra II DNA library kit for Illumina don’t include the core component of ATAC-seq (Tn5 transposase), so it can’t be used to ATAC-seq. We recommend the Illumina Tagment DNA Enzyme and Buffer Small Kit (20034197) or other Kit containing Tn5 transposase.</p>
</section>
  <section>
    <h2 id="references">References</h2>
    <p id="bib1">Chen, S., Zhou, Y., Chen, Y., and Gu, J. (2018). fastp: an ultra-fast all-in-one FASTQ preprocessor.
      Bioinformatics <i>34</i>, i884-i890. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref1" class="external-link">View at publisher</a></p>
    <p id="bib2">Ewels, P., Magnusson, M., Lundin, S., and Kaller, M. (2016). MultiQC: summarize analysis results for
      multiple tools and samples in a single report. Bioinformatics <i>32</i>, 3047-3048. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref2" class="external-link">View at publisher</a></p>
    <p id="bib3">Heinz, S., Benner, C., Spann, N., Bertolino, E., Lin, Y.C., Laslo, P., Cheng, J.X., Murre, C., Singh,
      H., and Glass, C.K. (2010). Simple combinations of lineage-determining transcription factors prime cis-regulatory
      elements required for macrophage and B cell identities. Mol. Cell <i>38</i>, 576-589. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref3" class="external-link">View at publisher</a></p>
    <p id="bib4">Khan, A. and Mathelier, A. (2017). Intervene: a tool for intersection and visualization of multiple
      gene or genomic region sets. BMC Bioinformatics <i>18</i>, 287. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref4" class="external-link">View at publisher</a></p>
    <p id="bib5">Langmead, B. and Salzberg, S.L. (2012). Fast gapped-read alignment with Bowtie 2. Nat. Methods
      <i>9</i>, 357-359. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref5" class="external-link">View at
        publisher</a></p>
    <p id="bib6">Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., and
      Durbin, R. (2009). The Sequence Alignment/Map format and SAMtools. Bioinformatics <i>25</i>, 2078-2079. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref6" class="external-link">View at publisher</a></p>
    <p id="bib7">Liao, Y., Smyth, G.K., and Shi, W. (2014). featureCounts: an efficient general purpose program for
      assigning sequence reads to genomic features. Bioinformatics <i>30</i>, 923-930. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref7" class="external-link">View at publisher</a></p>
    <p id="bib8">Love, M.I., Huber, W., and Anders, S. (2014). Moderated estimation of fold change and dispersion for
      RNA-seq data with DESeq2. Genome Biol. <i>15</i>, 550. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref8" class="external-link">View at publisher</a></p>
    <p id="bib9">Quinlan, A.R. and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic
      features. Bioinformatics <i>26</i>, 841-842. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref9" class="external-link">View at publisher</a></p>
    <p id="bib10">Ramirez, F., Dundar, F., Diehl, S., Gruning, B.A., and Manke, T. (2014). deepTools: a flexible
      platform for exploring deep-sequencing data. Nucleic Acids Res. <i>42</i>. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref10" class="external-link">View at publisher</a></p>
    <p id="bib11">Reske, J.J., Wilson, M.R., and Chandler, R.L. (2020). ATAC-seq normalization method can significantly
      affect differential accessibility analysis and interpretation. Epigenetics Chromatin <i>13</i>, 22. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref11" class="external-link">View at publisher</a></p>
    <p id="bib12">Robinson, J.T., Thorvaldsdottir, H., Winckler, W., Guttman, M., Lander, E.S., Getz, G., and Mesirov,
      J.P. (2011). Integrative genomics viewer. Nat. Biotechnol. <i>29</i>, 24-26. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref12" class="external-link">View at publisher</a></p>
    <p id="bib13">Ross-Innes, C.S., Stark, R., Teschendorff, A.E., Holmes, K.A., Ali, H.R., Dunning, M.J., Brown, G.D.,
      Gojis, O., Ellis, I.O., Green, A.R., et al. (2012). Differential oestrogen receptor binding is associated with
      clinical outcome in breast cancer. Nature <i>481</i>, 389-393. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref13" class="external-link">View at publisher</a></p>
    <p id="bib14">Tarasov, A., Vilella, A.J., Cuppen, E., Nijman, I.J., and Prins, P. (2015). Sambamba: fast processing
      of NGS alignment formats. Bioinformatics <i>31</i>, 2032-2034. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref14" class="external-link">View at publisher</a></p>
    <p id="bib15">Wang, F.X., Shang, G.D., Wu, L.Y., Xu, Z.G., Zhao, X.Y., and Wang, J.W. (2020). Chromatin
      accessibility dynamics and a hierarchical transcriptional regulatory network structure for plant somatic
      embryogenesis. Dev. Cell <i>54</i>, 742-757.e8. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref15" class="external-link">View at publisher</a></p>
    <p id="bib16">Yu, G., Wang, L.G., and He, Q.Y. (2015). ChIPseeker: an R/Bioconductor package for ChIP peak
      annotation, comparison and visualization. Bioinformatics <i>31</i>, 2382-2383. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref16" class="external-link">View at publisher</a></p>
    <p id="bib17">Zhang, Y., Liu, T., Meyer, C.A., Eeckhoute, J., Johnson, D.S., Bernstein, B.E., Nusbaum, C., Myers,
      R.M., Brown, M., Li, W., et al. (2008). Model-based analysis of ChIP-Seq (MACS). Genome Biol. <i>9</i>, R137. <a href="http://refhub.elsevier.com/S2666-1667(20)30276-8/sref17" class="external-link">View at publisher</a></p>
  </section>
  <section>
    <h2 id="article-info">Article Info</h2>
    <h3>Resource Availability</h3>
    <h4>Lead Contact</h4>
    <p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the
      Lead Contact, Jia-Wei Wang (<a href="mailto:jwwang@sippe.ac.cn">jwwang@sippe.ac.cn</a>).</p>
    <h4>Materials Availability</h4>
    <p>This study did not generate new unique materials.</p>
    <h4>Data and Code Availability</h4>
    <p>The data can be found online at <a href="https://www.sciencedirect.com/science/article/pii/S1534580720305517">https://www.sciencedirect.com/science/article/pii/S1534580720305517</a>
    </p>
    <p>The analysis code can be found online at <a href="https://github.com/WangLab-SIPPE/2020STARProtocols_pipeline">https://github.com/WangLab-SIPPE/2020STARProtocols_pipeline</a>.
    </p>
    <h3>Acknowledgments</h3>
    <p>We thank Hong Zhu for technical support in FACS (Cell Biology Core Facility of Shanghai Center for Plant Stress
      Biology, Center for Excellence in Molecular Plant Sciences, CAS); members of J.-W.W.’s lab for discussion and
      comments on the manuscript. This work was supported by <a href="https://doi.org/10.13039/501100001809">National
        Natural Science Foundation of China</a> grant numbers 31788103, 31525004, and 31721001 to (J.-W.W.) and the
      Strategic Priority Research Program of the <a href="https://doi.org/10.13039/501100002367">Chinese Academy of
        Sciences</a> grant number XDB27030101 to (J.-W.W.).</p>
    <h3>Author Contributions</h3>
    <p>F.-X.W., S.-G.D., and Y.-L.W. contributed equally. Y.-X.M. and J.G. helped with FACS. Z.-G.X. helped with
      bioinformatic analysis. F.-X.W. and S.-G.D. wrote the manuscript. J.-W.W. polished the paper and supervised the
      study.</p>
    <h3>Declaration of Interests</h3>
    <p>The authors declare no competing interests.</p>
  </section>
</article>