<article>
<section>
<h1 id="header">Measuring microenvironment-tuned nuclear stiffness of cancer cells with atomic force microscopy</h1>
<p><time datetime="2021-01-22">Published: January 22, 2021</time></p>
<p>Amlan Barai,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> Alakesh Das,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup> and Shamik Sen<sup><a href="#aff1">1</a>,<a href="#fn3">5</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biosciences and Bioengineering, Indian Institute of Technology Bombay, Mumbai 400076, India</p>
<p id="aff2"><sup>2</sup>Department of Biological Regulation, Weizmann Institute of Science, Rehovot 76100, Israel</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:amlan.hp@gmail.com">amlan.hp@gmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:adas1757@gmail.com">adas1757@gmail.com</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:shamiks@iitb.ac.in">shamiks@iitb.ac.in</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100296">10.1016/j.xpro.2021.100296</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Quantification of nuclear stiffness is challenging for cells encapsulated within a 3D extracellular matrix (ECM). Here, we describe an experimental setup for measuring microenvironment-dependent tuning of nuclear stiffness using an atomic force microscope (AFM). In our setup, ECM-coated polyacrylamide hydrogels mimic the stiffness of the microenvironment, enabling the measurement of nuclear stiffness using an AFM probe in live cancer cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib3">Das et al. (2019)</a> (<a href="https://doi.org/10.1016/j.matbio.2019.01.001">https://doi.org/10.1016/j.matbio.2019.01.001</a>).</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol to probe microenvironment-tuned nuclear stiffness of cancer cells with AFM</li>
<li>PA hydrogels mimic the bulk stiffness of the microenvironment</li>
<li>AFM can measure both cortical stiffness and nuclear stiffness in live cells</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Prepare reagents for polyacrylamide gels (PA gels) preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol>
<li>Acrylamide/bis-acrylamide stock solution preparation: Prepare a stock solution of acrylamide and bis-acrylamide in ultra-pure (UP) water. PA gels of varying stiffness can be prepared by mixing different concentrations of acrylamide and bis-acrylamide as per previous publications (<a href="#bib9">Syed et al., 2017</a>; <a href="#bib10">Tse and Engler, 2010</a>). In our studies, we have mixed 1.25 mL of 40% acrylamide solution and 0.75 mL of 2% (w/v) bis-acrylamide in 8 mL of UP water that produces a stiffness ~4.5 kPa (<a href="#bib10">Tse and Engler, 2010</a>). Sterile filter the solution using 0.22 μm syringe filters and store it at 4°C up to 30 days.</li>
<li>Ammonium Persulfate (APS) solution: Prepare 10% Ammonium Persulfate (APS) by dissolving appropriate amount of APS in UP water and then sterilize it using a 0.22 μm syringe filter in the hood. APS solution can be freshly prepared before use or can be stored as small aliquots of 200 μL at −20°C under dark conditions.</li>
<li>Cleaning of coverglass: 12 mm and 18 mm circular cover slips and rectangular glass slides (75 mm × 25 mm) are acid cleaned by incubating them in 1 M HCl solution for 20–30 min. After rinsing with UP water for 3–4 times, coverslips/coverslides are dried in between lint-free papers. These can be stored in 70% ethanol at room temperature and can be used later after washing with UP water.</li>
<li>Prepare HEPES buffer (50 mM, pH 8.5): Prepare HEPES buffer by dissolving appropriate amount of HEPES powder in autoclaved distilled water. After adjusting the pH to 8.5 using NaOH solution, filter sterilize the solution using 0.22 μm membrane filter and store in a sterile bottle.</li>
<li>Sulfo-SANPAH solution: Prepare 0.1 mg/μL stock solution in DMSO (TC grade) in sterile condition (inside TC hood) and store at −20°C in small aliquots of 500 μL under dark conditions. Each aliquot should not be thawed more than twice.</li>
<li>Prepare 0.1 M NaOH solution in UP water and then filter the solution using 0.22 μm syringe filters.</li>
</ol>
<h3 id="sec1.2">Cell culture methodology</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="7">
<li>Prepare plain DMEM by dissolving 13.7 g of high glucose powdered media and 3.7 g of NaHCO<sub>3</sub> in 1 liter of autoclaved distilled water; ensure that there is no particulate matter or aggregates. After adjusting pH to 7.0–7.2 using 0.1 N HCl, filter the entire solution through 0.22 μm membrane filters, and store the media under sterile conditions at 4°C.</li>
<li>Prepare 500 mL solution of culture medium by mixing 450 mL of plain DMEM prepared in step 1, 50 mL of fetal bovine serum (FBS), and 1% Antibiotic Antimycotic solution. If insoluble residues or particles are present in serum, filter it through a 0.22 μm membrane filter prior to media preparation.</li>
<li>Before thawing cells, pre-warm the media in a culture dish inside a CO<sub>2</sub> incubator. Make sure not to thaw cells for more than 10–15 min. Refer to Gibco Cell Culture Basics videos available on Thermo Fisher Scientific website (<a href="https://www.thermofisher.com/il/en/home/references/gibco-cell-culture-basics.html">https://www.thermofisher.com/il/en/home/references/gibco-cell-culture-basics.html</a>).</li>
<li>Change the media every 48 h. If cell density is around 70%–80% confluent, split cells at ratios of 1:2 to 1:4.</li>
<li>All experiments should be performed when cells are between 60%–70% confluent.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>40% acrylamide solution</td>
<td>Bio-Rad</td>
<td>1610140</td>
</tr>
<tr>
<td>2% (w/v) bis-acrylamide</td>
<td>Bio-Rad</td>
<td>1610143</td>
</tr>
<tr>
<td>TEMED (N,N,N′,N′-tetramethyl ethylenediamine)</td>
<td>Bio-Rad</td>
<td>1610801</td>
</tr>
<tr>
<td>Ammonium persulfate (APS)</td>
<td>Bio-Rad</td>
<td>1610700</td>
</tr>
<tr>
<td>25% glutaraldehyde solution</td>
<td>Sigma</td>
<td>G5882</td>
</tr>
<tr>
<td>Dichlorodimethylsilane</td>
<td>Sigma</td>
<td>440272</td>
</tr>
<tr>
<td>Sulfo-SANPAH (sulfosuccinimidyl 6-(4′-azido-2′-nitrophenylamino)hexanoate)</td>
<td>Thermo Fisher Scientific</td>
<td>22589</td>
</tr>
<tr>
<td>Collagen, type I solution from rat tail</td>
<td>Sigma</td>
<td>C3867</td>
</tr>
<tr>
<td>HEPES buffer (cell culture grade)</td>
<td>Sigma</td>
<td>H4034</td>
</tr>
<tr>
<td>3-(3-Aminopropyl)triethoxysilane</td>
<td>Sigma</td>
<td>A3648</td>
</tr>
<tr>
<td>Phosphate buffered saline, pH 7.4</td>
<td>Hi-media</td>
<td>TS1101</td>
</tr>
<tr>
<td>Hydrochloric acid (HCl)</td>
<td>Sigma</td>
<td>H1758</td>
</tr>
<tr>
<td>Ethanol (ACS, ISO, Reag. Ph Eur)</td>
<td>Merck</td>
<td>100983</td>
</tr>
<tr>
<td>Dulbecco’s modified Eagle’s medium, high glucose with 4,500 mg/L glucose, L-glutamine, and sodium pyruvate</td>
<td>Sigma</td>
<td>D7777</td>
</tr>
<tr>
<td>Sodium bicarbonate</td>
<td>Sigma</td>
<td>S5761</td>
</tr>
<tr>
<td>Fetal bovine serum (US origin)</td>
<td>Hi-media</td>
<td>RM10434</td>
</tr>
<tr>
<td>Trypsin-EDTA solution 1× w/ 0.025% Trypsin and 0.01% EDTA in Dulbecco’s phosphate buffered saline</td>
<td>Hi-media</td>
<td>TCL099</td>
</tr>
<tr>
<td>Dulbecco’s phosphate buffered saline (1×)</td>
<td>Hi-media</td>
<td>TS-1099</td>
</tr>
<tr>
<td>Phenol red-free DMEM with 25 mM HEPES buffer</td>
<td>Hi-media</td>
<td>AT241</td>
</tr>
<tr>
<td>Antibiotic antimycotic solution</td>
<td>Hi-media</td>
<td>A002</td>
</tr>
<tr>
<td>Sodium hydroxide (NaOH)</td>
<td>Sigma</td>
<td>S8045</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>MDA-MB-231</td>
<td>NCCS, Pune</td>
<td>NA</td>
</tr>
<tr>
<td>HT 1080</td>
<td>NCCS, Pune</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>AR Software (Igor Pro)</td>
<td>Asylum Research</td>
<td>NA</td>
</tr>
<tr>
<td>AtomicJ</td>
<td>NA</td>
<td>(<a href="#bib4">Hermanowicz et al., 2014</a>)</td>
</tr>
<tr>
<td>Origin (Optional)</td>
<td>OriginLab Corporation</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>12 mm and 18 mm coverslip, glass slides</td>
<td>Blue Star</td>
<td>NA</td>
</tr>
<tr>
<td>Lint-free papers (Kimtech Science Kimwipes)</td>
<td>Kimberley-Clark</td>
<td>34155</td>
</tr>
<tr>
<td>Membrane filters (0.22 μm)</td>
<td>Merck</td>
<td>GVWP04700</td>
</tr>
<tr>
<td>Forceps</td>
<td>Fisher Scientific</td>
<td>12-000-128, 12-000-123</td>
</tr>
<tr>
<td>24-well cell culture plates</td>
<td>Corning</td>
<td>CLS3527</td>
</tr>
<tr>
<td>Cell culture dishes (60 mm)</td>
<td>Corning</td>
<td>CLS430166</td>
</tr>
<tr>
<td>UV lamp (365 nm)</td>
<td>Vilber Lourmat, France</td>
<td>NA</td>
</tr>
<tr>
<td>AFM probes (Silicon Nitride Cantilevers)</td>
<td>Olympus</td>
<td>TR400PB</td>
</tr>
<tr>
<td>Optically clear double-sided acrylic adhesive tape</td>
<td>THORLABS</td>
<td>OCA8146-2</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p><b>Atomic force microscopy (AFM) setup:</b> AFM instrument should be equipped with an inverted microscope with long-distance 10× or 20× objectives (<a href="#fig1">Figure 1</a>). To measure nuclear stiffness, we have used the MPF-3D-BIO (Oxford instruments) that integrates an Atomic force microscope with an inverted optical microscope. The complete setup includes the MPF-3D-BIO mounted on top of a Zeiss Axio Observer Z1 fluorescent inverted microscope (Carl Zeiss) enclosed in an acoustic isolation chamber (<a href="#fig1">Figure 1</a>). The integrated baseplate of the MPF-3D-BIO allows sub-micrometer positioning of the AFM tip on top of the sample, which is an essential feature as the tip has to be positioned on top of the nucleus during indentation to estimate the nuclear stiffness. The setup includes a top view camera (TopViewIO functionality) that gives a top view of the probe and allows laser (SLD) alignment on top of the probe. The phase-contrast illumination can be used to visualize the probe and sample (cells) and thus the probe can be positioned to appropriately on top of the cells during force measurements.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Picture of the AFM head and optical microscope from different angles with a schematic diagram depicting the setup</div>
<p>The MPF-3D-BIOTM is mounted on top of a fluorescent inverted microscope Zeiss Axiovert model number (Carl Zeiss) enclosed in an acoustic isolation chamber. Some of the important components are labeled in the picture and include (A) the MPF-3D-BIOTM AFM, (B) fluorescent inverted microscope, (a) photodiode knob, (b) front leg thumb wheel, (c) LDX thumb wheels, (d) sample stage, (e) baseplate, (f) X-plate and Y-plate knobs (moves the whole base including AFM head and sample), (g, h) sample X and Y position knobs (moves the sample plate with respect to AFM).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Polyacrylamide hydrogels (PA gels) of tunable stiffness are first prepared on glass coverslips and the top surface of these hydrogels are functionalized and coated with ECM proteins. Cells cultured on these ECM-coated PA gels are used for AFM based stiffness measurement. The underlying ECM-coated PA gel mimics the bulk stiffness of the microenvironment and can be tuned accordingly using different concentrations of acrylamide and bis-acrylamide (<a href="#bib9">Syed et al., 2017</a>; <a href="#bib10">Tse and Engler, 2010</a>). For our experiments, cells were grown on collagen-coated ~4.5 kPa PA gels and nuclear stiffness was measured by probing live cells in the culture media using an AFM cantilever. The PA gel stiffness can also be measured from the same sample by analyzing force curves obtained from a cell free zone of the PA gel.</p>
<h3 id="sec3.1">Preparation of collagen type I coated PA gels and cell seeding</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This section of the protocol documents PA gel preparation, functionalization, ECM coating, and cell culture. PA gels prepared on coverslips are functionalized with Sulfo-SANPAH and coated with ECM protein prior to seeding cells. The accompanying <a href="#mmc1">Methods Video S1</a> demonstrates the major steps for PA gel preparation and ECM coating.</p>
<ol>
<li>Take acid cleaned 12 mm or 18 mm coverslips (up to 30 coverslips per batch, depending on the number of gels to be prepared) in a large (100 mm or 150 mm) glass petri dish and rinse with UP water 2–3 times. After rinsing, aspirate the water using a vacuum pump for suctioning.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid using plastic Petri dishes in subsequent steps as APTES and glutaraldehyde used in steps 4 and 5 can erode plastic.</div>
<ol start="2">
<li>Add 0.1 M NaOH solution onto the coverslips at room temperature (RT). After 20 min of incubation, wash coverslips/coverslides with UP water 2–3 times and then dry them carefully by placing between lint-free papers. After drying, put the coverslips/coverslides back into the large glass petri dish.
<ul>
<li>NaOH treatment is important to achieve uniform spreading of gel solution on the coverslips</li>
</ul>
</li>
<li>Add APTES onto dried coverslips; make sure that enough solution (2–5 mL) is added to cover all the coverslips. APTES activates the glass surface and provides free amine groups for further crosslinking.</li>
<li>After 20 min of incubation, remove the APTES solution and then wash with UP water at least 3–4 times to ensure complete removal of unreacted free APTES. Glass coverslips/coverslides having excess unreacted APTES can react with glutaraldehyde to form orange-brown precipitates which can interfere with later microscopic techniques.</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods Video S1. A detailed video demonstrating polyacrylamide gel preparation and ECM coating for cell culture, related to steps 1–17</div>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> APTES is corrosive, harmful, and can cause skin irritation. Wear protective gloves and goggles while handling it and follow other standard safety measures as per the manufacturer’s protocol.</div>
<ol start="5">
<li>Add 0.5% glutaraldehyde solution prepared in UP water onto APTES treated coverslips for 30 min. Make sure coverslips are completely immersed in the solution. As glutaraldehyde is light-sensitive, use aluminum foil to shield the coverslips from light.
<ul>
<li>Glutaraldehyde acts a crosslinking agent which facilitates covalent bonding of gel to the glass surface.</li>
</ul>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Glutaraldehyde is corrosive, toxic, harmful to health, and an environmental hazard. Wear gloves, goggles, and other protective equipment while handling it and follow other standard safety measures as per the manufacturer's protocol. It is recommended to dilute stock glutaraldehyde inside a fume hood (maintain sterility) to a working 0.5% solution.</div>
<ol start="6">
<li>Remove the solution completely and then wash with UP water at least 3–4 times. Make sure there are no traces of glutaraldehyde.</li>
<li>Lay coverslips out on a lint-free tissue paper using forceps making sure that the functionalized surfaces face up. Gently dry with lint-free tissues. The coverslips are now functionalized and ready to be used for PA gel preparation.</li>
<li>Prepare hydrophobic glass: Take acid cleaned glass slides (75 mm × 25 mm) on a glass petri dish, rinse with water then wipe with a lint-free tissue. Bring it inside the fume hood and put 100–200 μL of dichlorodimethylsilane on top of each slide and make a thin smear using another glass slide. Lay the treated glass slide on a paper towel with the hydrophobic surface facing up next to the functionalized coverslip.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Dichlorodimethylsilane is flammable, corrosive, and toxic. Hence follow standard safety measures as per the manufacturer's protocol while handling it.</div>
<ol start="9">
<li>Aliquot required amount of Acrylamide/bis-acrylamide stock solution in a 1.5 mL tube and add 1/1000 volume TEMED and 1/100 volume 10% APS solution (for example, add 1 μL TEMED and 10 μL APS solution for 1 mL stock Acrylamide/bis-acrylamide solution). Mix the solutions using a pipette avoiding formation of air bubbles.</li>
<li>Quickly pipette the mixture as small droplets on top of the hydrophobic glass slide and overlay the functionalized coverslips using forceps. The functionalized surface of the coverslip should be in contact with the solution (<a href="#fig2">Figure 2</a>, <a href="#mmc1">Methods video 1</a>) While pipetting small droplets, dispense 10 μL solution/cm<sup>2</sup> area to prepare ~80 μm thick gels (for example, dispense 8.9 μL droplets for 12 mm coverslip and 20 μL droplets for 18 mm coverslip).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. ECM-coated polyacrylamide gel (PA gel) preparation and cell seeding</div>
<p>(A–C) Polyacrylamide gel preparation picture (A) and schematic from different angles (B and C). Gel cocktail droplets are pipetted on top of a clean hydrophobic (dichlorodimethylsilane treated) glass slide and then functionalized glass coverslips were overplayed on top in such a way that the functionalized side faces the droplet.</p>
<p>(D and E) Once polymerized, gels are transferred to 24-well plates (D) and then coated with ECM protein overnight after functionalization (E).</p>
<p>(F) Cells are seeded on the gels after removing excess ECM protein solution.</p>
</figcaption>
</figure>
</li>
<li>Allow the solution to polymerize for 20–30 min to form gels. Check the leftover solution in the tube to see if the polymerization is complete.</li>
<li>After polymerization is complete, slide off the gels from the hydrophobic slide gently using forceps and put them in separate wells in a TC culture plate (12 mm coverslips in 24 well plate and 18 mm coverslips in 12 well plate) immersed in 50 mM HEPES buffer (pH 8.5). Make sure the polymerized gels are facing up (<a href="#fig2">Figures 2</a>D and 2E).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The gels can be stored for 14 days submerged in HEPES at 4°C. In such a case 0.5% sodium azide can be added to prevent contamination. If stored in 0.5% sodium azide HEPES, then the gels have to be cleaned 3–4 times with HEPES buffer (sodium azide free) before ECM coating to remove residual azide that might interfere with cell culture.</div>
<ol start="13">
<li>Prepare 0.5 mg/mL Sulfo-SANPAH working solution in 50 mM sterile HEPES buffer (pH 8.5) from 0.1 mg/μL stock (add 5 μL Sulfo-SANPAH stock solution per every 1 mL HEPES). The working solution can also be filtered through a 0.22 μm syringe filter.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not store the diluted solution and prepare it fresh from stock immediately before the ECM coating step.</div>
<ol start="14">
<li>Aspirate HEPES from the wells containing gels and add Sulfo-SANPAH working solution (per well ~300 μL for 24 well plate, ~600 μL for 12 well plate). Make sure the gels are fully covered with sulfo-SANPAH.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Be careful while aspirating buffers and other treatment reagents from wells containing gels. Tilt the plate and aspirate liquids using an aspirator or pipette from one side and avoid touching the gel with the aspirator/pipette tips as this might damage the surface.</div>
<ol start="15">
<li>After adding Sulfo-SANPAH, put the plates containing gels under λ365 nm UV lamp for 20 min inside a sterile place (preferably cell culture hood). Make sure plates are uncovered during UV treatment.</li>
<li>Aspirate Sulfo-SANPAH and then carefully wash 3 times with 50 mM sterile HEPES buffer (pH 8.5).</li>
<li>Dilute preferred ECM protein stock in an appropriate buffer in a sterile tube. Aspirate HEPES buffer entirely from the plate after final wash and add ECM protein (~300 μL/well for 24-well plate and ~600 μL/well for 12-well plate) and incubate overnight at 4°C or 2.5 h at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For our experiment, we have used collagen as ECM protein to coat the PA gels. We dilute Collagen Type I stock solution (Sigma, Cat# C3867) to a working concentration of 33.3 μg/mL in Dulbecco’s Phosphate Buffered Saline (DPBS). We added 300 μL of this diluted collagen solution to each well in a 24-well plate (growth area ~ 2 cm<sup>2</sup>) to obtain an effective collagen coating of 5 μg/cm<sup>2</sup>.</div>
<ol start="18">
<li>After incubation, aspirate ECM solution, wash 2 times with DPBS and then add complete cell culture media for stabilizing the gels at 37°C in a CO<sub>2</sub> incubator for 0.5–1 h (<a href="#fig2">Figure 2</a>F).</li>
<li>After incubation with complete media, seed cells at the desired seeding density from a 60%–70% confluent cell culture plate. Make sure cells are healthy before seeding them on gels.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For single-cell nuclear stiffness measurements, it is important to have a seeding density such that sufficient numbers of single cells can be measured. Excess cell seeding might result in a confluent or semi-confluent layer with very few single cells that can be probed. For MDA-MB-231 and HT 1080 cells, use a seeding density of 3,000–6,000 cells/cm<sup>2</sup>.</div>
<ol start="20">
<li>Culture cells seeded on PA gels for at least 12–24 h before doing AFM measurements.</li>
</ol>
<h3 id="sec3.2">Atomic force microscopy (AFM) measurements</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–90 min</div>
<p>This section of the protocol documents AFM setup process, AFM calibration, and cell stiffness measurement. The accompanying <a href="#mmc2">Methods Video S2</a> demonstrates the major steps of this section including AFM probe attachment, AFM calibration, coverslip (with cells grown on PA gel) attachment and force curve recording process.</p>
<ol start="21">
<li>Warm phenol red-free DMEM containing 25 mM HEPES buffer by incubating it in a 37°C water bath. While incubating, setup and calibrate the AFM instrument for measurement.</li>
</ol>
<ol start="22">
<li>AFM setup and calibration: (it is advised to operate the AFM instrument under the supervision of an experienced user). The AFM instrument and the accompanying software have to be set up and the probe has to be calibrated before force measurement. The standard control might vary based on the AFM manufacturer, model, and even the software version which are usually available in the user manuals. The aim at the end of this step is to set up the AFM for force measurement on samples (cells attached on culture plate) submerged in liquid media (culture media). In MPF-3D-BIO this can be done by selecting contact mode force measurement after attaching a liquid-protective membrane to the cantilever holder. The steps taken to set up the MPF-3D-BIO are described below:
<ol type="a">
<li>Start the AFM instrument and software: A detailed documentation about the software can be found in the MFP-3D SPM User Guide, Version 13, Revision: A-1715, Chapter- 3 (<a href="#bib1">Asylum Research, 2013</a>). Start the AFM instrument and connected computer. Once the instrument is powered up, start the AR software and wait for the initialization. Once initialized, select contact mode force from the standard selection menu (mode master &gt; force &gt; contact mode force).</li>
<li>Attaching liquid protective membrane to the cantilever holder: Remove the cantilever holder (<a href="#fig3">Figure 3</a>A.d, <a href="#mmc2">Methods video 2</a>) from the AFM head and mount it to the stand (<a href="#fig3">Figure 3</a>A.e) and remove if it has any old cantilever attached to it. Remove it from the stand and attach the Closed Cell Bellows black Viton (Fluid Cell Lite, part # 112.256.01, <a href="#fig3">Figure 3</a>A.b) and O-Ring Membrane Threaded Clamp (Fluid Cell Lite, part # 112.491, <a href="#fig3">Figure 3</a>A.c) using the Spanner Wrench (Fluid Cell Lite, part # 939.008, <a href="#fig3">Figure 3</a>A.c) and the blue ball (Fluid Cell Lite, part # 290.146, <a href="#fig3">Figure 3</a>A.c). The detailed attachment steps are given in the MFP-3D SPM User Guide, Version 13, Revision: A-1715, pp. 153–156 (<a href="#bib1">Asylum Research, 2013</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. AFM probe attachment and calibration process</div>
<p>(A) Essential components used to attach the cantilever to the cantilever holder and liquid protective membrane including a closed cell bellows black Viton (Fluid Cell Lite, part # 112.256.01, labeled b), O-ring membrane threaded clamp, and spanner wrench (Fluid Cell Lite, part # 112.491 and part # 939.008, labeled c), and the blue ball (Fluid Cell Lite, part # 290.146, labeled a). The cantilever holder along with the protective membrane attachment (d) is mounted on the stand (e). Finally, a single cantilever is taken out from the box (f) and is attached to the cantilever holder using forceps (g) and screwdriver (h).</p>
<p>(B) Cantilever holder mounted on the stand after protective membrane and cantilever attachment.</p>
<p>(C) Sum and deflection meter after laser alignment and setting zero deflection.</p>
<p>(D) Laser alignment on the BL-TR400PB small probe (marked ∗). The large probe (marked #) can be used to obtain cell cortical stiffness.</p>
<p>(E) GetReal calibration steps using the inbuilt software shown as steps 1–4.</p>
</figcaption>
</figure>
</li>
<li>After attaching the protective membrane put the cantilever holder back into the stand. Attach the cantilever using forceps to the cantilever holder. Be careful not to damage the probe as it is very delicate. Check if the probe is sitting properly and carefully install the cantilever holder back to the AFM head. Make sure not to accidentally touch the probe with anything.
<div class="note">
<span class="note-title">Note:</span> It is essential to use the right kind of probe during stiffness measurement. Using too soft or too stiff cantilever might produce an unexpected and wrong outcome. Since nuclear stiffness influences cell stiffness estimates, particularly for well spread cells (<a href="#bib11">Vichare et al., 2012</a>), it is preferable to probe cortical stiffness of cells using relatively soft cantilevers with spring constant . In comparison, to minimize the influence of cortical stiffness in estimating nuclear stiffness, stiffer cantilevers with spring constants is desirable. However, these values are also dependent on the type of cells we are probing. For example, for probing cortical stiffness of very soft cells such as embryonic stem cells, it is better to use soft cantilevers with spring constant (<a href="#bib8">Sthanam et al., 2017</a>). In comparison, stiffer cells such as fibroblasts may be probed with stiff probes (<a href="#bib7">Solon et al., 2007</a>).</div>
In our experiments, to probe nuclear stiffness, we used a stiff silicon nitride cantilevers (BL-TR400PB, Olympus) which have two probes attached to each cantilever. The large probe (Spring constant: 0.02 N/m, Resonance frequency: 10 kHz ) is a softer probe that can be used to obtain cortical stiffness of the cell. We have used the smaller probe (Spring constant: 0.09 N/m, Resonance frequency: 32 kHz) to obtain nuclear stiffness. AFM probes should be stored inside a vacuum desiccator as humidity and dust affect the quality of the cantilevers.</li>
<li>Laser alignment and setting zero deflection: After attaching the probe align the deflection laser using appropriate controls. Aligning the laser (SLD) spot in MPF-3D-BIO is achieved using the LDX and LDY thumb wheels that move the laser spot in the X and Y axis respectively. First, locate Sum and Deflection meter in the AR software, then bring the laser spot near the tip of the cantilever and finally do fine adjustment so that the Sum is highest (Sum signal displays the total reflected laser light) (<a href="#fig3">Figure 3</a>D). Once the laser is aligned, set the deflection meter to zero using the Sum and Deflection meter (this moves the reflected laser beam to the center on the photodetector) (<a href="#fig3">Figure 3</a>C).</li>
<li>The thermal calibration of the probe will yield the accurate spring constant of the probe. We used the inbuild GetReal probe calibration method (Thermal tab in Master panel of the AR software) as the BL-TR400PB probe is listed as a standard probe in GetReal tab (<a href="#fig3">Figure 3</a>E). If the used probe is not listed, the Custom Rectangular Probe option can be selected for the calibration if dimensions and estimated resonance frequency of the cantilever are known. The manual thermal capture method can also be used to obtain the spring constant of the probe. Calibration yields the inverse optical lever sensitivity (InvOLS) and Spring Constant of the AFM. Signal in the photo diode is recorded in Volts. InvOLS converts this voltage signal into distance.
<div class="note">
<span class="note-title">Note:</span> The thermal calibration is to be performed when the cantilever is not touching anything, i.e., it is withdrawn from the substrate. The thermal capture method is used to obtain the cantilever's resonance frequency from which the software calculates the spring constant.</div>
</li>
<li>Calculating InvOLS in liquid culture media: the InvOLS changes when the force measurement is done in a liquid medium. The InvOLS in liquid media can be calculated from a force plot on hard surface (culture disc) obtained in liquid media. To do so, place the AFM head upside down and carefully put a few droplets of cell culture media around the probe without touching the probe (<a href="#fig4">Figure 4</a>A). Fix a 60 mm Petri dish containing 4–5 mL liquid media on the stage using double-sided tape. Gently put the head back on to the base and slowly lower the head rotating the leg wheels. Once the cantilever is completely submerged into the liquid media re-align the laser (SLD) spot similarly as mentioned in step 22.d. After re-aligning, set up the force measurement parameter by going to the force tab (in the master panel) and set start dist 0 nm, force dist 2 μm, and velocity 2 μm/s. Now engage the tip and gradually bring the cantilever down by slowly rotating the front leg thumb wheel till it touches the surface indicated by a computer beep. Now rotate very slowly till the Z voltage reaches about 70 V in the Sum and Deflection meter. Be gentle and careful as lowering it too much might break the tip. Take a single force curve by clicking the single force button in the force tab. Now click on the force plot graph and select virtual deflection and check the fit that will re-calculate InvOLS for the liquid media. The AFM is now ready to be used for sample stiffness measurement.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. AFM positioning and cantilever calibration in liquid media</div>
<p>(A) Liquid droplets are overlaid around the probe to reduce probe bending due to surface tension during gradual immersion in liquid media.</p>
<p>(B) Plate containing culture media is attached to the sample stage using clear adhesive tape which will be used for probe calibration.</p>
<p>(C and D) A picture (C) and a schematic (D) of the AFM head over the media containing the plate used for calibration in liquid media.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure the liquid media is not overflowing on top of the protective membrane as it might damage the AFM.</div>
<div class="note">
<span class="note-title">Note:</span> While submerging into the liquid media, soft cantilevers might bend slightly due to their impact with the media. In such cases, waiting for some time (2–5 min) before proceeding with the laser alignment and calibration steps might straighten the cantilever.</div>
</li>
</ol>
</li>
</ol>
<ol start="23">
<li>Fixing gels on a plate for AFM: Attach a small piece of double-sided tape in the middle of a 60 mm cell culture disc inside the culture hood. Take the cell culture plate out from the incubator inside the hood. Using sterile fine forceps, lift the PA gel coverslip on which cells are seeded, gently tap the bottom on a lint-free tissue, and fix the coverslip on to the disc with the adhesive tape. Then pour 5–6 mL with warm phenol red-free DMEM containing 25 mM HEPES buffer and take it to the AFM room for measurement (<a href="#mmc2">Methods video 2</a>).</li>
<li>Mounting the cell culture disc: Withdraw the AFM probe from the surface and move up the head by rotating the thumbwheel. Very gently lift the AFM head and remove the media containing plate (from liquid InvOLS calibration, step 22.f), and fix the cell culture disc at the same place with double-sided tape. Now put the AFM head back to the base and slowly submerge the probe again by rotating the thumbwheel. Be careful not to touch the probe with anything and damage it.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure the liquid media is not overflowing on top of the protective membrane as it might damage the AFM. Using a pipette remove excess media from the plate if needed and then put the AFM head back.</div>
<ol start="25">
<li>Setting up the acquisition parameters: Set the acquisition parameters accordingly. For our experiment, we used the following: force distance (i.e., indentation depth) as 10 μm, tip velocity as 3–5 μm/s, and trigger point as 2.0 V. While lower tip speeds lead to an increase in acquisition time and may cause cell rearrangements and/or change in cell position, higher tip speeds may cause damage to cells and also lead to fewer data-points.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The default trigger point is set 1.0 V for cortical stiffness measurement. For nuclear stiffness measurements, it is recommended to use a trigger point of 2.0 V. For soft nuclei, we recommend using a lower value (~1.5 V). 10 μm indentation depth is sufficient to indent the nucleus through the cortex as most of the cells, when attached to the substrate, have a relatively thin cortex on top of the nucleus (2–4 μm). Due to the same reason, only the first 0.5–1 μm indentation data are fit during cortical stiffness measurements.</div>
<ol start="26">
<li>Positioning the AFM cantilever on top of the cell: Engage the cantilever and gradually lower the head till the cantilever reaches the sample (as the probe touches the surface is indicated by computer beep or change in Z voltage). Switch to the phase-contrast microscope eyepiece or camera to locate the probe. Move the whole base by rotating X-plate and Y-plate till the cantilever is visible in the eyepiece/camera. Now withdraw the cantilever from the surface and bring a cell nucleus below the cantilever using the sample X and Y position knobs. The X-plate and Y-plate move the whole base including the AFM head while the X and Y position knobs only move the sample plate.</li>
<li>Once the cantilever is positioned on top of the cell nucleus and all the parameters are set, take a single force curve by clicking the single force button on the master panel force tab. The obtained force curve is usually automatically saved within the assigned directory as a .ibw file (IGOR Pro binary wave file). Withdraw the tip from the surface and position it on top of the next cell nucleus by rotating X and Y position knobs and take a force curve again for the second cell. Repeat these steps for the desired number of nuclei (50–100 nuclei per condition). Occasionally, the cantilever might have to be moved up or down using the front leg thumb wheel.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is recommended to complete the measurements of each sample within 10–15 min as cells might undergo morphological changes when long term measurements are done at room temperature. For a rather long-time measurement or temperature-sensitive cells, an AFM on-stage incubator should preferably be used.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Force curves can also be recorded from a cell free part of the PA gel that can be analyzed to obtain PA gel stiffness (<a href="#fig5">Figure 5</a>C).</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods Video 2. Video demonstrating the steps for nuclear stiffness measurement using atomic force microscope (AFM), related to steps 22–27</div>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. AFM cantilever positioning on top of the nucleus and nuclear indentation</div>
<p>(A and B) Picture (A, left) and schematic (A, right) of sample plate preparation for attachment to the AFM sample stage (B) Cells growing on top of polyacrylamide gels (PA gels) are carefully picked up using forceps and attached to the plate using adhesive tape with gel and cells facing upwards. Liquid media is gently poured on top.</p>
<p>(C) A picture (top) and a schematic (bottom) showing the positioning of the probe on top of the nucleus. The cell nucleus is marked with dotted lines in the phase-contrast image. The left and the right panels show the position of the cantilever before and after positioning the probe on top of the nucleus. Force curve can be recorded from cell free zones (marked ∗) to accurately measure PA gel stiffness.</p>
<p>(D) Schematic of the probe movement during single force measurement when positioned on top of the cell nucleus.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> If the same sample is used for measuring PA gel stiffness, avoid recording force curves at the immediate neighborhood of cells to eliminate the effect of cell traction mediated strain stiffening on PA gel stiffness measurements.</div>
<ol start="28">
<li>Saving the force curve: While doing single force measurements each obtained force curve is saved within the assigned directory as a .ibw file (IGOR Pro binary wave file). The .ibw files can later be opened using the AR software (Igor Pro) and other AFM data analysis software like AtomicJ (See analysis section). The data can be exported to other formats as well.</li>
</ol>
<h3 id="sec3.3">Data analysis to obtain stiffness</h3>
<ol start="29">
<li>The obtained AFM force data can be analyzed using AtomicJ software: AtomicJ is an open-source software developed by <a href="#bib4">Hermanowicz et al. (2014</a>) that can be used for analysis of AFM force curves. We encourage the user to go through the published paper and documentation before analyzing data using this software. Below is a very short description of the analysis steps:
<ol type="a">
<li>Open the software and click on the process force curve and maps button (gear icon) to open the processing assistant. Click the "add" button to add force curve (.ibw or .csv files, using .ibw file allows the software to automatically read some parameters like spring constant and tip sensitivity) and browse to the folder containing force curve data and select curves (multiple force curves can be selected at once).</li>
<li>After selecting the force curve, click select the "next" button at the bottom to go to the processing assistant. Under the "general" tab, select a suitable batch name. For all the other parameters, we encourage the user to read the documentation first. We selected the parameters as follows: Processing as automatic, contact estimator as classical golden, Estimation method as Based on the contact model, Model fit as Classical (L2), Fit as Approach. In the model section, the appropriate model and the parameters are to be selected accordingly. We used a Pyramidal probe (cantilever) and the Half-angle was 35° (usually available from the cantilever manufacturer). Detailed documentation of all the different models offered in the AtomicJ software is available in the literature and the cited paper (<a href="#bib4">Hermanowicz et al., 2014</a>) and can help with accurate model selection. The pyramid model can be used for pyramidal probes like BL-TR400PB that was used in this protocol. The Poisson ratio for most adherent cells is set to 0.5 (<a href="#bib5">MacKay and Kumar, 2012</a>), however, it might vary in some cases (<a href="#bib5">MacKay and Kumar, 2012</a>). Curve baseline degree can be used for baseline correction. In the calibration section, the Spring constant and Sensitivity (InvOLS) can be entered manually, or when the .ibw file is loaded, it can automatically be read from the file by clicking Read-in. Go to the "advanced" tab and click on the infinity symbol () in Max load and Max indent (Constrain parameters). Max indent can be changed if the indentation data are fit for a certain indentation depth (for example, 0.5–1 μm data are usually fit to get cell cortical stiffness (<a href="#bib2">Das et al., 2017</a>)). However, for nuclear stiffness the entire indentation data are fit, i.e., Max indent is set to (). In the "output" tab, different graphs can be selected.</li>
<li>Optional step (when multiple batches are to be analyzed all at once): After clicking on the Processing settings, click the “next” batch button to add the next batch and add parameters accordingly after adding the batch name.</li>
<li>Finally, upon clicking the “finish” button, the software will analyze all the force curves and give you the results as two separate files. The "processing result" contains all the stiffness measurements, and can be saved as .csv or .tsv file from (from file &gt; save as option or by clicking Cmd+s/Ctrl+s). The "graphical result" contains different graphs including indentation and data fit curves and can also be saved in different formats.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>At the end of performing the protocol, the user will obtain single-cell force curves that have to be analyzed properly (Data analysis section). Analyzing the data will give cell nucleus stiffness along with data fit. Hence this protocol can be used to obtain nuclear stiffness across different conditions including different cell types, drug treatments, or other perturbations. Sample stiffness measurement for MDA-MB-231 (breast cancer) and HT 1080 (fibrosarcoma) is given in <a href="#fig6">Figure 6</a>. Further examples of nuclear property alterations as a result of drug treatment and its pathophysiological relevance can be found here (<a href="#bib3">Das et al., 2019</a>; <a href="#bib6">Mukherjee et al., 2020</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/430-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Quantification of nuclear stiffness of MDA-MB-231 and HT 1080 cells grown on 5 kPa polyacrylamide gels coated with collagen (5 μg/cm<sup>2</sup>)</div>
<p>(A and B) Representative force curves highlighting probe approach, withdraw, contact point, and fit (A), and the corresponding indentation and data fits (B).</p>
<p>(C) Boxplot showing nuclear stiffness of MDA-MB-231 and HT 1080 cells cultured on collagen-coated polyacrylamide gels.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>AFM force measurement is prone to error especially when the probe is not positioned properly on top of the cell nucleus. Hence it is recommended to obtained single force measurement data from 50–100 cells for each condition and then remove outliers using statistical tests like the Tietjen-Moore test or Generalized ESD Test. In general, outliers in cortical stiffness measurements may correspond to cases where well spread cells are probed at the cell periphery where the thickness may be or when experiments are performed with cells cultured on soft and thin gels (<a href="#bib12">Vichare et al., 2014</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol described here allows measurement of nuclear stiffness of adherent cells growing on ECM-coated polyacrylamide gels of tunable stiffness. This protocol eliminates the fluid cell and temperature controller with a rather quick and simple setup where the cell culture disc is directly fixed to the sample stage using double-sided tape. Since temperature is not maintained for a prolonged duration in this setup, all the measurements are to be taken within the initial 10–15 min. For temperature-sensitive cells, the fluid cell and the temperature controller have to be mounted. This protocol also requires the nucleus to be properly positioned under the AFM cantilever. Hence, this protocol cannot be used when the cell size is lesser than 10 μm, making it difficult to position the nucleus under the cantilever. The use of a narrow probe with a similar spring constant can overcome this problem to some extent. The cells also have to be sufficiently attached to the substrate and hence the protocol is applicable for adherent cells only and cannot be used for non-adherent cells.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Cells grown on PA gels are contaminated</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Cells cultured on PA gels can be contaminated due to contamination during gel preparation and coating. To test whether the contamination is due to the gel preparation issue, some cells can be seeded to an empty cell culture well (has no PA gels). If cells grown on empty culture well is also contaminated then the contamination can be because of cell handling issue.</p>
<p>Multiple steps can be taken to avoid contamination. Make sure reagents including HEPES buffer are properly sterilized with a syringe filter. Used UP water can be sterilized via autoclaving for subsequent use. Before the functionalization of PA gels (Sulfo-SANPAH treatment in step 13), gels can be incubated in HEPES buffer containing 0.5% sodium azide for 30 min for sterilization. This step prevents possible contaminations from gel preparation steps. However, after the treatment, it is important to maintain sterility. Also, the gels have to be washed with sterile 50 mM HEPES buffer (pH 8.5) inside the hood to completely remove azide. Also, make sure the diluted Sulfo-SANPAH working solution and the ECM protein used for coating PA gels are sterile. The diluted Sulfo-SANPAH solution can be sterilized using a syringe filter.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Cells are not attached/well spread on PA gels</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Cells grown on PA gels (especially on soft PA gels) will likely have a smaller spread-area compared to that on cell culture dishes, as PA gels are softer. However, low attachment and very small-spread areas can also be due to low abundance or even absence of ligand on the PA gel surface. Inappropriate functionalization can contribute to less ligand or even lack of ligand on the surface.</p>
<p>Make sure the used Sulfo-SANPAH and ECM protein solutions are properly prepared. Over-diluted ECM protein might result in reduced ligand attachment. Sulfo-SANPAH is a very unstable compound, hence inappropriately stored stock solution might also result in inappropriate functionalization. Also, make sure that the gels are fully covered in the Sulfo-SANPAH solution and are submerged and not floating in the liquid. It is also important to make sure that the gels are fully exposed to UV light during the Sulfo-SANPAH incubation step (Protocol step 15).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The nucleus cannot be located using a phase-contrast microscope</p>
<h3 id="sec7.6">Potential solution</h3>
<p>If the nucleus is not visible using the phase-contrast microscope, then fluorescent live cell nuclei stain like Hoechst 33342 can be used to stain the cell nucleus before attaching it to the stage. In such a scenario, it is important to ensure that the staining does not impact the nucleus stiffness as that might alter the measurements.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Asylum Research (2013). MFP 3D User Guide (Asylum Research). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref1">View at publisher</a></p>
<p id="bib2">Das, A., Monteiro, M., Barai, A., Kumar, S., and Sen, S. (2017). MMP proteolytic activity regulates cancer invasiveness by modulating integrins. Sci. Rep. <i>7</i>, 1-13. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref2">View at publisher</a></p>
<p id="bib3">Das, A., Barai, A., Monteiro, M., Kumar, S., and Sen, S. (2019). Nuclear softening is essential for protease-independent migration. Matrix Biol. <i>82</i>, 4-19. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref3">View at publisher</a></p>
<p id="bib4">Hermanowicz, P., Sarna, M., Burda, K., and Gabryś, H. (2014). AtomicJ: an open source software for analysis of force curves. Rev. Sci. Instrum. <i>85</i>, 063703. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref4">View at publisher</a></p>
<p id="bib5">MacKay, J.L. and Kumar, S. (2012). Measuring the elastic properties of living cells with atomic force microscopy indentation. Methods Mol. Biol. <i>931</i>, 313-329. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref5">View at publisher</a></p>
<p id="bib6">Mukherjee, A., Barai, A., Singh, R.K., Yan, W., and Sen, S. (2020). Nuclear plasticity increases susceptibility to damage during confined migration. PLoS Comput. Biol. <i>16</i>, e1008300. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref6">View at publisher</a></p>
<p id="bib7">Solon, J., Levental, I., Sengupta, K., Georges, P.C., and Janmey, P.A. (2007). Fibroblast adaptation and stiffness matching to soft elastic substrates. Biophys. J. <i>93</i>, 4453-4461. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref7">View at publisher</a></p>
<p id="bib8">Sthanam, L.K., Barai, A., Rastogi, A., Mistari, V.K., Maria, A., Kauthale, R., Gatne, M., and Sen, S. (2017). Biophysical regulation of mouse embryonic stem cell fate and genomic integrity by feeder derived matrices. Biomaterials <i>119</i>, 9-22. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref8">View at publisher</a></p>
<p id="bib9">Syed, S., Schober, J., Blanco, A., and Zustiak, S.P. (2017). Morphological adaptations in breast cancer cells as a function of prolonged passaging on compliant substrates. PLoS One <i>12</i>, e0187853. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref9">View at publisher</a></p>
<p id="bib10">Tse, J.R. and Engler, A.J. (2010). Preparation of hydrogel substrates with tunable mechanical properties. Curr. Protocol. Cell Biol. <i>47</i>, 10.16.1-10.16.16. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref10">View at publisher</a></p>
<p id="bib11">Vichare, S., Inamdar, M.M., and Sen, S. (2012). Influence of cell spreading and contractility on stiffness measurements using AFM. Soft Matter <i>8</i>, 10464-10471. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref11">View at publisher</a></p>
<p id="bib12">Vichare, S., Sen, S., and Inamdar, M.M. (2014). Cellular mechanoadaptation to substrate mechanical properties: contributions of substrate stiffness and thickness to cell stiffness measurements using AFM. Soft Matter <i>10</i>, 1174-1181. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00003-4/sref12">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact Shamik Sen (<a href="mailto:shamiks@iitb.ac.in">shamiks@iitb.ac.in</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate any unique dataset or code.</p>
<h3>Acknowledgments</h3>
<p>The authors acknowledge financial support from the <a href="https://doi.org/10.13039/501100001409">Department of Science and Technology</a> (Government of India) (DST/SJF/LSA-01/2016-17). A.D. was supported by fellowship from the Department of Biotechnology (Government of India). The authors would like to thank IRCC, IIT Bombay for providing the Bio-AFM facility.</p>
<h3>Author Contributions</h3>
<p>A.B., A.D., and S.S. conceived the idea; A.B. and A.D. executed the idea; A.B., A.D., and S.S. wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>