<article>
<section>
<h1 id="header">Dissociation of intact adult mouse cortical projection neurons for single-cell RNA-seq</h1>
<p><time datetime="2021-11-26">Published: November 26, 2021</time></p>
<p>Noa Golan<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> and William B. Cafferty<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Neurology, Yale University School of Medicine, New Haven, CT 06520, USA</p>
<p id="aff2"><sup>2</sup>Interdepartmental Neuroscience Program, Yale University, New Haven, CT 06520, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:noa.golan@yale.edu">noa.golan@yale.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:william.cafferty@yale.edu">william.cafferty@yale.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100941">10.1016/j.xpro.2021.100941</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol provides an improved pipeline for dissociating intact projection neurons from adult mouse cortex for applications including droplet and plate-based single-cell RNA sequencing, qPCR, immunocytochemistry, and long-term <i>in vitro</i> cell culture. This protocol provides a robust and reproducible dissociation pipeline that uses exclusively off-the-shelf reagents, not requiring the use of expensive dissociation kits. The unique incubation steps, in combination with the FACS gating strategy, results in unparalleled enrichment for intact cortical neurons from the adult brain.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1101/2021.2006.2002.446653">Golan et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol to dissociate neonatal and adult mouse cortical projection neurons for scRNA-seq</li>
<li>Gentle dissociation ensures reliable plasma membrane and cytoplasmic integrity</li>
<li>Neuronal subtype selection via fluorescent viral labeling and FACS</li>
<li>Uses exclusively off-the-shelf reagents, rather than expensive dissociation kits</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1196-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Viral tracing methods are powerful tools in the functional dissection of neuronal circuits. Precision labeling of anatomically defined subsets of neuronal pathways allows for exploring the molecular mechanism that build, refine, and define these circuits under homeostatic and pathological conditions. While cortical projection neurons are well labeled via retrograde AAV uptake, maintaining an intact soma during dissociation has proved difficult due to the large amount of membrane devoted to their axons, which are lost during the dissociation procedure. Here we provide step-by-step instructions to dissociate adult neurons from primary motor cortex that were traced from the spinal cord using retrograde AAVs (<a href="#bib9">Wang et al., 2018</a>; <a href="#bib6">Saunders et al., 2018</a>; <a href="#bib7">Tasic et al., 2018</a>; <a href="#bib8">Tervo et al., 2016</a>) that maintains cytosplasmic intregrity of these neurons for comprehensive mRNA sequencing analysis. Additionally, we have also used this protocol to isolate cortical neurons from adult transgenic mice that express a floxed stop codon 5′ to TdTomato to isolate intrinsically fluorescent and nonfluorescent neurons.</p>
<p>We recommend preparing all reagents the night before the experiment to allow solutions to be at the correct temperature for the following procedures.</p>
<h3 id="sec1.1">Preparation of pasteur pipette tips</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>Fire polish salinized pipettes tips using an oil burner flame to a final internal diameter of 600 μm, 300 μm and 150 μm (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1196-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Preparation of pasteur pipettes</div>
<p>(A) Suspend Pasteur pipette under an oil burner.</p>
<p>(B) Allow flame to heat the pipette tip.</p>
<p>(C) Measure the internal diameter using a calibration slide under a dissection microscope.</p>
<p>Scale bar = 200 μm.</p>
</figcaption>
</figure>
<ol type="a">
<li>Measure tip diameters using a calibration slide under a dissection microscope at 20X magnification.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Measure the internal diameter of flamed tips every 10 s of heat exposure to achieve the desired diameters. These times may depend on the temperature of the flame and will require optimization to avoid sealing the tip entirely.</div>
<div class="note">
<span class="note-title">Note:</span> Make fresh tips for each dissociation experiment.</div>
<h3 id="sec1.2">Preparation of artificial cerebrospinal fluid (aCSF)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<ol start="2">
<li>Prepare the aCSF
<ol type="a">
<li>Prepare the 1M stock solutions (following the recipe in “<a href="#materials-and-equipment">materials and equipment</a>”).</li>
<li>Add reagents to 500 mL of water to make aCSF. Do not add the HCl.</li>
</ol>
</li>
<li>Carbogenate the aCSF for 10 min.</li>
<li>pH the carbogenated aCSF and add HCl until the pH=7.3</li>
<li>Chill in the fridge until the aCSF is at 4°C, ∼3 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Carbogenate the aCSF before adding the HCl. The carbogen changes the pH of the solution.</div>
<h3 id="sec1.3">Preparation of enzyme solution for tissue dissociation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="6">
<li>Warm dissociation buffer (3 mL per sample; the recipe is in “<a href="#materials-and-equipment">materials and equipment</a>”) to 34°C.</li>
<li>Activation of papain
<ol type="a">
<li>Add 5 mL of warmed dissociation buffer into a vial of papain to a concentration of 26 units/mL and allow to activate at 34°C for 30 min.</li>
<li>When ready to dissociate, dilute papain solution 1:2 in dissociation buffer. Each sample should digest in 3 mL of total solution in a 5 mL microfuge tube.</li>
</ol>
</li>
</ol>
<h3 id="sec1.4">Setting up for the dissociation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="8">
<li>Prior to beginning the experiment, set up two petri dishes filled with aCSF. One of them on ice, one of them under a fluorescent dissection microscope.</li>
<li>Using ethanol-wiped binder clips, clip the tubing to each culture dish, ensuring that the aCSF gets continuously carbogenated throughout the entire experiment.</li>
<li>Set up the perfusion pump with clean tubing and run cold aCSF through the tubing in preparation for the perfusion.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the tubing for the perfusion pump has never been used to perfuse paraformaldehyde or any other fixative.</div>
<div class="critical">
<span class="critical-title">Critical:</span> See <a href="#fig2">Figure 2</a> for an example set-up that ensures all solutions remain cold and carbogenated.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1196-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Dissociation set-up</div>
<p>Proximity of all equipment is critical to ensure brevity of protocol to maintain health of dissociated neurons. All surfaces should be wiped with 70% ethanol.</p>
</figcaption>
</figure>
<h3 id="sec1.5">Pre-cooling the centrifuge</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="11">
<li>Prior to beginning the experiment, set centrifuge to cool to 4°C and ensure 15 mL tube inserts are installed.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>rAAV-CAG-GFP</td>
<td>Addgene</td>
<td>37825-AAVrg</td>
</tr>
<tr>
<td>rAAV-CAG-tdTomato</td>
<td>Addgene</td>
<td>59462-AAVrg</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Water</td>
<td>Milli-Q</td>
<td>N/A</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>J.T. Baker</td>
<td>1332–01</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>J.T. Baker</td>
<td>3818–01</td>
</tr>
<tr>
<td>KCl</td>
<td>Sigma-Aldrich</td>
<td>746436</td>
</tr>
<tr>
<td>sodium L-ascorbate</td>
<td>ChemCruz</td>
<td>sc215877A</td>
</tr>
<tr>
<td>Taurine</td>
<td>Alfa Aesar</td>
<td>A12403</td>
</tr>
<tr>
<td>Thiourea</td>
<td>Alfa Aesar</td>
<td>A12828</td>
</tr>
<tr>
<td>Glucose</td>
<td>Sigma-Aldrich</td>
<td>G5767</td>
</tr>
<tr>
<td>HEPES</td>
<td>AmericanBio</td>
<td>AB00892</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>
</td>
<td>AmericanBio</td>
<td>AB09013</td>
</tr>
<tr>
<td>myo-inositol</td>
<td>Sigma</td>
<td>15125</td>
</tr>
<tr>
<td>
<i>N</i>-acetylcysteine</td>
<td>Acros</td>
<td>160280250</td>
</tr>
<tr>
<td>N-Methyl-D-Glucamine (NMDG)</td>
<td>mpbio</td>
<td>191506</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>J.T. Baker</td>
<td>3506–01</td>
</tr>
<tr>
<td>Sodium Pyruvate</td>
<td>Gibco</td>
<td>11360070</td>
</tr>
<tr>
<td>Trehalose</td>
<td>Alfa Aesar</td>
<td>A19434</td>
</tr>
<tr>
<td>HCL</td>
<td>J.T. Baker</td>
<td>953500</td>
</tr>
<tr>
<td>Na<sub>2</sub>SO<sub>4</sub>
</td>
<td>Honeywell</td>
<td>239313</td>
</tr>
<tr>
<td>K<sub>2</sub>SO<sub>4</sub>
</td>
<td>Sigma Aldrich</td>
<td>P0772</td>
</tr>
<tr>
<td>MgC<sub>l2</sub>
</td>
<td>AmericanBio</td>
<td>AB09006-00100</td>
</tr>
<tr>
<td>Ovomucoid Protease Inhibitor</td>
<td>Worthington Biochemical Corporation</td>
<td>LK003182</td>
</tr>
<tr>
<td>Papain</td>
<td>Worthington Biochemical Corporation</td>
<td>LK003176</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>AmericanBio</td>
<td>AB01088</td>
</tr>
<tr>
<td>Phosphate Buffered Saline</td>
<td>Sigma</td>
<td>P4417-50TAB</td>
</tr>
<tr>
<td>Cytoplasmic Dye</td>
<td>Invitrogen</td>
<td>L34974</td>
</tr>
<tr>
<td>Laminin</td>
<td>Sigma-Aldrich</td>
<td>L2020</td>
</tr>
<tr>
<td>Poly-D-Lysine</td>
<td>Sigma-Aldrich</td>
<td>P7405</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>C57BL/6, Male and Female mice, P56</td>
<td>The Jackson Laboratory</td>
<td>000664</td>
</tr>
<tr>
<td>Ai14, Male and Female mice, P56</td>
<td>The Jackson Laboratory</td>
<td>007908</td>
</tr>
<tr>
<td>Rbp4 Cre, Male and Female mice, P56</td>
<td>Gifted from Dr. David Berson, Brown University</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Carbogen gas (95% O2 and 5% CO2)</td>
<td>Airgas</td>
<td>n/a</td>
</tr>
<tr>
<td>Petri Dish 100 X 20mm</td>
<td>Corning</td>
<td>353003</td>
</tr>
<tr>
<td>Isoflurane</td>
<td>Henry Schein NDC</td>
<td>11695-6776-2</td>
</tr>
<tr>
<td>Adult Mouse Brain Matrix, 0.5mm</td>
<td>Braintree Scientific, Inc.</td>
<td>BS-SS 505C</td>
</tr>
<tr>
<td>Tygon S3 E-3603 Bio-based Tubing for the carbogen</td>
<td>Saint-Gobain</td>
<td>ACF00001</td>
</tr>
<tr>
<td>Razor Blades</td>
<td>Personna</td>
<td>94-120-71</td>
</tr>
<tr>
<td>Cell Sorter</td>
<td>Sony</td>
<td>SH800</td>
</tr>
<tr>
<td>130 μm Microfluidic sorting Chip</td>
<td>Sony</td>
<td>LE-C3213</td>
</tr>
<tr>
<td>Flow Cytometry polystyrene particle standard, 8μm–12.9μm</td>
<td>Sphereotech</td>
<td>PPS-5</td>
</tr>
<tr>
<td>Flow Cytometry polystyrene particle standard, 13μm–17.9μm</td>
<td>Sphereotech</td>
<td>PPs-6</td>
</tr>
<tr>
<td>Perfusion Pump</td>
<td>Merck Millipore</td>
<td>XX80EL004</td>
</tr>
<tr>
<td>High-Performance Precision Pump Tubing</td>
<td>Masterflex</td>
<td>96410–15</td>
</tr>
<tr>
<td>9 Inch Silanized Glass Pasteur Pipette</td>
<td>BrainBits</td>
<td>FPP</td>
</tr>
<tr>
<td>Calibration slide</td>
<td>Motic</td>
<td>1101002300142</td>
</tr>
<tr>
<td>Eppendorf Tubes (5 mL)</td>
<td>Eppendorf</td>
<td>0030119460</td>
</tr>
<tr>
<td>Stovall Belly Dancer Shaker</td>
<td>Stovall Life Science, Inc.</td>
<td>n/a</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Eppendorf</td>
<td>5810 R</td>
</tr>
<tr>
<td>Epifluorescent Dissecting Microscope</td>
<td>ZEISS</td>
<td>495015-0008-000</td>
</tr>
<tr>
<td>5 mL polystyrene round-bottom tube with cells-strainer cap</td>
<td>Falcon</td>
<td>352235</td>
</tr>
<tr>
<td>Amnis ImageStream</td>
<td>Luminex</td>
<td>Mk II</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Artificial Cerebrospinal Fluid (aCSF) stock solutions</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>1M</td>
<td>1.47 g in 10 mL of water</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1M</td>
<td>1.19 g in 10 mL of water</td>
</tr>
<tr>
<td>KCl</td>
<td>1M</td>
<td>1 g in 13.43 mL of water</td>
</tr>
<tr>
<td>sodium L-ascorbate</td>
<td>1M</td>
<td>1.98 g in 10 mL of water</td>
</tr>
<tr>
<td>Taurine</td>
<td>1M</td>
<td>1.25 g in 10 mL of water ∗∗Heat up to get in solution</td>
</tr>
<tr>
<td>thiourea</td>
<td>1M</td>
<td>1 g in 13.14 mL of water</td>
</tr>
</tbody>
</table>
<p class="table-legend">Make stock solutions fresh for each preparation. Do not store more than 24 h. Store at room temperature (20°C–22°C).</p>
<table id="undtbl2">
<caption>Artificial Cerebrospinal Fluid (aCSF)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Water</td>
<td>n/a</td>
<td>Up to 500mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> (1M)</td>
<td>0.5mM</td>
<td>250 μL of 1M solution</td>
</tr>
<tr>
<td>Glucose</td>
<td>25mM</td>
<td>2.254g</td>
</tr>
<tr>
<td>HEPES</td>
<td>20mM</td>
<td>2.383g</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> (1M)</td>
<td>10mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub> (1M)</td>
<td>1.25mM</td>
<td>625μL of 1M solution</td>
</tr>
<tr>
<td>myo-inositol</td>
<td>3mM</td>
<td>0.27g</td>
</tr>
<tr>
<td>
<i>N</i>-acetylcysteine</td>
<td>12mM</td>
<td>0.97g</td>
</tr>
<tr>
<td>NMDG</td>
<td>96mM</td>
<td>9.3g</td>
</tr>
<tr>
<td>KCl (1M)</td>
<td>2.5mM</td>
<td>1.25 mL of 1M solution</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>25mM</td>
<td>1.05g</td>
</tr>
<tr>
<td>sodium L-ascorbate (1M)</td>
<td>5mM</td>
<td>2.5 mL of 1M solution</td>
</tr>
<tr>
<td>sodium pyruvate (100 mM)</td>
<td>3mM</td>
<td>15 mL</td>
</tr>
<tr>
<td>taurine (1 M)</td>
<td>0.01 mM</td>
<td>5μL of 1M solution</td>
</tr>
<tr>
<td>thiourea (1M)</td>
<td>2mM</td>
<td>1 mL of 1M solution</td>
</tr>
<tr>
<td>Trehalose (13.2 mM)</td>
<td>13.2mM</td>
<td>2.5g</td>
</tr>
<tr>
<td>HCl (96 mM)<sup>a</sup>
</td>
<td> </td>
<td>Add until pH = 7.3</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make aCSF fresh for each preparation. Do not store more than 24 h. Store at 4°C.<br> <sup>a</sup> Carbogenate for 10 min before adding HCl as the carbogen will change the pH of the solution</p>
<table id="undtbl3">
<caption>Dissociation buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Water</td>
<td>n/a</td>
<td>Up to 100mL</td>
</tr>
<tr>
<td>Na<sub>2</sub> SO<sub>4</sub> (82mM)</td>
<td>82mM</td>
<td>1.17g</td>
</tr>
<tr>
<td>K<sub>2</sub> SO<sub>4</sub> (30mM)</td>
<td>30mM</td>
<td>0.52g</td>
</tr>
<tr>
<td>HEPES (10 mM)</td>
<td>10 mM</td>
<td>0.24g</td>
</tr>
<tr>
<td>Glucose (10mM)</td>
<td>10mM</td>
<td>0.18g</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1M)</td>
<td>5mM</td>
<td>0.5 mL of 1M solution</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make this solution fresh for each preparation. Do not store for more than 24 h. Store at 4°C. Warm to 34°C prior to adding papain.</p>
<table id="undtbl4">
<caption>Stop solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dissociation Buffer</td>
<td>n/a</td>
<td>5mL</td>
</tr>
<tr>
<td>Ovomucoid Protease Inhibitor</td>
<td>n/a</td>
<td>5mg</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>n/a</td>
<td>10mg</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make this solution fresh for each preparation. Do not store for more than 24 h. Store at 4°C.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Perfusion and region of interest (ROI) dissection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<div class="note">
<span class="note-title">Note:</span> Adult mice that have been injected with retrograde AAVs to label cortical projection neurons in the region of interest (e.g., primary motor cortex) or adult transgenic mice with fluorescence-labeled cortical neurons have been used in this protocol.</div>
<ol>
<li>Perfusion
<ol type="a">
<li>Euthanize mice (postnatal day 28 and older, male and female) with isoflurane (open drop method), if using neonatal mice, euthanize on ice.</li>
<li>Transcardially perfuse with cold, carbogenated aCSF for 3 min at a rate of 3.75 mL/min.</li>
<li>Dissect out the entire brain and place in one of the petri dishes filled with aCSF and continuously bubbled with carbogen on ice for 3 min.</li>
</ol>
</li>
<li>Dissection
<ol type="a">
<li>Place the cold brain in the adult brain matrix and cut 500 μm thick sections through primary motor cortex using a fresh razor blade.
<ol type="i">
<li>Immediately place sections in cold carbogenated aCSF under the epifluorescent dissecting microscope.</li>
</ol>
</li>
<li>Under an epifluorescent dissecting microscope, macro dissect the regions of interest based on the location of fluorescent cell bodies as shown in <a href="#fig3">Figure 3</a>.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1196-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Tissue macro dissection</div>
<p>(A) 500 μm thick sections of cortex were cut using the adult brain matrix and placed in a petri dish with carbogenated aCSF.</p>
<p>(B) One section of cortex was visualized under epifluorescence. The area in the white box was dissected.</p>
<p>(C). High magnification of the area in the box from (B), showing individual cell bodies and axonal projections of corticospinal neurons.</p>
<p>Scale bar = 2 mm, 500 μm, 50 μm.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This step must be completed quickly to avoid neuronal death.</div>
<div class="note">
<span class="note-title">Note:</span> Dissociate an extra piece of tissue that does not contain your region of interest but is from a comparable location in cortex. This will serve as a negative fluorescent control for FACS. For our experiment, we dissected out primary motor cortex from a wildtype mouse that did not receive any viral labeling.</div>
<h3 id="sec3.2">Tissue digestion and manual dissociation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Variable depending on age of mouse, up to 70 min</div>
<ol start="3">
<li>Transfer dissected tissue to 5 mL microfuge tubes containing 3 mL enzyme solution (dissociation buffer + Papain).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This is the only solution to contain Papain.</div>
<ol start="4">
<li>Place 5 mL tubes on a tube rack and secure the rack onto an orbital shaker in an incubation oven set to 34°C.
<ol type="a">
<li>To calculate incubation time, add 15 min to the animal’s age in days to a max of 70 min). E.g., postnatal day 28 incubate for 43 min, postnatal day 56 incubate for 70 min.</li>
</ol>
</li>
<li>Remove papain solution with a P1000 pipette, allowing the tissue chunks to remain at the bottom of the tube. Add 2 mL of Stop solution and place on ice and incubate for 5 min. Do not shake or mix the solution at this step.</li>
<li>Remove stop solution with a P1000 pipette and replace with 800μL of cold dissociation buffer.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> for this step, ensure that the dissociation buffer is cold and does not contain any papain.</div>
<ol start="7">
<li>Triturate sample with fire-polished pipettes (<a href="#mmc1">Methods video S1</a>).
<ol type="a">
<li>Start with pipette with a diameter of 600 μm. Triturate 10 times.</li>
<li>Triturate 10 times with the pipette of 300 μm diameter.</li>
<li>Triturate 6 times with the pipette of 150 μm diameter.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Triturate very carefully to avoid air bubbles.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Tips should be thoroughly flushed with water and then dissociation buffer between samples.</div>
<ol start="8">
<li>Place samples on ice until ready for FACS.</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1196-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Sample trituration, related to step 7</div>
</figcaption>
</figure>
<h3 id="sec3.3">Preparation of sony cell sorter for FACS</h3>
<p>Based on morphological analyses, pyramidal neurons have a soma diameter of 10–20 μm (<a href="#bib5">Oswald et al., 2013</a>), while astrocytes, oligodendrocytes, and microglia have an average diameter of 6–8μms (<a href="#bib1">Chai et al., 2017</a>; <a href="#bib4">Karasek et al., 2004</a>; <a href="#bib2">Davis et al., 2017</a>). One of the key methods for neuronal enrichment is to set the initial size gate such that anything smaller than 10 μm is gated out.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="9">
<li>Turn on Sony cell sorter and insert a 130 μm Microfluidic sorting Chip.</li>
<li>Add a drop of 8μm–12.9 μm polystyrene particle standard beads and 13μm–17.9 μm polystyrene particle standard beads into 1<b>×</b> PBS.</li>
<li>Sort the bead solution with forward scatter area and back scatter area to establish the first gate (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1196-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Example FACS gating strategy for collecting <i>Rbp4</i>+ neurons from <i>Rbp4</i> cre mice crossed with tdTomato reporter mice (ai14)</div>
<p>1) Polystyrene beads allow for gating on known size. 2) Based on beads size, size gate of sample is set. 3) Backscatter height and width are used to remove cellular doublets or intact cells plus debris. 4) Non-fluorescent control sample is used to set fluorescence gate (tdTomato from reporter mouse). 5) Collection shows intact cells expressing tdTomato and enriched in cytoplasmic dye.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cortical neurons have a soma size of approximately 10μm–20μm. Anything &lt;10μm is likely debris.</div>
<div class="note">
<span class="note-title">Note:</span> This can be done while waiting for the enzymatic dissociation.</div>
<h3 id="sec3.4">Sample preparation for FACS</h3>
<p>Incubation in cytoplasmic dye will allow for selection of intact neurons based on visualization of the cytoplasm using FACS. As this protocol dissociates long-distance projection neurons, the dissociation itself results in removal of the primary axon of the neuron. This leads to inevitable membrane disruption/ permeability which allows for uptake of the cytoplasmic dye and binding to intracellular amines primarily in neurons (as other cell-types remain intact). Our experiments culturing these dissociated neurons after FACS confirm that while they do take up the dye, they remain viable during and after dissociation.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<ol start="12">
<li>Add cytoplasmic dye directly to the sample at a concentration of 1:1000.
<ol type="a">
<li>Gently flick tube to mix into solution.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The cytoplasmic labeling dye used here is marketed as a live/dead stain as it is used to differentiate between cells that have an intact or disrupted plasma membrane. It doesn’t identify any cellular biochemical processes that suggest the cell is overtly dying <i>per se.</i> Removal of the primary axon from projection neurons during our dissociation protocol will transiently disrupt the plasma membrane allowing entry of the dye and thus identification of intact neuronal somata as evidenced from our robust single cell sequencing data.</div>
<ol start="13">
<li>Incubate on ice for 5 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> such dyes can exhibit toxicity to cells, and therefore incubation should not exceed 20 min.</div>
<ol start="14">
<li>Spin cells at 300 <i>g</i> for 10 min at 4°C.</li>
<li>Carefully remove supernatant with a P1000 pipette, being careful to not disturb the loose pellet.</li>
<li>Resuspend in 500μL dissociation buffer and triturate very carefully with a P1000 pipette to resuspend the cells.</li>
<li>Run the final cell solution through a 35 μm nylon mesh cap of a round bottom polystyrene test tube to ensure no large debris is present in the sample.</li>
</ol>
<h3 id="sec3.5">Sorting the sample</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min</div>
<ol start="18">
<li>Load the nonfluorescent sample that has been incubated in cytoplasmic dye into the sorter to set the remaining gates (<a href="#fig4">Figure 4</a>).
<ol type="a">
<li>Backscatter width and backscatter height are set to exclude doublets.</li>
<li>Sorting based on far-red (the spectrum of the cytoplasmic dye) and tdTomato (fluorophore expressed by the neurons of interest) will allow for drawing of the gate to select for intact fluorescent neurons: tdTomato high, far-red (cytoplasmic dye) high.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For our experiments, we did not need to perform compensation using this sorter. This may differ if using different fluorophores or an alternate sorter.</div>
<div class="note">
<span class="note-title">Note:</span> When we sorted our retrogradely traced cells, the cells of interest were quite rare (0.88% of cells sorted) and the division between fluorescent and non-fluorescent cells could only be set by first sorting a negative control.</div>
<ol start="19">
<li>Using the gates drawn with the negative control, sort the samples.
<ol type="a">
<li>If doing 10X sequencing or other droplet-based sequencing methods, sort the samples directly into a microfuge tube containing 500 μL of cold dissociation buffer. Immediately proceed to sequencing step.
<div class="note">
<span class="note-title">Note:</span> Use the cell count obtained through sorting for estimation of the number of cells for 10X sequencing rather than using a cell counter- especially if collecting a rare or fragile population of neurons.</div>
<div class="note">
<span class="note-title">Note:</span> For our experiments, we collected between 2,000 and 10,000 cells and collected them into 500μl of cold dissociation buffer. As the chromium chip allows for a maximum loading volume of 34μl, we first spun down the collected cells at 500g for 5 min, and very carefully removed much of the supernatant. We then load all of the cells in 34μl of cold dissociation buffer into the chromium chip.</div>
</li>
<li>If using plate-based sequencing, sort directly into the plate.</li>
<li>If using cells for qPCR, sort up to 5 <b>×</b> 10<sup>5</sup> cells into 350μL of lysis buffer.</li>
<li>If using cells for immunocytochemistry, sort directly into a 96-well plate into fix.</li>
<li>If using cells for long-term tissue culture, sort into 20μL of warm media per 10,000 cells sorted. Add 100μL of warm media along with the sorted cells into each well of a 0.01% poly-D-lysine- and 10 μg/mL laminin-coated 24-well plate. For best results, plate 30,000 cells per well. Allow to adhere for 30 min. Remove media to remove debris and replace with 500μL of warm media. Maintain with 50% media replacement every other day.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol will enrich for intact neurons from adult mouse cortex (<a href="#fig5">Figure 5</a>A). scRNAseq of cells collected using this protocol shows that 98% of cells sequenced are neurons and 2% of cells sequenced are endothelial cells. Of the neurons, all of them express <i>Ywhaz</i> (Tyrosine 3-Monooxygenase/Tryptophan 5-Monooxygenase Activation Protein Zeta), a whole-neuron specific marker (<a href="#bib10">Yao et al., 2020</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1196-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Neurons following dissociation</div>
<p>(A) Corticospinal neurons retrogradely labeled with rAAV-CAG-tdTomato were dissociated according to the protocol above. Robust detection of tdTomato confirms an intact plasma membrane and retained cytoplasm.</p>
<p>(B) Enzymatic digestion with 1 mg/mL pronase solution at room temperature without mixing results in a loss of cell integrity and adhesion of non-specific fluorescent debris to nuclei. Neurons were imaged prior to FACS using an Amnis Imagestream.</p>
<p>Scale bar = 5 μm.</p>
</figcaption>
</figure>
<p>If tracing from the spinal cord, expect to collect approximately 500 neurons per mouse. If collecting all layer V cortical neurons from motor cortex, expect to collect 10,000+ neurons per mouse.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol dissects out primary motor cortex from adult mice after retrograde AAV labeling from the spinal cord. We found that viral production of a fluorophore (either GFP or tdTomato) makes cortical neurons more fragile and thus harder to keep intact during dissociation (<a href="#bib11">Suriano et al., 2021</a>). For our experiments, we dissected neurons 7–10 days after viral injection. Pilot experiments demonstrated robust expression of retrograde AVV-mediated fluorophore expression by this time point when tracing from either the cervical or lumbar spinal cord. We recommend using the protocol as early after fluorophore expression is confirmed. Additionally, neurons that were transduced with multiple fluorescent viruses simultaneously appeared too fragile to be dissociated using this protocol.</p>
<p>We also found that neuron fragility was associated with axon length. Cortical neurons with axons terminating in the lumbar spinal cord were more challenging to dissociate than cortical neurons with axons terminating in the cervical spinal cord.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Neurons are not intact following dissociation, before FACS (<a href="#fig5">Figure 5</a>B; step 4 and step 7).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>This protocol is designed to ensure optimal cell integrity. If neurons do not have a cytoplasm after going through the protocol, prior to FACS, the most likely reasons are the enzymatic and manual dissociation steps. For the enzymatic dissociation (step 4), ensure that the size of the tissue pieces is small (no more than 2 mm in diameter). Additionally, ensure that solutions are well carbogenated prior to adding tissue. Once the tissue has been added, proper shaking of the solution in the incubator at 34°C is vital, as it allows the enzyme to property access all the tissue.</p>
<p>For manual trituration (step 7), it is vital to avoid air bubbles. Be sure to triturate slowly to avoid causing neuronal damage during this step.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Neurons are intact prior to FACS, but do not maintain their integrity during the sort (step 18).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>In our experience, we were only able to achieve high cell viability using a chip-based sorter (such as the Sony Sh800) with a 130 μm chip. We failed to successfully maintain cell viability with a non-chip based FACS machine including the BD Aria, even with the 130 μm nozzle.</p>
<p>Additionally, ensuring that the FACS machine is pre-set to 4°C was critical.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Contamination of non-neuronal cells (step 12).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Incubation in cytoplasmic dye is key for neuronal enrichment (step 12). This protocol is specifically designed to dissociate projection neurons and therefore, even when being very gentle, the primary axon gets removed from the cell body. This results in slight permeability of the membrane only in neurons, as microglia and oligodendrocytes remain intact and do not take up the dye. Our sequencing supports this as less than 2% of sequencing cells were non-neuronal. If experiencing contamination, incubate with dye for an additional 5 min prior to spin.</p>
<p>Additionally, the FACS size gate is designed to exclude cells with a soma diameter of less than 10 μm, which based on morphological analysis, will only include pyramidal neurons.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Cell yield is lower than expected (step 15).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>If cell yield is lower than expected, one possibility is that after the spin step (step 15), the pellet is removed along with the supernatant. The pellet is likely quite small and very loosely packed. To avoid, aspirate the supernatant very slowly and very carefully. If that is not sufficient, skip spin step altogether and sort the cells directly. This is not recommended as the cytoplasmic dye can be toxic; however, if the sort is quick (less than 20 min), the cells remain viable.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Multiple mice need to be pooled together for an experiment (step 1).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>If multiple mice need to be pooled together, perfuse each mouse and proceed until step 2a. Once all of the brains have been cut into 500 μm thick sections, macrodissect the ROIs (step 2b) and leave the macrodissected chunks in cold bubbled aCSF. Only once all of the sections have been macrodissected, they should be transferred to the papain solution (step 3) and processed together moving forward. In our experiments, we routinely pooled 6–8 mice per prep.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chai, H., Diaz-Castro, B., Shigetomi, E., Monte, E., Octeau, J.C., Yu, X., Cohn, W., Rajendran, P.S., Vondriska, T.M., Whitelegge, J.P., et al. (2017). Neural circuit-specialized astrocytes: transcriptomic, proteomic, morphological, and functional evidence. Neuron <i>95</i>, 531-549 e539. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00647-X/sref1">View at publisher</a></p>
<p id="bib2">Davis, B.M., Salinas-Navarro, M., Cordeiro, M.F., Moons, L., and De Groef, L. (2017). Characterizing microglia activation: a spatial statistics approach to maximize information extraction. Sci. Rep. <i>7</i>, 1576. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00647-X/sref2">View at publisher</a></p>
<p id="bib3">Golan, N., Kauer, S., Ehrlich, D.B., Ravindra, N., van Dijk, D., and Cafferty, W.B. (2021). Single-cell transcriptional profiling of the adult corticospinal tract reveals forelimb and hindlimb molecular specialization. bioRxiv. <a class="external-link" href="https://doi.org/10.1101/2021.2006.2002.446653">https://doi.org/10.1101/2021.2006.2002.446653</a></p>
<p id="bib4">Karasek, M., Swiltoslawski, J., and Zieliniska, A. (2004). Ultrastructure of the central nervous system: the basics. Folia Neuropathol. <i>42</i>, 1-9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00647-X/sref4">View at publisher</a></p>
<p id="bib5">Oswald, M.J., Tantirigama, M.L., Sonntag, I., Hughes, S.M., and Empson, R.M. (2013). Diversity of layer 5 projection neurons in the mouse motor cortex. Front. Cell. Neurosci. <i>7</i>, 174. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00647-X/sref5">View at publisher</a></p>
<p id="bib6">Saunders, A., Macosko, E.Z., Wysoker, A., Goldman, M., Krienen, F.M., de Rivera, H., Bien, E., Baum, M., Bortolin, L., Wang, S., et al. (2018). Molecular diversity and specializations among the cells of the adult mouse brain. Cell <i>174</i>, 1015-1030 e1016. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00647-X/sref6">View at publisher</a></p>
<p id="bib11">Suriano, C.M., Verpeut, J.L., Kumar, N., Ma, J., Jung, C., and Boulanger, L.M. (2021). Adeno-associated virus (AAV) reduces cortical dendritic complexity in a TLR9-dependent manner. bioRxiv. <a class="external-link" href="https://doi.org/10.1101/2021.09.28.462148">https://doi.org/10.1101/2021.09.28.462148</a></p>
<p id="bib7">Tasic, B., Yao, Z., Graybuck, L.T., Smith, K.A., Nguyen, T.N., Bertagnolli, D., Goldy, J., Garren, E., Economo, M.N., Viswanathan, S., et al. (2018). Shared and distinct transcriptomic cell types across neocortical areas. Nature <i>563</i>, 72-78. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00647-X/sref7">View at publisher</a></p>
<p id="bib8">Tervo, D.G., Hwang, B.Y., Viswanathan, S., Gaj, T., Lavzin, M., Ritola, K.D., Lindo, S., Michael, S., Kuleshova, E., Ojala, D., et al. (2016). A designer AAV variant permits efficient retrograde Access to projection neurons. Neuron <i>92</i>, 372-382. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00647-X/sref8">View at publisher</a></p>
<p id="bib9">Wang, Z., Maunze, B., Wang, Y., Tsoulfas, P., and Blackmore, M. (2018). Global connectivity and function of descending spinal input revealed by 3D microscopy and retrograde transduction. J. Neurosci. <i>38</i>, 10566-10581. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00647-X/sref9">View at publisher</a></p>
<p id="bib10">Yao, Z., Liu, H., Xie, F., Fischer, S., Booeshaghi, A.S., Adkins, R.S., Aldridge, A.I., Ament, S.A., Pinto-Duarte, A., Bartlett, A., et al. (2020). An integrated transcriptomic and epigenomic atlas of mouse primary motor cortex cell types. bioRxiv. <a class="external-link" href="https://doi.org/10.1101/2020.2002.2029.970558">https://doi.org/10.1101/2020.2002.2029.970558</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, William Cafferty, <a href="mailto:william.cafferty@yale.edu">william.cafferty@yale.edu</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The published article includes all datasets generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>Funding sources: this work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grants R01NS095930 and R21NS108053 to W.B.C. Core facility: this work was conducted in the Yale Flow Cytometry Facility, The Anlyan Center, 300 Cedar Street, New Haven, CT 06511. Single-cell RNA sequencing was conducted by the Yale Center for Genomic Analysis (YCGA), Keck Biotechnology Resource Laboratory, Yale University, 300 George Street, New Haven, CT 06511.</p>
<h3>Author contributions</h3>
<p>Conceptualization, N.G. and W.B.C.; methodology, N.G. and W.B.C.; formal analysis, N.G. and W.B.C.; investigation, N.G., and W.B.C.; writing – original draft, N.G. and W.B.C.; supervision, W.B.C.; funding acquisition, W.B.C.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>