<article>
<section>
<h1 id="header">Depletion of nuclear LINE1 RNA in mouse ESCs and embryos</h1>
<p><time datetime="2021-08-13">Published: August 13, 2021</time></p>
<p>Michelle Percharde,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">5</a>,<a href="#fn2">6</a>,<a href="#cor1">*</a></sup> Chih-Jen Lin,<sup><a href="#aff3">3</a>,<a href="#fn1">5</a>,<a href="#cor2">**</a></sup> and Miguel Ramalho-Santos<sup><a href="#aff4">4</a></sup></p>
<p id="aff1"><sup>1</sup>MRC London Institute of Medical Sciences (LMS), London W12 0NN, UK</p>
<p id="aff2"><sup>2</sup>Institute of Clinical Sciences (ICS), Imperial College, London W12 0NN, UK</p>
<p id="aff3"><sup>3</sup>The University of Edinburgh, MRC Centre for Reproductive Health, Queen’s Medical Research Institute, Edinburgh EH16 4TJ, UK</p>
<p id="aff4"><sup>4</sup>Lunenfeld-Tanenbaum Research Institute and Department of Molecular Genetics, University of Toronto, Toronto M5T 3H7, Canada</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:m.percharde@lms.mrc.ac.uk">m.percharde@lms.mrc.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:Chih-Jen.Lin@ed.ac.uk">Chih-Jen.Lin@ed.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100726">10.1016/j.xpro.2021.100726</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>LINE1 is the most active and abundant family of retrotransposons; it is implicated in a number of pathologies, as well as in early embryo development. We present a protocol to specifically knockdown LINE1 in mouse embryonic stem cells and embryos, including details for the nucleofection and zygote microinjection of LINE antisense oligos, followed by RNA FISH validation. This protocol can be used in development, as well as other cell types where LINE1 is believed to be expressed.</p>
<p>For complete information on the use and execution of this protocol, please refer to <a href="#bib6">Percharde et al. (2018)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Nucleofection of antisense oligos (ASOs) into ESCs for specific depletion of LINE1 RNA</li>
<li>Zygote microinjection of LINE1 ASOs for embryonic knockdown</li>
<li>Visualization of LINE1 RNA and validation of depletion by RNA FISH</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/916-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol outlines the steps for successful knockdown of LINE1 in either mouse embryos (zygote and 2-cell) or ESCs, followed by RNA FISH validation. The main steps of the ESC protocol can be adapted for other cell types, with the use of different kits for nucleofection (available at Lonza) in order to optimize the delivery of antisense oligos (ASOs). For ESCs, follow steps 1–20, and for embryos, steps 21–32. Morpholino ASOs were designed using Gene Tools Oligo Design Request Form (<a href="https://www.gene-tools.com/">https://www.gene-tools.com/</a>), and chosen for their stability, extensive use in pre-implantation development (e.g., (<a href="#bib2">Lin et al., 2013</a>)), and specificity. The inter-orf region of LINE1 RNA was used as a template, and validated using L1Base (<a href="http://l1base.charite.de/l1base.php">http://l1base.charite.de/l1base.php</a>, (<a href="#bib5">Penzkofer et al., 2005</a>) to be perfectly homologous to at least 500 full-length elements. This region binds a key interactor, Nucleolin (<a href="#bib4">Peddigari et al., 2013</a>), with disruption of this interaction potentially contributing to the destabilization of LINE1 RNA. As a control, a predesigned non-targeting ASO (Ctl), or the LINE1 reverse-complement (RC) ASO sequence, is used. RNA FISH probes were designed online using the Stellaris design tool (<a href="https://www.biosearchtech.com/">https://www.biosearchtech.com/</a>) using the sequence for an active young L1 element (L1spa) as a template.</p>
<h3 id="sec1.1">Reagent preparation (for all sections)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>Resuspend Morpholino ASOs to 1.5 mM in RNAse free (not DEPC) water for embryo experiments and to 1 mM for ESC experiments, shaking at 37°C if needed to aid dissolution. Store at −20°C in aliquots protected from light. Lissaminated ASOs are used for ESCs, and non-lissaminated ASOs for embryos, which do not require confirmation of uptake by microscopy or flow cytometry.</li>
<li>Resuspend smRNA FISH probes in water to 12.5 μM. Store at −20°C in aliquots protected from light.</li>
</ol>
<h3 id="sec1.2">ESC culture (for section A: LINE1 depletion in ESCs)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–20 min per day, 2–3+ days</div>
<ol start="3">
<li>Prepare 0.1% gelatin/PBS solution by diluting 2% gelatin in PBS, store at RT.</li>
<li>Grow E14 ESCs on 0.1% gelatin/PBS pre-coated tissue culture dishes (we routinely use 6-well plates), splitting cells approximately 1:10 every other day in ESC medium. Cells are cultured in a humidified incubator, 5% CO<sub>2</sub> and 37°C. Split ESCs when 70%–80% confluent.</li>
<li>For general notes on splitting ESCs see method steps 1–3.</li>
<li>On non-splitting days, change the media.</li>
<li>Prepare to have at least 5 million cells on the day of nucleofection per ASO, at around 70% confluence.</li>
</ol>
<h3 id="sec1.3">Superovulation of mice for zygote and 2-cell collection (for section B: LINE1 depletion in embryos)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days for zygotes and 4 days for 2-cells</div>
<ol start="8">
<li>Day 0, ∼11:30 am–12:00, perform intraperitoneal (i.p.) injection of 7.5 IU pregnant mare's serum gonadotropin (PMSG) of C57BL/6 females (usually 8–10 mice).</li>
<li>Day 2, ∼9:45–10:00 am, perform intraperitoneal (i.p.) injection of 7.5 IU human chorionic gonadotropin (hCG) of C57BL/6 female mice. Place injected mice into male stud cages for mating.</li>
<li>Day3, take animals ∼24 h post-hCG for zygote collection.</li>
<li>Day4, take animals ∼43 h post-hCG for 2-cell collection.</li>
</ol>
<h3 id="sec1.4">Equilibration of embryo culture media (for section B: LINE1 depletion in embryos)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8–12 h or overnight before collection for step 12</div>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1 h <b>for step 13</b>
</div>
<ol start="12">
<li>Prepare KSOM medium in microdrops (∼80–100 μL per drop) in a 60 mm dish, cover with mineral oil and equilibrate overnight (8–12 h) at 37°C in 5% CO<sub>2</sub>.</li>
<li>Aliquot M2 medium into a Falcon 15 mL tube and warm on a 37°C heating plate for embryo flushing.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM high glucose, glutaMAX, pyruvate</td>
<td>Gibco</td>
<td>Cat# 10569010</td>
</tr>
<tr>
<td>LIF</td>
<td>Millipore</td>
<td>Cat# ESG1107</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Atlanta Biologicals</td>
<td>Cat# S11150</td>
</tr>
<tr>
<td>Beta-mercaptoethanol 100×</td>
<td>Millipore</td>
<td>Cat# ES-007-E</td>
</tr>
<tr>
<td>MEM Non-essential Amino Acids Solution (NEAA) 100×</td>
<td>Gibco</td>
<td>Cat# 11140050</td>
</tr>
<tr>
<td>TrypLE Express with phenol-red</td>
<td>Gibco</td>
<td>Cat# 12604013</td>
</tr>
<tr>
<td>2% Gelatin Solution</td>
<td>Sigma</td>
<td>Cat# G1393-100 mL</td>
</tr>
<tr>
<td>RNase-free yeast tRNA 20 mg/mL in RNAse-free H2O</td>
<td>Roche</td>
<td>Cat# 10109495001</td>
</tr>
<tr>
<td>RNase-free BSA</td>
<td>bioWORLD</td>
<td>Cat# 40200064-1</td>
</tr>
<tr>
<td>Vanadyl Ribonucleoside Complex (VRC)</td>
<td>NEB</td>
<td>Cat# 18068015</td>
</tr>
<tr>
<td>20× SSC</td>
<td>Ambion</td>
<td>Cat# AM9770</td>
</tr>
<tr>
<td>Formamide (deionized)</td>
<td>Ambion</td>
<td>Cat# AM9342</td>
</tr>
<tr>
<td>Dextran sulfate</td>
<td>Sigma</td>
<td>Cat# D8906</td>
</tr>
<tr>
<td>Nuclease-free Water</td>
<td>Ambion</td>
<td>Cat# AM9939</td>
</tr>
<tr>
<td>Rnase-OUT</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10777019</td>
</tr>
<tr>
<td>Poly L-Lysine</td>
<td>Sigma</td>
<td>Cat# P4707</td>
</tr>
<tr>
<td>Vectashield plus DAPI</td>
<td>Vector Labs</td>
<td>Cat# H-1000-10</td>
</tr>
<tr>
<td>Pregnant Mare Serum Gonadotropin, PMSG</td>
<td>Prospec Protein Specialists</td>
<td>Cat# HOR-272</td>
</tr>
<tr>
<td>Human Chorionic Gonadotropin, hCG</td>
<td>MSD Animal Health</td>
<td>Cat# CH-475-1</td>
</tr>
<tr>
<td>M2 medium</td>
<td>Sigma</td>
<td>Cat# M7167</td>
</tr>
<tr>
<td>KSOMaa medium</td>
<td>Sigma</td>
<td>Cat# MR-121D</td>
</tr>
<tr>
<td>Hyaluronidase</td>
<td>Sigma</td>
<td>Cat# H4272</td>
</tr>
<tr>
<td>Mineral oil</td>
<td>Sigma</td>
<td>Cat# M8410</td>
</tr>
<tr>
<td>Nuclease-Free water</td>
<td>Invitrogen</td>
<td>Cat# AM9937</td>
</tr>
<tr>
<td>Water for embryo transfer</td>
<td>Sigma</td>
<td>Cat# W1503</td>
</tr>
<tr>
<td>DAPI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# D3571</td>
</tr>
<tr>
<td>Sigmacote</td>
<td>Sigma</td>
<td>Cat# SL2</td>
</tr>
<tr>
<td>Formamide (embryo)</td>
<td>Sigma</td>
<td>Cat# F9037</td>
</tr>
<tr>
<td>20× SSC (embryo)</td>
<td>Alfa Aesar</td>
<td>Cat# J60561</td>
</tr>
<tr>
<td>PVA (Polyvinyl alcohol)</td>
<td>Sigma</td>
<td>Cat# P8136</td>
</tr>
<tr>
<td>TWEEN-20</td>
<td>Sigma</td>
<td>Cat# T8787</td>
</tr>
<tr>
<td>DPBS</td>
<td>Gibco</td>
<td>Cat# REF 14190-094</td>
</tr>
<tr>
<td>Beeswax</td>
<td>Sigma</td>
<td>Cat# 243221</td>
</tr>
<tr>
<td>Vaseline</td>
<td>Sigma</td>
<td>Cat# 16415</td>
</tr>
<tr>
<td>Stellaris RNA FISH Hybridization Buffer</td>
<td>Biosearchtech</td>
<td>CatSMF-HB1</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Mouse ES cell Nucleofector Kit</td>
<td>Lonza</td>
<td>Cat# VPH-1001</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Mouse ES cells E14Tg2A</td>
<td>B.Skarnes</td>
<td><a href="#bib9">(Hooper et al., 1987)</a></td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mice: C57BL/6NCrl (6–8 weeks old, female) (8–20 weeks old, male)</td>
<td>Charles River</td>
<td>Cat# 027</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Control ASO: CCTCTTACCTCAGTTACAATTTATA</td>
<td>Gene Tools</td>
<td>N/A</td>
</tr>
<tr>
<td>RC ASO: AGACAGCCACAAGAACAGAATGCCA</td>
<td>Gene Tools</td>
<td><a href="#bib6">Percharde et al., 2018</a></td>
</tr>
<tr>
<td>LINE1 ASO: TGGCATTCTGTTCTTGTGGCTGTCT</td>
<td>Gene Tools</td>
<td><a href="#bib6">Percharde et al., 2018</a></td>
</tr>
<tr>
<td>LINE1 RNA smFISH probes Quasar 670</td>
<td>Biosearch Technologies</td>
<td>(<a href="#bib6">Percharde et al., 2018</a>), Table S5</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>StarSearch</td>
<td><a href="#bib7">Raj et al., 2008</a></td>
<td><a href="http://rajlab.seas.upenn.edu/StarSearch/%20launch.html">http://rajlab.seas.upenn.edu/StarSearch/ launch.html</a></td>
</tr>
<tr>
<td>FIJI (Fiji is Just ImageJ)</td>
<td>ImageJ</td>
<td><a href="https://fiji.sc/">https://fiji.sc/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Hybridization oven (generic)</td>
<td>Techne</td>
<td>Hybridiser Model FHB1DG</td>
</tr>
<tr>
<td>Inverted widefield microscope</td>
<td>Leica</td>
<td>DMI 4000B</td>
</tr>
<tr>
<td>Amaxa Nucleofector 2b device, Lonza</td>
<td>Lonza</td>
<td>Cat# AAB-1001</td>
</tr>
<tr>
<td>13 mm Diameter Coverslips</td>
<td>SLS</td>
<td>Cat# MIC3300</td>
</tr>
<tr>
<td>Cover Slip forceps</td>
<td>Dumont</td>
<td>Cat# 11251-33</td>
</tr>
<tr>
<td>Tissue Culture 6-well plates</td>
<td>SLS</td>
<td>Cat# 3335</td>
</tr>
<tr>
<td>Tissue Culture 12-well plates</td>
<td> </td>
<td>Cat# 3336</td>
</tr>
<tr>
<td>50 mm dish for microinjection</td>
<td>Falcon</td>
<td>Cat# 351006</td>
</tr>
<tr>
<td>60 mm for embryo culture</td>
<td>Eppendorf</td>
<td>Cat# 003071011</td>
</tr>
<tr>
<td>96-well U-bottom plate Falcon</td>
<td>Falcon</td>
<td>Cat# 353077</td>
</tr>
<tr>
<td>SuperFrost Plus™ Adhesion slides</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# J1800AMNZ</td>
</tr>
<tr>
<td>1 mL Syringes</td>
<td>BD</td>
<td>Cat# 303172</td>
</tr>
<tr>
<td>30.5G needles</td>
<td>BD</td>
<td>Cat# 304000</td>
</tr>
<tr>
<td>Microinjection pipettes: Holding pipettes</td>
<td>Eppendorf VacuTip</td>
<td>Cat# 930001015</td>
</tr>
<tr>
<td>Microinjection pipettes: Injection pipettes</td>
<td>World Precision Instruments</td>
<td>Cat# BF-100-78-10</td>
</tr>
<tr>
<td>Stereo microscope</td>
<td>Leica</td>
<td>M165C</td>
</tr>
<tr>
<td>Inverted scope setup: Leica DMi8 equipped with Coarse manipulators (Narishiage MN-4), Micromanipulators (Narishiage MMO-202ND), and a Thermoplate(Leica TPX).</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Microinjector for injection pipettes</td>
<td>Eppendorf</td>
<td>FemtoJet 4i</td>
</tr>
<tr>
<td>Pneumatic Injector for holding pipettes</td>
<td>Narishige</td>
<td>IM-11-2</td>
</tr>
<tr>
<td>Heating plate</td>
<td>Minitube</td>
<td>Control Unit HT 50</td>
</tr>
<tr>
<td>Vibration Isolation Platform</td>
<td>Newport</td>
<td>VIP3200-L-DMi8A</td>
</tr>
<tr>
<td>Micropipette Puller</td>
<td>Sutter Instrument</td>
<td>P-97</td>
</tr>
<tr>
<td>Microforge</td>
<td>Narishige</td>
<td>MF-900</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Section A: LINE1 depletion in ESCs</h3>
<table id="undtbl1">
<caption>ESC culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>N/A</td>
<td>415 mL</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>15%</td>
<td>75 mL</td>
</tr>
<tr>
<td>NEAA 100×</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>Beta-mercaptoethanol 100×</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>LIF 10,000×</td>
<td>1×</td>
<td>50 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Pass medium through 0.2 um PES filtration system. Store at 4°C for up to 2 weeks</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> ES cells may also be cultured in N2B27/2i+LIF medium, <a href="#bib8">Ying et al., 2008</a>
</div>
<table id="undtbl2">
<caption>ESC smRNA FISH hybridization buffer (HB)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dextran sulfate</td>
<td>10%</td>
<td>1 g</td>
</tr>
<tr>
<td>Formamide 100%</td>
<td>15%</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>Yeast tRNA 20 mg/mL in H2O</td>
<td>1 mg/mL</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>SSC 20×</td>
<td>2×</td>
<td>1 mL</td>
</tr>
<tr>
<td>BSA 1% (50 mg/mL)</td>
<td>0.02%</td>
<td>40 μL</td>
</tr>
<tr>
<td>VRCs 200 mM</td>
<td>2 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>RNase-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>6.9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Allow formamide to come to room temperature (RT; 20°C–23°C) before opening. Work in RNase-free conditions at all times. Dissolve dextran sulfate in water at 20°C–23°C, then add the rest of the reagents. Filter buffer to remove undissolved clusters of Dextran. Freeze in aliquots and store at −20°C, use within 1–2 months. Add 1:1000 RNAse-OUT immediately before use.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> the Stellaris RNA FISH buffer may also be used as for embryos; although in our hands this recipe above worked better in ESCs.</div>
<table id="undtbl3">
<caption>ES Cell smRNA FISH wash buffer (WB)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Formamide 100%</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>SSC 20×</td>
<td>2×</td>
<td>50 mL</td>
</tr>
<tr>
<td>RNase-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>400 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Allow formamide to come to 20°C–23°C before opening. Make fresh each day. Work in RNase-free conditions at all times. Add 1:1000 RNAse-OUT immediately before use</div>
<h3 id="sec2.2">Section B: LINE1 depletion in embryos</h3>
<table id="undtbl4">
<caption>DPBS+PVA</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>100 mL</td>
</tr>
<tr>
<td>PVA</td>
<td>0.1%</td>
<td>0.1 g</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Work in RNase-free conditions at all times. Store at 20°C–23°C for 1–2 months. Add 1:1000 RNAse-OUT immediately before use</div>
<table id="undtbl5">
<caption>Embryo smRNA FISH wash buffer (eWB)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Formamide 100%</td>
<td>10%</td>
<td>10 mL</td>
</tr>
<tr>
<td>SSC 20×</td>
<td>2×</td>
<td>10 mL</td>
</tr>
<tr>
<td>RNase-free H<sub>2</sub>O</td>
<td>n/a</td>
<td>80 mL</td>
</tr>
<tr>
<td>PVA</td>
<td>0.1%</td>
<td>0.1 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Work in RNase-free conditions at all times. Make fresh each day at 20°C–23°C; add 1:1000 RNAse-OUT immediately before use</div>
<table id="undtbl6">
<caption>Embryo smRNA FISH wash buffer+ Tween-20 (eWB+T)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>eWB</td>
<td>n/a</td>
<td>49.5 mL</td>
</tr>
<tr>
<td>10% Tween-20</td>
<td>0.1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Work in RNase-free conditions at all times. Make fresh each day at 20°C–23°C; Add 1:1000 RNAse-OUT immediately before use</div>
<table id="undtbl7">
<caption>Embryo smRNA FISH wash buffer+Tween-20+DAPI (eWB+T+DAPI)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>eWB+T</td>
<td>n/a</td>
<td>25 mL</td>
</tr>
<tr>
<td>DAPI (5 mg/mL)</td>
<td>100 ng/mL</td>
<td>0.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Work in RNase-free conditions at all times. Make fresh each day at 20°C–23°C; add 1:1000 RNAse-OUT immediately before use</div>
<table id="undtbl8">
<caption>Soft wax for mounting embryos</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Vaseline</td>
<td>90%</td>
<td>27 g</td>
</tr>
<tr>
<td>Beeswax</td>
<td>10%</td>
<td>3 g</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Use a beaker for weighing the reagents and melt them on a 65°C hot plate. Store at 20°C–23°C for several months or more.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Section A: LINE1 depletion in ESCs</h3>
<h4 id="sec3.1.1">Nucleofection of ESCs with ASOs</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min for step<b>s</b> <b>1</b><b>-12</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 5–30 min for step 13</div>
<p>Due to the extremely high composition of LINE1 in the genome, it is not possible to delete it by a genome editing approach. Therefore, we applied an antisense oligo (ASO) knockdown approach, using lissamine-conjugated Morpholino ASOs.</p>
<ol>
<li>Before starting, warm up all media and trypsin to 37°C. Warm up nucleofection solution to 20°C–23°C. Add 2 mL 0.1% gelatin to 2 wells of a 6-well tissue culture plate per ASO and plate in the incubator. Thaw ASOs on ice. All reagents should be sterile.</li>
<li>Trypsinize ESCs: for ESCs grown in 6-well plates, wash each well of ESCs with 1–2 mL PBS before adding 0.5 mL TrypLE Express reagent to dissociate cells. Place in the incubator for 4–5 min to allow cells to detach. Scale all volumes accordingly for ESCs grown in larger dishes.</li>
<li>Pipette ESCs up and down 4–5 times with a P1000 pipette to achieve a single cell suspension. Transfer to a 15 mL conical tube and neutralize with &gt;2× volume of fresh ESC medium.</li>
<li>Use your preferred method of choice to count the total live number of ESCs in the tube. Transfer 5e<sup>6</sup> cells per ASO (for example take 10e<sup>6</sup> ESCs to perform nucleofection of ESCs with RC and LINE1 ASOs) to a new 15 mL conical tube, and centrifuge at 400 × g, 3 min to pellet cells.</li>
<li>Resuspend pellet in 10 mL PBS to wash, then pellet as before. In the meantime, get the Nucleofector device ready, and set it to the program, A023.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> alternative nucleofection devices may be used, although these have not been tried in our hands and would require additional optimization steps.</div>
<div class="critical">
<span class="critical-title">Critical:</span> get all reagents out ready for plating ESCs immediately after nucleofection, to minimize the time that the cells remain in the nucleofection solution. See <a href="#troubleshooting">troubleshooting 1</a>.</div>
<ol start="6">
<li>Remove the gelatin from the 6-well plate and add 2 mL ESC medium to each well to be used, leave this plate in the hood ready for the cells to be added.</li>
<li>Aspirate PBS and resuspend the ESC pellet in 95 μL nucleofection solution per sample, pipetting up and down only 1–2 times (there will be a small excess from residual PBS) and transfer 95 μL to the bottom of each nucleofection cuvette.</li>
<li>Add 5 μL (5 nmol) of ASO to the cells and flick gently a few times to mix.</li>
<li>Insert the cuvette into the nucleofector and press the button to nucleofect each sample, then take the cuvette(s) back to the tissue culture hood.</li>
<li>Pipette 0.5 mL ESC medium into each cuvette (a white precipitate may float to the top).</li>
<li>Using one of the supplied Pasteur pipettes, carefully obtain all the medium containing the nucleofected cells, avoiding any precipitate where possible. Add the cells dropwise to 2 wells of the 6-well plate, which already contains ESC media. Repeat for the other sample(s)</li>
<li>Gently shake the plate to evenly disperse the cells and return to the incubator to allow the ESCs to recover and adhere.</li>
<li>Next day: change the medium on each sample; there may be a small amount of death/floating cells but not much.</li>
<li>48 h post nucleofection, ESCs with RC ASOs should be 70%–90% confluent and 50%–70% confluent after LINE1 ASOs. Proceed to RNA FISH or other downstream applications.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> check for nucleofection efficiency 24 h after under a fluorescent microscope by observing the % red (lissamine-positive) cells. We routinely see 70%–95% positive cells (<a href="#fig1">Figure 1</a>A).</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/916-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. ESC nucleofection and RNA FISH</div>
<p>(A and B) (A) Step 14: Representative flow cytometry quantification of nucleofected ESCs with lissaminated ASOs. Adapted from <a href="#bib6">Percharde et al., 2018</a>. (step 19) Rehydrating coverslips with WB: (B) Slides overnight will be firmly attached to the parafilm and should not be directly picked up.</p>
<p>(C and D) (C) Use a P200 pipette tip placed just at the edge of the coverslip, pressing down into the parafilm to create a small gap to “flush” WB under the coverslip until (D) it floats upwards.</p>
<p>(E) This enables the coverslip to be lifted up without disturbing the cells.</p>
</figcaption>
</figure>
<h4 id="sec3.1.2">ESC RNA FISH</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="15">
<li>Perform RNA FISH 48 h after nucleofection. Before beginning, coat glass coverslips with poly-L-Lysine solution, one coverslip placed in each well of a 12-well plate. Apply enough solution to cover wells for 10–20 min. Rinse with sterile H2O and then leave to air dry. Chill 70% RNAse-free Ethanol at −20°C for later permeabilization.</li>
<li>Plating onto coverslips:
<ol type="a">
<li>Trypsinize ESCs and count as before. Resuspend in a volume of 150,000 ESCs/mL. ESCs must be single cell to allow effective FISH staining.</li>
<li>Transfer 1 mL ESCs (150,000 per well) to each well of the 12-well plate containing a coated coverslip. Gently shake to disperse the cells evenly and place into the incubator.</li>
<li>Leave the cells to adhere to the coverslips for at least 4 h.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> from now on, all reagents should be RNAse-free. Use clean gloves sprayed beforehand with anti-RNAse solution such as RNAse-ZAP, and dedicated RNAse-free filter tips.</div>
<ol start="17">
<li>Fixation and permeabilization:
<ol type="a">
<li>Check ESCs are adhered to coverslips before fixation by gently agitating the dish and observing whether ESCs move. Leave ESCs in the incubator for an extra hour if not adherent.</li>
<li>Wash once gently with 1–2 mL PBS before adding 1 mL 4% PFA solution per well to each well to fix cells without detaching them. Incubate 15 min at 20°C–23°C.</li>
<li>Discard the PFA and wash once with PBS</li>
<li>Wash once with 1–2 mL cold 70% ethanol</li>
<li>Remove and add a fresh change of 70% ethanol. Store plates at 4°C for a minimum of overnight (minimum 15–18 h) before proceeding to hybridization.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Plates may be stored for several days or up to 1–2 weeks at 4°C; make sure ethanol does not evaporate.</div>
<ol start="18">
<li>Hybridization:
<ol type="a">
<li>Rehydrate coverslips by replacing the ethanol with FISH Wash Buffer (WB). Replace with a second change of WB and leave to rehydrate for at least 5 min.</li>
<li>Thaw an aliquot of hybridization buffer (HB), you will require 100 μL HB per coverslip, and thaw the LINE1 Orf2 RNA FISH probes. Add 0.5 μL probe (appox 6.25 pmol) per 100 μL HB together with 1 μL RNAse-OUT inhibitor.</li>
<li>Wrap foil around a 15 cm tissue culture dish to make it lightproof. Place a piece of parafilm large enough for the coverslips in the dish, and label an area for each condition (Eg RC versus LINE1 ASO). Pipette 100 μL of probe in HB onto the parafilm per coverslip.</li>
<li>Use coverslip forceps to pick up each coverslip. Blot the edge of the coverslip on lint-free paper to remove excess liquid, before depositing face down onto the HB.</li>
<li>Incubate in a hybridization oven at 37°C for 45–60 min, then at 30°C overnight (approx. 16–18 h).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not humidify or seal the chamber, as some drying is meant to occur – evidenced by the coverslips needing to be rehydrated in situ before picking up for washes (step 19, <a href="#fig1">Figures 1</a>B–1E)</div>
<ol start="19">
<li>Washes, mounting and imaging:
<ol type="a">
<li>Prewarm WB to 30°C–37°C. Right before use, add RNAse-OUT to the correct volume needed for washes (1–2 mL per coverslip per wash).</li>
<li>Take out the dish containing coverslips. Note that the coverslips will seem almost stuck to the parafilm. With a p200 tip containing some WB, gently pipette some WB between the coverslip (still face down) and the parafilm; the coverslip will ‘float’ a little. Now remove the coverslip face up into a 12-well or 6-well dish containing WB (<a href="#fig1">Figures 1</a>B–1E)
<div class="critical">
<span class="critical-title">Critical:</span> Do not try to pick up the coverslip without having pipetted WB underneath; the cells will detach from the coverslip</div>
</li>
<li>Remove this wash and replace with fresh WB. Incubate at 30°C (e.g., back in the hybridization oven), protected from light.</li>
<li>Add fresh WB plus DAPI solution (1:1000) and leave for 15–30 min</li>
<li>Mount the coverslips. With forceps, blot the edge of the coverslip to remove excess WB. Place face down on a drop of Vectashield on a microscope slide; we typically use 2 coverslips per slide – e.g., the two conditions to be compared (RC versus LINE1 ASO).</li>
<li>Press down gently to remove excess Vectashield, blot away. Leave to dry protected from light for 15–30 min before sealing with nail varnish. Do not use hardening mounting medium, FISH signal will not be as strong.</li>
</ol>
</li>
</ol>
<ol start="20">
<li>Imaging:
<ol type="a">
<li>Image cells on a widefield microscope at 100× oil-immersion magnification (for best visualization of RNA FISH foci - See <a href="#troubleshooting">troubleshooting 2</a>). Make sure to collect data from the far-red channel (to visualize the RNA FISH foci) before other channels, e.g., DAPI, to prevent loss of signal from photobleaching.</li>
<li>On our setup we typically found an exposure time of 4–5 s was required. Collect Z-stacks every 0.25 μm using the Z stack function of the MetaMorph software (Molecular Devices), and create max projections in FIJI for subsequent quantification. See <a href="#troubleshooting">troubleshooting 2</a>.</li>
<li>Quantify RNA FISH foci numbers per cell in each condition using StarSearch free online software (<a href="#bib7">Raj et al., 2008</a>). Import in individual images and manually draw around each nuclear outline, then click to quantify FISH foci per nucleus. We maintained the same threshold across images.</li>
<li>Representative FISH results are shown in (<a href="#fig3">Figure 3</a>).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We found Quasar 670-labeled probes to give the best signal-to-noise ratio for quantification of RNA FISH foci. However, for other probes – eg a control probe, Gapdh Quasar 570 also works well (Cat# SMF-3002-1). Alternative software for quantification includes Rajlabimagetools (<a href="https://github.com/arjunrajlaboratory/rajlabimagetools/wiki">https://github.com/arjunrajlaboratory/rajlabimagetools/wiki</a>), or scripts in Cell Profiler - although these have not been tried by the authors and would require optimization.</div>
<h3 id="sec3.2">Section B: LINE1 depletion in embryos</h3>
<h4 id="sec3.2.1">Microinjection of Line1 ASO into zygotes and 2-cells</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>Successful microinjection is the key to deliver ASO into zygotes and 2-cells.</p>
<ol start="21">
<li>Embryo collection
<ol type="a">
<li>For zygote collection, sacrifice mice at ∼24 h post hCG administration. For 2-cell collection, sacrifice mice at ∼43 h post hCG administration. Use 20–30 zygotes per condition (8–10 C57Bl/6 mice, see <a href="#before-you-begin">before you begin</a>, step 8).</li>
<li>Zygotes are collected by either flushing M2 medium into infundibulum using a 30.5 gauge needle and syringe or tearing apart ampulla to release the embryos (<a href="#bib1">Behringer et al., 2014</a>).</li>
<li>Use 300 μL M2 medium with Hyaluronidase (300 μg/mL) to dissociate the remaining cumulus cells attached to the embryos.</li>
<li>Flushed embryos are then short-term cultured (1 h) in KSOMaa in the incubator at 37°C, 5% CO<sub>2</sub> before setting up for microinjection.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For 8–10 superovulated mice, 50–60 fertilised zygotes are expected.</div>
<ol start="22">
<li>Setting up microinjection
<ol type="a">
<li>Pull injection pipettes using P-97 micropipette puller. Injection parameters are adjusted according to the Ramp test. For example, for Ramp test of value 457, apply the following parameter:<br> Heat 496–506, Pull 90, velocity 70, and time 160. Pressure setting, 500.</li>
<li>Bend pulled injection pipettes by microforge. Use the heater setting at level ∼50. Adjust the platinum heater wire with glass bead to the correct position and bend the pipettes to an angle of 10–15 degrees (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/916-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Setting up embryo microinjection</div>
<p>(A) Bending the injection pipette by microforge (step 22b).</p>
<p>(B) Injection dish setup (step 22d). M2 medium drops (each of 5 ul; dome-shape drops are for washing and longer dome-shaped drops are for injection) are adding into a 50 mm dish and covered with mineral oil.</p>
<p>(C) Injection pipette loaded with ASO solution (step 22e). Before (left) and after (right) ASO loading into injection pipettes.</p>
<p>(D) Microinjection system (step 22f). (a) Inverted scope, (b) Micromanipulators, (c) FemtoJet microinjector (d) Pneumatic injector, (e) Thermoplate, (f) Anti-vibration platform.</p>
<p>See also <a href="#mmc1">Methods videos S1</a> and <a href="#mmc2">S2</a>.</p>
</figcaption>
</figure>
</li>
<li>Centrifuge ASOs at ∼16,000 <i>g</i> for 10 min to pellet any possible debris/particles.</li>
<li>Prepare injection dishes (<a href="#fig2">Figure 2</a>B) and set up both holding and injection pipettes.</li>
<li>Load ∼0.5 μl of injection solution to the end of injection pipettes by dispensing from a P2 pipette and let them fill by capillary action (<a href="#fig2">Figure 2</a>C).
<div class="critical">
<span class="critical-title">Critical:</span> You need to see the injection solution moving to the tip via capillary action.</div>
<div class="note">
<span class="note-title">Note:</span> Both Control (Ctl) ASOs or RC ASOs have been used successfully and interchangeably for embryo experiments (<a href="#bib6">Percharde et al., 2018</a>). Choice of which control to use depends on individual experiments – eg if simultaneous knockdown of different RNAs is required, the generic control may be more appropriate.</div>
</li>
<li>Fitting the holding pipette to the pneumatic injector, lower the pipette into the injection drops, aspirate the medium, equilibrate the pressure using the pressure release valve.</li>
<li>Operation of FemtoJet 4i. Before turning on the Femtojet 4i, disconnect the injection tube. Switch on and re-connect and wait for sufficient pressure to built up for injection. Apply Clean function to remove air bubbles and debris. Adjusting the injection parameters. Typical parameters are: automatic injection mode; pi (Injection pressure): 3–4, ti (Injection time): 1.5, pc (Compensation pressure): 2; pw (Compensation pressure, continuous flow): 1–2 if needed.
<div class="critical">
<span class="critical-title">Critical:</span> To avoid flow problems, make sure there is positive hydrostatic pressure, avoid negative pressure, sucking medium for injection</div>
</li>
<li>Microinjection: the micromanipulator platform is shown in <a href="#fig2">Figure 2</a>D. Microinjection is performed at room temp. Zygotes are microinjected with Standard Control, Reverse Complement (RC), or LINE1 ASOs. ASO solutions are injected into the cytoplasm from a stock concentration of 1.5 mM (1×) or 0.75 mM (0.5×), utilizing 2–5 pl of solution per injection (See details in <a href="#mmc1">Methods video S1</a>). For late 2-cell stage injection, embryos are microinjected into the cytoplasm of both blastomeres (see injection details in <a href="#mmc2">Methods video S2</a>). See <a href="#troubleshooting">troubleshooting 3</a> and <a href="#troubleshooting">4</a>.
<div class="critical">
<span class="critical-title">Critical:</span> To avoid embryo lysis, leave injected embryos at room temperature for 20 min to recover before conducting embryo culture. See <a href="#troubleshooting">troubleshooting 5</a>.</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/916-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Zygote microinjection, related to step 22h</div>
<p>Apply negative pressure to aspirate the zygote using the holding pipette. Adjust the focus on the injection pipette, use the x-axis control knob to insert the injection pipette into the cytoplasm of the zygote (avoid the pronuclei), apply the injection pressure using the foot control pedal. Quickly withdraw the injection pipette and finish the microinjection by releasing the zygote by applying positive pressure on the holding pipette. Move the injected zygote using the injection pipette and continue by repeating the procedure until all zygotes have been injected.</p>
</figcaption>
</figure>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/916-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. 2-cell embryo microinjection, related to step 22h</div>
<p>Apply negative pressure to aspirate the 2-cell embryo (preferably at the equatorial region) by using the holding pipette. Adjust the focus on the injection pipette, use the x-axis control knob to insert the injection pipette into the cytoplasm of one of the blastomeres (avoid the nuclei), apply the injection pressure by using the foot control pedal. Quickly withdraw the injection pipette and apply the same injection procedure to the other blastomere. Finish the microinjection by releasing the 2-cell embryo by applying positive pressure to the holding pipette. Move the injected 2-cell embryo using the injection pipette and repeat the procedure until all the 2-cell embryos have been injected.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<ol start="23">
<li>Embryo culture: culture embryos in equilibrated KSOM (at least 2 h in the incubator) in KSOM drops (∼80 μL, covered with mineral oil).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Some or all embryos may be cultured in KSOM medium for 2–3 days to observe embryo progression and validate the effect of LINE1 depletion. Embryos microinjected with LINE1, but not Control/RC ASOs, will arrest at the 2-cell stage, instead of progressing to blastocyst stage (<a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/916-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Typical results following RC/LINE1 ASOs in ESCs and embryos</div>
<p>(A and B) LINE1 RNA FISH foci are seen in the nucleus of ESCs (DAPI), and foci are significantly reduced upon knockdown with LINE1 ASOs, scale, 10 μm.</p>
<p>(C and D) Typical LINE1 RNA FISH results in mouse blastocysts (C) and 2-cell embryos (D), Scale, 20 μm. (C) adapted with permission from <a href="#bib6">Percharde et al., 2018</a>.</p>
<p>(E and F) <i>In vitro</i> culture of embryos for 3 days after injection of 2-cell embryos with ASOs reveals 2–4 cell arrest upon LINE1 knockdown, scale 250 μm.</p>
</figcaption>
</figure>
<h3 id="sec3.3">Embryo RNA FISH</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>Embryo RNA FISH is performed according to (<a href="#bib3">Lin et al., 2014</a>). For imaging 2-cell stage embryos, fix embryos injected at the zygote stage 24 h post microinjection; for blastocyst stage embryos, culture injected embryos for 4 days and select blastocysts for fixation.</p>
<ol start="24">
<li>Fixation: transfer embryos into pre-cooled 100% methanol (∼180 μL) in 96 well U bottom by pulled (∼150 μm in diameter), bended (∼70°–80°) Pasteur pipette connected to embryo-handling mouth pipette and leave at −20°C for 20 min.</li>
<li>Rehydration: quickly wash the embryos in ice cold 50% methanol and transfer into PBS/PVA for rehydration for 10 min</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> During rehydration, embryos usually float to the surface of the medium. To facilitate sample collection, make microdrops (80 μL) in a 60 mm dish for washing.</div>
<ol start="26">
<li>Permeabilisation: permeabilise embryos by incubating for 20 min in 70% ethanol (180 μL) at room temperature in a 96-well plate.</li>
<li>Equilibration: transfer the embryos to the next row of the 96 well plate into 10% formamide wash buffer (eWB, 180 μL) for 10 min at room temperature.</li>
<li>Hybridization: embryos are cultured with probes diluted in hybridization buffer in microdrops (80 μL) in a 60 mm dish overlaid with mineral at 37°C overnight (8–12 h).</li>
<li>Washes: embryos are washed in 10% formamide wash buffer with Tween-20 (eWB+T, ∼180 μL in 96-well U bottom plate) at 37°C for 30 min.</li>
<li>Counterstaining: embryos are counterstained in 10% wash buffer with Tween-20 containing DAPI (eWB+T+DAPI) in 37°C for 30 min.</li>
<li>Mounting: embryos are placed into the middle of a slide, add four small droplets of the soft wax around the area to act as a buffer space, apply a coverslip and use a pencil to gently squeeze down to immobilize the embryos without them being squashed. Apply Vectashield medium to the edge of the coverslip and allow it to fill the space and mount the embryos. Absorb the surplus medium using a Kimwipe tissue. Use clear nail polish to seal the coverslip.</li>
<li>Imaging: Same as described above (step 20) except defining the scanning Z-axis values according to the nuclear regions.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>RNA FISH performed in ESCs should give 100+ LINE1 RNA foci per nucleus in RC ASO nucleofected ESCs, and significantly fewer upon LINE1 ASO treatment (<a href="#fig3">Figure 3</a>). Bright LINE1 RNA foci are also apparent in embryos (<a href="#fig3">Figure 3</a>). Successful LINE1 depletion should also lead to 2-cell arrest in embryos, and reduction in self-renewal in ESCs concomitant with de-repression of 2-cell related transcripts. Self-renewal inhibition is observable as a reduction in proliferation of ESCs after LINE1 ASOs; this may also be quantified by plating ESCs at low density and scoring the number of emergent colonies after 6–7 days.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>As with any RNA-based protocol, the success of the RNA FISH is very sensitive to potential RNAse contamination. Work quickly and with RNAse-free materials wherever possible. For embryo manipulations, this protocol relies on the skill of the person injecting, to make sure that the introduction of ASOs does not trigger lysis or death. This protocol does not include steps for immunofluorescence for colocalization of protein markers with the FISH signal; details for this however have been described by the probe manufacturers (<a href="https://www.biosearchtech.com/support/resources/stellaris-protocols">https://www.biosearchtech.com/support/resources/stellaris-protocols</a>) as well as by the original authors <a href="#bib7">Raj et al., 2008</a>, on their website <a href="https://sites.google.com/site/singlemoleculernafish/">https://sites.google.com/site/singlemoleculernafish/</a>.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Excessive cell death (step 7–10)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Ensure that ESCs before nucleofection are healthy and in the logarithmic stage of cell growth.</p>
<p>Make sure that the correct number of ESCs are nucleofected; we see more cell death when too few cells are used.</p>
<p>Get plates, media and pipettes ready to take out ESCs immediately after nucleofection so that they do not remain in the cuvette for long.</p>
<p>Similarly, after resuspension in nucleofection solution, proceed quickly to the next steps to minimize time in nucleofection solution.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>No RNA FISH signal (step 20)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Ensure all instruments and solutions are RNAse free and have RNAseOUT added fresh.</p>
<p>Co-label samples with a control RNA FISH probe in a different channel – e.g., TexasRed (e.g., Gapdh Cat# SMF-3002-1) to verify the FISH set up is working.</p>
<p>The duration of permeabilization in 70% ethanol can be varied.</p>
<p>Make sure microscope is set up for FISH and a long enough exposure time is used – we found confocals do not let enough light in to visualize signals.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Injection flow problem (step 22)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Ensure the tubing is connected properly without any leaks.</p>
<p>Adjust the pi and pw when the flow pressure is unstable.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Injection pipette clogging (step 22)</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Apply Clean function to temporarily solve the issue (step 22g). Replace the injection pipette if the problem remains unsolved.</p>
<p>ASO related problems: centrifuge the ASO before injection (step 22c). Aspirate the upper-phase after centrifugation, pre-heat ASO to 55°C–65°C or autoclave ASO to increase the solubility.</p>
<p>Pipette related problems (step 22a): pre-clean the injection pipettes by washing the pipettes with 70% ethanol, rinse with nuclease-free water and air dry before use.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Embryo Lysis (step 23)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Optimize the pulling conditions of the injection pipettes (step 22a).</p>
<p>Adjust the injection pressure (step 22g).</p>
<p>Avoid the nuclear region during injection. Coat the injection pipettes with Sigmacote if necessary (step 22a).</p>
<p>Make sure to recover injected embryos for ∼20 min at room temperature before performing embryo culture (step 23):</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Behringer, R., Gertsenstein, M., Nagy, K.V., and Nagy, A. (2014). Manipulating the Mouse Embryo: A Laboratory Manual (Cold Spring Harbor Laboratory Press). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/sref1">View at publisher</a></p>
<p id="bib9">Hooper, M., Hardy, K., Handyside, A., Hunter, S., and Monk, M. (1987). HPRT- deficient (Lesch-Nyhan) mouse embryos derived from germline colonization by cultured cells. Nature <i>326</i>, 292-295. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/optGAE7agBTaf">View at publisher</a></p>
<p id="bib2">Lin, C.J., Conti, M., and Ramalho-Santos, M. (2013). Histone variant H3.3 maintains a decondensed chromatin state essential for mouse preimplantation development. Development <i>140</i>, 3624-3634. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/sref2">View at publisher</a></p>
<p id="bib3">Lin, C.J., Koh, F.M., Wong, P., Conti, M., and Ramalho-Santos, M. (2014). Hira- mediated H3.3 incorporation is required for DNA replication and ribosomal RNA transcription in the mouse zygote. Dev. Cell <i>30</i>, 268-279. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/sref3">View at publisher</a></p>
<p id="bib4">Peddigari, S., Li, P.W., Rabe, J.L., and Martin, S.L. (2013). hnRNPL and nucleolin bind LINE-1 RNA and function as host factors to modulate retrotransposition. Nucleic Acids Res. <i>41</i>, 575-585. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/sref4">View at publisher</a></p>
<p id="bib5">Penzkofer, T., Dandekar, T., and Zemojtel, T. (2005). L1Base: from functional annotation to prediction of active LINE-1 elements. Nucleic Acids Res. <i>33</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/sref5">View at publisher</a></p>
<p id="bib6">Percharde, M., Lin, C.J., Yin, Y., Guan, J., Peixoto, G.A., Bulut-Karslioglu, A., Biechele, S., Huang, B., Shen, X., and Ramalho-Santos, M. (2018). A LINE1- nucleolin partnership regulates early development and ESC identity. Cell <i>174</i>, 391-405. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/sref6">View at publisher</a></p>
<p id="bib7">Raj, A., van den Bogaard, P., Rifkin, S.A., van Oudenaarden, A., and Tyagi, S. (2008). Imaging individual mRNA molecules using multiple singly labeled probes. Nat. Methods <i>5</i>, 877-879. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/sref7">View at publisher</a></p>
<p id="bib8">Ying, Q.L., Wray, J., Nichols, J., Batlle-Morera, L., Doble, B., Woodgett, J., Cohen, P., and Smith, A. (2008). The ground state of embryonic stem cell self- renewal. Nature <i>453</i>, 519-523. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00433-0/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Michelle Percharde (<a href="mailto:m.percharde@lms.mrc.ac.uk">m.percharde@lms.mrc.ac.uk</a>)</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>No dataset or code was generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by a UKRI Future Leaders Fellowship (MC_EX_MR/S015930/1) and funding from the <a href="https://doi.org/10.13039/501100000265">Medical Research Council</a> (MRC, MC_UP_1605/4) to M.P, a Royal Society of Edinburgh Personal Research Fellowship funded by the <a href="https://doi.org/10.13039/100012095">Scottish Government</a> and <a href="https://doi.org/10.13039/501100000265">MRC</a> Centre Grant (MR/N022556/1) to C.-J.L., the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (R01GM123556) to M.R.-S., and the Canada 150 Research Chair in Developmental Epigenetics and the University of Toronto Medicine by Design Program to M.R.-S.</p>
<h3>Author contributions</h3>
<p>M.P. and C.-J.L. performed the experiments and data analysis. M.R-S. supervised the project. M.P., C-J.L., and M.R-S. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>