<article>
<section>
<h1 id="header">A high-throughput protocol for monitoring starvation-induced autophagy in real time in mouse embryonic fibroblasts</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Ada Nowosad<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a></sup> and Arnaud Besson<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Molecular, Cellular and Developmental Biology Department (MCD), Centre de Biologie Intégrative (CBI), University of Toulouse, CNRS, UPS, 31062 Toulouse, France</p>
<p id="aff2"><sup>2</sup>Laboratory of Molecular Cancer Biology, VIB Center for Cancer Biology, KU Leuven, Leuven, Belgium</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:arnaud.besson@univ-tlse3.fr">arnaud.besson@univ-tlse3.fr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100966">10.1016/j.xpro.2021.100966</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Autophagy measurement has been challenging due to the transient nature of autophagy vesicles, in which degradation of cargo occurs. Here, we present a protocol to monitor starvation-induced autophagy using a live high-throughput microscopy system in a fast and automated manner without the need for sample preparation. We provide a detailed protocol describing the generation of turboGFP-LC3B expressing mouse embryonic fibroblasts (MEFs), the measurement of autophagy over time and the analysis of data.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref12">Nowosad et al. (2020</a>, <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref11">2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detailed protocol to monitor autophagy in live cells based on turboGFP-LC3B dynamics</li>
<li>Stable expression of turboGFP-LC3B as a reporter for automated autophagy measurement</li>
<li>Rapid quantification of effects of starvation in live cells over time</li>
<li>Use of high-throughput microscopy to monitor metabolic response of cells to stress</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1186-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Generate cell lines stably expressing turboGFP-LC3B</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>The first step to measure autophagy in live cells is to generate cells stably expressing a protein that allows monitoring the autophagic response in real time. LC3B/MAP1LC3B/ATG8 is a gold standard for measuring autophagic activity. Upon autophagy induction, a cytoplasmic pool of LC3B (LCB I) is covalently conjugated to phosphatidylethanolamine. This lipidated form, called LC3B II, localizes on autophagosomes (<a href="#bib5">Kabeya, 2000</a>). Thus, the presence of autophagic vesicles may be assessed by measuring LC3B II levels (<a href="#bib8">Mizushima and Murphy, 2020</a>). Importantly, upon fusion of autophagosomes with lysosomes, LC3B II present on the inner membrane of autophagosomes is degraded by lysosomal enzymes, whereas LC3B II located on the outer membrane is converted back to the LC3B I form by cleavage from the membrane and recycled in the cytoplasm, resulting in an overall decrease of LC3B levels. Therefore, LC3B dynamics are routinely used in autophagy research (<a href="#bib6">Klionsky et al., 2021</a>). Previous studies have shown the utility of using a GFP-LC3B reporter to measure autophagy by monitoring lipidated LC3B II following saponin extraction of cytoplasmic LC3B I (<a href="#bib1">Eng et al., 2010</a>) or by detecting the free GFP fragment generated upon autophagy induction by immunoblotting (<a href="#bib10">Ni et al., 2011</a>). Since during nitrogen starvation both LC3B I and -II levels decline, we developed this protocol to automatically monitor the amount of LC3B in live cells without requirement for sample preparation. Another autophagy marker that can also be monitored using the same procedure is SQSTM1/p62 (<a href="#bib6">Klionsky et al., 2021</a>).</p>
<p>We chose to use turboGFP as a reporter as it is characterized by higher fluorescence intensity than eGFP and is easily detected using GFP/FITC filters (<a href="#bib2">Evdokimov et al., 2006</a>). LC3B and p62 were cloned as fusion proteins with turboGFP in their N-terminus in a retroviral expression vector carrying a puromycin resistance gene. TurboGFP was cloned in the EcoRI and XhoI sites of a modified pQCXIP vector (Clontech) and then either LC3B or p62 was inserted into the NotI and PmeI sites of the resulting pQCXIP-tGFP vector. In this vector, the GFP fusion protein and puromycin resistance are expressed from the same transcript due to the presence of an IRES. These constructs were validated in <a href="#bib11">Nowosad et al. (2021)</a> and are available from the Lead Contact upon request.</p>
<p>To establish the turboGFP-LC3B cell line, Mouse Embryonic Fibroblasts (MEFs) were retrovirally infected. Retrovirus production was performed using ecotropic Phoenix cells generated by Gary Nolan’s lab (Stanford University), which is based on a HEK293T cell line (human embryonic kidney cells transformed with adenovirus E1a and carrying a temperature sensitive T antigen co-selected with neomycin) modified for retroviral production. For more information on the original protocol and troubleshooting of viral production, please refer to the Nolan lab website: <a href="https://web.stanford.edu/group/nolan/_OldWebsite/protocols/pro_helper_dep.html">https://web.stanford.edu/group/nolan/_OldWebsite/protocols/pro_helper_dep.html</a></p>
<div class="note">
<span class="note-title">Note:</span> The protocol below describes the specific steps for measuring starvation-induced autophagy. We tested this protocol for different types of starvation media (no amino acids, no glucose, no serum) but it may be used to monitor autophagy induced by any kind of stress (e.g. drugs, oxygen levels…) or basal autophagy. However, it may require more optimization steps to determine LC3B dynamics and cell response to stress (see <a href="#troubleshooting">troubleshooting</a>).</div>
<div class="note">
<span class="note-title">Note:</span> This protocol has been optimized for immortalized MEFs. Use of different cells may require some modifications.</div>
<div class="note">
<span class="note-title">Note:</span> High-throughput imagers require a robust fluorescent signal such as turboGFP. The use of less bright fluorochromes such as eGFP is not recommended for this protocol (see <a href="#troubleshooting">troubleshooting</a>).</div>
<ol>
<li>DAY 1:</li>
</ol>
<p>PM: Seed 4 × 10<sup>6</sup> ecotropic (to infect murine cells) or amphotropic (to infect human cells) phoenix cells in 10 cm Petri dish.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Determine optimal antibiotic concentration for cell selection (this step needs to be done only once for each cell line to be infected). Seed cells to be infected into 24-well culture plate at a density that will result in approximately 75% confluence the following day in the appropriate culture medium and incubate overnight in a tissue culture incubator. The next day replace normal medium to antibiotic selection medium. To this end, dilute antibiotic (here puromycin) in culture media to obtain final concentrations from 0.5 to 10 μg/mL. Use medium without antibiotic as a control. Monitor cells daily starting 24 h after treatment using a microscope. Alternatively, colorimetric viability or cytotoxicity assays may be used to assess the cell death (e.g., CellTox, CellTiterGlow [Promega]). The duration of selection depends on the antibiotic used, for puromycin, the selection should be complete within 2–4 days. Determine the minimal concentration of antibiotic at which all uninfected cells die.</div>
<ol start="2">
<li>DAY 2:</li>
</ol>
<p>AM: Transfect phoenix cells with pQCXIP-turboGFP-LC3B or pQCXIP-turboGFP-p62 using a method of choice. We obtained good viral production (&gt;70% of infected cells) using a Calcium Phosphate transfection protocol and 10 μg of DNA.</p>
<p>Incubate cells for 24 h at 37°C.</p>
<ol start="3">
<li>DAY 3:</li>
</ol>
<p>AM: Change medium of transfected cells and replenish with 5–6 mL of fresh medium per dish.</p>
<p>PM: Seed cells to be infected at 20%–30% confluence.</p>
<ol start="4">
<li>DAY 4: AM
<ol type="a">
<li>Collect the supernatant of Phoenix cells using a 5–10 mL syringe and filter through 0.45 μm filter.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The viral supernatant can be concentrated by centrifugation or other methods (e.g. Retro-X™ Concentrator [Takara]).</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The supernatant with viral particles may be frozen and stocked at −80°C for later use. However, it may cause a major loss of viral titer and is not recommended unless using concentrated viral supernatants.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Users may determine viral titer using a method of choice. In our experiments, the viral load or infection rate was evaluated by determining the percentage of cells expressing turboGFP following infection using an immunofluorescence microscope or the Incucyte, but other methods such as FACS or qPCR may be used. Virus titration is especially useful when freezing supernatants for later use.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If you wish to repeat viral infection, add 5–6 mL of fresh medium to the phoenix cells and put them back at 37°C. We always do three rounds of infection before proceeding to antibiotic selection of infected cells.</div>
</li>
<li>Aspirate medium of cells to be infected and add 2–3 mL of filtered supernatant and 2–3 mL of fresh cell culture medium.</li>
<li>Immediately add Polybrene at a final concentration of 5 μg/mL to infected cells to improve transduction efficiency, swirl medium gently.
<div class="optional">
<span class="optional-title">Optional:</span> PM: Repeat infection (steps a–c) with the same procedure.</div>
</li>
</ol>
</li>
</ol>
<ol start="5">
<li>DAY 5:</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> AM: Repeat infection (steps a–c) using the same procedure.</div>
<ol start="6">
<li>DAY 7 (or 48 h after last infection):</li>
</ol>
<p>Start selection using the proper antibiotic (puromycin for pQCXIP). The concentration depends on the cell type used and should be determined experimentally (see Alternative in DAY 1 of infection protocol). For MEF selection, puromycin was used at a final concentration of 2 μg/mL. The selection should be complete within 48 h of antibiotic treatment but puromycin was added to the cell medium at all times to maintain selection pressure.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Infected cells may be selected by FACS based on turboGFP expression, which is detectable in the FITC/GFP channel found in most FACS instruments. The excitation/emission maxima of turboGFP are 482/502 nm.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Both antibiotic selection and FACS sorting will result in a polyclonal turboGFP+ population. A limitation of polyclonal populations is that expression levels of the protein of interest will not be homogenous within the cell population (See <a href="#sec6.9">problem 5</a>). If you prefer working on monoclonal populations, turboGFP-expressing cell lines should arise from single-cell clones. You can obtain single cell clones either by FACS sorting or by performing limited dilution assay in 96-well plates. Note that not all cell lines are compatible with single cell expansion.</div>
<ol start="7">
<li>DAY 9: Start to expand cells.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Once expanded, you can freeze expanded cells in DMSO-containing medium/FBS.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Viral production and infection should be handled using appropriate biosafety precautions and according to regulations in your institution/country. Personal protective equipment (PPE) should be worn at all times when working with viral particles and infected cells.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>LC3B</td>
<td>Cell signalling TECHNOLOGY</td>
<td>Cat#4108 RRID:AB_2137703</td>
</tr>
<tr>
<td>β-actin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A2228 RRID:AB_476697</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Polybrene (Hexadimethrine bromide)</td>
<td>Sigma-Aldrich</td>
<td>Cat#H9268</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Phoenix-ECO cells</td>
<td>ATCC</td>
<td>Cat#CRL-3214 RRID:CVCL_H717</td>
</tr>
<tr>
<td>Phoenix-AMPHO cells</td>
<td>ATCC</td>
<td>Cat#CRL-3213 RRID:CVCL_H716</td>
</tr>
<tr>
<td>Mouse embryo fibroblasts</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pQCXIP turboGFP LC3</td>
<td><a href="#bib11">Nowosad et al., (2021)</a></td>
<td>N/A</td>
</tr>
<tr>
<td>pQCXIP turboGFP p62</td>
<td><a href="#bib11">Nowosad et al., (2021)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>IncuCyte ZOOM 2018A</td>
<td>Essen BioScience</td>
<td>N/A</td>
</tr>
<tr>
<td>Microsoft Excel 2010</td>
<td>Microsoft</td>
<td>N/A</td>
</tr>
<tr>
<td>GraphPad Prism 8.0</td>
<td>Prism</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>DMEM w/o glucose cell medium</td>
<td>Sigma-Aldrich</td>
<td>Cat#D5030</td>
</tr>
<tr>
<td>DMEM w/o amino acids cell medium</td>
<td>USBiological life sciences</td>
<td>Cat#D9800-13</td>
</tr>
<tr>
<td>DMEM high glucose medium</td>
<td>Sigma-Aldrich</td>
<td>Cat#D6429,</td>
</tr>
<tr>
<td>Earle’s Balanced Salt Solution (EBSS)</td>
<td>Sigma-Aldrich</td>
<td>Cat#E7510</td>
</tr>
<tr>
<td>IncuCyte® FLR, ZOOM or S3, equipped with 10× or 20× objective</td>
<td>Sartorius</td>
<td>N/A</td>
</tr>
<tr>
<td>dialysis tubing 3,500 MW cut-off</td>
<td>Spectrum Labs</td>
<td>Cat#132111</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Conduct IncuCyte experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Varies in function of length of starvation, but typically 2–3 days</div>
<p>The monitoring of turboGFP-LC3B signal in live cells upon starvation was performed using an IncuCyte system (<a href="https://www.sartorius.com/en/products/live-cell-imaging-analysis/live-cell-analysis-instruments">https://www.sartorius.com/en/products/live-cell-imaging-analysis/live-cell-analysis-instruments</a>). This live-imaging platform consists of a microscope residing in a standard tissue culture incubator maintained at 37°C and 5% CO<sub>2</sub> that allows automated image acquisition and analysis through the IncuCyte software. This protocol was tested using IncuCyte FLR and ZOOM models but more recent versions should also be compatible. The main difference between FLR and ZOOM is that the latter has both green (Ex: 460 [440,480] nm; Em: 524 [504,544] nm) and red (Ex: 585 [565,605] nm; Em: 635 [625,705] nm) LEDs, whereas the FLR offers only the green fluorescence. Both have phase contrast imaging.</p>
<p>Importantly, this protocol may be used to monitor autophagy using any high-throughput imaging platform (i.e., N Cell Analyzer [GE] or Cytation Multi-Mode Reader [BioTek/Agilent]) but some adjustments may be required depending on the system.</p>
<p>During starvation experiments, DMEM high glucose supplemented with 2 μg/mL penicillin–streptomycin and 10% FBS was used as a control and cells were starved in starvation media (for example DMEM without glucose, DMEM without amino acids or Earle’s Balanced Salt Solution [EBSS], supplemented with 2 μg/mL penicillin–streptomycin and 10% dialyzed FBS).</p>
<p>FBS was dialyzed at 4°C in 3,500  MW cut-off dialysis tubing against PBS for 6 h and then overnight to remove amino acids and glucose.</p>
<p>For more detailed starvation protocol, refer to (<a href="#bib12">Nowosad et al., 2020</a>) and (<a href="#bib11">Nowosad et al., 2021</a>).</p>
<p>In the current manuscript, starvation was performed using EBSS medium supplemented with 10% FBS and non-essential amino acids.</p>
<ol>
<li>Seed turboGFP-LC3B expressing cells into multi-well plates.
<ol type="a">
<li>Trypsinize cells, inactivate trypsin with serum-containing medium and count cells using a method of choice.</li>
<li>Seed appropriate number of cells to obtain 50%–70% confluence the day of starvation (the exact density depends on the type and duration of treatment). MEFs were seeded at 12 000 cells per well in 24-well plates but the optimal number of cells should be determined for each cell types in a pilot experiment. Allow cells to adhere overnight in the incubator.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Seeding the cells at the right density and without cell clusters is crucial for the success of the experiment. Autophagy was shown to be affected by contact inhibition. Thus, particular attention should be paid to obtain the appropriate seeding density. See <a href="#troubleshooting">troubleshooting</a> for more information.</div>
<ol start="2">
<li>Starve cells.
<ol type="a">
<li>Remove medium and wash cells twice with warm PBS followed by one wash with starvation medium. Add full (control) or starvation medium to the wells.</li>
</ol>
</li>
<li>Begin plate imaging with IncuCyte.
<ol type="a">
<li>Place plate in available slot and ensure that plate is seated securely.</li>
<li>Open IncuCyte software and select the appropriate plate type and imaging pattern. Schedule scan every 2 or 4 h for 2–5 days (depending on the cell type and starvation medium), selecting phase and green fluorescence channels.</li>
</ol>
</li>
<li>Complete IncuCyte experiment.
<ol type="a">
<li>Remove plate from scan schedule in the IncuCyte software.</li>
<li>Remove plate from the IncuCyte.</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Generate processing definition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45–60 min</div>
<p>Creating a "processing definition" aims to “train the software” using the selected images to determine the optimal settings for automated object detection in all images generated in the experiment. This step is required only the first time the experiment is performed. Once defined and saved in the software, the same workflow can be applied to subsequent experiments.</p>
<div class="note">
<span class="note-title">Note:</span> This step is specific for the Zoom software and is not present in the FLR version.</div>
<div class="note">
<span class="note-title">Note:</span> You can also generate a new processing definition for previously analyzed experiments.</div>
<ol start="5">
<li>Add representative images to Image Collection. We recommend using at least 2–3 images for each condition representing different time points of the experiment.</li>
<li>Define parameters for object detection (<a href="#fig1">Figure 1</a>) for phase contrast and green fluorescence channels.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1186-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Processing definition for autophagy assay</div>
<p>Images represent MEFs expressing turboGFP-LC3B with and without masks using parameters set up in processing definition. The merge channel shows overlap between the green mask (purple) and the phase mask (yellow). For phase channel analysis, the following parameters were applied: segmentation adjustment: 1.1; adjust size: −1 pixel; area: min 200 μm. For green channel analysis, the following parameters were used: Top-Hat segmentation; radius: 90 μm; threshold (GCU): 2; area: min 200 μm. Scale bars are 300 μm.</p>
</figcaption>
</figure>
<ol type="a">
<li>Define the parameters for the phase contrast channel.
<ol type="i">
<li>Click <i>Preview Current</i> and apply the <i>Confluence Mask</i> to visualize the preset values already present in the analysis software.</li>
<li>Refine the parameters within the confluence mask:
<ul>
<li>Set up <i>Segmentation adjustment</i>, which aims at adjusting the Background/Cells slider bar. We used a 1.1 value.</li>
<li>Clean up by adjusting size. This parameters allows to either enlarge (by choosing a positive value) or shrink (by using a negative value) the mask by a specific number of pixels. We set up −1, which means that the mask was reduced by 1 pixel.</li>
<li>Define minimal object area to filter out dead cells and debris (in the Zoom software, distance measurement can be made by clicking a ruler icon on the right side of the image). In our analysis, we excluded objects smaller than 200 μm<sup>2</sup>.</li>
<li>Preview all the images. If you are satisfied with the parameters, proceed to the processing definition of the green channel.</li>
</ul>
</li>
</ol>
</li>
<li>Define the Analysis Parameters for the green Channel.
<ol type="i">
<li>Define the mode of object segmentation by selecting top-hat processing, which allows subtracting the local background from brightly fluorescent objects within a given radius</li>
<li>Set up a fluorescence threshold. Increasing the threshold will eliminate masking of background while decreasing the threshold will include dimmer objects. We used a threshold of 2 GCU.</li>
<li>Adjust edge sensitivity to optimize the segmentation mask for adjacent objects. As we will be extracting confluence and not the number or area of objects, there is no need to switch the Edge Split on.</li>
<li>As above, define minimal object area to filter out dead cells and debris. In our analysis, we excluded objects smaller than 200 μm<sup>2</sup>.</li>
<li>Preview all images to control the segmentation mask. If you are satisfied with the parameters, launch the analysis.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.3">Extracting and analyzing data</h3>
<p>This protocol explains how to exploit and further analyze data generated with the IncuCyte software.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 h (depending on the number of wells and conditions)</div>
<ol start="7">
<li>Extract raw counts from IncuCyte software.
<ol type="a">
<li>Export the results for confluence (percent) calculations for phase contrast and green channels.</li>
</ol>
</li>
<li>Copy raw data to Excel file and calculate the ratio of the percentage of confluence in the green channel by the confluence in the phase contrast channel using the following formula:
<ul>
<li>Green confluence (percent)/Phase contrast confluence (percent)</li>
</ul>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Instead of normalizing data with phase confluence, you can label cell nuclei using an appropriate red probe (e.g. IncuCyte® Nuclight Rapid Red Dye) and normalize the number of fluorescent green objects [in this case edge split should be turned on to precisely determine the number of objects] by the number of nuclei (only applicable in IncuCyte models equipped with more than one fluorescence channel).</div>
<ol start="9">
<li>Normalize the obtained values with that of the first scan (time 0 h).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Import your data into graphing software to create graphs. We used GraphPad Prism for statistical analysis and data visualization but this step may also be performed in Excel.</div>
<ol start="10">
<li>Generate a curve representing the levels of green fluorescence normalized by the number of cells (<a href="#fig2">Figure 2</a>). In the example shown <a href="#fig2">Figure 2</a>, cells were starved in EBSS supplemented with 10% serum and non-essential amino acids for the indicated times. Upon starvation, there is a progressive decrease of turboGFP-LC3B fluorescence (<a href="#fig2">Figure 2</a>A). A similar observation is made by immunoblotting endogenous LC3B (<a href="#fig2">Figure 2</a>B) at various times following starvation, indicating that turboGFP-LC3B behaves similarly to the endogenous protein in starvation conditions.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1186-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Quantification of autophagy in turboGFP-LC3B MEFs</div>
<p>(A) turboGFP-LC3B MEFs were kept either in full or starvation medium (EBSS supplemented with 10% serum and non-essential amino acids) and fluorescent object confluence was measured every 2 h for 72 h. Fluorescent object confluence results were divided by phase confluence results at the same time point and normalized with the first scan (0 h).</p>
<p>(B) Immunoblot of LC3B levels in MEFs starved for the indicated times as in (A). Actin was used as loading control.</p>
</figcaption>
</figure>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Green objects represent cells containing LC3B-positive autophagy vesicles. During starvation, LC3B fluorescence should decrease and its decline is inversely correlated with autophagic activity (<a href="#mmc1">Methods video S1</a> and <a href="#fig3">Figure 3</a>). Conversely, little or no loss of signal should be observed in control full medium conditions.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1186-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. TurboGFP-LC3B fluorescence during starvation</div>
<p>Examples of images showing MEFs expressing turboGFP-LC3B during a 3-day starvation in EBSS supplemented with 10% serum and non-essential amino acids. Scale bars are 300 μm.</p>
</figcaption>
</figure>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1186-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Effect of starvation on turboGFP-LC3B signal</div>
<p>MEFs expressing turboGFP-LC3B were starved in EBSS supplemented with 10% serum and non-essential amino acids for 72 h and scanned in IncuCyte every 2 h. The movie was generated using the IncuCyte software by assembling all images taken in a representative area within the starvation period (12 images per day for 3 days).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Data extracted from the IncuCyte software were exported to Excel to perform the normalization. Normalized values were copied to GraphPad 9 (Prism) to generate graphs. Results from at least 3 independent experiments are needed to perform statistical analysis. This may be done in GraphPad software by comparing each time point between several groups. This may be done using 2-way ANOVA followed by multiple testing correction (e.g., Bonferroni test). For examples and more details on statistical analyses, see <a href="#bib11">Nowosad et al., (2021)</a>.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol was validated for adherent monolayer cell cultures and does not work for 3D or suspension cultures. For the latter, an option may be to coat culture plates with agents that promote adhesion (e.g., collagen, poly-L-lysine) prior to cell seeding. In addition, suspension cells have a tendency to aggregate, which may make segmentation difficult during image processing.</p>
<p>The IncuCyte system has low sensitivity, requiring very bright fluorescent signal for detection, and some fluorochromes may not be compatible. For instance, while this protocol allows detecting turboGFP-LC3B. In similar experiments using eGFP-LC3B, the signal was too weak to distinguish it from background (<a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1186-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Comparison between turboGFP-LC3B and eGFP-LC3B</div>
<p>MEFs were infected with plasmids expressing either turboGFP-LC3B (left) or eGFP-LC3B (right) and imaged using the IncuCyte. Representative images are shown. Scale bars are 300 μm.</p>
</figcaption>
</figure>
<p>This protocol is based on LC3B degradation as autophagy indicator, assuming that there is no protein neosynthesis. Some pilot experiments with protein synthesis inhibitors (e.g., Cycloheximide) may be required to exclude the possibility of LC3B synthesis upon autophagy induction. We also recommend using another autophagy marker in parallel to validate the results. We obtained good results with MEFs expressing turboGFP-p62 (<a href="#mmc2">Methods video S2</a>). Furthermore, in addition to autophagic degradation, LC3B levels are regulated by multiple pathways, including ubiquitin-mediated proteasome degradation (<a href="#bib3">Jia and Bonifacino, 2019</a>, <a href="#bib4">2020</a>) and transcriptional regulation (<a href="#bib14">Settembre et al., 2011</a>). Therefore, it is important to determine the dynamics of LC3B before using LC3B as a reporter protein. This may be done by monitoring MAP1LC3 RNA levels by using RT-qPCR and LC3B protein levels by immunoblotting (<a href="#fig2">Figure 2</a>B).</p>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1186-Mmc2.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S2. Effect of starvation on turboGFP-p62 signal</div>
<p>MEFs expressing turboGFP-p62 were starved in EBSS supplemented with 10% serum and non-essential amino acids for 72 h and scanned in IncuCyte every 2 h. The movie was generated using the IncuCyte software by assembling all images taken in a representative area within the starvation period (12 images per day for 3 days).</p>
</figcaption>
</figure>
<p>The IncuCyte Zoom software does not allow analyzing fluorescence in single cells over time. However, the Incucyte® Cell-by-Cell Analysis Software permits to perform the segmentation and treat exported measurement as single-cell data. This module is available for IncuCyte® SX1, SX5 and S3 platforms as well as on other high-content imaging devices. Alternatively, recently developed microfluidic platforms (such as EVORION CellCity) can be used to refine the analysis and obtain single cell resolution. Furthermore, machine-learning approaches (<a href="#bib9">Morone et al., 2020</a>) may be helpful to classify fluorescent cells into different categories based on their fluorescence intensity or pattern.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low viral titer</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Split Phoenix cells regularly (1:5 ratio) and never let them reach confluence. Every ∼20 passages, select Phoenix cells by adding Hygromycin (at 300 ug/mL) and Diptheria Toxin (1 ug/mL) in culture medium for one week. Try to use low-passage Phoenix cells.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Cell clusters within wells.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>It is very important to seed cells homogeneously and at the appropriate density when performing autophagy assays as contact inhibition is known to affect autophagy (<a href="#bib7">Leontieva et al., 2014</a>; <a href="#bib13">Pavel et al., 2018</a>) (<a href="#fig5">Figure 5</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1186-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Impact of cell clustering on turboGFP-LC3B signal</div>
<p>(A) MEFs clusters due to incorrect cell seeding technique. Cell clusters exhibit high turboGFP-LC3B fluorescence levels due to induction of autophagy by contact inhibition.</p>
<p>(B) Correctly seeded cells. Scale bars are 300 μm.</p>
</figcaption>
</figure>
<p>To improve cell seeding, we recommend preparing the cell suspension at the final concentration in a larger volume in a tube, homogenize it by pipetting and distribute evenly to all wells from that same solution instead of adding a small volume of concentrated cell suspension to the wells.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>High background fluorescence in the green channel.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Low riboflavin culture media (&lt;0.2 mg/L) may be used to reduce green background fluorescence.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>High mortality of starved cells.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Every cell type responds differently to starvation. We recommend performing a pilot experiment to determine how cells respond to different types of metabolic stress (amino acid, glucose, serum starvation…). It will help adjusting the composition of the starvation medium and the duration of starvation.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Only a fraction of LC3B<sup>+</sup> cells are detected by the IncuCyte.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>As the antibiotic selection results in a polyclonal LC3B<sup>+</sup> population, LC3B levels may vary among cells, leading to heterogeneous signal. As fluorescence intensity is normalized by the initial number of cells expressing GFP-LC3B, this protocol may be also applied to heterogeneous population. If it is preferable to work on a more homogeneous cell population, single cell GFP<sup>+</sup> cells may be isolated by FACS and expanded in culture. However, be aware that the selection pressure that occurs through cell culture passaging may create deviations in how a clonal cell line responds to starvation over time. Therefore, we recommend to store your clonal cell lines at a low passage and to use similar low passage number in each experiment.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Loss of GFP signal in necrotic cells</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Necrosis-triggered increased in membrane permeability may lead to the leakage of LC3B to the cytoplasm and decrease in green fluorescence signal. The nature of cell death (apoptosis versus necrosis) depends on cell type and stress conditions. We recommend to discriminate cell death response in experimental conditions before starting to monitor autophagy using turboGFP-LC3B. To this end, we have successfully used the IncuCyte Caspase-3/7 Dye for Apoptosis (SARTORIUS) reagent but other approaches such as Annexin V combined with propidium iodide (PI) staining may be used for FACS experiment.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Cells lose turboGFP signal over time</p>
<h3 id="sec6.14">Potential solution</h3>
<p>It is common that cells lines lose GFP expression over time due to selective pressure in culture (turboGFP expression may results in a growth disadvantage compared to cells wild-type cells). Clonal cell expansion (see <a href="#sec6.9">problem 5</a>) or maintaining cells under antibiotic selection during the expansion step may help to overcome this problem. We recommend storing infected cells post-selection to be able to go back to early passage cells in case of inconsistent results.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Eng, K.E., Panas, M.D., Karlsson Hedestam, G.B., and McInerney, G.M. (2010). A novel quantitative flow cytometry-based assay for autophagy. Autophagy <i>6</i>, 634-641. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref1">View at publisher</a></p>
<p id="bib2">Evdokimov, A.G., Pokross, M.E., Egorov, N.S., Zaraisky, A.G., Yampolsky, I.V., Merzlyak, E.M., Shkoporov, A.N., Sander, I., Lukyanov, K.A., and Chudakov, D.M. (2006). Structural basis for the fast maturation of Arthropoda green fluorescent protein. EMBO Rep. <i>7</i>, 1006-1012. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref2">View at publisher</a></p>
<p id="bib3">Jia, R. and Bonifacino, J.S. (2019). Negative regulation of autophagy by UBA6-BIRC6-mediated ubiquitination of LC3. Elife <i>8</i>, e50034. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref3">View at publisher</a></p>
<p id="bib4">Jia, R. and Bonifacino, J.S. (2020). Regulation of LC3B levels by ubiquitination and proteasomal degradation. Autophagy <i>16</i>, 382-384. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref4">View at publisher</a></p>
<p id="bib5">Kabeya, Y. (2000). LC3, a mammalian homologue of yeast Apg8p, is localized in autophagosome membranes after processing. EMBO J. <i>19</i>, 5720-5728. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref5">View at publisher</a></p>
<p id="bib6">Klionsky, D.J., Abdel-Aziz, A.K., Abdelfatah, S., Abdellatif, M., Abdoli, A., Abel, S., Abeliovich, H., Abildgaard, M.H., Abudu, Y.P., Acevedo-Arozena, A., et al. (2021). Guidelines for the use and interpretation of assays for monitoring autophagy. Autophagy <i>17</i>, 1-382. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref6">View at publisher</a></p>
<p id="bib7">Leontieva, O.V., Demidenko, Z.N., and Blagosklonny, M.V. (2014). Contact inhibition and high cell density deactivate the mammalian target of rapamycin pathway, thus suppressing the senescence program. Proc. Natl. Acad. Sci. <i>111</i>, 8832-8837. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref7">View at publisher</a></p>
<p id="bib8">Mizushima, N. and Murphy, L.O. (2020). Autophagy assays for biological discovery and therapeutic development. Trends Biochem. Sci. <i>45</i>, 1080-1093. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref8">View at publisher</a></p>
<p id="bib9">Morone, D., Marazza, A., Bergmann, T.J., and Molinari, M. (2020). Deep learning approach for quantification of organelles and misfolded polypeptide delivery within degradative compartments. Mol. Biol. Cell <i>31</i>, 1512-1524. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref9">View at publisher</a></p>
<p id="bib10">Ni, H.M., Bockus, A., Wozniak, A.L., Jones, K., Weinman, S., Yin, X.M., and Ding, W.X. (2011). Dissecting the dynamic turnover of GFP-LC3 in the autolysosome. Autophagy <i>7</i>, 188-204. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref10">View at publisher</a></p>
<p id="bib11">Nowosad, A., Creff, J., Jeannot, P., Culerrier, R., Codogno, P., Manenti, S., Nguyen, L., and Besson, A. (2021). p27 controls autophagic vesicle trafficking in glucose-deprived cells via the regulation of ATAT1-mediated microtubule acetylation. Cell Death Dis <i>12</i>, 481. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref11">View at publisher</a></p>
<p id="bib12">Nowosad, A., Jeannot, P., Callot, C., Creff, J., Perchey, R.T., Joffre, C., Codogno, P., Manenti, S., and Besson, A. (2020). p27 controls Ragulator and mTOR activity in amino acid-deprived cells to regulate the autophagy-lysosomal pathway and coordinate cell cycle and cell growth. Nat. Cell Biol. <i>22</i>, 1076-1090. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref12">View at publisher</a></p>
<p id="bib13">Pavel, M., Renna, M., Park, S.J., Menzies, F.M., Ricketts, T., Fullgrabe, J., Ashkenazi, A., Frake, R.A., Lombarte, A.C., Bento, C.F., et al. (2018). Contact inhibition controls cell survival and proliferation via YAP/TAZ-autophagy axis. Nat. Commun. <i>9</i>, 2961. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref13">View at publisher</a></p>
<p id="bib14">Settembre, C., Malta, C.D., Polito, V.A., Arencibia, M.G., Vetrini, F., Erdin, S., Erdin, S.U., Huynh, T., Medina, D., Colella, P., et al. (2011). TFEB links autophagy to lysosomal biogenesis. Science <i>332</i>, 1429-1433. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00672-9/sref14">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Arnaud Besson (<a href="mailto:arnaud.besson@univ-tlse3.fr">arnaud.besson@univ-tlse3.fr</a>).</p>
<h4>Materials availability</h4>
<p>All unique/stable reagents generated in this study are available from the lead contact with a completed materials transfer agreement.</p>
<h4>Data and code availability</h4>
<p>This protocol does not involve any specific dataset or code.</p>
<h3>Acknowledgments</h3>
<p>The team of A.B. is supported by funds from the Fondation Toulouse Cancer Santé, the <a href="https://doi.org/10.13039/501100006364">Institut National du Cancer</a> (INCa_14854), and an “FRM Equipes” grant (EQU202103012639) from the <a href="https://doi.org/10.13039/501100002915">Fondation pour la Recherche Médicale</a>.</p>
<h3>Author contributions</h3>
<p>A.N. designed and performed the experiments. A.N. and A.B. wrote the paper.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>