<article>
<section>
<h1 id="header">A flow-cytometry-based assessment of global protein synthesis in human senescent cells</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Yeonghyeon Lee,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Jaejin Kim,<sup><a href="#aff1">1</a></sup> Taerang Jeon,<sup><a href="#aff1">1</a></sup> Kyeonghwan Roh,<sup><a href="#aff1">1</a></sup> Mi-Sung Kim,<sup><a href="#aff1">1</a></sup> and Chanhee Kang<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>School of Biological Sciences, Seoul National University, Seoul 08826, South Korea</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:chanhee.kang@snu.ac.kr">chanhee.kang@snu.ac.kr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100809">10.1016/j.xpro.2021.100809</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Senescent cells constantly experience stressful conditions and restrain their protein translation to cope with it. Here, we present a detailed protocol to measure the rate of global protein synthesis using L-azidohomoalanine (L-AHA)-based click chemistry in human senescent fibroblasts. We optimized several aspects of the procedure, including senescence induction, a flow cytometry analysis of senescent cells, and the duration of L-AHA incorporation. This protocol uses senescent human fibroblasts but can be applied to other types of cells or circumstances.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.devcel.2021.04.008">Lee et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>An optimized protocol to measure the global translation rate in senescent cells</li>
<li>Use of L-AHA-based click chemistry enables a sensitive measurement of protein translation</li>
<li>Flow cytometry allows quantitative analysis of the global translation rate</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/997-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps for the flow cytometry-based assessment and quantification of the global translation rate in primary human fibroblasts (IMR90) maintained under normal growing conditions as well as senescent conditions induced by either ionizing radiation or treatment with the radiomimetic agent bleomycin (<a href="#bib2">Coppe et al., 2008</a>; <a href="#bib15">Kang et al., 2015</a>). Specifically, the protocol offers a means to (1) cause DNA damage-induced senescence in human fibroblasts; (2) quantify the rate of global protein synthesis at a single cell resolution; and (3) perform a flow cytometry analysis of senescent cells with a special focus on drawing flow cytometry gates. This protocol has been successfully applied to another type of human fibroblasts (BJ cells) with a slight modification (e.g., cell density, L-AHA labeling time); thus we expect that it can be applied to other types of cells, including HEK293T and mouse embryonic fibroblasts (<a href="#bib4">Goldsmith et al., 2020</a>; <a href="#bib29">Zhang et al., 2014</a>), or circumstances, including cell cycle regulation (<a href="#bib12">Imami and Yasuda, 2019</a>), as well.</p>
<h3 id="sec1.1">Preparation of cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 40 min</div>
<ol>
<li>Provide a sufficient supply of nitrogen gas to maintain physiological O<sub>2</sub> concentrations inside the incubator (3% O<sub>2</sub>, 5% CO<sub>2</sub>, 92% N<sub>2</sub>; a physiological O<sub>2</sub> incubator).</li>
<li>Prepare sufficient culture medium and 1<b>×</b> Dulbecco’s Phosphate-Buffered Saline (DPBS) for cell culture of IMR90 cells as described in Materials and equipment.</li>
<li>Prepare methionine-free medium that enables the efficient incorporation of L-AHA into newly synthesized proteins as described in Materials and equipment.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We established this protocol using the human fibroblast IMR90 cells. Therefore, the use of another cell line is possible but needs adjustments and further troubleshooting for culture conditions and senescence induction. <a href="#troubleshooting">Troubleshooting 1</a>
</div>
<ol start="4">
<li>All medium, trypsin, and buffers for cell culture should be warmed up at 37°C for 30 min in advance.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="undtbl1">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagles Medium (DMEM) (High glucose)</td>
<td>HyClone</td>
<td>Cat#SH30243</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>HyClone</td>
<td>Cat#SH30919</td>
</tr>
<tr>
<td>Penicillin Streptomycin 100<b>×</b> Solution</td>
<td>HyClone</td>
<td>Cat#SV30010</td>
</tr>
<tr>
<td>MEM Non-essential Amino Acid Solution (100<b>×</b>)</td>
<td>Welgene</td>
<td>Cat#LS005-01</td>
</tr>
<tr>
<td>Dulbecco’s Phosphate-Buffered Saline (DPBS)</td>
<td>Welgene</td>
<td>Cat#LB001-02</td>
</tr>
<tr>
<td>Phosphate-Buffered Saline (PBS) (10<b>×</b>)</td>
<td>Welgene</td>
<td>Cat#LB204-01</td>
</tr>
<tr>
<td>Trypsin 0.25% protease (1<b>×</b>) HBSS with 0.2 g/L EDTA; without calcium nor magnesium</td>
<td>HyClone</td>
<td>Cat#SH30042</td>
</tr>
<tr>
<td>Bleomycin</td>
<td>Cayman Chemicals</td>
<td>Cat#13877</td>
</tr>
<tr>
<td>DMEM (high glucose, no glutamine, no methionine, no cysteine)</td>
<td>Gibco</td>
<td>Cat#21013024</td>
</tr>
<tr>
<td>L-Cysteine</td>
<td>Sigma-Aldrich</td>
<td>Cat#C7755</td>
</tr>
<tr>
<td>Dialyzed FBS</td>
<td>Gibco</td>
<td>Cat#26400044</td>
</tr>
<tr>
<td>GlutaMAX Supplement (100<b>×</b>)</td>
<td>Gibco</td>
<td>Cat#35050-061</td>
</tr>
<tr>
<td>L-Azidohomoalanine (AHA)</td>
<td>Click Chemistry Tools</td>
<td>Cat#1066-25</td>
</tr>
<tr>
<td>Cycloheximide</td>
<td>Sigma-Aldrich</td>
<td>Cat#01810</td>
</tr>
<tr>
<td>Paraformaldehyde</td>
<td>Sigma-Aldrich</td>
<td>Cat#158127</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Samchun Pure Chemical</td>
<td>Cat#T0819</td>
</tr>
<tr>
<td>Bovostar Bovine Serum Albumin (BSA)</td>
<td>Bovogen Biologicals</td>
<td>Cat#BSAS0.1</td>
</tr>
<tr>
<td>AFDye 405 alkyne</td>
<td>Click Chemistry Tools</td>
<td>Cat#1309-1</td>
</tr>
<tr>
<td>Sodium L-ascorbate</td>
<td>Sigma-Aldrich</td>
<td>Cat#A7631</td>
</tr>
<tr>
<td>Copper(II) sulfate (CuSO<sub>4</sub>)</td>
<td>Sigma-Aldrich</td>
<td>Cat#C1297</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>IMR90 cells</td>
<td>ATCC</td>
<td>ATCC CCL-186</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Attune Nxt Software (v3.1.2)</td>
<td>Thermo Fisher Scientific</td>
<td><a href="https://www.thermofisher.com/">https://www.thermofisher.com/</a></td>
</tr>
<tr>
<td>FlowJo (v10.6.2)</td>
<td>FlowJo</td>
<td><a href="https://www.flowjo.com/">https://www.flowjo.com/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>1300 Series A2 Class II, Type A2 Bio Safety Cabinets</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#1305</td>
</tr>
<tr>
<td>Heracell™ 240i CO<sub>2</sub> Incubator (3% O<sub>2</sub> 5% CO<sub>2</sub>)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#51026331</td>
</tr>
<tr>
<td>Digital Water Bath</td>
<td>Daihan Labtech</td>
<td>Cat#LWB-122D</td>
</tr>
<tr>
<td>EVOS M5000 Imaging System</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AMF5000</td>
</tr>
<tr>
<td>Countess II FL Automated Cell Counter</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AMQAF1000</td>
</tr>
<tr>
<td>Cell culture plate for adherent cells (6-well)</td>
<td>Sarstedt</td>
<td>Cat#83.3920</td>
</tr>
<tr>
<td>Serological pipettes</td>
<td>Sarstedt</td>
<td>Cat#86.1254.001</td>
</tr>
<tr>
<td>5 mL Round Bottom Polystyrene Test Tube</td>
<td>Falcon</td>
<td>Cat#352052</td>
</tr>
<tr>
<td>Benchtop centrifuge</td>
<td>LaboGene</td>
<td>Cat#1248</td>
</tr>
<tr>
<td>Attune NxT Acoustic Focusing Cytometer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A24858</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec3.1">Solutions for IMR90 cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span>  &lt;<b>30 min</b>
</div>
<div class="critical">
<span class="critical-title">Critical:</span> All solutions employed for cell culture need to be prepared and maintained in sterile conditions by operating under a Class II Biological Safety Cabinet</div>
<p>Sterilize Class II Biological Safety Cabinet with 70% ethanol before and after using it. All items placed in a Class II Biological safety cabinet must be sterilized with 70% ethanol.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Ethanol is highly flammable in liquid and vapor, and can cause serious eye damage or irritation. Therefore, it should be manipulated with appropriate personal protective equipment (PPE) and must not be handled near open flames. It should be stored in a dedicated cabinet for flammables.</div>
<p>Prepare complete culture medium for IMR90 cells by adding the supplements specified in the table below to DMEM:</p>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>15%</td>
<td>90.2 mL</td>
</tr>
<tr>
<td>Penicillin Streptomycin Solution (100<b>×</b>)</td>
<td>100 U/mL (Penicillin), 100 μg/mL (Streptomycin)</td>
<td>6.02 mL</td>
</tr>
<tr>
<td>MEM Non-essential Amino Acid Solution (100<b>×</b>)</td>
<td>100 μM of each nonessential amino acid (L-Alanine, L-Asparagine, L-Aspartic acid, L-Glutamic acid, Glycine, L-Proline, L-Serine)</td>
<td>6.02 mL</td>
</tr>
<tr>
<td>DMEM (High glucose)</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>∼602.24 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The optimal medium composition can vary considerably for different cell types.</div>
<div class="note">
<span class="note-title">Note:</span> Complete culture medium can be stored at 4°C for 1–3 months.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Penicillin/Streptomycin may cause an allergic skin reaction, may cause allergy or asthma symptoms or breathing difficulties if inhaled, and may damage fertility or the unborn child, and hence it should be manipulated by wearing appropriate recommended PPE.</div>
<h3 id="sec3.2">Solutions for L-AHA incorporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span>  &lt;<b>30 min</b>
</div>
<p>Prepare methionine-free medium for L-AHA incorporation by adding the supplements specified in the table below to DMEM without L-methionine:</p>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dialyzed FBS</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>GlutaMAX Supplement (100<b>×</b>)</td>
<td>2 mM L-alanyl-L-glutamine dipeptide</td>
<td>500 μL</td>
</tr>
<tr>
<td>L-Cysteine stock (200 mM, 1000<b>×</b>)</td>
<td>200 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>DMEM (High glucose, no glutamine, no methionine, no cysteine)</td>
<td>n/a</td>
<td>44.45 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> It is recommended to make methionine-free culture medium fresh for each experiment and use it immediately. Dialyzed FBS can reduce nonspecific incorporation of methionine into the newly synthesized proteins, enhancing their optimal labeling with L-AHA.</div>
<div class="note">
<span class="note-title">Note:</span> For Cysteine and GlutaMAX, follow the concentration defined in the medium for your cell line. Directly add L-AHA to the medium just before starting the pulse labeling.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Aqueous solutions of cysteine oxidize readily in air to give cystine at neutral or basic pH. Dissolve the L-Cysteine to ddH<sub>2</sub>O to make 200 mM (1000<b>×</b>) stock solution and filter with 0.45 μm filter just before use. Add it immediately to methionine-free medium.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> normal serum can be used if dialyzed serum affects the growth of your cell line.</div>
<h3 id="sec3.3">Solutions for flow cytometry sample preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h (4% Paraformaldehyde stock solution)</div>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min (1× permeabilization buffer)</div>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min (blocking buffer)</div>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;30 min (click reaction solution)</div>
<div class="timing">
<span class="timing-title">Timing:</span> &lt;10 min (flow cytometry solution)</div>
<table id="undtbl4">
<caption>4% Paraformaldehyde stock solution</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Paraformaldehyde powder</td>
<td>n/a</td>
<td>4%</td>
<td>4 g</td>
</tr>
<tr>
<td>NaOH (1N)</td>
<td>10 N</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>HCl</td>
<td>10 N</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>PBS</td>
<td>10<b>×</b>
</td>
<td>1<b>×</b>
</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p>For a 4% paraformaldehyde solution, add 4 g of paraformaldehyde to 50 mL of ddH<sub>2</sub>O. The powder will not immediately dissolve into solution. Slowly raise the pH by adding 10 N NaOH dropwise from a glass pipette while gently stirring on a heating block at 60°C until the paraformaldehyde is dissolved. Add 10 mL of 10<b>×</b> PBS and allow the mixture to cool to room temperature (20°C–25°C). Adjust the pH to 7.4 by adding 10 N HCl dropwise. Adjust the final volume to 100 mL with ddH<sub>2</sub>O. Keep the solution at 4°C for up to 2 weeks.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Aliquots can be stored at −20°C, but repeated freeze/thaw cycles should be avoided.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Paraformaldehyde can severely irritate and burn the skin and eyes, may cause irritation of the nose, mouth, throat, and lungs if inhaled. It may also cause a skin allergy and asthma-like allergy. It should be manipulated by wearing appropriate recommended PPE.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Hydrogen Chloride (HCl) is a highly corrosive chemical and contact can severely irritate and burn the skin and eyes. Inhaling HCl may irritate the respiratory system including lungs and may cause pulmonary edema. It should be manipulated by wearing appropriate recommended PPE.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Sodium Hydroxide (NaOH) is a highly corrosive chemical and contact can severely irritate and burn the skin and eyes. Inhaling NaOH may irritate the respiratory system including lungs and may cause pulmonary edema. It should be manipulated by wearing appropriate recommended PPE.</div>
<table id="undtbl5">
<caption>1<b>×</b> permeabilization buffer</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton X-100</td>
<td>10% in PBS (20<b>×</b>)</td>
<td>0.5% (1<b>×</b>)</td>
<td>5 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>n/a</td>
<td>95 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare 20<b>×</b> Triton X-100 stock solutions by diluting 1 mL Triton X-100 in 9 mL PBS, and allowing complete dissolution on a rotator for 30 min at 20°C–25°C.</div>
<div class="note">
<span class="note-title">Note:</span> 20<b>×</b> Triton X-100 stock solutions are stable at 20°C–25°C for at least one month.</div>
<div class="critical">
<span class="critical-title">Critical:</span> 1<b>×</b> permeabilization buffer (0.5% Triton X-100) must be prepared fresh at each experiment.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Triton X-100 is harmful if swallowed, may cause skin irritation, and causes serious eye damage, and hence should be manipulated by wearing appropriate recommended PPE.</div>
<table id="undtbl6">
<caption>Blocking buffer</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA (Bovine serum albumin)</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Let the solution on a rotator at 20°C–25°C for 30 min for complete dissolution of BSA.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Permeabilization and blocking buffers must be prepared fresh at each experiment.</div>
<table id="undtbl7">
<caption>Click reaction solution</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>AFDye 405 alkyne</td>
<td>1 mM in DMSO</td>
<td>1 μM</td>
<td>3 μL</td>
</tr>
<tr>
<td>Sodium L-ascorbate</td>
<td>100 mM in PBS</td>
<td>10 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>CuSO<sub>4</sub>
</td>
<td>100 mM in ddH<sub>2</sub>O</td>
<td>2 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>10<b>×</b>
</td>
<td>1<b>×</b>
</td>
<td>877 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare the click reaction solution fresh on the day of experiment and use it immediately.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not use Tris-based or amine-containing buffers instead of PBS because they will interfere with the click reaction by chelating Cu (<a href="#bib24">Presolski et al., 2011</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Cooper ligands can be additionally added for maintaining the preferred oxidation state of Cu in solution, further optimizing a click reaction (<a href="#bib11">Hong et al., 2009</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Sodium L-ascorbate stock solution (100 mM in 1<b>×</b> PBS) should be prepared just before use.</div>
<div class="note">
<span class="note-title">Note:</span> AFDye 405 alkyne stock solution (1 mM in DMSO) aliquots can be stored at −20°C for up to 6 months but multiple freeze-thaw cycles should be avoided.</div>
<div class="note">
<span class="note-title">Note:</span> CuSO<sub>4</sub> stock solution (100 mM in ddH<sub>2</sub>O) can be prepared in advance. Store the CuSO<sub>4</sub> stock solution at RT for up to 6 months.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> AFDye 405 alkyne can be replaced with any other fluorescent alkynes that can achieve chemoselective ligation (click chemistry). For example, it can be substituted with Alexa Fluor 488 alkyne (Thermo Fisher Scientific, Cat# A10267).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Copper (II) Sulfate (CuSO<sub>4</sub>) can irritate and burn the skin and eyes. Inhaling CuSO<sub>4</sub> may cause a sore and a hole in the bone, dividing the inner nose. It can cause headaches, nausea, vomiting, diarrhea, abdominal pain, and a skin allergy. It should be manipulated by wearing appropriate recommended PPE.</div>
<table id="undtbl8">
<caption>Flow cytometry solution</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>1 %</td>
<td>1 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>99 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare the flow cytometry solution fresh on the day of experiment and use it immediately.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec4.1">Cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 days, 15 min each day</div>
<p>In the next steps, the cell culture steps necessary for the preparation of senescence induction are described in detail.</p>
<ol>
<li>Thaw and seed IMR90 cells in a 10 cm culture plate with 10 mL complete culture medium.</li>
<li>Incubate the cells in a physiological O<sub>2</sub> incubator for 1–3 days so that they reach approximately 90% confluence.</li>
<li>Discard the culture medium and briefly rinse the cells with 5–10 mL prewarmed 1<b>×</b> DPBS to remove all traces of serum that contains trypsin inhibitors.</li>
<li>Add 1 mL prewarmed trypsin-EDTA (0.25%) to the cells and hold at 37°C until cell layer is dispersed.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Generally, it takes 2–3 min to completely detach the cells by trypsin-EDTA (0.25%) but might take longer if one does not use fresh trypsin-EDTA.</div>
<ol start="5">
<li>Neutralize trypsin-EDTA by adding 9 mL prewarmed complete culture medium and gently pipette 10 to 15 times to resuspend the cells. Check cell number using a cell counter.</li>
<li>Seed the cells into 6-well culture plates at a density of 1.2 X 10<sup>5</sup> cells/well. Add complete culture medium to a final volume of 2 mL/well. Incubate the cells in a physiological O<sub>2</sub> incubator for 16 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not seed the cells at a density of more than 2 × 10<sup>5</sup> cells/well to avoid excessive cell confluence during senescence induction. Appropriate cell confluence is essential to achieve senescence-induced morphological changes (e.g., cellular hypertrophy).</div>
<h3 id="sec4.2">Senescence induction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Overall timing for senescence induction: 7–10 days</div>
<div class="timing">
<span class="timing-title">Timing:</span>  &lt;30 min (step 8)</div>
<div class="timing">
<span class="timing-title">Timing:</span> for changing culture medium &lt;10 min (step 10)</div>
<p>DNA double-strand breaks (DSBs) are among the most potent inducers of senescence (<a href="#bib6">Gorgoulis et al., 2019</a>; <a href="#bib8">Hernandez-Segura et al., 2018</a>; <a href="#bib23">McHugh and Gil, 2018</a>), generating a relatively homogenous group of senescent cells (<a href="#bib2">Coppe et al., 2008</a>; <a href="#bib15">Kang et al., 2015</a>; <a href="#bib22">Lee et al., 2021</a>). In the next steps, the generation of DNA damage-induced senescent IMR90 cells is described in detail.</p>
<ol start="7">
<li>Check cell confluence to be approximately 10% before applying DNA damage (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/997-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Confluency of IMR90 cells before and after senescence induction</div>
<p>(A) represents appropriate confluency (approximately 10%) to irradiate the cells for DNA damage-induced senescence.</p>
<p>(B) represents cells that are too confluent for DNA damage-induced senescence; a sub-cultivation after irradiation is required in this case.</p>
<p>(C) represents morphological changes including cellular hypertrophy upon senescence induction. The arrows indicate representative hypertrophic cells.</p>
<p>Scale bar, 300 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<ol start="8">
<li>Expose the cells to a single ionizing radiation dose of 12 Gy with the GC3000 Elan irradiator (MDS Nordion, Canada).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Systems other than the GC3000 Elan irradiator can be used for irradiation if they offer control on total dose. In this case, one should follow a designated protocol for each equipment to get a reproducible dose of irradiation. If radiological equipment is not available, DNA damage can be induced by treating the cells with bleomycin. Systems other than the GC3000 Elan irradiator can be used for irradiation if they offer control on total dose. In this case, one should follow a designated protocol for each equipment to get a reproducible dose of irradiation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> normal (non-senescence-induced) cells should be maintained as a control.</div>
<ol start="9">
<li>24 h after irradiation, replace culture medium with 2 mL fresh, prewarmed complete culture medium.</li>
</ol>
<ol start="10">
<li>Every 3–4 days, replace exhausted culture medium with 2 mL fresh, prewarmed complete culture medium until the cells become senescent.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Generally, it takes 7–10 days after being irradiated for IMR90 cells to become fully senescent, based on the expression of multiple senescence markers, including Senescence-Associated Beta-Galactosidase (SA-β-Gal) activity, cyclin-dependent kinase inhibitors (e.g., p21 and p16), and Senescence-Associated Secretory Phenotype (SASP) (<a href="#bib1">Campisi et al., 2019</a>; <a href="#bib6">Gorgoulis et al., 2019</a>; <a href="#bib8">Hernandez-Segura et al., 2018</a>; <a href="#bib23">McHugh and Gil, 2018</a>). The use of different cell lines may need adjustments and further assessment of markers for senescence to achieve full senescence as described in this protocol (<a href="#bib5">Gonzalez-Gualda et al., 2021</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> If the cells become confluent (&gt; 80%) at 2–4 days after irradiation, a sub-cultivation at the ratio of 1:2 is necessary for allowing the cells to display senescent phenotypes robustly, including senescent hypertrophy. When the cells already start to show senescent hypertrophy (generally, 4–5 days after irradiation), however, a sub-cultivation is not recommended as senescent cells are relatively sensitive to trypsinization. <a href="#troubleshooting">Troubleshooting 2</a>
</div>
<h3 id="sec4.3">L-AHA incorporation and click chemistry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6–8 h</div>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h for step 11<b>–17</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 1–1.5 h for step 18<b>–21</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 1–1.5 h for step 22<b>–24</b>
</div>
<p>L-AHA is a non-radioactive and non-toxic amino acid analog of methionine that contains an azido moiety, allowing its quantitative detection through the chemoselective ligation (“click chemistry”) between an azide and a fluorescent alkyne (<a href="#bib19">Kolb et al., 2001</a>). L-AHA can be incorporated into the newly synthesized proteins in mammalian cells; thus, it can be utilized for measuring the rate of global protein synthesis (<a href="#bib3">Dieterich et al., 2006</a>; <a href="#bib12">Imami and Yasuda, 2019</a>; <a href="#bib27">Tom Dieck et al., 2012</a>). In the following steps, we describe how to measure incorporation of L-AHA into the newly synthesized proteins in senescent cells compared to that in normal (non-senescence-induced) cells using three different treatment conditions: (a) L-AHA; (b) L-AHA + cycloheximide (CHX); and (c) ddH<sub>2</sub>O (“unstained control”).</p>
<ol start="11">
<li>Replace complete culture medium with 1 mL prewarmed methionine-free medium to deplete methionine in normal and senescent IMR90 cells and incubate for 30 min at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step could be omitted if nutritional stress needs to be considered; however, in this case, L-AHA labeling time may need to be increased accordingly. <a href="#troubleshooting">Troubleshooting 3</a>
</div>
<ol start="12">
<li>Prepare fresh L-AHA working solution by diluting the stock solution in prewarmed methionine-free medium (100 μM final concentration).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> L-AHA working solution needs to be prepared fresh for every experiment. We also recommend testing and determining the optimal concentration of L-AHA for your experimental conditions (e.g., cell line, treatment). Generally, 25–100 μM L-AHA is used for measuring the global protein synthesis rate in mammalian cells (<a href="#bib9">Herranz et al., 2015</a>; <a href="#bib22">Lee et al., 2021</a>; <a href="#bib29">Zhang et al., 2014</a>). <a href="#troubleshooting">Troubleshooting 3</a>
</div>
<ol start="13">
<li>Label cells with 1 mL 100 μM L-AHA for 30 min at 37°C. Treatment with CHX (100 μg/mL) that interferes with the translocation step in protein synthesis can be used as a negative control (<a href="#bib25">Schneider-Poetsch et al., 2010</a>). In addition, prepare cells that are not labeled with L-AHA (ddH<sub>2</sub>O treated, unstained control) as an additional negative control for assessing a non-specific signal from unconjugated AFDye 405 alkyne.</li>
<li>Discard the L-AHA culture medium and briefly rinse cells with 1–2 mL prewarmed 1<b>×</b> DPBS per well to remove all traces of serum.</li>
<li>Add 250 μL prewarmed trypsin-EDTA (0.25%) to cells and hold at 37°C for 2–3 min until cell layer is dispersed.</li>
<li>Add 750 μL prewarmed methionine-free medium to cells and gently pipette 10 to 15 times to resuspend cells.</li>
<li>Collect cell suspension in U-bottom FACS tubes and centrifuge at 300 <b>×</b> <i>g</i>, 4°C for 5 min. Discard supernatants.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not use higher <b>×</b> <i>g</i> for centrifugation to avoid cell damage and clumping. From this step, we recommend using a U-bottom FACS tube instead of an Eppendorf tube to reduce cell loss during the procedure of click chemistry as shown in <a href="#fig2">Figure 2</a>. <a href="#troubleshooting">Troubleshooting 4</a>
</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/997-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. A comparison of cell loss during the click chemistry in Eppendorf and U-bottom FACS tubes</div>
<p>Using U-bottom FACS tubes for the click chemistry reduces cell loss during the click chemistry compared to using Eppendorf tubes. Red circle and arrow indicate cell pellet collected by centrifugation.</p>
</figcaption>
</figure>
<ol start="18">
<li>Wash cells with cold 1<b>×</b> DPBS: resuspend cells in 1 mL cold 1<b>×</b> DBPS per tube; centrifuge at 300 <b>×</b> <i>g</i>, 4°C for 5 min; and discard supernatants.</li>
<li>Fix cells with paraformaldehyde: resuspend cells in 200 μL of 4% paraformaldehyde (w/v in DPBS) per tube; incubate cells for 15 min at room temperature (20°C–25°C); centrifuge at 800 <b>×</b> <i>g</i>, 4°C for 10 min; and gently discard supernatants by pipetting.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Optimal fixation conditions may vary with cell type.</div>
<div class="critical">
<span class="critical-title">Critical:</span> After fixation, the cell pellet tends to get loose easily. Thus, caution is needed from this step. For example, we recommend slightly tilting a U-bottom FACS tube and pipetting slowly, not aspirating, to remove supernatants during the washing step. <a href="#troubleshooting">Troubleshooting 4</a>
</div>
<ol start="20">
<li>Permeabilize cells with the permeabilization buffer: resuspend cells in 200 μL of permeabilization buffer (1<b>×</b> Triton X-100 [0.5%]) per tube; incubate cells for 15 min at room temperature (20°C–25°C); centrifuge at 800 <b>×</b> <i>g</i>, 4°C for 10 min; and gently discard supernatants by pipetting.</li>
<li>Prepare the L-AHA click reaction cocktail as described in <a href="#materials-and-equipment">materials and equipment</a>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The L-AHA click reaction cocktail must be prepared fresh and used immediately at each experiment. <a href="#troubleshooting">Troubleshooting 3</a>
</div>
<ol start="22">
<li>Add 200 μL L-AHA click reaction solution per tube. Resuspend the cell pellet and incubate for 30 min at room temperature (20°C–25°C). Make sure to protect the sample from light (e.g., wrap sample tubes with aluminum foil when not in use) during the following steps.</li>
<li>Wash cells twice with 1% BSA in 1<b>×</b> DPBS: resuspend cells in with 500 μL 1% BSA in 1<b>×</b> DPBS per tube; centrifuge at 800 <b>×</b> <i>g</i>, 4°C for 10 min; gently discard supernatants by pipetting; and repeat this washing step once more.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Washing cells with 1% BSA is for blocking non-specific staining and reducing cell aggregation.</div>
<ol start="24">
<li>Resuspend cells in 500 μL 1% FBS in 1<b>×</b> DBPS. Place all tubes on ice in the dark.</li>
</ol>
<h3 id="sec4.4">Flow cytometry and quantitative analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>In the next steps, flow cytometry analysis including drawing gates for senescent cells and measuring the median fluorescence of L-AHA incorporation is described in detail. Turn on the Attune NxT Acoustic Focusing Cytometer.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other cytometers are likely suitable for this application once they provide a proper filter/laser setting for detecting AFdye405 (e.g., laser 405, filter 440/50).</div>
<ol start="25">
<li>Using the control unstained samples, draw the gate by the forward (FSC-A) and side scatter profiling (SSC-A) that defines the main live cell population, excluding cellular debris and dead cells with the Attune Nxt Software (v3.1.2) (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/997-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Example of flow cytometry gating strategy for the live cell population in senescent cells</div>
<p>Graph displaying all events detected by flow cytometry (left). Gates drawn to isolate debris and live cell populations. Graph displaying the levels of L-AHA incorporation in a selected population (right). Debris 1 and 2 populations show no L-AHA incorporation; Debris 3 population includes both ‘unstained’ debris and ‘stained’ live cells. The majority of the live population is positive for L-AHA incorporation.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Senescent cells exhibit significant morphological changes, such as cellular hypertrophy and increased granularity (<a href="#bib6">Gorgoulis et al., 2019</a>; <a href="#bib8">Hernandez-Segura et al., 2018</a>; <a href="#bib23">McHugh and Gil, 2018</a>). Furthermore, senescent samples tend to have a higher level of cell debris than control samples. Thus, we recommend adjusting the gate for the live cell population that excludes cell debris in senescent samples as much as possible while not significantly depleting live cells in control samples. As cell debris is negative for L-AHA staining, one can exclude the population completely negative for L-AHA staining as cell debris as shown in <a href="#fig3">Figure 3</a>. <a href="#troubleshooting">Troubleshooting 5</a>
</div>
<ol start="26">
<li>For the live cell population gated in step 25, record L-AHA incorporation (AFdye405) for at least 10,000 events.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As senescent cells are larger than control cells in size, the number of senescent cells recorded for L-AHA incorporation by FACS is less than that of normal cells if the same volume of samples is analyzed. In general, the number of senescent cells is about 70–85% less than that of normal cells.</div>
<ol start="27">
<li>Quantify L-AHA incorporation by measuring the median value of fluorescence with FlowJo software. Check whether both unstained samples and negative controls (samples treated with CHX) display a minimal level of L-AHA incorporation (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/997-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Quantification of the L-AHA incorporation in normal and senescent cells</div>
<p>(A) L-AHA incorporation is measured by FACS. CHX denotes cycloheximide.</p>
<p>(B) Median fluorescence intensity (MFI) was computed; Senescent cells have a higher level of L-AHA incorporation, suggesting a higher level of global protein synthesis.</p>
</figcaption>
</figure>
<ol type="a">
<li>Open FlowJo software and drag the data such as fcs format into a workspace.</li>
<li>Double click the icon of any data in the list and the pop-up window showing FSC and SSC will appear.</li>
<li>Draw the gate that includes live cells but excludes cell debris and dead cells as shown in <a href="#fig3">Figure 3</a>. Name this population as “live”.</li>
<li>Apply this gating to all samples by dragging the “live” icon in the list to the icon of “all samples”.</li>
<li>Double click the “live” icon in any sample and change the horizontal axis into the channel recording L-AHA incorporation (AFdye405).</li>
<li>Apply this change to all samples by dragging the icon of the modified one in the list to the icon of “all samples”.</li>
<li>Click the icon of “Layout Editor” and drag the “live” icon in the list to create graphical reports.</li>
<li>Double click the graph with the Layout Editor to customize the look of the graphical reports.</li>
<li>To calculate the median value of L-AHA incorporation, right-click the “live” icon and click “Add statistics”. Select “Median” (or any desired statistics) for the channel recording L-AHA incorporation (AFdye405) and click “Add”.</li>
<li>Apply the median statistics to all samples by dragging the “Median” icon in the list to the “live” icon under all samples.</li>
<li>Export the file in excel format.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Successful L-AHA incorporation not only displays a labeling time-dependent increase in fluorescent signals from both normal and senescent samples, but also shows minimal fluorescent signals from negative controls as shown in <a href="#fig4">Figure 4</a>. Furthermore, senescent cells display a higher level of L-AHA incorporation than normal cells, suggesting their increased levels of protein translation as described previously (<a href="#bib9">Herranz et al., 2015</a>; <a href="#bib21">Laberge et al., 2015</a>; <a href="#bib22">Lee et al., 2021</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Cellular senescence can be induced by a wide range of stresses, acting as a basic aging mechanism (<a href="#bib6">Gorgoulis et al., 2019</a>; <a href="#bib10">Higuchi-Sanabria et al., 2018</a>; <a href="#bib26">Smith et al., 2020</a>). Elimination of senescent cells or modulation of its inflammatory secretory phenotype holds great promise for treating several age-related diseases (<a href="#bib13">Kang, 2019</a>; <a href="#bib18">Kirkland and Tchkonia, 2017</a>; <a href="#bib28">van Deursen, 2019</a>). However, senescent cells display significant heterogeneity in their phenotypes, depending on the nature of senescence triggers and cell types (<a href="#bib7">Hernandez-Segura et al., 2017</a>; <a href="#bib14">Kang and Elledge, 2016</a>; <a href="#bib16">Kim et al., 2021a</a>; <a href="#bib20">Kwon et al., 2017</a>); this greatly limits the therapeutic value of targeting senescence (<a href="#bib17">Kim et al., 2021b</a>). Similarly, it is possible that not all types of senescent cells increase the rate of global protein synthesis.</p>
<p>Our protocol relies on the concept that L-AHA competes with Methionine to be inserted into newly synthesized proteins; therefore, the levels of its incorporation can serve as a surrogate marker for protein synthesis rate at a given time (<a href="#bib3">Dieterich et al., 2006</a>). It is critical to deplete intracellular methionine at a certain threshold before L-AHA labeling; otherwise, the assay’s sensitivity will not reach its significance. If one considers not depleting intracellular methionine to reduce potential cellular stresses, the incubation time for L-AHA labeling needs to increase accordingly. Moreover, the protocol is not suitable for measuring the protein synthesis rate of an individual protein. AFDye 405 alkyne needs to be replaced with alkyne-biotin in the protocol followed by pulldown of biotin-conjugated proteins and detection of a specific target protein by western blot.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Cell lines other than IMR90 cells need to be used for senescence induction (<a href="#before-you-begin">Before you begin</a>).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Each cell line has a unique requirement for cell culture, including the composition of cell culture medium. For cell lines other than IMR90 cells, one can contact their provider [e.g., American Type Culture Collection (ATCC)] to acquire information about optimal medium composition and culture guides. In addition, each cell line has a different sensitivity to DNA damage; thus, the amount of DNA damage to induce senescence needs to be determined individually for cell lines of interest. Expression of multiple senescence markers, including SA-β-Gal activity, p21, p16, and SASP genes, needs to be analyzed in response to a single ionizing radiation dose of 2–20 Gy (<a href="#bib5">Gonzalez-Gualda et al., 2021</a>). For cell lines that preferentially induce apoptosis instead of senescence upon DNA damage, other senescence-inducing stimuli (e.g., replicative exhaustion, oncogene activation, oxidative stress) can be employed as an alternative (<a href="#bib5">Gonzalez-Gualda et al., 2021</a>; <a href="#bib8">Hernandez-Segura et al., 2018</a>; <a href="#bib23">McHugh and Gil, 2018</a>).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Cellular senescence is not properly induced by DNA damage (step 10).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Senescence induction can be affected by several factors, including cell confluency. If the cells become confluent at 2–3 days after irradiation or treatment with bleomycin, the cells need to be split back to 50% confluence as described in the steps 3–6. When the cells start to display senescent hypertrophy, however, a sub-cultivation is not recommended as senescent cells are relatively sensitive to trypsinization; after a sub-cultivation, the cells become thin and long, which fail to express several markers of senescence, including SA-β-Gal activity. Serum starvation is also reported to suppress senescence induction; thus, it is important to maintain proper levels of serum in culture medium during senescence induction.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>No signal is detected for L-AHA incorporation (steps 11, 12, and 21).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>There could be multiple reasons for this problem: first, the L-AHA working solution goes bad during storage. Make sure that L-AHA working solution is prepared fresh for every experiment and used immediately; second, the concentration of L-AHA is not optimal for your cell line. We recommend determining the optimal concentration of L-AHA in your experimental setting first. Generally, 25–100 μM L-AHA is used for a wide range of mammalian cells (<a href="#bib9">Herranz et al., 2015</a>; <a href="#bib22">Lee et al., 2021</a>; <a href="#bib29">Zhang et al., 2014</a>); third, intracellular methionine is not depleted enough, and thus the incorporation of L-AHA into newly synthesized proteins is not efficient. In this case, it may be necessary to either extend the incubation time of the cells with methionine-free medium or prolong the labeling time of the cells with L-AHA; and last, make sure that the cells are supplemented with dialyzed FBS during L-AHA incorporation. Otherwise, protein synthesis is significantly halted due to serum starvation.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>The cell number is not enough for FACS analysis (steps 17–19).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Sample loss may occur after fixation (steps 19–23). To reduce sample loss, we recommend using a U-bottom FACS tube instead of an Eppendorf tube during the procedure (<a href="#fig2">Figure 2</a>). In general, using a U-bottom FACS tube reduces sample loss up to 250% compared to an Eppendorf tube. Be cautious when removing supernatants at each step. Alternatively, one can simply scale up cell culture and senescence induction from 6 well plates to 6 or 10 cm dishes.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>The cells gated in FACS analysis are not pure and contaminated by cell debris or dead cells (step 25).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Senescent cells exhibit significant morphological changes, such as cellular hypertrophy and increased granularity. Furthermore, the sample from senescent cells has an excessive level of cell debris compared to normal samples. Therefore, if the gate for the live cell population is set based on normal cells, it will include a high level of cell debris from senescent samples. We recommend adjusting the gate for the live cell population that excludes cell debris in senescent samples at the expense of losing part of live cells from normal samples. As cell debris is negative for L-AHA staining, one can use this as a basis to exclude cell debris from senescent samples as described in <a href="#fig3">Figure 3</a>.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Campisi, J., Kapahi, P., Lithgow, G.J., Melov, S., Newman, J.C., and Verdin, E. (2019). From discoveries in ageing research to therapeutics for healthy ageing. Nature <i>571</i>, 183-192. <a class="external-link" href="https://doi.org/10.1038/s41586-019-1365-2">https://doi.org/10.1038/s41586-019-1365-2</a></p>
<p id="bib2">Coppe, J.P., Patil, C.K., Rodier, F., Sun, Y., Munoz, D.P., Goldstein, J., Nelson, P.S., Desprez, P.Y., and Campisi, J. (2008). Senescence-associated secretory phenotypes reveal cell-nonautonomous functions of oncogenic RAS and the p53 tumor suppressor. PLoS Biol. <i>6</i>, 2853-2868. <a class="external-link" href="https://doi.org/10.1371/journal.pbio.0060301">https://doi.org/10.1371/journal.pbio.0060301</a></p>
<p id="bib3">Dieterich, D.C., Link, A.J., Graumann, J., Tirrell, D.A., and Schuman, E.M. (2006). Selective identification of newly synthesized proteins in mammalian cells using bioorthogonal noncanonical amino acid tagging (BONCAT). Proc. Natl. Acad. Sci. U S A <i>103</i>, 9482-9487. <a class="external-link" href="https://doi.org/10.1073/pnas.0601637103">https://doi.org/10.1073/pnas.0601637103</a></p>
<p id="bib4">Goldsmith, J., Marsh, T., Asthana, S., Leidal, A.M., Suresh, D., Olshen, A., and Debnath, J. (2020). Ribosome profiling reveals a functional role for autophagy in mRNA translational control. Commun. Biol. <i>3</i>, 388. <a class="external-link" href="https://doi.org/10.1038/s42003-020-1090-2">https://doi.org/10.1038/s42003-020-1090-2</a></p>
<p id="bib5">Gonzalez-Gualda, E., Baker, A.G., Fruk, L., and Munoz-Espin, D. (2021). A guide to assessing cellular senescence in vitro and in vivo. FEBS J. <i>288</i>, 56-80. <a class="external-link" href="https://doi.org/10.1111/febs.15570">https://doi.org/10.1111/febs.15570</a></p>
<p id="bib6">Gorgoulis, V., Adams, P.D., Alimonti, A., Bennett, D.C., Bischof, O., Bishop, C., Campisi, J., Collado, M., Evangelou, K., Ferbeyre, G., et al. (2019). Cellular senescence: defining a path forward. Cell <i>179</i>, 813-827. <a class="external-link" href="https://doi.org/10.1016/j.cell.2019.10.005">https://doi.org/10.1016/j.cell.2019.10.005</a></p>
<p id="bib7">Hernandez-Segura, A., de Jong, T.V., Melov, S., Guryev, V., Campisi, J., and Demaria, M. (2017). Unmasking transcriptional heterogeneity in senescent cells. Curr. Biol. <i>27</i>, 2652-2660 e2654. <a class="external-link" href="https://doi.org/10.1016/j.cub.2017.07.033">https://doi.org/10.1016/j.cub.2017.07.033</a></p>
<p id="bib8">Hernandez-Segura, A., Nehme, J., and Demaria, M. (2018). Hallmarks of cellular senescence. Trends Cell Biol. <i>28</i>, 436-453. <a class="external-link" href="https://doi.org/10.1016/j.tcb.2018.02.001">https://doi.org/10.1016/j.tcb.2018.02.001</a></p>
<p id="bib9">Herranz, N., Gallage, S., Mellone, M., Wuestefeld, T., Klotz, S., Hanley, C.J., Raguz, S., Acosta, J.C., Innes, A.J., Banito, A., et al. (2015). mTOR regulates MAPKAPK2 translation to control the senescence-associated secretory phenotype. Nat. Cell Biol. <i>17</i>, 1205. <a class="external-link" href="https://doi.org/10.1038/ncb3225">https://doi.org/10.1038/ncb3225</a></p>
<p id="bib10">Higuchi-Sanabria, R., Frankino, P.A., Paul, J.W., Tronnes, S.U., and Dillin, A. (2018). A futile battle? protein quality control and the stress of aging. Dev. Cell <i>44</i>, 139-163. <a class="external-link" href="https://doi.org/10.1016/j.devcel.2017.12.020">https://doi.org/10.1016/j.devcel.2017.12.020</a></p>
<p id="bib11">Hong, V., Presolski, S.I., Ma, C., and Finn, M.G. (2009). Analysis and optimization of copper-catalyzed azide-alkyne cycloaddition for bioconjugation. Angew. Chem. Int. Ed. Engl. <i>48</i>, 9879-9883. <a class="external-link" href="https://doi.org/10.1002/anie.200905087">https://doi.org/10.1002/anie.200905087</a></p>
<p id="bib12">Imami, K. and Yasuda, T. (2019). Measuring protein synthesis during cell cycle by azidohomoalanine (AHA) Labeling and flow cytometric analysis. Bio Protoc. <i>9</i>, e3215. <a class="external-link" href="https://doi.org/10.21769/BioProtoc.3215">https://doi.org/10.21769/BioProtoc.3215</a></p>
<p id="bib13">Kang, C. (2019). Senolytics and senostatics: a two-pronged approach to target cellular senescence for delaying aging and age-related diseases. Mol. Cells <i>42</i>, 821-827. <a class="external-link" href="https://doi.org/10.14348/molcells.2019.0298">https://doi.org/10.14348/molcells.2019.0298</a></p>
<p id="bib14">Kang, C. and Elledge, S.J. (2016). How autophagy both activates and inhibits cellular senescence. Autophagy <i>12</i>, 898-899. <a class="external-link" href="https://doi.org/10.1080/15548627.2015.1121361">https://doi.org/10.1080/15548627.2015.1121361</a></p>
<p id="bib15">Kang, C., Xu, Q.K., Martin, T.D., Li, M.Z., Demaria, M., Aron, L., Lu, T., Yankner, B.A., Campisi, J., and Elledge, S.J. (2015). The DNA damage response induces inflammation and senescence by inhibiting autophagy of GATA4. Science <i>349</i>. <a class="external-link" href="https://doi.org/10.1126/science.aaa5612">https://doi.org/10.1126/science.aaa5612</a></p>
<p id="bib16">Kim, J., Lee, Y., Jeon, T., Kim, M.S., and Kang, C. (2021a). All cells are created equal in the sight of autophagy: selective autophagy maintains homeostasis in senescent cells. Autophagy. <a class="external-link" href="https://doi.org/10.1080/15548627.2021.1953848">https://doi.org/10.1080/15548627.2021.1953848</a></p>
<p id="bib17">Kim, J., Lee, Y., Roh, K., Kim, M.S., and Kang, C. (2021b). Targeting the stress support network regulated by autophagy and senescence for cancer treatment. Adv. Cancer Res. <i>150</i>, 75-112. <a class="external-link" href="https://doi.org/10.1016/bs.acr.2021.01.003">https://doi.org/10.1016/bs.acr.2021.01.003</a></p>
<p id="bib18">Kirkland, J.L. and Tchkonia, T. (2017). Cellular senescence: a translational perspective. EBioMedicine <i>21</i>, 21-28. <a class="external-link" href="https://doi.org/10.1016/j.ebiom.2017.04.013">https://doi.org/10.1016/j.ebiom.2017.04.013</a></p>
<p id="bib19">Kolb, H.C., Finn, M.G., and Sharpless, K.B. (2001). click chemistry: diverse chemical function from a few good reactions. Angew. Chem. Int. Ed. Engl. <i>40</i>, 2004-2021. <a class="external-link" href="https://doi.org/10.1002/1521-3773(20010601)40:11&lt;2004::aid-anie2004&gt;3.3.co;2-x">https://doi.org/10.1002/1521-3773(20010601)40:11&lt;2004::aid-anie2004&gt;3.3.co;2-x</a></p>
<p id="bib20">Kwon, Y., Kim, J.W., Jeoung, J.A., Kim, M.S., and Kang, C. (2017). Autophagy is pro-senescence when seen in close-up, but anti-senescence in long-shot. Mol. Cells <i>40</i>, 607-612. <a class="external-link" href="https://doi.org/10.14348/molcells.2017.0151">https://doi.org/10.14348/molcells.2017.0151</a></p>
<p id="bib21">Laberge, R.M., Sun, Y., Orjalo, A.V., Patil, C.K., Freund, A., Zhou, L.L., Curran, S.C., Davalos, A.R., Wilson-Edell, K.A., Liu, S., et al. (2015). MTOR regulates the pro-tumorigenic senescence-associated secretory phenotype by promoting IL1A translation. Nat. Cell Biol. <i>17</i>, 1049-U1416. <a class="external-link" href="https://doi.org/10.1038/ncb3195">https://doi.org/10.1038/ncb3195</a></p>
<p id="bib22">Lee, Y., Kim, J., Kim, M.S., Kwon, Y., Shin, S., Yi, H., Kim, H., Chang, M.J., Chang, C.B., Kang, S.B., et al. (2021). Coordinate regulation of the senescent state by selective autophagy. Dev. Cell <i>56</i>, 1512-1525 e1517. <a class="external-link" href="https://doi.org/10.1016/j.devcel.2021.04.008">https://doi.org/10.1016/j.devcel.2021.04.008</a></p>
<p id="bib23">McHugh, D. and Gil, J. (2018). Senescence and aging: Causes, consequences, and therapeutic avenues. J. Cell Biol. <i>217</i>, 65-77. <a class="external-link" href="https://doi.org/10.1083/jcb.201708092">https://doi.org/10.1083/jcb.201708092</a></p>
<p id="bib24">Presolski, S.I., Hong, V.P., and Finn, M.G. (2011). Copper-catalyzed azide-alkyne click chemistry for bioconjugation. Curr. Protoc. Chem. Biol. <i>3</i>, 153-162. <a class="external-link" href="https://doi.org/10.1002/9780470559277.ch110148">https://doi.org/10.1002/9780470559277.ch110148</a></p>
<p id="bib25">Schneider-Poetsch, T., Ju, J., Eyler, D.E., Dang, Y., Bhat, S., Merrick, W.C., Green, R., Shen, B., and Liu, J.O. (2010). Inhibition of eukaryotic translation elongation by cycloheximide and lactimidomycin. Nat. Chem. Biol. <i>6</i>, 209-217. <a class="external-link" href="https://doi.org/10.1038/nchembio.304">https://doi.org/10.1038/nchembio.304</a></p>
<p id="bib26">Smith, H.J., Sharma, A., and Mair, W.B. (2020). Metabolic communication and healthy aging: where should we focus our Energy?. Dev. Cell <i>54</i>, 196-211. <a class="external-link" href="https://doi.org/10.1016/j.devcel.2020.06.011">https://doi.org/10.1016/j.devcel.2020.06.011</a></p>
<p id="bib27">Tom Dieck, S., Muller, A., Nehring, A., Hinz, F.I., Bartnik, I., Schuman, E.M., and Dieterich, D.C. (2012). Metabolic labeling with noncanonical amino acids and visualization by chemoselective fluorescent tagging. Curr. Protoc. Cell Biol. <i>7</i>, Unit7 11. <a class="external-link" href="https://doi.org/10.1002/0471143030.cb0711s56">https://doi.org/10.1002/0471143030.cb0711s56</a></p>
<p id="bib28">van Deursen, J.M. (2019). Senolytic therapies for healthy longevity. Science <i>364</i>, 636-637. <a class="external-link" href="https://doi.org/10.1126/science.aaw1299">https://doi.org/10.1126/science.aaw1299</a></p>
<p id="bib29">Zhang, J., Wang, J., Ng, S., Lin, Q., and Shen, H.M. (2014). Development of a novel method for quantification of autophagic protein degradation by AHA labeling. Autophagy <i>10</i>, 901-912. <a class="external-link" href="https://doi.org/10.4161/auto.28267">https://doi.org/10.4161/auto.28267</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Chanhee Kang (<a href="mailto:chanhee.kang@snu.ac.kr">chanhee.kang@snu.ac.kr</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate any new reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or codes.</p>
<h3>Acknowledgments</h3>
<p>We are grateful to all of the members of the Kang Lab for their support. This work was supported by the Suh Kyungbae Foundation (SUHF-17020068) and the <a href="https://doi.org/10.13039/501100003725">National Research Foundation of Korea</a> (NRF-2019R1C1C1006386, NRF-2020R1A5A1018081, and 2020R1I1A1A01072779).</p>
<h3>Author contributions</h3>
<p>Y.L. optimized the protocol and performed all experiments with input from J.K., T.J., K.R., M.-S.K., and C.K. C.K. supervised the project. Y.L. and C.K. wrote the manuscript. All authors discussed the results and commented on the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>