<article>
<section>
<h1 id="header">Purification of full-length recombinant human huntingtin proteins with allelic series of polyglutamine lengths</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Hyeongju Kim,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Kyung-gi Hyun,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Alejandro Lloret,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Ihn Sik Seong,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> and Ji-Joon Song<sup><a href="#aff1">1</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biological Sciences, KI for the BioCentury, Korea Advanced Institute of Science and Technology (KAIST), Daejeon 34141, South Korea</p>
<p id="aff2"><sup>2</sup>Molecular Neurogenetics Unit, Center for Genomic Medicine, Massachusetts General Hospital, Boston, MA 02114, USA</p>
<p id="aff3"><sup>3</sup>Department of Neurology, Harvard Medical School, Boston, MA 02114, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:songj@kaist.ac.kr">songj@kaist.ac.kr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100886">10.1016/j.xpro.2021.100886</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Huntington's disease (HD) is an autosomal dominant neurodegenerative disorder caused by the polyglutamine (polyQ) expansion in huntingtin (HTT) protein. The challenge of obtaining full-length HTT proteins with high purity limits the understanding of the HTT protein function. Here, we provide a protocol to generate and purify full-length recombinant human HTT proteins with various polyQ lengths, which is key to investigate the biochemical function of HTT proteins and the molecular mechanism underlying HD pathology.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.str.2020.06.008">Jung et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Expression and purification of full-length recombinant human huntingtin (HTT) proteins</li>
<li>Generation of HTT proteins with allelic series of polyglutamine length</li>
<li>Purified HTT proteins form complexes with HAP40 or HAP1.</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1103-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Prepare plasmids for generating baculovirus encoding full-length human huntingtin cDNAs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>To express HTT proteins in <i>Spodoptera frugiperda</i> (Sf9) insect cells using baculovirus, we modified pFastBac1 vector (Thermo Fisher Scientific) before cloning the Huntingtin (<i>H</i>TT) genes into the vector (<a href="#fig1">Figure 1</a>A) (<a href="#bib10">Seong et al., 2010</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1103-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The cloning strategy of full-length <i>H</i><i>TT</i> cDNA in a modified pFastBac1 vector</div>
<p>(A) A new linker cassette was inserted in pFastBac1 (Thermo Fisher Scientific, 4775 bp) vector. The linker cassette contains a FLAG tag, a 6<b>×</b>His-tag, a TEV protease recognition site and restriction enzyme sites (NcoI, XhoI, SacII and others, underlined and shown in italic).</p>
<p>(B) <i>Htt</i> cDNA fragment containing huntingtin amino acid 1–171 was inserted into the modified pFastBac1 vector with NcoI and XhoI restriction sites and then the rest of <i>Htt</i> cDNA fragment containing huntingtin amino acid 172-3,144 was cloned into the vector with XhoI and SacII restriction sites to express HTT tagged with FLAG, HIS and TEV sites. Each <i>Htt</i> DNA fragments and vector size are not up to scale.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> The <i>H</i><i>TT</i> plasmids with allelic series of CAG (polyQ) lengths are available upon a written request to the corresponding author (J.S.).</div>
<ol>
<li>We inserted a new linker cassette containing a FLAG tag, a 6<b>×</b>His tag, a TEV protease recognition site and new restriction enzyme sites (see the <a href="#fig1">Figure 1</a>A) into the pFastBac1 vector between the BamHI (GGATCC) and KpnI (GGTACC) restriction sites. Before inserting the cassette, the KpnI site in the vector was removed by incubation with a Klenow enzyme and the new linker cassette was ligated with BamHI and the blunt end KpnI site, which eliminates the KpnI site.</li>
<li>
<i>H</i><i>TT</i> cDNA fragments encoding amino acids 1–171 (based on polyQ23 human HTT) containing the CAG triplet repeat region were cloned between the NcoI (CCATGG) and XhoI (CTCGAG) restriction sites.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We can generate <i>H</i><i>TT</i> clones with various CAG repeats with this step.</div>
<div class="optional">
<span class="optional-title">Optional:</span> We usually add the final 1<b>×</b> Band Doctor (SolGent) to PCR mixtures, which improves PCR when <i>Htt</i> is used as a template.</div>
<ol start="3">
<li>A 9,046 bp fragment <i>H</i><i>TT</i> cDNA (GenBank accession number L12392) fragments encoding amino acids 172-3,144 was cloned with the newly inserted XhoI (CTCGAG) and SacII (CCGCGG) restriction sites located in the inserted cassette.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The original SacII restriction site in the pFastBac1 vector was deleted. In addition, the KpnI site was removed from the plasmid to utilize the KpnI site (1,761 bp, 587 a.a.) located at the loop region in N-HEAT, which has many phosphorylation sites, for the future usage.</div>
<p>An overall scheme of the plasmid containing <i>Htt</i> and the cloning steps are described in <a href="#fig1">Figure 1</a>B.</p>
<h3 id="sec1.2">Amplify the <i>H</i><i>TT</i> plasmids while stably maintaining the number of CAG repeat</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>When <i>H</i><i>TT</i> plasmids are amplified, it is difficult to maintain a constant number of CAG repeats in the clones as the CAG repeats are lost during plasmid amplification. In addition, the large size of the <i>H</i><i>TT</i> plasmid causes a low transformation efficiency and slow growth rate. To resolve these issues, we optimized the plasmid amplification protocol.</p>
<ol start="4">
<li>Remove 50 μL of XL10 Gold ultracompetent cells (Agilent) from the −80°C freezer and place them immediately on ice. Allow the cells to thaw for 5 min.</li>
<li>Add 100 ng of the <i>Htt</i> plasmid to be transformed and label the tubes. After incubation on ice for 30 min, incubate the tubes for 1 min at 42°C for heat shock.</li>
<li>Incubate for 10 min on ice after the heat shock and then add 900 μL of S.O.C. medium.</li>
<li>Incubate the tubes at 37°C for 45 min at a shaking incubator.</li>
<li>Centrifuge the cells at 4,000<b>×</b><i>g</i> for 10 min and decant 700 μL of the supernatant.</li>
<li>Resuspend the rest of the supernatant by slowly pipetting.</li>
<li>Plate the resuspended cells on an LB plate with 25 μg/mL of ampicillin.</li>
<li>Incubate the plate at 30°C for 2 days.</li>
<li>After 2 days, pick a single colony and proceed to a liquid culture with 25 μg/mL of ampicillin at 30°C for 2 days for mini-, midi-, or maxiprep (Qiagen).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To maintain the constant number of CAG triplet repeats, transformation and culture condition should be performed with LB media supplemented with 25 μg/mL of ampicillin and at 30°C for two days. The CAG lengths should be monitored by sequencing with the hHTT R146 reverse primer (see the <a href="#key-resources-table">key resources table</a>) whenever the plasmids are amplified.</div>
<h3 id="sec1.3">The critical points of this protocol</h3>
<p>Before starting the experiment, there are several critical points to follow this protocol.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The number of CAG tracks should be monitored by sequencing with the hHTT R146 reverse primer whenever the plasmids are amplified.</div>
<div class="critical">
<span class="critical-title">Critical:</span> For PCR amplification of the region containing CAG repeats, add the 5<b>×</b>Band Doctor™ to PCR reaction mixtures. If the 5<b>×</b>Band Doctor™ is not available, add a final 1–10% (v/v) DMSO as an alternative.</div>
<div class="critical">
<span class="critical-title">Critical:</span> For the optimal yield of purified HTT proteins, Sf9 cells should be kept healthy with a low level of dead cells. Healthy cells should double in their number in 24 h. Depending on the cell condition, the yield of HTT proteins can differ by up to 10-fold. When unhealthy cells are used, the purity and yield are greatly compromised.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The temperature and salt conditions of the buffer affect the behavior of HTT proteins. Exposure of HTT proteins to high salt (over 150 mM) for more than several hours was avoided.</div>
<div class="critical">
<span class="critical-title">Critical:</span> HTT purification should be completed within 2 days after cell lysis. Otherwise, HTT proteins degrade.</div>
<div class="critical">
<span class="critical-title">Critical:</span> HTT proteins can be stored at −80°C for several months to a year. However, when the proteins are thawed, the sample should be cleared by centrifugation at 17,000<b>×</b><i>g</i> for 20 min at 4°C to remove any precipitant formed during storage. Otherwise, all the HTT proteins start to be precipitated.</div>
<div class="note">
<span class="note-title">Note:</span> This protocol without any modification can be applied to all allelic series of HTT proteins regardless of the polyQ lengths.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Htt plasmids with allelic series of CAG (polyQ) lengths (Q2, Q23, Q46, Q67 and Q78)</td>
<td>The plasmids are available upon a written request to the corresponding author (J.S.).</td>
<td>N/A</td>
</tr>
<tr>
<td>pFastBac1 vector</td>
<td>Invitrogen</td>
<td>V011648</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>DNA Polymerase I, Klenow Fragment</td>
<td>NEB</td>
<td>M0210L</td>
</tr>
<tr>
<td>BamHI restriction enzyme</td>
<td>Enzynomics</td>
<td>CR003M</td>
</tr>
<tr>
<td>KpnI restriction enzyme</td>
<td>Enzynomics</td>
<td>R014M</td>
</tr>
<tr>
<td>XhoI restriction enzyme</td>
<td>Enzynomics</td>
<td>R007M</td>
</tr>
<tr>
<td>SacII restriction enzyme</td>
<td>Enzynomics</td>
<td>R036M</td>
</tr>
<tr>
<td>NcoI restriction enzyme</td>
<td>Enzynomics</td>
<td>R004M</td>
</tr>
<tr>
<td>5<b>×</b>Band Doctor™</td>
<td>SolGent</td>
<td>SBD41-B10k</td>
</tr>
<tr>
<td>2<b>×</b>TOPsimple™ PreMIX-Forte</td>
<td>Enzynomics</td>
<td>P610F</td>
</tr>
<tr>
<td>2<b>×</b>TOPsimple™ PreMIX-Tenuto</td>
<td>Enzynomics</td>
<td>P610</td>
</tr>
<tr>
<td>Plasmid DNA Purification Kit Mini</td>
<td>LaboPass</td>
<td>CMP0112</td>
</tr>
<tr>
<td>HiSPeed Plasmid Midi Kit</td>
<td>QIAGEN</td>
<td>Cat#12643</td>
</tr>
<tr>
<td colspan="3">Bacterial strains</td>
</tr>
<tr>
<td>
<i>Escherichia Coli</i> DH5α</td>
<td>Enzynomics</td>
<td>CPO11</td>
</tr>
<tr>
<td>DH10Bac</td>
<td>Thermo Fisher Scientific</td>
<td>LS10361012</td>
</tr>
<tr>
<td>XL10-Gold Ultracompetent Cells</td>
<td>Agilent</td>
<td>Cat#200314</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Sf9 cells in Sf-900<sup>TM</sup> II SFM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11496015</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>M13 forward primer</td>
<td>This paper</td>
<td>GTTTCCCAGTCACGAC</td>
</tr>
<tr>
<td>hHTT R146 reverse primer</td>
<td>This paper</td>
<td>CCTGACATCTGACTCTGCGTCATCAC</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Tryptone</td>
<td>LPS Solution</td>
<td>CHS-80</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>LPS Solution</td>
<td>CHS-02</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>LPS Solution</td>
<td>NACL05</td>
</tr>
<tr>
<td>Agar</td>
<td>LPS Solution</td>
<td>AGAR500</td>
</tr>
<tr>
<td>Tris</td>
<td>AMRESCO</td>
<td>0497-5kg</td>
</tr>
<tr>
<td>Glycerol</td>
<td>LPS Solution</td>
<td>GLYC01</td>
</tr>
<tr>
<td>Imidazole</td>
<td>Bio Basic</td>
<td>IB0277</td>
</tr>
<tr>
<td>HEPES</td>
<td>Sigma-Aldrich</td>
<td>H3375-550G</td>
</tr>
<tr>
<td>Hydrochloric acid (HCl)</td>
<td>Samchun</td>
<td>H0255</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Glentham</td>
<td>GA7355-100g</td>
</tr>
<tr>
<td>Kanamycin sulfate</td>
<td>Bio Basic</td>
<td>KB0286</td>
</tr>
<tr>
<td>Gentamicin</td>
<td>Bio Basic</td>
<td>Cat#105-41-0</td>
</tr>
<tr>
<td>Tetracycline</td>
<td>Sigma-Aldrich</td>
<td>87128-25G</td>
</tr>
<tr>
<td>Bluo-Gal</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15519-028</td>
</tr>
<tr>
<td>IPTG</td>
<td>UBPBio</td>
<td>P1010-100</td>
</tr>
<tr>
<td>Phenylmethylsulfonyl fluoride (PMSF)</td>
<td>Bio Basic</td>
<td>PB0425</td>
</tr>
<tr>
<td>DL-Dithiothreitol (DTT)</td>
<td>Soltec Ventures</td>
<td>M112_100g</td>
</tr>
<tr>
<td>Ethylenediaminetetraacetic acid (EDTA)</td>
<td>Sigma-Aldrich</td>
<td>E9884-1KG</td>
</tr>
<tr>
<td>cOmplete™, Mini, EDTA-free Protease inhibitor Cocktail (PIC)</td>
<td>Roche</td>
<td>Cat#4693159001</td>
</tr>
<tr>
<td>Cellfectin<sup>TM</sup> II Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10362100</td>
</tr>
<tr>
<td>HyQ CCM®3</td>
<td>Cytiva</td>
<td>SH30061</td>
</tr>
<tr>
<td>S.O.C. medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15544-034</td>
</tr>
<tr>
<td>2-Mercaptoethanol (BME)</td>
<td>Sigma-Aldrich</td>
<td>M6250-250ML</td>
</tr>
<tr>
<td>Anti-FLAG M2 affinity resin</td>
<td>Sigma-Aldrich</td>
<td>A2220-25ML</td>
</tr>
<tr>
<td>FLAG-peptide</td>
<td>Apeptide</td>
<td>APG1474</td>
</tr>
<tr>
<td>Nickel-nitrilotriacetic acid resin (Ni-NTA)</td>
<td>QIAGEN</td>
<td>Cat#30230</td>
</tr>
<tr>
<td>Glutathione Sepharose<sup>TM</sup> 4B</td>
<td>Cytiva</td>
<td>#17075605</td>
</tr>
<tr>
<td>Recombinant TEV protease</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Pyrex® 2,800 mL Fernbach culture flask</td>
<td>Pyrex</td>
<td>4425-2XL</td>
</tr>
<tr>
<td>MF-Millipore Membrane Filter, 0.22 μm pore size</td>
<td>Millipore</td>
<td>GSWP04700</td>
</tr>
<tr>
<td>6-Well plate</td>
<td>SPL Solutions</td>
<td>30066</td>
</tr>
<tr>
<td>75 cm<sup>2</sup> Culture flask</td>
<td>SPL Solutions</td>
<td>Cat#70075</td>
</tr>
<tr>
<td>150 cm<sup>2</sup> Cell culture dish</td>
<td>SPL Solutions</td>
<td>Cat#20151</td>
</tr>
<tr>
<td>Minisart® Syringe Filter</td>
<td>Sartorius</td>
<td>S6534-FMOSK</td>
</tr>
<tr>
<td>Steriflip-GP Sterile Centrifuge Tube Top Filter Unit</td>
<td>Millipore</td>
<td>SCGP00525</td>
</tr>
<tr>
<td>Corning® 500 mL Bottle Top Vacuum Filter, 0.22 μm Pore</td>
<td>Corning</td>
<td>431118</td>
</tr>
<tr>
<td>Superdex 200 HiLoad 26/60 Column</td>
<td>GE Healthcare</td>
<td>Cat#28–9893-36</td>
</tr>
<tr>
<td>Spectra/Por® 1 Dialysis Membrane</td>
<td>Spectrum Labs</td>
<td>Cat#132660</td>
</tr>
<tr>
<td>Amicon Ultra 100-KDa filter</td>
<td>Millipore</td>
<td>ACS510024</td>
</tr>
<tr>
<td>INFORS HT shaking incubator</td>
<td>INFORS HT</td>
<td>Multitron</td>
</tr>
<tr>
<td>Econo-Column® Chromatography Columns, 2.5 <b>×</b> 10 cm</td>
<td>Bio-Rad</td>
<td>#7372512</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Preparation of DH10Bac LB agar plates and medium</h3>
<ul>
<li>DH10Bac LB agar plates (1% Tryptone, 0.5% Yeast Extract, 1.0% NaCl, and 15 g/L agar) contain 50 μg/mL kanamycin, 7 μg/mL gentamycin, 10 μg/mL tetracycline, 100 μg/mL Bluo-Gal and 40 μg/mL IPTG.</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tryptone</td>
<td>1% (w/v)</td>
<td>5 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>0.5% (w/v)</td>
<td>2.5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>1% (w/v)</td>
<td>5 g</td>
</tr>
<tr>
<td>Agar</td>
<td>1.5% (w/v)</td>
<td>7.5 g</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>Up to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>500 mL</b></td>
</tr>
<tr>
<td>Kanamycin (50 mg/mL)</td>
<td>50 μg/mL</td>
<td>500 μL</td>
</tr>
<tr>
<td>Gentamycin (50 mg/mL)</td>
<td>7 μg/mL</td>
<td>70 μL</td>
</tr>
<tr>
<td>Tetracycline (5 mg/mL)</td>
<td>10 μg/mL</td>
<td>1 mL</td>
</tr>
<tr>
<td>Bluo-Gal (200 mg/mL)</td>
<td>100 μg/mL</td>
<td>250 μL</td>
</tr>
<tr>
<td>IPTG (119 mg/mL)</td>
<td>40 μg/mL</td>
<td>168 μL</td>
</tr>
</tbody>
</table>
<ul>
<li>DH10Bac LB medium contains 50 μg/mL kanamycin, 7 μg/mL gentamycin and 10 μg/mL tetracycline.</li>
</ul>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tryptone</td>
<td>1% (w/v)</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>0.5% (w/v)</td>
<td>0.25 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>1% (w/v)</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Agar</td>
<td>1.5% (w/v)</td>
<td>0.75 g</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
<tr>
<td>Kanamycin (50 mg/mL)</td>
<td>50 μg/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>Gentamycin (50 mg/mL)</td>
<td>7 μg/mL</td>
<td>7 μL</td>
</tr>
<tr>
<td>Tetracycline (5 mg/mL)</td>
<td>10 μg/mL</td>
<td>100 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The DH10Bac LB agar plates containing Bluo-Gal should be stored at 4°C in the dark place for less than one month. DH10Bac LB medium is freshly made from autoclaved LB medium by adding the antibiotics.</div>
<h3 id="sec2.2">Maintenance of insect cells</h3>
<ul>
<li>The optimal growth conditions of Sf9 cells with HyQ-CCM®3 medium (Cytiva) are 27°C and 100 rpm in an orbital shaker incubator.</li>
<li>We maintain the cell density of Sf9 between 1<b>×</b>10<sup>6</sup> to 8<b>×</b>10<sup>6</sup> cells/mL and split the cells at every 3 or 4 days. Once Sf9 cells are thawed from frozen cell stocks, the cells can be maintained over 6 months.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> The density of cells should be maintained above 0.5<b>×</b>10<sup>6</sup> cells/mL.</div>
<div class="note">
<span class="note-title">Note:</span> Healthy Sf9 cells should double every 24 h.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend using a PYREX® 2,800 mL Fernbach culture flask with a triple baffle and a membrane filtered cap for cell culture. The recommended cell culture volume in a Fernbach culture flask is 500 - 1,000 mL.</div>
<h3 id="sec2.3">Prepare buffers</h3>
<div class="note">
<span class="note-title">Note:</span> All the buffers should be kept at 4°C during the purification.</div>
<h4 id="sec2.3.1">TE buffer</h4>
<p>TE buffer contains 1 mM EDTA (pH8.0) and 10 mM Tris-HCl (pH8.0).</p>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.5 M EDTA (pH8.0)</td>
<td>1 mM</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl (pH8.0)</td>
<td>10 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>49.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> TE buffer must be filtered through a 0.22 μm MCE membrane filter (Millipore™) as this buffer will be treated directly with Sf9 cells for transfection. Filtered TE buffer can be stored at room temperature (RT; 20°C–24°C) for 1 yr.</div>
<h4 id="sec2.3.2">Cell harvest buffer</h4>
<p>Cell harvest buffer contains 100 mM NaCl, 50 mM Tris-HCl (pH8.0), 5% (v/v) glycerol, 1 mM PMSF and 1<b>×</b>Protease Inhibitor Cocktail (PIC) (Roche).</p>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>100 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl (pH8.0)</td>
<td>50 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>100% Glycerol</td>
<td>5% (v/v)</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>43 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
<tr>
<td>1 M PMSF</td>
<td>1 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>PIC (Roche)</td>
<td>N/A</td>
<td>1 tablet</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> We recommend adding PMSF and PIC immediately before harvesting the cells. The cell harvest buffer can be stored at 4°C for a month.</div>
<h4 id="sec2.3.3">Wash buffers</h4>
<p>Three different buffers are used to remove nonspecifically bound contaminants from the anti-FLAG M2 resin. The wash buffers can be stored at 4°C for a month.</p>
<p>Wash buffer A contains 100 mM NaCl, 50 mM Tris-HCl (pH8.0) and 5% (v/v) glycerol.</p>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>100 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl (pH8.0)</td>
<td>50 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>100% Glycerol</td>
<td>5% (v/v)</td>
<td>50 mL</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>880 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<p>Wash buffer B contains 300 mM NaCl, 50 mM Tris-HCl (pH8.0) and 5% (v/v) glycerol.</p>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>300 mM</td>
<td>60 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl (pH8.0)</td>
<td>50 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>100% Glycerol</td>
<td>5% (v/v)</td>
<td>50 mL</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>840 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<p>Wash buffer C contains 500 mM NaCl, 50 mM Tris-HCl (pH8.0) and 5% (v/v) glycerol.</p>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>500 mM</td>
<td>100 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl (pH8.0)</td>
<td>50 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>100% Glycerol</td>
<td>5% (v/v)</td>
<td>50 mL</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>800 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<h4 id="sec2.3.4">Anti-FLAG M2 resin elution buffer (freshly made)</h4>
<p>Anti-FLAG M2 elution buffer contains 150 mM NaCl, 50 mM Tris-HCl (pH8.0), 5% (v/v) glycerol, and 0.4 mg/mL FLAG peptides (sequence: DYKDDDDK).</p>
<table id="undtbl8">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>150 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl (pH8.0)</td>
<td>50 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>100% Glycerol</td>
<td>5% (v/v)</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>43.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
<tr>
<td>FLAG peptides</td>
<td>0.4 mg/mL</td>
<td>N/A</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> FLAG peptides (Apeptide) in powder were dissolved immediately before use.</div>
<h4 id="sec2.3.5">Dialysis buffer (freshly made)</h4>
<p>This dialysis buffer is used when incubation with TEV protease to remove tags. The dialysis buffer contains 150 mM NaCl, 50 mM Tris-HCl (pH8.0), 5% (v/v) glycerol, 1 mM DTT, 0.5 mM EDTA (pH8.0) and 20 mM Imidazole.</p>
<table id="undtbl9">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>150 mM</td>
<td>60 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl (pH8.0)</td>
<td>50 mM</td>
<td>100 mL</td>
</tr>
<tr>
<td>100% Glycerol</td>
<td>5% (v/v)</td>
<td>100 mL</td>
</tr>
<tr>
<td>1 M DTT</td>
<td>1 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>0.5 M EDTA (pH8.0)</td>
<td>0.5 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>1 M Imidazole (pH8.0)</td>
<td>20 mM</td>
<td>40 mL</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>1,696 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>2,000 mL</b></td>
</tr>
</tbody>
</table>
<h4 id="sec2.3.6">Gel filtration buffer</h4>
<p>Gel filtration buffer contains 150 mM NaCl and 20 mM HEPES (pH7.5).</p>
<table id="undtbl10">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 M NaCl</td>
<td>150 mM</td>
<td>30 mL</td>
</tr>
<tr>
<td>1 M HEPES (pH7.5)</td>
<td>20 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>N/A</td>
<td>950 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The buffer should be degassed and filtered through a 0.22 μm MCE membrane filter (Millipore™). The gel filtration buffer can be stored at 4°C for a month.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<div class="note">
<span class="note-title">Note:</span> The purification scheme was visualized with a flow diagram (<a href="#fig2">Figure 2</a>).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1103-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. A workflow of Huntingtin protein expression and purification</div>
</figcaption>
</figure>
<h3 id="sec3.1">DH10Bac transformation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>Generate DH10Bac strains containing the bacmid inserted with the <i>HTT</i> cDNA.</p>
<ol>
<li>Add 100 ng of pFastBac1 plasmid containing the full-length <i>H</i><i>TT</i> cDNA to 100 μL of DH10Bac™ (Thermo Fisher Scientific) competent cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Mix gently. Do not pipet.</div>
<ol start="2">
<li>Incubate the cells on ice for 30 min.</li>
<li>Heat-shock at 42°C for 45 s.</li>
<li>Incubate the cells on ice again for 2 min.</li>
<li>Add 900 μL of S.O.C. medium to the cells and incubate at 37°C for 4 h in a shaking incubator.</li>
<li>Prepare a series of 10<b>×</b>, 100<b>×</b>, and 1000<b>×</b> diluted cells with S.O.C. medium.</li>
<li>Plate 100 μL of each diluted cell sample on DH10Bac LB agar plates.</li>
<li>Incubate the plates for 2 days at 37°C in an incubator.</li>
<li>Pick 3–4 white colonies and restreak them onto new DH10Bac LB agar plates.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step is essential to distinguish true white colonies from blue colonies.</div>
<div class="note">
<span class="note-title">Note:</span> If it is difficult to distinguish between blue and white colonies due to the light blue color of some colonies, refer to the troubleshooting <a href="#sec6.1">problem 1</a>.</div>
<ol start="10">
<li>Incubate the restreaked plates for 2 days at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The resteaked plates can be stored at 4°C for 2–3 days.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not store the plate more than 2 days. Use fresh colonies for bacmid purification. This step substantially affects the quality of the viruses and protein production.</div>
<ol start="11">
<li>Inoculate a single white colony into 5 mL of DH10Bac LB medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We usually grow 3–5 cultures from several single white colonies.</div>
<ol start="12">
<li>Incubate for 2 days at 37°C in a shaking incubator.</li>
</ol>
<h3 id="sec3.2">Bacmid isolation and analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 h</div>
<p>Isolate bacmid from the culture of the selected white colony and confirm the <i>H</i><i>TT</i> cDNA transposition into the bacmid using Polymerase Chain Reaction (PCR)</p>
<div class="note">
<span class="note-title">Note:</span> We use Sol I, Sol II and Sol III buffers from the Plasmid DNA Purification Kit Mini (Labopass)</div>
<ol start="13">
<li>Harvest the cells by centrifugation at 3,000<b>×</b><i>g</i> for 20 min at 4°C.</li>
<li>Remove the supernatant and resuspend the cell pellet in 300 μL of Sol I buffer.</li>
<li>Add 300 μL of Sol II buffer and mix gently by inverting. Incubate the sample at room temperature (RT; 20°C–24°C) for 5 min.</li>
<li>Add 300 μL of Sol III buffer to the mixture and gently mix by inverting. Incubate the sample on ice for 10 min.</li>
<li>Centrifuge the sample at 17,000<b>×</b><i>g</i> for 20 min at 4°C.</li>
<li>Transfer 750 μL of the supernatant to 800 μL of 100% isopropanol and mix gently the sample by inverting. Incubate the sample on ice for 10 min.</li>
<li>Centrifuge the incubated sample at 17,000<b>×</b><i>g</i> for 20 min at RT.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After step #19, a white DNA pellet should be observed at the bottom of the tube.</div>
<ol start="20">
<li>Decant the supernatant and add 1 mL of 70% ethanol on the top of the white pellet. Wash the pellet by inverting the tube.</li>
<li>Centrifuge the sample at 17,000<b>×</b><i>g</i> for 5 min at RT.</li>
<li>After centrifugation, remove the supernatant with a pipet without disturbing the pellet. Dry the white DNA pellet with airdry while the tubes are upside-down on a Kimwipe until the white pellet becomes transparent.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the white pellet does not become transparent and stays white, it is an indication of impure bacmid. Refer to the troubleshooting <a href="#sec6.3">problem 2</a>.</div>
<ol start="23">
<li>Dissolve the transparent DNA pellet by adding 40 μL of TE buffer. Resuspend the DNA pellet gently by tapping. Do not use a pipette.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Store the bacmid at 4°C, and use the bacmid within a day.</div>
<div class="note">
<span class="note-title">Note:</span> In general, the concentration of DNA (bacmid) is ∼4 μg/μL in 40 μl TE (total 160 μg).</div>
<div class="note">
<span class="note-title">Note:</span> The purity of the bacmid can be monitored by the A<sub>260</sub>/A<sub>280</sub> ratio. The A<sub>260</sub>/A<sub>280</sub> ratio of the bacmid should be approximately 2.</div>
<ol start="24">
<li>Perform PCR using the M13 forward and hHTT R146 primers (the sequences of each primer are described in the <a href="#key-resources-table">key resources table</a>). The PCR product should appear at approximately 2.9 kbp when the <i>H</i><i>TT</i> c cDNA is properly integrated into bacmids.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Refer to <a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a>.</div>
<table id="tbl1">
<caption>Table 1. The detailed PCR composition</caption>
<thead>
<tr>
<th>Component</th>
<th>Amount (μl)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bacmid DNA (final 100 ng)</td>
<td>1</td>
</tr>
<tr>
<td>2<b>×</b>TOPsimple™ PreMIX-Tenuto</td>
<td>25</td>
</tr>
<tr>
<td>5<b>×</b>Band Doctor™</td>
<td>10</td>
</tr>
<tr>
<td>M13 forward primer (final 10 pmol)</td>
<td>1</td>
</tr>
<tr>
<td>hHTT R146 reverse primer (final 10 pmol)</td>
<td>1</td>
</tr>
<tr>
<td>Sterile Water</td>
<td>12</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50</b></td>
</tr>
</tbody>
</table>
<table id="tbl2">
<caption>Table 2. The reaction conditions of the PCR reaction for the bacmid plasmid containing <i>HTT</i> gene</caption>
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>94°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>94°C</td>
<td>45 s</td>
<td rowspan="3">30 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>45 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>3+α min <sup><a href="#tblfn1">a</a></sup>
</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>7 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">-</td>
</tr>
</tbody>
</table>
<p id="tblfn1" class="table-footnote"><sup>a</sup>Depending on the number of CAG repeats, add extra min for extension (α=0.18 s per one CAG repeat).</p>
<h3 id="sec3.3">Generating and amplify baculovirus</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 weeks</div>
<p>Transfect the bacmid into Sf9 cells to generate baculovirus and amplify the baculovirus for large scale infection to produce HTT proteins.</p>
<ol start="25">
<li>Plate 1.0<b>×</b>10<sup>6</sup> Sf9 cells in 2 mL CCM3 media (0.5<b>×</b>10<sup>6</sup> cells/mL) into each well in a 6-well plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Sf9 cells were used in the log phase of the growth where the cell density was between 2–4<b>×</b>10<sup>6</sup>.</div>
<ol start="26">
<li>Incubate the cells to attach to the bottom of the plate at 27°C for 1 h.</li>
<li>While the cells attach to the plate, prepare 1 μg of bacmid in 100 μL of CCM3 media and 6 μL of Cellfectin II (Thermo Fisher Scientific) in 100 μL of CCM3 media separately.</li>
<li>Combine the bacmid in CCM3 and the Cellfectin II in CCM3 from step #27, and mix gently by inverting. Incubate the mixture for 45 min at RT.</li>
<li>Remove the medium from each well from step #26 and add 2 mL of fresh CCM medium to each well.</li>
<li>After incubating the mixture of bacmid and Cellfectin II at step #28, add 800 μL of fresh CCM medium to the mixture and mix gently by inverting.</li>
<li>Remove the medium from each well again from step #29 and add the bacmid-Cellfectin II mixture (∼1 mL) to the cells. Incubate the plate at 27°C for 5 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure that one well remains as a negative control without bacmid at step #27–28.</div>
<ol start="32">
<li>After 5 h, remove the medium and add 2 mL of fresh CCM media to each well, and incubate the plate at 27°C for 3 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The media loss by evaporation during incubation can be prevented by incubating the cell plate in a sealed chamber such as a Ziploc container with wet Kimwipes.</div>
<ol start="33">
<li>After 3 days, collect the medium containing baculovirus from the wells and filter the medium by using a 0.22 μm syringe filter (Sartorius). This is a P1 virus stock.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The volume of P1 virus stock is approximately around 1.5 mL.</div>
<div class="note">
<span class="note-title">Note:</span> We do not observe clear indication of the infection such as enlarged cells and detached cells, at this stage.</div>
<ol start="34">
<li>Plate 5<b>×</b>10<sup>6</sup> Sf9 cells in 10 mL into a 75 cm<sup>2</sup> cell culture T-flask (or 10 cm<sup>2</sup> cell culture dish). Wait for 30 min to let the cells attach to the bottom of the plate. Add 50 μL of the harvested P1 virus from step #33 to the cells. Shake gently the flask to mix the virus in the cell media.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As a control, leave at least one plate of cells without baculovirus.</div>
<ol start="35">
<li>Incubate the flask at 27°C for 3–5 days</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> We recommend checking the condition of the infected cells compared with the control to confirm baculovirus infection. The infected Sf9 cells should become enlarged and detached from the plates especially during P2 and P3 virus generation, but not necessarily during P1 generation. Please see <a href="#fig3">Figure 3</a> for representative images. Refer to the troubleshooting <a href="#sec6.5">problem 3</a>.</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1103-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative images of baculovirus-infected Sf9 cells</div>
<p>(A and B) Images of Sf9 cells were taken under a light-microscopy after 5 days of post-infection of P2 baculovirus. The infected cells (B) are enlarged compared with the control cells (A).</p>
</figcaption>
</figure>
<ol start="36">
<li>Collect and filter using Steriflip filter unit (Merck Millipore). This is a P2 virus stock.</li>
<li>Plate 15<b>×</b>10<sup>6</sup> Sf9 cells in 25 mL (final 0.6<b>×</b>10<sup>6</sup> cells/mL) into 150 cm<sup>2</sup> cell culture dish. Wait for 30 min to let the cells attach to the bottom of the dish. In addition, add 50 μL of the P2 virus to cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As a control, leave at least one culture dish of cells without baculovirus.</div>
<ol start="38">
<li>Incubation at 27°C for 5–6 days.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> You should observe that most of the cells become enlarged and are floating in the media as the cells are detached. If the cells are not sufficiently detached, incubate 1–2 more days.</div>
<ol start="39">
<li>As there are a lot of cells in the media, centrifuge the media at 4,000<b>×</b><i>g</i> for 10 min and collect the media.</li>
<li>Filter the media using 0.22 μm bottle top filters (PES, Corning). This is a P3 virus stock.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We usually make 50 mL of P2 and 500 mL of P3.</div>
<div class="note">
<span class="note-title">Note:</span> Store the virus at 4°C and protect it from light.</div>
<div class="note">
<span class="note-title">Note:</span> The virus can be stored for one month without affecting the level of protein expression.</div>
<h3 id="sec3.4">Large-scale protein expression</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Infect baculovirus into Sf9 cells to express HTT proteins.</p>
<div class="note">
<span class="note-title">Note:</span> This protocol is based on a purification procedure from 8 L of Sf9 cell culture.</div>
<ol start="41">
<li>Prepare cells (final density of cells in 1,000 mL media: 1.0–1.5<b>×</b>10<sup>6</sup> cells/mL) in 1,000 mL of CCM3 media in a 2,800 mL Fernbach flask a day before the large-scale infection.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We dilute Sf9 cells on a large scale from Sf9 cells at 6–8<b>×</b>10<sup>6</sup> cells/mL.</div>
<ol start="42">
<li>Incubate the cell until 2.0–2.5<b>×</b>10<sup>6</sup> cells/mL density at 27°C with 100 rpm in an orbital shaker incubator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Typically, it will take 24 h to reach 2.0<b>×</b>10 <sup>6</sup> cells/mL from 1.0<b>×</b>10<sup>6</sup> cells/mL at 27°C at 100 rpm in an INFORS HT shaking incubator</div>
<ol start="43">
<li>When the proper cell density is reached, add 50 mL of the P3 virus to 1,000 mL of the cells in a 2,800 mL Fernbach flask.</li>
<li>Incubate at 27°C for 42–48 h.</li>
<li>Harvest the cells by centrifugation at 4,600<b>×</b><i>g</i> for 20 min at 4°C.</li>
<li>Decant the medium and resuspend the cell pellet with 30 mL of the cell harvest buffer for the cells from 1,000 mL of cell culture.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure that 1 mM PMSF and final 1<b>×</b> Protease inhibitor Cocktail are added to the cell harvest buffer before use.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Before resuspending the cells, wash the surface of the cell pellet with the cell harvest buffer. This step is important to reduce the degradation of HTT proteins.</div>
<ol start="47">
<li>Freeze the resuspended cells in liquid nitrogen and store them at −80°C.</li>
</ol>
<h3 id="sec3.5">FLAG affinity chromatography</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 9 h</div>
<p>Isolate the recombinant FLAG tag-HTT with anti-FLAG M2 affinity gel from cell lysate.</p>
<div class="critical">
<span class="critical-title">Critical:</span> From this step, perform all steps at 4°C.</div>
<ol start="48">
<li>The resuspended cells were lysed by repeatedly freezing and thawing in liquid nitrogen and in a water bath at 25°C for 3 times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> No other reagent such as DNase is necessary for cell lysis.</div>
<ol start="49">
<li>Centrifuge the cell lysate at 27,000<b>×</b><i>g</i> for 2 h using 50 mL centrifuge tubes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the supernatant is still turbid after this step, centrifuge again with the same condition for another 1 hr.</div>
<div class="note">
<span class="note-title">Note:</span> There should be clear supernatant and soft sponge like pellet. We don’t expect see solid pellet.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Centrifugation at lower speed (<i>g</i>) will result in little supernatant with huge amount of sponge like pellet.</div>
<ol start="50">
<li>Collect the supernatant by pouring out without disturbing the soft pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We usually obtain 30 mL of the supernatant from 1 L of Sf9 cell culture. Therefore, the volume of the supernatant for 8 L culture is approximately 240 mL.</div>
<div class="note">
<span class="note-title">Note:</span> Equally distribute the supernatant to six 50 mL conical tubes.</div>
<ol start="51">
<li>Equilibrate the anti-FLAG M2 affinity resin (Sigma) with the wash buffer A.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use 10 mL (packed bead volume) of M2 resin for 8 L of Sf9 cell culture.</div>
<ol start="52">
<li>Equally distribute 10 mL of the M2 resin to the supernatants from step #50.</li>
<li>Incubate the mixture of the supernatant and the M2 resin for 3 h at 4°C in a tube rotator or a tumbler at 10 turns per min.</li>
<li>Collect the M2 resin by centrifugation at 1,500<b>×</b><i>g</i> for 5 min.</li>
<li>After slowly pouring out the supernatant from the centrifuged M2 resin, collect all the resin in a single 50 mL tube with the wash buffer A and bring the solution to 50 mL by adding the wash buffer A.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After centrifugation, the M2 resin sits at the bottom of the 50 mL tubes. Slowly decant the supernatant by pouring out.</div>
<ol start="56">
<li>Collect the M2 resin by centrifugation at 1,500<b>×</b><i>g</i> for 5 min and decant the supernatant by pouring out.</li>
<li>Add 40 mL of the wash buffer A to the M2 resin and resuspend by inverting the tube.</li>
<li>Repeat step #56–57 for 3 times with the wash buffer A.</li>
<li>Repeat step #56–57 for 3 times with the wash buffer B.</li>
<li>Repeat step #56–57 for 3 times with the wash buffer C.</li>
<li>Repeat step #56–57 for 3 times with the wash buffer B.</li>
<li>Repeat step #56–57 for 3 times with the wash buffer A.</li>
<li>After washing, load the M2 resin to an Econo-Column® Chromatography Columns, 2.5 × 10 cm (Bio-Rad).</li>
<li>Wash the resin with the wash buffer A until the absorbance of the flow-through becomes lower than 0.01 at the OD<sub>280</sub> reading.</li>
<li>Decant the wash buffer A to the surface of the M2 resin.</li>
<li>Add 1 column volume (CV, 10 mL) of the M2 elution buffer slowly on the top of the resin without disturbing the M2 resin and incubate for 10 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> A 10 min incubation with the elution buffer is important for better elution.</div>
<ol start="67">
<li>Collect the incubated elution buffer.</li>
<li>Repeat step #66–67 until the OD<sub>280</sub> reading of the eluted sample is less than 0.01.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We usually collect HTT proteins with 5–6 elutions (total 50–60 mL).</div>
<div class="note">
<span class="note-title">Note:</span> The expected amount of the proteins at this stage is approximately 15–20 mg from 8 L of the Sf9 cell culture. The amounts of proteins were estimated by OD<sub>280</sub> with Abs 0.1% (1 g/L) of 0.78 (the molar extinction coefficient of Q23 HTT is 272,755 cm<sup>−1</sup>M<sup>−1</sup> at 280 nm) and the purity of the samples is higher than 80% based on estimation by SDS-PAGE.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The A<sub>260</sub>/A<sub>280</sub> ratio should be monitored at this stage. If the ratio is over 1.0, this indicates the contamination of nucleic acids in the sample, which results in significant loss of the proteins during the gel-filtration chromatography step. Refer to the troubleshooting <a href="#sec6.7">problem 4</a>.</div>
<h3 id="sec3.6">TEV protease treatment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 h</div>
<p>Remove the N-terminal tags of HTT with TEV protease.</p>
<ol start="69">
<li>Collect the eluted samples and add TEV protease.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use home-made TEV protease, which has an N-terminal 6<b>×</b>His-tag. Add 40 μg of TEV protease per 1 mg of the eluted samples (800 μg of TEV protease for 20 mg of HTT protein).</div>
<div class="note">
<span class="note-title">Note:</span> TEV proteases are commercially available (Thermo Fisher, Cat# 1257023)</div>
<ol start="70">
<li>Transfer the sample with TEV protease to Spectra/Por® 1 Dialysis membrane (30 Kd MWCO).</li>
<li>Dialyze the sample for 12–16 h against 2 L of the dialysis buffer.</li>
</ol>
<h3 id="sec3.7">Nickel affinity chromatography</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Remove the uncleaved protein and the remaining TEV protease with Ni affinity chromatography.</p>
<ol start="72">
<li>Load the Ni-nitrilotriacetic acid resin (Ni-NTA, Qiagen) to a Econo column (2.5<b>×</b>10 cm).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use 10 mL of Ni-NTA.</div>
<div class="note">
<span class="note-title">Note:</span> Ensure that Ni-NTA is resistant to 1 mM DTT and 0.5 mM EDTA in the dialysis buffer. We found that Ni-NTA from Qiagen is suitable for this step.</div>
<ol start="73">
<li>Equilibrate the Ni-NTA resin with the dialysis buffer.</li>
<li>Load the sample from step #71 to pass through the equilibrated Ni-NTA column and collect the flow-through.</li>
<li>Reload the collected flow-through to the Ni-NTA again.</li>
<li>Repeat step #74–75 for 3 times.</li>
<li>Concentrate the sample using an Amicon Ultra 100-KDa filter up to 10 mL (less than 2 mg/mL concentration) by repeated centrifugation at 1,500<b>×</b><i>g</i> for 5 min each time.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Repeated centrifugation at 1,500<b>×</b><i>g</i> for 5 min each time is critical as exceeded time or centrifugation at higher speed (g) can induce precipitant. Please pipetting the samples after every centrifuge cycle.</div>
<h3 id="sec3.8">Size exclusion chromatography</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<div class="note">
<span class="note-title">Note:</span> Elution profiles of the size exclusion chromatography and protein quality of final concentrated samples are shown in <a href="#fig4">Figure 4</a>.</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1103-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Size exclusion chromatography and PAGE analysis of recombinant HTT</div>
<p>(A) The elution profile of HTT purification from size exclusion chromatography with 10% SDS-PAGE analysis of the fractions. Asterisk indicates the position of the fraction in the gel.</p>
<p>(B) The purity of purified HTT was analyzed by 10% SDS-PAGE (lefts) and 5% native-PAGE (right). The proteins were visualized by Coomassie staining.</p>
</figcaption>
</figure>
<ol start="78">
<li>Equilibrate a Superdex 200 HiLoad 26/60 column (Cytiva) with the gel filtration buffer.</li>
<li>Load the concentrated sample from step #77 to the equilibrated column and run the column with a 2.0 mL/min flow rate and 5 mL fraction size.</li>
<li>Evaluate the purity of each fraction by SDS-PAGE.</li>
<li>Collect the fractions containing HTT proteins and concentrate the sample using an Amicon Ultra 100-KDa filter up to 5 mg/mL concentration.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Concentrate over 5 mg/mL can lead to precipitation.</div>
<ol start="82">
<li>Aliquot the sample into ∼20 μL per tube and snap freeze with liquid nitrogen. Store the samples at −80°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The sample can remain stable at −80°C up to 1 year.</div>
<div class="note">
<span class="note-title">Note:</span> With this protocol, we obtained 8 mg of recombinant HTT proteins from 8 L of Sf9 cell culture.</div>
<div class="note">
<span class="note-title">Note:</span> The expected purity of samples at this step is over 95%.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the samples are centrifuged after thawing the proteins to remove precipitant. When the proteins are thawed, the sample should be cleared by centrifugation at 17,000<b>×</b><i>g</i> for 20 min at 4°C to remove any precipitant formed during storage. Otherwise, all the HTT proteins will start to precipitate.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">SDS-PAGE and western blotting assay on HTT proteins</h3>
<p>To confirm the quality of the purified recombinant HTT proteins from this protocol, SDS-PAGE and western blotting assay were performed with HTT proteins. The purity of the purified proteins is more than 95%, as assessed by SDS- and native-PAGE gels stained with Coomassie blue (<a href="#fig4">Figure 4</a>B), showing that HTT proteins form multimers.</p>
<p>Using this protocol, we generated HTT proteins with allelic series of polyQ lengths (Q2, Q23, Q46, Q67 and Q78). The size differences of the samples with different polyQ lengths were distinguished by 5% SDS-PAGE (<a href="#fig5">Figure 5</a>A). The presence of expanded polyQ was examined with mAb2166 (Millipore) and 1F8 (<a href="#bib11">Wheeler et al., 2000</a>) monoclonal antibodies, which recognize N-HEAT (181–810 a.a) and expanded polyQ respectively (<a href="#fig5">Figure 5</a>B)</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1103-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. SDS-PAGE and western blot analysis on the purified HTT proteins with allelic series of polyQ lengths</div>
<p>(A) The purity of recombinant full-length HTT with various polyQ (Q2, Q23, Q46, Q67, Q78) was analyzed by 5% SDS-PAGE and Coomassie staining.</p>
<p>(B) Purified recombinant full-length HTT were analyzed via western blotting with mAb2166 (above) and 1F8 (below) recognizing N-HEAT (181–810 a.a.) and expanded polyQ respectively.</p>
</figcaption>
</figure>
<h3 id="sec4.2">Recombinant HTT forms a complex with HAP40</h3>
<p>HTT protein forms a complex with Huntingtin-associated protein 40 (HAP40) (<a href="#bib1">Guo et al., 2018</a>). To examine whether the HAP40 and HTT proteins form a complex using this protocol, we coinfected Sf9 cells with baculoviruses harboring either HAP40 or HTT expressing to induce HTT_HAP40 complex (50 mL of each virus to 1 L of Sf9 cells). After harvesting the expressed cells, the HTT_HAP40 complex was purified with the same protocol as HTT protein alone. The elution profile from Superdex S200 chromatography followed by SDS-PAGE analysis (<a href="#fig6">Figure 6</a>B) show that HTT forms a stoichiometric complex with HAP40. The quality of the complex was examined by SDS- and native-PAGE (<a href="#fig6">Figure 6</a>B). We also noticed that HTT_HAP40 complex forms a multimer, which is contrast to the previous works (<a href="#bib1">Guo et al., 2018</a>; <a href="#bib2">Harding et al., 2021</a>; <a href="#bib4">Huang et al., 2021</a>). However, it is possible that the multimer of HTT_HAP40 complex may be induced in high concentration, which is in our case. Furthermore, we analyzed whether isolated HTT can interact with isolated HAP40. We performed an <i>in-vitro</i> pulldown assay with recombinant GST-HAP40 (<a href="#fig6">Figure 6</a>C). GST or GST-HAP40 were expressed in Sf9 cells and the cells were harvested in a buffer A containing 150 mM NaCl, 20 mM HEPES (pH7.5). The cells were lysed by repeated freezing and thawing 3 times and cell debris was cleared with centrifugation at 17,000<b>×</b><i>g</i> for 1 h at 4°C. The proteins were purified with glutathione-Sepharose<sup>TM</sup> 4B resins (Cytiva). The GST resins bound with GST or GST-HAP40 were incubated with purified recombinant HTT (300 μg). After incubation the resins were washed with a buffer A to remove the unbound HTT proteins. The proteins were then eluted from the resins with a buffer B containing150 mM NaCl, 20 mM HEPES (pH7.5) and 20 mM reduced glutathione. The eluted proteins were analyzed by SDS-PAGE with Coomassie blue staining.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1103-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. HTT proteins form a complex with HAP40</div>
<p>(A) The elution profile of size exclusion chromatography and 10% SDS-PAGE analysis of the purified HTT-HAP40, which were co-expressed. Asterisk indicates the position of the fraction.</p>
<p>(B) The purity of the HTT and HTT_HAP40 complex was confirmed by SDS-PAGE (left) and Native-PAGE (right).</p>
<p>(C) GST pull-down assay with GST-HAP40 and isolated HTT protein.</p>
</figcaption>
</figure>
<p>Although the purified HTT protein can be pulled down with GST-HAP40, HTT_HAP40 does not form a stable complex when compared with the complex coexpressed, which is consistent with previous works (<a href="#bib1">Guo et al., 2018</a>; <a href="#bib2">Harding et al., 2021</a>; <a href="#bib4">Huang et al., 2021</a>). These data demonstrated that the HTT complexes can be generated through this protocol.</p>
<h3 id="sec4.3">Recombinant HTT proteins form a complex with HAP1</h3>
<p>Huntingtin-associated protein 1 (HAP1) is the first identified protein as a HTT interacting protein using yeast two-hybrid screen (Y2H) (<a href="#bib8">Li et al., 1995</a>). The HAP1 (278–599 a.a.) and HTT (1–253 a.a.) was shown to be an essential region for binding to each other (<a href="#bib7">Li et al., 1998</a>). To examine the binding of the recombinant HTT proteins purified from this protocol to HAP1, we performed a GST pulldown assay using GST-HAP1 (300–381 a.a.) (<a href="#fig7">Figure 7</a>A). Purified GST or GST-HAP1 were incubated with glutathione-Sepharose<sup>TM</sup> 4B resins (Cytiva) in a total volume of 500 μL of buffer A containting 150 mM NaCl, 20 mM HEPES (pH7.5) for 1 h at 4°C. The resins were washed with the buffer A followed by incubation with purified recombinant HTT (300 μg). After incubation, the resins were washed with the buffer A and the proteins were eluted with the buffer B and analyzed by SDS-PAGE with Coomassie blue staining. As shown in the results, the purified recombinant HTT can directly bind to the HAP1 fragments. These results indicate that the HTT protein generated through this protocol retains its HAP1 binding activity.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1103-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. <i>In</i> <i>vitro</i> binding assay using GST-HAP1 and recombinant HTT protein</div>
<p>The GST pull-down assay was used to test the binding between purified HTT and GST-HAP1 (300–381 a.a.). The result of the GST pull-down assay was analyzed by SDS-PAGE and Coomassie staining.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Since the normal function of HTT is still unclear, experiments to directly confirm the activity of recombinant HTT were not conducted in this paper. However, we have shown that the recombinant HTT purified by using this protocol activates the histone H3K27 tri-methylase activity of Polycomb Repressive Complex 2 (PRC2) in vitro (<a href="#bib10">Seong et al., 2010</a>). In addition, our cryo-EM analysis (EMDB: EMD-4944) (<a href="#bib6">Jung et al., 2020</a>) on full-length HTT proteins obtained using this method shows that they have similar structures to those previously resolved high-resolution cryo-EM structure of HTT (<a href="#bib1">Guo et al., 2018</a>; <a href="#bib2">Harding et al., 2021</a>; <a href="#bib4">Huang et al., 2021</a>) and maintains the similar phosphorylation pattern to the previously identified phosphorylation sites (<a href="#bib3">Harding et al., 2019</a>; <a href="#bib6">Jung et al., 2020</a>; <a href="#bib9">Saudou and Humbert, 2016</a>). These results suggest that this protocol to generate recombinant full-length HTT proteins with allelic series of polyQ lengths will provide key reagents for various experiments to investigate the functions of HTT and the molecular mechanism of HD pathology. It should be noted that other protocols have been used to generate recombinant full-length HTT proteins (<a href="#bib3">Harding et al., 2019</a>; <a href="#bib5">Huang et al., 2015</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The blue color of colonies is too weak in step #9.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>When Bluo-Gal is added to the autoclaved LB agar when it is too hot, Bluo-Gal goes bad and does not show clear blue color. Bluo-Gal was added when the temperature of the autoclaved LB agar was below 50°C.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The pellet was still turbid after dry in step #22.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This is an indication that bacmid is not pure. Pick new white colonies (step #11) and purify the bacmid again.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The infected Sf9 cells are not enlarged nor detached from the plates in step #35.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>You should see that the infected Sf9 cells become enlarged and detached from the plates during P2 and P3 virus generation, but not necessary during P1 virus generation. This is an indication that the quality of the virus is poor. We recommend returning to step #1.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The A<sub>260</sub>/A<sub>280</sub> ratio of the eluted samples is over 1.0 in step #68.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This indicates the presence of the genetic material contamination, which results in significant loss of the proteins during the gel-filtration chromatography step. This problem can be solved by washing with additional wash buffer C in step #60.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>The yield of final product is less than suggested in step #82.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Return to step #37–39 with the harvested P3 virus. This amplified virus is a P4 virus stock and is used in infection for expression. If that does not work, consider making a new virus from a fresh bacmid. Please refer to the critical points of this protocol in “<a href="#before-you-begin">before you begin</a>”.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Guo, Q., Bin, H., Cheng, J., Seefelder, M., Engler, T., Pfeifer, G., Oeckl, P., Otto, M., Moser, F., Maurer, M., et al. (2018). The cryo-electron microscopy structure of huntingtin. Nature <i>555</i>, 117-120. <a class="external-link" href="https://doi.org/10.1038/nature25502">https://doi.org/10.1038/nature25502</a></p>
<p id="bib2">Harding, R.J., Deme, J.C., Hevler, J.F., Tamara, S., Lemak, A., Cantle, J.P., Szewczyk, M.M., Zuo, X., Loppnau, P., Seitova, A., et al. (2021). HAP40 orchestrates huntingtin structure for differential interaction with polyglutamine expanded exon 1. bioRxiv. . <a class="external-link" href="https://doi.org/10.1101/2021.04.02.438217">https://doi.org/10.1101/2021.04.02.438217</a></p>
<p id="bib3">Harding, R.J., Loppnau, P., Ackloo, S., Lemak, A., Hutchinson, A., Hunt, B., Holehouse, A.S., Ho, J.C., Fan, L., Toledo-Sherman, L., et al. (2019). Design and characterization of mutant and wildtype huntingtin proteins produced from a toolkit of scalable eukaryotic expression systems. J. Biol. Chem. <i>294</i>, 6986-7001. <a class="external-link" href="https://doi.org/10.1074/jbc.RA118.007204">https://doi.org/10.1074/jbc.RA118.007204</a></p>
<p id="bib4">Huang, B., Guo, Q., Niedermeier, M.L., Cheng, J., Engler, T., Maurer, M., Pautsch, A., Baumeister, W., Stengel, F., Kochanek, S., and Fernandez-Busnadiego, R. (2021). Pathological polyQ expansion does not alter the conformation of the Huntingtin-HAP40 complex. Structure <i>29</i>, 804-809 e805. <a class="external-link" href="https://doi.org/10.1016/j.str.2021.04.003">https://doi.org/10.1016/j.str.2021.04.003</a></p>
<p id="bib5">Huang, B., Lucas, T., Kueppers, C., Dong, X., Krause, M., Bepperling, A., Buchner, J., Voshol, H., Weiss, A., Gerrits, B., and Kochanek, S. (2015). Scalable production in human cells and biochemical characterization of full-length normal and mutant huntingtin. PLoS One <i>10</i>, e0121055. <a class="external-link" href="https://doi.org/10.1371/journal.pone.0121055">https://doi.org/10.1371/journal.pone.0121055</a></p>
<p id="bib6">Jung, T., Shin, B., Tamo, G., Kim, H., Vijayvargia, R., Leitner, A., Marcaida, M.J., Astorga-Wells, J., Jung, R., Aebersold, R., et al. (2020). The polyglutamine expansion at the N-terminal of huntingtin protein modulates the dynamic configuration and phosphorylation of the C-terminal HEAT domain. Structure <i>28</i>, 1035-1050 e1038. <a class="external-link" href="https://doi.org/10.1016/j.str.2020.06.008">https://doi.org/10.1016/j.str.2020.06.008</a></p>
<p id="bib7">Li, S.H., Gutekunst, C.A., Hersch, S.M., and Li, X.J. (1998). Interaction of huntingtin-associated protein with dynactin P150Glued. J. Neurosci. <i>18</i>, 1261-1269. <a class="external-link" href="https://doi.org/10.1523/jneurosci.18-04-01261.1998">https://doi.org/10.1523/jneurosci.18-04-01261.1998</a></p>
<p id="bib8">Li, X.J., Li, S.H., Sharp, A.H., Nucifora, F.C., Schilling, G., Lanahan, A., Worley, P., Snyder, S.H., and Ross, C.A. (1995). A huntingtin-associated protein enriched in brain with implications for pathology. Nature <i>378</i>, 398-402. <a class="external-link" href="https://doi.org/10.1038/378398a0">https://doi.org/10.1038/378398a0</a></p>
<p id="bib9">Saudou, F. and Humbert, S. (2016). The biology of huntingtin. Neuron <i>89</i>, 910-926. <a class="external-link" href="https://doi.org/10.1016/j.neuron.2016.02.003">https://doi.org/10.1016/j.neuron.2016.02.003</a></p>
<p id="bib10">Seong, I.S., Woda, J.M., Song, J.J., Lloret, A., Abeyrathne, P.D., Woo, C.J., Gregory, G., Lee, J.M., Wheeler, V.C., Walz, T., et al. (2010). Huntingtin facilitates polycomb repressive complex 2. Hum. Mol. Genet. <i>19</i>, 573-583. <a class="external-link" href="https://doi.org/10.1093/hmg/ddp524">https://doi.org/10.1093/hmg/ddp524</a></p>
<p id="bib11">Wheeler, V.C., White, J.K., Gutekunst, C.A., Vrbanac, V., Weaver, M., Li, X.J., Li, S.H., Yi, H., Vonsattel, J.P., Gusella, J.F., et al. (2000). Long glutamine tracts cause nuclear localization of a novel form of huntingtin in medium spiny striatal neurons in HdhQ92 and HdhQ111 knock-in mice. Hum. Mol. Genet. <i>9</i>, 503-513. <a class="external-link" href="https://doi.org/10.1093/hmg/9.4.503">https://doi.org/10.1093/hmg/9.4.503</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Ji-Joon Song (<a href="mailto:songj@kaist.ac.kr">songj@kaist.ac.kr</a>).</p>
<h4>Materials availability</h4>
<p>Htt plasmids with allelic series of CAG are available at Song Lab, KAIST. As it is very tricky to maintain the stable number of CAG, the plasmid will be directly distributed from the lead contact upon request.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze datasets/code.</p>
<h3>Acknowledgments</h3>
<p>We thank the members of the Song Lab at KAIST. This work is partially supported by a grant (NRF-2016K1A1A2912057 to J.-J.S. and I.S.S.) from the <a href="https://doi.org/10.13039/501100003725">National Research Foundation of Korea</a>.</p>
<h3>Author contributions</h3>
<p>H.K. established a protocol for HTT protein purification. H.K., A.L., and I.S.S. established a protocol for plasmid amplification. H.K., K.-g.H., and J.-J.S. wrote the manuscript. All authors reviewed the data.</p>
<h3>Declaration of interests</h3>
<p>J.S. is a co-founder and CTO of PCG Biotech.</p>
</section>
</article>