<article>
<section>
<h1 id="header">Tumor dissociation of highly viable cell suspensions for single-cell omic analyses in mouse models of breast cancer</h1>
<p><time datetime="2021-09-24">Published: September 24, 2021</time></p>
<p>Laura Rodriguez de la Fuente,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">7</a></sup> Andrew M.K. Law,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> David Gallego-Ortega,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff5">5</a>,<a href="#fn1">6</a>,<a href="#cor1">*</a></sup> and Fatima Valdes-Mora<sup><a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn1">6</a>,<a href="#fn3">8</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Tumour Development Group, The Kinghorn Cancer Centre, Garvan Institute of Medical Research, Sydney, NSW 2010, Australia</p>
<p id="aff2"><sup>2</sup>St. Vincent’s Clinical School, Faculty of Medicine, University of New South Wales Sydney, Sydney, NSW 2010, Australia</p>
<p id="aff3"><sup>3</sup>Cancer Epigenetic Biology and Therapeutics Lab, Children’s Cancer Institute, Lowy Cancer Research Centre, University of New South Wales, Sydney, NSW 2052, Australia</p>
<p id="aff4"><sup>4</sup>School of Women’s and Children’s Health, Faculty of Medicine, University of New South Wales, Sydney, NSW 2052, Australia</p>
<p id="aff5"><sup>5</sup>School of Biomedical Engineering, Faculty of Engineering and IT, University of Technology Sydney, Ultimo, NSW 2007 Australia</p>
<p id="fn1"><sup>6</sup>Senior author</p>
<p id="fn2"><sup>7</sup>Technical contact</p>
<p id="fn3"><sup>8</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:david.gallegoortega@uts.edu.au">david.gallegoortega@uts.edu.au</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:fvaldesmora@ccia.org.au">fvaldesmora@ccia.org.au</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100841">10.1016/j.xpro.2021.100841</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Cell preparation with a high rate of viable cells is required to obtain reliable single-cell transcriptomic and epigenomic data. This protocol describes a technique for digestion and single-cell isolation from mouse mammary tumors to achieve ∼90% of viable cells, which can be subsequently processed in a diverse array of high-throughput single-cell “omic platforms,” both in an unbiased manner or after selection of a specific cell population.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00547-5/sref2">Valdes-Mora et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol yields cell suspensions with 90% of live cells from mouse mammary tumors</li>
<li>Necrosis and lymph nodes can bias the cell composition in breast tumors</li>
<li>Single-cell multiomic workflows require highly viable cell suspensions</li>
<li>Antibody-based cell selection (MACS or FACS) are compatible methods for cell selection</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1033-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Background</h3>
<p>Single-cell transcriptomics and epigenomics have become the preferred methods to characterize tumor cell diversity and functional heterogeneity. However, one of the limitations to obtain high quality data relies on the cell viability after single cell isolation. This protocol describes how to obtain high viable single-cell suspensions from mouse mammary tumors ready for single-cell genomic applications. Fail to produce cell preparations with a high content of viable cells for single-cell ’omic (sc-omic) applications can lead to poor data with low gene identification and high mitochondrial content. Poor quality data can often lead to the introduction of confounding cell stress signals unrelated to the biological scenario investigated and artifacts in cell identification and functional definition.</p>
<p>There are four specific steps and two variations depending on the experimental design and hypothesis to be tested. The first step involves tumor harvesting and enzymatic digestion, where tumor samples are collected and dissociated in single-cell suspension; the second step involves quality control and sample selection of the tumor areas with high-cellular content to represent heterogeneity as much as possible; the third step can either involve unbiased enrichment of all viable cells from tumors (step 3A) or antibody-based selection of specific cell subpopulations (step 3B), including rare cells. The fourth step consists of the evaluation of cell viability and preparation of cells for single-cell capture for downstream sequencing and analyisis.</p>
<p>We have tested this protocol in primary tumors of several mammary mouse models, including transgenic models such as MMTV-PyMT (<a href="#bib2">Valdes-Mora et al., 2021</a>) and syngeneic models of orthotopically injected cancer cell lines, such as the 4T1.2 and 67NR cell lines. This method has been also used in tumor metastasis from lungs, and in normal mouse lungs, liver and intestines. All experiments involving mice in this protocol have been approved by the St. Vincent’s Campus Animal Research Committee AEC #19/02, according to guidelines contained within the NSW (Australia) Animal Research Act 1985, the NSW (Australia) Animal Research Regulation 2010 and the Australian code of practice for the care and use of animals for scientific purposes, (8th Edition 2013, National Health and Medical Research Council (Australia)). For additional tissue types and cancer models, an optimization of this protocol may be required in the tumor digestion step.</p>
<p>This method has been applied for multiple sc-omic procedures, for step 3A (unbiased option) we have performed Drop-seq (<a href="#bib2">Valdes-Mora et al., 2021</a>), scRNA-seq Chromium 10<b>×</b> and scRNAseq BD-Rhapsody; and for step 3B (cell population isolation), we have performed scRNA-seq Chromium 10<b>×</b>, scRNAseq BD-Rhapsody and SureCell®ATACseq Bio-Rad ddSEQ.</p>
<h3 id="sec1.2">Set up of equipment and reagents</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol>
<li>Please, refer to the <a href="#key-resources-table">key resources table</a> for the complete list of materials and reagents.</li>
<li>Prepare reagent stocks (see below).</li>
<li>Pre-heat the incubator and water bath at 37°C.</li>
<li>Set centrifuge at 4°C.</li>
<li>Prepare the tissue collection material (dissecting scissors, serrated forceps and tweezers) (<a href="#fig1">Figure 1</a>A)
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1033-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Set up of the equipment for tissue harvesting and chopping</div>
<p>(A) Dissection kit for tissue collection, including two scissors (left), tweezers (middle) and two forceps (right).</p>
<p>(B) Steps for assembling the McIlwain tissue chopper.</p>
</figcaption>
</figure>
</li>
<li>Prepare tissue chopper (<a href="#fig1">Figure 1</a>B):
<ol type="a">
<li>Fit plastic disc on cutting platform (<a href="#fig1">Figure 1</a>B-I).</li>
<li>Remove blade clamping plate and set a double-edged razor blade onto the blade holder (<a href="#fig1">Figure 1</a>B-II).</li>
<li>Adjust the blade on the plastic disc and ensure that the blade is not in contact with the disc, as pieces of the plastic disc can contaminate the tissue sample (<a href="#fig1">Figure 1</a>B-III).</li>
<li>Replace the clamp plate and tighten the nut with the spanner (<a href="#fig1">Figure 1</a>B-IV).</li>
</ol>
</li>
<li>Aliquot 4.5 mL of 5% FBS/ DMEM High glucose in a 15 mL-Falcon tube per sample being collected and digested.</li>
<li>Thaw collagenase-hyaluronidase (10<b>×</b>), TEG (1<b>×</b>) and DNase (1 mg/mL) and then keep on ice.</li>
</ol>
<h3 id="sec1.3">Preparation of reagent stocks</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h 30 min</div>
<ol start="9">
<li>Collagenase-hyaluronidase (10<b>×</b>). To prepare 50 mL:
<ol type="a">
<li>Dissolve 182.6 mg of Collagenase (Sigma C9891-1G) and 62.4 mg of Hyaluronidase (Sigma H3506-1G) with 50 mL of MEM, Hanks' Balanced Salts (Gibco™ 11575032)</li>
<li>Aliquot in 1.5 mL Eppendorf tubes and store at −20°C until use.</li>
</ol>
</li>
<li>DNase (1mg/mL):
<ol type="a">
<li>Dissolve 100 mg DNase (Roche 10104159001) in 100 mL of DPBS (Gibco 14190-250)</li>
<li>Aliquot in 1.5 mL Eppendorf tubes and store at −20°C until use.</li>
</ol>
</li>
<li>TEG (1<b>×</b>):
<ol type="a">
<li>Dilute 10 mL of 2.5% Trypsin (Gibco™ 15090-046) in 90 mL of DPBS to prepare the stock of 0.25% Trypsin.</li>
<li>Dissolve 40 mg of EGTA (Sigma E3889-25G) and 10 mg of Polyvinyl alcohol (Sigma P8136-250G) in 0.25% Trypsin (might require vortex).</li>
<li>Aliquot in 1.5 mL Eppendorf tubes and store at −20°C until use.</li>
</ol>
</li>
<li>Dispase II solution (prepare fresh):
<ol type="a">
<li>Resuspend 25 mg of Dispase II in 5 mL of DPBS, mix well for full resuspension. Keep the solution 20°C–22°C.</li>
</ol>
</li>
<li>Red Blood Cell (RBC) Lysis Buffer. For 500 mL:
<ol type="a">
<li>Dissolve 4.15 g of Ammonium Chloride (Sigma A9434-500G) in 50 mL of 0.1M Tris HCl and 450 mL of reverse osmosis (RO) water.</li>
<li>Adjust pH to 7.5 +/− 0.2.</li>
<li>Store at 4°C.</li>
</ol>
</li>
<li>DAPI (Invitrogen D1306 - 10 mg):
<ol type="a">
<li>Stock preparation: to make 5 mg/mL stock solution dissolve the entire vial in 2 mL of DPBS. Then:
<ol type="i">
<li>Prepare 40 aliquots of 50 ul of DAPI 5 mg/mL in 1.5 mL Eppendorf tubes.</li>
<li>Freeze at −20°C (for long term storage).</li>
</ol>
</li>
<li>Working stock: to one aliquot of 50 μL of DAPI (5 mg/mL):
<ol type="i">
<li>Add 450 μL of DPBS (dilution 1:10 – Final concentration = 500 μg/mL).</li>
<li>Prepare 25 aliquots of 20 μL of DAPI 500 μg/mL.</li>
<li>Storage at 4°C for 6 months.</li>
</ol>
</li>
<li>For flow cytometry: take one aliquot from the fridge (DAPI 500 μg/mL) and:
<ol type="i">
<li>Dilute in 180 μL of DPBS (dilution 1:10 – Final concentration= 50 μg/mL)</li>
<li>Storage at 4°C for 1 month.</li>
<li>Dilute DAPI 50 ug/mL 1:100 into your sample (e.g., for 300 ul sample, add 3 ul of DAPI 50 μg/mL) – Final concentration = 0.5 μg/mL.</li>
</ol>
</li>
</ol>
</li>
<li>5% FBS/ DMEM High Glucose:
<ol type="a">
<li>Dilute 25 mL of FBS in 475 mL of DMEM High Glucose (GIBCO).</li>
<li>Store at 4°C for 1 month.</li>
</ol>
</li>
<li>2% FBS/ DPBS:
<ol type="a">
<li>Dilute 10 mL of FBS in 490 mL of DPBS.</li>
<li>Store at 4°C for 1 month.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Mammary gland carcinoma tissue</td>
<td>Mouse models from Australian BioResources (MossVale, NSW, Australia)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>PE/Cyanine7 anti-mouse CD45 (Dilution 1:400)</td>
<td>BioLegend</td>
<td>Cat#103114; RRID: AB_312979</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Collagenase</td>
<td>Sigma Aldrich</td>
<td>Cat#C9891; CAS: 9001-12-1</td>
</tr>
<tr>
<td>Hyaluronidase</td>
<td>Sigma Aldrich</td>
<td>Cat#H3506; CAS: 37326-33-3</td>
</tr>
<tr>
<td>Ammonium chloride</td>
<td>Sigma Aldrich</td>
<td>Cat#A9434; CAS: 12125-02-9</td>
</tr>
<tr>
<td>DNase I</td>
<td>Roche</td>
<td>Cat#10104159001</td>
</tr>
<tr>
<td>2.5% Trypsin</td>
<td>Gibco</td>
<td>Cat#15090-046</td>
</tr>
<tr>
<td>Dispase II (neutral protease, grade II)</td>
<td>Roche</td>
<td>Cat#04942078001</td>
</tr>
<tr>
<td>Ethylene glycol-bis(2-aminoethylether)-N,N,N′,N′-tetraacetic acid (EGTA)</td>
<td>Sigma</td>
<td>Cat# E3889; CAS: 67-42-5</td>
</tr>
<tr>
<td>Polyvinyl alcohol</td>
<td>Sigma</td>
<td>Cat# P8136; CAS: 9002-89-5</td>
</tr>
<tr>
<td>DMEM High Glucose</td>
<td>Gibco</td>
<td>Cat# 11995-073</td>
</tr>
<tr>
<td>MEM, Hanks' Balanced Salts</td>
<td>Gibco</td>
<td>Cat# 12571063</td>
</tr>
<tr>
<td>Dulbecco's phosphate-buffered saline (DPBS)</td>
<td>Gibco</td>
<td>Cat# 14190-250</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>GE Healthcare</td>
<td>Cat# SH30406.02</td>
</tr>
<tr>
<td>4',6-Diamidino-2-Phenylindole, Dihydrochloride (DAPI)</td>
<td>Invitrogen</td>
<td>Cat# D1306; RRID: AB_2629482</td>
</tr>
<tr>
<td>Trypan Blue Solution, 0.4%</td>
<td>ThermoFisher Scientific</td>
<td>Cat# 15250061</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Dead Cell Removal Kit</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-090-101</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>4T1.2</td>
<td>Gifted from Prof. Robin Anderson (Olivia Newton-John Cancer Research Institute)</td>
<td>N/A</td>
</tr>
<tr>
<td>67NR</td>
<td>Gifted from Prof. Robin Anderson (Olivia Newton-John Cancer Research Institute)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Female BALB/c mice intraductally injected at 8–10 weeks old with the 4T1.2 or 67NR cell lines</td>
<td>Inbred strain from Australian BioResources (origin The Jackson Laboratory)</td>
<td>Cat# JAX:000651; RRID:IMSR_JAX:000651</td>
</tr>
<tr>
<td>Mammary tumor virus-Polyoma Middle T (MMTV-PyMT) model of luminal breast cancer (females aged between 12–14 weeks old)</td>
<td>Inbred strain from Australian BioResources (origin The Jackson Laboratory)</td>
<td>Cat# JAX:002374; RRID:IMSR_JAX:002374</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo v10.6.2</td>
<td>Becton Dickinson</td>
<td><a href="https://flowjo.com">https://flowjo.com</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>BD FACSAria™ III sorter</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>McIlwain Tissue chopper</td>
<td>ProSciTech</td>
<td>Cat# U0800</td>
</tr>
<tr>
<td>Incu-Shaker™ 10 L Shaking Incubator</td>
<td>Benchmark Scientific</td>
<td>Part#: 6.284 19</td>
</tr>
<tr>
<td>Hemocytometer</td>
<td>Sigma-Aldrich</td>
<td>Cat# Z359629</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th>Collagenase-hyaluronidase 10×</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Collagenase</td>
<td>150,000 U</td>
<td>182.6 mg</td>
</tr>
<tr>
<td>Hyaluronidase</td>
<td>50,000 U</td>
<td>62.4 mg</td>
</tr>
<tr>
<td>MEM, Hanks' Balanced Salts</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for up to 12 months</p>
<table id="undtbl2">
<thead>
<tr>
<th>DNase solution</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNase</td>
<td>1 mg/mL</td>
<td>100 mg</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for up to 12 months</p>
<table id="undtbl3">
<thead>
<tr>
<th>Dispase II solution</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dispase II (neutral protease, grade II)</td>
<td>5 mg/mL</td>
<td>25 mg</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">This solution is freshly prepared and kept at 20°C–22°C</p>
<table id="undtbl4">
<thead>
<tr>
<th>TEG 10×</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Trypsin 2.5%</td>
<td>0.25% (1<b>×</b>)</td>
<td>10 mL</td>
</tr>
<tr>
<td>EGTA</td>
<td>1 mM</td>
<td>40 mg</td>
</tr>
<tr>
<td>Polyvinyl alcohol</td>
<td>n/a</td>
<td>10 mg</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>90 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for up to 12 months</p>
<table id="undtbl5">
<thead>
<tr>
<th>Red blood cell lysis buffer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ammonium Chloride</td>
<td>52.7 mM</td>
<td>4.15 g</td>
</tr>
<tr>
<td>0.1M Tris HCl</td>
<td>10 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>RO water</td>
<td>n/a</td>
<td>450 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 12 months</p>
<table id="undtbl6">
<thead>
<tr>
<th>DAPI</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DAPI</td>
<td>5 mg/mL</td>
<td>10 mg</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>2 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for long term storage</p>
<table id="undtbl7">
<thead>
<tr>
<th>5% FBS/ DMEM</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>5%</td>
<td>25 mL</td>
</tr>
<tr>
<td>DMEM High Glucose</td>
<td>n/a</td>
<td>475 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 2 months</p>
<table id="undtbl8">
<thead>
<tr>
<th>2% FBS/ DPBS</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>2%</td>
<td>10 mL</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>490 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 2 months</p>
<table id="undtbl9">
<thead>
<tr>
<th>10× Annexin V binding buffer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Hepes/NaOH (pH 7.4)</td>
<td>0.1 M</td>
<td>50 mL</td>
</tr>
<tr>
<td>5M NaCl</td>
<td>1.4 M</td>
<td>140 mL</td>
</tr>
<tr>
<td>1M CaCl<sub>2</sub>
</td>
<td>25 mM</td>
<td>12.5 mL</td>
</tr>
<tr>
<td>RO Water</td>
<td>n/a</td>
<td>297.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 2 months</p>
<div class="critical">
<span class="critical-title">Critical:</span> All reagents containing fetal bovine serum and/or culture medium should be prepared under sterilized conditions in a laminar flow cabinet.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> While this protocol uses the McIIwain Tissue chopper for cutting the tissue, this can be alternatively done manually using a scalpel blade. Additionally, any shaking incubator with temperature and speed control can be used as a substitute of the Incu-Shaker™ 10 L Shaking Incubator (Benchmark Scientific).</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Timings of this protocol are adjusted for one tumor from one mouse. By adding more samples, the timing will increase accordingly.</p>
<div class="note">
<span class="note-title">Note:</span> The maximum number of samples to handle at the same time is 8 for an experienced researcher to avoid compromising the quality and viability of the cells.</div>
<div class="note">
<span class="note-title">Note:</span> The time from tissue harvesting to single-cell capture should be the shortest possible, ideally 3.5–5.5 hours (depending on the abundance of your targeted cell population). It is not recommended long sample processing. In our hands, we have obtained good quality preparations from rare cell populations that require up to 12 h for alive single cell preparations.</div>
<h3 id="sec3.1">Tissue harvesting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<div class="timing">
<span class="timing-title">Timing:</span> for 8 samples: 1 h</div>
<p>In this section we explain the procedure for tumor harvesting. The most critical aspects are to avoid the areas containing necrosis and lymph nodes. This will ensure that we produce cell suspensions with a high content of viable cells and that the sample represents tumor heterogeneity.</p>
<p>Although sterilized conditions are not necessary, tissue collection should be performed in a clean area using clean scissors and forceps. To clean surfaces and instruments wipe with a 70% ethanol solution</p>
<ol>
<li>Euthanize mice with CO<sub>2</sub>, followed by cervical dislocation (or according to the specific ethics guidelines), and harvest tissue by collecting only the regions with high content of viable tumor cells from epithelial mammary carcinomas, trying to exclude the necrotic areas and lymph nodes (<a href="#fig2">Figure 2</a>). As these areas are very challenging to identify, we perform a QC step to be able to select the highest quality samples to improve the sorting timing and to eliminate bias due to necrosis or lymphocyte contamination. Based on our experience, here we provide some suggestions to avoid collecting those specific regions.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1033-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Selection of regions with high content of viable tumor cells during tissue harvesting</div>
<p>(A–C) Scatter plots that show an enrichment (red arrows) in lymphocytes that correspond to a lymph node (A), necrotic areas (B), and viable tumor tissue with high cellular content (C)</p>
</figcaption>
</figure>
<ol type="a">
<li>Necrotic areas can be identified by looking at the texture, consistence and location of the tumor tissue. Viable tissue is compact and hard, located in the edges of the tumor mass. Necrotic areas are softer and with “mash-potato”-like consistency, usually located in the inner of the tumor mass.</li>
<li>Lymph node’s location can be expected under the nipple, where most of the blood vessels converge within the tumor.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> As necrosis and lymph nodes can be difficult to identify, and they can misrepresent tumour heterogeneity (<a href="#sec6.3">problem 2</a>), we suggest to collect two separated samples from the same tumour or tissue (sample “A” and “B”). Then, in the QC of samples by flow cytometry step, the samples containing highly necrotic areas or lymph nodes can be discarded. This will avoid slowing down the sorting process and will increase the quality of the sample.</div>
<ol start="2">
<li>Manually dissect the tumor tissue into 3–5 mm pieces using a surgical scalpel blade.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The size range of the initial sample may vary depending on the tumour type and timepoint of harvesting. We recommend harvesting tumours of sizes between 500 – 2,000 mm3, or according to the specific Animal Ethics specifications.</div>
<ol start="3">
<li>Chop to 100 μm with maximum blade force on a McIlwain Tissue chopper for 2–3 passes.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure the cutting disc is not overloaded with more than 50% of the chopping surface covered by the tissue.</div>
<ol start="4">
<li>Place the chopped tissue in a 15 mL-Falcon tube containing 4.5 mL of 5% FBS/ DMEM High glucose.</li>
</ol>
<h3 id="sec3.2">Tissue digestion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<div class="timing">
<span class="timing-title">Timing:</span> for 8 samples: 1.5 h</div>
<p>This section describes the enzymatic digestion of the tumor samples to obtain a single cell suspension.</p>
<div class="note">
<span class="note-title">Note:</span> All the resuspension steps should be done by pipetting at least 10 times.</div>
<p>For tissue digestion, samples are incubated in the following order:</p>
<ol start="5">
<li>Add 500 μL of collagenase-hyaluronidase (10<b>×</b>) to the 15 mL-Falcon tube containing the sample and 4.5 mL of 5% FBS/ DMEM High glucose.</li>
<li>Incubate with 1<b>×</b> of collagenase-hyaluronidase at 37°C shaking at 200 rpm for 30 min in the Incu-Shaker™ 10 L Shaking Incubator.
<ol type="a">
<li>At 15 min of the incubation process, to ensure the tissue is sufficiently resuspended, disrupt the tissue with by pipetting up and down with a p1000 set at 1 mL for at least 10 times.</li>
<li>Although macroscopic tumor chunks can be still appreciated, a cloudy non-translucent medium indicates proper digestion of the tissue.</li>
</ol>
</li>
<li>Wash the sample by adding 10 mL of cold DPBS to neutralize the digestion.</li>
<li>Centrifuge at 200 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Decant the content of the tube to discard the supernatant.</li>
<li>Resuspend the pellet 100 μL of DNase solution.</li>
<li>Add 750 μL of TEG and pipette up and down for 1 min at 37°C in the water bath.</li>
<li>Fill up to 15 mL with 2% FBS/ DPBS.</li>
<li>Centrifuge at 200 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Decant the content of the tube to discard the supernatant.</li>
<li>Add 1.25 mL of Dispase solution to each pellet and incubate for 5 min in 37°C in the waterbath.</li>
<li>Resuspend the pellet in 50 μL of DNase solution and 450 mL of 2% FBS/ DPBS.</li>
<li>Add 2 mL of RBC lysis buffer, resuspend and incubate at 37°C in the water bath for 2 min.</li>
<li>Fill up to 15 mL with 2% FBS/ DPBS to neutralize.</li>
<li>Centrifuge at 200 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Decant the content of the tube to discard the supernatant</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After this point, samples should be kept on ice at all times.</div>
<ol start="21">
<li>Resuspend pellet in 5–10 mL of cold 2% FBS/ DPBS and filter into 50 mL-Falcon tubes using a 40 μm cell sterile strainer.</li>
<li>Count cells as explained in the cell count and viability step and prepare a single-cell suspension of 1 <b>×</b> 10<sup>6</sup>–1 <b>×</b> 10<sup>7</sup> cells/ mL.</li>
<li>Take between 50–150 K cells for the QC of samples by flow cytometry. The remaining cells are ready for options A or B.</li>
</ol>
<h3 id="sec3.3">Quality check (QC) of cell viability and cellular content of samples by flow cytometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<div class="timing">
<span class="timing-title">Timing:</span> for 8 samples: 45 min</div>
<p>During this step, cell viability and cellular content are analyzed and samples containing necrosis and/or lymph nodes are discarded.</p>
<p>To the single-cell suspension (50–150 K cells) from step 23 (in the <a href="#sec3.2">tissue digestion</a> section), do the QC as follows:</p>
<ol start="24">
<li>Fill up to 300 μL with 2% FBS/ DPBS.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> 3 μL of 50 μg/mL DAPI can be added before running the sample in the flow cytometer to verify viability after checking FSC and SSC.</div>
<ol start="25">
<li>Set a flow cytometer instrument.</li>
<li>Place the sample in the flow cytometer and acquire and/or record ∼100,000 events.</li>
<li>Categorize the cell preparation based on the FSC and SSC profile (<a href="#fig2">Figure 2</a>), identifying the presence of lymph nodes in the harvested tissue (<a href="#fig2">Figure 2</a>A, red arrow) or potential necrotic areas (<a href="#fig2">Figure 2</a>B, red arrow), as opposed to a typical tumor cell profile (<a href="#fig2">Figure 2</a>C, red arrow).</li>
<li>If all samples show a high percentage of viable cells from the tumor tissue with absence of necrosis and/or lymph nodes, samples “A” and “B” can be pooled (see step 1 in the <a href="#sec3.1">tissue harvesting</a> section).</li>
<li>Proceed with option A or B, depending on the type of single-cell experiment, Option A will be unbiased and Option B will be targeted to a specific cell population within the tumor.</li>
</ol>
<h3 id="sec3.4">Option A. Live population separation using autoMACS technology for unbiased single-cell downstream analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<div class="timing">
<span class="timing-title">Timing:</span> for 8 samples: 2 h</div>
<p>During autoMACS cleaning, the sample is enriched for live cells via Annexin-V antibody-based depletion. High viable cell suspensions are obtained using the Dead Cell Removal Kit (Miltenyi Biotec 130-090-101) and the autoMACS® Pro separator (Miltenyi Biotec).</p>
<ol start="30">
<li>Transfer highly viable tumor samples from step 22 (in the <a href="#sec3.2">tissue digestion</a> section) into a 5 mL-FACS tube.</li>
<li>Centrifuge at 200 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Resuspend the pellet (10<sup>7</sup> or fewer cells) in 100 μL of the Dead Cell Removal microbeads from the Dead Cell Removal Kit.</li>
<li>Incubate for 15 min at 20°C–22°C in the dark.</li>
<li>During the 15 min-incubation time:
<ol type="a">
<li>Prepare fresh 1<b>×</b> of Annexin V Binding buffer by diluting 100 mL of 10<b>×</b> Annexin V Binding Buffer in 900 mL of RO Water.</li>
<li>Label two extra 5 mL-FACS tubes as negative and positive fraction. These are the collection tubes.</li>
<li>Switch on and set up the autoMACS® Pro separator as indicated in the manual.</li>
<li>Equilibrate the column on the autoMACS Pro Separator with 1<b>×</b> of Annexin V Binding buffer and keep this buffer for the required washes.</li>
</ol>
</li>
<li>After 15 min of incubation (step 33), dilute the sample up to 800 μL by adding 700 μL of 1<b>×</b> Annexin V Binding buffer and place sample and the collection tubes in the 5 mL-rack from autoMACS® as explained in the manual.</li>
<li>Separate the live population (negative fraction) from the dead cells by using the DepleteS program in the autoMACS® Pro separator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During the autoMACS® separation is expected a ∼40%–50% cell loss.</div>
<ol start="37">
<li>Clean up and shutdown autoMACS® Pro separator following manufactures’ instructions.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Check the live fraction separated by autoMACS under flow cytometry using DAPI (0.5 μg/mL final concentration) to corroborate the high viability of the cell suspension (<a href="#fig3">Figure 3</a>)</div>
<ol start="38">
<li>The high viable cell fraction is ready for counting.</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1033-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. AutoMACS clean-up for the enrichment of alive cells</div>
<p>Bar plot showing the percentage of live cells before and after autoMACS clean up following tumor harvesting and single cell digestion of mouse primary mammary tumors (N= 8). Data represented as mean ± SD. ∗∗∗∗ p value &lt; 0.0001.</p>
</figcaption>
</figure>
<h3 id="sec3.5">Option B. Sorting of the population of interest using fluorescence-activated cell sorting (FACS) for targeted single-cell downstream analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> varies depending on the sorted population.</div>
<div class="timing">
<span class="timing-title">Timing:</span> for 100 K tumour infiltrated immune (CD45+) cells, 200 K cancer cells (mCherry+) and 10 K of stromal cells (CD45-, mCherry-): 3–3.5 h</div>
<p>During FACS sorting, the population of interest is selected by using antibodies against specific cell surface markers. As an example, in this protocol we focus on the sorting of immune cells (CD45+), tumor cells (expressing mCherry+) and stromal cells (defined as double negative population) from murine mammary tumors.</p>
<div class="note">
<span class="note-title">Note:</span> All the steps from this section should be performed at 4°C.</div>
<ol start="39">
<li>Transfer 2.5–5 <b>×</b> 10<sup>6</sup> cells from highly viable tumor samples from step 22 (in the <a href="#sec3.2">tissue digestion</a> section) into a 5 mL-FACS tube.</li>
<li>Centrifuge at 200 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Resuspend the pellet in a final volume of 500 μL of a solution containing 2% FBS/DPBS and anti-mouse CD45 antibody with the desired conjugated fluorophore at the recommended dilution. In the example shown here we used PE/Cyanine7 anti-mouse CD45 (Biolegend Cat#103114) Dilution 1:400.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure the fluorophore chosen doesn’t overlap with DAPI spectrum.</div>
<div class="note">
<span class="note-title">Note:</span> A flow cytometry antibody titration may be required to optimize the staining and to fix a dilution factor for that specific fluorophore-conjugated antibody.</div>
<ol start="42">
<li>Incubate on ice for 30 min.</li>
<li>During the 30 min-incubation time, prepare the collection tube by adding 100 μL of FBS in one 1.5 mL Eppendorf tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure the FBS cover all the wall of the tube to avoid cell dead when sorting as the sorted cells can be deposited in the walls instead on the bottom of the tube during the procedure.</div>
<ol start="44">
<li>After the 30-min incubation (step 42), wash the sample by adding 1 mL of 2% FBS/DPBS and centrifuge at 200 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Decant the contents of the tube to discard the supernatant.</li>
<li>Resuspend the pellet in 1 mL of 2% FBS/DPBS.</li>
<li>Keep the sample on ice and in the dark until FACS sorting.</li>
<li>Set up the nozzle size at 100 μm on the BD FACSAria™ III sorter.</li>
<li>Set up the gating strategy by selecting the cellular compartment in the FSC-A and SSC-A dot plot. Exclude doublets using FSC-A and FSC-H, select the negative fraction for DAPI and gate the positive cells for CD45 and mCherry. See <a href="#fig4">Figure 4</a> as an example of the gating strategy with the PE/Cyanine7 anti-mouse CD45 Antibody (Ex: 561 Em: 780) and mCherry (Ex: 561 Em: 610).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1033-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Example of gating strategy for cancer, stromal, and immune alive cells by FACS sorting</div>
<p>Immune cells are labeled with the CD45 -PE-Cy7 fluorophore and cancer cells are differentiated from stromal cells by the expression of the mCherry.</p>
</figcaption>
</figure>
</li>
<li>Add 10 μL of 50 μg/mL DAPI and incubate for 3–5 min.</li>
<li>Insert the DAPI-stained sample in the sample injection chamber of the BD FACSAria™ III sorter.</li>
<li>Adjust voltage for FSC, SSC and the lasers for the fluorophores used.</li>
<li>Insert the collection tube on the sort collection chamber.</li>
<li>Set up the desired number of events to collect.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As the number of cells returned from the sorter depends on different factors, including sort precision and efficiency, we usually assume that a 50% loss of the targeted cell population will be the “worse-case scenario.” Therefore, we recommend set up the desire number of events to 200 K when aiming 100 K cells.</div>
<ol start="55">
<li>Proceed with the sorting of CD45+ DAPI- (live immune cells), mCherry+ DAPI- (live cancer cells) and CD45-, mCherry- and DAPI- (live stromal cells) (<a href="#fig4">Figure 4</a>, right panel).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In mammary gland tumours, we estimate that ∼12%–25% of the total number of events corresponds to CD45+cells, ∼60–80% to cancer cells and ∼2%–8% to stromal cells. This translates to ∼ 1–2 h of sorting needed of immune cells, ∼20 min of cancer cells and &gt;2 h of stromal cells for 200 K cells of each type.</div>
<ol start="56">
<li>After sorting, proceed to count the total number of cells obtained and prepare the recommended cell concentration for your specific single-cell application.</li>
</ol>
<h3 id="sec3.6">Cell counting and viability measurements</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<div class="timing">
<span class="timing-title">Timing:</span> for 8 samples: 1 h</div>
<p>In this section we explain how to perform cell counting of viable cells using 0.4% Trypan blue solution and a hemocytometer.</p>
<div class="note">
<span class="note-title">Note:</span> For cell counting, a 0.4% Trypan blue solution is added to the sample to distinguish live from dead cells. Live cells will be bright under the microscope, while dead cells will be stained with the dye acquiring the blue color and losing the brightness. When preparing single cell suspension, the concentration should be based just of the live fraction of the sample.</div>
<ol start="57">
<li>Place a glass cover on the central area of a hemocytometer.</li>
<li>Adjust the cell concentration of your sample in the recommended range (2.5 <b>×</b> 10<sup>6</sup>–2 <b>×</b> 10<sup>5</sup> cells/ mL) by:
<ol type="a">
<li>Diluting with 2% FBS in PBS Buffer if it is too concentrated (if you estimate that you have more than 2.5 <b>×</b> 10<sup>6</sup> cells/ mL)</li>
<li>Centrifuging at 200 <b>×</b> <i>g</i> for 5 min at 4°C and resuspending in less volume of the used buffer if it is too diluted (if you estimate that you have less than 200,000 cells/ mL)</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The concentration can be estimated based on the turbidity of the sample and also based on the expected number of cells from step 36 from option A or step 55 from Option B. A very concentrated sample will look opaque, while a very diluted sample will look transparent. A light translucent sample should work for counting the cells with the hemocytometer.</div>
<ol start="59">
<li>Resuspend the sample by pipetting up and down 10 times and take 5 μL with a p20 pipette and mix with 5 μL of Trypan blue on a new tube.</li>
<li>Place the 10 μL of the mix in a hemocytometer, by positioning the loaded p20 pipette on the glass of the hemocytometer, close to the glass cover edge and releasing the sample carefully.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Count the cells right after adding the trypan blue. Incubating cells in trypan blue for too long can give false blue positive cells.</div>
<ol start="61">
<li>Place the loaded hemocytometer in the microscope, turn the light on and focus until you can see the grid squares.</li>
<li>Count the bright cells within the 16 squares of the 4 sets of big squares (A1, A2, A3, A4) (<a href="#fig5">Figure 5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1033-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Cell counting strategy of the final cell suspension</div>
<p>(A) Schematic representation of hemocytometer gridlines, showing 4 square areas for counting (A1–A4) and our approach to count each of the them (arrows).</p>
<p>(B) Zoom in of one of the grids containing 16 squares. The blue lines indicate the borders to be considered for counting the cells that sit on these borderlines. Cells in yellow indicate the cells excluded from the counting grid and blue cells represent dead cells labeled with trypan blue.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We generally obtain ∼90% of live cells, counted by trypan blue exclusion.</div>
<ol start="63">
<li>To calculate the concentration of the cells in your sample (cells/ mL) follow the formula:</li>
</ol>
<p class="math-ml"><math> <mrow> <mtext>Concentration </mtext> <mrow> <mo>(</mo> <mfrac> <mtext>cells</mtext> <mtext>mL</mtext> </mfrac> <mo>)</mo> </mrow> <mo>=</mo> <mrow> <mo>(</mo> <mfrac> <mrow> <mtext>A</mtext> <mn>1</mn> <mo>+</mo> <mtext>A</mtext> <mn>2</mn> <mo>+</mo> <mtext>A</mtext> <mn>3</mn> <mo>+</mo> <mtext>A</mtext> <mn>4</mn> </mrow> <mn>4</mn> </mfrac> <mo>)</mo> </mrow> <mo>×</mo> <mn>10,000</mn> <mfrac> <mtext>cells</mtext> <mtext>ml</mtext> </mfrac> <mo>×</mo> <mn>2</mn> </mrow> </math></p>
<div class="note">
<span class="note-title">Note:</span> A1-A4: represent the number of cells counted in each of the 4 sets of 16 squares, that is expected to be similar among them if there is a good resuspension of the sample. 4 is the number of sets that have been counted. 10,000 cells/mL is the equivalent of cell concentration of the surface of each set counted. 2 is the dilution factors used with trypan blue dye</div>
<ol start="64">
<li>Calculate the number of the total cells in your sample following the formula:</li>
</ol>
<p class="math-ml"><math> <mrow> <mtext>Total number of cells</mtext> <mo>=</mo> <mtext> cell concentration</mtext> <mrow> <mo>(</mo> <mfrac> <mtext>cells</mtext> <mtext>mL</mtext> </mfrac> <mo>)</mo> </mrow> <mo>×</mo> <mtext> Sample volume </mtext> <mrow> <mo>(</mo> <mtext>mL</mtext> <mo>)</mo> </mrow> </mrow> </math></p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Upon harvesting the regions with high cellular content and excluding the highly necrotic areas, the expected number of cells obtained from a 1,500 mm<sup>3</sup> tumor after dissociation is between 2.5-10 M. This number may vary depending on the tumor type and its cellularity.</p>
<p>The cellular fraction of the single cell suspension after digestion can be identified by flow cytometry (FSC vs SSC), and typically accounts for ∼60% of the total number of events (<a href="#fig4">Figure 4</a>, left panel).</p>
<p>For Option A, the percentage of viable cells in the samples is expected to reach &gt;90% after autoMACS clean up, (<a href="#fig3">Figure 3</a>), but up to 40–50% of cells can be lost during this procedure. This means we will obtain ∼1 and 5 M highly viable cell suspensions from a 1,500 mm<sup>3</sup> of mammary gland tumor for an unbiased single cell omic application. Alternatively, cell sorting of the viable (e.g., DAPI negative) population can be also performed. This is recommended for samples with limited material as it delivers less cell wastage but can be more time consuming than the autoMACS procedure.</p>
<p>For Option B, the percentage of viability in the sample will exceed 90%. The proportion of cells for a specific population can vary significantly among biological replicates due to tumor heterogeneity. Thus, the percentages given in this protocol for the CD45+ population may change between different mammary tumors</p>
<p>Typically, the mitochondrial-to-nuclear RNA ratio in single-cell RNAseq experiments is used as a quality metric. Using this protocol, we normally find 2–15% mitochondrial content depending on the cell type and the platform used for scRNAseq (<a href="#bib2">Valdes-Mora et al., 2021</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Although this protocol describes sorting of cell types from the three major tumor compartments, cancer cells, immune cells and stromal cells, different projects might aim for other type of cells including rare cell populations. The timing required for the sorting in this situation will exceed the standard times used in this protocol. If possible, it can also be considered to increase the number of tumors and sorting instruments to obtain the desired number of cells for single-cell capture in a reasonable timeframe in order to preserve quality. In this case, we recommend characterizing the cell content of your sample by FACS prior the single cell RNAseq experiment, so the number of biological replicates needed can be calculated in advance. Additionally, it is recommended to calculate the timing for the sorting of each specific population to make sure there is no more than 12 h from tissue harvesting to single-cell capture.</p>
<p>Another potential limitation is the application of this protocol to other tissue types. We have tested this protocol in mouse lungs, intestines and liver. However, for different tissues and human samples, and optimization of this protocol may be required.</p>
<p>When choosing the dead-cell clean up method it is important to check if specific single-cell applications have limitations for cell preparation. For example, we do not recommend using cell sorting to select the live population of a tumor for Drop-seq (<a href="#bib1">Salomon et al., 2019</a>; <a href="#bib2">Valdes-Mora et al., 2021</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Highly necrotic tumors at tissue harvesting (step 1).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Consider increasing the number of biological replicates to obtain more tissue and reach the minimum number of cells in the single-cell suspension for sorting the population of interest.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Necrotic tissue areas and lymph nodes may confound tumor heterogeneity (step 1).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>To avoid bias and increase the likelihood of representing the full tumor heterogeneity, it is recommended harvesting the whole tumor tissue. However, this can increase the risk of necrotic tissue and/or lymph node contamination. To avoid losing all the tumor sample due to this contamination and to maintain tumor heterogeneity, we split the tumor in two samples (“A” and “B”) during tissue harvesting and then, after the QC (step 28), combine them as long as there is no necrotic of lymph node contamination.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Clogging of the 40 μm cell strainer when filtering cells during the digestion (step 21).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>If the cell suspension is highly concentrated, the 40 μm cell strainer may be obstructed during filtration in the step 21 of the <a href="#sec3.2">tissue digestion</a> section. To avoid the obstruction, we recommend diluting the sample with 2% FBS/DPBS before filtration.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Cell stress during FACS sorting (step 55).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Reduce the event rate on the BD FACSAria™ III sorter by diluting the sample or by decreasing the speed of the sorting.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Cell clustering when counting (step 62).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>During cell counting with trypan blue 0.4%, cell clusters of 2 or more cells can lead to bias in the final calculated concentration. To minimize the proportion of cell clusters, make sure the sample is well-resuspended before loading it into the hemocytometer and the cell concentration is not too high. If necessary, the sample can be filtrated through a 40 μm cell strainer right before counting.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Salomon, R., Kaczorowski, D., Valdes-Mora, F., Nordon, R.E., Neild, A., Farbehi, N., Bartonicek, N., and Gallego-Ortega, D. (2019). Droplet-based single cell RNAseq tools: a practical guide. Lab. Chip <i>19</i>, 1706-1727. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00547-5/sref1">View at publisher</a></p>
<p id="bib2">Valdes-Mora, F., Salomon, R., Gloss, B.S., Law, A.M.K., Venhuizen, J., Castillo, L., Murphy, K.J., Magenau, A., Papanicolaou, M., Rodriguez de la Fuente, L., et al. (2021). Single-cell transcriptomics reveals involution mimicry during the specification of the basal breast cancer subtype. Cell Rep. <i>35</i>, 108945. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00547-5/sref2">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Fatima Valdes-Mora (<a href="mailto:FValdesMora@ccia.org.au">FValdesMora@ccia.org.au</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique data sets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants from <a href="https://doi.org/10.13039/501100001171">Cancer Institute New South Wales</a> (2019/CDF002) from F.V.-M. and (DG00625) to D.G.-O., <a href="https://doi.org/10.13039/501100001102">Cancer Council NSW</a> project grant (RG18-03) and <a href="https://doi.org/10.13039/501100001026">National Breast Cancer Foundation</a> Elaine Henry Fellowship (IIRS-21-096) to D.G.-O. L.R.F. holds a UIPA – University International Postgraduate Award from the <a href="https://doi.org/10.13039/100012481">University of New South Wales</a> (UNSW). The graphical abstract illustration and <a href="#fig2">Figure 2</a> were created with <a href="https://biorender.com">Biorender.com</a>. We thank the members of the Tumor Development Laboratory at Garvan Institute of Medical Research and the Cancer Epigenetics Biology and Therapeutics group at Children’s Cancer Institute for helpful discussions.</p>
<h3>Author contributions</h3>
<p>Writing – original draft preparation, protocol design and editing by L.R.d.l.F., A.M.K.L., F.V.-M., and D.G.-O.; figures designed by F.V.M. and L.R.F.; conceptualization and supervision by F.V.-M. and D.G.-O. All authors have read and agreed to the published version of the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>