<article>
<section>
<h1 id="header">Determining the effects of loss of function mutations in human cell lines</h1>
<p><time datetime="2022-03-18">Published: March 18, 2022</time></p>
<p>Nicola de Prisco,<sup><a href="#aff1">1</a>,<a href="#fn1">7</a></sup> Salvatore Botta,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Winston Lee,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a></sup> Sarallah Rezazadeh,<sup><a href="#aff1">1</a></sup> Alexei Chemiakine,<sup><a href="#aff1">1</a></sup> and Vincenzo A. Gennarino<sup><a href="#aff1">1</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#fn2">8</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Genetics and Development, Columbia University Irving Medical Center, New York, NY 10032, USA</p>
<p id="aff2"><sup>2</sup>Department of Translational Medical Science, University of Campania Luigi Vanvitelli, Caserta, 81100, Italy</p>
<p id="aff3"><sup>3</sup>Department of Ophthalmology, Columbia University Irving Medical Center, New York, NY 10032, USA</p>
<p id="aff4"><sup>4</sup>Departments of Pediatrics and Neurology, Columbia University Irving Medical Center, New York, NY 10032, USA</p>
<p id="aff5"><sup>5</sup>Columbia Stem Cell Initiative, Columbia University Irving Medical Center, New York, NY 10032, USA</p>
<p id="aff6"><sup>6</sup>Initiative for Columbia Ataxia and Tremor, Columbia University Irving Medical Center, New York, NY 10032, USA</p>
<p id="fn1"><sup>7</sup>Technical contact</p>
<p id="fn2"><sup>8</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:vag2138@cumc.columbia.edu">vag2138@cumc.columbia.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101232">10.1016/j.xpro.2022.101232</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Quantifying differences in the amount of protein and mRNA caused by missense mutations in a gene of interest can be challenging, especially when using patient-derived primary cells, which are intrinsically variable. In this protocol, we describe how to culture patient-derived lymphoblast and fibroblast cell lines for later mRNA and protein quantification. We also describe the steps to examine variants of PUM1 in HEK293T cells, but the protocol can be applied to other proteins of interest.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00112-5/sref1">Gennarino et al. (2018)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Studying gene variants in patient-derived cell lines</li>
<li>Culturing primary fibroblasts after skin biopsy</li>
<li>Culturing lymphoblastoid cell lines</li>
<li>Transfection of mutant proteins in HEK293T cells</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1503-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>We begin with the assumption that the gene of interest has been found to produce a phenotype in the model organism, which in our case was a mouse. To then identify human patients and establish the pathogenicity of genetic variants will require a more comprehensive analysis and will take some time.</p>
<div class="critical">
<span class="critical-title">Critical:</span> When working with DNA and RNA derived from patient cells, you must strenuously avoid contamination from your environment—else you risk unreliable results and the loss of months of work, labor, and materials. Wear gloves, clean your bench with a 70% ethanol solution, use RNaseZap or other commercially available cleaning solutions when extracting RNA; use nuclease-free disposable plasticware and filtered tips. Refer to ‘‘<a href="#materials-and-equipment">materials and equipment</a>’’ to prepare solutions, materials, and tools prior to the procedure.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER (CAT#)</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Expression vectors</td>
</tr>
<tr>
<td>
<i>PUM1</i>cDNA expression vector</td>
<td>OmickLink Expression Vector, GeneCopoeia</td>
<td>pEZ-M02, EX-E2337-M02-10</td>
</tr>
<tr>
<td>pEGFP-C1</td>
<td>Golenbock Lab</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Empty expression vector</td>
<td>OmickLink Expression Vector, GeneCopoeia</td>
<td>EX-NEG-M02</td>
</tr>
<tr>
<td>OmickLink<sup>TM</sup>Expression Vector pEZ-M02</td>
<td>GeneCopoeia</td>
<td>EX-E2337-M02-10</td>
</tr>
<tr>
<td>OmickLink<sup>TM</sup>Expression Vector</td>
<td>GeneCopoeia</td>
<td>EX-NEG-M02</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays/kits and instruments</td>
</tr>
<tr>
<td>jetPRIME Transfection Reagent</td>
<td>Polyplus-transfection</td>
<td>114-07</td>
</tr>
<tr>
<td>QuikChange Lightning Multi Site- Directed Mutagenesis Kit</td>
<td>Agilent Technologies</td>
<td>210515</td>
</tr>
<tr>
<td>miRNeasy Mini kit</td>
<td>QIAGEN</td>
<td>217004</td>
</tr>
<tr>
<td>QuantiTect Reverse Transcription kit</td>
<td>QIAGEN</td>
<td>205311</td>
</tr>
<tr>
<td>PowerUP SYBR Green Master Mix</td>
<td>Thermo Fisher Scientific</td>
<td>A25743</td>
</tr>
<tr>
<td>NanoDrop 1000 V3.8</td>
<td>Thermo Fisher Scientific</td>
<td>ND-ONE-W</td>
</tr>
<tr>
<td>CFX96 Touch Real-Time PCR Detection System</td>
<td>Bio-Rad Laboratories</td>
<td>Model 855196</td>
</tr>
<tr>
<td>Nitrocellulose Blotting membrane 0.45 μm</td>
<td>Amersham Protran</td>
<td>10600002</td>
</tr>
<tr>
<td colspan="3">Tissue culture</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle’s Medium (DMEM) – high glucose (4.5 g/L D-glucose)</td>
<td>Thermo Fisher Scientific</td>
<td>11965084</td>
</tr>
<tr>
<td>RPMI 1640 medium</td>
<td>Thermo Fisher Scientific</td>
<td>11875093</td>
</tr>
<tr>
<td>L-glutamine 200 mM (100×)</td>
<td>GenDepot</td>
<td>CA009-010</td>
</tr>
<tr>
<td>Normocin</td>
<td>InvivoGen</td>
<td>ant-nr-2</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>21103049</td>
</tr>
<tr>
<td>Heat Inactivated-Fetal Bovine Serum (FBS)</td>
<td>Novus Biologicals</td>
<td>S11150H</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>MilliporeSigma</td>
<td>T8787</td>
</tr>
<tr>
<td>NuPAGE 4%–12% Bis-Tris Gel</td>
<td>Thermo Fisher Scientific</td>
<td>NP0321PK2</td>
</tr>
<tr>
<td>Pierce BCA Protein Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>23225</td>
</tr>
<tr>
<td>Trypsin-EDTA 1×</td>
<td>MilliporeSigma</td>
<td>59417C</td>
</tr>
<tr>
<td>Flat Bottom Cell Culture Plates (6-Well)</td>
<td>Corning</td>
<td>07-200-83</td>
</tr>
<tr>
<td>Tissue-culture treated dishes 10 cm</td>
<td>Corning</td>
<td>CLS430167</td>
</tr>
<tr>
<td>Cell Culture Flask 25 cm<sup>2</sup> vent cap</td>
<td>Alkali scientific</td>
<td>TVN0025</td>
</tr>
<tr>
<td colspan="3">Chemicals and reagents</td>
</tr>
<tr>
<td>Formamide</td>
<td>MilliporSigma</td>
<td>FX0420-6</td>
</tr>
<tr>
<td>Tris-HCl (pH 7.5, 1 M)</td>
<td>MilliporeSigma</td>
<td>15567027</td>
</tr>
<tr>
<td>NP-40</td>
<td>Thermo Fisher Scientific</td>
<td>85124</td>
</tr>
<tr>
<td>Sodium Deoxycholate</td>
<td>Thermo Fisher Scientific</td>
<td>89905</td>
</tr>
<tr>
<td>Tris Base</td>
<td>MilliporeSigma</td>
<td>10708976001</td>
</tr>
<tr>
<td>Sodium Azide</td>
<td>Fisher Scientific</td>
<td>BP922I500</td>
</tr>
<tr>
<td>Yeast Extract</td>
<td>MilliporeSigma</td>
<td>BP1422500</td>
</tr>
<tr>
<td>Tris-Acetate-EDTA buffer (TAE)</td>
<td>Boston BioProducts</td>
<td>BM-250</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>MilliporeSigma</td>
<td>B0126-25G</td>
</tr>
<tr>
<td>Glycine</td>
<td>MilliporeSigma</td>
<td>410225-250G</td>
</tr>
<tr>
<td>Methanol</td>
<td>MilliporeSigma</td>
<td>322415-1L</td>
</tr>
<tr>
<td>HCl</td>
<td>MilliporeSigma</td>
<td>320331-500ML</td>
</tr>
<tr>
<td>NaOH</td>
<td>MilliporeSigma</td>
<td>221465</td>
</tr>
<tr>
<td>NaCl</td>
<td>MilliporeSigma</td>
<td>S3014</td>
</tr>
<tr>
<td>Polyethylene Glycol (PEG) 300</td>
<td>MilliporeSigma</td>
<td>807484100</td>
</tr>
<tr>
<td>Polyethylene Glycol (PEG) 400</td>
<td>MilliporeSigma</td>
<td>8074850050</td>
</tr>
<tr>
<td>Nonfat dry milk</td>
<td>Merck Millipore</td>
<td>M7409</td>
</tr>
<tr>
<td>NZ amine (casein hydrolysate)</td>
<td>Fisher Scientific</td>
<td>NC9874278</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Human embryonic kidney immortalized 293 cells (HEK293T)</td>
<td>Zoghbi Lab</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td colspan="3">Software and database</td>
</tr>
<tr>
<td>Primer3plus</td>
<td><a href="https://primer3plus.com/cgi-bin/dev/primer3plus.cgi">https://primer3plus.com/cgi-bin/dev/primer3plus.cgi</a></td>
<td>Whitehead Institute (<a href="#bib2">Untergasser et al., 2012</a>)</td>
</tr>
<tr>
<td>QuikChange software</td>
<td><a href="https://www.agilent.com/store/primerDesignProgram.jsp">https://www.agilent.com/store/primerDesignProgram.jsp</a></td>
<td>Agilent</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Human <i>GAPDH:</i> Forward, 5′-CGACCACTTTGTCAAGCTCA-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Human <i>GAPDH:</i> Reverse, 5′-TTACTCCTTGGAGGCCATGT-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Human <i>PUM1:</i> Forward, 5′-GCCCCAGTCTTTGCAATTTA-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Human <i>PUM1:</i> Reverse, 5′ AATCACTCGGCAGCCATAAG-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Human <i>ATXN1:</i> Forward 5′-CCAGGTCAGCGTTGAAGTTT-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Human <i>ATXN1:</i> Reverse 5′-CAAAGAGCTGGCTGGTTCTC-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Human <i>E2F3:</i> Forward 5′-AAAGCCCCTCCAGAAACAAG-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Human <i>E2F3:</i> Forward 5′-AATGGGCCCTTGGGTACTT-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Subject 10: 5′-CATGACGATCTTCCACTGGCCTGGCTCC-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Subject 11: 5′-GCGATGTGGGGCCAGATCTTATGCATGACGA-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Subject 12/13/14: 5′-GTACAAGCTGCTCTGAGTGCTGGTGAAGCTC-3′</td>
<td>n/a</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Goat polyclonal anti-PUM1</td>
<td>Bethyl Laboratories</td>
<td>A300-201A</td>
</tr>
<tr>
<td>Mouse monoclonal anti-GAPDH</td>
<td>MilliporeSigma</td>
<td>CB1001</td>
</tr>
<tr>
<td>Goat anti-Mouse IgG (H+L) Cross-Adsorbed Secondary Antibody, HRP</td>
<td>Fisher Scientific</td>
<td>G-21040</td>
</tr>
<tr>
<td>Goat anti-Rabbit IgG (H+L) Cross-Adsorbed Secondary Antibody, HRP</td>
<td>Fisher Scientific</td>
<td>G-21234</td>
</tr>
<tr>
<td colspan="3">Patient-derived cell lines</td>
</tr>
<tr>
<td>Subject 11</td>
<td>Sydney Children’s Hospital</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
<tr>
<td>Subject 12 and 13</td>
<td>Care4Rare Canada</td>
<td>(<a href="#bib1">Gennarino et al., 2018</a>)</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>All essential buffers and solutions should be freshly made prior to sample preparation and used the same day to avoid degradation or loss of efficacy. Make sure that there is enough of all solutions that are required for tissue homogenization, removal of interfering chemicals, RNA extraction, cDNA synthesis, protein extraction and western blot.</p>
<table id="undtbl1">
<thead>
<tr>
<th colspan="3">Radio immunoprecipitation assay (RIPA) buffer</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH 7.5, 1 M)</td>
<td>25 mM</td>
<td>12.5 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>4.4 g</td>
</tr>
<tr>
<td>NP-40</td>
<td>1%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Sodium Deoxycholate</td>
<td>1%</td>
<td>5 g</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate (SDS)</td>
<td>0.1%</td>
<td>0.5 g</td>
</tr>
<tr>
<td>H2O milliQ</td>
<td> </td>
<td>Up to 500 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> the solution can be stored at 4°C or room temperature and completed by adding protease 1× and phosphatase inhibitors 1× just before starting the protein extraction.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Adjust the pH to 7.5.</div>
<table id="undtbl2">
<thead>
<tr>
<th colspan="3">Transfer buffer 10×</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris Base</td>
<td>250 mM</td>
<td>60.4 g</td>
</tr>
<tr>
<td>Glycine</td>
<td>1.92 M</td>
<td>288 g</td>
</tr>
<tr>
<td>H2O milliQ</td>
<td> </td>
<td>Up to 2 L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 4°C.</div>
<table id="undtbl3">
<thead>
<tr>
<th colspan="3">Transfer buffer 1×</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Transfer Buffer 10×</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>Methanol</td>
<td>15%</td>
<td>150 mL</td>
</tr>
<tr>
<td>H2O milliQ</td>
<td> </td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>Caution:</b> Methanol is neurotoxic, use under chemical hood. Store at 4°C.</p>
<table id="undtbl4">
<thead>
<tr>
<th colspan="3">Tris-buffered saline-tween (TBS-T) 20×</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris Base</td>
<td>0.4 M</td>
<td>121.1 g</td>
</tr>
<tr>
<td>Tween20</td>
<td>0.1%</td>
<td>20 mL</td>
</tr>
<tr>
<td>H2O milliQ</td>
<td> </td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at room temperature.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Adjust pH of the solution to 7.5.</div>
<table id="undtbl5">
<thead>
<tr>
<th colspan="3">Tris-buffered saline-tween (TBS-T) 1×</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TBS-T 20×</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>H2O milliQ</td>
<td> </td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at room temperature.</div>
<table id="undtbl6">
<thead>
<tr>
<th colspan="3">TBS-T 1× – BSA- sodium azide</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TBS-T 1×</td>
<td>1×</td>
<td>500 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>3%</td>
<td>15 g</td>
</tr>
<tr>
<td>Sodium Azide</td>
<td>0.02%</td>
<td>10 mg</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Filter the solution with a 0.02 mm filter and store at 4°C. Sodium Azide is irritant, use under chemical hood.</div>
<table id="undtbl7">
<thead>
<tr>
<th colspan="3">Blocking solution for western blot</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TBS-T 1×</td>
<td>1×</td>
<td>50 mL</td>
</tr>
<tr>
<td>Nonfat dry milk</td>
<td>5%</td>
<td>2.5 g</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare fresh and store at 4°C between day 1 and day 2.</div>
<table id="undtbl8">
<thead>
<tr>
<th colspan="3">NZY<sup>+</sup> broth</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NZ amine (casein hydrolysate)</td>
<td>–</td>
<td>10 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>–</td>
<td>5 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>–</td>
<td>5 g</td>
</tr>
<tr>
<td>H2O milliQ</td>
<td>–</td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<div class="note">
<span class="note-title">Note:</span> The steps below are only for the protocols we modified from the manufacturer's suggestion.</div>
<h3 id="sec3.1">Culturing patient-derived fibroblast and lymphoblastoid cell lines</h3>
<p>The acquisition of patient samples is not always easy, particularly in the case of children who have already been subjected to multiple stressful tests. The patient’s clinician is in the best position to speak to the patient to see if they are willing to provide biological samples. Even so, we were able to procure fibroblasts or lymphoblastoid cell lines from only three unrelated patients carrying a single <i>de novo</i> variant in <i>PUM1</i> in addition to three age- and sex-matched healthy controls.</p>
<div class="note">
<span class="note-title">Note:</span> The informed consent from the patient is always needed. If the sample derives from a third party such as a biobank and the subject is de-identified, it is likely that the consent form was already acquired by the third party. Regulations can differ between countries and institutions, so we recommend that you procure Institutional Review Board (IRB) approval before attempting to work with human specimens or clinical data.</div>
<h3 id="sec3.2">Culturing patient-derived cell lines from frozen vials</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<div class="note">
<span class="note-title">Note:</span> Primary patient-derived cell lines can arrive either in frozen vials or in their culturing medium. To increase the chances of cell survival, we strongly suggest you re-culture the cells, as soon as you receive them, with fresh culturing medium.</div>
<div class="note">
<span class="note-title">Note:</span> Fibroblasts and lymphoblastoid cells are cultured at 37°C in a humidified chamber with 5% CO<sub>2.</sub>
</div>
<ol>
<li>Thaw the cells by placing the vial in a water bath set at 37°C for 2–5 min.</li>
<li>Transfer the contents of the vial to a 15 mL conical tube containing 5 mL of warm (∼37°C) complete culturing medium.</li>
<li>Centrifuge at 500 g for 5 min to obtain a pellet of cells.</li>
<li>Aspirate the supernatant without disturbing the cell pellet.</li>
<li>Add 10 mL of warm (∼37°C) complete medium and gently resuspend the pellet by pipetting up and down.</li>
<li>Transfer 10 mL of complete medium containing the cells to a T25 flask or 10 cm cell culture- treated dish.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Since lymphoblastoid cells grow as suspension, we recommend using T25 flasks.</div>
<ol start="7">
<li>The next day replace the medium with fresh complete medium.</li>
</ol>
<h3 id="sec3.3">Culturing patient-derived cell lines from cells in culture</h3>
<ol start="8">
<li>Aspirate the medium and add 10 mL of warm (∼37°C) fresh complete medium.</li>
<li>Replace the medium the next day with fresh medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Primary patient-derived cell lines are a precious resource and asking for new cells is usually not feasible. Therefore, before using the cells for any experiments, we recommend you expand the cells to have good number of vials as backup in liquid nitrogen for long storage. Having a backup will enable you to maintain the cells in low passage number (many experiments in human cells work better this way).</div>
<h3 id="sec3.4">Expanding fibroblast cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<table id="undtbl9">
<thead>
<tr>
<th colspan="3">Fibroblast culturing medium</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dulbecco’s Modified Eagle′s Medium (DMEM)</td>
<td>–</td>
<td>450 mL</td>
</tr>
<tr>
<td>Heat-inactivated fetal bovine serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Penicillin/streptomycin</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Normocin</td>
<td>–</td>
<td>1 mL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<ol start="10">
<li>Once the fibroblasts are at 85%–90% of plate confluency: Aspirate the medium from the plate, trying to avoid disturbing the cells.</li>
<li>Add 5 mL of phosphate buffer (PBS) 1× to rinse the remaining medium from the plate.</li>
<li>Add 1 mL of warm (∼37°C) Trypsin-EDTA 1× to the center of the plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> make sure the Trypsin-EDTA 1× contacts the entire plate surface.</div>
<ol start="13">
<li>Place the plate in the incubator for 5 min.</li>
<li>Using a P1000 pipette, mechanically detach the cells.</li>
<li>Add 4 mL of fibroblast culturing medium.</li>
<li>Transfer the volume in a 1:4 or 1:5 ratio to a new 10 cm cell culture treated dish.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Fibroblasts grow as a monolayer and should be re-fed with fresh medium or split every 3–4 days at a ratio of 1:4 or 1:5, depending on how fast the lineage grows.</div>
<h3 id="sec3.5">Expanding lymphoblastoid cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<table id="undtbl10">
<thead>
<tr>
<th colspan="3">Lymphoblastoid culturing medium</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI-1640</td>
<td>–</td>
<td>450 mL</td>
</tr>
<tr>
<td>Heat-inactivated fetal bovine serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Normocin</td>
<td>–</td>
<td>1 mL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> After 3–4 days from the thawing, the cells should be sub-cultured. The culturing plateau level for lymphoblastoid cells is 1 × 10<sup>6</sup> cells/mL; at this confluency the medium will be acidic and the phenol red used as pH indicator will turn yellow.</div>
<ol start="17">
<li>Pipette up and down for 5 times using a 10 mL serological pipette.</li>
<li>Transfer the cells in a new 50 mL conical.</li>
<li>Take 10 μl of the volume and count the cells.</li>
<li>Seed 200,000–300,000 cells/mL in a new T25 flask.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Lymphoblastoid cells grow in suspension with cells clumped in aggregates. They should be seeded at a concentration of 200,000–300,000 cells/mL and need subculture every 3–4 days.</div>
<h3 id="sec3.6">mRNA extraction and quantification from patient-derived cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>To extract total RNA from human patient or control fibroblast cell lines, cells were seeded in 6-well plates and harvested at 80% confluency (1 × 10<sup>6</sup> cells), whereas lymphoblastoid cell suspension cultures were collected at 6 × 10<sup>6</sup> confluence before proceeding for RNA extraction. Total RNA was obtained using the miRNeasy Mini kit (Qiagen).</p>
<div class="critical">
<span class="critical-title">Critical:</span> RNA is subject to degradation; even our hands contain RNase. It is therefore imperative to work in a clean environment: clean the work surface and the pipettes with RNase decontamination solutions such as RNaseZap, use only filtered pipette tips, clean your gloves with 70% alcohol after you put them on.</div>
<div class="critical">
<span class="critical-title">Critical:</span> QIAzol Lysis Reagent is a monophasic solution that consists of phenol and guanidine thiocyanate. Since phenol is toxic, the QIAzol solution should be handled with caution and always under a chemical fume hood. Laboratories working with phenol should use polyethylene glycol 300 or 400 (PEG-300 or PEG-400), rather than water, for immediate first aid treatment of dermal exposures. In addition, anyone using phenol should use appropriate personal protective equipment (PPE), including lab coat and nitrile gloves. If QIAzol contacts the gloves, it will penetrate the material within a couple of minutes, so immediately change your gloves. Similar precautions should be followed when working with chloroform.</div>
<ol start="21">
<li>Add 700 μL QIAzol Lysis Reagent to the cell pellet in a 1.7 mL tube and lyse the cells by pipetting rather vigorously, around 20 times up and down using a p200 pipette set at 200 μL.</li>
<li>Incubate at room temperature for 5 min.</li>
<li>Add 140 μL of chloroform (for each 700 μL of QIAzol) to the mix and shake vigorously for 15 s.</li>
<li>Incubate at room temperature for 3 min.</li>
<li>Centrifuge the lysate for 15 min, 12,000 g at 4°C. It is not necessary to use low brake for this step.</li>
<li>Carefully transfer the upper aqueous phase to a new collection tube and add 1.5 volumes of 100% ethanol. The phases should not be mixed.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If you accidentally mix or disturb the phases, you must repeat from step 25.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Perform steps 21–26 under chemical fume hood.</div>
<ol start="27">
<li>Load up to 700 μL of sample into a RNeasy Mini column placed in a 2 mL collection tube.</li>
<li>Centrifuge at 11,000 rpm for 15 s at room temperature, discarding the flow-through.</li>
<li>Add 350 μL buffer RWT to the RNAeasy Mini column and spin for 15 s at 11,000 rpm.</li>
<li>Discard the flow-through.</li>
<li>Add 10 μL DNAseI stock solution to 70 μL buffer RDD for each sample, mix gently by inverting the tube and spin briefly.</li>
<li>Add 80 μL of DNAseI reaction directly on the RNAeasy Mini column filter.</li>
<li>Incubate 15 min at RT.</li>
<li>Add 350 μL buffer RWT to the RNeasy spin column and centrifuge 15 s at 11,000 rpm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The DNAseI treatment purifies the RNA solution from genomic DNA contaminants. This treatment is optional, but we strongly recommend it if you plan to do qPCR; if you do not perform qPCR, you can skip steps 31–33.</div>
<ol start="35">
<li>Add 700 μL of RWT buffer to wash the column.</li>
<li>Centrifuge at 11,000 rpm for 30 s, discard the flow-through.</li>
<li>Add 500 μL of RPE wash buffer into the RNAeasy Mini column.</li>
<li>Centrifuge at 11,000 rpm for 30 s, discard the flow-through.</li>
<li>Add 500 μL of RPE wash buffer into the RNAeasy Mini column.</li>
<li>Centrifuge at 11,000 rpm for 2 min.</li>
<li>Place the RNeasy Mini column into a new 1.5 mL collection tube and centrifuge at full speed for 1 min.</li>
<li>Transfer the RNeasy Mini column to a new 1.5 mL collection tube.</li>
<li>Add pre-warmed (60°C) 30–50 μL RNease free water, directly into the column membrane.</li>
<li>After 5 min of incubation centrifuge at 11,000 rpm for 1 min.</li>
<li>Quantify RNA using NanoDrop 1000.</li>
<li>RNA can be stored at −20°C or −80°C for a longer preservation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Freezing and thawing the RNA could cause degradation. We recommend dividing the extracted RNA into small aliquots that can be thawed when needed.</div>
<h3 id="sec3.7">RNA quality assessment</h3>
<div class="note">
<span class="note-title">Note:</span> RNA quality can be assessed by 1% agarose gel electrophoresis in TAE running buffer. The RNA quality assessment through the electrophoresis gives a different information compared to the information given by the 260/280 and 260/230 absorbance ratios. From the electrophoresis the operator will be able to see an overall RNA degradation. This information cannot be obtained from the 260/280 or 260/230 ratio since these analyses will measure protein and DNA contamination, respectively, regardless if the RNA is degraded or not.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare a 3× bromophenol blue loading dye by diluting the commercial 6× loading dye 1:1 in formamide. Rinse the gel chamber and the comb once with NaOH 0.4N and four to five times with abundant deionized water; let them dry before using them.</div>
<div class="note">
<span class="note-title">Note:</span> 150 ng to 1 μg of total RNA can be loaded on the agarose gel. 1 μg of RNA, when possible, will give you a better resolution.</div>
<ol start="47">
<li>Mix 3 μL of 3× formamide bromophenol blue loading dye with 1 μg of the extracted RNA and bring the mix volume up to 9 μL with nuclease-free water in order to obtain a 1× loading dye solution.</li>
<li>Mix gently without vortexing.</li>
<li>Spin down quickly and incubate at 60°C for 10 min.</li>
<li>Place on ice for 2 min.</li>
<li>Load 9 μL of RNA/dye solution on the 1% agarose TAE gel.</li>
<li>Run the gel at low speed until the loading blue runs over half of the gel.</li>
<li>Image the gel in a UV transilluminator. If the RNA quality is optimal, three bands will be visible corresponding to rRNA 28S, 18S, and 5-5.8S. If the RNA degrades, you will see a smear between those three bands (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1503-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Agarose gel for RNA quality assessment</div>
<p>Representative image of agarose gel showing two RNA samples.</p>
<p>The three bands are the rRNA 28S, 18S and 5/5.8S.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Alternatively, the quality of the RNA can be assessed using the Bioanalyzer System which performs a microfluidic-based resolution automated electrophoresis. This system can provide information on both RNA integrity and quality.</div>
<h3 id="sec3.8">cDNA synthesis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Thaw gDNA wipeout buffer, Quantiscript Reverse Transcriptase, Quantiscript RT Buffer, RT Primer Mix, and RNase-free water on ice. Mix each solution by flicking the tubes. Centrifuge briefly to collect residual liquid from the sides of the tubes, and then store on ice.</p>
<div class="note">
<span class="note-title">Note:</span> The QuantiTect Reverse Transcription kit can retrotranscribe up to 1 μg of RNA (see <a href="#key-resources-table">key resources table</a>).</div>
<ol start="54">
<li>Thaw previously frozen RNA on ice.</li>
<li>Prepare the genomic DNA elimination reaction on ice, adding 2 μL of gDNA wipeout buffer to a tube containing 1 μg of RNA. Bring the volume up to 12 μL with nuclease-free water.</li>
<li>Incubate for 8 min at 42°C. Then place immediately on ice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not incubate at 42°C for longer than 10 min.</div>
<ol start="57">
<li>Add to each reaction tube 1 μL of RT Primer Mix, 4 μL of Quantiscript RT Buffer and 1 μL of Quantiscript Reverse Transcriptase.</li>
<li>Incubate for 30 min at 42°C.</li>
<li>Incubate for 3 min at 95°C to inactivate Quantiscript Reverse Transcriptase.</li>
<li>Store the cDNA at 4°C for short term storage or at −20°C for long term storage.</li>
</ol>
<h4 id="sec3.8.1">Quantitative RT-polymerase chain reaction (qRT-PCR)</h4>
<p>In order to quantify mRNA level changes between the patient-derived cells and the control cell lines, perform quantitative RT-polymerase chain reaction (qRT-PCR). In our original study (<a href="#bib1">Gennarino et al., 2018</a>), we used the CFX96 Touch Real-Time PCR Detection System with PowerUP SYBR Green Master Mix to analyze <i>PUM1</i>, <i>ATXN1</i> and <i>E2F3</i> (see <a href="#key-resources-table">key resources table</a>). Results were analyzed using the comparative Ct method normalized against the housekeeping gene <i>GAPDH</i> (<a href="#bib3">Vandesompele et al., 2002</a>). To ensure the efficacy of genomic DNA elimination, we ran negative control samples in the qRT-PCR that did not have reverse transcriptase (minus-RT) in the cDNA synthesis reaction.</p>
<h3 id="sec3.10">Protein extraction and quantification in patient-derived cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>To measure PUM1 quantities in patient-derived cells lines, we first tested different antibodies to find the one that gave the best signal with the lowest background. Since patient-derived cell lines are highly variable, we recommend repeating the experiment no less than 6–9 times after proper optimization.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Patient-derived cell lines need to be compared with healthy donor cell lines that must be matched according to cell type, sex, and age. Again, given the variability of primary human cells, we suggest using at least 2 to 3 healthy donors as control comparisons for each patient-derived cell line.</div>
<ol start="61">
<li>Aspirate the medium from the plate without disturbing the cells.</li>
<li>Add 5 mL of phosphate buffer (PBS) 1× to rinse the plate from the remaining medium.</li>
<li>Add 1 mL of warm (∼37°C) Trypsin-EDTA 1×.</li>
<li>Place the plate in the incubator for 5 min.</li>
<li>Using a P-1000 pipette, mechanically detach the cells by pipetting up and down.</li>
<li>Add 4 mL of fibroblast culturing medium.</li>
<li>Transfer the cells in a 15 mL conical and centrifuge at 500 g for 5 min.</li>
<li>Aspirate the medium without disturbing the pellet.</li>
<li>Add 300 μL of cold RIPA buffer completed with phosphatase and protease inhibitors 1×.</li>
<li>Using a P-1000 pipette dissolve the pellet pipetting up and down for 10–15 times.</li>
<li>Transfer the 300 μL of lysate in a 1.7 mL Eppendorf tube.</li>
<li>Place the tube in ice for 20 min.</li>
<li>Centrifuge for 20 min at 21,000 g at 4°C.</li>
<li>Transfer the supernatant in a new 1.7 mL Eppendorf tube and place it in ice.</li>
</ol>
<ol start="75">
<li>Quantify the protein concentration using the Pierce BCA Protein Assay Kit (Thermo Fisher Scientific).
<div class="note">
<span class="note-title">Note:</span> To evaluate the protein concentration with this method, prepare a bovine serum albumin (BSA) standard curve with 2 μg/μL, 1 μg/μL, 0.5 μg/μL and a sample only containing the BCA mix.</div>
<ol type="a">
<li>mix reagent A and B in a ratio of 50:1 for a total volume of 40 μL per sample.</li>
<li>add 38 μL of BCA mix to a new 1.7 mL Eppendorf tube.</li>
<li>add 2 μL of protein extract.</li>
<li>add 2 μL of the BSA curve for each standard point.</li>
<li>vortex and spin the tubes briefly.</li>
<li>incubate for 30 min at 37°C.</li>
<li>analyze the absorbance at 562 nm using the NanoDrop OneC (Thermo Scientific) and based on the standard curve calculate the protein concentration (mg/mL).</li>
</ol>
</li>
</ol>
<ol start="76">
<li>Mix the extracted protein with NuPAGE Reducing Agent 10× (Thermo Fisher Scientific) and NuPAGE LDS Sample Buffer 4×.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Consider a final volume of 30 μL–40 μL per well if using the 1.5 mm 10-wells gel or 20 μL–30 μL per well if using the 1.5 mm 15-wells gel.</div>
<ol start="77">
<li>Denature the proteins at 95°C for 10 min.</li>
<li>Pour 100–150 mL of running buffer to cover the bottom of the mini cell (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1503-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Western blot for the analysis of PUM1 levels in patient derived fibroblasts</div>
<p>(A) Picture of NuPage mini cell chamber containing running buffer to cover the bottom part.</p>
<p>(B) Lateral picture of the NuPage mini cell chamber loaded with a 1.5 mm 10 wells gel (asterisk) and a plastic adaptor on the opposite side.</p>
<p>(C) Components to assemble the transfer sandwich.</p>
<p>(D–H) Steps of the sandwich assembly (see text for details).</p>
<p>(I) PUM1 western blot redout from patient-derived cells. KDa: Kilodaltons; Ctrl1 and Ctrl2: age and sex match controls; S11: patient carrying the PUM1 <i>de novo</i> missense mutation (R1147W).</p>
</figcaption>
</figure>
</li>
<li>Place a NuPAGE 4%–14% Bis-Tris Gel (10 wells 1.5 mm) in a mini gel cassette (Thermo Fisher Scientific) (<a href="#fig2">Figure 2</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The wells of NuPAGE 4%–14% Bis-Tris Gel can sometimes contain residual part of gel, make sure of cleaning them by pipetting up and down 100–150 μL of running buffer into the wells.</div>
<ol start="80">
<li>Load the samples and run until the dye front reaches the bottom of the gel.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Run at 70 V until the dye front surpasses the stacking gel (upper part of the gel), then increase to 120 V. To have well-shaped bands in the western blot, we do not recommend running the gel any faster than 120 V, particularly for proteins with a high molecular weight.</div>
<ol start="81">
<li>Prepare 1 L of transfer solution to transfer the proteins from the acrylamide gel to a nitrocellulose membrane.</li>
<li>Prepare the transfer sandwich (<a href="#fig2">Figure 2</a>C):
<div class="critical">
<span class="critical-title">Critical:</span> Activate the nitrocellulose membrane in methanol or in transfer solution for 20–30 s.</div>
<ol type="a">
<li>Pour 500 mL of transfer buffer in the wide-set glass container and the remaining 500 mL in the transfer apparatus.</li>
<li>Place the sandwich with the black side facing down (<a href="#fig2">Figure 2</a>D).</li>
<li>Immerse for 10 s a sponge into the transfer buffer and place it over the black side of the sandwich (<a href="#fig2">Figure 2</a>D).</li>
<li>Open the gel support and with the help of a filter paper carefully detach the acrylamide gel from the plastic support (<a href="#fig2">Figures 2</a>E and 2F).</li>
<li>Place the filter paper and the acrylamide gel over the sponge with the gel facing up (<a href="#fig2">Figure 2</a>G).</li>
<li>Gently put the nitrocellulose membrane over the acrylamide gel.</li>
<li>Immerse a filter paper for 5 s in the transfer buffer and put it over the nitrocellulose membrane.</li>
<li>Immerse the roller in the transfer buffer and gently roll over the chromatography paper to make sure no air-bubble between the acrylamide gel and the nitrocellulose membrane will remain.</li>
<li>Immerse for 10 s the sponge into the running buffer.</li>
<li>Put it on top of the chromatography paper and close the sandwich.</li>
<li>Place it into the apparatus with the black side facing the black side of the apparatus (<a href="#fig2">Figure 2</a>H).</li>
<li>Transfer the 500 mL from the wide-set glass container to the transfer apparatus.
<div class="note">
<span class="note-title">Note:</span> If transferring only one acrylamide gel, put two sponges wet of running buffer in the other sandwich.</div>
</li>
</ol>
</li>
</ol>
<ol start="83">
<li>Run the transfer applying a constant current of 340 mA for 90 min in cold room (4°C).</li>
<li>Prepare blocking solution dissolving 2.5 g of nonfat milk in 50 mL of TBS-T 1× to obtain a solution of milk at 5%.</li>
<li>Remove the nitrocellulose membrane, with the transferred proteins, from the transfer apparatus and incubate (blocking) with milk 5% in TBS-T 1× for at least 30 min at room temperature.</li>
<li>Rinse the membrane 1–2 times with TBS-T 1× to remove the excess of milk.</li>
<li>Incubate the membrane with the antibody of interest dissolved in TBS-T 1× with 3% BSA and 0.02% of Sodium Azide to a final concentration indicated in the manufacturer’s data sheet overnight at 4°C.</li>
<li>Wash membranes with TBS-T 1× for 10 min each for three times.</li>
<li>Incubate the membrane with secondary antibody horseradish peroxidase, HRP-conjugated, dissolved in milk 5× at a final concentration of 1:10,000 for at least 1 h at room temperature.</li>
<li>To image the blot use ECL solution to reveal the images. All the blots are acquired on the G:BOX Chemi XX9 machine (Syngene; Frederick, MD) using GeneSys software 1.6.5.0. Gel exposures are determined by the software. (<a href="#fig2">Figure 2</a>I).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To appreciate modest differences in protein expression, we found that several steps required optimization: i) dilution rates for the primary and secondary antibodies can vary according to your antibody, protein of interest, cell lines, and tissue. Run a few western blots starting from the highest dilution ratio (as recommended by the manufacturer) to the lowest ratio until you will be able to visualize a clear band in your gel; ii) while primary antibodies can be diluted in TBS-T 1×-BSA 3%, other antibodies will work better in TBS-T 1× in 5% non-fat milk. Because antibodies in milk cannot be re-used more than twice, and therefore can become rather expensive, run a western with the primary antibody diluted in both solutions to see which will give you a better resolution; iii) to minimize smearing, avoid running the gel faster than 120 V for proteins weighing more than 90 kDa; iv) to transfer as much protein as possible from your gel to your membrane to ensure a correct quantification, be aware of the influence of protein size. Many labs use a fast transfer technique, but proteins &gt;90 kDa should be transferred at a power no greater than 340 mA for no less than 90 min but not more than 120 min; v) if your protein is highly expressed, use larger wells such as the 10-well gel at 1.5 mm. However, if you want to increase the signal for a protein that is expressed at low levels, use 15 wells at 1.5 mm. In the case of PUM1, 15-well gels did not show as clear a band as the10-well gels.</div>
<h3 id="sec3.11">Study the single gene variant in human cell lines</h3>
<p>HEK293T cells were incubated at 37°C in a humidified chamber supplemented with 5% CO<sub>2</sub>. These are fast-growing cells that require high glucose DMEM (Invitrogen) medium to survive. Culturing media should contain broad-spectrum antibiotics such as 1% penicillin/streptomycin to reduce the chances of bacterial contamination. We also add 10% of heat-inactivated fetal bovine serum (FBS) to supplement the basal medium with nutrients derived from the serum (vitamins, amino acids, growth factors, etc.) and 1% L-Glutamine.</p>
<p>The purpose of this section is to guide the reader testing the effect of one or more mutations of the disease-driving gene. In the specific case of PUM1, the readout of the overexpression using PUM1 mutants was to address the protein and mRNA levels of the PUM1 targets ATXN1 and E2F3.</p>
<h3 id="sec3.12">Transfection of mammalian gene expression vectors into HEK293T cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<div class="note">
<span class="note-title">Note:</span> To transfect HEK293T cells we used jetPRIME Transfection Reagent (see <a href="#key-resources-table">key resources table</a>).</div>
<ol start="91">
<li>Seed HEK293T cells in 12 well-plates at a density of 0.1 × 10<sup>6</sup> cells/well.</li>
<li>On day two, cells should be at 70% confluence, ready to be transfected with either full or mutant cDNA of <i>PUM1</i> (3567 nt) cloned into a mammalian expression vector OmickLink<sup>TM</sup>Expression Vector pEZ-M02.</li>
<li>Dilute the desired amount of DNA in 75 μL of jetPRIME buffer, vortex for 10 s and quickly spin down.</li>
<li>Add 1.6 μL of jetPRIME reagent, vortex briefly, spin down and incubate for 10 min at room temperature.</li>
<li>Add the transfection mix to each well and place the plate in the incubator for 48 h.</li>
<li>The day after transfection, aspirate the medium containing the transfection mix and replace with fresh complete culture medium.</li>
<li>After 48 h post-transfection, collect the cells and process for RNA or protein extraction.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our study, we transfected 0.5 μg, 1 μg and 2 μg of DNA to test the effect of different protein dosages on <i>PUM1</i> targets.</div>
<h4 id="sec3.12.1">Mutagenesis</h4>
<p>We created three <i>PUM1</i> mutants (R1139W, R1147W and T1035S) to insert into HEK293T cells by using the QuikChange Lightning Multi Site-Directed Mutagenesis Kit. We designed the primers for mutagenesis using QuikChange software (<a href="#key-resources-table">key resources table</a><b>)</b>.</p>
<ol start="98">
<li>Prepare the sample reaction mix as follow:</li>
</ol>
<table id="undtbl11">
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Reaction Buffer 10×</td>
<td>1×</td>
<td>5 μL</td>
</tr>
<tr>
<td>Oligonucleotide Forward (10 μM)</td>
<td>0.2 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Oligonucleotide Reverse (10 μM)</td>
<td>0.2 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>dNTP mix 10 μM</td>
<td>0.2 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>QuikSolution</td>
<td> </td>
<td>3 μL</td>
</tr>
<tr>
<td>DNA plasmid</td>
<td> </td>
<td>50 ng</td>
</tr>
<tr>
<td>
<i>PfuUltra</i> HF DNA polymerase</td>
<td>2.5 U</td>
<td>1 μL</td>
</tr>
<tr>
<td>H<sub>2</sub>O nuclease-free</td>
<td> </td>
<td>Up to 50 μL</td>
</tr>
</tbody>
</table>
<ol start="99">
<li>Run the PCR as follows:</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th colspan="4">PCR cycling setup</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>50 s</td>
<td rowspan="3">18 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>60°C</td>
<td>50 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>13 min (1 min/kb)</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>7 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">infinite</td>
</tr>
</tbody>
</table>
<ol start="100">
<li>Add 1 μL of the DpnI restriction enzyme (10 U/μL) directly to each reaction tube once the PCR is completed.</li>
<li>Gently but thoroughly mix each reaction mixture by pipetting the solution up and down several times. Spin down the reaction mixtures in a microcentrifuge for 1 min, then immediately incubate the reactions at 37°C for 1 h to digest the parental (i.e., the nonmutated) supercoiled dsDNA.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is important to adhere to the 18-cycle limit when cycling the mutagenesis reactions. More than 18 cycles can impair reaction efficiency.</div>
<ol start="102">
<li>Gently thaw the XL10-Gold ultracompetent cells on ice. For each control and sample reaction to be transformed, aliquot 45 μL of the ultracompetent cells to a prechilled 14-mL BD Falcon polypropylene round-bottom tube.</li>
<li>Add 2 μL of the β-ME mix.</li>
<li>Gently swirl the contents of the tube. Incubate the cells on ice for 10 min, swirling gently every 2 min.</li>
<li>Transfer 2 μL of the Dpn I-treated DNA from each control and sample reaction to separate aliquots of the ultracompetent cells. Swirl the transformation reactions gently to mix and incubate the reactions on ice for 30 min.</li>
<li>Preheat NZY+ broth in a 42°C water bath.</li>
<li>Heat-pulse the tubes in a 42°C water bath for 30 s. The duration of the heat pulse is critical for obtaining the highest efficiencies. Do not exceed 42°C.</li>
<li>Incubate the tubes on ice for 2 min.</li>
<li>Add 0.5 mL of preheated (42°C) NZY+ broth to each tube, then incubate the tubes at 37°C for 1 h with shaking at 225–250 rpm.</li>
<li>Plate the appropriate volume of each transformation reaction, as indicated in the table below, on agar plates containing the appropriate antibiotic for the plasmid vector.</li>
<li>Incubate the transformation plates at 37°C for &gt;16 h.</li>
<li>Screen the positive colonies by enzymatic digestions and sequencing.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The RNA yield from patient-derived cell lines should be more than enough to generate and repeat all the qRT-PCRs in this protocol. In case of <i>de novo</i> missense mutations, the qPCR analysis should indicate if the mutation has an effect on mRNA stability and decay. In case of deletions, the qPCR will indicate if the wild- type allele is able to compensate for the absence of one allele.</p>
<p>Protein quantification from patient-derived cell lines needs to be assessed multiple times in high-quality manner in order to quantify a difference of 25% in protein expression. In order to understand which condition will give you the better resolution, we highly recommend to use multiple antibodies, different protein concentrations, and different sizes and number of wells for the NuPage protein gel before starting the quantification experiments.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>RNA degradation is common; work as cleanly as possible. Fortunately, mRNA levels of a heterozygous missense mutation will not change when compared to the expression levels of the same gene in heathy age-matched cells. If this happens, it is likely because a mutation changing one amino acid in the final protein product was not detected by the decay machinery.</p>
<p>To better analyze the effect of a particular point mutation, we recommend analyzing protein levels using western blot analysis (WB), extracting proteins from patient-derived cell lines with RIPA buffer as described above. In the case of PUM1, we recommend using the same antibody and setup conditions used in this protocol. For other proteins, you will likely need to test several antibodies from different companies, in isolation and in combination, and/or tweak the conditions to get the best results.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Patient-derived fibroblasts can sometimes be slow in the growth, due to the mutation they carry.</p>
<h3 id="sec6.2">Potential solutions</h3>
<p>Increase the amount of fetal bovine serum (FBS) from 10% to 20%, especially during the first days after thawing.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Ct variability of the housekeeping gene for qPCR normalization.</p>
<h3 id="sec6.4">Potential solutions</h3>
<p>Test more than one housekeeping genes such as <i>GAPDH</i>, <i>HPRT</i> and <i>ACTB</i> in order to understand which one has less Ct variability between the samples. Normalizing on a variable housekeeping, can give false positive or false negative results, introducing a confounding effect.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>RNA degradation and contamination.</p>
<h3 id="sec6.6">Potential solutions</h3>
<p>When running total RNA in an agarose gel, the appearance of a smear indicates degradation. In this case, be sure all solutions and micropipettes are clean and nuclease-free. Also, use filter tips when dealing with RNA, to preserve its integrity, and keep the RNA on ice. For tissue, please take note that you will have a huge amount of RNA so if you take out only 20% of the supernatant after adding the chloroform will be enough to run all the planned qRT-PCRs. You should also control RNA purity by spectrophotometry. A 260/230 ratio below 2.0 might indicate solvent contamination, while a 260/280 ratio below 1.8 might indicate protein contamination.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Protein lysate can be sticky due to high amount of free nucleic acid.</p>
<h3 id="sec6.8">Potential solutions</h3>
<p>Before step 12 of protein quantification from patient-derived cell lines, aspirate the protein solution up and down 5–10 times using a 1 mL syringe with a 27G needle to break the long nucleic acid chains.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Low transformation efficiency of mutagenized plasmid.</p>
<h3 id="sec6.10">Potential solutions</h3>
<p>It is crucial to start from a good quality template DNA. Check the template plasmid DNA on agarose gel to ensure no degradation of the prep.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Gennarino, V.A., Palmer, E.E., McDonell, L.M., Wang, L., Adamski, C.J., Koire, A., See, L., Chen, C.A., Schaaf, C.P., Rosenfeld, J.A., et al. (2018). A mild PUM1 mutation is associated with adult-onset ataxia, whereas haploinsufficiency causes developmental delay and seizures. Cell <i>172</i>, 924-936.e11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00112-5/sref1">View at publisher</a></p>
<p id="bib2">Untergasser, A., Cutcutache, I., Koressaar, T., Ye, J., Faircloth, B.C., Remm, M., and Rozen, S.G. (2012). Primer3—new capabilities and interfaces. Nucleic Acids Res. <i>40</i>, e115. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00112-5/sref2">View at publisher</a></p>
<p id="bib3">Vandesompele, J., De Preter, K., Pattyn, F., Poppe, B., Van Roy, N., De Paepe, A., and Speleman, F. (2002). Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biol. <i>3</i>, 1-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00112-5/sref3">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources should be directed to and will be fulfilled by the lead contact, Vincenzo A. Gennarino (<a href="mailto:vag2138@cumc.columbia.edu">vag2138@cumc.columbia.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate unique reagents.</p>
<h4>Data and code availability</h4>
<p>No software was generated for this project. All softwares used in this study are publicly available and links are provided as appropriate in different sections of the methods and <a href="#key-resources-table">key resources table</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank members of the Gennarino lab for helpful discussions and the staff. We also thank V. Brandt for essential input on the manuscript. Patient-derived cell fibroblasts and lymphoblastoid cell lines were used and obtained from Sydney Children’s Hospital and Care4Rare Canada, see (<a href="#bib1">Gennarino et al., 2018</a>). This work was supported by the <a href="https://doi.org/10.13039/100000065">National Institute of Neurological Disorders and Stroke</a> (NINDS; R01NS109858 to V.A.G.) and the Paul A. Marks Scholar Program, Columbia University Vagelos College of Physicians and Surgeons (V.A.G.).</p>
<h3>Author contributions</h3>
<p>V.A.G. devised the original protocol. N.dP. wrote the technical sections and re-optimized each step of the protocol. S.B., W.L, S.R., and A.C. contributed to the technical sections. N.dP. and V.A.G. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>