<article>
<section>
<h1 id="header">Processing human lymph node samples for single-cell assays</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Tobias Roider,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">5</a>,<a href="#fn2">6</a>,<a href="#cor1">*</a></sup> Berit J. Brinkmann,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a></sup> and Sascha Dietrich<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup></p>
<p id="aff1"><sup>1</sup>European Molecular Biology Laboratory (EMBL), 69120 Heidelberg, Germany</p>
<p id="aff2"><sup>2</sup>Molecular Medicine Partnership Unit (MMPU), 69120 Heidelberg, Germany</p>
<p id="aff3"><sup>3</sup>Department of Medicine V, Hematology, Oncology and Rheumatology, University of Heidelberg, 69120 Heidelberg, Germany</p>
<p id="aff4"><sup>4</sup>Clinical Cooperation Unit Molecular Hematology/Oncology, German Cancer Research Center (DKFZ), 69120 Heidelberg, Germany</p>
<p id="fn1"><sup>5</sup>Lead contact</p>
<p id="fn2"><sup>6</sup>Technical contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:tobias.roider@embl.de">tobias.roider@embl.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100914">10.1016/j.xpro.2021.100914</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Most non-Hodgkin's lymphomas grow exclusively in the lymph node compartment protected by the tumor microenvironment. To better understand the cellular heterogeneity and the complex interaction between malignant and non-malignant cells, experiments with primary, patient-derived samples are often indispensable. Here, we describe a time-efficient but gentle protocol to process human lymph node samples. This protocol avoids enzymatic or mechanical stress and was optimized for the purpose of generating single-cell suspension suitable for delicate assays, such as single-cell RNA sequencing.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref7">Roider et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol to isolate and freeze lymphocyte suspensions from human lymph node samples</li>
<li>Avoiding enzymatic stress to preserve gene expression and epitope profiles</li>
<li>Optimized thawing and dead cell removal to meet the standards of single-cell sequencing</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1117-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Introduction</h3>
<p>Biobanking of lymph node-derived viable cells is not part of a standard routine in most clinical institutions. However, the majority of T and B cell non-Hodgkin lymphomas (NHL) grow exclusively in the lymph node compartment. Despite the large number of available cell line models (<a href="#bib5">Quentmeier et al., 2019</a>), primary cells are indispensable, especially for studies investigating tumor heterogeneity or tumor microenvironment.</p>
<p>Here, we describe a time-efficient but gentle protocol to isolate lymphocytes from human lymph node samples. Many protocols require enzymatic digestion to prepare single cell suspensions from tissues; however, enzymatic stress not only affects the expression of surface proteins (<a href="#bib6">Reichard and Asosingh, 2019</a>) but can also drastically alter single cell gene expression profiles (<a href="#bib4">Mattei et al., 2020</a>). Our protocol deliberately avoids enzymatic digestion or major mechanical stress and therefore provides an optimized procedure to prepare single cell suspensions from lymph node samples. Before freezing, quality of lymph-node-derived cells is assessed by flow cytometry. We further optimized the thawing procedure and included magnetic removal of dead cells so that thawed samples meet the standards for delicate assays, such as single cell RNA sequencing. Single cell suspensions processed and prepared this way can also be used for other kinds of assays, e.g., drug response profiling (<a href="#bib7">Roider et al., 2020, 2021</a>; ). Moreover, this protocol can be used to collect lymph node-derived exosomes (<a href="#bib1">Bordas et al., 2020</a>).</p>
<h3 id="sec1.2">General preparations</h3>
<p>As lymph node excisions represent relatively invasive diagnostic procedures, ethical approval adhering to regional and institutional regulations is of outstanding importance. Informed consent must be collected of all patients before surgery. Patients whose tissue is being processed must test negatively for hepatitis B, C and HI virus. Excised lymph node tissue must be processed in accordance with regional and institutional safety regulations.</p>
<p>Biobanking of viable lymph node tissue requires specific infrastructure to ensure a standardized workflow and consistently high quality of lymph node-derived cells. Surgeons and perioperative nurses must be instructed in details how to handle lymph node tissue for this specific purpose. We recommend splitting of lymph node tissue immediately after excision. One piece is handled according to the standard diagnostic routine, whereas the second piece is wrapped in a saline-soaked compress and then placed in a sterile container. Subsequent processing of lymph node samples should be performed as soon as possible.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>FITC anti-human CD8</td>
<td>BioLegend</td>
<td>RRID:AB_314110</td>
</tr>
<tr>
<td>BV421 anti-human CD19</td>
<td>BioLegend</td>
<td>RRID:AB_11142678</td>
</tr>
<tr>
<td>BV605 anti-human CD20</td>
<td>BioLegend</td>
<td>RRID:AB_2563398</td>
</tr>
<tr>
<td>PE anti-human Ig light chain κ</td>
<td>BioLegend</td>
<td>RRID:AB_493613</td>
</tr>
<tr>
<td>PE/Dazzle 597 anti-human Ig light chain λ</td>
<td>BioLegend</td>
<td>RRID:AB_2687261</td>
</tr>
<tr>
<td>PerCP/Cy5.5 anti-human CD3</td>
<td>BioLegend</td>
<td>RRID:AB_2561628</td>
</tr>
<tr>
<td>PE/Cy7 anti-human CD5</td>
<td>BioLegend</td>
<td>RRID:AB_2275812</td>
</tr>
<tr>
<td>AF700 anti-human CD4</td>
<td>BioLegend</td>
<td>RRID:AB_493743</td>
</tr>
<tr>
<td>Viability Dye eFluor506</td>
<td>Thermo Fisher Scientific</td>
<td>65-0866-18</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Gibco</td>
<td>10500-064</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>Gibco</td>
<td>15140-122</td>
</tr>
<tr>
<td>EDTA</td>
<td>Sigma Aldrich</td>
<td>E8008</td>
</tr>
<tr>
<td>RPMI medium</td>
<td>Gibco</td>
<td>21875-034</td>
</tr>
<tr>
<td>Trypan blue solution 0.4%</td>
<td>Gibco</td>
<td>11538886</td>
</tr>
<tr>
<td>DMSO</td>
<td>Serva</td>
<td>20385.01</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>MS Columns</td>
<td>Miltenyi</td>
<td>130-042-201</td>
</tr>
<tr>
<td>Dead Cell Removal Kit</td>
<td>Miltenyi</td>
<td>130-090-101</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo</td>
<td>BD Biosciences</td>
<td>RRID:SCR_008520</td>
</tr>
<tr>
<td>FACSDiva</td>
<td>BD Biosciences</td>
<td>RRID:SCR_001456</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Sterilizable forceps</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>Sterilizable scissors</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>Sterilizable mortar</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>Strainer 100 μm</td>
<td>Greiner Bio-One</td>
<td>542000</td>
</tr>
<tr>
<td>Strainer 70 μm</td>
<td>Greiner Bio-One</td>
<td>542070</td>
</tr>
<tr>
<td>Pasteur pipette</td>
<td>LP Italiana</td>
<td>135030</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Thermo Scientific</td>
<td>Heraeus Megafuge 16R</td>
</tr>
<tr>
<td>Cryo vials</td>
<td>Greiner Bio-One</td>
<td>122263</td>
</tr>
<tr>
<td>MACS Multistand</td>
<td>Miltenyi</td>
<td>130-042-303</td>
</tr>
<tr>
<td>OctoMACS Separator</td>
<td>Miltenyi</td>
<td>130-042-108</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Cell culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI (1X)</td>
<td>n/a</td>
<td>445 mL</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Penicillin 10,000 units/mLStreptomycin 10 mg/mL</td>
<td>100 units/mL0.1 mg/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for three months, prepare and use under sterile conditions.</p>
<table id="undtbl2">
<caption>Washing buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Phosphate-buffered saline (PBS) 1×</td>
<td>n/a</td>
<td>492.5 mL</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for three months.</p>
<table id="undtbl3">
<caption>Freezing medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>80%</td>
<td>40 mL</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>20%</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for three months, prepare and use under sterile conditions.</p>
<table id="undtbl4">
<caption>EDTA-free washing buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS 1×</td>
<td>n/a</td>
<td>47.5 mL</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for three months.</p>
<table id="undtbl5">
<caption>EDTA-containing cell culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cell culture medium</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>0.5%</td>
<td>0.25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>≈ 50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for three months, prepare and use under sterile conditions.</p>
<table id="undtbl6">
<caption>1X Binding buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Binding buffer 20×</td>
<td>n/a</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Water (sterile, distilled)</td>
<td>n/a</td>
<td>47.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare freshly for each experiment.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> You may replace cell culture medium and washing buffer by solutions you are most experienced with. However, we recommend the usage of EDTA, as indicated, because it binds calcium and thus prevents clumping of cells.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of single cell suspension from human lymph nodes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60 min</div>
<p>Lymph node tissue is minced to obtain single cell suspension. The following steps are performed under sterile conditions.</p>
<ol>
<li>Transfer lymph node into a sterile mortar (<a href="#fig1">Figure 1</a>A) and add 5 mL cell culture medium.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1117-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Key steps of lymph node processing</div>
<p>(A and B) Two different representative lymph nodes in a mortar (A) or on a saline-soaked compress (B) immediately after excision. B) Lymph node or excessive fat tissue is highlighted with blue or green arrows, respectively.</p>
<p>(C) Lymph node in cell culture medium after being cut into small pieces.</p>
<p>(D) Lymph node pieces in a cell strainer. Tube filled with approximately 50 mL medium containing lymph node-derived cells.</p>
<p>(E) Pellet of lymph node-derived cells after centrifugation. Reddish stain indicates slight contamination with erythrocytes.</p>
</figcaption>
</figure>
</li>
<li>Remove excessive fat and connective tissue (<a href="#fig1">Figure 1</a>B) using dissecting scissors or a scalpel.</li>
<li>Cut lymph node into small pieces of approx. 1–2 mm size using dissecting scissors and forceps (<a href="#fig1">Figure 1</a>C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The smaller the pieces, the higher the cell yield. However, for larger lymph node pieces, step 3 can be very time-consuming and defines the overall duration of this protocol. If you rely on fast processing rather than high cell yield, you can drastically shorten this step by cutting the lymph node into fewer pieces. We do not recommend using a pestle because it generates more debris.</div>
<ol start="4">
<li>Using a 5 mL serological pipette, thoroughly pipette up and down to gently wash out the lymphocytes.</li>
<li>Filter lymphocyte-containing medium through a 100 μm strainer into a sterile 50 mL tube (<a href="#fig1">Figure 1</a>C).</li>
<li>Add 5 mL cell culture medium and repeat steps 3–5 once or twice depending on the size of the lymph node. The strainer can be reused unless it becomes clogged.</li>
<li>Rinse cell strainer with fresh cell culture medium to fill up the tube to 50 mL.</li>
<li>Centrifuge tube at 400 <i>g</i> for 5 min at 18°C–25°C.</li>
<li>Carefully decant supernatant (<a href="#fig1">Figure 1</a>D).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> At this step, lymph node-derived exosomes can be collected. To do so, the use of exosome-free FBS and cell culture medium is required in all preceding steps (<a href="#bib1">Bordas et al., 2020</a>).</div>
<ol start="10">
<li>Gently resuspend the pellet in 10 mL cell culture medium and pass cell suspension through 70 μm strainer.</li>
<li>Centrifuge at 400 <i>g</i> for 5 min at 18°C–25°C.</li>
<li>Resuspend cells in cell culture medium. In case of clumps, repeat steps 10–12.</li>
<li>Count cells and set 1 × 10<sup>6</sup> cells aside for quality control by flow cytometry (see below).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> After this step, cells can be used freshly for any subsequent assays.</div>
<div class="note">
<span class="note-title">Note:</span> For single cell (RNA) sequencing non-frozen cells might generally be the better option. However, single cell (RNA) library preparation is an elaborate process and rather inefficient if performed for only one sample at a time. In addition, batch effects could mask interpatient differences. Therefore, we freeze all samples before use.</div>
<ol start="14">
<li>Fill 2 mL cryotubes with 700 μL of cold freezing medium.</li>
<li>Dilute cell suspensions to desired cell concentration using cell culture medium, for instance 5 × 10<sup>6</sup> cells per 700 μL medium.</li>
<li>Add 700 μL cell suspension drop-wise to cryotubes and gently invert tubes a few times.</li>
<li>Place cryotubes quickly into a slow freezing container and store at −80°C for ≥ 24 h. Long-term storage should be below −150°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the total yield of cells, we freeze 5 × 10<sup>6</sup>, 1 × 10<sup>7</sup>, 2 × 10<sup>7</sup> and 5 × 10<sup>7</sup> cells per vial. Aliquot sizes can be adjusted depending on the planned set of experiments. Please see the section <a href="#expected-outcomes">expected outcomes</a> for cell yield of an average lymph node sample.</div>
<h3 id="sec3.2">Output control using flow cytometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60–90 min</div>
<p>In order to check and document the cellular output, lymph node-derived cells are stained for viability, as well as basic B and T cell markers and analyzed on a flow cytometer. This step is crucial as the frequency of B and T cells varies significantly across patient samples.</p>
<ol start="18">
<li>Transfer leftover 1 × 10<sup>6</sup> cells (from step 13) into a 5 mL flow cytometry tube.</li>
<li>Add 1 mL of washing buffer and centrifuge cells at 400 <i>g</i> for 5 min at 18°C–25°C.</li>
<li>Decant supernatant and resuspend pellet in remaining buffer (ideally 50–100 μL). Then, add the antibodies, as shown in <a href="#tbl1">Table 1</a>.
<table id="tbl1">
<caption>Table 1. Suggested flow cytometry panel</caption>
<thead>
<tr>
<th>Antibody/Target</th>
<th>Conjugate/Dye</th>
<th>Recommended amount</th>
<th>Excitation laser</th>
</tr>
</thead>
<tbody>
<tr>
<td>Anti-human CD19</td>
<td>BV421</td>
<td>0.5 μL</td>
<td>Blue (405 nm)</td>
</tr>
<tr>
<td>Viability</td>
<td>eFluor 506</td>
<td>0.2 μL</td>
<td>Blue (405 nm)</td>
</tr>
<tr>
<td>Anti-human CD8</td>
<td>FITC</td>
<td>0.5 μL</td>
<td>Green (488 nm)</td>
</tr>
<tr>
<td>Anti-human CD3</td>
<td>PerCP/Cy5.5</td>
<td>1.0 μL</td>
<td>Green (488 nm)</td>
</tr>
<tr>
<td>Anti-human Ig light chain κ</td>
<td>PE</td>
<td>1.0 μL</td>
<td>Yellow (561 nm)</td>
</tr>
<tr>
<td>anti-human Ig light chain λ</td>
<td>PE/Dazzle</td>
<td>1.0 μL</td>
<td>Yellow (561 nm)</td>
</tr>
<tr>
<td>Anti-human CD5</td>
<td>PE/Cy7</td>
<td>1.0 μL</td>
<td>Yellow (561 nm)</td>
</tr>
<tr>
<td>Anti-human CD4</td>
<td>AF700</td>
<td>1.0 μL</td>
<td>Red (633 nm)</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Staining for kappa and lambda light chain enables quick detection of light chain-restricted populations proving the diagnosis of B cell lymphoma (<a href="#bib3">Horna et al., 2011</a>). The same is true for co-expression of CD5 and CD19 (<a href="#bib2">Hallek et al., 2008</a>). Quantification of B and CD4<sup>+</sup>/CD8<sup>+</sup> T cells might help to plan future experiments. The staining panel can be adjusted according to the titration or availability of antibodies and the configuration of flow cytometer at your means. To detect the above mentioned fluorophores, the flow cytometer must be equipped with suitable lasers (as shown) and filters.</div>
<ol start="21">
<li>Briefly vortex and incubate at 4°C protected from light for 30 min.</li>
<li>Add 1 mL of washing buffer, centrifuge cells at 400 <i>g</i> for 5 min and discard supernatant.</li>
<li>Repeat for a total of three washes.</li>
<li>Resuspend cells in 200 μL washing buffer.</li>
<li>Put cells on ice and analyze them on a flow cytometer within 24 h. Please see <a href="#fig2">Figure 2</a>C for suggested gating strategy.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1117-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Quality control of lymph node processing</div>
<p>(A) Lymph nodes were processed as described in the protocol and roughly grouped into four categories based on their size: fine needle biopsies, I (small piece), II (medium size piece), III (big piece) or IV (very big piece). The total number of harvested cells was compared to estimated size of the obtained lymph node piece. Box plots show minimum, first quartile, median, second quartile, and maximum.</p>
<p>(B) The total number of harvested cells across different lymphoma entities. Box plots show minimum, first quartile, median, second quartile, and maximum.</p>
<p>(C) Representative pseudocolor plots showing the usual output of harvested lymph node-derived lymphocytes.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.3">Thawing lymph node-derived cells and dead-cell removal</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60–90 min</div>
<p>Lymph node-derived cells are thawed using EDTA to prevent clumping of cells prior to removal of dead cells and library preparation for single cell sequencing. Until library preparation, the following steps are performed under sterile conditions.</p>
<ol start="26">
<li>Pre-warm water bath and EDTA-containing cell culture medium to 37°C.</li>
<li>Prepare 50 mL tubes with 10 mL of EDTA-containing cell culture medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> EDTA reduces the formation of cell clumps, and thereby increases the yield of viable cells after thawing.</div>
<ol start="28">
<li>Thaw frozen aliquots in pre-warmed water bath and quickly transfer cell suspension dropwise into the prepared tubes.</li>
<li>Centrifuge at 200 <i>g</i> for 5 min at 18°C–25°C and discard supernatant.</li>
<li>Resuspend cells in EDTA-free washing buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step aims to remove EDTA again from the cell suspension, as calcium is required for binding to the column of the dead-cell removal kit.</div>
<ol start="31">
<li>Filter cell suspensions through 75 μm strainer.</li>
<li>Count cells, for instance manually by means of trypan blue and a hemocytometer.</li>
<li>Centrifuge at 200 <i>g</i> for 5 min at 18°C–25°C and discard supernatant.</li>
<li>To magnetically label the dead cells, resuspend the cell pellet in 100 μL magnetic beads per 1 × 10<sup>7</sup> cells. For fewer than 1 × 10<sup>7</sup> cells, use 100 μL.</li>
<li>Incubate for 15 min protected from light at 18°C–25°C.</li>
<li>Meanwhile, prepare magnetic Multistand, OctoMACS™ separator, and one MS columns per sample (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1117-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Dead cell removal</div>
<p>(A) Magnetic multistand with OctoMACS™ separator and two MS columns.</p>
<p>(B) Columns are rinsed with 500 μL binding buffer (red arrow).</p>
<p>(C) To collect flow-through 15 mL tubes are placed beneath each column.</p>
<p>(D) 1000 μL binding buffer was add to each sample. Then, each sample (red arrow) is applied onto one column.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For single cell RNA sequencing, we strongly recommend the OctoMACS™ separator, as sequential handling of cells might confound gene expression profiles.</div>
<ol start="37">
<li>Wash each column with 500 μL 1X binding buffer (<a href="#fig3">Figure 3</a>B) and wait until the liquid has passed the column. Put a tissue beneath the columns to soak up the flow-through.</li>
<li>Place 15 mL tubes under each column to collect the flow-through (<a href="#fig3">Figure 3</a>C).</li>
<li>Add 1 mL 1X binding buffer to each sample and apply each sample onto one column (<a href="#fig3">Figure 3</a>D).</li>
<li>Wait until the liquid has passed the column before rinsing each column with 500 μL 1X binding buffer. Wait until the liquid has passed the column.</li>
<li>Perform a total of 4 washes in this way.</li>
<li>Centrifuge the collected flow-through at 200 <i>g</i> for 5 min at 18°C–25°C and discard supernatant.</li>
<li>Resuspend cells in washing buffer and from here on handle them at 4°C.</li>
<li>Count cells and assess viability, for instance by means of trypan blue and a hemocytometer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In 10%–15% of all samples that we processed by this protocol, viability did not exceed 85%. In these cases, we reevaluate continuation of the protocol. To avoid unexpected termination of the planned follow-up experiment, we usually thaw and process an additional sample as back-up. It should generally be considered that when thawing a vial of a vial of 1 × 10<sup>7</sup> cells, only a median of 3.5 × 10<sup>6</sup> viable cells can be recovered. Please see the <a href="#troubleshooting">troubleshooting</a> section on how to optimize conditions if you encounter low viability or exorbitant loss of cells.</div>
<div class="optional">
<span class="optional-title">Optional:</span> At this step, cells can be labelled with barcoded antibodies. We do not recommend staining before removal of dead cells.</div>
<ol start="45">
<li>Continue with library preparation for single cell sequencing according to the manufacturer’s protocol.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol describes how to prepare single cell suspensions from human lymph nodes without enzymatic digestions. The expected yield of cells strongly depends on the size of the processed lymph node (<a href="#fig2">Figure 2</a>A). Comparing the different B cell lymphoma entities, the lowest yield of cells is obtained from diffuse large B cell lymphoma (<a href="#fig2">Figure 2</a>B). Across several hundred lymph node samples, we harvested a median number of 9 × 10<sup>7</sup> cells per sample. Lymph node samples that we receive have an approximate size of 1.5–2.5 cm<sup>3</sup> and an average weight of 1.5–2.5 g. Flow cytometric analysis of isolated cells is indispensable to evaluate the quality and viability of isolated cells. <a href="#fig2">Figure 2</a>C illustrates representative pseudocolor plots of lymph node-derived lymphocytes of a patient with mantle cell lymphoma.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol was optimized to isolate lymphocytes as quickly as possible while reducing the level of cellular stress. For this reason, it includes neither enzymatic digestion nor harsh mechanical dissociation. This protocol was not designed for isolation or analysis of lymph node-derived myeloid cells. Among the isolated cells, the proportion of myeloid cells is below 5%. However, we have not systematically investigated whether this proportion is comparable to the proportion of myeloid cells <i>in vivo</i> or after enzymatic digestions of human lymph nodes.</p>
<p>This protocol is neither suitable to isolate or investigate lymph node-derived stroma cells, for which enzymatic digestion is required (<a href="#bib8">Takeda and Jalkanen, 2020</a>).</p>
<p>Another limitation is that fine needle biopsies, e.g., computer tomography-guided biopsies, might yield very low numbers of cells in the range of 1 × 10<sup>6</sup> (<a href="#fig2">Figure 2</a>A).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low cell yield during initial preparation (step 13)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>The number of isolated cells highly depends on the amount of lymph node tissue provided. In particular, ultrasound- or computer tomography-guided biopsies are only 1 mm thick and consequently yield very low numbers of cells.</p>
<p>Low yield of cells can also occur when lymph node tissue is mistaken for conjunctive tissue and then removed during the initial preparation (step 2). Differentiation of the tissue types (<a href="#fig1">Figure 1</a>B) by eye requires some experience. Alternatively, ensure that the excised lymph node tissue is already cleaned by surgeons or pathologists.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Low cell viability after initial preparation (step 13)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>There are multiple factors affecting the viability of lymph node-derived cells. To achieve the best possible results, reduce the timespan from excision to processing and freezing in the lab, e.g., by optimizing the transfer from operating room to laboratory. For susceptible assays, such as single cell (RNA) sequencing, we recommend to only use samples that were processed on the day of excision. However, it is possible to store lymph node samples for 12–16 h at 4°C if the harvested cells are to be used in protein, DNA, or flow cytometric assays.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Red cell pellet during initial preparation (step 12)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Similar to peripheral blood mononuclear cells, lymph node-derived lymphocytes can be contaminated by erythrocytes, which can potentially affect downstream experiments and results (<a href="#fig1">Figure 1</a>E). Potential solutions include gradient centrifugation of isolated cell suspensions or lysis of erythrocytes using a hypotonic solution. However, we discourage you from additional steps before susceptible assays, such as single cell (RNA) sequencing. More precisely, heterogeneous sample treatment can drastically confound gene expression profiles and introduce difficult-to-correct batch effects. Record any kind of deviation from the standard protocol and avoid using such samples for delicate assays.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Black/dark cell pellet during initial preparation (step 12)</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Black or dark cell pellets may be the result of tattoos in the draining region of the lymph node. As little is known about how the pigments may affect freezing of the cells and downstream experiments, isolated cells should be discarded.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Low cell yield or viability after removal of dead cells (step 44)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Ensure that cells were carefully resuspended before applying them to the column. Any clumps might drastically reduce the cellular output. In addition, low starting viability can also lead to low output or viability after dead cell removal.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bordas, M., Genard, G., Ohl, S., Nessling, M., Richter, K., Roider, T., Dietrich, S., Maass, K.K., and Seiffert, M. (2020). Optimized protocol for isolation of small extracellular vesicles from human and murine lymphoid tissues. Int. J. Mol. Sci. <i>21</i>, 5586. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref1">View at publisher</a></p>
<p id="bib2">Hallek, M., Cheson, B.D., Catovsky, D., Caligaris-Cappio, F., Dighiero, G., Dohner, H., Hillmen, P., Keating, M.J., Montserrat, E., Rai, K.R., et al. (2008). Guidelines for the diagnosis and treatment of chronic lymphocytic leukemia: a report from the International Workshop on Chronic Lymphocytic Leukemia updating the National cancer Institute-Working group 1996 guidelines. Blood <i>111</i>, 5446-5456. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref2">View at publisher</a></p>
<p id="bib3">Horna, P., Olteanu, H., Kroft, S.H., and Harrington, A.M. (2011). Flow cytometric analysis of surface light chain expression patterns in B-cell lymphomas using monoclonal and polyclonal antibodies. Am. J. Clin. Pathol. <i>136</i>, 954-959. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref3">View at publisher</a></p>
<p id="bib4">Mattei, D., Ivanov, A., van Oostrum, M., Pantelyushin, S., Richetto, J., Mueller, F., Beffinger, M., Schellhammer, L., Vom Berg, J., Wollscheid, B., et al. (2020). Enzymatic dissociation induces transcriptional and proteotype bias in brain cell populations. Int. J. Mol. Sci. <i>21</i>, 7944. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref4">View at publisher</a></p>
<p id="bib5">Quentmeier, H., Pommerenke, C., Dirks, W.G., Eberth, S., Koeppel, M., MacLeod, R.A.F., Nagel, S., Steube, K., Uphoff, C.C., and Drexler, H.G. (2019). The LL-100 panel: 100 cell lines for blood cancer studies. Sci. Rep. <i>9</i>, 8218. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref5">View at publisher</a></p>
<p id="bib6">Reichard, A. and Asosingh, K. (2019). Best practices for preparing a single cell suspension from solid tissues for flow cytometry. Cytometry A <i>95</i>, 219-226. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref6">View at publisher</a></p>
<p id="bib9">Roider, T., Brinkmann, B.J., Kim, V., Knoll, M., Kolb, C., Roessner, P.M., Bordas, M., Dreger, P., Mueller-Tidow, C., Huber, W., et al. (2021). Autologous culture model of nodal B-cell lymphoma identifies ex vivo determinants of response to bispecific antibodies. Blood Advances. . <a class="external-link" href="https://doi.org/10.1182/bloodadvances.2021005400">https://doi.org/10.1182/bloodadvances.2021005400</a></p>
<p id="bib7">Roider, T., Seufert, J., Uvarovskii, A., Frauhammer, F., Bordas, M., Abedpour, N., Stolarczyk, M., Mallm, J.P., Herbst, S.A., Bruch, P.M., et al. (2020). Dissecting intratumour heterogeneity of nodal B-cell lymphomas at the transcriptional, genetic and drug-response levels. Nat. Cell Biol. <i>22</i>, 896-906. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref7">View at publisher</a></p>
<p id="bib8">Takeda, A. and Jalkanen, S. (2020). Single-cell transcriptomics of human lymph node stroma. STAR Protoc. <i>1</i>, 100021. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00620-1/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources or reagents should be directed to the lead contact, Tobias Roider (<a href="mailto:tobias.roider@embl.de">tobias.roider@embl.de</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique materials or reagents.</p>
<h4>Data and code availability</h4>
<p>The study did not generate/analyze data sets or codes.</p>
<h3>Acknowledgments</h3>
<p>T.R. was supported by a physician scientist fellowship of the Medical Faculty of University Heidelberg. S.D. was supported by a grant of the <a href="https://doi.org/10.13039/100009162">Hairy Cell Leukemia Foundation</a>, the Heidelberg Research Centre for Molecular Medicine (HRCMM), and an e:med <a href="https://doi.org/10.13039/501100002347">BMBF</a> junior group grant and <a href="https://doi.org/10.13039/501100001659">Deutsche Forschungsgemeinschaft</a> through the SFB873 project B7. We thank Carolin Kolb, Angela Lenze, and Mareike Knoll (University Hospital Heidelberg) for their excellent technical assistance.</p>
<h3>Author contributions</h3>
<p>T.R. performed experiments; T.R. and S.D. designed and supervised the research; T.R., B.J.B., and S.D. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>