<article>
<section>
<h1 id="header">Isolation of human fetal intestinal cells for single-cell RNA sequencing</h1>
<p><time datetime="2021-10-29">Published: October 29, 2021</time></p>
<p>David Fawkner-Corbett,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Ana Sousa Gerós,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Agne Antanaviciute,<sup><a href="#aff1">1</a>,<a href="#aff4">4</a></sup> and Alison Simmons<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup></p>
<p id="aff1"><sup>1</sup>Medical Research Council (MRC) Human Immunology Unit, MRC Weatherall Institute of Molecular Medicine (WIMM), John Radcliffe Hospital, University of Oxford, Oxford OX3 9DS, UK</p>
<p id="aff2"><sup>2</sup>Translational Gastroenterology Unit, John Radcliffe Hospital, Oxford OX3 9DU, UK</p>
<p id="aff3"><sup>3</sup>Academic Paediatric Surgery Unit (APSU), Nuffield Department of Surgical Sciences, University of Oxford, Oxford OX3 9DU, UK</p>
<p id="aff4"><sup>4</sup>MRC WIMM Centre For Computational Biology, MRC Weatherall Institute of Molecular Medicine, John Radcliffe Hospital, University of Oxford, Oxford OX3 9DS, UK</p>
<p id="fn2"><sup>1</sup>Technical contact</p>
<p id="fn1"><sup>2</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:david.fawkner-corbett@imm.ox.ac.uk">david.fawkner-corbett@imm.ox.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100890">10.1016/j.xpro.2021.100890</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The intestine has a large number of cell types. Thus, digestion of pure and viable populations is necessary for downstream techniques including single-cell RNA sequencing. We outline a protocol to isolate both epithelial and non-epithelial cells from human fetal samples at high viability, which was used to produce a full thickness atlas of intestinal cells across human development. This protocol can also be adapted to adult endoscopy and surgical specimens.</p>
<p>For details on the use of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00596-7/sref1">Fawkner-Corbett et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Outlines preparation and digestion of intestinal tissue for scRNA-seq</li>
<li>Works with cryopreserved tissue</li>
<li>Adaptable to different tissue types such as biopsy and surgical tissue</li>
<li>Can generate samples for multiple downstream protocols</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1119-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> [1.5–3h]</div>
<div class="note">
<span class="note-title">Note:</span> All tissues should be collected following appropriate ethical and institutional approvals.</div>
<ol>
<li>Prepare the necessary buffers: Transport medium, HPGA, Wash medium, Chelation medium, Thaw medium and Buffer L replacement.</li>
<li>Set water bath to 37°C.</li>
<li>Set benchtop centrifuges to 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Set two centrifuges at 4°C. One for both 50 mL and 15 mL tubes, and a smaller one for 1.5 mL vials. All centrifuge steps in this protocol are performed at 300G, 4°C for 4 min.</div>
<ol start="4">
<li>If processing cryo-preserved tissue from liquid nitrogen storage, bring samples to a −80°C freezer for at least 1 h.</li>
<li>Thaw a 50 mL aliquot of sterile filtered Fetal Bovine Serum.</li>
<li>In a 24-well plate, add 700 μL of Buffer L replacement to one well and 2 mL of Buffer L replacement to another well. Repeat this for each sample being processed. Place the 24-well plate in an incubator set at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Protocol example is for fetal intestinal tissue, but can also be performed on adult endoscopic biopsies or surgical tissue with key adjustments mentioned in protocol. Compartment specific adaptations have also been used in adult tissue in (<a href="#bib3">Parikh et al., 2019</a>) and (<a href="#bib2">Kinchen et al., 2018</a>).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Sigma</td>
<td>Cat#P0781-100ML</td>
</tr>
<tr>
<td>HEPES Buffer Solution (1M)</td>
<td>Gibco</td>
<td>Cat#15630-056</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle’s Medium (DMEM)</td>
<td>Sigma</td>
<td>Cat#D5796-500ML</td>
</tr>
<tr>
<td>HBSS medium</td>
<td>Lonza</td>
<td>Cat#10–543F</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Sigma</td>
<td>Cat#F9665-500ML</td>
</tr>
<tr>
<td>Ultrapure 0.5M EDTA, pH 8.0</td>
<td>Invitrogen</td>
<td>Cat#15575-038</td>
</tr>
<tr>
<td>CryoStorCS10</td>
<td>Sigma</td>
<td>Cat#C2874-100ML</td>
</tr>
<tr>
<td>Phosphate-buffered saline (PBS, or similar)</td>
<td>Oxoid Ltd</td>
<td>Cat#BR0014G</td>
</tr>
<tr>
<td>ACK Lysis Buffer</td>
<td>Gibco</td>
<td>Cat#A1049201</td>
</tr>
<tr>
<td>TrypLE<sup>TM</sup> Express</td>
<td>Gibco</td>
<td>Cat#12605028</td>
</tr>
<tr>
<td>Deoxyribonuclease I from bovine pancreas (DNase)</td>
<td>Sigma</td>
<td>Cat#DN25-10mg</td>
</tr>
<tr>
<td>Trypan Blue Stain (0.4%)</td>
<td>Invitrogen</td>
<td>Cat# T10282</td>
</tr>
<tr>
<td>DAPI solution (dilution used 1:1500)</td>
<td>BD Pharmingen</td>
<td>Cat#564907</td>
</tr>
<tr>
<td>CD90(Thy1) FITC anti-human (dilution used 1:50)</td>
<td>BioLegend</td>
<td>Cat#328107 RRID: AB_893438</td>
</tr>
<tr>
<td>CD326(EpCAM) PE-vio770 (dilution used 1:50)</td>
<td>Milteyni Biotec</td>
<td>Cat# 130-099-742 RRID: AB_2660305</td>
</tr>
<tr>
<td>CD45 APC human (dilution used 1:75)</td>
<td>Miltenyi Biotec</td>
<td>Cat# 130-098-143 RRID: AB_2660416</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Costar 24-well cell culture plates (or similar)</td>
<td>Corning</td>
<td>Cat#3526</td>
</tr>
<tr>
<td>Blunt needle</td>
<td>Terumo</td>
<td>Cat#BN-1838</td>
</tr>
<tr>
<td>Terumo 5 mL syringe without needle (or similar)</td>
<td>Terumo</td>
<td>Cat#SS∗05SE1</td>
</tr>
<tr>
<td>VWR Cell Strainer 70 μm (or similar)</td>
<td>VWR</td>
<td>Cat#732-2758</td>
</tr>
<tr>
<td>Pre-Separation Filters 30 μm (or similar)</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-041-407</td>
</tr>
<tr>
<td>CoolCell<sup>TM</sup> LX Freezing Container (or similar)</td>
<td>Corning</td>
<td>Cat#BCS-405G</td>
</tr>
<tr>
<td>MACSmix™ Tube Rotator</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-090-753</td>
</tr>
<tr>
<td>Countess™ 3 Automated Cell Counter (or similar)</td>
<td>Thermo Fisher</td>
<td>Cat# AMQAX2000</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Transport medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM (high glucose)</td>
<td>n/a</td>
<td>490 mL</td>
</tr>
<tr>
<td>Penicillin / streptomycin</td>
<td>100 U/mL Penicillin, 0.1 mg/mL Streptomycin</td>
<td>5 mL</td>
</tr>
<tr>
<td>HEPES</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Total</td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>HPGA buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HBSS</td>
<td>n/a</td>
<td>490 mL</td>
</tr>
<tr>
<td>Penicillin / streptomycin</td>
<td>100U/mL Penicillin, 0.1 mg/mL Streptomycin</td>
<td>5 mL</td>
</tr>
<tr>
<td>HEPES</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Total</td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>Wash medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HPGA buffer</td>
<td>n/a</td>
<td>49.9 mL</td>
</tr>
<tr>
<td>EDTA (0.5M)</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>DTT (powdered)</td>
<td>1 mM</td>
<td>n/a (1 tube)</td>
</tr>
<tr>
<td>Total</td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>Chelation media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HPGA buffer</td>
<td>n/a</td>
<td>49 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>5 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>DTT</td>
<td>2 mM</td>
<td>n/a (2 tubes)</td>
</tr>
<tr>
<td>FBS</td>
<td>1%</td>
<td>500 μL</td>
</tr>
<tr>
<td>Total</td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>Thaw media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Transport Media</td>
<td>n/a</td>
<td>47.5 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Total</td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>Buffer L replacement</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM (high glucose)</td>
<td>n/a</td>
<td>46.25 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>HEPES</td>
<td>25 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>Total</td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> FBS should be sterile filtered through a 0.22 μm filter.</div>
<div class="note">
<span class="note-title">Note:</span> All buffers can be prepared up to 4 weeks before the experiment and stored at 4°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Precautions for reagents mentioned above include:</div>
<ul>
<li>EDTA causes serious irritation when in contact with the skin and eyes. To avoid this, wear protective gloves/clothing/eye and face protection, and wash skin thoroughly after handling.</li>
<li>DTT is harmful if swallowed and causes skin/eye/respiratory irritation. It is also harmful for aquatic life with long lasting effects. To avoid this, wear protective gloves/clothing/eye and face protection, and avoid breathing dust/fume/gas/mist/vapors/spray, preferably by handling inside a fume cabinet. Avoid release to the environment.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Processing of tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [&lt;30 min]</div>
<p>Prepare tissue for digestion protocol or storage.</p>
<ol>
<li>Receive intestinal tissue in 10 mL of transport medium on ice and proceed quickly with all dissection steps.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Transport of samples should be performed as quickly as possible but good results in viability are still seen up to 3 h after extraction of intestine from primary sample if kept on ice for the entirety of this process. Timing of dissection should aim to be completed in less than 30 min from starting to freezing or downstream processing, with all reagents pre-cooled on ice.</div>
<ol start="2">
<li>Place tissue on cutting board and examine fetal intestinal tissue under operating microscope. Identify anatomical landmarks for orientation (stomach, appendix, Meckel’s diverticulum) to inform selection of tissue for further processing.</li>
<li>Isolate and separate 2 cm regions of tissue from the anatomical compartments of interest (e.g., from terminal ileum, proximal colon and distal colon), or 1 cm in low gestation tissue (e.g., &lt;12 post conceptual weeks [pcw]).</li>
<li>Wash lumen with cold PBS to remove meconium. This can be achieved by flushing PBS into the lumen with the help of a blunt needle and syringe or with a pipette tip in larger samples. Wash with 500μL–1000μL depending on size of tissue for at least 3 washes. Late samples (e.g., after 15pcw) may require more to remove visible meconium. Early samples (e.g., &lt;12pcw) may not be large enough to permit a syringe or pipette tip, in which case proceed and the lumen can be cleaned by opening of shaking of the small fragments generated in latter steps.</li>
<li>Dissect each sample further by opening the lumen and cutting it into small (e.g., 0.2 cm<sup>2</sup>) tissue fragments.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A similar process can be performed with adult endoscopic biopsies or in adult surgical tissue, with the muscularis mucosae being removed and tissue being dissected into small fragments. In both instances the small fragments will then proceed in a similar manner to that outlined below.</div>
<ol start="6">
<li>Move all tissue fragments to tubes, washing each of them with 10 mL of cold transport media. Shake the tube and centrifuge.</li>
<li>[Optional] Tissue can be cryopreserved. For this, fully aspirate all transport media and resuspend the tissue fragments in 500 μL of Cryostor CS10. Place the tubes in a freezing container (e.g., CoolCell® or MrFrosty®) and move it to a −80°C freezer. This will benefit batch processing and multiplexed approaches, and it has shown good recovery of epithelial and non-epithelial fractions even after 6 months.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All dissection steps should be carried on ice using reagents pre-cooled on ice.</div>
<h3 id="sec3.2">Isolation of epithelial crypts</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h]</div>
<p>Obtain supernatant enriched for epithelial crypts through EDTA chelation.</p>
<ol start="8">
<li>Prepare tissue for chelation (This step is dependent on the processing method):
<ol type="a">
<li>If processing fresh tissue:
<ol type="i">
<li>Centrifuge the tubes containing tissue fragments in transport media.</li>
<li>Fully aspirate transport media and add 3 mL of ACK lysing buffer. Shake the tube and incubate it at room temperature (18°C–22°C) for 5 min.</li>
<li>Add 10 mL of wash media and centrifuge.</li>
<li>Fully aspirate wash media.</li>
<li>Repeat step iii then proceed to Step 9.</li>
</ol>
</li>
<li>If processing frozen tissue:
<ol type="i">
<li>Per sample, prepare a 50 mL tube containing 25 mL of thaw media using a serological pipette.</li>
<li>Obtain samples from −80°C freezer and transport them on ice to the water bath.</li>
<li>Thaw samples in the water bath until only small ice crystal remains (est. 2 min)</li>
<li>Immediately add thaw media to fill cryovial, mix x3, then transfer contents to the tube containing 25 mL of thaw media.</li>
<li>Agitate the tubes and centrifuge.</li>
<li>Aspirate thaw media and wash tissue with 10 mL of wash medium. Proceed to step 9.</li>
</ol>
</li>
</ol>
</li>
<li>Aspirate final wash media and replace with 10 mL of pre-warmed chelation media.</li>
<li>Shake then incubate tubes in a water bath for 15 min. There will be a total of three chelation steps of 15 min each during the epithelial isolation.</li>
<li>Vortex or shake tube every 5 min for 5–10 s.</li>
<li>After the last vortex, allow tissue to settle. Remove supernatant with FBS-coated pipette and transfer it to a new 15mL tube - crypt fraction 1. Place this tube on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A small amount of chelation media (est. 500 μL can be left in the two initial chelation stages, to preserve tissue remaining for subsequent repetition of chelation steps.</div>
<ol start="13">
<li>Add 5 mL of fresh chelation media to the tissue containing tube, and repeat steps 10–12. In the meantime, centrifuge the tube containing the crypt fraction 1. Resuspend small pellet in 2 mL cold transport medium in a 15 mL tube and place it on ice.</li>
<li>Once the second 15 min incubation is complete, repeat steps 12 and 13 to get crypt fraction 2. Pool with crypt fraction 1 and place the tube (now with 4mL transport media) on ice.</li>
<li>Repeat the chelation step for a final time by adding 5mL chelation medium and repeating steps 10–12 for crypt fraction 3.</li>
<li>In the meantime, during breaks between the EDTA chelation steps, make up stromal digestion enzymes (see below).</li>
<li>Repeat steps 12 and 13 (without re-adding chelation medium to the tissue) once again to obtain the third and last crypt fraction. Pool all 3 fractions (6 mL) in a 15 mL tube and centrifuge it.</li>
<li>Fully aspirate the all chelation media from the remaining tissue, and then transfer remaining tissue directly to stromal digestion in the incubator.</li>
<li>Resuspend all crypt fractions in 3 mL of warm TrypLE Express with 50 μg/mL DNAse, and move the tube into the incubator. Same volume of TrypLE is used for early and late gestation samples.</li>
<li>Incubate at 37°C for 35 min using continual agitation or rotation (e.g., MACSmix(Milteyni)).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Time crypt fraction incubation with stromal digestion: move remainder of crypt depleted tissue straight into the stromal digestion cocktail with blow out pipette (Step 18, prepared during chelation steps as detailed below).</div>
<h3 id="sec3.3">Digestion of crypt-depleted fraction and obtaining single epithelial cells</h3>
<ol start="21">
<li>During steps 14–17 of crypt isolate, prepare enzyme solution in the previously pre-warmed 24-well plate containing 700 μL of buffer L replacement.</li>
<li>Add the enzymes as follows: 100 μL of Enzyme D, 62.4 μL of Enzyme P, 6 μL of Enzyme B and 12 μL of Enzyme A. Small samples (e.g., &lt;14pcw, or endoscopic biopsies) may need only half volume of each enzyme in cocktail, subjected to optimization.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Enzyme concentrations and timings in this step can be optimized to volume of tissue processed or to time with the epithelial single cell isolation step by altering enzyme of buffer volumes.</div>
<ol start="23">
<li>Place the plate, with crypt depleted fraction added, in the incubator for 45 min, bringing it out every 15 min for mechanical dissociation using a blunt needle and a 2.5 mL syringe.</li>
<li>With the needle and syringe, pull up and flush down the sample repeatedly, washing the needle in the 2 mL buffer L replacement well in between uses. Samples may not pass through blunt needle initially.</li>
<li>After 45 min – to time with epithelial isolation - look at the sample under a microscope and check for tissue disruption and single cell suspension (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1119-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Example of timing of non-epithelial fraction digestion</div>
<p>Digestion of the crypt depleted fraction as seen by microscopy at 5 (left) and 40 min (right) in 24 well plate over 40 min. Right image demonstrated good dissociation of single cells at which stage digestion should be stopped (scale bar with respective measurement shown).</p>
</figcaption>
</figure>
</li>
<li>Collect all stromal and epithelial suspensions. Wash each tube with thaw media and centrifuge. In the meantime, place 70 μm cell strainers on top of 50 mL Falcons – 1 per condition - and pre-wet the filter membrane using FBS.</li>
<li>Vigorously resuspend each pellet in 1 mL of thaw media and flush it through the filter. Add thaw media up to 10 mL and centrifuge. Repeat the washing step.</li>
<li>Resuspend pellet in 1 mL of thaw media and flush it through a 30 μm strainer pre-wetted with FBS. Wash through with 3 mL of thaw media if using sample for downstream culturing/FACS, or with PBS+ 0.04% BSA for scRNA-seq. Centrifuge.</li>
<li>Resuspend cell pellet in 250 μL media or PBS+0.04% BSA depending on downstream application and transfer exactly 250 μL to a new Eppendorf.</li>
<li>Take sample for cell counting (e.g., 10 μL). Mix it with 10 μL of 0.1% Trypan Blue and add it to an Invitrogen Countess slide or view under hemocytometer for manual counting of cells and viability.</li>
<li>Single cell suspension can then proceed to further downstream use; loading for scRNA-seq on 10× Chromium platform, antibody staining for flow cytometry or scRNA-seq multiplexing or other desired experiments.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Step 28 is optional and can be omitted depending on downstream use of the cells (e.g., in culture or for flow cytometry) but is essential for scRNA-seq.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Expected yields of cells can vary depending on the gestational age of sample and the amount of tissue used in the initial dissection step. At initial count after isolation the epithelial depleted fraction is usually of higher cell number and viability, using a hemocytometer or automated cell counter. This can also be visualized in the cell pellet before filtering (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1119-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Cell pellet obtained from digestion of both compartments</div>
<p>Example cell pellet obtained from the epithelial fraction (left) and crypt-depleted fraction (right) at end of protocol, with less epithelial cells obtained which is expected.</p>
</figcaption>
</figure>
<p>For a recent digestion of n=25 samples from 8–22 post conceptual weeks, viabilities and total cell counts were assessed using a Countess II (ThermoFisher, example <a href="#fig3">Figure 3</a>.1a) both straight after the digestion protocol and then following a 30-min antibody stain to attach hashtag oligonucleotide antibodies with associated washes before undergoing multiplexed scRNA-seq (<a href="#bib1">Fawkner-Corbett et al., 2021</a>; <a href="#bib5">Stoeckius et al., 2018</a>). There is a significant drop in recovery and viability during the staining process, but the majority had viabilities &gt;70% for epithelial and &gt;80% for non-epithelial and were able to be run on the 10× Genomics platform with good result (Epithelial fraction: Mean recovery of viable cells after stain 230,122 cells (S.E.M 61,766) and viability 75.04% (S.E.M. 2.13%); non-epithelial fraction: Mean recovery of viable cells 555,938 cells (S.E.M. 79,324) and viability 81.92% (S.E.M. 1.95%) <a href="#fig3">Figure 3</a>.1b).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1119-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Assessment of final digested single cell product</div>
<p>(A) Example image from countess II (ThermoFisher) with non-epithelial fraction in single cell suspension showing good viability and dissociation (Dilution volume 250 μL, count profile set previously with hemocytometer).</p>
<p>(B) Overview of total number of viable cells and percentage viability from immediately after digestion and after hashtag oligo antibody staining protocol (<a href="#bib5">Stoeckius et al., 2018</a>) in n=25 samples of fetal intestine (Error bar representing mean +- standard error of the mean (SEM)).</p>
</figcaption>
</figure>
<p>Flow cytometry has also demonstrated that the enriched populations are highly purified, with only a small percentage of stromal cells (EPCAM-, CD90+) being identified in the crypt population and epithelial cells (EPCAM+) in the crypt depleted proportion (<a href="#fig4">Figure 4</a>). Similar results have been shown from healthy biopsied taken from adults during endoscopy although the proportion of immune cells (CD45+) can be much greater in both of the digested fractions, especially in the context of IBD associated inflammation (Data not shown).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1119-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Purity of digestion fractions assessed by flow cytometry</div>
<p>(A) Example gating strategy to quantify singlets (SSC-A vs FSC-A, not shown) which were viable (DAPI-) and identified as in epithelial (EpCAM+CD45-), immune (EPCAM-CD45+) or stromal (EPCAM-CD45-) compartments and quantified (B). (Epithelial digestion fraction top, non-epithelial digestion fraction bottom, n=3 repeats, error bar representing mean +SEM)</p>
</figcaption>
</figure>
<p>This protocol is designed to recover both the epithelial and non-epithelial populations containing all cell types in the full thickness of intestine. Further alterations can be performed if targeting one cell type of interest. This could include undertaking only the first part of protocol to obtain epithelial cells(<a href="#bib3">Parikh et al., 2019</a>).Alternatively, to obtain an enriched intestinal fibroblast population, the second enzyme digestion could be performed on the whole tissue, followed by bead separation of epithelial (CD326), immune (CD45) and erythrocyte (CD235a) populations using antibody conjugated beads (Miltenyi) as per(<a href="#bib2">Kinchen et al., 2018</a>). A similar method could select immune (CD45) cells by positive selection, or flow cytometry to sort a population of interest.</p>
<p>The epithelial fraction can be cultured to form epithelial organoids by placing the isolated cells into Matrigel in line with previous publications (<a href="#bib4">Sato et al., 2011</a>). The antibiotics contained in the media mean that, in our experience, these will culture without infection.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The number of cells obtained is dependent of the amount of tissue dissociated. This protocol has been validated on low gestation samples (e.g., 8pcw) with minimal intestinal tissue, as well as on 2 endoscopic biopsies. In these cases of low tissue input the final yield of epithelial cells is expected to be lower than the crypt-depleted fraction, but epithelial cells should still be obtained. Reduction of dilution volume to 100 μL in the penultimate step of “Digestion of crypt depleted fraction and obtaining single epithelial cells” (step 29) is advised.</p>
<p>A final viability check should be performed to ensure high viability (e.g., &gt;70% in epithelium and &gt;80% in non-epithelial fraction, although at discretion of user) before proceeding to scRNA-seq.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>When counting cells, or looking at them under microscope after digestion, there is a large amount of doublets and cell clumps in the sample (Identified at Step 30 of section “Digestion of crypt depleted fraction and obtaining single epithelial cells”).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>This is most commonly seen in the epithelial fraction, and can be solved by extending the TrypLE digestion for 5–10 more minutes and/or by adding an extra wash and filtering step to the end of the protocol. If necessary, adding 200 μL of buffer L replacement to the non-epithelial digestion can extend the umbilical cord kit (Milteyni) digestion and help account for the additional time required by epithelial fraction so that digestions of both fractions are completed together. This must be balanced with viability of final sample however.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The viability of cells in the final digested tissue sample appears to be too low. (Identified at Step 30 of section “Digestion of crypt depleted fraction and obtaining single epithelial cells”)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Again, this is more often seen in the epithelial fraction. Proceeding promptly through the protocol and reducing dissection time before starting the protocol is critical. Transporting and dissecting samples on ice is important. For fetal tissue it was found that the majority of samples will achieve a viability of &gt;70% for epithelial fraction and &gt;80% for non-epithelial (<a href="#fig3">Figure 3</a>). Viability can be further improved by use of live/dead bead separation kit (Miltenyi) or by flow cytometry sorting on a viability dye (DAPI or Zombie fixable violet) but has the limitation of a lower cell recovery. Use of frozen or cryopreserved samples thawed as described in this protocol has shown to have no significant effect on final viability, and has been validated in the more delicate epithelial fraction by flow cytometry and in pilot runs of scRNA-seq to compare cell type proportions (<a href="#fig5">Figure 5</a> and <a href="#bib1">Fawkner-Corbett et al., 2021</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1119-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Comparable purity of cryopreserved cells</div>
<p>(A) Flow cytometry was performed on epithelial digestion fraction as per example gating strategy in <a href="#fig4">Figure 4</a>A and cell populations quantified (n=3 frozen samples, n=4 fresh tissue from same tissue section, error bar representing mean +-SEM).</p>
<p>(B) In a scRNA-seq pilot run fresh and cryopreserved (x-axis) samples from early (12pcw) and late (19pcw, y-axis) intestine were run together with comparison of cell populations (Data represents samples re-analyzed from <a href="#bib1">Fawkner-Corbett et al., 2021</a>).</p>
</figcaption>
</figure>
<h3 id="sec6.5">Problem 3</h3>
<p>The total/live number of cells seems erroneous (Identified at Step 30 of section “Digestion of crypt depleted fraction and obtaining single epithelial cells”).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>To add confidence at final cell counting, it is optimum to repeat it twice, especially when exact enumeration is vital such as when loading the cells directly in single cell protocols. In instances where duplicate counts are vastly different, this is often due to low cell recovery and most often seen in low gestational samples or after antibody staining steps. The final dilution can be changed to 100 μL to improve cell counting accuracy. In addition, if using an automated cell counter, which are often optimized to quantify peripheral blood mononuclear cells, calibration of the pre-set settings with a manual hemocytometer can help improve speed and accuracy.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Large amount of mucus (seen on microscope or next to pellet) in the epithelial digestion (Identified by eye or looking at crypts under microscope after step 17 of section “Isolation of epithelial crypts”).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Sometimes mucus can be seen in the epithelial crypt fraction as a light, “fluffy” substance above the pellet either macroscopically, under the microscope when checking digestion or when counting. This appears to have no effect on the digestion process and it is lost in filtering steps. However, it can give the impression of losing the cell pellet. This can be removed by increasing the concentration of DTT in the chelation media.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>My cell type of interest does not appear using cryopreserved tissue (Identified by staining or by analysis at end of protocol)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>It is our experience that use of cryopserved tissue gives the full representation of cell types across the thickness of intestine with comparable mRNA signatures to previous freshly processed samples and allows for multiplexing of samples to increase throughput. This was of use for charting all cell types across developmental time. It is feasible that some cell types may be enriched or depleted with this method, and the cost of scRNA-seq means that direct comparisons of both are challenging although an example is given (<a href="#fig5">Figure 5</a>). If there is concern that cryopreservation depletes a cell type of interest, or reduces the response of a cell type to stimulation afterward then the protocol can still be used to process samples fresh as detailed in section “processing of tissue”.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Fawkner-Corbett, D., Antanaviciute, A., Parikh, K., Jagielowicz, M., Gerós, A.S., Gupta, T., Ashley, N., Khamis, D., Fowler, D., Morrissey, E., et al. (2021). Spatiotemporal analysis of human intestinal development at single-cell resolution. Cell <i>184</i>, 810-826.e23. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00596-7/sref1">View at publisher</a></p>
<p id="bib2">Kinchen, J., Chen, H.H., Parikh, K., Antanaviciute, A., Jagielowicz, M., Fawkner-Corbett, D., Ashley, N., Cubitt, L., Mellado-Gomez, E., Attar, M., et al. (2018). Structural remodeling of the human colonic mesenchyme in inflammatory bowel disease. Cell <i>175</i>, 372-386.e17. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00596-7/sref2">View at publisher</a></p>
<p id="bib3">Parikh, K., Antanaviciute, A., Fawkner-Corbett, D., Jagielowicz, M., Aulicino, A., Lagerholm, C., Davis, S., Kinchen, J., Chen, H.H., Alham, N.K., et al. (2019). Colonic epithelial cell diversity in health and inflammatory bowel disease. Nature <i>567</i>, 49-55. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00596-7/sref3">View at publisher</a></p>
<p id="bib4">Sato, T., Stange, D.E., Ferrante, M., Vries, R.G.J., Van Es, J.H., Van Den Brink, S., Van Houdt, W.J., Pronk, A., Van Gorp, J., Siersema, P.D., et al. (2011). Long-term expansion of epithelial organoids from human colon, adenoma, adenocarcinoma, and Barrett’s epithelium. Gastroenterology <i>141</i>, 1762-1772. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00596-7/sref4">View at publisher</a></p>
<p id="bib5">Stoeckius, M., Zheng, S., Houck-Loomis, B., Hao, S., Yeung, B.Z., Mauck, W.M., Smibert, P., and Satija, R. (2018). Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. Genome Biol. <i>19</i>, 1-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00596-7/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests should be directed to and will be fulfilled by the lead contact, David Fawkner-Corbett (<a href="mailto:David.fawkner-corbett@imm.ox.ac.uk">David.fawkner-corbett@imm.ox.ac.uk</a>)</p>
<h4>Materials availability</h4>
<p>This study did not generate any unique reagents.</p>
<h4>Data and code availability</h4>
<p>This protocol did not in itself generate any data or code availability. A recent example of data generated from this technique for scRNA-seq is publicly available (<a href="#bib1">Fawkner-Corbett et al., 2021</a>) and deposited on GEO (GEO: GSE158702).</p>
<h3>Acknowledgments</h3>
<p>Material used in the development of this protocol was kindly collected and provided by the Human Developmental Biology Resource (<a href="https://www.hdbr.org">https://www.hdbr.org</a>) and project funding from the joint <a href="https://doi.org/10.13039/501100000265">MRC</a>/Wellcome Trust developmental grant (MR/R006237/1). This protocol entails the support of a large number of individuals who contributed the original publication (<a href="#bib1">Fawkner-Corbett et al., 2021</a>) who we acknowledge the help of and we thank. We acknowledge the help of the WIMM Flow Cytometry Facility, WIMM Single Cell Facility, and WIMM Sequencing Facility. The graphical abstract was generated using <a href="https://biorender.com">https://biorender.com</a>. Funding for individuals was supported by an <a href="https://doi.org/10.13039/501100000272">NIHR</a>-Senior Investigator Award and Wellcome Investigator Award (219523/Z/19/Z to A.S.), the UK Medical Research Council (A.S.), and the <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> Doctoral Training Centre (216419/Z/19/Z to D.F.-C.). For the purpose of Open Access, the author has applied a CC BY public copyright licence to any Author Accepted Manuscript version arising from this submission.</p>
<h3>Author contributions</h3>
<p>D.F.-C. outlined and optimized protocol methods alongside A.A. A.S.G. tested protocol and generated data for write-up. A.S. supervised, conceived, and funded methods development and work that generated protocol design. All authors were involved in manuscript writing and generation.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>