<article>
<section>
<h1 id="header">CRISPR deactivation in mammalian cells using photocleavable guide RNAs</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Roger S. Zou,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> Yang Liu,<sup><a href="#aff2">2</a></sup> and Taekjip Ha<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biomedical Engineering, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA</p>
<p id="aff2"><sup>2</sup>Department of Biophysics and Biophysical Chemistry, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA</p>
<p id="aff3"><sup>3</sup>Department of Biophysics, Johns Hopkins University, Baltimore, MD 21205, USA</p>
<p id="aff4"><sup>4</sup>Howard Hughes Medical Institute, Baltimore, MD 21205, USA</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:rzou4@jhmi.edu">rzou4@jhmi.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:tjha@jhu.edu">tjha@jhu.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100909">10.1016/j.xpro.2021.100909</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The ability to deactivate CRISPR-Cas systems on demand would improve the safety and applicability of genome editing. Here, we detail a protocol using photocleavable guide RNAs (pcRNAs) to deactivate CRISPR-Cas9 inside cells. We verify that deactivation is both rapid and complete by checking for insertion-deletion (indel) mutations using Sanger sequencing. This protocol will be useful for researchers interested in using pcRNAs to improve genome editing specificity, characterize the timescales of genome editing, and study cellular DNA damage responses.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00615-8/sref5">Zou et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol using photocleavable guide RNAs to deactivate CRISPR-Cas9 inside cells</li>
<li>Deactivation is both rapid and complete by checking for insertion-deletion mutations</li>
<li>Useful for characterizing genome editing dynamics and DNA repair processes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1112-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps using the Cas9 nuclease in HEK293T cells targeting a sequence near the <i>DNMT3B</i> gene. The gRNA sequence is called <i>HEK293 site 4</i> (or <i>HEK site 4</i>) (<a href="#bib2">Hu et al., 2018</a>). However, we have also used this protocol targeting other genomic sequences, with cytosine base editors and in K562 and induced pluripotent stem cells. Sanger sequencing is used for readout of Cas9 editing, but we have also used high-throughput amplicon sequencing with an Illumina MiSeq.</p>
<h3 id="sec1.1">Preparing oligonucleotides for polymerase chain reaction (PCR)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>Dilute dried oligonucleotide primers to make 100 μM stock in IDTE pH 8.
<ol type="a">
<li>Pulse spin tubes containing dried DNA so that the DNA is in the bottom of the tube.</li>
<li>Add the expected volume of IDTE pH 8 into tubes containing dried primers (<i>HEKs4_F</i>, <i>HEKs4_R</i> from <a href="#tbl1">Table 1</a>) to have a concentration of 100 μM.
<table id="tbl1">
<caption>Table 1. Synthetic DNA sequences</caption>
<thead>
<tr>
<th>Name</th>
<th>Sequence (5′ to 3′)</th>
</tr>
</thead>
<tbody>
<tr>
<td>TracrRNA</td>
<td>AGCAUAGCAAGUUAAAAUAAGGCUAGUCCGUUAUCAACUUGAAAAAGUGGCACCGAGUCGGUGCUUU</td>
</tr>
<tr>
<td>Pcl5_HEKsite4</td>
<td>GGCAC/iPC-Linker/GCGGCUGGAGGUGGGUUUUAGAGCUAUGCUGUUUUG</td>
</tr>
<tr>
<td>gRNA_HEKsite4</td>
<td>GGCACTGCGGCTGGAGGTGGGUUUUAGAGCUAUGCU</td>
</tr>
<tr>
<td>HEKs4_F</td>
<td>CCAGTGGTTCAATGGTCATCC</td>
</tr>
<tr>
<td>HEKs4_R</td>
<td>GGCCAGTGAAATCACCCTG</td>
</tr>
</tbody>
</table>
</li>
<li>Briefly vortex and pulse spin on a microfuge.</li>
<li>Label tube accordingly.</li>
</ol>
</li>
<li>Mix forward and reverse primers, then dilute to 5 μM with nuclease free water.
<ol type="a">
<li>Add 10 μL of 100 μM forward primer stock (<i>HEKs4_F</i>) to Eppendorf tube.</li>
<li>Add 10 μL of 100 μM reverse primer stock (<i>HEKs4_R</i>) to Eppendorf tube.</li>
<li>Add 180 μL of nuclease free water for 5 μM working stock (i.e., 10 μM of total oligonucleotide DNA), store in 4°C.</li>
<li>Store 100 μM primer stocks in −20°C.</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Preparing Cas9 photocleavable crRNA (pcRNA) and tracrRNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="3">
<li>Dilute dried RNA to make 100 μM stock in Nuclease Free Duplex Buffer.
<ol type="a">
<li>Pulse spin tubes containing dried RNA so that the RNA is in the bottom of the tube.</li>
<li>Add the expected volume of Nuclease Free Duplex Buffer into two tubes containing dried photocleavable crRNA (the <i>Pcl5_HEKsite4</i> pcRNA from <a href="#tbl1">Table 1</a>, targeting <i>DNMT3B</i>) and tracrRNA (<a href="#tbl1">Table 1</a>) to each have a concentration of 100 μM.</li>
<li>Briefly vortex and pulse spin on a microfuge.</li>
<li>Label tube accordingly, store in −80°C.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For light-sensitive pcRNAs, be sure to cover with foil as much as possible. Store covered in foil or in a black microcentrifuge tube.</div>
<h3 id="sec1.3">Preparing cells for electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<ol start="4">
<li>Culture HEK293T cells following culture conditions in <a href="https://www.atcc.org/products/all/crl-3216.aspx">ATCC</a>.</li>
<li>Ensure culture of 1–10 million cells at approximately 80% confluency at the time of electroporation.</li>
<li>Cells should also be routinely tested for mycoplasma infection.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Alt-R® S.p. Cas9 Nuclease V3</td>
<td>Integrated DNA Technologies</td>
<td>Cat#1081059</td>
</tr>
<tr>
<td>IDTE pH 8.0 (1<b>×</b> TE Solution)</td>
<td>Integrated DNA Technologies</td>
<td>Cat#11-05-01-13</td>
</tr>
<tr>
<td>Nuclease Free Duplex Buffer</td>
<td>Integrated DNA Technologies</td>
<td>Cat#11-01-03-01</td>
</tr>
<tr>
<td>Corning™ DMEM with L-Glutamine, Glucose, Sodium Pyruvate</td>
<td>Fisher Scientific</td>
<td>Cat#MT10013CV</td>
</tr>
<tr>
<td>Corning™ Regular Fetal Bovine Serum</td>
<td>Fisher Scientific</td>
<td>Cat#MT35011CV</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15070063</td>
</tr>
<tr>
<td>3 M Sodium Acetate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R1181</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.05%), phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#25300062</td>
</tr>
<tr>
<td>Collagen I, rat tail</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A1048301</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>SF Cell Line 4D-Nucleofector™ X Kit S (32 RCT)</td>
<td>Lonza</td>
<td>Cat#V4XC-2032</td>
</tr>
<tr>
<td>DNeasy Blood &amp; Tissue Kit (50)</td>
<td>QIAGEN</td>
<td>Cat#69504</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit (50)</td>
<td>QIAGEN</td>
<td>Cat#28104</td>
</tr>
<tr>
<td>Q5® Hot Start High-Fidelity 2<b>×</b> Master Mix</td>
<td>New England BioLabs</td>
<td>Cat#M0494</td>
</tr>
<tr>
<td>E-Gel™ EX Agarose Gels, 2%</td>
<td>Invitrogen</td>
<td>Cat#G402022</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Human: HEK293T</td>
<td>ATCC</td>
<td>Cat#CRL-3216</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Alt-R® CRISPR-Cas9 crRNA (sequence ‘gRNA_HEKsite4’ in <a href="#tbl1">Table 1</a>)</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>Alt-R® CRISPR-Cas9 tracrRNA, 100 nmol</td>
<td>Integrated DNA Technologies</td>
<td>Cat#1072534</td>
</tr>
<tr>
<td>Alt-R® Cas9 Electroporation Enhancer, 10 nmol</td>
<td>Integrated DNA Technologies</td>
<td>Cat#1075916</td>
</tr>
<tr>
<td>tracrRNA (sequences in <a href="#tbl1">Table 1</a>)</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>photocleavable crRNA (sequence ‘Pcl5_HEKsite4’ in <a href="#tbl1">Table 1</a>)</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>HEKs4_F (sequences in <a href="#tbl1">Table 1</a>)</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>HEKs4_R (sequences in <a href="#tbl1">Table 1</a>)</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>SnapGene Viewer</td>
<td>SnapGene</td>
<td><a href="https://www.snapgene.com/snapgene-viewer/">https://www.snapgene.com/snapgene-viewer/</a></td>
</tr>
<tr>
<td>TIDE</td>
<td><a href="#bib1">Brinkman et al. (2014)</a></td>
<td><a href="http://shinyapps.datacurators.nl/tide/">http://shinyapps.datacurators.nl/tide/</a></td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Original/source data in the paper is available</td>
<td>This paper</td>
<td>Mendeley Data <a href="https://data.mendeley.com/datasets/w55kctmzzc">https://data.mendeley.com/datasets/w55kctmzzc</a>
</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>4D-Nucleofector Core Unit</td>
<td>Lonza</td>
<td>Cat#AAF-1002B</td>
</tr>
<tr>
<td>4D-Nucleofector X Unit</td>
<td>Lonza</td>
<td>Cat#AAF-1002X</td>
</tr>
<tr>
<td>Nalgene™ Rapid-Flow™ Sterile Single Use Vacuum Filter Units</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#09-741-02</td>
</tr>
<tr>
<td>Standard Hemocytometer</td>
<td>Weber Scientific</td>
<td>Cat#3048-12</td>
</tr>
<tr>
<td>C1000 Touch Thermal Cycler</td>
<td>Bio-Rad</td>
<td>Cat#1851148</td>
</tr>
<tr>
<td>Aluminum Foil</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>LED flashlight</td>
<td>JAXMAN</td>
<td>Cat#B06XW7S1CS (<a href="https://www.amazon.com/JAXMAN-Ultraviolet-365nm-Detector-Flashlight/dp/B06XW7S1CS/">https://www.amazon.com/JAXMAN-Ultraviolet-365nm-Detector-Flashlight/dp/B06XW7S1CS/</a>)</td>
</tr>
<tr>
<td>NanoDrop™ 2000 UV-Vis spectrophotometer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#ND-2000</td>
</tr>
<tr>
<td>E-Gel® iBase™ Power System with E-Gel™ Safe Imager™ Real-Time Transilluminator</td>
<td>Invitrogen</td>
<td>Cat#G6465</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p><b>Lonza nucleofector system for electroporation of Cas9:</b> For delivery of Cas9/gRNA particles into HEK293T cells using electroporation, we use the Lonza nucleofector system. The Lonza system allows electroporation in different volumes (e.g., 20 μL or 100 μL), and previous literature have optimized the electroporation parameters for each cell line. This protocol is written for electroporation of 800,000 HEK293T cells in a 20 μL volume, using <i>‘SF Cell Line 4D-Nucleofector™ X Kit S’</i>. Alternative strategies include Neon transfection system (Thermo Fisher) or Gene Pulser Xcell (Bio-Rad).</p>
<p><b>LED flashlight for deactivation:</b> We used a <a href="https://www.amazon.com/JAXMAN-Ultraviolet-365nm-Detector-Flashlight/dp/B06XW7S1CS/">LED flashlight (JAXMAN</a>) purchased from <a href="http://Amazon.com">Amazon.com</a>. There are many alternatives available on <a href="http://Amazon.com">Amazon.com</a> and other retailers. Ensure that the marketed wavelength is 365 nm.</p>
<p><b>Bio-Rad Thermo Cycler for PCR:</b> To generate samples for Sanger sequencing, we performed PCR on the Bio-Rad C1000 Touch Thermo Cycler. Numerous commercial alternatives exist.</p>
<p><b>Gel electrophoresis to evaluate PCR outcome:</b> We use E-Gel® EX Gel, 2% pre-cast gels on the E-Gel® iBase™ Power System/E-Gel™ Safe Imager™ Real-Time Transilluminator. Any gel electrophoresis system, including self-cast gels, will suffice.</p>
<table id="undtbl1">
<caption>DMEM complete</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>89%</td>
<td>445 mL</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>100%</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store in 4°C, maximum time of 1 year.</p>
<div class="optional">
<span class="optional-title">Optional:</span> Use vacuum filter unit (Nalgene 0.2 μm Polyethersulfone) to ensure sterility of media before storage.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Formation of Cas9/pcRNA ribonucleoprotein (RNP) complex</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>This step forms the ribonucleoprotein (RNP) complex composed of Cas9 in complex with the light-deactivatable pcRNA and tracrRNA. This RNP is allowed to stably form before direct delivery into the cell.</p>
<ol>
<li>Anneal light-deactivatable pcRNA with tracrRNA.
<ol type="a">
<li>Combine 1.2 μL pcRNA targeting <i>DNMT3B</i> with 1.2 μL tracrRNA (both at 100 μM in Nuclease Free Duplex Buffer) in PCR tube to form 2.4 μL of 50 μM annealed gRNA. Mix by pipetting up and down 5–10 times.</li>
<li>Heat at 95°C in thermocycler for 3 min, then cool on benchtop (room temperature; 20°C) for 5 min.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cover tubes with aluminum foil whenever possible and be sure to avoid direct sun exposure. This would minimize unintended deactivation of pcRNA.</div>
<ol start="2">
<li>Formation of RNP complex and Lonza nucleofection mixture.
<ol type="a">
<li>Add Cas9 to annealed gRNA, mix by pipetting up and down 5–10 times, add PBS, then mix by pipetting again.
<table id="undtbl2">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Annealed gRNA (50 μM)</td>
<td>2.4 μL</td>
</tr>
<tr>
<td>Cas9 (10 mg/mL)</td>
<td>1.7 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>1.9 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>5 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Cover the formed RNP complex with aluminum foil and leave at benchtop in room temperature (“RNP” in <a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1112-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Cas9 electroporation protocol into HEK293T cells</div>
<p>(A) Individual reagents necessary for electroporation, including Cas9/pcRNA RNP, EP enhancer, SF mix, and cuvette.</p>
<p>(B) Dissociating the cells using Trypsin, and the cells after mixing to the required single cell suspension before electroporation.</p>
<p>(C) After pelleting single cell suspension and resuspending in 1 mL PBS.</p>
<p>(D) After mix cell pellet with RNP solution.</p>
<p>(E) Electroporation settings on the Lonza electroporator.</p>
<p>(F) Beginning the electroporation after placing the cuvette in the machine.</p>
<p>(G) Zoom-in of the cuvette placement in the holder from panel f. Note the visible yellow pin that is visible only with the correct orientation (red box).</p>
<p>(H) Screen display a successful electroporation.</p>
</figcaption>
</figure>
</li>
<li>Open the Lonza SF kit (<a href="#fig1">Figure 1</a>A), which includes a cuvette, SF nucleofection reagent, and Supplement 1. In a separate tube, combine 16.4 μL of SF nucleofection reagent with 3.6 μL of Supplement 1 to form 20 μL of mixed nucleofection reagent (“SF mix” in <a href="#fig1">Figure 1</a>A). Mix by pipetting up and down 5–10 times.</li>
<li>Proceed to prepare cells for electroporation in cell culture hood.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">RNP delivery into HEK293T cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This step delivers the Cas9/pcRNA RNP into HEK293T cells using electroporation (<a href="#bib3">Liu et al., 2020</a>). The RNP is first mixed thoroughly with the cells, then an electric pulse of optimal voltage is applied. Nanometer-scale holes in the cell membrane are formed, allowing the charged RNP particles to enter the cell. All steps are performed inside a tissue culture hood unless stated otherwise.</p>
<ol start="3">
<li>Prepare 48-well plates with collagen.
<ol type="a">
<li>Prepare 1 mL of 1:100 collagen solution by mixing 10 μL of collagen with 1 mL of ice-cold PBS.</li>
<li>Add enough of 1:100 collagen to cover three wells of a 48-well plate (approximately 0.3 mL per well). Make sure the three wells are as spaced out as possible if on one plate. Incubate in 37°C cell culture incubator for 10 min.
<div class="note">
<span class="note-title">Note:</span> To better ensure that deactivation light does not affect more than the intended well, we suggest using one well each in different 48-well plates. Here, we use multiple wells in one 48-well plate for ease of presentation. Other options include using black plates that prevent deactivation light penetration or reflection.</div>
</li>
<li>Take out the plate from the incubator, move to inside the cell culture hood, remove the collagen solution, then let it dry in the cell culture hood with lid slightly ajar.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Collagen-coated plates allow for rapid cell adherence after electroporation.</div>
<ol start="4">
<li>Prepare 800,000 HEK293T cells for electroporation.
<ol type="a">
<li>Heat DMEM complete, PBS, and 0.05% Trypsin-EDTA in 37°C water bath until the solutions reach 37°C (usually takes at least 10 min).</li>
<li>Aspirate medium of cell culture (cells most easily grown on 1:100 collagen-coated 6-well or 10 cm plates).
<div class="note">
<span class="note-title">Note:</span> 60%–90% confluency is recommended for electroporation. A fully confluent well of a 6-well plate holds around 1 million HEK293T cells. A fully confluent 10 cm plate holds around 10 million HEK293T cells.</div>
</li>
<li>Add enough pre-warmed 0.05% Trypsin-EDTA into the culture flask to completely cover cells (0.6 mL to a well of a 6-well plate, or 3 mL to a 10 cm plate).</li>
<li>Incubate for 5 min in 37°C cell culture incubator to fully dissociate cells (<a href="#fig1">Figure 1</a>B).
<div class="critical">
<span class="critical-title">Critical:</span> Efficient transfection requires single cell suspension, which is attained with sufficiently long exposure to Trypsin.</div>
</li>
<li>Add equal volumes of DMEM complete, gently mix by pipetting 5–10 times to single cell suspension, then collect the cell suspension into a 15 mL centrifuge tube.</li>
<li>Count the number of cells using hemocytometer (Weber Scientific). Load 12 μL of cell suspension into hemocytometer and visualize using a light microscope. There are four square regions to count cells. Ensure at least 30 cells are in each region for accurate cell counts. After counting all four regions, determine the average number of cells in each square region. Multiply that number by 10,000 to yield the estimated number of cells per mL.</li>
<li>800,000 cells are required for each reaction. Add the volume of cell suspension that contain 800,000 cells into a separate 15 mL tube, centrifuge at 200 × <i>g</i> for 3 min.</li>
<li>Carefully aspirate the supernatant using a pipette, removing as much as possible without disturbing the pellet. Resuspend with 2 mL PBS to wash cells (<a href="#fig1">Figure 1</a>C).</li>
<li>Centrifuge at 200 × <i>g</i> for 3 min. Remove as much supernatant as possible using a pipette without disturbing the pellet.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> PBS wash is required to completely remove DMEM and trypsin before addition of Cas9/gRNA.</div>
<ol start="5">
<li>Electroporation of HEK293T cells using Lonza nucleofector.
<ol type="a">
<li>Transfer the 20 μL mixed nucleofection reagent into the cell pellet to form ∼25 μL cell suspension. Mix well by pipetting up and down 5 times.</li>
<li>Make sure that the cell culture hood lights are turned OFF. Then, move RNP mixture into hood and remove aluminum foil covering.</li>
<li>Transfer ∼25 μL cell suspension to the 5 μL RNP mixture; add 1 μL Cas9 EP enhancer (<a href="#fig1">Figure 1</a>D). Mix well by pipetting up and down 10 times.</li>
<li>Transfer all (∼30 μL) to one unused well of the 16-well electroporation cuvette using a P20 pipette.
<div class="critical">
<span class="critical-title">Critical:</span> Carefully add the liquid to avoid formation of bubbles. One or two bubbles is not a problem.</div>
</li>
<li>Go to 4D-Nucleofector and enter settings: select the cuvette well intended for electroporation in the machine touchscreen. Next, select “Cell Line SF” from the list of options. For the electroporation code, use CA-189 for HEK293T cells (<a href="#fig1">Figure 1</a>E). Press “OK” on touchscreen.</li>
<li>This brings up the next screen (<a href="#fig1">Figure 1</a>F). The 4D-Nucleofector opens a slot that allows placement of the cuvette. After placing the cuvette in the holder, confirm the settings and touch “START” on touchscreen to electroporate.
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the cuvette is placed in the right orientation in the nucleofector. The yellow pin must be visible on the side closest to the nucleofector body (<a href="#fig1">Figure 1</a>G). If the strip is mounted in the wrong orientation, the yellow pin is hardly visible because the cuvette will not be flush in the holder.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use some method of labeling which wells have already be used for electroporation. This can be done by labeling the cuvette cap accordingly, but please ensure that the orientation of the cuvette cap is consistent relative to the cuvette.</div>
<div class="note">
<span class="note-title">Note:</span> White precipitate may appear after electroporation and is not cause for concern. If anything, it is a visual cue that the electroporation worked.</div>
</li>
<li>Appearance of a green ‘+’ indicates successful electroporation (<a href="#fig1">Figure 1</a>H). After electroporation, go back to culture hood with lights OFF, add 50 μL of PBS into cuvette, transfer all to 250 μL PBS in an Eppendorf tube for ∼330 μL total.</li>
<li>Add 50 μL each to three wells of 48-well plate (A, B, C) (<a href="#tbl2">Table 2</a>).
<table id="tbl2">
<caption>Table 2. Organization of cell samples after electroporation</caption>
<thead>
<tr>
<th>Well A</th>
<th>Well B</th>
<th>Well C</th>
</tr>
</thead>
<tbody>
<tr>
<td>Light right after delivery</td>
<td>Light 1 h after delivery</td>
<td>No light applied</td>
</tr>
<tr>
<td>Indels measured at 3 days</td>
<td>Indels measured at 3 days</td>
<td>Indels measured at 3 days</td>
</tr>
</tbody>
</table>
</li>
<li>Add 300 μL “DMEM complete” media only to wells B and C (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1112-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Light illumination for deactivation of Cas9/pcRNA</div>
<p>(A) Sample A has 50 μL of PBS. Sample B and C have 50 μL PBS + 300 μL DMEM complete.</p>
<p>(B) Sample A is illuminated by the LED flashlight for 1 min. The flashlight is balanced and centered on the well. To avoid unintended illumination of multiple wells, we suggest separating each well into different 48-well plates. For this protocol, multiple wells were used in the same plate for ease of presentation in the figure.</p>
<p>(C) Cells in Sample A one hour after electroporation and light illumination. Cells are healthy and already adhering to the well. Scale bar corresponds to 100 μm.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Deactivation of Cas9/pcRNAs using light illumination</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="6">
<li>Deactivation right after electroporation to well A (containing 50 μL cells in PBS).
<ol type="a">
<li>Turn on 365 nm LED flashlight, hold it 2 cm above well A for 1 min (<a href="#fig2">Figure 2</a>B). Cells in well A are now deactivated.</li>
<li>Add 300 μL “DMEM complete” media to well A. Cover plate with aluminum foil, move back to incubator.</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Deactivation at 1 h after electroporation to well B.
<ol type="a">
<li>By 1 h after electroporation, HEK293T cells should be adherent if collagen is coated properly (<a href="#fig2">Figure 2</a>C).</li>
<li>First carefully remove most of the media from well B using a pipette. Be sure to not disturb the recently adhered cells.</li>
<li>Turn on 365 nm LED flashlight, hold it 2 cm above well B for 1 min.
<div class="critical">
<span class="critical-title">Critical:</span> DMEM complete can partially block the light, which greatly reduces deactivation efficiency.</div>
</li>
<li>Add back 300 μL fresh “DMEM complete” media. Cover plate with aluminum foil, move back to incubator.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Harvest genomic DNA (gDNA) and detect indel mutations via sanger sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>Genomic DNA from HEK293T cells exposed to various conditions are purified, then subject to PCR to amplify ∼400 bp product that includes the edited region of the genome. Approximately 10 ng of genomic DNA input is sufficient for high quality PCR that accurately quantifies editing efficiency using Sanger Sequencing.</p>
<ol start="8">
<li>Harvest genomic DNA (gDNA) from HEK293T cells.
<ol type="a">
<li>3 days after electroporation, wash cells off 48-well plates with 200 μL PBS, using pipette or cell scraper.
<div class="note">
<span class="note-title">Note:</span> Between 10,000 to 1 million cells should be sampled to yield enough gDNA and properly estimate the frequency of indels. In this experiment, there are approximately 100,000 cells initially plated to each 48-well after electroporation, which increases to approximately 300,000 after 3 days of cell growth and division.</div>
</li>
<li>From cells suspended in 200 μL PBS, purify genomic DNA (gDNA) using DNeasy Blood &amp; Tissue Kit following manufacturer’s protocol. Elute in 100 μL AE supplied with kit. Store gDNA in −20°C.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Measure DNA concentration using the NanoDrop 2000 UV-Vis spectrophotometer. Estimation of DNA concentration should be greater than 10 ng/μL.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> continue with PCR any time.</div>
<ol start="9">
<li>Perform PCR to amplify region around Cas9 target site. The forward and reverse primers used are <i>HEKs4_F</i> and <i>HEKs4_R</i>, respectively, from <a href="#tbl1">Table 1</a>.
<ol type="a">
<li>Combine for PCR reaction. Mix by pipetting up and down 5–10 times.
<table id="undtbl3">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease Free Water (NFW)</td>
<td>3 μL</td>
</tr>
<tr>
<td>Genomic DNA (10–50 ng)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Fwd/Rev primer set (5 μM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Q5® High-Fidelity 2<b>×</b> Master Mix</td>
<td>5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Start thermocycling protocol (for HEKs4_F/HEKs4_R primer set)
<table id="undtbl4">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>68°C</td>
<td>10 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>Extract PCR-amplified target DNA with QIAquick PCR Purification Kit following <a href="https://www.qiagen.com/us/resources/resourcedetail?id=e0fab087-ea52-4c16-b79f-c224bf760c39">manufacturer’s protocol</a>. The following reagents except 3 M sodium acetate (NaAc) are supplied with the kit. Briefly, adding 50 μL PB (binding buffer), 10 μL of 3 M NaAc, mix using a pipette, then loading a spin column. Spin in microcentrifuge at 17,900 × <i>g</i> for 1 min, decant the collection tube, then load 750 μL PE (wash buffer). Spin with same protocol, decant the collection tube, then spin again to dry spin column membrane. Elute in 30 μL EB.
<div class="optional">
<span class="optional-title">Optional:</span> Measure DNA concentration using the NanoDrop 2000 UV-Vis spectrophotometer. Estimation of DNA concentration should be greater than 1 ng/μL.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Run PCR product on gel electrophoresis. We use E-Gel® EX Gel, 2% pre-cast gels on the E-Gel® iBase™ Power System/E-Gel™ Safe Imager™ Real-Time Transilluminator. Running PCR product on gel electrophoresis should yield a band of the expected size.</div>
</li>
<li>Submit 10 μL for Sanger Sequencing, using the Fwd and/or Rev PCR primer as the sequencing primer. We use GeneWiz or a core facility at our institution.</li>
<li>Also sequence wild type HEK293T cells for a “negative control” sample with no genome editing.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> continue with Sanger sequencing analysis any time.</div>
<ol start="10">
<li>Evaluate genome editing efficiencies from Sanger sequencing results using TIDE: Tracking of Indels by DEcomposition (<a href="#bib1">Brinkman et al., 2014</a>).
<ol type="a">
<li>Obtain Sanger sequencing reads in .ab extension format.</li>
<li>Directly visualize the Sanger sequencing traces using a program such as SnapGene.</li>
<li>Visit the TIDE website.</li>
<li>Enter GGCACTGCGGCTGGAGGTGG as the guide sequence for <i>HEK site 4.</i>
</li>
<li>Enter the .ab sequencing file for the negative control under “Control Sample Chromatogram (.ab1 or .scf)”</li>
<li>Enter the .ab sequencing file for the intended sample under “Test Sample Chromatogram (.ab1 or .scf)”</li>
<li>Press the blue “Update View” button. The results will automatically update on the main website panel.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Direct visualization of Sanger sequencing traces (.ab1 files) should show a single dominant nucleotide signal at each base pair position for the samples without Cas9 (‘neg ctrl’) and with Cas9/pcRNA but light illumination as soon as possible after electroporation (‘light 1m’). In contrast, for samples with Cas9/pcRNA and not exposed to light, i.e., the Cas9 inside cells remains active (‘no light’), the base pair positions downstream of the target sequence will have variable nucleotide signal, indicating a mixture of different sequence species due to presence of indels (<a href="#fig3">Figure 3</a>A).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1112-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Sanger sequencing and indel measurement</div>
<p>(A) Sanger sequencing traces around the Cas9 target site for cells exposed to Cas9 followed by light right after electroporation (‘light 1m’), cells exposed to Cas9 but no light (‘no light), or cells without Cas9 exposure (‘neg ctrl’). 2.0%, 68.5%, 1.7% correspond to the estimated indel percentage at 3 days after electroporation for each sample.</p>
<p>(B) Output of TIDE indel analysis for cells exposed to Cas9 but no light (‘no light’).</p>
<p>(C) Output of TIDE indel analysis for cells exposed to Cas9 followed by light right after electroporation (‘light 1m’).</p>
<p>(D) Quantification of indel percentage at 3 days after electroporation for the samples described in panel a. Quantification for cells exposed to Cas9 followed by light at 1 h after electroporation is also included (‘light 1h’).</p>
</figcaption>
</figure>
<p>Entering the Sanger sequencing traces in TIDE (<a href="#bib1">Brinkman et al., 2014</a>) will estimate the identity of each indel product in the mixture. For the ‘no light’ sample, there is a wide distribution of deletions between 1 to 10 nucleotides, or an insertion of a single nucleotide (<a href="#fig3">Figure 3</a>B). For the ‘light 1m’ sample, there are almost no detected indels, because the cells had its intracellular Cas9/pcRNA deactivated before they had a chance to induce genome editing (<a href="#fig3">Figure 3</a>C). Quantification of expected indel results for all samples are shown (<a href="#fig3">Figure 3</a>D).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is reliable for all cell types compatible with electroporation using the Lonza system. We have successfully tested this protocol on HEK293T, K562, U2OS, and WTC-11 iPSCs with some variations to the protocol. We have mainly validated Cas9 deactivation after delivery of Cas9 protein in complex with light-deactivatable pcRNA. Because the pcRNA cannot be genetically encoded, delivery using other strategies such as lentivirus is likely infeasible. Direct delivery of pcRNA via lipofection is also feasible, but we found was significantly less efficient compared to electroporation.</p>
<p>The method for indel quantification using Sanger sequencing also has limitations. First, it is unable to detect lower levels of indels below 2%–3% due to the noise of Sanger sequencing. Second, there are some variations in quantification compared to other methods for indel measurement such as high-throughput sequencing and the T7E1 assay (<a href="#bib4">Sentmanat et al., 2018</a>). Sanger sequencing was chosen for its convenience, but other methods such as high-throughput sequencing can detect lower levels of indels (even below 0.1%) without potential miscalling of indels.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Electroporation protocol on the Lonza machine yields an error, represented by a red ‘–’ in the case of multiple errors or a yellow ‘+’ in the case of few errors (“warning”), rather than the green ‘+’ that indicates electroporation success (step 5g).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>We have never obtained electroporation errors with HEK293T cells but have encountered such errors on occasion using other cell types such as K562. To resolve these errors, select the problematic well on the graphical display to obtain more details including an error breakdown. Refer to the 4D-Nucleofector™ System Manual for a comprehensive explanation of each error code. Empirically, we found that a couple of changes may increase the chance of success: 1) decant as much PBS as possible from the cell pellet before nucleofection buffer is added – this increases the relative volume of nucleofection buffer compared to cells, Cas9, other buffers, etc. 2) Try the electroporation protocol using different numbers of cells. Reducing the number of cells may improve likelihood of success. 3) Ensure that bubbles are not introduced when transferring the cell mixture into the cuvette before electroporation. Excessive bubbles can also cause the electroporation to fail.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>HEK293T cells are not appropriately adherent by 1 h after electroporation or look otherwise unhealthy (step 7a).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>First, ensure that the collagen is fresh. Ensure that the tissue culture plate material is polystyrene and packaged with pre-treatment that makes the surface more amenable to cell attachment.</p>
<p>If the cells look unhealthy, one possibility is unintended temperature shock. Ensure that the trypsin, cell media, or PBS used are completely preheated to 37°C prior to removal from water bath/heating unit, sprayed with 70% ethanol, wiped with paper towels, and moved to inside the hood.</p>
<p>Finally, rule out mycoplasma infection as a cause of unexpected changes in cell morphology by performing mycoplasma testing.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Sanger sequencing failed for various reasons (mispriming, sequencing noise) (step 10a).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Run PCR product on gel before Sanger sequencing to verify presence of a single amplicon. If not, then the PCR failed and the PCR protocol should be further optimized. Another check is to ensure that Sanger sequencing works on samples without Cas9 exposure (genomic DNA from unmodified cells).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>No detected indels from Sanger sequencing in samples with Cas9/pcRNA delivered and without light illumination, despite successful electroporation (green ‘+’ observed on the 4D-Nucleofector after electroporation) (step 10g).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>First, verify that there are indels when the pcRNA is replaced with the wild type gRNA version of the target sequence. This involves performing the same electroporation experiment detailed in the protocol, but replacing the pcRNA (<i>Pcl5_HEKsite4</i> in <a href="#tbl1">Table 1</a>) with the wild type gRNA (<i>gRNA_HEKsite4</i> in <a href="#tbl1">Table 1</a>) version.</p>
<p>If there are indels when using the wild type gRNA, then the pcRNA itself may be problematic. This could be because all the pcRNA were already deactivated due to accidental exposure to sub 400 nm wavelength light illumination, the ordered target sequence was incorrect, or the company incorrectly synthesized the gRNA. To avoid accidental exposure to sub 400 nm wavelength light, be sure to cover the pcRNA whenever possible with aluminum foil, though exposure to ambient light is unlikely to cleave the pcRNA. Furthermore, during the light illumination protocol, separate each condition into a different plate so that light illumination to one well does not bleed through to adjacent wells, or use black wells that prevent light penetration or reflection.</p>
<p>If there are still no indels when using the wild type gRNA, the problem could arise from improper electroporation, defective Cas9, or defective tracrRNA.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Light illumination of cells with Cas9/pcRNA does not prevent indel formation (step 10g).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Ensure that the path of illumination from the flashlight to the cells is not inhibited by any plastic covering (such as plate cover) or excessive cell media (which could partially absorb the light). Finally, ensure that the illuminating flashlight is fully charged. Try increasing the intensity or duration of the light exposure to see if that results in any deactivation.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Brinkman, E.K., Chen, T., Amendola, M., and van Steensel, B. (2014). Easy quantitative assessment of genome editing by sequence trace decomposition. Nucleic Acids Res. <i>42</i>, 168. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00615-8/sref1">View at publisher</a></p>
<p id="bib2">Hu, J.H., Miller, S.M., Geurts, M.H., Tang, W., Chen, L., Sun, N., Zeina, C.M., Gao, X., Rees, H.A., Lin, Z., and Liu, D.R. (2018). Evolved Cas9 variants with broad PAM compatibility and high DNA specificity. Nature <i>556</i>, 57-63. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00615-8/sref2">View at publisher</a></p>
<p id="bib3">Liu, Y., Zou, R.S., He, S., Nihongaki, Y., Li, X., Razavi, S., Wu, B., and Ha, T. (2020). Very fast CRISPR on demand. Science <i>368</i>, 1265-1269. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00615-8/sref3">View at publisher</a></p>
<p id="bib4">Sentmanat, M.F., Peters, S.T., Florian, C.P., Connelly, J.P., and Pruett-Miller, S.M. (2018). A survey of validation strategies for CRISPR-Cas9 editing. Sci. Rep. <i>8</i>, 1-8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00615-8/sref4">View at publisher</a></p>
<p id="bib5">Zou, R.S., Liu, Y., Wu, B., and Ha, T. (2021). Cas9 deactivation with photocleavable guide RNAs. Mol. Cell <i>81</i>, 1553-1565. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00615-8/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Taekjip Ha (<a href="mailto:tjha@jhu.edu">tjha@jhu.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>Original data have been deposited to Mendeley Data: <a href="https://doi.org/10.17632/w55kctmzzc.1">https://doi.org/10.17632/w55kctmzzc.1</a>. The published article includes all remaining data generated during this study.</p>
<h3>Acknowledgments</h3>
<p>We thank Dr. Geraldine Seydoux for access to the Lonza 4D nucleofector. This work was supported by the grants from the <a href="https://doi.org/10.13039/100000002">NIH</a> (R35 GM 122569 to T.H., T32 GM 73009 and F30 CA 254160 to R.S.Z.), <a href="https://doi.org/10.13039/100000001">NSF</a> (PHY 1430124 to T.H., 1817447 to B.W.), and <a href="https://doi.org/10.13039/100000875">Pew Charitable Trust</a> (00030601 to B.W.). T.H. is an investigator of the Howard Hughes Medical Institute.</p>
<h3>Author contributions</h3>
<p>R.S.Z. and T.H. initially conceived the project. R.S.Z. and Y.L. performed experiments and analyzed the data. R.S.Z. wrote the manuscript with contributions from all authors. T.H. supervised the project.</p>
<h3>Declaration of interests</h3>
<p>The authors and Johns Hopkins University have authored patent application PCT/US20/57255, including the work described herein.</p>
</section>
</article>