<article>
<section>
<h1 id="header">CRISPR-Cas9-Based Genome Editing and Cytidine Base Editing in <i>Acinetobacter baumannii</i>
</h1>
<p><time datetime="2020-03-27">Published: March 27, 2020</time></p>
<p>Yu Wang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Zhipeng Wang,<sup><a href="#aff1">1</a></sup> and Quanjiang Ji<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>School of Physical Science and Technology, ShanghaiTech University, Shanghai 201210, China</p>
<p id="aff2"><sup>2</sup>College of Life Science and Engineering, Jiangxi Agricultural University, Nanchang, Jiangxi 330045, China</p>
<p id="fn1"><sup>3</sup>Technical Contact</p>
<p id="fn2"><sup>4</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:wangyu@shanghaitech.edu.cn">wangyu@shanghaitech.edu.cn</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:quanjiangji@shanghaitech.edu.cn">quanjiangji@shanghaitech.edu.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100025">10.1016/j.xpro.2020.100025</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Infections caused by drug-resistant <i>Acinetobacter baumannii</i> have posed a serious threat to global public health. However, genetic manipulation methods, the primary way to study pathogenesis and drug-resistance mechanisms, remain time consuming and inefficient. Here, we provide a detailed protocol for genetic manipulation, including gene deletion, insertion, and point mutation in <i>A. baumannii</i> using the platform.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib3">Wang et al. (2019)</a>.</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>An optimized protocol for <i>A. baumannii</i>-competent cell preparation and electroporation</li>
<li>Achieving chromosomal gene deletion, insertion, and point mutation in <i> A. baumannii</i>
</li>
<li>Inactivating the gene via cytidine deaminase-catalyzed C-to-T conversion in <i> A. baumannii</i>
</li>
<li>The editing plasmids can be cured via <i>sacB</i> counter-selection</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/69-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<h3>Examination of the Antibiotic Sensitivity of the Target Strain</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>To select the proper plasmids used for gene editing, the antibiotic sensitivity and the minimal inhibitory concentration (MIC) of the target <i>A. baumannii</i> strain needs to be examined beforehand.</p>
<ol>
<li>Streak the wild-type cells onto different lysogeny broth (LB) agar plates containing different antibiotics with serial 2-fold dilution concentrations. Use the antibiotic-free LB agar plate as a control.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Several commonly used antibiotics are recommended at a final concentration of 100 μg/mL of apramycin, 50 μg/mL of kanamycin, and 100 μg/mL of spectinomycin.</div>
<ol start="2">
<li>Incubate the plates for approximately 16 hours at 37°C, and then determine the antibiotic sensitivity and the MIC of the target strain.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and Virus Strains</td>
</tr>
<tr>
<td>
<i>E. coli</i> DH5α</td>
<td>WEIDI</td>
<td>CAT# DL1001</td>
</tr>
<tr>
<td>
<i>A. baumannii</i> ATCC 17978</td>
<td>ATCC</td>
<td>ATCC® 17978</td>
</tr>
<tr>
<td>
<i>A. baumannii</i> XH386</td>
<td>(<a href="#bib1">Fang et al., 2016</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>Apramycin sulfate</td>
<td>Sangon</td>
<td>CAT# A600090</td>
</tr>
<tr>
<td>Kanamycin sulfate</td>
<td>Sangon</td>
<td>CAT# A600286</td>
</tr>
<tr>
<td>Spectinomycin hydrochloride</td>
<td>Meilunbio</td>
<td>CAT# MB1497</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Hushi</td>
<td>CAT# 10010618</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sangon</td>
<td>CAT# A502792-0500</td>
</tr>
<tr>
<td>LB Broth Powder</td>
<td>Sangon</td>
<td>CAT# A507002-0250</td>
</tr>
<tr>
<td>LB Agar Powder</td>
<td>Sangon</td>
<td>CAT# A507003-0250</td>
</tr>
<tr>
<td>2×Es Taq MasterMix (Dye)</td>
<td>CoWin Biosciences</td>
<td>CAT# CW0690M</td>
</tr>
<tr>
<td>T4 Polynucleotide Kinase</td>
<td>TAKARA</td>
<td>CAT# 2021S</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>NEB</td>
<td>CAT# M0202S</td>
</tr>
<tr>
<td>
<i>Bsa</i>I-HFv2</td>
<td>NEB</td>
<td>CAT# R3733S</td>
</tr>
<tr>
<td>HiPure Plasmid Micro Kit</td>
<td>Magen</td>
<td>CAT# P1001-03C</td>
</tr>
<tr>
<td>Ezup Column Bacteria Genomic DNA Purification Kit</td>
<td>Sangon</td>
<td>CAT# B518255-0050</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pAT04</td>
<td>(<a href="#bib2">Tucker et al., 2014</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pCasAb-apr</td>
<td>This Paper</td>
<td>Addgene 121998</td>
</tr>
<tr>
<td>pSGAb-km</td>
<td>This Paper</td>
<td>Addgene 121999</td>
</tr>
<tr>
<td>PSGAb-spe</td>
<td>This Paper</td>
<td>Addgene 122000</td>
</tr>
<tr>
<td>PBECAb-apr</td>
<td>This Paper</td>
<td>Addgene 122001</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Gene Pulser Xcell<sup>TM</sup> Electroporation System</td>
<td>Bio-rad</td>
<td>CAT# 165-2660; CAT# 165-2662</td>
</tr>
<tr>
<td>2 mm electroporation cuvette</td>
<td>Bio-Rad</td>
<td>CAT# 1652082</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Spacer Cloning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol>
<li>Pick a 20 bp spacer ahead of the PAM sequence (5’-NGG-3’) in the target locus, and then synthesize the two oligonucleotides as follows:</li>
</ol>
<p>spacer-F: 5’-tagtNNNNNNNNNNNNNNNNNNNN-3’<br> spacer-R: 5’-aaacN’N’N’N’N’N’N’N’N’N’N’N’N’N’N’N’N’N’N’N’-3’</p>
<p>For example, the selected spacer and PAM sequence for the <i>oxyR</i> gene is 5’-ATTGTAGAGCGCCTAGAACA-TGG-3’. The sequences of <i>oxyR</i>_spacer oligonucleotides are as follows:</p>
<p>oxyR-spacer-F: 5’-tagtATTGTAGAGCGCCTAGAACA-3’<br> oxyR-spacer-R: 5’-aaacTGTTCTAGGCGCTCTACAAT-3’</p>
<div class="note">
<span class="note-title">Note:</span> A spacer design tool (e.g. sgRNAcas9 software) can be used for selecting a suitable spacer sequence (<a href="#bib4">Xie et al., 2014</a>). N’ denotes reverse complement of N in forward primer.</div>
<ol start="2">
<li>Phosphorylate the oligonucleotides as indicated below and incubate them at 37°C for 1 hour.</li>
</ol>
<p>5 μL 10 × T4 DNA Ligase buffer (TAKARA)<br> 1 μL Spacer-F (100 μM)<br> 1 μL Spacer-R (100 μM)<br> 1 μL T4 Polynucleotide Kinase (TAKARA)<br> 42 μL ddH<sub>2</sub>O</p>
<div class="note">
<span class="note-title">Note:</span> Gently mix all the reagents and collect them by a quick spin. The order of addition of the reagents can be random.</div>
<ol start="3">
<li>Add 0.5 μL of 5 M NaCl into the phosphorylated product. Incubate the reaction solution at 95 °C for 5 min and then slowly cool it down to room temperature (20-30°C) using a thermocycler, during the process of which the temperature decreases by 1°C per 10s.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> You can anneal the oligonucleotides using hot water instead of a thermocycler. Incubate the tube containing the aforementioned reaction solution in a hot water bath of boiling water for 5 min. Then get the tube out of the water bath and let the solution slowly cool down to room temperature (20-30°C).</div>
<ol start="4">
<li>Dilute the annealed oligonucleotides 20 folds with ddH<sub>2</sub>O. Clone the annealed oligonucleotide into the sgRNA expression plasmid as indicated below.</li>
</ol>
<p>1 μL sgRNA expression plasmid (50 ng/μL)<br> 1 μL Diluted annealed oligonucleotides<br> 1 μL 10 × T4 DNA Ligase buffer (NEB)<br> 0.5 μL T4 DNA Ligase<br> 0.5 μL BsaI-HFv2<br> 6 μL ddH<sub>2</sub>O</p>
<div class="note">
<span class="note-title">Note:</span> The sgRNA expression plasmids in this protocol contain pSGAb-km, pSGAb-spe and pBECAb-apr.</div>
<ol start="5">
<li>Perform the Golden Gate assembly reaction inside a thermocycler using the parameter indicated below (approximately 4 hours) (<a href="#fig1">Figure 1</a>).</li>
</ol>
<p>25 cycles at 37°C for 3 min, and 16°C for 4 min;<br> 50°C for 5 min;<br> 80°C for 10 min;<br> Hold at 16°C.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/69-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Inserting a 20 bp Spacer Fragment into the sgRNA Expression Plasmid via Golden Gate Assembly</div>
</figcaption>
</figure>
<ol start="6">
<li>Transform 10 μL assembled product into 100 μL <i>E. coli</i> DH5α competent cells according to the manufacturer's protocol. The cells are plated onto an LB agar plate supplemented with the corresponding antibiotic and the plate is incubated at 37°C for 14-16 hours.</li>
</ol>
<ol start="7">
<li>Randomly pick several colonies to verify the successful cloning of the spacer by colony PCR with the primers of Spacer-F/M13R. The M13R primer sequence is 5’-CAGGAAACAGCTATGACC-3’). Make a colony PCR mix as indicated below, and keep on ice.</li>
</ol>
<p>10 μL 2×Es Taq MasterMix (Dye)<br> 0.5 μL Spacer-F (10 μM)<br> 0.5 μL M13R (10 μM)<br> 0.5 μL bacteria solution<br> 8.5 μL ddH<sub>2</sub>O</p>
<div class="note">
<span class="note-title">Note:</span> Do not add more than 1 μL of bacteria solution into a 20 μL of colony PCR reaction.</div>
<ol start="8">
<li>Perform the PCR protocol inside a thermocycler using the parameter indicated below (approximately 1.5 hours). 5-10 colonies are recommended for the PCR screening.</li>
</ol>
<p>95°C for 10 min;<br> 28 cycles at 95°C for 30 s, 54°C for 30 s, and 72°C for 20 s;<br> 72°C for 5 min;<br> Hold at 16°C.</p>
<div class="critical">
<span class="critical-title">Critical:</span> It is crucial that an initial denaturation of 10 min at 95°C for colony PCR.</div>
<ol start="9">
<li>Run an agarose gel to check for the size of the PCR product, of which the theoretical size is 224 bp. Select 1∼2 positive colonies confirmed by colony PCR. Inoculate the colonies into 5 mL LB broth containing the corresponding antibiotic and incubate the cultures at 37°C for 16 hours with shaking at 250 rpm.</li>
<li>Extract the spacer-introduced plasmids by plasmid miniprep kit according to the manufacturer's protocol. Determine the concentration of the extracted plasmid using NanoDrop. Confirm the correct spacer insertion by DNA sequencing using the M13R primer.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The plasmids can be stored at −20°C for several months.</div>
<h3>Preparation of Plasmid-free Electrocompetent Cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<ol start="11">
<li>Streak a wild-type <i> A. baumannii</i> strain onto an antibiotic-free LB agar, and incubate at 37 °C until colonies are visible (approximately 16 hours).</li>
<li>Pick a fresh single colony, and inoculate it into 5 mL of LB broth and incubate the culture at 37 °C for 16 hours with shaking at 250 rpm.</li>
<li>Dilute 500 μL overnight culture into 50 mL of LB broth, and incubate the culture at 37°C with shaking at 250 rpm (approximately 3 hours).</li>
<li>Chill the culture on ice immediately for 15 min, when the optical density at 600 nm (OD<sub>600</sub>) of the cell culture reached to 0.5-0.7.</li>
<li>Collect the cells by centrifugation at 4000 × g for 5 min at 4°C and discard the supernatant.</li>
<li>Resuspend the cell pellet with 15 mL of sterile ice-cold ddH<sub>2</sub>O, and repeat step 15.</li>
<li>Resuspend the cell pellet with 15 mL of sterile ice-cold 10% v/v glycerol, and repeat step 15.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The 10% v/v glycerol can be replaced by ddH<sub>2</sub>O when the freshly prepared electrocompetent cells are used immediately without freezing.</div>
<ol start="18">
<li>Resuspend the cell pellet with 500 μL of sterile ice-cold 10% v/v glycerol.</li>
<li>Dispense the cell suspension into 50 μL aliquots in sterile 1.5 mL microfuge tubes. Freeze the electrocompetent cells in liquid nitrogen and store them at -80°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The sterile microfuge tubes at 0-25°C can meet the experimental requirement.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The plasmid-free <i>A. baumannii</i> electrocompetent cells can be stored at -–80°C for several months without significantly affecting the electroporation efficiency.</div>
<ol start="20">
<li>Perform the steps 21-26 to determine the transformation efficiency of the prepared competent cells.</li>
</ol>
<h3>Electroporation of pCasAb-apr Plasmid</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="21">
<li>Take a tube of the plasmid-free A. baumannii electrocompetent cells out from the -80°C freezer (see step 19), and thaw it on ice for several minutes.</li>
<li>Add approximately 50 ng pCasAb-apr plasmid into the thawed A. baumannii electrocompetent cells, and then gently transfer the mixture into an ice-cold 2 mm electroporation cuvette.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not add more than 5 μL of plasmid into 50 μL of electrocompetent cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the cuvette on ice until the next step.</div>
<div class="note">
<span class="note-title">Note:</span> The pAT04 plasmid can serve as positive transformation control; water can serve as a negative transformation control.</div>
<ol start="23">
<li>Perform the electroporation using a Gene Pulser Xcell<sup>TM</sup> Electroporation System with the following parameters: 2.5 kV, 200Ω and 25 μF.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Please adjust the appropriate parameters when using different electroporation systems.</div>
<ol start="24">
<li>After being pulsed, immediately add 1 mL of prechilled antibiotic-free LB broth into the cuvette to resuspend the cells.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To obtain a high electroporation efficiency, add the recovery medium as soon as possible following the electroporation.</div>
<ol start="25">
<li>Transfer the culture into a sterile 1.5 mL microfuge tube and incubate it for 1 h at 37°C with shaking.</li>
<li>Plate 50 μL of the transformed cells on an LB agar plate (90 mm in diameter) supplemented with 100 μg/mL apramycin, and incubate the plate at 37°C until the colonies are visible (14-16 hours).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Because the transformation efficiency should be high at this stage, do not plate more than 200 μL of cells.</div>
<h3>Preparation of pCasAb-harboring Electrocompetent Cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="27">
<li>Pick a single colony of the pCasAb-harboring <i>A. baumannii</i> strain from step 26, and inoculate it into 5 mL of LB broth containing 100 μg/mL apramycin and incubate the culture at 37°C for 16 hours with shaking at 250 rpm.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> A final concentration of 100 μg/mL of apramycin is critical to avoid plasmid loss.</div>
<ol start="28">
<li>Dilute 500 μL overnight culture into 50 mL of LB broth containing 100 μg/mL apramycin, and incubate the culture with shaking (250 rpm) for 1-1.5 hours at 37°C.</li>
<li>Add 50 μL of 1M IPTG into the culture to induce the expression of the RecAb recombinases and the Cas9 nuclease, when the OD<sub>600</sub> value of the culture reaches to 0.1-0.15.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> IPTG at a final concentration of 1 mM is recommended.</div>
<ol start="30">
<li>Incubate the culture with shaking (250 rpm) at 37°C for another 2 h.</li>
<li>Chill the culture on ice for 15 min.</li>
<li>Collect the cells by centrifugation at 4000 × g for 5 min at 4°C, and discard the supernatant.</li>
<li>Resuspend the cell pellet with 15mL of sterile ice-cold ddH2O, and repeat step 32.</li>
<li>Resuspend the cell pellet with 15mL of sterile ice-cold 10% v/v glycerol, and repeat step 32.</li>
<li>Resuspend the cell pellet with 500 μL of sterile ice-cold 10% v/v glycerol.</li>
<li>Dispense the cell suspension into 50 μL aliquots in sterile 1.5 mL microfuge tubes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The sterile microfuge tubes at 0-25°C can meet the experimental requirement.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To achieve a high editing efficiency, we suggest using the freshly-prepared pCasAb-harboring <i>A. baumannii</i> electrocompetent cells for editing. According to our experience, the genome editing efficiency mediated by the frozen pCasAb-harboring electrocompetent cells is less than 10% of the fresh cells.</div>
<h3>The Two-Plasmid pCasAb/pSGAb-Mediated Genome Editing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<ol start="37">
<li>Add approximately 200 ng spacer-introduced pSGAb-km plasmid and 3 μL of 100 μM ssDNA or 300 ng dsDNA donor repair template into a tube of freshly prepared pCasAb-harboring <i>A. baumannii</i> electrocompetent cells, and then gently transfer the mixture into an ice-cold 2 mm electroporation cuvette (<a href="#fig2">Figure 2</a>). For example, the sequence of 80-nt ssDNA donor DNA for <i>oxyR</i> gene deletion is 5’- TTGATGAAGTGCATCAACAATTACCGAAGATTCAGTTGCAATGATCTCGATTTATCACGTCTCATTTTGCTTGAAGAAGG-3’.</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/69-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Scheme for pCasAb/pSGAb-Mediated Genome Editing in <i>A. baumannii</i>
</div>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Do not add more than 5 μL of plasmid/donor DNA complex into 50 μL of electrocompetent cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the cuvette on ice until the next step.</div>
<div class="note">
<span class="note-title">Note:</span> The empty pSGAb-km plasmid can be used as positive transformation control. The spacer-introduced pSGAb-km plasmid without the insertion of a donor DNA can be used as negative transformation control. ssDNA donor template can be used for gene deletion and point mutation. dsDNA donor template can be used for point mutation and gene insertion. Long homologous template (&gt;200 bp each) can improve the efficiency of gene insertion. The deletion length flanking Cas9 cleavage sites should be similar and no more than 1 kb each.</div>
<ol start="38">
<li>Perform the electroporation using a Gene Pulser Xcell<sup>TM</sup> Electroporation System with the following parameters: 2.5 kV, 200Ω and 25 μF.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Adjust the appropriate parameters when using different electroporation systems.</div>
<ol start="39">
<li>After being pulsed, immediately add 1 mL of prechilled antibiotic-free LB broth into the cuvette to resuspend the cells.</li>
<li>Transfer the culture into a sterile 1.5 mL microfuge tube and incubate it for 1.5 h at 37°C with shaking for recovery and genome editing.</li>
<li>Plate 100 μL of the transformed cells on an LB agar plate (90 mm in diameter) supplemented with 100 μg/mL apramycin and 50 μg/mL kanamycin, and incubate the plate at 37°C until colonies are visible (approximately 16 hours).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Plating 100 μL of the transformed cells is sufficient to obtain enough colonies.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The transformation of the spacer-introduced pSGAb with the corresponding donor DNA can generally produce a few hundred colonies. The transformation of the spacer-free pSGAb plasmid can produce thousands of colonies (positive control). The transformation of the spacer-introduced pSGAb without a donor DNA can only produce less than 20 colonies (negative control).</div>
<ol start="42">
<li>Randomly pick several colonies to verify the success of the genome editing by colony PCR. For example, the sequences of two primers for verifying the <i>oxyR</i> gene deletion are as follows.</li>
</ol>
<p>oxyR-seq-F:5’-ACTTTATCGGGCGGCATTAT-3’<br> oxyR-seq-R:5’-GTACGGCTAGGTGCGTCTTC-3’</p>
<div class="note">
<span class="note-title">Note:</span> Screening of 10 colonies is recommended. In general, the genome editing efficiency is more than fifty percent.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The locations of the two primers used for verification should be outside of the donor template in the genomic DNA.</div>
<ol start="43">
<li>Perform DNA sequencing using one of the specific primers employed above (for example, oxyR-seq-F or oxyR-seq-R) to confirm the desired genome editing.</li>
</ol>
<h3>Cure Both the pCasAb-apr and pSGAb-km Plasmids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<ol start="44">
<li>Inoculate the desired A. baumannii mutant in 5 mL of antibiotic-free LB broth and shake it at 250 rpm for 14-16 hours at 37°C.</li>
<li>Streak a fraction of culture onto an LB agar plate supplemented with 5% w/v sucrose, and incubate it at 37°C until colonies are visible (approximately 16 hours).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Addition of sucrose is necessary to select the cells which have cured the plasmids via <i>sacB</i> counter-selection.</div>
<div class="note">
<span class="note-title">Note:</span> The LB agar supplemented with 5% w/v sucrose is sterilized at 115°C for 15-20 min.</div>
<ol start="46">
<li>Pick and streak several colonies onto LB agar plates supplemented with or without 100 μg/mL apramycin or 50 μg/mL kanamycin, respectively. Incubate the plates at 37°C overnight. If the colonies can only grow on the antibiotic-free LB agar plate, it indicates that both the pCasAb-apr plasmid and the pSGAb-km plasmid are cured successfully in the colonies (<a href="#fig3">Figure 3</a>).</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/69-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Curing of Plasmids pCasAb and pSGAb in <i>A. baumannii</i>
</div>
<p>The strain can only grow on the antibiotic-free LB agar plate, rather than the plate supplemented with apramycin or kanamycin, confirming that the pCasAb-apr and the pSGAb-km plasmids can be cured simultaneously. Apr, apramycin; Km, kanamycin.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Screening of 2–4 colonies is recommended.</div>
<div class="critical">
<span class="critical-title">Critical:</span> 50 μg/mL of kanamycin needs to be substituted by 100 μg/mL of spectinomycin for the genome editing using the pSGAb-spe plasmid.</div>
<h3>The Single-Plasmid pBECAb-Mediated Cytidine Base Editing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<ol start="47">
<li>Take a tube of the plasmid-free A. baumannii electrocompetent cells out from the -80°C freezer (see step 19), and thaw it on ice for several minutes.</li>
<li>Add 50-200 ng spacer-introduced pBECAb-apr plasmid into the thawed A. baumannii electrocompetent cells, and then gently transfer the mixture into an ice-cold 2 mm electroporation cuvette (<a href="#fig4">Figure 4</a>).</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/69-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Scheme for pBECAb-Mediated Cytidine Base Editing in <i>A. baumannii</i>
</div>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> The volume of the pBECAb-apr plasmid should be no more than 5 μL.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the cuvette on ice until the next step.</div>
<div class="note">
<span class="note-title">Note:</span> The spacer-free pBECAb-apr plasmid can serve as positive transformation control; water can serve as negative transformation control.</div>
<ol start="49">
<li>Perform the electroporation using a Gene Pulser Xcell<sup>TM</sup> Electroporation System with the following parameters: 2.5 kV, 200Ω and 25 μF.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Adjust the appropriate parameters when using different electroporation systems.</div>
<ol start="50">
<li>After being pulsed, immediately add 1 mL of prechilled antibiotic-free LB broth into the cuvette to resuspend the cells.</li>
<li>Transfer the culture into a sterile 1.5 mL microfuge tube and incubate it at 37°C for 1.5 hours with shaking (250 rpm) for recovery and base editing.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The sterile microfuge tubes at 0°C–25°C can meet the experimental requirement.</div>
<ol start="52">
<li>Plate 25 μL of the transformed cells on an LB agar plate (90 mm in diameter) supplemented with 100 μg/mL apramycin, and incubate the plate at 37°C until colonies are visible (approximately 16 hours).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> As transformation efficiency should be high at this stage, do not plate more than 50 μL of cells.</div>
<ol start="53">
<li>Randomly pick several colonies to amplify the target region covering the editable sites by colony PCR.</li>
</ol>
<p>For example, to convert the Q37(CAG) of the <i>blaTEM-1D</i> gene to a premature stop codon (TAG) in the <i>A. baumannii</i> XH386 strain, the spacer and PAM sequence 5’-AGATCAGTTGGGTGCACGAG-3’ is selected. The sequences of <i>blaTEM-1D</i>_spacer oligonucleotides are as follows:</p>
<p>blaTEM-1D-spacer-F: 5’-tagtAGATCAGTTGGGTGCACGAG-3’<br> blaTEM-1D-spacer-R: 5’-aaacCTCGTGCACCCAACTGATCT-3’</p>
<p>The sequences of the primers for amplifying the <i>blaTEM-1D</i>_spacer region are as follows:</p>
<p>blaTEM-1D-seq-F: 5’- TGGCACTTTTCGGGGAAATG -3’<br> blaTEM-1D-seq-R: 5’- CACGCTCGTCGTTTGGTATG -3’</p>
<ol start="54">
<li>Examine the desired C-to-T conversion by DNA sequencing using one of the specific primers employed above (for example, blaTEM-1D-seq-F or blaTEM-1D-seq-R).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Screening of 3 colonies is recommended.</div>
<h3>Cure the pBECAb-apr Plasmid</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<ol start="55">
<li>Inoculate the desired A. baumannii mutant into 5 mL of antibiotic-free LB broth and shake it at 250 rpm for 14-16 hours at 37°C.</li>
<li>Streak a fraction of culture onto an LB agar plate supplemented with 5% w/v sucrose, and incubate it at 37°C until colonies are visible (approximately 16 hours).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Addition of sucrose is necessary to select the cells which have cured the plasmids via <i>sacB</i> counter-selection.</div>
<div class="note">
<span class="note-title">Note:</span> The LB agar supplemented with 5% w/v sucrose is sterilized at 115°C for 15–20 min.</div>
<ol start="57">
<li>Pick and streak several colonies onto LB agar plates supplemented with or without 100 μg/mL apramycin, respectively. Incubate the plates at 37°C overnight. If the colonies can only grow on the antibiotic-free LB agar plate, it indicates that the pBECAb-apr plasmid is cured successfully in the colonies (<a href="#fig5">Figure 5</a>).</li>
</ol>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/69-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Curing of Plasmid pBECAb in <i>A. baumannii</i>
</div>
<p>The strain can only grow on the antibiotic-free LB agar plate, rather than the plate supplemented with apramycin, confirming that the pBECAb-apr plasmid can be cured easily cured in <i>A. baumannii</i>. Apr, apramycin.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Screening of 2–4 colonies is recommended.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>Rapid and precise chromosomal gene deletion, insertion, and point mutation were achieved in various <i>A. baumannii</i> strains with the utilization of the two-plasmid pCasAb/pSGAb system. Highly efficient gene inactivation was observed via cytidine deaminase-catalyzed programmed C-to-T conversion. After desired editing, all these editing plasmids were successfully cured using the <i>sacB</i> counter-selection. Details of genome editing and base editing can be found in our recent <i>Cell Chemical Biology</i> publication (<a href="#bib3">Wang et al., 2019</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>It is difficult to use the two-plasmid pCasAb/pSGAb system for gene editing in the sites located on mobile genetic elements, such as plasmids and transposons. The double-strand DNA streak (DSB) on the plasmids produced by Cas9 nuclease can lead to plasmid removal without repair. The DSB on the transposons may cause undesired recombination between repetitive sequences. Alternatively, the cytidine base editing system can be used for gene inactivation by converting four codons (CAA, CAG, CGA, and TGG) into premature stop codons (TAA, TAG, and TGA).</p>
<p>We have comprehensively investigated the activity window of the single plasmid pBECAb system, which is from positions 3 to 8 (counting the base distal from the PAM as position 1) (<a href="#fig6">Figure 6</a>). The Cs within the activity window can be converted into Ts efficiently. However, the conversion frequency of the Cs outside the activity window reduces dramatically. In addition, the editing efficiency is affected by the kind of base immediately at 5′ of the target C, which follows the order: TC ≥ CC ≥ AC &gt; GC.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/69-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Schematic Diagram Illustrating the Activity Window, which is from Positions 3 to 8 in the 20 bp Spacer</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem</h3>
<p>My <i>A. baumannii</i> strain is not sensitive to apramycin, kanamycin, or spectinomycin.</p>
<h3>Potential Solution</h3>
<p>Some of clinically isolated <i>A. baumannii</i> strains are usually resistant to multiple antibiotics. To modify the genome of MDR <i>A. baumannii</i> strains, you can first perform the drug sensitivity test for your strain, and then replace the antibiotic selection marker in pCasAb-apr, pSGAb-km, pSGAb-spe, and pBECAb-apr plasmids with any other proper selection markers by Gibson assembly.</p>
<h3>Problem</h3>
<p>Only a few colonies are obtained using the pCasAb/pSGAb system, and no colony is successfully edited.</p>
<h3>Potential Solution</h3>
<p>Does the transformation of the spacer-free pSGAb-km plasmid produce a lawn of colonies? If so, the result indicates that the CRISPR-Cas9 have cleaved the genomic DNA, but no recombination event occurs. Please confirm the correctness of the IPTG solution in your lab and the gene sequence of RecAb on the pCasAb plasmid you used. If all these are correct, the result indicates that the electroporation efficiency of the pCasAb-harboring cells is low. Please re-prepare the electrocompetent competent cells according to the preparation method described above.</p>
<h3>Problem</h3>
<p>For cytidine base editing, the results of DNA sequencing show that only overlapping peaks appearing in the target editing site.</p>
<h3>Potential Solution</h3>
<p>It indicates that the cells in a single colony are not 100% edited. You can obtain a pure mutant by streaking the partially edited colonies onto an LB agar containing the corresponding antibiotic and incubating it plate at 37°C overnight.</p>
<h3>Problem</h3>
<p>Only low efficiency of C-to-T conversion can be detected.</p>
<h3>Potential Solution</h3>
<p>Make sure the target C is located at the positions 3 to 8 (activity window) of the 20 bp spacer, and avoid use the spacer containing the GC target site.</p>
</section>
<section>
<h2 id="q-and-a">Q&amp;A</h2>
<p class="question"><b>Q:</b> I did not get any colonies after electroporation - what went wrong?</p>
<p class="answer"><b>A:</b> Many factors can affect the number of colonies you can get after transformation, such as the quality of the competent cells, the recombination efficiency of the target site and the target strain, the transformation efficiency of the target strain. It is better to do a positive control using the gene we successfully deleted in the paper when you are not able to get colonies.</p>
<p class="question"><b>Q:</b> Have the authors used this system in other organisms? Do they have any recommendations for places the protocol would need to be adapted?</p>
<p class="answer"><b>A:</b> We have not used this system in other organisms. The replication ori, the antibiotic marker, and the recombination system RecET probably need to changed when you want to adapt this system in other organisms.</p>
<p class="question"><b>Q:</b> For the 2-vector system, can both plasmids be introduced simultaneously during the electroporation step?</p>
<p class="answer"><b>A:</b> The two vectors cannot be transformed simultaneously. The RecET recombination system needs to be expressed before the CRISPR/Cas9 system. Otherwise, the cells will be killed by the CRISPR/Cas9 system without repair.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Fang, Y., Quan, J., Hua, X., Ye, F., Li, X., Wang, J., Zhi, R., Shang, S., and Yu, Y. (2016). Complete genome sequence of <i>Acinetobacter baumannii</i> XH386 (ST208), a multi-drug resistant bacteria isolated from pediatric hospital in China. Genomics Data <i>7</i>, 269-274.</p>
<p id="bib2">Tucker, A.T., Nowicki, E.M., Boll, J.M., Knauf, G.A., Burdis, N.C., M Stephen, T., and Davies, B.W. (2014). Defining gene-phenotype relationships in <i>Acinetobacter baumannii</i> through one-step chromosomal gene inactivation. Mbio <i>5</i>, e01313-e01314.</p>
<p id="bib3">Wang, Y., Wang, Z., Chen, Y., Hua, X., Yu, Y., and Ji, Q. (2019). A Highly Efficient CRISPR-Cas9-Based Genome Engineering Platform in <i> Acinetobacter baumanni</i>i to Understand the H<sub>2</sub>O<sub>2</sub>-Sensing Mechanism of OxyR. Cell Chem. Biol. <i>26</i>, 1732-1742.</p>
<p id="bib4">Xie, S., Shen, B., Zhang, C., Huang, X., and Zhang, Y. (2014). sgRNAcas9: a software package for designing CRISPR sgRNA and evaluating potential off-target cleavage sites. PLoS one <i>9</i>, e100448.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>We thank the support from Analytical Instrumentation Center (#SPST-AIC10112914), SPST, ShanghaiTech University.</p>
<h3>Author Contributions</h3>
<p>Y.W. and Z.W. wrote the protocol. Q.J. edited and revised the protocol.</p>
<h3>Declaration of Interests</h3>
<p>Two patents have been submitted for the two-plasmid pCasAb/pSGAb genome editing system and the single-plasmid pBECAb base editing system.</p>
</section>
</article>