<article>
<section>
<h1 id="header">Protocol for evaluating protein relocalization from the plasma membrane to chloroplasts</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Laura Medina-Puche,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor1">*</a></sup> Chanhong Kim,<sup><a href="#aff1">1</a></sup> Rosa Lozano-Duran,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Vivek Dogra<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Shanghai Center for Plant Stress Biology, CAS Center for Excellence in Molecular Plant Sciences, Chinese Academy of Sciences, Shanghai 201602, China</p>
<p id="aff2"><sup>2</sup>Department of Plant Biochemistry, Centre for Plant Molecular Biology (ZMBP), Eberhard Karls University, 72076 Tübingen, Germany</p>
<p id="aff3"><sup>3</sup>Biotechnology Division, CSIR-Institute of Himalayan Bioresource Technology, Palampur, Himachal Pradesh 176061, India</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:laura.medina-puche@zmbp.uni-tuebingen.de">laura.medina-puche@zmbp.uni-tuebingen.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:vivekdogra@ihbt.res.in">vivekdogra@ihbt.res.in</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100816">10.1016/j.xpro.2021.100816</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We present a protocol for analyzing and evaluating the relocalization of proteins from the plasma membrane to chloroplasts. Some plant membrane-bound proteins carry dual targeting signals, e.g., a membrane-anchoring N-myristoylation motif and a chloroplast transit peptide. These proteins are predominantly targeted to membranes; upon certain cues, however, they can undergo detachment from membranes and relocalization to chloroplasts. This protocol combines imaging and biochemical analyses to track in a reliable and quantitative manner the relocalization of proteins between subcellular organelles.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref6">Medina-Puche et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A method for tracking relocation of proteins from plasma membrane-to-chloroplasts</li>
<li>A method for a quantitative evaluation of the localization in a dynamic manner</li>
<li>Relocation of both host and foreign proteins can be studied</li>
<li>The protocol integrates biochemical and imaging approaches</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1003-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The experimental <i>Solanaceae</i> species <i>Nicotiana benthamiana</i> has become a cornerstone of host-pathogen research, particularly in the context of innate immunity and defense signaling. Moreover, its use as a plant model for cell biology has expanded considerably in recent years, rapidly gaining popularity, particularly in studies requiring protein localization, interaction, or plant-based systems for protein expression and purification. Transient protein expression requires four-week-old <i>N. benthamiana</i> plants. Therefore, in order to implement this protocol, researchers must first grow healthy <i>N. benthamiana</i> plants. Since <i>N. benthamiana</i> plants growth and <i>Agrobacterium</i>-mediated T-DNA transformation require optimal light, humidity, and temperature conditions, we recommend using environmentally-controlled growth facilities.</p>
<h3 id="sec1.1">Preparing plant material and setting growth conditions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 weeks</div>
<p>Start preparing the plant material four weeks in advance as follows:</p>
<ol>
<li>Sow <i>N. benthamiana</i> seeds in a well-watered soil mixture [3 parts of potting substrate (Pindstrup Mosebrug A/S, Denmark) and 1 part of inorganic substrate (e.g., vermiculite)] and cover with a transparent plastic lid or with cling-film. Stratify for 2 days at 4°C in the dark.</li>
<li>Move pots to a controlled environment room or growth chamber set at 25°C, long-day conditions (16 h light/8 h dark), and 100–150 μmol m<sup>−2</sup>s<sup>−1</sup>. Remove the plastic lids once the seedlings begin to emerge (after 5–7 days).</li>
<li>Transfer the seedlings individually to plastic pots (e.g., 10 <b>×</b> 10 <b>×</b> 9 cm) filled with the same soil mixture watered with fertilizer (20-20-20+TE, Xiamen Vastland Chemical Co., Ltd. China) and cover with a transparent plastic lid for the next 5 days.</li>
<li>Check the plants every 2 days and water them to ensure that the soil is humid, while carefully avoiding excess water in the tray, and fertilize once a week.</li>
<li>Four-week-old plants (five-leaf stage) are ideal to be used for transient expression and visualization of protein subcellular localization.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The quality and age of the plants are essential factors for achieving high protein accumulation. Healthy plants are obtained only following good practices in a well-conditioned growth chamber. A disease-free plant growth environment, careful watering, and regular fertilizing are recommended for desired protein accumulation.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol is suitable to be carried out in <i>N. benthamiana</i> leaves transiently expressing the protein of interest. Alternatively, if stable transgenic lines are available (e.g., Arabidopsis), it will be necessary to carry out a timely setup.</div>
<div class="note">
<span class="note-title">Note:</span> Plants can grow at different pace in different places, therefore rather than follow an age stage a leaf stage can be used instead.</div>
<div class="note">
<span class="note-title">Note:</span> This is an example of the conditions used to grow <i>N. benthamiana</i> plants, as shown in <a href="#fig1">Figure 1</a>. To ensure a continuous supply of plants, we recommend sowing new batches of seeds every week.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1003-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Growing <i>N. benthamiana</i> plants in controlled conditions to transiently express proteins</div>
<p>(A) <i>N. benthamiana</i> seeds sown and covered with cling-film for germination.</p>
<p>(B) Germinated seeds. Remove the plastic lid once the seedlings begin to emerge.</p>
<p>(C) Seedlings transferred individually into plastic square pots and covered with cling-film for 5 days.</p>
<p>(D) Ten-day-old plants.</p>
<p>(E) Four-week-old plants ready to be used for transient protein expression.</p>
<p>Bar = 2 cm.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Elicitor and inhibitor preparation and storage</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>For tracking the relocation of the protein of interest, translation must be stopped to chase steady-state levels, and an elicitation is required to stimulate its release. Cycloheximide (CHX) is a well-known eukaryotic translation inhibitor. In the case of effector or host defense proteins such as <i>Tomato yellow leaf curl virus</i> (TYLCV) C4 protein and <i>Arabidopsis thaliana</i> CPK16, respectively, relocation can be elicited by pathogenic elicitors, including the pathogen-associated molecular-pattern (PAMP) flg22 (<a href="#bib2">Felix et al., 1999</a>), and the immunogenic peptide from Arabidopsis <i>At</i>Pep1 (<a href="#bib11">Yamaguchi et al., 2006</a>). For simplicity, in this protocol, we present the effect of flg22 on the TYLCV protein C4. The impact of any other cues must be evaluated in each case.</p>
<ol start="6">
<li>The lyophilized flg22 peptide (TRLSSGLKINSAKDDAAGLQIA) from <i>Pseudomonas syringae</i> pv. <i>tomato</i> DC3000 (<a href="#bib2">Felix et al., 1999</a>; <a href="#bib3">Gomez-Gomez and Boller, 2000</a>) was chemically synthesized by ABclonal and carefully resuspended in ultra-pure distilled water to generate a storage stock at a concentration of 10 mM. Then, 10 μL aliquots were prepared and stored at −20°C for at least one year.</li>
<li>For protein synthesis inhibitor treatments, CHX was dissolved in dimethyl sulfoxide (DMSO) to a 30 mM stock solution. Then, 100 μL aliquots were prepared and stored at −20°C up to 2–3 months.</li>
<li>Elicitors and inhibitors used in this protocol can be prepared as:</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Solvent</th>
</tr>
</thead>
<tbody>
<tr>
<td>Flg22 peptide</td>
<td>10 mM</td>
<td>1 μM</td>
<td>Milli-Q water</td>
</tr>
<tr>
<td>Cycloheximide (CHX)</td>
<td>30 mM</td>
<td>180 μM</td>
<td>DMSO</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Peptide stocks from lyophilized peptides should be prepared with extreme care to avoid contamination.</div>
<div class="note">
<span class="note-title">Note:</span> Peptides can stick to plastics and other surfaces. To prevent peptide loss, stocks can be prepared using 0.1 M NaCl and 1 mg/mL bovine serum albumin (BSA) (<a href="#bib2">Felix et al., 1999</a>). Be also aware of this when reusing reagent reservoirs and other materials.</div>
<div class="note">
<span class="note-title">Note:</span> The stability of the flg22 peptide decreases at low concentrations. Therefore, it is recommended to keep peptides in concentrated stocks for long-term storage. The 10 mM flg22 stocks prepared in ultra-pure distilled water retain similar elicitor activity for at least one year when stored at −20°C.</div>
<div class="note">
<span class="note-title">Note:</span> Avoid thawing/freezing cycles of the working stock solutions. Flg22 solutions at lower concentration than 100 μM should not be stored for longer than 2–3 months.</div>
<h3 id="sec1.3">Plasmids and cloning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7–10 days</div>
<p>To amplify and clone the gene of interest in the designated binary vector, which is used for <i>Agrobacterium</i>-mediated transformation and <i>in planta</i> expression, use any standard method. Here we describe the plasmids and cloning protocol conducted for the viral ORF C4 from TYLCV, as described in <a href="#bib10">Rosas-Diaz et al. (2018)</a> and <a href="#bib6">Medina-Puche et al. (2020)</a>:</p>
<ol start="9">
<li>The gene sequence of C4 (NCBI: CAD33252) was amplified from the clone of TYLCV as the template (AJ489258; NCBI:txid220938) using specific primers containing recombination sites [for details, please refer to <a href="#bib10">Rosas-Diaz et al. (2018)</a>].</li>
<li>To generate the non-myristoylable version of C4, C4<sub>G2A</sub>, a single nucleotide was substituted in the forward primer to replace G with A in the resulting protein [for details, please refer to <a href="#bib10">Rosas-Diaz et al. (2018)</a>].</li>
<li>The fragment of interest cloned in the pENTR/D-TOPO® entry vector (Thermo Scientific) was then recombined into the corresponding destination vectors through a Gateway LR reaction (Thermo Scientific; <a href="https://www.thermofisher.com/in/en/home/life-science/cloning/gateway-cloning/protocols.html">https://www.thermofisher.com/in/en/home/life-science/cloning/gateway-cloning/protocols.html</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To generate C-terminal tagged proteins, the stop codon must be removed in the reverse primers.</div>
<div class="note">
<span class="note-title">Note:</span> Vectors from the pGWB and ImpGWB are published (<a href="#bib7">Nakagawa et al., 2007a</a>; <a href="#bib8">Nakagawa et al., 2007b</a>; <a href="http://shimane-u.org/nakagawa/gbv.htm">http://shimane-u.org/nakagawa/gbv.htm</a>) and available at Addgene. In this protocol, pGWB5 from the pGWB list and pGWB505 from the ImpGWB list were used as destination vectors for the Gateway LR reaction to generate the constructs pGWB5-C4 and pGWB505-C4<sub>G2A</sub>, respectively.</div>
<div class="note">
<span class="note-title">Note:</span> For proteins with signal sequences or transit peptide sequences at the N-terminal of the protein, it is advisable to fuse the tags at the C-terminus so as to not interfere with their native subcellular localization.</div>
<div class="note">
<span class="note-title">Note:</span> This protocol has been set up for a GFP tag. For other tags, both fluorescent and non- fluorescent, subcellular localization must be evaluated beforehand.</div>
<div class="note">
<span class="note-title">Note:</span> Gene sequences with specific deletions or site-directed mutagenesis can be amplified using PCR-based methods [e.g., using the QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent Technologies)].</div>
<h3 id="sec1.4">Preparation of the <i>Agrobacterium tumefaciens</i> inoculum</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–5 days</div>
<ol start="12">
<li>Three to five days before the experiment, streak out <i>A. tumefaciens</i> (strain GV3101) harboring the corresponding binary vectors on LB agar plates supplemented with appropriate antibiotics and incubate at 28°C for two days. A newly transformed <i>A. tumefaciens</i> clone or a clone streaked out from a glycerol stock stored at −80°C can be used.</li>
<li>One day before conducting the experiment, inoculate an <i>A. tumefaciens</i> clone harboring the corresponding binary vectors in 5 mL of liquid LB medium containing the appropriate antibiotics and grow overnight (12–16 h) at 28°C under continuous agitation (200 rpm).</li>
<li>Antibiotics used in this protocol can be prepared as:</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Solvent</th>
</tr>
</thead>
<tbody>
<tr>
<td>Rifampicin</td>
<td>25 mg/mL</td>
<td>25 μg/mL</td>
<td>DMSO</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>50 mg/mL</td>
<td>50 μg/mL</td>
<td>Milli-Q water (filter sterilized)</td>
</tr>
<tr>
<td>Gentamycin</td>
<td>50 mg/mL</td>
<td>50 μg/mL</td>
<td>Milli-Q water (filter sterilized)</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare 1 mL aliquots and store at −20°C up to 4–6 months.</p>
<div class="note">
<span class="note-title">Note:</span> Other <i>A. tumefaciens</i> strains, such as AGL1, MOG101, or C58C1, are also frequently used for agroinfiltration.</div>
<div class="note">
<span class="note-title">Note:</span> Avoid thawing/freezing cycles of the stock solutions. Rifampicin and kanamycin are light sensitive. Keep the aliquots tightly closed in light-resistant containers.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal anti-GFP (1:5,000)</td>
<td>Abiocode, Agoura Hills, CA, USA</td>
<td>Cat#M0802-3a</td>
</tr>
<tr>
<td>Goat anti-mouse IgG (H+L) HRP conjugate (1:10,000)</td>
<td>Bio-Rad</td>
<td>Cat#170-6516; RRID:AB_11125547</td>
</tr>
<tr>
<td>Goat anti-rabbit IgG-peroxidase (1:10,000)</td>
<td>Jackson ImmunoResearch Laboratories</td>
<td>Cat#111-035-003; RRID:AB_2313567</td>
</tr>
<tr>
<td>Rabbit anti-LhcB1 (1:10,000)</td>
<td>Agrisera</td>
<td>Cat#AS01 004; RRID:AB_1832079</td>
</tr>
<tr>
<td>Rabbit anti-RbcL (1:10,000)</td>
<td>Agrisera</td>
<td>Cat#AS03 037; RRID:AB_2175406</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>Escherichia coli</i> DB3.1</td>
<td>Transgen Biotech, China</td>
<td>Cat#CD531</td>
</tr>
<tr>
<td>
<i>E. coli</i> DH5α</td>
<td>Transgen Biotech, China</td>
<td>Cat#CD501</td>
</tr>
<tr>
<td>
<i>Agrobacterium tumefaciens</i> GV3101</td>
<td>Shanghai Yixin Biological Technology Co., LTD, China</td>
<td>Cat#AC1001</td>
</tr>
<tr>
<td>
<i>Tomato yellow leaf curl virus</i> (TYLCV)</td>
<td>Morilla et al., 2005(<a href="#bib12">Morilla et al., 2005</a>)</td>
<td>NCBI:txid220938</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>2-[N-Morpholino] ethanesulfonic acid (MES)</td>
<td>Amresco</td>
<td>Cat#E169</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#10000218</td>
</tr>
<tr>
<td>Acetone</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#1000418</td>
</tr>
<tr>
<td>Acetosyringone</td>
<td>Sigma-Aldrich</td>
<td>Cat#D134406</td>
</tr>
<tr>
<td>Acrylamide</td>
<td>Sangon Biotech</td>
<td>Cat#B546017</td>
</tr>
<tr>
<td>Agar for bacterial culture medium</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#10000561</td>
</tr>
<tr>
<td>Ammonium persulfate (APS)</td>
<td>Sigma</td>
<td>Cat#A3678</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Sangon Biotech</td>
<td>Cat#A600332</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>Sigma</td>
<td>Cat#B0126</td>
</tr>
<tr>
<td>Coomassie brilliant blue R-250</td>
<td>Amresco</td>
<td>Cat#0472</td>
</tr>
<tr>
<td>Cycloheximide (CHX)</td>
<td>Xiya Reagent, China</td>
<td>Cat#U5298</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Sangon Biotech</td>
<td>Cat#67-68-5</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>Sigma</td>
<td>Cat#43819</td>
</tr>
<tr>
<td>D-Sorbitol</td>
<td>Sigma</td>
<td>Cat#S6021</td>
</tr>
<tr>
<td>Ethylenediaminetetraacetic acid (EDTA)</td>
<td>Sangon Biotech</td>
<td>Cat#A500895</td>
</tr>
<tr>
<td>Ethylene glycol tetraacetic acid (EGTA)</td>
<td>Sigma</td>
<td>Cat# E3889</td>
</tr>
<tr>
<td>Ficoll 400</td>
<td>Sigma</td>
<td>Cat#F2637</td>
</tr>
<tr>
<td>Flg22 peptide (TRLSSGLKINSAKDDAAGLQIA)</td>
<td>ABclonal</td>
<td>N/A</td>
</tr>
<tr>
<td>Gentamycin</td>
<td>Sangon Biotech</td>
<td>Cat#A100304</td>
</tr>
<tr>
<td>GFP-Trap Agarose</td>
<td>ChromoTek, Munich, Germany</td>
<td>Cat#gta-20; RRID:AB_2631357</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sangon Biotech</td>
<td>Cat#A600232</td>
</tr>
<tr>
<td>Glycine</td>
<td>Sangon Biotech</td>
<td>Cat#A610235</td>
</tr>
<tr>
<td>HEPES</td>
<td>Sangon Biotech</td>
<td>Cat#A600264</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>GoldBio</td>
<td>Cat#K-120-50</td>
</tr>
<tr>
<td>Liquid nitrogen</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Magnesium chloride (MgCl<sub>2</sub>)</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#10012818</td>
</tr>
<tr>
<td>Methanol</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#A10014108</td>
</tr>
<tr>
<td>PEG 6000</td>
<td>Sigma</td>
<td>Cat#E3889</td>
</tr>
<tr>
<td>Percoll</td>
<td>GE Healthcare</td>
<td>Cat#17-0891-011</td>
</tr>
<tr>
<td>Ponceau S</td>
<td>Sangon Biotech</td>
<td>Cat#A100860</td>
</tr>
<tr>
<td>Potassium chloride (KCl)</td>
<td>Sigma</td>
<td>Cat#P9541</td>
</tr>
<tr>
<td>Potassium hydroxide (KOH)</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#100019718</td>
</tr>
<tr>
<td>Potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>)</td>
<td>Sangon Biotech</td>
<td>Cat#10017618</td>
</tr>
<tr>
<td>Protease Inhibitor</td>
<td>Sigma-Aldrich</td>
<td>Cat#P8340</td>
</tr>
<tr>
<td>Protease Inhibitor SIGMAFAST<sup>TM</sup>
</td>
<td>Sigma-Aldrich</td>
<td>Cat#S820</td>
</tr>
<tr>
<td>Q5® High-Fidelity DNA Polymerase</td>
<td>New England Biolabs</td>
<td>Cat#M0491S</td>
</tr>
<tr>
<td>Rifampicin</td>
<td>Sigma</td>
<td>Cat#R3501</td>
</tr>
<tr>
<td>Sodium bicarbonate (NaHCO<sub>3</sub>)</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#10018960</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#10019318</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate (SDS)</td>
<td>Sigma</td>
<td>Cat#L5752</td>
</tr>
<tr>
<td>Sodium fluoride (NaF)</td>
<td>Sigma</td>
<td>Cat#30105</td>
</tr>
<tr>
<td>Sodium hydroxide (NaOH)</td>
<td>Sinopharm Chemical Reagent</td>
<td>Cat#10019718</td>
</tr>
<tr>
<td>Sodium orthovanadate (Na<sub>3</sub>VO<sub>4</sub>)</td>
<td>Sigma</td>
<td>Cat#450243</td>
</tr>
<tr>
<td>Sodium phosphate dibasic anhydrous (Na<sub>2</sub>HPO<sub>4</sub>)</td>
<td>Sigma</td>
<td>Cat#S7907</td>
</tr>
<tr>
<td>N,N,N,N-Tetramethylethylenediamine (TEMED)</td>
<td>Sigma</td>
<td>Cat#T9281</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma</td>
<td>Cat#78787</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>Sangon Biotech</td>
<td>Cat#600194</td>
</tr>
<tr>
<td>Tryptone</td>
<td>Oxoid, Thermo</td>
<td>Cat#LP0042</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Sigma</td>
<td>Cat#P9416</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>Oxoid, Thermo</td>
<td>Cat#LP0021</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>pENTR/D-TOPO Cloning Kit</td>
<td>Invitrogen</td>
<td>Cat#45-0218</td>
</tr>
<tr>
<td>Gateway LR Clonase II Enzyme Mix</td>
<td>Invitrogen</td>
<td>Cat#11797-020</td>
</tr>
<tr>
<td>QuikChange Lightning Site-Directed Mutagenesis Kit</td>
<td>Agilent Technologies</td>
<td>Cat#210518-5</td>
</tr>
<tr>
<td>PierceTM BCA Protein Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#B23225</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td><i>Nicotiana benthamiana</i></td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Primer for amplification of C4, forward: CACCATGGGAACCACATC</td>
<td><a href="#bib10">Rosas-Diaz et al., 2018</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Primer for amplification of C4<sub>G2A</sub>, forward: CACCATGGCGAACCACATCTCCAT</td>
<td><a href="#bib10">Rosas-Diaz et al., 2018</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Primer for amplification of C4 or C4<sub>G2A</sub>, reverse without stop codon: ATATATTGAGGGCCTC</td>
<td><a href="#bib10">Rosas-Diaz et al., 2018</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>35S:<i>C4-GFP</i>
</td>
<td><a href="#bib10">Rosas-Diaz et al., 2018</a></td>
<td>pGWB5; NCBI:txid419549; AB289768</td>
</tr>
<tr>
<td>35S:<i>C4</i><sub><i>G2A</i></sub><i>-GFP</i>
</td>
<td><a href="#bib10">Rosas-Diaz et al., 2018</a></td>
<td>pGWB5; NCBI:txid419549; AB289768</td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>ImageJ</td>
<td>Schneider et al., 2012(<a href="#bib14">Schneider et al., 2012</a>)</td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td>GraphPad Prism 7</td>
<td>GraphPad Software</td>
<td><a href="https://www.graphpad.com/">https://www.graphpad.com/</a></td>
</tr>
<tr>
<td>LAS-X 2.01</td>
<td>Leica</td>
<td><a href="https://www.leica-microsystems.com/products/confocal-microscopes/">https://www.leica-microsystems.com/products/confocal-microscopes/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Soil substrate</td>
<td>Pindstrup Mosebrug A/S, Denmark</td>
<td>N/A</td>
</tr>
<tr>
<td>Inorganic substrate (vermiculite)</td>
<td>Shanghai Chunying Horticulture</td>
<td>N/A</td>
</tr>
<tr>
<td>Fertilizer (20-20-20+TE)</td>
<td>Xiamen Vastland Chemical Co., Ltd. China</td>
<td>N/A</td>
</tr>
<tr>
<td>Petri dishes (90Φ)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Sterilization syringe filter for ∼50 mL stock, 0.22 μm pore</td>
<td>Millipore Millex</td>
<td>Cat#SLGP033RS</td>
</tr>
<tr>
<td>0.22 μm Millipore Express PES membrane</td>
<td>Millipore Millex</td>
<td>Cat#SLGP033NB</td>
</tr>
<tr>
<td>2-mL Centrifuge tubes</td>
<td>Eppendorf</td>
<td>Cat#0030120094</td>
</tr>
<tr>
<td>15-mL Tubes</td>
<td>GenBrick</td>
<td>Cat#GP04-6500</td>
</tr>
<tr>
<td>50-mL Tubes</td>
<td>GenBrick</td>
<td>Cat#GP04-7500</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Eppendorf; Scilogex</td>
<td>5810R, 5424; D3024</td>
</tr>
<tr>
<td>Incubator (28°C)</td>
<td>Panasonic</td>
<td>MIR-264-PC</td>
</tr>
<tr>
<td>Incubator (37°C)</td>
<td>Panasonic</td>
<td>MIR-154-PC</td>
</tr>
<tr>
<td>Magnetic stirrer with heater</td>
<td>IKA</td>
<td>IKA C-MAG HS7</td>
</tr>
<tr>
<td>Biopsy Punch with Plunger, ID 4.0 mm, OD 4.36 mm, Green/Blue</td>
<td>Miltex®</td>
<td>Cat#15110-40</td>
</tr>
<tr>
<td>Microscope slides</td>
<td>FanYi</td>
<td>Cat#7105p</td>
</tr>
<tr>
<td>Coverslips</td>
<td>Citotest</td>
<td>Cat#85850-0005</td>
</tr>
<tr>
<td>Tissue homogenizer</td>
<td>QIAGEN</td>
<td>TissueLyser II</td>
</tr>
<tr>
<td>Waring blender</td>
<td>Local brand</td>
<td>N/A</td>
</tr>
<tr>
<td>Revolver lab rotator</td>
<td>Labnet</td>
<td>Cat# H5600</td>
</tr>
<tr>
<td>Miracloth</td>
<td>Millipore Calbiochem®</td>
<td>Cat#475855-1R</td>
</tr>
<tr>
<td>PVDF membranes</td>
<td>Millipore</td>
<td>Cat#IPVH00010</td>
</tr>
<tr>
<td>Autoclave</td>
<td>Sanyo</td>
<td>MLS-3782</td>
</tr>
<tr>
<td>Confocal microscope</td>
<td>Leica TCS SP8</td>
<td>Leica Microsystems</td>
</tr>
<tr>
<td>Water purification system</td>
<td>Millipore</td>
<td>Millipore integral 10</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Preparation of media</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p><b>LB medium</b></p>
<p>For making 1 L of LB medium, mix 10 g bacteriological tryptone, 5 g NaCl, 5 g yeast extract in 500 mL of double-distilled water (ddH<sub>2</sub>O) with a magnetic stirrer until the powders are dissolved completely, adjust the pH to 7.4 with NaOH, make up the volume to 1 L, and autoclave at 121°C for 20 min. Store at room temperature (23°C–25°C). For solid LB, prepare LB medium as described above and add 7.5 g agar after pH adjustment.</p>
<table id="undtbl3">
<caption>LB medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Weight/Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tryptone</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>5 g</td>
</tr>
<tr>
<td>Agar</td>
<td>7.5 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>To make up the volume</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at room temperature (23°C–25°C) for maximum 2 weeks.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Bacteriological tryptone can be replaced by bacteriological peptone.</div>
<h3 id="sec2.2">Preparation of stock solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<ul>
<li>
<b>Acetosyringone (AS, 150 mM)</b><i>:</i> Dissolve 294.3 mg AS in 10 mL of DMSO and store in 1 mL aliquots at −20°C for up to 1 month.</li>
<li>
<b>Ammonium persulfate (APS, 10% w/v)</b>: Dissolve 1 g APS in 10 mL of distilled water and store at 4°C for up to 3 weeks.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> APS undergoes decaying in solution, so replace the stock solution every 2–3 weeks.</div>
<ul>
<li>
<b>Bovine serum albumin (BSA, 1% w/v)</b>: Dissolve 100 mg BSA in 10 mL of distilled water, and store in 1 mL aliquots at 4°C or −20°C. If stored at 4°C or −20°C, BSA powders and BSA solutions are stable for 2 years.</li>
<li>
<b>Cycloheximide (CHX, 30 mM)</b>: Dissolve 84.4 mg CHX in 10 mL of Milli-Q water or DMSO. Sterilize with a 0.2 μm filter if necessary, and store in 1-mL aliquots at −20°C in the dark for up to 2–3 months.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> CHX is a potent inhibitor of cytosolic protein synthesis, promotes the formation of misfolded and toxic translation products, is a dermal irritant, and its ingestion can be lethal. Therefore, handle with extreme caution, wear gloves, goggles and lab coat. Follow the manufacturer’s recommendations when preparing, storing, and handling CHX. In the event of accidental exposure, consult the material safety data sheet provided by the manufacturer.</div>
<ul>
<li>
<b>Dithiothreitol (DTT, 1 M)</b>: Dissolve 1.54 g DTT in 10 mL of distilled water. Make 1 mL aliquots and store them at −20°C for 4–6 months.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Wear gloves and prepare in a fume hood.</div>
<ul>
<li>
<b>Ethylenediaminetetraacetic acid [EDTA, pH 8 (0.5 M)]</b>: Dissolve 14.61 g EDTA in 80 mL of distilled water. Adjust pH to 8 with 1 M NaOH solution. Sterilize by autoclaving at 121°C for 20 min, and store at 4°C for 4–6 months.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Initially, the solution will be turbid and turn clear only when the pH reaches 8.</div>
<ul>
<li>
<b>Ethylene glycol tetraacetic acid [EGTA, pH 8 (0.5 M)]</b>: Dissolve 19.02 g EGTA in 100 mL of distilled water. Adjust pH to 8 with 1 M NaOH solution. Sterilize by autoclaving at 121°C for 20 min, and store at 4°C for 4–6 months.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Initially, the solution will be turbid and turn clear only when the pH reaches 8.</div>
<ul>
<li>
<b>Ficoll 400</b>: Use fresh powder.</li>
<li>
<b>Flg22 peptide (10 mM)</b>: Dissolve the powder provided by the manufacturer in Milli-Q water to a 10 mM stock concentration. Store in 10 μL aliquots at −20°C for one year.</li>
<li>
<b>Gentamycin (50 mg/mL)</b>: Dissolve 1 g gentamycin in 20 mL of Milli-Q water. Sterilize with a 0.2 μm filter, and store in 1 mL aliquots at −20°C in the dark for 4–6 months.</li>
<li>
<b>Glycerol (50% v/v)</b>: Take 50 mL glycerol and add distilled water to make up the volume to 100 mL. Sterilize by autoclaving at 121°C for 20 min, and store at room temperature (23°C–25°C) for 4–6 months.</li>
<li>
<b>Hepes-KOH, pH 8 (1 M)</b>: Dissolve 23.83 g Hepes in 100 mL of distilled water. Adjust pH to 8 with 1 M KOH solution and store at 4°C for 4–6 months.</li>
<li>
<b>Hepes-KOH, pH 7.4 (1 M)</b>: Dissolve 23.83 g Hepes in 100 mL of distilled water. Adjust pH to 7.4 with 1 M KOH solution and store at 4°C for 4–6 months.</li>
<li>
<b>Kanamycin (50 mg/mL)</b>: Dissolve 1 g kanamycin in 20 mL of Milli-Q water. Sterilize with a 0.2 μm filter, and store in 1 mL aliquots at −20°C in the dark for 4–6 months.</li>
<li>
<b>Magnesium chloride (MgCl</b><sub><b>2</b></sub><b>, 1 M)</b>: Dissolve 9.521 g MgCl<sub>2</sub> in 100 mL of distilled water, and store at room temperature (23°C–25°C) for maximum 1 year.</li>
<li>
<b>2-[N-morpholino] ethanesulfonic acid (MES, 1 M)</b>: Dissolve 1.952 g MES in 10 mL of distilled water, and store at 4°C for maximum 1 month.</li>
<li>
<b>Percoll</b>: Use as provided.</li>
<li>
<b>PEG 6000</b>: Use fresh powder.</li>
<li>
<b>Rifampicin (25 mg/mL)</b>: Dissolve 500 mg rifampicin in 20 mL of DMSO. Sterilize with a 0.2 μm filter, and store in 1- mL aliquots at −20°C in the dark for 4–6 months.</li>
<li>
<b>Sodium bicarbonate (NaHCO</b><sub><b>3</b></sub><b>, 1 M)</b>: Dissolve 8.401 g NaHCO<sub>3</sub> in 100 mL of distilled water, and store at room temperature (23°C–25°C) for 4–6 months.</li>
<li>
<b>Sodium chloride (NaCl, 5 M)</b>: Dissolve 29.22 g NaCl in distilled water. Make up the volume to 100 mL with distilled water. Sterilize by autoclaving at 121°C for 20 min, and store at room temperature (23°C–25°C) for 4–6 months.</li>
<li>
<b>Sodium dodecyl sulfate (SDS, 10% w/v)</b>: Dissolve 10 g SDS in 80 mL distilled water in a beaker. Put a magnetic flea and place the beaker on a magnetic stirring plate to mix the solution. Once the powder is dissolved completely, make up the volume to 100 mL with distilled water and store at room temperature (23°C–25°C) for 4–6 months.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> SDS is highly hazardous. Wear gloves and masks and work in a fume hood while handling the powder to avoid inhalation.</div>
<ul>
<li>
<b>N,N,N′,N′-tetramethylethylenediamine (TEMED)</b>: Use fresh from the bottle.</li>
<li>
<b>Tris-HCl, pH 8.8 (1.5 M)</b>: Dissolve 181.65 g Tris Base in 400 mL distilled water and then add concentrated HCl to bring pH to 8.8. Make up the volume to 1 L, autoclave for sterilization, and store at room temperature (23°C–25°C) for 4–6 months.</li>
<li>
<b>Tris-HCl, pH 6.8 (1 M)</b>: Dissolve 121.1 g Tris Base in 400 mL distilled water and then add concentrated HCl to bring pH to 6.8. Make up the volume to 1 L, autoclave for sterilization, and store at room temperature (23°C–25°C) for 4–6 months.</li>
</ul>
<h3 id="sec2.3">Preparation of buffers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<table id="undtbl4">
<caption>Infiltration buffer. Freshly prepared at room temperature (23°C–25°C). Use immediately.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>1 M</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>MES</td>
<td>1 M</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>AS</td>
<td>150 mM</td>
<td>150 μM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Distilled water</td>
<td> </td>
<td> </td>
<td>Up to 100 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Always prepare fresh infiltration buffer before use.</div>
<table id="undtbl5">
<caption>Chloroplast isolation buffer. Stored at 4°C for maximum 1 month.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes-KOH, pH 8</td>
<td>1 M</td>
<td>20 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>1 M</td>
<td>5 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>EDTA, pH 8</td>
<td>0.5 M</td>
<td>5 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>EGTA, pH 8</td>
<td>0.5 M</td>
<td>5 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>1 M</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>D-sorbitol</td>
<td>--</td>
<td>0.33 M</td>
<td>6 g</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td> </td>
<td> </td>
<td>1 tablet</td>
</tr>
<tr>
<td>Distilled water</td>
<td> </td>
<td> </td>
<td>Up to 100 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>HS buffer. Stored at 4°C for maximum 1 month.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes-KOH, pH 8</td>
<td>1 M</td>
<td>50 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>D-sorbitol</td>
<td>--</td>
<td>0.33 M</td>
<td>6 g</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td> </td>
<td> </td>
<td>1 tablet</td>
</tr>
<tr>
<td>Distilled water</td>
<td> </td>
<td> </td>
<td>Up to 100 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl7">
<caption>HM buffer. Stored at 4°C for maximum 1 month.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes-KOH, pH 8</td>
<td>1 M</td>
<td>50 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>1 M</td>
<td>5 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td> </td>
<td> </td>
<td>1 tablet</td>
</tr>
<tr>
<td>Distilled water</td>
<td> </td>
<td> </td>
<td>Up to 100 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl8">
<caption>Percoll solution. Freshly prepared on ice or at 4°C. Use immediately.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Working concentration</th>
<th>Amount for 120 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Percoll</td>
<td>95% (w/v)</td>
<td>114 mL</td>
</tr>
<tr>
<td>PEG 6000</td>
<td>3% (w/v)</td>
<td>3.6 g</td>
</tr>
<tr>
<td>Ficoll 400</td>
<td>1% (w/v)</td>
<td>1.2 g</td>
</tr>
<tr>
<td>BSA</td>
<td>1% (w/v)</td>
<td>1.2 g</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td>--</td>
<td>1 tablet</td>
</tr>
<tr>
<td>Distilled water</td>
<td> </td>
<td>Up to 120 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Always prepare fresh percoll solution before use.</div>
<table id="undtbl9">
<caption>Gradient mixture. Freshly prepared on ice or at 4°C. Use immediately.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes-KOH, pH 8</td>
<td>1 M</td>
<td>25 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>EDTA pH 8</td>
<td>0.5 M</td>
<td>10 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>D-sorbitol</td>
<td> </td>
<td>5% (w/v)</td>
<td>5 g</td>
</tr>
<tr>
<td>Distilled water</td>
<td> </td>
<td> </td>
<td>Up to 100 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Always prepare fresh gradient mixture solution before use.</div>
<table id="undtbl10">
<caption>Percoll gradient solution. Stored at −20°C for maximum 1 month.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Percoll solution</th>
<th>Gradient mixture</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bottom</td>
<td>51 mL</td>
<td>9 mL</td>
</tr>
<tr>
<td>Top</td>
<td>58.8 mL</td>
<td>81.2 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store the Bottom and Top solutions at −20°C for maximum 1 month.</div>
<table id="undtbl11">
<caption>Percoll gradient preparation. Stored at 4°C for 12–16 h.</caption>
<thead>
<tr>
<th>Percoll solution</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bottom</td>
<td>3 mL</td>
</tr>
<tr>
<td>Top</td>
<td>7 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store the Bottom and Top solutions at −20°C for maximum 1 month.</div>
<div class="note">
<span class="note-title">Note:</span> First, take the Bottom solution in a glass vial and gently pour the Top solution without disturbing the Bottom. Be gentle while adding Top solution and avoid mixing.</div>
<div class="note">
<span class="note-title">Note:</span> Gradients can be poured a day before and stored at 4°C.</div>
<table id="undtbl12">
<caption>Protein extraction buffer. Stored at 4°C for maximum 1 month.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes-KOH, pH7.4</td>
<td>1 M</td>
<td>20 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>EDTA, pH 8</td>
<td>0.5 M</td>
<td>2 mM</td>
<td>0.4 mL</td>
</tr>
<tr>
<td>EGTA, pH 8</td>
<td>50 mM</td>
<td>2 mM</td>
<td>4 mL</td>
</tr>
<tr>
<td>NaF</td>
<td>Use fresh powder</td>
<td>25 mM</td>
<td>0.1049 g</td>
</tr>
<tr>
<td>Na<sub>3</sub>VO<sub>4</sub>
</td>
<td>Use fresh powder</td>
<td>1 mM</td>
<td>0.01839 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>50% (v/v)</td>
<td>10% (v/v)</td>
<td>20 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>5 M</td>
<td>150 mM</td>
<td>3 mL</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Directly from bottle</td>
<td>0.5% (v/v)</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td>-</td>
<td>-</td>
<td>1 tablet</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>-</td>
<td>To make up 100 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> It is recommended to use freshly prepared buffer. If required, buffer w/o protease inhibitor can be prepared and stored at 4°C for up to one month.</div>
<table id="undtbl13">
<caption>Laemmli buffer sample (4<b>×</b>) (Wang et al., 2016(<a href="#bib13">Wang et al., 2016</a>))</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl, pH 6.8</td>
<td>1 M</td>
<td>200 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>100% (v/v)</td>
<td>40% (v/v)</td>
<td>4 mL</td>
</tr>
<tr>
<td>SDS</td>
<td>Powder</td>
<td>8%</td>
<td>0.8 g</td>
</tr>
<tr>
<td>DTT</td>
<td>1 M</td>
<td>400 mm</td>
<td>0.4 mL</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>1%</td>
<td>0.02%</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>-</td>
<td>To make up 100 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Sterilize using filter sterilization. Prepare 1 mL aliquots and store at −20°C for maximum 6 months.</div>
<table id="undtbl14">
<caption>SDS-PAGE: Resolving Gel. Freshly prepared at room temperature (23°C–25°C). Use immediately.</caption>
<thead>
<tr>
<th rowspan="2">Reagent</th>
<th rowspan="2">Stock concentration</th>
<th colspan="2">Gel concentration (required for 10 mL)</th>
</tr>
<tr>
<th>10%</th>
<th>12%</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acrylamide mix</td>
<td>30%</td>
<td>3.3 mL</td>
<td>4 mL</td>
</tr>
<tr>
<td>Tris-HCl, pH 8.8</td>
<td>1.5 M</td>
<td>2.5 mL</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>SDS</td>
<td>10%</td>
<td>0.1 mL</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>APS</td>
<td>10%</td>
<td>0.1 mL</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>TEMED</td>
<td>-</td>
<td>0.004 mL</td>
<td>0.004 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>4 mL</td>
<td>3.3 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl15">
<caption>SDS-PAGE: Stacking Gel. Freshly prepared at room temperature (23°C–25°C). Use immediately.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Gel concentration4.5% (5 mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acrylamide mix</td>
<td>30%</td>
<td>0.83 mL</td>
</tr>
<tr>
<td>Tris-HCl, pH 6.8</td>
<td>1 M</td>
<td>0.63 mL</td>
</tr>
<tr>
<td>SDS</td>
<td>10%</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>APS</td>
<td>10%</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>TEMED</td>
<td>-</td>
<td>0.005 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>2.7 mL</td>
</tr>
</tbody>
</table>
<table id="undtbl16">
<caption>Running buffer (1<b>×</b>). Freshly prepared at room temperature (23°C–25°C). Use immediately.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl, pH 8.3</td>
<td>1 M</td>
<td>25 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>SDS</td>
<td>10% (v/v)</td>
<td>0.1%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Glycine</td>
<td>Powder</td>
<td>192 mM</td>
<td>144 g</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>-</td>
<td>To make up 1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl17">
<caption>Transfer buffer (1<b>×</b>). Freshly prepared at room temperature (23°C–25°C). Use immediately.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Working concentration</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl, pH 8.3</td>
<td>1 M</td>
<td>25 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>Glycine</td>
<td>Powder</td>
<td>192 mM</td>
<td>144 g</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>-</td>
<td>To make up 1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl18">
<caption>Coomassie blue brilliant (CBB) staining solution. Stored at room temperature (23°C–25°C) for maximum 1 year.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Working concentration</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Coomassie brilliant blue R-250</td>
<td>0.1% (w/v)</td>
<td>1 g</td>
</tr>
<tr>
<td>Methanol</td>
<td>50% (v/v)</td>
<td>500 mL</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>10% (v/v)</td>
<td>100 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>To make up 1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl19">
<caption>Coomassie blue brilliant (CBB) destaining solution. Stored at room temperature (23°C–25°C) for maximum 1 year.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Working concentration</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Methanol</td>
<td>50% (v/v)</td>
<td>500 mL</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>10% (v/v)</td>
<td>100 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>To make up 1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl20">
<caption>Ponceau S staining solution. Stored at room temperature (23°C–25°C) for maximum 1 year.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Working concentration</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ponceau S</td>
<td>0.1% (w/v)</td>
<td>1 g</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>5% (v/v)</td>
<td>50 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>To make up 1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl21">
<caption>PBS buffer (1<b>×</b>). Stored at room temperature (23°C–25°C) for maximum 1 year.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Working concentration</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>140 mM</td>
<td>8.2 g</td>
</tr>
<tr>
<td>KCl</td>
<td>30 mM</td>
<td>0.22 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>1 mM</td>
<td>1.42 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1.8 mm</td>
<td>0.24 g</td>
</tr>
<tr>
<td>Distilled water</td>
<td>--</td>
<td>To make up 1 L</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Adjust the pH to 7.4, and autoclave.</div>
<p>PBST buffer (membrane wash buffer) (1<b>×</b>). Freshly prepared at room temperature (23°C–25°C). Use immediately.</p>
<p>Add 0.02% (v/v) Tween-20 to PBS and mix well without making froth.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Transient expression in <i>N. benthamiana</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>Here we show an example with C4-GFP and C4<sub>G2A</sub>-GFP proteins. The non-myristoylable version of the C4 protein, C4<sub>G2A</sub>, which localizes in chloroplasts exclusively, was included as control to confirm in further steps that (i) our chloroplast fractionation is working properly, and (ii) there are no changes in the stability of the chloroplast-localized proteins upon flg22 treatment.</p>
<ol>
<li>Culture the <i>A. tumefaciens</i> strain GV3101 harboring the corresponding binary vectors in 10 mL liquid LB with the appropriate antibiotics at 28°C, shaking at 180 rpm overnight (12–16 h).</li>
<li>Next morning, harvest the bacteria by centrifugation at 4,000 × <i>g</i> for 10 min and resuspend in the infiltration buffer (10 mM MgCl<sub>2</sub>, 10 mM MES pH 5.6, 150 mM acetosyringone) to an OD<sub>600</sub> = 0.2-0.6. Volume of the resuspension buffer can be adjusted according to required OD.</li>
<li>Incubate the bacterial suspensions at room temperature (23°C–25°C) in dark for 2 h to allow acetosyringone to induce the expression of <i>vir</i> genes.</li>
<li>Infiltrate into the abaxial side of leaves of four-week-old <i>N. benthamiana</i> plants with a 1 mL needle-less syringe. According to our experience, it is suggested to infiltrate two leaves per plant to obtain the desired levels of protein. It is recommended to use the third and fourth youngest leaves, which would be fully expanded. See <a href="#fig2">Figure 2</a>.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1003-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Transient expression in N. benthamiana leaves, and confirmation of protein accumulation by confocal microscopy</div>
<p>(A) Selected leaves from a four-wee-old plant.</p>
<p>(B) Selected leaves agroinfiltrated into the abaxial side with a 1 mL needle-less syringe.</p>
<p>(C) Localization of the WT and the non-myristoylable (C4<sub>G2A</sub>) C4 versions fused to GFP. Zoom-out pictures: scale bar = 50 μm. Zoom-in pictures: scale bar = 10 μm. AF, autofluorescence.</p>
<p>(D) Two days post-agroinfiltration, CHX is infiltrated with a 1 mL needle-less syringe into the abaxial side of the same leaves previously agroinfiltrated.</p>
<p>(E) Two hours later, flg22 (or mock solution) is infiltrated with a 1 mL needle-less syringe into the abaxial side of the same leaves previously treated with CHX.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For experiments that require co-infiltration, <i>A. tumefaciens</i> suspensions carrying different constructs need to be mixed at a 1:1 ratio before infiltration.</div>
<ol start="5">
<li>Infiltrate the suspension until the whole leaf area appears translucent. Use as much of the bacterial suspension as necessary.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After the infiltration, ensure that the infiltrated leaves do not touch each other and gently dry the excess bacterial suspension with a paper towel.</div>
<div class="note">
<span class="note-title">Note:</span> To overcome host antiviral RNA silencing, viruses encode RNA silencing suppressors (VSRs) that efficiently inhibit host antiviral responses by interacting with the key components of cellular silencing machinery. The use of VSRs is justified when the accumulation of the protein of interest is low., Therefore, co-infiltration with a silencing suppressor (e.g., P19 from the tombusvirus <i>Tomato bushy stunt virus</i> (<a href="#bib9">Qu and Morris, 2002</a>)) is required to prevent silencing of the transgene and allow high protein accumulation.</div>
<div class="note">
<span class="note-title">Note:</span> In order to reduce the effects of plant-to-plant variation, it is recommended to express the protein of interest in at least six individual leaves from three different plants. Assays should be performed in at least three biological replicates.</div>
<h3 id="sec3.2">Confirmation of protein accumulation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 h</div>
<p>It is necessary to carry out this step before continuing with the experiment to confirm that the protein of interest accumulates homogeneously in the infiltrated leaves.</p>
<ol start="6">
<li>Two days after the A. tumefaciens infiltration, cut small leaf discs from the plants expressing C4-GFP and C4G2A-GFP with a corkborer with plunger (4 mm diameter).</li>
<li>Place the discs on a microscope slide. Put a drop of water and cover with a coverslip.</li>
<li>Observe under a confocal microscope using a GFP filter with excitation/emission: 488/500–550 nm. In this work, images were obtained with a Leica TCS SP8 confocal microscope (Leica Microsystems) LAS-X 2.01 software using the preset settings for GFP (Ex: 488 nm, Em: 500–550 nm). See <a href="#fig2">Figure 2</a>.</li>
<li>Chloroplast can be visualized using chlorophyll autofluorescence at excitation/emission: 488/650–700 nm. See <a href="#fig2">Figure 2</a>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> One may use any confocal microscope with the given settings.</div>
<h3 id="sec3.3">Inhibitor and elicitor treatments</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>Here, we show the example with the TYLCV C4 protein.</p>
<ol start="10">
<li>Two days after the <i>Agrobacterium</i> infiltration, and after verifying protein accumulation, infiltrate the <i>N. benthamiana</i> leaves expressing the viral protein C4-GFP with the translation inhibitor CHX (180 μM). See <a href="#fig2">Figure 2</a>.</li>
<li>After 2 h of incubation, infiltrate one-half of the leaf with the PAMP flg22 (1 μM), and the other half with water as mock. See <a href="#fig2">Figure 2</a>.</li>
<li>For protein and chloroplast isolations, harvest 6 leaves (2 leaves from each plant) after 6–8 h and proceed immediately using fresh tissue.</li>
<li>For visualization, harvest the leaf samples after 12–24 h and confirm the subcellular localization under the confocal microscope.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For steps 12 and 13, the incubation times must be adjusted according to the accumulation levels of the protein under study.</div>
<div class="note">
<span class="note-title">Note:</span> Under the working conditions indicated throughout this protocol, CHX incubations of less than 24 h did not produce a toxic effect on <i>N. benthamiana</i> leaves.</div>
<h3 id="sec3.4">Protein relocalization analysis using fluorescence microscopy</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h</div>
<h4 id="sec3.4.1">Visualization of subcellular localization and relocalization</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>(See <a href="#fig3">Figure 3</a>)</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1003-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Protein relocalization analysis in <i>N. benthamiana</i> leaves using fluorescence and biochemical methods</div>
<p>(A) Selected leaves from a four-week-old plant after treatment. A detail of the corkborer with plunger used to collect leaf discs for confocal microscopy is provided in the picture.</p>
<p>(B) Detail of the collected leaf discs placed on a microscope slide and cover with a coverslip.</p>
<p>(C) Leica TCS SP8 confocal microscope (Leica Microscopy) used in this protocol.</p>
<p>(D) Localization of C4-GFP following treatment with 1 μM flg22 (16 h post-treatment) in the presence of CHX (50 mg/mL, 2 h prior to flg22 treatment). Scale bar, 10 μm. AF, autofluorescence. For detailed information regarding protein quantification using ImageJ, see <a href="#fig4">Figure 4</a> and <a href="#mmc1">Methods video S1</a>.</p>
<p>(E) Intact chloroplasts separated at the interphase of a two-step Percoll gradient.</p>
<p>(F) Western blots showing the differential accumulation of C4-GFP protein in PM and chloroplasts upon flg22 treatment. The chloroplastic form C4<sub>G2A</sub>-GFP shows no change upon elicitation. Asterisks indicate unidentified bands, potential degradation products of the PM C4-GFP pool. Protein size in kDa are indicated. Numbers below blots indicate relative intensity (for further information, see <a href="#fig5">Figure 5</a> and <a href="#mmc2">Methods video S2</a>). Total: total protein; Chloroplast: protein from the isolated chloroplasts fraction. C4-GFP: PM-localized C4-GFP; cpC4-GFP: chloroplast form of C4-GFP. Panel reprinted with permission from <a href="#bib6">Medina-Puche et al. (2020)</a>.</p>
</figcaption>
</figure>
<ol start="14">
<li>After 16 h of flg22 or mock treatment, cut small leaf discs from the plants expressing C4-GFP and C4<sub>G2A</sub>-GFP with a corkborer with plunger (4 mm diameter).</li>
<li>Place the discs on a microscope slide. Put a drop of water and cover with a coverslip.</li>
<li>Observe under a confocal microscope using GFP filter with excitation/emission: 488/500–550 nm.</li>
<li>Chloroplast can be visualized using chlorophyll autofluorescence at excitation/emission: 488/650–700 nm.</li>
<li>The C4 protein localizes to both plasma membrane (PM) and chloroplasts, whereas C4<sub>G2A</sub> localizes to chloroplasts only.</li>
<li>To quantify the proportion of protein localized to PM and chloroplasts with and without elicitation by flg22, the ratio of GFP-fluorescence intensity at PM and chloroplast can be used. For this quantification, the confocal images can be analyzed using the ImageJ software (<a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a>) as described in the next section.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> For step 19 and further, the open-source image processing package Fiji (Fiji Is Just ImageJ; <a href="https://fiji.sc/">https://fiji.sc/</a>) can be used as well. Note: For step 14, elicitation time must be adjusted for every protein of interest.</div>
<h4 id="sec3.4.2">Quantification of protein subcellular localization using ImageJ</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 h</div>
<p>(See <a href="#fig3">Figures 3</a> and <a href="#fig4">4</a>, and <a href="#mmc1">Methods video S1</a>)</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1003-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Quantification of protein subcellular localization using ImageJ</div>
<p>(A) Screenshots showing step-by-step pipeline using ImageJ for confocal images.</p>
<p>(B) Analysis of the signal intensity values obtained from the confocal images corresponding to one pair of pictures from all the images from one replicate.</p>
<p>(C) Column chart representing the calculated PM-to-chloroplasts intensity ratio of C4-GFP protein reprinted with permission from <a href="#bib6">Medina-Puche et al. (2020)</a>. Bars represent SE of n = 6. Asterisks indicate a statistically significant difference (∗∗∗p = 0.0004) according to a two-tailed comparisons t test.</p>
</figcaption>
</figure>
<ol start="20">
<li>Analysis of the confocal images can be done with the ImageJ software (<a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a>; see <a href="#fig4">Figure 4</a> and <a href="#mmc1">Methods video S1</a>) as follows:
<ol type="a">
<li>Place all images for analysis into one folder. All images should have the same channel. Click on “Image” menu in the toolbar and make sure that “Green” (if your proteins are tagged to GFP) is selected among “Lookup Tables (LUT)” options.</li>
<li>Next, click on the “Process” menu in the toolbar and select “Substract Background”. Unclick all the options and set 50 by default.</li>
<li>Then, click on “Image” in the toolbar, select “Adjust”, and click on “Brightness/Contrast”. Set same value for all images.</li>
<li>In the toolbar, click on the “Hand lens” icon. Then, set a defined “Region of Interest” (ROI) size in the “ROI Manager menu”, save, and apply to all images during analysis.</li>
<li>Then check the “Grey Mean Value” icon in the “Set Measurements” icon of the “Analyze” menu in the toolbar.</li>
<li>Click “Measure” under the “Analyze” menu to get the intensity values.</li>
<li>Export the values of the selected areas into a spreadsheet.</li>
<li>Calculate the final relative quantification values as the ratio of PM to chloroplast fluorescence intensity.</li>
</ol>
</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1003-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. A step-by-step video with the quantification of the protein subcellular localization using ImageJ, related to step 20</div>
</figcaption>
</figure>
<h3 id="sec3.5">Biochemical analysis of protein relocalization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<h4 id="sec3.5.1">Chloroplast isolation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 h</div>
<div class="critical">
<span class="critical-title">Critical:</span> For isolation of intact chloroplasts (<a href="#fig3">Figure 3</a>), use fresh tissue. Do not freeze the samples.</div>
<div class="note">
<span class="note-title">Note:</span> Check the expression of the GFP-tagged protein by confocal microscopy prior to chloroplast isolation.</div>
<div class="note">
<span class="note-title">Note:</span> Carry out the chloroplast isolation in low or dim light (20 μmol m<sup>−2</sup> s<sup>−1</sup>).</div>
<div class="note">
<span class="note-title">Note:</span> It is better to perform the experiment in a cold room, if possible. Otherwise, keep everything at 4°C; solutions, rotors, equipment pre-cooled to 4°C; centrifugations should be performed at 4°C (unless otherwise indicated).</div>
<div class="note">
<span class="note-title">Note:</span> Plant leaf materials should be collected into a beaker placed on ice.</div>
<div class="note">
<span class="note-title">Note:</span> Use wide-bore 1 mL tips for the isolation of chloroplasts. Cut 2 mm off the tip of 1 mL pipetman tips using a razor blade, autoclave and dry them.</div>
<div class="note">
<span class="note-title">Note:</span> Please refer to (<a href="#bib1">Dogra et al., 2019</a>; <a href="#bib5">Kauss et al., 2012</a>).</div>
<ol start="21">
<li>Grind the tissue (around 6 leaves of four-weeks-old <i>N. benthamiana</i> plants in 50 mL of isolation buffer in a Waring blender (3 times, each for 3 s).</li>
<li>Filter the homogenate through 4 layers of Miracloth into a 50 mL polypropylene tube and centrifuge at 400 × g for 8 min. Collect the pellet and resuspend in 1 mL of isolation buffer.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> In case of using <i>Arabidopsis</i> transgenic plants, rosette leaves of 50 to 60 3-weeks-old plants should be used for chloroplasts isolation.</div>
<div class="note">
<span class="note-title">Note:</span> If using more than one tube, suspend the pellet of each tube in approximately 250 μl isolation buffer and pool all the suspensions together.</div>
<ol start="23">
<li>Load approximately 500–1000 μL of chloroplast suspension on the two-step Percoll gradient. Centrifuge in a swing-out rotor at 200 × <i>g</i> for 20 min with brakes off.</li>
<li>Remove broken chloroplasts and Percoll till the layer of intact chloroplasts between the two Percoll steps. Take intact chloroplasts using a wide-bore tip, place into 50 mL tube (chloroplasts from several gradients can be combined), fill with HS buffer up to 50 mL invert the tube several times, and centrifuge at 400 × <i>g</i> for 6 min.</li>
<li>Resuspend the chloroplast pellet in another 50 mL of HS buffer and centrifuge at 400 × <i>g</i> in for 6 min. Resuspend chloroplast pellet in 1 mL of HS buffer.</li>
<li>Take a 10 μL aliquot, add 990 μL of 80% acetone to it, vortex, centrifuge for 5 min at 21,000 × <i>g</i> at room temperature (23°C–25°C). Take clear supernatant and measure OD<sub>652</sub> using quartz or glass cuvette. Calculate chlorophyll concentration as:
<p class="chem">Chl (mg/mL) = (OD<sub>652</sub> <b>×</b> dilution factor)/36</p>
</li>
</ol>
<p>In our case, the dilution factor was 100. This normalization will be used in all downstream experiments.</p>
<ol start="27">
<li>For the lysis of the intact chloroplasts, take such a volume intact chloroplasts suspension that corresponds to a desired amount of chlorophyll, (you can take intact chloroplasts corresponding to 200 μg of chlorophyll amount as shown in #26) and pellet down at 2600 × <i>g</i> for 5 min. Resuspend the pellet in 150 μL of HM buffer and keep on ice for 10 min.</li>
<li>Centrifuge at 2600 <b>×</b> <i>g</i> for 5 min. Separate the supernatant as stroma fraction (150 μL) and the pellet as membrane (envelopes and thylakoids) fraction. Remove contaminants by washing the membrane fraction with HS buffer.</li>
</ol>
<h4 id="sec3.5.2">Protein isolation from plant tissue and isolated chloroplasts and western blotting</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>(See <a href="#fig3">Figure 3</a>)</p>
<ol start="29">
<li>For isolating total proteins from fresh plant tissue and total proteins from isolated chloroplasts:
<ol type="a">
<li>Collect 2 leaf discs with an 8-mm punch from each leaf (usually 6 leaves from 3 plants in case of <i>N. benthamiana</i>) to be used for total protein isolation. Grind the leaf discs in liquid nitrogen and collect in 2 mL tubes. Keep the tubes at −80°C or in liquid nitrogen, and wait until chloroplasts are isolated (as described in steps 21–28).</li>
<li>Isolate intact chloroplast from the leaves from which leaf discs are extracted (as described in steps 21–28).</li>
<li>Resuspend the chloroplast pellet from (b), corresponding to 1000 μg of chlorophyll (concentration usually 1 μg/ul), in 1 mL protein extraction buffer.</li>
<li>Resuspend the leaf disc powder from (a) in 500 ul protein extraction buffer.</li>
<li>Incubate the tubes from (c) and (d) for 30 min on a revolver lab rotator at 4°C.</li>
<li>Centrifuge at 21,000 <b>×</b> <i>g</i> for 30 min at 4°C. Collect the supernatant in a new tube and repeat the centrifugation step.</li>
<li>Final supernatants collected from the tubes from (c) and (d) represent chloroplast and total proteins, respectively.</li>
</ol>
</li>
<li>Take 150 μL of total and chloroplast protein samples and add 50 μL of 4<b>×</b> Laemmli sample buffer to make the final volume to 200 μL. Denature at 95°C for 10 min.</li>
<li>Load equal amounts of the solubilized protein fractions on 10% SDS-PAGE gels.</li>
<li>Transfer the gel to the PVDF membranes via wet or semi-dry transfer methods.</li>
<li>The target protein can be visualized using the specific antibody (tag-specific antibody, if the target protein is fused to tag). Light-harvesting complex proteins (LhcBs) and Rubisco large subunit proteins (RbcL) can be used as membrane and stroma marker proteins, respectively, and can be detected using specific antibodies commercially available (see <a href="#key-resources-table">key resources table</a>). Stain the gel with CBB and the membrane with Ponceau S staining to confirm equal loading.</li>
</ol>
<h4 id="sec3.5.3">Quantification of proteins bands in western blots using ImageJ</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>(See <a href="#fig3">Figures 3</a> and <a href="#fig5">5</a>, and <a href="#mmc1">Methods video S2</a>)</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1003-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Quantification of protein bands in western blots using ImageJ.</div>
<p>(A) Screenshoots of the step-by-step pipeline using ImageJ for the analysis of band intensity in western blots. The western blot and the acrylamide gel reprinted with permission from <a href="#bib6">Medina-Puche et al. (2020)</a>.</p>
<p>(B) Analysis of the signal intensity values obtained from the protein samples (WB) relative to the corresponding loading control (CBB).</p>
</figcaption>
</figure>
<ol start="34">
<li>This section will allow you to quantify the intensity of protein bands from western blots with the ImageJ software (<a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a>). The quantification will retrieve the relative amounts as a ratio of each protein band relative to the corresponding loading control band (see <a href="#fig5">Figure 5</a> and <a href="#mmc2">Methods video S2</a>).
<ol type="a">
<li>Before starting, make sure you make a copy of your original raw images, save them as JPEG file format, and transform the images mode to “Grayscale”. As reference, Photoshop (or similar software) may be used in this step.</li>
<li>Set up the measurement criteria you will apply to all the blots to analyze. In the toolbar, click on the “Analyze” menu and select “Set Measurements”. From the checkboxes, keep only the “Mean gray value” checked.</li>
<li>Next, click on the “File” menu in the toolbar, and open the JPEG file images generated previously. Maximize the window and adjust an appropriate zoom level.</li>
<li>Set a defined “Region of Interest” (ROI) size according to the largest band among the target bands of a sample on the blot. In the toolbar, click on the “Analyze” menu, and next on “Tools”. Then, set a defined “Region of Interest” (ROI) size in the “ROI Manager menu”. Select the “rectangle” tool and draw a frame around the largest band. Adjust the size of the frame so that it covers the minimum area. Click the “File” menu in the toolbar, “Save as”, “Selection” and save this frame. Use the same frame for all the protein bands you need to analyze from the same blot. If you accidentally resize the frame or lose it, go to the “File” menu, “Open”, and reimport the same frame again.</li>
<li>Put the frame so that the band is inside, click the “Analyze” menu, “Measure”, and get the intensity values. Move the frame to the following bands and repeat this step in order to get the intensity values for all the samples.</li>
<li>Next, export the values into a spreadsheet keeping track of which value corresponds to which sample.</li>
<li>Repeat the steps “e” and “f” to take a background measurement using the same frame as for the protein bands. Put the frame above or below every band on the row where there are no other bands, get the intensity values, and export into the spreadsheet.</li>
<li>For the loading control bands, repeat the same steps (“e” to “g”) as described for the sample bands.</li>
</ol>
</li>
<li>With all the values in the spreadsheet, you can proceed to do the calculations.
<ol type="a">
<li>Apply the formula 255 – X, where X is the intensity value recorded for each band or loading control and their respective background values, to invert the pixel density for all data.</li>
<li>Next, calculate the net value for the protein bands and the loading controls by deducting the inverted background value from the inverted band value calculated in “a”.</li>
<li>Finally, calculate the relative value as the ratio of a net band value over the corresponding net loading control.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Save high-resolution images or scans from the blots or films, respectively (600 dpi in TIFF file format is recommended). Save them as raw images, and do not save any change over these original files.</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1003-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. A step-by-step video with the quantification of protein bands in wetern blots using ImageJ, related to step 34</div>
</figcaption>
</figure>
<h3 id="sec3.6">Immunoprecipitation-based detection of low abundant proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 h</div>
<div class="note">
<span class="note-title">Note:</span> We recommend carrying out this section if the accumulation of the protein of interest is too low.</div>
<ol start="36">
<li>Isolate total proteins and total chloroplast proteins as described above.</li>
<li>Filter the isolated proteins through a 0.22 μm Millipore Express PES membrane.</li>
<li>Quantify the proteins using standard protocols, e.g., the PierceTM BCA protein assay kit (Thermo Fisher Scientific).</li>
<li>Prepare the isolated protein as the Input samples: take 150 μL of isolated chloroplast protein samples and add 50 μL of 4<b>×</b> Laemmli sample buffer to make the final volume to 200 μL. Denature at 95°C for 10 min.</li>
<li>Take the desired amount of protein (ideally 2–10 mg) and add 25 μL of antibody-conjugated beads, e.g., GFP-Trap Agarose beads (ChromoTek) in the case of GFP-tagged target protein.</li>
<li>Incubate the samples with agitation at 4°C for 1.5–2 h.</li>
<li>Centrifuge at 2500 <b>×</b> <i>g</i> for 30 s.</li>
<li>Take out the supernatant and wash the beads four times for 5 min with wash buffer (PBS with 0.05% Triton X-100).</li>
<li>Elute the bound proteins in 100 μL 1<b>×</b> Laemmli SDS sample buffer with incubation for 20 min at 70°C.</li>
<li>Load equal amounts of the input and IP-eluted samples on 10% SDS-PAGE gels (for detecting bait (target) protein, load 10 μL, whereas for the prey (interactor) the loading can vary depending upon the interaction strength and the protein abundance.</li>
<li>Transfer the gel to the PVDF membranes via wet or semi-dry transfer methods.</li>
<li>The protein of interest can be visualized using a specific antibody or a tag-specific antibody (e.g., anti-GFP antibody), if the target protein is fused to a tag. Stain the gel with CBB and the membrane with Ponceau S to show equal loading.</li>
<li>Quantification can be done as described in steps 34 and 35.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The multifunctionality of a protein may rely on its versatile accumulation in distinct subcellular compartments in response to internal and external stimuli. This dynamic relocalization of proteins can be visualized and quantified through different methods. By combining microscopy-based methods and biochemical approaches, we aim to provide an easy-to-do method that enables the reliable and robust visualization and quantification of the dynamic protein accumulation in different subcellular compartments.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p><i>Agrobacterium</i>-mediated transformation efficiency is sensitive to temperature, so the growing conditions must be controlled and optimized.</p>
<p>A desired protein accumulation can be compromised by posttranscriptional (e.g., gene silencing) or posttranslational (e.g., induced cell death) effects. In order to circumvent these issues, a silencing suppressor (e.g., P19) could be used, or the expression of the transgene can be driven by an inducible promoter.</p>
<p>The subcellular localization of proteins is highly dependent on posttranslational modifications, and these, in turn, can be modulated by various environmental and developmental signals. It is therefore vitally important that plants grow under controlled growing conditions, avoiding both abiotic and biotic stresses.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p><i>N. benthamiana</i> leaves are hard to infiltrate (steps 4, 5, 10, and 11).</p>
<div class="critical">
<span class="critical-title">Critical:</span> make sure not to press the needleless syringe so hard as to pierce the leaf through.</div>
<h3 id="sec6.2">Potential solution</h3>
<p>Water the <i>N. benthamiana</i> plants 30 min before using.</p>
<p>Create a small nick with a needle or blade in the epidermis on the back side of the leaf. Please, make sure not to scratch so hard as to pierce the leaf through otherwise, the <i>Agrobacteria</i> inoculum will pass through the puncture.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The protein accumulation is very low as observed under the confocal microscope (step 8).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Host gene silencing can prevent accumulation of transcripts of the transgene; in such cases, viral silencing suppressors, such as P19, can be co-expressed.</p>
<p>When transiently accumulated <i>in planta</i>, some proteins can trigger cell death, leading to their reduced accumulation. In that case, changing the constitutive promoter to an inducible one could enable a controlled expression.</p>
<p>Detect the fluorescence with a more sensitive detector for confocal imaging. In our protocol, the Leica TCS SP8 confocal microscope (Leica Microscopy) provides a Hybrid Detector (HyD) for super-sensitive imaging.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The protein accumulation is very low as observed in the western blot (<a href="#sec3.5.2">Protein isolation from plant tissue and isolated chloroplasts, and western blotting</a>section; steps 29–33).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Host gene silencing can prevent accumulation of transcripts of the transgene; in such cases, viral silencing suppressors, such as P19, can be co-expressed.</p>
<p>When transiently accumulated <i>in planta</i>, some proteins can trigger cell death, leading to their reduced accumulation. In that case, changing the constitutive promoter to an inducible one could enable a controlled expression.</p>
<p>Scale up the amount of agroinfiltrated tissue in order to increase the amount of accumulated protein.</p>
<p>For biochemical detection on western blot, low abundant proteins can be enriched by using immunoprecipitation (see <a href="#sec3.6">immunoprecipitation-based detection of low abundant proteins</a> section; steps 36–48).</p>
<p>Use high sensitivity enhanced chemiluminescence (ECL) substrates for western blot detection and/or long exposure times to enable the detection of low-abundance proteins.</p>
<p>When CCD-based imaging does not provide enough sensitivity, use X-ray film as is generally considered to be the most sensitive method for detecting relative protein levels.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Low yield of isolated intact chloroplasts (<a href="#sec3.5.1">chloroplast isolation</a> section; steps 21–28 ).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>While homogenizing the leaf samples in the blender, do not give more than 3 bursts of 3 s.</p>
<p>For separating intact chloroplasts in the Percoll gradient, make sure that both acceleration and deceleration are very slow during centrifugation.</p>
<p>Use cut tips for recovering the layer of intact chloroplasts.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>In some cases, when the size of the protein of interest is small, GFP-tagged proteins are hard to differentiate from free GFP. This is largely because the difference between the size of the GFP-tagged protein and free GFP is not big enough to nicely separate them on the gel (<a href="#sec3.5.2">protein isolation from plant tissue and isolated chloroplasts, and western blotting</a> section; steps 29–33).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Run the proteins on the gel for a longer time.</p>
<p>Use a longer gel.</p>
<p>Try higher acrylamide concentration or bis acrylamide/acrylamide ratio.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Dogra, V., Duan, J., Lee, K.P., and Kim, C. (2019). Impaired PSII proteostasis triggers a UPR-like response in the var2 mutant of Arabidopsis. J. Exp. Bot. <i>70</i>, 3075-3088. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref1">View at publisher</a></p>
<p id="bib2">Felix, G., Duran, J.D., Volko, S., and Boller, T. (1999). Plants have a sensitive perception system for the most conserved domain of bacterial flagellin. Plant J. <i>18</i>, 265-276. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref2">View at publisher</a></p>
<p id="bib3">Gomez-Gomez, L. and Boller, T. (2000). FLS2: an LRR receptor-like kinase involved in the perception of the bacterial elicitor flagellin in Arabidopsis. Mol. Cell <i>5</i>, 1003-1011. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref3">View at publisher</a></p>
<p id="bib5">Kauss, D., Bischof, S., Steiner, S., Apel, K., and Meskauskiene, R. (2012). FLU, a negative feedback regulator of tetrapyrrole biosynthesis, is physically linked to the final steps of the Mg++-branch of this pathway. FEBS Lett. <i>586</i>, 211-216. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref5">View at publisher</a></p>
<p id="bib6">Medina-Puche, L., Tan, H., Dogra, V., Wu, M., Rosas-Diaz, T., Wang, L., Ding, X., Zhang, D., Fu, X., Kim, C., et al. (2020). A defense pathway Linking plasma membrane and chloroplasts and Co-opted by pathogens. Cell <i>182</i>, 1109-1124 e1125. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref6">View at publisher</a></p>
<p id="bib12">Morilla, G., Janssen, S., García-Andrés, S., Moriones, E., Cuadrado, I.M., and Bejarano, E.R. (2005). Pepper (Capsicum annuum) is a dead-end host for Tomato yellow leaf curl virus. Phytopathology <i>95</i> (<i>9</i>), 1089-1097. <a class="external-link" href="https://doi.org/10.1094/PHYTO-95-1089">https://doi.org/10.1094/PHYTO-95-1089</a></p>
<p id="bib7">Nakagawa, T., Kurose, T., Hino, T., Tanaka, K., Kawamukai, M., Niwa, Y., Toyooka, K., Matsuoka, K., Jinbo, T., and Kimura, T. (2007a). Development of series of gateway binary vectors, pGWBs, for realizing efficient construction of fusion genes for plant transformation. J. Biosci. Bioeng. <i>104</i>, 34-41. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref7">View at publisher</a></p>
<p id="bib8">Nakagawa, T., Suzuki, T., Murata, S., Nakamura, S., Hino, T., Maeo, K., Tabata, R., Kawai, T., Tanaka, K., Niwa, Y., et al. (2007b). Improved Gateway binary vectors: high-performance vectors for creation of fusion constructs in transgenic analysis of plants. Biosci. Biotechnol. Biochem. <i>71</i>, 2095-2100. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref8">View at publisher</a></p>
<p id="bib9">Qu, F. and Morris, T.J. (2002). Efficient infection of Nicotiana benthamiana by tomato bushy stunt virus is facilitated by the coat protein and maintained by p19 through suppression of gene silencing. Mol. Plant Microbe Interact. <i>15</i>, 193-202. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref9">View at publisher</a></p>
<p id="bib10">Rosas-Diaz, T., Zhang, D., Fan, P., Wang, L., Ding, X., Jiang, Y., Jimenez-Gongora, T., Medina-Puche, L., Zhao, X., Feng, Z., et al. (2018). A virus-targeted plant receptor-like kinase promotes cell-to-cell spread of RNAi. Proc. Natl. Acad. Sci. U S A <i>115</i>, 1388-1393. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref10">View at publisher</a></p>
<p id="bib14">Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to ImageJ: 25 years of image analysis. Nat. Methods <i>9</i>, 671-675. <a class="external-link" href="https://doi.org/10.1038/nmeth.2089">https://doi.org/10.1038/nmeth.2089</a></p>
<p id="bib13">Wang, L., Kim, C., Xu, X., Piskurewicz, U., Dogra, V., Singh, S., and Apel, Klaus. (2016). Singlet oxygen- and EXECUTER1-mediated signaling is initiated in grana margins and depends on the protease FtsH2. Proc. Natl. Acad. Sci. U S A <i>113</i> (<i>26</i>), E3792-E3800. <a class="external-link" href="https://doi.org/10.1073/pnas.1603562113">https://doi.org/10.1073/pnas.1603562113</a></p>
<p id="bib11">Yamaguchi, Y., Pearce, G., and Ryan, C.A. (2006). The cell surface leucine-rich repeat receptor for AtPep1, an endogenous peptide elicitor in Arabidopsis, is functional in transgenic tobacco cells. Proc. Natl. Acad. Sci. U S A <i>103</i>, 10104-10109. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00522-0/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contacts, Laura Medina-Puche (<a href="mailto:laura.medina-puche@zmbp.uni-tuebingen.de">laura.medina-puche@zmbp.uni-tuebingen.de</a>), and Vivek Dogra (<a href="mailto:vivekdogra@ihbt.res.in">vivekdogra@ihbt.res.in</a>).</p>
<h4>Materials availability</h4>
<p>Plasmids generated in this study will be made available on request; a completed Materials Transfer Agreement may be requested if there is potential for commercial application.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>The authors thank Xinyu Jian, Aurora Luque, Huang Tan, and Rahul Mohan Singh for technical assistance. This research was supported by the <a href="https://doi.org/10.13039/100014717">National Natural Science Foundation of China</a> (NSFC) (grant numbers 31671994 and 31870250 to R.L.-D.), the <a href="https://doi.org/10.13039/501100002367">Chinese Academy of Sciences</a> (CAS) Strategic Priority Research Program (grant number XDB27040206), the 100 Talents Program from <a href="https://doi.org/10.13039/501100002367">CAS</a> (to R.L.-D.), the Talent Recruitment Program from the Shanghai government (to R.L.-D.), and the Shanghai Center for Plant Stress Biology, CAS. R.L.-D. is the recipient of a National Foreign Talents project (grant number G20200113006). L.M.-P. is the recipient of a Young Investigator Grant from the <a href="https://doi.org/10.13039/100014717">National Natural Science Foundation of China</a> (NSFC) (grant number 31850410467), a President’s International Fellowship Initiative (PIFI) postdoctoral fellowship (grant number 2018PB058 and 2020PB0080) from <a href="https://doi.org/10.13039/501100002367">CAS</a>, and a Foreign Youth Talent Program project (grant number 20WZ2503900) from the <a href="https://doi.org/10.13039/501100003399">Shanghai Science and Technology Commission</a>. Part of this research was supported by the Strategic Priority Research Program from the <a href="https://doi.org/10.13039/501100002367">Chinese Academy of Sciences</a> (Grant No. XDB27040102) and the <a href="https://doi.org/10.13039/100014717">National Natural Science Foundation of China</a> (NSFC) (Grant No. 31871397) to C.K. V.D. is the recipient of Research Fund for International Young Scientists Program of <a href="https://doi.org/10.13039/100014717">NSFC</a> (Grant No. 31850410478) and President’s International Fellowship Initiative (PIFI) postdoctoral fellowship from the <a href="https://doi.org/10.13039/501100002367">Chinese Academy of Sciences</a> (No. 2019PB0066). V.D. also acknowledges the financial assistance from Ramalingaswami Re-entry Fellowship, <a href="https://doi.org/10.13039/501100010803">Department of Biotechnology</a>, Government of India (grant number BT/RLF/Re-entry/02/2019).</p>
<h3>Author contributions</h3>
<p>Conceptualization, L.M.-P., V.D., C.K., and R.L.-D.; investigation, L.M.-P. and V.D.; resources, C.K. and R.L.-D.; writing – original draft, L.M.-P. and V.D.; writing – review &amp; editing, L.M.-P., V.D., C.K., and R.L.-D.; funding acquisition, L.M.-P., V.D., C.K., and R.L.-D.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>