<article>
<section>
<h1 id="header">Protocol for fungal infection following the induction of epithelial cell loss in larval zebrafish</h1>
<p><time datetime="2021-11-26">Published: November 26, 2021</time></p>
<p>Sebastian Wurster,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> Oscar E. Ruiz,<sup><a href="#aff2">2</a></sup> Alexander M. Tatara,<sup><a href="#aff3">3</a></sup> Dimitrios P. Kontoyiannis,<sup><a href="#aff1">1</a></sup> and George T. Eisenhoffer<sup><a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Infectious Diseases, Infection Control and Employee Health, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</p>
<p id="aff2"><sup>2</sup>Department of Genetics, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</p>
<p id="aff3"><sup>3</sup>Medical Scientist Training Program, Baylor College of Medicine, Houston, TX 77030, USA</p>
<p id="aff4"><sup>4</sup>Genetics and Epigenetics Graduate Program, MD Anderson Cancer Center UTHealth Graduate School of Biomedical Sciences, Houston, TX 77030, USA</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:stwurster@mdanderson.org">stwurster@mdanderson.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:gteisenhoffer@mdanderson.org">gteisenhoffer@mdanderson.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100963">10.1016/j.xpro.2021.100963</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Epithelia provide the first line of defense against foreign pathogens, and disruption of tissue homeostasis frequently allows for opportunistic infections. Here we provide a protocol for induction of epithelial cell loss in zebrafish larvae, followed by infection with fungal pathogens. Details are provided for monitoring larval survival after infection, assessment of fungal burden, and prophylactic treatment with antifungal compounds. Limitations of the protocol include potential antifungal toxicity and high fungal inoculums to induce lethal infection with some pathogenic fungal species.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref13">Wurster et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>An immersion-based fungal infection protocol for larval zebrafish</li>
<li>Applicable for different fungi associated with human disease</li>
<li>Description of post-infection monitoring of survival</li>
<li>Steps for determination of fungal burden</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1193-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps for <i>Rhizopus arrhizus</i> infection of zebrafish larvae after induced epithelial cell loss. The GAL4 enhancer trap line <i>zc1044a</i> (<a href="#bib6">Eisenhoffer et al., 2017</a>) is used to drive expression of the genetically encoded nitroreductase enzyme (NTR) fused to mCherry under the control of an upstream activating sequence (UAS) (<a href="#bib4">Davison et al., 2007</a>) in the surface epithelial cells. The expression of NTR is essential, as cells expressing this enzyme will convert exogenously added metronidazole (MTZ) to a cytotoxic byproduct that promotes DNA damage (<a href="#bib3">Curado et al., 2008</a>) and cell expulsion (<a href="#bib1">Atieh et al., 2021</a>). We further describe downstream assays to monitor epithelial pathogenesis and infection severity, as well as a protocol for antifungal treatment of <i>R. arrhizus</i>-infected larvae. In addition to <i>R. arrhizus</i>, we have also used this protocol to infect larvae with other agents of mucormycosis as well as pathogenic yeasts (e.g., <i>Candida albicans</i>).</p>
<div class="note">
<span class="note-title">Note:</span> All procedures described in this protocol have been approved by the University of Texas MD Anderson’s Institutional Animal Care and Use Committee (protocol #00001283-RN02 and #00001493-RN01). Investigators should consult their Institutional Animal Care and Use Committee for the regulations that apply to experiments performed following this protocol.</div>
<h3 id="sec1.1">Preparation of buffers and media</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h</div>
<ol>
<li>Prepare E3 medium according to the recipe provided below.</li>
<li>Prepare a 10 mM metronidazole (MTZ) solution by dissolving 1.71% w/v MTZ in 1× E3 medium. Filter-sterilize the solution.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The solubility of MTZ in cold E3 medium is insufficient to facilitate a 10 mM solution. In order to ensure the correct concentration of MTZ for addition to the zebrafish larvae, use pre-warmed E3 medium (28°C) and vortex the solution thoroughly prior to filter-sterilization. The MTZ working solution should always be freshly prepared on the day of use. We recommend using a fresh lot of MTZ powder on a quarterly basis. The working solution of MTZ should be kept in the dark.</div>
<ol start="3">
<li>To culture wild-type Mucorales strains, prepare yeast extract glucose agar (YAG) plates according to the recipe below.</li>
<li>The GFP-expressing <i>R. arrhizus</i> mutant required for live imaging needs to be cultured on an uracil-deficient selection medium (yeast nitrogen base agar with complete supplement mixture minus uracil, YNB + csm-ura), prepared according to the recipe below.
<ol type="a">
<li>First, autoclave 10 g agar in 450 mL ddH<sub>2</sub>O.</li>
<li>Add 50 mL of 10× YNB + csm-ura solution, prepared according to the recipe below.</li>
<li>Mix the ingredients well by inverting the bottle several times and pour 15–25 mL of agar in each sterile Petri dish.</li>
</ol>
</li>
<li>Prepare aliquots of 1% low-melt agarose and store at 42°C, if time-lapse imaging will be performed as a downstream application.</li>
</ol>
<h3 id="sec1.2">Fluorescence-based sorting of Gal4 enhancer trap larvae and assay preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<ol start="6">
<li>Set up mating for <i>Et(Gal4-VP16)</i><i><sup>zc1044a</sup></i><i>;Tg(UAS-E1b:nsfB-mCherry)</i><sup><i>c264</i></sup>
<ol type="a">
<li>Ensure that mating is set up 5 days prior to the experiment in order to have 4-day post-fertilization larvae ready for the day of the experiment.</li>
</ol>
</li>
<li>On day of mating, collect eggs in E3 media and discard all dead or unfertilized eggs. Place in an incubator at 28.5°C.</li>
<li>At 3 days post fertilization, sort larvae for mCherry fluorescence using a fluorescent dissecting scope and collect mCherry-positive embryos into a new dish with fresh E3.
<ol type="a">
<li>Also collect embryos that lack mCherry fluorescence into separate dishes. These can be used for enzyme negative controls (-NTR).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to provide fresh E3 and remove used chorions from the media. Failure to do this can prematurely damage the epithelia which will result in confounding data<b>.</b>
</div>
<h3 id="sec1.3">Preparation of fungal strains</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days for most wild-type isolates—strains grown on special selection media (e.g. GFP-<i>Rhizopus</i>) require longer incubation periods.</div>
<ol start="9">
<li>Streak clinical Mucorales strains (e.g., from a glycerol stock) on YAG plates. The GFP-expressing <i>R. arrhizus</i> mutant needs to be plated on a uracil-deficient YNB medium, prepared as described above.</li>
<li>Incubate fungal plates at 35°C–37°C for 48–72 h (wild-type isolates) or up to 5 days (mutants grown on selection plates).</li>
<li>To harvest the sporangiospores, overlay the mycelium with 5–10 mL of E3 medium and gently scrape the mycelium with a sterile glass rod or a single-use plastic spreader.</li>
<li>Collect the spore suspension with a serological pipette and pass the suspension through a 40-μM cell strainer in order to remove mycelial fragments.</li>
<li>Wash the spore suspension twice with 20 mL E3 medium (centrifugation steps at 5000 <i>g</i>, 5 min, RT).</li>
<li>After the last centrifugation step, discard the supernatant and resuspend the spores in 1 mL of sterile E3 medium.</li>
<li>Dilute an aliquot with E3 medium (1:10 or 1:100) and quantify the spore concentration with a hemocytometer.</li>
<li>Dilute the suspension to 6 × 10<sup>7</sup> spores per mL (recommended stock concentration to achieve our standard inoculum of 5 × 10<sup>6</sup> spores per mL in the infection assay, as described below). Depending on the experimental design, a higher or lower stock concentration may be required.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Fungal spore suspensions need to be prepared fresh on the day of the experiment. Prolonged storage and/or serial passaging of fungal pathogens can induce virulence drifts and introduce potential bias to infection outcomes (e.g., survival rates).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Metronidazole</td>
<td>Sigma-Aldrich</td>
<td>M-3761</td>
</tr>
<tr>
<td>Tricaine-S</td>
<td>Western Chemical</td>
<td>TRICMGR0100</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>DNeasy Blood and Tissue Kit</td>
<td>QIAGEN</td>
<td>69506</td>
</tr>
<tr>
<td>TaqMan Universal PCR Master Mix</td>
<td>Applied Biosystems / Thermo Fisher Scientific</td>
<td>4304437</td>
</tr>
<tr>
<td>DNAse Kit</td>
<td>QIAGEN</td>
<td>79254</td>
</tr>
<tr>
<td>Glass beads 3 mm</td>
<td>MilliporeSigma</td>
<td>1040150500</td>
</tr>
<tr>
<td>Low Melt Agarose</td>
<td>Invitrogen</td>
<td>16520</td>
</tr>
<tr>
<td>MatTek Glass Bottom Dish</td>
<td>MatTek Corporation</td>
<td>P35G-1.0-20-C</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Zebrafish: <i>Et(Gal4-VP16)</i><i><sup>zc1044a</sup></i><i>, Tg(UAS-1b:nsfB-mCherry)</i><sup><i>c264</i></sup>
</td>
<td>(<a href="#bib6">Eisenhoffer et al., 2017</a>)</td>
<td><i>zc1044a</i></td>
</tr>
<tr>
<td>Zebrafish: <i>Et(Gal4-VP16)</i><i><sup>zc1044a</sup></i><i>, Tg(UAS-1b:nsfB-mCherry)</i><sup><i>c264</i></sup> <i>,Tg(UAS-1b:Lifeact-EGFP)</i><sup><i>utm1</i></sup>
</td>
<td>(<a href="#bib6">Eisenhoffer et al., 2017</a>)</td>
<td>utm1</td>
</tr>
<tr>
<td>Zebrafish: <i>Tg(</i><i>k</i><i>rt4:GFP)</i>
</td>
<td>(<a href="#bib7">Gong et al., 2002</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Zebrafish: <i>Tg(-8.0cldnb:lynEGFP)</i> <sup><i>zf106</i></sup>
</td>
<td>(<a href="#bib8">Haas and Gilmour, 2006</a>)</td>
<td>zf106</td>
</tr>
<tr>
<td>
<i>Rhizopus arrhizus</i> 749</td>
<td>MD Anderson Cancer Center (clinical collection)</td>
<td>Ra-749</td>
</tr>
<tr>
<td>
<i>Rhizopus arrhizus</i> (FTR1)-GFP</td>
<td>A. S. Ibrahim, University of California, Los Angeles,</td>
<td>Rhizo-GFP</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>
<i>R. arrhizus</i> 18S forward amplification primer, GCGGATCGCATGGCC</td>
<td>Applied Biosystems / Thermo Fisher Scientific, (<a href="#bib9">Ibrahim et al., 2005</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>R. arrhizus</i> 18S reverse amplification primer, CCATGATAGGGCAGAAAATCG</td>
<td>Applied Biosystems / Thermo Fisher Scientific, (<a href="#bib9">Ibrahim et al., 2005</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>R. arrhizus</i> 18S hybridization probe, FAM-TGTGCCGGCGACGGTCCAC-TAMRA</td>
<td>Applied Biosystems / Thermo Fisher Scientific, (<a href="#bib9">Ibrahim et al., 2005</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism (version 7.03)</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td>Zeiss Zen Blue 2.6</td>
<td>ZEISS</td>
<td><a href="https://www.zeiss.com/microscopy/us/products/microscope-software/zen.html">https://www.zeiss.com/microscopy/us/products/microscope-software/zen.html</a></td>
</tr>
<tr>
<td>Olympus FV312S-SW</td>
<td>Olympus</td>
<td><a href="https://www.olympus-lifescience.com/en/support/downloads/">https://www.olympus-lifescience.com/en/support/downloads/</a></td>
</tr>
<tr>
<td>IncuCyte ZOOM software (version 2016B)</td>
<td>Sartorius / Essen Bioscience</td>
<td><a href="https://www.essenbioscience.com/en/products/software/">https://www.essenbioscience.com/en/products/software/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th colspan="3">50× E3 medium stock solution</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>1.46% w/v</td>
<td>14.6 g/L</td>
</tr>
<tr>
<td>KCl</td>
<td>0.063% w/v</td>
<td>0.63 g/L</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>•2H<sub>2</sub>O</td>
<td>0.243% w/v</td>
<td>2.43 g/L</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>•7H<sub>2</sub>O</td>
<td>0.407% w/v</td>
<td>4.07 g/L</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>ad 1 L</td>
</tr>
</tbody>
</table>
<p class="table-legend">Adjust pH to 7.2 with 0.1M NaOH solution, autoclave, and store at 4°C for up to 1 year.</p>
<table id="undtbl2">
<thead>
<tr>
<th colspan="3">1× E3 medium working solution</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>50× E3 stock solution</td>
<td>n/a</td>
<td>20 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>980 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave or filter-sterilize and store at RT (short-term) or 4°C (for up to 3 months).</p>
<table id="undtbl3">
<thead>
<tr>
<th colspan="3">YAG plates</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Agar</td>
<td>1.5% w/v</td>
<td>15 g</td>
</tr>
<tr>
<td>Glucose</td>
<td>1% w/v</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>0.5% w/v</td>
<td>5 g</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> [1 M]</td>
<td>10 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Vitamin mix<sup><a href="#dtbl4fna">a</a></sup>
</td>
<td>n/a</td>
<td>2 mL</td>
</tr>
<tr>
<td>Trace element mix<sup><a href="#dtbl4fnb">b</a></sup>
</td>
<td>n/a</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>ad 1 L</td>
</tr>
</tbody>
</table>
<p class="table-legend">Pour the autoclaved medium into sterile Petri dishes (15–25 mL per dish) and allow to solidify overnight at room temperature. Store at 4°C for up to 3 months.</p>
<p id="dtbl4fna" class="table-footnote"><sup>a</sup>Vitamin mix: 1 g p-aminobenzoic acid, 1 g niacin, 1 g pyridoxine HCl, 1 g riboflavin, 1 g thiamine HCl, 1 g cholin HCl, and 2 mg d-biotin in 1 L distilled water; autoclave and store at 4°C in the dark.</p>
<p id="dtbl4fnb" class="table-footnote"><sup>b</sup>Trace element mix: 100 mL 0.25 M EDTA pH 8.0, 1 g FeSO<sub>4</sub>·7H<sub>2</sub>O, 8.8 g ZnSO<sub>4</sub>·7H<sub>2</sub>O, 0.4 g CuSO<sub>4</sub>·4H<sub>2</sub>O, 0.15 g MnSO<sub>4</sub>·4H<sub>2</sub>O, 0.1 g Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>·10H<sub>2</sub>O, and 0.1 g NaMoO<sub>4</sub>·2H<sub>2</sub>O in 1 L distilled water; store at room temperature.</p>
<table id="undtbl4">
<thead>
<tr>
<th colspan="3">10× YNB solution</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast nitrogen base without amino acids (BD, #233520)</td>
<td>1.7% w/v</td>
<td>1.7 g</td>
</tr>
<tr>
<td>Glucose</td>
<td>20% w/v</td>
<td>20 g</td>
</tr>
<tr>
<td>CMS-URA (MP Biomedicals, #114511212)</td>
<td>0.77% w/v</td>
<td>0.77 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>ad 100 mL</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Exposure of larvae to metronidazole to induce epithelial cell loss</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min to 2 h hands-on time (depending on the number of larvae and experimental groups), followed by 5 h of incubation time (<a href="#fig1">Figure 1</a><b>).</b>
</div>
<p>Addition of MTZ to <i>Et(Gal4-VP16)</i><sup><i>zc1044a</i></sup><i>, Tg(UAS-1b:nsfB-mCherry)</i><sup><i>c264</i></sup> larvae causes a NTR-dependent conversion of MTZ to a cytotoxic agent, which induces the rapid elimination of apoptotic surface epithelial cells by extrusion, a mechanism used to remove unfit or aberrant cells.</p>
<ol>
<li>Carefully transfer larvae to 12-well suspension culture plates using a 1-mL Pasteur pipette.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If sufficient numbers of larvae are available, we recommend using 15 larvae per well (experimental group). However, do not exceed a number of 15 larvae per well as overcrowding of wells complicates manipulations and survival monitoring, and may result in increased mortality.</div>
<ol start="2">
<li>Hold the plate at an angle of about 30° and carefully remove the medium with a 1-mL microliter pipette. Do not touch or aspirate the larvae.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> No more than 100 μL of medium should remain in the well in order to ensure adequate MTZ exposure.</div>
<ol start="3">
<li>Add 1 mL of MTZ-supplemented E3 medium to the wells. Use plain E3 medium for control groups that shall retain epithelial homeostasis.</li>
<li>Incubate plates for 5 h at 28°C on a nutator (<a href="#fig1">Figure 1</a>).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> For therapeutic studies, drugs modulating epithelial restorative capacity (e.g., recombinant human epigen) and/or antifungal prophylaxis can be added to the medium during MTZ exposure.</div>
<div class="critical">
<span class="critical-title">Critical:</span> All experiments should include control larvae without epithelial cell loss. These may be either wild-type larvae or Gal4 enhancer trap larvae incubated in E3 medium lacking MTZ. In our experience, MTZ toxicity is not an issue as we did not observe noticeable morbidity or mortality upon exposure of larvae to concentrations of up to 10 mM MTZ.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The duration of MTZ exposure is critical for the extent of epithelial cell loss elicited. Do not exceed 5 h of MTZ exposure to prevent excess mortality of infected larvae and death of uninfected control animals. If necessary, the extent of epithelial damage caused by the cell loss can be fine-tuned by adjusting the MTZ concentration and/or shortening the exposure period.</div>
<div class="note">
<span class="note-title">Note:</span> Extrusion and the extent of associated tissue damage can be assessed by using the <i>Et(Gal4-VP16)</i><i><sup>zc1044a</sup></i><i>, Tg(UAS-1b:nsfB-mCherry)</i><sup><i>c264</i></sup> line in conjunction with other transgenic lines such as, <i>Tg(UAS-1b:Lifeact-EGFP)</i><sup><i>utm1</i></sup><i>; Tg(</i><i>k</i><i>rt4:GFP);</i> and <i>Tg(-8.0cldnb:lynEGFP)</i> <sup><i>zf106</i></sup> that allow for independent visualization of both the surface epithelial cells and cell-cell contacts. For additional details about quantification of the resulting epithelial tissue damage, please see (<a href="#bib13">Wurster et al., 2021</a>).</div>
<h3 id="sec3.2">Infection of larvae after induced epithelial cell loss</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min to 2 h hands-on time (depending on the number of larvae and experimental groups), 16 h of incubation time (<a href="#fig1">Figure 1</a><b>).</b>
</div>
<p>Induction of epithelial cell loss by MTZ treatment of <i>Et(Gal4-VP16)</i><sup><i>zc1044a</i></sup><i>, Tg(UAS-1b:nsfB-mCherry)</i><sup><i>c264</i></sup> larvae renders the animals susceptible to fungal infection. While larvae with epithelial homeostasis are highly resistant to opportunistic fungi, larvae with epithelial cell loss can be infected with agents of mucormycosis or other pathogenic fungi by adding spores to the medium. Unlike injection-based infection assays, this technique allows for rapid infection of hundreds of larvae by simple media changes.</p>
<ol start="5">
<li>Hold the 12-well plate at an angle of about 30° and remove the supernatant from each well with a 1-mL microliter pipette. Avoid touching or aspirating the larvae. Approximately 100 μL should remain in the well.</li>
<li>Add 1 mL of fresh E3 medium.</li>
<li>Repeat the wash cycle two more times and add 1 mL of fresh E3 medium.</li>
<li>Add 100 μL of a 6×10<sup>7</sup> <i>R. arrhizus</i> spore suspension to all wells that need to be infected (<a href="#fig2">Figures 2</a>A–2D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Considering a remainder of 100 μL E3 medium in the wells after the last wash step, addition of 1 mL of fresh E3 medium and the 100-μL volume of the spore inoculum itself, this inoculum results in our recommended standard spore concentration of 5×10<sup>6</sup>/mL. However, we have previously shown that the infection severity can be easily adjusted by using up to 10-fold lower or higher spore inoculums (as seen in Figure 1L in (<a href="#bib13">Wurster et al., 2021</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> All experiments should include control groups of uninfected larvae with epithelial homeostasis and induced epithelial cell loss, respectively, as a sentinel for non-infection-attributable baseline mortality. A standard experimental design is shown in <a href="#tbl1">Table 1</a>.</div>
<table id="tbl1">
<caption>Table 1. Example of a typical plate setup to test two therapeutic interventions (or two concentrations of a single drug), including relevant controls</caption>
<thead>
<tr>
<th colspan="4">
<u>Plate 1</u> <i>zc1044a</i> Gal4 enhancer trap larvae</th>
<th colspan="2">
<u>Plate 2</u> Wild-type larvae (optional when using no-MTZ controls on plate 1)</th>
</tr>
</thead>
<tbody>
<tr>
<td>MTZ: no<i>R. arr.</i>: noDrug: no</td>
<td>MTZ: yes<i>R. arr.</i>: noDrug: no</td>
<td>MTZ: no<i>R. arr.</i>: yesDrug: no</td>
<td>MTZ: yes<i>R. arr.</i>: yesDrug: no</td>
<td>MTZ: no<i>R. arr.</i>: noDrug: no</td>
<td>MTZ: no<i>R. arr.</i>: yesDrug: no</td>
</tr>
<tr>
<td>MTZ: no<i>R. arr.</i>: noDrug: 1</td>
<td>MTZ: yes<i>R. arr.</i>: noDrug: 1</td>
<td>MTZ: no<i>R. arr.</i>: yesDrug: 1</td>
<td>MTZ: yes<i>R. arr.</i>: yesDrug: 1</td>
<td>MTZ: no<i>R. arr.</i>: noDrug: 1</td>
<td>MTZ: no<i>R. arr.</i>: yesDrug: 1</td>
</tr>
<tr>
<td>MTZ: no<i>R. arr.</i>: noDrug: 2</td>
<td>MTZ: yes<i>R. arr.</i>: noDrug: 2</td>
<td>MTZ: no<i>R. arr.</i>: yesDrug: 2</td>
<td>MTZ: yes<i>R. arr.</i>: yesDrug: 2</td>
<td>MTZ: no<i>R. arr.</i>: noDrug: 2</td>
<td>MTZ: no<i>R. arr.</i>: yesDrug: 2</td>
</tr>
</tbody>
</table>
<ol start="9">
<li>Incubate plates for 16 h (± 1 h) at 28°C on a nutator (<a href="#fig1">Figure 1</a>).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> For defining interactions of fungal spores with the damaged epithelium, anesthetized (tricaine-S) infected and non-infected larvae can be mounted in a 4-well MatTek dish and high-resolution imaging can be performed using laser scanning or spinning disc confocal microscopy as in (<a href="#bib1">Atieh et al., 2021</a>; <a href="#bib5">Eisenhoffer and Rosenblatt, 2011</a>) or anesthetized larvae can be mounted in 24-well plates on soft agar and imaged using the IncuCyte ZOOM microscope system (<a href="#bib13">Wurster et al., 2021</a>).</div>
<h3 id="sec3.3">Post-infection monitoring and treatment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>We recommend a 16-h infection period to allow for spore adhesion to exposed extracellular matrix in larvae with induced epithelial cell loss and subsequent initiation of tissue invasion (<a href="#fig1">Figure 1</a>). Thereafter, non-invasive spores loosely attached to the larval epidermis are removed by a thorough washing routine and antifungal treatment may be initiated.</p>
<ol start="10">
<li>After the 16-h infection period, remove the medium from each well, as described above, and add 1 mL of fresh E3 medium.</li>
<li>Remove dead and moribund larvae from each well using a Pasteur pipette and tabulate the number of fatalities.
<ol type="a">
<li>Consider preserving the removed larvae for fungal burden determination (as described below).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Dead larvae can be commonly recognized by pale color and curled tails (<a href="#fig2">Figure 2</a>E). If uncertain, lack of heart pulsation may be verified with a stereomicroscope. With high-inoculum infection, it is not uncommon to even find some heavily disintegrated larvae. Moribund larvae are defined as displaying no swimming motions even after gentle probing with a Pasteur pipette.</div>
<ol start="12">
<li>Perform at least 3 additional wash steps with 1 mL of fresh E3.
<ol type="a">
<li>Make sure to remove about 90% of the medium in each wash step to ensure complete removal of residual spores.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> After the last wash step, (investigational) antifungal drugs may be added to the well. We recommend performing a) <i>in vitro</i>-efficacy testing against the studied fungus and b) toxicity testing in uninfected larvae with and without induced epithelial cell loss prior to the actual experiment. This will allow for selection of at least two potentially effective and non-toxic concentrations for <i>in vivo</i> efficacy studies. Even with prior toxicity testing, an uninfected toxicity control with and without induced epithelial cell loss should be part of each <i>in-vivo</i> experiment to monitor toxicity (<a href="#tbl1">Table 1</a>).</div>
<p><b>CAUTION:</b> Some antifungal agents (e.g., amphotericin B) are photosensitive and should be stored in a light-protected manner.</p>
<div class="note">
<span class="note-title">Note:</span> We do not recommend adding antifungal treatment during the 16-h infection period. Otherwise, effective drug concentrations would solely inhibit or kill the spores in the aqueous environment and no information would be obtained regarding antifungal <i>in-vivo</i> activity against the invasive fungal morphotypes.</div>
<ol start="13">
<li>Incubate plates at 28°C on a nutator.</li>
<li>Remove and count dead and moribund larvae at 24 h, 48 h, and 72 h post-infection.
<ol type="a">
<li>Consider preserving the removed larvae for fungal burden determination.</li>
</ol>
</li>
<li>At 72 h post-infection, euthanize the remaining larvae by addition of ice-cold E3 medium and either preserve them for fungal burden determination or dispose them in an appropriate bio-safety container.</li>
</ol>
<p><b>CAUTION:</b> The use of older larvae (≥ 8 dpf) may be associated with enhanced institutional and veterinarian oversight. Using our 72-h post-infection monitoring scheme, experiments are terminated on day 7 post-fertilization. Of note, our definition includes the 16-h exposure time as part of the 72-h monitoring window (<a href="#fig1">Figure 1</a>). Otherwise, the monitoring period would extend into day 8 post-fertilization.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1193-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Experimental timeline for induction of epithelial cell loss, fungal infection, treatment, and monitoring of larval zebrafish</div>
<p>Four days post-fertilization (dpf) zebrafish larvae expressing the genetically encoded enzyme NTR in the surface epithelial cells, which upon exposure to metronidazole (MTZ), induces epithelial cell extrusion and apoptosis, are seeded in 12-well plates (up to 15 larvae per well). Larvae are treated with 10 mM MTZ for 5 h, washed (W), and incubated in fungal spore suspensions for 16 h. Survival is monitored for 72 h post-infection. Dead or moribund larvae are collected at 16, 24, 48, and 72 h post-infection and preserved for fungal burden determination. Larvae surviving for 72 h post-infection are euthanized (E) and preserved for fungal burden determination. Treatments modulating epithelial cell loss and/or antifungal prophylaxis may be added during MTZ exposure (treatment period 1, T1). Antifungal therapy may be added after the 16-h infection period (T2).</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1193-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Monitoring <i>R. arrhizus</i> infection after induced cell loss in larval zebrafish</div>
<p>Four days post-fertilization (dpf) zebrafish larvae in plain E3 medium (A) and E3 medium inoculated with 5×10<sup>6</sup>/mL <i>R. arrhizus</i> sporangiospores (B). Individual larvae in plain (C) and <i>R. arrhizus</i>-infected medium (D).</p>
<p>(E) Representative example of a zebrafish larva that has succumbed to infection. Note the adherent fungal spores and hyphal elements and change in color of the body of the larva, two attributes associated with mortality.</p>
</figcaption>
</figure>
<h3 id="sec3.4">Fungal burden determination by quantitative PCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day (after collecting the larval samples during post-infection monitoring)</div>
<p>To quantify fungal tissue burden in <i>R. arrhizus</i>-infected larvae, we recommend using a well-established quantitative 18S PCR assay (<a href="#bib9">Ibrahim et al., 2005</a>). In addition to increasing mortality, verification of inoculum-dependently increasing fungal tissue burden is a suitable way to validate the infection assay when training new experimenters. Furthermore, we have shown that effective antifungal therapy significantly reduces fungal tissue burden in zebrafish larvae with induced epithelial cell loss (<a href="#bib13">Wurster et al., 2021</a>), suggesting that fungal burden determination is a suitable assay to corroborate therapeutic efficacy of investigational antifungal treatments in this <i>in vivo</i> platform.</p>
<ol start="16">
<li>Collect dead and moribund larvae at 16, 24, 48, and 72 h post-infection as well as the surviving larvae euthanized at 72 h post-infection in 2-mL microcentrifuge tubes.</li>
<li>Wash the collected larvae three times with 1 mL of cold E3 medium in order to remove loosely attached spores from the larval epidermis.</li>
<li>After the last wash step, remove the supernatant and add 500 μL of fresh E3 medium.</li>
<li>Store the samples at −80°C until further use.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be stored at −80°C for up to 12 months.</div>
<ol start="20">
<li>On the day of DNA extraction, thaw and combine all samples from different collection time points for each experimental cohort.</li>
<li>Centrifuge the samples for 5 min at 5000 <i>g</i>.</li>
<li>Remove the supernatant and resuspend the pellet in 2 mL of filter-sterilized cold E3 medium.</li>
<li>Transfer the suspension to a 2-mL screw-cap cryovial containing 15 acid-washed glass beads (3 mm, Sigma-Aldrich).</li>
<li>Homogenize samples with two 15-s bursts in a bead beater (e.g., Mini Bead Beater, Bio-Spec, Bartlesville, OK) and immediately place the homogenates on crushed ice.</li>
<li>Isolate genomic DNA from an aliquot of the larval homogenate (80 μL) using the DNeasy kit (Qiagen, Valencia, CA).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Isolated DNA can be stored at −80°C for several years.</div>
<ol start="26">
<li>Analyze each sample in triplicate by real-time quantitative PCR (qPCR) using primers and dually labeled hybridization probes specific for <i>R. arrhizus</i> 18S rRNA genes according to a previously published protocol (<a href="#bib9">Ibrahim et al., 2005</a>).</li>
<li>Interpolate the cycle threshold of each sample from a seven-point standard curve prepared by spiking batches of 15 uninfected larvae with known concentrations of sporangiospores of the <i>R. arrhizus</i> strain used for the infection experiment (10<sup>2</sup> to 10<sup>8</sup> spores / mL larval homogenate).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To quantify fungal tissue burden after yeast infection, qPCR can be replaced by a simple colony formation assay. To that end, aliquots of homogenized larval tissue are diluted 1:100 with sterile E3 medium and 10-μL aliquots of the dilution are streaked on yeast extract, peptone, dextrose (YPD) agar. Colonies are counted after 24 h of incubation at 37°C.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>In (<a href="#bib13">Wurster et al., 2021</a>), we demonstrated that epithelial cell loss in zebrafish larvae can facilitate pathogenic invasion of molds (e.g., <i>Rhizopus arrhizus</i>) and yeast (e.g., <i>Candida albicans</i>) associated with human disease. Using this immersion-based zebrafish infection protocol, we found that the amount of epithelial cell loss and mucosal damage was associated with increased mortality of fungal infected larvae. By contrast, epithelial cell loss and mucosal damage did not result in a decline of survival rates in the absence of infection. This <i>in vivo</i> platform now provides the opportunity to rapidly test anti-fungal compounds to attenuate fungal invasion in the setting of mucosal damage.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec4.1">Survival data</h3>
<p>For comparative virulence studies, evaluation of antifungal prophylaxis, or therapeutic interventions prior to the time of infection, survival data are normalized to the number of larvae infected (or mock-infected). Aggregate data from several replicate experiments are analyzed using the Mantel Cox log-rank test. For a sufficiently powered analysis, we recommend performing 3 to 5 independent replicates with 15 larvae per cohort and replicate. This number of animals (n = 45–75 per group) will allow for a two-tailed analysis of survival differences ≥30% at an alpha-error of 5% and a beta-error of less than 20%. Survival corridors in uninfected larvae and larvae infected with a representative clinical <i>R. arrhizus</i> isolate are shown in <a href="#fig3">Figure 3</a>.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1193-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative survival curves of zebrafish larvae with epithelial homeostasis and induced epithelial cell loss after <i>R. arrhizus</i> infection and mock infection</div>
<p>(A) Median, mean, and range of survival rates observed in wild-type larvae with epithelial (ep.) homeostasis after infection with 5×10<sup>6</sup>/mL spores of clinical <i>R. arrhizus</i> strain #749 (Ra-749).</p>
<p>(B and C) Survival rates of larvae with induced cell loss and epithelial damage (Ep. damage) and mock infection with sterile E3 medium (B) or <i>R. arrhizus</i> infection (C). Data of each plot are based on 9 experimental series, each consisting of 3–5 independent replicate wells (n = 27–75 per replicate). Median intra-experiment coefficients of variation (CVs) of survival rates in Ra-749-infected larvae with epithelial damage were 10.6%–16.3% and the inter-experiment CVs across all 9 series were 8.8%–12.1%. Survival rates were determined by dividing the number of animals alive at each monitoring time point by the number of animals alive at the time of (mock) infection.</p>
</figcaption>
</figure>
<p>For therapeutic interventions after infection (e.g., investigational antifungal therapy), data are normalized to the number of larvae alive at the time of treatment initiation. As the number of larvae dying during the 16-h infection period will reduce the sample size, at least 5 independent replicates with 15 larvae per cohort will be needed.</p>
<h3 id="sec4.2">Fungal burden determination by quantitative PCR</h3>
<p>The final readout of the qPCR assay for each replicate is the (average) <i>R. arrhizus</i> spore equivalent per larva. Based on multiple replicates (we recommend at least 5 independent replicates with 15 larvae per group and replicate), two-group comparisons between different treatments (or a treatment and control) are made using the two-sided Mann-Whitney U test for unpaired samples or paired Wilcoxon test for paired samples. Multi-group comparisons are made using the Kruskal-Wallis test and Dunn’s multiple comparison test for unpaired samples or Friedman test with Dunn’s multiple comparison test for paired samples.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>We have validated our immersion infection protocol with various clinical Mucorales strains, wild-type <i>Candida albicans</i> strains, as well as several hypovirulent Mucorales and <i>C. albicans</i> mutants and found reproducible, inoculum-dependent survival outcomes. In general, our protocol also works with <i>Aspergillus fumigatus</i>, the most common mold pathogen in cancer patients; however, based on our unpublished experience using a reference strains and several clinical isolates, very high conidial inoculums (5×10<sup>7</sup>/mL – 1×10<sup>8</sup>/mL) are required to establish lethal infection and survival outcomes of <i>A. fumigatus</i> infection are less reproducible compared to infection experiments with the aforementioned species.</p>
<p>Furthermore, the assays cannot be performed on later developmental stages or adult animals since expression of the NTR enzyme is limited to larval stages. Future studies can focus on examining other damage paradigms of Mucoralean pathogenesis and invasion in adult zebrafish, such as wounding (<a href="#bib11">Richardson et al., 2013</a>) or burn models (<a href="#bib10">Miskolci et al., 2019</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Poor fungal yield (Steps 9–16 in the <a href="#sec1.1">preparation of buffers and media</a> section). Mucorales isolates usually grow rapidly on common mold media such as YAG plates and produce a high abundance of sporangiospores. However, poor yield might be encountered when re-culturing a frozen stock after cryopreservation for extended periods. Furthermore, moisture on the inner side of the lid of the culture plate can lead to mycelial overgrowth and poor spore yields.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>We recommend preparation of multiple plates for large-scale experiments. Removal of moisture from wet plates or lids (e.g., with a sterile cotton swab) can improve spore yields.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Increased larval mortality outside of normal range (Steps 10–15 in the <a href="#sec3.3">post-infection monitoring and treatment</a> sub-section). The mortality corridors shown in <a href="#fig1">Figure 1</a> are based on a single clinical <i>R. arrhizus</i> isolate. Although we found similar results with other Mucorales isolates from our institution, increased mortality could be a naturally occurring phenomenon and does not necessarily represent a technical issue, especially when survival outcomes of mock-infected larvae are within the expected range. However, there could also be technical reasons for increased larval mortality. Specifically, long processing times of larvae and bacterial contamination can contribute to high mortality.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Proper training of new investigators is warranted prior to engaging in actual infection experiments. In particular, it is critical to add fresh medium quickly after removing the supernatants from the wells in order to prevent the larvae from drying out. Furthermore, we recommend splitting large-scale experiments into smaller subunits, that is, processing 1 or 2 plates at a time while keeping the remaining plates at 28°C. Lastly, bacterial contamination of the medium and/or fungal spore inoculum should be ruled out when encountering heightened mortality (e.g., by streaking aliquots of the E3 medium and fungal inoculum on a bacterial medium such as blood agar).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Decreased mortality in assay and/or lack of epithelial extrusion (Steps 10–15 in the <a href="#sec3.3">post-infection monitoring and treatment</a> sub-section). Decreased mortality of infected larvae can result from insufficient MTZ exposure or diminishing fungal virulence.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>A fresh MTZ solution should be prepared for each experiment (Step 2 in the <a href="#sec1.1">preparation of buffers and media</a> section). As explained above, it is critical to use pre-warmed E3 medium (28°C) and thoroughly vortex the suspension in order to fully dissolve the MTZ at a concentration of 10 mM prior to filtering. We also recommend purchasing fresh MTZ powder on a quarterly basis. As virulence of fungi can drift with serial plating, we further recommend re-growing fungal pathogens from frozen stocks or a refrigerated master stock to generate the working inoculum for each experiment in order to prevent repeated passaging.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Potential toxicity of antifungals or other pharmacological agents (Optional portion of Step 12 in the <a href="#sec3.3">post-infection monitoring and treatment</a> sub-section). A unique attribute of the zebrafish infection model presented here is that pharmacological agents can be rapidly assessed for mitigation of both mucosal damage or antifungal activity. Importantly, in contrast to infection models using cultured cells, the impact and potential toxicity of these compounds on distinct organ systems and organismal survival can be evaluated. Heightened toxicity of some antifungals might be encountered in larval zebrafish.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Upfront <i>in-vitro</i> efficacy and <i>in-vivo</i> toxicity testing of several concentrations is critical in order to define concentrations that can inhibit/kill the fungus while not impacting the fitness and viability of the larvae. For some agents, non-toxic yet effective concentrations might not be found, which is a potential limitation of the model.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Atieh, Y., Wyatt, T., Zaske, A.M., and Eisenhoffer, G.T. (2021). Pulsatile contractions promote apoptotic cell extrusion in epithelial tissues. Curr. Biol. <i>31</i>, 1129-1140 e4. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref1">View at publisher</a></p>
<p id="bib2">Brock, C.K., Wallin, S.T., Ruiz, O.E., Samms, K.M., Mandal, A., Sumner, E.A., and Eisenhoffer, G.T. (2019). Stem cell proliferation is induced by apoptotic bodies from dying cells during epithelial tissue maintenance. Nat. Commun. <i>10</i>, 1044. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref2">View at publisher</a></p>
<p id="bib3">Curado, S., Stainier, D.Y., and Anderson, R.M. (2008). Nitroreductase-mediated cell/tissue ablation in zebrafish: a spatially and temporally controlled ablation method with applications in developmental and regeneration studies. Nat. Protoc. <i>3</i>, 948-954. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref3">View at publisher</a></p>
<p id="bib4">Davison, J.M., Akitake, C.M., Goll, M.G., Rhee, J.M., Gosse, N., Baier, H., Halpern, M.E., Leach, S.D., and Parsons, M.J. (2007). Transactivation from Gal4-VP16 transgenic insertions for tissue-specific cell labeling and ablation in zebrafish. Dev. Biol. <i>304</i>, 811-824. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref4">View at publisher</a></p>
<p id="bib5">Eisenhoffer, G.T. and Rosenblatt, J. (2011). Live imaging of cell extrusion from the epidermis of developing zebrafish. J. Vis. Exp. <i>52</i>, 2689. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref5">View at publisher</a></p>
<p id="bib6">Eisenhoffer, G.T., Slattum, G., Ruiz, O.E., Otsuna, H., Bryan, C.D., Lopez, J., Wagner, D.S., Bonkowsky, J.L., Chien, C.B., Dorsky, R.I., and Rosenblatt, J. (2017). A toolbox to study epidermal cell types in zebrafish. J. Cell Sci. <i>130</i>, 269-277. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref6">View at publisher</a></p>
<p id="bib7">Gong, Z., Ju, B., Wang, X., He, J., Wan, H., Sudha, P.M., and Yan, T. (2002). Green fluorescent protein expression in germ-line transmitted transgenic zebrafish under a stratified epithelial promoter from keratin8. Dev. Dyn. <i>223</i>, 204-215. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref7">View at publisher</a></p>
<p id="bib8">Haas, P. and Gilmour, D. (2006). Chemokine signaling mediates self-organizing tissue migration in the zebrafish lateral line. Dev. Cell <i>10</i>, 673-680. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref8">View at publisher</a></p>
<p id="bib9">Ibrahim, A.S., Bowman, J.C., Avanessian, V., Brown, K., Spellberg, B., Edwards, J.E., and Douglas, C.M. (2005). Caspofungin inhibits Rhizopus oryzae 1,3-beta-D-glucan synthase, lowers burden in brain measured by quantitative PCR, and improves survival at a low but not a high dose during murine disseminated zygomycosis. Antimicrob. Agents Chemother. <i>49</i>, 721-727. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref9">View at publisher</a></p>
<p id="bib10">Miskolci, V., Squirrell, J., Rindy, J., Vincent, W., Sauer, J.D., Gibson, A., Eliceiri, K.W., and Huttenlocher, A. (2019). Distinct inflammatory and wound healing responses to complex caudal fin injuries of larval zebrafish. Elife <i>8</i>, e45976. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref10">View at publisher</a></p>
<p id="bib11">Richardson, R., Slanchev, K., Kraus, C., Knyphausen, P., Eming, S., and Hammerschmidt, M. (2013). Adult zebrafish as a model system for cutaneous wound-healing research. J. Invest. Dermatol. <i>133</i>, 1655-1665. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref11">View at publisher</a></p>
<p id="bib13">Wurster, S., Ruiz, O.E., Samms, K.M., Tatara, A.M., Albert, N.D., Kahan, P.H., Nguyen, A.T., Mikos, A.G., Kontoyiannis, D.P., and Eisenhoffer, G.T. (2021). EGF-mediated suppression of cell extrusion during mucosal damage attenuates opportunistic fungal invasion. Cell Rep. <i>34</i>, 108896. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00669-9/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and request for resources and reagents should be directed to and will be fulfilled by the lead contact, George Eisenhoffer (<a href="mailto:gteisenhoffer@mdanderson.org">gteisenhoffer@mdanderson.org</a>).</p>
<h4>Materials availability</h4>
<p>Transgenic zebrafish lines used in this study are available upon request.</p>
<h4>Data and code availability</h4>
<p>The published article includes all datasets and codes generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the <a href="https://doi.org/10.13039/100004917">Cancer Prevention Institute of Texas</a>, RR14007, <a href="https://doi.org/10.13039/100000057">National Institutes of General Medical Sciences</a>, R01GM124043, and the Linda and Mark Quick Award for Basic Science to GTE; and in part by the Cancer Center Support Grant (P30CA016672), the Cyrus Scholar Award to S.W., and the Texas 4000 distinguished professorship endowment and Robert C. Hickey Chair in Clinical Care endowment to D.P.K. We thank Jordan Pietz at the Creative Communications group at MD Anderson Cancer Center for the biomedical visualizations. Schematics in the graphical abstract were created with <a href="https://biorender.com">BioRender.com</a>. The zebrafish schematic in the graphical abstract reprinted with permission from <a href="#bib2">Brock et al., (2019)</a>. We would like to thank Nathaniel D. Albert, Krystin M. Samms, and Philip H. Kahan for their excellent technical support.</p>
<h3>Author contributions</h3>
<p>S.W., O.E.R., A.M.T., D.P.K., and G.T.E. conceived the method and protocol. S.W. and O.E.R. performed all the experiments and subsequent analysis. S.W., O.E.R., and G.T.E. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>D.P.K. reports honoraria and research support from Gilead Sciences, received consultant fees from Astellas Pharma and Gilead Sciences, and is a member of the Data Review Committee of Cidara Therapeutics. All other authors declare no competing interests.</p>
</section>
</article>