<article>
<section>
<h1 id="header">Induction of granule and Purkinje cells from primary cultured mouse cerebellar progenitors</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Tingting Zhang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Tengyuan Liu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Bassem A. Hassan<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Institut du Cerveau – Paris Brain Institute – ICM, Sorbonne Université, Inserm, CNRS, Hôpital Pitié-Salpêtrière, Paris, France</p>
<p id="aff2"><sup>2</sup>Doctoral School of Biomedical Sciences, KU Leuven, 3000 Leuven, Belgium</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:bassem.hassan@icm-institute.org">bassem.hassan@icm-institute.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100760">10.1016/j.xpro.2021.100760</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The architecturally stereotypical structure of cerebellum is ideal for investigating the generation of neuronal diversity, but <i>in vitro</i> models for assessing early cerebellar progenitor differentiation were lacking. Here, we report a detailed protocol for long-term <i>in vitro</i> generation of Pax6<sup>+</sup> granule cells and Calbindin<sup>+</sup> Purkinje cells from common Sox2<sup>+</sup> embryonic cerebellar progenitors. We describe the procedure for dissecting mouse cerebellar anlage, cell seeding, and tamoxifen-induced labeling of progenitor cells, followed by time-lapse video recording of clonal expansion and neuronal differentiation.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref7">Zhang et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Embryonic mouse cerebellar anlage isolation and primary cell culture</li>
<li>Induced differentiation of cerebellar progenitors into granule and Purkinje cells</li>
<li>Long-term time-lapse recording of clonal expansion and neuronal differentiation</li>
<li>Identification of granule neurons and Purkinje neurons in single progenitor clones</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/946-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The following protocol describes the detailed steps for primary cell collection carried out in accordance with animal welfare regulations and approved by the local Ethics Committee and local regulatory authorities of the institute. Mouse cerebellar anlage are harvested and processed under sterile conditions. Prepare sterilized surgical tools and buffers before sample collection. The cell seeding plates should be pretreated two days before sample processing. The detailed materials, reagents, concentrations and equipment that are required can be found below.</p>
<h3 id="sec1.1">Pregnant female (E11.5) preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 days</div>
<p>Set up overnight breeding crosses in the afternoon by using one male with one or two females in each cage. The next morning check plugs of the females.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid setting up new crosses by using females from the night before where no plugs were observed. Because the mouse cerebellum develops quite fast at early embryonic stages it is imperative to time the embryonic days as accurately as possible.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare more than one pregnant female for each experiment to make sure you can harvest enough cells for your experiments. You may need 10–15 embryos to ensure enough cells can be harvested to proceed further.</div>
<h3 id="sec1.2">Preparation of reagents</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol>
<li>Stock solutions for plate coating
<ol type="a">
<li>1 mg/mL Poly-L-ornithine hydrobromide: Dissolve the powder in Milli-Q water, aliquot as 1 mL/tube and store at −20°C (Can be stored for a year or longer).</li>
<li>1 mg/mL Laminin: Aliquot the Laminin solution as 100 μL/tube and store at −20°C (Can be stored for a year or longer) to avoid repeated freezing and thawing.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Stock for growth factors
<ol type="a">
<li>100 μg/mL bFGF: Dissolve recombinant mouse basic fibroblast growth factor to 100 μg/mL in sterile H<sub>2</sub>O, aliquot as 5 μL/tube and store at −20°C (Can be stored for a year or longer).</li>
<li>100 μg/mL EGF: Dissolve recombinant mouse epidermal growth factor to 100 μg/mL in sterile H<sub>2</sub>O, aliquot as 10 μL/tube and store at −20°C (Can be stored for a year or longer).</li>
<li>10 μM BMP4: Add 2 μL 36%–38% HCL to 6 mL Milli-Q water, then add 60 μL 10% bovine serum albumin to get 4 mM HCL containing 0.1% BSA. Filter the solution with 0.2 μm syringe filter. Reconstitute BMP4 at 1 mM in sterile 4 mM HCL containing 0.1% BSA by adding 763 μL of 4 mM HCl containing 0.1% BSA to 10 μg BMP4 (see the table below). Then dilute again to 10 μM concentration and aliquot as 50 μL/tube and store at −80°C (3 months under sterile conditions after reconstitution).
<table id="undtbl1">
<caption>1 mM BMP4</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Recombinant Mouse BMP-4 Protein</td>
<td>N/A</td>
<td>10 μg</td>
</tr>
<tr>
<td>4 mM HCl containing 0.1% BSA</td>
<td>N/A</td>
<td>763 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 mM</b></td>
<td>763 μL</td>
</tr>
</tbody>
</table>
</li>
<li>100 ng/μL Wnt3a: Dissolve recombinant mouse Wnt-3a protein to 100 ng/μL in sterile PBS containing 0.1% FBS, aliquot as 10 μL/tube and store at −20°C (3 months under sterile conditions after reconstitution).</li>
</ol>
</li>
</ol>
<ol start="3">
<li>10000 U/mL DNAse I: Dissolve Deoxyribonuclease I powder to ≈ 10000 U/mL in sterile H<sub>2</sub>O, aliquot as 120 μL/tube and store at −20°C (up to one year).</li>
<li>B27: Aliquot as 1 mL/tube and store at −20°C (up to one year).</li>
<li>200 mM L-Ascorbic acid: Dissolve the L-Ascorbic acid to 200 mM in sterile H2O, aliquot as 100 μL/tube and store at 4°C (Can be stored for at least 3 months) away from light.</li>
</ol>
<h3 id="sec1.3">Seeding plate preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="6">
<li>Day 1: Immerse 13 mm cover slips and a tweezer in 70% ethanol for 5 min for sterilization, then place the cover slips in a 24 well plate and dry the plate under the hood. Prepare the poly-L-ornithine hydrobromide solution with the final concentration of 20 μg/mL in sterile H<sub>2</sub>O and filter with 0.2 μm syringe filter. Add 500 μL filtered poly-L-ornithine hydrobromide in each well and incubate overnight (12–36 h).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Here you can also use ready to use Poly-L-ornithine solution.</div>
<ol start="7">
<li>Day 2: Aspirate the poly-L-ornithine hydrobromide solution, wash 3 times with sterile H<sub>2</sub>O and dry the plate under the hood or in the incubator. Dilute 1 mg/mL Laminin to 5–10 μg/mL in sterile H<sub>2</sub>O and filter with 0.2 μm syringe filter. Add 500 μL filtered Laminin in each well and incubate overnight (12–36 h).</li>
<li>Day 3: Aspirate the Laminin solution and dry the plate under the hood or in the incubator.</li>
</ol>
<h3 id="sec1.4">Medium preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Prepare the necessary dissection buffer, digestion mixes, proliferation medium and differentiation media. Please see the detail recipes in the materials and equipment section.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit anti-Calbindin</td>
<td>ImmunoStar (1:500)</td>
<td>Cat# 24427</td>
</tr>
<tr>
<td>Mouse anti-Pax6</td>
<td>Synaptic Systems (1:300)</td>
<td>Cat# 153011</td>
</tr>
<tr>
<td>Goat anti-Rabbit IgG (H+L), Alexa Fluor<sup>TM</sup>488</td>
<td>Invitrogen (1:500)</td>
<td>A-11008</td>
</tr>
<tr>
<td>Goat anti-Mouse IgG (H+L), Alexa Fluor<sup>TM</sup>647</td>
<td>Invitrogen (1:500)</td>
<td>A-21235</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DAPI</td>
<td>Sigma</td>
<td>Cat# D9564</td>
</tr>
<tr>
<td>Tamoxifen</td>
<td>Sigma</td>
<td>Cat#T5648</td>
</tr>
<tr>
<td>Mounting Medium</td>
<td>Vector Laboratories</td>
<td>Cat#H-1000</td>
</tr>
<tr>
<td>Poly-L-ornithine solution</td>
<td>Sigma</td>
<td>P4957</td>
</tr>
<tr>
<td>Poly-L-ornithine hydrobromide</td>
<td>Sigma</td>
<td>P3655</td>
</tr>
<tr>
<td>Laminin</td>
<td>Sigma</td>
<td>L2020</td>
</tr>
<tr>
<td>L15 medium</td>
<td>Gibco</td>
<td>11415064</td>
</tr>
<tr>
<td>0.05% Trypsin/EDTA</td>
<td>Gibco</td>
<td>25300-054</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Gibco</td>
<td>10270106</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Sigma</td>
<td>A9647</td>
</tr>
<tr>
<td>DNAse I</td>
<td>Serlabo</td>
<td>LS002138</td>
</tr>
<tr>
<td>Neurobasal medium without phenol red</td>
<td>Gibco</td>
<td>12348-017</td>
</tr>
<tr>
<td>B27 supplement</td>
<td>Gibco</td>
<td>17504-044</td>
</tr>
<tr>
<td>L-glutamax (100×)</td>
<td>Gibco</td>
<td>35050-061</td>
</tr>
<tr>
<td>Mouse epidermal growth factor (EGF)</td>
<td>Thermo Fisher</td>
<td>PMG8041</td>
</tr>
<tr>
<td>Mouse basic fibroblast growth factor (bFGF)</td>
<td>Thermo Fisher</td>
<td>PMG0035</td>
</tr>
<tr>
<td>HEPES</td>
<td>Gibco</td>
<td>12509079</td>
</tr>
<tr>
<td>Insulin</td>
<td>Sigma</td>
<td>I0516</td>
</tr>
<tr>
<td>KnockOut ™ Serum Replacement (KSR)</td>
<td>Gibco</td>
<td>10828010</td>
</tr>
<tr>
<td>2-Mercaptoethanol (50 mM) (2-ME)</td>
<td>Gibco</td>
<td>31350010</td>
</tr>
<tr>
<td>Pen/Strep (100×)</td>
<td>Gibco</td>
<td>15140-122</td>
</tr>
<tr>
<td>BMP4</td>
<td>R&amp;D</td>
<td>5020-BP-010</td>
</tr>
<tr>
<td>Recombinant mouse Wnt-3a protein</td>
<td>R&amp;D</td>
<td>1324-WNP-010</td>
</tr>
<tr>
<td>Neurobasal-Plus</td>
<td>Gibco</td>
<td>A3582901</td>
</tr>
<tr>
<td>B27-Plus</td>
<td>Gibco</td>
<td>A3582801</td>
</tr>
<tr>
<td>L-Ascorbic acid</td>
<td>Sigma</td>
<td>A4403</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>Sox2</i><sup><i>CreERT2</i></sup>
</td>
<td>The Jackson Laboratory</td>
<td>JAX stock #017593</td>
</tr>
<tr>
<td>
<i>Gt(ROSA)26Sor</i><sup><i>tdTom</i></sup>
</td>
<td>The Jackson Laboratory</td>
<td>JAX stock #007914</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Forward primer for Cre: 5′-CCAATTTACTGACCGTACACCAA-3′</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>Reverse primer for Cre: 5′-CTGATCCTGGCAATTTCGGCTA-3′</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>Forward primer for tdTomato: WT-5′-AAGGGAGCTGCAGTGGAGTA-3’;Mut-5′-CTGTTCCTGTACGGCATGG-3′</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>Reverse primer for tdTomato: WT-5′-CCGAAAATCTGTGGGAAGTC-3’;Mut-5′-GGCATTAAAGCAGCGTATCC-3′</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ZEISS ZEN Blue software</td>
<td>Zeiss</td>
<td><a href="https://www.zeiss.com/">https://www.zeiss.com/</a></td>
</tr>
<tr>
<td>ImageJ</td>
<td>ImageJ</td>
<td><a href="https://imagej.nih.gov/ij/download.html">https://imagej.nih.gov/ij/download.html</a></td>
</tr>
<tr>
<td>FV-OSR software</td>
<td>Olympus</td>
<td><a href="https://www.olympus-lifescience.com/en/">https://www.olympus-lifescience.com/en/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Videomicroscope</td>
<td>Zeiss</td>
<td>AxioObserver 7</td>
</tr>
<tr>
<td>Confocal microscope</td>
<td>Olympus</td>
<td>FV-1200</td>
</tr>
<tr>
<td>0.2 μm Syringe filter</td>
<td>Sartorius</td>
<td>16532-K</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p><b>Mouse line:</b> The <i>Sox2</i><sup><i>CreERT2</i></sup> mice are crossed with <i>Gt(ROSA)26Sor</i><sup><i>tdTom</i></sup> reporter mice to generate the lineage tracing line. Do genotyping for both males and females, and keep the Cre<sup>+</sup> and tdTomato<sup>+</sup> mice for pregnant female preparation.</p>
<table id="undtbl2">
<caption>Dissection buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>L15 medium</td>
<td>N/A</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Keep the solution cold and put it on ice before starting dissection.</div>
<table id="undtbl3">
<caption>Proliferation medium (NBC)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neurobasal medium without phenol red</td>
<td>N/A</td>
<td>47.5 mL</td>
</tr>
<tr>
<td>B27 (50×)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>Glutamax (100 ×, 200 mM)</td>
<td>2 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>HEPES 1M</td>
<td>5 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>EGF</td>
<td>20 ng/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>bFGF</td>
<td>10 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Insulin</td>
<td>20 μg/mL</td>
<td>20 μL</td>
</tr>
<tr>
<td>Pen/Strep (100×)</td>
<td>1×</td>
<td>500 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Use fresh-made medium, store the NBC medium at 4°C for up to 2 weeks.</div>
<div class="note">
<span class="note-title">Note:</span> For live imaging, Neurobasal medium without phenol red should be used, because phenol red is toxic for fluorescence.</div>
<div class="note">
<span class="note-title">Note:</span> Lower Pen/Strep concentration could be used or entirely skipped if needed.</div>
<table id="undtbl4">
<caption>Differentiation medium for Pax6<sup>+</sup> granule cells</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neurobasal medium without phenol red</td>
<td>N/A</td>
<td>46.5 mL</td>
</tr>
<tr>
<td>KSR</td>
<td>5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Glutamax (100 ×, 200 mM)</td>
<td>2 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>2-ME (50 mM)</td>
<td>0.1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Wnt3a (100 ng/μL)</td>
<td>20 ng/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>BMP4 (10 μM)</td>
<td>0.5 nM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Pen/Strep (100×)</td>
<td>1<b>×</b>
</td>
<td>500 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>Differentiation medium for Calbindin<sup>+</sup> Purkinje cells</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neurobasal medium without phenol red</td>
<td>N/A</td>
<td>48 mL</td>
</tr>
<tr>
<td>B27-Plus</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>Glutamax (100 ×, 200 mM)</td>
<td>2 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>L-Ascorbic acid (200 mM)</td>
<td>0.2 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Wnt3a (100 ng/μL)</td>
<td>20 ng/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>BMP4 (10 μM)</td>
<td>0.5 nM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Pen/Strep (100×)</td>
<td>1×</td>
<td>500 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Use fresh-made differentiation medium, store the medium at 4°C for up to 2 weeks. To increase the stability of growth factors, it is better to add fresh Wnt3a and BMP4.</div>
<div class="note">
<span class="note-title">Note:</span> Lower Pen/Strep concentrations could be used or entirely skipped if needed.</div>
<div class="note">
<span class="note-title">Note:</span> Wnt3a promotes the production of granule cells and Purkinje cells (<a href="#bib6">Su et al., 2006</a>), but it is not necessary.</div>
<table id="undtbl6">
<caption>Digestive termination buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Proliferation medium (NBC)</td>
<td>N/A</td>
<td>2680 μL</td>
</tr>
<tr>
<td>FBS</td>
<td>30%</td>
<td>1200 μL</td>
</tr>
<tr>
<td>DNAse I</td>
<td>100 U/mL</td>
<td>120 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>4 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Use fresh-made digestive termination buffer.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Dissection and isolation of mouse cerebellar anlage</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This section describes the procedures for embryonic cerebellar anlage dissection and isolation, which should be done for sterile operation (<a href="#fig1">Figure 1</a>). Because at this stage (E11.5) the mouse cerebellar anlage is very small, it is better to prepare two separate sets of tweezers: one pair of tweezers for the pregnant mice dissection and embryo isolation, the other pair of much finer microsurgical tweezers for further dissection of the required cerebellar regions. Using separate surgical tools could also reduce contamination risk.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/946-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic representation of mouse cerebellar anlage collection steps from E11.5 mouse embryos</div>
<p>Scalebars = 1 inches.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Pre-cool the dissection medium, and put the medium on ice during cerebellar anlage dissection. Try to finish sample collecting as fast as possible to keep high cell viability.</div>
<ol>
<li>Sterilize the surgery tools, operation area and dissection microscopes.</li>
<li>Euthanize one <i>Sox2</i><sup><i>CreERT2</i></sup><i>/ Gt(ROSA)26Sor</i><sup><i>tdTomtato</i></sup> pregnant mouse by using methods approved by the animal welfare committee.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Dissection of required cerebellar region needs relatively long time, so euthanize the pregnant mice one by one.</div>
<ol start="3">
<li>Take out all the embryos from the pregnant mouse under binocular microscope and put them in cold L15 medium.</li>
<li>Dissect the cerebellar anlage under binocular microscope by using fine microsurgical tweezers and collect the tissues into a 6 cm petri dish filled with cold L15 medium on ice. (<a href="#fig1">Figure 1</a>)</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> There is a transparent vascular membrane on the surface of the cerebellar anlage. If possible, try to remove the vascular membrane by using fine microsurgical tweezers carefully.</div>
<h3 id="sec3.2">Tissue dissociation and cell seeding</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>These steps describe how to digest collected cerebellar anlage tissues into single cells and the details for cell seeding and Rosa-Tomato signal activation.</p>
<ol start="5">
<li>Once dissection is finished, aspirate all the tissues in 2 mL L15 medium and transfer them to a 15 mL tube.</li>
<li>Add 2 mL of 0.05% Trypsin-EDTA to the 15 mL tube for digestion.</li>
<li>Incubate 10 min at room temperature (RT; 15°C–25°C), gently shaking to re-suspend the tissues every 2 min 10 min later, use a p1000 pipet to triturate the tissues by pipetting slowly and gently up and down about 20 times until few tissue clumps are visible.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> In the process of tissue digestion, check the samples regularly to avoid overdigestion. If it still has some tissue clumps left after pipetting, it means the digestion is inadequate. Put the 15 mL in ice for 5 min to wait to decant the big pieces of tissues, then collect the supernatant to a new 15 mL tube to follow step 8. Add new digestion buffer (2 mL L15 medium + 2 mL 0.05% Trypsin-EDTA) to the original 15 mL tube with precipitate and keep on ice to continue digestion for another 5 min, then repeat step 7.</div>
<ol start="8">
<li>Add 2 mL digestive termination buffer (NBC+FBS+DNAse I) to the 15 mL tube, and dissociate the cells by gently pipetting up and down.</li>
<li>Centrifuge for 5 min at 150 <i>g</i> at 4°C.</li>
<li>Remove the supernatant and resuspend the cell pellets in 2–4 mL NBC medium (without phenol red), depending on the size of cell pellets.</li>
<li>Count the cell number: 25 μL cell resuspension + 25 μL fresh NBC + 50 μL Trypan blue. Slowly mix well, then put 20 μL in a cell counting chamber and count at least 5 squares.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Calculation: mean of the cell numbers in square <b>×</b> 400 = cell number / μl.</div>
<p>Cell number / μL × cell suspension volume = total cell number.</p>
<ol start="12">
<li>Seed the cells at a cell density of 2–4 × 10<sup>5</sup> in pre-coated 24 well plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Because the cells will be cultured for 9–12 days, do not seed the plate with a very high cell density.</div>
<h3 id="sec3.3">Rosa-tomato activation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 h</div>
<ol start="13">
<li>Add tamoxifen to each well directly at a final 5–500 pg/mL concentration (<a href="#fig2">Figures 2</a>A–2D).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/946-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. RosaTomato activation in primary cultured cerebellar progenitors</div>
<p>(A–D) Fluorescence of RosaTomato signals 3 h and 12 days after Tamoxifen adding. Scalebar = 500 μm.</p>
<p>(E–H′) Fluorescence of RosaTomato signals (E–H) and phase contrast images (E′–H′) 2 weeks after cell proliferation and differentiation. Scalebars = 100 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The concentration of tamoxifen depends on the efficiency of Cre activation. Do not use very high tamoxifen concentration, otherwise too many progenitor cells will be labeled with Tomato and it will be hard to distinguish single cell clones (<a href="#fig2">Figures 2</a>E–2H′). It is critical to calibrate the concentration to be used in advance.</div>
<ol start="14">
<li>Incubate the cells in a 5% CO<sub>2</sub> 37°C incubator for 2–3 h. Check Rosa-Tomato signals under fluorescence microscope. When sparse Tomato signals could be detected (<a href="#fig2">Figures 2</a>A and 2C), begin the time-lapse video recording step.</li>
</ol>
<h3 id="sec3.4">Time-lapse video recording</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 9–12 days</div>
<p>This step outlines the procedures of live imaging for cell proliferation and differentiation.</p>
<ol start="15">
<li>2–3 h after cell seeding, transfer the 24-well plate to the videomicroscope (Zeiss AxioObserver 7) within a humidified incubator at 37°C with a constant 5% CO<sub>2</sub> supply.</li>
<li>Set up the parameters for cell proliferation stages: time-lapse images for both Cy3 and bright field are acquired every 30 min which lasts for 72 h, by using 10× objective (512∗512 resolution).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Due to cell movement, it is better to take images for the entire view of each well.</div>
<div class="note">
<span class="note-title">Note:</span> We prepared four wells for time-lapse video recording experiments each time. This results in very large datasets. To avoid crashing during acquisition, stop and immediately restart the acquisition every 24 hours, or more frequently if needed.</div>
<div class="note">
<span class="note-title">Note:</span> Check the temperature, CO<sub>2</sub> concentration and water regularly.</div>
<ol start="17">
<li>Three days later, single Tomato<sup>+</sup> clones will be formed (<a href="#fig3">Figure 3</a>). Take out the 24-well plate and change to differentiation medium for Pax6<sup>+</sup> granule cells. Use the same parameters to start the program again.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/946-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Live imaging showed that a single Sox2 progenitor cell (Tomato<sup>+</sup>, Red) could divide several times to generate a cluster</div>
<p>Scalebar = 100 μm.</p>
</figcaption>
</figure>
</li>
<li>At day 7, take out the plate and change to differentiation medium for Calbindin<sup>+</sup> Purkinje cells. Use the same parameters to start the program again.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After changing to differentiation medium, the position and morphology of each clone will not change as fast as proliferation stages, therefore, in order to reduce the files size, the time interval could be increased. In our study, we used 45 min for Pax6<sup>+</sup> granule cells differentiation and 60 min for Calbindin<sup>+</sup> Prukinje cells differentiation.</div>
<div class="note">
<span class="note-title">Note:</span> 7 days after cell proliferation and differentiation, the cell density will be very high, so the differentiation medium should be changed every 2–3 days.</div>
<h3 id="sec3.5">Identification of granule neurons and Purkinje neurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>Here we describe the procedure for immunofluorescence-based staining to confirm the identity of differentiated granule and Purkinje cells (<a href="#fig4">Figure 4</a>).</p>
<ol start="19">
<li>After time-lapse video recording, cells were fixed directly by using 4% PFA and incubated for 10 min at room temperature (RT).</li>
<li>Wash 3 × 10 min in 1 × PBS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To increase the permeability for primary antibody, especially for nuclear expression proteins, the first 10 min of washing should be performed with 1<b>×</b> PBT (1 <b>×</b> PBS containing 0.1% Triton-100) instead of PBS.</div>
<ol start="21">
<li>Block with 10% normal donkey or goat serum in 1 × PBS for 1 h at RT.</li>
<li>Incubate fixed cells with primary antibodies: Pax6 (1:300, mouse) for granule cells and Calbindin (1:500, rabbit) for Purkinje cells, diluted in 1 × PBS containing 1% normal donkey or goat serum, at 4°C overnight (12–36 h) or 3–4 h at RT.</li>
<li>Wash 3 × 10 min in 1 × PBS.</li>
<li>Incubate fixed cells with appropriate secondary antibodies conjugated with Alexa Fluor 488 (1:500) for Calbindin and Alexa Fluor 647 (1:500) for Pax6, diluted in 1 × PBS containing 1% normal donkey or goat serum for 1 h at RT.</li>
<li>Wash 3 × 10 min in 1 × PBS.</li>
<li>Incubate cells with DAPI (1:2000, Sigma) for 5 min at RT.</li>
<li>Wash 3 × 10 min in 1 × PBS. Then mount cells with Vectashield.</li>
<li>After staining, images were obtained by using confocal microscope (Olympus FV-1200).</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/946-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Cerebellar progenitors could differentiate into both granule and Purkinje neurons in vitro</div>
<p>(A) Immunolabeling of both PC marker (Calbindin, green) or GC marker (Pax6, gray) co-localized with Tomato (red) after cell differentiation, respectively.</p>
<p>(B–E’’’) Higher magnification of the rectangular region in (A).</p>
<p>Nuclei were stained with DAPI (blue). Arrows indicate Calbindin<sup>+</sup> / Tomato<sup>+</sup> double positive cells and arrow heads indicate Pax6<sup>+</sup> / Tomato<sup>+</sup> double positive cells. Scalebars=100 μm and 15 μm. Images were taken at the end of 12 days of live imaging.</p>
</figcaption>
</figure>
<h3 id="sec3.6">Data analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>This step offers time-lapse video recording data analysis and video reconstruction. Because the files are quite large, a super powerful computer should be provided.</p>
<ol start="29">
<li>Use Zen Pro software to separate the scene of each well to get relatively small files. Perform stitching for each scene to improve the image quality.</li>
<li>First, scan the stitched files day by day to follow the proliferation and differentiation of each single clone. Note the positions of selected clones.</li>
<li>Then use Image J to open the stitched files. According to written positions, crop the target areas one by one to a consistent size large enough to include the entire cell at the end of differentiation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is quite normal to spend 30–60 min to open one file, so crop and save all the target regions for each file.</div>
<div class="note">
<span class="note-title">Note:</span> Before cropping, check the size of the target clones that are taken photos on the last day of live imaging to make sure you can crop appropriate areas for the first several days. Because the first 3 days of cell proliferation, the clones are very small, they could grow into quite big clones after several days of cell differentiation.</div>
<ol start="32">
<li>Use Image J to open these cropped images and reconstruct videos for each clone. Here are two examples for 12 days time-lapse video recording (<a href="#mmc1">Methods video S1</a> and <a href="#mmc2">S2</a>).</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/946-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. 12-day time-lapse video recordings of single Sox2+ cerebellar progenitors generating neuronal clones, related to steps 29–32</div>
</figcaption>
</figure>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/946-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. 12-day time-lapse video recordings of single Sox2+ cerebellar progenitors generating neuronal clones, related to steps 29–32</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>There currently are no simple and efficient methods to perform long-term in vivo real-time observations and tracing for mouse embryonic cerebellar progenitors. In our study, our sparse lineage tracing combined with Mosaic Analysis with Double Markers (MADM)-based single cell lineage tracing results provide conclusive evidence that a single cerebellar progenitor can generate both inhibitory and excitatory neuronal lineages (<a href="#bib7">Zhang et al., 2021</a>). However, we sought to directly observe Purkinje neurons and granules neurons being generated from one mother progenitor cell. Because the MADM technique relies on very low chromosome recombination probability (<a href="#bib1">Beattie et al., 2017</a>; <a href="#bib2">Gao et al., 2014</a>; <a href="#bib8">Zong et al., 2005</a>) there is no absolute certainty that any given clone in fact derives from a single progenitor. Although cerebellar slices could be ex vivo-cultured for several weeks (<a href="#bib4">Kondru et al., 2020</a>), this was done during postnatal stages. For E10.5-E11.5 embryonic stages, the mouse cerebellum is still a very small anlage, making <i>ex-vivo</i> culture very challenging. Because it was reported that it is possible to induce both granule cells and Purkinje cells from embryonic stem (ES) cells (<a href="#bib3">Kawasaki et al., 2000</a>; <a href="#bib6">Su et al., 2006</a>), we sought to establish a feasible protocol to culture and very sparsely label embryonic cerebellar progenitors and try to differentiate them into both glutamatergic and GABAergic lineages, which when combined with time-lapse video recording, would provide direct evidence that a single Sox2+ cerebellar progenitor can give rise to both Pax6<sup>+</sup> granule cell lineage and Calbindin<sup>+</sup> Purkinje cell lineage. In principle, this approach could be applied to any CNS progenitors of interest.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<h3 id="sec5.1">Sufficient cell collection</h3>
<p>The protocol described here is used for investigating the proliferation and differentiation of bi-potent cerebellar progenitors. The time points for cell harvest are very important. According to our study, E10.5 and E11.5 are the best stages for this protocol (<a href="#bib7">Zhang et al., 2021</a>). There are two reasons: first, although the mouse cerebellum anlage is induced around E8.5 (<a href="#bib5">Martinez et al., 2013</a>), it has clear distinguishable morphology/anatomy at around E10.5-E11.5. Second, the mouse cerebellum contains abundant bi-potential Sox2<sup>+</sup> cerebellar progenitors. However, the cerebellar anlage is still quite small, so it is difficult to do tissue dissection, especially when trying to remove the vascular membrane from the tissue, and the final number of cells collected is small. Normally, we get ∼0.8–2 × 10<sup>6</sup> cells from 10 embryos. Therefore, both tissue dissection and embryo numbers are critical factors in this protocol.</p>
<h3 id="sec5.2">Cell seeding density and tamoxifen concentration</h3>
<p>Optimal cell seeding density should be optimized according to the study purpose and the required assay. In our study, we tested 1–5 × 10<sup>5</sup> cells per well for 24-well plate by using a 10-fold gradient increase. Cells could survive for all of these concentrations. But for long-term differentiation, lower seeding densities are recommended, because cells will die after ∼12–14 days differentiation.</p>
<p>For Rosa-Tomato labeled single cell clone differentiation assay, an appropriate Tamoxifen concentration should be tested in advance to avoid too many Tomato signals to distinguish each single cell differentiated cluster.</p>
<p>Therefore, according to the aim of experiments and required assay, preliminary experiments should be done for further optimizations of this protocol.</p>
<h3 id="sec5.3">Time-lapse recording data saving and analysis</h3>
<p>To ensure the cells of observed clones are always within the field of microscope vision, it is better to image the entire well, which will generate a huge amount of data and the computer may not have enough memory to save all the data in time and this may lead to frequent crashes. Therefore, during the live imaging period, regularly checking the process of imaging is needed. In our study, we stop and restart the program every 24 h to make sure the computer works well. Due to the size of the data, for data analysis and video reconstruction, a powerful computer is needed.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low viability and cell yield after isolation and dissociation (steps 4 and 7).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Try to shorten the dissection and isolation time and keep the tissues in cold buffer on ice to reduce the decrease of cell viability. Try to remove the vascular membrane because it will have a negative influence on the cell yield and cell survival: first, some cells will stick to membrane to form tissue clumps during digestion step. In order to dissociate the cells to single cell, we needed to pipet more times, resulting in more cell death. Second, some vascular cells will mix with primary cultured progenitors and these cells could divide and finally affect the growth of cerebellar progenitors. For the tissue digestion step, avoid overdigestion and pipet gently to dissociate the cells.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Bad adherence of the seeding cells on pre-coated cover slips (step 12).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This may be caused by coating solutions. Try to use freshly prepared coating solution and fresh coated plate. We usually reuse Poly-L-ornithine hydrobromide once but never reuse Laminin. After coated, it is better to use the plate in one week.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Cells death during time-lapse recording (step 16).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Avoid using medium with phenol red for fluorescence labeled signals, because phenol red is toxic under fluorescence scanning.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Low differentiation efficiency of Purkinje cells (step 18).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Fresh made Neurobasal-Plus supplemented with B27-Plus medium is recommended since some growth factors are not stable. Fgf8b could be added to the differentiation medium (<a href="#bib6">Su et al., 2006</a>). Prolong the differentiation time for Purkinje cells. In our study, we found that compared with 3 days differentiation, 6 days differentiation could significantly increase Calbindin<sup>+</sup> cell number.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Microscope frequently out of focus during time-lapse recording (step 16).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>In addition to reducing the vibrations of the movement of the microscope, double check for plate deformation. We found that the TPP® tissue culture plates (Sigma, Z707791) will be slightly deformed after several hours scanning causing the images to become out of focus. Plates from Thermofisher are recommended.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Beattie, R., Postiglione, M.P., Burnett, L.E., Laukoter, S., Streicher, C., Pauler, F.M., Xiao, G., Klezovitch, O., Vasioukhin, V., Ghashghaei, T.H., et al. (2017). Mosaic analysis with double markers reveals distinct sequential Functions of Lgl1 in neural stem cells. Neuron <i>94</i>, 517-533.e3. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref1">View at publisher</a></p>
<p id="bib2">Gao, P., Postiglione, M.P., Krieger, T.G., Hernandez, L., Wang, C., Han, Z., Streicher, C., Papusheva, E., Insolera, R., Chugh, K., et al. (2014). Deterministic progenitor behavior and unitary production of neurons in the neocortex. Cell <i>159</i>, 775-788. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref2">View at publisher</a></p>
<p id="bib3">Kawasaki, H., Mizuseki, K., Nishikawa, S., Kaneko, S., Kuwana, Y., Nakanishi, S., Nishikawa, S.I., and Sasai, Y. (2000). Induction of midbrain dopaminergic neurons from ES cells by stromal cell-derived inducing activity. Neuron <i>28</i>, 31-40. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref3">View at publisher</a></p>
<p id="bib4">Kondru, N., Manne, S., Kokemuller, R., Greenlee, J., Greenlee, M.H.W., Nichols, T., Kong, Q., Anantharam, V., Kanthasamy, A., Halbur, P., et al. (2020). an ex vivo brain slice culture model of chronic wasting disease: implications for disease pathogenesis and therapeutic development. Sci. Rep. <i>10</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref4">View at publisher</a></p>
<p id="bib5">Martinez, S., Andreu, A., Mecklenburg, N., and Echevarria, D. (2013). Cellular and molecular basis of cerebellar development. Front. Neuroanat. <i>7</i>, 18. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref5">View at publisher</a></p>
<p id="bib6">Su, H.L., Muguruma, K., Matsuo-Takasaki, M., Kengaku, M., Watanabe, K., and Sasai, Y. (2006). Generation of cerebellar neuron precursors from embryonic stem cells. Dev. Biol. <i>290</i>, 287-296. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref6">View at publisher</a></p>
<p id="bib7">Zhang, T., Liu, T., Mora, N., Guegan, J., Bertrand, M., Contreras, X., Hansen, A.H., Streicher, C., Anderle, M., Danda, N., et al. (2021). Generation of excitatory and inhibitory neurons from common progenitors via Notch signaling in the cerebellum. Cell Rep. <i>35</i>, 109208. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref7">View at publisher</a></p>
<p id="bib8">Zong, H., Espinosa, J.S., Su, H.H., and Muzumdar, M.D. (2005). Resource mosaic analysis with double markers in mice spatial specificity of the loss of the candidate gene by and Liqun Luo 1,2, ∗ intrachromosomal excision in animals homozygous for. Cell <i>121</i>, 479-492. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00466-4/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Bassem Hassan (<a href="mailto:bassem.hassan@icm-institute.org">bassem.hassan@icm-institute.org</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the program “<a href="https://doi.org/10.13039/501100006072">Investissements d’avenir</a>” ANR-10-IAIHU-06, <a href="https://doi.org/10.13039/100013327">ICM</a>, a <a href="https://doi.org/10.13039/100012946">Sorbonne Université</a> Emergence grant, an Allen Distinguished Investigator Award, a Neuro-Glia foundation grant, and the <a href="https://doi.org/10.13039/501100008236">Roger De Spoelberch Foundation</a> Prize (to B.A.H.). T.Z. and T.L. were supported by doctoral fellowships from the <a href="https://doi.org/10.13039/501100004543">China Scholarship Council</a>. All animal work was conducted at the PHENO-ICMice facility. The core is supported by two “<a href="https://doi.org/10.13039/501100006072">Investissements d’avenir</a>” (ANR-10- IAIHU-06 and ANR-11-INBS-0011-NeurATRIS) and the “<a href="https://doi.org/10.13039/501100002915">Fondation pour la Recherche Médicale.</a>” Light microscopy work was carried out at ICM’s imaging core facility, ICM.Quant, and primary culture work followed by time-lapse recording were carried out at the CELIS core facility.</p>
<h3>Author contributions</h3>
<p>B.A.H. and T.Z. designed the experimental protocols. T.Z. and T.L. performed experiments. T.Z. performed the data analysis and prepared the manuscript. B.A.H. revised the protocols.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>