<article>
<section>
<h1 id="header">Protocol for purification and identification of MHC class I immunopeptidome from cancer cell lines</h1>
<p><time datetime="2021-03-12">Published: March 12, 2021</time></p>
<p>Sonali V. Mohan,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">6</a>,<a href="#cor1">*</a></sup> Keshava K. Datta,<sup><a href="#aff1">1</a></sup> Rebekah Ziegman,<sup><a href="#aff1">1</a></sup> Corey Smith,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a></sup> and Harsha Gowda<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Cancer Precision Medicine Group, QIMR Berghofer Medical Research Institute, Herston, QLD, Australia</p>
<p id="aff2"><sup>2</sup>Faculty of Medicine, The University of Queensland, Herston, QLD, Australia</p>
<p id="aff3"><sup>3</sup>School of Biomedical Sciences, Faculty of Health, Queensland University of Technology, Brisbane, QLD, Australia</p>
<p id="aff4"><sup>4</sup>QIMR Centre for Immunotherapy and Vaccine Development, QIMR Berghofer Medical Research Institute, Brisbane, QLD, Australia</p>
<p id="aff5"><sup>5</sup>Translational and Human Immunology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane, QLD, Australia</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:sonali.mohan@qimrberghofer.edu.au">sonali.mohan@qimrberghofer.edu.au</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:harsha.gowda@qimrberghofer.edu.au">harsha.gowda@qimrberghofer.edu.au</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100385">10.1016/j.xpro.2021.100385</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Major histocompatibility complexes (MHC) play a critical role in immunity by presenting peptides on the cell surface for T cell recognition. Identification of these peptides can be valuable to develop vaccines or immunotherapeutic strategies for infectious diseases and cancers. Mass spectrometry is the only tool available for unbiased identification of the immunopeptidome. Here, we describe a protocol for purification and identification of MHC class I peptides, including in-house purification of anti-MHC-antibody from hybridoma cells and the LC-MS/MS analysis of MHC-I bound peptides.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>In-house production and Sepharose conjugation of anti-MHC class I antibodies</li>
<li>Affinity purification of MHC-I complex from cultured cancer cell lysates</li>
<li>LC-MS/MS analysis of purified MHC-I bound peptides</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/517-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="note">
<span class="note-title">Note:</span> Substantial amount (10–20 mg) of anti-MHC class I antibody is required to pull down MHC complex and identify bound peptides. This antibody is commercially available and can be purchased in bulk. However, if it is cost prohibitive, the following section provides step-by-step protocol to culture the hybridoma cells and purify the antibodies in-house</div>
<h3 id="sec1.1">Hybridoma culture for antibody production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–6 weeks</div>
<ol>
<li>Thaw one vial of W6/32 (HB-95) hybridoma cells in DMEM supplemented with 10% FBS at a density of 2e5/mL.</li>
<li>Change media on 2nd day and maintain them until they reach a density of 1e6/mL.</li>
<li>Split the cells at a ratio of 1:1 and change media from DMEM with 10% FBS to HyClone CDM4MAb serum free media (SFM).</li>
<li>Maintain cells until they reach a density of 1e6/mL.</li>
<li>Transfer the cells to Corning Roller Bottles and maintain them on constant agitation (1 revolution per minute) so that cells do not settle down.</li>
<li>Add 100 mL of SFM every day until the total volume reaches 1 L.</li>
<li>Keep the cells on rotation for 1 month.</li>
<li>After a month, pellet the cells down by centrifuging them at 300 × <i>g</i> for 10 min at 20°C.</li>
<li>Collect the secretome and filter using 0.2 μm Nalgene Rapid-Flow Sterile Single Use Vacuum Filter Units.</li>
</ol>
<h3 id="sec1.2">Purification of antibody W6/32 from cell secretome</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12–24 h</div>
<div class="note">
<span class="note-title">Note:</span> Perform antibody purification at 4°C.</div>
<ol start="10">
<li>Transfer 4 mL of protein A Sepharose beads to a glass column.</li>
<li>Wash the column with 10 column volumes (c.v.) of PBS, pH 7.4.</li>
<li>Pass the secretome through the column (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/517-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Setup for purification of MHC class I antibodies from secretome of HB-95 cells</div>
</figcaption>
</figure>
</li>
<li>Wash the antibody-bound protein A column with 50 c.v. of PBS.</li>
<li>Elute W6/32 antibody with 5 c.v. of 100 mM glycine/HCl, pH 3.0 and collect fractions in tubes containing 200 μL 2 M Tris-HCl, pH 7.5.</li>
<li>Wash the column with 10 c.v. of 50 mM Tris-HCl, pH 8.0.</li>
<li>Pass 20 c.v. of PBS through the column.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The pH of solution flowing out of the column should be &gt; 7.</div>
<ol start="17">
<li>Store the column in 0.02% sodium azide/PBS at 4°C.</li>
<li>Take 5 μL from each fraction and run an SDS-PAGE. Pool all fractions containing antibody (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/517-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. SDS-PAGE of fractions collected during antibody purification</div>
</figcaption>
</figure>
</li>
<li>Estimate the concentration and yield of antibody using NanoDrop or any protein estimation strategy.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The final pH of antibody fractions collected should be between 7.5–8 to make sure antibodies are not denatured.</div>
<div class="note">
<span class="note-title">Note:</span> The column prepared for purification can be re-used if stored at 4°C to avoid fungal growth. Antibodies can also be procured commercially</div>
<h3 id="sec1.3">Cancer cell line culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 weeks</div>
<div class="note">
<span class="note-title">Note:</span> Number of cells required to identify MHC bound peptides varies based on expression level of MHC, sensitivity of mass spectrometer, and antibody affinity during immunoprecipitation. Start the experiment with 250–500 million cells to achieve sufficient depth and coverage of MHC peptides.</div>
<ol start="20">
<li>Culture the cancer cell lines of interest in appropriate media</li>
<li>Harvest suspension cell lines as follows -
<ol type="a">
<li>Transfer the cells to centrifuge tubes.</li>
<li>Take an aliquot of cell suspension to count the number of cells.</li>
<li>Pellet the cells down by centrifuging them at 350 × <i>g</i> for 10 min at 20°C–25°C.</li>
<li>Wash cell pellets 3 times with PBS at 20°C–25°C.</li>
<li>Aspirate PBS and freeze the cell pellets at −80°C if you plan to carry out lysis and subsequent steps later.</li>
</ol>
</li>
<li>Harvest the adherent cells using versene
<ol type="a">
<li>Aspirate cell culture media from the culture flask.</li>
<li>Wash the cells with 2–5 mL versene solution and incubate the flask for 10 min at 20°C–25°C.</li>
<li>Add PBS to the flask and harvest the cells.</li>
<li>Take an aliquot of cell suspension to count the number of cells.</li>
<li>Wash cell pellets 3 times with PBS at 20°C–25°C.</li>
<li>Aspirate PBS and freeze the cell pellets at −80°C if you plan to carry out lysis and subsequent steps later.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to ensure cell viability of ≥90% after harvesting</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>InVivoMAb anti-human MHC Class I (HLA-A, HLA-B, HLA-C) (optional)</td>
<td>Assay Matrix Pty Ltd</td>
<td>Cat#BE0079</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Tris hydrochloride</td>
<td>Sigma-Aldrich</td>
<td>Cat#10812846001</td>
</tr>
<tr>
<td>Hydrochloric acid 37% (HCl)</td>
<td>Sigma-Aldrich</td>
<td>Cat#320331-500ML</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>Sigma-Aldrich</td>
<td>Cat#S2002-5G</td>
</tr>
<tr>
<td>Versene solution</td>
<td>Life Technologies</td>
<td>Cat#15040066</td>
</tr>
<tr>
<td>Boric acid</td>
<td>Sigma-Aldrich</td>
<td>Cat#B0394-100G</td>
</tr>
<tr>
<td>Potassium chloride (KCl)</td>
<td>Sigma-Aldrich</td>
<td>Cat#P9333-500G</td>
</tr>
<tr>
<td>Triethanolamine</td>
<td>Sigma-Aldrich</td>
<td>Cat#90279-100ML</td>
</tr>
<tr>
<td>Ethanolamine (optional)</td>
<td>Sigma-Aldrich</td>
<td>Cat# E9508-100ML</td>
</tr>
<tr>
<td>Sodium hydroxide (NaOH)</td>
<td>Sigma-Aldrich</td>
<td>Cat#221465-500G</td>
</tr>
<tr>
<td>Citric acid</td>
<td>Sigma-Aldrich</td>
<td>Cat#251275-100G</td>
</tr>
<tr>
<td>IGEPAL CA-630 (NP-40)</td>
<td>Sigma-Aldrich</td>
<td>Cat#I8896-50ML</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>Sigma-Aldrich</td>
<td>Cat#D6750-10G</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Sigma-Aldrich</td>
<td>Cat#S7653-250G</td>
</tr>
<tr>
<td>cOmplete Protease Inhibitor Cocktail</td>
<td>Merck</td>
<td>Cat#11697498001</td>
</tr>
<tr>
<td>Iodoacetamide (IAA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#I1149-5G</td>
</tr>
<tr>
<td>CaptivA Protein A Affinity Resin</td>
<td>REPLIGEN</td>
<td>Cat#CA-PRI-0025</td>
</tr>
<tr>
<td>Dimethyl pimelimidate dihydrochloride (DMP)</td>
<td>Sigma-Aldrich</td>
<td>Cat#D8388-1G</td>
</tr>
<tr>
<td>LM- CDM4MAb W/ GLUT</td>
<td>GE Healthcare Life Sciences</td>
<td>Cat#SH30801.01</td>
</tr>
<tr>
<td>DMEM</td>
<td>Life Technologies</td>
<td>Cat#11995073</td>
</tr>
<tr>
<td>RPMI</td>
<td>Life Technologies</td>
<td>Cat#11875119</td>
</tr>
<tr>
<td>FBS</td>
<td>Life Technologies</td>
<td>Cat#10437-028</td>
</tr>
<tr>
<td>Acetonitrile, Optima LC/MS grade, Fisher Chemical</td>
<td>Fisher Scientific</td>
<td>Cat# A955-212</td>
</tr>
<tr>
<td>Water, Optima LC/MS grade, Fisher Chemical</td>
<td>Fisher Scientific</td>
<td>Cat# W6212</td>
</tr>
<tr>
<td>Trifluoroacetic acid, Optima LC/MS grade, Fisher Chemical</td>
<td>Fisher Scientific</td>
<td>Cat# A116-50</td>
</tr>
<tr>
<td>Formic acid, 99.0+%, Optima LC/MS grade, Fisher Chemical</td>
<td>Fisher Scientific</td>
<td>Cat# A117-50</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>ATCC W6/32; hybridoma; mouse (<i>Mus musculus</i>) BSL1, 1 mL</td>
<td>ATCC</td>
<td>ATCHB95</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Table S1</td>
<td>This study</td>
<td>Mendeley data (<a href="https://data.mendeley.com">https://data.mendeley.com</a> 10.17632/4bk7w8298s.1)</td>
</tr>
<tr>
<td>Mass spectrometry raw file</td>
<td>This study</td>
<td>Mendeley data (<a href="https://data.mendeley.com">https://data.mendeley.com</a> 10.17632/4bk7w8298s.1)</td>
</tr>
<tr>
<td>Proteome Discoverer Suite (2.2)</td>
<td>Thermo Scientific</td>
<td><a href="https://www.thermofisher.com/store/products/OPTON-30945#/OPTON-30945">https://www.thermofisher.com/store/products/OPTON-30945#/OPTON-30945</a></td>
</tr>
<tr>
<td>Mascot</td>
<td>Matrix Science</td>
<td><a href="https://www.matrixscience.com/">https://www.matrixscience.com/</a></td>
</tr>
<tr>
<td>Peaks X+</td>
<td>N/A</td>
<td><a href="https://www.bioinfor.com/peaks-studio/">https://www.bioinfor.com/peaks-studio/</a></td>
</tr>
<tr>
<td>MaxQuant</td>
<td><a href="#bib1">Cox and Mann, 2008</a></td>
<td><a href="https://www.maxquant.org/">https://www.maxquant.org/</a></td>
</tr>
<tr>
<td>Seq2Logo 2.0</td>
<td><a href="#bib2">Thomson and Nielsen, 2012</a></td>
<td><a href="http://www.cbs.dtu.dk/biotools/Seq2Logo/">http://www.cbs.dtu.dk/biotools/Seq2Logo/</a></td>
</tr>
<tr>
<td>NetMHCpan 4.1</td>
<td><a href="#bib3">Reynisson et al., 2020</a></td>
<td><a href="http://www.cbs.dtu.dk/services/NetMHCpan/">http://www.cbs.dtu.dk/services/NetMHCpan/</a></td>
</tr>
<tr>
<td>SYEPEITHI</td>
<td><a href="#bib4">Schuler et al., 2007</a></td>
<td><a href="http://www.syfpeithi.de/">http://www.syfpeithi.de/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>WaterXBridge column</td>
<td>Waters</td>
<td>Cat# 186008982</td>
</tr>
<tr>
<td>Acquity UPLC M-Class V/M symmetry trap column</td>
<td>Waters</td>
<td>Cat# 186007496</td>
</tr>
<tr>
<td>Acquity UPLC M-Class Peptide BEH C18 nanoAcquity Column 10K psi</td>
<td>Waters</td>
<td>Cat# 186003544</td>
</tr>
<tr>
<td>SpeedVac vacuum concentrator</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>Orbitrap Fusion Lumos Tribrid Mass Spectrometer</td>
<td>Thermo Scientific</td>
<td>NA</td>
</tr>
<tr>
<td>LC-MS certified total recovery vial</td>
<td>Waters</td>
<td>Cat# 600000671CV</td>
</tr>
<tr>
<td>HPLC with fraction collector</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>HPLC vials</td>
<td>Shimadzu</td>
<td>Cat# 226-50512-00</td>
</tr>
<tr>
<td>HPLC vial inserts</td>
<td>Shimadzu</td>
<td>Cat# 226-50523-00</td>
</tr>
<tr>
<td>Waters Acquity UHPLC</td>
<td>Waters</td>
<td>NA</td>
</tr>
<tr>
<td>Needles 18G × 38 mm</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>10–50 mL syringes</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>ECONO column 1 × 10, 2/Pack</td>
<td>Bio-Rad</td>
<td>Cat#7371012</td>
</tr>
<tr>
<td>Protein LoBind tubes</td>
<td>Eppendorf</td>
<td>Cat#0030108116</td>
</tr>
<tr>
<td>0.2 μm Nalgene rapid-flow sterile single use vacuum filter units</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#153-0020</td>
</tr>
<tr>
<td>Millex-HV syringe filter unit, 0.45 μm, PVDF, 33 mm</td>
<td>Merck</td>
<td>Cat# SLHV033RS</td>
</tr>
<tr>
<td>Corning 50 mL centrifuge tubes</td>
<td>Merck</td>
<td>Cat# CLS4558-300EA</td>
</tr>
<tr>
<td>Corning roller bottles, CellBIND cell culture surface, bottle surface area 1,700 cm<sup>2</sup>
</td>
<td>Merck</td>
<td>Cat# CLS431134</td>
</tr>
<tr>
<td>cRAP protein sequences</td>
<td> </td>
<td><a href="https://www.thegpm.org/crap/">https://www.thegpm.org/crap/</a></td>
</tr>
<tr>
<td rowspan="2">HLA allele expressed in cell lines</td>
<td>ATCC</td>
<td><a href="https://www.atcc.org/">https://www.atcc.org/</a></td>
</tr>
<tr>
<td>Cellosaurus</td>
<td><a href="https://web.expasy.org/cellosaurus/">https://web.expasy.org/cellosaurus/</a></td>
</tr>
<tr>
<td>NanoDrop</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Refrigerated centrifuge capable of achieving 20,000 ×<i>g</i>
</td>
<td>NA</td>
<td>NA</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p><b>Borate wash buffer:</b> 0.5 M boric acid and 0.5 M KCl, adjust the pH to 9 with NaOH. Add 0.31 g of boric acid and 0.38 g of KCl in 50 mL of water. Adjust pH to 9 with NaOH.</p>
<p><b>0.2 M Ethanolamine, pH 8</b> – Add 2.7 mL of triethanolamine to 100 mL of Milli-Q water and 1 mL of 33% HCl. Check the pH. If pH &gt;8.3 add more HCl and if it is less than 8 adjust the pH with 10% NaOH</p>
<div class="note">
<span class="note-title">Note:</span> pH range of 7.9–8.3 is acceptable to proceed further</div>
<p><b>40 mM DMP/0.2 M Ethanolamine, pH 8.3:</b> Add 100 mg of DMP in 9 mL of 0.2 M ethanolamine. The resulting pH will be around 7. Add 200 μL of 50% NaOH and check the pH. If pH is &lt;8.2 adjust it with NaOH and if it is &gt;8.4 adjust the pH with HCl</p>
<div class="note">
<span class="note-title">Note:</span> The rate of reaction is highest at pH 8.3. 5 mL of this solution is sufficient to bind 10 mg of antibody to 1 mL of protein A Sepharose beads.</div>
<div class="critical">
<span class="critical-title">Critical:</span> NP-40, sodium deoxycholate, IAA, TFA, formic acid, and acetonitrile are harmful and toxic. Work with these reagents in a chemical hood. Ethanolamine can be used as a substitute for triethanolamine. If working with ethanolamine, add 1.2 mL of ethanolamine to 60 mL of Milli-Q water. Add 1 mL of 33% HCl and check the pH. If pH is &lt;8.2 adjust it with NaOH and if it is &gt;8.4 adjust the pH with HCl</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Generation of MHC immunoaffinity column</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>The following steps describe the preparation of an immunoaffinity column for MHC class I pull-down.</p>
<div class="note">
<span class="note-title">Note:</span> Wash the glass column with 1% TFA before packing the column for the first time to ensure no contamination from the glass column comes into the antibody-Sepharose column or in pre-column packed with Sepharose beads only.</div>
<ol>
<li>Transfer the required amount of protein A Sepharose to a 15 mL Falcon tube and wash the beads three times with 10 mL PBS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Conjugate 10–20 mg of W6/32 antibody per mL of protein A Sepharose beads.</div>
<ol start="2">
<li>Dilute the antibody solution to a concentration of 0.5–2 mg/mL in PBS.
<ol type="a">
<li>Retain an aliquot of 5 μL of antibody solution and label it as pre-mAb.</li>
</ol>
</li>
<li>Add antibody to protein A Sepharose beads. Rotate gently end-over-end at 20°C–25°C for 1 h to allow binding.</li>
<li>Transfer the antibody-bound Sepharose beads to a glass column.</li>
<li>Collect the flow through in a fresh tube.
<ol type="a">
<li>Retain 5 μL of this flow through and label as post-mAb.</li>
</ol>
</li>
<li>Wash antibody-bound Sepharose beads with 10 c.v. of borate wash buffer at 20°C–25°C.</li>
<li>Wash antibody-bound Sepharose beads with 10 c.v. of 0.2 M triethanolamine, pH 8.2, at 20°C–25°C.</li>
<li>Add 4–5 c.v. of freshly prepared 40 mM DMP cross-linker to the column and let it pass through until only a small meniscus is left at the top of the Sepharose beads. Allow the reaction to proceed at 20°C–25°C for 1 h.</li>
<li>Wash the antibody-bound Sepharose beads with 10 c.v. of ice-cold 0.2 M Tris-HCl, pH 8.0, at 20°C–25°C.</li>
<li>Wash the antibody-bound Sepharose beads with 10 c.v. of 0.1 M citrate buffer, pH 3. at 20°C–25°C.</li>
<li>Wash the antibody-bound Sepharose beads with 10 c.v. of PBS, at 20°C–25°C.</li>
<li>Run an SDS-PAGE for the samples aliquoted at steps 2 and 5. Both the heavy chain and light chain bands should be present in the pre-mAb sample and absent in post-mAb.</li>
</ol>
<p><a href="#sec6.1">Troubleshooting 1</a></p>
<div class="critical">
<span class="critical-title">Critical:</span> Maintaining the pH of triethanolamine at 8.2 is important.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The antibody column can be stored at 4°C for 2–3 months in the presence of 0.02% sodium azide/PBS</div>
<h3 id="sec3.2">Preparation of Sepharose protein A column for pre-clearing of cell lysate</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>This section describes the steps to make a column for pre-clearing the lysate before IP.</p>
<ol start="13">
<li>Transfer 1 mL protein A Sepharose beads to glass column.</li>
<li>Wash the column with 10 c.v. of PBS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The pre-column can be used 10 times or till the column gets blocked.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> This column can be stored at 4°C for 2–3 months in the presence of 0.02% sodium azide/PBS.</div>
<h3 id="sec3.3">Preparation of cell lysate</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>This section describes steps involved in preparation of cell lysate.</p>
<ol start="15">
<li>Add 2 mL cell lysis buffer (<a href="#tbl1">Table 1</a>) for 100 M cells and let it thaw on ice.
<table id="tbl1">
<caption>Table 1. Composition of cell lysis buffer</caption>
<thead>
<tr>
<th>Cell lysis buffer (50 mL)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>500 mM Tris-HCl buffer, pH 8</td>
<td>50 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>NP-40</td>
<td>1%</td>
<td>500 μL</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.25%</td>
<td>125 mg</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>0.88 mg</td>
</tr>
<tr>
<td>IAA</td>
<td>20 mM</td>
<td>185 mg</td>
</tr>
<tr>
<td>cOmplete Protease Inhibitor Cocktail</td>
<td>-</td>
<td>1 tablet</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Homogenize the sample using 18G × 38mm needle. Repeat this 4–5 times in 15 min intervals.</li>
<li>Centrifuge the lysate at 2,000 × <i>g</i> at 4°C for 10 min.</li>
<li>Transfer the supernatant to a fresh tube and centrifuge it at 20,000 × <i>g</i> for 60 min at 4°C.</li>
<li>Pass the supernatant obtained in step 18 through a 0.45 μm filter to remove any debris and aliquot 30 μg equivalent cell lysate in a microcentrifuge tube and label it as pre-IP.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Lysis of cells should be performed at 4°C. Do not store the cell lysate for more than 12 h as proteins tend to aggregate. Best practice is to lyse the cells and immediately carry out immunoprecipitation</div>
<div class="optional">
<span class="optional-title">Optional:</span> Combine the pellets obtained in steps 17 and 18. Add 2% SDS/50 mM TEABC, pH 8.0 to lyse the membrane bound organelles obtained. Sonicate and centrifuge it at 20,000 × <i>g</i> for 30 min at 4°C. Perform western blot on the lysate obtained after adding 2% SDS/50 mM TEABC and at step 19, using anti-MHC class I antibody. MHC class I band corresponding to 40 kDa should be in the lysate from step 19 while it should be absent in the cell lysate obtained with SDS.</div>
<h3 id="sec3.4">Immunoprecipitation of MHC class-I complex</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>This section describes steps for carrying out immunoprecipitation of MHC class I complex from cell lysate</p>
<div class="note">
<span class="note-title">Note:</span> Same approach can be used for carrying out immunoprecipitation of MHC class I complex from tissue lysate</div>
<ol start="20">
<li>Pass 10 c.v. of PBS through protein A Sepharose pre-column to remove sodium azide.</li>
<li>Pass cell lysate obtained in step 19 through the pre-column.</li>
<li>Collect flow through in a fresh tube.</li>
<li>Pass 10 c.v. of PBS through the antibody column to remove sodium azide.</li>
<li>Pass 5 c.v. of cell lysis buffer (without detergent) through the antibody column.</li>
<li>Pass the pre-cleared cell lysate obtained in step 22 through the antibody column and collect flow through in a fresh tube.</li>
<li>Pass flow through obtained in step 25 through the antibody column.
<ol type="a">
<li>Collect flow through in a fresh tube.</li>
<li>Aliquot 30 μg of flow through to a microcentrifuge tube and label it as post-IP.</li>
</ol>
</li>
<li>Wash antibody column with 10 c.v. of cell lysis buffer (without detergent).</li>
<li>Wash antibody column with 10 c.v. of 50 mM Tris, pH 8.0, and 150 mM NaCl.</li>
<li>Wash antibody column with 10 c.v. of 50 mM Tris, pH 8.0, 450 mM NaCl.</li>
<li>Wash antibody column with 10 c.v. of 50 mM Tris, pH 8.0, and 150 mM NaCl.</li>
<li>Wash antibody column with 10 c.v. of 50 mM Tris, pH 8.0.</li>
<li>Elute the proteins bound on antibody column with 10% (v/v) acetic acid in 10 fractions of 500 μL each.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Regenerate the column by passing 10 c.v. of 100 mM Tris-HCl, pH 8.0 and then 10 c.v. of PBS. The pH of liquid flowing out from the column should be more than 7.</div>
<ol start="33">
<li>Pool all fractions collected in step 32.</li>
</ol>
<p><a href="#sec6.3">Troubleshooting 2</a></p>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The samples can be stored for a couple of days without much loss.</div>
<div class="note">
<span class="note-title">Note:</span> Clear the protein from pre-column by washing the column with 5 c.v. of 10% acetic acid and regenerate by passing 10 c.v. of 100 mM Tris-HCl, pH 8.0 followed by 10 c.v. of PBS. The pH of liquid flowing out from the column should be more than 7. If reusing pre-column, ensure no protein is left on the column by running later fractions of acid wash on SDS-PAGE. Flow through can be run on SDS-PAGE to ensure no protein is bound to pre-column to avoid cross contamination.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Perform western blot for the samples labeled pre- and post-IP using anti-HLA-A/B/C antibody. 40 kDa band should be detectable in pre-IP sample and absent in post-IP sample</div>
<h3 id="sec3.5">Purification of peptides from MHC- class I complex by reverse phase liquid chromatography</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 h</div>
<p>This section describes the steps and parameters for purifying peptides from MHC complex.</p>
<ol start="34">
<li>Set the flow rate of HPLC system to 2 mL/min (<a href="#tbl2">Table 2</a>).
<table id="tbl2">
<caption>Table 2. Solvent A and B for separation of peptides from MHC-I complex (for step 34)</caption>
<thead>
<tr>
<th>HPLC solvents</th>
<th>LC-MS grade water (mL)</th>
<th>LC-MS grade ACN (mL)</th>
<th>TFA (mL)</th>
<th>Total (L)</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>Solvent A</b></td>
<td>999</td>
<td>0</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td><b>Solvent B</b></td>
<td>199</td>
<td>800</td>
<td>1</td>
<td>1</td>
</tr>
</tbody>
</table>
</li>
<li>Equilibrate the column with 2% Solvent B.</li>
<li>Add trifluoroacetic acid (TFA) to the sample obtained at step 33 such that the final concentration of TFA is 1%.</li>
<li>Transfer the eluate to auto sampler vial.</li>
<li>Resolve peptides using the following LC gradient:</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Time (min)</th>
<th>Flow (mL/min)</th>
<th>% A</th>
<th>% B</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.01</td>
<td>2</td>
<td>98</td>
<td>2</td>
</tr>
<tr>
<td>0.5</td>
<td>2</td>
<td>85</td>
<td>15</td>
</tr>
<tr>
<td>4.5</td>
<td>2</td>
<td>70</td>
<td>30</td>
</tr>
<tr>
<td>12.5</td>
<td>2</td>
<td>60</td>
<td>40</td>
</tr>
<tr>
<td>22.5</td>
<td>2</td>
<td>55</td>
<td>45</td>
</tr>
<tr>
<td>24.5</td>
<td>2</td>
<td>0</td>
<td>100</td>
</tr>
<tr>
<td>26.5</td>
<td>2</td>
<td>0</td>
<td>100</td>
</tr>
<tr>
<td>27</td>
<td>2</td>
<td>98</td>
<td>2</td>
</tr>
<tr>
<td>33</td>
<td>2</td>
<td>98</td>
<td>2</td>
</tr>
</tbody>
</table>
<ol start="39">
<li>Collect 500 μL fractions in micro centrifuge tubes.</li>
<li>Concatenate the fractions into 6 samples by combining fractions eluted at different hydrophobicity range (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/517-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Separation of peptides from proteins using high performance liquid chromatography</div>
</figcaption>
</figure>
</li>
<li>Concentrate the samples using speedvac.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use protein low bind tubes to concentrate samples. Do not dry the samples completely. Concentrate the samples to 5–10 μL</div>
<p><a href="#sec6.5">Troubleshooting 3</a></p>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Samples can be stored after this step at −80°C</div>
<h3 id="sec3.6">Mass spectrometry analysis of peptides eluted from MHC complex</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This section describes the parameters to be used to acquire data on a Thermo Scientific Orbitrap Fusion mass spectrometer paired with a Waters nanoAcquity UHPLC.</p>
<ol start="42">
<li>Centrifuge the samples at 15,000 × <i>g</i> for 15 min.</li>
<li>Transfer the samples to total recovery glass vials or 96-well plate compatible with the LC connected to mass spectrometer.</li>
<li>Set nanoLC flow rate to 300 nL/min (<a href="#tbl3">Table 3</a>).
<table id="tbl3">
<caption>Table 3. Solvent A and B for nano-flow LC for mass spec analysis (for step 44)</caption>
<thead>
<tr>
<th>LC-MS solvents</th>
<th>LC-MS grade water (mL)</th>
<th>LC-MS grade ACN (mL)</th>
<th>Formic acid (mL)</th>
<th>Total (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>Solvent A</b></td>
<td>200</td>
<td>0</td>
<td>0.2</td>
<td>200</td>
</tr>
<tr>
<td><b>Solvent B</b></td>
<td>0</td>
<td>200</td>
<td>0.2</td>
<td>200</td>
</tr>
</tbody>
</table>
</li>
<li>Load the peptides on trap column at 5 μL /min using 5% solvent B.</li>
<li>Resolve peptides on analytical column using the following gradient:</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th>Time (min)</th>
<th>Flow (μL /min)</th>
<th>% A</th>
<th>% B</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial</td>
<td>0.300</td>
<td>92</td>
<td>8</td>
</tr>
<tr>
<td>44</td>
<td>0.300</td>
<td>70</td>
<td>30</td>
</tr>
<tr>
<td>49</td>
<td>0.300</td>
<td>30</td>
<td>70</td>
</tr>
<tr>
<td>54</td>
<td>0.300</td>
<td>30</td>
<td>70</td>
</tr>
<tr>
<td>55</td>
<td>0.300</td>
<td>95</td>
<td>5</td>
</tr>
<tr>
<td>60</td>
<td>0.300</td>
<td>95</td>
<td>5</td>
</tr>
</tbody>
</table>
<ol start="47">
<li>Acquire mass spectrometry data using following parameters:
<ol type="a">
<li>MS1: Resolution – 60,000; Mass Range – 350–1,800 <i>m/z</i>; AGC Target – 1e<sup>6</sup>, Maximum injection time 50 ms.</li>
<li>Precursor selection: Top-speed mode with 2 s cycle time (for top N mode, set N = 10), Include charge state – 1–4; Dynamic exclusion – 5 s</li>
<li>MS2: Isolation window – 1.2 <i>m/z</i>; Activation type – HCD, Collision energy mode – Stepped; HCD Collision Energy – 25,27,30; Orbitrap resolution – 15,000; AGC target – 2e<sup>5</sup>, Maximum injection time – 120 ms</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Data acquisition parameters vary based on the instrument on which data is acquired. The above parameters are for Orbitrap Fusion mass spectrometer</div>
<h3 id="sec3.7">Database searches of mass spectrometry data</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h to 2 days</div>
<p>This section describes mass spectrometry data analysis using Proteome Discoverer suite 2.2 with Sequest/Mascot as search engine</p>
<ol start="48">
<li>Select spectra based on following parameters: Lowest charge state – 1; Highest charge state – 4; Minimum precursor mass – 350 Da; maximum precursor mass – 5,000 Da</li>
</ol>
<ol start="49">
<li>Sequest/Mascot search parameters:
<ol type="a">
<li>Database: Reference protein database of relevant species.
<div class="note">
<span class="note-title">Note:</span> Add cRAP protein sequences (common Repository of Adventitious Proteins) that are commonly found as contaminant proteins in a proteomics experiment</div>
</li>
<li>Enzyme Name: No-Enzyme (Unspecific) on SequestHT, NoCleave on Mascot</li>
<li>Precursor Mass tolerance: 10 ppm</li>
<li>Fragment mass tolerance: 0.05 Da</li>
<li>Dynamic modifications: Oxidation on Methionine, Carbamidomethylation on C</li>
</ol>
</li>
</ol>
<ol start="50">
<li>1% FDR at PSM and Peptide level</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For identifying MHC peptides with post translational modifications like phosphorylation at S/T/Y, deamidation at N/Q/R can be given as variable modifications for MS/MS search.</div>
<p><a href="#sec6.7">Troubleshooting 4</a></p>
<p><a href="#sec6.9">Troubleshooting 5</a></p>
<div class="optional">
<span class="optional-title">Optional:</span> Other MS/MS search algorithms like Peaks and MaxQuant (<a href="#bib1">Cox and Mann, 2008</a>) can also be used. If carrying out mutant/non-canonical peptide analysis, include relevant protein sequences along with reference protein database in step 49a.</div>
<h3 id="sec3.8">Data analysis and interpretation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–24 h</div>
<p>This step describes downstream analysis of MHC peptides</p>
<ol start="51">
<li>Remove peptides originating from known contaminant proteins to get the list of MHC presented peptides.</li>
<li>Carry out sequence length analysis of peptides. Majority of peptides should be between 8 to 14 amino acids.</li>
<li>Motif analysis can be done using tools such as Seq2Logo 2.0 (<a href="#bib2">Thomson and Nielsen, 2012</a>), SYEPEITHI (<a href="#bib4">Schuler et al., 2007</a>)</li>
<li>NetMHCpan-4.1 (<a href="#bib3">Reynisson et al., 2020</a>) can be used to predict the binding affinity of peptides to MHC class I molecule.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For many cancer cell lines, HLA allele information can be found at Cellosaurus, ATCC website, or HLA typing can be performed for the cell lines.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Thousands of peptides can be identified by following the above protocol (number of peptides identified depends on expression level of MHC complex, efficiency of MHC pull-down, and sensitivity of mass spectrometer). Using the protocol, we identified 1,300 peptides from 25 million SK-MEL-28 cells (Table S1). Using more cells as starting material will result in identification of more MHC bound peptides.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is applicable for identifying MHC class I peptides for cell line based system and can be expanded for MHC class II peptides by selecting an appropriate antibody for carrying out immunoprecipitation of MHC class II. As MHC class II peptides are longer, data acquisition parameters need to be modified accordingly. The protocol can also be adapted for immunopeptidome studies using primary cells or tissues. However, some optimization may be required to overcome limitations of cell numbers or limited tissue availability</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Covalent cross linking of antibody to the Sepharose beads failed (see “<a href="#sec3.1">Generation of MHC immunoaffinity column</a>”).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Check the pH of 0.2 M triethanolamine. If the pH of triethanolamine is &gt;10, it could result in stripping of antibody before the coupling reaction. Also, pH 8.3 of DMP-triethanolamine is critical for the coupling reaction. The efficiency of the reaction reduces with decrease or increase of pH.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>HLA band is detectable in post IP sample (see “<a href="#sec3.4">Immunoprecipitation of MHC class-I complex</a>”).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Make a new column with fresh antibody and Sepharose beads.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The resolution of separation of peptides from proteins during offline fractionation was poor (see “<a href="#sec3.5">Purification of peptides from MHC- class I complex by reverse phase liquid chromatography</a>”).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The bRPLC method given here works best for the Waters XBridge column. If using a different column, the gradient method may have to be optimized again. Also, TFA gives better resolution than FA. It is advisable to use 0.1% TFA than 0.1% FA to acidify the solvents. Equilibrate the column with the same gradient before injecting the sample of interest.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Identification of many tryptic peptides outside the expected size range for MHC bound peptides (see “<a href="#sec3.7">Database searches of mass spectrometry data</a>”).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>It could be carryover from previous mass spectrometry runs on the same column. Flush the column with high organic to remove peptides bound to column, equilibrate, and then load MHC peptidome for mass spectrometry analysis</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Number of MHC peptides identified was considerably less than expected (see “<a href="#sec3.7">Database searches of mass spectrometry data</a>”).</p>
<h3 id="sec6.10">Potential solution</h3>
<ol>
<li>Check the expression level of MHC class I complex in the sample of interest. If MHC expression level is low, increase the number of cells.</li>
<li>Check the efficiency of IP by running a western blot of pre and post IP samples.</li>
<li>Check whether the mass spec is running at optimal condition by acquiring a quality control sample.</li>
</ol>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cox, J. and Mann, M. (2008). MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol. <i>26</i>, 1367-1372. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00092-7/sref1">View at publisher</a></p>
<p id="bib2">Thomson, M.C.F. and Nielsen, M. (2012). Seq2Logo: a method for construction and visualization of amino acid binding motifs and sequence profiles including sequence weighting, pseudo counts and two-sided representation of amino acid enrichment and depletion. Nucleic Acids Res. <i>40</i>, W281-W287. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00092-7/sref2">View at publisher</a></p>
<p id="bib3">Reynisson, B., Alvarez, B., Paul, S., Peters, B., and Nielsen, M. (2020). NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. Nucleic Acids Res. <i>48</i>, W449-W454. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00092-7/sref3">View at publisher</a></p>
<p id="bib4">Schuler, M.M., Nastke, M.D., and Stevanovikć, S. (2007). SYFPEITHI: database for searching and T-cell epitope prediction. Methods Mol. Biol. <i>409</i>, 75-93. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00092-7/sref4">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Harsha Gowda (<a href="mailto:Harsha.Gowda@qimrberghofer.edu.au">harsha.gowda@qimrberghofer.edu.au</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents</p>
<h4>Data and Code Availability</h4>
<p>Raw and processed mass spectrometry data presented in this protocol is available on Mendeley data (<a href="https://data.mendeley.com">https://data.mendeley.com</a>) with <a href="https://doi.org/10.17632/4bk7w8298s.1">https://doi.org/10.17632/4bk7w8298s.1</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank the QIMR Berghofer Medical Research Institute for funding. H.G. is supported by a <a href="https://doi.org/10.13039/501100000925">National Health and Medical Research Council</a> (NHMRC) Career Development Fellowship. S.V.M. is supported by <a href="https://doi.org/10.13039/100013103">QIMR Berghofer Medical Research Institute</a> and <a href="https://doi.org/10.13039/501100001794">The University of Queensland</a>.</p>
<h3>Author Contributions</h3>
<p>S.V.M. optimized the protocol and conducted experiments. S.V.M. and R.Z. carried out fractionation and mass spectrometry data acquisition. S.V.M. and K.K.D. carried out data analysis. K.K.D., C.S., and H.G. supervised the study. S.V.M. wrote the first draft. K.K.D., C.S., and H.G. critically read and edited the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>