<article>
<section>
<h1 id="header">Genetic Modulation of RNA Splicing with a CRISPR-Guided Cytidine Deaminase</h1>
<p><time datetime="2019-12-20">Published: December 20, 2019</time></p>
<p>Yunqing Ma,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup> Juanjuan Yuan,<sup><a href="#aff3">3</a></sup> and Xing Chang<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Key Laboratory of Structural Biology of Zhejiang Province, School of Life Sciences, Westlake University, 18 Shilongshan Road, Hangzhou 310024, Zhejiang Province, China</p>
<p id="aff2"><sup>2</sup>Institute of Basic Medical Sciences, Westlake Institute for Advanced Study, 18 Shilongshan Road, Hangzhou 310024, Zhejiang Province, China</p>
<p id="aff3"><sup>3</sup>Institute of Nutrition and Health Sciences, Chinese Academy of Sciences, Shanghai, China</p>
<p id="fn1"><sup>4</sup>Lead Contact</p>
<p id="fn2"><sup>5</sup>Technical Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yqma@sibs.ac.cn">yqma@sibs.ac.cn</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:changxing@westlake.edu.cn">changxing@westlake.edu.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2019.100005">10.1016/j.xpro.2019.100005</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol uses lipofectamine to deliver base editors (i.e., dCas9 and AIDx fusion protein) and sgRNA expression vectors into Duchenne Muscular Dystrophy (DMD) patient-derived human induced pluripotent stem cells (hiPSCs). This protocol details mutation of the 5′ splice site of DMD exon50 with TAM (targeted AID-induced mutagenesis) followed by amplicon-based NGS library preparation for high-throughput sequencing analysis. This protocol can be generalized for base editing in other hIPSCs and for correcting aberrant splicing associated with other genetic diseases.</p>
<p>For complete information on the generation and use of this protocol, please refer to <a href="#bib2">Yuan et al. (2018)</a>.</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>TAM-expressing vectors can be delivered into hiPSCs by lipofectamine transfection.</li>
<li>Cardiomyocytes from patients’ iPSCs can be used to mimic Dystrophic cardiomyopathy.</li>
<li>TAM is applicable to base editing in hIPSCs.</li>
<li>TAM can be used for correcting aberrant splicing associated with genetic disease.</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/17-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<ol>
<li>Design appropriate targeting sgRNAs; we recommend using <a href="https://zlab.bio/guide-design-resources">https://zlab.bio/guide-design-resources</a>. PAM sequences should be within 15 bps to 20 bps 3′ end of the splice sites.</li>
<li>Verify sgRNAs’ efficacies in inducing desired mutations in HEK293T cells (<a href="#bib1">Ma et al., 2016</a>).</li>
</ol>
<ol type="a">
<li>We typically will try 2–3 sgRNAs in 293T cells.</li>
<li>Co-transfect 293T with plasmids expressing base editors and sgRNAs. Select the transfected cells with corresponding antibiotics for 3 days.</li>
<li>Then extract the genomic DNA, PCR the region of interest and send the PCR product for Sanger sequencing. Compare the efficacies with Sanger sequencing and select the most efficient one for experiments in hiPSCs (<a href="#bib2">Yuan et al., 2018</a>).</li>
</ol>
<ol start="3">
<li>Clone sgRNA into expressing vectors containing antibiotic-resistance open reading frames. We highly recommend including sgRNA and base editors or nuclease in a single expression vector because of the low transfection efficiency of hiPSCs.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend puromycin for the selection because hiPSCs are sensitive to puromycin.</div>
<ol start="4">
<li>The sequence information of the vector we used in our study can be found here.</li>
<li>Prepare necessary buffers and primers listed in the <a href="#krt">Key Resources Table</a>.</li>
<li>Design amplicon primers for sequencing: The overhang adaptor sequence must be added to the locus-specific primer for the region to be targeted. The Illumina overhang adaptor sequences are</li>
</ol>
<p>Forward overhang: 5′TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG - [locus specific sequence]</p>
<p>Reverse overhang: 5′GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG - [locus specific sequence]</p>
<p>Suggestions for designing locus-specific primers:</p>
<ol type="a">
<li>Illumina recommends that amplicon sequenced with paired-end reads should have at least ∼50 bp of overlapping sequence in the middle.</li>
<li>The locus-specific portion of primer must have a melting temperature (Tm) of 60°–65°C.</li>
<li>Illumina recommends ordering oligo primer sets with standard desalting purification.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>Phusion High-Fidelity DNA Polymerase</td>
<td>NEB</td>
<td>M530S</td>
</tr>
<tr>
<td>Lipofectamine™ LTX Reagent with PLUS™ Reagent</td>
<td>Invitrogen™</td>
<td>15338100</td>
</tr>
<tr>
<td>Agencourt AMPure XP</td>
<td>Beckman Coulter</td>
<td>A63881</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>MicroElute Genomic DNA Kit</td>
<td>Omega</td>
<td>D3096-01</td>
</tr>
<tr>
<td colspan="3">Experimental Models: Cell Line</td>
</tr>
<tr>
<td>DMD iPS</td>
<td>The Third Affiliated Hospital of Guangzhou Medical University, Guangzhou, Guangdong, China</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>DMD exon50-F</td>
<td>CTGAGTGGAAGGCGGTAAAC</td>
<td>N/A</td>
</tr>
<tr>
<td>DMD exon50-R</td>
<td>TCTCACCCAGTCATCACTTCA</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Lenti-V2-Ugi-nCas9-AIDx</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Lenti-V2-AIDx-nSaCas9 (KKH)-Ugi</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pcDNA3 Ugi</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>Primer3</td>
<td>Whitehead Institute</td>
<td><a href="http://bioinfo.ut.ee/primer3-0.4.0/">http://bioinfo.ut.ee/primer3-0.4.0/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<p>Please see the <a href="#krt">Key Resources Table</a> for the most important resources (material and sequences) to have before starting.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> We used a BioRad Tetrad for PCR, but an equivalent PCR machine can be used.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Nuclease/sgRNA Delivery Method</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>The purpose of this step is to deliver base editors and sgRNA vectors into hiPSCs. The transfected cells are subsequently selected with antibiotics (puromycin in this protocol).</p>
<h4>A. Plating Cells</h4>
<ol>
<li>For feeder-free culture, hiPSCs are maintained on matrigel in mTeSR1 medium. The day before transfection, hiPSCs should reach 60%–80% confluency.</li>
<li>Remove the 6-well plate from the incubator and aspirate the mTeSR1 medium. Add 1 mL of PBS and aspirate it.</li>
<li>Next, add 0.5 mL of (pre-warmed) Accutase (3 times dilution with PBS, final concentration, 16.7 ug/mL) to the well, and treat it for 6 min in a 37°C incubator for detachment.</li>
<li>When each colony is ∼80%–90% detached, dissociate the colonies into single cells by resuspending them with 2 mL mTeSR1.</li>
<li>Pipette the mixture up and down 6–8 times and centrifuge it at 1500 rpm (300 g) for 3 min at 25 °C. At the same time, count the number of cells in a 10 μL volume using a hemocytometer.</li>
<li>Aspirate the supernatant and resuspend the cells in mTeSR1 at a final density equal to 1*10<sup>4</sup> cells/ul.</li>
<li>Plate 4 × 10<sup>5</sup> cells in 6-well plate coated with matrigel by 2 mL mTeSR1 medium supplemented with 10 μM ROCK inhibitor to each well (Y-27632; STEMCELL Technologies).</li>
</ol>
<h4>B. Transfection of Base Editors and sgRNA-Expressing Vectors</h4>
<div class="critical">
<span class="critical-title">Critical:</span> Transfection should be conducted within 24 hours after plating to ensure the cells are actively dividing.</div>
<ol>
<li>Recover lipofectamine and opti-MEM medium to 25°C.</li>
<li>Cell density should reach 60% confluency on the day after plating. Change the medium to fresh mTeSR1.</li>
<li>Dilute 12 ul Lipofectamine LTX Reagent in 125 ul Opti-MEM Medium and mix well with a pipette.</li>
<li>Dilute Lenti-V2-AIDx-nSaCas9 (KKH)-Ugi (2.5 ug) and pCDNA3 Ugi (0.5 ug) in 125 ul Opti-MEM Medium, then add 2.5 ul PLUS Reagent and mix well with a pipette.</li>
<li>Add diluted DNA to diluted Lipofectamine LTX Reagent (1:1 ratio) and mix well with a pipette.</li>
<li>Incubate for 5 min at 25°C.</li>
<li>Add DNA-lipid complex dropwise and evenly to cells.</li>
<li>Incubate cells for 2 days at 37°C.</li>
</ol>
<h4>C. Antibiotic Selection</h4>
<div class="critical">
<span class="critical-title">Critical:</span> To determine the minimum antibiotic concentration to select transfected cells over the course of one week, a killing curve of untransfected cells is highly recommended with every selection antibiotic when used for the first time or used on a new cell line (Steps 1-3).</div>
<ol>
<li>Plate 1*10<sup>5</sup> cells in 0.5 mL complete mTeSR1 medium with 10 μM ROCK inhibitor per well in a 24-well tissue culture plate coated with matrigel one day prior to antibiotic selection. Ideally cells should reach ∼60% confluence prior to the antibiotic selection.</li>
<li>Add increasing amounts of the antibiotic (e.g., 0, 0.25, 1, 2, 3, 4, 5, 6, 7, 8, 9, and 10 μg/mL puromycin) to duplicate wells of cells plated in complete mTeSR1 media.</li>
<li>Observe the cells under a light microscope until cells die out within 7 days. The duration of culture may be extended to 14 days depending on the cell line used.</li>
<li>2 days after transfection, change the medium to fresh mTeSR1 supplemented with 2 μg/mL puromycin and 10 μM ROCK inhibitor.</li>
<li>Select for 3 days and then change the medium every day.</li>
<li>5–10 days after the transfection, collect half of the cells for analysis of induced mutations and keep the remaining cells for single-cell clone isolation and downstream function test.</li>
</ol>
<h4>D. Single-Cell Clone Isolation</h4>
<ol>
<li>Plate single cell suspension at low densities (1x10<sup>4</sup>-3x10<sup>4</sup> cells per 10cm dish in 6 mL mTeSR1 plus 10 mM ROCK inhibitor). Change the culture medium every day.</li>
<li>Ten days later, single cells should grow into small cell masses which are detached to each other.</li>
<li>Aspirate the supernatant and add 15 mL PBS.</li>
<li>Incubate for 10 min at 37°C to detach the cells gently.</li>
<li>Transfer the individual cell mass gently into a well of 48-well plates under a light microscope.</li>
</ol>
<h4>E. Genomic DNA Extraction</h4>
<div class="note">
<span class="note-title">Note:</span> Genomic DNA is extracted with MicroElute Genomic DNA Kit (Omega) following the manufacturer’s instructions.</div>
<div class="note">
<span class="note-title">Note:</span> DNA Wash Buffer Concentrate must be diluted with absolute ethanol before use.</div>
<div class="note">
<span class="note-title">Note:</span> Work at 25 °C unless stated otherwise.</div>
<ol>
<li>Re-suspend the sample cell pellet (no less than 1*10<sup>5</sup> to avoid low yield) volume with 100 ul with PBS Buffer.</li>
<li>Add 20 ul of Protease solution (included in the kit) and mix well by vortexing.</li>
<li>Add 120 ul Buffer BL and vortex to mix. Incubate at 70°C for 10 min. If there are less than 1x10<sup>5</sup> cells, 4 ul of Linear acrylamide is recommended to be added to each sample (Linear Polyacrylamide is an efficient neutral carrier for precipitating picogram amounts of nucleic acids with ethanol).</li>
<li>Add 120 ul absolute ethanol and mix thoroughly by vortexing for 15 s at maxi speed. Bring down any liquid drop from inside of lid by brief centrifugation.</li>
<li>Assemble a HiBind® MicroElute column in a 2 mL collection tube (provided with the kit).</li>
<li>Transfer the entire solution from Step 4 into the column, including any precipitate that may have formed. Close the lid and centrifuge at 8,000 x g for 1 min at 25°C to bind DNA. Discard the collection tube and flow-through liquid.</li>
<li>Place the column into a new collection tube (supplied). Add 500 ul of Buffer HB in the column. Close the lid and centrifuge at 8000 x g for 1 min at 25°C. Discard the flow-through and re-use the collection tube.</li>
<li>Place the column into the same 2 mL tube (supplied) and wash by pipetting 650 ul of DNA Wash Buffer diluted with ethanol. Centrifuge at 8,000 x g for 1 min at 25°C. Again, dispose of the collection tube and flow-through liquid.</li>
<li>Using a new collection tube, wash the column with a second 650 ul of DNA Wash Buffer diluted with ethanol and centrifuge as above. Discard flow-through and re-use the collection tube.</li>
<li>Using the same 2 mL collection tube, centrifuge empty column at maximum speed (≥20,000 x g) for 3 min to dry the HiBind® membrane.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This step is crucial for ensuring optimal elution in the following step.</div>
<ol start="11">
<li>Place the column into a sterile 1.5 mL microfuge tube and add 10-50 ul of preheated (70°C) Elution Buffer or water onto the center of the membrane. Allow tubes to sit for 3 min at 25°C.</li>
<li>To elute DNA from the column, centrifuge at 20,000 x g for 1 min.</li>
<li>The DNA concentration is measured with a NanoDrop. We typically get ∼25 ug gDNA from 1*10<sup>6</sup> cells.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Incubation at 70°C rather than 25 °C will give a modest increase in DNA yield per elution.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> the second elution may be performed using the first eluate or using the second 10-50 ul of preheated Elution Buffer or water.</div>
<h3>NGS Library Preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>The goal of this step is to quantitively analyze the editing efficiencies using high-throughput sequencing.</p>
<h4>A. Primary PCR Amplification</h4>
<div class="note">
<span class="note-title">Note:</span> See <a href="#before-you-begin">Before you Begin</a> section for notes on primer design.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Work on ice unless stated otherwise.</div>
<ol>
<li>If desired, normalize samples to a consistent concentration (concentration (100 ng/ul) in a sterile 1.5 mL microfuge tube. Always include DNA from un-edited cells as controls.</li>
<li>Vortex samples to mix and spin-down in a centrifuge.</li>
<li>Thaw the Phusion® High-Fidelity DNA Polymerase kit reagents.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Remember to vortex and centrifuge the plate(s) and reagents (when thawed) before using them.</div>
<ol start="4">
<li>Make a 1 x Phusion® PCR master mix using the following recipe (see below):</li>
</ol>
<p>10 μL 5x HF buffer</p>
<p>4 μL 2.5 mM dNTPs</p>
<p>0.5 μL Phusion® High-Fidelity DNA Polymerase</p>
<p>2.5 μL forward primer (dilute stock(100uM) to 10 uM)</p>
<p>2.5 μL reverse primer (dilute stock(100uM) to 10 uM).</p>
<p>29.5 μL Nuclease-free water</p>
<ol start="5">
<li>Add 1 μL DNA template (100 ng/ul) to the PCR mix, mix and spin down.</li>
<li>Run the following PCR protocol on the PCR machine. Typically, this should be run for 25 cycles and stopped at the end of the final extension phase.</li>
</ol>
<p>Cycling Conditions:</p>
<p>98°C – 30 s</p>
<p>25-35 cycles of:</p>
<p>98°C – 10 s</p>
<p>55°C – 20 s</p>
<p>72°C – 10 min</p>
<p>Hold at 4°C</p>
<ol start="7">
<li>The PCR product can be Sanger sequenced to determine if the desired mutations occur in the edited cells.</li>
<li>Run 5 μL of PCR product on the agarose gel to confirm the size.</li>
</ol>
<h4>B. First PCR Clean Up</h4>
<ol>
<li>Bring the AMPure XP beads to 25°C. The AMPure XP beads are paramagnetic beads to selectively bind nucleic acids by type and size which are powered by patented Solid Phase Reversible Immobilization (SPRI) technology.</li>
<li>Vortex the AMPure XP beads for 30 s to make sure that the beads are evenly dispersed. Add an equal volume of beads with the PCR product to the 1.5 mL microfuge tube.</li>
<li>Gently pipette entire volume up and down 10 times.</li>
<li>Incubate at 21°C without shaking for 5 min.</li>
<li>Place 1.5 mL microfuge tube on a magnetic stand for 2 min or until the supernatant has cleared.</li>
<li>With 1.5 mL microfuge tube on the magnetic stand, use a pipette to remove and discard the supernatant. Change tips between samples.</li>
<li>With 1.5 mL microfuge tube on the magnetic stand, wash the beads with freshly prepared 80% ethanol. Incubate the tube on the magnetic stand for 30 s. Carefully remove and discard the supernatant.</li>
<li>With 1.5 mL microfuge tube on the magnetic stand, perform a second ethanol wash.</li>
<li>With 1.5 mL microfuge tube still on the magnetic stand, allow the beads to air-dry for 10 min.</li>
<li>Remove 1.5 mL microfuge tube from the magnetic stand. add 25 μL of ddH2O to each well of the tube.</li>
<li>Gently mix with pipette up and down 10 times, changing tips after each tube.</li>
<li>Incubate at 25°C for 2 min.</li>
<li>Place 1.5 mL microfuge tube on the magnetic stand for 2 min or until the supernatant has cleared. Carefully transfer 20 μL of the supernatant from 1.5 mL microfuge tube to a new one.</li>
</ol>
<h4>C. Indexing PCR</h4>
<ol>
<li>Determine an i5 and i7 dual-indexing scheme and ensure no index conflicts between samples to be pooled for sequencing (e.g., N701 and N702). The HiSeq and MiSeq Systems use a green laser to sequence G/T and a red laser to sequence A/C. At each cycle at least 1 of 2 nucleotides for each color channel must be read to ensure proper registration.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to maintain color balance for each base of the Index Read being sequenced, otherwise, Index Read sequencing may fail due to registration failure of clusters.</div>
<ol start="2">
<li>Thaw the Phusion® High-Fidelity DNA Polymerase (New England Biolab, MA) kit reagents.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Remember to vortex and centrifuge reagents (when thawed) before using them.</div>
<ol start="3">
<li>Make a 1 x Phusion® PCR master mix using the following recipe:</li>
</ol>
<p>10 μL 5x HF buffer</p>
<p>4 μL 2.5 mM dNTPs</p>
<p>0.5 μL Phusion® High-Fidelity DNA Polymerase</p>
<p>2.5 μL forward primer (dilute stock(100uM) to 10 uM)</p>
<p>2.5 μL reverse primer (dilute stock(100uM) to 10 uM).</p>
<p>25.5 μL Nuclease-free water</p>
<p>Add 5 μL template to the PCR mix.</p>
<ol start="4">
<li>Run the following PCR protocol on the BioRad Tetrad (or equivalent PCR machine). Typically, this should be run for 5-8 cycles and stopped at the end of the final extension phase.</li>
</ol>
<p>Cycling Conditions:</p>
<p>98°C – 30 s</p>
<p>5 cycles of 98°C – 10 s</p>
<p>55°C – 20 s</p>
<p>72°C – 10 min</p>
<p>Hold at 4°C</p>
<div class="note">
<span class="note-title">Note:</span> Total PCR cycles should be limited to 33 cycles.</div>
<ol start="5">
<li>Run 5 μL of PCR product on the agarose gel to check product size.</li>
</ol>
<h4>D. Second PCR Clean Up</h4>
<ol>
<li>Repeat steps B.1-B.13 above.</li>
<li>The library can be analyzed for quality, quantification, and sent to high-throughput sequencing. A good quality library should only contain desired PCR products without primer dimers. The 170 bp primer dimers indicated as below should be removed and the size of the amplicon should be consistent with what is expected (<a href="#fig1">Figure 1</a>).</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/17-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. An Example of DNA Analysis during the NGS Workflow using the High-Sensitivity D1000 ScreenTape</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem</h3>
<p>DNA samples couldn’t be amplified.</p>
<h3>Potential Solution</h3>
<p>Samples can be amplified with increased cycle numbers but no more than 35 cycles are suggested to avoid non-specific amplification. In addition, each pair of primers should be tested for suitable Tm temperatures.</p>
<h3>Problem</h3>
<p>Non-specific amplification.</p>
<h3>Potential Solution</h3>
<p>We recommend including a water blank to exclude potential contamination in reagents.</p>
<h3>Problem</h3>
<p>Lower than expected editing efficiencies.</p>
<h3>Potential Solution</h3>
<p>Further isolate single cell clones from iPSCs (steps D) and re-select corrected clones for analysis.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The sequence context of splice sites and availability of PAM may limit the application of this approach. By inducing efficient G &gt; A conversions in a window of 12- to 20-bp upstream of the PAM sequence, the current strategy is estimated to target over 80% of all human exons. With expansion and evolution of Cas9 proteins with fewer PAM restrictions, we believe most of the exons will be targetable using a similar concept.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>In our experience, the efficiencies as well as accuracies of using this protocol for genomic editing in hiPSCs, are very similar (or even better) to what one could achieve using the same editing scheme in 293T cells with &gt; 50% intended G&gt;A conversion. If the editing efficiencies do not reach expectations, it will be helpful to further isolate single-cell clones from iPSCs and select corrected clones for downstream analysis.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ma, Y., Zhang, J., Yin, W., Zhang, Z., Song, Y., and Chang, X. (2016). Targeted AID-mediated mutagenesis (TAM) enables efficient genomic diversification in mammalian cells. Nat. Methods <i>13</i>, 1029-1035.</p>
<p id="bib2">Yuan, J., Ma, Y., Huang, T., Chen, Y., Peng, Y., Li, B., Li, J., Zhang, Y., Song, B., Sun, X., et al. (2018). Genetic Modulation of RNA Splicing with a CRISPR-Guided Cytidine Deaminase. Mol. Cell <i>72</i>, 380-394.e7.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>This work was supported by 2018YFA0107500 and 2018YFA0801401 from <a href="https://doi.org/10.13039/501100002855">MOST</a> (X.C.) and by 31370858 and 81671552 from <a href="https://doi.org/10.13039/501100001809">NSFC</a> (X.C.).</p>
<h3>Author Contributions</h3>
<p>X.C. and Y.M. wrote the manuscript, and J.Y. discussed the contents of the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare that they have no competing interests.</p>
</section>
</article>