<article>
<section>
<h1 id="header">IFI16 knockdown in primary HIV-1 target cells</h1>
<p><time datetime="2021-01-08">Published: January 8, 2021</time></p>
<p>Matteo Bosso,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Caterina Prelli Bozzo,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Meta Volcic,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Frank Kirchhoff<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Molecular Virology, Ulm University Medical Center, 89081 Ulm, Germany</p>
<p id="fn1"><sup>2</sup>These authors contributed equally</p>
<p id="fn2"><sup>3</sup>Technical Contact</p>
<p id="fn3"><sup>4</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:matteo.bosso@uni-ulm.de">matteo.bosso@uni-ulm.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:frank.kirchhoff@uni-ulm.de">frank.kirchhoff@uni-ulm.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100236">10.1016/j.xpro.2020.100236</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>IFI16 is an important player of the host intrinsic immune response. Among others, it has been reported to sense intermediate products of HIV-1 reverse transcription in the cytosol and to sequester the transcription factor Sp1 in the nucleus to attenuate viral gene expression. Here, we present three different methods to reduce IFI16 protein expression levels in HIV-1 primary target cells. These techniques can be adapted for the investigation of other cellular factors in primary macrophages and CD4<sup>+</sup> T lymphocytes.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Hotter et al. (2019)</a> and <a href="#bib1">Bosso et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>IFI16 knockdown by siRNA achieves about 50% efficiency in primary macrophages</li>
<li>shRNA-mediated IFI16 knockdown in lymphocytes reduces protein levels by about 40%</li>
<li>Nucleofection of gRNA/Cas9 reduces IFI16 expression in lymphocytes with high efficacy</li>
<li>Knockdown of IFI16 by gRNA/Cas9 nucleofection did not cause off-target effects</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/360-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Isolation of peripheral blood mononucleate cells (PBMCs) from buffy coats</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol>
<li>Decide how many macrophages you will need for the experiment and adapt the volume of buffy coat you prepare, accordingly. Usually, 12.5∗10<sup>6</sup> cells are sufficient to perform one experiment in technical duplicates. Three donors are sufficient for one complete experiment.
<ol type="a">
<li>Dilute 1:1 the buffy coat in 1× PBS (pH 7.2).</li>
<li>Prepare n 50 mL Falcons (according to the required amount of diluted buffy coat) with 15 mL Bicoll in each.</li>
<li>Carefully layer 15 mL of buffy coat/PBS dilution on top of the Bicoll.</li>
<li>Centrifuge 20 min at 1,200 rcf at 22°C–23°C (no brake, acceleration 5).</li>
<li>Collect the white blood cell ring from the tubes and transfer the cells from the tubes into new 50 mL Falcons (approximately 15 mL each).</li>
<li>Fill up with PBS to 50 mL.</li>
<li>Centrifuge 10 min at 453 rcf at 22°C–23°C (acceleration 9, brake 9).</li>
<li>Remove supernatant by pouring.</li>
<li>Resuspend the cell pellet by scratching the Falcons on the grid of the flow bench and add 10 mL EDTA (1 mM) in PBS.</li>
<li>Fill up with PBS to 50 mL.</li>
<li>Centrifuge 5 min at 453 rcf at 22°C–23°C (acceleration 9, brake 9).</li>
<li>Remove supernatant, resuspend, and add 50 mL PBS.</li>
<li>Centrifuge 5 min at 453 rcf at 22°C–23°C (acceleration 9, brake 9).</li>
<li>Discard supernatant, resuspend cell pellet, pool cells from the same donor within one tube and add 10 mL RPMI (supplemented with 10% (v/v) heat-inactivated FCS, L-glutamine (2 mM), streptomycin (100 μg/mL) and penicillin (100 U/mL)) (RPMIxxx).</li>
<li>Dilute cells 1:100 and count them.</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Differentiation of PBMCs into monocyte-derived macrophages (MDMs)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="2">
<li>Dilute cells with medium RPMIxxx supplemented with 10% (v/v) heat-inactivated human serum from AB donors and 15 ng/mL M-CSF to a final concentration of 2.5∗10<sup>6</sup> cells/mL.</li>
<li>Seed cells in a 12 well plate (NUNC) (1 mL/well).
<ol type="a">
<li>On day 3 (72 h post plating) replace the medium with MDM medium (DMEM(0xx)) + 10% (v/v) heat-inactivated human AB serum + 15 ng/mL M-CSF.</li>
</ol>
</li>
<li>Cells will be ready for siRNA transfection on day 7 after isolation.</li>
</ol>
<h3 id="sec1.3">Isolation of CD4<sup>+</sup> T lymphocytes from buffy coats</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="5">
<li>Isolation of CD4<sup>+</sup> T lymphocytes from buffy coat
<ol type="a">
<li>Bring to room temperature (22°C–25°C ) PBS + 2% (v/v) FCS and Biocoll separating solution (Biochrom).</li>
<li>Dilute buffy coats with equal amount of PBS + 2% (v/v) FCS and gently mix.</li>
<li>For each mL of blood add 50 μL of RosetteSep Human CD4<sup>+</sup> T cell Enrichment Cocktail and gently mix.</li>
<li>Incubate 20 min at 22°C–23°C.</li>
<li>Prepare density medium, 15 mL in 50 mL tube.</li>
<li>Dilute sample with equal amount of PBS + 2% (v/v) FCS and mix gently.</li>
<li>Layer 15 mL blood sample on top of the density medium.</li>
<li>Centrifuge 20 min, 1,200 rcf, acceleration 5, no breaking, 22°C–23°C.</li>
<li>Remove the enriched CD4<sup>+</sup> T cell layer (between the Plasma on top and density medium under).</li>
<li>Wash with PBS + 2% (v/v) FCS.</li>
<li>Remove supernatant and add 5 mL of ACK Lysing Buffer for 5 min to remove contaminating erythrocytes, wash with PBS + 2% (v/v) FCS.</li>
<li>Resuspend cells in 10 mL RPMIxxx 20% (v/v) FCS.</li>
<li>Count the cells.</li>
</ol>
</li>
</ol>
<h3 id="sec1.4">Stimulation of CD4<sup>+</sup> T lymphocytes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="6">
<li>Stimulate cells with IL-2 (10 ng/mL) and 25 μL of anti-CD3/CD28 magnetic beads per 1 million cells (1 bead/cell).</li>
<li>Incubate stimulated CD4<sup>+</sup> T lymphocytes in a concentration of 1 million cells/mL in RPMIxxx 20% (v/v) FCS.</li>
<li>Cultivate for 3 days at 37°C, in a 5% CO<sub>2</sub> incubator.
<ol type="a">
<li>Regularly check the cell culture under the light microscope to monitor cell growth and avoid overgrowth.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> use magnetic beads for the lymphocyte’s stimulation. They will be easy to remove after 3 days once the activation of the cells is completed.</div>
<h3 id="sec1.5">Amplification and isolation of shRNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="9">
<li>Transformation of competent <i>Escherichia coli</i> XL-2 blue
<ol type="a">
<li>Mix 5 μL of competent <i>Escherichia coli</i> XL-2 blue with 0.5 μg/μL of shRNA.</li>
<li>Incubate 20 min on ice.</li>
<li>Heat shock 42°C for 45 s.</li>
<li>Incubate on ice 2 min.</li>
<li>Add 200 μL SOC medium.</li>
<li>Incubate at 37°C with gently shaking for 30 min.</li>
<li>Plate 30 μL on agar plate with selective antibiotic.</li>
<li>Incubate for 14 h at 37°C.</li>
<li>Select one colony and inoculate liquid broth (LB) medium, incubate for 14 h at 37°C.</li>
<li>For the isolation of shRNA use the QIAGEN midi kit (Cat# 12145); it is critical to use a DNA isolation kit that gives high purity DNA!</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>IRDye 680RD goat anti-mouse IgG (H + L), 1:20,000</td>
<td>LI-COR</td>
<td>Cat# 926-68070RRID:AB_10956588</td>
</tr>
<tr>
<td>IRDye 800CW goat anti-mouse IgG (H + L), 1:20,000</td>
<td>LI-COR</td>
<td>Cat# 926-32210RRID:AB_621842</td>
</tr>
<tr>
<td>IRDye 680RD goat anti-rat IgG (H + L), 1:20,000</td>
<td>LI-COR</td>
<td>Cat# 926-68076RRID: AB_10956590</td>
</tr>
<tr>
<td>IRDye 800CW goat anti-rat IgG (H + L), 1:20,000</td>
<td>LI-COR</td>
<td>Cat# 926-32219, RRID: AB_1850025</td>
</tr>
<tr>
<td>Mouse monoclonal anti-IFI16, 1:150</td>
<td>Santa Cruz</td>
<td>Cat# sc-8023 RRID: AB_627775</td>
</tr>
<tr>
<td>Rat monoclonal anti-GAPDH, 1:1,000</td>
<td>BioLegend</td>
<td>Cat# 607902RRID: AB_2734503</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-actin, 1:5,000</td>
<td>Abcam</td>
<td>Cat# ab8226;RRID: AB_306371</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>XL2-Blue MRF′ ultracompetent cells</td>
<td>Agilent Technologies</td>
<td>Cat# 200151</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human: peripheral blood mononuclear cells</td>
<td>DRK-Blutspendedienst Baden-Württemberg-Hessen, Ulm, Germany</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Non-targeting (NT) gRNA:ACGGAGGCTAAGCGTCGCAA</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>IFI16 gRNA:GACCAGCCCTATCAAGAAAG</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>CTRL siRNA:UUCUCCGAACGUGUCACGUdTdT</td>
<td>Eurofins</td>
<td>N/A</td>
</tr>
<tr>
<td>IFI16 siRNA (1);GGUGCUGAACGCAACAGAAUCAUUUAAAUGAUUCUGUUGCGUUCAGCACC</td>
<td> </td>
<td>Cat# IFI16HSS105205</td>
</tr>
<tr>
<td>IFI16 siRNA (2):GGAUUGAUUAGAAGUGCCAGCGUAAUUACGCUGGCACUUCUAAUCAAUCC</td>
<td>Invitrogen</td>
<td>Cat# IFI16HSS105206</td>
</tr>
<tr>
<td>IFI16 siRNA (3):UUAGUUUGUGAGCAAAGCUGAUUGAUCAAUCAGCUUUGCUCACAAACUAA</td>
<td>Invitrogen</td>
<td>Cat# IFI16HSS105207</td>
</tr>
<tr>
<td>IFI16 shRNA</td>
<td>Origine</td>
<td>Cat# TR312254</td>
</tr>
<tr>
<td colspan="3">Chemicals and reagents</td>
</tr>
<tr>
<td>Amaxa 4D-Nucleofector Human Activated T cell P3 Lonza Kit</td>
<td>Lonza</td>
<td>Cat# V4XP-3032</td>
</tr>
<tr>
<td>Phosphate-buffer saline (PBS)</td>
<td>Thermo Fisher</td>
<td>Cat#14190144</td>
</tr>
<tr>
<td>ACK lysis buffer (1×)</td>
<td>Lonza</td>
<td>Cat# 10-548E</td>
</tr>
<tr>
<td>HiFi Cas9 nuclease V3</td>
<td>IDT</td>
<td>Cat# 1081061</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Pan Biotech</td>
<td>Cat# P04-80100</td>
</tr>
<tr>
<td>Dulbecco’s modified Eagle’s medium (DMEM)</td>
<td>Gibco</td>
<td>Cat# 41965039</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Gibco</td>
<td>Cat# M3942</td>
</tr>
<tr>
<td>Roswell Park Memorial Institute (RPMI)</td>
<td>Gibco</td>
<td>Cat# 22400089</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>Thermo Fisher</td>
<td>Cat# 15140122</td>
</tr>
<tr>
<td>FCS</td>
<td>Gibco</td>
<td>Cat# 10270106</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>Sigma-Aldrich</td>
<td>Cat# 444203</td>
</tr>
<tr>
<td>Recombinant human IL-2</td>
<td>MACS Miltenyi Biotec</td>
<td>Cat# 170076147</td>
</tr>
<tr>
<td>Lipofectamine RNAiMAX</td>
<td>Thermo Fisher</td>
<td>Cat# 13778075</td>
</tr>
<tr>
<td>Anti-CD3/CD28 magnetic beads</td>
<td>Life Technologies</td>
<td>Cat# 11131D</td>
</tr>
<tr>
<td>Biocoll separating solution</td>
<td>Biochrom</td>
<td>Cat# L6113</td>
</tr>
<tr>
<td>RosetteSep human CD4<sup>+</sup> T cell enrichment cocktail</td>
<td>STEMCELL Technologies</td>
<td>Cat# 15022</td>
</tr>
<tr>
<td>4× protein sample loading buffer</td>
<td>LI-COR</td>
<td>Cat# 928-40004</td>
</tr>
<tr>
<td>Complete ULTRA inhibitor cocktail tablet</td>
<td>Roche</td>
<td>Cat# 05892791001</td>
</tr>
<tr>
<td>Human M-CSF</td>
<td>R&amp;D Systems</td>
<td>Cat# 216-MC-01M</td>
</tr>
<tr>
<td>Plasmid midi kit</td>
<td>QIAGEN</td>
<td>Cat# 12145</td>
</tr>
<tr>
<td>SOC medium</td>
<td>Invitrogen/Life Technologies</td>
<td>Cat# 15544034</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Corel DRAW 2017</td>
<td>Corel Corporation</td>
<td><a href="https://www.coreldraw.com/">https://www.coreldraw.com/</a></td>
</tr>
<tr>
<td>GraphPad Prism version 8</td>
<td>GraphPad Software</td>
<td>
<a href="https://www.graphpad.com/">https://www.graphpad.com</a>RRID: SCR_002798</td>
</tr>
<tr>
<td>LI-COR Image Studio Lite version 5.0</td>
<td>LI-COR</td>
<td>
<a href="http://www.licor.com/">www.licor.com/</a>RRID: SCR_013715</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> isolation of MDMs can be performed using the RosetteSep Human Monocyte Enrichment Cocktail, following the manufacturer instructions.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">siRNA transfection of macrophages</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 11 days</div>
<p>This step explains the procedure to transfect siRNAs in MDMs.</p>
<ol>
<li>Remove medium from plated macrophages and add 500 μL fresh DMEMxxx (supplemented with 10% (v/v) heat-inactivated FCS, L-glutamine (2 mM), streptomycin (100 μg/mL) and penicillin (100 U/mL))</li>
<li>Transfect with siRNA (45 nM final molarity) using Lipofectamine RNAimax (preferentially done during late afternoon to reduce Lipofectamine-derived cellular toxicity)
<ol type="a">
<li>Prepare (scale according to the number of samples) the following master mixes
<ol type="i">
<li>Tube 1: 3 μL Lipofectamine + 75 μL OptiMEM</li>
<li>Tube 2: 1.46 μL siRNA (stock concentration: 20 μM) + 75 μL OptiMEM</li>
</ol>
</li>
<li>Add contents of tube 1 to tube 2 and mix by pipetting</li>
<li>Incubate for 10 min at 22°C–23°C</li>
<li>Drop 150 μL of the Lipofectamine-siRNA complex solution onto the cells in a dropwise manner. When all the required wells have been transfected, gently rock the plate to evenly distribute the liposome complexes.</li>
</ol>
</li>
<li>Morning of the following day: remove medium and add 750 μL fresh MDM medium (DMEM0xx + 10% (v/v) human serum + 15 ng/ mL M-CSF) (this step is roughly done at 14 h post siRNA transfection)</li>
<li>Early morning of three days post transfection, remove medium from cultured macrophages and add 500 μL fresh DMEMxxx
<ol type="a">
<li>Repeat point 2.</li>
</ol>
</li>
<li>Incubate cells for 1 week at 37°C in a 5% CO<sub>2</sub> incubator.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> always consider doing siRNA transfection in technical duplicates. siRNA master mixes have to be prepared under the flow hood. Disinfect the working area with 70% isopropanol and inactivate any residual nucleases with RNase away.</div>
<h3 id="sec3.2">Validation of knockdown efficiency</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step illustrates processing of cells for western blot analysis</p>
<ol start="6">
<li>At 1 week post transfection, harvest and lyse cells for western blot
<ol type="a">
<li>Remove supernatants.</li>
<li>Add 500 μL PBS.</li>
<li>Remove PBS and add 100 μL western blot lysis buffer (150 mM NaCl, 50 mM HEPES, 5 mM EDTA, 0.1% NP40, 500 μM Na<sub>3</sub>VO<sub>4</sub>, 500 μM NaF, pH 7.5) supplemented with Complete ULTRA protease inhibitor cocktail tablet (1 pill/10 mL) (this step should be done one plate at the time).</li>
<li>Leave the plate for 3 min on ice.</li>
<li>Carefully rinse the well and pool cell lysates from the same technical replicates into a 1.5 mL Eppendorf tube.</li>
<li>Vortex 1 s and incubate on ice for 20 min.</li>
<li>Vortex 1 s and centrifuge lysates at 20,817 rcf for 20 min at 4°C.</li>
<li>Add 90 μL of supernatants to 30 μL of loading dye (90% (v/v) 4× loading dye + 10% (v/v) β-mercaptoethanol).</li>
<li>Heat at 95°C for 5 min.</li>
<li>Freeze at −20°C.</li>
</ol>
</li>
<li>SDS-PAGE Immunoblot
<ol start="11" type="a">
<li>Load 15–20 μL of cell lysates on a NuPAGE 4%–12% Bis-Tris gel (Invitrogen) and run them for 90 min at 120 V.</li>
<li>Transfer proteins on an Immobilion-FL Transfer PVDF Membranes (Merck Millipore) with the Trans-Blot SD Semi-Dry Transfer Cell for 2 h.</li>
<li>Block the membrane with 5% milk (w/v) in PBS-T (0.02% Tween (v/v)).</li>
</ol>
</li>
</ol>
<p>Stain membranes with anti-IFI16 (1G7) mouse (stock concentration: 200 μg/mL; working dilution: 1:150) and anti-GAPDH (<a href="https://www.biolegend.com/en-us/search-results?Clone=W17079A">W17079A</a>) rat (stock concentration: 0.5 μg/μL; working dilution: 1:1,000) antibodies.</p>
<ol start="8">
<li>Quantify band intensities using Image Studio Lite Ver 5.0.</li>
</ol>
<h3 id="sec3.3">Transfection of CD4+ T lymphocytes with shRNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Nucleofection of CD4<sup>+</sup> T lymphocytes with shRNA to downregulate expression of IFI16.</p>
<ol start="9">
<li>Remove the anti-CD3/CD28 magnetic beads from CD4<sup>+</sup> T lymphocytes
<ol type="a">
<li>Resuspend cells thoroughly to separate cells from the beads.
<ol type="i">
<li>Transfer cells in an appropriate tube size and place it on a magnet.</li>
<li>Wait 10 min.</li>
<li>Transfer cells into a fresh tube.</li>
</ol>
</li>
</ol>
</li>
<li>Count the cells.</li>
<li>Prepare 12 well plate with 1.5 mL of RPMI 20% (v/v) FCS, IL-2 (10 ng/mL) and L-glutamine (2 mM) <b>without antibiotics</b> (RPMIxx)/well. Place it into the cell culture incubator (37°C, 5% CO<sub>2</sub>).</li>
<li>Pre-warm Human T cell nucleofector solution with added supplement (delivered together with the kit).</li>
<li>Prepare cuvettes.</li>
<li>Take 6 million CD4<sup>+</sup> T lymphocytes per sample.</li>
<li>Centrifuge cells at 393 rcf for 3 min at 22°C–23°C and remove the supernatant as much as possible.</li>
<li>Add 100 μL of T cell nucleofector solution, add 1.5 μg of IFI16 shRNA, gently mix, and transfer into the cuvette.</li>
<li>Place the cuvette into the Amaxa device and pulse the sample with U15 program.</li>
<li>Immediately transfer the sample into the medium in the 12 well plate (one sample/well).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For primary CD4<sup>+</sup> T lymphocytes transfection, it is important to use high purity DNA.</div>
<h3 id="sec3.4">Validation of knockdown efficiency</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes how to determine the knockdown efficiency via western blot analysis.</p>
<ol start="19">
<li>Harvest and lyse cells
<ol type="a">
<li>72 h after transfection, harvest one million cells and centrifuge them at 393 rcf for 5 min at 22°C–23°C.</li>
<li>Remove the medium and add 500 μL of PBS.</li>
<li>Centrifuge at 393 rcf for 5 min at 22°C–23°C.</li>
<li>Remove the supernatant and add 100 μL of western blot lysis buffer (150 mM NaCl, 50 mM HEPES, 5 mM EDTA, 0.1% (v/v) NP40, 500 μM Na<sub>3</sub>VO<sub>4</sub>, 500 μM NaF, pH 7.5) supplemented with protease inhibitor.</li>
<li>Vortex and incubate on ice for 10 min.</li>
<li>Centrifuge for 20 min at 20,817 rcf at 4°C.</li>
<li>Transfer the supernatant in a new 1.5 mL Eppendorf tube.</li>
<li>Measure protein concentration using the Nanodrop. Normalize protein concentration for each donor with CoIP lysis buffer.</li>
<li>Transfer 30 μL of supernatants in a new 1.5 mL Eppendorf tube and add 10 μL of loading dye (90% (v/v) 4× loading dye + 10% (v/v) β-mercaptoethanol).</li>
<li>Heat at 95°C for 10 min.</li>
<li>Freeze at −20°C.</li>
</ol>
</li>
<li>Perform western blot analysis as described in the siRNA section (points 8 and 9).</li>
</ol>
<h3 id="sec3.5">gRNA/Cas9 nucleofection of CD4<sup>+</sup> T lymphocytes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>This step accomplishes the gRNA/Cas9 nucleofection</p>
<ol start="21">
<li>Mix 80 pmol Cas9 and 240 pmol sgRNA in a final volume of 5 μL.</li>
<li>Incubate the complex for 20 min at 22°C–23°C.</li>
<li>Prior to nucleofection, prepare the nucleofector master mix.
<ol type="a">
<li>For the mix use 82% (v/v) of the Lonza P3 Primary Cell Solution and 18% (v/v) of the supplement 1 for a total volume of 20 μL for one nucleofection.</li>
</ol>
</li>
<li>In the meantime, remove the anti-CD3/CD28 magnetic beads from the CD4<sup>+</sup> T lymphocyte culture and transfer 1 million cells in a new 1.5 mL Eppendorf tube (use one tube per reaction).
<ol type="a">
<li>Spin down the cells and carefully remove the medium.</li>
</ol>
</li>
<li>Add 20 μL of nucleofector master mix and straight afterward the gRNA/Cas9 complex to the cells, mix gently, and transfer it in the nucleofection strip. Avoid making the bubbles.</li>
<li>Insert the strip in the 4D Nucleofector Core Unit and chose the pulse code EO115.</li>
<li>Add 80 μL warm RPMIxx supplemented with 20% (v/v) FCS and IL-2 (10 ng/mL) to the strip and incubate in cell culture incubator for 10 min.</li>
<li>Add 200 μL warm RPMIxx with 20% (v/v) FCS and IL-2 (10 ng/mL) in a 48 well plate, transfer the electroporated cells in the plate and culture for 14 h at 37°C, 5% CO<sub>2</sub>.</li>
<li>On the following day, add 300 μL of warm RPMIxx with 20% (v/v) FCS and IL-2 (10 ng/mL). Transfer cells in a 24 well plate. (See <a href="#sec7.1">Troubleshooting 1</a>)</li>
<li>48 h post nucleofection, supplement the culture with 500 μL fresh RPMIxx with 20% (v/v) FCS and IL-2 (10 ng/mL). If necessary, transfer the cells in a bigger culture format.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep in mind the toxicity of the nucleofector master mix and never leave the cells in it for more than 15 min before nucleofection.</div>
<h3 id="sec3.6">Proving knockout efficiency via western blot</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes how to determine the knockout efficiency via western blot analysis.</p>
<ol start="31">
<li>Harvest and lyse cells
<ol type="a">
<li>96 h after nucleofection, harvest one million cells and centrifuge them at 393 rcf for 5 min at 22°C–23°C.</li>
<li>Remove the medium and add 500 μL of PBS.</li>
<li>Centrifuge at 393 rcf for 5 min at 22°C–23°C.</li>
<li>Remove the supernatant and add 100 μL of western blot lysis buffer (150 mM NaCl, 50 mM HEPES, 5 mM EDTA, 0.1% (v/v) NP40, 500 μM Na<sub>3</sub>VO<sub>4</sub>, 500 μM NaF, pH 7.5) supplemented with protease inhibitor.</li>
<li>Vortex and incubate on ice for 10 min.</li>
<li>Centrifuge for 20 min at 20,817 rcf at 4°C.</li>
<li>Transfer the supernatant in a new 1.5 mL Eppendorf tube.</li>
<li>Measure protein concentration using the Nanodrop. Normalize protein concentration for each donor with CoIP lysis buffer.</li>
<li>Transfer 30 μL of supernatants in a new 1.5 mL Eppendorf tube and add 10 μL of loading dye (90% (v/v) 4× loading dye + 10% (v/v) β-mercaptoethanol).</li>
<li>Heat at 95°C for 10 min.</li>
<li>Freeze at −20°C.</li>
</ol>
</li>
<li>Perform western blot analysis as described in the siRNA section (points 8 and 9). (See <a href="#sec7.3">Troubleshooting 2</a>)</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>According to the siRNA transfection process, it is expected to see a reduction of IFI16 expression levels by an average of 50% (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/360-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. IFI16 knockdown in primary macrophages</div>
<p>Human monocyte-derived macrophages were treated with IFI16-specific or control siRNA and knockdown efficiencies were determined by western blot analysis (indicated below) (<a href="#bib2">Hotter et al., 2019</a>)</p>
</figcaption>
</figure>
<p>An efficient shRNA transfection results in a 40% reduction of IFI16 protein levels. <a href="#fig2">Figure 2</a> is an example of an IFI16 knockdown in primary CD4<sup>+</sup> T lymphocytes.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/360-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. IFI16 knockdown in primary CD4<sup>+</sup> T lymphocytes</div>
<p>Human CD4<sup>+</sup> T lymphocytes isolated from three different donors were transfected with either a control or an IFI16-specific shRNA. Knockout efficiency was assessed by western blot. The right panel shows the quantification of IFI16 and PYHIN1 band intensities in the presence of the non-targeting (nt) control or an IFI16-specific shRNA. Data are represented as means ± SD.</p>
</figcaption>
</figure>
<p>A successful gRNA/Cas9 complex nucleofection in CD4<sup>+</sup> T lymphocytes should result in a 70%–90% knockout. The IFI16 gRNA should specifically target IFI16 and not cross react with any other PYHIN protein.</p>
<p><a href="#fig3">Figure 3</a> depicts an example of western blot showing an efficient IFI16 knockout using gRNA/Cas9.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/360-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. IFI16 knockout in primary CD4<sup>+</sup> T lymphocytes</div>
<p>Human CD4<sup>+</sup> T lymphocytes isolated from three different donors were nucleofected with either a control or an IFI16-specific gRNA and Cas9. Knockout efficiency was assessed by western blot (left panel). Quantification of IFI16 and PYHIN1 band intensity (right panel) verified the specificity of the IFI16 gRNA. Data are represented as means ± SD.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Western blot band intensities were quantified using the LI-COR Image Studio Software.</p>
<p>Statistical analyses were performed using GraphPad PRISM 8 (GraphPad Software). P-values were determined using a two-tailed Student’s t test. Unless otherwise stated, data are shown as the mean of at least three independent experiments ± SD. Significant differences are indicated as: ∗p &lt;0.05; ∗∗p &lt;0.01; ∗∗∗p &lt;0.001.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Cell culture viability represents the main obstacle concerning experiments with primary macrophages as this is donor-dependent. On day 7 post macrophages isolation, cells should be fully attached to the bottom of the wells and have either a fibroblast-like or rounded shape. If cells are still floating, most likely the differentiation did not work properly and it is pointless to carry on the experiment. From our experience, this is very often linked to over-counting mistakes, leading to a reduced cell density in the well. Using an automatic cell counter would help in preventing this.</p>
<p>Concerning the knockdown approaches, using either IFI16 siRNA or shRNA may result in potential off-target effect with PYHIN1-mRNA. To undergo this limitation, we suggest to use the gRNA/Cas9 approach. Off-targets of siRNA and shRNAs are mainly due to their instability. On the other hand, gRNAs are known to be much more stable due to their structure which includes R-loop dynamics (<a href="#bib3">Mullally et al., 2020</a>).</p>
<p>However, the gRNA/Cas9 approach may have economical restrictions given the high costs of both reagents but still remains the preferable approach, given its very high efficiency compared to the mRNA-targeting techniques.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low CD4<sup>+</sup> T lymphocytes viability upon nucleofection.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Compared to chemical transfection, nucleofection induces membrane-pores formation via an external voltage. The main drawback is represented by a partial successful membrane repair, which could be overcome by some easy steps:</p>
<ul>
<li>Make sure that the anti-CD3/CD28 magnetic beads were completely removed;</li>
<li>Do not leave the cells for more than 15 min in the Lonza (P3 Primary Cell Solution or Human T cell nucleofector solution);</li>
<li>After nucleofection immediately add medium or transfer the cells in to the medium without antibiotics;</li>
<li>Avoid cells overgrowth by splitting the cells in a bigger plate format; make sure to use 20% (v/v) FCS in the RPMI culture medium.</li>
</ul>
<h3 id="sec7.3">Problem 2</h3>
<p>Low IFI16 knockdown/knockout efficiency.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>As these procedures involve working with RNA, it is important to work in a nuclease free environment. This can be achieved by carefully inactivating DNases and RNases with RNase away and by using sterile filter tips. Concerning the IFI16 gRNA, multiple freeze and thaw cycles will reduce its stability. Therefore, it is recommended to aliquot the gRNA upon resuspension in nuclease free water.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bosso, M., Prelli Bozzo, C., Hotter, D., Volcic, M., Stürzel, C.M., Rammelt, A., Ni, Y., Urban, S., Becker, M., Schelhaas, M., et al. (2020). Nuclear PYHIN proteins target the host transcription factor Sp1 thereby restricting HIV-1 in human macrophages and CD4+ T cells. PLoS Pathog. <i>16</i>, e1008752. <a href="https://doi.org/10.1371/journal.ppat.1008752">https://doi.org/10.1371/journal.ppat.1008752</a></p>
<p id="bib2">Hotter, D., Bosso, M., Jønsson, K.L.K.L., Krapp, C., Stürzel, C.M.C.M., Das, A., Littwitz-Salomon, E., Berkhout, B., Russ, A., Wittmann, S., et al. (2019). IFI16 targets the transcription factor Sp1 to suppress HIV-1 transcription and latency reactivation. Cell Host Microbe <i>25</i>. <a href="https://doi.org/10.1016/j.chom.2019.05.002">https://doi.org/10.1016/j.chom.2019.05.002</a></p>
<p id="bib3">Mullally, G., van Aelst, K., Naqvi, M.M., Diffin, F.M., Karvelis, T., Gasiunas, G., Siksnys, V., and Szczelkun, M.D. (2020). 5′ modifications to CRISPR–Cas9 gRNA can change the dynamics and size of R-loops and inhibit DNA cleavage. Nucleic Acids Res. <i>48</i>, 6811-6823. <a href="https://doi.org/10.1093/nar/gkaa477">https://doi.org/10.1093/nar/gkaa477</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Frank Kirchhoff (<a href="mailto:frank.kirchhoff@uni-ulm.de">frank.kirchhoff@uni-ulm.de</a>).</p>
<h4>Materials availability</h4>
<p>No materials were generated in this study.</p>
<h4>Data and code availability</h4>
<p>No data or code were generated in this study.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants from the <a href="https://doi.org/10.13039/501100001659">German Research Foundation</a> (DFG CRC 1279 and SPP 1923) to F.K.</p>
<h3>Author Contributions</h3>
<p>M.B., C.P.B., and M.V. developed the protocols and prepared this manuscript. F.K. supervised the studies and edited the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>