<article>
<section>
<h1 id="header">Intranuclear immunostaining-based FACS protocol from embryonic cortical tissue</h1>
<p><time datetime="2021-02-05">Published: February 5, 2021</time></p>
<p>M. Sadman Sakib,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a></sup> Godwin Sokpor,<sup><a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Huu Phuc Nguyen,<sup><a href="#aff2">2</a></sup> Andre Fischer,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#cor1">*</a></sup> and Tran Tuoc<sup><a href="#aff2">2</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>German Center for Neurodegenerative Diseases, 37077 Goettingen, Germany</p>
<p id="aff2"><sup>2</sup>Ruhr University of Bochum, 44791 Bochum, Germany</p>
<p id="aff3"><sup>3</sup>Cluster of Excellence “Multiscale Bioimaging: from Molecular Machines to Networks of Excitable Cells (MBExC),” 37077 Goettingen, Germany</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:andre.fischer@dzne.de">andre.fischer@dzne.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:Tran.Tuoc@ruhr-uni-bochum.de">Tran.Tuoc@ruhr-uni-bochum.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100318">10.1016/j.xpro.2021.100318</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Cell sorting can be used to purify cell populations for cell type-specific molecular probing. Fluorescence-activated cell sorting (FACS) coupled with high-throughput sequencing affords molecular signature identification for specific cell types. FACS has many challenges that limit comprehensive cell purification from the brain, leading to incomplete molecular characterization. Here, we present the intranuclear immunostaining-based FACS protocol with several modified steps, which allows optimized nuclei/cell sorting from mouse or human embryonic cortical tissue for distinct downstream molecular investigation of basal intermediate progenitors.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>The FACS technique allows effective nuclei and cell sorting for molecular profiling</li>
<li>The protocol is optimized for nuclei and cell purification from embryonic cortical tissue</li>
<li>Modified steps ensured isolation of TBR2+ cells for mass spectroscopy analysis</li>
<li>The protocol was used to generate extensive RNA-seq and epigenome data for TBR2+ nuclei</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/448-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Decontaminating FACS machine</h3>
<div class="timing">
<span class="timing-title">Timing:</span> day before and on the day of sorting</div>
<ol>
<li>Make sure the FACS machine is optically aligned, lasers fully functioning with all the fluidics of the machine running in optimum condition.
<ol type="a">
<li>For BD FACS machines, perform routine laser checkup with CST beads.</li>
<li>While the machine is running, put 2–3 mL of RNaseZAP solution in a FACS tube and perform flow cell cleanup with the closed loop nozzle inside. Keep it for 5 min and use Sigma nuclease-free water afterwards to clean the flow cell 2 times.</li>
</ol>
</li>
<li>On the day of sorting, for BD machines, optimum sorting accuracy should be attained by performing automatic drop delay setup with BD Accudrop beads according to manufacturer’s protocol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Standard operating procedures of FACS machine applies as well. 85 μm Nozzle with 4,000–6,000 events/s sample flow rate should be used while sorting.</div>
<div class="critical">
<span class="critical-title">Critical:</span> For preparing PBS buffer (Sheath fluid) for FACS, use cell culture grade, filtered 10×.</div>
<p>PBS (PAN-Biotech) and dilute with milli-Q water and autoclave afterwards. Do not use non-autoclaved PBS.</p>
<div class="critical">
<span class="critical-title">Critical:</span> For preparing 70% ethanol, always filter the 100% ethanol with 0.2 μm filtration unit, then use autoclaved milli-Q water to dilute.</div>
<h3 id="sec1.2">Preparation of cortical tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–2 h</div>
<ol>
<li>Start with E13.5–E16.5 mouse brain or GW14–GW20 human brain.</li>
<li>Dissect, and extract cortical regions in cold PBS with DEPC.</li>
<li>Remove meninges as well to reduce clumps.</li>
<li>Pool at least 4 mouse cortices or equal amount of human cortex per replicate.</li>
<li>Collect them in 1.5 mL tubes (for RNA-seq, ChIP-seq protocols) or 15 mL tubes (MS protocol) on ice and proceed to subsequent steps described below.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Multimedia presentation for the preparation of cortical tissue can be obtained from <a href="#bib7">Viesselmann et al. (2011)</a> and (<a href="#bib6">Tuoc and Stoykova, 2008</a>) and freely accessible at PubMed Central portal.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Tissue can be kept for a day in PBS (for below ChIP-seq or MS protocol) or in RNAlater solution (for RNA-seq protocol).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Alexa Fluor 488-conjugated rat EOMES/TBR2 antibody</td>
<td>eBioscience</td>
<td>53-4875-82 (Clone: Dan11mag)</td>
</tr>
<tr>
<td>Alexa Fluor 488</td>
<td>Invitrogen</td>
<td>A-11006; RRID: AB_221544</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>E13.5–E16.5 mouse cortical tissue</td>
<td>Charles River Laboratories</td>
<td>CD-1 IGS</td>
</tr>
<tr>
<td>GW14–GW20 human cortical tissue</td>
<td>Medical University-Varna</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Nuclei EZ prep lysis buffer</td>
<td>Sigma</td>
<td>NUC101-1KT</td>
</tr>
<tr>
<td>BSA, RNase free</td>
<td>Merck</td>
<td>2905-5GM</td>
</tr>
<tr>
<td>BSA, fraction V</td>
<td>Roth</td>
<td>8076.4</td>
</tr>
<tr>
<td>Complete Protease inhibitor cocktail EDTA free</td>
<td>Roche</td>
<td>4693132001</td>
</tr>
<tr>
<td>RNasin Plus RNase inhibitor</td>
<td>Promega</td>
<td>N2615</td>
</tr>
<tr>
<td>10× PBS, RNase free</td>
<td>Invitrogen</td>
<td>AM9625</td>
</tr>
<tr>
<td>DPBS, w/o Ca and Mg</td>
<td>PAN-Biotech</td>
<td>P04-361000</td>
</tr>
<tr>
<td>Sucrose for molecular biology</td>
<td>Sigma</td>
<td>S0389</td>
</tr>
<tr>
<td>Calcium chloride (CaCl<sub>2</sub>)</td>
<td>Roth</td>
<td>5239.2</td>
</tr>
<tr>
<td>Magnesium acetate (MgAc<sub>2</sub>)</td>
<td>Roth</td>
<td>P026.1</td>
</tr>
<tr>
<td>EDTA (0.5 M), pH 8.0</td>
<td>Invitrogen</td>
<td>AM9260G</td>
</tr>
<tr>
<td>HEPES (1 M)</td>
<td>Gibco</td>
<td>15630056</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma</td>
<td>T8787</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Sigma</td>
<td>P9416-50ML</td>
</tr>
<tr>
<td>1,4-Dithiothreitol (DTT)</td>
<td>Roth</td>
<td>6908.3</td>
</tr>
<tr>
<td>RNase-free molecular biology grade water</td>
<td>Sigma</td>
<td>W4502-1L</td>
</tr>
<tr>
<td>TRIzol LS reagent</td>
<td>Invitrogen</td>
<td>10296010</td>
</tr>
<tr>
<td>Trichloromethane/chloroform</td>
<td>Roth</td>
<td>4432.1</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Roth</td>
<td>9065.1</td>
</tr>
<tr>
<td>RNA 6000 Pico Bioanalyzer Kit</td>
<td>Agilent</td>
<td>5067-1513</td>
</tr>
<tr>
<td>Qubit RNA HS Assay Kit</td>
<td>Invitrogen</td>
<td>Q32852</td>
</tr>
<tr>
<td>SMART-Seq v4 Ultra Low Input RNA Kit for sequencing</td>
<td>Takara</td>
<td>634888 (for 12 reactions)</td>
</tr>
<tr>
<td>Nextera XT DNA Library Preparation Kit</td>
<td>Illumina</td>
<td>FC-131-1024</td>
</tr>
<tr>
<td>37% Formaldehyde solution</td>
<td>Sigma</td>
<td>F8775</td>
</tr>
<tr>
<td>Glycine (1.25 M stock)</td>
<td>Millipore</td>
<td>357002</td>
</tr>
<tr>
<td>NEBNext Ultra II DNA Library Prep Kit for Illumina</td>
<td>NEB</td>
<td>E7645S</td>
</tr>
<tr>
<td>Foxp3/Transcription Factor Staining Buffer Set</td>
<td>eBioscience</td>
<td>00-5523</td>
</tr>
<tr>
<td>Neural tissue dissociation kit</td>
<td>MACS Miltenyi Biotec</td>
<td>130-092-628 P</td>
</tr>
<tr>
<td>RNaseZap</td>
<td>Ambion</td>
<td>9780.9782</td>
</tr>
<tr>
<td>RNeasy Mini Kit</td>
<td>Qiagen</td>
<td>Cat#74104</td>
</tr>
<tr>
<td>Fetal calf serum (FCS)</td>
<td>Sigma</td>
<td>C8056</td>
</tr>
<tr>
<td>PBS (1×)</td>
<td>Invitrogen</td>
<td>Cat#20012-019</td>
</tr>
<tr>
<td>SUPERase⋅In RNase Inhibitor (20 U/μL)</td>
<td>Invitrogen Thermo Fisher</td>
<td>Cat#AM2696, 2,500 units</td>
</tr>
<tr>
<td>Complete (protein inhibitors)</td>
<td>Roche</td>
<td>Mini: #1 836 153</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>BD FACS Aria III sorter</td>
<td>BD Biosciences</td>
<td><a href="https://www.bdbiosciences.com/enus/instruments/research-instruments%20/research-cell-sorters/facsaria-iii">https://www.bdbiosciences.com/enus/instruments/research-instruments /research-cell-sorters/facsaria-iii</a></td>
</tr>
<tr>
<td>Falcon tube roller mixer</td>
<td>Stuart</td>
<td>SRT9D</td>
</tr>
<tr>
<td>2100 Bioanalyzer Instrument</td>
<td>Agilent</td>
<td>G2939BA; RRID:SCR_018043</td>
</tr>
<tr>
<td>Qubit 4 fluorometer</td>
<td>Invitrogen</td>
<td>Q33238; RRID:SCR_018095</td>
</tr>
<tr>
<td>T 10 basic ULTRA-TURRAX</td>
<td>IKA</td>
<td>0003737000</td>
</tr>
<tr>
<td>S10N-5G dispersing tool for ULTRA-TURRAX</td>
<td>IKA</td>
<td>0003304000</td>
</tr>
<tr>
<td>Sterile syringe filter unit, 0.22 μm, PES, 33 mm</td>
<td>Millipore</td>
<td>SLGP033RS</td>
</tr>
<tr>
<td>DNA LoBind tubes (1.5 mL)</td>
<td>Eppendorf</td>
<td>0030108051</td>
</tr>
<tr>
<td>DNA LoBind tubes (2 mL)</td>
<td>Eppendorf</td>
<td>0030108078</td>
</tr>
<tr>
<td>Oak ridge PSF tubes</td>
<td>Thermo</td>
<td>3137-0050</td>
</tr>
<tr>
<td>70 μm cup-type strainer (sterile)</td>
<td>BD Medimachine</td>
<td>340633</td>
</tr>
<tr>
<td>Aluminum heating/cooling block</td>
<td>Sigma</td>
<td>Z743497</td>
</tr>
<tr>
<td>Falcon 5 mL round bottom polystyrene test tube, with cell strainer snap cap</td>
<td>Corning</td>
<td>352235</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="note">
<span class="note-title">Note:</span> Use Sigma molecular biology grade water unless otherwise stated.</div>
<div class="note">
<span class="note-title">Note:</span> One Roche Protease inhibitor tablet can be dissolved into 500 μL and kept at −20°C for storage.</div>
<div class="note">
<span class="note-title">Note:</span> Dilute Triton X-100 into water (Sigma) to make 10% stock concentration. Protect from light.</div>
<div class="note">
<span class="note-title">Note:</span> Dilute Tween 20 with water (Sigma)to make 20% stock concentration. Protect from light.</div>
<div class="note">
<span class="note-title">Note:</span> Make Salt stock solutions (e.g., CaCl<sub>2</sub> and MgAc<sub>2</sub>) and syringe filter.</div>
<table id="undtbl1">
<caption>Nuclei suspension buffer (NSB) for sorted nuclear RNA: (5,500 μL/sample required)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Unit</th>
<th>Stock conc</th>
<th>Final conc</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA, RNase free</td>
<td>%</td>
<td>Powder</td>
<td>0.5</td>
</tr>
<tr>
<td>10× PBS</td>
<td>x</td>
<td>10</td>
<td>1.00</td>
</tr>
<tr>
<td>RNasin Plus RNase inhibitor</td>
<td>x</td>
<td>100</td>
<td>0.5</td>
</tr>
<tr>
<td>Protease inhibitor</td>
<td>x</td>
<td>100</td>
<td>1</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Use only freshly prepared NSB. After adding the BSA and PBS, with sufficient water, keep it on a rotator to mix without forming bubbles. Add water up to necessary, syringe filter, then add the inhibitors before use.</div>
<table id="undtbl2">
<caption>Low Sucrose Buffer (LSB): (1,500 μL/sample needed)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Units</th>
<th>Stock concentration</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose (342.3 g/mol)</td>
<td>mM</td>
<td>powder</td>
<td>320</td>
</tr>
<tr>
<td>CaCl2</td>
<td>mM</td>
<td>2,500</td>
<td>5</td>
</tr>
<tr>
<td>MgAc<sub>2</sub>
</td>
<td>mM</td>
<td>300</td>
<td>5</td>
</tr>
<tr>
<td>EDTA</td>
<td>mM</td>
<td>500</td>
<td>0.1</td>
</tr>
<tr>
<td>HEPES</td>
<td>mM</td>
<td>1,000</td>
<td>10</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>%</td>
<td>10</td>
<td>0.1</td>
</tr>
<tr>
<td>Protease inhibitor (PI)∗</td>
<td>x</td>
<td>100</td>
<td>1</td>
</tr>
<tr>
<td>DTT∗</td>
<td>mM</td>
<td>500</td>
<td>1</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> ∗Add PI and DTT freshly.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make the buffer without PI and DTT, and store at 4°C maximum 2–3 weeks. Due to presence of Sucrose, Fungal growth can easily occur. Therefore, make sure to discard after usage.</div>
<table id="undtbl3">
<caption>High Sucrose buffer (HSB): 6,000 μL/sample needed</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Units</th>
<th>Stock concentration</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose (342.3 g/mol)</td>
<td>mM</td>
<td>powder</td>
<td>1,000</td>
</tr>
<tr>
<td>MgAc<sub>2</sub>
</td>
<td>mM</td>
<td>300</td>
<td>3</td>
</tr>
<tr>
<td>HEPES</td>
<td>mM</td>
<td>1,000</td>
<td>10</td>
</tr>
<tr>
<td>DTT∗</td>
<td>mM</td>
<td>500</td>
<td>1</td>
</tr>
<tr>
<td>Protease inhibitor(PI)∗</td>
<td>x</td>
<td>100</td>
<td>1</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> ∗Add PI and DTT freshly.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make the buffer without PI and DTT, store at 4°C maximum 2–3 weeks. Due to presence of Sucrose, Fungal growth can easily occur. Therefore, make sure to throw away after usage.</div>
<table id="undtbl4">
<caption>PBS containing Tween and BSA (PBTB): 3,500 μL/sample needed</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Units</th>
<th>Stock concentration</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA, fraction V</td>
<td>%</td>
<td>Powder</td>
<td>1</td>
</tr>
<tr>
<td>Tween 20</td>
<td>%</td>
<td>20</td>
<td>0.2</td>
</tr>
<tr>
<td>Protease inhibitor(PI)∗</td>
<td>x</td>
<td>100</td>
<td>1</td>
</tr>
<tr>
<td>DPBS (1×)</td>
<td>x</td>
<td colspan="2">Use as a diluent for the whole buffer</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare freshly, syringe filter after preparation.</div>
<h3 id="sec2.1">Neural tissue dissociation solution (MACS Miltenyi Biotec, Cat. 130-092-628 P)</h3>
<p>(The kit contains: 2.5 mL of Enzyme P, 2×50 mL of Buffer X, 1.5 mL of Buffer Y, 1 vial of Enzyme A, 1 mL of Buffer A)</p>
<ol>
<li>Resuspend the lyophilized powder in the vial labeled Enzyme A with 1 mL of Buffer A. Do not vortex. Aliquot and store solution at −20°C for later use.</li>
<li>Enzyme mix 1: Prepare 1,950 μL enzyme mix 1 (50 μL Enzyme P + 1,900 μL Buffer X) for up to 400 mg tissue and vortex. Pre-heat the mixture at 37°C for 10–15 min before use.</li>
<li>Enzyme mix 2: Take 20 μL Buffer Y + 10 μL Enzyme A to make 30 μL enzyme mix 2 for each sample.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For one experiment of 6–10 E16.5 cortices.</div>
<h3 id="sec2.2">Foxp3/Transcription factor staining buffer set</h3>
<ol>
<li>Mix 1 part of Foxp3 Fixation/Permeabilization concentrate with 3 parts of Foxp3 Fixation/Permeabilization Diluent to prepare fresh Foxp3 Fixation/Permeabilization working solution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each sample requires 0.5 mL of the working solution. Add 1× complete (Protein inhibitors) for protein application (e.g., mass spectrometry, western blot).</div>
<ol start="2">
<li>Prepare 1× working solution of Permeabilization Buffer by mixing 1 part of 10× Permeabilization Buffer with 9 parts of H<sub>2</sub>O + diethylpyrocarbonate (DEPC).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each sample requires 0.5 mL of the working solution.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Spray RNaseZAP Solution onto the surface of bench and tools for RNA-free condition.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use PBS with DEPC.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use filter-tips.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Control water bath temperature.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Cool down temperature of centrifuges.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>The three protocols below describe different immunostaining-based sorting procedures for nuclei and cells from mouse and human cortices. The sorted materials were used for three downstream applications (RNA-seq, ChIP-seq, and Mass Spectrometry analyses).</p>
<h3 id="sec3.1">Nuclei isolation for TBR2+ nuclei sorting for nuclear RNA and sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h</div>
<p>Since typical cell sorting methods comprise of protease treatment for cell dissociation that can lead to unwanted effects on gene expression profile, we opted for unbiased approach to minimize biasness due to sample processing. The cell sorting protocol requires stringent reagents for cell staining, which usually lead to degradation of RNA. To eliminate such drawback, we opted for TBR2+ nuclei sorting from cortex, instead of cell sorting.</p>
<div class="note">
<span class="note-title">Note:</span> Timing is dependent on the size of the sample to be processed and sorted. Always use</div>
<p>DNA LoBind tubes (Eppendorf) unless otherwise stated.</p>
<h3 id="sec3.2">Nuclei sorting</h3>
<ol>
<li>Directly add 500 μL ice-cold EZ prep lysis buffer, homogenize with plastic pestle (30 times up-down, also circulating movements). Add additional 10–15 strokes until majority of tissue chunks are homogenized and no big chunks remain.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Clean the plastic pestle with RNaseZap and then RNase-free water before using.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Aluminum cooling block for 1.5 mL tubes (kept on ice) should be used to keep the temperature low while homogenizing to avoid RNA degradation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To ensure complete homogenization make sure the plastic pestle fits in the 1.5 mL tube tightly with no gap at the bottom or on the sides.</div>
<ol start="2">
<li>Transfer lysate into a 2 mL tube. Add additional lysis buffer to make up to 2 mL.</li>
<li>Incubate on ice for 7 min (Additional few min due to multiple samples being processed make no difference). Centrifuge at 500 × <i>g</i> for 5 min at 4°C. Discard supernatant.</li>
<li>Resuspend the pellet in 2 mL ice-cold EZ prep lysis buffer. Incubate on ice for 7 min. Filter the lysate through 70 μm cup-type strainer into a new 2 mL tube (To remove large debris and leftover meninges). Afterwards, centrifuge at 500 × <i>g</i> for 5 min at 4°C. Discard supernatant.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Straining lysate through 70 μm filter is crucial. Otherwise, the nuclei will be clumped after addition of NSB leading to failure of the experiment.</div>
<ol start="5">
<li>Resuspend in 1,800 μL NSB. Centrifuge at 500 × <i>g</i> for 5 min at 4°C. Discard supernatant.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Step 5 helps remove residual lysis buffer from the tube.</div>
<ol start="6">
<li>Resuspend in 510 μL NSB. The nuclei suspension is now ready for staining.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> A small 5 mL aliquot of the nuclei suspension can be taken and stained with DAPI to check the nuclei preparation.</div>
<ol start="7">
<li>Take 10 μL as negative control for FACS gating, add 90 μL NSB and keep it on ice. Mix the remaining nuclei suspension with anti TBR2-Alexa488 conjugated antibody (eBioscience) at 1:100 dilution (<a href="#bib5">Tuoc et al., 2013</a>) and incubated for 45 min in a rotator at 4°C (in the dark). In parallel, prepare collection tubes for sorting: add 2× 1 mL NSB into 2× 15 mL falcon tubes (2 tubes per sample), vortex for 10 s, then keep on a roller mixer to coat the tubes at 4°C for at least 30 min or kept rotating until time for sorting).</li>
<li>After staining, centrifuge at 500 × <i>g</i> for 5 min, at 4°C. Discard Supernatant. Wash pellet once with 500 μL NSB by resuspending pellet and centrifuging at 500 × <i>g</i> for 5 min at 4°C.</li>
<li>Resuspend pellet in 500 μL NSB and proceed to the cell sorter immediately.</li>
<li>Flow the sample through 40 μm strainer attached to the FACS tube. Perform sorting. Follow the FACS gating strategy suggested in <a href="#fig1">Figure 1</a>. Since nuclei contains lower amount of RNA than whole cell, sort at least 100,000‒200,000 nuclei for RNA to be detected by Bioanalyzer or Qubit concentration measurements. After sorting, vortex the collection tubes and keep it on ice until end of FACS.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/448-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. FACS gating strategy for mouse and human TBR2 sorting</div>
<p>First P1 is selected to get all nuclei separated from debris, then P2 and P3 population is selected for single nuclei, rejecting doublets, or multiplets. For mouse (A), since the TBR2 population was rather smear like, instead of completely separated population, TBR2+ gating was done conservatively to make sure no TBR2- nuclei mis-sorted. This heavily depends upon sorting efficiency and accuracy of a given FACS machine. (B) Human TBR2 sorting.</p>
</figcaption>
</figure>
</li>
<li>To collect sorted nuclei in the tubes, centrifuge in a swinging bucket for 500 × <i>g</i> for 5 min at 4°C. Carefully take out liquid from top by pipetting and leave out approximately 250 μL solution containing nuclei pellet.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to carefully remove supernatant by slow pipetting with a P200 pipette from the upper surface of the liquid to prevent significant nuclei loss, which can lead to low RNA yield.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Perform post sort imaging of the sorted TBR2+ nuclei to validate their purity. An example is given in <a href="#fig2">Figure 2</a>.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/448-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Post sorting validation for sorting efficiency</div>
<p>Notice that in TBR2− nuclei, there is no signal, while all TBR2+ DAPI nuclei have an Alexa 488 (green) signal, confirming 100% pure population. Scale bar, 5 μm.</p>
</figcaption>
</figure>
<h3 id="sec3.3">RNA isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>For further details, see <a href="#bib1">Halder et al. (2016)</a>.</p>
<ol start="12">
<li>Add 750 μL Trizol LS reagent with the remaining 250 μL suspension. Vortex well. Transfer into 1.5 mL tube and incubate at 20°C‒25°C for 5 min.</li>
<li>Add 200 μL Chloroform, vortex vigorously for 10 s and incubate for 5 min at 20°C‒25°C. Centrifuge for 15 min for 12,000 × <i>g</i> at 4°C. Following centrifugation, collect the upper aqueous phase into a new 1.5 mL tube and add equal volume of 100% ethanol to the aqueous phase. Vortex.</li>
<li>Follow RNA clean and concentrator-5 kit instructions, including on-column DNase treatment to purify RNA (Zymo, catalog no. R1013, instruction manual version 2.3.1, pp. 3 and 4).</li>
<li>At the last step, collect RNA into 12 μL water. Use 1 μL for measuring RNA integrity (RIN value) in Bioanalyzer with RNA 6000 pico kit (Agilent) and 1 μL for concentration measurement with Qubit RNA HS kit (Invitrogen).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Due to very low amount of RNA, it is sometimes difficult to get concentration measurement from Qubit. In that case, use concentration determined by Bioanalyzer. Typically, 28S rRNA is more prone to degradation. The RNA integrity value is usually low in RNA samples, which have the high content of 28S rRNA. The nuclear RNA prepared from progenitor cells from embryonic cortex contains a higher amount of 28S rRNA and lower amount of 18S rRNA than the RNA isolated from adult tissue in the Bioanalyzer quantification. This resulted in unmeasurable RNA integrity number (RIN). Therefore, RNA quality should also be carefully determined by checking the regions (e.g., 200‒2000 nucleotides) in the gel image where typically degraded RNA could be found. with our protocol, the degraded RNA was not seen, and the 28S rRNA is found to be in intact form (<a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/448-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Bioanalyzer image of sorted TBR2 nuclear RNA</div>
<p>Notice the unusually low amount of 18S rRNA, which results in an unmeasurable RNA integrity number (RIN). However, intact 28S rRNA and no visible degraded RNA in regions between 200 and 2000 nucleotide shows no apparent RNA degradation.</p>
</figcaption>
</figure>
<h3 id="sec3.4">Sequencing library preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>For further details, see <a href="#bib1">Halder et al. (2016)</a>. Sequencing libraries can be prepared with any appropriate kits, but below is a simple guideline for performing standard libraries from very low input sorted nuclear RNA.</p>
<ol start="16">
<li>Use 100 pg–10 ng RNA for cDNA synthesis with SMART-Seq v4 Ultra Low Input RNA Kit (Takara) for optimal results. Follow instruction manual provided with the kit for cDNA synthesis with appropriate PCR cycles for avoiding over amplified library.</li>
<li>After cDNA synthesis, prepare Illumina sequencing libraries using 1 ng cDNA with Illumina Nextera XT library preparation kit. Follow instruction manual to perform library preparation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Takara now provides SMART-Seq v4 Ultra low input Plus kit, that includes both cDNA and Illumina sequencing library preparation reagents. In that case, Nextera XT kit is not needed.</div>
<ol start="18">
<li>Perform Illumina high-throughput sequencing with minimum 50 base pair single end reads and 30 million raw reads per sample. Perform bioinformatic analysis for sequencing quality control and data analysis. After using STAR aligner (2.5.2b) for read mapping to reference genome, feature counts (Subread package v1.5.1) for generating raw read counts, DESeq2(v1.30) for normalizing raw read counts, and taking base mean &gt;5, we detected around 20,000 genes under the above mentioned sequencing conditions.</li>
</ol>
<h3 id="sec3.5">Nuclei preparation for TBR2+ nuclei sorting and chromatin immunoprecipitation (ChIP) for epigenome profiling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>In principle, both sorted TBR2+ nuclei (above protocol) and sorted TBR2+ cells (see below protocol) can be used for ChIP-seq experiment. We experienced however that the ChIP-seq results have less background with sorted nuclei than those with sorted cells. Therefore, the sorted nuclei protocol is recommended for ChIP experiments.</p>
<div class="note">
<span class="note-title">Note:</span> Always use DNA LoBind tubes (Eppendorf) unless otherwise stated.</div>
<ol start="19">
<li>Add 250 μL of low sucrose buffer (LSB) to freshly dissected embryonic mouse cortical tissues (pool 4/5 mice tissue per replicate) in a 1.5 mL tube and homogenize with plastic pestle (30 times up-down, also in circulating movements). Add additional 10–15 strokes until majority of tissue chunks are homogenized and no large chunks remain. Keep it cold as much as possible while homogenizing by using aluminum block for 1.5 mL tubes on ice. The solution should have no tissue chunks left. Make it up to 500 μL with LSB.</li>
<li>Add 13.5 μL of 37% formaldehyde solution (to have 1% as final concentration). Keep at 20°C‒25°C for 10 min in a rotator for cross-linking.</li>
<li>Stop cross-linking reaction by adding 52 μL of 1.25 M glycine. Rotate for 5 min on rotator at 20°C‒25°C. Centrifuge at 2000 × <i>g</i> for 3 min at 4°C.</li>
<li>Discard the supernatant and resuspend pellet in 1,000 μL LSB. Now, homogenize again using mechanical homogenizer (IKA Ultraturax T10, with S10N-5G dispersion tool) at power 3 until the suspension gets clear (10–20 s) with no visible clumps.</li>
<li>Carefully layer homogenized solution on top of 6 mL high sucrose buffer (HSB) as sucrose cushion in oak ridge tubes, followed by centrifugation at ~3,148 × <i>g</i> in a swinging bucket rotor for 10 min at 4°C.</li>
<li>A nuclei pellet would be visible at the bottom of the tube (<a href="#fig4">Figure 4</a>). Carefully remove the supernatant by pipetting from top layer. (<b>Recommended to use:</b> Eppendorf E3x with 5 mL Combitips advanced for smooth pipetting). Leave less than 2 mL. Resuspend in the remaining buffer and transfer into 2 mL tube.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/448-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Oak ridge tube nuclei pellet after centrifugation</div>
<p>Take note of the ring-shaped pellet forming instead of a regular pellet at the bottom.</p>
</figcaption>
</figure>
</li>
<li>Centrifuge for 3 min at 2000 × <i>g</i> in 4°C. Remove supernatant, resuspend into 610 μL PBTB (10 μL extra for negative control staining later).</li>
</ol>
<ol start="26">
<li>Nuclei Staining for Sorting:
<div class="note">
<span class="note-title">Note:</span> The nuclei staining steps must be performed on ice.</div>
<div class="note">
<span class="note-title">Note:</span> Multimedia presentation for the nuclei staining steps can be obtained from <a href="#bib2">Menon et al. (2014)</a> and freely accessible at PubMed Central portal.</div>
<ol type="a">
<li>Pool 10 μL from all tubes into a new 1.5 mL tube. Dilute accordingly for negative control.</li>
<li>Primary antibody staining: Add 1:100 TBR2+AF488 conjugated antibody. Incubate on rotator for 20 min at 4°C, together with negative control. Perform step 9 in parallel.</li>
<li>Centrifuge at 2000 × <i>g</i> for 3 min at 4°C. Wash nuclei twice with 1 mL DPBS. Each wash entails resuspending in DPBS and centrifuging for 3 min at 2000 × <i>g</i> at 4°C.</li>
<li>After washing, resuspend in 500 μL PBTB. Keep it in the dark on ice.</li>
<li>Flow the sample through 40 μm strainer attached to the FACS tube. Proceed to FACS machine with the collection tubes prepared in step 9.</li>
</ol>
</li>
</ol>
<ol start="27">
<li>Coating tubes for collecting sorted nuclei
<ol type="a">
<li>While staining with antibody, take 15 mL falcon tubes (twice the number of samples, excluding negative control). Add 1 mL of PBTB. Vortex to evenly distribute the buffer on sides of the tubes.</li>
<li>Leave it for rolling at 4°C for the time nuclei is staining. Take them with you when are going for the sorter.</li>
</ol>
</li>
<li>Perform FACS sorting of TBR2 positive and negative nuclei. Use negative control prepared in step 8a to determine TBR2 expressing nuclei population. Sort minimum of 100,000‒200,000 nuclei for measurable amounts of chromatin. After sorting, perform the following to collect nuclei for ChIP:
<ol type="a">
<li>After each sorting, vortex each falcon tube slightly to collect nuclei from the sides of the tubes. Keep on ice until all samples are sorted.</li>
<li>Balance the tubes before centrifuging. Centrifuge for 15 min at 3,148 × <i>g</i> at 4°C in a swinging bucket rotor.</li>
<li>Very carefully decant the supernatant without shaking. There should be some buffer left inside. Mix the pellet with the remaining buffer and transfer into 1.5 mL tubes.</li>
<li>Centrifuge again for 10,000 × <i>g</i> for 2 min at 4°C. Remove supernatant completely by pipetting with P200 and P10 pipettes. Flash freeze the sorted nuclei pellet in liquid nitrogen, then store at −80°C until further processing.</li>
</ol>
</li>
<li>ChIP-seq was done according to <a href="#bib3">Narayanan et al. (2015)</a> and <a href="#bib4">Nguyen et al. (2018)</a>. Use updated library preparation kit (NEB Next Ultra II DNA library preparation kit).</li>
</ol>
<h3 id="sec3.6">TBR2+ cell sorting for mass spectrometry (MS) analysis of epigenetic marks</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>The protocol for sorted nuclei precludes isolation of cell type-specific cytosolic protein for downstream applications. We therefore established a cell preparation protocol using solution with stringent reagents (Foxp3/Transcription Factor Staining Buffer Set) for cell staining. Histones, chromatin, and proteins can be extracted from these sorted cells for mass spectrometry, western blot, and ChIP analyses. Due to the harsh fixation with the stringent solution during cell staining process, we were not successful in using these sorted cells for RNA-based investigation.</p>
<ol start="30">
<li>Spin (i.e., 1,000 rpm for 2 min) cortical tissue in 15 mL tubes and take off supernatant.</li>
<li>Add 1,950 μL of pre-heated enzyme mix 1 (refer to solution preparation) to a maximum of 400 mg tissue. Add Protein inhibitors.</li>
<li>Incubate at 37°C for 15 min in water bath (mix by hand every 2–3 min).</li>
<li>Prepare fire-polished Pasteur pipettes and keep in PBS (DEPC).</li>
<li>Add 30 μL of enzyme mix 2 (refer to solution preparation) to sample.</li>
<li>Invert gently to mix.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
<ol start="36">
<li>Pipette up/down with P1000 to mechanically dissociate.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> In addition to pipetting up/down, the tissue can be dissociated mechanically with a fire-polished Pasteur pipette. 10–15 up and down trituration is enough.</div>
<ol start="37">
<li>Stop digestion and immunocytochemistry (ICC) block by adding 2 mL 5% FCS in PBS (DEPC).</li>
<li>Filter with cell strainer to eliminate clumps and debris. Collect cell suspension in a 50 mL falcon tube.</li>
<li>Rinse the 15 mL tube and strainer with PBS (DEPC).</li>
<li>Spin for 2000 rpm (400–600 × <i>g</i>) for 10 min in centrifuge in tool lab.</li>
<li>Take off supernatant.</li>
<li>Wash cells: resuspend cells in 2.0 mL PBS (DEPC), Spin for 2,000 rpm (400–600 × <i>g</i>) for 5 min at 4<sup>o</sup>C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cells may be in viscous form, but it should be well dissociated in Fixation/Permeabilization solution in next step.</div>
<ol start="43">
<li>For cell labeling with TBR2 antibody, repeat washing of cells by resuspending cells in 2.0 mL PBS (DEPC), spin for 2000 rpm (400–600 × <i>g</i>) for 5 min at 4°C</li>
<li>Carefully pipette to take off the supernatant, otherwise you might lose the cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Approximately 100 μL of residual volume typically remains.</div>
<ol start="45">
<li>Add 0.5 mL of Foxp3 Fixation/Permeabilization working solution to each tube and pipette up and down to dissolve the cells.</li>
<li>Incubate for 30 min or longer at 2°C–8°C or on ice (in the dark) with slight shaking.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Longer incubation time is possible for mouse samples, i.e., up to 18 h at 2°C–8°C in the dark.</div>
<ol start="47">
<li>Add 1 mL of 1× Permeabilization Buffer to each tube and centrifuge samples at 2000 rpm (400–600 × <i>g</i>) for 5 min at 4°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Pipette to take off the supernatant carefully, otherwise you might lose the cells.</div>
<div class="note">
<span class="note-title">Note:</span> Approximately 100 μL of residual volume typically remains.</div>
<ol start="48">
<li>Resuspend pellet in remaining volume of 1× Permeabilization Buffer (usually 100 μL after decanting) plus additional 350 μL 1× Permeabilization Buffer. Thus, we have about 450 μL.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the amount of cells, you may want to scale up or down rather than 450 μL.</div>
<ol start="49">
<li>Save a tube as blank/control tube.</li>
<li>Without washing, add the required amount of conjugated TBR2 antibody (1:200) to cells for detection of corresponding intracellular antigen.</li>
<li>Incubate for at least 30 min at 2°C–8°C (on ice) with slight shaking and no exposure to light.</li>
<li>Add 2 mL of 1× Permeabilization Buffer to each tube and centrifuge samples at 2000 rpm (400–600 × <i>g</i>) for 5 min at 2°C–8°C. Pipette with 1,000 μL, then 200 μL pipettor to carefully and completely take off the supernatant, to prevent losing the cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> You have to take off the supernatant completely. Otherwise, staining will have high background.</div>
<ol start="53">
<li>Repeat the previous washing step. Resuspend stained cells in 0.5 mL PBS plus 1× complete (Protein inhibitors).</li>
<li>Perform FACS as described above in steps 10 and 11 under Part I.</li>
<li>Centrifuge to collect the pellet of the sorted cells. 2.5 × 10<sup>6</sup> sorted TBR2+ cells should be used for each MS sample.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Samples should be stored in −80°C if the experiment will not be done immediately.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The protocol is expected to ensure effective cell and nuclei sorting to yield substantial amount of isolated materials with high enough quality for downstream probing such as RNA, chromatin, and protein analyses.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol for isolating and collecting nuclear material essentially precludes collection of cell type-specific cytosolic RNA or protein for downstream applications. While protocol for TBR2+ cell sorting for mass spectrometry analysis can get whole cell including cytosolic parts, it should be noted that it presents RNA degradation challenges due to the application of stringent reagents. Therefore, this part of the protocol is not suitable for RNA-based investigation. Because the TBR2- cell population contains many cell subtypes (e.g., radial glia progenitors, neuron, microglia…), it is not suitable for investigating cell subtype-specific molecular profile. Given the small size of the mouse embryonic brain, multiple mouse embryonic cortices need to be pooled (at least 5 per replicate) to achieve enough material for sorting TBR2+ nuclei.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>No visible pellet or viable nuclei after spinning down.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>While preparing nuclei for RNA, after staining, a good measure of viable nuclei is to see pellet after spinning down. If no pellet is seen, it is likely considerable amounts of nuclei might have been lost. This can be avoided by milder mechanical dissociation at the first step.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The isolated Tbr2 nuclear RNA is typically in very low amount, making concentration measurement quite troublesome.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Consider using Bioanalyzer for detecting trace amounts of RNA if the level cannot be detection by routine measurement methods. In difficult cases, it is advisable to re-purify RNA and elute into lowest volume possible (say 7 μl) to increase RNA concentration.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>No or indecipherable reads after ChIP experiment.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>It is good practice to perform qPCR for some known target regions as quality control.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Halder, R., Hennion, M., Vidal, R.O., Shomroni, O., Rahman, R.U., Rajput, A., Centeno, T.P., Van Bebber, F., Capece, V., Garcia Vizcaino, J.C., et al. (2016). DNA methylation changes in plasticity genes accompany the formation and maintenance of memory. Nat. Neurosci. <i>19</i>, 102-110. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00025-3/sref1">View at publisher</a></p>
<p id="bib2">Menon, V., Thomas, R., Ghale, A.R., Reinhard, C., and Pruszak, J. (2014). Flow cytometry protocols for surface and intracellular antigen analyses of neural cell types. J. Vis. Exp. 52241. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00025-3/sref2">View at publisher</a></p>
<p id="bib3">Narayanan, R., Pirouz, M., Kerimoglu, C., Pham, L., Wagener, R.J., Kiszka, K.A., Rosenbusch, J., Seong, R.H., Kessel, M., Fischer, A., et al. (2015). Loss of BAF (mSWI/SNF) complexes causes global transcriptional and chromatin state changes in forebrain development. Cell Rep. <i>13</i>, 1842-1854. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00025-3/sref3">View at publisher</a></p>
<p id="bib4">Nguyen, H., Kerimoglu, C., Pirouz, M., Pham, L., Kiszka, K.A., Sokpor, G., Sakib, M.S., Rosenbusch, J., Teichmann, U., Seong, R.H., et al. (2018). Epigenetic regulation by BAF complexes limits neural stem cell proliferation by suppressing Wnt signaling in late embryonic development. Stem Cell Reports <i>10</i>, 1734-1750. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00025-3/sref4">View at publisher</a></p>
<p id="bib5">Tuoc, T.C., Boretius, S., Sansom, S.N., Pitulescu, M.E., Frahm, J., Livesey, F.J., and Stoykova, A. (2013). Chromatin regulation by BAF170 controls cerebral cortical size and thickness. Dev. Cell <i>25</i>, 256-269. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00025-3/sref5">View at publisher</a></p>
<p id="bib6">Tuoc, T.C. and Stoykova, A. (2008). Trim11 modulates the function of neurogenic transcription factor Pax6 through ubiquitin-proteosome system. Genes Dev. <i>22</i>, 1972-1986. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00025-3/sref6">View at publisher</a></p>
<p id="bib7">Viesselmann, C., Ballweg, J., Lumbard, D., and Dent, E.W. (2011). Nucleofection and primary culture of embryonic mouse hippocampal and cortical neurons. J. Vis. Exp. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00025-3/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Tran Tuoc (<a href="mailto:Tran.Tuoc@ruhr-uni-bochum.de">Tran.Tuoc@ruhr-uni-bochum.de</a>)</p>
<h4>Materials Availability</h4>
<p>Materials are available upon request.</p>
<h4>Data and Code Availability</h4>
<p>The complete datasets generated with the protocol have not been deposited in a public repository because the study is currently under consideration for publication, but are available from the corresponding author on request.</p>
<h3>Acknowledgments</h3>
<p>We thank L. Pham for preparation of mouse cortical tissue and A.B. Tonchev for providing human fetal brain tissue. This work was supported by TU432/1-1, TU432/3-1, TU432/6-1 <a href="https://doi.org/10.13039/501100001659">DFG</a> grants (T.T.), Schram-Stiftung (T.T.). A.F. was supported by the DZNE, the ERC consolidator grant DEPICODE (648898), the SFB1286, and the DFG under Germany’s Excellence Strategy - EXC 2067/1390729940.</p>
<h3>Author Contributions</h3>
<p>M.S.S. and T.T. contributed to establishment of TBR2+ cell sorting protocols for RNA-seq, ChIP-seq, and MS, respectively. A.F. and H.P.N. provided research tools and contributed to discussions. G.S., M.S.S., and T.T. wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>