<article>
<section>
<h1 id="header">Protocol for preparation and measurement of intracellular and extracellular modified RNA using liquid chromatography-mass spectrometry</h1>
<p><time datetime="2021-10-01">Published: October 1, 2021</time></p>
<p>Akiko Ogawa<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> and Fan-Yan Wei<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Modomics Biology and Medicine, Institute of Development, Aging and Cancer (IDAC), Tohoku University, Sendai 980-8575, Japan</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:accogawa@tohoku.ac.jp">accogawa@tohoku.ac.jp</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:fanyan.wei.d3@tohoku.ac.jp">fanyan.wei.d3@tohoku.ac.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100848">10.1016/j.xpro.2021.100848</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>About 150 modifications have been identified in RNA species. Besides their regulatory roles in the intracellular gene expression, abundant modified RNA nucleosides are catabolized from RNA and released into extracellular fluids, which can impact extracellular signaling as ligands for receptors. Here, we describe a protocol to prepare samples from biological specimens, including cultured cells, extracellular fluid, and tissues, to measure both intracellular and extracellular RNA modifications using mass spectrometry.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00554-2/sref1">Ogawa et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Isolation of various RNA species from total RNA for analysis of modified nucleosides</li>
<li>Extraction of modified nucleosides-containing metabolites from biological fluids</li>
<li>Methods and parameters for measurement of modified nucleosides by LC-MS/MS</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1040-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="note">
<span class="note-title">Note:</span> Abbreviations: LMS, L-methionine sulfone; FBS, fetal bovine serum; LC-MS/MS, liquid chromatography-mass spectrometry</div>
<ol>
<li>RNA purification. Fresh biological samples should be processed immediately after collection as described below. Otherwise, samples should be snap-frozen in liquid nitrogen and stored at –80°C until preparation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To purify RNA from tissues or cells, a phenol-chloroform extraction is recommended to isolate all RNA species. If a silica spin column is used to extract RNA, confirm that the column will retain the desired RNA species. Conventional spin columns will only retain large RNA species (&gt; 200nt).</div>
<div class="note">
<span class="note-title">Note:</span> For cell culture samples, it is highly recommended to use phenol red-free medium to prevent contamination of the HPLC column and mass spectrometer with phenol red.</div>
<div class="note">
<span class="note-title">Note:</span> For tissue samples, record the wet weight after collection or before homogenization. To extract modified nucleosides, ∼100 mg of tissues is sufficient.</div>
<ol start="2">
<li>Prepare nuclease P1 solution and 200 mM filtered-HEPES solution (pH 7.0) to digest RNA. Dissolve 500 U nuclease P1 in 1 mL of 10 mM ammonium acetate (pH 5.3) to make a 0.5 U/μL solution.</li>
<li>Prepare standard nucleoside solutions diluted in water (for example, 10 nM, 100 nM, and 1 μM).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If available, stable isotope standard is recommended. Standard solutions should be aliquoted in small volumes and stored at temperatures lower than –20°C. Avoid repeated freeze–thaw cycles.</div>
<ol start="4">
<li>Precondition ultrafiltration device before use
<ol type="a">
<li>Add 300 μL ultrapure water to the centrifugal filter device.</li>
<li>Centrifuge at 14,000 × g at room temperature for 30 min.</li>
<li>Invert the inner filter device and perform a flash centrifugation to completely remove residual water.</li>
<li>The device is ready for addition of cell or tissue extracts.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> According to the manufacturer’s datasheet (<a href="https://www.merckmillipore.com/JP/en/product/Amicon-Ultra-0.5mL-Centrifugal-Filters-for-DNA-and-Protein-Purification-and-Concentration">https://www.merckmillipore.com/JP/en/product/Amicon-Ultra-0.5mL-Centrifugal-Filters-for-DNA-and-Protein-Purification-and-Concentration</a>,MM_NF-C82301#documentation), the Amicon Ultr-0.5 ultrafiltration membrane contains a trace amount of glycerin, which may interfere with the evaporation step and LC-MS/MS analysis. We commend performing this preconditioning step to minimize the unnecessary interference.</div>
<div class="note">
<span class="note-title">Note:</span> Perform preconditioning steps right before extraction process.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Microman E (Gilson) is recommended for accurately dispensing volatile liquids such as methanol.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Ultrapure Water (LC/MS grade)</td>
<td>Wako</td>
<td>Cat#214-01301</td>
</tr>
<tr>
<td>Methanol (LC/MS grade)</td>
<td>Wako</td>
<td>Cat#134-14523</td>
</tr>
<tr>
<td>Chloroform (HPLC grade)</td>
<td>Wako</td>
<td>Cat#033-08631</td>
</tr>
<tr>
<td>Acetonitrile (LC/MS grade)</td>
<td>Wako</td>
<td>Cat#018-19853</td>
</tr>
<tr>
<td>2-Propanol (LC/MS grade)</td>
<td>Wako</td>
<td>Cat#168-25531</td>
</tr>
<tr>
<td>10× Loading dye</td>
<td>Takara</td>
<td>Cat#9157</td>
</tr>
<tr>
<td>Agarose S</td>
<td>Nippon Gene</td>
<td>Cat#316-01191</td>
</tr>
<tr>
<td>Alkaline phosphatase (<i>E. coli</i> C75)</td>
<td>Takara</td>
<td>Cat#2120A</td>
</tr>
<tr>
<td>Nuclease P1</td>
<td>Wako</td>
<td>Cat#145-08221</td>
</tr>
<tr>
<td>L-Methionine sulfone</td>
<td>Alfa Aesar</td>
<td>Cat# A17027</td>
</tr>
<tr>
<td>Pseudouridine</td>
<td>Tokyo Chemical Industry</td>
<td>Cat#P2396; CAS:1445-07-4</td>
</tr>
<tr>
<td>5-Methylcytidine</td>
<td>Tokyo Chemical Industry</td>
<td>Cat#M1931; CAS:2140-61-6</td>
</tr>
<tr>
<td>2′-<i>O</i>-Methylcytidine</td>
<td>Tokyo Chemical Industry</td>
<td>Cat#M2317; CAS:2140-72-9</td>
</tr>
<tr>
<td>2′-<i>O</i>-Methyluridine</td>
<td>Tokyo Chemical Industry</td>
<td>Cat#M2290; CAS:2140-76-3</td>
</tr>
<tr>
<td>2′-<i>O</i>-Methylguanosine</td>
<td>Tokyo Chemical Industry</td>
<td>Cat#M2318; CAS:2140-71-8</td>
</tr>
<tr>
<td>2′-<i>O</i>-Methyladenosine</td>
<td>Tokyo Chemical Industry</td>
<td>Cat#M2291; CAS:2140-79-6</td>
</tr>
<tr>
<td>5,6-Dihydrouridine</td>
<td>Toronto Research Chemicals</td>
<td>Cat#D449668; CAS:5627-05-4</td>
</tr>
<tr>
<td>5-Formylcytidine</td>
<td>Toronto Research Chemicals</td>
<td>Cat#F698965; CAS:148608-53-1</td>
</tr>
<tr>
<td>
<i>N</i><sup>6</sup>, 2′-<i>O</i>-Dimethyladenosine</td>
<td>Toronto Research Chemicals</td>
<td>Cat#D447415; CAS:57817-83-1</td>
</tr>
<tr>
<td>3-Methyluridine</td>
<td>Sigma-Aldrich</td>
<td>Cat#M4129; CAS:2140-69-4</td>
</tr>
<tr>
<td>7-Methylguanosine</td>
<td>Sigma-Aldrich</td>
<td>Cat#M0627; CAS:20244-86-4</td>
</tr>
<tr>
<td>1-Methylguanosine</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat#sc-500889; CAS:2140-65-0</td>
</tr>
<tr>
<td>1-Methylinosine</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat#sc-483758; CAS:2140-73-0</td>
</tr>
<tr>
<td>2′-<i>O</i>-Methylinosine</td>
<td>Santa Cruz Biotechnology</td>
<td>Cat#sc-283498; CAS:3881-21-8</td>
</tr>
<tr>
<td>
<i>N</i><sup>2</sup>-Methylguanosine</td>
<td>Biosynth Carbosynth</td>
<td>Cat#NM35522; CAS:2140-77-4</td>
</tr>
<tr>
<td>1-Methyladenosine</td>
<td>Cayman Chemical</td>
<td>Cat#16937; CAS:15763-06-1</td>
</tr>
<tr>
<td>
<i>N</i><sup>6</sup>-Methyladenosine</td>
<td>Abcam</td>
<td>Cat#ab145715; CAS:1867-73-8</td>
</tr>
<tr>
<td>
<i>N</i><sup>4</sup>-Acetylcytidine</td>
<td>Combi-Blocks</td>
<td>Cat#QB-9019; CAS:3768-18-1</td>
</tr>
<tr>
<td>
<i>N</i><sup>2</sup>,<i>N</i><sup>2</sup>-Dimethylguanosine</td>
<td>MedChem Express</td>
<td>Cat#HY-113137; CAS:2140-67-2</td>
</tr>
<tr>
<td>
<i>N</i><sup>6</sup>,<i>N</i><sup>6</sup>-dimethyladenosine</td>
<td>MedChem Express</td>
<td>Cat#HY-101984; CAS:2620-62-4</td>
</tr>
<tr>
<td>RNA Clean &amp; Concentrator-5</td>
<td>Zymo Research</td>
<td>Cat#R1015</td>
</tr>
<tr>
<td>Oligotex-dT30&lt;Super&gt; mRNA Purification Kit (From Total RNA)</td>
<td>Takara</td>
<td>Cat#9086</td>
</tr>
<tr>
<td>Zymoclean Gel RNA Recovery Kit</td>
<td>Zymo Research</td>
<td>Cat#R1011</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Submerge-Mini (electrophoresis device)</td>
<td>ATTO</td>
<td>Cat#WSE-1710</td>
</tr>
<tr>
<td>UV transilluminator</td>
<td>Bio Craft</td>
<td>Cat#CI-110B</td>
</tr>
<tr>
<td>TissueRuptor Disposable Probes</td>
<td>QIAGEN</td>
<td>Cat#990890</td>
</tr>
<tr>
<td>TissueRuptor II</td>
<td>QIAGEN</td>
<td>Cat#9002754</td>
</tr>
<tr>
<td>Amicon Ultra-0.5 Centrifugal Filter Devices</td>
<td>Millipore</td>
<td>Cat#UFC500396</td>
</tr>
<tr>
<td>Rotating incubator</td>
<td>Taitec</td>
<td>Cat#RT-50</td>
</tr>
<tr>
<td>SpeedVac vacuum concentrators</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#SPD1010</td>
</tr>
<tr>
<td>High Performance Liquid Chromatograph Triple Quadrupole Mass Spectrometer</td>
<td>Shimadzu</td>
<td>Cat#LCMS-8050</td>
</tr>
<tr>
<td>Inertsil ODS-3 column</td>
<td>GL Sciences</td>
<td>Cat#5020-84655</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>[Nuclease P1 solution]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease P1</td>
<td>0.5 U/μL</td>
<td>500 U</td>
</tr>
<tr>
<td>Ammonium acetate (pH 5.3)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>1</b> <b>mL</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Storage: Dispense the solution into aliquots after the initial dissolution and store for up to 1 year at –20°C. Avoid repeated freeze and thaw.</p>
<table id="undtbl2">
<caption>[Extraction buffer 1]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>LMS (1mM)</td>
<td>1 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>99.7% methanol</td>
<td>99.7%</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Storage: Store for up to 6 months at –80°C.</p>
<table id="undtbl3">
<caption>[Extraction buffer 2]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>LMS (1mM)</td>
<td>1 μM</td>
<td>45 μL</td>
</tr>
<tr>
<td>99.7% methanol</td>
<td>n/a</td>
<td>25 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>10 mL</td>
</tr>
<tr>
<td>Chloroform</td>
<td>n/a</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Storage: Store for up to 6 months at –80°C.</p>
<table id="undtbl4">
<caption>[Extraction buffer 3]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>LMS (1mM)</td>
<td>1 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>99.7% methanol</td>
<td>n/a</td>
<td>25 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Storage: Store for up to 6 months at –80°C.</p>
<table id="undtbl5">
<caption>[1% TAE-Agarose gel for the extraction of rRNA]</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Agarose</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>50<b>×</b> TAE Buffer</td>
<td>1<b>×</b>
</td>
<td>2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>98 mL</td>
</tr>
<tr>
<td>Ethidium bromide solution (10 mg/mL)</td>
<td>300 ng/mL</td>
<td>3 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>100</b> <b>mL</b>
</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Before use, treat all equipment to inactivate RNases, such as by using RNaseZAP.</div>
<p>Storage: Do not store the gel. Use the gel immediately after solifification.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Many popular ethidium bromide alternatives, such as SYBR Gold (Thermo Fischer Scientific) can be used instead.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h for step a</div>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h for step b</div>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h for step c</div>
<div class="timing">
<span class="timing-title">Timing:</span> 3–24 h for step d</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h for step e</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h for step f</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h for step g</div>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h for step h</div>
<div class="timing">
<span class="timing-title">Timing:</span> 33 min/sample for step 2</div>
<ol>
<li>Sample preparation
<ol type="a">
<li>Isolation of small RNAs This section describes procedures to isolate small RNAs from total RNA for enzymatic digestion and LC-MS/MS analysis.
<ol type="i">
<li>Purify total RNA from cells or tissues by using a phenol-containing reagent, such as TRIzol (Thermo Fisher), according to the manufacturer's instruction (<a href="https://tools.thermofisher.com/content/sfs/manuals/trizol_reagent.pdf">https://tools.thermofisher.com/content/sfs/manuals/trizol_reagent.pdf</a>). Use of a reagent similar to TRIzol will isolate cell debris and protein isolated to the acidic phenol-containing layer whereas the aqueous layer will contain the desired total RNA.</li>
<li>Resuspend RNA in ultrapure water to a concentration of 1 μg/μL. More than 100 μg RNA is needed to perform all downstream procedures.</li>
<li>Use an RNA clean &amp; concentrator kit to separate small (17–200 nt) and large (&gt; 200 nt) RNA species from total RNA, following the manufacturer’s instruction (<a href="https://files.zymoresearch.com/protocols/_r1013_r1014_r1015_r1016_rna_clean_concentrator-5.pdf">https://files.zymoresearch.com/protocols/_r1013_r1014_r1015_r1016_rna_clean_concentrator-5.pdf</a>). Briefly, add the appropriate volume of RNA Binding Buffer and ethanol to each sample. Transfer the mixture to a silica-based Zymo-Spin IC Column and centrifuge. Large RNAs are retained in the column and small RNAs are in the flow-through. Mix the flow-through with ethanol and load the mixture to a new Zymo-Spin IC Column and centrifuge. After several cleanup procedures with wash buffer, large and small RNAs can be eluted from the column, respectively.
<div class="note">
<span class="note-title">Note:</span> The yield of small RNAs is usually less than one-fourth the yield of large RNAs.</div>
<div class="note">
<span class="note-title">Note:</span> Alternatively, other commercially available RNA extraction kit which can isolate small RNAs can be used such as miRNeasy Mini Kit (QIAGEN).</div>
</li>
<li>Measure the concentration of both small RNAs and large RNAs. If the concentration of small RNAs is more than one-fourth that of large RNAs, it is likely that the small RNAs are contaminated by large RNAs. Proceed to step d i. for digestion. RNA can be stored at −80°C if it is not immediately subjected to digestion.</li>
</ol>
</li>
<li>Isolation of mRNAs This section describes procedures to isolate mRNA from total RNA for enzymatic digestion and LC-MS/MS analysis.
<ol type="i">
<li>Purify polyA<sup>+</sup> mRNA from either large RNA or total RNA by using the Oligotex-dT30&lt;Super&gt; mRNA Purification Kit according to the manufacturer’s instruction (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/rna-purification/mrna/oligotex-mrna-kits/">https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/rna-purification/mrna/oligotex-mrna-kits/</a>). Briefly, mix RNA with Binding Buffer and Oligotex-dT30 magnetic beads. Heat-denature the RNA at 70°C for 3 min and then let stand at room temperature for 10 min to hybridize magnetic beads and mRNA. After centrifugation, remove the supernatant and suspend the magnetic particles with wash buffer, then transfer the mixture to a spin column and centrifuge the column. Repeat the washing step and polyA<sup>+</sup> mRNA can be eluted from the column with RNase free dH<sub>2</sub>O.</li>
<li>Measure the concentration of mRNA, which should be approximately 1%–2% of the input concentration. Proceed to step d i. for digestion. RNA can be stored at −80°C if it is not directly subjected to digestion.
<div class="critical">
<span class="critical-title">Critical:</span> Do not exceed the maximum capacities of either spin columns or Oligo-dT beads. If excessive RNA is applied, mRNA fraction will be easily contaminated with rRNA and tRNA.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If contamination is suspected, it is recommended to perform agarose gel electrophoresis to check the purity. 1% TAE-agarose gel containing ethidium bromide and RNA denature with formamide can be used (as described below). Purified mRNA fraction can be subject to a second round of purification if contamination is confirmed.</div>
</li>
</ol>
</li>
<li>Isolation of 18s rRNA and 28s rRNA<br> This section describes procedures to isolate 28S rRNA and 18S rRNA from large RNA or total RNA by agarose gel extraction.
<ol type="i">
<li>Prepare 1% TAE-agarose gel. Measure 1 g agarose. Mix agarose powder with 100 mL 1× TAE (2 mL RNAse-free 50× TAE + 98 mL ultrapure water) in a microwavable flask. Microwave for 1–3 min until the agarose is completely dissolved. After the complete melt of agarose, let agarose solution cool down to about 50°C. Add 3 μL ethidium bromide solution (10 mg/mL) and swirl. Pour the agarose solution into an assembled gel tray and cover the tray with saran wrap. Let the agarose gel solidify for about 60 min at room temperature.</li>
<li>Mix 10 μL RNase-free formamide with 10 μL total RNA or large RNA (≤5 μg RNA), and denature samples by heating at 60°C for 15 min followed by quick cooling on ice.</li>
<li>Add 3 μL of 10× loading buffer composed of 50% glycerol, 0.9% SDS, and 0.05% Bromophenol Blue.</li>
<li>Electrophorese the entire sample (23 μL) with voltage 100V for 20 min (until the dye has migrated two-thirds of the length of the gel). RNA molecular size marker such as DynaMarker (BioDynamics Laboratory) can be used to confirm the size of rRNAs.</li>
<li>Visualize the gel on a UV transilluminator and excise the 28S or 18S rRNA bands using razor blade. Usually RNA shows a 28S rRNA band at 4.5 kb and an 18S rRNA band at 1.9 kb, and small RNA species that possibly contained tRNA and other small RNA species. For a typical photograph of the gel, please refer to the Figure 4G of <a href="#bib1">Ogawa et al. (2021)</a>.</li>
<li>Extract 28S or 18S rRNAs from the excised gel slices by using the Zymoclean Gel RNA Recovery Kit. When using 5 μg RNA as starting materials, the yield of 28S rRNA should be &gt;900 ng; the yield of 18S rRNA &gt;450 ng. The ratio of 28S rRNA to 18S rRNA should be approximately 2:1. Proceed to step d i. for digestion. RNA can be stored at −80°C if it is not directly subject to digestion.</li>
</ol>
</li>
<li>RNA digestion
<ol type="i">
<li>Mix up to 2.5 μg of each RNA sample with 2 μL nuclease P1 solution (0.5 U/μL), 0.5 μL bacteria alkaline phosphatase (BAP), and 2.5 μL of 200 mM HEPES (pH 7.0), and bring the total volume to 25 μL with ultrapure water.</li>
<li>Digest RNA for 3 h at 37°C. For 2′-O-methylated nucleosides, which are resistant to RNase (<a href="#bib2">Sproat et al., 1989</a>), prolonged digestion (up to 24 h) can increase the yield. For a prolonged digestion, use a PCR instrument to prevent evaporation.</li>
<li>After digestion, the digested samples should be immediately subjected to LC-MS/MS analysis.</li>
</ol>
</li>
<li>Extracellular nucleosides from cultuerd cells This section describes extraction of metabolites containing modified nucleosides from supernatant of cell culture.
<ol type="i">
<li>Thaw cell culture medium (if frozen) and gently vortex.</li>
<li>Transfer 50 μL to a 1.5 mL tube.</li>
<li>Add 250 μL of pre-cooled extraction buffer 1 and gently vortex.</li>
<li>Rotate the mixture using a rotator for 15 min at 4°C.</li>
<li>Centrifuge at 15,000 × g for 10 min at 4°C.</li>
<li>Transfer 250 μL supernatant to a new 1.5 mL tube.</li>
<li>Add 100 μL ultrapure water to the sample and gently vortex.</li>
<li>Transfer the mixture into a preconditioned ultrafiltration tube.</li>
<li>Centrifuge at 16,000 × g for 1 h at 4°C.</li>
<li>Transfer the entire ultrafiltrate volume into a new 1.5 mL tube and puncture the lid by using a 20-gauge needle.</li>
<li>Evaporate samples by using a SpeedVac Vacuum Concentrator.</li>
<li>Dissolve each dried sample in 25 μL ultrapure water.</li>
<li>Centrifuge at 16,000 × g for 10 min at 10°C.</li>
<li>Transfer 20 μL of the supernatant into a new tube suitable for LC-MS/MS measurement.
<div class="note">
<span class="note-title">Note:</span> Samples should preferably be subjected to LC-MS/MS analysis immediately after preparation, but samples can be kept frozen below –20°C after evaporation (step xi.). In this case, cover the punctured lid with parafilm to avoid contamination.</div>
</li>
</ol>
</li>
<li>Intracellular nucleosides from cultured cells This section describes procedures to extract intracellular metabolites containing modified nucleosides from HEK 293A cells plated in 6-well plates at a density of 400,000–500,000 cells per well.
<ol type="i">
<li>Aspirate supernatant from cell culture dish.</li>
<li>Wash cells with ice-cold 5% mannitol or PBS, and completely aspirate the fluid.</li>
<li>Add 500 μL pre-chilled extraction buffer 1 and incubate for 10 min on ice.</li>
<li>Transfer the liquid to a 1.5 mL tube and gently vortex.</li>
<li>Add 400 μL pre-chilled chloroform and vortex.</li>
<li>Add 200 μL pre-chilled ultrapure water and vortex.</li>
<li>Further mix the tube contents on a rotating mixer for 15 min at 4°C.</li>
<li>Centrifuge at 10,000 × g for 5 min at 4°C.</li>
<li>Transfer 350 μL of the top layer into a preconditioned ultrafiltration tube.</li>
<li>Centrifuge at 16,000 × g for 1 h at 4°C.</li>
<li>Transfer the entire ultrafiltrate volume into a new 1.5 mL tube and puncture the lid by using a 20-gauge needle.</li>
<li>Evaporate samples by using a SpeedVac Vacuum Concentrator.</li>
<li>Dissolve each dried sample in 25 μL ultrapure water.</li>
<li>Centrifuge at 16,000 × g for 10 min at 10°C.</li>
<li>Transfer 20 μL of the supernatant into a new tube suitable for LC-MS/MS measurement.
<div class="note">
<span class="note-title">Note:</span> See section 4 in “<a href="#before-you-begin">before you begin</a>” for the procedures of preconditioning ultrafiltration device.</div>
<div class="note">
<span class="note-title">Note:</span> Samples should preferably be subjected to LC-MS/MS analysis immediately after preparation, but samples can be kept frozen below –20°C after evaporation (step f xii.). In this case, cover the punctured lid with parafilm to avoid contamination.</div>
</li>
</ol>
</li>
<li>Nucleosides in mammalian extracellular fluid This section describes extraction of metabolites containing modified nucleosides from extracellular fluids such as plasma, urine, and aqueous humor.
<ol type="i">
<li>Thaw biological fluids (if frozen) and gently vortex.</li>
<li>Transfer 25 μL of each sample into a 1.5 mL tube.</li>
<li>Add 450 μL pre-chilled extraction buffer 2 (water: methanol: chloroform = 1 : 2.5 : 1) and gently vortex.</li>
<li>Mix the tube contents on a rotator for 15 min at 4°C.</li>
<li>Centrifuge at 16,000 × g for 3 min at 4°C.</li>
<li>Transfer 315 μL of the top layer into a new 1.5 mL tube.</li>
<li>Add 140 μL ultrapure water to the sample and gently vortex.</li>
<li>Centrifuge at 16,000 × g for 3 min at 4°C.</li>
<li>Transfer 300 μL supernatant into a preconditioned ultrafiltration tube.</li>
<li>Centrifuge at 16,000 × g for 1 h at 4°C.</li>
<li>Transfer the entire ultrafiltrate volume into a new 1.5 mL tube and puncture the lid by using a 20-gauge needle.</li>
<li>Evaporate samples by using a SpeedVac Vacuum Concentrator.</li>
<li>Dissolve each dried sample in 25 μL ultrapure water.</li>
<li>Centrifuge at 16,000 × g for 10 min at 10°C.</li>
<li>Transfer 20 μL of the supernatant into a new tube suitable for LC-MS/MS measurement.
<div class="note">
<span class="note-title">Note:</span> Samples should preferably be subjected to LC-MS/MS analysis immediately after preparation, but samples can be kept frozen below –20°C after evaporation (step xii.). In this case, cover the punctured lid with parafilm to avoid contamination.</div>
</li>
</ol>
</li>
<li>Nucleosides in mammalian tissues This section describes extraction of metabolites containing modified nucleosides from tissues.
<ol type="i">
<li>Add 1.2 mL pre-chilled extraction buffer 3 (water: methanol = 1 : 1) to each tissue sample (∼ 100 mg) and homogenize in an ice bucket by using a TissueRuptor II with a disposable probe.</li>
<li>Transfer entire homogenate into a new 1.5 mL tube and centrifuge at 16,000 × g for 15 min at 4°C.</li>
<li>Transfer 500 μL supernatant into a preconditioned ultrafiltration tube.</li>
<li>Centrifuge at 16,000 × g for 1 h at 4°C.</li>
<li>Transfer the entire ultrafiltrate volume into a new 1.5 mL tube and puncture the lid by using a 20-gauge needle.</li>
<li>Evaporate samples by using a SpeedVac Vacuum Concentrator.</li>
<li>Dissolve each dried sample in 25 μL ultrapure water.</li>
<li>Centrifuge at 16,000 × g for 10 min at 10°C.</li>
<li>Transfer 20 μL of the supernatant into a new tube suitable for LC-MS/MS measurement.
<div class="note">
<span class="note-title">Note:</span> If a sample is too big or too hard to homogenize, chop the sample to reduce the size before adding extraction buffer 3.</div>
<div class="note">
<span class="note-title">Note:</span> Samples should preferably be subjected to LC-MS/MS analysis immediately after preparation, but samples can be kept frozen below –20°C after evaporation (step h vi.). In this case, cover the punctured lid with parafilm to avoid contamination.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Measurement of nucleosides by LC-MS/MS<br> This section describes measurement of modified/unmodified nucleosides after preparation of the samples using LCMS-8050 (Shimadzu) with Inertsil ODS-3 Column (GL science). Obtained data were analyzed using LabSolutions software (Shimadzu). These parameters have been developed and optimized for the Shimadzu LCMS-8050 mass spectrometer equipped with binary pump, autosampler and electrospray ionization (ESI) using Inertsil ODS-3 Column (reversed phase C18 Column, particle size, 2 μm, I.D., 2.1 mm, length, 150mm, GL science), but these parameters can be adapted for other instruments.
<ol type="a">
<li>Set up parameters for liquid chromatography.</li>
<li>Flow rate = 0.4 mL/min</li>
<li>Mobile phase: Buffer A: 5 mM ammonium acetate (pH 5.3) in water; Buffer B: 60% acetonitrile in water</li>
<li>The LC time gradient was created using two Nexera LC-30AD solvent delivery unit (Shimadzu) as follows:
<table id="undtbl6">
<thead>
<tr>
<th>Time (min)</th>
<th>Buffer B (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>0</td>
</tr>
<tr>
<td>10</td>
<td>22.1</td>
</tr>
<tr>
<td>15</td>
<td>63.1</td>
</tr>
<tr>
<td>17</td>
<td>100</td>
</tr>
<tr>
<td>22</td>
<td>100</td>
</tr>
<tr>
<td>23</td>
<td>0.6</td>
</tr>
<tr>
<td>33</td>
<td>0.6</td>
</tr>
</tbody>
</table>
</li>
<li>Set up the temperature parameters for mass spectrometer:
<ol type="i">
<li>Interface temperature = 300°C</li>
<li>Desolvation line temperature = 250°C</li>
<li>Heat block temperature = 400°C</li>
</ol>
</li>
<li>Nitrogen gas was supplied by an N<sub>2</sub> supplier Model T24FD (System Instruments) for nebulization and drying, and argon gas was used for collision-induced dissociation. Nebulizing gas flow is 3.0 L/min, heating gas flow is 10.0 L/min, and drying gas flow is 10.0 L/min.</li>
<li>Set capillary needle in the interface and needle protrusion to 1.0 mm.</li>
<li>The MS/MS conditions for each target is optimized using the automated MRM optimization procedure in LabSolutions software (Shimadzu). The MRM transitions as shown in the following table:
<table id="undtbl7">
<thead>
<tr>
<th>Nucleosides</th>
<th>Precursor ion (m/z)</th>
<th>Product ion (m/z)</th>
<th>Retention time</th>
<th>Q1 pre-Bias (V)</th>
<th>CE</th>
<th>Q3 pre-Bias (V)</th>
<th>Polarity</th>
</tr>
</thead>
<tbody>
<tr>
<td>LMS</td>
<td>182</td>
<td>56</td>
<td>1.078</td>
<td>16</td>
<td>21</td>
<td>24</td>
<td>+</td>
</tr>
<tr>
<td>D</td>
<td>247</td>
<td>115</td>
<td>1.771</td>
<td>12</td>
<td>9</td>
<td>27</td>
<td>+</td>
</tr>
<tr>
<td>Y</td>
<td>245</td>
<td>209</td>
<td>1.78</td>
<td>14</td>
<td>10</td>
<td>23</td>
<td>+</td>
</tr>
<tr>
<td>τm<sup>5</sup>U</td>
<td>380</td>
<td>124</td>
<td>1.802</td>
<td>19</td>
<td>25</td>
<td>18</td>
<td>-</td>
</tr>
<tr>
<td>C</td>
<td>244</td>
<td>112</td>
<td>2.168</td>
<td>12</td>
<td>15</td>
<td>22</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>3</sup>C</td>
<td>258</td>
<td>126</td>
<td>2.23</td>
<td>11</td>
<td>14</td>
<td>21</td>
<td>+</td>
</tr>
<tr>
<td>acp<sup>3</sup>U</td>
<td>346</td>
<td>214</td>
<td>2.371</td>
<td>20</td>
<td>18</td>
<td>24</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>1</sup>A</td>
<td>282</td>
<td>150</td>
<td>2.481</td>
<td>10</td>
<td>20</td>
<td>30</td>
<td>+</td>
</tr>
<tr>
<td>U</td>
<td>245</td>
<td>113</td>
<td>2.716</td>
<td>12</td>
<td>11</td>
<td>20</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>7</sup>G</td>
<td>298</td>
<td>166</td>
<td>2.956</td>
<td>12</td>
<td>16</td>
<td>30</td>
<td>+</td>
</tr>
<tr>
<td>τm<sup>5</sup>s<sup>2</sup>U</td>
<td>396</td>
<td>124</td>
<td>3.436</td>
<td>19</td>
<td>25</td>
<td>18</td>
<td>-</td>
</tr>
<tr>
<td>m<sup>3</sup>U</td>
<td>259</td>
<td>127</td>
<td>3.445</td>
<td>14</td>
<td>11</td>
<td>26</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>5</sup>C</td>
<td>258</td>
<td>126</td>
<td>3.454</td>
<td>11</td>
<td>14</td>
<td>21</td>
<td>+</td>
</tr>
<tr>
<td>I</td>
<td>269</td>
<td>137</td>
<td>3.654</td>
<td>14</td>
<td>21</td>
<td>28</td>
<td>+</td>
</tr>
<tr>
<td>Um</td>
<td>259</td>
<td>113</td>
<td>3.706</td>
<td>13</td>
<td>11</td>
<td>21</td>
<td>+</td>
</tr>
<tr>
<td>Cm</td>
<td>258</td>
<td>112</td>
<td>3.745</td>
<td>13</td>
<td>13</td>
<td>21</td>
<td>+</td>
</tr>
<tr>
<td>G</td>
<td>284</td>
<td>152</td>
<td>3.835</td>
<td>14</td>
<td>15</td>
<td>30</td>
<td>+</td>
</tr>
<tr>
<td>f<sup>5</sup>C</td>
<td>272</td>
<td>140</td>
<td>4.288</td>
<td>12</td>
<td>15</td>
<td>22</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>1</sup>I</td>
<td>283</td>
<td>151</td>
<td>4.946</td>
<td>20</td>
<td>18</td>
<td>29</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>1</sup>G</td>
<td>298</td>
<td>166</td>
<td>5.001</td>
<td>12</td>
<td>16</td>
<td>30</td>
<td>+</td>
</tr>
<tr>
<td>Im</td>
<td>283</td>
<td>137</td>
<td>5.027</td>
<td>15</td>
<td>12</td>
<td>25</td>
<td>+</td>
</tr>
<tr>
<td>Gm</td>
<td>298</td>
<td>152</td>
<td>5.069</td>
<td>13</td>
<td>12</td>
<td>30</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>2</sup>G</td>
<td>298</td>
<td>166</td>
<td>5.353</td>
<td>12</td>
<td>16</td>
<td>30</td>
<td>+</td>
</tr>
<tr>
<td>ac<sup>4</sup>C</td>
<td>286</td>
<td>154</td>
<td>5.476</td>
<td>15</td>
<td>13</td>
<td>28</td>
<td>+</td>
</tr>
<tr>
<td>A</td>
<td>268</td>
<td>136</td>
<td>5.512</td>
<td>12</td>
<td>17</td>
<td>24</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>2</sup><sub>2</sub>G</td>
<td>312</td>
<td>180</td>
<td>6.552</td>
<td>11</td>
<td>18</td>
<td>11</td>
<td>+</td>
</tr>
<tr>
<td>Am</td>
<td>282</td>
<td>136</td>
<td>6.89</td>
<td>15</td>
<td>16</td>
<td>26</td>
<td>+</td>
</tr>
<tr>
<td>t<sup>6</sup>A</td>
<td>413</td>
<td>281</td>
<td>6.916</td>
<td>21</td>
<td>15</td>
<td>20</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>6</sup>A</td>
<td>282</td>
<td>150</td>
<td>7.855</td>
<td>10</td>
<td>20</td>
<td>30</td>
<td>+</td>
</tr>
<tr>
<td>ms<sup>2</sup>t<sup>6</sup>A</td>
<td>459</td>
<td>327</td>
<td>9.157</td>
<td>25</td>
<td>14</td>
<td>25</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>6</sup>Am</td>
<td>296</td>
<td>150</td>
<td>9.73</td>
<td>12</td>
<td>21</td>
<td>29</td>
<td>+</td>
</tr>
<tr>
<td>m<sup>6</sup><sub>2</sub>A</td>
<td>296</td>
<td>164</td>
<td>11.176</td>
<td>12</td>
<td>21</td>
<td>29</td>
<td>+</td>
</tr>
<tr>
<td>i<sup>6</sup>A</td>
<td>336</td>
<td>204</td>
<td>14.548</td>
<td>10</td>
<td>15</td>
<td>13</td>
<td>+</td>
</tr>
<tr>
<td>ms<sup>2</sup>i<sup>6</sup>A</td>
<td>382</td>
<td>182</td>
<td>16.939</td>
<td>17</td>
<td>29</td>
<td>12</td>
<td>+</td>
</tr>
</tbody>
</table>
</li>
<li>Set samples into the rack in autosampler (SIL-30AC, Shimadzu), and measure the nucleoside levels. We recommend running standard nucleoside solution before actual samples for quantification of results and validation of the sensitivity of the mass spectrometer. The representative mass chromatographs of extracellular fluids are shown below. The representative peak area of individual standard modified or unmodified nucleoside is also listed below.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Modified nucleosides and unmodified nucleosides in the prepared sample are trapped in C18 reverse phase column and eluted from the column at different time points in response to the increase in acetonitrile concentration. Each nucleoside is introduced into the mass spectrometer and detected using the specific MRM parameter. The mass chromatograph of each nucleoside is generated by the software and the area of the peak can be used to calculate the concentration of the −ucleoside.</p>
<p>Here we show the representative peak areas of standard nucleosides (<a href="#fig1">Figures 1</a>, <a href="#fig2">2</a>, and <a href="#fig3">3</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1040-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Representative LC-MS/MS peaks of human plasma</div>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1040-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative LC-MS/MS peaks of human urine</div>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1040-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative LC-MS/MS peaks of human aqueous humor</div>
</figcaption>
</figure>
<table id="undtbl8">
<thead>
<tr>
<th> </th>
<th>A</th>
<th>m<sup>1</sup>A</th>
<th>m<sup>6</sup>A</th>
<th>t<sup>6</sup>A</th>
<th>ms<sup>2</sup>t<sup>6</sup>A</th>
<th>m<sup>6</sup><sub>2</sub>A</th>
<th>i<sup>6</sup>A</th>
<th>ms<sup>2</sup>i<sup>6</sup>A</th>
</tr>
</thead>
<tbody>
<tr>
<td>10 nM</td>
<td>295,277</td>
<td>213,584</td>
<td>123,372</td>
<td>43,402</td>
<td>10,679</td>
<td>1,026,826</td>
<td>170,483</td>
<td>94,871</td>
</tr>
<tr>
<td>100 nM</td>
<td>2,683,129</td>
<td>1,908,278</td>
<td>1,340,535</td>
<td>360,191</td>
<td>130,869</td>
<td>9,599,842</td>
<td>1,685,403</td>
<td>1,272,560</td>
</tr>
</tbody>
</table>
<table id="undtbl9">
<thead>
<tr>
<th> </th>
<th>m<sup>6</sup>Am</th>
<th>Am</th>
<th>I</th>
<th>m<sup>1</sup>I</th>
<th>Im</th>
<th>U</th>
<th>Y</th>
<th>D</th>
</tr>
</thead>
<tbody>
<tr>
<td>10 nM</td>
<td>835,757</td>
<td>277,998</td>
<td>70,039</td>
<td>270,216</td>
<td>65,669</td>
<td>1,501</td>
<td>6,568</td>
<td>24,642</td>
</tr>
<tr>
<td>100 nM</td>
<td>8,545,363</td>
<td>2,651,182</td>
<td>765,996</td>
<td>2,710,965</td>
<td>756,950</td>
<td>18,044</td>
<td>48,492</td>
<td>150,425</td>
</tr>
</tbody>
</table>
<table id="undtbl10">
<thead>
<tr>
<th> </th>
<th>m<sup>3</sup>U</th>
<th>acp<sup>3</sup>U</th>
<th>Um</th>
<th>G</th>
<th>m<sup>1</sup>G</th>
<th>m<sup>2</sup>G</th>
<th>m<sup>7</sup>G</th>
<th>m<sup>2</sup><sub>2</sub>G</th>
</tr>
</thead>
<tbody>
<tr>
<td>10nM</td>
<td>5,613</td>
<td>2,142</td>
<td>1,708</td>
<td>190,829</td>
<td>407,747</td>
<td>304,146</td>
<td>479,999</td>
<td>423,290</td>
</tr>
<tr>
<td>100 nM</td>
<td>59,429</td>
<td>33,564</td>
<td>17,355</td>
<td>1,275,513</td>
<td>3,684,919</td>
<td>2,690,115</td>
<td>4,507,921</td>
<td>3,805,035</td>
</tr>
</tbody>
</table>
<table id="undtbl11">
<thead>
<tr>
<th> </th>
<th>Gm</th>
<th>C</th>
<th>m<sup>3</sup>C</th>
<th>m<sup>5</sup>C</th>
<th>ac<sup>4</sup>C</th>
<th>f<sup>5</sup>C</th>
<th>Cm</th>
</tr>
</thead>
<tbody>
<tr>
<td>10 nM</td>
<td>173,000</td>
<td>89,806</td>
<td>556,717</td>
<td>199,492</td>
<td>68,001</td>
<td>27,231</td>
<td>102,212</td>
</tr>
<tr>
<td>100 nM</td>
<td>1,531,681</td>
<td>750,689</td>
<td>5,879,203</td>
<td>1,599,454</td>
<td>577,325</td>
<td>233,768</td>
<td>905,150</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> These values depend on LC-MS/MS type and manufacture, and on column type and usage status. Optimization is needed for each LC-MS/MS machine.</div>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Quantification of modified nucleosides in biological samples can be performed using standard nucleosides with known concentration. Prepare a serial dilution of a standard nucleoside, followed by mass spectrometry analysis. Use the peak areas of the standard nucleosides to generate calibration curve. For any given sample, use the peak area to fit this calibration curve to obtain the concentration. For modified nucleosides in RNA, the unit can be nmol / ng RNA; for modified nucleosides in biological samples, the unit can be nmol / μL. Representative calibration curve of i<sup>6</sup>A is shown in <a href="#fig4">Figure 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1040-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Example of calibration curve for quantitative analysis of i<sup>6</sup>A</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol only covers a subset of the RNA modifications that have been identified so far. Matrix effect is one of the main concerns in quantitative LC-MS/MS analysis, especially when using biological fluid samples (such as serum and urine). It is strongly recommended to use stable isotope-labeled nucleosides as internal standards for reliable measurement and for comparing samples.</p>
<p>This protocol was developed to screen all of the modified nucleosides that are listed above. If only a few of these nucleosides are being studied, then optimize the extraction and detection conditions (which are determined by each nucleoside’s polarity) accordingly.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>RNA yield is low (steps 1-a, 1-b, and 1-c).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Low yield is caused either by overloading the column (which can clog the column or prevent optimal RNA binding) or by RNase contamination. Confirm the binding capacity of the column before use. Also use nuclease-free grade tubes and tips. Wear masks while handling RNA and use an RNase-cleaning agent. RNA contents vary between tissues and cell types. If you cannot improve the yield by addressing these concerns, then increase the amount of starting material to obtain more RNA.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Evaporation process takes too much time (steps 1-e-xi, 1-f-xii, 1-g-xii, and 1-h-vi).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>The primary cause of this problem is the residual glycerin of ultrafiltration membrane. Confirm the preconditioning steps are done right before the extraction process. Extend preconditioning centrifugation time from 30 min to 60 min.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Results are not stable when the sample is repeatedly measured (step 2-i).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>This problem can be caused by HPLC malfunction, such as the abnormal sample intake or leakage. Prepare a vial containing 50 μL ultrapure water and weigh the vial using an analytic balance. Perform a test run by injecting 5 μL water and weigh the vial again. If HPLC works properly, a weight reduction of 5 mg is expected. If not, then contact the technical service of the manufacturer. Alternatively, check the pressure endurance of the HPLC following the instruction. When the leakage is suspected, then contact the technical service and replace the appropriate parts.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Intensities of peaks are lower than expected (step 2-i).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>This is the most frequent problem. Check the peak areas or height of the standard nucleosides. If the results of standard nucleosides are lower than expected, then check mass spectrometer. Usually, accumulation of sample remaining in the mass spectrometer can cause the decrease in peak intensity. From our experience working with Shimazu LCMS-8050, replacing Orifice (Part # S225-15479) or ESI Capillary (Part # S225-14948-91) will often restore the intensity.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Retention time of nucleosides is different (step 2-i).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>The primary cause of this problem is the inappropriate pH of buffer A. Check and adjust the pH to 5.3. Alternatively, a shift of retention time can be caused by the composition (or matrix) of a sample. For example, samples prepared from serum or urine contain thousands of molecules in addition to nucleosides, whereas samples prepared from purified RNA basically only contain nucleosides. The difference of the sample composition can potentially cause the change of retention time. Nucleoside with an early retention time, such as uridine and m<sup>1</sup>A, tends to be easily affected when compared to nucleosides with a late retention time. If samples with different origins are subject to measurement, it is highly recommended not to mix the samples. For example, if an experimenter has 3 blood samples and 3 digested RNA samples, run blood samples and RNA samples separately. It is also recommended to run 2–3 dummy blood samples or RNA samples before the sample of interest.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ogawa, A., Nagiri, C., Shihoya, W., Inoue, A., Kawakami, K., Hiratsuka, S., Aoki, J., Ito, Y., Suzuki, T., Suzuki, T., et al. (2021). <i>N</i><sup><i>6</i></sup>-methyladnenosine (m<sup>6</sup>A) is an endogenous A3 adenosine receptor ligand. Mol. Cell <i>81</i>, 1-16. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00554-2/sref1">View at publisher</a></p>
<p id="bib2">Sproat, B.S., Lamond, A.I., Beijer, B., Neuner, P., and Ryder, U. (1989). Highly efficient chemical synthesis of 2'-O-methyloligoribonucleotides and tetrabiotinylated derivatives; novel probes that are resistant to degradation by RNA or DNA specific nucleases. Nucleic Acids Res. <i>17</i>, 3373-3386. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00554-2/sref2">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Akiko Ogawa (<a href="mailto:accogawa@tohoku.ac.jp">accogawa@tohoku.ac.jp</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The published article includes all generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>We are grateful for suggestions from all members of the Department of Molecular Physiology, Kumamoto University, and the Department of Modomics Biology &amp; Medicine, Institute of Development, Aging and Cancer (IDAC), Tohoku University. We thank Y. Takahata and H. Miyamoto for technical assistance. This work was supported by <a href="https://doi.org/10.13039/501100001691">JSPS</a> KAKENHI grants 18H02599 (F.-Y.W.), 18K19521 (F.-Y.W.), 20H05309 (F.-Y.W.), and 20K18371 (A.O.), and by grants from the <a href="https://doi.org/10.13039/501100002241">Japan Science and Technology Agency</a> (JST), SAKIGAKE JPMJPR1532 (F.-Y.W.), Exploratory Research for Advanced Technology (ERATO) JPMJER2002 (F.-Y.W.), Fusion Oriented Research for disruptive Science and Technology (FOREST) JPMJFR205Y (F.-Y.W.), the <a href="https://doi.org/10.13039/100007449">Takeda Science Foundation</a> (F.-Y.W.), the <a href="https://doi.org/10.13039/100008732">Uehara Memorial Foundation</a> (F.-Y.W.), the <a href="https://doi.org/10.13039/501100007263">Astellas Foundation for Research on Metabolic Disorders</a> (A.O.), and the <a href="https://doi.org/10.13039/501100001696">Japan Medical Women’s Association Foundation</a> (A.O.).</p>
<h3>Author contributions</h3>
<p>A.O. performed experiments and analysis used in this study. F.-Y.W. developed the protocol. A.O. and F.-Y.W. wrote and revised this manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>