<article>
<section>
<h1 id="header">Analysis of lysosomal hydrolase trafficking and activity in human iPSC-derived neuronal models</h1>
<p><time datetime="2021-02-19">Published: February 19, 2021</time></p>
<p>Leah K. Cuddy<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Joseph R. Mazzulli<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>The Ken and Ruth Davee Department of Neurology, Northwestern University Feinberg School of Medicine, Chicago, IL 60611, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:leah.cuddy@northwestern.edu">leah.cuddy@northwestern.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:jmazzulli@northwestern.edu">jmazzulli@northwestern.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100340">10.1016/j.xpro.2021.100340</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Lysosomes are critical for maintaining protein homeostasis and cellular metabolism. Lysosomal dysfunction and disrupted protein trafficking contribute to cell death in neurodegenerative disorders, including Parkinson's disease and dementia. We describe three complementary protocols—the use of protein glycosylation, western blotting, immunofluorescence, and hydrolase activity measurement—to analyze the trafficking and activity of lysosomal proteins in patient-derived neurons differentiated from iPSCs. These methods should help to identify lysosomal phenotypes in patient-derived cultures and aid the discovery of therapeutics that augment lysosomal function.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib1">Cuddy et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Hydrolase maturation can be analyzed by protein glycosylation and western blot</li>
<li>Immunofluorescence is used to validate hydrolase trafficking to the lysosome</li>
<li>Hydrolase function is assessed in living cells with fluorescent activity probes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/474-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol is divided into three sections that provide distinct but complementary readouts of lysosomal hydrolase trafficking and function. The first section describes a biochemical method to measure protein maturation, by analyzing glycosidase sensitivity and molecular weight of hydrolases extracted from cell culture lysates using western blot techniques. The second section describes the analysis of hydrolase subcellular location in fixed cultures using immunofluorescence techniques, through colocalization of the hydrolase of interest with endoplasmic reticulum (ER) and lysosomal markers. The third section describes a method to assess hydrolase activity in living cultures, specifically within lysosomal compartments.</p>
<h3 id="sec1.1">Prepare iPSC-derived midbrain cultures to be analyzed</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 months</div>
<ol>
<li>Culture, expand, and groom iPSCs into a full 6 well culture plate. Procedures for culturing iPSCs are described elsewhere (<a href="https://www.atcc.org/~/media/7E031EF950594BC3B85A411AE1DC9684.ashx">https://www.atcc.org/∼/media/7E031EF950594BC3B85A411AE1DC9684.ashx</a>). We use a feeder free culturing system that utilizes vitronectin (STEMCELL Technologies Catalog # 07004) coated plates with mTesR Plus (STEMCELL Technologies Catalog # 100-0276).</li>
<li>Pass iPSCs for midbrain differentiation. iPSCs should be passed using accutase at a range between 500,000 and 1,000,000 cells per individual 6 well-sized well (35 mm diameter), depending on the particular growth and attachment properties of the iPSC line, onto Matrigel coated plates (Corning Catalog #354277). ROCK inhibitor Y-27632 at 10 μM should be added to the media to promote cell viability during passaging, and for the first day after passaging iPSCs. Tips for coating plates with Matrigel can be found here: <a href="https://www.stemcell.com/coating-plates-with-matrigel-for-pluripotent-stem-cell-culture.html">https://www.stemcell.com/coating-plates-with-matrigel-for-pluripotent-stem-cell-culture.html</a> .</li>
<li>Allow the cells to reach 95%–98% confluency, then start the differentiation protocol (described in detail in the following references: <a href="#bib3">Kriks et al., 2011</a>; <a href="#bib4">Mazzulli et al., 2016a</a>). After 40 days of differentiation, the cultures are allowed to mature for an additional 20–50 days in neurobasal media containing Neurocult SM1 supplement (STEMCELL Technologies Catalog # 05711).</li>
</ol>
<h3 id="sec1.2">Prepare reagents for the analysis of protein trafficking by glycosidase digestion and western blot</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–4 h</div>
<ol start="4">
<li>Purchase Endo H and PNGase F glycosidases from New England Biolabs or a similar vendor.</li>
</ol>
<ol start="5">
<li>If required, prepare buffers for protein extraction, Laemmli and other buffers necessary for standard SDS-PAGE, and reagents for western blotting.
<ol type="a">
<li>Protein extraction base buffer (stored at 4°C for 2–3 years), [final]: 1% Triton X-100, 10% glycerol, 150 mM NaCl, 25 mM HEPES at pH 7.4, 1 mM EDTA, 1.5 mM MgCl<sub>2</sub>).</li>
<li>On the day of the extraction, add in to the base buffer [final]: 1× cOmplete mini EDTA-free protease inhibitor cocktail (Roche), 50 mM NaF, 1 mM PMSF, 10 mM Na orthovanadate.
<div class="note">
<span class="note-title">Note:</span> PMSF and other protease inhibitors expire quickly at 20°C in aqueous solutions. Once added, the completed extraction buffer should be kept on ice and used within 1 day.</div>
</li>
<li>Standard buffers for running Tris-glycine SDS-PAGE gels and western blot transfer can be purchased from Boston BioProducts or a similar vendor. For gels, 30% acrylamide/bis-acrylamide (29:1) will provide optimal resolution for proteins that range from 30 to 80 kilodaltons (kDa). A ratio of 37.5:1 is optimal for proteins larger than 80 kDa, such as LAMP1 or LAMP2a. For transfer, we use a Tris-glycine buffer with 20% MeOH (final).</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Prepare SDS-PAGE gels using a Tris-glycine buffer system and 10% or 12% polyacrylamide, depending on the size of the protein of interest.</li>
<li>Pre-heat a dry bath to 100°C, used to denature proteins before de-glycosylation.</li>
<li>Prepare control samples to aid in the analysis and identify immature protein forms by western blot. For example, acutely treat cultures with an established inhibitor of protein maturation such as tunicamycin or brefeldin A.</li>
</ol>
<h3 id="sec1.3">Prepare reagents for immunofluorescence staining of iPSC-derived neurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<ol start="9">
<li>Prepare buffers required for immunofluorescence analysis.
<ol type="a">
<li>Make 4% paraformaldehyde (PFA) in PBS just before use. Dilute 10% PFA stock to 4% in water and PBS. For 10 mL, add 4 mL of 10% PFA, 1 mL of 10× PBS, and 5 mL of Milli-Q water. This should be prepared fresh each time.</li>
<li>Blocking solution (all final concentrations): 2% fatty acid-free bovine serum albumin (BSA) Fraction V (Sigma # 10775835001), 5% normal goat serum (NGS) (Jackson Immunoresearch #AB_2336990) in 1× PBS pH 7.4. Note that the serum source should match the source host of the labeled secondary antibodies. This should be prepared fresh each time.</li>
<li>Triton permeabilization/wash solutions: The base buffer to be used is 1× PBS, pH 7.4 with 0.1% Triton X-100. This can be stored at 4°C for 2–3 years.</li>
<li>Saponin permeabilization/wash solutions: 1× PBS with 0.2% w/v saponin and 0.2% w/v glycine. This can be stored at 20°C for 6 months.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The detergent to be used will depend on the protein of interest. For GCase, 0.2% saponin is essential, as Triton X-100 compromises detection when the anti-GCase antibody described here (clone 8E4) is used. For most other lysosomal proteins, a range of 0.1%–0.3% Triton X-100 should be sufficient. The end-user should optimize the conditions (detergent and concentration) as subtle variations can occur in different laboratories.</div>
<ol start="10">
<li>Prepare positive and negative controls. Ideally, overexpressing the protein of interest will provide a positive control, while knocking out the protein of interest will provide an optimal negative control. In parallel with the staining procedure, omitting the primary antibody will provide an essential secondary alone control. If available, add in the peptide or protein epitope to compete for the signal during primary antibody incubation to determine antibody specificity. As described above for Endo H analysis, include a tunicamycin-treated control to determine the maximal location shift that is detectable in the assay since this should cause a dramatic accumulation of immature, ER-localized protein.</li>
</ol>
<h3 id="sec1.4">Prepare reagents for live-cell hydrolase activity assays</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="11">
<li>Setting up the culture plate: Plate iPSC-derived neurons into 96 well black wall, clear bottom plates at approximately 8 × 10<sup>4</sup> cells per well. Given the assay and cell type variability, seed n = 8–10 wells per genotype or treatment, and seed 2–3 extra wells to serve as untreated background controls. Half of the wells will be treated with a lysosomal inhibitor, Bafilomycin A1 (Baf A1) on the day of the assay to obtain activity within lysosomal compartments. Coat the plates with poly-D-lysine (PDL)/laminin, as with any iPSC-neuron experiment. Other cell types can be analyzed using this assay; dividing cell types will likely require plating at a lower density to allow for a few rounds of cell division. The cells should have a flat, evenly distributed attachment pattern in the well to achieve the best results. Mature iPSC-derived midbrain neurons may clump over time. Normalization methods and well-scan read patterns in the microplate can help to mitigate technical issues that occur in undesirable cell distribution, but only to a certain degree. Cells that have excessive clumps or that attach only along the well edges will not be measurable. Before the experiment, examine the cells and exclude wells that have detached, demonstrate excessive clumping, or appear unhealthy.</li>
<li>Prepare a stock solution of 100 μM Bafilomycin A1 (Baf A1) in tissue culture grade DMSO. This stock solution can be stored at −20°C for up to 12 months. Just before the experiment, dilute this stock into phenol red free Neural basal medium to make a 2 μM intermediate dilution. This step limits the amount of DMSO added into the media and allows pipetting of a larger, more accurate volume to the cultures during the assay. For this experiment, this 2 μM intermediate dilution is equivalent to a 10× stock since it will be diluted to 200 nM final concentration on the day of the assay.</li>
<li>Prepare a stock solution of 100× (100 mg/mL) dextran cascade blue. Dextran blue measures lysosomal mass and will be used to normalize lysosomal hydrolase activity. The 25 mg compound size from Thermo Fisher requires reconstitution in 250 μL of 1× PBS to make a 100 mg/mL stock concentration. Filter the stock solution and store at −20°C in aliquots. This stock solution can be stored for up to 12 months. Once thawed, it should be stored in the dark at 4°C and used within 2 weeks. Repeated freeze and thaw cycles are not recommended.</li>
<li>After the addition of dextran cascade blue to the 96-well plate, and for each step after that, the experimenter should take care to protect the cells from bright light for extended periods of time.</li>
<li>Set up parameters in the microplate reader. This protocol is optimized for use with a Molecular Devices Spectramax M5 or similar model. Set up the plate reader for the particular type of 96-well plate that is used. Set the temperature to 37°C, bottom reading, and PMT sensitivity to medium (6). Assign the wells used for background subtraction that will not receive any fluorescent substrate, dextran blue, or any other dyes. Set a well-scan reading for 6 to 9 points per well.
<div class="note">
<span class="note-title">Note:</span> most microplate reader's default read setting is to measure 6 reads/well directly through the middle of the well in a single strip. For iPSC-derived neurons, this setting may yield high well-to-well variability since the cells rarely attach to the well in an even, consistent manner. Changing the default setting to well-scan, which samples fluorescence in multiple well regions, will reduce this variability. It will take the plate reader approximately 5–10 min to scan each 96-well plate, so it is important to have the temperature set to 37°C on the plate reader to help maintain physiological conditions. It is also helpful to fill all empty wells and in-between regions of the 96 well plate with sterile water to prevent evaporation of media from the culture wells.</div>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th colspan="2">IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="4">Antibodies</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-GCase (for western blot only)</td>
<td>Sigma</td>
<td colspan="2">Cat #4171</td>
</tr>
<tr>
<td>Mouse monoclonal anti-GAPDH (6C5)</td>
<td>Millipore</td>
<td colspan="2">Cat #CB1001</td>
</tr>
<tr>
<td>Mouse polyclonal anti-b-iii tubulin (TUJI)</td>
<td>BioLegend</td>
<td colspan="2">Cat # 801201</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-hexosaminidase A (H-40)</td>
<td>Santa Cruz</td>
<td colspan="2">Cat #sc-134577</td>
</tr>
<tr>
<td>Sheep polyclonal anti-B-galactosidase (GLB1)</td>
<td>R&amp;D Systems</td>
<td colspan="2">Cat #AF6464</td>
</tr>
<tr>
<td>Goat polyclonal anti-cathepsin B</td>
<td>R&amp;D Systems</td>
<td colspan="2">Cat #AF953</td>
</tr>
<tr>
<td>Mouse monoclonal anti-iduronate-2-sulfatase (3B10)</td>
<td>Novus Biologicals</td>
<td colspan="2">Cat #NBP2-01760</td>
</tr>
<tr>
<td>Mouse monoclonal anti-lamp1</td>
<td>Santa Cruz</td>
<td colspan="2">Cat #sc-20011</td>
</tr>
<tr>
<td>Anti-LAMP2a</td>
<td>Thermo Fisher Scientific</td>
<td colspan="2">Cat #51-2210</td>
</tr>
<tr>
<td>Monoclonal anti-GCase 8E4 (this is the only GCase antibody that we know of to work in a specific manner by indirect immunofluorescence) (1:100 dilution)</td>
<td>Johannes Aerts Lab (University of Leiden)</td>
<td colspan="2">N/A</td>
</tr>
<tr>
<td>Mouse monoclonal anti-lamp1 (lysosome marker; 1:50 dilution)</td>
<td>Santa Cruz</td>
<td>sc-20011</td>
<td> </td>
</tr>
<tr>
<td>Rabbit polyclonal anti-lamp2a (lysosome marker; 1:100 dilution)</td>
<td>Thermo Fisher</td>
<td colspan="2">51-2200</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-PDI (ER marker; 1:100 dilution)</td>
<td>Abcam</td>
<td colspan="2">ab11432</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-GM130 (Golgi marker; 1:100 dilution)</td>
<td>Abcam</td>
<td colspan="2">ab52649</td>
</tr>
<tr>
<td>Mouse monoclonal anti-neurofilament antibody</td>
<td>BioLegend</td>
<td colspan="2">SMI-312R</td>
</tr>
<tr>
<td colspan="4">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Protease inhibitor cocktail</td>
<td>Roche</td>
<td colspan="2">Cat #11-836-170-001</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma</td>
<td colspan="2">Cat #T-8787</td>
</tr>
<tr>
<td>Endo H</td>
<td>New England BioLabs</td>
<td colspan="2">Cat #P0702S</td>
</tr>
<tr>
<td>PNGase F</td>
<td>New England BioLabs</td>
<td colspan="2">Cat #P0704S</td>
</tr>
<tr>
<td>RNase B (control glycoprotein for Endo H or PNGase digest)</td>
<td>New England BioLabs</td>
<td colspan="2">Cat #P7817S</td>
</tr>
<tr>
<td>Odyssey block buffer</td>
<td>LI-COR</td>
<td colspan="2">Cat #927-70001</td>
</tr>
<tr>
<td>Tunicamycin</td>
<td>Enzo Life Sciences</td>
<td colspan="2">BMLCC1040010</td>
</tr>
<tr>
<td>Normal goat serum</td>
<td>Jackson Immunoresearch</td>
<td colspan="2">#AB_2336990</td>
</tr>
<tr>
<td>Bovine serum albumin, Fraction V</td>
<td>Sigma</td>
<td colspan="2">10775835001</td>
</tr>
<tr>
<td>DAPI-Fluoromount G</td>
<td>Southern Biotech</td>
<td colspan="2">0100-20</td>
</tr>
<tr>
<td>Saponin</td>
<td>Sigma</td>
<td colspan="2">47036-50G-F</td>
</tr>
<tr>
<td>Phenol red-free neurobasal medium</td>
<td>Thermo Fisher</td>
<td colspan="2">12348-017</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Sigma</td>
<td colspan="2">10775835001</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO), Hybri-max</td>
<td>Sigma</td>
<td colspan="2">D2650</td>
</tr>
<tr>
<td>Bafilomycin A1</td>
<td>Enzo Life Sciences</td>
<td colspan="2">BML-CM110</td>
</tr>
<tr>
<td>Conduritol-b-epoxide</td>
<td>EMD Millipore</td>
<td colspan="2">234599</td>
</tr>
<tr>
<td>Dextran, Cascade blue, MW 10,000 kDa, lysine fixable</td>
<td>Thermo Fisher</td>
<td colspan="2">D1976</td>
</tr>
<tr>
<td>PFB-FD-Gluc (GCase substrate)</td>
<td>Thermo Fisher</td>
<td colspan="2">P11947</td>
</tr>
<tr>
<td>CellTag700</td>
<td>Li-Cor Biosciences</td>
<td colspan="2">926-41090</td>
</tr>
<tr>
<td>PFB-FD-Gluc (substrate for GCase)</td>
<td>Thermo Fisher</td>
<td colspan="2">P11947</td>
</tr>
<tr>
<td>Fluorescein acetylglucosamine (substrate for hexosaminidase)</td>
<td>Abcam/MarkerGene</td>
<td colspan="2">ab274878</td>
</tr>
<tr>
<td>LysoLive sulfatase kit (substrate for Sulfatase)</td>
<td>Abcam/MarkerGene</td>
<td colspan="2">ab253377</td>
</tr>
<tr>
<td>2′, 7′-Dichlorofluorescein di-β-D-lactoside (substrate for β-lactosidase)</td>
<td>Abcam/MarkerGene</td>
<td colspan="2">ab275366</td>
</tr>
<tr>
<td>2′,7′-Dichlorofluorescein di-β-D-galactopyranoside (substrate for β-galactosidase)</td>
<td>Abcam/MarkerGene</td>
<td colspan="2">ab275920</td>
</tr>
<tr>
<td>MagicRed Cathepsin B kit</td>
<td>Immunochemistry Technologies</td>
<td colspan="2">937</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<ul>
<li>For Endo H/western blot analysis, most standard materials and equipment including electrophoresis apparatus are suitable for this analysis. For gel transfers, we use a semi-wet (Thermo Fisher Mini Blot module, #B1000) or completely wet transfer system (e.g., Bio-Rad Mini-Trans blot Cell) for efficient transfers of glycoproteins. We have found unsatisfactory results using rapid (&lt;10 min) and completely dry transfer systems due to inefficient transfer of proteins out of gels. An efficient transfer is critical for high molecular weight proteins that are difficult to elute from gels. Postfixing the membrane after transfer either in 100% MeOH for 1 min or 0.4% paraformaldehyde for 20 min can also increase the signal : noise ratio.</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> Unpolymerized acrylamide is a neurotoxin. Exercise caution when handling acrylamide solutions (e.g., wearing double gloves and face shields while preparing gels) and discard waste chemicals in containers designated by the institution's research safety office. Please read the MSDS sheets of all chemicals to be used in the protocol before usage. All toxic and carcinogenic reagents should be identified and documented with the institution's research safety office.</div>
<ul>
<li>For the immunofluorescence studies, a confocal microscope with a laser excitation source should be used for optimal signal: noise ratio, such as a Leica TCS SPE laser with CTR4000/DMI4000B microscope. Confocality is required for colocalization studies of any cell type, and this is especially true with iPSC-derived neurons that often form large clumps and cluster in culture wells.</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> Paraformaldehyde solutions are volatile and carcinogenic. Only use PFA-containing solutions in a chemical fume hood.</div>
<ul>
<li>For live-cell activity assays, a microplate reader that contains dual monochromators is best suited since it will allow the user to precisely define the excitation, emission, and bandwidth parameters. Microplate readers with other excitation systems are sufficient, as long as excitation at the optimal wavelength of the fluorophore can be measured. A microplate reader with a well-scan option is also recommended, particularly when analyzing iPSC-derived neurons that tend to aggregate or grow unevenly in the well, as discussed above. For this reason, we recommend the use of a Molecular Devices Spectramax model that is capable of fluorescence and temperature control, such as the M5 model or later.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Analysis of protein trafficking by western blot and endo H analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>The analysis method described here relies on the differential PAGE mobility and molecular weights of distinct forms of lysosomal proteins that occur during the maturation process, such as proteolytic cleavage and N-linked glycosylation. N-linked glycosylation of lysosomal proteins occurs in the endoplasmic reticulum (ER), which permits the proper folding and export out of the ER. Once proteins have passed the quality control checkpoints in the ER, transport into the Golgi occurs, where the enzyme -mannosidase II further modifies N-linked glycans. N-linked glycans are then resistant to endoglycosidase H (Endo H), a bacterium-derived glycosidase commonly used in protein-glycan analysis. Therefore, by determining the amount that is Endo H resistant, the approximate subcellular location of lysosomal proteins is revealed. Endo H resistant and sensitive forms are identified by SDS-PAGE/western blot by molecular weight analysis. Below, we focus on the lysosomal protein β-glucocerebrosidase (GCase) as a representative example. However, similar principles apply to other proteins that mature through the lysosomal and secretory pathways. Analysis of protein extracts from any source, including cell cultures, whole organism model systems, or tissues, can be performed using this protocol.</p>
<p>Hydrolases measured by this method include hexosaminidase, B-galactosidase, cathepsin B, iduronate-2-sulfatase, and GCase.</p>
<p>When setting up the culture experiment, it is useful to include a tunicamycin-treated culture well as a control. Tunicamycin will prevent glycosylation and maturation of the protein of interest, allowing for a direct comparison of this control sample and your experimental sample, aiding in the assay's interpretation. Each assay should be done with at least three technical replicates and confirmed in two to three independent experiments or iPSC passages. If desired, digestion of purified RNase B can be used as an additional control to confirm the activity of activity of Endo H and PNGase F. Glycosylated RNase B can be purchased from New England Biolabs.</p>
<ol>
<li>Making an extract (4 h)
<ol type="a">
<li>Using a cell scraper, scrape cell cultures from a 12 or 6-well dish of iPSC-derived neurons into 1 mL cold 1× PBS and pellet by centrifugation at 400 × <i>g</i>, 5 min, 4°C.</li>
<li>Discard the supernatant and resuspend pelleted material in an appropriate volume (200–400 μL per well of a 6-well dish) of cold protein extraction buffer containing 1% Triton X-100.</li>
<li>Use a Teflon homogenizer with a motor-driven spindle to homogenize the extract in a conical glass vessel 20–40 times at 2,000 × <i>g</i>. Then, incubate the extract in an ice-water slurry with gentle rotation for 20 min.
<div class="note">
<span class="note-title">Note:</span> When homogenizing the cell pellets, use care not to generate excessive froth. Homogenize the extract until there are no particles; it should be even and slightly translucent. If a pipet tip is not visible when inserted in the solution, add more lysis buffer to the extract and re-homogenize.</div>
</li>
<li>Clarify the extract by centrifugation at 20,000 × <i>g</i>, 4°C for 10 min. Transfer the supernatants to a new 1.5 mL tube.
<ol type="i">
<li>Perform a protein assay on the supernatant using a Pierce micro BCA kit (Thermo Fisher # 23235) or standard Bradford reagent (Boston Bioproducts # BPA-200).</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep the pellet fraction if desired, but the supernatant fraction of this extract will contain most hydrolases.</div>
<ol start="2">
<li>Digest the lysate with Endoglycosidase H (Endo H) and PNGase F (3 h).
<ol type="a">
<li>Denature 50 μg of protein lysate from each sample by adding 1× denaturation buffer (included with Endo H and PNGase F from New England Biolabs).
<ol type="i">
<li>Add 50 μg of protein lysate + water, up to 18 μL.</li>
<li>Add 2 μL of 10× denaturation buffer and incubate at 100°C for 10 min in a dry bath.</li>
<li>Pulse-centrifuge the sample to spin down any evaporated liquid.</li>
</ol>
</li>
<li>Digest the denatured lysate with glycosidase enzymes Endo H and PNGase F in two separate tubes:
<ol type="i">
<li>For the addition of Endo H: add 2.5 μL of 10× G5 digest buffer (included with the enzymes from NEB) + 1.5 μL of water + 1 μL of Endo H and incubate for 2 h at 37°C.</li>
<li>For the addition of PNGase F: add 2.6 μL of 10× G5 buffer + 2.6 μL of 10% NP-40 (included with the enzymes from NEB) + 1 μL PNGase F and incubate for 2 h at 37°C.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In parallel, prepare an undigested control sample by using water instead of enzymes. Incubate the control sample under the same conditions as experimental samples.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> After the 2-h incubation, the digest can be stored at −80°C until western blot analysis is performed.</div>
<ol start="3">
<li>Run a western blot (3 h).
<ol type="a">
<li>Boil digest samples in 1× Laemmli sample buffer for 10 min at 100°C.
<div class="critical">
<span class="critical-title">Critical:</span> Boil for at least 10 min to fully separate the different glycosylated forms of the proteins. After boiling, do <i>not</i> place the sample on ice since this may cause SDS to precipitate and prevent efficient separation of protein. The sample will be stable at 20°C for up to 1 h until the gel is loaded.</div>
</li>
<li>Run the samples on 10% Tris-glycine gels that are 1.5 mm thick and can hold 10 wells. Set the initial voltage at 120 V for 20 min, followed by running at 150 V for 1–3 h depending on the protein of interest. GCase migrates between 55 kDa and 70 kDa. To achieve sufficient separation of distinct GCase forms, run the gel until a 36 kDa marker of the ladder is 1 cm from the bottom of the gel, which usually takes about 3 h.
<div class="note">
<span class="note-title">Note:</span> The mature forms of some hydrolases (e.g., Cathepsins and Hexosaminidase B) migrate at 10–20 kDa. To prevent run-off of these forms, stop gels at the appropriate time.</div>
</li>
<li>Transfer onto PVDF membranes according to the particular transfer apparatus used. For the Thermo Fisher mini blot apparatus, transfer for 1 h at 20°C at a constant voltage of 20–30 V (approximately 300 mA starting current per gel). This protocol utilizes fluorescent secondary reagents, and therefore Millipore Immobilon-FL 0.45 μm membranes (# IPFL00005) are used to prevent high background signals.</li>
<li>Remove the membrane and postfix in 100% MeOH for 1 min, then wash in 1× Tris-buffered saline (pH 7.4) (TBS) for 2–5 min until the membrane sinks. Stain the gel in Coomassie brilliant blue for 12–24 h, followed by washing in water for 2 h. This gel is to be used as a loading control. Under these transfer conditions, a portion of the total protein remains within the gel due to the presence of 20% MeOH in the transfer buffer. While the MeOH promotes protein binding to membranes, it slows the elution from SDS-PAGE gels. This is particularly noticeable in high and middle molecular weight proteins that do not elute out of the gel as easily as low molecular weight proteins.
<div class="note">
<span class="note-title">Note:</span> Alternatively, postfix the membrane in 0.4% PFA. The PFA is made in 1× PBS and incubated for 20 min on a slow rotator, followed by 3 × 5 min washes in water. The optimal postfix method should be determined by the end-user and depends on the protein of interest and antibody. For GCase, either MeOH or PFA postfixing methods work equally well.</div>
</li>
<li>Block the membrane in a 1:1 mix of 100% Odyssey Block Buffer (OBB) and 1 × TBS (No Tween) for 1–1.5 h.
<div class="note">
<span class="note-title">Note:</span> This protocol is optimized for fluorescent-conjugated secondaries using the Li-Cor Odyssey scanner. Other scanners can be used with equal success. Other secondaries, such as HRP and ECL reagents, are not recommended due to the limited quantitative range.</div>
</li>
<li>Add primary antibodies for the lysosomal hydrolases of interest to a 1:1 mix of OBB: 1× TBS-Tween 0.2%. Add this solution to the membranes and incubate for 12–24 h 4°C. For GCase antibodies from Sigma, a dilution of 1:500 to 1:1,000 should work, depending on the secondary reagent's antibody lot and sensitivity. Through pilot studies, the end-user can determine optimal antibody conditions. It is crucial to record the antibody lot number, as different batches tend to change in both antibody concentration and binding efficiency.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Incubate the membrane for 12–24 h at 4°C until performing western blot imaging and analysis the following day.</div>
<ol start="4">
<li>Image and quantify western blots (4 h).
<ol type="a">
<li>Wash the membranes three times, 10 min each, in 1× TBS-Tween 0.2% at 20°C.</li>
<li>Add secondary antibodies in at 1:10,000 in OBB : 1× TBS-Tween 0.2% 1:1 mix and incubate the membranes at 20°C for 1–2 h.</li>
<li>Wash membranes three times, 10 min each, in 1× TBS-Tween 0.2% at 20°C.</li>
<li>Scan and analyze using an Odyssey infrared imaging system (or similar system), and analyze with Imaging software (e.g., Image Studio Software (Li-Cor) or Image J).</li>
<li>By comparing the undigested, Endo H digested, and PNGase F digested samples, identify and quantify the post-ER/ER forms of each protein. Quantify protein maturation by calculating a ratio of post-ER/ER forms. Each form can also be quantified separately and normalized to a loading control (e.g., GAPDH, tubulin, or Coomassie blue).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> During the lysis step, efficient homogenization is necessary, and a fresh enzyme is required for the complete digestion of samples. To achieve a clean separation of individual glycosylated forms of lysosomal hydrolases, avoid gradient gels and use the acrylamide percentage suited for the protein of interest. For most proteins that average 50–70 kDa in weight, 10% polyacrylamide is suitable. Using an antibody that is capable of detecting mature and immature forms is also critical. During the maturation process, portions of most hydrolases are cleaved in the ER, and therefore some epitopes may be removed. It is best to use a specific polyclonal antibody that will react with multiple regions of the protein of interest and to know the exact amino acid sequence where the antibody reacts.</div>
<h3 id="sec3.2">Analysis of protein trafficking by immunofluorescence staining of iPSC-derived neurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This assay is used in combination with Endo H/western blot analysis to confirm the location of lysosomal hydrolases in fixed cells. Changes in hydrolase trafficking are a dynamic process but can be estimated by quantifying the hydrolase’s steady-state levels within ER, Golgi, and lysosomal compartments by colocalization analysis. Complementation of the Endo H analysis with this immunostaining protocol will provide stronger evidence for changes in protein trafficking. Immunofluorescence staining does not involve cell lysis, thereby eliminating confounding biochemical processes that could occur post-lysis.</p>
<ol start="5">
<li>Fix and immunostain cells (2 days).
<ol type="a">
<li>Plate cells on PDL and laminin-coated coverglass in 24 or 12 well plastic culture plates.
<div class="note">
<span class="note-title">Note:</span> iPSC-neuron cultures require plates with PDL and laminin coating; other cell types may require different coating conditions. Cells should be plated at a density that allows for imaging of individual cells.</div>
</li>
<li>Quickly remove the culture media and immediately add 4% paraformaldehyde (PFA) in 1× PBS for 20 min at 20°C under a fume hood. The cells should not be left dry for more than a few seconds, so this may require the fixation of only a few wells at a time.</li>
<li>Remove the 4% PFA and wash the cells 3 times for 5 min in cold PBS to completely remove PFA. Place both the 4% PFA and washes in a designated waste container according to the institution's hazardous waste guidelines.</li>
<li>Permeabilize and block cells: For GCase, use 0.2% saponin (w/v)/1× PBS for 5 min, then 0.2% (w/v) glycine/0.2% saponin in 1× PBS for min. Next, block the cells with 2% bovine serum albumin, 5% normal goat serum in 0.2% saponin/1× PBS for 30 min.</li>
<li>Make the primary antibody dilutions in blocking buffer, preparing 50 μL of solution per 24 well-sized cover glass. For anti-GCase, use a 1:1,000 dilution. Place a 50 μL droplet of the primary antibody solution for each cover glass onto a parafilm sheet inside a humidity chamber.
<div class="note">
<span class="note-title">Note:</span> To create a humidity chamber, place a water-saturated paper towel along the side of a large square culture plate (e.g., Thermo Fisher #166508).</div>
</li>
<li>Carefully remove the cover glasses from the plastic culture plate while the block solution is still in the well, and flip onto the primary antibody solution on parafilm so that the cell-side is facing down. Place the chamber in a safe location and incubate for 12–24 h at 4°C. Keep the plastic culture plate at 4°C for the washing steps the next day.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The cells will be incubated in primary antibodies for 12–24 h at 4°C. This step could extend to 2 days, which may enhance the signal. However, water should be added to the chamber once or twice a day to prevent drying. The frequency depends on the humidity conditions of the surrounding environment.</div>
</li>
<li>Place the cover glasses back into the plastic culture plate. Wash the cells very gently 3 times for 20 min each in 0.1% Triton X-100 in 1× PBS at 20°C.
<div class="note">
<span class="note-title">Note:</span> During the wash steps, it is advisable to avoid using a vacuum aspirator since iPSC-derived neurons are occasionally not well attached to the cover glass. Gentle pipetting using a P-1000 tip is advised.</div>
</li>
<li>Add secondary antibodies at 1:300 or 1:400 dilutions in blocking buffer for 2 h at 20°C.
<div class="note">
<span class="note-title">Note:</span> Alexa conjugated antibodies may aggregate over time. To avoid adding aggregates to the samples, spin down the secondary antibody stocks before each use for 10 min, 20,000 × <i>g</i>, 4°C.</div>
</li>
<li>Wash cells 3 times for 20 min each in 0.1% Triton in 1× PBS at 20°C.</li>
<li>Place a 5–10 μL droplet of DAPI-containing Fluoromount G onto a glass slide.</li>
<li>Quickly rinse the cover glasses in pure water, then invert the cover glasses onto the slides containing Fluoromount G. Remove any bubbles if necessary and allow to dry for 10 min. Once dry, seal the edges with a thin layer of clear nail polish.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the confluency and number of cell clumps, more than 10 μL of Fluoromount G may be required. Any large three-dimensional cell clumps should be removed or flattened to provide a tight seal when mounting. Background autofluorescence will occur if a tight seal is not achieved.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dry the slides at 20°C for 24 h in the dark, followed by long-term storage at 4°C. Image the cells within 1 week.</div>
<ol start="6">
<li>Image and quantify (4+ h).
<ol type="a">
<li>Examine cells by confocal microscopy through a 10 μM section by Z-stack at 1 μM per section.</li>
<li>Using Image J software, outline individual cells and define them as regions of interest to be calculated individually.</li>
<li>Obtain Pearson's correlation values for colocalization using ImageJ software using the Coloc 2 plugin.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The fixation and permeabilization steps of this protocol have been optimized to detect Lamp2 and GCase. The specific methods used for the permeabilization (permeabilization reagent, concentration, time) and fixation step should be adjusted and optimized for the antibody and epitope of interest.</div>
<h3 id="sec3.3">Measurement of hydrolase trafficking and activity in lysosomal compartments of living cultures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This portion of the protocol complements the preceding trafficking assays by assessing functionality of hydrolases through activity measurements, specifically within acidic cellular compartments in living cells. Hydrolase activity is measured by adding an artificial substrate into the media of living cell cultures that fluoresces upon cleavage within lysosomes. Cell-permeable substrates are available for multiple hydrolases, but the protocol below focuses on the GCase substrate, 5-(Pentafluorobenzoylamino) Fluorescein Di-β-D-Glucopyranoside (PFB – FD-Gluc). This substrate is internalized into cells within 30 min and trafficked to lysosomes through the endolysosomal system. Once cleaved by GCase, the substrate emits fluorescence at ~510 nm of the spectrum (similar to GFP or FITC) and accumulates within lysosomal compartments. Therefore, the fluorescein signal increases gradually over time as the substrate accumulates. Specific lysosomal activity is estimated by treated parallel cultures with a lysosomal inhibitor such as Bafilomycin A1 (Baf A1) that reduces lysosomal acidity. The difference between control and Baf A1 treated cultures is a measurement of hydrolase activity within acidic cellular compartments. In this case, it is safe to assume that most of this activity is lysosomal. The presence of PFB-FD-Gluc in lysosomes can be confirmed, if desired, by fixing cells treated with PFB-FD-Gluc, immunostaining with a lysosomal marker, and examining by confocal microscopy (see previous section). The PFB-FD-Gluc signal should dramatically decline in cultures treated with a GCase inhibitor, such as conduritol-β-epoxide (CBE).</p>
<p>The advantage of this protocol is that it can assess enzymatic activity <i>in situ</i>, within lysosomes of living neurons. Alternative methods employ activity measurements from cell lysates, which can help determine the amount of potentially active GCase inside cells, but confounding variables may occur post-cell lysis. For example, the buffers used in lysate assays are artificially optimized for enzymatic activity, including the optimal pH and required co-factors for activity (such as sodium taurocholate, in the case of GCase). The live-cell assay described here eliminates these issues, since substrate cleavage occurs naturally within the lysosomal environment. When identifying lysosomal phenotypes in cells, this issue is critical, as many artificial factors could synergize and contribute to declined activity. In Parkinson's disease and other synucleinopathies, one mechanism underlying lysosomal dysfunction occurs from defects in hydrolase trafficking, where immature hydrolases accumulate within the ER or other compartments (<a href="#bib1">Cuddy et al., 2019</a>). This phenotype is difficult to detect in lysate assays since the immature enzymes stuck in the ER are still capable of substrate turnover, once liberated in a lysate and placed into an optimal buffer system <i>in vitro</i>. In this case, using a live-cell or <i>in situ</i> lysosomal activity assay is a more accurate method for detecting a deficiency that occurs from enzyme mislocalization.</p>
<ol start="7">
<li>Day 1: Add dextran cascade blue to cells at a final concentration of 1 mg/mL 12–24 h before starting the activity measurements. Do not treat one or two wells; these will serve as a blank during the microplate readings.</li>
</ol>
<ol start="8">
<li>Day 2: Measure lysosomal activity (5 h).
<ol type="a">
<li>In a typical assay, n = 8–10 culture wells are required per condition. To half of the wells, add Baf A1 to a final concentration of 200 nM. For example, if a single well contains 100 μL media, add 11 μL of 2 μM Baf A1 to achieve 200 nM Baf A1 [final]. The control wells should get the same volume or final percentage of the vehicle (DMSO). A multi-channel pipettor will be useful here.
<div class="note">
<span class="note-title">Note:</span> A complete media change can be done but is not necessary or recommended. Excessive media changes can cause loosely attached iPSC-derived neurons to lift off of the plate. Add Baf A1 directly to the wells so that treatment can be done while the dextran blue remains in the media.</div>
</li>
<li>Incubate the cells for 1 h with Baf A1 and dextran blue in a tissue culture incubator at 37°C, 5% CO<sub>2</sub>.</li>
<li>After 1 h, aspirate off the media to remove the dextran blue. Replace with media containing the fluorescent substrate of the lysosomal enzyme being analyzed, along with DMSO or Baf A1 in the appropriate wells.
<div class="note">
<span class="note-title">Note:</span> For GCase, use PFB-FD-Gluc at 100 μg/mL for 30–45 min. It is not recommended to exceed 100 μg/mL or to increase incubation time past 1 h. Be sure to replace the Baf A1 or DMSO in the appropriate wells during incubation with the fluorescent substrate.</div>
</li>
<li>Incubate at 37°C, 5% CO<sub>2</sub> for 45 min to allow the substrate to be fully taken up by cells and into lysosomes.</li>
<li>Gently wash the cells once with complete media, and replace with the same media containing either DMSO or Baf A1 in the appropriate wells. The volume of media in each well should be precise between wells (including the wells used for blanks) since the volume will greatly influence the background signal in the plate reader. For iPSC-neurons, use complete Neural basal (NB) media supplemented with B27 (Thermo Fisher) or Neurocult SM1 (Stem Cell Technologies), and replace the media with 100 μL/well of SM1-NB media <i>without</i> PFB-FD-Gluc and phenol red. Be careful not to let the cells dry out or lift off during the media changes.</li>
<li>After the media change, immediately read the plate in the microplate reader to get the initial fluorescent reading. Depending on the plate reader speed and number of wells to be read, this step could require 5–10 min to read through the plate.
<div class="note">
<span class="note-title">Note:</span> Before starting, set the temperature of the plate reader to 37°C and assure it will read fluorescence from the bottom. Some plate readers require a plate-well adapter for bottom reading applications. Start with default sensitivity and change if necessary (6, PMT at medium). As mentioned above, default settings in the Molecular Devices Plate reader will normally read one strip in the middle, left to right. If the cells are clumped together, use a well-scan setting (in the Softmax settings) to read multiple areas of the well to reduce well-to-well error. There should be a minimal signal at time = 0 since the substrate requires approximately 30 min post-chasing time to be degraded and accumulate inside lysosomes at a detectable level. However, the dextran blue signal should be strong at t = 0 and should not change much between the initial and final readings.</div>
</li>
<li>When the first reading is complete, cover the plate with foil to protect the cells from light and place the plate back into the incubator. Continue to read every 30 min for up to 4 h to get a full kinetic activity curve. After each read, place the cells back into a humidified tissue culture incubator. No further media changes are required.</li>
<li>After the final read, remove the media and fix the cells in 4% PFA for 20 min. If activity is measured in neurons, staining the cells with a primary antibody for neurofilament followed by a secondary antibody IRDye800, will detect neurite degeneration. If a decline in neurofilament staining is observed, this indicates neurotoxicity in the cells that will affect results.</li>
<li>Simultaneously, CellTag700 can be used as a total cell volume staining, or Draq5 to specifically stain nuclei. After a 1 h incubation at 20°C, wash the stained cells with 1× PBS and scan on an infrared imager such as the Li-cor Odyssey, Azure Sapphire, or Typhoon system. Both CellTag700 and Draq5 should indicate the total cellular material per well and can be used to normalize the lysosomal substrate signal. Even though plates should be seeded with an equal number of cells/well at the start of the experiment, normalization is required since cell numbers can change dramatically between the time the iPSC-neurons were seeded and the time of the lysosomal assay.</li>
<li>To analyze the data, the relative fluorescent units (RFUs) of the hydrolase substrate for each time point should be normalized to either cell volume or lysosomal mass and plotted vs. time. If lysosomal mass is used, it is best to normalize the values to only the last time point collected of dextran cascade blue. The dextran fluorescence should not vary between time 0 and the final time point collected at 4 h. However, using final time point values will assure that the signal occurs from lysosomal compartments instead of endosomes and other vesicles en route to the lysosome. Once plotted, the area between the DMSO and Baf A1 curves is the “lysosomal” value (<a href="#fig4">Figure 4</a>). The remaining area under the Baf A1 curve is the activity that occurs outside of acidic subcellular compartments (<a href="#fig4">Figure 4</a>). For example, for the PFB-FD-Gluc substrate, some cleavage may occur from cytosolic GBA2 or <i>GBA1</i> that is mislocalized in non-lysosomal compartments.</li>
</ol>
</li>
</ol>
<p>These experiments can be done to compare the effect of investigational small molecules that activate lysosomes, or to identify lysosomal phenotypes in patient-derived culture systems of lysosomal storage disorders (<a href="#bib2">Kim et al., 2019</a>; <a href="#bib7">Zunke and Mazzulli, 2019</a>) or Parkinson's disease (<a href="#bib1">Cuddy et al., 2019</a>; <a href="#bib4">Mazzulli et al., 2016a</a>). Each experiment should be done using 8–10 wells (4–5 each for DMSO and Baf A1) and 2 or 3 distinct cell passages/differentiations. If possible, the analysis should be done in a blinded manner.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>For the Endo H/western blot analysis, mature (post-ER) hydrolases are determined by dividing Endo H resistant protein by the Endo H sensitive forms, using the signal from the Endo H digested gel lanes. PNGase F is used to detect the molecular weight (MW) of the fully deglycosylated protein. In the case of GCase, the fully deglycosylated form runs at 55 kDa on Tris-glycine gels. MW may vary on different gel or buffer systems. For example, the fully deglycosylated GCase migrates slightly faster in MOPS gels using a Bis-tris buffer system, at 51 kDa. On Tris-glycine SDS-PAGE gels, post-ER mature bands of GCase migrate at higher molecular weights than immature bands. Mature GCase = 64–75 kDa, immature GCase runs between 55 and 62 kDa (<a href="#fig1">Figure 1</a>). The MW of other hydrolases is as follows: Hexosaminidase A MW: Mature = 54 kDa (alpha), immature = 67 kDa. B-Galactosidase, MW: Mature = 64 kDa; Immature = 85 kDa. Cathepsin B, MW: Two distinct mature bands at 30 kDa, 25 kDa; Immature = 52 kDa, Iduronate-2-Sulfatase MW: Two distinct mature bands at 45 kDa, 44 kDa; Immature 74 kDa, 55 kDa. Most N-linked glycans are located on the outer surface or solvent-exposed region of the protein and are therefore subject to further modification in the Golgi. However, some proteins may contain buried glycans that are inaccessible to further modification in the Golgi. Therefore, some Endo H sensitive N-glycans could occur in fully mature proteins.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/474-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Expected results of Endo H-digested western blot analysis of GCase</div>
<p>The top image shows the blot probed with anti-GCase antibodies, while the bottom image shows the same blot probed with anti-beta-iii-tubulin and GAPDH as loading controls. Three different samples were analyzed. Sample 2 shows an example of a condition where Endo H sensitive GCase (immature, most likely in the ER) are elevated compared to the other samples. This is apparent in the (–) Endo H condition by the stronger intensity of the 62 kDa form (red dotted box in (–) Endo H sample 2). When digested with Endo H, the 62 kDa band collapses completely into 55 and 58k Da forms (red dotted box in (+) Endo H sample 2). The band at 64 kDa shows a partial decrease in intensity upon Endo H digestion, indicating partial sensitivity. The Endo H resistant and sensitive labels on the right side of the blot indicate those bands from (+) Endo H lanes used to quantify post-ER-ER ratios.</p>
</figcaption>
</figure>
<p>For the immunofluorescence studies, control cells at steady-state should show that 80%–90% of the 8E4 GCase signal colocalizes with the LAMP2a signal, whereas minimal colocalization of GCase with ER or Golgi compartments should occur. See <a href="#fig2">Figure 2</a> for a representative example, as well as figures within these references (<a href="#bib1">Cuddy et al., 2019</a>; <a href="#bib5">Mazzulli et al., 2016b</a>). If trafficking perturbations have occurred, then the enzyme levels will either be apparent within ER, Golgi, or throughout the cytoplasm. Often, trafficking disruptions result in the aberrant secretion of the enzyme into the media, e.g., in the lysosomal storage disorder, I-cell disease (<a href="#bib6">Wiesmann et al., 1971</a>), or when essential trafficking proteins are knocked down (<a href="#bib1">Cuddy et al., 2019</a>). Aberrant secretion of hydrolases will be evident by a low/undetectable hydrolase signal inside cells. To confirm aberrant hydrolase secretion, concentrate the media and perform a western blot or activity assay.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/474-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative example of an immunofluorescence result showing the colocalization of GCase and LAMP2a</div>
<p>Control H4 neuroglioma cells (top) or control iPSC-derived midbrain neurons were fixed and stained using anti-GCase (green) or anti-LAMP2a (red) antibodies and analyzed by confocal microscopy as described above. Nuclei are DAPI-stained and shown in blue.</p>
</figcaption>
</figure>
<p>In live-cell activity assays, the signal of the lysosomal substrate should increase over time and eventually peak at around 3 h. Baf A1 should inhibit the lysosomal enzyme and dextran blue. The wells should be free of dead or clumped cells. Pay careful attention to possible technical errors that may arise, such as signal drift of the raw values (i.e., values changing from left to right or top to bottom, possibly occurring from toxicity/cell drying from slow media changes). Often the dextran blue signal will be higher in the Baf A1 treated cells due to the accumulation of non-functional lysosomes.</p>
<p>There are two methods for normalizing the lysosomal substrate signal depending on the desired measurement. To obtain activity per cell, normalize the signal to the cell volume dye or nuclear stain. To obtain a measurement of lysosomal efficiency, normalize the substrate signal to the dextran blue (lysosomal mass) obtained from the plate reader. Both activity per cell and activity per lysosomal mass are useful readouts depending on the specific question. For example, potential small molecule lysosomal activators may influence lysosomal efficiency (more enzymes per lysosome, <a href="#fig3">Figure 3</a>A) or number (more lysosomes per cell, <a href="#fig3">Figure 3</a>B). Some lysosomal activators increase both lysosomal efficiency and number.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/474-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Possible outcomes and interpretations of the live-cell lysosomal activity assay</div>
<p>(A) In a hypothetical experiment, treating cells with compound A increases lysosomal efficiency by providing more hydrolase enzyme per lysosome (L). This would be observed upon normalizing to both cell number and lysosomal mass (dextran blue).</p>
<p>(B) Upon treating cells with hypothetical compound B, increased lysosomal activity is observed only when normalized to total cell number, but not lysosomal mass (dextran blue) since the total number of lysosomes also increases. In this case, compound B may increase lysosomal biogenesis. Total lysosomal mass can be further quantified by quantifying the dextran blue signal and normalizing to cell volume (CellTag 700). In each case A and B, further experiments are required to delineate the mechanism of action definitively.</p>
</figcaption>
</figure>
<p>For each time point, the relative fluorescent units (RFUs) of the hydrolase substrate should be normalized to either cell volume or lysosomal mass and plotted vs. time. If lysosomal mass is used, it is best to normalize the values to only the last time point collected of dextran cascade blue. The dextran fluorescence should not vary between time 0 and the final time point collected at 4 h. However, using final time point values will assure that the signal occurs from lysosomal compartments. Once plotted, the area between the DMSO and Baf A1 curves is the “lysosomal” value (<a href="#fig4">Figure 4</a>). The remaining area under the Baf A1 curve is the activity that occurs outside of acidic subcellular compartments (<a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/474-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. A typical activity assay result using a probe to detect hexosaminidase activity in healthy control iPSC-derived neuronal midbrain cultures</div>
<p>The amount of lysosomal and non-lysosomal activity observed in control cells at baseline will depend on the specific hydrolase measured, the cell type, and culture conditions of the laboratory.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Quantification methods are discussed within each subsection. Statistical analysis is dependent on the number of samples to be compared. An unpaired Student’s t test should be done when two samples are compared. For more than two conditions, ANOVA with either a Tukey or Dunnett's post-hoc test should be done. P-values that are &lt;0.05 are considered significant.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Analysis of hydrolase trafficking by quantifying Endo H resistant forms on western blots assumes that the protein has passed through the Golgi compartment and into lysosomes. In some diseases, trafficking may be perturbed post-Golgi, and therefore Endo H resistant hydrolases may not accurately represent fully matured lysosomal forms. Additionally, hydrolases may mature through non-classical mechanisms, such as secretion followed by uptake through the endolysosomal system. For this reason, the Endo H resistance assay should be complemented with the other two assays described here to gain a complete representation of hydrolase trafficking.</p>
<p>Immunofluorescence analysis and colocalization studies can validate studies from protocols 1 and 3 described here. These studies have the advantage of allowing the observer to directly visualize the hydrolase of interest within lysosomal compartments. However, detecting a hydrolase within lysosomes does not guarantee its functionality, and therefore activity assays described in section <a href="#step-by-step-method-details">3</a> should be performed.</p>
<p>Measurement of hydrolase activity in living cultures provides an accurate, sensitive readout of lysosomal functionality within a physiological setting. Disruption of cellular membranes or organelles is not required for measurement, allowing the observer to gain insight into the hydrolase activity within its natural environment. However, this assay has technical limitations controlled by the seeding density and distribution pattern within the well. Cells that clump or aggregate, leaving bare areas on the well, will not be readable with this assay. This assay may not be suitable for measuring activity from mixtures of cells or co-cultures of neurons and glia since only global measures can be obtained from the plate reader. While robust, well characterized iPSC lines can achieve efficient neuronal differentiation resulting in nearly 80%–90% of neurons (<a href="#bib4">Mazzulli et al., 2016a</a>), the cultures can also contain a minor percentage of undifferentiated cells or other cell types that could contribute to global hydrolase activity measurement. However, the majority of the lysosomal activity obtained should occur from neurons that make up the majority of the culture. If a less-than-optimal iPSC line is used that fails to differentiate efficiently into neurons, then it may be required to image the PFB-FD-Gluc signal in individual neurons directly and quantify the signal using Image J or similar pixel quantification software. The differentiation protocol must be optimized before starting experiments to minimize the impact of unwanted cell types on the experimental outcome. Co-staining for markers of cell types of interest can also be done in parallel during experiments, where possible. Single-cell measurements may be possible through high-content imaging systems that can detect colocalization of hydrolase substrates with lysosomal markers and cell-type-specific markers.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Endo H digest/western blot: Inability to detect multiple glycosylated forms of the lysosomal protein by western blot.</p>
<h3 id="sec7.2">Potential solution 1</h3>
<p>Ensure that the antibody used for western probing is capable of binding to a region of the protein that is exposed/present during all stages of maturation and not cleaved during processing in the ER. Most proteins that move through the secretory pathway are cleaved in the ER then glycosylated. In some cases, it is best to use a polyclonal antibody that will bind specifically to multiple epitopes of the hydrolase to ensure that cleaved forms can be detected. Validating antibody specificity using a lysate where the hydrolase of interest is knock-out is optimal. Some commercially available antibodies against GCase only react with one particular form of the hydrolase and cannot be used to determine changes in trafficking.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Endo H digest/western blot: There are no mobility changes upon digestion with glycosidases.</p>
<h3 id="sec7.4">Potential solution 2</h3>
<p>This could be due to: (1) an inactive glycosidase enzyme, (2) interfering substance within the digest buffer, (3) failure of the antibody to detect all immature and mature forms of the hydrolase.</p>
<ol>
<li>Glycosidase activity can be validated with a control glycoprotein such as purified RNase B that is run on a separate gel, followed by silver staining to detect mobility differences. To preserve activity, the enzyme should be carefully stored at −20°C or lower, and freeze/thaw cycles should be avoided.</li>
<li>If the lysis buffer recipe provided above is used, there should be no interfering substances that inhibit glycosidase activity. If another lysis buffer is used, it should be confirmed to be compatible with the glycosidase protocol. In most cases, the components of the lysis buffer are diluted out by 10 to 20 fold by addition of water and 10× glycosidase buffer, which will also limit interference of enzyme activity.</li>
<li>As noted above, it is optimal to use a well characterized polyclonal antibody that is capable of detecting multiple epitopes that span different regions of the hydrolase, or use a monoclonal antibody that contains an epitope that is common to both immature and mature forms of the protein. The antibodies described above have been characterized and validated to be useful to detect multiple forms of each hydrolase. However, antibody reactivity and efficiency can change between lots and therefore should always be tested when a new stock of antibody is ordered.</li>
</ol>
<h3 id="sec7.5">Problem 3</h3>
<p>Immunofluorescence studies: Weak signal or high background is observed with immunofluorescence colocalization studies.</p>
<h3 id="sec7.6">Potential solution 3</h3>
<p>Make sure that the antibody used is capable of detecting the hydrolase of interest under fixed conditions. Fixing in 4% PFA can destroy the epitopes of some proteins, or certain epitopes may be buried or not exposed. An alternative fixation method could be employed in this case, which involves treating cells with ice-cold Methanol and Acetone (1:1 vol/vol mix) for 20 min followed by washing in PBS. Perform a positive control using an established antibody against an abundant protein such as tubulin or actin to confirm that fixation and the detection reagents worked properly. If available, overexpressing the hydrolase of choice with a plasmid will also provide an optimal positive control. Finally, high background signals may occur from sub-optimal settings on the fluorescence microscope. Using laser excitation is preferred compared to other sources such as Hg bulbs or LEDs since it can achieve a better signal to noise ratio.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Live-cell lysosomal assay: Lack of increase in hydrolase substrate fluorescence during the time course of the assay.</p>
<h3 id="sec7.8">Potential solution 4</h3>
<p>If cells are healthy, the vehicle (DMSO) treated wells should show a gradual elevation in signal over 3–4 h of measurements. If the signal remains flat over time, it is an indication of cellular toxicity. The cells should be examined before, after, and during the microplate readings to determine if cell death has occurred. Often, toxicity will occur from cells drying out from prolonged air exposure during the media changes. Toxicity will be observable from the neurofilament/CellTag700 staining, which will show a low intensity. If toxicity occurs, double check that all concentrations of DMSO, Baf A1, and substrates are correct. Changing media using a multi-channel aspirator and pipettor will decrease the time cells are exposed to air, limiting toxicity.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Live-cell lysosomal assay: The addition of Baf A1 does not inhibit enzyme activity.</p>
<h3 id="sec7.10">Potential solution 5</h3>
<p>The stock of Baf A1 should be verified to be active and not expired. This is most easily determined by quantification of the dextran blue signal, which should dramatically increase in Baf A1 treated cells. This is due to an inhibition of lysosomal activity, causing a back-up of the endolysosomal and autolysosomal pathways and enlargement of dextran blue containing vacuoles. If the Baf A1 is a newly ordered reagent or has been validated to be active through independent assays, it is possible that the wrong concentration was used or that toxicity occurred. While we have not observed toxicity at the concentration and incubation time noted above, it is possible that some patient lines will be more sensitive to lysosomal inhibition. If so, the toxicity assay will indicate cell or neurite loss. The assay should then be repeated using a lower dose of Baf A1.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cuddy, L.K., Wani, W.Y., Morella, M.L., Pitcairn, C., Tsutsumi, K., Fredriksen, K., Justman, C.J., Grammatopoulos, T.N., Belur, N.R., Zunke, F., et al. (2019). Stress-induced cellular clearance is mediated by the SNARE protein ykt6 and disrupted by alpha-synuclein. Neuron <i>104</i>, 869-884.e1. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00047-2/sref1">View at publisher</a></p>
<p id="bib2">Kim, J., Hu, C., Moufawad El Achkar, C., Black, L.E., Douville, J., Larson, A., Pendergast, M.K., Goldkind, S.F., Lee, E.A., Kuniholm, A., et al. (2019). Patient-customized oligonucleotide therapy for a rare genetic disease. N. Engl. J. Med. <i>381</i>, 1644-1652. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00047-2/sref2">View at publisher</a></p>
<p id="bib3">Kriks, S., Shim, J.W., Piao, J., Ganat, Y.M., Wakeman, D.R., Xie, Z., Carrillo-Reid, L., Auyeung, G., Antonacci, C., Buch, A., et al. (2011). Dopamine neurons derived from human ES cells efficiently engraft in animal models of Parkinson's disease. Nature <i>480</i>, 547-551. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00047-2/sref3">View at publisher</a></p>
<p id="bib4">Mazzulli, J.R., Zunke, F., Isacson, O., Studer, L., and Krainc, D. (2016a). alpha-Synuclein-induced lysosomal dysfunction occurs through disruptions in protein trafficking in human midbrain synucleinopathy models. Proc. Natl. Acad. Sci. U S A <i>113</i>, 1931-1936. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00047-2/sref4">View at publisher</a></p>
<p id="bib5">Mazzulli, J.R., Zunke, F., Tsunemi, T., Toker, N.J., Jeon, S., Burbulla, L.F., Patnaik, S., Sidransky, E., Marugan, J.J., Sue, C.M., et al. (2016b). Activation of beta-glucocerebrosidase reduces pathological alpha-synuclein and restores lysosomal function in Parkinson's patient midbrain neurons. J. Neurosci. <i>36</i>, 7693-7706. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00047-2/sref5">View at publisher</a></p>
<p id="bib6">Wiesmann, U.N., Lightbody, J., Vassella, F., and Herschkowitz, N.N. (1971). Multiple lysosomal enzyme deficiency due to enzyme leakage?. N. Engl. J. Med. <i>284</i>, 109-110. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00047-2/sref6">View at publisher</a></p>
<p id="bib7">Zunke, F. and Mazzulli, J.R. (2019). Modeling neuronopathic storage diseases with patient-derived culture systems. Neurobiol. Dis. <i>127</i>, 147-162. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00047-2/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Joseph R. Mazzulli, <a href="mailto:jmazzulli@northwestern.edu">jmazzulli@northwestern.edu</a></p>
<h4>Materials Availability</h4>
<p>All of the reagents described here are commercially available through the indicated vendors. There are no unique reagents or restrictions to availability of reagents.</p>
<h4>Data and Code Availability</h4>
<p>There are no unpublished custom code, software, or algorithms in this protocol.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants RF1NS109157 and R01NS092823 from the <a href="https://doi.org/10.13039/100000065">NINDS</a>/NIA to J.R.M.</p>
<h3>Author Contributions</h3>
<p>The protocols were written by J.R.M. and L.K.C.</p>
<h3>Declaration of Interests</h3>
<p>J.R.M. has received consulting fees from SK Biopharmaceuticals, Neuron23, Sanofi-Genzyme, and Lysosomal Therapeutics. J.R.M. has ownership/investment interests in Lysosomal Therapeutics.</p>
</section>
</article>