<article>
<section>
<h1 id="header">Direct IgG epitope mapping on bacterial AB toxins by cryo-EM</h1>
<p><time datetime="2021-10-01">Published: October 1, 2021</time></p>
<p>Tri Nguyen<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Jeongmin Song<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Microbiology &amp; Immunology, Cornell University, Ithaca, NY 14853, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:tn332@cornell.edu">tn332@cornell.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:jeongmin.song@cornell.edu">jeongmin.song@cornell.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100852">10.1016/j.xpro.2021.100852</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This cryo-EM protocol was used to determine the B cell epitope map on the CdtB subunit of typhoid toxin, an A<sub>2</sub>B<sub>5</sub> toxin secreted by <i>Salmonella</i> Typhi during infection. Immunoglobulin G (IgG) was directly mixed with typhoid toxin in this protocol, different from our previous cryo-EM protocol that uses the Fab fragments in place of IgG. This simple approach requires smaller amounts of materials, supporting the broader use of this protocol for determining antibody recognition sites on various antigens.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref2">Ahn et al. (2021)</a> and <a class="external-link" href="https://doi.org/10.1016/j.celrep.2021.109654">Nguyen et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>This protocol describes the entire procedures for determining antibody target sites</li>
<li>One bacterial AB toxin and toxin-recognizing mAb pair was used as an example</li>
<li>IgG can be directly used to determine the B cell epitope map</li>
<li>This protocol offers insights into other projects concerning mAb-antigen complexes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1056-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes specific steps for determining the B cell epitope map of IgG targeting the CdtB subunit of typhoid toxin.</p>
<h3 id="sec1.1">Buffer preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<h4 id="sec1.1.1">Antibody purification</h4>
<ol>
<li>Binding buffer – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 1 L of the Protein G binding buffer containing 20 mM sodium phosphate, pH 7.0.</p>
<ol start="2">
<li>Elution buffer – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 100 mL of the elution buffer containing 0.1 M glycine-HCl, pH 2.7.</p>
<ol start="3">
<li>Equilibration buffer – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 100 mL of the equilibration buffer containing 1 M Tris-HCl, pH 8.0.</p>
<h4 id="sec1.1.2">Antigen purification</h4>
<ol start="4">
<li>Buffer A – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 200 mL buffer A containing 15 mM Tris-HCl, pH 8.0, 150 mM NaCl, 20 mM imidazole.</p>
<table id="undtbl1">
<caption>Buffer A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl, pH 8.0</td>
<td>15 mM</td>
<td>3 mL</td>
</tr>
<tr>
<td>2 M NaCl</td>
<td>150 mM</td>
<td>15 mL</td>
</tr>
<tr>
<td>Imidazole</td>
<td>20 mM</td>
<td>0.27 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Make up the volume 200 mL with ddH<sub>2</sub>O</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 1 month</p>
<ol start="5">
<li>Buffer B – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 50 mL buffer B containing 20 mM Tris-HCl, pH 8.0, 200 mM NaCl, 300 mM imidazole.</p>
<table id="undtbl2">
<caption>Buffer B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl, pH 8.0</td>
<td>20 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>2 M NaCl</td>
<td>200 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Imidazole</td>
<td>300 mM</td>
<td>1.02 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Make up the volume 50 mL with ddH<sub>2</sub>O</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 1 month</p>
<ol start="6">
<li>Buffer C – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 1 L buffer C containing 10 mM 2-(N-morpholino)ethanesulfonic acid (MES), pH 6.0.</p>
<ol start="7">
<li>Buffer D – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 500 mL buffer D containing 10 mM MES, pH 6.0, 1 M NaCl.</p>
<ol start="8">
<li>Buffer E – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 500 mL buffer E containing 15 mM Tris-HCl, pH 8.0, 150 mM NaCl.</p>
<h4 id="sec1.1.3">Antibody-antigen complex preparation</h4>
<ol start="9">
<li>Buffer F – Store at 4°C for up to 1 month</li>
</ol>
<p>Prepare 1 L buffer F containing 15 mM Tris-HCl, pH 7.5 and 150 mM NaCl.</p>
<div class="critical">
<span class="critical-title">Critical:</span> When you make buffers, add only 75% of H<sub>2</sub>O and let the buffer cool for at least 18 hours at 4°C before adjusting pH.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Imidazole</td>
<td>Sigma</td>
<td>I0125</td>
</tr>
<tr>
<td>IPTG, Dioxane-Free</td>
<td>Thermo Fisher</td>
<td>R0393</td>
</tr>
<tr>
<td>Pierce Protease Inhibitor Tablet, EDTA-free</td>
<td>Thermo Fisher</td>
<td>A32965</td>
</tr>
<tr>
<td>Lysozyme from chicken egg white</td>
<td>Sigma</td>
<td>L6876</td>
</tr>
<tr>
<td>DNase I</td>
<td>Alfa Aesar</td>
<td>J62229</td>
</tr>
<tr>
<td>PROTEINDEX HiBond Ni-NTA Agarose 6 Fastflow</td>
<td>Marvelgent</td>
<td>11-0225-010</td>
</tr>
<tr>
<td>XK16/20 column</td>
<td>GE Scientific</td>
<td>28988937</td>
</tr>
<tr>
<td>HiTrap SP HP</td>
<td>Cytiva</td>
<td>17-1152-01</td>
</tr>
<tr>
<td>SD200 10/300GL Increase columns</td>
<td>Cytiva</td>
<td>28-9909-44</td>
</tr>
<tr>
<td>Pierce NHS-Activated Agarose</td>
<td>Thermo Fisher</td>
<td>26200</td>
</tr>
<tr>
<td>Bradford reagent</td>
<td>Bio-Rad</td>
<td>5000006</td>
</tr>
<tr>
<td>Amicon Ultra-15 30 kDa cutoff</td>
<td>Millipore</td>
<td>UFC903024</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RNeasy Mini Kit</td>
<td>QIAGEN</td>
<td>74104</td>
</tr>
<tr>
<td>iScript cDNA Synthesis Kit</td>
<td>Bio-Rad Laboratories</td>
<td>1708891</td>
</tr>
<tr>
<td>Herculase II Fusion DNA polymerase</td>
<td>Agilent</td>
<td>600677</td>
</tr>
<tr>
<td>Phusion DNA polymerase</td>
<td>New England Biolabs</td>
<td>M0530</td>
</tr>
<tr>
<td>QIAEX II DNA Extraction Kit</td>
<td>QIAGEN</td>
<td>20021</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Typhoid toxin bound to TyTx11 MAb</td>
<td>RCSB Protein Data Bank</td>
<td>PDB ID of 6VX4</td>
</tr>
<tr>
<td>Typhoid toxin bound to TyTx1 and TyTx4 Fab</td>
<td>RCSB Protein Data Bank</td>
<td>PDB ID of 7K7H and 7K7I</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Hybridomas</td>
<td>(<a href="#bib2">Ahn et al., 2021</a>; <a href="#bib6">Nguyen et al., 2021</a>)</td>
<td>See Methods Details of the referred papers</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>See <a href="#tbl1">Table 1</a>
</td>
<td>Integrated DNA Technologies (IDT)</td>
<td><a href="#tbl1">Table 1</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Unicorn 6.3</td>
<td>GE Healthcare Life Sciences</td>
<td><a href="https://www.gelifesciences.com/en/us/shop/unicorn-6-3-p-01118">https://www.gelifesciences.com/en/us/shop/unicorn-6-3-p-01118</a></td>
</tr>
<tr>
<td>CryoSparc 2</td>
<td>(<a href="#bib7">Punjani et al., 2017</a>)</td>
<td><a href="https://cryosparc.com">https://cryosparc.com</a></td>
</tr>
<tr>
<td>Relion 3</td>
<td>(<a href="#bib10">Zivanov et al., 2018</a>)</td>
<td><a href="https://www3.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page">https://www3.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page</a></td>
</tr>
<tr>
<td>PyMol</td>
<td>Schrodinger</td>
<td><a href="https://pymol.org/2/">https://pymol.org/2/</a></td>
</tr>
<tr>
<td>Chimera</td>
<td>UCSF Chimera</td>
<td><a href="https://www.cgl.ucsf.edu/chimera/">https://www.cgl.ucsf.edu/chimera/</a></td>
</tr>
<tr>
<td>PHENIX</td>
<td>(<a href="#bib1">Adams et al., 2010</a>)</td>
<td><a href="https://www.phenix-online.org/">https://www.phenix-online.org/</a></td>
</tr>
<tr>
<td>Coot</td>
<td>(<a href="#bib3">Emsley et al., 2010</a>)</td>
<td><a href="https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/">https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Below we provide a step-by-step protocol describing the sequencing of antibody variable regions, purification of antibody and antigen, and structure determination of the antibody-antigen complex (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1056-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Typical size-exclusion chromatograms showing antibody only (orange line), antigen only (blue line), and antibody-antigen complex (gray line).</div>
</figcaption>
</figure>
<h3 id="sec2.1">Antibody variable region sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>This step allows for the accurate determination of the amino acid sequences of the variable regions of the heavy and light chains of the antibody. These sequences are vital for the atomic structural model building and fitting into the electron density data from cryo-EM analyses. The antibody variable region sequencing protocol was adopted from (<a href="#bib4">Meyer et al., 2019</a>).</p>
<ol>
<li>Total RNA was extracted from the hybridoma cells using the RNeasy mini kit by following the vendor’s recommendations (Qiagen).</li>
<li>cDNA synthesis was performed using the iScript cDNA synthesis system (Bio-Rad) with the mouse IgG reverse transcription primers listed in <a href="#tbl1">Table 1</a>. The reaction condition is listed in <a href="#tbl2">Table 2</a>.
<table id="tbl1">
<caption>Table 1. Primer sequences used in this study</caption>
<thead>
<tr>
<th>Application</th>
<th>Primer name</th>
<th>Forward or reverse</th>
<th>Primer sequence (5′–3′)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="4">Reverse transcription (<a href="#bib4">Meyer et al., 2019</a>)</td>
<td>Template-switch oligo</td>
<td>Universal forward</td>
<td>AGGCAGTGGTATCAACGCAGAGTACATGrGrGr (GrGrGr: 3 riboguanines)</td>
</tr>
<tr>
<td>mIGK RT</td>
<td>Kappa chain reverse</td>
<td>TTGTCGTTCACTGCCATCAATC</td>
</tr>
<tr>
<td>mIGL RT</td>
<td>Lambda chain reverse</td>
<td>GGGGTACCATCTACCTTCCAG</td>
</tr>
<tr>
<td>mIGHG RT</td>
<td>Heavy chain reverse</td>
<td>AGCTGGGAAGGTGTGCACAC</td>
</tr>
<tr>
<td rowspan="4">PCR (<a href="#bib4">Meyer et al., 2019</a>)</td>
<td>ISPCR</td>
<td>Universal forward</td>
<td>AAGCAGTGGTATCAACGCAGAG</td>
</tr>
<tr>
<td>mIGK PCR</td>
<td>Kappa chain reverse</td>
<td>ACATTGATGTCTTTGGGGTAGAAG</td>
</tr>
<tr>
<td>mIGL PCR</td>
<td>Lambda chain reverse</td>
<td>ATCGTACACACCAGTGTGGC</td>
</tr>
<tr>
<td>mIGHG PCR</td>
<td>Heavy chain reverse</td>
<td>GGGATCCAGAGTTCCAGGTC</td>
</tr>
<tr>
<td rowspan="4">Gibson assembly</td>
<td>pET-F</td>
<td>Forward</td>
<td>CTCTGCGTTGATACCACTGCTTGATCCGGCTGCTAAC</td>
</tr>
<tr>
<td>pET-R for kappa chain</td>
<td>Reverse</td>
<td>CTTCTACCCCAAAGACATCAATGTCATGGTATATCTCCTTC</td>
</tr>
<tr>
<td>pET-R for lambda chain</td>
<td>Reverse</td>
<td>GCCACACTGGTGTGTACGATCATGGTATATCTCCTTC</td>
</tr>
<tr>
<td>pET-R for heavy chain</td>
<td>Reverse</td>
<td>GACCTGGAACTCTGGATCCCCATGGTATATCTCCTTC</td>
</tr>
<tr>
<td rowspan="2">Sequencing</td>
<td>T7 promoter</td>
<td>Forward</td>
<td>TAATACGACTCACTATAGGG</td>
</tr>
<tr>
<td>T7 terminator</td>
<td>Reverse</td>
<td>GCTAGTTATTGCTCAGCGG</td>
</tr>
</tbody>
</table>
<table id="tbl2">
<caption>Table 2. cDNA synthesis reaction</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Priming</td>
<td>25°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Reverse transcription</td>
<td>42°C</td>
<td>60 min</td>
<td>1</td>
</tr>
<tr>
<td>RT Inactivation</td>
<td>95°C</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>To amplify the antibody variable regions from the cDNA, a two-stage PCR reaction (<a href="#tbl3">Tables 3</a> and <a href="#tbl4">4</a>) was run with the universal forward PCR primer and reverse PCR primer based on the antibody chain (<a href="#tbl1">Table 1</a>).
<table id="tbl3">
<caption>Table 3. 1<sup>st</sup>-stage PCR amplification of the antibody variable region</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>15 s</td>
<td rowspan="3">10 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<table id="tbl4">
<caption>Table 4. 2<sup>nd</sup>-stage PCR reactions for the antibody variable region and vector</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>20 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min for insert,3 min for vector</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>The amplicons from the 1<sup>st</sup> PCR reaction (<a href="#tbl3">Table 3</a>) appeared between 550–600 base pairs on 1 % agarose gel.</li>
<li>The amplicon DNA was extracted from the gel using the QIAEX II DNA Extraction Kit by following the vendor’s recommendations (Qiagen).</li>
<li>The extracted DNA was used to set up the 2<sup>nd</sup>-stage PCR (<a href="#tbl4">Table 4</a>). The PCR amplicon was extracted and Gibson-assembled to the pET28a<sup>+</sup> plasmid (EMD Biosciences). The pET28a<sup>+</sup> vector amplicon was prepared using the primer pair matching the light and heavy chains (<a href="#tbl1">Tables 1</a> and <a href="#tbl4">4</a>).</li>
<li>The Gibson-assembled circular plasmid was transformed into <i>E. coli</i> DH5α, a single colony was isolated, and the isolated plasmid was Sanger-sequenced using either T7 promoter or T7 terminator primer (<a href="#tbl1">Table 1</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other plasmids and <i>E. coli</i> strains can be used.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The PCR amplicons could be directly sequenced to determine the antibody variable regions via Sanger sequencing. However, we found out that the sequencing outcomes were less satisfactory and counter-productive due to ambiguous base calls in our cases.</div>
<h3 id="sec2.2">Purification of antibody</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="8">
<li>Cell culture supernatants of the hybridomas generated as part of (<a href="#bib2">Ahn et al., 2021</a>; <a href="#bib6">Nguyen et al., 2021</a>) were prepared by centrifugation for 15 min at approximately 6,000 × <i>g</i> at 4°C using JLA8.1 rotor, followed by filtration through 0.2 μm cellulose acetate filters.</li>
<li>Protein G resins were packed in a 25 mL gravity plastic column and equilibrated by flowing 20 mL of the binding buffer (20 mM sodium phosphate buffer, pH 7.0).</li>
<li>The cleared hybridoma supernatants from step 8 were submitted to the prepared Protein G resin from step 9 three times.</li>
<li>The antibody-bound resins were washed with 25–50 mL of the binding buffer.</li>
<li>The bound antibodies were eluted by applying 10 mL of the elution buffer (0.1 M glycine-HCl, pH 2.7), which was harvested in a conical tube containing 1 mL of the equilibration buffer (1 M Tris-HCl, pH 8.0) for immediate neutralization.</li>
<li>The purity and yield of purified antibodies were monitored via 15% SDS-PAGE.</li>
<li>Purified antibodies were divided in 50 μL aliquots and stored at −80°C until use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Flash freezing with liquid nitrogen was not required in this case. Purified antibodies were stored stably without excipients such as glycerol. However, if desired, flash freezing and the addition of excipients like glycerol can be considered in this step.</div>
<div class="optional">
<span class="optional-title">Optional:</span> IgG was directly used for the complex structure determination for TyTx11 mAb (<a href="#bib2">Ahn et al., 2021</a>). However, if desired, the following steps can be carried out to prepare the Fab fragment, as described in (<a href="#bib6">Nguyen et al., 2021</a>).</div>
<ol start="15">
<li>Fab fragment generation:
<ol type="a">
<li>Concentrate purified mAbs to 20 mg/mL in 0.1 mL of a buffer containing 20 mM sodium phosphate, pH 7.0, and 10 mM EDTA.</li>
<li>Mix 250 μL of equilibrated immobilized Papain (ThermoFisher) with 1 mL of concentrated mAbs and incubate for at least 16 h at 37°C.</li>
<li>Elute the digested mAb samples in a buffer containing 10 mM Tris-HCl, pH 7.5.</li>
<li>Separate the Fab from uncut mAbs and the Fc by carrying out using a Superdex 75 10/300 Increase column (GE Healthcare) with a buffer containing 10 mM Tris-HCl, pH 7.5 at a constant flow of 0.5 mL/min.</li>
</ol>
</li>
</ol>
<h3 id="sec2.3">Purification of typhoid toxin (antigen)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>A highly purified antigen is vital for the generation and purification of the antigen-antibody complex. The uniformity of antigen and subsequent antigen-antibody complex improves the quality and reproducibility of data obtained during cryo-EM analysis.</p>
<ol start="16">
<li>Three subunits of typhoid toxin, PltB, PltA, and CdtB, were cloned together into pET28a<sup>+</sup> with a C-terminal His<sub>6</sub> tag (<a href="#bib8">Song et al., 2013</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The in-frame ORFs of PltB and PltA were cloned downstream of a T7 promoter provided by pET28a<sup>+</sup>. The ORF for CdtB was added downstream of PltB and PltA to be in-frame with the C-terminal His<sub>6</sub> tag presented on pET28a<sup>+</sup>. This design allowed for the expression of all three genes by the induction of T7 promoter.</div>
<ol start="17">
<li>Preparing bacterial cell pellet:
<ol type="a">
<li>Transform the typhoid toxin plasmid in <i>E. coli</i> BL21(DE3) competent cells.</li>
<li>Pick, inoculate, and culture a single colony in 10 mL of LB for at least 18 h at 37°C in a glass culture tube.</li>
<li>The next day, transfer 5 mL of the 10 mL bacterial culture into 50 mL of LB in a 250-mL baffled flask.</li>
<li>After 3 h of culture at 37°C, transfer 10 mL of this culture to each 1 L of LB in a 2-L baffled flask and continue to culture at 37°C, 200 rpm for 1.5–2 h.</li>
<li>When OD<sub>600</sub> reaches 0.5, change the setting to 28°C and 130 rpm.</li>
<li>After 30 min, add 250 μL of 0.5 M isopropyl β-D-1-thiogalactopyranoside (IPTG) to each culture (final concentration, 125 μM) and stir before placing the culture back to the incubator for 17–18 h.</li>
<li>Harvest the bacterial cells in a 1 L centrifuge bottle with 5000 × <i>g</i> for 15 min at 4°C.</li>
<li>Pour off the supernatant into an empty flask and bleached (10% v/v) to discard.
<div class="note">
<span class="note-title">Note:</span> If desired, scoop out the pellet and combine up to 2 flasks worth of pellets into a single 50 mL tube.</div>
</li>
<li>Store the pellet at −80°C if desired.</li>
</ol>
</li>
</ol>
<ol start="18">
<li>Bacteria lysis and centrifugal clarification:
<ol type="a">
<li>In a 50 mL conical tube containing bacterial pellets from up to 2 L culture, add 25 mL of buffer A and vortex to resuspend.</li>
<li>Add a protease inhibitor cocktail solution (final, 1x), 2 mg/mL of lysozyme, and 40 μg/mL of DNAse I and invert the tube several time to mix.</li>
<li>Subject the bacteria solution to 3 repeats of freeze-thaw in liquid nitrogen.</li>
<li>Lyse the cells using a 200-Watt ultrasonic homogenizer with a 1/8” probe (VWR).</li>
<li>Use the ‘Time – Pulse’ setup with the 75% power, 20 sec-on, and 20 sec-off for 4 min.</li>
<li>Clarify the lysate by centrifugation for 30 min at 18,000 × <i>g</i> at 4°C.</li>
</ol>
</li>
<li>Ni-Immobilized Metal Affinity chromatography (IMAC):
<ol type="a">
<li>Load the clarified lysate onto 10 mL of Ni-NTA equilibrated with buffer A.</li>
<li>Wash the column with 100 mL of buffer A.</li>
<li>Elute the protein with 50 mL of buffer B and collect 7 mL fractions.</li>
<li>Combine elute fractions containing desired protein and concentrate using Amicon columns, 30 kDa cutoff (ThermoFisher) until the volume reaches less than 5 mL.</li>
</ol>
</li>
</ol>
<ol start="20">
<li>HiTrap SP HP cation exchange chromatography:
<ol type="a">
<li>In a 50 mL conical tube, add 45 mL of buffer C and up to 5 mL of the concentrated Ni-IMAC eluate to lower the pH and salt level.</li>
<li>Load the protein solution onto a 5 mL SP HiTrap column equilibrated with buffer C.</li>
<li>Connect the column to the AKTA system.</li>
<li>Run the ‘Toxin SP – Opt (or any name that you prefer to use)’ program:<br> AKTA A-line equilibrated with buffer C, B-line equilibrated with buffer D.
<ol type="i">
<li>A flow rate of 1 mL/min.</li>
<li>Fractionation of 1 mL for 60 mL.</li>
<li>Gradient going from 0 to 40% buffer D for 50 mL.</li>
<li>At 50 mL, gradient going from 40% to 100% for 1 mL.</li>
<li>At 70 mL, gradient going from 100% to 0% for 1 mL.</li>
<li>At 90 mL, stop method.</li>
</ol>
</li>
<li>Run SDS-PAGE of the fractions containing UV-peaks to identify the fractions of interest.</li>
<li>Pool the fractions containing the toxin and concentrate using a 5 mL Amicon column, 30 kDa cutoff, by spinning at 3500 × <i>g</i> for ∼15 min at 4°C until the volume reaches 400–500 μL.</li>
</ol>
</li>
</ol>
<ol start="21">
<li>Size Exclusion Chromatography:
<div class="optional">
<span class="optional-title">Optional:</span> PBS can be used in place of buffer E if the downstream procedure is not compatible with Tris.</div>
<ol type="a">
<li>Equilibrate a Superdex 200 10/300GL increase column with buffer E.</li>
<li>Load the concentrated protein elute from the HiTrap SP chromatography (up to 500 μL) through the manual injection port of AKTA into the 500 μL sample loop.</li>
<li>Run the ‘Size Ex −500 μL (or any name that you prefer to use)‘ program: AKTA A-line equilibrated with buffer E
<ol type="i">
<li>A flow rate of 0.25 mL/min.</li>
<li>Fractionation of 0.5 mL for 28 mL.</li>
<li>Inject 0.5 mL from the 500 μL sample loop (Cytiva).</li>
<li>At 28 mL, stop method.
<div class="note">
<span class="note-title">Note:</span> In the case of typhoid toxin, you should see the UV peak around 12 mL–14.5 mL, which equates to an estimate of 110–120 kDa.</div>
</li>
</ol>
</li>
<li>Aliquot the toxin and freeze at −80°C until use.</li>
<li>Assess the toxin concentration and analyze on SDS-PAGE.
<div class="note">
<span class="note-title">Note:</span> Western blot and/or mass spectrometry can be carried out if further validation is desired.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If the antibody recognition site overlaps with the location of the tag used for protein purification, the tagless version of proteins can be prepared. The following is the procedure that we used to obtain the tagless typhoid toxin (<a href="#bib6">Nguyen et al., 2021</a>).</div>
</li>
</ol>
</li>
</ol>
<ol start="22">
<li>Preparation of a column packed with the TyTx1 antibody-conjugated agarose resin:
<ol type="a">
<li>Transfer 1 mL of the settled Pierce NHS-activated agarose resin (ThermoFisher, cat# 26200), wash with PBS, and incubate with PBS containing 14 mg of the TyTx1 antibody (<a href="#bib6">Nguyen et al., 2021</a>) for 1 h at 25°C with gentle rocking for the conjugation to occur.</li>
<li>Wash the resin four times with 5 mL of PBS.</li>
<li>Quench the resin by repeated washing with 10 mL of 0.5 M Tris-HCl, pH 8.0, and store at 4°C until use.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The resin should be stable for longer than a year. Proper handling of the resin including buffer selection and cleaning will ensure a long shelf-life of the resin.</div>
<ol start="23">
<li>Purification of the tagless typhoid toxin:
<ol type="a">
<li>Culture <i>E. coli</i> BL21(DE3) harboring pET28a<sup>+</sup> carrying tagless typhoid toxin subunits in LB to approximately OD<sub>600</sub>=0.7 at 37°C, transfer to 28°C, add with 0.5 mM IPTG, and incubate for 16 h.</li>
<li>Resuspend the harvested bacteria pellet in buffer E containing 1x EDTA-free protease inhibitor cocktail, 0.2 mg/mL lysozyme, and 80 μg/mL DNAse I, and subject to 3 times of the freeze-thaw step in liquid nitrogen, and lyse the bacterial cells by sonication. See steps 17–18 for details.</li>
<li>Clarify the lysate by centrifugation at 18,000 × <i>g</i> for 30 min at 4°C.</li>
<li>Mix the clarified lysate with the prepared TyTx1-conjugated agarose and incubate at 4°C for 30 min with gentle rocking.</li>
<li>Recapture the resin by passing the lysate through a gravity column and wash with buffer E.</li>
<li>Elute the protein with 5 mL of 100 mM Glycine, pH 3.0, and quickly neutralize the eluted protein by adding 1 mL of 1 M Tris-HCl, pH 8.8.</li>
<li>Apply the purified protein to size exclusion chromatography.</li>
</ol>
</li>
</ol>
<h3 id="sec2.4">Antibody-antigen complex and cryo-EM grid preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="24">
<li>Antibody-antigen complex preparation:
<ol type="a">
<li>Prepare TyTx11 mAbs bound to wild-type purified typhoid toxin (PltB, PltA, CdtB) by at least 16 h incubation of the 1:1 mixture of antibody and toxin (250 μL each of ∼1.2 mg/mL proteins) at 4°C.</li>
<li>Submit the antibody-antigen mixture to a Superdex 200 10/300 increase column (Cytiva) using a constant flow of 0.5 mL/min of buffer F (15 mM Tris-HCl, pH 7.5 and 150 mM NaCl).</li>
<li>Collect fractions corresponding to the antibody-antigen complex (<a href="#fig1">Figure 1</a>). Measure the concentrations using the Bradford solution (Bio-Rad, cat# 5000006) and use the purified complexes immediately for Cryo-EM grid/sample preparations using buffer F.</li>
</ol>
</li>
<li>Grid preparation:
<ol type="a">
<li>Dilute IgG-toxin complex samples with buffer F to a final concentration of 0.1 and 0.2 mg/mL providing a variable for grid screening.</li>
<li>Apply samples (3.5 μL) to glow-discharged copper Quantifoil R1.2/1.3 300 mesh cryo-EM grid (EMS) mounted, blotted, and plunge frozen in liquid ethane using a Vitrobot (ThermoFisher) at 100% humidity at 4°C.
<ol type="i">
<li>One of the common settings for Vitrobot was: Blot total – 1; Blot force – 1; Blot time – 2 to 4 s; Drain time −1 s; Wait time – 0 s.</li>
</ol>
</li>
<li>Store frozen grids in liquid nitrogen until analysis.</li>
</ol>
</li>
</ol>
<h3 id="sec2.5">Cryo-EM grid screening and data acquisition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="26">
<li>Grid Screening:
<ol type="a">
<li>Test two variables per antibody-toxin complex: blot times and protein concentrations.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Optimal protein concentrations and blot times may vary depending on samples, which can be empirically determined. As shown in <a href="#fig2">Figure 2</a>, good grids should have wells containing thin vitrified ice with good protein complex dispersal. In our case, using two different protein concentrations, 0.1 mg/mL and 0.2 mg/mL, we tested 4 different blot times: 2.5, 3, 3.5 and 4s to screen for good grid, resulting in 8 different grids. The initial screen indicated that a concentration of 0.1 mg/mL was suitable for imaging.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1056-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Cryo-EM sample and grid preparation</div>
<p>Examples of good and bad ice.</p>
<p>(A) The panel in the middle (thin vitrified ice) is a good example that typically shows protein particles with suitable dispersion.</p>
<p>(B) Samples with thin vitrified ice were examined for particle dispersion and marked for data collection (i). During data processing, suitable particle picking parameters were set to auto-pick most particles (ii). Subsequent rounds of particle picking were carried out to exclude non-protein debris particles (iii).</p>
</figcaption>
</figure>
<ol start="27">
<li>Data Acquisition:
<ol type="a">
<li>In our case, data were acquired at a nominal magnification of 63,000× for TyTx11 using a Talos Arctica (ThermoFisher) operating at 200 kV equipped with a Gatan energy filter set to a slit width of 20 eV and K3 detector operating in super-resolution counting mode using a defocus range of −0.8 to −1.8 μm.</li>
<li>In our case, 50 frame movies with a super-resolution pixel size of 1.23 were collected over a 4-s exposure, resulting in a total dose of 54.7 e<sup>-</sup> Å<sup>−2</sup> (1.06 e<sup>-</sup> Å<sup>−2</sup>/frame) for the TyTx11/Toxin complex.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To provide an example, in the case of our TyTx1 and TyTx4 study, we took approximately 150–200 movies that successfully yielded the complex structures for TyTx1 and TyTx4 (<a href="#bib6">Nguyen et al., 2021</a>).</div>
<div class="note">
<span class="note-title">Note:</span> A newer software like CryoSparc Live processes images captured by camera real-time, informing the researcher when to stop.</div>
<h3 id="sec2.6">Cryo-EM data processing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 weeks</div>
<p>The simplified version of the cryo-EM data processing workflow is shown in <a href="#fig3">Figure 3</a>. See Ahn et al. (<a href="#bib2">Ahn et al., 2021</a>) for additional details.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1056-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Typical workflow of cryo-EM data processing</div>
<p>See also Ahn et al. (<a href="#bib2">Ahn et al., 2021</a>) for 2D classification and other details.</p>
</figcaption>
</figure>
<ol start="28">
<li>Image import, motion correction, and contrast transfer function (CTF) estimation:
<ol type="a">
<li>Perform motion correction of each image stack using MotionCor2 function in RELION (<a href="#bib9">Zheng et al., 2017</a>; <a href="#bib5">Nakane et al., 2018</a>).</li>
<li>Following CTF estimation by CTFFIND-4.1 via Relion 3, subject the movies (i.e., a total of 935 movies collected during data acquisition) to Laplacian-of-Gaussian particle picking in RELION 3.</li>
</ol>
</li>
<li>Approximately 1000 particles were manually picked, extracted, and 2D classified to serve as initial templates for automated particle picking in RELION 3.</li>
<li>The resulting 1,009,626 particles were extracted at 3.0457 Å/pixel to reduce the memory requirements during computation and to speed up image processing (<a href="#fig3">Figure 3</a>).</li>
<li>The 2D classification was used to remove incorrectly picked particles (<a href="#fig3">Figure 3</a> left panel).</li>
<li>CryoSparc 2 was used to generate an 3D ab initio model from the cleaned particle set.</li>
<li>This 3D ab initio model was used as a reference map for subsequent 3D classifications and refinement in RELION 3.</li>
<li>Initial 3D classification revealed three out of five classes containing particles resembling the structure of Fab (<a href="#fig3">Figure 3</a> right panel).</li>
<li>The particles from those classes were combined and submitted to another round of 3D classification, which revealed one of four classes contained high-quality particles resulting in a high-resolution estimate for the model (<a href="#fig3">Figure 3</a> middle-right panel).</li>
<li>Particles from this class were then used for subsequent per-particle CTF refinements and 3D refinement (<a href="#fig3">Figure 3</a> bottom-right panel).</li>
<li>When no further improvement could be made, the particles were then un-binned and re-extracted at 1.23 Å/pixel.</li>
<li>After subsequent CTF refinements and 3D refinements, a Bayesian polishing step followed by a final 3D classification was used to remove any last junk particles.</li>
<li>Final reconstruction using 61,478 particles was generated. The final 3D reconstruction has a 0.143 FSC cutoff resolution of 3.12 Å according to RELION3 (Figure 43 bottom left panel).</li>
</ol>
<h3 id="sec2.7">Structural model building</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="40">
<li>Chimera (<a href="http://www.cgl.ucsf.edu">www.cgl.ucsf.edu</a>) was used to place the crystal structures of typhoid toxin (PDB 4K6L) and PltB pentamer (PDB 4RHR) into the sharpened reconstructions as preliminary atomic models. See Ahn et al. (<a href="#bib2">Ahn et al., 2021</a>) for additional details.</li>
<li>The preliminary atomic models for TyTx11 Fab variable region were obtained from the following models selected based on sequence similarity through BLAST: the light chain from PDB 6GFF and the heavy chain from PDB 1F11. The constant regions of the TyTx11 were too flexible for the proper density fitting and thus left unbuilt in the final deposited model.</li>
<li>Coot (<a href="http://www.biop.ox.ac.uk/coot">www.biop.ox.ac.uk/coot</a>) was then used to manually rebuild these preliminary models. The flexibility of the antibody constant regions, CdtB subunit, and the majority of the PltA subunit prevented us from confidently building these portions of the maps, so they were left unbuilt.</li>
<li>Cyclical refinement step involving model manually modified in Coot was submitted to automated real-space refinement in Phenix (<a href="http://phenix-online.org">phenix-online.org</a>), which was manually modified in Coot to lower clashes and fix outliers. The quality of the final models was validated using MolProbity (<a href="http://molprobity.biochem.duke.edu">molprobity.biochem.duke.edu</a>).</li>
<li>Figures were generated using Chimera and Pymol (Schrodinger, Inc). Cryo-EM density maps and refined complex structures were deposited under EMD-21429 and PDB 6VX4 for the TyTx11 MAb-toxin complex.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Here, we have described step-by-step experimental procedures that we used to obtain the sequences of antibody variable regions, purifications of antibody and antigen, and structure determination of the antibody-antigen complex. This protocol provides practical insights into other projects aimed at solving the structures of the antibody and bacterial toxin complexes.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Obtaining ultrapure antibodies and antigens will be vital to high-resolution B cell epitope mapping of antibodies. The uniformity of antigen and subsequent antigen-antibody complexes will also be critical to the quality and reproducibility of data obtained during cryo-EM analysis.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>The direct use of IgG has several advantages such as the requirement of smaller amounts of materials, less time and labor. Also note that in some cases that need to be determined empirically, the use of IgG is required as the Fab fragment is not stable. In addition, the use of IgG can improve the signal contrast for particle picking and help avoid orientation bias among particle groups. Conversely, the use of the Fab fragment can be required for some cases and therefore the use of Fabs and IgG can be empirically determined depending on the experimental needs and the questions to address.</p>
<p>In step 24, if the direct use of IgG is not suitable for yielding reliable antigen-antibody complexes, the Fab fragments can be prepared and used in place of IgG.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Detailed procedures relevant to obtaining the Fab fragments are described in step 15 using two antibodies, TyTx1 and TyTx4, as examples.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>In step 24, the hexahistidine tag used for antigen purification might be located on/near the epitope recognized by the antibody. Standard ELISA can address whether the tag location and amino acid residues recognized by the antibody overlap.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Suppose the antibody recognition site overlaps with the location of the epitope tag used for protein purification. In that case, it can be resolved by preparing the tagless version of proteins or relocating the tag to somewhere else on the antigen.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>In step 4, the PCR amplicons could be directly sequenced in antibody sequencing to determine the antibody variable regions via Sanger sequencing. However, the direct sequencing of PCR amplicons might yield unsatisfactory outcomes.</p>
<h3 id="sec5.6">Potential solution</h3>
<p>As described in steps 6–7, PCR amplicons can be inserted into a plasmid before sequencing the antibody variable region.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>In step 26, the grids screening variables might not yield any good grids.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Good grids and subsequent good dataset can be varied from proteins to proteins. Depending on the size of the protein complex, protein concentrations can be altered for better particle dispersions. Other factors to optimize include tweezer handling of grids, awareness of cryogenic fluids throughout the preparation steps, and careful transfer of grids to cryo-EM machine. Seasonal awareness of humidity can also help optimize blot-time for ice formation.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>In step 29, during particle picking, depending on the proteins, a large portion of particles can appear clumping in large patches in the micrographs and therefore the number of single particles appear to be insufficient for picking.</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Instead of picking only isolated single particles, both isolated single particles and particles found in clump patches can be used by performing the following steps. Both types of particles are picked manually, followed by automated picks using a 2D template generated by the initial manual picks. For instance, after picking around 1,000 particles from different defocus, a 2D template can be generated and served a guide for automated particle picks. 2D and 3D classification will improve the signal to noise ratio in the particles found in clumps.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Adams, P.D., Afonine, P.V., Bunkoczi, G., Chen, V.B., Davis, I.W., Echols, N., Headd, J.J., Hung, L.W., Kapral, G.J., Grosse-Kunstleve, R.W., et al. (2010). PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D Biol. Crystallogr. <i>66</i>, 213-221. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref1">View at publisher</a></p>
<p id="bib2">Ahn, C., Yang, Y.-A., Neupane, D.P., Nguyen, T., Richards, A.F., Sim, J.H., Mantis, N.J., and Song, J. (2021). Mechanisms of typhoid toxin neutralization by antibodies targeting glycan receptor binding and nuclease subunits. iScience <i>24</i>, 102454. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref2">View at publisher</a></p>
<p id="bib3">Emsley, P., Lohkamp, B., Scott, W.G., and Cowtan, K. (2010). Features and development of Coot. Acta Crystallogr. D Biol. Crystallogr. <i>66</i>, 486-501. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref3">View at publisher</a></p>
<p id="bib4">Meyer, L., Lopez, T., Espinosa, R., Arias, C.F., Vollmers, C., and Dubois, R.M. (2019). A simplified workflow for monoclonal antibody sequencing. PLoS One <i>14</i>, e0218717. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref4">View at publisher</a></p>
<p id="bib5">Nakane, T., Kimanius, D., Lindahl, E., and Scheres, S.H. (2018). Characterisation of molecular motions in cryo-EM single-particle data by multi-body refinement in RELION. Elife <i>7</i>, e36861. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref5">View at publisher</a></p>
<p id="bib6">Nguyen, T., Richards, A.F., Neupane, D.P., Feathers, J.R., Yang, Y.A., Sim, J.H., Byun, H., Lee, S., Ahn, C., Van Slyke, G., Fromme, J.C., Mantis, N.J., and Song, J. (2021). The structural basis of Salmonella A2B5 toxin neutralization by antibodies targeting the glycan-receptor binding subunits. Cell Rep. <i>36</i>. . <a class="external-link" href="https://doi.org/10.1016/j.celrep.2021.109654">https://doi.org/10.1016/j.celrep.2021.109654</a></p>
<p id="bib7">Punjani, A., Rubinstein, J.L., Fleet, D.J., and Brubaker, M.A. (2017). cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Nat. Methods <i>14</i>, 290. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref7">View at publisher</a></p>
<p id="bib8">Song, J., Gao, X., and Galan, J.E. (2013). Structure and function of the Salmonella Typhi chimaeric A(2)B(5) typhoid toxin. Nature <i>499</i>, 350-354. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref8">View at publisher</a></p>
<p id="bib9">Zheng, S.Q., Palovcak, E., Armache, J.P., Verba, K.A., Cheng, Y., and Agard, D.A. (2017). MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods <i>14</i>, 331-332. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref9">View at publisher</a></p>
<p id="bib10">Zivanov, J., Nakane, T., Forsberg, B.O., Kimanius, D., Hagen, W.J., Lindahl, E., and Scheres, S.H. (2018). New tools for automated high-resolution cryo-EM structure determination in RELION-3. Elife <i>7</i>, e42166. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00558-X/sref10">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jeongmin Song (<a href="mailto:jeongmin.song@cornell.edu">jeongmin.song@cornell.edu</a>).</p>
<h4>Materials availability</h4>
<p>Unique and stable reagents generated as part of this study are available upon request via an appropriate material transfer agreement.</p>
<h4>Data and code availability</h4>
<p>The cryo-EM map included in this study has been deposited in the Electron Microscopy Data Bank with accession code: EMD-21429. The atomic coordinate of TyTx11 bound to typhoid toxin has been deposited in the RCSB Protein Data Bank under PDB ID: 6VX4. The cryo-EM maps of TyTx1 and TyTx4 bound to typhoid toxin, where we used the Fab fragments in place of IgG, have also been deposited in the EMDB with accession codes: EMD-22699 and EMD-22700. The atomic coordinates have been deposited in the PDB ID: 7K7H and 7K7I.</p>
<h3>Acknowledgments</h3>
<p>We thank Nicholas J. Mantis, Angelene F. Richards, and Greta Van Slyke for the generation and characterization of the hybridomas producing anti-PltB or anti-CdtB antibodies; J. Christopher Fromme and J. Ryan Feathers for their inputs on cryo-EM data analysis; and Mariena S. Ramos and Katie Spoth of the Cornell Center for Material Research (CCMR) for their assistance in using FEI Vitrobot and Talos Artica S/TEM. This study was supported in part by a multi-PI R03 award from <a href="https://doi.org/10.13039/100000060">NIAID</a> (AI135767) to J.S. and N.J.M.; <a href="https://doi.org/10.13039/100000002">NIH</a> R01 AI137345, AI139625, and AI141514 to J.S.; and <a href="https://doi.org/10.13039/100000001">NSF</a> DMR-1719875 to the CCMR. The funders had no role in the study design, data collection, analysis, decision to publish, or manuscript preparation.</p>
<h3>Author contributions</h3>
<p>T.N. conducted the experiment relevant to the procedures described in this protocol. J.S. and T.N. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>