<article>
<section>
<h1 id="header">Differentiation of human pluripotent stem cells into pancreatic duct-like organoids</h1>
<p><time datetime="2021-12-10">Published: December 10, 2021</time></p>
<p>Markus Breunig,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Jessica Merkle,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Michael Karl Melzer,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Sandra Heller,<sup><a href="#aff1">1</a></sup> Thomas Seufferlein,<sup><a href="#aff1">1</a></sup> Matthias Meier,<sup><a href="#aff3">3</a></sup> Meike Hohwieler,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor1">*</a></sup> and Alexander Kleger<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#fn4">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Internal Medicine I, Ulm University Hospital, 89081 Ulm, Germany</p>
<p id="aff2"><sup>2</sup>Department of Urology, Ulm University, 89081 Ulm, Germany</p>
<p id="aff3"><sup>3</sup>Helmholtz Pioneer Campus, Helmholtz Zentrum München, 85764 München, Germany</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Senior authors</p>
<p id="fn3"><sup>6</sup>Technical contact</p>
<p id="fn4"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:meike.hohwieler@uni-ulm.de">meike.hohwieler@uni-ulm.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:alexander.kleger@uni-ulm.de">alexander.kleger@uni-ulm.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100913">10.1016/j.xpro.2021.100913</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The recapitulation of human developmental processes and pathological manifestations requires access to specific cell types and precursor stages during embryogenesis and disease. Here, we describe a scalable <i>in vitro</i> differentiation protocol to guide human pluripotent stem cells stepwise into pancreatic duct-like organoids. The protocol mimics pancreatic duct development and was successfully used to model the onset and progression of pancreatic ductal adenocarcinoma; the approach is suitable for multiple downstream applications. However, the protocol is cost- and time-intensive.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00619-5/sref1">Breunig et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Step-by step protocol to generate pancreatic duct-like organoids (PDLOs) from hPSCs</li>
<li>Recovery, processing, and splitting of the 3D Matrigel organoid culture</li>
<li>
<i>In vitro</i> differentiation mimics pancreatic duct development</li>
<li>Successfully used to model the onset and progression of pancreatic cancer</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1219-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Culture of pluripotent stem cells</h3>
<p>The hereafter described protocol provides detailed instructions for generating pancreatic duct-like organoids (PDLOs) from human pluripotent stem cells (hPSCs). The protocol was initially established for the human embryonic stem cell (hESC) line HUES8 but was successfully applied to several other hPSC lines. The culture and maintenance of hPSCs is central for an efficient differentiation. Stem cells were kept as single cell culture in mTeSR1 or mTeSR Plus medium on wells coated with hESC-qualified Matrigel, diluted in DMEM/F12 with 0.1% HSA according to manufacturer’s recommendations and batch-specific dilution factors (<a href="https://www.corning.com/emea/en/products/life-sciences/resource-library.html">https://www.corning.com/emea/en/products/life-sciences/resource-library.html</a>). Further details on the culture of hPSCs and the generation of iPSCs can be found in <a href="#bib1">Breunig et al. (2021)</a>. Before initializing a differentiation, hPSCs were kept in culture for at least one week with two passaging steps using TrypLE Select or TrypLE Express (equivalent) for cell detachment and singularization of cells. Please ensure that all required legal permissions from national authorities and approval from institutional review boards are in place.</p>
<h3 id="sec1.2">Growth factor, cytokine, small molecules, and solution preparation</h3>
<ol>
<li>Matrigel, GFR
<ol type="a">
<li>Thaw Matrigel vial on ice for 10–16 h at 4°C.</li>
<li>Use precooled tips and precooled reaction tubes to make 1 mL aliquots.</li>
<li>Store aliquots at −20°C, stable for around 2 years (see certificate of analysis for each batch).</li>
<li>Before use, thaw the required number of aliquots on ice (takes around 30 min to 1 h).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We have used only Matrigel protein concentrations between 8.8 and 11.3 mg/mL and the endotoxin content should be 1.5 EU/mL or lower.</div>
<ol start="2">
<li>0.1% BSA in PBS for preparing stock solutions
<ol type="a">
<li>Dissolve 30 mg BSA in 30 mL PBS (without Ca<sup>2+</sup> and Mg<sup>2+</sup>).</li>
<li>Mix by pipetting up and down.</li>
<li>Collect the solution in a 30-mL syringe and filtrate it through a 0.22-μm filter.</li>
<li>Make 1 mL aliquots and store at −20° for up to 2 years.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>4 mM HCl solution for preparing stock solutions
<ol type="a">
<li>Add 3.32 μL of 37% HCl to 10 mL of ddH<sub>2</sub>O for a 4 mM HCl solution.</li>
<li>Collect the solution in a 15-mL syringe and filtrate it through a 0.22-μm filter.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>5 mM sodium phosphate buffer for preparing stock solutions
<ol type="a">
<li>Prepare 5 mM sodium phosphate buffer (pH 7.4) by adding 101 mg of Na<sub>2</sub>HPO<sub>4</sub>•7H<sub>2</sub>O and 17 mg of NaH<sub>2</sub>PO<sub>4</sub>•H<sub>2</sub>O to 80 mL of ddH<sub>2</sub>O.</li>
<li>Mix on a magnetic stirrer until completely dissolved.</li>
<li>Adjust the pH to 7.4 with HCl or NaOH.</li>
<li>Fill up with ddH<sub>2</sub>O for a final volume of 100 mL.</li>
<li>Filter the solution through a 0.22-μm filter.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>ROCK inhibitor Y-27632 (10 mM)
<div class="note">
<span class="note-title">Note:</span> The specific molecular weight varies from batch to batch. Therefore, volumes need to be adjusted accordingly.</div>
<ol type="a">
<li>Exemplary volumes are calculated for a molecular weight of 338.28 g/mol.</li>
<li>Centrifuge vial prior to opening.</li>
<li>10 mg are dissolved in 2.956 mL sterile water.</li>
<li>Mix by pipetting up and down.</li>
<li>Make 100 µL aliquots and store at -20°C, stable for 6 months.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Activin A (100 μg/mL, PeproTech)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Add 250 μL sterile ddH<sub>2</sub>O to 250 μg Activin A.</li>
<li>Mix by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Add additionally 2250 μL 0.1% BSA/PBS.</li>
<li>Mix until homogeneously dissolved.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Make 50 μL aliquots and store at −20°C, stable for 12 months.</li>
</ol>
</li>
</ol>
<ol start="7">
<li>
<b><i>Alternatives:</i></b> Activin A (100 μg/mL, R&amp;D)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Reconstitute 50 μg Activin A with 500 μL 4 mM HCl solution<b>.</b>
</li>
<li>Mix by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Make 30 μL aliquots and store at −20°C, stable for 3 months.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>CHIR-99021 (10 mM)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Add 1074 μL DMSO to 5 mg CHIR-99021.</li>
<li>Pipet up and down until dissolved.</li>
<li>Make 50 μL aliquots and store at −20°C, stability after reconstitution is not indicated by the company.</li>
</ol>
</li>
</ol>
<ol start="9">
<li>FGF2 (50 μg/mL)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Add 500 μL 0.1% BSA/PBS to 25 μg.</li>
<li>Mix by pipetting up and down.</li>
<li>Make 50 μL aliquots and store at −20°C, stable for 3 months.</li>
</ol>
</li>
</ol>
<ol start="10">
<li>FGF10 (50 μg/mL, R&amp;D)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Add 5 mL 0.1% BSA/PBS to 250 μg FGF10.</li>
<li>Carefully dissolve by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Make 100 μL aliquots and store at −20°C, stable for 3 months.</li>
</ol>
</li>
<li>
<b><i>Alternatives:</i></b> FGF10 (50 μg/mL, PeproTech)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Add 5 mL 5 mM sodium phosphate buffer to 500 μg FGF10.</li>
<li>Carefully dissolve by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Further dilute 1:2 by adding 5 mL of 0.1% BSA/PBS.</li>
<li>Mix by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Make 100 μL aliquots and store at −20°C, stable for 3 months.</li>
</ol>
</li>
</ol>
<ol start="12">
<li>Dorsomorphin (5 mM)
<div class="note">
<span class="note-title">Note:</span> The specific molecular weight varies from batch to batch depending on the degree of hydration. Therefore, volumes need to be adjusted accordingly.</div>
<ol type="a">
<li>Exemplary volumes are calculated for a molecular weight of 399.49 g/mol.</li>
<li>Centrifuge vial prior to opening.</li>
<li>5 mg are dissolved in 2.5 mL DMSO.</li>
<li>Mix by pipetting up and down.</li>
<li>Make 50 μL aliquots and store at −20°C, stability after reconstitution is not indicated by the company.</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Wnt3A (25 μg/mL)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Dissolve 10 μg in 100 μL ddH<sub>2</sub>O.</li>
<li>Dissolve by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Add 300 μL 0.1% BSA/PBS.</li>
<li>Mix by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Make 15 μL aliquots and store at −20°C, stable for 3 months.</li>
</ol>
</li>
</ol>
<ol start="14">
<li>LDN-193189 (1 mM)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Dissolve 5 mg LDN in 12.3 mL ddH<sub>2</sub>O.</li>
<li>Mix by pipetting up and down.</li>
<li>Make 100 μL aliquots and store at −20°C, stability after reconstitution is not indicated by the company.</li>
</ol>
</li>
</ol>
<ol start="15">
<li>SANT-1 (1 mM)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Add 13.39 mL DMSO to 5 mg of SANT-1.</li>
<li>Pipet up and down until completely resolved.</li>
<li>Make 100 μL aliquots and store at −20°C, stability after reconstitution is not indicated by the company.</li>
</ol>
</li>
</ol>
<ol start="16">
<li>Retinoic Acid (10 mM)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>For a 10 mM stock concentration dissolve 50 mg in 16.64 mL DMSO.</li>
<li>Pipet up and down until completely resolved.</li>
<li>Make 150 μL aliquots and store at −20°C, stability after reconstitution is not indicated by the company.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Protect from light.</div>
<ol start="17">
<li>Glucose (1 M)
<ol type="a">
<li>Weigh 9 g of cell culture qualified glucose to 50 mL of ddH<sub>2</sub>O.</li>
<li>Collect the solution in a 50-mL syringe and filter it through a 0.22-μm filter to make it sterile.</li>
<li>Store at 4°C, stable for 12 months.</li>
</ol>
</li>
</ol>
<ol start="18">
<li>EGF (50 μg/mL)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>First dissolve 200 μg of EGF in 400 μL PBS by pipetting up and down.</li>
<li>Then add 3600 μL PBS.</li>
<li>Mix by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Make 100 μL aliquots and store at −20°C, stable for 3 months.</li>
</ol>
</li>
</ol>
<ol start="19">
<li>Nicotinamide (1 M)
<ol type="a">
<li>Weigh 2.44 g Nicotinamide and dissolve in 18 mL ddH<sub>2</sub>O.</li>
<li>Fill up with ddH<sub>2</sub>O for a total volume of 20 mL.</li>
<li>Collect the solution in a 30-mL syringe and filter it through a 0.22-μm filter for sterilization.</li>
<li>Make 1 mL aliquots and store at −20°C, stability after reconstitution is not indicated by the company.</li>
</ol>
</li>
</ol>
<ol start="20">
<li>Indolactam V (660 μM)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>Add 1508 μL DMSO to 300 μg Indolactam V.</li>
<li>Dissolve by pipetting up and down.</li>
<li>Make 100 μL aliquots and store at −20°C, stability after reconstitution is not indicated by the company.</li>
</ol>
</li>
</ol>
<ol start="21">
<li>ZnSO4 (10 mM)
<ol type="a">
<li>Weigh 287.56 mg ZnSO4 and dissolve in 100 mL ddH2O.</li>
<li>Collect the solution in a 50-mL syringe and filter it through a 0.22-μm filter to make it sterile.</li>
<li>Store at 4 °C, stability after reconstitution is not indicated by the company.</li>
</ol>
</li>
</ol>
<ol start="22">
<li>KGF (50 μg/mL)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>First dissolve 100 µg KGF in 100 µL PBS by pipetting up and down.</li>
<li>Then add 1900 µL 0.1 % BSA/PBS.</li>
<li>Mix by pipetting up and down.
<div class="note">
<span class="note-title">Note:</span> Do not vortex.</div>
</li>
<li>Make 50 µL aliquots and store at -20°C, stable for 3 months.</li>
</ol>
</li>
</ol>
<ol start="23">
<li>MSC2530818 (5 mM)
<ol type="a">
<li>Centrifuge vial prior to opening.</li>
<li>For a 5 mM stock concentration dissolve 5 mg MSC2530818 in 2.934 mL DMSO.</li>
<li>Pipet up and down until completely resolved.</li>
<li>Make 30 µL and store at -80°C, stable for 2 years.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Due to extensive costs, the in this section described working solutions were successfully used for 12 months (stored at −20°C or −80°C), albeit some companies do not guarantee stability for more than 3–6 months for some compounds (indicated for each compound above). We strictly avoid thawing and re-freezing of compounds but store compounds for around 10 days at 4°C. Please see specific notes for the usage of GFR Matrigel above. Differentiation efficiency has to be tightly controlled by including standard hPSC, such as HUES8, or iPSC lines with known high differentiation capacities in the experiment in parallel as quality control.</div>
<h3 id="sec1.3">Preparation of solutions for splitting of organoids and downstream applications</h3>
<ol start="24">
<li>Collagenase/dispase solution (Stock: 100 mg/mL, working solution: 1 mg/mL)
<ol type="a">
<li>Add 5 mL dH<sub>2</sub>O to 500 mg collagenase/dispase.</li>
<li>Mix by pipetting up and down.</li>
<li>Make 1 mL stock solutions and store them at −20°C.</li>
<li>For preparation of working solutions, defrost an aliquot of the stock solution and dilute collagenase/dispase 1:100 in DMEM/F12. Freshly prepare working solutions before use.</li>
<li>Filter the solution through a 0.22 μm filter and collect it in a falcon.</li>
<li>Store the remaining collagenase/dispase stock solution at 4°C for up to 4 weeks.</li>
</ol>
</li>
</ol>
<ol start="25">
<li>Neutralization solution (1% BSA, 1% Penicillin/Streptomycin (P/S) in DMEM/F12)
<ol type="a">
<li>Add 5 mL 100<b>×</b> P/S solution to 495 mL of DMEM/F12.</li>
<li>Add 5 g fatty-acid-free BSA to the solution.</li>
<li>Incubate on a shaker until BSA is completely dissolved.</li>
<li>Filter the solution through a 0.22 μm filter and collect it in a sterile glass bottle.</li>
<li>Store the solution at 4°C for up to 6 months.</li>
</ol>
</li>
</ol>
<ol start="26">
<li>1 M sucrose solution
<ol type="a">
<li>Add 342.3 g sucrose to 1 L PBS.</li>
<li>Mix on a magnetic stirrer until completely dissolved.</li>
<li>Make 50 mL aliquots and store them at −20°C for up to 1 year.</li>
</ol>
</li>
</ol>
<ol start="27">
<li>4% PFA solution with 100 mM sucrose
<ol type="a">
<li>Add 20 g paraformaldehyde (PFA) to 400 mL of PBS (without Ca<sup>2+</sup> and Mg<sup>2+</sup>).</li>
<li>Add 32 drops of 1 M NaOH with a 1 mL serological pipette (around 1.1 mL).</li>
<li>Put the solution at 56°C in a water bath and incubate until the solution is clear.</li>
<li>Wait until the solution is cooled down to 20°C–23°C.</li>
<li>Adjust the pH to 7.3 with 37% HCl.</li>
<li>Fill up with PBS to reach a volume of 450 mL.</li>
<li>Add 50 mL 1 M sucrose and mix by pipetting up and down.</li>
<li>Make 50 mL aliquots and store them at −20°C for up to 1 year.</li>
</ol>
</li>
</ol>
<ol start="28">
<li>1% BSA in PBS
<ol type="a">
<li>Add 0.5 g BSA to 50 mL PBS.</li>
<li>Mix on a shaker until the BSA is completely dissolved.</li>
<li>The solution can be stored for several weeks at 4°C.</li>
</ol>
</li>
</ol>
<ol start="29">
<li>2% agarose in PBS
<ol type="a">
<li>Add 2 g agarose to 100 mL of PBS.</li>
<li>Boil in the microwave.</li>
<li>Solution can be cooled down and stored at 20°C–23°C.</li>
<li>Boil before reusing.</li>
</ol>
</li>
</ol>
<ol start="30">
<li>25% sucrose in PBS
<ol type="a">
<li>Dissolve 12.5 g sucrose in 40 mL PBS.</li>
<li>Mix on a shaker until completely dissolved.</li>
<li>Fill up to a final volume of 50 mL with PBS.</li>
<li>Store at 4°C for up to 6 months.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>CXCR4 antibody (1:50),Mouse monoclonal PE-conjugated</td>
<td>Life Technologies</td>
<td>Cat# MHCXCR404; RRID:AB_10373097</td>
</tr>
<tr>
<td>EPCAM antibody (1:33.33),Mouse monoclonal APC-conjugated</td>
<td>BD</td>
<td>Cat# 347200; RRID:AB_400570</td>
</tr>
<tr>
<td>GATA6 (1:500),Rabbit monoclonal unconjugated</td>
<td>Cell Signaling Technology</td>
<td>Cat# 5851;RRID:AB_10705521</td>
</tr>
<tr>
<td>GP2 (1:5000),Mouse monoclonal unconjugated</td>
<td>MBL International</td>
<td>Cat# D277-3;RRID:AB_10598500</td>
</tr>
<tr>
<td>KIT antibody (1:100),Mouse monoclonal APC-conjugated</td>
<td>Thermo</td>
<td>Cat# CD11705; RRID:AB_2536476</td>
</tr>
<tr>
<td>NKX6-1 (1:150-&gt;final conc. 1μg/mL),Mouse monoclonal unconjugated</td>
<td>Developmental Studies Hybridoma Bank (DSHB)</td>
<td>Cat# F55A12 concentrate; RRID:AB_532379</td>
</tr>
<tr>
<td>NKX6-1 (1:35),Mouse monoclonal APC-conjugated</td>
<td>BD</td>
<td>Cat# 563338; RRID:AB_2738144</td>
</tr>
<tr>
<td>PDX1 (1:500),Goat polyclonal unconjugated</td>
<td>R&amp;D</td>
<td>Cat# AF2419; RRID:AB_355257</td>
</tr>
<tr>
<td>PDX1-PE (1:35),Mouse monoclonal unconjugated</td>
<td>BD</td>
<td>Cat# 562161; RRID:AB_10893589</td>
</tr>
<tr>
<td>SOX17 (1:500),Goat polyclonal unconjugated</td>
<td>R&amp;D</td>
<td>Cat# AF1924;RRID:AB_355060</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Accutase</td>
<td>Merck</td>
<td>Cat# A6964</td>
</tr>
<tr>
<td>Activin A</td>
<td>PeproTech</td>
<td>Cat# 120-14;SDS: 25-120-14</td>
</tr>
<tr>
<td>Activin A</td>
<td>R&amp;D</td>
<td>Cat# 338-AC</td>
</tr>
<tr>
<td>Agarose</td>
<td>Merck</td>
<td>Cat# A9539</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>Axon MedChem</td>
<td>Cat# 1386;CAS: 252917-06-9</td>
</tr>
<tr>
<td>Collagenase/dispase</td>
<td>Roche</td>
<td>Cat# 11097113001</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>Gibco</td>
<td>Cat# 12634010</td>
</tr>
<tr>
<td>DMSO</td>
<td>Merck</td>
<td>Cat# D2650</td>
</tr>
<tr>
<td>Dorsomorphin</td>
<td>Merck</td>
<td>Cat# P5499;CAS: 866405-64-3</td>
</tr>
<tr>
<td>Ethanol (EtOH)</td>
<td>Thermo</td>
<td>Cat# 32205</td>
</tr>
<tr>
<td>EGF human</td>
<td>R&amp;D</td>
<td>Cat# 236-EG-200</td>
</tr>
<tr>
<td>Fatty acid free BSA</td>
<td>Proliant</td>
<td>Cat# 68700;CAS: 9048-46-8</td>
</tr>
<tr>
<td>FGF2</td>
<td>R&amp;D</td>
<td>Cat# 233-FB</td>
</tr>
<tr>
<td>FGF10</td>
<td>PeproTech</td>
<td>Cat# 100-26</td>
</tr>
<tr>
<td>FGF10</td>
<td>R&amp;D</td>
<td>Cat# 345-FG</td>
</tr>
<tr>
<td>Glucose</td>
<td>Merck</td>
<td>Cat# G7528CAS: 50-99-7</td>
</tr>
<tr>
<td>Hydrochloric acid (HCL) (37%)</td>
<td>Merck</td>
<td>Cat# 30721-M</td>
</tr>
<tr>
<td>Insulin-Transferrin-Selenium-Ethanolamine (ITS-X)</td>
<td>Thermo</td>
<td>Cat# 51500056</td>
</tr>
<tr>
<td>KGF</td>
<td>PeproTech</td>
<td>Cat# 100-19;SDS: 25-100-19</td>
</tr>
<tr>
<td>L-Ascorbic acid</td>
<td>Merck</td>
<td>Cat# A4544CAS: 50-81-7</td>
</tr>
<tr>
<td>L-glutamine (Glutamax)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 35050038</td>
</tr>
<tr>
<td>LDN-193189</td>
<td>Merck</td>
<td>Cat# SML0559;CAS: 1062368-24-4</td>
</tr>
<tr>
<td>Matrigel Basement Membrane Matrix Growth Factor Reduced (GFR)</td>
<td>Corning</td>
<td>Cat# 354230</td>
</tr>
<tr>
<td>Matrigel hESC-qualified Matrix</td>
<td>Corning</td>
<td>Cat# 354277</td>
</tr>
<tr>
<td>MCDB131 medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10372019</td>
</tr>
<tr>
<td>MSC2530818</td>
<td>Selleckchem</td>
<td>Cat# S8387;CAS: 1883423-59-3</td>
</tr>
<tr>
<td>mTeSR1</td>
<td>STEMCELL Technologies</td>
<td>Cat# 85850</td>
</tr>
<tr>
<td>mTesR Plus</td>
<td>STEMCELL Technologies</td>
<td>Cat# 100-0276</td>
</tr>
<tr>
<td>Nicotinamide (NA)</td>
<td>Merck</td>
<td>Cat# N0636;CAS: 98-92-0</td>
</tr>
<tr>
<td>NutriFreez D10</td>
<td>Sartorius</td>
<td>Cat# 05-713</td>
</tr>
<tr>
<td>Paraformaldehyde (PFA)</td>
<td>Merck</td>
<td>Cat# P6148</td>
</tr>
<tr>
<td>Penicillin/Streptomycin (P/S)</td>
<td>Merck</td>
<td>Cat# P4333</td>
</tr>
<tr>
<td>Retinoic acid (RA)</td>
<td>Merck</td>
<td>Cat# R2625;CAS: 302-79-4</td>
</tr>
<tr>
<td>ROCK inhibitor (Y-27632)</td>
<td>Abcam</td>
<td>ab120129;CAS: 129830-38-2</td>
</tr>
<tr>
<td>SANT1</td>
<td>Merck</td>
<td>Cat# S4572;CAS: 304909-07-7</td>
</tr>
<tr>
<td>Sodium bicarbonate (NaHCO<sub>3</sub>)</td>
<td>Merck</td>
<td>Cat# S5761CAS: 144-55-8</td>
</tr>
<tr>
<td>Sodium dihydrogen phosphate mnohydrate (NaH<sub>2</sub>PO<sub>4</sub> <b>×</b> H<sub>2</sub>O)</td>
<td>Fluka</td>
<td>71507</td>
</tr>
<tr>
<td>Disodium hydrogen phosphate heptahydrate (Na<sub>2</sub>HPO<sub>4</sub> <b>×</b> 7H<sub>2</sub>O)</td>
<td>Sigma</td>
<td>S9390</td>
</tr>
<tr>
<td>Sodium hydroxide (NaOH)</td>
<td>AppliChem</td>
<td>AP131687.1211</td>
</tr>
<tr>
<td>Succrose D (+)</td>
<td>Fluka</td>
<td>84100</td>
</tr>
<tr>
<td>Tissue-Tek O.C.T. compound</td>
<td>Sakura</td>
<td>Cat# 4583</td>
</tr>
<tr>
<td>TrypLE™ Express Enzyme, no phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12604013</td>
</tr>
<tr>
<td>TrypLE™ Select Enzyme, no phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12563011</td>
</tr>
<tr>
<td>Wnt3a mouse</td>
<td>PeproTech</td>
<td>Cat# 315-20;SDS: 25-315-20</td>
</tr>
<tr>
<td>Zinksulfate (ZnSO<sub>4</sub>)</td>
<td>Merck</td>
<td>Cat# Z0251;CAS: 7446-20-0</td>
</tr>
<tr>
<td>(-)-Indolactam V</td>
<td>STEMCELL Technologies</td>
<td>Cat# 72312;CAS: 90365-57-4</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Tissue Genomic DNA Purification Mini Prep Kit</td>
<td>Genaxxon</td>
<td>Cat# S5378.0050</td>
</tr>
<tr>
<td>GeneJET RNA Purification Kit</td>
<td>Thermo</td>
<td>Cat# K0732</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HUES8 hESC line (male, NIH approval number NIHhESC-10-0021)</td>
<td>HSCI iPS Core, Harvard University, Cambridge, MA, USA</td>
<td>hES Cell Line: HUES-8; RRID:CVCL_B207</td>
</tr>
<tr>
<td>iPSCs (male, Institutional review board of Ulm University (registration number 68/11-UBB/bal)</td>
<td><a href="#bib1">Breunig et al. (2021)</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Counting chamber</td>
<td>VWR</td>
<td>Cat# HIRS8100104</td>
</tr>
<tr>
<td>Freezing container</td>
<td>VWR</td>
<td>Cat# 479-3200</td>
</tr>
<tr>
<td>LSR II flow cytometer</td>
<td>BD</td>
<td>n/a</td>
</tr>
<tr>
<td>SuperFrost plus slides</td>
<td>VWR</td>
<td>Cat# 631-0108</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p><a href="#fig1">Figure 1</a> provides an illustrative summary of the medium composition during all stages of hPSC differentiation into PDLOs (related to step I-2 and II-4). Please find additional details in tables below.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1219-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overview of medium composition</div>
<p>Schematic overview of medium composition during all stages of hPSC differentiation into PDLOs. Colored circles indicate the day of medium change (Related to step I-2 and II-4).</p>
</figcaption>
</figure>
<table id="undtbl1">
<caption>Preparation of basal differentiation media</caption>
<thead>
<tr>
<th>BE1a medium</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>MCDB131</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td>Fatty-acid-free BSA</td>
<td><b>0.1%</b></td>
<td>0.5 g</td>
</tr>
<tr>
<td>Sodium bicarbonate</td>
<td>1.174 g/L</td>
<td>587 mg</td>
</tr>
<tr>
<td>Glucose</td>
<td>0.8 g/L</td>
<td>400 mg</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>505 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">After preparation of the media, mix on a magnetic stirrer until everything is completely dissolved (at least 20 min). Filter the solution through a 0.22 μm filter and collect it in a sterile glass bottle. We recommend preparing 50 mL aliquots and store in the dark at 4°C for up to 4 weeks.</p>
<table id="undtbl2">
<thead>
<tr>
<th>BE1b medium</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>MCDB131</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td>Fatty-acid-free BSA</td>
<td><b>0.5%</b></td>
<td>2.5 g</td>
</tr>
<tr>
<td>Sodium bicarbonate</td>
<td>1.174 g/L</td>
<td>587 mg</td>
</tr>
<tr>
<td>Glucose</td>
<td>0.8 g/L</td>
<td>400 mg</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>505 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">After preparation of the media, mix on a magnetic stirrer until everything is completely dissolved (at least 20 min). Filter the solution through a 0.22 μm filter and collect it in a sterile glass bottle. We recommend preparing 50 mL aliquots and store in the dark at 4°C for up to 4 weeks.</p>
<table id="undtbl3">
<thead>
<tr>
<th>BE3 medium</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>MCDB131</td>
<td>n/a</td>
<td>1000 mL</td>
</tr>
<tr>
<td>Fatty-acid-free BSA</td>
<td><b>2%</b></td>
<td>20 g</td>
</tr>
<tr>
<td>Sodium bicarbonate</td>
<td>1.754 g/L</td>
<td>1.754 g</td>
</tr>
<tr>
<td>Glucose</td>
<td>0.44 g/L</td>
<td>440 mg</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>2 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>L-Ascorbic acid</td>
<td>44 mg/L</td>
<td>44 mg</td>
</tr>
<tr>
<td>ITS-X</td>
<td>0.5<b>×</b>
</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1015 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix on a magnetic stirrer until everything is completely dissolved (at least 40 min) by protecting the solution from light (for example with an aluminum foil). Filter the solution through a 0.22 μm filter and collect it in a sterile glass bottle. We recommend preparing 50 mL aliquots and store in the dark at 4°C for up to 4 weeks.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Only use reagent grade, fatty-acid-free BSA. Batch to batch variation of BSA should be controlled by determining optimal BSA concentrations for each batch. Adjust the concentration of BSA in the basal media (BE1a/b: 0.1–0.5%, BE3: 1–2%) for each batch accordingly. We additionally recommend comparing different batches and placing a bulk order for the best batch.</div>
<table id="undtbl4">
<caption>Preparation of stage-specific differentiation media</caption>
<thead>
<tr>
<th>Day</th>
<th>Additives to the media</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="5">BE1a</td>
</tr>
<tr>
<td rowspan="2">0–1 (d0)</td>
<td>Act A</td>
<td>100 μg/mL</td>
<td>100 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>CHIR-99021</td>
<td>10 mM</td>
<td>2 μM</td>
<td>1:5,000</td>
</tr>
<tr>
<td rowspan="2">1–3 (d1,2)</td>
<td>Act A</td>
<td>100 μg/mL</td>
<td>100 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>FGF2</td>
<td>50 μg/mL</td>
<td>5 ng/mL</td>
<td>1:10,000</td>
</tr>
<tr>
<td colspan="5">BE1b</td>
</tr>
<tr>
<td rowspan="3">3–6 (d3,4,5)</td>
<td>FGF10</td>
<td>50 μg/mL</td>
<td>50 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>DorsoM</td>
<td>5 mM</td>
<td>0.75 μM</td>
<td>1:6,666</td>
</tr>
<tr>
<td>Wnt3a</td>
<td>25 μg/mL</td>
<td>3 ng/mL</td>
<td>1:8,333</td>
</tr>
<tr>
<td colspan="5">BE3</td>
</tr>
<tr>
<td rowspan="5">d6–9 (d6,7,8)</td>
<td>FGF10</td>
<td>50 μg/mL</td>
<td>50 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>LDN-193189</td>
<td>1 mM</td>
<td>200 nM</td>
<td>1:5,000</td>
</tr>
<tr>
<td>SANT-1</td>
<td>1 mM</td>
<td>0.25 μM</td>
<td>1:4,000</td>
</tr>
<tr>
<td>RA</td>
<td>10 mM</td>
<td>2 μM</td>
<td>1:5,000</td>
</tr>
<tr>
<td>Glucose</td>
<td>1 M</td>
<td>16 mM∗</td>
<td>1:62.5</td>
</tr>
<tr>
<td rowspan="5">d9–13 (d9,10,11,12)</td>
<td>LDN-193189</td>
<td>1 mM</td>
<td>200 nM</td>
<td>1:5,000</td>
</tr>
<tr>
<td>EGF</td>
<td>50 μg/mL</td>
<td>100 ng/mL</td>
<td>1:500</td>
</tr>
<tr>
<td>Nicotinamide</td>
<td>1 M</td>
<td>10 mM</td>
<td>1:100</td>
</tr>
<tr>
<td>Indolactam V</td>
<td>660 μM</td>
<td>330 nM</td>
<td>1:2,000</td>
</tr>
<tr>
<td>Glucose</td>
<td>1 M</td>
<td>16 mM∗</td>
<td>1:62.5</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare differentiation media at the day of medium change; day of medium change is given in brackets; d: day.</p>
<table id="undtbl5">
<thead>
<tr>
<th>Day</th>
<th>Additives to the media</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Dilution</th>
</tr>
<tr>
<th colspan="5">BE3</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="8">d13–20 (d13,17)</td>
<td>Nicotinamide</td>
<td>1 M</td>
<td>10 mM</td>
<td>1:100</td>
</tr>
<tr>
<td>ZnSO<sub>4</sub>
</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1:1,000</td>
</tr>
<tr>
<td>Rock inhibitor Y-27632</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1:1,000</td>
</tr>
<tr>
<td>EGF</td>
<td>50 μg/mL</td>
<td>50 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>FGF10</td>
<td>50 μg/mL</td>
<td>50 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>KGF</td>
<td>50 μg/mL</td>
<td>50 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>MSC2530818</td>
<td>5 mM</td>
<td>50 nM</td>
<td>1:100,000</td>
</tr>
<tr>
<td>GFR Matrigel∗</td>
<td>100%</td>
<td>5%</td>
<td>1:20</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare differentiation media at the day of medium change; day of medium change is given in brackets; d: day. ∗ Add GFR with precooled tips directly before use.</p>
<table id="undtbl6">
<thead>
<tr>
<th>Day</th>
<th>Additives to the media</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Dilution</th>
</tr>
<tr>
<th colspan="5">BE3</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="5">d20–&gt; (d20,24,27,…)</td>
<td>Nicotinamide</td>
<td>1 M</td>
<td>10 mM</td>
<td>1:100</td>
</tr>
<tr>
<td>ZnSO<sub>4</sub>
</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1:1,000</td>
</tr>
<tr>
<td>EGF</td>
<td>50 μg/mL</td>
<td>50 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>FGF10</td>
<td>50 μg/mL</td>
<td>50 ng/mL</td>
<td>1:1,000</td>
</tr>
<tr>
<td>GFR Matrigel∗</td>
<td>100%</td>
<td>5%</td>
<td>1:20</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare differentiation media at the day of medium change; day of medium change is given in brackets; d: day. ∗ Add GFR with precooled tips directly before use. Add Rock inhibitor Y-27632 also for splitting and thawing of PDLOs.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Step I: Differentiation of hPSCs into PPs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 14–15 days</div>
<div class="timing">
<span class="timing-title">Timing:</span> 30–45 min for <a href="#o0730">step 1b</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 13 days for <a href="#olist0210">step 2</a>
</div>
<p>Pancreatic progenitors (PPs) are multipotent progenitors capable of differentiating into acinar, ductal, and endocrine cells during pancreatic development and several distinct protocols have been established for the <i>in vitro</i> generation of such PPs. PSCs have been successfully differentiated into PDLOs (<a href="#sec3.2">step II</a>) in combination with the presented PP protocol (<a href="#sec3.1">step I</a>), while alternative PP protocols have not been investigated. The herein described protocol was established and optimized for the hESC line HUES8 but also other PSC lines including induced pluripotent control stem cells were successfully applied.</p>
<ol>
<li>Cell seeding 1–2 days prior to differentiation
<ol type="a">
<li>Precoat the desired number of wells.
<ol type="i">
<li>Dilute GFR-Matrigel 1:18 in DMEM/F12. Evenly distribute 220 μL of the Matrigel solution per well of a 24-well plate and incubate plates for 1–2 h at 37°C.
<div class="note">
<span class="note-title">Note:</span> Take care that the Matrigel does not dry out at some regions of the well during the incubation.</div>
</li>
<li>Remove the Matrigel solution after incubation and add 300 μL mTeSR1 supplemented with 10 μM Rock inhibitor Y-27632.
<div class="note">
<span class="note-title">Note:</span> We recommend precoating of wells directly prior to harvesting and seeding.</div>
</li>
</ol>
</li>
<li>Harvest and seed hPSCs (volumes for 12-well plate).
<ol type="i">
<li>Remove medium from hPSC culture.</li>
<li>Add 500 μL PBS, gently swirl the plate, remove PBS.</li>
<li>Add 300 μL TrypLE Express or TrypLE Select and incubate at 37°C until cells are detached.
<div class="note">
<span class="note-title">Note:</span> Detachment of cells takes approximately 3–5 min but might vary between cell lines sometimes exceeding 5 min.</div>
</li>
<li>Dilute the dissociation reagent by adding 700 μL DMEM/F12. Resuspend cells by carefully pipetting up and down without mechanical dissociation and transfer the cell suspension into a tube or falcon.</li>
<li>Centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min and remove supernatant. Resuspend cells in 1 mL mTeSR1 supplemented with 10 μM Y-27632.</li>
<li>Count viable cells using a counting chamber and adjust the cell suspension to a final concentration of 1–1.5 <b>×</b> 10<sup>6</sup> cells/mL.</li>
<li>Dropwise add 200 μL of the cell suspension to each precoated 24-well (200,000 to 300,000 cells per well). In order to prevent cell aggregation in the middle of the wells, add the cell suspension in an outer circle to the wells and perform 2–3 times 8-shaped movements followed by one sudden ↕↔ movement of the plate.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We describe our strategy to achieve homogenous cell seeding. Equivalent strategies can also be used.</div>
<ol start="2">
<li>Stepwise differentiation of hPSCs into PPs (<a href="#sec6.1">Troubleshooting 1+2</a>).
<div class="note">
<span class="note-title">Note:</span> Initiate the differentiation 24–48 h after cell seeding at a confluency of 75%–95% (see <a href="#fig2">Figure 2</a>). Ideal cell confluency at the start of differentiation might be cell line dependent. In our hands starting the differentiation at a high confluency entailed better differentiation efficiencies than at a low confluency.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1219-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Seeding densities</div>
<p>Bright field overview images of hPSCs 1–2 days after cell seeding. Recommended confluency for the initiation of a differentiation is indicated.</p>
</figcaption>
</figure>
<ol type="a">
<li>D0-d1: Differentiation of hPSCs into mesendoderm.
<ol type="i">
<li>Remove mTeSR medium from hPSCs.</li>
<li>Add 500 μL PBS, gently swirl the plate, remove PBS.</li>
<li>Add 500 μL BE1a + 100 ng/mL Activin A + 2 μM CHIR99021 (medium change at d0).</li>
</ol>
</li>
<li>D1-d3: Differentiation of mesendoderm into definitive endoderm (DE).
<ol type="i">
<li>Remove medium and add 500 μL BE1a + 100 ng/mL Activin A + 5 ng/mL FGF2 (medium change at d1 and d2).</li>
<li>Perform quality check at d3. Flow cytometry analysis of surface marker CXCR4 and KIT as described in <a href="#bib1">Breunig et al. (2021)</a>. Alternatively, perform intracellular marker staining of SOX17 and GATA6.
<div class="critical">
<span class="critical-title">Critical:</span> More than 95% of DE cells should be double positive for the respective marker combination (<a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1219-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Quality check via flow cytometry</div>
<p>Differentiation efficiencies at definitive endoderm (DE, d3), pancreatic endoderm (PE, d9), and pancreatic progenitor (PP, d13) stage are measured by flow cytometry. Our quality standards are 95% CXCR4<sup>+</sup>/KIT<sup>+</sup> cells at DE stage, 90% PDX1<sup>+</sup> cells at PE stage, and 60% PDX1<sup>+</sup>/NKX6-1<sup>+</sup> cells at PP stage.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>D3-d6: Differentiation of DE into gut tube endoderm (GTE).
<ol type="i">
<li>Remove medium and add 500 μL BE1b + 50 ng/mL FGF10 + 0.75 μM Dorsomorphin + 3 ng/mL Wnt3a (medium change at d3, d4, and d5).</li>
</ol>
</li>
<li>D6-d9: Differentiation of GTE into pancreatic endoderm (PE).
<ol type="i">
<li>Remove medium and add 500 μL BE3 + 50 ng/mL FGF10 + 200 nM LDN-193189 + 0.25 μM SANT-1 + 2 μM retinoic acid + 16 mM glucose (medium change at d6, d7, and d8).</li>
<li>Perform quality check at d9. Intracellular flow cytometry analysis of PDX1 as described in <a href="#bib1">Breunig et al. (2021)</a>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> More than 90% of PE cells should be positive for PDX1 (<a href="#fig3">Figure 3</a>).</div>
</li>
<li>D9-d13: Differentiation of PE into pancreatic progenitors (PPs).
<ol type="i">
<li>Remove medium and add 500 μL BE3 + 100 ng/mL EGF + 200 nM LDN-193189 + 10 mM Nicotinamide + 330 nM Indolactam V + 16 mM glucose (medium change every day from d9 to d12).</li>
<li>Perform quality check at d13. Intracellular flow cytometry analysis of PDX1 and NKX6-1 as described in <a href="#bib1">Breunig et al. (2021)</a>. Duplicates should be analyzed for each cell line and measurements should precede harvesting and cell seeding at d13.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> More than 60% of PP cells should be double positive for PDX1 and NKX6-1 (<a href="#fig3">Figure 3</a>).</div>
<div class="note">
<span class="note-title">Note:</span> We observed slight differences in PP efficiency between different well formats and use 24-well formats as standard for differentiation.</div>
<div class="note">
<span class="note-title">Note:</span> Cell death within the first 2 days after initiation of differentiation can be observed, as expected. Such cell death might vary between different cell lines but usually does not affect the differentiation efficiency. In case of excessive and prolonged cell death, however, leading to a disruption of the cell layer and, therefore, cell-cell interactions, differentiation can be hampered. For the successful differentiation of such cell lines, we recommend supplementing the differentiation media from d0 to d6 with 5–10 μM Rock inhibitor Y-27632.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The seeding density is important and ideal cell numbers depend on the selected cell line and confluency of hPSCs at the time point of seeding. Also, homogenous cell seeding is important. Quality criteria at d3, d9, and d13 have to be met to seed PPs for the generation of PDLOs. If PP efficiency does not meet the quality check, glycoprotein 2 (GP2)-sorted cells can be used for PDLO differentiation. GP2 is described as cell surface marker that is specific for a multipotent pancreatic progenitor population (<a href="#bib2">Cogger et al., 2017</a>; <a href="#bib5">Ramond et al., 2017</a>). A step-by-step protocol for GP2 magnetic-activated cell sorting (MACS) can be found in <a href="#bib2">Cogger et al. (2017)</a>.</div>
<h3 id="sec3.2">Step II: Differentiation of PPs into PDLOs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 14–17 days</div>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min–1 h for <a href="#o0865">step 3b</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min–1 h for <a href="#o0930">step 3d</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 14–17 days for <a href="#olist0275">step 4</a>
</div>
<p>While in step I generated PPs are unspecified progenitor cells, step II allows the specification of PPs into pancreatic duct-like organoids resembling functional human pancreatic ducts. Sandwich culture and dome culture can be used equivalently. Please see <a href="#fig4">Figure 4</a> for temporally resolved bright field images from PSC to PDLO stage.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1219-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Time course of hPSC differentiation to PDLOs</div>
<p>While PPs are differentiated in a 2D monolayer culture format, PPs are reseeded as single cells on day 13 to allow 3D growth of PDLOs. Representative bright field images for each stage are shown.</p>
</figcaption>
</figure>
<ol start="3">
<li>Cell seeding of PPs – Sandwich culture (<a href="#sec6.5">Troubleshooting 3</a>)
<ol type="a">
<li>Precoating of wells and preparation.
<ol type="i">
<li>Precoat the desired number of wells. Evenly distribute 150 μL of undiluted GFR-Matrigel per well of a 12-well plate in continuous z-shape movements on a precooled culture plate using precooled tips and incubate plates for around 30 min to 1 h at 37°C.
<div class="note">
<span class="note-title">Note:</span> Take care that the Matrigel is evenly distributed across the whole well and avoid tilting the plate. For details on handling, please refer to <a href="#mmc1">Methods video S1</a>.</div>
</li>
</ol>
</li>
<li>Harvest PPs for PDLO differentiation.
<ol type="i">
<li>Remove medium from d13 differentiation culture.</li>
<li>Add 500 μL PBS, gently swirl the plate, remove PBS.</li>
<li>Add 200 μL TrypLE Express or TrypLE Select and incubate at 37°C until cells are detached.
<div class="note">
<span class="note-title">Note:</span> Takes approximately 10–15 min. Complete detachment of cells is important to obtain a high number of cells.</div>
</li>
<li>Dilute the dissociation reagent by adding 800 μL DMEM/F12. Resuspend cells by carefully pipetting up and down without mechanical dissociation and transfer the cell suspension into a tube or falcon.</li>
<li>Filter the cells through a 0.22 μm syringe filter.</li>
<li>Centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min and remove the supernatant. Resuspend cells in 1 mL BE3 medium.</li>
<li>Count viable cells using a counting chamber.</li>
<li>Centrifuge the desired number of cells at 200 <b>×</b> <i>g</i> for 4–5 min and remove the supernatant. Resuspend cells in PDLO medium (d13-d20) to a final concentration of 1.05 <b>×</b> 10<sup>5</sup> cells/mL.</li>
<li>Put cells on ice and add 5% GFR Matrigel to the cell suspension.</li>
<li>Pipet up and down and dropwise add 1 mL of the cell suspension to each of the precoated 12-wells (100,000 cells per well).
<div class="note">
<span class="note-title">Note:</span> To prevent cell aggregation in the middle of the wells, perform 2–3 times 8-shaped movements followed by one sudden ↕↔ movement of the plate.</div>
<div class="note">
<span class="note-title">Note:</span> We use 12-well plates as standard format for the sandwich culture. For 6-well plates, please double the herein described volumes and cell numbers.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Homogenous coating and cell seeding is important. Our strategy to ensure homogenous coating of the sandwich culture is based on <a href="#bib4">Xiang and Muthuswamy (2006)</a> and is shown in <a href="#mmc1">Methods video S1</a>. While wells can be homogenously coated for 12-well and 6-well formats, we do not recommend coating of lower well formats, as equal distribution of Matrigel in smaller well formats is more difficult and non-homogenous coating leads to disrupted organoid formation.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Cell seeding in Matrigel-domes (step II-3) (<a href="#sec6.5">Troubleshooting 3</a>)</div>
</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1219-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Matrigel-coating for sandwich culture, related to step II-3a</div>
<p>Perform narrow z-formed movement to evenly distribute undiluted GFR-Matrigel. Use precooled tips and well plates and avoid any air bubbles.</p>
</figcaption>
</figure>
</li>
<li>Well preparation.
<ol type="i">
<li>Prewarm 24-well plates for dome culture at 37°C for 10 min.</li>
</ol>
</li>
<li>Harvest PPs for PDLO differentiation.
<ol type="i">
<li>Harvest PPs. See step II-3b (i-vii) (sandwich culture)</li>
<li>Centrifuge the desired number of cells at 200 <b>×</b> <i>g</i> for 4–5 min and remove the supernatant. Put cells on ice and resuspend cells with 50 μL GFR Matrigel per 30,000 cells.</li>
<li>Pipet up and down and add 50 μL drops of the cell suspension per well of a 24-well plate to the uncoated but prewarmed plate (30,000 cells per well).</li>
<li>Incubate domes for around 10 min at 37°C to allow Matrigel solidification, then add 500 μL PDLO medium (d13-d20) to each well.
<div class="note">
<span class="note-title">Note:</span> No additional Matrigel is added to the medium for the dome culture.</div>
<div class="note">
<span class="note-title">Note:</span> Sandwich and dome cultures can be used as equivalent alternatives. The choice for sandwich or dome depends on the intended application with the use of a sandwich format allowing for example improved in-well imaging and automated quantification of organoids.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Differentiation of PPs into PDLOs (<a href="#sec6.1">Troubleshooting 1+4</a>).
<ol type="a">
<li>D13-d20: Differentiation of PPs into pancreatic trunk-like organoids (PTLOs)/ductal precursors.
<ol type="i">
<li>PDLO medium (d13–20) is added after the transfer of PPs into a 3D culture format on d13. PDLO medium (d13–20) consists of BE3 + 50 ng/mL EGF + 50 ng/mL FGF10 + 50 ng/mL KGF + 50 nM MSC2530818 + 10 mM Nicotinamide + 10 μM Rock inhibitor Y-27632 + 10 μM ZnSO<sub>4</sub>. For the sandwich culture, 5% GFR Matrigel was additionally added to the medium at each medium change (medium change at d17).</li>
<li>For medium change, carefully remove medium without mechanically disrupting the thick Matrigel layer and add 1 mL of PDLO medium(d13–20).</li>
</ol>
</li>
<li>D20-d27/d30: Differentiation of PTLOs into PDLOs.
<ol type="i">
<li>Carefully remove the medium without mechanically disrupting the thick Matrigel layer and add 1 mL of PDLO medium(d20-&gt;). PDLO (d20-&gt;) medium consists of BE3 + 50 ng/mL EGF + 50 ng/mL FGF10 + 10 mM Nicotinamide + 10 μM ZnSO<sub>4</sub>. For the sandwich culture, 5% GFR Matrigel was additionally added to the medium for each medium change (medium change at d20, d24, and d27).</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For the medium change, the media is first prepared without Matrigel and the Matrigel is added on ice to the medium directly before the medium change.</div>
<div class="note">
<span class="note-title">Note:</span> Depending on the density of the PDLO culture, PDLOs are harvested between d27 and d30.</div>
<div class="note">
<span class="note-title">Note:</span> For the initial characterization of PDLO differentiation, we recommend performing a comprehensive characterization including functional assays and detailed marker profiling. Please refer for further information to <a href="#bib1">Breunig et al. (2021)</a>. After successful establishment of the PDLO differentiation, examination of bright field images can be used as primary quality control as outlined in <a href="#fig5">Figure 5</a>.</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1219-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Illustrative PDLO cultures</div>
<p>While PDLOs in the two bright field images on the left constitute homogenous cultures of ring-like organoids, PDLOs on the two right images represent poor cultures. While many PDLOs on the third image have not formed ring-like organoids or have collapsed, the fourth image highlights 2D structurers arising due to inhomogeneous coating.</p>
</figcaption>
</figure>
<h3 id="sec3.3">Step III: PDLO culture and processing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–7 h for <a href="#olist0300">step 5a</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 5–7 h for <a href="#o1050">step 5b</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h and 30 min for <a href="#o1075">step 5c</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 5–8 h for <a href="#olist0325">step 6a</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 3–5 h for <a href="#o1140">step 6b</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 5–7 h for <a href="#o1155">step 6c</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 3–5 h for <a href="#o1185">step 6d</a>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h for <a href="#o1205">step 6e</a>
</div>
<p>Step III describes the harvesting of PDLOs generated in step II for subsequent sampling to further cultivation, DNA, RNA, or protein analysis, or staining. PDLOs can be cultured for several passages, albeit PDLOs have not been systematically characterized beyond d59 (<a href="#bib1">Breunig et al., 2021</a>). PDLOs are viable after freezing and thawing but potential changes in PDLO composition have also not been comprehensively investigated.</p>
<ol start="5">
<li>Further culturing of PDLOs
<ol type="a">
<li>Splitting PDLOs.
<ol type="i">
<li>Remove medium, wash twice with 1 mL PBS, remove PBS.</li>
<li>Add 1 mL of 1 mg/mL collagenase/dispase solution and incubate at 37°C until the Matrigel is mostly dissolved and PDLOs are detached (2–4 h).
<div class="note">
<span class="note-title">Note:</span> The duration of the incubation should be adjusted for the different applications. While the Matrigel should be completely dissolved for the purification of protein (4 h incubation recommended), some Matrigel can be carried over for splitting of cells (2 h incubation is sufficient).</div>
</li>
<li>Add 2 mL neutralization solution and carefully rinse the well, transfer cell suspension to a 15 mL falcon and centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min.</li>
<li>Remove supernatant, add 5 mL PBS, and centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min.
<div class="critical">
<span class="critical-title">Critical:</span> If the Matrigel has not been completely resolved, Matrigel remnants will be carried over. In this case, perform an additional PBS washing step and be careful not to disrupt the pellet. The subsequent accutase step will further dissolve the Matrigel in addition to the singularization of cells.</div>
</li>
<li>Remove supernatant, add 0.5 mL accutase, and incubate at 37°C in the water bath for 30 min.
<div class="note">
<span class="note-title">Note:</span> For improved singularization of the cells, mix the solution by pipetting up and down (10 <b>×</b> before incubation, 10 <b>×</b> after 15 min of incubation, and 10 <b>×</b> at the end of incubation). When cell clusters are adverse as in flow cytometry experiments or for single cell seeding, the accutase step should be extended to 1 h.</div>
</li>
<li>Add 1 mL neutralization solution and centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min.</li>
<li>Remove supernatant, resuspend cells in 1 mL BE3.</li>
<li>Count viable cells using a counting chamber.</li>
<li>Centrifuge the desired number of cells at 200 <b>×</b> <i>g</i> for 4–5 min and remove the supernatant; resuspend cells into PDLO medium (d20-&gt;) supplemented with 10 μM Rock inhibitor Y-27632 to a final concentration of 1.05 <b>×</b> 10<sup>5</sup> cells/mL.</li>
<li>Put cells on ice and add 5% GFR Matrigel to the cell suspension.</li>
<li>Pipet up and down and dropwise add 1 mL of the cell suspension to one precoated well of the 12-well plate (100,000 cells per well).
<div class="note">
<span class="note-title">Note:</span> In order to prevent cell aggregation in the middle of the wells, perform 2–3 times 8-shaped movements followed by one sudden ↕↔ movement of the plate.</div>
<div class="note">
<span class="note-title">Note:</span> PDLOs were successfully cultured until around day 100 (5–6 passages). While no differences in the expression of selected markers (SOX9, HNF1B, KRT19, ECAD) were observed in immunoflourescence staining, we have not assessed the PDLOs beyond d59 in more detail (<a href="#bib1">Breunig et al., 2021</a>).</div>
</li>
</ol>
</li>
<li>Cell preparation for xenotransplantation.
<ol type="i">
<li>Harvest and dissociate PDLOs and count single cells. See step III-5a (i-viii).</li>
<li>Centrifuge 1 <b>×</b> 10<sup>6</sup> cells per mouse at 200 <b>×</b> <i>g</i> for 4–5 min and remove the supernatant. Resuspend cells in 25 μL PDLO medium (d20-&gt;) supplemented with 20 μM Rock inhibitor Y-27632.</li>
<li>Put cells on ice and add 25 μL undiluted GFR Matrigel to the cell suspension.</li>
<li>Proceed with the transplantation of the 50 μL cell/Matrigel suspension as detailed in <a href="#bib1">Breunig et al. (2021)</a>.</li>
</ol>
</li>
<li>Freezing and thawing of PDLOs.
<ol type="i">
<li>Harvest and dissociate PDLOs to single cells. See step III-5a (i-vi).</li>
<li>Remove supernatant. Resuspend cells in 1 mL NutriFreez D10 cryopreservation media supplemented with 10 μM Rock inhibitor Y-27632 and transfer cells into a cryopreservation tube.</li>
<li>Transfer tubes into a freezing container and put into a −80°C freezer for around 24 h before transferring into a liquid nitrogen tank.</li>
<li>For thawing, quickly defrost vials at 37°C, add 5 mL DMEM/F12, and centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min.</li>
<li>Remove supernatant. Resuspend in BE3 medium and count viable cells using a counting chamber.</li>
<li>Centrifuge the desired number of cells at 200 <b>×</b> <i>g</i> for 4–5 min and remove the supernatant. Resuspend cells in PDLO medium (d20-&gt;) supplemented with 10 μM Rock inhibitor Y-27632 to a final concentration of 2–4 <b>×</b> 10<sup>5</sup> cells/mL.
<div class="note">
<span class="note-title">Note:</span> PDLO cells should be seeded at a higher density after thawing to ensure proper growth after freezing and thawing.</div>
</li>
<li>Put cells on ice and add 5% GFR Matrigel to the cell suspension.</li>
<li>Pipet up and down and dropwise add 1 mL of the cell suspension to one well of a precoated 12-well plate (200,000–400,000 cells per well).
<div class="note">
<span class="note-title">Note:</span> To prevent cell aggregation in the middle of the wells, perform 2–3 times 8-shaped movements followed by one sudden ↕↔ movement of the plate.</div>
<div class="note">
<span class="note-title">Note:</span> Thawing of frozen PDLOs (iv – viii) is described in detail for the sandwich culture. Seeding as dome cultures can be performed as equivalent alternative.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend freezing multiple vials in one round and performing a test thaw after storage of cells in the liquid nitrogen tank for several weeks.</div>
<div class="note">
<span class="note-title">Note:</span> Monolayered ring-like PDLOs are formed within 8–12 days after thawing but the exact marker profile and cell composition has not been systematically analyzed after freezing and thawing. Repeated freezing and thawing are not recommended.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Harvesting and sampling of PDLOs
<ol type="a">
<li>Sampling for flow cytometry.
<ol type="i">
<li>Harvest and dissociate PDLOs to single cells. See step III-5a (i-vi).
<div class="note">
<span class="note-title">Note:</span> Singularization of PDLOs with accutase should be extended to 1 h (step v).</div>
</li>
<li>Remove supernatant, proceed with surface staining or paraformaldehyde fixation in case of intracellular marker staining as detailed in <a href="#bib1">Breunig et al. (2021)</a>.</li>
</ol>
</li>
<li>RNA isolation.
<ol type="i">
<li>Recover PDLOs from the Matrigel. See step III-5a (i-iv).</li>
<li>Remove supernatant. Perform RNA lysis according to manufacturer’s instructions. Either freeze samples after lysis or directly continue with RNA isolation. RNA isolation using Gene Jet RNA Purification Kit resulted in very good RNA yields in our hands.</li>
</ol>
</li>
<li>Protein isolation.
<ol type="i">
<li>Recover PDLOs from the Matrigel. See step III-5a (i-iv).
<div class="note">
<span class="note-title">Note:</span> The Matrigel should be completely dissolved in step ii (incubation for at least 4 h).</div>
</li>
<li>Remove supernatant, wash with 5 mL PBS, and centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min.</li>
<li>Remove supernatant, add 1 mL PBS, transfer cell suspension to a 1.5-mL tube, and centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min.</li>
<li>Completely remove supernatant.</li>
<li>Either freeze the dry pellet at −80°C for protein isolation at a later time point or directly continue with protein lysis according to the downstream application. The lysis with RIPA buffer for Western Blotting and with urea buffer for mass spectrometry are detailed in <a href="#bib1">Breunig et al. (2021)</a>.
<div class="note">
<span class="note-title">Note:</span> Use PBS without Ca<sup>2+</sup> and Mg<sup>2+</sup> for the isolation steps, as the respective cations might interfere with protein lysis.</div>
</li>
</ol>
</li>
<li>DNA isolation.
<ol type="i">
<li>Recover PDLOs from the Matrigel. See step III-5a (i-iv).</li>
<li>Remove supernatant, wash with 5 mL PBS, and centrifuge at 200 <b>×</b> <i>g</i> for 4–5 min.</li>
<li>Perform DNA lysis according to manufacturer’s instructions. DNA isolation using <a href="https://www.genaxxon.com/shop-all-products/rna-dna-protein-reinigung/dna-reinigungskits/554/tissue-genomic-dna-purification-mini-spin-column-kit-50-columns?c=59">Animal Tissue Genomic DNA Purification Mini Prep Kit</a> resulted in very good DNA yields in our hands.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> No singularization of PDLOs is required for the isolation of RNA, protein, or DNA.</div>
</li>
<li>Paraffin-embedding.
<ol type="i">
<li>Remove medium, wash with 1 mL PBS, remove PBS.
<div class="note">
<span class="note-title">Note:</span> PBS should be 20°C–23°C, not ice cold to prevent depolymerization of the Matrigel.</div>
</li>
<li>Add 1 mL 4% PFA solution (100 mM sucrose) for 14–18 h at 8°C.
<div class="note">
<span class="note-title">Note:</span> PFA should be 20°C–23°C, not ice cold to prevent uncontrolled depolymerization of the Matrigel.</div>
<div class="note">
<span class="note-title">Note:</span> The Matrigel will be partly dissolved allowing to arrange organoids next to each other and simultaneously decreasing background staining.</div>
</li>
<li>Transfer organoids that have been detached by the PFA treatment to a 1.5-mL tube first and then transfer remaining organoids in PBS by detaching them with a tip. Rinse the well again with PBS to transfer as many organoids as possible.
<div class="note">
<span class="note-title">Note:</span> Coat pipet tips with 1% BSA/PBS to prevent sticking of the organoids on the wall of the tips and cut pipet tip to prevent mechanical shearing.</div>
</li>
<li>Centrifuge organoids at 3000 <b>×</b> <i>g</i> for 5 min, remove supernatant, and add 1 mL PBS.</li>
<li>Repeat the washing step in iv.</li>
<li>For pre-embedding, prepare 2% Agarose/PBS (small glass bottle). In parallel, heat water in a beaker. Let the agarose cool down to around 60°C in the hot water beaker (measure temperature).</li>
<li>Spin down organoids at 3000 <b>×</b> <i>g</i> for 3 min.
<div class="note">
<span class="note-title">Note:</span> If the organoid pellet is rather distributed along the tube, you can perform a second centrifugation step where you change the orientation of the tubes in the centrifuge.</div>
</li>
<li>Carefully take off half of the supernatant of your sample.</li>
<li>Transfer organoids with a 200 μL pipet with as little liquid as possible to cryomolds and carefully remove excess liquid with a 10 μL pipet.
<div class="note">
<span class="note-title">Note:</span> The usage of 1% BSA/PBS-coated tips is especially important for transferring the organoids to the cryomold.</div>
</li>
<li>Add around 100 μL agarose to organoids to form a layer of agarose covering all organoids.
<div class="note">
<span class="note-title">Note:</span> Do not pipet the agarose directly onto the organoids but add it dropwise at opposite edges of the mold to avoid floating of organoids towards one corner but arranging them in the center. Also take care that organoids are placed in the same plain. If not, you can arrange the position of organoids with a 200 μL pipette tip before the agarose starts to polymerize.</div>
</li>
<li>Wait some minutes until the agarose starts to polymerize, then fill up the mold with additional agarose.</li>
<li>When agarose has cooled down to 20°C–23°C (takes around 5 min), transfer the block into a tissue cassette with filter paper and start with pre-dehydration.</li>
<li>For an initial pre-dehydration, incubate the tissue cassettes with the agarose blocks in 30% EtOH for 30 min.</li>
<li>Then incubate in 50% EtOH for 30 min.</li>
<li>Incubate in 70% EtOH for 20 min, and store at 4°C until continuing with routine histological processing including dehydration (starting at 70% EtOH), paraffin-embedding, sectioning to 4 μm thick serial section, and immunohistochemical or immunofluorescence staining.</li>
</ol>
</li>
<li>Cryo-embedding:
<ol type="i">
<li>Remove medium, wash with 1 mL PBS, remove PBS.
<div class="note">
<span class="note-title">Note:</span> PBS should be 20°C–23°C, not ice cold to prevent depolymerization of the Matrigel.</div>
</li>
<li>Add 1 mL 4% PFA solution (100 mM Sucrose) for 30 min at 20°C–23°C.
<div class="note">
<span class="note-title">Note:</span> PFA should be 20°C–23°C, not ice cold to prevent depolymerization of the Matrigel.</div>
</li>
<li>Remove PFA solution and wash twice with PBS.
<div class="note">
<span class="note-title">Note:</span> PBS should be 20°C–23°C, not ice cold to prevent depolymerization of the Matrigel. When the Matrigel detaches during the washing steps, transfer organoids first and wash in 1.5 mL tubes.</div>
</li>
<li>Transfer organoids with PBS to the 1.5 mL tube by scrapping off the organoid/Matrigel layer with a 1000 μL pipette tip. Rinse the well again with PBS to transfer nearly all organoids.
<div class="note">
<span class="note-title">Note:</span> Coat pipet tips with 1% BSA/PBS to prevent sticking of the organoids on the wall of the tips and cut pipet tip to prevent mechanical shearing.</div>
</li>
<li>Centrifuge organoids at 200 <b>×</b> <i>g</i> for 5 min, remove supernatant, and add 1 mL 25% sucrose solution (in PBS) for 14–18 h at 4°C.</li>
<li>For cryo-embedding, transfer organoids from the bottom of the tube in as less of sucrose solution as possible to cryomolds and carefully remove excess liquid with a 10 μL pipet.
<div class="note">
<span class="note-title">Note:</span> The usage of 1% BSA/PBS-coated tips is especially important for transferring the organoids to the cryomold.</div>
</li>
<li>Add Tissue-Tek O.C.T. compound to form a layer covering all organoids.
<div class="note">
<span class="note-title">Note:</span> Try to avoid air bubbles. Aspirate remaining air bubbles using a 200 μL pipette.</div>
<div class="note">
<span class="note-title">Note:</span> Do not add the Tissue-Tek O.C.T. directly onto the organoids but add it dropwise at opposite edges of the mold to avoid floating of organoids towards one corner but arranging them in the center. Also take care that organoids are placed at the bottom of the cryomold in the same plain. If not, the position of organoids can be arranged with a needle.</div>
</li>
<li>Wait for around 5 min to allow the Tissue-Tek O.C.T. to enter the tissue and for vanishing of remaining air bubbles.</li>
<li>Snap-freeze in liquid nitrogen and store at −80°C.</li>
<li>Use a cryotome for generating 6–10 μm thick sections on SuperFrost plus slides according to standard histological procedures.</li>
</ol>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>While differentiation into PPs was analyzed by flow cytometry as illustrated in <a href="#fig3">Figure 3</a>, quality criteria for the generation of PDLOs were based on morphology. Only a homogenous ring-like PDLO culture should be used for downstream applications. Examples of respective PDLO cultures are shown in <a href="#fig5">Figure 5</a> and typical yields of RNA, protein, and DNA are given in <a href="#tbl1">Table 1</a>.</p>
<table id="tbl1">
<caption>Table 1. Typical RNA, protein, and DNA yields from the PDLO sandwich culture<sup><a href="#tblfn1">a</a></sup>
</caption>
<thead>
<tr>
<th>Well format</th>
<th>Sample</th>
<th>Typical yields</th>
</tr>
</thead>
<tbody>
<tr>
<td>12-well sandwich culture</td>
<td>RNA</td>
<td>2.5–7.5 μg</td>
</tr>
<tr>
<td>6-well sandwich culture</td>
<td>Protein (RIPA lysis)</td>
<td>100–200 μg</td>
</tr>
<tr>
<td>12-well sandwich culture</td>
<td>DNA</td>
<td>2.5–7.5 μg</td>
</tr>
</tbody>
</table>
<p id="tblfn1" class="table-footnote"><sup>a</sup>PDLOs were harvested at day 27–30 or 10–12 days after splitting.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While the import/generation, culture, and study of hESCs requires specific approval from National authorities, the use of iPSCs demands approval from institutional review boards. For reprogramming of iPSCs, informed consent of all patients has to be in place. Such limitations make PDLOs a model system which might not be applicable for all laboratories. The implementation also demands the acquisition of various cytokines. Albeit we have shown broad applicability of the protocol to hPSC lines other than the hESC line HUES8, the development of the protocol was driven in HUES8 cells (<a href="#bib1">Breunig et al., 2021</a>). Therefore, successful PDLO generation from other hPSC lines should be first thoroughly characterized before using the system for disease modeling. The use of the current PDLO system has been also only used in combination with the presented PP protocol. If other PP protocols can be equivalently used prior to PDLO generation needs to be tested. Variations in cytokine preparation or activity might affect the performance of the protocol and troubleshooting can be cumbersome to identify potential bugs. We, therefore, recommend implementing a stringent control system monitoring the dissolving date and the expiration date of all applied cytokines, which is especially important when the throughput of cytokines might be low.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Problems to establish the PP (step I-2) and PDLO (step II-4) differentiation protocols.</p>
<h3 id="sec6.2">Potential solution</h3>
<ul>
<li>Check pluripotency of the applied hPSCs.</li>
<li>Make sure that the applied hPSC culture is suitable. We use a Matrigel- and mTeSR-based single cell culture format. Conversion of PSCs to a different culture format might take several weeks and might affect the differentiation efficiency.</li>
<li>We recommend establishing the presented protocol with HUES8 cells first.</li>
<li>Try to establish the protocol stage by stage to first achieve efficient DE, PE, and PP formation before starting to generate PDLOs.</li>
<li>Test different batches of BSA.</li>
<li>Stick to the herein presented compounds and solutions. We do not recommend testing alternative compounds before the protocol has been successfully established in the laboratory.</li>
<li>
<a href="#bib3">Nostro et al. (2015)</a> has provided additional options for the cell line-dependent generation of PPs including the variation of the stage durations and the concentrations of key cytokines.</li>
</ul>
<h3 id="sec6.3">Problem 2</h3>
<p>Suddenly, low differentiation efficiencies at PP stage, albeit differentiation was successfully established (step I-2).</p>
<h3 id="sec6.4">Potential solution</h3>
<ul>
<li>Make sure that differentiation efficiencies at intermediate stages (DE, PE) fulfill the quality criteria.</li>
<li>Use control lines with known differentiation efficiencies.</li>
<li>Keep track of the date of cytokine preparation and overall efficiencies in the laboratory to identify putative bugs.</li>
<li>Take cytokines and solutions from all stages in consideration. Differences in cytokines in early phases might also affect PP efficiency albeit intermediate quality controls are met.</li>
</ul>
<h3 id="sec6.5">Problem 3</h3>
<p>Formation of 2D structures in a PDLO sandwich culture (step II-3) (see <a href="#fig5">Figure 5</a>).</p>
<h3 id="sec6.6">Potential solution</h3>
<ul>
<li>Carefully control homogenous Matrigel coating. Avoid any thin-coated regions which becomes especially problematic, when Matrigel is pushed to the edges of the well. Therefore, avoid forceful movements to edges of the well.</li>
</ul>
<h3 id="sec6.7">Problem 4</h3>
<p>No homogenous ring-like PDLO culture is formed (step II-4) (see <a href="#fig5">Figure 5</a>).</p>
<h3 id="sec6.8">Potential solution</h3>
<ul>
<li>Carefully control homogenous cell seeding and cell numbers. Too thin and too dense cell cultures will not lead to homogenous organoid cultures.</li>
<li>Make sure that differentiation efficiencies at intermediate stages (DE, PE, PP) fulfill the quality criteria.</li>
<li>Use control lines with known morphology.</li>
<li>Keep track of the date of cytokine preparation and overall efficiencies in the laboratory to identify putative bugs.</li>
<li>Take cytokines and solutions from all stages in consideration. Differences in cytokines in early phases might also affect PDLO differentiation albeit intermediate quality controls are met.</li>
<li>GP2 sorting can be applied at d13 to purify the PP culture to improve PDLO formation.</li>
<li>EPCAM sorting can be applied at d27-d30 to purify the PDLO culture and get rid of any non-epithelial contamination.</li>
<li>Homogenous coating is easier for larger well formats. We usually differentiate PDLOs in a 12-well format.</li>
<li>The use of a Matrigel-dome culture might overcome problems related to inhomogeneous coating.</li>
<li>A PDX1/NKX6-1 double negative PP population might give rise to non-epithelial 2D structures. Control differentiation efficiencies and consider GP2 or EPCAM sorting for increased purity.</li>
</ul>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Breunig, M., Merkle, J., Wagner, M., Melzer, M.K., Barth, T.F.E., Engleitner, T., Krumm, J., Wiedenmann, S., Cohrs, C.M., Perkhofer, L., et al. (2021). Modelling plasticity and dysplasia of pancreatic ductal organoids derived from human pluripotent stem cells. Cell Stem Cell <i>28</i>, 1105-1124.e19. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00619-5/sref1">View at publisher</a></p>
<p id="bib2">Cogger, K.F., Sinha, A., Sarangi, F., Mcgaugh, E.C., Saunders, D., Dorrell, C., Mejia-Guerrero, S., Aghazadeh, Y., Rourke, J.L., Screaton, R.A., et al. (2017). Glycoprotein 2 is a specific cell surface marker of human pancreatic progenitors. Nat. Commun. <i>8</i>, 331. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00619-5/sref2">View at publisher</a></p>
<p id="bib3">Nostro, M.C., Sarangi, F., Yang, C., Holland, A., Elefanty, A.G., Stanley, E.G., Greiner, D.L., and Keller, G. (2015). Efficient generation of NKX6-1+ pancreatic progenitors from multiple human pluripotent stem cell lines. Stem Cell Rep. <i>4</i>, 591-604. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00619-5/sref3">View at publisher</a></p>
<p id="bib5">Ramond, C., Glaser, N., Berthault, C., Ameri, J., Schlichting Kirkegaard, J., Semb, H., and Scharfmann, R. (2017). Reconstructing human pancreatic differentiation by mapping specific cell populations during development. Elife <i>21</i>, e27564. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00619-5/optUq3doPUHzK">View at publisher</a></p>
<p id="bib4">Xiang, B. and Muthuswamy, S.K. (2006). Using three-dimensional acinar structures for molecular and cell biological assays. Methods Enzymol. <i>406</i>, 692-701. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00619-5/sref4">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Alexander Kleger (<a href="mailto:alexander.kleger@uni-ulm.de">alexander.kleger@uni-ulm.de</a>)</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze data sets/code.</p>
<h3>Acknowledgments</h3>
<p>The authors thank Katrin Köhn, Ulrike Mayr-Beyrle, Rashmi Bijegatte, and Ralf Köhntop for excellent technical assistance, and Jana Krüger, Sarah Merz, Paul Lopatta, Martin Wagner, and Franz Oswald for helpful discussions. The authors acknowledge O.D. Madsen and the Developmental Studies Hybridoma Bank (DSHB) for providing the NKX6-1 antibody. Main funding is provided by the Deutsche Forschungsgemeinschaft (<a href="https://doi.org/10.13039/501100001659">DFG</a>) “Sachbeihilfe” KL 2544/7-1 and “Heisenberg-Programm” KL 2544/6-1, as well as the <a href="https://doi.org/10.13039/100008316">Baden-Württemberg-Foundation</a> ExPoChip. Alexander Kleger and Thomas Seufferlein are PIs in the HEIST RTG funded by the <a href="https://doi.org/10.13039/501100001659">DFG</a> GRK 2254/1. Additional funding comes from the <a href="https://doi.org/10.13039/501100005972">German Cancer Aid grant</a> to Alexander Kleger. (111879), the <a href="https://doi.org/10.13039/501100001659">DFG</a> (K.L. 2544/1-1 and 1–2 and 5-1), the INDIMED-Verbund PancChip, and the <a href="https://doi.org/10.13039/501100003042">Else Kröner-Fresenius-Stiftung</a> supporting Alexander Kleger with an Excellence grant and Meike Hohwieler with a First Application grant. Meike Hohwieler and Sandra Heller received supportive funds by the Bausteinprogramm of <a href="https://doi.org/10.13039/501100013812">Ulm University</a>.</p>
<h3>Author contributions</h3>
<p>A.K. and M.B. performed the literature research identifying potential candidates allowing the differentiation of PPs into the ductal lineage. The underlying PP protocol was mainly established and optimized in the laboratory by M.H. and S.H. M.B. developed the actual PDLO differentiation protocol by several rounds of cytokine screening. J.M., M.K.M., and M.H. contributed to the conduction of these comprehensive screens. J.M., M.K.M., and M.H. further optimized the downstream applications including the processing of PDLOs for cryo- and paraffin-embedding and the use of Matrigel domes. M.B., J.M., and M.H. wrote the manuscript. M.H., A.K., and T.S. initiated the study.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>