<article>
<section>
<h1 id="header">Nuclei isolation from surgically resected human hippocampus</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Fatma Ayhan,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Connor Douglas,<sup><a href="#aff1">1</a></sup> Bradley C. Lega,<sup><a href="#aff2">2</a></sup> and Genevieve Konopka<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Neuroscience, UT Southwestern Medical Center, Dallas, TX 75390, USA</p>
<p id="aff2"><sup>2</sup>Department of Neurosurgery, UT Southwestern Medical Center, Dallas, TX 75390, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:genevieve.konopka@utsouthwestern.edu">genevieve.konopka@utsouthwestern.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100844">10.1016/j.xpro.2021.100844</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Single-nucleus RNA sequencing (snRNA-seq), where nuclear transcriptomes are a proxy to cellular transcriptomes, has been used to profile human brain. snRNA-seq is sensitive to tissue processing, tissue quality, postmortem interval time, and cellular debris. This protocol outlines steps for the isolation of high-quality nuclei from surgically resected human brain tissue followed by a sucrose gradient yielding neuronal and non-neuronal nuclei enabling unbiased analysis of various cell types.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00550-5/sref1">Ayhan et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol describing nuclei isolation from human brain for single-nuclei RNA-seq.</li>
<li>A sucrose gradient adapted for a low amount of tissue helps cleaning debris.</li>
<li>Various quality control measures ensuring RNA and nuclei integrity.</li>
<li>Details regarding biological safety and reproducible record keeping are considered.</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1025-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>All experiments using human tissues must be performed with the informed consent of participants. For this study and our previous work (<a href="#bib1">Ayhan et al., 2021</a>), all hippocampal tissues were obtained from patients with temporal lobe epilepsy with the informed consent of patients and approval of the UTSW Institutional Review Board (IRB).</li>
<li>All unfixed human tissue must be assumed to be infectious. Use appropriate PPE including lab coats, double gloves, and safety googles.</li>
<li>As RNA polymerases and RNases show minimal activity at 4°C, it is critical to keep the dissected tissue at ice-cold temperatures after surgical resection. Prepare the solutions listed in <a href="#materials-and-equipment">materials and equipment</a> and prechill all solutions to 4°C on ice. Cool all centrifuges, rotors, and tubes to 4°C.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Nuclei EZ lysis buffer</td>
<td>Sigma</td>
<td>Cat# NUC-101</td>
</tr>
<tr>
<td>UltraPure™ BSA (50 mg/mL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #AM2618</td>
</tr>
<tr>
<td>SUPERase•In™ RNase Inhibitor (20 U/μL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #AM2694</td>
</tr>
<tr>
<td>Trypan Blue Solution, 0.4%</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15250061</td>
</tr>
<tr>
<td>ProLong Diamond Antifade Reagent with DAPI</td>
<td>Life Technologies</td>
<td>Cat#P36971</td>
</tr>
<tr>
<td>Neurobasal A Medium</td>
<td>Invitrogen</td>
<td>Cat#10888-022</td>
</tr>
<tr>
<td>Ethidium Homodimer-1</td>
<td>Invitrogen</td>
<td>Cat#E1169</td>
</tr>
<tr>
<td>1<b>×</b> PBS</td>
<td>Corning</td>
<td>Cat# 21040CM</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Nuclei PURE Prep Isolation Kit</td>
<td>Sigma</td>
<td>Cat# NUC201-1KT</td>
</tr>
<tr>
<td>RNA Screen Tape</td>
<td>Agilent Technologies</td>
<td>Cat# 5067-5576</td>
</tr>
<tr>
<td>Chromium Single Cell 3′ kit</td>
<td>10<b>×</b> Genomics</td>
<td>Cat#1000153</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Surgically resected human hippocampus tissue (mixed sex, 24–60 years old)</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Eppendorf® Centrifuge 5810/5810R</td>
<td>Eppendorf</td>
<td>Cat# EP022628188</td>
</tr>
<tr>
<td>1.5 mL Eppendorf tubes</td>
<td>Eppendorf</td>
<td>Cat# 022431021</td>
</tr>
<tr>
<td>2 mL Eppendorf tubes</td>
<td>Eppendorf</td>
<td>Cat# EP022363344</td>
</tr>
<tr>
<td>15 mL conical tubes</td>
<td>Corning</td>
<td>Cat# CLS430791</td>
</tr>
<tr>
<td>Dounce homogenizer</td>
<td>DWK Life Sciences</td>
<td>Cat# 8853000002</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p><b>Neurobasal A medium</b></p>
<p>Aliquot ∼30 mL of Neurobasal A Medium in a 50 mL conical tube for each specimen to be transported from the operation room. Prechill on ice.</p>
<p><b>Nuclei EZ lysis buffer</b></p>
<p>Aliquot 8 mL for each sample to be processed and prechill on ice.</p>
<table id="undtbl1">
<caption>Nuclei suspension buffer (NSB)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>4 mL</td>
</tr>
<tr>
<td>5% BSA</td>
<td>1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>RNase inhibitor</td>
<td>0.2U/ul</td>
<td>50 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh and keep on ice.</p>
<table id="undtbl2">
<caption>Sucrose solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclei PURE 2M Sucrose Cushion Solution</td>
<td>N/A</td>
<td>2.7 mL</td>
</tr>
<tr>
<td>Nuclei PURE Sucrose Cushion Buffer</td>
<td>N/A</td>
<td>0.3 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh and keep on ice.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Tissue collection, transport, and banking</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>Here we describe the best practices to transport and store the resected hippocampal tissue after surgical removal.</p>
<ol>
<li>The neurosurgeon collects the anterior and posterior hippocampal tissue by initially resecting the lateral temporal cortex to expose the hippocampus after opening the temporal horn of the ventricle.
<ol type="a">
<li>Following division of the amygdalo-hippocampal sulcus, the hippocampus is reflected inferiorly on its vascular pedicle until the fibria and adjacent choroidal fissure are visualized. After opening the alveus just lateral to the fimbria, an anterior-posterior line of dissection is carried along the hippocampal axis until the arteries feeding the hippocampus are identified and sequentially divided, freeing the hippocampus from its pedicle in an en bloc fashion. The tail of the hippocampus is severed posterior to the lateral geniculate nucleus.
<div class="note">
<span class="note-title">Note:</span> Only specimens greater than 3.5 cm in length in the anterior-posterior dimension are included.</div>
</li>
<li>An approximately 0.5 cm segment is removed from the anterior and posterior poles of this specimen for processing. Anterior specimens are taken from a location anterior to the uncal notch.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Send the middle portion of the hippocampus for neuropathological analysis.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This pathological analysis is a key step of the tissue exclusion criteria. Specimens with mesial temporal sclerosis (MTS) are excluded unless the scope of the study includes studying the pathogenesis related to MTS. Other neuropathological findings such as astrogliosis are noted and shared along with the study results.</div>
<ol start="3">
<li>Carefully drop anterior and posterior hippocampal specimens into separate 50 mL conical tubes containing ice-cold Neurobasal A Medium after surgical resection. Place the tubes on ice and transport the tissue back to the laboratory within 20 min for processing.</li>
<li>Wash tissue in ice cold 1<b>×</b> PBS. Using a scalpel, dissect out ∼25% of each specimen for RNA integrity analysis.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Since it is not possible to get RNA integrity (RIN) values once tissue is processed as single nuclei, it is important to isolate RNA from a small piece of adjacent tissue. RIN values for each specimen will provide insights into RNA quality for a given sample and can be used as a covariate for computational analysis.</div>
<ol start="5">
<li>Transfer specimens into prechilled and prelabelled Eppendorf tubes. Flash-freeze the tissue in liquid nitrogen.</li>
<li>RNA isolation and RIN: Isolate total RNA from the dissected piece (from Step 4) using a standard RNA isolation. Determine RIN values for each sample using a RNA Screen Tape Analysis system (Agilent Technologies) according to the manufacturer’s instructions</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Label the tubes with both case identification number and date of the tissue collection. Update the tissue inventory spreadsheet accordingly. Using the date and time of the surgery can be a more reliable way to track the samples and helps avoid any confusion between the lab and surgical department.</div>
<h3 id="sec3.2">Nuclei isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This section includes steps for the complete digestion of the plasma membrane yielding single-nuclei dissociation of human hippocampus. Similar approaches have been used in previous studies (<a href="#bib2">Habib et al., 2017</a>). We adopted this dissociation protocol and the coupled it with a modified sucrose gradient.</p>
<ol start="7">
<li>Using a scalpel, dissect the surgically resected hippocampal tissue into smaller than ∼0.3 cm<sup>3</sup> on a petri dish placed on ice. This step should be performed quickly without allowing the tissue to thaw.</li>
<li>Lyse the tissue pieces
<ol type="a">
<li>Transfer tissue pieces to a Dounce homogenizer containing 2 mL of ice-cold Nuclei EZ lysis buffer.</li>
<li>Use pestle A for 20 strokes, followed by 20 strokes with pestle B while on ice.</li>
<li>Transfer the homogenate to a 15 mL conical tube, add 2 mL of ice-cold Nuclei EZ lysis buffer and mix by pipetting with a glass pipette. Incubate on ice for 5 min.</li>
<li>Collect the nuclei by centrifugation at 500 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Discard supernatant and carefully resuspend nuclei in 4 mL of ice-cold Nuclei EZ lysis buffer. Incubate on ice for 5 min.</li>
<li>Repeat the centrifugation at 500 <b>×</b> <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
</ol>
</li>
<li>Resuspended the nuclei in 500 μL of nuclei suspension buffer (NSB) consisting of 1<b>×</b>PBS, 1%BSA and 0.2 U/ul RNAse inhibitor.</li>
<li>Filter nuclei suspension using a Flowmi Cell Strainer (Bel-Art, H13680-0040). Based on the amount of debris in the suspension, it might be hard to filter. Try filtering smaller volumes and using a new filter for every ∼100 μL volume.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this point of the protocol tissue lysis is complete and nuclei are dissociated in the suspension. However, using adult human brain as starting tissue results in a large amount of debris. Filtering is effective in removing debris larger than nuclei however smaller size cellular and/or myelin debris often remains after filtering. We implemented the following sucrose gradient to obtain a cleaner nuclei suspension.</div>
<h3 id="sec3.3">Sucrose gradient</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Preparing a nuclei suspension with minimal debris is key to minimize technical noise in snRNA-seq datasets. Typically, ultracentrifugation at higher speeds has been used to separate subcellular compartments such as nuclei (<a href="https://www.protocols.io/view/nuclei-isolation-from-human-brain-using-sucrose-gr-scneave">https://www.protocols.io/view/nuclei-isolation-from-human-brain-using-sucrose-gr-scneave</a>). Working with relatively smaller tissue amounts from human hippocampus, we noticed that there is a high degree of nuclei loss during ultracentrifugation potentially due to the larger volume of tubes used in the ultracentrifuge. Adopting the modified sucrose gradient using a tabletop centrifuge as is detailed below minimized the nuclei loss.</p>
<ol start="11">
<li>Remove the debris using a density gradient centrifugation
<ol type="a">
<li>Add 500 μL of the sucrose solution (see <a href="#materials-and-equipment">materials and equipment</a>) to a 2 mL Eppendorf tube.</li>
<li>Add 900 μL of the sucrose solution to 500 μL of isolated nuclei in NSB.</li>
<li>Layer 1400 μL nuclei suspension (from Step 11b) to the top of the sucrose solution (from Step 11a).</li>
<li>Centrifuge this gradient at 13,000 <b>×</b> <i>g</i> for 45 min at 4°C.</li>
</ol>
</li>
<li>Resuspend and wash nuclei pellet
<ol type="a">
<li>Carefully remove the myelin layer on the top and the supernatant containing the cellular debris (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1025-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Sucrose gradient for isolation of high-quality nuclei</div>
<p>(A) Images showing phases separated by sucrose gradient.</p>
<p>(B) Nuclei isolated using this protocol stained by Ethidium Homodimer-1. Insets show closeup displaying individual nuclei without DNA blebbing. Scale bar = 50 microns.</p>
</figcaption>
</figure>
</li>
<li>Resuspend the nuclei pellet in 500 μL NSB and transfer the solution to a 1.5 mL Eppendorf tube.</li>
<li>Centrifuge at 500 <b>×</b> <i>g</i> for 5 min at 4°C</li>
<li>Remove the supernatant and resuspend the pellet in 100–300 μL NSB</li>
<li>Filter the suspension through a 40-μm Flowmi Cell Strainer (#H13680-0040, Bel-Art).</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Quality control and nuclei count</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Here we describe how we evaluate the quality of the nuclei suspension and calculate the nuclei concentration. In terms of quality control, we consider this protocol successful if 1) nuclei appear as a single-nuclei solution and are not clumped together, and 2) nuclei look intact. For these quality control steps the following staining approaches were used.</p>
<ol start="13">
<li>Assess nuclei integrity with Ethidium Homodimer-1 (EthD-1) staining.
<ol type="a">
<li>Stain 10 μL of nuclei suspension (from 11e) with 4 μM EthD-1 (<a href="#fig1">Figure 1</a>B). EthD-1 emits red fluorescence upon binding to DNA and will show DNA blebbing if the sample quality is low. Visualize stained nuclei under an epifluorescence microscope in the red channel.</li>
</ol>
</li>
<li>Count nuclei with Trypan Blue staining
<ol type="a">
<li>Stain 10 μL of nuclei suspension (from Step 11e) with 10 μL of premade 0.4% Trypan Blue.</li>
<li>Load the stained sample on a hemocytometer and visualize the nuclei under a bright field microscope. Count blue/purple stained nuclei.</li>
</ol>
</li>
<li>Adjust the final concentration of nuclei to 1000 nuclei/μL with NSB.</li>
<li>Continue with preparation of droplet-based single-nuclei RNA-seq libraries using the Chromium Single Cell 3′ kit (10<b>×</b> Genomics) according to the manufacturer’s protocol</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol results in obtaining a total of ∼50,000–100,000 nuclei per 0.3 cm<sup>3</sup> tissue sample. The final number of nuclei varies based on how much white matter is present in the starting tissue piece. In general, anterior hippocampal samples contained more white matter tracks. Single-nuclei RNA-sequencing libraries prepared following these preparations consistently enabled sampling of excitatory neurons, inhibitory neurons, astrocytes, oligodendrocytes, microglia and oligodendrocyte progenitor cells.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>A sucrose gradient, washes and filtering steps are implemented to obtain intact nuclei without cellular and myelin debris. However, some degree of ambient RNA will still exist in the nuclei suspension and will result in barcoded droplets containing low RNA count during snRNA-seq. These artifacts will then be removed from the dataset by filtering based on the number of transcripts and genes found associated with each cell barcode. Please see (<a href="#bib1">Ayhan et al., 2021</a>) for an examples of filtering criteria.</p>
<p>Here, we used a commercially available sucrose buffer for the sucrose gradient. Theoretically, homemade sucrose solution could be used (<a href="https://www.protocols.io/view/nuclei-isolation-from-human-brain-using-sucrose-gr-scneave">https://www.protocols.io/view/nuclei-isolation-from-human-brain-using-sucrose-gr-scneave</a>). However, centrifuge times, speed and the molar concentration for the gradient will need to be optimized for the tabletop centrifuge. Similarly, we used a commercially available lysis buffer. It is possible to use a homemade lysis buffer (e.g., 10 mM Tris-HCl, 10 mM NaCl, 3 mM MgCl2, and 0.1% Nonidet™ P40 Substitute) but lysis times will need to be tested.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Nuclei form clumps rather than a single-nuclei solution (step 14b).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Nuclei clumping indicates that nuclei are over lysed. This could be due to lysis time that was too long for the tissue of interest. Repeat the experiment using shorter times for lysis and/or fewer number of strokes with the Dounce homogenizer. In fact, the number of strokes and lysis will have to be optimized for different tissue types and amount of tissue used. Alternatively, this could also occur if the reagents used were warmer than ice-cold. If the tissue sample is too valuable to be discarded, repeating a filtering step might help to remove clumps if there is enough non clumping single-nuclei in the solution.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>No visible pellet after sucrose gradient (step 12d).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>If the starting tissue amount is low, the pellet containing nuclei after the sucrose gradient will be too small to visualize. In that case, resume the protocol as directed without touching the potential invisible pellet. Adjust the final resuspension volume accordingly as the number of nuclei will be low.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Cell bodies are found in the nuclei suspension (step 14b).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The presence of cell bodies in the trypan blue stained nuclei suspension indicates incomplete lysis. Increasing lysis time and/or cutting the tissue into smaller pieces before starting lysis is recommended.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Number of genes detected is lower than expected after single-nuclei sequencing (downstream result).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Low quality tissue preparations can lead to low quality data after sequencing. The steps for tissue processing should be considered carefully. Addition of RNase inhibitors to the lysis buffer can also help with protecting the RNA during lysis.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ayhan, F., Kulkarni, A., Berto, S., Sivaprakasam, K., Douglas, C., Lega, B.C., and Konopka, G. (2021). Resolving cellular and molecular diversity along the hippocampal anterior-to-posterior axis in humans. Neuron <i>109</i>, 2091-2105.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00550-5/sref1">View at publisher</a></p>
<p id="bib2">Habib, N., Avraham-Davidi, I., Basu, A., Burks, T., Shekhar, K., Hofree, M., Choudhury, S.R., Aguet, F., Gelfand, E., Ardlie, K., et al. (2017). Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat. Methods <i>14</i>, 955-958. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00550-5/sref2">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Genevieve Konopka (<a href="mailto:Genevieve.konopka@UTSouthwestern.edu">Genevieve.konopka@UTSouthwestern.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique data sets or code. Please see (<a href="#bib1">Ayhan et al., 2021</a>) for data sets generated using this protocol.</p>
<h3>Acknowledgments</h3>
<p>The authors thank the patients for consenting to participate in this research and Haley Moore for providing the images shown in <a href="#fig1">Figure 1</a>A. G.K. is a Jon Heighten Scholar in Autism Research at UT Southwestern. This work was supported by the <a href="https://doi.org/10.13039/100000002">NIH</a> (NS106447), a UT BRAIN Initiative Seed Grant (366582), the Chilton Foundation, and the National Center for Advancing Translational Sciences of the <a href="https://doi.org/10.13039/100000002">NIH</a> under the Center for Translational Medicine’s award number UL1TR001105 to G.K. and B.C.L., the <a href="https://doi.org/10.13039/100000002">NIH</a> (T32DA007290 and T32HL139438) to F.A., the <a href="https://doi.org/10.13039/100000002">NIH</a> (NS107357) to B.C.L., and the Chan Zuckerberg Initiative, an advised fund of <a href="https://doi.org/10.13039/100000923">Silicon Valley Community Foundation</a> (HCA-A-1704-01747) and the <a href="https://doi.org/10.13039/100000913">James S. McDonnell Foundation</a> 21<sup>st</sup> Century Science Initiative in Understanding Human Cognition – Scholar Award (220020467) to G.K. Some images in the Graphical Abstract are reprinted with permission from <a href="#bib1">Ayhan et al. (2021)</a>.</p>
<h3>Author contributions</h3>
<p>B.C.L. resected the hippocampal tissue. F.A. designed and optimized the protocol. C.D. assisted with tissue collection and banking. F.A. and G.K. wrote the manuscript. G.K. supervised the study.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>