<article>
<section>
<h1 id="header">Fluorescent <i>in situ</i> mRNA detection in the adult mouse cochlea</h1>
<p><time datetime="2021-08-13">Published: August 13, 2021</time></p>
<p>Giovanni H. Diaz<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> and Stefan Heller<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Otolaryngology – Head &amp; Neck Surgery, Stanford University School of Medicine, Stanford, CA 94305, USA</p>
<p id="aff2"><sup>2</sup>Institute for Stem Cell Biology and Regenerative Medicine, Stanford University School of Medicine, Stanford, CA 94305, USA</p>
<p id="aff3"><sup>3</sup>Department of Developmental Biology, Stanford University School of Medicine, Stanford, CA 94305</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ghdiaz@stanford.edu">ghdiaz@stanford.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:hellers@stanford.edu">hellers@stanford.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100711">10.1016/j.xpro.2021.100711</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Ossification and the delicateness of the cochlear duct make histologic assessments of the mature cochlea a challenging endeavor. Treatments to soften the bone facilitate sectioning and dissection of the cochlear duct but limit <i>in situ</i> mRNA detection in such specimens. Here, we provide a protocol for <i>in situ</i> mRNA detection using hybridization chain reaction in whole-mount preparations of the adult mouse cochlea. We show examples for multi-probe detection of different mRNAs and describe combination of this method with conventional immunohistochemistry.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Instructions for preparing the adult mouse cochlea for whole-mount histology applications</li>
<li>Modifications for hybridization chain reaction protocol</li>
<li>Adult cochlea mRNA <i>in situ</i> detection paired with immunostaining and imaging</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/910-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Preparation of mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [15 min]</div>
<ol>
<li>Reporter mice that express fluorescent tdTomato protein in cochlear hair cells are used in this protocol for visualization of the organ of Corti during cochlear dissection. Using fluorescent reporter proteins is helpful but not absolutely necessary. The example presented in this protocol utilizes transgenic mice that carry heterozygous Myo15Cre and Ai14tdTomato alleles.</li>
<li>Mice were euthanatized in accordance with an IACUC-approved animal study protocol, which included CO<sub>2</sub> inhalation for 8 min, followed by cervical dislocation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Homozygous Myo15Cre mice are deaf (<a href="#bib4">Pepermans et al., 2014</a>). Mating homozygous Myo15Cre mice with homozygous Ai14tdTomato mice results in mice with normal hearing thresholds.</div>
<div class="note">
<span class="note-title">Note:</span> Heterozygous Myo15Cre/Ai14tdTomato mice express tdTomato at equivalent levels between cochlear inner and outer hair cells in neonates. At four weeks of age, the red fluorescence in outer hair cells is much lower when compared with inner hair cells. Additionally, a subset of neurons in the spiral ganglia expresses tdTomato.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit anti-Myosin7a (1:1000)</td>
<td>Proteus</td>
<td>Cat# 256790; RRID: AB_10015251</td>
</tr>
<tr>
<td>Mouse anti-Ctbp2 (1:200)</td>
<td>Becton Dickinson</td>
<td>Cat# 612044;RRID: AB_399431</td>
</tr>
<tr>
<td>Alexa Fluor 647 donkey anti-rabbit IgG(H+L) secondary (1:200)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A31573;RRID: AB_2536183</td>
</tr>
<tr>
<td>Alexa Fluor 647 donkey anti-mouse IgG(H+L) secondary (1:200)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A31571;RRID: AB_162542</td>
</tr>
<tr>
<td>DAPI (1:1000)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# D1306, RRID: AB_2629482</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Ocm HCR DNA probe</td>
<td>NM_033039.3</td>
<td>Lot# PRC932</td>
</tr>
<tr>
<td>Otof HCR DNA probe</td>
<td>NM_001100395.1</td>
<td>Lot# PRC934</td>
</tr>
<tr>
<td>Efhd2 HCR DNA probe</td>
<td>NM_025994.3</td>
<td>Lot# PRC935</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>1× DPBS without Ca<sup>2+/</sup>Mg<sup>2+</sup>
</td>
<td>Corning</td>
<td>Cat# 21-031-CV</td>
</tr>
<tr>
<td>10× DPBS without Ca<sup>2+/</sup>Mg<sup>2+</sup>
</td>
<td>Corning</td>
<td>Cat# 46-013-CM</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM9932</td>
</tr>
<tr>
<td>RNAlater™ Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM7021</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Sigma-Aldrich</td>
<td>Cat# P9416</td>
</tr>
<tr>
<td>20× Sodium chloride citrate (SSC)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM9770</td>
</tr>
<tr>
<td>Proteinase K, Molecular Biology Grade (800 units/mL)</td>
<td>New England Biolabs</td>
<td>Cat# P8107S</td>
</tr>
<tr>
<td>Bovine Serum Albumin (BSA)</td>
<td>Millipore Sigma</td>
<td>Cat# 9048-46-8</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat# X100</td>
</tr>
<tr>
<td>4,6-Diamidino-2-phenylindole (DAPI, 1 μg/mL PBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# D1306</td>
</tr>
<tr>
<td>FluorSave™ Reagent</td>
<td>Calbiochem</td>
<td>Cat# 345789-20 mL</td>
</tr>
<tr>
<td>16% Paraformaldehyde Aqueous Solution</td>
<td>Electron Microscopy Sciences</td>
<td>Cat# 15710</td>
</tr>
<tr>
<td>HCR probe hybridization buffer</td>
<td>Molecular Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>HCR probe wash buffer</td>
<td>Molecular Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>HCR amplification buffer</td>
<td>Molecular Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Myo15-Cre mice</td>
<td>Dr. Christine Petit, Pasteur Institute</td>
<td>N/A</td>
</tr>
<tr>
<td>Ai14-tdTomato mice</td>
<td>The Jackson Laboratory</td>
<td>Stock# 007908; MGI:3817869; RRID: IMSR_JAX:007908</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Fiji/ImageJ</td>
<td>Fiji</td>
<td>RRID:SCR_002285 <a href="http://fiji.sc">http://fiji.sc</a>
</td>
</tr>
<tr>
<td>Black Zen</td>
<td>Zeiss</td>
<td>RRID:SCR_018163 <a href="http://www.zeiss.com/microscopy/en_us/products/microscope-software/zen.html#introduction">http://www.zeiss.com/microscopy/en_us/products/microscope-software/zen.html#introduction</a>
</td>
</tr>
<tr>
<td>Blue Zen</td>
<td>Zeiss</td>
<td>RRID:SCR_013672 <a href="http://www.zeiss.com/microscopy/en_us/products/microscope-software/zen.html#introduction">http://www.zeiss.com/microscopy/en_us/products/microscope-software/zen.html#introduction</a>
</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Heratherm OMS60 Lab Oven</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 51028121</td>
</tr>
<tr>
<td>Secure-Seal™ hybridization sealing system, (9 mm diameter, Electron Microscopy Sciences)</td>
<td>Electron Microscopy Sciences</td>
<td>Cat# 70333-40</td>
</tr>
<tr>
<td>Adhesive Port Seal Tabs</td>
<td>Electron Microscopy Sciences</td>
<td>Cat# 70328-00</td>
</tr>
<tr>
<td>Eppendorf tube revolver rotator (Hematology/Chemistry Mixer 346)</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Falcon 50 mL conical centrifuge tube</td>
<td>Corning</td>
<td>Cat# 352070</td>
</tr>
<tr>
<td>Microcentrifuge tube storage box for Eppendorf tubes (Axygen brand)</td>
<td>VWR</td>
<td>Cat# TR8300-BLK</td>
</tr>
<tr>
<td>Micro Dissecting Curette/Spoon</td>
<td>Biomedical Research Instruments</td>
<td>Cat# BRI 15-1020</td>
</tr>
<tr>
<td>SYLGARD® 184 silicone</td>
<td>World Precision Instruments</td>
<td>Cat# SYLG184</td>
</tr>
<tr>
<td>Vannas Micro Dissecting Spring Scissors; Straight; 2 mm Cutting Edge; 0.05 mm Tip Width; 3.5" Overall Length</td>
<td>Roboz</td>
<td>Cat# RS-5640</td>
</tr>
<tr>
<td>Scalpel Blades - Breakable</td>
<td>Fine Science Tools</td>
<td>Cat# 10050-00</td>
</tr>
<tr>
<td>Blade Holder &amp; Breaker - Flat Jaws</td>
<td>Fine Science Tools</td>
<td>Cat# 10052-11</td>
</tr>
<tr>
<td>Superfrost Plus microscope slides</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12-550-15</td>
</tr>
<tr>
<td>18G 1 Sterile Precision Glide Needles</td>
<td>Becton Dickinson</td>
<td>Cat# 305195</td>
</tr>
<tr>
<td>30G ½ Hypodermic needle</td>
<td>Becton Dickinson</td>
<td>Cat# 305106</td>
</tr>
<tr>
<td>1 mL Syringe Luer Lok Tip</td>
<td>Becton Dickinson</td>
<td>Cat# 309628</td>
</tr>
<tr>
<td>35×10 mm Dish</td>
<td>Corning</td>
<td>Cat# 353001</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Preparation of 4% paraformaldehyde (PFA) solution</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [5 min]</div>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× PBS (without Ca<sup>2+/</sup>Mg<sup>2+</sup>)</td>
<td>1×</td>
<td>4 mL</td>
</tr>
<tr>
<td>RNase-free water</td>
<td>N/A</td>
<td>26 mL</td>
</tr>
<tr>
<td>16% PFA</td>
<td>4% PFA</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> 4% PFA solution is made fresh before each cochlear dissection.</div>
<div class="note">
<span class="note-title">Note:</span> Using PBS with Ca<sup>2+</sup> and Mg<sup>2+</sup> will result in increased autofluorescence.</div>
<h3 id="sec2.2">Preparation of 5× saline sodium citrate buffer with Tween-20 (SSCT)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [5 min]</div>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>20× SSC</td>
<td>5×</td>
<td>10 mL</td>
</tr>
<tr>
<td>10% Tween-20</td>
<td>0.1%</td>
<td>400 μL</td>
</tr>
<tr>
<td>DEPC-treated water</td>
<td>N/A</td>
<td>30 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare a stock solution of 10% Tween-20 in RNase-free water. This solution can be stored at 4°C for several months.</div>
<h3 id="sec2.3">Preparation of PBS with Tween-20 (PBST)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [5 min]</div>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS without Ca<sup>2+</sup>/Mg<sup>2+</sup>
</td>
<td>N/A</td>
<td>50 mL</td>
</tr>
<tr>
<td>10% Tween-20</td>
<td>0.1%</td>
<td>500 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This solution can be stored at 4°C for several months.</div>
<h3 id="sec2.4">Preparation of immunostaining solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [120 min]</div>
<table id="undtbl4">
<caption>Antibody wash buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS (without Ca<sup>2+</sup>/Mg<sup>2+</sup>)</td>
<td>N/A</td>
<td>20 mL</td>
</tr>
<tr>
<td>10% Triton X-100</td>
<td>0.2%</td>
<td>400 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare a stock solution of 10% Triton X-100 in PBS without Ca<sup>2+</sup>/Mg<sup>2+</sup>. This solution can be stored at 4°C for several months.</div>
<table id="undtbl5">
<caption>Antibody blocking buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS with 0.2% Triton X-100</td>
<td>N/A</td>
<td>20 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>1%</td>
<td>0.2 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>Antibody incubation buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS (without Ca<sup>2+</sup>/Mg<sup>2+</sup>)</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
<tr>
<td>PBS with 0.2% Triton X-100</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>0.5%</td>
<td>0.1 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> PBS with 0.2% Triton X-100 can be stored at 4°C for several months.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare blocking and incubation buffers fresh prior to each immunostaining for consistent results. Store solutions at 4°C between primary and secondary antibody application.</div>
<div class="note">
<span class="note-title">Note:</span> BSA dissolves slowly. Preparing the solution and then placing the tube on a shaker set to gentle rocking for 90–120 min is recommended.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Extraction of the otic capsule</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [60 min, followed by incubation for 12–16 h]</div>
<ol>
<li>Euthanize the four-week-old mice (postnatal day (P) 28). Male and female mice are used.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The workflow and timing information assumes that three mice (6 cochleae) are used.</div>
<ol start="2">
<li>Decapitate the mouse and bisect the skull with two scissor cuts, the first from the base of the brainstem above the skull to the nose and the second from the base of the brainstem to the lower jaw.</li>
<li>Remove the brain tissue to expose the otic capsules (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/910-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Dissection of the cochlear apical turn</div>
<p>(A) Bisected P28 mouse head. The otic capsule containing the cochlea and the vestibular system is outlined with a green dashed line.</p>
<p>(B) Dissected mouse otic capsule. Green arrowhead denotes the apical turn crease that is targeted to create a window in the apex.</p>
<p>(B′) Magnified image of the cochlea’s apex. The arrow points to the crease of the apical turn that is used as a toehold for creating a window in the apex.</p>
<p>(B’’) Otic capsule with the apical bone removed. The pried-off bone piece is seen to the right of the otic capsule.</p>
<p>(B’’’) A higher magnified view of the window made at the apex of the otic capsule.</p>
<p>(C) Otic capsule after removal of bone to create window in the apex. The location of the round window is indicated by a blue dashed line and the oval window is outlined with a purple dashed line.</p>
<p>(C′) Magnified image of the round window (blue dashed line) and oval window (purple dashed line).</p>
<p>(C’’) Otic capsule after the removal of the bone covering the sensory epithelium.</p>
<p>(C’’’) Magnified image of the exposed cochlear turns after bone removal.</p>
<p>(D) Dissected apical turn after excess stria vascularis and spiral ganglion tissue was removed.</p>
<p>(D′) Strong Myo15Cre-tdTomato expression is associated with P28 inner hair cells and lower levels are observed in outer hair cells in the shown apical turn. Some spiral ganglion cells also express tdTomato.</p>
</figcaption>
</figure>
</li>
<li>Extract each otic capsule by applying pressure with a gloved thumbnail above the apex of the cochlea. Hold a bisected skull in the left hand and apply pressure to the tissue behind the otic capsule so that the capsule is pushed upwards, while simultaneously pushing down on the semicircular canal with the other thumb. This will lift the otic capsule so that it can be removed from the surrounding tissues using RNase-free forceps and placed directly into 4% PFA solution.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Transfer the extracted otic capsules directly into 4% PFA solution at room temperature (18°C–24°C).</div>
<p><b>Caution:</b> PFA is a hazardous material, and the following steps need to be conducted using a dissection microscope inside a chemical hood.</p>
<ol start="5">
<li>Perfuse 4% PFA solution into the round and oval windows of the isolated otic capsule.
<ol type="a">
<li>Transfer the isolated otic capsules into 4% PFA solution in a 35×10 mm dish, coated with black Sylgard for dissections (<a href="#bib3">Montgomery and Cox, 2016</a>).</li>
<li>Using a curved pair of sharp forceps or an 18-gauge needle that has been curved on a sharpening/grinding block, place the curved sharp tip along the crease of the apical cochlear turn and apply gentle pressure to remove a fragment of the apical bone (<a href="#fig1">Figures 1</a>B and 1B′). This movement generates a small window at the apex (<a href="#fig1">Figures 1</a>B’ and 1B’’).</li>
<li>Using a 30-gauge needle, slowly perfuse about 100 μL of 4% PFA solution into the round and oval windows (blue and purple dotted circles in <a href="#fig1">Figure 1</a>C, respectively). Slowly push the solution through the cochlea using gentle pressure, at a flow of ≈50 μL per minute. The solution will flow out of the window made in the apex.
<div class="note">
<span class="note-title">Note:</span> Remove the stapes with forceps to allow for easier perfusion.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Thorough perfusion with 4% PFA solution is essential for completely fixing the tissue. Additionally, this process makes the sensory epithelium more rigid, mediating easier removal from the bone.</div>
</li>
<li>Place the perfused cochleae into a 2 mL Eppendorf tube containing 1.8 mL of 4% PFA solution.</li>
<li>Incubate 12–16 h on a shaker set to gentle rocking at 4°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Separate the left and right cochleae into different Eppendorf tubes to help with subsequent orientation of the tissue.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Limit the number of otic capsules that are placed in the 1.8 mL of fixative to six to ensure thorough fixation.</div>
<h3 id="sec3.2">Dissection of all three cochlear turns from decalcified otic capsules</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [<b>4 h 30 min</b><b>]</b>
</div>
<ol start="6">
<li>After overnight fixation, transfer the cochleae into RNase-free 0.5 M EDTA solution pH 8.0.</li>
<li>Incubate for 3 h with slow agitation with an Eppendorf tube revolver rotator (Fisher Scientific Hematology/Chemistry Mixer 346) at room temperature.</li>
<li>Rinse the cochleae once in RNase-free PBS and then transfer the cochleae into RNase-free PBS at room temperature for dissection.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The cochlear bone will not be fully decalcified after 3 h, however it will be softened enough to remove the cochlear turns. Additional decalcification will result in easier dissection, but prolonged EDTA treatment can reduce downstream mRNA detection efficacy (<a href="#bib1">Belluoccio et al., 2013</a>, <a href="#bib5">Shao et al., 2006</a>). For immunostaining, we have not noticed adverse effects of extended decalcification times (for up-to 6 days).</div>
<div class="note">
<span class="note-title">Note:</span> Even with a short 3 h EDTA decalcification time, we have noted that <i>Otof</i> mRNA detection levels where affected when directly compared to non-decalcified tissue.</div>
<div class="note">
<span class="note-title">Note:</span> Protocols have suggested that co-incubation of EDTA with RNAlater solution (Invitrogen) improves RNA preservation and detection (<a href="#bib1">Belluoccio et al., 2013</a>). We have not systematically investigated the effects of RNAlater on wholemount samples, but have noted no clear improvement of mRNA detection in experiments on cryosectioned cochlear tissue. Future optimization is warranted to see if this protective step can further improve mRNA detection in wholemount cochleae.</div>
<div class="note">
<span class="note-title">Note:</span> Decalcified otic capsule dissection was done according to (<a href="#bib3">Montgomery and Cox, 2016</a>).</div>
<ol start="9">
<li>After decalcification, insert one blade of Vannas Spring Scissors into the oval window (purple dotted line in <a href="#fig1">Figures 1</a>C and 1C′) and cut along the turn towards the apex.</li>
<li>To separate the basal turn from the middle and apical turn, place one scissor blade into the region cut in step 9 with the other scissor blade on the back of the cochlea and cut.</li>
<li>Hold the apical/middle turn so that the apical end points down. Cut along the scala media.</li>
<li>To separate the middle and apical turns, place one scissor blade into the region that was cut in step 11 and place the other blade on the outside of the bony labyrinth and cut.</li>
<li>To remove the basal turn, cut the spiral ganglion fibers that connect to the basal turn and cut along the lateral wall.</li>
<li>Use the scissors to remove excess bone and spiral ganglion tissue.</li>
<li>To help with removal of surrounding tissue, as it is sometimes challenging to get the correct angle with the scissors, transfer turns to a 35×10 mm plastic Petri dish filled with RNase-free PBS and use a 0.1 mm thick scalpel blade mounted to a blade holder to remove excess tissue.</li>
</ol>
<h3 id="sec3.3">Dissection of the apical turn, without decalcification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [45 min]</div>
<p>We describe in the following an alternative dissection that focuses on the apical turn. The benefit of this dissection is that it can be done without decalcification, which increases the sensitivity of subsequent mRNA detection. The following steps should be conducted after step 5 and instead of steps 6–15.</p>
<ol start="16">
<li>Following the overnight fixation, transfer the cochleae into RNase-free PBS without Ca<sup>2+</sup>/Mg<sup>2+</sup> for dissection.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The samples should be processed immediately for best signals and not stored at 4°C.</div>
<ol start="17">
<li>Use the sharp tip of a curved 18-gauge needle to chip away the bone from the cochlear apical turn (<a href="#fig1">Figures 1</a>B and 1B′).
<ol type="a">
<li>Use the curved tip to gently hook the bone and to chip it away from the apical window opening that was previously made (<a href="#fig1">Figures 1</a>B’’ and 1B’’’) until all the bone surounding the sensory epithelium is removed. An example of a cochlea after the bone surrounding the sensory epithelium is removed is shown in (Figures 1C’’ and 1C’’’).
<div class="critical">
<span class="critical-title">Critical:</span> Scrape the region between the stria vascularis and the bone with the sharp needle to gently separate the two compartments. This step decreases the chance of removing the sensory epithelium along with the chipped bone.</div>
<div class="note">
<span class="note-title">Note:</span> Carefully bend the tip of the 18-gauge needle to generate a sharp curved “hook”.</div>
</li>
<li>Use Vannas Spring Scissors to dissect the apical turn.</li>
<li>Remove excess stria vascularis tissue with the scissors.</li>
<li>To aid with removal of surrounding tissue, as it is sometimes challenging to get the correct angle with the micro scissors, transfer apical turns to a 35×10 mm plastic Petri dish filled with RNase-Free PBS, and use a 0.1 mm thick scalpel blade mounted to blade holder to remove excess tissue. After excess tissue is removed you will be left with an apical turn with hair cells visualization as presented in (<a href="#fig1">Figures 1</a>D and 1D′).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A disposable scalpel may not be thin enough to aid in this dissection step. The thinner 0.1 mm breakable FST blades (Cat# 10050-00) are recommended.</div>
<div class="note">
<span class="note-title">Note:</span> It is possible to remove parts of the middle and basal turns from non-decalcified cochleae. However, these turns are difficult to reliably extract because the tissue tends to tear between the outer and inner hair cells resulting in removal of outer hair cells. For analyses of inner hair cells only and potentially also spiral ganglion cells, one can use such tissue for downstream RNA detection.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use a micro spoon to transfer fully immersed tissue.</div>
<h3 id="sec3.4">Mounting of cochlear turns to slide</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [20 min]</div>
<ol start="18">
<li>Cut in half a sheet of Secure-Seal™ hybridization chamber system (hybridization chamber) for two stripes of four chambers each (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/910-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Mounting Hybridization chambers and humidifying chamber</div>
<p>(A) One stripe of four hybridization chambers placed onto a microscope slide.</p>
<p>(B) A piece of Post-it paper is cut to the dimensions of a microscope slide and four ≈ 9 mm circles or dots are made that align with the circular hybridization chambers to create a visual guide. The sticky section of the Post-it aligns with the frosted end of the slide.</p>
<p>(C) A Superfrost Plus microscope slide is placed on top of the visual guide and the sticky part of the Post-it paper is used to hold the visual guide in place.</p>
<p>(D) Microcentrifuge storage box used as humidifying chamber. The slots are filled with pieces of paper towel and water.</p>
<p>(E) A plastic inset of a pipette tip box is used as a slide holder.</p>
</figcaption>
</figure>
</li>
<li>Use a marker to outline four 9 mm diameter circles on a piece of paper that align with the four hybridization chambers. (<a href="#fig2">Figure 2</a>B). A Post-It note is useful for this because it can be temporarily attached to the back of a microscopy slide, preventing the visual guide from moving (<a href="#fig2">Figure 2</a>C).</li>
<li>Place 10 μL of RNAse-free PBS without Ca<sup>2+</sup>/Mg<sup>2+</sup> onto the front of the Superfrost Plus microscope slide corresponding to each marked region (<a href="#fig2">Figure 2</a>C).</li>
<li>Transfer the sensory epithelium turns into the 10 μL PBS drop using a micro spoon and orient the turns using a pair of forceps using a dissection microscope.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is possible to place and orient up to five cochlear turns into each 10 μL drop as long as they do not overlap when mounted.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Orient the tissue so that the hair cells are facing upwards with the bundles pointing toward the surface.</div>
<ol start="22">
<li>After the tissue specimens are correctly oriented, carefully remove excess PBS with a pipette, and allow to air dry for about 5 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Carefully monitor sample drying, allowing just enough time for the solution to evaporate and avoiding over-drying of the samples.</div>
<ol start="23">
<li>Attach the hybridization chambers over the designated regions and fill the wells carefully with 50 μL RNase-free PBS.</li>
</ol>
<h3 id="sec3.5">Hybridization chain reaction (HCR) protocol modifications</h3>
<p>The HCR v3.0 Protocol (<a href="#bib2">Choi et al., 2018</a>) was used for mRNA detection in the presented examples. HCR has been commercialized and additional information is available at <a href="http://www.molecularinstruments.com">www.molecularinstruments.com</a>.</p>
<p>For mRNA detection in cochlear whole mount tissues, we use a modified HCR protocol. The original protocol can be found in the supplemental section of the HCR publication (<a href="#bib2">Choi et al., 2018</a>) and an updated version of the protocol can be found at <a href="https://files.molecularinstruments.com/MI-Protocol-HCRv3-frozentissue-Rev1.pdf">https://files.molecularinstruments.com/MI-Protocol-HCRv3-frozentissue-Rev1.pdf</a>.</p>
<table id="undtbl7">
<thead>
<tr>
<th>Reagents</th>
<th>Composition</th>
</tr>
</thead>
<tbody>
<tr>
<td>HCR probe hybridization buffer</td>
<td>30% formamide5× sodium chloride citrate (SSC)9 mM citric acid (pH 6.0)0.1% Tween 2050 μg/mL heparin1× Denhardt’s solution10% dextran sulfate</td>
</tr>
<tr>
<td>HCR probe wash buffer</td>
<td>30% formamide5× sodium chloride citrate (SSC)9 mM citric acid (pH 6.0)0.1% Tween 2050 μg/mL heparin</td>
</tr>
<tr>
<td>HCR amplification buffer</td>
<td>5× sodium chloride citrate (SSC)0.1% Tween 20</td>
</tr>
<tr>
<td>PBST</td>
<td>1× PBS without Ca<sup>2+/</sup>Mg<sup>2+</sup>0.1% Tween 20</td>
</tr>
<tr>
<td>Proteinase K solution</td>
<td>Proteinase K (800 units/mL) 1:400 in PBST</td>
</tr>
<tr>
<td colspan="2">Details of HCR probes/amplifiers and other reagents are provided in the key resource table.</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> a detailed composition of the reagents used including probe hybridization buffer, probe wash buffer, and amplification buffer are provided by Molecular Instruments and can be found in their protocol (<a href="#bib2">Choi et al., 2018</a>). For convenience and reproducibility, we used the commercially available buffers from Molecular Instruments.</div>
<div class="note">
<span class="note-title">Note:</span> Detection step modifications:</div>
<p>We used 2× the probe concentration that is recommended in the original protocol.</p>
<div class="note">
<span class="note-title">Note:</span> Amplification step modifications:</div>
<p>Snap-cool amplifier hair pins on ice for 5 min, instead of the suggested 30 min at room temperature.</p>
<h4 id="sec3.5.1">Sample preparation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> [20 min]</div>
<ol start="24">
<li>Gently remove the PBS from each well in the hybridization chamber and add 50 μL of diluted Proteinase K solution to each chamber and incubate for 2.5 min at room temperature.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We dilute Proteinase K (800 units/mL) 1:400 in PBST and prepare the solution fresh for each use.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Proteinase K concentration and duration may need to be optimized as each batch is slightly different. A final concentration of 2 units/mL is a good starting point.</div>
<ol start="25">
<li>Gently remove the Proteinase K solution using a pipette and perform three 30 s washes with 50 μL PBST followed by 50 μL PBST for 5 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Manually aspirate the solution with a P200 pipette and be careful not to dislodge the mounted tissue.</div>
<h4 id="sec3.5.2">Detection</h4>
<div class="timing">
<span class="timing-title">Timing:</span> [30 min, followed by incubation for 12–16 h]</div>
<div class="timing">
<span class="timing-title">Timing:</span> [90 min] for step 34</div>
<ol start="26">
<li>Pre-warm a humidified plastic chamber to 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A plastic microcentrifuge tube storage box makes a good humidifying chamber when the individual compartments are filled with a piece of paper towel and water is added (<a href="#fig2">Figure 2</a>D). A plastic inset from a P1000 pipette box is used as a slider holder and avoids excess condensation on the slide (<a href="#fig2">Figure 2</a>E).</div>
<ol start="27">
<li>Replace the PBS in each hybridization chamber with 50 μL of HCR probe hybridization buffer for pre-hybridization.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Pre-heat probe hybridization buffer to 37°C.</div>
<div class="caution">
<span class="caution-title">Caution:</span> Probe hybridization buffer contains formamide, a hazardous material.</div>
<ol start="28">
<li>Pre-hybridize for 10 min inside the humidified chamber at 37°C.</li>
<li>During pre-hybridization, prepare probe hybridization solution by adding 2 μL of 1 μM probe stock to 250 μL of hybridization buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> if multiple probes are used, combine them all into the 250 μL hybridization buffer.</div>
<ol start="30">
<li>Remove the pre-hybridization solution with a P200 pipette.</li>
<li>Immediately add 50 μL of the probe hybridization solution into each of the hybridization chambers.</li>
<li>Place sealing tabs over the chamber port holes, press them tightly onto the holes, and incubate overnight (12–16 h) at 37°C in the humidified chamber.</li>
<li>The next day, remove the probe hybridization solution and replace with HCR probe wash buffer that has been pre-heated to 37°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Be sure to use different pipette tips to avoid cross contamination of probe combinations if performing multiple HCR probe combinations.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to perform buffer exchange steps carefully using a pipette. Vacuum aspiration increases the likelihood of dislodging and aspirating the samples.</div>
<p><b>Caution:</b> Probe wash buffer contains formamide, a hazardous material.</p>
<ol start="34">
<li>Remove excess probes by incubating sample hybridization chambers at 37°C in 50 μL of:
<ol type="a">
<li>75% of probe wash buffer / 25% 5× SSCT for 15 min</li>
<li>50% of probe wash buffer / 50% 5× SSCT for 15 min</li>
<li>25% of probe wash buffer / 75% 5× SSCT for 15 min</li>
<li>100% 5× SSCT for 15 min</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Pre-heat probe wash buffer and 5<b>×</b> SSCT solutions to 37°C.</div>
<ol start="35">
<li>Incubate samples in 5× SSCT for 5 min at room temperature.</li>
</ol>
<h4 id="sec3.5.3">Amplification</h4>
<div class="timing">
<span class="timing-title">Timing:</span> [60 min, followed by incubation for 12–16 h]</div>
<div class="timing">
<span class="timing-title">Timing:</span> [75 min] for step 44</div>
<ol start="36">
<li>Add 50 μL of amplification buffer into each hybridization chamber and incubate in a humidified chamber (<a href="#fig2">Figures 2</a>D and 2E) for 30 min at room temperature.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Equilibrate an aliquot of HCR amplification buffer to room temperature before use.</div>
<div class="note">
<span class="note-title">Note:</span> The final amount of hairpin solution needed depends on the number of experiments/hybridization chambers used; each hybridization chamber requires 50 μL to fill.</div>
<ol start="37">
<li>For each hybridization chamber, aliquot 1 μL of hairpin H1 (3 μM stock) of desired fluorophore into a fresh tube.</li>
<li>For each hybridization chamber, aliquot 1 μL of hairpin H2 (3 μM stock) of desired fluorophore into a fresh tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use at least 4 μL of solution for each tube for heat shock treatment.</div>
<div class="note">
<span class="note-title">Note:</span> We have used up to 3 different hairpins-fluorophore combinations simultaneously.</div>
<ol start="39">
<li>Heat the tubes containing hairpins at 95<sup>◦</sup>C for 90 s and then snap-cool on ice in the dark for 5 min.</li>
<li>Prepare hairpin solution by adding snap-cooled H1 hairpins and snap-cooled H2 hairpins to amplification buffer at room temperature. Add 1 μL of each hairpin per 50 μL.</li>
<li>Remove the amplification buffer from the hybridization chambers.</li>
<li>Add 50 μL of the hairpin solution in amplification buffer into each of the hybridization chambers.</li>
<li>Seal the hybridization chambers with adhesive seal tabs that come with Secure-Seal™ hybridization chamber system and incubate overnight (12–16 h) in a dark humidified chamber at room temperature.</li>
<li>Remove excess hairpins by incubating slide hybridization chamber in 5× SSCT at room temperature twice for 30 min, followed by a single wash for 15 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this stage, if the goal is to perform immunostaining along with the HCR protocol, continue to the <b>Immunostaining following HCR mRNA detection</b> section (below). Otherwise, complete the HCR only protocol with mounting the samples for microscopy (<a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/910-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Wholemount hybridization chain reaction <i>in situ</i> mRNA detection in cochlear hair cells</div>
<p>(A) P28 mouse cochlear apical turn results after hybridization chain reaction protocol. <i>Ocm</i>-mRNA is shown in green and <i>Otof</i>-mRNA is shown in magenta.</p>
<p>The magnified panel (A′) shows expression of <i>Ocm</i>-mRNA in outer hair cells and <i>Otof</i>-mRNA in inner hair cells.</p>
<p>(B) tdTomato fluorescence labels inner hair cells and outer hair cells.</p>
<p>(B′) shows a magnified region.</p>
<p>(C) <i>Ocm</i>-mRNA expression.</p>
<p>(C′) shows a magnified region.</p>
<p>(D) <i>Otof</i>-mRNA expression.</p>
<p>(D′) shows a magnified region.</p>
</figcaption>
</figure>
<h4 id="sec3.5.4">Sample mounting for microscopy</h4>
<div class="timing">
<span class="timing-title">Timing:</span> [2 h]</div>
<ol start="45">
<li>Incubate with DAPI at 1:1000 in PBST for 15 min at room temperature.</li>
<li>Wash with PBST for 15 min at room temperature.</li>
<li>Replace PBST with 50 μL of 50% glycerol for each hybridization chamber.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not allow the samples to dry after removal of the hybridization chambers. Brief equilibration in 50% glycerol lowers the chances of drying out the sample.</div>
<ol start="48">
<li>Remove glycerol solution and then remove the adhesive Secure-Seal™ hybridization system.</li>
<li>Add a drop of FluorSave™ Reagent onto each sample.</li>
<li>Place cover slip and allow the mounting solution to dry in the dark at room temperature for 1 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Gently lower coverslip to avoid bubble formation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid excessive coverslip adjustments, as this will dislodge and possibly damage the sensory epithelium.</div>
<ol start="51">
<li>Add a small amount of clear nail polish along edge of the coverslip to permanently secure the coverslip.</li>
<li>Proceed with confocal microscopy image acquisition and analysis.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Completion of the HCR only protocol results in detection of mRNA expression within the wholemount cochlea apical turn (<a href="#fig3">Figure 3</a>).</div>
<h3 id="sec3.6">Immunostaining following HCR mRNA detection</h3>
<p>Directly after the removal of excess hairpins with 5× SSCT in step 44 of the amplification stage HCR protocol, continue with the following steps.</p>
<h4 id="sec3.6.1">Primary antibody incubation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> [2 h, followed by incubation for 12–16 h]</div>
<ol start="53">
<li>Equilibrate the sample for 5 min in antibody wash buffer.</li>
<li>Replace the antibody wash buffer with blocking buffer and incubate at room temperature for 2 h.</li>
<li>Prepare diluted primary antibody solution by diluting the primary antibody in antibody incubation buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Briefly centrifuge the tube of primary antibody prior to taking aliquot to collect all solution at the bottom of the tube for easier pipetting.</div>
<div class="note">
<span class="note-title">Note:</span> Dilute primary antibody as needed. We used mouse anti-Ctbp2 (1:200) and rabbit anti-Myo7a (1:1000) for the presented examples. Amount of primary antibody used is dependent on the number of hybridization chambers used; each hybridization chamber requires 50 μL to fill.</div>
<ol start="56">
<li>Add 50 μL of diluted primary antibody incubation solution to each hybridization chamber and incubate in a humidified box overnight (12–16 h) at 4°C.</li>
</ol>
<h4 id="sec3.6.2">Secondary antibody incubation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> [2 h 30 min]</div>
<ol start="57">
<li>Remove excess primary antibodies by washing three times with 50 μL of antibody wash buffer, each time for 15 min at room temperature.</li>
<li>Prepare a diluted secondary antibody solution by diluting the secondary antibody in antibody incubation buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Briefly centrifuge the tube of secondary antibody prior to taking aliquot to collect all solution at the bottom of the tube for easier pipetting.</div>
<div class="note">
<span class="note-title">Note:</span> Dilute secondary antibody as needed. We used Donkey anti-mouse Alexa-fluor 647 (1:200) or donkey anti-rabbit Alexa-fluor 647 (1:200) to correspond with species of the primary antibody used for the presented examples.</div>
<ol start="59">
<li>Add 50 μL of secondary antibody solution to each hybridization chamber and incubate at room temperature for 60 min.</li>
<li>Remove excess secondary antibodies by washing the samples three times with 50 μL of antibody wash buffer, each time for 15 min at room temperature.</li>
<li>Add 50 μL of DAPI solution (1:1000 DAPI dilution in PBST) and incubate for 15 min at room temperature.</li>
<li>Replace DAPI solution with 50 μL of PBST and incubate for 15 min.</li>
</ol>
<h4 id="sec3.6.3">Sample mounting for microscopy</h4>
<div class="timing">
<span class="timing-title">Timing:</span> [90 min]</div>
<ol start="63">
<li>Remove PBST and add 50 μL of 50% glycerol to each hybridization chamber.</li>
<li>Remove glycerol solution and adhesive Secure-Seal™ hybridization system.</li>
<li>Add a drop of FluorSave™ Reagent onto each sample.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not allow the samples to dry after removal of the hybridization chambers. Brief equilibration in 50% glycerol lowers the chances of drying out the sample.</div>
<ol start="66">
<li>Place cover slip and allow to dry in the dark for 1 h at room temperature.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid excessive coverslip adjustments, as this will dislodge and possibly damage the sensory epithelium.</div>
<ol start="67">
<li>Add a small amount of clear nail polish along edge of the coverslip to permanently secure the coverslip.</li>
<li>Proceed with confocal microscopy image acquisition and analysis (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/910-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Examples for hybridization chain reaction <i>in situ</i> mRNA detection coupled with immunohistochemistry</div>
<p>(A) Immunolabeling and HCR <i>in situ</i> mRNA detection on whole mount P28 mouse cochlear apical turns without decalcification. Hair cells are labeled with antibodies against Myo7a (white). HCR detection of <i>Efhd2</i>-mRNA (red) and <i>Otof</i>-mRNA (green). A magnified image of the inner hair cells that demonstrate detection of <i>Otof</i> mRNA (green) and immunolabelling for antibodies against Myo7a (white).</p>
<p>(B) Myo15Cre-tdTomato(red) transgenic mouse was used to label hair cells. Presynaptic ribbons in the inner and outer hair cells were stained with antibodies against Ctbp2 (white). Hybridization chain reaction provides individual cell mRNA resolution of the hair cell marker <i>Otof</i>-mRNA (green). A magnified image of the inner hair cells demonstrating mRNA expression of <i>Otof</i> mRNA (green) and immunolabelling for antibodies against Ctbp2 (white). Results are from an apical turn that was not decalcified.</p>
</figcaption>
</figure>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The provided protocol will allow for the simultaneous detection of multiple mRNA transcripts and proteins of interest. Because HCR is a quantitative <i>in situ</i> mRNA detection method, it can be used to quantify mRNA expression levels using confocal microscopy.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>With this protocol, we can reproducibly dissect the apical turn of the P28 mouse cochlea for mRNA and protein detection without decalcification of the tissue. Although we can obtain pieces of the mid and basal turns, we are not able to reliably obtain whole middle and basal turns without decalcifying. Decalcifying the adult cochlea enables dissection of apical, middle, and basal turns. It is possible to detect mRNA using HCR on decalcified cochlear tissue. However, by comparing decalcified with non-decalcified apical turns, we noted a decrease in the amount of mRNA HCR signal in the decalcified apical hair cells. Additional experiments are warranted to see if the addition of RNAlater (<a href="#bib1">Belluoccio et al., 2013</a>) will improve RNA preservation during decalcification. Mounting of cochlear tissue to a microscopy slide can be challenging as not all turns are oriented flat on the slide. However, when the imaged cochleae are visualized with confocal microscopy and the resulting volumes are analyzed with software capable of 3-dimensional manipulations, such as Imaris, it is possible to quantify mRNA expression in the organ of Corti.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Cochlear turns detach from the hybridization chamber bottom during medium change (step 30).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Avoid a vacuum aspiration system. Pipetting should be conducted manually using a 200 μL pipette. The sample needs to dry sufficiently during the attachment step (step 22). We have also noted that prolonged Proteinase K treatment or too high a concentration of the enzyme results in poor attachment.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>No HCR signal from any of the hairpin amplifiers (step 52).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Be sure to use a matching fluorescent H1 and H2 hairpin combination. Do not mix two different fluorescent H1 or H2 hairpins.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>No HCR signal (step 52).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Samples may not have been fixed long enough. We have noted that fixation for a minimum of three hours is needed for HCR to work properly.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Low tdTomato fluorescence (step 52).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Samples may not have been fixed long enough. We have noted that fixation for less than one hour results in a low fluorescent protein signal.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Adhered cochlear turn becomes dislodged during solution changes (step 34).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The detached sample can be salvaged and moved through the HCR/immunostaining protocols. Careful pipetting and visualization with a dissection microscope can help with salvaging the dislodged tissue.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>No HCR signal, even with highly expressed genes (step 52).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Use a different amplifier. For example, we have noticed performance issues with one batch of 647 nm hairpin amplifiers, as the mRNA signal was detectable with 488 nm and 546 nm amplifiers. In these experiments, we instead utilized the 647 nm channel for immunohistochemical detection of a hair cell marker protein.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>No antibody signal (step 68).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>The examples shown in <a href="#fig3">Figures 3</a> and <a href="#fig4">4</a> used antibodies that work well in a variety of applications and conditions. Each new antibody should be thoroughly validated before using it for a combined HCR/immunostaining procedure.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Belluoccio, D., Rowley, L., Little, C.B., and Bateman, J.F. (2013). Maintaining mRNA integrity during decalcification of mineralized tissues. PLoS One <i>8</i>, e58154. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00418-4/sref1">View at publisher</a></p>
<p id="bib2">Choi, H.M.T., Schwarzkopf, M., Fornace, M.E., Acharya, A., Artavanis, G., Stegmaier, J., Cunha, A., and Pierce, N.A. (2018). Third-generation in situ hybridization chain reaction: multiplexed, quantitative, sensitive, versatile, robust. Development <i>145</i>, dev165753. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00418-4/sref2">View at publisher</a></p>
<p id="bib3">Montgomery, S.C. and Cox, B.C. (2016). Whole mount dissection and immunofluorescence of the adult mouse cochlea. J. Vis. Exp. <i>107</i>, 53561. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00418-4/sref3">View at publisher</a></p>
<p id="bib4">Pepermans, E., Michel, V., Goodyear, R., Bonnet, C., Abdi, S., Dupont, T., Gherbi, S., Holder, M., Makrelouf, M., Hardelin, J.P., et al. (2014). The CD2 isoform of protocadherin-15 is an essential component of the tip-link complex in mature auditory hair cells. EMBO Mol. Med. <i>6</i>, 984-992. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00418-4/sref4">View at publisher</a></p>
<p id="bib5">Shao, Y.Y., Wang, L., Hicks, D.G., and Ballock, R.T. (2006). Analysis of gene expression in mineralized skeletal tissues by laser capture microdissection and RT-PCR. Lab. Invest. <i>86</i>, 1089-1095. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00418-4/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Stefan Heller (<a href="mailto:hellers@stanford.edu">hellers@stanford.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate new datasets/code.</p>
<h3>Acknowledgments</h3>
<p>We thank Dr. Amanda Janesick (Stanford University), Dr. Brad Walters (University of Mississippi), and Dr. Taha A Jan (University of California San Francisco) for advice. We also thank the members of the Heller Laboratory for support and comments on the study and the manuscript and Lily Yang (Saratoga High School) for creating the graphical abstract for this manuscript. We acknowledge the Stanford Animal Care Facility and the Otolaryngology Imaging Core. This work was supported by <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> Grants R01DC015201 (S.H.), the <a href="https://doi.org/10.13039/100002046">Hearing Health Foundation</a>’s Hearing Restoration Project (S.H.), and the Stanford Developmental Biology and Genetics Training Grant 2T32GM007790-38 (G.D.).</p>
<h3>Author contributions</h3>
<p>Conceptualization, G.D. and S.H.; methodology and analysis, G.D.; writing – original draft, G.D.; writing – review &amp; editing, G.D. and S.H.; funding acquisition, S.H.</p>
<h3>Declaration of interests</h3>
<p>S.H. is a paid consultant of Pipeline Therapeutics.</p>
</section>
</article>