<article>
<section>
<h1 id="header">Detection and enumeration of Lak megaphages in microbiome samples by endpoint and quantitative PCR</h1>
<p><time datetime="2021-12-24">Published: December 24, 2021</time></p>
<p>Marco A. Crisci,<sup><a href="#aff1">1</a>,<a href="#fn1">6</a>,<a href="#cor3">***</a></sup> Paula M. Corsini,<sup><a href="#aff2">2</a></sup> Nicola Bordin,<sup><a href="#aff1">1</a></sup> Lin-Xing Chen,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a></sup> Jillain F. Banfield,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#cor2">**</a></sup> and Joanne M. Santini<sup><a href="#aff1">1</a>,<a href="#fn2">7</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Structural and Molecular Biology, Division of Biosciences, University College London, London, UK</p>
<p id="aff2"><sup>2</sup>Quadram Institute Bioscience, Norwich Research Park, Norwich, UK</p>
<p id="aff3"><sup>3</sup>Department of Earth and Planetary Science, University of California Berkeley, Berkeley, CA, USA</p>
<p id="aff4"><sup>4</sup>Innovative Genomics Institute, University of California Berkeley, Berkeley, CA, USA</p>
<p id="aff5"><sup>5</sup>The University of Melbourne, Melbourne, VIC, Australia</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:j.santini@ucl.ac.uk">j.santini@ucl.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:jbanfield@berkeley.edu">jbanfield@berkeley.edu</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:marco.crisci.18@ucl.ac.uk">marco.crisci.18@ucl.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101029">10.1016/j.xpro.2021.101029</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Lak megaphages are prevalent across diverse gut microbiomes and may potentially impact animal and human health through lysis of <i>Prevotella</i>. Given their large genome size (up to 660 kbp), Lak megaphages are difficult to culture, and their identification relies on molecular techniques. Here, we present optimized protocols for identifying Lak phages in various microbiome samples, including procedures for DNA extraction, followed by detection and quantification of genes encoding Lak structural proteins using diagnostic endpoint and SYBR green-based quantitative PCR, respectively.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00735-8/sref2">Crisci et al., (2021</a>).</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Optimized DNA extraction procedure for use with various microbiome samples</li>
<li>Amplification of Lak phage structural protein genes for detection and sequencing</li>
<li>Quantification of major capsid protein genes to estimate Lak phage abundance</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1287-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes specific steps for processing and screening fecal, digesta and mucosal samples for Lak phages, but can be adapted for different sample types. We have also implemented the protocol for use with liquid samples (e.g., jejunal and rumen fluid) and mucosal brushes suspended in sodium-magnesium (SM) buffer.</p>
<h3 id="sec1.1">Sample collection and storage</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min–2 h</div>
<ol>
<li>Avoid cross-contamination between samples and environmental contaminants.
<ol type="a">
<li>Change gloves between samples.</li>
<li>Use clean, sterile collection equipment.
<ol type="i">
<li>For mucosal samples, dissection utensils can be sterilized in 100% ethanol.</li>
<li>For fecal/digesta samples, use sterile, disposable equipment.</li>
</ol>
</li>
<li>Collect fecal samples from the centre of the stool where possible.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use appropriate personal protective equipment and take care when collecting biological samples which may contain pathogens.</div>
<ol start="2">
<li>Transport and storage.
<ol type="a">
<li>Samples should be immediately flash-frozen on dry ice or liquid nitrogen on collection.
<ol type="i">
<li>If flash-freezing is not possible, transfer samples at 4°C or at room temperature (20°C–22°C)</li>
</ol>
</li>
<li>Samples should be stored at −80°C until analysis, or analyzed immediately.
<ol type="i">
<li>Thaw samples to room temperature (20°C–22°C) before DNA extraction.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Prepare stocks of oligonucleotide primers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="3">
<li>Order or synthesize primers detailed in <a href="#tbl1">Table 1</a>.
<table id="tbl1" class="dense-table">
<caption>Table 1. Lak phage PCR and qPCR primer pairs</caption>
<thead>
<tr>
<th>Use</th>
<th>Application</th>
<th>Primer pair</th>
<th>Sequence</th>
<th>Target</th>
<th>Product size (bp)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">PCR</td>
<td rowspan="2">General Lak phage diagnostics/ amplicon sequencing</td>
<td>LakMC581-F</td>
<td>5′-GGAGTCATACGAACACCAGAAGT-3′</td>
<td rowspan="2">Major capsid protein</td>
<td rowspan="2">473</td>
</tr>
<tr>
<td>LakMC1053-R</td>
<td>5′-GTAGTTCTTACACTTCACGCTCCTC-3′</td>
</tr>
<tr>
<td rowspan="2">PCR</td>
<td rowspan="2">General Lak phage diagnostics/amplicon sequencing</td>
<td>LakPV767-F</td>
<td>5′-CATGGTCAACAGGTATGTATGG-3′</td>
<td rowspan="2">Portal vertex protein</td>
<td rowspan="2">495</td>
</tr>
<tr>
<td>LakPV1261-R</td>
<td>5′-CCTCTCGTGTTATACTTGCATCA-3′</td>
</tr>
<tr>
<td rowspan="2">PCR</td>
<td rowspan="2">General Lak phage diagnostics/amplicon sequencing</td>
<td>LakTS3039-F</td>
<td>5′-CTTCCATCTAAGAGACAGTTTGA-3′</td>
<td rowspan="2">Tail sheath monomer</td>
<td rowspan="2">689</td>
</tr>
<tr>
<td>LakTS3781-R</td>
<td>5′-CTTCCATCTAAGAGACAGTTTGA-3′</td>
</tr>
<tr>
<td rowspan="2">PCR</td>
<td rowspan="2">660 kbp Lak phage diagnostics/amplicon sequencing</td>
<td>LakHMC185-F</td>
<td>5′-CTCTTAACGAGGATGCACAGT-3′</td>
<td rowspan="2">Major capsid protein</td>
<td rowspan="2">799</td>
</tr>
<tr>
<td>LakHMC984-R</td>
<td>5′-ACCTGCACCGGTATAACCAA-3′</td>
</tr>
<tr>
<td rowspan="2">qPCR</td>
<td rowspan="2">General Lak phage quantification</td>
<td>LakMCP683-F</td>
<td>5′-CAACCAAGAGCGAACAAACGAG-3′</td>
<td rowspan="2">Major capsid protein</td>
<td rowspan="2">121</td>
</tr>
<tr>
<td>LakMC803-R</td>
<td>5′-TAACAGACCTTCAGAAACAGTGGG-3′</td>
</tr>
<tr>
<td rowspan="2">qPCR</td>
<td rowspan="2">660 kbp Lak phage quantification</td>
<td>LakHMC224-F</td>
<td>5'-GGTGGCGTTTACAGCCCATA-3′</td>
<td rowspan="2">Major capsid protein</td>
<td rowspan="2">197</td>
</tr>
<tr>
<td>LakHMC421-R</td>
<td>5'-CAACGGTCTTAGCAGCAACCT-3′</td>
</tr>
</tbody>
</table>
<p class="table-legend">Primers detailed in this table were designed by <a href="#bib2">Crisci et al. (2021</a>).</p>
</li>
<li>Prepare stocks. For lyophilized primers:
<ol type="a">
<li>Centrifuge for 30 s at 13,000 × <i>g</i>.</li>
<li>Resuspend in sterile Tris-EDTA (TE) buffer to stock concentration (e.g., 100 μM).</li>
<li>Vortex for 10 min at maximum speed.</li>
<li>Centrifuge briefly.</li>
</ol>
</li>
<li>Dilute primers to working concentration of 7 μM with nuclease-free water.
<ol type="a">
<li>Centrifuge briefly.</li>
<li>Store both stock and working solutions at −20°C.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Microbiome samples (e.g., feces)</td>
<td>Acquired by protocol users</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Trizma® Base</td>
<td>Sigma-Aldrich</td>
<td>Cat. No. T6066</td>
</tr>
<tr>
<td>Hydrochloric Acid (HCl; 37%)</td>
<td>VWR</td>
<td>Cat. No. 20252.290</td>
</tr>
<tr>
<td>Ethylenediaminetetraacetic acid disodium salt dihydrate (EDTA-Na<sub>2</sub>; 372.24 g/mol)</td>
<td>Sigma-Aldrich</td>
<td>Cat. No. E5134</td>
</tr>
<tr>
<td>Sodium hydroxide pellets (NaOH; 98.5%–100.5%)</td>
<td>VWR</td>
<td>Cat. No. 28244.262</td>
</tr>
<tr>
<td>Deoxynucleotide set (including 100 mM each: TTP in H<sub>2</sub>O, dCTP (pH 7), dGTP (pH 7) dATP (pH 7)).</td>
<td>Sigma-Aldrich</td>
<td>Cat. No. DNTP100A</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>QIAquick PCR purification kit</td>
<td>Qiagen</td>
<td>Cat. No. 28106</td>
</tr>
<tr>
<td>QIAquick gel extraction kit</td>
<td>Qiagen</td>
<td>Cat. No. 28706</td>
</tr>
<tr>
<td>QIAamp PowerFecal Pro DNA kit</td>
<td>Qiagen</td>
<td>Cat. No. 51804</td>
</tr>
<tr>
<td>QuantiNova SYBR green PCR kit (including master mix, nuclease-free water, ROX passive reference dye and template dilution buffer)</td>
<td>Qiagen</td>
<td>Cat. No. 208054</td>
</tr>
<tr>
<td>BIOTAQ<sup>TM</sup> DNA polymerase kit (including polymerase, NH<sub>4</sub> reaction buffer, and MgCl<sub>2</sub>)</td>
<td>Bioline</td>
<td>Cat. No. BIO-21060</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Results using this method</td>
<td><a href="#bib2">Crisci et al. (2021)</a></td>
<td><a href="https://www.sciencedirect.com/science/article/pii/S2589004221008439">https://www.sciencedirect.com/science/article/pii/S2589004221008439</a></td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>See <a href="#tbl1">Table 1</a> for Lak qPCR and PCR primers</td>
<td><a href="#bib2">Crisci et al. (2021)</a></td>
<td><a href="https://www.sciencedirect.com/science/article/pii/S2589004221008439">https://www.sciencedirect.com/science/article/pii/S2589004221008439</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>JMP® Pro 14.1</td>
<td>SAS Institute Inc., NC, USA, 2019</td>
<td><a href="https://www.jmp.com/en_gb/home.html">https://www.jmp.com/en_gb/home.html</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>OrionTM 3-Star benchtop pH meter</td>
<td>Thermo Scientific<sup>TM</sup>
</td>
<td>Cat. No. 13-644-928</td>
</tr>
<tr>
<td>NanoDrop<sup>TM</sup> 2000 spectrophotometer</td>
<td>Thermo Scientific<sup>TM</sup>
</td>
<td>Cat. No. ND2000LAPTOP</td>
</tr>
<tr>
<td>Piko PCR plate, 96-well</td>
<td>Thermo Scientific<sup>TM</sup>
</td>
<td>Cat. No. SPL0961</td>
</tr>
<tr>
<td>Mastercylcler® nexus gradientPCR thermal cycler</td>
<td>Eppendorf</td>
<td>Cat No. 6331000041</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Tris-EDTA (TE) buffer</h3>
<p>Preparation of TE buffer is recommended for storage of oligonucleotide primers. This can be prepared as follows:</p>
<ul>
<li>Prepare Tris-HCl (1 M, pH 8):
<ul>
<li>Dissolve 121.14 g Trizma® base (121.14 g/mol) in 800 mL deionized water.</li>
<li>Using a pH meter, adjust to pH 8 by adding HCl (37%) as required.</li>
<li>Adjust volume to 1000 mL with deionized water.</li>
</ul>
</li>
<li>Prepare EDTA (0.5 M, pH 8)
<ul>
<li>Dissolve 186.12 g EDTA-Na<sub>2</sub> (372.24 g/mol) in 800 mL deionized water.</li>
<li>Using a pH meter, adjust to pH 8 by adding NaOH pellets (<b>≥</b> 98.5%) as required.</li>
<li>Adjust volume to 1000 mL with deionized water.</li>
</ul>
</li>
<li>Then, prepare the TE buffer (pH 8):</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume to add for 100 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Deionized water</td>
<td>n/a</td>
<td>98.8 mL</td>
</tr>
<tr>
<td>Tris-HCl (1 M, pH 8)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M, pH 8)</td>
<td>1 mM</td>
<td>0.2 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterilize by autoclaving (110°C for 15 min), or filtration (0.22 μm membrane filter), and store at room temperature (20°C–22°C).</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Purchase pre-prepared TE buffer (e.g., Invitrogen<sup>TM</sup>, Cat. no. 12090015)</div>
<h3 id="sec2.2">Deoxynucleotide triphosphates (dNTPs)</h3>
<p>Preparation of dNTPs is essential for the proposed PCR assay. Prepare solution as follows:</p>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume to add for 100 μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>60 μL</td>
</tr>
<tr>
<td>dATP (100 mM, pH 7)</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>TTP (100 mM, in H<sub>2</sub>O)</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>dCTP-Na<sub>2</sub> (100 mM, pH 7)</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>dGTP-Na<sub>3</sub> (100 mM, pH 7)</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Purchase pre-prepared dNTPs (e.g., Thermo Scientific<sup>TM</sup>, Cat. no. R0181)</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">DNA extraction from microbiome samples</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Validated procedure for DNA extraction using PowerFecal Pro DNA kit (Qiagen, Hilden, Germany) on a range of microbiome samples, including feces/solid digesta, mucosal tissues and fluid digesta. Follow manufacturer instructions with adaptations as below:</p>
<ol>
<li>Lysis.
<ol type="a">
<li>Mix solid samples in collection tubes with a sterile spatula.</li>
<li>Place approximately 0.25 g of solid sample in a bead tube with 800 μL lysis buffer.
<ol type="i">
<li>For liquid samples, use 200 μL of sample with 600 μL lysis buffer.</li>
</ol>
</li>
<li>Heat tubes at 65°C for 10 min, using a water bath or heat block.
<div class="note">
<span class="note-title">Note:</span> Heating will aid disassembly of the phage capsid and the release of genetic material.</div>
</li>
<li>Using a vortex adapter, vortex tubes for 10 min at maximum speed.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Inhibitor removal and washing (as specified in manufacturer instructions)</li>
</ol>
<ol start="3">
<li>Elution.
<ol type="a">
<li>Use 60 μL of elution buffer and incubate spin column at room temperature (20°C–22°C) for 10 min.
<ol type="i">
<li>. Centrifuge at 13,000 ×<i>g</i> for 1 min.</li>
<li>. Once eluted, pass the same 60 μL eluate back through the same spin column (without incubation) and repeat centrifugation.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Measure DNA concentration by UV spectrophotometry.</li>
<li>Store DNA at −20°C until analysis.</li>
</ol>
<p><b>RECOMMENDED:</b> Use of filter tips when processing microbiome samples.</p>
<h3 id="sec3.2">Amplification of Lak phage structural protein-coding genes by PCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h 30 min</div>
<p>Amplification of Lak phage major capsid protein (MCP), portal vertex protein (PVP) and tail sheath monomer (TSM) gene fragments can be used for detection of Lak phages in biological samples. These signature genes were selected from metagenomic datasets, due to their specificity and conservation across Lak phage genomes. The general screening primers are specific to Lak phages found in humans, primates, pigs and most other animals. An additional set was designed to detect the divergent ∼660 kbp variant discovered in a racehorse microbiome (<a href="#bib2">Crisci et al., 2021</a>). Primer sequences are reported in <a href="#tbl1">Table 1</a>.</p>
<ol start="6">
<li>On ice, prepare master mixes for each assay containing the following:</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration per 25 μL reaction</th>
<th>Volume to add per 25 μL reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>dNTPs</td>
<td>20 μM (of each dNTP)</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Forward primer (variable)</td>
<td>0.14 μM</td>
<td>0.5 μL (of 7 μM primers)</td>
</tr>
<tr>
<td>Reverse primer (variable)</td>
<td>0.14 μM</td>
<td>0.5 μL (of 7 μM primers)</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>3 mM</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>NH<sub>4</sub> reaction buffer</td>
<td>1×</td>
<td>2.5 μL</td>
</tr>
</tbody>
</table>
<ol start="7">
<li>Calculate required volume of DNA to achieve 150 ng/μL in 25 μL.</li>
<li>Calculate required volume of nuclease-free water (e.g., 19.25 μL – DNA volume).</li>
<li>On ice, prepare 25 μL PCR reaction:
<ol type="a">
<li>Add water and DNA.</li>
<li>Add 5.5 μL pre-prepared master mix.</li>
<li>Add 0.25 μL Taq DNA Polymerase (1.25 U; BIOTAQ<sup>TM</sup> DNA Polymerase, Bioline, London, UK).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Although this protocol has been validated with BIOTAQ<sup>TM</sup> DNA Polymerase, users may adapt for other DNA polymerases or master mixes.</div>
<ol start="10">
<li>Centrifuge briefly, and place in a thermocycler programmed under the following conditions:</li>
</ol>
<table id="undtbl4">
<thead>
<tr>
<th colspan="5">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Target</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>MCP, PVP, TSM and MCP_660</td>
<td>96°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>MCP, PVP, TSM and MCP_660</td>
<td>96°C</td>
<td>10 s</td>
<td rowspan="8">40 cycles</td>
</tr>
<tr>
<td rowspan="4">Annealing</td>
<td>MCP</td>
<td>61°C</td>
<td>30 s</td>
</tr>
<tr>
<td>PVP</td>
<td>58°C</td>
<td>30 s</td>
</tr>
<tr>
<td>TSM</td>
<td>57°C</td>
<td>30 s</td>
</tr>
<tr>
<td>MCP_660</td>
<td>57°C</td>
<td>30 s</td>
</tr>
<tr>
<td rowspan="3">Extension</td>
<td>MCP and PVP</td>
<td>72°C</td>
<td>15 s</td>
</tr>
<tr>
<td>TSM</td>
<td>72°C</td>
<td>20 s</td>
</tr>
<tr>
<td>MCP_660</td>
<td>72°C</td>
<td>25 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>MCP, PVP, TSM and MCP_660</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>MCP, PVP, TSM and MCP_660</td>
<td>4°C</td>
<td colspan="2">thereafter</td>
</tr>
</tbody>
</table>
<h3 id="sec3.3">Endpoint analysis of PCR products</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="11">
<li>Presence of Lak phage in microbiome DNA can be visualized using agarose gel electrophoresis.</li>
<li>Purified PCR products can be sequenced for confirmation and comparison. We recommend using either QIAquick PCR purification or gel extraction kits (Qiagen, Hilden, Germany) prior to sequencing. To maximise yield, follow manufacturer instructions with adaptations as below:
<ol type="a">
<li>Use 30 μL of elution buffer and incubate spin column at room temperature for 10 min.</li>
<li>Centrifuge at 13,000 ×<i>g</i> for 1 min.</li>
<li>Once eluted, pass the same 30 μL eluate back through the same spin column (without incubation) and repeat centrifugation.</li>
</ol>
</li>
<li>Measure DNA concentration by UV spectrophotometry.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> PCR purification of Lak MCP gene amplicons is a prerequisite in generating standards for the following qPCR assay.</div>
<h3 id="sec3.4">qPCR quantification of Lak phage major capsid protein-coding gene</h3>
<p>This protocol has been validated for use with the QuantiNova SYBR Green PCR kit (Qiagen, Hilden, Germany), and PikoReal<sup>TM</sup> real-time PCR system (Thermo Fisher Scientific, MA, USA) (<a href="#bib2">Crisci et al., 2021</a>). However, the following procedure can be adapted for any SYBR green-based qPCR kit and compatible instrument. As Lak phage genomes are generally AT-rich, the MCP gene was selected due to its conservation and higher GC content (∼41%) than other genes, which provides better thermal stability in qPCR reactions.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The use of PCR product as qPCR standard is expected to yield an efficiency over 100% (<a href="#bib2">Crisci et al., 2021</a>). Efficiencies between 90 and 110% are considered acceptable for quantification. It is advisable to calculate amplification efficiencies using standards and primers prior to qPCR quantification.</div>
<p><b>e.g.</b> (−1+10<sup>−1/slope</sup>) × 100</p>
<h3 id="sec3.5">Prepare standards for qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Lak phage MCP gene amplicons (MCP: LakMC581-F/LakMC1053-R and MCP_660: LakHMC185-F/LakHMC984-R) are used as qPCR standards. The qPCR target regions are within the regions amplified by the endpoint PCR primers (<a href="#tbl1">Table 1</a>). Once a PCR product is obtained, prepare qPCR standards as follows.</p>
<ol start="14">
<li>Prepare 9 × 1:10 serial dilutions of PCR amplicon (starting at 5 ng).
<ol type="a">
<li>In the first tube, adjust Lak major capsid PCR product to 5 ng/μL using nuclease-free water or 1× template dilution buffer.</li>
<li>Then, add 2 μL from the first tube to 18 μL nuclease-free water or 1× template dilution buffer, mix by pipetting up and down, and discard the tip.
<ol type="i">
<li>Repeat for remaining serial dilutions. For example:</li>
</ol>
</li>
</ol>
</li>
</ol>
<table id="undtbl5">
<thead>
<tr>
<th>Dilution order</th>
<th>DNA concentration (ng/μL)</th>
<th>Water volume (μL)</th>
<th>DNA volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>5</td>
<td>Variable</td>
<td>Variable</td>
</tr>
<tr>
<td>2</td>
<td>0.5</td>
<td rowspan="8">18</td>
<td rowspan="8">2</td>
</tr>
<tr>
<td>3</td>
<td>0.05</td>
</tr>
<tr>
<td>4</td>
<td>0.005</td>
</tr>
<tr>
<td>5</td>
<td>0.0005</td>
</tr>
<tr>
<td>6</td>
<td>0.00005</td>
</tr>
<tr>
<td>7</td>
<td>0.000005</td>
</tr>
<tr>
<td>8</td>
<td>0.0000005</td>
</tr>
<tr>
<td>9</td>
<td>0.00000005</td>
</tr>
</tbody>
</table>
<h3 id="sec3.6">Prepare DNA dilutions for qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="15">
<li>Adjust sample DNA to 10 ng/μL using nuclease-free water or 1× template dilution buffer (so that 1 μL provides 10 ng DNA per 10 μL qPCR reaction).</li>
</ol>
<h3 id="sec3.7">qPCR amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="16">
<li>Prepare master mixes for each assay containing the following:</li>
</ol>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration per 10 μL reaction</th>
<th>Volume to add per 10 μL reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>1 μL</td>
</tr>
<tr>
<td>Forward primer (variable)</td>
<td>0.7 μM</td>
<td>1 μL (of 7 μM primers)</td>
</tr>
<tr>
<td>Reverse primer (variable)</td>
<td>0.7 μM</td>
<td>1 μL (of 7 μM primers)</td>
</tr>
<tr>
<td>ROX passive reference dye</td>
<td>1×</td>
<td>1 μL∗</td>
</tr>
<tr>
<td>QuantiNova SYBR Green master mix</td>
<td>1×</td>
<td>5 μL</td>
</tr>
</tbody>
</table>
<div class="∗optional">
<span class="∗optional-title">∗optional:</span> ROX reference dye is used for normalization of fluorescence signals. ROX can be replaced with the same volume of nuclease-free water if not required.</div>
<ol start="17">
<li>Load 9 μL master mix into 96-well plates.</li>
<li>Load 1 μL diluted DNA:
<ol type="a">
<li>Standards (pre-prepared serial dilutions)</li>
<li>Sample DNA (10 ng per reaction)</li>
</ol>
</li>
<li>Seal plates with optical heat seals.</li>
<li>Program thermocycler according to kit instructions for quantification with SYBR green using the standard curve method.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A minimum of 3 technical replicates is recommended, along with use of no template controls (NTC; water in place of DNA). Melt curve analysis can be performed to identify non-specific binding.</div>
<ol start="21">
<li>Extrapolate major capsid gene quantities (ng) from standard curves (quantification cycle (Cq) vs. Log DNA dilution).
<ol type="a">
<li>Extrapolated quantities (ng) can be analysed as is, or converted to copy numbers.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The MCP gene is single copy. Thus, gene copy number = genome copies.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">Expected PCR outcomes: DNA extraction</h3>
<p>Across 169 samples, the present method for total DNA extraction yielded an average concentration of 367.1 ng μL<sup>−1</sup>. For mucosal samples, yields ∼1000 ng μL<sup>−1</sup> were common, which are likely attributed by host DNA. For fecal samples and fresh digesta, ∼70 ng μL<sup>−1</sup> to ∼400 ng μL<sup>−1</sup> DNA was obtained.</p>
<h3 id="sec4.2">Expected PCR outcomes: endpoint PCR results</h3>
<p>The present endpoint PCR method using phage structural proteins is effective for detection of Lak phages in a variety of microbiome samples. <a href="#fig1">Figure 1</a> shows positive PCR results for the Lak MCP assay, derived from pig digesta samples.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1287-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Representative Lak phage major capsid PCR products visualised by agarose gel electrophoresis</div>
<p><b>GR</b> = 1 kb+ GeneRuler; <b>+</b> = Swine positive control; <b>Lak MCP PCR amplicons from pig gastrointestinal samples = JE</b>=Jejunum; <b>IL</b> = Ileum; <b>CE</b> = Caecum; <b>PS</b> = Proximal spiral; <b>DS</b> = Distal spiral. See <a href="#tbl1">Table 1</a> for primer details. Data relating to this figure has been published previously by <a href="#bib2">Crisci et al. (2021</a>).</p>
</figcaption>
</figure>
<h3 id="sec4.3">Expected qPCR outcomes</h3>
<p><a href="#fig2">Figure 2</a> shows representative standard curves and efficiencies obtained using the proposed qPCR assays for Lak phages (based on most known genomes; <a href="#fig2">Figure 2</a>A), and for the ∼660 kbp variant (<a href="#fig2">Figure 2</a>B).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1287-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Standard curves generated using Lak phage qPCR primers</div>
<p>Cq = Cycle of quantification; R<sup>2</sup> = Coefficient of determination; RMSE = Root mean square error. Black dots represent technical replicates for each serial dilution of Lak major capsid amplicon (MCP: LakMC581-F/LakMC1053-R; MCP_660: LakHMC185-F/ LakHMC984-R): 1:10 starting at 5 ng. Primer efficiencies (E) calculated from slopes ((−1+10<sup>−1/slope</sup>) × 100).</p>
<p>(A) Lak MCP amplicons used as standards with LakMC683-F/LakMC803-R primers, E = 102.8%.</p>
<p>(B) Lak MCP_660 variant amplicons used as standards with LakHMC224-F/LakHMC421-R primers, E = 110.1%. See <a href="#tbl1">Table 1</a> for primer details. Data relating to this figure has been published previously by <a href="#bib2">Crisci et al. (2021</a>).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Acceptable qPCR primer efficiencies for reliable quantification should be between 90 and 100%. If efficiencies outside of this range are encountered, see <a href="#troubleshooting">troubleshooting</a>.</div>
<p><a href="#fig3">Figure 3</a> shows representative melt curves with no non-specific binding using the proposed qPCR assays for Lak phages (based on most known genomes; <a href="#fig3">Figure 3</a>A), and for the ∼660 kbp variant (<a href="#fig3">Figure 3</a>B).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1287-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative melt curves for Lak qPCR primer pairs</div>
<p>RFU = relative fluorescence units using SYBR green. Single peak generated from 3 technical replicates indicates no nonspecific binding or secondary structures.</p>
<p>(A) Representative result shown for Lak MCP primers (LakMC683-F/LakMC803-R).</p>
<p>(B) Representative result shown for Lak MCP_660 primers (LakHMC224-F/LakHMC421-R). See <a href="#tbl1">Table 1</a> for primer details. A version of this figure has been published previously by <a href="#bib2">Crisci et al. (2021</a>).</p>
</figcaption>
</figure>
<p>Quantification results obtained using the present qPCR assays for Lak phages are published in <a href="#bib2">Crisci et al. (2021)</a>. For pig gut luminal contents and mucosal tissues (using Lak_MCP) and horse faeces (using Lak MCP_660), Log copies ranged from ∼2 to ∼7.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Depending on the users’ requirements, statistical analysis of qPCR data will vary. However, for comparison of Lak phage abundance between different treatments or sample groups, we briefly describe steps employed by <a href="#bib2">Crisci et al. (2021)</a>. The proposed method for statistical analysis is compatible with JMP® Pro 14.1 (SAS Institute Inc., NC, USA, 2019), but can be adapted to virtually any statistical package.</p>
<ol>
<li>Prepare data for analysis.
<ol type="a">
<li>Collate data into a table, including headings for all known variables.</li>
<li>Calculate averages for technical replicates, and log-transform.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Log transformation simplifies the analysis of data that does not usually conform to normality.</div>
<ol start="2">
<li>Analyse distribution.
<ol type="a">
<li>Perform distribution analysis by group (e.g., treatment, or body site).</li>
<li>Identify outliers and remove as required.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Lak phage abundance may vary beyond the normal distribution for a given population. JMP® identifies outliers as 1.5∗IQR (interquartile range).</div>
<ol start="3">
<li>Design appropriate statistical model.
<ol type="a">
<li>If feasible, use least squared mean comparisons or ANOVA.</li>
<li>Identify covariates so that they can be accounted for in the statistical model.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Covariates may be factors such as sex, age, and location.</div>
<ol start="4">
<li>Means separation.
<ol type="a">
<li>When comparing &gt;2 means, we recommend using Tukey’s HSD test. When comparing 2 means, a t test or similar can be used.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>It should be considered that Lak phage genomes which are genetically distinct to those described by <a href="#bib2">Crisci et al. (2021</a>) and <a href="#bib3">Devoto et al. (2019)</a> may not be detected using the proposed methods. Given that qPCR assays yield efficiencies over 100% (due to use of PCR amplicons as standards), quantification results should be considered as estimations, useful for comparison purposes, rather than definitive quantities.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>DNA concentration too low for PCR or sequencing (step 4).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>During DNA extraction, prepare multiple lysates for each sample, but pass all through the same spin column for washing and elution into one tube.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>False negatives (step 11).</p>
<h3 id="sec7.4">Potential solution 1</h3>
<p>To confirm whether negative PCR results are true or due to the absence of Lak phages, it is recommended to use a positive control alongside unknown samples. Once Lak phages are confirmed within sample DNA, the sample should be used thereon as a positive control.</p>
<h3 id="sec7.5">Potential solution 2</h3>
<p>During DNA spectrophotometry, check 260/280 and 260/230 ratios, as this can indicate contamination which may interfere with PCR. Furthermore, check that microbiome DNA is not degraded prior to use by electrophoresing 5 uL on a 1% agarose gel with 6× loading dye. If DNA is contaminated, it can be diluted or re-extracted.</p>
<h3 id="sec7.6">Problem 3</h3>
<p>False positives (step 11).</p>
<p>Thus far, the present PCR and qPCR primers have not resulted in nonspecific binding. However, should nonspecific binding or secondary structures arise (e.g., PCR amplicon sizes not as expected), the following potential solutions can be implemented.</p>
<h3 id="sec7.7">Potential solution 1</h3>
<p>It is advisable to sequence PCR products and perform a BLASTN (<a href="#bib1">Altschul et al., 1990</a>) search for confirmation. Published Lak genomes are deposited in NCBI (<a href="#bib2">Crisci et al., 2021</a>; <a href="#bib3">Devoto et al., 2019</a>).</p>
<h3 id="sec7.8">Potential solution 2</h3>
<p>New primers can be designed using published Lak phage genes. Before synthesis, check primer sets for secondary structures (e.g., OligoAnalyzer; Integrated DNA Technologies Inc., Iowa, USA), and validate the assay using methods described in this protocol.</p>
<h3 id="sec7.9">Problem 4</h3>
<p>Insufficient PCR product yield for sequencing (step 12).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Repeat the PCR, using a larger reaction volume until a sufficient product yield is obtained.</p>
<h3 id="sec7.11">Problem 5</h3>
<p>Lak phage abundance below that reliably detected using the standard curve (step 15).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>If Cq values for sample DNA are outside of those plotted in the standard curve (e.g., lower than highest standard dilution), the starting concentration of DNA may be increased.</p>
<h3 id="sec7.13">Problem 6</h3>
<p>Inhibition of qPCR (step 15).</p>
<p>If Lak phages are detected in sample DNA, but are not quantifiable by qPCR, impurities in the DNA preparation may be accountable.</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Use further dilutions of sample DNA to reduce inhibition (e.g., reduce template DNA concentration to less than 10 ng).</p>
<h3 id="sec7.15">Problem 7</h3>
<p>qPCR primer efficiencies outside of an acceptable range (step 16).</p>
<p>For efficiencies below 90% or above 110%, the following potential solutions can be implemented.</p>
<h3 id="sec7.16">Potential solution 1</h3>
<p>Check primer concentrations are correct, prepare fresh master mix, and repeat the experiment.</p>
<h3 id="sec7.17">Potential solution 2</h3>
<p>Use a larger qPCR reaction volume (e.g., greater than 10 uL), as this will decrease the propensity for pipetting errors.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Altschul, S.F., Gish, W., Miller, W., Myers, E.W., and Lipman, D.J. (1990). Basic local alignment search tool. J. Mol. Biol. <i>215</i>, 403-410. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00735-8/sref1">View at publisher</a></p>
<p id="bib2">Crisci, M.A., Chen, L.-X., Devoto, A.E., Borges, A.L., Bordin, N., Sachdeva, R., Tett, A., Sharrar, A.M., Segata, N., Debenedetti, F., et al. (2021). Closely related Lak megaphages replicate in the microbiomes of diverse animals. iScience <i>24</i>, 102875. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00735-8/sref2">View at publisher</a></p>
<p id="bib3">Devoto, A.E., Santini, J.M., Olm, M.R., Anantharaman, K., Munk, P., Tung, J., Archie, E.A., Turnbaugh, P.J., Seed, K.D., Blekhman, R., et al. (2019). Megaphages infect Prevotella and variants are widespread in gut microbiomes. Nat. Microbiol. <i>4</i>, 693-700. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00735-8/sref3">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Joanne M. Santini (<a href="mailto:j.santini@ucl.ac.uk">j.santini@ucl.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate any new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The original/source data for figures in the paper are available (<a href="#bib2">Crisci et al., 2021</a>).</p>
<h3>Acknowledgments</h3>
<p>M.A.C. was funded by the <a href="https://doi.org/10.13039/501100000268">BBSRC</a> (BB/M009513/1) through the London Interdisciplinary Doctoral Program. N.B. acknowledges funding by the <a href="https://doi.org/10.13039/501100000268">BBSRC</a> (BB/R009597/1). Metagenomic sequencing that facilitated primer design for this protocol was funded by the <a href="https://doi.org/10.13039/100014220">Innovative Genomics Institute</a>, <a href="https://doi.org/10.13039/100006978">UC Berkeley</a>, the Chan Zuckerberg Biohub, and the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (RAI092531A) to J.F.B. For assisting with pig microbiome sampling, the authors would like to thank Mick Bailey, Francesco Debenedetti, and Rachel Burt (University of Bristol). For assisting with racehorse sampling, the authors also thank Roger Ingram and Sharron Ingram (Wendover Stables, Epsom).</p>
<h3>Author contributions</h3>
<p>This protocol was designed and optimised by M.A.C. with input from J.M.S. and P.M.C. N.B. advised and assisted with Lak signature gene acquisition from metagenomic datasets. L.-X.C. and J.F.B. curated and provided sequences for the 660 kbp Lak variant from a horse gut metagenome. P.M.C. advised during qPCR optimization and analysis. M.A.C. wrote the protocol with input from J.M.S., P.M.C., N.B., L.-X.C., and J.F.B.</p>
<h3>Declaration of interests</h3>
<p>The authors of this paper declare no competing interests.</p>
</section>
</article>