<article>
<section>
<h1 id="header">Application of dual <i>Nr4a1</i>-GFP <i>Nr4a3</i>-Tocky reporter mice to study T cell receptor signaling by flow cytometry</h1>
<p><time datetime="2021-01-22">Published: January 22, 2021</time></p>
<p>Emma K. Jennings,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> David A.J. Lecky,<sup><a href="#aff1">1</a></sup> Masahiro Ono,<sup><a href="#aff2">2</a></sup> and David Bending<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Immunology and Immunotherapy, College of Medical and Dental Sciences, University of Birmingham, Birmingham B15 2TT, UK</p>
<p id="aff2"><sup>2</sup>Department of Life Sciences, Imperial College London, London SW7 2AZ, UK</p>
<p id="fn1"><sup>3</sup>Present address: Leeds Institute for Medical Research, Faculty of Medicine and Health, University of Leeds, St James’ University Hospital, Leeds LS9 7TF, UK</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:d.a.bending@bham.ac.uk">d.a.bending@bham.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100284">10.1016/j.xpro.2020.100284</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol uses <i>Nr4a1</i>-GFP <i>Nr4a3</i>-Tocky mice to study T cell receptor (TCR) signaling using flow cytometry. It identifies the optimal mouse transgenic status and fluorochromes compatible with the dual reporter. This protocol has applications in TCR signaling, and we outline how to obtain high-quality datasets. It is not compatible with cellular fixation, and cells should be analyzed immediately after staining.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib3">Jennings et al., 2020</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol uses <i>Nr4a1</i>-GFP and <i>Nr4a3</i>-Tocky mice to study TCR signal duration</li>
<li>Optimized for use with homozygous <i>Nr4a3</i>-Tocky mice</li>
<li>Guidance on fluorochrome choices compatible with GFP and Timer-Blue and Timer-Red</li>
<li>Use of an open “dump” channel on flow cytometer can improve signal to noise ratio</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/420-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Determine zygosity of <i>Nr4a3</i>-Tocky mice</h3>
<p><i>Nr4a1</i>-GFP (<a href="#bib4">Moran et al., 2011</a>) and <i>Nr4a3</i>-Tocky (<a href="#bib2">Bending et al., 2018b</a>) are Bacterial Artificial Chromosome (BAC) transgenic mice. The protocol is optimized for use of “homozygous” <i>Nr4a3</i>-Tocky BAC and “heterozygous” <i>Nr4a1</i>-GFP mice.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Use of heterozygous <i>Nr4a3</i>-Tocky in combination with <i>Nr4a1</i>-GFP leads to reduced expression of Nr4a3-Timer (<a href="#bib3">Jennings et al., 2020</a>). Therefore, it is strongly recommended that <i>Nr4a3</i>-Tocky homozygous mice are used when these are mated to <i>Nr4a1</i>-GFP mice.</div>
<div class="note">
<span class="note-title">Note:</span> The following primers are made up to 100 μM final concentration as master stocks. From these 1 in 10 dilutions of working 10 μM stocks are made.</div>
<table id="undtbl1">
<thead>
<tr>
<th>Primer</th>
<th>Sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<i>Timer BAC</i> F</td>
<td>CGCGGAACTAACTTCCCCTC</td>
</tr>
<tr>
<td>
<i>Timer BAC</i> R</td>
<td>GTCTTGACCTCAGCGTCGTA</td>
</tr>
<tr>
<td>
<i>Il2ra</i> F</td>
<td>CAGGAGTTTCCTAAGCAACG</td>
</tr>
<tr>
<td>
<i>Il2ra</i> R</td>
<td>CTGTGTCTGTATGACCCACC</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> <i>Timer BAC</i> F+R produce a PCR product of approximately 180 bp; <i>Il2ra</i> F+R product is approximately 200 bp.</div>
<div class="timing">
<span class="timing-title">Timing:</span> 2–6 h (DNA extraction typically takes 4 h, RT-PCR 1.5 h with 30 min preparation)</div>
<ol>
<li>DNA extraction from ear tissue biopsy using PureLink Genomic DNA mini Kit (Thermo Fisher) according to manufacturer’s instructions.</li>
</ol>
<ol start="2">
<li>Preparation of real-time (RT)-PCR reaction using PowerUP SYBR green and primers designed for the <i>Timer BAC</i> transgene and an endogenous <i>Il2ra</i> control gene.
<ol type="a">
<li>Prepare reaction mix for <i>Timer BAC</i> and genomic <i>Il2ra.</i>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2× PowerUP SYBR Green</td>
<td>1×</td>
<td>5 μL</td>
</tr>
<tr>
<td>10 μM forward primer</td>
<td>250 nM</td>
<td>0.25 μL</td>
</tr>
<tr>
<td>10 μM reverse primer</td>
<td>250 nM</td>
<td>0.25 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>3.5 μL</td>
</tr>
<tr>
<td>DNA template</td>
<td>n/a</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Pipette 9 μL of reaction mix onto the wells of a 384 well PCR plate (or 96 well plate for compatible machines) in technical triplicates.</li>
<li>Add 1 μL of template DNA (concentration 10–50 ng/μL) to each reaction.</li>
<li>Seal the plate with optical adhesive and centrifuge for 15 s at 500 × <i>g</i>.</li>
<li>Set up real-time PCR on ABI 7900HT machine (or other SYBR green compatible machine) with the following cycling conditions:
<ul>
<li>55°C 2 min</li>
<li>95°C 2 min</li>
<li>35 cycles of the following three steps:</li>
<li>95°C 15 s</li>
<li>60°C 30 s</li>
<li>72°C 30 s</li>
</ul>
</li>
<li>Confirm reaction specificity by melt curve analysis – single peak should be observable.</li>
<li>Set same threshold of 0.2 for both <i>Il2ra</i> and <i>BAC Timer</i> to calculate Ct.</li>
<li>Calculate delta (Δ)Ct using the formula:<br> ΔCt=Ct (<i>Il2ra</i>)-Ct(<i>BAC Timer</i>)
<div class="note">
<span class="note-title">Note:</span> Depending on machine and SYBR green master mix, the ΔCt may vary. Controls with known zygosity should be used to identify ΔCt ranges.</div>
</li>
<li>
<a href="#fig1">Figure 1</a> shows typical ΔCt using this protocol and ABI 7900HT machine.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/420-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Typical delta (Δ)Ct values for homozygote and heterozygote <i>Nr4a3</i>-Tocky mice</div>
<p>ΔCt values based on Ct for <i>Timer Bac</i> – Ct for <i>Il2ra</i> for heterozygote (n = 5) and homozygote (n = 7) <i>Nr4a3</i>-Tocky mice. Dots represent individual mice and bars represent mean ± SEM.</p>
</figcaption>
</figure>
</li>
<li>Determine zygosity of line and use <i>Nr4a3</i>-Tocky homozygous in combination with <i>Nr4a1</i>-GFP for experiment.</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Panel design</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<div class="critical">
<span class="critical-title">Critical:</span> Key to high-quality analysis is optimal panel design and this should be validated before undertaking experiments. Here we illustrate the compensation matrix and optimized fluorochrome combinations using a BD LSR Fortessa X-20 machine. It is by no means exhaustive but represents a good starting point for panel design.</div>
<ol start="3">
<li>GFP, Timer-Blue, and Timer-Red are excitable by different lasers and detected by different filters (see <a href="#materials-and-equipment">Materials and equipment</a>). The assumption is that Timer-Blue and Timer-Red do not spectrally overlap (<a href="#bib5">Subach et al., 2009</a>), and therefore no compensation is required between these two channels (<a href="#bib2">Bending et al., 2018b</a>). Single color controls can be generated through using a <i>Nr4a1</i>-GFP line (or equivalent GFP expressing cell line), and <i>Nr4a3</i>-Tocky cells. To generate a single Blue and Red color control:
<ol type="a">
<li>Place 1 million <i>Nr4a3</i>-Tocky splenocytes into a 96 well round bottom plate containing 10% FBS RPMI (v/v) and stimulate with 5 μg/mL soluble anti-CD3 for 4 h.
<div class="note">
<span class="note-title">Note:</span> Nr4a3-Blue expression will remain for long as as a TCR stimulus is applied in vitro; however, the transition of Blue to Red has a half-life of approximately 4 h, meaning stimulus for longer periods will increase Red fluorescence of cells.</div>
</li>
<li>Harvest cells and maintain on ice until analysis on flow cytometer, gate on Nr4a3-Blue<sup>+</sup> (BV421 channel) and Nr4a3-Red<sup>−</sup> (PE-Texas Red channel) cells to use as a single color Blue control.</li>
<li>Use unstimulated Nr4a3-Tocky T cells and gate on Nr4a3-Blue<sup>−</sup>Nr4a3-Red<sup>+</sup> to generate a single Red color control.
<div class="note">
<span class="note-title">Note:</span> <a href="#fig2">Figure 2</a> shows that Blue, Red and GFP colors show no overlap (see also <a href="#tbl1">Table 1</a>)</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/420-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. GFP, Timer-Blue, and Timer-Red show no spectral overlap</div>
<p>(A) Thymocytes from OTI <i>Nr4a3</i>-Tocky <i>Nr4a1</i>-GFP mice (which do not express Nr4a3-Timer in the thymus; <a href="#bib3">Jennings et al., 2020</a>) were analyzed for expression of Nr4a3-Blue versus Nr4a1-GFP and Nr4a3-Red versus Nr4a1-GFP channels.</p>
<p>(B) Splenocytes from <i>Nr4a3</i>-Tocky (pure) line were analyzed for Nr4a1-GFP versus Nr4a3-Red (left) or stimulated for 3 h with anti-CD3 and then Nr4a3-Blue<sup>+</sup>Nr4a3-Red<sup>−</sup> cells analyzed for Nr4a1-GFP versus Nr4a3-Blue. These single color controls were used to make the compensation matrix in <a href="#tbl1">Table 1</a>.</p>
</figcaption>
</figure>
<table id="tbl1">
<caption>Table 1. Compensation matrix for GFP, Timer-Blue, Timer-Red, and tested channels on the BD LSR FORTESSA X-20</caption>
<thead>
<tr>
<th> </th>
<th>GFP</th>
<th>Blue</th>
<th>Red</th>
<th>APC-A</th>
<th>Alexa 700-A</th>
<th>APC-Cy7-A</th>
<th>PerCP-Cy5-5-A</th>
<th>BUV737-A</th>
<th>BUV395-A</th>
<th>PE-Cy7-A</th>
<th>BV605-A</th>
<th>BV711-A</th>
<th>PE-A</th>
</tr>
</thead>
<tbody>
<tr>
<td>GFP</td>
<td> </td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>1.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
</tr>
<tr>
<td>Blue</td>
<td>0.00%</td>
<td> </td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>8.00%</td>
<td>0.00%</td>
<td>0.00%</td>
</tr>
<tr>
<td>Red</td>
<td>0.00%</td>
<td>0.00%</td>
<td> </td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>4.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>2.20%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>16.00%</td>
</tr>
<tr>
<td>APC-A</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.90%</td>
<td> </td>
<td>40.00%</td>
<td>11.00%</td>
<td>2.00%</td>
<td>13.00%</td>
<td>0.00%</td>
<td>3.40%</td>
<td>0.00%</td>
<td>4.60%</td>
<td>0.00%</td>
</tr>
<tr>
<td>Alexa 700-A</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>3.30%</td>
<td> </td>
<td>27.00%</td>
<td>8.00%</td>
<td>28.50%</td>
<td>0.00%</td>
<td>5.50%</td>
<td>0.00%</td>
<td>11.00%</td>
<td>0.00%</td>
</tr>
<tr>
<td>APC-Cy7-A</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.60%</td>
<td>1.10%</td>
<td>5.50%</td>
<td> </td>
<td>0.30%</td>
<td>12.00%</td>
<td>0.00%</td>
<td>18.00%</td>
<td>0.00%</td>
<td>0.40%</td>
<td>0.20%</td>
</tr>
<tr>
<td>PerCP-Cy5-5-A</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>7.00%</td>
<td>9.50%</td>
<td>3.30%</td>
<td> </td>
<td>25.00%</td>
<td>0.00%</td>
<td>4.00%</td>
<td>0.00%</td>
<td>29.00%</td>
<td>0.00%</td>
</tr>
<tr>
<td>BUV737-A</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.10%</td>
<td>3.40%</td>
<td>1.70%</td>
<td>5.30%</td>
<td> </td>
<td>0.95%</td>
<td>0.20%</td>
<td>0.00%</td>
<td>0.80%</td>
<td>0.00%</td>
</tr>
<tr>
<td>BUV395-A</td>
<td>0.00%</td>
<td>0.20%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td> </td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
</tr>
<tr>
<td>PE-Cy7-A</td>
<td>0.10%</td>
<td>0.00%</td>
<td>3.50%</td>
<td>0.00%</td>
<td>0.30%</td>
<td>5.80%</td>
<td>5.20%</td>
<td>5.00%</td>
<td>0.00%</td>
<td> </td>
<td>0.00%</td>
<td>0.00%</td>
<td>2.20%</td>
</tr>
<tr>
<td>BV605-A</td>
<td>0.00%</td>
<td>3.20%</td>
<td>46.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>1.40%</td>
<td>9.10%</td>
<td>0.20%</td>
<td>0.70%</td>
<td> </td>
<td>11.70%</td>
<td>8.30%</td>
</tr>
<tr>
<td>BV711-A</td>
<td>0.00%</td>
<td>9.40%</td>
<td>0.00%</td>
<td>1.00%</td>
<td>14.00%</td>
<td>3.60%</td>
<td>25.00%</td>
<td>84.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.60%</td>
<td> </td>
<td>0.00%</td>
</tr>
<tr>
<td>PE-A</td>
<td>0.40%</td>
<td>0.00%</td>
<td>135.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>12.00%</td>
<td>0.00%</td>
<td>0.00%</td>
<td>0.20%</td>
<td>8.00%</td>
<td>0.00%</td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Compensation matrix generated on FlowJo software indicating percentage spectral overlap between various channels. Single color controls were generated using activated splenocytes from pure <i>Nr4a3</i>-Tocky mice (for Timer-Blue and Timer-Red; <a href="#fig2">Figure 2</a>), or thymus from OTI <i>Nr4a1</i>-GFP <i>Nr4a3</i>-Tocky (for Nr4a1-GFP single color; <a href="#fig2">Figure 2</a>). Non-transgenic cells were stained with the following antibodies to generate single color controls: anti-mouse CD5 APC; viability dye eFluor780 (APC-Cy7); anti-mouse TCRbeta PerCP-Cy5.5; anti-mouse CD8 BUV737; anti-mouse CD4 BUV395; anti-mouse CD8 PE-Cy7; anti-mouse-CX3CR1 BV605; anti-mouse CD11c BV711; anti-mouse CD103 PE.</p>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>For panel design it is recommended to use a viability dye (such as eFluor780) to exclude dead cells. A typical compensation matrix involving commonly used channels is shown (<a href="#tbl1">Table 1</a>). Typical compensation requirements between GFP, Timer-Blue, and Timer-Red with other channels are bold highlighted. <a href="#sec7.1">Troubleshooting 1</a>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Caution should be used when using BV711 and BV605, and PE-Cy7 due to the spectral overlaps between them and Timer-Red and Timer-Blue channels. We recommend that single color compensation controls for all tandem dyes use the dye that is in the final panel. We advise not using PE or BV510 due to the close proximity to Nr4a3-Red and Nr4a3-Blue. FITC is not compatible with GFP.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti CD3ϵ Armenian hamster monoclonal 145-2C11</td>
<td>BioLegend</td>
<td>Cat# 100301; RRID: AB_312666</td>
</tr>
<tr>
<td>Anti-CD4 BUV737 rat monoclonal GK1.5</td>
<td>BD Biosciences</td>
<td>Cat# 564298; RRID: AB_2738734</td>
</tr>
<tr>
<td>Anti CD8 BUV395 rat monoclonal 53-6.7</td>
<td>BD Biosciences</td>
<td>Cat# 563786; RRID: AB_2732919</td>
</tr>
<tr>
<td>Anti CD69 AF700 Armenian hamster monoclonal H1.2F3</td>
<td>BioLegend</td>
<td>Cat# 104539; RRID: AB_2566304</td>
</tr>
<tr>
<td>Anti CD103 PE Armenian hamster monoclonal 2E7</td>
<td>BioLegend</td>
<td>Cat# 121406; RRID: AB_535948</td>
</tr>
<tr>
<td>Anti CX3CR1 BV605 mouse monoclonal SA011F11</td>
<td>BioLegend</td>
<td>Cat# 149027; RRID: AB_2565937</td>
</tr>
<tr>
<td>Anti CD11c BV711 Armenian hamster monoclonal N418</td>
<td>BioLegend</td>
<td>Cat# 117349; RRID: AB_2563905</td>
</tr>
<tr>
<td>Anti CD8 PE-cy7 rat monoclonal 53-6.7</td>
<td>BioLegend</td>
<td>Cat# 100722; RRID: AB_312760</td>
</tr>
<tr>
<td>Anti CD5 APC rat monoclonal 52-7.3</td>
<td>BioLegend</td>
<td>Cat# 100625; RRID: AB_2563928</td>
</tr>
<tr>
<td>Anti-TCRbeta PerCP-Cy5.5 Armenian hamster monoclonal H57-597</td>
<td>Tonbo Biosciences</td>
<td>Cat# 65-5961-UO25; RRID: AB_2621911</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>PP2</td>
<td>Sigma-Aldrich</td>
<td>Cat# P0042; CAS: 172889-27-9</td>
</tr>
<tr>
<td>RPMI 1640 (referred to as RPMI)</td>
<td>Gibco</td>
<td>21875034</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Gibco</td>
<td>10500064</td>
</tr>
<tr>
<td>PBS</td>
<td>Sigma-Aldrich</td>
<td>D8537</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Applied Biosystems SYBR green power up master mix</td>
<td>Thermo Fisher</td>
<td>Cat# A25752</td>
</tr>
<tr>
<td>eFluor-780 fixable viability dye</td>
<td>eBioscience</td>
<td>Cat# 65-0865-14</td>
</tr>
<tr>
<td>PureLink genomic DNA mini kit</td>
<td>Thermo Fisher</td>
<td>K182002</td>
</tr>
<tr>
<td>Red blood cell lysis buffer</td>
<td>Thermo Fisher</td>
<td>00-4333-57</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: <i>Nr4a3</i>-Tocky founder line 323</td>
<td>Dr. Masahiro Ono, Imperial College London</td>
<td>PMID: 29941474(<a href="#bib2">Bending et al., 2018b</a>)</td>
</tr>
<tr>
<td>Mouse: Nr4a1/Nur77 GFP</td>
<td>Jackson Laboratory</td>
<td>Strain code: 016617</td>
</tr>
<tr>
<td>Mouse: OTI</td>
<td>Charles River Laboratories</td>
<td>Strain Code: 642</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>
<i>Timer BAC</i> F<i>:</i> CGCGGAACTAACTTCCCCTC</td>
<td>Sigma-Aldrich</td>
<td>This paper</td>
</tr>
<tr>
<td>
<i>Timer BAC</i> R<i>: GTCTTGACCTCAGCGTCGTA</i>
</td>
<td>Sigma-Aldrich</td>
<td>This paper</td>
</tr>
<tr>
<td>
<i>Il2ra</i> F<i>: CAGGAGTTTCCTAAGCAACG</i>
</td>
<td>Sigma-Aldrich</td>
<td>This paper</td>
</tr>
<tr>
<td>
<i>Il2ra</i> R<i>:</i> CTGTGTCTGTATGACCCACC</td>
<td>Sigma-Aldrich</td>
<td>This paper</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism 8 and 9</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td>FlowJo v10.5.3</td>
<td>FlowJo</td>
<td><a href="https://www.flowjo.com/solutions/flowjo">https://www.flowjo.com/solutions/flowjo</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>BD LSR FORTESSA X-20 machine</td>
<td>BD Biosciences</td>
<td>Custom machine</td>
</tr>
<tr>
<td>ABI 7900HT</td>
<td>Thermo Fisher</td>
<td>Custom machine</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Mice</h3>
<p>All animal experiments were performed in accordance with local Animal Welfare and Ethical Review Body at the University of Birmingham and under the authority of a Home Office project license, P18A892E0A held by D.B. Animals were housed in specific pathogen-free conditions with appropriate housing conditions and husbandry as specified by NC3Rs. <i>Nr4a3</i>-Tocky mice expressing a BAC containing FT Fast mCherry mutant (<a href="#bib5">Subach et al., 2009</a>) under the influence of <i>Nr4a3</i> regulatory regions on the C57BL/6J background as previously described (<a href="#bib1">Bending et al., 2018a</a>; <a href="#bib2">Bending et al., 2018b</a>) were bred to Nr4a1/Nur77-GFP mice expressing a BAC containing <i>GFP</i> transgene under the influence of <i>Nr4a1</i> regulatory regions on C57BL/6J background as previously described (<a href="#bib4">Moran et al., 2011</a>). These were mated to OTI mice (expressing the TCR (Vα2, Vβ5) of the OVA257-264-specific CTL clone 149.42) to generate OTI <i>Nr4a3</i>-Tocky <i>Nr4a1</i>-GFP mice.</p>
<h3 id="sec2.2">Flow cytometer – BD LSR FORTESSA X-20</h3>
<p>Data were acquired on a BD LSR FORTESSA X-20 flow cytometer, with the configuration detailed below. Dyes require careful compensation considerations are marked with an asterisk. Other cytometers with similar setups would also be suitable.</p>
<table id="undtbl3">
<thead>
<tr>
<th colspan="3">BD LSR FORTESSA X-20 dyes tested</th>
</tr>
<tr>
<th>Laser</th>
<th>Filter</th>
<th>Tested dye(s)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">355 nm (<b>UV</b>)</td>
<td>740/35</td>
<td>BUV737</td>
</tr>
<tr>
<td>379/28</td>
<td>BUV395</td>
</tr>
<tr>
<td rowspan="6">405 nm (<b>violet</b>)</td>
<td>450/50</td>
<td>Timer-Blue</td>
</tr>
<tr>
<td>525/50</td>
<td><b>not advised BV510</b></td>
</tr>
<tr>
<td>610/20</td>
<td>BV605∗</td>
</tr>
<tr>
<td>670/30</td>
<td>not tested</td>
</tr>
<tr>
<td>710/50</td>
<td>BV711∗</td>
</tr>
<tr>
<td>780/60</td>
<td>not tested</td>
</tr>
<tr>
<td rowspan="2">488 nm (<b>blue</b>)</td>
<td>530/30</td>
<td>GFP</td>
</tr>
<tr>
<td>670/30</td>
<td>PerCP-Cy5.5 (or DUMP)</td>
</tr>
<tr>
<td rowspan="5">561 nm (<b>yellow</b>)</td>
<td>586/15</td>
<td><b>not advised PE</b></td>
</tr>
<tr>
<td>610/20</td>
<td>Timer-Red</td>
</tr>
<tr>
<td>670/30</td>
<td>not tested</td>
</tr>
<tr>
<td>710/50</td>
<td>not tested</td>
</tr>
<tr>
<td>795/71</td>
<td>PE-Cy7∗</td>
</tr>
<tr>
<td rowspan="3">642 nm (<b>red</b>)</td>
<td>671/30</td>
<td>APC</td>
</tr>
<tr>
<td>722/44</td>
<td>AF700</td>
</tr>
<tr>
<td>795/70</td>
<td>efluor780</td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>BD LSR FORTESSA X-20 settings</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Laser</th>
<th>Filter</th>
<th>Gain</th>
<th>Voltage</th>
</tr>
</thead>
<tbody>
<tr>
<td>BUV395-A</td>
<td>355 nm (UV)</td>
<td>379/28</td>
<td>1</td>
<td>576</td>
</tr>
<tr>
<td>BUV737-A</td>
<td>355 nm (UV)</td>
<td>740/35</td>
<td>1</td>
<td>750</td>
</tr>
<tr>
<td>BV421-A</td>
<td>405 nm (violet)</td>
<td>450/50</td>
<td>1</td>
<td>406</td>
</tr>
<tr>
<td>BV605-A</td>
<td>405 nm (violet)</td>
<td>610/20</td>
<td>1</td>
<td>549</td>
</tr>
<tr>
<td>BV711-A</td>
<td>405 nm (violet)</td>
<td>710/50</td>
<td>1</td>
<td>514</td>
</tr>
<tr>
<td>FITC-A</td>
<td>488 nm (blue)</td>
<td>530/30</td>
<td>1</td>
<td>405</td>
</tr>
<tr>
<td>PerCP-Cy5-5-A</td>
<td>488 nm (blue)</td>
<td>670/30</td>
<td>1</td>
<td>583</td>
</tr>
<tr>
<td>PE-A</td>
<td>561 nm (yellow)</td>
<td>586/15</td>
<td>1</td>
<td>503</td>
</tr>
<tr>
<td>PE-Texas Red-A</td>
<td>561 nm (yellow)</td>
<td>610/20</td>
<td>1</td>
<td>612</td>
</tr>
<tr>
<td>PE-Cy7-A</td>
<td>561 nm (yellow)</td>
<td>795/71</td>
<td>1</td>
<td>500</td>
</tr>
<tr>
<td>APC-A</td>
<td>642 nm (red)</td>
<td>671/30</td>
<td>1</td>
<td>520</td>
</tr>
<tr>
<td>APC-Alexa 700-A</td>
<td>642 nm (red)</td>
<td>722/44</td>
<td>1</td>
<td>487</td>
</tr>
<tr>
<td>APC-Cy7-A</td>
<td>642 nm (red)</td>
<td>795/70</td>
<td>1</td>
<td>493</td>
</tr>
</tbody>
</table>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Alternative fluorochrome formats are available for many channels, however these should be tested in advance for compatibility to Nr4a3-Blue, Nr4a3-Red, and Nr4a1-GFP. Alternative flow cytometer settings and filters are also available, but each user should optimize voltages and panel design to their own machines.</div>
<h3 id="sec2.3">Buffers</h3>
<p>Staining and washing buffers: consisted of 1× PBS supplemented with 2% FBS (v/v).</p>
<p>Culture media: 10% FBS (v/v) RMPI 1640 was supplemented with 1% penicillin/streptomycin (v/v, made from 100× stock at 5,000 units/mL penicillin, 5,000 μg/mL streptomycin, Life Technologies).</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Assessing T cell receptor signal duration in the regulation of <i>Nr4a1</i> and <i>Nr4a3</i> expression using <i>Nr4a1</i>-GFP and <i>Nr4a3</i>-Tocky dual reporter mice</h3>
<h4 id="sec3.1.1">Generation of single-cell splenocyte suspension</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>This part generates a single-cell suspension of splenocytes for use in experimental cultures.</p>
<ol>
<li>Harvest spleen from <i>Nr4a1</i>-GFP <i>Nr4a3</i>-Tocky mice (<a href="#bib3">Jennings et al., 2020</a>) in TC hood.
<ol type="a">
<li>Mice are euthanized by cervical dislocation, and death confirmed by cessation of circulation.</li>
<li>Spleen is removed, ensuring complete detachment of the pancreas and placed into a 1.5 mL Eppendorf containing 0.5 mL 10% FBS RPMI.</li>
</ol>
</li>
<li>Generation of single-cell suspension in tissue culture hood.
<ol type="a">
<li>Into a sterile 5 mL petri dish, place a 70 μm cell strainer and add the spleen and 1 mL of 10% FBS RPMI buffer. (Alternatively, a strainer can be placed in a 50 mL Falcon tube for processing).</li>
<li>Using the syringe plunger from a 5 mL syringe, gently force the spleen through the cell strainer.</li>
<li>Using a P1000 pipette add 1 mL 10% FBS RPMI buffer to wash the strainer.</li>
<li>Filter the splenocyte solution back through the strainer by taking from suspension in the petri dish and pipetting back onto the strainer. Repeat this 8–10 times using P1000 pipette until suspension is homogenous.</li>
<li>Pool the splenocyte suspension by transferring it into a 15 mL Falcon tube.</li>
<li>Wash strainer with 1 mL 10% FBS RMPI and transfer this to the 15 mL Falcon tube.</li>
<li>Centrifuge Falcon tube at 500 × <i>g</i> for 5 min at 15°C–20°C.</li>
<li>Decant supernatant and proceed to perform red blood cell lysis by resuspending the cell pellet in 1 mL of RBC lysis buffer.</li>
<li>Incubate for 2 min on ice.</li>
<li>Top up the 15 mL Falcon tube with 9 mL 10% FBS RPMI and centrifuge at 500 × <i>g</i> for 5 min at 15°C–20°C.</li>
<li>Decant supernatant and resuspend cell pellet at cell concentration of 5 million/mL in 10% FBS RPMI and keep cells on ice.</li>
</ol>
</li>
</ol>
<h4 id="sec3.1.2">Culture setup</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 0–4 h</div>
<p>This part established how to modulate the TCR signal length in splenocyte cultures from <i>Nr4a1</i>-GFP <i>Nr4a3</i>-Tocky mice.</p>
<div class="optional">
<span class="optional-title">Optional:</span> Cultures are set up on 96 well round bottom plates (Corning) in a final volume of 190 μL, with 10 μL volume used to add the PP2 inhibitor, however depending on cell numbers and desired interactions other plate formats may also be suitable.</div>
<div class="note">
<span class="note-title">Note:</span> PP2 is reconstituted in sterile DMSO to a master stock concentration of 20 mM.</div>
<ol start="3">
<li>Prepare master mixes of anti-CD3 and PP2, a src-kinase family inhibitor which will pharmacologically terminate TCR signaling in culture.
<ol type="a">
<li>20 μg/mL anti-CD3, (pre-warmed to 37°C).</li>
<li>200 μm PP2 (keep on ice).</li>
</ol>
</li>
<li>Pipette 100 μL of splenocyte suspension which is at 5 million/mL into 8 wells of 96 well round bottom plate. For this experiment there are 8 conditions representing 0, 2.5, 5, 15, 30, 60, 120, and 240 min of TCR stimulation.</li>
<li>Add 40 μL of 10% FBS RPMI to each well.</li>
<li>Spin plate for 1 min at 100 × <i>g</i> to settle cells to bottom of plate.</li>
<li>Place plate in incubator for 30 min to equilibrate at 37°C.</li>
<li>Remove plate and add 50 μL of 20 μg/mL anti-CD3 stock (final concentration 5 μg/mL) to wells.</li>
<li>Add 10 μL of 200 μM PP2 to 0-min stimulation (gives final concentration 10 μM).</li>
<li>Place plate in incubator with 5% CO<sub>2</sub> and 37°C and start timer.</li>
<li>At 2.5-, 5-, 15-, 30-, 60-, 120-min remove plate and add 10 μL of 200 μM PP2 inhibitor. Place plate back in incubator until 240 min has expired.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During incubation, the staining mix can be prepared (see below).</div>
<ol start="12">
<li>After 240 min proceed immediately to staining.</li>
</ol>
<h4 id="sec3.1.3">Staining for flow cytometric analysis</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–1 h</div>
<ol start="13">
<li>Centrifuge culture plate at 500 × <i>g</i> for 3 min, flick plate to decant supernatant and while still inverted blot against paper towel.</li>
<li>Add 25 μL of staining mix prepared as below in 2% FBS PBS (v/v). In our hands viability dye staining can be performed simultaneously with surface staining without adversely affecting performance.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Include single color compensation controls in addition to GFP, Timer-Blue, and Timer-Red.</div>
<table id="undtbl5">
<thead>
<tr>
<th>Stain</th>
<th>Stain dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>Viability Dye eFluor780</td>
<td>1 in 2,000</td>
</tr>
<tr>
<td>CD4 BUV395</td>
<td>1 in 200</td>
</tr>
<tr>
<td>CD8 BUV737</td>
<td>1 in 200</td>
</tr>
<tr>
<td>CD69 AF700</td>
<td>1 in 50</td>
</tr>
<tr>
<td>
<i>Optional</i> PEcy7</td>
<td><i>Optional stain</i></td>
</tr>
<tr>
<td>
<i>Optional</i> APC</td>
<td><i>Optional stain</i></td>
</tr>
<tr>
<td>
<i>Optional</i> PerCP-Cy5.5</td>
<td><i>Optional stain / DUMP</i></td>
</tr>
</tbody>
</table>
<ol start="15">
<li>Incubate for 20 min at 4°C.</li>
<li>Add 180 μL of wash buffer to each well and centrifuge plate at 500 × <i>g</i> for 3 min, flick plate to decant supernatant and while still inverted blot against paper towel.</li>
<li>Resuspend pellet in 150 μL 2% FBS PBS and transfer to 5 mL FACS tube (BD Biosciences). Wash well with 150 μL 2% FBS PBS and transfer to FACS tube to make 300 μL final volume.</li>
<li>Place on ice in the dark and acquire samples on the flow cytometer within 2 h of staining.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cells should be analyzed as soon as possible following staining. In our hands, slow maturation of Blue to Red can occur after prolonged periods (e.g., overnight) at 4°C.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Using this protocol, it is expected to see an early increase in Nr4a1-GFP expression commencing within as little as 5–10 min of stimulation time. <a href="#sec7.3">Troubleshooting 2</a>Nr4a3-Blue expression will start to increase after 30–60 min of stimulation. After 12 min of TCR signaling 50% of T cells should be Nr4a1-GFP<sup>+</sup>. After 70 min of TCR stimulation 50% of T cells should be Nr4a3-Blue<sup>+</sup>. With 4 h of TCR signaling &gt;85%–90% of CD4<sup>+</sup> and CD8<sup>+</sup> T cells should be Nr4a1-GFP<sup>+</sup>Nr4a3-Blue<sup>+</sup> (see Figure 4D in (<a href="#bib3">Jennings et al., 2020</a>)) <a href="#sec7.7">Troubleshooting 4</a></p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>To determine the percentage positivity, a non-transgenic line can be used to set the threshold for Nr4a1-GFP and Nr4a3-Blue positivity. Given that Nr4a1-GFP has higher background expression it is important that the data are normalized to the minimum at T=0 min and maximum at T=240 min using the following formulas:</p>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/420-Si1.jpg" alt="Si1.jpg"></p>
<p><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/420-Si2.jpg" alt="Si2.jpg"></p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is designed for the analysis of Nr4a1-GFP and Nr4a3-Timer expression patterns by flow cytometry. Other fluorescence-based analysis techniques such as confocal microscopy have not been tested or validated on the dual reporter. This protocol also relies on the analysis occurring within 2 h of staining, and samples being kept on ice in the dark to prevent the maturation of Timer-Blue into Timer-Red. This protocol is also not compatible with fixation and intranuclear factor staining with kits such as the eBioscience Foxp3 / Transcription Factor Staining Buffer Set.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>A diagonal population of Blue<sup>+</sup>Red<sup>+</sup> cells appear in unstimulated controls.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>This is typically seen in preparations from tissues that have increased autofluorescence of some cells. To reduce background, we advise having a “dump” channel open such as the PerCP-Cy5.5 channel. If no dye is used in this channel, background noise is reduced substantially (<a href="#fig3">Figure 3</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/420-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Use of PerCP-Cy5.5 channel as a dump to reduce noise</div>
<p>(A) Lymphocytes from wild-type mice displaying autofluorescence in the Timer-Blue versus Timer-Red channels, which appears as a diagonal line.</p>
<p>(B) By gating on FSC-A versus PerCP-Cy5.5 (open channel no stain), the noise can be significantly reduced by excluding PerCP-Cy5.5<sup>+</sup> cells.</p>
<p>(C) The result is that lymphocytes from wild-type mice gated as PerCP-Cy5.5<sup>−</sup> display very low noise in the Timer-Blue versus Timer-Red channels.</p>
</figcaption>
</figure>
<h3 id="sec7.3">Problem 2</h3>
<p>Weak signal in Blue channel but strong signal in GFP after stimulation.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>This is typically caused by using <i>Nr4a3</i>-Tocky heterozygous BAC mice in combination with <i>Nr4a1</i>-GFP BAC mice. While <i>Nr4a3</i>-Tocky heterozygous mice are usable alone, when crossed with <i>Nr4a1</i>-GFP BAC mice the Nr4a3-Timer signal is quenched. We advise checking zygosity of line and repeating experiment with a confirmed <i>Nr4a3</i>-Tocky homozygote with <i>Nr4a1</i>-GFP heterozygote.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Ex vivo Timer-Red signal is dim for single color staining control.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Splenocytes can be activated with soluble anti-CD3 for 4 h before washing and keeping in fridge for 24 h. The Blue proteins will mature to the terminal Red form and provide a brighter single Red color control.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Cells show a clear Blue<sup>+</sup>Red<sup>+</sup> pattern, even after a short 4-h stimulation.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>This may occur if cells are not analyzed immediately. The half-life of the timer protein is 4 h, and this process can be stopped by keeping cells on ice before acquisition. We strongly recommend analysis within 2 h of staining of samples. Overnight incubation in the fridge is likely to reduce Timer-Blue and increase Timer-Red signal.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bending, D., Paduraru, A., Ducker, C.B., Prieto Martin, P., Crompton, T., and Ono, M. (2018a). A temporally dynamic Foxp3 autoregulatory transcriptional circuit controls the effector Treg programme. EMBO J. <i>37</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30271-9/sref1">View at publisher</a></p>
<p id="bib2">Bending, D., Prieto Martin, P., Paduraru, A., Ducker, C., Marzaganov, E., Laviron, M., Kitano, S., Miyachi, H., Crompton, T., and Ono, M. (2018b). A timer for analyzing temporally dynamic changes in transcription during differentiation in vivo. J. Cell Biol. <i>217</i>, 2931-2950. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30271-9/sref2">View at publisher</a></p>
<p id="bib3">Jennings, E., Elliot, T.A.E., Thawait, N., Kanabar, S., Yam-Puc, J.C., Ono, M., Toellner, K.M., Wraith, D.C., Anderson, G., and Bending, D. (2020). Nr4a1 and Nr4a3 reporter mice are differentially sensitive to T cell receptor signal strength and duration. Cell Rep. <i>33</i>, 108328. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30271-9/sref3">View at publisher</a></p>
<p id="bib4">Moran, A.E., Holzapfel, K.L., Xing, Y., Cunningham, N.R., Maltzman, J.S., Punt, J., and Hogquist, K.A. (2011). T cell receptor signal strength in Treg and iNKT cell development demonstrated by a novel fluorescent reporter mouse. J. Exp. Med. <i>208</i>, 1279-1289. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30271-9/sref4">View at publisher</a></p>
<p id="bib5">Subach, F.V., Subach, O.M., Gundorov, I.S., Morozova, K.S., Piatkevich, K.D., Cuervo, A.M., and Verkhusha, V.V. (2009). Monomeric fluorescent timers that change color from blue to red report on cellular trafficking. Nat. Chem. Biol. <i>5</i>, 118-126. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(20)30271-9/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, David Bending (<a href="mailto:d.a.bending@bham.ac.uk">d.a.bending@bham.ac.uk</a> ).</p>
<h4>Materials Availability</h4>
<p><i>Nr4a3</i>-Tocky mice are available under MTA from Dr Masahiro Ono, (Imperial College London, UK). Nr4a1/Nur77-GFP mice are available from the Jax lab (strain number 016617).</p>
<h4>Data and Code Availability</h4>
<p>All data from the study are present here or in <a href="#bib3">Jennings et al., 2020</a>.</p>
<h3>Acknowledgments</h3>
<p>D.B. is funded by the <a href="https://doi.org/10.13039/501100000855">University of Birmingham</a>, the <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> (214018/Z/18/Z), and the <a href="https://doi.org/10.13039/501100000265">MRC</a> (MR/V009052/1). D.A.L. is funded by a <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> 4-year Basic Science PhD program. E.K.J. is supported by a studentship from the MRC Discovery Medicine North (DiMeN) Doctoral Training Partnership (MR/N103840/1). M.O. is funded by the <a href="https://doi.org/10.13039/501100000268">BBSRC</a> (BB/J013951/2) and <a href="https://doi.org/10.13039/501100000265">MRC</a> (MR/S000208/1). The graphical abstract was created with <a href="https://biorender.com/">https://biorender.com</a>.</p>
<h3>Author Contributions</h3>
<p>E.K.J., D.A.L., and D.B. collected data, and together wrote and edited the manuscript. M.O. provided the <i>Nr4a3</i>-Tocky line and designed the primers for zygosity typing.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>