<article>
<section>
<h1 id="header">Preclinical testing of oncolytic adenovirus sensitivity in patient-derived tumor organoids</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Silvia Pascual-Sabater,<sup><a href="#aff1">1</a>,<a href="#fn1">6</a>,<a href="#cor1">*</a></sup> Giulia Raimondi,<sup><a href="#aff1">1</a>,<a href="#fn1">6</a></sup> Ana Mato-Berciano,<sup><a href="#aff1">1</a></sup> Eva C. Vaquero,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Fabio Ausania,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a></sup> and Cristina Fillat<sup><a href="#aff1">1</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#fn2">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Catalunya 08036, Spain</p>
<p id="aff2"><sup>2</sup>Gastroenterology and Pancreatic Oncology Research Group, Hospital Clínic, CiberEHD, Barcelona, Catalunya 08036, Spain</p>
<p id="aff3"><sup>3</sup>Hepatobiliary and Pancreatic Surgery Unit, Hospital Clínic, Barcelona, Catalunya 08036, Spain</p>
<p id="aff4"><sup>4</sup>Centro de Investigación Biomédica en Red de Enfermedades Raras (CIBERER), Barcelona, Catalunya 08036, Spain</p>
<p id="aff5"><sup>5</sup>Facultat de Medicina I Ciències de la Salut, Universitat de Barcelona, Barcelona, Catalunya 08036, Spain</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:sipascual@clinic.cat">sipascual@clinic.cat</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:cfillat@clinic.cat">cfillat@clinic.cat</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101017">10.1016/j.xpro.2021.101017</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Patient-derived organoids (PDOs) have shown the potential to reflect patient sensitivity to chemotherapeutic or targeted drugs. Recently, we showed that organoid models can also serve as a platform to screen for selectivity and potency of oncolytic adenoviruses (OAds). In this protocol, we describe the steps for tumor organoid adenoviral infection and functional assessment of patient-specific responses to OAds. We provide methods to determine OAd relative efficacy by evaluation of PDO viability after infection and adenoviral replication within cancer cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref6">Raimondi et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for infection of organoids with oncolytic adenoviruses</li>
<li>Steps to quantify adenoviral replication within organoids</li>
<li>Evaluation of viability after infection to determine relative oncolytic efficacy</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1233-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>For the completion of this protocol, availability of both organoids (PDOs) and oncolytic adenoviruses (OAd) is assumed.</p>
<h3 id="sec1.1">Organoid generation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–6 weeks</div>
<ol>
<li>Generate organoids from tumor tissue following standard protocols. Detailed instructions for the establishment of PDAC organoids can be found in <a href="#bib2">Driehuis et al. (2020)</a>.</li>
<li>For culture maintenance, grow organoids in 80% Matrigel (diluted in F12+++ medium) and replace organoid growth medium every 3–4 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Matrigel lots with a protein concentration &gt; 9 mg/mL are preferred to ensure organoid 3D morphology. Therefore, the final protein concentration in drops containing organoids is 7.0–7.5 mg/mL, both in culture maintenance and in organoid seeding after infection.</div>
<ol start="3">
<li>Split organoids 6–7 days before organoid infection to ensure &gt; 70% confluency at the day of the experiment, saving 2 of the seeded wells for cell count. Change organoid growth medium every 2–3 days.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To maximize reproducibility of results, organoids should be passaged at a constant split ratio. Each organoid line may require different split ratios depending on its growth rate (for PDAC organoids, split ratios usually range between 1:3–1:6 every 7–10 days). PDOs should ideally have a similar size and confluence at the time of seeding and infection to compare outcomes of different experiments.</div>
<div class="note">
<span class="note-title">Note:</span> Once established, we usually maintain organoid cultures by seeding a drop of 50 μL of Matrigel 80% (10 μL F12+++ + 40 μL Matrigel) containing organoids in each well of a 24-well plate and supplementing with 450 μL of organoid growth medium. Therefore, approximate numbers given in this protocol refer to cultures grown in this format, even though other organoid handling techniques may also be valid.</div>
<h3 id="sec1.2">Generation and titration of oncolytic adenoviruses</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Variable</div>
<ol start="4">
<li>Oncolytic adenoviruses (OAds) should be available before starting this protocol. The production and titration of OAds used here were performed in-house as described in <a href="#bib7">Rovira-Rigau et al. (2019)</a>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ready-to-use virus aliquots must be stored in a −80°C freezer and repeated freeze-thaw cycles, which can negatively impact infective capacity (there can be a 10% drop for each freeze-thaw cycle), should be avoided. Thaw a new OAd aliquot on ice for each experiment and ensure that viruses to be compared have undergone the same number of freeze-thaw cycles and have been titrated in parallel.</div>
<div class="note">
<span class="note-title">Note:</span> It is advisable to titer OAds again before use if they have been stored for more than 12 months.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Reporter genes (e.g., EGFP) can be additionally cloned in the OAd genome to easily follow infection and viral spreading within organoids (<a href="#fig1">Figure 1</a>). Alternatively, for non-tagged OAds, viral genomes can be quantified by qPCR (<a href="#fig2">Figure 2</a>). For detailed instructions on viral genome quantification from infected PDOs, see section ‘<a href="#step-by-step-method-details">step-by-step method details</a>’.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1233-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Time-course of PDO infection with EGFP-OAd</div>
<p>Organoids infected with an EGFP-containing OAd (2 MOI) were imaged every 24 h after infection in an inverted fluorescence microscope. Viral replication and spreading within organoids can be confirmed visually by an increase in EGFP expression with time. Scale bar, 100 μm.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1233-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Quantification of viral genomes by qPCR to confirm OAd replication in PDOs</div>
<p>PDOs were infected with 2 different OAds (OAd1 and OAd2) at 2 MOI, subjected to freeze-thaw cycles and to DNA extraction (passage 1, P1). Fifteen μL of lysate-derived supernatants were used to infect fresh PDOs, and the same process was repeated to quantify viral genomes in organoids (passage 2, P2). Adenoviral genomes were quantified by qPCR amplification of hexon genes using a standard curve consisting of DNA dilutions of known copy numbers (10<sup>2</sup>–10<sup>7</sup>). High viral yields were detected in P1 and P2 with both viruses, proving that the OAd effectively replicates within PDOs. Data are represented as mean ± SEM (n = 3).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human pancreatic ductal adenocarcinoma tissue (surgical resection specimens, fine-needle biopsies)</td>
<td>Hospital Clínic (Barcelona)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>A 83-01</td>
<td>Tocris Bioscience (Bio-Techne)</td>
<td>Cat#2939</td>
</tr>
<tr>
<td>Advanced DMEM/F-12</td>
<td>Gibco</td>
<td>Cat#12634010</td>
</tr>
<tr>
<td>B-27 supplement (50x), serum-free</td>
<td>Gibco (Thermo Fisher Scientific)</td>
<td>Cat#17504044</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A3294; CAS: 9048-46-8</td>
</tr>
<tr>
<td>Gastrin I, human (hGastrin I)</td>
<td>Tocris Bioscience (Bio-Techne)</td>
<td>Cat#3006</td>
</tr>
<tr>
<td>GlutaMAX™ Supplement</td>
<td>Gibco</td>
<td>Cat#35050038</td>
</tr>
<tr>
<td>HEPES solution (1M, pH 7.0–7.6)</td>
<td>Sigma-Aldrich</td>
<td>Cat#H0887; CAS: 7365-45-9</td>
</tr>
<tr>
<td>Human EGF Recombinant Protein (hEGF)</td>
<td>Gibco</td>
<td>Cat#PHG0311</td>
</tr>
<tr>
<td>Matrigel Basement Membrane Matrix, Phenol Red-free</td>
<td>Corning</td>
<td>Cat#356231</td>
</tr>
<tr>
<td>N-Acetylcysteine</td>
<td>Sigma-Aldrich</td>
<td>Cat#A9615; CAS: 616-91-1</td>
</tr>
<tr>
<td>Nicotinamide</td>
<td>Sigma-Aldrich</td>
<td>Cat#N0636; CAS: 98-92-0</td>
</tr>
<tr>
<td>Penicillin/Streptomycin (P/S)</td>
<td>Life Technologies</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>Phosphate Buffer Saline (PBS) (10X), pH 7.2</td>
<td>Gibco</td>
<td>Cat#70013065</td>
</tr>
<tr>
<td>Primocin</td>
<td>InvivoGen</td>
<td>Cat#ANT-PM-1</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Blood DNA Isolation Mini Kit</td>
<td>Norgen Biotek</td>
<td>Cat#46380</td>
</tr>
<tr>
<td>CellTiter-Glo® 3D Cell Viability Assay</td>
<td>Promega</td>
<td>Cat#G9681</td>
</tr>
<tr>
<td>LightCycler® 480 SYBR Green I Master (for qPCR)</td>
<td>Roche</td>
<td>Cat#04707516001</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Hexon Forward (Fw): GCCGCAGTGGTCTTACATGCACATC</td>
<td>(<a href="#bib8">Villanueva et al., 2017</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Hexon Reverse (Rv): CAGCACGCCGCGGATGTCAAAG</td>
<td>(<a href="#bib8">Villanueva et al., 2017</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Excel</td>
<td>Microsoft Office</td>
<td>RRID:SCR_016137</td>
</tr>
<tr>
<td>Prism 8</td>
<td>GraphPad</td>
<td>RRID:SCR_002798</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>BackSeal-96/384, White Adhesive Bottom Seal for 96-well and 384-well Microplate</td>
<td>PerkinElmer</td>
<td>Cat#6005199</td>
</tr>
<tr>
<td>CulturPlate-96, White Opaque 96-well Microplate, Sterile and Tissue Culture Treated</td>
<td>PerkinElmer</td>
<td>Cat#6005680</td>
</tr>
<tr>
<td>Glass Pasteur pipettes (pre-narrowed in the flame)</td>
<td>Deltalab</td>
<td>Cat#702</td>
</tr>
<tr>
<td>MicroAmp<sup>TM</sup> Optical 384-Well Reaction Plate with Barcode</td>
<td>Applied Biosystems</td>
<td>Cat#4309849</td>
</tr>
<tr>
<td>Microamp<sup>TM</sup> Optical Adhesive Film</td>
<td>Applied Biosystems</td>
<td>Cat#4311971</td>
</tr>
<tr>
<td>Nanodrop 1000</td>
<td>Thermo Scientific</td>
<td>RRID:SCR_016517</td>
</tr>
<tr>
<td>Olympus IX51 Inverted Microscope</td>
<td>Olympus</td>
<td>N/A</td>
</tr>
<tr>
<td>Synergy<sup>TM</sup> HT Multi-mode Microplate Reader</td>
<td>BioTek</td>
<td>RRID:SCR_020536</td>
</tr>
<tr>
<td>ViewPlate-96, White 96-well Microplate with Clear Bottom, Sterile and Tissue Culture Treated, Lid Included</td>
<td>PerkinElmer</td>
<td>Cat#6005181</td>
</tr>
<tr>
<td>ViiA 7 Real-Time PCR System</td>
<td>Applied Biosystems</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> To perform CellTiter-Glo® 3D cell viability assay, white-walled, clear-bottom plates can also be used to facilitate visual inspection of infected PDOs. However, we recommend the use of white-bottom plates to minimize signal crosstalk between wells.</div>
<table id="undtbl1">
<caption>Basal medium, with (F12+++B) or without (F12+++) BSA</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Advanced DMEM/F-12</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
<tr>
<td>HEPES (1 M)</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Pen/Strep</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>2 mM (1%)</td>
<td>5 mL</td>
</tr>
<tr>
<td>BSA 30% (for F12+++B)</td>
<td>0.1%</td>
<td>1.72 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>516.72 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Once prepared, store at 4°C</p>
<table id="undtbl2">
<caption>30% BSA solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>30%</td>
<td>6 g</td>
</tr>
<tr>
<td>PBS 1X (sterile)</td>
<td>N/A</td>
<td>Up to 20 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Once prepared, filter the solution through a 0.22 μm filter and store in 1.8 mL aliquots at −20°C</p>
<table id="undtbl3">
<caption>Pancreatic tumor organoid growth medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>F12+++</td>
<td>N/A</td>
<td>14 mL</td>
</tr>
<tr>
<td>Wnt3a-conditioned medium∗</td>
<td>50%</td>
<td>25 mL</td>
</tr>
<tr>
<td>R-spondin-1-conditioned medium∗</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>mNoggin-conditioned medium∗</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>B27 (50×)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>Nicotinamide (1 M)</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>N-Acetylcysteine (0.5 M)</td>
<td>1.25 mM</td>
<td>125 μL</td>
</tr>
<tr>
<td>hEGF (200 μg/mL)</td>
<td>50 ng/mL</td>
<td>12.5 μL</td>
</tr>
<tr>
<td>hFGF-10 (100 μg/mL)</td>
<td>100 ng/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>hGastrin I (100 μM)</td>
<td>10 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>A83-01 (500 μM)</td>
<td>500 nM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Primocin (50 mg/mL)</td>
<td>100 μg/mL</td>
<td>100 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Once prepared, store 10 mL aliquots at −20°C for up to 3 months.<br> ∗For production and activity testing of conditioned media, see <a href="#bib3">Farin et al. (2012)</a> and <a href="#bib1">Boj et al. (2017)</a>.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>In this protocol, we first detail quantification of OAd genomes as a readout of viral replication inside PDOs 5 days after infection. This is a step that can be performed to prove replication capabilities of designed OAds when fluorescent tags are not incorporated in the viral genome. To compare OAd infectivity and cytotoxicity among different PDOs, we explain two complementary approaches to assess PDO cell viability after viral infection: the CellTiter-Glo® 3D Cell Viability Assay and an organoid formation assay. To perform the organoid formation assay, a sublethal dose of OAd is needed to allow for formation of enough organoids to be counted, so previous cell viability experiments may be needed to establish optimal experimental conditions.</p>
<h3 id="sec3.1">Organoid harvesting for infection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Either to evaluate viral replication or effects on cell viability, organoids are infected as whole entities, since in our experience, when infected as single cells, they have significant problems to recover. Even though variability between biological replicates can be an issue, this can be minimized by performing a cell count using 2 additional wells (see ‘<a href="#before-you-begin">before you begin</a>’) to estimate the number of cells to seed. Prior to harvesting, it is very important to check organoids under the microscope and confirm similar confluency between wells used for infection and seeding. At this point, organoid diameter should not exceed 200 μm, as bigger sizes can lead to higher variability (<a href="#sec7.1">troubleshooting 1</a>). The density to be expected in organoid cultures harvested is usually of about 50,000–100,000 cells/well.</p>
<div class="note">
<span class="note-title">Note:</span> Organoid harvesting and infection times have been calculated for 1 PDO line infected with 2 doses of 2 different OAds to be compared.</div>
<ol>
<li>Thaw Matrigel aliquots needed on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For a 500-μL aliquot to be thawed, 30–45 min are needed, so by the end of the infection protocol Matrigel should already be a pipettable liquid. Calculate the volume of Matrigel needed to seed the experiment to minimize freeze-thaw cycles. See <a href="#tbl1">Table 1</a> for orientation on how much Matrigel will be needed.</div>
<table id="tbl1">
<caption>Table 1. Proportion of PDOs, OAds, and Matrigel in 96- and 24-well plates</caption>
<thead>
<tr>
<th>Format</th>
<th>Component</th>
<th colspan="2">Amount (1 well)</th>
<th colspan="2">Amount per OAd dose (4 wells)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3">96-well plate</td>
<td>PDOs</td>
<td>5000 cells</td>
<td rowspan="2">2 μL</td>
<td>20000 cells</td>
<td rowspan="2">8 μL</td>
</tr>
<tr>
<td>OAd</td>
<td>MOI ⋅ DF/titer</td>
<td>(MOI ⋅ DF/titer) ⋅ 4</td>
</tr>
<tr>
<td>Matrigel</td>
<td colspan="2">8 μL</td>
<td colspan="2">32 μL</td>
</tr>
<tr>
<td rowspan="3">24-well plate</td>
<td>PDOs</td>
<td>25,000 cells</td>
<td rowspan="2">10 μL</td>
<td>100,000 cells</td>
<td rowspan="2">40 μL</td>
</tr>
<tr>
<td>OAd</td>
<td>MOI ⋅ DF/titer</td>
<td>(MOI  DF/titer) ⋅ 4</td>
</tr>
<tr>
<td>Matrigel</td>
<td colspan="2">40 μL</td>
<td colspan="2">200 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">DF = dilution factor. Numbers are given for 1 dose of 1 OAd to infect 1 PDO line.</p>
<ol start="2">
<li>Pre-cool a centrifuge to 4°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All centrifugation steps need to be performed in a chilled centrifuge to ensure that Matrigel is removed from PDOs during washes with basal medium.</div>
<ol start="3">
<li>Prepare 15 mL-tubes with 10 mL of cold basal medium and keep them on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Due to Matrigel density, a maximum of 3 wells of a 24-well plate (containing 150,000–300,000 cells) can be harvested in a single tube. If more organoids need to be collected, split wells into different tubes to perform the first centrifugation step.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Collect organoids used for cell count and organoids used for OAd infection in different tubes (tube 1 or tubes 2 to X, respectively) and process them in parallel. Tubes should always be kept on ice unless otherwise stated.</div>
<ol start="4">
<li>Take an aliquot of basal medium to prepare OAd dilutions. This can be kept at room temperature (20°C–25°C) inside the cell culture hood during organoid processing until use.</li>
</ol>
<ol start="5">
<li>Harvest organoids.<br> <b>Tube 1:</b> 2 wells for cell count <b>Tubes 2-X:</b> 2–3 wells per tube for organoid infection
<ol type="a">
<li>Remove organoid growth medium from the wells without touching the Matrigel dome.</li>
<li>Add 500 μL of basal medium from the corresponding tube to each well to disaggregate the Matrigel dome with the help of a P1000 pipette, and collect the content into the tube.
<div class="note">
<span class="note-title">Note:</span> Either F12+++ or F12+++B basal media can be used for this step. BSA present in F12+++B helps prevent attachment of organoids to the walls of the tube, which can lead to loss of cells during the process.</div>
</li>
<li>Repeat step 5b 2–3 times until removing most of Matrigel remainders inside the wells.</li>
<li>Centrifuge at 100<i>g</i> 5 min at 4°C.</li>
<li>Remove supernatant carefully, trying to remove as much Matrigel as possible but without touching the pellet.
<div class="note">
<span class="note-title">Note:</span> If organoids are not cleanly pelleted under a Matrigel layer, remove medium leaving the Matrigel and add 10 mL of fresh basal medium to repeat centrifugation in step 5d at higher speed (400<i>g</i>, 5 min, 4°C).</div>
</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Disaggregate organoids.
<div class="note">
<span class="note-title">Note:</span> Organoids that will be used for cell count (Tube 1) are dissociated to single cells using TrypLE, while those used for infection with OAd (Tube 2) are mechanically disaggregated. See Figure 4 in <a href="#bib2">Driehuis et al. (2020)</a> for examples of how organoids look like before and after disaggregation.</div>
<ol type="a">
<li>Resuspend the pellet of organoids in Tube 1 (cell count) in 1 mL TrypLE Select by pipetting up and down 10–15 times with a P1000 pipette to facilitate disaggregation.</li>
<li>Incubate Tube 1 for 5 min at 37°C.</li>
<li>During the incubation step, disaggregate organoids in Tube 2 (infection) mechanically. For that, resuspend the pellet in 2 mL of basal medium and use a pre-narrowed, pre-wet (with F12+++B) sterile glass Pasteur pipette to pipette the volume up and down 8–10 times and keep the tube on ice. Try not to create bubbles when pipetting.
<div class="note">
<span class="note-title">Note:</span> For preparation of pre-narrowed glass Pasteur pipettes, after lighting a Bunsen burner, hold the pipette horizontally with both hands and roll it 3–4 times while keeping the tip in the flame. Make sure that the tip is equally narrowed by all sides, and that it has been reshaped to a quarter of its original diameter. Narrowed tips should be sterilized by autoclaving before being used. For pictures on how pre-narrowed pipettes look like, refer to <a href="#bib2">Driehuis et al. (2020)</a>.</div>
</li>
<li>Check the suspension in Tube 1 under a microscope. If big organoids are still visible, pipette up and down a few times and incubate 5 min more at 37°C.</li>
<li>Add cold basal medium up to 13 mL to Tube 1, and up to 10 mL to Tube 2.</li>
<li>Centrifuge at 100<i>g</i> 5 min at 4°C. Like before, repeat the centrifugation step at a higher speed if organoids are not nicely pelleted.
<div class="note">
<span class="note-title">Note:</span> During this time, thaw ready-to-use virus aliquots (stored at −80°C) on ice, and label one 1.5-mL tube for each experimental condition. E.g., for infection of 1 PDO with 2 doses of 2 different OAds (+ non-infected control), prepare 5 tubes.</div>
</li>
<li>Remove the supernatant of both tubes carefully and resuspend each pellet in 1 mL of basal medium.
<div class="note">
<span class="note-title">Note:</span> If more than 3 wells were harvested for infection (more than 1 tube for infection), this is the moment to pool all organoids together in a single tube, maintaining a proportion of 2 wells/mL. It is very important to maintain the same concentration of organoids in Tube 1 and Tube 2 to have a comparable cell number that can be used for calculations of the volume needed for infection and seeding.</div>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Count cells and calculate the volume of organoid suspension needed for OAd infection (V<sub>PDO</sub>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The number of cells to seed in each well and the type of multiwell plate used varies depending on the assay to be performed after infection. For viral genome quantification, we seed 25,000 cells/well in a 24-well plate. For cell viability assessment using CellTiter-Glo 3D or organoid formation assay, we seed 5000 cells/well in a white-walled (with either white or clear bottom) or transparent 96-well plate, respectively.</div>
<div class="note">
<span class="note-title">Note:</span> Doses used for infection vary depending on the experiment to be performed. Generally, doses used to infect PDOs lie in a range between 0.2 MOI and 20 MOI. However, for organoid formation assay a lower dose may be needed to allow for enough structures to grow after infection and re-seeding. Try to add similar volumes of viral dilution to each tube. For this, preparation of several serial dilutions is recommended.</div>
<h3 id="sec3.2">Organoid infection and plating</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min (preparation of OAd dilutions) + 30 min (incubation with OAds) + 15 min (plating)</div>
<p>Since adenoviruses are not able to cross Matrigel, infection of PDOs is performed in the absence of extracellular matrix, which is added right before seeding.</p>
<p>To infect organoids, both cell homogenate-derived supernatants or purified OAds can be used. However, for experiments that will be compared, the virus source used should be the same. Calculate in advance the volume of virus dilution needed for PDO infection (V<sub>OAd</sub>), considering that it should not represent more than 20% of the total volume to seed. For this reason, purified, high-titer (∼1·10<sup>10</sup>–1.10<sup>11</sup> PFU/mL) OAds are preferred.</p>
<ol start="8">
<li>Add the volume calculated in step 7 (V<sub>PDO</sub>) to each of the prepared 1.5-mL tubes.</li>
<li>Bring the volume up to 700–800 μL with basal medium.</li>
<li>Centrifuge at 400<i>g</i> 5 min at 4°C.
<ol type="a">
<li>During this time, prepare virus dilutions in basal medium according to calculated viral titers and doses to be used for infection.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To accurately prepare dilutions, pipette the volume thoroughly to ensure homogeneous distribution of the virus. Try not to pipette volumes smaller than 2 μL.</div>
<ol start="11">
<li>Remove the supernatant carefully, leaving organoids in the volume of basal medium needed to infect. We generally perform infections in a final volume of 10 μL per well to seed.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Remember that basal medium will represent 20% of the total volume to be seeded (the remaining 80% is Matrigel, which is added after infection), that is, 2 μL per well. When calculating how much volume to leave in the tube, also subtract V<sub>OAd</sub> to the total volume to avoid excessive dilution of Matrigel and virus. This will ensure that all PDO infections are performed in the same volume, guaranteeing that the contact between organoids and viral particles will be equivalent, and thus comparable, in all the tested conditions.</div>
<ol start="12">
<li>Add V<sub>OAd</sub> to each 1.5-mL tube.</li>
<li>Mix well by tapping with the finger or by carefully pipetting up and down.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use the same mixing method (tapping or pipetting) for all the Eppendorf tubes in the experiment to further ensure comparable PDO-OAd contacts.</div>
<ol start="14">
<li>Incubate 30 min at 37°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> During incubation, chill tip boxes in the refrigerator (4°C). This will prevent premature Matrigel polymerization inside tips during seeding.</div>
<ol start="15">
<li>Add Matrigel to each tube and mix well by pipetting, avoiding bubble creation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Once Matrigel has been added, put the tubes containing PDOs and OAds on ice to avoid premature polymerization of Matrigel.</div>
<ol start="16">
<li>Seed organoid-containing Matrigel drops (10 μL in 96-well plates, 50 μL in 24-well plates) (<a href="#sec7.3">troubleshooting 2</a>).</li>
<li>Seed drops of 80% Matrigel (diluted in basal medium) in triplicate to read background signals.</li>
<li>Let Matrigel solidify for &gt; 15 min at 37°C (in the incubator).</li>
<li>Add 90 μL (96-well plate) or 450 μL (24-well plate) of organoid growth medium to each well.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> If white-bottom plates are used, a representative well per condition can be seeded in a transparent 96-well plate for visual inspection of infected organoids before cell viability assessment (<a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1233-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Bright-field images of PDAC PDOs infected with OAd</div>
<p>Representative images of PDAC organoids infected with increasing doses of an oncolytic adenovirus (5 days after infection). Arrowheads indicate dead organoids. Scale bar, 200 μm.</p>
</figcaption>
</figure>
<h3 id="sec3.3">Viral yield quantification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days (incubation with OAds) + 30 min (organoid harvesting) + 1 h (DNA isolation) + 2.5 h (qPCR plate preparation + run)</div>
<p>OAd replication inside organoids can be assessed through the quantification of viral genomes by qPCR amplification of the adenoviral hexon gene in infected organoids. To this end, organoids infected with known doses from OAd stock (passage 1, P1) are grown for 5 days and lysed by freeze and thaw to release viral particles. After centrifugation of the lysate, a fraction of the resulting virus-containing supernatant is used to infect new organoids (passage 2, P2) and proceed as with P1.</p>
<p>For viral yield quantification, supernatants from both P1 and P2 are analyzed by qPCR for the presence of viral genomes. The detection of viral genomes in the P2 supernatant is a further proof of viral replication in organoids. Before starting, and for absolute quantification of OAd genomes, a standard curve of known viral genome copies must be generated. See ‘<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>’ for calculations and instructions on how to build the standard curve.</p>
<p>For this procedure, organoids should be infected in a 24-well format to guarantee obtention of enough DNA for qPCR amplification.</p>
<ol start="20">
<li>Prepare a cooling bath by adding dry ice and ethanol to an appropriate container.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Alternatively, organoids can be frozen by storing them in a −80°C freezer (even though the freezing process will be slower).</div>
<ol start="21">
<li>Harvest each well of infected organoids (passage 1, P1) in a 2-mL tube, and 1 well of non-infected organoids as explained in the section ‘<a href="#sec3.1">organoid harvesting for infection</a>’.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In the case of P1 organoids, for the washing steps, bring the volume up to 1.9 mL with cold basal medium (instead of 10 mL used when harvesting organoids in 15-mL tubes). In addition, after the 1<sup>st</sup> centrifugation step, remove all the supernatant carefully to resuspend the pellet in the basal medium.</div>
<ol start="22">
<li>After the 2<sup>nd</sup> centrifugation step, remove the supernatant carefully and resuspend the pellet in 150 μL of cold basal medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Organoids infected with different OAds should be resuspended in the same volume to ensure that viral genome concentrations will be comparable.</div>
<ol start="23">
<li>Perform 3 freeze-thaw cycles to release viral particles from cells.
<ol type="a">
<li>To freeze, put the tubes in the cooling bath.</li>
<li>To thaw, submerge the tubes partially in a 37°C water bath with gentle agitation.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Pelleted organoids can be left frozen at −80°C (before 1<sup>st</sup> freeze-thaw cycle) to continue the protocol in the following weeks.</div>
<ol start="24">
<li>Centrifuge at 1000<i>g</i> 5 min to remove cell debris.</li>
<li>Transfer 120 μL of P1 supernatant to a new 1.5-mL tube. Not taking the whole volume of supernatant will prevent the transfer of any cell debris. From this volume of supernatant:
<ol type="a">
<li>Use 10 μL to infect harvested, non-infected PDOs (P2) as explained in steps 8–19, treating this supernatant as V<sub>OAd</sub>. Add 10 μL of basal medium and 80 μL of Matrigel to perform a 1:2 split, seeding organoids in 2 wells of a 24-well plate.</li>
<li>Save 25 μL of P1 supernatant to perform DNA extraction (step 27).</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Supernatants can be frozen at −20°C or −80°C to isolate DNA from P1 and P2 organoids in parallel.</div>
<ol start="26">
<li>After 5 days, repeat the process described in steps 20–25 to obtain 120 μL of supernatant containing viral particles produced by P2 PDOs.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> The supernatant derived from P2 organoids can also be used to infect new PDOs (10 μL per well of PDOs growing in a 24-well plate) and obtain viral particles released in subsequent passages.</div>
<ol start="27">
<li>Isolate DNA from P1 and P2 supernatants using the Blood DNA Isolation Mini Kit following the manufacturer's protocol. Since the starting volume required for this kit is 200 μL, bring the supernatant sample up to this volume by adding PBS.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Isolated DNA can be stored at 4°C (a few days) or −20°C (months) for further qPCR analysis.</div>
<ol start="28">
<li>Quantification of viral genomes by qPCR amplification of hexon gene.
<ol type="a">
<li>Prepare a qPCR master mix and add 9 μL to each well of a 384-well microplate. For each sample:
<p class="chem"><b>qPCR master mix:</b> 5 μL SYBR Green 2X + 0,3 μL Hexon Primer Mix 10 μM (Fw + Rv) + 3,7 μL Nuclease-Free H<sub>2</sub>O</p>
<div class="note">
<span class="note-title">Note:</span> Extra master mix volume should be prepared, considering that each sample needs to be run in triplicate. As a rule of thumb, make a master mix for 1 extra well for every 9 wells to be prepared.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Protect the master mix from light with aluminum foil and keep it on ice while working.</div>
</li>
<li>Add 1 μL of isolated DNA into corresponding wells, trying to minimize bubble formation.</li>
<li>Carefully seal the microplate with an optical adhesive film.
<div class="optional">
<span class="optional-title">Optional:</span> Centrifuge the microplate briefly to ensure that the volume is correctly placed at the bottom of the wells.</div>
</li>
<li>Insert the microplate in a qPCR system (e.g., ViiA 7 Real-Time PCR System).</li>
<li>Assign samples and targets to corresponding wells and start the qPCR program:
<table id="undtbl4">
<thead>
<tr>
<th colspan="4">qPCR cycling conditions</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>10 sec</td>
<td rowspan="3">40</td>
</tr>
<tr>
<td>Annealing</td>
<td>58°C</td>
<td>30 sec</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>20 sec</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td rowspan="3">(Melt curve)</td>
<td>95°C</td>
<td>5 sec</td>
<td rowspan="3">-</td>
</tr>
<tr>
<td>65°C</td>
<td>1 min</td>
</tr>
<tr>
<td>97°C</td>
<td>15 sec</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
</ol>
<ol start="29">
<li>Quantify OAd genomes as explained in ‘<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>’. Detection of OAd genomes in the different passages is indicative of OAd replication within cancer cells (<a href="#fig2">Figure 2</a>).</li>
</ol>
<h3 id="sec3.4">Cell viability assessment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days (incubation with OAds) + 1 h (CellTiter-Glo 3D assay) or 24 h (Organoid formation assay)</div>
<p>Adenoviruses complete their lytic infection cycle in 36 h. To observe effects on cell viability derived from the infection of organoids by newly formed viral particles, organoids are incubated with OAds for 5 days (120 h), allowing for the completion of 3 viral cycles (<a href="#fig3">Figure 3</a>).</p>
<ol start="30">
<li>Cell viability (CellTiter-Glo® 3D Cell Viability Assay)
<div class="note">
<span class="note-title">Note:</span> The protocol is based on the manufacturer's instructions, with minor modifications.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Since this is an ATP-based assay, microbial contamination may alter results. For this reason, steps 30b–e should be performed in a cell culture hood and CellTiterGlo-3D should be handled with care. White-walled, 96-well plates must be used to reduce luminescence crosstalk.</div>
<ol type="a">
<li>Thaw CellTiter-Glo® 3D reagent at 4°C (for 2–48 h).</li>
<li>Equilibrate the plate/s containing infected organoids, CellTiter-Glo® 3D reagent and basal medium (F12+++) for 30 min at room temperature (20°C–25°C).
<div class="optional">
<span class="optional-title">Optional:</span> During this time, take brightfield pictures of organoids seeded in the transparent plate using an inverted microscope.</div>
</li>
<li>Prepare a mix of basal medium and CellTiter-Glo® 3D at a 1:1 ratio.</li>
<li>Remove medium from the wells of the plate containing infected organoids using a glass Pasteur pipette equipped with a 10 μL-tip, taking care not to disturb the Matrigel domes. Change the tip between different experimental conditions.
<div class="optional">
<span class="optional-title">Optional:</span> If using white-walled, clear-bottom plates, this is the moment to carefully stick the sealing film to the bottom of the plate.</div>
</li>
<li>Transfer the mix to a sterile plastic container and add 100 μL of it to each well using a multichannel pipette.</li>
<li>Mix the plate/s vigorously (450 rpm) in a microplate shaker for 5 min at room temperature (20°C–25°C) to lyse organoids.</li>
<li>Incubate the plate for 25 min at room temperature (20°C–25°C).</li>
<li>Quantify luminescence in a microplate reader (<a href="#sec7.5">troubleshooting 3</a>).
<div class="note">
<span class="note-title">Note:</span> We use a Biotek Synergy HT for quantification, programmed with the following features: Integration Time 0.1 s, Plate Costar 96w white opaque, Sensitivity 100.</div>
</li>
<li>Subtract background (Matrigel) luminescence values and normalize values of infected organoids to non-infected organoids, which correspond to 100% viability (<a href="#fig4">Figure 4</a>, <a href="#sec7.7">troubleshooting 4</a>).
<div class="note">
<span class="note-title">Note:</span> See <a href="#tbl2">Table 2</a> for approximate luminescence values expected under described conditions.</div>
</li>
</ol>
</li>
</ol>
<ol start="31">
<li>Organoid formation assay
<ol type="a">
<li>Harvest organoids as described in the ‘<a href="#sec3.1">organoid harvesting for infection</a>’ section, performing gentle mechanical disaggregation after the first centrifugation step.
<div class="note">
<span class="note-title">Note:</span> Pool 2 wells of every experimental condition into one tube. In this case, since they are seeded in a 96-well plate, smaller tubes (1.5 mL) can be used to harvest organoids. Wash wells carefully with basal medium to recover all organoids.</div>
</li>
<li>After the second centrifugation step, carefully remove the supernatant to leave organoids in a final volume of 8 μL.</li>
<li>Add 32 μL of Matrigel, pipetting up and down to mix.</li>
<li>Seed 10 μL drops in a new 96-well plate, performing a 1:2 split (if 2 wells were collected, seed 4).</li>
<li>Let Matrigel drops solidify for &gt; 15 min at 37°C.</li>
<li>Add 90 μL of organoid growth medium in each well.</li>
<li>Incubate at 37°C for 24 h.</li>
<li>After 24 h, take brightfield pictures of the wells seeded using an inverted microscope.
<div class="note">
<span class="note-title">Note:</span> Take several pictures that include different fields so that the whole well can be observed for organoid count.</div>
</li>
<li>Count organoids in the wells by visually inspecting the pictures (<a href="#sec7.9">troubleshooting 5</a>).
<div class="note">
<span class="note-title">Note:</span> Only organoids that have not been infected with OAd, or with a low viral progeny inside, will be able to reassemble and form a new, round-shaped organoid. For manual organoid count, we consider as viable organoids those structures that are rounded and have a diameter bigger than 20 μm (<a href="#fig5">Figure 5</a>). Therefore, organoid count can be interpreted as a readout of cell viability, which is inversely correlated with sensitivity to the OAd.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1233-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Comparison of sensitivity to oncolytic adenoviruses between PDOs</div>
<p>Two different PDOs were infected with 2 or 20 MOI of two different oncolytic adenoviruses (OAds) and assessed for viability using CellTiter-Glo 3D 5 days after infection. Data are represented as mean ± SEM (n ≥ 3). Luminescence units were normalized to values of non-infected (mock) organoids.</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1233-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Organoid formation assay for survival assessment after PDO infection</div>
<p>(A) Representative bright-field images of non-infected (mock) or OAd-infected PDOs (0.02 MOI for 5 days) and re-seeded at a 1:2 split ratio. Examples of organoids (round-shaped, &gt; 20 μm diameter) formed 24 h after reseeding are indicated with arrowheads.</p>
<p>(B) Quantification of organoids formed 24 h after re-seeding. Data are represented as mean ± SEM (n ≥ 5).</p>
</figcaption>
</figure>
<table id="tbl2">
<caption>Table 2. Example of raw luminescence data of a CellTiter-Glo® 3D cell viability assay</caption>
<thead>
<tr>
<th rowspan="2">Technical replicate</th>
<th rowspan="2">Negative control (mock)</th>
<th colspan="3">OAd 1</th>
<th colspan="3">OAd 2</th>
<th rowspan="2">Blank (Matrigel)</th>
</tr>
<tr>
<th>0.2 MOI</th>
<th>2 MOI</th>
<th>20 MOI</th>
<th>0.2 MOI</th>
<th>2 MOI</th>
<th>20 MOI</th>
</tr>
</thead>
<tbody>
<tr>
<td>#1</td>
<td>1065780</td>
<td>1140390</td>
<td>800990</td>
<td>210150</td>
<td>954530</td>
<td>726390</td>
<td>94080</td>
<td>40</td>
</tr>
<tr>
<td>#2</td>
<td>1205090</td>
<td>1102890</td>
<td>788530</td>
<td>212610</td>
<td>1120560</td>
<td>638680</td>
<td>102600</td>
<td>10</td>
</tr>
<tr>
<td>#3</td>
<td>1296840</td>
<td>1343930</td>
<td>781060</td>
<td>222520</td>
<td>1302120</td>
<td>639860</td>
<td>91450</td>
<td>50</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol describes the steps to infect pancreatic tumor PDOs with OAds, as well as methods to analyze viral replication and assess PDO sensitivity to therapeutic viruses. If the OAd genome encodes a fluorescent protein (e.g., EGFP), spreading of the virus to adjacent cells across organoid structures should be observed within 1–4 days after infection (<a href="#fig1">Figure 1</a>). Alternatively, viral replication can be confirmed by quantification of viral genomes in infected organoids. After a 5-day incubation period with OAds, which allows for the completion of 2 viral lytic cycles, we generally obtain viral genome concentrations that range between 10<sup>5</sup>-10<sup>6</sup> copies/μL for organoids initially infected at a dose of 2 MOI. If the supernatant of infected organoids (P1) is used to newly infect organoids, similar results should be obtained (P2), confirming that viral particles can be efficiently produced within PDOs (<a href="#fig2">Figure 2</a>).</p>
<p>We suggest 2 different approaches to assess PDO sensitivity to OAds. Five days after infection, PDOs can be either analyzed for cell viability using CellTiter-Glo3D (<a href="#fig4">Figure 4</a>), or harvested and re-seeded to evaluate organoid formation after 24 h (<a href="#fig5">Figure 5</a>).</p>
<p>Since oncolytic viruses should selectively replicate in and kill tumor cells, infection of normal organoids should not lead to viral replication or diminished cell viability. Comparison of viral replication and OAd-induced cell death between tumor and normal organoids can be found in <a href="#bib6">Raimondi et al. (2020)</a>.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">Quantification of OAd genomes from qPCR data</h3>
<p>To quantify OAd genomes present in supernatants derived from infected organoids, it is necessary to first build a standard curve of known genomic DNA copies. In our case, this curve covers a range from 10<sup>2</sup> to 10<sup>7</sup> copies of viral genomes.</p>
<ol>
<li>Quantify a sample of viral DNA in a spectrophotometer.</li>
<li>Calculate the weight of 1 copy of OAd genome with the following equation:</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>G</mi> <mi>e</mi> <mi>n</mi> <mi>o</mi> <mi>m</mi> <mi>e</mi> <mspace></mspace> <mi>m</mi> <mi>a</mi> <mi>s</mi> <mi>s</mi> <mo>=</mo> <mi>n</mi> <mspace></mspace> <mrow> <mo>(</mo> <mrow> <mi>g</mi> <mi>e</mi> <mi>n</mi> <mi>o</mi> <mi>m</mi> <mi>e</mi> <mspace></mspace> <mi>s</mi> <mi>i</mi> <mi>z</mi> <mi>e</mi> </mrow> <mo>)</mo> </mrow> <mo>·</mo> <mn>1,096</mn> <mo>·</mo> <msup> <mn>10</mn> <mrow> <mo>−</mo> <mn>21</mn> </mrow> </msup> <mfrac> <mi>g</mi> <mrow> <mi>b</mi> <mi>p</mi> </mrow> </mfrac> <mo>·</mo> <msup> <mn>10</mn> <mn>12</mn> </msup> <mfrac> <mrow> <mi>p</mi> <mi>g</mi> </mrow> <mi>g</mi> </mfrac> <mo>=</mo> <mi>X</mi> <mspace></mspace> <mi>p</mi> <mi>g</mi> </mrow> </math></p>
<ol start="3">
<li>Dilute the sample until obtaining a concentration of 10<sup>7</sup> copies/μL and prepare serial 1:10 dilutions containing from 10<sup>2</sup> to 10<sup>7</sup> copies.</li>
<li>Amplify the hexon gene by qPCR as explained in step 28, loading 1 μL of gDNA to each reaction (4 technical replicates).</li>
<li>Represent CT (cycle threshold) values versus DNA copies in Excel or GraphPad Prism.</li>
<li>Transform DNA concentration to logarithmic values and fit the curve using linear regression (<a href="#fig6">Figure 6</a>).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1233-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Example of a standard curve for viral genome quantification by qPCR</div>
<p>The viral hexon gene was amplified by qPCR in samples with known DNA copies (from 10<sup>2</sup> to 10<sup>7</sup>). Results expressed as CT (cycle threshold) for each sample were represented and fitted to a linear regression model. The resulting equation is shown in a green square. Data are represented as mean ± SEM (n = 4).</p>
</figcaption>
</figure>
</li>
</ol>
<ol start="7">
<li>In Excel, use the equation obtained to interpolate qPCR results derived from PDO infection in Excel and calculate viral genomes/μL (<a href="#tbl3">Table 3</a>; <a href="#fig2">Figure 2</a>):
<div class="note">
<span class="note-title">Note:</span> Since CT values represent the number of PCR cycles required for the fluorescent signal to exceed a threshold (i.e., background), the slope of the equation obtained should be negative.</div>
<ol type="a">
<li>Calculate the mean and standard deviation (SD) of the technical replicates’ CT values.
<div class="note">
<span class="note-title">Note:</span> SD of technical replicates should be &lt; 0.3 for results to be considered.</div>
</li>
<li>Calculate the mean and standard deviation (SD) of the technical replicates’ CT values.</li>
<li>Using the equation obtained, calculate the DNA copies (log<sub>10</sub>DNA) corresponding to the mean CT value (<i>y</i>).</li>
<li>Transform log<sub>10</sub>DNA to DNA copies (from logarithmic to linear scale).</li>
<li>Plot results in Excel or GraphPad Prism.</li>
</ol>
</li>
</ol>
<table id="tbl3">
<caption>Table 3. Example of viral genome quantification by qPCR</caption>
<thead>
<tr>
<th>Replicate</th>
<th>CT value</th>
<th>Mean CT</th>
<th>SD</th>
<th>LogDNA</th>
<th>OAd genomes/μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>#1</td>
<td>20.807</td>
<td rowspan="3">20.949</td>
<td rowspan="3">0.247</td>
<td rowspan="3">4.582</td>
<td rowspan="3">38205.46</td>
</tr>
<tr>
<td>#2</td>
<td>20.806</td>
</tr>
<tr>
<td>#3</td>
<td>21.234</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Even though seeding of PDOs as whole entities is necessary for infection with OAds, this leads to higher variability compared to seeding single cells. If inconsistent seeding densities are observed after plating, a relatively high number of replicates may be required (n ≥ 4). As described in this protocol, we generally keep 2 wells (of a 24-well plate) for cell count and infect an approximate number of organoids by extrapolation of this cell count, even though other strategies to consistently seed whole organoids have been reported elsewhere (<a href="#bib5">Ooft et al., 2019</a>). Importantly, cell numbers described here could vary depending on growth rates of individual PDO lines, so they may need to be adjusted before starting screening of OAds by seeding increasing cell densities. Proper maintenance of organoids and splitting them at a constant ratio and frequency are also key points for consistent results.</p>
<p>Due to time-consuming steps, like organoid harvesting and manual seeding of Matrigel drops, this method is limited to low-throughput screenings of OAds. Comparing more than 4 viruses at a time may become too laborious, especially if testing more than one dose. If a higher throughput is needed, automated seeding in a semi-solid format (5%–10% of Matrigel) might be used.</p>
<p>This protocol only describes methods to compare relative sensitivities to OAds among different PDOs in a preclinical setting. While being a very important aspect for virotherapy, OAd selectivity is not assessed here. Nevertheless, it can be easily evaluated by any of the methods described above by infecting normal and tumor-tissue derived PDOs in parallel. Since OAds are designed to complete their lytic cycle in a tumor background, they should therefore not be able to replicate in normal organoids and kill them. Experiments showing OAd selectivity can be found in <a href="#bib6">Raimondi et al. (2020)</a>. To ensure specific replication of OAds inside cancer cells, it is also recommended to check that pathways on which OAd design is based are active (e.g., by qPCR analysis of downstream effectors), especially if a dose-response decrease in cell viability is not observed in tumor PDOs.</p>
<p>CellTiter-Glo 3D evaluates cell viability based on ATP levels. Other viability assays based on the measurement of metabolic activity, such as the MTT assay, can also be used (<a href="#bib4">Grabinger et al., 2014</a>; <a href="#bib6">Raimondi et al., 2020</a>). Moreover, complementary techniques to inspect cell death induced by viral infection could be applied.</p>
<p>This protocol has only been applied to pancreatic ductal adenocarcinoma (PDAC) organoids. However, we anticipate that it could be applied to patient-derived organoids derived from many solid organ malignancies.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Organoids are too big at the day of harvesting and infection (<a href="#sec3.1">organoid harvesting for infection</a>).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Harvest organoids as explained in steps 1–5 and disaggregate them enzymatically by incubating with TrypLE Select for 5–10 min at 37°C (1 mL for every 3 wells of a 24-well plate). Vigorously pipette up and down (about 10 times) using a narrowed Pasteur pipette every 2–3 min and check organoids under the microscope to ensure that they do not become a single-cell suspension, since this may lead to poor recovery from disaggregation.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Matrigel/organoid suspension hardens before seeding all PDO wells (step 16).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Always keep tubes on ice when working with Matrigel and during seeding. Chill pipette tips inside the refrigerator, too.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Too low, or too high, luminescence levels are obtained after CellTiter-Glo® 3D Cell Viability Assay (step 30).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>If luminescence levels are too low to compare between experimental conditions, incubation time with CellTiter-Glo® 3D reagent can be extended to maximize organoid lysis. Alternatively, they may indicate that a higher number of cells is needed to detect differences in cell viability (likewise, viral dose should be reduced). In case luminescence levels are too high, the integration time can be reduced. However, if biological replicates show substantial differences in luminescence signals, the culture should be examined for potential microbial contaminations. It is recommended to set up a protocol for luminescence reading in the multiplate reader’s software before starting screenings.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Variability among technical replicates is very high (step 30).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>To ensure homogeneous seeding, it is very important to thoroughly pipette the organoid suspension before dividing the volume into different tubes to infect, especially when seeding is performed in a 96-well format. Pipette well the OAd-PDO-Matrigel mixture right before seeding to ensure that the percentage of Matrigel is similar in all wells. Try to seed Matrigel drops in such a way that they do not come in contact with the walls of the wells, since this may lead to irregular growth of organoids (<a href="#fig7">Figure 7</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1233-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Uneven seeding of Matrigel drops</div>
<p>Red crosses indicate Matrigel drops in contact with the well’s walls.</p>
</figcaption>
</figure>
<h3 id="sec7.9">Problem 5</h3>
<p>There are almost no organoids in the wells, or no big organoids (with a diameter &gt; 20 μm) are observed (step 31).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Some organoids are always lost during harvesting and seeding. Since the organoid formation assay is performed in a 96-well format, too many organoids could be lost during the process of reseeding. To compensate for that, harvest more wells for re-seeding or perform mechanical disaggregation with a p200 pipette pre-wet in F12+++B medium. The OAd dose may also need to be reduced to ensure that enough organoids are alive for count after re-seeding.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Boj, S.F., Vonk, A.M., Statia, M., Su, J., Dekkers, J.F., Vries, R.R.G., Beekman, J.M., and Clevers, H. (2017). Forskolin-induced swelling in intestinal organoids: an in vitro assay for assessing drug response in cystic fibrosis patients. J. Vis. Exp. <i>120</i>, 55159. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref1">View at publisher</a></p>
<p id="bib2">Driehuis, E., Gracanin, A., Vries, R.G.J., Clevers, H., and Boj, S.F. (2020). Establishment of pancreatic organoids from normal tissue and tumors. STAR Protoc. <i>1</i>, 100192. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref2">View at publisher</a></p>
<p id="bib3">Farin, H.F., Van Es, J.H., and Clevers, H. (2012). Redundant sources of Wnt regulate intestinal stem cells and promote formation of paneth cells. Gastroenterology <i>143</i>, 1518-1529. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref3">View at publisher</a></p>
<p id="bib4">Grabinger, T., Luks, L., Kostadinova, F., Zimberlin, C., Medema, J.P., Leist, M., and Brunner, T. (2014). Ex vivo culture of intestinal crypt organoids as a model system for assessing cell death induction in intestinal epithelial cells and enteropathy. Cell Death Dis. <i>5</i>, e1228. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref4">View at publisher</a></p>
<p id="bib5">Ooft, S.N., Weeber, F., Dijkstra, K.K., McLean, C.M., Kaing, S., van Werkhoven, E., Schipper, L., Hoes, L., Vis, D.J., van de Haar, J., et al. (2019). Patient-derived organoids can predict response to chemotherapy in metastatic colorectal cancer patients. Sci. Transl. Med. <i>11</i>, 1-10. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref5">View at publisher</a></p>
<p id="bib6">Raimondi, G., Mato-Berciano, A., Pascual-Sabater, S., Rovira-Rigau, M., Cuatrecasas, M., Fondevila, C., Sánchez-Cabús, S., Begthel, H., Boj, S.F., Clevers, H., et al. (2020). Patient-derived pancreatic tumour organoids identify therapeutic responses to oncolytic adenoviruses. EBioMedicine <i>56</i>, 102786. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref6">View at publisher</a></p>
<p id="bib7">Rovira-Rigau, M., Raimondi, G., Marín, M.Á., Gironella, M., Alemany, R., and Fillat, C. (2019). Bioselection reveals miR-99b and miR-485 as enhancers of adenoviral oncolysis in pancreatic cancer. Mol. Ther. <i>27</i>, 230-243. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref7">View at publisher</a></p>
<p id="bib8">Villanueva, E., Navarro, P., Rovira-Rigau, M., Sibilio, A., Méndez, R., and Fillat, C. (2017). Translational reprogramming in tumour cells can generate oncoselectivity in viral therapies. Nat. Commun <i>8</i>, 14833. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00723-1/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Cristina Fillat (<a href="mailto:cfillat@clinic.cat">cfillat@clinic.cat</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze any datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants from the Spanish Ministerio de Ciencia e Innovación BIO2017-89754-C2-2R and PID2020-119692RB-C22, co-financed by <a href="https://doi.org/10.13039/501100008530">Fondo Europeo de Desarrollo Regional</a> (FEDER); from the <a href="https://doi.org/10.13039/100008666">Fundació La Marató de TV3</a> (301/C/2019); and from the <a href="https://doi.org/10.13039/100008061">Fundación Mutua Madrileña</a> (AP172132019), with partial support from the <a href="https://doi.org/10.13039/501100002809">Generalitat de Catalunya</a> SGR17/861. CIBER de Enfermedades Raras is an initiative of the ISCIII. We also acknowledge the support of the CERCA Programme/Generalitat de Catalunya. This work was developed at the Centro Esther Koplowitz, Barcelona, Spain. S.P.-S. is recipient of an FPU predoctoral contract (FPU19/05036) from the Spanish Ministerio de Universidades. G.R. has been recipient of an FPI predoctoral contract (BES-2015-071612). A.M.-B. was the recipient of an FPU predoctoral contract (AP2010-5297). Graphical abstract was created using <a href="https://app.biorender.com/">Biorender.com</a>.</p>
<h3>Author contributions</h3>
<p>Conceptualization, S.P.-S, G.R, A.M.-B. and C.F.; writing – original draft, S.P.-S.; writing – review &amp; editing, G.R. and C.F.; investigation, S.P.-S, G.R, A.M.-B.; methodology, G.R., S.P.-S., and A.M.-B.; resources, F.A.; funding acquisition, C.F. and E.C.V.; supervision, C.F.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>