<article>
<section>
<h1 id="header">Analysis of local protein accumulation kinetics by live-cell imaging in yeast systems</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Hiroki Okada,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Brittany MacTaggart,<sup><a href="#aff1">1</a></sup> and Erfei Bi<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Cell and Developmental Biology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-6058, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:okad@pennmedicine.upenn.edu">okad@pennmedicine.upenn.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:ebi@pennmedicine.upenn.edu">ebi@pennmedicine.upenn.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100733">10.1016/j.xpro.2021.100733</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Microscopy-based analysis of protein accumulation at a given subcellular location in real time provides invaluable insights into the function of a protein in a specific process. Here, we describe a detailed protocol for determining protein accumulation kinetics at the division site in the budding yeast <i>Saccharomyces cerevisiae</i> and fission yeast <i>Schizosaccharomyces pombe</i>. This protocol can be adapted for the analysis of any protein involved in any process as long as the protein is localized to a discrete region of the cell.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00440-8/sref16">Okada et al. (2021)</a> and <a href="http://refhub.elsevier.com/S2666-1667(21)00440-8/sref17">Okada et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Critical factors for live-cell imaging in yeast systems</li>
<li>Detailed protocol for analyzing protein localization kinetics</li>
<li>Kinetic analysis of myosin-II accumulation at the division site</li>
<li>Limitations of kinetic analysis in uncovering biological mechanisms</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps used for the kinetic analysis of more than twenty different cytokinetic proteins at the division site in the budding yeast <i>S. cerevisiae</i> (<a href="#bib16">Okada et al., 2021</a>). The same protocol was also used to define the roles of myosin-II isoforms in cytokinesis under normal and stress conditions in the fission yeast <i>S. pombe</i> (<a href="#bib17">Okada et al., 2019</a>).</p>
<h3 id="sec1.1">Strain construction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 weeks</div>
<ol>
<li>Introduce a cell cycle marker.
<ol type="a">
<li>The kinetic profile of a protein during the cell cycle represents data from many cells, so a cell cycle marker for aligning the individual curves of protein accumulation is essential. A tight alignment indicates data reliability. The cell cycle marker must: (1) represent a definitive cell cycle event, (2) provide an unambiguous time point for alignment, and (3) have no adverse effect on the biological process of interest. Choose an appropriate marker for your analysis.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We routinely use tubulin tagged with a fluorescent protein (FP) (e.g., RFP-Tub1 in budding yeast and mCherry-Atb2 in fission yeast) as our cell cycle marker and use the time of spindle breakage to indicate the onset of cytokinesis (<a href="#fig1">Figure 1</a>) (<a href="#bib16">Okada et al., 2021</a>; <a href="#bib17">Okada et al., 2019</a>).</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The spindle morphogenesis during the cell cycle in budding yeast and fission yeast</div>
<p>The localization of (A) mScarlet-Tub1 in budding yeast or (B) mCherry-Atb2 in fission yeast. Montages of a representative cell from the yeast strain YEF10879 or YCW0130 are created from selected frames of time-lapse series taken with a 2-min or 1.5-min interval, respectively. The gray dotted line represents the cell outline. The images defined by the orange boxes represent spindle breakage.</p>
</figcaption>
</figure>
<ol start="2">
<li>Select an FP for tagging the protein.
<ol type="a">
<li>Although many FPs are available (<a href="#bib4">Bialecka-Fornal et al., 2016</a>), we prefer the classic EGFP due to its excellent brightness and photostability (<a href="#bib6">Botman et al., 2019</a>). Alternatively, GFP<sup>E</sup><sup>nvy</sup> can be used, as it is also bright, fast-folding, and photostable (<a href="#bib20">Slubowski et al., 2015</a>). In our hands, GFP<sup>E</sup><sup>nvy</sup> performed better than EGFP under certain conditions, such as high temperature (37°C). We use yeast codon-optimized FPs (e.g., yEGFP and yoEGFP) to avoid an expression bottleneck caused by rare-codon usage (<a href="#bib14">Lee et al., 2013</a>).</li>
<li>For dual-color imaging with EGFP, we recommend ymScarlet-I, a bright and fast-folding RFP (<a href="#bib5">Bindels et al., 2017</a>).</li>
</ol>
</li>
<li>Choose the location and method for tagging the endogenous gene.
<ol type="a">
<li>The N- and C-termini of proteins are usually exposed at the surface of their structures and are more distant from their structural cores (<a href="#bib12">Jacob and Unger, 2007</a>). This is why FP tags are usually placed at the N- or C-termini of the proteins. Due to its simplicity, C-terminal tagging is routinely used in budding yeast (<a href="#bib11">Huh et al., 2003</a>; <a href="#bib15">Longtine et al., 1998</a>). Another advantage is that the tagged gene is still under the control of its endogenous promoter, so protein expression is unlikely to be affected.</li>
<li>For C-terminal tagging, the simple and efficient PCR-based method has been widely used (<a href="#bib15">Longtine et al., 1998</a>). Various methods have also been developed to allow for N-terminal tagging with native promoter control or tag insertion within an open reading frame (ORF), but these can be more time-consuming (<a href="#bib10">Hawkins et al., 2003</a>; <a href="#bib24">Yofe et al., 2016</a>). CRISPR/Cas9-based methods have become increasingly popular because they allow for tagging multiple genes simultaneously and offer more choices for tagging locations (<a href="#bib7">Buchmuller et al., 2019</a>; <a href="#bib23">Torres-Garcia et al., 2020</a>).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For convenience and simplicity, low-copy or centromere plasmids have often been used for protein localization analysis. However, this approach is not recommended for protein kinetic analysis, as the plasmid copy number varies among individual cells, which causes a variation in the level of protein expression that might lead to changes in kinetic behavior.</div>
<ol start="4">
<li>Check for the functionality of the tagged protein.
<ol type="a">
<li>The functionality of a protein tagged with an FP can be examined by various methods, including analyzing the growth rate, morphology, or chemical sensitivity of cells carrying the tagged protein. Choose the appropriate method for your analysis. <a href="#sec7.1">Troubleshooting 1</a>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is essential to make sure that tagging does not compromise protein function. In the study of cytokinetic protein kinetics (<a href="#bib16">Okada et al., 2021</a>), we co-expressed Mlc2-mApple along with the GFP-tagged cytokinetic protein in every strain, which allowed us to monitor the actomyosin ring and to ensure that the GFP-tagged proteins did not perturb actomyosin ring constriction.</div>
<h3 id="sec1.2">Yeast media preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="5">
<li>YPD agar medium (1 L, rich medium for budding yeast)
<ol type="a">
<li>Prepare two mixtures in separate flasks. Mixture 1
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast extract</td>
<td>10 g</td>
</tr>
<tr>
<td>Bacto peptone</td>
<td>20 g</td>
</tr>
<tr>
<td>Agar</td>
<td>20 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Add distilled water up to 900 mL.<br> Put a magnetic stir bar in the flask.</p>
Mixture 2
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glucose</td>
<td>20 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Add distilled water up to 100 mL.</p>
</li>
<li>Autoclave both mixtures (at 121°C for 20 min).</li>
<li>Add mixture 2 to mixture 1 with stirring by a magnetic stir bar.</li>
<li>Leave medium at 22°C–24°C to cool off (until 50°C–55°C) and pour into Petri dishes (∼30 plates).
<div class="note">
<span class="note-title">Note:</span> At the ideal temperature, the medium is not solidified but not too hot to hold the flask by hand.</div>
<div class="note">
<span class="note-title">Note:</span> During pouring, if air bubbles form on the surface of the unsolidified medium, instant searing by a burner should eliminate them.</div>
</li>
<li>Leave plates to dry for 1–2 days at 22°C–24°C and store them at 4°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During storage, place plates in a plastic bag to prevent them from drying out.</div>
<div class="note">
<span class="note-title">Note:</span> The medium can be stored at 4°C for at least 6 months without a noticeable change in quality.</div>
<ol start="6">
<li>SC liquid medium (1 L, minimal medium for budding yeast)
<ol type="a">
<li>Prepare two mixtures in separate glass bottles. Mixture 1
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast nitrogen base without amino acids and with ammonium sulfate</td>
<td>6.7 g</td>
</tr>
<tr>
<td>Amino acid mix (<a href="#tbl1">Table 1</a>)</td>
<td>2 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Add distilled water up to 900 mL.</p>
<div class="note">
<span class="note-title">Note:</span> Mix all the reagents by grinding with a mortar and pestle.</div>
<div class="note">
<span class="note-title">Note:</span> The powder mix can be stored in a dark place at 22°C–24°C for at least one year without a noticeable change in quality.</div>
<div class="optional">
<span class="optional-title">Optional:</span> This step can be skipped if you purchase a premixed powder (#1300-030, Sunrise Science, TN, USA).</div>
<div class="optional">
<span class="optional-title">Optional:</span> If your protein of interest is tagged with GFP and exposed to extracellular space, the GFP signal will be quenched due to the acidity of SC medium (pH ∼4.5) (<a href="#bib8">Colman-Lerner et al., 2001</a>; <a href="#bib22">Thomas et al., 2018</a>). To resolve this issue, adjust the mixture to pH 7 with sodium hydroxide solution.</div>
<table id="tbl1">
<caption>Table 1. Recipe for an amino acid mix (93.4 g)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>4-aminobenzoic acid</td>
<td>0.4 g</td>
</tr>
<tr>
<td>Adenine</td>
<td>1 g</td>
</tr>
<tr>
<td>Alanine</td>
<td>4 g</td>
</tr>
<tr>
<td>Arginine</td>
<td>4 g</td>
</tr>
<tr>
<td>Asparagine</td>
<td>4 g</td>
</tr>
<tr>
<td>Aspartic acid</td>
<td>4 g</td>
</tr>
<tr>
<td>Cysteine</td>
<td>4 g</td>
</tr>
<tr>
<td>Glutamic acid</td>
<td>4 g</td>
</tr>
<tr>
<td>Glutamine</td>
<td>4 g</td>
</tr>
<tr>
<td>Glycine</td>
<td>4 g</td>
</tr>
<tr>
<td>Histidine</td>
<td>4 g</td>
</tr>
<tr>
<td>Isoleucine</td>
<td>4 g</td>
</tr>
<tr>
<td>Leucine</td>
<td>8 g</td>
</tr>
<tr>
<td>Lysine</td>
<td>4 g</td>
</tr>
<tr>
<td>Methionine</td>
<td>4 g</td>
</tr>
<tr>
<td>myo-Inositol</td>
<td>4 g</td>
</tr>
<tr>
<td>Phenylalanine</td>
<td>4 g</td>
</tr>
<tr>
<td>Proline</td>
<td>4 g</td>
</tr>
<tr>
<td>Serine</td>
<td>4 g</td>
</tr>
<tr>
<td>Threonine</td>
<td>4 g</td>
</tr>
<tr>
<td>Tryptophan</td>
<td>4 g</td>
</tr>
<tr>
<td>Tyrosine</td>
<td>4 g</td>
</tr>
<tr>
<td>Uracil</td>
<td>4 g</td>
</tr>
<tr>
<td>Valine</td>
<td>4 g</td>
</tr>
</tbody>
</table>
Mixture 2
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glucose</td>
<td>20 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Add distilled water up to 100 mL.</p>
</li>
<li>Autoclave both mixtures (at 121°C for 20 min).</li>
<li>Leave medium at 22°C–24°C to cool off. Store in a dark place.
<div class="note">
<span class="note-title">Note:</span> The medium can be stored at 22°C–24°C for at least three months without a noticeable change in quality.</div>
</li>
<li>Just before use, mix two solutions (mixture 1:mixture 2 = 9:1).</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> You can mix the entirety of the two solutions immediately after autoclaving, but this increases the risk of contamination.</div>
<ol start="7">
<li>YE5S agar or liquid medium (1 L, rich medium for fission yeast)
<ol type="a">
<li>Prepare the mixture in a flask (for agar medium) or bottle (for liquid medium).
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast extract</td>
<td>5 g</td>
</tr>
<tr>
<td>Glucose</td>
<td>30 g</td>
</tr>
<tr>
<td>Adenine</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Uracil</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Histidine</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Leucine</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Lysine</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Agar (for agar medium only)</td>
<td>20 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Add distilled water up to 1 L.</p>
</li>
<li>Autoclave the mixture (at 121°C for 20 min).</li>
<li>(If agar medium) Leave medium at 22°C–24°C to cool off (until 50°C–55°C) and pour into Petri dishes (∼30 plates). Dry plates for 1–2 days at 22°C–24°C and store them at 4°C.</li>
<li>(If liquid medium) Leave medium at 22°C–24°C to cool off and store at 22°C–24°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The medium can be stored for at least 3 months without a noticeable change in quality.</div>
<h3 id="sec1.3">Concanavalin A (Con A)/lectin coating of imaging dish</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<ol start="8">
<li>Prepare the stock solution.
<ol type="a">
<li>Dissolve Con A (2 mg/mL) in distilled water with 5 mM MnCl<sub>2</sub> and 5 mM CaCl<sub>2</sub>. Store aliquots (1 mL) at −20°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Frozen aliquots can be stored for several years without a noticeable loss of efficiency.</div>
<ol start="9">
<li>Con A coating
<ol type="a">
<li>Thaw Con A solution and place 100–150 μL on the glass bottom of the dish.</li>
<li>Incubate for 15 min at 22°C–24°C and remove the Con A solution.
<div class="note">
<span class="note-title">Note:</span> Collected Con A solution can be reused at least 5 times.</div>
</li>
<li>Wash by vigorous pipetting with 200 μL of distilled water. Repeat this step 5 times.</li>
<li>Remove water and leave the dish until the remaining water evaporates completely. Put the lid on to avoid dust falling into the dish.</li>
<li>Store the coated dish at 4°C.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For the fission yeast protocol, replace Con A with lectin (L1395, Sigma, MA, USA) because Con A does not provide efficient adhesion to fission yeast.</div>
<div class="optional">
<span class="optional-title">Optional:</span> This step can be skipped if you purchase a precoated dish (BioMedTech Laboratories, FL, USA).</div>
<div class="note">
<span class="note-title">Note:</span> Coated dishes can be stored for at least 6 months without noticeable loss of efficiency.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Adenine</td>
<td>Acros Organics</td>
<td>Cat#: 16363-1000</td>
</tr>
<tr>
<td>Agar</td>
<td>Fisher Scientific</td>
<td>Cat#: BP1423-500</td>
</tr>
<tr>
<td>Bacto peptone</td>
<td>BD Biosciences</td>
<td>Cat#: 211677</td>
</tr>
<tr>
<td>L-Alanine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP369-100</td>
</tr>
<tr>
<td>L-Arginine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP372-100</td>
</tr>
<tr>
<td>L-Asparagine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP373-100</td>
</tr>
<tr>
<td>L-Aspartic Acid</td>
<td>Fisher Scientific</td>
<td>Cat#: BP374-100</td>
</tr>
<tr>
<td>Calcium Chloride, Dihydrate</td>
<td>Fisher Scientific</td>
<td>Cat#: C79-500</td>
</tr>
<tr>
<td>L-Cysteine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP376-100</td>
</tr>
<tr>
<td>Concanavalin A</td>
<td>Millipore Sigma</td>
<td>Cat#: L7647</td>
</tr>
<tr>
<td>D-Glucose</td>
<td>Fisher Scientific</td>
<td>Cat#: D16-1</td>
</tr>
<tr>
<td>L-Glutamic Acid</td>
<td>Fisher Scientific</td>
<td>Cat#: BP378-100</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP379-100</td>
</tr>
<tr>
<td>Glycine</td>
<td>Fisher Scientific</td>
<td>Cat#: G46-500</td>
</tr>
<tr>
<td>L-Histidine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP382-100</td>
</tr>
<tr>
<td>L-Isoleucine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP384-100</td>
</tr>
<tr>
<td>L-Leucine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP385-100</td>
</tr>
<tr>
<td>Lectin from Glycine max (soybean)</td>
<td>Millipore Sigma</td>
<td>Cat#: L1395</td>
</tr>
<tr>
<td>L-Lysine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP386-100</td>
</tr>
<tr>
<td>L-Methionine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP388-100</td>
</tr>
<tr>
<td>Manganese Chloride Tetrahydrate</td>
<td>Fisher Scientific</td>
<td>Cat#: M87-100</td>
</tr>
<tr>
<td>myo-Inositol</td>
<td>Millipore Sigma</td>
<td>Cat#: I5125</td>
</tr>
<tr>
<td>L-Phenylalanine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP391-100</td>
</tr>
<tr>
<td>L-Proline</td>
<td>Fisher Scientific</td>
<td>Cat#: BP392-100</td>
</tr>
<tr>
<td>L-Serine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP393-100</td>
</tr>
<tr>
<td>L-Threonine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP394-100</td>
</tr>
<tr>
<td>L-Tryptophan</td>
<td>Fisher Scientific</td>
<td>Cat#: BP395-100</td>
</tr>
<tr>
<td>L-Tyrosine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP396-100</td>
</tr>
<tr>
<td>Uracil</td>
<td>Millipore Sigma</td>
<td>Cat#: U0750</td>
</tr>
<tr>
<td>L-Valine</td>
<td>Fisher Scientific</td>
<td>Cat#: BP397-100</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>Fisher Scientific</td>
<td>Cat#: BP1422-500</td>
</tr>
<tr>
<td>Yeast nitrogen base without amino acids</td>
<td>Research Products International</td>
<td>Cat#: Y20040–250.0</td>
</tr>
<tr>
<td>4-Aminobenzoic acid</td>
<td>Acros Organics</td>
<td>Cat#: 14621-2500</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Example imaging dataset (Myo1-GFP)</td>
<td>Mendeley Data</td>
<td><a href="https://dx.doi.org/10.17632/sv9d9hmvrm.1">https://dx.doi.org/10.17632/sv9d9hmvrm.1</a></td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>S. cerevisiae</i>: Strain YEF9609: <i>MATa trp1-Δ63 leu2-Δ1 ura3-52 his3-Δ200 lys2-801 TUB1::HPH-pHIS3-mRuby2-TUB1 MLC2-mApple-URA3MX MYO1-GFP-His3MX6</i>
</td>
<td>(<a href="#bib16">Okada et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>S. cerevisiae</i>: Strain YEF10879: <i>MATα his3Δ1 leu2Δ0 lys2Δ0 ura3Δ0 TUB1::HPH-pHIS3-ymScarlet-I-TUB1 GFP</i><sup><i>Envy</i></sup><i>-EGT2</i>
</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>S. pombe</i>: Strain YCW0130: <i>h</i><sup><i>-</i></sup> <i>kanMX6-Pmyo2-mEGFP-myo2 ade6-M210 leu1-32 ura4-D18 mCherry-atb2:hphMX6</i>
</td>
<td>(<a href="#bib17">Okada et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Fiji</td>
<td>(<a href="#bib19">Schindelin et al., 2012</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>MetaMorph, version 7.8.10.0</td>
<td>Molecular Devices</td>
<td>N/A</td>
</tr>
<tr>
<td>Microsoft Excel</td>
<td>Microsoft</td>
<td>N/A</td>
</tr>
<tr>
<td>NIH ImageJ (1.51j)</td>
<td>(<a href="#bib19">Schindelin et al., 2012</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>R (ver 3.0.1)</td>
<td>The R Project for Statistical Computing</td>
<td><a href="https://www.r-project.org/">https://www.r-project.org/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>4-Chamber 35-mm glass bottom dish</td>
<td>Cellvis</td>
<td>Cat#: D35C4-20-1.5-N</td>
</tr>
<tr>
<td>Sonicator</td>
<td>Qsonica</td>
<td>Q55</td>
</tr>
<tr>
<td>Water bath shaker</td>
<td>New Brunswick Scientific</td>
<td>Model G76</td>
</tr>
<tr>
<td>Spectrophotometer</td>
<td>Thermo Scientific</td>
<td>NanoDrop 2000c</td>
</tr>
<tr>
<td>Microscope</td>
<td>Nikon</td>
<td>Eclipse Ti-U</td>
</tr>
<tr>
<td>Objective lens</td>
<td>Nikon</td>
<td>CFI Apo TIRF 100<b>×</b>/1.49NA</td>
</tr>
<tr>
<td>Light source</td>
<td>Spectral Applied Research</td>
<td>ILE-400</td>
</tr>
<tr>
<td>EMCCD camera</td>
<td>Photometrics</td>
<td>Evolve 512 Delta</td>
</tr>
<tr>
<td>Confocal system</td>
<td>Yokogawa</td>
<td>CSU-X1</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>To perform analysis of protein accumulation kinetics in yeast cells, a spinning-disk confocal microscope is the first choice due to its fast acquisition ability with minimal photobleaching and phototoxicity. Details of the suitable device configuration are described previously (<a href="#bib18">Okada et al., 2017</a>). See the <a href="#key-resources-table">key resource table</a> under the heading "Other" for the devices used for the imaging examples shown in this protocol.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>Yeasts were typically stored in glycerol stock at −80°C, and this step explains how to revitalize frozen cells in preparation for live-cell imaging.</p>
<ol>
<li>Streak out the strain(s) of interest on YPD plates. Incubate them for 1–2 days at 25°C or 30°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use YE5S plates for fission yeast.</div>
<div class="note">
<span class="note-title">Note:</span> Cells on rich media can survive for 3–4 weeks at 4°C. To prevent the plates from drying out, they should be sealed with parafilm. Minimal media are not recommended for storage purposes because cells on such media are prone to lose viability more quickly.</div>
<ol start="2">
<li>A day before the planned imaging, inoculate cells from a single colony on the plate (step 1) into 4 mL of SC liquid medium in a test tube (16 <b>×</b> 150 mm, round bottom) and shake them at 25°C for 6–8 h in a water bath shaker (at 210 rpm).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For fission yeast, we use 8 mL of YE5S liquid medium in a test tube (25 <b>×</b> 120 mm, flat bottom).</div>
<div class="critical">
<span class="critical-title">Critical:</span> YPD liquid medium should be avoided for imaging purposes due to its high background of autofluorescence.</div>
<div class="optional">
<span class="optional-title">Optional:</span> As an alternative, make a thin cell patch on a fresh plate from a single colony and then incubate at 25°C. This also works sufficiently well.</div>
<ol start="3">
<li>Transfer preculture (inoculation volume needs to be calculated, see note) to 4 mL of SC medium in a test tube (16 <b>×</b> 150 mm, round bottom) and shake them at 25°C overnight for 12–16 h in a water bath shaker (at 210 rpm). Cells are cultured until they reach the mid-log phase (OD<sub>600</sub> = 0.3–0.7, 0.4–1 × 10<sup>7</sup> cells/mL).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For fission yeast, we use 8 mL of YE5S liquid medium in a test tube (25 <b>×</b> 120 mm, flat bottom).</div>
<div class="note">
<span class="note-title">Note:</span> Use the spreadsheet to calculate the inoculation volume (See <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Mmc1.xlsx">Table S1</a>)</div>
<div class="optional">
<span class="optional-title">Optional:</span> To increase the likelihood of capturing cells at the right phase of growth for imaging, prepare multiple cultures by serial dilution to gain a broader range of cell density at the time of collection.</div>
<h3 id="sec3.2">Live-cell imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>This step explains how to prepare the sample and acquire time-lapse images.</p>
<ol start="4">
<li>Sample preparation
<ol type="a">
<li>Transfer 1 mL of mid-log culture to a 1.5 mL microcentrifuge tube.
<div class="note">
<span class="note-title">Note:</span> Use a low retention tube for harvesting cells because cells in a minimal medium tend to adhere to the tube's inner wall and are not pelleted efficiently in a regular tube.</div>
</li>
<li>Declump cells by sonication for 5 s at 15% power (Q55, Qsonica).
<div class="note">
<span class="note-title">Note:</span> The sonication step can be skipped if cells do not form aggregates.</div>
</li>
<li>Concentrate cells by centrifugation (8,000 <b>×</b> <i>g</i>) and remove the supernatant.</li>
<li>Resuspend the pellet to 150 μL fresh SC medium.
<div class="note">
<span class="note-title">Note:</span> Use YE5S medium for fission yeast.</div>
</li>
<li>Place the entire cell suspension to the Con A/Lectin-coated glass-bottom dish.</li>
<li>Mount the dish on the microscope stage and incubate 5–15 min to sediment the cells. During cell sedimentation, check the cell density on the glass bottom occasionally by microscope. <a href="#sec7.3">Troubleshooting 2</a>
</li>
<li>When the cell density reaches an appropriate level (see details below), gently remove the supernatant and wash cells with 200 μL fresh SC medium. Repeat the wash step 2–3 times to get rid of floating and unattached cells. Gently remove the supernatant and add 1–1.5 mL of SC medium.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use YE5S medium for fission yeast.</div>
<div class="note">
<span class="note-title">Note:</span> While changing medium, leave a small amount of supernatant to prevent cells from drying out.</div>
<div class="note">
<span class="note-title">Note:</span> Indicated medium volume (1–1.5 mL) is for a single chamber in the 4-chamber 35 mm dish (D35C4-20-1.5-N, Cellvis). If you use a regular non-chambered 35 mm dish, add 4–5 mL to provide cells with sufficient nutrition.</div>
<div class="critical">
<span class="critical-title">Critical:</span> We prefer the cell density at 50–70% field coverage by cells (<a href="#fig2">Figure 2</a>A). Overcrowded fields (90–100% field coverage, <a href="#fig2">Figure 2</a>B) should be avoided because the cell-free area will be used as a background reference in the following data analysis.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Examples of cell densities in a microscopic field for imaging in budding yeast</div>
<p>(A) A field of cells at an appropriate density for analysis (50–70% coverage).</p>
<p>(B) A field of cells at an excessive density for analysis (90–100% coverage).</p>
</figcaption>
</figure>
<div class="optional">
<span class="optional-title">Optional:</span> We recommend this preparation method because it is simple and does not require special equipment or reagents. This method can keep cells healthy for prolonged imaging times (&gt;2 h) and allows for drug perfusion. There are many alternative ways to prepare specimens for live-cell imaging, e.g., using an agarose pad (<a href="#bib18">Okada et al., 2017</a>). If you need special conditions, such as accurate temperature regulation, you should select the appropriate method for your experiment (<a href="#bib1">Babic et al., 2018</a>).</div>
<ol start="5">
<li>Time-lapse imaging
<ol type="a">
<li>These are typical settings for our routine imaging (<a href="#tbl2">Table 2</a>). The conditions should be adjusted to be appropriate for the protein of interest and your imaging system. <a href="#sec7.5">Troubleshooting 3</a>, <a href="#sec7.7">4</a>, and <a href="#sec7.9">5</a>
</li>
</ol>
</li>
</ol>
<table id="tbl2">
<caption>Table 2. Settings for imaging</caption>
<thead>
<tr>
<th>GFP excitation/emission</th>
<th>RFP excitation/emission</th>
<th>Laser power</th>
<th>Exposure time</th>
<th>EM gain</th>
<th>Z-stack</th>
<th>Imaging duration</th>
<th>Time interval</th>
</tr>
</thead>
<tbody>
<tr>
<td>488/505 nm</td>
<td>561/595 nm</td>
<td>3–5%</td>
<td>30–100 ms</td>
<td>200–300</td>
<td>11 sections with 1.0 μm spacing</td>
<td>60–90 min</td>
<td>1 or 1.5 min</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>By following the steps described above and the <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a> described below, you can acquire local protein accumulation kinetics. <a href="#fig3">Figure 3</a> shows an analysis scheme and the kinetics of myosin-II at the division site during cytokinesis in both budding yeast (Myo1-GFP, <a href="#fig3">Figure 3</a>A) and fission yeast (GFP-Myo2, <a href="#fig3">Figure 3</a>B). Accumulation kinetics are an information-rich measurement. Different aspects of a kinetic curve, including the specific rate of rise or fall and the peak accumulation time, could reveal different molecular behaviors of a protein, including its functional time, a localization switch, or a potential structural transition during the cell cycle. Kinetic analysis of cytokinetic proteins at the system level has led to some general rules that can be used to interpret or predict protein behavior (<a href="#bib16">Okada et al., 2021</a>). For example, proteins sharing similar kinetics might reflect similar molecular functions and/or functioning in the same complex. In addition, mechanistic insights into distinct behaviors of different members of a protein family can be obtained by kinetic analysis. For example, the disappearance pattern of myosin-II in budding yeast (Myo1-GFP), but not in fission yeast (GFP-Myo2), accurately reflects the kinetics of actomyosin ring constriction during cytokinesis (<a href="#fig3">Figure 3</a>C). Moreover, the plot created by superimposing the kinetics of many different proteins is exceptionally informative. The analysis of more than twenty proteins has depicted a kinetic landscape of cytokinesis, revealing a strict temporal regulation of budding yeast cytokinesis (<a href="#fig3">Figure 3</a>D).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Quantification of myosin-II kinetics at the division site</div>
<p>(A) Kinetic analysis of the sole myosin-II (Myo1-GFP) in budding yeast. Fluorescent intensity of Myo1-GFP at the division site (blue box) of the budding yeast strain YEF9609 was measured from a series of time-lapse images. Montages of a representative cell were created from selected frames of time-lapse series taken with a 1-min interval. The gray dotted line represents the cell outline. All data from individual cells were aligned in relation to the timing of mitotic spindle breakage. Bold lines and associated shaded bands represent mean and s.d. values, respectively. Modified and reproduced from <a href="#bib16">Okada et al., 2021</a>.</p>
<p>(B) Kinetic analysis of the essential myosin-II (GFP-Myo2) in fission yeast. Time-lapse imaging and data analysis of the fission yeast strain YCW0130 were conducted as described in (A) with slight modifications to fit for fission yeast (see text and note for details). Similar results were published in <a href="#bib17">Okada et al., 2019</a>.</p>
<p>(C) Diameter and intensity of Myo1-GFP and GFP-Myo2 during constriction. Plots were created from the data in (A) and (B). For Myo1-GFP, reproduced from <a href="#bib16">Okada et al., 2021</a>; for GFP-Myo2, similar data were published in <a href="#bib17">Okada et al., 2019</a>.</p>
<p>(D) The kinetic landscape of cytokinesis. As the cell cycle progresses from the onset of mitotic exit to cell separation, distinct functional modules accumulate at the division site in a strict temporal order to carry out their specific roles in cytokinesis. Reproduced from <a href="#bib16">Okada et al., 2021</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>To establish the kinetics of protein accumulation from a sequence of cell images, we need to extract the temporal changes in protein intensity at a specific subcellular location. This step describes how to analyze protein kinetics using our published protocol (<a href="#bib18">Okada et al., 2017</a>) with a slight modification. The example datasets are deposited at Mendeley Data: <a href="https://dx.doi.org/10.17632/sv9d9hmvrm.1">https://dx.doi.org/10.17632/sv9d9hmvrm.1</a>.</p>
<ol>
<li>Import and process an image sequence
<ol type="a">
<li>Open the image sequence (<i>File&gt;Open</i>, select nd file) in Fiji (<a href="#bib19">Schindelin et al., 2012</a>) and create a z-stack of images by combining a series of images taken at different focal planes using sum projections (<i>Image&gt;Stacks&gt;Z Project,</i> choose <i>sum slices</i> for projection type).</li>
<li>Apply the StackReg plugin (<a href="#bib21">Thevenaz et al., 1998</a>, <a href="http://bigwww.epfl.ch/thevenaz/stackreg/">http://bigwww.epfl.ch/thevenaz/stackreg/</a>) to align the stack of images (<i>Plugins&gt;Registration&gt;StackReg</i>, choose <i>Rigid Body</i> for transformation).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> StackReg takes quite a long time if your stack is large and contains many slices. Run the plugin on a computer with a powerful processor. For example, processing of one z-stack (512<b>×</b>512 pixels, 55 slices) takes ∼15 seconds by our workstation (CPU: Intel Xeon W-2223 3.60GHz), while ∼30 seconds by our laptop (CPU: Intel Core i5-2410M 2.30GHz).</div>
<ol start="2">
<li>Selection of the ROI (region of interest)
<ol type="a">
<li>Draw a polygon (click <i>Polygon selections</i> in the toolbar) around the ROI ("a" in <a href="#fig4">Figure 4</a>) and add it to ROI manager (<i>Edit&gt;Selection&gt;Add to Manager</i> or press the T key on a keyboard) (<a href="#fig5">Figure 5</a>A).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Drawing an appropriate ROI for the quantification of Myo1-GFP kinetics during cytokinesis</div>
<p>Please note that the ROIs (yellow polygons) for the protein of interest (Myo1-GFP, "a") and for the background ("b") must cover the intended locations throughout the process of cytokinesis. The gray dotted line represents the cell outline.</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Configuration and quantification</div>
<p>(A) Make sure that the two ROIs ("a" and "b") intended for quantification are selected (in blue highlight).</p>
<p>(B) Example of measurement results.</p>
<p>(C) Make sure that the two boxes are checked before running a measurement.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Ensure that the polygon contains the ROI throughout the biological process of interest by scrolling through the image slices. <a href="#sec7.7">Troubleshooting 4</a>
</div>
<div class="note">
<span class="note-title">Note:</span> In this example, we monitored myosin-II (marked by Myo1-GFP) until the end of cytokinesis.</div>
</li>
<li>Click and drag the polygon to the background area ("b" in <a href="#fig4">Figure 4</a>) and add it to the ROI manager.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure that the polygon stays in the background area throughout the biological process of interest by scrolling through the image slices.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Using a cell-free area as the background may artificially increase the signal for the protein of interest at a subcellular location. To avoid this problem, some researchers might prefer to use the signal in an intracellular area as the background. However, yeast cytosol is not uniform in composition due to the movement of organelles during the cell cycle. In addition, the cytosolic concentrations of some proteins may change as the result of changes in their expression and/or localization during the cell cycle. These factors would increase the signal fluctuations in the background, thus, rendering this alternative less favorable.</div>
<ol start="3">
<li>Perform measurement and data visualization.
<ol type="a">
<li>Ensure that the integrated density box is checked in the measurements (<i>Analyze&gt;Set Measurements</i>).</li>
<li>In the ROI manager, select both ROIs ("a" and "b" from step 2; click + Shift key, <a href="#fig4">Figures 4</a> and <a href="#fig5">5</a>A) and perform Multi Measure (<i>More&gt;Multi Measure</i>). Results will appear in a pop-up window (<a href="#fig5">Figure 5</a>B).
<div class="note">
<span class="note-title">Note:</span> Ensure that the two boxes (<i>Measure all x slices</i> and <i>One row per slice</i>) are checked (<a href="#fig5">Figure 5</a>C).</div>
</li>
<li>Save and open the results in Excel or a statistical program, such as GraphPad or R. To measure the protein intensity, subtract the background integrated density (<i>IntDen2</i> measured from the ROI "b", <a href="#fig4">Figure 4</a>) from the integrated density of the ROI of the biological process (<i>IntDen1</i> measured from the ROI "a", <a href="#fig4">Figure 4</a>). To review the data, plot the changes of fluorescent intensity over time (<a href="#fig6">Figure 6</a>).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Visualization of the result from a single ROI by Microsoft Excel</div>
<p>Fluorescent intensity of the ROI is visualized by line plot after subtraction of background intensity.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Generate a summary of the results by aligning the individual plots.
<ol type="a">
<li>For each dataset from a specific ROI, mark the image number (yellow highlight in <a href="#fig6">Figure 6</a>) in which a cell cycle event can be used as a reference point for plot analysis.
<div class="note">
<span class="note-title">Note:</span> In this example, we used spindle (marked by RFP-Tub1, <a href="#fig1">Figure 1</a>) breakage monitored in a different channel as the reference point (time "0" for the x-axis).</div>
</li>
<li>Align all the data from individual ROIs by the marked slice as time 0, and plot summarized data, such as mean with standard deviation (<a href="#fig7">Figure 7</a>).
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Summarizing the result from multiple ROIs</div>
<p>It is informative to draw both a summarized chart (bottom-right, mean and s.d.) and all individual plots (top-right) to inspect data quality and reliability.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While kinetic analysis of protein accumulation at a specific subcellular location provides a significant amount of information about the function of a protein in a real-time biological process, such an analysis alone is not adequate for defining the mechanism of action for that specific protein. Kinetic analysis must be combined with genetic and/or biochemical analysis to draw any meaningful conclusions.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Compromised function caused by FP tagging (step 4 in <a href="#before-you-begin">Before you begin</a>)</p>
<h3 id="sec7.2">Potential solution</h3>
<p>The difference in FP could result in a difference in phenotype (<a href="#bib13">Jiang et al., 2017</a>). Consistent with this, we observed that mNeonGreen tagging of Myo1 caused an FP-specific growth defect in budding yeast. Consider changing the FP if a similar problem is encountered during strain construction.</p>
<p>Although C-terminal tagging has been widely used, it might compromise the localization and/or function of some proteins. For example, Ras2 in budding yeast requires C-terminal modification for its targeting to the plasma membrane (<a href="#bib3">Bhattacharya et al., 1995</a>). In such cases, N-terminal tagging or tag insertion within the ORF should be attempted. For example, FP insertion at a specific site within the <i>cdc42</i><sup><i>+</i></sup> ORF in fission yeast does not compromise its function (<a href="#bib2">Bendezu et al., 2015</a>). Examining relevant literature and GFP-based localization databases, such as Yeast RGB (<a href="http://shmoo.weizmann.ac.il/elevy/YeastRGB/HTML/YeastRGB.html">http://shmoo.weizmann.ac.il/elevy/YeastRGB/HTML/YeastRGB.html</a>) (<a href="#bib9">Dubreuil et al., 2019</a>), is informative and helpful for deciding the best location for tagging a specific gene.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Insufficient or overcrowded cell density in a microscopic field (step 4 in <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>If cell density in the microscopic field has not reached adequate coverage during the incubation, you can add more cell suspension to the imaging dish. If the cell density is too high, you can strip cells off by pipetting.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Insufficient signal intensity (step 5 in <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Optimize imaging conditions. To have a better signal-to-noise ratio, you can try increasing the laser power. However, this can cause adverse effects, such as enhancing photobleaching/phototoxicity. You can also change other imaging conditions, such as reducing the number of z-sections and/or setting a longer imaging interval. It also helps if you can upgrade your equipment, such as a more sensitive camera and an optimal filter cube.</p>
<p>Change in strain construction. Different FPs possess different signal intensities. Switching to a brighter FP (e.g., mNeonGreen) is the first thing to try. To increase the fluorescence intensity per molecule, tag your protein of interest with tandem copies (2<b>×</b> or 3<b>×</b>) of an FP. Alternatively, to increase the number of molecules per cell, change the promoter to a stronger one (e.g., <i>ADH1</i> promoter). This is done only when the pattern of protein localization is not affected by its expression level. However, it is worth noting that these strategies may also increase the probability of compromising protein function and/or introducing an artefact. Thus, it is essential to ensure that the biological process of interest is not affected by the molecular modifications.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Field drift (step 5 in <a href="#step-by-step-method-details">step-by-step method details</a> and step 2 in <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>If it is x-y drift, image processing by the StackReg plugin can reduce the drift effect. If the StackReg does not solve the issue, it is possible to manually move the ROI to trace the location for the analysis. When changing the location of the polygon, make sure that the polygon's size and shape are not changed. If it is z drift, it is difficult to fix the problem through post-acquisition image processing. Thus, make sure that your focal plane is in the appropriate position during time-lapse imaging and, if necessary, adjust the focal position between acquisitions. Deploying an autofocus system will certainly help. We noticed that turning on the imaging system 1–2 h before imaging significantly improves the stability of the focal plane.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Unhealthy cells (step 5 in <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Cells could slow down growth and lose viability during imaging because of the phototoxicity caused by light illumination. To reduce the toxicity, use minimal laser power and/or change an FP, which requires a longer wavelength for excitation. Differences in specimen preparation also affect cell health. We noticed that some other documented methods (e.g., agarose pad) are unsuitable for more prolonged time-lapse imaging, probably due to insufficient nutrition and/or gas exchange in the culture medium. Cell freshness is a critical factor for cell health. Thus, use freshly streaked-out cells for your experiments.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Babic, J., Griscom, L., Cramer, J., and Coudreuse, D. (2018). An easy-to-build and re-usable microfluidic system for live-cell imaging. BMC Cell Biol. <i>19</i>, 8. <a class="external-link" href="https://doi.org/10.1186/s12860-018-0158-z">https://doi.org/10.1186/s12860-018-0158-z</a></p>
<p id="bib2">Bendezu, F.O., Vincenzetti, V., Vavylonis, D., Wyss, R., Vogel, H., and Martin, S.G. (2015). Spontaneous Cdc42 polarization independent of GDI-mediated extraction and actin-based trafficking. PLoS Biol. <i>13</i>, e1002097. <a class="external-link" href="https://doi.org/10.1371/journal.pbio.1002097">https://doi.org/10.1371/journal.pbio.1002097</a></p>
<p id="bib3">Bhattacharya, S., Chen, L., Broach, J.R., and Powers, S. (1995). Ras membrane targeting is essential for glucose signaling but not for viability in yeast. Proc. Natl. Acad. Sci. U S A <i>92</i>, 2984-2988. <a class="external-link" href="https://doi.org/10.1073/pnas.92.7.2984">https://doi.org/10.1073/pnas.92.7.2984</a></p>
<p id="bib4">Bialecka-Fornal, M., Makushok, T., and Rafelski, S.M. (2016). A review of fluorescent proteins for use in yeast. Methods Mol. Biol. <i>1369</i>, 309-346. <a class="external-link" href="https://doi.org/10.1007/978-1-4939-3145-3_21">https://doi.org/10.1007/978-1-4939-3145-3_21</a></p>
<p id="bib5">Bindels, D.S., Haarbosch, L., van Weeren, L., Postma, M., Wiese, K.E., Mastop, M., Aumonier, S., Gotthard, G., Royant, A., Hink, M.A., et al. (2017). mScarlet: a bright monomeric red fluorescent protein for cellular imaging. Nat. Methods <i>14</i>, 53-56. <a class="external-link" href="https://doi.org/10.1038/nmeth.4074">https://doi.org/10.1038/nmeth.4074</a></p>
<p id="bib6">Botman, D., de Groot, D.H., Schmidt, P., Goedhart, J., and Teusink, B. (2019). In vivo characterisation of fluorescent proteins in budding yeast. Sci. Rep. <i>9</i>, 2234. <a class="external-link" href="https://doi.org/10.1038/s41598-019-38913-z">https://doi.org/10.1038/s41598-019-38913-z</a></p>
<p id="bib7">Buchmuller, B.C., Herbst, K., Meurer, M., Kirrmaier, D., Sass, E., Levy, E.D., and Knop, M. (2019). Pooled clone collections by multiplexed CRISPR-Cas12a-assisted gene tagging in yeast. Nat. Commun. <i>10</i>, 2960. <a class="external-link" href="https://doi.org/10.1038/s41467-019-10816-7">https://doi.org/10.1038/s41467-019-10816-7</a></p>
<p id="bib8">Colman-Lerner, A., Chin, T.E., and Brent, R. (2001). Yeast Cbk1 and Mob2 activate daughter-specific genetic programs to induce asymmetric cell fates. Cell <i>107</i>, 739-750. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00440-8/sref8">View at publisher</a></p>
<p id="bib9">Dubreuil, B., Sass, E., Nadav, Y., Heidenreich, M., Georgeson, J.M., Weill, U., Duan, Y., Meurer, M., Schuldiner, M., Knop, M., et al. (2019). YeastRGB: comparing the abundance and localization of yeast proteins across cells and libraries. Nucleic Acids Res. <i>47</i>, D1245-D1249. <a class="external-link" href="https://doi.org/10.1093/nar/gky941">https://doi.org/10.1093/nar/gky941</a></p>
<p id="bib10">Hawkins, N.C., Garriga, G., and Beh, C.T. (2003). Creating precise GFP fusions in plasmids using yeast homologous recombination. Biotechniques <i>34</i>, 74-78, 80. <a class="external-link" href="https://doi.org/10.2144/03341st03">https://doi.org/10.2144/03341st03</a></p>
<p id="bib11">Huh, W.K., Falvo, J.V., Gerke, L.C., Carroll, A.S., Howson, R.W., Weissman, J.S., and O'Shea, E.K. (2003). Global analysis of protein localization in budding yeast. Nature (London) <i>425</i>, 686-691. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00440-8/sref11">View at publisher</a></p>
<p id="bib12">Jacob, E. and Unger, R. (2007). A tale of two tails: why are terminal residues of proteins exposed?. Bioinformatics <i>23</i>, 225-230. <a class="external-link" href="https://doi.org/10.1093/bioinformatics/btl318">https://doi.org/10.1093/bioinformatics/btl318</a></p>
<p id="bib13">Jiang, Y., Di Gregorio, S.E., Duennwald, M.L., and Lajoie, P. (2017). Polyglutamine toxicity in yeast uncovers phenotypic variations between different fluorescent protein fusions. Traffic <i>18</i>, 58-70. <a class="external-link" href="https://doi.org/10.1111/tra.12453">https://doi.org/10.1111/tra.12453</a></p>
<p id="bib14">Lee, S., Lim, W.A., and Thorn, K.S. (2013). Improved blue, green, and red fluorescent protein tagging vectors for <i>S. cerevisiae</i>. PLoS One <i>8</i>, e67902. <a class="external-link" href="https://doi.org/10.1371/journal.pone.0067902">https://doi.org/10.1371/journal.pone.0067902</a></p>
<p id="bib15">Longtine, M.S., McKenzie, A., DeMarini, D.J., Shah, N.G., Wach, A., Brachat, A., Philippsen, P., and Pringle, J.R. (1998). Additional modules for versatile and economical PCR-based gene deletion and modification in <i>Saccharomyces cerevisiae</i>. Yeast <i>14</i>, 953-961. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00440-8/sref15">View at publisher</a></p>
<p id="bib16">Okada, H., MacTaggart, B., Ohya, Y., and Bi, E. (2021). The kinetic landscape and interplay of protein networks in cytokinesis. iScience <i>24</i>, 101917. <a class="external-link" href="https://doi.org/10.1016/j.isci.2020.101917">https://doi.org/10.1016/j.isci.2020.101917</a></p>
<p id="bib17">Okada, H., Wloka, C., Wu, J.Q., and Bi, E. (2019). Distinct roles of myosin-II isoforms in cytokinesis under normal and stressed conditions. iScience <i>14</i>, 69-87. <a class="external-link" href="https://doi.org/10.1016/j.isci.2019.03.014">https://doi.org/10.1016/j.isci.2019.03.014</a></p>
<p id="bib18">Okada, S., Wloka, C., and Bi, E. (2017). Analysis of protein dynamics during cytokinesis in budding yeast. Methods Cell. Biol. <i>137</i>, 25-45. <a class="external-link" href="https://doi.org/10.1016/bs.mcb.2016.04.002">https://doi.org/10.1016/bs.mcb.2016.04.002</a></p>
<p id="bib19">Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="https://doi.org/10.1038/nmeth.2019">https://doi.org/10.1038/nmeth.2019</a></p>
<p id="bib20">Slubowski, C.J., Funk, A.D., Roesner, J.M., Paulissen, S.M., and Huang, L.S. (2015). Plasmids for C-terminal tagging in <i>Saccharomyces cerevisiae</i> that contain improved GFP proteins, Envy and Ivy. Yeast <i>32</i>, 379-387. <a class="external-link" href="https://doi.org/10.1002/yea.3065">https://doi.org/10.1002/yea.3065</a></p>
<p id="bib21">Thevenaz, P., Ruttimann, U.E., and Unser, M. (1998). A pyramid approach to subpixel registration based on intensity. IEEE Trans. Image Process. <i>7</i>, 27-41. <a class="external-link" href="https://doi.org/10.1109/83.650848">https://doi.org/10.1109/83.650848</a></p>
<p id="bib22">Thomas, V.N., Weiss, E.L., and Brace, J.L. (2018). Asymmetric secretion in budding yeast reinforces daughter cell identity. bioRxiv. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00440-8/sref22">View at publisher</a></p>
<p id="bib23">Torres-Garcia, S., Di Pompeo, L., Eivers, L., Gaborieau, B., White, S.A., Pidoux, A.L., Kanigowska, P., Yaseen, I., Cai, Y., and Allshire, R.C. (2020). SpEDIT: A fast and efficient CRISPR/Cas9 method for fission yeast. Wellcome Open Res. <i>5</i>, 274. <a class="external-link" href="https://doi.org/10.12688/wellcomeopenres.16405.1">https://doi.org/10.12688/wellcomeopenres.16405.1</a></p>
<p id="bib24">Yofe, I., Weill, U., Meurer, M., Chuartzman, S., Zalckvar, E., Goldman, O., Ben-Dor, S., Schutze, C., Wiedemann, N., Knop, M., et al. (2016). One library to make them all: streamlining the creation of yeast libraries via a SWAp-Tag strategy. Nat. Methods <i>13</i>, 371-378. <a class="external-link" href="https://doi.org/10.1038/nmeth.3795">https://doi.org/10.1038/nmeth.3795</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Erfei Bi (<a href="mailto:ebi@pennmedicine.upenn.edu">ebi@pennmedicine.upenn.edu</a>).</p>
<h4>Materials availability</h4>
<p>Reagents used in the Expected Outcomes are available upon request.</p>
<h4>Data and code availability</h4>
<p>The example datasets generated in this study are deposited at Mendeley Data: <a href="https://dx.doi.org/10.17632/sv9d9hmvrm.1">https://dx.doi.org/10.17632/sv9d9hmvrm.1</a>.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/942-Mmc1.xlsx">Table S1. Inoculation volume table, relates to step 3 of step-by-step method details</a></p>
<h3>Acknowledgments</h3>
<p>We thank Andrea Stout of the CDB Imaging Core for assistance, Satoshi Okada and Takashi Ito for the plasmid carrying ymScarlet-I, and Joseph Marquardt, Kangji Wang, and Xi Chen for stimulating discussions and critically reading the manuscript. This work was supported by the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> grant (GM115420 to E.B.).</p>
<h3>Author contributions</h3>
<p>Conceptualization, H.O. and E.B.; methodology, H.O. and E.B.; validation, H.O.; investigation, H.O. and B.M.; writing – original draft, H.O., B.M., and E.B.; writing – review &amp; editing, H.O., B.M., and E.B.; supervision, E.B.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>