<article>
<section>
<h1 id="header">Optogenetic axon guidance in embryonic zebrafish</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>James M. Harris,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor1">*</a></sup> Andy Yu-Der Wang,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a></sup> and Paola Arlotta<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#fn4">7</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Stem Cell and Regenerative Biology, Harvard University, Cambridge, MA 02138, USA</p>
<p id="aff2"><sup>2</sup>Harvard-MIT Division of Health Sciences and Technology, Harvard Medical School, Boston, MA 02115, USA</p>
<p id="aff3"><sup>3</sup>Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA 02138, USA</p>
<p id="fn1"><sup>4</sup>Present address: Tufts University School of Medicine, Boston, MA 02115, USA</p>
<p id="fn2"><sup>5</sup>These authors contributed equally</p>
<p id="fn3"><sup>6</sup>Technical contact</p>
<p id="fn4"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:james_harris@meei.harvard.edu">james_harris@meei.harvard.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:paola_arlotta@harvard.edu">paola_arlotta@harvard.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100947">10.1016/j.xpro.2021.100947</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Axons form the long-range connections of biological neuronal networks, which are built through the developmental process of axon guidance. Here, we describe a protocol to precisely and non-invasively control axonal growth trajectories in live zebrafish embryos using focal light activation of a photoactivatable Rac1. We outline techniques for photostimulation, time-lapse imaging, and immunohistochemistry. These approaches enable engineering of long-range axonal circuitry or repair of defective circuits in living zebrafish, despite a milieu of competing endogenous signals and repulsive barriers.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref6">Harris et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Directed outgrowth of axons using focal optogenetic activation of Rac1</li>
<li>Time-lapse imaging of guided axonal growth in embryonic zebrafish</li>
<li>Immunohistochemical assessment of novel synapse formation</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes a method for optogenetic axon guidance <i>in vivo</i> using stable transgenic zebrafish lines expressing a photoactivatable Rac1 (PA-Rac1) in spinal motor neurons as a model of <i>in vivo</i> axon growth. We used this approach to create novel neuromuscular axonal circuitry in healthy zebrafish with intact axon guidance signaling. We have also had success using a similar protocol for optogenetic axon guidance <i>in vivo</i> with transiently transgenic zebrafish and in cultured zebrafish spinal motor neurons. Finally, this protocol was used to rescue genetic mutants with disrupted axon guidance signaling and development to restore the morphology and function of zebrafish spinal motor neurons.</p>
<h3 id="sec1.1">Zebrafish husbandry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<div class="note">
<span class="note-title">Note:</span> Transiently or stably transgenic zebrafish may be used for optogenetic axon guidance. Transiently transgenic animals are created by injecting zebrafish at the 1-cell stage with a transgenesis construct, while stably transgenic zebrafish are bred from fish already carrying a transgene. Stably transgenic lines are preferred, since all cells in the organism have an equal number of integrated transgenes resulting in less variable expression levels. Transiently transgenic animals are mosaic and thus have more variability in transgene expression between cells.</div>
<div class="note">
<span class="note-title">Note:</span> We use <i>casper (mitfa</i><sup><i>w2/w2</i></sup><i>;mpv17</i><sup><i>a9/a9</i></sup><i>)</i> zebrafish as a background strain, since they lack melanocytes and iridophores and are thus almost entirely transparent (<a href="#bib12">White et al., 2008</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Zebrafish lines are grown under standard conditions (16:8 hour light dark cycle at 28.5°C) (<a href="#bib11">Westerfield, 1993</a>).</div>
<ol>
<li>To generate zebrafish embryos for experiments, breed fish by placing animals from the desired line (<i>casper</i> fish for initial transgenesis or <i>mnx1:mCherry-PA-Rac1</i> transgenic fish if already derived) in a breeding tank, separating males from females with a temporary divider (<a href="#fig1">Figure 1</a>A). The number of fish per breeding tank is determined by the size of the tank. We aim for a male:female ratio between 1:1 and 1:3. For additional information on standard zebrafish husbandry practices, please see (<a href="#bib1">Aljiboury et al., 2021</a>; <a href="#bib11">Westerfield, 1993</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Zebrafish husbandry and embryo generation</div>
<p>(A) Male and female zebrafish are set up in a breeding tank separated by a divider.</p>
<p>(B) The following morning, the divider is removed allowing zebrafish to breed. Embryos are collected by pouring fish water through a tea strainer.</p>
<p>(C) Collected embryos in a tea strainer.</p>
<p>(D) Embryos are transferred to a petri dish.</p>
<p>(E) Embryos in a petri dish.</p>
<p>(F) Fish water is removed with a transfer pipette and replaced with E3 with 0.1% methylene blue.</p>
</figcaption>
</figure>
</li>
<li>The following morning, remove the divider to allow males and females to breed.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Dividers must be pulled within several hours of the room lights turning on to allow for appropriate breeding. Zebrafish naturally breed just after sunrise and exhibit this behavior in laboratory environments as well.</div>
<ol start="3">
<li>Wait 20 min for fertilization, then collect embryos. Remove fish from breeding tanks and filter the water containing the embryos through a tea strainer (<a href="#fig1">Figures 1</a>B and 1C). Transfer embryos to petri dishes by rinsing the collected embryos out of the tea strainer with fish water (<a href="#fig1">Figures 1</a>D and 1E).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Embryos may be injected immediately with a transgenic construct to derive stable lines or to create transiently transgenic animals for subsequent axon guidance experiments.</div>
<ol start="4">
<li>Remove the fish water from the petri dish using a transfer pipette while being careful not to damage or remove the embryos (<a href="#fig1">Figure 1</a>F). Add sterile E3 solution with 0.1% methylene blue.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Sterile E3 with methylene blue helps prevent microbial contamination of embryo cultures (<a href="#bib11">Westerfield, 1993</a>).</div>
<ol start="5">
<li>Place embryos in a 28.5°C incubator and grow to 24 h post fertilization (hpf) for optogenetic axon guidance experiments.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Wrap petri dishes tightly in tinfoil to block ambient light from activating PA-Rac1.</div>
<h3 id="sec1.2">Zebrafish transgenesis injection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼4 h</div>
<p>To create zebrafish expressing PA-Rac1 (<a href="#bib13">Wu et al., 2009</a>) in genetically defined neuronal subtypes, embryos are injected at the 1-cell stage with a transgenesis construct carrying the PA-Rac1 gene under a cell-type specific enhancer or promoter. To express PA-Rac1 specifically in spinal motor neurons, we use 3 tandem repeats of a 125bp-enhancer from the <i>mnx1</i> gene (<a href="#bib15">Zelenchuk and Bruses, 2011</a>).</p>
<ol start="6">
<li>Collect <i>casper</i> zebrafish embryos at the one-cell stage as described in “Zebrafish Husbandry”</li>
<li>Inject embryos at the one-cell stage with 1 nL of 25 ng/μL of <i>pDestTol2CG2-mnx1:mCherry-PA-Rac1</i> transgenesis construct (Addgene Plasmid #169044) along with 35 ng/μL of Tol2 mRNA (<a href="#bib7">Kawakami and Shima, 1999</a>; <a href="#bib8">Kwan et al., 2007</a>). Please see the following references for standard injection techniques (<a href="#bib1">Aljiboury et al., 2021</a>; <a href="#bib4">Engerer et al., 2016</a>; <a href="#bib5">Graeden and Sive, 2009</a>; <a href="#bib11">Westerfield, 1993</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Tol2 mRNA is transcribed <i>in vitro</i> from a NotI-linearized <i>pCS2FA-transposase</i> plasmid using mMessage mMachine SP6 (<a href="#bib8">Kwan et al., 2007</a>). Transcribed RNA is then purified with the RNAeasy Mini Kit. All reactions are performed according to the kit manufacturers’ instructions.</div>
<ol start="8">
<li>Place embryos in 28.5°C incubator. Remove dead, opaque embryos at the end of the injection day or early the following morning to limit microbial contamination. Grow until 30 h post fertilization (hpf) for transgenic line founder screening.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To establish stable transgenic lines, screen injected embryos for mCherry expression in target neurons and raise these transgene-expressing embryos (F0) to reproductive maturity. Transgenesis will occur only in a subset of the injected embryos and these embryos will be mosaic for the transgene. Outcross F0 transgenic fish and screen progeny (F1) for stable transgene expression in target neurons. Since F0 fish are mosaic for the transgene, only a subset of their offspring will carry the transgene. Incross heterozygous F1 fish to create homozygous F2 fish. F2 homozygote fish can be used as a stable transgenics to produce embryos for optogenetic axon guidance experiments without need for injection.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal anti-synaptic vesicle 2 (Clone SV2) (1:100)</td>
<td>Developmental Studies Hybridoma Bank - DSHB</td>
<td>Cat#SV2; RRID: AB_2315387</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-RFP (1:500)</td>
<td>Rockland Immunochemicals</td>
<td>Cat#600-401-379S Lot 35868; RRID: AB_2209751</td>
</tr>
<tr>
<td>Rat monoclonal anti-RFP (Clone 5F8) (1:100)</td>
<td>Chromotek</td>
<td>Cat#5F8; RRID: AB_2336064</td>
</tr>
<tr>
<td>AlexaFluor 488 conjugated goat anti-mouse (1:750)</td>
<td>Invitrogen</td>
<td>Cat#A11001; RRID: AB_2534069</td>
</tr>
<tr>
<td>AlexaFluor 488 conjugated goat anti-rabbit (1:750)</td>
<td>Invitrogen</td>
<td>Cat#A11008; RRID: AB_143165</td>
</tr>
<tr>
<td>AlexaFluor 546 conjugated goat anti-rabbit (1:750)</td>
<td>Invitrogen</td>
<td>Cat#A11035; RRID: AB_143051</td>
</tr>
<tr>
<td>AlexaFluor 546 conjugated goat anti-rat (1:750)</td>
<td>Invitrogen</td>
<td>Cat#A11081; RRID: AB_141738</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Alpha-bungarotoxin, Alexa Fluor 647 conjugate</td>
<td>Thermo Fisher</td>
<td>Cat#B35450</td>
</tr>
<tr>
<td>Ethyl 3-aminobenzoate (Tricaine, MS222)</td>
<td>Sigma-Aldrich</td>
<td>Cat#E10505</td>
</tr>
<tr>
<td>Agarose type II (low melting point)</td>
<td>Sigma-Aldrich</td>
<td>Cat#A6877</td>
</tr>
<tr>
<td>NotI-HF restriction enzyme</td>
<td>NEB</td>
<td>Cat#R3189S</td>
</tr>
<tr>
<td>Collagenase</td>
<td>Sigma-Aldrich</td>
<td>Cat#C9891</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A4161</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat#X100</td>
</tr>
<tr>
<td>Tween20</td>
<td>Sigma-Aldrich</td>
<td>Cat#P1379</td>
</tr>
<tr>
<td>Paraformaldehyde 16% EM Grade</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#50-980-487</td>
</tr>
<tr>
<td>Phosphate-buffered saline 1x</td>
<td>Thermo Fisher</td>
<td>Cat#10010031</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RNAeasy Mini Kit</td>
<td>Qiagen</td>
<td>Cat#74104</td>
</tr>
<tr>
<td>mMessage mMachine SP6 Transcription Kit</td>
<td>Thermo Fisher</td>
<td>Cat#AM1340</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Zebrafish: <i>casper (mitfa</i><sup><i>w2</i></sup><i>;mpv17</i><sup><i>a9</i></sup>: w2/w2;a9/a9)</td>
<td>Zon Lab (<a href="#bib12">White et al., 2008</a>; ZIRC.)</td>
<td>ZIRC Cat#ZL1714; RRID: ZFIN_ZDB-GENO-080326-11</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Plasmid: <i>pDestTol2CG2-mnx1:mCherry-PA-Rac1</i>
</td>
<td><a href="#bib6">Harris et al., 2020</a></td>
<td>Addgene Cat#169044</td>
</tr>
<tr>
<td>Plasmid: <i>pCS2FA-transposase</i>
</td>
<td><a href="#bib7">Kawakami and Shima, 1999</a></td>
<td>Tol2Kit plasmid #396</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Zen Black 2.3 SP1</td>
<td>ZEISS</td>
<td>
<a href="https://www.zeiss.com/microscopy/us/products/microscope-software/zen.html">https://www.zeiss.com/microscopy/us/products/microscope-software/zen.html</a>; RRID:SCR_018163</td>
</tr>
<tr>
<td>Fiji (ImageJ 2.0.0-rc-69/1.52p)</td>
<td>Schneider et al., 2012</td>
<td>
<a href="https://fiji.sc/">https://fiji.sc/</a>; RRID:SCR_002285</td>
</tr>
<tr>
<td>VAST (1.4.0)</td>
<td>Berger et al., 2018</td>
<td><a href="https://software.rc.fas.harvard.edu/lichtman/vast/">https://software.rc.fas.harvard.edu/lichtman/vast/</a></td>
</tr>
<tr>
<td>Imaris</td>
<td>Bitplane</td>
<td>
<a href="https://imaris.oxinst.com/packages">https://imaris.oxinst.com/packages</a>; RRID:SCR_007370</td>
</tr>
<tr>
<td>RStudio 3.5.1</td>
<td>RStudio</td>
<td>
<a href="https://rstudio.com/">https://rstudio.com/</a>; RRID:SCR_000432</td>
</tr>
<tr>
<td>Illustrator 25.2.3</td>
<td>Adobe Studios</td>
<td>
<a href="https://www.adobe.com/products/illustrator.html">https://www.adobe.com/products/illustrator.html</a>; RRID:SCR_018012</td>
</tr>
<tr>
<td>BioRender for graphical abstract</td>
<td>BioRender</td>
<td>
<a href="https://biorender.com/">https://biorender.com/</a>; RRID:SCR_018361</td>
</tr>
<tr>
<td>Statistical analyses and image processing scripts</td>
<td><a href="#bib6">Harris et al., 2020</a></td>
<td><a href="https://github.com/jharris9/optogenetic_axon_guidance_analysis">https://github.com/jharris9/optogenetic_axon_guidance_analysis</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Zeiss LSM 880 Microscope with GaAsP detector</td>
<td>ZEISS</td>
<td>N/A</td>
</tr>
<tr>
<td>Zeiss LSM 880 Microscope with Airyscan detector</td>
<td>Zeiss</td>
<td>N/A</td>
</tr>
<tr>
<td>Gel loading pipette tips</td>
<td>Eppendorf</td>
<td>Cat#022351656</td>
</tr>
<tr>
<td>Transfer pipettes</td>
<td>VWR</td>
<td>Cat#414004-006</td>
</tr>
<tr>
<td>Glass bottom dish (35mm) with 20mm micro-well and #1.5 cover glass</td>
<td>Cellvis</td>
<td>Cat#D35-20-1.5-N</td>
</tr>
<tr>
<td>Miniature scalpel 15-degree blade angle, 5mm depth</td>
<td>Roboz</td>
<td>Cat#37-7515</td>
</tr>
<tr>
<td>Forceps Dumont #5SF</td>
<td>Fine Science Tools</td>
<td>Cat#FST 11252-00</td>
</tr>
<tr>
<td>42°C water bath</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>Digital Optical Power and Energy Meter</td>
<td>ThorLabs</td>
<td>Cat#PM100D</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl2">
<caption>1X E3 solution with methylene blue</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>5.03 mM</td>
<td>2.94 g</td>
</tr>
<tr>
<td>KCl</td>
<td>0.17 mM</td>
<td>0.13 g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>·2H<sub>2</sub>O</td>
<td>0.33 mM</td>
<td>0.49 g</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>·6H<sub>2</sub>O</td>
<td>0.33 mM</td>
<td>0.81 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>10 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at room temperature (15°C–25°C) for up to one year.</p>
<table id="undtbl3">
<caption>0.0168% tricaine anesthetic solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tricaine</td>
<td>0.0168%</td>
<td>16.8 mg</td>
</tr>
<tr>
<td>1x E3</td>
<td>1x</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C protected from light for up to 1 month.</p>
<table id="undtbl4">
<caption>1% agarose mounting solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Agarose type II (low melting point)</td>
<td>1%</td>
<td>0.1 g</td>
</tr>
<tr>
<td>1x E3 with 0.0168% tricaine</td>
<td>1x</td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for same day use. Must be heated first to dissolve into solution and stored at 42°C until use to avoid solidification.</p>
<table id="undtbl5">
<caption>Fixing solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Paraformaldehyde</td>
<td>4%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Phosphate buffered saline (PBS)</td>
<td>1x</td>
<td>30 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh for same day use.</p>
<table id="undtbl6">
<caption>Washing solution (PBT)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tween20</td>
<td>0.1%</td>
<td>50 μL</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>1%</td>
<td>500 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>1x</td>
<td>49.45 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at room temperature (15°C–25°C) for up to one month.</p>
<table id="undtbl7">
<caption>Blocking solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bovine Serum Albumin (BSA)</td>
<td>2%</td>
<td>1 g</td>
</tr>
<tr>
<td>TritonX</td>
<td>0.5%</td>
<td>250 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>1x</td>
<td>49.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to one month.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Zebrafish agarose mounting for imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Zebrafish embryos are anesthetized and mounted in agarose on glass-bottom dishes for photostimulation and confocal imaging experiments.</p>
<ol>
<li>Prepare 1% agarose mounting solution. Each 35 mm glass-bottom dish requires 300–500 uL of agarose.
<ol type="a">
<li>Weigh out the appropriate amount of agarose (0.05 g agarose/5 mL E3 with 0.0168% tricaine).</li>
<li>Add agarose to E3 with 0.0168% tricaine and stir to form a homogenous suspension.</li>
<li>Microwave suspension until completely dissolved, around 10 s at high power.</li>
<li>Place dissolved agarose solution in a pre-equilibrated water bath at 42°C to cool for about 30 min.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Perform the following steps in a dark room using &gt;500 nm wavelength light, outside of the PA-Rac1 activation spectrum (400–500 nm) to reduce baseline PA-Rac1 activity (<a href="#bib14">Wu et al., 2011</a>).</div>
<ol start="2">
<li>Ensure zebrafish are the appropriate age for stimulation experiments. For CaP spinal motor neurons, we aim for 23–25 h post fertilization (hpf). The heart starts beating at 24hpf and is a useful developmental milestone for staging embyros.</li>
<li>Dissect the chorion away from the zebrafish embryos with fine forceps.
<ol type="a">
<li>Pinch the chorion with a set of fine forceps (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Zebrafish mounting in agarose for imaging</div>
<p>(A) Embryos are dissected using fine forceps by first pinching the chorion with forceps.</p>
<p>(B) A second pair of forceps is then used to peel away the chorion.</p>
<p>(C) Embryos are then anesthetized in 0.0168% tricaine in E3 and oriented with a gel-loading pipette tip in 1% agarose.</p>
<p>(D) Multiple embryos can be mounted in a glass bottom dish. Orienting the fish in a column allows them to be easily located on the confocal microscope. Scale bar 1cm.</p>
</figcaption>
</figure>
</li>
<li>With a second pair of fine forceps, pinch the chorion next to the first pair of forceps (<a href="#fig2">Figure 2</a>B).</li>
<li>Separate the forceps, peeling the chorion apart, and allowing the embryo to gently fall out onto the dish.</li>
</ol>
</li>
<li>Transfer embryos to tricaine anesthetic solution for 5 min in a petri dish. Ensure adequate anesthesia by lightly brushing the tail with a gel loading pipette tip. Embryos should be unresponsive to light touch.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Appropriate anesthetic titration is essential since spontaneous movement from embryos will impair the ability to correctly position embryos in agarose.</div>
<ol start="5">
<li>Transfer 3-6 zebrafish embryos to a 1.5 mL microcentrifuge tube and remove anesthetic solution, taking care not to damage or remove the embryos.</li>
<li>Add agarose mounting solution pre-cooled to 42°C</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> There is a short time window of ∼3 minutes in which to position the embryos correctly before agarose hardens. Work quickly, but gently while mounting.</div>
<ol start="7">
<li>Transfer zebrafish to glass bottom dishes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Add the minimum volume of agarose necessary to cover the surface of the glass. Ensure the embryo is positioned near the bottom of the dish so that it is within the working distance of the microscope objective. Avoid introducing air bubbles to the agarose, which will cause optical distortions. If bubbles occur, pop them with a pipette tip.</div>
<ol start="8">
<li>Using a gel loading pipette tip, position embryos on their side, such that the longitudinal axis is parallel to the bottom of the dish with an eye facing up. The eyes should be aligned such that the top eye is directly above the bottom eye. Monitor the embryo position continuously as the agarose solidifies, adjusting as needed (<a href="#fig2">Figure 2</a>C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When mounting multiple embryos, we arrange fish in a single column to make them easier to locate under the microscope (<a href="#fig2">Figure 2</a>D).</div>
<ol start="9">
<li>After the agarose completely solidifies for 15 min, slowly add 1 mL of E3 solution with tricaine anesthetic to the glass bottom dish to keep the agarose hydrated.</li>
<li>Wrap the glass bottom dish with mounted zebrafish in tinfoil to avoid ambient light exposure.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Repeat steps 3-10 as needed to mount additional zebrafish. Mounting additional zebrafish is recommended, since zebrafish may be injured or misaligned in the mounting process.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Embryos can be mounted up to 1 hour before photostimulation experiments.</div>
<h3 id="sec3.2">Confocal live imaging setup</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>Optogenetic axon guidance requires focal illumination over extended periods of live imaging, which imposes several constraints on microscope selection. The microscope must be sufficiently sensitive to detect fluorophores in growing axons with low levels of laser stimulation to minimize photobleaching of the fluorophore and phototoxicity to the organism during extended imaging sessions. Additionally, microscope software must allow focal stimulation within a region of interest. Finally, laser scanning speeds must be sufficiently fast to allow alternating periods of imaging and photostimulation in accordance with the parameters described below. We use the Zeiss LSM 880 Microscope with an optional gallium arsenide phosphide (GaAsP) or Airyscan high-sensitivity detector.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Ambient light must be reduced as much as possible to prevent baseline activation of PA-Rac1. Turn off microscope room lights, cover the microscope with a blackout curtain, and turn computer screens away from the microscope. Microscope indicator lights beneath the blackout curtain can be covered with tape. Background illumination should be measured and minimized, as described in Step 34.</div>
<ol start="11">
<li>Power on the laser, microscope, and computer. Launch the microscope control software (Zen Black). If equipped with an incubation chamber and stage incubator, preheat to 28.5°C.</li>
<li>Move a tinfoil-wrapped glass-bottom dish under the blackout curtain. Working under the curtain with a &gt;500nm wavelength light source, unwrap the tinfoil and position the dish on the microscope stage.</li>
<li>Set up the imaging laser, light path, filters, and standard detector with Smart Setup.
<ol type="a">
<li>Click “Smart Setup” in the top left panel.</li>
<li>Select mCherry from the dropdown menu under “Dye” (<a href="#fig3">Figure 3</a>A). False color the greyscale image by selecting a color under the “Color” dropdown menu.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Smart setup of mCherry fluorescent imaging in Zeiss Zen Black software</div>
<p>(A) Select mCherry as the “Dye” from a library of fluorophores.</p>
<p>(B) The “Fastest” imaging strategy is chosen.</p>
</figcaption>
</figure>
</li>
<li>Several different imaging strategies are available (“Fastest”, “Best Signal”, “Smartest”). Choose “Fastest” and click “Apply” (<a href="#fig3">Figure 3</a>B). Doing this enables the appropriate excitation laser and emission filters to allow for optimal visualization of this fluorophore.</li>
</ol>
</li>
<li>Click the checkbox under “T-PMT” in the “Imaging Setup” panel to enable a brightfield channel that is obtained from the transmitted fluorescent light (<a href="#fig4">Figure 4</a>A). Ensure that the line with the standard “Ch1” detector is checked, the dye is “mCherry”, and the range (emission spectrum) is 578–697nm. The emission spectrum is illustrated with a histogram and the excitation wavelength (561nm) is illustrated with a vertical line.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Enable the appropriate detectors and DIC optics</div>
<p>(A) Enable the T-PMT and optionally the GaAsP2 detector and ensure the appropriate filters are in place.</p>
<p>(B) The Zeiss microscope control touch screen “Home” panel provides information on the appropriate DIC condenser for the current objective.</p>
<p>(C) Input the parameters from B into the DIC optics under the “Locate” tab in the Zen software.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The T-PMT brightfield channel is useful for locating zebrafish and assessing gross morphology.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Enable any high-sensitivity detectors such as a GaAsP or Airyscan detector for higher quality imaging. The GaAsP detector may be used for time-lapse imaging during optogenetic stimulation or high-quality Z-stacks. The Airyscan detector is too slow for time-lapse imaging during optogenetic stimulation, but provides excellent quality Z-stack images.</div>
<ol start="15">
<li>If using a GaAsP detector, click the checkbox next to “GaAsP2” in the “Imaging Setup” panel to enable. Ensure this detector is behind an appropriate bandpass filter for the mCherry emission spectrum (575–610nm) (<a href="#fig4">Figure 4</a>A).</li>
<li>If using an Airyscan detector:
<ol type="a">
<li>In the “Imaging Setup” panel, click the checkbox next to the “ChA” detector. This will disable the standard “Ch1” detector.</li>
<li>Ensure the appropriate filter set is upstream of the “ChA” detector. This filter must allow light from the mCherry emission spectrum (575–610nm) to pass through.</li>
<li>In the “Channels” panel, select “SR” (super resolution) as the Airyscan Mode.</li>
<li>Ensure the Airyscan detector is properly aligned. Click ”Continuous” to generate a live image. Focus on a small fluorescent feature in the mCherry channel. Go to the “Maintain tab” and open the “Airyscan” panel. Adjust the X and Y positions until the central hexagons are brightest in the Airyscan window and the “Quality and Status” indicator reads “Good (Waiting)”. Click “Store Current Pos” to save this position.</li>
</ol>
</li>
<li>Set up differential interference contrast (DIC) optics if available. Determine the DIC condenser and numerical aperture (NA) of the objective that will be used for imaging (DICII condenser and 0.45 NA for a Plan-Apochromat 20x). This information can be found under the “Home” tab of the microscope control touchscreen (<a href="#fig4">Figure 4</a>B). Under the “Locate” tab in the Zen software, input these parameters such that they match those of the objective (<a href="#fig4">Figure 4</a>C).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Always keep laser power as low as possible (&lt;2%) to prevent photobleaching and toxicity.</div>
<ol start="18">
<li>Locate mounted zebrafish embryos using a low magnification objective (Plan-Apochromat 10x M27- Air). Adjust the focus, laser power, and master gain until the zebrafish and the fluorescent target neurons appear clearly. See <a href="#tbl1">Table 1</a> for recommended starting parameters.
<table id="tbl1">
<caption>Table 1. Suggested parameters for imaging acquisition</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Suggested initial values (GaAsP optional)</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Laser</td>
</tr>
<tr>
<td>Laser Lines [nm]</td>
<td>561nm458nm</td>
<td>Excitation wavelengths for the fluorophore (561nm for mCherry) and PA-Rac1 (458nm). “Smart Setup” determines the mCherry excitation laser automatically from a library of fluorophores.</td>
</tr>
<tr>
<td colspan="3">Acquisition mode</td>
</tr>
<tr>
<td>Objective</td>
<td>Plan-Apochromat 20x/0.8 M27- Air</td>
<td>Choose the highest resolution objective with a working distance and field of view sufficient to image the target neurons (CaP spinal motor neurons) within the body of the living zebrafish</td>
</tr>
<tr>
<td>Scan Mode</td>
<td>Frame</td>
<td>Each track is imaged for the entire frame before switching to another track.</td>
</tr>
<tr>
<td>Frame Size</td>
<td>1240 x 1240</td>
<td>The image size in pixels. Use ZEN’s internal algorithm to set the appropriate pixel size for the current imaging parameters by clicking the “Optimal” button. Large images require more time to image.</td>
</tr>
<tr>
<td>Speed</td>
<td>Speed: 12 Pixel Dwell: 0.28 μsScan time: 2s</td>
<td>Fast scan speeds reduce the time required for image capture, limiting the exposure of the sample to light at the expense of signal strength. “Scan time” is calculated based on the current parameters and indicates the time required to capture each image. This determines the duty cycle of optogenetic stimulation.</td>
</tr>
<tr>
<td>Averaging</td>
<td>Number: 2 Mode: Line Method: MeanBit Depth: 8 bitDirection: unidirectional</td>
<td>Multiple scans can be performed, and pixel values combined to reduce noise. “Number” sets the number of scans that are collected. “Mode” determines when repeat information is collected (spot, line, or frame). “Method” indicates the computation performed on pixel values from repeat scans (mean or sum). “Bit Depth” is the number of bits allocated to encode the signal at each pixel. “Direction” determines how the laser is scanned over the sample (unidirectional or bidirectional).</td>
</tr>
<tr>
<td>Zoom</td>
<td>2.0</td>
<td>Digital zoom. Smaller areas can be imaged faster.</td>
</tr>
<tr>
<td colspan="3">Channels</td>
</tr>
<tr>
<td>Lasers</td>
<td>561</td>
<td>Wavelength of the excitation laser.</td>
</tr>
<tr>
<td>Laser Power</td>
<td>1.1%</td>
<td>Use the lowest laser power that produces a clear fluorescent image to reduce photobleaching and phototoxicity.</td>
</tr>
<tr>
<td>Pinhole Diameter</td>
<td>599.4 (17.01 Airy Units, 30.5 μm section)</td>
<td>For Z-stack confocal imaging, click the “1 AU” button to set the pinhole diameter to correspond to 1 Airy Unit. If using the Airyscan detector or for time-lapse optogenetic imaging, set the pinhole to “max”.</td>
</tr>
<tr>
<td>Gain (Master)</td>
<td>mCherry: 725T-PMT: 194GaAsP2: 585</td>
<td>Voltage amplification of the photomultiplier tubes that allows the signal to be scaled to fill the available pixel values without saturation. Use the “Range Detector” to visualize saturation and adjust accordingly.</td>
</tr>
<tr>
<td>Digital Offset</td>
<td>mCherry: 0 T-PMT: 0GaAsP2: 0</td>
<td>Sets the background level of the image.</td>
</tr>
<tr>
<td>Digital Gain</td>
<td>mCherry: 2.8T-PMT: 3.0GaAsP2: 1.0</td>
<td>Multiplies the pixel value by the specified number. This adjustment is performed after image acquisition.</td>
</tr>
<tr>
<td>GaSaP2 Mode (optional)</td>
<td>Integration</td>
<td> </td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Within each somite of the embryonic zebrafish, there are three bilaterally symmetric pairs of primary spinal motor neurons that innervate the musculature. For optogenetic guidance, we target the caudal primary (CaP) spinal motor neurons, since they are easily visualized and have highly stereotyped morphology within each somite (<a href="#bib2">Beattie, 2000</a>). These features allow the effects of optogenetic axon guidance to be easily assessed (<a href="#fig5">Figure 5</a>).</div>
<ol start="19">
<li>Screen mounted zebrafish for appropriately positioned healthy fish at the correct developmental timepoint (23-25hpf) with robust mCherry-PA-Rac1 expression in the CaP spinal motor neurons.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Adjusting the digital zoom can help to quickly locate and screen zebrafish.</div>
<ol start="20">
<li>Once a zebrafish is identified as an appropriate candidate for optogenetic stimulation, select a CaP spinal motor neuron to target for optogenetic axon guidance (<a href="#fig5">Figure 5</a>). We typically aim for central CaP axons above the end of the yolk sac extension (<a href="#fig5">Figure 5</a>A).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Anatomy of zebrafish embryonic primary motor neurons</div>
<p>(A) The caudal primary (CaP) spinal motor neurons extend their sole axon into the ventral myotome at this point in development.</p>
<p>(B) Individual CaP neurons are targeted for optogenetic stimulation with their neighbors serving as internal controls. Somitic boundaries are shown by dashed white chevrons (scale bar, 20 μm). R, rostral; C, caudal; D, dorsal; V, ventral; DM, dorsal myotome; SC, spinal cord; VM, ventral myotome; HMS, horizontal myoseptum; VMS, vertical myoseptum; CaP, MiP, and RoP, caudal, middle, and rostral primary spinal motor neurons, respectively.</p>
<p>Reproduced from <a href="#bib6">Harris et al. (2020)</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.3">High-quality Z-stack image acquisition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼10 min per sample</div>
<p>During cycles of optogenetic stimulation, the time available to capture images is short, which restricts image quality and limits acquisition to a single Z-plane. Thus, it can be helpful to acquire a high-quality 3D image of the target neuron prior to initiating optogenetic axon guidance to capture its initial morphology.</p>
<ol start="21">
<li>Switch to a higher-power objective (Plan-Apochromat 20x/0.8 M27) and bring the target neuron into focus.</li>
<li>Click the “Snap” button to generate a single image using the current settings.</li>
<li>Adjust the frame size, scan speed, pixel averaging, digital zoom, laser power (keep &lt;2%), and master gain of the detector until snapshots of the target neurons are of sufficient quality. See <a href="#tbl1">Table 1</a> for suggested initial parameter settings and advice for adjusting each parameter. <a href="#sec7.1">Troubleshooting 1</a>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The laser power should be set at the lowest level possible (&lt;2%) that still produces a clear image to prevent photobleaching. The “Range Indicator” located in the “Dimensions” panel can be helpful to optimize laser power and gain to maximize information collected without saturation.</div>
<div class="optional">
<span class="optional-title">Optional:</span> A high-sensitivity detector such as a GaAsP or Airyscan detector is recommended to improve the quality of Z-stack images and better capture the axonal morphology.</div>
<div class="note">
<span class="note-title">Note:</span> We typically image a (200–300 μm)<sup>2</sup> region allowing us to capture both the target axon and CaP neurons in neighboring somites in the field of view. The neighbors function as internal controls for assessing the effects of the optogenetic stimulation on the growth trajectory of the target axon.</div>
<div class="note">
<span class="note-title">Note:</span> To enhance visualization following image capture, adjust the settings in the “Display” panel. Min and max thresholds can be adjusted to discard pixels values with no information and the gamma level can be adjusted to further separate the range of intensity values for each channel.</div>
<ol start="24">
<li>Once the parameters for individual frames are optimized, enable Z-stack imaging by clicking the “Z-Stack” checkbox (<a href="#fig6">Figure 6</a>).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Microscope settings for Z-stack acquisition</div>
<p>Ensure the Z-stack function is enabled and set the first and last image planes by focusing the microscope at the top and bottom of the desired Z-stack and clicking the corresponding button. Use the “Optimal” button to set the number of slices and distance between slices.</p>
</figcaption>
</figure>
<ol type="a">
<li>Set the total thickness of the Z-stack by adjusting the focus to the top and bottom most Z-planes that encompass the target neuron and click the “Set First” and “Set Last” buttons at these positions respectively.</li>
<li>Press the "Optimal" button to automatically calculate the appropriate slice number and thickness based on the current parameters.</li>
</ol>
</li>
<li>Click “Start Experiment” to acquire high-quality Z-stack images.</li>
<li>If using the Airyscan detector, perform post-imaging deconvolution. Go to the “Processing” tab and click “Airyscan Processing.” Select the appropriate image, ensure the 3D deconvolution is selected, and click “Apply.” The strength of the Wiener filter can be adjusted to optimize the deconvolution.</li>
<li>Save the current imaging settings to a profile. These settings can be easily loaded for subsequent high-quality Z-stack imaging during or after optogenetic stimulation or for future experiments. Alternatively, a previously acquired image can be opened and the imaging settings restored by clicking the “Reuse” button in the “Dimensions” panel.</li>
</ol>
<h3 id="sec3.4">Optogenetic stimulation and time lapse imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–6 h</div>
<p>Optogenetic axon guidance requires delivery of appropriate light stimulation and time-lapse imaging to monitor the growth of the target axon. A brief (roughly 2 millisecond) light stimulus is given in a focal region at the tip of the growth cone every 1–3 s (1-1/3 Hz), although we have had success up to every 7 s (1/7 Hz). In the period between optogenetic stimulations, an image is acquired to assess the growth of the axon. Image acquisition and optogenetic stimulation are alternated without any pause or delay. This stimulation-imaging cycle is repeated until the axon has grown far enough to require an update to the region of light stimulation (<a href="#fig7">Figure 7</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Time lapse imaging and optogenetic stimulation paradigm</div>
<p>To establish the initial and final axonal morphologies, high-quality 3D Z-stack images (stacked yellow rectangles) are acquired in the mCherry channel before and after the optogenetic stimulation experiment. During the axon guidance experiment, a 2D single plane image in the mCherry channel is acquired (yellow rectangles) over a period of 1–7 s using the 561nm laser. Immediately following image acquisition, PA-Rac1 is photoactivated (blue rectangle) for a period of 2ms using the 458nm laser within a region of interest 3–30 μm. This imaging-stimulation cycle is repeated to induce axonal outgrowth into the region of interest. For both 2D imaging and optogenetic stimulation, each pixel is illuminated for 0.28μs.</p>
</figcaption>
</figure>
<ol start="28">
<li>Focus on the leading edge of the growth cone of the neuron targeted for optogenetic axon guidance. Continue using a higher-power objective (Plan-Apochromat 20x/0.8 NA M27).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To improve image quality during time-lapse imaging, high sensitivity GaAsP detectors are recommended. Airyscan detectors, while very sensitive, are too slow for imaging during optogenetic stimulation. We have had equal success with optogenetic axon guidance using standard detectors, though time-lapse imaging quality is lower.</div>
<ol start="29">
<li>Input the image capture parameters (<a href="#fig8">Figure 8</a>). These parameters control how each frame of the time-series is collected. See <a href="#tbl1">Table 1</a> for suggested initial settings and notes for optimization.
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Image capture settings for optogenetic axon guidance</div>
<p>These parameters allow axonal growth to be monitored without phototoxicity or photobleaching over extended imaging periods.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> We have found that imaging periods (during which an mCherry image of the axonal morphology is acquired using the 561nm laser between optogenetic stimulations) of 1–3 seconds result in reproducible axon guidance, though we have had success with periods of up to 7 seconds. Additionally, low imaging laser powers (&lt;2%) are critical for preventing photobleaching and phototoxicity. Imaging scan speed, averaging, frame size, and zoom affect the duration of image capture.</div>
<div class="note">
<span class="note-title">Note:</span> Both 10x and 20x objectives have been used to optogenetically guide axons.</div>
<ol start="30">
<li>Acquire a test image by clicking the “Snap” button. Adjust the parameters above until satisfactory images are acquired, ensuring that the image duration (“Scan time”) is &lt;3 s and the laser power is &lt;2%. <a href="#sec7.1">Troubleshooting 1</a>
</li>
<li>Next, enable time-lapse imaging by checking the “Time Series” box and unchecking the “Z-Stack” box. Enter the maximal number of cycles (100,000) with an interval of 0 msec between imaging cycles (<a href="#fig9">Figure 9</a>, <a href="#tbl2">Table 2</a>). These settings allow stimulation and imaging to run continuously with no delays. When the axon has grown sufficiently to require an update to the region of illumination, the time series acquisition is manually stopped, and the region of illumination is redrawn.
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Parameters for time-lapse imaging</div>
<p>Imaging is alternated with stimulation without any delay and stopped manually to update the region of stimulation.</p>
</figcaption>
</figure>
<table id="tbl2">
<caption>Table 2. Suggested parameters for time-lapse imaging</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Suggested value</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Time series</td>
</tr>
<tr>
<td>Cycles</td>
<td>As many as possible (100,000)</td>
<td>Number of time points collected during the time-lapse experiment. We set this parameter to the maximum value, since time-lapse imaging is stopped manually once the axon has grown sufficiently to require an update to the region of stimulation.</td>
</tr>
<tr>
<td>Interval</td>
<td>0 msec</td>
<td>Optogenetic stimulation and imaging are alternated without any intervening delay.</td>
</tr>
</tbody>
</table>
</li>
<li>Enable focal optogenetic stimulation by checking “Bleaching” and “Regions”. This allows illumination with laser light of a specific wavelength (458 nm) and power (5%) in a user-drawn region of interest (ROI) (<a href="#fig10">Figure 10</a>, <a href="#tbl3">Table 3</a>). These parameters can be saved to a profile for future experiments.
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Bleaching settings for focal illumination</div>
<p>458nm light is delivered in a circular region of interest targeted at the leading edge of the growth cone to induce outgrowth via PA-Rac1 activation.</p>
</figcaption>
</figure>
<table id="tbl3">
<caption>Table 3. Suggested parameters for optogenetic stimulation</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Suggested value</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bleaching</td>
</tr>
<tr>
<td>Start Bleaching after # scans</td>
<td>1</td>
<td>Number of images acquired prior to optogenetic stimulation.</td>
</tr>
<tr>
<td>Repeat Bleach after # scans</td>
<td>1</td>
<td>Number of images acquired between optogenetic stimulations. Alternate imaging and stimulation.</td>
</tr>
<tr>
<td>Iterations</td>
<td>2</td>
<td>Number of times the ROI is bleached per cycle. More iterations increase the duration and thus amount of light delivered.</td>
</tr>
<tr>
<td>Excitation of Bleach</td>
<td>Wavelength: 458 nmPower: 5.0</td>
<td>Wavelength and power of the laser used for bleaching within the ROI. We stimulate PA-Rac1 with 458nm laser light. We have found that 5% laser power (300–650nW depending on ROI diameter) consistently results in robust optogenetic axon guidance. Too little laser power will not elicit a growth response in the target axon, while too much power can cause retraction.</td>
</tr>
<tr>
<td colspan="3">Regions</td>
</tr>
<tr>
<td>Type</td>
<td>Circle</td>
<td>Circular ROIs are drawn at the leading edge of the growth cone in the desired direction of growth</td>
</tr>
<tr>
<td>Diameter</td>
<td>3–30 μm</td>
<td>ROIs are between 3-30 μm, depending on the size of the growth cone.</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For zebrafish CaP motor neurons, a stimulation period of ∼2 milliseconds (0.28 μs per pixel) with 5% (350–600nW) 458nm laser power results in reproducible optogenetic axon guidance.</div>
<ol start="33">
<li>Acquire an image by clicking the “Snap” button. On this acquired image, draw a 3-30 μm diameter circular ROI using the circle icon at the leading edge of the growth cone facing the desired direction of growth. We aim for half of the ROI to contain the growth cone and the other half to cover the space immediately adjacent to the leading edge, into which the growth cone should expand during optogenetic stimulation.</li>
<li>Use an optical power meter (ThorLabs PM100D) to measure the amount of light delivered during the optogenetic stimulation.
<ol type="a">
<li>Remove the sample dish with the mounted zebrafish embryo from the stage. Take a baseline optical power reading while the microscope is turned on, but not active to measure background illumination.</li>
<li>Minimize background illumination by turning off or blocking lights with tape or blackout curtains.</li>
<li>Once background light is minimized, increase the iterations of bleaching to the maximum value.</li>
<li>Click “Start Experiment” to initiate the imaging experiment.</li>
<li>Measure the laser power during the stimulation period. Adjust laser power such that the power meter reads between 300–650 nW total power in a 3–10 um diameter circular region of interest (21–32 mW/mm<sup>2</sup>).</li>
<li>Replace the sample dish and refocus on the growth cone of the target axon. Repeat Step 33 to ensure the ROI is drawn appropriately on the growth cone. Reduce the bleaching iterations to the specified value.</li>
</ol>
</li>
<li>Start time-lapse imaging and optogenetic axon guidance by clicking the “Start Experiment” button.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We typically see unambiguous axonal outgrowth into the ROI 20 minutes after initiation of optogenetic stimulation. If there is no growth in the direction of illumination, further parameter optimization may be required. <a href="#sec7.3">Troubleshooting 2</a>. Movement (<a href="#sec7.5">troubleshooting 3</a>), injury <a href="#sec7.7">troubleshooting 4</a>, or photobleaching (<a href="#sec7.9">troubleshooting 5</a>) of the zebrafish may impair the ability to optogenetically guide axons.</div>
<ol start="36">
<li>Monitor axonal growth, adjusting the focus as needed, until the growth cone fully occupies the ROI. At this point, the imaging experiment must be manually terminated so the ROI can be redrawn to reflect the growth.
<ol type="a">
<li>To stop the imaging experiment, click “Finish current step”.</li>
<li>Refocus on the growth cone as necessary.</li>
<li>Delete the previous ROI and draw an updated ROI as in Step 33.</li>
<li>Resume time lapse imaging and optogenetic stimulation with the new ROI by pressing the “Start Experiment” button.</li>
</ol>
</li>
<li>Repeat step 36 until axon grows to the desired location.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The rate of axonal growth is dependent on the surrounding environment and strength of competing axonal guidance signaling. For example, when growing CaP axons across vertical myosepta, a repulsive developmental barrier, we noticed that the rate of directed axonal guidance would often slow. In these cases, we continue using the same light stimulation and imaging settings. After roughly 40 minutes, the axon typically crossed the barrier and quickly returned to its previous growth rate.</div>
<ol start="38">
<li>After optogenetic axon guidance is complete, acquire a high-quality Z-stack by repeating Steps 21-26 to capture detailed axonal morphology after axonal guidance. Previously saved settings can be loaded to expedite the imaging setup.</li>
<li>Following optogenetic axon guidance, living zebrafish can be recovered from the agarose and allowed to develop further or analyzed with other methods. To remove the zebrafish from agarose, use a small scalpel to carefully dissect the embedding media away from the fish and transfer to 28.5°C E3 culture media without Tricaine anesthetic.</li>
</ol>
<h3 id="sec3.5">Immunohistochemistry of newly formed synapses</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>After stimulation, immunohistochemistry can be performed to investigate the effects of optogenetic stimulation. Here we outline a procedure to visualize pre- and postsynaptic proteins to provide evidence of novel synapse formation between the guided axon and its target.</p>
<div class="note">
<span class="note-title">Note:</span> This whole mount immunohistochemistry protocol was adapted from previous methods (<a href="#bib10">Panzer et al., 2005</a>).</div>
<ol start="40">
<li>To allow time for synapse formation, grow zebrafish for 1.5 h following optogenetic axon guidance in E3 culture media without tricaine anesthetic at 28.5°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Solutions must be changed slowly to avoid damaging fragile zebrafish tissue.</div>
<ol start="41">
<li>Fix embryos in fixing solution overnight (12–16 h) at 4°C.</li>
<li>Wash embryos three times for five minutes in washing solution (PBT).</li>
<li>To increase antibody penetration, use a miniature scalpel to separate the zebrafish head from the tail.</li>
<li>Incubate tails for 45 min in 1 mg/mL collagenase in PBS.</li>
<li>Wash three times in washing solution for five minutes, and incubate in blocking solution for several hours.</li>
<li>Incubate with mouse anti-synaptic vesicle 2 (1:100) and rabbit anti-RFP (1:500) primary antibodies in blocking solution overnight (12–16hr) at 4°C.</li>
<li>Wash embryo tails 5X with PBT and incubate overnight (12–16 h) at 4°C with AlexaFluor 488 conjugated goat anti-mouse (1:750) and AlexaFluor 546 conjugated goat anti-rabbit (1:750) secondary antibodies in blocking solution.</li>
<li>In the last 45 min of secondary antibody incubation, add AlexFluor 647-conjugated α-bungarotoxin (1:500).</li>
<li>Wash embryo tails three times in washing solution for five minutes each.</li>
<li>Mount in 1% low melting point agarose in PBS for confocal imaging following as above in “Zebrafish Agarose Mounting for Imaging”.</li>
<li>Image samples. We use a similar procedure for imaging processed samples as described above in “High-Quality Z-Stack Acquisition” with adjustments to account for the different fluorophores used in the immunohistochemistry.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Focal, asymmetric light activation of PA-Rac1 in the growth cone should result in axonal outgrowth in the direction of simulation. When sustained over time, axonal growth can be guided along a user-defined trajectory through a living organism, in opposition to intrinsic developmental signals, and across repulsive barriers. Following optogenetic stimulation, we found that 73.9% of CaP spinal motor neurons grew in the direction of optogenetic stimulation, while 26.1% either had no response or retracted following optogenetic stimulation (<a href="#bib6">Harris et al., 2020</a>). Responsive axons grew roughly 30–40% faster than unstimulated controls and could be deviated 130° from their initial trajectory (<a href="#bib6">Harris et al., 2020</a>). Axons could also be guided across repulsive vertical myoseptal barriers, which are never crossed by control axons (<a href="#fig11">Figure 11</a>). CaP motor neurons could be guided within the musculature both rostrally and caudally into neighboring somites as well as ventrally out of the musculature completely to kidney duct. The observed axonal outgrowth is likely due to PA-Rac1 photoactivation and not a direct effect of laser illumination, since illumination of axons expressing a light-insensitive mutant PA-Rac1 results in no differences in growth compared to unilluminated controls in cultured zebrafish spinal motor neurons (<a href="#bib6">Harris et al., 2020</a>). Zebrafish that survived the optogenetic axon guidance did not appear to have any obvious impairments in health, function or ability to swim following excision from the agarose mounting media.</p>
<figure id="fig11"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig11.jpg" alt="Fig11.jpg">
<figcaption>
<div class="figcaption-title">Figure 11. Optogenetic axon guidance of zebrafish CaP spinal motor neurons across repulsive myoseptal boundaries</div>
<p>Time series of unstimulated axons (upper series) and axons subjected to focal optogenetic activation of PA-Rac1 at the growth cone (blue circle). The initial axonal position (white arrowhead) and vertical myoseptal boundaries (white dashed line) are shown (scale bar, 20 μm).</p>
<p>Reproduced from <a href="#bib6">Harris et al. (2020)</a>.</p>
</figcaption>
</figure>
<p>Immunohistochemistry for pre- and postsynaptic proteins can reveal the formation of new synaptic connections with the intended target following guidance and provide evidence for successful repair or creation of novel neuronal circuitry. We observe colocalization of these markers 1.5 h after axon guidance in 3/3 fish with axons guided to neighboring somites and 3/3 fish with repaired axonal circuitry (<a href="#bib6">Harris et al., 2020</a>), suggesting that synapses formed between the guided axon and the novel target of the neighboring somitic muscle (<a href="#fig12">Figure 12</a>).</p>
<figure id="fig12"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1174-Fig12.jpg" alt="Fig12.jpg">
<figcaption>
<div class="figcaption-title">Figure 12. Immunohistochemical markers of novel synapse formation</div>
<p>Synaptic vesicle 2 (SV2, green, middle left) and α-bungarotoxin (αBT, cyan, middle right) are colocalized (merge, right) within a spinal motor axon expressing mCherry (red, left) that has been optogenetically guided to a novel target (scale bar, 5 μm).</p>
<p>Reproduced from <a href="#bib6">Harris et al. (2020)</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The effect of optogenetic axon guidance can be assessed using several metrics including distance of axonal growth, speed of growth, and angle of deviation from the initial axonal trajectory. The distance of axonal growth is calculated by subtracting the initial axonal length from the axonal length after stimulation. Axon length is calculated by tracing the axonal process from the cell body to the tip of the growth cone using the manual neurite trace function in the Imaris (Bitplane) software. Speed of growth is calculated by dividing the distance of axonal growth by the duration of optogenetic stimulation. The angle of deviation from the initial axonal trajectory is calculated by first overlaying maximum intensity projections from Z-stacks of the target axons before and after stimulation. The angle tool in FIJI (ImageJ 2.0.0-rc-69/1.52p) is then used to measure the deviation of the axonal trajectory after stimulation compared with its initial trajectory.</p>
<p>Additional measurements can be made to assess an axon’s position relative to neighboring anatomic landmarks. In our case, we counted the number of axons that crossed the repulsive vertical myoseptal boundary and the distance from this boundary to the tip of the growth cone using the “Straight Segment” tool in FIJI on maximum intensity Z-stack projections.</p>
<p>To compare optogenetically stimulated axons and unstimulated controls with null hypothesis significance testing, a Student’s t-test (“t.test” function in RStudio) is sufficient. However, in our case, since zebrafish CaP motor neurons grow in highly stereotyped repeating somitic units, we can use the neighboring, unstimulated axons as internal controls. Rostral and caudal neighbors must be treated separately though, since somitic units develop asynchronously in a rostral-to-caudal gradient. Furthermore, since neighboring axons reside in the same organism, they are not independent observations. To account for these considerations, we model the data as a function of a fixed effect of axon category, which has three levels (rostral unstimulated, caudal unstimulated, or stimulated) using the “lmerTest” package (version 3.0.1, RStudio 3.5.1). This mixed linear model also includes a random effect corresponding to the animal in which axonal measurements were made. Once modeled, p-values for pairwise comparisons of the axon categories are computed using the least square means method. Finally, these p-values are adjusted for multiple comparisons using the Benjamini-Hochberg procedure to control the false discovery rate.</p>
<div class="textbox">
<p><code>&gt; require('lmerTest')</code></p>
<p><code>&gt; model = lmer(measurement ∼ axon_category (1+axon_category|fish),</code></p>
<p><code>data=zfish_data)</code></p>
<p><code>&gt; ls_means_table = ls_means(model, which="axon_category", pairwise =</code></p>
<p><code>TRUE)</code></p>
<p><code>&gt; p.adjust(ls_means_table$`Pr(&gt;|t|)`, method="fdr")</code></p>
</div>
<p>For immunohistochemical staining, the degree of pre- and postsynaptic protein (SV2 and α-BT) colocalization can be quantified from 3D regions of interest obtained from Z-stacks. We use the Coloc2 package in FIJI (ImageJ) to calculates a Pearson’s correlation coefficient between the pixel values of the SV2 and α-BT channels. To determine if this correlation is greater than what would be expected by chance, Coloc2 generates a random distribution of correlations calculated after randomly shuffling blocks of the original images 100 times to obtain an empiric p-value for the input images (<a href="#bib3">Costes et al., 2004</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Optogenetic axon guidance using this protocol requires the ability to quickly visualize the target axon and illuminate a user-defined subcellular region of the growth cone for an extended period while minimizing the effects of toxicity from light exposure. Sensitive detectors are needed to capture images with low laser powers required for extended time-lapse imaging. The microscope must also be able to illuminate a user-defined region of interest. Sufficient resolution is required to both visualize the growth cone and deliver light within a small area. Additionally, to reduce baseline activation of PA-Rac1, users must not exposure the specimen to ambient 400–500nm light prior to or during optogenetic stimulation, which may not be possible for all experiments.</p>
<p>Optogenetic axon guidance in the 3D tissue of a living organism is challenging. Due to the short time allowed for imaging between stimulation periods, the quality of images obtained during optogenetic stimulation is limited and restricted to a single Z-plane. The lack of real-time 3D spatial information of axonal morphology can make 3D guidance challenging, though high-quality Z-stacks can be obtained before, after, and intermittently during the stimulation period. Additionally, by virtue of using a scanning confocal microscope, laser stimulation is delivered with much greater precision along the X- and Y-axes than the Z-axis. As a result, optogenetic axon guidance in the 2D XY-plane is much easier than in the Z-direction.</p>
<p>Here, we used CaP spinal motor neurons in the embryonic zebrafish as a model of axonal growth. Rac1 is a central mediator of axon guidance across a wide variety of neuronal types and species (<a href="#bib9">Ng et al., 2002</a>) suggesting broad applicability. However, the precise dosage of Rac1 required for guidance via optogenetic stimulation may be different in other systems and may require further optimization. Additionally, zebrafish are transparent. Other organisms may require alternative methods of light-delivery for less optically accessible locations. Furthermore, the embryonic zebrafish and its axons grow rapidly, allowing optogenetic axon guidance over biologically significant distances in a single time-lapse imaging session. Slow-growing axons in larger organisms may require extended periods of optogenetic stimulation. It is also not clear if optogenetic axon guidance is possible outside of the developmental window of growth once axons have reached and synapsed with their targets. Lastly, we found that stable transgenic lines produced more consistent expression of the PA-Rac1. The creation of transgenic lines may not be suitable for all applications.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Target axons cannot be imaged clearly with the confocal microscope. Step 23, Step 30.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Image quality may be degraded by issues with either the microscope or the sample. Use a positive control slide or sample to determine if the microscope is functioning appropriately and the parameters are set correctly. If not, ensure the hardware is working properly and the objectives are clean and free of scratches. Check the software for any errors and ensure imaging parameters match those described above (Step 29). Restart the system if the control sample is still not appearing correctly.</p>
<p>If the imaging set up can capture clear images of the control sample, the issue likely lies with the zebrafish specimen itself. Ensure that the thickness of the glass-bottom dish is appropriate for the objective and that the glass is clean. When mounting, position the zebrafish embryo as close to the bottom of the glass-bottom dish as possible to ensure it remains within the working distance of the objective (Step 7). Use the minimal amount of agarose needed to cover the glass-bottom dish to help keep the zebrafish close to the glass (Step 8). Remove all bubbles from the agarose while it is solidifying. Confirm that the glass-bottom dish is mounted flat on the stage.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Zebrafish axons exhibit no outgrowth after optogenetic activation of PA-Rac1. Step 36.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>We typically notice extension of the growth cone around 20 min after stimulation in our experimental paradigm (Step 36). However, it may take longer in other organisms or neuronal subtypes. If axons are unresponsive after extended periods of stimulation, there may be issues with the specimen or delivery of the optogenetic stimulation.</p>
<p>The specimen must be amenable to optogenetic stimulation. Zebrafish must be stationary (Problem 3) and healthy (Problem 4). Ensure that the organism is the appropriate age such that the target axons are growing and have not formed terminal connections with their target. PA-Rac1 must be expressed at a sufficient level in the target axons. Different promoters or gene expression strategies may be required to increase expression (Before You Begin, Step 7).</p>
<p>Alternatively, the optogenetic stimulation protocol may not activate PA-Rac1 sufficiently to induce axonal outgrowth. First, ensure that the stimulation (Step 32) and imaging (Step 29) parameters match those described above. The ROI must be drawn appropriately at the leading edge of the growth cone. Ensure the ROI is receiving laser stimulation by checking the box in the “Bleaching” column in the “Regions” panel (Step 32). Ensure that ambient light levels are sufficiently low in all workspaces in which zebrafish expressing PA-Rac1 are handled by measuring background illumination with a light meter (Step 34). If high, turn off room lights, cover microscopes with blackout curtains, and keep zebrafish wrapped in tinfoil whenever possible. When working with zebrafish expressing PA-Rac1, work using light outside the PA-Rac1 activation spectrum (&gt;500nm).</p>
<p>Next, measure the power of the light applied by the microscope during bleaching using an optical power meter and adjust the laser power as needed (Step 34). If outgrowth is still not observed, further optimize the light stimulation parameters (<a href="#tbl3">Table 3</a>). Start by lengthening the duration of photoactivation by increasing the bleaching iterations. Next, increase laser power or shorten the imaging period. If the growth cone retracts after stimulation, the laser power is likely too high and phototoxic. Decrease the laser power in this case. Consider optimizing optogenetic stimulation parameters in dissociated monolayer cultures of the target neurons. This simplified environment reduces competing signaling and removes the tissue complexity found <i>in vivo</i>, facilitating optimization of the light stimulation protocol.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Zebrafish embryos move during time-lapse imaging. Step 36.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Movement of the zebrafish can be caused by zebrafish muscle activation, failure of the mounting media, or growth of the embryo. Ensure zebrafish are sufficiently anesthetized with 0.0168% Tricaine in all E3 solutions (the anesthetic solution prior to mounting, the agarose mounting media, and the E3 solution added to the glass bottom dish) (Step 1, Step 4, Step 9). If zebrafish are still not sufficiently anesthetized, remake all solutions as Tricaine can lose potency over time, especially with exposure to light.</p>
<p>Alternatively, the embedding agarose can detach from the glass-bottom dish, causing the embedded zebrafish to move as the agarose floats through the E3 media. Prevent agarose detachment by adding E3 media slowly to the glass bottom dish after mounting the embryos (Step 9). Also, ensure the agarose has sufficiently solidified before adding the E3 media.</p>
<p>Zebrafish embryos grow rapidly, which causes some apparent motion as the embryo develops. At 24hpf and 28.5°C, this growth is slow enough to not interfere with optogenetic stimulation experiments. If zebrafish are much younger or exposed to higher temperatures, the apparent motion due to development may be faster. Consider decreasing the incubation temperature to slow the development of the zebrafish if its growth interferes with optogenetic axon guidance (Step 11).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Zebrafish embryos die during optogenetic stimulation. Step 36.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Healthy zebrafish are largely transparent. When embryo tissue starts to die, it becomes opaque. Tissue death can result from unhealthy embryos, damage during mounting, or phototoxicity. Compare illuminated embryos to control embryos (mounted but not illuminated and not mounted) to determine the cause of damage.</p>
<p>Unhealthy embryos can result from unhealthy parent fish, damage during transient transgenesis injection, or poor culture conditions. Use young, healthy parent fish that are bred regularly (Before You Begin, Step 1). Ensure fresh, uncontaminated E3 culture media with methylene blue is used to inhibit microbial growth and raise embryos at 28.5°C.</p>
<p>Injury can occur during the mounting procedure by exposing embryos to mounting media that is too hot or dense, toxicity from the anesthetic, or by rough manipulation while positioning the embryos. Ensure that mounting media is completely equilibrated to 42°C before attempting to mount zebrafish (Step 1). Mounting media should be 1% low melting point agarose to allow embryos to grow during optogenetic stimulation. Ensure that Tricaine is used at no more than 0.0168% to prevent toxicity. When positioning embryos, use a flexible gel loading pipette tip and lightly manipulate the embryo with soft nudges (Step 8).</p>
<p>During imaging, damage to the embryos can occur if the agarose dries out or if the photostimulation is toxic. Ensure E3 solution with 0.0168% Tricaine is added to the glass-bottom dish after mounting to keep the agarose hydrated (Step 9). Phototoxicity during time-lapse imaging is likely due to laser powers that are too high during the imaging phase (Step 23, Step 29). Lower the laser power and increase the master gain if possible.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>The fluorescent signal from target axons decreases during imaging. Step 25, Step 36.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Excessive laser stimulation may bleach the fluorophore causing loss of signal (Step 23, Step 29). Lower the laser power and increase the master gain if possible. Alternatively, the sample may have drifted out of focus (see <a href="#sec7.5">problem 3</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aljiboury, A.A., Mujcic, A., Cammerino, T., Rathbun, L.I., and Hehnly, H. (2021). Imaging the early zebrafish embryo centrosomes following injection of small-molecule inhibitors to understand spindle formation. STAR Protoc. <i>2</i>, 100293. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref1">View at publisher</a></p>
<p id="bib2">Beattie, C.E. (2000). Control of motor axon guidance in the zebrafish embryo. Brain Res. Bull. <i>53</i>, 489-500. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref2">View at publisher</a></p>
<p id="bib3">Costes, S.V., Daelemans, D., Cho, E.H., Dobbin, Z., Pavlakis, G., and Lockett, S. (2004). Automatic and quantitative measurement of protein-protein colocalization in live cells. Biophysical J. <i>86</i>, 3993-4003. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref3">View at publisher</a></p>
<p id="bib4">Engerer, P., Plucinska, G., Thong, R., Trovo, L., Paquet, D., and Godinho, L. (2016). Imaging subcellular structures in the living zebrafish embryo. J. Vis. Exp. <i>110</i>, e53456. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref4">View at publisher</a></p>
<p id="bib5">Graeden, E. and Sive, H. (2009). Live imaging of the zebrafish embryonic brain by confocal microscopy. J. Vis. Exp. <i>26</i>, 1217. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref5">View at publisher</a></p>
<p id="bib6">Harris, J.M., Wang, A.Y., Boulanger-Weill, J., Santoriello, C., Foianini, S., Lichtman, J.W., Zon, L.I., and Arlotta, P. (2020). Long-range optogenetic control of axon guidance overcomes developmental boundaries and defects. Develop. Cell <i>53</i>, 577-588.e577. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref6">View at publisher</a></p>
<p id="bib7">Kawakami, K. and Shima, A. (1999). Identification of the Tol2 transposase of the medaka fish Oryzias latipes that catalyzes excision of a nonautonomous Tol2 element in zebrafish Danio rerio. Gene <i>240</i>, 239-244. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref7">View at publisher</a></p>
<p id="bib8">Kwan, K.M., Fujimoto, E., Grabher, C., Mangum, B.D., Hardy, M.E., Campbell, D.S., Parant, J.M., Yost, H.J., Kanki, J.P., and Chien, C.B. (2007). The Tol2kit: a multisite gateway-based construction kit for Tol2 transposon transgenesis constructs. Develop. Dyn. <i>236</i>, 3088-3099. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref8">View at publisher</a></p>
<p id="bib9">Ng, J., Nardine, T., Harms, M., Tzu, J., Goldstein, A., Sun, Y., Dietzl, G., Dickson, B.J., and Luo, L. (2002). Rac GTPases control axon growth, guidance and branching. Nature <i>416</i>, 442-447. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref9">View at publisher</a></p>
<p id="bib10">Panzer, J.A., Gibbs, S.M., Dosch, R., Wagner, D., Mullins, M.C., Granato, M., and Balice-Gordon, R.J. (2005). Neuromuscular synaptogenesis in wild-type and mutant zebrafish. Develop. Biol. <i>285</i>, 340-357. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref10">View at publisher</a></p>
<p id="bib11">Westerfield, M. (1993). The Zebrafish Book : A Guide for the Laboratory Use of Zebrafish (<i>Brachydanio rerio</i>) (M. Westerfield). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref11">View at publisher</a></p>
<p id="bib12">White, R.M., Sessa, A., Burke, C., Bowman, T., LeBlanc, J., Ceol, C., Bourque, C., Dovey, M., Goessling, W., Burns, C.E., et al. (2008). Transparent adult zebrafish as a tool for in vivo transplantation analysis. Cell Stem Cell <i>2</i>, 183-189. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref12">View at publisher</a></p>
<p id="bib13">Wu, Y.I., Frey, D., Lungu, O.I., Jaehrig, A., Schlichting, I., Kuhlman, B., and Hahn, K.M. (2009). A genetically encoded photoactivatable Rac controls the motility of living cells. Nature <i>461</i>, 104-108. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref13">View at publisher</a></p>
<p id="bib14">Wu, Y.I., Wang, X., He, L., Montell, D., and Hahn, K.M. (2011). Spatiotemporal control of small GTPases with light using the LOV domain. Methods Enzymol. <i>497</i>, 393-407. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref14">View at publisher</a></p>
<p id="bib15">Zelenchuk, T.A. and Bruses, J.L. (2011). In vivo labeling of zebrafish motor neurons using an mnx1 enhancer and Gal4/UAS. Genesis <i>49</i>, 546-554. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00653-5/sref15">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Paola Arlotta (<a href="mailto:paola_arlotta@harvard.edu">paola_arlotta@harvard.edu</a>).</p>
<h4>Materials availability</h4>
<p>The <i>pDestTol2CG2-mnx1:mCherry-PA-</i>Rac1 transgenesis construct is available from Addgene (Plasmid #169044).</p>
<h4>Data and code availability</h4>
<p>The datasets and code generated during this study are available on github (<a href="https://github.com/jharris9/optogenetic_axon_guidance_analysis">https://github.com/jharris9/optogenetic_axon_guidance_analysis</a>) and from the corresponding author upon request.</p>
<h3>Acknowledgments</h3>
<p>We would like to thank members of the Arlotta lab for insightful discussions and editing of the manuscript. We are grateful to Isaac Adatto and members of the Zon lab for assistance with zebrafish methodology and husbandry and to Doug Richardson and the Harvard Center for Biological Imaging staff for assistance with microscopy. J.M.H. acknowledges support from the Harvard Stem Cell Institute MD/PhD Training Fellowship, the <a href="https://doi.org/10.13039/100000065">National Institute of Neurological Disorders and Stroke</a> (F30 NS095520-02), the <a href="https://doi.org/10.13039/100000057">National Institute of General Medical Sciences</a> (T32GM007753), and the HCBI Simmons Award. Work in P.A. laboratory is supported by the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (P50MH094271, U01MH115727, U19MH114821 and R01NS103758 ), The Harvard Brain Initiative, and the Stanley Center for Psychiatric Research at the Broad Institute of MIT and Harvard.</p>
<h3>Author contributions</h3>
<p>J.M.H. and A.Y.-D.W. performed all experiments. P.A. supervised the work. J.M.H., A.Y.-D.W., and P.A. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>P.A. is a consultant for Herophilus, Foresite Labs, and the New York Stem Cell Foundation. P.A. is a co-founder of Serqet Therapeutics. A patent application is pending based on this work (applicants: President and Fellows of Harvard College; inventors: P.A. and J.M.H). All other authors declare no competing interests.</p>
</section>
</article>