<article>
<section>
<h1 id="header">Protocol for RNA fluorescence <i>in situ</i> hybridization in mouse meningeal whole mounts</h1>
<p><time datetime="2022-03-25">Published: March 25, 2022</time></p>
<p>Olof Rickard Nilsson,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Laszlo Kari,<sup><a href="#aff1">1</a></sup> Rebecca Rosenke,<sup><a href="#aff2">2</a></sup> and Olivia Steele-Mortimer<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Laboratory of Bacteriology, Rocky Mountain Laboratories, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Hamilton, MT 59840-2932, USA</p>
<p id="aff2"><sup>2</sup>Rocky Mountain Veterinary Branch, Rocky Mountain Laboratories, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Hamilton, MT 59840-2932, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:olof.nilsson@nih.gov">olof.nilsson@nih.gov</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:omortimer@niaid.nih.gov">omortimer@niaid.nih.gov</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101256">10.1016/j.xpro.2022.101256</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The multilayered meninges surrounding the brain and spinal cord harbor distinct immune cell populations with prominent roles in health and diseases. Here we present an optimized protocol for RNA fluorescence <i>in situ</i> hybridization (RNA FISH) in meningeal whole mounts, allowing the visualization of gene expression. We also describe the combination of this protocol with immunohistochemistry for simultaneous visualization of mRNA and proteins. This protocol can be used for assessing spatial gene expression within the meninges.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol for the isolation of mouse meninges and preparation of dried whole mounts</li>
<li>Optimization of whole mount <i>in situ</i> hybridization for assessing spatial gene expression</li>
<li>RNA FISH protocol can be combined with immunohistochemistry</li>
<li>Allows simultaneous visualization of gene expression and protein localization</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>In this protocol, we describe the steps for preparing whole mounts of mouse meninges and the specific steps for <i>in situ</i> hybridization assay (ISH) for detection of target RNA within the meninges. We use the commercially available RNAscope technology (<a href="#bib13">Wang et al., 2012</a>), an ISH based on a probe design by Advanced Cell Diagnostics (<a href="https://acdbio.com">https://acdbio.com</a>), that optimizes signal amplification and background noise suppression. Here we use probes to <i>Pecam1</i> (<i>Cd31</i>), <i>Aif1</i>-C3 (<i>Iba1</i>) and <i>Cx3cr1</i> to target macrophages and endothelial cells in the meninges. We also demonstrate that, with additional optimization, immunostaining with antibodies to specific proteins can be multiplexed in the tissue.</p>
<p>All animal experiments in this study adhered to the Guide for Care and Use of Laboratory Animals, 8<sup>th</sup> edition (National Research Council). Animal protocols were reviewed and approved by the Rocky Mountain Laboratories Animal Care and Use Committee (Animal Study Protocol number 2019-043).</p>
<h3 id="sec1.1">Order target RNA probes and opal reagents</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Up to 3 weeks</div>
<ol>
<li>A wide variety of probes can be ordered from the ACD catalog and are usually shipped within 2 weeks. Alternatively, probes can be Made-to-Order, but will take longer.</li>
<li>Select the correct combination of Opal reagents for detection of probes. One fluorophore must be assigned to each probe channel (C1, C2 or C3). In this protocol, we have successfully used Opal 520 (green), Opal 570 (red) and Opal 690 (far red) reagents. Make sure your microscope is set up correctly for viewing the selected fluorophores. Additional guidance for fluorophore selection can be found at <a href="https://acdbio.com/rnascope-fluorescent-multiplex-assay">https://acdbio.com/rnascope-fluorescent-multiplex-assay</a>. A list of available Opal Fluorophore reagent packs can be found at <a href="https://www.akoyabio.com/phenoimager/assays/">https://www.akoyabio.com/phenoimager/assays/</a>.</li>
</ol>
<h3 id="sec1.2">Practice extraction of meninges</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<ol start="3">
<li>It is recommended to practice the collection of the skull cap, extraction and mounting of meninges before beginning any ISH experiment. This is to ensure high quality samples with properly preserved structure.
<ol type="a">
<li>Following O/N fixation, but prior to meningeal extraction, calvaria can be split down the midline using a razor blade, yielding two pieces for practicing extraction and mounting of meninges.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit anti-Mouse CD31 (1:1000)</td>
<td>Abcam</td>
<td>Cat# ab124432RRID: AB_2802125</td>
</tr>
<tr>
<td>Rabbit anti-Rat IBA1 (1:1000)</td>
<td>Wako</td>
<td>Cat# 019-19741RRID: AB_839504</td>
</tr>
<tr>
<td>Goat anti-Rabbit IgG, HRP-conjugated (1:2000)</td>
<td>Cell Signaling Technology</td>
<td>Cat# 7074RRID: AB_2099233</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Opal 520 Reagent Pack</td>
<td>Akoya Biosciences</td>
<td>Cat# FP1487001KT</td>
</tr>
<tr>
<td>Opal 570 Reagent Pack</td>
<td>Akoya Biosciences</td>
<td>Cat# FP1488001KT</td>
</tr>
<tr>
<td>Opal 690 Reagent Pack</td>
<td>Akoya Biosciences</td>
<td>Cat# FP1497001KT</td>
</tr>
<tr>
<td>RNAscope 3-Plex Positive Control Probe-Mm</td>
<td>ACD</td>
<td>Cat# 320881</td>
</tr>
<tr>
<td>RNAscope 3-Plex Negative Control Probe-Mm</td>
<td>ACD</td>
<td>Cat# 320871</td>
</tr>
<tr>
<td>RNAscope probe-Mm-Pecam1</td>
<td>ACD</td>
<td>Cat# 316721</td>
</tr>
<tr>
<td>RNAscope probe-Mm-Aif1-C3</td>
<td>ACD</td>
<td>Cat# 319141-C3</td>
</tr>
<tr>
<td>RNAscope probe-Mm-Cx3cr1</td>
<td>ACD</td>
<td>Cat# 314221</td>
</tr>
<tr>
<td>10% Neutral Buffered Formalin</td>
<td>Cancer Diagnostics</td>
<td>Cat# FX1000</td>
</tr>
<tr>
<td>Sterile Pharmaceutical Grade Saline</td>
<td>Vet One</td>
<td>Cat# NDC 13985-807-50</td>
</tr>
<tr>
<td>Heparin</td>
<td>Sagent Pharmaceuticals</td>
<td>Cat# NDC 25021-400-30</td>
</tr>
<tr>
<td>Phosphate Buffered Saline</td>
<td>Rocky Mountain Laboratories</td>
<td> </td>
</tr>
<tr>
<td>Trizma Base</td>
<td>Sigma-Aldrich</td>
<td>Cat# T6066-1KG</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Calbiochem</td>
<td>Cat# 12659</td>
</tr>
<tr>
<td>ProLong Glass Antifade Mountant</td>
<td>Invitrogen</td>
<td>Cat# P36980</td>
</tr>
<tr>
<td>Tween-20</td>
<td>BIO-RAD</td>
<td>Cat# 1706531</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RNAscope Multiplex Fluorescent Reagent Kit v2</td>
<td>ACD</td>
<td>Cat# 323100</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>RML mice, (2–12 months), male or female</td>
<td>Rocky Mountain Laboratories</td>
<td>N/A</td>
</tr>
<tr>
<td>B6.129P2(Cg)-<i>Cx3cr1</i><sup><i>tm1Litt</i></sup>/J (<i>Cx3cr1</i><sup><i>GFP</i></sup>) mice, (6–12 weeks), male or female (CX3CR1 KO mice)</td>
<td>JAX</td>
<td>Cat# 005582</td>
</tr>
<tr>
<td>B6.129(Cg)-<i>Ccr2</i><sup><i>tm2.1lfc</i></sup>/J (<i>Ccr2</i><sup><i>RFP</i></sup>) mice, (6–12 weeks), male or female (CCR2 KO mice)</td>
<td>JAX</td>
<td>Cat# 017586</td>
</tr>
<tr>
<td>B6.129(Cg)F1-<i>Cx3cr1</i><sup><i>tm1Litt</i></sup> <i>Ccr2</i><sup><i>tm2.1lfc</i></sup> mice, (6–12 weeks), male or female (CX3CR1 heterozygous mice)</td>
<td>Rocky Mountain Laboratories</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Fiji (ImageJ)</td>
<td><a href="#bib11">Schneider et al. (2012)</a></td>
<td><a href="https://imagej.net/software/fiji/">https://imagej.net/software/fiji/</a></td>
</tr>
<tr>
<td>SVI Huygens Essential</td>
<td>Scientific Volume Imaging</td>
<td><a href="https://svi.nl/HomePage">https://svi.nl/HomePage</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>ImmEdge Hydrophobic Barrier Pen</td>
<td>ACD</td>
<td>Cat# 310018</td>
</tr>
<tr>
<td>HybEz II Hybridization System</td>
<td>ACD</td>
<td>Cat# 321710</td>
</tr>
<tr>
<td>TOMO Adhesion Microscope Slides</td>
<td>Matsunami</td>
<td>Cat# TOM-11/90</td>
</tr>
<tr>
<td>Vegetable Steamer, 5.5 quart</td>
<td>Hamilton Beach</td>
<td>Model 37530A</td>
</tr>
<tr>
<td>Tissue Tek staining dish</td>
<td>Sakura Finetek</td>
<td>Cat# 4457</td>
</tr>
<tr>
<td>Tissue Tek slide holder</td>
<td>Sakura Finetek</td>
<td>Cat# 4465</td>
</tr>
<tr>
<td>Fine Scissors</td>
<td>Fine Science Tools</td>
<td>Cat# 14958-11</td>
</tr>
<tr>
<td>Spring Scissors</td>
<td>Fine Science Tools</td>
<td>Cat# 15024-10</td>
</tr>
<tr>
<td>Graefe Forceps</td>
<td>Fine Science Tools</td>
<td>Cat# 11050-10</td>
</tr>
<tr>
<td>Dumont #7 Forceps (fine)</td>
<td>Fine Science Tools</td>
<td>Cat# 11271-30</td>
</tr>
<tr>
<td>Nikon Eclipse <i>Ti2</i> fluorescence microscope</td>
<td>Nikon</td>
<td><a href="https://www.microscope.healthcare.nikon.com/products/inverted-microscopes/eclipse-ti2-series">https://www.microscope.healthcare.nikon.com/products/inverted-microscopes/eclipse-ti2-series</a></td>
</tr>
<tr>
<td>Nikon SMZ1500 stereomicroscope (dissection microscope)</td>
<td>Nikon</td>
<td><a href="https://www.microscopyu.com/museum/model-smz1500-stereomicroscope">https://www.microscopyu.com/museum/model-smz1500-stereomicroscope</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol uses a Hamilton Beach 5.5 quart vegetable steamer, model 37530A, for performing the target retrieval. However, other steamers can be used.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> For RNA FISH involving 1–3 targets qualitative analysis can be done using most research grade epi-fluorescence wide field microscopes with a CCD or CMOS camera or confocal microscopes. We used a Nikon <i>Ti2</i> widefield fluorescence microscope equipped with an Orca-Flash 4.0 sCMOS camera (Hamamatsu Photonics). Specific computer software, e.g., ImageJ (Fiji) (<a href="#bib11">Schneider et al., 2012</a>), is required for post-acquisition analysis.</div>
<table id="undtbl1">
<caption>Buffers</caption>
<thead>
<tr>
<th>Buffer name</th>
<th>Components</th>
</tr>
</thead>
<tbody>
<tr>
<td>TBST (TBS wash buffer)</td>
<td>TBS + 0.005% Tween-20 (v/v).</td>
</tr>
<tr>
<td>TBSB</td>
<td>TBS + 0.1% Bovine serum albumin (w/v).</td>
</tr>
<tr>
<td>1× wash buffer</td>
<td>50× ACD wash buffer diluted 1:50 in distilled water.</td>
</tr>
<tr>
<td>1× target retrieval solution</td>
<td>10× ACD target retrieval solution diluted 1:10 in distilled water.</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>PBS, pH 7.4</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>137 mM</td>
<td>8 g</td>
</tr>
<tr>
<td>KCl</td>
<td>2.7 mM</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>10 mM</td>
<td>1.42 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1.8 mM</td>
<td>0.24 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>TBS, pH 7.6</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>8.766 g</td>
</tr>
<tr>
<td>Tris base</td>
<td>50 mM</td>
<td>6.057 g</td>
</tr>
<tr>
<td>HCl, 6 M</td>
<td>n/a</td>
<td>To pH 7.6</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">50× wash buffer and 10× target retrieval solution are supplied in the RNAscope Multiplex Fluorescent Reagent Kit v2, ACD, #323100.<br> RNAse-free reagents are not required for this protocol.<br> All buffers are stored at room temperature (RT; in our lab 20°C–21°C) for up to one year.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of dried meningeal whole mounts</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Meninges are fixed in situ, extracted, and dried onto microscope slides to prepare them for RNA FISH.</p>
<ol>
<li>Harvest calvaria (skull cap) containing meninges and fix O/N (<a href="#bib9">Louveau et al., 2015</a>, <a href="#bib8">2018</a>) (<a href="#fig1">Figure 1</a>):
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Illustration describing the harvest of calvaria containing meninges from euthanized mice</div>
<p>Top left panel indicates the cut sequence. Panels A–E illustrate each step of the harvest of the calvaria. See also <a href="#mmc2">Methods video S1</a>.</p>
</figcaption>
</figure>
<ol type="a">
<li>Following euthanasia, immediately transcardially perfuse mice with 10 mL sterile pharmaceutical grade saline containing heparin (100 U/mL) followed by 6 mL 10% neutral buffered formalin.</li>
<li>Using fine scissors, cut away fur and skin covering the skull. While holding mouse down firmly with a pair of forceps grasping around the neck, make a transverse cut between the eye sockets using fine scissors (<a href="#fig1">Figure 1</a>A).</li>
<li>Using spring scissors, carefully cut posteriorly along the sides of the skull, starting at the transverse cut made in 1b (<a href="#fig1">Figure 1</a>B).</li>
<li>Finally, use a pair of forceps to lift off the calvaria with meninges attached (<a href="#fig1">Figures 1</a>C and 1D). Fix calvaria with associated meninges in 10% neutral buffered formalin O/N at RT in a 15 mL conical tube.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When removing skull caps, pay close attention to make sure the meninges are extracted along with the skull cap. Sometimes the meninges will come loose, remaining associated with the brain. If this happens, use spring scissors to carefully separate the meninges from the brain while removing the skull cap (<a href="#fig1">Figure 1</a>C).</div>
<div class="note">
<span class="note-title">Note:</span> Complete perfusion is vital for an optimal RNA FISH result, as red blood cells are highly autofluorescent. Perfusion is effective when muscle twitches and blanching of the liver are observed. For more information, see <a href="#sec7.1">troubleshooting problem 1</a>.</div>
<div class="note">
<span class="note-title">Note:</span> While this protocol uses 10% formalin for fixation, we have used 4% paraformaldehyde with similar results. If performing immunohistochemistry, fixation with 4% paraformaldehyde is preferred as formalin fixation can reduce immunoreactivity of target proteins with antibodies. If the fixation time, typically 18–20 h, is changed adjustments may have to be made to protease digestion duration to facilitate proper penetration of target probes into the tissue. Lowering the fixation temperature, e.g., by fixation at 4°C, may also increase target binding of some antibodies, by better preservation of antibody epitopes.</div>
<ol start="2">
<li>The next day, extract the fixed meninges from the calvaria (<a href="#mmc2">Methods video S1</a>):
<ol type="a">
<li>Prepare a 24 well plate with two wells containing 1 mL of PBS for each meninges.</li>
<li>Briefly rinse a calvaria in PBS and place it under a stereomicroscope (optional but recommended). Make sure meninges do not dry out by adding approximately 100 μL PBS.</li>
<li>Extract the meningeal layers from the skull cap using fine forceps (<a href="#bib9">Louveau et al., 2015</a>, <a href="#bib8">2018</a>) by carefully loosening the meninges around the perimeter. Then, starting at the olfactory bulb area, gently start peeling off meninges. If areas of the meninges stay attached, carefully loosen them with the forceps before continue pulling. Peel meninges towards the midline of the calvaria. Once meninges are gathered in the middle, pull meninges loose, starting at the cerebellum and ending at the olfactory bulbs.</li>
<li>Place extracted meninges in the first well with PBS. Wash meninges for 5 min on a shaker (70 rpm).</li>
<li>Using forceps, gently transfer meninges to the second well containing PBS and wash for another 5 min on a shaker (70 rpm). Meninges can be left in PBS for some time (e.g., if mounting several meninges sequentially), but should not be stored in the PBS for prolonged periods of time (maximum 1 h at RT).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To facilitate testing of new probes or antibodies, split the calvaria down the midline using a razor blade before extracting the meninges. This generates two samples from each calvaria.</div>
<ol start="3">
<li>Mount extracted meninges onto microscope slides (<a href="#mmc3">Methods video S2</a>):
<ol type="a">
<li>Place approximately 100 μL PBS in the middle of a microscope slide. Add the washed meninges to the droplet.</li>
<li>Using fine forceps, gently spread meninges out. Carefully remove some of the liquid using a pipette and spread meninges out further. When meninges are sufficiently spread out, remove as much as possible of the remaining liquid using a paper tissue.</li>
<li>Dry meninges on a flat surface O/N at RT.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Proper drying of meninges is crucial for subsequent steps (<a href="#fig2">Figure 2</a>). Improperly dried meninges will detach from slides during target retrieval. Drying time depends on the type of microscope slide. The drying times listed above were optimized using Matsunami TOMO adhesion microscope slides. For more information, see <a href="#sec7.3">troubleshooting problem 2</a>.</div>
<ol start="4">
<li>The next day, wash dried meninges by dipping them about 10 times in distilled water. Dry meninges for 24 h at 42°C or for at least 3 days at RT.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Images of meningeal whole mounts dried onto microscope slides</div>
<p>Examples of meningeal whole mount properly dried onto a microscope slide (A) and of improperly dried whole mounts after target retrieval (B and C). Due to insufficient drying, meninges detach during the target retrieval process. Scale bar indicates approximate size of a whole mount, 13 mm. See also <a href="#mmc3">Methods video S2</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Once meninges have been dried, they can be stored at RT in a dry place. We have successfully used samples that were stored for up to 1 year.</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Mmc2.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Extraction of meninges from calvaria, related to step 2</div>
</figcaption>
</figure>
<figure id="mmc3"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Mmc3.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S2. Mounting of extracted meninges onto microscope slides, related to step 3</div>
</figcaption>
</figure>
<h3 id="sec3.2">Run the RNA FISH assay on dried meningeal whole mounts</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7–9 h depending on the number of probes</div>
<p>Dried meninges are subjected to the RNA FISH procedure to visualize targeted mRNA.</p>
<p>This protocol follows the manufacturer’s protocol, with a few modifications, and can be found using the following link (document #323100-USM).</p>
<p><a href="https://acdbio.com/sites/default/files/USM-323100%20Multiplex%20Fluorescent%20v2%20User%20Manual_10282019_0.pdf">https://acdbio.com/sites/default/files/USM-323100%20Multiplex%20Fluorescent%20v2%20User%20Manual_10282019_0.pdf</a></p>
<ol start="5">
<li>Set up equipment for target retrieval: Add 1 L distilled water to the water chamber of a Hamilton Beach vegetable steamer and put one container with 200 mL distilled water and one container with 200 mL 1× target retrieval solution in the steaming compartment. Start the preset steaming program, setting the duration for at least 1 h. Use a thermometer to monitor the temperature of the target retrieval solution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It may be difficult to reach the target temperature of 100°C for the target retrieval solution. In our laboratory, located at an altitude of approx. 3,500 feet, the maximum temperature was 94°C.</div>
<ol start="6">
<li>Lay slide with dried meninges flat on the bench. When the target retrieval solution in the steamer reaches 90°C, add 4–5 drops of hydrogen peroxide to meninges and incubate for 10 min at RT. Briefly wash meninges by placing slides in a Tissue Tek slide holder and dipping them in a Tissue Tek staining dish filled with distilled water. Repeat wash. Keep slides in the Tissue Tek slide holder. By now the target retrieval solution should have reached the target temperature.</li>
<li>Carefully open the steamer and submerge slides in the water for 15 s. Transfer slides to the target retrieval solution and incubate for 10–15 min. Carefully remove slides from the steamer and rinse them in a Tissue Tek staining dish filled with distilled water for 15 s and then incubate them in a Tissue Tek staining dish filled with 100% ethanol for 3 min at RT.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> If a steamer is not available, target retrieval can be performed by boiling samples in the target retrieval solution. Heat 700 mL 1× target retrieval solution in a 1 L beaker on a hot plate until a mild boil is reached. Slowly submerge a slide rack with slides into the solution and boil for 10–15 min. Remove slides, rinse them briefly in distilled water and then incubate them in 100% ethanol for 3 min. Proceed to step 8. Do note that this alternative uses more reagents and that bubbles formed during the boiling increases the risk of meninges detaching from slides.</div>
<div class="note">
<span class="note-title">Note:</span> For RNA FISH alone, 10 min of target retrieval is sufficient. However, increasing the duration of this step may improve antibody staining. In our experience, a target retrieval of 15 min yielded improved antibody staining, with no apparent loss of RNA FISH staining, although this may be antibody dependent.</div>
<div class="note">
<span class="note-title">Note:</span> If aiming to perform immunohistochemistry following the RNA FISH protocol, do not let slides dry out after incubation in 100% ethanol. Instead, following ethanol incubation, rinse slides in distilled water. Carefully wipe around the meninges with a piece of paper tissue, making sure not to touch the tissue, and draw the hydrophobic barrier. Then add a small amount of distilled water, without contacting the barrier, to the meninges to avoid them drying out while barrier dries, about 10 min. Continue to step 9.</div>
<ol start="8">
<li>Remove slides from ethanol and dry at RT for 5 min. Using a hydrophobic barrier pen, draw a border around the tissue on the slide. Dry completely at RT, at least 10 min. A continuous barrier is required for steps 9–13 of the RNA FISH protocol and steps 14–18 of the immunohistochemistry protocol.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> It is possible to leave the slides at RT O/N and continue the assay the next day.</div>
<div class="note">
<span class="note-title">Note:</span> Before starting steps 9–12, make sure the hybridization oven is set to 40°C and contains the humidity control tray with a wet filter paper, allow to equilibrate for 30 min. Oven should be set to 40°C for all incubation steps.</div>
<ol start="9">
<li>Transfer slides to the HybEz II hybridization system slide holder. Add 4–5 drops of Protease Plus to each slide and incubate for 15–30 min at 40°C in the hybridization oven. Rinse slides in distilled water. Repeat rinse with fresh distilled water.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During protease incubation, remove desired probes from the fridge, warm to 40°C (about 10 min) in an incubator and then cool to RT.</div>
<div class="note">
<span class="note-title">Note:</span> For RNA FISH alone, 15 min of protease digestion is sufficient. If also performing immunohistochemistry using antibodies, protease digestion time will need to be optimized. In our hands, we achieved the best results using a protease digestion of 15 (anti-CD31 antibody) to 30 min (anti-IBA1 antibody).</div>
<ol start="10">
<li>Remove excess liquid from slides by holding the slide holder vertically and carefully dabbing the lower portion of each hydrophobic area with a piece of tissue.
<ol type="a">
<li>Add 100–150 μL (4–6 drops) probe mixture to each sample so that the tissue is completely covered.</li>
<li>Incubate in hybridization oven for 2 h.</li>
<li>Wash slides, still locked in slide holder, in 200 mL 1× wash buffer for 2 min with gentle agitation. Repeat wash. Unless otherwise noted, all subsequent washes are performed similarly.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Probes must be mixed to the correct concentration before use. Channel 1 (C1) probes come at a ready-to-use concentration while C2 and C3 probes must be diluted 1:50 in the C1 probe or in probe diluent if no C1 probe is used.</div>
<div class="note">
<span class="note-title">Note:</span> During probe hybridization, remove AMPs and HRPs from the fridge and equilibrate to RT.</div>
<ol start="11">
<li>Hybridize AMPs:
<ol type="a">
<li>Remove excess liquid from slides and add 4–6 drops AMP1, incubate in the hybridization oven for 30 min.</li>
<li>Wash slides with 1× wash buffer.</li>
<li>Remove excess liquid and add 4–6 drops AMP2. Incubate in hybridization oven for 30 min.</li>
<li>Wash slides with 1× wash buffer.</li>
<li>Remove excess liquid. Add 4–6 drops AMP3 and incubate in hybridization oven for 15 min.</li>
<li>Wash slides with 1× wash buffer.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During AMP3 hybridization, remove desired Opal dye from fridge and equilibrate to RT, in the dark.</div>
<ol start="12">
<li>Develop HRP signal:
<ol type="a">
<li>Remove excess liquid and add 4–6 drops of the correct HRP (e.g., HRP-C1 if probe is in C1) to the slides.</li>
<li>Incubate in the hybridization oven for 15 min.</li>
<li>Wash slides with 1× wash buffer.</li>
<li>Remove excess liquid, add 100–150 μL of the appropriate Opal dye (as substrate for HRP).</li>
<li>Incubate in the hybridization oven for 30 min.</li>
<li>Wash slides with 1× wash buffer.</li>
<li>Remove excess liquid, add 4–6 drops HRP blocker to each slide.</li>
<li>Incubate in the hybridization oven for 15 min.</li>
<li>Wash slides with 1× wash buffer.</li>
<li>If using more than one probe, repeat step 12 with other HRPs and opal dyes.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Any Opal dye color can be assigned to any probe channel. We have used the Opal 520, Opal 570 and Opal 690 dyes. After being reconstituted following manufacturers’ directions, dyes are used at a dilution of 1:1500, diluted in TSA buffer.</div>
<div class="note">
<span class="note-title">Note:</span> Once opal dyes have been added to the sample, make sure to protect the sample from light as much as possible.</div>
<div class="note">
<span class="note-title">Note:</span> If meninges detach from the microscope slide during the staining, refer to <a href="#sec7.5">troubleshooting problem 3</a>.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If desired, perform immunohistochemistry at this point, starting at step 14 below.</div>
<ol start="13">
<li>Counterstain and mount slides:
<ol type="a">
<li>Remove excess liquid and add a few drops of DAPI to the slide. Incubate for 30 s at RT.</li>
<li>Tap off excess DAPI and mount slides using a coverslip and mounting medium, e.g., ProLong Glass Antifade Mountant.</li>
<li>Let slides cure, protected from light, O/N at RT before viewing.</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Perform optional immunohistochemistry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Immunohistochemistry allows for the visualization of proteins in the meningeal whole mounts.</p>
<p>This protocol follows the manufacturer's protocol, with a few modifications, and can be found using the following link (document #323100-TN). <a href="https://acdbio.com/sites/default/files/323100-TN%20Multiplex%20Fluorescent%20V2%20with%20IF_incOpal780.pdf">https://acdbio.com/sites/default/files/323100-TN%20Multiplex%20Fluorescent%20V2%20with%20IF_incOpal780.pdf</a></p>
<ol start="14">
<li>Block tissue:
<ol type="a">
<li>Following the last HRP-blocker step and washes (step 12 above), wash slides two times with TBST wash buffer for two min each.</li>
<li>Add 100–150 μL 10% normal serum diluted in TBSB.</li>
<li>Incubate at RT for 30 min, protected from light, in a humidity chamber.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For optimal blocking, use serum from the same species where the secondary antibody was raised.</div>
<ol start="15">
<li>Add primary antibody:
<ol type="a">
<li>Tap slides to remove excess blocking solution and add 100–150 μL primary antibody diluted in TBSB within the hydrophobic barrier.</li>
<li>Incubate for 2 h at RT, protected from light, in a humidified chamber.</li>
<li>Wash slides in TBST with gentle agitation three times for 5 min each.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Primary antibody incubation time and dilution should be adjusted to optimize staining and minimize background.</div>
<ol start="16">
<li>Add secondary antibody:
<ol type="a">
<li>Add 100–150 μL HRP-conjugated secondary antibody diluted in TBSB.</li>
<li>Incubate for 30 min at RT.</li>
<li>Wash slides in TBST with gentle agitation three times for 5 min each.</li>
</ol>
</li>
<li>Add Opal reagent:
<ol type="a">
<li>Add 100–150 μL of the appropriate Opal dye diluted in TSA buffer to each sample.</li>
<li>Incubate for 10 min at RT.</li>
<li>Wash slides in TBST with gentle agitation three times for 2 min each.</li>
</ol>
</li>
<li>Counterstain and mount slides as described in step 13 above.</li>
</ol>
<h3 id="sec3.4">Image slides</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–4 h</div>
<p>Image slides in a fluorescence microscope to visualize mRNA and protein expression.</p>
<ol start="19">
<li>Image slides using a fluorescence microscope equipped with an appropriate light source and filter sets. If experiencing problems with RNA FISH or immunohistochemistry signals, refer to <a href="#sec7.7">troubleshooting problem 4</a> and <a href="#sec7.9">problem 5</a>. We commonly use the following setup on a Nikon <i>Ti2</i> widefield fluorescence microscope:
<ol type="a">
<li>Objectives: Plan APO 20×/0.8 NA (air) or Plan Fluor 40×/1.3 NA (oil).</li>
<li>Filter sets:</li>
</ol>
</li>
</ol>
<table id="undtbl4">
<thead>
<tr>
<th>Dye</th>
<th>Excitation range (nm)</th>
<th>Emission range (nm)</th>
<th>Common filter set name</th>
</tr>
</thead>
<tbody>
<tr>
<td>DAPI</td>
<td>338–372</td>
<td>414–480</td>
<td>DAPI</td>
</tr>
<tr>
<td>Opal 520</td>
<td>443–489</td>
<td>497–551</td>
<td>GFP</td>
</tr>
<tr>
<td>Opal 570</td>
<td>540–568</td>
<td>579–640</td>
<td>TRITC</td>
</tr>
<tr>
<td>Opal 690</td>
<td>590–645</td>
<td>659–736</td>
<td>Cy5</td>
</tr>
</tbody>
</table>
<ol start="20">
<li>Analyze acquired images using an image analysis program, e.g., Fiji (ImageJ) (<a href="#bib11">Schneider et al., 2012</a>).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Please see figures for expected RNA FISH (<a href="#fig3">Figures 3</a>, <a href="#fig4">4</a>, and <a href="#fig5">5</a>) and immunohistochemistry (<a href="#fig6">Figures 6</a> and <a href="#fig7">7</a>) outcomes. All images were acquired as Z-stacks. The only image processing performed was assembling of images into maximum intensity projections and pseudo coloring of channels in Fiji (<a href="#bib11">Schneider et al., 2012</a>). No alterations of brightness or contrast were performed.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. mRNA control probes in dried meningeal whole mounts</div>
<p>Positive (A and B) and negative (C) control RNA FISH probes in dried meningeal whole mounts. Dried meningeal whole mounts were stained with probes targeting <i>Polr2a</i> (low expression), <i>Ppib</i> (medium expression) and <i>Ubc</i> (high expression) mRNA. Negative control probe targets <i>dapB</i>, an mRNA from <i>Bacillus subtilis</i> and thus absent from mouse tissues. Images were captured with a 20× (A and C) or a 40× (B) objective on a Nikon Ti2 widefield fluorescence microscope. Insert in (A) indicates area imaged in (B). In the top panels (merge): <i>Polr2a</i> = magenta, <i>Ppib</i> = yellow, <i>Ubc</i> = cyan, DAPI = gray. Scale bars, 100 μm.</p>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. <i>Cd31</i> (<i>Pecam1</i>) and <i>Iba1</i> (<i>Aif1</i>) mRNA signal from RNA FISH in dried meningeal whole mounts</div>
<p>Images were captured with a 40× objective on a Nikon Ti2 widefield fluorescence microscope. In the top panel (merge): <i>Cd31</i> = cyan, <i>Iba1</i> = magenta, DAPI = gray. Scale bar, 100 μm.</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Loss of mRNA signal in KO mice</div>
<p><i>Cx3cr1</i> and <i>Iba1</i> (<i>Aif1</i>) mRNA signal in <i>Cx3cr1</i> heterozygous (A) and knock out (B) mice. Notice the greatly reduced <i>Cx3cr1</i> signal in the KO mice. Images were captured with a 40× objective on a Nikon Ti2 widefield fluorescence microscope. In the top panel (merge): <i>Cx3cr1</i> = cyan, <i>Iba1</i> = magenta, DAPI = gray. Scale bars, 100 μm.</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. mRNA and protein staining in dried meningeal whole mounts</div>
<p>(A) <i>Cd31</i> and anti-PECAM-1/CD31 and (B) <i>Iba1</i> and anti-AIF1/IBA1. Whole mounts were first subjected to RNA FISH followed by immunohistochemistry targeting the corresponding protein. Notice the overlap of the mRNA and protein signals. Images were captured with a 40× objective on a Nikon Ti2 widefield fluorescence microscope. In top panels (merge): mRNA probe (FISH) signal = magenta, protein (immunohistochemistry) signal = cyan, DAPI = gray. Scale bars, 100 μm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The punctate mRNA signal produced by the RNAscope assay can be quantified using an image analysis program such as Fiji (ImageJ). We have quantified low and medium expression genes by thresholding the appropriate channel followed by the “analyze particles” function. However, with high expression targets it is often not possible to separate individual objects. Guidelines for how to quantify high expression targets can be found on the ACD website (<a href="https://acdbio.com/get-most-out-your-rnascope%C2%AE-multiplex-fluorescent-assays">https://acdbio.com/get-most-out-your-rnascope%C2%AE-multiplex-fluorescent-assays</a>).</p>
<p>It is also worth noting that deconvolution of images captured on a widefield microscope, using e.g., the SVI Huygens software (<a href="https://svi.nl/HomePage">https://svi.nl/HomePage</a>), can improve quantification by enabling enhanced separation of objects during thresholding (<a href="#fig8">Figure 8</a>). As always when using deconvolution, care should be taken not to lose data.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The RNA FISH part of this protocol is robust and consistently yields good results in our laboratory, showing little to no variation in binding between probes, whereas antibody staining is less predictable and often requires optimization of multiple steps (e.g., target retrieval, protease digestion, antibody dilution, incubation time). Since the protocol requires a protease digestion step, antibodies whose epitopes are sensitive to proteases may not work (<a href="#bib1">Baleriola et al., 2015</a>; <a href="#bib2">Duncan et al., 2015</a>; <a href="#bib6">Kersigo et al., 2018</a>). Furthermore, antibody staining tends to be inconsistent across the dried meninges (<a href="#fig7">Figure 7</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Examples of inconsistent antibody staining with antibodies in the periphery of dried meningeal whole mounts</div>
<p>(A) anti-PECAM-1/CD31 and (B) anti-AIF1/IBA1. Samples were first subjected by RNA FISH followed by immunohistochemistry targeting the corresponding protein. Notice absence of antibody staining (filled arrowheads) but not probe staining (open arrowheads) in some areas. Images were captured with a 40× objective on a Nikon <i>Ti2</i> widefield fluorescence microscope. In top panels (merge): mRNA probe (FISH) signal = magenta, protein (immunohistochemistry) signal = cyan, DAPI = gray. Scale bars, 100 μm.</p>
</figcaption>
</figure>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Thresholding of mRNA signal for quantification</div>
<p>Examples of thresholding for <i>Iba1</i> (<i>Aif1</i>) mRNA signals in original (A) and deconvolved (B) images. The original image is the same as in panel A of <a href="#fig3">Figure 5</a>; ; . Image was deconvolved using the SVI Huygens Essential software. Only the DAPI and <i>Iba1</i> stain is shown for clarity. Lower panels show thresholding of <i>Iba1</i> signal (black). Notice the better separation of objects in the deconvolved image. In top panel (merge): <i>Iba1</i> = cyan, DAPI = gray. Scale bars, 100 μm.</p>
</figcaption>
</figure>
<p>Fluorescent proteins, such as GFP, typically lose their fluorescence during processing for RNA FISH. However, major adjustments to the protocol such as fixation with 4% PFA, omitting the heat-induced target retrieval or lowering protease incubation time may allow the fluorescence of endogenous proteins to be retained (<a href="#bib5">Gross-Thebing et al., 2014</a>; <a href="#bib6">Kersigo et al., 2018</a>) Alternatively, antibodies or mRNA probes targeting fluorescent proteins or their transcription can be used to look for their expression.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Residual red blood cells present in tissue (step 1).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Blood and red blood cells can cause significant background fluorescence. Thus, performing an effective perfusion of mice prior to tissue collection is vital to good assay results. While perfusing, look for blanching of the liver and muscle twitching, which indicates draining of the blood. Apply a slow, steady pressure on the syringe. Too high pressure can damage vessels and lead to artifacts.</p>
<p>Conditions causing clotting or activation of the blood vessel endothelium, such as bacteremia, will impair the perfusion as blood cells will remain attached in blood vessels. This is mostly unavoidable, depending on the nature of the study.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Meninges detach from microscope slides during target retrieval (step 7).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Meninges must be properly dried to stay attached during the target retrieval. For the microscope slides utilized in this protocol, 3 days at RT (temperature and humidity range over 24 h of 20°C–21°C and 25%–30%, respectively) or 24 h at 42°C are required for proper attachment (<a href="#fig2">Figure 2</a>). Furthermore, it is crucial that meninges are completely flat on the microscope slide as wrinkles or bubbles may cause them to detach during target retrieval and washes (<a href="#bib4">Grabinski et al., 2015</a>). Aim to dry meninges with the ventral side facing upwards, as this will minimize any space forming between the tissue and the microscope slide.</p>
<p>Even if dried as outlined in this protocol, prolonged target retrieval will cause meninges to detach. In our hands, target retrieval longer than 15–20 min causes partial detachment and 30 min results in almost complete detachment from slides.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Meninges detach from microscope slides during staining (steps 10–12).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Similar to how meninges may detach during target retrieval, they may also detach or lose integrity during the staining and wash steps of the protocol. As in problem 2, make sure meninges are dried appropriately on the microscope slide, taking care to avoid wrinkles and bubbles forming in the tissue. Prolonged incubation time with protease may cause degradation of the tissue and detachment from the microscope slide.</p>
<p>Take care to perform wash steps with very gentle and slow agitation of the slides. Intense rocking of the slides will eventually cause the sides of the tissue to detach from the microscope slide.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Low signal for mRNA probe. Hard to separate signal from autofluorescence (step 19).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>In general, mRNA probes produce a strong signal, visible as distinct puncta in the meningeal tissue. Signal strength depends on the number of ZZ target probes binding to the target mRNA and if a probe contains a low number of ZZ pairs, signal will be weaker. To increase signal strength, make sure to use optimal assay conditions.</p>
<p>RNA degradation within the sample or the provided probes has not been a problem in our hands. Probes have a shelf life of 2 years when stored as recommended.</p>
<p>Autofluorescence is relatively high in meningeal tissue, especially in the green range of the visible spectrum (see second row panels in <a href="#fig3">Figure 3</a>). Since any color can be assigned to any probe channel, try to use an Opal dye in the red or far-red channels, where autofluorescence is less of a problem. Alternatively, try using a bright probe or highly expressed mRNA target in the green channel. Lastly, red blood cells are highly autofluorescent, thus it is vital to exsanguinate mice completely (by flushing with a minimum of 10 mL saline) before fixation.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Unspecific, weak or non-existent antibody staining (step 19).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>The processes of fixation, drying, target retrieval and protease digestion can alter the epitopes recognized by antibodies (<a href="#bib3">Ge et al., 2006</a>; <a href="#bib5">Gross-Thebing et al., 2014</a>; <a href="#bib10">Ramos-Vara and Miller, 2014</a>; <a href="#bib14">Young et al., 2020</a>). Thus, the protocol presented here will need to be optimized for use with antibodies other than the ones described herein. Even with optimization, some areas of the meninges may display inconsistent antibody staining, usually on the edges of the sample (<a href="#fig7">Figure 7</a>).</p>
<p>In our experience, key parameters to adjust are the target retrieval and protease digestion times. Try varying the target retrieval time from 10 to 20 min. A longer target retrieval may not be possible due to detachment of meninges from slides. A shorter target retrieval may result in reduced mRNA signal. Digestion times can be varied from 15 min up to 30 min without loss of mRNA signal and may increase antibody signal. However, a protease digestion time of 45 min or more will result in loss of mRNA signal.</p>
<p>If unspecific staining is encountered, try diluting the primary antibody more. This may reduce off target staining.</p>
<p>Following target retrieval and incubation in 100% ethanol, do not let slides dry out if performing immunohistochemistry. Keeping slides wet will make staining more consistent for some antibodies.</p>
<p>While it is possible to use secondary antibodies conjugated to fluorophores, we recommend using an HRP-conjugated secondary antibody in combination with one of the Opal dyes, which are based on tyramide signal amplification and yields strong fluorescence signals (<a href="#bib7">Kerstens et al., 1995</a>; <a href="#bib12">van Gijlswijk et al., 1997</a>). This has consistently worked well in our hands.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Baleriola, J., Jean, Y., Troy, C., and Hengst, U. (2015). Detection of axonally localized mRNAs in brain sections using high-resolution in situ hybridization. J. Vis. Exp. e52799. <a class="external-link" href="https://doi.org/10.3791/52799">https://doi.org/10.3791/52799</a></p>
<p id="bib2">Duncan, J.S., Elliott, K.L., Kersigo, J., Gray, B., and Fritzsch, B. (2015). Combining whole-mount in situ hybridization with neuronal tracing and immunohistochemistry. In Neuromethods, G. Hauptmann, ed. (Springer Science), pp. 339-352. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00136-8/sref2">View at publisher</a></p>
<p id="bib3">Ge, S., Crooks, G.M., McNamara, G., and Wang, X. (2006). Fluorescent immunohistochemistry and in situ hybridization analysis of mouse pancreas using low-power antigen-retrieval technique. J. Histochem. Cytochem. <i>54</i>, 843-847. <a class="external-link" href="https://doi.org/10.1369/jhc.5B6902.2006">https://doi.org/10.1369/jhc.5B6902.2006</a></p>
<p id="bib4">Grabinski, T.M., Kneynsberg, A., Manfredsson, F.P., and Kanaan, N.M. (2015). A method for combining RNAscope in situ hybridization with immunohistochemistry in thick free-floating brain sections and primary neuronal cultures. PLoS One <i>10</i>, e0120120. <a class="external-link" href="https://doi.org/10.1371/journal.pone.0120120">https://doi.org/10.1371/journal.pone.0120120</a></p>
<p id="bib5">Gross-Thebing, T., Paksa, A., and Raz, E. (2014). Simultaneous high-resolution detection of multiple transcripts combined with localization of proteins in whole-mount embryos. BMC Biol. <i>12</i>, 55. <a class="external-link" href="https://doi.org/10.1186/s12915-014-0055-7">https://doi.org/10.1186/s12915-014-0055-7</a></p>
<p id="bib6">Kersigo, J., Pan, N., Lederman, J.D., Chatterjee, S., Abel, T., Pavlinkova, G., Silos-Santiago, I., and Fritzsch, B. (2018). A RNAscope whole mount approach that can be combined with immunofluorescence to quantify differential distribution of mRNA. Cell Tissue Res. <i>374</i>, 251-262. <a class="external-link" href="https://doi.org/10.1007/s00441-018-2864-4">https://doi.org/10.1007/s00441-018-2864-4</a></p>
<p id="bib7">Kerstens, H.M., Poddighe, P.J., and Hanselaar, A.G. (1995). A novel in situ hybridization signal amplification method based on the deposition of biotinylated tyramine. J. Histochem. Cytochem. <i>43</i>, 347-352. <a class="external-link" href="https://doi.org/10.1177/43.4.7897179">https://doi.org/10.1177/43.4.7897179</a></p>
<p id="bib8">Louveau, A., Filiano, A.J., and Kipnis, J. (2018). Meningeal whole mount preparation and characterization of neural cells by flow cytometry. Curr. Protoc. Immunol. <i>121</i>, e50. <a class="external-link" href="https://doi.org/10.1002/cpim.50">https://doi.org/10.1002/cpim.50</a></p>
<p id="bib9">Louveau, A., Smirnov, I., Keyes, T.J., Eccles, J.D., Rouhani, S.J., Peske, J.D., Derecki, N.C., Castle, D., Mandell, J.W., Lee, K.S., et al. (2015). Structural and functional features of central nervous system lymphatic vessels. Nature <i>523</i>, 337-341. <a class="external-link" href="https://doi.org/10.1038/nature14432">https://doi.org/10.1038/nature14432</a></p>
<p id="bib10">Ramos-Vara, J.A. and Miller, M.A. (2014). When tissue antigens and antibodies get along: revisiting the technical aspects of immunohistochemistry--the red, brown, and blue technique. Vet. Pathol. <i>51</i>, 42-87. <a class="external-link" href="https://doi.org/10.1177/0300985813505879">https://doi.org/10.1177/0300985813505879</a></p>
<p id="bib11">Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to ImageJ: 25 years of image analysis. Nat. Methods <i>9</i>, 671-675. <a class="external-link" href="https://doi.org/10.1038/nmeth.2089">https://doi.org/10.1038/nmeth.2089</a></p>
<p id="bib12">van Gijlswijk, R.P., Zijlmans, H.J., Wiegant, J., Bobrow, M.N., Erickson, T.J., Adler, K.E., Tanke, H.J., and Raap, A.K. (1997). Fluorochrome-labeled tyramides: use in immunocytochemistry and fluorescence in situ hybridization. J. Histochem. Cytochem. <i>45</i>, 375-382. <a class="external-link" href="https://doi.org/10.1177/002215549704500305">https://doi.org/10.1177/002215549704500305</a></p>
<p id="bib13">Wang, F., Flanagan, J., Su, N., Wang, L.C., Bui, S., Nielson, A., Wu, X., Vo, H.T., Ma, X.J., and Luo, Y. (2012). RNAscope: a novel in situ RNA analysis platform for formalin-fixed, paraffin-embedded tissues. J. Mol. Diagn. <i>14</i>, 22-29. <a class="external-link" href="https://doi.org/10.1016/j.jmoldx.2011.08.002">https://doi.org/10.1016/j.jmoldx.2011.08.002</a></p>
<p id="bib14">Young, A.P., Jackson, D.J., and Wyeth, R.C. (2020). A technical review and guide to RNA fluorescence in situ hybridization. PeerJ <i>8</i>, e8806. <a class="external-link" href="https://doi.org/10.7717/peerj.8806">https://doi.org/10.7717/peerj.8806</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Olivia Steele-Mortimer (<a href="mailto:omortimer@niaid.nih.gov">omortimer@niaid.nih.gov</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>All data reported in this paper will be shared by the lead contact upon request. This paper does not report original code.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1521-Mmc1.pdf">Document S1</a></p>
<h3>Acknowledgments</h3>
<p>We would like to thank Ryan Kissinger for assistance with the graphical abstract and figure 1 and Austin Athman for assistance with figures, videos, and video voice over. We would also like to thank Clayton Winkler, Adam Nock, and members of the Steele-Mortimer lab for critical reading of the protocol. This work was supported by the <a href="https://doi.org/10.13039/100006492">Division of Intramural Research, National Institute of Allergy and Infectious Diseases</a>, <a href="https://doi.org/10.13039/100000002">NIH</a> (to O.S-M.).</p>
<h3>Author contributions</h3>
<p>Conceptualization, O.R.N. and R.R.; Investigation, O.R.N. and L.K.; Writing – Original Draft, O.R.N. and O.S-M.; Writing – Review &amp; Editing, O.R.N., L.K., R.R., and O.S-M.; Funding Acquisition, O.S-M.; Supervision, L.K. and O.S-M.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>