<article>
<section>
<h1 id="header">VirMutSig: Discovery and assignment of viral mutational signatures from sequencing data</h1>
<p><time datetime="2021-10-29">Published: October 29, 2021</time></p>
<p>Davide Maspero,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">6</a>,<a href="#cor1">*</a></sup> Fabrizio Angaroni,<sup><a href="#aff2">2</a></sup> Danilo Porro,<sup><a href="#aff1">1</a></sup> Rocco Piazza,<sup><a href="#aff4">4</a>,<a href="#fn1">5</a></sup> Alex Graudenzi,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#fn1">5</a>,<a href="#cor2">**</a></sup> and Daniele Ramazzotti<sup><a href="#aff4">4</a>,<a href="#fn1">5</a>,<a href="#fn3">7</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Inst. of Molecular Bioimaging and Physiology, Consiglio Nazionale delle Ricerche (IBFM-CNR), Segrate, Milan, Italy</p>
<p id="aff2"><sup>2</sup>Department of Informatics, Systems and Communication, Univ. of Milan-Bicocca, Milan, Italy</p>
<p id="aff3"><sup>3</sup>Bicocca Bioinformatics, Biostatistics and Bioimaging Centre – B4, Milan, Italy</p>
<p id="aff4"><sup>4</sup>Department of Medicine and Surgery, Univ. of Milan-Bicocca, Monza, Italy</p>
<p id="fn1"><sup>5</sup>Senior author</p>
<p id="fn2"><sup>6</sup>Technical contact</p>
<p id="fn3"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:d.maspero@campus.unimib.it">d.maspero@campus.unimib.it</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:alex.graudenzi@ibfm.cnr.it">alex.graudenzi@ibfm.cnr.it</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:daniele.ramazzotti@unimib.it">daniele.ramazzotti@unimib.it</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100911">10.1016/j.xpro.2021.100911</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We describe the procedures to perform the following: (1) the <i>de novo</i> discovery of mutational signatures from raw sequencing data of viral samples and (2) the association of existing viral mutational signatures to the samples of a given dataset. The goal is to identify and characterize the nucleotide substitution patterns related to the mutational processes that underlie the origination of variants in viral genomes. The VirMutSig protocol is available at this link: <a href="https://github.com/BIMIB-DISCo/VirMutSig">https://github.com/BIMIB-DISCo/VirMutSig</a>.</p>
<p>For complete information on the theoretical aspects of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref11">Graudenzi et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Distinct mutational processes underlie the origination of viral variants</li>
<li>VirMutSig implements variant calling from raw sequencing data of viral genomes</li>
<li>The tool performs de novo discovery of mutational signatures (substitution patterns)</li>
<li>Robust assignment of signature activity to samples is performed via bootstrap</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1132-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Problem description</h3>
<p>The VirMutSig protocol aims at identifying mutational signatures from raw sequencing data of viral samples, as originally proposed in the context of cancer evolution in (<a href="#bib1">Alexandrov et al., 2013</a>) and in the analysis of SARS-CoV-2 in (<a href="#bib11">Graudenzi et al., 2021</a>). Mutational signatures represent the decomposition of the categorical distribution of nucleotide substitutions that are observed in the samples of a given dataset, and which might be due to distinct mutational processes. Such processes could be endogenous (e.g., APOBEC deaminase activity causes mainly C to T substitutions) or exogenous (e.g., tobacco smoke causes mainly C to A substitutions).</p>
<p>We formulated the problem of <i>de novo</i> signature discovery and assignment as a Non-negative Matrix Factorization (NMF) problem (<a href="#bib16">Lal et al., 2021</a>; <a href="#bib11">Graudenzi et al., 2021</a>).</p>
<p>In brief, the counts of nucleotide substitutions observed in all viral samples of a given dataset, after preprocessing and variant calling, result in an input data matrix <b>D</b>, composed by <i>n</i> samples (rows) X <i>m</i> nucleotide substitution categories (e.g., the C to T category will include the count of all variants with such substitution in any given sample) (columns).</p>
<p>Formally, <b>D</b> is factorized in the following matrices:</p>
<p>Signature matrix (<b>B</b>): this matrix specifies the composition of every detected signature in terms of the selected substitution categories.</p>
<p>Exposure matrix (<b>A</b>): it represents the coefficients of the linear combination of signatures present in a sample. This matrix evaluates the activity of the various signatures in each sample of the dataset.</p>
<p>As we are dealing with noisy data, we propose a stochastic optimization performed via a Monte Carlo Markov Chain (MCMC) to find the best <b>A</b> and <b>B</b> matrices such that <b>A</b>x<b>B</b> ∼ <b>D</b> (see the <a href="#abssec0015">graphical abstract</a>). The method computes <b>A</b> and <b>B</b> for a number of signatures in a user-defined range (e.g., from 2 to 10) and allows one to select the optimal number by assessing different metrics (see below).</p>
<p>Specifically, we release two R scripts named respectively: denovo.R and assignment.R.</p>
<ol>
<li>denovo.R allows one to perform the inference of both <b>A</b> and <b>B.</b> In particular:
<ol type="a">
<li>
<b>B</b> is randomly initialized.</li>
<li>
<b>A</b> is inferred given <b>B</b> via non-negative least squares (NNLS).</li>
<li>
<b>B</b> is then inferred given <b>A</b> via NNLS.<br> Tasks “b” and “c” are repeated until convergence. The whole procedure is repeated from task “a” for a sufficient number of times (e.g., at least 100), by testing the rank of <b>B</b> (i.e., the number of signatures) within a user specified range.<br> The output of this procedure is:</li>
</ol>
</li>
</ol>
<p><b>B</b>, obtained as the consensus from all the proposed matrices; plots regarding different metrics, so to estimate the optimal number of signatures present in the dataset (see below).</p>
<ol start="2">
<li>The assignment.R script employs as input a <b>B</b> matrix that could be provided by the user or obtained with the script denovo.R. Then, it infers <b>A</b> similarly as done in the task “b” of the denovo.R script (see above).</li>
</ol>
<p>Finally, a <i>bootstrap</i> procedure is performed to assess the statistical significance of the signature activities. To do this, we consider each sample as a categorical distribution over the given contexts, and we extract, with resampling, from such distributions the same number of mutations as observed in the sample. Then, given the bootstrapped dataset, we fit again <b>A</b>.</p>
<p>We repeat this process multiple times (e.g., 100) to obtain a distribution for each entry of <b>A</b>. So, a p-value of the significance of the activity of each signature in a given sample can be returned.</p>
<h3 id="sec1.2">Overall structure of the VirMutSig protocol</h3>
<p>The protocol is subdivided into 4 distinct parts:</p>
<p>Part 1) INSTALLATION</p>
<p>Part 2) DATA PREPROCESSING PIPELINE (via Nextflow)</p>
<p>Part 3) DE NOVO DISCOVERY OF MUTATIONAL SIGNATURES (denovo.R)</p>
<p>Part 4) ASSIGNMENT OF EXISTING MUTATIONAL SIGNATURES (assignment.R)</p>
<p>The protocol is publicly available at this link: <a href="https://github.com/BIMIB-DISCo/VirMutSig">https://github.com/BIMIB-DISCo/VirMutSig</a>.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>Reagent or RESOURCE</th>
<th>Source</th>
<th>Identifier</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Public database analyzed [example]: RNA-seq</td>
<td>NCBI</td>
<td>[<i>example</i>] PRJNA610428 (<a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA610428">https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA610428</a>)</td>
</tr>
<tr>
<td>Public database analyzed [example]: Amplicon</td>
<td>NCBI</td>
<td>[<i>example</i>] PRJNA645906 (<a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA645906">https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA645906</a>)</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>VirMutSig</td>
<td><a href="#bib11">Graudenzi et al. (2021)</a></td>
<td><a href="https://github.com/BIMIB-DISCo/VirMutSig">https://github.com/BIMIB-DISCo/VirMutSig</a></td>
</tr>
<tr>
<td>Unix/MacOS operating system</td>
<td>Canonical Ltd.Apple Inc.</td>
<td>Ubuntu 20.04 Focal macOS 10.15 Catalina</td>
</tr>
<tr>
<td>Docker</td>
<td>Merkel, (2014).</td>
<td>
<a href="https://www.docker.com/get-started">https://www.docker.com/get-started</a> (v. 20)</td>
</tr>
<tr>
<td>Docker Image</td>
<td>Docker Hub</td>
<td>dcblab/virmutsig_img:latest (<a href="https://hub.docker.com/r/dcblab/virmutsig_img">https://hub.docker.com/r/dcblab/virmutsig_img</a>)</td>
</tr>
<tr>
<td>Nextflow</td>
<td><a href="#bib22">Di Tommaso et al. (2017)</a></td>
<td>
<a href="https://www.nextflow.io/">https://www.nextflow.io/</a> (v. 21)</td>
</tr>
<tr>
<td>R</td>
<td>The R Foundation</td>
<td>
<a href="https://www.r-project.org">https://www.r-project.org</a> (v. 4)</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Computational requirements</h3>
<p>To execute the VirMutSig protocol, the user requires a computer with the following software specifications:</p>
<h4 id="sec2.1.1">Software</h4>
<ul>
<li>Unix/OS operating system (for details on the usage of the protocol on Windows OS, please refer to the <a href="#sec7.9">troubleshooting problem 5</a>).</li>
<li>Nextflow (version = 21) (<a href="https://www.nextflow.io/">https://www.nextflow.io/</a>)</li>
<li>Docker (version = 20) (<a href="https://www.docker.com/get-started">https://www.docker.com/get-started</a>)</li>
<li>R (version = 4) with the installed packages listed below (<a href="https://www.r-project.org">https://www.r-project.org</a><u>)</u>
</li>
<li>The Docker image of VirMutSig (<a href="https://hub.docker.com/r/dcblab/virmutsig_img">https://hub.docker.com/r/dcblab/virmutsig_img</a>)</li>
<li>A working internet connection (<i>for installation only</i>).</li>
</ul>
<p>All the other required tools and libraries will already be installed in the provided Docker image.</p>
<table id="undtbl1">
<thead>
<tr>
<th colspan="3">Software and R packages</th>
</tr>
<tr>
<th>Part</th>
<th>Package</th>
<th>Version</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3">1–2) INSTALLATION and DATA PREPROCESSING PIPELINE</td>
<td>Nextflow (<a href="#bib22">Di Tommaso et al., 2017</a>)</td>
<td>version = 21</td>
</tr>
<tr>
<td>Docker (<a href="#bib17">Merkel 2014</a>)Docker image: dcblab/virmutsig_img</td>
<td>version = 20.10.8version = latest</td>
</tr>
<tr>
<td>R</td>
<td>version = 4.0.1</td>
</tr>
<tr>
<td rowspan="9">3) DE NOVO DISCOVERY OF MUTATIONAL SIGNATURES (denovo.R)</td>
<td>BiocManager (<a href="#bib18">Morgan 2021</a>)Biobase (<a href="#bib12">Huber et al., 2015</a>)</td>
<td>version = 1.30.16version = 2.46</td>
</tr>
<tr>
<td>data.table (<a href="#bib7">Dowle and Srinivasan, 2021</a>)</td>
<td>version = 1.14.0</td>
</tr>
<tr>
<td>ggplot2 (<a href="#bib24">Wickham 2016</a>)</td>
<td>version = 3.3.5</td>
</tr>
<tr>
<td>gridExtra (<a href="#bib3">Auguie 2017</a>)</td>
<td>version = 2.3</td>
</tr>
<tr>
<td>NMF (<a href="#bib10">Gaujoux and Seoighe, 2020</a>)</td>
<td>version = 0.23.0</td>
</tr>
<tr>
<td>nnls (<a href="#bib19">Mullen and van Stokkum, 2012</a>)</td>
<td>version = 1.4</td>
</tr>
<tr>
<td>Optparse (<a href="#bib23">Davis, 2020</a>)</td>
<td>version = 1.6.6</td>
</tr>
<tr>
<td>Stringi (<a href="#bib9">Gagolewski 2021</a>)</td>
<td>version = 1.7.4</td>
</tr>
<tr>
<td>Yaml (<a href="#bib21">Stephens, 2020</a>)</td>
<td>version = 2.2.1</td>
</tr>
<tr>
<td rowspan="4">4) ASSIGNMENT OF EXISTING MUTATIONAL SIGNATURES (assignment.R)</td>
<td colspan="2">All the above denovo packages</td>
</tr>
<tr>
<td>Glmnet (<a href="#bib14">Friedman et al., 2010</a>)</td>
<td>version = 4.1–2</td>
</tr>
<tr>
<td>Lsa (<a href="#bib8">Wild, 2020</a>)</td>
<td>version = 0.73.2</td>
</tr>
<tr>
<td>Matrix (<a href="#bib6">Bates and Maechler, 2021</a>)</td>
<td>version = 1.3.4</td>
</tr>
</tbody>
</table>
<h4 id="sec2.1.2">Hardware</h4>
<ul>
<li>Parts 1–2</li>
</ul>
<p>The resources required for processing a single sample are limited, but we suggest running Part 2 in parallel, if possible. Memory: 8Gb required, processors: 1 required, 8 recommended.</p>
<ul>
<li>Parts 3–4</li>
</ul>
<p>Memory: 4GB required, processors: 1 required, 4 recommended.</p>
<h3 id="sec2.2">Input data</h3>
<p>The VirMutSig protocol requires as input either: (<i>i</i>) RNA-seq, or (<i>ii</i>) Amplicon Illumina raw data, generated from sequencing experiments of viral samples (obtained, e.g., from primary isolates), and typically available as FASTQ files.</p>
<p>Generally, one will have a FASTQ file for each sample of the dataset.</p>
<p>Notice that the protocol works with both (<i>i</i>) single- and (<i>ii</i>) paired-end sequencing library preparation layout.</p>
<h4 id="sec2.2.1">FASTQ files</h4>
<p>FASTQ files can be collected in different ways.</p>
<p>If one is interested in the analysis of public datasets, e.g., those available on NCBI SRA database (<a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>), one can create a file with a list of SRR accession number (one per line) to use as input data.</p>
<p>The VirMutSig will automatically download the proper FASTQ files; see <a href="#sec3.2">Part 2</a>.</p>
<p>Instead, if one has produced her/his own FASTQ files, or if they are available in other databases, one must aggregate them in one directory and specify its path as input; see <a href="#sec3.2">Part 2</a>.</p>
<h4 id="sec2.2.2">Reference genome file</h4>
<p>A reference genome is required to perform variant calling. In (<a href="#bib20">Ramazzotti et al., 2021</a>) we proposed the SARS-CoV-2-ANC as reference genome. Such genome is already available in the VirMutSig GitHub repository and is used as default.</p>
<p>However, any viral genome can be used as reference. To do so, the chosen genome must be provided as a FASTA file; see <a href="#sec3.2">Part 2</a>.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Part 1. Installation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<ol>
<li>Downloading VirMutSig<br> The installation of VirMutSig is done by downloading the GitHub repository in a local directory, which includes the VirMutSig scripts and the example files.<br> After the installation, a new folder named VirMutSig will be generated. Please install VirMutSig by moving to your local directory and using GitHub with the following command in the terminal:
<div class="textbox">
<p><code>&gt;git clone https://github.com/BIMIB-DISCo/VirMutSig.git</code></p>
</div>
VirMutSig includes 4 directories with the following names:
<ol type="a">
<li>“preprocessing”<br> This directory contains all the files and directories required to perform Part 2, such as:
<ol type="i">
<li>SRR_to_SNVlist.nf: the Nextflow pipeline file</li>
<li>nextflow.config: the config file with the preprocessing settings and parameters</li>
<li>reference: a directory with the SARS-CoV-2-ANC reference file</li>
<li>bin: a directory with an R script used to create the SNVs list for Parts 3–4.</li>
</ol>
</li>
<li>“denovo” This directory contains the corresponding R script to perform the <i>de novo</i> discovery of mutational signatures (Part 3). The files included are:
<ol type="i">
<li>denovo.R<br> This script performs the discovery of viral mutational signatures from the list of selected SNVs taken as input, by employing a NMF approach described in the above section or in (<a href="#bib16">Lal et al., 2021</a>). The user must also specify the number of contexts, i.e., the flanking bases to the genome positions. They may either be 6 or 96, please refer to (<a href="#bib16">Lal et al., 2021</a>) for further details.</li>
<li>denovo_config.yaml<br> This file contains the parameters explained above. It could be modified using any text editor.</li>
<li>denovo_utils.R This R file contains some functions used by the main denovo.R scripts.
<div class="note">
<span class="note-title">Note:</span> please do not modify this file.</div>
</li>
</ol>
</li>
<li>“assignment” This directory contains the corresponding R script to perform the <i>assignment</i> of the activity of mutational signatures to each sample (Part 4). The files included are:
<ol type="i">
<li>assignment.R<br> This script takes as input the signatures.txt file generated by the denovo.R script or provided by the user, and the list of SNVs to perform the assignment of the signatures to each sample. Bootstrap can be employed to assess the statistical confidence of the signature assignments.</li>
<li>assignment_config.yaml<br> This file contains the parameters explained above. It could be modified using any text editor.</li>
<li>assignment_utils.R This R file contains some functions used by the main assignment.R scripts.
<div class="note">
<span class="note-title">Note:</span> please do not modify this file.</div>
</li>
</ol>
</li>
<li>“example”<br> This directory includes an example of the VirMutSig analysis performed on 150 FASTQ files obtained from samples of SARS-CoV-2 via RNA sequencing experiments.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Download docker image</li>
</ol>
<p>The preprocessing pipeline executes all the steps using a Docker image in which all the requested software is already installed to avoid compatibility issues.</p>
<p>Docker can be obtained following the instruction at this link: (<a href="https://www.docker.com/get-started">https://www.docker.com/get-started</a>).</p>
<p>Once the installation is completed, please get the protocol image called ‘dcblab/virmutsig_img’ by digiting the following command in a terminal:</p>
<div class="textbox">
<p><code>&gt;docker pull dcblab/virmutsig_img:latest</code></p>
</div>
<ol start="3">
<li>Verify docker image installation</li>
</ol>
<p>To test if the image is correctly installed in your system, please execute the following code:</p>
<div class="textbox">
<p><code>&gt;docker image ls</code></p>
</div>
<p>which should display the following lines:</p>
<div class="textbox">
<p><code>REPOSITORY       TAG    IMAGE ID      CREATED    SIZE</code></p>
<p><code>dcblab/virmutsig_img  latest    223f27c12b45    4 hours ago  1.56GB</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> in this case, it is possible to proceed to the next steps.</div>
<h3 id="sec3.2">Part 2. Data preprocessing pipeline</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min download + 70 min pipeline execution for each sample (can be parallelized)</div>
<p>The data preprocessing pipeline included in the VirMutSig protocol is provided as a Nextflow pipeline file, named “SRR_to_SNVlist.nf” and included in the “preprocessing” folder of the GitHub repository: <a href="https://github.com/BIMIB-DISCo/VirMutSig">https://github.com/BIMIB-DISCo/VirMutSig</a>.</p>
<p>The folder also includes a file named: “nextflow.config”, which must be opportunely modified according to the specific experimental settings (see below for the parameter description).</p>
<p>The preprocessing pipeline includes the following processes (detailed in the following): data acquisition, trimming, alignment, remove duplicated reads (optional), get depth information, variant calling, and variant filtering.</p>
<p>As output, the preprocessing pipeline returns a file named: “SNV_list.txt” in which:</p>
<p>rows correspond to all the single nucleotide variants (SNVs) detected in all the samples of the dataset</p>
<p>columns correspond to: sample ID, variant ID, genome position, reference allele, alternative allele, reference three nucleotides (flanking bases), supporting reads, coverage, variant frequency (VF), and p-value (returned by the variant caller).</p>
<p>To run the preprocessing processes of the protocol, simply move to the “preprocessing” folder and execute the following command from the terminal:</p>
<div class="textbox">
<p><code>&gt;nextflow run SRR_to_SNVlist.nf</code></p>
</div>
<p>In the following, we describe the preprocessing processes and the related settings in detail.</p>
<ol start="4">
<li>Data acquisition<br> From now on, we will consider 'preprocessing' as a working directory from the one specified during the installation: ‘user_local_directory/VirMutSig/preprocessing' Input data can be either (<i>i</i>) downloaded from public repositories, or (<i>ii</i>) provided from local folders, if available.
<ol type="a">
<li>Input data acquisition from public repositories Input data can be downloaded from public repositories such as, e.g., the NCBI Sequence Read Archive (SRA) (<a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>). In this case, one can use the SRA Run Selector web interface (<a href="https://www.ncbi.nlm.nih.gov/Traces/study/">https://www.ncbi.nlm.nih.gov/Traces/study/</a>) to search for a dataset and download the “Accession List” by using the related button. The obtained list of SRR IDs (one per line) can be passed as input to download the files via SRA-toolkit, by editing the following parameter of the “nextflow.config”:
<div class="textbox">
<p><code>params.FASTQ_input = '../example/SRAlist_paired.txt'</code></p>
</div>
Also in this case, the library preparation layout (single-end or paired-end) must be specified by editing the following parameter in “nextflow.config” file:
<div class="textbox">
<p><code>params.library_preparation = ‘paired’ // or ‘single’</code></p>
</div>
By default, the downloaded FASTQ files will be stored in the following path: '/intermediate/FASTQ' (relative to SRR_to_SNVlist.nf file). It possible to change the directory by editing the following parameter in in “nextflow.config” file:
<div class="textbox">
<p><code>params.FASTQdir = 'intermediate/FASTQ'</code></p>
</div>
</li>
<li>Input data from local folders</li>
</ol>
</li>
</ol>
<p>In this case, the user must indicate the directory where the FASTQ files are located, editing the following parameter of the “nextflow.config”:</p>
<div class="textbox">
<p><code>params.FASTQ_input = '/Path/To/Directory/'</code></p>
</div>
<p>The library preparation layout (single-end or paired-end) must be specified by editing the following parameter in “nextflow.config” file:</p>
<div class="textbox">
<p><code>params.library_preparation = ‘paired’ //</code><code>or ‘single’</code></p>
</div>
<p>Notice that with the “paired” configuration two FASTQ files are expected for each sample, formatted as (sampleID_1.fastq.gz and sampleID_2.fastq.gz).</p>
<p>Conversely, with the “single” configuration one sampleID.fastq.gz file is expected for each sample.</p>
<ol start="5">
<li>Trimming</li>
</ol>
<p>This step aims at removing from the sequence the nucleotides with low sequencing quality. To perform this step, the Nextflow pipeline exploits the tool Trimmomatic (<a href="http://www.usadellab.org/cms/?page=trimmomatic">http://www.usadellab.org/cms/?page=trimmomatic</a>).</p>
<p>To tune the additional parameters used by Trimmomatic please edit the following parameter in “nextflow.config” file:</p>
<div class="textbox">
<p><code>  params.trimmomatic_setting = 'LEADING:20 TRAILING:20</code></p>
<p><code>SLIDINGWINDOW:4:20 MINLEN:40'</code></p>
</div>
<p>In brief, LEADING and TRAILING parameters cut the bases that display a quality below a certain threshold, respectively at the start and the end of a read (20).</p>
<p>The SLIDINGWINDOW parameter sets the size of a sliding window (4 in our example) and deletes all the bases from the leftmost position of the window to the end of the read, when the average quality detected in the window drops below a given threshold (e.g., 20).</p>
<p>Finally, the MINLEN parameter specifies the minimum length of a read to be kept (40 bases in our example). Please, refer to the Trimmomatic documentation for more details.</p>
<ol start="6">
<li>Alignment<br> Reads need to be aligned to a reference genome, which can be selected by the user, e.g.,
<ol type="a">
<li>SARS-CoV-2-ANC (<a href="#bib20">Ramazzotti et al., 2021</a>),</li>
<li>EPI_ISL_405839 / GeneBank ID: MN975262.1 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/MN975262.1">https://www.ncbi.nlm.nih.gov/nuccore/MN975262.1</a>) (<a href="#bib4">Bastola et al., 2020</a>),</li>
<li>EPI_ISL_402125 / NCBI ID: NC_045512.2 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/1798174254">https://www.ncbi.nlm.nih.gov/nuccore/1798174254</a>) (<a href="#bib2">Andersen et al., 2020</a>).</li>
</ol>
</li>
</ol>
<p>In the subfolder “preprocessing/reference” of the GitHub repository, we provide the SARS-CoV-2-ANC genome in FASTA format.</p>
<p>The user can specify the reference genome file by editing following parameter in “nextflow.config” file:</p>
<div class="textbox">
<p><code>params.fasta = 'reference/SARS-CoV-2-ANC.fasta'</code></p>
</div>
<p>The alignment is performed with BWA-MEM (<a href="https://github.com/lh3/bwa">https://github.com/lh3/bwa</a>), which generates a SAM file including the aligned reads. All the associated files used by the BWA aligner will be automatically generated and placed in the same reference genome folder.</p>
<p>Each SAM file will be sorted and compressed into a BAM file with Samtools (<a href="http://www.htslib.org/">http://www.htslib.org/</a>).</p>
<p>By default, the BAM files will be stored in the following path: '/intermediate/BAM' (relative to SRR_to_SNVlist.nf file). It is possible to change the directory by editing the following parameter in in “nextflow.config” file:</p>
<div class="textbox">
<p><code>params.BAMdir = 'intermediate/BAM'</code></p>
</div>
<ol start="7">
<li>Remove duplicated reads (optional)</li>
</ol>
<p>Often, after obtaining an aligned BAM file, is useful to mark and remove duplicated reads to reduce the impact of the amplification bias, especially with RNA-seq experiments. Otherwise, when dealing with Amplicon data, several duplicated reads are expected and should not be removed. For this reason, we made this step optional.</p>
<p>To include or skip this step in the preprocessing pipeline, please set accordingly the following parameter in “nextflow.config” file:</p>
<div class="textbox">
<p><code>params.remove_duplicates = “true” // or “false”</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> the tool used to perform this task is Picard (<a href="https://broadinstitute.github.io/picard/">https://broadinstitute.github.io/picard/</a>).</div>
<ol start="8">
<li>Get depth information</li>
</ol>
<p>The aim of this step is to obtain the depth information for each sample in every genome position (i.e., number of reads mapped on each position of the viral genome).</p>
<p>To perform this task, we used Samtools.</p>
<p>By default, the coverage files will be stored in the following path: '/intermediate/COVERAGE’ (relative to SRR_to_SNVlist.nf file). It is possible to change the directory by editing the following parameter in in “nextflow.config” file:</p>
<div class="textbox">
<p><code>params.COVERAGEdir = 'intermediate/COVERAGE'</code></p>
</div>
<ol start="9">
<li>Variant calling</li>
</ol>
<p>Variants can be called comparing the aligned reads with the reference genome. To do so, we used Samtools (<a href="http://www.htslib.org/">http://www.htslib.org/</a>) and VarScan (<a href="http://varscan.sourceforge.net/">http://varscan.sourceforge.net/</a>). More in detail, we used the mpileup command included in Samtools which converts the BAM file into the pileup format required by VarScan.</p>
<p>Then we used the VarScan pileup2snp for variant calling to produce a VCF file for each BAM file. The VarScan setting can be adjusted by editing the following parameter included in “nextflow.config” file.</p>
<div class="textbox">
<p><code>params.varscan = '--min-var-freq 0.01 --p-value 1'</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> For more information, please refer to the VarScan documentation.</div>
<p>By default, the VCF files will be stored in the following path: '/intermediate/VCF’ (relative to SRR_to_SNVlist.nf file). It is possible to change the directory by editing the following parameter in in “nextflow.config” file:</p>
<div class="textbox">
<p><code>params.VCFdir = 'intermediate/VCF'</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> We suggest performing the alignment step according to the manufacturer’s recommendations for all sequencing technologies.</div>
<div class="critical">
<span class="critical-title">Critical:</span> All the above steps can be performed also by applying different pipelines for variant calling, such as the one proposed in [<a href="https://github.com/andersen-lab/ivar">https://github.com/andersen-lab/ivar</a>], which was specifically designed for handling viral amplicon data obtained via the <i>artic protocol</i>.</div>
<ol start="10">
<li>Variant filtering
<ol type="a">
<li>In order to reduce the impact of noise in the data (due, e.g., to sequencing issues), we adopted multiple quality control (QC) filters to select only the reliable SNVs. The last step of the preprocessing pipeline applies different filtering criteria on the detected SNVs. The following parameters determine the filtering threshold, and they can be set by changing the corresponding numeric value included into a string assigned to VirMutSig_QCfilter parameter present in the “nextflow.config” file.
<div class="textbox">
<p><code>params.SNV_filters = 'PV_THR:0.01 VAR_FREQ_THR:0.05 MIN_COV:20 ALT_READ_THR:3'</code></p>
</div>
<ol type="i">
<li>p-value on variant calling significance (PV_THR). The filter removes all the variants called with a significance p-value larger than a given threshold (default = 0.01).</li>
<li>Frequency threshold (VAR_FREQ_THR)<i>.</i> The filter keeps only variants observed in the data with a variant frequency that exceed the specified threshold (default = 0.05).</li>
<li>Minimum coverage (MIN_COV). The filter keeps only variants with the specified minimum coverage (default = 20).</li>
<li>Minimum alternative read count (ALT_READ_THR)<i>.</i> The filter keeps only variants with a minimum number of reads showing the alternative allele equal to the given threshold. (default = 3)
<div class="note">
<span class="note-title">Note:</span> Indels are not considered in the analysis.</div>
</li>
</ol>
</li>
<li>By default, the SNV_list.txt file will be stored in the 'example’ directory. It is possible to change the directory by editing the following parameter in in “nextflow.config” file:
<div class="textbox">
<p><code>params.SNVlistdir = '../example</code></p>
</div>
This step uses a R script included in the “preprocessing/bin” directory of the GitHub repository named <a href="https://github.com/BIMIB-DISCo/VirMutSig/blob/main/preprocessing/bin/makeVirMutSigInput.R">makeSNVlist.R</a>. It takes as input different arguments with the following fixed order:
<ol type="i">
<li>A string with the path of the directory containing the VCF files generated by the variant caller.</li>
<li>A string with the path of the directory containing the depth files generated by step 8. Such files must end with '.depth.txt'.</li>
<li>Reference file in fasta format. (e.g., SARS-CoV-2-ANC.fasta)
<div class="note">
<span class="note-title">Note:</span> If another pipeline has been used to perform variant calling, the R script can be used to aggregate the vcf files into a SNV list. Instead, if VirMutSig preprocessing steps have been used, the script is automatically executed via Nextflow.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Parameters must be set according to the specific features of the datasets (see, e.g., the guidelines proposed on the website: <a href="https://virological.org/">https://virological.org/</a>).</div>
<ol start="11">
<li>Further information</li>
</ol>
<p>The Trimming and Alignment step require greater computational resources than the other processes. For this reason, it is possible to specify the maximum number of cores to be used, by changing the cpus value in the following setting of the “nextflow.config” file:</p>
<div class="textbox">
<p><code>process {</code></p>
<p><code>      withName: 'Trimming_single' {cpus = 4}</code></p>
<p><code>      withName: 'Trimming_paired' {cpus = 4}</code></p>
<p><code>      withName: 'Alignment_and_sorting_single' {cpus = 8}</code></p>
<p><code>      withName: 'Alignment_and_sorting_paired' {cpus = 8}</code></p>
<p><code>      }</code></p>
</div>
<p>An increase of the cpus number assigned to the processes reduces the computational time required to trim and align the reads, but it also reduces the number of samples that can be analyzed in parallel. For these reasons, one should select a proper value based on the available computational resources and number of samples.</p>
<p>All parameters (“params.”) can be specified by overriding when the pipeline is launched. To do so, please specify the corresponding parameter name (the string following “params.”) and specify the opportune argument of the Nextflow command.</p>
<p>In the example, the SRR list file path and the output directory path of the SNVs list file will be specified without editing the “nextflow.config” file.</p>
<div class="textbox">
<p><code>&gt;nextflow run SRR_to_SNVlist.nf \</code></p>
<p><code>--FASTQ_input ‘SRRfile/custom/path’ \</code></p>
<p><code>--SNVlistdir ‘SNVlist/output/path’</code></p>
</div>
<p>For a summary of the settings and processes executed with the example configuration please see <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1132-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Summary of the settings and processes executed with the example configuration</div>
<p>We analyzed 150 samples with a paired-end library layout, downloaded from the SRA database. The related processes are automatically executed via Nextflow.</p>
</figcaption>
</figure>
<h3 id="sec3.3">Part 3. <i>De novo</i> discovery of mutational signatures (denovo.R)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼20 min (approximate time required to perform the step on a dataset with ∼100 samples and ∼10,000 mutations, with a significant variability related to the signature rank range)</div>
<ol start="12">
<li>Input files and setup The denovo.R script requires two input files to be executed.
<ol type="a">
<li>List of selected SNVs [SNV_list.txt] This file is generated following the preprocessing step. It is a semicolon-separated file with a SNV for each line. It includes (at least) the following column headers:
<ol type="i">
<li>SampleId: The ID of the sample.</li>
<li>Position: The genome position.</li>
<li>Reference: the reference allele (A, T, C, G).</li>
<li>Alternative: the alternative allele (A, T, C, G).</li>
<li>VariantCount: the number of the reads including the alternative allele.</li>
<li>TotalCount: the total number of reads covering the genome position.<br> For the 96-contexts analysis (see below) the following column is also required:</li>
<li>ReferenceTrinucleotide: the triplet with the reference bases before and after the variant position (e.g., ATC where T is the reference). Please see the file SNV_list.txt contained in the “/example” directory for an example of input formatting (see <a href="#tbl1">Table 1</a>).
<table id="tbl1" class="dense-table">
<caption>Table 1. First 5 lines of the SNV_list.txt file imported in Rstudio</caption>
<thead>
<tr>
<th>SampleID</th>
<th>VariantID</th>
<th>Position</th>
<th>Reference</th>
<th>Alternative</th>
<th>Reference Trinucleotide</th>
<th>VariantCount</th>
<th>TotalCount</th>
<th>Variant frequency</th>
<th>Pvalue</th>
</tr>
</thead>
<tbody>
<tr>
<td>SRR12661198</td>
<td>9996_C_T</td>
<td>9996</td>
<td>C</td>
<td>T</td>
<td>TCA</td>
<td>494</td>
<td>5720</td>
<td>0.0864</td>
<td>2.85E-144</td>
</tr>
<tr>
<td>SRR12661096</td>
<td>9965_C_T</td>
<td>9965</td>
<td>C</td>
<td>T</td>
<td>TCT</td>
<td>68</td>
<td>1263</td>
<td>0.0538</td>
<td>4.82E-20</td>
</tr>
<tr>
<td>SRR12833654</td>
<td>9960_G_T</td>
<td>9960</td>
<td>G</td>
<td>T</td>
<td>TGT</td>
<td>49</td>
<td>641</td>
<td>0.0764</td>
<td>6.85E-16</td>
</tr>
<tr>
<td>SRR12661080</td>
<td>995_C_T</td>
<td>995</td>
<td>C</td>
<td>T</td>
<td>ACG</td>
<td>165</td>
<td>2618</td>
<td>0.063</td>
<td>5.54E-48</td>
</tr>
<tr>
<td>SRR12661132</td>
<td>9945_G_T</td>
<td>9945</td>
<td>G</td>
<td>T</td>
<td>AGA</td>
<td>41</td>
<td>389</td>
<td>0.1054</td>
<td>1.50E-13</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>Configuration file [denovo_config.yaml] The parameters to run denovo.R are set in a text file called as default ‘denovo_config.yaml’. The file must contain a parameter in each line formatted as:
<div class="textbox">
<p><code>PARAMETER NAMES: value</code></p>
</div>
The parameters that can be modified are the following:<br> [variant selection]
<ol type="i">
<li>CLONAL_SNV_THR: Double [0,1]. default = 0.9.<br> This parameter defines the variant frequency threshold that determines whether a given variant is clonal.</li>
<li>MINOR_SNV_SEL: String {‘always’, ‘all’}. default = ‘always’.<br> To reduce the bias induced by mutations transmitted and inherited in the population during the epidemic spread, for the signature analysis we select and employ only the SNVs that are never observed with a frequency higher than <i>CLONAL_SNV_THR</i> in any sample. The selected SNVs are defined as ‘<i>always minor’</i>. This parameter together with the parameter CLONAL_SNV_THR are critical as they determine the variants that are selected for the signatures analysis; the default parameters that we suggest here (i.e., CLONAL_SNV_THR = 0.90 and MINOR_SNV_SEL ="always") are very conservative and they should be determined based on the aim of the study. For further details, please refer to (<a href="#bib20">Ramazzotti et al., 2021</a>).</li>
<li>MAX_SNV_SAMPLE: Integer [1, Inf]. default = 100.<br> In order to reduce the impact of highly mutated samples, likely due to degradation of their biological isolation, we suggest removing the samples exhibiting more than a user selected number of variants.</li>
<li>MIN_SNV_SAMPLE: Integer [1, Inf]. default = 6.<br> To assess the existence of statistically significant mutational signatures, we remove all the samples with number of detected SNVs lower than this value.<br> [analysis parameters]</li>
<li>NUM_CONTEXTS: Integer {6, 96}. default = 6.<br> This parameter specifies the number of different nucleotide substitution types (based on the flanking bases) that are considered.<br> 6 indicates that no flanking bases are considered. In this case, the possible substitution types are: C&gt;A (or G&gt;T), C&gt;G (or G&gt;C), C&gt;T (or G&gt;A), T&gt;A (or A&gt;T), T&gt;C (or A&gt;G), T&gt;G (or A&gt;C).<br> 96 indicates that the two flanking bases are considered. In this case, the possible substitution types include, e.g.,: A<b>C</b>A&gt;A<b>A</b>A (or A<b>G</b>A&gt;A<b>T</b>A), A<b>C</b>G&gt;A<b>A</b>G (or A<b>G</b>G&gt;A<b>T</b>G), A<b>T</b>A&gt;A<b>C</b>A (or A<b>A</b>A&gt;A<b>G</b>A), etc. For further details, please refer to (<a href="#bib16">Lal et al., 2021</a>).</li>
<li>MIN_NUM_SIG: Integer [1, <i>NUM_CONTEXTS</i>]. default = 1.<br> This parameter sets the lower bound of the range for the number of distinct signatures to be searched.</li>
<li>MAX_NUM_SIG: Integer [<i>MIN_NUM_SIG</i>, <i>NUM_CONTEXTS</i>]. default = 6<br> This parameter sets the upper bound of the range for the number of distinct signatures to be searched.</li>
<li>NMF_ITER: Integer [1, inf]. default = 100.<br> This parameter sets the number of Negative Matrix Factorization iterations performed during the inference of <b>A</b> and <b>B</b> matrices</li>
<li>SEED: Integer [0, Inf]. default = 0 (with 0 the seed will be randomly set).<br> This parameter initializes the random number generator. It is used to obtain reproducible results by keeping the same SEED.</li>
<li>N_CORE: Integer [0, Inf]. default = 0 (with 0 the number of cores available will automatically be detected).<br> This parameter indicates the maximum number of computational units (cpus) available for the inference.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> MAX_NUM_SIG and MIN_SNV_SAMPLE must be set accordingly with the NUM_CONTEXTS parameter. To obtain robust results, we suggest setting the former equal to half the number of contexts (i.e., 3 or 48) and the latter larger than the number of contexts.</div>
<ol start="13">
<li>Run denovo.R
<ol type="a">
<li>denovo.R can be run with the following command:
<div class="textbox">
<p><code>&gt;Rscript denovo.R \</code></p>
<p><code>--SNV_list path/to/SNV_list.txt \</code></p>
<p><code>--config_file path/to/denovo_config.txt \</code></p>
<p><code>--output_dir path/to/output/dir</code></p>
</div>
The script will generate in the output directory (specified by the user) the following three subdirectories:
<ol type="i">
<li>
<i>“</i>Signatures<i>”</i>:<br> This directory will contain a file for each number of searched signatures named ‘x_signatures.txt’. Each file will be formatted as a table with the context as header and a row for each signature.</li>
<li>
<i>“</i>Graphical<i>”</i>:<br> This directory will contain the graphical representation of each signature set found in the data.</li>
<li>
<i>“</i>Rank<i>”</i>:<br> This directory will contain a set of files to determine the optimal number of signatures.</li>
</ol>
</li>
<li>Unfortunately, no automated procedures are available to find the correct number of signatures, so we here provide a set of metrics to determine it. The denovo.R script provides four metrics as output and the relative plots:
<ol type="i">
<li>explained variance (<a href="#bib13">Hutchins et al., 2008</a>)</li>
<li>cophenetic coefficient (<a href="#bib5">Brunet et al., 2004</a>),</li>
<li>dispersion coefficient (<a href="#bib15">Kim and Park, 2007</a>),</li>
<li>silhouette consensus coefficient.</li>
</ol>
</li>
</ol>
</li>
</ol>
<p>The four metrics are shown in <a href="#fig2">Figure 2</a> and <a href="#tbl2">Table 2</a>.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1132-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Metrics to evaluate the best number of signatures</div>
<p>(A) shows the explained variance at different ranks; in this case we observed a bend at 3.</p>
<p>(B–D) show stability-based coefficients which report the consistency of NMF solutions across multiple iterations.</p>
</figcaption>
</figure>
<table id="tbl2">
<caption>Table 2. Metric coefficients for each different signature rank searched via denovo.R script</caption>
<thead>
<tr>
<th>Rank</th>
<th>Cophenetic coefficient</th>
<th>Dispersion coefficient</th>
<th>Silhouette consensus</th>
<th>Explained variance</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td><i>NA</i></td>
<td>1</td>
<td><i>NA</i></td>
<td>0.85</td>
</tr>
<tr>
<td>2</td>
<td>1</td>
<td>0.96</td>
<td>0.82</td>
<td>0.92</td>
</tr>
<tr>
<td>3</td>
<td>0.93</td>
<td>0.78</td>
<td>0.63</td>
<td>0.97</td>
</tr>
<tr>
<td>4</td>
<td>1</td>
<td>0.99</td>
<td>0.66</td>
<td>0.99</td>
</tr>
<tr>
<td>5</td>
<td>0.91</td>
<td>0.85</td>
<td>0.16</td>
<td>0.99</td>
</tr>
<tr>
<td>6</td>
<td>0.94</td>
<td>0.92</td>
<td><i>NA</i></td>
<td>1</td>
</tr>
</tbody>
</table>
<p>The first one measures how good is the fit of each given number of signatures, considering the observed mutational profiles. This metric is useful to estimate when the number of signatures is too high (i.e., overfitting). To this end, we suggest choosing the optimal number of signatures based on the <i>bend rule</i> that selects the number of signatures corresponding to the elbow in the explained variant plot (see the example in <a href="#fig2">Figure 2</a>A).</p>
<p>Often there is uncertainty to select a unique <i>elbow point</i> on the plot, so the latter three metrics can be used as additional evaluation of the optimal rank. Roughly, they provide a measure of stability of the NMF solutions over multiple runs (<i>NMF_ITER</i> parameter). These coefficients range from 0 to 1 and higher values indicate higher consistency among NMF solutions.</p>
<p>Considering all the above, our suggestion is to select one or few elbow points and among them select the one corresponding to higher cophenetic, dispersion, or silhouette coefficient. If still there is ambiguity it is reasonable to take the lower one.</p>
<p>Considering the example, and the metrics shown in <a href="#fig2">Figure 2</a>, we conservatively selected 3 as the optimal number of signatures, as at this rank we have a first bend in the explained variance and high values of the other metrics.</p>
<div class="note">
<span class="note-title">Note:</span> Please get from the ‘Signatures’ directory the corresponding file that should be then used in the following assignment step.</div>
<div class="note">
<span class="note-title">Note:</span> Each signature file contains a different number of signatures (identified with SIG_NUM) found in the data. In the ‘graphical’ directory, denovo.R generates pdf files with the same name as the corresponding signatures set. Those files contain a plot of the categorical distributions.</div>
<h3 id="sec3.4">Part 4. Assignment of existing mutational signatures (assignment.R)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min (approximate time required to perform the step on a dataset with ∼100 samples and ∼10 , 000 mutations, with significant variability related to bootstrap iterations)</div>
<ol start="14">
<li>Input files and setup<br> The assignment.R script requires three input files to be executed.
<ol type="a">
<li>List of selected SNVs [SNV_list.txt]<br> This file is the same used in Part 3, please see above.</li>
<li>Signatures file [x_signatures.txt]<br> This file contains for each signature the frequency of substitutions. Their values are grouped by context and normalized up to 1. The file must be formatted as a table with the context as header and a row for each signature.<br> The header must be formatted as: G&gt;T:C&gt;A;G&gt;C:C&gt;G;G&gt;A:C&gt;T;A&gt;T:T&gt;A;A&gt;G:T&gt;C;A&gt;C:T&gt;G.<br> Notice that, for instance, variants from G to T and C to A are aggregated together in the cases of 6-context. For further details, especially for the 96-contexts representation, please refer to (<a href="#bib1">Alexandrov et al., 2013</a>).<br> This file can be generated via Part 3 or can be directly passed by the user.</li>
<li>Configuration file [assignment_config.yaml]<br> The parameters to run assignment.R are set in a text file called as default ‘assignment_config.txt’. The file must contain a parameter in each line formatted as PARAMETER_NAMES: value<br> The parameters that can be modified are the following:<br> [variant selection]
<ol type="i">
<li>CLONAL_SNV_THR: Double [0,1]. default = 0.9. This parameter defines whether a given SNV is considered as clonal (default = 0.9).</li>
<li>MINOR_SNV_SEL: String {‘always’, ‘all’}. default = ‘always’. To reduce the bias induced by the mutation transmitted and inherited among the population during the epidemic spread we select for the signature analysis only SNVs never observed with a frequency higher than CLONAL_SNV_THR in any sample. The remaining SNVs are defined as ‘always minor’. For further details, please refer to (<a href="#bib11">Graudenzi et al., 2021</a>).</li>
<li>MAX_SNV_SAMPLE: Integer [1, Inf]. default = 100. In order to reduce the impact of highly mutated samples likely due to degradation of their biological isolation. We suggest removing the samples exhibiting more than a user selected number of variants.</li>
<li>MIN_SNV_SAMPLE: Integer [1, Inf]. default = 6. In order to assess the existence of statistically significant mutational signatures we have to remove all the samples with less than a given number of detected SNVs.<br> [analysis parameters]</li>
<li>BOOTSTRAP: String {‘yes’, ‘no’}. default = ‘yes’.</li>
<li>GOODNESS_FIT_THR: Double [0.5,1]. default = 0.95. This threshold indicates the minimum level of goodness of fit until when keep adding signatures (among the signatures.txt file) to the fit, in a given sample. The goodness of fit is measured with the cosine similarity between observed and predicted counts in each sample.</li>
<li>MIN_SIG_FREQ: Double [0,1]. default = 0.05. Each signature is considered to be present in a given sample if its activity (alpha value) is greater than the value of this parameter.</li>
<li>P_VALUE: threshold to be used when assessing significance of the exposure of samples to signatures. To this extent, Mann–Whitney U test is performed whose results are evaluated with the given P_VALUE threshold. default = 0.05</li>
<li>NUM_ITER: Integer [1, Inf]. default = 100</li>
<li>SEED: Integer [0, Inf]. default = 0 (with 0 the seed will be set randomly)</li>
<li>N_CORE: Integer [0, Inf]. default = 0 (with 0 the number of core available will automatically detected)</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> We suggest setting the parameters in accordance with the number of contexts. Similar to Part 3 MIN_SNV_SAMPLE should be larger than the number of contexts to provide reasonable results.</div>
<ol start="15">
<li>Run assignment.R</li>
</ol>
<p>assignment.R can be run by the following command:</p>
<div class="textbox">
<p><code>&gt;Rscript assignment.R \</code></p>
<p><code>--</code><code>SNV_list path_to_SNV_list.txt \</code></p>
<p><code>--</code><code>signature path_to_signature_x.txt \</code></p>
<p><code>--</code><code>config_file path_to_assignment_config.txt \</code></p>
<p><code>--</code><code>output_file path_to_output_file</code></p>
</div>
<p>The script will generate a table in a text file specified by the user (default: assignment_result.txt). This table will have a row for each selected sample and one column for each signature, called <i>“</i>Sn_exposure<i>”</i>. This column reports the alpha values of each signature found in each sample (i.e., a numeric value measuring the level of activity of the signature in that sample).</p>
<p>If the bootstrap procedure is performed, another column for each signature is added into file. This column, called ‘Sn_pvalue’, shows the p-value of significance of observing each signature in a given sample obtained with the harmonic mean of the one-sided (greater) Mann–Whitney U-test.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">denovo.R</h3>
<p>This step provides as output:</p>
<p>The signature matrix (<b>B</b>) for each considered number of signatures and the corresponding graphical visualization (see <a href="#fig3">Figure 3</a>). These files are stored into the “signatures” directory.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1132-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Output of the denovo.R script</div>
<p>(A) The table reports the frequency of the nucleotide substitutions for each context.</p>
<p>(B) The same values are represented with bar-plots. The figure is saved as 3_signatures.pdf file located in “VirMutSig/example/denovo_results/signature/figures/”.</p>
</figcaption>
</figure>
<p>The metrics to estimate the optimal number of signatures in a text file (<i>“</i>metric_coefficients.txt<i>”)</i> and the corresponding plots (see <a href="#fig2">Figure 2</a> and <a href="#tbl2">Table 2</a>). These files are stored into the “rank” directory.</p>
<p>Among them the user should select the optimal solution (<i>“</i>signatures/files/x_signatures.txt<i>”</i>) and use it as input for the following step.</p>
<h3 id="sec4.2">assignment.R</h3>
<p>This step returns a matrix of the activity of each signature assigned to each sample, saved as a text file (“assignment_result.txt”). See <a href="#tbl3">Table 3</a>.</p>
<table id="tbl3">
<caption>Table 3. First 5 entries of the assignment_results.txt file obtained after assignment.R execution</caption>
<thead>
<tr>
<th>Sample</th>
<th>S1_exposure</th>
<th>S2_exposure</th>
<th>S3_exposure</th>
<th>S1_pvalue</th>
<th>S2_pvalue</th>
<th>S3_pvalue</th>
</tr>
</thead>
<tbody>
<tr>
<td>SRR12351622</td>
<td>4.63</td>
<td>0.13</td>
<td>1.92</td>
<td>1.98e-18</td>
<td>1</td>
<td>1.98e-18</td>
</tr>
<tr>
<td>SRR12351634</td>
<td>1.14</td>
<td>0.44</td>
<td>4.48</td>
<td>1.98e-18</td>
<td>1.98e-18</td>
<td>1.98e-18</td>
</tr>
<tr>
<td>SRR12351645</td>
<td>0.81</td>
<td>1.84</td>
<td>3.68</td>
<td>1.98e-18</td>
<td>1.98e-18</td>
<td>1.98e-18</td>
</tr>
<tr>
<td>SRR12351651</td>
<td>2.32</td>
<td>0.47</td>
<td>1.05</td>
<td>1.98e-18</td>
<td>1.98e-18</td>
<td>1.98e-18</td>
</tr>
<tr>
<td>SRR12351655</td>
<td>4.65</td>
<td>0.26</td>
<td>2.06</td>
<td>1.98e-18</td>
<td>1</td>
<td>1.98e-18</td>
</tr>
</tbody>
</table>
<p class="table-legend">For each sample, an exposure value is assigned for each signature. The corresponding harmonic mean p-value of the one-sided Mann–Whitney U-test is computed with a bootstrap procedure.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The produced signature-assignment matrix (“assignment_result.txt”) can be used as input in downstream analyses.</p>
<p>For example, as shown in (<a href="#bib11">Graudenzi et al., 2021</a>), the following tasks can be performed:</p>
<ol>
<li>Stratification of samples into signature-based clusters</li>
</ol>
<p>Samples can be stratified by applying k-means (or any other clustering methods) on the signature-assignment matrix. Standard heuristics should be employed to determine the optimal number of clusters.</p>
<ol start="2">
<li>Corrected-for-signatures dN/dS analysis</li>
</ol>
<p>dN/dS analysis is a standard population genetics method to assess the selection pressure acting on the virus. After the identification of viral mutational signatures, it is possible to investigate whether the SNVs generated by the corresponding mutational process are positively or negatively selected in the population. To this end, the dN/dS analysis must be corrected for the expected mutation frequency specific for each signature profiles, as proposed in (<a href="#bib11">Graudenzi et al., 2021</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<h3 id="sec6.1">Reference genome</h3>
<p>Any reference genome can be used with VirMutSig. Using different reference genomes may slightly change the results. For SARS-CoV-2 analysis we highlight the presence of two other reference genomes besides SARS-CoV-2-ANC used in this protocol (see “<a href="#sec6.1">reference genome</a>” in “<a href="#materials-and-equipment">materials and equipment</a>” section):</p>
<p>EPI_ISL_405839 (<a href="#bib4">Bastola et al., 2020</a>)</p>
<p>EPI_ISL_402125 (<a href="#bib2">Andersen et al., 2020</a>)</p>
<p>Notice that the number of mismatches between those two reference genomes and the one SARS-CoV-2-ANC is only 5 nucleotides.</p>
<h3 id="sec6.2">Dataset quality</h3>
<p>In these kinds of analyses using good quality data is mandatory because the level of random mutations (sequencing artifacts or samples biological degradation) could affect the significance of the signature identified and assigned.</p>
<p>However, we suggested parameter settings that can mitigate this issue. Depending on the specific quality of the data, the stringency of such parameters could be increased.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>It may be difficult to choose the optimal rank (i.e., the number of mutational signatures) on the basis of the available metrics (see step 13).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>The estimation of the optimal number of signatures is a critical task. We have already discussed in the text how one should consider the results of different metrics to estimate the optimal rank. However, such metrics may sometimes lead to ambiguous (or conflicting) indications, making it difficult to select the optimal rank. In this case, one may try to improve the stability of the results by increasing the number of NMF iterations, since a higher number of iterations is expected to deliver more reliable and stable results<b>.</b> To this end, please increase the value of the NMF_ITER parameter in the ‘denovo_config.yalm’ file.</p>
<p>As an alternative option, we suggest increasing the number of samples in the datasets, if possible.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>The input file for both the <i>de novo</i> and the <i>assignment</i> analyses is a matrix with samples (rows) x mutations (columns) that is automatically generated from the provided list of SNVs. If the mutations kept after the quality filter are too few, the resulting matrix might be too sparse to obtain robust results (see <a href="#sec3.3">parts 3</a> and <a href="#sec3.4">4</a>).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>The de novo extraction of mutational signatures is harder with very sparse matrices. If the input data are too sparse, one may increase the number of mutations by loosening the quality check filters or lowering the threshold to filter out fixed variants. In both cases, more mutations will be employed in the analysis.</p>
<p>However, one should be cautious when relaxing quality control filters and should aim at preserving a good tradeoff between the number of variants and the quality of the data, because including lower quality mutations may reduce the robustness of the results and the reliability of the inference.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>An issue similar to the one presented in Problem 2 may arise when the number of samples that satisfies the quality criteria (e.g., minimum number of mutations) is too low (see <a href="#sec3.3">parts 3</a> and <a href="#sec3.4">4</a>).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Increasing the number of samples is extremely beneficial for the de novo inference of mutational signatures. We provide a set of quality checks parameters to set the minimum and maximum number of mutations used to filter out samples (i.e., MIN_SNV_SAMPLE, MAX_SNV_SAMPLE). One may set these parameters to increase the sample size of the datasets, but as stated in the previous point, one should keep in mind that including lower quality samples may also reduce the quality of the inference.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>After performing the bootstrap (see <a href="#sec3.4">part 4</a>) no significant signatures were assigned to the samples.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>In the bootstrap procedure, the algorithm detects the signatures that are significantly exceeding a given exposure for each sample. The method repeats the fit until a given ‘GOODNESS_FIT_THR’ threshold is reached (to reduce overfitting) and identifies the signatures significantly exceeding ‘MIN_SIG_FREQ’ percentage of mutations for each sample. Reducing these two parameters will result in a larger number of significant signatures assigned to each sample. A correct tuning of such parameters depends on the quality of the data and on the minimum number of mutations which are considered to be significant to assess if a signature is active.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>The user may face issues with software compatibility during the setup of VirMutSig or one needs to execute it on a computer with a Windows operating system (see <a href="#sec3.1">parts 1</a>, <a href="#sec3.2">2</a>, <a href="#sec3.3">3</a>, and <a href="#sec3.4">4</a>). .</p>
<h3 id="sec7.10">Potential solution</h3>
<p>We provide a Docker image (see <a href="#sec3.1">part 1</a>) including all the VirMutSig scripts and data already preinstalled and tested. It is also possible to execute all the VirMutSig steps within the Docker image even on a computer with Windows OS.</p>
<p>To do this, please start by selecting a directory (e.g., “C:\user\data”) on the local machine and copy all the files requested for the user analysis (e.g., SRA_list or fastq files, reference genome of choice, etc.). This directory will be the only local directory available within the Docker image.</p>
<p>After obtaining the dcblab/virmutsig_img, the user will need to access it by executing the following command in a terminal or in the command prompt window.</p>
<div class="textbox">
<p><code>&gt;docker run -it --mount \</code></p>
<p><code>type=bind,source="C:</code><code>\\user\</code><code>\</code><code>data</code><code>”,target=/VirMutSig/UserData/ \</code></p>
<p><code>dcblab/virmutsig_img</code></p>
</div>
<p>Please, change "C:\user\data” with the absolute path of the selected directory.</p>
<p>The terminal will changce, and the username will be replaced by something similar to:</p>
<div class="textbox">
<p><code>root@d55f578b7306:/#</code></p>
</div>
<p>Now, it is possible to execute all the protocol steps within the new terminal, following the same instructions described above.</p>
<p>The user files are located in the /VirMutSig/UserData directory. To get the VirMutSig result files please copy them into this location, and they will also be available in “C:\user\data\” local directory.</p>
<p>All the files in VirMutSig_img are writable. For example, the config files (i.e., “nextflow.config”, “denovo_config.yalm”, and “assignment_config.yalm”) can be modified using the “nano” editor with the following command:</p>
<div class="textbox">
<p><code>root@d55f578b7306:/# nano /VirMutSig/denovo/denovo_config.yaml</code></p>
</div>
<p>Once completed the analyses, please digit exit to close the VirMutSig image. Notice that, only the files modified or created within the /VirMutSig/UserData directory will be permanently available (in C:\user\data), while all other edits will be discarded.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Alexandrov, L.B., Nik-Zainal, S., Wedge, D.C., Aparicio, S.A., Behjati, S., Biankin, A.V.,., and Stratton, M.R. (2013). Signatures of mutational processes in human cancer. Nature <i>500</i>, 415-421. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref1">View at publisher</a></p>
<p id="bib2">Andersen, K.G., Rambaut, A., Lipkin, W.I., Holmes, E.C., and Garry, R.F. (2020). The proximal origin of SARS-CoV-2. Nat. Med. <i>26</i>, 450-452. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref2">View at publisher</a></p>
<p id="bib3">Auguie, B. (2017). gridExtra: Miscellaneous Functions for ”Grid” Graph- Ics. R Package Version 2.3. <a class="external-link" href="https://CRAN.R-project.org/package=gridExtra">https://CRAN.R-project.org/package=gridExtra</a></p>
<p id="bib4">Bastola, A., Sah, R., Rodriguez-Morales, A.J., Lal, B.K., Jha, R., Ojha, H.C.,., and Pandey, B.D. (2020). The first 2019 novel coronavirus case in Nepal. Lancet Infect. Dis. <i>20</i>, 279-280. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref4">View at publisher</a></p>
<p id="bib6">Bates, D. and Maechler, M. (2021). Matrix: Sparse and Dense Matrix Classes and Methods. R Package Version 1.3-4. <a class="external-link" href="https://CRAN.R-project.org/package=Matrix">https://CRAN.R-project.org/package=Matrix</a></p>
<p id="bib5">Brunet, J.P., Tamayo, P., Golub, T.R., and Mesirov, J.P. (2004). Metagenes and molecular pattern discovery using matrix factorization. PNAS <i>101.12</i>, 4164-4169. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref5">View at publisher</a></p>
<p id="bib7">Dowle, M. and Srinivasan, A. (2021). data.table: Extension of ‘data.Frame‘. R Package Version 1.14.0. <a class="external-link" href="https://CRAN.R-project.org/package=data.table">https://CRAN.R-project.org/package=data.table</a></p>
<p id="bib14">Friedman, J., Hastie, T., and Tibshirani, R. (2010). Regularization paths for Generalized linear models via coordinate descent. J. Stat. Softw. <i>33</i>, 1-22. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref14">View at publisher</a></p>
<p id="bib9">Gagolewski, M. (2021). Stringi: Fast and Portable Character String Processing in R. R Package Version 1.7.4. <a class="external-link" href="https://stringi.gagolewski.com/">https://stringi.gagolewski.com/</a></p>
<p id="bib10">Gaujoux, R. and Seoighe, C. (2020). The Package NMF: Manual Pages. R Package Version 0.23.0. CRAN. <a class="external-link" href="https://cran.r-project.org/package=NMF">https://cran.r-project.org/package=NMF</a></p>
<p id="bib11">Graudenzi, A., Maspero, D., Angaroni, F., Piazza, R., and Ramazzotti, D. (2021). Mutational signatures and heterogeneous host response revealed via large-scale characterization of SARS-CoV-2 genomic diversity. iScience <i>24</i>, 102116. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref11">View at publisher</a></p>
<p id="bib12">Huber, W., Carey, V.J., Gentleman, R., Anders, S., Carlson, M., Carvalho, B.S.,., and Morgan, M. (2015). Orchestrating high-throughput genomic analysis with Bioconductor. Nat. Methods <i>12</i>, 115-121. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref12">View at publisher</a></p>
<p id="bib13">Hutchins, L.N., Murphy, S.M.,, Singh, P., and Graber, J.H. (2008). Position-dependent motif characterization using non-negative matrix factorization. Bioinformatics <i>24</i>, 2684-2690. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref13">View at publisher</a></p>
<p id="bib15">Kim, H. and Park, H. (2007). Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis. Bioinformatics <i>23</i>, 1495-1502. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref15">View at publisher</a></p>
<p id="bib16">Lal, A., Liu, K., Tibshirani, R., Sidow, A., and Ramazzotti, D. (2021). De novo mutational signature discovery in tumor genomes using SparseSignatures. PLoS Comput. Biol. <i>17</i>, e1009119. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref16">View at publisher</a></p>
<p id="bib17">Merkel, D. (2014). Docker: lightweight Linux containers for consistent development and deployment. Linux J. <i>2014</i>, 2. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref17">View at publisher</a></p>
<p id="bib18">Morgan, M. (2021). BiocManager: Access the Bioconductor Project Package Repository. R Package Version 1.30.16. <a class="external-link" href="https://CRAN.R-project.org/package=BiocManager">https://CRAN.R-project.org/package=BiocManager</a></p>
<p id="bib19">Mullen, K.M. and van Stokkum, I.H.M. (2012). Nnls: The Lawson- Hanson Algorithm for Non-negative Least Squares (NNLS). R Package Version 1.4. <a class="external-link" href="https://CRAN.R-project.org/package=nnls">https://CRAN.R-project.org/package=nnls</a></p>
<p id="bib20">Ramazzotti, D., Angaroni, F., Maspero, D., Gambacorti-Passerini, C., Antoniotti, M., Graudenzi, A., and Piazza, R. (2021). VERSO: a comprehensive framework for the inference of robust phylogenies and the quantification of intra-host genomic diversity of viral samples. Patterns <i>2</i>, 100212. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref20">View at publisher</a></p>
<p id="bib21">Stephens, J. (2020). Yaml: Methods to Convert R Data to YAML and Back. R Package Version 2.2.1. <a class="external-link" href="https://CRAN.R-project.org/package=yaml">https://CRAN.R-project.org/package=yaml</a></p>
<p id="bib23">Davis, T.L. (2020). Optparse: Command Line Option Parser. R Package Version 1.6.6. <a class="external-link" href="https://CRAN.R-project.org/package=optparse">https://CRAN.R-project.org/package=optparse</a></p>
<p id="bib22">Di Tommaso, P., Chatzou, M., Floden, E.W., Barja, P.P., Palumbo, E., and Notredame, C. (2017). Nextflow enables reproducible computational workflows. Nat. Biotechnol. <i>35</i>, 316-319. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00617-1/sref22">View at publisher</a></p>
<p id="bib24">Wickham, H. (2016). ggplot2: Elegant Graphics for Data Analysis (Springer-Verlag). <a class="external-link" href="https://ggplot2.tidyverse.org">https://ggplot2.tidyverse.org</a></p>
<p id="bib8">Wild, Fridolin. (2020). Lsa: Latent Semantic Analysis. R Package Version 0.73.2. <a class="external-link" href="https://CRAN.R-project.org/package=lsa">https://CRAN.R-project.org/package=lsa</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Daniele Ramazzotti (<a href="mailto:daniele.ramazzotti@unimib.it">daniele.ramazzotti@unimib.it</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique data sets. The FASTQ files used in this protocol are public available at NCBI BioProject database (<a href="https://www.ncbi.nlm.nih.gov/bioproject/">https://www.ncbi.nlm.nih.gov/bioproject/</a>) using the following access numbers NCBI: PRJNA610428, PRJNA645906</p>
<p>The VirMutSig protocol and the example files are available at: <a href="https://github.com/BIMIB-DISCo/VirMutSig">https://github.com/BIMIB-DISCo/VirMutSig</a>.</p>
<h3>Acknowledgments</h3>
<p>This work was partially supported by the Elixir Italian Chapter and the SysBioNet project, a <a href="https://doi.org/10.13039/501100003407">Ministero dell’Istruzione, dell’Università e della Ricerca</a> initiative for the Italian Roadmap of European Strategy Forum on Research Infrastructures, and by the <a href="https://doi.org/10.13039/501100005010">Associazione Italiana per la Ricerca sul Cancro</a> (AIRC)-IG grant 22082. D.R. and F.A. were partially supported by a <a href="https://doi.org/10.13039/501100002954">Bicocca 2020 Starting Grant</a>. D.R. was supported by a Premio Giovani Talenti dell'<a href="https://doi.org/10.13039/100012352">Università degli Studi di Milano-Bicocca</a>. We thank Marco Antoniotti, Giulio Caravagna, Chiara Damiani, Lucrezia Patruno, and Francesco Craighero for helpful discussions.</p>
<h3>Author contributions</h3>
<p>Writing – original draft and writing – review &amp; editing, D.M., D.R., and A.G.; software, D.M., D.R., and F.A.; validation, D.M., D.R., F.A., and A.G.; funding acquisition, A.G., D.R., D.P., R.P.; supervision, A.G. and D.R. All authors read, revised, and approved the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>