<article>
<section>
<h1 id="header">Protocol for computationally evaluating the loss of stoichiometry and coordinated expression of proteins</h1>
<p><time datetime="2022-03-18">Published: March 18, 2022</time></p>
<p>Stefan Hinz,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Michael E. Todhunter,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> and Mark A. LaBarge<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Population Sciences, Beckman Research Institute, City of Hope, 1500 E. Duarte Road, Duarte, CA 91010, USA</p>
<p id="aff2"><sup>2</sup>Center for Cancer and Aging Research, City of Hope, 1500 E. Duarte Road, Duarte, CA 91010, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:shinz@coh.org">shinz@coh.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:mlabarge@coh.org">mlabarge@coh.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101182">10.1016/j.xpro.2022.101182</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Dysregulation of the transcriptional or translational machinery can alter the stoichiometry of multiprotein complexes and occurs in natural processes such as aging. Loss of stoichiometry has been shown to alter protein complex functions. We provide a protocol and associated code that use omics data to quantify these stoichiometric changes via statistical dispersion utilizing the interquartile range of expression values per grouping variable. This descriptive statistical approach enables the quantification of stoichiometry changes without additional data acquisition.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00062-4/sref1">Hinz et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol to quantify stoichiometry changes of protein complexes</li>
<li>Robust and versatile output based on interquartile range of expression</li>
<li>Lightweight R functions easily loaded into data pipeline via GitHub</li>
<li>Conveniently plot measured stoichiometry changes with ggplot2 wrapper function</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1508-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">General considerations</h3>
<p>This protocol uses numerical expression data to evaluate the changing stoichiometry of gene or protein complexes over a condition variable (e.g., age, time, or treatment). The script assumes that all genes or proteins are present at all tested conditions. The protocol assumes that the expression of gene/protein complexes must change concordantly to maintain stoichiometry (coordinated change of expression). A progressive reduction in the correlation between protein and mRNA causes a progressive loss of stoichiometry in several protein complexes, including ribosomes (<a href="#bib2">Kelmer Sacramento et al., 2020</a>), which is observed as an uncoordinated change of expression. The figure of merit is the interquartile range (IQR) of expression. IQR describes the difference between the 75th and 25th percentile (<i>x</i><sub>75</sub> -<i>x</i><sub>25</sub>) and, if the proteins complexed are unchanged between condition variables, the IQR stays unchanged, whereas the IQR changes given coordination changes (<a href="#fig1">Figure 1</a>). These analyses have been used to identify changes in proteostasis. (<a href="#bib1">Hinz et al., 2021</a>; <a href="#bib2">Kelmer Sacramento et al., 2020</a>)</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1508-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Examples of coordinated and uncoordinated change of expression</div>
<p>The provided function requires numerical expression data with n samples for m conditions (exemplar conditions A and B) to calculate the interquartile range.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT OR RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>R (V3.6.1)</td>
<td>CRAN</td>
<td>r-project.org</td>
</tr>
<tr>
<td>RStudio (V1.2.1335)</td>
<td>CRAN</td>
<td>rstudio.com</td>
</tr>
<tr>
<td>dplyr (V1.0.5)</td>
<td>CRAN</td>
<td><a href="https://cran.r-project.org/web/packages/dplyr/index.html">https://cran.r-project.org/web/packages/dplyr/index.html</a></td>
</tr>
<tr>
<td>ggrepel (V0.8.2)</td>
<td>CRAN</td>
<td><a href="https://cran.r-project.org/web/packages/ggrepel/index.html">https://cran.r-project.org/web/packages/ggrepel/index.html</a></td>
</tr>
<tr>
<td>ggplot2 (V3.3.3)</td>
<td>CRAN</td>
<td><a href="https://cran.r-project.org/web/packages/ggplot2/index.html">https://cran.r-project.org/web/packages/ggplot2/index.html</a></td>
</tr>
<tr>
<td>ggsci (V2.9)</td>
<td>CRAN</td>
<td><a href="https://cran.r-project.org/web/packages/ggsci/index.html">https://cran.r-project.org/web/packages/ggsci/index.html</a></td>
</tr>
<tr>
<td>readr (V1.4.0)</td>
<td>CRAN</td>
<td><a href="https://cran.r-project.org/web/packages/readr/index.html">https://cran.r-project.org/web/packages/readr/index.html</a></td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Numerical expression data</td>
<td>GitHub</td>
<td><a href="https://raw.githubusercontent.com/LaBargeLab/IQR_test/main/example_gene_data.csv">https://raw.githubusercontent.com/LaBargeLab/IQR_test/main/example_gene_data.csv</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Computer</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>R functions</td>
<td>GitHub/zenodo</td>
<td>
<a href="https://github.com/LaBargeLab/IQR_test">https://github.com/LaBargeLab/IQR_test</a><a href="https://doi.org/10.5281/zenodo.5879559">https://doi.org/10.5281/zenodo.5879559</a>
</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Source IQR functions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<p>To run this protocol, a sourcing of provided convenience R functions from GitHub is required to calculate and visualize the IQR analyses. The functions, example data, and a tutorial are available at <a href="https://github.com/LaBargeLab/IQR_test">https://github.com/LaBargeLab/IQR_test</a>.</p>
<ol>
<li>Download IQR functions in R.</li>
</ol>
<div class="textbox">
<p><code>if(!require(devtools)){ install.packages("devtools") # If not already installed }</code></p>
<p><code>source_url("https://raw.githubusercontent.com/LaBargeLab/IQR_test/main/clean_functions.R")</code></p>
</div>
<h3 id="sec2.2">Format input data</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–30 min</div>
<p>The functions require input data with gene name, sample name, expression value, and grouping variable columns. Any parametric units are valid for expression values, such as counts-per-million, reads-per-kilobase transcript for RNA-seq, or protein abundance data. An example can be downloaded through the provided GitHub page.</p>
<ol start="2">
<li>Import expression matrix.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Data must be in long format - i.e., every combination of gene name and sample name must have its own row.</div>
<div class="textbox">
<p><code>urlfile &lt;- “</code><a href="https://raw.githubusercontent.com/LaBargeLab/IQR_test/main/example_gene_data.csv">https://raw.githubusercontent.com/LaBargeLab/IQR_test/main/example_gene_data.csv</a><code>”</code></p>
<p><code>data &lt;- read.csv(urlfile)</code></p>
<p><code>data</code></p>
<p><code># Symbol name value variable</code></p>
<p><code>## &lt;chr&gt; &lt;chr&gt; &lt;dbl&gt; &lt;chr&gt;</code></p>
<p><code># 1 gene1 sample1 99 A</code></p>
<p><code># 2 gene1 sample2 131 A</code></p>
<p><code># 3 gene1 sample3 74 A</code></p>
<p><code># 4 gene1 sample4 145 A</code></p>
<p><code>#</code></p>
<p><code>#</code></p>
<p><code>#...</code></p>
<p><code>#10000 gene1000 sample10 0 B</code></p>
<p>.</p>
</div>
<h3 id="sec2.3">Run analyses</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 min</div>
<ol start="3">
<li>Execute the stoichiometry function with the long format input data from step 2 to calculate the IQR for every sample by conditions and return a data frame with the results.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The stoichiometry function input requires the following arguments:</div>
<p><b>symbol</b> - character vector of gene symbols</p>
<p><b>expression</b> - numeric vector of expression values</p>
<p><b>variable</b> - character or factor vector with condition information</p>
<p><b>sample</b> - character vector of sample IDs</p>
<p><b>geneset</b> - character vector of interested genes (same nomenclature as symbol); if no geneset is supplied IQR analyses are performed based on all provided symbols</p>
<div class="textbox">
<p><code>stoi &lt;- stoichiometry(</code> <code>expression = data$value,</code></p>
<p>                          <code>symbol = data$Symbol,</code></p>
<p>                          <code>variable = data$variable,</code></p>
<p>                          <code>geneset = c("gene1", "gene2", "gene3", "gene4", "gene5"),</code></p>
<p>                          <code>sample = data$name)</code></p>
</div>
<ol start="4">
<li>Use the output from the stoichiometry function as input for the plotting function for visualization.</li>
</ol>
<div class="textbox">
<p><code>stoi_plot(stoi)</code></p>
</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The provided functions output a data frame with the IQR values on a sample level, and these can be conveniently plotted using boxplots (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1508-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Results of example IQR analyses</div>
<p>Boxes of box whisker plot represent 25–75 percentile ranges, vertical lines represent 1.5 × inter quartile range, and horizontal bars represent medians.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Statistical significance can be calculated using an appropriate statistical test, such as the Welch two-sample t-test for comparing two conditions or ANOVA for more than two conditions. The tests utilize the per-sample IQR data as input. Therefore, a power estimation is recommended to assess minimum sample size for meaningful analyses.</p>
<div class="textbox">
<p><code>t.test(stoi$IQR ∼ stoi$variable)</code></p>
<p><code>#  Welch Two Sample t-test</code></p>
<p><code>#  data  : stoi$IQR by stoi$variable</code></p>
<p><code>#  alternative hypothesis: true difference in means is not equal to 0</code></p>
<p><code>#  -216.46803 -65.53197</code></p>
<p><code>#  sample estimates:</code></p>
<p><code>#  mean in group A mean in group B</code></p>
<p><code>#          36.6       177.6 95 percent confidence interval:</code></p>
<p><code>#    -15.48451 22.68451</code></p>
<p><code>#  sample estimates:</code></p>
<p><code>#  mean in group A mean in group B</code></p>
<p><code>#          36.6        33.0</code></p>
</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The method described here is a statistical approach to assess the deregulation of protein complexes and does not replace confirmatory experiments.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The provided function does not load or work (step 3).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Confirm that all dependencies are installed (see <a href="#key-resources-table">key resources table</a>). In case issues persist, an issue can be opened through the GitHub page.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The expression data includes NA values (step 1).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Remove data with NA values or impute expression data if appropriate.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Expression data is in wide format not the required long format (step 2).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>There are multiple tools to reshape data in R. The authors suggest the use of pivot_longer() function from the tidyR package.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Where to find curated genesets (step 3)?</p>
<h3 id="sec6.8">Potential solution</h3>
<p>There are multiple databases of curated genesets. The authors of this protocol recommend Molecular Signatures Database (MSigDB), Kyoto Encyclopedia of Genes and Genomes (KEGG), Drug Signatures Database (DSigDB), Gene Ontology Resource (GO), or HUGO Gene Nomenclature Committee as starting points.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>The geneset does not match any symbols provided in the dataset (step 3).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Confirm that the symbol nomenclature matches between genset and dataset symbol. In case of differing format, consider utilizing symbol conversion tools (e.g.,: biomaRt).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Hinz, S., Manousopoulou, A., Miyano, M., Sayaman, R.W., Aguilera, K.Y., Todhunter, M.E., Lopez, J.C., Sohn, L.L., Wang, L.D., and LaBarge, M.A. (2021). Deep proteome profiling of human mammary epithelia at lineage and age resolution. iScience <i>24</i>, 103026. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00062-4/sref1">View at publisher</a></p>
<p id="bib2">Kelmer Sacramento, E., Kirkpatrick, J.M., Mazzetto, M., Baumgart, M., Bartolome, A., Di Sanzo, S., Caterino, C., Sanguanini, M., Papaevgeniou, N., Lefaki, M., et al. (2020). Reduced proteasome activity in the aging brain results in ribosome stoichiometry loss and aggregation. Mol. Syst. Biol. <i>16</i>, e9596. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00062-4/sref2">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Mark A. LaBarge, <a href="mailto:mlabarge@coh.org">mlabarge@coh.org</a></p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>Code and data is available through the GitHub repository: <a href="https://github.com/LaBargeLab/IQR_test">https://github.com/LaBargeLab/IQR_test</a>.</p>
<p>This repository has been archived at Zenodo: <a href="https://doi.org/10.5281/zenodo.5879559">https://doi.org/10.5281/zenodo.5879559</a>.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by awards from the Department of Defense/Army Breast Cancer Era of Hope Scholar Award (BC141351), City of Hope Center for Cancer and Aging to M.A.L.; National Institutes of Health/National Cancer Institute (NIH/NCI) grants R01CA237602, U01CA244109, R33AG059206, and R01EB024989 to M.A.L.; American Cancer Society – Fred Ross Desert Spirit Postdoctoral Fellowship (PF-21-184-01-CSM) to S.H. and American Cancer Society Postdoctoral Fellowship (131311-PF-18-188-01-TBG) to M.E.T. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. Graphical abstract and <a href="#fig1">Figure 1</a> created in part with BioRender.com.</p>
<h3>Author contributions</h3>
<p>S.H. and M.A.L. conceived the protocol; S.H. and M.E.T. wrote code; and S.H. wrote the manuscript. All authors discussed and commented on the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>