<article>
<section>
<h1 id="header">A laser-mediated photo-manipulative toolbox for generation and real-time monitoring of DNA lesions</h1>
<p><time datetime="2021-08-06">Published: August 6, 2021</time></p>
<p>Bindhu K. Madhavan,<sup><a href="#aff1">1</a></sup> Zhe Han,<sup><a href="#aff1">1</a></sup> Albert Sickmann,<sup><a href="#aff2">2</a></sup> Rainer Pepperkok,<sup><a href="#aff3">3</a></sup> Peter P. Nawroth,<sup><a href="#aff1">1</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a></sup> and Varun Kumar<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn1">6</a>,<a href="#fn2">7</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Internal Medicine-I and Clinical Chemistry, University Hospital of Heidelberg, Im Neuenheimer Feld 410, Heidelberg 69120, Germany</p>
<p id="aff2"><sup>2</sup>Leibniz Institute for Analytical Sciences, Dortmund 44227, Germany</p>
<p id="aff3"><sup>3</sup>European Molecular Biology Laboratory, Advanced Light Microscopy Facility, Heidelberg 69117, Germany</p>
<p id="aff4"><sup>4</sup>German Center for Diabetes Research (DZD), Neuherberg 85764, Germany</p>
<p id="aff5"><sup>5</sup>Joint Heidelberg-IDC Translational Diabetes Program, Helmholtz-Zentrum, Heidelberg 69120, Germany</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:varun.kumar@med.uni-heidelberg.de">varun.kumar@med.uni-heidelberg.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100700">10.1016/j.xpro.2021.100700</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>With the advancement of laser-based microscopy tools, it is now possible to explore mechano-kinetic processes occurring inside the cell. Here, we describe the advanced protocol for studying the DNA repair kinetics in real time using the laser to induce the DNA damage. This protocol can be used for inducing, testing, and studying the repair mechanisms associated with DNA double-strand breaks, interstrand cross-link repair, and single-strand break repair.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib12">Kumar et al. (2017</a>, <a href="#bib11">2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Laser micro-irradiation can be used to induce and study DNA repair mechanisms</li>
<li>This protocol can be used as a tool to study the kinetics of DNA repair</li>
<li>Repair factor mutation(s) affects the recruitment kinetics and DNA repair process</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Principle</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>This protocol is based on two principles; first, it utilizes the inherent property of the DNA repair factors to respond to DNA damage signaling rapidly, recruits to the site of damage, become transiently immobilized and assembles into an active repair complex to timely repair the damage. From the damage, this protocol is based on the original observation that cellular DNA, when pre-sensitized with halogenated thymidine analogs or DNA intercalating agents becomes hypersensitive to UV light. Therefore, these two together can be used for studying the DNA repair process. For more details of the DNA repair process please refer to the original work (<a href="#bib21">Zhou and Elledge, 2000</a>, <a href="#bib9">Jackson, 2002</a>, <a href="#bib4">Caldecott, 2007</a>, <a href="#bib6">Deans and West, 2011</a>).</p>
<p>In this preparatory phase, plan the pre-experiment set up as per the steps mentioned here.</p>
<ol>
<li>Set aside a dish of healthy, well-growing study cells to be used in DNA repair studies. Usually, 0.5–1 × 10<sup>6</sup> cells are sufficient for most multi-factor studies.</li>
<li>Prepare the required reagents, medium and dishes, and ensure they are sufficient for the experiments planned.</li>
<li>Prepare phenol red-free DMEM medium supplemented with FCS (10%), penicillin-streptomycin (1%), L-glutamine (1%). Here one is free to use any other media best suited to the study cells. However, to avoid the auto-fluorescence of the medium, a phenol red-free medium is better.</li>
<li>Prepare the study or control plasmids with an endotoxin-free plasmid kit (refer to <a href="#materials-and-equipment">materials and equipment</a> and <a href="#key-resources-table">key resources table</a>)</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti γH2AX</td>
<td>Cell Signaling</td>
<td>Cat# 9718 (dilution 1:500)</td>
</tr>
<tr>
<td>Anti-Rabbit Alexa-555</td>
<td>Abcam</td>
<td>Cat# 150062 (dilution 1:2000)</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Calbiochem</td>
<td>Cat# 648466</td>
</tr>
<tr>
<td>DAPI</td>
<td>Sigma</td>
<td>Cat# MBD0015</td>
</tr>
<tr>
<td>DMEM with phenol red</td>
<td>Gibco</td>
<td>Cat# 31885-023</td>
</tr>
<tr>
<td>Fetal calf serum (FCS)</td>
<td>Gibco</td>
<td>Cat# A4766801</td>
</tr>
<tr>
<td>DMEM without phenol red</td>
<td>Sigma</td>
<td>Cat# D4947</td>
</tr>
<tr>
<td>Paraformaldehyde</td>
<td>Boster Biological</td>
<td>Cat# AR1068</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Sigma</td>
<td>Cat# P0781</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Sigma</td>
<td>Cat# G7513</td>
</tr>
<tr>
<td>PBS</td>
<td>Sigma</td>
<td>Cat# D8537</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Sigma</td>
<td>Cat# A7030</td>
</tr>
<tr>
<td>355 nM Laser</td>
<td>JDS Uniphase</td>
<td>#PNV-001525-0x0</td>
</tr>
<tr>
<td>Turbofect transfection reagent</td>
<td>Thermo</td>
<td>#R0533</td>
</tr>
<tr>
<td>BrdU (<a href="#bib14">Lukas et al. 2005</a>)</td>
<td>Sigma</td>
<td>Cat# B9285</td>
</tr>
<tr>
<td>Hochest 33342</td>
<td>Sigma</td>
<td>Cat# B2261</td>
</tr>
<tr>
<td>TMP (trimethyl-psoralen) (<a href="#bib13">Lachaud et al. 2014</a>)</td>
<td>Cayman</td>
<td>Cat# 17162</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>U2OS</td>
<td>ATCC</td>
<td>ATCC® HTB-96</td>
</tr>
<tr>
<td>A549</td>
<td>Sigma</td>
<td># Cat 86012804-1VL</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ</td>
<td>Fiji</td>
<td><a href="https://fiji.sc">https://fiji.sc</a></td>
</tr>
<tr>
<td colspan="3">Others</td>
</tr>
<tr>
<td>15 μ Slide 8-well</td>
<td>ibidi</td>
<td># Cat 80826</td>
</tr>
<tr>
<td>15 μ Slide 8-well Grid</td>
<td>ibidi</td>
<td># Cat 80826 – G500</td>
</tr>
<tr>
<td>15 μ Slide 8-well poly-L-lysine</td>
<td>ibidi</td>
<td># Cat 80824</td>
</tr>
<tr>
<td>CELLview culture dish 35/10 mm</td>
<td>Greiner</td>
<td># Cat 627870</td>
</tr>
<tr>
<td>Cover glass ⍉ 8 mm # 1</td>
<td>Thermo</td>
<td># Cat 12052712</td>
</tr>
<tr>
<td>GenElute HP Endotoxin-Free Plasmid Maxiprep Kit</td>
<td>Sigma</td>
<td># NA0410-1KT</td>
</tr>
<tr>
<td>Repair factor plasmids</td>
<td>Addgene</td>
<td><a href="https://www.addgene.org/">https://www.addgene.org/</a></td>
</tr>
<tr>
<td>Repair factor plasmids</td>
<td>MRC Laboratory</td>
<td><a href="https://mrcppureagents.dundee.ac.uk/reagents-cdna-clones">https://mrcppureagents.dundee.ac.uk/reagents-cdna-clones</a></td>
</tr>
<tr>
<td>FV1200 (Microscope access at EMBL Heidelberg)</td>
<td>EMBL/Olympus</td>
<td><a href="https://www.eurobioimaging.eu/nodes/advanced-light-microscopy-facility-embl">https://www.eurobioimaging.eu/nodes/advanced-light-microscopy-facility-embl</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Materials</h3>
<p>The materials that are required to plan and execute the laser induced DNA damage are described in the <a href="#key-resources-table">key resources table</a>. Moreover, depending upon the DNA damage pathway or a particular step of the repair process experimenter is planning to study, the positive control can be chosen from the plasmid table (<a href="#tbl1">Table 1</a>).</p>
<table id="tbl1">
<caption>Table 1. Plasmid table</caption>
<thead>
<tr>
<th>S.No.</th>
<th>Repair factor</th>
<th>Repair pathway</th>
<th>Step involved</th>
<th>Our lab</th>
<th>Addgene</th>
<th>MRC</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>hNBS1-EGFP</td>
<td>DSB (HR)</td>
<td>Detection/Signaling</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>2</td>
<td>hNBS1-mCherry</td>
<td>DSB (HR)</td>
<td>Detection/Signaling</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>3</td>
<td>hMRE11-EGFP</td>
<td>DSB (HR)</td>
<td>Detection/Signaling</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>4</td>
<td>hMRE11-mCherry</td>
<td>DSB (HR)</td>
<td>Detection/Signaling</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>5</td>
<td>hRAD50-EGFP</td>
<td>DSB (HR)</td>
<td>Detection/Signaling</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>6</td>
<td>hBLM-EGFP</td>
<td>DSB (HR)</td>
<td>Long range resection</td>
<td>+</td>
<td>+ (#80070)</td>
<td>n/a</td>
</tr>
<tr>
<td>7</td>
<td>hBLM-mCherry</td>
<td>DSB (HR)</td>
<td>Long range resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>8</td>
<td>hEXO1-EGFP</td>
<td>DSB (HR)</td>
<td>Long range resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>9</td>
<td>hDna2-EGFP</td>
<td>DSB (HR)</td>
<td>Long range resection</td>
<td>-</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>10</td>
<td>hRPA2-EGFP</td>
<td>DSB (HR)/NER</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>+</td>
</tr>
<tr>
<td>11</td>
<td>hRPA2-mCherry</td>
<td>DSB (HR)/NER</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>12</td>
<td>hRPA2-EBFP</td>
<td>DSB (HR)/NER</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>13</td>
<td>mRPA2-TagBF2</td>
<td>DSB (HR)/NER</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>14</td>
<td>mRPA2-mVenus</td>
<td>DSB (HR)/NER</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>15</td>
<td>mRPA2-AsRed2</td>
<td>DSB (HR)/NER</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>16</td>
<td>hSOSS1-EGFP</td>
<td>DSB (HR)</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>17</td>
<td>hCTIP-EGFP</td>
<td>DSB (HR)/ICL</td>
<td>End-processing</td>
<td>+</td>
<td>+ (#71109)</td>
<td>+</td>
</tr>
<tr>
<td>18</td>
<td>hPARP1-EGFP</td>
<td>DSB (HR)/SS-DNA repair</td>
<td>Detection/ processing</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>19</td>
<td>hPARP1-mCherry</td>
<td>DSB (HR)/SS-DNA repair</td>
<td>Detection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>20</td>
<td>hXRCC1-GFP</td>
<td>SS break repair</td>
<td>Detection/ processing</td>
<td>-</td>
<td>n/a</td>
<td>+</td>
</tr>
<tr>
<td>21</td>
<td>hXRCC1-mCherry</td>
<td>SS break repair</td>
<td>Detection/ processing</td>
<td>-</td>
<td>n/a</td>
<td>+</td>
</tr>
<tr>
<td>22</td>
<td>mH2A1.1-EYFP</td>
<td>DSB (HR)</td>
<td>Detection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>23</td>
<td>mH2A1.1-mCherry</td>
<td>DSB (HR)</td>
<td>Detection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>24</td>
<td>53BP1-EGFP</td>
<td>DSB (HR)/ICL</td>
<td>Detection and Processing</td>
<td>+</td>
<td>+ (#110301)</td>
<td>n/a</td>
</tr>
<tr>
<td>25</td>
<td>hXRCC4-EGFP</td>
<td>DSB (NHEJ)</td>
<td>Processing</td>
<td>+</td>
<td>+ (#46959)</td>
<td>n/a</td>
</tr>
<tr>
<td>26</td>
<td>hGcn5-EGFP</td>
<td>DSB (HR and NHEJ)</td>
<td>Signaling</td>
<td>+</td>
<td>+ (#65386)</td>
<td>n/a</td>
</tr>
<tr>
<td>27</td>
<td>hKu70-EGFP</td>
<td>DSB (NHEJ)</td>
<td>End protection</td>
<td>+</td>
<td>+ (#46957)</td>
<td>n/a</td>
</tr>
<tr>
<td>28</td>
<td>hKu80-EGFP</td>
<td>DSB (NHEJ)</td>
<td>End protection</td>
<td>+</td>
<td>+ (#46958)</td>
<td>n/a</td>
</tr>
<tr>
<td>29</td>
<td>hLig4-EGFP</td>
<td>DSB (NHEJ)</td>
<td>Ligation</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>30</td>
<td>hDDX17-EGFP</td>
<td>DSB (HR)</td>
<td>Processing</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>31</td>
<td>hDDX17-mCherry</td>
<td>DSB (HR)</td>
<td>Processing</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>32</td>
<td>hRAD51-EYFP</td>
<td>DSB (HR)</td>
<td>Strand invasion</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>33</td>
<td>hRAD51-TagBF2</td>
<td>DSB (HR)</td>
<td>Strand invasion</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>34</td>
<td>hRAGE-EGFP</td>
<td>DSB (HR)</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>35</td>
<td>hRAGE-mCherry</td>
<td>DSB (HR)</td>
<td>Resection</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>36</td>
<td>hFANCD2-EGFP</td>
<td>ICL</td>
<td>Signaling and Repair</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>37</td>
<td>hSLX4-EGFP</td>
<td>ICL</td>
<td>Repair adapter</td>
<td>+</td>
<td>n/a</td>
<td>+</td>
</tr>
<tr>
<td>38</td>
<td>hBRCA1-EGFP</td>
<td>DSB (HR)</td>
<td>Signaling and Strand Invasion</td>
<td>+</td>
<td>+ (#71116)</td>
<td>n/a</td>
</tr>
<tr>
<td>39</td>
<td>hNPM1-EGFP</td>
<td>DSB (HR)</td>
<td>Signaling</td>
<td>+</td>
<td>+ (#17578)</td>
<td>n/a</td>
</tr>
<tr>
<td>40</td>
<td>hKIF3A-EGFP</td>
<td>DSB (HR/NHEJ)</td>
<td>Mobility</td>
<td>+</td>
<td> </td>
<td>n/a</td>
</tr>
<tr>
<td>41</td>
<td>hLamin A-EGFP</td>
<td>DSB (HR)</td>
<td>Activator/ resection</td>
<td>+</td>
<td>+ (#134867)</td>
<td>n/a</td>
</tr>
<tr>
<td>42</td>
<td>rKIF5C-EGFP</td>
<td>DSB (HR/NHEJ)</td>
<td>Activator</td>
<td>+</td>
<td>+ (#71853)</td>
<td>n/a</td>
</tr>
<tr>
<td>43</td>
<td>hSIRT1-EGFP</td>
<td>DSB (HR and NHEJ)</td>
<td>Epigenetic and regulatory roles</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>44</td>
<td>hSIRT6-EGFP</td>
<td>DSB (HR and NHEJ)</td>
<td>Epigenetic and regulatory roles</td>
<td>+</td>
<td>n/a</td>
<td>+</td>
</tr>
<tr>
<td>45</td>
<td>hSIRT7-EGFP</td>
<td>DSB (HR and NHEJ)</td>
<td>Epigenetic regulations</td>
<td>+</td>
<td>n/a</td>
<td>n/a</td>
</tr>
</tbody>
</table>
<p class="table-legend">Here HR means homologous recombination repair, NHEJ represents Non-homologous end-joining repair, NER: Nucleotide excision repair (<a href="#bib1">Anindya, 2020</a>). Resection (<a href="#bib18">Polo et al., 2012</a>), strand invasion (<a href="#bib2">Bhattacharya et al., 2017</a>) are general terminologies commonly used in DNA repair.</p>
<h3 id="sec2.2">Buffer</h3>
<p><b>CSK Buffer:</b> For the pre-extraction CSK buffer can be prepared by adding the following components in ddH<sub>2</sub>O. These are 10 mM PIPES (pH=7.5), 3 mM MgCl<sub>2</sub>, 100 mM NaCl, 300 mM Sucrose and Triton X-100 (0.01%–0.5% depends on the cell types). Prepare fresh, store at ± 25°C.</p>
<h3 id="sec2.3">Equipment</h3>
<p><b>Features required in the microscope:</b> For the induction and timely tracking of the DNA damage, the microscope needs to be equipped with the following or similar setup:</p>
<p>For laser-induced DNA damage, you need a microscope equipped with two laser scanning units, as shown in figure (<a href="#fig1">Figure 1</a>). The first laser scanner will be utilized in imaging the cells (like cells expressing fluorescence tagged DNA repair factors). This scanner is present in all the laser-equipped microscopes (405, 488, 560 nm), whereas the second scanner plays a role in photo-stimulation only (DNA damage). This scanner could be equipped with various lasers, but the prominent lasers used for inducing DNA damage are 405 or 355 nm lasers.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Laser-microirradiation microscope setup: Schematic view of confocal microscope with imaging (Scanner-1) and Photo-manipulative (Scanner-2) scanners</div>
</figcaption>
</figure>
<p>Most of the latest confocal systems are equipped with imaging and photo-stimulation scanners in two different configurations: in the first configuration, one dedicated scanner is assigned for each process (i.e., for imaging or photo-stimulation), whereas in another configuration, the scanner can perform simultaneous photo-stimulation and imaging. Moreover, any old instrument can also be modified to these configurations. More details of these modifications can be obtained from either the microscope manufacturer or Rapp OptoElectronic GmbH (<a href="#tbl2">Table 2</a>).</p>
<table id="tbl2">
<caption>Table 2. Alternative laser set up already in use or potential modifications to the existing microscopes</caption>
<thead>
<tr>
<th> </th>
<th><i>Alternative systems</i></th>
<th> </th>
<th> </th>
<th> </th>
</tr>
<tr>
<th>S.No.</th>
<th>Microscope system used</th>
<th>Vendor</th>
<th>Laser type used</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>1</b></td>
<td>Zeiss AxioObserver Z1 spinning-disk equipped with AxioCam HRm CCD camera</td>
<td>Zeiss</td>
<td>DPSL-355/14, Rapp OptoElectronics</td>
<td>(<a href="#bib8">Golia et al., 2017</a>)</td>
</tr>
<tr>
<td><b>2</b></td>
<td>UltraVIEW VoX spinning disc equipped with a FRAP unit</td>
<td>Perkin Elmer</td>
<td>405, 488, 561 or 633 nm</td>
<td>(<a href="#bib17">Muster et al., 2017</a>)</td>
</tr>
<tr>
<td><b>3</b></td>
<td>Ti-E inverted microscope equipped with a CSU-X1 spinning-disk head from Yokogawa</td>
<td>Nikon</td>
<td>405</td>
<td>(<a href="#bib20">Smith et al., 2019</a>)</td>
</tr>
<tr>
<td> </td>
<td><b><i>To modify the existing systems</i></b></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td> </td>
<td><b>Vendor</b></td>
<td><b>Link</b></td>
<td><b>Laser setup available</b></td>
<td><b>Reference</b></td>
</tr>
<tr>
<td><b>1</b></td>
<td>Rapp OptoElectronic GmbH</td>
<td><a href="https://rapp-opto.com/publications/#dna_damage">https://rapp-opto.com/publications/#dna_damage</a></td>
<td>DPSL-266, DPSS-355/14, DPSL-355, DL-405</td>
<td>Various (available in the link)</td>
</tr>
<tr>
<td> </td>
<td><b><i>Objective lenses used in this study</i></b></td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td> </td>
<td><b>Magnification</b></td>
<td><b>Immersion type</b></td>
<td><b>Catalog</b></td>
<td><b>Vendor</b></td>
</tr>
<tr>
<td><b>1</b></td>
<td>UPlanSApo 60×1.20</td>
<td>Water</td>
<td>#N1480800</td>
<td>Olympus</td>
</tr>
<tr>
<td><b>2</b></td>
<td>UPlanSApo 60×/1.35</td>
<td>Oil</td>
<td>#N1480700</td>
<td>Olympus</td>
</tr>
</tbody>
</table>
<p>Hence, in general, any microscope equipped with these described scanners along with the lasers of suitable wavelengths which can be selectively positioned in regions of interest (ROI), in principle can be used for DNA damage experiments. The most common available setup for such an experiment is a confocal laser-scanning microscope (Confocal LSM) with continuous wavelength laser (355 nm or 405 nm). If this setup has a single scanner only (as in most confocal LSMs), a ‘FRAP wizard’ (Fluorescence Recovery After Photo-bleaching wizard) can be used to induce the DNA damage. In this case, the scanner is used for imaging the repair factors and the drawn ROI to induce DNA damage <i>by</i> using the AOTF control of the selected damaging laser (355 nm, or 405 nm).</p>
<p>Another important feature required for live microscopy is the environmental incubation chamber surrounding the microscope stage. This chamber system must be equipped with respective controls to regulate temperature, CO<sub>2</sub> concentration, and the chamber’s relative humidity during the experiment.</p>
<p><b>Equipment used in this protocol:</b> Here in this protocol, we used the Olympus FV1200 microscope equipped with two scanners along with a synchronizing (SIM) unit and a variety of lasers of various wavelengths (405, 488, 559, and 635 nm in imaging scanner) for imaging and a pulsed 355 nm laser (JDS Uniphase PNV-001525-0 × 0; max pulse energy 30 μJ, pulse width &lt;500 ps), in photo-stimulation scanner, for DNA damage. This microscope was equipped with EMBL environmental chamber with all the respective controls mentioned above.</p>
<h3 id="sec2.4">Image and DNA damage settings</h3>
<p>The successful induction and tracking of DNA damage involve two different settings. The first set consists of the imaging part, and the second set consists of the laser microirradiation part.</p>
<p><b>Imaging setup:</b> This part involves a simple microscope setting commonly used during any microscopy imaging. To begin with, adjust the standard image settings such as laser power and high voltage (current-voltage towards photomultiplier tubes; PMT), scan speed, zoom (zoom factor 3-3.2×), and other general imaging parameters. This step aims to clearly locate and visualize the cellular morphology and nuclear periphery for specific targeting of laser irradiation. We recommend using water- or oil-based 40 ×-60 × objectives for better visualization. This part utilizes the imaging scanner of the microscope and prominently stays the same across various microscopes.</p>
<p>In our instrument (FV1200), we used either UPlanSApo 60 × 1.20 (water-based immersion, working distance ∼0.28 mm) or UPlanSApo 60×/1.35 (oil-based immersion, working distance ∼0.15 mm) objective (<a href="#tbl2">Table 2</a>) and high sensitivity Gallium-Arsenide Phosphide (GaAsP) photomultiplier tubes (PMTs) as detectors to obtain high signal/noise ratio at exceptionally low excitation light.</p>
<div class="note">
<span class="note-title">Note:</span> In all the confocal systems, it is possible to image the cells in Z-stacks. Still, considering the fast recruitment kinetics of DNA repair factors (∼ms) and the time required to image/reflect all the Z-stacks, it is advisable to stick to only one plane. We also used single plane imaging during all the data presented in this protocol. This helps the experimenter track even the minor changes occurring at the site of damage in a small interval of time (like 1–2 s). Moreover, if required, one can opt for Z-stack-based imaging of the cell, provided the kinetic steps one is studying must fall in the time gap required for imaging these stacks.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To prevent the non-specific excitation of other cellular biomolecules, it is advised to keep the fluorescence intensity of each channel low to medium. Similarly, to avoid respective fluorochromes bleeding into each other, we suggest using sequential scanning mode.</div>
<h3 id="sec2.5">Laser irradiation setup</h3>
<p>This setup will independently utilize the second photo-manipulation scanner of the microscope to micro-irradiate the cell nucleus. From the FRAP wizard of the microscope program, chose any shape to position the laser spot in the nucleus with a scan speed of 200 μs/pixel at 40%–50% laser power (<a href="#fig2">Figure 2</a>). However, for uniform applicability, we recommend using a straight line (8–12 μm in length) across the length of the nucleus, since this geometry usually does not occur in nuclear structures or endogenous DNA damage pattern in nature, rather straightforward to analyze.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Laser-microirradiation and region of interest: Schematic and still views of various options to choose for the region of interest (ROI) shape in laser-induced DNA damage settings</div>
<p>To prevent unnecessary cellular stress and to obtain a clear damage zone, a straight line marked in the green dotted line box is the most preferred ROI shape (Scale 10 μm).</p>
</figcaption>
</figure>
<p>Now choose the photo-manipulative laser you are planning to use for DNA damage induction (355 nm, or 405 nm), and set the time of exposure in milliseconds (For 355 nm laser, time of exposure to create SSBs (in total 150 ms), or DSBs and ICLs (in total 250 ms) DNA damage. This will lead to about 6-12 (for DSBs and ICL) or 5–8 (for SSBs) line/frame (if using other shapes) scans, respectively, to induce the DNA damage. Usually, with a 60× objective, a straight line creates damage of width 0.2–0.5 μm across the length of the line.</p>
<p>For most of the DNA repair factors, 150 ms (for SSBs) 250 ms (for DSBs or ICLs) total time of exposure, which in total yields about unidirectional 6–12 line/frame scans, are sufficient. In contrast, several factors such as hnRNPU A1 or RBMX also participate in RNA processing steps, may require 300–500 ms time of exposure, which yields 20 line/frame scans (line, because we use line-shaped damage, term Frame is used for indicating any other shape). Therefore, if you are testing a new factor, then one should try to standardize the recruitment of test factor, along with a positive control (<a href="#tbl1">Table 1</a>), by testing different time of exposure from low to high (150–200 ms in total for SSBs and 250–500 ms in total for DSBs/ICLs). The minimal number of line/frame scans needed for clear recruitment can be used for future experiments along with the respective experimental controls. Furthermore, for all the comparative studies, it is essential to keep all the parameters constant for DNA damage induction and imaging between the different experiments. This includes laser power (imaging and induction), pixel size and number, zoom (zoom factor 3–3.2×), scan speed, shape, and induction time of the DNA damaging event.</p>
<p>Moreover, in the data presented in this protocol, we used the UV laser (355 nm laser at 46% power) of the second photo-manipulative scanner <i>via</i> SIM/FRAP wizard to scan an 8–10 μm long line across the nucleus with a scan speed of 200 μs/pixel for about 150 ms or 250 ms for SS-breaks or DSBs/ICL damage, respectively. In our hand, we have tested all the positive factors suggested in the table (<a href="#tbl1">Table 1</a>) in U2OS, Hela, and A549 cells, with 8-10 line/frame scans that are more than sufficient to induce the indicated DNA damages and for the recruitment of test factors without any detectable non-specific cytological effects. The snap view of our SIM window is shown in figure (<a href="#fig3">Figure 3</a>)</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. SIM setting window of FV software: The representative view of SIM setting window of Olympus FV software showing the indicated parameters for laser-induced studies</div>
<p>This is similar to any other FRAP/FRET window of other microscopy software.</p>
</figcaption>
</figure>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> In our studies, we used a pulsed UV-A laser (355 nm; see above for details). However, many other laser types ranging from 266 nm to NIR-lasers were successfully used for inducing DNA damage with or without pre-sensitization (<a href="#bib10">Kong, Mohanty et al., 2009</a>). For more details on laser possibilities, please refer to <a href="#tbl2">Table 2</a>. In this study, we used an Olympus FV1200 microscope; however, laser-induced DNA damage can be induced with any other microscope as long as it has sufficiently powerful laser sources and the ability to illuminate suitable ROIs efficiently. It has been attached to a SIM port and coupled to a high energy laser (355 nm or 405 nm). Low energy or weak lasers can be compensated by increasing the total exposure time of the photo-manipulative scanner (like from 250 ms to 400 ms or more). However, it takes more time in total, and if the lasers are very weak, the damage may not be uniform.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Lasers can be used for inducing various types of DNA damage. In this protocol, we describe the detailed procedure for generating the following kinds of DNA damage, which can be implemented to any cell type growing in an anchorage-dependent manner, irrespective of their size or origin:</p>
<ul>
<li>DNA <b>D</b>ouble-<b>S</b>trand <b>B</b>reaks (<b>DSBs</b>)</li>
<li>DNA <b>I</b>nterstrand <b>C</b>ross <b>L</b>inks (<b>ICLs</b>)</li>
<li>DNA <b>S</b>ingle-<b>S</b>trand <b>B</b>reaks (<b>SSBs</b>)</li>
</ul>
<p>Protocol for studying double-strand break (DSB) repair:</p>
<p>DNA DSBs are a highly cytotoxic form of DNA lesions, which, if stays unrepaired, leads to various cellular anomalies, including oncogenic transformation (<a href="#bib15">Madhavan et al., 2021</a>). Moreover, all eukaryotic cells respond to DSBs by a series of events, namely DNA damage response (DDR), to detect, signal and repair the damage by either an error-free homologous recombination (HR), or via an error-prone non-homologous end joining (NHEJ) repair pathway (<a href="#bib3">Branzei and Foiani, 2008</a>). DSBs lead to the activation of the DSB-sensing kinase (such as Ataxia Telangiectasia Mutated kinase; ATM). This activated sensor kinase then directs the recruitment of various DNA repair or auxiliary factors to the site of damage to timely repair the damage (<a href="#bib16">Matsuoka et al., 2007</a>). Thus, on-demand activation of the DNA repair pathway plays an important role in preventing the most devastating diseases of western society, including neurodegeneration, organ fibrosis and cancer.</p>
<ol>
<li>Protocol to study recruitment and repair kinetics of factors involved in DNA DSB repair by live microscopy. DAY-1 (Estimated time required: &lt;1 h)
<ol type="a">
<li>To begin with, seed cells in the chambered slide (15μ slide 8-well) in phenol red-free DMEM medium in the evening. Seed around ∼2 × 10<sup>4</sup> (U2OS), or ∼3 × 10<sup>4</sup> (HeLa/A549) cells in 250 μL of indicated medium/well. Incubate under normal cell growth conditions such as temperature 37°C, 5% CO2, and 90%–95% relative humidity (RH).
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The glass bottom chambered slide or dish can also be purchased from any other vendor providing them with high-quality coverslip. Its growth surface needs to provide a suitable attachment of the applied cell type. Please ensure that the materials used do not affect the morphological and physiological properties of the study cells.</div>
DAY-2 (Estimated time required: 1–2 h)</li>
<li>Replace the culture medium from each well with fresh medium DMEM medium (250 μL/well).</li>
<li>Selecting the study plasmids and planning for transfection: Choosing the proper positive control depends on various aspects of the repair process you are studying. For DSB repair, we suggest using fluorescent-tagged early DNA damage sensing factor such as Nbs1 or nuclear sirtuins as a positive control. More importantly, Nbs1 recruits to the site of the damage within &lt;1 s and stays there for a very long time; thus, if you plan to study the repair process for a long time or late stages of the repair, Nbs1 serve an excellent option. Similarly, nuclear sirtuin such as SIRT1, SIRT6 or SIRT7, also recruits to the site of the damage within 20 s, thus can serve a perfect positive control. The choice of a second plasmid depends on the experimental plans. For example, a cell uses two different pathways to complete long-range end-resection, such as (i) BLM-DNA2-RPA-MRN, or (ii) EXO1-BLM-RPA-MRN pathway; thus, to plan your study, respective markers of each pathway can be chosen from the <a href="#tbl1">Table 1</a>.
<div class="note">
<span class="note-title">Note:</span> The empty vector backbone expressing (EGFP, RFP, mCherry, YFP, EBFP or any other) fluorescent tag alone can be used as a negative control of the experiment. This ensures the experimenter to distinguish between clear recruitment and exclusion (although very rare, but also from any passive diffusion) at the site of damage.</div>
</li>
<li>Prepare the transfection mix as per table (<a href="#tbl3">Table 3</a>). We used a Turbofect transfection reagent, but any other transfection reagent can be used for the transfection by following the manufacturer's instructions.
<table id="tbl3">
<caption>Table 3. Preparation of transfection mix</caption>
<thead>
<tr>
<th>Cell transfection</th>
<th>-</th>
<th>-</th>
<th>-</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>Part-A (DNA mix)</b></td>
<td> </td>
<td><b>Part-B (Transfection mix)</b></td>
<td> </td>
</tr>
<tr>
<td>Opti-MEM/DMEM</td>
<td>25-(X+Y) μL</td>
<td>Opti-MEM/DMEM</td>
<td>25-A μL</td>
</tr>
<tr>
<td>Plasmid-1 (EGFP-)</td>
<td>0.4 μg (X μL)</td>
<td rowspan="2">Transfection Reagent (If using Turbofect TR)</td>
<td rowspan="2">A μL (2.0 μL)</td>
</tr>
<tr>
<td>Plasmid-2 (Test)</td>
<td>0.4 μg (Y μL)</td>
</tr>
<tr>
<td>Total volume</td>
<td>25 μL</td>
<td>Total volume</td>
<td>25 μL</td>
</tr>
<tr>
<td colspan="4">Mix both Part-A and Part-B, incubate at room temperature (25°C) for 15 min</td>
</tr>
<tr>
<td colspan="4">Now drop wise transfer the transfection mix to the respective well of the slide.</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The above table represents the condition we successfully tested in Hela, U2OS and A549 cells using Turbofect transfection reagent. However, for other cells, one needs to standardize the transfection conditions. Alternatively, follow the manufacturer’s instruction for other transfection reagents by keeping the ratio of the plasmid 1 and 2 as 1:1.</div>
</li>
<li>
<b>Pre-sensitization-cum-medium change:</b> Six hours post-transfection, replace the culture medium with a phenol red-free complete DMEM medium supplemented with 10 μM BrdU (BrdU stock: 10 mM in H<sub>2</sub>O) in each well. Transfer as well as incubate them in a cell culture incubator for 24 h (Note down the time).
<div class="note">
<span class="note-title">Note:</span> Pre-sensitization with BrdU, or any other halogenated thymidine analogs, increases the sensitivity of DNA to irradiation in many ways (<a href="#bib7">Fujii, Genet et al., 2013</a>), but the generalized viewpoint states that upon UV irradiation, the single-strand break formation from BrdU radicals results in the formation of lethal DSBs. Therefore, this pre-sensitization step increases the frequency of laser-induced DSB formation.</div>
DAY-3 (Estimated time required: 2–4 h)</li>
<li>After about 22 h of the last medium change, turn on the microscope incubator and set the incubator temperature to 37°C, CO<sub>2</sub> to 5%, and relative humidity to 90%–95%. At this stage, one can also turn on the microscope, laser boxes, open the software and load the SIM settings/parameters described in “Equipment”. Now let the associated things warm up and monitor them.</li>
<li>After 24 h of pre-sensitization, transfer chambered slide containing transfected cells onto the pre-warmed incubator-microscope stage.</li>
<li>Locate, focus, and zoom-in (zoom factor 3-3.2×) over transfected cells (single or multiple cells). Check for intact nuclear morphology and uniform fluorescence in the nucleus, as shown for EGFP and mCherry channels (<a href="#fig4">Figure 4</a>A).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Uniform expression of fluorescently tagged repair factor</div>
<p>(A) The representative image of focused, zoom-in of U2OS cell expressing EGFP-SIRT6 and vRAGE-mCherry showing the uniform fluorescence of these repair factors across the nuclei and nucleolus.</p>
<p>(B) The representative image of one such cell expressing EGFP-tagged BLM helicase before (pre-) and after irradiation. The brightfield represents the morphology of the nucleus (Scale 10 μm).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Some factors show prominent normal sub-nuclear localization (BLM helicase in nucleolus). These cells can be used for laser-induced DNA damage studies without any issue, as shown (<a href="#fig4">Figure 4</a>B).</div>
</li>
<li>In the SIM setting wizard (described in <a href="#fig3">Figure 3</a>), load/write the parameters for laser settings (described in the Equipment section). As mentioned, we used a 63×/1.15 water to scan a straight line across a single plane, but not Z-stacks, of the nucleus of individual cells (zoom factor 3-3.2×) with a scan speed of 200 μs/pixel to yield a line of DNA damage of 0.3 μm width. The total time of cell exposure to laser radiation was 250 ms, with a total of about 8 frames scans. The recruitment kinetics was recorded over a gap of 15 s. The unirradiated scan (pre) served as an internal negative control.
<div class="note">
<span class="note-title">Note:</span> For most of the factors, we successfully used an activation time of 250 ms. This condition was tested for U2OS, Hela, A549, HCT, HEK, Fibroblasts, and other primary cells.</div>
</li>
<li>Now, decide the region of interest (ROI) shape and nuclear area you want to irradiate. Although it is possible to draw any shape like alphabets, squares, circles, and many other shapes to be irradiated by laser (<a href="#fig2">Figure 2</a>), we recommend using a single straight line across the nucleus, as highlighted in <a href="#fig2">Figure 2</a>, marked in the green box. The laser irradiation will induce DNA damage in the respective area of these drawn shapes.
<div class="note">
<span class="note-title">Note:</span> Several repair factors (Such as BLM, SIRT1, SIRT6, etc.) prominently localize to the nucleolus (as shown in <a href="#fig4">Figure 4</a>B), forming a very intense nucleolar structure (can be circular or oval-shaped). Hence straight lines are much easier to distinguish from these nucleolar structures than an irradiated circle. In addition, the straight-line limits stress to a small region and create a clear structure at high contrast that is easy to detect.</div>
</li>
<li>After ROI shape setup, next set the time interval and total duration of live microscopy. Adjust conditions for time-lapse imaging according to the dynamics of the sub-process studied. For example, Nbs1 serves as a DNA damage sensor, thus gets recruited to the site of the damage within 1 s. It stays there for a very long period, where other factors, such as SIRT6, SIRT1, vRAGE, come into play after Nbs1; therefore, while studying these factors along with NBS1, we use time interval 5–10 s post DNA damage. However, depending upon the study factor, one can change this parameter accordingly. While using FV1000/FV1200, one could even track the recruitment kinetics over an interval as low as 0.5 s.
<div class="critical">
<span class="critical-title">Critical:</span> The total duration of the live-cell microscopy (number and interval between the time points) depends on the repair factor, a process to be studied and its role in DNA repair, supposed, you plan to study DNA damage sensor complex related factors (like Nbs1, MRE11, or RAD50). Their recruitment starts immediately after laser-induced DNA damage, thus this process can be tracked from 1- to 120-sec(s). The MRN (MRE11-RAD50-Nbs1) complex reaches its maximum saturation intensity within 10–90 s and stays as such over a very long time. Moreover, If you plan to study end-processing events like resection, then one can track the recruitment from 2–5 min (for short-range resection), or longer (for long-range resection). Keep in mind that both short and long-range resection may take hours to complete. Therefore, depending upon the test factor's potential role, the total duration of the post-damage studies can be designed accordingly.</div>
</li>
<li>Now, start the acquisition and track the following changes.
<ol type="i">
<li>The Laser-induced minor bleaching across the selected shape as shown in <a href="#fig5">Figure 5</a>A.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Recruitment kinetics of SIRT7 and vRAGE at the site of DNA DSBs</div>
<p>(A) The minor photo-bleaching of SIRT7-EGFP expressed in U2OS cells induced by laser irradiation recovers within seconds. Subsequently, repair factors get recruited to the site of DSBs damage.</p>
<p>(B) Representative time-points of the recruitment of EGFP-SIRT7 (green) and vRAGE-mCherry (red) at the site of DNA DSBs in BrdU pre-sensitized U2OS cells.</p>
<p>(C) Quantification of the fluorescence intensity (measured at the place of damage) of EGFP-SIRT7, and mCherry-vRAGE before and after laser-induced DNA damage. The fluorescence intensity (in percentage) of the indicated factors in <a href="#fig5">Figure 5</a>B of irradiated versus unirradiated regions are plotted as the relative fluorescence intensity at each time point (Error bars indicate SEM; n &gt; 3. Scale 10 μm).</p>
</figcaption>
</figure>
</li>
<li>Recruitment of positive control or any other established repair factor (Here, for DSB repair, we used EGFP-SIRT7) at the site of irradiation <a href="#fig5">Figures 5</a>B and 5C.</li>
<li>Now check the recruitment kinetics of the study factor (Here, for DSB repair, we used mCherry-vRAGE).</li>
</ol>
</li>
<li>Save the recorded video and process the data accordingly with <i>ImageJ</i> or any other comparative software. The following link can be used as a guide to arrange the video microscopy data with <i>ImageJ</i> (<a href="https://imagej.nih.gov/ij/docs/guide/">https://imagej.nih.gov/ij/docs/guide/</a>). To analyze the recruitment kinetics data, the fluorescence intensity values of the recruited factors can be evaluated. The recorded videos can be opened in ImageJ. Now select and draw a straight line precisely on the damaged area from ROI manager of the <i>ImageJ.</i> Now select the multi-measure (for all the slices) from the “more” window of ROI manager. Copy the result values in an excel file. Now repeat the measurements in the undamaged area for all the time points/slices. Repeat this for other cells too. For graphical presentation, the mean fluorescence intensity of irradiated region (ROIirr)/Fluorescence intensity of unirradiated (ROIunirr), then normalized with the fluorescence intensity at pre-damage (t=0) of the experiment. The final normalized values for each can be plotted against the time interval used during video microscopy. Representative still images and quantification of live recruitment EGFP-tagged human SIRT7 and mCherry-tagged human vRAGE for this DSB repair assay are shown in <a href="#fig5">Figures 5</a>B and 5C.<br> Moreover, to verify the application of this protocol in a different cell line, we also used A549 expressing EGFP-Nbs1 and vRAGE-mCherry, and the representative movie (<a href="#mmc1">Methods video S1</a>) shows the DSBs-dependent recruitment kinetics of these factors.
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Representative video file showing the live recruitment kinetics of the indicated repair factors after laser mediated DNA DSBs in A549 cells, step 1 m</div>
<p>Here green or red fluorescence represents EGFP-Nbs1, or mCherry-vRAGE, respectively.</p>
</figcaption>
</figure>
Protocol to study DNA DSBs repair by immunofluorescence (IF) staining method:<br> This protocol describes the method to study DNA DSBs without tracking the live recruitment of repair factors. This protocol does not involve the transfection step but instead uses Paraformaldehyde (PFA, 4% in 1 × PBS) based fixation and IF staining with desired antibodies.<br> DAY-1 (Estimated time required: &lt;1 h)</li>
<li>Seed cells in respective wells of the chambered slide (15μ slide 8-well Grid-500) in complete phenol red-free DMEM medium, supplemented with 10 μM BrdU in the morning. Seed around ∼2 × 10<sup>4</sup> (U2OS), or ∼3–4 × 10<sup>4</sup> (HeLa) cells in 250 μL of indicated medium/well. Incubate under normal cell growth conditions (temperature 37°C, 5% CO<sub>2</sub>, and 90%–95% relative humidity (RH), Note down the time).<br> DAY-2 (Estimated time required: 2–3 h)</li>
<li>After about 22–23 h of BrdU pre-sensitization, turn on the microscope incubator and set the incubator temperature to 37°C, CO<sub>2</sub> to 5%, and relative humidity to 90%–95%.<br> At this stage, you can also turn on the microscope, laser boxes and start the software and load the settings/parameters described in DSBs Protocol (step-f). Now let the environment chamber to warm up for at least an hour.</li>
<li>After 24 h of BrdU pre-sensitization, transfer slide containing cells (Grid-500) onto the pre-warmed incubator-microscope stage.</li>
<li>Focus and locate the well-grown areas of cells in the culture well. Now find the region of the chamber where cells are well spaced and healthy. Note down the lane markings in the surface grid, as shown in <a href="#fig6">Figure 6</a>.
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Bird’s eye view of a grid slide well</div>
<p>The representative drawing of the Grid slide (15μ slide 8-well Grid-500) and individually zoomed well (zoom factor 3–3.2×), indicating the surface labels for quickly locating the region of irradiation.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Please do not forget to note the lane and well descriptions in the notebook (e.g., like well-5 lane 9D). Any error in noting can make it very difficult to locate the irradiated zone after immunofluorescence (IF) staining.</div>
</li>
<li>Now, draw the line(s) to damage across the cell nucleus. One can target a single cell or multiple cells at a time, as described in <a href="#fig7">Figure 7</a>.
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Plan to irradiate individual or multiple cells</div>
<p>The images show two options of drawing the irradiation regions to either target single cells or multiple cells at a time. The Bright field images showing the representative unzoomed window view of U2OS cells. The red line(s) shows the selected ROIs for laser irradiation. Scale 10 μm.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> To ensure the uniform damage and best tracking of the recruitment, we suggest using individual targeting or targeting a limited number of cells at a time.</div>
</li>
<li>Use the laser settings described in DSBs Protocol (step-i) and activate the damage pulse for 250 ms.</li>
<li>Repeat steps (steps- q to s) quickly to irradiate the desired number of cells in the same or other wells. The faster the irradiation can be carried out, the closer the timing of the induced phenotypes is.</li>
<li>Processing of the irradiated cells for fixation:
<ol type="i">
<li>The time gap between the cells' irradiation and processing depends upon the repair factor/ step of DNA DSBs repair you are studying. For broad guidance, please refer to <a href="#tbl4">Table 4</a>.
<table id="tbl4">
<caption>Table 4. Average recruitment time of the factors involved in DSBs or ICL repair</caption>
<thead>
<tr>
<th>S.No.</th>
<th>Process</th>
<th>Post irradiation time</th>
<th>Recommended markers</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>a)</b></td>
<td colspan="3">HR repair</td>
</tr>
<tr>
<td>1.</td>
<td>DNA Damage response</td>
<td>1–10 min(s)</td>
<td>γH2AX, MRE11, Nbs1, pATM etc</td>
</tr>
<tr>
<td>2.</td>
<td>DNA DSBs processing</td>
<td>10–60 min</td>
<td>pRPA2, SOSS1, ATRIP etc</td>
</tr>
<tr>
<td>3.</td>
<td>Strand Invasion /exchange</td>
<td>3–6 h</td>
<td>RAD51, Mus81 etc</td>
</tr>
<tr>
<td><b>b)</b></td>
<td colspan="3">NHEJ repair</td>
</tr>
<tr>
<td>1.</td>
<td>DNA Damage response</td>
<td>1–10 min(s)</td>
<td>γH2AX, PARP etc</td>
</tr>
<tr>
<td>2.</td>
<td>End protection/processing</td>
<td>10–60 min</td>
<td>Ku70, Ku80etc</td>
</tr>
<tr>
<td>3.</td>
<td>End-Ligation</td>
<td>4–6 h</td>
<td>Lig-4</td>
</tr>
<tr>
<td><b>c)</b></td>
<td colspan="3">ICLs repair</td>
</tr>
<tr>
<td>1.</td>
<td>ICLs detection</td>
<td>0.5–4 min(s)</td>
<td>FANCD2, SLX4</td>
</tr>
<tr>
<td>2.</td>
<td>Crosslink processing</td>
<td>&gt;4 min</td>
<td>SLX1. Mus81, Gen1 etc</td>
</tr>
</tbody>
</table>
</li>
<li>After a suitable post-irradiation time slot, remove the cell culture medium from the irradiated well and wash the growth surface three times with 1× PBS. Now fix the cells using ice-cold PFA (250 μL/well), and incubate at 4°C for 15 min. Remove Paraformaldehyde (PFA), incubate/wash the fixed cells once with 50 mM Tris-Cl (pH=7.5; to inactivate the residual PFA), and then with 1× PBS for 5 min each time.
<div class="note">
<span class="note-title">Note:</span> Suppose the expression of the factor studied is low or the intensity of its recruitment is weak, one can pre-extract the irradiated cells (before fixation) in the cytoskeleton extraction (CSK) buffer. CSK buffer treatment partially permeabilizes the live cells to release their unbound cytosolic/nucleoplasmic contents without affecting the factors bound to sub-cellular structures and chromatin. For more details on CSK mediated pre-extraction, please follow ((<a href="#bib22">Sawasdichai et al., 2010</a>) #281). The recipe of the CSK buffer is provided in <a href="#materials-and-equipment">materials and equipment</a> section. The time of CSK extraction needs to be empirically determined with the respective cell types used in the study. The following Triton X-100 concentration and time we successfully used for the indicated cells in our laboratory, such as fibroblasts (0.1%, 1–2 min), U2OS (0.2%, 3 min), Hela and A549 (0.3%, 4 min).</div>
DAY-3 (Estimated time required: 2–3 h)</li>
</ol>
</li>
<li>Proceed with immunofluorescence (IF) staining protocol (<a href="#bib12">Kumar et al., 2017</a>). After IF staining, locate (as noted in step-q), and image the irradiated cells.
<div class="note">
<span class="note-title">Note:</span> Although the pre-extraction with CSK buffer is not mandatory, permeabilization associated with IF staining protocol is mandatory and is irrespective of CSK extractions.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>DNA Interstrand Cross-Link (ICLs) damage and repair<br> DNA interstrand cross-links are highly cytotoxic forms of DNA lesions, leading to oncogenic transformations, premature aging, and death, if not repaired in time. The ICLs repair involves a Fanconi-anemia-dependent or an independent mechanism to repair the damage. Moreover, the molecular event that occurs after ICLs DNA damage can provide valuable information about novel factors that might be exploited to develop modern anti-cancer therapeutics. Therefore, a study involving the recruitment kinetics of known and potential repair factors plays an important role in better understanding ICLs repair pathways.<br> This protocol involves many similar steps as described for DSBs studies, but it significantly differs at several points. Moreover, like DSBs repair, this method is also provided in two formats: one involves the live recruitment based analysis of the potential factors, whereas the other involves an antibody-based immunofluorescence staining approach. A detailed description of these methods is given below:<br> Protocol to study recruitment and repair kinetics of factors involved in DNA ICL repair by live microscopy. DAY-1 (Estimated time required: &lt; 1 h)
<ol type="a">
<li>Seed cells in the chambered slide (15μ slide 8-well) in complete phenol red-free DMEM medium in the evening. Seed around ∼1.5 × 10<sup>4</sup> (U2OS) or ∼2 × 10<sup>4</sup> (HeLa) cells in 250 μL of indicated medium/well. Incubate under normal cell growth conditions such as temperature 37°C, 5% CO2, and 90%–95% relative humidity (RH).<br> DAY-2 (Estimated time required: 1–2 h)</li>
<li>Replace the culture medium from each well with fresh medium DMEM medium (250 μL/well).</li>
<li>Prepare and transfect the cells with control (consider <a href="#tbl1">Table 1</a> for positive control suggestions) and desired study plasmid as per <a href="#tbl3">Table 3</a>. We suggest using EGFP-FANCD2 or EGFP-SLX4 as a positive control to check if ICL repair is induced. These markers get recruited to the site of ICL within 15 s(s), therefore, serve as a very good positive control.<br> The choice of the second plasmid depends upon one’s experimental plan; like experimenter can choose another ICL repair plasmid from <a href="#tbl1">Table 1</a> and <a href="#tbl4">Table 4</a>, or it can be their own test factor.</li>
<li>After 6 h of transfection, replace the medium with fresh phenol red-free DMEM and incubate the cells for 24–36 h in a cell culture incubator (since FANCD2 and SLX1/4 are very large proteins. Therefore, significant expression/intense fluorescence of these factors need at least 30–36 h post-transfection).<br> DAY-3 (Estimated time required: 2–4 h)</li>
<li>After 33–34 h of transfection, turn on the microscope incubator and set the incubator temperature to 37°C degrees, CO<sub>2</sub> to 5%, and relative humidity to 90%–95%. At this stage, one can also turn on the microscope, laser boxes and start the microscope program software and load the settings/parameters described in “Equipment”. Now let the system warm-up.</li>
<li>After 35–36 h post-transfection, prepare cells for ICL damage by pre-sensitizing them with 25 μM of TM-Psoralen for 30 min (Prepare TMP solution using DMSO as a solvent; discard the leftover solution as per disposal guidelines). Do not use a TMP solution older than 12 h.
<div class="note">
<span class="note-title">Note:</span> Psoralen derivatives are planar cyclic compounds that can readily enter and intercalate in the DNA strands. Activation of intercalated psoralen molecules with UV irradiation (355–365 nm) forms covalent crosslinks/adducts by involving pyrimidine bases (mostly Thymidine; T) of the dsDNA and the reactive groups of psoralen. TM-psoralen has two reactive groups. Therefore, it simultaneously forms bifunctional adducts involving two DNA bases and prominently used for inducing ICLs.</div>
</li>
<li>After TM-Psoralen pre-sensitization, transfer chambered slide with sensitized/transfected cells onto the microscope stage and select suitable cells expressing the EGFP and mCherry (or other fluorescent tags).</li>
<li>Now fine focus and zoom-in (zoom factor 3-3.2×) to the respective well to locate an extended, well-grown cell with intact nuclear morphology and uniform fluorescence in the nucleus as described in DSBs Protocol (step-h).</li>
<li>In the SIM settings window, select the 355 nm laser and load/write the parameters for other settings described in DSBs Protocol (step-i). For ICL factors, we need an exposure time of 300–400 ms. These conditions have been thoroughly tested for U2OS, Hela, A549, HCT, HEK, fibroblasts, and other primary cells).
<div class="note">
<span class="note-title">Note:</span> For laser-induced ICL damage, we need a pulsed 355 nm laser. Therefore, if you are planning to develop a new laser-induced DNA damage setup, please consider spending on an appropriate 355 nm laser instead of a 405 nm laser.</div>
</li>
<li>Now, decide the shape and nuclear area you want to expose to radiation and draw the ROI as described in DSBs Protocol (step-j), as well as in <a href="#fig2">Figure 2</a>.</li>
<li>After setting the region to irradiate, the next step is to set the time interval and total duration. FANCD2 gets recruited to the site of ICL damage within 10 s; therefore, to begin with, one can check the DNA ICL damage response/recruitment kinetics using an interval of 15–20 s for the desired duration (<a href="#tbl4">Table 4</a>).
<div class="note">
<span class="note-title">Note:</span> If the study intervals are very close, one can adopt the time and total duration settings accordingly.</div>
</li>
<li>Start the time-lapse and track the recruitment kinetics of the ICL repair positive control and test factor in a similar way as described in DSBs Protocol (step-l). Here for representation, we have used EGFP-tagged human FANCD2 and mCherry-tagged human vRAGE for this ICL repair assay. Save the recorded video and arrange the data accordingly presented in DSBs Protocol (step-m). A representative movie and the still images of live recruitment EGFP-tagged human FANCD2 and mCherry-tagged human vRAGE for the ICL repair assay are shown in <a href="#fig8">Figures 8</a>A and 8B and <a href="#mmc2">Methods video S2</a>.
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Recruitment of ICLs repair factors at the site of damage</div>
<p>(A) The representative still images of live microscopy showing time-dependent recruitment of FANCD2 and RAGE variant-<i>V</i> at the site of TMP mediated interstrand crosslink’s for ICLs repair in U2OS cells. The green fluorescence indicates GFP-tagged-FANCD2, whereas red fluorescence indicates mCherry tagged-vRAGE.</p>
<p>(B) Quantification of the fluorescence intensity of EGFP-FANCD2 and mCherry-vRAGE before and after laser-induced DNA damage. The fluorescence intensity (in percentage) of the indicated factors in <a href="#fig8">Figure 8</a>A of irradiated versus unirradiated regions were plotted as the relative fluorescence intensity at each time point (Error bars indicate SEM; n &gt; 3. Scale 10 μm).</p>
</figcaption>
</figure>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. Representative video file showing the live recruitment kinetics of the indicated repair factors after laser mediated DNA ICLs in U2OS cells, step 2 l</div>
<p>Here green or red fluorescence represents EGFP-FANCD2, or mCherry-vRAGE, respectively.</p>
</figcaption>
</figure>
Protocol to study DNA ICL repair process by immunofluorescence (IF) staining method:<br> This protocol describes the method to induce DNA ICLs, but it does not include live tracking of the repair process. It does not involve the transfection step but instead uses Paraformaldehyde (PFA) based fixation and later IF staining with the antibodies.<br> DAY-1 (Estimated time required: &lt;1 h)</li>
<li>Seed cells in phenol red-free complete DMEM medium in the chambered slide (15μ slide 8-well Grid-500; 2–3 × 10<sup>4</sup> cells/well for U2OS and 3-4 × 10<sup>4</sup> cells/well for Hela cells, or Cellview culture dish 3 × 10<sup>4</sup> cells/well) in the evening (Note down the time).<br> DAY-2 (Estimated time required: 2–4 h)</li>
<li>Next morning, turn on the microscope incubator and set the incubator temperature to 37°C degrees, CO<sub>2</sub> to 5%, and relative humidity to 90%–95%.<br> At this stage, one can also turn on the microscope, laser boxes, and open the software and load the settings/parameters described in the instrumental setup. Now let the system warm-up.</li>
<li>Now, pre-sensitize the growing cells with 25 μM of TM-Psoralen for 30 min as described in ICLs Protocol (step-f).</li>
<li>Focus and zoom (zoom factor 3–3.2×) into the culture well and locate the region of the chamber where cells are well spaced and healthy. Note down the lane markings in the surface grid, as described in <a href="#fig6">Figure 6</a>.
<div class="note">
<span class="note-title">Note:</span> Please ensure that you note down the lane and well description in the notebook (like 6D or 9H). Any error in noting can make it challenging to locate the irradiated cells after IF staining.</div>
</li>
<li>In the settings, select the 355 nm laser and load/write the parameters, as described for ICL live microscopy in ICLs Protocol (step-i).</li>
<li>Now, decide the shape and nuclear area you want to expose to radiation and draw the ROI accordingly and as described in <a href="#fig2">Figures 2</a> and <a href="#fig7">7</a>.</li>
<li>Laser irradiates the cell as described in live-cell experiment ICLs Protocol (step-l).</li>
<li>Quickly repeat steps from ICLs Protocol (step-q to t) for inducing the desired number of cells in the same or other wells in a very short time to keep the damage repair in the same time scale.</li>
<li>After the empirically determined post-irradiation time (5, 10, 15, 30, or 60 min), process the irradiated cells as described earlier for DSBs in Protocol (step-u to v). In short, if you are planning to study the ICLs sensor signaling/recruitment (like SLX4), we recommend fixing the cells in 15 min, whereas if you are studying the nuclease mediated unhooking of the ICLs, and we suggest you to wait at least 30–45 min before fixation.</li>
<li>Stain the processed cells with respective antibodies and locate the irradiated cells using the noted Grid coordinates from ICLs Protocol (step-p). For the immunofluorescence staining guide, please refer to the details given in DSBs Protocol (step-v). Inspect and process the data with <i>ImageJ</i> or similar software.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Single-strand breaks (SSBs) and repair<br> In this protocol, we describe the method to study single-strand breaks damage (SSBs damage). This protocol involves many similar steps as followed for DNA DSBs or ICLs, but differs at a few critical steps. As in the DSBs and ICL protocol, we divided this protocol into 2-sub categories.<br> Protocol to study recruitment and repair kinetics of factors involved in single-strand breaks (SSBs) repair by live microscopy: DAY-1 (Estimated time required: &lt;1 h)
<ol type="a">
<li>Seed cells in phenol red-free complete DMEM medium in the chambered slide (15μ slide 8-well; ∼3 × 10<sup>4</sup> (U2OS) or ∼4 × 10<sup>4</sup> (Hela) cells/well for Hela cells in 250 μL medium volume in the evening.<br> DAY-2 (Estimated time required: 1–2 h)</li>
<li>Replace the culture medium from each well with fresh medium DMEM medium (250 ul/well).</li>
<li>Prepare the transfection mix with the study plasmids as shown in <a href="#tbl1">Tables 1</a> and <a href="#tbl3">3</a>. We suggest using EGFP-PARP1 as a positive control. PARP1 gets recruited to SSBs within 10 s, therefore serving as a perfect positive control for SSBs. The choice of the second plasmid depends upon the experimental plan, as described in <a href="#tbl1">Table 1</a>.</li>
<li>About 6 h of post-transfection, replace the medium with phenol red-free DMEM, and incubate the cell for 24 h.<br> DAY-3 (Estimated time required: 2–4 h)</li>
<li>After 22–23 h of the last medium change, turn on the microscope incubator, laser, and load the settings described in the method. Set the microscope incubator temperature to 37°C, 5% CO<sub>2,</sub> and RH to 90%–95%.</li>
<li>After 24 h of post-transfection, transfer chambered slide containing transfected cells on the microscope stage and visualize the EGFP and other fluorescence mCherry, as shown in <a href="#fig4">Figure 4</a>.</li>
<li>Now, focus as well as zoom in (zoom factor 3–3.2×) same as mentioned earlier into the well extended/grown cell with intact nuclear morphology and uniform fluorescence in the nucleus as described in Protocol 1 (step-h).</li>
<li>In the settings window, select 355 nm Laser and load/write the other settings' parameters. For most of the ssDNA damage factors, we need an exposure time of 100–150 ms (These conditions were tested U2OS, Hela, A549, HCT, HEK, fibroblasts, and other primary cells).</li>
<li>Now, decide the shape and nuclear area you want to expose to radiation and draw the ROI as needed and described in DSBs Protocol (step-j).</li>
<li>After setting the ROI, the next step is to set the time interval and total duration of live microscopy. PARP gets recruited to the site of the damage within 10–20 s(s); therefore, to begin with, you can check the DNA damage response/recruitment kinetics after every 15–20 s for the desired time (<a href="#tbl4">Table 4</a>).
<div class="note">
<span class="note-title">Note:</span> If the recruitment kinetics are very fast, one can adapt the time and total duration settings accordingly.</div>
</li>
<li>Now, start the time-lapse and track the recruitment kinetics of the single-strand break repair positive control and test factor in a similar way as described in DSBs Protocol (step-l). For the representation, we have used mCherry tagged human PARP1 and EGFP-tagged human Cockayne syndrome group B (Csb).</li>
<li>Save the recorded time-lapse and process the data accordingly with <i>ImageJ</i> or any other similar software as mentioned in DSBs Protocol (step-m). Here we show the representative still images of live recruitment EGFP-CSB and mCherry-PARP1 for the SSB repair assay are shown in <a href="#fig9">Figure 9</a>.
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Recruitment kinetics of PARP1 and CSB at the site of ssDNA breaks</div>
<p>The representative still images of live recruitment of PARP1-mCherry and CSB-EGFP at the site of SSBs in U2OS cells at indicated time points. The red fluorescence indicates mCherry tagged-PARP1, and the green fluorescence indicates EGFP tagged-CSB (Scale 10 μm).</p>
</figcaption>
</figure>
Protocol for studying the single-strand DNA damage and study the repair process by IF staining:<br> This protocol describes the method to study ssDNA damage without tracking live recruitment studies. This protocol also can be implemented to any cell type. It does not involve the transfection step but instead uses Paraformaldehyde-based fixation and IF staining with the antibodies of interest.<br> DAY-1 (Estimated time required: &lt;1 h)</li>
<li>Seed cells in Phenol red-free complete DMEM medium in the chambered slide (15μ slide 8-well Grid-500; 2–3 × 10<sup>4</sup> cells/well for U2OS and 3–4 × 10<sup>4</sup> cells/well for Hela cells, or Cellview culture dish 3 × 10<sup>4</sup> cells/well) in the evening (Note down the time).<br> DAY-2 (Estimated time required: 2–4 h)</li>
<li>Turn on the microscope, laser, and load settings/parameters as described in Equipment Setup. Set the microscope incubator temperature to 37°C degrees and RH to 90%–95%.</li>
<li>Transfer chambered grid slide containing cells onto the microscope stage and focus and zoom-in (zoom factor 3–3.2×) to the well extended/grown cell with intact nuclear morphology as described in DSBs Protocol (step-h).</li>
<li>In the settings window, select 355 nm Laser and load/write the parameters described for live microscopy SSBs Protocol (step-h).</li>
<li>Now, decide the shape and nuclear area you want to expose to radiation and draw the ROI as needed and described in DSBs Protocol (step-j).</li>
<li>Use the laser settings described in ssDNA damage live microscopy protocol and irradiate the nuclei accordingly. Work quickly and repeat steps as per requirements.</li>
<li>After a suitable post-irradiation time (as described for ss-DNA damage in <a href="#tbl4">Table 4</a>), process the irradiated cells as described in DSBs Protocol (step-u to v) with or without CSK pre-extraction and proceed to IF staining with study markers.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Images and movies of the expected outcome of the protocol are shown in <a href="#fig5">Figures 5</a>, <a href="#fig8">8</a>, and <a href="#fig9">9</a> and <a href="#mmc1">Methods video S1</a> and <a href="#mmc2">S2</a>.</p>
<p>This protocol covers the detection of three kinds of DNA damage: DNA DSBs, DNA ICL, and ss DNA damage (<a href="#bib5">Ciccia and Elledge, 2010</a>, <a href="#bib19">Rosado et al., 2011</a>, <a href="#bib1">Anindya, 2020</a>); it can provide a useful tool to screen the participation of a novel repair factor in these 3-pathways. Furthermore, this method can be coupled with inhibitor studies and yields more precise data than other radiomimetic drug-based studies. In addition, the pathways or step-specific involvement (duration, time, etc.) of the test factor can be studied. Another important detail that needs to be considered is that laser-based damage studies are very fast and provide better chances to interpret your data in real-time.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Highlighted below are some potential limitations to consider during experimental design and data analysis.</p>
<p>DNA damage repair is a highly coordinated sequence of events that are tightly regulated by various signaling cascades. Therefore the completion of the DNA repair process takes time, and it depends on the state of the cell. Furthermore, the early repair events such as detecting damage, signaling, and assembly of repair complexes are very fast processes. Still, the very late events of the repair process, such as strand invasion (in HR repair of DSBs), strand ligation (in NHEJ repair of DSBs), nucleolytic processing, and processing of crosslinks (in ICLs repair), although trackable in live studies, require longer study time and duration.</p>
<p>Practically it is possible to induce laser-induced DNA damage in suspension culture, but due to their various axial or aggregation properties, post-damage live tracking of these cells can be problematic.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Slow out of focus movement of cells during live tracking in DSBs Protocol (step-l), ICLs Protocol (step-l), and SSBs Protocol (step-k).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>These types of movements are associated with microscope incubator temperature settings. Turn-on the microscope associated incubator in advance and let it condition to 37°C. Moreover, most of the eukaryotic cells show a very high rate of movements below 33°C. Another option to stabilize the focus is to apply a hardware autofocus device.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The intensity of DNA damage seems low.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>In a few cases, we observed that the recruitment intensity of the positive factor seemed weak. Although uncommon, it is mostly associated with a degraded pre-sensitizing agent such as BrdU or TMP stocks. This problem can be solved by properly storing the prepared BrdU solutions and using freshly made TMP solutions. Never use more than 24 h old TMP stocks for pre-sensitization.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>After laser irradiation, positive control was recruited, but the test factor was completely photo-bleached.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>This is an infrequent problem, but some proteins are too sensitive to photo-bleaching, very weak in their fluorescence or low in expression. This can be overcome by the fusion of 2 molecules of the same fluorescent tag to your study factor, which is showing a high photo-bleaching rate (<a href="#fig10">Figure 10</a>). This should solve this issue.</p>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/897-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. The solution to weak protein expression or photolabile factors</div>
<p>Schematic view of the suggested solution to reduce photo-bleaching or increase fluorescence intensity repair factors showing weak expression or fluorescence.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Anindya, R. (2020). Single-stranded DNA damage: Protecting the single-stranded DNA from chemical attack. DNA Repair (Amst) <i>87</i>, 102804. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref1">View at publisher</a></p>
<p id="bib2">Bhattacharya, S., Srinivasan, K., Abdisalaam, S., Su, F., Raj, P., Dozmorov, I., Mishra, R., Wakeland, E.K., Ghose, S., Mukherjee, S., and Asaithamby, A. (2017). RAD51 interconnects between DNA replication, DNA repair and immunity. Nucleic Acids Res. <i>45</i>, 4590-4605. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref2">View at publisher</a></p>
<p id="bib3">Branzei, D. and Foiani, M. (2008). Regulation of DNA repair throughout the cell cycle. Nat. Rev. Mol. Cell Biol. <i>9</i>, 297-308. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref3">View at publisher</a></p>
<p id="bib4">Caldecott, K.W. (2007). Mammalian single-strand break repair: mechanisms and links with chromatin. DNA Repair (Amst) <i>6</i>, 443-453. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref4">View at publisher</a></p>
<p id="bib5">Ciccia, A. and Elledge, S.J. (2010). The DNA damage response: making it safe to play with knives. Mol. Cell <i>40</i>, 179-204. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref5">View at publisher</a></p>
<p id="bib6">Deans, A.J. and West, S.C. (2011). DNA interstrand crosslink repair and cancer. Nat. Rev. Cancer <i>11</i>, 467-480. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref6">View at publisher</a></p>
<p id="bib7">Fujii, Y., Genet, M.D., Roybal, E.J., Kubota, N., Okayasu, R., Miyagawa, K., Fujimori, A., and Kato, T.A. (2013). Comparison of the bromodeoxyuridine-mediated sensitization effects between low-LET and high-LET ionizing radiation on DNA double-strand breaks. Oncol. Rep. <i>29</i>, 2133-2139. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref7">View at publisher</a></p>
<p id="bib8">Golia, B., Moeller, G.K., Jankevicius, G., Schmidt, A., Hegele, A., Preisser, J., Tran, M.L., Imhof, A., and Timinszky, G. (2017). ATM induces MacroD2 nuclear export upon DNA damage. Nucleic Acids Res. <i>45</i>, 244-254. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref8">View at publisher</a></p>
<p id="bib9">Jackson, S.P. (2002). Sensing and repairing DNA double-strand breaks. Carcinogenesis <i>23</i>, 687-696. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref9">View at publisher</a></p>
<p id="bib10">Kong, X., Mohanty, S.K., Stephens, J., Heale, J.T., Gomez-Godinez, V., Shi, L.Z., Kim, J.S., Yokomori, K., and Berns, M.W. (2009). Comparative analysis of different laser systems to study cellular responses to DNA damage in mammalian cells. Nucleic Acids Res. <i>37</i>, e68. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref10">View at publisher</a></p>
<p id="bib11">Kumar, V., Agrawal, R., Pandey, A., Kopf, S., Hoeffgen, M., Kaymak, S., Bandapalli, O.R., Gorbunova, V., Seluanov, A., Mall, M.A., et al. (2020). Compromised DNA repair is responsible for diabetes-associated fibrosis. EMBO J. <i>39</i>, e103477. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref11">View at publisher</a></p>
<p id="bib12">Kumar, V., Fleming, T., Terjung, S., Gorzelanny, C., Gebhardt, C., Agrawal, R., Mall, M.A., Ranzinger, J., Zeier, M., Madhusudhan, T., et al. (2017). Homeostatic nuclear RAGE-ATM interaction is essential for efficient DNA repair. Nucleic Acids Res. <i>45</i>, 10595-10613. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref12">View at publisher</a></p>
<p id="bib13">Lachaud, C., Castor, D., Hain, K., Munoz, I., Wilson, J., MacArtney, T.J., Schindler, D., and Rouse, J. (2014). Distinct functional roles for the two SLX4 ubiquitin-binding UBZ domains mutated in Fanconi anemia. J. Cell Sci. <i>127</i>, 2811-2817. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref13">View at publisher</a></p>
<p id="bib14">Lukas, C., Bartek, J., and Lukas, J. (2005). Imaging of protein movement induced by chromosomal breakage: tiny 'local' lesions pose great 'global' challenges. Chromosoma <i>114</i>, 146-154. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref14">View at publisher</a></p>
<p id="bib15">Madhavan, B., Han, Z., Singh, B., Bordt, N., Kaymak, S., Bandapalli, O.R., Kihm, L., Shahzad, K., Isermann., B., Herzig, S., et al. (2021). Elevated expression of the RAGE variant-V in SCLC mitigates the effect of chemotherapeutic drugs. Cancers <i>13</i>, 2843. <a class="external-link" href="https://doi.org/10.3390/cancers13112843">https://doi.org/10.3390/cancers13112843</a></p>
<p id="bib16">Matsuoka, S., Ballif, B.A., Smogorzewska, A., McDonald, E.R., Hurov, K.E., Luo, J., Bakalarski, C.E., Zhao, Z., Solimini, N., Lerenthal, Y., et al. (2007). ATM and ATR substrate analysis reveals extensive protein networks responsive to DNA damage. Science <i>316</i>, 1160-1166. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref16">View at publisher</a></p>
<p id="bib17">Muster, B., Rapp, A., and Cardoso, M.C. (2017). Systematic analysis of DNA damage induction and DNA repair pathway activation by continuous wave visible light laser micro-irradiation. AIMS Genet. <i>4</i>, 47-68. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref17">View at publisher</a></p>
<p id="bib18">Polo, S.E., Blackford, A.N., Chapman, J.R., Baskcomb, L., Gravel, S., Rusch, A., Thomas, A., Blundred, R., Smith, P., Kzhyshkowska, J., et al. (2012). Regulation of DNA-end resection by hnRNPU-like proteins promotes DNA double-strand break signaling and repair. Mol. Cell <i>45</i>, 505-516. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref18">View at publisher</a></p>
<p id="bib19">Rosado, I.V., Langevin, F., Crossan, G.P., Takata, M., and Patel, K.J. (2011). Formaldehyde catabolism is essential in cells deficient for the Fanconi anemia DNA-repair pathway. Nat. Struct. Mol. Biol. <i>18</i>, 1432-1434. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref19">View at publisher</a></p>
<p id="bib22">Sawasdichai, A., Chen, H.T., Hamid, N.A., Jayaraman, P.-S., and Gaston, K. (2010). In situ subcellular fractionation of adherent and non-adherent mammalian cells. J Vis Exp <i>41</i>, 1958. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/optKM11RNveyj">View at publisher</a></p>
<p id="bib20">Smith, R., Lebeaupin, T., Juhasz, S., Chapuis, C., D'Augustin, O., Dutertre, S., Burkovics, P., Biertumpfel, C., Timinszky, G., and Huet, S. (2019). Poly(ADP-ribose)-dependent chromatin unfolding facilitates the association of DNA-binding proteins with DNA at sites of damage. Nucleic Acids Res. <i>47</i>, 11250-11267. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref20">View at publisher</a></p>
<p id="bib21">Zhou, B.B. and Elledge, S.J. (2000). The DNA damage response: putting checkpoints in perspective. Nature <i>408</i>, 433-439. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00407-X/sref21">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contacts, Varun Kumar (<a href="mailto:varun.kumar@med.uni-heidelberg.de">varun.kumar@med.uni-heidelberg.de</a>, or <a href="mailto:varun.kumar@embl.de">varun.kumar@embl.de</a>, or <a href="mailto:kumarvarun3@gmail.com">kumarvarun3@gmail.com</a>).</p>
<h4>Materials availability</h4>
<p>Various plasmids for expressing the fluorescent-tagged repair factors are available from our group to institutional researchers. Requests for these can be made to Varun Kumar (<a href="mailto:dsbiclssb@gmail.com">dsbiclssb@gmail.com</a>)</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This study was supported by the <a href="https://doi.org/10.13039/501100001659">Deutsche Forschungsgemeinschaft</a> (SFB 1118) and DZD. We also thank all members of our group for their support. We also thank Dr. Stefan Terjung, Dr. Marco Lampe, Dr. Beate Neumann, Dr. Sabine Reither, and other members of ALMF-EMBL Heidelberg for their friendly support and suggestions.</p>
<h3>Author contributions</h3>
<p>Methodology, B.K.M., Z.H., and V.K.; validation, B.K.M. and V.K.; writing – original draft, B.K.M., V.K., and P.P.N.; writing-review &amp; editing, B.K.M., V.K., A.S., R.P., and P.P.N.; supervision, R.P., V.K., and P.P.N. All authors read this manuscript and agreed for the final submission.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>