<article>
<section>
<h1 id="header">Co-staining human PBMCs with fluorescent antibodies and antibody-oligonucleotide conjugates for cell sorting prior to single-cell CITE-Seq</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Xiaoshan Shi,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Gisele V. Baracho,<sup><a href="#aff2">2</a></sup> Woodrow E. Lomas,<sup><a href="#aff1">1</a></sup> Stephanie J. Widmann,<sup><a href="#aff2">2</a></sup> and Aaron J. Tyznik<sup><a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>BD Biosciences, 2350 Qume Drive, San Jose, CA 95131, USA</p>
<p id="aff2"><sup>2</sup>BD Biosciences, 11077 N Torrey Pines Road, La Jolla, CA 92037, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:aaron_j_tyznik@bd.com">aaron_j_tyznik@bd.com</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100893">10.1016/j.xpro.2021.100893</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The dual interrogation of the transcriptome and proteome with single-cell resolution provides exquisite insights into immune mechanisms in health and disease. Here, we describe a cutting-edge protocol wherein we combine Cellular Indexing of Transcriptomes and Epitopes by Sequencing (CITE-Seq), a technique utilizing antibody-oligonucleotide conjugates (AOCs), with fluorescence-activated cell sorting to enrich rare cell populations. Our protocol incorporates co-staining of cells with both fluorescent antibodies and AOCs simultaneously for subsequent input into the cell sorting and CITE-Seq pipeline.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref9">Mair et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Flow cytometry analysis to evaluate signals from cells at co-staining</li>
<li>Step-by-step protocol for co-staining cells with fluorescent antibodies and AOCs</li>
<li>Strategy to choose clones for the same protein marker with two antibodies</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1105-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Our protocol described herein are performed using fresh peripheral blood mononuclear cells (PBMCs) isolated from human whole blood. In this section, we provide a method for their isolation. Frozen PBMCs can also be used. Our co-staining protocol may also be suitable for co-staining other cell types from different species. Separately, we suggest preparing the appropriate reagents to be utilized in the assay in advance, as detailed below.</p>
<h3 id="sec1.1">Prepare cells: Isolation of PBMCs from human whole blood using SepMate<sup>TM</sup>-50</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Here, we present a method to isolate PBMCs from human whole blood using the SepMate<sup>TM</sup>-50 system (see product information sheet for detailed instructions). SepMate<sup>TM</sup>-50 tubes contain an insert that creates a physical barrier between the density gradient medium and blood, therefore eliminating the need for carefully layering blood over density gradient medium and reducing the total isolation time. SepMate<sup>TM</sup>-15 tubes can also be used if fewer PBMCs are required. Other isolation methods such as EasySep<sup>TM</sup> Direct Human PBMC Isolation Kit can also be used.</p>
<ol>
<li>Prepare PBS + 2% FBS without calcium or magnesium for diluting whole blood.</li>
<li>Obtain whole blood collected in anticoagulant-containing tubes such as BD Vacutainer® EDTA tubes.</li>
<li>Add 15 mL density gradient medium Ficoll to the 50 mL SepMate<sup>TM</sup> tube by carefully pipetting it through the central hole of the SepMate<sup>TM</sup> insert.</li>
<li>Dilute whole blood with an equal volume of PBS + 2% FBS and mix the sample gently. For example, dilute 5 mL of whole blood with 5 mL of PBS + 2% FBS.</li>
<li>Add the diluted blood on top of the Ficoll layer by pipetting down the side of the SepMate<sup>TM</sup> tube to preserve the gradient.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> No more than 17 mL of diluted blood should be added per SepMate<sup>TM</sup> capacity.</div>
<ol start="6">
<li>Centrifuge at 1200 × <i>g</i> for 10 min at room temperature (20°C–22°C), with the brake on.</li>
<li>Collect the top layer containing the enriched PBMCs using a Pipette Aid into a new 50 mL conical tube.</li>
<li>Pellet cells by centrifuging at 300 × <i>g</i> for 8 min.</li>
<li>Wash the enriched PBMCs with PBS + 2% FBS twice by centrifuging at 300 × <i>g</i> for 8 min.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Ammonium chloride–based lysis buffer can be added to the cell pellet prior to washes to remove red blood cells based on the optimal input for downstream applications. For example, dilute the 10X concentrate BD Pharm Lyse<sup>TM</sup> solution with H<sub>2</sub>O. Resuspend cell pellet derived from 20 mL blood with 3 mL 1X working solution and incubate at room temperature for 5 minutes followed by two washes.</div>
<ol start="10">
<li>Resuspend cells in a volume of staining buffer to obtain cells at ∼1 × 10<sup>7</sup> cells/mL for co-staining or flow proxy assay, as detailed below.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Approximately 1 million PBMCs can be isolated from 1 mL whole blood. Use this as an approximation to decide the volume of staining buffer. For example, resuspend PBMCs isolated from 5 mL blood into 0.5 mL staining buffer to obtain a concentration of ∼1 × 10<sup>7</sup> cells/mL.</div>
<h3 id="sec1.2">Designing a flow cytometry panel for cell sorting prior to single-cell CITE-Seq</h3>
<p>Isolating a specific cell population of interest using a cell sorter is a common step prior to single-cell CITE-Seq. This allows researchers to exclusively interrogate the cell populations of interest during single-cell sequencing and saves sequencing costs by excluding unwanted cells. Once the protein markers in the sorting panel are decided, successful panel design relies on optimal protein marker-fluorochrome assignment. Key considerations for building the sorting panel include, but are not limited to, instrument configuration, fluorescence spillover, dye brightness, antigen density, and co-expression (<a href="#bib1">Chattopadhyay and Roederer, 2012</a>; <a href="#bib5">Flores-Montero et al., 2019</a>). Evaluating the spread spillover matrix is useful to aid flow cytometry panel design specific to an instrument (<a href="#bib10">Nguyen et al., 2013</a>). Additional panel optimization may be needed to improve upon the initial panel design (<a href="#bib4">Ferrer-Font et al., 2020</a>; <a href="#bib8">Mahnke and Roederer, 2007</a>). Biotinylated primary antibodies, followed by secondary streptavidin-conjugated fluorescent antibodies for cell sorting are compatible with a downstream single-cell CITE-Seq workflow.</p>
<h3 id="sec1.3">Prepare stock solution of flow proxy oligos</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>We designed a flow proxy experiment that allows detection of the antibody-oligonucleotide conjugates (AOC) signal using flow cytometry. First, cells are stained with AOC. Fluorochrome oligonucleotide conjugates as flow proxy oligos containing the complimentary sequence to the AOC oligo are then added. With the binding of flow proxy oligos to AOC, the fluorescent signal is correlated to the AOC signal.</p>
<p>The flow proxy experiment is useful for different applications. It is possible to examine the AOC binding or identify the optimal quantity of any AOC reagent for a particular cell type using a flow proxy assay. Compared to using a fluorescent antibody with the same clone, this method allows researchers to titrate the exact lot of the AOC reagent, avoiding the scenario that the same clone of AOC and fluorescent antibody show different phenotypes (<a href="#bib9">Mair et al., 2020</a>). Additionally, identification of the optimal concentration of the AOC reagent will not be affected by the dye brightness of the flow antibody chosen. It is also possible to confirm oligo and antibody conjugation using this assay. In this work, we co-stained cells with fluorescent antibodies and AOC with the same protein marker. With the secondary labeling of the flow proxy oligo, we then evaluated the signal change upon co-staining compared to single staining.</p>
<p>We utilized two different flow proxy oligos conjugated to Alexa Fluor™ 647 (AF647). AF647-dT contains 25 thymines (T) and binds to the polyA tail of the BD® AbSeq oligonucleotide (<a href="#fig1">Figure 1</a>). The other flow proxy oligo, the AF647-AbSeq PCR handle, is complementary to the conserved sequence in the 5′ end of the AbSeq oligonucleotide (<a href="#fig1">Figure 1</a>). Using these two flow proxy oligos simultaneously may improve the detection of the AOC signal, based on our experience.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1105-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Graphical illustration of the binding between AOC and flow proxy oligos</div>
<p>The AF647-dT flow proxy oligo binds the polyA tail of the oligonucleotide in the AOC reagent, whereas the AF647-AbSeq PCR handle binds the 5′ region of the AOC oligonucleotide. With this secondary labeling, the fluorescence signal on the AF647 channel is correlated to the AOC signal.</p>
</figcaption>
</figure>
<p>According to vendor’s instructions, resuspend the lyophilized flow proxy oligos in DNA suspension buffer to make 100 μM stocks.</p>
<p>For this work, the read-out fluorophore for AOCs is always AF647, although other fluorochromes should also work. Here, the fluorescent antibodies are conjugated to various dyes, as derived from an existing multicolor cell sorting panel. Fluorochrome incompatibility of the co-stained sample in the flow proxy assay may arise from imbalanced sizes between fluorescent antibodies and AOC-dye oligo conjugate complexes competing to bind with the cells (<a href="#bib15">Zhang et al., 2017</a>). In that case, changing dyes may be advisable.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>BD Horizon<sup>TM</sup> BUV395 Mouse Anti-Human CD4</td>
<td>BD Biosciences</td>
<td>Cat# 563550; RRID: AB_2738273</td>
</tr>
<tr>
<td>BD Horizon<sup>TM</sup> BB515 Mouse Anti-Human CD45RO</td>
<td>BD Biosciences</td>
<td>Cat# 564529; RRID: AB_2744408</td>
</tr>
<tr>
<td>BD Pharmingen<sup>TM</sup> APC-H7 Mouse Anti-Human HLA-DR</td>
<td>BD Biosciences</td>
<td>Cat# 561358; RRID: AB_10611876</td>
</tr>
<tr>
<td>BD Horizon<sup>TM</sup> BV711 Mouse Anti-Human CD27</td>
<td>BD Biosciences</td>
<td>Cat# 564893; RRID: AB_2739003</td>
</tr>
<tr>
<td>BD Pharmingen<sup>TM</sup> PE Mouse Anti-Human CD11c</td>
<td>BD Biosciences</td>
<td>Cat# 555392; RRID: AB_395793</td>
</tr>
<tr>
<td>BD Horizon<sup>TM</sup> BV480 Mouse Anti-Human CD3</td>
<td>BD Biosciences</td>
<td>Cat# 566105; RRID: AB_2739507</td>
</tr>
<tr>
<td>BD Horizon<sup>TM</sup> BV421 Mouse Anti-Human CD19</td>
<td>BD Biosciences</td>
<td>Cat# 562440; RRID: AB_11153299</td>
</tr>
<tr>
<td>BD Horizon<sup>TM</sup> BV605 Mouse Anti-Human CD8</td>
<td>BD Biosciences</td>
<td>Cat# 564116; RRID: AB_2869551</td>
</tr>
<tr>
<td>BD Horizon<sup>TM</sup> BV786 Mouse Anti-Human CCR7</td>
<td>BD Biosciences</td>
<td>Cat# 566758; RRID: AB_2869852</td>
</tr>
<tr>
<td>BD Pharmingen<sup>TM</sup> PE-Cy7 Mouse Anti-Human CD56</td>
<td>BD Biosciences</td>
<td>Cat# 557747; RRID: AB_396853</td>
</tr>
<tr>
<td>BD OptiBuild<sup>TM</sup> BV786 Mouse Anti-Rat CD90/Mouse CD90.1</td>
<td>BD Biosciences</td>
<td>Cat# 740916; RRID: AB_2740558</td>
</tr>
<tr>
<td>BD OptiBuild<sup>TM</sup> BV480 Rat Anti-Mouse CD90.2</td>
<td>BD Biosciences</td>
<td>Cat# 746840; RRID: AB_2744090</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD4</td>
<td>BD Biosciences</td>
<td>Cat# 940001; RRID: AB_2875892</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD45RO</td>
<td>BD Biosciences</td>
<td>Cat# 940022; RRID: AB_2875913</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human HLA-DR</td>
<td>BD Biosciences</td>
<td>Cat# 940010; RRID: AB_2875901</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD27</td>
<td>BD Biosciences</td>
<td>Cat# 940018; RRID: AB_2875909</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD11c</td>
<td>BD Biosciences</td>
<td>Cat# 940024; RRID: AB_2875915</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD3</td>
<td>BD Biosciences</td>
<td>Cat# 940000; RRID: AB_2875891</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD3</td>
<td>BD Biosciences</td>
<td>Cat# 940307; RRID: AB_2876183</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD19</td>
<td>BD Biosciences</td>
<td>Cat# 940004; RRID: AB_2875895</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD19</td>
<td>BD Biosciences</td>
<td>Cat# 940247; RRID: AB_2876128</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD8</td>
<td>BD Biosciences</td>
<td>Cat# 940003; RRID: AB_2875894</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Rat Anti-Human CCR7</td>
<td>BD Biosciences</td>
<td>Cat# 940014; RRID: AB_2875905</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CCR7</td>
<td>BD Biosciences</td>
<td>Cat# 940394; RRID: AB_2876258</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD56</td>
<td>BD Biosciences</td>
<td>Cat# 940380; RRID: AB_2876248</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Human CD56</td>
<td>BD Biosciences</td>
<td>Cat# 940007; RRID: AB_2875898</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Mouse Anti-Mouse CD90/Mouse CD90.1</td>
<td>BD Biosciences</td>
<td>Cat# 940148; RRID: AB_2876037</td>
</tr>
<tr>
<td>BD® AbSeq Oligo Rat Anti-Mouse CD90.2</td>
<td>BD Biosciences</td>
<td>Cat# 940344; RRID: AB_2876216</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Whole blood</td>
<td>Any supplier</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>BD Pharm Lyse<sup>TM</sup> solution</td>
<td>BD Biosciences</td>
<td>Cat# 555899</td>
</tr>
<tr>
<td>BD Horizon™ Brilliant Stain Buffer (BSB) Plus</td>
<td>BD Biosciences</td>
<td>Cat# 566385</td>
</tr>
<tr>
<td>Cell Culture Phosphate Buffered Saline (1X)</td>
<td>Corning</td>
<td>Cat# 21-040-CV</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 26140079</td>
</tr>
<tr>
<td>Ficoll-Paque PLUS (Density Gradient Medium)</td>
<td>Cytiva</td>
<td>Cat# 17144003</td>
</tr>
<tr>
<td>Cell Stain Buffer (FBS)</td>
<td>BD Biosciences</td>
<td>Cat# 554656</td>
</tr>
<tr>
<td>Human Fc block</td>
<td>BD Biosciences</td>
<td>Cat# 564220</td>
</tr>
<tr>
<td>Pre-Sort Buffer</td>
<td>BD Biosciences</td>
<td>Cat# 563503</td>
</tr>
<tr>
<td>DAPI</td>
<td>BD Biosciences</td>
<td>Cat# 564907</td>
</tr>
<tr>
<td>7AAD</td>
<td>BD Biosciences</td>
<td>Cat# 559925</td>
</tr>
<tr>
<td>DNA Suspension Buffer</td>
<td>Teknova</td>
<td>Cat# T0221</td>
</tr>
<tr>
<td>Flow proxy oligos: 5′ Alexa Fluor 647-TTTTTTTTTTTTTTTTTTTTTTTTT 3' (AF647-dT) and 5′ Alexa Fluor 647-AGATCGGAAGAGCACACGTCTG 3' (AF647-AbSeq PCR handle)</td>
<td>Any supplier</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>BD Rhapsody<sup>TM</sup> Targeted mRNA and AbSeq Amplification Kit</td>
<td>BD Biosciences</td>
<td>Cat# 633774</td>
</tr>
<tr>
<td>BD Rhapsody<sup>TM</sup> Immune Response Panel Mm</td>
<td>BD Biosciences</td>
<td>Cat# 633753</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Seven Bridges (bioinformatic pipeline for processing the raw sequencing data)</td>
<td>BD Biosciences</td>
<td>RRID: SCR_008308 <a href="https://www.sevenbridges.com/">https://www.sevenbridges.com/</a>
</td>
</tr>
<tr>
<td>FlowJo™ v10.7.2 software (analysis and visualization of flow cytometry data)</td>
<td>BD Biosciences</td>
<td>RRID: SCR_008520 <a href="https://www.flowjo.com/solutions/flowjo">https://www.flowjo.com/solutions/flowjo</a>
</td>
</tr>
<tr>
<td>SeqGeq™ v1.6.0 software (analysis and visualization of single-cell CITE-Seq data)</td>
<td>BD Biosciences</td>
<td><a href="https://www.flowjo.com/solutions/seqgeq">https://www.flowjo.com/solutions/seqgeq</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Plastic whole blood tube with K2EDTA</td>
<td>BD</td>
<td>Cat# 366643</td>
</tr>
<tr>
<td>Falcon 50 mL Conical Centrifuge Tubes</td>
<td>Corning</td>
<td>Cat# 352070</td>
</tr>
<tr>
<td>Pipet Aid</td>
<td>Any supplier</td>
<td>N/A</td>
</tr>
<tr>
<td>SepMate<sup>TM</sup>-50</td>
<td>STEMCELL Technologies</td>
<td>Cat# 85450</td>
</tr>
<tr>
<td>Falcon round bottom polystyrene test tubes</td>
<td>Corning</td>
<td>Cat# 352054</td>
</tr>
<tr>
<td>Latch Rack for AbSeq Ab-Oligo tubes</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 4900-BR</td>
</tr>
<tr>
<td>8-Channel Screw Cap Tube Capper</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 4105MAT</td>
</tr>
<tr>
<td>1.5 mL DNA LoBind Tubes</td>
<td>Eppendorf</td>
<td>Cat# 0030108051</td>
</tr>
<tr>
<td>1.5 mL Protein LoBind Tubes</td>
<td>Eppendorf</td>
<td>Cat# 0030108442</td>
</tr>
<tr>
<td>Pipettes (P20, P200, P1000)</td>
<td>Major supplier</td>
<td>N/A</td>
</tr>
<tr>
<td>Low retention filtered pipette tips (20 μL, 200 μL, 1000 μL)</td>
<td>Major supplier</td>
<td>N/A</td>
</tr>
<tr>
<td>Flow cytometry cell analyzer</td>
<td>Major supplier</td>
<td>N/A</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Many reagents such as BD stain buffer and BSB Plus contain sodium azide. Contact with acidic solutions and metal compounds over time may form potentially explosive metal azides. Flush with adequate amounts of water when introducing these materials into a sanitary sewer system.</div>
<p>BD, the BD Logo, FACS, FlowJo, Horizon, OptiBuild, Pharm Lyse, Pharmingen, Rhapsody, SeqGeq and Vacutainer are trademarks of Becton, Dickinson and Company or its affiliates. Alexa Fluor is a trademark of Life Technologies Corporation. Cy is a trademark of Global Life Sciences Solutions Germany GmbH or an affiliate doing business as Cytiva. All other trademarks are the property of their respective owners.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Single-cell CITE-Seq can provide great insight to complex biological systems, uncover cellular heterogeneity and reveal novel characteristics (<a href="#bib11">Peterson et al., 2017</a>; <a href="#bib14">Stoeckius et al., 2017</a>). Combining single-cell sorting with single-cell CITE-Seq provides unparalleled resolution to deeply characterize rare cell populations (<a href="#bib9">Mair et al., 2020</a>; <a href="#bib13">See et al., 2018</a>). This protocol was generated to streamline the process of staining cells with flow cytometry antibodies and AOCs. The stained cells can be directly utilized for downstream cell sorting followed by single-cell capture.</p>
<h3 id="sec2.1">Optional step: Flow proxy experiment to evaluate the compatibility for the same protein marker of fluorescent antibody and AOC</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Approximately 2 h</div>
<p>When the fluorescent antibody and AOC against the same protein marker are used simultaneously, regardless of the binding epitopes, it is highly recommended to perform a flow proxy experiment to evaluate the compatibility of the two antibodies and the possible signal loss of the co-stained sample compared to the single-stained sample.</p>
<p>If there are no overlapping proteins between the fluorescent panel and the AOC panel, move directly to the “<a href="#sec2.2">co-staining PBMCs with a fluorescent panel and an AOC panel</a>” section.</p>
<ol>
<li>Obtain PBMCs at a concentration of ∼1 × 10<sup>7</sup> cells/mL. Use 0.5–1 million cells for each sample. It is essential to account for sufficient cells for use in 6 different samples per protein marker.</li>
<li>For cells containing myeloid cells and B cells, blocking potential non-specific Fc receptor-mediated false positive signals with human Fc block is recommended: use 5 μL human Fc block for each 100 μL PBMCs.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To block more cells, increase the amount of human Fc block proportionally. Incubate cells at room temperature for 10 minutes.</div>
<ol start="3">
<li>Label 6 polystyrene tubes for each protein marker to be tested and add 100 μL PBMCs (0.5–1 million cells/tube) into all 6 tubes.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A plate can also be used for cell staining instead of polystyrene tubes.</div>
<ol start="4">
<li>Add the corresponding antibodies to tube 3, 4, 5 and 6 for the first stain (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1105-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Graphical illustration of co-staining cells with the same protein marker of fluorescent antibody and AOC</div>
<p>Six tubes need to be prepared and stained with one or two antibodies to examine the possible signal change of the co-stained sample compared to the single stained samples.</p>
</figcaption>
</figure>
<table id="tbl2">
<caption>Table 2. Antibody quantities used for co-staining</caption>
<thead>
<tr>
<th>Protein marker</th>
<th>Fluorochrome</th>
<th>Flow antibody clone</th>
<th>Flow antibody optimal amount per test (μg)</th>
<th>AOC clone</th>
<th>AOC optimal amount per test (μg)</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="6">The same clones</td>
</tr>
<tr>
<td>CD4</td>
<td>BUV395</td>
<td>SK3</td>
<td>0.125<sup><a href="#tblfn1">a</a></sup>
</td>
<td>SK3</td>
<td>0.06</td>
</tr>
<tr>
<td>CD11c</td>
<td>PE</td>
<td>B-ly6</td>
<td>0.5<sup><a href="#tblfn1">a</a></sup>
</td>
<td>B-ly6</td>
<td>0.5<sup><a href="#tblfn1">a</a></sup>
</td>
</tr>
<tr>
<td>CD27</td>
<td>BV711</td>
<td>M-T271</td>
<td>1<sup><a href="#tblfn1">a</a></sup>
</td>
<td>M-T271</td>
<td>0.25</td>
</tr>
<tr>
<td>CD45RO</td>
<td>BB515</td>
<td>UCHL1</td>
<td>1<sup><a href="#tblfn1">a</a></sup>
</td>
<td>UCHL1</td>
<td>0.5</td>
</tr>
<tr>
<td>HLA-DR</td>
<td>APC-H7</td>
<td>G46-6</td>
<td>1<sup><a href="#tblfn1">a</a></sup>
</td>
<td>G46-6</td>
<td>1<sup><a href="#tblfn1">a</a></sup>
</td>
</tr>
<tr>
<td>CD56</td>
<td>PE-Cy7</td>
<td>B159</td>
<td>1<sup><a href="#tblfn1">a</a></sup>
</td>
<td>B159</td>
<td>0.5</td>
</tr>
<tr>
<td>CCR7</td>
<td>BV786</td>
<td>2-L1-A</td>
<td>2<sup><a href="#tblfn1">a</a></sup>
</td>
<td>2-L1-A</td>
<td>1</td>
</tr>
<tr>
<td>CD3</td>
<td>BV480</td>
<td>UCHT1</td>
<td>0.75<sup><a href="#tblfn1">a</a></sup>
</td>
<td>UCHT1</td>
<td>0.5</td>
</tr>
<tr>
<td>CD19</td>
<td>BV421</td>
<td>HIB19</td>
<td>1<sup><a href="#tblfn1">a</a></sup>
</td>
<td>HIB19</td>
<td>0.5</td>
</tr>
<tr>
<td>CD90.1</td>
<td>BV786</td>
<td>HIS51</td>
<td>0.5<sup><a href="#tblfn1">a</a></sup>
</td>
<td>HIS51</td>
<td>0.5<sup><a href="#tblfn1">a</a></sup>
</td>
</tr>
<tr>
<td>CD90.2</td>
<td>BV480</td>
<td>30-H12</td>
<td>0.5<sup><a href="#tblfn1">a</a></sup>
</td>
<td>30-H12</td>
<td>0.5<sup><a href="#tblfn1">a</a></sup>
</td>
</tr>
<tr>
<td colspan="6">Different clones</td>
</tr>
<tr>
<td>CCR7</td>
<td>BV786</td>
<td>2-L1-A</td>
<td>2</td>
<td>3D12</td>
<td>0.5</td>
</tr>
<tr>
<td>CD8</td>
<td>BV605</td>
<td>SK1</td>
<td>1</td>
<td>RPA-T8</td>
<td>0.25</td>
</tr>
<tr>
<td>CD3</td>
<td>BV480</td>
<td>UCHT1</td>
<td>0.75</td>
<td>SK7</td>
<td>0.25</td>
</tr>
<tr>
<td>CD19</td>
<td>BV421</td>
<td>HIB19</td>
<td>1</td>
<td>SJ25C1</td>
<td>0.06</td>
</tr>
<tr>
<td>CD56</td>
<td>PE-Cy7</td>
<td>B159</td>
<td>1</td>
<td>NCAM16.2</td>
<td>0.125</td>
</tr>
</tbody>
</table>
<p id="tblfn1" class="table-footnote"><sup>a</sup>antibody quantity used for co-staining with the same clones; for different clones of the same protein marker, use the corresponding optimal amount per test.</p>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Because of the possible competition of the fluorescently labeled antibody with AOC, it is important to add the two antibodies at the same time into the cells for tube 6. Alternatively, cells can be added after adding antibodies to the tubes.</div>
<div class="note">
<span class="note-title">Note:</span> For tube 6, if the fluorescent antibody and the AOC are of the same clone, use an equivalent quantity of the two antibodies based on the optimal amount from the vendor’s recommendations. If the optimal amount is different, select one amount to be used for both antibodies (see <a href="#fig3">Figure 3</a> as an example of identification of optimal antibody amount to use). If the two antibodies are different clones, use the corresponding optimal amount provided by the antibody vendor, which can be different.</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1105-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. An example of co-staining PBMCs with the same CD4 clone and conjugate utilized for surface detection and AOC</div>
<p>PBMCs were co-stained with BUV395 CD4 (clone SK3) and CD4 AOC (clone SK3). Histograms of unstained, single stained, and co-stained samples were overlaid to compare the flow signal (left plot) and AOC signal (right plot). Median fluorescence intensity (MFI) was reduced by approximately half for both fluorescent CD4 surface expression and CD4 AOC signals. The optimal quantity is 0.125 μg for BUV395 CD4 and 0.06 μg for CD4 AOC. 0.125 μg of each antibody was added for co-staining to utilize equal amounts of the two antibodies. 7AAD was used for live/dead cell discrimination.</p>
</figcaption>
</figure>
<ol start="5">
<li>Pipet mix and incubate the mixture at the room temperature for 30 min.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If staining murine cells or delicate human cells, incubation should be performed on ice.</div>
<ol start="6">
<li>Add 2 mL of stain buffer to tube 3, 4, 5 and 6 and centrifuge at 400 × <i>g</i> for 5 min.</li>
<li>Aspirate or decant supernatant and gently blot the tube on a lint-free wipe to remove residual liquid from the rim of the tube.</li>
<li>Add 2 mL of stain buffer a second time and centrifuge at 400 × <i>g</i> for 5 min.</li>
<li>Resuspend tube 3 in 350 μL stain buffer and store at 4°C until flow cytometry acquisition.</li>
<li>Resuspend tubes 4, 5 and 6 in 100 μL stain buffer.</li>
<li>Dilute 100 μM flow proxy oligos (AF647-dT and AF647-AbSeq PCR handle) using stain buffer to 3.75 μM.</li>
<li>Add 1 μL of each of 3.75 μM flow proxy oligos (2 μL in total) into tubes 2, 4, 5 and 6 containing 100 μL cells. Mix by pipetting 10 times.</li>
<li>Incubate the cells at room temperature for 20 min, protected from light.</li>
<li>Add 2 mL of stain buffer to tubes 1, 2, 4, 5, 6 and resuspend by pipet-mixing.</li>
<li>Centrifuge the tube at 400 × <i>g</i> for 5 min.</li>
<li>Add 2 mL of stain buffer to each tube a second time and mix the solution.</li>
<li>Centrifuge the tube at 400 × <i>g</i> for 5 min.</li>
<li>Resuspend cell pellets in 350 μL stain buffer to prepare the 6 tubes for the next step.</li>
<li>Samples can be analyzed using any flow cytometry analyzer with appropriate detectors. An example of co-staining PBMCs with CD4 AOC and CD4 fluorescent antibody is shown in <a href="#fig3">Figure 3</a>.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Add appropriate amount of live/dead marker to all tubes prior to flow cytometry analysis. Because fluorescent antibodies and flow proxy oligos may non-specifically bind with dead cells, we recommend including a live/dead dye to each tube to gate on live cells during flow cytometry analysis. For example, 7AAD or DAPI can be used.</div>
<div class="note">
<span class="note-title">Note:</span> Single-color control tubes need to be prepared for compensation calculation if there is spillover among fluorescent antibody, flow proxy dye and live/dead marker. If no live/dead marker is added, tube 3 and tube 5 can be used for single color compensation controls.</div>
<h3 id="sec2.2">Co-staining PBMCs with a fluorescent panel and an AOC panel</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>This protocol describes the procedure of co-staining cells with AOCs and fluorescent antibodies. After staining, cell sorting can be performed directly to enrich cell populations of interest, followed by single-cell CITE-Seq workflow.</p>
<ol start="20">
<li>Obtain the optimal volume per test of all antibodies in both the sorting panel and the AOC panel based on vendor recommendations or flow proxy experiment for the shared protein marker. For example, the optimal volume of BUV395 CD4 (BD Catalog #: 563550) is 5 μL per 1 million cells. The optimal volume of BD® AbSeq AOC reagents is 2 μL for staining 1 million cells.</li>
<li>Calculate the total volume of all antibodies to be used in the assay for both the sorting panel and the AOC panel.
<ol type="a">
<li>Subtract the total volume of the two panels from 200 μL to obtain the volume of stain buffer per sample. An example of calculation is in <a href="#tbl1">Table 1</a>.
<table id="tbl1">
<caption>Table 1. An example of co-staining</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Volume/test (1 million cells)</th>
</tr>
</thead>
<tbody>
<tr>
<td>10 plex fluorescent antibody panel</td>
<td>50 μL (5 μL × 10 protein markers)</td>
</tr>
<tr>
<td>20 plex AOC panel</td>
<td>40 μL (2 μL × 20 protein markers)</td>
</tr>
<tr>
<td>Cells in stain buffer</td>
<td>110 μL</td>
</tr>
<tr>
<td>Total volume</td>
<td>200 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Note: Antibody cocktail volumes can be scaled up based on the number of samples or amount of cells</p>
</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> when two or more BD Horizon™ Brilliant dyes are present in the sorting panel, Brilliant Stain Buffer (BSB) Plus should be used to mitigate dye to dye interaction and staining artifacts.</div>
<div class="note">
<span class="note-title">Note:</span> To stain more than 1 million cells, increase the antibody reagent volume proportionally.</div>
<ol start="22">
<li>Resuspend the PBMCs in the calculated volume of stain buffer in a 5 mL polystyrene tube.</li>
<li>Make the fluorescent antibody cocktail and the AOC cocktail separately and place them on ice until cells are ready for staining.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> If multiple tubes of BD® AbSeq AOCs are used, tubes can be organized in a Latch Rack. Centrifuge the Latch Rack with a plate adaptor at 400 × <i>g</i> for 30 seconds. Open the tubes using an 8-Channel Screw Cap Tube Capper to make the AOC cocktail (<a href="#fig4">Figure 4</a>).</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1105-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Latch rack for BD® AbSeq Ab-Oligo tubes and 8-channel screw cap tube capper</div>
<p>BD® AbSeq tubes can be organized in the Latch Rack, as shown. The tubes can be opened and closed using the Screw Cap Tube Capper.</p>
</figcaption>
</figure>
<ol start="24">
<li>Blocking non-specific binding of Fc receptors: add 5 μL of human Fc block per 1 million cells in suspension and incubate cells for 10 min at room temperature. Increase the volume of Fc block appropriately for increasing cell numbers.</li>
<li>Combine the two cocktail mixtures and add PBMCs to the antibody cocktail mixtures.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Pre-staining cells with the competing antibodies or the antibody that is particularly important for downstream characterization may be a good strategy to preserve the staining resolution. For example, researchers can add the same protein marker present in the AOC panel and the fluorescent panel, if there is any, to the cells first. After 10 minutes of incubation, the two remaining cocktail mixtures can be added.</div>
<ol start="26">
<li>Pipet to mix and incubate for 30–60 min at room temperature.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If staining murine cells or delicate human cells, incubation should be performed on ice.</div>
<ol start="27">
<li>Add 2 mL stain buffer to the labeled cells and mix by pipetting.</li>
<li>Centrifuge at 400 × <i>g</i> for 5 min.</li>
<li>Aspirate or decant supernatant and gently blot the tube on a lint-free wipe to remove residual liquid from the rim of the tube.</li>
<li>Repeat the wash for an additional two times by centrifuging at 400 × <i>g</i> for 5 min in between.</li>
<li>Resuspend the cell pellet in Pre-Sort buffer for cell sorting.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The co-stained cells can be kept at 4°C in the dark before FACS sorting. However, we highly recommend proceeding to cell sorting as quickly as possible to preserve the cell viability.</div>
<h3 id="sec2.3">Fluorescence-activated cell sorting to isolate cell population of interest</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Approximately 1 h</div>
<p>Cell sorting on co-stained cells can be performed using any type of cell sorter. A high level of procedure overview is provided below. Refer to the specific user guide of the instrument used.</p>
<ol start="32">
<li>Start up the cytometer and the computer.</li>
<li>Perform quality control of the cell sorter such as running Cytometer Setup and Tracking (CS&amp;T) beads.</li>
<li>Prepare the instrument for sorting following specific instrument user guidelines and biosafety regulations (<a href="#bib12">Schmid and Dean, 1997</a>).</li>
<li>Visualization of the fluorescent sorting panel using co-stained cells.
<ol type="a">
<li>Run all the single-color controls and calculate the compensation.</li>
<li>Run a small portion of the co-stained cells to visualize the fluorescent sorting panel and ensure proper staining.</li>
</ol>
</li>
<li>Proceed to sorting the co-stained sample.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To avoid potential transcriptional and proteomic changes and preserve cell viability, keep sorted cells on ice and proceed immediately to downstream single-cell capture (<a href="#bib2">Cossarizza et al., 2019</a>).</div>
<h3 id="sec2.4">Single-cell capture using BD Rhapsody<sup>TM</sup> single-cell analysis system</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Approximately 2 h</div>
<p>A detailed protocol for single-cell capture using the nano-well based BD Rhapsody™ system is described in <a href="#bib3">Erickson et al. (2020)</a> or here.</p>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The Exonuclease-treated beads can be stored at 2°C–8°C for ≤ 3 months. During the library preparation process, there are multiple stopping points that allow researchers to break down the process into different days if needed.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Using the flow proxy experiment described above, we evaluated the compatibility of several protein markers of AOC and fluorescent antibodies with the same specificity for co-staining, including either the same clone or different clones of the two specificities (<a href="#fig5">Figure 5</a>). For protein markers of the same clone, reduction of signal is typically observed for both the fluorescent antibody and AOC due to their competition (<a href="#fig5">Figure 5</a>A). One exception is CD56 AOC, which showed significant reduction of signal intensity (red arrow in <a href="#fig5">Figure 5</a>A), whereas the remaining protein markers showed an acceptable signal loss with co-staining. However, results are more nuanced when using different clones of the same protein marker (<a href="#fig5">Figure 5</a>B). For example, despite signal retention when co-staining using different clones of CD8, suboptimal staining of CCR7 AOC was observed with single staining. Additionally, CD3 AOC and CD19 AOC exhibited drastic signal reduction upon co-staining compared to single AOC staining. We therefore swapped different clones of these three protein markers to utilize the same clone. Signal loss was reduced with co-staining (<a href="#fig5">Figure 5</a>C). Interestingly, when we switched CD56 AOC into a non-competing clone, the AOC signal was restored.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1105-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Examples of co-staining PBMCs with the same protein marker of AOC and fluorescent antibody</div>
<p>(A) Co-staining PBMCs with the same clone of AOC and fluorescent antibody. Equivalent amount of the two antibodies were used for co-staining. Reduction of signal was observed for almost all cases at co-staining. Notably, when co-staining PBMCs with the same clone of CD56 PE-Cy7 and CD56 AOC, the signal of AOC was significantly reduced (red arrow) whereas the fluorescent signal remained comparable to the single stained sample. Either 7AAD or DAPI was used as a live/dead cell marker.</p>
<p>(B) Co-staining PBMCs with different clones of the same protein marker of AOC and fluorescent antibody. Diverse results of signal loss were observed during co-staining. 7AAD was used for discriminating dead cells from live cells.</p>
<p>(C) The signal loss was reduced when changing different clones of CCR7, CD3 and CD19 into the same clone. Additionally, changing CD56 AOC from clone B159 to clone NCAM16.2 restored the AOC signal. DAPI was included as a live/dead cell marker. Approximately 1 million PBMCs were used per sample. The percent and MFI of the positive population were labeled in all plots to evaluate signal loss. See <a href="#tbl2">Table 2</a> for antibody concentrations used for co-staining.</p>
</figcaption>
</figure>
<p>Based on the data observed, we generated a decision tree on how to choose the same protein marker of AOC and fluorescent antibody for co-staining (<a href="#fig6">Figure 6</a>). Because results are less predictable when using different clones of two antibodies, we recommend starting from the same clone for co-staining—preferably a clone that you know may work well. Signal for both fluorescent antibody and AOC can be evaluated using the median fluorescence intensity (MFI) in the corresponding channels. By comparing the MFI between the single-stained and co-stained samples, researchers can evaluate the signal loss quantitatively. If researchers are unable to identify the cell population of interest and the percent of positive cells changes (see CD56 AOC signal loss in <a href="#fig5">Figure 5</a>A as an example), seeking non-competing clones for the same protein marker is required. By switching the clone of one antibody, it is possible to restore the signal intensity of either the AOC or the fluorescent antibody, such as CD56 in <a href="#fig5">Figure 5</a>C. However, signal restoration of the two antibodies cannot always be expected. In this regard, researchers need to evaluate the signal loss of the AOC and fluorescent antibody combination on a case-by-case basis and choose a clone from a practical standpoint.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1105-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Decision tree for choosing AOC and fluorescent antibodies with the same specificity</div>
<p>We recommend users start from the same clone of the two antibodies and use a flow proxy experiment to evaluate the signal change of co-staining compared to single-staining conditions. If changing into a different clone still shows signal loss in the flow proxy experiment, researchers can choose the best combination of the two antibodies that is most suitable for cell sorting and single-cell CITE-Seq.</p>
</figcaption>
</figure>
<p>Using the decision tree, we identified the optimal amount and clone of CD90.1 and CD90.2 fluorescent antibodies and AOC reagents for co-staining. We performed the comprehensive workflow including co-staining, cell sorting and single-cell CITE-Seq. We could clearly gate on CD90.1 positive and CD90.2 positive cells during cell sorting (<a href="#fig7">Figure 7</a>A) and identify cell subsets characterized by CD90.1 and CD90.2 protein expression in single-cell CITE-Seq (<a href="#fig7">Figure 7</a>B). This supports the protocol presented and the strategy of choosing antibodies for co-staining cells with the same protein marker of AOC and flow antibody.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1105-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. An example of single-cell CITE-Seq on sorted cells co-stained with the same protein marker of AOCs and fluorescent antibodies</div>
<p>Murine splenocytes were co-stained with a 21-color fluorescent antibody panel and a 34-plex AOC panel containing overlapped protein markers of CD90.1 and CD90.2. The same clones of these two specificities were used and antibody quantities are provided in <a href="#tbl2">Table 2</a>.</p>
<p>(A) CD90.1<sup>+</sup> cells (pink) and CD90.2<sup>+</sup> cells (green) were clearly resolved and sorted during FACS sorting.</p>
<p>(B) Different sorted populations including CD90.1<sup>+</sup> and CD90.2<sup>+</sup> cells were pooled together and processed for single-cell CITE-Seq using the BD Rhapsody<sup>TM</sup> single-cell analysis system. Approximately 80% sequencing saturation (∼6500 reads/cell) was achieved for the AOC library. Similar to the flow cytometry data, CD90.1<sup>+</sup> and CD90.2<sup>+</sup> cell subsets can also be separated using bivariate plot in single-cell CITE-Seq. Additionally, these two populations can be identified via t-distributed stochastic neighbor embedding (tSNE) plots using CD90.1 and CD90.2 protein marker expression.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<h3 id="sec4.1">Limitations to choosing antibodies for the overlapping protein marker in the AOC and fluorescent panels</h3>
<p>If the signal loss while using the same clone of two antibodies is not acceptable and changing clones of one antibody still shows signal loss, researchers need to evaluate which antibody configuration is best for the downstream applications. If obtaining a strong signal for single-cell CITE-Seq is of top priority, researchers can choose the scenario in which AOC intensity is well preserved at co-staining whereas the fluorescent signal for cell sorting is compromised. If signal resolution for the flow antibody is significantly impacted, researchers may consider changing the dye into a brighter fluorochrome if the sorting panel allows.</p>
<h3 id="sec4.2">Limitations to cell washes post co-staining</h3>
<p>Sufficient washing post-AOC staining is important for reducing the noise from unbound AOCs being captured by BD Rhapsody™ system beads during single-cell capture and also help preserve the resolution of fluorescent antibodies. However, each wash results in a certain level of cell loss. Washing AOC-labeled cells 3 times is necessary for reducing the unbound antibodies. Additionally, the process of cell sorting replenishes the buffer where cells reside, which also helps to remove unbound AOCs.</p>
<h3 id="sec4.3">Considerations of optimal cell sorting conditions</h3>
<p>The conditions of cell sorting profoundly affect the cell viability of sorted cells (<a href="#bib2">Cossarizza et al., 2019</a>), which is crucial for the data quality of single-cell CITE-Seq. The optimal input cell concentration is critical to ensure that cells are delivered in a consistent concentration to the sorter. Too many cells per second will compromise the purity and yield of sorted cells whereas too few cells per second can increase the sorting time, potentially impacting the quality of sorted cells. Secondly, consider the optimal cell sorting nozzle size to best fit the cell type to be sorted. Thirdly, resuspend the cells to be sorted in an optimal buffer to avoid clumps and preserve cell viability and ensure that the catch buffer is optimized to support the nutrient requirements for the sorted cells. Note that Pre-Sort buffer is well suited for both resuspending cells to be sorted and collecting sorted cells.</p>
<h3 id="sec4.4">Considerations of optimal cell staining workflow</h3>
<p>There are two different methods to stain cells for cell sorting followed by single-cell CITE-Seq: co-staining cells with the fluorescent and AOC panels before sorting; or staining sorted cells with AOC panel after sorting. Co-staining may be a better approach in that it saves time by streamlining the workflow and reducing processing time, improving viability and cell resilience. Sorted cells labeled with AOCs can be directly processed for single-cell capture without any additional steps. Additionally, for rare cell populations, washing sorted cells multiple times post-AOC staining, can be detrimental in that the procedure can lead to severe cell loss. In addition, a workflow where staining with a fluorescent antibody followed by cell sorting, and then staining with AOC of the same specificity will have limited success as target antigens are likely bound with fluorescently conjugated antibodies that will block the binding of AOCs and limit their utility for downstream analysis.</p>
<h3 id="sec4.5">Correlation between flow proxy assay results and single-cell CITE-Seq results</h3>
<p>The flow proxy experiment is a useful tool to evaluate the protein marker expression level in single-cell CITE-Seq. This is supported by high concordance of cellular composition identified by the two readouts using the same biological sample (<a href="#bib6">Lawlor et al., 2021</a>; <a href="#bib9">Mair et al., 2020</a>). For example, if significant signal loss is observed at co-staining in the flow proxy experiment, researchers need to be cautious when using the antibody combination for cell sorting followed by single-cell CITE-Seq. However, the measurement unit between flow cytometry (median fluorescence intensity) and single-cell CITE-Seq (protein molecule count per cell) is not the same. Additionally, the same antibody clone can yield a different result in a multiomic sequencing experiment relative to flow cytometry (<a href="#bib9">Mair et al., 2020</a>). Hence, the change of fluorescence signal intensity correlated to AOC reagent in the flow proxy assay at co-staining may not be directly translated to the change of protein expression in single-cell CITE-Seq.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>When adding the live/dead marker 7AAD into the AOC-stained cells, the 7AAD negative population shifts slightly in the positive direction, possibly due to the binding of 7AAD with the oligonucleotides in the AOC reagent (step 19 or in the <a href="#sec2.4">single-cell capture using BD RhapsodyTM single-cell analysis system</a> section).</p>
<h3 id="sec5.2">Potential solution</h3>
<p>If the 7AAD negative population shifts to the positive side in the flow proxy experiment, ensure that the correct concentration of flow proxy oligos has been utilized. If the shift takes place during the sorting for co-stained cells, make a less stringent gate for the live cells. If the shift is too significant and interferes with accurately gating on live cells, either titrating 7AAD or utilizing an alternative live/dead marker is recommended. For example, DAPI shows less this type of issue compared to 7AAD.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>Compromised resolution of the fluorescent cell sorting panel (Step 35b).</p>
<h3 id="sec5.4">Potential solution</h3>
<p>For high-parameter fluorescent sorting panels, pre-testing the panel may be needed before executing the co-staining workflow. For fluorescent antibodies that generate non-optimal staining, titrate the reagents and incorporate isotype controls when needed. Use the optimal PMT voltage settings of the instrument (<a href="#bib7">Maecker and Trotter, 2006</a>). Ensure the compensation matrix is calculated accurately using single-stained controls. For low expressed protein markers, single-stained beads may be needed for compensation calculation.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>Cell clumping before sorting (after step 31).</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Be sure to filter co-stained cells prior to cell sorting. Larger nozzle size and resuspending cells using Pre-Sort buffer may be helpful to prevent sorter clogging due to cell aggregation.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>Low cell viability prior to single-cell capture (in the <a href="#sec2.4">single-cell capture using BD RhapsodyTM single-cell analysis system</a> section).</p>
<h3 id="sec5.8">Potential solution</h3>
<p>This indicates that cells have been stressed during the workflow. Make sure to work swiftly, spin cells properly, pipette cells gently and keep cells on ice as much as possible throughout the workflow. Be sure to monitor cell viability during the workflow. If cell viability drops after single-cell isolation (e.g., tumor dissociation process), optimize the cell preparation protocol if needed. If cell sorting affects cell viability, include a live/dead marker into the sorting panel and seek to optimize the sorting conditions. For example, using polypropylene collection tubes is helpful to prevent cell trapping by the tube wall. Precoating the collection tubes with FBS provides another approach to improve the cell viability of sorted cells.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>Unexpected low cell viability of sorted cells measured by BD Rhapsody<sup>TM</sup> scanner (in the <a href="#sec2.4">single-cell capture using BD RhapsodyTM single-cell analysis system</a> section).</p>
<h3 id="sec5.10">Potential solution</h3>
<p>The measurement of cell viability using the BD Rhapsody<sup>TM</sup> scanner prior to single-cell capture relies on DRAQ7<sup>TM</sup>, which binds to double-stranded DNA of dead cells. If the sorted cells are also stained with a fluorescent antibody conjugated to an APC dye, due to the similar excitation and emission spectral profile between DRAQ7<sup>TM</sup> and APC, an overestimation of the dead cell count of the sorted cell population may occur. In that case, researchers may need to count cell viability under a microscope with a hemocytometer using a non-fluorescent live/dead cell marker such as trypan blue.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chattopadhyay, P.K. and Roederer, M. (2012). Cytometry: today's technology and tomorrow's horizons. Methods <i>57</i>, 251-258. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref1">View at publisher</a></p>
<p id="bib2">Cossarizza, A., Chang, H.D., Radbruch, A., Acs, A., Adam, D., Adam-Klages, S., Agace, W.W., Aghaeepour, N., Akdis, M., Allez, M., et al. (2019). Guidelines for the use of flow cytometry and cell sorting in immunological studies (second edition). Eur. J. Immunol. <i>49</i>, 1457-1973. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref2">View at publisher</a></p>
<p id="bib3">Erickson, J.R., Mair, F., Bugos, G., Martin, J., Tyznik, A.J., Nakamoto, M., Mortimer, S., and Prlic, M. (2020). AbSeq protocol using the nano-well cartridge-based rhapsody platform to generate protein and transcript expression data on the single-cell level. STAR Protoc. <i>1</i>, 100092. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref3">View at publisher</a></p>
<p id="bib4">Ferrer-Font, L., Pellefigues, C., Mayer, J.U., Small, S.J., Jaimes, M.C., and Price, K.M. (2020). Panel design and optimization for high-dimensional immunophenotyping assays using spectral flow cytometry. Curr. Protoc. Cytom. <i>92</i>, e70. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref4">View at publisher</a></p>
<p id="bib5">Flores-Montero, J., Kalina, T., Corral-Mateos, A., Sanoja-Flores, L., Pérez-Andrés, M., Martin-Ayuso, M., Sedek, L., Rejlova, K., Mayado, A., Fernández, P., et al. (2019). Fluorochrome choices for multi-color flow cytometry. J. Immunol. Methods <i>475</i>, 112618. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref5">View at publisher</a></p>
<p id="bib6">Lawlor, N., Nehar-Belaid, D., Grassmann, J.D.S., Stoeckius, M., Smibert, P., Stitzel, M.L., Pascual, V., Banchereau, J., Williams, A., and Ucar, D. (2021). Single cell analysis of blood mononuclear cells stimulated through either LPS or anti-CD3 and anti-CD28. Front Immunol. <i>12</i>, 636720. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref6">View at publisher</a></p>
<p id="bib7">Maecker, H.T. and Trotter, J. (2006). Flow cytometry controls, instrument setup, and the determination of positivity. Cytometry A <i>69</i>, 1037-1042. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref7">View at publisher</a></p>
<p id="bib8">Mahnke, Y.D. and Roederer, M. (2007). Optimizing a multicolor immunophenotyping assay. Clin. Lab Med. <i>27</i>, 469-485. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref8">View at publisher</a></p>
<p id="bib9">Mair, F., Erickson, J.R., Voillet, V., Simoni, Y., Bi, T., Tyznik, A.J., Martin, J., Gottardo, R., Newell, E.W., and Prlic, M. (2020). A targeted multi-omic analysis approach measures protein expression and low-abundance transcripts on the single-cell level. Cell Rep. <i>31</i>, 107499. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref9">View at publisher</a></p>
<p id="bib10">Nguyen, R., Perfetto, S., Mahnke, Y.D., Chattopadhyay, P., and Roederer, M. (2013). Quantifying spillover spreading for comparing instrument performance and aiding in multicolor panel design. Cytometry A <i>83</i>, 306-315. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref10">View at publisher</a></p>
<p id="bib11">Peterson, V.M., Zhang, K.X., Kumar, N., Wong, J., Li, L., Wilson, D.C., Moore, R., McClanahan, T.K., Sadekova, S., and Klappenbach, J.A. (2017). Multiplexed quantification of proteins and transcripts in single cells. Nat. Biotechnol. <i>35</i>, 936-939. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref11">View at publisher</a></p>
<p id="bib12">Schmid, I. and Dean, P.N. (1997). Introduction to the biosafety guidelines for sorting of unfixed cells. Cytometry <i>28</i>, 97-98. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref12">View at publisher</a></p>
<p id="bib13">See, P., Lum, J., Chen, J., and Ginhoux, F. (2018). A single-cell sequencing guide for immunologists. Front Immunol. <i>9</i>, 2425. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref13">View at publisher</a></p>
<p id="bib14">Stoeckius, M., Hafemeister, C., Stephenson, W., Houck-Loomis, B., Chattopadhyay, P.K., Swerdlow, H., Satija, R., and Smibert, P. (2017). Simultaneous epitope and transcriptome measurement in single cells. Nat. Methods <i>14</i>, 865-868. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref14">View at publisher</a></p>
<p id="bib15">Zhang, L., Fang, Y., Kopeček, J., and Yang, J. (2017). A new construct of antibody-drug conjugates for treatment of B-cell non-Hodgkin's lymphomas. Eur. J. Pharm. Sci. <i>30</i>, 36-46. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00599-2/sref15">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Aaron Tyznik (<a href="mailto:Aaron_J_Tyznik@bd.com">Aaron_J_Tyznik@bd.com</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate code.</p>
<h3>Acknowledgments</h3>
<p>The authors would like to thank Adam Wright and Jody Martin for their guidance and expertise on the BD® AbSeq antibody-oligo conjugates and flow proxy assay. We are grateful to Ludovic Monheim, Nihan Kara, Chad Sisouvanthong, Gunjan Choudhary, and Bipulendu Jena for reviewing the manuscript. Provision of editorial assistance by Laura Richert-Spuhler, who significantly contributed to writing the manuscript, was funded by BD. Daren Passolt and Shirin Darbani generated the graphical abstract for this manuscript. We thank John Sedy and Wai Lin in Sanford Burnham Prebys Medical Discovery Institute for providing us mouse tissues for the single-cell CITE-Seq experiment.</p>
<h3>Author contributions</h3>
<p>X.S., S.J.W., and A.J.T. wrote the original draft. X.S., G.V.B., and W.E.L. generated the data for figures.</p>
<h3>Declaration of interests</h3>
<p>All authors are employees of BD Biosciences.</p>
</section>
</article>