<article>
<section>
<h1 id="header">Quantification of adipocyte numbers in transgenic mice via the Cre-LoxP recombination sites</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Caroline Moser,<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup> Leon G. Straub,<sup><a href="#aff1">1</a></sup> Yael Rachamin,<sup><a href="#aff1">1</a></sup> and Christian Wolfrum<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Food Nutrition and Health, Eidgenössische Technische Hochschule Zürich (ETH), Schwerzenbach 8603, Switzerland</p>
<p id="fn1"><sup>2</sup>Lead contact</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:christian-wolfrum@ethz.ch">christian-wolfrum@ethz.ch</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:caroline-moser@ethz.ch">caroline-moser@ethz.ch</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100761">10.1016/j.xpro.2021.100761</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes a method to assess adipocyte numbers within a specific depot based on their inducible genomic label. By extracting DNA from a complete adipose tissue depot stemming from two transgenic mouse lines (Adipoq-CreERT2 x ROSA26-tdRFP and Ucp1-CreERT2 x ROSA26-tdRFP), the number of adipocytes can be determined based on the quantification of the recombined LoxPRed sites. This highly sensitive system allows for the quantification of white, brown, and brite/beige adipocytes in a spatially unbiased and size-independent manner.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00467-6/sref5">Moser et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Employ transgenic mouse lines to label specific genomic sites in adipocytes</li>
<li>Isolate genomic DNA from a complete adipose tissue depot</li>
<li>Quantify adipocytes number, based on the genomic labels</li>
<li>The principle of the protocol can be adapted to other transgenic mouse lines and cells</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was established to enable the quantification of adipocytes. With the Adipoq-tracer mouse line (Adipoq-CreERT2 x ROSA26-tdRFP), all adipocytes can be inducible labeled and quantified, while the Ucp1-tracer mouse line (Ucp1-CreERT2 x ROSA26-tdRFP) exclusively labels all brown and brite/beige adipocytes. By qPCR the Cre-driven LoxPRed recombination sites are specifically quantified and compared to the genomic DNA copy number (ApoB locus) [<a href="#fig1">Figure 1</a>] and thus the number of adipocytes and the total cell number are determined. In comparison to other methods, like histology or flow cytometry, this highly sensitive method allows for the quantification of adipocytes unbiased regarding their size or spatial location. Between 6 – 10 mice per condition are suggested to obtain reproducible results. Currently, this protocol is based on the quantification of the recombination sites obtained by the Ucp1-creERT2 or Adipoq-creERT2 driven ROSA26–tdRFP recombination (<a href="#bib4">Luche et al., 2007</a>). This method can be also be adapted to other Cre-driver mouse lines crossed to the ROSA26-tdRFP construct (<a href="#bib4">Luche et al., 2007</a>), hence enabling the quantification of other cell types than adipocytes. Further, by designing primers binding to another recombination site and generating a plasmid harboring the new recombination site of interest, the principle of the method can be also applied to other already existing transgenic mouse lines.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Transgenic mouse lines</div>
<p>The transgenic Adipoq-tracer mouse line (Adipoq-CreERT2 × ROSA26-tdRFP) allows white, brown, and brite/beige adipocytes to be labeled by RFP, while the Ucp1-tracer mouse line (Ucp1-CreERT2 <b>×</b> ROSA26-tdRFP) exclusively labels brown and brite/beige adipocytes. The tamoxifen-inducible recombination site of the LoxPRed is used for the quantification of adipocytes.</p>
</figcaption>
</figure>
<p>The inducible CreERT2-LoxP system is a very useful tool for the temporal-controlled and tissue-specific recombination of DNA. However, at higher doses of tamoxifen, toxic effects were reported (<a href="#bib6">Ye et al., 2015</a>). With the very low dose of 2<b>×</b> 2 mg, we use 30%–80% less tamoxifen compared to others (<a href="#bib1">Hesselbarth et al., 2015</a>; <a href="#bib3">Lee et al., 2015</a>; <a href="#bib6">Ye et al., 2015</a>). However, we only established the minimal effective dose for the adipose tissue. If analyses of other tissues/cell types are planned, the ideal dose of tamoxifen has yet to be determined.</p>
<h3 id="sec1.1">Induction of recombination</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0–1 h on 2 consecutive days</div>
<ol>
<li>Oral gavage of 2 mg tamoxifen per animal per day.
<ol type="a">
<li>100 μL of the tamoxifen solution (as described in the <a href="#materials-and-equipment">materials and equipment</a> section) are orally gavaged with a gavage needle or a plastic feeding tube.</li>
<li>Repeat the step described in the section above the next day.</li>
</ol>
</li>
<li>Full recombination can be observed latest two days after the last tamoxifen treatment.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> One dose of 2 mg tamoxifen was described to sufficiently induce full recombination in the main adipose tissue depots (<a href="#bib5">Moser et al., 2021</a>). Two applications on two consecutive days were chosen as a precaution in case technical issues with the oral gavage happen to occur during the first treatment with tamoxifen. The minimal dose required to sufficiently label all cells of interest has yet to be determined for other cell types/tissues.</div>
<div class="note">
<span class="note-title">Note:</span> In this protocol tamoxifen ordered from Sigma Aldrich was used, however is can be also ordered from other companies (eg. VWR).</div>
<h3 id="sec1.2">Tissue harvest</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0–30 min per mouse</div>
<ol start="3">
<li>Euthanize the mouse using carbon dioxide.</li>
<li>Harvest the adipose tissue depots of interest. The fat depot nomenclature ascribed by de Jong et al (<a href="#bib2">de Jong et al., 2015</a>) was utilized. One lobe of each depot is to be used for the quantification of adipocytes.
<ol type="a">
<li>From the inguinal white adipose tissue depot, the popliteal lymph nodes should be carefully removed [<a href="#fig2">Figure 2</a>].
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Lymph node in the inguinal white adipose tissue depot</div>
<p>As the lymph nodes contain a high number of immune cells that can heavily influence the number of total cells, said lymph nodes should be carefully removed from the samples during the dissection of the tissue.</p>
</figcaption>
</figure>
</li>
<li>If two lobes per adipose tissue depot are available, the other lobe can be used for further analysis (e.g., histology or gene expression).</li>
</ol>
</li>
<li>Place the adipose tissue samples in 2 mL Eppendorf tubes and snap-freeze them in liquid nitrogen.</li>
<li>Store the samples at −20°C until further processing.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is critical to dissect the adipose tissue depots very precisely. Depots tightly connected to skeletal muscles must be removed especially carefully, as remnants of skeletal muscle can drastically influence total cell numbers [<a href="#fig3">Figure 3</a>].</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Skeletal muscle adhering to brown adipose tissue</div>
<p>Other tissues adhering to the tissue of interest can influence the number of total cells, thus they should be carefully removed from the samples during the dissection of the tissue.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> The tissue harvesting process can influence the results. When adhering to the classification of the depots by de Jong et al. (<a href="#bib2">de Jong et al., 2015</a>) there was no interpersonal variation observed (<a href="#bib5">Moser et al., 2021</a>)<b>.</b> If this method is to be applied to other organs/cell types, similar schemes/guidelines for tissue harvesting are suggested.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The samples can be stored at −20°C until further processing.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Tamoxifen</td>
<td>Sigma-Aldrich</td>
<td>Cat# T5648</td>
</tr>
<tr>
<td>Fast SYBR™ Green Master Mix</td>
<td>Thermo Scientific</td>
<td>Cat# 4385610</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Ucp1-CreERT2 × ROSA26-tdRFP × Ucp1-DTR-GFP</td>
<td>Moser et al.</td>
<td>N/A</td>
</tr>
<tr>
<td>Adipoq-CreERT2 × ROSA26-tdRFP × Ucp1-DTR-GFP</td>
<td>Moser et al.</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>qPCR primer LoxPRed and ApoB</td>
<td>Microsynth</td>
<td>Listed in <a href="#materials-and-equipment">materials and equipment</a>
</td>
</tr>
<tr>
<td>Plasmid pUC57recloxPRed-ApoB</td>
<td>Moser et al.</td>
<td>Listed in <a href="#materials-and-equipment">materials and equipment</a>
</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>QuantStudio Real-Time PCR software</td>
<td>Applied Biosystems</td>
<td>Version 1.3</td>
</tr>
<tr>
<td>GraphPad Prism 9</td>
<td>GraphPad Software</td>
<td>Version 9.0.1</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Chow diet</td>
<td>Kliba-Nafag, Kaiseraugst</td>
<td>#2222</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Tamoxifen Solution (20 mg/mL):</p>
<p>Please work in a biosafety cabinet while preparing the tamoxifen solution. Add 600 mg of Tamoxifen powder to a 50 mL falcon tube. Distribute the tamoxifen powder at the wall of the falcon tube. Carefully and slowly add 30 mL of sunflower oil to the tamoxifen powder. Wrap the falcon tube in aluminum foil, as tamoxifen is light-sensitive, and shake the solution for ca. 1 h until the tamoxifen powder is fully dissolved and no crystals are visible anymore. Dissolving can be eased by heating the solution to 37°C while shaking e.g., by putting it onto a shaker placed in an incubator <a href="#troubleshooting">Troubleshooting 1.</a> When fully dissolved, prepare 1 mL aliquots and store them for up to 12 months at −20°C. Avoid repeated freeze/thaw cycles. When thawing the tamoxifen aliquots, make sure the tamoxifen is fully dissolved; no crystals should be visible.</p>
<table id="undtbl1">
<caption>50 mM NaOH</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaOH</td>
<td>50 mM</td>
<td>2 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 20°C–25°C with no time limit.</p>
<table id="undtbl2">
<caption>1 M Tris-HCl pH 8</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>1 M</td>
<td>60.57 g</td>
</tr>
<tr>
<td>HCL</td>
<td>n/a</td>
<td>Until pH 8 is reached</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>0.5 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>0.5 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 20°C–25°C with no time limit.</p>
<table id="undtbl3">
<caption>Primer sequences</caption>
<thead>
<tr>
<th>Primer name</th>
<th>Primer sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>Recombined LoxPRed forward</td>
<td>GCGCATGAACTCTTTGATGAC</td>
</tr>
<tr>
<td>Recombined LoxPRed reverse</td>
<td>TCGCGGTTGAGGACAAACTC</td>
</tr>
<tr>
<td>ApoB forward</td>
<td>GTCCAGGTTGAATCACGGGT</td>
</tr>
<tr>
<td>ApoB reverse</td>
<td>AGGATCCTGCAAGGTCAAGC</td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>Primer mix recombined LoxPRed:</caption>
<thead>
<tr>
<th>Primer name</th>
<th>Final concentration in mix</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Recombined LoxPRed forward (100 μM Stock)</td>
<td>2.5 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>Recombined LoxPRed reverse (100 μM Stock)</td>
<td>2.5 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>380 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>400 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C with no time limit. The final concentration of each primer in the qPCR reaction should be 250 nM.</p>
<table id="undtbl5">
<caption>Primer mix ApoB</caption>
<thead>
<tr>
<th>Primer name</th>
<th>Final concentration in mix</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>ApoB forward (100 μM Stock)</td>
<td>2.5 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ApoB reverse (100 μM Stock)</td>
<td>2.5 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>380 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>400 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C with no time limit. The final concentration of each primer in the qPCR reaction should be 250 nM.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of genomic DNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>Isolation of the genomic DNA from the adipose tissue depots</p>
<ol>
<li>Add two stainless steel metal beads per sample to the 2 mL Eppendorf tube.</li>
<li>Add 1 mL of NaOH (50 mM) to the samples.</li>
<li>Lyse the samples for 5–10 min with the TissueLyser LT (Qiagen). Carefully check that the tissue is fully lysed. If, not repeat the lysis for another 5 min.</li>
<li>Shake the samples in a thermoshaker for 1 h at 1000 rpm and 92°C.</li>
<li>Give the tubes a quick spin to remove any condensation from the lid. It is not necessary to let the tubes cool down to RT first.</li>
<li>Add 250 μL of Tris-HCl (1 M, pH 8) to the samples to neutralize the pH.</li>
<li>Centrifuge the samples at 13000 rcf for 5 min at 4°C.
<ol type="a">
<li>A fat cake will emerge on top of an aqueous phase and a pellet will be seen in the bottom of the tube [<a href="#fig4">Figure 4</a>].
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Three layers of the sample after centrifugation</div>
<p>On top of the sample, there will be a layer of fat followed by an aqueous phase and the beads and a pellet in the bottom of the tube. Carefully move through the layer of fat and transfer part of the aqueous phase to a fresh tube.</p>
</figcaption>
</figure>
</li>
<li>The cooling of the centrifuge helps make the fat cake more stable and eases the separation of the fat from the aqueous phase.</li>
</ol>
</li>
<li>Carefully transfer 200 μL of the aqueous phase to a fresh tube.
<ol type="a">
<li>If a lot of fat is sticking to the outside of the pipette tip, use a paper tissue to carefully remove the fat before transferring the sample to a fresh tube [<a href="#fig5">Figure 5</a>].
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Fat sticking to the outside of the pipet tip</div>
<p>If fat is sticking to the outside of the pipet tip, carefully remove the fat with a paper tissue before transferring the aqueous phase to a fresh tube.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Centrifuge the samples again at 13000 rcf for 5 min at 4°C. <a href="#troubleshooting">Troubleshooting 2</a>
</li>
<li>To obtain a 1:30 dilution of the genomic DNA, carefully transfer 2 μL of the aqueous phase from the samples to 58 μL of H<sub>2</sub>O.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is highly critical to remove all lipids from the sample before continuing with the next step.</div>
<div class="note">
<span class="note-title">Note:</span> Other tissue lysing machines or methods e.g., the MagNA lyser (Roche), the BeadBug (Benchmark Scientific) or other hand held grinders can be used alternatively.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The samples can be stored at −20°C until further processing.</div>
<h3 id="sec3.2">Preparation of the standard curve</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0–1 h</div>
<p>To determine the exact number of recombined LoxPRed or ApoB molecules, a standard curve generated from a synthesized plasmid pUC57recloxPRed-ApoB needs to be prepared [<a href="#tbl1">Table 1</a>].</p>
<table id="tbl1">
<caption>Table 1. recloxPRed-ApoB sequence of the pUC57 plasmid</caption>
<thead></thead>
<tbody>
<tr>
<td>CTTGAAGCGCATGAACTCTTTGATGACGTCCTCGGAGGAGGCCAGCATGGATCCAGCGCTAGCTTGGCTGGACGTAAACTCCTCTTCAGACCTAATAACTTCGTATAGCATACATTATACGAAGTTATGCGGCCGACCGGTAAGCTTATCGATACCGTCGATCCCCACTGGAAAGACCGCGAAGAGTTTGTCCTCAACCGCGAGCTGTGGAGGTGGGGTCCAGGTTGAATCACGGGTTCTTCAGCACAATGCACAGTTCTCCAATGACCAAGAAGAAATACGGCTTGACCTTGCAGGATCCTTAGACGGAGCCGGAGAACCTGCGTGCAATCCATCTTGTTCAATGGCCGATCCCATGGCGGCACAGATGAATTCTTAATAACTTCGTATAGCATACATTATACGAAGTTATGCGGCCGACCGGTAAGCTTATCGATACCGTCGATCCCCACTGGAAAGACCGCGAAGAGTTTGTCCTCAACCGC</td>
</tr>
</tbody>
</table>
<ol start="11">
<li>Prepare a standard curve from the pUC57recloxPRed-ApoB plasmid with the concentration depicted in [<a href="#tbl2">Table 2</a>]. It is recommended to test the standard curve by qPCR at this point before assaying the samples. <a href="#troubleshooting">Troubleshooting 3</a>
<table id="tbl2">
<caption>Table 2. pUC57recloxPRed-ApoB plasmid standard curve</caption>
<thead>
<tr>
<th>Concentration [μg/μL]</th>
</tr>
</thead>
<tbody>
<tr>
<td>2.2 × 10<sup>7</sup>
</td>
</tr>
<tr>
<td>1.1 × 10<sup>7</sup>
</td>
</tr>
<tr>
<td>5.5 × 10<sup>8</sup>
</td>
</tr>
<tr>
<td>2.2 × 10<sup>8</sup>
</td>
</tr>
<tr>
<td>1.1 × 10<sup>8</sup>
</td>
</tr>
<tr>
<td>5.5 × 10<sup>9</sup>
</td>
</tr>
<tr>
<td>2.2 × 10<sup>9</sup>
</td>
</tr>
<tr>
<td>1.1 × 10<sup>9</sup>
</td>
</tr>
<tr>
<td>5.5 × 10<sup>10</sup>
</td>
</tr>
<tr>
<td>2.2 × 10<sup>10</sup>
</td>
</tr>
<tr>
<td>1.1 × 10<sup>10</sup>
</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The standard curve is essential to determine the absolute numbers of adipocytes and total cells. We recommend preparing 500 μL–1000 μL of each standard, so that the full study can be completed with the same standard. Aliquots of 100 μL can be prepared and stored at −20°C. The correct cell quantification depends heavily on stable plasmid concentrations of the individual solutions of the standard curve. The precise determination of the exact plasmid concentration before the preparation of the standard curve can be achieved using a NanoDrop microvolume spectrometer.</div>
<div class="note">
<span class="note-title">Note:</span> The plasmid is available upon request from the lead contact. However, the principle of the method can be adapted either to any other vector backbone or to any other sequence of interest, if another recombination site is investigated.</div>
<h3 id="sec3.3">Determination of recombined sites by qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0–2 h</div>
<p>The number of recombined LoxPRed or ApoB molecules are quantified by qPCR using a standard curve generated from a synthesized plasmid pUC57recloxPRed-ApoB in addition to primers that specifically bind at the ApoB or recombined LoxPRed site [<a href="#fig6">Figure 6</a>].</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Schematic description of the quantitative recombination analysis</div>
<p>The primer pair depicted in the upper part binds to the ApoB locus and serves to quantify the total number of cells in a depot. The second primer pair depicted in the lower part binds to the recombined transgene at the ROSA26 locus. The primer pair only produces a PCR product when LoxP Stop LoxP casette is genetically deleted by the Cre recombinase.</p>
</figcaption>
</figure>
<ol start="12">
<li>Perform a real time PCR with the samples and the standard curve using Fast SYBR Green according to the following protocol:
<ol type="a">
<li>Reaction per well:Sample/Standard = 2 μL, Primer mix (as described in <a href="#materials-and-equipment">materials and equipment</a>) = 1 μL, H<sub>2</sub>O = 2 μL, Fast SYBR Green Mastermix = 5 μL</li>
</ol>
</li>
<li>Run the Fast SYBR Green program on your qPCR machine [<a href="#tbl3">Table 3</a>].
<table id="tbl3">
<caption>Table 3. Fast SYBR green qPCR protocol</caption>
<thead>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>95°C</td>
<td>20 s</td>
<td> </td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>10 s</td>
<td rowspan="2">40 cycles</td>
</tr>
<tr>
<td>Annealing and Extension</td>
<td>60°C</td>
<td>20 s</td>
</tr>
</tbody>
</table>
</li>
<li>When exporting the data from the qPCR machine, it is important that the thresholds of LoxPRed samples and ApoB samples are set at the same value. However, be careful to set the threshold only within the geometric phase of the amplification blot.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We established the protocol with Fast SYBR Green (ThermoScientific). However, also SYBR Green from other companies (e.g., KAPA SYBR® FAST, Roche) or other dyes/enzymes (e.g TagMan, ThermoScientific) compatible with qPCR can be used.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Expected total cell numbers as well as recombined Adipoq+ or recombined Ucp1+ cell numbers for various adipose tissue depots under various housing conditions can be found in Moser et al., (<a href="#bib5">Moser et al., 2021</a>). The cell numbers at 20°C–22°C and after three weeks of cold exposure (8°C) can be found in the table below [<a href="#tbl4">Table 4</a>].</p>
<table id="tbl4">
<caption>Table 4. Example of Ucp1+ and Adipoq+ cell numbers at 20°C–22°C and after one week of cold exposure (8°C)</caption>
<thead>
<tr>
<th rowspan="2"> </th>
<th colspan="2">20°C–22°C</th>
<th colspan="2">One week of cold exposure (8°C)</th>
</tr>
<tr>
<th>Number of Ucp1+ cells</th>
<th>Number of Adipoq+ cells</th>
<th>Number of Ucp1+ cells</th>
<th>Number of Adipoq+ cells</th>
</tr>
</thead>
<tbody>
<tr>
<td>iBAT</td>
<td>28.9 ± 7.5 M</td>
<td>25.0 ± 1.1 M</td>
<td>53.2 ± 5.2 M</td>
<td>58.1 ± 3.2 M</td>
</tr>
<tr>
<td>ingWAT</td>
<td>0.4 ± 0.1 M</td>
<td>18.1 ± 0.8 M</td>
<td>2.3 ± 0.7 M</td>
<td>n/a</td>
</tr>
<tr>
<td>eWAT</td>
<td>n/a</td>
<td>6.9 ± 0.9 M</td>
<td>n/a</td>
<td>n/a</td>
</tr>
</tbody>
</table>
<p class="table-legend">M = 10<sup>6</sup>. Data presented are SEM ± mean</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Analysis of the qPCR data to determine the number of recombined LoxPRed sites and ApoB sites. A full example and a calculation template can be found online [<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Mmc1.xlsx">Table S1</a>].</p>
<ol>
<li>Standard curve calculation:
<ol type="a">
<li>The pUC57recloxPRed-ApoB plasmid is 3200 bp and the molecular weight of one bp is 650 [g/mol]. Thus, the molecular weight of the plasmid is 2.010 × 10<sup>6</sup> [g/mol]</li>
<li>To determine the molarity [M] of your plasmid, divide the concentration of your plasmid by the molecular weight of your plasmid,e.g., 2.2 × 10 <sup>7</sup> [μg/μL] / 2.010 × 10<sup>6</sup> [g/mol] = 1.06 × 10<sup>−13</sup> [M]</li>
<li>To calculate the number of molecules in 2 μL sample, multiply the molarity by the volume of the sample and the Avogadro constante.g., 1.06 × 10<sup>−13</sup> [M] ∗ 2 × 10<sup>−6</sup> [L] ∗ 6.02214076×10<sup>23</sup>= 1.28 × 10<sup>5</sup> molecules</li>
<li>Blot the average of your CT values obtained for the standard curve against the Log10(Molecules) to draw the standard curve</li>
<li>Calculate the standard curve for both the recombined LoxPRed and ApoB and determine the slope and y-axis intercept for both the LoxPRed and ApoB [<a href="#fig7">Figure 7</a>]
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Example of the standard curves</div>
<p>(A) The standard curve generated from the pUC57recloxPRed-ApoB analyzed for ApoB.</p>
<p>(B) The standard curve generated from the pUC57recloxPRed-ApoB analyzed for LoxPRed.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ideally the slope of the standard curve should be −3.32, which indicates 100% efficiency. However, this is often difficult to achieve. If the range of the CT values is not too spread out, the removal of the highest/lowest standard point can help to bring the slope towards 100% efficiency. We generally accepted an efficiency of 93%–100%; if this was not achieved, the qPCR run was repeated. <a href="#troubleshooting">Troubleshooting 4</a>
</div>
<ol start="2">
<li>Calculation of the number ApoB sites
<ol type="a">
<li>Take the average of the CT value replicates</li>
<li>Determine the Log10(Molecules) via the standard curveLog10(Molecules ApoB) = (average CT - y-axis intercept) / axis slope</li>
<li>Determine the number of ApoB molecules in 1 μLApoB molecules in 1 μL = (10<sup>ˆ</sup>(Log10(Molecules ApoB))/2</li>
<li>To get the number of cells in 1 μL, divide the number of ApoB molecules in 1 μL by two, as each cell harbors two ApoB alleles</li>
<li>To get the total number of cells, multiply the number of cells in 1 μL by the dilution factor of 30 and the total volume the tissue was lysed in (e.g., 1250 μL)</li>
</ol>
</li>
<li>Calculation of the number recombined LoxPRed sites
<ol type="a">
<li>Take the average of the CT value replicates</li>
<li>Determine the Log10(Molecules rec.) via the standard curveLog10(Molecules rec.) = (average CT - y-axis intercept) / axis slope</li>
<li>Determine the number of recombined LoxPRed molecules in 1 μL rec. molecules in 1 μL = (10ˆ(Log10(Molecules rec.))/2</li>
<li>To get the number of recombined cells in 1 μL, divide the number of recombined LoxPRed molecules in 1 μL by two if the animal is homozygous for the LoxPRed allele. If the animal is heterozygous, do not divide the number of recombined LoxPRed molecules in 1 μL by two.</li>
<li>To get the total number of recombined cells, multiply the number of cells in 1 μL by the dilution factor of 30 and the total volume you lysed the tissue in (e.g., 1250 μL). <a href="#troubleshooting">Troubleshooting 5</a>
</li>
</ol>
</li>
<li>Determination of the percentage of recombined LoxPRed site
<ol type="a">
<li>To determine the percentage of recombined LoxPRed sites, divide the number of recombined LoxPRed sites in 1 μL by the number of all cells present in 1 μL and multiply it by 100.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol is currently limited to the analysis of recombination sites obtained by LoxPRed recombination sites. Thus, it can be only applied to mouse strains harboring the ROSA26–tdRFP construct (<a href="#bib4">Luche et al., 2007</a>). However, the principle can be applied to other mouse strains if the plasmid and the primers are adapted accordingly. The absolute cell numbers are determined by the standard curve. Therefore, variations or inaccuracies in the standard curve will drastically affect the results.</p>
<p>Further, the number of cells is greatly influenced by the tissue harvesting; thus, the tissues should ideally always be harvested in the same manner. When adhering to a clear tissue harvesting protocol, we observed no interpersonal variation among adipocyte cell numbers. Additionally, other tissue types that adhere to the tissue of interest should be carefully removed (e.g., in the case of analyzing adipose tissue depots adhering to skeletal muscles). Lastly, given that the presence of lymph nodes in the tissue depot of interest can dramatically influence the cell numbers, they should be carefully removed.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The tamoxifen is not dissolving in the oil.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Heat the solution to 37°C while shaking for 1 h e.g., on a shaker in an incubator or by taping the falcon tube onto a thermoshaker.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>There is still a lot of fat in the genomic DNA solution after the second centrifugation step.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Repeat the centrifugation a third time before preparing the 1:30 dilution of the genomic DNA.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The slope of the standard curve is not equal to −3.32.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>We recommend to not work with a serial dilution approach, because if an error is introduced in an early dilution then this error will be present in the whole series. We rather recommend to work with a series of dilutions e.g., diluting 2.2 × 10<sup>7</sup> by a 1:10 dilution to 2.2 × 10<sup>8</sup></p>
<h3 id="sec7.7">Problem 4</h3>
<p>The slope of the standard curve is not equal to −3.32 or 93%–100% efficiency is not achieved in the run with the samples.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Carefully check your duplicates of the standard curve, or eventually remove the highest/lowest points of the standard curve if the CT values of your sample nicely cluster around the middle part of the standard curve.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>The recombined LoxPRed cell numbers are extraordinarily high.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Double check the genotype of the animal. If the animal is homozygous for the LoxPRed site, then divide the recombined LoxPRed cell numbers by two, as this animal is harboring two LoxPRed alleles.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Hesselbarth, N., Pettinelli, C., Gericke, M., Berger, C., Kunath, A., Stumvoll, M., Blüher, M., and Klöting, N. (2015). Tamoxifen affects glucose and lipid metabolism parameters, causes browning of subcutaneous adipose tissue and transient body composition changes in C57BL/6NTac mice. Biochem. Biophys. Res. Commun. <i>464</i>, 724-729. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00467-6/sref1">View at publisher</a></p>
<p id="bib2">de Jong, J.M., Larsson, O., Cannon, B., and Nedergaard, J. (2015). A stringent validation of mouse adipose tissue identity markers. Am. J. Physiol. Endocrinol. Metab. <i>308</i>, E1085-E1105. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00467-6/sref2">View at publisher</a></p>
<p id="bib3">Lee, Y.H., Petkova, A.P., Konkar, A.A., and Granneman, J.G. (2015). Cellular origins of cold-induced brown adipocytes in adult mice. FASEB J. <i>29</i>, 286-299. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00467-6/sref3">View at publisher</a></p>
<p id="bib4">Luche, H., Weber, O., Nageswara Rao, T., Blum, C., and Fehling, H.J. (2007). Faithful activation of an extra-bright red fluorescent protein in “knock-in” cre-reporter mice ideally suited for lineage tracing studies. Eur. J. Immunol. <i>37</i>, 43-53. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00467-6/sref4">View at publisher</a></p>
<p id="bib5">Moser, C., Straub, L.G., Rachamin, Y., Dapito, D.H., Kulenkampff, E., Ding, L., Sun, W., Modica, S., Balaz, M., and Wolfrum, C. (2021). Quantification of adipocyte numbers following adipose tissue remodeling. Cell Rep. <i>35</i>, 109023. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00467-6/sref5">View at publisher</a></p>
<p id="bib6">Ye, R., Wang, Q.A., Tao, C., Vishvanath, L., Shao, M., McDonald, J.G., Gupta, R.K., and Scherer, P.E. (2015). Impact of tamoxifen on adipocyte lineage tracing: Inducer of adipogenesis and prolonged nuclear translocation of Cre recombinase. Mol. Metab. <i>4</i>, 771-778. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00467-6/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Christian Wolfrum, <a href="mailto:christian-wolfrum@ethz.ch">christian-wolfrum@ethz.ch</a></p>
<h4>Materials availability</h4>
<p>Plasmids and animal lines are available from the lead contact upon request.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze dataset or codes.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/949-Mmc1.xlsx">Table S1. Template for recombination calculation, related to quantification and statistical analysis steps 1–4</a></p>
<h3>Acknowledgments</h3>
<p>The authors would like to thank Manuel Klug for his great support with the animal work and Ian Mitchell for editing this manuscript. The work was supported by the <a href="https://doi.org/10.13039/501100001711">Swiss National Science Foundation</a>, Switzerland.</p>
<h3>Author contributions</h3>
<p>C.M., L.G.S., and Y.R. established the method. C.M. wrote the manuscript and generated the figures, with the help of feedback from all other authors. C.W. supervised the study.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>