<article>
<section>
<h1 id="header">Global Run-On sequencing to measure nascent transcription in <i>C. elegans</i>
</h1>
<p><time datetime="2021-12-10">Published: December 10, 2021</time></p>
<p>Piergiuseppe Quarato<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Germano Cecere<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institut Pasteur, Mechanisms of Epigenetic Inheritance, Department of Developmental and Stem Cell Biology, UMR3738, CNRS, 75724 Cedex 15 Paris, France</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:piergiuseppe.quarato@pasteur.fr">piergiuseppe.quarato@pasteur.fr</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:germano.cecere@pasteur.fr">germano.cecere@pasteur.fr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100991">10.1016/j.xpro.2021.100991</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Global Run-On sequencing (GRO-seq) is one of the most sensitive techniques to detect nascent transcription from RNA polymerase (Pol) at a genome-wide level. The protocol incorporates labeled ribonucleotides into nascent RNAs from Pol I, II, and III. We have adapted the GRO-seq protocol to the nematode <i>Caenorhabditis elegans</i> to measure transcription from embryos and adult worms. Here, we provide a detailed overview of the protocol highlighting the critical steps for generating successful libraries.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1038/s41467-021-21691-6">Quarato et al. (2021)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Isolation of nuclei from <i>C. elegans</i> adults or embryos</li>
<li>Labeling of nascent transcripts with a biotinylated nucleotide</li>
<li>Three immunoprecipitation steps allow the efficient removal of the background</li>
<li>The library preparation strategy allows further removal of the background</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1210-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Overview of the protocol</h3>
<p>GRO-seq is a powerful technique to map RNA polymerase transcription at the genome-wide level. The assay has been initially developed for human cells (<a href="#bib3">Core et al., 2008</a>) and has been updated and adapted to work in fission yeasts, <i>Drosophila melanogaster,</i> and plant cells (<a href="#bib4">Mahat et al., 2016</a>). Here we describe the specific steps for producing GRO-seq libraries from <i>Caenorhabditis elegans</i> worms and embryos. We have used this protocol to generate GRO-seq libraries from as low as 1,000 worms or 40,000 embryos (<a href="#bib1">Barucci et al., 2020</a>; <a href="#bib5">Quarato et al., 2021</a>; <a href="#bib6">Singh et al., 2021</a>).</p>
<p>The protocol can be divided into three main steps:</p>
<ol>
<li>Nuclei isolation and nuclear run-on with biotinylated-UTP</li>
<li>Enrichment of biotin-labeled nascent RNAs and library preparation</li>
<li>Library amplification and sequencing</li>
</ol>
<h3 id="sec1.2">Collection of synchronized worms and early embryos</h3>
<p>To grow synchronous populations of worms, collect embryos from gravid adults by bleaching and allow hatching for 12–20 h at 20°C to obtain synchronized L1 larvae. Seed 40,000 L1 larvae onto one 150 mm NGM plate and harvest worms at the desired developmental stage. Perform three washes with M9 buffer, remove the supernatant, and flash-freeze in dry ice. Worms can be stored at −80°C for up to a year.</p>
<p>To collect populations of early embryos, grow four 150 mm NGM plates of synchronous worms until adulthood. Carefully monitor worms using a stereomicroscope and perform bleaching shortly after they start to produce the first embryos. Immediately after bleaching, perform three washes with a cold M9 buffer, remove the supernatant and immediately freeze in dry ice to block cell divisions. Embryos can be stored at −80°C for up to a year.</p>
<div class="note">
<span class="note-title">Note:</span> we seed 150 mm NGM plates with a layer of concentrated <i>Escherichia Coli</i> culture (100<b>×</b>). For information about NGM plates and M9 buffer composition, please refer to the WormBook (<a href="http://www.wormbook.org/">http://www.wormbook.org/</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> the first embryonic divisions happen in a relatively short time. Therefore, handling time should be minimized to avoid the collection of embryos at more advanced developmental stages. Always use cold reagents after bleaching to slow down embryonic cell divisions.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If needed, embryos can be stained with DAPI and visualized using a fluorescent microscope to assess the developmental stage of the population. For more information about this step, please refer to (<a href="#bib5">Quarato et al., 2021</a>).</div>
<div class="note">
<span class="note-title">Note:</span> In principle, the protocol described for collecting early embryos can be adapted to obtain embryos at different developmental stages. Also, embryos already laid on the plate can be physically separated from gravid adults by filtration, and two populations of embryos, early- and late-embryos, can be collected. However, we have never performed GRO-seq on embryos older than the 100-cell stage.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> (1 M)</td>
<td>Sigma-Aldrich</td>
<td>Cat#21115</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9530G</td>
</tr>
<tr>
<td>Tris-HCI Buffer, pH 7.5 (1 M)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15567027</td>
</tr>
<tr>
<td>TERGITOL™ solution Type NP-40, 70% in H2O</td>
<td>Sigma-Aldrich</td>
<td>Cat#NP40S</td>
</tr>
<tr>
<td>Glycerol for molecular biology</td>
<td>Sigma-Aldrich</td>
<td>Cat#G5516</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#P2325</td>
</tr>
<tr>
<td>Tris-HCI Buffer, pH 8 (1 M)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#BP1758</td>
</tr>
<tr>
<td>EDTA (500 mM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15575</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9759</td>
</tr>
<tr>
<td>NaOH</td>
<td>Sigma-Aldrich</td>
<td>Cat#S5881</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat#T8787</td>
</tr>
<tr>
<td>TWEEN® 20</td>
<td>Sigma-Aldrich</td>
<td>Cat#P9416</td>
</tr>
<tr>
<td>KCl (1 M)</td>
<td>Sigma-Aldrich</td>
<td>Cat#60142</td>
</tr>
<tr>
<td>Ribonucleotide Solution Set</td>
<td>New England Biolabs</td>
<td>Cat#N0450</td>
</tr>
<tr>
<td>DNase/RNase-Free Distilled Water</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15667708</td>
</tr>
<tr>
<td>N-Lauroylsarcosine sodium salt (sarkosyl)</td>
<td>Sigma-Aldrich</td>
<td>Cat#L5777</td>
</tr>
<tr>
<td>Biotin-11-UTP</td>
<td>Jena Biosciences</td>
<td>Cat#NU-821-BIOX</td>
</tr>
<tr>
<td>Halt™ Protease Inhibitor Cocktail, EDTA-free (100<b>×</b>)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#78425</td>
</tr>
<tr>
<td>RiboLock RNase Inhibitor (40 U/μL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EO0381</td>
</tr>
<tr>
<td>TRI Reagent™ Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9738</td>
</tr>
<tr>
<td>TRIzol™ LS Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10296028</td>
</tr>
<tr>
<td>1-Bromo-3-chloropropane (BCP)</td>
<td>Sigma-Aldrich</td>
<td>Cat#B9673</td>
</tr>
<tr>
<td>GlycoBlue™ Coprecipitant</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9515</td>
</tr>
<tr>
<td>2-Propanol for molecular biology</td>
<td>Sigma-Aldrich</td>
<td>Cat#I9516</td>
</tr>
<tr>
<td>Ethanol, Absolute (200 Proof), Molecular Biology Grade</td>
<td>Fisher Scientific</td>
<td>Cat#BP2818100</td>
</tr>
<tr>
<td>Dynabeads™ MyOne™ Streptavidin C1</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#65001</td>
</tr>
<tr>
<td>DynaMag™-2 Magnet</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12321D</td>
</tr>
<tr>
<td>T4 Polynucleotide Kinase</td>
<td>New England Biolabs</td>
<td>Cat#M0201</td>
</tr>
<tr>
<td>Phenol/Chloroform/Isoamyl Alcohol, 25:24:1 (v/v)</td>
<td>Sigma-Aldrich</td>
<td>Cat#516726</td>
</tr>
<tr>
<td>5PRIME Phase Lock Gel™ – Heavy</td>
<td>Quantabio</td>
<td>Cat#2302830</td>
</tr>
<tr>
<td>Sodium Acetate (3 M), pH 5.5, RNase-free</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9740</td>
</tr>
<tr>
<td>T4 RNA Ligase 2, truncated KQ</td>
<td>New England Biolabs</td>
<td>Cat#M0373</td>
</tr>
<tr>
<td>Agencourt RNAClean XP beads</td>
<td>Beckman Coulter</td>
<td>Cat#A63987</td>
</tr>
<tr>
<td>T4 RNA Ligase</td>
<td>Ambion</td>
<td>Cat#AM2141</td>
</tr>
<tr>
<td>SuperScript™ IV Reverse Transcriptase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#18090010</td>
</tr>
<tr>
<td>NEBNext® Ultra™ II Q5® Master Mix</td>
<td>New England Biolabs</td>
<td>Cat#M0544</td>
</tr>
<tr>
<td>Agencourt AMPure XP beads</td>
<td>Beckman Coulter</td>
<td>Cat#A63880</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>High Sensitivity D1000 ScreenTape</td>
<td>Agilent</td>
<td>Cat#5067-5584</td>
</tr>
<tr>
<td>High Sensitivity D1000 Reagents</td>
<td>Agilent</td>
<td>Cat#5067-5585</td>
</tr>
<tr>
<td>Qubit™ dsDNA HS Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#Q32851</td>
</tr>
<tr>
<td>NextSeq 500/550 High Output Kit v2.5 (75 Cycles)</td>
<td>Illumina</td>
<td>Cat#20024906</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>3’ end adapter: 3SR adapter: /5rApp/NNNNTGGAATTCTCGGGTGCCAAGG/3ddC/</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>5’ end adapter: 5SR adapter: rGrUrUrCrArGrArGrUrUrCrUrArCrArGrUrCrCrGrArCrGrArUrCrNrNrNrN</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>Reverse transcription primer: SR_RT : GCCTTGGCACCCGAGAATTCCA</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>PCR barcoded primers</td>
<td>IDT</td>
<td>See <a href="#tbl1">Table 1</a>
</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>bcl2fastq</td>
<td><a href="https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html">https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Cutadapt</td>
<td><a href="https://cutadapt.readthedocs.io/en/stable/index.html">https://cutadapt.readthedocs.io/en/stable/index.html</a></td>
<td>N/A</td>
</tr>
<tr>
<td>bowtie2</td>
<td><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td>
<td>N/A</td>
</tr>
<tr>
<td>featureCounts</td>
<td><a href="https://sourceforge.net/projects/subread/">https://sourceforge.net/projects/subread/</a></td>
<td>N/A</td>
</tr>
<tr>
<td>DESeq2</td>
<td><a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html">https://bioconductor.org/packages/release/bioc/html/DESeq2.html</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Samtools</td>
<td><a href="http://www.htslib.org/">http://www.htslib.org/</a></td>
<td>N/A</td>
</tr>
<tr>
<td>bamCoverage</td>
<td><a href="https://deeptools.readthedocs.io/en/develop/content/tools/bamCoverage.html">https://deeptools.readthedocs.io/en/develop/content/tools/bamCoverage.html</a></td>
<td>N/A</td>
</tr>
<tr>
<td>Integrative Genomic Viewer (IGV)</td>
<td><a href="https://software.broadinstitute.org/software/igv/download">https://software.broadinstitute.org/software/igv/download</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>WHEATON® DOUNCE DURA-GRIND™Stainlesss Steel Tissue Grinder, 7 mL</td>
<td>DWK Life Sciences</td>
<td>Cat#357572</td>
</tr>
<tr>
<td>DNA LoBind® Tubes</td>
<td>Eppendorf</td>
<td>Cat#0030108051</td>
</tr>
<tr>
<td>ART™ Wide Bore Filtered Pipette Tips</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#2069G</td>
</tr>
<tr>
<td>ThermoMixer®</td>
<td>Eppendorf</td>
<td>Cat#5382000015</td>
</tr>
<tr>
<td>ThermoTop®</td>
<td>Eppendorf</td>
<td>Cat#5308000003</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="note">
<span class="note-title">Note:</span> we suggest purchasing molecular biology grade reagents for the preparation of the following buffers. However, it is possible to prepare in the lab most of the solutions needed. If so, solutions should be prepared with nuclease-free water and filter-sterilized with a 0.22 μm filter.</div>
<table id="undtbl1">
<caption>Nuclei extraction buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub> (1 M)</td>
<td>3 mM</td>
<td>150 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>2 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Tris-HCI Buffer, pH 7.5 (1 M)</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>NP-40 (10%)</td>
<td>0.25%</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>Glycerol (50%)</td>
<td>10%</td>
<td>10 mL</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>0.5 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>RNase inhibitor (40 U/μL)</td>
<td>4 U/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Protease inhibitors (100<b>×</b>)</td>
<td>1<b>×</b>
</td>
<td>500 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>38 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer without DTT, RNase inhibitor, and protease inhibitors can be stored at 4°C for up to 3 months.</p>
<table id="undtbl2">
<caption>Freezing buffer NO-Glycerol</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCI Buffer, pH 8 (1 M)</td>
<td>50 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>5 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>EDTA (500 mM)</td>
<td>0.1 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>47.24 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at 4°C for up to 3 months.</p>
<table id="undtbl3">
<caption>Freezing buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCI Buffer, pH 8 (1 M)</td>
<td>50 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Glycerol (50 %)</td>
<td>40 %</td>
<td>40 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>5 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>EDTA (500 mM)</td>
<td>0.1 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>7.24 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at 4°C for up to 3 months.</p>
<table id="undtbl4">
<caption>NRO 2<b>×</b> mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl Buffer, pH 8 (1 M)</td>
<td>12.5 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>6.25 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>DTT (100 mM)</td>
<td>1.25 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>KCl (1 M)</td>
<td>375 mM</td>
<td>30 μL</td>
</tr>
<tr>
<td>Sarkosyl (5%)</td>
<td>1.25%</td>
<td>20 μL</td>
</tr>
<tr>
<td>ATP (100 mM)</td>
<td>0.625 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>CTP (100 mM)</td>
<td>0.625 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>GTP (100 mM)</td>
<td>0.625 mM</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>RNase inhibitor (40 U/mL)</td>
<td>1 U/mL</td>
<td>2 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>24 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>80 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare immediately before use and store it at room temperature (20°C–25˚C).</p>
<div class="critical">
<span class="critical-title">Critical:</span> Sarkosyl is associated with acute inhalation toxicity and can cause skin corrosion and serious eye damage. Handle with care.</div>
<div class="critical">
<span class="critical-title">Critical:</span> always keep the NRO 2<b>×</b> mix at room temperature (20°C–25˚C). Low temperatures would cause Sarkosyl to precipitate. Allow all reagents to equilibrate at room temperature before mixing. Prepare immediately before use and visually inspect the buffer before adding to the sample.</div>
<div class="note">
<span class="note-title">Note:</span> The table shows the volumes to perform one Nuclear Run-On (NRO) reaction. Scale-up accordingly if you are performing an NRO reaction on multiple samples. NRO reaction is performed in 200 μL final volume: 100 μL nuclei suspension + 80 μL NRO 2<b>×</b> mix + 20 μL Bio-11-UTP.</div>
<table id="undtbl5">
<caption>Binding &amp; Washing Buffer 2<b>×</b>
</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl Buffer, pH 7.5 (1 M)</td>
<td>10 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>2 M</td>
<td>2 mL</td>
</tr>
<tr>
<td>EDTA (500 mM)</td>
<td>1 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>2.94 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at room temperature (20°C–25˚C) for up to 3 months.</p>
<div class="optional">
<span class="optional-title">Optional:</span> 0.04% Tween 20 can be added to the Binding &amp; Washing Buffer 2<b>×</b> to reduce non-specific binding.</div>
<table id="undtbl6">
<caption>Solution A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaOH (10 M)</td>
<td>100 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>50 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1.96 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare fresh every time you use it, keep at room temperature (20°C–25˚C), and use it over the day.</p>
<div class="critical">
<span class="critical-title">Critical:</span> NaOH can cause chemical burns. Handle with care and avoid any contact with skin and eyes.</div>
<div class="optional">
<span class="optional-title">Optional:</span> 0.02% Tween 20 can be added to Solution A to prevent beads to adhere to tube walls during washes.</div>
<table id="undtbl7">
<caption>Solution B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl (5 M)</td>
<td>100 mM</td>
<td>40 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1.96 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at room temperature (20°C–25˚C) for up to 3 months.</p>
<div class="optional">
<span class="optional-title">Optional:</span> 0.02% Tween 20 can be added to Solution B to prevent beads to adhere to tube walls during washes.</div>
<table id="undtbl8">
<caption>High Salt Wash buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl (5 M)</td>
<td>2 M</td>
<td>20 mL</td>
</tr>
<tr>
<td>Tris-HCl Buffer, pH 7.5 (1 M)</td>
<td>50 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Triton X-100 (10%)</td>
<td>0.5%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at 4°C for up to 3 months.</p>
<table id="undtbl9">
<caption>Medium Salt Wash buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl (5 M)</td>
<td>300 mM</td>
<td>3 mL</td>
</tr>
<tr>
<td>Tris-HCl Buffer, pH 7.5 (1 M)</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>Triton X-100 (10%)</td>
<td>0.1%</td>
<td>500 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>46 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at 4°C for up to 3 months.</p>
<table id="undtbl10">
<caption>Low Salt Wash buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl Buffer, pH 7.5 (1 M)</td>
<td>5 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>Triton X-100 (10%)</td>
<td>0.1%</td>
<td>500 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>49.25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Buffer can be stored at 4°C for up to 3 months.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<div class="critical">
<span class="critical-title">Critical:</span> all steps of the protocol are performed at 4°C unless stated differently. Pre-chill all reagents and equipment before use. Nuclease-free reagents need to be used in all steps of the protocol.</div>
<h3 id="sec3.1">Nuclei isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>The following steps describe the procedure to isolate intact nuclei from worms or embryos. The protocol has been tested with 1,000 to 40,000 worms and 40,000 to 300,000 early embryos.</p>
<ol>
<li>Resuspend worms or embryos in 1.5 mL of cold Nuclei extraction buffer (NEB) and transfer to a pre-chilled stainless steel tissue grinder on ice.</li>
<li>Lyse worms to release nuclei by applying 40 dounce strokes.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> take a small aliquot of the lysate (10–20 μL) and check that the worms are lysed using a stereomicroscope. Intact bodies and embryos should be absent. If intact corpses are still present, increase the number of dounce strokes.</div>
<ol start="3">
<li>Transfer the lysate to a pre-chilled 1.5 mL microcentrifuge tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Multiple samples can be processed simultaneously. If so, keep lysate from step 3 on ice while lysing the other samples. In this case, wash the metal dounce well between each sample with milliQ water. After processing all the samples, proceed immediately to step 4.</div>
<ol start="4">
<li>Remove debris by centrifuging at 100×<i>g</i> for 4 min at 4°C.</li>
<li>Transfer the supernatant to a new 1.5 mL low binding microcentrifuge tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> carefully collect the supernatant without disturbing the debris pellet. If necessary, repeat steps 4 and 5 until lysate is clear from debris. A residual 50–100 μL can be left in the tube to avoid debris carryover.</div>
<ol start="6">
<li>Pellet nuclei by centrifuging at 1,000<b>×</b><i>g</i> for 4 min at 4°C.</li>
<li>Remove supernatant and wash nuclei with 1 mL NEB.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> wash nuclei by pipetting up and down several times using a wide-bore tip. Gently pipette nuclei to preserve their integrity.</div>
<ol start="8">
<li>Pellet nuclei by centrifuging at 1,000<b>×</b><i>g</i> for 4 min at 4°C.</li>
<li>Repeat steps 7 and 8 three more times for a total of 4 washes with NEB.</li>
<li>Wash nuclei in 1 mL Freezing buffer NO-Glycerol.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> wash nuclei by pipetting up and down several times using a wide-bore tip. Gently pipette nuclei to preserve their integrity.</div>
<ol start="11">
<li>Pellet nuclei by centrifuging at 1,000<b>×</b><i>g</i> for 4 min at 4°C and discard the supernatant.</li>
<li>Resuspend nuclei in 100 μL Freezing buffer.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> nuclei in the Freezing buffer can be stored at −80°C for up to 1 year.</div>
<h3 id="sec3.2">Nuclear run-on (NRO) reaction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>NRO is performed on isolated nuclei in the presence of Bio-11-UTP. Sarkosyl is added to prevent <i>de novo</i> assembly of the pre-initiation complex and avoid new transcriptional initiation. A negative NRO control reaction can be added at this step. In the negative NRO reaction, UTP is used instead of Biotinylated-UTP. If you are performing the experiment for the first time, the addition of negative control is strongly recommended.</p>
<ol start="13">
<li>Prepare the NRO 2x mix (the recipe is listed in the “<a href="#materials-and-equipment">Materials and equipment</a>” section), and preheat it at 30°C before use.</li>
<li>Dispense 80 μL of the NRO 2<b>×</b> mix in a new low binding microcentrifuge tube.</li>
<li>Add 20 μL of Bio-11-UTP 10 mM to the NRO 2<b>×</b> mix (final Bio-11-UTP concentration: 1 mM) and gently mix.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> if running a negative NRO reaction, replace Bio-11-UTP with 0.5 μL of UTP 100 mM and 19.5 μL of nuclease-free water.</div>
<ol start="16">
<li>Add 100 μL of lysed nuclei from the previous step 12. Gently mix the reaction well using a wide bore tip.</li>
<li>Incubate the reaction for 3 min at 30°C.</li>
<li>Transfer the tubes on ice and add 50 μL ice-cold nuclease-free water and 750 μL TRIzol LS reagent and mix well to stop the reaction.</li>
<li>Incubate nuclei in TRIzol for 5 min at room temperature (20°C–25˚C) to permit complete dissociation of the nucleoprotein complexes.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> TRIzol has an acute oral and dermal toxicity. Always handle in a chemical hood. Contact with eyes and skin should be avoided.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Nuclei in TRIzol can be stored at −80°C for up to a week (we have not tested longer storage).</div>
<h3 id="sec3.3">RNA isolation using TRIzol</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h 15 min</div>
<p>Total RNA, including biotinylated nascent RNA, is isolated using TRIzol.</p>
<ol start="20">
<li>Add 100 μL of 1-Bromo-3-chloropropane (BCP) and mix it well by vortexing.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> BCP is toxic upon inhalation. Always handle in a chemical hood.</div>
<ol start="21">
<li>Incubate at RT for 15 min.</li>
<li>Centrifuge at 12,000<b>×</b><i>g</i> for 15 min at 4°C, then transfer the aqueous phase (upper phase) containing the RNA to a new low binding microcentrifuge tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Avoid transferring any of the interphase or organic layer into the pipette when removing the aqueous phase. The tube can be angled at 45° to facilitate the process. Approximately 450 μL of aqueous phase can be collected at this step.</div>
<ol start="23">
<li>Add 2 μL of GlycoBlue and mix it well by vortexing for 5–10 s.</li>
<li>Add 500 μL of isopropanol and mix it vigorously by vortexing for 10–15 s.</li>
<li>Incubate at RT for 15 min to precipitate the RNA.</li>
<li>Centrifuge at 12,000<b>×</b><i>g</i> for 15 min at 4°C to pellet the RNA.</li>
<li>Remove the supernatant and add 1 mL of 75 % Ethanol to the RNA pellet.</li>
<li>Centrifuge at 7,500<b>×</b><i>g</i> for 5 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After centrifugation, a blue pellet should be visible at the bottom of the tube.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Carefully remove the supernatant without disturbing the RNA pellet.</div>
<ol start="29">
<li>Remove the supernatant and air dry the pellet for 1–2 min.
<div class="note">
<span class="note-title">Note:</span> to completely remove the supernatant without disturbing the pellet you can use the following procedure:</div>
<ol type="a">
<li>Remove the supernatant using a p1000 micropipette.</li>
<li>Briefly centrifuge to collect all residual supernatant at the bottom of the tube. At this point, the pellet should be on the side of the tube.</li>
<li>Using a p200 micropipette, gently push at the bottom-center of the tube and remove all residual ethanol. Always check that the pellet is not entering the tip.</li>
<li>Briefly air dry the pellet.
<div class="critical">
<span class="critical-title">Critical:</span> do not over-dry the RNA pellet as this would make it difficult to resuspend it.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">RNA fragmentation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8 min</div>
<ol start="30">
<li>Resuspend the RNA pellet from the previous step 29 in 8 μL water.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> RNA can be stored at −80°C for up to a week.</div>
<div class="note">
<span class="note-title">Note:</span> The analysis of RNA quality/integrity is not required at this step. This is because degradation fragments cannot be cloned with the library preparation strategy described in this protocol. Only RNA molecules with 3′-hydroxyl biotinylated ends will be suitable for cloning. In addition, the RNA quality control methods will primarily detect mature mRNAs and not Polymerase-bound nascent RNAs, which are much less abundant.</div>
<ol start="31">
<li>Add 2 μL of 5<b>×</b> reverse transcriptase buffer (provided with SuperScript™ IV Reverse Transcriptase) and mix it well by pipetting or vortexing. Spin down the samples using a benchtop centrifuge.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> RNA fragmentation can be performed using different buffers, including commercially available RNA fragmentation reagents, based on MgCl<sub>2</sub> or ZnCl<sub>2</sub>. If using a different buffer, check that the final library size is correct and adjust the fragmentation time accordingly.</div>
<ol start="32">
<li>Incubate the samples at 95°C for 7 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> incubation can be performed in a ThermoMixer equipped with Thermo Top. Alternatively, the sample can be transferred to a PCR tube and incubated in a thermocycler with the lid temperature set at 105°C. During the fragmentation time, it is possible to start preparing the Dynabeads magnetic beads to isolate biotinylated molecules.</div>
<ol start="33">
<li>Immediately block fragmentation by shifting the samples on ice.</li>
<li>Add 50 μL of ice-cold nuclease-free water and keep on ice for at least 1 min before proceeding to the next step.</li>
</ol>
<h3 id="sec3.5">Preparation of dynabeads MyOne streptavidin C1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>Dynabeads are washed following the <a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https://assets.thermofisher.com/TFS-Assets%2FLSG%2Fmanuals%2Fdynabeads_myone_savC1_man.pdf">manufacturer’s instructions</a> before binding to biotinylated RNA.</p>
<div class="critical">
<span class="critical-title">Critical:</span> This step is crucial since the beads are not supplied in RNase-free solutions.</div>
<div class="note">
<span class="note-title">Note:</span> all the washing steps are performed at room temperature (20°C–25˚C) unless stated differently.</div>
<ol start="35">
<li>Resuspend the Dynabeads™ magne/tic beads in the vial (i.e., vortex for &gt;30 s, or tilt and rotate for 5 min).</li>
<li>Transfer 30 μL of Dynabeads™ magnetic beads per sample to a low binding microcentrifuge tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 30 μL of beads are sufficient for a sample of 1,000 to 40,000 adult worms or 40,000 to 300,000 embryos. If performing the experiment with a different number of worms/embryos, the volume of beads should be scaled accordingly.</div>
<ol start="37">
<li>Add 1 mL of Binding &amp; Washing Buffer 1<b>×</b> and resuspend by pipetting or gentle vortexing for 5–10 s. Then, briefly centrifuge the tube using a benchtop centrifuge.</li>
<li>Place the tube on a magnet for 1 mi/n or until the solution appears clear and discard the supernatant.</li>
<li>Remove the tube from the magnet and resuspend the magnetic beads in a Binding &amp; Washing Buffer volume 1<b>×</b> equal to the initial volume of beads taken from the vial at step 36. Resuspend beads by pipetting or gentle vortexing for 5–10 s. Then, briefly centrifuge the tube using a benchtop centrifuge.</li>
<li>Place the tube on a magnetic stand for 1 min or until the solution appears clear and discard the supernatant.</li>
<li>Repeat steps 39–40 twice for a total of 3 washes.</li>
<li>Wash the beads twice in Solution A for 2 min. Use a volume of Solution A equal to or larger than the initial volume of Dynabeads™ magnetic beads initially taken from the vial.</li>
<li>Wash the beads once in Solution B. Use a volume of Solution B equal to the volume used for Solution A.</li>
<li>Resuspend the beads in 60 μL per sample of 2<b>×</b> Binding &amp; Washing Buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The washed beads can be prepared in advance and stored at 4°C for 1 day.</div>
<h3 id="sec3.6">1<sup>st</sup> enrichment of biotinylated RNA molecules</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 50 min</div>
<p>Biotinylated nascent RNAs are enriched by binding to Dynabeads™ magnetic beads. After binding, the beads coated with the biotinylated molecules are washed using high salt buffers to efficiently remove non-biotinylated molecules.</p>
<ol start="45">
<li>Add 60 μL of washed beads from previous step 44 to 60 μL of fragmented RNA from previous step 34.</li>
<li>Incubate for 30 min at 25°C with 1200 rpm agitation (15’’ ON 45” OFF) in a ThermoMixer.</li>
<li>Briefly spin the tubes using a benchtop centrifuge and place on a magnetic rack for 1 min or until the solution appears clear.</li>
<li>Remove the supernatant and add 1 mL of High salt buffer. Incubate on rotation for 5 min at 4°C.</li>
<li>Briefly spin the tubes using a benchtop centrifuge and place on a magnet for 1 min or until the solution appears clear.</li>
<li>Remove the supernatant and add 1 mL of Medium salt buffer. Incubate on rotation for 5 min at 4°C.</li>
<li>Briefly spin the tubes using a benchtop centrifuge and place on a magnet for 1 min or until the solution appears clear.</li>
<li>Remove the supernatant and add 1 mL of Low salt buffer. Incubate on rotation for 5 min at 4°C.</li>
<li>Briefly spin the tubes using a benchtop centrifuge and place on a magnet for 1 min or until the solution appears clear.</li>
<li>Remove the supernatant, add 1mL TRI Reagent solution, and proceed to RNA isolation as previously described (Refer to “RNA isolation using TRIzol”, steps 20–29).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Beads in TRIzol can be stored at −80°C for up to a week.</div>
<h3 id="sec3.7">RNA 5′ phosphorylation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 35 min</div>
<p>The RNA fragmentation step produces fragmented RNA molecules with 3′-phosphate and 5′-hydroxyl termini, not suitable for the adapter ligation steps, which require a 3′-hydroxyl and a 5′-phosphate terminus. Therefore, with the following steps, 5′-hydroxyl ends are phosphorylated using the Polynucleotide kinase.</p>
<div class="note">
<span class="note-title">Note:</span> Since incorporating Bio-11-UTP stall RNA polymerases, all nascent RNAs should terminate with a 3′-hydroxyl biotinylated end. Therefore, RNA molecules terminating with a 3′-phosphate end, result from the fragmentation steps and represent the background for this protocol. For this reason, 3′-phosphate ends do not need to be repaired to allow ligation of the adapter.</div>
<ol start="55">
<li>Resuspend RNA after TRIzol purification in 20 μL water.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> RNA can be stored at −80°C for up to a week.</div>
<ol start="56">
<li>Setup RNA 5′ phosphorylation reaction in 100 μL final volume by adding to the RNA from the previous step:
<ol type="a">
<li>10 μL 10<b>×</b> Polynucleotide Kinase Buffer(</li>
<li>1 μL 100 mM ATP</li>
<li>0.5 μL 40 U/μL RNase inhibitor</li>
<li>2μL 100 U/μL T4 Polynucleotide Kinase</li>
<li>66.5 μL water.</li>
</ol>
</li>
<li>Incubate the reaction at 37°C for 30 min.</li>
<li>Add 200 μL of ice-cold nuclease-free water, keep the sample on ice and proceed to the next step.</li>
</ol>
<h3 id="sec3.8">RNA purification using Phenol:Chloroform</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="59">
<li>Spin “5PRIME Phase Lock Gel™ – Heavy” tubes for 2 min at 12,000<b>×</b><i>g</i> at room temperature (20°C–25˚C) to pellet the gel.</li>
<li>Add the solution from the previous step 58 to a pre-spun Phase Lock tube.</li>
<li>Add 1 volume (300 μL) of Phenol/Chloroform/Isoamyl Alcohol and mix thoroughly by inverting 10–15 times.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> do not vortex the tube to avoid disturbing the Phase Lock gel.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Phenol:Chloroform is toxic upon inhalation. Always handle in a chemical hood. Contact with eyes and skin should be avoided.</div>
<ol start="62">
<li>Centrifuge at 12,000<b>×</b><i>g</i> for 5 min at room temperature (20°C–25˚C) and transfer the aqueous phase containing the RNA (upper phase) to a fresh low binding microcentrifuge tube.</li>
<li>Add 30 μL of 3 M sodium acetate and 2 μL of GlycoBlue and vortex for 5–10 s.</li>
<li>Add 900 μL ethanol and vortex vigorously for 10–15 s.</li>
<li>Precipitate the RNA by incubating for 1h at −20°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> RNA can be precipitated overnight or over-weekend.</div>
<ol start="66">
<li>Centrifuge to pellet the RNA at 12,000<b>×</b><i>g</i> for 30 min at 4°C.</li>
<li>Remove the supernatant and add 1 mL of 75 % Ethanol to the RNA pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After centrifugation, a blue pellet should be visible at the bottom of the tube.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Carefully remove the supernatant without disturbing the RNA pellet.</div>
<ol start="68">
<li>Centrifuge at 12,000<b>×</b><i>g</i> for 5 min at 4°C.</li>
<li>Remove the supernatant and air dry the pellet for 1–2 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> do not over-dry the RNA pellet as this would make it difficult to resuspend it.</div>
<ol start="70">
<li>Resuspend the RNA in 5 μL water.</li>
</ol>
<h3 id="sec3.9">3′ adapter ligation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 16 h</div>
<p>Biotinylated RNA molecules are ligated to the 5′ pre-adenylated DNA adapter using an optimized version of the T4 RNA ligase. This strategy reduces the formation of ligation artifacts such as concatemers or circles.</p>
<ol start="71">
<li>Perform adapter ligation in 20 μL final volume by adding to the RNA from the previous step:
<ol type="a">
<li>1 μL 10 μM 3SR adapter</li>
<li>2 μL 10<b>×</b> Ligation Buffer</li>
<li>2 μL “T4 RNA Ligase 2, truncated KQ”</li>
<li>10 μL 50% PEG8000</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> PEG8000 is very viscous. Carefully pipette the right amount of PEG in the solution and mix well by carefully pipetting up and down at least 20 times or vortexing. Visually inspect that the solution is well mixed.</div>
<ol start="72">
<li>Incubate the reaction for 16 h at 16°C in a ThermoMixer equipped with a Thermo top.</li>
</ol>
<h3 id="sec3.10">RNA purification using agencourt RNAClean XP beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 40 min</div>
<p>SPRI (Solid Phase Reversible Immobilization) beads are used to purify the RNA from the ligation reaction. No size selection is performed at this step.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> equivalent SPRI beads can be used at this step instead of Agencourt RNAClean XP beads.</div>
<ol start="73">
<li>Add 3 volumes (60 μL) of isopropanol to the ligation reaction from the previous step.</li>
<li>Add 1.8 volumes (36 μL) of Agencourt RNAClean XP beads.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> beads are very viscous. Mix well before use and carefully pipette the right volume of beads.</div>
<ol start="75">
<li>Mix well by pipetting up and down at least 15 times or by gentle vortexing and incubate for 20 min on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> during this time, it is possible to start washing Dynabeads (steps 35–44) for the 2<sup>nd</sup> enrichment of biotinylated RNA molecules.</div>
<ol start="76">
<li>Briefly spin and place the tube onto a magnetic tube rack for 5 min to separate the beads from the solution.</li>
<li>Slowly pipette the cleared solution from the tube and discard.</li>
<li>While on the magnetic rack, dispense 200 μL of 75% ethanol into the tube, incubate for 30 s and discard the ethanol.</li>
<li>Repeat the previous step for a total of 2 washes.</li>
<li>Remove the tube from the magnetic rack and briefly spin down residual ethanol using a benchtop centrifuge.</li>
<li>Place the tube onto a magnetic tube rack and remove residual ethanol.</li>
<li>Let the beads air dry for 5 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Over-drying the sample may result in a lower recovery.</div>
<ol start="83">
<li>Remove tubes from the rack and elute purified RNA from the beads by adding 61 μL water.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> manually resuspend the beads by pipetting up and down several times. Expel the elution buffer down the side of the tube to ensure the entire bead mass comes into contact with the buffer. Incubate for 30 s before proceeding to the next step.</div>
<ol start="84">
<li>Place the tubes onto a magnetic tube rack for 1 min to separate the beads from the solution. Then, slowly collect 60 μL RNA solution and transfer to a fresh tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> slowly collect the solution while keeping the tubes onto the magnetic rack to avoid beads' carryover. 1 μL of excess water is added at step 83 is added to avoid beads' carryover.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> RNA can be stored at −80°C for up to a week.</div>
<h3 id="sec3.11">2<sup>nd</sup> enrichment of biotinylated RNA molecules</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="85">
<li>Heat purified RNA at 65°C for 5 min to denature secondary structures and place on ice for 1 min.</li>
<li>Perform a 2<sup>nd</sup> enrichment of biotinylated RNA molecules by repeating steps 45–54.</li>
</ol>
<h3 id="sec3.12">5′ adapter ligation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="87">
<li>Resuspend RNA after trizol purification in 15 μL water.</li>
<li>Perform 5′ end ligation in a total volume of 20 μL by adding to RNA from the previous step:
<ol type="a">
<li>1 μL 5SR adapter 10 μM</li>
<li>2 μL Ligation buffer 10<b>×</b>
</li>
<li>2 μL T4 RNA Ligase</li>
</ol>
</li>
<li>Mix well and incubate the reaction at 25°C for 2 h.</li>
<li>Proceed to RNA purification using Agencourt RNAClean XP beads as described in steps 73–84.</li>
</ol>
<h3 id="sec3.13">3<sup>rd</sup> enrichment of biotinylated RNA molecules</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="91">
<li>Heat purified RNA (60 μL) at 65°C for 5 min to denature secondary structures and place on ice for 1 min.</li>
<li>Proceed to the 3<sup>rd</sup> enrichment of biotinylated molecules by repeating steps 45–54.</li>
</ol>
<h3 id="sec3.14">Reverse transcription</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h 20 min</div>
<p>Ligated RNA molecules are reverse transcribed to produce complementary DNA molecules suitable for library amplification.</p>
<ol start="93">
<li>Resuspend RNA after TRIzol purification in 11 μL of water.</li>
<li>Perform reverse transcription reaction in a final volume of 20 μL by adding to RNA from the previous step:
<ol type="a">
<li>1 μL SR_RT primer 50 mM</li>
<li>1 μL dNTP mix 10 mM</li>
<li>4 μL SSIV Buffer 5<b>×</b>
</li>
<li>1 μL DTT 100 mM</li>
<li>1 μL RNase inhibitor</li>
<li>1 μL SuperScript IV reverse transcriptase</li>
</ol>
</li>
<li>Mix well and incubate the reaction at 50°C for 1 h in a thermal cycler with the lid temperature set at 85°C.</li>
<li>Inactivate the reaction by heating the sample at 80°C for 10 min.</li>
<li>Keep the sample at 4°C until ready to proceed to the next step.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> reverse-transcribed cDNA can be stored at −20°C for up to a month.</div>
<h3 id="sec3.15">Test PCR amplification of libraries</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min</div>
<p>After reverse transcription, libraries are amplified by PCR using a universal primer and a barcoded primer to allow sample multiplexing. Primer sequences are listed in <a href="#tbl1">Table 1</a>.</p>
<table id="tbl1">
<caption>Table 1. Primers for library amplification</caption>
<thead>
<tr>
<th>Primer name</th>
<th>Primer sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>SR universal primer</td>
<td>AATGATACGGCGACCACCGAGATCTACACGTTCAGAGTTCTACAGTCCG∗A</td>
</tr>
<tr>
<td>SR barcode primer 1</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>CGTGAT</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 2</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>ACATCG</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 3</td>
<td>AAGCAGAAGACGGCATACGAGATG<b>CCTAA</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 4</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>TGGTCA</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 5</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>CACTGT</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 6</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>ATTGGC</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 7</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>GATCTG</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 8</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>TCAAGT</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 9</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>CTGATC</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 10</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>AAGCTA</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 11</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>GTAGCC</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
<tr>
<td>SR barcode primer 12</td>
<td>CAAGCAGAAGACGGCATACGAGAT<b>TACAAG</b>GTGACTGGAGTTCCTTGGCACCCGAGAATTCC∗A</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗ = Phosphorothioate bond.</p>
<p>The number of amplification cycles required to amplify the libraries is influenced by many factors and needs to be determined experimentally for each sample (for example, very early embryos have lower transcription levels than late embryos). Therefore, we suggest performing a test PCR amplification of the libraries using ¼ of the reverse transcription reaction.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The number of PCR cycles should be minimized to avoid overamplification, which increases PCR artifacts. Please refer to <a href="#fig1">Figure 1</a> for an example of a correctly amplified library.</div>
<div class="note">
<span class="note-title">Note:</span> if multiple samples need to be sequenced together, use a different SR barcode primer per sample. The sequence of the SR barcode primers is available in <a href="#tbl1">Table 1</a>.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1210-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Tapestation analysis showing a proper library preparation result</div>
<p>Example electropherograms from Tapestation analysis of GRO-seq libraries using 1,000 worms (A), 100,000 embryos (B), and 100,000 embryos after a negative NRO control reaction using UTP instead of Bio-11-UTP (C). No residual primers (&lt; 100 bp) nor overamplification products (&gt; 600 bp) are visible. The peak height is in the same range as the lower and upper markers for Figures (A and B). Number of PCR cycles used to amplify each library is indicated in the figure.</p>
</figcaption>
</figure>
<ol start="98">
<li>Setup PCR reaction in a final volume of 50 μL by combining:
<ol type="a">
<li>17 μL water</li>
<li>5 μL reverse-transcribed cDNA library</li>
<li>1.5 μL 10 μM SR universal primer</li>
<li>1.5 μL 10 μM SR barcode primer 10</li>
<li>25 μL NEBNext® Ultra™ II Q5® Master Mix 2<b>×</b>
</li>
</ol>
</li>
<li>Place the tube on a thermocycler with the heated lid set to 105°C and perform PCR amplification using the following PCR cycling conditions:</li>
</ol>
<table id="undtbl11">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>15 s</td>
<td rowspan="3">16–22 cycles∗</td>
</tr>
<tr>
<td>Annealing</td>
<td>61°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>45 s</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> We generally perform 20 PCR cycles for the test PCR for both adults and embryos.</div>
<h3 id="sec3.16">Purification of the PCR reaction using agencourt AMPure XP beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>AMPure SPRI beads are used to remove salts, dNTP, primers, and primer dimers from the PCR reaction. 1.8 volumes of AMPure beads are used to recover fragments &gt; 100 bp (expected size of amplicons 140–350 bp).</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> equivalent SPRI beads can be used at this step instead of Agencourt AMPure XP beads. If different beads are used, the beads/sample ratio might vary. Refer to manufacturer guidelines to select the proper beads/sample ratio to purify fragments &gt; 100 bp.</div>
<ol start="100">
<li>Add 90 μL AMPure XP beads to 50 μL PCR reaction.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> beads are very viscous. Mix well before use and carefully pipette the right volume of beads.</div>
<ol start="101">
<li>Mix well by pipetting up and down at least 15 times or by gentle vortexing and incubate for 5 min at RT.</li>
<li>Briefly spin using a benchtop centrifuge and place the tube onto a magnetic tube rack for 5 min to separate the beads from the solution.</li>
<li>Slowly pipette the cleared solution from the tube and discard.</li>
<li>While on the magnetic rack, dispense 200 μL of 75% ethanol into the tube, incubate for 30 s and remove the Ethanol.</li>
<li>Repeat the previous step for a total of 2 washes.</li>
<li>Remove the tube from the magnetic rack and briefly spin down residual ethanol using a benchtop centrifuge.</li>
<li>Place the tube onto a magnetic tube rack and remove residual ethanol.</li>
<li>Let the beads air fry for 5 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Over-drying the sample may result in a lower recovery.</div>
<ol start="109">
<li>Remove the tube from the rack and elute purified DNA from the beads by adding 18 μL water.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> manually resuspend the beads by pipetting up and down several times. Expel the elution buffer down the side of the tube to ensure the entire bead mass comes into contact with the buffer. Incubate for 30 s before proceeding to the next step.</div>
<ol start="110">
<li>Place the tubes onto a magnetic tube rack for 1 min to separate the beads from the solution. Then, slowly collect 17 μL of cleaned DNA libraries and transfer them to a fresh tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> slowly pipette the solution while keeping the tubes onto the magnetic rack to avoid beads' carryover 1 μL of excess water is added at step 110 to prevent beads carryover.</div>
<h3 id="sec3.17">Quality control of purified DNA libraries and full-scale PCR amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>Purified DNA libraries are run on an Agilent Tapestation instrument using the High Sensitivity D1000 reagents to check for library size, purity, and amount.</p>
<ol start="111">
<li>Run 2 μL purified DNA libraries on an Agilent Tapestation instrument using the High Sensitivity D1000 reagents following <a href="https://www.agilent.com/cs/library/usermanuals/Public/ScreenTape_HSD1000_QG.pdf">manufacturer conditions</a>.</li>
<li>Examine the result of the Tapestation run and determine the right PCR cycles for the full-scale PCR amplification. An example of Tapestation results for both worms and embryos can be found in the <a href="#expected-outcomes">expected outcomes</a> section and <a href="#fig1">Figure 1</a>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Adjust the PCR cycles of the final PCR amplification, calculating that each library should be in the same amount range of the upper and lower markers. Check for the presence of over-amplification products (high molecular size products) and reduce the number of PCR cycles to eliminate them (See <a href="#expected-outcomes">expected outcomes</a>, <a href="#troubleshooting">troubleshooting</a> section, and <a href="#fig3">Figure 3</a>).</div>
<ol start="113">
<li>Setup a full-scale PCR reaction in a final volume of 50 μL by combining:
<ol type="a">
<li>7 μL water</li>
<li>15 μL reverse-transcribed cDNA library (from step 97)</li>
<li>1.5 μL SR universal primer 10 μM</li>
<li>1.5 μL SR barcode primer 10 μM</li>
<li>25 μL NEBNext® Ultra™ II Q5® Master Mix 2<b>×</b>
</li>
</ol>
</li>
<li>Place the tube on a thermocycler with the heated lid set to 105°C and perform PCR amplification using the following PCR cycling conditions:</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>15 s</td>
<td rowspan="3">Determined at step 112</td>
</tr>
<tr>
<td>Annealing</td>
<td>61°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>45 min</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">forever</td>
</tr>
</tbody>
</table>
<ol start="115">
<li>Purify the PCR reaction by repeating steps 100–110.</li>
<li>Run 2 μL of purified DNA libraries on an Agilent Tapestation instrument using the High Sensitivity D1000 reagents following <a href="https://www.agilent.com/cs/library/usermanuals/Public/ScreenTape_HSD1000_QG.pdf">manufacturer conditions</a>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This step is crucial to assess the amount of each DNA library, the amplification status, and the absence of residual PCR primers before quantifying and sequencing. An example of TapeStation results for both worms and embryos can be found in the section <a href="#expected-outcomes">expected outcomes</a> and <a href="#fig1">Figure 1</a>.</div>
<h3 id="sec3.18">Quantification of DNA libraries and pooling for next-generation sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>In the following steps, libraries are precisely quantified and combined in an equimolar ratio based on the size distribution and the concentration.</p>
<ol start="117">
<li>Quantify libraries using the Qubit™ dsDNA HS Assay Kit Always measure Qubit standard solutions before quantifying the DNA libraries.
<ol type="a">
<li>Prepare 200 μL of working solution for each sample by diluting the Qubit® dsDNA HS Reagent 1:200 in Qubit® dsDNA HS Buffer.</li>
<li>To quantify the standard solutions transfer 190 μL working solution and 10 μL standard solution into a new Qubit 0.5 mL tube (for both standards #1 and #2).</li>
<li>To quantify the libraries, transfer 198 μL of working solution and 2 μL of sample into a new Qubit 0.5 mL tube.</li>
<li>Mix well, briefly spin using a benchtop centrifuge, and incubate for 2 min at RT.</li>
<li>Read sample concentration by selecting the dsDNA High Sensitivity as the assay type on the Qubit instrument.</li>
</ol>
</li>
<li>Combine the average peak size (step 116) and the concentration value to calculate the molarity of each library. Pool all libraries that have to be sequenced together in an equimolar ratio.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Library pooling can be performed by using the “Illumina Pooling Calculator” at <a href="https://support.illumina.com/help/pooling-calculator/pooling-calculator.htm">https://support.illumina.com/help/pooling-calculator/pooling-calculator.htm</a>
</div>
<h3 id="sec3.19">High-throughput sequencing of GRO-seq libraries</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12–24 h</div>
<ol start="119">
<li>Sequence pooled GRO-seq libraries using an Illumina Sequencing platform compatible with the TruSeq approach.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> we use a NextSeq-500 instrument and the NextSeq 500/550 High Output Kit v2.5 (75 Cycles) (Single-read sequencing). Generally, 20–30 million reads per sample provide enough coverage for both worms and embryos.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>After amplification, cDNA libraries are run on a Tapestation to check their quality and quantity. Expected library size range between 140–350 bp (<a href="#fig1">Figures 1</a>A and 1B). Residual PCR primers, primer-dimers, and empty products should be absent since they could negatively influence the library quantification and sequencing steps. The negative NRO control reaction should not show any library amplification product when amplified with the same number of PCR cycles as the biotin-UTP NRO reaction (<a href="#fig1">Figure 1</a>C).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The steps below describe how to analyze GRO-seq raw data after sequencing to perform differential gene expression analysis and visualize data on the “integrative genomic viewer” (IGV).</p>
<ol>
<li>Use bcl2fastq to demultiplex the samples and convert base call files (bcl) to fastq.gz files.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> NextSeq 500 flowcell has 4 different lanes. Therefore, the result of the previous step will be 4 fastq.gz per sample. We usually concatenate all files corresponding to the same sample and work with a single file per sample. Files from different lanes can also be processed separately to assess differences between sequencing lanes.</div>
<ol start="2">
<li>Remove adapter sequence from raw reads using cutadapt or any equivalent software. Adapter sequence: TGGAATTCTCGGGTGCCAAGG</li>
<li>Remove random tetramers at both 5′ and 3′ ends using cutadapt or any equivalent software.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> random tetramers at both ends can also be used for removing PCR duplicates.</div>
<div class="note">
<span class="note-title">Note:</span> To improve the mapping step, we remove reads that are too short for mapping. Therefore, we apply a size selection criteria during raw reads processing, keeping only reads of 24 bp or longer after adapter trimmings. Thus, after random tetramers removal, reads will be 16 nucleotides or longer.</div>
<ol start="4">
<li>Map the trimmed reads on <i>C. elegans</i> genome using bowtie2.</li>
<li>Count features using featureCounts or any equivalent software. This step will produce a count table for the selected feature (for example, protein-coding genes, transposons, tRNA, rRNA, etc.).</li>
<li>Compute TPM or use deseq2 software to perform differential gene expression analysis.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> to visualize genomic data, we suggest generating normalized bigwig files (<a href="#fig2">Figure 2</a>). This can be done by sorting and indexing bam files using samtools and converting them to normalized bigwig files using the bamCoverage package from deeptools.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1210-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Genome browser example of a GRO-seq library compared to RNA-seq libraries</div>
<p>A genomic view of two protein-coding genes showing normalized GRO-seq (orange) or RNA-seq (light purple) reads from early embryos or adults. oma-1 (left) is highly expressed in adult worms and is among the embryo's most abundant maternally inherited mRNAs. However, oma-1 is not transcribed in early embryos. vet-6 (right) is not expressed in adult worms, is not maternally inherited, and is among the earliest transcribed genes in embryos.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Here, we presented a protocol to perform GRO-seq on as low as 1,000 L4/adult worms (around 2,000,000 nuclei) or 40,000 early embryos (around 800,000 nuclei). The input required for this protocol is much lower than the first reported GRO-seq in <i>C. elegans,</i> which required 100,000 worms (<a href="#bib2">Cecere et al., 2014</a>). Nonetheless, this minimum number of worms or embryos might still be difficult to achieve for specific samples, such as mutant strains with fertility defects. The main limitation of the protocol is the nuclei extraction procedure which has an estimated efficiency of 50%. Therefore, improvements in this step might help decrease the input material required to run GRO-seq in the future.</p>
<p>Another limitation is the high cost of the method. Indeed, reagents such as biotinylated nucleotides, streptavidin-conjugated beads, and ligases drastically influence the cost per sample. Home-production of the required enzymes and order of large batches of biotinylated nucleotides helps reduce the cost per sample of this protocol.</p>
<p>Ultimately, to understand the advantages and the limitations of the general GRO-seq procedure, we suggest referring to (<a href="#bib4">Mahat et al., 2016</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No library product or &gt; 25 PCR amplification cycles are required to obtain the libraries (step 112).</p>
<p>Possible causes:</p>
<p>Poor nuclei preparation quality.</p>
<p>Problems in the NRO reaction.</p>
<p>RNA degradation.</p>
<p>Low recovery in the SPRI purification steps.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Potential solutions are listed below:</p>
<p>Stain nuclei with DAPI and check for their integrity by fluorescence microscopy. If nuclei are not intact, reduce the number of dounce strokes at step 2.</p>
<p>Check for precipitates in the NRO 2<b>×</b> mix. If present, prepare the solution again.</p>
<p>Always work on ice and use RNase-free reagents at all steps.</p>
<p>Be careful not to over-dry the beads after purification. Dry beads are difficult to resuspend and may result in low recovery of RNA or DNA.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Libraries show the presence of unused primers or primer-dimers (&lt; 100 bp) after Tapestation analysis (step 116).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>If primers are present (&lt; 100 bp), perform a new round of library size selection using AMPure SPRI beads and perform a new Tapestation analysis.</p>
<div class="note">
<span class="note-title">Note:</span> We do not recommend performing gel purification to remove primer-dimers as this would result in sample loss. In our experience, an additional round of SPRI beads purification is sufficient to remove unused primers or primer-dimers and to obtain libraries of high quality.</div>
<h3 id="sec7.5">Problem 3</h3>
<p>Libraries are overamplified and show peaks corresponding to high molecular weight products (&gt; 600 bp) (step 112).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Reduce PCR amplification by 2–4 cycles and perform a new Tapestation analysis. <a href="#fig3">Figure 3</a> shows an example of the same library amplified with different PCR cycles.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1210-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Tapestation analysis of an overamplified library</div>
<p>(A) GRO-seq library from 100,000 embryos amplified with 22 PCR cycles. The electropherogram shows the presence of high molecular weight products (arrow).</p>
<p>(B) Same library show in (A). amplified with 2 PCR cycles less. PCR artifacts are not present, and the size distribution of the library is correct.</p>
</figcaption>
</figure>
<h3 id="sec7.7">Problem 4</h3>
<p>Streptavidin magnetic beads adhere to the tube wall after washing with solution A (step 42).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>The addition of 0.02% tween to Solution A can help to prevent this issue. Also, the use of low binding microcentrifuge tubes is suggested.</p>
<p>Furthermore, we have noticed that solution A might form precipitates few days after preparation. Therefore, we suggest preparing a new batch of solution A for each experiment.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Small or no pellet recovered after TRIzol or Phenol:Chloroform purification (steps 26 or 66).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>The absence of pellet after RNA precipitation might be due to incomplete mixing of the aqueous phase containing the RNA with GlycoBlue and Isopropanol/Ethanol (steps 23–24 for TRIzol purification and 64–65 for Phenol:Chloroform purification).</p>
<p>Mix the sample again by vortexing at maximum speed for 15 s and repeat the precipitation and centrifugation steps.</p>
<p>If the pellet is still not visible, add 2 μL of GlycoBlue, mix thoroughly, and repeat the precipitation and centrifugation steps by increasing the centrifugation speed to 16,000<b>×</b><i>g</i>.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Barucci, G., Cornes, E., Singh, M., Li, B., Ugolini, M., Samolygo, A., Didier, C., Dingli, F., Loew, D., Quarato, P., et al. (2020). Small-RNA-mediated transgenerational silencing of histone genes impairs fertility in piRNA mutants. Nat. Cell Biol. <i>22</i>, 235-245. <a class="external-link" href="https://doi.org/10.1038/s41556-020-0462-7">https://doi.org/10.1038/s41556-020-0462-7</a></p>
<p id="bib2">Cecere, G., Hoersch, S., O’keeffe, S., Sachidanandam, R., and Grishok, A. (2014). Global effects of the CSR-1 RNA interference pathway on the transcriptional landscape. Nat. Struct. Mol. Biol. <i>21</i>, 358-365. <a class="external-link" href="https://doi.org/10.1038/nsmb.2801">https://doi.org/10.1038/nsmb.2801</a></p>
<p id="bib3">Core, L.J., Waterfall, J.J., and Lis, J.T. (2008). Nascent RNA sequencing reveals widespread pausing and divergent initiation at human promoters. Science <i>322</i>, 1845-1848. <a class="external-link" href="https://doi.org/10.1126/science.1162228">https://doi.org/10.1126/science.1162228</a></p>
<p id="bib4">Mahat, D.B., Kwak, H., Booth, G.T., Jonkers, I.H., Danko, C.G., Patel, R.K., Waters, C.T., Munson, K., Core, L.J., and Lis, J.T. (2016). Base-pair-resolution genome-wide mapping of active RNA polymerases using precision nuclear run-on (PRO-seq). Nat. Protoc. <i>11</i>, 1455-1476. <a class="external-link" href="https://doi.org/10.1038/nprot.2016.086">https://doi.org/10.1038/nprot.2016.086</a></p>
<p id="bib5">Quarato, P., Singh, M., Cornes, E., Li, B., Bourdon, L., Mueller, F., Didier, C., and Cecere, G. (2021). Germline inherited small RNAs facilitate the clearance of untranslated maternal mRNAs in C. elegans embryos. Nat. Commun. <i>12</i>. <a class="external-link" href="https://doi.org/10.1038/s41467-021-21691-6">https://doi.org/10.1038/s41467-021-21691-6</a></p>
<p id="bib6">Singh, M., Cornes, E., Li, B., Quarato, P., Bourdon, L., Dingli, F., Loew, D., Procaccia, S., and Cecere, G. (2021). Translation and codon usage regulate Argonaute slicer activity to trigger small RNA biogenesis. Nat. Commun. <i>12</i>, 1-18. <a class="external-link" href="https://doi.org/10.1038/s41467-021-23615-w">https://doi.org/10.1038/s41467-021-23615-w</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and fulfilled by the lead contact, Germano Cecere (<a href="mailto:germano.cecere@pasteur.fr">germano.cecere@pasteur.fr</a>).</p>
<h4>Materials availability</h4>
<p>All materials are available commercially. A list of manufacturers for all reagents is provided in the <a href="#key-resources-table">key resources table</a> section.</p>
<h4>Data and code availability</h4>
<p>Data are available upon reasonable request.</p>
<h3>Acknowledgments</h3>
<p>G.C. has received funding from the <a href="https://doi.org/10.13039/501100003762">Institut Pasteur</a>, the <a href="https://doi.org/10.13039/501100004794">CNRS</a>, and from the <a href="https://doi.org/10.13039/100010663">European Research Council</a> (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement no. 679243 and no. 101002999). P.Q. was supported by Ligue Nationale Contre le Cancer (S-FB19032). N2 worms used for the setup of the protocol were provided by the CGC, funded by <a href="https://doi.org/10.13039/100000002">NIH</a> Office of Research Infrastructure Programs (P40 OD010440).</p>
<h3>Author contributions</h3>
<p>P.Q. performed all the experiments presented in the protocol and wrote the method together with G.C.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>