<article>
<section>
<h1 id="header">RHybridFinder: An R package to process immunopeptidomic data for putative hybrid peptide discovery</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Frederic Saab,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a></sup> David J. Hamelin,<sup><a href="#aff1">1</a></sup> Qing Ma,<sup><a href="#aff4">4</a></sup> Kevin A. Kovalchik,<sup><a href="#aff1">1</a></sup> Isabelle Sirois,<sup><a href="#aff1">1</a></sup> Pouya Faridi,<sup><a href="#aff2">2</a></sup> Chen Li,<sup><a href="#aff2">2</a></sup> Anthony W. Purcell,<sup><a href="#aff2">2</a></sup> Peter Kubiniok,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> and Etienne Caron<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>CHU Sainte-Justine Research Center, Montreal, QC H3T 1C5, Canada</p>
<p id="aff2"><sup>2</sup>Infection and Immunity Program and Department of Biochemistry and Molecular Biology, Biomedicine Discovery Institute, Monash University, Clayton, VIC 3800, Australia</p>
<p id="aff3"><sup>3</sup>Department of Pathology and Cellular Biology, Faculty of Medicine, Université de Montréal, Montreal, QC H3T 1J4, Canada</p>
<p id="aff4"><sup>4</sup>School of Electrical Engineering and Computer Science, Faculty of Engineering, University of Ottawa, Ottawa, ON K1N 6N5, Canada</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:peterkubiniok@gmail.com">peterkubiniok@gmail.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:etienne.caron@umontreal.ca">etienne.caron@umontreal.ca</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100875">10.1016/j.xpro.2021.100875</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Identification of proteasomal spliced peptides (PSPs) by mass spectrometry (MS) is not possible with traditional search engines. Here, we provide a protocol for running RHybridFinder (RHF), an R package for the computational inference of putative PSPs detected by MS. RHF extracts high confidence scored de novo sequenced peptides identified by PEAKS software. Those peptides are then matched to protein databases to infer cis- or trans-spliced major histocompatibility complex (MHC)-associated peptides. RHF is relatively fast and straightforward. PSPs have to be validated experimentally.</p>
<p>For complete details on the use and execution of the original protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref5">Faridi et al. (2018)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>RHybridFinder (RHF) is an improved R package for the discovery of spliced peptides</li>
<li>RHF builds upon the algorithm published in <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref5">Faridi et al. (2018)</a>
</li>
<li>RHF uses MS data analyzed in PEAKS</li>
<li>The spliced peptide candidates generated by RHF need to be validated experimentally</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The proteasome is recognized as the core enzymatic machinery of the antigen processing and presentation pathway wherein peptides derived from proteasomal proteolysis are selectively presented on the cell surface by MHC (major histocompatibility complex)-I molecules (<a href="#bib14">Neefjes et al., 2011</a>). In 2004, Hanada et al. discovered that the proteasome could cleave and splice peptide fragments to generate immunogenic epitopes presented by MHC class I molecules (<a href="#bib6">Hanada et al., 2004</a>). Following this groundbreaking discovery, other research groups have been able to uncover additional T cell spliced epitopes generated by the proteosome, referred in this protocol as proteasomal spliced peptides (PSPs) (<a href="#bib1">Berkers et al., 2015</a>; <a href="#bib3">Dalet et al., 2011</a>; <a href="#bib4">Ebstein et al., 2016</a>; <a href="#bib12">Michaux et al., 2014</a>; <a href="#bib18">Vigneron et al., 2004</a>).</p>
<p>More recently, MS-based immunopeptidomics has been used to expedite the identification of PSPs in a systematic manner, including cis- and trans-spliced peptides (<a href="#bib1">Berkers et al., 2015</a>; <a href="#bib5">Faridi et al., 2018</a>; <a href="#bib9">Liepe et al., 2010</a>, <a href="#bib10">2016</a>; <a href="#bib16">Rolfs et al., 2019</a>; <a href="#bib17">Specht et al., 2020</a>). However, MS-based studies using different computational approaches have led to a debate around the proportion of those PSPs in the MHC class I immunopeptidome (<a href="#bib8">Lichti, 2021</a>; <a href="#bib13">Mylonas et al., 2018</a>; <a href="#bib19">Wilhelm et al., 2021</a>).</p>
<p>Here, we provide a protocol to run RHybridFinder (RHF), an open access and improved R package built upon the computational workflow developed by <a href="#bib5">Faridi et al. (2018)</a> for the analysis of MS data to systematically identify putative PSPs (<a href="#bib5">Faridi et al., 2018</a>). High speed performance is the main strength of RHF in addition to be relatively straightforward to run. The main limitation is that the PSPs identified by RHF may not be genuily spliced by the proteasome in vivo. Their source and presentation should therefore be validated experimentally to move the debate forward (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overview of suggested workflow for the discovery of PSPs</div>
<p>We propose a four-step workflow for the identification of PSPs. The first three steps (blue squares: sample preparation, MS data acquisition and RHbridFinder enable computational exploration of putative PSPs followed by experimental validations (green square). A non-exhaustive list of possible experiments is shown for validating/gaining confidence in the identification of MHC-I peptides that are genuinely catalyzed by proteasomal splicing.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Recommended folder structure</div>
<p>The parent folder includes two child folders. The child folders include the various files that are necessary for running RHybridFinder. The dotted line (second_run) indicates that the DB search psm.csv file is added after the second DB search.</p>
</figcaption>
</figure>
<p>RHybridFinder is available on CRAN (<a href="https://cran.r-project.org/package=RHybridFinder">https://cran.r-project.org/package=RHybridFinder</a>) to enable more researchers to explore those debated peptides.</p>
<h3 id="sec1.1">Data collection</h3>
<p>For demonstration of the output of the different RHybridFinder functions, we have used datasets from the HLA Ligand Atlas (<a href="#bib11">Marcu et al., 2021</a>) deposited in PRIDE (Proteomics IDentification Database) <a href="http://www.ebi.ac.uk/pride/archive/projects/PXD019643">PXD019643</a>.</p>
<ol>
<li>Download the following mzML files and analyzed them in PEAKS:<br> 171002_AM_AUT01-DN17_Liver_W6-32_10%_DDA_3_400-650mz_msms4, 171002_AM_AUT01-DN17_Liver_W6-32_10%_DDA_3_400-650mz_msms5, 171002_AM_AUT01-DN17_Liver_W6-32_10%_DDA_3_400-650mz_msms6.</li>
<li>Analyze these files in PEAKS.</li>
</ol>
<h3 id="sec1.2">Installing Rstudio/R</h3>
<p>RHybridFinder package has been developed in RStudio and implemented in R programming language.</p>
<ol start="3">
<li>Download &amp; install Rstudio if not already installed: (<a href="https://www.rstudio.com/products/rstudio/download/">https://www.rstudio.com/products/rstudio/download/</a>).</li>
</ol>
<h3 id="sec1.3">Installing and loading RHybridFinder</h3>
<p>Below are the lines needed to install the RHybridFinder package from CRAN (the Comprehensive R Archive Network) and then load it.</p>
<ol start="4">
<li>Install and load RHybridFinder by typing “install.packages(“RHybridFinder”) in the R console.</li>
</ol>
<div class="textbox">
<p><code>&gt; install.packages(“RHybridFinder“)</code></p>
</div>
<ol start="5">
<li>Load RHybridFinder by typing “library(RHybridFinder)” in the R console</li>
</ol>
<div class="textbox">
<p><code>&gt; library(RHybridFinder)</code></p>
</div>
<div class="critical">
<span class="critical-title">Critical:</span> if you copy the lines of code from here, keep in mind that you might have to re-write the quotation marks yourself.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Human liver sample from autologous donor 17 - HLA Ligand Atlas</td>
<td>(<a href="#bib11">Marcu et al., 2021</a>)</td>
<td><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD019643">PXD019643</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>RStudio (version 1.3.1093)</td>
<td>RStudio website <a href="https://www.rstudio.com/">https://www.rstudio.com</a>
</td>
<td>SCR_000432</td>
</tr>
<tr>
<td>R (&gt;3.5.0)</td>
<td>R statistical software (<a href="https://www.r-project.org/">https://www.r-project.org/</a>)</td>
<td>SCR_001905</td>
</tr>
<tr>
<td>PEAKS (PEAKS X studio)</td>
<td>PEAKS website: <a href="https://www.bioinfor.com//">https://www.bioinfor.com//</a>
</td>
<td>N/A</td>
</tr>
<tr>
<td>RHybridFinder (v.0.2.0)</td>
<td><a href="https://cran.r-project.org/web/packages/RHybridFinder/index.html">https://cran.r-project.org/web/packages/RHybridFinder/index.html</a></td>
<td>N/A</td>
</tr>
<tr>
<td>seqinr (v. 4.2-5)</td>
<td>CRAN - (<a href="#bib2">Charif and Lobry, 2007</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>foreach (v.1.5.1), doParallel (v. 1.0.16)</td>
<td>CRAN</td>
<td>N/A</td>
</tr>
<tr>
<td>netMHCpan (v. 4.0 &amp;4.1)</td>
<td>DTU health tech: <a href="https://services.healthtech.dtu.dk/">https://services.healthtech.dtu.dk</a> (<a href="#bib15">Reynisson et al., 2020</a>)</td>
<td>SCR_018182</td>
</tr>
<tr>
<td>hybrid finder</td>
<td>
<a href="#bib5">Faridi et al. (2018)</a> (workflow on which the package is based)</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Step 1: Load inputs into R</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 min</div>
<p>Before running HybridFinder, the inputs need to be loaded into R. We propose the following way of loading the files into R in order to facilitate the process <a href="#fig2">Figure 2</a>.</p>
<ol>
<li>Create an object (folder_Exp1) for the path to the parent folder (Mel_Exp1) (but both can be named otherwise).</li>
</ol>
<div class="textbox">
<p><code>&gt; folder_Exp1 &lt;- file.path(“/Users/YOURUSERNAME/Desktop/Mel_Exp1”)</code></p>
</div>
<ol start="2">
<li>Import the <i>de novo</i> sequencing as well as the database results, both of which are located in the first_run child folder.
<ol type="a">
<li>
<i>de novo</i> sequencing results file
<div class="textbox">
<p><code>&gt; denovo_Exp1 &lt;- read.csv(file = file.path(folder_Exp1, “first_run”, "all_denovo_candidates.csv"), header=TRUE, sep=",", stringsAsFactors = FALSE)</code></p>
</div>
</li>
<li>database search results file</li>
</ol>
</li>
</ol>
<div class="textbox">
<p><code>&gt; db_search_Exp1&lt;- read.csv (file=file.path(folder_Exp1, “first_run”,“DB seach psm.csv”), header=TRUE, sep=“,”, stringsAsFactors=FALSE)</code></p>
</div>
<ol start="3">
<li>Create an object for the path to the proteome file, located in the parent folder (folder_Exp1) (see refproteome_Exp1, in the example below). The fasta proteome will be imported in R during the HybridFinder function.</li>
</ol>
<div class="textbox">
<p><code>&gt; refproteome_Exp1 &lt;- file.path(folder_Exp1, “uniprothuman-20379entries-Nov2019_validated.fasta”)</code></p>
</div>
<div class="critical">
<span class="critical-title">Critical:</span> Please note that if you copy the file access path (in windows), you will need to switch the backslash (“\”) to a normal slash ( “/”).</div>
<h4 id="sec2.1.1"><b>Access the datasets included in the R package</b></h4>
<p>The RHybridFinder package also includes demonstration datasets from the HLA Ligand Atlas that have already been analyzed in PEAKS. These datasets include PEAKS de novo sequencing results and PEAKS database search results.</p>
<div class="textbox">
<p><code># access denovo dataset</code></p>
<p><code>&gt; data(package= “RHybridFinder”, “denovo_Human_Liver_AUTD17”)</code></p>
<p><code># access database search dataset</code></p>
<p><code>&gt; data(package=”RHybridFinder”, “db_Human_Liver_AUTD17”)</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> that due to size constraints the proteome database (.fasta) file is not included in the package. It can be downloaded from the <a href="https://www.uniprot.org/proteomes/UP000005640">Uniprot database</a>.</div>
<div class="note">
<span class="note-title">Note:</span> In the environment tab, the denovo_Human_Liver_AUTD17 and db_Human_Liver_AUTD17 should appear. Note that if you see &lt;promise&gt;, after clicking on the objects, the data would appear.</div>
<h3 id="sec2.2">Step 2: Run HybridFinder</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–5 min (with parallelism, 8 cores) - 10–15 min (without parallelism)</div>
<p>In order to have a relatively short runtime, we have implemented an option to use parallel computing. However, please note that because parallel computing requires a certain amount of processing units for proper functioning, it has been made possible to also run HybridFinder without parallel computing.</p>
<p>Based on default parameters in the HybridFinder function, the “all de novo candidates.csv” file contains 16,286 peptide sequences and the runtime (parallelism with 8 cores) is of 2 min 17 s ∼5 min are required for double the number of peptides. Without parallelism, the runtime ranged between 10 and 15 min for 16,286 peptide sequences.</p>
<ol start="4">
<li>Run Hybridfinder (Please refer to <a href="#tbl1">Table 1</a> in order to know more about the inputs needed) and export the results in the parent folder.
<table id="tbl1">
<caption>Table 1. HybridFinder function parameters</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Description</th>
<th>Default value</th>
</tr>
</thead>
<tbody>
<tr>
<td>de novo_candidates</td>
<td>the dataframe containing the <i>de novo</i> sequencing results</td>
<td>No defaults. Necessary input.</td>
</tr>
<tr>
<td>db_search</td>
<td>the data frame containing the database search results</td>
<td>No defaults. Necessary input.</td>
</tr>
<tr>
<td>db_search</td>
<td>the data frame containing the database search results</td>
<td>No defaults. Necessary input.</td>
</tr>
<tr>
<td>proteome_db</td>
<td>the file path to the proteome used for the database search</td>
<td>No defaults. Necessary input.</td>
</tr>
<tr>
<td>(Optional) customALCcutoff</td>
<td>A custom score cutoff that can be set by the user as long as it would be at least 85</td>
<td>NULL. (ALC cutoff calculated automatically as median of matching peptide sequences of assigned spectra). If set manually, minimum is 85.</td>
</tr>
<tr>
<td>with_parallel : boolean (True or False)</td>
<td>representing whether parallel computing should be employed for running the function.</td>
<td>TRUE</td>
</tr>
<tr>
<td>(Optional) customCores</td>
<td>If with_parallel is set to TRUE and the PC has &gt;5 cores, the user can set a custom amount of cores to be used by the function.</td>
<td>6</td>
</tr>
<tr>
<td>(Optional) export_files : boolean (True or False)</td>
<td>by default it is set to False, however, if set to True, then the following input is essential.</td>
<td>FALSE</td>
</tr>
<tr>
<td>(Optional) export_dir</td>
<td>file path to the directory where the output files should be stored. This parameter is necessary for the export.</td>
<td>NULL</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="textbox">
<p><code>&gt; HybridFinder_results_Exp1&lt;- HybridFinder(denovo_candidates = denovo_Exp1, db_search = db_search_Exp1, proteome_db = refproteome_Exp1,customALCcutoff = NULL, with_parallel=TRUE, customCores = 8, export_files= TRUE, export_dir = folder_Exp1)</code></p>
</div>
<div class="critical">
<span class="critical-title">Critical:</span> if you use the datasets included in the package, please note that they are named differently so for instance the “denovo_candidates” and “db_search” parameters should be set to the datasets loaded from the package: denovo_Human_Liver_AUTD17 and db_Human_Liver_AUTD17, respectively.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to store the HybridFinder results in an object (i.e HybridFinder_results_Exp1), as the HybridFinder output dataframe will come in handy in the second function.</div>
<div class="note">
<span class="note-title">Note:</span> At the end of the hybrid proteome will be the concatenated hybrid fake proteins with the name pattern ‘sp|denovo_HF_fake_protein_[#]’.</div>
<div class="note">
<span class="note-title">Note:</span> with_parallel is activated if set to true and if the PC has more than 5 cores.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Please ensure to have a minimal number of other windows open and to save any work in other softwares prior to using HybridFinder with parallelism.</div>
<p>The function will output a list (<a href="#fig3">Figure 3</a>) containing: (1) the HybridFinder output containing all the denovo peptides along with their potential splice type explanation cis-/trans-, (2) a list of the step1 hybrid candidate peptides, (3) the hybrid proteome (merged proteome: the original user proteome along with the hybrid proteome composed of the concatenated candidate hybrid peptide sequences).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Screenshot of the HybridFinder function results</div>
<p>In the results list you will find 3 items: 1) a dataframe containing the HybridFinder output. 2) a character vector containing the candidate spliced peptides. 3) a list which is in a seqinr class (<a href="#bib2">Charif and Lobry, 2007</a>) containing the merged hybrid proteome.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> In the example above, export_files have been set to TRUE and the export_dir has been defined which means that the files are also automatically exported. If these two parameters were not specified or were set to FALSE &amp; NULL, the results are only stored in the Exp1_HybridFinder_results. In this case, you can still use “export_HybridFinder_results” as in the code below, where HybridFinder_results_Exp1 is the object created above for the storage of HybridFinder results.</div>
<div class="textbox">
<p><code>&gt; export_HybridFinder_results(HybridFinder_results_Exp1, export_dir= folder_Exp1)</code></p>
</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If you would like to conduct the rest of the protocol at a later time, either use the export functionality and then load the HybridFinder output in order to use it for the second step. Alternatively, save the objects in R in a .rda file as follows and once you want to use it again for the step 4, load checknetMHCpan inputs into R.</div>
<div class="textbox">
<p><code>&gt; save (HybridFinder_results_Exp1, file=file.path(folder_Exp1, ”HybridFinder_results_Exp1.rda”)</code></p>
<p><code>&gt;load (file.path(folder_Exp1, ”HybridFinder_results_Exp1.rda”))</code></p>
</div>
<h3 id="sec2.3">Step 3: Database search using hybrid Fasta</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>An essential interim step must follow the HybridFinder function and consists of running a database search in PEAKS with the merged proteome. Importantly, now that a merged hybrid proteome has been obtained from the HybridFinder function, it can be used to obtain potential PSPs whose quality is comparable with all other database search peptides while filtering all peptides at the same FDR (False Discovery Rate) cutoff which can be adjusted by the users in PEAKS. In the original workflow by <a href="#bib5">Faridi et al. (2018)</a>, the database search peptides in both runs were filtered in PEAKS at a 1% FDR.</p>
<ol start="5">
<li>Perform a database search in PEAKS using the original raw MS file (while using the same settings as in the beginning) however, this time while using the merged hybrid proteome (.fasta) file generated with the HybridFinder function.</li>
</ol>
<h3 id="sec2.4">Step 4: Load checknetMHCpan inputs into R</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 min</div>
<p>Prior to running checknetMHCpan, please ensure that netMHCpan (versions 4.0 or 4.1) is installed. checknetMHCpan is the last step of the hybrid finder workflow, the function uses the database search results from the second PEAKS analysis and provides the binding affinity results of all the peptides along with their categorizations.</p>
<ol start="6">
<li>Create an object for the location of the netMHCpan executable</li>
</ol>
<div class="textbox">
<p><code>&gt; netmhcpan_dir &lt;- file.path(“/usr/local/bin”)</code></p>
</div>
<ol start="7">
<li>Create an object (vector) for storing the HLA-I alleles that you would like to have binding affinity predictions for.</li>
</ol>
<div class="textbox">
<p><code>&gt; alleles_Exp1 &lt;- c(“HLA-A∗02:01”, “HLA-A∗03:01”, “HLA-B∗07:02”)</code></p>
</div>
<ol start="8">
<li>Retrieve the HybridFinder output from the HybridFinder function results</li>
</ol>
<div class="textbox">
<p><code>&gt; HF_output_Exp1 &lt;- HybridFinder_results_Exp1[[1]]</code></p>
</div>
<ol start="9">
<li>Import the database search results (from step 3: Database search using hybrid fasta)</li>
</ol>
<div class="textbox">
<p><code>&gt; rerun_db_search_Exp1 &lt;- read.csv(file.path(folder_Exp1, “second_run”, “DB search psm.csv”), sep=“,”, head = TRUE, stringsAsFactors = FALSE)</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> in case your computer’s OS is “Windows” (netMHCpan is not compatible with Windows) the web version of netMHCpan (<a href="http://www.cbs.dtu.dk/services/NetMHCpan-4.1/instructions.php">http://www.cbs.dtu.dk/services/NetMHCpan-4.1/instructions.php</a>) would come in handy. In this case, we propose to use a separate function from this package instead (step2_wo_netmhcpan) which outputs a netMHCpan-ready input of sequences in .pep format.</div>
<h4 id="sec2.4.1"><b>Access the datasets included in the R package</b></h4>
<p>The demonstration datasets from the HLA Ligand Atlas included in this package also include datasets for the checknetMHCpan/step2_wo_netMHCpan functions. After having run the HybridFinder function and stored the results in HyrbidFinder_results_Exp1, PEAKS was run using the merged hybrid proteome. Below is a way to retrieve the second PEAKS run dataset included in the package:</p>
<div class="textbox">
<p><code>&gt; data(package= “RHybridFinder”, “db_rerun_Human_Liver_AUTD17”)</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> The merged proteome used for the second database search is based on the customALCcutoff being set to NULL (default parameter value).</div>
<div class="critical">
<span class="critical-title">Critical:</span> The merged proteome database would change between different samples, and if the customALCcutoff parameter is changed. The same merged hybrid proteome cannot be used for separate analyses.</div>
<h3 id="sec2.5">Step 5: Run checknetMHCpan</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 1 min</div>
<p>The checknetMHCpan function embodies the second major step of the workflow. The categorizations of the hybrid peptides from the HybridFinder output are retrieved for matched peptides found in the second PEAKS database results. Then, peptide-MHC class I binding predictions for the entire database search results (for peptides between 9 and 12 amino acids) are computed using netMHCpan and are tidied in order to summarize the results.</p>
<ol start="10">
<li>Run checknetMHCpan using the code below (Please refer to <a href="#tbl2">Table 2</a> in order to know more about the inputs needed) and export the results in the same folder:
<table id="tbl2">
<caption>Table 2. checknetMHCpan function parameters</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Description</th>
<th>Default value</th>
</tr>
</thead>
<tbody>
<tr>
<td>netmhcpan_directory</td>
<td>the directory where netMHCpan is installed (i.e., ‘/usr/bin’ or ‘/usr/local/bin’, depending on where you have it installed)</td>
<td>No defaults. Necessary input.</td>
</tr>
<tr>
<td>netmhcpan_alleles</td>
<td>a vector composed of the alleles the peptides will be tested against.</td>
<td>No defaults. Necessary input.</td>
</tr>
<tr>
<td>peptide_rerun</td>
<td>the database search results from the second peaks run</td>
<td>No defaults. Necessary input.</td>
</tr>
<tr>
<td>HF_step1_output</td>
<td>the data frame from the HybridFinder function of the containing the spliced peptide potential explanations as well as RT, m/z, ALC, Scan &amp; Fraction</td>
<td>No defaults. Necessary input.</td>
</tr>
<tr>
<td>(Optional) export_files : boolean (True or False)</td>
<td>by default it is set to False, however, if set to True, then the following input is essential.</td>
<td>FALSE</td>
</tr>
<tr>
<td>(Optional) export_dir</td>
<td>file path to the directory where the output files should be stored. This parameter is necessary for the export.</td>
<td>NULL</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="textbox">
<p><code>&gt; checknetMHCpan_results_Exp1 &lt;- checknetMHCpan(netmhcpan_directory = netmhcpan_dir, netmhcpan_alleles = alleles_Exp1, peptide_rerun = rerun_db_search_Exp1, HF_step1_output = HF_output_Exp1, export_files= TRUE, export_dir = folder_Exp1)</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> checknetMHCpan is compatible with the exports from both netMHCpan 4.0 &amp; netMHCpan 4.1.</div>
<div class="critical">
<span class="critical-title">Critical:</span> if you use the datasets included in the package, please note that they are named differently so for instance the “peptide_rerun” parameter should be set to dataset loaded from the package db_rerun_Human_Liver_AUTD17.</div>
<p>After running the code above, a results list should be returned (<a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Screenshot of the checknetMHCpan results list</div>
<p>In the results list you will find 3 items: 1) a dataframe containing the netMHCpan results. 2) a dataframe containing the tidied netMHCpan results. 3) the database search results with the “Potential_spliceType” for the hybrid peptides retrieved from step1.</p>
</figcaption>
</figure>
<p>These results are also exportable with the export_checknetMHCpan_results function.</p>
<div class="textbox">
<p><code>&gt; export_checknetMHCpan_results(step2_RHF_results_Exp1 , export_dir = folder_Exp1)</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> If you intend on using the web version of netMHCpan (especially useful for windows OS users) or another software for peptide binding affinity, the step2_wo_netMHCpan function does the same as checknetMHCpan but without running netMHCpan. The function should return a list (<a href="#fig5">Figure 5</a>) containing the updated database search results as well as a list of the peptides which can be used as input in the web version of netMHCpan.</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Screenshot of the step2_wo_netMHCpan results list</div>
<p>In the results list you will find 2 items: 1) a character vector containing the netMHCpan-ready input. 2) the database search results with the “Potential_spliceType” for the hybrid peptides retrieved from step1.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec3.1">HybridFinder</h3>
<p>The HybridFinder function follows the same rationale as indicated in <a href="#bib5">Faridi et al. (2018)</a>. After high-confidence <i>de novo</i> peptides are extracted, these are searched sequentially for an exact hit, followed by a search of pair fragments within one protein and then within two proteins (<a href="#fig6">Figure 6</a>). Finally, the sequences of all hybrid peptides are concatenated to create fake proteins, which are added at the bottom of the proteome database in order to constitute a merged hybrid proteome.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. HybridFinder function</div>
<p>HybridFinder extracts high confidence <i>de novo</i> peptides by using a ALC cutoff based on the median ALC of common spectrum groups &amp; sequence of peptides between the <i>de novo</i> and the database search. The ALC cutoff is used to filter unassigned <i>de novo</i> spectrum groups in order to obtain high confidence <i>de novo</i> spectra. All sequences are then searched in the proteome for the entire sequence, those that match are filtered and considered “Linear”, the remainder of the peptide spectrum groups are “cut” in order to create peptide fragment combinations. These are then searched in the proteome for whether fragment combinations exist within a same protein, matches are considered as cis-spliced and further filtered. Finally, fragment combinations are created from those that didn’t match in the previous step and are searched whether they exist in two proteins. If there is a match, these are considered as trans-spliced peptides. The remaining uncategorized spectrum groups are considered not to have a biological explanation (NBE) and are therefore discarded.</p>
</figcaption>
</figure>
<p>Typically, when the HybridFinder function is run, 3 messages are printed representing each major stage of the algorithm and finally ‘Done!’ is printed once the processing is finished. The function returns a list containing 3 items: the HybridFinder output (<a href="#fig7">Figure 7</a>) where the predicted splice type is displayed, a character vector containing only the list of hybrid candidates (<a href="#fig8">Figure 8</a>) and finally the merged hybrid proteome (<a href="#fig9">Figure 9</a>) where the hybrid peptide candidates have been concatenated as fake proteins.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Screenshot of the HybridFinder output dataframe</div>
<p>(5 rows), The <i>Fraction</i> column represents the LC-MS run, the <i>Scan</i> column is a number representing a unique index for the tandem mass spectra (F[Fraction#]:Scan#), <i>m/z</i> is the precursor mass-to-charge ratio, <i>RT</i> is the Retention Time (elution time) for the spectrum, <i>Peptide</i> corresponds to the peptide sequences. The <i>Length</i> column represents the number of amino acids for a given peptide, <i>ALC</i> (Average Local Confidence), is a score calculated in PEAKS as the total of the residue local confidence scores in the peptide divided by the peptide length. These columns are not provided by the HybridFinder function, they are columns found in any PEAKS <i>de novo</i> sequencing export. For more information, please visit the PEAKS user manual. The <i>Potential_spliceType</i> corresponds to the resulting categorization from the HybridFinder function. Finally, the <i>proteome_database_used</i> is the filename of the fasta proteome provided by the user (this column is mainly for helping the user keep track of the proteome used) in the HybridFinder function.</p>
</figcaption>
</figure>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Screenshot of the HybridFinder hybrid peptide candidates vector (5 rows)</div>
</figcaption>
</figure>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Screenshot of the bottom of the HybridFinder merged hybrid proteome (5 proteins)</div>
</figcaption>
</figure>
<p>The results might differ if the customALCcutoff score parameter is changed. If the results are exported, these are stored in a folder as .csv files and the merged proteome database is saved as .fasta file. The peptide sequences predicted as spliced are considered as preliminary candidates. Performing the rest of the steps is essential in order to obtain the final list.</p>
<h3 id="sec3.2">checknetMHCpan and step2_wo_netMHCpan</h3>
<p>The checknetMHCpan &amp; step2_wo_netMHCpan functions represent the last step in <a href="#bib5">Faridi et al.’s (2018)</a> workflow. After a database search is performed using the merged hybrid proteome in step 1, these two functions can be used. Both of these functions retrieve the potential splice type categorization established in step 1. However, with checknetMHCpan the user can directly obtain MHC-I binding affinity predictions computed for all peptides between 9 and 12 amino acids using netMHCpan (<a href="#bib7">Jurtz et al., 2017</a>; <a href="#bib15">Reynisson et al., 2020</a>).</p>
<p>The checknetMHCpan function returns two formats of the netMHCpan results and the updated database search results from the second run with the potential splice type. The first format of the netMHCpan represents the results as they are (<a href="#fig10">Figure 10</a>). The second format is a tidied version of the netMHCpan results (<a href="#fig11">Figure 11</a>), where the rows are summarized into different columns, to allow quick analysis of the netMHCpan results (especially when more than one HLA-I allele is used); in these columns are summed the number of HLA-I alleles that a given peptide is a strong or weak binder to as well as the corresponding alleles. Finally, the database search results dataframe (from the second PEAKS run) updated with the potential splice type determined in the HybridFinder function for each peptide (<a href="#fig12">Figure 12</a>). Additionally, any sequence not identified in the HybridFinder output and solely attributed to the fake proteins created is removed. If exported, these are stored in a folder containing 2 .csv files and a .tsv (tab-separated values) corresponding to these different outputs.</p>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Screenshot of the checknetMHCpan netMHCpan results</div>
<p>(5 rows) <i>HLA/MHC</i> is the allele, <i>Peptide</i> is the amino acid sequence of the potential ligand, <i>Core</i> is the minimal 9 amino acid sequence core to enable HLA binding, <i>Of</i> is the starting position of the Core within the peptide, <i>Gp</i> and <i>Gl</i> are the position and the length of the deletions (respectively), if any. <i>Ip</i> and <i>Il</i> are the position and the length of the insertions (respectively), if any. <i>Icore</i> is the interaction core<i>, Identity</i> is PEPLIST (which indicates that peptides were used as input as opposed to proteins in fasta-format). <i>Score</i> is the raw prediction, <i>Aff(nM)</i> is the predicted IC50 value in nanoMolar units, <i>%Rank</i> is the percentile rank of the predicted affinity compared to a set of random natural ligands. <i>BindLevel</i> is designated by 3 qualifiers: Strong binder, Weak binder, None binder. <i>Potential_spliceType</i> is the categorization retrieved from the HybridFinder output on the potential splice type explanation of the peptide (i.e., linear, cis, trans).</p>
</figcaption>
</figure>
<figure id="fig11"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig11.jpg" alt="Fig11.jpg">
<figcaption>
<div class="figcaption-title">Figure 11. Screenshot of the checknetMHCpan tidied netMHCpan results</div>
<p>(5 rows) <i>Peptide</i> is the amino acid sequence of the potential ligand, the <i>strongBinder, weakBinder, noneBinder</i> (this column not shown in this figure) columns correspond to the alleles to which a given peptide is a strong/weak/none binder to, respectively. If more than one allele, these are separated by commas. For each peptide, there will be %Rank columns per allele (e.g., If 3 alleles were specified in the checknetMHCpan command, then each peptide will have 3%Rank columns). <i>strongBinder_count, weakBinder_count, noneBinder_count</i> represent the number of alleles to which a peptide is a strong/weak/none binder to. Lastly, the <i>Potential_spliceType</i> column is the categorization retrieved from the HybridFinder output on the potential splice type explanation of the peptide (i.e., linear, cis, trans).</p>
</figcaption>
</figure>
<figure id="fig12"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig12.jpg" alt="Fig12.jpg">
<figcaption>
<div class="figcaption-title">Figure 12. Screenshot of the checknetMHCpan database search results updated with the Potential_spliceType column</div>
<p>(5 rows) <i>Peptide</i> is the amino acid sequence of the potential ligand, <i>X.log10P</i> represents the best -10logP identification score for the corresponding peptide. <i>Mass</i> represents the monoisotopic mass of the peptide, <i>Length</i> is the number of amino acid residues that constitute the given peptide, <i>ppm</i> is the precursor mass error, the <i>m.z</i> is the precursor mass-to-charge ratio, <i>Z</i> is the precursor charge, <i>RT</i> is the Retention Time (elution time) for the spectrum, <i>Area</i> represents the area underthe curve of the peptide feature found at the same m/z and retention time as the MS/MS scan, <i>Fraction</i> is the LC-MS run, <i>id</i> represents the precursor ID associated with the PSM, <i>Scan</i> is a number representing a unique index for tandem mass spectra (F[Fraction#]:Scan#), <i>from.Chimera</i> (this column is not shown in this figure) displays whether the identified peptide is from chimeric spectra, <i>Source.File</i> is the mzML/mzXML file used in the PEAKS analysis, <i>PTM</i> is the type of the post-translational modification, <i>Ascore</i> is the localization score assigned to modifications on the peptide, <i>Found.By</i> represents the analysis (in this case PEAKS DB). <i>Peptide_no_mods</i> represents the peptide sequence without modifications, <i>Potential_spliceType</i> is linear, cis or trans and is retrieved from the HybridFinder function.</p>
</figcaption>
</figure>
<p>The step2_wo_netMHCpan is the equivalent of checknetMHCpan with the exception of computing binding affinity. The function returns a netMHCpan-ready list of peptides (<a href="#fig13">Figure 13</a>), as well as the updated the database search results (<a href="#fig14">Figure 14</a>). If exported, the results are exported into a folder containing a .pep file and a .csv file.</p>
<figure id="fig13"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig13.jpg" alt="Fig13.jpg">
<figcaption>
<div class="figcaption-title">Figure 13. Screenshot of the step2_wo_netMHCpan netMHCpan-ready input (5 rows)</div>
</figcaption>
</figure>
<figure id="fig14"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1118-Fig14.jpg" alt="Fig14.jpg">
<figcaption>
<div class="figcaption-title">Figure 14. Screenshot of the checknetMHCpan database search results updated with the Potential_spliceType column</div>
<p>(5 rows) The dataframe contains the same columns as in <a href="#fig13">Figure 13</a></p>
</figcaption>
</figure>
<p>After running checknetMHCpan or step2_wo_netMHCpan the final list of hybrid candidate peptides should be explored for further experimental validation (<a href="#fig1">Figure 1</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The presented package was developed and optimized for exports from PEAKS software. Therefore, results from other search engines or <i>de novo</i> sequencing softwares might not work while using this package. Limitations related to the workflow include the possible introduction of bias towards having results containing a higher proportion of Leucine residues. This is due to the workaround proposed by <a href="#bib5">Faridi et al. (2018)</a> which is also used in this package, entailing a switch of all Isoleucines to Leucines in the database search and the proteome since <i>de novo</i> sequencing does not differentiate between Isoleucine and leucine. As mentioned above, it is also important to emphasize that this protocol does not enable the direct identification of high-confidence PSPs that are genuinely spliced by the proteasome in vivo. However, this protocol enables the computational identification of putative PSPs, which should then be validated experimentally in a rigorous manner as shown in <a href="#fig1">Figure 1</a>.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>While installing the .tar.gz file for the package, in case you run into the following error: ”Error in install.packages : type == ”both” cannot be used with ’repos = NULL’”</p>
<h3 id="sec5.2">Potential solution</h3>
<p>The solution would be to simply invoke the install.packages function while specifying where the package is located, setting the repository (repos) to NULL and setting the type as source (source package).</p>
<div class="textbox">
<p><code>&gt; install.packages("∼/Downloads/RHybridFinder_0.1.0.tar", repos = NULL, Type="source")</code></p>
</div>
<h3 id="sec5.3">Problem 2</h3>
<p>While running HybridFinder, in case you run into the following error: ”Error in prepare_input_for_HF(de novo_candidates, db_search): Please make sure you have the right input. N.B: The <i>de novo</i> results data frame should be the first input”</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Verify the <i>de novo</i> data frame has been correctly imported. Since the <i>de novo</i> results file is in .csv format, the separator should be a comma “,”, stringsAsFactors should be set to FALSE and lastly the header should be set to TRUE. Please refer to step1: Loading inputs into R.</p>
<p>Verify that the HybridFinder parameters are properly typed. The <i>de novo</i> sequencing results data frame is indicated first and then the database search results. Alternatively, write the parameters and assigned them their appropriate objects (i.e., de novo_candidates = de novo_results_human_liver_Exp1). Please refer to step2: Run HybridFinder.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>While running HybridFinder, in case you run into the following error: ”Error in $&lt;-.dataframe`(`∗tmp`, “db_id”, value = character ( 0 ) ) : replacement has 0 rows, data has[…]”</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Verify the database search data frame, make sure it has been correctly imported. Since the database search results file is in .csv format the separator should be a comma “,”, stringsAsFactors should be set to FALSE and lastly the header should be set to TRUE. Please refer to step1: Loading inputs into R.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>While running checknetMHCpan, if the following error is displayed: ”Error in checknetMHCpan[…]:Please provide the proper input”</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Verify that the <i>de novo</i> and database search data frames are not switched. Please refer to step 5: Run checknetMHCpan.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>While running checknetMHCpan, if the following error is displayed: ”Please check the input alleles: […]”</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Ensure that the alleles are in the right format, or that the allele is written correctly (i.e., HLA-A03<b>:</b>01, HLA-A∗03:01). Please refer to step 4: Load checknetMHCpan inputs into R.</p>
<h3 id="sec5.11">Problem 6</h3>
<p>While running checknetMHCpan, if the path to netMHCpan is not correct, the following error might appear: sh: 1: [/temporary directory/netMHCpan]: not found error in running command</p>
<h3 id="sec5.12">Potential solution</h3>
<p>The issue could either be that the directory does not contains the netMHCpan file or that the directory was not well written I.e(‘usr/bin/' vs. ‘/usr/bin’ or ‘/usr/bin/, where the first example is wrong and the other two are correct). Please refer to step 4: Load checknetMHCpan inputs into R</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Berkers, C.R., Jong, A. de, Schuurman, K.G., Linnemann, C., Geenevasen, J.A.J., Schumacher, T.N.M., Rodenko, B., and Ovaa, H. (2015). Peptide splicing in the proteasome creates a novel type of antigen with an isopeptide linkage. J. Immunol. <i>195</i>, 4075-4084. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref1">View at publisher</a></p>
<p id="bib2">Charif, D. and Lobry, J.R. (2007). SeqinR 1.0-2: a contributed package to the R project for statistical computing devoted to biological sequences retrieval and analysis. In Structural Approaches to Sequence Evolution, Molecules, Networks, Populations, Ugo Bastolla, Markus Porto, H. Eduardo Roman, and Michele Vendruscolo, eds. (Springer Verlag)), pp. 207-232. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref2">View at publisher</a></p>
<p id="bib3">Dalet, A., Robbins, P.F., Stroobant, V., Vigneron, N., Li, Y.F., El-Gamil, M., Hanada, K., Yang, J.C., Rosenberg, S.A., and Eynde, B.J.V. den. (2011). An antigenic peptide produced by reverse splicing and double asparagine deamidation. Proc. Natl. Acad. Sci. USA <i>108</i>, E323-E331. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref3">View at publisher</a></p>
<p id="bib4">Ebstein, F., Textoris-Taube, K., Keller, C., Golnik, R., Vigneron, N., Eynde, B.J.V. den, Schuler-Thurner, B., Schadendorf, D., Lorenz, F.K.M., Uckert, W., et al. (2016). Proteasomes generate spliced epitopes by two different mechanisms and as efficiently as non-spliced epitopes. Sci. Rep. <i>6</i>, 24032. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref4">View at publisher</a></p>
<p id="bib5">Faridi, P., Li, C., Ramarathinam, S.H., Vivian, J.P., Illing, P.T., Mifsud, N.A., Ayala, R., Song, J., Gearing, L.J., Hertzog, P.J., et al. (2018). A subset of HLA-I peptides are not genomically templated: Evidence for cis- and trans-spliced peptide ligands. Sci. Immunol. <i>3</i>, eaar3947. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref5">View at publisher</a></p>
<p id="bib6">Hanada, K., Yewdell, J.W., and Yang, J.C. (2004). Immune recognition of a human renal cancer antigen through post-translational protein splicing. Nature <i>427</i>, 252-256. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref6">View at publisher</a></p>
<p id="bib7">Jurtz, V., Paul, S., Andreatta, M., Marcatili, P., Peters, B., and Nielsen, M. (2017). NetMHCpan-4.0: improved peptide–MHC class I interaction predictions integrating eluted ligand and peptide binding affinity data. J. Immunol. <i>199</i>, 3360-3368. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref7">View at publisher</a></p>
<p id="bib8">Lichti, C.F. (2021). Identification of spliced peptides in pancreatic islets uncovers errors leading to false assignments. Proteomics <i>21</i>, e2000176. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref8">View at publisher</a></p>
<p id="bib9">Liepe, J., Mishto, M., Textoris-Taube, K., Janek, K., Keller, C., Henklein, P., Kloetzel, P.M., and Zaikin, A. (2010). The 20S proteasome splicing activity discovered by SpliceMet. Plos. Comput. Biol. <i>6</i>, e1000830. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref9">View at publisher</a></p>
<p id="bib10">Liepe, J., Marino, F., Sidney, J., Jeko, A., Bunting, D.E., Sette, A., Kloetzel, P.M., Stumpf, M.P.H., Heck, A.J.R., and Mishto, M. (2016). A large fraction of HLA class I ligands are proteasome-generated spliced peptides. Science <i>354</i>, 354-358. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref10">View at publisher</a></p>
<p id="bib11">Marcu, A., Bichmann, L., Kuchenbecker, L., Kowalewski, D.J., Freudenmann, L.K., Backert, L., Mühlenbruch, L., Szolek, A., Lübke, M., Wagner, P., et al. (2021). HLA Ligand Atlas: a benign reference of HLA-presented peptides to improve T-cell-based cancer immunotherapy. J. Immunother. Cancer <i>9</i>, e002071. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref11">View at publisher</a></p>
<p id="bib12">Michaux, A., Larrieu, P., Stroobant, V., Fonteneau, J.-F., Jotereau, F., Eynde, B.J.V. den, Moreau-Aubry, A., and Vigneron, N. (2014). A spliced antigenic peptide comprising a single spliced amino acid is produced in the proteasome by reverse splicing of a longer peptide fragment followed by trimming. J. Immunol. <i>192</i>, 1962-1971. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref12">View at publisher</a></p>
<p id="bib13">Mylonas, R., Beer, I., Iseli, C., Chong, C., Pak, H.-S., Gfeller, D., Coukos, G., Xenarios, I., Müller, M., and Bassani-Sternberg, M. (2018). Estimating the contribution of proteasomal spliced peptides to the HLA-I Ligandome. Mol. Cell. Proteomics <i>17</i>, 2347-2357. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref13">View at publisher</a></p>
<p id="bib14">Neefjes, J., Jongsma, M.L.M., Paul, P., and Bakke, O. (2011). Towards a systems understanding of MHC class I and MHC class II antigen presentation. Nat. Rev. Immunol. <i>11</i>, 823-836. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref14">View at publisher</a></p>
<p id="bib15">Reynisson, B., Alvarez, B., Paul, S., Peters, B., and Nielsen, M. (2020). NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. Nucleic Acids Res. <i>48</i>, W449-W454. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref15">View at publisher</a></p>
<p id="bib16">Rolfs, Z., Müller, M., Shortreed, M.R., Smith, L.M., and Bassani-Sternberg, M. (2019). Comment on “A subset of HLA-I peptides are not genomically templated: Evidence for cis- and trans-spliced peptide ligands. Sci. Immunol. <i>4</i>, eaaw1622. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref16">View at publisher</a></p>
<p id="bib17">Specht, G., Roetschke, H.P., Mansurkhodzhaev, A., Henklein, P., Textoris-Taube, K., Urlaub, H., Mishto, M., and Liepe, J. (2020). Large database for the analysis and prediction of spliced and non-spliced peptide generation by proteasomes. Sci. Data <i>7</i>, 146. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref17">View at publisher</a></p>
<p id="bib18">Vigneron, N., Stroobant, V., Chapiro, J., Ooms, A., Degiovanni, G., Morel, S., Bruggen, P. van der, Boon, T., and Eynde, B.J.V. den. (2004). An antigenic peptide produced by peptide splicing in the proteasome. Science <i>304</i>, 587-590. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref18">View at publisher</a></p>
<p id="bib19">Wilhelm, M., Zolg, D.P., Graber, M., Gessulat, S., Schmidt, T., Schnatbaum, K., Schwencke-Westphal, C., Seifert, P., Krätzig, N. de A., Zerweck, J., et al. (2021). Deep learning boosts sensitivity of mass spectrometry-based immunopeptidomics. Nat. Commun. <i>12</i>, 3346. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00581-5/sref19">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources should be directed to and will be fulfilled by the lead contact, Etienne Caron <a href="mailto:etienne.caron@umontreal.ca">etienne.caron@umontreal.ca</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The package is available on CRAN and includes data (PEAKS analyses) from HLA Ligand Atlas (<a href="#bib11">Marcu et al., 2021</a>) deposited in PRIDE (Proteomics IDentification Database) <a href="http://www.ebi.ac.uk/pride/archive/projects/PXD019643">PXD019643</a> (were analyzed in PEAKS and used in this protocol for demonstration purposes only).</p>
<h3>Acknowledgments</h3>
<p>This work was supported by funding from the Funds de recherche du Québec - Santé (FRQS), the Cole Foundation, <a href="https://doi.org/10.13039/501100010779">CHU Sainte-Justine</a> and the Charles-Bruneau Foundations, <a href="https://doi.org/10.13039/501100000196">Canada Foundation for Innovation</a>, the <a href="https://doi.org/10.13039/501100000038">National Sciences and Engineering Research Council</a> (NSERC) (#RGPIN-2020-05232), and the Canadian Institutes of Health Research (CIHR) (#174924). K.A.K. is a recipient of IVADO’s postdoctoral scholarship (#3879287150). C.L. is currently supported by a <a href="https://doi.org/10.13039/501100000925">National Health and Medicine Research Council</a> (NHMRC) of Australia CJ Martin Early Career Research Fellowship (1143366). A.W.P. is supported by a <a href="https://doi.org/10.13039/501100000925">NHMRC</a> Principal Research fellowship (1137739). P.F. is supported by a <a href="https://doi.org/10.13039/100008018">Victorian Cancer Agency</a> (Australia) Mid-Career Fellowship.</p>
<h3>Author contributions</h3>
<p>R code, conceptualization, and authorship, F.S. and P.K.; validation and conceptualization, Q.M., D.J.H., K.A.K., and I.S.; conceptualization and authors of the original workflow, P.F., C.L., and A.W.P.; conceptualization and authorship, E.C.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>