<article>
<section>
<h1 id="header">Glycomics-informed glycoproteomic analysis of site-specific glycosylation for SARS-CoV-2 spike protein</h1>
<p><time datetime="2020-12-18">Published: December 18, 2020</time></p>
<p>Katelyn E. Rosenbalm,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Michael Tiemeyer,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Lance Wells,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Kazuhiro Aoki,<sup><a href="#aff1">1</a>,<a href="#cor1">*</a></sup> and Peng Zhao<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Complex Carbohydrate Research Center, University of Georgia, Athens, GA 30602, USA</p>
<p id="aff2"><sup>2</sup>Department of Biochemistry and Molecular Biology, University of Georgia, Athens, GA 30602, USA</p>
<p id="aff3"><sup>3</sup>Department of Chemistry, University of Georgia, Athens, GA 30602, USA</p>
<p id="fn1"><sup>4</sup>Technical Contact</p>
<p id="fn2"><sup>5</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kaoki@ccrc.uga.edu">kaoki@ccrc.uga.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:pengzhao@uga.edu">pengzhao@uga.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100214">10.1016/j.xpro.2020.100214</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes an integrated approach for analyzing site-specific N- and O-linked glycosylation of SARS-CoV-2 spike protein by mass spectrometry. Glycoproteomics analyzes intact glycopeptides to examine site-specific microheterogeneity of glycoproteins. Glycomics provides structural characterization on any glycan assignments by glycoproteomics. This procedure can be modified and applied to a variety of N- and/or O-linked glycoproteins. Combined with bioinformatics, the glycomics-informed glycoproteomics may be useful in generating 3D molecular dynamics simulations of certain glycoproteins alone or interacting with one another.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib10">Zhao et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Glycan structures are characterized by glycomic analysis of released glycans</li>
<li>Site-specific glycan microheterogeneity is characterized by glycoproteomic analysis</li>
<li>This protocol has broad applicability for a variety of glycoproteins</li>
<li>Bioinformatics and protein 3D structures can generate molecular dynamic simulations</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/329-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used in a recent publication (<a href="#bib10">Zhao et al., 2020</a>) to characterize site-specific microheterogeneity of glycosylation for a recombinant trimer SARS-CoV-2 spike mimetic immunogen and for a soluble version of human ACE2. The analysis quantitated the site-specific N-linked and O-linked glycosylation for SARS-CoV-2 spike as well as human ACE2 proteins. In combination with bioinformatic analyses of natural variants and with existing 3D-structures of both glycoproteins, the results generated molecular dynamics simulations of each glycoprotein alone and interacting with one another and highlighted roles for glycans in sterically masking polypeptide epitopes and directly modulating Spike-ACE2 interactions. Additionally, this protocol has broad applicability for a variety of glycoproteins that can be enriched and purified.</p>
<p>Prior to the experiment, clean glassware and plastic sample tubes, prepare solvents and reaction buffers, and store them as stock. These may be done several months in advance with the buffers stored at room temperature (20°C–25°C) or 4°C. Protein reduction, alkylation reagents, and proteases should be added to the buffers right before their respective steps of the experiments.</p>
<h3 id="sec1.1">Prepare clean glassware and plastic tubes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 h</div>
<ol>
<li>Prepare ultrapure water by passing deionized water through 0.22-μm Millipak membrane filter.</li>
<li>Soak all glassware (glass pipettes, bottles, and tubes) in ultrapure water, sonicate for 45 min, and discard water.</li>
<li>Soak all glassware in 50% methanol, sonicate for 45 min, and discard methanol.</li>
<li>Soak all glassware in 100% methanol, sonicate for 45 min, and discard methanol.</li>
<li>Air dry all glassware.</li>
<li>Rinse all plastic sample tubes with 100% methanol.</li>
<li>Air dry all plastic sample tubes.</li>
</ol>
<h3 id="sec1.2">Prepare solvents and reaction buffers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="8">
<li>Prepare 10 mL of trypsin reaction buffer: 100 mM Tris-HCl, 10 mM calcium chloride, pH 8.2. Store in a clean glass bottle at room temperature (20°C–25°C).</li>
<li>Prepare 10 mL of PNGase F reaction buffer: 100 mM sodium phosphate, pH 7.5. Store in a clean glass bottle at room temperature (20°C–25°C).</li>
<li>Activate cation exchange resin:
<ol type="a">
<li>Soak 20 g of cation exchange resin in 200 mL of 1 M hydrochloric acid.</li>
<li>Decant the supernatant and wash the resin with 200 mL of ultrapure water twice.</li>
<li>Remove water and soak the resin in 200 mL of 1 M sodium hydroxide.</li>
<li>Decant the supernatant and wash the resin with 200 mL of ultrapure water twice.</li>
<li>Remove water and soak the resin in 200 mL of 1 M hydrochloric acid.</li>
<li>Decant the supernatant and wash the resin with ultrapure water till its pH becomes neutral.</li>
</ol>
</li>
<li>Add 5% of acetic acid to the resin to make 50/50 (v/v) slurry.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use ultrapure water to prepare for any solvents and reaction buffers. All glassware and plastic tubes need to be pre-washed and dried.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>SARS-CoV-2 S protein</td>
<td>(<a href="#bib10">Zhao et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO), anhydrous</td>
<td>Sigma-Aldrich</td>
<td>276855-1L</td>
</tr>
<tr>
<td>Sodium hydroxide solution, 50% w/w</td>
<td>Fisher</td>
<td>SS254-1</td>
</tr>
<tr>
<td>Sodium borohydride</td>
<td>Sigma-Aldrich</td>
<td>213462-25G</td>
</tr>
<tr>
<td>Methyl iodine</td>
<td>Sigma-Aldrich</td>
<td>289566-100G</td>
</tr>
<tr>
<td>Methanol, anhydrous</td>
<td>Sigma-Aldrich</td>
<td>322415-100ML</td>
</tr>
<tr>
<td>Dichloromethane</td>
<td>Sigma-Aldrich</td>
<td>34856-4L</td>
</tr>
<tr>
<td>Methanol, HPLC grade</td>
<td>Sigma-Aldrich</td>
<td>34860-4L-R</td>
</tr>
<tr>
<td>Acetonitrile, HPLC grade</td>
<td>Sigma-Aldrich</td>
<td>34998-4L</td>
</tr>
<tr>
<td>2-Propanol, HPLC grade</td>
<td>Sigma-Aldrich</td>
<td>34863-4L</td>
</tr>
<tr>
<td>Acetic acid, glacial</td>
<td>Fisher</td>
<td>A38-500</td>
</tr>
<tr>
<td>Hydrochloric acid, 37%, ACS reagent</td>
<td>Sigma-Aldrich</td>
<td>320331</td>
</tr>
<tr>
<td>Dithiothreitol</td>
<td>Sigma-Aldrich</td>
<td>43815</td>
</tr>
<tr>
<td>Iodoacetamide</td>
<td>Sigma-Aldrich</td>
<td>I1149</td>
</tr>
<tr>
<td>Trypsin</td>
<td>Promega</td>
<td>V5111</td>
</tr>
<tr>
<td>Lys-C</td>
<td>Promega</td>
<td>V1671</td>
</tr>
<tr>
<td>Arg-C</td>
<td>Promega</td>
<td>V1881</td>
</tr>
<tr>
<td>Glu-C</td>
<td>Promega</td>
<td>V1651</td>
</tr>
<tr>
<td>Asp-N</td>
<td>Promega</td>
<td>VA1160</td>
</tr>
<tr>
<td>Endoglycosidase H</td>
<td>Promega</td>
<td>V4871</td>
</tr>
<tr>
<td>PNGaseF</td>
<td>Promega</td>
<td>V4831</td>
</tr>
<tr>
<td>Chymotrypsin</td>
<td>Athens Research and Technology</td>
<td>16-19-030820</td>
</tr>
<tr>
<td>Alpha lytic protease</td>
<td>New England BioLabs</td>
<td>P8113</td>
</tr>
<tr>
<td>Water (17O, 10%)</td>
<td>Cambridge Isotope Laboratories</td>
<td>OLM-782-10-1</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>pGlyco v2.2.2</td>
<td>(<a href="#bib5">Liu et al., 2017</a>)</td>
<td><a href="http://pfind.ict.ac.cn/software/pGlyco/index.html">http://pfind.ict.ac.cn/software/pGlyco/index.html</a></td>
</tr>
<tr>
<td>Byonic v3.8.13</td>
<td>Protein Metrics Inc. (<a href="#bib3">Bern et al., 2012</a>)</td>
<td><a href="https://www.proteinmetrics.com/products/byonic/">https://www.proteinmetrics.com/products/byonic/</a></td>
</tr>
<tr>
<td>GRITS Toolbox v1.1</td>
<td>(<a href="#bib8">Weatherly et al., 2019</a>)</td>
<td><a href="http://www.gritstoolbox.org">http://www.gritstoolbox.org</a></td>
</tr>
<tr>
<td>GlyGen v1.5</td>
<td>(<a href="#bib9">York et al., 2020</a>)</td>
<td><a href="https://www.glygen.org">https://www.glygen.org</a></td>
</tr>
<tr>
<td>GNOme v1.5.5</td>
<td>OBO Foundry</td>
<td><a href="https://github.com/glygen-glycan-data/GNOme/blob/master/README.md">https://github.com/glygen-glycan-data/GNOme/blob/master/README.md</a></td>
</tr>
<tr>
<td>GlyTouCan v3.1.0</td>
<td>(<a href="#bib7">Tiemeyer et al., 2017</a>)</td>
<td><a href="https://glytoucan.org">https://glytoucan.org</a></td>
</tr>
<tr>
<td>Xcalibur v2.0</td>
<td>Thermo Fisher Scientific</td>
<td><a href="https://www.thermofisher.com/order/catalog/product/OPTON-30965">https://www.thermofisher.com/order/catalog/product/OPTON-30965</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Millipak Filter</td>
<td>Millipore</td>
<td>C85052</td>
</tr>
<tr>
<td>Sep-Pak C18 Vac cartridge ,1 mL (100 mg sorbent)</td>
<td>Waters</td>
<td>WAT023590</td>
</tr>
<tr>
<td>2-mL screw cap microcentrifuge tube</td>
<td>USA Scientific</td>
<td>1420-9700</td>
</tr>
<tr>
<td>1.5-mL microcentrifuge tubes</td>
<td>USA Scientific</td>
<td>1615-5500</td>
</tr>
<tr>
<td>PYREX 13 × 100 mm disposable round bottom threaded culture tubes</td>
<td>Corning</td>
<td>99447-13</td>
</tr>
<tr>
<td>Phenolic caps with PTFE-faced rubber liner</td>
<td>Kimble</td>
<td>45066C-13</td>
</tr>
<tr>
<td>AG 50W-X8 cation exchange resin, hydrogen form, 100–200 mesh</td>
<td>Bio-Rad</td>
<td>142-1441</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<ul>
<li>5% Acetic acid stock solution</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetic acid, glacial</td>
<td>5%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>95%</td>
<td>950 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean glass bottle at room temperature (20°C–25°C).</div>
<ul>
<li>10% Acetic acid stock solution</li>
</ul>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetic acid, glacial</td>
<td>10%</td>
<td>100 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>90%</td>
<td>900 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean glass bottle at room temperature (20°C–25°C).</div>
<ul>
<li>20% 2-Propanol stock solution</li>
</ul>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>2-Propanol, HPLC grade</td>
<td>20%</td>
<td>20 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>80%</td>
<td>80 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean glass bottle at room temperature (20°C–25°C).</div>
<ul>
<li>40% 2-Propanol stock solution</li>
</ul>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>2-Propanol, HPLC grade</td>
<td>40%</td>
<td>40 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>60%</td>
<td>60 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean glass bottle at room temperature (20°C–25°C).</div>
<ul>
<li>50 mM Sodium hydroxide stock solution</li>
</ul>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sodium hydroxide solution, 50% w/w</td>
<td>50 mM</td>
<td>0.13 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>49.87 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean plastic tube at 4°C.</div>
<div class="note">
<span class="note-title">Note:</span> The sodium hydroxide solution (50%, w/w) is viscous and needs to be treated with care.</div>
<ul>
<li>1 M Hydrochloric acid stock solution</li>
</ul>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hydrochloric acid, 37%, ACS reagent</td>
<td>1 M</td>
<td>82 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>918 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean glass bottle at room temperature (20°C–25°C).</div>
<ul>
<li>1 M Sodium hydroxide stock solution</li>
</ul>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sodium hydroxide solution, 50% w/w</td>
<td>1 M</td>
<td>53 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>947 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean plastic bottle at 4°C.</div>
<div class="note">
<span class="note-title">Note:</span> The sodium hydroxide solution (50%, w/w) is viscous and needs to be treated with care.</div>
<ul>
<li>10% Acetic acid in methanol stock solution</li>
</ul>
<table id="undtbl8">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetic acid, glacial</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Methanol, HPLC grade</td>
<td>90%</td>
<td>45 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean glass bottle at room temperature (20°C–25°C).</div>
<ul>
<li>1 mM Sodium hydroxide in 50% methanol stock solution</li>
</ul>
<table id="undtbl9">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sodium hydroxide solution, 50% w/w</td>
<td>1 mM</td>
<td>5.3 μL</td>
</tr>
<tr>
<td>Methanol, HPLC grade</td>
<td>50%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>50%</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store in a clean glass bottle at room temperature (20°C–25°C).</div>
<div class="note">
<span class="note-title">Note:</span> The sodium hydroxide solution (50%, w/w) is viscous and needs to be treated with care.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Trypsin digestion of glycoproteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>It is crucial to use pre-washed, clean glassware such as glass tubes and glass pipettes as well as clean plastic tubes in order to reduce the amount of contaminant during sample preparation. Those contaminants may interfere with glycan profiling by MS analysis. The product from trypsin digestion contains peptides, salts, and other contaminants that may interfere with the subsequent PNGase F digestion and therefore needs to be purified.</p>
<ol>
<li>Prepare 2 mg/mL trypsin solution in trypsin reaction buffer and mix on vortex mixer.</li>
<li>Reconstitute the purified glycoproteins (25 μg of SARS-CoV-2 spike protein) in 200 μL trypsin buffer.</li>
<li>Heat the sample for 5 min at 98°C prior to digestion.</li>
<li>Cool down on ice and add 25 μL trypsin solution to the sample</li>
<li>Incubate the sample at 37°C for 18 h.</li>
<li>Stop the reaction by boiling the sample for 5 min.</li>
<li>Cool down on ice and add equal volume (225 μL) of 10% acetic acid to the sample.</li>
<li>Equilibrate a C18 cartridge with 3 mL of 100% acetonitrile, followed by 3 mL of 5% acetic acid.</li>
<li>Load the sample onto the cartridge, and discard the flow-through.</li>
<li>Rinse the sample tube with 0.5 mL of 5% acetic acid for three times, apply each rinse to the cartridge, and discard the flow-through.</li>
<li>Wash the cartridge with an additional 5 mL of 5% acetic acid, and discard the flow-through.</li>
<li>Elute and collect glycopeptides into three separate 2-mL screw cap tubes:
<ol type="a">
<li>First elution with 2 mL of 20% 2-propanol</li>
<li>Second elution with 2 mL of 40% 2-propanol</li>
<li>Third elution with 2 mL of 100% 2-propanol</li>
</ol>
</li>
<li>Evaporate water and 2-propanol with a vacuum concentrator.</li>
<li>Reconstitute the dried second elution (40% 2-propanol elution) in 0.5 mL of 5% acetic acid and pooled with the first elution (20% 2-propanol elution). The third elution (100% 2-propanol) can be used to prepare O-linked glycans.</li>
<li>Evaporate water and acetic acid with a vacuum concentrator.</li>
<li>Store the dry glycopeptides at −20°C until use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It can be helpful to briefly vortex digestion solution every 15 min for the first hour.</div>
<div class="note">
<span class="note-title">Note:</span> Use glass pipettes for any organic solvents and acid solutions.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dry glycopeptides can be stored at −20°C for 3–4 weeks.</div>
<h3 id="sec3.2">Isolating N-linked glycans released by PNGase F digestion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<p>N-linked glycans are released from glycopeptides by enzymatic digestion with PNGase F and purified using C18 cartridges. The purified N-linked glycans are ready for permethylation.</p>
<ol start="17">
<li>Reconstitute dry glycopeptides in 20 μL PNGase F reaction buffer with vortexing and sonication.</li>
<li>Add 28 μL of ultrapure water and adjust pH to 7.5 with additional PNGase F reaction buffer if necessary. Adjust total volume of the sample by evaporating excessive buffer with a vacuum concentrator.</li>
<li>Add 2 μL PNGase F (0.75 mg/mL) and incubate for 18 h at 37°C.</li>
<li>After incubation, evaporate the reaction with a vacuum concentrator.</li>
<li>Reconstitute the sample in 0.2 mL of 5% acetic acid.</li>
<li>Equilibrate a C18 cartridge with 3 mL of 100% acetonitrile, followed by 3 mL of 5% acetic acid.</li>
<li>Place a clean screw cap glass tube under the cartridge, load the sample, and collect the flow-through.</li>
<li>Rinse the sample tube with 0.2 mL of 5% acetic acid for three times, apply each rinse to the cartridge and collect the flow-through.</li>
<li>Elute with an additional 1.2 mL of 5% acetic acid and collect for a total elution volume of 2 mL.</li>
<li>Elute and collect residual glycopeptides with 2 mL of 100% 2-propanol.</li>
<li>Dry the purified glycans by lyophilization.</li>
<li>Combine elution from steps 12c and 26 in a glass tube with screw top, and dry under nitrogen stream. The combined glycopeptides can be used to prepare O-linked glycans.</li>
<li>Store the dry glycans and glycopeptides at −20°C until use. (See <a href="#sec7.1">Troubleshooting 1</a>)</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use glass pipettes for any organic solvents and acid solutions.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dry glycans and glycopeptides can be stored at −20°C for 3–4 weeks.</div>
<h3 id="sec3.3">Releasing O-linked glycans by reductive β-elimination</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<p>O-linked glycans are released from deglycosylated glycopeptides obtained from steps 1–29 by reductive β-elimination. After reaction, the product contains a large amount of sodium and borate salts as well as small peptides and other contaminants, and therefore needs to be purified. Following desalting and C18 purification, O-linked glycans are ready for permethylation.</p>
<ol start="30">
<li>Preparation of reaction reagents for reductive β-elimination.
<ol type="a">
<li>Prepare 1 M sodium borohydride in 50 mM sodium hydroxide in a glass tube.</li>
<li>Add 0.3 mL of the sodium borohydride solution to the O-linked glycopeptides from step 27.</li>
<li>Cap the glass tube with a PTFE-lined screw cap and dissolve the glycopeptides by sonication for 30 s.</li>
</ol>
</li>
<li>Incubate at 45°C for 18 h.</li>
<li>Remove the glass tube from incubation and place it on ice for 10 min.</li>
<li>Keep the tube on ice while adding 10% acetic acid dropwise into the sample solution until bubbling stops. Vortex and centrifuge the tube repeatedly to avoid any spillover caused by bubbling.</li>
<li>Prepare a small glass column by breaking the tip of a Pasteur glass pipette using a ceramic cutter.</li>
<li>Plug the bottom of the glass column with some glass wool, and transfer the activated resin to the glass column (1 mL bed volume).</li>
<li>Wash the resin with 5 mL of 5% acetic acid.</li>
<li>Place a clean glass tube under the glass column for collection, load the sample onto the column, and collect the flow-through.</li>
<li>Elute glycans with additional 3 mL of 5% acetic acid and collect the flow-through.</li>
<li>Dry the glycans by lyophilization.</li>
<li>Add 200–300 μL of 10% acetic acid in methanol to the dried glycans. Evaporate under nitrogen stream at 37°C.</li>
<li>Repeat step 40 for three to five times.</li>
<li>Reconstitute the dry glycans in 200 μL of 5% acetic acid.</li>
<li>Equilibrate a C18 cartridge with 3 mL of 100% acetonitrile, followed by 5 mL of 5% acetic acid.</li>
<li>Place a clean glass tube under the cartridge for collection, load the sample, and collect the flow-through.</li>
<li>Elute with additional 3 mL of 5% acetic acid and collect the flow-through.</li>
<li>Dry the glycans by lyophilization.</li>
<li>Store the dry glycans at −20°C until use. (See <a href="#sec7.3">Troubleshooting 2</a>)</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use glass pipettes for any organic solvents and acid solutions.</div>
<div class="note">
<span class="note-title">Note:</span> When terminating the reductive β-elimination with 10% acetic acid, the neutralizing reaction can generate a large volume of bubbles in a short period of time and cause spills. Therefore, the acetic acid needs to be slowly added dropwise with intermittent vortexing and centrifugation.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dry glycans can be stored at −20°C for 3–4 weeks.</div>
<h3 id="sec3.4">Permethylation of purified N- and O-linked glycans</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<p>Following lyophilization, the purified dry glycans are permethylated for MS analysis. (See <a href="#sec7.5">Troubleshooting 3</a>)</p>
<ol start="48">
<li>Prepare the base solution for permethylation
<ol type="a">
<li>Transfer 300 μL of sodium hydroxide solution (50%, w/w) to a clean glass tube with screw top.</li>
<li>Add 600 μL of anhydrous methanol to the same tube and vortex for 30 s.</li>
<li>Add 4 mL of anhydrous dimethyl sulfoxide and vortex for 30 s.</li>
<li>Centrifuge for 30 to 60 s.</li>
<li>Remove and discard the supernatant with a glass pipette.</li>
<li>Repeat steps c–e for three to five times until the base slurry becomes translucent.</li>
<li>Dissolve the precipitated base in 3 mL of anhydrous dimethyl sulfoxide by pipetting with a new clean glass pipette</li>
</ol>
</li>
<li>Permethylate the dry glycans
<ol type="a">
<li>Add 100 μL of anhydrous dimethyl sulfoxide to the dry glycans with a glass pipette and sonicate for 30 s to dissolve the glycans.</li>
<li>Transfer 200 μL of the base solution to the glycans with a glass pipette.</li>
<li>Add 50 μL of methyl iodine quickly with a glass syringe.</li>
<li>Vortex vigorously for 5 min.</li>
<li>Place the sample tube on ice.</li>
<li>Add 2 mL of dichloromethane to the glycans with a clean glass pipette.</li>
<li>Add 2 mL of ultrapure water to the glycans with a glass pipette.</li>
<li>Vortex for 30 s, and centrifuge for 30 s.</li>
<li>Remove and discard the supernatant (the water phase).</li>
<li>Repeat steps g–i for three to five times.</li>
<li>After removing water phase as much as possible, transfer the lower phase (organic phase) carefully with a new clean glass pipette to a new glass tube.</li>
<li>Dry the sample solution under nitrogen stream at 37°C.</li>
</ol>
</li>
<li>Store the permethylated glycans at −20°C until use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use plastic pipettes for the sodium hydroxide solution.</div>
<div class="note">
<span class="note-title">Note:</span> Use glass pipettes for any organic solvents and acid solutions.</div>
<div class="note">
<span class="note-title">Note:</span> The base solution can be stored up to 6 h at room temperature (20°C–25°C).</div>
<div class="note">
<span class="note-title">Note:</span> Methyl iodine used in permethylation is stored at 4°C and needs to sit at room temperature (20°C–25°C) for at least 30 min prior to the reaction.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Permethylated glycans can be stored at −20°C for 3–4 months.</div>
<h3 id="sec3.5">Analyzing N- and O-linked glycans by mass spectrometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>The permethylated glycans are reconstituted and analyzed by directly infused into a mass spectrometer. Data annotation and assignment of glycan accession identifiers are performed manually and can be facilitated by software.</p>
<ol start="51">
<li>Reconstitute permethylated glycans in 100% methanol: glycans derived from the equivalent of 5–15 μg of purified glycoproteins are reconstituted in 30–50 μL of 100% methanol.</li>
<li>Transfer 10 μL of the reconstituted glycans to a small glass vial using a clean glass syringe.</li>
<li>Add 40 μL of 1 mM sodium hydroxide or sodium acetate in 50% methanol to the glycans and mix by vortexing.</li>
<li>Glycans are loaded into a glass syringe, directly infused into the nano-electrospray ion source of an LTQ-Orbitrap Discovery mass spectrometer (Thermo Fisher Scientific), and analyzed by MS/MS.
<ol type="a">
<li>The flow rate is set at 0.4–0.6 μL/min and the capillary temperature is set at 210°C.</li>
<li>The total ion mapping (TIM) functionality of the Xcalibur software package (v 2.0, Thermo Fisher Scientific) is utilized to detect and quantify the prevalence of individual glycans in the total glycan profile. Through TIM, automated MS and MS/MS spectra (at 30–40% normalized collision energy via collision induced dissociation) are acquired in collection windows that are 2.8 mass units in width. Five scans, each 150 ms in duration, are averaged for each collection window. The m/z range from 500 to 2,000 is scanned in successive 2.8 mass unit windows with a window-to-window overlap of 2 mass units.</li>
</ol>
</li>
<li>Glycan species are identified by manual interpretation of the raw spectra as well as by using tools such as GRITS Toolbox (<a href="#bib8">Weatherly et al., 2019</a>), GlyGen (<a href="#bib9">York et al., 2020</a>), GNOme (OBO Foundry), and GlyTouCan (<a href="#bib1">Aoki-Kinoshita et al., 2016</a>). Manual interpretation of glycan mass spectra may be carried out as reported previously (<a href="#bib2">Ashline et al., 2014</a>). N-linked glycan peaks of all charge states are deconvoluted by charge state and summed for quantification using the Xtract functionality of the Xcalibur software package (v 2.0, Thermo Fisher Scientific). O-linked glycan peaks of all charges are manually collected and summed for quantification.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of glycans required for MS analysis depends on sample quality, such as the purity and the origin of the sample (tissue or cell types). Information on the level of glycosylation (the number of glycosylation sites and their occupancy) of the sample is also very important and sometimes unknown. The amount of injection may need to be adjusted to enhance quality of MS profile (<a href="#bib6">Mehta et al., 2016</a>).</div>
<div class="note">
<span class="note-title">Note:</span> In TIM, the 2.8 mass unit window allows signals from the naturally occurring isotopes of individual glycans to be summed into a single response, increasing detection sensitivity for minor structures. The 2 mass unit overlap ensures that minor glycans, whose masses place them at the edge of an individual window, would be sampled in a representative fashion.</div>
<div class="note">
<span class="note-title">Note:</span> For detail MS<sup>n</sup> analysis to determine glycan linkage, 1 mM of lithium acetate in 50% methanol is preferred.</div>
<div class="note">
<span class="note-title">Note:</span> Manual interpretation of glycan spectra can be facilitated by the resources provided by Expasy (<a href="http://www.expasy.org">www.expasy.org</a>). The computational tools provided via Expasy can help calculating the m/z values of unmodified peptides, peptides modified with glycans, and released glycans with and/or without derivatization.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Permethylated glycans can also be analyzed by MALDI-TOF-MS.</div>
<h3 id="sec3.6">Preparing intact N-linked glycopeptides</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Preparing intact N-linked glycopeptides for mass spectrometry analysis.</p>
<ol start="56">
<li>Prepare four aliquots of SARS-CoV-2 spike protein (20–100 pmol for each aliquot) and reduce the proteins by incubating each aliquot with 10 mM of dithiothreitol at 56°C for 1 h.</li>
<li>Alkylate each aliquot by incubating with 27.5 mM of iodoacetamide at room temperature (20°C–25°C) in dark for 45 min.</li>
<li>Digest four aliquots of proteins by incubating them with respective proteases:
<ol type="a">
<li>Incubate aliquot #1 with alpha lytic protease at an enzyme-to-protein ratio of 1:30 (w/w) at 37°C for 18 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
<li>Incubate aliquot #2 with chymotrypsin at a ratio of 1:20 (w/w) at 25°C for 8 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
<li>Incubate aliquot #3 with trypsin at a ratio of 1:20 (w/w) at 37°C for 18 h, and deactivate the enzyme by incubating at 100°C for 5 min.
<ol type="i">
<li>Incubate aliquot #3 with Glu-C at a ratio of 1:20 (w/w) at 37°C for 18 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
</ol>
</li>
<li>Incubate aliquot #4 with Glu-C at a ratio of 1:20 (w/w) at 37°C for 18 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
<li>Incubate aliquot #4 with Asp-N at a ratio of 1:20 (w/w) at 37°C for 8 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
</ol>
</li>
<li>Acidify and dry down each aliquot in a vacuum concentrator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For choosing proper proteases to be used in the digestion of the target protein, inspect the protein sequence for cleavable amino acids as well as the N-linked glycosylation sequons (Asn-x-Ser/Thr/Cys, Asn-Gly-x sequences) and choose the proteases that can generate reasonable sized peptides (5 to 15 amino acids) containing the sequons for mass spectrometry analysis.</div>
<h3 id="sec3.7">Analyzing intact N-linked glycopeptides by mass spectrometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 weeks</div>
<p>Mass spectrometry analysis of intact N-linked glycopeptides.</p>
<ol start="60">
<li>Reconstitute each aliquot in 0.1% formic acid, inject approximately 20–30 pmol of each aliquot, and analyze by LC-MS/MS.
<ol type="a">
<li>Separate peptides of each aliquot on an Acclaim PepMap RSLC C18 column (75 μm × 15 cm) while eluting into the nano-electrospray ion source of an Orbitrap Fusion Lumos Tribrid mass spectrometer (Thermo Fisher Scientific) at a flow rate of 0.2 μL/min. The elution gradient consists of 1%–40% acetonitrile in 0.1% formic acid over 370 min followed by 10 min of 80% acetonitrile in 0.1% formic acid.</li>
<li>Full MS scans are acquired from m/z 200 to 2,000 at 60k resolution, and MS/MS scans following higher-energy collisional dissociation (HCD) with stepped collision energy (15%, 25%, 35%) are collected in the orbitrap at 15k resolution.</li>
</ol>
</li>
<li>Analyze the raw spectra by pGlyco (<a href="#bib5">Liu et al., 2017</a>) and Byonic (<a href="#bib3">Bern et al., 2012</a>) followed by manual validation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Injection volumes depend on the sample loop equipped with individual LC instruments. For our particular LC instrument, the injection loop is 20-μL in volume, so our injection volume was approximately 17 μL.</div>
<div class="note">
<span class="note-title">Note:</span> Injection amounts depend on the capacity of the analytical columns equipped with individual LC instruments. For our particular LC instrument, the analytical column performs best at approximately 20–30 pmol of peptides (3–4 μg of SARS-CoV-2 spike protein).</div>
<div class="note">
<span class="note-title">Note:</span> Manual interpretation of glycopeptide spectra can be facilitated by the resources provided by Expasy (<a href="http://www.expasy.org">www.expasy.org</a>). The computational tools provided via Expasy can help calculating the m/z values of unmodified peptides, peptides modified with glycans, and released glycans with and/or without derivatization.</div>
<h3 id="sec3.8">Preparing intact O-linked glycopeptides</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>Preparing intact O-linked glycopeptides for mass spectrometry analysis.</p>
<ol start="62">
<li>Prepare three aliquots of SARS-CoV-2 spike protein (20–100 pmol for each aliquot) and reduce the proteins by incubating each aliquot with 5 mM of dithiothreitol at 56°C for 1 h.</li>
<li>Alkylate each aliquot by incubating with 13.75 mM of iodoacetamide at room temperature (20°C–25°C) in dark for 45 min.</li>
<li>Digest three aliquots of proteins by incubating them with respective proteases:
<ol type="a">
<li>Incubate aliquot #1 with Lys-C/trypsin mix at an enzyme-to-protein ratio of 1:20 (w/w) at 37°C for 18 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
<li>Incubate aliquot #2 with Lys-C at a ratio of 1:20 (w/w) at 37°C for 18 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
<li>Incubate aliquot #3 with Arg-C at a ratio of 1:20 (w/w) at 37°C for 18 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
</ol>
</li>
<li>Deglycosylate each aliquot by incubating with PNGase F at 37°C for 40 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
<li>Perform a second digest of three aliquots by incubating them with respective proteases:
<ol type="a">
<li>Incubate aliquot #1 with Lys-C/trypsin mix at an enzyme-to-protein ratio of 1:20 (w/w) at 37°C for 8 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
<li>Incubate aliquot #2 with Lys-C at a ratio of 1:20 (w/w) at 37°C for 8 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
<li>Incubate aliquot #3 with Arg-C at a ratio of 1:20 (w/w) at 37°C for 8 h, and deactivate the enzyme by incubating at 100°C for 5 min.</li>
</ol>
</li>
<li>Acidify and dry down each aliquot in a vacuum concentrator.</li>
</ol>
<h3 id="sec3.9">Analyzing intact O-linked glycopeptides by mass spectrometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 weeks</div>
<p>Mass spectrometry analysis of intact O-linked glycopeptides.</p>
<ol start="68">
<li>Reconstitute each aliquot in 0.1% formic acid, inject approximately 20–30 pmol of each aliquot, and analyze by LC-MS/MS.
<ol type="a">
<li>Separate peptides of each aliquot on an Acclaim PepMap RSLC C18 column (75 μm × 15 cm) while eluting into the nano-electrospray ion source of an Orbitrap Fusion Lumos Tribrid mass spectrometer (Thermo Fisher Scientific) at a flow rate of 0.2 μL/min. The elution gradient consists of 1%–40% acetonitrile in 0.1% formic acid over 370 min followed by 10 min of 80% acetonitrile in 0.1% formic acid.</li>
<li>Full MS scans are acquired from m/z 200 to 2,000 at 60k resolution, and MS/MS scans following higher-energy collisional dissociation (HCD) with stepped collision energy (15%, 25%, 35%) or electron transfer dissociation (ETD) are collected in the orbitrap at 15k resolution.</li>
</ol>
</li>
<li>Analyze raw spectra by Byonic (<a href="#bib3">Bern et al., 2012</a>) followed by manual validation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Injection volumes depend on the sample loop equipped with individual LC instruments. For our particular LC instrument, the injection loop is 20-μL in volume, so our injection volume was approximately 17 μL.</div>
<div class="note">
<span class="note-title">Note:</span> Injection amounts depend on the capacity of the analytical columns equipped with individual LC instruments. For our particular LC instrument, the analytical column performs best at approximately 20–30 pmol of peptides (3–4 μg of SARS-CoV-2 spike protein).</div>
<div class="note">
<span class="note-title">Note:</span> Manual interpretation of glycopeptide spectra can be facilitated by the resources provided by Expasy (<a href="http://www.expasy.org">www.expasy.org</a>). The computational tools provided via Expasy can help calculating the m/z values of unmodified peptides, peptides modified with glycans, and released glycans with and/or without derivatization.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol is designed to fully characterize the N-linked and O-linked glycosylation of a purified glycoprotein and was optimized for SARS-CoV-2 spike protein. The glycomic analysis of released glycans allows for the elucidation of glycan structures while providing a quantitative glycan profile of the target glycoprotein. (See <a href="#sec7.7">Troubleshooting 4</a>) The glycoproteomic analysis characterizes and quantitates site-specific glycan topologies and provides insights into the roles of glycosylation microheterogeneity. By integrating the cross-validating glycomic and glycoproteomic analyses, the use of this protocol will provide a detailed understanding of the glycosylation states of the target glycoprotein and facilitate exploring a variety of essential roles for glycosylation. (See <a href="#sec7.9">Troubleshooting 5</a>)</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>For glycomic analysis of released glycans, process the raw spectra by deconvolution and manual interpretation. The explicit identities of individual monosaccharide residues are assigned based on known human biosynthetic pathways. Data annotation and assignment of glycan accession identifiers are performed manually and can be facilitated by GRITS Toolbox (<a href="#bib8">Weatherly et al., 2019</a>), GlyTouCan (<a href="#bib7">Tiemeyer et al., 2017</a>), GNOme (OBO Foundry), and GlyGen (<a href="#bib4">Kahsay et al., 2020</a>; <a href="#bib9">York et al., 2020</a>). Quantification are performed based on peak intensities. N-linked glycan peaks of all charge states are deconvoluted by charge state and summed for quantification using the Xtract functionality of the Xcalibur software package (v 2.0, Thermo Fisher Scientific). O-linked glycan peaks of all charges are manually collected and summed for quantification.</p>
<p>For glycoproteomic analysis of intact glycopeptides, perform database searches on the raw spectra using pGlyco v2.2.2 (<a href="#bib5">Liu et al., 2017</a>) and Byonic v3.8.13 (<a href="#bib3">Bern et al., 2012</a>). Search output from pGlyco and Byonic can be filtered to reach certain false discovery rate (1%–10%) followed by manual validation. To minimize the effect of ionization suppression suffered.by glycopeptides compared to non-glycopeptides, quantification of glycopeptides are performed based on spectral counts (measuring the base of the peak) instead of peak intensity or area (measuring the height of the peak). (See <a href="#sec7.11">Troubleshooting 6</a>)</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While this protocol can be applied to characterize a variety of glycoproteins individually, it may become less reliable when analyzing a mixture of glycoproteins. In glycomic analysis, glycans are released universally from the source material and cannot be correlate to any individual source proteins. Glycoproteomic analysis can differentiate individual proteins in a mixture to certain extent based on their respective sequences, but may fail to do so if the target proteins are highly conserved. Therefore, we highly recommend performing individual enrichment for each target protein by biochemical methods, such as immunoprecipitation, prior to starting this protocol.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low signal intensity of N-linked glycans in glycomic analysis (steps 21–29)</p>
<h3 id="sec7.2">Potential solution</h3>
<p>If high-intensity signals from contaminants are detected, then additional purification may need to be performed to remove the interferences. Before N-linked glycans are permethylated, a porous graphitized carbon column can be used to remove contaminants in addition to a C18 column. After permethylation, the permethylated glycans can be purified using a C18 column before being analyzed by a mass spectrometer. If no high-intensity signal is detected in general, then the amount of starting material may need to be increased.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Low signal intensity of O-linked glycans in glycomic analysis (steps 42–47)</p>
<h3 id="sec7.4">Potential solution</h3>
<p>If high-intensity signals from sources other than O-linked glycans are detected, such as N-linked glycans or other contaminants, then additional deglycosylation of N-linked glycans may need to be performed using PNGase F or PNGaseA as well as additional purification by C18 and/or porous graphitized carbon columns. If no high-intensity signal is detected in general, then the amount of starting material may need to be increased.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Single glycan species is observed as multiple peaks differed by 14 Da in glycomic analysis (steps 48–50).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>This problem is likely caused by under-permethylation of the released glycans and can be addressed by repeating the permethylation step (steps 48–50) for a second time. Additional purification of glycans prior to permethylation using C18 and/or porous graphitized carbon columns is also beneficial to reduce under-permethylation.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>High signal intensity of singly-charged polymer peaks are observed and target species (glycans or glycopeptides) are at very low intensity.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>The cause of this problem is likely polymer contamination. The source of polymer contamination may come from the glassware, plastic sample tubes, solvents, and buffers used in the sample preparation steps as well as the mass spectrometer. Make sure to clean all glassware and plastic tubes as described in the preparation steps and use HPLC grade reagents to prepare any solvents and buffers. If the source of contamination is in the mass spectrometer, replace any liquid transfer parts, and clean the ion transfer tube and lenses as recommended by the manufacturer.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Low signal intensity of glycopeptides in glycoproteomic analysis.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Make sure the lens radio frequency (RF Lens) is lower than 35% and the source fragmentation is turned off to avoid losing the glycan moieties during ionization. With proper ionization parameters, if the non-glycopeptides from the target protein are detected at reasonable intensity, then the target protein may not have been sufficiently denatured prior to initial proteolysis. A possible solution is to use mass spectrometer-compatible detergent and/or incubate the target proteins at 95°C for 5–10 min prior to reduction. If no peptides are detected at reasonable intensity, then the amount of starting material may need to be increased.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Potential glycans and glycopeptides are detected at reasonable signal intensity, but no confident identification from data analysis.</p>
<h3 id="sec7.12">Potential solution</h3>
<p>A possible cause is that the mass spectrometer is out of calibration. Make sure to perform regular cleaning and calibration of the mass spectrometer and perform test runs of standard glycans or peptides prior to analyzing target proteins.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aoki-Kinoshita, K., Agravat, S., Aoki, N.P., Arpinar, S., Cummings, R.D., Fujita, A., Fujita, N., Hart, G.M., Haslam, S.M., Kawasaki, T., et al. (2016). GlyTouCan 1.0--The international glycan structure repository. Nucleic Acids Res. <i>44</i>.</p>
<p id="bib2">Ashline, D.J., Hanneman, A.J., Zhang, H., and Reinhold, V.N. (2014). Structural documentation of glycan epitopes: sequential mass spectrometry and spectral matching. J. Am. Soc. Mass Spectrom. <i>25</i>, 444-453.</p>
<p id="bib3">Bern, M., Kil, Y.J., and Becker, C. (2012). Byonic: advanced peptide and protein identification software. Curr. Protoc. Bioinformatics <i>Chapter 13</i>, Unit 13 20.</p>
<p id="bib4">Kahsay, R., Vora, J., Navelkar, R., Mousavi, R., Fochtman, B.C., Holmes, X., Pattabiraman, N., Ranzinger, R., Mahadik, R., Williamson, T., et al. (2020). GlyGen data model and processing workflow. Bioinformatics <i>36</i>, 3941-3943.</p>
<p id="bib5">Liu, M.Q., Zeng, W.F., Fang, P., Cao, W.Q., Liu, C., Yan, G.Q., Zhang, Y., Peng, C., Wu, J.Q., Zhang, X.J., et al. (2017). pGlyco 2.0 enables precision N-glycoproteomics with comprehensive quality control and one-step mass spectrometry for intact glycopeptide identification. Nat. Commun. <i>8</i>, 438.</p>
<p id="bib6">Mehta, N., Porterfield, M., Struwe, W.B., Heiss, C., Azadi, P., Rudd, P.M., Tiemeyer, M., and Aoki, K. (2016). Mass spectrometric quantification of N-linked glycans by reference to exogenous standards. J. Proteome Res. <i>15</i>, 2969-2980.</p>
<p id="bib7">Tiemeyer, M., Aoki, K., Paulson, J., Cummings, R.D., York, W.S., Karlsson, N.G., Lisacek, F., Packer, N.H., Campbell, M.P., Aoki, N.P., et al. (2017). GlyTouCan: an accessible glycan structure repository. Glycobiology <i>27</i>, 915-919.</p>
<p id="bib8">Weatherly, D.B., Arpinar, F.S., Porterfield, M., Tiemeyer, M., York, W.S., and Ranzinger, R. (2019). GRITS Toolbox-a freely available software for processing, annotating and archiving glycomics mass spectrometry data. Glycobiology <i>29</i>, 452-460.</p>
<p id="bib9">York, W.S., Mazumder, R., Ranzinger, R., Edwards, N., Kahsay, R., Aoki-Kinoshita, K.F., Campbell, M.P., Cummings, R.D., Feizi, T., Martin, M., et al. (2020). GlyGen: computational and informatics resources for glycoscience. Glycobiology <i>30</i>, 72-73.</p>
<p id="bib10">Zhao, P., Praissman, J.L., Grant, O.C., Cai, Y., Xiao, T., Rosenbalm, K.E., Aoki, K., Kellman, B.P., Bridger, R., Barouch, D.H., et al. (2020). Virus-receptor interactions of glycosylated SARS-CoV-2 spike and human ACE2 receptor. Cell Host Microbe <i>28</i>, 586-601.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Peng Zhao (<a href="mailto:pengzhao@uga.edu">pengzhao@uga.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was funded by <a href="https://doi.org/10.13039/100000002">NIH</a> P41GM103490 (M.T. and L.W.), U01GM125267 (M.T.), and R01GM130915 (L.W.).</p>
<h3>Author Contributions</h3>
<p>K.E.R., K.A., and P.Z. contributed to the writing of the manuscript; K.E.R., L.W., K.A., and P.Z. edited the manuscript. Figures generated by P.Z. Initial optimization by P.Z. Lab infrastructure and resources by M.T. and L.W.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>