<article>
<section>
<h1 id="header">Protocol for intracellular and extracellular metabolite detection in human embryonic stem cells</h1>
<p><time datetime="2021-08-13">Published: August 13, 2021</time></p>
<p>Faxiang Xu,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Chengcheng Song,<sup><a href="#aff1">1</a></sup> Weiwei Liu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Guokai Chen<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor4">*</a></sup></p>
<p id="aff1"><sup>1</sup>Centre of Reproduction, Development and Aging, Faculty of Health Sciences, University of Macau, Macau, China</p>
<p id="aff2"><sup>2</sup>Bioimaging and Stem Cell Core Facility, Faculty of Health Sciences, University of Macau, Macau, China</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor4"><sup>*</sup>Correspondence: <a href="mailto:guokaichen@um.edu.mo">guokaichen@um.edu.mo</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100740">10.1016/j.xpro.2021.100740</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Metabolic homeostasis is critical for cell pluripotency and differentiation in human embryonic stem cells (hESCs). It has been reported that metabolic changes specifically regulate cellular signaling during hESC differentiation. This protocol describes procedures for both cell culture and detection of intracellular and extracellular metabolites in hESCs by liquid chromatography-mass spectrometry. Metabolites in glycolysis, citric acid cycle, pentose phosphate pathway, and other metabolic processes can be detected using this approach.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib9">Song et al., (2019</a>), <a href="#bib10">Yang et al., (2019</a>), <a href="#bib6">Meng et al., (2018</a>), and <a href="#bib3">Chen et al., (2011b</a>).</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol and tips for stem cell culture suitable for metabolic analysis</li>
<li>Protocol for semi-quantification of metabolic changes in the cell culture medium</li>
<li>Protocol for semi-quantification of intracellular metabolites in hESCs</li>
<li>MRM and SIR mass spectrometry parameters for common metabolites</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="note">
<span class="note-title">Note:</span> To develop multiple reaction monitoring (MRM) and selected ion recording (SIR) assays for LC-MS in this protocol, the specialty of a skilled mass spectrometrist is needed.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>Thermo Fisher</td>
<td>Cat#11330-032</td>
</tr>
<tr>
<td>UltraPure™ DNase/RNase-Free Distilled Water</td>
<td>Invitrogen</td>
<td>Cat#10977015</td>
</tr>
<tr>
<td>PBS (1X), pH 7.4</td>
<td>Gibco</td>
<td>Cat#10010023</td>
</tr>
<tr>
<td>DPBS (1X)</td>
<td>Gibco</td>
<td>Cat#14190144</td>
</tr>
<tr>
<td>TrypLE Select</td>
<td>Thermo Fisher</td>
<td>Cat#12563011</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Gibco</td>
<td>Cat#10270106</td>
</tr>
<tr>
<td>DMSO</td>
<td>Sigma</td>
<td>Cat#D4540</td>
</tr>
<tr>
<td>2-Phospho-L-ascorbic acid trisodium salt</td>
<td>Sigma-Aldrich</td>
<td>Cat#49752</td>
</tr>
<tr>
<td>Sodium selenite</td>
<td>Sigma-Aldrich</td>
<td>Cat#S5261</td>
</tr>
<tr>
<td>Human holo-Transferrin</td>
<td>Sigma-Aldrich</td>
<td>Cat#T0665</td>
</tr>
<tr>
<td>Insulin</td>
<td>Sigma</td>
<td>Cat#I9278</td>
</tr>
<tr>
<td>Recombinant Human TGF-β1</td>
<td>PeproTech</td>
<td>Cat#100-21</td>
</tr>
<tr>
<td>Recombinant human serum albumin</td>
<td>Sigma</td>
<td>Cat#A9731</td>
</tr>
<tr>
<td>Recombinant Human FGF2</td>
<td>(<a href="#bib4">Chen et al., 2012</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Matrigel</td>
<td>Corning</td>
<td>Cat#354230</td>
</tr>
<tr>
<td>ROCK Inhibitor Y-27632</td>
<td>DC Chemical</td>
<td>Cat#DC1028</td>
</tr>
<tr>
<td>0.5M EDTA</td>
<td>Thermo Fisher</td>
<td>Cat#AM9262</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Sigma</td>
<td>Cat#S5886</td>
</tr>
<tr>
<td>Hydrochloric acid</td>
<td>Sigma</td>
<td>Cat#30721</td>
</tr>
<tr>
<td>Acetonitrile LC-MS grade</td>
<td>Merck</td>
<td>Cat#1000304000</td>
</tr>
<tr>
<td>Methanol LC-MS grade</td>
<td>Merck</td>
<td>Cat#1060354000</td>
</tr>
<tr>
<td>Water (for preparing saline and extraction solution)</td>
<td>Milli-Q</td>
<td>N/A</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Sigma</td>
<td>Cat#S5886</td>
</tr>
<tr>
<td>Formic acid</td>
<td>Thermo Fisher</td>
<td>Cat#A117-50</td>
</tr>
<tr>
<td>Ammonium bicarbonate</td>
<td>Sigma</td>
<td>Cat#5.33005</td>
</tr>
<tr>
<td>Norvaline</td>
<td>Acros</td>
<td>Cat#6600-40-4</td>
</tr>
<tr>
<td>Chloroform (for lipid extraction)</td>
<td>Supelco</td>
<td>Cat#102444</td>
</tr>
<tr>
<td>Heptadecanoate</td>
<td>Sigma</td>
<td>Cat#51633</td>
</tr>
<tr>
<td>Coprostan-3-ol</td>
<td>Sigma</td>
<td>Cat#5000206</td>
</tr>
<tr>
<td>Bovine Serum Albumin Standard (for protein quantification)</td>
<td>Bio-Rad</td>
<td>Cat#A7030</td>
</tr>
<tr>
<td>Urea (for protein quantification)</td>
<td>BBI</td>
<td>Cat#A510907</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>BCA Protein Assay Kit (for protein quantification)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#23225</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>H1 hESC line</td>
<td>WiCell Research Institute</td>
<td>NIHhESC-10-0043</td>
</tr>
<tr>
<td>H9 hESC line</td>
<td>WiCell Research Institute</td>
<td>NIHhESC-10-0062</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>MassLynx (v4.1)</td>
<td>Waters</td>
<td>N/A</td>
</tr>
<tr>
<td>TargetLynx</td>
<td>Waters</td>
<td>N/A</td>
</tr>
<tr>
<td>Skyline (optional)</td>
<td>MacCoss Lab</td>
<td><a href="https://skyline.ms/project/home/software/Skyline/begin.view">https://skyline.ms/project/home/software/Skyline/begin.view</a></td>
</tr>
<tr>
<td>Prism (v9.1.1) (optional)</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td>SIMCA (optional)</td>
<td>Sartorius</td>
<td><a href="https://www.sartorius.com/en/products/process-analytical-technology/data-analytics-software/mvda-software/simca">https://www.sartorius.com/en/products/process-analytical-technology/data-analytics-software/mvda-software/simca</a></td>
</tr>
<tr>
<td>RStudio (optional)</td>
<td>(<a href="#bib7">Racine, 2012</a>)</td>
<td><a href="https://rstudio.com/products/r%20studio/">https://rstudio.com/products/r studio/</a></td>
</tr>
<tr>
<td>SigmaStat (v4.0) (optional)</td>
<td>Systat Software Inc.</td>
<td><a href="https://systatsoftware.com/products/sigmastat/">https://systatsoftware.com/products/sigmastat/</a></td>
</tr>
<tr>
<td>SPSS (v28) (optional)</td>
<td>IBM</td>
<td><a href="https://www.ibm.com/products/spss-statistics">https://www.ibm.com/products/spss-statistics</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Cell scraper</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#07-200-364</td>
</tr>
<tr>
<td>Eppendorf™ Polypropylene Tube (1.5 mL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#0030120086</td>
</tr>
<tr>
<td>Conical-bottom glass centrifuge tubes (10 mL) (for lipid)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#05-569-2</td>
</tr>
<tr>
<td>PTFE black phenolic screw-thread closures (for lipid)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#05-569-5</td>
</tr>
<tr>
<td>LCMS Screw-Top Kitpack</td>
<td>Waters</td>
<td>Cat#600000669CV</td>
</tr>
<tr>
<td>Low volume insert (150 μL)</td>
<td>Waters</td>
<td>Cat#WAT094171</td>
</tr>
<tr>
<td>Acquity UPLC BEH amide column (2.1 × 100 mm, 1.7 μm) (for amino acids, ribonucleotides, TCA intermediates, and so on)</td>
<td>Waters</td>
<td>186004801</td>
</tr>
<tr>
<td>Acquity UPLC BEH HILIC column (2.1 × 100 mm, 1.7 μm) (For PC, LPC, LPA and intermediates of glycolysis, PPP, and so on)</td>
<td>Waters</td>
<td>186003461</td>
</tr>
<tr>
<td>Acquity UPLC BEH C18 column (2.1 × 100 mm, 1.7 μm) for lipids)</td>
<td>Waters</td>
<td>186002352</td>
</tr>
<tr>
<td>Waters Xevo Tandem Quadrupole Mass Spectrometry</td>
<td>Waters</td>
<td>201000252</td>
</tr>
<tr>
<td>Infinite M200 pro (plate reader)</td>
<td>Tecan</td>
<td><a href="https://lifesciences.tecan.com/plate_readers/infinite_200_pro">https://lifesciences.tecan.com/plate_readers/infinite_200_pro</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Sodium selenite stock (0.7 mg/mL)</caption>
<thead></thead>
<tbody>
<tr>
<td>Sodium selenite stock</td>
<td>21 mg sodium selenite, dissolve and fill up to 30 mL with UltraPure™ water</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C up to one year.</p>
<div class="note">
<span class="note-title">Note:</span> Filter the sodium selenite stock solution and aliquot 500 μL/tube.</div>
<table id="undtbl2">
<caption>Ascorbic acid and selenium stock (50X)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>L-Ascorbic acid 2-phosphate sesquimagnesium salt hydrate</td>
<td>50X</td>
<td>1.6 g</td>
</tr>
<tr>
<td>Sodium selenite stock (0.7 mg/mL)</td>
<td>50X</td>
<td>485 μL</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>N/A</td>
<td>499.515 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store aliquots at −80°C up to one year.</p>
<div class="note">
<span class="note-title">Note:</span> Filter the stock solution and aliquot 10 mL/tube.</div>
<table id="undtbl3">
<caption>Transferrin stock (1,000X)</caption>
<thead></thead>
<tbody>
<tr>
<td>Transferrin stock</td>
<td>500 mg holo-transferrin, dissolve and fill up to 50 mL with 1X DPBS</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C up to one year.</p>
<div class="note">
<span class="note-title">Note:</span> Filter the transferrin stock solution and aliquot 500 μL/tube.</div>
<table id="undtbl4">
<caption>Insulin stock (1,000X)</caption>
<thead></thead>
<tbody>
<tr>
<td>Insulin stock</td>
<td>2.5g insulin, fill up to 250 mL with UltraPure™ water. Adjust pH to 4 to dissolve.</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C up to one year.</p>
<div class="note">
<span class="note-title">Note:</span> Filter the insulin stock solution and aliquot 500 μL/tube.</div>
<table id="undtbl5">
<caption>TGF-β1 stock (10,000X)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Human TGF-β1</td>
<td>17.4 μg/mL</td>
<td>1 mg</td>
</tr>
<tr>
<td>Human Serum Albumin</td>
<td>0.5 mg/mL</td>
<td>28.5 mg</td>
</tr>
<tr>
<td>1N HCl</td>
<td>N/A</td>
<td>230 μL</td>
</tr>
<tr>
<td>PBS (1X)</td>
<td>N/A</td>
<td>57.241 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>57. 471 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C up to one year.</p>
<div class="note">
<span class="note-title">Note:</span> Aliquot 50 μL/tube.</div>
<table id="undtbl6">
<caption>FGF2 stock (1,000X)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Human FGF2</td>
<td>100 μg/mL</td>
<td>5 mg</td>
</tr>
<tr>
<td>Human Serum Albumin</td>
<td>0.15 mg/mL</td>
<td>7.5 mg</td>
</tr>
<tr>
<td>PBS (1X)</td>
<td>N/A</td>
<td>50 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C up to one year.</p>
<div class="note">
<span class="note-title">Note:</span> Filter the FGF2 stock and aliquot 500 μL/tube.</div>
<table id="undtbl7">
<caption>Home-made E8 medium (1X)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ascorbic acid and selenium stock (50X)</td>
<td>1X</td>
<td>10 mL</td>
</tr>
<tr>
<td>Transferrin stock (1,000X)</td>
<td>10 μg/mL</td>
<td>500 μL</td>
</tr>
<tr>
<td>Insulin stock (1,000X)</td>
<td>10 μg/mL</td>
<td>500 μL</td>
</tr>
<tr>
<td>TGF-β1 stock (10,000X)</td>
<td>1.74 ng/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>FGF2 stock (1,000X)</td>
<td>100 ng/mL</td>
<td>500 μL</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>N/A</td>
<td>488.45 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C up to two weeks.</p>
<table id="undtbl8">
<caption>DPBS-EDTA solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.5 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>1.8 g/L</td>
<td>0.9 g</td>
</tr>
<tr>
<td>DPBS</td>
<td>N/A</td>
<td>499.5 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C up to one month.</p>
<div class="note">
<span class="note-title">Note:</span> Filter the DPBS-EDTA solution.</div>
<table id="undtbl9">
<caption>ROCK inhibitor Y-27632 (10 mM)</caption>
<thead></thead>
<tbody>
<tr>
<td>Y-27632 stock</td>
<td>100 mg Y-27632, fill up to 31.225 mL with DMSO</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C up to 6 months.</p>
<div class="note">
<span class="note-title">Note:</span> Aliquot 300 μL/tube.</div>
<table id="undtbl10">
<caption>0.9% saline solution</caption>
<thead></thead>
<tbody>
<tr>
<td>0.9% saline</td>
<td>9 g NaCl, fill up to 1 L with Milli-Q water</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 15°C–30°C up to one month.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Commercial saline can also be used. For example: Sigma-Aldrich Cat#S8776; Fisher scientific Cat#Z1376.</div>
<div class="note">
<span class="note-title">Note:</span> For homemade saline, use boiled water to prepare the saline or sterilize the saline solution by autoclaving at 121°C for 15 min or filter the solution through 0.45 μm filter. Aliquot and store at 15°C–30°C up to one month.</div>
<table id="undtbl11">
<caption>Medium extraction solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetonitrile</td>
<td>75%</td>
<td>75 mL</td>
</tr>
<tr>
<td>Methanol</td>
<td>25%</td>
<td>25 mL</td>
</tr>
<tr>
<td>Formic acid</td>
<td>0.2%</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 15°C–30°C up to one year.</p>
<div class="note">
<span class="note-title">Note:</span> Adapted from a published paper (<a href="#bib5">Jain et al., 2012</a>). Homogenize ultrasonically for 15 min.</div>
<div class="note">
<span class="note-title">Note:</span> In this protocol, acetonitrile/methanol/water is used to extract metabolites from medium. Compared to the methanol/water system with and without pre-chilling (−80°C) of the extraction solution, as shown in <a href="#fig1">Figure 1</a>, the acetonitrile/methanol/water system retains more abundant metabolites in most cases. Pre-chilling the extraction solution can help quench metabolism, but the cold-shock itself may cause metabolic changes, so whether or not to pre-chill the extraction solution should be decided according to needs.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Comparison of different methods used for extracting metabolites from medium.</div>
<p>E8 medium was extracted by nine volumns of 75% acetonitrile, 25% methanol and 0.2% formic acid (ACN_E8), methanol (MeOH_E8), −80°C pre-chilled 75% acetonitrile, 25% methanol and 0.2% formic acid (−80°C ACN_E8) and −80°C pre-chilled methanol (−80°C MeOH_E8) as described in the “Medium collection for metabolic analysis” section. Peak area of each metabolite is normalized to the average of that in ACN_E8 group. Data are shown as mean ± SD, n = 5.</p>
</figcaption>
</figure>
<table id="undtbl12">
<caption>Cell extraction solution for Method 1 (Pre-chilled at –80°C for at least one hour)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Methanol</td>
<td>80%</td>
<td>800 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>20%</td>
<td>200 mL</td>
</tr>
<tr>
<td>Norvaline</td>
<td>10 μg/mL</td>
<td>10 mg</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>1 L</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C up to one year.</p>
<div class="note">
<span class="note-title">Note:</span> Adapted from a published paper (<a href="#bib11">Zhang et al., 2016</a>). Homogenize ultrasonically for 15 min.</div>
<p>Solvents for lipid extraction in <b>Method 2</b></p>
<ol type="a">
<li>Ice-cold Milli-Q water</li>
<li>−80°C methanol</li>
<li>−80°C chloroform containing 1 μg/mL heptadecanoate and 1 μg/mL coprostan-3-ol</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Refer to step 13 of Step-by-Step method for details.</div>
<table id="undtbl13">
<caption>Protein denaturing solution (for protein quantification)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Urea</td>
<td>8 M</td>
<td>48.48 g</td>
</tr>
<tr>
<td>Ammonium bicarbonate</td>
<td>100 mM</td>
<td>0.79 g</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>N/A</td>
<td>100 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 15°C–30°C up to one month.</p>
<div class="note">
<span class="note-title">Note:</span> Homogenize ultrasonically for 15 min.</div>
<div class="critical">
<span class="critical-title">Critical:</span> List of −80°C pre-chilled items: (1) cell extraction solution, (2) methanol, (3) chloroform containing 1 μg/mL heptadecanoate and 1 μg/mL coprostan-3-ol, and (4) cell scrapers.</div>
<div class="critical">
<span class="critical-title">Critical:</span> List of on-ice pre-chilled items: (1) saline, (2) Milli-Q water, and (3) empty, labeled tubes or glass vials.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of home-made E8 medium</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>30 min</b>
</div>
<ol>
<li>Preparation of 500 mL E8 medium</li>
</ol>
<p>The components of E8 medium are: DMEM/F12, NaHCO<sub>3</sub>, 2-Phospho-L-ascorbic acid trisodium salt, sodium selenite, FGF2, TGF-β1, insulin, and transferrin. The concentrations of E8 medium components are listed in Materials and equipment.</p>
<p>For more details on E8 medium preparation and stem cell culture maintenance, please refer to <a href="#bib2">Chen et al. (2011a)</a> and <a href="#bib1">Beers et al. (2012)</a>. For metabolic analysis in the same set of samples, the same batch of medium should be used.</p>
<h3 id="sec3.2">Preparation of Matrigel-coated plate</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>2 days</b> <b>for step 2</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>30 min</b> <b>for step 3</b>
</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Commercial Matrigel pre-coated plates may also be used. For example: Corning® BioCoat™ Matrigel® Matrix 6-well clear flat bottom TC-treated multiwell plate (Cat#354432). For metabolic analysis in chemically defined conditions, vitronectin or other coating substrates could be used in place of Matrigel.</div>
<ol start="2">
<li>Preparation of Matrigel aliquots
<ol type="a">
<li>One day before the experiment, thaw 1 bottle of Matrigel on ice for 12–24 h and pre-chill empty 2-mL Eppendorf tubes in −80°C refrigerator.</li>
<li>Aliquot 2 mg Matrigel into each pre-chilled 2-mL Eppendorf tube, and freeze immediately in −80°C refrigerator.</li>
<li>Store aliquoted Matrigel at −80°C. One tube of 2 mg Matrigel will be used to coat six cell culture plates.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Preparation of Matrigel-coated cell culture plates
<div class="note">
<span class="note-title">Note:</span> Pre-chill DMEM/F12 at 4°C for at least 1 h.</div>
<ol type="a">
<li>Dilute 2 mg of aliquoted Matrigel in 36 mL cold DMEM/F12.
<div class="critical">
<span class="critical-title">Critical:</span> Be quick at this step.</div>
</li>
<li>Aliquot 1 mL of diluted Matrigel into each well of a 6-well plate. Scale down for other sizes of plates. Gently shake the plate till the medium covers the whole surface without air bubble. 2 mg of Matrigel can be used to coat 6 multi-well plates.
<div class="note">
<span class="note-title">Note:</span> 0.5 mL/well diluted Matrigel for 12-well plates; 0.25 mL/well diluted Matrigel for 24-well plates.</div>
</li>
<li>Leave the plate in a 37°C incubator for at least 30 min before use.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Coated plates can be wrapped with foil and stored in 4°C for up to 1 week.</div>
<div class="note">
<span class="note-title">Note:</span> Medium remaining in the coated plate will continuously evaporate even at 4°C. Avoid using the plate if part of the culturing surface has dried.</div>
<h3 id="sec3.3">Recovery of human ES/iPS cells from frozen stock</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>20 min</b>
</div>
<ol start="4">
<li>Recovery of human ES/iPS cells
<ol type="a">
<li>Take a Matrigel-coated 6-well plate. Aspirate medium from the wells you plan to use, and then add 2 mL/well E8 medium containing 10 μM Y-27632.
<div class="note">
<span class="note-title">Note:</span> (Optional) The plate with E8 medium can be put into the incubator to pre-equilibrate for 30 min.</div>
</li>
<li>Prepare a 37°C water bath.</li>
<li>Take the cryotube with frozen cell stock out of liquid nitrogen tank and put it directly in the 37°C water bath.
<div class="critical">
<span class="critical-title">Critical:</span> When there is only a small ice particle floating, transfer the tube into a biosafety cabinet.</div>
</li>
<li>Transfer cells into a 15-mL conical tube. Drop-wisely add 10 mL E8 medium + 10 μM Y-27632, while continuously mixing the solution.</li>
<li>Pellet cells by centrifugation at 300 × <i>g</i> for 5 min.</li>
<li>Carefully aspirate the supernatant, resuspend the cells with 1 mL E8 + 10 μM Y-27632, and then gently seed into the wells of Matrigel-coated plate containing E8 (step 4a above).</li>
<li>Change medium daily by aspirating the spent medium and adding 2 mL fresh E8.</li>
<li>When cells reach 60–70% confluence, passage the cells following the steps below. If the survival is not good, passage cells after 5–6 days regardless of the confluence.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Passaging and maintenance of human ES/iPS cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>30 min</b> <b>for step 5</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>30 min</b> <b>for step 6</b>
</div>
<ol start="5">
<li>Passaging of human ES/iPS cells with DPBS-EDTA solution
<ol type="a">
<li>Take a Matrigel-coated 6-well plate. Aspirate the remaining medium and add 2 mL E8 medium containing 5 μM Y-27632 in each well.
<div class="note">
<span class="note-title">Note:</span> (Optional) The plate with E8 medium can be put into the incubator to pre-equilibrate for 30 min.</div>
</li>
<li>For cells maintained in 6-well plates that need to be passaged, remove the spent medium, and rinse cells twice by adding 1 mL/well DPBS-EDTA solution and aspirating the solution without incubation.
<div class="critical">
<span class="critical-title">Critical:</span> Be careful when rinsing cells and aspirating EDTA to avoid losing cells.</div>
</li>
<li>Add 1 mL DPBS-EDTA solution per well, and incubate in 37°C incubator for 5 min.
<div class="note">
<span class="note-title">Note:</span> The optimal incubation time depends on the cell line. 5–10 min of incubation works for most cell lines.</div>
</li>
<li>Aspirate the DPBS-EDTA solution. Harvest cells with 1 mL of E8 medium + 5 μM Y-27632 by pipetting up and down.</li>
<li>Take the desired amount of cell suspension and seed the cells into the Matrigel-coated plate containing E8 medium and 5 μM Y-27632 (step 5a).
<div class="note">
<span class="note-title">Note:</span> The passaging ratio depends on the cell line and experimental design. For H1/H9 hESCs, 1:6 passaging is conducted for maintenance.</div>
</li>
<li>Gently shake the plate until cells are evenly distributed. Put the plate in a 37°C incubator with 5% CO<sub>2</sub>.</li>
<li>The next day, change medium to 2 mL/well E8 medium without ROCK inhibitor.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Maintenance of human ES/iPS cells
<div class="critical">
<span class="critical-title">Critical:</span> For metabolic analysis, it is recommended to equibilize E8 medium in 37°C incubator with loosened lid for 15 min before medium change.</div>
<ol type="a">
<li>Culture medium is usually changed every day. For one well of a 6-well plate, aspirate the spent medium and add 2 mL fresh E8 medium.</li>
<li>Cells are passaged every 3–4 days when the confluence is about 70%. Avoid over-confluence of cell culture.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> ROCK inhibitor is usually added during cell passaging and removed after 24 h.</div>
<h3 id="sec3.5">Human ES/iPS cell preparation for metabolite detection</h3>
<h4 id="sec3.5.1">Day 0</h4>
<div class="timing">
<span class="timing-title">Timing:</span> <b>30 min</b>
</div>
<ol start="7">
<li>Seed enough cells into 6-well plates for LC-MS analysis as described above (step 5). Prepare 8 wells for each treatment. Five wells of cells will be collected individually as five replicates, and additional 3 wells will be used to determine cell number for normalization. Multiple plates will be needed.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The minimum number of cells required for metabolomic profiling is 1 million. When H1/H9 hESCs reach ∼80% confluence, the cell number in each well of a 6-well plate is over one million. This number may vary for different cell lines. It is important to avoid over-confluence of culture, so cells need to be monitored under microscope daily.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Cell number is important for normalization. Before metabolic analysis, multiple rounds of experiments are often conducted to evaluate the impact of specific treatment on cell proliferation. As an alternative approach for normalization, cellular protein concentration from each well may also be used as internal control. However, if albumin or FBS is used in cell culture, it is important to make sure that exogenous proteins are rinsed off effectively before protein measurement.</div>
<h4 id="sec3.5.2">Day 1</h4>
<div class="timing">
<span class="timing-title">Timing:</span> <b>10 min</b>
</div>
<ol start="8">
<li>Change medium. For 6-well plates, add 2 mL fresh E8 medium per well. Add appropriate treatments.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Treatment period is flexible depending on the experiment design. Medium should be changed every day until the day of metabolite analysis. In our experiments, medium is collected for extraction 24 h after the final medium change.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid over-confluence of cells.</div>
<h4 id="sec3.5.3">Day 2</h4>
<div class="timing">
<span class="timing-title">Timing:</span> <b>1 h</b>
</div>
<ol start="9">
<li>Count cells. When cells are ready for metabolic analysis, collect 3 wells of cells from each condition for cell count determination. Aspirate medium and add 500 μL/well TrypLE Select, incubate in 37°C for 5 min, and neutralize with 500 μL 5% FBS in DMEM/F12. Determine the cell count using a hemocytometer or flow cytometer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Metabolic data is normalized to cell number or protein level.</div>
<h3 id="sec3.6">Medium collection for metabolic analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>1 h</b>
</div>
<ol start="10">
<li>Collect medium and extract with medium extraction solution (see Materials and equipment).
<ol type="a">
<li>Transfer 1 mL culture medium from each well to a 1.5-mL Eppendorf tube. Spin at 1,000 x <i>g</i> for 5 min.</li>
<li>Transfer 100 μL of the supernatant into a new 1.5-mL Eppendorf tube. Add 900 μL medium extraction solution (nine volumes) and vortex.</li>
<li>Spin at 18,000 × <i>g</i> at 4°C for 10 min. Take 200 μL supernatant for LC-MS analysis.</li>
</ol>
</li>
</ol>
<h3 id="sec3.7">Cell collection for metabolic analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>1 h</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>1 h</b> <b>for step 14</b>
</div>
<ol start="11">
<li>Put 0.9% saline and labeled tubes on ice. Make sure cell scrapers and cell extraction solution (see Materials and equipment) are pre-chilled at −80°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To ensure effective quenching, keep the cell extraction solution in −80°C until use. We advise to prepare two bottles of cell extraction solution. Take one bottle out of the freezer for use with one plate, put it back immediately and then take the other bottle for the next plate. Try to decrease the time the extraction solution is left at room temperature (15°C–30°C).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Perform the rinse quickly to avoid interference with results. Rinsing with saline is to remove residual medium and floating cell debris. However, it is reported that the rinsing procedure itself could significantly affect determination of metabolites (<a href="#bib8">Ser et al., 2015</a>). Our data also validated that washing cells with saline could decrease signal intensities of metabolites (<a href="#fig2">Figure 2</a>). But considering that components in residual medium would lead to distortion of quantification, the washing procedure is retained in this protocol.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Impact of extraction procedures on quantification</div>
<p>H1 cells (1 million) were washed twice with 0.9% saline and then extracted as instructed in Method 1 (saline wash), or directly quenched by −80°C cell extraction solution after spent medium is aspirated (W/O wash). Ovnight incubation at −80°C (−80°C, 12 h) was also compared to immediate quantification (saline wash). Data are shown as mean ± SD, n = 5.</p>
</figcaption>
</figure>
<ol start="12">
<li>
<b>Method 1</b> (for amino acids and intermediates in glycolysis, PPP and TCA cycle):
<ol type="a">
<li>Rinse cells with 1 mL/well ice-cold 0.9% saline twice.</li>
<li>Aspirate the saline and add 500 μL/well cold cell extraction solution (kept in −80°C). Scrape off cells by cell scraper and transfer the suspension into a labeled, pre-chilled 1.5-mL Eppendorf tube (Tube 1). Put tubes on ice.
<div class="critical">
<span class="critical-title">Critical:</span> To ensure a correct volume, use a new pipette tip to add the cold cell extraction solution into each well.</div>
</li>
<li>Wash the well with 100 μL cell extraction solution and combine it into Tube 1.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Sample in Tube 1 can be stored in −80°C fridge for 24 h (see Note after step 12d).</div>
</li>
<li>Vortex each sample. Centrifuge at 18,000 × <i>g</i> for 5 min at 4°C. Transfer the supernatant into a new 1.5-mL tube (Tube 2). Put tubes on ice.
<div class="note">
<span class="note-title">Note:</span> The pellet could be preserved to measure protein concentration as described below (step 14).</div>
<div class="note">
<span class="note-title">Note:</span> After step 12d, Tube 2 could be preserved at −80°C for 24 h. This 24 h preservation was evaluated in <a href="#fig2">Figure 2</a>. Peak intensities of most metabolites were comparable with or without 24 h-storage. Because of the risk of losing metabolites caused by storage, we recommend immediate LC-MS analysis after extraction. Long term storage of extracted metabolites, even in −80°C refrigerator, should be avoided.</div>
</li>
<li>Vortex Tube 2, put tube on ice, and then transfer 200 μL extract into a sample vial for LC-MS analysis.</li>
</ol>
</li>
</ol>
<ol start="13">
<li>
<b>Method 2</b> (for lipid determination):
<div class="critical">
<span class="critical-title">Critical:</span> To avoid plastic contamination, glass vials are recommended. Polypropylene pipette tip is conditionally resistant to chloroform for a minimized contact time. A fresh tip must be used each time chloroform is dispensed.</div>
<ol type="a">
<li>Put Milli-Q water and labeled glass vials on ice. Make sure methanol and chloroform containing 1 μg/mL heptadecanoate and 1 μg/mL coprostan-3-ol (see Materials and equipment) are pre-chilled at −80°C.
<div class="note">
<span class="note-title">Note:</span> Heptadecanoate is used as internal control for fatty acids. coprostan-3-ol is used as internal control for cholesterol.</div>
</li>
<li>Rinse cells with 1 mL/well ice-cold 0.9% saline twice.</li>
<li>Aspirate the saline. Add 250 μL/well −80°C methanol and then add 100 μL/well ice-cold Milli-Q water.</li>
<li>Scrape off cells and transfer the suspension into glass vials. Put vials on ice.</li>
<li>Add 250 μL/vial −80°C chloroform containing 1 μg/mL heptadecanoate and 1 μg/mL coprostan-3-ol. Vortex and then centrifuge at 18,000 × <i>g</i> for 5 min at 4°C.</li>
<li>Remove the upper hydrophilic phase. Protein pellet at the interface is carefully pushed aside on the wall of the vial. A fresh tip is used to reach the bottom of the vial to collect the hydrophobic phase. Stop collecting before reaching the interface.</li>
<li>Transfer the hydrophobic phase into a new glass vial. Evaporate under nitrogen gas till dry. Re-dissolve the residue in 100 μL 2:1 chloroform/methanol (v/v) for LC-MS analysis.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Nitrogen but not regular air is used to dry samples, because lipids, especially unsaturated lipids are easily oxidized. The time of nitrogen blowdown is mainly affected by the amount of water in the sample.</div>
<ol start="14">
<li>Determine protein mass (optional)
<ol type="a">
<li>Decant supernatant and add 200 μL methanol to the protein pellet. Vigorously vortex and then centrifuge at 18,000 × <i>g</i> for 5 min at 4°C.</li>
<li>Decant the supernatant into waste. Let the protein pellet dry by putting the tube upside down on a Kimwipe in a biological safety cabinet for 5 min.</li>
<li>Dissolve the protein pellet with 400 μL protein denaturing solution (see Materials and equipment).</li>
<li>Bicinchoninic acid (BCA) protein assay kit is used to quantify the protein concentration. Bovine serum albumin standard (2 mg/mL) is diluted to a series of concentrations in 1X PBS, typically 0.1–1 mg/mL, to construct a standard curve.</li>
<li>Prepare the working reagent as instructed by the BCA kit manual. Samples in urea are diluted 1/10. Mix diluted samples or BSA standards with the working reagent in a 96-well microplate.
<div class="critical">
<span class="critical-title">Critical:</span> The BCA assay is compatible with up to 3 M urea, so appropriate dilution of the protein pellet dissolved in protein denaturing solution (8 M urea + 100 mM NH<sub>4</sub>HCO<sub>3</sub>) is necessary.</div>
</li>
<li>Incubate the plate at 37°C for 30 min, and then check absorptance at 562 nm by a spectrophotometer.</li>
<li>Use the standard curve to determine the protein concentration of each sample.</li>
</ol>
</li>
</ol>
<h3 id="sec3.8">LC-MS warm up</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1 h</div>
<ol start="15">
<li>Calibrate the mass spectrometer following manufacturer’s recommendation. For the Waters TQD used in this protocol, use IntelliStart software to automatically tune and mass calibrate the instrument.</li>
<li>Make sure mobile phases and system wash solution are sufficient.</li>
<li>Install the UPLC column. For example, install HILIC column for the quantification of metabolites involved in glycolysis as described in Run LC-MS section.</li>
<li>Purge the system with mobile phases. In this protocol, two bottles of mobile phases are used for the methods, purge each bottle of mobile phase at 4 mL/min for 4 min. Equilibrate the column with initial gradient and flow rate until delta pressure (maximum pressure minus minimum pressure) is below 30 psi.</li>
<li>Check LC and MS performance. Check column pressure, and take a record of the column pressure after equilibration. An increase of over 300 psi in column pressure could be considered as abnormal increase. Wash the column as instructed if abnormal column pressure increase happens.</li>
</ol>
<h3 id="sec3.9">Run LC-MS</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>2 days</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>20 min/sample</b> <b>for step 20</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>10 min/sample</b> <b>for step 21</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>10 min/sample</b> <b>for step 22</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>15 min/sample</b> <b>for step 23</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>15 min/sample</b> <b>for step 24</b>
</div>
<div class="note">
<span class="note-title">Note:</span> Waters Xevo TQD coupled with Waters Acquity UPLC system is used for quantification in this protocol, though other LC-MS systems with similar capabilities can be used. The MS parameters are listed in <a href="#tbl1">Table 1</a>.</div>
<table id="tbl1">
<caption>Table 1. TQD/MS parameter settings</caption>
<thead>
<tr>
<th>Parameters</th>
<th>Values</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ionization</td>
<td>ESI (electrospray ionization)</td>
</tr>
<tr>
<td>Collision gas</td>
<td>Argon</td>
</tr>
<tr>
<td>API gas</td>
<td>Nitrogen</td>
</tr>
<tr>
<td>Capillary voltage</td>
<td>3500 V</td>
</tr>
<tr>
<td>Desolvation temperature</td>
<td>500°C</td>
</tr>
<tr>
<td>Column temperature</td>
<td>40°C</td>
</tr>
<tr>
<td>Autosampler sample temperature</td>
<td>4°C</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Polarity is important for the column separation of metabolites. Different metabolites are acquired using different polarity.</div>
<p>As shown in <a href="#fig3">Figure 3</a>, multiple MRM settings of each standard compound were tested during method development. Retention time of one compound in different MRM channels should be aligned, which excludes peaks at other retention time. The MRM setting which generates the most intense response is selected for later quantification.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Examples of amino acid chromatograms</div>
<p>Different MRM settings for twenty amino acids were shown as examples of how the MRM is set in this protocol. To establish an MRM setting, multiple daughter ions are used for the same parent ion, the elution time of different daughter ions should be the same, and the one with the highest response is used for later quantification.</p>
</figcaption>
</figure>
<p><u>Q</u>uality control (QC) sample is prepared by mixing equal volumes of all samples. It is recommended to run the QC sample six times at the beginning of the sample sequence and subsequently inject the QC sample after every 5–10 sample injections. QC sample is used to verify the performance of LC-MS.</p>
<ol start="20">
<li>
<b>Program 1:</b> For the quantification of norvaline, amino acids, GSH, GSSG, SAH, SAM, ascorbic acid and myo-inositol. LC-MS/MS conditions and parameters are displayed in <a href="#tbl2">Table 2</a>. Representative chromatograms are shown in <a href="#fig3">Figures 3</a> and <a href="#fig4">4</a>.
<table id="tbl2">
<caption>Table 2. LC-MS/MS conditions and parameters for program 1</caption>
<thead>
<tr>
<th>Name</th>
<th>MRM</th>
<th>Dwell (s)</th>
<th>CV (V)</th>
<th>CE (V)</th>
<th>Mode</th>
<th>Retention time (min)</th>
<th>Time window ± (min)</th>
<th>Performance</th>
</tr>
</thead>
<tbody>
<tr>
<td>Norvaline</td>
<td>117.9 → 54.9</td>
<td>0.005</td>
<td>24</td>
<td>16</td>
<td>ESI+</td>
<td>7.2</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Tryptophan (Trp)</td>
<td>205.1 → 118.0</td>
<td>0.005</td>
<td>22</td>
<td>26</td>
<td>ESI+</td>
<td>7.4</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Tyrosine (Tyr)</td>
<td>182.0 → 136.1</td>
<td>0.005</td>
<td>20</td>
<td>12</td>
<td>ESI+</td>
<td>8.0</td>
<td>0.15</td>
<td>∗∗</td>
</tr>
<tr>
<td>Arginine (Arg)</td>
<td>175.1 → 70.0</td>
<td>0.005</td>
<td>34</td>
<td>20</td>
<td>ESI+</td>
<td>10.6</td>
<td>0.4</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Phenylalanine (Phe)</td>
<td>166.0 → 120.1</td>
<td>0.005</td>
<td>24</td>
<td>12</td>
<td>ESI+</td>
<td>7.1</td>
<td>0.3</td>
<td>∗∗</td>
</tr>
<tr>
<td>Histidine (His)</td>
<td>156.0 → 110.0</td>
<td>0.005</td>
<td>26</td>
<td>14</td>
<td>ESI+</td>
<td>10.7</td>
<td>0.5</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Methionine (Met)</td>
<td>150.0 → 56.0</td>
<td>0.005</td>
<td>22</td>
<td>16</td>
<td>ESI+</td>
<td>7.6</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Glutamic acid (Glu)</td>
<td>148.0 → 84.0</td>
<td>0.005</td>
<td>20</td>
<td>16</td>
<td>ESI+</td>
<td>8.8</td>
<td>0.35</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Lysine (Lys)</td>
<td>147.0 → 67.0</td>
<td>0.005</td>
<td>22</td>
<td>26</td>
<td>ESI+</td>
<td>10.8</td>
<td>0.3</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Glutamine (Gln)</td>
<td>147.0 → 41.0</td>
<td>0.005</td>
<td>22</td>
<td>24</td>
<td>ESI+</td>
<td>9.3</td>
<td>0.35</td>
<td>∗∗</td>
</tr>
<tr>
<td>Aspartic acid (Asp)</td>
<td>134.0 → 73.9</td>
<td>0.005</td>
<td>20</td>
<td>12</td>
<td>ESI+</td>
<td>9.1</td>
<td>0.4</td>
<td>∗</td>
</tr>
<tr>
<td>Asparagine (Asn)</td>
<td>133.0 → 73.9</td>
<td>0.005</td>
<td>20</td>
<td>14</td>
<td>ESI+</td>
<td>9.4</td>
<td>0.35</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Isoleucine (Ile)</td>
<td>132.0 → 69.0</td>
<td>0.005</td>
<td>20</td>
<td>16</td>
<td>ESI+</td>
<td>7.3</td>
<td>0.25</td>
<td>∗∗</td>
</tr>
<tr>
<td>Leucine (Leu)</td>
<td>132.0 → 43.4</td>
<td>0.005</td>
<td>20</td>
<td>22</td>
<td>ESI+</td>
<td>7.0</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Cysteine (Cys)</td>
<td>122.0 → 75.9</td>
<td>0.005</td>
<td>18</td>
<td>12</td>
<td>ESI+</td>
<td>8.2</td>
<td>0.35</td>
<td>∗</td>
</tr>
<tr>
<td>Threonine (Thr)</td>
<td>120.0 → 56.0</td>
<td>0.005</td>
<td>48</td>
<td>14</td>
<td>ESI+</td>
<td>8.7</td>
<td>0.35</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Valine (Val)</td>
<td>118.0 → 72.0</td>
<td>0.005</td>
<td>20</td>
<td>10</td>
<td>ESI+</td>
<td>7.7</td>
<td>0.3</td>
<td>∗∗</td>
</tr>
<tr>
<td>Proline (Pro)</td>
<td>116.0 → 70.0</td>
<td>0.005</td>
<td>30</td>
<td>12</td>
<td>ESI+</td>
<td>7.8</td>
<td>0.4</td>
<td>∗∗</td>
</tr>
<tr>
<td>Serine (Ser)</td>
<td>106.0 → 60.0</td>
<td>0.005</td>
<td>22</td>
<td>8</td>
<td>ESI+</td>
<td>9.2</td>
<td>0.4</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Alanine (Ala)</td>
<td>90.0 → 44.0</td>
<td>0.005</td>
<td>20</td>
<td>8</td>
<td>ESI+</td>
<td>8.4</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Glycine (Gly)</td>
<td>75.9 → 29.8</td>
<td>0.005</td>
<td>18</td>
<td>6</td>
<td>ESI+</td>
<td>8.8</td>
<td>0.3</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Glutathione (GSH)</td>
<td>308 → 179</td>
<td>0.005</td>
<td>20</td>
<td>10</td>
<td>ESI+</td>
<td>8.4</td>
<td>0.2</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Glutathione disulfide (GSSG)</td>
<td>613 → 355</td>
<td>0.005</td>
<td>40</td>
<td>20</td>
<td>ESI+</td>
<td>11.5</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Ascorbic acid</td>
<td>175 → 115</td>
<td>0.005</td>
<td>40</td>
<td>10</td>
<td>ESI-</td>
<td>4.4</td>
<td>0.25</td>
<td>∗</td>
</tr>
<tr>
<td><i>S-Adenosyl-L-methionine (SAM)</i></td>
<td>399 → 250</td>
<td>0.005</td>
<td>20</td>
<td>10</td>
<td>ESI+</td>
<td>11.3</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td><i>S-Adenosyl-L-homocysteine (SAH)</i></td>
<td>385 → 136</td>
<td>0.005</td>
<td>20</td>
<td>20</td>
<td>ESI+</td>
<td>9.8</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td>Myo-inositol</td>
<td>179.2 → 86.9</td>
<td>0.005</td>
<td>40</td>
<td>12</td>
<td>ESI-</td>
<td>9.1</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b><i>Note:</i></b> Italicized metabolites are referenced from a database (<a href="https://metlin.scripps.edu">https://metlin.scripps.edu</a>) but not confirmed with standards. CV: cone voltage; CE: collision energy; Performance is evaluated based on how easily peak identification could be done with 10 μL injected hESC sample that was extracted as described in Method 1. ∗∗∗, single peak, easily identified; ∗∗, multiple peaks, target peak could be identified based on the retention time of the standard or by comparing the isotopic ratio, and target peak is not overlapped with other peaks; ∗, multiple peaks, very weak (below 3 folds of baseline) target peak intensity and/or target peak overlaps with other peaks.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Representative chromatogram of metabolites listed in <a href="#tbl2">Table 2</a>
</div>
</figcaption>
</figure>
<ol type="a">
<li>Amide column (Acquity UPLC BEH amide column, 2.1 × 100 mm, 1.7 μm) is used for the separation.</li>
<li>Acetonitrile with 0.1% formic acid (A) and water with 0.1% formic acid (B) are used as mobile phases. The gradient setting is listed in <a href="#tbl3">Table 3</a>.
<table id="tbl3">
<caption>Table 3. Gradient setting for program 1</caption>
<thead>
<tr>
<th>Time (min)</th>
<th>Flowrate (mL/min)</th>
<th>A%</th>
<th>B%</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial</td>
<td>0.4</td>
<td>99</td>
<td>1</td>
</tr>
<tr>
<td>4</td>
<td>0.4</td>
<td>90</td>
<td>10</td>
</tr>
<tr>
<td>10</td>
<td>0.4</td>
<td>67</td>
<td>33</td>
</tr>
<tr>
<td>13</td>
<td>0.4</td>
<td>1</td>
<td>99</td>
</tr>
<tr>
<td>15</td>
<td>0.4</td>
<td>1</td>
<td>99</td>
</tr>
<tr>
<td>16.5</td>
<td>0.4</td>
<td>99</td>
<td>1</td>
</tr>
<tr>
<td>20</td>
<td>0.4</td>
<td>99</td>
<td>1</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>
<b>Program 2:</b> For the quantification of ribonucleotides and metabolites involved in TCA cycle. LC-MS/MS conditions and parameters are displayed in <a href="#tbl4">Table 4</a>. Representative chromatograms are shown in <a href="#fig5">Figure 5</a>.
<table id="tbl4">
<caption>Table 4. LC-MS/MS conditions and parameters for program 2</caption>
<thead>
<tr>
<th>Name</th>
<th>MRM</th>
<th>Dwell (s)</th>
<th>CV (V)</th>
<th>CE (V)</th>
<th>Mode</th>
<th>Retention time (min)</th>
<th>Time window ± (min)</th>
<th>Performance</th>
</tr>
</thead>
<tbody>
<tr>
<td>Citrate (Cit)</td>
<td>191 → 111</td>
<td>0.006</td>
<td>20</td>
<td>10</td>
<td>ESI-</td>
<td>1.2</td>
<td>0.1</td>
<td>∗</td>
</tr>
<tr>
<td>α-Ketoglutarate (αKG)</td>
<td>145 → 101</td>
<td>0.006</td>
<td>20</td>
<td>6</td>
<td>ESI-</td>
<td>1.1</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Succinate (Suc)</td>
<td>117 → 73</td>
<td>0.006</td>
<td>24</td>
<td>12</td>
<td>ESI-</td>
<td>0.72</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td>Fumarate (Fum)</td>
<td>115 → 71</td>
<td>0.006</td>
<td>18</td>
<td>8</td>
<td>ESI-</td>
<td>0.68</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td>Malate (Mal)</td>
<td>133 → 115</td>
<td>0.006</td>
<td>20</td>
<td>10</td>
<td>ESI-</td>
<td>1.05</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Oxaloacetate (Oaa)</td>
<td>131 → 87</td>
<td>0.006</td>
<td>40</td>
<td>10</td>
<td>ESI-</td>
<td>0.78</td>
<td>0.1</td>
<td>∗</td>
</tr>
<tr>
<td>Pyruvate</td>
<td>86.9 → 42.9</td>
<td>0.006</td>
<td>18</td>
<td>8</td>
<td>ESI-</td>
<td>0.77</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td>Lactate</td>
<td>89 → 43</td>
<td>0.006</td>
<td>20</td>
<td>10</td>
<td>ESI-</td>
<td>0.79</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>Nicotinic acid adenine dinucleotide phosphate (NAADP)</i></td>
<td>745 → 604</td>
<td>0.006</td>
<td>30</td>
<td>10</td>
<td>ESI+</td>
<td>0.56</td>
<td>0.1</td>
<td>∗</td>
</tr>
<tr>
<td><i>Nicotinamide adenine dinucleotide phosphate (NADP)</i></td>
<td>744 → 604</td>
<td>0.006</td>
<td>26</td>
<td>18</td>
<td>ESI+</td>
<td>4.68</td>
<td>0.3</td>
<td>∗</td>
</tr>
<tr>
<td>Reduced nicotinamide adenine dinucleotide (NADH)</td>
<td>666 → 649</td>
<td>0.006</td>
<td>26</td>
<td>20</td>
<td>ESI+</td>
<td>4.66</td>
<td>0.3</td>
<td>∗</td>
</tr>
<tr>
<td><i>Nicotinic acid adenine dinucleotide (NAAD)</i></td>
<td>665 → 428</td>
<td>0.006</td>
<td>24</td>
<td>24</td>
<td>ESI+</td>
<td>4.21</td>
<td>0.15</td>
<td>∗</td>
</tr>
<tr>
<td>Nicotinamide adenine dinucleotide (NAD<sup>+</sup>)</td>
<td>664 → 428</td>
<td>0.006</td>
<td>26</td>
<td>26</td>
<td>ESI+</td>
<td>4.15</td>
<td>0.15</td>
<td>∗∗</td>
</tr>
<tr>
<td><i>Adenosine 5′-diphosphate (ADP)</i></td>
<td>428 → 136</td>
<td>0.006</td>
<td>30</td>
<td>26</td>
<td>ESI+</td>
<td>4.2</td>
<td>0.15</td>
<td>∗</td>
</tr>
<tr>
<td><i>Inosine monophosphate (IMP)</i></td>
<td>349 → 137</td>
<td>0.006</td>
<td>14</td>
<td>22</td>
<td>ESI+</td>
<td>3.85</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Adenosine 5′-monophosphate (AMP)</td>
<td>348 → 136</td>
<td>0.006</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>3.88</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>Uridine monophosphate (UMP)</i></td>
<td>325 → 97</td>
<td>0.006</td>
<td>20</td>
<td>12</td>
<td>ESI+</td>
<td>3.72</td>
<td>0.2</td>
<td>∗</td>
</tr>
<tr>
<td><i>Cytidine monophosphate (CMP)</i></td>
<td>324 → 112</td>
<td>0.006</td>
<td>16</td>
<td>22</td>
<td>ESI+</td>
<td>3.91</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
<tr>
<td>Uridine</td>
<td>245 → 113</td>
<td>0.006</td>
<td>18</td>
<td>16</td>
<td>ESI+</td>
<td>1.01</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
</tbody>
</table>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Representative chromatogram of metabolites listed in <a href="#tbl4">Table 4</a>
</div>
</figcaption>
</figure>
<ol type="a">
<li>Amide column (Acquity UPLC BEH amide column, 2.1 × 100 mm, 1.7 μm) is used for the separation.</li>
<li>Acetonitrile with 0.1% formic acid (A) and water with 0.1% formic acid (B) are used as mobile phases. The gradient setting is listed in <a href="#tbl5">Table 5</a>.
<table id="tbl5">
<caption>Table 5. Gradient setting for program 2</caption>
<thead>
<tr>
<th>Time (min)</th>
<th>Flowrate (mL/min)</th>
<th>A%</th>
<th>B%</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial</td>
<td>0.4</td>
<td>80</td>
<td>20</td>
</tr>
<tr>
<td>2</td>
<td>0.4</td>
<td>80</td>
<td>20</td>
</tr>
<tr>
<td>3</td>
<td>0.4</td>
<td>20</td>
<td>80</td>
</tr>
<tr>
<td>5</td>
<td>0.4</td>
<td>20</td>
<td>80</td>
</tr>
<tr>
<td>6</td>
<td>0.4</td>
<td>80</td>
<td>20</td>
</tr>
<tr>
<td>10</td>
<td>0.4</td>
<td>80</td>
<td>20</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>
<b>Program 3:</b> For the quantification of acetate, acetyl-CoA and metabolites involved in glycolysis and pentose phosphate pathway. LC-MS/MS conditions and parameters are displayed in <a href="#tbl6">Table 6</a>. Representative chromatograms are shown in <a href="#fig6">Figure 6</a>.
<table id="tbl6">
<caption>Table 6. LC-MS/MS conditions and parameters for program 3</caption>
<thead>
<tr>
<th>Name</th>
<th>MRM</th>
<th>Dwell (s)</th>
<th>CV (V)</th>
<th>CE (V)</th>
<th>Mode</th>
<th>Retention time (min)</th>
<th>Time window ± (min)</th>
<th>Performance</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glucose (Glc)</td>
<td>179 → 89</td>
<td>0.021</td>
<td>22</td>
<td>6</td>
<td>ESI-</td>
<td>1.41</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td>Glucose 6-phosphate (G6P)</td>
<td>259 → 97</td>
<td>0.021</td>
<td>24</td>
<td>14</td>
<td>ESI-</td>
<td>1.09</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td>Fructose 6-phosphate (F6P)</td>
<td>259 → 97</td>
<td>0.021</td>
<td>24</td>
<td>14</td>
<td>ESI-</td>
<td>1.09</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td>Fructose 1,6-bisphosphate (FBP)</td>
<td>339 → 97</td>
<td>0.021</td>
<td>36</td>
<td>18</td>
<td>ESI-</td>
<td>1.02</td>
<td>0.15</td>
<td>∗∗</td>
</tr>
<tr>
<td>Glyceraldehyde 3-phosphate (GADP)</td>
<td>169 → 97</td>
<td>0.021</td>
<td>14</td>
<td>8</td>
<td>ESI-</td>
<td>1.12</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td>3-phosphoglycerate (3PG)</td>
<td>185 → 79</td>
<td>0.021</td>
<td>26</td>
<td>14</td>
<td>ESI-</td>
<td>1.18</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>2-phosphoglycerate (2PG)</td>
<td>185 → 79</td>
<td>0.021</td>
<td>26</td>
<td>14</td>
<td>ESI-</td>
<td>1.18</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>Phosphoenolpyruvate (PEP)</td>
<td>167 → 79</td>
<td>0.021</td>
<td>18</td>
<td>14</td>
<td>ESI-</td>
<td>1.09</td>
<td>0.1</td>
<td>∗∗</td>
</tr>
<tr>
<td><i>3-phosphonopyruvate (3-P-Pyr)</i></td>
<td>185 → 87</td>
<td>0.021</td>
<td>30</td>
<td>20</td>
<td>ESI+</td>
<td>2.17</td>
<td>0.2</td>
<td>∗</td>
</tr>
<tr>
<td><i>Sedoheptulose 7-phosphate (S7P)</i></td>
<td>289 → 97</td>
<td>0.021</td>
<td>40</td>
<td>10</td>
<td>ESI-</td>
<td>1.11</td>
<td>0.1</td>
<td>∗</td>
</tr>
<tr>
<td>Ribulose 5-phosphate (R5P)</td>
<td>229 → 97</td>
<td>0.021</td>
<td>26</td>
<td>12</td>
<td>ESI-</td>
<td>1.12</td>
<td>0.1</td>
<td>∗</td>
</tr>
<tr>
<td>Erythrose 4-phosphate (E4P)</td>
<td>199 → 97</td>
<td>0.021</td>
<td>100</td>
<td>8</td>
<td>ESI-</td>
<td>1.09</td>
<td>0.1</td>
<td>∗</td>
</tr>
<tr>
<td>Acetate</td>
<td>61 → 44.4</td>
<td>0.011</td>
<td>36</td>
<td>80</td>
<td>ESI+</td>
<td>1.45</td>
<td>0.1</td>
<td>∗</td>
</tr>
<tr>
<td>Acetyl-CoA</td>
<td>808 → 408</td>
<td>0.011</td>
<td>82</td>
<td>58</td>
<td>ESI-</td>
<td>1.0</td>
<td>0.1</td>
<td>∗</td>
</tr>
</tbody>
</table>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Representative chromatogram of metabolites listed in <a href="#tbl6">Table 6</a>
</div>
</figcaption>
</figure>
<ol type="a">
<li>HILIC column (Acquity UPLC BEH HILIC column, 2.1 × 100 mm, 1.7 μm) is used for the separation.</li>
<li>Acetonitrile (A) and 10 mM ammonium bicarbonate aqueous solution (B) are used as mobile phases. The gradient setting is listed in <a href="#tbl7">Table 7</a>.
<table id="tbl7">
<caption>Table 7. Gradient setting for program 3</caption>
<thead>
<tr>
<th>Time (min)</th>
<th>Flowrate (mL/min)</th>
<th>A%</th>
<th>B%</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial</td>
<td>0.2</td>
<td>10</td>
<td>90</td>
</tr>
<tr>
<td>2</td>
<td>0.2</td>
<td>10</td>
<td>90</td>
</tr>
<tr>
<td>5</td>
<td>0.2</td>
<td>5</td>
<td>95</td>
</tr>
<tr>
<td>6</td>
<td>0.2</td>
<td>10</td>
<td>90</td>
</tr>
<tr>
<td>10</td>
<td>0.2</td>
<td>10</td>
<td>90</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>
<b>Program 4:</b> For the quantification of lysophosphatidic acid (LPA), lysophosphatidylcholine (LPC) and phosphatidylcholine (PC). LC-MS/MS conditions and parameters are displayed in <a href="#tbl8">Table 8</a>. Representative chromatograms are shown in <a href="#fig7">Figure 7</a>.
<table id="tbl8">
<caption>Table 8. LC-MS/MS conditions and parameters for program 4</caption>
<thead>
<tr>
<th>Name</th>
<th>MRM</th>
<th>Dwell (s)</th>
<th>CV (V)</th>
<th>CE (V)</th>
<th>Mode</th>
<th>Retention time (min)</th>
<th>Time window ± (min)</th>
<th>Performance</th>
</tr>
</thead>
<tbody>
<tr>
<td>16:0 Lyso PA</td>
<td>409 → 153</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI-</td>
<td>5.86</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>18:0 Lyso PA</td>
<td>437 → 153</td>
<td>0.009</td>
<td>30</td>
<td>32</td>
<td>ESI-</td>
<td>5.83</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>18:1 Lyso PA</td>
<td>435 → 153</td>
<td>0.009</td>
<td>30</td>
<td>33</td>
<td>ESI-</td>
<td>5.84</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td>18:2 Lyso PA</td>
<td>433 → 153</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI-</td>
<td>5.87</td>
<td>0.3</td>
<td>∗∗</td>
</tr>
<tr>
<td>20:4 Lyso PA</td>
<td>457 → 153</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI-</td>
<td>5.85</td>
<td>0.2</td>
<td>∗∗</td>
</tr>
<tr>
<td><i>16:0 Lyso PC</i></td>
<td>496.5 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>6.02</td>
<td>0.15</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>18:0 Lyso PC</i></td>
<td>524.6 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>5.96</td>
<td>0.15</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>18:1 Lyso PC</i></td>
<td>522.6 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>5.97</td>
<td>0.15</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>18:2 Lyso PC</i></td>
<td>520.6 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>6.03</td>
<td>0.15</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>20:4 Lyso PC</i></td>
<td>544.6 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>6.01</td>
<td>0.15</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>34:1 PC</i></td>
<td>760.7 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>5.71</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>34:2 PC</i></td>
<td>758.7 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>5.7</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>36:2 PC</i></td>
<td>786.7 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>5.69</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>36:3 PC</i></td>
<td>784.7 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>5.68</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
<tr>
<td><i>36:4 PC</i></td>
<td>782.7 → 184.2</td>
<td>0.009</td>
<td>30</td>
<td>30</td>
<td>ESI+</td>
<td>5.69</td>
<td>0.1</td>
<td>∗∗∗</td>
</tr>
</tbody>
</table>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Representative chromatogram of metabolites listed in <a href="#tbl8">Table 8</a>
</div>
</figcaption>
</figure>
<ol type="a">
<li>HILIC column (Acquity UPLC BEH HILIC column, 2.1 × 100 mm, 1.7 μm) is used for the separation.</li>
<li>Acetonitrile (A) and 10 mM ammonium bicarbonate aqueous solution (B) are used as mobile phases. The gradient setting is listed in <a href="#tbl9">Table 9</a>.
<table id="tbl9">
<caption>Table 9. Gradient setting for program 4</caption>
<thead>
<tr>
<th>Time (min)</th>
<th>Flowrate (mL/min)</th>
<th>A%</th>
<th>B%</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial</td>
<td>0.2</td>
<td>95</td>
<td>5</td>
</tr>
<tr>
<td>2</td>
<td>0.2</td>
<td>95</td>
<td>5</td>
</tr>
<tr>
<td>4</td>
<td>0.2</td>
<td>10</td>
<td>90</td>
</tr>
<tr>
<td>7</td>
<td>0.2</td>
<td>10</td>
<td>90</td>
</tr>
<tr>
<td>9</td>
<td>0.2</td>
<td>95</td>
<td>5</td>
</tr>
<tr>
<td>15</td>
<td>0.2</td>
<td>95</td>
<td>5</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>
<b>Program 5:</b> For the quantification of lipids. LC-MS/MS conditions and parameters are displayed in <a href="#tbl10">Table 10</a>. Representative chromatograms are shown in <a href="#fig8">Figure 8</a>.
<table id="tbl10">
<caption>Table 10. LC-MS/MS conditions and parameters for program 5</caption>
<thead>
<tr>
<th>Name</th>
<th>SIR</th>
<th>Dwell (s)</th>
<th>CV (V)</th>
<th>Mode</th>
<th>Retention time (min)</th>
<th>Time window ± (min)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Arachidonic acid</td>
<td>303.23</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>7.94</td>
<td>0.1</td>
</tr>
<tr>
<td>Cholesterol</td>
<td>369.4</td>
<td>0.005</td>
<td>40</td>
<td>ESI+</td>
<td>12.37</td>
<td>0.2</td>
</tr>
<tr>
<td>DL-alpha-tocopherol acetate</td>
<td>429.37</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>8.96</td>
<td>0.1</td>
</tr>
<tr>
<td>Linoleic acid</td>
<td>279.23</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>8.05</td>
<td>0.1</td>
</tr>
<tr>
<td>Linolenic acid</td>
<td>277.22</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>7.81</td>
<td>0.1</td>
</tr>
<tr>
<td>Myristic acid</td>
<td>227.2</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>7.98</td>
<td>0.1</td>
</tr>
<tr>
<td>Oleic acid</td>
<td>281.2</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>8.42</td>
<td>0.1</td>
</tr>
<tr>
<td>Palmitic acid</td>
<td>255.2</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>8.41</td>
<td>0.1</td>
</tr>
<tr>
<td>Palmitoleic acid</td>
<td>253.22</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>8.0</td>
<td>0.1</td>
</tr>
<tr>
<td>Stearic acid</td>
<td>283.26</td>
<td>0.005</td>
<td>40</td>
<td>ESI-</td>
<td>8.98</td>
<td>0.1</td>
</tr>
</tbody>
</table>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Representative chromatogram of metabolites listed in <a href="#tbl10">Table 10</a>
</div>
</figcaption>
</figure>
<ol type="a">
<li>C18 column (Acquity UPLC BEH C18 column, 2.1 × 100 mm, 1.7 μm) is used for the separation.</li>
<li>Acetonitrile (A) and 10 mM ammonium bicarbonate aqueous solution (B) are used as mobile phases. The gradient setting is listed in <a href="#tbl11">Table 11</a>.
<table id="tbl11">
<caption>Table 11. Gradient setting for program 5</caption>
<thead>
<tr>
<th>Time (min)</th>
<th>Flowrate (mL/min)</th>
<th>A%</th>
<th>B%</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial</td>
<td>0.4</td>
<td>0.1</td>
<td>99.9</td>
</tr>
<tr>
<td>5</td>
<td>0.4</td>
<td>0.1</td>
<td>99.9</td>
</tr>
<tr>
<td>6</td>
<td>0.4</td>
<td>99.9</td>
<td>0.1</td>
</tr>
<tr>
<td>11</td>
<td>0.4</td>
<td>99.9</td>
<td>0.1</td>
</tr>
<tr>
<td>12</td>
<td>0.4</td>
<td>0.1</td>
<td>99.9</td>
</tr>
<tr>
<td>15</td>
<td>0.4</td>
<td>0.1</td>
<td>99.9</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The method we developed for the quantification of lipids listed in <a href="#tbl10">Table 10</a>, mainly free fatty acids, is used to quantify that in serum products but not in hESCs. Most of the free fatty acids, except PA and SA, extracted from 1 million hESCs are below the limit of detection. Because we define “Performance” as how easily peak identification could be done in hESC sample, “Performance” is not listed in <a href="#tbl10">Table 10</a>.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Fresh mobile phases are prepared before running LC-MS. Mobile phases are degassed by ultrasonic bathing for 15 min.</div>
<div class="note">
<span class="note-title">Note:</span> Status of column and the length of pipeline could affect retention time.</div>
<div class="note">
<span class="note-title">Note:</span> Guard columns are not listed here but recommended to be used to protect the main columns. Guard column would increase column pressure and extend retention time.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Instrument performance is assessed by acquiring a QC sample (mixture of all samples). Column pressure, retention time and peak intensity of metabolites in different injections of the QC sample should be consistent.</div>
<div class="note">
<span class="note-title">Note:</span> Exact quantification of target compound could be achieved if a standard curve is constructed.</div>
<h3 id="sec3.10">LC-MS data analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>1 h</b> <b>for step 25</b>
</div>
<div class="timing">
<span class="timing-title">Timing:</span> <b>1 h</b> <b>for step 26</b>
</div>
<p>Each sample processed by the LC-MS system will generate a Total Ion Chromatogram (TIC) which is the sum of different channels obtained under MRM or SIR settings corresponding to specific metabolites. Metabolite identification could be processed manually by comparing the retention time of target compound in the standard and the sample or using software like TargetLynx.</p>
<ol start="25">
<li>The software TargetLynx XS is incorporated in MassLynx (Waters). Peak identification is described in <a href="#fig9">Figure 9</a>.
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/923-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Procedure of metabolite identification using TargetLynx</div>
<p>(A) Selection of “Edit Method” function in TargetLynx.</p>
<p>(B) Add a new compound profile.</p>
<p>(C) Select a retention range to enable automatic information input in method editor inset (B).</p>
<p>(D) Name the compound and choose the criteria for peak selection and response types.</p>
<p>(E) Selection of “Process Samples” function in TargetLynx.</p>
<p>(F) Selection of sample range and process method.</p>
<p>(G) Chromatogram showing peak integration using the process method.</p>
</figcaption>
</figure>
<ol type="a">
<li>Edit Method (<a href="#fig9">Figure 9</a>A).</li>
<li>Generate a new profile of compound (<a href="#fig9">Figure 9</a>B).</li>
<li>Open the chromatogram of standards, select the channel of target compound, and right drag to set a range of retention time (<a href="#fig9">Figure 9</a>C). The information of MRM and time range will be automatically recorded in TargetLynx XS Method Editor (<a href="#fig9">Figure 9</a>D).</li>
<li>The nearest peak or the largest peak within the time range should be set as the target, and peak area or peak height could be chosen as response. Multiple compound settings could be integrated into one TargetLynx Method. Edited method is saved for future data processing.
<div class="note">
<span class="note-title">Note:</span> For metabolites with overlapping peaks, reduce the injection volume until peaks are separated.</div>
</li>
<li>Process Samples (<a href="#fig9">Figure 9</a>E).</li>
<li>Dataset setting (<a href="#fig9">Figure 9</a>F).
<ol type="i">
<li>Set the range of samples.</li>
<li>Select TargetLynx method.</li>
</ol>
</li>
<li>Review results (<a href="#fig9">Figure 9</a>G). Verify if peaks are correctly found and integrated. If not, you could go back and modify the TargetLynx Method or manually choose and integrate the peaks.</li>
<li>Finally, the result could be saved and exported for further analysis.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> There are other freely available softwares like Skyline that can be used to analyze RAW data.</div>
<ol start="26">
<li>Normalization of data.</li>
</ol>
<p>Exported peak responses (peak area or peak intensity) from TargetLynx are then normalized by internal control, cell number or protein level and control group responses, respectively.</p>
<div class="note">
<span class="note-title">Note:</span> We used peak area for quantification. Theoretically, if peak width remains the same, peak intensity could also be used for quantification.</div>
<p>Take 2 groups of treatments and 5 replicates per treatment for example:</p>
<p>Peak areas of internal control (norvaline, heptadecanoate or coprostan-3-ol) in 5 replicates of 2 groups:</p>
<p>Control group: A1i, A2i, A3i, A4i, A5i; Treatment group: A6i, A7i, A8i, A9i, A10i.</p>
<p>Peak areas of a certain metabolite Q in 5 replicates of 2 groups:</p>
<p>Control group: A1, A2, A3, A4, A5; Treatment group: A6, A7, A8, A9, A10.</p>
<p>Cell number ratio:</p>
<p class="chem">Control group: 1; Treatment group: N, calculated by <math> <mrow> <mtext>N</mtext> <mo>=</mo> <mfrac> <mrow> <mi>A</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>u</mi> <mi>n</mi> <mi>t</mi> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>t</mi> <mi>r</mi> <mi>e</mi> <mi>a</mi> <mi>t</mi> <mi>e</mi> <mi>d</mi> <mspace></mspace> <mi>w</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mi>s</mi> </mrow> <mrow> <mi>A</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mi>c</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>u</mi> <mi>n</mi> <mi>t</mi> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>n</mi> <mi>t</mi> <mi>r</mi> <mi>o</mi> <mi>l</mi> <mspace></mspace> <mi>w</mi> <mi>e</mi> <mi>l</mi> <mi>l</mi> <mi>s</mi> </mrow> </mfrac> </mrow> </math></p>
<p>Protein concentration:</p>
<p>Control group: P1, P2, P3, P4, P5; Treatment group: P6, P7, P8, P9, P10.</p>
<p>Normalized to cell number:</p>
<p class="chem">Relative peak area ratio of metabolite Q is: <math> <mrow> <mover> <mrow> <mfrac> <mrow> <mi>A</mi> <mn>6</mn> <mo>×</mo> <mi>A</mi> <mn>1</mn> <mi>i</mi> </mrow> <mrow> <mi>N</mi> <mo>×</mo> <mi>A</mi> <mn>6</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>1</mn> </mrow> </mfrac> <mo>+</mo> <mfrac> <mrow> <mi>A</mi> <mn>7</mn> <mo>×</mo> <mi>A</mi> <mn>2</mn> <mi>i</mi> </mrow> <mrow> <mi>N</mi> <mo>×</mo> <mi>A</mi> <mn>7</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>2</mn> </mrow> </mfrac> <mo>+</mo> <mfrac> <mrow> <mi>A</mi> <mn>8</mn> <mo>×</mo> <mi>A</mi> <mn>3</mn> <mi>i</mi> </mrow> <mrow> <mi>N</mi> <mo>×</mo> <mi>A</mi> <mn>8</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>3</mn> </mrow> </mfrac> <mo>+</mo> <mfrac> <mrow> <mi>A</mi> <mn>9</mn> <mo>×</mo> <mi>A</mi> <mn>4</mn> <mi>i</mi> </mrow> <mrow> <mi>N</mi> <mo>×</mo> <mi>A</mi> <mn>9</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>4</mn> </mrow> </mfrac> <mo>+</mo> <mfrac> <mrow> <mi>A</mi> <mn>10</mn> <mo>×</mo> <mi>A</mi> <mn>5</mn> <mi>i</mi> </mrow> <mrow> <mi>N</mi> <mo>×</mo> <mi>A</mi> <mn>10</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>5</mn> </mrow> </mfrac> </mrow> <mo>¯</mo> </mover> </mrow> </math>  ± SD</p>
<p>Normalized to protein level:</p>
<p class="chem">Relative peak area ratio of metabolite Q is: <math> <mrow> <mover> <mrow> <mfrac> <mrow> <mi>A</mi> <mn>6</mn> <mo>×</mo> <mi>A</mi> <mn>1</mn> <mi>i</mi> <mo>×</mo> <mi>P</mi> <mn>1</mn> </mrow> <mrow> <mi>A</mi> <mn>6</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>1</mn> <mo>×</mo> <mi>P</mi> <mn>6</mn> </mrow> </mfrac> <mo>+</mo> <mfrac> <mrow> <mi>A</mi> <mn>7</mn> <mo>×</mo> <mi>A</mi> <mn>2</mn> <mi>i</mi> <mo>×</mo> <mi>P</mi> <mn>2</mn> </mrow> <mrow> <mi>A</mi> <mn>7</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>2</mn> <mo>×</mo> <mi>P</mi> <mn>7</mn> </mrow> </mfrac> <mo>+</mo> <mfrac> <mrow> <mi>A</mi> <mn>8</mn> <mo>×</mo> <mi>A</mi> <mn>3</mn> <mi>i</mi> <mo>×</mo> <mi>P</mi> <mn>3</mn> </mrow> <mrow> <mi>A</mi> <mn>8</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>3</mn> <mo>×</mo> <mi>P</mi> <mn>8</mn> </mrow> </mfrac> <mo>+</mo> <mfrac> <mrow> <mi>A</mi> <mn>9</mn> <mo>×</mo> <mi>A</mi> <mn>4</mn> <mi>i</mi> <mo>×</mo> <mi>P</mi> <mn>4</mn> </mrow> <mrow> <mi>A</mi> <mn>9</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>4</mn> <mo>×</mo> <mi>P</mi> <mn>9</mn> </mrow> </mfrac> <mo>+</mo> <mfrac> <mrow> <mi>A</mi> <mn>10</mn> <mo>×</mo> <mi>A</mi> <mn>5</mn> <mi>i</mi> <mo>×</mo> <mi>P</mi> <mn>5</mn> </mrow> <mrow> <mi>A</mi> <mn>10</mn> <mi>i</mi> <mo>×</mo> <mi>A</mi> <mn>5</mn> <mo>×</mo> <mi>P</mi> <mn>10</mn> </mrow> </mfrac> </mrow> <mo>¯</mo> </mover> </mrow> </math>  ± SD</p>
<div class="critical">
<span class="critical-title">Critical:</span> Sub-optimal data and outlier data should not be included.</div>
<p>Finally, the normalized data are visualized as heatmap or other graphs using softwares like Prism (Graphpad), Simca-P (Umetrics), SigmaStat (Systat), SPSS (IBM) and RStudio coupled with statistical packages.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The protocol we provided is a semi-quantitative measurement of extracellular and intracellular metabolites. It could be modified to be fully quantitative if a standard curve is built for each metabolite. The sample response of the corresponding metabolite should be within the lineage range, which could be pre-determined by looking in the literature to see how many pmols are consumed/produced by hESCs and then adjusting the final concentration and the injection volume of samples to fit in the range.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is designed for analysis of targeted metabolites using triple quadrupole LC-MS. Due to the requirement of setting parameters for each metabolite before running samples, triple quadrupole LC-MS is not ideal for untargeted analysis or global metabolomics. However, the method we provided for metabolite extraction can be applied for other MS assays. Standards should be used to confirm parameter setting, align eluting time and establish the range of concentration for quantification. More complete metabolite standards could boost the application of this protocol to analyze metabolic processes in more details. Because derivatization reagents and antioxidants are not involved during sample preparation in this protocol, one needs to consider the possibility of getting low response or even failure to detect some components.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>How to maintain cell viability while thawing them? (step 4)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Frozen cell stock, typically 0.5–1 mL per cryovial, should be stored in liquid nitrogen or −150°C freezer to avoid losing viability. When thawing cells, the vial of frozen stock from liquid nitrogen should be put directly into a 37°C water bath. Avoid letting cells thaw at room temperature (15°C–30°C), and be quick during the whole process. Gently stir the water while holding the cryovials upright (do not shake the tubes), watch closely and then transfer cells (wipe the cryovials with 70% ethanol before moving them into biosafety cabinet) into an empty 15-mL tube in biosafety cabinet when only a small ice particle is left floating. When diluting the cells with E8 + 10 μM Y-27632, add the medium drop by drop and gently shake the tube during the process to minimize the shock from osmolarity changes. Other steps are described in the step 4 of this protocol. Be careful when aspirating and discarding the supernatant after centrifugation to avoid losing cells.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Should extracted solution, for instance solution in Tube 2 described in cell metabolic extraction Method 1, be nitrogen blowdown and redissolved for LC-MS analysis? (step 12)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Nitrogen blowdown and redissolving with small volume of solvent is a good way to increase sample concentration. However, more steps of sample preparation lead to increased risk of losing or altering metabolites. Bypassing the drying and redissolving steps can save time and decrease sample loss, but the downside is the possibility of precipitation when sample is eluted through the column. Based on our own experience, we have not encountered the problem of blocking the column when skipping the drying &amp; redissolving steps.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>How to develop an MRM assay? (step 20)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Target compound is dissolved with methanol (LC/MS grade) to a final concentration of 0.1–1 μg/mL, and 1 mL of this solution is used to directly infuse into ion source (bypass column separation). For the Waters TQD spectrometer used in this protocol, a built-in add-on in Waters Console called IntelliStart is used for developing MRM and SIR methods. After inputting the molecular weight of target compound and setting the lowest fragment ion mass, it will come out with a report with details about parent and daughter ion masses together with the corresponding optimized cone voltage and collision energy. Meanwhile, search the literature or public databases like HMDB (<a href="https://hmdb.ca">https://hmdb.ca</a>), LIPID MAPS (<a href="https://www.lipidmaps.org">https://www.lipidmaps.org</a>) and METLIN (<a href="https://metlin.scripps.edu">https://metlin.scripps.edu</a>) to find reported parameters.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>How to identify the target peak if there are multiple peaks in one channel of the chromatogram? (step 25)</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Running standards before and after the sequence of samples is recommended. The correct target peak should have the same retention time as the standards. In the absence of a standard, isotopic ratio should be checked to match the target compound.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>How reliable is the provided information about identified metabolites? (step 25)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Identification of metabolite in this protocol is based on comparison with standards or matching multiple fragmentation patterns (MRM) and isotopic ratio. Mass spectrometry has its limitations in identifying structure and chiral isomers unless they could be well separated by chromatography.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Beers, J., Gulbranson, D.R., George, N., Siniscalchi, L.I., Jones, J., Thomson, J.A., and Chen, G. (2012). Passaging and colony expansion of human pluripotent stem cells by enzyme-free dissociation in chemically defined culture conditions. Nat Protoc <i>7</i>, 2029-2040. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref1">View at publisher</a></p>
<p id="bib2">Chen, G., Gulbranson, D.R., Hou, Z., Bolin, J.M., Ruotti, V., Probasco, M.D., Smuga-Otto, K., Howden, S.E., Diol, N.R., Propson, N.E., et al. (2011a). Chemically defined conditions for human iPSC derivation and culture. Nat Methods <i>8</i>, 424-429. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref2">View at publisher</a></p>
<p id="bib3">Chen, G., Gulbranson, D.R., Hou, Z., Bolin, J.M., Ruotti, V., Probasco, M.D., Smuga-Otto, K., Howden, S.E., Diol, N.R., Propson, N.E., et al. (2011b). Chemically defined conditions for human iPSC derivation and culture. Nature Methods <i>8</i>, 424-U76. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref3">View at publisher</a></p>
<p id="bib4">Chen, G., Gulbranson, D.R., Yu, P., Hou, Z., and Thomson, J.A. (2012). Thermal stability of fibroblast growth factor protein is a determinant factor in regulating self-renewal, differentiation, and reprogramming in human pluripotent stem cells. Stem Cells <i>30</i>, 623-630. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref4">View at publisher</a></p>
<p id="bib5">Jain, M., Nilsson, R., Sharma, S., Madhusudhan, N., Kitami, T., Souza, A.L., Kafri, R., Kirschner, M.W., Clish, C.B., and Mootha, V.K. (2012). Metabolite profiling identifies a key role for glycine in rapid cancer cell proliferation. Science <i>336</i>, 1040-1044. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref5">View at publisher</a></p>
<p id="bib6">Meng, Y., Ren, Z., Xu, F., Zhou, X., Song, C., Wang, V.Y.-F., Liu, W., Lu, L., Thomson, J.A., and Chen, G. (2018). Nicotinamide promotes cell survival and differentiation as kinase inhibitor in human pluripotent stem cells. Stem cell reports <i>11</i>, 1347-1356. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref6">View at publisher</a></p>
<p id="bib7">Racine, J.S. (2012). RStudio: a platform-independent IDE for R and Sweave. JSTOR. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref7">View at publisher</a></p>
<p id="bib8">Ser, Z., Liu, X., Tang, N.N., and Locasale, J.W. (2015). Extraction parameters for metabolomics from cultured cells. Analytical biochemistry <i>475</i>, 22-28. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref8">View at publisher</a></p>
<p id="bib9">Song, C.C., Xu, F.X., Ren, Z.L., Zhang, Y.M., Meng, Y., Yang, Y.Q., Lingadahalli, S., Cheung, E., Li, G., Liu, W.W., et al. (2019). Elevated exogenous pyruvate potentiates mesodermal differentiation through metabolic modulation and AMPK/mTOR pathway in human embryonic stem cells. Stem Cell Reports <i>13</i>, 338-351. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref9">View at publisher</a></p>
<p id="bib10">Yang, Y., Ren, Z., Xu, F., Meng, Y., Zhang, Y., Ai, N., Long, Y., Fok, H.I., Deng, C., and Zhao, X. (2019). Endogenous IGF signaling directs heterogeneous mesoderm differentiation in human embryonic stem cells. Cell Reports <i>29</i>, 3374-3384.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref10">View at publisher</a></p>
<p id="bib11">Zhang, H., Badur, M.G., Divakaruni, A.S., Parker, S.J., Jäger, C., Hiller, K., Murphy, A.N., and Metallo, C.M. (2016). Distinct metabolic states can support self-renewal and lipogenesis in human pluripotent stem cells under different culture conditions. Cell Reports <i>16</i>, 1536-1547. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00447-0/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Guokai Chen (<a href="mailto:guokaichen@um.edu.mo">guokaichen@um.edu.mo</a>).</p>
<h4>Materials availability</h4>
<p>No new materials were generated in this study.</p>
<h4>Data and code availability</h4>
<p>The study did not generate unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>We would like to thank the technical support by the Proteomics, Metabolomics and Drug Development Core Facility at the Faculty of Health Sciences, University of Macau. This project was funded by the <a href="https://doi.org/10.13039/501100004733">University of Macau</a> (file nos. MYRG2018-00135-FHS and MYRG2019-00147-FHS) and also by the <a href="https://doi.org/10.13039/501100006469">Science and Technology Development Fund, Macau</a> SAR (file nos. 0059/2019/A1 and 0123/2019/A3).</p>
<h3>Author contributions</h3>
<p>G.C. and F.X. conceived and designed the study. F.X., C.S., and W.L. conducted LC-MS metabolome analysis. F.X., C.S., W.L., and G.C. wrote the manuscript. Most authors contributed to the editing and proofreading of the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>