<article>
<section>
<h1 id="header">SpineRacks and SpinalJ for efficient analysis of neurons in a 3D reference atlas of the mouse spinal cord</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Felix Fiederling,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Luke A. Hammond,<sup><a href="#aff1">1</a></sup> David Ng,<sup><a href="#aff1">1</a></sup> Carol Mason,<sup><a href="#aff1">1</a></sup> and Jane Dodd<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Zuckerman Institute, Columbia University, New York, NY 10027, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ff2397@columbia.edu">ff2397@columbia.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:jd18@columbia.edu">jd18@columbia.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100897">10.1016/j.xpro.2021.100897</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Spatial analysis of spinal neurons is currently limited by a lack of tools for efficient preparation and imaging of the whole spinal cord (SC) and the absence of a 3D reference atlas. Here, we describe protocols for efficient sectioning of whole SC using SpineRacks and subsequent image registration, atlas mapping, and 3D analysis of cells and projections, using SpinalJ. Together, these tools enable high-throughput analyses of adult mouse SC and direct comparison of spatial information of neurons between animals and studies.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref5">Fiederling et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>SpineRacks facilitate efficient sectioning and imaging of whole mouse spinal cord</li>
<li>SpinalJ allows registration, mapping and analysis of spinal sections</li>
<li>A 3D atlas provides a common framework for comparative analyses of neurons</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1177-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Given the technical limitations of traditional histological methods and more recent clearing approaches, SC analyses in mice have typically focused on sparse sampling through the entire SC or on subsets of spinal segments, resulting in incomplete characterization of the distributions and diversity of spinal neurons and their projections along the rostro-caudal (r-c) axis. Moreover, data generated by sparse sampling of whole SC is difficult to map accurately to defined r-c positions without the context of neighboring sections or vertebral landmarks. To overcome these problems, we have developed tools to improve sectioning efficiency of whole SC and to facilitate section registration, atlas mapping and 3D analysis of SC sections (<a href="#bib5">Fiederling et al., 2021</a>). These tools fill the prevailing gap in methodology for SC analysis that limits detailed characterization of spinal cell types and circuitry. Moreover, mapping of section data to a common coordinate framework using SpinalJ permits SC wide analysis with high spatial precision and comparison of results from different samples and across research groups.</p>
<p>In the protocol below, we describe in detail the use of SpineRacks and SpinalJ for efficient analysis of whole mouse SC. To follow along with this protocol and for easy testing and troubleshooting of SpinalJ, we provide an example dataset that comprises raw image data of a full spinal cord sample, all intermediate processing files generated during analysis, and our resulting output files. A video tutorial that guides the user through all steps is also included. Links to these resources are provided in the <a href="#key-resources-table">key resources table</a>.</p>
<h3 id="sec1.1">Print SpineRacks</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h print time for 50 SpineRacks on the Ultimaker3 printer (may vary using other printers)</div>
<ol>
<li>Download SpineRack.stl print files (<a href="https://github.com/felixfiederling/SpinalJ/blob/main/SpineRack.stl">https://github.com/felixfiederling/SpinalJ/blob/main/SpineRack.stl</a>).
<ol type="a">
<li>Slice file for printing.<br> We used the following settings in Cura v4.8 for printing on the Ultimaker3 printer: Material: natural PVA; print core: BB 0.4; layer height: 0.15 mm; print temp: 220°C, bed: 60°C; infill: 20%; build plate adhesion: 3 mm brim.</li>
</ol>
</li>
<li>Print using PVA filament.</li>
<li>Store SpineRacks in a sealed container (such as a 50 mL screw top tube) at room temperature (20°C–22°C) in the dark with a desiccant bag to prevent absorption of ambient moisture.</li>
</ol>
<h3 id="sec1.2">SpinalJ installation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15–30 min</div>
<p>We have successfully tested SpinalJ on systems running Microsoft Windows 10 using Fiji version 1.53c. Memory requirements depend strongly on the size of the input images, but at least 16 GB RAM are recommended. SpinalJ is generally compatible with MacOS, but does not support machine learning detection of cells and projections with Ilastik (step 20 c).</p>
<ol start="4">
<li>Download and install Fiji software (<a href="http://fiji.sc/Fiji">http://fiji.sc/Fiji</a>).</li>
<li>Download and install SpinalJ for Fiji (<a href="https://github.com/felixfiederling/SpinalJ">https://github.com/felixfiederling/SpinalJ</a>).
<ol type="a">
<li>Copy the SpinalJ.jar file into your Fiji plugins folder and restart Fiji to install the plugin (Alternatively, use <i>Plugins &gt; Install</i>).</li>
<li>Store the segmentation masks “Split_mask.roi” and “Split_mask_scaled.roi” on your computer.</li>
<li>Download the SpinalJ atlas files (<a href="https://data.mendeley.com/datasets/4rrggzv5d5/1">https://data.mendeley.com/datasets/4rrggzv5d5/1</a>) and store on your computer.</li>
</ol>
</li>
<li>Install Attenuation Correction (<a href="#bib3">Biot et al., 2008</a>). Download and copy into your Fiji plugins folder and restart Fiji.<br> Attenuation Correction can be downloaded at: <a href="https://imagejdocu.tudor.lu/plugin/stacks/attenuation_correction/start">https://imagejdocu.tudor.lu/plugin/stacks/attenuation_correction/start</a>
</li>
<li>Install TurboReg and MultiStackReg (<a href="#bib10">Thévenaz et al., 1998</a>). Download and copy into your Fiji plugins folder and restart Fiji.<br> TurboReg can be downloaded at: <a href="http://bigwww.epfl.ch/thevenaz/turboreg/">http://bigwww.epfl.ch/thevenaz/turboreg/</a><br> MultiStackReg can be downloaded at: <a href="http://bradbusse.net/sciencedownloads.html">http://bradbusse.net/sciencedownloads.html</a>
</li>
<li>Install Elastix 5.0.0 (<a href="#bib7">Klein et al., 2010</a>).<br> Elastix can be downloaded at: <a href="https://elastix.lumc.nl/download.php">https://elastix.lumc.nl/download.php</a>
</li>
<li>Install Ilastik 1.3.3post3 (<a href="#bib2">Berg et al., 2019</a>). Only required for machine learning detection of cells and projections (step 20 c).<br> Ilastik can be downloaded at: <a href="https://www.ilastik.org/download.html">https://www.ilastik.org/download.html</a>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Paraformaldehyde 16% aqueous solution</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#15700</td>
</tr>
<tr>
<td>PBS 10x</td>
<td>Corning</td>
<td>Cat#46-013-CM</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Fisher Scientific</td>
<td>Cat#AC177140050</td>
</tr>
<tr>
<td>Tissue-Tek OCT Compound</td>
<td>Sakura Finetek</td>
<td>Cat#4583</td>
</tr>
<tr>
<td>Fluoro-Gel mounting medium</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#17985-11</td>
</tr>
<tr>
<td>Neurotrace 500/525 (1:200)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#N21480</td>
</tr>
<tr>
<td>Neurotrace 640/660 (1:200)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#N21483</td>
</tr>
<tr>
<td>DAPI (1:1000)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#D1306</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>C57BL/6J mice, adult (25–90 weeks old), males and females</td>
<td>The Jackson Laboratory</td>
<td>Stock#000664</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>3D SC reference atlas p56</td>
<td><a href="#bib5">Fiederling et al. (2021)</a></td>
<td><a href="https://data.mendeley.com/datasets/4rrggzv5d5/1">https://data.mendeley.com/datasets/4rrggzv5d5/1</a></td>
</tr>
<tr>
<td>SpinalJ example dataset:processed files</td>
<td>This protocol</td>
<td><a href="https://data.mendeley.com/datasets/mkc37w4rrv/1">https://data.mendeley.com/datasets/mkc37w4rrv/1</a></td>
</tr>
<tr>
<td>SpinalJ example dataset:raw files (1/6)</td>
<td>This protocol</td>
<td><a href="https://data.mendeley.com/datasets/dsn9kggdpx/1">https://data.mendeley.com/datasets/dsn9kggdpx/1</a></td>
</tr>
<tr>
<td>SpinalJ example dataset:raw files (2/6)</td>
<td>This protocol</td>
<td><a href="https://data.mendeley.com/datasets/mrbjz83xf6/1">https://data.mendeley.com/datasets/mrbjz83xf6/1</a></td>
</tr>
<tr>
<td>SpinalJ example dataset:raw files (3/6)</td>
<td>This protocol</td>
<td><a href="https://data.mendeley.com/datasets/dfyvhrh4cd/1">https://data.mendeley.com/datasets/dfyvhrh4cd/1</a></td>
</tr>
<tr>
<td>SpinalJ example dataset:raw files (4/6)</td>
<td>This protocol</td>
<td><a href="https://data.mendeley.com/datasets/43ky5by55m/1">https://data.mendeley.com/datasets/43ky5by55m/1</a></td>
</tr>
<tr>
<td>SpinalJ example dataset:raw files (5/6)</td>
<td>This protocol</td>
<td><a href="https://data.mendeley.com/datasets/xdj4kxkvnp/1">https://data.mendeley.com/datasets/xdj4kxkvnp/1</a></td>
</tr>
<tr>
<td>SpinalJ example dataset:raw files (6/6)</td>
<td>This protocol</td>
<td><a href="https://data.mendeley.com/datasets/4hx2cj4j67/1">https://data.mendeley.com/datasets/4hx2cj4j67/1</a></td>
</tr>
<tr>
<td>SpinalJ video tutorial</td>
<td>This protocol</td>
<td><a href="https://data.mendeley.com/datasets/63m34hg3pt/1">https://data.mendeley.com/datasets/63m34hg3pt/1</a></td>
</tr>
<tr>
<td>SpineRack .stl print file</td>
<td><a href="#bib5">Fiederling et al. (2021)</a></td>
<td>
<a href="https://doi.org/10.5281/zenodo.5177185">https://doi.org/10.5281/zenodo.5177185</a>Also available through: <a href="https://github.com/felixfiederling/SpinalJ/blob/main/SpineRack.stl">https://github.com/felixfiederling/SpinalJ/blob/main/SpineRack.stl</a>
</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Cura v4.8</td>
<td>Ultimaker</td>
<td><a href="https://ultimaker.com/software/ultimaker-cura">https://ultimaker.com/software/ultimaker-cura</a></td>
</tr>
<tr>
<td>Fiji (ImageJ)</td>
<td><a href="#bib8">Schindelin et al. (2012)</a></td>
<td><a href="https://fiji.sc/">https://fiji.sc/</a></td>
</tr>
<tr>
<td>SpinalJ</td>
<td><a href="#bib5">Fiederling et al. (2021)</a></td>
<td>
<a href="https://doi.org/10.5281/zenodo.5177185">https://doi.org/10.5281/zenodo.5177185</a>Also available through: <a href="https://github.com/felixfiederling/SpinalJ">https://github.com/felixfiederling/SpinalJ</a>
</td>
</tr>
<tr>
<td>Attenuation Correction</td>
<td><a href="#bib3">Biot et al. (2008)</a></td>
<td><a href="https://imagejdocu.tudor.lu/plugin/stacks/attenuation_correction/start#attenuation_correction">https://imagejdocu.tudor.lu/plugin/stacks/attenuation_correction/start#attenuation_correction</a></td>
</tr>
<tr>
<td>TurboReg</td>
<td><a href="#bib10">Thévenaz et al. (1998)</a></td>
<td><a href="http://bigwww.epfl.ch/thevenaz/turboreg/">http://bigwww.epfl.ch/thevenaz/turboreg/</a></td>
</tr>
<tr>
<td>MultiStackReg</td>
<td><a href="#bib10">Thévenaz et al. (1998)</a></td>
<td><a href="http://bradbusse.net/sciencedownloads.html">http://bradbusse.net/sciencedownloads.html</a></td>
</tr>
<tr>
<td>3D Viewer</td>
<td><a href="#bib9">Schmid et al. (2010)</a></td>
<td><a href="https://imagej.nih.gov/ij/plugins/3d-viewer/">https://imagej.nih.gov/ij/plugins/3d-viewer/</a></td>
</tr>
<tr>
<td>Elastix</td>
<td><a href="#bib7">Klein et al. (2010)</a></td>
<td><a href="https://elastix.lumc.nl/download.php">https://elastix.lumc.nl/download.php</a></td>
</tr>
<tr>
<td>Ilastik</td>
<td><a href="#bib2">Berg et al. (2019)</a></td>
<td><a href="https://www.ilastik.org/">https://www.ilastik.org/</a></td>
</tr>
<tr>
<td>NIS-Elements JOBS</td>
<td>Nikon Instruments</td>
<td><a href="https://www.microscope.healthcare.nikon.com/en_EU/products/software/nis-elements/nis-elements-jobs">https://www.microscope.healthcare.nikon.com/en_EU/products/software/nis-elements/nis-elements-jobs</a></td>
</tr>
<tr>
<td>JOBS settings for imaging</td>
<td><a href="#bib5">Fiederling et al. (2021)</a></td>
<td><a href="https://doi.org/10.5281/zenodo.5177185">https://doi.org/10.5281/zenodo.5177185</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Ultimaker3 3D printer (or equivalent)</td>
<td>Ultimaker</td>
<td><a href="https://ultimaker.com/de/3d-printers/ultimaker-3">https://ultimaker.com/de/3d-printers/ultimaker-3</a></td>
</tr>
<tr>
<td>PVA filament</td>
<td>Ultimaker</td>
<td>Cat#9731</td>
</tr>
<tr>
<td>Truncated embedding mold Peel-A-Way(R) T12</td>
<td>Polysciences</td>
<td>Cat#189861</td>
</tr>
<tr>
<td>Cryostat</td>
<td>Leica Biosystems</td>
<td>CM3050S or equivalent</td>
</tr>
<tr>
<td>Disposable microtome blades</td>
<td>Sakura Finetec</td>
<td>Cat#4685</td>
</tr>
<tr>
<td>Microscope slides, Fisherbrand Superfrost Plus</td>
<td>Fisher Scientific</td>
<td>Cat#12-550-15</td>
</tr>
<tr>
<td>Wheaton dish</td>
<td>DWK Life Sciences</td>
<td>Cat#900200</td>
</tr>
<tr>
<td>Slide Scanning Microscope e.g., Nikon AZ100 Multizoom</td>
<td>Nikon Instruments</td>
<td>Model AZ100, discontinued</td>
</tr>
<tr>
<td>Automated slide feeder</td>
<td>Prior Scientific Inc.</td>
<td>Model PL-200</td>
</tr>
<tr>
<td>Andor Zyla camera</td>
<td>Oxford Instruments</td>
<td>Model Zyla 4.2 sCMOS</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Tissue preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1 h hands on + 2 days unattended</div>
<p>This protocol has been established on perfusion-fixed SC tissue of adult C57BL/6J mice.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to acquire approval for anesthesia and perfusion of mice and to comply with institutional guidelines on the care and use of animals in research.</div>
<ol>
<li>Perfuse the deeply anesthetized animal with PBS followed by 4% PFA in PBS and expose the SC by standard laminectomy or by hydraulic extrusion (<a href="#bib6">Kennedy et al., 2013</a>). Postfix spinal tissue in 4% PFA in PBS on an orbital shaker for 2–12 h at 4°C.</li>
<li>Wash SC 3x (10 min each) in PBS on ice, remove dura and meninges, then transfer into a 15 mL tube filled with PBS containing 30% sucrose and incubate at 4°C for 1–2 days or until the tissue has sunk, for cryo-protection.</li>
</ol>
<h3 id="sec2.2">Spinal cord embedding using SpineRacks</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–15 min hands on + 20–30 min unattended</div>
<p>After fixation and isolation of the SC, the tissue is divided into nine short pieces that are embedded with precise upright orientation and in rostrocaudal (r-c) order within the SpineRack support scaffold. Placing tissue pieces in the same block, in order, ensures parallel cryosectioning that reduces whole SC sectioning time and preserves section identity.</p>
<ol start="3">
<li>Prepare a ∼5 cm × 5 cm piece of Parafilm, and pour ∼5 mL OCT onto it. This pool of OCT will be used to wash and orient all tissue pieces prior to embedding.</li>
<li>Transfer the SC together with the sucrose solution to a square petri dish (<a href="#fig1">Figure 1</a>A). Cut the SC into three equal parts (<a href="#fig1">Figures 1</a>B and 1C) using sharp spring scissors while viewing with a dissecting microscope.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1177-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Embedding of SC tissue using SpineRacks</div>
<p>(A and B) Isolated SC in sucrose solution.</p>
<p>(C) SC cut into 3 equal tissue pieces. Numbers 1–9 indicate r-c orientation and placement of next cuts. Ruler scale in mm.</p>
<p>(D) SC cut into 9 equal pieces arranged according to position in SpineRack in OCT.</p>
<p>(E) Submerging SpineRack in embedding mold filled with OCT.</p>
<p>(F–I) Placement of tissue pieces 1–9 into SpineRack.</p>
<p>(J) Vertical orientation of tissue pieces in the mold.</p>
<p>(K and L) Incubation of SpineRack in OCT expands and softens the material for sectioning (K: before, L: after 20 min incubation).</p>
<p>(M) Freezing the block.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to cut perpendicular to the long axis of the SC (angled cuts produce incomplete cross sections that cannot be used for registration). Keep track of the identity and r-c orientation of the tissue pieces.</div>
<ol start="5">
<li>Cut each piece from step 4 into three equal sized pieces of 3–4 mm (<a href="#fig1">Figures 1</a>C and 1D). Starting with the most rostral segment, after each cut gently transfer the resulting tissue pieces 1–3 into the OCT puddle with their rostral end facing the top side of the Parafilm square, using forceps. Repeat with the other pieces and collect resulting tissue pieces 4–6 and 7–9 in OCT.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keeping track of the identity and orientation of each tissue piece is essential. For maximal efficiency of cryo-sectioning and subsequent imaging, all tissue pieces should be of equal length.</div>
<ol start="6">
<li>Fill a truncated embedding mold with OCT and mark one side for orientation (here marked “top”). Submerge a SpineRack into OCT and push all the way to the bottom of the mold using forceps (<a href="#fig1">Figure 1</a>E). Release any trapped air bubbles.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> After immersing the SpineRack into OCT, the PVA material will start to dissolve and will soften and expand in the process, and this may prevent positioning or placement of tissue pieces. Thus, tissue must be embedded immediately, within the first 15 min after immersing the SpineRack in OCT.</div>
<ol start="7">
<li>Transfer one tissue piece at a time into the SpineRack, taking care to load the pieces into wells from top-left to bottom-right in r-c order (<a href="#fig1">Figures 1</a>F–1I). Using forceps, carefully push each tissue piece within a well to the bottom of the mold with its rostral end facing down and the dorsal side facing the marked (top) side of the mold (<a href="#fig1">Figure 1</a>J). Wide cervical segments 1–3 are rotated by ∼45° to fit the well.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> If interested only in a smaller sub-region of SC (e.g., L3-L6), SpineRacks can also be used to embed and section multiple samples in parallel.</div>
<div class="optional">
<span class="optional-title">Optional:</span> SpineRacks (with same or adjusted geometry) can also be used to embed other tissues or organisms that have a high length-to-width ratio (muscles, fish embryos) or that require parallel sectioning in a single orientation (eg. mouse eyes).</div>
<ol start="8">
<li>After embedding all tissue pieces, leave mold containing SpineRack and tissue at room temperature for 20–30 min. This allows the PVA material to soften for ease of sectioning (<a href="#fig1">Figures 1</a>K and 1L).</li>
<li>Freeze the block on dry ice in a slush of ethanol and crushed dry ice until the block is solid (<a href="#fig1">Figure 1</a>M).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Use immediately OR, if the histology protocol in use for the tissue permits, store blocks at −80°C until use. For NeuroTrace or DAPI labelling, blocks can be stored for at least 6 months (longer has not been tested).</div>
<h3 id="sec2.3">Sectioning and staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h hands on + incubation time for immunohistochemistry (optional)</div>
<p>Using SpineRacks, &gt;1000 25μm sections comprising spinal segments C1-L6 can be produced within 1h and collected on as few as 16 slides. Cutting thicker sections can further reduce sectioning and processing time at the expense of r-c resolution.</p>
<ol start="10">
<li>Place the block into the cryostat chamber for 20–30 min to allow temperatures to equilibrate. We section at approximately −21°C using a Leica Cryostat CM3050S and Sakura Finetec microtome blade. For sectioning, remove the mold, keeping track of orientation, and mount the block on the block holder with the marked side up and set sectioning thickness to 25μm (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1177-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Sectioning, collecting and staining of block sections</div>
<p>(A) Tissue block mounted in cryostat.</p>
<p>(B) Schematic arrangement of 8 block sections on a slide in a left-and-up order. See also <a href="#fig4">Figure 4</a>A.</p>
<p>(C) Example csv. list of lost sections.</p>
<p>(D) Slides washing in PBS.</p>
<p>(E) Slide after staining and mounting.</p>
</figcaption>
</figure>
</li>
<li>Section the block and collect all sections continuously on numbered slides. To permit automated imaging, collect eight block sections in two rows of four with equal spacing on each slide (<a href="#fig2">Figure 2</a>B). Keep note of the order and orientation of sections on each slide, as this information is crucial to maintain r-c order of images. Make notes of lost sections (slide # and position on slide; lost sections can be replaced with neighboring sections during image processing; <a href="#fig2">Figure 2</a>C).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Unless user-specific antibody or histology procedure of choice does not support it, slides can now be stored at −80°C. For subsequent NeuroTrace or DAPI labelling slides can be stored for at least 3 months (longer has not been tested).</div>
<ol start="12">
<li>Wash slides in a glass Wheaton dish filled with PBS for 5–10 min (gentle shaking at room temperature) to wash away SpineRack material and OCT (<a href="#fig2">Figure 2</a>D). Proceed with immunohistochemistry, or other staining methods, following standard protocols (<a href="#bib4">Evilsizor et al., 2015</a>) and mount slides under coverslips (<a href="#fig2">Figure 2</a>E).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For subsequent image registration and atlas mapping, co-staining with NeuroTrace (preferred) or DAPI is required.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Mounted slides can be stored at 4°C in the dark. Slides can be kept for at least 3 months after NeuroTrace or DAPI labelling. User specific antibodies or stains will dictate other storage times.</div>
<h3 id="sec2.4">Imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–15 min hands on + 3–5 h unattended</div>
<ol start="13">
<li>Image slides on a slide scanning microscope.
<div class="note">
<span class="note-title">Note:</span> Processing in SpinalJ requires either images of individual tissue sections or images of block sections (array of 9 tissue sections), and alphanumeric file naming that reflects section order. In this study, we used a motorized Nikon AZ100 Multizoom microscope equipped with an automated slide feeder and Andor Zyla sCMOS camera to acquire images of block sections. Any other automated or manual systems can be used to collect images, but may require additional manual intervention, depending on the capabilities of the individual system to identify, scan and save block sections on the slide automatically. For example, images taken on slide scanners that scan the whole slide have to be cropped to individual block section images. Block section images should be saved as .nd2 or .tif files with file naming “SlideXX_SectionYY” (e.g., Slide12_Section04.nd2). For non-motorized systems, images of individual sections have to be acquired and saved manually. Single section images should be saved as .tif files with file naming “SegmentXX_SlideYY_ImageZZ” (e.g., Segment06_Slide12_Image01.tif) if pre-processing in SpinalJ is desired.</div>
The workflow for automatic scanning of block sections using the Nikon system, running NIS-Elements JOBS software is described below (<a href="#fig3">Figure 3</a>; JOBS file available for download):
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1177-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Scan settings for imaging block sections using NIS-Elements JOBS</div>
<p>(A) Load a pre-scan image of a slide and open pre-scan settings in JOBS file.</p>
<p>(B) Check ‘rolling ball correction’.</p>
<p>(C) Set lower and upper threshold to isolate pixel intensities within SC sections.</p>
<p>(D) Apply size cutoff to filter out objects smaller and larger than SC sections.</p>
<p>(E) Apply dilation factor to create a mask that connects all sections of a block section, but does not touch sections from neighboring block sections. Save settings and start scan.</p>
</figcaption>
</figure>
Setting up Nikon slide scanner for scanning block sections with NIS-Elements JOBS:
<ol type="a">
<li>Load all slides in order into a slide magazine (1–16, bottom to top) and initialize the slide scanner using 4x objective, 2.1x zoom.</li>
<li>Open JOBS scan file for SC sections and pre-scan 1–2 slides using the autofocusing channel (DAPI or NT). To save the pre-scan, edit settings before scanning as follows (<a href="#fig3">Figure 3</a>A):
<ol type="i">
<li>Under PreScanSlide set Save Image → always.</li>
<li>Disable Region Loop.</li>
</ol>
</li>
<li>After pre-scanning, open the pre-scan image.</li>
<li>Open JOBS scan file and open scan parameters (<a href="#fig3">Figure 3</a>B).
<ol type="i">
<li>Set lower and upper intensity threshold to filter out background and unspecific signals (<a href="#fig3">Figure 3</a>C).</li>
<li>Set size cutoff to exclude objects that are not tissue sections (<a href="#fig3">Figure 3</a>D).</li>
<li>Set dilation factor to merge all sections (1-9) of a block section into a single object (<a href="#fig3">Figure 3</a>E). Avoid merging multiple block sections.</li>
</ol>
</li>
<li>Save settings, disable save pre-scan, enable Region Loop and start the scanning process.</li>
</ol>
</li>
</ol>
<h3 id="sec2.5">Image pre-processing in SpinalJ</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–1 h hands on + 1–2 h unattended</div>
<p>SpinalJ provides a series of tools to pre-process SC section images for registration and analysis. These include segmentation of block section images, compensation for lost sections, r-c ordering of sections, cleaning out damaged sections, as well as re-orientation and horizontal alignment of sections. If preprocessing has already been performed without using SpinalJ, skip to step 19.</p>
<p>Achieving r-c order of images critically depends on how sections were collected on the slide. SpinalJ supports a series of collection orders, as defined in <a href="#fig4">Figure 4</a>A.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1177-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Image pre-processing</div>
<p>(A) Schematic of commonly used arrangements of sections on the slide.</p>
<p>(B) Pre-processing settings (for details see descriptions in step 14).</p>
<p>(C) For manual segmentation of block section images, position the segmentation mask (yellow) to include all tissue sections 1-9.</p>
<p>(D) Batch rotation of images from tissue pieces embedded upside down. Zoom in (blue box) reveals that tissue piece 4 (yellow box) needs 180° rotation.</p>
<p>(E) Cleaning out of compromised and empty section images using SPACE, CTRL and ALT keys.</p>
<p>(F–M) Example NT images of sections to keep (green), replace (orange), or delete (red).</p>
<p>(N) Montage of all horizontally aligned sections from a sample after pre-processing.</p>
</figcaption>
</figure>
<ol start="14">
<li>Setting Pre-Processing Settings: To prepare image pre-processing, create a parameter file that contains information on how the images of a sample are processed (<a href="#fig4">Figure 4</a>B). Run <i>Plugins &gt; SpinalJ &gt; 1 Set Preprocessing Settings</i>. In the GUI that appears, complete the fields below.
<ol type="a">
<li>Image Data - Specify location of the folder containing the raw sections.
<div class="note">
<span class="note-title">Note:</span> Files in .nd2 format cannot exceed 2GB. If image resolution does not match 1micron/pixel, scale images using Fiji&gt;Image&gt;Scale or create custom segmentation masks (see <a href="#sec5.7">troubleshooting 4</a>).</div>
</li>
<li>Reference Channel - Select channel containing NeuroTrace or DAPI.</li>
<li>Transform all Block Section Images - Select transformation option to flip images as necessary so that sections are oriented with their dorsal side up and array order as follows: Top row, left to right: 1, 2, 3. Middle row: 4, 5, 6. Bottom row: 7, 8, 9 (see <a href="#fig4">Figure 4</a>C).</li>
<li>Determine r-c File Order - Select method used to bring image files into r-c order (using stage coordinates or alphanumerical file names).</li>
<li>Order of Sections on Slide - Indicate order of sections on slide, see <a href="#fig4">Figure 4</a>A. Required only if choosing stage coordinate sorting.</li>
<li>Segmentation Masks - Specify the location of the folder containing the segmentation masks .roi files. If using images with resolution other than 1micron\pixel, see <a href="#sec5.7">troubleshooting 4</a>.</li>
<li>Automatic Segmentation:
<ol type="i">
<li>Min. Object Size – Specify lower area cutoff for auto-detection of tissue sections. Default: 1e6.</li>
<li>Max. Object Size – Specify upper area cutoff for auto-detection of tissue sections. Default: 5e6.</li>
<li>Min. Circularity – Specify lower circularity cutoff for auto-detection of tissue sections. Default: 0.05.</li>
<li>Max. Circularity – Specify upper circularity cutoff for auto-detection of tissue sections. Default: 1.</li>
</ol>
</li>
<li>Replace Lost Sections - Check ‘yes’ to compensate for lost sections during cryo-sectioning by duplicating neighboring sections. Requires a .csv list of lost sections (see below).</li>
<li>Lost Sections - Specify the location of the folder containing a .csv list of lost sections. Required only if choosing to replace lost sections. To create the list, enter pairs of values that specify the slide (column A) and the position (column B) on which the section was lost e.g., if the seventh section on the second slide was lost, enter 2, 7 (see <a href="#fig2">Figure 2</a>C). If several consecutive sections are lost, repeat the list entry. The first row contains the titles ‘Slide’ and ‘Section’, respectively. Save list as ‘_Lost_Sections.csv’.</li>
<li>Horizontal Alignment Sampling Interval - Specify n. Total number of sections / n sections are randomly selected to determine the horizontal orientation for each tissue piece. Default: 10.</li>
<li>Minimal Test Angle - Specify negative test angle extreme. Must be &gt; −90°. Default: −50°.</li>
<li>Maximal Test Angle - Specify positive test angle extreme. Must be &lt; 90°. Default: 50°.</li>
<li>Test Angle Increment - Specify interval between test angles. Default: 10.</li>
<li>Masking Channel - Select channel for section segmentation. NeuroTrace or DAPI.</li>
<li>Thresholding Option for Masking - Choose automatic thresholding option. Empirically, we found ‘Percentile dark’ to work best for NT and ‘Min error’ for DAPI.</li>
</ol>
</li>
</ol>
<ol start="15">
<li>Segmentation of block section images:<br> Run <i>Plugins &gt; SpinalJ &gt; 2 Block Section Segmentation</i> to split block section images into individual tissue section images. Confirm path to the folder containing raw image data. Input files need to contain slide information as first part of the filename, e.g., “Slide4_Section3.nd2”. Images in.nd2 format will be automatically sorted rostrocaudally using stage coordinates when selecting the corresponding option in step 14. Images in other file formats have to be renamed manually to reflect r-c order. After ordering, block section images are automatically segmented. Images that contain less than 9 tissue sections cannot be automatically segmented and are saved to folder <i>\_Temp\_0_Scaled\</i> as a down-scaled version and displayed at the end of the script for manual segmentation. For these files, position the segmentation mask displayed on top of each image so that each tissue section falls within the appropriate outline of the section array mask, reflecting segment identity (<a href="#fig4">Figure 4</a>C). Click <i>OK</i> to proceed to the next section. If identification of segment identity is impossible because of missing sections, hold <i>SHIFT</i> + click <i>OK</i> to skip segmentation of the current image. Output files are saved to folder <i>\_Temp\_I_Split\</i> with names indicating tissue piece (segment) identity, e.g., “Segment05_Slide12_Image02.tif”.</li>
<li>Segment Rotation:<br> Run <i>Plugins &gt; SpinalJ &gt; 3 Batch Rotation</i> to re-orient section images from tissue pieces embedded upside down or rotated by 90° (e.g., piece 4 in <a href="#fig4">Figure 4</a>C). Confirm path to the folder containing raw image data. A preview montage of all sections will show the reference channel of all section images ordered in rows representing tissue pieces 1–9. Identify rows in which all sections are oriented at an angle ≥ 90° or ≤ −90° and select those rows for rotation (<a href="#fig4">Figure 4</a>D). Set rotation value and click <i>OK</i>. Confirm proper orientation in the updated preview montage saved at <i>\_Temp\_II_Preview_Split\</i>.</li>
<li>Section Clean up:<br> Run <i>Plugins &gt; SpinalJ &gt; 4 Clean Stack</i> to clean out compromised and empty sections and to obtain a continuous stack of intact tissue sections. Confirm path to the folder containing raw image data. A reference channel preview of the image stack of individual tissue sections will be displayed. Navigate through the stack and decide for each section whether to ‘keep’, ‘replace’, or ‘delete’ the current image by pressing the <i>SPACE</i>, <i>CTRL</i> or <i>ALT</i> key, respectively (<a href="#fig4">Figures 4</a>E–4M). To the left and right of the stack window, additional windows show the previous (left) and next (right) image in the stack. Use ‘replace’ only after at least one previous image has been selected for ‘keep’. Processed files are saved to folder <i>\_Temp\_III_Clean\</i> and a preview montage of the reference channel is saved to <i>\_Temp\_IV_Preview_Clean\</i> for validation.</li>
<li>Horizontal alignment of section images: Run <i>Plugins &gt; SpinalJ &gt; 5_Horizontal Alignment</i> to align tissue sections horizontally in order to improve image registration performance<i>.</i> Confirm path to the folder containing raw image data. After automatic alignment is completed, a preview montage of the cleaned and aligned sections is created for validation (<a href="#fig4">Figure 4</a>N). In addition, pre-processed images are scaled to match the resolution of the atlas and saved to a new folder <i>/_Output/ 1_Reformatted_Sections/</i> with subfolders for each image channel.
<div class="note">
<span class="note-title">Note:</span> Data generated during image pre-processing will be saved to subfolder <i>\_Temp\</i> and can be deleted after reformatted images have been created (step 18).</div>
</li>
</ol>
<h3 id="sec2.6">Image registration, atlas mapping, and analysis in SpinalJ</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–30 min hands on + 1–8 h unattended</div>
<p>All subsequent steps assume a single folder containing intact, consecutive single section images of a sample with an alphanumerical name format that reflects r-c order.</p>
<ol start="19">
<li>Setting Registration and Analysis Parameters for SpinalJ:<br> SpinalJ is designed to batch process multiple samples, even when the experimental conditions or analysis settings are different. This step creates a parameter file containing analysis information for individual samples. Run <i>Plugins &gt; SpinalJ &gt; 6 Set Analysis Settings</i>. In the GUI that appears, complete the fields below:
<ol type="a">
<li>Select folder - Specify the location of the folder containing the ordered tissue sections (/_Output/).</li>
<li>Final resolution of image output (μm/px) - Specify the resolution used by the rest of the analysis, default: 2μm/pixel.</li>
<li>Counterstain channel (DAPI or NeuroTrace) – Select channel containing NeuroTrace, or DAPI.</li>
<li>Background intensity of counterstain channel - this value should be slightly higher than the intensity of the background surrounding the tissue. <i>To determine the background intensity: open a section image in Fiji and hover the cursor over the tissue and background to find a suitable intensity level for this setting.</i> If set too low, the sections will not be adequately isolated, and if set too high, the sections will be patchy and over-cropped.</li>
<li>Background removal prior to segmentation (rolling ball radius in px, 0 if none) - Specify the radius of a rolling ball filter used to clean up images before cell detection and projection analysis. A value of 7 works well for the recommend image pixel size of 2μm/px.</li>
<li>Section cut thickness - Indicate section cut thickness. If using a sub-series, multiply accordingly. E.g., every second section, cut at 20μm = 40μm.</li>
<li>Spinal cord range (start segment, end segment) - Indicate the range of segments included in the dataset (e.g., C2-L5). <i>To determine range, open preview stack of sections and determine the segment identity of the first and last section based on morphological landmarks and by comparing to neighboring sections.</i>
</li>
<li>Reference section - Specify the starting section used for initial section-to-section registration. This should be roughly in the middle of the dataset. e.g., section 500 in a 1000 section dataset.</li>
<li>Generate full resolution registered image of reference channel – Check only if attempting to perform cell or projection analysis on the reference channel.</li>
<li>Perform a second pass section registration - This feature can be used to register badly damaged sections with the help of fiducial markers but is not required for normal use.</li>
<li>Method for cell detection:
<ol type="i">
<li>Manual cell detection takes a list of cell locations created by manual cell counting – see step 20a.</li>
<li>Find Maxima will locate the center of cells and is fast and effective for cells with clearly labeled soma – see step 20b.</li>
<li>Machine learning segmentation with Ilastik can yield better results, especially when cells are difficult to discern from brightly labeled axons and dendrites. This will require training of an Ilastik Project for each channel – see step 20c.</li>
</ol>
</li>
<li>Channel selection - Select up to three channel/s for cell analysis. Set the first selector to the channel to analyze. If no remaining channels will be analyzed, set the remaining selectors to 0.</li>
<li>Minimum intensity threshold: Specify the minimum threshold used for detecting cells when using Find Maxima – see step 20b.</li>
<li>Minimum cell area: Specify the area used for cell detection (when using Ilastik for segmentation).</li>
<li>Method for projection detection:
<ol type="i">
<li>Binary Threshold: fluorescence above this threshold will be included in the density analysis. This is a fast and effective method, useful when it’s not necessary to remove cell bodies from the analysis.</li>
<li>Machine learning segmentation with Ilastik can yield better results than binary thresholding, especially when it is necessary to differentiate between cells and projections. This will require training of an Ilastik Project for each channel – see 20c.</li>
</ol>
</li>
<li>Channel selection: Select up to three channel/s for projection analysis. Set the first selector to the channel to analyze. If no remaining channels will be analyzed, set the remaining selectors to 0.</li>
<li>Minimum intensity threshold: Choose the minimum threshold used for detecting projections when using Binary Threshold – see 20b.</li>
<li>Ilastik location: Specify the directory where Ilastik is installed.</li>
<li>Elastix location: Specify the directory where Elastix is installed.</li>
</ol>
</li>
</ol>
<ol start="20">
<li>After setting up the Registration and Analysis Parameter file, follow the steps below according to the selected option for cell detection (skip if interested only in intensity mapping, see step 21 i):
<ol type="a">
<li>Manual cell detection (19 k.i.):
<ol type="i">
<li>Run through steps 1–19 of this guide. In step 19 k, select ‘Manual Cell Count’.</li>
<li>Run step 21 a-d. The experiment folder will now contain a sub-folder <i>\3_Registered_Sections\.</i> Inside this folder, you will find subfolders for each channel e.g., Folder <i>\2\</i> corresponds to channel 2.</li>
<li>Drag the entire channel folder on Fiji. An ‘Open Folder’ window will appear. Click ‘Yes’.</li>
<li>Under <i>Edit &gt; Options &gt; Input/Output…</i> turn off rows and columns headers.</li>
<li>Under <i>Analyze &gt; Set Measurements</i> deselect everything except centroid. Set decimal places to 0.</li>
<li>Under <i>Image &gt; Properties…</i> set unit of length to pixel. Pixel width, pixel height, voxel depth = 1.</li>
<li>Select the Multipoint Tool.</li>
<li>Go through slice by slice and click on every cell. If you miss a cell, it is ok to go back through the stack and add new cells. <i>CTRL</i>+ click removes markers.</li>
<li>After counting, you can save this image if you wish to keep a record of how the cells were counted.</li>
<li>Press <i>CTRL+M</i> to get the centroid and slice position of each cell.</li>
<li>Save this list as ‘Cell_Points_Ch#.csv’ into the subfolder <i>\4_Processed_Sections\Detected_Cells\</i> (if you are counting cells on channel 2 this file would be called Cell_Points_Ch2.csv).</li>
<li>Run step 21. Uncheck c and d - already performed. Check e to perform cell detection and analysis, and any other desired analysis steps.</li>
</ol>
</li>
<li>Determining thresholds for Find Maxima cell detection and binary thresholds for projection analysis (19 k.ii):
<ol type="i">
<li>Run through steps 1–19 of this guide. Run steps 21 a-e, as described in step 21.</li>
<li>The experiment folder will now contain a sub-folder <i>\3_Registered_Sections\.</i> Inside this folder, you will find subfolders for each channel e.g., Folder <i>\2\</i> corresponds to channel 2.</li>
<li>Apply a rolling ball filter with the same diameter used within SpinalJ, then confirm the typical intensities of the cells you wish to detect using Find Maxima, and use this value in the analysis settings. For binary thresholding of axons/dendrites use the threshold tool (<i>CTRL+SHIFT+T</i>) to determine the minimum threshold value to be used.</li>
</ol>
</li>
<li>Training Ilastik for machine learning based mesoscale mapping / cell detection (19 k.iii): We recommend using Ilastik for best results, especially when analyzing densely labeled neurons, although other cell detection methods are provided. Be aware that oversaturation of axons/dendrites may result in false positive cell detections when using automated analysis. If analyzing both cells and neuronal projections, it is important to ensure your images are free from oversaturated pixels. For best results in quantifying cell numbers, use a probe with nuclear localization, such as GFP fused to histone 2B.
<ol type="i">
<li>Prepare representative images for training the Ilastik project.<br> Training Ilastik’s pixel classifier requires several representative images. A minimum of 5 (better 10–20) sections from different r-c positions of the SC is recommended. If training a project on multiple animals or if the label is highly variable between sections, additional training images may be required. Training images can come from the <i>\1_Reformated_Sections\</i> or <i>\3_Registered_Sections\</i> folders. During the SpinalJ pipeline, background subtraction is applied to all sections prior to running Ilastik. After providing a list of sections, images suitable for training will be created and saved to folder <i>\Ilastik_Training_Images\</i>.
<div class="critical">
<span class="critical-title">Critical:</span> When adding images to the Ilastik project for training, ensure they also have this same background subtraction applied. You can do this by running <i>Plugins &gt; SpinalJ &gt; Other Tools &gt; Create Ilastik training images</i> in Fiji.</div>
</li>
<li>Open the provided Ilastik project file.<br> An example project is provided as a template to build the projects for each of your channels containing all settings required to export the necessary probability images for use in SpinalJ.</li>
<li>Select the existing images in the project file. Right-click and choose ‘Reset’ - this clears out the existing training data, but keeps project file settings, allowing you to add new images for training.</li>
<li>Copy in the images selected for training.<br> We recommend saving these images into the project file for auditing and to avoid problems if further training is required. To do so, Right-click and select ‘Dataset Properties…’, in the window that appears, change ‘Storage’ to ‘Copy into project file’. Then save the project.</li>
<li>Train the project file.<br> The Ilastik Project comes preconfigured with three labels: Cells (for soma/cell bodies), Projections (for axons/dendrites), and Background.<br> If you are not detecting cells, you do not have to train the cells label (likewise for projections).<br> For best results when training on 16-bit images, it can be helpful to adjust how the image is displayed in Ilastik. You can do this by right-clicking on ‘Raw Input’ in Group Visibility list (lower left-hand corner), then selecting ‘Adjust thresholds’. Uncheck the ‘automatic range’ option and reduce the ‘maximum value’ so that you can clearly visualize low intensity structures.<br> For further advice on training an Ilastik project:<br> <a href="https://www.ilastik.org/documentation/pixelclassification/pixelclassification">https://www.ilastik.org/documentation/pixelclassification/pixelclassification</a>
</li>
<li>Save the project file as: ‘Ilastik_Project_Channel_#’ (e.g.,‘Ilastik_Project_Channel_1’ for channel 1) in the <i>\Ilastik_Projects\</i> folder.</li>
<li>Repeat steps 20 c.i-vi to create an Ilastik project for each channel to be analyzed. These projects can be copied to other animals/datasets to be analyzed.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="21">
<li>Atlas registration analysis This step will automatically process each folder and generate three subfolders that contain 1) the registered sections at the resolution set for analysis (<i>\3_Registered_Sections\</i>), 2) the processed images used for analysis, such as the output from Ilastik, and validation images for confirming the accuracy of cell segmentation (<i>\4_Processed_Sections\</i>; <a href="#fig5">Figure 5</a>A) and 3) the final analysis tables and images (<i>\5_Analysis_Output\</i>; <a href="#fig5">Figures 5</a>B and 5C). Run <i>Plugins &gt; SpinalJ &gt; 7 Registration and Atlas Analysis</i>. In the GUI that appears, complete the fields below:
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1177-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Quality control and output</div>
<p>(A) Overlay of automatically detected cells and corresponding image channel for visual validation of object detection. Shown is data of the example dataset (ChatCre;Ai14tdTomato); green: channel 1 (555nm), red: objects detected using Ilastik.</p>
<p>(B) Overlay of the atlas template (green) and the experimental example dataset (red, Neurotrace channel) after atlas alignment.</p>
<p>(C) SpinalJ output graphs like this heatmap montage of cell densities within channel 1 of the example dataset provide a quick visual summary of the data. Scale: 500μm.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> For efficient processing, we recommend confirming good section registration prior to performing further analyses steps. The registered sections are stored in folder <i>/_Output/3_Registered_Sections/</i> after performing step 21 a-c and can be visualized in 3D using the ImageJ 3D Viewer plugin (<a href="#bib9">Schmid et al., 2010</a>), see <a href="#mmc1">Methods video S1</a>.</div>
<ol type="a">
<li>Select experiment/spinal cord folder(s) - Specify the location(s) of the folder(s) for analysis (/_Output/).
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1177-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Registered sections, related to step 21</div>
<p>Animation through a stack of 1302 25μm registered sections (Neurotrace channel) covering segments C1-S3 and the same data re-sliced in the horizontal and sagittal plane. Data is provided in the example dataset.</p>
</figcaption>
</figure>
</li>
<li>Select spinal cord atlas to be used - Specify the directory where the atlas files are located.</li>
<li>1. Perform section registration? - Choose to perform initial rigid body registration of all tissue sections.</li>
<li>2. Perform atlas registration? - Choose to perform affine and elastic registration for atlas mapping to experimental data.</li>
<li>3. Perform cell detection and analysis? - Choose to perform cell detection using the method selected in the previous step. If performing manual cell counting, run both registration steps first, manually count cells, then rerun this step with this and the following options enabled. The cell analysis generates two tables for each channel used for cell detection. The Detected Cells table provides information on the coordinates of each cell, the region and segment it is contained in, and the measured mean intensity in each channel. The Detected Cells Summary table provides a total cell count for each region and segment.</li>
<li>Generate cell analysis images and heatmaps? - Choose to generate images showing cell locations in the atlas colored according to annotation ID.</li>
<li>4. Perform mesoscale mapping projection analysis? - Choose to perform projection analysis. The mesoscale mapping analysis uses reverse mapping to generate output tables for each channel containing information on the density of axons/dendrites present in each atlas region, and on the relative density according to the volume of the entire labeled volume. Importantly, all density measurements are calculated using the region volume as represented in the dataset, allowing for accurate measurement even when a region is only partially represented in the experimental data.</li>
<li>Generate mesoscale mapping visualization images and heatmaps? - Choose to generate images showing the projections mapped to the atlas and density heatmaps.</li>
<li>5. Measure mean intensities of annotated regions? - Choose to perform region specific intensity measurements. Mean intensities for each channel, atlas region and segment are saved to output tables.</li>
<li>Generate intensity based atlas images? - Choose to generate an atlas image with regions colored with their measured mean intensity.</li>
<li>6. Extract specific regions at full resolution (requires &gt;128GB RAM)</li>
<li>Provide annotation IDs for extraction (e.g., 4,10) - Specify location of a list (comma separated) with annotation regions for extraction.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After the analysis is complete, all output tables and graphs (e.g., cell density heatmaps, as shown in <a href="#fig5">Figure 5</a>C) can be found in folder <i>\5_Analysis_Output\</i>. In order to re-run analysis with different settings, delete this folder and repeat step 21.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>We provide a toolbox for the analysis of neuronal number and position of cells and processes within the full r-c extent of mouse SC. We have developed SpineRacks for oriented embedding and efficient sectioning, SpinalJ for user-friendly image processing and analysis, and a 3D SC atlas that provides a standardized reference for analysis of spinal neurons and projections. The availability of a common coordinate framework and 3D anatomical annotations permits comparative mapping of spinal neurons across samples and labs.</p>
<p>We have validated the accuracy and reproducibility of SpinalJ mapping with reference to published manual studies, attesting to its usefulness for a variety of experiments (<a href="#bib5">Fiederling et al., 2021</a>).</p>
<p>Analysis output data including intensity values, cell coordinates as well as cell/projection density is provided in table (csv.) format and summarized in heatmap graphs for each spinal segment (<a href="#fig5">Figure 5</a>C).</p>
<p>An example dataset including the raw image data, parameter settings files and output images are provided and available for download (see <a href="#key-resources-table">key resources table</a>). <a href="#tbl1">Table 1</a> lists all folders and important files within this dataset. To reproduce this experiment and re-run all pre-processing and analysis steps (14–21), copy the raw .nd2 files (provided in a total of 6 subfolders, 120 GB) to a new location and start with step 14. Alternatively, to skip pre-processing and start directly with section registration (19–21), copy folder “example_data/_Output/1_Reformatted_Sections/” (21.3 GB) and start with step 19. The settings we used for processing and analysis are provided in files “_PreProcessing_Parameters.csv” and “Analysis_Settings.csv”. To repeat parts of the analysis with adjusted settings make sure to delete folder “…/5_Analysis_Output/” before re-running step 21.</p>
<table id="tbl1">
<caption>Table 1. List of files included in example dataset</caption>
<thead>
<tr>
<th>Folder</th>
<th>File</th>
<th>Description</th>
<th>Size</th>
<th>Step</th>
</tr>
</thead>
<tbody>
<tr>
<td>SpinalJ_example_data_processed/</td>
<td> </td>
<td>Example dataset</td>
<td>299 GB</td>
<td>n/a</td>
</tr>
<tr>
<td> </td>
<td>_Sample_Info.csv</td>
<td>Mouse genetics, staining and imaging info</td>
<td>1 KB</td>
<td>n/a</td>
</tr>
<tr>
<td> </td>
<td>_Lost_Sections.csv</td>
<td>Sections lost during cryosectioning</td>
<td>1 KB</td>
<td>11</td>
</tr>
<tr>
<td> </td>
<td>_PreProcessing_Parameters.csv</td>
<td>Parameters used for pre-processing</td>
<td>1 KB</td>
<td>14</td>
</tr>
<tr>
<td> </td>
<td>_Cleaning_log.csv</td>
<td>User input during section clean up</td>
<td>10 KB</td>
<td>17</td>
</tr>
<tr>
<td> </td>
<td>_II_Preview_Montage.tif</td>
<td>Down scaled single section images (reference channel montage)</td>
<td>48.9 MB</td>
<td>15</td>
</tr>
<tr>
<td> </td>
<td>_IV_Preview_Montage_Clean.tif</td>
<td>Down scaled cleaned single section images (reference channel montage)</td>
<td>43.5 MB</td>
<td>17</td>
</tr>
<tr>
<td> </td>
<td>_V_Preview_Montage_Aligned.tif</td>
<td>Down scaled, cleaned and aligned single section images (reference channel montage)</td>
<td>341 MB</td>
<td>18</td>
</tr>
<tr>
<td>SpinalJ_example_data_processed/_Output/</td>
<td> </td>
<td>Pre-processed image data, temporary files created during analysis, as well as resulting files</td>
<td>38.5 GB</td>
<td>19, 21</td>
</tr>
<tr>
<td> </td>
<td>Analysis_Settings.csv</td>
<td>Parameters used for registration and analysis</td>
<td>2 KB</td>
<td>19</td>
</tr>
<tr>
<td>…/_Output/1_Reformatted_Sections/</td>
<td> </td>
<td>Single section images in rostro-caudal order, split by channel, and reference channel montage</td>
<td>21.3 GB</td>
<td>18</td>
</tr>
<tr>
<td>…/_Output/3_Registered_Sections/</td>
<td> </td>
<td>Registered single section images, split by channel</td>
<td>8.7 GB</td>
<td>21</td>
</tr>
<tr>
<td>…/_Output/4_Processed_Sections/</td>
<td> </td>
<td>Temporary processing data</td>
<td>6.3 GB</td>
<td>21</td>
</tr>
<tr>
<td>…/4_Processed_Sections/Detected_Cells/</td>
<td> </td>
<td>Cell detection data</td>
<td>2 MB</td>
<td>21</td>
</tr>
<tr>
<td>…4_Processed_Sections/Enhanced/</td>
<td> </td>
<td>Images enhanced for Ilastik pixel classification; split by channel</td>
<td>4.1 GB</td>
<td>20, 21</td>
</tr>
<tr>
<td>…/4_Processed_Sections/Measured_Intensities/</td>
<td> </td>
<td>Intensity data</td>
<td>3 MB</td>
<td>21</td>
</tr>
<tr>
<td>…/4_Processed_Sections/Object_Detection_Validation/</td>
<td> </td>
<td>Overlay of section images and detected cells; split by channel</td>
<td>2.1 GB</td>
<td>21</td>
</tr>
<tr>
<td>…/4_Processed_Sections/Probability_Masks/</td>
<td> </td>
<td>Probability masks created by Ilastik</td>
<td>66 MB</td>
<td>20, 21</td>
</tr>
<tr>
<td>…/4_Processed_Sections/Resampled_Cells/</td>
<td> </td>
<td>Cells mapped to atlas</td>
<td>2 MB</td>
<td>21</td>
</tr>
<tr>
<td>…/_Output/5_Analysis_Output/</td>
<td> </td>
<td>Analysis output</td>
<td>2 GB</td>
<td>21</td>
</tr>
<tr>
<td> </td>
<td>Annotated_Volumes_XY_10_Z_25micron.csv</td>
<td>Volumes of atlas regions</td>
<td>34 KB</td>
<td>21</td>
</tr>
<tr>
<td> </td>
<td>Transformed_Segment_Annotations.csv</td>
<td>Segment annotations transformed to match template length</td>
<td>9 KB</td>
<td>21</td>
</tr>
<tr>
<td> </td>
<td>Template_Brain_Aligned.tif</td>
<td>Overlay of experimental SC data and SC template</td>
<td>341 MB</td>
<td>21</td>
</tr>
<tr>
<td> </td>
<td>Transformed_Annotations.tif</td>
<td>Transformed atlas annotations</td>
<td>341 MB</td>
<td>21</td>
</tr>
<tr>
<td> </td>
<td>Transformed_Hemisphere_Annotations.tif</td>
<td>Transformed hemisegment boundaries</td>
<td>86 MB</td>
<td>21</td>
</tr>
<tr>
<td> </td>
<td>Transformed_Segments.tif</td>
<td>Transformed segment boundaries</td>
<td>86 MB</td>
<td>21</td>
</tr>
<tr>
<td>…/5_Analysis_Output/Cell_Analysis/</td>
<td> </td>
<td>Cell analysis output lists and graphs</td>
<td>1.1 GB</td>
<td>21</td>
</tr>
<tr>
<td>…/5_Analysis_Output/ Region_Mean_Intensity_Measurements_XY_10_Z_25micron/</td>
<td> </td>
<td>Intensity analysis output lists and graphs</td>
<td>51 MB</td>
<td>21</td>
</tr>
<tr>
<td>…./5_Analysis_Output/ Transform_Parameters/</td>
<td> </td>
<td>Transformation data</td>
<td>9 MB</td>
<td>21</td>
</tr>
<tr>
<td>…/_Output/Ilastik_Projects/</td>
<td> </td>
<td>Ilastik training file(s)</td>
<td>255 MB</td>
<td>20 c</td>
</tr>
<tr>
<td>SpinalJ_example_data_raw 1-6</td>
<td> </td>
<td>Raw .nd2 images of block sections</td>
<td>120 GB</td>
<td>13</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The quality of section registration relies critically on intact spinal sections. Images of damaged and incomplete sections, as well as out-of-focus images, cannot be used and must be replaced with neighboring sections. In the samples analyzed, we had to replace up to 15% of sections distributed throughout the length of any given SC. Comparative analysis of multiple samples suggests that this intervention did not confound the real distribution of cells (<a href="#bib5">Fiederling et al., 2021</a>). Replacing larger numbers of sections may, however, substantially reduce axial resolution locally and introduce artifacts in cell distributions, especially when analyzing small, local cell clusters. Thus, good section quality is critical and tissue segment break points must be avoided in areas of particular interest.</p>
<p>To minimize damage and loss of information when cutting SC tissue for parallel embedding, careful placement of cuts orthogonal to the long axis of the SC is essential to reduce incomplete transverse sections. To generate a complete dataset, cuts can be placed at slightly staggered intervals along the r-c axis of SC in different samples, allowing smooth analysis through breakpoints.</p>
<p>SpinalJ provides the first 3D atlas and a common coordinate framework for mouse SC. The creation of this 3D atlas from 2D annotations that comprise only one annotated section per spinal segment (<a href="#bib1">Allen Institute for Brain Science, 2008</a>) does not provide additional r-c resolution. Rather, the new atlas brings the existing annotations into r-c context and into a format that allows 3D registration and atlas mapping. Thus, the 3D atlas does not account for anatomical differences within the length of individual segments. Moreover, it is important to note that this atlas and the Allen Spinal Cord Atlas annotations are based on sections of a single animal and therefore cannot account for inter-animal variability in SC anatomy.</p>
<p>We propose, however, that iterative mapping of defined populations of neurons to the standard reference in SpinalJ can be used to create and improve 3D annotations and segment boundaries. Providing our tools as an open resource, we hope progressively to improve the quality of the atlas with increasing numbers of datasets and markers mapped by individual labs and/or shared within the community.</p>
<p>Atlas mapping in SpinalJ is achieved by the registration of the DAPI or NT experimental volume to the 3D Nissl template of the atlas. We use an affine 3D transformation to prevent deformation and warping of the experimental data during registration. With this, the width-to-length ratio of the experimental dataset is fixed and must match the template. However, loss of sections can shorten the experimental data along the r-c axis (the average shortening in the samples processed in <a href="#bib5">Fiederling et al., 2021</a> was 39 sections = 975μm), resulting in mismapping. To compensate for this, SpinalJ matches the total r-c length of the experimental dataset to the length of the corresponding segment range in the template by interpolating additional sections. The current compensation approach assumes random/equal spacing of missing sections and does not account for locally concentrated losses, which can, however, be manually accommodated.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Embedding - Tissue pieces do not fit into the wells of the SpineRack.</p>
<p>Related to step 7. Notable resistance when placing tissue pieces into the wells of the SpineRack.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Make sure that the dimensions of the SpineRack match the print file. If the printed product is smaller than expected, calibration of the 3D printer may solve this issue. The SpineRack should also be checked for rough surfaces and print irregularities (especially in the first print layer) protruding into the wells. In such cases, affected print layers can be peeled off.</p>
<p>Alternatively, the print file can be sliced and printed slightly enlarged (e.g., at 110%).</p>
<h3 id="sec5.3">Problem 2</h3>
<p>Sectioning - Difficulties in sectioning through the block.</p>
<p>Related to step 11. Notable resistance when cutting through the block including SpineRack and tissue; sections deform or split frequently.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>After embedding tissue pieces, increase SpineRack softening wait time by a few minutes before freezing the block (step 8). Make sure to remove air bubbles that may prevent contact of OCT with the SpineRack. Use a fresh, sharp blade on the cryostat.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>SpinalJ - Automatic segmentation of block section images fails.</p>
<p>Related to step 15. Automatic segmentation of block section images fails on a high proportion of images, even if image contains nine tissue sections. Segmented images in folder “\Temp\_I_Split\” show empty images or partial sections.</p>
<h3 id="sec5.6">Potential solution</h3>
<p>Adjust segmentation settings in step 14. The lower circularity cutoff value is critical to distinguish tissue sections from debris/auto-fluorescence and may be adjusted upwards (0.1–0.2) if too many objects are detected or downwards (0.0–0.1) if tissue sections are not reliably detected. Optimal settings can be determined manually by running the following sequence of operations on the reference channel of a representative block section image:</p>
<div class="textbox">
<p><code>run("Enhance Contrast", "saturated=0.35");</code></p>
<p><code>run("8-bit");</code></p>
<p><code>setAutoThreshold("Mean dark");</code></p>
<p><code>run("Convert to Mask");</code></p>
<p><code>run("Fill Holes");</code></p>
<p><code>roiManager("reset");</code></p>
<p><code>run("Analyze Particles...", "size=1000000-5000000 pixel circularity=0.1–1.00 show=Nothing include add");</code></p>
</div>
<p>If manual segmentation is preferred, setting extremely stringent auto-segmentation values (e.g., minimal circularity = 0.9) will essentially negate any automatic segmentation.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>SpinalJ – Mask for manual segmentation of block section images does not fit.</p>
<p>Related to step 15. Mask for manual segmentation does not fit the section array.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Check if arrays of tissue sections appear regularly spaced. If not, sections may have been compressed during sectioning → see <a href="#sec5.3">problem 2</a>. If this is the case for all sections, a custom segmentation mask can be created, see below.</p>
<p>Check if the input image resolution matches 1um/px. If not, scale images accordingly, using Fiji&gt;Image&gt;Scale and re-run the analysis. Alternatively, a custom segmentation mask can be created based on an example input image using SpinalJ &gt; Other Tools &gt; Create Segmentation Mask. As input, browse a single channel tiff image of a block section that shows nine tissue sections.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>SpinalJ – Bad atlas alignment.</p>
<p>Related to steps 19 &amp; 21. Experimental dataset does not align well to reference template.</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Make sure to select the correct segment range for the experimental dataset. To identify start and end segments, refer to the Allen Spinal Cord Atlas (<a href="https://mousespinal.brain-map.org/imageseries/showref.html">https://mousespinal.brain-map.org/imageseries/showref.html</a>). When re-adjusting segment range, make sure to delete “DAPI_25.tif” and “DAPI_25_Mask.tif” in folder <i>/3_Registered_Sections/</i> and re-run section registration and atlas mapping.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Allen Institute for Brain Science (2008). Allen Spinal Cord Atlas (Allen Institute for Brain Science), Available at: <a class="external-link" href="https://mousespinal.brain-map.org/imageseries/showref.html">https://mousespinal.brain-map.org/imageseries/showref.html</a>.</p>
<p id="bib2">Berg, S., Kutra, D., Kroeger, T., Straehle, C.N., Kausler, B.X., Haubold, C., Schiegg, M., Ales, J., Beier, T., Rudy, M., et al. (2019). Ilastik: Interactive machine learning for (Bio)Image analysis. Nat. Methods <i>16</i>, 1226-1232. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref2">View at publisher</a></p>
<p id="bib3">Biot, E., Crowell, E., Höfte, H., Maurin, Y., Vernhettes, S., and Andrey, P. (2008). A new filter for spot extraction in N-dimensional biological imaging. In 2008 5th IEEE Int. Symp. Biomed. Imaging from Nano to Macro, Proceedings, ISBI, pp. 975-978. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref3">View at publisher</a></p>
<p id="bib4">Evilsizor, M.N., Ray-Jones, H.F., Lifshitz, J., and Ziebell, J. (2015). Primer for immunohistochemistry on cryosectioned rat brain tissue: example staining for microglia and neurons. J. Vis. Exp. <i>2015</i>, 1-9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref4">View at publisher</a></p>
<p id="bib5">Fiederling, F., Hammond, L.A., Ng, D., Mason, C., and Dodd, J. (2021). Tools for efficient analysis of neurons in a 3D reference atlas of whole mouse spinal cord. Cell Rep. Methods <i>1</i>, 100074. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref5">View at publisher</a></p>
<p id="bib6">Kennedy, H.S., Jones, C., and Caplazi, P. (2013). Comparison of standard laminectomy with an optimized ejection method for the removal of spinal cords from rats and mice. J. Histotechnol. <i>36</i>, 86-91. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref6">View at publisher</a></p>
<p id="bib7">Klein, S., Staring, M., Murphy, K., Viergever, M.A., and Pluim, J.P.W. (2010). Elastix: a toolbox for intensity-based medical image registration. IEEE Trans. Med. Imaging <i>29</i>, 196-205. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref7">View at publisher</a></p>
<p id="bib8">Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref8">View at publisher</a></p>
<p id="bib9">Schmid, B., Schindelin, J., Cardona, A., Longair, M., and Heisenberg, M. (2010). A high-level 3D visualization API for Java and ImageJ. BMC Bioinformatics <i>11</i>, 274. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref9">View at publisher</a></p>
<p id="bib10">Thévenaz, P., Ruttimann, U.E., and Unser, M. (1998). A pyramid approach to subpixel registration based on intensity. IEEE Trans. Image Process. <i>7</i>, 27-41. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00603-1/sref10">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by Jane Dodd (<a href="mailto:jd18@columbia.edu">jd18@columbia.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The SpinalJ package created in <a href="#bib5">Fiederling et al. (2021)</a> is freely accessible through Github: <a href="https://github.com/felixfiederling/SpinalJ">https://github.com/felixfiederling/SpinalJ</a> and Zenodo: <a href="https://doi.org/10.5281/zenodo.5177185">https://doi.org/10.5281/zenodo.5177185</a>. Example raw and processed image data has been deposited at Mendeley Data: <a href="https://data.mendeley.com/datasets/mkc37w4rrv/1">https://data.mendeley.com/datasets/mkc37w4rrv/1</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank Samaher Fageiry for technical advice on SC tissue preparation and histology, Marcela Carmona and Anna Kim for help with cortical virus injections, Crystal K. Colón Ortiz for providing mouse eye samples, and Amy Norovich and Young Mi Kwon for providing betta fish brains. We thank Susan B. Morton and the Jessell Lab for gifts of custom antibodies. We are grateful to Carl E. Schoonover and Sarah Ohashi for help with 3D printing and thank Isobel Jessell for the term “SpineRack”. Thank you to the Mason-Dodd and Bendesky labs for discussion and Christoph Gebhardt for critical comments on the manuscript. Imaging was performed with support from the Zuckerman Institute’s Cellular Imaging platform. This work was funded by <a href="https://doi.org/10.13039/100000002">NIH</a> grants 1R21NS120665-01, 5U19NS104649, and <a href="https://doi.org/10.13039/501100001659">DFG</a> grant FI 2367/1-1.</p>
<h3>Author contributions</h3>
<p>The study was conceptualized and directed by F.F., J.D., and C.M. F.F. developed SpineRacks and performed histology studies, imaging, and data interpretation. L.A.H. and F.F. developed SpinalJ software. D.N. provided VAChT/Foxp1 mice for validation. F.F., L.A.H., and J.D. wrote the manuscript with input from all authors.</p>
<h3>Declaration of interests</h3>
<p>The authors declare that they have no competing interests.</p>
</section>
</article>