<article>
<section>
<h1 id="header">Protocol to study starvation-induced autophagy in developing rat neurons</h1>
<p><time datetime="2021-08-13">Published: August 13, 2021</time></p>
<p>José Wojnacki,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> Sébastien Nola,<sup><a href="#aff1">1</a></sup> and Thierry Galli<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn3">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Université de Paris, Institute of Psychiatry and Neuroscience of Paris (IPNP), INSERM U1266, Membrane Traffic in Healthy &amp; Diseased Brain, 75014 Paris, France</p>
<p id="aff2"><sup>2</sup>GHU PARIS psychiatrie &amp; neurosciences, 75014 Paris, France</p>
<p id="fn1"><sup>3</sup>Present address: Centre for Genomic Regulation, The Barcelona Institute of Science and Technology, 08003 Barcelona, Spain</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:jose.wojnacki@crg.eu">jose.wojnacki@crg.eu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:thierry.galli@inserm.fr">thierry.galli@inserm.fr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100713">10.1016/j.xpro.2021.100713</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Autophagy is being involved in an increasing number of cellular pathways. It now appears that autophagy stimulation and inhibition have complex effects in neurons. Here, we present a simple yet powerful protocol to induce autophagy in primary neurons in culture by partial nutrient deprivation, in neurons with or without transfection of plasmids encoding the Longin domain of VAMP7 or a nanobody directed against VAMP7. Although limited to cells in culture, this protocol can facilitate the study of autophagy in neurons.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib16">Wojnacki et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Primary neuron hippocampal culture for morphological analysis after starvation</li>
<li>In vitro neuronal starvation by culture media dilution</li>
<li>Fixation and immuno staining for morphological analysis</li>
<li>Image acquisition and determination of axonal length after 24 h starvation</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/911-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps to prepare, transfect, starve, and quantify axonal extension of primary rat hippocampal neurons in culture. In many cases where it has been tested, cortical and hippocampal neurons behave very similarly in cell culture. Given the high number of cortical neurons compared to hippocampal neurons, it has become a common practice to use the former for Western Blot analysis while using hippocampal neurons for morphological characterization. We have used an almost identical protocol to prepare and starve rat cortical cultures for Western Blot analysis of autophagy activation by nutrient deprivation.</p>
<h3 id="sec1.1">Preparation of glass coverslips for primary culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [3 days]</div>
<ol>
<li>Clean and sterilize glass coverslips.
<ol type="a">
<li>Place glass coverslips in a ceramic rack and immerse in nitric acid for 10–18 h at 18°C–25°C.
<div class="note">
<span class="note-title">Note:</span> We routinely use ø12 mm coverslips as this provides an optimal ratio between coverslip’s surface / number per petri dish for treatments and microscopic analysis.</div>
</li>
<li>Wash coverslips 4 times, 30 min each with ddH<sub>2</sub>O using a stirring bar.</li>
<li>Sterilize at 160°C–180°C for 4 h.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Steps 1 a and b should be performed under a chemical hood and using personal protective equipment as nitric acid is a highly volatile and corrosive chemical.</div>
<ol start="2">
<li>Coat coverslips with poly-L-lysine.
<ol type="a">
<li>Prepare a solution of borate buffer (0.05 M boric acid and 0.01 M sodium tetrahydroborate in ddH<sub>2</sub>O) and adjust pH to 8.9 with a concentrated solution of HCl or Tris-base as appropriate.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Many laboratories prefer to use ddH<sub>2</sub>O to dissolve poly-L-lysine and it seems to work fine but we haven’t specifically tested the efficacy of this preparation.</div>
</li>
<li>Dissolve poly-L-lysine to 1 mg/mL in borate buffer and filter-sterilize with a 0.2 μm, very low protein-binding filter membrane such as polyvinylidene fluoride (PVDF).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Aliquots of dissolved poly-L-lysine can be stored at −20°C for at least 6 months.</div>
</li>
<li>Place cleaned and sterilized glass coverslips in a plastic petri dish with sterile forceps.</li>
<li>Completely cover each coverslip with sufficient poly-L-lysine 1 mg/mL solution and incubate at 18°C–25°C for 10–18 h.
<div class="note">
<span class="note-title">Note:</span> We usually place 5 ø12 mm coverslips in one ø35 mm petri dish as this is convenient for transfection and treatments but other sizes/numbers should work equally well.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The volume of PLL solution added to each coverslip doesn’t have a significant impact on posterior steps of this protocol, therefore, instead of individually covering each coverslip, the poly-L-lysine solution could be added to the entire petri dish with the only drawback being the amount of the solution used.</div>
</li>
<li>Remove the excess of poly-L-lysine with a vacuum pump.</li>
<li>Wash coated coverslips with ∼1.5 mL sterile ddH<sub>2</sub>O 3 times 60 min each.</li>
<li>Remove the water from the petri dish and add 1.5 mL of plating medium (<a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Preparation of the plating media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Storage</th>
<th>Amount for 50 mL</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>MEM with Earle’s salts and L -glutamine</td>
<td>4°C</td>
<td>45 mL</td>
<td>0.9 X</td>
</tr>
<tr>
<td>d-Glucose</td>
<td>18°C–25°C</td>
<td>0.3 g</td>
<td>0.6% wt/vol</td>
</tr>
<tr>
<td>Horse serum</td>
<td>4°C</td>
<td>5 mL</td>
<td>10% vol/vol</td>
</tr>
</tbody>
</table>
</li>
<li>Store in a CO<sub>2</sub> incubator until use.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Coated coverslips can be kept in the CO<sub>2</sub> incubator for at least one week.</div>
<div class="note">
<span class="note-title">Note:</span> Step 2 manipulations should be performed in a laminar flow hood.</div>
<div class="note">
<span class="note-title">Note:</span> Several variations for storing coated coverslips exist such as drying coated coverslips and storing them at 4°C but we have not specifically tested the advantages or disadvantages of them.</div>
<div class="note">
<span class="note-title">Note:</span> Filter-sterile after preparation.</div>
<div class="note">
<span class="note-title">Note:</span> Once prepared and filtered, the plating media can be stored for at least 3 weeks at 4°C.</div>
<div class="note">
<span class="note-title">Note:</span> Cortical neuronal culture is almost identical to the hippocampal cell culture as one of the steps of it is to remove the cortex to gain access to the hippocampus. Once isolated and cleaned off the meninges, the cortex should be cut into smaller pieces and trypsinized in at least 5 times larger trypsin solution. We usually plate 300,000 neurons/cm<sup>2</sup> in a ø60 mm petri dish (<a href="#bib16">Wojnacki et al,2020</a>).</div>
<h3 id="sec1.2">Primary culture of hippocampal neurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [4–6 h]</div>
<p>Here we describe a summarized protocol to prepare a rat hippocampal neurons cell culture. For a more detailed protocol please refer to previous publications (<a href="#bib5">Kaech and Banker, 2006</a>; <a href="#bib15">Wilson et al., 2020</a>).</p>
<ol start="3">
<li>Hippocampus dissection.
<ol type="a">
<li>Euthanize an E18 - E19 pregnant rat with an ethically approved method. Typically use a CO<sub>2</sub> chamber.</li>
<li>Wet the ventral part of the pregnant rat with 70% ethanol and make an incision.</li>
<li>Remove the uterus.</li>
<li>Remove each embryo from the uterus and separate the head.</li>
<li>Isolate the brain.</li>
<li>Isolate the hippocampus.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Manipulations from step 3 d on should be performed in a laminar flow hood. The hippocampi should be kept submerged in cold HBSS at all times. Ideally, the dissection should be performed in a ø60 mm petri dish on top of a flat ice pack wrapped in absorbent paper soaked with ethanol 70%.</div>
<div class="optional">
<span class="optional-title">Optional:</span> We also isolate the cerebral cortex to culture in parallel primary cortical neurons. As the cortex contains more neurons than the hippocampus, we typically use cortical neurons for Western Blot analyses.</div>
<ol start="4">
<li>Hippocampus dissociation.
<ol type="a">
<li>Collect the hippocampi with a P1000 pipette and place them in a 15 mL conical tube.
<div class="note">
<span class="note-title">Note:</span> The amount of embryos in pregnant rats usually ranges from 10 to 16 depending on the specific strain. We normally recover all the hippocampi from all the embryos and pool them in a single collection tube to reduce the workload. Hippocampi from each embryo can also be processed separately, particularly when individual genotyping is needed.</div>
</li>
<li>Bring the volume up to 4.5 mL with Hanks Balanced Salt Solution (HBSS).</li>
<li>Add 0.5 mL of Trypsin 2.5% and incubate for 15 min at 37°C. Gently mix the solution 2 or 3 times during this incubation period.
<div class="critical">
<span class="critical-title">Critical:</span> Use a fast-transmitting heater such a water bath to quickly warm up the trypsin solution. The conductance of air or metal pellets heating systems is much slower and it may take too long for the trypsin solution to reach the temperature for maximal enzymatic activity.</div>
</li>
<li>Carefully remove the trypsin solution and replace with 5 mL of fresh HBSS. Repeat this step two more times. The final volume after the final wash should be 2–3 mL.</li>
<li>Dissociate the hippocampi by gently pipetting up and down with a P1000 tip 8–10 times. Then place a P200 tip over the P1000 tip and repeat 8–10 times.</li>
<li>Determine cell number (Typically with a Neubauer chamber).
<div class="note">
<span class="note-title">Note:</span> From a typical culture with 12 to 14 embryos we obtain between 7 to 10 millions hippocampal neurons.</div>
</li>
<li>Plate 9,000 neurons per square centimeter in each petri dish with coated coverslips and with plating media (<a href="#tbl1">Table 1</a>).</li>
<li>Put into the CO<sub>2</sub> incubator.</li>
<li>After 2–4 h replace the culture media with complete N2 media (<a href="#tbl2">Table 2</a>).
<table id="tbl2">
<caption>Table 2. Preparation of the N2 media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Storage</th>
<th>Amount for 50 mL</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>MEM with Earle’s salts and L -glutamine</td>
<td>4°C</td>
<td>49.5 mL</td>
<td>n/a</td>
</tr>
<tr>
<td>d-Glucose</td>
<td>18°C–25°C</td>
<td>0.3 g</td>
<td>0.6 % wt/vol</td>
</tr>
<tr>
<td>N2 supplement</td>
<td>−20°C</td>
<td>0.5 mL</td>
<td>1 X</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Filter-sterile after preparation.</div>
<div class="note">
<span class="note-title">Note:</span> Once prepared and filtered, the N2 media can be stored at 4°C for at least 2 weeks.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Preparation of the starvation media</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h]</div>
<p>Neurobasal/B27 is a common culture media for hippocampal neurons but the composition of B27 is proprietary and remains undisclosed. In order to have more control on the diluted components we decided to use the N2 culture media (<a href="#bib5">Kaech and Banker, 2006</a>).</p>
<p>We have tried the dilution of several of the components of the N2 culture media. Insulin dilution had no effect unless completely removed (<a href="#bib16">Wojnacki et al., 2020</a>). Glucose dilution has highly deleterious effects on neuronal survival so we maintained it constant during starvation (<a href="#tbl4">Table 4</a>). If no variations of the N2 composition are needed then it’s more convenient to use the 10× solution available from commercial suppliers as described in this protocol, otherwise the N2 media should be prepared according to the user’s need following <a href="#bib5">Kaech and Banker, 2006</a> description.</p>
<ol start="5">
<li>Dilute 1 to 5 times complete N2 media with Hanks Balanced Salt Solution (HBSS) and add glucose (<a href="#tbl3">Table 3</a>) so its concentration remains unaltered in the diluted media compared to complete N2 media (<a href="#tbl4">Table 4</a>). Store the starvation media at 4°C until use (up to two weeks).
<table id="tbl3">
<caption>Table 3. Preparation of the starvation media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Storage</th>
<th>Amount for 50 mL</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Complete N2 media</td>
<td>4°C</td>
<td>10 mL</td>
<td>0.2 X</td>
</tr>
<tr>
<td>HBSS</td>
<td>4°C</td>
<td>40 mL</td>
<td>0.8 X</td>
</tr>
<tr>
<td>Glucose</td>
<td>18°C–25°C</td>
<td>0.24 g</td>
<td>0.6 % wt / vol</td>
</tr>
</tbody>
</table>
<table id="tbl4">
<caption>Table 4. Composition of the N2 and starvation culture media</caption>
<thead>
<tr>
<th> </th>
<th>N2 media</th>
<th>Starvation media</th>
</tr>
<tr>
<th> </th>
<th>mM</th>
<th>mM</th>
</tr>
<tr>
<th colspan="3">rowheadAmino acids</th>
</tr>
</thead>
<tbody>
<tr>
<td>L-Arginine hydrochloride</td>
<td>0.59716</td>
<td>0.11943</td>
</tr>
<tr>
<td>L-Cystine 2HCl</td>
<td>0.09904</td>
<td>0.01981</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>2.00000</td>
<td>0.40000</td>
</tr>
<tr>
<td>L-Histidine hydrochloride-H2O</td>
<td>0.20000</td>
<td>0.04000</td>
</tr>
<tr>
<td>L-Isoleucine</td>
<td>0.39695</td>
<td>0.07939</td>
</tr>
<tr>
<td>L-Leucine</td>
<td>0.39695</td>
<td>0.07939</td>
</tr>
<tr>
<td>L-Lysine hydrochloride</td>
<td>0.39891</td>
<td>0.07978</td>
</tr>
<tr>
<td>L-Methionine</td>
<td>0.10067</td>
<td>0.02013</td>
</tr>
<tr>
<td>L-Phenylalanine</td>
<td>0.19394</td>
<td>0.03879</td>
</tr>
<tr>
<td>L-Threonine</td>
<td>0.40336</td>
<td>0.08067</td>
</tr>
<tr>
<td>L-Tryptophan</td>
<td>0.04902</td>
<td>0.00980</td>
</tr>
<tr>
<td>L-Tyrosine disodium salt dihydrate</td>
<td>0.19923</td>
<td>0.03985</td>
</tr>
<tr>
<td>L-Valine</td>
<td>0.39316</td>
<td>0.07863</td>
</tr>
<tr>
<td colspan="3">Vitamins</td>
</tr>
<tr>
<td>Choline chloride</td>
<td>0.00714</td>
<td>0.00143</td>
</tr>
<tr>
<td>D-Calcium pantothenate</td>
<td>0.00210</td>
<td>0.00042</td>
</tr>
<tr>
<td>Folic Acid</td>
<td>0.00227</td>
<td>0.00045</td>
</tr>
<tr>
<td>Niacinamide</td>
<td>0.00820</td>
<td>0.00164</td>
</tr>
<tr>
<td>Pyridoxal hydrochloride</td>
<td>0.00490</td>
<td>0.00098</td>
</tr>
<tr>
<td>Riboflavin</td>
<td>0.00027</td>
<td>0.00005</td>
</tr>
<tr>
<td>Thiamine hydrochloride</td>
<td>0.00297</td>
<td>0.00059</td>
</tr>
<tr>
<td>i-Inositol</td>
<td>0.01111</td>
<td>0.00222</td>
</tr>
<tr>
<td colspan="3">N2 supplement</td>
</tr>
<tr>
<td>Sodium Pyruvate (mM)</td>
<td>1.000000</td>
<td>0.200000</td>
</tr>
<tr>
<td>Putrescine (mM)</td>
<td>0.100000</td>
<td>0.020000</td>
</tr>
<tr>
<td>Progesterone (mM)</td>
<td>0.000020</td>
<td>0.000004</td>
</tr>
<tr>
<td>Selenium Dioxide (mM)</td>
<td>0.000030</td>
<td>0.000006</td>
</tr>
<tr>
<td>Bovine Transferrin (g/L)</td>
<td>0.100000</td>
<td>0.020000</td>
</tr>
<tr>
<td>Insulin (g/L)</td>
<td>0.005000</td>
<td>0.001000</td>
</tr>
<tr>
<td colspan="3">Other components</td>
</tr>
<tr>
<td>D-Glucose (Dextrose) g/l</td>
<td>6</td>
<td>6</td>
</tr>
<tr>
<td colspan="3">Inorganic salts</td>
</tr>
<tr>
<td>Calcium Chloride (CaCl2) (anhyd.)</td>
<td>1.80180</td>
<td>1.80180</td>
</tr>
<tr>
<td>Magnesium Sulfate (MgSO4) (anhyd.)</td>
<td>0.81392</td>
<td>0.81392</td>
</tr>
<tr>
<td>Potassium Chloride (KCl)</td>
<td>5.33333</td>
<td>5.33333</td>
</tr>
<tr>
<td>Sodium Bicarbonate (NaHCO3)</td>
<td>26.19048</td>
<td>26.19048</td>
</tr>
<tr>
<td>Sodium Chloride (NaCl)</td>
<td>117.24138</td>
<td>117.24138</td>
</tr>
<tr>
<td>Sodium Phosphate monobasic</td>
<td>1.01449</td>
<td>1.01449</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Filter-sterile after preparation.</div>
<div class="note">
<span class="note-title">Note:</span> Once prepared and filtered, the starvation media can be stored at 4°C for at least 2 weeks.</div>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-Tau1</td>
<td>Merck Millipore</td>
<td>Cat# MAB3420; RRID:AB_11213630</td>
</tr>
<tr>
<td>Anti-beta tubulin</td>
<td>N/A</td>
<td>Clone E7</td>
</tr>
<tr>
<td>Anti-RTN3</td>
<td>Abcam</td>
<td>Cat# ab187764;</td>
</tr>
<tr>
<td>Anti-ULK1</td>
<td>Cell Signaling Technology</td>
<td>Cat# 8054; RRID:AB_11178668</td>
</tr>
<tr>
<td>Anti-pULK1</td>
<td>Cell Signaling Technology</td>
<td>Cat# 14202; RRID:AB_2665508</td>
</tr>
<tr>
<td>Anti-cleaved caspase 3</td>
<td>Cell Signaling Technology</td>
<td>Cat# 9664; RRID:AB_2070042</td>
</tr>
<tr>
<td>DAPI (4′,6-Diamidino-2-Phenylindole, Dilactate)</td>
<td>Invitrogen</td>
<td>D3571</td>
</tr>
<tr>
<td>Beta-Tubulin</td>
<td>Developmental Studies Hybridoma Bank</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>MEM</td>
<td>Invitrogen</td>
<td>Cat# 11095</td>
</tr>
<tr>
<td>Poly-L-lysine hydrobromide</td>
<td>Sigma-Aldrich</td>
<td>Cat# P2636</td>
</tr>
<tr>
<td>D(+)Glucose</td>
<td>Sigma-Aldrich</td>
<td>Cat# G7021</td>
</tr>
<tr>
<td>Donor Horse serum</td>
<td>Biosera</td>
<td>Cat# DH-291H/500</td>
</tr>
<tr>
<td>N2 supplement</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 17502048</td>
</tr>
<tr>
<td>Lipofectamine 2000</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11668030</td>
</tr>
<tr>
<td>HBSS (Hanks’ Balanced Salt solution)</td>
<td>Sigma-Aldrich</td>
<td>Cat# H4641</td>
</tr>
<tr>
<td>Paraformaldehyde</td>
<td>Sigma-Aldrich</td>
<td>Cat# 441244</td>
</tr>
<tr>
<td>Nitric acid</td>
<td>VWR Chemicals</td>
<td>Cat# 20425.297</td>
</tr>
<tr>
<td>Prolong™ Gold Antifade Mountant</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# P36930</td>
</tr>
<tr>
<td>Borid acid</td>
<td>Sigma-Aldrich</td>
<td>Cat# B7660</td>
</tr>
<tr>
<td>Sodium tetraborate decahydrate</td>
<td>Sigma-Aldrich</td>
<td>Cat# B9876</td>
</tr>
<tr>
<td>Ammonium Chloride</td>
<td>Sigma-Aldrich</td>
<td>Cat# A9434</td>
</tr>
<tr>
<td>Bovine Serum Albumine (BSA)</td>
<td>Euromedex</td>
<td>Cat# 04-100-812-E</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat# T9284</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Euromedex</td>
<td>Cat# 2001-C</td>
</tr>
<tr>
<td>cOmplete™ Mini Protease Inhibitor Cocktail</td>
<td>Roche</td>
<td>Cat# 11836153001</td>
</tr>
<tr>
<td>Trypsin 2.5% (10×)</td>
<td>Gibco</td>
<td>Cat# 25090-028</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Protein assay dye reagent concentrate</td>
<td>Bio-Rad</td>
<td>Cat# 500-0006</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Rattus norvergicus</td>
<td>Janvier Labs (France)</td>
<td>Cat# RN-SD-F</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Nanobody mCherry-F1.1</td>
<td>
<a href="#bib16">Wojnacki et al. 2020</a>Hybrigenics SA</td>
<td>N/A</td>
</tr>
<tr>
<td>VAMP7 Longin domain</td>
<td><a href="#bib8">Martinez-Arca et al., 2000</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ</td>
<td>
<a href="#bib14">Schindelin et al., 2012</a>; <a href="#bib13">Rueden et al., 2017</a>
</td>
<td><a href="https://imagej.nih.gov/ij">https://imagej.nih.gov/ij</a></td>
</tr>
<tr>
<td>Simple neurite tracer</td>
<td>
<a href="#bib7">Longair et al., 2011</a>.</td>
<td><a href="https://imagej.net/Simple_Neurite_Tracer">https://imagej.net/Simple_Neurite_Tracer</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Glass coverslips 12 mm Menzel</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# CB00120RA120MNZ0</td>
</tr>
<tr>
<td>Ceramic rack</td>
<td>Life Technologies</td>
<td>Cat# C14784</td>
</tr>
<tr>
<td>Western blotting membranes, nitrocellulose 0.45μm</td>
<td>GE Healthcare</td>
<td>Cat# 10600002</td>
</tr>
<tr>
<td>Spectrophotometer cuvette</td>
<td>Sigma-Aldrich</td>
<td>Cat# BR759015</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Function blocking proteins / nanobodies transfection (optional)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 h]</div>
<p>Full gene KO might cause cells to develop compensatory changes, as shown from experiments comparing full gene KO with knock-down (<a href="#bib12">Rossi et al., 2015</a>). The acute over-expression of a function-blocking protein or synthetic antibody overcomes the potential compensatory changes that cells may develop over long periods of time. As an example of the efficacy of this approach here we present a protocol describing the over-expression of the VAMP7 Longin domain. This domain of the protein was shown to have an auto inhibitory function (<a href="#bib10">Martinez-Arca et al., 2003</a>) and when over-expressed it prevents neurite growth (<a href="#bib8">Martinez-Arca et al., 2000</a>, <a href="#bib9">2001</a>) and inhibits VAMP7-dependent secretion (<a href="#bib4">Gupton and Gertler, 2010</a>). We further used a synthetic single chain variable fragment (ScFv) nanobody directed against VAMP7 (F1.1) (<a href="#bib16">Wojnacki et al., 2020</a>). Indeed, nanobodies directed against specific proteins are another powerful tool to inhibit a protein’s function acutely (<a href="#bib11">Moutel et al., 2016</a>). Here we present a protocol based on a nanobody- or a function-blocking protein-transfection approach to inhibit VAMP7’s function in developing neurons.</p>
<div class="note">
<span class="note-title">Note:</span> Generation and characterization of a fully-functional nanobody is complex and time-consuming. We recommend to call on some companies able to provide VHH screens by phage-display or yeast two hybrid screens.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Not all nanobodies are function-blocking. Some nanobodies may specifically recognize their target protein without blocking its function. This needs to be specifically assayed beforehand.</div>
<ol>
<li>Mix 1 μg of plasmid DNA, encoding either a nanobody or a function-blocking expression protein, with 100 μL of MEM.</li>
<li>Mix 2.5 μL of Lipofectamine 2000 reagent with 100 μL of MEM.</li>
<li>Mix the 100 μL of DNA/MEM and 100 μL of Lipofectamine 2000/MEM and leave at 18°C–25°C for 20 min.</li>
<li>Add the 200 μL solution to a ø35 mm petri dish with coverslips containing 2 days <i>in vitro</i> (DIV) hippocampal neurons and leave for 90 min in the CO<sub>2</sub> incubator.</li>
<li>Change culture media with starvation media (next step).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The transfection could be performed as soon as neurons are plated following an identical protocol. Then the starvation could be started at 2DIV when neurons are already expressing the proteins of interest. This should be avoided if the proteins of interest are known to affect neuronal polarization. Another alternative is to transfect neurons at 2DIV <i>in vitro</i> as here described but changing to starvation media at 3DIV.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Neuron transfection is in general a low-efficiency procedure. Usually between 3% and 5% of neurons are transfected. Highly pure plasmid increases the percentage of transfected neurons and diminishes the toxic effects of the procedure.</div>
<h3 id="sec2.2">Primary neuron starvation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h–2 days]</div>
<p>Primary neurons are remarkably sensitive to culture conditions, including the culture media composition. Several culture media have been established and most of them are formulated to support long-term neuronal development. Starvation conditions imply the lack of one or more of the components that would be required for normal, long-term development of neurons in culture.</p>
<p>Here we describe a procedure and culture media that induces autophagy in cultured neurons without inducing apoptosis or death whilst still allowing axonal extension for at least 24 h.</p>
<ol start="6">
<li>Pre-equilibrate the starvation media in the CO2 incubator for a couple of hours.</li>
<li>Remove the N2 media with a vacuum pump and replace with the starvation media (<a href="#tbl3">Table 3</a>)</li>
<li>Incubate in CO<sub>2</sub> incubator for 24–48 h.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Starvation media can also be applied to cortical neurons culture in an identical way to test autophagy induction by Western Blot.</div>
<h3 id="sec2.3">Fixation and immunostaining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [4–6 h]</div>
<p>The most common way to describe morphological changes in neuronal cultures is to stain components of the cytoskeleton. These components allow to discriminate axons from minor processes or dendrites and at the same time allow for clear measurement of axonal length, ramifications and the number of axons present in each neuron.</p>
<ol start="9">
<li>Fixation.
<ol type="a">
<li>Remove and discard the starvation media.</li>
<li>Add pre-warmed (37°C) paraformaldehyde 4% in PBS to completely cover the coverslips.</li>
<li>Leave for 20 min at 18°C–25°C.</li>
<li>Remove the paraformaldehyde solution and replace with PBS
<div class="note">
<span class="note-title">Note:</span> Dispose the PFA solution according to environmental / safety regulations.</div>
</li>
<li>Wash twice, 4 min each, with PBS.
<div class="note">
<span class="note-title">Note:</span> Steps 9 b, c and d should be performed under a chemical hood.</div>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Immuno-staining.
<ol type="a">
<li>Quench 20 min at 18°C–25°C with 50 mM NH<sub>4</sub>Cl in PBS.</li>
<li>Wash twice, 4 min each, with PBS.</li>
<li>Permeabilize 6 min with 0.1% Triton X-100 in PBS.</li>
<li>Wash twice, 4 min each, with PBS.</li>
<li>Transfer coverslips onto a parafilm sheet in a humid chamber (cells upward).</li>
<li>Block for 30–60 min at 18°C–25°C with 1% BSA in PBS-0.01% Tween-20 (PBST).</li>
<li>Incubate with the primary antibody for 90 min at 18°C–25°C diluted in 1% BSA/PBST.
<div class="note">
<span class="note-title">Note:</span> This incubation protocol works well for the antibodies here described (<a href="#tbl5">Table 5</a>) but adjustments may be necessary if different antibodies are used. Incubation of primary antibodies for 10–18 h at 4°C is a common alternative and in some cases it may increase specificity and reduce noise. Other blocking reagents may also be tried such as secondary antibody same-species serum or gelatin.</div>
</li>
<li>Wash three times, 4 min each, with PBS.
<table id="tbl5">
<caption>Table 5. Primary antibodies used to describe neuronal morphology</caption>
<thead>
<tr>
<th>Antibody</th>
<th>Region labeled</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>Anti Tau1</td>
<td>Axon</td>
<td>1/2000</td>
</tr>
<tr>
<td>Beta Tubulin</td>
<td>All processes</td>
<td>1/8000</td>
</tr>
<tr>
<td>DAPI</td>
<td>Cell nucleus</td>
<td>∼ 0.5μg/mL</td>
</tr>
</tbody>
</table>
</li>
<li>Incubate with the secondary antibody for 60 min at 18°C–25°C diluted in 1% BSA/PBS.</li>
<li>Wash three times, 4 min each, with PBS.</li>
<li>Rinse once in ddH<sub>2</sub>O.</li>
<li>Lay out a drop of ProlongGold (5–9 μL) in a microscopic slide. Place the coverslips on top of the ProlongGold, neurons facing down.
<div class="note">
<span class="note-title">Note:</span> Other mounting medium may be used if preferred. We have tried Mowiol mounting medium with similar results. Yet in our hands ProlongGold has a subjectively superior quality for maintaining fluorescence intensity.</div>
</li>
<li>Let ProlonGold dry over-night.
<div class="note">
<span class="note-title">Note:</span> DAPI staining is not directly used in our protocol for the quantification process itself but it could be useful to discriminate two adjacent neurons. DNA/nucleus staining is a very simple method to assess the number of cells present in the field. We routinely add DAPI (0.5 μg/mL) along with the secondary antibodies incubation. One common alternative is to use mounting medium with DAPI incorporated.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Paraformaldehyde is a highly toxic and volatile chemical. Preparation and usage of it should be performed by experienced personnel, always under a chemical hood and using personal protective equipment. Disposal of this chemical follows safety regulations.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Other molecular markers can be used to detect different neuronal regions. Common options to specifically detect the axon include Tiam1 (<a href="#bib6">Kunda et al., 2001</a>), MAP1B (<a href="#bib3">Gonzalez-Billault et al., 2001</a>) and Cdc42 (<a href="#bib2">Chuang et al., 2005</a>).</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> After the ProlongGold mounting media has dried, coverslips can be stored in the dark and between 4°C - 10°C for at least 10 days before imaging.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.4">Determination of autophagy induction in cortical primary neurons (optional)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 days]</div>
<p>Detection of the cleaved and lipidated LC3B-II by western blot is one of the most common procedures to detect autophagy activation in several cell types. In our hands however, electrophoretic detection of LC3B-II was technically challenging in primary neurons, probably because of high autophagic flux in neurons as already noted (<a href="#bib1">Ariosa and Klionsky, 2016</a>). We have thus relied on the detection of signaling intermediaries of the autophagy pathway: phospho ULK1 (pULK1) by western blot.</p>
<ol start="11">
<li>After starvation lyse neurons with 500 μL of lysis buffer composed of Tris 50 mM; NaCl 150 mM; Triton X-100 1%; 3 mM EDTA and 1× complete protease inhibitor cocktail.</li>
<li>Centrifuge samples at 12,000 <i>g</i> at 4°C for 20 min.</li>
<li>Recover the supernatant and quantify protein concentration.
<ol type="a">
<li>Dilute 5 times the Bio-Rad Bradford reagent with ddH<sub>2</sub>O and dispense 1 mL in as many spectrophotometer cuvettes as samples to be measured and 8 more for the reference curve.</li>
<li>Prepare a reference protein concentration curve with bovine serum albumin (BSA). We usually prepare a curve with 0, 0.5, 1, 2, 4, 8, 16 and 20 μg/μL of BSA.</li>
<li>Dilute 2 μL of the sample in one cuvette and mix thoroughly.</li>
<li>Measure the optic density and 595 nm.</li>
<li>Make a regression curve with the reference values and then determine the sample’s protein concentration.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> With this preparation we usually obtain ∼100 μl of cell lysates of 1 μg/μl to 1.5 μg/μl of protein content.</div>
<ol start="14">
<li>Add Laemmli loading buffer to the samples and boil at 95°C for 5 min.</li>
<li>Load 20–30 μg of total protein extract into a 12% SDS-PAGE gels and run at 80 V for 30 min and then 110 V for another 90 min approximately until the front of migration reaches the lower edge of the gel.</li>
<li>Transfer proteins to a 0.2 μm nitrocellulose membrane at 400 mA for 2–3 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Protein transfer produces large amounts of heat. The transfer chamber should be loaded with chilled transfer buffer and kept in a cold room or in a bucket of ice to prevent over-heating.</div>
<ol start="17">
<li>Block membranes with a solution of 2.5% non-fat dry milk in a Tris-based saline solution containing 0.1% Tween-20 (TBST) for 30–60 min at 18°C–25°C.</li>
<li>Incubate primary antibodies (<a href="#tbl6">Table 6</a>) at 4°C for 10–18 h in TBST supplemented with 2.5% non-fat dry milk solution.
<table id="tbl6">
<caption>Table 6. Primary antibodies used to determine autophagy activation by WB</caption>
<thead>
<tr>
<th>Antibody</th>
<th>Catalog number</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>Anti ULK1</td>
<td>Cell Signaling-8054</td>
<td>1/2000</td>
</tr>
<tr>
<td>Anti pULK1</td>
<td>Cell Signaling-14202</td>
<td>1/2000</td>
</tr>
</tbody>
</table>
</li>
<li>Incubate with secondary antibodies and develop the signal by ECL or fluorescence detection depending on the choice of secondary antibodies.
<div class="note">
<span class="note-title">Note:</span> Anti ULK1 and pULK1 antibodies are both raised in the same species therefore we run 2 gels in order to incubate these antibodies in different membranes.</div>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec3.1">Neuron transfection with function-blocking expression proteins / nanobodies</h3>
<p>As the VAMP7-Longin domain (<a href="#fig1">Figure 1</a>B) and the F1.1 nanobody (<a href="#fig1">Figure 1</a>A) are fused with a fluorescent protein, transfected neurons are simply detectable. The low efficiency of neuronal transfection (3%–5% with this protocol) mostly leads to isolated expressing cells where branching can be easily identifiable and unlikely to overlap with neurites from another positive cell, and then facilitates the microscopic analysis.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/911-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Neurons expressing function-blocking proteins</div>
<p>(A) Neurons transfected with the F1.1 VAMP7 nanobody (mCherry, red) at 2 DIV and left for additional 24 h. During this 24 h period following transfection neurons were placed in starvation media. Neuronal processes were stained with anti-beta tubulin (green). Scale bars represent 30 micrometers.</p>
<p>(B) High magnification of cell bodies from neurons transfected with control red fluorescent protein (RFP) or the VAMP7-Longin domain coupled to RFP. In these images we can see that the VAMP7-Longin domain affects the distribution of endogenous reticulon-3 (RTN3; green) after chemical induction of autophagy. The scale bar represents 5 micrometers.</p>
</figcaption>
</figure>
<p>Neurons transfected with the F1.1 nanobody lost the response to starvation and bear shorter axons compared to control-transfected neurons after 24 h of transfection and starvation (<a href="#fig1">Figure 1</a>A). As another example of a function blocking protein, here we describe the altered distribution of the endogenous protein reticulon-3 after transfection with the VAMP7-Longin domain and 4 h of chemical induction of autophagy by rapamycin treatment (<a href="#fig1">Figure 1</a>B). In the case of expression of the Longin domain, defective secretory reticulophagy supposedly explains the appearance of Reticulon 3 in the whole ER after autophagy induction instead of it being restricted to tubular ER as in control conditions (<a href="#bib16">Wojnacki et al. 2020</a>).</p>
<h3 id="sec3.2">Neuron starvation</h3>
<p>Neuronal starvation for 24 h induces autophagy as detected by Ulk1 phosphorylation (<a href="#fig2">Figure 2</a>B) but does not induce apoptosis as shown by the lack of cleaved caspase 3 by western blot, contrary to staurosporine treatment, a classical apoptosis inducer (<a href="#fig2">Figure 2</a>C). During this 24 h period axonal length increased by ∼50% on average when compared to control neurons (<a href="#fig2">Figure 2</a>A and <a href="#bib16">Wojnacki et al. 2020</a>). Neuronal polarity, as shown by the number of Tau1-positive processes per cell, was not altered by starvation.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/911-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Neuron starvation</div>
<p>(A) Hippocampal neurons in culture stained with an antibody against the axonal protein Tau1. Left image are 3 DIV neurons in culture with control media. Right image are 3 DIV neurons which have been starved for the last 24 h in culture by diluting media five times with HBSS. Tau1 signal is in gray scale and DAPI staining is in blue. Here we show a maximal projection image of a 3D image acquisition using a TLSM Leica SP8 confocal microscope. To generate this image, several fields of view were stitched together with the confocal’s proprietary software. Arrows indicate uni-polar neurons. Arrowheads indicate multi-polar neurons. Green dashed lines are examples of quantified axons and ramifications that equal to the total axonal length. Scale bars represent 100 micrometers.</p>
<p>(B) Western blot of samples from cortical neurons starved for 4 h with diluted N2 culture media. Different membranes were incubated with either anti ULK1 or anti phospho ULK1.</p>
<p>(C) Western blot of samples from neurons starved for 24 h or treated with staurosporine to induce apoptosis as a positive control. The membrane was incubated with anti cleaved caspase 3 antibody.</p>
</figcaption>
</figure>
<h3 id="sec3.3">Autophagy induction</h3>
<p>Nutrient starvation triggers autophagy by inhibiting the phosphorylation of ULK1 downstream mTOR. This ultimately leads to the formation of the phagophore with membrane contributions from different donor compartments. Therefore after 24 h of nutrient starvation a decrease in the level of pULK1 is observed in primary neurons treated with starvation media and this can be used an index of autophagy induction (<a href="#fig2">Figure 2</a>B).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Neuronal culture requires skills and practice. Even under the most exceptional culture conditions, the morphological characteristics of individual neurons in the culture dish can vary. Under a large enough field of view neurons with very long or no axons at all can be seen, particularly in the first few days <i>in vitro</i>. Identifying neurons which are subjectively “sick” or not properly developed is also common. This poses a particular challenge to rigorous quantification of the morphological characteristics that define neuronal maturation and growth even in double-blind image acquisition and quantification.</p>
<p>We highly encourage the acquisition of uninterrupted and very large fields holding several tens of neurons and then quantifying the morphological characteristics of interest (axonal length in the case of this particular protocol) of all the neurons included in the field.</p>
<p>Most modern microscope settings allow for tile-image acquisition mode in which the user can select several millimeters of the sample and the software bundled with the microscope will direct image acquisition and stitch the different fields of view together almost effortlessly.</p>
<p>To measure the axonal length we have used the Fiji/ImageJ’s simple neurite tracer . For a detail protocol on how to use this plug in visit <a href="https://imagej.net/SNT:_Step-By-Step_Instructions">https://imagej.net/SNT:_Step-By-Step_Instructions</a>.</p>
<p>Here we describe a summarized step-by-step protocol of how we have used this plug in to quantify the longest unbranhed axonal length and total axonal length.</p>
<ol>
<li>Open the image to be analyzed with Simple Neurite Trace ImageJ plugin.</li>
<li>Once in the overview page, identify an axon based on Tau1 staining and whose length is at least 3 times longer than the minor processes (usually over 90 μm).</li>
<li>Select a starting point in the cell body and click once.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Simple Neurite Tracer by default snaps the cursor to the highest value pixel in the vicinity.</div>
<ol start="4">
<li>Select the furthest possible point of the axon and click a second time. Simple Neurite Tracer will automatically draw a line over the axon from the first to the second point and store this value.</li>
<li>Confirm the temporary path by pressing “Y” or cancel the path if it’s incorrect by pressing “N”.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This segment represents the longest unbranched axonal segment in the neuron which is sometimes used as a measure of axonal development but it doesn’t include the axonal branches length.</div>
<div class="note">
<span class="note-title">Note:</span> If the axon is highly complex or too curved, Simple Neurite Tracer may not find the correct path. In that case, process by selecting intermediate points along the axon to fully cover it.</div>
<ol start="6">
<li>Include axonal branches by starting a new path on the existing axonal path by pressing “G” near the axonal path.</li>
<li>Holding down “alt” + “shift” keys click with the mouse on the fork point.</li>
<li>Extend the branch with a click at the end of it. Intermediate points can also be selected if necessary.</li>
<li>Confirm the temporary path by pressing “Y”.</li>
<li>When all the axonal branches have been measured select another neurons and repeat the process.</li>
<li>When done, export the data table into a spreadsheet table.</li>
<li>The length of the longest axonal process will appear with “child” length measurements. These are the branches lengths that need to be added to the length of the longest process to obtain the total axonal length.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The longest unbranched and total axonal length may not always change in the same way as these can be independently affected by cellular/molecular processes. Total axonal length may potentially be affected much more than longest unbranched length if a process affecting axonal branching is affected by an experimental treatment.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The starvation culture media we describe in this protocol is based on the dilution of all the amino acids, vitamins and trophic factors present in the complete N2 media formulation. The dilution of individual components present in the N2 culture media, particularly some of the amino acids, could have a stronger impact than dilution of others. Future works should address whether or not all these components have equal effect on autophagy induction and axonal extension.</p>
<p>Neurons in culture are isolated but in the brain they grow side by side with accompanying cells which could have significant effects on neuronal development under nutrient-restriction conditions. This level of complexity is inherently absent in neuronal cultures. Organotypic cultures could overcome some of these limitations but eventually <i>in vivo</i> studies will be required to address the effect of starvation in axonal extension in the developing brain.</p>
<p>Longer than two days <i>in vitro</i> starvation has shown to be deleterious to neurons. This could be a limitation of the culture conditions themselves. Perhaps in the developing brain other factors could allow for longer periods of mild starvation. It’s also possible that improved culture conditions could allow for longer starvation periods.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1 (step 8)</h3>
<p>Neurons die or show clear signs of development arrest before 24 h of starvation.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Glucose is critical for the development of neurons. Check if the diluted medium contains 1 mM glucose.</p>
<h3 id="sec6.3">Problem 2 (step 6)</h3>
<p>Neurons show clear signs of stress soon after changing the N2 culture media with the starvation media.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>The more developed neurons are the more sensitive they become to changes in the culture conditions. Make sure the starvation media was pre-equilibrated in the CO<sub>2</sub> incubator for a couple of hours before replacement.</p>
<h3 id="sec6.5">Problem 3 (steps 1–4)</h3>
<p>No transfection or mayor neuronal death after transfection.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The amount of lipofectamine used may be further reduced and optimized if neuronal viability is compromised. The quality of the plasmid is also very important, use highly pure plasmid DNA.</p>
<h3 id="sec6.7">Problem 4 (optional steps 1–5)</h3>
<p>No observable phenotype after nanobody transfection.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Not all nanobodies are function-blocking. There may recognize its target but not inhibit its function. Test by an independent assay that the nanobody block or alters the function of the target protein.</p>
<h3 id="sec6.9">Problem 5 (step 7)</h3>
<p>Little or no effect of the starvation media on axonal length.</p>
<h3 id="sec6.10">Potential solution:</h3>
<p>Make sure that autophagy is being induced by WB of pULK1 on a parallel neuronal culture.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ariosa, A.R. and Klionsky, D.J. (2016). Autophagy core machinery: overcoming spatial barriers in neurons. J. Mol. Med. <i>94</i>, 1217-1227. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref1">View at publisher</a></p>
<p id="bib2">Chuang, J.-Z., Yeh, T.-Y., Bollati, F., Conde, C., Canavosio, F., Caceres, A., and Sung, C.-H. (2005). The dynein light chain Tctex-1 has a dynein-independent role in actin remodeling during neurite outgrowth. Dev. Cell <i>9</i>, 75-86. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref2">View at publisher</a></p>
<p id="bib3">Gonzalez-Billault, C., Avila, J., and Cáceres, A. (2001). Evidence for the role of MAP1B in axon formation. Mol. Biol. Cell <i>12</i>, 2087-2098. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref3">View at publisher</a></p>
<p id="bib4">Gupton, S.L. and Gertler, F.B. (2010). Integrin signaling switches the cytoskeletal and exocytic machinery that drives neuritogenesis. Dev. Cell <i>18</i>, 725-736. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref4">View at publisher</a></p>
<p id="bib5">Kaech, S. and Banker, G. (2006). Culturing hippocampal neurons. Nat. Protoc. <i>1</i>, 2406-2415. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref5">View at publisher</a></p>
<p id="bib6">Kunda, P., Paglini, G., Quiroga, S., Kosik, K., and Caceres, A. (2001). Evidence for the involvement of Tiam1 in axon formation. J. Neurosci. <i>21</i>, 2361-2372. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref6">View at publisher</a></p>
<p id="bib7">Longair, M.H., Baker, D.A., and Armstrong, J.D. (2011). Simple Neurite Tracer: open source software for reconstruction, visualization and analysis of neuronal processes. Bioinformatics <i>27</i>, 2453-2454. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref7">View at publisher</a></p>
<p id="bib8">Martinez-Arca, S., Alberts, P., Zahraoui, A., Louvard, D., and Galli, T. (2000). Role of tetanus neurotoxin insensitive vesicle-associated membrane protein (TI-VAMP) in vesicular transport mediating neurite outgrowth. J. Cell Biol. <i>149</i>, 889-900. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref8">View at publisher</a></p>
<p id="bib9">Martinez-Arca, S., Coco, S., Mainguy, G., Schenk, U., Alberts, P., Bouillé, P., Mezzina, M., Prochiantz, A., Matteoli, M., Louvard, D., et al. (2001). A common exocytotic mechanism mediates axonal and dendritic outgrowth. J. Neurosci. <i>21</i>, 3830-3838. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref9">View at publisher</a></p>
<p id="bib10">Martinez-Arca, S., Rudge, R., Vacca, M., Raposo, G., Camonis, J., Proux-Gillardeaux, V., Daviet, L., Formstecher, E., Hamburger, A., Filippini, F., et al. (2003). A dual mechanism controlling the localization and function of exocytic v-SNAREs. Proc. Natl. Acad. Sci. U S A <i>100</i>, 9011-9016. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref10">View at publisher</a></p>
<p id="bib11">Moutel, S., Bery, N., Bernard, V., Keller, L., Lemesre, E., de Marco, A., Ligat, L., Rain, J.-C., Favre, G., Olichon, A., et al. (2016). NaLi-H1: A universal synthetic library of humanized nanobodies providing highly functional antibodies and intrabodies. Elife <i>5</i>, e16228. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref11">View at publisher</a></p>
<p id="bib12">Rossi, A., Kontarakis, Z., Gerri, C., Nolte, H., Hölper, S., Krüger, M., and Stainier, D.Y.R. (2015). Genetic compensation induced by deleterious mutations but not gene knockdowns. Nature <i>524</i>, 230-233. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref12">View at publisher</a></p>
<p id="bib13">Rueden, C.T., Schindelin, J., Hiner, M.C., DeZonia, B.E., Walter, A.E., Arena, E.T., and Eliceiri, K.W. (2017). ImageJ2: ImageJ for the next generation of scientific image data. BMC Bioinformatics <i>18</i>, 529. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref13">View at publisher</a></p>
<p id="bib14">Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref14">View at publisher</a></p>
<p id="bib15">Wilson, C., Rozés-Salvador, V., and Cáceres, A. (2020). Protocol for evaluating neuronal polarity in murine models. STAR Protoc. <i>1</i>, 100114. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref15">View at publisher</a></p>
<p id="bib16">Wojnacki, J., Nola, S., Bun, P., Cholley, B., Filippini, F., Pressé, M.T., Lipecka, J., Man Lam, S., N’guyen, J., Simon, A., et al. (2020). Role of VAMP7-dependent secretion of reticulon 3 in neurite growth. Cell Rep. <i>33</i>, 108536. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00420-2/sref16">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Thierry Galli (<a href="mailto:thierry.galli@inserm.fr">thierry.galli@inserm.fr</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This protocol did not generate/need datasets.</p>
<h3>Acknowledgments</h3>
<p>Work in our group was funded by grants from <a href="https://doi.org/10.13039/100007393">Association Française contre les Myopathies</a> (16612), the <a href="https://doi.org/10.13039/501100001665">French National Research Agency</a> (NeuroImmunoSynapse ANR-13-BSV2-0018-02; MetDePaDi ANR-16-CE16-0012), the <a href="https://doi.org/10.13039/501100006364">Institut National Du Cancer</a> (PLBIO 2018-149), the <a href="https://doi.org/10.13039/501100002915">Fondation pour la Recherche Médicale</a> (FRM, Labélisation T Galli), the Fondation de France (00096652), Who am I? <a href="https://doi.org/10.13039/100013349">Labex</a> (Idex ANR-11-IDEX-0005-01), and awards of the Association Robert Debré pour la Recherche Médicale and <a href="https://doi.org/10.13039/501100007492">Fondation Bettencourt-Schueller (Coup d'Elan)</a> to T.G. Imaging was carried out at NeurImag Imaging core facility, part of the IPNP, INSERM U1266, Université de Paris. We thank <a href="https://doi.org/10.13039/501100001674">Leducq Foundation</a> (LEDUCQ RE-TP_2016_U894_GALLI_TANTER) for funding the Leica SP8 Confocal/STED 3DX system. We acknowledge B&amp;B technological core facility of IPNP for technical support and the ImagoSeine core facility of the Institut Jacques Monod, member of the France BioImaging (ANR-10-INBS-04), for their services.</p>
<h3>Author contributions</h3>
<p>Conceptualization, J.W. and T.G.; methodology, J.W. and S.N.; investigation, J.W. and T.G.; data analysis, J.W. and T.G.; writing – original draft, J.W., S.N., and T.G.; funding acquisition, T.G.; supervision, T.G.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>