<article>
<section>
<h1 id="header">An efficient and versatile CRISPR-Cas9 system for genetic manipulation of multi-drug resistant <i>Klebsiella pneumoniae</i>
</h1>
<p><time datetime="2021-03-12">Published: March 12, 2021</time></p>
<p>Thomas H. McConville,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Marla J. Giddins,<sup><a href="#aff1">1</a></sup> and Anne-Catrin Uhlemann<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Division of Infectious Diseases, Department of Medicine, Columbia University Irving Medical Center, 630 West 168th Street, New York, NY 10032, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:au2110@columbia.edu">au2110@columbia.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100373">10.1016/j.xpro.2021.100373</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Multi-drug resistant (MDR) <i>Klebsiella pneumoniae</i> remains an urgent public health threat. While whole-genome sequencing has helped identify genetic changes underlying resistance, functional validation remains difficult due to a lack of genetic manipulation systems for MDR <i>K. pneumoniae</i>. CRISPR-Cas9 has revolutionized molecular biology, but its use was only recently adapted in bacteria by overcoming the lack of genetic repair systems. We describe a CRISPR-Cas9/lambda recombineering system utilizing a zeocin resistance cassette allowing efficient and versatile genetic manipulation of <i>K. pneumoniae</i>.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib9">McConville et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Gene editing for multi-drug resistant <i>Klebsiella pneumoniae</i> utilizing CRISPR-Cas9</li>
<li>Description of plasmid design, cloning, genetic manipulation, and mutant confirmation</li>
<li>Approach allows for gene knockouts and single nucleotide polymorphism editing</li>
<li>“Scarless” editing allows for serial modifications in a single bacterial isolate</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/507-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The purpose of this protocol is to generate a gene specific CRISPR plasmid to allow for a gene knockout or editing of single nucleotides in MDR <i>Klebsiella pneumoniae.</i> This protocol consists of two major sections.</p>
<p>The first section involves designing primers and cloning the CRISPR plasmid for the desired genetic change; either a 120–150 bp deletion or a single nucleotide polymorphism (SNP) insertion. The second section describes the gene deletion and editing procedures to generate the desired genetic change in <i>K. pneumoniae</i> isolates. Please read the entire protocol first and ensure you have all the necessary resources and reagents to carry out the entire protocol.</p>
<p>Of note, the restriction enzymes used for cloning are specific to the pUC19_CRISPR_<i>ΔpmrA</i> plasmid (<a href="#bib3">El-Halfawy et al., 2017</a>; <a href="#bib5">Jiang et al., 2013</a>; <a href="#bib6">Jiang et al., 2016</a>; <a href="#bib7">Jiang et al., 2015</a>; <a href="#bib8">Kovach et al., 1995</a>; <a href="#bib10">Zhao et al., 2016</a>). <i>Aar</i>I <i>+ Bsa</i>I will release the gRNA template DNA + zeocin cassette and <i>Bsa</i>I <i>+ Ahd</i>I will release the homology cassette (<a href="#fig1">Figure 1</a>). The gRNA template DNA is transcribed into the gRNA, which aids the Cas9 protein in inducing a double-strand DNA break necessary for gene editing.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/507-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic of the pUC19_CRISPR_<i>ΔpmrA</i> plasmid and gRNA primers</div>
<p>The CRISPR plasmid template is shown with the necessary primers over-laid to indicate how the gRNA cassette is initially amplified. Noted in yellow are the new N20 sequences for <i>yfg</i> that do not have homology to the CRISPR plasmid used for initial amplification.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Zeocin selection antibiotic</td>
<td>Invivogen</td>
<td>Cat# Ant-zn-5b</td>
</tr>
<tr>
<td>Phusion high-fidelity DNA polymerase</td>
<td>NEB</td>
<td>Cat# M0530S</td>
</tr>
<tr>
<td>
<i>Taq</i> DNA polymerase with standard <i>Taq</i> buffer</td>
<td>NEB</td>
<td>Cat# M0273S</td>
</tr>
<tr>
<td>
<i>Ahd</i>I restriction enzyme</td>
<td>NEB</td>
<td>Cat# R0584S</td>
</tr>
<tr>
<td>
<i>Bsa</i>I HF V2 restriction enzyme</td>
<td>NEB</td>
<td>Cat# R3733S</td>
</tr>
<tr>
<td>
<i>Aar</i>I restriction enzyme</td>
<td>Thermo Fisher</td>
<td>Cat# ER1581</td>
</tr>
<tr>
<td>
<i>Xba</i>I restriction enzyme</td>
<td>NEB</td>
<td>Cat# R0145S</td>
</tr>
<tr>
<td>T4 DNA ligase</td>
<td>NEB</td>
<td>Cat# M0202S</td>
</tr>
<tr>
<td>L-(+)-Arabinose</td>
<td>Thermo Fisher</td>
<td>Cat# A3256-25G</td>
</tr>
<tr>
<td>Molecular biology grade water</td>
<td>GE Life Sciences</td>
<td>Cat# SH30538.02</td>
</tr>
<tr>
<td>Polymyxin B sulfate</td>
<td>Thermo Fisher</td>
<td>Cat# 2180029</td>
</tr>
<tr>
<td>S.O.C. medium</td>
<td>Thermo Fisher</td>
<td>Cat# 15544034</td>
</tr>
<tr>
<td>Mueller-Hinton Broth 2 for microbiology, cation-adjusted</td>
<td>Millipore Sigma</td>
<td>Cat# 90922</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>QIAprep spin mini-prep kit</td>
<td>Qiagen</td>
<td>Cat# 27104</td>
</tr>
<tr>
<td>QIAquick gel extraction kit</td>
<td>Qiagen</td>
<td>Cat# 28115</td>
</tr>
<tr>
<td>QIAquick PCR purification kit</td>
<td>Qiagen</td>
<td>Cat# 28104</td>
</tr>
<tr>
<td>DNeasy Ultraclean microbial kit</td>
<td>Qiagen</td>
<td>Cat# 12224-250</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>One shot TOP10 chemically competent <i>E. coli</i>
</td>
<td>Thermo Fisher</td>
<td>Cat# C404010</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td><i>K. pneumoniae</i></td>
<td>ATCC</td>
<td>ATCC# BAA-1705D-5</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pUC19_CRISPR_<i>ΔpmrA</i>
</td>
<td>Addgene</td>
<td>Addgene# 160903 <a href="https://www.addgene.org/160903/">https://www.addgene.org/160903/</a>
</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Please see <a href="#tbl1">Table 1</a>
</td>
<td>This study</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>SeqBuilder Pro</td>
<td>DNASTAR</td>
<td>Product Name: SeqBuilder Pro</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Bio-Rad T100 thermo cycler</td>
<td>Bio-Rad</td>
<td>Product# 1861096</td>
</tr>
<tr>
<td>Centrifuge 5415 D</td>
<td>Eppendorf</td>
<td>Cat# Z604062</td>
</tr>
<tr>
<td>Qubit 2.0 fluorometer</td>
<td>Thermo Fisher</td>
<td>Cat# Q32866</td>
</tr>
<tr>
<td>Incubator Labnet 211DS</td>
<td>Labnet</td>
<td>SKU# I-5211-DS</td>
</tr>
<tr>
<td>0.5 Polymer McFarland Standard</td>
<td>Thermo Fisher</td>
<td>Cat# E1041</td>
</tr>
<tr>
<td>Disposable culture tube</td>
<td>Fisher</td>
<td>Cat# 1496126</td>
</tr>
<tr>
<td>96 well cell culture plate round bottom with lid</td>
<td>Corning</td>
<td>Product# 3799</td>
</tr>
<tr>
<td>Whole genome sequencing</td>
<td>Genewiz</td>
<td>Website: <a href="https://www.genewiz.com/en/Public/Services/Next-Generation-Sequencing/Whole-Genome-Sequencing">https://www.genewiz.com/en/Public/Services/Next-Generation-Sequencing/Whole-Genome-Sequencing</a>
</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Equipment and basic materials</h3>
<p>This protocol requires standard microbiology laboratory equipment. This includes a PCR machine, microcentrifuge, a device for quantification of nucleic acids, for example a Qubit machine from Thermo Fisher, and two incubators, set at 30°C and 37°C, respectively.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> We have optimized our protocol using enzymes from New England Biolabs, including the Phusion DNA polymerase, <i>Ahd</i>I, <i>Bsa</i>I, <i>Xba</i>I restriction enzymes and the T4 DNA ligase. Equivalent restriction enzymes and polymerases, from companies like Thermo Fisher or Promega, can likely be used interchangeably in this protocol.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol uses Qiagen DNA extraction kits. These can be substituted with the corresponding kits from other suppliers, including Thermo Fisher, Zymo or Promega (plasmid mini-prep, agarose gel extraction, PCR clean up).</div>
<h3 id="sec2.2">Solutions</h3>
<ul>
<li>10% arabinose – Dissolve 1 g of L-arabinose in 9 mL of water and slowly adjust the volume to 10 mL. Filter sterilize this solution. Storage condition: room temperature (20°–25°C) for up to 6 months.</li>
<li>10% glycerol – Add 5 mL of glycerol to 45 mL of PCR grade H<sub>2</sub>O. Always prepare freshly.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Primer design (for gene knockouts)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<div class="note">
<span class="note-title">Note:</span> Please review <a href="#fig2">Figure 2</a>A for the primer design concept. This is a schematic of <i>yfg</i> (<i>your favorite gene –</i> the gene of interest) with all necessary primers and N20 sequences. This will help put in context the different steps for primer design. As a positive control, primers needed to generate a CRISPR plasmid for a <i>mgrB</i> gene knockout are included in <a href="#tbl1">Table 1</a>.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/507-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic of <i>“your favorite gene” (yfg)</i> and homology primers</div>
<p>(A) Depicted is <i>yfg</i> annotated with the necessary primers to PCR amplify the homology for a 120 bp deletion, including the start codon. The chosen N20 sequences are also labeled, along with the Del_F/R primers used to identify the appropriate mutants.</p>
<p>(B) Depicted is the actual sequence and primers necessary to create a <i>pmrA</i> gene knockout.</p>
<p>(C) Primers are shown to generate SNPs in <i>yfg</i>. Notably in the <i>yfg</i>_Up_Hom_SNP_Silent_R the primer is designed to have 15–20 bp of overlapping sequence with the <i>yfg</i>_Dwn_Hom_Silent_F primer.</p>
</figcaption>
</figure>
<table id="tbl1">
<caption>Table 1. Oligonucleotide template sequences for gene knockouts and SNP editing</caption>
<thead>
<tr>
<th>Primer name</th>
<th>Sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<i>yfg</i> _Up_Hom_F_<i>Bsa1</i>
</td>
<td>ATT GGT CTC GCG GT + selected sequence</td>
</tr>
<tr>
<td>
<i>yfg_</i>Up_Hom_R_<i>XbaI</i>
</td>
<td>ATT TCT AGA + reverse complement of selected sequence</td>
</tr>
<tr>
<td>
<i>yfg</i>_Up_Hom_SNP_Silent_R</td>
<td>variable based on desired SNP and inserted silent mutations</td>
</tr>
<tr>
<td>
<i>yfg_</i>Dwn_Hom_F_<i>XbaI</i>
</td>
<td>ATT TCT AGA + selected sequence</td>
</tr>
<tr>
<td>
<i>yfg</i>_Dwn_Hom_Silent_F</td>
<td>variable based on N20 sequence and inserted silent mutations</td>
</tr>
<tr>
<td>
<i>yfg_</i>Dwn_Hom_R_<i>AhdI</i>
</td>
<td>ATT GAC CCC CTG TC + reverse complement of selected sequence</td>
</tr>
<tr>
<td>
<i>yfg_</i>gRNA_1_F</td>
<td>N20 Sequence_1 + GTT TTA GAG CTA GAA ATA GC</td>
</tr>
<tr>
<td>
<i>yfg_</i>gRNA_1_R</td>
<td>reverse complement N20 sequence_1 + CCA CAC ATT ATA CGA GCC GA</td>
</tr>
<tr>
<td>
<i>yfg_</i>gRNA_2_F</td>
<td>N20 sequence_2 + GTT TTA GAG CTA GAA ATA GC</td>
</tr>
<tr>
<td>
<i>yfg_</i>gRNA_2_R</td>
<td>reverse complement N20 sequence_2 + CCA CAC ATT ATA CGA GCC GA</td>
</tr>
<tr>
<td>
<i>yfg_</i>Del_F</td>
<td>selected sequence directly upstream of Up_Hom_F_<i>Bsa</i>I primer</td>
</tr>
<tr>
<td>
<i>yfg_</i>Del_R</td>
<td>reverse complement of selected sequence directly downstream of Dwn_Hom_R_<i>Ahd</i>I primer</td>
</tr>
<tr>
<td>
<i>yfg_</i>SNP_Confirm_F primer</td>
<td>sequence between desired SNP and N20 sequence with inserted silent mutations</td>
</tr>
<tr>
<td>
<i>yfg_</i>SNP_Confirm_R primer</td>
<td>reverse complement of selected sequence directly downstream of Dwn_Hom_R_<i>AhdI</i> primer</td>
</tr>
<tr>
<td>
<i>yfg_</i>gRNA_1_N20_F</td>
<td>N20 Sequence_1</td>
</tr>
<tr>
<td>
<i>yfg_</i>gRNA_2_N20_F</td>
<td>N20 Sequence_2</td>
</tr>
<tr>
<td>CRISPR_Vector_F</td>
<td>5′-GTT GTG AAG AAA AAG TGA ATG ATG TAG CCG</td>
</tr>
<tr>
<td>gRNA_Vector_R</td>
<td>5′-ATT ACC GCG AGA CCG AAA CCA TTA TTA TCA TGA CAT TAA CC</td>
</tr>
<tr>
<td>
<i>mgrB</i> _Up_Hom_F_<i>Bsa1</i>
</td>
<td>5′-ATT GGT CTC GCG GTC GTT TTC CTC ACT GTG GAA T</td>
</tr>
<tr>
<td>
<i>mgrB_</i>Up_Hom_R_<i>XbaI</i>
</td>
<td>5′-ATT TCT AGA TCC ACT CTC CGC ATC AGT AC</td>
</tr>
<tr>
<td>
<i>mgrB_</i>Dwn_Hom_F_<i>XbaI</i>
</td>
<td>5′-ATT TCT AGA GAC ATT TTT CTG CCG ACT GA</td>
</tr>
<tr>
<td>
<i>mgrB_</i>Dwn_Hom_R_<i>AhdI</i>
</td>
<td>5′-ATT GAC CCC CTG TCA ACA ATA AAC GCG GCG AGA A</td>
</tr>
<tr>
<td>
<i>mgrB_</i>gRNA_1_F</td>
<td>5′-GCA TCT GAG TCC ACA GCA ACG TTT TAG AGC TAG AAA TAG C</td>
</tr>
<tr>
<td>
<i>mgrB_</i>gRNA_1_R</td>
<td>5′-GTT GCT GTG GAC TCA GAT GCC CAC ACA TTA TAC GAG CCG A</td>
</tr>
<tr>
<td>
<i>mgrB_</i>Del_F</td>
<td>5′-ATG AAC ATA AAG TGC GGT GCT GAC T</td>
</tr>
<tr>
<td>
<i>mgrB_</i>Del_R</td>
<td>5′-AGT CGT CAC GCT AAA AAT AGT GCT A</td>
</tr>
<tr>
<td>
<i>mgrB</i>_N20_F</td>
<td>5′-GCA TCT GAG TCC ACA GCA AC</td>
</tr>
</tbody>
</table>
<ol>
<li>Copy the first 200 bp (from the ATG start codon) of <i>yfg</i> into the <a href="https://crispr.dbcls.jp/">https://crispr.dbcls.jp/</a> website and click “design.” Please see <a href="#fig2">Figure 2</a>B, which shows the sequence, annotated primers and annotated N20 sequences for the <i>K. pneumoniae pmrA</i> gene as a reference.</li>
<li>Choose two appropriate guide RNA N20 candidate sequences with a melting temperature of 70°C–75°C and a GC content between 40% and 60%.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Be sure to note which strand the N20 sequence is found on (forward or reverse complement strand). PAM sequences (NGG) are found at the 3′ end of the N20 sequence. Two different guide RNA sequences are chosen for each knockout in case the efficiency of one is low in the knockout protocol. This allows for rapid use of the second guide RNA without having to re-design the CRISPR plasmid.</div>
<ol start="3">
<li>Click on the gRNA button next to the two selected N20 sequences to generate the full gRNA sequences. Change RNA to DNA by clicking the U→T button on the next page. This will provide the template DNA sequence for the gRNA.</li>
<li>Copy the first 40 bp of the DNA sequence:
<ul>
<li>N20 Sequence + GTT TTA GAG CTA GAA ATA GC</li>
<li>These sequences will be the gRNA_1_F and gRNA_2 F primers (see <a href="#fig1">Figure 1</a> and <a href="#tbl1">Table 1</a>).</li>
</ul>
</li>
<li>Copy and paste these primer sequences into an excel spreadsheet based on <a href="#tbl1">Table 1</a>.</li>
<li>Create a reverse complement of the two N20 sequences to generate the two gRNA_R primers. The additional sequence shown below is part of the pTAC promoter found directly upstream of the gRNA template DNA in the pUC19_CRISPR_<i>ΔpmrA</i> plasmid.
<ul>
<li>Reverse complement N20 Sequence + CCA CAC ATT ATA CGA GCC GA</li>
<li>These sequences will be the gRNA_1 and gRNA_2 R primers.</li>
</ul>
</li>
<li>Copy and paste these primer sequences into the excel spreadsheet.</li>
<li>Open the full gene sequence for the desired gene to be knocked out in a sequencing analyzing software, like SeqBuilder Pro from DNASTAR. Ensure there is at least 300 bp upstream of the start codon.</li>
<li>Annotate the two N20 sequences in the gene sequence. Ideally, the N20 sequences are within the first 150 bp of the gene.</li>
</ol>
<ol start="10">
<li>Design primers to amplify two areas of homology for the puc19_CRISPR_<i>Δyfg</i> plasmid. Ideally the area of homology (upstream and downstream of the Cas9 cut site) is between 500–1,000 bp. For example, design the upstream homology primers to amplify approximately 250 bp upstream of the Cas9 cut site and the downstream homology to amplify 500 bp downstream of the Cas9 cut site (<a href="#fig2">Figure 2</a>). The sequence between the two areas of homology will be deleted following the knockout protocol.
<div class="note">
<span class="note-title">Note:</span> Four primers will be designed as indicated below. The two middle primers contain an imbedded restriction site. <i>Xba</i>I is shown below, but an alternative restriction site can be substituted. This site is used to ligate the two areas of homology together to create the full homology cassette. The 5′ and 3′ primers contain <i>Bsa</i>I and <i>Ahd</i>I sites respectively, allowing for ligation into the digested pUC19_CRISPR plasmid.</div>
<ol type="a">
<li>
<i>yfg</i>_Up_Hom_F_<i>Bsa</i>I</li>
<li>
<i>yfg</i>_Up_Hom_R_<i>Xba</i>I</li>
<li>
<i>yfg</i>_Dwn_Hom_F_<i>Xba</i>I</li>
<li>
<i>yfg</i>_Dwn_Hom_R_<i>Ahd</i>I</li>
</ol>
</li>
</ol>
<ol start="11">
<li>Choose 20 bp approximately 250 bp upstream of the <i>yfg</i> start codon.
<ul>
<li>ATT + [<i>Bsa</i>I site:GGT CTC GCG GT] + chosen 20 bp</li>
<li>This sequence will be: <i>yfg</i>_Up_Hom_F_<i>Bsa</i>I.</li>
</ul>
</li>
<li>Choose 20 bp around the start codon of <i>yfg</i>
<ul>
<li>ATT + [<i>Xba</i>I site: TCTAGA] + reverse complement of chosen 20 bp</li>
<li>This sequence will be: <i>yfg</i>_Up_Hom_R_<i>Xba</i>I<i>.</i>
</li>
</ul>
</li>
<li>Choose 20 bp approximately 120–150 bp from the start codon of <i>yfg.</i> Ensure the start of this primer is downstream of the second N20 sequence.
<ul>
<li>ATT + [<i>Xba</i>I site: TCTAGA] + chosen 20 bp</li>
<li>This sequence will be: <i>yfg_</i>Dwn_Hom_F_<i>Xba</i>I<i>.</i>
</li>
</ul>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The area between the <i>yfg_</i>Up_Hom_R<i>_XbaI</i> and <i>yfg_</i>Dwn_Hom_F_<i>XbaI</i> primers, which are used to amplify the two homology arms, will be the region within <i>yfg</i> that will be deleted in the final knockout strain. If the area to be deleted exceeds approximately 150 bp the efficiency of homologous recombination decreases.</div>
<ol start="14">
<li>Choose 20 bp approximately 500 bp downstream of the <i>yfg_</i>Dwn_Hom_F primer.
<ul>
<li>ATT + [<i>Ahd</i>I site: GAC CCC CTG TC] + reverse complement of chosen 20 bp</li>
<li>This sequence will be: <i>yfg</i>_Dwn_Hom_R_<i>AhdI.</i>
</li>
</ul>
</li>
<li>Copy and paste the four primer sequences into the excel spreadsheet.</li>
<li>Choose 20–25 bp directly upstream of the <i>yfg</i>_Up_Hom_F primer sequence.
<ul>
<li>This sequence will be: <i>yfg</i>_Del_F.</li>
</ul>
</li>
<li>Choose 20–25 bp directly downstream of the <i>yfg</i>_Dwn_Hom_R<i>_AhdI</i> primer sequence.
<ul>
<li>The reverse complement of this sequence will be: <i>yfg</i>_Del_R.</li>
</ul>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> These primers will be used after the knockout protocol is completed to screen for the appropriate knockout mutant. Mutants that have undergone the appropriate CRISPR-Cas9 gene editing will have a smaller band (based on the amount of the gene deleted) than the WT parent strain.</div>
<ol start="18">
<li>Create two primers with the two N20 sequences used in the two gRNA_F primers. These will be named N20_1_F and N20_2_F. The N20_1_F and N20_2_F and the <i>yfg_</i>Dwn_Hom_R_<i>AhdI</i> primers will create an approximately 1,800 bp band if the final plasmid has ligated together appropriately. Please see the plasmid cloning section for further details.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> These primers will be used to identify the correctly ligated final CRISPR plasmid after the ligation is transformed into chemically competent <i>E. coli</i> cells.</div>
<ol start="19">
<li>Primers CRISPR_Vector F: 5′-GTT GTG AAG AAA AAG TGA ATG ATG TAG CCG and gRNA_Vector_R: 5′-ATT ACC GCG AGA CC G AAA CCA TTA TTA TCA TGA CAT TAA CC are included in the excel spreadsheet and are required to PCR amplify the gRNA cassette from the pUC19_CRISPR_Δ<i>pmrA</i> plasmid.</li>
<li>Order primers from IDTdna or an equivalent company that synthesizes DNA oligonucleotides.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> <a href="#tbl1">Table 1</a> includes validated primers for a knockout of the <i>mgrB</i> gene in <i>K. pneumoniae.</i> These primers can be used as a positive control. The knockout of <i>mgrB</i> induces polymyxin resistance, which is an easy to measure phenotype via broth microdilution or Etest.</div>
<h3 id="sec3.2">Primer design (for SNP insertions)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<div class="note">
<span class="note-title">Note:</span> Please review the schematic in <a href="#fig2">Figure 2</a>C, which illustrates all the necessary primers and N20 sequence for a SNP insertion in an example <i>yfg</i>. For SNP insertions we use the same CRISPR plasmid backbone as for deletions, but replace the origin of replication with pBBR1. This is a broad-host range, low copy number, origin of replication. The lower copy number origin of replication is more efficient for SNP editing. The pBBR1MCS2 plasmid can also be obtained via Addgene and the origin of replication can be amplified and used to replace the pUC origin of replication in the pUC19_CRISPR<i>_ΔpmrA</i> plasmid.</div>
<ol start="21">
<li>Copy the 200 bp downstream of the location of the desired SNP in <i>yfg</i> into the <a href="https://crispr.dbcls.jp/">https://crispr.dbcls.jp/</a> website and click “design.”</li>
<li>Choose <b>one</b> appropriate guide RNA N20 candidate sequences based on the following parameters: a melting temperature of 70°C–75°C, a GC content between 40% and 60%, and approximately 30–60 bp from the desired SNP.</li>
</ol>
<ol start="23">
<li>Follow steps 3–7 of the primer design section for gene knockouts to create the primers to amplify the gRNA template DNA for SNP insertion.
<div class="note">
<span class="note-title">Note:</span> For SNP insertion only one N20 sequence is necessary. The amplified homology cassette will be specific to the selected N20 sequence.</div>
<div class="note">
<span class="note-title">Note:</span> Two primer pairs will be designed as indicated below. One of the two middle primers will contain the desired SNP and both will contain approximately 40 bp of overlapping sequence to allow for a downstream fusion PCR reaction to amplify the full homology cassette. This overlapping sequence will contain <i>yfg</i> gene sequence with a series of silent mutations that help to improve the efficiency of homologous recombination during the knock-in protocol (<a href="#fig2">Figure 2</a>C). The 5′ and 3′ primers will contain <i>Bsa</i>I and <i>Ahd</i>I sites respectively, allowing for future ligation into the digested pBBR1_CRISPR plasmid.</div>
<ol type="a">
<li>
<i>yfg</i>_Up_Hom_F_<i>BsaI</i>
</li>
<li>
<i>yfg</i>_Up_Hom_SNP_Silent_R</li>
<li>
<i>yfg</i>_Dwn_Hom_Silent_F</li>
<li>
<i>yfg</i>_Dwn_Hom_R_<i>AhdI</i>
</li>
</ol>
</li>
</ol>
<ol start="24">
<li>Choose 20 bp approximately 250–500 bp upstream of the desired SNP.
<ul>
<li>ATT + [<i>Bsa</i>I site:GGTCTCGCGGT] + chosen 20 bp</li>
<li>This sequence will be: <i>yfg</i>_Up_Hom_F_<i>Bsa</i>I.</li>
</ul>
</li>
<li>Ensure the sequence analysis software displays the amino acid sequence for <i>yfg</i> in the desired reading frame.</li>
<li>Change the gene sequence to include the desired SNP.</li>
<li>Highlight the sequence directly downstream of the desired SNP and the end of the selected N20 + PAM (NGG) sequence. Identify all the codons that have a wobble bp and change the 3<sup>rd</sup> bp in these codons to insert a silent mutation. Ensure the resulting amino acid sequence (other than with the desired SNP) does not change.</li>
<li>Highlight approximately 15 bp upstream of the SNP and 45 bp downstream of the SNP to create an approximately 60 bp sequence. The reverse complement of this sequence will be: <i>yfg_</i>Up_Hom_SNP_Silent_R.</li>
<li>Identify an approximately 60 bp sequence that includes at least 15 bp of overlap with the <i>yfg_</i>Up_Hom_SNP_Silent_R primer, the modified N20 + PAM sequence and 15 bp of un-edited <i>yfg</i> sequence downstream of the N20 + PAM sequence. This sequence will be: <i>yfg_</i>Dwn_Hom_Silent_F.</li>
<li>Choose 20 bp approximately 250–500 bp downstream of the <i>yfg_</i>Dwn_Hom_Silent_F primer.
<ul>
<li>ATT + [<i>Ahd</i>I site: GAC CCC CTG TC] + reverse complement of the chosen 20 bp</li>
<li>This sequence will be: <i>yfg</i>_Dwn_Hom_R_<i>AhdI.</i>
</li>
</ul>
</li>
<li>Chose 20–25 bp between the SNP and the N20 sequence that contains the inserted silent mutations. This sequence will be: <i>yfg_</i>SNP_Confirm_F.</li>
<li>Choose 20–25 bp directly downstream of the <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i> primer. The reverse complement of this will be: <i>yfg_</i>SNP_Confirm_R.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> These two primers will be used to confirm the successful homologous recombination of the <i>yfg</i> homology into the genome. This primer combination will only amplify genomic DNA that has been modified with the completed <i>yfg</i> SNP CRISPR plasmid.</div>
<ol start="33">
<li>Order primers from IDTdna or an equivalent company that synthesizes DNA oligonucleotides.</li>
</ol>
<h3 id="sec3.3">CRISPR plasmid cloning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–7 days</div>
<ol start="34">
<li>
<b>Day 1:</b> PCR amplify the two areas of homology (Up homology, Down homology) for the homology cassette and the two gRNA PCR products for the gRNA cassettes. Use the Phusion high-fidelity DNA polymerase.
<ol type="a">
<li>Primer Set + Template:
<ol type="i">
<li>
<i>yfg</i>_Up_Hom_F_<i>BsaI</i> + <i>yfg</i>_Up_Hom_R_<i>XbaI</i> – Template: 50–100 ng of genomic DNA<br> Expected product size: approximately 250 bp</li>
<li>
<i>yfg</i>_Dwn_Hom_F_<i>XbaI</i> + <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i> – Template: 50–100 ng of genomic DNA<br> Expected product size: approximately 500 bp</li>
<li>CRISPR_Vector_F + gRNA_1_R – Template: 50–100 ng of pUC19_CRISPR_Δ<i>pmrA</i><br> Expected product size: approximately 380 bp</li>
<li>gRNA_1_F + gRNA_Vector_R – Template: 50–100 ng of pUC19_CRISPR_Δ<i>pmrA</i><br> Expected product size: approximately 1100 bp</li>
</ol>
</li>
</ol>
</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th colspan="5">PCR reaction setup for homology and gRNA cassettes</th>
</tr>
<tr>
<th>Reagent</th>
<th>Homology cassette up</th>
<th>Homology cassette down</th>
<th>gRNA cassette forward</th>
<th>gRNA cassette reverse</th>
</tr>
</thead>
<tbody>
<tr>
<td>H<sub>2</sub>O</td>
<td>Variable</td>
<td>Variable</td>
<td>Variable</td>
<td>Variable</td>
</tr>
<tr>
<td>Phusion 5× Reaction Buffer</td>
<td>10 μL</td>
<td>10 μL</td>
<td>10 μL</td>
<td>10 μL</td>
</tr>
<tr>
<td>Primer 1</td>
<td>2.5 μL <i>yfg</i>_Up_Hom_F_<i>BsaI</i>
</td>
<td>2.5 μL <i>yfg</i>_Dwn_Hom_F_<i>XbaI</i>
</td>
<td>2.5 μL gRNA_1_F</td>
<td>2.5 μLCRISPR_Vector_F</td>
</tr>
<tr>
<td>Primer 2</td>
<td>2.5 μL <i>yfg</i>_Up_Hom_R_<i>XbaI</i>
</td>
<td>2.5 μL <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i>
</td>
<td>2.5 μL gRNA_Vector_R</td>
<td>2.5 μL gRNA_1_R</td>
</tr>
<tr>
<td>2.5 mM dNTPs</td>
<td>4 μL</td>
<td>4 μL</td>
<td>4 μL</td>
<td>4 μL</td>
</tr>
<tr>
<td>Template</td>
<td>50–100 ng genomic DNA</td>
<td>50–100 ng genomic DNA</td>
<td>50–100 ng CRISPR Plasmid DNA</td>
<td>50–100 ng CRISPR Plasmid DNA</td>
</tr>
<tr>
<td>Phusion Polymerase</td>
<td>0.5 μL</td>
<td>0.5 μL</td>
<td>0.5 μL</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Total</td>
<td>50 μL</td>
<td>50 μL</td>
<td>50 μL</td>
<td>50 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Please see <a href="#fig1">Figures 1</a> and <a href="#fig2">2</a> for visualization of the expected PCR products. Since the PCR products will be used in downstream applications we recommend a 50 μL reaction volume following the NEB Phusion polymerase protocol. The PCR parameters shown below can be used for all four PCR reactions.</div>
<table id="undtbl2">
<thead>
<tr>
<th colspan="4">PCR cycling parameters for the Phusion polymerase: CRISPR plasmid cloning</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>40 s</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<ol start="35">
<li>Load all 50 μL of each PCR product on a 1% agarose gel to visualize and ensure the appropriate band sizes indicated above.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The size of the homology arms will depend on the primers chosen above.</div>
<ol start="36">
<li>Gel purify the four PCR products according to the QIAquick gel purification kit protocol (<a href="https://www.qiagen.com/us/resources/resourcedetail?id=a72e2c07-7816-436f-b920-98a0ede5159a&amp;lang=en">https://www.qiagen.com/us/resources/resourcedetail?id=a72e2c07-7816-436f-b920-98a0ede5159a&amp;lang=en</a>). Elute the DNA product in 45 μL of H<sub>2</sub>O. Store the two gRNA PCR products at −20°C</li>
<li>Set up a restriction enzyme digestion with <i>Xba</i>I for the up and down areas of homology. Incubate at 37°C for approximately 1 h.</li>
<li>Clean up the two restriction enzyme digests according to the QIAquick PCR purification kit protocol (<a href="https://www.qiagen.com/us/resources/resourcedetail?id=a72e2c07-7816-436f-b920-98a0ede5159a&amp;lang=en">https://www.qiagen.com/us/resources/resourcedetail?id=a72e2c07-7816-436f-b920-98a0ede5159a&amp;lang=en</a>).</li>
<li>Quantify the concentration of the two areas of homology and set up a ligation at a 1:1 molar ratio of each with T4 DNA ligase. Follow the protocol from NEB to setup a 20 μL ligation (<a href="https://www.neb.com/protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202">https://www.neb.com/protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202</a>). Allow the ligation to incubate at room temperature (20°C–25°C) for approximately 1 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This ligation reaction will generate three different product sizes, assuming 100% ligation efficiency. A 500 bp product (up homology + up homology) a 750 bp product (up homology + down homology) and a 1000 bp product (down homology + down homology). The 750 bp product is the desired homology cassette that will induce the desired deletion after homologous recombination into <i>yfg</i>.</div>
<ol start="40">
<li>Run the entire ligation reaction on a 1% agarose gel, allowing for appropriate separation of the three expected bands: 500 bp, 750 bp and 1,000 bp.</li>
<li>Gel purify the 750 bp band using the QIAquick gel purification kit. Elute product in 30 μL of H<sub>2</sub>O.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For amplification of the homology for a SNP insertion the amplification and gel extraction steps are the same. Since the second PCR reaction utilizes a fusion PCR technique, restriction enzyme digestion + ligation is not necessary. It will be necessary, however, to quantify the concentration of the two areas of homology after gel purification to generate a 1:1 molar ratio in the downstream fusion PCR reaction.</div>
<ol start="42">
<li>Set up PCR reactions using the Phusion polymerase to amplify the full gRNA cassette, the ligated <i>yfg</i> homology cassette or the <i>yfg</i> homology cassette for a SNP insertion. Since these PCR products will be used for downstream cloning, we recommend a 100 μL reaction to ensure enough product.
<ol type="a">
<li>
<i>yfg</i>_Up_Hom_F_BsaI + <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i> (gene deletion primers)– Template: 1 μL of gel purified ligation. Expected band size: ~750 bp</li>
<li>
<i>yfg</i>_Up_Hom_F_BsaI + <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i> (SNP insertion primers)– Template: gel purified PCR products at a 1:1 molar ratio (10–100 ng of DNA) – Expected band size – depends on selected primers</li>
<li>CRISPR_Vector_F + gRNA_Vector_R – Template: the two gRNA PCR products at a 1:1 molar ratio (10–100 ng of DNA) – Expected Band size: approximately 1500 bp</li>
</ol>
</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th colspan="4">PCR reaction setup for homology and gRNA cassettes</th>
</tr>
<tr>
<th>Reagent</th>
<th>Full homology cassette – gene deletion</th>
<th>Full homology cassette – SNP insertion</th>
<th>Full gRNA cassette</th>
</tr>
</thead>
<tbody>
<tr>
<td>H<sub>2</sub>O</td>
<td>Variable</td>
<td>Variable</td>
<td>Variable</td>
</tr>
<tr>
<td>Phusion 5× Reaction Buffer</td>
<td>10 μL</td>
<td>10 μL</td>
<td>10 μL</td>
</tr>
<tr>
<td>Primer 1</td>
<td>2.5 μL <i>yfg</i>_Up_Hom_F_<i>BsaI</i>
</td>
<td>2.5 μL <i>yfg</i>_Up_Hom_F_<i>BsaI</i>
</td>
<td>2.5 μL CRISPR_Vector_F</td>
</tr>
<tr>
<td>Primer 2</td>
<td>2.5 μL <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i>
</td>
<td>2.5 μL <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i>
</td>
<td>2.5 μL gRNA_Vector_R</td>
</tr>
<tr>
<td>2.5 mM dNTPs</td>
<td>4 μL</td>
<td>4 μL</td>
<td>4 μL</td>
</tr>
<tr>
<td>Template</td>
<td>1 μL ligated homology</td>
<td>1:1 molar ratio of the Up / Down homology PCR products</td>
<td>1:1 molar ratio of the two gRNA PCR products</td>
</tr>
<tr>
<td>Phusion Polymerase</td>
<td>0.5 μL</td>
<td>0.5 μL</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Total</td>
<td>50 μL</td>
<td>50 μL</td>
<td>50 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The PCR reactions to generate the full gRNA cassette and the SNP homology are fusion PCR reactions. The N20 sequence in the gRNA_F primer and the reverse complement of the N20 sequence in the gRNA_R primer will bind via homology allowing for amplification of the full 1500 bp cassette. Similarly, the overlapping area of silent mutations in the <i>yfg</i>_Up_Hom_SNP_Silent_R and <i>yfg</i>_Dwn_Hom_Silent_F primers will bind via homology allowing for amplification of the full cassette (<a href="#fig2">Figure 2</a>C). The PCR parameters below can be used for all three PCR reactions.</div>
<table id="undtbl4">
<thead>
<tr>
<th colspan="4">PCR cycling parameters for the Phusion polymerase: CRISPR plasmid cloning</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This is an appropriate stopping point for day 1. The PCR reactions will be stable overnight, 12–18 h, in the PCR machine at 4°C–12°C, or in the refrigerator.</div>
<ol start="43">
<li>
<b>Day 2:</b> Run each of the 100 μL PCR reactions (homology cassette and gRNA cassette) on a 1% agarose gel. Gel purify the cassettes using the QIAquick gel purification kit. Expected band sizes are:
<ol type="a">
<li>The gRNA cassette: approximately 1,500 bp,</li>
<li>
<i>yfg</i> homology cassette for knockout: 750 bp,</li>
<li>
<i>yfg</i> homology cassette for SNP insertion: variable.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> With the gRNA cassette amplification, the fusion PCR technique will yield several smaller, non-specific bands. Make sure to only extract the top most band corresponding to the full 1,500 bp cassette.</div>
<ol start="44">
<li>Set up three separate digests shown below.
<ol type="a">
<li>gRNA cassette: <i>Aar</i>I</li>
<li>
<i>yfg</i> homology cassette: <i>Ahd</i>I and <i>Bsa</i>I HF-V2</li>
<li>pUC19_CRISPR_<i>ΔpmrA</i>: <i>AarI</i>
</li>
</ol>
</li>
</ol>
<table id="undtbl5">
<thead>
<tr>
<th>Components</th>
<th>gRNA cassette</th>
<th>
<i>Yfg</i> homology cassette</th>
<th>pUC19_CRISPR_<i>ΔpmrA</i>
</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNA</td>
<td>41.5 μL</td>
<td>40 μL</td>
<td>41.5 μL</td>
</tr>
<tr>
<td>Restriction Enzyme 10× Buffer</td>
<td>5 μL</td>
<td>5 μL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Oligo (<i>Aar</i>I specific)</td>
<td>1 μL</td>
<td>N/A</td>
<td>1 μL</td>
</tr>
<tr>
<td>Restriction Enzyme</td>
<td>2.5 μL</td>
<td>2.5 μL <i>Ahd</i>I2.5 μL <i>Bsa</i>I HF-V2</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Total</td>
<td>50 μL</td>
<td>50 μL</td>
<td>50 μL</td>
</tr>
</tbody>
</table>
<p>Allow the digests to incubate for 4 h at 37°C.</p>
<ol start="45">
<li>Purify the three digestion reactions using the QIAquick PCR purification kit. Elute the digestion reactions in 45 μL of H<sub>2</sub>O. Store the <i>yfg</i> homology cassette at −20°C.</li>
<li>Set up secondary digestion reactions for the gRNA cassette and the CRISPR plasmid vector.
<ol type="a">
<li>gRNA cassette <i>Aar</i>I digested: <i>Bsa</i>I HF-V2</li>
<li>pUC19_CRISPR_<i>ΔpmrA AarI</i> digested: <i>Ahd</i>I</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The secondary <i>Bsa</i>I digest in the gRNA cassette allows it to be ligated to the 5′ end of the homology cassette in the ligation reaction described below. The secondary <i>Ahd</i>I digest in the pUC19_CRISPR_<i>ΔpmrA</i> plasmid will release the <i>pmrA</i> gRNA and homology cassettes. The flanking <i>Aar</i>I and <i>Ahd</i>I sites on the <i>yfg</i> gRNA and homology cassettes will ligate into the digested pUC19_CRISPR_<i>ΔpmrA</i> plasmid.</div>
<table id="undtbl6">
<thead>
<tr>
<th>Components</th>
<th>gRNA cassette</th>
<th>pUC19_CRISPR_<i>ΔpmrA</i>
</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNA</td>
<td>42.5 μL</td>
<td>42.5 μL</td>
</tr>
<tr>
<td>Restriction Enzyme 10× Buffer</td>
<td>5 μL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Restriction Enzyme</td>
<td>2.5 μL</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Total</td>
<td>50 μL</td>
<td>50 μL</td>
</tr>
</tbody>
</table>
<p>Allow the digests to incubate at 37°C for 4 h.</p>
<ol start="47">
<li>Purify the two repeat digestion reactions using the QIAquick PCR purification kit. Elute the digestion reactions in 30 μL of H<sub>2</sub>O.</li>
<li>Quantify the DNA concentration of the three digestion products.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> This is a safe stopping point. The purified DNA products can be stored at −20°C for several weeks.</div>
<ol start="49">
<li>Set up a 20 μL ligation reaction using the T4 DNA ligase. Ensure a 1:3:3 molar ratio of the pUC19 CRISPR vector : the gRNA cassette : the <i>yfg</i> homology cassette. Use at least 100 ng of the pUC19_CRISPR vector. Incubate the ligation at 16°C for 12–18 h (overnight).</li>
</ol>
<table id="undtbl7">
<thead>
<tr>
<th>Components</th>
<th>Full CRISPR plasmid ligation</th>
</tr>
</thead>
<tbody>
<tr>
<td>T4 DNA Ligase Buffer</td>
<td>2 μL</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>Variable</td>
</tr>
<tr>
<td>pUC19 CRISPR Vector</td>
<td>100–200 ng</td>
</tr>
<tr>
<td>gRNA cassette</td>
<td>3× molar ratio of vector</td>
</tr>
<tr>
<td>
<i>yfg</i> homology</td>
<td>3× molar ratio of vector</td>
</tr>
<tr>
<td>T4 DNA ligase</td>
<td>1.0</td>
</tr>
<tr>
<td>Total</td>
<td>20 μL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Since the final plasmid is between 11,000 to 12,000 bp and this ligation reaction includes a vector and two inserts, the final efficiency will be low. As a result, the extended incubation for 12–16 h (overnight) is necessary to maximize transformation efficiency.</div>
<ol start="50">
<li>
<b>Day 3:</b> Heat inactivate the ligation reaction at 65°C for 10 min.</li>
<li>Store reaction at −20°C until ready for transformation.</li>
<li>Transform 5 μL of the ligation reaction into an aliquot of TOP10 cells. Follow the provided protocol from Invitrogen for chemical transformation (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Foneshottop10_man.pdf&amp;title=T25lIFNob3QgVE9QMTAgQ29tcGV0ZW50IENlbGxz">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Foneshottop10_man.pdf&amp;title=T25lIFNob3QgVE9QMTAgQ29tcGV0ZW50IENlbGxz</a>).</li>
<li>Add 5 μL of the ligation reaction to a thawed aliquot of TOP10 cells on ice.</li>
<li>Incubate the TOP10 Cells on ice for 30 min.</li>
<li>Heat shock the TOP10 cells for 30 s at 42°C. Let sit on ice for 2 min.</li>
<li>Add 250 μL of S.O.C. media to the TOP10 cells and shake at 37°C and 220 RPM for 1 h.</li>
<li>Spin down the transformation reaction in a microcentrifuge at 4,000 × <i>g</i> for 10–15 min.</li>
<li>Re-suspend the bacterial pellet in 50–100 μL of residual S.O.C. media.</li>
<li>Spread plate the transformation reaction on Luria Bertani (LB) plates supplemented with zeocin at 37.5 μg/ml.</li>
<li>Incubate plate at 37°C overnight (12–18 h).</li>
<li>
<b>Day 4:</b> Perform colony PCR to identify resulting transformants that contain the desired final pUC19_CRISPR<i>_Δyfg</i> plasmid.</li>
<li>Pick 24–48 colonies and re-suspend in 25μL of H<sub>2</sub>O.</li>
<li>Set up a PCR master mix using the gRNA_N20_F primer(s) and the <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i> primer and a standard <i>Taq</i> polymerase. NEB <i>Taq</i> polymerase works efficiently for this reaction. Follow the NEB protocol for a 25 μL reaction (<a href="https://www.neb.com/protocols/0001/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273">https://www.neb.com/protocols/0001/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273</a>)</li>
<li>Add 1 μL of each picked colony as template for the PCR reaction.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The gRNA_N20_F primer(s) only contains the N20 sequence found in the gRNA cassette(s). This primer combined with <i>yfg</i>_Dwn_Hom_R_<i>AhdI</i> primer will yield a 1,800–2,000 bp product if the two inserts ligated into the pUC19_CRISPR vector in the appropriate orientation. See below for the appropriate PCR reaction conditions using the NEB <i>Taq</i> polymerase.</div>
<table id="undtbl8">
<thead>
<tr>
<th colspan="4">PCR cycling parameters for standard <i>Taq</i> polymerase: Final CRISPR plasmid confirmation</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>2 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>68°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<ol start="65">
<li>Run the PCR products on a 1% agarose gel.</li>
<li>Identify the corresponding picked colonies that yielded a 1,800–2,000 bp band indicating the desired final pUC19_CRISPR<i>_yfg</i> plasmid.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We often see between 50–100 colonies on the transformation plate. In our experience approximately 10%–50% of picked colonies contain the desired plasmid.</div>
<ol start="67">
<li>Inoculate the remaining volume of the PCR-confirmed picked colonies into 5 mL of LB broth supplemented with 37.5 μg/mL of zeocin. Incubate shaking at 220 RPM and 37°C overnight, 12–18 h.</li>
<li>
<b>Day 5:</b> Mini-prep the overnight cultures using the QIAprep plasmid mini kit (<a href="https://www.qiagen.com/us/resources/resourcedetail?id=22df6325-9579-4aa0-819c-788f73d81a09&amp;lang=en">https://www.qiagen.com/us/resources/resourcedetail?id=22df6325-9579-4aa0-819c-788f73d81a09&amp;lang=en</a>). Elute the final CRISPR plasmid in 40 μL of H<sub>2</sub>O.</li>
<li>Quantify the DNA concentration of the resulting plasmids. Expected concentration: 50–200 ng/μL of DNA.</li>
<li>Send the prepped plasmids for Sanger sequencing to ensure the desired sequence is present.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Send three sequencing reactions for each plasmid to confirm the most important portions of the plasmid. The CRISPR_Vector_F primer can be used to sequence the gRNA template DNA and ensure the new N20 sequence has been inserted. The <i>yfg</i>_Up_Hom_F_<i>BsaI</i> primer can be used to sequence the <i>yfg</i> homology ensuring the up / down areas of homology are in the correct orientation with the imbedded <i>Xba</i>I site, or the desired SNP and silent mutations are present. Finally, the M13R primer, which is a universal primer at most Sanger sequencing companies, can be used to ensure that sequences for the <i>ara</i>BAD promoter and the <i>cas9</i> gene are intact.</div>
<h3 id="sec3.4">
<i>MgrB</i> knockout/SNP editing and mutant confirmation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7–10 days</div>
<ol start="71">
<li>
<b>Days 1–3: Create an electrocompetent stock of the K. pneumoniae isolate to be modified.</b> Streak the desired isolate onto a non-selective LB plate. Incubate at 37°C overnight for 12–16 h.</li>
<li>Pick a single colony and start an overnight seed culture in 5 mL of non-selective LB broth. Incubate shaking at 220 RPM and 37°C overnight, 12–18 h.</li>
<li>Sub-culture the overnight culture into 25 mL of LB at a 1:100 dilution.</li>
<li>Grow the culture, shaking at 220 RPM and 37°C until an OD<sub>600</sub> of 0.35–0.45 is reached.</li>
<li>Chill the culture on ice for 20 min.</li>
<li>Spin the culture down at 4,000 × <i>g</i> for 20 min and pour off the supernatant.</li>
<li>Re-suspend the bacterial pellet in 2.5 mL of ice-cold 10% glycerol.</li>
<li>Spin the suspension down at 4,000 × <i>g</i> for 20 min and discard the supernatant.</li>
<li>Repeat the wash step with 2.5 mL of ice-cold 10% glycerol.</li>
<li>Spin the suspension at 4,000 × <i>g</i> for 20 min and pour off supernatant.</li>
<li>Re-suspend the bacterial pellet in 330 μL of ice-cold 10% glycerol.</li>
<li>Aliquot 50 μL of the re-suspension into 0.5 or 1.5 mL microcentrifuge tubes.</li>
<li>Snap freeze the 50 μL aliquots in liquid nitrogen and store at −80°C for future use.</li>
<li>Days 3–4: Electroporate sequence confirmed CRISPR plasmid into the electrocompetent stock and confirm uptake of plasmid via colony PCR. Thaw a 50 μL aliquot of the electrocompetent isolate on ice.</li>
<li>Add 100–200 ng of the pUC19_CRISPR_<i>ΔmgrB</i> or pBBR1_CRISPR<i>_yfg</i>_SNP plasmid (no more than 5 μL total) to the thawed cells.</li>
<li>Transfer the re-suspension to an ice-cold 1 mm cuvette.</li>
<li>Electroporate with a voltage of 1800 V, capacitance of 25 μF and a resistance of 200 ohms.</li>
<li>Immediately add 1 mL of S.O.C media. Recover cells shaking at 220 RPM and 37°C for 1 h.</li>
<li>Spin down the bacterial culture at 4,000 × <i>g</i> for 10 min. Re-suspend the bacterial pellet in 50–100 μL of residual media.</li>
<li>Plate the entire re-suspension onto LB supplemented with zeocin at 1000 μg/mL. Incubate at 37°C overnight, 12–18 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Plating a lower concentration of the transformation reaction (only 50 μL or 1:10 / 1:100 dilutions) may be necessary depending on the underlying competency of the <i>K. pneumoniae</i> strain to be modified. The ST258 carbapenem resistant <i>K. pneumoniae</i> isolates are minimally competent and thus require plating a higher concentration of the transformation reaction. Additionally, it may be necessary to change the final concentration of zeocin in the LB plates. Performing a broth microdilution with zeocin and the isolate to be modified will allow for more precise prediction of the isolate’s minimum inhibitory concentration (MIC) to zeocin. 1000 μg/mL is an appropriate zeocin concentration for the majority of the carbapenem resistant ST258 <i>K. pneumoniae</i> isolates.</div>
<ol start="91">
<li>Pick between 8 and 32 colonies on the resulting plate for colony PCR to confirm the CRISPR plasmid has been taken up.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> There is low-level background resistance to zeocin that develops in vitro for some of the <i>K. pneumoniae</i> isolates. As a result, this colony PCR step to confirm the desired plasmid has been taken up by the <i>K. pneumoniae</i> isolate is necessary.</div>
<ol start="92">
<li>Re-suspend the picked colonies in 25 μL of water.</li>
<li>Use the CRISPR_Vector_F and gRNA_Vector_R primers to confirm the presence of the CRISPR plasmid using colony PCR with standard <i>Taq</i> polymerase. Follow the NEB protocol for a 25 μL reaction (<a href="https://www.neb.com/protocols/0001/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273">https://www.neb.com/protocols/0001/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273</a>). The pUC19_CRISPR_<i>ΔpmrA</i> plasmid can be used as a positive control. The expected band is approximately 1,500 bp. Use 1 μL of each picked colony as a template. Below are the appropriate PCR reaction conditions.</li>
</ol>
<table id="undtbl9">
<thead>
<tr>
<th colspan="4">PCR cycling parameters for standard <i>Taq</i> polymerase: plasmid uptake confirmation</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">35 Cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>1:30 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>68°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<ol start="94">
<li>Run the resulting PCR products on a 1% agarose gel to identify colonies that harbor the CRISPR plasmid, demonstrated by a 1,500 bp band.</li>
<li>Store the picked colonies that have taken up the plasmid at −20°C for future use in the knockout protocol.</li>
<li>
<b>Day 5: MgrB knockout or SNP editing protocol.</b> Inoculate 10 μL of each picked colony into two 5 mL cultures of LB supplemented with 1000 μg/mL of zeocin. Label one culture “induced” and one culture “control”.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The knockout and SNP editing protocols are identical for either a deletion mutant or a SNP mutant.</div>
<div class="critical">
<span class="critical-title">Critical:</span> There is variability in the efficiency of the CRISPR-Cas9 protocol. As a result, choosing 2–4 colonies that contain the CRISPR plasmid to use in the knockout / SNP editing protocol allows for higher likelihood of success.</div>
<ol start="97">
<li>Grow the two cultures per picked colony shaking at 220 RPM and 30°C for 2 h.</li>
<li>After 2 h add 100 μL of 10% arabinose to the “induced” culture to achieve a final concentration of 0.2% arabinose. The arabinose will induce the expression of the <i>cas9</i> and lambda red recombineering genes (see below).</li>
<li>Grow the induced and control cultures shaking at 220 RPM and 30°C for 6–8 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Since the plasmid contains two arabinose promoters, one upstream of the <i>cas9</i> gene and one upstream of the lambda red recombineering genes, only the induced cultures will express the CRISPR-Cas9 machinery. Since the double-strand break induced by the combination of Cas9 and the <i>mgrB</i> gRNA is toxic to growing bacterial cells, only those isolates that undergo homologous recombination with the <i>mgrB</i> homology will survive. Since the homologous recombination event is not 100% efficient, the induced culture should be significantly less turbid than the control culture (<a href="#fig3">Figure 3</a>).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/507-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Differential growth of the induced versus control cultures</div>
<p>Shown in the left panel are two cultures from a single picked colony containing a CRISPR plasmid. The left culture was induced with 2% arabinose and has grown significantly slower than the control culture, which has not been induced. On the right is the resulting plate showing significantly fewer colonies on the induced side compared with the control side.</p>
</figcaption>
</figure>
<ol start="100">
<li>Measure the OD<sub>600</sub> of the induced and control cultures after the 6–8 h of growth. If the OD<sub>600</sub> is greater than 0.10 (visibly turbid) make a 1:100 dilution of the cultures to ensure single colonies upon plating. If the OD<sub>600</sub> is less 0.1 (not visibly turbid) plate directly from the resulting culture</li>
<li>Plate 10 μL of the induced culture dilution on LB + 1000 μg/mL of zeocin + 0.2%
<ul>
<li>arabinose and 10 μL of the control culture dilution on LB + 1000 μg/mL of zeocin only.</li>
<li>Incubate the plates at 30°C overnight, 12–18 h.</li>
</ul>
</li>
<li>
<b>Day 6: mgrB knockout mutant or SNP edited mutant confirmation.</b> Perform colony PCR on 24–48 colonies from the induced plate using the <i>mgrB_</i>Del F/R primers to detect a deletion within the genome. Use the standard <i>Taq</i> polymerase and follow the NEB protocol for a 25 μL reaction (<a href="https://www.neb.com/protocols/0001/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273">https://www.neb.com/protocols/0001/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273</a>). Run a control reaction with a colony of the wild type parent <i>K. pneumoniae</i> isolate. Use 1 μL of each picked colony as a template.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> These primers bind genomic DNA upstream and downstream of the homology contained in the pUC19_CRISPR_<i>ΔmgrB</i> plasmid, thus will not amplify the plasmid and will only amplify genomic DNA (see <a href="#fig2">Figure 2</a>A). If the double-strand break and homologous recombination event have occurred there will be a deletion in the genomic DNA and the PCR band from the desired mutant will be smaller than from the WT parent isolate (<a href="#fig4">Figure 4</a>).</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/507-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Representative gel confirming a CRISPR mediated gene knockout</div>
<p>The first four wells demonstrate a smaller band, consistent with an approximately 140 bp knockout in four mutants.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> For confirmation of SNP editing the <i>yfg_</i>SNP_Confirm_F/R primers are used. The forward primer binds within the area of homology containing the silent mutations; only binding genomic DNA of the desired mutant, or the pBBR1_CRISPR_<i>yfg_</i>SNP plasmid. The reverse primer will only bind genomic DNA and as a result only the desired mutant will produce a band with this primer set. See <a href="#fig2">Figure 2</a>C for a schematic to see relative locations of primer binding. Below are the PCR reaction conditions for both the knockout and SNP editing confirmation using standard <i>Taq</i> polymerase.</div>
<table id="undtbl10">
<thead>
<tr>
<th colspan="4">PCR cycling parameters for standard <i>Taq</i> polymerase: mutant confirmation</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">35 Cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>68°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<ol start="103">
<li>Identify the picked colonies that contain the appropriate deletion or SNP based on the PCR reaction. Streak up to 4 on non-selective LB agar and incubate at 37°C overnight, 12–18 h.</li>
<li>
<b>Days 7–9: Confirming the Mutant via Sanger Sequencing, curing the CRISPR plasmid and Stocking the Final Mutant.</b> Pick 4–8 colonies from each sub-cultured mutant and repeat the colony PCR with the <i>mgrB_</i>Del_F/R or <i>yfg_</i>SNP_Confirm_F/R primers to ensure the mutation is stable.</li>
<li>Send 1–4 PCR products for Sanger sequencing to confirm the deletion. Use the <i>mgrB_</i>Del F or <i>yfg_</i>SNP_Confirm_R primers to confirm the desired mutation. This will confirm the presence of the knockout or SNP in the genomic DNA.</li>
<li>Streak the same 1–4 picked colonies onto non-selective LB agar. Incubate at 37°C overnight, 12–18 h.</li>
<li>Pick up to 48 colonies from the plates that were sequence confirmed to run colony PCR to evaluate for loss of the CRISPR plasmid. Use the CRISPR_Vector_F and gRNA_Vector_R primers with standard <i>Taq</i> polymerase in a 25 μL reaction. Follow the NEB protocol for a 25 μL reaction (<a href="https://www.neb.com/protocols/0001/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273">https://www.neb.com/protocols/0001/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273</a>). Below are the appropriate PCR reaction conditions.</li>
</ol>
<table id="undtbl11">
<thead>
<tr>
<th colspan="4">PCR cycling parameters for standard <i>Taq</i> polymerase: plasmid cure confirmation</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">35 Cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>1:30 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>68°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> After one sub-culture onto non-selective media, loss of the CRISPR plasmid occurs in 1%–5% of picked colonies. After two rounds of sub-culturing onto non-selective media this increases to approximately 20%–50% of picked colonies. Screening 48 picked colonies will likely yield at least one colony that lacks a band with the CRISPR_Vector_F / gRNA_Vector_R primers, indicating loss of the CRISPR plasmid.</div>
<ol start="108">
<li>Select 1–2 picked colonies that lack the CRISPR plasmid and inoculate these colonies into 5 mL of LB broth. Shake at 220 RPM and 37°C overnight 12–18 h.</li>
<li>Stock the mutant in a final concentration of LB + 25% glycerol. Store at −80°C for future use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We frequently also carry out whole genome sequencing of edited isolates to confirm the deletion, true loss of the plasmid and to ensure no off-target effects have occurred. This can be performed with multiple commercial agencies including Genewiz (<a href="https://www.genewiz.com/en/Public/Services/Next-Generation-Sequencing/Whole-Genome-Sequencing">https://www.genewiz.com/en/Public/Services/Next-Generation-Sequencing/Whole-Genome-Sequencing</a>).</div>
<div class="note">
<span class="note-title">Note:</span> The resulting <i>mgrB</i> deletion mutant will be polymyxin resistant; an easy way to test the resulting phenotype is to perform a broth microdilution to polymyxin B as described below.</div>
<ol start="110">
<li>Dilute 55 mg of polymyxin B sulfate in 8.27 mL of molecular grade water. This makes a 5120 μg/mL stock solution. Vortex and separate into 500 μL aliquots.</li>
<li>Create 11 solutions in cation-adjusted Mueller-Hinton broth (CAMHB) based on the table below:</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th>Solution</th>
<th>CAMHB volume</th>
<th>Volume of solution</th>
<th>Polymyxin B concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Solution 1</td>
<td>4.5 mL</td>
<td>0.5 mL of 5120 μg/mL polymyxin B</td>
<td>512 μg/mL</td>
</tr>
<tr>
<td>Solution 2</td>
<td>2 mL</td>
<td>2 mL of solution 1</td>
<td>256 μg/mL</td>
</tr>
<tr>
<td>Solution 3</td>
<td>3 mL</td>
<td>1 mL of solution 1</td>
<td>128 μg/mL</td>
</tr>
<tr>
<td>Solution 4</td>
<td>7 mL</td>
<td>1 mL of solution 1</td>
<td>64 μg/mL</td>
</tr>
<tr>
<td>Solution 5</td>
<td>2 mL</td>
<td>2 mL of solution 4</td>
<td>32 μg/mL</td>
</tr>
<tr>
<td>Solution 6</td>
<td>3 mL</td>
<td>1 mL of solution 4</td>
<td>16 μg/mL</td>
</tr>
<tr>
<td>Solution 7</td>
<td>7 mL</td>
<td>1 mL of solution 4</td>
<td>8 μg/mL</td>
</tr>
<tr>
<td>Solution 8</td>
<td>2 mL</td>
<td>2 mL of solution 7</td>
<td>4 μg/mL</td>
</tr>
<tr>
<td>Solution 9</td>
<td>3 mL</td>
<td>1 mL of solution 7</td>
<td>2 μg/mL</td>
</tr>
<tr>
<td>Solution 10</td>
<td>7 mL</td>
<td>1 mL of solution 7</td>
<td>1 μg/mL</td>
</tr>
<tr>
<td>Solution 11</td>
<td>2 mL</td>
<td>2 mL of solution 10</td>
<td>0.5 μg/mL</td>
</tr>
</tbody>
</table>
<ol start="112">
<li>This will create enough solution for approximately 4 broth microdilution 96 well plates. Add 100 μL of the solutions to a 96 well round bottom cell culture plate based on the table below:</li>
</ol>
<table id="undtbl13">
<thead></thead>
<tbody>
<tr>
<td>Column</td>
<td>1</td>
<td>2</td>
<td>3</td>
<td>4</td>
<td>5</td>
<td>6</td>
<td>7</td>
<td>8</td>
<td>9</td>
<td>10</td>
<td>11</td>
<td>12</td>
</tr>
<tr>
<td>Solution</td>
<td>2</td>
<td>3</td>
<td>4</td>
<td>5</td>
<td>6</td>
<td>7</td>
<td>8</td>
<td>9</td>
<td>10</td>
<td>11</td>
<td>CAMHB</td>
<td>CAMHB</td>
</tr>
<tr>
<td>Polymyxin B concentration (μg/mL)</td>
<td>256</td>
<td>128</td>
<td>64</td>
<td>32</td>
<td>16</td>
<td>8</td>
<td>4</td>
<td>2</td>
<td>1</td>
<td>0.5</td>
<td>0</td>
<td>0</td>
</tr>
<tr>
<td>Final Polymyxin B concentration after bacterial dilution (μg/mL)</td>
<td>128</td>
<td>64</td>
<td>32</td>
<td>16</td>
<td>8</td>
<td>4</td>
<td>2</td>
<td>1</td>
<td>0.5</td>
<td>0.25</td>
<td>0</td>
<td>0</td>
</tr>
</tbody>
</table>
<ol start="113">
<li>Wrap plates in parafilm and store at −80°C for future use for up to 2 weeks.</li>
<li>Re-suspend bacterial isolates from picked colonies into 3 to 5 mL normal saline in a disposable culture glass tube to reach a 0.5 McFarland standard of the bacterial suspension.</li>
<li>Dilute 15 μL of this 0.5 McFarland bacterial suspension in 1485 μL of CAMHB.</li>
<li>Inoculate 100 μL of this dilution into one row of the 96 well broth microdilution plate. Each 96 well plate can test up to 8 isolates.</li>
<li>Wrap the plate in parafilm and incubate at 37°C for 16–20 h.</li>
<li>The MIC corresponds to the final polymyxin B concentration in the first well where there is no bacterial growth.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The expected polymyxin B MIC of the <i>mgrB</i> deletion mutant will be 8–32 μg/mL, with resistance defined as a MIC of &gt;2 μg/mL.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Most genetic work studying multi-drug resistant Gram-negative bacteria has been performed in antibiotic susceptible laboratory strains of <i>K. pneumonia</i> (<a href="#bib1">Cheng et al., 2016</a>, <a href="#bib2">2018</a>). However, antibiotic resistance frequently develops in a previously resistant strain (<a href="#bib4">Jeannot et al., 2017</a>). This single plasmid CRISPR-Cas9 / lambda red recombineering system allows for efficient and versatile genetic manipulation in MDR <i>K. pneumoniae.</i> Depending on the efficiency of the selected gRNA one should expect 50%–100% efficiency in obtaining the desired mutant, which is confirmed in the PCR reaction from step 102 and shown in <a href="#fig4">Figure 4</a>. The mutation will be inserted in a “scarless” manner so that only what is found on the homology cassette will be recombined into the genome. This will allow for more robust phenotypic work given minimal polar effects of the inserted mutations as well as allowing for serial rounds of editing and targeting of multiple loci.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>While zeocin is an effective antibiotic at high concentrations against most MDR <i>K. pneumoniae</i> isolates, resistance can develop <i>de novo</i> during growth of <i>K. pneumoniae.</i> As a result, there can be background colonies after transforming the CRISPR plasmid into <i>K. pneumoniae</i>. Therefore, screening the resulting colonies for presence of the plasmid is critical prior to moving forward with the knockout / SNP editing protocol.</p>
<p>The use of restriction enzymes to generate the CRISPR plasmid does pose issues at times if a desired restriction site (<i>Ahd</i>I<i>, Bsa</i>I<i>, Xba</i>I) is found within the homology region to be used. We overcome this, at times, but deleting the area of the gene that contains the restriction site to make cloning easier. Additionally, we use alternative restrictions enzymes, including <i>BamH</i>I<i>,</i> if <i>Xba</i>I cannot be used to ligate the two homology arms together. The Gibson method of cloning would also be a potential option for cloning in the gRNA and the homology cassettes.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Difficulty amplifying the homology cassette or the gRNA cassette by PCR (steps 34–43).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>In our experience almost all PCR reactions to assemble the CRISPR plasmids work well at an annealing temperature of 55°C. However, to determine the optimal annealing temperature for difficult PCR reactions we perform a gradient PCR first to optimize the annealing temperature.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Difficulty obtaining the full CRISPR plasmid after the ligation reaction with the pUC19_CRISPR vector, the gRNA cassette and the homology cassette (steps 49–61).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>For ligation reactions that yield few or no colonies we take several steps to increase the efficiency of the reaction. First, we increase the concentration of the different components. For the vector we increase from approximately 100 ng of input to 200 ng of input. The molar ratio of the gRNA and homology will increase as well. Additionally, we ensure the ligation reaction lasts for at least 16–20 h at 16°C. We also ensure inactivation of the ligation reaction at 65°C for 10 min to remove the ligase from the formed plasmid and to improve the efficiency of the transformation. Finally, if these steps do not improve the yield of the ligation reaction we pool two aliquots of chemically competent Top10 cells and add 10 μL of the ligation reaction. If these troubleshooting steps still do not yield the desired plasmid, a Gibson assembly technique could be used.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Difficulty transforming the <i>K. pneumoniae</i> isolate of interest for genetic modification (steps 84–91).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Given the noted background found with zeocin as a resistance cassette, it can sometimes be difficult to isolate <i>K. pneumoniae</i> transformants that have taken up the CRISPR plasmid. To overcome this limitation, we initially add a higher concentration of the CRISPR plasmid to the electrocompetent stock of <i>K. pneumoniae</i> (up to 500 ng in 5 μL / 50 μL aliquot). If there are resulting colonies, but none of the selected colonies show uptake of the CRISPR plasmid via colony PCR we increase the concentration of zeocin in the transformation plates. For standard transformation reactions into carbapenem resistant ST258 <i>K. pneumoniae</i> we use a zeocin concentration of 1,000 μg/mL in LB, which can be increased to 1,250, 1,500 or even 2,000 μg/mL.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The induced culture grows as quickly as the control culture and no mutants are found after screening the resulting colonies (steps 96–103).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This may indicate overgrowth of a sub-population that developed zeocin resistance <i>in vitro</i> and subsequently lost the CRISPR plasmid due to the potential negative fitness effects of plasmid replication. This may also indicate inappropriate selection of a transformant that did not actually contain the CRISPR plasmid. We repeat the transformation reaction and identify additional <i>K. pneumoniae</i> colonies that have taken up the CRISPR plasmid and repeat the knockout protocol on these new colonies. If this troubleshooting does not work, it may indicate that the selected gRNA has little to no efficiency at inducing the double-strand break with the Cas9 protein. This occurs in approximately 5%–10% of our selected gRNA sequences. This appears to occur randomly and is not predictable by any of the aforementioned parameters for selecting appropriate N20 sequences. We then move forward with generating a second gRNA CRISPR plasmid. It is highly unlikely to select two gRNA sequences that are both non-functional.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>There is differential growth of the induced cultures, and significantly fewer colonies on the induced plate, but all screened colonies are still wild type (steps 96–103).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>This indicates that the gRNA and Cas9 combination are inducing the double-strand break, but the homologous recombination is occurring with minimal efficiency. This may be due to characteristics of the selected homology versus the timing of arabinose induction. We find that lengthening the induction time with arabinose from 6–8 h to at least 12 h, and at times up to 24–36 h, can help increase the number of resulting colonies that contain the desired mutation.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cheng, Y.H., Lin, T.L., Lin, Y.T., and Wang, J.T. (2016). Amino acid substitutions of CrrB responsible for resistance to colistin through CrrC in <i>Klebsiella pneumoniae</i>. Antimicrob. Agents Chemother. <i>60</i>, 3709-3716. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref1">View at publisher</a></p>
<p id="bib2">Cheng, Y.H., Lin, T.L., Lin, Y.T., and Wang, J.T. (2018). A putative RND-type efflux pump, H239_3064, contributes to colistin resistance through CrrB in <i>Klebsiella pneumoniae</i>. J. Antimicrob. Chemother. <i>73</i>, 1509-1516. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref2">View at publisher</a></p>
<p id="bib3">El-Halfawy, O.M., Klett, J., Ingram, R.J., Loutet, S.A., Murphy, M.E., Martin-Santamaria, S., and Valvano, M.A. (2017). Antibiotic capture by bacterial lipocalins uncovers an extracellular mechanism of intrinsic antibiotic resistance. MBio <i>8</i>, e00225-17. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref3">View at publisher</a></p>
<p id="bib4">Jeannot, K., Bolard, A., and Plesiat, P. (2017). Resistance to polymyxins in Gram-negative organisms. Int. J. Antimicrob. Agents <i>49</i>, 526-535. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref4">View at publisher</a></p>
<p id="bib5">Jiang, W., Bikard, D., Cox, D., Zhang, F., and Marraffini, L.A. (2013). RNA-guided editing of bacterial genomes using CRISPR-Cas systems. Nat. Biotechnol. <i>31</i>, 233-239. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref5">View at publisher</a></p>
<p id="bib6">Jiang, X., Zhu, C., Lin, J., Li, J., Fu, S., and Gong, H. (2016). Vector promoters used in <i>Klebsiella pneumoniae</i>. Biotechnol. Appl. Biochem. <i>63</i>, 734-739. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref6">View at publisher</a></p>
<p id="bib7">Jiang, Y., Chen, B., Duan, C., Sun, B., Yang, J., and Yang, S. (2015). Multigene editing in the <i>Escherichia coli</i>genome via the CRISPR-Cas9 system. Appl. Environ. Microbiol. <i>81</i>, 2506-2514. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref7">View at publisher</a></p>
<p id="bib8">Kovach, M.E., Elzer, P.H., Hill, D.S., Robertson, G.T., Farris, M.A., Roop, R.M., and Peterson, K.M. (1995). Four new derivatives of the broad-host-range cloning vector pBBR1MCS, carrying different antibiotic-resistance cassettes. Gene <i>166</i>, 175-176. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref8">View at publisher</a></p>
<p id="bib9">McConville, T.H., Annavajhala, M.K., Giddins, M.J., Macesic, N., Herrera, C.M., Rozenberg, F.D., Bhushan, G.L., Ahn, D., Mancia, F., Trent, M.S., et al. (2020). CrrB positively regulates high-level polymyxin resistance and virulence in <i>Klebsiella pneumoniae</i>. Cell Rep. <i>33</i>, 108313. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref9">View at publisher</a></p>
<p id="bib10">Zhao, D., Yuan, S., Xiong, B., Sun, H., Ye, L., Li, J., Zhang, X., and Bi, C. (2016). Development of a fast and easy method for <i>Escherichia coli</i>genome editing with CRISPR/Cas9. Microb. Cell Fact. <i>15</i>, 205. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00080-0/sref10">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Additional information and inquiries for additional reagents and resources are best directed to and will be handled by Dr. Anne-Catrin Uhlemann, <a href="mailto:Au2110@columbia.edu">au2110@columbia.edu</a></p>
<h4>Materials Availability</h4>
<p>The pUC19_CRISPR_<i>ΔpmrA</i> plasmid necessary for this protocol is available at Addgene: catalog #160903.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate or analyze any new data sets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was funded through <a href="https://doi.org/10.13039/100000002">NIH</a>/<a href="https://doi.org/10.13039/100000060">NIAID</a> T32 AI100852-04 and K08 AI146284 (to T.H.M.), NIH/NIAID R01 AI116939, and NIH/NIDDK U54 DK104309 (to A.-C.U.)</p>
<h3>Author Contributions</h3>
<p>T.H.M., M.J.G., and A.C.U. devised the initial CRISPR plasmid design. T.H.M. and M.J.G. completed the initial cloning to create the initial CRISPR plasmid. T.H.M. optimized the CRISPR knockout and SNP editing protocol.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>