<article>
<section>
<h1 id="header">A protocol to quantify chromatin compaction with confocal and super-resolution microscopy in cultured cells</h1>
<p><time datetime="2021-10-01">Published: October 1, 2021</time></p>
<p>Laura Martin,<sup><a href="#aff1">1</a></sup> Chiara Vicario,<sup><a href="#aff1">1</a></sup> Álvaro Castells-García,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Melike Lakadamyali,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a></sup> Maria Victoria Neguembor,<sup><a href="#aff1">1</a>,<a href="#fn1">8</a>,<a href="#cor1">*</a></sup> and Maria Pia Cosma<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#aff7">7</a>,<a href="#fn2">9</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, 08003 Barcelona, Spain</p>
<p id="aff2"><sup>2</sup>Bioland Laboratory, Guangzhou Regenerative Medicine and Health Guangdong Laboratory, Guangzhou 510005, China</p>
<p id="aff3"><sup>3</sup>Perelman School of Medicine, Department of Physiology, University of Pennsylvania, Philadelphia, PA 19104, USA</p>
<p id="aff4"><sup>4</sup>Perelman School of Medicine, Department of Cell and Developmental Biology, University of Pennsylvania, Philadelphia, PA 19104, USA</p>
<p id="aff5"><sup>5</sup>Universitat Pompeu Fabra (UPF), Dr Aiguader 88, 08003 Barcelona, Spain</p>
<p id="aff6"><sup>6</sup>ICREA, Pg. Lluís Companys 23, 08010 Barcelona, Spain</p>
<p id="aff7"><sup>7</sup>CAS Key Laboratory of Regenerative Biology, Guangdong Provincial Key Laboratory of Stem Cell and Regenerative Medicine, Guangzhou Institutes of Biomedicine and Health, Chinese Academy of Sciences, Guangzhou 510530, China</p>
<p id="fn1"><sup>8</sup>Technical contact</p>
<p id="fn2"><sup>9</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:victoire.neguembor@crg.es">victoire.neguembor@crg.es</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:pia.cosma@crg.es">pia.cosma@crg.es</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100865">10.1016/j.xpro.2021.100865</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we describe three complementary microscopy-based approaches to quantify morphological changes of chromatin organization in cultured adherent cells: the analysis of the coefficient of variation of DNA, the measurement of DNA-free nuclear areas, and the quantification of chromatin-associated proteins at the nuclear edge. These approaches rely on confocal imaging and stochastic optical reconstruction microscopy and allow a fast and robust quantification of chromatin compaction. These approaches circumvent inter-variability between imaging conditions and apply to every type of adherent cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.molcel.2021.06.009">Neguembor et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for three complementary techniques to measure chromatin compaction</li>
<li>Chromatin compaction is quantified from confocal and super-resolution images</li>
<li>CV and Black Space analyses score for heterogeneity of nuclear DNA distribution</li>
<li>Edge analysis measures the enrichment of a protein signal around the nuclear edge</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the steps for the confocal- and Stochastic Optical Reconstruction Microscopy (STORM)-based quantifications of morphological changes in chromatin organization in the nucleus of HeLa cells treated with Actinomycin D, an inhibitor of transcription whose action drives major changes in nuclear topology (<a href="#bib4">Neguembor et al., 2021</a>). In addition, we have also successfully used this protocol with other drugs (as α-amanitin, β-Lapachone, which are a transcription inhibitor and a torsional stress inhibitor, respectively) and in different cellular systems (mAID-mClover-POLR2A DLD-1 cells, in which transcriptional inhibition is obtained by degrading the endogenous RNA polymerase II) (<a href="#bib4">Neguembor et al., 2021</a>).</p>
<p>Specifically, the protocol offers a means to (1) quantify the coefficient of variation of DAPI signal; (2) quantify the percentage of the nuclear DNA-free areas; and (3) quantify the percentage of H3K9me3 spatially localizing in the nuclear edge with respect to the total nuclear signal.</p>
<p>Coefficient of variation is a method to assess the degree of heterogeneity of DNA signal across the nucleus. It has been previously used to quantify changes in DNA compaction upon drug treatments or genetic modifications affecting chromatin organization (<a href="#bib1">Grezy et al., 2016</a>; <a href="#bib3">Jeanblanc et al., 2012</a>; <a href="#bib4">Neguembor et al., 2021</a>). The coefficient of variation (CV) of individual nuclei is calculated as <math> <mrow> <mo>=</mo> <mfrac> <mi>σ</mi> <mi>μ</mi> </mfrac> </mrow> </math> , where σ represents the standard deviation of the DAPI intensity values and μ represents the mean value of intensity of the nucleus.</p>
<p>DNA-free area analysis is an alternative method to assess DNA compaction previously reported in <a href="#bib4">Neguembor et al. (2021)</a>. Upon DNA compaction, DNA signal becomes redistributed and accumulates in densely packed areas. This drives the appearance of areas with no DNA signal or low-density DNA signal (i.e., DNA free areas). DNA-free nuclear areas are quantified from DNA STORM images by applying a binary threshold based on signal intensity on Gaussian filtered density maps. Regions with DNA signal below the threshold are considered DNA-free areas and the percentage of DNA-free areas over the imaged nuclear area is estimated for each nucleus.</p>
<p>Nuclear edge analysis is a method to quantify the enrichment of specific proteins close to the nuclear rim with respect to the protein signal across the entire nuclear section. The method was previously reported in <a href="#bib4">Neguembor et al. (2021)</a> to assess the redistribution of H3K9me3 and Lamin A/C signals in STORM images. Here, the percentage of H3K9me3 STORM localizations at the edge of nucleus is calculated by first defining an edge region around the border of each nucleus of a width chosen by the user and then calculating the percentage of localizations falling in this area, i.e., the number of localizations inside the edge region divided by the total number of localizations inside the nucleus.</p>
<p>We expect this protocol to be suitable for quantitatively assessing the changes in spatial localization and density of any nuclear protein of interest other than DNA or H3K9me3 with respect to a control condition, provided that such proteins are suitable for confocal or STORM imaging, and that variable parameters of analysis are adjusted and optimized according to the particular experiment.</p>
<p>Solutions are prepared following the recipes in the <a href="#materials-and-equipment">materials and equipment</a> section. Solutions that can be prepared in advance and stored are indicated. A complete list of materials and equipment required is given in the <a href="#key-resources-table">key resources table</a>.</p>
<h3 id="sec1.1">Installation of software and custom codes for analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol>
<li>Download and install (Fiji Is Just) ImageJ (FIJI) (<a href="#bib5">Rueden et al., 2017</a>) from ImageJ website: download the relevant .zip file depending on the operating system and run installation executable. The codes illustrated in this protocol were tested in ImageJ 2.0.0 and 2.1.0/1.53c versions.
<ol type="a">
<li>Upload ImageJ script CV PART 1 IMAGEJ within ChromatinCompactionAnalysis.zip (see <a href="#key-resources-table">key resources table</a>).</li>
<li>Extract the .zip file in the folder of your choice.</li>
</ol>
</li>
<li>Download and install MATLAB from <a href="https://www.mathworks.com/products/matlab.html">https://www.mathworks.com/products/matlab.html</a>. The codes illustrated in this protocol were tested in MATLAB R2021a and older versions, such as R2016a, R2016b versions.
<ol type="a">
<li>Upload script packages <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc1.extension">Data S1</a>.zip, <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc2.extension">Data S2</a>.zip, and <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc3.extension">Data S3</a>.zip (see <a href="#key-resources-table">key resources table</a>).</li>
<li>Extract the .zip file in the folder of your choice.</li>
<li>In the home section of MATLAB, hit ‘‘Set Path’’.</li>
<li>Select ‘‘Add with Subfolders…’’ and browse to the folder containing the extracted files.</li>
<li>Hit ‘‘Save’’. MATLAB can now find and run all the scripts necessary for the analyses.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A digest version of the image analysis steps of this protocol can also be found in the Videos as STEP BY STEP simplified tutorial videos (<a href="#mmc4">Methods videos S1</a>, <a href="#mmc5">S2</a>, and <a href="#mmc6">S3</a>). The protocol does not require an extensive knowledge of MATLAB. However, we recommend looking at the tutorial available at Matlab website, particularly the section about setting up paths and running scripts. We ran this analysis successfully on laptop computers (both Windows and Mac) with <i>ad minima</i> a 1.80 GHz Processor and 16 GB of RAM. Higher specifications can lead to reduced times of analysis.</div>
<figure id="mmc4"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc4.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Step by step tutorial for CV analysis, related to steps 14–17</div>
</figcaption>
</figure>
<figure id="mmc5"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc5.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. Step by step tutorial for DNA-free regions analysis, related to step 18</div>
</figcaption>
</figure>
<figure id="mmc6"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc6.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S3. Step by step tutorial for Edge analysis, related to step 19</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-H3K9me3 (dilution 1:50)</td>
<td>Abcam</td>
<td>Cat#ab8898; RRID: AB_306848</td>
</tr>
<tr>
<td>Goat anti-Rabbit IgG AF647 (dilution 1:250)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A-21244</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMSO</td>
<td>Sigma-Aldrich</td>
<td>Cat#D2650</td>
</tr>
<tr>
<td>Actinomycin D (ActD)</td>
<td>Sigma-Aldrich</td>
<td>Cat#A9415</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle Medium (DMEM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#41965039</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10270106</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (Pen Strep)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#35050038</td>
</tr>
<tr>
<td>Dulbecco’s Phosphate Buffered Saline (DPBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#14190094</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.25%), phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#25200056</td>
</tr>
<tr>
<td>HEPES</td>
<td>Sigma-Aldrich</td>
<td>Cat#H4034</td>
</tr>
<tr>
<td>Hydrochloric acid fuming 37% (HCl)</td>
<td>Millipore</td>
<td>Cat#100317</td>
</tr>
<tr>
<td>L-Ascorbic acid</td>
<td>Sigma-Aldrich</td>
<td>Cat#A92902</td>
</tr>
<tr>
<td>Amino guanidine</td>
<td>Sigma-Aldrich</td>
<td>Cat#396494</td>
</tr>
<tr>
<td>Glucose oxidase</td>
<td>Sigma-Aldrich</td>
<td>Cat#G2133</td>
</tr>
<tr>
<td>Catalase</td>
<td>Sigma-Aldrich</td>
<td>Cat#C100</td>
</tr>
<tr>
<td>Cysteamine (MEA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#30070</td>
</tr>
<tr>
<td>CuSO<sub>4</sub>
</td>
<td>Sigma-Aldrich</td>
<td>Cat#C1297</td>
</tr>
<tr>
<td>Glucose</td>
<td>Sigma-Aldrich</td>
<td>Cat#G8270</td>
</tr>
<tr>
<td>5-Ethynyl-2′-deoxycytidine (EdC)</td>
<td>Sigma-Aldrich</td>
<td>Cat#T511307</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#A7906</td>
</tr>
<tr>
<td>Paraformaldehyde 16% methanol-free (PFA)</td>
<td>Alfa Aesar</td>
<td>Cat#43368</td>
</tr>
<tr>
<td>4′,6-Diamidino-2-phenylindole (DAPI)</td>
<td>Roche</td>
<td>Cat#10236276</td>
</tr>
<tr>
<td>Alexa Fluor™ 647 Azide, Triethylammonium Salt (AF647)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A10277</td>
</tr>
<tr>
<td>Drierite</td>
<td>Sigma-Aldrich</td>
<td>Cat#238961</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Images files (Mendeley data)</td>
<td>This study</td>
<td>DOI: <a href="https://doi.org/10.17632/pbnc4zhcwd.1">https://doi.org/10.17632/pbnc4zhcwd.1</a>
</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HeLa cells (human cervix epithelioid carcinoma, female)</td>
<td>Maria Pia Cosma Lab</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ (2.0.0 and 2.1.0/1.53c)</td>
<td>(<a href="#bib5">Rueden et al., 2017</a>)</td>
<td><a href="https://imagej.nih.gov/ij/download.html">https://imagej.nih.gov/ij/download.html</a></td>
</tr>
<tr>
<td>MATLAB</td>
<td>MathWorks</td>
<td><a href="https://www.mathworks.com/products/matlab.html">https://www.mathworks.com/products/matlab.html</a></td>
</tr>
<tr>
<td>
<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc1.extension">Data S1</a>. CV analysis, related to steps 14–17, containing CV-Part1-ImageJ (steps 14–16) and CV-Part2-MATLAB (step 17)</td>
<td>This paper</td>
<td>
<a href="https://github.com/CosmaLab/CV-Analysis">https://github.com/CosmaLab/CV-Analysis</a>All processing scripts can be found also in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc1.extension">Data S1</a>.</td>
</tr>
<tr>
<td>
<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc2.extension">Data S2</a>. Black Space analysis, related to step 18, used for the quantification of DNA-free areas</td>
<td>This paper</td>
<td>
<a href="https://github.com/CosmaLab/Black-Space-Analysis">https://github.com/CosmaLab/Black-Space-Analysis</a>All processing scripts can be found also in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc2.extension">Data S2</a>.</td>
</tr>
<tr>
<td>
<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc3.extension">Data S3</a>. Edge analysis, related to step 19, used for the quantification of the percentage of H3K9me3 signal at nuclear edges</td>
<td>This paper</td>
<td>
<a href="https://github.com/CosmaLab/Edge-Analysis">https://github.com/CosmaLab/Edge-Analysis</a>All processing scripts can be found also in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc3.extension">Data S3</a>.</td>
</tr>
<tr>
<td>Step-by-step simplified tutorial videos.mp4, a digest version of the protocol</td>
<td>This paper</td>
<td>STEP BY STEP simplified tutorial videos.mp4</td>
</tr>
<tr>
<td>LAS X</td>
<td>Leica</td>
<td><a href="https://www.leica-microsystems.com/products/microscope-software/details/product/leica-las-x-ls/">https://www.leica-microsystems.com/products/microscope-software/details/product/leica-las-x-ls/</a></td>
</tr>
<tr>
<td>NIS software STORM module</td>
<td>Nikon</td>
<td><a href="https://www.microscope.healthcare.nikon.com/products/software/nis-elements">https://www.microscope.healthcare.nikon.com/products/software/nis-elements</a></td>
</tr>
<tr>
<td>Graphpad Prism</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td>Insight3</td>
<td>(<a href="#bib2">Huang et al., 2008</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Borosilicate glass bottom 8-well chambers (Lab-Tek)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#155411</td>
</tr>
<tr>
<td>N-STORM 4.0 microscope equipped with highly inclined and laminated optical sheet illumination (HILO)</td>
<td>Nikon</td>
<td>N/A</td>
</tr>
<tr>
<td>LU-NV laser unit with 405 nm (LD, 20 mW output), 488 nm (DPSS, 45 mW output), 560 nm (DPSS, 45 mW output), and 647 nm (fiber, 125 mW output) lasers</td>
<td>Nikon</td>
<td>N/A</td>
</tr>
<tr>
<td>405/488/561/647nm Laser Quad Band Set</td>
<td>Chroma</td>
<td>Cat#TRF89902</td>
</tr>
<tr>
<td>CFI HP Apochromat TIRF 100XC Oil (NA 1.49) objective</td>
<td>Nikon</td>
<td>N/A</td>
</tr>
<tr>
<td>iXON Ultra DU-897U EMCCD camera</td>
<td>Andor Technology</td>
<td>N/A</td>
</tr>
<tr>
<td>Leica TCS SP8 confocal microscopy equipped with laser/filter sets to acquire DAPI signal</td>
<td>Leica</td>
<td>N/A</td>
</tr>
<tr>
<td>HC Plan-Apochromat CS2 63x/1.40 oil objective</td>
<td>Leica</td>
<td>N/A</td>
</tr>
<tr>
<td>Type F Immersion liquid (RI 1.518)</td>
<td>Leica</td>
<td>Cat#11944399</td>
</tr>
<tr>
<td>Immersion oil (Type F) (RI 1.515)</td>
<td>Nikon</td>
<td>Cat#MXA22168</td>
</tr>
<tr>
<td>Stericup Quick Release Millipore Express PLUS 0.22 μm PES, 500 mL</td>
<td>Millipore</td>
<td>Cat#11290</td>
</tr>
<tr>
<td>Telstar™ Soporte de mesa para Bio II Advance 4</td>
<td>Telstar</td>
<td>Cat#15138614</td>
</tr>
<tr>
<td>BINDER CO<sub>2</sub> incubator CB 160, with O<sub>2</sub> control</td>
<td>Binder</td>
<td>Cat# 9040-0092</td>
</tr>
<tr>
<td>WHEATON® Dry-Seal vacuum desiccator</td>
<td>Sigma-Aldrich</td>
<td>Cat#Z114375</td>
</tr>
<tr>
<td>Sartorius Stedim Biotech SI-215D Analytical Balance</td>
<td>Denver Instrument</td>
<td>Cat#EW-11422-04</td>
</tr>
<tr>
<td>Fisherbrand™ accumet™ AE150 Benchtop pH Meter</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#13-636-AE152</td>
</tr>
<tr>
<td>Millipore Milli-Q Synthesis A10 Water Purification</td>
<td>Millipore</td>
<td>Cat#15882</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Media for HeLa cell culture</h3>
<table id="undtbl1">
<caption>HeLa’s standard cell culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dulbecco’s Modified Eagle Medium (DMEM)</td>
<td>88%</td>
<td>44 mL</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>HeLa’s resting medium to induce cell cycle arrest (optional)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dulbecco’s Modified Eagle Medium (DMEM)</td>
<td>96%</td>
<td>48 mL</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>2%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>1%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Hela’s culture media can be stored at 4°C for 1 month.</div>
<div class="note">
<span class="note-title">Note:</span> Optimal medium composition can vary considerably for different cell types.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend filtering FBS under the hood through Stericup 0.22 μm prior to use.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Penicillin/Streptomycin may cause an allergic skin reaction (H317), may cause allergy or asthma symptoms or breathing difficulties if inhaled (H334), and may damage fertility or the unborn child (H360), and hence it should be manipulated by wearing appropriate recommended PPE.</div>
<h3 id="sec2.2">EdC solution</h3>
<p>Ethynil-deoxy-cytidine (EdC) (#T511307, Sigma-Aldrich) is resuspended in DMSO to a concentration of 10 mM and aliquots can be stored at −20°C for 1 year.</p>
<h3 id="sec2.3">ActD stock solution</h3>
<p>Actinomycin D (ActD) (#A9415, Sigma-Aldrich) is resuspended in DMSO to a concentration of 5 mM and aliquots can be stored foil-wrapped vials at −20°C for up to 1 month. Do not store in glass vials.</p>
<div class="critical">
<span class="critical-title">Critical:</span> ActD is fatal if swallowed (H300), may cause cancer (H350), and may damage fertility or the unborn child (H360), and hence it should be manipulated by wearing appropriate recommended PPE.</div>
<h3 id="sec2.4">Solutions for microscopy</h3>
<h4 id="sec2.4.1">Fixative solution</h4>
<p>Fixative solution is prepared by diluting PFA 16% in PBS to obtain a final solution of PFA 4%.</p>
<div class="note">
<span class="note-title">Note:</span> PFA 4% is prepared freshly the same day of the fixation. PFA 16% can be stored at 4°C for 3 months.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Paraformaldehyde is harmful in contact with skin (H312), harmful if inhaled (H332), causes skin irritation (H315), may cause an allergic skin reaction (H317), causes serious eye irritation (H319), may cause respiratory irritation (H335), is suspected of causing genetic defects (H341), and may cause cancer (H350), and hence it should be manipulated by wearing appropriate recommended PPE.</div>
<h4 id="sec2.4.2">Permeabilization solution</h4>
<p>Permeabilization solution is prepared by diluting TritonX-100 in PBS to obtain a final solution of 0.4%.</p>
<div class="note">
<span class="note-title">Note:</span> TritonX-100 0.4 % stock solution is stable at 20°C–25°C for 1 month.</div>
<div class="critical">
<span class="critical-title">Critical:</span> TritonX-100 is classified as harmful if swallowed (H302), causes skin irritation (H315), causes serious eye damage (H318), and hence should be manipulated by wearing appropriate recommended PPE.</div>
<table id="undtbl3">
<caption>Blocking buffer for Immuno-Fluorescence (IF) and STORM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>10%</td>
<td>1 g</td>
</tr>
<tr>
<td>Triton-X 100 0.4 %</td>
<td>0.01%</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>9.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Blocking solution is prepared freshly before use.</div>
<table id="undtbl4">
<caption>Washing buffer for IF and STORM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>2%</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Triton-X 100 0.4 %</td>
<td>0.01%</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>9.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Washing buffer is prepared freshly before use.</div>
<h3 id="sec2.5">DAPI</h3>
<p>DAPI stock is prepared by dissolving DAPI in ultrapure water to 1 mg/mL.</p>
<div class="note">
<span class="note-title">Note:</span> Stock solution is stable for 6 months and repeated use if stored protected from light at −20°C.</div>
<table id="undtbl5">
<caption>Glox solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glucose Oxidase</td>
<td>0.1%</td>
<td>14 mg</td>
</tr>
<tr>
<td>Catalase</td>
<td>
<code>∼</code>200–500 U/μL</td>
<td>50 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>200 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<code>∼</code>250 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Glucose Oxidase is stored at −20°C and it is stable for 5 years.</div>
<div class="note">
<span class="note-title">Note:</span> Catalase is prepared as specified in the Product Data Sheet (Sigma-Aldrich) and the aliquots can be stored at 4°C for 12 months.</div>
<div class="note">
<span class="note-title">Note:</span> Thawed Glox aliquots can be stored at 4°C for 1 week, do not re-freeze.</div>
<p>Preparation:</p>
<ul>
<li>Weigh 14 mg of Glucose Oxidase.</li>
<li>Dissolve in 200 μL of PBS. Gently vortex in order to dissolve.</li>
<li>Add 50 μL of Catalase 1 mg/mL.</li>
<li>Centrifuge at 20 000 × <i>g</i> for 2 min in order to pellet insolubilities.</li>
<li>Prepare 200 μL eppendorf tubes. Pipet the supernatant in aliquots of <code>∼</code>20 μL.</li>
<li>Store at −20°C or use immediately.</li>
</ul>
<table id="undtbl6">
<caption>Click Chemistry buffer solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES 152 mM pH 8.2</td>
<td>n/a</td>
<td>1492 μL</td>
</tr>
<tr>
<td>CuSO<sub>4</sub> solution (100 mM stock)</td>
<td>1 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Amino Guanidin (500 mM stock)</td>
<td>50 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>Ascorbic acid (250 mM)</td>
<td>25 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>AlexaFluor647-azide (10 mM stock)</td>
<td>10 μM</td>
<td>2 μL</td>
</tr>
<tr>
<td>Glucose 50%</td>
<td>2%</td>
<td>80 μL</td>
</tr>
<tr>
<td>Glox solution (1000X)</td>
<td>0.1%</td>
<td>6 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> HEPES 152 mM pH 8.2, CuSO<sub>4</sub> 100 mM solution, and Glucose 50% solution can be prepared in advance and can be stored at 20°C–25°C. Amino Guanidin 500 mM solution, AlexaFluor647-azide and Glox can be prepared in advance and should be stored for 3 months at −20°C in the dark. Ascorbic acid solution should be prepared freshly before use.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare HEPES 152 mM pH 8.2 stock solution, HEPES is resuspended in MilliQ water to a concentration of 152 mM, pH is adjusted to pH 8.2 with HCl using a pH-meter, solution is brought to final desired volume with milliQ water. Stock solution can be prepared in advance and can be stored for 1 year at 20°C–25°C.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare CuSO<sub>4</sub> 100 mM stock solution, CuSO<sub>4</sub> is resuspended in MilliQ water to a concentration of 100 mM. Stock solution can be prepared in advance and can be stored for 1 year at 20°C–25°C, protected from light.</div>
<div class="note">
<span class="note-title">Note:</span> Glucose 50% stock solution is prepared by resuspending Glucose in MilliQ water to a concentration of 50% (w/v). Stock solution can be prepared in advance and can be stored for 1 year at 20°C–25°C.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare Amino Guanidin 500 mM stock solution, Amino Guanidin is resuspended in MilliQ water to a concentration of 500 mM. Stock solution can be prepared in advance and can be stored in aliquots for 1 year at −20°C.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare Ascorbic acid solution, Ascorbic acid is resuspended in MilliQ water to a concentration of 44 mg/mL. Solution should be prepared freshly before use and remaining solution should be discarded.</div>
<div class="note">
<span class="note-title">Note:</span> To prepare AlexaFluor647-azide 10 mM stock solution, Alexa Fluor647-azide is resuspended in DMSO to a concentration of 10 mM. Stock solution can be prepared in advance and can be stored in aliquots for 1 year at −20°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Click Chemistry buffer solution should be prepared fresh 15 minutes before performing the reaction. Mix the reagents by pipetting carefully and let stabilize for some minutes until the color of the solution slightly changes to dark blue.</div>
<h3 id="sec2.6">MEA 1M solution</h3>
<p>MEA 1M is prepared by dissolving 77 mg of MEA in 1 mL of HCl 360 mM.</p>
<div class="note">
<span class="note-title">Note:</span> MEA powder is aliquoted in eppendorfs containing 77 mg each. Eppendorfs should be wrapped in parafilm and stored with Drierite under the vacuum desiccator, at 4°C for 6 months.</div>
<div class="note">
<span class="note-title">Note:</span> MEA solution can be stored at 4°C for 1 month.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If MEA aliquots show signs of humidity they should be thrown away and the status of both the desiccator and the drierite stones should be check.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Drierite may cause an allergic skin reaction (H317), may cause allergy or asthma symptoms or breathing difficulties if inhaled (H334), and is toxic to aquatic life with long lasting effects (H411), and hence it should be manipulated by wearing appropriate recommended PPE.</div>
<div class="critical">
<span class="critical-title">Critical:</span> HCl may be corrosive to metals (H290), causes severe skin burns and eye damage (H314), may cause respiratory irritation (H335), and hence it should be manipulated by wearing appropriate recommended PPE.</div>
<table id="undtbl7">
<caption>Imaging buffer for STORM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>237 μL</td>
</tr>
<tr>
<td>Glucose 50%</td>
<td>5%</td>
<td>30 μL</td>
</tr>
<tr>
<td>MEA 1M solution</td>
<td>100 mM</td>
<td>30 μL</td>
</tr>
<tr>
<td>Glox solution (100×)</td>
<td>1%</td>
<td>3 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>300 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Imaging buffer should be prepared freshly right before use using PBS at 20°C–25°C.</div>
<div class="note">
<span class="note-title">Note:</span> Always keep MEA and Glox solution on ice during imaging sessions.</div>
<div class="note">
<span class="note-title">Note:</span> Glucose 50% can be stored at 20°C–25°C or 4°C for 1 year.</div>
<div class="note">
<span class="note-title">Note:</span> MEA solution can be stored at 4°C for 1 month.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Reagents listed for Click chemistry reaction can be substituted with the components of the commercial kit Click-iT EdU Cell proliferation kit for imaging Alexa Fluor 647 dye (Cat#C10340). However, we recommend the use of EdC instead of EdU.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>We begin by describing how to culture and treat HeLa cells with Actinomycin D. We then illustrate the protocols for staining DNA with DAPI for confocal microscopy, for labeling DNA through Click Chemistry for STORM, and for immunostaining of H3K9me3 for STORM. Next, we describe the protocol for imaging acquisition, both for confocal imaging and STORM imaging. Finally, we explain the workflows for the analysis and quantification of acquired images through customized codes for ImageJ and MatLab.</p>
<p>For a global overview of the step-by-step workflow see <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Diagram of the workflow for chromatin compaction quantification</div>
</figcaption>
</figure>
<h3 id="sec3.1">Cell plating, synchronization, and treatment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<div class="critical">
<span class="critical-title">Critical:</span> Cell culture needs to be performed entirely under a Class II Biological Safety Cabinet to avoid contamination.</div>
<div class="note">
<span class="note-title">Note:</span> Pre-warming the complete cell culture medium and Trypsin-EDTA is recommended to avoid exposing culture cells to excessive thermal shift and minimizing detachment time, respectively.</div>
<p>Cells are seeded in 8-well chambers and allowed to attach for 24 h before challenging them with drug treatments or other experimental conditions that may induce changes in the degree of chromatin compaction.</p>
<ol>
<li>Detach HeLa cells from maintenance cultures in T75 flasks or p10 dishes.
<ol type="a">
<li>Discard exhausted culture medium.</li>
<li>Wash gently with 5 mL of PBS, and discard.</li>
<li>Add 1 mL Trypsin-EDTA (0.25%), phenol red to each flask.</li>
<li>Incubate flasks for 3 min at 37°C in cell incubator.
<div class="note">
<span class="note-title">Note:</span> As detachment time can vary considerably with cell type, the plate should be visually inspected for complete cell detachment before proceeding.</div>
<div class="note">
<span class="note-title">Note:</span> Tapping gently on the sides of the flask/dish can favour the detachment.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Trypsin without Phenol Red/Versene can be employed as an alternative agent for cell detachment.</div>
</li>
<li>Add 9 mL of complete culture medium and pipette gently up and down do disaggregate cells.</li>
<li>Collect detached cells in 15 mL Falcon tubes.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>HeLa cells counting.
<ol type="a">
<li>Count cells with a standard or automated cell counter.</li>
<li>Prepare a cell suspension to achieve 1 × 10<sup>5</sup> cells/mL.
<ol type="i">
<li>Put in a 15 mL Falcon the volume of cell suspension containing 800 000 cells.</li>
<li>Dilute it with complete culture medium until reaching the volume of 8 mL.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The suggested protocol is sufficient to plate 3 chamberslides at the density of 37 500 cells/cm<sup>2</sup>. In the case of using different cell lines and biological treatments, we recommend scaling up/down the cell density as well as the volumes for culturing cells into higher/lower number of wells. Particular attention should be taken in modifying density plating, since this protocol requires cells to be in a monolayer when imaged and superimposition of nuclei would compromise the analysis.</div>
<div class="note">
<span class="note-title">Note:</span> To perform DAPI confocal imaging or H3K9me3 STORM imaging proceed to step 3, then to steps 5a and 6a. To perform EdC incorporation for DNA STORM imaging proceed directly to step 4, then to steps 5b and 6b.</div>
<ol start="3">
<li>HeLa seeding for DAPI confocal imaging or H3K9me3 STORM imaging
<ol type="a">
<li>Seed 350 μL of this suspension in 8-well chambers and return plates to the incubator.</li>
</ol>
</li>
<li>HeLa seeding for DNA STORM imaging
<ol type="a">
<li>Add 4 μL of EdC 10 mM solution to the 8 mL of cells suspension (1:2000 dilution, final concentration 5 μM).</li>
<li>Seed 350 μL of this suspension in 8-well chambers and return plates to the incubator.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> For steps 3 and 4, Borosilicate glass bottom 8-well chambers (Ibidi, #80827) can be used alternatively to Borosilicate glass bottom 8-well chambers (Lab-Tek, #155411), but volumes per well should be adjusted to maximum 250 μL to avoid spill-over. Adjust concentration of cell suspension accordingly.</div>
<ol start="5">
<li>Serum-starvation of HeLa cells
<ol type="a">
<li>After 24 h from seeding, replace culture medium in all wells with 350 μL of pre-warmed resting medium for DAPI confocal imaging or H3K9me3 STORM imaging.</li>
<li>For wells designated to DNA STORM imaging, replace culture medium with 350 μL of pre-warmed resting medium plus EdC in the dilution of 1:2000, as the day before.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Step 5 can be skipped if cell cycle arrest is not required for your experimental design.</div>
<ol start="6">
<li>HeLa cells treatment with ActD (5 h)
<ol type="a">
<li>After 43 h from seeding, replace resting medium in all wells with 350 μL of fresh resting medium plus ActD at 1:1000 dilution (final concentration 5 μg/mL).</li>
<li>For wells designated to DNA STORM imaging, also add EdC in the dilution of 1:2000, as the days before.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Different treatment lengths and concentrations or alternative drug treatments can be performed at this stage. Adjust timing and drug concentration accordingly.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If you wish to compare experimental conditions other than drug treatments (for example, genetically modified cell lines), steps 5 and 6 can be skipped.</div>
<h3 id="sec3.2">Cell fixation, permeabilization, and staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>After 5 h of ActD incubation, or corresponding alternative treatment, HeLa cells are subjected to fixation, permeabilization and labeling for IF microscopy-based assessment of DNA compaction and STORM microscopy-based assessment of DNA-free areas and H3K9me3 redistribution.</p>
<div class="note">
<span class="note-title">Note:</span> Optimal fixation and permeabilization conditions may vary with cell type.</div>
<div class="note">
<span class="note-title">Note:</span> Fixative solution should be at 20°C–25°C before being added to the cells, as cell fixation should be carried out at 20°C–25°C.</div>
<ol start="7">
<li>Fixation.
<ol type="a">
<li>Discard medium.</li>
<li>Wash 3 × 5 minutes with PBS.</li>
<li>Fix cells with 300 μL of PFA 4% for 10 min at 20°C–25°C.</li>
<li>Wash 3 × 5 minutes with PBS.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Fixed cells can be stored (before staining) for up to 1 month at 4°C, provided that chamber lids are covered with 400 μL PBS and that plates are sealed with parafilm to avoid PBS evaporation. In this case, wells should be visually inspected to ensure normal cellular morphology and absence of contamination before resuming the protocol. Also, bring cells to 20°C–25°C before proceeding with permeabilization.</div>
<ol start="8">
<li>Permeabilization.
<ol type="a">
<li>Permeabilize with Triton 0.4% for 15 min.</li>
<li>Wash 3 × 5 minutes with PBS.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For Click-chemistry reaction for DNA STORM imaging of EdC labelled cells proceed to step 9, then to step 13 (STORM imaging).</div>
<div class="note">
<span class="note-title">Note:</span> For immunostaining of H3K9me3 and STORM imaging, proceed to step 10, then to step 13 (STORM imaging).</div>
<div class="note">
<span class="note-title">Note:</span> For DAPI labeling for confocal imaging proceed directly to step 11, then to step 12 (confocal imaging).</div>
<ol start="9">
<li>Click Chemistry reaction for DNA STORM imaging.
<ol type="a">
<li>Incubate cells with Click Chemistry solution for 30 min protecting samples from light.</li>
<li>Wash with PBS three times for 5 min each protecting samples from light.</li>
<li>Proceed directly with STORM imaging (step 13) or store at 4°C until imaging.</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Immunostaining for H3K9me3 STORM imaging.
<ol type="a">
<li>Incubate wells with blocking buffer for 1 h at 20°C–25°C.</li>
<li>Incubate wells with Rabbit polyclonal anti-H3K9me3 diluted in blocking buffer 1:50 for 12–16 h at 4°C.</li>
<li>Wash with Washing buffer three times for 5 min each.</li>
<li>Incubate cells with 1:250 Goat anti-Rabbit IgG AF647 in blocking buffer for 1 h at 20°C–25°C in the dark.
<div class="note">
<span class="note-title">Note:</span> From now on, keep the chamberslides in the dark to avoid bleaching of the AF647 fluorophore.</div>
</li>
<li>Wash secondary antibody with Washing buffer twice for 5 min each.</li>
<li>Wash once in PBS.</li>
<li>Proceed directly with STORM imaging (step 13) or store at 4°C until imaging.</li>
</ol>
</li>
</ol>
<ol start="11">
<li>DAPI staining for DNA confocal imaging
<ol type="a">
<li>Incubate wells with Blocking buffer plus DAPI diluted 1:1000 in PBS (final concentration 1 μg/mL) for 15 min at 20°C–25°C in the dark.</li>
<li>Wash three times with PBS.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Hoechst 33342 at a 1:1000 dilution can be employed as an alternative agent for DNA staining instead of DAPI. Also, DNA labeled with EdC-AF647 can be imaged with confocal imaging.</div>
<p>Slight changes in the concentration of the dye (in the range of 1:500 -1:2000) are not crucial for the positive outcome of the protocol as far as the adjusted concentration is kept constant across all conditions and biological replicates.</p>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Stained cells can be stored for up to 2 days at 4°C in the dark, provided that chamber lids are covered with 400 μL PBS and that plates are sealed with parafilm to avoid PBS evaporation. In this case, wells should be visually inspected to ensure normal cellular morphology and absence of contamination before resuming the protocol. Also, cells should be inspected for abnormalities in cell density and in fluorescent signal and intensity before proceeding with imaging (see <a href="#sec7.1">troubleshooting 1</a>).</div>
<h3 id="sec3.3">Image acquisition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h (Confocal) to 2 days (STORM)</div>
<p>Stained chambers are mounted onto the microscopes (Confocal and STORM respectively) and images are acquired.</p>
<ol start="12">
<li>Confocal microscopy imaging.
<ol type="a">
<li>Set parameters of image acquisition for the corresponding image size and zoom. We suggest to use an image size of 1024 × 1024, pinhole 1, scan speed 400 Hz, bidirectional scan, and a zoom of 1, in order to have around 50 nuclei per field at a high resolution.</li>
<li>Adjust laser power and gain for the DAPI channel to satisfy the dynamic range of the experiment by imaging at least 10 fields per condition. We normally use a laser power in a range between 10%–20% and a gain between 300–600V, and we suggest adjusting these parameters if needed, as far as they are kept constant across the whole experiment.</li>
<li>Visually inspect the objective for proper state. Place one drop of Type F oil (Leica) (RI 1.518) in the center of the objective lens.</li>
<li>Quickly but carefully, mount the chamber on the stage-holder.</li>
<li>Adjust the focus on the central stack of the cells and proceed with image acquisition. Repeat the step for at least other 9 fields per condition, in order to have enough nuclei to analyze.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Laser power and gain should be adjusted to prevent DAPI signal saturation even in the brightest condition.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Good quality confocal images are critical for further analysis. Thus, prior training of confocal imaging and microscope setup are required. Specifically, images are considered of good quality when nuclei are in focus, signal is well detectable in all nuclei but not saturated, and the field is illuminated homogeneously.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Images can be stored indefinitely prior to analysis.</div>
<ol start="13">
<li>STORM imaging
<ol type="a">
<li>Visually inspect the objective for proper state. Place one drop of Type F oil (Nikon) (RI 1.515) in the center of the objective lens.</li>
<li>Quickly but carefully, mount the chamber on the stage-holder.</li>
<li>Replace PBS with freshly prepared STORM imaging buffer in the well to image.</li>
<li>Set camera to 17 MHz, EM Gain 100V and image size to 256 × 256 pixels.</li>
<li>Detect the AF647 fluorescence in the sample by using a laser compatible with AF647 fluorophore excitation (peak at 647 nm) and starting at a low power coupled with a 100 ms exposure, to avoid blinking and sample bleaching. Adjust focus and illumination angle for optimal signal on the central stack of the cells.</li>
<li>Take a single snapshot of the nucleus/nuclei to be acquired. It can be used for further detection of nuclear structures (e.g., nucleoli), as well as for assaying the intensity of fluorescence prior to single-molecule acquisition.</li>
<li>Set the camera exposure time at 10 ms and increase the 647 nm laser power to 2 kW/cm<sup>2</sup> approx., to visualize single molecule blinking.</li>
<li>Launch image acquisition with simultaneous illumination of the sample with 647 nm and 405 nm lasers. Specifically, 647 nm laser was used at constant ∼2 kW/cm<sup>2</sup> power density and 405 nm laser power was gradually increased over the imaging, from 0.005 kW/cm<sup>2</sup> to 0.5 kW/cm<sup>2</sup>. Typically, 60 000 frames were acquired and at least 10 fields were imaged per condition.</li>
<li>Generate localization lists from STORM images and perform drift correction with Insight3 (<a href="#bib2">Huang et al., 2008</a>) or Nikon NIS elements by selecting the command “Automatic drift correction”.<br> In Insight3, select “Save Molecule List”, and a .bin file will be generated.<br> To generate .bin files in NIS elements, go to Input &amp; Output in the Molecule analysis section. Select Export and choose “Export to BIN file (M425)”.</li>
<li>To generate .bin files of individual selected nuclei in Nikon NIS elements go to Molecule analysis, Turn ON ROI button, open ROI Menu and select “Draw Bezier ROI…”. Check “Autofit ROI” option. Click “Process All Frames”. Export .bin file as described in point i).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend the use of 405-nm laser activation during acquisition to maintain a good rate of single molecule blinking per frame throughout the entire image acquisition (approx. 30–50 localizations per frame). Nevertheless, EdC-AF647 signal typically does not require 405 nm laser activation during acquisition as enough single molecules are detected per frame.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Discard STORM imaging buffer every hour by aspiration with a pipette and replace it with freshly prepared buffer as buffer acidifies with time. Imaging buffer should be added at 20°C–25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Replace STORM imaging buffer with PBS and store samples in the dark at 4°C when not performing imaging.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Good quality STORM images are critical for further analysis. Thus, prior training of STORM imaging, image processing and STORM microscope setup are required. More in details, STORM images are considered of good quality when nuclei are in focus, density of localizations is consistent among images belonging to the same experimental condition, there is no significant background noise, and the field is illuminated homogeneously.</div>
<div class="critical">
<span class="critical-title">Critical:</span> DNA-free areas and Edge analyses require .bin files as input. In case localization lists are generated with a software different from Insight3 or NIS elements, export data in .bin format and verify that the structure of the file is compatible with downstream analyses.</div>
<h3 id="sec3.4">Coefficient of variation analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>For step-by step visual guide of the protocol please refer to <a href="#mmc4">Methods videos S1</a>.</p>
<p>Single-channel confocal microscopy images are processed for the assessment of chromatin compaction through the measurement of the coefficient of variation.</p>
<div class="note">
<span class="note-title">Note:</span> (Fiji Is Just) ImageJ imaging software is employed to perform this automated analysis.</div>
<div class="note">
<span class="note-title">Note:</span> Before to begin, create one folder containing the .tiff files to be analyzed and the script FIJI_MACRO_CV.txt, contained in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc1.extension">Data S1</a>.</div>
<p>Output files will be saved automatically in the same folder. The name of the output file will include both the name of the source image and the index of the selected ROI.</p>
<div class="note">
<span class="note-title">Note:</span> We also suggest generating a shortcut for the command “Run” to make the process of analysis even faster. To do this, select from (Fiji Is Just) ImageJ menu: &gt; Plugins &gt; Shortcuts &gt; Add Shortcut…, and associate the Command “Run…” to the Shortcut of your preference.</div>
<p>In that case, step 14b will be substituted by “Click your personalized Shortcut for the command “Run…”.</p>
<ol start="14">
<li>Identification of single nuclei and generation of cropped single nuclei .tiff files.
<ol type="a">
<li>Open one single-channel .tiff image.</li>
<li>Plugins &gt; Macros &gt; Run…</li>
<li>Run FIJI_MACRO_CV.txt.</li>
</ol>
</li>
<li>Check generated masks if they correctly match single nuclei.
<ol type="a">
<li>Visually inspect if masks generated are correct.</li>
<li>If yes, &gt; Close all.</li>
<li>If not, in FIJI_MACRO_CV.txt change the minimum and maximum threshold values set to generate the masks:
<ol type="i">
<li>Open the .txt.</li>
<li>In line 10,
<ul>
<li>
<code>&gt; //setThreshold(12, 255)</code>;</li>
<li>the former parameter (i.e., “12”) is the threshold of the minimum intensity value detected by the mask, while the latter (i.e., “255”) is the threshold of the maximum intensity one. Change value “12” with customized values between 0 and 255 adapted to your particular condition. Note that lowering the minimum intensity value (i.e., &lt;”12”) would allow the detection of weaker signal, while increasing it (i.e., &gt;”12”) would increase the stringency. The maximum intensity value should be kept as “255”, to include the broadest range of intensity in the image.</li>
</ul>
</li>
<li>Save modified .txt.</li>
<li>Re-analyze images from step 14.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Although this code has been successfully tested on several different conditions, we are aware that the density of cell seeding and the quality of labeling can both affect the recognition of nuclei performed by FIJI_MACRO_CV.txt script. For this reason, we recommend modifying the ROI analyzed (see <a href="#sec7.1">troubleshooting 1</a>) and the command “Watershed” if needed (see <a href="#sec7.3">troubleshooting 2</a>).</div>
<div class="note">
<span class="note-title">Note:</span> If after applying <a href="#sec7.3">troubleshooting 2</a> and <a href="#sec7.5">3</a> some nuclei are still incorrectly identified, these can be discarded easily in step 15 by checking the mask numbers associated to them and eliminating the correspondent .tiff files in the folder.</div>
<ol start="16">
<li>Run step 14 and step 15 for every image you want to analyze.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend analyzing at least 5 images per condition for a total of at least 100 nuclei per condition, and multiple biological replicates (at least 3).</div>
<ol start="17">
<li>Calculation of the Coefficient of Variation.
<ol type="a">
<li>Open MatLab software.</li>
<li>Open the MATLAB_CV_Confocal.m script, contained in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc1.extension">Data S1</a>.</li>
<li>Change the following parameters according to your experiment:
<ol type="i">
<li>
<code>&gt;categ =</code>; with the number of experimental conditions.</li>
<li>
<code>&gt;Folder =</code> ; with the path of the data folder given as a string.</li>
<li>
<code>&gt;categname =</code> ; with the names of the experimental categories given in cell format.</li>
<li>
<code>&gt;colors =[]</code> ; with a 3-columns matrix with the color code for each category in each subsequent row. You can use predefined functions as<code>&gt;colors = colormap(jet());</code>to define color series.</li>
</ol>
</li>
<li>Click Run.</li>
<li>Load the single nuclei .tiff files generated in step 15. One different window will open for each category.</li>
<li>Click Done.</li>
<li>Two pop-up windows with graphic representations of the results will appear. <a href="#fig1">Figure 1</a> represents the distribution of pixel intensities of single cells (“CV_Histplot_individualcells”) (see <a href="#mmc5">Methods videos S1</a>, 00:02:34), while <a href="#fig2">Figure 2</a> plots the mean and standard deviation of the distribution of pixel intensities for each category (“CV_Histplot_smooth”) (see <a href="#mmc4">Methods videos S1</a>, 00:02:36).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic view of the code to crop images in batch (left) of the corresponding subROI selection (right)</div>
</figcaption>
</figure>
</li>
<li>In the Command Window, the line
<ul>
<li>
<code>&gt;Do Boxplot (Y/N)&gt;</code>will appear.</li>
</ul>
<br> Write “Y” (Yes) to additionally visualize the boxplot of the CV calculated (“Coefficient_Variation_Boxplot”) or “<code>N</code>” (No) to skip the visualization, and press Enter. All the .fig visualized, plus the .xls output file generated and the CV workspace are automatically saved in your folder of choice defined in step 17) c) ii).</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">DNA-free area analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>For step-by step visual guide of the protocol please refer to <a href="#mmc5">Methods videos S2</a>.</p>
<p>Single-color STORM microscopy images of EdC-AF647 labeled DNA are processed for the assessment of chromatin compaction through the measurement of the percentage of DNA free areas. The same procedure can be applied for STORM images of core histones or histone modifications.</p>
<ol start="18">
<li>Calculation of the percentage of DNA-free areas
<ol type="a">
<li>Open MatLab software.</li>
<li>Open Main code: “BlackSpace_MAIN”, contained in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc2.extension">Data S2</a>.</li>
<li>Change the following parameters according to your experiment:
<ol type="i">
<li>
<code>&gt;categ =</code>; with the number of experimental conditions.</li>
<li>
<code>&gt;Directorio =</code> ; with the path of the data folder given as a string.</li>
<li>
<code>&gt;categname =</code> ; with the names of the experimental categories given in cell format.</li>
<li>Check and adjust to image dimension by defining<br> <code>&gt;maxX = ;</code><br> <code>&gt;maxY = ;</code><br> values (e.g., 256 or 512 pixels).</li>
<li>
<code>&gt;h = fspecial('gaussian',[5 5], 2);</code> (line 44) with “2” being the value of Gaussian sigma recommended. Adjust it if needed depending on your dataset.<br> Increasing the value will increase the level of blurring of the image from which the masks of DNA and nucleus are generated.</li>
<li>
<code>&gt;MaskDNA = imbinarize(Dens,'adaptive','Sensitivity',0.001);</code> (line 48) with “0.001” being the value of adaptive threshold recommended. Adjust it if needed depending on your dataset.</li>
</ol>
</li>
<li>Click Run.</li>
<li>Load .bin files for each experimental category.</li>
<li>Click Done.</li>
<li>One pop-up window with graphic representations of the results will appear for each analyzed file. Each figure is a panel displaying the density rendering of the nucleus, the DNA mask and the nuclear mask generated.</li>
<li>Calculated values of percentage of DNA-free areas are automatically saved as the .xls file called “Blackspace_results.xlsx” in your folder of choice defined in step 18) c) ii).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Although this code has been successfully tested on several different conditions, we are aware that the quality of labeling and alternative DNA distributions across cell types can affect the generation of nuclear masks and the identification of DNA-free areas performed by this script. For this reason, we recommend to visually verify the generated masks and modify the adaptive threshold value, pixel size and Gaussian sigma if needed (see <a href="#sec7.5">troubleshooting 3</a>).</div>
<div class="note">
<span class="note-title">Note:</span> We recommend analyzing at least 10 images per condition.</div>
<h3 id="sec3.6">Edge analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>For step-by step visual guide of the protocol please refer to <a href="#mmc6">Methods videos S3</a>.</p>
<p>Single-color STORM microscopy images of H3K9me3 are processed for the assessment of chromatin compaction through the measurement of the percentage of localizations found at the nuclear edge versus the total number of nuclear localizations. The same procedure can be applied to STORM images of any nuclear staining (for example: core histones, lamins, DNA) to quantify differences in signal enrichment at nuclear edge.</p>
<ol start="19">
<li>Calculation of the percentage of localizations at nuclear edge and nuclear interior
<ol type="a">
<li>Open MatLab software.</li>
<li>Open Main script: "edgeLocDensity_main.m", contained in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc3.extension">Data S3</a>.</li>
<li>Change the following parameters according to your experiment:
<ol type="i">
<li>
<code>&gt;categ =</code>; with the number of experimental conditions.</li>
<li>
<code>&gt;Folder =</code> ; with the path of the data folder given as a string.</li>
<li>
<code>&gt;CategLabels =</code>; with the names of the experimental categories given in cell format.</li>
<li>
<code>&gt;colors =[]</code>; with a 3-columns matrix with the color code for each category in each subsequent row. You can use predefined functions as<br> <code>&gt;colors = colormap(jet());</code>to define color series.</li>
<li>
<code>&gt;SR_px = ;</code> with the size of super resolved pixels. We recommend starting with the value “20” and change it if needed.</li>
<li>
<code>&gt;NPx_edge = ;</code> with the width in pixels of the edge mask desired. We recommend starting with the value “50”and adjust it if needed depending on your dataset. Note that very small values could cause a partial loss of the enrichment while too high values would cause the inclusion of also the inner part of the nucleus, not-being specific for the edge area anymore. Both conditions would cause an error of underestimation of the percentage of enrichment.</li>
<li>
<code>&gt;nmperpx = ;</code> with the original pixel size in nm. Adjust this value (e.g., “160”) depending on the microscope model.</li>
<li>
<code>&gt;sigma = ;</code> with the desired Gaussian Sigma value. We recommend starting with the value “1.5”. This parameter regulates the degree of blurring of the signal from which the nuclear mask is generated (the lower the value, the higher the stringency). If the signal of the protein labeled is not present across the whole nuclear area, we suggest to increase the value of blurring (lowering the stringency) in order to avoid the generation of artifactual holes.</li>
</ol>
</li>
<li>Click Run.</li>
<li>Load .bin files for each experimental category.</li>
<li>Two pop-up windows with graphic representations of the results will appear for each analyzed file. <a href="#fig1">Figure 1</a> is the super resolution image of the nucleus, while <a href="#fig2">Figure 2</a> is a panel including the nuclear mask, the edge mask, and their overlay.</li>
<li>In the Command Window, the line<br> <code>&gt;Do Density rendering (Y/N)&gt;</code>will appear.<br> Write “ Y” (Yes) to visualize additionally the density renderings or “<code>N</code>” (No) to skip the visualization and press Enter.</li>
<li>Calculated values of percentage of localizations at nuclear edge and nuclear interior, as well as average and median density values, are automatically saved in a single file called ‘EDGE_analysis_results.xlsx’ in your folder of choice defined in step 19) c) ii).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Although this code has been successfully tested on several different conditions, we are aware that the quality of labeling can affect the generation of nuclear masks performed by this script. For this reason, we recommend to visually verify the generated masks and change the Gaussian sigma if needed (see <a href="#sec7.7">troubleshooting 4</a>).</div>
<div class="note">
<span class="note-title">Note:</span> We recommend analyzing at least 10 images per condition.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Confocal and STORM images of cells exposed to transcription inhibition treatment processed as described above reveal the compaction of chromatin preferentially around the nuclear and nucleolar rims.</p>
<p>See Figures S2B, S2D, S2F and S2I in <a href="#bib4">Neguembor et al. (2021)</a>, for comparison of coefficient of variation analysis between control and treated cells. See Figures 2B and 6C in <a href="#bib4">Neguembor et al. (2021)</a>, for comparison of DNA-free areas between control and treated cells. See Figures 4D and 4F in <a href="#bib4">Neguembor et al. (2021)</a>, for both comparison of percentage of Lamin A/C and of H3K9me3 at the nuclear rim between control and WAPL mutant conditions.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Quantitative results obtained upon image analysis as detailed above are processed with GraphPad Prism software as follows.</p>
<p>Coefficient of variation data generated through the MATLAB_CV_Confocal.m script were automatically saved as .xls file (step 17). The .xls was imported in a new GraphPad Prism file and plotted as column scatter plot with mean and standard deviation per conditions.</p>
<p>DNA-free areas data generated through the BlackSpace_MAIN.m script were automatically saved as .xls file (step 18). The .xls was imported in a new GraphPad Prism file and plotted as column scatter plot with mean and standard deviation per conditions.</p>
<p>Edge quantification data generated through the edgeLocDensity_main.m script were automatically saved as .xls file (step 19). The .xls was imported in a new GraphPad Prism file and plotted as column scatter plot with mean and standard deviation per conditions.</p>
<p>For all the above-mentioned quantification approaches, statistical analysis has been performed in Graphpad Prism. For every dataset, normality tests (D’Agostino-Pearson and Shapiro-Wilk) were run to assess normal distribution. For datasets with Gaussian distribution of values, parametric tests were applied. In all cases, two-tailed tests were run and multiple comparison corrections were applied for datasets with more than two groups and multiple comparisons. For datasets with no Gaussian distribution, non-parametric tests were applied. Statistical significance was represented in the following manner: ns p &gt; 0.05, ∗ p ≤ 0.05, ∗∗ p ≤ 0.01, ∗∗∗ p ≤ 0.001, ∗∗∗∗ p ≤ 0.0001.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Our protocol has been carried out only on adherent cells. Non-adherent cells pose a challenge for imaging because of their constant movements in the medium, even when sedimented. That can be addressed by coating the support with components allowing immobilization of cells. Since, EdC incorporation requires DNA replication, Click chemistry labeling can be performed only on replicating cells. If cell cycle arrest is desired, perform EdC incorporation before cell cycle arrest treatment.</p>
<p>Moreover, the staining of cells described in the protocol requires a step of permeabilization; thus, it is not compatible with live imaging. Nevertheless, in the case of coefficient of variation analysis this limitation can be overcome by skipping the step of fixation and performing nuclear staining with Hoechst instead of DAPI.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Not homogeneous cell density (steps 2, 3, and 4):</p>
<p>Homogeneous cell seeding and attachment is critical to ensure robust conditions for labeling, imaging and downstream analysis. Cell clumps and overlaps should be avoided as well as cell detachment from the well.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>To ensure homogeneous cell seeding, it is important to disaggregate cell clumps by pipetting well and to prepare cell dilutions at the final concentration desired for plating, as described in the protocol, instead of plating cells at higher concentrations and adjusting volumes with medium in the wells. Make sure the cells are homogeneously seeded by gentle agitation of the chamberslide before placing them in the incubator. Avoid circular movements that tend to bring cells to the center of the well. Steps of pipetting and aspiration should be done carefully to avoid cell detachment. We recommend inclining the wells and placing the tip at a bottom corner of the wells.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Not homogeneous illumination (step 15):</p>
<p>The fields of confocal imaging could present regions more illuminated than others. Even though our method of analysis is independent on absolute values of pixel intensities belonging to distinct nuclei, the CV quantification can be affected if one single nucleus shows inhomogeneous illumination.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>A simple solution is to analyze a subROI with homogeneous illumination.</p>
<p>The subROI can be either drawn by hand in individual files or a series of images displaying the same problem can be cropped systematically in batch.</p>
<p>The former method is suggested for single cases and requires the use of the tool Draw.</p>
<p>The latter is preferred when analyzing in batch and is performed by modifying the FIJI_MACRO_CV.txt file as following:</p>
<p>- The commands</p>
<p><code>&gt;makeRectangle(x<sub>A</sub>, y<sub>A</sub>, width, height);</code></p>
<p><code>&gt;run(“Crop”);</code></p>
<p>should be added as line 8 and 9, respectively.</p>
<p>The brackets of <code>&gt;makeRectangle(xA, yA, width, height);</code> should contain the pixel coordinates of the upper left (x<sub>A</sub>, y<sub>A</sub>) vertex and the width and height of the rectangular subROI (see <a href="#fig2">Figure 2</a>).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Incorrect nuclei masks generation (step 15):</p>
<p>In rare cases, it can occur that one single nucleus is recognized as two different ones (<a href="#fig3">Figure 3</a>). This occurs most frequently in ROIs with cells at very low confluence or with very irregular shapes.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Example of nuclei and relative masks displaying erroneous nuclear segmentation</div>
</figcaption>
</figure>
<h3 id="sec7.6">Potential solution</h3>
<p>This problem can be circumvented by deleting the line 15</p>
<p><code>&gt;run("Watershed");</code></p>
<p>from the script FIJI_MACRO_CV.txt and running this modified script only on the ROI displaying the erroneous nuclei division. Alternatively, if this problem is very infrequent, individual masks presenting problems of segmentation can be discarded from the analysis (see step 15, note 2).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Incorrect nuclear mask generation and associated wrong recognition of DNA-free areas (step 18):</p>
<p>In rare cases, insufficient nuclear labeling or peculiar DNA distributions can lead to incorrect generation of nuclear masks with parts of nuclei being excluded from the analysis.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>When setting up the analysis, visually inspect the nuclear masks generated for the different cells. If masks are suboptimal, modify the adaptive threshold value and Gaussian sigma until nuclear area and DNA-free areas are correctly segmented. We suggest starting with values of 0.001 for adaptive threshold and 2 for Gaussian sigma, and adjust them if needed.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Incorrect nuclear mask generation and associated wrong recognition of nuclear edges (step 19):</p>
<h3 id="sec7.10">Potential solution</h3>
<p>When setting up the analysis, visually inspect the nuclear masks generated for the different cells. If masks are suboptimal, modify the Gaussian sigma until nuclear areas are correctly segmented. We recommend starting with a Gaussian sigma value of 1.5, and adjust it if needed. We suggest to increase the value of Gaussian sigma if the signal is not homogeneously distributed across the nuclear area, since in this way the increase in blurring and the decrease in stringency would help in generating a correct nuclear mask.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Grezy, A., Chevillard-Briet, M., Trouche, D., and Escaffit, F. (2016). Control of genetic stability by a new heterochromatin compaction pathway involving the Tip60 histone acetyltransferase. Mol. Biol. Cell <i>27</i>, 599-607. <a class="external-link" href="https://doi.org/10.1091/mbc.E15-05-0316">https://doi.org/10.1091/mbc.E15-05-0316</a></p>
<p id="bib2">Huang, B., Wang, W., Bates, M., and Zhuang, X. (2008). Three-dimensional super-resolution imaging by stochastic optical reconstruction microscopy. Science <i>319</i>, 810-813. <a class="external-link" href="https://doi.org/10.1126/science.1153529">https://doi.org/10.1126/science.1153529</a></p>
<p id="bib3">Jeanblanc, M., Ragu, S., Gey, C., Contrepois, K., Courbeyrette, R., Thuret, J.Y., and Mann, C. (2012). Parallel pathways in RAF-induced senescence and conditions for its reversion. Oncogene <i>31</i>, 3072-3085. <a class="external-link" href="https://doi.org/10.1038/onc.2011.481">https://doi.org/10.1038/onc.2011.481</a></p>
<p id="bib4">Neguembor, M.V., Martin, L., Castells-Garcia, A., Gomez-Garcia, P.A., Vicario, C., Carnevali, D., AlHaj Abed, J., Granados, A., Sebastian-Perez, R., Sottile, F., et al. (2021). Transcription-mediated supercoiling regulates genome folding and loop formation. Mol. Cell <i>81</i>, 3065-3081.e12. <a class="external-link" href="https://doi.org/10.1016/j.molcel.2021.06.009">https://doi.org/10.1016/j.molcel.2021.06.009</a></p>
<p id="bib5">Rueden, C.T., Schindelin, J., Hiner, M.C., DeZonia, B.E., Walter, A.E., Arena, E.T., and Eliceiri, K.W. (2017). ImageJ2: ImageJ for the next generation of scientific image data. BMC Bioinformatics <i>18</i>, 529. <a class="external-link" href="https://doi.org/10.1186/s12859-017-1934-z">https://doi.org/10.1186/s12859-017-1934-z</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Maria Pia Cosma (<a href="mailto:pia.cosma@crg.es">pia.cosma@crg.es</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The codes generated during this study are available at Github (<a href="https://github.com/CosmaLab/">https://github.com/CosmaLab/</a>) and are provided as <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc1.extension">Data S1</a>, <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc2.extension">S2</a>, <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc3.extension">S3</a>.</p>
<p>Original/source data for figures in the paper are related to Neguembor et al. and available at Mendeley Data: <a href="https://doi.org/10.17632/pbnc4zhcwd.1">https://doi.org/10.17632/pbnc4zhcwd.1</a>.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc1.extension">Data S1. CV analysis, related to steps 14–17</a></p>
<p><a id="mmc2" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc2.extension">Data S2. Black Space analysis, related to step 18</a></p>
<p><a id="mmc3" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1051-Mmc3.extension">Data S3. Edge analysis, related to step 19</a></p>
<h3>Acknowledgments</h3>
<p>The authors acknowledge the support from European Union's <a href="https://doi.org/10.13039/501100007601">Horizon 2020</a> Research and Innovation Programme (CellViewer no. 686637 to M.P.C. and M.L.); <a href="https://doi.org/10.13039/501100004837">Ministerio de Ciencia e Innovación</a>, grant (BFU2017-86760-P [AEI/FEDER, UE] to M.P.C.), and an AGAUR grant from Secretaria d’Universitats i Recerca del Departament d’Empresa i Coneixement de la Generalitat de Catalunya ([2017 SGR 689] to M.P.C.); Centro de Excelencia Severo Ochoa (2013–2017 to M.P.C.); CERCA Programme/Generalitat de Catalunya (to M.P.C.); the <a href="https://doi.org/10.13039/501100004837">Spanish Ministry of Science and Innovation</a> to the EMBL partnership (to M.P.C.); <a href="https://doi.org/10.13039/501100001809">National Natural Science Foundation of China</a> (31971177 to M.P.C.); Innovative Team Program of Guangzhou Regenerative Medicine and Health Guangdong Laboratory (2018GZR110103001 to M.P.C.); People Program (Marie Curie Actions) FP7/2007–2013 under REA grant 608959 (to M.V.N.); Juan de la Cierva-Incorporación 2017 (to M.V.N.); grant for the recruitment of early-stage research staff FI-2020 (Operational Program of Catalonia 2014-2020 CCI 2014ES05SFOP007 of the <a href="https://doi.org/10.13039/501100004895">European Social Fund</a> to L.M.); "La Caixa" Foundation Fellowship (ID 100010434, #LCF/BQ/DR20/11790016) (to L.M.); and "La Caixa-Severo Ochoa" pre-doctoral fellowship (to A.C.-G.). The authors acknowledge the Advanced Light Microscopy Unit from CRG for their excellent technical support.</p>
<h3>Author contributions</h3>
<p>Methodology, M.V.N., L.M., and C.V.; software, L.M., C.V., and A.C.-G.; formal analysis, M.V.N. and L.M.; investigation, M.V.N., L.M., and A.C.-G; visualization, M.V.N. and L.M.; writing – original draft, L.M., M.V.N., and M.P.C.; writing – review &amp; editing, C.V., A.C.-G., and M.L.; funding acquisition, M.P.C. and M.L.; supervision, M.V.N. and M.P.C.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>