<article>
<section>
<h1 id="header">Quantitative analysis of m<sup>6</sup>A RNA modification by LC-MS</h1>
<p><time datetime="2021-08-06">Published: August 6, 2021</time></p>
<p>Lavina Mathur,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Sunhee Jung,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Cholsoon Jang,<sup><a href="#aff2">2</a>,<a href="#cor1">*</a></sup> and Gina Lee<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Microbiology and Molecular Genetics, Chao Family Comprehensive Cancer Center, University of California Irvine School of Medicine, Irvine, CA 92697-4025, USA</p>
<p id="aff2"><sup>2</sup>Department of Biological Chemistry, Chao Family Comprehensive Cancer Center, University of California Irvine School of Medicine, Irvine, CA 92697-4025, USA</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:choljang@uci.edu">choljang@uci.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:ginalee@uci.edu">ginalee@uci.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100724">10.1016/j.xpro.2021.100724</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p><i>N</i><sup>6</sup>-adenosine methylation (m<sup>6</sup>A) of messenger RNA (mRNA) plays key regulatory roles in gene expression. Accurate measurement of m<sup>6</sup>A levels is thus critical to understand its dynamic changes in various biological settings. Here, we provide a protocol to quantitate the levels of adenosine and m<sup>6</sup>A in cellular mRNAs. Using nuclease and phosphatase, we digest mRNA into nucleosides, which are subsequently quantified using liquid chromatography mass spectrometry.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib2">Cho et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Straightforward protocol for quantitation of m<sup>6</sup>A levels in cellular mRNA</li>
<li>Purified mRNA is processed to nucleosides by nuclease and phosphatase</li>
<li>Modified and unmodified nucleoside levels are measured by LC-MS</li>
<li>The protocol can be applied to quantitate various chemical modifications in RNA and DNA</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/907-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol requires preparation of several buffers and enzymes beforehand. To prevent degradation of RNA samples, it is necessary to follow general precautions for RNA experiments including preparation of RNase-free plastic wares and wiping working surfaces with RNase inactivating agents. To avoid contamination of buffers with RNase, we recommend that the users purchase RNase-free buffers (list provided in the <a href="#key-resources-table">key resources table</a>). Using these raw materials, prepare working solutions and enzyme mixtures as described in the Materials and Equipment Section. The quality of mass spectrometry reagents (e.g., organic solvents, water) is also critical to reduce contamination of RNA and nucleotides from external sources.</p>
<h3 id="sec1.1">HEK293E cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>The protocol was used to measure m<sup>6</sup>A levels in HEK293E cell line but can be adapted for any cells and tissue samples. Prepare enough number of cells and tissues to isolate &gt;50 μg total RNA.</p>
<ol>
<li>Seed around 350,000 cells in a 60 mm cell plate.</li>
<li>After two days, harvest the cells for total RNA isolation.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, buffers and enzymes</td>
</tr>
<tr>
<td>
<i>N</i><sup>6</sup>-methyladenosine (m<sup>6</sup>A)</td>
<td>Selleckchem</td>
<td>Cat#S3190</td>
</tr>
<tr>
<td>Adenosine</td>
<td>Sigma-Aldrich</td>
<td>Cat#A9251</td>
</tr>
<tr>
<td>10× Phosphate Buffered Saline (PBS)</td>
<td>Corning</td>
<td>Cat#46-013-CM</td>
</tr>
<tr>
<td>EDTA 0.5 M (pH8.0)</td>
<td>Promega</td>
<td>Cat#V4231</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma-Aldrich</td>
<td>Cat#G5516-100ML</td>
</tr>
<tr>
<td>LiCl 8 M</td>
<td>Sigma-Aldrich</td>
<td>Cat#L7026-100ML</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> 25 mM</td>
<td>New England Biolabs (NEB)</td>
<td>Cat#B9021S</td>
</tr>
<tr>
<td>ZnCl<sub>2</sub> 0.1M</td>
<td>Sigma-Aldrich</td>
<td>Cat#39059-1ML-F</td>
</tr>
<tr>
<td>NaCl 5M</td>
<td>Quality Biological Inc</td>
<td>Cat#351-036-721EA</td>
</tr>
<tr>
<td>Ammonium Bicarbonate</td>
<td>Sigma-Aldrich</td>
<td>Cat#09830</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>HyClone</td>
<td>Cat#SH30538.FS</td>
</tr>
<tr>
<td>200 Proof Ethanol</td>
<td>Sigma-Aldrich</td>
<td>Cat#459836</td>
</tr>
<tr>
<td>RNase decontamination solution</td>
<td>Genesee Scientific</td>
<td>Cat#10-456</td>
</tr>
<tr>
<td>PCR grade water</td>
<td>IBI Scientific</td>
<td>Cat#IB42301</td>
</tr>
<tr>
<td>Sodium acetate 3 M (pH5.2)</td>
<td>Corning</td>
<td>Cat#46-033-CI</td>
</tr>
<tr>
<td>Tris Hydrochloride (Tris-HCl) 1 M (pH7.5)</td>
<td>Fisher Scientific</td>
<td>Cat#BP1757-500</td>
</tr>
<tr>
<td>Beta-Mercaptoethanol</td>
<td>Sigma-Aldrich</td>
<td>Cat#M3148-25ML</td>
</tr>
<tr>
<td>Hydrochloric Acid (HCl) 6 M</td>
<td>Fisher Scientific</td>
<td>Cat# S25857</td>
</tr>
<tr>
<td>Nuclease P1 from <i>Penicillium citrinum</i>
</td>
<td>Sigma-Aldrich</td>
<td>Cat#N8630-1VL</td>
</tr>
<tr>
<td>Alkaline phosphatase from Escherichia coli</td>
<td>Sigma-Aldrich</td>
<td>Cat#P5931-100UN</td>
</tr>
<tr>
<td>Oligo(dT)<sub>25</sub> magnetic beads</td>
<td>New England Biolabs</td>
<td>Cat#S1419S</td>
</tr>
<tr>
<td>Water (HPLC Grade)</td>
<td>Fisher Scientific</td>
<td>Cat#AA22934M6</td>
</tr>
<tr>
<td>Acetonitrile (HPLC Grade)</td>
<td>Fisher Scientific</td>
<td>Cat#6000247</td>
</tr>
<tr>
<td>Ammonium Acetate</td>
<td>Spectrum Chemical</td>
<td>Cat#A2149-1KG</td>
</tr>
<tr>
<td>Ammonium Hydroxide</td>
<td>Spectrum Chemical</td>
<td>Cat#A1195</td>
</tr>
<tr>
<td>23G Syringe</td>
<td>Fisher Scientific</td>
<td>Cat#309571</td>
</tr>
<tr>
<td>1.5 mL low-bind tube</td>
<td>Corning</td>
<td>Cat#3207</td>
</tr>
<tr>
<td>Cell scraper</td>
<td>Corning</td>
<td>Cat#3008</td>
</tr>
<tr>
<td>0.2 μm PES filter</td>
<td>Whatman Puradisc</td>
<td>Cat#6780-2502</td>
</tr>
<tr>
<td>1 mL Syringe</td>
<td>Becton Dickinson</td>
<td>Cat#329654</td>
</tr>
<tr>
<td>Snap Cap, pre-slit</td>
<td>Fisher Scientific</td>
<td>Cat#14-823-480</td>
</tr>
<tr>
<td>Stepvial System™ Crimp/Snap Vial</td>
<td>Fisher Scientific</td>
<td>Cat#05-704-225</td>
</tr>
<tr>
<td>Xbridge BEH amide column (150 × 2.1 mm, 3 μm particle size)</td>
<td>Waters</td>
<td>Cat#186006724</td>
</tr>
<tr>
<td>DMEM</td>
<td>Gibco</td>
<td>Cat#11965118</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Sigma-Aldrich</td>
<td>Cat#F0926-500ML</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>PureLink RNA Mini Kit</td>
<td>Ambion</td>
<td>Cat#12183018A</td>
</tr>
<tr>
<td>RNA Clean &amp; Concentrator kit</td>
<td>Zymo Research</td>
<td>Cat#R1016</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell line</td>
</tr>
<tr>
<td>Human: HEK293E</td>
<td>ATCC</td>
<td>Cat#293c18; RRID: CVCL_6974</td>
</tr>
<tr>
<td colspan="3">Software</td>
</tr>
<tr>
<td>Xcalibur</td>
<td>Thermo Scientific</td>
<td>n/a</td>
</tr>
<tr>
<td>ProteoWizard</td>
<td>ProteoWizard</td>
<td>n/a</td>
</tr>
<tr>
<td>MAVEN</td>
<td>Elucidata</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Refrigerated centrifuge</td>
<td>Eppendorf</td>
<td>Cat#2231000655</td>
</tr>
<tr>
<td>Chemical Hood</td>
<td>Labconco</td>
<td>Cat#3746704</td>
</tr>
<tr>
<td>Magnetic Rack</td>
<td>Bio-Rad</td>
<td>Cat#1614916</td>
</tr>
<tr>
<td>Mini centrifuge</td>
<td>Benchmark Scientific</td>
<td>Cat#C1008-R</td>
</tr>
<tr>
<td>NanoDrop 2000c</td>
<td>Thermo Scientific</td>
<td>Cat#ND2000C</td>
</tr>
<tr>
<td>Thermomixer C</td>
<td>Eppendorf</td>
<td>Cat#5382000023</td>
</tr>
<tr>
<td>Dry heat block</td>
<td>Benchmark Scientific</td>
<td>Cat#BSH6000</td>
</tr>
<tr>
<td>Vortex</td>
<td>Benchmark Scientific</td>
<td>Cat#BV1003</td>
</tr>
<tr>
<td>Thermo Q Exactive Plus Hybrid Quadrupole-Orbitrap Mass Spectrometer</td>
<td>Thermo Scientific</td>
<td>n/a</td>
</tr>
<tr>
<td>Vanquish UHPLC System</td>
<td>Thermo Scientific</td>
<td>n/a</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Below is the list of reagents that need to be prepared before experiments. Use raw materials in the <a href="#key-resources-table">key resources table</a> or other available reagents with similar grade (i.e., RNase-free materials). Individual procedures take 10–30 min. The buffers and reconstituted enzymes are good for use for 3–6 months.</p>
<h3 id="sec2.1">PureLink RNA Mini Kit</h3>
<p>The kit provides PureLink Lysis Buffer, Wash Buffer I, Wash Buffer II, and Spin cartridges for total RNA isolation. Reconstitute the Lysis Buffer with beta-mercaptoethanol and Wash Buffer II with ethanol according to the <a href="https://www.thermofisher.com/order/catalog/product/12183020#/12183020">manufacturer’s protocol</a>.</p>
<h3 id="sec2.2">Zymo RNA Clean &amp; Concentrator Kit</h3>
<p>The kit provides Zymo RNA Binding Buffer, RNA Prep Buffer, RNA Wash Buffer, and Spin cartridges for removal of salts from RNA samples. Reconstitute the RNA Wash Buffer with ethanol according to the <a href="https://www.zymoresearch.com/products/rna-clean-concentrator-5">manufacturer’s protocol</a>.</p>
<h3 id="sec2.3">Oligo(dT) Binding Buffer</h3>
<p>After mixing below components, store the buffer at 4°C.</p>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH7.5)</td>
<td>1 M</td>
<td>20 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>LiCl</td>
<td>8 M</td>
<td>1 M</td>
<td>6.25 mL</td>
</tr>
<tr>
<td>EDTA</td>
<td>500 mM</td>
<td>2 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>n/a</td>
<td>42.55 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.4">Oligo(dT) wash buffer</h3>
<p>After mixing below components, store the buffer at 4°C.</p>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH7.5)</td>
<td>1 M</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>LiCl</td>
<td>8 M</td>
<td>150 mM</td>
<td>937.5 μL</td>
</tr>
<tr>
<td>500 mM EDTA</td>
<td>500 mM</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>n/a</td>
<td>48.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.5">Oligo(dT) Elution Buffer</h3>
<p>Dilute 100 μL of 1M Tris-HCl (pH7.5) with 9.9 mL nuclease-free water to make 10 mM Tris-HCl. Store at 4°C.</p>
<h3 id="sec2.6">Reconstitution of nuclease P1</h3>
<p>Prepare 2 unit/μL nuclease P1 stock by dissolving nuclease P1 powder in nuclease P1 reconstitution buffer. Aliquot into 10 μL and store at −20°C.</p>
<h3 id="sec2.7">Nuclease P1 reconstitution buffer</h3>
<p>After mixing below components, store the buffer at 4°C.</p>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaOAc (pH5.3)</td>
<td>3 M</td>
<td>50 mM</td>
<td>60 μL</td>
</tr>
<tr>
<td>ZnCl2</td>
<td>100 mM</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>100%</td>
<td>25%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>n/a</td>
<td>7.34 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.8">Reconstitution of alkaline phosphatase</h3>
<p>Prepare 2 unit/μL alkaline phosphatase stock by dissolving alkaline phosphatase powder in alkaline phosphatase reconstitution buffer. Aliquot into 10 μL and store at −20°C.</p>
<h3 id="sec2.9">Alkaline phosphatase reconstitution buffer</h3>
<p>After mixing below components, store the buffer at 4°C.</p>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH7.5)</td>
<td>1 M</td>
<td>5 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>25 mM</td>
<td>0.5 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>100%</td>
<td>25%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>n/a</td>
<td>n/a</td>
<td>7.35 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.10">2 M ammonium bicarbonate</h3>
<p>Dissolve 158 mg ammonium bicarbonate in 1 mL PCR grade water. Filter through 0.2 μM PES filter using 1 mL syringe. Prepare fresh ammonium bicarbonate solution on the day of experiment.</p>
<h3 id="sec2.11">1.2 M HCl</h3>
<p>Dilute 100 μL 6 M HCl in 500 μL PCR grade water. Store at 4°C.</p>
<h3 id="sec2.12">Preparation of m<sup>6</sup>A and adenosine standards</h3>
<p>Dissolve 1 mg of m<sup>6</sup>A or adenosine powder in 1 mL of 75% acetonitrile (acetonitrile: water, 75:25, v/v). Dilute each standard as 1 mg/L. Mix m<sup>6</sup>A and adenosine standard solutions with the same volume (1:1) to make a standard solution containing 500 μg/L of both m<sup>6</sup>A and adenosine. Aliquot this standard solution mix into 100 μL and store at −80°C. To make standard calibration curves, make serial dilution of standards at 0.5, 1, 2, 5, 10, 20 and 50 μg/L. Then, obtain <i>y</i> = <i>ax</i> equation (<i>a</i> is constant) with ion counts (<i>y</i>) and standard concentrations (<i>x</i>) using linear regression.</p>
<h3 id="sec2.13">LC-MS setting</h3>
<p>Thermo Q Exactive™ Plus Hybrid Quadrupole-Orbitrap™ Mass Spectrometer coupled with Vanquish UHPLC system was used. LC-MS system was controlled by Xcalibur software (Thermo). Metabolite separation was conducted by Xbridge BEH amide column (150 × 2.1 mm, 3 μm particle size). LC gradient was generated using LC solvents A and B (<a href="#tbl1">Table 1</a>). Autosampler temperature was set at 4°C and the column temperature was set at 25°C. MS analysis was performed with a full-scan mode for measurement of samples (m/z range 250–300, positive ion mode). MS<sup>2</sup> fragmentation was used to confirm m<sup>6</sup>A and adenosine (<a href="#tbl2">Table 2</a>).</p>
<table id="tbl1">
<caption>Table 1. LC gradient method</caption>
<thead>
<tr>
<th>min</th>
<th>Flow (mL/min)</th>
<th>LC solvent A (%)</th>
<th>LC solvent B (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>0.35</td>
<td>25</td>
<td>75</td>
</tr>
<tr>
<td>3</td>
<td>0.35</td>
<td>25</td>
<td>75</td>
</tr>
<tr>
<td>4</td>
<td>0.35</td>
<td>50</td>
<td>50</td>
</tr>
<tr>
<td>5</td>
<td>0.35</td>
<td>90</td>
<td>10</td>
</tr>
<tr>
<td>7</td>
<td>0.35</td>
<td>90</td>
<td>10</td>
</tr>
<tr>
<td>7.5</td>
<td>0.35</td>
<td>25</td>
<td>75</td>
</tr>
<tr>
<td>11</td>
<td>0.35</td>
<td>25</td>
<td>75</td>
</tr>
</tbody>
</table>
<table id="tbl2">
<caption>Table 2. MS parameters</caption>
<thead>
<tr>
<th>Parameter</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sheath gas flow rate</td>
<td>40 psi</td>
</tr>
<tr>
<td>Aux gas flow rate</td>
<td>10 psi</td>
</tr>
<tr>
<td>Sweep gas flow rate</td>
<td>2 psi</td>
</tr>
<tr>
<td>Spray voltage</td>
<td>2.7 kV</td>
</tr>
<tr>
<td>Capillary temperature</td>
<td>300°C</td>
</tr>
<tr>
<td>Collision energy</td>
<td>40 eV</td>
</tr>
<tr>
<td>Peak width</td>
<td>6 s</td>
</tr>
<tr>
<td>S-lens RF level</td>
<td>50</td>
</tr>
<tr>
<td>AGC target</td>
<td>3E+06</td>
</tr>
<tr>
<td>Maximum injection time</td>
<td>500 msec</td>
</tr>
</tbody>
</table>
<h3 id="sec2.14">LC solvent A</h3>
<p>After mixing below components, store the solvent at 25°C.</p>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ammonium acetate dissolved in water</td>
<td>1 M</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Ammonium hydroxide dissolved in water</td>
<td>1 M</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Acetonitrile</td>
<td>100%</td>
<td>5%</td>
<td>25 mL</td>
</tr>
<tr>
<td>Water</td>
<td>n/a</td>
<td>n/a</td>
<td>455 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.15">LC solvent B</h3>
<p>100% Acetonitrile</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Isolation of total RNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>In this step, total RNA is isolated from the cells using PureLink RNA Mini Kit. Prepare the buffers in the kit according to the manufacturer’s protocol before starting the experiment.</p>
<ol>
<li>Sample harvest and homogenization
<ol type="a">
<li>Remove medium from cells and rinse with 1× PBS (e.g., 5 mL for 60 mm plates).</li>
<li>Add 350 μL of PureLink Lysis buffer to the plate. Scrape the cell lysate thoroughly using a cell scraper.</li>
<li>Transfer the viscous liquid into a new 1.5 mL tube.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be frozen at −80°C.</div>
</li>
<li>Homogenize the sample with a 23G syringe needle. Repeat the suction-release step 5–10 times.
<div class="note">
<span class="note-title">Note:</span> Try not to generate too many bubbles during homogenization (samples can overflow the tubes).</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Purification of total RNA
<ol type="a">
<li>Add 350 μL of 70% ethanol to sample (Sample: 70% ethanol = 1:1) and vortex.</li>
<li>Transfer 700 μL of the sample into the PureLink Spin cartridge and centrifuge for 15 s at 12,000 × <i>g</i> at 25°C. Discard the flow through.</li>
<li>Add 700 μL of PureLink Wash Buffer I and centrifuge for 15 s at 12,000 × <i>g</i> at 25°C. Discard the flow through.</li>
<li>Add 500 μL of PureLink Wash Buffer II and centrifuge for 15 s at 12,000 × <i>g</i> at 25°C. Discard the flow through. Repeat the step twice.</li>
<li>Centrifuge the column for 2 min at 12,000 × <i>g</i> to ensure complete removal of the wash buffer.</li>
<li>Transfer the column to a new 1.5 mL tube.
<div class="note">
<span class="note-title">Note:</span> Leave the column on the tube for 5 min to <b>evaporate</b> any residual ethanol from the wash buffer.</div>
</li>
<li>Add 50 μL nuclease-free water directly to the column matrix and incubate for 5 min.</li>
<li>Centrifuge for 2 min at 12,000 × <i>g</i> at 25°C. The flow through contains total RNA.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be frozen at −80°C.</div>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Measure RNA concentration using Nanodrop with absorbance at 260 nm.</li>
</ol>
<p><b>Note:</b> 80–90% confluent HEK293E cells from a 60 mm plate result in &gt;75 μg total RNA.</p>
<h3 id="sec3.2">Purification of mRNA using Oligo(dT) beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>In this step, polyadenylated [poly(A)] mRNA is isolated from total RNA using oligo(dT) beads. Except heat block and ice incubation steps, all procedures are performed at 25°C. When not in the reaction (i.e., while preparing beads or kits), RNA samples should be kept on ice. Before starting the experiment, bring the oligo(dT) Binding and oligo(dT) Wash buffers to 25°C, and reconstitute the buffers in the Zymo RNA Clean &amp; Concentrator kit.</p>
<ol start="4">
<li>Preparation of RNA
<ol type="a">
<li>Spin down the samples.</li>
<li>Adjust the sample amount to contain 50 μg of total RNA in 100 μL nuclease-free water.</li>
<li>Heat the samples in dry heat block at 65°C for 2 min to disrupt RNA secondary structures.</li>
<li>Place the sample on ice immediately.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Rapid cool down of the heated RNA samples is the key to minimize secondary structures for efficient binding of RNAs with Oligo(dT) beads.</div>
<ol start="5">
<li>Preparation of Oligo(dT) beads
<ol type="a">
<li>Resuspend Oligo(dT) beads (vortex &gt;30 s or tilt for 5 min).</li>
<li>Transfer beads to a new 1.5 mL tube. Use 250 μg beads for 50 μg total RNA.
<div class="note">
<span class="note-title">Note:</span> For example, the concentration of NEB #S1419S bead suspension is 5 mg/mL. Use 50 μL of NEB #S1419S bead suspension to get 250 μg beads.</div>
</li>
<li>Add 0.5 mL Oligo(dT) Binding Buffer to the beads and rinse by pipetting.</li>
<li>Place the tube on the magnetic rack until the solution is clear (i.e., incubate for 1 min on the magnetic rack).</li>
<li>Carefully remove the supernatant using a pipette.
<div class="note">
<span class="note-title">Note:</span> Do not use aspirator to prevent loss of the beads.</div>
</li>
<li>Add 200 μL Oligo(dT) Binding Buffer to the beads and mix well.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Isolation of mRNA
<ol type="a">
<li>Add 100 μL RNA solution (prepared in step 4) to 200 μL Oligo(dT) bead suspension (i.e., RNA solution: Bead suspension = 1:2). Mix thoroughly by pipetting.</li>
<li>Incubate the samples in Thermomixer for 5 min with agitation at 800 rpm at 25°C.</li>
<li>Place the tube on the magnetic rack until the solution is clear, and then remove the supernatant.</li>
<li>Wash the mRNA-bead complex with 200 μL Oligo(dT) Wash Buffer by pipetting.
<div class="note">
<span class="note-title">Note:</span> When handling multiple samples, stagger steps 6a–6d to decrease differences in bead incubation times among the samples.</div>
</li>
<li>Incubate on the magnetic rack and remove the supernatant.</li>
<li>Repeat steps 6d and 6e.</li>
<li>To discard the wash buffer completely, centrifuge at 200 × <i>g</i> for 10 s at 25°C.
<div class="note">
<span class="note-title">Note:</span> Do not centrifugate the beads at speeds higher than 200 × <i>g</i>. Place the tube in a metal rack and remove the residual wash buffer.</div>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Elution
<ol type="a">
<li>Add 50 μL of Oligo(dT) Elution Buffer to the beads. Mix well by pipetting.</li>
<li>To elute mRNA from the beads, heat the samples at 75°C for 2 min.</li>
<li>Immediately place the tube on the magnetic rack and incubate until the solution is clear.</li>
<li>Transfer the supernatant (i.e., eluted mRNAs) to a new 1.5 mL tube.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be frozen at −80°C.</div>
<ol start="8">
<li>Repeat one more round of Oligo(dT) purification (i.e., steps 4–7) to achieve higher mRNA purity.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be frozen at −80°C.</div>
<ol start="9">
<li>Conduct RNA clean-up using Zymo RNA Clean &amp; Concentrator kit to remove residual salts from mRNA samples for the m<sup>6</sup>A processing step.
<ol type="a">
<li>Add 100 μL Zymo RNA Binding Buffer to 50 μL mRNA sample and mix (RNA Binding Buffer: mRNA sample = 2:1).
<div class="note">
<span class="note-title">Note:</span> To decrease variations in the isolated mRNA amount among the samples, use same amount of total RNA as a starting material (e.g., Adjust total RNA amount as 50 μg across all samples).</div>
</li>
<li>Add 150 μL of 100% ethanol and mix (mRNA-RNA Binding Buffer: 100% ethanol = 1:1)</li>
<li>Transfer the sample to the Zymo Spin cartridge.</li>
<li>Centrifuge at 12,000 × <i>g</i> for 30 s at 25°C. Discard the flow through.</li>
<li>Add 400 μL Zymo RNA Prep Buffer to the column and centrifuge at 12,000 × <i>g</i> for 30 s at 25°C. Discard the flow through.</li>
<li>Add 700 μL Zymo RNA Wash Buffer to the column and centrifuge at 12,000 × <i>g</i> for 30 s at 25°C. Discard the flow through.</li>
<li>Add 400 μL Zymo RNA Wash Buffer to the column and centrifuge for 2 min to completely remove the wash buffer.</li>
<li>Transfer the column carefully into a new 1.5 mL tube.
<div class="note">
<span class="note-title">Note:</span> Leave the column on the tube for 5 min to evaporate any residual ethanol from the wash buffer.</div>
</li>
<li>Add 15 μL nuclease-free water directly to the column matrix and incubate for 5 min.</li>
<li>Centrifuge at 16,000 × <i>g</i> for 30 s at 25°C. The flow through contains purified mRNA.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be frozen at −80°C.</div>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Measure mRNA concentration using Nanodrop with absorbance at 260 nm. 50 μg of total RNA results in &gt;300 ng mRNA.</li>
</ol>
<h3 id="sec3.3">Processing of mRNA samples for m<sup>6</sup>A analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 h</div>
<p>In this step, nuclease P1 hydrolyzes phosphodiester bonds in mRNA to generate nucleoside 5′-monophosphates (nucleotides). Nucleotides are further processed with alkaline phosphatase to remove phosphate groups for LC-MS analysis. Efficient enzymatic processing is critical for the LC-MS to accurately detect each nucleoside signal. Use 100–200 ng of mRNAs for sample processing. Include negative control (water-only sample) to estimate and subtract background signals.</p>
<ol start="11">
<li>Nuclease P1 digestion
<ol type="a">
<li>In each sample tube, add the components listed below. Make a master mix and distribute to the samples to reduce pipetting error.
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>mRNA</td>
<td>n/a</td>
<td>200 ng</td>
<td>20 μL</td>
</tr>
<tr>
<td>Nuclease P1</td>
<td>2 unit/μL</td>
<td>1 unit</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>NaCl</td>
<td>5 M</td>
<td>25 mM</td>
<td>0.4 μL</td>
</tr>
<tr>
<td>ZnCl<sub>2</sub>
</td>
<td>0.1 M</td>
<td>2.5 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>PCR grade water</td>
<td>n/a</td>
<td>n/a</td>
<td>17.1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>40 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Vortex briefly and spin down the samples.</li>
<li>Incubate the samples in Thermomixer for 2 h at 37°C with agitation at 800 rpm for 30 s every 5 min.</li>
</ol>
</li>
</ol>
<ol start="12">
<li>Phosphatase treatment
<ol type="a">
<li>Add 2 μL of 2 M ammonium bicarbonate solution.</li>
<li>Vortex briefly and spin down the samples.</li>
<li>Add 1 unit of alkaline phosphatase.</li>
<li>Vortex briefly and spin down the samples.</li>
<li>Incubate the samples in Thermomixer for 2 h at 37°C with agitation at 800 rpm for 30 s every 5 min.</li>
</ol>
</li>
<li>Re-neutralization of the solution
<ol type="a">
<li>To neutralize the reaction, add 1 μL of 1.2 M HCl. Vortex briefly to mix the samples.</li>
<li>Centrifuge the samples for 30 min at 16,000 × <i>g</i> at 4°C to precipitate any insoluble parts.</li>
<li>Transfer 20 μL supernatant to a new 1.5 mL tube.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be frozen at −80°C.</div>
<h3 id="sec3.4">LC-MS analysis of m<sup>6</sup>A</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="14">
<li>Preparation of LC-MS samples
<ol type="a">
<li>Mix 20 μL of the purified nucleoside samples with 40 μL of acetonitrile.
<div class="note">
<span class="note-title">Note:</span> This is to have a similar proportion of organic solvent in the LC-MS sample with the starting LC mobile phase (75% acetonitrile).</div>
</li>
<li>Centrifuge the samples at 16,000 × <i>g</i> for 10 min at 4°C to precipitate any insoluble parts.</li>
<li>Carefully transfer 40 μL of supernatant to a new LC-MS vial.
<div class="note">
<span class="note-title">Note:</span> Do not touch the pellet.</div>
</li>
</ol>
</li>
</ol>
<ol start="15">
<li>Inject 3 μL of samples to the LC-MS system with the setting parameters described above.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The 3 μL sample now contains ∼4.5 ng mRNA if the m<sup>6</sup>A processing was performed with 200 ng mRNA as a starting material.</div>
<ol start="16">
<li>Run m<sup>6</sup>A and adenosine standards in the same LC-MS setting with the samples.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Under suggested conditions, m<sup>6</sup>A and adenosine are eluted at 1.65 min and 1.86 min, respectively (<a href="#fig1">Figure 1</a> and <a href="#tbl3">Table 3</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/907-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. LC-MS chromatograms of m<sup>6</sup>A and adenosine</div>
<p>Representative LC-MS (left) and MS/MS (right) chromatogram of m<sup>6</sup>A (A) and adenosine (B) derived from standards and purified mRNA samples (see also <a href="#tbl3">Table 3</a>).</p>
</figcaption>
</figure>
<table id="tbl3">
<caption>Table 3. Summary of LC-MS results (see also <a href="#fig1">Figure 1</a>)</caption>
<thead>
<tr>
<th>Metabolite</th>
<th>Formula</th>
<th>m/z</th>
<th>Ion species</th>
<th>Retention time (min)</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<i>N</i><sup>6</sup>-Methyladenosine (m<sup>6</sup>A)</td>
<td>C11H15N5O4</td>
<td>282.1197</td>
<td>[M+H]+</td>
<td>1.65</td>
</tr>
<tr>
<td>Adenosine</td>
<td>C10H13N5O4</td>
<td>268.1041</td>
<td>[M+H]+</td>
<td>1.86</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<ol>
<li>Convert LC-MS raw data files to mzXML using Proteowizard software.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> MAVEN software (<a href="https://resources.elucidata.io/elmaven">https://resources.elucidata.io/elmaven</a>) or other software can be used to use mzXML file for peak visualization and quantitation.</div>
<ol start="2">
<li>Export ion counts of m<sup>6</sup>A and adenosine for each sample.</li>
<li>Calculate the concentration of m<sup>6</sup>A and adenosine using standard calibration curves (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/907-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Standard calibration curves of m<sup>6</sup>A and adenosine.</div>
</figcaption>
</figure>
</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol details quantitative measurement of m<sup>6</sup>A modification in mRNAs using LC-MS. While this protocol is straightforward and easy to follow, it has some limitations.</p>
<p>First, to measure m<sup>6</sup>A modification of mRNAs, we purified mRNA from total RNA. However, contamination of abundant RNA species such as ribosomal RNA (rRNA) can occur. To measure m<sup>6</sup>A levels specifically from the mRNA m<sup>6</sup>A modification sequence (GA∗C; A∗ is methylated adenosine), the users can adopt RNase T1-based assays such as 2D thin-layer chromatography (TLC) (<a href="#bib1">Bodi and Fray, 2017</a>). In the TLC assay, mRNAs are processed with RNase T1 (specifically cleaves after G) followed by <sup>32</sup>P labeling of nucleotides, which enables specific labeling of m<sup>6</sup>A from mRNAs.</p>
<p>Second, this protocol quantitates m<sup>6</sup>A levels from a total pool of mRNAs and cannot distinguish differential m<sup>6</sup>A modification levels in individual genes. This requires site-specific m<sup>6</sup>A detection using qPCR or TLC. Transcriptome-wide m<sup>6</sup>A sequencing methods have also been developed by several groups (reviewed in <a href="#bib7">Zaccara et al., 2019</a>).</p>
<p>Finally, while this protocol provides an optimized LC-MS condition for efficient measurement of m<sup>6</sup>A and adenosine in mRNA, users can adjust RNA purification and mass spectrometry methods to quantitate other modifications in various RNA species and DNA (<a href="#bib3">Su et al., 2014</a>; <a href="#bib4">Thüring et al., 2017</a>; <a href="#bib5">Wei et al., 2018</a>; <a href="#bib6">Wein et al., 2020</a>). Comprehensive analysis of nucleotide chemical modifications using LC-MS technology will provide valuable tools and resources in the field of transcriptomics, genomics, and metabolomics.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low yield of mRNA (Related step: <a href="#sec3.2">purification of mRNA using Oligo(dT) beads</a>).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Since mRNA is only 1–5% of total RNA, preparation of enough amount of total RNA is key to get enough amount of mRNA (e.g., 30–100 μg total RNA as a starting material). Also, use nucleic acid low-bind tubes and low retention pipette tips to minimize loss of mRNAs during purification. We recommend calculating mRNA purification yield using the amount of starting material (total RNA, step 3) and final mRNA product (step 10; consider that the yield of Zymo RNA Clean &amp; Concentrator Kit is 70–80%). To decrease differences in mRNA yield among the samples, stagger the 5 min Oligo(dT) bead-RNA incubation step when handling several samples. Randomization of the sample order during the reaction also helps to decrease the incubation time differences caused by the sample order (i.e., randomization of sample order prevents Sample #1, 2, 3 being incubated longer with the beads than Sample #22, 23, 24).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Contamination of other RNA species (Related step: <a href="#sec3.2">purification of mRNA using Oligo(dT) beads</a>).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Contamination of abundance RNA species such as rRNA can interfere with accurate measurement of m<sup>6</sup>A and adenosine levels from mRNAs. To increase the purity of mRNA, we recommend conducting the oligo(dT) bead isolation twice. For the same sample, beads can be reused after washing the beads with oligo(dT) Wash Buffer. To evaluate rRNA contamination, conduct qPCR with primers that are specific to rRNAs. rRNAs can be further removed by RiboMinus Eukaryote Kit (Invitrogen Cat#A15020).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Background nucleoside signal (Related step: <a href="#sec3.3">processing of mRNA samples for m6A analysis</a>).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Due to the nucleic acids contaminated from the environment and reagents, background m<sup>6</sup>A and adenosine signals can be detected. To prevent this, we recommend using PCR grade (i.e., nucleic acid-free) water during the m<sup>6</sup>A processing step and subtracting the background signals detected in the water-only sample. Also, conduct m<sup>6</sup>A processing step in a clean chemical fume hood.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Confirmation of m<sup>6</sup>A peak (Related steps: <a href="#sec3.4">LC-MS analysis of m6A</a> and <a href="#expected-outcomes">expected outcomes</a>).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Some samples may show m<sup>6</sup>A isomers as shown in the <a href="#fig1">Figure 1</a>A (left panel). To avoid mis-annotation of peaks, we recommend running m<sup>6</sup>A standards in parallel with the samples to obtain accurate retention time. MS/MS profile can be used to confirm correct m<sup>6</sup>A peak (<a href="#fig1">Figure 1</a>A, right panel).</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Alternative reagents and equipment (Related step: <a href="#key-resources-table">key resources table</a>).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>In the <a href="#key-resources-table">key resources table</a>, we provided catalog numbers of the reagents and equipment that this protocol used, which may not be available in other circumstances. Users can use any reagents with equivalent grade (e.g., RNase-free reagents for RNA isolation and processing steps; HPLC-grade reagents for LC-MS). Regarding the equipment, (1) Oligo(dT)-RNA sample can be mixed using orbital shakers or rotators instead of Thermomixer; (2) Shaking nuclease P1 and alkaline phosphatase reactions in Thermomixer is optional (i.e., reaction can be performed in a regular heat block without agitation); (3) for mass spectrometry analysis of nucleoside samples, any type of high-sensitivity tandem mass spectrometers such as triple quadrupole, quadrupole-time of flight, and quadrupole-orbitrap can be used.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bodi, Z. and Fray, R.G. (2017). Detection and quantification of N6-methyladenosine in messenger RNA by TLC. In Methods in Molecular Biology (Humana Press Inc.), pp. 79-87. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00431-7/sref1">View at publisher</a></p>
<p id="bib2">Cho, S., Lee, G., Pickering, B.F., Jang, C., Park, J.H., He, L., Mathur, L., Kim, S.-S., Jung, S., Tang, H.-W., et al. (2021). mTORC1 promotes cell growth via m6A-dependent mRNA degradation. Mol. Cell. <i>81</i>, 2064-2075.e8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00431-7/sref2">View at publisher</a></p>
<p id="bib3">Su, D., Chan, C.T.Y., Gu, C., Lim, K.S., Chionh, Y.H., McBee, M.E., Russell, B.S., Babu, I.R., Begley, T.J., and Dedon, P.C. (2014). Quantitative analysis of ribonucleoside modifications in tRNA by HPLC-coupled mass spectrometry. Nat. Protoc. <i>9</i>, 828-841. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00431-7/sref3">View at publisher</a></p>
<p id="bib4">Thüring, K., Schmid, K., Keller, P., and Helm, M. (2017). LC-MS analysis of methylated RNA. In Methods in Molecular Biology (Humana Press Inc.), pp. 3-18. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00431-7/sref4">View at publisher</a></p>
<p id="bib5">Wei, J., Liu, F., Lu, Z., Fei, Q., Ai, Y., He, P.C., Shi, H., Cui, X., Su, R., Klungland, A., et al. (2018). Differential m 6 A, m 6 A m , and m 1 A demethylation mediated by FTO in the cell nucleus and cytoplasm. Mol. Cell <i>71</i>, 973-985.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00431-7/sref5">View at publisher</a></p>
<p id="bib6">Wein, S., Andrews, B., Sachsenberg, T., Santos-Rosa, H., Kohlbacher, O., Kouzarides, T., Garcia, B.A., and Weisser, H. (2020). A computational platform for high-throughput analysis of RNA sequences and modifications by mass spectrometry. Nat. Commun. <i>11</i>, 1-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00431-7/sref6">View at publisher</a></p>
<p id="bib7">Zaccara, S., Ries, R.J., and Jaffrey, S.R. (2019). Reading, writing and erasing mRNA methylation. Nat. Rev. Mol. Cell Biol. <i>20</i>, 608-624. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00431-7/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Gina Lee (<a href="mailto:ginalee@uci.edu">ginalee@uci.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique materials, reagents, or cell lines.</p>
<h4>Data and code availability</h4>
<p>The published article includes all datasets generated and analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>This research was supported by <a href="https://doi.org/10.13039/100000005">Department of Defense</a> TS200022 (G.L.) and AASLD Foundation Pinnacle Research Award in Liver Disease (C.J.). The research was also partially supported by <a href="https://doi.org/10.13039/100000002">NIH</a> P30CA062203 (Chao Family Comprehensive Cancer Center) and NIH U54CA217378 (UCI Center for Cancer Systems Biology). S.J. was supported by a postdoctoral fellowship from <a href="https://doi.org/10.13039/501100003725">National Research Foundation of Korea</a> (2021R1A6A3A14039681). We thank co-authors of (<a href="#bib2">Cho et al., 2021</a>) and members of the Lee and Jang laboratories for helpful discussions. We apologize to authors whose work was not cited due to space limitations.</p>
<h3>Author contributions</h3>
<p>L.M. and S.J. performed the experiments and constructed the protocol. S.J. developed LC-MS method and analyzed the results. C.J. and G.L. conceived the project and provided supervision.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>