<article>
<section>
<h1 id="header">A protocol to study bacteriophage adaptation to new hosts</h1>
<p><time datetime="2021-08-27">Published: August 27, 2021</time></p>
<p>Raquel Luzon-Hidalgo,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Valeria A. Risso,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Asuncion Delgado,<sup><a href="#aff1">1</a></sup> Beatriz Ibarra-Molero,<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> and Jose M. Sanchez-Ruiz<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Departamento de Quimica Fisica, Facultad de Ciencias, Unidad de Excelencia de Quimica Aplicada a Biomedicina y Medioambiente (UEQ), Universidad de Granada, 18071 Granada, Spain</p>
<p id="fn1"><sup>2</sup>These authors contributed equally</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:beatriz@ugr.es">beatriz@ugr.es</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sanchezr@ugr.es">sanchezr@ugr.es</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100784">10.1016/j.xpro.2021.100784</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>A general protocol for the experimental assessment of bacteriophage adaptation to new hosts is described. We use as a model system the lytic phage T7 and an <i>engineered E. coli</i> strain modified to hamper the recruitment of a known proviral factor. Our protocol includes steps of phage amplification, plaque and liquid lysis assays, and DNA extraction for next-generation sequencing of the viral genome over several rounds of laboratory evolution thus allowing the investigation of the sequence determinants of viral adaptation.</p>
<p>For complete information on the generation and use of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref8">Luzon-Hidalgo et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for experimental characterization of phage adaptation to new hosts</li>
<li>DNA extraction for NGS of viral genome to unwind the molecular basis of adaptation</li>
<li>Potential application in the development of anti-microbial phage therapies</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/967-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Bacteriophage T7 is an obligate lytic phage that infects most <i>E. coli</i> strains. Its genome is a linear, double stranded DNA molecule that comprises about 57 genes encoding for 60 potential proteins (https://www.ncbi.nlm.nih.gov/nuccore/NC_001604.1; <a href="#bib4">Chan et al., 2005</a>). Genes are grouped into class I (early genes, along with the RNA polymerase), class II (DNA replication related genes) and class III (virus structure and assembly related genes). Some of the early genes, including RNA polymerase, are initially transcribed by the <i>E. coli</i> RNA polymerase. T7 functional replisome is one of the simplest known and is comprised of three viral proteins (gp5 DNA polymerase, a primase-helicase and a ssDNA binding protein) and a proviral host factor, thioredoxin (<a href="#bib7">Hamdan and Richardson, 2009</a>). Remarkably, bacterial thioredoxin is required to bind to gp5 DNA polymerase, through well-known interactions with its TBD (<i>thioredoxin binding domain</i>), acting as a necessary processivity factor for efficient DNA replication (<a href="#bib6">Etson et al., 2010</a>).</p>
<p>We have challenged bacteriophage T7 to infect an <i>engineered</i> host with a modified thioredoxin. It is actually a functional ancestral thioredoxin that significantly differs in sequence from <i>E. coli</i> thioredoxin, its natural proviral factor (<a href="#bib5">Delgado et al., 2017</a>). In the absence of thioredoxin, T7 infection does not proceed for the aforementioned reason. Previous work in our group reported an extremely low efficiency of virus propagation in the <i>engineered</i> host (∼5·10<sup>−8</sup>), probably as a result of the perturbation of relevant interactions between thioredoxin and the TBD (<a href="#bib5">Delgado et al., 2017</a>). Therefore, T7 is unable to recruit the alternative thioredoxin to be part of its replisome, at least in an efficient manner. In the view of this, we wonder if the virus eventually adapts to the <i>new</i> proviral factor and, if so, which is the molecular mechanism behind virus adaptation.</p>
<p>To address this question, in <a href="#bib8">Luzón-Hidalgo et al. 2021</a>, we have carried out a number of rounds of adaptation of T7 to the <i>engineered</i> host with an ancestral thioredoxin that shares only 57% of its sequence with <i>E. coli</i> thioredoxin, its natural proviral factor, and which hinders virus replication around 6–7 orders of magnitude.</p>
<p>Here we present our detailed protocol for the experimental assessment of viral evolution. This process is intended to generate virus particles able to overcome the molecular hindrance imposed by the <i>engineered</i> host. It comprises the following steps that are developed below (see <a href="#fig2">Figure 2</a> for a schematic depiction of the complete experimental procedure): i) Phage amplification is typically required to ensure evolution experiments start from a highly infective stock; ii) Titer determination of the phage stock is described based on plaque assays; iii) A complete round of T7 adaptation to the <i>new</i> host is depicted and its capability of infection is monitored based on plaque assays; iv) Determination of lysis profiles for evolved phages corresponding to different rounds of adaptation allows for infectivity efficiency measurements; v) genomic DNA extraction from evolved phage suspensions is carried out with the aim of detecting specific mutations responsible for phage adaptation by Illumina new generation sequencing.</p>
<p>We believe our protocol may be applied to study bacteriophage adaptation to any new host and this could be of great interest in the context of antibacterial therapies. The appearance of multidrug-resistant bacteria is considered an emergent global threat and a major public health problem. In this regard, phage therapy has revealed as a promising strategy for the treatment of infections by antibiotic-resistant bacteria. Among the numerous advantages of using phage therapy (<a href="#bib3">Bragg et al., 2014</a>), we highlight the fact that concerns related with bacteriophage resistance are expected to be of less relevance than the resistance to antibiotics, in part because coevolution bacteria-phages does take place naturally. Thus, we reckon that performing phage adaptation studies may be crucial in guiding and improving phage therapy and that our protocol could be of general interest in the field.</p>
<p>Finally, it is worth noting that proviral factor modification or repression provides a mechanism for hosts to generate resistance to viruses. Therefore, the methodology we describe here may potentially be used to explore virus-host arm races and the strategies for virus counter-adaptation to the resistant hosts.</p>
<div class="note">
<span class="note-title">Note:</span> <i>E. coli</i> strains used in this work as hosts for the T7 bacteriophage are the following (see <a href="#key-resources-table">key resources table</a>): DHB3 (<i>araD139</i> Δ<i>(ara-leu)7697 ΔlacX74 galE galK rpsL phoR</i> Δ(<i>phoA</i>)<i>Pvu</i>II Δ<i>malF3 thi</i>) and FA41 (DHB4 <i>thioredoxin minus</i>) strains were kindly donated by Jon Beckwith (Harvard Medical School). DHB4, the parental strain of FA41, is a DHB3 derivative bearing F' (<a href="#bib2">Boyd et al., 1987</a>) unable to support T7 growth due to the presence of the F′ factor. For this reason, DHB3 instead of DHB4 is used throughout this work.</div>
<p>FA41 cells were cured of the F′ factor to allow infectivity by T7 and lysogenized so the T7 RNA polymerase gene was inserted in the bacterial chromosome using the λDE3 Lysogenization Kit (<a href="#bib5">Delgado et al., 2017</a>). In addition, cured and lysogenized FA41 strain was transformed with a pET30a(+) derivative plasmid containing either the <i>E. coli</i> thioredoxin gene (<i>original host</i>) or the ancestral thioredoxin (<i>engineered host</i>), under the T7 promoter.</p>
<h3 id="sec1.1">Bacteriophage amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Typically, T7 phage suspensions are stored at −20°C and keep viable virus particles for months. Nevertheless, it should be noted that after every freeze/thaw cycle, the titer usually decreases about 1–2 orders of magnitude. For this reason, before every evolution experiment an amplification of the phage stock should be included, in order to make sure that the experiment is started out from a highly infective sample (an initial titer within the range 10<sup>9</sup>–10<sup>11</sup> pfu/mL is recommended). For this purpose an <i>E. coli</i> liquid culture is infected with the T7 phage stock solution as follows:</p>
<ol>
<li>Bacterial growth:
<ol type="a">
<li>Take 375 μL of a 5 mL overnight or 15 h <i>E. coli</i> liquid culture and add LB broth to a final volume of 15 mL in a 50 mL tube (1/40 dilution).</li>
<li>Incubate the bacterial suspension with shaking at 37°C until OD<sub>600nm</sub> reaches 0.5 (3–4 h approximately).</li>
<li>Transfer 5 mL to a new tube and use it as bacterial growth control. In the absence of infection, a typical bacterial growth curve is expected to be obtained.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For wild type phage amplification we use DHB3 cells as thioredoxin is expressed endogenously. Please note that this strain lacks of antibiotic resistance so great care must be taken avoiding contamination. Thus, it is highly recommended setting up a tube with uninoculated LB broth as control for contamination during the process.</div>
<ol start="2">
<li>Phage infection: add 100 μL of phage suspension into 10 mL of grown bacterial culture (from step 1b).</li>
<li>Let the infection proceed with shaking at 37°C for 4–5 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Time for complete bacterial lysis depends on each particular culture. Although 2–3 hours are apparently enough to complete cell lysis, it is recommended allowing the infection to proceed an extra time just to assure the minimum number of unlysed cells in the suspension. Reaching complete lysis is important in order to avoid bacterial contamination of the phage stock and also to achieve higher phage titer.</div>
<ol start="4">
<li>Centrifugate to eliminate bacterial debris. Phage particles are at the supernatant.
<ol type="a">
<li>Aliquot the bacterial lysate into 1.5 mL tubes and centrifuge at 15871 <i>g</i> for 5 min at 4°C.</li>
<li>Carefully transfer the supernatants to new 1.5 mL tubes and, again, centrifuge at 15871 <i>g</i> for 5 min at 4°C.</li>
<li>Pool all supernatants from the second centrifugation in the same tube to guarantee the homogeneity of the amplified phage suspension.</li>
<li>Aliquot the amplified phage stock into 1.5 mL tubes. You can preserve them at −20°C (note that the actual phage suspension is in LB medium). One of these aliquots, V0, will be the starting point for the initial propagation of the virus in the engineered host.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> You may or may not see a pellet after the first centrifugation depending on the amount of unbroken cells. In any case, proceed with the second centrifugation to make sure all bacterial debris is removed.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend storing your phage in small aliquots in order to avoid successive freeze/thaw cycles.</div>
<div class="note">
<span class="note-title">Note:</span> Please be aware that, in principle, phage-resistant bacteria may potentially be present in the phage stock. To yield a bacterial cell-free phage suspension, filter the phage supernatant using a 0.22 μm sterilized filter.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point, amplified phage stock may be stored at −20°C. In case titer determination is performed within the following days, please keep the phage stock at 4°C .</div>
<h3 id="sec1.2">Phage plaque assay for titer determination</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>In this step the titration of the amplified wild-type phage suspension (obtained in the preceding step 4d) is carried out based on the number of plaque forming units (pfu) in serially diluted virus samples:</p>
<ol start="5">
<li>Bacterial culture should be grown up to an OD<sub>600nm</sub> of 0.5. If needed, the culture might be kept on ice until use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If titer determination is done immediately after the amplification step, then the same bacterial culture (kept at 4°C in the meantime) may be used for the infection</div>
<div class="note">
<span class="note-title">Note:</span> For wild type phage plaque assay, DHB3 cells should be used as previously done in the amplification step. Set up the indicated controls to check for any contamination of the LB broth as cells carry no antibiotic resistance. Also, set up a control for monitoring normal bacterial growth in the absence of infection (see step 1c, previous section).</div>
<ol start="6">
<li>Prepare serial dilutions of your stock phage suspension as described in <a href="#fig1">Figure 1</a>. Ten-fold dilutions are commonly arranged as follows:
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/967-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Phage plaque assay for titer determination</div>
<p>Upper panel: Preparation of ten-fold serial dilutions of phage stock is followed by bacterial culture infection. The corresponding mixtures are plated according to the double agar method and, after incubation, the number of well-isolated plaques allows for pfu/mL determinations. Lower panel: Representative examples of T7 plaque formation after inoculating the <i>original</i> host at different phage dilutions.</p>
</figcaption>
</figure>
<ol type="a">
<li>Transfer 100 μL of stock phage suspension V0 into a sterile 1.5 mL tube containing 900 μL of phage dilution buffer. This tube is labeled as 10<sup>−1</sup>. Gently vortex at 25°C.</li>
<li>Next, withdraw 100 μL of this solution, 10<sup>−1</sup>, into another 1.5 mL tube, labeled as 10<sup>−2</sup>, containing 900 μL of phage dilution buffer as well.</li>
<li>Continue this process until dilution 10<sup>−10</sup> is prepared. As a result, a total of 10 serial dilutions will be made.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare in advance your dilution tubes in order to minimize waiting times. 1.5 mL tubes with 1× phage dilution buffer may be prepared while waiting for phage suspension to melt, if necessary.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare in advance for next step as many LB agar plates as needed.</div>
<ol start="7">
<li>Inoculate fresh <i>E. coli</i> cultures with T7 dilutions. The following sub-steps must be completed for each dilution prepared in step 6:
<ol type="a">
<li>Gently mix 100 μL of the corresponding phage dilution with 100 μL of <i>E. coli</i> culture from step 5.</li>
<li>Transfer the inoculated culture to another tube with 4 mL of molten top agar, previously melted in the microwave, as described in the <a href="#materials-and-equipment">materials and equipment</a> section. While preparing, keep the molten top agar in a water bath set at 50°C.</li>
<li>Pour the resulting 4.2 mL of cells/phages mixture in top agar onto a LB agar plate. This is called the <i>double agar method</i> (<a href="#bib1">Adams, 1959</a>) and allows for infection to proceed as diffusion of the progeny phages is favored on the upper layer as it contains less amount of agar.</li>
<li>Allow the mixture to solidify, invert the plate and incubate it at 37°C overnight or for 15 h.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure that molten top agar is properly mixed with the inoculated culture to avoid any concentration gradient.</div>
<div class="note">
<span class="note-title">Note:</span> Typically, wild type phage dilutions up to 10<sup>−4</sup> contain a high number of phages leading to either uncountable confluent plaques or complete lysis of the bacterial lawn; therefore, plating those dilutions might not be required.</div>
<div class="note">
<span class="note-title">Note:</span> Routine control plates should be set up. In order to verify media sterility, spread a 4 mL aliquot of molten top agar onto an LB plate and incubate overnight or for 15 hours along with the test plates; no bacterial growth should be obtained in this plate. In addition, to check viability of uninfected host cells, add 100 μL of the <i>E. coli</i> culture to 4 mL of molten top agar, gently mix and spread onto an LB plate; after overnight or 15 hours incubation a uniform bacterial lawn with no lytic plaques should be visible.</div>
<ol start="8">
<li>Quantification of phage titer:
<ol type="a">
<li>The next day, visual inspection of each plate allows for plaque number determinations (N). Choose the three plates corresponding to the highest phage dilutions, typically within the 10<sup>−6</sup>–10<sup>−9</sup> dilution factor range, where individual and well-isolated circular clear areas, the so-called plaques, appear in the bacterial lawn (<a href="#fig1">Figure 1</a>, lower panel). It is assumed that well-defined plaques develop from a single phage-infected cell and the release of progeny virions subsequently infect the proximal cells, resulting in a characteristic circular shape plaque. Typically, plaque numbers vary within 300-1, depending on the particular dilution.</li>
<li>The concentration of infective virus particles in the stock may be calculated by using the following equation:</li>
</ol>
</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>Phage</mi> <mspace></mspace> <mi>titer</mi> <mo>=</mo> <mi>N</mi> <mo>×</mo> <mfrac> <mn>1</mn> <mrow> <mi>D</mi> <mi>F</mi> </mrow> </mfrac> <mo>×</mo> <mfrac> <mn>1</mn> <mi>V</mi> </mfrac> </mrow> </math></p>
<p>Where N is the plaque number in a particular plate, DF is the phage dilution factor for that plate and V is the initial withdrawn volume of phage stock, in mL. The resulting number is given as plaque forming units per milliliter (pfu/mL). Typically, an average of at least 3 different measurements for phage titer corresponding to different dilutions is performed.</p>
<div class="note">
<span class="note-title">Note:</span> Usually, an ideal plaque count is within the 5–15/200 range.</div>
<div class="note">
<span class="note-title">Note:</span> We prefer to average plaque counts from plates obtained using different dilution factors instead of averaging replica of the same dilution factor. In this way, some common sources of error in plate counting, as inaccurate pipetting, presence of cell clumps, etc. (<a href="#bib9">Madigan et al., 2019</a>) would be easily detected.</div>
<div class="note">
<span class="note-title">Note:</span> Plates might be stored at 4°C for six months (plates should be wrapped with sealing film and stored upside down) and plaques could continue to be used although the titer could significantly decrease up to 10<sup>6</sup>.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended taking pictures of plates so morphology and size of plaques can be compared with future infections (<a href="#fig1">Figure 1</a>, lower panel).</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> In evolution experiments you can pause at this point and plaques may be stored at 4°C. Plaque count and titer calculation may also be done another day.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>DHB3 (araD139 Δ(ara-leu)7697 ΔlacX74 galE galK rpsL phoR Δ(phoA)PvuII ΔmalF3 thi)</td>
<td>Jon Beckwith Lab</td>
<td>n/a</td>
</tr>
<tr>
<td>FA41</td>
<td>Jon Beckwith Lab</td>
<td>n/a</td>
</tr>
<tr>
<td>FA41 F- λDE3 pET30a(+)::<i>trxA</i> (<i>Original host</i>)</td>
<td>Our lab</td>
<td>n/a</td>
</tr>
<tr>
<td>FA41 F- λDE3 pET30a(+)::<i>ancestral trxA</i> (<i>Engineered host</i>)</td>
<td>Our lab</td>
<td>n/a</td>
</tr>
<tr>
<td>
<i>Escherichia coli</i> bacteriophage T7</td>
<td>ATCC</td>
<td>BAA-1025-B2™</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>Fisher Bioreagents</td>
<td>BP153-1</td>
</tr>
<tr>
<td>NaCl</td>
<td>Merck</td>
<td>106404</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> · 7 H<sub>2</sub>O</td>
<td>Merck</td>
<td>105886</td>
</tr>
<tr>
<td>Bacto<sup>TM</sup> Yeast extract</td>
<td>BD</td>
<td>212750</td>
</tr>
<tr>
<td>Bacto<sup>TM</sup> Tryptone</td>
<td>Gibco</td>
<td>211705</td>
</tr>
<tr>
<td>Bacto<sup>TM</sup> Agar</td>
<td>BD</td>
<td>214010</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>Alfa Aesar</td>
<td>J61272</td>
</tr>
<tr>
<td>Chloroform</td>
<td>Sigma-Aldrich</td>
<td>C2432</td>
</tr>
<tr>
<td>Agarose</td>
<td>Bio-Rad</td>
<td>161–3101</td>
</tr>
<tr>
<td>Tris Base</td>
<td>Sigma-Aldrich</td>
<td>T6066</td>
</tr>
<tr>
<td>Boric Acid</td>
<td>Sigma-Aldrich</td>
<td>B7901</td>
</tr>
<tr>
<td>EDTA</td>
<td>Sigma-Aldrich</td>
<td>E5134</td>
</tr>
<tr>
<td>RedSafe™</td>
<td>iNtRON</td>
<td>21141</td>
</tr>
<tr>
<td>2-Log DNA Ladder</td>
<td>New England BioLabs</td>
<td>N0550S</td>
</tr>
<tr>
<td>Gel Loading Dye, Purple (6<b>×</b>), no SDS</td>
<td>New England Biolabs</td>
<td>B7025S</td>
</tr>
<tr>
<td>Isopropanol</td>
<td>Sigma-Aldrich</td>
<td>154970</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Phage DNA Isolation Kit</td>
<td>Norgen Biotek</td>
<td>Cat. #46800</td>
</tr>
<tr>
<td>λDE3 Lysogenization Kit</td>
<td>Novagen</td>
<td>69734-3</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>We recommend preparing buffers, plates, tubes, and media in advance. Phage dilution buffer and media must be autoclaved prior to use and stored at 25°C. Plates could be stored at 4°C to avoid contamination. Kanamycin stock solution is sterilized by filtration using 0.22 μm filters.</p>
<p>All manipulations involving bacteria and phage suspensions must be carried out at a laminar flow cabinet to guarantee a sterile environment. In addition, strict protocols for waste disposal should be followed according to the corresponding institution guidelines. In particular, biological waste and related laboratory material should be properly autoclaved before being disposed according to your local biosafety protocols.</p>
<ul>
<li>Buffer and media preparation: LB broth, LB agar, top agar, phage dilution buffer (10×)
<ul>
<li>Prepare as many LB agar plates as your experiment requires. Do not forget to add antibiotic if necessary, kanamycin at 30 μg/mL final concentration when using bacteria complemented with the pET30a(+) plasmid or derivatives.</li>
<li>Prepare as many molten top agar aliquots as your experiment requires. For instance, if ten serial dilutions of phage stock are prepared and used to infect the respective <i>E. coli</i> cultures (see <a href="#fig1">Figure 1</a>), then ten top agar aliquots are required in order to plate the mixtures. Also, additional top agar aliquots might be needed for the appropriate controls according to Note in step 7. Place the top agar aliquots in a water bath at 50°C to keep them melted while you wait for bacteria to grow.</li>
<li>Prepare your 1× phage dilution buffer stock from autoclaved 10× phage dilution buffer and dilute with autoclaved ddH<sub>2</sub>O.</li>
</ul>
</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Keep in mind how many phage dilutions are prepared in the plaque assay in order to estimate the volume of reagents to prepare in advance.</div>
<div class="note">
<span class="note-title">Note:</span> Set up the appropriate controls for the particular infection depending on whether it is done in liquid medium or on a plate and also depending on the bacterial strains used.</div>
<h3 id="sec2.1">Phage dilution buffer</h3>
<p>The following recipe is 10 times more concentrated (10×) than used:</p>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (1 M), pH 7.4</td>
<td>0.2 M</td>
<td>20 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>1 M</td>
<td>20 mL</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> · 7 H<sub>2</sub>O (1 M)</td>
<td>0.2 M</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td>Total</td>
<td>10×</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Tris-HCl may cause skin, eye or respiratory tract irritation. Avoid breathing dust. To avoid skin irritation, wear protective gloves and clothing. If there is any skin or eye contact, wash with water for several minutes.</div>
<table id="undtbl2">
<caption>LB Broth</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>0.17 M</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>0.5%</td>
<td>5 g</td>
</tr>
<tr>
<td>Tryptone</td>
<td>1%</td>
<td>10 g</td>
</tr>
<tr>
<td>dH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>LB agar</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>0.17 M</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>0.5%</td>
<td>5 g</td>
</tr>
<tr>
<td>Tryptone</td>
<td>1%</td>
<td>10 g</td>
</tr>
<tr>
<td>Agar</td>
<td>1.5%</td>
<td>15 g</td>
</tr>
<tr>
<td>dH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1 L</td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>Top agar</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>0.086 M</td>
<td>5 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>0.5%</td>
<td>5 g</td>
</tr>
<tr>
<td>Tryptone</td>
<td>1%</td>
<td>10 g</td>
</tr>
<tr>
<td>Agar</td>
<td>0.7%</td>
<td>7 g</td>
</tr>
<tr>
<td>dH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1 L</td>
</tr>
</tbody>
</table>
<h3 id="sec2.2">Kanamycin</h3>
<p>Stock solution prepared at 30 mg/mL and stored at −20°C, used at a final concentration of 30 μg/mL</p>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Kanamycin</td>
<td>30 mg/mL</td>
<td>0.3 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>10 mL</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Kanamycin may cause allergy skin or asthma symptoms; it also may difficult breathing, damage fertility or the unborn child. Wear protective gloves and avoid breathing dust.</div>
<h3 id="sec2.3">TBE</h3>
<p>The following recipe is 10 times more concentrated (10×) than used:</p>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris Base</td>
<td>0.89 M</td>
<td>108 g</td>
</tr>
<tr>
<td>Boric Acid</td>
<td>0.89 M</td>
<td>55 g</td>
</tr>
<tr>
<td>EDTA</td>
<td>0.02 M</td>
<td>7.5 g</td>
</tr>
<tr>
<td>dH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1 L</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Boric Acid may cause skin, eye or respiratory tract irritation. Avoid breathing dust. To avoid skin irritation, wear protective gloves and clothing. If there is any skin or eye contact, wash with water for several minutes.</div>
<div class="critical">
<span class="critical-title">Critical:</span> EDTA may cause respiratory tract irritation. Avoid breathing dust by using a mask. Wear protective gloves, eye shields and clothing. If there is any skin or eye contact, wash with water for several minutes</div>
<table id="undtbl7">
<caption>Agarose gel</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Agarose</td>
<td>0.5%</td>
<td>0.25 g</td>
</tr>
<tr>
<td>TBE</td>
<td>0.5×</td>
<td>50 mL</td>
</tr>
<tr>
<td>Red Safe</td>
<td>1×</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Before proceeding with the step-by-step protocol, please be aware that virus adaptation to a new host can be revealed through changes in different features such as number, size and morphology of the plaques when performing plaque assays, as will be described in detail. Furthermore, efficiency of infection, measured as the time required for complete lysis of a bacterial culture by the virus, is also a crucial feature to be monitored in virus adaptation studies and is actually incorporated in our protocol (see <a href="#fig2">Figure 2</a> for a schematic depiction of the complete experimental procedure):</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/967-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Bacteriophage T7 evolution experiments, lysis curve determinations, and genomic DNA extraction</div>
<p>Depiction of the general protocol for characterizing T7 adaptation to the <i>new</i> host, including pfu determinations based on plaque assays and lysis profiles of the evolved phages. In addition, a final step including DNA extraction allows for new generation sequencing of viral genome. Figure adapted from <a href="#bib8">Luzon-Hidalgo et al., 2021</a>.</p>
</figcaption>
</figure>
<h3 id="sec3.1">Bacteriophage T7 evolution experiments</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days per round of adaptation</div>
<p>Three parallel experiments are followed up in our evolution rounds (<a href="#fig3">Figure 3</a>): propagation of evolved T7 in the <i>engineered</i> and in the <i>original</i> hosts (panel A) and a control experiment in which wild type T7 infection in the <i>original</i> host is monitored (panel B). The general scheme of evolution experiments can be represented as follows:</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/967-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Description of infections performed in a typical evolution experiment</div>
<p>(A) The amplified T7 stock is used to infect the <i>new</i> host. The resulting evolved T7 suspension, V1, inoculates both the original and the engineered hosts. The new evolved T7 suspension, V2, is subjected to a new round of evolution.</p>
<p>(B) Control experiment to monitor the infection of <i>original</i> host with wild type T7.</p>
</figcaption>
</figure>
<p class="math-ml"><math> <mrow> <mi>V</mi> <mn>0</mn> <mo>⇒</mo> <mi>V</mi> <mn>1</mn> <mover> <mo>→</mo> <mrow> <mi>R</mi> <mn>1</mn> </mrow> </mover> <mi>V</mi> <mn>2</mn> <mover> <mo>→</mo> <mrow> <mi>R</mi> <mn>2</mn> </mrow> </mover> <mi>V</mi> <mn>3</mn> <mover> <mo>→</mo> <mrow> <mi>R</mi> <mn>3</mn> </mrow> </mover> <mi>V</mi> <mn>4</mn> <mover> <mo>→</mo> <mrow> <mi>R</mi> <mn>4</mn> </mrow> </mover> <mspace></mspace> <mo>⋯</mo> <mo>⋯</mo> </mrow> </math></p>
<ol>
<li>Amplified phage stock from step 4d of <a href="#before-you-begin">before you begin</a> section, V0, is actually the starting point for evolution experiments. The titer should be within the 10<sup>9</sup>–10<sup>11</sup> pfu/mL range, as quantified according to step 8b of the same section.</li>
<li>Let the wild type virus sample infect the <i>engineered</i> host (<i>E. coli Trx-</i> complemented with a plasmid encoding for the ancestral thioredoxin). To this purpose, a plaque assay as described in steps 5–7 of <a href="#before-you-begin">before you begin</a> section should be carried out. Thus, successive ten-fold dilutions of phage suspension are used to inoculate our <i>engineered</i> host culture following the same protocol described previously.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Typically, high wild-type phage dilutions, above 10<sup>−3</sup>–10<sup>−4</sup>, do not infect efficiently the <i>engineered</i> host in the first/second rounds. Therefore, plating those dilutions might not be required at this point since a bacterial lawn will develop, and no plaques will be observed.</div>
<div class="note">
<span class="note-title">Note:</span> Do not forget to add kanamycin to LB media, 30 μg/mL final concentration, as the <i>engineered</i> host strain carries antibiotic resistance gene.</div>
<div class="note">
<span class="note-title">Note:</span> A number of routine controls should be set up including: control to verify LB broth + kanamycin sterility (no growth should be observed after overnight or 15 hours incubation of media); control for <i>engineered</i> host growth in the absence of infection (a typical bacterial growth curve is expected to be obtained); and a negative control of phage infection using FA41 cured cells without plasmid complementation (in the absence of thioredoxin, T7 infection should not be successful).</div>
<ol start="3">
<li>For this first infection, plaques show up only occasionally at the highest phage concentrations. Thus, it is advisable to set around 10 plate replicates for those concentrations to assure obtaining a minimum number of plaques. Incubate the plates overnight or for 15 h.</li>
<li>Proceed with pfu quantification as explained in step 8 of <a href="#before-you-begin">before you begin</a> section.</li>
<li>Plaques size is remarkably small, reflecting the fact that propagation at this point (double-line arrow in the previous scheme) is inefficient. Pick a single, well-isolated plaque (the bigger the better) by using an inoculation loop. Then resuspend the plaque into a 1.5 mL tube containing 1 mL of phage dilution buffer, vortex gently and let it stand at 25°C for two hours, to allow phage particles to diffuse towards the solution (<a href="#fig4">Figure 4</a>). This first plaque suspension actually corresponds to phage suspension V1 in previous scheme and <a href="#fig2">Figures 2</a> and <a href="#fig3">3</a>A. It is obtained from the initial propagation of wild type T7 in the <i>new</i> host and is used for the first evolution round R1 (scheme, <a href="#fig2">Figures 2</a> and <a href="#fig3">3</a>A).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/967-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Schematic representation for phage stock preparation from a plaque</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To remove the plaque from the plate, use a thin inoculation loop to carefully excise the actual piece of soft agar (solidified on top of the standard agar plate) containing the plaque. It should come out easily after gently stabbing the top agar around the plaque.</div>
<div class="note">
<span class="note-title">Note:</span> If you are not going to use the phage immediately after the two-hour diffusion step, keep the suspension at 4°C for the day or freeze it.</div>
<div class="note">
<span class="note-title">Note:</span> V1 could be also used as starting point for lysis profile determinations (see Determination of lysis profile for the evolved phage section below).</div>
<div class="note">
<span class="note-title">Note:</span> Infection of V0 with the <i>original</i> host (FA41 cells complemented with <i>E. coli</i> thioredoxin) must be also carried out in parallel, as mentioned before (<a href="#fig3">Figure 3</a>B). This control is crucial in order to extract future conclusions regarding the performance of wild type and evolved phages. To this purpose, repeat steps 2 to 5 of this section.</div>
<div class="note">
<span class="note-title">Note:</span> Plaques resulting from the first infection of the <i>engineered</i> host are clearly smaller and display a different morphology when compared with those from the <i>original</i> host. In particular, the typical circular shape is not so well defined, presenting irregular borders.</div>
<ol start="6">
<li>First round of phage evolution, R1, is initiated with standard plaque assays, as previously described, where V1 stock infects both, the <i>engineered</i> host and the <i>original</i> host, in parallel experiments (see <a href="#fig3">Figure 3</a>A). We are interested in following up infectivity of evolved phages in the two hosts to gain experimental evidence on the virus adaptation mechanism.</li>
</ol>
<p>Take 100 μL directly from the top agar suspension V1 and prepare serial dilutions as indicated in step 6 of <a href="#sec1.2">phage plaque assay for titer determination</a> section.</p>
<ol start="7">
<li>Perform the two plaque assays in parallel, using the different phage dilutions to inoculate the <i>engineered</i> and the <i>original</i> hosts, respectively, according to steps 7a and 7b of the cited section.</li>
<li>Plate the resulting suspensions (steps 7c, 7d).</li>
<li>Quantify pfu/mL (step 8 of <a href="#sec1.2">phage plaque assay for titer determination</a> section) for both infections. The comparison between pfu/mL results for both infections is a measurement of the capability of the evolved virus to propagate in the two hosts.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Take pictures of plates to compare morphology and size of plaques.</div>
<ol start="10">
<li>The new evolved phage suspension, V2, for the next round of evolution is obtained from an appropriate plaque of the <i>engineered</i> host infection, following step 5 (<a href="#fig4">Figure 4</a>) in the current section.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Evolved phage stock V2 is used for lysis profiles determinations (<a href="#fig2">Figure 2</a>), according to next section.</div>
<ol start="11">
<li>Repeat as many evolution rounds as required.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A second round of evolution for the wild type phage (control experiment) infecting the <i>original</i> host should be also performed in parallel (<a href="#fig3">Figure 3</a>B).</div>
<ol start="12">
<li>Plot log<sub>10</sub> pfu/mL values <i>versus</i> rounds of evolution. As shown in <a href="#fig5">Figure 5</a>, after the second round adaptation of evolved phage to the <i>engineered</i> host takes place. Furthermore, the size and the number of plaque forming units increase reaching typical values measured for the infection of the <i>original</i> host. This surprising result reveals not only a fast adaptation of the phage but also a promiscuous adaptation, meaning that infective virions from evolved phage sample are capable of replicating in both, the engineered and the original hosts.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/967-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Results of plaque assays over several rounds of laboratory evolution</div>
<p>(A) Pictures showing the increase in size and number of plaques along with rounds of adaptation of T7 to the <i>engineered</i> host.</p>
<p>(B) Number of plaque forming units for virus samples from two laboratory evolution experiments, using the <i>engineered</i> host (red) and the <i>original</i> host (blue). A control experiment where wild type T7 inoculates the original host is also shown (black). Figure taken from <a href="#bib8">Luzon-Hidalgo et al., 2021</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Results presented in <a href="#fig5">Figure 5</a>B for two laboratory evolution experiments demonstrate host promiscuity of evolved T7 but they do not provide experimental evidence regarding the efficiency of virus replication. To shed light into this relevant aspect, determination of lysis profiles is also integrated in our protocol for assessment of virus evolution (<a href="#fig2">Figure 2</a>).</div>
<h3 id="sec3.2">Determination of the lysis profile for the evolved phage</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day per round of adaptation</div>
<p>Pfu determinations based on plaque assays shed light on the capability of virion particles to propagate in a particular host although do not provide information on virus replication efficiency. Thus, we aim to address this issue by integrating in our experimental protocol, combined with plaque assays, the characterization of lysis profiles in liquid medium. Please note that both, plaque assays and lysis profile determinations, can be carried out alongside adaptation experiments as indicated in <a href="#fig2">Figure 2</a>:</p>
<ol start="13">
<li>Leave a 5 mL bacterial pre-inoculum to grow overnight or for 15 h at 37°C, with shaking.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In fact, two parallel bacterial cultures corresponding to the <i>engineered</i> and the <i>original</i> hosts should be grown to be inoculated by the evolved T7 and the wild type T7, following the general scheme in <a href="#fig6">Figure 6</a>.</div>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/967-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Determination of the lysis profile for the evolved phage</div>
<p>Upper panel: Schematic representations of the different infections carried out. Control experiments for negative infection of thioredoxin minus cells (knock-out) with T7 are also included. Lower panel: experimental profiles of turbidity versus time obtained for each infection, according to color code. Figures adapted from <a href="#bib8">Luzon-Hidalgo et al., 2021</a>.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> In addition, a pre-inoculum of the knock-out thioredoxin minus cells (FA41 F- strain without plasmid complementation) should be grown to be used as a control of negative infection.</div>
<ol start="14">
<li>Transfer 350 μL of the overnight culture into 70 mL of fresh LB broth (1/200 dilution), with antibiotic if necessary, and let bacteria grow until an OD<sub>600nm</sub> of 0.25 is reached (typically after 2–3 h approximately).</li>
<li>Split up the bacterial culture into three tubes (20 mL each): one will be inoculated with evolved T7, another one with wild type T7 and the last one is set as a bacterial growth control (in the absence of virus).</li>
<li>Phage infection starts when a volume of 150 μL of the corresponding phage stock is added to the bacterial culture from step 14.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For the evolved phage infection, use suspension V1 corresponding to the first round of viral adaptation.</div>
<ol start="17">
<li>Leave the infection to proceed at 37°C with shaking.</li>
<li>Take culture aliquots to monitor turbidity, OD<sub>600nm</sub>, as a function of time. The decrease in absorbance is taken as a measurement of cell lysis. Typically, measurements are carried out every 10, 15 or 30 min depending on the infectivity of the phage, until complete lysis.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is advisable to prepare in advance a worksheet to keep track of raw data corresponding to parallel experiments, in an organized manner. See <a href="#tbl1">Table 1</a>, as an example:</div>
<table id="tbl1">
<caption>Table 1. Example of worksheet for lysis profiles measurements</caption>
<thead>
<tr>
<th colspan="4">
<i>Original</i> host</th>
<th colspan="4">
<i>Engineered</i> host</th>
</tr>
<tr>
<th rowspan="2">Time (min)</th>
<th colspan="3">OD<sub>600</sub>
</th>
<th rowspan="2">Time (min)</th>
<th colspan="3">OD<sub>600</sub>
</th>
</tr>
<tr>
<th>Bacterial growth control</th>
<th>Wild-type T7</th>
<th>Evolved T7</th>
<th>Bacterial growth control</th>
<th>Wild-type T7</th>
<th>Evolved T7</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>0.462</td>
<td>0.311</td>
<td>0.285</td>
<td>0</td>
<td>0.259</td>
<td>0.259</td>
<td>0.259</td>
</tr>
<tr>
<td>30</td>
<td>0.625</td>
<td>0.561</td>
<td>0.571</td>
<td>30</td>
<td>0.692</td>
<td>0.562</td>
<td>0.663</td>
</tr>
<tr>
<td>35</td>
<td> </td>
<td>0.545</td>
<td>-</td>
<td>60</td>
<td>0.961</td>
<td>0.849</td>
<td>0.928</td>
</tr>
<tr>
<td>40</td>
<td>0.761</td>
<td>0.595</td>
<td>0.656</td>
<td>90</td>
<td>1.070</td>
<td>1.090</td>
<td>1.150</td>
</tr>
<tr>
<td>50</td>
<td> </td>
<td>0.056</td>
<td>0.707</td>
<td>120</td>
<td>1.150</td>
<td>1.180</td>
<td>1.220</td>
</tr>
<tr>
<td>60</td>
<td>0.933</td>
<td>0.065</td>
<td>0.973</td>
<td>170</td>
<td>-</td>
<td>-</td>
<td>1.170</td>
</tr>
<tr>
<td>70</td>
<td>-</td>
<td>-</td>
<td>0.944</td>
<td>180</td>
<td>1.320</td>
<td>1.330</td>
<td>1.110</td>
</tr>
<tr>
<td>80</td>
<td>-</td>
<td>-</td>
<td>1.050</td>
<td>190</td>
<td>-</td>
<td>-</td>
<td>0.745</td>
</tr>
<tr>
<td>90</td>
<td>1.060</td>
<td>-</td>
<td>1.140</td>
<td>200</td>
<td>-</td>
<td>-</td>
<td>0.331</td>
</tr>
<tr>
<td>110</td>
<td>-</td>
<td>-</td>
<td>0.885</td>
<td>210</td>
<td>1.340</td>
<td>1.360</td>
<td>0.394</td>
</tr>
<tr>
<td>120</td>
<td>1.150</td>
<td>-</td>
<td>0.343</td>
<td>220</td>
<td>-</td>
<td>-</td>
<td>0.263</td>
</tr>
<tr>
<td>140</td>
<td>-</td>
<td>-</td>
<td>0.181</td>
<td>240</td>
<td>1.360</td>
<td>1.430</td>
<td>0.377</td>
</tr>
<tr>
<td>150</td>
<td>1.250</td>
<td>-</td>
<td>0.156</td>
<td colspan="4"> </td>
</tr>
</tbody>
</table>
<ol start="19">
<li>Plot OD<sub>600</sub> values versus time for each infection. Typical lysis profiles are shown in <a href="#fig6">Figure 6</a>, lower panel.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To avoid handling large volumes, microtiter plates can be used to monitor the lysis profiles. If so, please be aware that this may prevent the extraction of enough phage genomic DNA for NGS.</div>
<ol start="20">
<li>Once the cell lysis is completed, you can freeze these samples or extract phage DNA (see next section).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point samples may be stored at −20°C and continue with DNA extraction another day.</div>
<ol start="21">
<li>Repeat the above protocol (steps 13–20) to monitor lysis profiles for evolved phage suspensions V2, V3, V4, etc. obtained from successive evolution rounds (see <a href="#fig2">Figure 2</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When performing lysis experiments for the first time, we recommend doing a “quick and dirty” experiment to estimate the time frame for complete lysis. Otherwise, you can run out of culture if too many turbidity measurements are taken.</div>
<div class="note">
<span class="note-title">Note:</span> Avoid bacterial debris when you withdraw media to make your absorbance measurements.</div>
<h3 id="sec3.3">Phage genomic DNA extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day per round of adaptation</div>
<p>A step forward can be taken in order to tentatively elucidate the molecular bases for virus adaptation. We include here the protocol for genomic DNA extraction from lysed <i>engineered</i> host samples to be used for Illumina next generation sequencing (<a href="#fig2">Figure 2</a>):</p>
<ol start="22">
<li>Add 0.1 mL of chloroform to 6 mL of virus lysate from step 20, at 25°C (thaw on ice if necessary). Vortex the mixture. Chloroform helps to separate DNA from bacterial lipids and proteins.</li>
<li>Centrifuge at 4000 <i>g</i>, 10 min, 4°C.</li>
<li>Take 1 mL of the supernatant to extract viral DNA and freeze the rest.</li>
<li>We use the phage DNA Isolation Kit from Norgen Biotek Corp. and follow its instructions. In the last step, DNA sample is eluted in 50 μL of water. To evaporate possible traces of organic solvents from the kit that might interfere with the sequencing process, leave the sample open at 50°C for 30 min. Typical yield obtained is within 3–15 μg DNA when starting from a 10<sup>6</sup>–10<sup>10</sup> pfu/mL enriched phage solution, in good agreement with specifications of the commercial kit. Moreover, the quality of the resulting DNA sample is appropriate for Illumina new generation sequencing of phage genome. Absorbance measurements were taken using a Nanodrop One spectrophotometer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Please, be aware that the resulting phage DNA sample (50 μL of a concentration within 50–300 μg/mL) is extracted from 6 mL of virus lysate obtained according to steps 16–20.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Lysis Buffer B from DNA Isolation Kit may cause eye or skin irritation and may be harmful if swallowed. Wear protective gloves, eye shields and clothing. If there is any skin or eye contact, wash with water for several minutes.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point samples may be stored at −20°C and continue with agarose gel electrophoresis another day.</div>
<ol start="26">
<li>Finally, the purity of the phage DNA is assessed by 0.5% agarose gel electrophoresis. Mix 10 μL of DNA sample with 2 μL of loading dye and load 12 μL in each well. Run the gel at 100 V.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Include an appropriate molecular weight ladder into the first lane of the gel. T7 DNA is ∼40 Kb long (<a href="#fig7">Figure 7</a>).</div>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/967-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Phage genomic DNA extraction</div>
<p>Agarose gel corresponding to phage DNA extracted from lysed engineered host samples and subsequently used for Illumina next generation sequencing.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Number of plaque forming units and lysis profiles over several rounds of laboratory evolution can be determined providing a detailed phenomenological description of the process of viral adaptation to the new host. In addition, purified DNA can be further used for Next Generation Sequencing of the viral genome over the several rounds of laboratory evolution, allowing the sequence- and molecular-determinants of virus adaptation to be explored and potentially determined.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>One potential limitation of our protocol could come from the actual amount of active virus particles present in the evolved phage suspensions used either for plaque assays or lysis measurements. Results might be biased in case a lower phage concentration is used for infection. Thus, in plaque assays special care must be taken in picking up plaques of similar size, if possible, to prepare the V0, V1, V2, etc, suspensions to be used for next adaptation round. The same applies to lysis measurements as larger lysis times could be due to the presence of a lower amount of active virus particles instead of a worse adaptation to the host.</p>
<p>It is recommended performing parallel experiments, infecting simultaneously the <i>engineered</i> and the <i>original</i> hosts in order to obtain comparative measurements of capability and efficiency of infection. Likewise, a number of replicates of experimental data is mandatory to get reliable results.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>(Related to step: Phage Plaque Assay for Titer Determination)</p>
<p>Low titer of phage stock suspension.</p>
<h3 id="sec6.2">Potential Solution</h3>
<p>Repeat the amplification step as many times as needed.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>(Related to step: Bacteriophage T7 Evolution Experiments)</p>
<p>No plaques found on plates the first time phage propagates in the <i>new</i> host.</p>
<h3 id="sec6.4">Potential Solution</h3>
<p>Set up a higher number of replicates when plating infections corresponding to high phage concentrations in serial dilutions.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>(Related to step: Phage Plaque Assay for Titer Determination)</p>
<p>In plaque assays, either confluent plaques or very low number of plaques appear.</p>
<h3 id="sec6.10">Potential Solution</h3>
<p>Optimization of phage dilutions range is required.</p>
<h3 id="sec6.6">Problem 4</h3>
<p>(Related to step: Determination of the Lysis Profile for the evolved phage)</p>
<p>Large lysis times in liquid medium.</p>
<h3 id="sec6.7">Potential Solution</h3>
<p>Optimize the time between OD measurements to avoid running out of cell/phage suspension before lysis is completed.</p>
<h3 id="sec6.8">Problem 5</h3>
<p>(Related to step: Determination of the Lysis Profile for the evolved phage)</p>
<p>Noisy lysis time profiles.</p>
<h3 id="sec6.9">Potential Solution</h3>
<p>The quality of the absorbance profiles might be improved, if needed, if the measurements are performed automatically in a plate-reader spectrophotometer, using well-plates.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Adams, M.H. (1959). Bacteriophages (Interscience Publisher Inc). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref1">View at publisher</a></p>
<p id="bib2">Boyd, D., Manoil, C., and Beckwith, J. (1987). Determinants of membrane protein topology. Proc. Natl. Acad. Sci. U S A <i>84</i>, 8525-8529. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref2">View at publisher</a></p>
<p id="bib3">Bragg, R., van der Westhuizen, W., Lee, J.Y., Coetsee, E., and Boucher, C. (2014). Bacteriophages as potential treatment option for antibiotic resistant bacteria. In Infectious diseases and nanomedicine I. Advances in experimental medicine and biology, <i>807</i>, R. Adhikari and S. Thapa, eds. (Springer). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref3">View at publisher</a></p>
<p id="bib4">Chan, L.Y., Kosuri, S., and Endy, D. (2005). Refactoring bacteriophage T7. Mol. Syst. Biol. <i>1</i>, 2005.0018. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref4">View at publisher</a></p>
<p id="bib5">Delgado, A., Arco, R., Ibarra-Molero, B., and Sanchez-Ruiz, J.M. (2017). Using resurrected ancestral proviral proteins to engineer virus resistance. Cell Rep. <i>19</i>, 1247-1256. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref5">View at publisher</a></p>
<p id="bib6">Etson, C.M., Hamdan, S.M., Richardson, C.C., and van Oijen, A.M. (2010). Thioredoxin suppresses microscopic hopping of T7 DNA polymerase on duplex DNA. Proc. Natl. Acad. Sci. U S A <i>107</i>, 1900-1905. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref6">View at publisher</a></p>
<p id="bib7">Hamdan, S.M. and Richardson, C.C. (2009). Motors, switches and contacts in the replisome. Annu. Rev. Biochem. <i>78</i>, 205-243. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref7">View at publisher</a></p>
<p id="bib8">Luzon-Hidalgo, R., Risso, V.A., Delgado, A., Andrés-León, E., Ibarra-Molero, B., and Sanchez-Ruiz, J.M. (2021). Evidence for a role of phenotypic mutations in virus adaptation. iScience <i>24</i>, 102257. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref8">View at publisher</a></p>
<p id="bib9">Madigan, M.T., Bender, K.S., Buckley, D.H., Sattley, W.M., and Stahl, D.A. (2019). Brock Biology of Microorganisms, Fifteenth Edition (Pearson Education Limited). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00490-1/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Jose M. Sanchez-Ruiz (<a href="mailto:sanchezr@ugr.es">sanchezr@ugr.es</a>).</p>
<h4>Materials availability</h4>
<p>All strains generated in this study are available upon request. This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by Spanish Ministry of Economy and Competitiveness/<a href="https://doi.org/10.13039/501100002924">FEDER</a> Funds Grant RTI2018-097142-B-100 and by <a href="https://doi.org/10.13039/100004412">Human Frontier Science Program</a> Grant RGP0041/2017. We also thank Dr. John Beckwith and Dr. Dana Boyd (Harvard University) for kindly providing knockout strains used in this work.</p>
<h3>Author contributions</h3>
<p>Conceptualization: B.I.-M., V.A.R., and J.M.S.-R.; methodology: A.D., R.L.-H., and V.A.R.; experiments: R.L.-H.; formal analyses: R.L.-H., B.I.-M., and V.A.R.; writing - original draft: B.I.-M.; writing - review and editing: all authors; supervision and funding acquisition: J.M.S.-R.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>