<article>
<section>
<h1 id="header">Generation and isolation of single cells from mouse brain with mosaic analysis with double markers-induced uniparental chromosome disomy</h1>
<p><time datetime="2020-12-18">Published: December 18, 2020</time></p>
<p>Susanne Laukoter,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Nicole Amberg,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Florian M. Pauler,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Simon Hippenmeyer<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Science and Technology Austria, Am Campus 1, 3400 Klosterneuburg, Austria</p>
<p id="fn1"><sup>2</sup>Technical Contact</p>
<p id="fn2"><sup>3</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:simon.hippenmeyer@ist.ac.at">simon.hippenmeyer@ist.ac.at</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100215">10.1016/j.xpro.2020.100215</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Mosaic analysis with double markers (MADM) technology enables concomitant fluorescent cell labeling and induction of uniparental chromosome disomy (UPD) with single-cell resolution. In UPD, imprinted genes are either overexpressed 2-fold or are not expressed. Here, the MADM platform is utilized to probe imprinting phenotypes at the transcriptional level. This protocol highlights major steps for the generation and isolation of projection neurons and astrocytes with MADM-induced UPD from mouse cerebral cortex for downstream single-cell and low-input sample RNA-sequencing experiments.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib10">Laukoter et al. (2020b)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Generation of cells with uniparental chromosome disomy (UPD) using MADM technology</li>
<li>Isolation of cells with MADM-induced UPD for single-cell downstream applications</li>
<li>Analysis of cortical cells with MADM-induced UPD using scRNA-seq</li>
<li>Analysis of neurons and astrocytes with MADM-induced UPD to probe genomic imprinting</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/333-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Background</h3>
<p>Genomic imprinting is an epigenetic phenomenon in mammalian genomes that results in monoallelic parent-of-origin dependent gene expression. Therefore, certain genes are only expressed from the paternally inherited allele and others only from the maternally inherited allele. A key feature of imprinted genes is their imperative dosage-sensitivity. Uniparental chromosome disomy (UPD, somatic cells containing either two maternal or two paternal chromosomes) has previously been used to probe genomic imprinting mainly in whole animal, and revealed major principles of genomic imprinting (<a href="#bib13">Tucci et al., 2019</a>).</p>
<p>MADM is a genetic technology that allows concomitant fluorescent labeling and induction of UPD with unprecedented single-cell resolution in a defined genetic lineage (<a href="#bib5">Contreras et al., 2020</a>, <a href="#bib7">Hippenmeyer et al., 2013</a>, <a href="#bib9">Laukoter et al., 2020a</a>, <a href="#bib10">Laukoter et al., 2020b</a>). MADM is based on the use of split marker genes that consist of partial coding sequences for green fluorescent protein (GFP) and tandem dimer Tomato (TdT), interspersed by LoxP sites. MADM requires crossing of two different mouse lines, the GT (N-GFP/C-TdT) line and the TG (N-TdT/C-GFP) line, one of which also carries a Cre driver of interest, for the generation of experimental animals (<a href="#fig1">Figures 1</a>A and 1B).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/333-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Experimental MADM paradigm for generating UPD with single-cell resolution</div>
<p>MADM utilizes Cre/LoxP-mediated interchromosomal recombination to reconstitute two reciprocally chimeric marker genes (GT/TG). The two reciprocally chimeric marker genes are targeted separately to identical loci on homologous chromosomes. The marker genes consist of partial coding sequences for green (GFP, G) and red (TdT, T) fluorescent proteins separated by an intron containing a LoxP site. Recombination in G2 phase of the cell cycle followed by X-Segregation (the two recombinant chromosomes are transmitted to distinct daughter cells) leads to individual green (GFP<sup>+</sup>) and red (TdT<sup>+</sup>) labeled cells, respectively, with near complete uniparental chromosome disomy (UPD). Recombination in G2 phase of the cell cycle with segregation of both recombinant chromosomes (including fully reconstituted TdT and GFP marker genes) to the same daughter cell (Z-Segregation) does not alter the genotype and results in one yellow cell which serves as control. Z-Segregation at the same time also gives rise to an unlabeled cell (unchanged genotype).</p>
<p>(A) MADM paradigm to generate UPD using forward crossing scheme: If the TG MADM cassette is inherited from the father and the GT MADM cassette from the mother as indicated, cells labeled in red show unimaternal chromosome disomy (MM; matUPD) and green cells unipaternal chromosome disomy (PP, patUPD). Consequently, maternally expressed genes (MAT, shown in red) are expressed at the double dose and paternally expressed genes are not expressed in red matUPD cells. In contrast, green cells with patUPD express paternally expressed genes (PAT, shown in blue) at twice the normal dose and maternally expressed genes are not expressed. Thus, imprinted genes are differentially expressed depending on the UPD. Biallelically expressed genes (BAE, non-imprinted genes, shown in black) are not affected.</p>
<p>(B) MADM paradigm to generate UPD using reverse crossing scheme. Here, the TG MADM cassette is inherited from the mother and the GT MADM cassette from the father. Thus, matUPD (2x maternally expressed genes, 0x paternally expressed genes) are labeled in green (GFP<sup>+</sup>) and patUPD (2x paternally expressed genes, 0x maternally expressed genes) are labeled in red (TdT<sup>+</sup>). Symbols are detailed in the key.</p>
<p>Parts of the figure and its legend are reused but adapted from previous publications (<a href="#bib7">Hippenmeyer et al., 2013</a>, <a href="#bib8">Hippenmeyer et al., 2010</a>, <a href="#bib10">Laukoter et al., 2020b</a>) with permission.</p>
</figcaption>
</figure>
<p>Cells with MADM-induced UPD are generated by Cre/LoxP-dependent mitotic recombination during the G2 phase in dividing stem cells followed by X-segregation. G2-X events produce near complete UPD of a particular chromosome carrying the MADM cassettes (<a href="#fig1">Figures 1</a>A and 1B). Here, we established an experimental pipeline for the isolation of MADM-labeled cells carrying UPD for downstream applications, including small sample and single-cell RNA-sequencing (scRNAseq). Our protocol is based on the use of <i>Emx1</i>-Cre driver in combination with MADM cassettes targeted to chr. 7, which harbors many imprinted genes, but can be applied to any chromosome with MADM cassettes (<a href="#bib5">Contreras et al., 2020</a>).</p>
<p>In cells with MADM-induced UPD, imprinted genes located on the chromosome carrying the MADM cassettes are either homozygosed to be expressed in double dose or are not expressed. MADM-induced UPD thus provide a functional approach to simultaneously test loss-of-function and gain-of-function of imprinted gene dosage. Cells with MADM-induced UPD are visualized with distinct fluorescent colors, e.g., patUPD cells labeled in green (GFP<sup>+</sup>) and matUPD cells labeled in red (tdT<sup>+</sup>) (<a href="#fig1">Figure 1</a>A) (or vice versa (<a href="#fig1">Figure 1</a>B), also outlined in breeding strategy in <a href="#fig2">Figures 2</a>A and 2B). Importantly, the dosage of imprinted gene expression is predictable in MADM-induced UPD. Thus, ensuing cellular and transcriptional phenotypes, originating from imbalanced imprinted gene expression, can be analyzed in a cell-type-specific manner. MADM-induced UPD thus provide a unique functional assay to probe genomic imprinting with single-cell resolution.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/333-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Breeding strategy for experimental mice</div>
<p>(A) Breeding strategy A illustrates the generation of experimental MADM mice for sorting of cells originating from a genetically defined lineage. Experimental mice contain red (TdT<sup>+</sup>) cells with unimaternal disomy (matUPD: 2x maternally expressed genes, 0x paternally expressed genes). Green (GFP<sup>+</sup>) cells carry unipaternal disomy (patUPD: 2x paternally expressed genes, 0x maternally expressed genes).</p>
<p>(B) Breeding strategy B illustrates reversal of fluorescent colors in cells with MADM-induced UPD. In this setup green (GFP<sup>+</sup>) cells contain matUPD and red (TdT<sup>+</sup>) cells carry patUPD.</p>
<p>(C) Breeding strategy C shows the generation of experimental MADM-LacZ mice for astrocyte sorting. Astrocytes are identified based on LacZ expression under the control of the human <i>GFAP</i> promoter. GFP<sup>+</sup>, LacZ<sup>+</sup> astrocytes harbor matUPD, while TdT<sup>+</sup>, LacZ<sup>+</sup> positive cells carry patUPD.</p>
<p>(D) Breeding strategy D indicates reversal of fluorescent colors in cells with MADM-induced UPD in experimental animals for MADM+LacZ astrocyte sorting. TdT<sup>+</sup>, LacZ<sup>+</sup> cells carry matUPD and GFP<sup>+</sup>, LacZ<sup>+</sup> cells harbor patUPD.</p>
<p>Parts of the figure and its legend are reused but adapted from previous publications (<a href="#bib7">Hippenmeyer et al., 2013</a>, <a href="#bib8">Hippenmeyer et al., 2010</a>, <a href="#bib10">Laukoter et al., 2020b</a>) with permission.</p>
</figcaption>
</figure>
<p>For more details on the MADM technology, please refer to (<a href="#bib1">Beattie et al., 2017</a>, <a href="#bib2">Beattie et al., 2020</a>, <a href="#bib5">Contreras et al., 2020</a>, <a href="#bib7">Hippenmeyer et al., 2013</a>, <a href="#bib8">Hippenmeyer et al., 2010</a>, <a href="#bib9">Laukoter et al., 2020a</a>, <a href="#bib10">Laukoter et al., 2020b</a>, <a href="#bib14">Zong et al., 2005</a>).</p>
<h3 id="sec1.2">Generation of experimental MADM mice for FACS isolation of cells with MADM-induced UPD</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 days and up to 9 weeks</div>
<ol>
<li>Cross <i>MADM-7</i><sup>TG/TG</sup> animals with <i>MADM-7</i> <sup>GT/GT</sup><i>;Emx1</i><sup>Cre/+</sup> animals (<a href="#fig2">Figures 2</a>A and 2B).
<div class="note">
<span class="note-title">Note:</span> Breeding strategy A: cross <i>MADM-7</i><sup>TG/TG</sup> males (homozygous for the MADM cassette that contains the N-terminal domain of TdT and C-terminal domain of GFP) with <i>MADM-7</i><sup>GT/GT</sup><i>;Emx1</i><sup>Cre/+</sup> females (homozygous for the MADM cassette that contains the N-terminal domain of GFP and C-terminal domain of TdT). In the experimental animals (F1 generation), red cells (TdT<sup>+</sup>) will carry unimaternal disomy (matUPD; double dose of maternally expressed genes, no expression of paternally expressed genes) and green cells (GFP<sup>+</sup>) will contain unipaternal disomy (patUPD; 2x expression dose of paternally expressed genes and no expression of maternally expressed genes).</div>
Breeding strategy B: Fluorescent colors of UPD cells are swapped. Reversing fluorescent colors in cells with UPD serves as control for potential effects resulting from the expression of the fluorescent marker genes themselves. Cross <i>MADM</i><sup>TG/TG</sup> females with <i>MADM</i><sup>GT/GT</sup><i>;Emx1</i><sup>Cre/+</sup> males. In the F1 experimental generation red cells contain patUPD (2x expression dose of paternally expressed genes and no expression of maternally expressed genes) and green cells carry matUPD (2x expression dose of maternally expressed genes and no expression of paternally expressed genes).
<ol type="a">
<li>Check for vaginal plug to monitor successful mating and to facilitate the experimental planning.</li>
<li>Males and females of the F1 generation can be used as experimental animals (genotype is <i>MADM-7</i><sup>GT/TG</sup><i>, Emx1</i><sup>Cre/+</sup>).</li>
<li>Perform experiment at developmental stage(s) of choice.
<div class="note">
<span class="note-title">Note:</span> Experimental animals can be used at embryonic stages, P0, early postnatal stages and up to adulthood (P21, P42). Here we made use of <i>Emx1</i>-Cre that primarily targets radial glial progenitors in the dorsal forebrain, which give rise to excitatory projection neurons and glial cells in cerebral cortex (<a href="#bib6">Gorski et al., 2002</a>). Distinct proportions of these cortical cell types are present at different developmental time points. For details refer to <a href="#tbl1">Table 1</a>.</div>
<table id="tbl1">
<caption>Table 1. Developmental time point and corresponding cell types present in <i>Emx1</i>-lineage</caption>
<thead>
<tr>
<th>Time point</th>
<th>Radial glial progenitor cells</th>
<th>Astrocyte intermediate progenitors</th>
<th>Excitatory projection neurons</th>
<th>Astrocytes</th>
<th>Oligodendrocytes</th>
<th>Olfactory bulb neuroblasts</th>
</tr>
</thead>
<tbody>
<tr>
<td>E15</td>
<td>fraction</td>
<td>–</td>
<td>majority</td>
<td>–</td>
<td>–</td>
<td>fraction</td>
</tr>
<tr>
<td>P0</td>
<td>fraction</td>
<td>fraction</td>
<td>majority</td>
<td>fraction</td>
<td>fraction</td>
<td>fraction</td>
</tr>
<tr>
<td>P7</td>
<td>–</td>
<td>–</td>
<td>majority</td>
<td>fraction</td>
<td>fraction</td>
<td>fraction</td>
</tr>
<tr>
<td>P14</td>
<td>–</td>
<td>–</td>
<td>majority</td>
<td>fraction</td>
<td>fraction</td>
<td>fraction</td>
</tr>
<tr>
<td>P21</td>
<td>–</td>
<td>–</td>
<td>majority</td>
<td>fraction</td>
<td>fraction</td>
<td>fraction</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Here we used MADM cassettes targeted to chr. 7. In principle the protocol for isolation of cells with MADM-induced UPD can be applied to any MADM chromosome (<a href="#bib5">Contreras et al., 2020</a>). Besides <i>Emx1-</i>Cre<i>,</i> any other Cre driver can be used, provided Cre is expressed in dividing stem and progenitor cells, to target cell types and lineages of interest. Depending on the use of specific chr. with targeted MADM cassettes and the particular Cre driver, interchromosomal recombination and MADM-labeling efficiencies in experimental animals might vary.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Generation of experimental MADM mice for FACS isolation of astrocytes with MADM-induced UPD</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 weeks and up to 5 weeks</div>
<div class="note">
<span class="note-title">Note:</span> Astrocytes are specifically targeted using a <i>LacZ</i> transgene expressed under the control of the human <i>GFAP</i> promoter (<a href="#bib3">Brenner et al., 1994</a>). The transgene is located on the X-chromosome. Due to random X-inactivation, labeling frequency varies in females. For consistency (i.e., maximal <i>LacZ</i> expression) it is therefore recommended to primarily use males as experimental animals.</div>
<ol start="2">
<li>Cross <i>MADM-7</i><sup>GT/GT</sup><i>;Emx1</i><sup>Cre/+</sup> males with <i>MADM-7</i><sup>TG/TG</sup><i>;XGFAP-LacZ</i><sup>+/−</sup> females (UPDs are colored as follows: matUPD in green; patUPD in red) (as shown in breeding strategy C in <a href="#fig2">Figure 2</a>C).
<div class="critical">
<span class="critical-title">Critical:</span> Make sure that the <i>XGFAP-LacZ</i> transgene is transmitted from females in order to obtain <i>XGFAP, LacZ</i> positive male experimental animals.</div>
<div class="note">
<span class="note-title">Note:</span> In order to reverse the fluorescent colors in cells with UPD (matUPD in red and patUPD in green) use <i>MADM-7</i><sup>TG/TG</sup> males and <i>MADM</i><sup>GT/GT</sup><i>;Emx1</i><sup>Cre/+</sup><i>;XGFAP-LacZ</i><sup>+/−</sup> females (<a href="#fig2">Figure 2</a>D).</div>
<ol type="a">
<li>Check for vaginal plug to monitor successful mating and to facilitate the experimental planning.</li>
<li>Males of the F1 generation can be used as experimental animals (genotype is <i>MADM-7</i><sup>GT/TG</sup><i>;Emx1</i><sup>Cre/+</sup><i>;XGFAP-LacZ</i><sup>+</sup>).</li>
<li>Include a negative control by also preparing single-cell suspension of one <i>MADM-7</i><sup>GT/TG</sup><i>;Emx1</i><sup>Cre/+</sup> (no <i>XGFAP-LacZ</i> expression) animal. The negative control will allow proper set up of the FACS gating strategy (see below).</li>
<li>Perform experiment at developmental stage of choice (in our lab we have successfully applied the protocol at P0, P4, and up to P14).</li>
</ol>
</li>
</ol>
<h3 id="sec1.4">Prepare equipment (on the day of experiment)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<ol start="3">
<li>Clean all dissection tools (2 curved forceps and scissors) with 70% ethanol and RNase Away, a decontamination reagent that removes RNase from equipment and surfaces.</li>
<li>Pre-warm shaking water bath (37°C) for single-cell suspension of <i>Emx1</i>-lineage or Thermoshaker (37°C) for single-cell suspension of astrocytes.</li>
<li>Prepare PCR tube holders for sorting:
<ol type="a">
<li>Take 1.5 mL tubes and cut off the lid.</li>
<li>Fill the 1.5 mL tubes with paper tissue or Parafilm (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/333-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Self-made PCR tube holders for collecting sorted cells at FACS Aria</div>
</figcaption>
</figure>
</li>
<li>Insert a PCR tube in the middle of the stuffed 1.5 mL tube.</li>
<li>After sorting, PCR tubes can be taken out of the 1.5 mL tubes with forceps and PCR tube holders can be reused by inserting a new PCR tube.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Papain vial source</td>
<td>Worthington</td>
<td>Cat#PAP2</td>
</tr>
<tr>
<td>DNase vial source</td>
<td>Worthington</td>
<td>Cat#D2</td>
</tr>
<tr>
<td>Inhibitor vial source</td>
<td>Worthington</td>
<td>Cat#OI-BSA</td>
</tr>
<tr>
<td>EBSS</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#24010043</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#21041025</td>
</tr>
<tr>
<td>Heat-inactivated fetal bovine serum (FBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10082147</td>
</tr>
<tr>
<td>Heat-inactivated horse serum (HS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#26050088</td>
</tr>
<tr>
<td>Triton X 100</td>
<td>Sigma-Aldrich</td>
<td>Cat#T878</td>
</tr>
<tr>
<td>RNase inhibitor</td>
<td>Takara</td>
<td>Cat#2313A</td>
</tr>
<tr>
<td>RNase away</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10328011</td>
</tr>
<tr>
<td>RNase-free water</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10977035</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>FACS Blue LacZ beta galactosidase detection kit</td>
<td>Abcam</td>
<td>Cat#ab189815</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: <i>MADM-7-GT</i>
</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:021457</td>
</tr>
<tr>
<td>Mouse: <i>MADM-7-TG</i>
</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:021458</td>
</tr>
<tr>
<td>Mouse: <i>Emx1</i>-Cre</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:005628</td>
</tr>
<tr>
<td>Mouse: XGFAP-LacZ</td>
<td>The Jackson Laboratory</td>
<td>RRID: IMSR JAX: 003487</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FACS Diva</td>
<td>BD Biosciences</td>
<td><a href="https://www.bdbiosciences.com/en-us/instruments/research-instruments/research-software/flow-cytometry-acquisition/facsdiva-software">https://www.bdbiosciences.com/en-us/instruments/research-instruments/research-software/flow-cytometry-acquisition/facsdiva-software</a></td>
</tr>
<tr>
<td>R statistical environment</td>
<td> </td>
<td><a href="https://www.r-project.org/">https://www.r-project.org/</a></td>
</tr>
<tr>
<td>DESeq2</td>
<td><a href="#bib11">Love et al., 2014</a></td>
<td><a href="http://www.bioconductor.org/">http://www.bioconductor.org/</a></td>
</tr>
<tr>
<td>Seurat</td>
<td><a href="#bib4">Butler et al., 2018</a></td>
<td><a href="http://www.bioconductor.org/">http://www.bioconductor.org/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>FACS Aria III</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>AluminaSeal</td>
<td>Merck</td>
<td>Cat#Z740251-100EA</td>
</tr>
<tr>
<td>Hard-Shell PCR Plates (96-well)</td>
<td>Bio Rad</td>
<td>Cat#HSP9631</td>
</tr>
<tr>
<td>PCR tubes and caps, RNase-free, 0.2 mL (8-strip format)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM12230</td>
</tr>
<tr>
<td>Falcon 5 mL polystyrene round-bottom tube with cell strainer cap (Falcon 352235) (40 μm cell strainer)</td>
<td>Fisher Scientific</td>
<td>Cat#10585801</td>
</tr>
<tr>
<td>Microfuge tubes 1.5 mL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM12450</td>
</tr>
<tr>
<td>TPP TubeSpin bioreactor tubes</td>
<td>Merck</td>
<td>Cat#Z761028-180EA</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Solutions from Worthington kit</h3>
<p>Resuspend Papain containing L-cysteine and EDTA (in 5 mL EBSS), DNase (in 0.5 mL EBSS) and Ovomucoid protease inhibitor (in 32 mL EBSS) according to manufacturer’s instructions.</p>
<p>Papain and DNase should be prepared freshly on the day of the experiment.</p>
<p>Resuspended Ovomucoid protease inhibitor can be stored at 4°C and used for several months.</p>
<table id="undtbl1">
<caption>Papain-DNase solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Papain</td>
<td>20 units/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>DNase</td>
<td>1,000 units/mL</td>
<td>250 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>Solution 2</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EBSS</td>
<td>n/a</td>
<td>5,350 μL</td>
</tr>
<tr>
<td>DNase</td>
<td>166.7 units/mL</td>
<td>250 μL</td>
</tr>
<tr>
<td>Ovomucoid protease inhibitor</td>
<td>0.67 mg</td>
<td>400 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>6,000 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>Media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM/F12</td>
<td>n/a</td>
<td>14.4 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>1,800 μL</td>
</tr>
<tr>
<td>HS</td>
<td>10%</td>
<td>1,800 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>18 mL</td>
</tr>
</tbody>
</table>
<h3 id="sec2.2">Solutions of FACS blue LacZ beta galactosidase detection kit</h3>
<p>Resuspend and aliquot reagents according to manufacturer’s instructions.</p>
<table id="undtbl4">
<caption>Lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton X-100</td>
<td>0.2%</td>
<td>0.08 μL</td>
</tr>
<tr>
<td>RNase Inhibitor</td>
<td>2 units/μL</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>RNase-free water</td>
<td>n/a</td>
<td>3.72 μL</td>
</tr>
<tr>
<td><b>Total/well</b></td>
<td>n/a</td>
<td>4 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This lysis buffer can be used to collect single cells for scRNAseq or low inputs of cells (up to 400 cells). RNA-sequencing with these amounts of cells was successfully performed using the Smart-seq V2 protocol (<a href="#bib12">Picelli et al., 2014</a>).</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of single-cell suspension for sorting cells with MADM-induced UPD of the <i>Emx1</i> lineage in developing cerebral cortex</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5–3 h</div>
<p>This section of the protocol documents the dissection and preparation of single-cell suspension from mouse cerebral cortex for sorting cells with MADM-induced UPD and fluorescent labeling derived from <i>Emx1</i> lineage.</p>
<div class="critical">
<span class="critical-title">Critical:</span> There is no stopping point in the protocol after starting with dissection of the experimental animal until the end of sorting.</div>
<ol>
<li>Prepare all solutions required for the tissue dissociation (details are listed above)
<ol type="a">
<li>Papain-DNase solution</li>
<li>Solution 2</li>
<li>DMEM/F12</li>
<li>Media</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Adjust volumes of Papain-DNase solution (in step 3) and Solution 2 (in steps 5 and 7) according to amount of used tissue.</div>
<table id="undtbl5">
<caption>Volumes of reagents are adjusted according to the amount of used tissue/sample</caption>
<thead>
<tr>
<th>Tissue</th>
<th>Papain-DNase solution</th>
<th>Solution 2 (step 5)</th>
<th>Solution 2 (step 7)</th>
<th>DMEM/F12</th>
</tr>
</thead>
<tbody>
<tr>
<td>Single cortex (embryonic brain)</td>
<td>650 μL</td>
<td>500 μL</td>
<td>250 μL</td>
<td>1,000 μL</td>
</tr>
<tr>
<td>One hemisphere (P0 animal)</td>
<td>650 μL</td>
<td>500 μL</td>
<td>250 μL</td>
<td>1,000 μL</td>
</tr>
<tr>
<td>Multiple embryonic cortices (up to 3 cortices)</td>
<td>1,300 μL</td>
<td>1,200 μL</td>
<td>300 μL</td>
<td>1,000 μL</td>
</tr>
<tr>
<td>Cortex of single individual (P0, adult animals)</td>
<td>1,300 μL</td>
<td>1,200 μL</td>
<td>300 μL</td>
<td>1,000 μL</td>
</tr>
<tr>
<td>Multiple cortices (up to 3 cortices at any postnatal age)</td>
<td>2,600 μL</td>
<td>2,400 μL</td>
<td>500 μL</td>
<td>2,000 μL</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Sacrifice experimental animal
<ol type="a">
<li>Decapitate the animal.</li>
<li>Remove the skin overlaying the skull. Then perform a long cut of the skull along the midline, a short cut at the naso-frontal suture (in front of the olfactory bulbs) and two short cuts along the parietal-interparietal sutures. By flipping the bone parts to the side, the brain is easily accessible. Place curved forceps underneath the entire brain, and thereby lift up the entire brain to remove it from the skull (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/333-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Dissection of the cortex from postnatal brain</div>
<p>(A) After removing the skull-overlaying skin (not shown), cut the bone tissue of the skull along the midline.</p>
<p>(B) Cut along the parietal-interparietal sutures.</p>
<p>(C) After removing the bone parts the brain is easily accessible.</p>
<p>(D) Carefully place the curved forceps underneath the brain in order to release it from the skull.</p>
<p>(E) Whole brain after removal from the skull.</p>
<p>(F) The whole hemisphere can be flipped to the side. Hippocampus is visible and accessible now.</p>
<p>(G) Removal of hippocampus.</p>
<p>(H) Hemisphere after hippocampus was removed.</p>
<p>(I) Dissection of the cortex.</p>
<p>(J) Dissected cortex which is used as starting material for preparation of single-cell suspension.</p>
</figcaption>
</figure>
</li>
<li>Remove the hippocampus to avoid contamination from other cell types.</li>
<li>Dissect the cortex. Here we specifically focus on the isolation of cortical cell types. In our hands, careful and precise dissection of the area of interest is most efficient to avoid contamination from cell types originating from other brain areas (<a href="#fig4">Figure 4</a>).</li>
</ol>
</li>
<li>Transfer cortex to Papain-DNase solution in a 50 mL bioreactor tube harboring a filter-cap and incubate for 30 min at 37°C in a shaking water bath.</li>
<li>During the incubation time, prepare lysis buffer and the respective collection tubes (PCR tubes) or 96-well plate.</li>
<li>After 30 min, remove the bioreactor tube containing the dissociated cortical tissue from the water bath and add Solution 2. Mix thoroughly by pipetting up and down to dissolve remaining tissue parts.</li>
<li>Centrifuge for 5 min with 100 × <i>g</i> at 20°C–22°C.</li>
<li>Remove supernatant and resuspend the pellet in Solution 2. Add DMEM/F12 as a wash buffer. Mix well.</li>
<li>Centrifuge for 5 min with 200 × <i>g</i> at 20°C–22°C.</li>
<li>Remove supernatant and resuspend pellet in 300 – 500 μL media. Single-cell suspension is now ready for FACS. Keep suspension on ice until sorting can be started.</li>
<li>For collection of a low number of cells, proceed with step 11. For sorting single cells, proceed with step 20.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Perform all steps in a timely manner because cell death rate increases over time.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Although volumes are rather small, ventilated cap bioreactor tubes (50 mL) were used during the whole procedure in order to allow high oxygen exposure.</div>
<h3 id="sec3.2">Isolation of cells with MADM-induced UPD using FACS (collection of low cell numbers)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min to 3 h</div>
<p>In this section of the protocol MADM<sup>+</sup> cells are sorted according to their fluorescent colors and collected for RNA-sequencing.</p>
<ol start="11">
<li>Ensure that the cytometer (here: FACS Aria III) is set up for your sorting experiment.
<ol type="a">
<li>Use the 100 μm nozzle.</li>
<li>Set the sample temperature and the temperature for the collection tubes to 4°C.</li>
<li>Flush the system with RNase Away for 1–2 min.</li>
<li>Flush the system with RNase-free water for 1–2 min.</li>
<li>Adjust the side streams so that they hit the middle of the collection tubes.</li>
</ol>
</li>
<li>Filter your entire sample through a 40 μm cell strainer. Dilute your sample with 200–300 μL media, which is additionally added through the cell strainer.</li>
<li>Prepare 0.2 mL PCR tubes containing 4 μL lysis buffer as collection tubes for your sorted cells.</li>
<li>Place the collection tubes into 1.5 mL tubes, which are then placed into the appropriate FACS Aria tube holder.</li>
<li>Load your sample and record 30,000 - 50,000 events. Adjust the gates to select the surviving cell population (see <a href="#troubleshooting">Troubleshooting</a> Problem 1), for removing doublets and for sorting GFP<sup>+</sup> and TdT<sup>+</sup> (optional: GFP<sup>+</sup> TdT<sup>+</sup> double positive) cells (<a href="#fig5">Figure 5</a>A) (see <a href="#troubleshooting">Troubleshooting</a> Problem 2).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/333-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Gating strategy for FACS experiments</div>
<p>(A) Gating steps for sorting MADM-labeled cells.</p>
<p>(B) Gating for red LacZ<sup>−</sup> population is set up based on a negative control (no <i>XGFAP-LacZ</i> expression).</p>
<p>(C) Gating for red astrocytes (TdT<sup>+</sup>, LacZ<sup>+</sup>).</p>
<p>(D) Gating for green LacZ<sup>−</sup> population is set up based on a negative control (no <i>XGFAP-LacZ</i> expression).</p>
<p>(E) Gating for green astrocytes (GFP<sup>+</sup>, LacZ<sup>+</sup>).</p>
</figcaption>
</figure>
</li>
<li>Open the sort layout and select 4-way purity as the sorting mode to sort your sample.</li>
<li>Start sorting. Depending on your experimental needs, either set a maximum number of labeled cells that shall be collected or sort the entire volume of sample.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Lysis buffer was used and optimized for single cells and low amounts of cells (max. 400 cells). If a greater number of cells will be collected with FACS, a different lysis buffer needs to be used (<a href="#bib10">Laukoter et al., 2020b</a>).</div>
<ol start="18">
<li>Once sorting is completed, transfer the entire volume of the collected sample into an empty well in a 96-well plate, which should remain on dry ice throughout the procedure. Be fast with pipetting since samples freeze in the pipette tip if this step is executed too slowly.</li>
<li>Collect all replicates for your RNA-sequencing experiment in this 96-well plate. Either fill the entire plate on a single day, or add few samples on a day-to-day basis. When adding few samples over a time course of several days or weeks, store the plate at −80°C until further processing or additional samples can be added.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This setup with collecting samples on a 96-well plate proved to be very useful for preparing samples for RNA-sequencing. However, sample collection and handling can be adapted to individual use and respective purposes.</div>
<div class="note">
<span class="note-title">Note:</span> Since sorting can be done only for one sample at the time, it is important to decide on the number of cells you want to collect for your sequencing experiment. The longer a sample is kept on ice before it is processed by FACS, the higher the number of dead cells (this is of critical importance when cells from postnatal brains are sorted).</div>
<h3 id="sec3.3">Isolation of cells with MADM-induced UPD using FACS for single-cell RNA-sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min to 1.5 h</div>
<p>This section of the protocol lists details on sorting MADM<sup>+</sup> cells for single-cell RNA-sequencing.</p>
<ol start="20">
<li>Ensure that the cytometer (here: FACS Aria III) is set up for your sorting experiment.
<ol type="a">
<li>Use the 100 μm nozzle.</li>
<li>Set the sample temperature and the plate holder temperature to 4°C.</li>
<li>Flush the system with RNase Away for 1–2 min.</li>
<li>Flush the system with RNase-free water for 1–2 min.</li>
<li>Make sure that the stream is hitting the wells of the 96-well plate properly in the middle (use “test sort” of unlabeled cells on the cover of the plate).</li>
</ol>
</li>
<li>Filter your entire sample through a 40 μm cell strainer. Dilute your sample with 200–300 μL media, which is additionally added through the cell strainer.</li>
<li>Place 96-well plate in the plate holder. All wells need to be filled with 4 μL lysis buffer.</li>
<li>Load your sample and record 30,000 – 50,000 events. Adjust the gates to select the live cell population (see <a href="#troubleshooting">Troubleshooting</a> Problem 1), for removing doublets and for sorting GFP<sup>+</sup> and TdT<sup>+</sup> (optional: GFP<sup>+</sup> TdT<sup>+</sup> double positive) cells (see <a href="#fig5">Figure 5</a>A) (same comments as above) (see <a href="#troubleshooting">Troubleshooting</a> Problem 2).</li>
<li>Open the sort layout and set the purity to single cells.</li>
<li>Select the collection of a single cell per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Design a layout for collecting your single cells. Keep in mind that you may want to sort a higher number of cells (up to 400 cells) as a positive control for your sequencing experiment in one of the wells. For this purpose, you can also make use of the MADM-unlabeled cells as there is no limitation with cell numbers.</div>
<p>Consult with your sequencing facility regarding the plate layout since empty wells and a positive control might be needed at specific positions. An example for a plate layout is provided in <a href="#tbl2">Table 2</a>.</p>
<table id="tbl2">
<caption>Table 2. Potential plate layout for collection of single cells which are later processed for RNA-sequencing</caption>
<thead>
<tr>
<th> </th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
<th>9</th>
<th>10</th>
<th>11</th>
<th>12</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
</tr>
<tr>
<td>B</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
</tr>
<tr>
<td>C</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
</tr>
<tr>
<td>D</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
</tr>
<tr>
<td>E</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
</tr>
<tr>
<td>F</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
</tr>
<tr>
<td>G</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
</tr>
<tr>
<td>H</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>1 GFP<sup>+</sup> cell</td>
<td>1 TdT<sup>+</sup> cell</td>
<td>empty</td>
<td>empty</td>
<td>50 cells</td>
</tr>
</tbody>
</table>
<ol start="26">
<li>Start sorting.</li>
<li>After sorting is completed, seal the plate with AluminaSeal, quick freeze immediately on dry ice and store at −80°C until further processing. In order to ensure maximum sequencing efficiency plates should be processed within three months after collection.</li>
</ol>
<h3 id="sec3.4">Preparation of single-cell suspension for sorting astrocytes with MADM-induced UPD</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5–3 h</div>
<p>This section of the protocol explains the dissection and preparation of single-cell suspension for sorting MADM<sup>+</sup> astrocytes with MADM-induced UPD.</p>
<div class="critical">
<span class="critical-title">Critical:</span> There is no stopping point in the protocol after starting with dissection of the experimental animal until the end of sorting.</div>
<ol>
<li>Prepare all solutions required for the tissue dissociation (details are listed above)
<ol type="a">
<li>Papain-DNase solution</li>
<li>Solution 2</li>
<li>DMEM/F12</li>
<li>Media</li>
<li>Substrate Reagent of FACS Blue LacZ beta galactosidase detection kit</li>
</ol>
</li>
</ol>
<table id="undtbl6">
<caption>Volumes of reagents for one replicate are adjusted depending on the age of the experimental animal</caption>
<thead>
<tr>
<th>Age</th>
<th>Papain-DNase solution</th>
<th>Solution 2 (step 5)</th>
<th>Solution 2 (step 7)</th>
<th>DMEM/F12</th>
</tr>
</thead>
<tbody>
<tr>
<td>P0/P4/P7</td>
<td>400 μL</td>
<td>400 μL</td>
<td>200 μL</td>
<td>800 μL</td>
</tr>
<tr>
<td>P10/P14</td>
<td>500 μL</td>
<td>500 μL</td>
<td>250 μL</td>
<td>1,000 μL</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Sacrifice experimental animal. Individual animals were used as biological replicates for downstream bioinformatics analyses upon RNA-seq.
<ol type="a">
<li>Decapitate the animal.</li>
<li>Remove the skin overlaying the skull. Then perform a long cut of the skull along the midline, a short cut at the naso-frontal suture (in front of the olfactory bulbs) and two short cuts along the parietal-interparietal sutures. By flipping the bone parts to the side, the brain is easily accessible. Place curved forceps underneath the entire brain, and thereby lift up the entire brain to remove it from the skull (<a href="#fig4">Figure 4</a>).</li>
<li>Remove the hippocampus to avoid contamination from other cell types.</li>
<li>Dissect the cortex. In this protocol, we specifically focus on the isolation of cortical cell types. Thus, careful and precise dissection of the area of interest is most efficient to avoid contamination from cell types originating from other brain areas (<a href="#fig4">Figure 4</a>).</li>
</ol>
</li>
<li>Transfer cortex into Papain-DNase solution in a 1.5 mL tube and incubate for 30 min at 37°C in a Thermoshaker.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to not close the lid completely. Some air should get into the tube to allow oxygenation.</div>
<ol start="4">
<li>During the enzymatic digestion, prepare LacZ staining solution by diluting LacZ substrate 1:50 in DMEM/F12. Pre-activate the staining solution by incubating at 37°C until step 10.</li>
<li>Also prepare lysis buffer and the respective collection tubes (PCR tubes).</li>
<li>Add Solution 2 to the Tissue-Papain-DNase mix. Mix well by pipetting up and down and dissociate remaining pieces of tissue.</li>
<li>Centrifuge for 8 min at 100 × <i>g</i> at 20°C–22°C.</li>
<li>Remove the supernatant with a pipette and resuspend the pellet in Solution 2. Add DMEM/F12 as wash buffer.</li>
<li>Centrifuge for 8 min at 200 × <i>g</i> at 20°C–22°C.</li>
<li>Remove supernatant and resuspend pellet in 100 μL pre-activated LacZ staining solution. Incubate for 25 min at 37°C in a Thermoshaker.</li>
<li>Stop the reaction by adding 100 μL media. Mix well.</li>
<li>Centrifuge for 8 min at 200 × <i>g</i> at 20°C–22°C.</li>
<li>Resuspend the cell pellet in 200 μL media. Keep samples at 20°C–22°C and in the dark until sorting is started.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> LacZ staining is temperature sensitive. Staining is most stable at RT.</div>
<h3 id="sec3.5">Isolation of astrocytes with MADM-induced UPD using FACS (collection of low cell numbers)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min to 3 h</div>
<p>In this section of the protocol, astrocytes are sorted according to MADM-labeling and LacZ substrate fluorescent signal, and are then collected for bulk RNA-sequencing.</p>
<ol start="14">
<li>Ensure that the cytometer (here: FACS Aria III) is set up for your sorting experiment.
<ol type="a">
<li>Use the 100 μm nozzle.</li>
<li>Set the sample temperature to 20°C–22°C and the temperature for the collection tubes to 4°C.</li>
<li>Flush the system with RNase Away for 1–2 min.</li>
<li>Flush the system with RNase-free water for 1–2 min.</li>
<li>Adjust the side streams so that they hit the middle of the collection tubes.</li>
</ol>
</li>
<li>Filter your entire sample through a 40 μm cell strainer. Dilute your sample with 200 μL media, which is additionally added through the cell strainer.</li>
<li>First load the LacZ negative, MADM positive sample and record 100,000 cells. Use the negative control to adjust the gates for the live cell population (see <a href="#troubleshooting">Troubleshooting</a> Problem 1) as well as for the MADM<sup>+</sup> cells (see <a href="#troubleshooting">Troubleshooting</a> Problem 2), and also adjust the LacZ gate (<a href="#fig5">Figures 5</a>A–5E).</li>
<li>Prepare 0.2 mL PCR tubes containing 4 μL lysis buffer as collection tubes for your sorted cells.</li>
<li>Place the collection tubes into 1.5 mL tubes, which are then placed in the appropriate FACS Aria tube holder.</li>
<li>Load your LacZ positive sample and start sorting. Astrocytes are either GFP<sup>+</sup> LacZ<sup>+</sup> or TdT<sup>+</sup> LacZ<sup>+</sup> double positive. LacZ<sup>−</sup> but GFP<sup>+</sup> and TdT<sup>+</sup> cells are enriched for neurons of the <i>Emx1</i><sup>+</sup> lineage, but also contain oligodendrocytes and olfactory bulb neuroblasts. Either set a maximum number of labeled cells that shall be collected, or sort the entire volume of sample (max. 400 cells).</li>
<li>Once sorting is completed, transfer the entire volume of the collected sample into an empty well in a 96-well plate, which should remain on dry ice throughout the whole procedure. Be fast with pipetting since samples freeze in the pipette tip in case this step is executed too slowly.</li>
<li>Collect all replicates for your RNA-sequencing experiment in this 96-well plate. Either fill the entire plate on a single day, or add few samples on a day-to-day basis. When adding few samples over a time course of several days or weeks, store the plate at −80°C until further processing or additional samples can be added.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The setup with collecting samples on a 96-well plate proved very useful for preparing samples for RNA-sequencing. However, this can be adapted to individual use and respective purposes.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>In our experience, the fraction of living cells in suspension is higher in younger animals. However, sorting of MADM<sup>+</sup> cells using <i>Emx1</i>-Cre for single-cell RNA-sequencing was always successful, even at adult stages. Collection of 400 TdT<sup>+</sup> cells and 400 GFP<sup>+</sup> cells was easily feasible from individual animals dissected at P0, P7, and up to P21. The time required for sorting increased with the age of the experimental animal due to an increase of myelin-containing debris in the single-cell suspension.</p>
<p>Sorting MADM<sup>+</sup> astrocytes from a single animal usually yields relatively low cell numbers. We typically aim for 200 cells at P0, 50–150 cells at P7 and 10–50 cells at P14 per MADM fluorescent color from a single cortex. In order to reach these cell numbers, usually the entire volume of single-cell suspension was sorted.</p>
<h3 id="sec4.1">RNA-sequencing and statistical analysis</h3>
<p>Preparation of cDNA libraries from sorted cells was performed as described in (<a href="#bib12">Picelli et al., 2014</a>).</p>
<p>Standard bioinformatics quality control and statistical analysis are used to analyze the RNA-seq data. We used R statistical environment for all RNA-seq analyses (<a href="https://www.r-project.org/">https://www.r-project.org/</a>). In our hands DESeq2 and Seurat packages proved to be especially suitable to analyze bulk and single-cell data, respectively. For detailed description on data analysis with these packages, we refer the interested reader to the publications, which link to documentations and online tutorials (<a href="#bib4">Butler et al., 2018</a>, <a href="#bib11">Love et al., 2014</a>). Since cells with MADM-induced UPD contain predictable imbalances of imprinted gene dosage, transcriptional analysis provided a powerful approach to study genomic imprinting (<a href="#bib10">Laukoter et al., 2020b</a>).</p>
<p>Besides the probing of genomic imprinting using UPD, MADM technology can be utilized to perform functional analysis of candidate genes (<a href="#bib1">Beattie et al., 2017</a>, <a href="#bib2">Beattie et al., 2020</a>, <a href="#bib8">Hippenmeyer et al., 2010</a>, <a href="#bib9">Laukoter et al., 2020a</a>). In MADM-based functional genetic analysis, fluorescent labeling of individual cells corresponds to mutant and wild-type genotypes, respectively (e.g., green is mutant and red is wild type). Application of the above protocol in MADM-based mutant analysis allows investigation of mutant-specific transcriptional responses (<a href="#bib9">Laukoter et al., 2020a</a>). In such cases, we recommend to always verify that the mutation is linked to one particular UPD in order to gain reproducible results between different replicates, experimental batches, or genotypes. Importantly, reverse breeding schemes (resulting in color swap of mutant and wild-type MADM cells) permit the separation of UPD phenotypes from genetic loss of gene function phenotypes.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Both protocols for preparing single-cell suspensions to isolate neurons and glial cells, or specifically astrocytes, have been optimized for early postnatal stages. Application of the protocol for late adult stages might require additional optimization due to increased cell death and increased myelin-containing debris.</p>
<p>MADM is a technology that can be used to sparsely induce UPD and fluorescently label cells (<a href="#bib5">Contreras et al., 2020</a>). Thus, collection of more than 20–30K cells is not easily possible. For instance, if using MADM-7 to isolate cells from the <i>Emx1</i> lineage at P0 multiple cortices were pooled to increase cell numbers in the samples. Consequently, MADM is most suitable for downstream applications that require limited biological material, up to a few thousand cells and/or for single-cell applications.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>High amount of dead cells</p>
<h3 id="sec6.2">Potential solution 1</h3>
<ol>
<li>Make sure to not deprive cells of air during single-cell suspension preparation (use ventilated cape tubes or leave 1.5 mL tubes partly open).</li>
<li>Limit the time between sample preparation and sorting. When many samples need to be processed at the same time, it is preferable to work in a team so that certain protocol steps can be done in parallel. Alternatively, split sample preparation into batches of maximum 3–5 samples, and sort these before moving on to the next batch.</li>
</ol>
<h3 id="sec6.3">Problem 2</h3>
<p>Low number of MADM-labeled cells</p>
<h3 id="sec6.4">Potential solution 2</h3>
<ol>
<li>If individual animals were used for the single-cell suspension, pool multiple cortical samples to prepare the single-cell suspension for sorting.</li>
<li>Cover your sample tubes with aluminum foil to protect the samples from light.</li>
<li>MADM-labeled cells might have died during the preparation. Please refer to section above “Problem: High amount of cell death.”</li>
</ol>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Beattie, R., Postiglione, M.P., Burnett, L.E., Laukoter, S., Streicher, C., Pauler, F.M., Xiao, G., Klezovitch, O., Vasioukhin, V., Ghashghaei, T.H., and Hippenmeyer, S. (2017). Mosaic analysis with double markers reveals distinct sequential functions of Lgl1 in neural stem cells. Neuron <i>94</i>, 517-533.e3.</p>
<p id="bib2">Beattie, R., Streicher, C., Amberg, N., Cheung, G., Contreras, X., Hansen, A.H., and Hippenmeyer, S. (2020). Lineage tracing and clonal analysis in developing cerebral cortex using mosaic analysis with double markers (MADM). J. Vis. Exp. <a href="https://doi.org/10.3791/61147">https://doi.org/10.3791/61147</a></p>
<p id="bib3">Brenner, M., Kisseberth, W.C., Su, Y., Besnard, F., and Messing, A. (1994). GFAP promoter directs astrocyte-specific expression in transgenic mice. J. Neurosci. <i>14</i>, 1030-1037.</p>
<p id="bib4">Butler, A., Hoffman, P., Smibert, P., Papalexi, E., and Satija, R. (2018). Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat. Biotechnol. <i>36</i>, 411-420.</p>
<p id="bib5">Contreras, X., Davaatseren, A., Amberg, N., Hansen, A.H., Sonntag, J., Andersen, L., Bernthaler, T., Heger, A., Johnson, R., Schwarz, L.A., Luo, L., Rülicke, T., and Hippenmeyer, S. (2020). A genome-wide library of MADM mice for single-cell genetic mosaic analysis. bioRxiv <a href="https://doi.org/10.1101/2020.06.05.136192">https://doi.org/10.1101/2020.06.05.136192</a></p>
<p id="bib6">Gorski, J.A., Talley, T., Qiu, M., Puelles, L., Rubenstein, J.L., and Jones, K.R. (2002). Cortical excitatory neurons and glia, but not GABAergic neurons, are produced in the Emx1-expressing lineage. J. Neurosci. <i>22</i>, 6309-6314.</p>
<p id="bib7">Hippenmeyer, S., Johnson, R.L., and Luo, L. (2013). Mosaic analysis with double markers reveals cell-type-specific paternal growth dominance. Cell Rep. <i>3</i>, 960-967.</p>
<p id="bib8">Hippenmeyer, S., Youn, Y.H., Moon, H.M., Miyamichi, K., Zong, H., Wynshaw-Boris, A., and Luo, L. (2010). Genetic mosaic dissection of Lis1 and Ndel1 in neuronal migration. Neuron <i>68</i>, 695-709.</p>
<p id="bib9">Laukoter, S., Beattie, R., Pauler, F.M., Amberg, N., Nakayama, K.I., and Hippenmeyer, S. (2020a). Imprinted Cdkn1c genomic locus cell-autonomously promotes cell survival in cerebral cortex development. Nat. Commun. <i>11</i>, 195.</p>
<p id="bib10">Laukoter, S., Pauler, F.M., Beattie, R., Amberg, N., Hansen, A.H., Streicher, C., Penz, T., Bock, C., and Hippenmeyer, S. (2020b). Cell-type specificity of genomic imprinting in cerebral cortex. Neuron <i>107</i>, 1160-1179.e9.</p>
<p id="bib11">Love, M.I., Huber, W., and Anders, S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. <i>15</i>, 550.</p>
<p id="bib12">Picelli, S., Faridani, O.R., Bjorklund, A.K., Winberg, G., Sagasser, S., and Sandberg, R. (2014). Full-length RNA-seq from single cells using Smart-seq2. Nat. Protoc. <i>9</i>, 171-181.</p>
<p id="bib13">Tucci, V., Isles, A.R., Kelsey, G., Ferguson-Smith, A.C., and Erice Imprinting, G. (2019). Genomic imprinting and physiological processes in mammals. Cell <i>176</i>, 952-965.</p>
<p id="bib14">Zong, H., Espinosa, J.S., Su, H.H., Muzumdar, M.D., and Luo, L. (2005). Mosaic analysis with double markers in mice. Cell <i>121</i>, 479-492.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact Simon Hippenmeyer (<a href="mailto:simon.hippenmeyer@ist.ac.at">simon.hippenmeyer@ist.ac.at</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>No custom code is necessary to perform this protocol. All datasets mentioned in this protocol have been described elsewhere (<a href="#bib10">Laukoter et al., 2020b</a>).</p>
<h3>Acknowledgments</h3>
<p>This research was supported by the Scientific Service Units (SSU) at <a href="https://doi.org/10.13039/100014988">IST Austria</a> through resources provided by the Bioimaging (BIF) and Preclinical Facilities (PCF). N.A received support from the <a href="https://doi.org/10.13039/501100002428">FWF</a> Firnberg-Programm (T 1031). This work was also supported by <a href="https://doi.org/10.13039/100014988">IST Austria</a> institutional funds; <a href="https://doi.org/10.13039/501100002428">FWF</a> SFB F78 to S.H.; <a href="https://doi.org/10.13039/501100013691">NÖ Forschung und Bildung</a> n[f+b] life science call grant (C13-002) to S.H.; the People Programme (Marie Curie Actions) of the <a href="https://doi.org/10.13039/100011102">European Union’s Seventh Framework Programme</a> (FP7/2007-2013) under REA grant agreement no. 618444 to S.H.; and the <a href="https://doi.org/10.13039/100010663">European Research Council</a> (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement no. 725780 LinPro) to S.H.</p>
<h3>Author Contributions</h3>
<p>All authors conceived the project. S.L. established and optimized the protocol for single-cell suspension of MADM<sup>+</sup> cells with input from F.M.P. and S.H. N.A. optimized isolation and FACS of astrocytes. F.M.P. established bioinformatics analysis. All authors edited and proofread the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>