<article>
<section>
<h1 id="header">Analytical protocol to identify local ancestry-associated molecular features in cancer</h1>
<p><time datetime="2021-09-24">Published: September 24, 2021</time></p>
<p>Jian Carrot-Zhang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">10</a>,<a href="#cor1">*</a></sup> Seunghun Han,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Wanding Zhou,<sup><a href="#aff4">4</a>,<a href="#aff5">5</a></sup> Jeffrey S. Damrauer,<sup><a href="#aff6">6</a></sup> Anab Kemal,<sup><a href="#aff7">7</a></sup> Ashton C. Berger, Matthew Meyerson, Katherine A. Hoadley, Ina Felau, Samantha Caesar-Johnson, John A. Demchok, Michael K.A. Mensah, Roy Tarnuzzer, Zhining Wang, Liming Yang, Jean C. Zenklusen, Nyasha Chambwe, Theo A. Knijnenburg, A. Gordon Robertson, Christina Yau, Christopher Benz, Kuan-lin Huang, Justin Newberg, Garret Frampton, R. Jay Mashl, Li Ding, Alessandro Romanel, Francesca Demichelis, Rosalyn W. Sayaman, Elad Ziv, Peter W. Laird, Hui Shen, Christopher K. Wong, Joshua M. Stuart, Alexander J. Lazar, Xiuning Le, Ninad Oak, Andrew D. Cherniack,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">11</a>,<a href="#cor2">**</a></sup> and Rameen Beroukhim<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff8">8</a>,<a href="#aff9">9</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>The Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA</p>
<p id="aff2"><sup>2</sup>Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</p>
<p id="aff3"><sup>3</sup>Harvard Medical School, Boston, MA 02115, USA</p>
<p id="aff4"><sup>4</sup>Center for Computational and Genomic Medicine, Children’s Hospital of Philadelphia, PA, 19104, USA</p>
<p id="aff5"><sup>5</sup>Department of Pathology and Laboratory Medicine, University of Pennsylvania, Philadelphia, PA, 19104, USA</p>
<p id="aff6"><sup>6</sup>Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</p>
<p id="aff7"><sup>7</sup>National Cancer Institute, Bethesda, MD 20892, USA</p>
<p id="aff8"><sup>8</sup>Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</p>
<p id="aff9"><sup>9</sup>Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</p>
<p id="fn1"><sup>10</sup>Technical contact</p>
<p id="fn2"><sup>11</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:zhangj@broadinstitute.org">zhangj@broadinstitute.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:achernia@broadinstitute.org">achernia@broadinstitute.org</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:rameen_beroukhim@dfci.harvard.edu">rameen_beroukhim@dfci.harvard.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100766">10.1016/j.xpro.2021.100766</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>People of different ancestries vary in cancer risk and outcome, and their molecular differences may indicate sources of these variations. Determining the “local” ancestry composition at each genetic locus across ancestry-admixed populations can suggest causal associations. We present a protocol to identify local ancestry and detect the associated molecular changes, using data from the Cancer Genome Atlas. This workflow can be applied to cancer cohorts with matched tumor and normal data from admixed patients to examine germline contributions to cancer.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref3">Carrot-Zhang et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocols for local ancestry identification using the TCGA data</li>
<li>Detecting local ancestry associated with cancer risk</li>
<li>Statistical analysis associating molecular changes with local ancestry</li>
<li>Understanding the contribution of genetic ancestry in admixed patients</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1034-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Prepare input files</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 h</div>
<p>This protocol uses genotyping data, ancestry assignments, and molecular features associated with ancestry generated by The Cancer Genome Atlas (TCGA) pan-cancer atlas.</p>
<ol>
<li>Download TCGA Affymetrix SNP 6.0 microarray data from Genomic Data Commons Legacy Archive (<a href="https://portal.gdc.cancer.gov/legacy-archive/">https://portal.gdc.cancer.gov/legacy-archive/</a>) for each matched normal sample. Select the Birdseed outputs (TXT under Data format), which are the genotyping files processed from raw SNP array CEL files.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Please refer to the following link to request access to controlled germline genotyping data:</div>
<p><a href="https://gdc.cancer.gov/access-data/obtaining-access-controlled-data">https://gdc.cancer.gov/access-data/obtaining-access-controlled-data</a></p>
<p>This protocol runs local ancestry identification by cancer type. It is recommended to select samples of a specific cancer type and copy their Birdseed genotyping files to a separate folder.</p>
<ol start="2">
<li>Download ancestry assignments from Table S1 of <a href="#bib3">Carrot-Zhang et al., 2020</a> (10.1016/j.ccell.2020.04.012), which includes percentage of EUR, EAS or AFR ancestry for each individual.</li>
<li>Download a table mapping TCGA SNP array ID to Sample and Patient ID from Genomic Data Commons Legacy Archive or download an example file:(<a href="https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/geno_sample_map.txt">https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/geno_sample_map.txt</a>)</li>
<li>Download somatic mutations associated with ancestry from Table S2 of <a href="#bib3">Carrot-Zhang et al. 2020</a> (<a href="http://10.1016/j.ccell.2020.04.012">10.1016/j.ccell.2020.04.012</a>).</li>
<li>Download methylation differences associated with ancestry from Table S3 from <a href="#bib3">Carrot-Zhang et al. 2020</a> (<a href="http://10.1016/j.ccell.2020.04.012">10.1016/j.ccell.2020.04.012</a>).</li>
<li>Download mRNA expression associated with ancestry from Table S4 from <a href="#bib3">Carrot-Zhang et al. 2020</a> (<a href="http://10.1016/j.ccell.2020.04.012">10.1016/j.ccell.2020.04.012</a>).</li>
</ol>
<p>This protocol uses 1000 Genomes samples as a reference panel to identify haplotypes and assign ancestry to genomic loci by chromosome (local ancestry identification).</p>
<ol start="7">
<li>Download 1000 Genomes reference panel version 5a (<a href="http://bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/b37.vcf/">http://bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/b37.vcf/</a>) and convert it into PLINK binary PED format using the PLINK --make-bed option, and join all chromosomes into one file, using the PLINK --merge-list option. Next, remove duplicate markers and markers exceeding missingness in the 1000 Genomes samples.
<ol type="a">
<li>Identify duplicate germline variants (markers) in the ∗.bim file and write them to a separate file. Then, exclude those duplicate markers using the PLINK --exclude option.</li>
<li>Identify markers excessed missingness (a minimum frequency of 5% genotype missing rate allowed) using the PLINK --freq option, and then, exclude markers excessed missingness in the ∗frq.counts file using the PLINK --exclude option. After filtering, 32,161,998 markers are left.</li>
</ol>
</li>
<li>To convert Birdseed genotyping files into VCF format, download hg19 human genome reference: ftp://<u>ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/human_g1k_v37.fasta.gz</u> Unzip and index the fasta file using Samtools faidx. Then, download the genome annotation file for Affymetrix SNP array 6.0: <a href="https://software.broadinstitute.org/cancer/cga/sites/default/files/data/tools/contest/GenomeWideSNP_6.na30.annot.hg19.csv.pickle.gz">https://software.broadinstitute.org/cancer/cga/sites/default/files/data/tools/contest/GenomeWideSNP_6.na30.annot.hg19.csv.pickle.gz</a>
</li>
<li>Download the population information for 1000 Genomes samples from: ftp://<u>ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/integrated_call_samples_v3.20130502.ALL.panel</u>
</li>
<li>Download the 1000 Genomes phased haplotypes (∗.hap.gz and ∗.legend.gz): <a href="https://mathgen.stats.ox.ac.uk/impute/1000GP_Phase3.html">https://mathgen.stats.ox.ac.uk/impute/1000GP_Phase3.html</a>
</li>
<li>Download the genetic map file (genetic_map∗) containing at least three columns (chromosome, physical position in bp and genetic position in cM):<a href="https://mathgen.stats.ox.ac.uk/impute/1000GP_Phase3.html">https://mathgen.stats.ox.ac.uk/impute/1000GP_Phase3.html</a>
</li>
</ol>
<h3 id="sec1.2">Software installation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–5h</div>
<p>Because some algorithms take time and memory, this protocol recommends installing the following software in a high-performance compute cluster.</p>
<ol start="12">
<li>Install the most recent version of Samtools, Bcftools and Bedtools as described in <a href="#key-resources-table">key resources table</a>.</li>
<li>Install PLINK v1.9, Shapeit v2 and RFMix v1.5.4 as described in <a href="#key-resources-table">key resources table</a>.</li>
<li>Install Python 2.7 library to support downstream analyses.</li>
<li>To convert Birdseed genotyping files into VCF format, install pyfasta (<a href="https://pypi.org/project/pyfasta">https://pypi.org/project/pyfasta</a>) and add the module to your Python classpath.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>TCGA Affymetrix SNP 6.0 microarray data</td>
<td>Genomic Data Commons Legacy Archive</td>
<td><a href="https://portal.gdc.cancer.gov/legacy-archive/">https://portal.gdc.cancer.gov/legacy-archive/</a></td>
</tr>
<tr>
<td>TCGA cancer type and subtype</td>
<td>(<a href="#bib15">Sanchez-Vega et al., 2018</a>)</td>
<td><a href="https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx">https://ars.els-cdn.com/content/image/1-s2.0-S0092867418303593-mmc1.xlsx</a></td>
</tr>
<tr>
<td>TCGA patient ancestry call</td>
<td>(<a href="#bib3">Carrot-Zhang et al., 2020</a>)</td>
<td>Table S1 of <a href="http://10.1016/j.ccell.2020.04.012">10.1016/j.ccell.2020.04.012</a>
</td>
</tr>
<tr>
<td>Somatic mutation, methylation and mRNA expression associated with ancestry in TCGA</td>
<td>(<a href="#bib3">Carrot-Zhang et al., 2020</a>)</td>
<td>Tables S2–S4 of <a href="http://10.1016/j.ccell.2020.04.012">10.1016/j.ccell.2020.04.012</a>
</td>
</tr>
<tr>
<td>TCGA SNP array and sample ID map</td>
<td>(<a href="#bib3">Carrot-Zhang et al., 2020</a>)</td>
<td><a href="https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/geno_sample_map.txt">https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/geno_sample_map.txt</a></td>
</tr>
<tr>
<td>TCGA patient gender and age</td>
<td>(<a href="#bib8">Liu et al., 2018</a>)</td>
<td><a href="https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81">https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81</a></td>
</tr>
<tr>
<td>Resource website for the local ancestry calls</td>
<td>(<a href="#bib3">Carrot-Zhang et al., 2020</a>)</td>
<td><a href="https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020">https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020</a></td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>PLINK v1.9</td>
<td>(<a href="#bib12">Purcell et al., 2007</a>)</td>
<td><a href="https://www.cog-genomics.org/plink/1.9/">https://www.cog-genomics.org/plink/1.9/</a></td>
</tr>
<tr>
<td>Samtools</td>
<td>(<a href="#bib7">Li et al., 2009</a>)</td>
<td><a href="http://www.htslib.org/">http://www.htslib.org/</a></td>
</tr>
<tr>
<td>Bcftools</td>
<td>n/a</td>
<td><a href="http://samtools.github.io/bcftools/bcftools.html">http://samtools.github.io/bcftools/bcftools.html</a></td>
</tr>
<tr>
<td>Bedtools</td>
<td>(<a href="#bib13">Quinlan and Hall, 2010</a>)</td>
<td><a href="https://bedtools.readthedocs.io/en/latest/">https://bedtools.readthedocs.io/en/latest/</a></td>
</tr>
<tr>
<td>SHAPEIT v2</td>
<td>(<a href="#bib5">Delaneau et al., 2011</a>)</td>
<td><a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html</a></td>
</tr>
<tr>
<td>RFMIX v1.5.4</td>
<td>(<a href="#bib10">Maples et al., 2013</a>)</td>
<td><a href="https://sites.google.com/site/rfmixlocalancestryinference/">https://sites.google.com/site/rfmixlocalancestryinference/</a></td>
</tr>
<tr>
<td>birdseed2vcf</td>
<td>n/a</td>
<td><a href="https://github.com/ding-lab/birdseed2vcf">https://github.com/ding-lab/birdseed2vcf</a></td>
</tr>
<tr>
<td>Local ancestry identification</td>
<td>(<a href="#bib11">Martin et al., 2017</a>)</td>
<td><a href="https://github.com/armartin/ancestry_pipeline">https://github.com/armartin/ancestry_pipeline</a></td>
</tr>
<tr>
<td>Local ancestry association analysis</td>
<td>(<a href="#bib3">Carrot-Zhang et al., 2020</a>)</td>
<td><a href="https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM">https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Local ancestry identification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3h for step 1 and step 2; 72–120h for step 3; 48–72h for step 4; 100–180h for step 5 (alternative step)</div>
<p>This step phases the TCGA samples into haplotypes, produces local ancestry calls that assign African (AFR), European (EUR) and East Asian (EAS) ancestry to each genomic locus (<a href="#fig1">Figure 1</a>), and estimates global ancestry proportions to ensure accuracy of local ancestry calls.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1034-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Analytical workflow for identifying local ancestry-associated molecular features in cancer</div>
<p>Part 1: Preparation of TCGA and 1000 Genomes genotyping data; Part 2: Haplotype estimation and local ancestry assignments; Part 3: Examples of correlation analyses utilizing local ancestry information.</p>
</figcaption>
</figure>
<ol>
<li>First, for each cancer type, merge all samples into PLINK input format (binary PED files):
<ol type="a">
<li>Exclude variants with genotype call confidence greater than 0.1 in the Birdseed files downloaded from the Genomic Data Commons Legacy Archive.</li>
<li>Convert the Birdseed genotyping files into VCF format, using birdseed2vcf (<a href="https://github.com/ding-lab/birdseed2vcf">https://github.com/ding-lab/birdseed2vcf</a>) following the instructions.</li>
<li>Select samples of a specific cancer type and copy the VCF files to a folder named for that cancer type.</li>
<li>Compress and index the VCF files using bgzip and tabix from Samtools. Prepare a list of VCF files for each cancer type, then, use the Bcftools --merge option to merge all samples in each cancer type into one VCF file.<br> An example Unix bash command:<br> <code>bcftools merge -l $vcflist.txt -Oz &gt; $OUT.vcf.gz</code>
</li>
<li>For each merged file, use PLINK to convert VCF into format to binary PED files (BED/BIM/FAM) using the PLINK --make-bed option.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended to merge TCGA samples by cancer type and run local ancestry identification for each cancer type in parallel to reduce the computational runtime.</div>
<ol start="2">
<li>To prepare for phasing, perform quality control for genome-wide SNP genotyping data and all TCGA germline samples, and then merge TCGA samples from the same cancer type with 1000 Genomes reference samples into a single PLINK file.
<ol type="a">
<li>Perform basic quality control of markers using PLINK:
<ol type="i">
<li>Exclude markers that are not on autosomes using the PLINK –not-chr option.</li>
<li>Identify markers excessed missingness (a minimum frequency of 5% genotype missing rate allowed) using the --freq option, and exclude markers excessed missingness in the ∗frq.counts file using the PLINK --exclude option.</li>
<li>Calculate Hardy Weinberg equilibrium using the PLINK --hardy option, and identify markers failing Hardy Weinberg equilibrium test (P&lt;1e-6) due to excess heterogeneousness in the ∗.hwe file using the PLINK --exclude option.</li>
</ol>
</li>
<li>Perform basic quality control of samples:
<ol type="i">
<li>Compute pairwise relationships using the PLINK --genome option.</li>
<li>Exclude samples due to duplicates, with the PI_HAT column in the ∗genome file greater than or equal to 0.7.</li>
<li>Exclude samples due to relatedness (e.g., sibling), with the PI_HAT column in the ∗genome file between 0.35 and 0.7, and the Z1 column (IBD=1) less than 0.7.</li>
</ol>
</li>
<li>Merge the TCGA samples with 1000 Genomes project phase 3 samples:
<ol type="i">
<li>Identify markers with minor allele frequency (MAF) of at least 1% in 1000 Genomes project phase 3 samples, using the PLINK --maf option.</li>
<li>Extract markers that passed quality control from both the TCGA data set identified in step 2a using the PLINK --extract option.</li>
<li>Merge the two datasets using the PLINK --bmerge option.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended to merge TCGA samples with reference samples into one PLINK binary PED file and analyze them together to avoid technical errors, such as strand flip in the phasing step.</div>
<ol start="3">
<li>To prepare for local ancestry identification, for each TCGA germline sample, phase alleles to haplotypes using Shapeit2 by chromosome.
<ol type="a">
<li>Split the genome into 22 files corresponding to each autosomal chromosome using the --chr option in PLINK</li>
<li>Perform phasing using Shapeit2 for each chromosome by running the following bash command line as an example:
<ol type="i">
<li>Marker inspection to exclude SNPs that are incompatible (e.g., exceeding missing data rate; strand error etc.) for phasing using Shapeit2. SNPs to be excluded are shown in the output file (∗.mendel.snp.strand.exclude).<br> <code>for i in {1..22}; do</code><br> <code>shapeit -check --input-ref 1000GP_Phase3_chr${chr}.hap.gz \</code><br> <code>1000GP_Phase3_chr${chr}.legend.gz \</code><br> <code>1000GP_Phase3.sample \</code><br> -<code>B BRCA/chr${chr} \</code><br> -<code>-input-map genetic_map_chr${chr}_combined_b37.txt --input-thr 1 \</code><br> -<code>-output-log BRCA/chr${chr}.mendel</code><code>;</code><br> <code>done</code>
</li>
<li>Phasing algorithm using Shapeit2. The output files include ∗.haps and ∗ sample files.</li>
</ol>
</li>
</ol>
</li>
</ol>
<p><code>for i in {1..22}; do</code></p>
<p><code>shapeit --input-ref 1000GP_Phase3_chr${chr}.hap.gz \</code></p>
<p><code>1000GP_Phase3_chr${chr}.legend.gz 1000GP_Phase3.sample \</code></p>
<p>-<code>B BRCA/chr${chr} \</code></p>
<p>-<code>-duohmm --input-map genetic_map_chr${chr}_combined_b37.txt \</code></p>
<p>-<code>-output-max BRCA/chr${chr}.haps.gz BRCA}/chr${chr}.sample \</code></p>
<p>-<code>-exclude-snp BRCA/chr${chr}.mendel.snp.strand.exclude \</code></p>
<p>-<code>-thread 4</code><code>;</code></p>
<p><code>done</code></p>
<div class="critical">
<span class="critical-title">Critical:</span> Phased 1000 Genomes samples (1000GP_Phase3_chr${chr}.hap.gz) are used as a reference panel for phasing the TCGA samples (BRCA/chr${chr}). This step takes time, so it is recommended to parallelize the tasks by chromosome. In the following, four cores are requested for each phasing job.</div>
<p>Imputation is not required in this step, as the number of germline alleles used to capture local ancestry is sufficient, as a linkage disequilibrium generated from continental-scale admixtures that occurred recently (e.g., African American and Latin American populations) is expected to span many centimorgans.</p>
<div class="note">
<span class="note-title">Note:</span> Other tools, such as Eagle2 (<a href="#bib9">Loh et al. 2016</a>), Beagle (<a href="#bib2">Browning and Browning 2007</a>) etc. can also be used for phasing larger datasets. Moreover, a larger reference panel, such as Haplotype Reference Consortium (<a href="http://www.haplotype-reference-consortium.org/home">http://www.haplotype-reference-consortium.org/home</a>) consisting of 64,976 haplotypes, may improve phasing accuracy. One may use the Michigan Imputation Server (<a href="https://imputationserver.sph.umich.edu/index.html#!pages/home">https://imputationserver.sph.umich.edu/index.html#!pages/home</a>) to run Eagle phasing with Haplotype Reference Consortium as the reference panel. One may write a custom script to convert Eagle output files into RFMix input files for step 4.</div>
<ol start="4">
<li>Perform local ancestry identification to assign AFR, EUR or EAS ancestry to each genomic region by chromosome using RFMix.
<ol type="a">
<li>Choose parental populations from 1000 Genomes samples to use for local ancestry identification. An example file with 1668 samples from AFR, EUR and EAS populations can be found: <a href="https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/BRCA.ref">https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/BRCA.ref</a>
<div class="note">
<span class="note-title">Note:</span> It is recommended by RFMix to balance the size of continental-scale ancestries in the reference samples to avoid biases caused by skewed reference sample sizes. For example, one can randomly choose between 400 to 500 1000 Genomes samples from AFR, EUR, or EAS population.</div>
</li>
<li>Read in the ancestry assignment from Table S1 of <a href="#bib3">Carrot-Zhang et al., 2020</a>, exclude samples with South Asian (SAS) or American (AMR) ancestry greater than 20%, and then generate a list of TCGA samples by cancer type (for example, BRCA.notref).
<div class="note">
<span class="note-title">Note:</span> Because most TCGA samples were from AFR, EUR or EAS populations, the identification of local SAS or AMR ancestry is not performed in this protocol.</div>
</li>
<li>For each chromosome, convert phased shapeit outputs into RFMix input files, and prepare other RFMix input files (∗.alleles; ∗.classes; ∗.snp_locations). Detailed information on how to prepare the RFMix input files can be found in the manual:<br> <a href="https://www.dropbox.com/s/cmq4saduh9gozi9/RFMix_v1.5.4.zip?file_subpath=%2FRFMix_v1.5.4%2FManual.pdf">https://www.dropbox.com/s/cmq4saduh9gozi9/RFMix_v1.5.4.zip?file_subpath=%2FRFMix_v1.5.4%2FManual.pdf</a><br> An example script (shapeit2rfmix.py) can be found in the following Github repository: <a href="https://github.com/armartin/ancestry_pipeline">https://github.com/armartin/ancestry_pipeline</a><br> This script requires ∗haps.gz files, a list of TCGA samples (<a href="https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/BRCA.notref">https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/BRCA.notref</a>), a list of reference samples (<a href="https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/BRCA.ref">https://github.com/jcarrotzhang/ancestry-from-panel/blob/master/GDAN_AIM/BRCA.ref</a>), and genetic_map∗ files downloaded in <a href="#before-you-begin">before you begin</a> section step 11.<br> <code>for i in {1..22}; do</code><br> <code>python ∼bin/ancestry_pipeline-master/shapeit2rfmix.py \</code><br> -<code>-shapeit_hap_ref BRCA/chr${i}.haps.gz \</code><br> -<code>-shapeit_hap_admixed BRCA/chr${i}.haps.gz \</code><br> -<code>-shapeit_sample_ref BRCA/chr${i}.sample \</code><br> -<code>-shapeit_sample_admixed BRCA/chr${i}.sample \</code><br> -<code>-ref_keep BRCA.ref \</code><br> -<code>-admixed_keep BRCA.notref \</code><br> -<code>-chr ${i} \</code><br> -<code>-genetic_map 1000GP_Phase3/genetic_map_chr${i}_combined_b37.txt \</code><br> -<code>-out BRCA/rfmix</code><code>;</code><br> <code>done</code>
</li>
<li>Run RFMix, with a minimum window size of 0.2 cM and a node size of 5 for random forest trees, using the following bash command as an example:<br> <code>for i in {1..22}; do</code><br> <code>python RunRFMix.py --forward-backward -w 0.2 -n 5 \</code><br> -<code>-num-threads 4 \</code><br> <code>PopPhased BRCA/rfmix_chr${i}.alleles \</code><br> <code>BRCA_rfmix.classes \</code><br> <code>BRCA/rfmix_chr${i}.snp_locations \</code><br> -<code>o BRCA/outputPopPhased.chr.${i}.withref</code><code>;</code> <code>done</code>
<div class="note">
<span class="note-title">Note:</span> Other parameters in RFMix can be considered. For example, RFMix can perform expectation-maximization (EM) optimization of the model, using the -e option. Although it is also recommended to use -e to increase the accuracy of local ancestry identification, the -e option will significantly increase computational complexity.</div>
</li>
<li>Collapsing RFMix outputs (∗.Viterbi.txt) into bed files. This step generates two ∗.bed files for each allele (A.bed and B.bed) for all samples, with the first four columns as chromosome, start position, end position and ancestry assignment, respectively.<br> An example script (collapse_ancestry.py) can be found in the following Github repository: <a href="https://github.com/armartin/ancestry_pipeline">https://github.com/armartin/ancestry_pipeline</a>
</li>
<li>Visualize the local ancestry calls using karyogram plots for inspection of technical artifacts (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1034-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Accuracy assessment of local ancestry identification</div>
<p>(A) Karyograms of six individuals from the TCGA cohort. Upper panel: none-admixed individuals from EUR, AFR and EAS populations, respectively. Lower panel: individuals with mixed ancestries.</p>
<p>(B–D) (B) Correlation of global ancestry indicated by the proportions of EUR, (C) AFR and (D) EAS estimated from RFMix and ADMIXTURE. Upper panel: TCGA breast cancer cohort; Lower panel: TCGA prostate cancer cohort. r values are calculated from Pearson’s correlation.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended to visually inspect the reference samples to avoid upstream errors, such as mislabeling population assignments of reference samples in the class file etc. If the --forward-backward option is used in RFMix, local ancestry calls can be filtered with the posterior probability of a given ancestry less than 0.9.</div>
</li>
<li>Apply a custom script to calculate global ancestry (the percentage of AFR, EUR and EAS ancestry per individual) based on RFMix generated local ancestry calls. This step is to ensure that the local ancestry assignment can accurately reflect the global ancestry generated by a different algorithm.<br> An example script (lai_global.py) can be found in the Github repository: <a href="https://github.com/armartin/ancestry_pipeline">https://github.com/armartin/ancestry_pipeline</a>
</li>
<li>Exclude samples with ADMIXTURE estimated global SAS or AMR ancestry greater than 20% from Table S1. in <a href="#bib3">Carrot-Zhang et al., 2020</a>, and compare the global ancestry estimated from RFMix to ADMIXTURE estimated global ancestry using a Pearson correlation.
<div class="note">
<span class="note-title">Note:</span> Because most TCGA samples were from AFR, EUR or EAS populations, only samples in the 1000 Genomes Project from AFR, EUR or EAS populations were used for local ancestry identification in <a href="#bib3">Carrot-Zhang et al., 2020</a>. However, other parental populations should be used when analyzing other populations with mixed ancestries. For example, 1000 Genomes samples from Peru and Mexico can be used as reference samples when analyzing cancer samples from the Latin American populations.</div>
</li>
</ol>
</li>
</ol>
<ol start="5">
<li>
<b>Alternative:</b> It is now recommended to use RFMix2 (<a href="https://github.com/slowkoni/rfmix">https://github.com/slowkoni/rfmix</a>) - a more user friendly version. RFMix2 takes phased VCF files as input directly, without requiring users to generate ∗.alleles and ∗.snp_locations. To use RFMix2, it is recommended to use Beagle for phasing, which outputs phased variants in VCF format that can be used as an RFMix2 input file. RFMix2 also reports global ancestry estimated based on local ancestries identified by their algorithm in their standard output file (∗results.rfmix.Q) corresponding to the .Q output files from ADMIXTURE that can be compared to ADMIXTURE generated global ancestry directly.
<ol type="a">
<li>To run Beagle, the following reference files are needed:
<ol type="i">
<li>Beagle reference vcf: <a href="http://bochet.gcc.biostat.washington.edu/beagle/genetic_maps/">http://bochet.gcc.biostat.washington.edu/beagle/genetic_maps/</a>
</li>
<li>Beagle reference map: <a href="http://bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a">http://bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a</a>
</li>
</ol>
</li>
<li>An example bash command to run RFMix2 on Beagle phased VCF file:<br> <code>for i in {1..22}; do</code><br> <code>java -Xmx10 g -jar ∼bin/beagle.08Jun17.d8b.jar \</code><br> <code>gt=BRCA/chr${chr}.vcf.gz \</code><br> <code>ref=∼beagle/chr${chr}.1 kg.phase3.v5a.vcf.gz \</code><br> <code>out=BRCA/$chr.beagle_phased \</code><br> <code>map=∼reference/beagle/plink.chr${chr}.GRCh37.map \</code><br> <code>chrom=${chr} \</code><br> <code>impute=false</code><code>;</code><br> <code>done</code><br> For more information about phasing using Beagle, please refer to <a href="http://faculty.washington.edu/browning/beagle/beagle.html">http://faculty.washington.edu/browning/beagle/beagle.html</a>
</li>
<li>An example bash command to run RFMix2 on Beagle phased VCF file:</li>
</ol>
</li>
</ol>
<p><code>for i in {1..22}; do</code></p>
<p><code>∼bin/rfmix-master/rfmix -f BRCA/$chr.beagle_phased.vcf \</code></p>
<p>-<code>r ∼reference/beagle/chr${chr}.1 kg.phase3.v5a.vcf.gz \</code></p>
<p>-<code>g genetic_map_chr${chr}_combined_b37.txt \</code></p>
<p>-<code>m BRCA.ref \</code></p>
<p>-<code>o $BRCA/chr${chr}.results --chromosome=${chr}</code><code>;</code></p>
<p><code>done</code></p>
<h3 id="sec2.2">Association of local ancestry with cancer features and risk</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–8h</div>
<p>This step demonstrates two types of analyses leveraging local ancestry calls to investigate germline influence on cancer development (<a href="#fig1">Figure 1</a>). The first analysis identifies genomic loci with enrichment of certain ancestry in the pan-cancer cohort, to investigate the contributions of ancestry at those loci to cancer risk. In the second analysis, ancestries are compared between loci harboring genes whose mRNA expression (or methylation) levels are associated with ancestry and those that do not, in order to evaluate whether ancestry at these specific loci--loci with identified molecular correlates of ancestry--contributes to cancer risk. These analyses should be focused on an admixed population, for example, the African American population with mixed AFR and EUR ancestry that has a relatively larger sample size compared to other admixed populations in the TCGA cohort.</p>
<p>For the genome-wide, pan-cancer analysis, perform the following:</p>
<ol start="6">
<li>Read in global and local ancestries for all TCGA samples:
<ol type="a">
<li>Read in the ancestry assignment from Table S1 of <a href="#bib3">Carrot-Zhang et al., 2020</a>. Select samples with AFR ancestry between 20%–80% for the downstream analyses.</li>
<li>Read in bed files converted from RFMix outputs, and output regions with local ancestry (48,939 genomic loci) across all samples.</li>
</ol>
</li>
</ol>
<p>An example script (find_local_ancestry_count_blocks.py) can be found in the following Github repository: <a href="https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM">https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM</a></p>
<ol start="7">
<li>Code regions identified from step 4 with both alleles from AFR ancestry as 2, with one AFR allele as 1, and loci with both alleles from EUR as 0.</li>
<li>Compute a z-score for each region with unique local ancestry identified from step 4. In Python, one can store coded ancestry of each locus in an array and then use the scipy.stats.zscore module to compute the z-score in each locus relative to the genome. This step again, only uses samples with AFR ancestry between 20%–80%.</li>
</ol>
<p>An example script (local_ancestry_enrichment.py) can be found in the following Github repository: <a href="https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM">https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM</a></p>
<ol start="9">
<li>Plot each locus and its corresponding z-score of local ancestry across the genome. Apply FDR correction to identify regions passed multiple testing corrections.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In theory, certain loci might be enriched for one ancestry over others across the admixed population so that the observed enriched regions may not be specific to cancer risks. Therefore, filtering out known regions associated with ancestry in healthy individuals, or a case-control study, is necessary when applicable. In practice, populations that have mixed within the last several hundred years are unlikely to exhibit such biases.</div>
<p>For the gene-specific analysis in pan-cancers, perform the following:</p>
<ol start="10">
<li>Similarly, the correlation of somatic alterations in a specific gene and the local ancestry of the locus harboring the gene of interest, while controlling for global ancestry, can be tested, using the following logistic regression model in Python:</li>
</ol>
<p>mutation status ∼ local ancestry + global EUR ancestry + global AFR ancestry</p>
<p>Global ancestries can be obtained from Table S1 of <a href="#bib3">Carrot-Zhang et al., 2020</a>. A significant p-value indicates a <i>cis</i> effect of germline variation to somatic alteration in the gene of interest.</p>
<p>An example script (find_local_ancestry_by gene.py) can be found in the following Github repository: <a href="https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM">https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM</a></p>
<p>This script selects samples with AFR ancestry between 20%–80%, finds the local ancestry of the locus of interest in each sample, and correlates the local ancestry with mutation status of the gene of interest, using the regression model described above. The local ancestry is coded as 0 for homozygous EUR ancestry, 1 for heterozygous AFR ancestry and 2 for homozygous AFR ancestry.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Because local ancestry can be significantly correlated with global ancestry, when testing the association between local ancestry and somatic mutation status, global ancestry should be added as a covariate to the regression model. A significant local ancestry--phenotype correlation independent of global ancestry may suggest germline influence on the phenotype, as local ancestry is not expected to be associated with ancestry-associated environmental exposure.</div>
<p>This analysis is performed in a pan-cancer fashion. Due to limited sample size, we were unable to perform tissue-type specific analysis.</p>
<p>For the second analysis, restricted to loci with known molecular correlates of ancestry (in this case, mRNA expression in <i>cis</i>), perform the following:</p>
<ol start="11">
<li>Keep loci associated with AFR ancestry (z-score &gt;3) or EUR ancestry (z-score &lt;−3) calculated from step 8, or downloaded from Table S1 <a href="#bib3">Carrot-Zhang et al., 2020</a>.</li>
<li>For the remaining loci from step 11, use the Bedtools intersect option to locate loci with differentially expressed genes between populations from Table S4 of <a href="#bib3">Carrot-Zhang et al., 2020</a>. Label those loci as “ancestry-associated expression” loci. Label other loci as “non-ancestry-associated expression” loci.</li>
<li>Compare z-scores between “ancestry-associated loci” and “non-ancestry-associated loci”, using Wilcoxon’s rank test.</li>
</ol>
<p>Step 11 to 13 can also be applied to other loci with ancestry-associated molecular features, such as methylation differences, to assess their association with cancer risk.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec3.1">Local ancestry identification accuracy</h3>
<p>When inspecting the karyograms of TCGA samples (<a href="#fig2">Figure 2</a>A) from self-reported African American patients, one should expect varying admixture contributions from AFR and EUR ancestries, but less contribution from EAS ancestry. If the local ancestry calls are filtered by the posterior probability, one would expect few regions with unknown ancestry assignment, especially for regions close to the ancestry switch locations.</p>
<p>One should expect a high correlation (Pearson’s r &gt; 0.98) between global ancestry estimated from ADMIXTURE (<a href="#bib1">Alexander and Lange, 2011</a>) and the genome-wide average of local ancestry-based determinations obtained from RFMix (<a href="#fig2">Figure 2</a>B), suggesting an agreement of the ancestry estimates based on two different algorithms. The local-global ancestry comparison can be performed using global ancestry inferred from other methods (i.e., principal component analysis-based approaches).</p>
<h3 id="sec3.2">Association of local ancestry with cancer risk</h3>
<p>See <a href="#bib3">Carrot-Zhang et al., 2020</a> Figure 1D for expected outcomes of genome-wide pan-cancer association of local ancestry with cancer risk.</p>
<p>A case-case admixture mapping can be used to identify a marker locus associated with a certain cancer phenotype, by comparing the observed frequency of the risk ancestry at a marker locus to the rest of the genomes of the cancer cases. Sample size may limit the detection of a specific locus that is significantly enriched in AFR or EUR ancestry in TCGA samples from admixed patients, even in the pan-cancer analysis. A power calculation, considering risk allele frequency, penetrance, and degree of admixture in the population may be helpful prior to this type of analysis.</p>
<p>A case-control admixture mapping compares the observed frequency of the risk ancestry at a marker locus from the cases to the frequency observed in the controls. The controls can be samples without the molecular feature of interest. For example, to test the association of local ancestry with a certain somatic mutation, samples from the same cancer type without carrying the mutation can be used as controls. However, none of the TCGA cancer types provide sufficient statistical power for a genome-wide case-control comparison, and therefore, one would not expect to identify a significant local ancestry-somatic alteration association using the TCGA data set.</p>
<h3 id="sec3.3">Association of local ancestry at loci known ancestry-associated molecular features, with cancer risk</h3>
<p>See <a href="#bib3">Carrot-Zhang et al., 2020</a> Figure S7D and S7E for expected outcomes.</p>
<p>By restricting analyses of ancestry-associated cancer risk to loci for which ancestry-associated molecular features have been detected, one would hope to reduce the number of hypotheses tested while enriching for signal, thereby improving power. Indeed, this analysis detected an enrichment of AFR ancestry in loci with ancestry-associated mRNA expression (<a href="#bib3">Carrot-Zhang et al., 2020</a>). Notably, datasets such as TCGA provide methylation and gene expression data for tumor samples; those data types are not available for matched normal samples. However, most ancestry-associated molecular features appear to be shared between tumor and normal tissue (<a href="#bib3">Carrot-Zhang et al., 2020</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Studying the differences between cancer genomes from patients with different ancestries, and more broadly germline effects on somatic alterations, can be particularly challenging because it requires both germline and somatic data from the same patient. The TCGA cohort of over 10,000 cases across 33 cancer types with matched germline samples is such a data set for joint genomic and ancestry analysis. However, it is important to note that population diversity for most TCGA cancer types is poor, especially for patients from admixed populations. For example, we were unable to perform case-case admixture mapping analysis in tissue-type specific fashion. Moreover, although we identified several ancestry-specific somatic alterations in cancers using the TCGA cohort (e.g., <i>FBXW7</i> in multiple cancer types), we did not identify a genomic region harboring a germline allele associated with <i>FBXW7</i>-mutant cancers (either in case-case studies or using the case-control approach) in the pan-cancer analysis, possibly due to limited sample size of admixed samples with AFR and EUR ancestries.</p>
<p>Another mixed patient population, Latin American patients, is severely under-represented in the TCGA cohort. We were therefore unable to evaluate molecular features and cancer risk that correlate with Native American (NAT) vs. other ancestries (<a href="#bib11">Martin et al. 2017</a>; <a href="#bib4">Carrot-Zhang et al. 2021</a>; <a href="#bib14">Rodriguez et al. 2020</a>). The NAT ancestry is also under-represented in 1000 Genomes reference samples. A larger data set from diverse populations will enable the discovery of germline alleles associated with cancer molecular features and ancestry-associated risks.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Step 3–4</p>
<p>A lack of local ancestry association analysis for admixed patients with NAT ancestry.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>To include AMR and SAS patients in the TCGA cohort for the association analysis with local ancestry, local ancestry identification can be performed by adding AMR and SAS samples to the reference panel, and then assigning five (AFR, EUR, NAT, SAS and EAS) ancestries to each patient (<a href="#bib11">Martin et al. 2017</a>; <a href="#bib3">Carrot-Zhang et al. 2020</a>; <a href="#bib14">Rodriguez et al. 2020</a>). Because the AMR population is an admixed population, one may want to only include samples with inferred NAT ancestry greater than 20% in the reference panel.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>Step 3–4</p>
<p>The reference panel for local ancestry identification contains samples from multiple sub-populations.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>To avoid local ancestry identification biases caused by skewed reference sample sizes from sub-populations, it is recommended to use a minimum node size of 5 (-n 5) in RFMix, as shown in step 4d.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>Step 3–4</p>
<p>The reference panel for local ancestry identification contains samples with mixed ancestries.</p>
<h3 id="sec5.6">Potential solution</h3>
<p>If the reference panel contains samples with mixed ancestries, one can use the --use-reference-panels-in-EM in RFMix v1.5.4, or --reanalyze-reference in RFMix2, coupled with the EM optimization (-e) to update the ancestry assignment for reference haplotypes. Including the reference panel in the EM can also improve the results when the reference panel is small.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>Step 10</p>
<p>A lack of genome-wide association analysis of local ancestry to somatic alterations in ancestry-associated genes.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>If the sample size is sufficient for a genome-wide association analysis, step 10 can be extended to test any region to identify <i>trans</i> alleles associated with somatic alterations in certain ancestry-associated cancer genes. Multiple testing correction (e.g., Bonferroni) should be performed in the genome-wide analysis.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>Step 10</p>
<p>A lack of matched normal samples in the cancer cohort for local ancestry and somatic association analysis.</p>
<h3 id="sec5.10">Potential solution</h3>
<p>It was suggested that tumor purity and somatic alterations had minimal influence on the accuracy of inferring germline haplotype and genetic ancestry (<a href="#bib4">Carrot-Zhang et al. 2021</a>; <a href="#bib6">Gusev et al. 2021</a>). It is possible to accurately infer both global and local ancestry from tumor-only DNA, and correlate ancestry with somatic phenotypes, when genotyping data from matched germline samples is not available. One may consider controlling for certain somatic features (e.g., tumor purity, somatic copy number alterations) in the local ancestry and somatic association analysis.</p>
</section>
<section>
<h2 id="consortia">Consortia</h2>
<p>Ashton C. Berger, Matthew Meyerson, Katherine A. Hoadley, Ina Felau, Samantha Caesar-Johnson, John A. Demchok, Michael K.A. Mensah, Roy Tarnuzzer, Zhining Wang, Liming Yang, Jean C. Zenklusen, Nyasha Chambwe, Theo A. Knijnenburg, A. Gordon Robertson, Christina Yau, Christopher Benz, Kuan-lin Huang, Justin Newberg, Garret Frampton, R. Jay Mashl, Li Ding, Alessandro Romanel, Francesca Demichelis, Rosalyn W. Sayaman, Elad Ziv, Peter W. Laird, Hui Shen, Christopher K. Wong, Joshua M. Stuart, Alexander J. Lazar, Xiuning Le, Ninad Oak</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Alexander, D.H. and Lange, K. (2011). Enhancements to the ADMIXTURE algorithm for individual ancestry estimation. BMC Bioinformatics <i>12</i>, 246. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref1">View at publisher</a></p>
<p id="bib2">Browning, S.R. and Browning, B.L. (2007). Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. Am. J. Hum. Genet. <i>81</i>, 1084-1097. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref2">View at publisher</a></p>
<p id="bib3">Carrot-Zhang, J., Chambwe, N., Damrauer, J.S., Knijnenburg, T.A., Robertson, A.G., Yau, C., Zhou, W., Berger, A.C., Huang, K.-L., Newberg, J.Y., et al. (2020). Comprehensive analysis of genetic ancestry and its molecular correlates in cancer. Cancer Cell <i>37</i>, 639-654.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref3">View at publisher</a></p>
<p id="bib4">Carrot-Zhang, J., Soca-Chafre, G., Patterson, N., Thorner, A.R., Nag, A., Watson, J., Genovese, G., Rodriguez, J., Gelbard, M.K., Corrales-Rodriguez, L., et al. (2021). Genetic ancestry contributes to somatic mutations in lung cancers from admixed Latin American populations. Cancer Discov. <i>11</i>, 591-598. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref4">View at publisher</a></p>
<p id="bib5">Delaneau, O., Marchini, J., and Zagury, J.-F. (2011). A linear complexity phasing method for thousands of genomes. Nat. Methods <i>9</i>, 179-181. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref5">View at publisher</a></p>
<p id="bib6">Gusev, A., Groha, S.M., Taraszka, K., Semenov, Y.R., and Zaitlen, N. (2021). Constructing germline research cohorts from the discarded reads of clinical tumor sequences. medRxiv. . <a class="external-link" href="https://doi.org/10.1101/2021.04.09.21255197">https://doi.org/10.1101/2021.04.09.21255197</a></p>
<p id="bib7">Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R., and 1000 Genome Project Data Processing Subgroup. (2009). The sequence alignment/map format and SAMtools. Bioinformatics <i>25</i>, 2078-2079. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref7">View at publisher</a></p>
<p id="bib8">Liu, J., Lichtenberg, T., Hoadley, K.A., Poisson, L.M., Lazar, A.J., Cherniack, A.D., Kovatich, A.J., Benz, C.C., Levine, D.A., Lee, A.V., et al. (2018). An integrated TCGA pan-cancer clinical data resource to drive high-quality survival outcome analytics. Cell <i>173</i>, 400-416.e11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref8">View at publisher</a></p>
<p id="bib9">Loh, P.-R., Danecek, P., Palamara, P.F., Fuchsberger, C., A Reshef, Y., K Finucane, H., Schoenherr, S., Forer, L., McCarthy, S., Abecasis, G.R., et al. (2016). Reference-based phasing using the Haplotype Reference Consortium panel. Nat. Genet. <i>48</i>, 1443-1448. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref9">View at publisher</a></p>
<p id="bib10">Maples, B.K., Gravel, S., Kenny, E.E., and Bustamante, C.D. (2013). RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference. Am. J. Hum. Genet. <i>93</i>, 278-288. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref10">View at publisher</a></p>
<p id="bib11">Martin, A.R., Gignoux, C.R., Walters, R.K., Wojcik, G.L., Neale, B.M., Gravel, S., Daly, M.J., Bustamante, C.D., and Kenny, E.E. (2017). Human Demographic History Impacts Genetic Risk Prediction across Diverse Populations. Am. J. Hum. Genet. <i>100</i>, 635-649. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref11">View at publisher</a></p>
<p id="bib12">Purcell, S., Neale, B., Todd-Brown, K., Thomas, L., Ferreira, M.A.R., Bender, D., Maller, J., Sklar, P., de Bakker, P.I.W., Daly, M.J., et al. (2007). PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. <i>81</i>, 559-575. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref12">View at publisher</a></p>
<p id="bib13">Quinlan, A.R. and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics <i>26</i>, 841-842. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref13">View at publisher</a></p>
<p id="bib14">Rodriguez, D.A., Sanchez, M.I., Decatur, C.L., Correa, Z.M., Martin, E.R., and Harbour, J.W. (2020). Impact of genetic ancestry on prognostic biomarkers in uveal melanoma. Cancers <i>12</i>, 3208. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref14">View at publisher</a></p>
<p id="bib15">Sanchez-Vega, F., Mina, M., Armenia, J., Chatila, W.K., Luna, A., La, K.C., Dimitriadoy, S., Liu, D.L., Kantheti, H.S., Saghafinia, S., et al. (2018). Oncogenic signaling pathways in the cancer genome atlas. Cell <i>173</i>, 321-337.e10. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00472-X/sref15">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and fulfilled by the lead contact Andrew Cherniack (<a href="mailto:achernia@broadinstitute.org">achernia@broadinstitute.org</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique materials or reagents.</p>
<h4>Data and code availability</h4>
<p>Data used and generated are listed in the <a href="#key-resources-table">key resources table</a>, and the GDC Publication Website (<a href="https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020">https://gdc.cancer.gov/about-data/publications/CCG-AIM-2020</a>). Code generated for local ancestry-related analyses is available from the following URL <a href="https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM">https://github.com/jcarrotzhang/ancestry-from-panel/tree/master/GDAN_AIM</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank the Cancer Genome Atlas Research Network, the U.S. <a href="https://doi.org/10.13039/100000054">National Cancer Institute</a> for funding through U24 grants CA210999, CA210974, CA211006, CA210949, CA210978, CA210952, CA210989, CA210957, CA210990, CA211000, CA210950, CA210969, CA210988, and K24CA169004 and R01CA1845851. J.C.-Z. holds a NCI K99 award. We thank Anab Kemal for project administration. We are grateful for advice from numerous colleagues, TCGA and GDAN collaborators, and the GDC technical support team.</p>
<h3>Author contributions</h3>
<p>J.C.-Z., A.D.C., and R.B. wrote the manuscript. J.C.-Z., S.H., J.S.D., and W.Z. provided data and performed the analyses. A.K. provided project administration.</p>
<h3>Declaration of interests</h3>
<p>A.D.C. receives research funding from Bayer. R.B. owns equity in and consults for Scorpion and Ampressa Therapeutics and receives research funding from Novartis.</p>
</section>
</article>