<article>
<section>
<h1 id="header">Microinjection for precision genome editing in <i>Caenorhabditis elegans</i>
</h1>
<p><time datetime="2021-09-03">Published: September 3, 2021</time></p>
<p>Krishna S. Ghanta,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Takao Ishidate,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a></sup> and Craig C. Mello<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>RNA Therapeutics Institute, University of Massachusetts Medical School, 368 Plantation Street, Worcester, MA 01605, USA</p>
<p id="aff2"><sup>2</sup>Program in Molecular Medicine, University of Massachusetts Medical School, 368 Plantation Street, Worcester, MA 01605, USA</p>
<p id="aff3"><sup>3</sup>Howard Hughes Medical Institute, University of Massachusetts Medical School, 368 Plantation Street, Worcester, MA 01605, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:craig.mello@umassmed.edu">craig.mello@umassmed.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100748">10.1016/j.xpro.2021.100748</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>In <i>Caenorhabditis elegans,</i> targeted genome editing techniques are now routinely used to generate germline edits. The remarkable ease of <i>C. elegans</i> germline editing is attributed to the syncytial nature of the pachytene ovary which is easily accessed by microinjection. This protocol describes the step-by-step details and troubleshooting tips for the entire CRISPR-Cas genome editing procedure, including gRNA design and microinjection of ribonucleoprotein complexes, followed by screening and genotyping in <i>C. elegans</i>, to help accessing this powerful genetic animal system.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref10">Ghanta and Mello (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Step-by-step microinjection protocol for CRISPR-Cas genome editing in <i>C. elegans</i>
</li>
<li>Tips to troubleshoot microinjection and genome editing protocols</li>
<li>Melting dsDNA donor molecules potentiates precision genome editing</li>
<li>A single injected hermaphrodite can produce dozens of precisely edited progenies</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Culturing worms</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Worms grown for microinjection should be healthy — fed well, contamination free and under controlled temperature. An approach that ensures ideal worms for injection is to culture broods from individual animals. Animals cultured by chunking Nematode Growth Medium (NGM) agar tend to vary significantly in their quality for the purpose of injections. If chunked worms must be used it is helpful to pick several L3 and L4 hermaphrodites onto a fresh plate one day prior to injection. Synchronized worms obtained by bleaching gravid adults can also be used, but synchrony limits the variety of stages available from the culture on a given day to a much narrower developmental window than is achieved by brood culture.</p>
<p>The choice of animals may also depend on the needle. A blunt or broken needle can still work for injections into older larger adults but will invariably kill young adults. When attempting the first injections for a new batch of animals, pay close attention to the size of the gonad cytoplasm. Recently starved adults may appear healthy otherwise but will have a shrunken core gonadal cytoplasm that provides a very poor target for microinjection. Unhealthy animals are a very common reason for unsuccessful injections. Refer to the WormBook chapter for detailed protocols on maintenance of <i>C. elegans</i> (<a href="#bib18">Stiernagle, 2006</a>).</p>
<h3 id="sec1.2">Preparation of injection pads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>Dried 2% agarose pads (also known as injection pads) are used to immobilize the worms during the microinjection procedure (<a href="#bib8">Fire, 1986</a>; <a href="#bib12">Mello and Fire, 1995</a>). If not prepared properly the pads can either dehydrate the worms too quickly and/or worms may not stick to them. High quality injection pads are critical to performing good injections.</p>
<ol>
<li>Prepare 2% (w/v) agarose solution in distilled water by briefly bringing to a boil in a microwave oven. Keep the solution as liquid and well mixed on a stir plate with medium heat (with a stir bar).</li>
<li>Use a Pasteur pipette to place two drops (∼50 μL each) of hot agarose solution on a 24 x 60 mm glass coverslip and immediately drop another coverslip on top, forming an agarose sandwich, such that the coverslips are parallel to each other.</li>
<li>After the flattened agarose solidifies (∼5 min) remove one coverslip by gently sliding.</li>
<li>Place the coverslips with wet agarose facing up on Whatman paper cut to fit on an oven tray for baking. Array the coverslips singly so they do not stack or overlap.</li>
<li>Repeat the agarose sandwich process until about 100 pads are arrayed on the Whatman paper for baking.</li>
<li>Bake the agarose pads at 80°C for 1 h.</li>
<li>Once dried, the agarose pads will not stick to each other can be transferred to a 10 cm petri dish and stored together indefinitely. In areas with high humidity, it may be necessary to use a sealed desiccated container, or to re-bake the pads periodically as needed. For convenience, correct side (agarose side) of the slide can be marked with an asymmetric letter before storing in the container.</li>
</ol>
<h3 id="sec1.3">Preparation of needles</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>Here we describe needle preparation method that we routinely use in the lab, other labs may use slightly different techniques. Injection needles are prepared by pulling and etching borosilicate glass capillaries (1.2 mm) with inner filaments. The inner glass filament provides capillary action and aids in easier back filling of the needles. Although pre-pulled needles (e.g., Eppendorf Femtotips or Tritech Research, MINJ-PP) can be purchased, they are unnecessary and cost-prohibitive in our opinion. Needles can be stored and used indefinitely (at least in the order of months) without losing their shape due to glass flow.</p>
<ol start="8">
<li>
<b>Pulling the capillaries:</b> Needles suitable for penetrating the outer cuticle of adult worms must be both rigid and sharp. Such needles are easily prepared using an economical needle puller. There are three key adjustment parameters: the “heat” setting, a “gentle pull” setting, and a “strong pull” setting. During the gentle pull the needle gradually lengthens as the glass closest to the filament warms, upon reaching a predetermined length, this slow pull triggers a strong pull that draws the needle out and breaks the filament into two halves. If your device pulls from one end only, the end closer to the filament will have a longer taper, however both needles should be nearly identical at the tip and thus both halves should be saved for use.
<ol type="a">
<li>Adjusting the needle puller is an empirical process that can be greatly aided by keeping a reference needle to use for comparison. Needles should taper to a sharp stiff point. These parameters are attained by adjusting the “heat” and “pull-strength” settings on the puller. In general, lower heat and greater pull strength give stiffer needles. Occasionally, the heating filament is bumped by the blank needle and becomes bent. Use extra care when loading or unloading the needles to avoid this. Examine the first few needles under a dissecting scope comparing their shapes to a reference needle kept for this purpose. The settings we use may not translate to other systems and are only approximate starting values (Heat: 97°C, magnetic sub:22 and magnetic main: 90 on Narishige PN-30 needle puller). These settings should be optimized for your system.</li>
<li>Once the settings are optimized, pull needles and monitor needle shape by eye and by periodically re-checking the needles under the dissecting scope. If the needles begin to deviate in shape or if the puller begins to delay during the slow pull phase or stops pulling altogether, the filament was likely bumped out of alignment. Carefully, re-align the filament, repeat the adjustments to the heat and pull strength, and then continue pulling until you have made enough needles to last for several weeks.</li>
<li>Load the pulled needles into a tray or box (with a lid) by propping them up in rows on Time Tape (rolled lengthwise with sticky side out) or along a thin line of rolled clay.</li>
</ol>
</li>
<li>
<b>Etching Needles:</b> Etching the needles takes less than 1 min per needle and is worth the added time as it saves hours that will otherwise be lost attempting to brake open needles on the inverted microscope. To etch the needles, pressurize each needle one at a time by loading them onto the microinjection system set to 80 psi. This is easily done by sliding the Tygon tubing from the system’s compressed air source directly over the blunt end of the needle. While holding the needle in your hand, apply pressure (using the foot pedal from the injector system). Keep the needle pressurized throughout the test. If the needle tubing interface leaks (emits a hissing sound) it is necessary to seat the needle more firmly.
<ol type="a">
<li>Pipette 50 μL of distilled water near the center of a 35 mm plastic petri dish (do not use a glass dish). Pipette a second drop of 15 μL water nearby about a half inch from the larger drop. Carefully, using a plastic pipette tip add 15 μL of a commercially sourced 48% HF solution to the 15 μL drop of water to obtain a drop containing 24% HF.</li>
<li>Bubble test: Place the petri dish containing the water drop and 24% HF droplets under the dissection scope and carefully, at low magnification, bring the pressurized needle close to the (larger) water drop, pause before touching. If the needle is badly broken, then before the needle touches the water drop you will see the water drop distort from air blown from the needle. Immediately discard such needles. Otherwise, while keeping the needle pressurized, insert the tip of the needle into the water drop. If the needle emits a slow stream of tiny air bubbles, immediately remove it from the water while keeping it pressurized (to prevent water from entering the needle). The needle can then be transferred directly to a final storage tray, with lid (marked as “bubbles in water”). These needles have coarse openings and though useful are not ideal, unless for example an injection solution contains particulates that are clogging finer tipped needles.</li>
<li>If the needle fails to bubble in water, transfer the needle (while keeping it pressurized) into the drop of HF solution. In about one or two seconds it should start to bubble in HF. Immediately transfer back to water, (again maintaining pressure at all times, including during the transfer). Usually, the needle will not bubble in water. If so, transfer it to a tray labeled “bubbles in HF.” These are ideal needles. If the needle produces a slow stream of bubbles in the second water test, it can be saved in the “bubbles in water” tray. A fast bubble stream in water, indicates a faulty needle that is over etched, and should be discarded. If this happens repeatedly add additional water to further dilute the HF solution.</li>
<li>Store the etched needles in a closed container on rolled tape or clay so that the needles are firmly secured.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the needle pressurized throughout the entire process. If HF enters the needle, it will quickly over-etch and destroy the needle. If water is allowed to enter the needle it will leave residues when it evaporates and could also dilute your injection mixes.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Validating the needles in the bubble tests is a major time saver. When using pre-tested needles, a single needle can be loaded with each injection mix and will almost always yield excellent flow. If untested needles are used, or if a needle becomes clogged while in use, the needle can be opened by physically breaking the tip against the agarose pad or by touching it to the edge of a coverslip.</div>
<div class="note">
<span class="note-title">Note:</span> After several needles are etched it may be necessary to trim a few mm off of the Tygon tubing to remove cracks that prevent a good seal. Etching can also be done by loading each needle into a needle holder that creates a seal with a screw lock and gaskets. However, we find this takes more time than using the simpler direct tubing interface.</div>
<div class="critical">
<span class="critical-title">Critical:</span> HF is highly corrosive and in large quantities is very dangerous, handle the solution with great care and follow all product safety guidelines. Appropriate personal protection equipment such as eye googles, lab coat and gloves should be used. Purchase the 48% HF solution in a small quantity. If possible, purchase HF in an anti-spill bottle that permits micropipetting from a small reservoir in the lid.</div>
<h3 id="sec1.4">Preparation of stock solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<div class="note">
<span class="note-title">Note:</span> proceed to either step 10 or 11 based on the Cas protein used in this protocol.</div>
<ol start="10">
<li>If using Cas9, prepare Cas9 RNPs
<ol type="a">
<li>Cas9 protein- aliquot 0.5 μL (5 μg or 30 pmol) either in PCR tubes or 1.5 mL tubes and store at −80°C (avoid freeze/thaw cycles)</li>
<li>tracrRNA – 0.4 μg/μL (18 μM) in IDT nuclease free duplex buffer, store at −20°C (store 30 μL aliquots at −80°C for long term storage)</li>
<li>crRNA – 0.4 μg/μL (34 μM) in TE pH 7.5, store at −20°C (store 10 μL aliquots at −80°C for long term storage)</li>
</ol>
</li>
<li>If using Cas12a, prepare Cas12a RNPs
<ol type="a">
<li>Cas12a protein- aliquot 0.5 μL (5 μg or 32 pmol) and store at −80°C (avoid freeze/thaw cycles)</li>
<li>Cas12a-crRNA – 40 μM in TE pH 7.5, store at −20°C (store 10 μL aliquots at −80°C for long term storage)</li>
</ol>
</li>
<li>ssDNA oligo donor – 1 μg/μL in nuclease free water, store at −20°C</li>
<li>PRF4::<i>rol-6 (su1006)</i> plasmid (midi prep)<i>:</i> 500 ng/μL working solution in nuclease free water, store at −20°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Commercial Cas9 protein can be diluted 1:1 with 1× PBS and stored as 2.5 μg (0.5 μL) aliquots to reduce the total injection volume to 10 μL. We chose to reconstitute tracrRNA in Duplex buffer to provide salts to the Cas9 RNP complexes. We have not explored the long-term stability and cutting efficiencies of the injection mixtures prepared with tracrRNA reconstituted in other solvents such as TE or water. We have also not explored replacing TE or water with Duplex buffer in the injection mixture. Aliquots of Cas9 protein can be stored at −80°C for at least a year without compromising efficiencies. crRNAs and tracrRNA can be stored at −80°C for at least two years and at −20°C for at least a year without compromising efficiencies. Refer to previously published protocols if Cas9 and guide RNA plasmid-based approach is used (<a href="#bib4">Dickinson et al., 2015</a>; <a href="#bib17">Schwartz and Jorgensen, 2016</a>).</div>
<h3 id="sec1.5">Guide RNA design</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Depending on the GC content and the availability of Protospacer Adjacent Motif (PAM) sites either Cas9 (PAM: NGG; N is any DNA base) or Cas12a (Cpf1) (PAM: TTTV; V is A, C, or G) can be used as nuclease to introduce double strand breaks. In our experience Cas9 guide RNAs (gRNA) with low GC content perform poorly. It may be more efficient to use Cas12a (<a href="#bib6">Ebbing et al., 2017</a>; <a href="#bib10">Ghanta and Mello, 2020</a>) at those AT-rich target sites.</p>
<ol start="14">
<li>To generate loss of function deletions, choose two gRNA target sites such that majority of the gene is deleted. To generate HDR mediated knock-in alleles, choose a PAM site that is as close as possible to the site of insertion. Bioinformatics tools such as CRISPOR (<a href="http://crispor.tefor.net/">http://crispor.tefor.net/</a>) (<a href="#bib3">Concordet and Haeussler, 2018</a>) or CHOPCHOP (<a href="https://chopchop.cbu.uib.no">https://chopchop.cbu.uib.no</a>) (<a href="#bib11">Labun et al., 2019</a>) help in identifying gRNAs that have no off-target cleavage activity. When the choice of guide RNA is more restricted (e.g., to insert tags) these tools can be used to identify potential off-target sites that can be screened for later and crossed out of the strains. If using <i>C. elegans</i> strains other than N2, polymorphisms specific to your strains need to be considered for gRNA design and off-target assessment.</li>
</ol>
<h3 id="sec1.6">Templated precise editing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h for step 18</div>
<p>Although non-homologous end joining (NHEJ) has been shown to be the dominant type of repair mechanism in many cell types, this is not the case in the pachytene germline of <i>C. elegans</i>. When using properly prepared donor molecules, up to two thirds of the post injection progeny will exhibit homology-directed repair (<a href="#bib10">Ghanta and Mello, 2020</a>) (<a href="#bib9">Ghanta et al., 2021</a>). Depending on the length of the desired edit, one can choose to use short single stranded oligodeoxynuleotides (ssODN) (<a href="#bib1">Arribere et al., 2014</a>; <a href="#bib5">Dokshin et al., 2018</a>; <a href="#bib7">Farboud et al., 2019</a>; <a href="#bib15">Paix et al., 2017</a>; <a href="#bib16">Prior et al., 2017</a>; <a href="#bib22">Zhao et al., 2014</a>; <a href="#bib21">Ward, 2015</a>) or linear double stranded DNA (dsDNA) (<a href="#bib5">Dokshin et al., 2018</a>; <a href="#bib7">Farboud et al., 2019</a>; <a href="#bib10">Ghanta and Mello, 2020</a>; <a href="#bib14">Paix et al., 2015</a>; <a href="#bib15">Paix et al., 2017</a>; <a href="#bib20">Vicencio et al., 2019</a>) as donor templates. Refer to (<a href="#bib10">Ghanta and Mello, 2020</a>) (<a href="#bib9">Ghanta et al., 2021</a>) for HDR efficiencies achieved using the protocols described here.</p>
<ol start="15">
<li><b>ssODN Donor Design</b></li>
</ol>
<p>ssODNs up to 200 nt can be obtained commercially at reasonable prices. Therefore, any edit that is shorter than 130 bp can easily be obtained using a synthetic ssODN donor (35 nt homology arms) (<a href="#bib5">Dokshin et al., 2018</a>; <a href="#bib15">Paix et al., 2017</a>).</p>
<ol type="a">
<li>To generate short inserts: To generate short inserts (&lt;130 bp), add 5′ homology sequence (∼35 nt) in front of the tag (or mutations) and 3′ homology sequence (∼35 nt) at the end. To prevent recutting of the edited allele, introduce silent mutations into the PAM site or into the guide binding sequence in the donor molecule if it is not already split by the insert. Three or four silent mutations in the PAM proximal region should suffice and 35nt after the final silent mutation should be used as homology arm (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Donor design strategy</div>
<p>(A) If the site of insertion of a tag (knock-in) and the site of double strand break are not close to each other, disrupting the internal homology between these two sites increases the efficiency of precise repair. Sequence with disrupted homology should be considered as part of the insert and the homology arm should begin after the last silent mutation.</p>
<p>(B) Schematic design for a set of locus-specific homology arms (HA) with PCR primers which also serve as linkers between the tag and the protein of interest. For a given locus, the same set of oligos can be used to generate dsDNA donors to knock-in any tag. Plasmid containing the tag flanked by linkers is used as PCR template. Diagrams are not drawn to scale.</p>
</figcaption>
</figure>
</li>
</ol>
<ol start="2" type="a">
<li>To generate point mutations: Choose 35 nt sequence upstream and 35 nt sequence downstream of the DSB site as homology arms for the donor. Introduce the desired nucleotide changes in the donor and the PAM or guide binding sequences. Silent mutations can be introduced to create restriction sites to easily genotype precisely edited progeny of the injected animals.</li>
<li>To generate large deletions: Use two guides that remove majority of the target locus (for example, to precisely remove a structural domain or an entire coding region). Order an ssODN that contains 35 nt of sequence upstream of the left DSB and 35 nt downstream of the right DSB to act as a donor. You may wish to incorporate a new gRNA binding site on this donor so that you can re-cut the locus for any downstream purposes. Deletions of 1 kb are routinely achieved with this strategy but it can also work for larger deletions. Donor can be omitted if precise deletion is not necessary.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For both ssODNs and dsDNA donors, if the cut site is offset by several nucleotides from the desired point of donor insertion, it is critical to mutate the intervening nucleotides by recoding amino acids. Minimizing sequence identity in the donor internal to the desired insertion prevents partial insertions due to template switching (<a href="#fig1">Figure 1</a>A) (<a href="#bib5">Dokshin et al., 2018</a>; <a href="#bib15">Paix et al., 2017</a>).</div>
<ol start="16">
<li>dsDNA Donor design and generation Generate dsDNA donor templates using unmodified or 5′ Triethylene Glycol (TEG) modified primers. TEG modification is also known as spacer 9 (SP9). Melting the dsDNA donors is critical to achieve high HDR efficiencies (<a href="#bib10">Ghanta and Mello, 2020</a>). Using end-modified donors (and melting) further increases HDR efficiencies (<a href="#bib9">Ghanta et al., 2021</a>).
<ol type="a">
<li>Synthesize unmodified (or 5′ SP9 modified) PCR primers with standard desalting (IDT); 35 nt as homology arms and 20 nt complementary to the desired insert (e.g., <i>gfp</i>). SP9 modifications are available at 100 nmol scale from IDT.</li>
<li>Perform PCRs to amplify the insert (e.g., <i>gfp</i>) from a plasmid using High-Fidelity polymerase.</li>
<li>Perform agarose gel electrophoresis to ensure that a single specific band is obtained. If non-specific amplification is observed, set up a temperature gradient (50°C–64°C) and find the optimal annealing temperature.</li>
<li>Purification of donor DNA: Use one of the following methods depending on your experimental conditions.
<ol type="i">
<li>Perform PCR purification using spin-columns (e.g., Qiaquick Minelute) and elute DNA in 15–20 μL of nuclease free water. Generally, column purification is sufficient, and you may proceed to step 18e. However, some primer (55 nt long) pairs produce long dimers that may contain the homology arms. Spin-columns may not be able to remove long (&gt;80 bp) dimers completely. These short “dimer donors” out compete the full-length donors (e.g., <i>gfp</i>) and integrate heavily into the DSB site.
<div class="note">
<span class="note-title">Note:</span> Dimers may or may not be visible on the agarose gel. It is helpful to use a primer analysis tool (e.g., OligoAnalyzer-IDT) to predict homo- or hetero-dimer formation.</div>
</li>
<li>If primer-dimer formation is a concern, perform PCR clean-up using 0.6× SPRI paramagnetic beads (AMPure XP) to size-exclude shorter (&lt;200 bp) products. For example, incubate 100 μL of PCR product with 60 μL of beads, wash the beads with 70% ethanol twice and elute with nuclease free water (refer to the manufacturer’s protocol for detailed procedure).</li>
<li>Gel-extract the donor DNA if primer dimers are clearly visible on agarose gel. However, gel extracted DNA can be toxic due to the presence of guanidine hydrochloride (component of binding buffer) in the final elute. Strong absorbance at 230 nm on Nanodrop suggests presence of GuHCl in the elute. During the wash steps, incubating the spin-columns with wash buffer for 10 min before centrifugation helps in reducing salt contamination; repeat the washes 2 to 3 times. For best results, gel-extracted DNA should be further purified with 1× to 1.5× AMPure-XP beads (strongly recommended).</li>
</ol>
</li>
<li>Dilute a portion of the purified donor to 100 ng/μL and transfer about 5.5 μL (for 20 μL injection mixture) to a PCR strip tube and proceed to the melting step.</li>
<li>“Melt” the donor DNA using thermal cycler:</li>
</ol>
</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Temperature</th>
<th>Time</th>
</tr>
</thead>
<tbody>
<tr>
<td>95°C</td>
<td>2 min</td>
</tr>
<tr>
<td>85°C</td>
<td>10 s</td>
</tr>
<tr>
<td>75°C</td>
<td>10 s</td>
</tr>
<tr>
<td>65°C</td>
<td>10 s</td>
</tr>
<tr>
<td>55°C</td>
<td>1 min</td>
</tr>
<tr>
<td>45°C</td>
<td>30 s</td>
</tr>
<tr>
<td>35°C</td>
<td>10 s</td>
</tr>
<tr>
<td>25°C</td>
<td>10 s</td>
</tr>
<tr>
<td>4°C</td>
<td>hold</td>
</tr>
</tbody>
</table>
<p class="table-legend">Ramp down at 1°C/s at every step</p>
<div class="note">
<span class="note-title">Note:</span> We store purified donors at −20°C and melt them right before adding to the injection mix. We have not explored storage and re-use of melted donors. We have only tested the editing efficiencies with donors melted at 50–150 ng/μL concentrations. Melting donors at concentrations that are too high or too low may negatively influence editing efficiencies.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If the goal is to insert a common tag such as FLAG, GFP, degron etc. then it is a good idea to order a set of homology arms with “universal” PCR primers which also serve as linkers between the tag and the protein of interest. A single set of gene-specific homology arm primers can then be used to insert any tag into your locus of interest (<a href="#fig1">Figure 1</a>B). It is almost always more cost effective to order universal primers, even if inserting a short tag such as FLAG could be achieved with ssODN donors. Any gene you decide to study will be tagged in multiple different ways during the course of your work, and universal primers will save both time and money. Plasmids containing a set of commonly used tags flanked by these linkers are available from Addgene (see <a href="#key-resources-table">key resources table</a>).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Hydrofluoric acid (HF)</td>
<td>Sigma-Aldrich</td>
<td>339261</td>
</tr>
<tr>
<td>Halocarbon oil- 700 CAS#9002-83-9</td>
<td>Sigma-Aldrich</td>
<td>H8898</td>
</tr>
<tr>
<td>Agarose</td>
<td>Genesee Scientific</td>
<td>20-102GP</td>
</tr>
<tr>
<td>tracrRNA</td>
<td>IDT</td>
<td>Cat# 1072532</td>
</tr>
<tr>
<td>TE 7.5 (10 mM Tris, 0.1 mM EDTA)</td>
<td>IDT</td>
<td>Cat#11-01-02-02</td>
</tr>
<tr>
<td>Nuclease-Free Duplex Buffer (30 mM HEPES, pH 7.5; 100 mM potassium acetate)</td>
<td>IDT</td>
<td>Cat#11-01-03-01</td>
</tr>
<tr>
<td>Polyethylene Glycol 8000 (PEG)</td>
<td>MP</td>
<td>#195445</td>
</tr>
<tr>
<td>Q5 polymerase</td>
<td>NEB</td>
<td>Cat# M0491S</td>
</tr>
<tr>
<td>S. pyogenes Cas9 3NLS protein</td>
<td>IDT</td>
<td>Cat#1081058</td>
</tr>
<tr>
<td>Cas12a protein</td>
<td>IDT</td>
<td>Cat#10001272</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Gel Extraction Kit</td>
<td>QIAGEN-Qiaquick</td>
<td>#28706</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/Strains</td>
</tr>
<tr>
<td>
<i>C. elegans</i> N2 strain</td>
<td>Caenorhabditis Genetics Center</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>CRISPR-Cas9 crRNA, 2 nmol</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>A.s. Cas12a crRNA, 2 nmol</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>ssODN donors (ultramer)</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>GFP forward primer</td>
<td>This study</td>
<td>AGTAAAGGAGAAGAACTT</td>
</tr>
<tr>
<td>GFP reverse primer</td>
<td>This study</td>
<td>TTTGTATAGTTCATCCATGC</td>
</tr>
<tr>
<td>Universal linker forward</td>
<td>This study</td>
<td>TCCGGAGGGAGTGGA</td>
</tr>
<tr>
<td>Universal linker reverse</td>
<td>This study</td>
<td>AGAACCTCCGCCACC</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>GFP-linker plasmid</td>
<td>Addgene</td>
<td>N/A</td>
</tr>
<tr>
<td>FLAG-TEV-degron-linker plasmid</td>
<td>Addgene</td>
<td>N/A</td>
</tr>
<tr>
<td>mCherry linker plasmid</td>
<td>Addgene</td>
<td>N/A</td>
</tr>
<tr>
<td>PRF4::rol-6 (su1006); now named pCCM958</td>
<td>Addgene</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Glass capillaries</td>
<td>World Precision Instruments</td>
<td>#1B120F-4</td>
</tr>
<tr>
<td>Cover slips 24 x 60 mm No.1</td>
<td>Globe Scientific Inc</td>
<td>#1419-10</td>
</tr>
<tr>
<td>Cover slips 22 x 22 mm No.1</td>
<td>Globe Scientific Inc</td>
<td>#1404-10</td>
</tr>
<tr>
<td>Mouth pipette -15 In Drummond aspirator tube assembly</td>
<td>Thermo Fisher Scientific</td>
<td>#2118010</td>
</tr>
<tr>
<td>PCR purification columns</td>
<td>QIAGEN-Minelute</td>
<td>#28604</td>
</tr>
<tr>
<td>Ampure XP beads</td>
<td>Beckman Coulter</td>
<td>Ref# A63880</td>
</tr>
<tr>
<td>Tygon tubing E-3603 (ID:1/32 in; OD:3/32 in; Wall: 1/32 in)</td>
<td>Saint-Gobain</td>
<td>#00444</td>
</tr>
<tr>
<td>Microloader Tips (Femtotips)</td>
<td>Eppendorf</td>
<td>930001007</td>
</tr>
<tr>
<td>Dissecting scope</td>
<td>Nikon</td>
<td>SMZ745</td>
</tr>
<tr>
<td>Inverted microscope (DIAPHOT 200)</td>
<td>Nikon</td>
<td>Current successor: Eclipse Ti2</td>
</tr>
<tr>
<td>Fluorescence dissecting microscope</td>
<td>Zeiss</td>
<td>Axio Zoom. V16</td>
</tr>
<tr>
<td>Needle Puller Narishige PN-30</td>
<td>Narishige</td>
<td>Current successor: PC-100</td>
</tr>
<tr>
<td>Microinjector</td>
<td>Tritech Research, Inc</td>
<td>Analog MINJ-1</td>
</tr>
<tr>
<td>Micromanipulator</td>
<td>Narishige</td>
<td>MN-151</td>
</tr>
<tr>
<td>Thermal Cycler</td>
<td>Bio-Rad</td>
<td>1851148</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl2">
<caption>Reaction setup:</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5× Q5 Reaction Buffer</td>
<td>10 μL</td>
</tr>
<tr>
<td>10 mM dNTPs</td>
<td>1 μL</td>
</tr>
<tr>
<td>10 μM Forward Primer</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>10 μM Reverse Primer</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Q5 DNA Polymerase</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Template Plasmid (5 ng)</td>
<td>1.0 μL</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td>33.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>Thermocycling conditions:</caption>
<thead>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>98°C</td>
<td>1 min</td>
</tr>
<tr>
<td rowspan="3">Amplification (30 cycles)</td>
<td>98°C</td>
<td>10 s</td>
</tr>
<tr>
<td>64°C</td>
<td>20 s</td>
</tr>
<tr>
<td>72°C</td>
<td>40 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>3 min</td>
</tr>
<tr>
<td> </td>
<td>4°C</td>
<td>Hold</td>
</tr>
</tbody>
</table>
<p>If necessary, find optimal annealing temperature by performing a thermal gradient (50°C–64°C).</p>
<p>While this protocol has focused on GFP knock-in with the microinjection equipment detailed in the <a href="#key-resources-table">key resources table</a>, the general principles of the procedure described here can be used for inserts of comparable size using other equipment. GFP sequence is provided in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-Mmc1.zip">Data S1</a>.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<div class="note">
<span class="note-title">Note:</span> If using Cas9, follow steps 1–8; if using Cas12a, follow steps 9–16.</div>
<h3 id="sec3.1">Genome editing using Cas9 nuclease</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Add components of the injection mixture to the tube containing Cas9 in the following order:</p>
<ol>
<li>Prepare Cas9 – 0.5 μL of 10 μg/μL stock (30 pmol)</li>
<li>Add tracrRNA – 5 μL of 0.4 μg/μL stock (90 pmol)</li>
<li>Add crRNA – 2.8 μL of 0.4 μg/μL stock (95 pmol) (if two guides are needed add 1.4 μL of each)</li>
<li>Pipette the mixture gently several times and incubate at 37°C for 15 min.</li>
<li>Add ssODN donor – 2.2 μL of 1 μg/μL stock (or)
<ul>
<li>Add melted dsDNA – 500 ng (final concentration: 25 ng/μL for ∼1 kb donors or 45 fmol/μL)</li>
</ul>
</li>
<li>Add PRF4::<i>rol-6 (su1006)</i> plasmid – 1.6 μL of 500 ng/μL solution</li>
<li>Add nuclease free water to bring the final volume to 20 μL and pipette gently several times.</li>
<li>To avoid needle clogging, centrifuge the mixture 13000 x g for 2 min, transfer about 17 μL of the mixture to a fresh tube and keep the tube on ice; proceed to loading the needles.</li>
</ol>
<h3 id="sec3.2">Genome editing using Cas12a nuclease</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Add components of the injection mixture to the tube containing Cas12a in the following sequence:</p>
<ol start="9">
<li>Cas12a – 0.5 μL of 10 μg/μL stock (32 pmol)</li>
<li>Add Cas12a-crRNA – 2.5 μL of 40 μM stock (100 pmol)</li>
<li>Add TE pH 7.5– 3.0 μL</li>
<li>Pipette the mixture gently several times and incubate at 37°C for 15 min</li>
<li>Add ssODN donor – 2.2 μL of 1 μg/μL stock (or)
<ul>
<li>Add melted dsDNA – 500 ng (final concentration: 25 ng/μL for ∼1 kb donors or 45 fmol/μL)</li>
</ul>
</li>
<li>Add PRF4::<i>rol-6 (su1006)</i> plasmid – 1.6 μL of 500 ng/μL stock</li>
<li>Add nuclease free water to bring the final volume to 20 μL and pipette gently several times.</li>
<li>To avoid needle clogging, centrifuge the mixture at 13000 x g for 2 min, transfer about 17 μL of the mixture to a fresh tube and keep the tube on ice; proceed to loading the needles.</li>
</ol>
<div class="note"><span class="note-title">Note:</span></div>
<p>Injection mixtures can be prepared at room temperature.</p>
<p>1:3 molar ratio of nuclease to guide RNA is used to saturate the nuclease. We have not explored the cutting efficiencies with other molar ratios.</p>
<p>Protein aggregation or precipitation is not an issue at these RNP concentrations.</p>
<p>TE is added in step 11 of Cas12a mixtures for easier pipetting. This step can be omitted by further reducing the concentration of crRNA stock.</p>
<p>Although we haven’t explored the optimal dose range for ssODNs, given the efficiencies obtained with dsDNA at 25 ng/μL, much lower doses of ssODN could be used (<a href="https://star-protocols.cell.com/groups/7/protocols/973/versions/bib15">Paix et al., 2017</a>).</p>
<p>Final injection mixture can be stored at 4°C and re-used for more than a year without compromising efficiencies. We have injected 16 months old mixes and confirmed that HDR efficiencies are high.</p>
<p>In our experience adding any double stranded DNA before RNP complex formation reduces HDR efficiency.</p>
<h3 id="sec3.3">Loading needles and preparing the microscope</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<ol start="17">
<li>Using microloader tips, load 1.5 μL of the injection mixture into an etched needle.</li>
<li>Leave the needle pointing tip downward for several minutes to allow air bubbles to work their way out of the tip. This can be done easily by sticking the needle to double sided tape or clay on a vertical surface.</li>
<li>Before mounting your needle onto the micro-manipulator make sure to align and adjust the microscope. To do this place a worm on an injection pad under halocarbon oil. Bring the worm into focus under the 10× objective, then close down the condenser iris and make sure the shadow of the iris is centered in the field of view and sharply in focus. Also adjust the Nomarski optics and ensure that you have the proper filters and polarizers in the light path. Consult your manual for appropriate steps to focus the condenser and adjust the microscope. These adjustments are essential and must be done properly or the injections will be difficult if not impossible.</li>
<li>Insert the needle into the tube connected to the pressure regulator and lock it in place on the needle holder, then position the holder downward (∼15°) relative to the stage. Position the needle using the Z-axis adjustment such that it doesn’t touch the injection pad or the condenser when the microscope head is lowered.</li>
<li>With the light on and the condenser iris closed down partially, position the needle using the coarse lateral adjustments so that the needle tip is illuminated in the center of the light. If your manipulator is mounted to the condenser arm, these adjustments can be made while the arm is tilted back away from the stage. Do not look through the eyepiece when making these coarse adjustments. Instead, look directly at the needle tip as you move the lateral and longitudinal adjustments on the manipulator. First, with the needle tip pushed past the center of the light path, move it forward or backward with the coarse adjustments until it is illuminated. You will see a spark of light when the needle is in the light path (close down the condenser iris to further pinpoint the needle, via this spark of light, in the center of the field of view). Next, move the needle laterally (to the right if using a right-handed system) until the spark of light disappears, and then move it back toward the left until it lights again. Now raise the needle up close to the condenser to ensure ample working distance when you lower the condenser arm. Assuming you successfully aligned your microscope light source as described in step 19 above, your needle tip is now in the center of the field of view and will only need to be lowered in order to bring it into focus when needed.</li>
<li>Using the worm loaded earlier as a reference, focus on the worm using the 10× objective and then while watching the needle directly (not through the eyepiece) use the Z-axis adjustment to lower the needle until it touches the oil above the worm.</li>
<li>Look through the eyepiece and slowly center and bring the tip of needle into focus using X-, Y- and Z-axis adjustments on the micromanipulator. Once the tip is in focus, do not change the positions of X- or Y- axis.</li>
<li>Using 40× objective, examine the needle tip and apply pressure continuously for one or two seconds. One can judge the size of the opening (hence the quality of the needle) based on the diameter of the bubble produced in this interval. The bubble should be about the diameter of the gonad after one second. Adjust the pressure to the needle as needed. Pressures in the range of 40 to 80 psi are typically used during injection.</li>
</ol>
<h3 id="sec3.4">Mounting worms</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 min</div>
<p>Worms move by swimming in a thin aqueous layer that surrounds their body surface. Agarose pads immobilize the animals by drying this layer of moisture, sticking the animals to the pad surface. If the pad is too thick it will quickly desiccate the animal, if too thin, the animal will crawl away. Even high-quality injection pads will dry the worms out after a few minutes, so do not mount more worms than you can easily inject in a 10-to-15-min interval. With experience mounting each worm should take less than one minute.</p>
<ol start="25">
<li>Determine the upper side of the injection pad by scratching an edge of the dried agarose with a razor blade or other metal object. This scratch mark can also be used for breaking needles later if needed. Mark on the agarose side of the pad can also be used if the slide is marked with an asymmetric letter.</li>
<li>If desired, you can mark the boundaries of the agarose with a thin marker on either surface of the coverslip. Since the dried agarose is virtually transparent marking the boundaries helps in placing the worms on the agarose.</li>
<li>Use a Pasteur Pipette to transfer a small drop of halocarbon oil onto the injection pad away from the agarose (<a href="#fig2">Figure 2</a>). Use surface tension (not suction) on the pipette to move the oil to the injection pad. For working under the dissecting microscope place the injection pad on an inverted lid of petri dish for easier handling.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Mounting worms</div>
<p>Use halocarbon oil to mount worms onto the injection pad. Align all the pachytene zones (sites of injection) of all the gonad arms towards the needle. Diagrams are not drawn to scale.</p>
</figcaption>
</figure>
</li>
<li>Touch your worm pick to the surface of the halocarbon drop, and then use the viscous oil on the bottom of your pick to collect one or a few worms from your culture plate using the dissecting microscope. It is usually possible to choose worms that are off the bacterial lawn, but it may be necessary to move the worms briefly to an unseeded plate prior to picking them up with the halocarbon oil.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Excess bacteria can also be removed from the worms after transferring them to the large halocarbon oil droplet on the injection pad. After depositing the worms into the large oil droplet, flame the pick and return to the oil drop. First quench the pick in the oil away from the worms and then move the pick underneath the worms and then quickly pull the worms through the oil to remove bacteria. Once the desired number of animals are floating in the oil drop and free of bacteria, select the worms for injection by placing the pick under them in the oil and moving them by lifting them on the pick to a dry area of the pad. Depending on your abilities and your experimental needs you can transfer either one (recommended for beginners), or up to several animals onto your pad for the first round of injections.</div>
<ol start="29">
<li>Touch the pick down to the surface and move it laterally to deposit the worms in a row one after another (<a href="#fig2">Figure 2</a>). They should float off the pick in the oil. Carefully orient the worms with vulva and row of eggs facing to the left (if a right-handed system is being used). Gently push the worm down with the worm-pick by rubbing the pick along the body of the animal. If the animal is mostly stuck, then move onto the next animal. Make sure all the animals remain stuck and properly oriented. If possible, keep the worms within about 2–3 mm of each other so that you can move from one to the next without difficulty while working on the inverted microscope (<a href="#fig2">Figure 2</a>). Finally, after the worms are stuck down dip your pick back into the large oil drop on the edge of the pad, then use your pick to deposit additional oil to fully cover the immobilized worms. This additional oil will prevent the worms from desiccating during the procedure.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If you accidently orient some worms incorrectly you can either reorient them by gently pushing the worm off the pad with your pick and re-sticking them, or you can inject the worms oriented one way and then flip the slide to inject the ones facing the other way.</div>
<h3 id="sec3.5">Microinjection and Worm recovery</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<p>Microinjection into syncytium of hermaphrodite gonad (<a href="#bib13">Mello et al., 1991</a>; <a href="#bib19">Stinchcomb et al., 1985</a>) delivers Cas9-guideRNA complexes into dozens of germ nuclei simultaneously. Aim to inject the animals in the mid pachytene region. Injecting either in the distal or in the proximal end of the gonad is not ideal. Since the number of embryos produced by wildtype animals is limited by the availability of sperm, germ cells present in the distal region of the gonad arms do not get fertilized. Therefore, you should target the germ cells in the mid to late pachytene region. It should take approximately one minute to inject both gonad arms of each worm, and another minute or two to remove the injected animals to culture plates.</p>
<ol start="30">
<li>Focus the objective (40×) onto the gonad (pachytene region) so that you can see two rows of nuclei (as shown in <a href="#fig3">Figure 3</a>A). This focal plane ensures that when inserted, the tip of the needle enters the core of the gonad and not above or below the somatic gonad. Do not focus on the honeycomb structure of the germ nuclei.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Microinjection</div>
<p>(A) Using the 40× objective, focus on the two rows of germline nuclei as shown and insert the needle into the pachytene zone. Flow of the injection mixtures should reach mature oocytes in the proximal end of the germline as shown with yellow dotted line.</p>
<p>(B) Snapshots from <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-Mmc1.zip">Methods Video S1</a> are shown to illustrate the flow of the injection mixture. Germline is shown (1) before and (2) after injection. Arrows marks the region in the gonad until where the mixture has reached. 10× magnification is shown for illustration purposes, use 40× objective for the microinjection procedure.</p>
</figcaption>
</figure>
</li>
<li>Without changing the focus and while looking through the eyepiece, slowly lower the needle into the oil drop. By adjusting the position of the stage bring the cuticle (at the pachytene region) very close to the needle tip (<a href="#fig3">Figure 3</a>A).</li>
<li>Gently tap the micromanipulator so that the needle penetrates the cuticle and the somatic gonad, and enters the core germplasm. Gently move the stage towards the needle to adjust the position of the tip in the gonad. Do not insert the needle too far into the gonad as it ruptures the cuticle.</li>
<li>Press the foot pedal to inject the mixture into the germplasm and make sure that the injection mixture flows smoothly onto either side of the needle tip over a period of two seconds (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-Mmc1.zip">Methods Video S1</a>). You should see the fluid entering the proximal region of the gonad and pushing the oocytes towards the spermatheca (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-Mmc1.zip">Methods Video S1</a> and <a href="#fig3">Figure 3</a>B). If the flow is excessive the worm or gonad can explode, reduce the pressure to achieve a lower flow rate. Inject into both arms of the gonad.</li>
</ol>
<ol start="34">
<li>Raise the needle to remove the injection pad. Move the pad onto the inverted lid under the dissecting scope.</li>
<li>Recover the injected worms using mouth pipette (pulled glass capillary attached to Drummond aspirator tube) and sterile M9 buffer or MPEG (0.5% PEG in M9). Occasionally, worms stick to the glass pipette during the recovery process, and they are less likely to stick when MPEG is used. Other buffers are not necessary. Avoid using plastic pipette tips to recover the injected worms because they will stick to the tips.</li>
<li>If the injections differed in quality, for example if only one arm was injected, these differences should be noted, and the worms separated, NGM plates labeled according to injection quality. When all the worms are injected place each injected animal singly onto a fresh NGM plate and culture for about 3.5 days at room temperature (22°C–23°C) unless genetic or experimental conditions demand other temperature conditions.</li>
</ol>
<h3 id="sec3.6">Screening and genotyping</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>If both arms of the hermaphrodite gonad are injected, a good injection should yield 20–40 F1 Rollers. Screening strategy depends on the type of edit generated.</p>
<ol start="37">
<li>About 72 h post injection, score the number of F1 Rollers and choose two plates with the highest number. The Rollers provide an important trouble-shooting metric (see below) and should always be included whenever possible.
<ol type="a">
<li>For indels (without donor template) or ssODN donors, choose two P0 plates that segregate the highest number of F1 Rollers; pick 12–24 young adult F1 Rollers onto separate plates and allow them to have progeny prior to genotyping by PCR.</li>
<li>For long dsDNA donors screening should be performed differently depending on the quality of the injections and the type of edit desired. If excellent bilateral injections were performed resulting in greater than 20 Roller progeny per injected animal, then choose two plates that segregate the highest number of F1 Rollers. If a fluorescent protein tag was inserted a quick look under the compound fluorescence microscope (or a dissecting Fluorescence microscope if the fluorescence is expected to be bright) will allow you to gauge how many F1 progeny to pick. To check under the compound microscope (preferred for genes whose expression levels are low or unknown) place ∼50 gravid adult F1 progeny younger than the rollers onto 2% wet agarose pads on a microscope slide (do not use dry injection pads) and immobilize with levamisole, then push the worms into proximity with the pick, gently place a cover slip on the top and check for fluorescent signal. If you wish you can memorize the location of positive animals and while working on a dissecting scope push the cover slip aside (do not flick it open) to directly recover the fluorescent positive animals. If PCR genotyping is the desired approach for detection, choose about 24 non-Rollers that are younger than the Rollers and place them onto separate NGM plates. Younger animals among Rollers can also be picked. For inexperienced injectors, we recommend using 5′ end-modified dsDNA donors and picking F1 Rollers.</li>
</ol>
</li>
<li>Genotype the F1 adults directly after allowing them to lay progeny. Alternatively, to avoid PCR false positives due to mosaicism, lyse several F2s from each plate and genotype. At least one of the genotyping primers should lie outside of the homology arms to avoid amplification from transiently retained donor molecules.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Strains obtained by genome editing should be back crossed to eliminate off-target indels. However, off-target indels on the same chromosome and close to the target site may be linked to the target site. Therefore, generating at least two independent alleles using different gRNAs is recommended.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol is used to generate precise genome edits in <i>C. eleg</i>ans germline. Loss of function alleles, point mutants and fluorescent tags can be generated with high efficiencies. A single injected animal can produce precision edits in among as many as two thirds of progeny from the post-injection cohort (<a href="#bib10">Ghanta and Mello, 2020</a>). These advances increase the utility of <i>C. elegans</i> for understanding the fundamentals of animal biology and for investigating the physiology of alleles implicated in disease.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Obtaining high knock-in efficiencies of large inserts (for example, 3 kb or more) is still a challenge. Future improvements in protocols and development of new strategies are required to address this limitation.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Worms mounted onto injection pads dry immediately (step 29)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Ensure that the agarose solution is made with water. Beginners may mistakenly use DNA gel running buffers instead of water.</p>
<p>Ensure that the pads are thoroughly dried by baking and that they are not too thick</p>
<p>Animals that are too young tend to dry much faster</p>
<p>Ensure that the animals have never starved and are well fed</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Worms don’t stick to the injection pads (step 29)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Injection pads may not be dry. Re-bake the pads at 80°C for an hour</p>
<p>Worms may be covered with excess amounts of bacteria. Make sure that bacteria are not transferred onto the injection pad</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Needles clog during injections (step 33)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Dust particles in the injection mixture clog the needles. Make sure that the mixtures are spun down at high speed for at least 2 min prior to loading into the needles</p>
<p>High concentrations of protein could clog the needles. Ensure that the correct concentrations are used in the injection mixture</p>
<p>Needle opening may be too small: gently touch the tip of the needle to the agarose to break it open</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Needle opening is either too wide or too narrow, or tip is too flexible (bends when pushed against cuticle (step 32)).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Adjust the settings of the needle puller to obtain tapered stiff needles</p>
<p>Adjust the contact time of the needles in HF</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Low survival rate of the injected animals (step 35)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Choose young adults for injection</p>
<p>Blunt needles with wide openings could severely damage the cuticle</p>
<p>Injected worms should be recovered with care using M9 buffer</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Small brood sizes after injection (step 37)</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Small brood sizes are observed if the target locus of the guide RNA is essential for viability</p>
<p>Make sure that the donor DNA is free of contaminants such as high salts and that the concentrations of all the injection mixture components are optimal</p>
<p>Use young adults for injections. Old animals do not yield many progeny</p>
<p>Do not inject excess amounts of the mixture. If flow is excessive, reduce the pressure to lower the flow rate</p>
<h3 id="sec6.13">Problem 7</h3>
<p>No Rollers (step 37)</p>
<h3 id="sec6.14">Potential solution</h3>
<p>Check the integrity of the <i>rol-6</i> marker plasmid on agarose gel. If degradation is observed, re-extract plasmid DNA (midi-prep).</p>
<p>Use appropriate concentrations of the Cas9 RNPs, donor DNA and <i>r</i><i>ol-6</i> plasmid as suggested in the protocol. Excess amounts of Cas9 RNPs or DNA can be toxic.</p>
<p>For beginners, practicing microinjection procedure by injecting Prf-4: <i>r</i><i>ol-6</i> plasmid alone can be helpful. More than 40 F1 Rollers can be obtained from each injected animal.</p>
<p>If loss of function alleles of the target locus impair gamete or embryo viability, the number of F1 Rollers produced will be dramatically low.</p>
<h3 id="sec6.15">Problem 8</h3>
<p>No indels (step 38)</p>
<h3 id="sec6.16">Potential solution</h3>
<p>AT rich Cas9 guide RNAs can be inefficient. If the target sequence is AT rich, switch to Cas12a based editing.</p>
<p>Use appropriate concentrations of Cas9 and crRNAs as suggested in the protocol. We recommend using a final concentration of 1.5 μM Cas9 RNPs in the mixture. Increasing the concentration to 3 μM does not increase cutting efficiency (<a href="#bib5">Dokshin et al., 2018</a>).</p>
<h3 id="sec6.17">Problem 9</h3>
<p>Indels at homologous locus (step 38)</p>
<h3 id="sec6.18">Potential solution</h3>
<p>If an identical target site exists in another locus, majority of the F1 progeny may have indels at both the loci. Use a different guide to avoid indels at the homologous locus.</p>
<p>If the guide choice is limited, reduce Cas9 RNP concentration by five-fold. Overall indel efficiencies may drop but single mutants can be easily isolated.</p>
<h3 id="sec6.19">Problem 10</h3>
<p>No GFP insertions (steps 37 and 38)</p>
<h3 id="sec6.20">Potential solution</h3>
<p>Make sure that the dsDNA donor is melted before adding to the Cas9 RNP mixture</p>
<p>Donor DNA should be free of any primer dimer or smaller DNA species that can potentially inhibit HDR efficiencies</p>
<p>It is difficult to obtain HDR insertions with inefficient guide RNA. Genotype and sanger sequence the target locus in about 12-24 Rollers and ensure that cutting efficiency of the guide is high. Alternatively, TIDE analysis on pooled F1 genomic DNA can be performed (<a href="#bib5">Dokshin et al., 2018</a>) (<a href="#bib2">Brinkman et al., 2014</a>). Also, rule out partial insertion of the donor (e.g., due to primer dimer insertion).</p>
<p>If the expression pattern of the protein is unknown it may be helpful to perform fluorescence-based screens with animals of mixed stages including older embryos on the plates.</p>
<p>If the initial screening was performed by fluorescence microscopy, genotype the locus by PCR and sanger sequence the target site. Low expression levels may be the reason for “No GFP”.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Arribere, J.A., Bell, R.T., Fu, B.X., Artiles, K.L., Hartman, P.S., and Fire, A.Z. (2014). Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in Caenorhabditis elegans. Genetics <i>198</i>, 837-846. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref1">View at publisher</a></p>
<p id="bib2">Brinkman, E.K., Chen, T., Amendola, M., and van Steensel, B. (2014). Easy quantitative assessment of genome editing by sequence trace decomposition. Nucleic Acids Res. <i>42</i>, e168. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref2">View at publisher</a></p>
<p id="bib3">Concordet, J.P. and Haeussler, M. (2018). CRISPOR: intuitive guide selection for CRISPR/Cas9 genome editing experiments and screens. Nucleic Acids Res. <i>46</i>, W242-W245. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref3">View at publisher</a></p>
<p id="bib4">Dickinson, D.J., Pani, A.M., Heppert, J.K., Higgins, C.D., and Goldstein, B. (2015). Streamlined genome engineering with a self-excising drug selection cassette. Genetics <i>200</i>, 1035-1049. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref4">View at publisher</a></p>
<p id="bib5">Dokshin, G.A., Ghanta, K.S., Piscopo, K.M., and Mello, C.C. (2018). Robust genome editing with short single-stranded and long, partially single-stranded DNA donors in Caenorhabditis elegans. Genetics <i>210</i>, 781-787. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref5">View at publisher</a></p>
<p id="bib6">Ebbing, A., Shang, P., Geijsen, N., and Korswagen, H. (2017). Extending the CRISPR toolbox for C. elegans: Cpf1 as an alternative gene editing system for AT-rich sequences. MicroPubl Biol. <i>2017</i>, 10.17912/W2237D. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref6">View at publisher</a></p>
<p id="bib7">Farboud, B., Severson, A.F., and Meyer, B.J. (2019). Strategies for efficient genome editing using CRISPR-Cas9. Genetics <i>211</i>, 431-457. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref7">View at publisher</a></p>
<p id="bib8">Fire, A. (1986). Integrative transformation of Caenorhabditis elegans. EMBO J. <i>5</i>, 2673-2680. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref8">View at publisher</a></p>
<p id="bib9">Ghanta, K.S., Chen, Z., Mir, A., Dokshin, G.A., Krishnamurthy, P.M., Yoon, Y., Gallant, J., Xu, P., Zhang, X.-O., Ozturk, A., et al. (2021). 5′ modifications improve potency and efficacy of DNA donors for precision genome editing. bioRxiv, 354480. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref9">View at publisher</a></p>
<p id="bib10">Ghanta, K.S. and Mello, C.C. (2020). Melting dsDNA donor molecules greatly improves precision genome editing in Caenorhabditis elegans. Genetics <i>216</i>, 643-650. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref10">View at publisher</a></p>
<p id="bib11">Labun, K., Montague, T.G., Krause, M., Torres Cleuren, Y.N., Tjeldnes, H., and Valen, E. (2019). CHOPCHOP v3: expanding the CRISPR web toolbox beyond genome editing. Nucleic Acids Res. <i>47</i>, W171-W174. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref11">View at publisher</a></p>
<p id="bib12">Mello, C. and Fire, A. (1995). DNA transformation. Methods Cell Biol. <i>48</i>, 451-482. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref12">View at publisher</a></p>
<p id="bib13">Mello, C.C., Kramer, J.M., Stinchcomb, D., and Ambros, V. (1991). Efficient gene transfer in C.elegans: extrachromosomal maintenance and integration of transforming sequences. EMBO J. <i>10</i>, 3959-3970. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref13">View at publisher</a></p>
<p id="bib14">Paix, A., Folkmann, A., Rasoloson, D., and Seydoux, G. (2015). High efficiency, homology-directed genome editing in Caenorhabditis elegans using CRISPR-Cas9 ribonucleoprotein complexes. Genetics <i>201</i>, 47-54. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref14">View at publisher</a></p>
<p id="bib15">Paix, A., Folkmann, A., and Seydoux, G. (2017). Precision genome editing using CRISPR-Cas9 and linear repair templates in C. elegans. Methods <i>121-122</i>, 86-93. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref15">View at publisher</a></p>
<p id="bib16">Prior, H., Jawad, A.K., MacConnachie, L., and Beg, A.A. (2017). Highly efficient, rapid and Co-CRISPR-independent genome editing in Caenorhabditis elegans. G3 (Bethesda) <i>7</i>, 3693-3698. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref16">View at publisher</a></p>
<p id="bib17">Schwartz, M.L. and Jorgensen, E.M. (2016). SapTrap, a toolkit for high-throughput CRISPR/Cas9 gene modification in Caenorhabditis elegans. Genetics <i>202</i>, 1277-1288. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref17">View at publisher</a></p>
<p id="bib18">Stiernagle, T. (2006). Maintenance of C. elegans. In WormBook, pp. 1-11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref18">View at publisher</a></p>
<p id="bib19">Stinchcomb, D.T., Shaw, J.E., Carr, S.H., and Hirsh, D. (1985). Extrachromosomal DNA transformation of Caenorhabditis elegans. Mol. Cell Biol. <i>5</i>, 3484-3496. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref19">View at publisher</a></p>
<p id="bib20">Vicencio, J., Martinez-Fernandez, C., Serrat, X., and Ceron, J. (2019). Efficient generation of endogenous fluorescent reporters by nested CRISPR in Caenorhabditis elegans. Genetics <i>211</i>, 1143-1154. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref20">View at publisher</a></p>
<p id="bib21">Ward, J.D. (2015). Rapid and precise engineering of the Caenorhabditis elegans genome with lethal mutation co-conversion and inactivation of NHEJ repair. Genetics <i>199</i>, 363-377. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref21">View at publisher</a></p>
<p id="bib22">Zhao, P., Zhang, Z., Ke, H., Yue, Y., and Xue, D. (2014). Oligonucleotide-based targeted gene editing in C. elegans via the CRISPR/Cas9 system. Cell Res. <i>24</i>, 247-250. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00455-X/sref22">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Craig C. Mello (<a href="mailto:craig.mello@umassmed.edu">craig.mello@umassmed.edu</a>).</p>
<h4>Materials availability</h4>
<p>All materials are available upon reasonable request.</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze any data sets</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/974-Mmc1.zip">Data S1. GFP sequence</a></p>
<h3>Acknowledgments</h3>
<p>We thank the members of the Mello lab and the worm community for sharing their observations and difficulties which helped in making the protocols better. This work was funded by <a href="https://doi.org/10.13039/100000002">NIH</a> grant R37GM058800 to C.C.M. C.C.M is a Howard Hughes Medical Investigator. Some strains were provided by the CGC, which is funded by <a href="https://doi.org/10.13039/100000002">NIH</a> Office of Research Infrastructure Programs (P40 OD010440).</p>
<h3>Author contributions</h3>
<p>K.S.G. and C.C.M. wrote the manuscript. K.S.G., T.I., and C.C.M. performed the experiments. C.C.M. supervised the study.</p>
<h3>Declaration of interests</h3>
<p>Some of the findings described here are part of the patent applications filed by the University of Massachusetts Medical School with K.S.G and C.C.M. as co-inventors. C.C.M. is a scientific founder and advisory board member of CRISPR therapeutics.</p>
</section>
</article>