<article>
<section>
<h1 id="header">Unveiling the K<sup>+</sup>-sensitivity of cell metabolism using genetically encoded, FRET-based K<sup>+</sup>, glucose, and ATP biosensors</h1>
<p><time datetime="2021-09-24">Published: September 24, 2021</time></p>
<p>Helmut Bischof,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> Sandra Burgstaller,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Wolfgang F. Graier,<sup><a href="#aff2">2</a>,<a href="#aff4">4</a></sup> Robert Lukowski,<sup><a href="#aff1">1</a></sup> and Roland Malli<sup><a href="#aff2">2</a>,<a href="#aff4">4</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pharmacology, Toxicology and Clinical Pharmacy, Institute of Pharmacy, University of Tuebingen, Auf der Morgenstelle 8, 72076 Tuebingen, Germany</p>
<p id="aff2"><sup>2</sup>Gottfried Schatz Research Center, Molecular Biology and Biochemistry, Medical University of Graz, Neue Stiftingtalstraße 6/6, 8010 Graz, Austria</p>
<p id="aff3"><sup>3</sup>Natural and Medical Sciences Institute, University of Tuebingen, 72770 Reutlingen, Germany</p>
<p id="aff4"><sup>4</sup>BioTechMed-Graz, Mozartgasse 12/II, 8010 Graz, Austria</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:helmut.bischof@uni-tuebingen.de">helmut.bischof@uni-tuebingen.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:roland.malli@medunigraz.at">roland.malli@medunigraz.at</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100843">10.1016/j.xpro.2021.100843</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Investigating dynamic changes of mitochondrial ATP and cytosolic glucose levels of single living cells over time by genetically encoded biosensors provides an informative readout of their metabolic activities. Here, we describe how to monitor the metabolic K<sup>+</sup>-sensitivity of HEK293 cells exploiting ATP-, glucose-, and K<sup>+</sup> probes. Fluorescence live-cell imaging of these Förster resonance energy transfer-based biosensors over time in response to gramicidin, an ionophoric peptide, indicated an absolute dependency of cellular ATP homeostasis on high intracellular K<sup>+</sup> levels.</p>
<p>For complete information on the generation and use of this protocol please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00549-9/sref2">Bischof et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Single live-cell imaging allows the metabolic phenotyping of (cancer) cell lines</li>
<li>FRET-based probes allow high-temporal resolution imaging</li>
<li>Protocol allows us to investigate the K<sup>+</sup> sensitivity of (cancer) cell metabolism</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1035-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The workflow described in this protocol takes several days and requires preparation and access to special equipment (see <a href="#materials-and-equipment">materials and equipment</a>). Before starting the experiment, buffers and media should be prepared.</p>
<div class="note">
<span class="note-title">Note:</span> As the optimal conditions for cell growth, we suggest to follow the guidelines recommended by the provider of the cell lines. Accordingly, the cell number, cell culture medium, supplements, and times for preparing cells that express the genetically encoded probes indicated in this protocol might also vary depending on the cell type used. This protocol will describe the specific steps for HEK293 cells. However, we have also used this protocol with other cell lines including HeLa, INS-1 832/13, MCF-7 or MDA-MB-453 cells. For each cell line, a 10 cm cell culture dish at a confluency of ∼90% yielded enough cells for several experiments. For other cell lines, we expect, as mentioned above, that only minor adjustments of some experimental procedures will be necessary.</div>
<h3 id="sec1.1">Purification of sensor plasmids from <i>E.coli</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 0</div>
<ol>
<li>Sufficient amounts of the plasmids FLII12Pglu-700μδ6 (<a href="#bib5">Takanaga et al., 2008</a>), mtAT1.03 (<a href="#bib3">Imamura et al., 2009</a>), mt lc-LysM GEPII 1.0 and NES lc-LysM GEPII 1.0 (<a href="#bib1">Bischof et al., 2017</a>) encoding for the respective genetically-encoded, FRET-based probes need to be prepared before the experiments.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Conventional plamid Maxiprep systems including an endotoxin removal step can be used for DNA purification. DNA preparations should be performed according to the manufacturer’s instructions. Upon plasmid purification, the concentration of the DNA (μg/μL) should be determined and can be stored at 4°C for up to 6 months or at −18°C for up to 1 year.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The DNA purity needs to be checked (<a href="#bib4">Lucena-Aguilar et al., 2016</a>), as RNA or protein contaminations might affect transfection efficiency or cell viability. After the DNA purification, the DNA should importantly be verified by sequencing to check for potential DNA mutations affectingsensor functionalities.</div>
<h3 id="sec1.2">Culture of cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 0</div>
<ol start="2">
<li>HEK293 cells utilized in this protocol were cultivated in DMEM + 10% fetal calf serum (FCS) + 1<b>×</b> penicillin-streptomycin + 1<b>×</b> sodium pyruvate. Cells were cultivated in a humidified incubator at 37°C and 5% CO<sub>2</sub>. Cell splitting and trypsinization were performed when cells reached a confluency of 90% using 0.05% Trypsin-EDTA in PBS.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Any disturbances that could potentially influence cell metabolism and –activities (temperature fluctuations, CO<sub>2</sub> fluctuations, altering the nutrient composition of the medium,…) should be avoided as far as possible.</div>
<h3 id="sec1.3">Preparation of the assay buffers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 2</div>
<ol start="3">
<li>For the metabolic phenotyping of the different cell lines by fluorescence microscopy using genetically-encoded probes, different buffers are required. The composition of these buffers can be found in the “<a href="#materials-and-equipment">materials and equipment</a>” section below.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>3-Bromo-2-oxopropionic acid (3-BP)</td>
<td>Sigma Aldrich</td>
<td>N/A</td>
</tr>
<tr>
<td>50<b>×</b> MEM Amino Acids Solution</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>100<b>×</b> MEM Vitamin Solution</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>100<b>×</b> Sodium Pyruvate Solution</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Antimycin-A</td>
<td>Abcam</td>
<td>N/A</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> ⋅ 2H<sub>2</sub>O</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>D-Glucose ⋅ H<sub>2</sub>O</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>DMEM, high Glucose, HEPES</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Fetal Calf Serum</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Gramicidin</td>
<td>Sigma Aldrich</td>
<td>N/A</td>
</tr>
<tr>
<td>HEPES</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>KCl</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>L-Glutamine (200 mM)</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> ⋅ 6H<sub>2</sub>O</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>NaCl</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>NaOH</td>
<td>Carl Roth</td>
<td>N/A</td>
</tr>
<tr>
<td>Oligomycin-A</td>
<td>Tocris</td>
<td>N/A</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (100<b>×</b>)</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Phosphate Buffered Saline</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.5%)</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Human: HEK293</td>
<td>ATCC</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>mt lc-LysM GEPII 1.0, K<sup>+</sup> sensor targeted to the mitochondrial matrix</td>
<td>Next Generation Fluorescence Imaging GmbH</td>
<td>N/A</td>
</tr>
<tr>
<td>NES lc-LysM GEPII 1.0, K<sup>+</sup> sensor targeted to the cytosol</td>
<td>Next Generation Fluorescence Imaging GmbH</td>
<td>N/A</td>
</tr>
<tr>
<td>pcDNA3.1 FLII<sup>12</sup>Pglu-700μδ6, glucose sensor targeted to the cytosol</td>
<td>Addgene</td>
<td>N/A</td>
</tr>
<tr>
<td>mtAT1.03, ATP-sensor targeted to the mitochondrial matrix</td>
<td>Imamura et al.</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Excel Office 365</td>
<td>Microsoft</td>
<td>N/A</td>
</tr>
<tr>
<td>Prism 5</td>
<td>GraphPad</td>
<td>N/A</td>
</tr>
<tr>
<td>VisiView</td>
<td>Visitron Systems</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>6-well plates</td>
<td>Corning</td>
<td>N/A</td>
</tr>
<tr>
<td>15 mL Centrifuge tubes</td>
<td>Corning</td>
<td>N/A</td>
</tr>
<tr>
<td>30 mm circular glass coverslips</td>
<td>Paul Marienfeld GmbH &amp; Co. KG, Lauda-Königshofen, Germany</td>
<td>N/A</td>
</tr>
<tr>
<td>37°C incubator, 5% CO<sub>2</sub>, humidified</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>37°C water-bath</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Bürker-Türk cell counting chamber</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Omicron LEDHub</td>
<td>Omicron, Klaus, Austria</td>
<td>N/A</td>
</tr>
<tr>
<td>xcitation filter</td>
<td>IDEX Health &amp; Science, Rochester, NY, USA</td>
<td>N/A</td>
</tr>
<tr>
<td>Emission filter</td>
<td>IDEX Health &amp; Science, Rochester, NY, USA</td>
<td>N/A</td>
</tr>
<tr>
<td>Multichannel perfusion system</td>
<td>NGFI GmbH, Graz, Austria</td>
<td>N/A</td>
</tr>
<tr>
<td>Optosplit II Image Splitter</td>
<td>Cair Research, Faversham, UK</td>
<td>N/A</td>
</tr>
<tr>
<td>PC</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Pco.panda 4.2 bi</td>
<td>PCO AG, Kelheim, Germany</td>
<td>N/A</td>
</tr>
<tr>
<td>Perfusion Chamber PC30</td>
<td>NGFI GmbH, Graz, Austria</td>
<td>N/A</td>
</tr>
<tr>
<td>Perfusion System PS-9D</td>
<td>NGFI GmbH, Graz, Austria</td>
<td>N/A</td>
</tr>
<tr>
<td>Sterile work bench</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Vacuum pump</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Zeiss Axio Observer Z1</td>
<td>Carl Zeiss AG, Oberkochen, Germany</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>We highly recommend preparing stock solutions in ddH<sub>2</sub>O to minimize inaccuracies.</p>
<table id="undtbl1">
<caption>Formulation of cell equilibration buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub> ⋅ 2H<sub>2</sub>O</td>
<td>2 mM</td>
<td>20 mL of 0.1 M</td>
</tr>
<tr>
<td>D-Glucose ⋅ H<sub>2</sub>O</td>
<td>10 mM</td>
<td>10 mL of 1 M</td>
</tr>
<tr>
<td>HEPES</td>
<td>10 mM</td>
<td>1 mL of 1 M</td>
</tr>
<tr>
<td>KCl</td>
<td>5 mM</td>
<td>50 mL of 0.1 M</td>
</tr>
<tr>
<td>
<span class="small-caps">l</span>-Glutamine</td>
<td>2 mM</td>
<td>10 mL of 100<b>×</b>
</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>0.44 mM</td>
<td>4.4 mL of 0.1 M</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> ⋅ 6H<sub>2</sub>O</td>
<td>1 mM</td>
<td>10 mL of 0.1 M</td>
</tr>
<tr>
<td>NaCl</td>
<td>135 mM</td>
<td>135 mL of 1 M</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>2.6 mM</td>
<td>2.6 mL of 1 M</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>0.34 mM</td>
<td>3.4 mL of 0.1 M</td>
</tr>
<tr>
<td>MEM amino acids</td>
<td>1<b>×</b>
</td>
<td>20 mL of 50<b>×</b>
</td>
</tr>
<tr>
<td>MEM vitamins</td>
<td>1<b>×</b>
</td>
<td>10 mL of 100<b>×</b>
</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>1<b>×</b>
</td>
<td>10 mL of 100<b>×</b>
</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">After mixing all components, the pH should be adjusted to 7.4 using NaOH. The cell equilibration buffer should be sterile filtered and can be stored at 4°C in the dark for up to 2 months.</p>
<table id="undtbl2">
<caption>Formulation of glucose free imaging buffer (IB – GLU)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub> ⋅ 2H<sub>2</sub>O</td>
<td>2 mM</td>
<td>20 mL of 0.1 M</td>
</tr>
<tr>
<td>HEPES</td>
<td>10 mM</td>
<td>1 mL of 1 M</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> ⋅ 6H<sub>2</sub>O</td>
<td>1 mM</td>
<td>10 mL of 0.1 M</td>
</tr>
<tr>
<td>NaCl</td>
<td>138 mM</td>
<td>135 mL of 1 M</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">After mixing all components, pH should be adjusted to 7.4 using NaOH. Nutrient-free imaging buffer can be stored at 4°C for up to 1 week.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Cell counting and seeding</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 1, 1 h</div>
<p>Handling the cells for this experiment does not require special treatment. We recommend optimizing the seeded cell number empirically.</p>
<div class="note">
<span class="note-title">Note:</span> The following steps will describe the protocol for using HEK293 cells. The protocol might be adjusted for the use with other cell lines. All media and buffers getting in contact with the cells should be prewarmed to 37°C using a water bath.</div>
<ol>
<li>Remove the cells cultivated within a 10 cm dish to a confluency of ∼90% from the incubator and detach the cells under sterile conditions.
<ol type="a">
<li>Dilute Trypsin-EDTA to the desired final concentration with PBS. Typically, 0.05% of Trypsin-EDTA is sufficient for the trypsinization of HEK293 cells.</li>
<li>Put the plate under the sterile cell culture hood, remove the cell culture medium, and replace the cell culture medium twice with 10 mL of pre-warmed PBS.</li>
<li>Remove PBS and add 3 mL of a prewarmed, 0.05% Trypsin-EDTA in PBS.</li>
<li>Put the cells back into the incubator and check for cell detachment after 2 min. If cells have not detached after 5 min, keep incubating them at 37°C and check for detachment every 1 min.</li>
<li>When cells are floating, stop trypsinization by adding 10 mL of pre-warmed complemented medium (DMEM + 10% FCS + 1<b>×</b> penicillin-streptomycin + 1<b>×</b> sodium pyruvate).</li>
<li>Transfer the cells to a 15 mL conical tube and pellet them at 200 × g for 5 min.</li>
<li>After centrifugation, remove the supernatant from the cells without disturbing the cell pellet and fully resuspend the cell pellet in 10 mL of complemented-medium (DMEM + 10% FCS + 1<b>×</b> penicillin-streptomycin + 1<b>×</b> sodium pyruvate) carefully.</li>
</ol>
</li>
<li>Determine the cell number/milliliter by counting the cells using a Bürker-Türk counting chamber or any other method of choice.
<ol type="a">
<li>When using a Bürker-Türk counting chamber, use 15 μL of cell suspension for one chamber. We recommend the determination of the cell number in duplicates by counting 2 × 4 large squares.</li>
<li>For each of the two chambers, divide the number of total cells counted in the 4 large squares by 4 and take the average of the two chambers.</li>
<li>Multiply the number of cells × 10.000, which gives you the cell number/milliliter.</li>
</ol>
</li>
<li>Seed HEK293cells into the desired number of wells in a 6-well plate containing 30 mm circular glass coverslips and cultivate the cells for 16–24 h.
<ol type="a">
<li>Prepare the desired number of 6-well plates.
<ol type="i">
<li>Put 30 mm circular glass coverslips in the desired number of wells in 6-well plates.</li>
<li>Add 1 mL of complemented medium (DMEM + 10% FCS, + penicillin/ streptomycin, + 1<b>×</b> sodium pyruvate) and distribute the medium in the wells.</li>
<li>Using a sterile pipette tip, push the coverslips to the bottom of the 6-well plate to ensure that the coverslips firmly stick to the bottom of the plate to prevent cell growth beneath the glass coverslip.</li>
</ol>
</li>
<li>Dilute the cells to an appropriate cell number/milliliter and seed into the wells of the 6-well plate containing the glass coverslips.
<ol type="i">
<li>The cell number that needs to be seeded varies with the time between seeding the cells and performing the experiment. For HEK293 cells that should be investigated 2 days after seeding, typically 300.000 cells per well are sufficient to achieve a density of approximately 70–80% on the day of the experiment, which is ideal for the assays.</li>
<li>Ideally, the final culture volume per well is 1.5 mL. Dilute the cell number of the HEK293 cells to 600.000/mL and add 500 μL of the cell suspension to each well of the 6-well plate containing the glass coverslip, yielding a final culture volume of 1.5 mL and a cell number of 300.000 cells/well.</li>
</ol>
</li>
<li>Put the 6-well plates into a humidified incubator with 37°C and 5% CO<sub>2</sub> for 16–24 h and cultivate the cells.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Cell transfection with the sensor plasmids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 2, 1 h</div>
<p>The genetic information of the sensors, i.e., the plasmids, needs to be delivered to the cells. Therefore, different transfection methods, electroporation, or viral transduction may be used.</p>
<div class="note">
<span class="note-title">Note:</span> In this protocol, we describe how to transfect HEK293 cells using the PolyJet transfection reagent. Depending on the transfection reagent, the protocol may vary, and other reagents/methods for plasmid delivery might be required for different cell types.</div>
<ol start="4">
<li>Perform a medium change of the cells seeded the day before.
<ol type="a">
<li>Remove the cells from the incubator (humidified, 37°C, 5% CO<sub>2</sub>) and check the cells using a cell culture microscope for their morphology, density, and possible bacterial or fungal contaminations.
<ol type="i">
<li>If the cells should be measured at the fluorescence microscope the next day, HEK293 cells should show a density of approximately 40%–50% on the day of the transfection.</li>
</ol>
</li>
<li>Remove the medium from each well that will be transfected with a sensor plasmid and carefully add 1 mL of fresh and prewarmed complemented medium (DMEM + 10% FCS, + penicillin/ streptomycin, + 1<b>×</b> sodium pyruvate) to each well.</li>
<li>After changing the medium, return the cells to the incubator for 30 min.</li>
</ol>
</li>
<li>Prepare the transfection mixture using the plasmid of choice and PolyJet transfection reagent. For PolyJet transfection also see: <a href="https://signagen.com/DataSheet/SL100688.pdf">https://signagen.com/DataSheet/SL100688.pdf</a>
<ol type="a">
<li>Approximately 20 min after putting the cells back into the incubator, start with preparing the transfection mixture.
<ol type="i">
<li>Therefore, 50 μL of FCS-free DMEM medium are prepared for each well that will be transfected with a plasmid. Hence, if two 6-well plates will be transfected with two different probes (probe A and B), 2 × 300 μL of serum-free DMEM are required and put in separate tubes (tubes A and B). Per transfected well, 1.0 μg of plasmid DNA should be added. For mitochondria-targeted constructs, the DNA amount should be reduced to 0.3 μg/well to ensure proper mitochondrial targeting of the probes.</li>
<li>For each well that will be transfected, 50 μL of FCS-free DMEM are prepared in a separate tube (tube C) for the transfection reagent. Hence, if two 6-well plates will be transfected (this is independent of the number of different plasmids), 600 μL of serum-free DMEM are pipetted in tube C. Subsequently, 3 μL of PolyJet transfection reagent are added for each well that should be transfected and the tube is mixed by pipetting.</li>
<li>After the DNA (tube A and B) and PolyJet transfection reagent (tube C) were prepared, the DNA-containing medium and PolyJet-medium mixtures are mixed at a 1:1 ratio. Hence, if 300 μL of the DNA-medium mixture are in tube A, 300 μL of the PolyJet-media mixture from tube C are added, followed by thorough mixing by pipetting. The same applies to the second sensor in tube B, which is also mixed in a 1:1 ratio with 300 μL of tube C. Mixtures are incubated for 10 min at 20°C–25°C.</li>
</ol>
</li>
<li>After 10 min of incubating the DNA-transfection mixtures, the transfection mixtures are added to the cells.
<ol type="i">
<li>The cells are removed from the incubator and 100 μL of each transfection mixture is added in a drop-by-drop manner to each well of the 6-well plate that should be transfected with the respective probe.</li>
<li>Once the mixtures are added, the cells are returned to the incubator and incubated at 37°C with 5% CO<sub>2</sub> for 12–16 h.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the cell type used, other transfection reagents, incubation times, plasmid- or transfection reagent amounts may be required.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The transfection mixture containing the transfection reagent and the plasmid DNA must be prepared in FCS free medium. Furthermore, after mixing the reagent and plasmid DNA together, the vial should incubate without perturbations to allow the transfection complex to form.</div>
<h3 id="sec3.3">Preparing cells for the experiments</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 3, 20 min</div>
<p>On the day of an experiment, the cells should be removed from the incubator and the transfection mixture is replaced with the cell equilibration buffer. This allows the cells to acclimate to the environment of the recording conditions, which ensures data consistency.</p>
<ol start="6">
<li>12–16 h after adding the transfection mixture to the cells, the transfection mixture should be removed and cells should be equilibrated in the cell equilibration buffer.
<ol type="a">
<li>Pre-warm PBS and the cell-equilibration buffer to 37°C using a water bath.</li>
<li>Remove the cells from the incubator and check the morphology, density, and possible contaminations using a cell culture microscope.</li>
<li>Exchange the medium of the cells for the cell-equilibration buffer.
<ol type="i">
<li>Remove all medium from the cells.</li>
<li>Carefully replace the medium 1<b>×</b> with PBS by adding 1.5 mL of PBS / well to remove any residual medium.</li>
<li>Subsequently, remove the PBS and carefully add 1.5 mL/well of the cell-equilibration buffer. Keep cells in the dark at 20°C–25°C until they are measured. We recommend pre-equilibrating the cells for at least 30 min to allow their adaption to their new environment.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cell pre-equilibration with the cell-equilibration buffer may be omitted if desired and cells might directly be measured from the incubator.</div>
<div class="optional">
<span class="optional-title">Optional:</span> The cells can also be cultivated for a longer period after their transfection (e.g. additional 24 h). In such cases, the transfection mixture should be exchanged for fresh complemented medium (DMEM + 10% FCS + 1<b>×</b> penicillin-streptomycin + 1<b>×</b> sodium pyruvate) after 12–16 h. The cells can then be further cultivated in DMEM + 10% FCS + 1<b>×</b> penicillin-streptomycin + 1<b>×</b> sodium pyruvate until the experiments will be conducted.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to remove the cationic lipids contained in the transfection mixture after 16 hours as a prolonged exposure will increase the toxcicity of the mixture.</div>
<h3 id="sec3.4">Complementing buffers for the experiments</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 3, 30 min</div>
<p>Depending on the information that should be obtained from the experiments, a nutrient-free imaging buffer should be complemented with specific nutrients or modulators of cell metabolism such as inhibitors/activators of oxidative or glycolytic activity.</p>
<div class="note">
<span class="note-title">Note:</span> In this protocol, we will describe how to complement the buffers for extracting information about the mitochondrial activity, as well as cell metabolic glucose and HKII dependency, and intracellular K<sup>+</sup> sensitivity. This analysis is based on the use of 3-bromo-2-oxopropionic acid (3-BP), oligomycin-A, and antimycin-A which either inhibit hexokinase-2 or the ATP-synthase and complex III, respectively. Additionally, gramicidin is used to deplete intracellular K<sup>+</sup> stores.</div>
<ol start="7">
<li>Complement the buffers with the required compounds.
<ol type="a">
<li>Pre-warm nutrient-free imaging buffer to 20°C–25°C.</li>
<li>Aliquot 200 mL of buffer in a new bottle without adding additional compounds (= IB -GLU).</li>
<li>Add 10 mM of <span class="small-caps">d</span>-glucose to the residual 800 mL of buffer (= IB + GLU).</li>
<li>Aliquot 200 mL of basal imaging buffer in a new bottle and add 3-BP at a final concentration of 300 μM (= IB + 3-BP) from a 300 mM H<sub>2</sub>O stock (1:1000 dilution).</li>
<li>Aliquot 200 mL of basal imaging buffer in a new bottle and add oligomycin-A and antimycin-A at final concentrations of 3 μM and 5 μM (= IB + O/A) from 10 mM DMSO stocks, respectively (1:3333 and 1:2000 dilutions, respectively).</li>
<li>Aliquot 200 mL of basal imaging buffer in a new bottle and add gramicidin at final concentrations of 15 μM (= IB + GRAM) from a 15 mM DMSO stock (1:1000 dilution).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other concentrations, modulators, or combinations of metabolic modulators may be used to receive further information about cell metabolic activity.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ideally, buffers are freshly complemented with glucose and the metabolic modulators (3-BP, oligomycin-A and antimycin-A, and gramicidin) on the day of the imaging experiments due to the potential instability of the compounds in an aqueous solution.</div>
<h3 id="sec3.5">Preparing the microscope and the perfusion system for the experiments</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 3, 30 min</div>
<p>Prior to the measurements, the microscope, PC, imaging software, and perfusion system software should be started, followed by the adjustment of the parameters for the subsequent measurements.</p>
<ol start="8">
<li>Start the PC and open the perfusion system software to prepare the perfusion system.
<ol type="a">
<li>Wash the perfusion system to get rid of any residual components left from older experiments.
<ol type="i">
<li>Open the number of channels that are necessary for the measurements + one for cleaning, and wash every syringe of the perfusion system with 10 mL of 70% ethanol.</li>
<li>When the ethanol has passed, add 20 mL of ddH<sub>2</sub>O per syringe to remove residual ethanol.</li>
</ol>
</li>
<li>Close the valves of the perfusion system using the perfusion system software and prepare the system with selected buffers.
<ol type="i">
<li>Fill one buffer in each of the syringes.</li>
<li>Shortly open the valves of the perfusion system to remove air from the tubings.</li>
<li>As the buffers contain toxic components, repeat steps 5.a.i.–5.a.ii. with the channel used for cleaning to remove residual compounds.</li>
<li>Once all air is removed from the tubes and the system is cleaned, the channel containing the “basal buffer” (IB + GLU) should be opened to prepare the system for the first experiment and the flow rate of the perfusion system should be adjusted to approximately 1 mL/min using tube clamps. Subsequently, perfusion can be stopped by closing all valves in the software.</li>
</ol>
</li>
</ol>
</li>
<li>Open the Imaging software and adjust the imaging parameters.
<ol type="a">
<li>All the sensors presented in this protocol represent FRET-based probes utilizing CFP and YFP, hence, the excitation maximum of all probes is close to 430 nm and emission maxima are close to 475 nm (CFP) and 530 nm (FRET).
<ol type="i">
<li>Using the described microscope setup, at a binning of 4, a LED intensity at 455 nm of ∼1%–2%, an exposure time of ∼200 ms and an interval time of ∼3 s are good starting points.</li>
<li>Adjust the settings of the microscope for simultaneous detection of CFP- and FRET emission upon excitation with the 455 nm LED and the 427/10 excitation filter.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend using a 40<b>×</b> oil immersion objective to investigate the proper localization of the fluorescent probes upon expression in the mammalian cells.</div>
<div class="note">
<span class="note-title">Note:</span> Light intensities should be minimized to prevent phototoxic effects and photobleaching as much as possible. To maintain suitable fluorescent intensities for measurements, the camera binning can be increased to yield higher fluorescence emission signals from the cells, while keeping excitation light intensities low.</div>
<div class="note">
<span class="note-title">Note:</span> Depending on the expected duration of the experiment, imaging intervals should be adjusted to prevent phototoxic effects and photobleaching as much as possible. For longer measurements (e.g. slow responses/effects), imaging intervals of several seconds (for example 10 s) might be chosen instead of imaging every 3 s.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Although imaging parameters have to be adjusted for every sensor, these parameters should not be changed when measuring the same sensor on a different day. Keeping the same settings for one sensor guarantees consistency (e.g. comparable dynamic range) throughout various measurements (either on the same day or on different days).</div>
<h3 id="sec3.6">Preparing the experiments</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 3, 10 min</div>
<ol start="10">
<li>After 30 min of cell equilibration in the cell equilibration buffer, experiments can be started.
<ol type="a">
<li>Carefully transfer a glass coverslip containing the cells from the 6-well plate to the perfusion chamber.
<ol type="i">
<li>A 25G needle and sharp forceps are helpful for the coverslip transfer. Alternatively, a commercially available lens lifter can be used.</li>
<li>Ensure that no edge of the coverslip is damaged or broken, as this will lead to leakiness of the perfusion chamber.</li>
<li>Properly place the coverslip in the notch of the lower part of the perfusion chamber (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1035-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Representation of the perfusion chamber</div>
<p>Photograph of the (A) unmounted and (B) mounted perfusion chamber.</p>
<p>(C) Displays the perfusion chamber placed on top of a microscope stage. Inlet and outlet of the chamber are indicated.</p>
</figcaption>
</figure>
</li>
<li>Subsequently, close the perfusion chamber by inserting the sealing ring in the upper part of the chamber (<a href="#fig1">Figure 1</a>A) and put the upper part on top of the lower part (<a href="#fig1">Figure 1</a>B). Add 1 mL of the cell equilibration buffer to ensure hydration of the cells.</li>
<li>Tightly close the perfusion chamber by twisting (<a href="#fig1">Figure 1</a>B) and remove excess buffer with a paper towel. Check the tightness of the perfusion chamber by using a fresh, dry paper towel and carefully move around the bottom of the glass coverslip. If the chamber is tight, the towel will remain dry.</li>
</ol>
</li>
<li>Place the perfusion chamber on top of the microscope and start perfusing the cells with IB + GLU.
<ol type="i">
<li>If using an oil immersion objective, add a drop of fluorescence-suitable immersion oil on the objective.</li>
<li>Put the perfusion chamber on top of the microscope and fix it (<a href="#fig1">Figure 1</a>C).</li>
<li>Attach the tube of the perfusion system to the inlet (on one side) of the perfusion chamber (<a href="#fig1">Figure 1</a>C).</li>
<li>Open the valve of IB + GLU buffer to start perfusing the cells.</li>
<li>Check for functional perfusion (i.e., increasing volume in the chamber) and attach the tube of the vacuum pump to the other side of the chamber (outlet) to prevent buffer overflow and to ensure a laminar flow of the buffer across the coverslip (<a href="#fig1">Figure 1</a>C).</li>
</ol>
</li>
<li>Screen the coverslip for a suitable imaging section with cells showing a good signal to noise ratio and proper targeting of the respective fluorescent probe (<a href="#fig2">Figure 2</a>). <a href="#sec7.1">Troubleshooting 1</a>, <a href="#sec7.3">2</a> and <a href="#sec7.5">3</a>.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1035-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative fluorescence wide-field images of HEK293 cells</div>
<p>Images display fluorescence wide-field images of the CFP- (cyan, upper images) and FRET-fluorescence (yellow, lower images) of HEK293 cells expressing (from left to right): FLII<sup>12</sup>Pglu-700μδ6, mtAT1.03, NES lc-LysM GEPII 1.0 or mt lc-LysM GEPII 1.0. Arrow indicates a cell where a mitochondrial construct is mistargeted in the cytosol/ nucleus. Scale bar represents 20 μm.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Once a section was found, the imaging experiment can be started.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For fluorescent probes targeted to the mitochondrial matrix (mtAT1.03 and mt lc-LysM GEPII 1.0), ensure that cells show a correct targeting of these sensors (solely mitochondrial fluorescence) and do not show cytosolic and/or nuclear mistargeting. We highly recommend performing a preliminary experiment testing for correct targeting and expression of the sensor. Excessive mistargeting can also be caused by the usage of too high DNA or PolyJet transfection reagent concentrations. Also ensure that the cell confluency within the area of interest is not too high, as this will lead to slower cell responses due to limited buffer/compound diffusion.</div>
<h3 id="sec3.7">Measurement of cytosolic glucose and K<sup>+</sup> and mitochondrial ATP and K<sup>+</sup> dynamics</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 3, ∼25 min per experiment</div>
<p>Upon identification of cells suitable for imaging, the actual experiment can be started.</p>
<ol start="12">
<li>Select your cells of interest and follow the fluorescence over-time of the CFP- and FRET emissions.
<ol type="a">
<li>Start recording the images at a distinct light intensity, exposure time, binning, and interval. Make sure to not select the brightest cells, as it will then be difficult to keep the settings for the next experiments. Cells possessing average fluorescence intensities that are representative for the whole population might be measured instead.</li>
<li>Draw regions of interest around every cell of interest. Also include one background region, which is important for later analysis. Make sure to select a background region that does not show any fluorescence signal coming from neighboring cells.</li>
<li>Record the baseline of both fluorescence intensities in IB+GLU buffer for 5 min. <a href="#sec7.7">Troubleshooting 4</a>.</li>
<li>After 5 min, switch to one of the experimental buffers, either containing no glucose (IB -GLU), 3 μM oligomycin-A and 5 μM antimycin-A (IB + O/A), 300 μM 3-BP (IB +3-BP) or 15 μM gramicidin (IB + GRAM).</li>
<li>After switching to the experimental buffers, keep recording the fluorescence over-time values for at least 10 min or until the ratio over-time has stabilized again. <a href="#sec7.9">Troubleshooting 5</a>.</li>
<li>Perform steps a-e for cells transfected with each sensor (mtAT1.03, FLII<sup>12</sup>Pglu-700μδ6, NES lc-LysM GEPII 1.0, and mt lc-LysM GEPII 1.0, at least 5 times on 3 different days (with the same settings for each sensor) to ensure data consistency and to ensure a stable average for each treatment and sensor (<a href="#bib1">Bischof et al., 2017</a>; <a href="#bib3">Imamura et al., 2009</a>; <a href="#bib5">Takanaga et al., 2008</a>).</li>
<li>After the experiment has finished, stop the image acquisition, export the fluorescence over-time values, e.g., to Microsoft Excel, stop the perfusion system, withdraw the coverslip and extensively wash the perfusion system and -chamber with ethanol and subsequently with ddH<sub>2</sub>O.</li>
<li>Start the next experiment by repeating steps 7–9.</li>
</ol>
</li>
</ol>
<h3 id="sec3.8">Analyze data</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Day 4, 2 h</div>
<p>After performing the experiments and data export to Microsoft Excel, analysis can be performed for data interpretation and representation.</p>
<ol start="13">
<li>Correct the fluorescence over-time values of every cell measured for the corresponding background intensity.
<ol type="a">
<li>Subtract the fluorescence over-time of the cyan channel from the CFP signal of the cells and subtract the fluorescence over-time values of the FRET-channel from the FRET signal of the cells.</li>
</ol>
</li>
<li>Calculate the FRET-ratio signal by dividing the background-corrected values of FRET/CFP and plot the FRET-ratio signal over-time (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1035-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative responses of HEK293 cells upon gramicidin treatment</div>
<p>(A) FRET-ratio signals over-time of several HEK293 cells either expressing FLII<sup>12</sup>Pglu-700μδ6 (upper left), mtAT1.03 (upper right), NES lc-LysM GEPII 1.0 (lower left) or mt lc-LysM GEPII 1.0 (lower right), respectively, in response to the administration of 15 μM gramicidin at time-point indicated in the panels.</p>
<p>(B) Shows the representative single cell fluorescence over-time courses of CFP- (cyan) and FRET-signal (yellow) of the respective FRET-based probes as demonstrated in (A) of one single cell in response to the same treatment.</p>
</figcaption>
</figure>
<ol type="a">
<li>In case of observing a signal in the FRET-ratio over-time, ensure that the signal is ratiometric, i.e., CFP- and FRET-intensities show opposite responses (<a href="#fig3">Figure 3</a>B). A de- or increase in both channels will most likely point to other effects rather than a FRET change due to binding of the respective ion/metabolite.</li>
</ol>
</li>
<li>If desired, correct the FRET-ratio over-time for potential fluorescence bleaching effects using the following equation:</li>
</ol>
<p class="math-ml"><math> <mrow> <msub> <mi>R</mi> <mn>0</mn> </msub> <mo>=</mo> <msub> <mi>R</mi> <mrow> <mi>i</mi> <mi>n</mi> <mi>i</mi> <mi>t</mi> <mi>i</mi> <mi>a</mi> <mi>l</mi> </mrow> </msub> <mo>⋅</mo> <mi>exp</mi> <mrow> <mo>(</mo> <mrow> <mo>−</mo> <mi>K</mi> <mo>⋅</mo> <mi>T</mi> <mi>i</mi> <mi>m</mi> <mi>e</mi> </mrow> <mo>)</mo> </mrow> <mo>+</mo> <msub> <mi>R</mi> <mrow> <mi>p</mi> <mi>l</mi> <mi>a</mi> <mi>t</mi> <mi>e</mi> <mi>a</mi> <mi>u</mi> </mrow> </msub> </mrow> </math></p>
<p>R<sub>0</sub> = function of the bleaching correction curve</p>
<p>R<sub>initial</sub> = maximal fluorescence ratio signal once imaging has started</p>
<p>K = rate constant of fluorescence bleaching over-time</p>
<p>R<sub>plateau</sub> = minimal fluorescence ratio reached by bleaching over-time</p>
<p>One of the easiest ways to correct for photobleaching using the given formula is to use the one phase decay function provided by GraphPad Prism (<a href="#fig4">Figure 4</a>A).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1035-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Representative bleaching correction of a single-cell response</div>
<p>(A) FRET-ratio signal over-time (R, black solid line) of a HEK293 cell expressing NES lc-LysM GEPII 1.0 in response to gramicidin treatment and the corresponding, proper bleaching correction curve (R<sub>0</sub>, gray dashed line). R<sub>0</sub> was generated using GraphPad Prism 5 software.</p>
<p>(B) Displays the normalized and bleaching corrected FRET-ratio signal over-time of the response demonstrated in (A) as a result of dividing R/R<sub>0</sub>.</p>
<p>(C and D) demonstrate an example of an overcompensated bleaching correction of the same curve as demonstrated in (A).</p>
</figcaption>
</figure>
<ol start="16">
<li>Normalize the data by dividing the FRET-ratio signal by the R<sub>0</sub> function over-time (<a href="#fig4">Figure 4</a>B) and check the bleaching correction for its correctness. An example of an overcompensated photobleaching correction is demonstrated in <a href="#fig4">Figures 4</a>C and 4D.</li>
<li>To receive the metabolic phenotype of cells, plot the average of the normalized [GLU]<sub>cyto</sub> response of every cell analyzed over the average of the normalized [ATP]<sub>mito</sub> alteration of every cell analyzed for each of the different treatments. Subtracting the value “1” from both average responses will yield the net change of [GLU]<sub>cyto</sub> and [ATP]<sub>mito</sub> from the basal value (<a href="#fig5">Figure 5</a>). Such representation significantly reduces the number of panels and allows to observe the effect of metablic interventions directly at one glance.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1035-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Representative results of [GLU]<sub>cyto</sub> and [ATP]<sub>mito</sub> measurements in HEK293 cells upon gramicidin treatment</div>
<p>(A and B) (A) [GLU]<sub>cyto</sub> and (B) [ATP]<sub>mito</sub> recordings over-time of HEK293 cells in response to treatment with gramicidin at time point indicated in the panels. Single cell responses (gray lines) and average response of the whole cell population (black line) are shown.</p>
<p>(C) Displays the result of plotting the averaged [GLU]<sub>cyto</sub> over [ATP]<sub>mito</sub> after baseline subtraction. In total, the figure shows the gramicidin sensitivity of [GLU]<sub>cyto</sub> and [ATP]<sub>mito</sub> of HEK293 cells in response to gramicidin treatment.</p>
</figcaption>
</figure>
</li>
<li>To receive the gramicidin sensitivity of the cytosolic and mitochondrial [K<sup>+</sup>] of cells, plot the average of the normalized [K<sup>+</sup>]<sub>cyto</sub> alteration of every cell analyzed over the average of the normalized [K<sup>+</sup>]<sub>mito</sub> alteration of every cell analyzed for each of the different treatments by subtracting the value “1” from both average responses will yield the net change of [K<sup>+</sup>]<sub>cyto</sub> and [K<sup>+</sup>]<sub>mito</sub> from the basal value (<a href="#fig6">Figure 6</a>). Such representation significantly reduces the number of panels and allows to observe the effect of metablic interventions directly at one glance.
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1035-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Representative results of [K<sup>+</sup>]<sub>cyto</sub> and [K<sup>+</sup>]<sub>mito</sub> measurements in HEK293 cells upon gramicidin treatment</div>
<p>(A and B) (A) [K<sup>+</sup>]<sub>cyto</sub> and (B) [K<sup>+</sup>]<sub>mito</sub> recordings over-time of HEK293 cells in response to gramicidin treatment as indicated in the panels. Single cell responses (gray lines) and average response of the whole cell population (black line) are shown.</p>
<p>(C) Displays the result of plotting the averaged [GLU]<sub>cyto</sub> over [ATP]<sub>mito</sub> after baseline subtraction. In total, the shows displays the gramicidin sensitivity of [K<sup>+</sup>]<sub>cyto</sub> and [K<sup>+</sup>]<sub>mito</sub> of HEK293 cells in response to gramicidin treatment.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Photobleaching represents an unwanted side-effect. Although it can be reduced to an absolute minimum by adjusting imaging parameters, corrections, as described, may be necessary.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Check the correctness of the photobleaching correction procedure by comparing the uncorrected raw curves/responses with the curves after correction. Bad curve fitting may lead to over- or underestimation of responses/ effects, hence, always double-check. Other corrections than the one-phase-decay might also be tested in case of bad fitting. The exact shape of the plots shown in <a href="#fig5">Figures 5</a>C and <a href="#fig6">6</a>C requires an extremly thoroughful and exact timing of the experiments. For example, the total duration of the entire experiment as well as the superfusion of the cells with e.g., gramicidin needs to follow identical schedules for all measurements that will be transferred to XY plots.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Performing the experiments as described above, will yield the metabolic phenotype of cell lines concerning their glucose and ATP homeostasis and will demonstrate the correlation between glucose/ATP dynamics. Also the other treatments can be represented as described, which will yield the metabolic phenotype of cells (<a href="#fig7">Figure 7</a>A) and the K<sup>+</sup>-sensitivity/phenotype (<a href="#fig7">Figure 7</a>B). Additionally, the experiments will demonstrate mitochondrial and/or cytosolic K<sup>+</sup> fluctuations in response to the different metabolic interventions and their interrelation.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1035-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Results of the metabolic- and K<sup>+</sup> homeostasis phenotyping of HEK293 cells</div>
<p>[GLU]<sub>cyto</sub> and [ATP]<sub>mito</sub>sensitivity (A) and [K<sup>+</sup>]<sub>cyto</sub> and [K<sup>+</sup>]<sub>mito</sub>sensitivity (B) of HEK293 cells in response to different metabolic interventions as indicated, including the removal of extracellular glucose (-Glucose, gray curves), or the administration of gramicidin (+ Gramicidin, magenta curves), the administration of oligomycin-A and Antimycin-A (+Oligo/ Anti, green curves) or 3-bromo-2-oxopropionic acid (3-BP). Part of the figure has been published in <a href="#bib2">Bischof et al. (2021)</a>. Figure reuse with permission from <a href="#bib2">Bischof et al. (2021)</a>.</p>
</figcaption>
</figure>
<p>Besides visualizing metabolite/K<sup>+</sup> dynamics over-time, already the basal FRET-ratio signal contains information about the basal concentration of the analytes in the cytosol or mitochondria, which can differ significantly between cell types and/ or pretreatment conditions and should therefore also be analyzed separately by comparing the basal values of different treatments/ conditions using appropriate statistical tests.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Data can be represented and analyzed using any program of choice. We recommend either plotting every single cell response over-time in a separate graph for each treatment or represent data as described above. Experiments should at least be replicated for a minimum of 3 times on 3 independent days to ensure data consistency, reproducibility, and a stable means.</p>
<p>In case of a statistical comparison of treatments is desired, we recommend testing the data for normal distribution to choose the proper statistical test. Usually, an unpaired t-test (if data are normally distributed) or a Mann-Whitney-U test (if data are not normally distributed) should be used for pairwise comparison of two conditions, or a One-Way ANOVA (if data are normally distributed) or Kruskal Wallis test (if data are not normally distributed) should be used for comparison of multiple conditions.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>First, due to the use of a perfusion system, cells that should be investigated should show good adherence to the glass coverslip, to not detach them when starting the experiment.</p>
<p>Second, cells that are investigated are exposed to slight shear stress, which can, however, be regulated by adjusting the flow rate. To compare measurements, make sure to adjust the perfusion system to the same speed as a higher speed might lead to faster cell responses and, in turn, to false interpretation.</p>
<p>Third, cells are exposed to potential effects of phototoxicity. Hence, optimizing the imaging parameters in terms of using low light intensities and exposure times may avoid possible toxic side effects caused by excitation light.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No fluorescence signal is detected when observing the cells under the microscope (steps 10 - 12).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>First, check the imaging settings of the microscope, as something might be wrong with these parameters (e.g., excitation light, filter settings, light path).</p>
<p>Second, check transfection efficiency of your cells of interest using other, in your lab well-known plasmids and eventually check transfection by fluorescence imaging, western blot, qPCR or any other method of choice. If transfection efficiency confirms to be at a low level, try changing to other transfection reagents or use other methods of transduction, for example, viral delivery systems or electroporation.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Cells detach when perfusion is started (steps 10–12).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Reduce the flow rate of perfusion. If this does not help and cells still start to detach, try to coat the glass coverslips with a coating reagent of choice. We recommend using poly-L-lysine, for HEK293 cells at a concentration of 0.25 mg/mL.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Sensors targeted to mitochondria show high levels of mistargeting to the cytosol/nucleus (steps 10 - 12).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>In case of high levels of mistargeting are observed upon expression of mitochondrial targeted biosensors, a change of the transfection reagent or reducing the amount of plasmid used for transfection might be helpful. Reducing the time between transfection and imaging might be also helpful.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>No stable baseline can be recorded, as FRET ratio values are continuously raising or dropping (step 12).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>If no stable baseline appears during imaging, photobleaching may be the reason. Try to optimize the imaging conditions by reducing light intensity, exposure time and/or acquisition intervals.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Cells do not respond to any treatment (step 12).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Ensure that cells are not pre-stimulated with any of the compounds used in the perfusion system. Excessive washing of the perfusion system and -chamber may solve the problem.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bischof, H., Rehberg, M., Stryeck, S., Artinger, K., Eroglu, E., Waldeck-Weiermair, M., Gottschalk, B., Rost, R., Deak, A.T., Niedrist, T., et al. (2017). Novel genetically encoded fluorescent probes enable real-time detection of potassium in vitro and in vivo. Nat. Commun. <i>8</i>, 1422. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00549-9/sref1">View at publisher</a></p>
<p id="bib2">Bischof, H., Burgstaller, S., Springer, A., Matt, L., Rauter, T., Bachkönig, O.A., Schmidt, T., Groschner, K., Schindl, R., Madl, T., et al. (2021). Potassium ions promote hexokinase-II dependent glycolysis. Iscience <i>24</i>, 102346. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00549-9/sref2">View at publisher</a></p>
<p id="bib3">Imamura, H., Huynh Nhat, K.P., Togawa, H., Saito, K., Iino, R., Kato-Yamada, Y., Nagai, T., and Noji, H. (2009). Visualization of ATP levels inside single living cells with fluorescence resonance energy transfer-based genetically encoded indicators. PNAS <i>106</i>, 15651-15656. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00549-9/sref3">View at publisher</a></p>
<p id="bib4">Lucena-Aguilar, G., Sánchez-López, A.M., Barberán-Aceituno, C., Carrillo-Ávila, J.A., López-Guerrero, J.A., and Aguilar-Quesada, R. (2016). DNA source selection for downstream applications based on DNA quality indicators analysis. Biopreserv. Biobank. <i>14</i>, 264-270. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00549-9/sref4">View at publisher</a></p>
<p id="bib5">Takanaga, H., Chaudhuri, B., and Frommer, W.B. (2008). GLUT1 and GLUT9 as major contributors to glucose influx in HepG2 cells identified by a high sensitivity intramolecular FRET glucose sensor. Biochim. Biophys. Acta <i>1778</i>, 1091-1099. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00549-9/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact Roland Malli, <a href="mailto:roland.malli@medunigraz.at">roland.malli@medunigraz.at</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>Original/source data for the paper is available from the lead contact upon request.</p>
<h3>Acknowledgments</h3>
<p>This work was funded by the <a href="https://doi.org/10.13039/100003579">Alexander von Humboldt Foundation</a> (H.B.), the <a href="https://doi.org/10.13039/501100002428">Austrian Science Fund</a> (FWF), project numbers I3716 (R.M.) and W1226 (W.F.G.), the German Research Foundation (DFG), project numbers RL 1490/10-1 (R.L.), and the ICEPHA Graduate Program ‘‘Membrane-associated Drug Targets in Personalized Cancer Medicine’’ (R.L.).</p>
<h3>Author contributions</h3>
<p>H.B., S.B., W.F.G., R.L., and R.M. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>