<article>
<section>
<h1 id="header">Purification of mouse hepatic non-parenchymal cells or nuclei for use in ChIP-seq and other next-generation sequencing approaches</h1>
<p><time datetime="2021-03-12">Published: March 12, 2021</time></p>
<p>Ty D. Troutman,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Hunter Bennett,<sup><a href="#aff2">2</a></sup> Mashito Sakai,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Jason S. Seidman,<sup><a href="#aff2">2</a></sup> Sven Heinz,<sup><a href="#aff1">1</a></sup> and Christopher K. Glass<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Medicine, University of California, San Diego, La Jolla, CA 92093, USA</p>
<p id="aff2"><sup>2</sup>Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla, CA 92093, USA</p>
<p id="aff3"><sup>3</sup>Department of Biochemistry &amp; Molecular Biology, Nippon Medical School, 1-1-5 Sendagi, Bunkyo-ku, Tokyo 113-8602, Japan</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ttroutman@health.ucsd.edu">ttroutman@health.ucsd.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:ckg@health.ucsd.edu">ckg@health.ucsd.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100363">10.1016/j.xpro.2021.100363</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Significant advancements in understanding disease mechanisms can occur through combined analysis of next-generation sequencing datasets generated using purified cell populations. Here, we detail our optimized protocol for purification of mouse hepatic macrophages (or other liver non-parenchymal populations) suitable for use in various next-generation sequencing protocols. An alternative framework is described for sorting pre-fixed hepatic nuclei populations. This strategy has the advantage of rapidly preserving the nuclei and can facilitate success with ChIP-seq for more challenging molecules.</p>
<p>For complete details on the use and execution of these protocols, please refer to <a href="#bib8">Muse et al. (2018)</a>, <a href="#bib12">Sakai et al. (2019)</a>, and <a href="#bib14">Seidman et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Isolation of high-purity hepatic non-parenchymal cells or hepatic nuclei using FACS</li>
<li>Recovered cells are suitable for (sc)RNA-seq, (sc)ATAC-seq, ChIP-seq, etc</li>
<li>Recovered nuclei are suitable for ChIP-seq, chromatin conformation assays, etc</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/513-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Experimental considerations</h3>
<p>This protocol contains two modular strategies for preparing mouse liver cell populations for chromatin immunoprecipitation with sequencing (ChIP-seq), or other next-generation sequencing assays. In the first module, we describe an optimized protocol for preparing single-cell suspensions of mouse liver non-parenchymal cells (NPCs) for immunolabeling with fluorescent antibodies and population purification using fluorescence-activated cell sorting (FACS). We describe in detail a strategy for FACS purification of Kupffer cells and provide abbreviated examples of FACS strategies for purification of either liver sinusoidal endothelial cells or hepatic stellate cells. Additional cell populations can also be defined as suited. At the end of this module, the cells are ready for use in diverse next-generation sequencing assays. Examples are described in processing FACS-purified cells for RNA-seq, ATAC-seq, or ChIP-seq.</p>
<p>The second module describes an alternative strategy for cell population purification better suited for ChIP-seq of challenging transcriptional regulators. The rationale for this is that preparation of cells and purification by FACS can take many hours, in which time transcriptional regulators can dissociate from the genome. Instead, in this module mouse livers are perfused immediately after euthanasia with fixative and nuclei are prepared for sorting. This module contains a detailed strategy for purification of Kupffer cell nuclei using either a genetic nuclear reporter or immunolabeling of nuclei with fluorescent antibodies to the macrophage lineage determining transcription factor PU.1. This strategy can be extended to other liver populations of interest. Samples from either of these strategies are suitable for use in an extensively validated ChIP-seq library preparation protocol described by <a href="https://star-protocols.cell.com/protocols/491">Texari et al.</a></p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-mouse/human CD11b PE (clone M1/70)</td>
<td>BioLegend</td>
<td>101208; RRID: AB_312791</td>
</tr>
<tr>
<td>Anti-mouse CD146 PE/Cy7 (clone ME-9F1)</td>
<td>BioLegend</td>
<td>134714; RRID: AB_2563109</td>
</tr>
<tr>
<td>Anti-mouse CD16/32 (clone 93)</td>
<td>BioLegend</td>
<td>101302; RRID: AB_312801</td>
</tr>
<tr>
<td>Anti-mouse CD45 BB515 (clone 30-F11)</td>
<td>BD Biosciences</td>
<td>564590; RRID: none</td>
</tr>
<tr>
<td>Anti-mouse F4/80 BV421 (clone BM8)</td>
<td>BioLegend</td>
<td>123132; RRID: AB_11203717</td>
</tr>
<tr>
<td>Anti-mouse Tim4 Alexa647 (clone RMT4-54)</td>
<td>BioLegend</td>
<td>130008; RRID: AB_2271648</td>
</tr>
<tr>
<td>Anti-PU.1 Alexa647 (7C2C34)</td>
<td>BioLegend</td>
<td>681304; RRID:AB_2566591</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>0.5 M EGTA, pH 8.0, DNase- and RNase-free (ethyleneglycol-bis(2-aminoethylether)-N,N,N′,N′-tetraacetic acid)</td>
<td>bioWORLD</td>
<td>40121266-3</td>
</tr>
<tr>
<td>0.5 M UltraPure EDTA, pH 8.0 (ethylenediaminetetraacetic acid)</td>
<td>Thermo Fisher</td>
<td>15575020</td>
</tr>
<tr>
<td>10% Tween 20</td>
<td>Teknova</td>
<td>T0027; CAS: 900564-5</td>
</tr>
<tr>
<td>1 M 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES)</td>
<td>Teknova</td>
<td>H1090</td>
</tr>
<tr>
<td>1 M magnesium chloride</td>
<td>Millipore Sigma</td>
<td>63069</td>
</tr>
<tr>
<td>1 M UltraPure Tris-HCl pH 8.0</td>
<td>Invitrogen</td>
<td>15568-025</td>
</tr>
<tr>
<td>1× RBC lysis buffer</td>
<td>Invitrogen</td>
<td>00-4333-57</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Millipore Sigma</td>
<td>D8418</td>
</tr>
<tr>
<td>Disuccinimidyl glutarate (DSG)</td>
<td>ProteoChem</td>
<td>c1104-100mg; CAS: 79642-50-5</td>
</tr>
<tr>
<td>DNase I</td>
<td>Worthington</td>
<td>LS002139</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Flavopiridol</td>
<td>Sigma-Aldrich</td>
<td>F3055; CAS: 131740-09-5</td>
</tr>
<tr>
<td>Formaldehyde</td>
<td>Thermo Fisher Scientific</td>
<td>BP531-500; CAS: 7732-18-5, 50-00-0, 67-56-1</td>
</tr>
<tr>
<td>Glycine</td>
<td>Fisher Scientific</td>
<td>BP381-5</td>
</tr>
<tr>
<td>Hank’s balanced salt solution (no calcium, no magnesium, no phenol red)</td>
<td>Thermo Fisher</td>
<td>14175095</td>
</tr>
<tr>
<td>Hank’s balanced salt solution (with calcium, with magnesium, no phenol red)</td>
<td>Thermo Fisher</td>
<td>14025092</td>
</tr>
<tr>
<td>HBSS (10×), no calcium, no magnesium, no phenol red</td>
<td>Thermo Fisher</td>
<td>14185052</td>
</tr>
<tr>
<td>IGEPAL CA-630</td>
<td>Millipore Sigma</td>
<td>I8896</td>
</tr>
<tr>
<td>Invitrogen OneComp eBeads</td>
<td>Thermo Fisher</td>
<td>01-1111-42</td>
</tr>
<tr>
<td>Liberase™</td>
<td>Sigma-Aldrich</td>
<td>5401127001</td>
</tr>
<tr>
<td>OptiPrep</td>
<td>Sigma-Aldrich</td>
<td>D1556; CAS: 92339-11-2</td>
</tr>
<tr>
<td>Percoll</td>
<td>Sigma-Aldrich</td>
<td>P1644; CAS: 65455-52-9</td>
</tr>
<tr>
<td>Phosphate buffered saline (PBS)</td>
<td>Millipore Sigma</td>
<td>806552-500ML</td>
</tr>
<tr>
<td>Sucrose</td>
<td>MP Biomedicals</td>
<td>194747</td>
</tr>
<tr>
<td>Surgical glue (Loctite 4013)</td>
<td>All-Spec</td>
<td>237041-30769</td>
</tr>
<tr>
<td>Tris-EDTA (TE) buffer, pH 8.0</td>
<td>Invitrogen</td>
<td>AM9849</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Millipore Sigma</td>
<td>T8787</td>
</tr>
<tr>
<td>Trizol reagent (optional)</td>
<td>Thermo Fisher Scientific</td>
<td>15596018; CAS: 108-95-2, 1762-95-4, 593-84-0</td>
</tr>
<tr>
<td>UltraPure DNase/RNase-free distilled water</td>
<td>Invitrogen</td>
<td>10977023</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>LIVE/DEAD Fixable Near-IR Dead Cell Stain Kit</td>
<td>Invitrogen</td>
<td>L10119</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: C57BL/6J-<i>Clec4f</i><sup><i>em1(cre)Glass</i></sup>/J</td>
<td>Glass Lab</td>
<td>JAX stock #033296</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo</td>
<td>Becton, Dickinson &amp; Company</td>
<td><a href="https://www.flowjo.com/">https://www.flowjo.com/</a></td>
</tr>
<tr>
<td>Summit</td>
<td>Beckman Coulter</td>
<td>NA</td>
</tr>
<tr>
<td>Sony Cell Sorter Software</td>
<td>Sony</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>0.2 mL PCR Strip</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>0.22 μm syringe filters</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>1.5 inch 18-gauge needles</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>1.5 mL DNA LoBind tubes</td>
<td>Eppendorf</td>
<td>022431021</td>
</tr>
<tr>
<td>10 mL syringes</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>15 mL conical tubes</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>22-gauge needles</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>24-gauge catheters</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>50 mL conical tubes</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Adjustable flow rate peristalsis pump</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Adjustable temperature swinging bucket centrifuge</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Cell sorter</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Dissection tray</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Falcon 40 μm cell strainer</td>
<td>Corning</td>
<td>352340</td>
</tr>
<tr>
<td>Falcon 100 μm cell strainer</td>
<td>Corning</td>
<td>352360</td>
</tr>
<tr>
<td>Falcon 5 mL round bottom polystyrene test tube, with cell strainer snap cap</td>
<td>Corning</td>
<td>352235</td>
</tr>
<tr>
<td>Heat lamp(s)</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Incubator at 37°C</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Liquid nitrogen or dry ice</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>MACSMix (or similar)</td>
<td>Miltenyi</td>
<td>130-090-753</td>
</tr>
<tr>
<td>Micropipettes and sterile DNase-/RNase-free filter tips</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Nalgene Super Versi-dry surface protectors</td>
<td>Thermo Fisher Scientific</td>
<td>74000-00</td>
</tr>
<tr>
<td>Pasteur pipettes</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Pipette aid</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Primary IV solution set (Example: Baxter Healthcare Model: 2C8401)</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Serological pipettes</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Surgical glue (Loctite4013 or similar)</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Sutures</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td>Water bath at 37°C</td>
<td>NA</td>
<td>NA</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="critical">
<span class="critical-title">Critical:</span> Many chemicals used in this protocol are hazardous and/or toxic. Institutional safety guidelines should be followed for safe use and disposal.</div>
<h3 id="sec2.1">Cell sorter</h3>
<p>This equipment is necessary for <a href="#sec3.1">Module 1: Preparation of hepatic Kupffer cells (or other NPCs)</a> and <a href="#sec3.7">Module 2, alternative strategy: in situ fixation and isolation of nuclei from Kupffer cells (or other liver cells)</a>. Cell sorters can be configured differently between institutes. Prior to beginning experiment, ensure the chosen fluorescent antibodies are suitable for the available equipment. It is recommended to use a cell sorter equipped with an ultraviolet laser to identify hepatic stellate cells using retinoid autofluorescence. Good results for FACS of cells prepared from Module 1 have been obtained using a BD FACSAria and a Beckman Coulter MoFlo Astrios. Good results for FACS of nuclei prepared from Module 2 have been obtained using a Sony SH800 and a Sony MA900.</p>
<h3 id="sec2.2">DNase I (2 mg/mL)</h3>
<p>This enzyme is used for <a href="#sec3.1">Module 1: Preparation of hepatic Kupffer cells (or other NPCs)</a>. Prepare 100 mg lyophilized DNase I at 2 mg/mL in 50 mL ice cold phosphate buffered saline. Do not vortex or vigorously pipette! Store single use aliquots at −80°C. Aliquots of 10.5 mL are adequate for preparations of 8 healthy mice. Single use aliquots can be thawed in a 22°C–25°C water bath, then stored on ice immediately after thawing.</p>
<h3 id="sec2.3">Flavopiridol (10 mM)</h3>
<p>This reagent is used for <a href="#sec3.1">Module 1: Preparation of hepatic Kupffer cells (or other NPCs)</a>. Prepare flavopiridol in dimethyl sulfoxide to 10 mM. Store single use aliquots at −80°C.</p>
<h3 id="sec2.4">Glycine (2.625 M)</h3>
<p>Place 9.85 g glycine in a 50 mL conical and add 45 mL 20°C–25°C UltraPure DNase/RNase-free distilled water. Mix (e.g., rocking, stir-bar) at 20°C–25°C until dissolved and adjust the final volume to 50 mL with 20°C–25°C UltraPure DNase/RNase-free distilled water. Store at 20°C–25°C.</p>
<h3 id="sec2.5">Nuclei isolation buffer</h3>
<p>This buffer is used for <a href="#sec3.7">Module 2, alternative strategy: in situ fixation and isolation of nuclei from Kupffer cells (or other liver cells)</a>. Prepare nuclei isolation buffer at 20°C–25°C using the table below. Store at 4°C and discard after 2–3 weeks.</p>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>UltraPure DNase/RNase-free distilled water</td>
<td> </td>
<td>500 mL</td>
</tr>
<tr>
<td>1 M Tris (pH 8.0)</td>
<td>0.5 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.5 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M magnesium chloride</td>
<td>5 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>Sucrose</td>
<td>0.1 M</td>
<td>17.11 g</td>
</tr>
<tr>
<td>10% Triton X-100</td>
<td>0.05%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>~500 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Triton X-100 is toxic and should be disposed of following approve institutional guidelines.</div>
<h3 id="sec2.6">Nuclei sorting buffer</h3>
<p>This buffer is used for <a href="#sec3.7">Module 2, alternative strategy: in situ fixation and isolation of nuclei from Kupffer cells (or other liver cells)</a>. Prepare 500 mL nuclei sorting buffer by adding 5 mL 0.5 M EDTA to a new bottle of phosphate buffered saline. Store on ice or at 4°C. Discard after 1 week.</p>
<h3 id="sec2.7">Peristalsis pump</h3>
<p>Prepare necessary tubing for perfusion with the peristalsis pump. The tubing must permit transfer of liquids from a reservoir (example: media bottle, conical tubes) to a 22-gauge catheter. Primary IV solution sets are ideal for at the delivery end as they contain a bubble trap and a Luer lock for connecting to the catheter. Determine pump settings delivering 5 and 7 mL of fluid per minute.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Follow institutional guidelines and appropriate safety considerations when perfusing hazardous chemicals! The tubing connections can forcibly disconnect if adequate back pressure builds up in the peristalsis tubing. This event can lead to chemical spillage and potential chemical spraying. It is strongly recommended to use engineering controls (examples: safety hood, custom shielding) and personal protective equipment (lab coat, safety glasses, and disposable gloves). In our experience, this event is uncommon but occasionally occurs at the bubble trap connection points.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to use tubing separately dedicated for each module to eliminate the risk of cross contamination. At the conclusion of module 1 experiments, clean the tubing by sequentially perfusing 500 mL of deionized or distilled water, 500 mL of 70% ethanol, and finally 500 mL of deionized or distilled water. At the conclusion of module 2 experiments, clean the tubing by sequentially perfusing 500 mL of deionized or distilled water, 500 mL of 0.1 M glycine, and finally 500 mL of deionized or distilled water.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A good option is the Cole Palmer Masterflex L/S Precision Variable-Speed Console Drive with a 4 or 8 channel pump head (1 channel per parallel perfused liver). It is more expensive but more adaptable long-term, with less flow-rate variability observed. Alternatively, the Fisherbrand Variable-Flow Peristaltic Pumps (1 pump unit per parallel-perfused liver) is an inexpensive single channel option, but each unit requires frequent calibration of the flow rate. Many additional options are available.</div>
<h3 id="sec2.8">Triton X-100 (10%)</h3>
<p>This reagent is necessary for and <a href="#sec3.7">Module 2, alternative strategy: in situ fixation and isolation of nuclei from Kupffer cells (or other liver cells)</a>. Prepare 40 mL 10% Triton X-100 by adding 10 mL (using a 10 mL syringe) of the detergent to 35 mL UltraPure water in a 50 mL conical. Seal the tube with parafilm and rock overnight at 20°C–25°C. Adjust to a final 40 mL volume using UltraPure water and store at 4°C for up to 1 year.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>This protocol was developed specifically for isolation of hepatic macrophages and downstream use in diverse next-generation sequencing technologies. Conceptually this framework can easily be extended for other liver non-parenchymal cells (NPCs) including sinusoidal endothelial cells, stellate cells, or alternative lineages of immune cells.</p>
<h3 id="sec3.1">Module 1: preparation of hepatic Kupffer cells (or other NPCs)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> &gt;10 h</div>
<p>Liver tissue is enzymatically digested <i>in situ</i> to generate a single-cell suspension enriched for liver non-parenchymal cells that are then immunolabeled for fluorescence-activated cells sorting (FACS). The protocol for preparing non-parenchymal cells was optimized from several other available protocols (<a href="#bib6">Mederacke et al., 2015</a>; <a href="#bib7">Movita et al., 2012</a>; <a href="#bib15">Seki et al., 2009</a>). Excellent protocols with videos detailing the procedure for perfusion-based digestion of a mouse liver exist and can be referred to (<a href="#bib2">Charni-Natan and Goldstein, 2020</a>; <a href="#bib6">Mederacke et al., 2015</a>).</p>
<div class="note">
<span class="note-title">Note:</span> The protocol benefits from two people working in parallel if processing more than four mice. Mouse dissection and digestion is time-consuming and delays at this stage can affect sample quality.</div>
<h3 id="sec3.2">Prepare necessary buffers and reagents</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<p>In this section, necessary buffers for are made ready for non-parenchymal cell preparations.</p>
<ol>
<li>Freshly prepare the following solutions on the same day:
<ol type="a">
<li>1 mM flavopiridol: dilute 10 mM aliquot of flavopiridol stock to 1 mM using water. Do not use phosphate buffered saline as this can cause a precipitate to form. Store the 1 mM solution on ice and discard at the end of the day.
<div class="critical">
<span class="critical-title">Critical:</span> Tissue digestion leads to cell stress responses and the potential for confounding changes in measured analytes. To minimize the effect of tissue digestion on profiled transcriptional regulators, the RNA polymerase inhibitor flavopiridol is added to perfusion and digestion steps where the hepatic cells are at 37°C and active. Use of flavopiridol can affect the viability of purified cells in culture though and it can be excluded or titrated as appropriate for experimental purposes. For comparison of genomics data though, the digestion protocol used should not be altered.</div>
</li>
<li>Wash buffer 1: store and use at 4°C or on ice. The below table is adequate for 8 mice. Discard at the end of the day.
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hank’s Balanced Salt Solution (with calcium, with magnesium, no phenol red)</td>
<td> </td>
<td>500 mL</td>
</tr>
<tr>
<td>DNase I</td>
<td>20 μg/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>2% volume/volume</td>
<td>10 mL</td>
</tr>
<tr>
<td>1 M HEPES</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>10× Hank’s Balanced Salt Solution (no calcium, no magnesium, no phenol red)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>512.5 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Wash buffer 2: store and use at 4°C or on ice. The below table is adequate for 16 mice. Discard within 1 week.
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hank’s Balanced Salt Solution (no calcium, no magnesium, no phenol red)</td>
<td> </td>
<td>500 mL</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>2% volume/volume</td>
<td>10 mL</td>
</tr>
<tr>
<td>1 M HEPES</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>5 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>10× Hank’s Balanced Salt Solution (no calcium, no magnesium, no phenol red)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td><b>512.5 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Isotonic Percoll. Percoll must be made isotonic. Mixing 90% Percoll with 10% 10× Hank’s Balanced Salt Solution (no calcium, no magnesium, no phenol red) renders the solution safe for use with cells. Store the solution on ice and discard at the end of the day.
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Percoll</td>
<td> </td>
<td>9 mL</td>
</tr>
<tr>
<td>10× Hank’s Balanced Salt Solution (no calcium, no magnesium, no phenol red)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Liberase TM (5 mg/mL). Prepare and store on ice. Do not vortex or vigorously pipette! Immediately freeze unused enzyme at −80°C in single use aliquots (1.25 mL per healthy mouse liver).
<div class="critical">
<span class="critical-title">Critical:</span> The enzyme blend Liberase TM is preferable to sequential digestion with Pronase and Collagenase. This preference is due to improved reproducibility in tissue digestion quality and reduced loss of surface antigens used in subsequent FACS purification that are cleaved by Pronase (examples: CD31 and Tim4).</div>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>50 mg lyophilized Liberase TM</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>Phosphate buffered saline</td>
<td> </td>
<td>10 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Liver clearing buffer. Store and use at 37°C. The below table is adequate for 8 mice.
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hank’s Balanced Salt Solution <b>(no calcium, no magnesium, no phenol red)</b>
</td>
<td> </td>
<td>500 mL</td>
</tr>
<tr>
<td>0.5 M EGTA</td>
<td>0.5 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.5 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M HEPES</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>1 mM flavopiridol</td>
<td>1 μM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>10× Hank’s Balanced Salt Solution (no calcium, no magnesium, no phenol red)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>~500 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Liver digestion buffer A. Store and use at 37°C.
<div class="critical">
<span class="critical-title">Critical:</span> Collagenases require calcium to function. Do not use Hank’s Balanced Salt Solution not containing calcium or magnesium.</div>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>Hank’s Balanced Salt Solution (with calcium, with magnesium, no phenol red)</b></td>
<td> </td>
<td>500 mL</td>
</tr>
<tr>
<td>1 M HEPES</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>1 mM flavopiridol</td>
<td>1 μM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>10× Hank’s Balanced Salt Solution (no calcium, no magnesium, no phenol red)</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>~500 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Liver digestion buffer B. Prepare immediately prior to euthanizing a group of mice. Prepare 60 mL per mouse, or 80 mL if the liver is expected to be enlarged or diseased.
<table id="undtbl8">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Liver digestion buffer A</td>
<td> </td>
<td>60 mL</td>
</tr>
<tr>
<td>Liberase TM (5 mg/mL)</td>
<td>0.1 mg/mL</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>DNase I (2 mg/mL)</td>
<td>1 μM</td>
<td>0.625 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>61.88 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>OptiPrep (28%): OptiPrep is a commercially available iso-osmolar solution of iodixanol (60%) and water. With this solution, the final iodixanol concentration is 16.8%. Prepare 10 mL per liver and store at 4°C or on ice. Discard at the end of the day.
<div class="critical">
<span class="critical-title">Critical:</span> Mix the OptiPrep stock solution prior to use. It has a tendency to settle.</div>
<table id="undtbl9">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>OptiPrep</td>
<td>28%</td>
<td>2.8 mL</td>
</tr>
<tr>
<td>Wash buffer 2</td>
<td>72%</td>
<td>7.2 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
</li>
<li>PBS + 0.1% Tween 20: Prepare 50 mL by adding 50 μL of 10% Tween 20 to 50 mL PBS. Store on ice or at 4°C for 1 week.</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Digestion and removal of liver tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 h</div>
<p>In this section, mouse livers are digested <i>in situ</i>, excised, then further digested <i>in vitro</i>. Following digestion, non-parenchymal cells from the crude cell preparation are enriched by successive washing and centrifugation, as well as density gradient centrifugation.</p>
<ol start="2">
<li>Flush blood from the liver of a mouse freshly euthanized following institutional approved guidelines.
<ol type="a">
<li>Pin down the mouse and carefully open the abdominal cavity.</li>
<li>Gently move the intestines to the side exposing the inferior vena cava and the portal vein.</li>
<li>Insert a 24-gauge catheter into the inferior vena (<a href="#fig1">Figure 1</a>A). Attach the line of a peristalsis pump connected to a 0.22-μm filter to the catheter and pin into place. Nick the portal vein with scissors to release fluid pressure. Close the inferior vena cava above the diaphragm using a bulldog clamp. Pin the lines in place at the filter (<a href="#fig1">Figure 1</a>B).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/513-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Catheter placement for fluid perfusion through the mouse inferior vena cava</div>
<p>(A) Example of successful insertion of a catheter into the inferior vena cava. Often the catheter hub will fill with blood. If it does not, it is sometimes helpful to apply a slight vacuum using a 1 mL syringe.</p>
<p>(B) Example of an ongoing successful perfusion. The catheter and peristalsis lines are secured by pinning the filter in place on the dissection mount (Styrofoam board). Note the blanched appearance of the liver in comparison to (A) as a result of blood evacuation through the portal vein.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> All lobes of the liver should blanch (<a href="#fig1">Figure 1</a>B) as erythrocytes drain from the organ. If the liver is not blanching, carefully adjust the position of the catheter in the vessel.</div>
</li>
<li>Apply a small amount of surgical glue to hold the catheter securely in place. Alternatively, the catheter can be secured using a suture.</li>
<li>Turn on the heat lamp to maintain liver temperature at around 37°C, which is ideal for enzymatic digestion.</li>
<li>Perfuse with liver clearing buffer for 3 min at flow rate of 5 mL/min.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Digest liver tissue <i>in situ.</i>
<div class="note">
<span class="note-title">Note:</span> Enlarged fatty livers and fibrotic livers may require a higher flow rate and increased digestion volume (60 mL at 7 mL/min). This is described in the troubleshooting section.</div>
<ol type="a">
<li>Move the pump intake line to a tube with 40 mL of liver digestion buffer B and allow the contents to perfuse through the liver until the line is empty. The liver should begin to lose its shape during digestion, and the Glisson’s capsule may start to detach from the liver cells.</li>
<li>Remove the gallbladder using scissor and tweezers, the gallbladder contains digestive enzymes that can digest surface markers used for FACS.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Continue digestion of liver tissue <i>in vitro.</i>
<ol type="a">
<li>At this point the liver is now often “mush” and can be removed from the abdominal cavity by aspirating into a 1 mL syringe. Alternatively, remove the lobes with scissors and forceps and finely mince the tissue using a razor blade. Transfer the digested contents into a 50 mL conical filled with 20 mL of liver digestion buffer B. Refer to Troubleshooting for more details.</li>
<li>Mix digested liver tissue with digestion buffer by pipetting up and down 3–5 times or by gentle inversion of tubes.</li>
<li>Incubate digested liver tissue for 20 min at 37°C with Miltenyi MACSmix tube rotator at the highest speed.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Remove hepatocytes from cell suspension by low-speed centrifugation followed by Percoll centrifugation.
<div class="note">
<span class="note-title">Note:</span> If working with a fatty liver then perform the centrifugations described in this step at 20°C–25°C.</div>
<div class="note">
<span class="note-title">Note:</span> The Percoll wash (step 4) can be skipped if the liver is healthy and non-steatotic, instead simply centrifuge at 500 × <i>g</i> for 10 min at 4°C to pellet non-parenchymal cells. Furthermore, the Percoll wash depletes stellate cells. If stellate cells are desired, then skip the Percoll wash and simply centrifuge at 500 × <i>g</i> for 10 min to pellet non-parenchymal cells.</div>
<ol type="a">
<li>Filter 20 mL of cells suspended in liver digestion buffer through a 70 μM cell strainer into a 50 mL conical to remove undigested debris and connective tissue.</li>
<li>Add 20 mL of wash buffer 1 to dilute liver digestion buffer and slow enzymatic activity during hepatocyte removal.</li>
<li>Sediment the dense hepatocytes by low-speed centrifugation at 50 × <i>g</i> for 3 min at 4°C.</li>
<li>Transfer the supernatant to a new 50 mL conical and repeat spin at 50 × <i>g</i> for 3 min at 4°C and discard the pellet containing hepatocytes.</li>
<li>Transfer the supernatant enriched for NPCs to a new 50 mL conical.</li>
<li>Add 10 mL isotonic Percoll to 40 mL cell suspension. If the volume of cell suspension is less than 40 mL at this step adjust amount of isotonic Percoll that is added to maintain a 20% final concentration of Percoll.</li>
<li>Centrifuge cell suspension in 20% Percoll at 600 × <i>g</i> for 10 min 4°C.</li>
<li>Remove supernatant and resuspend NPCs in 20 mL of wash buffer 1.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Isolate NPCs using OptiPrep gradient centrifugation (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/513-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Setup of the OptiPrep density gradient</div>
<p>(A) Depiction of how to set up the gradient underlay.</p>
<p>(B) An example of a prepared gradient prior to centrifugation.</p>
<p>(C) Depiction of the resulting gradient layers after centrifugation. Collect the desired NPC layers by pipetting and discard the debris fraction. It can be helpful to assess a small portion of debris layer by flow cytometry if the yields for the desired cell population are consistently lower than expected. This could necessitate optimization of the gradient percentages.</p>
<p>Created with <a href="http://BioRender.com">BioRender.com</a>.</p>
</figcaption>
</figure>
<ol type="a">
<li>Centrifuge the NPC cell suspension at 500 × <i>g</i> for 10 min at 4°C to pellet NPCs once more.</li>
<li>Resuspend pellet in 10 mL of OptiPrep (28%) using a 10 mL syringe with an 18-gauge needle attached (<a href="#fig2">Figure 2</a>). Take care to resuspend the cell pellet gently at this step.</li>
<li>Fill a 15 mL conical with 3 mL of wash buffer 1 and rest a glass Pasteur pipette at the bottom of the tube. Transfer the 10 mL cell suspension in OptiPrep buffer through the Pasteur pipette to create the cell layer underneath the less dense wash buffer 1 layer (<a href="#fig2">Figures 2</a>A and 2B). The narrow opening of the Pasteur pipette controls the flow rate and greatly simplifies the layering process.</li>
<li>Centrifuge gradient suspension at 1,400 × <i>g</i> for 25 min at 4°C with centrifuge acceleration set to very low and with brake mechanism turned off.</li>
<li>Following gradient centrifugation, non-parenchymal cells will be enriched at the interface between the upper 0% OptiPrep layer and the lower 28% OptiPrep layer (<a href="#fig2">Figure 2</a>C). Use a 1 mL pipette tip to extract all cells at this interface and transfer to a new 15 mL conical tube, then fill the tube to 12 mL with wash buffer 2.</li>
</ol>
</li>
<li>Lyse red blood cells remaining in non-parenchymal cell mixture.
<ol type="a">
<li>Centrifuge cell suspension at 500 × <i>g</i> for 10 min at 4°C.</li>
<li>Remove supernatant and resuspend in 1 mL of 4°C RBC lysis buffer, incubate for 5 min on ice to lyse red blood cells.</li>
<li>Dilute the RBC lysis buffer cell suspension with 9 mL of 4°C phosphate buffered saline and pellet NPCs via centrifugation at 400 × <i>g</i> for 7 min at 4°C.</li>
<li>Remove the supernatant and wash the cells with 10 mL of 4°C phosphate buffered saline.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Staining of non-parenchymal cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5 h</div>
<p>In this section, the enriched non-parenchymal cells are labeled with fluorescent antibodies in preparation for FACS.</p>
<ol start="8">
<li>Stain non-parenchymal cells with viability dye. Set aside some unstained cells which can be helpful for determining background fluorescence.
<ol type="a">
<li>Centrifuge cell suspension at 500 × <i>g</i> for 10 min at 4°C.</li>
<li>Resuspend cell pellet in 250 to 500 μL of 4°C PBS containing anti-CD16/CD32 to block FC and Live/Dead NIR Viability Dye or similar to label dead cells.
<div class="note">
<span class="note-title">Note:</span> A 250 μL stain volume per healthy liver is often appropriate for steps 8b and 9. Enlarged fatty livers may require a 500 μL stain volume.</div>
<table id="undtbl10">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Phosphate buffered saline</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>anti-CD16/CD32 (0.5 mg/mL)</td>
<td>1 μg/mL</td>
<td>1:500</td>
</tr>
<tr>
<td>Live/Dead NIR</td>
<td>1 μM</td>
<td>1:300</td>
</tr>
</tbody>
</table>
</li>
<li>Stain at 4°C for 10 min.</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Freshly prepare 250 to 500 μL of the antibody staining cocktail per liver.</li>
</ol>
<table id="undtbl11">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Wash buffer 2</td>
<td> </td>
<td> </td>
</tr>
<tr>
<td>Anti-CD16/CD32 (0.5 mg/mL)</td>
<td>1 μg/mL</td>
<td>1:500</td>
</tr>
<tr>
<td>Anti-F4/80 BV421 (0.2 mg/mL)</td>
<td>2 μg/mL</td>
<td>1:100</td>
</tr>
<tr>
<td>Anti-CD45 BB515 (0.2 mg/mL)</td>
<td>2 μg/mL</td>
<td>1:100</td>
</tr>
<tr>
<td>Anti-CD11b PE (0.2 mg/mL)</td>
<td>2 μg/mL</td>
<td>1:100</td>
</tr>
<tr>
<td>Anti-CD146 PE/Cy7 (0.2 mg/mL)</td>
<td>2 μg/mL</td>
<td>1:100</td>
</tr>
<tr>
<td>Anti-Tim4 Alexa 647 BV421 (0.5 mg/mL)</td>
<td>2 μg/mL</td>
<td>1:250</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Many excellent alternative antibody cocktails may be used. We refer the readers to the literature for examples (<a href="#bib4">Devisscher et al., 2017</a>; <a href="#bib10">Ramachandran et al., 2012</a>; <a href="#bib11">Remmerie et al., 2020</a>; <a href="#bib14">Seidman et al., 2020</a>).</div>
<ol start="10">
<li>Add an equivalent volume used in step 8b of the 2× antibody staining cocktail to the cells. Mix together by inversion 20 times or using gentle flicking.</li>
<li>Stain at 4°C for 20 min.</li>
<li>Fill the tube with wash buffer 2 at the end of the staining process.</li>
<li>Pellet the NPCs by centrifugation at 500 × <i>g</i> for 10 min at 4°C, remove the supernatant, then resuspend pellet in 2 mL wash buffer 2.</li>
<li>Filter the cells through a 35- or 40-μm cell strainer to remove particulate matter that may clog the cell sorter. Flush the strainer with wash buffer 2, then fill the tube with wash buffer 2.</li>
<li>Pellet the non-parenchymal cell suspension at 500 × <i>g</i> for 10 min at 4°C.</li>
<li>Remove supernatant and resuspend in 500 μL wash buffer 2. Keep the cells on ice and in the dark (covered). Cells are now ready for FACS.</li>
</ol>
<h3 id="sec3.5">Sorting of non-parenchymal cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 h to sort through an entire preparation of one liver</div>
<p>In this section, the stained single-cell suspension is sorted into purified cell populations using FACS.</p>
<ol start="17">
<li>Prepare single-color controls using compensation beads. We use OneComp eBeads Compensation Beads sourced from Invitrogen. Follow the manufacturer’s protocol. Typically, using 1–2 μg of antibody per compensation control provides necessary fluorescent signal higher than the signal of stained cells. Of note, we have found the popular UltraComp eBeads Compensation Beads are not easily detected by this lab’s MoFlo Astrios or Sony MA900. Yet, these beads are an excellent option for use on BD systems.</li>
</ol>
<ol start="18">
<li>Set a gating strategy for Kupffer cells (<a href="#fig3">Figure 3</a>). We have found success with the following strategy:
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/513-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Gating strategy for purification of mouse hepatic macrophages and sinusoidal endothelial cells</div>
<p>Representative flow cytometry gating to isolate Kupffer cells. The cells displayed here are from a mouse fed a NASH-inducing diet and display two population of Kupffer cells that can be segregated based on expression of Tim4. High expression of Tim4 is a useful surrogate for determine cells of long-term residence or embryonic origin (<a href="#bib4">Devisscher et al., 2017</a>; <a href="#bib12">Sakai et al., 2019</a>; <a href="#bib13">Scott et al., 2016</a>; <a href="#bib14">Seidman et al., 2020</a>).</p>
</figcaption>
</figure>
<ol type="a">
<li>Generate a dot plot of CD45 versus CD146 and set gates for CD146<sup>−</sup>CD45<sup>+</sup> immune cells and CD146<sup>+</sup>CD45<sup>−</sup> endothelial cells.</li>
<li>Generate a dot plot CD11b versus F4/80 for the CD146<sup>−</sup>CD45<sup>+</sup> population identified in step 20a. Set gate for CD11b<sup>lo</sup>F4/80<sup>hi</sup> tissue macrophages.</li>
<li>Generate a dot plot of side scatter pulse height versus side scatter pulse area to identify single cells and exclude doublets. Doublet particles will have an increased ratio of area signal relative to height signal. It is also acceptable to identify singlets in an analogous manner using forward scatter.</li>
<li>Generate a dot plot of forward scatter versus Zombie NIR to identify live cells and exclude dead cells. Live cells will be Zombie NIR negative.
<div class="note">
<span class="note-title">Note:</span> It is helpful to use a fluorescence minus one control for Zombie NIR to define your viable cell gate.</div>
</li>
<li>Generate a dot plot of Tim4-A647. Tim4<sup>+</sup> cells are long-term resident or embryonic-derived Kupffer cells while Tim4<sup>−</sup> cells are monocyte-derived macrophages recruited during disease to the Kupffer cell niche.</li>
</ol>
</li>
</ol>
<ol start="19">
<li>Optional: Set gating strategy for other hepatic macrophages which may arise during the course of a chosen disease model.
<ol type="a">
<li>Select the CD11b<sup>hi</sup>F4/80<sup>lo</sup> population displayed in step 11b.</li>
<li>Select CX3CR1<sup>+</sup>Ly6G<sup>−</sup> cells to exclude neutrophils.</li>
<li>These cells can be further divided based on expression of Ly6C into Ly6C<sup>lo</sup> recruited macrophages and Ly6C<sup>hi</sup> recruited macrophages.</li>
<li>Identify singlets and live cells as in step 18c and 18d. Further clarification on identification of these cells is described in <a href="#bib10">Ramachandran et al., 2012</a> and <a href="#bib14">Seidman et al., 2020</a>.</li>
</ol>
</li>
<li>Optional: Set gating strategy for liver sinusoidal endothelial cells.
<ol type="a">
<li>Identify liver sinusoidal endothelial cells by gating on CD146<sup>+</sup>CD45<sup>−</sup> cells described in step 18a and shown in <a href="#fig3">Figure 3</a> at left.</li>
<li>Identify singlets and live cells as in step 18c and 18d. Further clarification on identification of these cells is described in <a href="#bib12">Sakai et al., 2019</a>.</li>
</ol>
</li>
<li>Optional: Set gating strategy for stellate cells.
<ol type="a">
<li>Generate a plot of retinoid autofluorescence versus side scatter to identify autofluorescent stellate cells.</li>
<li>Identify singlets and live cells as in step 18c and 18d. Further clarification on identification of these cells is described in <a href="#bib5">Kisseleva et al., 2012</a> and <a href="#bib6">Mederacke et al., 2015</a>.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our experience detection of retinoid autofluorescence is best when excited by an ultraviolet (355 nm) laser and detected at approximately 500 nm.</div>
<ol start="22">
<li>Purity of sorted cells can be assessed by flow cytometry of a small number of sorted cells diluted in wash buffer 2.</li>
</ol>
<h3 id="sec3.6">Storage of sorted non-parenchymal cells</h3>
<p>FACS-purified cell populations are now suitable for many next-generation sequencing applications. We provide example storage strategies for RNA-seq, ATAC-seq, and ChIP-seq below. Our labs have also successfully used cells prepared in this manner for scRNA-seq and Hi-C as examples of other suitable applications.</p>
<div class="optional">
<span class="optional-title">Optional:</span> To prepare cells for RNA-seq, we recommend using 50,000 - 100,000 cells for common RNA-seq library preparation protocols. Cells can be processed as recommended using a supplied lysis buffer found in many commercial RNA-seq kits or alternatively with Trizol.</div>
<div class="optional">
<span class="optional-title">Optional:</span> To prepare cells for ATAC-seq, we recommend using 50,000 cells (<a href="#bib1">Buenrostro et al., 2015</a>; <a href="#bib3">Corces et al., 2017</a>). Of note, sample preparation and tagmentation reaction conditions may require optimization for each cell type to yield the best quality data.</div>
<p>Prepare sorted viable cells for ChIP-seq. We recommend starting with 0.5 to 1 million cells for ChIP-seq library preparation. For Kupffer cells, this can often be attained with 1–3 livers. For sinusoidal endothelial cells, this can be often be attained with 1 liver. Other populations of non-parenchymal cells would require testing to determine necessary sample size. This approach works well for assessment of histone modifications and genomic PU.1 occupancy. Other ChIP targets may require optimization.</p>
<ol start="23">
<li>Wash the desired number of cells with wash buffer 2 and centrifuge at 500 × <i>g</i> for 5 min at 4°C.</li>
<li>Remove the supernatant and resuspend pellet in 500 μL wash buffer 2 and transfer to 1.5 mL DNA LoBind tube. Wash the starting tube with an additional 500 μL wash buffer 2 and combine into the 1.5 mL DNA LoBind tube.</li>
<li>Centrifuge the cells in the 1.5 mL DNA LoBind tube at 500 × <i>g</i> for 5 min at 4°C.</li>
<li>Resuspend the pellet in 20°C–25°C 1% formaldehyde using approximately 1 million cells per mL. Incubate for 10 min 20°C–25°C with gentle rocking or rotation to cross link proteins and DNA.</li>
<li>Quench the cross-linking reaction by adding 20°C–25°C glycine to a final 0.125 M concentration from the 2.625 M glycine stock solution. Additionally, add Tween 20 to a final 0.1% concentration from a 10% Tween 20 stock. Tween 20 aids in pelleting the cross-linked cells during centrifugation.</li>
<li>Centrifuge cross-linked cells at 800 × <i>g</i> for 10 min at 4°C. Carefully confirm the presence of a cell pellet. If a pellet is not apparent, centrifuge again at 1,200 × <i>g</i> for 10 min at 4°C. Aspirate the supernatant taking care to avoid disturbing the cell pellet.</li>
<li>Suspend the cross-linked cells in 1 mL PBS + 0.1% Tween 20 at 4°C and centrifuge at 800–1,200 × <i>g</i> for 10 min at 4°C, as in step 28. Aspirate the supernatant taking care to avoid disturbing the cell pellet.</li>
<li>Wash the cells once more by repeating step 29.</li>
<li>Quickly centrifuge the sample for a few seconds at 800–1,200 × <i>g</i> collect the residual fluid to the tube bottom. Aspirate as much residual supernatant as possible using a p10 pipette tip. Be very careful to avoid disturbing the cell pellet. Left over liquid in the sample at this step will crystallize at −80°C and can interfere with ChIP-seq library quality.</li>
<li>Snap-freeze the pellet using liquid nitrogen and store the cross-linked cells at −80°C until ready for ChIP-seq library preparation described by Texari et al. (link to “An optimized protocol for rapid, sensitive, and robust on-bead ChIP-seq”). Alternatively, the samples can be snap-frozen using an alcohol and dry ice bath. Take care to protect the tube labeling!</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It can be beneficial for challenging ChIPs to double crosslink the sample at 20°C–25°C with gentle rocking for 30 min using 1 mL of a 2 mM solution of disuccinimidyl glutarate (DSG) per million cells. Then add formaldehyde to a final concentration of 1% and incubate for an additional 10 min of 20°C–25°C incubation. More details on this approach can be found in the linked protocol described by Texari et al. (link to “An optimized protocol for sensitive and robust on-bead ChIP-seq”). However, in our experience much higher quality ChIP-seq data for more challenging ChIP factors can be obtained through use of nuclei fixed <i>in situ</i> prior to fluorescence-activated nuclei sorting. This approach is described in the next section.</div>
<h3 id="sec3.7">Module 2, alternative strategy: <i>in situ</i> fixation and isolation of nuclei from Kupffer cells (or other liver cells)</h3>
<p>This section describes an approach for preparing nuclei from fixed livers for sorting that was optimized from other protocols (<a href="#bib9">Nott et al., 2019</a>). We have found that using around 1 million Kupffer cell nuclei prepared in this manner yield higher quality ChIP-seq data than material from Module 1. This is particularly true for transcription factors that are challenging to ChIP with limited material. Examples of the data quality can be found in <a href="#bib12">Sakai et al., 2019</a> and <a href="#bib14">Seidman et al., 2020</a>.</p>
<h3 id="sec3.8">Prepare buffers and reagents necessary for module 2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>In this section, the buffers needed for <i>in situ</i> fixation are prepared.</p>
<ol start="33">
<li>Freshly prepare 25 mL liver clearing buffer 2 per mouse liver and store at 20°C–25°C.</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td> </td>
<td>24.95 mL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.5 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>0.5 M EGTA</td>
<td>0.5 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<ol start="34">
<li>Freshly prepare 180 mL 1 mg/mL DSG per mouse liver.
<ol type="a">
<li>Dissolve 100 mg vials of DSG using 4 mL of DMSO.
<div class="critical">
<span class="critical-title">Critical:</span> DSG is toxic. Wear appropriate personal protective equipment (lab coat, safety glasses, and disposable gloves), refer to the safety data sheet and follow institutional guidelines for safe handling and disposal!</div>
</li>
<li>Dilute the DSG/DMSO solution to 1 mg/mL by rapidly dispensing it into 176 mL 20°C–25°C PBS followed by immediate mixing by inverting 5–6 times and keep at 20°C–25°C until use.
<div class="note">
<span class="note-title">Note:</span> Occasionally the solution will form an immediate precipitate. In this event discard the solution and prepare fresh.</div>
</li>
</ol>
</li>
</ol>
<ol start="35">
<li>Freshly prepare 60–65 mL of 1% formaldehyde per mouse liver.
<ol type="a">
<li>Dilute stock formaldehyde solution to 1% volume/volume using 20°C–25°C PBS. Keep at 20°C–25°C until use.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Formaldehyde is toxic. Wear appropriate personal protective equipment (lab coat, safety glasses, and disposable gloves), refer to the safety data sheet and follow institutional guidelines for safe handling and disposal!</div>
<ol start="36">
<li>Freshly prepare 20 mL 0.25 M glycine per mouse liver by diluting 1.9 mL 2.625 M stock glycine into 18.1 mL 20°C–25°C PBS. Store at 20°C–25°C.</li>
<li>Chill and use nuclei isolation buffer and nuclei sorting buffer on ice.</li>
</ol>
<h3 id="sec3.9">
<i>In situ</i> fixation and nuclei preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>In this section, mouse livers are quickly fixed by <i>in situ</i> perfusion. Then nuclei are liberated and prepared for FACS.</p>
<div class="critical">
<span class="critical-title">Critical:</span> It is strongly recommended to use engineering controls (examples: safety hood, custom shielding) and personal protective equipment (lab coat, safety glasses, and disposable gloves) when perfusing hazardous chemicals!</div>
<div class="critical">
<span class="critical-title">Critical:</span> Safely collect and dispose of the perfusion waste as a chemical hazard. Waste can be pooled in a sufficiently deep tray for later disposal. Alternatively, it can be captured in multiple layers of super absorbent bench liners.</div>
<ol start="38">
<li>Flush blood from the liver of a mouse freshly euthanized following institutional approved guidelines.
<ol type="a">
<li>Pin down the mouse and carefully open the abdominal cavity.</li>
<li>Gently move the intestines to the side exposing the inferior vena cava and the portal vein.</li>
<li>Carefully insert a 24-gauge catheter into the inferior vena cava. Attach the line of a peristalsis pump connected to a 0.22-μm filter to the catheter and pin into place. Nick the portal vein with scissors to release fluid pressure. Close the inferior vena cava above the diaphragm using a bulldog clamp.</li>
<li>Insert a 24-gauge catheter into the inferior vena (<a href="#fig1">Figure 1</a>A). Attach the line of a peristalsis pump connected to a 0.22-μm filter to the catheter and pin into place. Nick the portal vein with scissors to release fluid pressure. Close the inferior vena cava above the diaphragm using a bulldog clamp (<a href="#fig1">Figure 1</a>B).
<div class="note">
<span class="note-title">Note:</span> All lobes of the liver should blanch as erythrocytes drain from the organ. If the liver is not blanching, carefully adjust the position of the catheter in the vessel.</div>
</li>
<li>Apply a small amount of surgical glue to hold the catheter securely in place. Alternatively, the catheter can be secured using a suture.</li>
<li>Perfuse 20°C–25°C Liver clearing buffer 2 at 5 mL/min for 3 min.</li>
</ol>
</li>
</ol>
<ol start="39">
<li>
<i>In situ</i> double-fixation of the liver.
<ol type="a">
<li>Switch the perfusion line to a solution of 1 mg/mL DSG prepared in 20°C–25°C phosphate buffered saline. Perfuse for 30 min, which requires approximately 180 mL/mouse.
<div class="note">
<span class="note-title">Note:</span> Some ChIP targets, (for example PU.1, H3K27ac, and RNA Polymerase II), do not require double cross-linking with DSG and step 39a should be excluded. This determination should be made empirically. Further, optimal fragmentation conditions for single and double-cross-linked samples will not be the same.</div>
</li>
<li>Switch the perfusion line to a solution containing 1% formaldehyde prepared in 20°C–25°C phosphate buffered saline. Perfuse for 10 min, which requires approximately 60 mL per mouse.</li>
</ol>
</li>
</ol>
<ol start="40">
<li>Quenching fixation.
<ol type="a">
<li>Switch the perfusion line to a vessel containing 20 mL of 20°C–25°C 0.25 M glycine and allow the perfusion line to fully empty.</li>
</ol>
</li>
<li>Homogenize the liver.
<ol type="a">
<li>Remove the fixed liver and dispose of the hazardous mouse carcass following institutional guidelines.</li>
<li>Finely mince the liver with a razor in a 5 cm plastic dish, then transfer the minced tissue to a 50 mL conical.</li>
<li>Wash the liver twice by suspending in 20 mL nuclei isolation buffer and centrifuging for 1,200 × <i>g</i> for 7 min at 4°C. Discard the supernatant after each wash.</li>
<li>Suspend the washed liver tissue in 15 mL nuclei isolation buffer then transfer to a 15 mL Dounce homogenizer.</li>
<li>Using the loose pestle, homogenize slowly 10 times with ¼ to ½ rotation on the downward stroke and no rotation on the upward stroke.</li>
<li>Incubate on ice for 30 min.</li>
<li>Using the tight pestle, homogenize slowly 10 times with ¼ to ½ rotation on the downward stroke and no rotation on the upward stroke.</li>
<li>Transfer the homogenate to a 15 mL conical tube and centrifuge at 1,200 × <i>g</i> for 7 min at 4°C.</li>
<li>Repeat this step by resuspending the pellet in 10 mL fresh nuclei isolation buffer each time for a total of 2–4 times. The goal by this step is to have sufficiently cleaned the pellet of debris yielding a clear supernatant. Removing this debris is beneficial to reduce the frequency of clogging the cell sorter and the overall time spent sorting. A cleaner starting sample will also result in increased sort purity and yield.</li>
<li>Examine nuclei under a microscope using trypan blue and a hemocytometer. If the cells are intact, consider repeating steps 41g–i. Then suspend at ∼20 million nuclei per mL.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> At this point, nuclei can be suspended in 10 mL 90% FBS and 10% DMSO (10 mL per liver) and aliquots stored at −80°C for downstream processing at a later time. We have not tested other nuclei preservation methods. To recover the cryopreserved sample, warm the frozen suspension at 37°C in a water bath, then wash the nuclei twice with nuclei isolation buffer.</div>
<div class="note">
<span class="note-title">Note:</span> At this time nuclei can be stained for nuclear antigens specific for the population of interest. For the purpose of this protocol, we use fluorescence signal provided genetically via nuclear-localized tdTomato of the Clec4f-Cre-Reporter (<a href="#bib12">Sakai et al., 2019</a>) which is available at Jackson labs under Stock No: 033296. As shown in <a href="#fig4">Figure 4</a>, Kupffer cell nuclei can also be enriched at the steady state by staining the nuclei with anti-PU.1. We describe this procedure in the optional step below. This concept can be extended to labeling of other populations with lineage-specific antibodies.</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/513-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Gating strategy for purification of mouse hepatic Kupffer cell nuclei</div>
<p>Representative flow cytometry gating for isolation of Kupffer cell nuclei. The use of a Clec4f nuclear reporter is necessary to ensure separation of Kupffer cell nuclei from nuclei of other PU1+ recruited macrophages that can arise during disease. At the steady state, there are very few recruited macrophages in the liver isolation of Kupffer cell nuclei using PU1 staining alone is adequate.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Liver disease can promote population expansion of many cell lineages, especially myeloid cells. In these instances, purification of nuclei by labeling with a single antibody may not result in a consistent nuclei population. During disease, we suggest considering genetically encoded drivers of nuclear-localized reporters.</div>
<ol start="42">
<li>Optional: Stain nuclei population(s) of interest using antibodies for lineage-specific transcription factors. We have found success labeling hepatic macrophage nuclei using anti-PU.1. This process is described below as a template for user specific applications. Save a small portion of the sample for use as an unstained control.
<ol type="a">
<li>Add 1 μg/mL anti-CD16/CD32 to prevent nonspecific binding of antibodies to FC receptors and incubate at 4°C for 20 min.</li>
<li>Add 1 μg/mL Alexa Fluor 647 conjugated anti-PU.1 and stain at 4°C for 30 min or overnight as optimized empirically.</li>
<li>Fill the tube with nuclei isolation buffer and centrifuge at 1,200 × <i>g</i> for 7 min at 4°C.</li>
<li>Aspirate the supernatant and refill the tube with nuclei isolation buffer, then repeat the previous centrifugation.</li>
<li>Aspirate the supernatant.</li>
</ol>
</li>
</ol>
<h3 id="sec3.10">Nuclei sorting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h per liver, depending on sample purity and liver size</div>
<p>In this section, the prepared nuclei are filtered and the desired populations are purified using FACS.</p>
<ol start="43">
<li>Prepare the isolated nuclei for sorting by suspending the washed nuclei in nuclei sorting buffer, then slowly pass the suspension through a 22-gauge needle and a 40-μm cell strainer. Suspending nuclei at 10–20 million/mL is a good starting point. Store on ice until sorting.</li>
<li>Purify desired nuclei population using fluorescence-activated nuclei sorting.
<ol type="a">
<li>During sorter setup, pay careful attention to instrument detector thresholds and voltages/gains for light scatter.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> While sorting carefully monitor droplet profile and event rate stability.</div>
<ol start="45">
<li>After purification, verify yield by microscopy and purity by flow cytometry.</li>
<li>Centrifuge nuclei at 800 to 1,200 × <i>g</i> for 10 min at 4°C in PBS + 0.01% Tween 20</li>
<li>Carefully aspirate all buffer then quick spin the nuclei once more at 800 to 1,200 × <i>g</i>.</li>
<li>Carefully remove all buffer without disturbing the nuclei pellet.</li>
<li>Snap-freeze the pellet using liquid nitrogen and store at −80°C until use for ChIP.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> At this point, purified nuclei can be stored long-term. We have generated high-quality results using fixed nuclei stored for more than a year at −80°C.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>At the end of Module 1, yields of 0.3 to 0.75 million Kupffer cells, a million or more sinusoidal endothelial cells, and 50,000 to 250,000 stellate cells. Yields of other recruited macrophage populations are usually less than 50,000 in mice with disease and are not present in healthy mice. Variables contributing to cell yields include genetic background, the presence of disease, digestion quality, and cell sorter abort rates. Yields of 1–2 million Kupffer cell nuclei are typical of Module 2.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Acquiring sufficient material for downstream assays can require extensive time investment. Further, yields can be variable, especially from livers with fatty liver disease or fibrosis. The procedure described in module 1 uses a Percoll centrifugation as a way to eliminate debris and fatty hepatocytes. This can also lead to loss of stellate cells, especially in fatty livers. If the primary experimental goal is to yield hepatic stellate cells, consider instead using the excellent protocol described elsewhere (<a href="#bib6">Mederacke et al., 2015</a>). Further, autofluorescence of liver NPC preparations can make robust population discrimination challenging during FACS.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Liver tissue is poorly digested following <i>in situ</i> digestion.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Fibrotic, fatty, or hypertrophied liver tissue may not enzymatically digest as well as healthy liver tissue. If this is expected, we have found success by increasing the perfusion rate to 7 mL/min and perfusing with 60 mL of liver digestion buffer. Further, perfusion quality will impact liver digestion efficiency. Thorough descriptions of strategies to improve liver digestions have already been described elsewhere. (<a href="#bib2">Charni-Natan and Goldstein, 2020</a>; <a href="#bib6">Mederacke et al., 2015</a>).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Cell populations are difficult to discriminate in setup for FACS.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Residual hepatocytes and stellate cells have significant autofluorescence that interferes with detection of true signals of interest. It is helpful during setup of the cell sorter to crudely identify your population(s) of interest using fluorescent signals then determine their SSC profiles. This information can then be used to apply a first gate on SSC versus autofluorescence of retinoids to exclude the undesired residual hepatocytes and stellate cells from other downstream gates. Additionally, Kupffer cells also have significant autofluorescence and many antigen:fluorophor combinations do have adequate signal to noise profiles. It is valuable to create and carefully assess fluorescence minus one controls, especially when testing new antibodies. We have often found success with Kupffer cells using the following fluorophores: BUV395, BUV737, BV421, BV786, Alexa Fluor 488 or BB515, PE or TdTomato, PE-Cy7 or PE-eFluor-770, Alexa Fluor 647 or APC, and Live/dead NIR. We have found use of detector channels for PE-Dazzle, BV510, BV650, PerCP-Cy5.5, and Alexa Fluor 700 often have inadequate signal to noise ratio for reliable protein detection on mouse Kupffer cells without very careful attention to proper controls and antibody titrations.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Detection and/or yields for nuclei sorts are lower than expected.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Detecting nuclei of some cell populations using flow cytometry can be trickier than intact cells due to the smaller particle size. It can be helpful to co-stain the nuclei with titrated DAPI. Nuclei will be very brightly labeled with DAPI and the resulting signal can be used to assist setup of detector voltages and gains for proper detection of the desired nuclei population. It is also important to check the trigger threshold setting to ensure the desired population is not simply being excluded as noise by the controlling software.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Buenrostro, J.D., Wu, B., Chang, H.Y., and Greenleaf, W.J. (2015). ATAC-seq: a method for assaying chromatin accessibility genome-wide. Curr. Protoc. Mol. Biol. <i>109</i>, 21.29.1-21.29.9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref1">View at publisher</a></p>
<p id="bib2">Charni-Natan, M. and Goldstein, I. (2020). Protocol for primary mouse hepatocyte isolation. STAR Protoc. <i>1</i>, 100086. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref2">View at publisher</a></p>
<p id="bib3">Corces, M.R., Trevino, A.E., Hamilton, E.G., Greenside, P.G., Sinnott-Armstrong, N.A., Vesuna, S., Satpathy, A.T., Rubin, A.J., Montine, K.S., Wu, B., et al. (2017). An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. Nat. Methods <i>14</i>, 959-962. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref3">View at publisher</a></p>
<p id="bib4">Devisscher, L., Scott, C.L., Lefere, S., Raevens, S., Bogaerts, E., Paridaens, A., Verhelst, X., Geerts, A., Guilliams, M., and Van Vlierberghe, H. (2017). Non-alcoholic steatohepatitis induces transient changes within the liver macrophage pool. Cell. Immunol. <i>322</i>, 74-83. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref4">View at publisher</a></p>
<p id="bib5">Kisseleva, T., Cong, M., Paik, Y., Scholten, D., Jiang, C., Benner, C., Iwaisako, K., Moore-Morris, T., Scott, B., Tsukamoto, H., et al. (2012). Myofibroblasts revert to an inactive phenotype during regression of liver fibrosis. Proc. Natl. Acad. Sci. U S A <i>109</i>, 9448-9453. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref5">View at publisher</a></p>
<p id="bib6">Mederacke, I., Dapito, D.H., Affò, S., Uchinami, H., and Schwabe, R.F. (2015). High-yield and high-purity isolation of hepatic stellate cells from normal and fibrotic mouse livers. Nat. Protoc. <i>10</i>, 305-315. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref6">View at publisher</a></p>
<p id="bib7">Movita, D., Kreefft, K., Biesta, P., Oudenaren, A. van, Leenen, P.J.M., Janssen, H.L.A., and Boonstra, A. (2012). Kupffer cells express a unique combination of phenotypic and functional characteristics compared with splenic and peritoneal macrophages. J. Leukoc. Biol. <i>92</i>, 723-733. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref7">View at publisher</a></p>
<p id="bib8">Muse, E.D., Yu, S., Edillor, C.R., Tao, J., Spann, N.J., Troutman, T.D., Seidman, J.S., Henke, A., Roland, J.T., Ozeki, K.A., et al. (2018). Cell-specific discrimination of desmosterol and desmosterol mimetics confers selective regulation of LXR and SREBP in macrophages. Proc. Natl. Acad. Sci. U S A <i>115</i>, E4680-E4689. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref8">View at publisher</a></p>
<p id="bib9">Nott, A., Holtman, I.R., Coufal, N.G., Schlachetzki, J.C.M., Yu, M., Hu, R., Han, C.Z., Pena, M., Xiao, J., Wu, Y., et al. (2019). Brain cell type-specific enhancer-promoter interactome maps and disease-risk association. Science <i>366</i>, 1134-1139. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref9">View at publisher</a></p>
<p id="bib10">Ramachandran, P., Pellicoro, A., Vernon, M.A., Boulter, L., Aucott, R.L., Ali, A., Hartland, S.N., Snowdon, V.K., Cappon, A., Gordon-Walker, T.T., et al. (2012). Differential Ly-6C expression identifies the recruited macrophage phenotype, which orchestrates the regression of murine liver fibrosis. Proc. Natl. Acad. Sci. U S A <i>109</i>, E3186-E3195. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref10">View at publisher</a></p>
<p id="bib11">Remmerie, A., Martens, L., Thoné, T., Castoldi, A., Seurinck, R., Pavie, B., Roels, J., Vanneste, B., De Prijck, S., Vanhockerhout, M., et al. (2020). Osteopontin expression identifies a subset of recruited macrophages distinct from Kupffer cells in the fatty liver. Immunity <i>53</i>, 641-657. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref11">View at publisher</a></p>
<p id="bib12">Sakai, M., Troutman, T.D., Seidman, J.S., Ouyang, Z., Spann, N.J., Abe, Y., Ego, K.M., Bruni, C.M., Deng, Z., Schlachetzki, J.C.M., et al. (2019). Liver-derived signals sequentially reprogram myeloid enhancers to initiate and maintain Kupffer cell identity. Immunity <i>51</i>, 655-670.e8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref12">View at publisher</a></p>
<p id="bib13">Scott, C.L., Zheng, F., De Baetselier, P., Martens, L., Saeys, Y., De Prijck, S., Lippens, S., Abels, C., Schoonooghe, S., Raes, G., et al. (2016). Bone marrow-derived monocytes give rise to self-renewing and fully differentiated Kupffer cells. Nat. Commun. <i>7</i>, 10321. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref13">View at publisher</a></p>
<p id="bib14">Seidman, J.S., Troutman, T.D., Sakai, M., Gola, A., Spann, N.J., Bennett, H., Bruni, C.M., Ouyang, Z., Li, R.Z., Sun, X., et al. (2020). Niche-specific reprogramming of epigenetic landscapes drives myeloid cell diversity in nonalcoholic steatohepatitis. Immunity <i>52</i>, 1057-1074.e7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref14">View at publisher</a></p>
<p id="bib15">Seki, E., de Minicis, S., Inokuchi, S., Taura, K., Miyai, K., van Rooijen, N., Schwabe, R.F., and Brenner, D.A. (2009). CCR2 promotes hepatic fibrosis in mice. Hepatology <i>50</i>, 185-197. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00070-8/sref15">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Christopher K. Glass (<a href="mailto:ckg@health.ucsd.edu">ckg@health.ucsd.edu</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new materials.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate new data or code.</p>
<h3>Acknowledgments</h3>
<p>These studies were supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grants DK091183, HL088083, DK063491, and GM085764 and <a href="https://doi.org/10.13039/501100001674">Fondation Leducq</a> grant 16CVD01. T.D.T. was supported by P30 DK063491, T32DK007044, and NRSA T32CA009523. H.B. was supported by the <a href="https://doi.org/10.13039/100000002">NIH</a> grants T32DK007202 and F30DK124980. M.S. was supported by the <a href="https://doi.org/10.13039/501100007664">Manpei Suzuki Diabetes Foundation</a> of Tokyo, Japan, and the Osamu Hayaishi Memorial Scholarship for Study Abroad, Japan. J.S.S. was supported by a <a href="https://doi.org/10.13039/100000968">American Heart Association</a> Fellowship 16PRE30980030 and <a href="https://doi.org/10.13039/100000002">NIH</a> Predoctoral Training grant T32DK007541. S.H. received additional support from NIH grants R01GM129523, P30DK063491 and P30DK120515.</p>
<h3>Author Contributions</h3>
<p>T.D.T., protocol development and optimization, manuscript conceptualization and writing, experimentation and data analysis, and manuscript final approval; H.R.B., protocol optimization, manuscript writing, experimentation; M.S., protocol development and optimization, manuscript editing, experimentation; J.S.S., protocol development and optimization, draft writing; S.H., protocol development and draft writing; C.K.G, manuscript conceptualization, manuscript editing, and manuscript final approval.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>