<article>
<section>
<h1 id="header">Generation of Inexpensive, Highly Labeled Probes for Fluorescence <i>In Situ</i> Hybridization (FISH)</h1>
<p><time datetime="2020-01-31">Published: January 31, 2020</time></p>
<p>Rahul Sharma<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> and Peter Meister<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Cell Fate and Nuclear Organization Laboratory, Institute for Cell Biology, University of Bern, Baltzerstrasse 4, CH 3012 Bern</p>
<p id="aff2"><sup>2</sup>Present address: Molecular and Cell Biology Laboratory, Salk Institute for Biological Studies, La Jolla, CA 92037, USA</p>
<p id="fn1"><sup>3</sup>Technical Contact</p>
<p id="fn2"><sup>4</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:rsharma@salk.edu">rsharma@salk.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:peter.meister@izb.unibe.ch">peter.meister@izb.unibe.ch</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2019.100006">10.1016/j.xpro.2019.100006</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>DNA-FISH remains the method of choice to visualize genomic regions <i>in situ</i> ranging from a single locus to entire chromosomes. Current methods to generate probes rely on expensive kits that vary in labeling efficiency and are limited by the size and/or amount of starting material and by the choice of fluorophores. Here we describe a protocol to prepare inexpensive ($20) DNA-FISH probes using an isothermal polymerase, incorporating labeled nucleotides while amplifying minute amounts of any template (PCR fragments/BAC/YAC/fosmids).</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib1">Grosmaire et al. (2019)</a> and <a href="#bib4">Sharma et al. (2014)</a>.</p>
<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Direct incorporation of modified nucleotides during isothermal template amplification</li>
<li>Adaptable to the size, amount, type of template DNA, and choice of fluorophores</li>
<li>Inexpensive probes and easily scalable from single locus to multiple chromosomes</li>
<li>Method consistently yields good quantity of highly labeled probes (∼1 dye/100 bases)</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/22-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<h3>DNA Template Preparation</h3>
<p>The amount of total starting material required is between 10-100 ng. This amount can be easily recovered from a single miniprep (for plasmids/cosmids/fosmids) or a PCR reaction.</p>
<p>A variety of methods are available to find suitable starting material. For example, the UCSC genome browser provides a track with BACs and YACs covering the mouse and the human genome. Additional methods have been published elsewhere (<a href="#bib2">Nedbal et al., 2012</a>) and are available online (<a href="https://github.com/webfish/">https://github.com/webfish/</a>).</p>
<p>The isothermal polymerase uses a rolling circle amplification mechanism (RCA) which works optimally with a circular template. If starting from shorter (&lt; 2 kb) PCR fragments, use the following method to circularize the fragments:</p>
<table>
<thead>
<tr>
<th>Reagents</th>
<th>Volume in microliter (ul)</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNA</td>
<td>up to 1 ug</td>
</tr>
<tr>
<td>T4 ligase buffer (10x)</td>
<td>5 ul</td>
</tr>
<tr>
<td>Water</td>
<td>to 42 ul</td>
</tr>
<tr>
<td>T4 ligase</td>
<td>2 ul</td>
</tr>
<tr>
<td>T4 PNK</td>
<td>1 ul</td>
</tr>
</tbody>
</table>
<p>Incubate at 37°C for one h.</p>
<p>Purify either with a PCR purification kit (e.g., QIAGEN minElute) or with SPRI beads (2x Ampure) according to the manufacturer’s instructions.</p>
<div class="note">
<span class="note-title">Note:</span> Although not tested here, purified genomic DNA can also be directly used for amplification. Please refer to <a href="#bib3">Pan et al. (2008)</a>.</div>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>Phi29 DNA polymerase</td>
<td>NEB</td>
<td>M0269S</td>
</tr>
<tr>
<td>Phi29 DNA polymerase 10X reaction buffer</td>
<td>NEB</td>
<td>Included in M0269S</td>
</tr>
<tr>
<td>BSA</td>
<td>NEB</td>
<td>Included in M0269S</td>
</tr>
<tr>
<td>Deoxynucleotide (dNTP) Solution Set</td>
<td>NEB</td>
<td>N0446S</td>
</tr>
<tr>
<td>Aminoallyl-dUTP</td>
<td>Fisher Scientific</td>
<td>R0091</td>
</tr>
<tr>
<td>ATTO-488 NHS ester</td>
<td>Attotec</td>
<td>AD 488-3</td>
</tr>
<tr>
<td>ATTO-647N NHS ester</td>
<td>Attotec</td>
<td>AD 647N-3</td>
</tr>
<tr>
<td>ATTO-565 NHS ester</td>
<td>Attotec</td>
<td>AD 565-3</td>
</tr>
<tr>
<td>Glycogen from oyster</td>
<td>Millipore Sigma</td>
<td>G8751</td>
</tr>
<tr>
<td>Ribonucleic acid, transfer (tRNA)</td>
<td>Millipore Sigma</td>
<td>R9001</td>
</tr>
<tr>
<td>Formamide</td>
<td>Millipore Sigma</td>
<td>47670</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>NEB</td>
<td>M0202</td>
</tr>
<tr>
<td>T4 PNK</td>
<td>NEB</td>
<td>M0201</td>
</tr>
<tr>
<td>Phenol solution</td>
<td>Millipore sigma</td>
<td>P4557</td>
</tr>
<tr>
<td>Ethanol 100%, 95% and 70%</td>
<td>Lab Stock</td>
<td>N/A</td>
</tr>
<tr>
<td>3M Sodium acetate pH 5.2</td>
<td>Lab Stock</td>
<td>N/A</td>
</tr>
<tr>
<td>0.2M Sodium bicarbonate solution pH 9</td>
<td>Lab Stock</td>
<td>N/A</td>
</tr>
<tr>
<td>AG 501-X8 mixed bed resin</td>
<td>Biorad</td>
<td>1426425</td>
</tr>
<tr>
<td>MilliQ or Nuclease Free Water</td>
<td>Lab Stock</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>MinElute PCR Purification Kit</td>
<td>QIAGEN</td>
<td>2800a4</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Random Nonamer NNNNNNN*N*N (* phosphorothiolate bond)</td>
<td>Most oligonucleotides providers</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>DNA template (e.g., purified fosmid minipreps, PCR amplicons and plasmids)</td>
<td>User defined</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Heat block</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Minicentrifuge</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>−80 freezer</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Branson Ultrasonics 250 Digital Sonifier or similar Thermocycler (Optional for PCR amplification and incubations)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Magnetic stand (Optional if using magnetic beads for DNA purification)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<h3>Deoxynucleotide Master Mix (dAGC) Preparation</h3>
<p>Prepare the following mix of dATP, dGTP, dCTP to obtain a final concentration of 10 mM:</p>
<table>
<thead>
<tr>
<th>Reagent</th>
<th>Volume in microliter (ul)</th>
</tr>
</thead>
<tbody>
<tr>
<td>100 mM dATP</td>
<td>10 ul</td>
</tr>
<tr>
<td>100 mM dGTP</td>
<td>10 ul</td>
</tr>
<tr>
<td>100 mM dCTP</td>
<td>10 ul</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>70 ul</td>
</tr>
<tr>
<td><b>Final Volume</b></td>
<td><b>100 ul</b></td>
</tr>
</tbody>
</table>
<p>Prepare 10 ul aliquots and store at −20°C.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A master mix with all nucleotides, including dTTP (1 mM final concentration) and amino-dUTP (1mM final concentration can be prepared by replacing the 70 ul of nuclease free water with 0.5 ul of dTTP 100 mM, 5 ul of amino-dUTP 10 mM and 64.5 ul of nuclease free water). In any case, we recommend aliquoting the premixed nucleotides in 3 ul volume/vial to avoid multiple freeze-thaw cycles.</div>
<h3>Sodium Bicarbonate Buffer Preparation</h3>
<p>Prepare 0.2 M sodium bicarbonate solution and adjust pH to 9 using sodium hydroxide.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Maintaining the pH of this buffer is critical. Frequent use might change the pH. Therefore, make aliquots and store at −20°C.</div>
<h3>ATTO NHS Ester Dye Preparation</h3>
<p>ATTO dyes come as a lyophilized powder. Reconstitute the dye by adding DMSO to a final concentration of 1 uM, prepare 5-10 ul aliquots and store at −20°C indefinitely.</p>
<div class="critical">
<span class="critical-title">Critical:</span> NHS ester groups are highly reactive to moisture and amine groups; this negatively affects dye coupling to DNA. Therefore, it is advisable to use the aliquots only once.</div>
<h3>Random Nonamer Preparation (100 uM)</h3>
<p>Most primer providers produce these random primers. Order NNNNNNNNN. In case the GC/AT content of the target organism genome is more than 20% away from 50%, the ratio between the different nucleotides can be adjusted. Reconstitute the lyophilized primers to a final concentration of 100 uM in water.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The phi29 polymerase has 3′ exonuclease activity, therefore it is essential to use phosphorothioate bonds for the last two bonds of the random nonamer to avoid digestion of these by the polymerase.</div>
<h3>Purified Glycogen Preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Glycogen is used an inert carrier to significantly improve nucleic acid precipitation in ethanol. Once precipitated, it forms a visible pellet, facilitating the localization of the co-pelleted DNA. While commercial alternatives exist, glycogen can be quickly prepared from powder at much lower costs and stored indefinitely at −20°C as frozen aliquots. (This protocol is adapted from a protocol for glycogen preparation available on the website of the Molecular Oncology laboratory, University of Chicago Medical Center)</p>
<ol>
<li>Dissolve 5 g of glycogen (from Oyster) in 10 mL of MilliQ water. Transfer 0.5-1ml to a 2ml microcentrifuge tube.</li>
<li>Add an equal volume of aqueous phenol (Millipore Sigma P4557; this is not the phenol/chloroform/isoamyl alcohol mixture used for DNA extraction). Mix vigorously by shaking the tubes or vortexing. Handle phenol carefully by using proper personal protective equipment and working in a chemical hood.</li>
<li>Spin tube at 13,000 rpm for 10 min at 4C. Recover the aqueous upper phase and transfer to a fresh tube.</li>
<li>Add an equal volume of chloroform and mix thoroughly by vortexing.</li>
<li>Spin tube at 13,000 rpm for 10 min at 4°C. Recover the aqueous phase and transfer to a fresh tube.</li>
<li>Add an equal volume of 100% ethanol to the obtained supernatant. This will precipitate the glycogen.</li>
<li>Spin at 13,000 rpm for 10 min at 4°C and remove supernatant.</li>
<li>Air dry pellet. Weigh and resuspend in MilliQ water to obtain a final concentration of 20 mg/mL. Make aliquots and store at −20°C.</li>
</ol>
<h3>Purified Glycogen Preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1-2 h</div>
<div class="critical">
<span class="critical-title">Critical:</span> Formamide is a hazardous material. Use proper personal protective equipment while handling and work under a chemical hood.</div>
<p>Add 5 g of AG 501-X8 mixed bed resin (Biorad 1426425) to 100 mL of Formamide (Sigma 47670). Stir for 1 h and filter through Whatman paper. Prepare 1 mL aliquots and store at −20°C indefinitely.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Cost for commercial kits that apply similar chemistry begins at about $500 for the preparation of 10 probes ($50/probe). Our cost estimate for all reagents required to setup our protocol is about a $1000 for 50 reactions ($20/probe).</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Set Up Amplification Reaction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 16 h</div>
<p>To generate high quality FISH probes, our approach relies on efficient incorporation of an amine-modified nucleotide (dUTP) during the amplification of template DNA. Nick translation only moderately amplifies DNA while PCR based probe preparation methods are not possible with large fragments. Therefore, we used the phi29 DNA polymerase, an isothermal high fidelity DNA replicator capable of whole genome amplification from single cells (<a href="#bib3">Pan et al., 2008</a>). Using random nonamer primers, phi29 DNA polymerase generates large concatemers of template DNA in an unbiased rolling circle amplification reaction. phi29 is able to incorporate aminoallyl-dUTP instead of dTTP at a ratio of up to 1:1 without compromising the amplification yield.</p>
<ol>
<li>Put 50 ng of purified template DNA in a final volume of 2 ul in an eppendorf tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of initial template DNA is important. It is reported that even 10 ng of total DNA template is sufficient for amplification and more than 50 ng does not increase the final yield (<a href="#bib3">Pan et al., 2008</a>).</div>
<ol start="2">
<li>Denature the DNA at 95°C for 3 min and immediately chill on ice for 5 min.</li>
<li>Add the following to the DNA in the following order (or make a master mix for multiple tubes):</li>
</ol>
<table>
<thead>
<tr>
<th>Reagent</th>
<th>Volume in microliter (ul)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>15.4 ul</td>
</tr>
<tr>
<td>10X reaction buffer</td>
<td>3 ul</td>
</tr>
<tr>
<td>BSA 10 mg/mL</td>
<td>0.6 ul</td>
</tr>
<tr>
<td>Nonamer 100 uM</td>
<td>2 ul</td>
</tr>
<tr>
<td>dAGC (10 mM) mixture</td>
<td>3 ul (final concentration 1 mM)</td>
</tr>
<tr>
<td>dTTP (1 mM)</td>
<td>1.5 ul (final concentration 50 uM)</td>
</tr>
<tr>
<td>Aminoallyl-dUTP (1 mM)</td>
<td>1.5 ul (final concentration 50 uM)</td>
</tr>
<tr>
<td>Phi29 DNA polymerase</td>
<td>1 ul</td>
</tr>
<tr>
<td><b>Final Volume</b></td>
<td><b>30 ul</b></td>
</tr>
</tbody>
</table>
<p>Alternatively, when using a nucleotide mixture with all nucleotides including amino-dUTP, use 3 ul of this mixture and add an additional 3 ul of nuclease free water (total volume of water 18.4 ul).</p>
<div class="critical">
<span class="critical-title">Critical:</span> The dTTP:aminoallyl-dUTP ratio is a critical variable. We tested a variety of ratios of dUTP:dTTP, keeping the final concentration at 100 uM. Based on the final amplification yield and labeling efficiency, a 1:1 ratio gave the best results.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Change the Phi29 10x reaction buffer and BSA after 20-25 reactions as the amplification yield decreases with the number of freeze-thaw cycles. Alternatively, make aliquots of the reaction buffer. According to the manufacturer, older reaction buffers can be supplemented with 4 mM DTT to restore activity.</div>
<ol start="4">
<li>Vortex briefly, spin and incubate the tube at 30°C for 12-16 h. Immediately proceed to the next step.</li>
<li>Heat inactivate the enzyme at 65°C for 10 min. Cool on ice.</li>
<li>Run 1-2 ul of the reaction on 1% agarose gel. A single faint band is visible with a high molecular weight that runs above the 10 kb band of the DNA ladder (<a href="#fig1">Figure 1</a>). Given the rolling circle amplification process, the size of the band will be independent of the starting material size.</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/22-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Detection of Amplified Template DNA</div>
<p>Agarose gel picture shows the size of the amplified product (lane 2 and 3) compared to DNA ladder (lane 1).</p>
</figcaption>
</figure>
<h3>Alternate Option for Multiple Fosmid Amplification</h3>
<ol>
<li>If more than one template is to be amplified together (example: chromosome paints made of multiple fosmids), separate master mixes can be prepared, as follows:</li>
</ol>
<ol type="a">
<li>Pool 100 ng of each template</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Empirical testing showed that mixing more than 10 fosmids/plasmids per tube led to uneven staining, likely due to small differences in template concentrations leading to larger differences between probes after amplification. We therefore recommend to make another master mix if more than 10 different templates have to be amplified simultaneously.</div>
<ol start="2" type="a">
<li>Calculate the concentration (ng/ul) of the new master mix of fosmids since the corresponding volumes are not the same for the individual purified fosmid stocks.</li>
<li>Take a volume corresponding to 50 ng of this new mix (should not be more than 2 ul) in a separate tube and amplify using the protocol above.</li>
<li>Independently amplified master mixes can then be pooled together for further downstream processing.</li>
</ol>
<p>The idea is to maintain equal stoichiometry between the different fosmids present in the same tube thereby providing an equal chance for each independent fosmid to be amplified in the mix and achieve homogeneity.</p>
<h3>Sonication and Purification of the Amplification Products</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1-2 h</div>
<p>DNA fragments obtained by RCA are concatemers of hundreds of copies of the original template. To facilitate entry into the nucleus and efficient hybridization, the DNA probe needs to be sheared into smaller fragments of 300-700 bp (average 500bp). For this purpose, we use a Branson sonicator to shear DNA into fragments of desirable probe length.</p>
<ol>
<li>Add 370 ul of MilliQ water to the amplification reaction.</li>
<li>Sonicate the sample in an ice box using a Branson sonifier with the following parameters:</li>
</ol>
<p>Amplitude: 10%</p>
<p>Time: 80 s</p>
<p>Pulse: 0.5 s (ON and OFF)</p>
<ol start="3">
<li>Purify the reaction using a PCR clean up kit or SPRI beads (1.8x Ampure) according to the manufacturer’s instructions. Elute in 80 ul MilliQ water. Estimate the DNA concentration using Nanodrop. (Under ideal conditions, the reaction would yield a total of 1.5 to 2.4 ug of DNA)</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Sonication time should be modified to achieve an average fragment length of 500 base pairs (<a href="#fig2">Figure 2</a>). It is desirable to check the sonicated product on a gel for the first time to ensure fragment length.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/22-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Size Distribution of Fragmented DNA</div>
<p>Agarose gel picture shows the size of the fragmented DNA (lane 2) post sonication compared to the DNA ladder (lane 1).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> The reaction volume should be raised to 400 ul in order to avoid frothing of the sample.</div>
<div class="note">
<span class="note-title">Note:</span> If using a different sonicator, it is recommended to optimize conditions for efficient shearing.</div>
<h3>DNA Precipitation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2-18 h</div>
<p>DNA fragments obtained post sonication are precipitated before fluorophore labeling for two reasons: a) to remove any unincorporated amine labeled dUTP that could possibly react with the amine reactive dyes leading to inefficient labeling; and b) to concentrate DNA for buffer exchange in the labeling step. We recommend using purified glycogen as a carrier to efficiently precipitate DNA.</p>
<ol>
<li>Add the following to the reaction:</li>
</ol>
<table>
<thead>
<tr>
<th>Reagent</th>
<th>Volume in microliter (ul)</th>
</tr>
</thead>
<tbody>
<tr>
<td>3M sodium acetate (pH 5.2)</td>
<td>10 ul (or 0.1 volume of the total sample volume)</td>
</tr>
<tr>
<td>Glycogen (purified, 20 mg/mL)</td>
<td>1.5 ul (see ‘Protocol to prepare purified glycogen’)</td>
</tr>
<tr>
<td>Chilled −20°C 100% ethanol</td>
<td>250 ul</td>
</tr>
</tbody>
</table>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> 1 ul of tRNA (10 mg/ml) can be added instead of glycogen as a precipitation carrier.</div>
<ol start="2">
<li>Vortex and spin briefly and incubate the reaction at −20°C for at least 1 h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The reaction can be left up to 16 hours at −20°C. Longer incubation times improve precipitation.</div>
<ol start="3">
<li>Spin the tube at 16000 g for 12 min at 4°C.</li>
<li>Remove supernatant and wash pellet with 500 ul of 70% Ethanol.</li>
<li>Spin again at 16000 g for 5 min at 4°C.</li>
<li>Air dry the pellet.</li>
<li>Add 6 ul of nuclease free water to the tube and incubate at 37°C for 15 min. Pipette gently to dissolve the pellet.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Tubes can be stored at −20°C for days to weeks.</div>
<h3>Labeling the DNA Fragments with Fluorescent Dye</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2-16 h</div>
<p>The aminoallyl group linked to dUTP reacts readily with the N-hydroxysuccinimidyl (NHS)-ester group of fluorescent dyes to form a stable amide bond, thereby coupling the dye to the modified nucleotide (<a href="#bib5">Wiederschain 2011</a>). To maintain the NHS ester in its reactive state, the pH of the buffer has to be adjusted to be between 7 and 9 while the buffer cannot contain any other amine groups (e.g., Tris). At this stage, the DNA fragments can be coupled to virtually any fluorophore of choice.</p>
<ol>
<li>Denature the DNA by incubating the tube at 95°C for 3 min and immediately cool on ice.</li>
<li>Spin briefly and add in the following order:</li>
</ol>
<table>
<thead>
<tr>
<th>Reagent</th>
<th>Volume in microliter (ul)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sodium bicarbonate solution</td>
<td>3 ul</td>
</tr>
<tr>
<td>Fluorescent dye (in DMSO)</td>
<td>1-1.5 ul (1000-1500 picomoles)</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The ideal amplification yields ∼500 picomoles of 1:1 dUTP/dTTP incorporated DNA. Therefore, 1000 picomoles of the fluorescent dye are in large excess for the coupling reaction.</div>
<ol start="3">
<li>Vortex for 15 s, spin briefly and incubate at 18-24°C in the dark for at least one h.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Reaction can be left up to 3 days in the dark at 18-24°C. Increased incubation times do not improve dramatically the labeling yield.</div>
<h3>Purification and Precipitation of the Labeled DNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3-20 h</div>
<p>DNA fragments coupled to fluorescent dyes as described in section IV, need to be purified to remove excess dye. These fragments can then be precipitated individually or pooled together with other probes depending on the experiment. For example, probes targeting different regions of single/multiple chromosomes can be pooled together at this step to generate probes for chromosome painting.</p>
<ol>
<li>Add 340ul of MilliQ water to the reaction and purify using the PCR purification kit or SPRI beads (1.8x Ampure) according to the manufacturer’s instructions. Elute in 20 ul of MilliQ water.</li>
<li>Check the probe concentration and absorption of the fluorochrome at the specific wavelength using the ‘microarray’ function of Nanodrop (<a href="#fig3">Figure 3</a>).</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/22-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Quantitative Analysis of the FISH Probe</div>
<p>Snapshot of a nanodrop screen showing the absorbance spectrum of the final probe. The readings have been generated using the microarray function on the Nanodrop. Nucleotides absorb at 260 nm whereas the dye (ATTO-488) absorbs at 488nm. The spectrum shows a clear absorbance peak at the latter wavelength. The ratio between the two absorbances is used to calculate the labeling efficiency (see text for details).</p>
</figcaption>
</figure>
<ol start="3">
<li>Labeling efficiency can be calculated using a Dye:Base ratio calculator (<a href="http://www.genelink.com/tools/gl-BDratiores.asp">http://www.genelink.com/tools/gl-BDratiores.asp</a>)</li>
<li>Transfer a volume corresponding to 400 ng of DNA of each probe (in case there is more than one probe required for simultaneous hybridization) into a separate tube.</li>
<li>To precipitate, add:</li>
</ol>
<table>
<thead>
<tr>
<th>Reagent</th>
<th>Volume in microliter (ul)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sodium acetate</td>
<td>10 ul (for every 90 ul)</td>
</tr>
<tr>
<td>tRNA (10mg/mL)</td>
<td>1 ul</td>
</tr>
<tr>
<td>Chilled 95% ethanol</td>
<td>400 ul</td>
</tr>
</tbody>
</table>
<ol start="6">
<li>Vortex vigorously and incubate at −20°C for at least 2 h</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Reaction can be left up to 16 hours at −20°C. Longer precipitation improves purification yield.</div>
<ol start="7">
<li>Spin the reaction at 11,000 rpm for 20 min at 4°C.</li>
<li>Decant the supernatant and wash with cold 70% ethanol. Spin for 5 min with the same conditions.</li>
<li>Remove the supernatant carefully and dry the pellet in the dark. At this point, the pellet should be colored and fluoresce under a dissecting scope (as in <a href="#fig4">Figure 4</a>).</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/22-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Visual Assessment of the FISH Probe</div>
<p>Images of microtubes containing final dissolved probes labeled with ATTO-488 and ATTO-565 under a fluorescent dissecting scope (monochrome pictures acquired sequentially, displayed in false colors).</p>
</figcaption>
</figure>
<ol start="10">
<li>Dissolve the obtained pellet in 20 ul deionised formamide and store at −20°C indefinitely (<a href="#fig4">Figure 4</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not use regular formamide. Probes resuspended in deionised formamide hybridize optimally in FISH experiments. See ‘before you begin’ section for the preparation of deionised formamide.</div>
<p>If probes of different concentration have to be mixed before being applied to the sample, the precipitated probes can be reconstituted in nuclease free water before adding deionized formamide to the hybridization mixture. The efficacy and long term storage of probes suspended in water should be determined by the user.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>We tested directly incorporating fluorophore labeled dUTP during DNA amplification but in our hands this strategy did not work. We failed to obtain any product suggesting that phi29 is unable to incorporate directly labeled nucleotides.</p>
<p>Our protocol generates double stranded probes that are used for DNA-FISH but cannot be used for RNA-FISH.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem 1</h3>
<p>Low yield of amplified product</p>
<h3>Potential Solution A</h3>
<p>Change phi29 reaction buffer.</p>
<h3>Potential Solution B</h3>
<p>Check if random nonamers have phosphorothioate bonds.</p>
<h3>Problem 2</h3>
<p>Poor fragmentation of DNA post sonication.</p>
<h3>Potential Solution</h3>
<p>Increase sonication time and amplitude on test reaction.</p>
<h3>Problem 3</h3>
<p>Poor labeling efficiency</p>
<h3>Potential Solution A</h3>
<p>Check pH of sodium bicarbonate buffer and make sure it is 9.</p>
<h3>Potential Solution B</h3>
<p>Use fresh aliquot of NHS-ester dye.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>Expected labeling varies with the kind of dye being used. In our experience, ATTO-488 gives a labeling of 1-1.5 dye molecules/100 bases, on average. On the other hand, ATTO-647N gives 0.5-1 dye molecules/100 bases and ATTO-565 gives 0.8-1.2 dye molecules/100 bases. These ranges of probe labels provide good signals upon hybridization under a standard confocal microscope (<a href="#fig5">Figure 5</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/22-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Probes Generated Using This Protocol Efficiently Label a Single Locus or Entire Chromosomes</div>
<p>Partial z stacks of <i>C. elegans</i> embryos with 2-color DNA-FISH performed with probes of different sizes. Top panel shows simultaneous hybridization of (i) a probe generated from a plasmid stably integrated as a single copy in the worm genome (red, ATTO-488) and (ii) a locus on chromosome X (green, ATTO-647N). Bottom panel shows chromosome X (red, ATTO-488) and V (green, ATTO-647N) painting probes generated from a mixture of 50 fosmids spread across each chromosome. Blue scale bar is 5 μm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Grosmaire, M., Launay, C., Siegwald, M., Brugière, T., Estrada-Virrueta, L., Berger, D., Burny, C., Modolo, L., Blaxter, M., Meister, P., et al. (2019). Males as somatic investment in a parthenogenetic nematode. Science <i>363</i>, 1210-1213.</p>
<p id="bib2">Nedbal, J., Hobson, P.S., Fear, D.J., Heintzmann, R., and Gould, H.J. (2012). Comprehensive FISH probe design tool applied to imaging human immunoglobulin class switch recombination. PLoS One <i>7</i>, e51675.</p>
<p id="bib3">Pan, X., Urban, A.E., Palejev, D., Schulz, V., Grubert, F., Hu, Y., Snyder, M., and Weissman, S.M. (2008). A procedure for highly specific, sensitive, and unbiased whole-genome amplification. Proc. Natl. Acad. Sci. USA <i>105</i>, 15499-15504.</p>
<p id="bib4">Sharma, R., Jost, D., Kind, J., Gómez-Saldivar, G., van Steensel, B., Askjaer, P., Vaillant, C., and Meister, P. (2014). Differential spatial and structural organization of the X chromosome underlies dosage compensation in C. elegans. Genes Dev. <i>28</i>, 2591-2596.</p>
<p id="bib5">Wiederschain, G.Y. (2011). Molecular probes handbook. A guide to fluorescent probes and labeling technologies. Biochemistry Moscow, <i>76</i>, 1276.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Acknowledgments</h3>
<p>We thank the past and present members of the Meister laboratory for their support during the development of this technique and for careful reading of this manuscript. The Meister laboratory is supported by the <a href="https://doi.org/10.13039/501100001711">Swiss National Foundation</a> (PP00P3_133744/159320, 31003A_176226), the <a href="https://doi.org/10.13039/501100004784">Novartis Biomedical Research Foundation</a>, and the <a href="https://doi.org/10.13039/100009068">University of Bern</a>.</p>
<h3>Author Contributions</h3>
<p>P.M. conceived the idea. R.S. optimized the conditions, standardized the protocol, and performed the FISH and imaging experiments. P.M. and R.S. analyzed the data and wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no conflicts of interest.</p>
</section>
</article>