<article>
<section>
<h1 id="header">Activation of a diluted <i>E. coli</i> cell-free transcription-translation system within liposomes by hypertonic concentration</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Yukino Matsui,<sup><a href="#aff1">1</a></sup> Toshiki Akui,<sup><a href="#aff1">1</a></sup> Nobuhide Doi,<sup><a href="#aff1">1</a></sup> and Kei Fujiwara<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biosciences &amp; Informatics, Keio University, 3-14-1 Hiyoshi, Kohokuku, Yokohama, Kanagawa 223-8522, Japan</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:fujiwara@bio.keio.ac.jp">fujiwara@bio.keio.ac.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101003">10.1016/j.xpro.2021.101003</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We present a protocol for activating protein synthesis in liposomes encapsulating a diluted <i>E. coli</i> cell extract-based TX-TL (transcription-translation) system by hypertonic concentration. Protein expression is turned on in the liposome-encapsulated TX-TL system by simple treatment with a concentrated external solution. The expression of sfGFP is demonstrated here, but it can be applied to other proteins. This protocol can be applied to the development of artificial cells utilizing the switch-on mechanism to activate protein expression, responding to the outer environment.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00709-7/sref1">Akui et al. (2021)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A diluted cell extract-based transcription translation system encapsulated in liposomes</li>
<li>Protein synthesis is activated inside liposomes by hypertonic treatment</li>
<li>Protocol can be applied to developing artificial cells utilizing the switch-on mechanism</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1241-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Prepare S30 cell extract for cell-free TX-TL system</h3>
<p>The TX-TL system required for this protocol can either be prepared from an <i>E. coli</i> cell-extract (<a href="#bib2">Fujiwara and Doi, 2016</a>) or reconstituted from individual components (PURE system, <a href="#bib5">Shimizu et al., 2001</a>). This section describes detailed steps for preparing an E coli cell extract-based TX-TL system.</p>
<h4 id="sec1.1.1">Day 1</h4>
<ol>
<li>Prepare and autoclave the buffers and media as described below.
<ol type="a">
<li>1 L of 2× YTPG media [Yeast Extract 10 g, Tryptone 16 g, NaCl 5 g, K<sub>2</sub>HPO<sub>4</sub> 7 g, KH<sub>2</sub>PO<sub>4</sub> 3 g, 15 mL of 1 M KOH to adjust pH7.2, and then, filled up with distilled water to 950 mL]. After autoclave, add 50 mL of 2 M filter-sterilized glucose.
<div class="note">
<span class="note-title">Note:</span> Glucose should be added to the media after autoclave.</div>
<div class="note">
<span class="note-title">Note:</span> We typically use 2 L baffled culture flasks with foil for cell culture. It is no problem to use different sizes of baffled flasks or Sakaguchi flasks.</div>
</li>
<li>S30A Buffer [10 mM BisTris, 60 mM GluK (potassium-glutamate), 10 mM Mg(oAc)<sub>2</sub>]
<div class="note">
<span class="note-title">Note:</span> Prepare 10 mL per 1 L culture.</div>
</li>
<li>S30B Buffer [10 mM BisTris, 60 mM GluK, 14 mM Mg(oAc)<sub>2</sub>]
<div class="note">
<span class="note-title">Note:</span> The amount needed varies according to the buffer exchange protocol. In the case of the Amicon filter method, prepare 100 mL per 1 L culture. In the case of dialysis protocol, prepare 1 L or more (it depends on the volume of cell extract).</div>
</li>
<li>400 mM Sucrose
<div class="note">
<span class="note-title">Note:</span> Prepare 200 mL per 1 L culture. Store in the refrigerator and cool it on ice prior to use.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Cell Culture
<ol type="a">
<li>Inoculate <i>Escherichia coli</i> BL21(DE3) in 10 mL of 2× YTPG or LB.
<div class="note">
<span class="note-title">Note:</span> This protocol is not suitable for some <i>Escherichia coli</i> strains. It may be possible to obtain cell extract from other <i>E. coli</i> strains. For example, we confirmed that it was possible with MG1655 and BL21-CodonPlus(DE3) strains. However, some strains, such as BL21 Star (DE3) (it has a mutation in RNaseE gene) are not suitable. In most cases, we recommend using BL21(DE3) or BL21-CodonPlus(DE3) strains.</div>
</li>
<li>Cultivate the cells for overnight (16–20 h) at 37°C with shaking (120–200 rpm).</li>
</ol>
</li>
</ol>
<h4 id="sec1.1.2">Day 2</h4>
<ol start="3">
<li>Cell Culture, Harvest
<ol type="a">
<li>Inoculate 1 L of 2×YTPG media with 10 mL of the o/n culture.</li>
<li>Grow the culture at 37°C with shaking at 170 rpm until OD<sub>600</sub> reaches 1.5–1.8, which requires 2–3 h cultivation.
<div class="optional">
<span class="optional-title">Optional:</span> When your template for TX-TL uses T7 promoter for gene expression, express T7 RNA polymerase encoded in DE3 gene by adding IPTG (0.1 mM final) at OD<sub>600</sub> = 0.1.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Collect cells at late log phase. OD<sub>600</sub> should be measured after dilution of the cell culture with water or medium to ensure linear correlation between OD<sub>600</sub> and cell number.</div>
</li>
<li>Before harvesting, leave the culture for 15 min at room temperature (20°C–30°C). During the process, pre-cool the centrifuge and rotor for cell collection.</li>
<li>Collect cells by centrifugation. For example, 15,000 × <i>g</i> centrifugation for 5 min at 4°C. Centrifugation speed is not critical.</li>
<li>Remove the supernatant and keep the cell pellet on ice.
<div class="critical">
<span class="critical-title">Critical:</span> Keep the sample on ice all the time after this step.</div>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Lysozyme treatment
<ol type="a">
<li>Wash a paint brush with 5–10 mL of 400 mM sucrose in a 50 mL tube (<a href="#fig1">Figure 1</a>).</li>
<li>Add 20 mL of 400 mM ice-cold sucrose to the cell pellet and resuspend softly using the brush. If you have more than one pellet, you can pour this cell suspension into the other pellet to gather cells in a single tube (<a href="#mmc1">Methods video S1</a>).
<div class="note">
<span class="note-title">Note:</span> Resuspend all the cell pellets as homogenous as possible.</div>
</li>
<li>After resuspending, pour the total cell suspension to a single 50 mL tube and centrifuge at 15,000 × <i>g</i> for 2 min at 4°C.</li>
<li>Remove the supernatant and resuspend the cell pellet with 20 mL sucrose using the brush. Centrifuge again.</li>
<li>Remove the supernatant and resuspend the cell pellet in 30 mL sucrose.</li>
<li>Add 150 μL of 20 mg/mL Lysozyme to the cell suspension and mix the cell suspension with moderate shakes until the lysozyme solution is homogeneously diffused. Repeat this step 3 times (in total you will add 600 μL lysozyme solution).</li>
<li>Keep the sample on ice for 1 h (lysozyme treatment).</li>
<li>Centrifuge at 15,000 × <i>g</i> for 2 min at 4°C.</li>
<li>Wash the cell pellet twice by repetitively resuspending in 30 mL sucrose and centrifuging to remove the supernatant.</li>
<li>After the final centrifugation step, remove the supernatant and measure the wet cell weight (typically 2 g/1 L culture).</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Freeze and thaw
<ol type="a">
<li>Resuspend the cell pellet with the cold S30A Buffer (1.5 mL per gram wet cell pellet weight).</li>
<li>Aliquot the cell suspension into 1.7 mL tubes (0.5–1 mL for each tube).</li>
<li>Dip the tubes into liquid nitrogen for 5 min.</li>
<li>Transfer the tubes to ice water until fully thawed (Use a floating tube rack and take care not to immerse the lid of the tube in ice water). To facilitate thawing, any ice that has formed around the tubes should be removed at about 3, 6, 9 min after the transfer. This step would take around 30 min.</li>
</ol>
</li>
<li>Centrifuge for 1 h at 30,000 × <i>g</i> at 4°C.</li>
</ol>
<ol start="7">
<li>Run-off reaction
<ol type="a">
<li>Dispense the supernatant into 1.7 mL tubes (1 mL for each tube).</li>
<li>Incubate the tubes for 1 h at 37°C with shaking at 200 rpm.
<div class="note">
<span class="note-title">Note:</span> The supernatant should become milky white during this step.</div>
<div class="note">
<span class="note-title">Note:</span> Pre-warm centrifuge to 37°C for the next step (7c).</div>
</li>
<li>Centrifuge at 10,000 × <i>g</i> for 20 min at 4°C.</li>
<li>Collect the supernatants in a single 15 mL tube or multiple 1.7 mL tubes (The merit of collection in multiple 1.7 mL tubes is faster thawing in step 8a).</li>
<li>Store the collected supernatants as S30 cell extract at −20°C or below (this step can be skipped if the next step is going to be performed on the same day).
<div class="note">
<span class="note-title">Note:</span> Flash-freezing is not required.</div>
</li>
</ol>
</li>
</ol>
<h4 id="sec1.1.3">Day 3</h4>
<ol start="8">
<li>Buffer exchange
<ol type="a">
<li>Leave the S30 extract on ice until completely thawed if frozen.</li>
<li>Exchange the buffer with S30B buffer by dialysis (Usage of pore membrane smaller than 14 kDa MWCO and two-rounds dialysis is recommended) or Amicon Ultra centrifugal filter (10 kDa MWCO). In the case of Amicon filter, concentrate and dilute with the S30B Buffer, and repeat this cycle until the S30A Buffer is diluted 500 to 1000 times with the S30B buffer.</li>
<li>Using the Amicon filter, concentrate the S30 extract until it reaches less than half the volume of the original S30 extract in step a</li>
<li>Measure the concentration of the S30 cell extract using a BCA assay kit.</li>
<li>Dispense 10–100 μL of S30 extract into tubes and store them at −20°C or less. Typically, about 30 mg cell extract is obtained from 1 g wet cells weight.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Flash-freezing is not required. But aliquot S30 extract for storage to avoid multiple freeze-thaw cycles. (Although we have not tested how many freeze-thaw cycles could significantly decrease the TX-TL activity, we find that the activity is not affected by up to 2 freeze-thaw cycles.)</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1241-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. A paint brush washed with the sucrose solution.</div>
</figcaption>
</figure>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1241-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Suspension of cell pellet with the sucrose solution using a paint brush, related to before you begin, prepare S30 cell extract for cell-free TX-TL system, step 4b</div>
</figcaption>
</figure>
<h3 id="sec1.2">Prepare an indicator for observation of liposomes</h3>
<p>We recommend using a fluorescent indicator to aid identification of liposomes in the microscopic field; without a fluorescent indicator it is not easy to judge whether spherical objects observed in transmitted light are liposomes or not (since contaminant oil in water emulsion is also spherical).</p>
<p>In this section, we outline the steps for the preparation of mCherry.</p>
<div class="note">
<span class="note-title">Note:</span> mCherry can be replaced with other fluorescent molecules if desired.</div>
<h4 id="sec1.2.1">Day 1</h4>
<ol start="9">
<li>Transform <i>E. coli</i> BL21 (DE3) with pET29-mCherry-His (<a href="#bib1">Akui et al., 2021</a>) using competent cells, and cultivate overnight (16–20 h) at 37°C on LB agar medium supplemented with 50 μg/mL kanamycin at final.</li>
</ol>
<h4 id="sec1.2.2">Day 2</h4>
<ol start="10">
<li>In the evening, inoculate the transformant into 1 mL LB medium supplemented with 50 μg/mL kanamycin and cultivate the cells overnight (16–20 h) at 37°C with shaking at 120–200 rpm.</li>
</ol>
<h4 id="sec1.2.3">Day 3</h4>
<ol start="11">
<li>Inoculate 1 L of LB medium containing 50 μg/mL kanamycin with 1 mL of the precultured cells.</li>
<li>Cultivate the cells at 37°C with shaking 120–200 rpm to reach OD<sub>600</sub> = 0.1–0.4.</li>
<li>Add 0.1 mM IPTG and cultivate overnight (16–20 h) at 18°C with shaking at 120–200 rpm.</li>
</ol>
<h4 id="sec1.2.4">Day 4</h4>
<ol start="14">
<li>In the morning, collect the cells and resuspend the cell pellet in 10 mL of Binding Buffer [50 mM Tris-HCl (pH 8.0), 500 mM NaCl, 40 mM Imidazole].</li>
<li>Disrupt the cells by sonication. During sonication, the tubes should be cooled with ice water.</li>
<li>Collect the supernatant after 15,000 × <i>g</i> centrifuge for 10 min at 4°C.</li>
<li>Purify the mCherry from the supernatant by using HisTrap column (or equivalent) following the manufacturer’s instructions. For wash and elution, the Binding Buffer and Elution Buffer [50 mM Tris-HCl (pH 8.0), 500 mM NaCl, and 400 mM Imidazole] are used, respectively.</li>
<li>Exchange the buffer with 20 mM Hepes-KOH (pH 7.6) by dialysis or the Amicon Filter method. Because 16 μM mCherry is required for the preparation of the TX-TL system solution below, concentrate mCherry by the Amicon Filter if its concentration is lower than 16 μM.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to reduce the osmotic pressure derived from the buffer for the hypertonic concentration for protein expression. NaCl and Imidazole concentration derived from mCherry purification should be less than 1 mM in the final solution for the TX-TL system. We do not recommend the addition of glycerol to the purification buffers.</div>
<ol start="19">
<li>Dispense 10–100 μL of the mCherry solution into tubes and store at −20°C or below.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Flash-freezing is not required.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>Escherichia coli</i> BL21 (DE3)</td>
<td>New England BioLabs</td>
<td>Cat# C2527</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Folinic acid</td>
<td>Tokyo Chemical Industry</td>
<td>Cat# C2235</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10977023</td>
</tr>
<tr>
<td>Potassium glutamate</td>
<td>Sigma-Aldrich</td>
<td>Cat# G1501-1KG</td>
</tr>
<tr>
<td>Magnesium glutamate</td>
<td>Sigma-Aldrich</td>
<td>Cat# 49605-250G</td>
</tr>
<tr>
<td>Magnesium acetate</td>
<td>Nacalai Tesque</td>
<td>Cat# 208-21</td>
</tr>
<tr>
<td>tRNA</td>
<td>Roche</td>
<td>Cat# 10109541001</td>
</tr>
<tr>
<td>Creatine kinase</td>
<td>Roche</td>
<td>Cat# 10127566001</td>
</tr>
<tr>
<td>ATP</td>
<td>Wako Chemicals</td>
<td>Cat# 014-16913</td>
</tr>
<tr>
<td>GTP</td>
<td>Wako Chemicals</td>
<td>Cat# 073-03113</td>
</tr>
<tr>
<td>CTP</td>
<td>Tokyo Chemical Industry</td>
<td>Discontinued</td>
</tr>
<tr>
<td>UTP</td>
<td>Affymetrix</td>
<td>Discontinued</td>
</tr>
<tr>
<td>IPTG</td>
<td>Nacalai Tesque</td>
<td>Cat# 19742-94</td>
</tr>
<tr>
<td>Tris(hydroxymethyl)aminomethane</td>
<td>Nacalai Tesque</td>
<td>Cat# 35434-05</td>
</tr>
<tr>
<td>NaCl</td>
<td>Nacalai Tesque</td>
<td>Cat# 31320-05</td>
</tr>
<tr>
<td>Imidazole</td>
<td>Nacalai Tesque</td>
<td>Cat# 19004-35</td>
</tr>
<tr>
<td>HEPES</td>
<td>Nacalai Tesque</td>
<td>Cat# 17546-05</td>
</tr>
<tr>
<td>HCl</td>
<td>Nacalai Tesque</td>
<td>Cat# 18321-05</td>
</tr>
<tr>
<td>KOH</td>
<td>Nacalai Tesque</td>
<td>Cat# 28616-45</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Nacalai Tesque</td>
<td>Cat# 30404-45</td>
</tr>
<tr>
<td>Ala</td>
<td>Nacalai Tesque</td>
<td>Cat# 01115-22</td>
</tr>
<tr>
<td>Cys</td>
<td>Nacalai Tesque</td>
<td>Cat# 10309-12</td>
</tr>
<tr>
<td>Asp</td>
<td>Nacalai Tesque</td>
<td>Cat# 03561-12</td>
</tr>
<tr>
<td>Glu</td>
<td>Nacalai Tesque</td>
<td>Cat# 16911-22</td>
</tr>
<tr>
<td>Phe</td>
<td>Nacalai Tesque</td>
<td>Cat# 26910-22</td>
</tr>
<tr>
<td>Gly</td>
<td>Nacalai Tesque</td>
<td>Cat# 17109-35</td>
</tr>
<tr>
<td>His</td>
<td>Nacalai Tesque</td>
<td>Cat# 18116-92</td>
</tr>
<tr>
<td>Ile</td>
<td>Nacalai Tesque</td>
<td>Cat# 20330-02</td>
</tr>
<tr>
<td>Lys</td>
<td>Tokyo Chemical Industry</td>
<td>Cat# L0129</td>
</tr>
<tr>
<td>Leu</td>
<td>Nacalai Tesque</td>
<td>Cat# 20327-62</td>
</tr>
<tr>
<td>Met</td>
<td>Nacalai Tesque</td>
<td>Cat# 21719-02</td>
</tr>
<tr>
<td>Asn</td>
<td>Nacalai Tesque</td>
<td>Cat# 03427-82</td>
</tr>
<tr>
<td>Pro</td>
<td>Nacalai Tesque</td>
<td>Cat# 29001-42</td>
</tr>
<tr>
<td>Gln</td>
<td>Nacalai Tesque</td>
<td>Cat# 16919-42</td>
</tr>
<tr>
<td>Arg</td>
<td>Nacalai Tesque</td>
<td>Cat# 03321-65</td>
</tr>
<tr>
<td>Ser</td>
<td>Nacalai Tesque</td>
<td>Cat# 30608-72</td>
</tr>
<tr>
<td>Thr</td>
<td>Nacalai Tesque</td>
<td>Cat# 33820-82</td>
</tr>
<tr>
<td>Val</td>
<td>Nacalai Tesque</td>
<td>Cat# 36108-42</td>
</tr>
<tr>
<td>Trp</td>
<td>Wako Chemicals</td>
<td>Cat# 202-03383</td>
</tr>
<tr>
<td>Tyr</td>
<td>Nacalai Tesque</td>
<td>Cat# 35709-02</td>
</tr>
<tr>
<td>Spermidine</td>
<td>Nacalai Tesque</td>
<td>Cat# 32108-91</td>
</tr>
<tr>
<td>Creatine phosphate</td>
<td>Wako Chemicals</td>
<td>Cat# 030-04584</td>
</tr>
<tr>
<td>Chloroform</td>
<td>Nacalai Tesque</td>
<td>Cat# 08402-55</td>
</tr>
<tr>
<td>1-Palmitoyl-2-Oleoylphosphatidylcholine (POPC)</td>
<td>Avanti Polar Lipids</td>
<td>Cat# 850457C</td>
</tr>
<tr>
<td>Lysozyme</td>
<td>Nacalai Tesque</td>
<td>Cat# 19499-04</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Nacalai Tesque</td>
<td>Cat# 02739-32</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>Nacalai Tesque</td>
<td>Cat# 19860-44</td>
</tr>
<tr>
<td>Yeast Extract</td>
<td>Gibco</td>
<td>Cat# 212750</td>
</tr>
<tr>
<td>Tryptone</td>
<td>Nacalai Tesque</td>
<td>Cat# 35640-95</td>
</tr>
<tr>
<td>NaCl</td>
<td>Nacalai Tesque</td>
<td>Cat# 31319-45</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Nacalai Tesque</td>
<td>Cat# 28727-95</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>Nacalai Tesque</td>
<td>Cat# 28721-55</td>
</tr>
<tr>
<td>Mineral oil</td>
<td>Nacalai Tesque</td>
<td>Cat# 23306-84</td>
</tr>
<tr>
<td>Glucose</td>
<td>Nacalai Tesque</td>
<td>Cat# 16805-35</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>HisTrap HP</td>
<td>GE Healthcare</td>
<td>Cat# 17524701</td>
</tr>
<tr>
<td>Amicon-Ultra 15 filter 10k</td>
<td>Merck Millipore</td>
<td>Cat# UFC901096</td>
</tr>
<tr>
<td>BCA Assay kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 23225</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pET29-mCherry-His</td>
<td><a href="#bib1">Akui et al. (2021)</a></td>
<td>N/A</td>
</tr>
<tr>
<td>pOR2OR1-sfGFP</td>
<td><a href="#bib2">Fujiwara and Doi (2016)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Others</td>
</tr>
<tr>
<td>molecular sieve 4A</td>
<td>Nacalai Tesque</td>
<td>Cat# 04171-05</td>
</tr>
<tr>
<td>Silicone rubber sheet</td>
<td>TOGAWA Rubber Co. Ltd.</td>
<td>Cat# K-125(50)</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> All reagents and supplies can be replaced with alternatives.</div>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Mineral oil</h3>
<p>Just after opening the mineral oil tube, we always treat the oil with a molecular sieve to remove water and other small chemicals which can be contaminated from environments. Add 10 g of molecular sieve 4A to 100 mL mineral oil. [Store at room temperature (20°C–30°C)].</p>
<table id="undtbl1">
<caption>2× YTPG medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast Extract</td>
<td>10 g</td>
</tr>
<tr>
<td>Tryptone</td>
<td>16 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>5 g</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>7 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>3 g</td>
</tr>
<tr>
<td>KOH (1M)</td>
<td>15 mL</td>
</tr>
<tr>
<td>Glucose (2M)</td>
<td>50 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>935 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Glucose should be added after autoclaving the medium that contains all the rest.</div>
<table id="undtbl2">
<caption>S30A Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration (mM)</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes-KOH pH7.6</td>
<td>2000</td>
<td>10</td>
<td>0.05</td>
</tr>
<tr>
<td>GluK</td>
<td>4000</td>
<td>60</td>
<td>0.15</td>
</tr>
<tr>
<td>Mg(oAc)<sub>2</sub>
</td>
<td>1000</td>
<td>10</td>
<td>0.14</td>
</tr>
<tr>
<td>Ultrapure Water</td>
<td> </td>
<td>n/a</td>
<td>9.66</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td>n/a</td>
<td>10</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave before use.</div>
<div class="note">
<span class="note-title">Note:</span> A month’s storage at 4°C is assured.</div>
<table id="undtbl3">
<caption>S30B Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration (mM)</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes-KOH pH7.6</td>
<td>2000</td>
<td>10</td>
<td>0.5</td>
</tr>
<tr>
<td>GluK</td>
<td>4000</td>
<td>60</td>
<td>1.5</td>
</tr>
<tr>
<td>Mg(oAc)<sub>2</sub>
</td>
<td>1000</td>
<td>14</td>
<td>1.4</td>
</tr>
<tr>
<td>Ultrapure Water</td>
<td> </td>
<td>n/a</td>
<td>96.6</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td>n/a</td>
<td>100</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave before use.</div>
<div class="note">
<span class="note-title">Note:</span> A month’s storage at 4°C is assured.</div>
<table id="undtbl4">
<caption>Binding Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration (mM)</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH8.0)</td>
<td>1000</td>
<td>50</td>
<td>10</td>
</tr>
<tr>
<td>NaCl</td>
<td>1000</td>
<td>500</td>
<td>100</td>
</tr>
<tr>
<td>Imidazole</td>
<td>1000</td>
<td>40</td>
<td>8</td>
</tr>
<tr>
<td>Ultrapure Water</td>
<td>n/a</td>
<td>n/a</td>
<td>82</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>n/a</td>
<td>200</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave before use.</div>
<div class="note">
<span class="note-title">Note:</span> Two years storage at 4°C is assured.</div>
<table id="undtbl5">
<caption>Elution Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration (mM)</th>
<th>Final concentration (mM)</th>
<th>Volume (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH8.0)</td>
<td>1000</td>
<td>50</td>
<td>10</td>
</tr>
<tr>
<td>NaCl</td>
<td>1000</td>
<td>500</td>
<td>100</td>
</tr>
<tr>
<td>Imidazole</td>
<td>1000</td>
<td>400</td>
<td>80</td>
</tr>
<tr>
<td>Ultrapure Water</td>
<td>n/a</td>
<td>n/a</td>
<td>10</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>n/a</td>
<td>200</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Autoclave before use.</div>
<div class="note">
<span class="note-title">Note:</span> Two years storage at 4°C is assured.</div>
<table id="undtbl6">
<caption>50× NTP mixture</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration (mM)</th>
<th>50× Final concentration (mM)</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>ATP</td>
<td>500</td>
<td>100</td>
<td>20</td>
</tr>
<tr>
<td>GTP</td>
<td>500</td>
<td>100</td>
<td>20</td>
</tr>
<tr>
<td>CTP</td>
<td>500</td>
<td>65</td>
<td>13</td>
</tr>
<tr>
<td>UTP</td>
<td>500</td>
<td>65</td>
<td>13</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>n/a</td>
<td>34</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>100</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C or below. Two years storage is assured.</p>
<div class="note">
<span class="note-title">Note:</span> 1× NTP indicates a mixture of 2.0 mM ATP, 2.0 mM GTP, 1.3 mM UTP, and 1.3 mM CTP.</div>
<table id="undtbl7">
<caption>5× reaction mixture</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration</th>
<th>5× Final concentration</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes-KOH pH7.6</td>
<td>2000 mM</td>
<td>250 mM</td>
<td>12.5</td>
</tr>
<tr>
<td>GluK</td>
<td>4000 mM</td>
<td>500 mM</td>
<td>12.5</td>
</tr>
<tr>
<td>tRNA mixture</td>
<td>51840 μg/mL</td>
<td>990 μg/mL</td>
<td>1.9</td>
</tr>
<tr>
<td>FD (folinic acid)</td>
<td>33.9 mM</td>
<td>0.34 mM</td>
<td>1</td>
</tr>
<tr>
<td>Creatine phosphate</td>
<td>1000 mM</td>
<td>400 mM</td>
<td>40</td>
</tr>
<tr>
<td>20 amino acids mixture</td>
<td>20 mM each</td>
<td>3 mM each</td>
<td>15</td>
</tr>
<tr>
<td>Spermidine</td>
<td>500 mM</td>
<td>5 mM</td>
<td>1</td>
</tr>
<tr>
<td>50× NTP mixture</td>
<td>50×</td>
<td>5×</td>
<td>10</td>
</tr>
<tr>
<td>IPTG</td>
<td>1000 mM</td>
<td>5 mM</td>
<td>0.5</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>n/a</td>
<td>5.6</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>n/a</td>
<td>100</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at -20°C or below. Two years storage is assured.</p>
<div class="note">
<span class="note-title">Note:</span> 1 μL of 5× reaction mixture is used to prepare a 5 μL TX-TL system. Scale up or down the total stock amount for your purpose.</div>
<div class="note">
<span class="note-title">Note:</span> 1× reaction mixture indicates a mixture of 50 mM Hepes-KOH (pH7.6), 100 mM GluK, 198 μg/mL tRNA mixture, 0.068 mM FD, 80 mM Creatine phosphate, 0.6 mM 20 amino acids mixture, 1 mM Spermidine, 1× NTP mixture, and 1 mM IPTG.</div>
<div class="note">
<span class="note-title">Note:</span> 20 amino acids mixture is prepared by dissolving 20 amino acids in Ultrapure water.</div>
<table id="undtbl8">
<caption>TX-TL system solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration</th>
<th>Final concentration</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>5× reaction mixture</td>
<td>5×</td>
<td>1×</td>
<td>1</td>
</tr>
<tr>
<td>Cell extract</td>
<td>40 mg/mL</td>
<td>10 mg/mL</td>
<td>1.25</td>
</tr>
<tr>
<td>GluMg</td>
<td>200 mM</td>
<td>11.5 mM</td>
<td>0.3</td>
</tr>
<tr>
<td>mCherry</td>
<td>16 μM</td>
<td>4 μM</td>
<td>1.25</td>
</tr>
<tr>
<td>Creatine kinase</td>
<td>3000 μg/mL</td>
<td>300 μg/mL</td>
<td>0.5</td>
</tr>
<tr>
<td>DNA (pOR2OR1-sfGFP)</td>
<td>200 nM</td>
<td>20 nM</td>
<td>0.5</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>n/a</td>
<td>0.2</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>5 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare before use.</p>
<div class="note">
<span class="note-title">Note:</span> In this protocol, the final concentration of magnesium in the TX-TL system is set as 15 mM in total. Because S30 cell extract contains 14 mM magnesium, 3.5 mM magnesium in the TX-TL system is derived from the S30 cell extract. GluMg added is to compensate for the rest.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> 5× reaction mixture, Cell extract, GluMg and Creatine kinase can be replaced with PURE system.</div>
<table id="undtbl9">
<caption>External buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration</th>
<th>Final concentration</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>5× reaction mixture</td>
<td>5×</td>
<td>1 x</td>
<td>20</td>
</tr>
<tr>
<td>S30B Buffer</td>
<td>1×</td>
<td>0.25×</td>
<td>25</td>
</tr>
<tr>
<td>GluMg</td>
<td>200 mM</td>
<td>11.5 mM</td>
<td>5.8</td>
</tr>
<tr>
<td>Hepes-KOH (pH 7.6)</td>
<td>20 mM</td>
<td>5 mM</td>
<td>25</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>n/a</td>
<td>24.3</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>100</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C or below. Two years storage is assured.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The osmotic pressure of the External buffer should be the same as TX-TL system solution. S30B Buffer is added to compensate for the small molecules that cell extract contains in the TX-TL system solution. Hepes-KOH (pH 7.6) is the replacement for mCherry.</div>
<div class="note">
<span class="note-title">Note:</span> 20 μL of the External buffer is used to prepare one liposome sample. Scale up or down the total stock amount for your purpose.</div>
<table id="undtbl10">
<caption>2× External buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration</th>
<th>Final concentration</th>
<th>Volume (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>5× reaction mixture</td>
<td>5×</td>
<td>2×</td>
<td>40</td>
</tr>
<tr>
<td>S30B Buffer</td>
<td>1×</td>
<td>0.5×</td>
<td>50</td>
</tr>
<tr>
<td>GluMg</td>
<td>1000 mM</td>
<td>23 mM</td>
<td>2.3</td>
</tr>
<tr>
<td>Hepes-KOH (pH 7.6)</td>
<td>200 mM</td>
<td>10 mM</td>
<td>5</td>
</tr>
<tr>
<td>Ultrapure water</td>
<td>n/a</td>
<td>n/a</td>
<td>2.7</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>100</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C or below. Two years storage is assured.</p>
<div class="note">
<span class="note-title">Note:</span> 20 μL of 2× External buffer is used for one hypertonic treatment. Scale up or down the total stock amount for your purpose.</div>
<h3 id="sec2.2">Silicon spacer</h3>
<p>To make the silicon spacer for microscopic observation, cut a silicone rubber sheet (0.5 or 1 mm thick) into 1 cm squares and make a hole in it by using a hole punch.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of liposomes encapsulating a diluted <i>E. coli</i> TX-TL system</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 90 min</div>
<p>In this step, liposomes are prepared by a modified droplet-transfer method (<a href="#bib3">Fujiwara and Yanagisawa, 2014</a>)</p>
<ol>
<li>Preparation of lipid solution
<ol type="a">
<li>Dry 40 μL of 25 mg/mL POPC (dissolved in chloroform and stored at −20°C in a round glass microtube) under Ar or N<sub>2</sub> gas flow. We typically perform this process without using a vacuum desiccator.</li>
<li>Add 500 μL mineral oil to the dried POPC.</li>
<li>Incubate the lipid solution at 80°C for 1 h (e.g., in an Aluminum block incubator). Do not cap the tube in this process.</li>
<li>Cool the tube to room temperature (20°C–30°C).</li>
<li>After capping the tube by an attached screw cap, vortex the lipid solution for 1 min.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The lipid solution can be stored at room temperature (20°C–30°C) for at least more than several hours with the tube wrapped by aluminum foil. Vortex again before the emulsification (step 2).</div>
<ol start="2">
<li>Emulsification
<ol type="a">
<li>Prepare the TX-TL system solution on ice.</li>
<li>Add 20 μL of ultrapure water to 5 μL of the TX-TL system to generate a 1/5-fold dilution.</li>
<li>Add 50 μL of the lipid solution into a 0.6 mL tube and pipette 1 μL of the diluted TX-TL system to the lipid solution.
<div class="note">
<span class="note-title">Note:</span> From this step, prepare samples at room temperature (20°C–30°C) .</div>
</li>
<li>Tap the tube about 10 times for emulsification.
<div class="note">
<span class="note-title">Note:</span> The size range of liposomes depends on how strong and how many times you tap the tube in this step. We recommend preparing liposomes larger than 5 μm (diameter) for microscopic observation. You may need to optimize this step in order to get an appropriate size range.</div>
<div class="note">
<span class="note-title">Note:</span> Keep the emulsified solution at room temperature (20°C–30°C) until used in step 3d. Tapping the tube might be required to form emulsions again just before step 3d.</div>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Preparation of liposomes
<ol type="a">
<li>Prepare the External buffer.</li>
<li>Add 80 μL of ultrapure water to 20 μL of the External buffer in a 1.7 mL tube to generate a 1/5-fold dilution (in order to balance the osmotic pressure between the inside and outside of the liposomes).</li>
<li>Carefully add 50 μL of the lipid solution (prepared in step 1) on top of the diluted External buffer with pipette (<a href="#fig2">Figure 2</a>).</li>
<li>Add 50 μL of the emulsified solution (prepared in step 2) onto the oil phase with the lipids (<a href="#fig2">Figure 2</a>).
<div class="critical">
<span class="critical-title">Critical:</span> Be careful not to mix the emulsified solution with the phase of diluted External buffer (water phase).</div>
</li>
<li>Centrifuge at 20,000 × <i>g</i> for 5 min at 4°C and remove the oil phase.
<div class="note">
<span class="note-title">Note:</span> Remove as much of the oil phase as possible. Contaminant oil forms a liposome-like sphere structure in the solution (oil in water emulsion).</div>
</li>
<li>Recover liposomes by gently pipetting the bottom of the tube and transfer the liposome solution into a new 0.6 mL tube.</li>
</ol>
</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1241-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Pipetting emulsion onto the oil phase</div>
<p>(A) Lipids in oil is added on top of the diluted External buffer.</p>
<p>(B) Emulsion is pipetted onto the oil phase.</p>
<p>(C) A resultant solution with phases. The solution is centrifuged to form liposomes at the next step.</p>
</figcaption>
</figure>
<h3 id="sec3.2">Activation of the TX-TL system inside liposomes by hypertonic treatment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–16 h. (Depending on reaction time.)</div>
<p>In this step, the prepared liposomes are treated by osmolytes to concentrate molecules within liposomes, and the activation of the TX-TL system is confirmed by measuring sfGFP fluorescence. A fluorescent microscope is used to obtain the fluorescent images. We use sfGFP expression as a demonstration of this protocol. Other genes should be expressed by the protocol. We have confirmed the expression of α-hemolysin by the protocol (<a href="#bib1">Akui et al., 2021</a>).</p>
<ol start="4">
<li>For the experimental condition, mix 20 μL of 2× External buffer with 20 μL of liposome solution in a 1.7 mL tube by gently pipetting. As a negative control (without hypertonic treatment), mix 20 μL 1/5-diluted External buffer with 20 μL of liposome solution in another 1.7 mL tube by gently pipetting.</li>
<li>Incubate the sample at 25°C for an appropriate time (1–16 h).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The activation of the TX-TL system starts as soon as you add a 2× External buffer to the liposome solution. It may take 30 min to an hour for the sfGFP to be expressed to a detectable level. (In the case that T7 promoter is used for DNA template, the speed of protein synthesis and the final amount will be higher than when the endogenous promoter is used.)</div>
<ol start="6">
<li>Pipette the reacted sample (hypertonic treatment) into the central chamber made of silicon spacer on a slide glass (<a href="#fig3">Figure 3</a>). Do the same for the negative control on a separate glass slide.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1241-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. A unit spaced with silicon spacer for microscopic observation of liposomes</div>
<p>(A) A silicon spacer spotted on the cover glass.</p>
<p>(B) Spotting liposome solution.</p>
<p>(C) The silicon spacer chamber covered with another cover glass.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To prevent the reacted sample from drying, cover the sample with another slide glass (<a href="#fig3">Figure 3</a>), especially if the pipetted sample volume is a few microliters or the observation takes a long time.</div>
<ol start="7">
<li>Observe the fluorescence of mCherry (indicator) and sfGFP (synthesized protein), respectively, by fluorescent microscope.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Tens to hundreds of liposomes are collected by the protocol, and at least 72% liposomes show synthesis of sfGFP in cytosols (see also <a href="#bib1">Akui et al, 2021</a>). Typical data obtained by the protocol is shown in <a href="#fig4">Figure 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1241-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. A representative result obtained by this protocol (16 h incubation)</div>
<p>The thresholds of Indicator (mCherry) images are differently adjusted among the two conditions to clearly show the position of liposomes in both cases. Thus, the fluorescent levels of indicator (mCherry) do not correlate with the actual mCherry concentrations: mCherry is concentrated by the hypertonic treatment.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The TX-TL activity inside liposomes can’t be switched on by more than 5-fold hypertonic treatment, although it was confirmed that macromolecules inside liposomes can be concentrated up to 30 times by hypertonic treatment (<a href="#bib1">Akui et al, 2021</a>). (The osmotic pressure ratio of the outer solution to the inner solution of liposomes determines the concentration factor by hypertonic treatment. if the ratio is 5, the concentration of macromolecules inside liposomes become 5-times higher by the treatment.)</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>No or low TX-TL activity of prepared S30 cell extract. (<a href="#before-you-begin">Before you begin</a>, <a href="#sec1.1">prepare S30 cell extract for cell-free TX-TL system</a>)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>(i) Check the timing of the cell collection in step 3b. The activity of the TX-TL system will significantly decrease if stationary phase cells were collected. Optimal OD<sub>600</sub> for cultivation may depend on the apparatus to measure OD<sub>600</sub>. Typically, less than half of the maximal OD<sub>600</sub> is the best OD<sub>600</sub> for collecting cells. (ii) Cell lysis and the centrifugation to collect the extract after cell lysis can be a critical step for activity. It takes at least 30 min to collect sufficient volume of the supernatant (0.3–0.7 mL per 1 mL cell suspension) by centrifugation in step 6. If cell debris were completely pelleted within a few min centrifugation, either the cell disruption failed (In this case, cell extract seems to be clear) or the timing of cell harvest was too late (In this case, cell extract shows yellow color). A small volume of supernatant (less than 0.3 mL per 1 mL cell suspension) suggests extensive cell lysis. In this case, the timing of cell harvest was too early, or lysozyme treatment was too strong. In the latter case, decrease amounts of lysozyme or shorten the reaction time. (iii) Otherwise, try other preparation methods, which use French press, sonication, or other apparatus for cell disruption. (iv) Cell extract can be replaced with a reconstituted transcription-translation system (PURE system) in the original study (<a href="#bib1">Akui et al, 2021</a>).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Poor emulsification. (<a href="#step-by-step-method-details">Step-by-step method details</a>, <a href="#sec3.1">preparation of liposomes encapsulating a diluted E. coli TX-TL system</a>, step 2 Emulsification). Poor emulsification will lead to poor liposome formation. In some cases, emulsion disappears after several minutes, with water and oil phases separating.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>(i) Minimize the time between emulsification and liposome preparation (i.e., these two steps should be performed within about 5 min of each other). Tap the tube and form emulsions again just before adding to the next step. If the time is not the cause, it is mainly attributed to the condition of lipids solution. Primarily, (ii) Vortex the lipid solution (prepared in <a href="#step-by-step-method-details">step-by-step method details</a>, 1. Preparation of lipid solution) just before pipetting the diluted TX-TL system for emulsification. (iii) Try an alternative method for preparing lipids solution, such as sonication in a hot water bath (<a href="#bib4">Kohyama et al., 2019</a>, (<a href="#bib7">Kohyama et al., 2020</a>)). Finally, (iv) Water contamination in lipids should be concerned: Any water contamination should be evaporated in a vacuum desiccator for several hours after drying chloroform.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Poor liposome yield. (<a href="#step-by-step-method-details">Step-by-step method details</a>, <a href="#sec3.1">preparation of liposomes encapsulating a diluted E. coli TX-TL system</a>, step 3 Preparation of liposomes)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Primarily, check if emulsions are successfully prepared by using a microscope. (step 2 Emulsification). If emulsions are not formed, refer to Problem 2 &amp; Potential solution. If emulsions are successfully formed, (i) check if the osmotic pressure between the inner solution (TX-TL system) and the outer solution (External Buffer) are balanced in step 3 Preparation of liposomes. Refer to the CRITICAL in <a href="#materials-and-equipment">materials and equipment</a>, External buffer. (ii) Alternative solutions may be found in the condition of lipid solution. Use the lipid solution prepared on the same day. If liquid paraffin is used instead of mineral oil in step 1 Preparation of lipid solutions, we recommend to use liquid paraffin that is just after opened or stored in the freezer after it was opened.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>No liposome was found in the microscopic field. (<a href="#step-by-step-method-details">Step-by-step method details</a>, <a href="#sec3.2">activation of the TX-TL system inside liposomes by hypertonic treatment</a>, step 7)</p>
<h3 id="sec6.8">Potential solution</h3>
<p>(i) Increase the concentration of fluorescent proteins or molecules, the indicator of liposome position: mCherry in this protocol. (ii) Refer to Problem 3 &amp; Potential solution.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Protein expression could not be detected in liposomes after hypertonic treatment (e.g., the fluorescence intensity of GPF in the experimental condition was not significantly stronger than that in the negative control). (<a href="#step-by-step-method-details">Step-by-step method details</a>, <a href="#sec3.2">activation of the TX-TL system inside liposomes by hypertonic treatment</a>, step 7)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Primarily, check if the TX-TL system solution has the activity in a test tube. If TX-TL works in the case of the tube reaction, lipid membrane or semipermeability is the cause of the low TX-TL activity inside liposomes. If polar lipids or other anionic lipids are used, negative charges of lipids affect TX-TL activity. In this case, add 10 mg/mL or more BSA to the TX-TL system like in our previous study (<a href="#bib6">Yoshida et al., 2019</a>). If lipids with neutral charges are used, but the TX-TL activity is low, some essential small molecules for TX-TL may leak out from liposomes due to the semi-permeability of lipid membranes or the hypertonic treatment. In this case, please make sure that the External buffer contains the essential small molecules for TX-TL reaction.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Akui, T., Fujiwara, K., Sato, G., Takinoue, M., Nomura, S.M., and Doi, N. (2021). System concentration shift as a regulator of transcription-translation system within liposomes. iScience <i>24</i>, 102859. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00709-7/sref1">View at publisher</a></p>
<p id="bib2">Fujiwara, K. and Doi, N. (2016). Biochemical preparation of cell extract for cell-free protein synthesis without physical disruption. PLoS One <i>11</i>, e0154614. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00709-7/sref2">View at publisher</a></p>
<p id="bib3">Fujiwara, K. and Yanagisawa, M. (2014). Generation of giant unilamellar liposomes containing biomacromolecules at physiological intracellular concentrations using hypertonic conditions. ACS Synth. Biol. <i>3</i>, 870-874. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00709-7/sref3">View at publisher</a></p>
<p id="bib7">Kohyama, S., Fujiwara, K., Yoshinaga, N., and Doi, N. (2020). Conformational equilibrium of MinE regulates the allowable concentration ranges of a protein wave for cell division. Nanoscale <i>12</i>, 11960-11970. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00709-7/optQpz5BXhTsM">View at publisher</a></p>
<p id="bib4">Kohyama, S., Yoshinaga, N., Yanagisawa, M., Fujiwara, K., and Doi, N. (2019). Cell-sized confinement controls generation and stability of a protein wave for spatiotemporal regulation in cells. eLife <i>8</i>, e44591. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00709-7/sref4">View at publisher</a></p>
<p id="bib5">Shimizu, Y., Inoue, A., Tomari, Y., Suzuki, T., Yokogawa, T., Nishikawa, K., and Ueda, T. (2001). Cell-free translation reconstituted with purified components. Nat. Biotechnol. <i>19</i>, 751-755. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00709-7/sref5">View at publisher</a></p>
<p id="bib6">Yoshida, A., Kohyama, S., Fujiwara, K., Nishikawa, S., and Doi, N. (2019). Regulation of spatiotemporal patterning in artificial cells by a defined protein expression system. Chem. Sci. <i>10</i>, 11064-11072. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00709-7/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>[Kei Fujiwara] (fujiwara@bio.keio.ac.jp).</p>
<h4>Materials availability</h4>
<p>Plasmids constructed in this study are available from the corresponding author on request.</p>
<h4>Data and code availability</h4>
<p>This paper does not report original code. For questions regarding this study, please contact the corresponding author.</p>
<h3>Acknowledgments</h3>
<p>We thank Prof. Masahiro Takinoue (Tokyo Institute of Technology) and Prof. Shin-ichiro M. Nomura (Tohoku university) for collaborating on the original work. We thank the financial support by <a href="https://doi.org/10.13039/501100001691">JSPS KAKENHI</a> Grant Number JP15H00826 and JP17H05235 for KF.</p>
<h3>Author contributions</h3>
<p>Y.M., T.A., and K.F. established the protocol. T.A., N.D., and K.F. contributed to the original set-up of the experimental system. Y.M. and K.F. wrote the paper.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>