<article>
<section>
<h1 id="header">
<i>Ιn vivo</i> imaging of DNA-bound minichromosome maintenance complex in embryonic mouse cortex</h1>
<p><time datetime="2021-01-08">Published: January 8, 2021</time></p>
<p>Maria Mougkogianni,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup> Argyro Kalogeropoulou,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Nickolaos Nikiforos Giakoumakis,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Zoi Lygerou,<sup><a href="#aff3">3</a></sup> and Stavros Taraviras<sup><a href="#aff1">1</a>,<a href="#fn3">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Physiology, School of Medicine, University of Patras, Rio, Patras 26504, Greece</p>
<p id="aff2"><sup>2</sup>Advanced Digital Microscopy Facility, Institute for Research in Biomedicine-IRB Barcelona, 08028 Barcelona, Spain</p>
<p id="aff3"><sup>3</sup>Department of General Biology, School of Medicine, University of Patras, Rio, Patras 26504, Greece</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:m.mougkogianni@upnet.gr">m.mougkogianni@upnet.gr</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:taraviras@med.upatras.gr">taraviras@med.upatras.gr</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100234">10.1016/j.xpro.2020.100234</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The recruitment of the minichromosome maintenance complex (MCM) on DNA replication origins is a critical process for faithful genome duplication termed licensing. Aberrant licensing has been associated with cancer and, recently, with neurodevelopmental diseases. Investigating MCM loading in complicated tissues, such as brain, remains challenging. Here, we describe an optimized approach for the qualitative and quantitative analysis of DNA-bound MCMs in the developing mouse cortex through direct imaging, offering an innovative insight into the research of origin licensing <i>in vivo</i>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detailed protocol for the generation of sections from freshly isolated mouse brain</li>
<li>Removal of unbound soluble MCM complexes by a pre-extraction procedure</li>
<li>Classification in distinct cell cycle phases according to MCM and EdU staining</li>
<li>Application in a variety of tissues to investigate DNA replication licensing</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/354-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>DNA replication is a strictly organized procedure to ensure that the genetic information will be accurately transmitted into the ensuing generations. In eukaryotes, DNA replication initiates from multiple origins of replication which are dispersed across the genome. From late mitosis and during the G1 phase of the cell cycle, multi-protein complexes are formed on the origins in a process that is known as origin licensing (<a href="#bib2">Fragkos et al., 2015</a>; <a href="#bib12">Symeonidou et al., 2012</a>). Licensing is completed with the recruitment to the origins of the inactive form of the minichromosome maintenance (MCM) complex, which is a ring-shape heterohexamer (MCM2–7) with helicase activity. Upon G1-S transition and along the S phase, the MCM complexes become activated to catalyze the unwinding of DNA at replication forks. MCM complexes are gradually released from the DNA as the replication progresses and remain in the nucleus as soluble molecules until the next round of licensing.</p>
<p>MCM proteins are constitutively present in the cell nucleus as diffused molecules whereas a subset of the MCMs are loaded to DNA in a cell cycle-dependent manner (<a href="#bib11">Symeonidou et al., 2013</a>). Given their direct correlation and their compulsory presence in DNA replication, analysis of total MCMs frequently substitutes known proliferation markers, such as Ki-67 and PCNA, in confirmation of proliferation capacity (<a href="#bib4">Juríková et al., 2016</a>). On the other hand, analysis of DNA-bound MCMs is used in the study of the early events of DNA replication like origin licensing and initiation of replication.</p>
<p>Here, we present a method to identify the DNA-bound MCMs in brain slices derived from mouse embryos from embryonic day (E) 10.5 to E18.5. To distinguish the different cell cycle phases of the proliferating cells, the incorporation of a thymidine analog was examined in parallel with the presence of MCM proteins in the nucleus. A detailed protocol is provided here, covering all the steps from the isolation of the brain and the generation of thick brain slices to the quantification of MCM fluorescent intensity.</p>
<h3 id="sec1.1">Preparation of the low melting temperature agarose solution</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 30 min</div>
<p>For the embedding of 4 embryonic brains prepare 10 mL of low-temperature melting agarose (LMA) 3% (Troubleshooting 1).</p>
<ol>
<li>Prewarm 10 mL of PBS 1× in a flask on a heating magnetic stirrer at 70°C for 15 min.</li>
<li>Add 0.3 g agarose and stir with a magnetic bar at 70°C until the agarose melts and starts boiling. Cap the flask loosely to avoid evaporation.</li>
<li>Once agarose is completely dissolved and the solution is clear and smooth reduce heat at 45°C.</li>
<li>Place a thermometer in the flask to monitor the temperature and leave it stirring until tissue embedding.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is crucial to maintain the temperature of the LMA solution between 45°C and 50°C. Higher temperature will destroy the tissue during embedding while in lower temperature (&lt;36°C) LMA is solidified.</div>
<h3 id="sec1.2">Preparation of the cytoskeleton (CSK) buffer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 20 min</div>
<p>Prepare the CSK buffer fresh before use. CSK buffer is applied for the extraction of the cytoplasmic proteins (<a href="#bib3">Hua and Ferland, 2017</a>; <a href="#bib10">Sawasdichai et al., 2010</a>; <a href="#bib11">Symeonidou et al., 2013</a>). The sucrose that is included in the buffer preserves the internal cell structures.</p>
<ol start="5">
<li>Mix the following reagents to the final concentration:</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent (stock concentration)</th>
<th>Final concentration</th>
<th>Quantity for 1 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES (1 M)</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>Sucrose (1 M)</td>
<td>300 mM</td>
<td>300 μL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>100 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (0.5 M)</td>
<td>3 mM</td>
<td>6 μL</td>
</tr>
<tr>
<td>EGTA (0.5 M)</td>
<td>1 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>Triton (20%)</td>
<td>0.2%</td>
<td>10 μL</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>1 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>BSA</td>
<td>2%</td>
<td>20 mg</td>
</tr>
<tr>
<td>PMSF (0.1 M)</td>
<td>1 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>Na<sub>3</sub>VO<sub>4</sub> (1 M)</td>
<td>1 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>NaF (1 M)</td>
<td>10 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>Protease inhibitor tab (25×)</td>
<td>1×</td>
<td>40 μL</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>n/a</td>
<td>590 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<ol start="6">
<li>Vortex gently and maintain the buffer at 4°C until use.</li>
</ol>
<h3 id="sec1.3">EdU labeling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 70 min</div>
<p>Labeling with a thymidine analog will permit the cells undergo DNA replication detection of proliferating neural stem cells and the identification of the different stages of S phase (early – mid – late).</p>
<ol start="7">
<li>Prepare an EdU solution at 1 mg/mL concentration.</li>
<li>Perform an intraperitoneal injection of 100 μL EdU solution to the pregnant mice 1 h before the tissue dissection procedure begins.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The same exposure time to EdU (1 h) can be applied in every developmental stage that is examined given that, the length of the S phase is relatively constant during brain development (<a href="#bib13">Takahashi, 1995</a>).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>MCM2 XP rabbit mAb #3619</td>
<td>Cell Signaling Technology</td>
<td>Cat#D7G11; RRID:AB_2142137</td>
</tr>
<tr>
<td>Goat anti-rabbit IgG (H+L) highly cross-adsorbed secondary antibody, Alexa Fluor 568</td>
<td>Life Technologies</td>
<td>Cat#A11036; RRID:AB_10563566</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>EdU Cell Proliferation Kit for high-throughput screening</td>
<td>BaseClick</td>
<td>Cat#BCK-HTS488</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>SeaPlaque agarose (low melting temperature agarose)</td>
<td>Lonza</td>
<td>Cat#50101</td>
</tr>
<tr>
<td>N-(2-Hydroxyethyl)piperazine-N′-2-ethane sulfonic acid (HEPES)</td>
<td>Serva</td>
<td>Cat#25245</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma-Aldrich</td>
<td>Cat#S7903</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>Merck</td>
<td>Cat#106404</td>
</tr>
<tr>
<td>Magnesium chloride hexahydrate (MgCl<sub>2</sub>·6H<sub>2</sub>O)</td>
<td>Merck</td>
<td>Cat#105833</td>
</tr>
<tr>
<td>Ethylene glycol-bis(2-aminoethylether)-N,N,N′,N′-tetraacetic acid (EGTA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#E3889</td>
</tr>
<tr>
<td>Triton X-100 molecular biology grade</td>
<td>Applichem</td>
<td>Cat#A4975</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>Applichem</td>
<td>Cat#A1101</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#A4503</td>
</tr>
<tr>
<td>PMSF</td>
<td>Sigma-Aldrich</td>
<td>Cat# P7626</td>
</tr>
<tr>
<td>Sodium orthovanadate (Na<sub>3</sub>VO<sub>4</sub>)</td>
<td>Sigma-Aldrich</td>
<td>Cat#S6508</td>
</tr>
<tr>
<td>Sodium fluoride (NaF)</td>
<td>Sigma-Aldrich</td>
<td>Cat#201154</td>
</tr>
<tr>
<td>cOmplete, EDTA-free Protease Inhibitor Cocktail (Protease inhibitor tab)</td>
<td>Roche</td>
<td>Cat#11873580001</td>
</tr>
<tr>
<td>Paraformaldehyde (PFA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#P6148</td>
</tr>
<tr>
<td>Gibco fetal bovine serum (FBS)</td>
<td>Gibco</td>
<td>Cat#10270106</td>
</tr>
<tr>
<td>Mowiol 4-88 reagent</td>
<td>Calbiochem</td>
<td>Cat#475904</td>
</tr>
<tr>
<td>2-(4-Amidinophenyl)-6-indolecarbamidine dihydrochloride (DAPI)</td>
<td>Sigma-Aldrich</td>
<td>Cat#D9542</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Wild-type Parkes mouse</td>
<td>NA</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Leica VT1000 S Vibrating blade microtome</td>
<td>Leica</td>
<td>NA</td>
</tr>
<tr>
<td>Confocal TCS SP5 equipped with a Leica fluorescence microscope (DMI600B)</td>
<td>Leica</td>
<td>NA</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> DTT, PFA, and PMSF are harmful in case of swallowing and cause skin and eye irritation. Triton X-100 causes corrosion and environmental hazard. It is advised to always work in a fume hood and wear the appropriate protective equipment (gloves, protective goggles, and lab coat).</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Tissue dissection and embedding in LMA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 75 min</div>
<p>The subsequent procedure describes the embedding of the dissected brain tissues in low melting agarose in order to facilitate the generation of robust tissue sections.</p>
<ol>
<li>Expose pregnant mice to lethal overdose of isoflurane and euthanize by cervical dislocation.</li>
<li>Section the abdomen to reveal the uterus. Remove the uterine horns and place them in cold PBS 1×.</li>
<li>Carefully remove the embryos from their yolk sac and isolate their heads. For embryos at E10.5-E12.5 it is advised to dissect the heads while for embryos at E14.5 or later stages it is suggested to dissect out the brains.</li>
<li>Clean the isolated tissues by performing 3 PBS 1× washes.</li>
<li>Put a 48-well plate on ice and add 500 μL of LMA 3% while still in liquid form (45°C). Wait 1 min until the temperature becomes tolerable.</li>
<li>Remove the excess liquid from each tissue using a perforated spoon and a piece of absorbent paper(∼40°C).</li>
<li>Place the tissue in the well and gently swirl with a pipette tip until the tissue is totally covered with LMA.</li>
<li>Place the tissue at the appropriate orientation carefully and support with the pipette tip until LMA starts to solidify.</li>
<li>Leave the 48-well plate on ice until LMA is completely solid and afterwards place it at 4°C for 30 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important for the tissue to be totally covered with LMA in order not to touch the bottom of the well plate and not to be exposed on the top.</div>
<div class="note">
<span class="note-title">Note:</span> The orientation of the tissue inside the well depends on the desired sectioning plane (coronal, sagittal, or transverse). For generation of coronal sections place the brain cerebellum vertically pointing the bottom of the well and the olfactory bulbs facing upwards.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Place the remaining quantity of LMA that does not contain any tissue residue in a tube and when solid store at 4°C for up to 1 week.</div>
<h3 id="sec2.2">Generation of brain sections</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 15 min, depending on the tissue size</div>
<p>The following steps describe the generation of thick sections from freshly isolated mouse brain by using a vibrating microtome and highlight the appropriate settings depending on the developmental stage that is examined.</p>
<ol start="10">
<li>Remove the embedded tissue from the 48-well plate by gently slide a micro spatula toward the well circumference until the embedded tissue springs out.</li>
<li>Remove the excess amount of LMA with a scalpel and create a shape of trapezoid so that its bigger base touches the vibratome plate and its smaller base faces toward the vibratome blade. To generate coronal sections at the level of the cortex, the rostral part of the brain should face toward the blade</li>
<li>Stabilize the embedded tissue on the vibratome platform with a thin layer of instant glue and ensure that the sample will not be displaced by the vibratome blade. Wait 2 min until the glue is dried.</li>
<li>Place the vibratome plate in the tank with the appropriate screwdrivers and fill with cold PBS 1× to fully cover the sample.</li>
<li>Cut serial sections of 250 μm thickness. Adjust the vibratome settings according to the developmental stage studied. For later embryonic stages (E14.5 to E18.5) adjust the speed at 0,225 mm/s and frequency at 50 Hz while for earlier developmental stages (E10.5 to E12.5) and smaller tissues it is preferable to modify the settings of the vibratome (speed: 0,125 mm/s and frequency 70 Hz) to avoid destroying the tissue. In general, the higher the speed and the lower the frequency, the more suitable it is for vulnerable tissues.</li>
<li>Prepare a 24-well plate with cold PBS 1× and keep it on ice.</li>
<li>Collect consecutive sections of the brain with thin paint brushes and place them in separate wells of the 24-well plate on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Given that the tissue at this stage is unfixed, the sections are fragile. Thus, it is advised to gently handle the sections by using the appropriate brushes. In case it is not possible for the sections to be removed of the brush try to slowly move the brush inside the well containing PBS 1× or pour additional PBS 1× over the brush. If the section tangles within the brush hairs, it becomes difficult to come out completely intact.</div>
<h3 id="sec2.3">Pre-extraction and fixation of brain sections</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>The pre-extraction procedure aims to remove the freely diffusing proteins while the DNA-bound proteins remain intact within the cells. The following steps are performed in floating sections. It is important that the sections remain moist during the entire protocol to maintain tissue integrity and in order to avoid auto-fluorescence effect during subsequent imaging.</p>
<ol start="17">
<li>Select the sections that include the dorsal cortex. To get representative results, it is preferable to select two sections per sample in order to cover frontal and caudal cortical regions. For earlier embryonic stages one section is sufficient.</li>
<li>To remove the cytoplasmic soluble proteins, incubate the sections in the freshly prepared CSK buffer for 20 min on ice. Avoid shaking of the sections at this stage since the tissue is very fragile before fixation (Troubleshooting 2).</li>
<li>Wash once with cold PBS 1× on ice.</li>
<li>Fix the sections with PFA 4% for 30 min at 20°C–23°C.</li>
<li>Wash once with cold PBS 1× on ice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep 1–2 sections as control samples by skipping the incubation with the CSK buffer. Control sections are immediately fixed with PFA 4%.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Sections can be stored in 1 mM NaN<sub>3</sub> solution in PBS at 4°C for 1 day in case that it is not possible to perform the following procedure at the same day.</div>
<h3 id="sec2.4">Immunofluorescent staining and EdU detection (day 1)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> at least 23 h</div>
<p>This process elaborates on the details of immunofluorescence, detection of thymidine analog, and mounting to the previously generated sections. Perform the staining in multiwell plates by removing and filling with the appropriate solutions the wells that contain the sections. The use of 24-well plates is recommended. Avoid transferring the sections within wells to maintain their morphology.</p>
<ol start="22">
<li>Wash the floating sections with PBT (PBS-Triton 0.1%) - BSA 1% three times for at least 30 min each. The first two washes should be performed at 4°C while the last one at 20°C–23°C.</li>
<li>Incubate the sections in PBS-Triton 0.5% for 20 min at 20°C–23°C.</li>
<li>Wash with PBT-BSA 1% three times at 4°C for at least 30 min each.</li>
<li>Block the sections in blocking solution (1% FBS in PBT) for at least 1 h at 20°C–23°C.</li>
<li>Wash with PBT-BSA 1% for at least 30 min at 4°C.</li>
<li>Perform the step of EdU staining by incubating the sections with the EdU detection solution (reaction buffer 10×, catalyst solution, dye azide 10 mM, buffer additive 10×) at 20°C–23°C for 30 min. From this point the sections should be protected from light.</li>
<li>Wash with PBT three times at 4°C for at least 30 min each.</li>
<li>Incubate the sections with the primary antibody against MCM2 (dilution 1:500 in blocking solution) for16 h at 4°C.</li>
</ol>
<h3 id="sec2.5">Immunofluorescent staining and EdU detection (day 2)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> at least 22 h</div>
<ol start="30">
<li>Remove the primary antibody solution.</li>
<li>Wash the sections with PBT at 4°C three times for at least 2 h each.</li>
<li>Incubate with fluorescent secondary antibody (dilution 1:1,000 in blocking solution) for16 h at 4°C.</li>
</ol>
<h3 id="sec2.6">Immunofluorescent staining and EdU detection (day 3)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5 h</div>
<ol start="33">
<li>Remove the secondary antibody solution.</li>
<li>Wash the sections with PBT at 4°C three times for at least 30 min each.</li>
<li>Perform nuclear DNA staining with DAPI (dilution 1:1,500 in PBS 1×) for 7 min at 20°C–23°C.</li>
<li>Wash the sections with PBT at 4°C three times for at least 30 min each.</li>
<li>Add a drop of PBS 1× on a microscope slide and transfer the sections by using thin paint brushes and sharp forceps.</li>
<li>Remove the excess amount of PBS with a piece of absorbent paper.</li>
<li>Add Mowiol 4-88 solution on the top of the section until to fully covered (60–100 μL per section) and gently drop the coverslip.</li>
<li>Keep the slides horizontally at 20°C–23°C until the Mowiol is polymerized. For long-term storage keep them at 4°C.</li>
</ol>
<h3 id="sec2.7">Imaging and analysis of DNA-bound MCM complexes</h3>
<ol start="41">
<li>For the imaging of the mounted brain sections use a confocal system equipped with a fluorescent microscope. Use at least a 63× high NA oil immersion lens to focus on the dorsal cortex. To analyze the MCM complexes, acquire images with an electronic zoom of 3 or higher with at least a 1,024 × 1,024 pixels resolution.</li>
<li>Before the analysis, it is important to confirm that the soluble proteins have been sufficiently removed while the DNA-bound proteins are intact on the tissue. In the section incubated with CSK the cells that are in S phase (distinguished as EdU+) exhibit distinct patterns of nuclear MCM2 staining, which represent the recruitment of MCM complexes in the origins of DNA replication. In contrast, the cells in the section incubated without CSK present a uniform nuclear staining for MCM2, corresponding to both bound and soluble fractions of MCM proteins. A representative image of confocal acquisition of brain sections incubated or not in CSK buffer is shown in <a href="#fig1">Figure 1</a>.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/354-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. DNA-bound MCMs in the neural stem cells of the mouse dorsal cortex</div>
<p>Representative images of mouse embryos cortexes at E14.5 with (+CSK) or without pre-extraction (-CSK) after immunofluorescent staining for MCM2 and EdU. DNA was stained with DAPI (blue). Scale bars, 10 μm.</p>
</figcaption>
</figure>
</li>
<li>To analyze multiple cells at the same time, acquire serial z-stacks with a step size of 0.7 μm to a total z-volume 7.608 μm and pinhole set to 1 Airy unit. The nucleus volume of a mammalian cell is between 600–1,500 μm<sup>3</sup>, and therefore images of higher volume will include more than two layers of cells, making the analysis of single cells impossible.</li>
<li>The intensity of MCM staining in CSK treated sections is representative of the DNA-bound MCM complexes. Fluorescent intensity per nucleus is quantified with the open-source platform ImageJ.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Intensities measurements are performed in sum slices projection. If necessary, use the subtract background plugin of ImageJ with a rolling ball radius equal to the average size of the nuclei to remove the intensities of the background due to unspecific staining.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The detailed protocol that is described here allows the analysis of the DNA-bound MCM complexes in intact brain slices of mouse embryos. In <a href="#fig2">Figure 2</a>, we show a representative image of a brain section derived from E10.5 embryos, which has been subjected to pre-extraction followed by immunofluorescent staining for MCM2 and EdU according to protocol. Note that the structure and the cytoarchitecture of the cortex are well preserved allowing the detailed analysis of the proliferating cells that reside in this area. A similar analysis can be applied in different developmental stages and in different areas of the embryonic brain according to the cell population of interest.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/354-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Analysis of the DNA-bound MCM complexes in the neural stem cells residing in the cortex</div>
<p>Brain slices in the level of the cortex were generated from mouse embryos at E10.5. The embryos have been exposed to EdU 1 h before the sacrifice. The slices were subjected to pre-extraction followed by immunofluorescence staining for EdU (green) and MCM2 (red). DNA was stained with DAPI. The dotted lines restrict the area of the dorsal cortex. Scale bar, 10 μm. LV, lateral ventricle.</p>
</figcaption>
</figure>
<p>The presented protocol permits the classification of the proliferating cells to distinct phases of the cell cycle according to the MCM2 and EdU staining, as it is indicated in <a href="#fig3">Figure 3</a>. MCM complexes are bound to origins of replication during late mitosis, G1, and S phase. Thus, after pre-extraction, MCM2 positive cells represent the population of cells that belong in G1 or S phases. The incorporation of the thymidine analog was used to further distinguish between these two phases and thus, the cells that are in G1 are identified as MCM2+EdU- whereas, MCM2+EdU+ cells represent the S phase. Further analysis, results in accurate identification of different stages of the S phase based on MCM2 and EdU patterns (<a href="#bib1">Aparicio et al., 2012</a>). Consequently, pan-nuclear staining for both EdU and MCM2 indicates early S phase, dotted signal for both markers characterizes mid S phase whereas intensely dotted EdU signal concurrently with slight or no MCM2 signal is typical of late S phase. Once the different cell cycle phases are identified, the fluorescent intensity of MCM2 can be quantified. The MCM2 intensity of G1 cells (MCM2+EdU-) represents the licensed origins.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/354-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative results of cell phase identification in cortical neural progenitors of E10.5 mouse embryos</div>
<p>Cells were classified in cell cycle phases according to EdU and MCM2 patterns. Cells positive for MCM2 (red) and negative for EdU (green) belong in G1 phase while double positive cells belong in S phase. Among the latter, cells with pan-nuclear signal of EdU and MCM2 represent early S phase, cells with dotted signal of EdU and more dotted signal of MCM2 represent mid S phase, and cells with intensely dotted EdU signal and slight or none MCM2 signal represent late S phase or early G2 phases. Nuclei were stained with DAPI (blue). Scale bars, 10 μm.</p>
</figcaption>
</figure>
<p>DNA replication licensing has a critical role in the maintenance of genome integrity by ensuring the sufficient duplication of the genome within the cell cycle Several methods have been developed in order to quantify the amount of the MCM complexes that are loaded to DNA, however, these methods are usually applied in synchronized cells or they require high number of cells or the tagging of MCM molecules (<a href="#bib6">Moreno et al., 2016</a>; <a href="#bib7">Morino et al., 2014</a>; <a href="#bib8">Nishitani et al., 2014</a>; <a href="#bib11">Symeonidou et al., 2013</a>). All these steps are difficultly applied in freshly isolated tissue and thus the existing protocols are not suitable for the in vivo analysis of DNA-bound MCMs.</p>
<p>Here, we describe a detailed protocol that permits the imaging of DNA-bound MCM complexes in freshly isolated mouse embryonic brains. Our protocol is based on the removal of the cytoplasmic fraction of the MCM proteins and the identification of the bound fraction through immunofluorescence. Microscopy analysis of the tissues allows the qualitative analysis of origins licensing by examining the pattern of the MCM staining within the nucleus as well as the quantification of bound MCMs through the fluorescence intensity. We believe that with the appropriate modifications, our protocol can be applied in different tissues offering a new perspective in the study of DNA replication licensing.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Perturbations in origin licensing result in inadequate replication leading to genomic instability, which is associated with malignant transformation and cancer as well as with several developmental syndromes (<a href="#bib5">Kalogeropoulou et al., 2019</a>; <a href="#bib9">Petropoulos et al., 2019</a>). The complex of the MCM proteins comprise one of the key factors that mediate origin licensing and therefore the study of these proteins and their presence in DNA have proven to be a useful tool in the research of replication licensing. Here, we describe a detailed protocol that permits the imaging of DNA-bound MCM complexes in freshly isolated mouse embryonic brains. Our protocol is based on the removal of the cytoplasmic fraction of the MCM proteins and the identification of the bound fraction through immunofluorescence. Qualitative analysis of origins licensing can be performed by examining the pattern of the MCM staining in respect to phase of the cell cycle. Quantification of the DNA-bound MCM complexes for each phase of the cell cycle can be achieved by the analysis of the fluorescent intensity for the MCM staining.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>How to ensure that the tissue does not separate from the LMA coverage during sectioning (step 7)?</p>
<h3 id="sec5.2">Potential solution</h3>
<p>It is crucial to remove all the excess liquid from the tissue before being embedded in LMA so that the tissue remains stable in its block after the LMA becomes solid. In case the problem persists, use higher concentration of LMA solution (e.g., 4% w/v).</p>
<h3 id="sec5.3">Problem 2</h3>
<p>How to confirm the efficacy of pre-extraction procedure (step 18)?</p>
<h3 id="sec5.4">Potential solution</h3>
<p>It is important to adjust the duration of sections’ incubation in CSK buffer and PFA solution, since these steps may need several modifications depending on the thickness of the sections and the origin of the tissue. Thus, we suggest to compare non pre-extracted and CSK treated sections in order to identify that the soluble proteins have been efficiently removed.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aparicio, T., Megías, D., and Méndez, J. (2012). Visualization of the MCM DNA helicase at replication factories before the onset of DNA synthesis. Chromosoma <i>121</i>, 499-507.</p>
<p id="bib2">Fragkos, M., Ganier, O., Coulombe, P., and Méchali, M. (2015). DNA replication origin activation in space and time. Nat. Rev. Mol. Cell Biol. <i>16</i>, 360-374.</p>
<p id="bib3">Hua, K. and Ferland, R.J. (2017). Fixation methods can differentially affect ciliary protein immunolabeling. Cilia <i>6</i>, 1-17.</p>
<p id="bib4">Juríková, M., Danihel, Ľ., Polák, Š., and Varga, I. (2016). Ki67, PCNA, and MCM proteins: Markers of proliferation in the diagnosis of breast cancer. Acta Histochem. <i>118</i>, 544-552.</p>
<p id="bib5">Kalogeropoulou, A., Lygerou, Z., and Taraviras, S. (2019). Cortical development and brain malformations: insights from the differential regulation of early events of DNA replication. Front. Cell Dev. Biol. <i>7</i>, 29.</p>
<p id="bib6">Moreno, A., Carrington, J.T., Albergante, L., Mamun, M. Al, Haagensen, E.J., Komseli, E.S., Gorgoulis, V.G., Newman, T.J., and Blow, J.J. (2016). Unreplicated DNA remaining from unperturbed S phases passes through mitosis for resolution in daughter cells. Proc. Natl. Acad. Sci. U S A <i>113</i>, E5757-E5764.</p>
<p id="bib7">Morino, M., Tanaka, M., Shiomi, Y., and Nishitani, H. (2014). Imaging analysis to determine chromatin binding of the licensing factor MCM2-7 in mammalian cells. Methods Mol. Biol. <i>1170</i>, 529-537.</p>
<p id="bib8">Nishitani, H., Morino, M., Murakami, Y., Maeda, T., and Shiomi, Y. (2014). Chromatin fractionation analysis of licensing factors in mammalian cells. Methods Mol. Biol. <i>1170</i>, 517-527.</p>
<p id="bib9">Petropoulos, M., Tsaniras, S.C., Taraviras, S., and Lygerou, Z. (2019). Replication licensing aberrations, replication stress, and genomic instability. Trends Biochem. Sci. <i>44</i>, 752-764.</p>
<p id="bib10">Sawasdichai, A., Chen, H.T., Hamid, N.A., Jayaraman, P.S., and Gaston, K. (2010). In situ subcellular fractionation of adherent and non-adherent mammalian cells. J. Vis. Exp. 1958.</p>
<p id="bib11">Symeonidou, I.E., Kotsantis, P., Roukos, V., Rapsomaniki, M.A., Grecco, H.E., Bastiaens, P., Taraviras, S., and Lygerou, Z. (2013). Multi-step loading of human minichromosome maintenance proteins in live human cells. J. Biol. Chem. <i>288</i>, 35852-35867.</p>
<p id="bib12">Symeonidou, I.E., Taraviras, S., and Lygerou, Z. (2012). Control over DNA replication in time and space. FEBS Lett. <i>586</i>, 2803-2812.</p>
<p id="bib13">Takahashi, T. (1995). The cell cycle of the pseudostratified embryonic murine cerebral wall. J. Neurosci. <i>15</i>, 6046-6057.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Dr. Stavros Taraviras (<a href="mailto:taraviras@med.upatras.gr">taraviras@med.upatras.gr</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate datasets.</p>
<h3>Acknowledgments</h3>
<p>This research is co-financed by Greece and the European Union (European Regional Development Fund) and supported by the INSPIRED (MIS-5002550), which is implemented under the action Reinforcement of the Research and Innovation Infrastructure funded by the Operational Program Competitiveness, Entrepreneurship and Innovation (NSRF 2014-2020) and the Operational Program Human Resources Development, Education and Lifelong Learning in the context of the project Reinforcement of Postdoctoral Researchers - 2nd Cycle (MIS-5033021), implemented by the State Scholarships Foundation (IKY). We thank the Advanced Light Microscopy Facility at the Medical School of the University of Patras and the Experimental Animal Facility of the University of Patras for support with experiments. We would also like to thank Dr. Meletios Verras for professional assistance with the graphical abstract design.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, M.M., A.K., and S.T.; formal analysis, N.N.G.; funding acquisition, S.T.; investigation, M.M. and A.K.; methodology, M.M. and A.K.; resources, S.T.; supervision, Z.L. and S.T.; visualization, M.M. and A.K.; writing – original draft, M.M. and A.K.; writing – review &amp; editing, Z.L. and S.T.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>