<article>
<section>
<h1 id="header">Isolation and analysis of fractions enriched in WrappER-associated mitochondria from mouse liver</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Nicolò Ilacqua,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Irene Anastasia,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Luca Pellegrini<sup><a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Graduate Program in Neuroscience, Faculty of Medicine, Laval University, Quebec, QC, Canada</p>
<p id="aff2"><sup>2</sup>Mitochondria biology laboratory, Brain Research Center, Quebec, QC, Canada</p>
<p id="aff3"><sup>3</sup>Dept. of Molecular Biology, Medical Biochemistry, and Pathology, Faculty of Medicine, Laval University, Quebec, QC, Canada</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:nicolo.ilacqua.1@ulaval.ca">nicolo.ilacqua.1@ulaval.ca</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:luca.pellegrini@fmed.ulaval.ca">luca.pellegrini@fmed.ulaval.ca</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100752">10.1016/j.xpro.2021.100752</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The endoplasmic reticulum (ER) plays a central role in lipid homeostasis, but the role of individual ER subdomains in lipid biology has not been elucidated. WrappER is a curved wrapping type of rough-ER that establishes extensive contacts with almost every mitochondria of the hepatocyte in the mouse liver. Here, we describe a protocol for isolation of fractions enriched in wrappER-associated mitochondria from the mouse liver. We also provide techniques for assessing its quality by electron microscopy and biochemical/proteomic analysis.</p>
<p>For complete information on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00459-7/sref1">Anastasia et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for isolating fractions enriched in wrappER-associated mitochondria (WAM)</li>
<li>Companion protocol for isolating a control ER-enriched fraction</li>
<li>Procedure to validate WAM- and ER-enriched fractions by electron microscopy analysis</li>
<li>Procedure to validate WAM- and ER-enriched fractions by immunoblot analysis</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/935-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Procedure to modify the tissue homogenizer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<div class="critical">
<span class="critical-title">Critical:</span> this protocol requires the use of a modified tissue grinder</div>
<ol>
<li>Take a new 5 mL Teflon Tissue Grinder Wheaton #358034.</li>
<li>Fill the potter with 5 mL of water (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/935-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Procedure to modify the liver homogenizer</div>
<p>(A) Steps required to modify a Teflon Tissue Grinder Wheaton #358034. Note that in this figure red-colored water is used only for convenience, to better visualize the white potter Teflon head.</p>
</figcaption>
</figure>
</li>
<li>Hold the pestle as shown in <a href="#fig1">Figure 1</a>A.</li>
<li>Drop the pestle and measure the time it takes to reach the bottom of the glass potter.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> typically, an unmodified 5 mL Teflon potter Wheaton #358034 takes 13–16 seconds to reach the bottom of the potter.</div>
<ol start="5">
<li>Increase the clearance between the Teflon pestle and the potter:
<ol type="a">
<li>Thin the Teflon head of the pestle evenly with 240-grit sandpaper.</li>
<li>Smooth the Teflon head of the pestle with 1200-grit sandpaper.</li>
</ol>
</li>
<li>Repeat step 4. The pestle should now take 9–10 s to reach the bottom of the potter (<a href="#fig1">Figure 1</a>A). If not, repeat steps 5 and 4 as long as necessary.</li>
<li>The potter is now ready to be used.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>EGTA</td>
<td>Sigma-Aldrich</td>
<td>#E4378</td>
</tr>
<tr>
<td>Halt Protease Inhibitor Cocktail (PIC) (100<b>×</b>)</td>
<td>Thermo Fisher Scientific</td>
<td>#78429</td>
</tr>
<tr>
<td>Magnesium chloride hexahydrate</td>
<td>Sigma-Aldrich</td>
<td>#M9272</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Thermo Fisher Scientific</td>
<td>#BP220-212</td>
</tr>
<tr>
<td>Trizma-base</td>
<td>Thermo Fisher Scientific</td>
<td>#T395</td>
</tr>
<tr>
<td>Osmium tetroxide</td>
<td>Electron Microscopy Sciences</td>
<td>#19152</td>
</tr>
<tr>
<td>Uranyl acetate</td>
<td>Electron Microscopy Sciences</td>
<td>#22400</td>
</tr>
<tr>
<td>Propylene oxide</td>
<td>Electron Microscopy Sciences</td>
<td>#82320</td>
</tr>
<tr>
<td>Lead nitrate</td>
<td>Electron Microscopy Sciences</td>
<td>#17900</td>
</tr>
<tr>
<td>Sodium nitrate</td>
<td>Electron Microscopy Sciences</td>
<td>#21140</td>
</tr>
<tr>
<td>Durcupan, single component A</td>
<td>Sigma-Aldrich</td>
<td>#44611</td>
</tr>
<tr>
<td>Durcupan, single component B</td>
<td>Sigma-Aldrich</td>
<td>#44612</td>
</tr>
<tr>
<td>Durcupan, single component C</td>
<td>Sigma-Aldrich</td>
<td>#44613</td>
</tr>
<tr>
<td>Durcupan, single component D</td>
<td>Sigma-Aldrich</td>
<td>#44614</td>
</tr>
<tr>
<td>Glutaraldehyde solution, 25% in H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>#G5882</td>
</tr>
<tr>
<td>Calcium chloride, anhydrous</td>
<td>Thermo Fisher Scientific</td>
<td>#S25223</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Teflon Dounce homogenizer 5 mL</td>
<td>Wheaton</td>
<td>#358034</td>
</tr>
<tr>
<td>30 mL Round-bottomed glass centrifuge tubes</td>
<td>Kimble</td>
<td>#45500-30</td>
</tr>
<tr>
<td>Rotor adapter for 30 mL glass centrifuge tubes</td>
<td>Kimble</td>
<td>#45550-30</td>
</tr>
<tr>
<td>Rotor adapter</td>
<td>Sigma-Aldrich</td>
<td>#EP022637266-2EA</td>
</tr>
<tr>
<td>Glass disposable Pasteur pipettes</td>
<td>VWR International</td>
<td>#14673-043</td>
</tr>
<tr>
<td>Straight operating scissors: sharp/blunt blades</td>
<td>Stoelting Co</td>
<td>#52136-12P</td>
</tr>
<tr>
<td>Sandpaper 240 and 1200 grit</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>15 mL and 50 mL Polypropylene Falcon tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>50 mL Pyrex glass beaker</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>50 mL Plastic beaker</td>
<td>Electron Microscopy Sciences</td>
<td>#60968</td>
</tr>
<tr>
<td>2 mL Plastic transfer pipette</td>
<td>SARSTEDT</td>
<td>#86.1176.300</td>
</tr>
<tr>
<td>Single Slot Copper grids</td>
<td>Electron Microscopy Sciences</td>
<td>#G2010-Cu</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Adult male C57BL/6N mice (8 weeks of age)</td>
<td>Charles River</td>
<td>Stain code: 027</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="critical">
<span class="critical-title">Critical:</span> All buffers should be freshly prepared on the morning of the experiment.</div>
<table id="undtbl1">
<caption>Buffer A (BA)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH 7.4)</td>
<td>1 M</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>0.5 M</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>EGTA (pH 7.4)</td>
<td>0.1 M</td>
<td>0.1 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the BA at 20°C.</div>
<table id="undtbl2">
<caption>Buffer 10 (B10)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>n/a</td>
<td>250 mM</td>
<td>4.28 g</td>
</tr>
<tr>
<td>Tris-HCl (pH 7.4)</td>
<td>1 M</td>
<td>10 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>0.5 M</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>EGTA (pH 7.4)</td>
<td>0.1 M</td>
<td>0.1 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the B10 always on ice.</div>
<table id="undtbl3">
<caption>Buffer 27 (B27)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>n/a</td>
<td>27% (w/w)</td>
<td>6.75 g</td>
</tr>
<tr>
<td>Buffer A</td>
<td>n/a</td>
<td>73% (w/w)</td>
<td>18.25 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>25 g</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the B27 at 20°C.</div>
<table id="undtbl4">
<caption>CaCl<sub>2</sub> 8 mM solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl2</td>
<td>800 mM</td>
<td>8 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the CaCl<sub>2</sub> 8 mM solution always on ice.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Mouse liver WAM-enriched fraction preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>The following protocol describes the steps required to obtain WAM-enriched fractions from a mouse liver weighing between 1.0 and 1.4 grams.</p>
<div class="note">
<span class="note-title">Note:</span> In our experience, this protocol allows the simultaneous processing of up to 3 livers, but no more.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not clean glassware with detergents. Always thoroughly rinse all glassware with ddH<sub>2</sub>O before and after using them.</div>
<ol>
<li>Prepare all the buffers and cool down the centrifuge to 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use a fixed angle rotor type F34-6-38 for Eppendorf centrifuge 5800 (<a href="#fig2">Figure 2</a>A).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/935-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Some of the equipment needed to isolate WAM-enriched mouse liver fractions</div>
<p>(A) Beaker containing mouse chopped liver in ice-cold B10 buffer.</p>
<p>(B) Motorized tissue homogenizer.</p>
<p>(C) Fixed angle rotor type F34-6-38 for Eppendorf centrifuge 5800 equipped with adapters for 50 mL Falcon tubes.</p>
<p>(D) 1 mL and 2 mL plastic transfer pipettes.</p>
</figcaption>
</figure>
<ol start="2">
<li>To prepare B10<sub>PIC</sub>, transfer 11 mL of B10 to a 15mL-Falcon tube, then add 110 μL of protease inhibitor cocktail (PIC).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not use a protease inhibitor cocktail containing EDTA, as chelating Mg<sup>++</sup> interferes with retaining ribosomes attached to the ER membrane.</div>
<ol start="3">
<li>Transfer 15 mL of B10 to a 50 mL-glass beaker. Keep it on ice.</li>
<li>Euthanize the mouse, then incise the abdomen with surgical scissors to expose the liver. Excide the liver, discard the gallbladder, and transfer it to the 50 mL-glass beaker prepared in step 3.</li>
<li>Use clean surgical scissors to cut the liver into small pieces (<a href="#fig2">Figure 2</a>B), then rinse three times with 15 mL of B10 to remove the blood.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Let the small liver pieces decant, then remove most of the buffer by gently tilting the beaker. Use a P1000 pipette to remove the last few mL of B10.</div>
<ol start="6">
<li>Add 4 mL of B10<sub>PIC</sub> to the chopped, rinsed liver and transfer it into the modified 5 mL Teflon Potter by tilting the beaker.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use surgical scissors or a wooden spatula to assist in the transfer of the small liver pieces.</div>
<div class="note">
<span class="note-title">Note:</span> Do not fill the potter past the fill line. Depending on the size of the liver, it may be necessary to remove excess buffer with a P1000 before proceeding to the next step.</div>
<ol start="7">
<li>Place the modified glass potter with the chopped liver inside a 50 mL Falcon tube containing ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep the glass potter cold during liver homogenization.</div>
<ol start="8">
<li>Use an electric homogenizer (<a href="#fig2">Figure 2</a>C) to homogenize the liver by stroking up and down the potter.
<ol type="a">
<li>Set the homogenizer speed to 1,600 rpm, then perform 16 strokes: 8 ups and 8 downs, 2 s per stroke; then pause for 30 s.</li>
</ol>
</li>
<li>Repeat step 8a.</li>
<li>Transfer the liver homogenate by tilting the glass potter into a 15 mL Falcon tube.</li>
<li>Rinse the potter with the remaining 7 mL of cold B10<sub>PIC</sub> and add them to the liver homogenate. Mix gently by inverting the tube a few times and pour the homogenate into two 30 mL glass centrifuge tubes.</li>
<li>Spin at 400 <b>×</b> <i>g</i> for 10 min at 4°C to remove nuclei and tissue debris.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use a fixed angle rotor type F34-6-38 for Eppendorf centrifuge 5800 (<a href="#fig2">Figure 2</a>A).</div>
<ol start="13">
<li>Collect the supernatant (∼5 mL per tube) and transfer it to a clean 30 mL glass centrifuge tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Stay away from the pellet: it contains unbroken cells and nuclei that would contaminate the WAM-enriched fraction.</div>
<div class="note">
<span class="note-title">Note:</span> Use a P1000 pipette tip, cut ∼2 mm from the end to enlarge the opening.</div>
<ol start="14">
<li>Repeat steps 12 and 13.</li>
<li>Repeat step 12 one more time, collect the two supernatants and gently mix them into a 15 mL Falcon tube (the final volume should be ∼ 7.5 mL).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Stay well away from the pellet.</div>
<div class="note">
<span class="note-title">Note:</span> Use a P1000 pipette tip, cut ∼2 mm from the end to enlarge the opening.</div>
<div class="note">
<span class="note-title">Note:</span> This is what, from here on, we call post-nuclear hepatic homogenate. You can set aside up to 500 μL and use it as a control in assays concurrent with biochemical validation of the WAM-enriched fraction.</div>
<ol start="16">
<li>Pour 7 mL of B27 (maintained at 20°C) into a 50 mL Falcon tube and carefully lay on top of it 7 mL of post-nuclear hepatic homogenate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use a P1000 pipette tip, cut ∼2 mm from the end to enlarge the opening.</div>
<ol start="17">
<li>Spin at 2,000 <b>×</b> <i>g</i> for 20 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use a fixed angle rotor type F34-6-38 for Eppendorf centrifuge 5800 (<a href="#fig2">Figure 2</a>A).</div>
<ol start="18">
<li>Collect the top 5 mL of the supernatant and use it to prepare the control ER-enriched fraction (see below “<a href="#sec3.2">complementary protocol: preparation of ER-enriched fraction</a>”).</li>
<li>Thoroughly remove the supernatant using a disposable glass Pasteur pipette connected to the vacuum line. The pellet is the WAM-enriched fraction (<a href="#fig3">Figures 3</a>A and 3B).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/935-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Schematic representation of the procedure used to prepare wrappER-mitochondria (WAM)-enriched fractions</div>
<p>(A) 7 mL of B27 are added into a 50 mL Falcon tube, on top of which 7 mL of post-nuclear liver homogenate are gently layered. The solution is centrifugated in a fixed rotor. The pellet is enriched in WAM.</p>
<p>(B) An image of the pellet enriched in WAM. Note that its appearance depends on the nutritional status of the mouse liver from which it was prepared.</p>
</figcaption>
</figure>
<ol type="a">
<li>For omics analysis, resuspend the pellet in 500 μL B10<sub>PIC</sub> and store at −80°C.</li>
<li>For Electron Microscopy (EM) analysis, gently add 500 μL of B27<sub>PIC</sub> on top of the pellet. Using a paintbrush, gently detach it from the side of the tube; then, follow “<i>Complementary Protocol: EM Analysis of WAM-Enriched Fractions</i>”.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Step 19b goal is to obtain a flaky pellet, limiting its resuspension in the buffer.</div>
<h3 id="sec3.2">Complementary protocol: Preparation of ER-enriched fraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>The following procedure describes how to obtain ER-enriched fractions from WAM-depleted liver homogenates. They can be used <i>per se</i> or as a control to the proteomic/functional analysis of the WAM-enriched fractions. This protocol has been modified from (<a href="#bib2">Hamilton et al., 1999</a>).</p>
<ol start="20">
<li>Starting from step 18, aliquot the supernatant into two 2 mL Eppendorf tubes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When collecting the supernatant in step 18, stay away from the fat layer that accumulates at the top and along the wall of the 50 mL Falcon tube.</div>
<ol start="21">
<li>Spin at 12,000 <b>×</b> <i>g</i> for 10 min at 4°C to remove heavy organelles; collect the supernatants with a P1000 pipette tip and transfer them into two clean 2 mL Eppendorf tubes.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Stay away from the pellet: it contains heavy organelles such as peroxisomes, lysosomes, mitochondria that would contaminate the ER-enriched fraction.</div>
<ol start="22">
<li>Repeat step 21.</li>
<li>Collect 1.5 mL of supernatant from each tube and gently mix them into a 50 mL Falcon tube. Gently add 22.5 mL of ice-cold CaCl<sub>2</sub> 8 mM solution.</li>
<li>Incubate for 15 min at 4°C, with gentle rocking.</li>
<li>Pour gently into two 30 mL glass tubes (12 mL/tube), then spin at 8,000 <b>×</b> <i>g</i> for 10 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use a fixed angle rotor type F34-6-38 for Eppendorf centrifuge 5800.</div>
<ol start="26">
<li>Discard the supernatants; the pellet is the ER-enriched fraction.
<ol type="a">
<li>For omics analysis, resuspend the pellet in 500 μL B10<sub>PIC</sub> and store at −80°C.</li>
<li>For Electron Microscopy (EM) analysis, gently add 500 μL of B27<sub>PIC</sub> on top of the pellet. Using a paintbrush, gently detach it from the side of the tube; then, follow “<i>Complementary Protocol: EM Analysis of WAM-Enriched Fractions</i>”.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Step 26b goal is to obtain a flaky pellet, limiting its resuspension in the buffer.</div>
<div class="note">
<span class="note-title">Note:</span> Proteomic analysis of this fraction should indicate that ∼40% of the proteins are ER proteins (<a href="#fig4">Figure 4</a>C).</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/935-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Analysis of mouse liver WAM-enriched fractions</div>
<p>(A–D) The pellet shown in <a href="#fig3">Figure 3</a>B is analyzed by EM (A), immunoblot (B and D), and proteomic studies (C). Adapted from (<a href="#bib1">Anastasia et al., 2021</a>).</p>
<p>(E) EM analysis of the control mouse liver ER-enriched fractions.</p>
</figcaption>
</figure>
<h3 id="sec3.3">Complementary protocol: EM analysis of WAM-enriched fractions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Here we describe how to fix and embed WAM- and ER-enriched fractions for EM analysis, a reliable method for verifying their quality.</p>
<ol start="27">
<li>Transfer the flaky pellet (step 19b or 26b) into a 2 mL Eppendorf tube and add 500 μL of B27<sub>PIC</sub>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use a tube with a round, not conical, bottom.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use a P200 pipette tip, cut ∼5 mm from the end to enlarge the opening, and avoid resuspending the pellet. The pellet must remain visibly flaky.</div>
<ol start="28">
<li>Spin at 3,000 <b>×</b> <i>g</i> for 10 min at 4°C. Remove and discard the supernatant.</li>
<li>Gently add to the pellet 500 μL of B27 solution supplemented with 2% glutaraldehyde (freshly prepared), and incubate overnight (16 h) at 4°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> from this point on, the pellet must remain intact, possibly sticking against the wall of the Eppendorf tube. Through all the next steps, pay attention to always be gentle when adding and removing all various solutions; to this end, we recommend using plastic transfer pipettes (<a href="#fig2">Figure 2</a>D).</div>
<div class="note">
<span class="note-title">Note:</span> Glutaraldehyde is highly toxic: work under a chemical hood.</div>
<ol start="30">
<li>Remove the glutaraldehyde solution with a plastic transfer pipette, paying attention to leave the pellet undisturbed</li>
<li>Rinse the pellet three times to eliminate any residual glutaraldehyde. To this end, gently apply onto the wall of the Eppendorf tube 700 μL of B27 solution (use a P1000 pipette).</li>
<li>Incubate the pellet in a solution of 1% (w/v) osmium tetroxide in ddH<sub>2</sub>O for 30 min at 20°C, keeping the Eppendorf tube protected from light by wrapping it in aluminum foil.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Osmium tetroxide is highly toxic: operate under a chemical hood.</div>
<ol start="33">
<li>During the incubation (step 32), reconstitute the Durcupan epoxy resin:
<ol type="a">
<li>In a 50 mL plastic beaker placed on a scale, add 5 g of Durcupan component A, 5 g of Durcupan component B, 0.15 g of Durcupan component C, and 0.15 g of Durcupan component D.
<div class="note">
<span class="note-title">Note:</span> Durcupan components are dense liquids: use plastic pipettes (<a href="#fig2">Figure 2</a>D).</div>
</li>
<li>Stir well with a wooden spatula until the resin changes color and becomes orange.
<div class="note">
<span class="note-title">Note:</span> The reconstituted Durcupan epoxy resin is now ready to be used but it can be stored for up to 2 weeks at −20°C.</div>
</li>
</ol>
</li>
</ol>
<ol start="34">
<li>Rinse the pellet (step 32) three times to eliminate any residual osmium tetroxide. To this end, gently apply onto the wall of the Eppendorf tube 700 μL of ddH<sub>2</sub>O (use a P1000 pipette).</li>
</ol>
<ol start="35">
<li>Pellet dehydration:
<div class="note">
<span class="note-title">Note:</span> Use plastic transfer pipettes (<a href="#fig2">Figure 2</a>D) to add/remove the solutions used in this step.</div>
<ol type="a">
<li>Add 1 mL of 50% ethanol:water and incubate for 15 min. Discard ethanol solution.</li>
<li>Add 1 mL of 1% uranyl acetate in 70% ethanol:water and incubate for 40 min keeping the Eppendorf tube protected from light by wrapping it with aluminum foil. Remove and discard the uranyl acetate solution.</li>
<li>Add 1 mL of 90% ethanol:water and incubate for 15 min. Discard ethanol solution.</li>
<li>Add 1 mL of 100% ethanol and incubate for 15 min. Discard ethanol solution.</li>
<li>Repeat step 35d.</li>
<li>Add 1 mL of 100% propylene oxide to the pellet and incubate for 15 min. Discard propylene oxide.</li>
<li>Repeat step 35f.
<div class="critical">
<span class="critical-title">Critical:</span> Work rapidly as propylene oxide evaporates quickly at 20°C. The sample must not dry out when changing the solutions; perform all steps at 20°C.</div>
<div class="note">
<span class="note-title">Note:</span> Propylene oxide is highly toxic: operate under a chemical hood.</div>
</li>
</ol>
</li>
</ol>
<ol start="36">
<li>Embed the pellet in the Durcupan epoxy resin:
<div class="note">
<span class="note-title">Note:</span> Step 36b-d: leave the cap of the Eppendorf tube open.</div>
<ol type="a">
<li>Remove and discard the 100% propylene oxide and quickly embed the pellet in ∼500 μL of reconstituted Durcupan epoxy resin.
<div class="note">
<span class="note-title">Note:</span> From this stage, the sample can be handled on the lab bench.</div>
</li>
<li>Incubate for 1 h at 37°C.</li>
<li>Remove as much epoxy resin as possible, leaving the pellet undisturbed and, ideally, attached to the side of the Eppendorf tube. Add 500 μL epoxy resin and move the sample in an oven at 58°C.</li>
<li>Incubate 48–72 h, to polymerize the reconstituted Durcupan epoxy resin.</li>
</ol>
</li>
</ol>
<ol start="37">
<li>Release the hardened epoxy resin block that contains the sample to analyze by EM; this can be done by cutting the 2 mL Eppendorf tube with a sharp blade or scissors.</li>
<li>Cut the epoxy resin-embedded sample in ∼50 nm -thick slices using a Leica Ultramicrotome and place them on single-slot formvar coated copper grids (up to 5 slices/grid).</li>
<li>Counterstain slices by incubating the grid for 5 min with Reynold’s lead citrate.</li>
<li>The WAM and ER-enriched fractions can be now imaged under a transmission electron microscope (<a href="#fig4">Figures 4</a>A and 4E).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>At the end of the protocol that yields the WAM-enriched fraction (step 19), the size and color of the pellet depend on the nutritional status of the animal when the liver was collected (<a href="#fig3">Figure 3</a>B). Regardless, at step 19a the expected protein concentration should be in the 5–10 μg/μL range (determined by Bradford assay).</p>
<p>Electron microscopy analysis is the most reliable method to verify that the protocol described here has been performed correctly (<a href="#fig4">Figure 4</a>A). EM analysis of WAM-enriched fractions should show that at least ∼75% of mitochondria are associated with wrappER (<a href="#fig4">Figure 4</a>A).</p>
<p>Immunoblot analysis of the WAM-enriched fraction (10 μg/lane) must reveal the enrichment of mitochondrial and ER proteins (membrane-anchored and soluble in the matrix/ER lumen); this can be done using post-nuclear liver homogenate as control (10 μg/lane; step 15). This analysis must also show enrichment of ribosomal proteins (<a href="#fig4">Figure 4</a>B). In contrast, contaminant lysosome, nucleus, plasma membrane, and Golgi proteins should be absent or minimally present. Note that peroxisomal proteins might be present because WAM-enriched fractions contain some wrappER-associated peroxisomes and peroxisome-wrappER-mitochondria complexes (our unpublished study).</p>
<p>Proteomic analysis of WAM-enriched fractions should show that ∼80% of the proteins are from mitochondria, ER, and ribosomes (<a href="#fig4">Figure 4</a>C) (<a href="#bib1">Anastasia et al., 2021</a>).</p>
<p>When compared to ER-enriched fractions (see above, step 20–26), the pattern of expression of the Rrbp1 protein should be as shown in <a href="#fig4">Figure 4</a>D (<a href="#bib1">Anastasia et al., 2021</a>).</p>
<p>This protocol is designed to retain the wrappER ribosomes and mRNAs, allowing to perform transcriptomic analysis. Such analyses should reveal a massive presence of ER protein transcripts and a minimal presence of nuclear-encoded mitochondrial protein transcripts (<a href="#bib1">Anastasia et al., 2021</a>); however, mitochondrial DNA-encoded transcripts should be highly expressed.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Preparing WAM-enriched fractions from fatty livers can be problematic. The yield of WAM may diminish due to ineffective liver homogenate preparation and/or separation during the early phases of the protocol (steps 8 and 9). The presence of large lipid droplets may indeed cause wrappER shredding during sample homogenization. Therefore, any WAM isolation should be done by validating the integrity of this inter-organelle contact through EM analysis (step 27–40).</p>
<p>Recent studies from our laboratory have shown that the wrappER, in addition to mitochondria, also forms contacts with peroxisomes and that the protocol described here also yields wrappER-associated peroxisomes and peroxisome-wrappER-mitochondria (PEWM) complexes.</p>
<p>This protocol has been optimized for the isolation of WAM-enriched fractions from the mouse liver. We expect it to be equally effective in other model organisms, such as rats or fish, as long as the amount of liver used is not different from that given here.</p>
<p>Isolation of WAM-enriched fractions has not been attempted in other tissues and cultured cells, although we expect it to be possible in soft tissues such as the pancreas and thymus. As for cultured cells, the main hurdle is to adapt this protocol to a smaller scale and to modify the homogenization step.</p>
<p>Membrane-permeable molecules (including protease inhibitors) hurt the yield and structure of the WAM. Membrane-permeabilizing detergents must also be avoided, which is why indicated to clean the glassware by rinsing it with water.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Small pellet (step 19; <a href="#fig3">Figure 3</a>B). This typically means poor WAM yield, which can be caused either by poor liver homogenization (steps 8 and 9) or excessive organelle disruption/shredding (see <a href="#sec6.3">problem 2</a>).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Increase liver homogenization.</p>
<p>Start by chopping the liver into smaller pieces (step 5; <a href="#fig2">Figure 2</a>B); also, consider increasing the number of strokes (steps 8 and 9).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Small pellet (step 19; <a href="#fig3">Figure 3</a>B) and a) EM analysis shows either mitochondria without wrappER or broken mitochondria (step 40), or b) immunoblot shows low level of soluble mitochondria matrix markers (such as MnSOD, due to broken mitochondria).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Decrease liver homogenization.</p>
<p>In general, keep in mind that high hydraulic pressure destroys the integrity of any organelles, particularly in a viscous media. High hydraulic pressure is typically caused by pushing hard and fast the Teflon pestle into the glass potter containing the liver homogenate (step 8). Therefore, considering go slower in this step.</p>
<p>If hydraulic pressure is OK, then consider reducing the number of strokes to 10–12 (steps 8 and 9) and/or slow down the speed of the rotor to 1,200 rpm.</p>
<p>Consider also thinning a little more the Teflon head of the pestle (to 6–8 s), as this decreases hydraulic pressure too.</p>
<p>Excessive organelle shredding can also be caused by the presence of large masses of lipid droplets during homogenization (steps 8 and 9). Use 20% less liver without changing the volume of B10 solution (4 mL; step 6).</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Electron microscopy analysis shows the presence of intact organelles such as, for example, lysosomes and peroxisomes (step 40).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Improve the homogenization described in steps 8 and 9. This requires increasing the number of strokes to 20–24. Also, increase the volume of B27 solution from 7 to 10 mL (step 16).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Immunoblot analysis of ER-enriched fractions detects membrane proteins from lysosomes, plasma membrane, peroxisomes, Golgi, etc (step 26a).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Decrease liver homogenization to limit the disruption of the organelles' integrity. Consider increasing the potter-pestle clearance (steps 5 and 6 of “<a href="#sec1.1">procedure to modify the tissue homogenizer</a>”) and/or reducing the number and speed of the strokes (steps 8 and 9).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Pellet breaks apart during steps 29–34 of “<a href="#sec3.3">complementary protocol: EM analysis of WAM-enriched fractions</a>”.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Centrifuge the sample at 6,000 <b>×</b> <i>g</i> for 10 min at 4°C.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Anastasia, I., Ilacqua, N., Raimondi, A., Lemieux, P., Ghandehari-Alavijeh, R., Faure, G., Mekhedov, S.L., Williams, K.J., Caicci, F., Valle, G., et al. (2021). Mitochondria-rough-ER contacts in the liver regulate systemic lipid homeostasis. Cell Rep. <i>34</i>, 108873. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00459-7/sref1">View at publisher</a></p>
<p id="bib2">Hamilton, R.L., Moorehouse, A., Lear, S.R., Wong, J.S., and Erickson, S.K. (1999). A rapid calcium precipitation method of recovering large amounts of highly pure hepatocyte rough endoplasmic reticulum. J. Lipid Res. <i>40</i>, 1140-1147. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00459-7/sref2">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Luca Pellegrini (<a href="mailto:luca.pellegrini@fmed.ulaval.ca">luca.pellegrini@fmed.ulaval.ca</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The published article (<a href="#bib1">Anastasia et al., 2021</a>) includes all datasets generated and analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>N.I. and I.A. are recipients of a PhD scholarship from the Centre Thématique de Recherche en Neurosciences (CTRN) and from Fonds de la recherche en santé du Québec (FRQS). This study was funded by grants from the <a href="https://doi.org/10.13039/501100000024">Canadian Institutes of Health Research</a> (CIHR, 201603PJT-365052) and the <a href="https://doi.org/10.13039/501100000038">Natural Sciences and Engineering Research Council of Canada</a> (NSERC, RGPIN-2017-06130) to L.P.</p>
<h3>Author contributions</h3>
<p>Conceptualization, N.I., I.A., and L.P.; investigation, N.I. and I.A.; writing – original draft, N.I., I.A., and L.P.; writing – review &amp; editing, N.I. and L.P.; funding acquisition, L.P.; supervision, L.P.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>