<article>
<section>
<h1 id="header">Synthesis and application of POLYseq for profiling human liver organoids</h1>
<p><time datetime="2021-12-10">Published: December 10, 2021</time></p>
<p>Andrew W. Dunn,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Yuqi Cai,<sup><a href="#aff1">1</a></sup> Kentaro Iwasawa,<sup><a href="#aff1">1</a></sup> Masaki Kimura,<sup><a href="#aff1">1</a></sup> and Takanori Takebe<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Division of Gastroenterology, Hepatology &amp; Nutrition, Developmental Biology and Center for Stem Cell and Organoid Medicine (CuSTOM), Cincinnati Children’s Hospital Medical Center, Cincinnati, OH, USA</p>
<p id="aff2"><sup>2</sup>Institute of Research, Tokyo Medical and Dental University (TMDU), 1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510, Japan</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:takanori.takebe@cchmc.org">takanori.takebe@cchmc.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100976">10.1016/j.xpro.2021.100976</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Detailed herein is the protocol for synthesis, characterization, and application of POLYseq for cell pooling in single-cell sequencing runs. POLYseq is synthesized through commercially available reagents and is highly tailorable. Synthesis is easily performed in a two-step protocol, utilizing Michael addition only requiring a temperature-stable hot bath capable of holding 90°C. However, care must be taken when mixing reagents for synthesis, as the final product is sensitive to initial mixing ratios of POLYseq reagents.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref1">Dunn et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detailed protocols for POLYseq using inexpensive, commercially available reagents</li>
<li>POLYseq application involves fluorescent imaging and DNA barcoding</li>
<li>POLYseq-based multiplexing for single-cell RNA sequencing with cells and organoids</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1213-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Before beginning synthesis, a temperature controllable hot bath using sand or silicone oil will be required.</p>
<h3 id="sec1.1">Creations of POLYseq reagent working solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h]</div>
<ol>
<li>Dissolve POLYseq reagents in anhydrous DMSO to a final concentration of 200 mg/mL. Care should be taken to include listed densities of reagents in determination of volume of DMSO to add to achieve the final concentration.
<ol type="a">
<li>Table of reagent densities and DMSO volumes to add to stock reagents is listed below. Volume of <b>DMSO (ml)</b> is calculated for <b>200 mg</b> of respective reagent to achieve a final concentration of 200 mg/mL. Reagent mass and respective DMSO volume may be linearly scaled for establishment of larger working solutions.</li>
</ol>
</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Identifier</th>
<th>Density (g/mL)</th>
<th>DMSO (ml)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Poly(ethylene glycol) diacrylate, Mn = 250</td>
<td>D8</td>
<td>1.11</td>
<td>0.820</td>
</tr>
<tr>
<td>Di(trimethylolpropane) tetraacrylate</td>
<td>V5</td>
<td>1.101</td>
<td>0.818</td>
</tr>
<tr>
<td>3-amino-1-propanol</td>
<td>S3</td>
<td>0.982</td>
<td>0.796</td>
</tr>
<tr>
<td>1,4-Bis(3-aminopropyl)piperazine</td>
<td>C1</td>
<td>0.973</td>
<td>0.794</td>
</tr>
<tr>
<td>Spermine</td>
<td>C2</td>
<td>0.937</td>
<td>0.787</td>
</tr>
<tr>
<td>Polyethylenimine, Mn = 600</td>
<td>C3</td>
<td>1.05</td>
<td>0.810</td>
</tr>
<tr>
<td>2,2-dimethyl-1,3-propanediamine</td>
<td>C4</td>
<td>0.851</td>
<td>0.765</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Maintain all dissolved reagents in the dark at – 20°C once dissolved. Dissolved reagents are stable for at least one year. Reagents may be stored in standard borosilicate screw cap vials. Take care to use barrier pipette tips and solvent resistant pipettes.</li>
</ol>
<h3 id="sec1.2">Creation of stock solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [10 min]</div>
<ol start="3">
<li>Dissolve HEPES with pure ddH2O to a final concentration of 25 mM, pH to 7.4.
<ol type="a">
<li>Sterile filter 25 mM HEPES with 0.22 μm filter.</li>
</ol>
</li>
<li>Suspend ssDNA barcodes to a final concentration of 1 μg/μL in HEPES.</li>
<li>Suspend 1 mg of DyLight NHS powder with 100 μL of DMSO directly in the manufacturer’s tube to create a 10 mg/mL stock solution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Store dissolved ssDNA barcodes aliquoted at −20°C for long-term storage.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Poly(ethylene glycol) diacrylate, Mn = 250</td>
<td>Sigma-Aldrich</td>
<td>Cat#475629; CAS: 26570-48-9</td>
</tr>
<tr>
<td>Di(trimethylolpropane) tetraacrylate</td>
<td>Sigma-Aldrich</td>
<td>Cat#408360; CAS: 94108-97-1</td>
</tr>
<tr>
<td>3-amino-1-propanol</td>
<td>Sigma-Aldrich</td>
<td>Cat#239844; CAS: 156-87-6</td>
</tr>
<tr>
<td>1,4-Bis(3-aminopropyl)piperazine</td>
<td>Sigma-Aldrich</td>
<td>Cat#239488; CAS: 7209-38-3</td>
</tr>
<tr>
<td>Spermine</td>
<td>Sigma-Aldrich</td>
<td>Cat#85590; CAS:71-44-3</td>
</tr>
<tr>
<td>Polyethylenimine, Mn = 600</td>
<td>Sigma-Aldrich</td>
<td>Cat#408719; CAS:26913-06-4</td>
</tr>
<tr>
<td>2,2-dimethyl-1,3-propanediamine</td>
<td>Sigma-Aldrich</td>
<td>Cat#226920; CAS:7328-91-8</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Sigma-Aldrich</td>
<td>Cat#276855; CAS: 67-68-5</td>
</tr>
<tr>
<td>Dimethyl sulfoxide-d<sub>6</sub> (DMSO-d<sub>6</sub>)</td>
<td>Sigma-Aldrich</td>
<td>Cat#1035910050; CAS: 2206-27-1</td>
</tr>
<tr>
<td>Supelco vials, pre-cleaned and assembled</td>
<td>Sigma-Aldrich</td>
<td>Cat#27347</td>
</tr>
<tr>
<td>PYREX 12×75 mm Disposable Rimless Culture Tubes</td>
<td>Corning</td>
<td>Cat#99445-12</td>
</tr>
<tr>
<td>Falcon Snap Caps for 12×75 mm Test Tubes, Sterile</td>
<td>Corning</td>
<td>Cat#352032</td>
</tr>
<tr>
<td>Silicone oil</td>
<td>Sigma-Aldrich</td>
<td>Cat#1077420100; CAS: 68083-14-7</td>
</tr>
<tr>
<td>HEPES</td>
<td>Thermo Fisher</td>
<td>Cat#15630106; CAS: 7365-45-9</td>
</tr>
<tr>
<td>Agarose low EEO</td>
<td>Fisher Scientific</td>
<td>Cat#AAJ6636922; CAS: 9012-36-6</td>
</tr>
<tr>
<td>TAE, 50×</td>
<td>Fisher Scientific</td>
<td>Cat#AAJ636931K7; CAS: 77-86-1, 60-00-4, 6850-28-8</td>
</tr>
<tr>
<td>SYBR Safe</td>
<td>Thermo Fisher</td>
<td>Cat#S33102; CAS: 1030826-36-8</td>
</tr>
<tr>
<td>Gel Loading Dye, Purple (6×), no SDS</td>
<td>New England BioLabs</td>
<td>Cat#B7025S</td>
</tr>
<tr>
<td>iMatrix-511 (recombinant laminin-511)</td>
<td>Takara Bio Inc.</td>
<td>Cat#T304</td>
</tr>
<tr>
<td>Y-27632-Dihydrochloride</td>
<td>Tocris Bioscience</td>
<td>Cat#1254; CAS: 129830-38-2</td>
</tr>
<tr>
<td colspan="3">Molecular/chemical probes</td>
</tr>
<tr>
<td>DyLight 650-NHS</td>
<td>Thermo Fisher</td>
<td>Cat#62265</td>
</tr>
<tr>
<td>DyLight 488-NHS</td>
<td>Thermo Fisher</td>
<td>Cat#46402</td>
</tr>
<tr>
<td>Image-iT TMRM Reagent</td>
<td>Thermo Fisher</td>
<td>Cat#I34361; CAS: 152842-32-5</td>
</tr>
<tr>
<td>LysoTracker Blue DND-22</td>
<td>Thermo Fisher</td>
<td>Cat#L7525</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Chromium Next GEM Single Cell 3’ Kit v3.1, 4 rxns</td>
<td>10× Genomics</td>
<td>Cat#1000269</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Raw HLO</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>Cat#GSM4992600</td>
</tr>
<tr>
<td>Raw HLO + OA</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>Cat#GSM4992603</td>
</tr>
<tr>
<td>Raw HLO + OA + MGL</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>Cat#GSM4992605</td>
</tr>
<tr>
<td>Raw POLYseq Pool</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>Cat#GSM4992607</td>
</tr>
<tr>
<td>Raw TotalSeq Pool</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>Cat#GSM4992608</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Hep G2</td>
<td>ATCC</td>
<td>Cat#HB-8065</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>10× Additive Primer:GTGACTGGAGTTCAGACGTGTGC∗T∗C</td>
<td>This paper; C∗T∗ indicates 3’ phosphorothioate</td>
<td>N/A</td>
</tr>
<tr>
<td>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b><u>CATCTTGTGATC</u></b><i>B</i>(A)<sub>30</sub>
</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b><u>AGAAGGACGAGT</u></b><i>B</i>(A)<sub>30</sub>
</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b><u>CACCATGTACCA</u></b><i>B</i>(A)<sub>30</sub>
</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b>GGCTCTTCTCGG</b><i>B</i>(A)<sub>30</sub>
</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b><u>CATGGTTGGTAC</u></b><i>B</i>(A)<sub>30</sub>
</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b><u>TGACTCCTCAGT</u></b><i>B</i>(A)<sub>30</sub>
</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b><u>ATCCTGGTCCTA</u></b><i>B</i>(A)<sub>30</sub>
</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td>GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b><u>GTCTACCATCTG</u></b><i>B</i>(A)<sub>30</sub>
</td>
<td><a href="#bib1">Dunn et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Seurat</td>
<td>Satija Lab</td>
<td><a href="https://satijalab.org/seurat/index.html">https://satijalab.org/seurat/index.html</a></td>
</tr>
<tr>
<td>Loupe Browser v4.0.0</td>
<td>10× Genomics</td>
<td><a href="https://www.10xgenomics.com/products/loupe-browser">https://www.10xgenomics.com/products/loupe-browser</a></td>
</tr>
<tr>
<td>ImageJ</td>
<td>NIH</td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Wilmad NMR tubes, precision, 600 MHz</td>
<td>Sigma-Aldrich</td>
<td>Cat#Z272027</td>
</tr>
<tr>
<td>DyLight 650-NHS</td>
<td>Thermo Fisher</td>
<td>Cat#62265</td>
</tr>
<tr>
<td>Countess II Automated Cell Counter</td>
<td>Thermo Fisher</td>
<td>Cat#A27978</td>
</tr>
<tr>
<td>Opti-MEM Reduced Serum Medium</td>
<td>Thermo Fisher</td>
<td>Cat#31985062</td>
</tr>
<tr>
<td>DMEM, high glucose</td>
<td>Thermo Fisher</td>
<td>Cat#11965092</td>
</tr>
<tr>
<td>DMEM/F-12</td>
<td>Thermo Fisher</td>
<td>Cat#11320033</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS), heat inactivated</td>
<td>Thermo Fisher</td>
<td>Cat#16140071</td>
</tr>
<tr>
<td>ACCUTASE</td>
<td>STEMCELL Technologies</td>
<td>Cat#07922</td>
</tr>
<tr>
<td>Costar 6-well clear TC-treated plates, individually wrapped, sterile</td>
<td>Corning</td>
<td>Cat#3516</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>An RCT basic package (IKA) hotplate equipped with an ETS-D5 electronic contact thermometer was used in conjunction with a silicone oil bath. Silicone oil may be replaced with silica sand (<a href="#tbl1">Table 1</a>).</p>
<table id="tbl1">
<caption>Table 1. POLYseq vector formulation nomenclature</caption>
<thead>
<tr>
<th>POLYseq nomenclature</th>
<th>Formulation</th>
<th>Reagent ratio D:V:S or D:S (molar)</th>
<th>POLY-ac:capping (mass ratio)</th>
</tr>
</thead>
<tbody>
<tr>
<td>POLY1</td>
<td>D8V5S3C1</td>
<td>1.05:0.7:1</td>
<td>100:75</td>
</tr>
<tr>
<td>POLY2</td>
<td>D8V5S3C2</td>
<td>1.05:0.7:1</td>
<td>100:75</td>
</tr>
<tr>
<td>POLY3</td>
<td>D8V5S3C3</td>
<td>1.05:0.7:1</td>
<td>150:250</td>
</tr>
<tr>
<td>POLY4</td>
<td>D8V5S3C4</td>
<td>1.05:0.7:1</td>
<td>100:50</td>
</tr>
<tr>
<td>POLY5</td>
<td>D8S3C1</td>
<td>1.1:1</td>
<td>100:10</td>
</tr>
<tr>
<td>POLY6</td>
<td>D8S3C2</td>
<td>1.1:1</td>
<td>100:10</td>
</tr>
<tr>
<td>POLY7</td>
<td>D8S3C3</td>
<td>1.1:1</td>
<td>100:20</td>
</tr>
<tr>
<td>POLY8</td>
<td>D8S3C4</td>
<td>1.1:1</td>
<td>100:10</td>
</tr>
</tbody>
</table>
<p class="table-legend">POLY-ac denotes uncapped acrylate backbone.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> ssDNA barcode may be modified to include a barcode of choice. The bold, underlined sequence in <a href="#tbl2">Table 2</a> (<b><u>CATCTTGTGATC</u></b>) may be changed to a sequence of choice. All other sequences of the barcode should remain the same.</div>
<table id="tbl2">
<caption>Table 2. Single-stranded DNA oligonucleotide barcoding sequence</caption>
<thead></thead>
<tbody>
<tr>
<td>5′-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT<b><u>CATCTTGTGATC</u></b><i>B</i>(A)<sub>30</sub>-3′</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Synthesis of POLYseq</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 days]</div>
<p>This section achieves the synthesized POLYseq vector dissolved in DMSO at a final concentration of 200 mg/mL using Michael addition (<a href="#bib4">Mather et al., 2006</a>).</p>
<ol>
<li>If reagent working solutions were stored at −20°C, let reagents warm to 20°C–25°C. Vortex dissolved reagents to ensure proper mixing following thawing. Reagents may be frozen/thawed multiple times without detriment.</li>
<li>The acrylate backbone is created through mixing of the acrylate and alkanolamine monomers at a determined molar ratio. The molar ratio for a sample vector (POLY2) is as follows: 1.05:0.7:1. This equates to a mass ratio of 349.5:434.3:100 mg. The required mass of each component is calculated using equation 1. Here, V5 (Di(trimethylolpropane) tetraacrylate) is used in the example calculation:</li>
</ol>
<p class="math-ml"><math> <mrow> <msub> <mi>M</mi> <mrow> <mi>V</mi> <mn>5</mn> </mrow> </msub> <mrow> <mo>(</mo> <mi>mass</mi> <mo>)</mo> </mrow> <mo>=</mo> <mfrac> <mrow> <msub> <mi>X</mi> <mrow> <mi>V</mi> <mn>5</mn> </mrow> </msub> <msub> <mi>MW</mi> <mrow> <mi>V</mi> <mn>5</mn> </mrow> </msub> <mtext>∗</mtext> <msub> <mi>M</mi> <mrow> <mi>Scaled, S</mi> <mn>3</mn> </mrow> </msub> </mrow> <mrow> <msub> <mi>X</mi> <mrow> <mi>S</mi> <mn>3</mn> </mrow> </msub> <msub> <mi>MW</mi> <mrow> <mi>S</mi> <mn>3</mn> </mrow> </msub> </mrow> </mfrac> </mrow> </math></p>
<p>X<sub>V5</sub> = molar ratio of V5 (0.7), MW<sub>V5</sub> = molecular weight of V5 (466 g/mol), X<sub>S3</sub> = molar ratio of S3 (1), MW<sub>S3</sub> = molecular weight of S3 (75.11 g/mol), M<sub>Scaled,S3</sub> = scaled mass of S3 (100 g).</p>
<div class="note">
<span class="note-title">Note:</span> This equation may be used to calculate the required masses of all other components in the backbone through substitution of the respective mass ratios and molecular weights. The final scaled mass of S3 (3-amino-1-propanol) may be chosen as desired. <a href="#tbl1">Table 1</a> lists mass ratios used during synthesis of respective POLYseq vectors.</div>
<ol start="3">
<li>Add all backbone components (acrylate monomers and alkanolamine) into glass vial with suitable airtight cap. 12 x 75 mm borosilicate glass culture tubes with corresponding polypropylene (PP) plastic caps may be used.</li>
</ol>
<p><b>Caution:</b> Take care if using Polystyrene (PS) caps as DMSO may react with the cap.</p>
<ol start="4">
<li>Mix the monomers either by pipetting or through vortex mixing for 30 s to achieve homogeneity. Vortex mixing is recommended. Inhomogeneity may be visually observed by the presence of fine stratified layers.</li>
<li>Heat the mixture in an oil or sand bath affixed to a temperature controllable hotplate at 90°C for 20 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After 20 hours, an aliquot may be directly taken from the synthesis tube and dissolved in deuterated DMSO (DMSO-d6) for optional <sup>1</sup>H NMR. Adjust the volume added in Step 5 accordingly.</div>
<div class="note">
<span class="note-title">Note:</span> Uncapped POLYseq backbone will show NMR peaks at δ 6.2–5.6 (<a href="#fig1">Figure 1</a>A). Successful capping will eliminate these resonant peaks (<a href="#fig1">Figure 1</a>B). <sup>1</sup>H NMR was acquired on a Bruker Ascend 600 MHz spectrometer. Acquisition may be easily acquired on spectrometers with other operational frequencies.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1213-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Typical <sup>1</sup>H NMR (d6-DMSO with 0.03% v/v TMS, 600 MHz) spectrum form acquisition of uncapped POLYseq (backbone)</div>
<p><sup>1</sup>H NMR spectrum demonstrates resonance at δ 6.2–5.6 from terminal acrylate groups (a) and POLY2 (Capped) showing absence of terminal acrylate groups following successful capping (b).</p>
</figcaption>
</figure>
<ol start="6">
<li>Adjust the hotplate to 50°C, remove backbone from hot bath.</li>
<li>Allow the backbone to cool to 25°C and add respective amount of desired caping agent dropwise while vortex mixing. The mass capping ratio of POLY2 is 100:75 (backbone:cap). For the above example in Step 6, this equates to 662.8 mg of spermine (M<sub>C2</sub>) and is calculated using the summed masses of D8 (M<sub>D8</sub>), V5 (M<sub>V5</sub>), and S3 (M<sub>S3</sub>) in equation 2.</li>
</ol>
<p class="math-ml"><math> <mrow> <msub> <mtext>M</mtext> <mrow> <mtext>C</mtext> <mn>2</mn> </mrow> </msub> <mrow> <mo>(</mo> <mtext>mass</mtext> <mo>)</mo> </mrow> <mo>=</mo> <mrow> <mo>(</mo> <msub> <mtext>M</mtext> <mrow> <mtext>D</mtext> <mn>8</mn> </mrow> </msub> <msub> <mrow> <mo>+</mo> <mtext>M</mtext> </mrow> <mrow> <mtext>V</mtext> <mn>5</mn> </mrow> </msub> <msub> <mrow> <mo>+</mo> <mtext>M</mtext> </mrow> <mrow> <mtext>S</mtext> <mn>3</mn> </mrow> </msub> <mo>)</mo> </mrow> <mtext>∗</mtext> <mfrac> <mn>75</mn> <mn>100</mn> </mfrac> </mrow> </math></p>
<ol start="8">
<li>Heat the mixture at 50°C for 24 h after addition of capping agent to generate the final capped POLYseq vector. An aliquot may be directly taken from the synthesis tube after 24 h and dissolved in deuterated DMSO for optional <sup>1</sup>H NMR.</li>
<li>The final POLYseq concentration is 200 mg/mL in DMSO if all starting reagents were initially dissolved in DMSO at 200 mg/mL. Directly dissolve capped POLYseq in 25 mM HEPES, pH = 7.4 at a final concentration of 10 mg/mL to achieve the working solution (50 μL DMSO stock in 950 μL HEPES buffer). Working solution can be further diluted with 25 mM HEPES buffer if desired.
<div class="optional">
<span class="optional-title">Optional:</span> DyLight NHS Ester may be conjugated to 10 mg/mL working solution by the addition of 1 μL of DyLight NHS stock solution to each 100 μL of 10 mg/mL POLYseq working solution. <b><i>Vortex</i></b> unconjugated POLYseq during addition of DyLight NHS Ester. <b><i>Maintain</i></b> POLYseq at 20°C–25°C in the dark for 30 min following DyLight NHS Ester addition to complete conjugation. <i><b>Aliquot</b></i> as desired and freeze unused POLYseq at −20°C.</div>
<div class="note">
<span class="note-title">Note:</span> Other NHS Ester dyes may be conjugated to POLYseq but may require optimization of conjugation conditions.</div>
<div class="note">
<span class="note-title">Note:</span> POLYseq vectors may be stored directly in DMSO at −20°C without detriment to performance for longer than one year. Vectors can be directly dissolved in 25 mM HEPES buffer, pH = 7.4 at a final concentration of 10 mg/ml and stored at −20°C. Binding ability of vectors should be checked by agarose gel electrophoresis if vectors are stored dissolved in HEPES buffer for longer than six months.</div>
<div class="note">
<span class="note-title">Note:</span> Free induction decay (FID) files generated by NMR by be loaded into an academic licensed version of TopSpin (Bruker) for free.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure the correct hot bath temperature prior to backbone synthesis and capping. Ensure homogeneous mixing during synthesis and dropwise addition of the capping agent.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> POLYseq stock solution may be stored in DMSO at −20°C for longer than one year.</div>
</li>
</ol>
<h3 id="sec3.2">Testing POLYseq barcode binding performance</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h, 30 min]</div>
<p>This section checks the ability for POLYseq vectors to bind and retain ssDNA barcodes following synthesis using agarose gel electrophoresis. Optional DyLight conjugation may be tested using electrophoresis and a gel imaging station with fluorescent capability.</p>
<ol start="10">
<li>Create a 2.5 % agarose gel by dissolving 2.5 grams of agarose, low electroendoosmotic (EEO) in 100 mL of TAE buffer (0.5× concentration).
<ol type="a">
<li>Following heating to dissolve agarose, add 10 μL of 10,00× SYBR Safe stock solution prior to casting the gel. A BioRad gel station with a 20-well comb setup was used for casting. Alternate casting stations may be used.</li>
</ol>
</li>
</ol>
<ol start="11">
<li>POLYseq vectors may be examined at a range of mass ratios (w/w). A w/w of 10 indicates 10 μg of POLYseq for 1 μg ssDNA barcode. For synthesis of POLY2, w/w ratios of 0, 2, 5, 10, 20 may be used. If imaging DyLight conjugated POLYseq, skip every other well. w/w = 0 corresponds to a control well without POLYseq (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1213-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Gel electrophoresis (2.5% w/v) at various w/w ratios</div>
<p>(A) DyLight 650 fluorescence acquired by a ChemiDoc MP showing DyLight 650 band migration in conjugated POLYseq wells (POLY1, POLY2, POLY3, POLY4) compared to Control (DyLight only).</p>
<p>(B) ssDNA barcode migration captured by a ChemiDoc MP using SYBR Safe diluted directly into the gel during casting. a) free ssDNA (control). b) partially complexed ssDNA with POLYseq demonstrating impeded ssDNA migration.</p>
<p>c)Fully complexed ssDNA with POLYseq demonstrating band migration reversal. Various POLYseqs mirror formulations listed in <a href="#tbl1">Table 1</a>.</p>
</figcaption>
</figure>
<ol type="a">
<li>Dilute 10 mg/mL POLYseq solutions to 2 mg/mL with 25 mM HEPES.</li>
<li>Create a 1 mg/mL (1 μg/μL) solution of ssDNA barcode in 25 mM HEPES.</li>
<li>For select w/w ratios, set up the following conditions.
<table id="undtbl2">
<thead>
<tr>
<th>Mass ratio</th>
<th>0×</th>
<th>2×</th>
<th>5×</th>
<th>10×</th>
<th>20×</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>HEPES (μl)</b></td>
<td>12.5</td>
<td>11.5</td>
<td>10</td>
<td>7.5</td>
<td>2.5</td>
</tr>
<tr>
<td><b>ssDNA Barcode (μl)</b></td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td><b>POLYseq (μl)</b></td>
<td>0</td>
<td>1</td>
<td>2.5</td>
<td>5</td>
<td>10</td>
</tr>
</tbody>
</table>
</li>
<li>Wait 10 min following mixing and add 2.3 μL of loading dye without SDS
<div class="note">
<span class="note-title">Note:</span> Take care to use loading dye without SDS as SDS interferes with POLYseq/barcode binding.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If testing DyLight conjugation, create a control solution of free DyLight by pipetting 1 μL of stock DyLight solution into 500 μL of HEPES buffer. This is the DyLight Control Solution in the following table. The following control (C) mixes may be made for observation of migration of unconjugated DyLight along with unbound ssDNA for juxtaposition against POLYseq vectors mixed in step 10c:</div>
<table id="undtbl3">
<thead>
<tr>
<th>DyLight control</th>
<th>2× C</th>
<th>5× C</th>
<th>10× C</th>
<th>20× C</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>HEPES (μl)</b></td>
<td>11.5</td>
<td>10</td>
<td>7.5</td>
<td>2.5</td>
</tr>
<tr>
<td><b>ssDNA Barcode (μl)</b></td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td><b>DyLight Control Solution (μl)</b></td>
<td>1</td>
<td>2.5</td>
<td>5</td>
<td>10</td>
</tr>
</tbody>
</table>
</li>
<li>Load mixtures into cast agarose gel and run at 100 V for 30 min. If using DyLight control wells (C), a migrating dye band will be visible. Correctly conjugated POLY2 should show ssDNA barcode retained within loaded wells beginning at a mass ratio of 10×.</li>
</ol>
</li>
</ol>
<ol start="12">
<li>Image ssDNA barcode migration using a ChemiDoc MP or similar gel station capable of imaging SYBR safe.
<ol type="a">
<li>If imaging DyLight conjugated POLYseq, image gel with ChemiDoc MP with pre-set fluorescent mode or similar gel station with fluorescent imaging.</li>
</ol>
</li>
<li>Re-freeze POLYseq working solution and ssDNA aliquots at −20°C. POLYseq working solutions, when dissolved in HEPES, are stable at −20°C for at least 6 months. Avoid freeze-thaw cycles.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> POLYseq vectors may be conjugated with DyLight fluorescent dyes at a conjugation ratio of 10 μg DyLight-NHS : 1 mg POLYseq.</div>
<div class="note">
<span class="note-title">Note:</span> Gel images may be loaded into ImageJ for imaging processing and quantification of fluorescent intensity.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid the use of Sodium dodecyl sulfate (SDS) in loading dye during gel electrophoresis. SDS can cause ssDNA to become unbound from POLYseq. Gel Loading Dye, Purple (6×), no SDS offered by New England Biolabs has been successfully used.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> POLYseq may be stored diluted in HEPES at −20°C without detriment for at least six months.</div>
<h3 id="sec3.3">POLYseq single-cell barcoding</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h]</div>
<p>Single cell transcriptomics offers a high-resolution method to investigate transcribed genes in heterogeneous populations (<a href="#bib6">Shalek et al., 2013</a>), (<a href="#bib8">Stubbington et al., 2017</a>). This section details basic POLYseq barcoding of Hep G2 cells for pooling during 10× single-cell RNA sequencing runs. This protocol is applicable to other cell types and can be run in parallel for the barcoding of other cells. However, specific culture and labeling conditions may need to be tailored for different cell lines/types.</p>
<div class="note">
<span class="note-title">Note:</span> POLYseq labeling for scRNA-seq has so far only been tested on single-cell suspensions of live cells. Compatibility with cell fixation or organelle isolation methods for scRNA-seq may work but have not been fully verified with the currently published methods.</div>
<ol start="14">
<li>Culture Hep G2 cells in Minimal Essential Medium (MEM) supplemented with 10 % FBS in tissue culture treated 6-well 10 cm dishes at 37°C under 5 % CO<sub>2</sub>. Split with confluency reaches 70–80%. If recovering cells from frozen stock, allow for at least one passage prior to barcoding to allow for recovery.</li>
<li>Prepare complexed barcoding vector:
<ol type="a">
<li>Add 1 μg ssDNA barcode to 25 μL Opti-MEM in a 1.5 mL microcentrifuge tube.</li>
<li>Add 10 μg POLY2 to 25 μL Opti-MEM in a new 1.5 microcentrifuge tube.</li>
<li>Mix (a) into (b) with pipette mixing and allow 10 min for complexation.</li>
</ol>
</li>
<li>Remove medium from Hep G2 cultures.</li>
<li>Add 1 mL Accutase into each well of a 6-well plate or 3 mL Accutase into one 10 cm dish.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other enzymatic or enzyme-free cell dissociating reagent may be used.</div>
<ol start="18">
<li>Warm in 37°C cell culture incubator for 5 min.</li>
<li>Gently pipette Accutase solution to dislodge cells from culture dish.</li>
<li>Add 1 mL of dissociated cells to 9 mL of MEM + 10 % FBS to quench Accutase.</li>
<li>Spin cells at 280×<i>g</i> for 3 min to pellet.</li>
<li>Carefully aspirate supernatant using a vacuum flask or pipette and resuspend cells in 2 mL Opti-MEM for cells harvested from 6-well plates or 5 mL for cells harvested from 10 cm dishes.</li>
<li>Count number of live cells/mL on Countess II automated cell counter.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other methods for counting number of live cells may be used.</div>
<ol start="24">
<li>Aliquot 1E6 (1×10<sup>6</sup>) number of live cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For successful 10× sequencing runs, cell viability should be at least 70 %.</div>
<ol start="25">
<li>Spin aliquoted cells at 400×<i>g</i> for 3 min to pellet.</li>
<li>Carefully aspirate supernatant using a vacuum flask or pipette and resuspend cell pellet in 1 mL Opti-MEM.</li>
<li>Aliquot 10 μL of POLYseq barcoding vector prepared in 2c into a new microcentrifuge tube.</li>
<li>Remove a 100 μL aliquot (100k cells) and quickly add to the 10 μL aliquot of prepared POLYseq barcoding vector.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> With 1 μg of prepared POLYseq barcoding vector, up to 500k (500 μl) of cells can be barcoded. Volumes can be linearly scaled so that 100 μl of POLYseq barcodes 1E6 (1×10<sup>6</sup>) cells.</div>
<ol start="29">
<li>Incubate with Hep G2 at 37°C for 5 min. Cell lines with 5 min labeling times have been investigated (<a href="#bib1">Dunn et al., 2021</a>). These cells include: mouse ES, 3T3, B16-F10, human ESH9, HUVEC, and MEG-01. DyLight conjugated POLYseq may be used to identify if 5 min of labeling time is sufficient for new cell lines. When running labeled cells on a flow cytometer, single cells should show &gt;95% of cells labeled relative to an unlabeled control.</li>
<li>Dilute cells with 1 mL of ice-cold PBS + 0.2 % FBS.</li>
<li>Spin cells at 400×<i>g</i> for 3 min.</li>
<li>Carefully aspirate supernatant using a vacuum flask or pipette and resuspend cells with 1 mL ice-cold PBS + 0.2 % FBS. Gently pipette to re-suspend pellet.</li>
<li>Spin cells at 400×<i>g</i> for 3 min.</li>
<li>Carefully aspirate supernatant using a vacuum flask or pipette and resuspend cells with 1 mL ice-cold PBS + 0.2 % FBS. Gently pipette to resuspend pellet.</li>
<li>Spin cells at 400×<i>g</i> for 3 min.</li>
<li>Carefully aspirate supernatant using a vacuum flask or pipette and resuspend cells in 100 μL ice-cold PBS + 0.2 % FBS. Gently pipette to resuspend pellet.</li>
<li>Mix Hep G2 with other barcoded cells.</li>
<li>Count number of live cells/mL on Countess II automated cell counter.</li>
<li>Spin cells at 400×<i>g</i> for 3 min.</li>
<li>Carefully aspirate supernatant using a vacuum flask or pipette and resuspend cells at a final concentration of 1E6 (1×10<sup>6</sup>) live cells/mL. Gently pipette to re-suspend pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Maintain cells on ice prior to running on the 10× Genomics Chromium Single Cell 3′ GEM, Library &amp; Gel Bead Kit v3.1 in the following step.</div>
<h3 id="sec3.4">POLYseq single-cell library generation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1–2 days]</div>
<div class="critical">
<span class="critical-title">Critical:</span> Cell viability should be at least 70 % for generation of high quality libraries.</div>
<ol start="41">
<li>Generate standard DNA libraries from cells in <b>step 41</b> using the 10× Genomics Chromium Single Cell 3′ GEM, Library &amp; Gel Bead Kit v3.1 (<i>Chromium Next GEM Single Cell 3′ Reagent Kits v3.1</i>, Document Number CG000204 Rev D, 2019, November 22) (<a href="https://assets.ctfassets.net/an68im79xiti/1eX2FPdpeCgnCJtw4fj9Hx/7cb84edaa9eca04b607f9193162994de/CG000204_ChromiumNextGEMSingleCell3_v3.1_Rev_D.pdf">https://assets.ctfassets.net/an68im79xiti/1eX2FPdpeCgnCJtw4fj9Hx/7cb84edaa9eca04b607f9193162994de/CG000204_ChromiumNextGEMSingleCell3_v3.1_Rev_D.pdf</a>)</li>
<li>Follow CG000204 up until Step 2.2</li>
<li>Add 1 μL of 10× Additive Primer (0.2 μM). Subtract this volume from the volume of cDNA Primers (Step 2.2a).</li>
<li>Follow the Cell Hashing Protocol (CITE-seq) beginning at <b><u>After cDNA amplification</u></b> (Document Number, 2019-02-13) employed for amplification and isolation of ssDNA barcodes (<a href="#bib7">Stoeckius et al., 2018</a>) (<a href="https://citeseq.files.wordpress.com/2019/02/cell_hashing_protocol_190213.pdf">https://citeseq.files.wordpress.com/2019/02/cell_hashing_protocol_190213.pdf</a>).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Fluorescent labeling of cells is an optional method for microscopy evaluation of POLYseq colocalization within cell populations or to fluorescently label cells of interest. Fluorescent labeling is separate from single-cell barcoding.</div>
<h3 id="sec3.5">POLYseq fluorescent cell labeling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [16 h <b>to</b> <b>3 days]</b>
</div>
<p>This section details how to fluorescently label cell cultures with DyLight conjugated POLYseq which may be directly imaged with widefield or confocal microscopy or analyzed by a flow cytometer. In this section, fusion of anterior and posterior gut cultures derived using established protocols is used (<a href="#bib2">Koike et al., 2019</a>). This method is applicable to other cell types. Human liver organoids have been successfully stained directly in culture. (<a href="#bib5">Ouchi et al., 2019</a>), (<a href="#bib3">Koike et al., 2021</a>). Other cell types may require optimization of incubation time during labeling.</p>
<ol start="45">
<li>Culture anterior and posterior gut cells in 6-well cell culture treated dishes.</li>
<li>Spike DyLight conjugated POLYseq directly into cell culture medium to a final concentration of 20 μg/mL. Each distinct culture should receive a distinct DyLight color.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If performing flow cytometry of mixed cultures, maintain an unlabeled population as a control.</div>
<ol start="47">
<li>Culture cells at 37°C, 5 % CO<sub>2</sub> for 24 h.</li>
<li>Aspirate cell culture solution containing DyLight conjugated POLYseq.</li>
<li>Wash cells twice with DMEM/F-12 to remove unbound POLYseq.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cells may be directly imaged at this time to check for labeling (<a href="#fig3">Figures 3</a>A and <a href="#fig4">4</a>B).</div>
<ol start="50">
<li>Add 1 mL Accutase into each well of a 6-well plate or 0.5 mL to each well of a 24-well plate.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1213-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Quantification and localization of POLYseq labeling in organoids</div>
<p>(A) Typical FACS analysis of Anterior and Posterior spheroids tagged DyLight conjugated POLYseq (POLY2) demonstrating string single staining and low double staining following spheroid fusion.</p>
<p>(B) Typical intracellular fluorescent distribution at 3 h post tagging with DyLight 488 conjugated POLY2 in Human Liver Organoid (HLO) culture. Scale bar = 50 μm. Scale bar of inset images = 10 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other enzymatic or enzyme-free cell dissociating reagent may be used.</div>
<ol start="51">
<li>Warm in 37°C cell culture incubator for 5 min.</li>
<li>Gently pipette Accutase solution to dislodge cells from culture dish.</li>
<li>Dilute dissociated cells in 4 mL of cell culture buffer to quench Accutase.</li>
<li>Count number of live cells/mL on Countess II automated cell counter.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other methods for counter number of live cells may be used.</div>
<ol start="55">
<li>Aliquot 1E6 number of live cells.</li>
<li>Spin cells at 280×<i>g</i> for 3 min to pellet.</li>
<li>Resuspend cells in mTeSR supplemented with Y-27632 and Laminin-511 at a final concentration of 200,000 cells per mL.</li>
<li>Pipette 100 μL (20,000 live cells) into each well of a 96-well ultra-low attachment U-bottom plate.</li>
<li>Centrifuge plates at 160×<i>g</i> for 2 min to pellet cells.</li>
<li>Allow spheroids to form for 24 h.</li>
<li>Pipette single anterior spheroids into wells containing single posterior spheroids.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If performing flow cytometry of mixed cultures, maintain single color controls.</div>
<ol start="62">
<li>Allow two spheroids to fuse for 24 h.</li>
<li>Fused spheroids may be directly imaged under widefield or confocal microscopy (<a href="#fig3">Figure 3</a>C).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1213-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. POLYseq imaging in iPSC and organoids</div>
<p>(A) Bright field image of iPSC cultures tagged with fluorescent POLYseq (scale bar = 100 μm).</p>
<p>(B) Fluorescent composite of (A) showing successful labeling with POLYseq conjugated with one of three fluorescent colors (DyLight 488, DyLight 550, DyLight 650).</p>
<p>(C) Fluorescent composite of Anterior (white) and Posterior (red) cultures tagged with POLYseq conjugated with DyLight 650 (pseudocolor white) or DyLight 550 (pseudocolor red). Scale bar = 200 μm.</p>
</figcaption>
</figure>
</li>
<li>Dissociate fused spheroids at 24 h following step 17 through digestion with a mixture of 0.9× Accutase + 1.0× TrypLE Express at 37°C with gentle pipetting.</li>
<li>Filter samples through 40 μm mesh following dissociation to remove large debris.</li>
<li>Run unstained, single-stained controls, and multi-stained samples on a flow cytometer capable of detecting conjugated colors for analysis of total labeling and double labeled populations (<a href="#fig4">Figure 4</a>A).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1213-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. POLYseq barcoding based multiplexing for scRNAseq</div>
<p>(A) Barcode detection in HLO cultures visualized in Loupe Browser v4.0.0.</p>
<p>(B) Automatic Seurat clustering (9 clusters) of mixed culture conditions pooled following sample barcoding by POLYseq and subsequent doublet detection based on barcode reads.</p>
<p>(C) Automatic Seurat clustering of a mixed sample comprising 8 POLYseq barcoded cell lines.</p>
<p>(D) Doublet removal and cluster identification of 3T3, B16-F10, ES, ESH9, Hep G2, Hep G2 + high glucose, HUVEC, and MEG-01 cultures based on barcode expression.</p>
<p>(E) Cluster verification based on transcriptome markers for respective cells lines.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If performing high-resolution microscopy of fluorescent-conjugated POLYseq. Counter staining may be performed on live cells using Tetramethylrhodamine, methyl ester at a concentration of 1 μM to stain mitochondria, LysoTracker Blue DND-22 at a concentration of 1 μM to stain lysosomes, and SiR-Actin at a concentration of 500 nM directly in the culture medium for 1 hour (<a href="#fig4">Figure 4</a>B).</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Example barcode detection in single-cell samples following pooling, along with subsequent cell line identification is shown below. Expected fluorescent labeling and sub-cellular localization is demonstrated (<a href="#fig5">Figure 5</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Quantification of backbone capping through 1H NMR is achieved through resonant peak integration between δ 6.2–5.6. Integration should equate to a value of near 0 relative to uncapped backbone (<a href="#fig1">Figure 1</a>).</p>
<p>Quantification of barcode binding percentage by gel electrophoresis may be conducted using ImageJ. Calculation is completed by comparing regions of bound barcode to free barcode (<a href="#fig2">Figure 2</a>A), normalized by their respective integrated areas. Background is taken as a random spot in the fluorescent image without fluorescence from ssDNA. When taking a fluorescent image of ssDNA migration, it is important to take an image without saturation. This method may be used to investigate DyLight conjugation.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>POLYseq barcoding has been successfully tested on a variety of cell lines from model lines to stem cell cultures (<a href="#bib1">Dunn et al., 2021</a>). Barcoding time (Step 28) may require optimization for adaptation to new cell lines or cell types and may be tested in a range of 5–15 min. Successful barcoding may be quantified by flow cytometry where successful labeling of a single-cell suspension shows &gt;95% labeling relative to an unlabeled control. Barcoding has not yet been optimized for barcoding during continuous low temperature conditions (4°C). Barcoding at low temperature conditions may be quickly checked using DyLight conjugated POLYseq in conjunction with flow cytometry. Simple detection of barcode inclusion into cell populations following single-cell dissociation may be probed using quantitative PCR.</p>
<p>Barcoding is non-specific to allow for cell hashing and pooling of heterogenous populations. When pooling multiple samples, excess POLYseq not removed during washing caries the potential to interact with other samples.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Step 7. A highly viscous solution or hard acrylic is achieved following backbone formation.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Verify correct reagent dilutions and initial mixing volumes. Backbone reagents mixed in incorrect molar ratios may produce highly viscous or hard products which cannot mixed with capping reagents in subsequent steps. Diluted backbone reagents are stable for at least one year at −20°C. If kept for longer than one year, fresh dilutions from stock reagents are recommended.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Step 12. Gelatinous material appears in DMSO stock solution following capping.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Specific capping agents may begin to react quickly with the uncapped POLYseq backbone. Care should be taken to slowly add capping agent dropwise while stirring or vortexing the newly synthesized backbone. Inhomogeneity during capping addition can lead to formation of highly crosslinked clumps which appear as gelatinous regions.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Step 14. POLYseq does not dissolve in HEPES buffer.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Incorrect buffer pH, especially if the buffer pH is above 8, may cause precipitation of POLYseq. After establishment of a 25 mM HEPES solution, check pH and adjust to 7.4; pH does not need to be re-checked following sterile filtration.</p>
<p>Ensure correct hot bath temperature during capping. Uncapped POLYseq contains terminal double bonded acrylate groups which severely limit the solubility of uncapped POLYseq in aqueous solutions. If capping temperature is too low, inefficient conjugation of capping agents to the backbone is possible. Correctly capped backbone can be observed by <sup>1</sup>H NMR as the terminal acrylate groups which resonate at δ 6.2–5.6 will disappear.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Step 17. POLYseq is dissolved but does not bind ssDNA barcodes</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Synthesized POLYseq may require different w/w ratios to fully bind ssDNA. Ratios can be adjusted to probe various missing ratios. For POLY2, the onset of binding was found to occur at a w/w ratio of 10.</p>
<p>Sodium dodecyl sulfate (SDS) interferes with complexation of POLYseq with DNA barcodes. Ensure loading dye is SDS free as SDS is commonly included in loading dyes to disrupt DNA/protein interactions.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Step 19. POLYseq no longer binds ssDNA barcodes after storage at −20°C long term</p>
<h3 id="sec7.10">Potential solution</h3>
<p>POLYseq may be stored at −20°C without detriment for at least six months in HEPES buffer and long term as a stock solution in DMSO. Check frozen working solution POLYseq aliquots dissolved in HEPES buffer six months after initial storage as binding ability may deteriorate. Dissolve fresh aliquots from DMSO stock if binding ability of HEPES working solution deteriorates.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Step 35. POLYseq does not sufficiently bind cells of interest under labeling conditions tested.</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Increase labeling time or elevate incubation temperature used during labeling if temperatures below 37°C are used. 5 min has been successfully tested for a range of mouse and human cell types at 37°C. Successful labeling may be quickly checked by running cells labeled with DyLight conjugated POLYseq on a flow cytometer in conjunction with unlabeled control cells.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Dunn, A., Cai, Y., Iwasawa, K., Kimura, M., and Takebe, T. (2021). POLYseq: a poly(B-amino ester)-based vector for multifunctional cellular barcoding. Stem Cell. Rep. <i>16</i>, 2149-2158. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref1">View at publisher</a></p>
<p id="bib2">Koike, H., Iwasawa, K., Ouchi, R., Maezawa, M., Giesbrecht, K., Saiki, N., Ferguson, A., Kimura, M., Thompson, W.L., Wells, J.M., and Takebe, T. (2019). Modelling human hepato-biliary-pancreatic organogenesis from the foregut-midgut boundary. Nature <i>574</i>, 112-116. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref2">View at publisher</a></p>
<p id="bib3">Koike, H., Iwasawa, K., Ouchi, R., Maezawa, M., Kimura, M., Kodaka, A., Nishii, S., Thompson, W.L., and Takebe, T. (2021). Engineering human hepato-biliary-pancreatic organoids from pluripotent stem cells. Nat. Protoc. <i>16</i>, 919-936. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref3">View at publisher</a></p>
<p id="bib4">Mather, B.D., Viswanathan, K., Miller, K.M., and Long, T.E. (2006). Michael addition reactions in macromolecular design for emerging technologies. Prog. Polym. Sci. <i>31</i>, 487-531. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref4">View at publisher</a></p>
<p id="bib5">Ouchi, R., Togo, S., Kimura, M., Shinozawa, T., Koido, M., Koike, H., Thompson, W., Karns, R.A., Mayhew, C.N., McGrath, P.S., et al. (2019). Modeling steatohepatitis in humans with pluripotent stem cell-derived organoids. Cell Metab. <i>30</i>, 374-384.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref5">View at publisher</a></p>
<p id="bib6">Shalek, A.K., Satija, R., Adiconis, X., Gertner, R.S., Gaublomme, J.T., Raychowdhury, R., Schwartz, S., Yosef, N., Malboeuf, C., Lu, D., and Trombetta, J.J. (2013). Single-cell transcriptomics reveals bimodality in expression and splicing in immune cells. Nature <i>498</i>, 236-240. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref6">View at publisher</a></p>
<p id="bib7">Stoeckius, M., Zheng, S., Houck-Loomis, B., Hao, S., Yeung, B.Z., Mauck, W.M., Smibert, P., and Satija, R. (2018). Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. Genome Biol. <i>19</i>, 1-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref7">View at publisher</a></p>
<p id="bib8">Stubbington, M.J., Rozenblatt-Rosen, O., Regev, A., and Teichmann, S.A. (2017). Single-cell transcriptomics to explore the immune system in health and disease. Science <i>358</i>, 58-63. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00682-1/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Takanori Takebe (<a href="mailto:Takanori.Takebe@cchmc.org">Takanori.Takebe@cchmc.org</a>).</p>
<h4>Materials availability</h4>
<p>All materials generated are available through the lead study contact and upon completing relevant Materials Transfer Agreements.</p>
<h4>Data and code availability</h4>
<p>Loupe Browser is readily available from 10× Genomcis. <a href="https://www.10xgenomics.com/products/loupe-browser">https://www.10xgenomics.com/products/loupe-browser</a>).</p>
<p>Seurat may be obtained from Satija Lab and installed from CRAN through RStudio (<a href="https://satijalab.org/seurat/articles/install.html">https://satijalab.org/seurat/articles/install.html</a>).</p>
<p>RStudio is available open source for Windows, Mac, and Linux. (<a href="https://www.rstudio.com/products/rstudio/">https://www.rstudio.com/products/rstudio/</a>).</p>
<p>Single-Cell datasets are publicly available on Gene Expression Omnibus (GEO).</p>
<h3>Acknowledgments</h3>
<p>This work was supported by Cincinnati Children’s Research Foundation grant, Ohio TVSF grant, CCHMC Innovation Acceleration fund, and NIH Director's New Innovator Award (DP2 DK128799-01). This work was also supported by an NIH grant UH3 DK119982, Cincinnati Center for Autoimmune Liver Disease Fellowship Award, PHS Grant P30 DK078392 (Integrative Morphology Core and Pluripotent Stem Cell and Organoid Core) of the Digestive Disease Research Core Center in Cincinnati, the Falk Catalyst Research Awards Program, JST Moonshot R&amp;D Grant Number JPMJMS2033 and JPMJMS2022, Takeda Science Foundation award, Mitsubishi Foundation award and AMED CREST (20gm1210012h0001), JP19fk0210037, JP19bm0704025, JP19fk0210060, JP19bm0404045, and JSPS JP18H02800, 19K22416. T.T. is a New York Stem Cell Foundation – Robertson Investigator. The authors would like to express their gratitude to Drs. Jim Wells, Jorge Bezerra, and Aaron Zorn and the Takebe lab members for their directional support and excellent technical assistance during development of the POLYseq system; Wendy Thompson and Mary Koch for kind administrative/technical assistance; the CCHMC NMR-based Metabolomics Core for access to <sup>1</sup>H NMR instrumentation; the CCHMC Confocal Core for providing access to high-resolution confocal microscopy; the CCHMC Flow Core for flow cytometry instrumentation; and the CCHMC Gene Expression Core for preparation of 10× Chromium libraries and access to a NovaSeq 6000 system for generation of sequencing reads.</p>
<h3>Author contributions</h3>
<p>A.D. designed the POLYseq synthesis, application protocols, and prepared figures. Y.C., K.I., and M.K. optimized cell culture protocols and HLO generation. Y.C. assisted in barcoding optimization. K.I. assisted in fluorescent POLYseq tagging optimization. T.T. and A.D. prepared the manuscript.</p>
<h3>Declaration of interests</h3>
<p>A.D. and T.T. are listed as inventors for a POLY-seq related patent.</p>
</section>
</article>