<article>
<section>
<h1 id="header">Protocol for isolating young adult parvalbumin interneurons from the mouse brain for extraction of high-quality RNA</h1>
<p><time datetime="2021-08-06">Published: August 6, 2021</time></p>
<p>Donald J. Joseph,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup> Markus Von Deimling,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#fn1">4</a></sup> Yuiko Hasegawa,<sup><a href="#aff1">1</a></sup> Ana G. Cristancho,<sup><a href="#aff1">1</a></sup> Rashmi Risbud,<sup><a href="#aff1">1</a></sup> Almedia J. McCoy,<sup><a href="#aff1">1</a></sup> and Eric D. Marsh<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn3">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Division of Child Neurology, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</p>
<p id="aff2"><sup>2</sup>Department of Neurology, Perelman School of Medicine at the University of Pennsylvania, Philadelphia, PA 19104, USA</p>
<p id="aff3"><sup>3</sup>Department of Intensive Care Medicine, University Medical Center Hamburg-Eppendorf, Martinistraße 52, 20246 Hamburg, Germany</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:josephd1@chop.edu">josephd1@chop.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:marshe@chop.edu">marshe@chop.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100714">10.1016/j.xpro.2021.100714</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Dysfunction in the parvalbumin (PV) subclass of GABAergic interneurons is implicated in several neurodevelopmental disorders that evolve in severity with postnatal developmental stages. Understanding the molecular underpinnings of the postnatal changes in the function of PV interneurons has been limited by the difficulty in the isolation of pure adult PV interneurons and high-quality RNA. Here, we describe our protocol for the isolation of pure young adult PV interneurons and preparation of high-quality RNA from these cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib4">Joseph et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol for isolation of fluorescently labeled young adult parvalbumin interneurons</li>
<li>FACS sorting of cortical/hippocampal parvalbumin interneurons</li>
<li>Generation of high-quality RNA for genome-wide sequencing</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/899-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>We have found that various protocols used for the isolation of embryonic and early postnatal interneurons result in low survival of young adult PV interneurons and low-quality RNA. This is likely due to the sensitivity of these young adult interneurons to damage by enzymatic dissociation and mechanical trituration, leading to cell death and low-quality RNA. Therefore, the main improvements in this protocol to increase yield and quality of RNA are the following: preparing acute slices with focus on ensuring viability; using a mild enzymatic and mechanical dissociation; supplementing the dissociating solutions with trehalose, a disaccharide that has been shown to improve cell viability in dissociated neuronal preparations (<a href="#bib7">Saxena et al., 2012</a>).</p>
<h3 id="sec1.1">Experimental considerations</h3>
<ol>
<li>This protocol can also be used for isolation of young adult pyramidal cells and other GABAergic interneuron subtypes. Ensure that your core facility has the instruments compatible for fluorescence-activated cell sorting (FACS) and for RNA integrity assessment before starting.</li>
<li>To ensure success, perform pilot experiments to improve on speed and to set appropriate conditions for FACS.</li>
<li>Wear protective gear (gloves, goggles, masks, lab coat).</li>
</ol>
<h3 id="sec1.2">Generation of transgenic mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 months</div>
<p>The PV<sup>cre</sup>TdTomato mouse is now commercially available from the Jackson lab (027395 - C57BL/6-Tg(Pvalb-TdTomato)15Gfng/J). However, we generated our PV<sup>cre</sup>TdTomato by crossing heterozygous PV<sup>Cre</sup> mice [B6;129P2-Pvalbtm1(cre)Arbr/J; RRID:IMSR_JAX:008069] to homozygous Ai14 mice. If the goal is to compare conditional knock-out (CKO) of a particular gene of interest in PV interneurons to controls, then female PV<sup>Cre</sup>TdTomato double heterozygotes mice can be crossed to male mice with the gene of interest flanked by <i>LoxP</i> sites mice to generate CKO PV<sup>Cre</sup>TdTomato mice and PV<sup>Cre</sup>TdTomato littermate controls. Mice should be genotyped for Cre and TdTomato using end-PCR. We typically use the primers listed in key resources table for Cre and TdTomato (<a href="#bib1">Fulp et al., 2008</a>; <a href="#bib2">Guo et al., 2000</a>). Cre primers should result in gene products of ∼350 bp and the TdTomato primers in gene products of ∼350 bp in PV<sup>Cre</sup>TdTomato mice.</p>
<div class="note">
<span class="note-title">Note:</span> Please make sure to follow all relevant guidelines and regulations set by the National Institutes of Health Guide for the Care and Use of Laboratory Animals and approved by your Institutional Animal Care and Use Committee.</div>
<h3 id="sec1.3">Preparation on the night before the experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–45 min</div>
<ol start="4">
<li>
<b>Set up bubbling station near the vibratome</b>: We use refillable compressed gas cylinders attached to an in-wall line served by our building facilities (<a href="#fig1">Figure 1</a>A, left panel). However, if such infrastructure is not available, attach the line from the regulator of the gas cylinder to a Masterflex Manifold Fittings with Female Luer Locks (Cole-Farmer, SK-30600-43; <a href="#fig1">Figure 1</a>A, red arrow to right panel). Each fitting of the Masterflex Manifold can be connected to gas diffusers and inlet solvent filters using proper size tubing. Note that an additional line with small tubing and no bubbler attached is needed to equilibrate small volumes. Make sure the gas cylinder is secured and the regulator is properly fitted.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/899-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Bubbling station and initial setup for brain slicing</div>
<p>(A) A secured refillable compressed 95 O<sub>2</sub>/5 CO<sub>2</sub> mixed gas tank attached to an in-wall line (Left panel). Right panel shows the various ports at the end of the in-wall line and the connected glass diffusers used to bubble microdissection and cutting solutions. Red arrow in the right panel points to a Masterflex Manifold Fittings setup that can be used as an alternative to an in-wall line.</p>
<p>(B) Cutting solution with ice crystals on the walls of the bottle bubbling on ice.</p>
<p>(C) Photograph of the vibratome setup for the maintenance of a cold cutting environment. Ice is tightly packed in the accessible spaces between the buffer chamber (Dashed arrow) and the vibratome ice tray (Solid arrow). Scale bars: 20 mm.</p>
</figcaption>
</figure>
</li>
<li>
<b>Set an anesthesia induction area near the vibratome:</b> We use a portable isoflurane induction system supplied the Veterinary Department of the Children’s Hospital of Philadelphia. This system is connected to an induction chamber and isoflurane is delivered through a vaporizer with 100% O<sub>2</sub> as a carrier gas. However, if such mobile anesthesia induction system is not available, attach a line from the regulator of a 100% O<sub>2</sub> gas cylinder to a Vaporizer. Connect the vaporizer to an induction chamber and a line from the chamber to a fume hood to allow for scavenging of excess anesthetic.</li>
<li>Make two separate 500 mg/mL trehalose stock solutions (One in EBSS and the other in NB) and store at −20°C. This solution should be used as fresh as possible and not be stored for future preparations.</li>
<li>Make sure gas cylinders (O<sub>2</sub> and mixed O<sub>2</sub>/CO<sub>2</sub>) have enough volume to complete the whole procedure.</li>
<li>Prepare ice dissection trays (2) by filling the chamber outside the 35 mm Sylgard dish with water and freeze at −20°C. See materials and equipment section below for detailed instructions on how to construct dissection trays.</li>
<li>Prepare cutting solution and store overnight at 4°C.</li>
</ol>
<table id="undtbl1">
<caption>Cutting solution (pH 7.35, 310–315 mOsm), 500 mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>180</td>
<td>30.81 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1.25</td>
<td>0.1 g</td>
</tr>
<tr>
<td>KCl</td>
<td>2.5</td>
<td>0.1 g</td>
</tr>
<tr>
<td>Glucose</td>
<td>11</td>
<td>1g</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>26</td>
<td>1.1 g</td>
</tr>
<tr>
<td>NaAscorbate</td>
<td>5</td>
<td>0.5g</td>
</tr>
<tr>
<td>NaPyruvate</td>
<td>3</td>
<td>0.17g</td>
</tr>
<tr>
<td>Thiourea</td>
<td>2</td>
<td>0.08g</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>
</td>
<td>10</td>
<td>5 mL of 1M stock</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>0.5</td>
<td>250μL of 1M stock</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Add to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Use the volumetric flask when adding water to final volume. Do not add MgSO<sub>4</sub> and CaCl<sub>2</sub> to solution until after bubbling with carbogen (95% O<sub>2</sub> /5% CO<sub>2</sub>) for 30 min on the morning of the experiment to saturate cutting solution.</div>
<h3 id="sec1.4">Preparation on the morning of the experiment:</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60–90 min</div>
<ol start="10">
<li>Get the mice from the animal facility and hold them in a cage with bedding, food, and water until anesthesia induction.</li>
<li>Saturate cutting solution with oxygen for ∼ 25–30 min by bubbling with carbogen (95% O<sub>2</sub> /5% CO<sub>2</sub>) then add drugs as suggested above in the critical section (<a href="#fig1">Figure 1</a>A, right panel). Place the saturated solution in a −80°C or −20°C freezer until ice crystals start to form on the surface of the solution and walls of the bottle. This should take about ∼30 min at −80°C and ∼50 min at −20°C.</li>
<li>Fill a bucket with ice and place it next to the bubbling station. Take out the cutting solution out of the freezer and shake it or break ice chunks into a slushy and place the bottle on the ice with constant bubbling using the glass diffuser (See <a href="#fig1">Figure 1</a>B). Place ice around the vibratome slicing chamber to maintain a cold cutting environment (<a href="#fig1">Figure 1</a>C).</li>
<li>Prepare microdissection solution.</li>
</ol>
<table id="undtbl2">
<caption>Microdissection solution (pH 7.35, 310–315 mOsm), 500 mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>115</td>
<td>3.36 g</td>
</tr>
<tr>
<td>KCl</td>
<td>3.5</td>
<td>0.26 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1.25</td>
<td>0.1 g</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>26</td>
<td>1.1 g</td>
</tr>
<tr>
<td>Glucose</td>
<td>11</td>
<td>1.98 g</td>
</tr>
<tr>
<td>NaAscorbate</td>
<td>5</td>
<td>0.5g</td>
</tr>
<tr>
<td>NaPyruvate</td>
<td>3</td>
<td>0.17g</td>
</tr>
<tr>
<td>Thiourea</td>
<td>2</td>
<td>0.08g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>2.5</td>
<td>1.25 mL of 1M stock</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>
</td>
<td>1.0</td>
<td>0.5 mL of 1M stock</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Add to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Use the volumetric flask when adding water to final volume. As in the case of the cutting solution, do not add MgSO<sub>4</sub> and CaCl<sub>2</sub> to microdissection solution until after bubbling with carbogen (95% O<sub>2</sub> /5% CO<sub>2</sub>) for 30 min.</div>
<div class="note">
<span class="note-title">Note:</span> All brain extraction solutions should be made as fresh as possible and not be stored for future use. We also recommend using water of high purity to avoid trace metals and other contaminants that can compromise slice health. Dedicate a set of glassware for fresh tissue preparations and make sure that glassware used, at least the very time, are thoroughly cleaned by rinsing with dilute (0.1N) nitric acid followed by copious amounts of pure water.</div>
<ol start="14">
<li>Turn on the hybridization oven/incubator and set it at 39°C. When our oven/incubator is set at that temperature, the actual chamber is 37°C according to thermometer reading.</li>
</ol>
<ol start="15">
<li>Prepare solutions from papain kit with few modifications to manufacturer’s instructions (Worthington Biochemical Corporation, see <a href="#fig2">Figure 2</a> for reagents included in the kit).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/899-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Components of the papain dissociation system</div>
<p>(A) Photograph of the reagents used in the preparation of enzymatic and dissociation solutions including deoxyribonuclease 1 (5 small glass bottles), papain (5 medium size glass bottles), ovomucoid protease inhibitor (1 large glass bottle), and Earl’s balanced salt solution (1 large plastic media bottle).</p>
<p>(B) Closeup view of these reagents with an already prepared ovomucoid inhibitor-Albumin mixture (Large glass bottle). Scale bars (A) 10 mm and (B) 20 mm.</p>
</figcaption>
</figure>
<ol type="a">
<li>EBSS with bicarbonate and phenol red (vial 1), equilibrate with sterile O<sub>2</sub>: CO<sub>2</sub>
</li>
<li>Ovomucoid protease inhibitor (OPI) with BSA (Ovomucoid inhibitor 10 mg with albumin 10 mg, vial 4)
<div class="note">
<span class="note-title">Note:</span> Add 32mL EBSS (vial 1) to OPI with BSA (vial 4). Allow contents to dissolve. Aliquot OPI into 5mL and keep at 4°C for up to 6 months.</div>
<ol type="i">
<li>
<b><u>Prepare EBSS#1 solution in a 15 mL Falcon tube:</u></b> 9 mL EBSS from papain kit, and 1 mL of 500 mg/mL trehalose in EBSS</li>
<li>
<b>Prepare EBSS#1-papain solution:</b> Add 5 mL of <b><u>EBSS#1 solution</u></b> to one vial of papain and incubate the vial for 10 min at 37°C or until solution appears clear and then transfer it to 15 mL falcon tube to equilibrate with 95/5 O<sub>2</sub>CO<sub>2</sub> until color turns orangeish (see pH scale included in kit). Bubble above the liquid using a tubing line without inlet solvent filter to avoid spillage.</li>
<li>
<b>Prepare DNase 1 solution:</b> Add 500μL of regular EBSS to vial 3 (1000 units of DNAse1) and mix gently (<u>AVOID VIGOROUS MIXING</u>). Keep on ice until use.</li>
<li>
<b><u>EBSS#2-Papain-DNAse1 solution</u></b> <u>(20units/mL papain and 0.005% DNAse)</u>: Add 250μL of DNAse solution to Papain-EBSS#1 solution for a total of 5.25 mL volume just prior to adding dissected tissue.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="16">
<li>Prepare FACS solution and keep it on ice.</li>
</ol>
<table id="undtbl3">
<caption>FACS sorting solution, 20mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (%)</th>
<th>Amount (mL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neural basal (NB)</td>
<td>n/a</td>
<td>15</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>10</td>
<td>2</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA; 30%)</td>
<td>1.5</td>
<td>1</td>
</tr>
<tr>
<td>Trehalose</td>
<td>5</td>
<td>2 mL of 500mg/mL stock</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Do not store this solution for future use.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Papain dissociation system</td>
<td>Worthington</td>
<td>Cat#: LK003150</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Neurobasal A medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: 12348017</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: A3840301</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Sigma-Aldrich</td>
<td>Cat#: A9576</td>
</tr>
<tr>
<td>Trehalose</td>
<td>Sigma-Aldrich</td>
<td>Cat#: 90208</td>
</tr>
<tr>
<td>KCl</td>
<td>Sigma Millipore</td>
<td>Cat#: 7447407</td>
</tr>
<tr>
<td>NaCl</td>
<td>Sigma Millipore</td>
<td>Cat#: 7647145</td>
</tr>
<tr>
<td>Sodium ascorbate</td>
<td>Sigma Millipore</td>
<td>Cat#: A4034</td>
</tr>
<tr>
<td>Sodium Pyruvate</td>
<td>Sigma Millipore</td>
<td>Cat#: P8574</td>
</tr>
<tr>
<td>Thiourea</td>
<td>Sigma Millipore</td>
<td>Cat#: T8656</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma Millipore</td>
<td>Cat#: S7903</td>
</tr>
<tr>
<td>Sodium bicarbonate</td>
<td>Sigma Millipore</td>
<td>Cat#: S5761</td>
</tr>
<tr>
<td>Sodium Phosphate Monobasic</td>
<td>Sigma Millipore</td>
<td>Cat#: S381</td>
</tr>
<tr>
<td>Glucose</td>
<td>Sigma Millipore</td>
<td>Cat#: G7021</td>
</tr>
<tr>
<td>Magnesium Sulfate</td>
<td>Sigma Millipore</td>
<td>Cat#: M7506</td>
</tr>
<tr>
<td>Calcium Chloride</td>
<td>Sigma Millipore</td>
<td>Cat#: C1016</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Agar</td>
<td>Invitrogen</td>
<td>Cat # 30391-023</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Isoflurane</td>
<td>VetOne</td>
<td>Cat#: V1-502017</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Acetone</td>
<td>Fisher Scientific</td>
<td>Cat#: A18-500</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> DEPC-Treated water</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: AM-9915</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> RNA ZAP</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: AM9780</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> TRIzol LS</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: 10296010</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Isopropanol</td>
<td>VWR</td>
<td>Cat#: 76204-296</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Chloroform</td>
<td>VWR</td>
<td>Cat#: IC0219400290</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Sodium acetate (3M)</td>
<td>Sigma Millipore</td>
<td>Cat#: 71196</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Linear Acrylamide</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: AM9520</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Ethanol absolute</td>
<td>VWR</td>
<td>Cat#: 71001-754</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Cre (Forw: GGC GCG CGG TCT GGC AGT AAA AAC</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Rev: CCC TGA TCC TGG CAA TTT CGG CTA TA</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>TdTom (Forw: CTC TGC TGC CTC CTG GCT TCT</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Rev: CGA GGC GGA TCA CAA GCA ATA</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: Rosa-CAG-LSL-tdTomato-WPRE</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:007914</td>
</tr>
<tr>
<td>Mouse: B6; 129P2-Pvalbtm1 (cre)Arbr/J</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:008069</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Vapor pressure osmometer 5520</td>
<td>Wescor</td>
<td>Cat#: SS-140</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Mettler Toledo SevenEasy pH meter</td>
<td>Sigma Millipore</td>
<td>Cat#: Z654272</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Milli-Q® Water Purification System</td>
<td>Sigma Millipore</td>
<td>Cat#: Z00QSV0US</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Solvent inlet filters</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: A-302</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Gas dispersion tube</td>
<td>Sigma Millipore</td>
<td>Cat#: Z408700</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Storage media bottle (500 mL)</td>
<td>VWR</td>
<td>Cat#: 10754-818</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Storage media bottle (100 mL)</td>
<td>VWR</td>
<td>Cat#: 89000-242</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Volumetric flask (500 mL)</td>
<td>VWR</td>
<td>Cat#: 10545-998</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Beaker (800 mL)</td>
<td>VWR</td>
<td>Cat#: 10754-958</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Magnetic Stirring Bars</td>
<td>VWR</td>
<td>Cat#: 58948-025</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Magnetic Stirrer</td>
<td>VWR</td>
<td>Cat#: 10153-314</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Analytical balance</td>
<td>Sartorius</td>
<td>Cat#: BSA124S</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Tubing (1/2” ID)</td>
<td>VWR</td>
<td>Cat#: 89412-478</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Tubing (1/4” ID)</td>
<td>VWR</td>
<td>Cat#: 89412-468</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Tubing (1/8” ID)</td>
<td>VWR</td>
<td>Cat#: 89412-464</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Tubing (1/16” ID)</td>
<td>VWR</td>
<td>Cat#: 89412-462</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fna">a</a></sup> Carbogen: 95% O<sub>2</sub>/5% CO<sub>2</sub>
</td>
<td>Air gas</td>
<td>Cat#: Z02OX952000033</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Leica VT1200S Vibratome</td>
<td>VWR</td>
<td>Cat#: 76001-018</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Oxygen (100%)</td>
<td>Air gas</td>
<td>Cat#: OX 200</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Isoflurane Vaporizer</td>
<td>VWR</td>
<td>Cat#: 89012-506</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Induction chamber</td>
<td>VetEquip</td>
<td>Cat#: 89012-702</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Micro Dissecting scissors</td>
<td>Roboz Surgical Store</td>
<td>Cat#: RS-5914SC</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Forceps</td>
<td>Roboz Surgical Store</td>
<td>Cat#: RS-4902</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Standard Scissors</td>
<td>Fine Science Tools</td>
<td>Cat#: 14014-17</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Scalpel Handle</td>
<td>Roboz Surgical Store</td>
<td>Cat#: RS-9843</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Scalpel Blades</td>
<td>Roboz Surgical Store</td>
<td>Cat#: RS-9801-10</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Bone Rongeurs</td>
<td>Roboz Surgical Store</td>
<td>Cat#: RS-8300</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Tissue Forceps</td>
<td>Roboz Surgical Store</td>
<td>Cat#: RS-8102</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Feather Double Edge Blade</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#: 72002-01</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Flat Squared Tip Spatula</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#: 78326-13</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Long Curved Tip Spatula</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#: 78326-12</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Sable Brushes</td>
<td>Ted Pella</td>
<td>Cat#: 11812</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Filter Paper (42.5 mm)</td>
<td>VWR</td>
<td>Cat#: 28460-019</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Filter Paper (9 cm)</td>
<td>VWR</td>
<td>Cat#: 28313-057</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Loctite Super Glue</td>
<td>Office Depot</td>
<td>Cat#: A11364076</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Crystallizing Dishes</td>
<td>VWR</td>
<td>Cat#: 10754-772</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Transfer pipette</td>
<td>VWR</td>
<td>Cat#: 16001-174</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Nylon Mesh</td>
<td>Warner Instruments</td>
<td>Cat#: 13890-046</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Disposable Beakers (100 mL)</td>
<td>VWR</td>
<td>Cat#: 13890-046</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Sylgard</td>
<td>World Precision Instruments</td>
<td>Cat#: SYLG184</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Norit A Charcoal</td>
<td>Sigma Millipore</td>
<td>Cat#: 53663</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Petri Dish (35 mm)</td>
<td>VWR</td>
<td>Cat#: 10799-192</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Petri Dish (150 mm)</td>
<td>VWR</td>
<td>Cat#: 25382-470</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Petri Dish (60 mm)</td>
<td>VWR</td>
<td>Cat#: 25384-328</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnb">b</a></sup> Glass petri dish (20 mm)</td>
<td>VWR</td>
<td>Cat#: 75845-524</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnc">c</a></sup> Benchtop centrifuge</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: 75009912</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnc">c</a></sup> Falcon Tube (15 mL)</td>
<td>VWR</td>
<td>Cat#: 62406-200</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnc">c</a></sup> Falcon Tube (50 mL)</td>
<td>VWR</td>
<td>Cat#: 21008-940</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnc">c</a></sup> Serological Pipettes (10 mL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: 07-000-376</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnc">c</a></sup> Hybridization Oven/Incubator with Rocker</td>
<td>VWR</td>
<td>Cat#: 97005-252</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnc">c</a></sup> Trypan Blue solution</td>
<td>Sigma Millipore</td>
<td>Cat#: T8154</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnc">c</a></sup> Hemocytometer</td>
<td>Sigma Millipore</td>
<td>Cat#: Z359629</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Micro-centrifuge</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: UX-17703-05</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Bioanalyzer 2100</td>
<td>Agilent</td>
<td>Cat#: G2939BA</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> MoFlo Astrios Cell Sorter</td>
<td>Beckman Coulter</td>
<td>Cat#: B52102</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Nanodrop 8000</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#: ND-8000-GL</td>
</tr>
<tr>
<td>
<sup><a href="#dtbl4fnd">d</a></sup> Eppendorf tube (1.5 mL)</td>
<td>VWR</td>
<td>Cat#: 89166-280</td>
</tr>
</tbody>
</table>
<p id="dtbl4fna" class="table-footnote"><sup>a</sup>For Cutting &amp; Recovery solutions</p>
<p id="dtbl4fnb" class="table-footnote"><sup>b</sup>For Brain Slicing</p>
<p id="dtbl4fnc" class="table-footnote"><sup>c</sup>For Micro-dissection and Dissociation</p>
<p id="dtbl4fnd" class="table-footnote"><sup>d</sup>For FACS and RNA Prep</p>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Animal and dissociation reagents</h3>
<p>Young adult male PV<sup>cre</sup>TdTomato mice postnatal days of age 35–40 (P35-40).</p>
<p>Papain kit with:</p>
<ul>
<li>Earle's Balanced Salt Solution (EBSS) with bicarbonate and phenol red.</li>
<li>Papain containing L-cysteine and EDTA.</li>
<li>Deoxyribonuclease I (DNase).</li>
<li>Ovomucoid protease inhibitor.</li>
</ul>
<p>Neurobasal (NB) A medium without phenol red.</p>
<p>Trehalose (500 mg/mL stock in EBBS and NB).</p>
<p>Bovine Serum Albumin (BSA) solution (30%).</p>
<p>Fetal bovine serum (FBS), heat inactivated.</p>
<div class="note">
<span class="note-title">Note:</span> We used male mice because of our general interest in the role of the X linked Aristaless related homeobox gene in the functional maintenance of maturing GABAergic interneurons, but female mice can also be used for the isolation of PV interneurons using this protocol. While the TdTomato reporter line used in our protocol offers superior brightness, reporters such as mCherry and enhanced GFP amongst others can also be used. In addition, we used NB as sorting and collection buffer since it is the most commonly used medium in neuronal cell culture. Although other types of media have been used successfully in the isolation of embryonic and early postnatal neurons, we strongly recommend prior testing of their efficacy in supporting healthy young adult PV interneurons before a complete run through this protocol. Regardless of the chosen medium, we strongly recommend that it is free of phenol red in order to avoid autofluorescence during FACS sorting.</div>
<h3 id="sec2.2">Materials and Equipment for preparing Cutting and Microdissection solutions</h3>
<p>Glassware and tools for preparing solutions: Storage media bottle (2×, 500 and 100 mL) volumetric flasks (500 mL), large beakers (800 mL), glass dispersion tubes, and magnetic stirrer and bars, solvent inlet filter 1/16” OD, carbogen (95 O<sub>2</sub>/5 CO<sub>2</sub> mixed gas), tubing (½” ¼, 1/8, and 1/16 inner diameters).</p>
<p>Vapor Pressure Osmometer. It must be calibrated frequently, and the thermocouple must be cleaned regularly in order to obtain accurate measurements.</p>
<p>Analytical balance</p>
<p>pH meter: Must be strictly dedicated for fixatives-free usage.</p>
<p>Ultrapure Water: We use the Milli-Q high purity water system from Sigma-Millipore, but other commercially available water systems provide water of comparable purity.</p>
<h3 id="sec2.3">Materials and Equipment for brain slicing</h3>
<p><i><u>Surgical tools</u>:</i> Standard scissors, micro-dissecting scissors, fine forceps, tissue forceps, long curved tip spatula, flat squared tip spatula, scalpel handle and #10 blades, plastic transfer pipette, Loctite super glue, filter paper (9 cm and 42.5 mm diameter), glass petri dish (20 mm), double-edge razor blade, and acetone.</p>
<p><i><u>Agar block:</u></i> Prepare 4% agar block to be used as support in the back of the mounted brain during slicing. We typically prepare it by adding 8 g agar to 200 mL Milli-Q-water and microwave until agar is completely dissolved. Pour it in a 150 mm petri dish and allow it to cure into a block at RT (21°C–23°C) or at 4°C. The agar block can be stored at 4°C for up two months.</p>
<p><i>Ice tray for brain extraction and microdissection:</i> We use a custom-made ice tray with an inner dish with darkened Sylgard to aid with the microdissection (See <a href="#fig3">Figure 3</a>A).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/899-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Custom-made ice tray and multi-slice holder</div>
<p>(A) The ice tray consists of a 35 mm dish filled up to ½ with Sylgard glued to the center of a 60 mm dish. The resultant outer zone of the 60 mm dish is then filled with water and the whole contraption is kept in the -20C until use.</p>
<p>(B) The multi-slice holder was constructed first by cutting the middle third of the wall of two 100 mL Polypropylene disposable beakers, which were then inserted into one another with a Nylon mesh in between. Three screws were attached to the side of the chamber for height and weight in order to prevent slice holder from floating. Scale bars: 20 mm.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Sylgard dishes are commercially available at Living Systems Instrumentations (Cat#: DD-50-S-BLK), but we prepare our own by thoroughly mixing 300g of liquid A and 30g of liquid B of the Sylgard 184 kit. To darken sylgard, we add to 2g of Norit A Charcoal Powder to the mixture and thoroughly mix. The darkened mixture is then poured into a 35mm Petri dishes to about a 1/3 to ½ full and allowed to cure in a dust-free area (e.g., Fume hood) for at least 24 h prior to using. During the pouring process, bubbles will inevitably form, and it is important to get rid of them. This can be done by using a small butane torch above the dish but take care not to overheat Sylgard.</div>
<p><i><u>Anesthesia induction</u></i>: Vaporizer, pure oxygen gas, induction chamber, isoflurane.</p>
<div class="note">
<span class="note-title">Note:</span> For anesthesia induction, we use isoflurane (1–2%) delivered via 100% Oxygen, but other anesthesia cocktails (e.g., mixture of 100 mg/kg ketamine and 10 mg/kg xylazine) can be used. Please make sure that the anesthetic cocktail of choice is approved by your institutional care committee.</div>
<p>Vibratome and multi-slice incubation chamber.</p>
<div class="note">
<span class="note-title">Note:</span> We use the Leica VT1200S, but healthy slices can be obtained by using other models when optimally calibrated. Also, we use a custom-made multi-slice holder placed in a crystallization dish as a slice incubation chamber (See <a href="#fig3">Figure 3</a>B). However, the commercially available Brain Slice Keeper-4 quad (BSK- 4, AutoMate Scientific) and the Brain/Tissue slice system (BSC-PC, cat#: 65–0076; Warner Instruments) incubation chambers are reliable alternatives.</div>
<h3 id="sec2.4">Materials and Equipment for microdissection and tissue dissociation</h3>
<p>Tools &amp; equipment: Scalpel with blade, sable brush, ice tray, tabletop centrifuge, and hybridization oven/incubator with a rocker platform.</p>
<p>Trituration: Falcon tubes (15 and 50 mL) and individually wrapped serological pipettes (10 mL).</p>
<h3 id="sec2.5">Materials and Equipment for FACS and RNA Preparation</h3>
<p><i><u>Reagents</u></i>: RNAse ZAP, DEPC-treated water, Ethanol (100% and 75% in DEPC-treated water), isopropanol, chloroform, and TRIzol LS, low bind Eppendorf tubes (1.5 mL), linear acrylamide.</p>
<p><i><u>Equipment</u></i>: Micro-centrifuge, Nanodrop 8000 and Bioanalyzer 2100, FACS machine.</p>
<div class="note">
<span class="note-title">Note:</span> Please use personal protective equipment when handling the reagents as they may be quite hazardous. Also, we use the MoFlo Astrios (Beckman Coulter), but similar results can be obtained with other cell sorters using proper settings.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Slicing and microdissection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45–60 min</div>
<ol>
<li>Fill another bucket with ice and pour ∼80 mL of the slushy cutting solution in a 100 mL storage bottle and place it in the ice bucket along with a tissue forceps and a pair of standard and micro-dissecting scissors (See <a href="#fig4">Figure 4</a>A for setup). Fill the custom-made or commercial incubation chamber with micro-dissecting solution and place it on ice next to the bubbling cutting solution. Keep this solution saturated using a solvent inlet bubbler line.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/899-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Brain extraction and mounting setup</div>
<p>(A) Ice bucket containing the dissection tools, perfusion syringe with slushy cutting solution, and a 100 mL storage bottle with slushy cutting solution as well.</p>
<p>(B) An extracted brain in an ice tray containing bubbling cutting solution and the arrow points to the inlet solvent filter used for bubbling. This brain was perfused with slushy cutting solution prior to extraction in order to obtain a high-quality image.</p>
<p>(C) Image of the brain following removal of the cerebellum and olfactory bulbs.</p>
<p>(D) Specimen holder with a glued agar block used a bumper support to prevent dislodging of the brain during slicing. Scale bars: (Main panels) 20 mm and (Insets) 5 mm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We generally do not perform transcardial perfusion prior to brain extraction, but it is a common practice that may prove to be beneficial especially if this protocol is applied to aged mice. In such scenario, perfusion can be done manually using a 30mL syringe filled with approximately 25mL of slushy solution. This perfusion syringe should be prepared in advance and kept on ice until processing (See <a href="#fig4">Figure 4</a>A).</div>
<ol start="2">
<li>Place a double-edge razor blade in a 20 mm glass petri dish with acetone for 5 min. Scrape residual glue off the blade and rinse with copious amount of water. Mount the blade on the vibratome (See <a href="#fig1">Figure 1</a>C) and set up the optimal slicing settings for your machine. We cut 350μm-thick slices using an advancing speed of 0.1–0.15 mm/s and 1 mm amplitude to obtain healthy slices with the Leica VT-1200, but slices of similar quality can be obtained using other models, providing that proper settings are used.</li>
<li>Move the ice bucket with the cutting solution containing syringe and 100 mL storage bottle as well as tools to the dissection area. Remove the ice tray from the freezer and rinse the inner dish containing the Sylgard in order to dispose ice from accidental water spillage. Now add the slushy cold cutting solution from the 100 mL storage bottle to the Sylgard dish and bubble it with the inlet solvent diffuser in the dissection area.</li>
<li>Place the mouse in the induction chamber and turn on oxygen tank and vaporizer filled with isoflurane (1–2%) and wait until limb and eye reflexes have disappeared. Note that approved anesthetic cocktails can be used instead of isoflurane (i.e., intraperitoneal injection of the ketamine–xylazine mixture). When preparing slices from two animals (e.g., a control and an experimental) at the same time, we recommend processing each animal separately from anesthesia to brain extraction (See step 7). We do not recommend the processing of more than two animals in the same preparation. Also, a second ice tray is required to keep the brains from the animals apart in order to avoid mix up.</li>
<li>Decapitate the animal using the standard scissors. Place the head in the slushy cutting solution in the ice tray and cut the skin longitudinally along the skull median line and laterally as well using the micro-dissecting scissors in order to fully expose the skull.</li>
<li>Cut the skull bone longitudinally along the median line, advancing from back to front with extreme care in order to prevent damage to the brain with the scissors tip. Make one last transverse cut along the nasal bone, then use the bone rongeurs to pull the skull top from the brain.</li>
<li>Gently scoop the brain using a long-curved tip spatula and cut out the cerebellum and olfactory bulbs while in the slushy bubbling solution (<a href="#fig4">Figures 4</a>B and 4C). Allow the brain to recover for 2–3 min in the ice tray while bubbling using the solvent inlet filter line. At this point, it is possible to begin the brain extraction process for another animal beginning with the anesthesia induction in step 4.</li>
<li>Place a 42.5 mm round filter on the back of the glass petri dish and dampen it with some cold cutting solution. Gently place the brain on the soaked filter paper using a flat squared tip spatula. Remove excess cutting solution using a transfer pipette or 2–3 pieces of filter paper sequentially.</li>
<li>Apply a dab of glue on the surface of the specimen holder in front of the agar block (<a href="#fig4">Figure 4</a>D) and use a strip of filter paper to lift the brain from the back of the petri dish and place it on the glue. Make sure the back of the brain is in contact with the agar block. Note that lifting the brain with the filter paper strip can be done by first folding ¼ inch of the filter to create an L shape strip and touch the surface of the brain with the smaller folded portion to lift it and place the ventral side of the brain on the glue. This approach allows for the slicing of both brain hemispheres in the transverse plane. If you are processing two brains, we recommend mounting the brains in a line to mitigate the risk of mixing the slices as they come off the blade. This arrangement, however, prolongs the slicing time as opposed to mounting side-by-side.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> we generally place a 4% agar block on the specimen holder prior to mounting the brain in order to prevent advancing force of the blade from dislodging it during slicing (See <a href="#fig4">Figure 4</a>D). The brain is mounted between the block and the advancing blade holder. We strongly recommend using such agar block as it serves an important role in generating healthy slices.</div>
<ol start="10">
<li>Place the specimen holder with brain attached to the stage chamber and quickly fill it with ice-slushy cutting solution (<a href="#fig5">Figure 5</a>A). Set up a cutting window if your vibratome allows for it as it significantly reduces the cutting time. Remove the inlet solvent bubbler from the ice tray and place it in the stage chamber as far away as possible from the cutting blade (<a href="#fig5">Figure 5</a>A) and cut the tip of a transfer pipette to create an opening large enough to capture the slices as they come off the blade. Cut 350μm-thick transverse slices. If you are slicing two brains, take the necessary care to prevent mixing them (See steps 4 and 9 for tips).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/899-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Slicing and microdissection setup</div>
<p>(A) Final view of the slicing setup with the mounted on the specimen holder and submerged in bubbling slushy cutting solution.</p>
<p>(B) Slices in bubbling recovering or microdissection solution. The slices are generally kept on ice, but they were briefly taken off the ice for the picture.</p>
<p>(C) Final set up for microdissection with the slices in an ice tray with bubbling microdissection solution, microscope, and an additional light source on the left.</p>
<p>(D) A close up view of the transverse or horizontal slices prior to microdissection of the cortex and hippocampus.</p>
<p>(E) A close up view of the slices following microdissection of the cortex (Dashed arrow) and hippocampus (Solid arrow). Scale bars: (Main panels) 20 mm and (Insets) 5 mm.</p>
</figcaption>
</figure>
</li>
<li>Carefully pick up each slice as they come off the blade using the wide mouth transfer pipette and place the slices inside the bubbling incubation chamber sitting on ice (<a href="#fig5">Figure 5</a>B, pictured off ice for clarity). If you are slicing two brains, make sure to have a dedicated commercial (See note in <a href="#materials-and-equipment">materials and equipment</a>/brain slicing for suggestions) or custom (<a href="#fig3">Figure 3</a>B) incubation chamber for each brain.</li>
<li>Move the slices to an ice tray with bubbling recovery solution for microdissection under a microscope immediately after collecting the last slice from the vibratome (<a href="#fig5">Figure 5</a>C).</li>
<li>Adjust the optical settings on the dissection microscope in order to have clear view of the cortex and hippocampus (<a href="#fig5">Figure 5</a>D). Using a sable brush to gently hold the slice in place, cut the cortex and hippocampus from the subcortical brain structures of both hemispheres with a scalpel blade (<a href="#fig5">Figure 5</a>E). Mince tissue into chunks as small as possible and collect the hippocampal/cortical tissue chunks along with some microdissection solution in a 15 mL Falcon tube and proceed to incubation.</li>
</ol>
<h3 id="sec3.2">Incubation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 90–120 min</div>
<ol start="14">
<li>Remove microdissection solution from 15 mL falcon tube with minced cortical and hippocampal tissue chunks using a transfer pipette and quickly add <b>∼2.6 mL of EBSS#1-papain-DNase mixture</b> to the Falcon tube.</li>
<li>Displace the air in vial with a carbogen tubing line with no inlet solvent filter attached at room temperature (RT; 21°C–23°C) for 2–3 min and immediately cap the vial. <b>Do not bubble gas through the solution.</b>
</li>
<li>Seal the cap of the Falcon tube with parafilm and wrap it in foil. Tape the sealed and wrapped Falcon tube to a rocking platform in the hybridization oven/incubator and <b>incubate for ∼90 min at 37⁰C.</b>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> We highly recommend test runs prior to the final experiment to evaluate the appropriate duration (30–90min) of the incubation. We noticed that the RNA yield was best using a 90min incubation.</div>
<ol start="17">
<li>In the meantime, prepare the following:
<ol type="a">
<li>Set a centrifuge for 15 mL tubes to RT (21°C–23°C) for short spins.</li>
<li>Precoat Eppendorf LoBind Microcentrifuge or 15 mL Falcon tubes for collecting FACS cells by filling them with 30% BSA incubate at RT (21°C–23°C) until after final resuspension. We generally collect our FACS cells in a 15 mL falcon tube.</li>
<li>
<b><u>EBSS#2 (RESUSPENSION) solution in a 50 mL Falcon tube</u></b>: <i>9.6 mL EBSS + 700μL OPI + 500μL DNAse 1, and 1.2cc 50% Trehalose in EBSS for a total volume of 12 mL for each brain and place on ice.</i>
</li>
<li>
<b>OPI discontinuous density gradient medium:</b> 5 mL OPI in EBSS + 0.6 mL of 50% Trehalose in EBSS for a total volume 5.6 mL per brain and place on ice.</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Trituration</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–40 min</div>
<ol start="18">
<li>Triturate the mixture 8–10 times using a 10 mL serological pipette. Let the non-dissociated chunks settle for a couple minutes and collect the cloudy suspension for centrifugation in a 15 mL Falcon tube. <b>This is important for FACS efficiency.</b>
</li>
<li>Centrifuge the suspension at <b><u>300</u> <i><u>g</u></i> <u>(∼1800 RPM) for 5 min at RT (21–23°)</u>.</b>
</li>
<li>Discard the supernatant and gently re-suspend the pellet in 3 mL of mixed EBSS#2 solution by trituration with a 1 mL micropipette while avoiding air bubbles.</li>
<li>Slowly and gently layer the cell suspension to the <b><i>OPI discontinuous density gradient medium</i></b> previously prepared and do not agitate. The cell suspension can be transferred to the OPI medium by slowly expelling it from the transfer pipette along the wall of the Falcon tube. This step is critical for a cell suspension with minimal cellular debris.</li>
<li><b>Centrifuge at 70 <i>g</i> (∼800 RPM) for 6 min at RT (21–23°).</b></li>
<li>Discard the supernatant and re-suspend the pellet 2 mL of FACS solution prepared in the morning using a 1 mL micropipette.</li>
<li>
<b><u>Count cells:</u></b> Take 50 μL of cell suspension and add it to 250μL of 0.4% trypan blue. Mix well and mount on hemocytometer. Count the live, unstained cells (Live cells do not take up trypan blue) in 4 corner squares on the hemocytometer. Take the average cell count and multiply by 10,000 (1×10<sup>4</sup>). Multiply by five to correct for the 1:5 dilution from the trypan blue addition. The final value is the number of viable cells/mL in the original cell suspension.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For FACS efficiency, it may be necessary to dilute the cell suspension further if the cell density is too high. We generally aim for a density ≤1<b>×</b>10<sup>7</sup> cells/mL. Staining of surface antigens with antibodies may be done at this point. In addition, cell suspension can be labeled with 4′,6-diamidino-2-phenylindole (DAPI, Cat#: 62247; ThermoFisher Scientific) or propidium iodide to label dead cells, allowing for gating of dead cells from live cells. Alternatively, live cells can be labeled with the membrane permeable dye calcein acetomethoxyis (Cat#: 65-0853-39; ThermoFisher Scientific), which emits bright green fluorescence signals upon cleavage of the acemethoxy group by active cellular esterase. We did not use fluorescent dyes as they prolong the preparation, but they are certainly useful in the first few preparations as they allow for the assessment of progress in generating suspensions with predominantly live and healthy cells.</div>
<h3 id="sec3.4">FACS sorting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="25">
<li>
<b>Prepare FACS collection tubes</b>: Aspirate BSA from 15 mL Falcon tubes or the Eppendorf LoBind Microcentrifuge tubes and add the desired solution for collecting FACS cells (e.g., 0.1–2 mL FACS solution or 750μL TRIzol LS/250μL sorted volume). Note that we generally collect our cells in a 15 mL Falcon tube with a starting volume of 2 mL FACS solution.</li>
<li>Confirm that the flow cytometer tubing was rinsed with one run of RNase free dH<sub>2</sub>O, then bleach followed by RNase free dH<sub>2</sub>O.</li>
<li>Confirm that the nozzle size is set up to ensure smooth passage of neurons. We normally use a 100 μm nozzle with a sheath pressure of 45 psi. We generally excite TdTomato with an argon-ion laser (488 nm) and detect using emission spectra between 564 nm and 606 nm. Finally, limit sorting to 60–75 min per sample.</li>
<li>Confirm the sorting will be conducted at slow speed to prevent damage to neurons.</li>
<li>Take a few cells at the beginning of sorting and observe under a fluorescent microscope to ensure cells are:
<ol type="a">
<li>Well dissociated and viable.</li>
<li>Tomato red positive.</li>
<li>Use proper forward (FSC) and side scatter (SSC) properties to exclude debris and doublets (<a href="#fig6">Figure 6</a>).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/899-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Fluorescence activated cell sorting of PV interneurons with high TdTomato expression</div>
<p>FACS plots from two representative preparations of dissociated neuronal suspension showing modest (A) and high (B) yield of TdTom+ cells. (I) Total neuronal population in both preparations were selected by forward and side scatter profiles. (II) Fluorescence gates were established by comparing TdTomato positive and negative cells. (III) TdTomato positive cells were selected for RNA extraction. (IV) Histograms of fluorescence emission of sorted TdTomato positive cells versus event number.</p>
</figcaption>
</figure>
</li>
<li>check the intensity of the fluorescence and report it.</li>
<li>Gating can be adjusted at this stage to ensure that fluorescence positive cells alone are collected.</li>
</ol>
</li>
<li>Set up a timer at the start of sorting and collect TdTomato+ cells in either Eppendorf micro-centrifuge or 15 mL Falcon tubes with FACS solution or TRIzol LS. If you choose to collect directly in TRIzol LS, limit your collection volume to 250μL as noted above in step 25 because TRIzol LS allows at most a 30% sample volume for optimal RNA isolation as per manufacturer’s instructions (ThermoFisher Scientific). More importantly, restrict total sorting time to 1–2 h to prevent RNA degradation.</li>
<li>After sorting, spin the cells at 140 <i>g</i> (∼1300 RPM) for 10 min at 4°C and discard the supernatant. Cells pellet can be kept at −80°C for until RNA preparation at a later time point. While freezing cells in TRIzol LS at −80°C for up to a month is generally tolerable, we recommend immediate RNA extraction from cells collected in such solution in order to maximize recovery.</li>
</ol>
<h3 id="sec3.5">RNA extraction and purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>We use the TRIzol LS method for RNA extraction and purification with few modifications from the manufacturer’s instructions (ThermoFisher Scientific), but other methods such as the Invitrogen purelink kit can be used with modifications to obtain high quality RNA (<a href="#bib7">Saxena et al., 2012</a>).</p>
<ol start="32">
<li>Preparation:
<ol type="a">
<li>Reagents: Isopropanol, Chloroform, TRIzol LS, RNase Zap, linear acrylamide, DEPC-treated water, sodium acetate (3M), 200 proof ethanol.</li>
<li>75% ethanol in DEPC-treated water on ice.</li>
<li>Turn on centrifuge and set temperature to 4°C</li>
<li>Take out linear acrylamide out of the freezer and keep at RT.</li>
<li>Set up water bath to 37°C.</li>
<li>Clean and wipe tools, working area, and tubes with RNase Zap.</li>
</ol>
</li>
<li>Thaw your FACS sample on ice. If you have a frozen pellet of FACS cells, add 1 mL of TRIzol LS while thawing. Homogenize sample by intermittent vortexing for 2 min and then by pipetting up and down (10×).</li>
<li>Incubate the homogenized sample at room temp for 5 min to permit complete dissociation of the nucleoprotein complex.</li>
<li>Add 267μL of chloroform per 1 mL of TRIzol LS reagent used for homogenization. Cap the tube securely and shake vigorously by hand for 15 s.</li>
<li>Incubate for 15 min at RT (21°–23°) and then spin a centrifuge at 12,000 <i>g</i> for 15 min at 4°C.</li>
<li>Remove the aqueous phase by decanting into a clean new tube. Avoid the organic phase as it contains mostly DNA and, thus, will affect the quality of the extracted RNA.</li>
<li>Add 2μL of linear acrylamide to the aqueous phase and gently mix by inverting the tube.</li>
<li>Add 666μL of 100% isopropanol per 1 mL TRIzol LS used for homogenization and gently mix by inverting the tube for 15 s.</li>
<li>Incubate at RT (21°–23°) for 10 min and then spin at 12,000 <i>g</i> for 15 min at 4°C.</li>
<li>Remove the supernatant without disturbing the RNA pellet and resuspend the pellet in 100μL of DEPC-treated water.</li>
<li>Add 200μL of 100% ethanol and 10μL of 3M sodium acetate to the resuspended RNA. Briefly Vortex to mix the solution and store in a −20°C freezer for at least 1 h.</li>
<li>Spin at 14000 <i>g</i> for 30 min at 4°C and discard the supernatant.</li>
<li>Wash the RNA pellet with 1 mL of 75% cold ethanol (Made with DEPC-treated water) by gently inverting the tube.</li>
<li>Spin at 14000 <i>g</i> for 10 min at 4°C and discard the supernatant. Air dry the pellet for about 5 min to allow residual ethanol to evaporate. Do not allow pellet to over dry as that tends to interfere with solubility.</li>
<li>Resuspend pellet in ∼25μL DEPC-treated water and obtain a rough estimate of RNA yield using a nanodrop or fluorescent based methods (Qubit for example). Accurate yield and quality of RNA can be readily obtained using a bioanalyzer 2100 with a Pico/Nano chip. At this point RNA sample can be stored at −80°C until use for RNAseq or other experimental plans.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Bioanalyzers are expensive, but they are often available for use in Flow Cytometry core facilities for a fee. If your core facility does not have one, RNA integrity can be checked by running some of your sample on a 1% standard agarose gel and examine the ribosomal RNA (rRNA) bands. High integrity RNA should have an upper 28S band of twice the density of the lower 18S band. Both bands should be tight and crisp.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">PV interneurons/TdTomato+ cells density and FACS</h3>
<p>Recent advances in Cre recombination technologies have led to the development of fluorescently labeled GABAergic interneuron subtypes, paving the way for the identification and collection of individual subtype of GABAergic interneurons for molecular profiling (<a href="#bib8">Taniguchi et al., 2011</a>). Significant progress has been made in the molecular profiling of embryonic and early postnatal GABAergic interneurons, partly due to the relative ease in which these immature neurons can be efficiently isolated. Evidently, the complex nature of the extracellular matrix and sensitivity to enzymatic treatment have hampered the isolation of fluorescently labeled adult GABAergic. While emergent studies have described various approaches to overcome the technical barriers limiting the isolation of sufficient healthy GABAergic neurons from mature adult and aging animals (<a href="#bib3">Hrvatin et al., 2018</a>; <a href="#bib5">Mardinly et al., 2016</a>; <a href="#bib6">Paul et al., 2017</a>; <a href="#bib9">Tasic et al., 2016</a>; <a href="#bib11">Zeisel et al., 2015</a>), we ran into difficulties in applying any particular published method when isolating sufficient young adult PV interneurons for the extraction of high integrity RNA. Here, we describe a protocol that leads to highly purified young adult PV interneurons from the PV<sup>cre</sup>TdTomato mouse. Using the papain dissociation enzyme, protective slice preparation, and gentle mechanical dissociation, we typically sort on average more than 670,000 TdTomato+ PV interneurons per brain (n=36 mice) by using proper FACS gating procedures (<a href="#fig6">Figure 6</a>).</p>
<h3 id="sec4.2">RNA quality</h3>
<p>High quality RNA is a limiting factor for genome-wide sequencing and other sequencing method. From our experience, the amount and quality of RNA appears to be dependent on yield, suggesting that RNAse accumulation as a result of excessive cell death affects the quality of RNA extracted from the surviving FACS of young adult PV interneurons. We typically recover on average 2.5 μg/μL of RNA from sorted PV interneurons in each brain. While the integrity of RNA varies from preparation to preparation, we routinely measure RIN scores in the range of 7–9 (<a href="#fig7">Figure 7</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/899-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Total RNA samples from Sorted TdTomato positive interneurons show high integrity scores</div>
<p>(A) RNA samples from eleven different preparations were ran on an Agilent 2100 Bioanalyzer: L, Ladder/marker; A-K denote samples 1–11.</p>
<p>(B–L) The fluorescence plots or electropherograms of samples 1–11, showing lower marker, 18S, and 28S peaks in sequential order.</p>
<p>(M) RIN scores and RNA concentration for the 11 samples. Note that sample 11 or K was our very first preparation and the low RIN was likely due very low TdTomato+ cell yield and unhealthy sorted cells.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is designed for isolation of PV interneurons from freshly dissociated brains from young adult mice (P35-40). While it represents a good starting point for aged mice, modifications may be needed in order to obtain enough viable PV interneurons for high quality RNA. N-Methyl-D-glucamine-based (NMDG) cutting/recovery solution has been described as an improved approach to prepare healthy slices from aged animals (<a href="#bib10">Ting et al., 2014</a>) and should be considered for animals older than P35-40. In addition, alternatives to enzymatic dissociation may need to be considered in addition to modifications to density gradient filtration as the older tissue may lead to higher amount of cellular debris, which can interfere with the flow sorting and RNA quality.</p>
<p>Another limitation is the amount of time required to perform this protocol. As noted in the step-by-step instructions, FACS of healthy young adult PV interneurons requires meticulous processing with careful trituration and density gradient filtration to remove cellular debris and clumps that might otherwise clog the nozzle or impede sorting. Although manual sorting (<a href="#bib6">Paul et al., 2017</a>) circumvents many of these issues, it is limited by the number of cells that can be collected and by the inherent subconscious bias of the selector. However, it must be noted that FACS represents a more non-bias approach for isolating cells for the comprehensive cataloging of their cellular profile. The time-consuming nature of this process and the many centrifugation steps along with sorting itself could contribute to the resultant transcriptomic or proteomic profile of these cells. For that purpose, we highly recommend that control and experimental animals are processed simultaneously.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Incomplete dissociation of tissue or dissociation takes too long (steps 16 and 18).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>If this problem occurs, it is likely that the enzyme-tissue ratio is off or enzymatic digestion is not effective enough.</p>
<p>In the case of tissue-enzyme ratio, it is important to follow manufacturer’s instruction in regard to the amount of enzyme to use per mg of tissue. In addition, we advise using fresh batch of enzymes or adjust to higher concentrations should incomplete dissociation persists. Finally, consider limiting the time enzymes are kept at RT (21°–23°) or 37°C before the dissociation to maximize activity.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Excessive amounts of debris in cell suspension (steps 21 and 24).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>This problem could be due to incomplete enzymatic and/or mechanical disruption or technical issue with OPI gradient.</p>
<p>Due to sensitivity of young adult PV interneurons, we recommend minor adjustment to the trituration steps by adding slightly more mechanical disruption. This should be done by pipetting up and down a bit more with a 10 mL serological pipette rather than using pipettes with smaller openings. We did not use a cell strainer, but the use of proper cell strainer size (∼70 μm or larger) might be a viable option to reduce debris. Note that, in this case, cell suspension should be added gently to the strainer with a 10 mL pipette to reduce the amount of debris seeping through. Smaller mesh size is likely to produce cleaner suspension, but the yield is likely to be lower. Finally, we recommend that cell suspension be layered gently to the OPI discontinuous density gradient solution to prevent complete mixture and excessive dense cellular debris in the cellular fraction upon centrifugation.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Low cell yield in suspension (step 24).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>PV interneurons represent a small percentage of cells in the cortical/hippocampal suspension. Therefore, it is important to have a high cell yield in the suspension in order for enough PV interneurons to be sorted in a timely manner. Some potential reasons for low cell yield in suspension includes issues with slicing/microdissection solutions, slow preparation of slices, incomplete digestion, damaged cells due to prolonged enzymatic dissociation, and aggressive trituration leading to excessive cell death.</p>
<p>We recommend several practice sessions in order to perform all the steps outlined in the protocol in a precise and timely manner. Ideally, brain extraction should take 45 s or less. Mounting of the brain on the vibratome and slicing should be completed in about 20 min. Also, it is critical that the cutting and recovery solutions are properly made (pH measures at 7.3–7.4 and osmolarity at 310–315mOm). Optimize the enzymatic dissociation by using higher enzyme concentrations if the digestion takes too long or lower concentrations if cells are too sensitive. To reduce the impact of prolonged exposure of cells to papain, equal amount of EBSS#2 or FACS solution can be added to the enzyme solution to dilute its concentration just prior to trituration. We elected to triturate without diluting the enzymatic solution because of the mild nature of papain, but we cannot rule out the possibility that it could have improved our preparations even more. Mechanical disruption is important for the dissociation of cells from tissue chunks, but it will also damage cells, so find the proper balance between mechanical disruption force and cell yield. We recommend slightly prolonged trituration with the low mechanical force using a 10 mL serological pipette to achieve that balance. Work to minimize the time spent between steps in the protocol from anesthesia induction to the preparation of RNA from isolated PV interneurons. If slicing continues to be a point of delay, consider transcardial perfusion of the anesthetized mouse with cold cutting solution prior to brain extraction. This approach can decrease body temperature, thereby reducing hypoxic damage to the brain prior to extraction. Finally, we recommend regular confirmation of cell viability at every step between digestion and homogenization.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Low RNA yield (step 46).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>This problem could be due to low cell yield, excessive cell death, and RNase contamination.</p>
<p>See potential solutions for problems 1–3 to increase cell yield. Excessive cell death could lead to the release of RNAse in cell suspension, leading to the isolation of degraded RNA. We routinely generate cell suspension with less than 2% of dead or dying cells. Finally, ensure that your RNA workstation and pipettes are cleaned with RNAse ZAP and dedicated to RNA use only. Given that RNAse ZAP contains a fair amount of detergent, we do not recommend its use in the dissection area nor on tools use for the preparation of acute slices for electrophysiological recordings.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Low RIN Values (&lt;7) (step 46).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>While a RIN value of 7 is reasonable for a lot of applications, we recommend RIN values in the range of 8–9 for RNAseq. The problem with low RIN is likely due to degradation of RNA as a result of RNase contamination.</p>
<p>As mentioned in the solution to problem 4, ensure RNase-free techniques are employed throughout the protocol by using a bench and equipment dedicated to RNA use only. It is important that bacterial DNA preps are not carried out in the RNA area as RNAse is commonly used at high concentrations in such preps. In addition, change gloves frequently, use filter pipette tips, and make all solutions using RNase-free water. Finally, always keep samples and solutions on ice unless otherwise stated.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Fulp, C.T., Cho, G., Marsh, E.D., Nasrallah, I.M., Labosky, P.A., and Golden, J.A. (2008). Identification of Arx transcriptional targets in the developing basal forebrain. Hum. Mol. Genet. <i>17</i>, 3740-3760. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref1">View at publisher</a></p>
<p id="bib2">Guo, H., Mao, C., Jin, X.L., Wang, H., Tu, Y.T., Avasthi, P.P., and Li, Y. (2000). Cre-mediated cerebellum- and hippocampus-restricted gene mutation in mouse brain. Biochem. Biophys. Res. Commun. <i>269</i>, 149-154. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref2">View at publisher</a></p>
<p id="bib3">Hrvatin, S., Hochbaum, D.R., Nagy, M.A., Cicconet, M., Robertson, K., Cheadle, L., Zilionis, R., Ratner, A., Borges-Monroy, R., Klein, A.M., et al. (2018). Single-cell analysis of experience-dependent transcriptomic states in the mouse visual cortex. Nat. Neurosci. <i>21</i>, 120-129. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref3">View at publisher</a></p>
<p id="bib4">Joseph, D.J., Von Deimling, M., Hasegawa, Y., Cristancho, A.G., Ahrens-Nicklas, R.C., Rogers, S.L., Risbud, R., McCoy, A.J., and Marsh, E.D. (2021). Postnatal Arx transcriptional activity regulates functional properties of PV interneurons. iScience <i>24</i>, 101999. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref4">View at publisher</a></p>
<p id="bib5">Mardinly, A.R., Spiegel, I., Patrizi, A., Centofante, E., Bazinet, J.E., Tzeng, C.P., Mandel-Brehm, C., Harmin, D.A., Adesnik, H., Fagiolini, M., and Greenberg, M.E. (2016). Sensory experience regulates cortical inhibition by inducing IGF1 in VIP neurons. Nature <i>531</i>, 371-375. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref5">View at publisher</a></p>
<p id="bib6">Paul, A., Crow, M., Raudales, R., He, M., Gillis, J., and Huang, Z.J. (2017). transcriptional architecture of synaptic communication delineates GABAergic neuron identity. Cell <i>171</i>, 522-539 e520. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref6">View at publisher</a></p>
<p id="bib7">Saxena, A., Wagatsuma, A., Noro, Y., Kuji, T., Asaka-Oba, A., Watahiki, A., Gurnot, C., Fagiolini, M., Hensch, T.K., and Carninci, P. (2012). Trehalose-enhanced isolation of neuronal sub-types from adult mouse brain. Biotechniques <i>52</i>, 381-385. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref7">View at publisher</a></p>
<p id="bib8">Taniguchi, H., He, M., Wu, P., Kim, S., Paik, R., Sugino, K., Kvitsiani, D., Fu, Y., Lu, J., Lin, Y., et al. (2011). A resource of Cre driver lines for genetic targeting of GABAergic neurons in cerebral cortex. Neuron <i>71</i>, 995-1013. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref8">View at publisher</a></p>
<p id="bib9">Tasic, B., Menon, V., Nguyen, T.N., Kim, T.K., Jarsky, T., Yao, Z., Levi, B., Gray, L.T., Sorensen, S.A., Dolbeare, T., et al. (2016). Adult mouse cortical cell taxonomy revealed by single cell transcriptomics. Nat Neurosci <i>19</i>, 335-346. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref9">View at publisher</a></p>
<p id="bib10">Ting, J.T., Daigle, T.L., Chen, Q., and Feng, G. (2014). Acute brain slice methods for adult and aging animals: application of targeted patch clamp analysis and optogenetics. Methods Mol. Biol. <i>1183</i>, 221-242. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref10">View at publisher</a></p>
<p id="bib11">Zeisel, A., Munoz-Manchado, A.B., Codeluppi, S., Lonnerberg, P., La Manno, G., Jureus, A., Marques, S., Munguba, H., He, L., Betsholtz, C., et al. (2015). Brain structure. Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq. Science <i>347</i>, 1138-1142. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00421-4/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Eric D. Marsh (<a href="mailto:marshe@chop.edu">marshe@chop.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>The authors thank Dr. Florin Tuluc from the Flow Cytometry Core at the Children’s Hospital of Philadelphia. We are also grateful for advice and suggestions from members of the Marsh laboratory during the conduct of this research. This work was supported by the <a href="https://doi.org/10.13039/100000065">National Institute of Neurological Disorders and Stroke</a> grant (Grant #: RO1 NS082761 to E.D.M., 2018); The Stiftung zur Förderung der Medizinischen Forschung der <a href="https://doi.org/10.13039/501100002869">Christian-Albrechts-Universität zu Kiel</a> (Fellowship grant to M.V.D., 2017); the <a href="https://doi.org/10.13039/100000071">National Institute of Child Health and Human Development</a> (Grant #: 5U54HD086984 to the Institutional Intellectual Developmental Disabilities Research Center Cellular Neuroscience Core at the Children’s Hospital of Philadelphia, 2019); and the <a href="https://doi.org/10.13039/100001605">Epilepsy Foundation</a> (Postdoctoral fellowship #: 367394 to D.J.J., 2015).</p>
<h3>Author contributions</h3>
<p>Conceptualization, E.D.M., M.V.D., and D.J.J.; methodology, E.D.M., D.J.J, Y.H., A.G.C., and M.V.D.; investigation, D.J.J., Y.H., M.V.D., A.J.M., and R.R.; writing – original draft, D.J.J. and E.D.M.; writing – review &amp; editing, A.G.C., A.J.M., and M.V.D.; visualization, D.J.J., A.G.C., and A.J.M.; supervision, E.D.M.; funding acquisition, E.D.M., M.V.D., and D.J.J.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>