<article>
<section>
<h1 id="header">Derivation of dorsal spinal sensory interneurons from human pluripotent stem cells</h1>
<p><time datetime="2021-02-05">Published: February 5, 2021</time></p>
<p>Sandeep Gupta,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Ken Yamauchi,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Bennett G. Novitch,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> and Samantha J. Butler<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Neurobiology, University of California Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="aff2"><sup>2</sup>Eli &amp; Edythe Broad Center of Regenerative Medicine and Stem Cell Research, University of California Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="aff3"><sup>3</sup>Intellectual &amp; Developmental Disabilities Research Center, University of California Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:butlersj@ucla.edu">butlersj@ucla.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100319">10.1016/j.xpro.2021.100319</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We describe two differentiation protocols to derive sensory spinal interneurons (INs) from human embryonic stem cells (hESCs) and induced pluripotent stem cells (iPSCs). In protocol 1, we use retinoic acid (RA) to induce pain, itch, and heat mediating dI4/dI6 interneurons, and in protocol 2, RA with bone morphogenetic protein 4 (RA+BMP4) is used to induce proprioceptive dI1s and mechanosensory dI3s in hPSC cultures. These protocols provide an important step toward developing therapies for regaining sensation in spinal cord injury patients.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib4">Gupta et al. (2018)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Detailed methods to derive key spinal sensory interneurons from human stem cells</li>
<li>Methods assess the identity and efficiency of sensory interneuron generation <i>in vitro</i>
</li>
<li>Descriptions of troubleshooting common problems during the differentiation protocol</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/455-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days to a week</div>
<p>For background information on the development of the protocol, please refer to <a href="#bib1">Andrews et al., 2017</a> and <a href="#bib2">Andrews et al., 2019</a>.</p>
<ol>
<li>Order the qPCR primers found in the key resource table for quality control and assessment of conversion efficiency.</li>
<li>Acquire critical reagents mentioned in the key resource table.</li>
<li>Acquire/thaw hPSCs and maintain as undifferentiated cultures in mTeSR based medium. Cells need to be passaged at least twice before starting the differentiation.</li>
<li>Prepare all stock solutions listed below and make the necessary working aliquots.</li>
<li>Prepare media using recipes below, only when needed.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Both the ESC maintenance and differentiation media are stable for up to 2 weeks at 4°C. Thus, it is <u>not</u> advisable to prepare all media at once, rather it is better to make it when needed at the specific step of the protocol.</div>
<div class="critical">
<span class="critical-title">Critical:</span> All procedures are performed in a BSL-2 certified laboratory equipped with class II type A2 biosafety cabinets. Cultures are grown and maintained at 37°C with 5% CO<sub>2</sub>.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Goat polyclonal anti-Sox2 (1:1,000)</td>
<td>Santa Cruz Biotechnology</td>
<td>sc-17320</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-Nanog (1:500)</td>
<td>Cell Signaling</td>
<td>D73G4</td>
</tr>
<tr>
<td>Mouse monoclonal anti-Pax6 (1:20)</td>
<td>DSHB</td>
<td>AB528427</td>
</tr>
<tr>
<td>Goat polyclonal anti-Pax3 (1:500)</td>
<td>R&amp;D Systems</td>
<td>AF2457</td>
</tr>
<tr>
<td>Goat polyclonal anti-HoxA5 (1:1,000)</td>
<td>Santa Cruz Biotechnology</td>
<td>sc-13199</td>
</tr>
<tr>
<td>Mouse monoclonal anti-Tubb3 (1:1,000)</td>
<td>BioLegend</td>
<td>801202</td>
</tr>
<tr>
<td>Goat polyclonal anti-Lhx2 (1:250)</td>
<td>Santa Cruz Biotechnology</td>
<td>sc-19344</td>
</tr>
<tr>
<td>Goat polyclonal anti-Isl1 (1:500)</td>
<td>R&amp;D Systems</td>
<td>AF1837</td>
</tr>
<tr>
<td>Mouse monoclonal anti-Lhx1/5 (1:100)</td>
<td>DSHB</td>
<td>AB531784</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-Pax2 (1:500)</td>
<td>Invitrogen</td>
<td>PA5-81235</td>
</tr>
<tr>
<td>Goat polyclonal anti-Robo3 (1:500)</td>
<td>R&amp;D Systems</td>
<td>AF3076</td>
</tr>
<tr>
<td>Goat polyclonal anti-DCC (1:500)</td>
<td>R&amp;D Systems</td>
<td>AF844</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>mTeSR1 complete kit (basal medium plus 5× supplement)</td>
<td>STEMCELL Technologies</td>
<td>85850</td>
</tr>
<tr>
<td>Dulbecco's modified Eagle’s medium/nutrient mixture F-12 (DMEM-F12)</td>
<td>Hyclone</td>
<td>SH30023.02</td>
</tr>
<tr>
<td>Neurobasal Media</td>
<td>Thermo Fisher</td>
<td>21103049</td>
</tr>
<tr>
<td>Iscove's modified Dulbecco's medium (IMDM)</td>
<td>Thermo Fisher</td>
<td>12440053</td>
</tr>
<tr>
<td>Non-essential amino acid (NEAA) (100×)</td>
<td>Thermo Fisher</td>
<td>11140050</td>
</tr>
<tr>
<td>Penicillin/streptomycin (Pen-Strep)</td>
<td>Thermo Fisher</td>
<td>15140122</td>
</tr>
<tr>
<td>Glutamax (100×)</td>
<td>Thermo Fisher</td>
<td>35050</td>
</tr>
<tr>
<td>N2 supplement (100×)</td>
<td>Thermo Fisher</td>
<td>17502-048</td>
</tr>
<tr>
<td>B27 supplement (50×)</td>
<td>Thermo Fisher</td>
<td>17504-044</td>
</tr>
<tr>
<td>B27 supplement without vitamin A (100×)</td>
<td>Thermo Fisher</td>
<td>12587010</td>
</tr>
<tr>
<td>Accutase (100 mL)</td>
<td>STEMCELL Technologies</td>
<td>7920</td>
</tr>
<tr>
<td>ReLeSR</td>
<td>STEMCELL Technologies</td>
<td>05872</td>
</tr>
<tr>
<td>Ascorbic acid</td>
<td>Sigma-Aldrich</td>
<td>A8960-5G</td>
</tr>
<tr>
<td>Geltrex</td>
<td>Thermo Fisher</td>
<td>A1413301</td>
</tr>
<tr>
<td>Human recombinant bone morphogenetic protein 4 (BMP4)</td>
<td>Thermo Fisher</td>
<td>PHC9534</td>
</tr>
<tr>
<td>Retinoic acid</td>
<td>Sigma-Aldrich</td>
<td>R2625</td>
</tr>
<tr>
<td>Rho kinase (ROCK) inhibitor (Y-27632), 10 mM</td>
<td>BioPioneer</td>
<td>SM-008</td>
</tr>
<tr>
<td>Primocin</td>
<td>Invitrogen</td>
<td>Ant-pm-05</td>
</tr>
<tr>
<td>Dulbecco (d) PBS (without calcium, magnesium)</td>
<td>Thermo Fisher</td>
<td>14190250</td>
</tr>
<tr>
<td>Poly-L-ornithine</td>
<td>Sigma-Aldrich</td>
<td>P4957</td>
</tr>
<tr>
<td>Laminin</td>
<td>Sigma-Aldrich</td>
<td>L2020</td>
</tr>
<tr>
<td>Dorsomorphin</td>
<td>Sigma-Aldrich</td>
<td>P5499</td>
</tr>
<tr>
<td>SB431542</td>
<td>Stemgent</td>
<td>04-0010-10</td>
</tr>
<tr>
<td>Dibutyryl cyclic-AMP salt</td>
<td>Tocris</td>
<td>1141</td>
</tr>
<tr>
<td>Hydrochloric acid (HCl)</td>
<td>Millipore-Sigma</td>
<td>1003170510</td>
</tr>
<tr>
<td>Heat inactivated horse serum</td>
<td>Thermo Fisher</td>
<td>16050130</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>X100</td>
</tr>
<tr>
<td>Vectashield mounting medium</td>
<td>Vector laboratories</td>
<td>H-1000-10</td>
</tr>
<tr>
<td>ProLong Gold mounting medium</td>
<td>Thermo Fisher</td>
<td>P36930</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Sigma-Aldrich</td>
<td>A9418-5G</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>H9 hESCs</td>
<td>WA09, NIH registry #0062</td>
<td>N/A</td>
</tr>
<tr>
<td>iPSC lines</td>
<td>(<a href="#bib7">Sareen et al., 2013</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>
<i>HOXA5</i> F-AAGTCATGACAACATAGGCGGR-TTCAATCCTCCTTCTGCGGG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>LHX2</i> F-TGGACCGAGGAACAACTTGGR- TCGCTCAGTCCACAAAACTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>ISL1</i> F- GATTTGGAATGGCATGCGGCR- GCGCATTTGATCCCGTACAA</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>LHX1</i> F- CAACATGCGCGTCATTCAGGR- ACTCGCTCTGGTAATCTCCG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>LHX5</i> F- GCGTCATCCAGGTGTGGTTTR- GGTGGACCCCAACATCTCAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>TUBB3 F- GGCCAAGGGTCACTACACGR- GCAGTCGCAGTTTTCACACTC</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>6-well plate</td>
<td>Corning</td>
<td>490007-412</td>
</tr>
<tr>
<td>Ultra-low-attachment 24-well plate</td>
<td>Corning</td>
<td>CLS3473</td>
</tr>
<tr>
<td>Optimum cutting temperature (OCT) compound</td>
<td>Sakura-Finetek</td>
<td>4583</td>
</tr>
<tr>
<td>Peel-A-Way Embedding molds</td>
<td>Polysciences Inc.</td>
<td>18986-1</td>
</tr>
<tr>
<td>EZ Passage tool</td>
<td>Thermo Fisher</td>
<td>23181010</td>
</tr>
<tr>
<td>RNeasy Plus mini kit (RNA extraction)</td>
<td>QIAGEN</td>
<td>74136</td>
</tr>
<tr>
<td>LightCycler 480 SYBR Green master mix</td>
<td>Roche</td>
<td>04707516001</td>
</tr>
<tr>
<td>QIAshredder</td>
<td>QIAGEN</td>
<td>79656</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Growth factor/cytokine stock solutions</caption>
<thead>
<tr>
<th>Retinoic acid (RA) stock solution (100 mM)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>All trans RA</td>
<td>100 mM</td>
<td>50 mg</td>
</tr>
<tr>
<td>DMSO</td>
<td>-</td>
<td>1.66 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>-</td>
<td><b>1.66 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Retinoic acid (RA) is a light sensitive compound. Work under low-light conditions when working with RA. To make a less concentrated solution, dilute RA stock in 100% ethanol. The 100 mM stock can be stored at −80°C for up to 2 years. The diluted stock can be stored at 4°C in dark and must be used within a week.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Retinoic acid (RA) is a potent teratogen. Dispose concentrated stock solutions responsibly according to the manufacturer’s recommendations.</div>
<table id="undtbl2">
<thead>
<tr>
<th>BMP4 stock solution (10 μg/mL)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BMP4</td>
<td>10 μg/mL</td>
<td>10 μg</td>
</tr>
<tr>
<td>4 mM HCl with 0.1% BSA</td>
<td>-</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>-</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Make 25 μL aliquots and avoid multiple freeze-thaw cycle. Aliquots can be stored at −30°C for up to 1 year.</div>
<table id="undtbl3">
<thead>
<tr>
<th>Ascorbic acid stock solution (10 mg/) mL</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ascorbic acid</td>
<td>10 mg/mL</td>
<td>10 mg</td>
</tr>
<tr>
<td>dPBS</td>
<td>-</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>-</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Ascorbic acid is light sensitive. Protect the stock solution from light. Make 20 μL working aliquots to prevent frequent freeze-thaw. Aliquots can be stored at −20°C for up to 6 months.</div>
<table id="undtbl4">
<thead>
<tr>
<th>Dibutyryl cyclic-AMP stock solution (10 mM)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dibutyryl cyclic-AMP salt</td>
<td>10 mM</td>
<td>10 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>2 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Determine solute required to make 10 mM solution from the manufacturer’s datasheet as batch-to-batch variation may affect the volume of water required to dissolve cAMP. Store aliquots at −20°C for up to 6 months.</div>
<table id="undtbl5">
<caption>Maintenance and differentiation media composition</caption>
<thead>
<tr>
<th>mTeSR1 medium (commercially available)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Basal media</td>
<td>-</td>
<td>400 mL</td>
</tr>
<tr>
<td>5× supplement</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>-</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Store complete media at 2°C–8°C for up to 2 weeks or −20°C for up to 6 months. Avoid warming the medium for extended periods of time in the 37°C water bath, because bFGF is unstable at this temperature.</div>
<table id="undtbl6">
<thead>
<tr>
<th>SaND medium</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>IMDM</td>
<td>-</td>
<td>470 mL</td>
</tr>
<tr>
<td>N2 supplement</td>
<td>1% (v/v)</td>
<td>5 mL</td>
</tr>
<tr>
<td>B27 supplement (-Vit A)</td>
<td>2% (v/v)</td>
<td>10 mL</td>
</tr>
<tr>
<td>Primocin</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>Glutamax (100×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>NEAA (100×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>-</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Store complete medium at 2°C–8°C for up to 2 weeks.</div>
<table id="undtbl7">
<thead>
<tr>
<th>NBD medium (NBDM)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Neurobasal media</td>
<td>-</td>
<td>470 mL</td>
</tr>
<tr>
<td>N2 supplement</td>
<td>1% (v/v)</td>
<td>5 mL</td>
</tr>
<tr>
<td>B27 supplement with vitamin A</td>
<td>2% (v/v)</td>
<td>10 mL</td>
</tr>
<tr>
<td>Primocin</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>Non-essential amino acid (NEAA)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>Glutamax</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>-</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Store complete medium at 2°C–8°C for up to 2 weeks.</div>
<div class="note">
<span class="note-title">Note:</span> The B27 supplement is commercially available both with and without vitamin A (retinyl acetate). This medium requires B27 containing vitamin A, because vitamin A has been shown to enhance neuronal differentiation and maturation in the stem cell cultures.</div>
<table id="undtbl8">
<thead>
<tr>
<th>Maturation medium</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM:F12</td>
<td>-</td>
<td>475 mL</td>
</tr>
<tr>
<td>N2 supplement</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>B27 supplement with Vit A</td>
<td>2%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Pen-Strep</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>Non-essential amino acid (NEAA)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>Dibutyric cyclic-AMP stock solution</td>
<td>1 μM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Ascorbic acid stock solution</td>
<td>200 ng/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>-</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Store complete medium at 2°C–8°C for up to 2 weeks.</div>
<div class="note">
<span class="note-title">Note:</span> medium composition of SaND, NBDM and maturation media was adapted from (<a href="#bib7">Sareen et al., 2013</a>)</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparing hESCs/hiPSCs cultures for neural differentiation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6–10 days</div>
<p>This step permits the expansion of hPSC lines for cryostorage and produces healthy pluripotent colonies of hESCs/ hiPSCs suitable for neural differentiation. hESCs/ hiPSCs are cultured under feeder-free conditions and passaged when cells approach 70% confluency. Cells are passaged at least twice before starting the differentiation.</p>
<ol>
<li>Prepare Geltrex matrix coated plates before starting the hESC/hiPSC culture. One 6-well plate is sufficient to start the differentiation.
<ol type="a">
<li>Thaw Geltrex matrix for 12–16 h at 2°C–8°C. Make smaller aliquots (500 μL-1 mL) using refrigerated pipette tips to avoid multiple freeze-thaw cycles. Aliquots can be stored at −30°C for up to 1 year.</li>
<li>Dilute Geltrex matrix 1:100 in cold DMEM:F12 medium using refrigerated pipette tips to avoid precocious gelling. (For example- 100 μL Geltrex in 10 mL DMEM:F12 media)
<div class="note">
<span class="note-title">Note:</span> If hPSC colonies show enhanced differentiation and low adherence, use lower dilution (1:75) of Geltrex for the maintenance cultures.</div>
</li>
<li>Add 1.5 mL diluted Geltrex to each well of the 6-well plate. Make sure to cover the entire surface of the well.</li>
<li>Incubate plates at 25°C for minimum of 1 h.</li>
<li>Aspirate off Geltrex before adding medium to the wells and rinse with 1× PBS once before use.
<div class="note">
<span class="note-title">Note:</span> Coated plates can be stored at 2°C–8°C for 1 week in DMEM:F12 if wrapped in parafilm to prevent evaporation.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Thaw hESCs/hiPSC lines by warming cryovials containing hESCs/hiPSCs in a 37°C water bath. At the first signs of thawing, cells must immediately be transferred into 15 mL tube containing 10 mL of warm mTesR1 medium to reduce cryo-preservant mediated damage.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Resuspending thawed cells in 10 times volume of the warm mTesR1 medium enhances the cell survival.</div>
<ol start="3">
<li>Spin cells at 112 × <i>g</i> for 5 min.</li>
<li>Resuspend cells in 12 mL mTesR1 medium and transfer 2 mL of the cell suspension into each well of the Geltrex-coated plate.</li>
<li>Change medium every day to reduce probability of spontaneous differentiation (see <a href="#fig1">Figures 1</a>B and 1C for examples of differentiated colonies). Cells will be ready for the first passage in 3–4 days. At this point, the healthy hESCs/hiPSCs colonies will appear as a tight, flat colonies separated from each other (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/455-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Images of healthy and differentiated hESC/hiPSC colonies</div>
<p>(A) Healthy hESC/hiPSC cultures should contain flat hESCs/hiPSC colonies with smooth edges.</p>
<p>(B and C) In contrast, differentiated cultures have colonies that appear as clusters of scattered cells (B) or pigmented mounds (C). These colonies need to be removed before passaging.</p>
<p>Scale bar (all panels), 500 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Unhealthy hESCs/hiPSCs cultures can be distinguished by having more differentiated clones (see examples of differentiated clones in <a href="#fig1">Figures 1</a>B and 1C). In this case, try a) optimizing the Geltrex concentration for the coating, b) use earlier passage cultures, or c) remove the differentiated cells either by scratching them using a sterile pipette tip or careful aspiration with a pipette tip attached to a vacuum line.</div>
<ol start="6">
<li>Once cells reach 70% confluency, start the first cell passage by preparing another Geltrex-coated 6-well plate. This 6-well plate will be directly used for the differentiation.</li>
</ol>
<ol start="7">
<li>Cells are passaged at 1:3 ratio. This way, two wells of the first 6-well plate (starting culture plate) will be passaged into the freshly prepared Geltrex-coated 6-well plate (differentiation plate).
<ol type="a">
<li>Wash wells to be passaged with 2 mL 1× PBS.</li>
<li>Add 1 mL ReLeSR reagent and aspirate after 1 min to ensure that the colonies are only exposed to a thin film of ReLeSR reagent.
<div class="note">
<span class="note-title">Note:</span> ReLeSR circumvents removing the differentiated clones manually, since it only causes the pluripotent stem cell clones to detach, leaving most differentiated clones adhered to the plate.</div>
</li>
<li>Incubate plate at 37°C for approximately 3 min. This incubation step may need to be optimized for the individual cell lines. For example, some hiPSC lines require 2 min while the H9 hESC line requires 3 min.</li>
<li>Add 2 mL mTesR1 medium to the wells treated with ReLeSR reagent. Large fragments of stem cell colonies should be seen floating in the medium. Triturate 2–3 times with a 5 mL glass pipette to break colonies in smaller pieces. Avoid generating a single-cell suspension because it will result in poor cell survival.
<div class="note">
<span class="note-title">Note:</span> glass pipettes are recommended for this step to prevent losing cell clusters during the transfer process. Cells show less attachment to a glass surface.</div>
</li>
<li>Collect medium containing the cell colonies in a 15 mL tube and add extra mTesR1 medium up to 12 mL and gently triturate 2 times for homogeneous distribution of the cell clusters. Transfer 2 mL cell suspension in each well of the newly coated 6-well plate.</li>
<li>Place 6-well plate in incubator at 37°C. mTesR1 medium should be changed daily to avoid spontaneous differentiation.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>The newly passaged 6-well plate should reach 80%–90% confluency in 4–5 days and can be used to start the differentiation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend using hESCs/hiPSCs culture with 2–4 passages for this differentiation protocol. Cultures with more than 5 passages tend to spontaneously differentiate at higher rates.</div>
<div class="note">
<span class="note-title">Note:</span> Cells at 80%–90% confluency should be immediately used in the differentiation procedure. Leaving cells at higher confluency for long periods (&gt;1–2 days) can lead to spontaneous differentiation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure there are no differentiated colonies in these expanding cultures. These colonies can be identified by the apparent scattered morphology of cells (arrows, <a href="#fig1">Figure 1</a>B) or as pigmented clustered cells (arrows, <a href="#fig1">Figure 1</a>C). Remove the differentiated clones by scratching them using sterile pipette tip or careful aspiration with pipette tip attached to vacuum.</div>
<h3 id="sec3.2">Two-dimensional differentiation to the neuroectoderm stage</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 days</div>
<p>The next step is to direct pluripotent hESCs/ hiPSCs toward a spinal neuroectodermal fate, which can be assessed by the generation of SOX1<sup>+</sup>/ PAX6<sup>+</sup> neural progenitors. For more details see (<a href="#bib4">Gupta et al., 2018</a>).</p>
<ol start="9">
<li>Prepare SaND medium and pre-heat in 37°C water bath before use.</li>
<li>Remove 6-well plate prepared in step 8 from the incubator.</li>
<li>Aspirate mTeSR1 medium from all wells of the 6-well plate.</li>
<li>Add 2 mL SaND medium to each well of the 6-well plate and return the plate to the incubator.</li>
<li>Feed cells every other day with SaND medium for up to 6 days.</li>
<li>Cells should appear as a monolayer with cells packed together tightly at the end of day 6 (<a href="#fig2">Figure 2</a>A). This confluency is optimal for embryoid body (EB) formation in the next step.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/455-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Formation of embryoid bodies</div>
<p>(A) By day 6 in SaND medium, the cells have formed a tight monolayer.</p>
<p>(B) Embryoid bodies are formed after these cells are dissociated and seeded in ultra-low attachment plate as a single-cell suspension.</p>
<p>(C) Large EBs should be observed in the culture, by day 3 after the seeding into the ultra-low attachment plate.</p>
<p>Scale bar (all panels), 500 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See “<a href="#troubleshooting">Troubleshooting</a>” section below</div>
<div class="optional">
<span class="optional-title">Optional:</span> After 6 days, plate can be fixed for immunohistochemistry or RNA can be isolated from the cells to check for the expression of neuroectodermal markers, i.e., SOX1 and PAX6.</div>
<h3 id="sec3.3">Embryoid body formation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>The next step is to grow the neuroectodermal cells as embryoid bodies (EBs) in suspended 3d culture. One 6-well differentiation plate, prepared in step 14, can be used to seed four 24 well plates. This protocol requires ultra-low attachment 24 well plates which prevent cells from attaching to the bottom of the plate and allow 3d embryoid body formation. Plan the experiment according to the number of 24 well plates required for all necessary end points.</p>
<ol start="15">
<li>Prepare enough SaND medium with 1 μM RA, and 10 μM ROCK inhibitors.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> RA stock is made in DMSO at 100 mM concentration. Dilute the stock to 1 mM in 100% ethanol. This working stock can be stored at 4°C and can be used for up to 1 week.</div>
<div class="note">
<span class="note-title">Note:</span> RA is both light sensitive and a potent teratogen. All work should be performed under low-light conditions and the concentrated RA stock must be disposed of properly.</div>
<div class="note">
<span class="note-title">Note:</span> The hESCs are vulnerable to apoptosis upon dissociation which leads to poor EB formation. Adding ROCK inhibitor Y-27632 to the cell dissociation medium has been shown to significantly inhibit apoptosis and enhance the EB formation (<a href="#bib8">Watanabe et al., 2007</a>, <a href="#bib6">Pettinato et al., 2015</a>). The concentration of ROCK inhibitor needed, may have to be empirically determined with different hESCs/hiPSCs lines, if poor EB formation is observed.</div>
<ol start="16">
<li>Pre-heat SaND medium in 37°C water bath.</li>
<li>Remove 6-well differentiation plate, prepared in step 14, from the incubator.</li>
<li>Aspirate SaND medium from the well and wash cells once with 1× PBS.</li>
<li>Add 1 mL Accutase to each well of the 6-well plate and incubate at 37°C for 10 min.</li>
<li>Remove plate from the incubator and gently tap the bottom of the plate to dislodge the cell clusters.</li>
<li>Add 2 mL of SaND medium + 1 μM RA with 10 μM ROCK inhibitors in each well and collect the cell suspension.</li>
<li>Triturate the cell suspension 1–2 times using a glass pipette to break apart the larger cell clusters into a single-cell suspension.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> glass pipettes are recommended for this step to prevent losing cell clusters during the transfer process. Cells show less attachment to a glass surface.</div>
<ol start="23">
<li>Equally distribute cell suspension in two 50 mL tubes. Centrifuge cells at 112 × <i>g</i> for 5 min. Aspirate liquid and gently resuspend cells in 50 mL SaND medium with 1 μM RA + 10 μM ROCK inhibitors. The total cell suspension at this stage will be 100 mL.</li>
<li>Transfer 1 mL of cells in SaND medium into each well of the ultra-low attachment 24 well plate to promote EB formation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ultra-low attachment plates have a specialized surface coated with a hydrophobic hydrogel which inhibits cell attachment and forces them to remain in the suspension to make embryoid bodies (EBs).</div>
<div class="note">
<span class="note-title">Note:</span> Embryoid bodies (EBs) are 3-dimensional stem cell aggregates. Stem cells differentiate better as EBs because they recapitulate the cell-cell interactions found in growing tissues.</div>
<ol start="25">
<li>Return 24-well plates to 37°C. After 2 days, the resuspended cells should have formed small EBs in each well (see <a href="#fig2">Figures 2</a>B and 2C).</li>
<li>Change the medium on EB cultures on the next day and then every 2 days.
<ol type="a">
<li>Tilt the ultra-low attachment plate to collect the EBs against the wall of the well. This will prevent EBs from drying out.</li>
<li>Gently aspirate medium, without disturbing the EB pellet.</li>
<li>Add fresh 1 mL SaND medium+ 1 μM RA (without ROCK inhibitor) in each well.</li>
<li>Alternatively, medium can be changed by collecting all the EBs in one 50 mL conical tube and allow EBs to settle at the bottom of the tube by gravity. This process takes 5–10 min.</li>
<li>Add 50 mL of fresh SaND medium with 1 μM RA in the tube and resuspend the EBs.</li>
<li>Transfer 1 mL EBs + medium back to the ultra-low attachment 24-well plate.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is <u>not</u> required to use fresh ultra-low attachment plate with every media change.</div>
<div class="note">
<span class="note-title">Note:</span> ROCK inhibitor is only required on the day of EB formation, i.e., steps 21 and 23. The SaND medium that is used to feed EBs does not require ROCK inhibitors.</div>
<h3 id="sec3.4">Induction of dorsal sensory interneurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days</div>
<p>This step directs 3d EBs toward different cell fates, i.e., dI4/6 (<b>protocol 1</b>) or dI1/3 (<b>protocol 2</b>), by the addition of either RA alone or both RA and BMP4 (RA+BMP4).</p>
<ol start="27">
<li>Prepare SaND medium supplemented with either 1 μM RA for <b>protocol 1</b> or 1 μM RA+10 ng/mL human recombinant BMP4 for <b>protocol 2.</b>
</li>
<li>Pre-heat medium in 37°C water bath.</li>
<li>
<b>Protocol 1:</b> Induce dI4 and dI6 by transferring EBs into SaND medium + 1 μM RA on day 10. <b>Protocol 2:</b> Induce dI1 and dI3s by transferring EBs into SaND medium with 1 μM RA + 10 ng/mL human recombinant BMP4 on day 10.
<ol type="a">
<li>Transfer EBs to fresh SaND medium by collecting EBs in 50 mL conical tube.</li>
<li>Allow EBs to settle at the bottom of the tube by gravity. This step will take 5–10 min.</li>
<li>Aspirate the liquid from the tube and resuspend in either SaND medium +1 μM RA (<b>protocol 1</b>) or SaND medium with 1 μM RA+ 10 ng/mL BMP4 (<b>protocol 2</b>).</li>
<li>Resuspend EBs and transfer 1 mL EBs+ medium to low attachment 24-well plate.</li>
</ol>
</li>
<li>Feed EBs by transferring them into fresh SaND medium + growth factors after every 2 days., as described in step 26.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> BMP4 aliquots should be freshly thawed and not be subjected to multiple freeze-thaw cycles.</div>
<h3 id="sec3.5">Neuronal differentiation and maturation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 19 days</div>
<p>This step permits dorsal EBs to differentiate and then mature into dI4/6 and dI1/3 neurons. EBs are passaged through both differentiation and maturation media. Ascorbic acid in the maturation medium enhances the differentiation process. At the end of the protocol, EBs can either be fixed, sectioned, and stained directly, or dissociated and plated onto laminin coated slides to examine the neuronal morphology.</p>
<ol start="31">
<li>Prepare neural differentiation medium (NBDM) and maturation medium supplemented with either 1 μM RA for <b>protocol 1</b> or 1 μM RA+10 ng/mL human recombinant BMP4 for <b>protocol 2.</b>
</li>
<li>Transfer EBs from <b>protocol 1</b> into NBDM supplemented with 1 μM RA.</li>
<li>Transfer EBs from <b>protocol 2</b> into NBDM supplemented with 1 μM RA + 10 ng/mL BMP4.
<ol type="a">
<li>Transfer EBs to fresh NBDM by collecting EBs in 50 mL conical tube.</li>
<li>Allow EBs to settle down at the bottom of the tube by gravity. This step will take 5–10 min.</li>
<li>Aspirate the liquid from the tube and resuspend in either NBDM +1 μM RA <b>(protocol 1)</b> or NBDM +1 μM RA+ 10 ng/mL BMP4 <b>(protocol 2).</b>
</li>
<li>Resuspend EBs and transfer 1 mL of EB suspension to low attachment 24 well plate.</li>
</ol>
</li>
<li>Transfer EBs to fresh NBDM medium supplemented with growth factors after every 2 days.</li>
<li>At day 28, transfer EBs from <b>protocol 1</b> into maturation medium supplemented with 1 μM RA and, EBs from <b>protocol 2</b> into maturation medium supplemented with 1 μM RA + 10 ng/mL BMP4.</li>
<li>Repeat steps 33 and 34, now with maturation medium, to continue feeding EBs every 2 days until day 36 or any earlier desired endpoint for the quality assessment.</li>
<li>At the end of the differentiation, EBs can either be a) fixed directly or dissociated for immunostaining, see steps 39–63 or b) lysed to obtain RNA for a qPCR analysis (see steps 64–67).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> EBs can be dissociated at the end of the differentiation using Accutase and plated onto laminin coated coverslips or Ibidi chamber slides for immunostaining procedures.</div>
<h3 id="sec3.6">Quality control analyses of EBs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–5 days</div>
<ol start="38">
<li>
<b>Immunostaining analysis of fixed EBs.</b> Prepare fresh 4% PFA in 1× PBS to fix the EBs.</li>
<li>Collect EBs in a 50 mL conical tube, allow them to settle at the bottom of the tube. Remove medium and wash EBs with 1× PBS. Allow EBs to settle at the bottom of the tube by gravity and aspirate PBS from the tube.</li>
<li>Add 5–10 mL 4% PFA in the tube and incubate EBs on an orbital shaker at 25°C for 20 min.</li>
<li>Wash EBs twice with 1× PBS.</li>
<li>Embedding for cryo-sectioning: replace PBS with 30% sucrose and allow EBs to equilibrate and settle at the bottom of the tube. This step can take from 30 min to 1 h.</li>
<li>Carefully transfer the EBs using wide bore pipette tips into a cryo-block filled with OCT (optimum cutting compound). Gently swirl them around in the OCT using 19G–20G hypodermal needle to equilibrate and arrange EBs at bottom-center of the block. Transfer blocks onto dry ice to freeze.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Try to aspirate all EBs in a 100 μL pipette volume to minimize the amount of sucrose transferred into the OCT.</div>
<ol start="44">
<li>Prepare 12 μm sections of EBs on glass slides, using standard cryo-sectioning methods.</li>
<li>Incubate slides with an antibody blocking solution (10% heat inactivated serum in 1× PBS + 0.2% Triton X-100) for 30 min.</li>
<li>Dilute desired primary antibodies in the antibody blocking solution. Replace blocking solution, with 500 μL of diluted antibody solution/slide and incubate for 12–16 h at 4°C in a humidified chamber.</li>
<li>Remove primary antibody and wash slides twice with 1× PBS + 0.2% Triton (PBTN).</li>
<li>Add species appropriate secondary antibodies in PBTN and incubate slides for 1 h in secondary antibody.</li>
<li>Remove secondary antibody solution and wash slides twice with PBTN.</li>
<li>Counterstain with DAPI to detect nuclei.</li>
<li>Coverslip slides using either Vectashield or ProLong Gold mounting media.</li>
<li>Examine slides using a microscope. Cell numbers can be counted using the ImageJ cell counter plugin.</li>
<li>
<b>Immunostaining analysis of dissociated EBs.</b> Prepare Poly-L-ornithine (PLO) + laminin coated sterile 12 mm coverslips:
<ol type="a">
<li>Sterile the entire pack of 12 mm coverslips by autoclaving. Place one coverslip in each well of a 24-well plate.</li>
<li>Prepare 1.5 mg/mL (100×) PLO stock solution in H<sub>2</sub>O. Dilute to 1× using 1× PBS. Prepare enough diluted PLO solution required for the number of wells to be coated. One well of 24 well plate will be coated with 0.5 mL 1× PLO solution.</li>
<li>Add 0.5 mL PLO solution in wells containing the coverslips. Make sure that the coverslips are completely submerged in the solution.</li>
<li>Incubate at 25°C for 2 h, aspirate the PLO solution, and wash 2 times with H<sub>2</sub>O. Coverslips are ready to be coated with Laminin.</li>
<li>Thaw laminin for 12–16 h at 2°C–8°C and prepare 10 μg/mL working solution in sterile PBS.</li>
<li>Add 0.2–0.5 mL laminin on PLO coated coverslips and incubate at RT for 2 h.</li>
<li>Aspirate laminin and wash two times with sterile PBS. Coverslips are ready to be used.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> PBS is only removed when ready to plate cells. Drying of coated coverslips lead to poor cell attachment.</div>
<ol start="54">
<li>Collect EBs that are to be dissociated in a 50 mL conical tube. Aspirate supernatant and add 1 mL Accutase. Incubate EBs+Accutase in 37°C incubator for 3–5 min.</li>
<li>Triturate 5–6 times to break EBs into smaller clumps. Medium will appear cloudy at this point because of breaking of EBs into multiple smaller clumps.</li>
<li>Spin the medium with dissociated EBs at 112 × <i>g</i> for 5 min and resuspend cells in the required maturation medium.</li>
<li>Plate dissociated EBs onto the coated coverslips (present in the wells of 24-well plate) at the density of 25,000–50,000 cells/well. Place the plate in 37°C incubator for 1–2 days, until cells start to take on a neuronal morphology.</li>
<li>
<b>Immunostaining analysis of the dissociated EBs.</b> Wash wells containing neuron-plated coverslips with 1× PBS.</li>
<li>Add cold 4% PFA in each well and incubate the plate on ice for 8–10 min.</li>
<li>Aspirate the PFA and wash the coverslips with 1× PBS two times.</li>
<li>Follow steps 46–51 for blocking and antibody staining.</li>
<li>To mount the stained coverslips onto a glass slide for the microscopic examination, place a drop of ProLong Gold (40–50 μL) onto a glass slide. Carefully place the stained coverslip in a manner that the sample surface faces the mounting medium. Avoid air bubbles.</li>
<li>Air dry slides for 10–15 min. Coverslips are now ready for the microscopy.</li>
</ol>
<ol start="64">
<li>
<b>qPCR analysis.</b> Collect RNA lysate for RNA extraction.
<div class="note">
<span class="note-title">Note:</span> this method of RNA isolation has been optimized for the RNeasy Plus mini kit (QIAGEN). Other RNA extraction methods can be used, but the protocol may need modification.</div>
<ol type="a">
<li>Collect EBs in a 15 mL conical tube. Wash once with 1× PBS.</li>
<li>Remove PBS and add 500 μL RLT buffer, which is included in the RNeasy Plus mini kit (QIAGEN).</li>
<li>Dissociate EBs in RLT buffer by passing through QIAshredder (QIAGEN).</li>
<li>Collect the elutant and proceed for RNA isolation according to the manufacturer’s instructions.</li>
</ol>
</li>
</ol>
<ol start="65">
<li>Prepare cDNA samples from the isolated RNA according to the manufacturer’s instructions.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> the instructions for preparing cDNA can be accessed on the following link-<a href="https://www.thermofisher.com/order/catalog/product/18091050#/18091050">https://www.thermofisher.com/order/catalog/product/18091050#/18091050</a>.</div>
<ol start="66">
<li>Set up 10 μL qRT-PCR reaction with the LightCycler SYBR green master mix using the following cycling parameters:</li>
</ol>
<table id="undtbl9">
<caption>qRT-PCR setup reaction (10 μL)</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>92°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>92°C</td>
<td>15 s</td>
<td rowspan="3">40 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>60°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>Forever</td>
<td> </td>
</tr>
</tbody>
</table>
<ol start="67">
<li>Analyze qPCR results using the ΔΔ CT method (<a href="#bib5">Livak and Schmittgen, 2001</a>).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The anticipated outcome of this protocol is as follows.</p>
<p>P<b>rotocol 1</b> should yield 30%–40% population of LHX1/5<sup>+</sup> PAX2<sup>+</sup> dI4-dI6 classes of neurons (<a href="#fig3">Figure 3</a>A).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/455-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Immunostaining of D36 embryoid bodies to detect dI4-6, dI1, and dI3 neurons</div>
<p>By day 36, EBs can be fixed, sectioned, and immunostained for dI specific markers.</p>
<p>(A) The EBs derived from <b>protocol 1</b> will contain Pax2/Lhx1/5<sup>+</sup> dI4-dI6 neurons (A, A′, and A″) (B and C) In contrast, the EBs derived from <b>protocol 2</b> contain a mixture of Lhx2<sup>+</sup> dI1s (B) and Isl1<sup>+</sup> Tlx3<sup>+</sup> dI3s (C, C′, and C″).</p>
<p>Scale bar (all panels), 100 μm.</p>
</figcaption>
</figure>
<p>P<b>rotocol 2</b> should yield 10%–20% of LHX2<sup>+</sup> dI1s and Isl1<sup>+</sup> TLX3<sup>+</sup> dI3s neurons (<a href="#fig3">Figure 3</a>B).</p>
<p>In addition to the cell fate specific transcription factors expressed by these dI populations, they should also express pan-neural marker-beta III tubulin (TUJ1) and spinal cord specific axonal markers such Dcc and Robo3 (see (<a href="#bib4">Gupta et al., 2018</a>) for more information and images of the axonal markers).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>To quantify the differentiation efficiency, count 1) the number of nuclei expressing a specific cell fate marker and 2) the total number of DAPI<sup>+</sup> nuclei within an embryoid body, counterstained with DAPI. The area of the EB can be measured using ImageJ, to calculate the percentage of a specific cell type/unit area.</p>
<p>All qPCRs should be performed in triplicate with at least two biological replicates (independent differentiations).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The current protocol generates a mixture of either dI4/dI6 (<b>protocol 1</b>) or dI1/dI3 (<b>protocol 2</b>). We are currently assessing protocol modifications that would generate pure populations of specific dIs.</p>
<p>This protocol has been tested with H9 hESCs and two patient derived iPSC lines (<a href="#bib7">Sareen et al., 2013</a>, <a href="#bib4">Gupta et al., 2018</a>). However, the efficiency of the protocol may vary with other human pluripotent stem cell lines and may require the adjustment of the RA and BMP4 concentrations.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1: poor neural differentiation</h3>
<p>Poor neural differentiation is observed when directing pluripotent hESCs/ hiPSCs toward a spinal neuroectodermal fate (step 14).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>In our hands, SaND medium is sufficient to direct hESCs/hiPSCs toward a neuroectodermal fate. However, if poor neural differentiation is observed, SMAD inhibitors e.g., dorsomorphin (BMP signaling inhibitor, 1.5 μM) and SB431542 (TGF beta inhibitor, 10 μM) can be added to the SaND medium (dual-SMAD inhibition). Inhibition of BMP/TGFβ signaling has been shown to enhance neuroectodermal fate commitment in hESCs (<a href="#bib3">Chambers et al., 2009</a>, <a href="#bib9">Wattanapanitch et al., 2014</a>).</p>
<h3 id="sec7.3">Problem 2: poor viability</h3>
<p>During EB formation, the single-cell dissociation step, described in steps 22–24, can result in poor viability for certain cell lines, even after the addition of ROCK inhibitors. This issue will dramatically affect the efficiency and yield of the dI differentiation process mentioned in step 67 and in the <a href="#expected-outcomes">Expected outcomes</a>.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>In this case, EBs can be formed by cutting the sheet of the neuroectodermal cells (obtained in step 14) using EZ passage tool (Thermo Fisher). This tool contains roller blades to cut the stem cell colonies in uniform manner. If the EZ passage tool is used, Accutase treatment is not required because cutting the stem cell colonies is sufficient to detach them from the bottom of the dish.</p>
<p>To use the EZ passage tool, remove the medium from the well and roll the EZ passage tool first in one direction and then at an 90° angle to cut colonies approximately into squares (arrows, <a href="#fig4">Figure 4</a>A). Add 2 mL of SaND media+1 μM RA+ 10 μM ROCK inhibitors in each well to collect the freshly cut stem cell clusters and transfer them into ultra-low attachment plates. EBs are formed by next day (<a href="#fig4">Figure 4</a>B).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/455-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Formation of embryoid bodies using the EZ passage tool</div>
<p>(A) As an alternative method of making EBs, the EZ passage tool can be used to cut the monolayer of neuroectodermal cells at day 6, resulting in the square-shaped cell clusters (arrows).</p>
<p>(B) Embryoid bodies are formed by day 7 after these clusters are transferred to an ultra-low attachment plate.</p>
<p>Scale bar (all panels), 500 μm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Andrews, M.G., Del Castillo, L.M., Ochoa-Bolton, E., Yamauchi, K., Smogorzewski, J., and Butler, S.J. (2017). BMPs direct sensory interneuron identity in the developing spinal cord using signal-specific not morphogenic activities. eLife <i>6</i>, e30647. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref1">View at publisher</a></p>
<p id="bib2">Andrews, M.G., Kong, J., Novitch, B.G., and Butler, S.J. (2019). New perspectives on the mechanisms establishing the dorsal-ventral axis of the spinal cord. Curr. Top Dev. Biol. <i>132</i>, 417-450. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref2">View at publisher</a></p>
<p id="bib3">Chambers, S.M., Fasano, C.A., Papapetrou, E.P., Tomishima, M., Sadelain, M., and Studer, L. (2009). Highly efficient neural conversion of human ES and iPS cells by dual inhibition of SMAD signaling. Nat. Biotechnol. <i>27</i>, 275-280. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref3">View at publisher</a></p>
<p id="bib4">Gupta, S., Sivalingam, D., Hain, S., Makkar, C., Sosa, E., Clark, A., and Butler, S.J. (2018). Deriving dorsal spinal sensory interneurons from human pluripotent stem cells. Stem Cell Reports <i>10</i>, 390-405. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref4">View at publisher</a></p>
<p id="bib5">Livak, K.J. and Schmittgen, T.D. (2001). Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods <i>25</i>, 402-408. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref5">View at publisher</a></p>
<p id="bib6">Pettinato, G., Wen, X., and Zhang, N. (2015). Engineering strategies for the formation of embryoid bodies from human pluripotent stem cells. Stem Cells Dev. <i>24</i>, 1595-1609. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref6">View at publisher</a></p>
<p id="bib7">Sareen, D., O'Rourke, J.G., Meera, P., Muhammad, A.K., Grant, S., Simpkinson, M., Bell, S., Carmona, S., Ornelas, L., Sahabian, A., et al. (2013). Targeting RNA foci in iPSC-derived motor neurons from ALS patients with a C9ORF72 repeat expansion. Sci. Transl. Med. <i>5</i>, 208ra149. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref7">View at publisher</a></p>
<p id="bib8">Watanabe, K., Ueno, M., Kamiya, D., Nishiyama, A., Matsumura, M., Wataya, T., Takahashi, J.B., Nishikawa, S., Muguruma, K., and Sasai, Y. (2007). A ROCK inhibitor permits survival of dissociated human embryonic stem cells. Nat. Biotechnol. <i>25</i>, 681-686. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref8">View at publisher</a></p>
<p id="bib9">Wattanapanitch, M., Klincumhom, N., Potirat, P., Amornpisutt, R., Lorthongpanich, C., U-Pratya, Y., Laowtammathron, C., Kheolamai, P., Poungvarin, N., and Issaragrisil, S. (2014). Dual small-molecule targeting of SMAD signaling stimulates human induced pluripotent stem cells toward neural lineages. PLoS One <i>9</i>, e106952. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00026-5/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by Samantha Butler (<a href="mailto:butlersj@ucla.edu">butlersj@ucla.edu</a>).</p>
<h4>Materials Availability</h4>
<p>The majority of materials required in this protocol are commercially available. The primer sequences used for the qPCRs can be obtained in the supplementary material of (<a href="#bib4">Gupta et al., 2018</a>).</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate/analyze any datasets or code that were submitted to any repositories.</p>
<h3>Acknowledgments</h3>
<p>We would like to thank Madeline Andrews and Daniel Sivalingam for discussions. This work was supported by training fellowships from <a href="https://doi.org/10.13039/100008623">Eli and Edythe Broad Center of Regenerative Medicine and Stem Cell Research at UCLA</a> (BSCRC) to K.Y. and S.G., and grants from the BSCRC, Ablon Foundation, the <a href="https://doi.org/10.13039/100000900">California Institute for Regenerative Medicine</a> (CIRM) (RB1-01367 and RB5-07480 to B.G.N. and RB5-07320 to S.J.B), and the <a href="https://doi.org/10.13039/100000002">NIH</a> (R21NS115012 to B.G.N. and NS085097 to S.J.B).</p>
<h3>Author Contributions</h3>
<p>S.G. and K.Y. performed both the experiments and analysis. S.J.B., S.G., B.G.N., and K.Y. wrote and edited the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>