<article>
<section>
<h1 id="header">A detailed protocol for RNA cleavage assay in sympathetic neurons</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Catia Andreassi<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Antonella Riccio<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>MRC Laboratory for Molecular Cell Biology, University College London, London WC1E 6BT, UK</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:c.andreassi@ucl.ac.uk">c.andreassi@ucl.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:a.riccio@ucl.ac.uk">a.riccio@ucl.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101001">10.1016/j.xpro.2021.101001</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol illustrates the use of an <i>in vitro</i> assay to study the cleavage of the <i>IMPA1</i> 3′UTR by the endonuclease Ago2 in sympathetic neurons. The procedure includes the preparation of cytoplasmic protein extracts and also describes the synthesis and labeling of the RNA probe. The protocol can be applied to other cell systems, RNA transcripts, and endonucleases to confirm the role of known cleavage site(s) and cleavage proteins, or to investigate new ones.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref3">Andreassi et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Describes cleavage assay of the <i>IMPA1</i> 3′UTR by the endonuclease Ago2</li>
<li>Use of radionuclides enables high sensitivity</li>
<li>Adaptable to other RNA sequences and endonucleases</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Post-transcriptional endonucleolytic cleavage is a widespread processing mechanism that controls cytoplasmic mRNA metabolism in a variety of conditions, including miRNA-mediated silencing (<a href="#bib7">Jin et al., 2021</a>), mRNA decay (<a href="#bib5">Harigaya and Parker, 2010</a>), mRNA targeting by viral endonucleases (<a href="#bib9">Rodriguez et al., 2021</a>) and generation of non-coding RNA from mRNA (<a href="#bib2">Andreassi et al., 2018</a>). We have modified and integrated previously published protocols (<a href="#bib8">Miyoshi et al., 2008</a>; <a href="#bib6">Jenal et al., 2012</a>) to develop an <i>in vitro</i> assay to study <i>IMPA1</i> 3′UTR cleavage by the endonuclease Ago2, in the presence of cytoplasmic extracts from sympathetic neurons. The protocol includes preliminary steps to prepare cytoplasmic extracts, and to synthesize and label the RNA probe.</p>
<h3 id="sec1.1">Cytoplasmic extracts from sympathetic neurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<p>This step describes the preparation of cytoplasmic protein extracts that will be added to the cleavage reaction mix.</p>
<div class="note">
<span class="note-title">Note:</span> Mechanical force and hypotonic cell lysis can be used to separate cytosolic and nuclear proteins fractions (<a href="#bib4">DeCaprio and Kohl, 2019</a>). Please, note that optimization of lysis conditions may be necessary depending on the cell type. We routinely use the NE-PER kit for Nuclear and cytoplasmic extractions as it ensures reproducible results with both cell lines (N2A cells, HEK293 cells, PC12 cells) and primary neurons (cortical neurons, sympathetic neurons).</div>
<div class="critical">
<span class="critical-title">Critical:</span> From the cell harvest step onwards, use RNAse-free reagents and techniques (<a href="#sec6.1">Troubleshooting Problem1</a>).</div>
<ol>
<li>All protocols for animal studies were approved by the Institutional Animal Care and Use Committees at University College London. Culture 2,500,000–3,000,000 sympathetic neurons dissected from P0/P1 Wistar rats on one collagen-coated 35 mm dish for 5 days (<a href="#bib1">Andreassi et al., 2010</a>). Rinse cells with RNAse-free PBS and scrape them in 1 mL of RNAse-free PBS. Centrifuge the cells at 500×<i>g</i> for 5 min at 22°C–25°C and remove the PBS leaving the rinsed cells as dry as possible.
<ol type="a">
<li>Add 100 μL of CERI solution+1:100 Protease inhibitor cocktail added fresh.</li>
<li>Vortex the tube vigorously on the highest setting for 15 s to fully suspend the cell pellet and incubate on ice for 10 min.</li>
<li>Add 5.5 μL of CERII solution and vortex for 5 s on the highest setting. Incubate on ice for 1 min.</li>
<li>Vortex for 5 s on the highest setting and centrifuge for 5 min at maximum speed (∼16,000 × <i>g</i>) in a microcentrifuge at 4°C.</li>
<li>Immediately transfer the supernatant (cytoplasmic extract) to a clean pre-chilled tube. Add 1.25 μL of RNasin Plus Ribonuclease Inhibitor (40 U/μL stock). To avoid repeated freezing and thawing, store in 35 μL aliquots (sufficient for 3 experiments) at −80°C. Proceed with nuclear protein extraction if necessary as a control for step 2, otherwise discard the nuclear fraction.</li>
</ol>
</li>
<li>Run an aliquot (generally 5%–10% of the total volume) of the cytosolic (and nuclear, if available) fraction samples on a denaturing SDS-PAGE and perform western blotting for known nuclear and cytoplasmic markers (for example HDAC2 and Impa1) to verify the purity of the fractions (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Western blotting to check purity of nuclear and cytosolic fractions</div>
<p>Aliquots of proteins purified from nuclear or cytosolic fractions of sympathetic neurons cultured <i>in vitro</i> were separated by PAGE, blotted and probed with antibodies for nuclear or cytosolic protein markers (HDAC2 and Impa1, respectively).</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec1.2">
<i>In vitro</i> transcription, de-phosphorylation and 5′end-labeling of the RNA marker and of the probe, and full-length probe purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>This section provides a detailed description of 4 preliminary steps (<i>in vitro</i> transcription, de-phosphorylation, labeling and isolation of full length probe by band isolation) necessary to generate the probe that will be used in the cleavage reaction mix, and to label the RNA marker that will serve as size reference when separating the products of the cleavage reaction.</p>
<div class="note">
<span class="note-title">Note:</span> Although any suitable protocol for <i>in vitro</i> transcription will work to generate the RNA probe, we routinely use the <i>mir</i>Vana miRNA Probe Construction kit, as we found it to be flexible and very reliable in terms of yield. By using a primer to incorporate the promoter for the T7 RNA polymerase (see below), time-consuming cloning steps are avoided. Below we describe the protocol in its essential steps, and we refer to the manufacturer’s user manual (for more details see the <a href="http://tools.thermofisher.com/content/sfs/manuals/fm_1550.pdf">http://tools.thermofisher.com/content/sfs/manuals/fm_1550.pdf</a>). For the RNA transcription, a DNA oligonucleotide about 120 nt long whose sequence encompasses the expected cleavage site should be used. The position of the expected cleavage site within the oligonucleotide sequence should be chosen considering the size of the expected fragments and potential additional restrictions due to the surrounding sequences. If the probe will be uniformly labeled by incorporation of a labeled nucleotide during the <i>in vitro</i> transcription, it is advisable to design the oligonucleotide such that the cleavage site is not centered in the oligo sequence. This will allow both fragments generated after cleavage to be easily resolved by electrophoresis. The sequence of the oligonucleotide should be antisense to the RNA strand to be synthesized and should include the leader sequence CCTGTCTC at its 3′end (<a href="#fig2">Figure 2</a>).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Oligonucleotide design and steps of <i>in vitro</i> transcription reaction</div>
<p>Schematics of the <i>IMPA1</i> 3′UTR sequence targeted by Ago2 (upper panel), and of the oligonucleotide template, intermediate products and RNA from the <i>in vitro</i> transcription reaction for probe synthesis (lower panel).</p>
</figcaption>
</figure>
<ol start="3">
<li>To synthesize the RNA probe, first resuspend the purchased oligonucleotide to 100 μM concentration in H<sub>2</sub>O using RNAse-free conditions and use it to prepare the dsDNA template for the <i>in vitro</i> transcription. The leftovers of resuspended oligo can be stored at −80°C.
<ol type="a">
<li>Thaw, mix by vortexing and keep at 22°C–25°C the following reagents from the <i>mir</i>Vana kit: T7 promoter primer, DNA hybridization buffer, 10× Klenow buffer, 10× dNTP mix, nuclease-free water. Keep the tube of Exo-Klenow Enzyme mix at −20°C and do not vortex it. Add the reagents to a PCR tube in the following order:
<table id="undtbl1">
<thead></thead>
<tbody>
<tr>
<td>T7 Promoter Primer</td>
<td>2 μL</td>
</tr>
<tr>
<td>DNA Hybridization Buffer</td>
<td>6 μL</td>
</tr>
<tr>
<td>100 μM oligonucleotide template</td>
<td>2 μL</td>
</tr>
</tbody>
</table>
Incubate at 70°C for 5 min and then at 22°C–25°C for 5 min, before adding in this order:
<table id="undtbl2">
<thead></thead>
<tbody>
<tr>
<td>10× Klenow Reaction Buffer</td>
<td>2 μL</td>
</tr>
<tr>
<td>10× dNTP Mix</td>
<td>2 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>4 μL</td>
</tr>
<tr>
<td>Exo–Klenow enzyme</td>
<td>2 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix by pipetting, quick spin and incubate at 37°C for 30 min, then proceed with the <i>in vitro</i> transcription, or store at −20°C for several months.</p>
</li>
<li>To set up the <i>in</i> vitro transcription reaction, thaw and mix by vortexing 10× transcription buffer, nucleotide solutions (NTPs), and nuclease-free water from the <i>mir</i>Vana kit. Keep NTPs always on ice and the 10× transcription buffer at 22°C–25°C. Keep the tube of T7 Enzyme at −20°C and do not vortex it. Prepare the reaction mix at 22°C–25°C by mixing in a PCR tube in the following order:
<table id="undtbl3">
<thead></thead>
<tbody>
<tr>
<td>Nuclease-free water</td>
<td>11 μL</td>
</tr>
<tr>
<td>dsDNA template (step 3a.)</td>
<td>1 μL</td>
</tr>
<tr>
<td>10× transcription buffer</td>
<td>2 μL</td>
</tr>
<tr>
<td>10 mM ATP</td>
<td>1 μL</td>
</tr>
<tr>
<td>10 mM CTP</td>
<td>1 μL</td>
</tr>
<tr>
<td>10 mM GTP</td>
<td>1 μL</td>
</tr>
<tr>
<td>10 mM UTP</td>
<td>1 μL</td>
</tr>
<tr>
<td>T7 RNA polymerase</td>
<td>2 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Gently pipette up and down several times to mix, then centrifuge briefly to collect the reaction mixture at the bottom of the tube, and incubate the reaction mixture at 37°C for at least 60 min.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure that the transcription buffer is completely thawed and at 22°C–25°C before adding it to the reaction tube that should also be kept at 22°C–25°C. The step is critical to avoid precipitation of the oligonucleotide by the spermidine contained in the buffer.</div>
</li>
<li>Digest the dsDNA template by adding:
<table id="undtbl4">
<thead></thead>
<tbody>
<tr>
<td>RNase-free DNase I</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix well by pipetting and incubate at 37°C for 10 min.</p>
</li>
<li>Add nuclease-free water to 50 μL total volume and purify the probes using G50 columns as per manufacturer’s instructions <a href="https://www.cytivalifesciences.com/en/us/shop/molecular-biology/purification/gel-filtration-columns/illustra-probequant-g-50-micro-columns-p-00055#tech-spec-table">https://www.cytivalifesciences.com/en/us/shop/molecular-biology/purification/gel-filtration-columns/illustra-probequant-g-50-micro-columns-p-00055#tech-spec-table</a> to eliminate unincorporated nucleotides.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>De-phosphorylate the <i>in vitro</i> transcribed RNA probe using Calf Intestinal Phosphatase (CIP) and purify it using a phenol/chloroform extraction followed by ethanol precipitation.
<ol type="a">
<li>Dilute CIP to 2.5 U/μL using 1× CutSmart buffer (dispose of unused diluted enzyme) and proceed as follows by mixing:
<table id="undtbl5">
<thead></thead>
<tbody>
<tr>
<td>Purified IVT rxn (step 3.d)</td>
<td>50 μL</td>
</tr>
<tr>
<td>10× CutSmart buffer</td>
<td>5.5 μL</td>
</tr>
<tr>
<td>Diluted CIP</td>
<td>1.1 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Incubate at 37°C for 30 min.</p>
</li>
<li>Remove the CIP by phenol/chloroform extraction by adding to each reaction:
<table id="undtbl6">
<thead></thead>
<tbody>
<tr>
<td>RNAse-free water</td>
<td>75 μL</td>
</tr>
<tr>
<td>2.0 M sodium acetate pH 4.3</td>
<td>15 μL</td>
</tr>
<tr>
<td>Phenol/chloroform 25:24:1</td>
<td>150 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix well by vigorous shaking of the tube by hand, then spin at 17000×<i>g</i> at 22°C–25°C for 5 min in a microcentrifuge. Transfer the aqueous (top) phase to a clean 1.5 mL tube.</p>
</li>
<li>Precipitate the RNA by adding</li>
</ol>
</li>
</ol>
<table id="undtbl7">
<thead></thead>
<tbody>
<tr>
<td>Linear acrylamide</td>
<td>1 μL</td>
</tr>
<tr>
<td>100% ethanol</td>
<td>750 μL</td>
</tr>
</tbody>
</table>
<p>Vortex and precipitate at −20°C &gt;30 min <b>STOP point</b>. Spin at the maximum speed at 4°C for 15 min in a microcentrifuge. Carefully remove the supernatant and discard. Wash the pellet once with 1.5 mL of ice cold 70% ethanol. Spin at the max speed at 4°C for 5 min in a microcentrifuge. Carefully withdraw nearly all supernatants, spin for 1 min at max speed to collect the residual ethanol on the wall of the tube and discard the liquid using an extended pipette tip. Resuspend the pellet in 15 μL of nuclease-free water and proceed with labeling <b>STOP point</b>.</p>
<ol start="5">
<li>To achieve high sensitivity, the probe is labeled with a radionuclide.
<div class="note">
<span class="note-title">Note:</span> The probe can be either uniformly labeled, by including a labeled nucleotide in the <i>in vitro</i> transcription reaction (step 3.b), or labeled at its 5′end. We prefer the latter as the results are generally cleaner and easier to interpret (<a href="#fig3">Figure 3</a>). The amount of radioactive ATP to add to the labeling reaction depends on the reference activity date of the batch used. 1 μL of a fresh batch is usually sufficient. Increase the volume up to 5 μL if the batch is 2 weeks old, adjusting the reaction volume to 25 μL total. For best results, do not use a radioactive batch that is more than 3 weeks old.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The protocol involves the use of radiolabeled nucleotides. Strictly follow institutional guidelines for working with radionuclides.</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Comparison between total labeling and 5′end-labeling of the probe</div>
<p>(A and B) The probes can be labeled during the <i>in vitro</i> transcription reaction (total labeling, A) or by treatment with T4 PNK (5′end-labeling, B). The latter gives better results as there is a substantial decrease in background signal (asterisk in A). Non-contiguous lanes from the same experiment and autoradiography blot are shown side by side. Gray lines show where irrelevant lanes have been removed.</p>
</figcaption>
</figure>
<ol type="a">
<li>Perform 5′-end labeling using radioactive ATP ([γ-<sup>32</sup>P ATP] and T4 Polynucleotide Kinase (PNK). Mix by pipetting:
<table id="undtbl8">
<thead></thead>
<tbody>
<tr>
<td>CIP-treated IVT RNA (step 4.c)</td>
<td>15 μL</td>
</tr>
<tr>
<td>10× T4 PNK Buffer A</td>
<td>2 μL</td>
</tr>
<tr>
<td>[γ-<sup>32</sup>P] ATP ∼25 pmol</td>
<td>x μL</td>
</tr>
<tr>
<td>T4 Polynucleotide Kinase</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Incubate at 37°C for more than 1 h and less than 3 h, then add 1 μL of 0.5 mM EDTA, incubate 2 min at 22°C–25°C and heat-inactivate at 95°C for 2 min.</p>
</li>
<li>Eliminate unincorporated nucleotides using G-50 columns as in step 3.d.</li>
<li>Purify the labeled RNA using 1 mL of TRIzol according to the manufacturer’s protocol <a href="https://tools.thermofisher.com/content/sfs/manuals/trizol_reagent.pdf">https://tools.thermofisher.com/content/sfs/manuals/trizol_reagent.pdf</a> and by adding 1 μL of Linear acrylamide as co-precipitant. Resuspend the pellet in 10 μL of RNAse-free water, add 10 uL of 2× Gel Loading Buffer from <i>mir</i>Vana kit (or NEB supplemented with 18 mM EDTA) and either freeze <b>STOP point</b> or proceed with gel purification (step 7).</li>
</ol>
</li>
</ol>
<ol start="6">
<li>The RNA marker that will be used as a size reference for the gel electrophoresis needs to be labeled as well.
<div class="note">
<span class="note-title">Note:</span> Any RNA marker of suitable size can be used, but it will need to be dephosphorylated (see point 4). For this reason, we prefer to use the Decade Marker system that is ready to be labeled following manufacturer’s instructions <a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2F4386532D.pdf=UEkgU2hlZXQ6IERlY2FkZSBNYXJrZXJzIFN5c3RlbQ==">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2F4386532D.pdf=UEkgU2hlZXQ6IERlY2FkZSBNYXJrZXJzIFN5c3RlbQ==</a>. Upon receiving the markers, they should be dissolved by adding 60 μL of nuclease-free water, aliquoted in 1.5 mL tubes (7 μL each) and stored at −80°C. See preparation point 5 for information about the amount of radioactive label that will be used in the labeling reaction.</div>
<ol type="a">
<li>To label the RNA marker, use the reagents provided in the Decade Marker systems kit to set up the reaction as follows:
<table id="undtbl9">
<thead></thead>
<tbody>
<tr>
<td>Decade™ Marker RNA (100 ng) +H<sub>2</sub>O</td>
<td>7 μL</td>
</tr>
<tr>
<td>10× T4 PNK Buffer A</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>up to 4.5 μL</td>
</tr>
<tr>
<td>[γ-32P] ATP</td>
<td>x μL</td>
</tr>
<tr>
<td>T4 Polynucleotide Kinase</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Incubate at 37°C for 1 h.</p>
</li>
<li>Add to the reaction tube:
<table id="undtbl10">
<thead></thead>
<tbody>
<tr>
<td>Nuclease-free Water</td>
<td>3 μL</td>
</tr>
<tr>
<td>10×Cleavage Reagent</td>
<td>2 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Incubate at 22°C–25°C for 5 min, then add 20 μL of 2× Gel Loading Buffer from <i>mir</i>VANA kit (or NEB supplemented with 18 mM EDTA) and store at −20°C or use.</p>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Separate the probe by denaturing polyacrylamide gel electrophoresis (PAGE) and purify the probe by band isolation.
<div class="note">
<span class="note-title">Note:</span> <i>In vitro</i> transcription reactions generally produce minor products of intermediate length that may confound the results of the cleavage reaction. Thus, it is necessary to purify the full-length probe by band isolation using a denaturing polyacrylamide gel. Precast polyacrylamide gels can be used, but the size of the gel is usually not adequate to obtain an optimal band separation. Thus, we recommend pouring home-cast gels.</div>
<ol type="a">
<li>Cast a denaturing polyacrylamide gel as follows. Volumes in this protocol are set for the Biometra Maxi-gel system.
<ol type="i">
<li>Clean plates and combs with soapy water, dry and rinse with 100% Ethanol. Treat all parts with RNAseZap and rinse with RNAse-free water. Dry and set up the plates for casting the gels according to the manufacturer’s instructions <a href="https://manualzz.com/doc/6464039/minigel-twin-user-manual">https://manualzz.com/doc/6464039/minigel-twin-user-manual</a>.</li>
<li>Prepare an 8% acrylamide-7M urea gel according to the recipe table below. In a 50 mL tube, weigh the urea, add 5× TBE buffer and water to 25.36 mL final volume, vortex and place in a water bath at 42°C vortexing at regular intervals until all crystals are dissolved (about 30 min). Syringe filter with a 0.45 μm filter. Add freshly made APS and TEMED, swirl to mix and pour the gel.
<div class="note">
<span class="note-title">Note:</span> To avoid introducing air bubbles, do not completely empty the pipette and make sure to have excess solution. If air bubbles are present, hold the plates at an angle of about 45° and use a pair of scissors, or similar, to gently bang the plates and dislodge the bubbles. Insert the comb slowly, at an angle, to avoid trapping air bubbles in the wells. After polymerizing, the gel can be stored at 4°C for several days, as long as it is kept moist by wrapping it in water-soaked tissue and sealed with cling film.</div>
</li>
<li>Before loading the samples, it is advisable to pre-run the gel to warm it and to keep the samples denatured. Remove the comb, rinse the wells with water or a running buffer and set up the gel in the running apparatus. Fill the reservoirs with 1× TBE running buffer and pre-run the gel (empty) at 20–25 mA for about 45 min.</li>
</ol>
</li>
<li>Prepare the samples for the gel loading and run electrophoresis.
<ol type="i">
<li>While the gel is pre-running, complete the probe purification (step 4.c) or thaw on ice the purified probe stored at −20°C. Aliquot the needed amount of labeled marker. The amount of marker depends on the activity of the batch of the radioactive labeling used. Use 10 μL or more to get an intense labeling upon autoradiography.</li>
<li>Heat markers and samples for 3 min at 100°C, incubate on ice for 3 min and quick spin in a cold centrifuge.</li>
<li>While heating the samples, flush the gel wells with a syringe and running buffer.</li>
<li>After spinning, load both samples and marker on the pre-warmed denaturing gel using long gel loading tips. Leave empty lanes between marker and samples to prevent carry over and over-shadowing of the bands.</li>
<li>Run at 20–25 mA for 1.5 h. Use the migration pattern of the loading dyes according to the table below, to decide when to interrupt the electrophoresis.
<table id="undtbl11">
<thead>
<tr>
<th>% Acrylamide (denaturing conditions)</th>
<th>Blue bromophenol (nt)</th>
<th>Xylene cyanol (nt)</th>
</tr>
</thead>
<tbody>
<tr>
<td>8</td>
<td>19</td>
<td>75</td>
</tr>
<tr>
<td>12</td>
<td>10</td>
<td>70</td>
</tr>
<tr>
<td>15</td>
<td>8</td>
<td>28</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>Identify the full-length probe by autoradiography.
<ol type="i">
<li>Remove the glass plates from the running apparatus.</li>
<li>Cool down only one glass side by placing ice on it. This will prevent the gel from detaching from both glass plates when separating them.</li>
<li>Open the glass plates, by using a spatula as a lever in between the glass plates, and place a transparency film on the gel. Secure with tape the position of the transparency film to the gel and use a fluorescent sticker for orientation (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Band-isolation of full-length probe</div>
<p>Schematics of the set-up to identify the area of the gel to cut in order of purifying the full length probe.</p>
</figcaption>
</figure>
</li>
<li>In a dark room, expose the gel to an X-ray film.
<div class="note">
<span class="note-title">Note:</span> The probe is very radioactive, thus an exposure of 30 s-few minutes at 22°C–25°C (or at −80°C if the X-ray cassette has intensifying screens) is usually sufficient to identify the full-length probe that should be the most intense band at the highest size (<a href="#fig5">Figure 5</a>). Do not overexpose as a faint gray band is ideal for cutting and will allow to minimize the amount of acrylamide.</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Band isolation of full-length probe</div>
<p>(A and B) The band corresponding to the full length probe is identified by its size (120 nt in this example) on the auto-radiography (A) and, after being superimposed to the gel, it is used to mark the area of the gel to cut. After cutting, the gel is exposed to film again (B). Absence of the marked band (dotted oval) suggests successful isolation of the correct band. Non-contiguous lanes from the same experiment and autoradiography blot are shown side by side. Gray lines show where irrelevant lanes have been removed.</p>
</figcaption>
</figure>
</li>
<li>To cut out the band of the full length probe, superimpose the film and the gel using the fluorescent stickers for reference, label the band on the glass plate, remove the film and transparency and cut with a clean disposable scalpel tightly around the band (<a href="#fig4">Figure 4</a>).</li>
<li>Roughly chop the band and transfer it to a 500 μL tube, pierced on the bottom with a 25G needle and placed on top of a 1.5 mL tube. Spin for 1 min at max speed to crush the gel band.</li>
<li>Elute the RNA from the gel by incubation with 400 μL of elution buffer from the <i>mir</i>Vana Kit for 14–16 h at 22°C–25°C on a rotor wheel. Expose the gel again to make sure that the band isolation was effective (<a href="#fig5">Figure 5</a>). <a href="#sec6.3">Troubleshooting Problems 2</a>, <a href="#sec6.5">3</a>, and <a href="#sec6.7">5</a>:
<div class="note">
<span class="note-title">Note:</span> Throughout the next step, the use of a Geiger counter will help to make sure that radioactivity signal (i.e. sample) is not lost during the extraction.</div>
</li>
<li>To precipitate the eluted probe, centrifuge for 1 min at max speed to pellet the gel fragments and collect the supernatant. Add about 3× volumes of 100% ET-OH and incubate for at least 30 min at −20°C.</li>
<li>Centrifuge at max speed for 30 min at 4°C, aspirate the supernatant and wash the pellet with ice-cold 75% EtOH. Centrifuge at max speed for 5 min at 4°C and remove the supernatant until the pellet is dry. Dissolve the pellet in 20 μL (or other suitable volume) of nuclease-free water for 5 min at 22°C–25°C. The purified probe can be stored at −20°C for a few days <b>STOP point</b> but it is advisable to use it fresh to minimize radiolysis and decay of the radioactive signal.</li>
</ol>
</li>
<li>Using a liquid scintillation β-counter, count 1 μL of the purified probe and 1 μL of the isotope stock opportunely diluted (i.e., 1:100). Calculate probe concentration as follows:
<p class="chem">P = Total activity of the probe = measured cpm/μL probe × dilution factor × total volume of probe</p>
<p class="chem">S = Total amount of radioactivity used in the labeling reaction = measured cpm/μL ATP stock × dilution factor × volume of ATP stock used in the reaction. If necessary, correct for decay of the signal using the activity reference date of the ATP stock</p>
<p class="chem">F = Fraction of incorporated radioactivity = P/S</p>
<p class="chem">A = Amount (pmol) of probe used in the reaction. From the concentration of the stock calculate the amount of ATP in pmol used for the labeling reaction (i.e., if the ATP stock is 3000Ci/mmol, 10mCi/mL, 1 μL ATP stock used in the reaction corresponds to 3.3pmol)</p>
<p class="chem">Y = Probe yield (pmol) = A × F</p>
<p class="chem">Concentration of the probe (pM) = (Y/volume of the probe in mL) × 1000</p>
</li>
</ol>
</li>
</ol>
<p>Probe concentration varies between purifications, but it is usually in between 0.5 nM and 10 nM.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit anti-IMPA1 (use 1:10000 for western blot)</td>
<td>Abcam</td>
<td>ab184165</td>
</tr>
<tr>
<td>Mouse anti-HDAC2 (use 1:1000 for western blot)</td>
<td>Abcam</td>
<td>ab12169</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Recombinant Ago2</td>
<td>Active Motif</td>
<td>31486</td>
</tr>
<tr>
<td>Rat tail collagen</td>
<td>Homemade</td>
<td>n/a</td>
</tr>
<tr>
<td>PBS 0.0095M(PO4) w/o Ca and Mg</td>
<td>Lonza</td>
<td>BE17-516F</td>
</tr>
<tr>
<td>Protease Inhibitor Cocktail</td>
<td>Merck</td>
<td>P8340</td>
</tr>
<tr>
<td>RNasin Plus Ribonuclease Inhibitor</td>
<td>Promega</td>
<td>N2611</td>
</tr>
<tr>
<td>RNAseZap</td>
<td>Merck</td>
<td>R2020</td>
</tr>
<tr>
<td>Calf Intestinal Phosphatase (CIP)</td>
<td>New England Biolabs</td>
<td>M0290S</td>
</tr>
<tr>
<td>Sodium Acetate buffer solution</td>
<td>Merck</td>
<td>S7899</td>
</tr>
<tr>
<td>Linear acrylamide</td>
<td>Thermo Fisher Scientific</td>
<td>AM9520</td>
</tr>
<tr>
<td>Phenol:Chloroform:Isoamyl Alcohol (25:24:1)</td>
<td>Fisher Scientific</td>
<td>15875408</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Merck</td>
<td>32221</td>
</tr>
<tr>
<td>γ-<sup>32</sup>P ATP</td>
<td>PerkinElmer</td>
<td>BLU002A100UC</td>
</tr>
<tr>
<td>T4 Polynucleotide Kinase</td>
<td>Thermo Fisher Scientific</td>
<td>EK0031</td>
</tr>
<tr>
<td>0.5M EDTA pH 8</td>
<td>Merck</td>
<td>E7889</td>
</tr>
<tr>
<td>TRIzol Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>15596018</td>
</tr>
<tr>
<td>RNA Loading Dye, (2×)</td>
<td>New England Biolabs</td>
<td>B0363S</td>
</tr>
<tr>
<td>ProtoGel (30% Solution at 37.5:1 Ratio)</td>
<td>GENEFLOW</td>
<td>EC-890</td>
</tr>
<tr>
<td>Ammonium persulfate</td>
<td>Merck</td>
<td>A3678</td>
</tr>
<tr>
<td>TEMED</td>
<td>VWR</td>
<td>443083g</td>
</tr>
<tr>
<td>Trizma base</td>
<td>Merck</td>
<td>T1503</td>
</tr>
<tr>
<td>Boric acid</td>
<td>Meck</td>
<td>B6768</td>
</tr>
<tr>
<td>Urea</td>
<td>Merck</td>
<td>U5578</td>
</tr>
<tr>
<td>HEPES Buffer, 1M Solution</td>
<td>Fisher Scientific</td>
<td>83264</td>
</tr>
<tr>
<td>Potassium hydroxide</td>
<td>Merck</td>
<td>60377</td>
</tr>
<tr>
<td>Potassium acetate</td>
<td>Merck</td>
<td>60035</td>
</tr>
<tr>
<td>Magnesium Acetate</td>
<td>Merck</td>
<td>M5661</td>
</tr>
<tr>
<td>DL-Dithiothreitol</td>
<td>Merck</td>
<td>D9779</td>
</tr>
<tr>
<td>Yeast tRNA (10 mg/mL)</td>
<td>Thermo Fisher Scientific</td>
<td>AM7119</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>rat sympathetic neuron extracts</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Wild-type P0/P1 Wistar rats, either sex</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>CTGACACATTAGAAGCAGGAATTGCAGCTTCAAGCACTCAGTGTGAGGTGTCGATTGGAGATCGCCTACTGTCTGACTGGCTCTGCACTAGGTGCTGTATTACAGTGATTAACCTGTCTC</td>
<td>Sigma-Genosys</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>NE-PER™ Nuclear and Cytoplasmic Extraction Reagents</td>
<td>Thermo Fisher Scientific</td>
<td>78833</td>
</tr>
<tr>
<td>Decade Markers System</td>
<td>Thermo Fisher Scientific</td>
<td>AM7778</td>
</tr>
<tr>
<td>
<i>mi</i>rVana™ miRNA Probe Construction Kit</td>
<td>Thermo Fisher Scientific</td>
<td>AM1550</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Related to preparation steps 3.d and 5.d</p>
<p>ProbeQuant G-50 Micro Columns Cytiva 28903408</p>
<p>Related to preparation step 7.a and main protocol step 3</p>
<p>Maxi gel system Biometra 010-400</p>
<p>Related to main protocol step 3.f</p>
<p>Grade 3MM Chr Cellulose Chromatography Papers Cytiva 3030-917</p>
<p>Related to preparation step 7.c and main protocol step 3.g</p>
<p>Hyperfilm™ MP Merck GE28-9068-44</p>
<p>Hypercassette Merck GERPN11649</p>
<table id="undtbl12">
<caption>5× TBE buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Trizma base</td>
<td>89 mM</td>
<td>54 g</td>
</tr>
<tr>
<td>Boric acid</td>
<td>89 mM</td>
<td>27.5 g</td>
</tr>
<tr>
<td>0.5M EDTA pH 8</td>
<td>2 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at RT. After prolonged storage, EDTA may precipitate out of solution.</p>
<table id="undtbl13">
<caption>8% PAA gel</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>30% solution Acrylamide to Bisacrylamide</td>
<td>8%</td>
<td>10.4 mL</td>
</tr>
<tr>
<td>urea</td>
<td>7M</td>
<td>16.8 g</td>
</tr>
<tr>
<td>5× TBE</td>
<td>1×</td>
<td>8 mL</td>
</tr>
<tr>
<td>100mg/mL Ammonium persulfate</td>
<td>0.5 mg/mL</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>TEMED</td>
<td>0.1%</td>
<td>0.04 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Before adding ammonium persulfate and TEMED, the solution may be stored for some days in the dark, but we advise to use it immediately to pour the gel.</p>
<table id="undtbl14">
<caption>5× cleavage buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>500 mM Hepes-KOH pH 7.5</td>
<td>125 mM</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>1M KOAc</td>
<td>250 mM</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>250m Mg(OAc)<sub>2</sub>
</td>
<td>25 mM</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>1M DTT</td>
<td>25 mM</td>
<td>0.025 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>0.375 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Stock solutions can be stored at RT for several months. Make the working solution fresh.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The following reagents are toxic and need to be handled with precautions like stated below:</div>
<p><b>TRIzol/Phenol Chloroform</b>: Toxic by inhalation and contact. Use PPE and handle in the fume hood.</p>
<p><b>Ethanol:</b> Flammable.</p>
<p><b>Acrylamide-bis acrylamide solution (liquid):</b> Toxic and carcinogenic. Use PPE.</p>
<p><b>TEMED</b>: Toxic by inhalation and contact. Use PPE and handle in the fume hood.</p>
<p><b>Tris base</b>: Toxic and irritant. Use PPE.</p>
<p><b>Boric acid</b>: Irritant and carcinogenic. Use PPE.</p>
<p><b>EDTA</b>: Irritant. Use PPE.</p>
<p><b>Urea</b>: Use dust-mask.</p>
<p><b>DTT</b>: Corrosive. Use PPE.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> All reagents and resources listed could be sourced by other suppliers or replaced by equivalent items; however, the impact of these substitutions on the protocol performance has not been tested.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Cleavage reaction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–5 h</div>
<p>The <i>in vitro</i> cleavage reaction will result in digestion of the labeled probe by the endonuclease(s).</p>
<div class="note">
<span class="note-title">Note:</span> We have successfully used both commercial and home-made preparations of recombinant Ago2 in the cleavage reaction. To set the reaction conditions, it is advisable to perform preliminary experiments with variable ratios of the amount of probe and enzyme. A 10- to 100 fold-molar excess of the enzyme over the probe and 5 nM maximum probe concentration is normally used (see table below).</div>
<table id="undtbl15">
<thead>
<tr>
<th> </th>
<th>Test 1</th>
<th>Test 2</th>
<th>Test 3</th>
<th>Test 4</th>
</tr>
</thead>
<tbody>
<tr>
<td>Probe</td>
<td>0.5 nM</td>
<td>1 nM</td>
<td>2.5 nM</td>
<td>5 nM</td>
</tr>
<tr>
<td>Recombinant protein</td>
<td>50 nM</td>
<td>50 nM</td>
<td>50 nM</td>
<td>50 nM</td>
</tr>
</tbody>
</table>
<p>If using a different amount of recombinant protein, change the amount of probe accordingly.</p>
<ol>
<li>In our system, the cleavage reaction required pre-incubation of the probe with cytosolic protein extracts (preparation step 1. e). This step might be omitted if it is not necessary for the cleavage reaction under study. Various control reactions for the specificity of the cleavage site and of the cleavage enzyme can be performed, as suggested in the <a href="#expected-outcomes">expected outcomes</a> section.
<ol type="a">
<li>To pre-incubate the probe with the cytosolic extracts, dilute the purified probe to the desired concentration and thaw on ice one 10 μL aliquot of cytoplasmic extracts/reaction. To each reaction tube add (<a href="#fig6">Figure 6</a>):
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Diagram of a typical cleavage reaction.</div>
</figcaption>
</figure>
<table id="undtbl16">
<thead></thead>
<tbody>
<tr>
<td>Cytoplasmic extract</td>
<td>10 μL</td>
</tr>
<tr>
<td>RNAse inhibitor</td>
<td>1 μL</td>
</tr>
<tr>
<td>Purified probe</td>
<td>According to need</td>
</tr>
</tbody>
</table>
<p class="table-legend">Incubate at 26°C for 90 min, then move the tubes to ice.</p>
</li>
<li>Just before starting the cleavage reaction, thaw the recombinant Ago2 enzyme on ice and dilute it to the desired concentration using a 1× cleavage buffer. Discard diluted unused enzyme. Add to each tube from step a (except the probe only tube) the following and incubate at 26°C for 90 min.
<table id="undtbl17">
<thead></thead>
<tbody>
<tr>
<td>5× cleavage buffer</td>
<td>6 μL</td>
</tr>
<tr>
<td>RNAse inhibitor</td>
<td>1 μL</td>
</tr>
<tr>
<td>0.5 μg/μL yeast RNA</td>
<td>1 μL</td>
</tr>
<tr>
<td>Diluted recombinant Ago2</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Stop the reaction by adding 0.5 mL/tube of TRIzol and purify samples.
<ol type="i">
<li>Samples can be purified according to TRIzol manufacturer’s protocol <a href="https://tools.thermofisher.com/content/sfs/manuals/trizol_reagent.pdf">https://tools.thermofisher.com/content/sfs/manuals/trizol_reagent.pdf</a> and adding 1 μL of Linear acrylamide as co-precipitant, or stored at −20°C <b>STOP point</b>. Avoid prolonged storage to prevent radiolysis of the samples.</li>
<li>After purification, each pellet is resuspended in 20 μL of 2× Gel Loading Buffer from <i>mir</i>Vana kit (or NEB supplemented with 18 mM EDTA) and stored at −20°C <b>STOP point</b> or used immediately for PAGE. In case of storage, the samples should be resolved by PAGE in the next few days to prevent radiolysis.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Denaturing PAGE and auto-radiography</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>The products of the cleavage reaction will be separated by denaturing PAGE and visualized by auto-radiography.</p>
<ol start="2">
<li>Separate the purified products of the cleavage reaction by PAGE.
<ol type="a">
<li>Prepare and pre-run a 8% denaturing polyacrylamide gel (see preparation step 7.a). If needed, change acrylamide concentration according to the size of the reaction products to separate.</li>
<li>Thaw 1 μL of labeled marker (see preparation point 5.a) and add 19 μL of 2× Gel Loading Buffer from <i>mir</i>Vana kit (or NEB supplemented with 18 mM EDTA). 1 μL of marker will suffice, as the exposure will be long enough to get a strong signal.</li>
<li>Denature marker and purified samples from step 2.b for 3 min at 100°C and incubate on ice for 3 min.</li>
<li>Load marker and samples on the pre-warmed 8% denaturing polyacrylamide gel (see preparation point 7.a).</li>
<li>Run at 20–25 mA for about 2.5 h. If possible, leave an empty lane in between the marker and samples to prevent carry over and over-shadowing of the bands. Use the migration pattern of the loading dyes (see table in preparation point 7.b) to decide when to interrupt the electrophoresis.</li>
<li>Remove the glass plates from the running apparatus and cool one of the glass plates by placing ice on it. This will prevent the detachment of the gel from both glass plates when separating them. Open the glass plates by using a spatula as a lever in between and detach the gel from the sides of the glass plates using a scalpel. Wet the gel dropwise with 1–2 mL of RNAse-free water and place on it a piece of 3MM filter paper, roughly cut to size, to lift the gel off the glass plate. Place the gel in between the sheets of a clear polypropylene plastic wallet and use a fluorescent sticker for orientation, if needed.</li>
<li>Expose to an X-ray film in an X-ray cassette or to a PhosphorImager screen. If using films, it is better to expose at −80°C using intensifier screens. It is also useful to place two films, one on top and one underneath the gel, to get a long and short exposure. Using tape, secure the film underneath the gel to both the cassette and the gel, so that it will not move when opening the cassette to develop the top film. <a href="#sec6.5">Troubleshooting Problems 3</a>, <a href="#sec6.7">4</a>, and <a href="#sec6.9">5</a>:</li>
<li>Develop the top film after 18–24 h exposure. Depending on the intensity of the bands, expose the film underneath the gel for a longer time or place a new film on top of the gel for a shorter exposure. <a href="#sec6.11">Troubleshooting Problem 6</a>.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A successful cleavage reaction should show bands at the expected size with little or no background signal (<a href="#fig7">Figure 7</a>). If the enzyme has additional enzymatic activities (such as RNA “trimming” for Ago2), further bands can be observed. The specificity of the cleavage site can be assessed using probes bearing mutations of the cleavage sequence. When testing a known endonuclease for potential cleavage of a new target, a control reaction including a known target for the desired enzyme should be included. Recombinant protein preparations (commercially or home-made generated) should be tested for enzymatic activity on known substrates before performing an experiment on the target of interest. The use of enzymes with defective catalytic activity can further confirm the specific involvement of the endonuclease under consideration in the cleavage reaction and the absence of RNases contaminating the recombinant protein preparations. Finally, the use of probes carrying mutations of the sequence targeted by the endonuclease under study can help to narrow down the exact position of the cleavage site, if not known. When reaction conditions (such as temperature for the cleavage reaction, concentration of enzyme and probe, gel running conditions) are established for the desired enzyme and probe, the protocol can be reproduced very reliably.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Cleavage of the <i>Impa1</i> probe gives rise to a fragment of expected size</div>
<p>A 120 nt probe encompassing the cleavage site for <i>Impa1</i> 3′UTR is digested by Ago2 to produce a fragment of 67 nt. Fragments of smaller size (labeled by an asterisk) are due to a further trimming reaction performed by Ago2 after the cleavage reaction.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Previous knowledge of the cleavage sequence targeted by the endonuclease is necessary to design the appropriate oligo template for the probe synthesis. If the sequence is not known, other techniques like RML-RT-PCR followed by cloning of the cleaved fragment can help to identify the target sequence (<a href="#bib3">Andreassi et al., 2021</a>).</p>
<p>The protocol relies on the use of nucleotides labeled with <sup>32</sup>P. This allows very high sensitivity, however their use is limited by the relative fast decay of this radionucleotide and by the potential hazard for the operator. Training by experienced users and facilities dedicated to the use of radionuclides should be provided before attempting the protocol.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Smears in all lanes or no signal. This could be caused by RNAse contamination of solutions or equipment.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>The protocol assesses cleavage of RNA substrates. As such, strict RNAse-free conditions should be followed in all steps. Working material (gloves, benches, pipettes…) should be cleaned with RNase inactivating solutions and DNase and RNase free tubes and tips should be used.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>No bands in marker and/or sample lanes. This could be caused by failure in the labeling reaction or degradation (see <a href="#sec1.2">in vitro transcription, de-phosphorylation and 5′end-labeling of the RNA marker and of the probe, and full-length probe purification</a>, see preparation steps 3 and 4)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Use fresh reagents. Prior to labeling, run an aliquot of the marker and/or the in vitro transcribed RNA on a Bioanalyzer to verify integrity.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Double bands in any lane (<a href="#fig8">Figure 8</a>). This could be caused by incomplete denaturation of the samples (see <a href="#sec1.2">in vitro transcription, de-phosphorylation and 5′end-labeling of the RNA marker and of the probe, and full-length probe purification</a>, preparation step 7.b and <a href="#sec3.2">denaturing PAGE and auto-radiography</a>, step 3.c).</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Incomplete denaturation of the samples</div>
<p>(A and B) The altered migration pattern caused by incomplete denaturation results in the detection of unexpected bands and the migration of the marker bands at the wrong size (compare A to B).</p>
</figcaption>
</figure>
<h3 id="sec6.6">Potential solution</h3>
<p>Before loading on the gel, boil samples longer and make sure to cool them quickly on ice. Pre-warm the gel before loading. Check urea concentration in the gel mix.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Size of the marker bands are as expected but sample bands are at the wrong size, smeared or there are no bands. This could be caused by RNAse contamination of the samples or failure in the probe synthesis (see <a href="#sec1.2">in vitro transcription, de-phosphorylation and 5′end-labeling of the RNA marker and of the probe and full-length probe purification</a>, preparation steps 3 and 4).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Use fresh reagents. Prior to labeling, run an aliquot of the in vitro transcribed RNA on a Bioanalyzer to verify integrity.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Wobbly bands. This could be caused by poor gel polymerization or the presence of urea in the gel wells (see <a href="#sec1.2">in vitro transcription, de-phosphorylation and 5′end-labeling of the RNA marker and of the probe, and full-length probe purification</a>, preparation steps 7.a and <a href="#sec3.2">denaturing PAGE and auto-radiography</a>, 3.a).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Filter the gel solutions before pouring the gel to remove particulate contaminants. Carefully flush the gel wells with the running buffer before loading the samples.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Bands seem to move after longer exposure (<a href="#fig9">Figure 9</a>). This could be due to radioactivity non-specifically bound by the loading buffer dye that keeps diffusing in the gel when it is left too moist before exposure (See <a href="#sec3.2">denaturing PAGE and auto-radiography</a>, step 3.f).</p>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1240-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Smeared bands due to diffusion of the dye</div>
<p>Appearance of new bands that move during prolonged exposure (labeled by the white asterisks) is due to radioactivity bound nonspecifically by the loading buffer dye that keeps diffusing in the gel when it is left too moist before exposure to X-ray.</p>
</figcaption>
</figure>
<h3 id="sec6.12">Potential solution</h3>
<p>Gently blot excess buffer from the gel before exposure or dry the gel before exposing it.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Andreassi, C., Zimmermann, C., Mitter, R., Fusco, S., De Vita, S., Saiardi, A., and Riccio, A. (2010). An NGF-responsive element targets <i>myo-inositol monophosphatase-1</i> mRNA to sympathetic neuron axons. Nat. Neurosci. <i>13</i>, 291-301. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref1">View at publisher</a></p>
<p id="bib2">Andreassi, C., Crerar, H., and Riccio, A. (2018). Post-transcriptional processing of mRNA in neurons: the vestiges of the RNA world drive transcriptome diversity. Front. Mol. Neurosci. <i>11</i>, 304. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref2">View at publisher</a></p>
<p id="bib3">Andreassi, C., Luisier, R., Crerar, H., Darsinou, M., Blokzijl-Franke, S., Lenn, T., Luscombe, N.M., Cuda, G., Gaspari, M., Saiardi, A., and Riccio, A. (2021). Cytoplasmic cleavage of <i>IMPA1</i> 3′UTR is necessary for maintaining axon integrity. Cell Rep. <i>34</i>, 108778. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref3">View at publisher</a></p>
<p id="bib4">DeCaprio, J. and Kohl, T.O. (2019). Using dounce homogenization to lyse cells for immunoprecipitation. Cold Spring Harb. Protoc. <a class="external-link" href="https://doi.org/10.1101/pdb.prot098574">https://doi.org/10.1101/pdb.prot098574</a></p>
<p id="bib5">Harigaya, Y. and Parker, R. (2010). No-go decay: a quality control mechanism for RNA in translation. Wiley Interdiscip. Rev. RNA <i>1</i>, 132-141. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref5">View at publisher</a></p>
<p id="bib6">Jenal, M., Elkon, R., Loayza-Puch, F., van Haaften, G., Kühn, U., Menzies, F.M., Oude Vrielink, J.A.F., Bos, A.J., Drost, J., Rooijers, K., et al. (2012). The poly(A)-binding protein nuclear 1 suppresses alternative cleavage and polyadenylation sites. Cell <i>149</i>, 538-553. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref6">View at publisher</a></p>
<p id="bib7">Jin, S., Zhan, J., and Zhou, Y. (2021). Argonaute proteins: structures and their endonuclease activity. Mol. Biol. Rep. <i>48</i>, 4837-4849. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref7">View at publisher</a></p>
<p id="bib8">Miyoshi, K., Uejima, H., Nagami-Okada, T., Siomi, H., and Siomi, M.C. (2008). In vitro RNA cleavage assay for Argonaute-family proteins. Methods Mol. Biol. <i>442</i>, 29-43. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref8">View at publisher</a></p>
<p id="bib9">Rodriguez, W., Macveigh-Fierro, D., Miles, J., and Muller, M. (2021). Fated for decay: RNA elements targeted by viral endonucleases. Semin. Cell Dev. Biol. <i>111</i>, 119-125. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00707-3/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Antonella Riccio (<a href="mailto:a.riccio@ucl.ac.uk">a.riccio@ucl.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze datasets/code.</p>
<h3>Acknowledgments</h3>
<p>We are grateful to Fabricio Loayza-Puch and Reuven Agami for advice and sharing their cleavage protocol. We thank Adolfo Saiardi for helping with Ago2 purification and all members of the Riccio lab for helpful discussions. This work was supported by <a href="https://doi.org/10.13039/100004440">Wellcome Trust Investigator</a> Awards 103717/Z/14/Z and 217213_Z_19_Z to A.R. and a Wellcome Trust Institutional Strategic Support Fund 2014 to C.A.</p>
<h3>Author contributions</h3>
<p>C.A. established this protocol and co-wrote the manuscript. A.R. co-wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>