<article>
<section>
<h1 id="header">Protocol to measure contraction, calcium, and action potential in human-induced pluripotent stem cell-derived cardiomyocytes</h1>
<p><time datetime="2021-10-08">Published: October 8, 2021</time></p>
<p>Joe Z. Zhang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Shane Rui Zhao,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Chengyi Tu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Paul Pang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Mao Zhang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Joseph C. Wu<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Stanford Cardiovascular Institute, Stanford University School of Medicine, Stanford, CA, USA</p>
<p id="aff2"><sup>2</sup>Division of Cardiovascular Medicine, Department of Medicine, Stanford University School of Medicine, Stanford, CA, USA</p>
<p id="aff3"><sup>3</sup>Department of Radiology, Stanford University School of Medicine, Stanford, CA, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:joezhang@szbl.ac.cn">joezhang@szbl.ac.cn</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:joewu@stanford.edu">joewu@stanford.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100859">10.1016/j.xpro.2021.100859</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Multiple strategies have been developed to efficiently differentiate human-induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs). Here, we describe a protocol for measuring three key functional parameters of hiPSC-CMs, including contractile function, calcium (Ca<sup>2+</sup>) handling, and action potential.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref15">Zhang et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Automated measurement of hiPSC-CM contractile function</li>
<li>Single-cell calcium imaging of hiPSC-CMs loaded with Fura-2 AM</li>
<li>Imaging-based action potential measurement of hiPSC-CMs</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1064-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Human-induced pluripotent stem cells (hiPSCs) can be requested from Stanford Cardiovascular Institute iPSC Biobank (<a href="https://med.stanford.edu/scvibiobank.html">https://med.stanford.edu/scvibiobank.html</a>) or obtained from various commercial sources. For maintenance of hiPSCs, differentiation of hiPSCs into cardiomyocytes (hiPSC-CMs), expansion and cryopreservation of hiPSC-CMs, please refer to other STAR protocols (<a href="#bib8">Lin and Zou, 2020</a>; <a href="#bib9">Lyra-Leite et al., 2021</a>; <a href="#bib11">Maas et al., 2021</a>). Here, we used a small molecule-based cardiomyocyte differentiation method as described in <a href="#bib15">Zhang et al. (2021)</a>.</p>
<h3 id="sec1.1">Preparation of customized cell chambers for culturing hiPSC-CMs for Ca<sup>2+</sup> imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<div class="note">
<span class="note-title">Note:</span> The following steps should be conducted in a biosafety cabinet.</div>
<ol>
<li>Pour 10 mL of 75% ethanol into a petri dish.</li>
<li>Sterilize glass coverslips by submerging them in 75% ethanol for 5 min.</li>
<li>Air dry coverslips.</li>
<li>Cut off the lid and bottom half of 0.5 mL Eppendorf tubes using a razor blade (<a href="#fig1">Figures 1</a>A–1E).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1064-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Customized cell chambers for Ca<sup>2+</sup> imaging</div>
<p>(A) Required materials for making customized cell chambers.</p>
<p>(B–G) The steps of making customized chambers. The small tubes are made by cutting off the lid and bottom half of 0.5 mL Eppendorf tubes using a razor blade. The grease is used for attaching tubes on coverslips. The paint brush is used for painting grease on the bottom side (not the cut side) of tubes.</p>
<p>(H) Example of customized cell chambers. Tubes with grease are put at the center of the coverslip. By gentle pressing the top of tubes, tubes are attached to the coverslips with a good seal.</p>
</figcaption>
</figure>
</li>
<li>Carefully paint a thin layer of grease around the ring (not the cut side) of the Eppendorf tube (<a href="#fig1">Figure 1</a>F).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not put too much grease.</div>
<ol start="6">
<li>Put the tube on top of the glass coverslip and gently press the top of the tube to form a seal (<a href="#fig1">Figure 1</a>G).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend putting two tubes on one coverslip.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the tube does not slide on the coverslip when pressing, as this may leave a grease smear on the coverslip and affect cell attachment.</div>
<ol start="7">
<li>Place two coverslips in a petri dish (<a href="#fig1">Figure 1</a>H).</li>
<li>Leave the customized cell chambers under UV exposure in a biosafety cabinet for 1 h.</li>
<li>After 1 h, seal the petri dish using parafilm for further use.</li>
</ol>
<h3 id="sec1.2">Preparation of Matrigel-coated plates/coverslips/dishes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="10">
<li>Thaw a bottle of Growth Factor Reduced (GFR) Matrigel (10 mL) for 18 h at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The following steps should be conducted in a biosafety cabinet.</div>
<ol start="11">
<li>Aliquot 500 μL of Matrigel in sterile 1.5 mL Eppendorf tubes and store aliquoted Matrigel in a −20°C freezer for further use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Matrigel should be kept on ice at all times during the procedure to avoid solidification.</div>
<ol start="12">
<li>Pipette 200 mL of ice-cold DMEM/F12 in a sterile plastic bottle and keep the bottle on ice.</li>
<li>Take out one aliquot of Matrigel and leave it on ice to thaw.</li>
<li>Add 500 μL of Matrigel in ice-cold DMEM/F12 medium at a concentration of 1:400 dilution (v/v).</li>
<li>Rinse the Eppendorf tube with 1 mL of DMEM/F12 to collect the remaining Matrigel in the Eppendorf tube and add it to the bottle.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Diluted Matrigel medium (DMEM/F12/Matrigel) can be stored in a 4°C fridge. We recommend using up diluted Matrigel medium within one week.</div>
<ol start="16">
<li>Add 100 μL of DMEM/F12/Matrigel to each well of 96-well plates that will be used for seeding hiPSC-CMs for contraction measurement.</li>
<li>Add 100 μL of DMEM/F12/Matrigel to each customized cell chamber that will be used for seeding hiPSC-CMs for Ca<sup>2+</sup> imaging.</li>
<li>Add 500 μL of DMEM/F12/Matrigel to each 35 mm glass-bottom cell culture dish that will be used for seeding hiPSC-CMs for action potential measurement.</li>
<li>Transfer coated plates/coverslips/dishes to a humidified cell culture incubator at 37°C for 18 h before use.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>RPMI 1640 Medium</td>
<td>Life Technologies</td>
<td>11875-119</td>
</tr>
<tr>
<td>B27 supplements</td>
<td>Life Technologies</td>
<td>17504-044</td>
</tr>
<tr>
<td>Growth Factor Reduced (GFR) Basement Membrane Matrix, Phenol Red-Free, LDEV-Free</td>
<td>Corning</td>
<td>356231</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>Gibco/Life Technologies</td>
<td>12634028</td>
</tr>
<tr>
<td>Accutase</td>
<td>Sigma-Aldrich</td>
<td>A6964</td>
</tr>
<tr>
<td>KnockOut Serum Replacement</td>
<td>Thermo Fisher Scientific</td>
<td>10828-028</td>
</tr>
<tr>
<td>Pluronic™ F-127 (20% Solution in DMSO)</td>
<td>Thermo Fisher Scientific</td>
<td>P3000MP</td>
</tr>
<tr>
<td>Fura-2, AM, cell permeant</td>
<td>Thermo Fisher Scientific</td>
<td>F1221</td>
</tr>
<tr>
<td>DPBS, no calcium, no magnesium</td>
<td>Fisher Scientific</td>
<td>14-190-250</td>
</tr>
<tr>
<td>NaCl</td>
<td>Sigma-Aldrich</td>
<td>S9888</td>
</tr>
<tr>
<td>KCl</td>
<td>Sigma-Aldrich</td>
<td>529552</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> 1 M in H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>21115</td>
</tr>
<tr>
<td>Glucose</td>
<td>Sigma-Aldrich</td>
<td>G8270</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>Sigma-Aldrich</td>
<td>M8266</td>
</tr>
<tr>
<td>HEPES</td>
<td>Sigma-Aldrich</td>
<td>H3375</td>
</tr>
<tr>
<td>NaOH 10 M in H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>72068</td>
</tr>
<tr>
<td>Trypan blue stain</td>
<td>Life Technologies</td>
<td>T10282</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Human induced pluripotent stem cell lines</td>
<td>Stanford Cardiovascular Institute Biobank</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>ASAP2 lentivirus</td>
<td>ASAP2 plasmid is from Michael Lin’s Lab</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Polystyrene Storage Bottle, 250 mL</td>
<td>VWR</td>
<td>8390</td>
</tr>
<tr>
<td>6-Well culture plate</td>
<td>E &amp; K Scientific</td>
<td>EK-27160</td>
</tr>
<tr>
<td>96-Well flat clear bottom black polystyrene TC-treated microplates</td>
<td>Corning</td>
<td>3603</td>
</tr>
<tr>
<td>35 mm Glass bottom dish with 20 mm micro-well #1.5 cover glass</td>
<td>Cellvis</td>
<td>D35-20-1.5-N</td>
</tr>
<tr>
<td>Rectangular glass coverslips</td>
<td>Warner Instruments</td>
<td>64-0709</td>
</tr>
<tr>
<td>100 mm × 20 mm Petri dish, sterile</td>
<td>Sigma-Aldrich</td>
<td>P5606</td>
</tr>
<tr>
<td>Dow Corning High Vacuum Grease</td>
<td>Sigma-Aldrich</td>
<td>Z273554-1EA</td>
</tr>
<tr>
<td>Fisherbrand Razor Blades</td>
<td>Fisher Scientific</td>
<td>12-640</td>
</tr>
<tr>
<td>Nikon Immersion Oil (Type F) 30cc PFS</td>
<td>Morrell Instrument Company</td>
<td>MXA22168</td>
</tr>
<tr>
<td>Conical tubes, 15 mL</td>
<td>Corning Falcon</td>
<td>352097</td>
</tr>
<tr>
<td>LoBind micro-centrifuge tubes</td>
<td>Fisher Scientific</td>
<td>13-698-794</td>
</tr>
<tr>
<td>5 mL Pipettes sterile, individually wrapped</td>
<td>Fisher Basix</td>
<td>14-955-232</td>
</tr>
<tr>
<td>10 mL Pipettes sterile, individually wrapped</td>
<td>Fisher Basix</td>
<td>14-955-234</td>
</tr>
<tr>
<td>2 mL Aspiration pipettes sterile, individually wrapped</td>
<td>Corning Falcon</td>
<td>357558</td>
</tr>
<tr>
<td>Reservoir, 10 mL sterile</td>
<td>MTC Bio</td>
<td>P8010-5S</td>
</tr>
<tr>
<td>Tube micro centrifuge, 0.5 mL</td>
<td>Dynalon Labware</td>
<td>299265</td>
</tr>
<tr>
<td>Nalgene™ Rapid-Flow™ Sterile Single Use Vacuum Filter Units</td>
<td>Fisher Scientific</td>
<td>09-741-04</td>
</tr>
<tr>
<td>Paint brush</td>
<td>Amazon</td>
<td>N/A</td>
</tr>
<tr>
<td>Narrow bath chamber with field stimulation</td>
<td>Warner Instruments</td>
<td>RC-27NE2</td>
</tr>
<tr>
<td>Magnetic heated platforms</td>
<td>Warner Instruments</td>
<td>PM-6</td>
</tr>
<tr>
<td>Class II, type A2 or better biosafety cabinet</td>
<td>Thermo Fisher Scientific</td>
<td>1300 Series Class II</td>
</tr>
<tr>
<td>CO<sub>2</sub> laboratory incubator</td>
<td>Laboratory Equipment Company</td>
<td>MCO-170AICUVL-PA</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>VWR</td>
<td>Centrifuge 5810 R</td>
</tr>
<tr>
<td>pH meter</td>
<td>Hanna Instruments</td>
<td>HI2002-01</td>
</tr>
<tr>
<td>Lab companion stirring hot plate, ceramic top</td>
<td>Cole-Parmer</td>
<td>8657808</td>
</tr>
<tr>
<td>Magnetic stirring bars</td>
<td>Fisher Scientific</td>
<td>14-513-68</td>
</tr>
<tr>
<td>Aspirator vacuum</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Countess® Cell Counting Chamber Slides</td>
<td>Thermo Fisher Scientific</td>
<td>C10312</td>
</tr>
<tr>
<td>Luna-FL automated fluorescence cell counter</td>
<td>WISBIOMED</td>
<td>LB-L20001</td>
</tr>
<tr>
<td>Sony SI8000 Cell Motion Imaging System</td>
<td>Sony Biotechnology</td>
<td>N/A</td>
</tr>
<tr>
<td>Lambda DG 4</td>
<td>Sutter Instrument Company</td>
<td>N/A</td>
</tr>
<tr>
<td>Zeiss LSM 710 confocal microscope</td>
<td>Zeiss</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism version 7.0</td>
<td>GraphPad Software</td>
<td>N/A</td>
</tr>
<tr>
<td>ImageJ</td>
<td>NIH</td>
<td>N/A</td>
</tr>
<tr>
<td>Sony SI8000 cardiac analysis software</td>
<td>Sony Biotechnology</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<ul>
<li>
<b>hiPSC-CM culture medium:</b> thaw a bottle of 50x B27 supplements (10 mL) and add B27 supplements to a bottle of RPMI 1640 medium (500 mL) (RPMI/B27).</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> We recommend aliquoting 50x B27 supplements (stored in −20°C and used before the expiration date) and preparing RPMI/B27 weekly according to the required amount.</div>
<ul>
<li>
<b>hiPSC-CM seeding medium:</b> thaw a bottle of KnockOut serum replacement (KOSR) (500 mL) and aliquot the KOSR in 50 mL Falcon tubes (20 mL/tube and stored in −20°C) and prepare 10% KOSR RPMI/B27 by adding 20 mL of KOSR into 180 mL of RPMI/B27.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> This is the seeding medium that can be stored at 4°C for up to a month.</div>
<ul>
<li>
<b>hiPSC-CM lentivirus infection medium:</b> thaw a vial of the accelerated sensor of action potentials 2 (ASAP2) lentivirus (1.0 × 10<sup>8</sup> TU/mL) on ice, then add 1 μL of the virus into 1 mL of hiPSC-CM culture medium.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Keep the virus on ice at all times. If the virus titer is different, the dilution should be adjusted according to the virus titer. The infection medium should be prepared on the day of use.</div>
<ul>
<li>
<b>10× Tyrode’s stock solution:</b> Dissolve the chemicals (see the table below) in 500 mL of distilled water to make 10× Tyrode’s stock solution with the defined concentrations.</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Amount (g)</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>1350</td>
<td>39.45</td>
</tr>
<tr>
<td>KCl</td>
<td>54</td>
<td>2.01</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>10</td>
<td>0.48</td>
</tr>
<tr>
<td>HEPES</td>
<td>100</td>
<td>11.92</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>To final volume of 500 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span>  The stock solution can be stored at 4°C for up to one year.</div>
<ul>
<li>
<b>Tyrode’s solution:</b> add 5 mL of 10× Tyrode’s stock solution, 90 μL of 1 M CaCl<sub>2</sub>, and 0.045 g of glucose in 40 mL of distilled water in a beaker (see the table below). Adjust the pH of the solution to 7.4 with NaOH. Add additional distilled water to bring the solution to a final volume of 50 mL. Consequently, Tyrode’s solution contains NaCl (135 mM), KCl (5.4 mM), MgCl<sub>2</sub> (1 mM), CaCl<sub>2</sub> (1.8 mM), glucose (5 mM), and HEPES (10 mM).</li>
</ul>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× Tyrode’s stock solution</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>1 M CaCl<sub>2</sub>
</td>
<td>1.8 mM</td>
<td>90 μL</td>
</tr>
<tr>
<td>Glucose</td>
<td>5 mM</td>
<td>0.045 g</td>
</tr>
<tr>
<td>Distilled water</td>
<td>-</td>
<td>To final volume of 50 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span>  As Tyrode’s solution contains glucose that can be easily contaminated, we do not recommend storing it for a long time. Tyrode’s solution should be prepared in fresh on the day of the experiment.</div>
<ul>
<li>
<b>Fura-2 AM indicator:</b> Add 125 μL of Pluronic™ F-127 (20% solution in DMSO) into one vial of Fura-2 AM (50 μg) to make 0.4 mM stock of Fura-2 AM indicator solution. Aliquot 5 μL of Fura-2 AM indicator in a 1.5 mL Eppendorf tube, store it at −20°C, and protect it from light.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Fura-2 AM indicator may take a few minutes to fully dissolve. Aliquoted Fura-2 AM can be stored up to one year.</div>
<ul>
<li>
<b>Fura-2 AM loading solution:</b> Take out one aliquot (5 μL) of Fura-2 indicator solution and add 995 μL of Tyrode’s solution to the Eppendorf tube to make the loading solution with 2 μM Fura-2 AM and 0.1% F-127.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> Loading solution should be prepared in fresh on the day of the experiment and protected from light.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> For contraction measurement, this protocol uses Sony SI8000 Cell Motion Imaging System for imaging and analyses. Other microscopes equipped with a high-speed camera can also be used for recording hiPSC-CM contraction motion. For analyses, open-source motion tracking software is also available (<a href="#bib4">Huebsch et al., 2015</a>).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> For Ca<sup>2+</sup> imaging, this protocol uses Lambda DG 4 ultra-high-speed wavelength switching illumination system to excite Fura-2 AM. Others systems such as IonOptix can also be used for Fura-2 AM imaging.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> For action potential measurement, this protocol uses ASAP2 lentivirus to record membrane potential (<a href="#bib1">Chamberland et al., 2017</a>; <a href="#bib2">Chour et al., 2021</a>; <a href="#bib16">Zhang et al., 2019</a>, <a href="#bib15">2021</a>). Other probes such as FluoVolt can also be used for action potential measurement. This protocol uses Zeiss LSM 710 confocal microscope for action potential imaging. Other confocal microscopes such as Nikon AX/AXR can also be used for imaging.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Enzymatic dissociation of hiPSC-CMs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>Day 23 hiPSC-CMs cultured in 6-well plates can be dissociated by enzymes for re-plating into desired imaging plates/coverslips/dishes at a defined cell density.</p>
<div class="note">
<span class="note-title">Note:</span> hiPSC-CMs with the purity over 95% is necessary for the reproducible measurement.</div>
<ol>
<li>Aspirate the medium, wash hiPSC-CMs with DPBS twice, and incubate cells with Accutase (1 mL/well) for 6 min at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The incubation time may vary for different cell lines or culture conditions. We recommend observing cells under a microscope regularly to ensure that cells are not over-digested. A gentle tap on one side of the plate helps to agitate cells and improve the dissociation efficiency. Prolonged incubation of cells with enzymes may reduce cell viability.</div>
<ol start="2">
<li>After incubation for 6 min, gently shake the plate and pipette the solution using a 1 mL pipette several times to fully dissociate the monolayer into single cells.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> hiPSC-CMs should be dissociated into single cells to ensure accurate cell count.</div>
<ol start="3">
<li>Transfer the dissociated cell suspension into a 15 mL Falcon tube containing 8 mL of seeding medium.</li>
<li>Add 1 mL of seeding medium to the well and wash off the remaining cells from the well; then transfer the cell suspension to the Falcon tube to give a final volume of 10 mL.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If a large number of cells are required, multiple wells of hiPSC-CMs can be dissociated together and the cell suspension can be collected in a 50 mL Falcon tube.</div>
<ol start="5">
<li>Centrifuge the cell suspension at 300 × <i>g</i> for 3 min at room temperature (22°C).</li>
<li>Aspirate the supernatant and re-suspend cells in 1 mL of 37°C pre-warmed seeding medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If multiple wells of hiPSC-CMs are dissociated, we recommend re-suspending the cell pellet according to the rule of 1 mL of seeding medium per well of 6-well plate.</div>
<ol start="7">
<li>Mix 20 μL of cell suspension with 1:1 Trypan blue in a 0.5 mL Eppendorf tube.</li>
<li>Transfer 15 μL of stained cell suspension in a Countess® Cell Counting Chamber Slide (each slide has a duplicate) and count cell number by the Luna-FL automated fluorescence cell counter. The cell suspension is ready for re-plating.</li>
</ol>
<h3 id="sec3.2">Seeding cells for imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min (15 min every other day)</div>
<ol start="9">
<li>Remove the coating Matrigel from each well/dish/customized chamber.</li>
<li>Adjust the cell suspension to the density of 500,000 cells/mL in the seeding medium, transfer the cell suspension into a 10 mL reservoir, and add 100 μL of the cell suspension to each well of the 96-well plate.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To re-plate cells in a 96-well plate, a multi-channel pipette should be used. To avoid cells from settling down at the bottom of the reservoir, we recommend pipetting up and down the cell suspension several times before re-plating.</div>
<ol start="11">
<li>Add 50 μL of cell suspension and 50 μL of seeding medium to each customized cell chamber on the coverslip for Ca<sup>2+</sup> imaging.</li>
<li>Add 200 μL of cell suspension and 300 μL of seeding medium to each 35 mm glass-bottom dish for action potential measurement.</li>
<li>After 24 h, aspirate the seeding medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend putting a 1 mL non-filtered tip over the 2 mL aspiration pipette and ensure that the tip does not touch cells.</div>
<ol start="14">
<li>Add 100 μL of hiPSC-CM culture medium to each well in a 96-well plate or customized cell chamber. Replace the medium every other day until recording.</li>
<li>Add 500 μL of the hiPSC-CM culture medium to 35 mm glass-bottom dishes. Replace the medium every other day until recording.</li>
</ol>
<h3 id="sec3.3">Automated recording of hiPSC-CM contraction motion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h/96-well plate</div>
<ol start="16">
<li>Turn on the microscope, camera, CO<sub>2</sub> supply, and heating unit 10 min before use and set the heating unit temperature to 37°C.</li>
<li>Add the desired amount of water to the water jacket of the plate holder (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1064-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Measurement of contractile function</div>
<p>(A) Sony SI8000 Cell Motion Imaging System (hardware).</p>
<p>(B) The interface of Sony recording software.</p>
<p>(C) Representative contraction-relaxation traces of hiPSC-CMs and schematic diagrams showing the measurement of contraction velocity (CV) and relaxation velocity (RV).</p>
</figcaption>
</figure>
</li>
<li>Put the 96-well plate on the plate holder and allow cells to stabilize for 5 min before recording.</li>
<li>In the software interface, activate the auto-focus and auto-brightness functions. Adjust the camera frame rate to 75 frames per second (fps) and the video/image format to 1024 × 1024 pixels (<a href="#fig2">Figure 2</a>B). The total recording time is 10 s. Live-cell imaging is performed by an inverted Nikon Eclipse Ti microscope (USA) equipped with a 10× objective (NA: 0.3).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To avoid the biased selection of a region of interest (ROI), the center of each well is imaged. During recording, ensure the vibration of the microscope table is minimal.</div>
<ol start="20">
<li>Open the Sony SI8000 cardiac analysis software; the motion detection and analyses are conducted automatically (<a href="#fig2">Figure 2</a>C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is essential to detect contraction start, contraction peak, contraction end, relaxation peak, and relaxation end for each beating cycle to complete analyses.</div>
<h3 id="sec3.4">Single-cell Ca<sup>2+</sup> imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="21">
<li>Prepare Tyrode’s solution as described in the <a href="#materials-and-equipment">materials and equipment</a> section.</li>
<li>Prepare the Fura-2 AM loading solution as described in the <a href="#materials-and-equipment">materials and equipment</a> section.</li>
<li>Aspirate the culture medium in the customized cell chambers and wash the hiPSC-CMs twice with 100 μL of Tyrode’s solution.</li>
<li>Load 100 μL of Fura-2 AM loading solution to hiPSC-CMs in the customized cell chambers.</li>
<li>Incubate the hiPSC-CMs at room temperature for 10 min.</li>
<li>Remove Fura-2 AM loading solution from the hiPSC-CMs and wash twice with 100 μL of Tyrode’s solution.</li>
<li>Incubate the hiPSC-CMs in Tyrode’s solution for another 5 min at room temperature to allow de-esterification of Fura-2 AM.</li>
<li>Put the coverslips in the magnetic heated platform (<a href="#fig3">Figure 3</a>A, step 1).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1064-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Equipment used for Ca<sup>2+</sup> imaging</div>
<p>(A) Steps of assembling coverslips, bath chamber, and the magnetic platform. Step 1: put the coverslip with tubes in the slot of magnetic heated platform; Step 2: remove tubes and put the bath chamber on top of the coverslip; and Step 3: clamp the bath chamber in place by magnetic clamps and add 1 mL of Tyrode’s solution.</p>
<p>(B) Picture of the bath chamber that has been connected with the heating and stimulation components.</p>
<p>(C) Picture of the microscope used for Ca<sup>2+</sup> imaging.</p>
</figcaption>
</figure>
</li>
<li>Remove tubes on the coverslip and put the narrow bath chamber with grease at the bottom on top of the coverslip (<a href="#fig3">Figure 3</a>A, step 2).</li>
<li>Secure the bath chamber in place using the magnetic clamps and ensure there is no leak (<a href="#fig3">Figure 3</a>A, step 3).</li>
<li>Add 1 mL of Tyrode’s solution into the chamber (<a href="#fig3">Figure 3</a>A, step 3).</li>
<li>Add a drop of immersion oil on the objective.</li>
<li>Place the magnetic heated platform in a stage adapter on the stage of an inverted epifluorescence microscope (Nikon Eclipse Ti-S, USA) equipped with a 40× oil immersion objective (0.95 NA) (<a href="#fig3">Figures 3</a>B and 3C).</li>
<li>Connect the temperature controller wires with the bath chamber.</li>
<li>Put the temperature sensor in the bath chamber.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Secure the sensor in place during the recording.</div>
<ol start="36">
<li>Set the temperature controller at 37°C.</li>
<li>Connect the field stimulation electrodes with a stimulator.</li>
<li>Set the field stimulation at 0.5 Hz with a pulse duration of 20 ms.</li>
<li>Fura-2 AM loaded cells are excited with Lambda DG 4 ultra-high-speed wavelength switching light source (Sutter Instrument, USA) at both 340 and 380 nm at fast-switching mode and the emission fluorescence signals are collected at 510 nm with the iXon Ultra 897 EMCCD camera (Andor Technology, UK). Find a region of interest (ROI) under the eye pieces first and then switch to the live view by camera.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The light source switching mode can be set as indicated in the operation manual of Lambda DG 4. Ensure cells are focused under the live view by camera.</div>
<ol start="40">
<li>Click “Run now” to start recording under the ND Acquisition mode in NIS Elements AR software (<a href="#fig4">Figure 4</a>A). Exposure time for each wavelength is 20 ms and the total recording time is 20 s.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1064-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Software interface of Ca<sup>2+</sup> transient recording and the first-step analysis</div>
<p>(A) The picture of the acquisition mode interface.</p>
<p>(B) Example of the region of interest showing the cell selection. Scale bar is 100 μm.</p>
<p>(C) Example of the time measurement of Ca<sup>2+</sup> imaging, the color code corresponds to the cells shown in panel B.</p>
<p>(D) Representative trace of Ca<sup>2+</sup> transients.</p>
</figcaption>
</figure>
</li>
<li>The photons are detected by the camera and converted to arbitrary intensity values. Changes in fluorescence intensity of individual hiPSC-CMs over time are recorded by the NIS Elements AR software, which permits the recording of multiple cells.</li>
<li>After recording, using the ROI selection tool to select individual hiPSC-CM in ROI as well as an area without any cells for background subtraction (<a href="#fig4">Figure 4</a>B).</li>
<li>Measure the fluorescence intensity using the time measurement function (<a href="#fig4">Figure 4</a>C).</li>
<li>Export the data to Excel for further analyses.</li>
<li>Analyze the data using an open-source Excel-based program as previously published (<a href="#bib3">Greensmith, 2014</a>) (<a href="#fig4">Figure 4</a>D).</li>
</ol>
<h3 id="sec3.5">Fluorescence-based single-cell action potential imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="46">
<li>Three days before imaging, prepare the lentivirus infection medium as described in <a href="#materials-and-equipment">materials and equipment</a> and add 300 μL to each dish with the multiplicity of infection (MOI) of three.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use the minimum required volume for infection to improve efficiency.</div>
<ol start="47">
<li>After 24 h, aspirate the infection medium.</li>
<li>Add 500 μL of hiPSC-CM culture medium to each dish until imaging.</li>
<li>On the day of imaging, turn on the confocal microscope and incubation system.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The incubation system is essential for live-cell imaging. The temperature of the imaging chamber is kept at 37°C and is humidified to avoid evaporation of solution during imaging.</div>
<ol start="50">
<li>Remove the medium and wash cells twice with 1 mL of Tyrode’s solution.</li>
<li>Remove the washing solution, add 1 mL of Tyrode’s solution to the dish, and place the dish onto a holding adapter to allow cells to stabilize for 5 min before imaging. Live-cell imaging is performed by an inverted Zeiss LSM 710 confocal microscope equipped with a 20× objective (NA: 0.8). The line-scan mode is used for measuring the action potential of single cells (<a href="#fig5">Figures 5</a>A and 5B).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1064-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Imaging-based fluorescence measurement of the action potential</div>
<p>(A) The acquisition interface of the line-scan mode.</p>
<p>(B) Representative line-scan image of the action potential.</p>
<p>(C) Representative fluorescence signals of action potential trace.</p>
<p>(D) Example of single action potential trace (shown in the box in panel C) showing the measurement of amplitude and action potential duration (APD).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is essential to allow cells to adapt to Tyrode’s solution before imaging. Cells usually start to show changes in fluorescence (changes in membrane potential) within 1 min after switching to Tyrode’s solution based on our experience.</div>
<ol start="52">
<li>Live cell imaging of single cells is conducted in a line scan mode with an excitation wavelength of 488 nm and the emission is filtered with a band-pass filter (493–630 nm). The scanning line is selected across the cell membrane with the maximal scanning speed. The image size is 512 × 1 pixels with the total recording time of 8 s. The recording is conducted using the Zen software (USA) and images are saved as LSM files.</li>
<li>For analyses, line-scan images are opened in Image J (USA). The fluorescence intensity of images is inverted using the Invert function under the Edit in ImageJ. The entire image is selected (Ctrl+A) to plot the profile of images (Ctrl+Alt+K). The amplitude, action potential duration (APD) 30, 50, 70, and 90 can be determined accordingly (<a href="#fig5">Figures 5</a>C and 5D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The cycle of scanning can be adjusted according to the experimental condition. If hiPSC-CMs beat slow, more recording time is required to allow recording multiple cycles of action potential.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Here we introduce a protocol to measure hiPSC-CM contraction, Ca<sup>2+</sup> handling, and action potential. Our previous studies have applied these approaches to investigate inherited cardiomyopathies and drug-induced cardiotoxicity (<a href="#bib5">Kitani et al., 2019</a>; <a href="#bib6">Lan et al., 2013</a>; <a href="#bib7">Lee et al., 2019</a>; <a href="#bib10">Ma et al., 2018</a>; <a href="#bib12">Seeger et al., 2019</a>; <a href="#bib13">Sharma et al., 2018</a>; <a href="#bib14">Wu et al., 2019</a>). This protocol provides an imaging-based assessment of hiPSC-CM functions, which does not require extensive technical training. When following the protocol, individuals who have cell culture and microscope operation experience will usually succeed in the recording in the first attempt. For the automated contraction measurement, over 100 experimental conditions with triplicates for each condition can be screened within a few hours. Moreover, this approach provides an opportunity for experimenters to repeatedly monitor the contractile function of the same ROI at different time points.</p>
<p>We have measured the stimulated Ca<sup>2+</sup> transients in hiPSC-CMs by Ca<sup>2+</sup> imaging. The arbitrary values of fluorescence intensity are measured by NIS Elements AR software and exported to an Excel sheet. After analyzing with the open-source Excel-based program (<a href="#bib3">Greensmith, 2014</a>), we can obtain the F340/F380 ratio over time for each cell. A representative raw trace of stimulated Ca<sup>2+</sup> transients is shown in <a href="#fig4">Figure 4</a>D. In addition, we can obtain the values of three parameters (diastolic Ca<sup>2+</sup>, amplitude, and Ca<sup>2+</sup> decay tau of stimulated Ca<sup>2+</sup> transients) based on the Excel-based program. According to our experience, up to 1000 cells can be recorded per coverslip in one hour if necessary.</p>
<p>Optical imaging of hiPSC-CM action potential provides an accessible method to examine the electrophysiology of hiPSC-CMs. Although the single-cell patch-clamp is regarded as the gold-standard approach to determine the electrophysiological function of hiPSC-CMs, it requires highly specified equipment that is not always accessible. Moreover, as it requires extensive training for new users to master the skills, the labor-intensive nature of the patch-clamp hinders its application for high throughput studies. As an alternative, here we introduce an imaging-based fluorescence measurement of action potential using a highly sensitive and fast voltage sensor. This approach only requires minimal training in operating the confocal microscope. The brightness of fluorescence signals represents the resting membrane potential. When hiPSC-CMs depolarize, conformational changes in the ASAP2 lead to a reduction in the fluorescence signals. The line scan mode is normally utilized, and the detector gain is set at 600 with a digital gain of three. The scanning line is across the cell membrane and each line comprises 512 pixels. After a sequential scanning, a two-dimensional (2D) image of 512 × 2000 lines can be generated and stored for offline analyses. The fluorescence-based imaging approach allows experimenters to record over 100 cells per hour, which significantly improves the scale of throughput.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analyses">Quantification and statistical analyses</h2>
<p>Statistical analyses and graph generation can be done using GraphPad Prism 9 (Graphpad, USA).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Motion-based hiPSC-CM contraction measurement provides an easy and accessible approach to determine the hiPSC-CM contractile function. However, the major disadvantage of this approach is that it records the motion velocity when hiPSC-CMs contract and relax, so it does not report contractile force directly. Another disadvantage of this approach is that if hiPSC-CMs do not form a syncytium and contract coherently, the camera could capture multiple contraction peaks, which may make the analysis difficult.</p>
<p>Fura-2 AM, a ratiometric Ca<sup>2+</sup> indicator, has been widely used because of advantages such as allowing the quantitative determination of the cytosolic Ca<sup>2+</sup> level and being unaffected by uneven dye distribution and photo-bleaching. However, UV-excitation may lead to high background noise and bring photo-toxicity due to the high energy. Dual excitation dye requires fast filter switching, which is not suitable for confocal microscopy. In addition, like other Ca<sup>2+</sup> dyes, Fura-2 AM may compartmentalize into intracellular compartments, making it challenging to indicate the cytosolic Ca<sup>2+</sup> level. Either loading at lower temperatures or using a dextran-conjugated dye can be used to minimize this problem.</p>
<p>ASAP2 is a fast and sensitive voltage indicator (<a href="#bib1">Chamberland et al., 2017</a>). However, given the difficulty of transfection of hiPSC-CMs, the construct of the ASAP2 viral vector is required to achieve the desired efficiency of ASAP2 transduction in hiPSC-CMs. Another disadvantage of fluorescence-based imaging is the measurement of relative changes in membrane potential. In other words, this approach cannot provide the absolute recording of maximal diastolic membrane potential.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>There are a lot of dead hiPSC-CMs floating 24 h after re-plating (step 11).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Over-digestion or under-digestion (leading to a forceful blow to attached cells) usually reduces cell viability. Continuous monitoring of cells during dissociation every minute under a microscope may be necessary. Adding 10 μM Y27632 in the hiPSC-CM seeding medium could improve cell viability. Another reason is the low attachment of hiPSC-CMs, and it may be necessary to use newly prepared Matrigel or increase the dilution to 1:200.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>No monolayer is formed for contraction measurement (step 11).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>We typically observe a monolayer when seeding 50,000 hiPSC-CMs per well in a 96-well plate with over 90% viability and attachment 1-week after re-plating. If cell viability is low, more cells could be seeded per well.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>If hiPSC-CMs are clustered in the center on coverslips/dishes, is this a problem (step 13)?</p>
<h3 id="sec7.6">Potential solution</h3>
<p>The hiPSC-CMs need to be equally distributed over coverslips/dishes to allow single-cell recording. After seeding cells, using a pipette to give a gentle blow may help equalize the distribution. Do not swirl the dish, as this may lead to cell aggregation in the center.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>There are small spikes on the contraction-relaxation graphs (step 21).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>These small spikes could be caused by the vibration of the microscope table, or floating dead cells moving around during recording. If this “noise” does not affect the detection of contraction start, contraction peak, contraction end, relaxation peak, and relaxation end, it can be ignored. Ensure the microscope table is stable during recording. Change the medium the day before recording can help remove the cell debris, which may solve the problem.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>No Ca<sup>2+</sup> transients are observed and recorded in Ca<sup>2+</sup> imaging (step 40).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Several points may cause this problem. First, make sure that hiPSC-CMs are in good conditions and beat normally before the start of Ca<sup>2+</sup> imaging. Second, check whether Tyrode's stock solution has bacterial contamination. If there is contamination, filter the solution using a Rapid-Flow sterile filter unit with a pore size of 0.2 μm. Finally, the pH of the freshly prepared Tyrode's solution should be accurately adjusted.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Motion artifact during Ca<sup>2+</sup> or action potential recording (steps 41 and 53).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>When single-cells are recorded, we hardly detect the motion artifact. Avoid the regions where monolayers are formed.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>No spontaneous electrical signals (step 52).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Ensure hiPSC-CMs are in good condition. After switching to Tyrode’s solution, give hiPSC-CMs more time to adapt to the new solution.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chamberland, S., Yang, H.H., Pan, M.M., Evans, S.W., Guan, S., Chavarha, M., Yang, Y., Salesse, C., Wu, H., Wu, J.C., et al. (2017). Fast two-photon imaging of subcellular voltage dynamics in neuronal tissue with genetically encoded indicators. Elife <i>6</i>, e25690. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref1">View at publisher</a></p>
<p id="bib2">Chour, T., Tian, L., Lau, E., Thomas, D., Itzhaki, I., Malak, O., Zhang, J.Z., Qin, X., Wardak, M., Liu, Y., et al. (2021). Method for selective ablation of undifferentiated human pluripotent stem cell populations for cell-based therapies. JCI insight <i>6</i>, 10. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref2">View at publisher</a></p>
<p id="bib3">Greensmith, D.J. (2014). Ca analysis: an Excel-based program for the analysis of intracellular calcium transients including multiple, simultaneous regression analysis. Comput. Methods Programs Biomed. <i>113</i>, 241-250. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref3">View at publisher</a></p>
<p id="bib4">Huebsch, N., Loskill, P., Mandegar, M.A., Marks, N.C., Sheehan, A.S., Ma, Z., Mathur, A., Nguyen, T.N., Yoo, J.C., Judge, L.M., et al. (2015). Automated video-based analysis of contractility and calcium flux in human-induced pluripotent stem cell-derived cardiomyocytes cultured over different spatial scales. Tissue Eng. Part C Methods <i>21</i>, 467-479. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref4">View at publisher</a></p>
<p id="bib5">Kitani, T., Ong, S.G., Lam, C.K., Rhee, J.W., Zhang, J.Z., Oikonomopoulos, A., Ma, N., Tian, L., Lee, J., Telli, M.L., et al. (2019). Human-induced pluripotent stem cell model of trastuzumab-induced cardiac dysfunction in patients with breast cancer. Circulation <i>139</i>, 2451-2465. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref5">View at publisher</a></p>
<p id="bib6">Lan, F., Lee, A.S., Liang, P., Sanchez-Freire, V., Nguyen, P.K., Wang, L., Han, L., Yen, M., Wang, Y., Sun, N., et al. (2013). Abnormal calcium handling properties underlie familial hypertrophic cardiomyopathy pathology in patient-specific induced pluripotent stem cells. Cell Stem Cell <i>12</i>, 101-113. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref6">View at publisher</a></p>
<p id="bib7">Lee, J., Termglinchan, V., Diecke, S., Itzhaki, I., Lam, C.K., Garg, P., Lau, E., Greenhaw, M., Seeger, T., Wu, H., et al. (2019). Activation of PDGF pathway links LMNA mutation to dilated cardiomyopathy. Nature <i>572</i>, 335-340. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref7">View at publisher</a></p>
<p id="bib8">Lin, Y. and Zou, J. (2020). Differentiation of cardiomyocytes from human pluripotent stem cells in fully chemically defined conditions. STAR Protoc. <i>1</i>, 100015. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref8">View at publisher</a></p>
<p id="bib9">Lyra-Leite, D.M., Fonoudi, H., Gharib, M., and Burridge, P.W. (2021). An updated protocol for the cost-effective and weekend-free culture of human induced pluripotent stem cells. STAR Protoc. <i>2</i>, 100213. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref9">View at publisher</a></p>
<p id="bib10">Ma, N., Zhang, J.Z., Itzhaki, I., Zhang, S.L., Chen, H., Haddad, F., Kitani, T., Wilson, K.D., Tian, L., Shrestha, R., et al. (2018). Determining the pathogenicity of a genomic variant of uncertain significance using CRISPR/Cas9 and human-induced pluripotent stem cells. Circulation <i>138</i>, 2666-2681. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref10">View at publisher</a></p>
<p id="bib11">Maas, R.G.C., Lee, S., Harakalova, M., Snijders Blok, C.J.B., Goodyer, W.R., Hjortnaes, J., Doevendans, P.A.F.M., Van Laake, L.W., van der Velden, J., Asselbergs, F.W., et al. (2021). Massive expansion and cryopreservation of functional human induced pluripotent stem cell-derived cardiomyocytes. STAR Protoc. <i>2</i>, 100334. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref11">View at publisher</a></p>
<p id="bib12">Seeger, T., Shrestha, R., Lam, C.K., Chen, C., McKeithan, W.L., Lau, E., Wnorowski, A., McMullen, G., Greenhaw, M., Lee, J., et al. (2019). A premature termination codon mutation in MYBPC3 causes hypertrophic cardiomyopathy via chronic activation of nonsense-mediated decay. Circulation <i>139</i>, 799-811. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref12">View at publisher</a></p>
<p id="bib13">Sharma, A., McKeithan, W.L., Serrano, R., Kitani, T., Burridge, P.W., Del Álamo, J.C., Mercola, M., and Wu, J.C. (2018). Use of human induced pluripotent stem cell-derived cardiomyocytes to assess drug cardiotoxicity. Nat. Protoc. <i>13</i>, 3018-3041. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref13">View at publisher</a></p>
<p id="bib14">Wu, H., Yang, H., Rhee, J.W., Zhang, J.Z., Lam, C.K., Sallam, K., Chang, A.C.Y., Ma, N., Lee, J., Zhang, H., et al. (2019). Modelling diastolic dysfunction in induced pluripotent stem cell-derived cardiomyocytes from hypertrophic cardiomyopathy patients. Eur. Heart J. <i>40</i>, 3685-3695. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref14">View at publisher</a></p>
<p id="bib15">Zhang, J.Z., Belbachir, N., Zhang, T., Liu, Y., Shrestha, R., and Wu, J.C. (2021). Effects of cryopreservation on human induced pluripotent stem cell-derived cardiomyocytes for assessing drug safety response profiles. Stem Cell Reports <i>16</i>, 168-181. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref15">View at publisher</a></p>
<p id="bib16">Zhang, J.Z., Termglinchan, V., Shao, N.Y., Itzhaki, I., Liu, C., Ma, N., Tian, L., Wang, V.Y., Chang, A.C.Y., Guo, H., et al. (2019). A human iPSC double-reporter system enables purification of cardiac lineage subpopulations with distinct function and drug response profiles. Cell Stem Cell <i>24</i>, 802-811.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00565-7/sref16">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Joseph C. Wu, Email: <a href="mailto:joewu@stanford.edu">joewu@stanford.edu</a>.</p>
<h4>Materials availability</h4>
<p>hiPSC lines used in this study are available upon request from the lead contact, Joseph C. Wu, Email: <a href="mailto:joewu@stanford.edu">joewu@stanford.edu</a>.</p>
<h4>Data and code availability</h4>
<p>The published article includes all datasets generated or analyzed during this study. No new data have been generated for this protocol.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> R01 HL123968, R01 HL141371, R01 HL145676, R01 HL150693, and Leducq 18CVD05 (J.C.W.).</p>
<h3>Author contributions</h3>
<p>J.Z.Z. wrote the manuscript, performed the experiments, and analyzed the data. S.R.Z., C.T., P.P., and M.Z. co-wrote the manuscript. J.Z.Z. and S.R.Z developed the full protocol. J.C.W. supervised the experimental studies, reviewed the manuscript, and provided funding support.</p>
<h3>Declaration of interests</h3>
<p>J.C.W. is a cofounder of Khloris Biosciences but has no competing interests, as the work presented here is completely independent. The other authors declare no competing interests.</p>
</section>
</article>