<article>
<section>
<h1 id="header">Transformation and CRISPR-Cas9-mediated homologous recombination in the fungus <i>Rhizopus microsporus</i>
</h1>
<p><time datetime="2022-03-18">Published: March 18, 2022</time></p>
<p>Carlos Lax,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> María Isabel Navarro-Mendoza,<sup><a href="#aff2">2</a></sup> Carlos Pérez-Arques,<sup><a href="#aff2">2</a></sup> Eusebio Navarro,<sup><a href="#aff1">1</a></sup> Francisco Esteban Nicolás,<sup><a href="#aff1">1</a>,<a href="#cor3">***</a></sup> and Victoriano Garre<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Departamento de Genética y Microbiología, Facultad de Biología, Universidad de Murcia, 30100 Murcia, Spain</p>
<p id="aff2"><sup>2</sup>Department of Molecular Genetics and Microbiology, Duke University Medical Center, Durham, NC 27710, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:carlos.lax@um.es">carlos.lax@um.es</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:vgarre@um.es">vgarre@um.es</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:fnicolas@um.es">fnicolas@um.es</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101237">10.1016/j.xpro.2022.101237</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we describe a reliable approach for targeted DNA integrations in the genome of <i>R. microsporus</i>, one of the main causal agents of mucormycosis. We provide a strategy for stable, targeted integration of DNA templates by homologous recombination (HR) based on the CRISPR-Cas9 technology. This strategy opens a wide range of possibilities for the genetic modification of <i>R. microsporus</i> and will be useful for the study of mucormycosis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.2139/ssrn.3860385">Lax et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Genetic manipulation of <i>Rhizopus microsporus</i> using CRISPR-Cas9</li>
<li>Optimized conditions for protoplast generation and electroporation parameters</li>
<li>Generation of stable and reproducible mutant strains by homologous recombination</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1495-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol presents a reliable approach for targeted DNA integrations in the genome of <i>R. microsporus,</i> one of the main causal agents of mucormycosis. To develop this protocol, we have adapted the procedures for protoplast generation and transformation developed for <i>Mucor lusitanicus</i> (<a href="#bib3">Gutiérrez et al., 2011</a>). This protocol can be used to introduce self-replicative plasmids in the cells or to perform targeted gene integration. In this work, we describe the steps for the latter because those for plasmid transformation are encompassed in it. For targeted integration, we have adjusted the CRISPR-Cas9 strategy described by (<a href="#bib1">Abdallah et al., 2017</a>) combining the use of <i>in vitro</i> assembled CRISPR-Cas9 ribonucleoprotein (RNP) complexes and DNA templates flanked with very short homology regions (microhomology repair templates) that allow for homologous recombination after the RNP targeted cut.</p>
<p>While the main core of the protocol is conducted in one day, in which protoplasts are generated and electroporated, the whole procedure is extended before and after this day. The fungus is grown for 4–5 days to obtain the spores used in the transformation. After electroporation, protoplasts must be cultured in selective media for 2 or 3 days, and then 1 or 2 extra weeks will be required to check the template integration. Isolation of homokaryons for the integration will require about a month.</p>
<p>This strategy opens a wide range of possibilities for the genetic modification of <i>R. microsporus,</i> such as gene knockout, genetic complementation, gene tagging, etc. This protocol describes the steps for targeted gene knockout using the uridine selectable marker <i>pyrF</i> (<a href="#fig1">Figure 1</a>). A previously generated uridine auxotrophic strain, UM1, derived from <i>R. microsporus</i> ATCC 11559 was used as recipient strain (<a href="#bib6">Lax et al., 2021</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1495-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic representation of the strategy described in this protocol</div>
<p>(A) The double strand break (DSB) in the DNA produced by the guided Cas9 and selectable marker used as DNA repair template are shown. The DNA template has 38-bp tails to allow homology directed repair.</p>
<p>(B) Result of the targeted integration showing primers (red arrows) used to check the template integration and homokaryosis validation.</p>
<p>(C) Detailed view of the DSB and microhomology regions attached to Templ_F_primer and Templ_R_primer for template amplification.</p>
</figcaption>
</figure>
<h3 id="sec1.1">Media and buffer preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 h</div>
<ol>
<li>Preparation of solid YPG media pH 4.5 and MMCS media pH 3.2.
<ol type="a">
<li>After autoclaving, MMC media must be supplemented with niacin and thiamine.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> YPG plates need to be supplemented with uridine because a uridine auxotrophic strain will be transformed.</div>
<ol start="2">
<li>Preparation of liquid YPG media pH 4.5, YPGS media pH 4.5, YNBS media pH 4.5, 0.5 M sorbitol solution, and PS buffer.
<ol type="a">
<li>After autoclaving, YNBS media must be supplemented with niacin and thiamine.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Concentration of media supplements are: Niacin (1 mg/L) Thiamine (1 mg/L), Uridine (200 mg/L).</div>
<h3 id="sec1.2">Culture and collection of <i>R. microsporus</i> fresh spores</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 days</div>
<ol start="3">
<li>UM1 spores are plated in YPG media pH 4.5 and grow for 4–5 days at 30°C.
<ol type="a">
<li>To release spores from the mycelia, pour 10 mL/plate of sterile distilled water and spread it by a Drigalski spatula. Transfer spore solution with a pipette to a 50 mL sterile tube and centrifuge for 5 min at 440 × <i>g</i>. Wash spores twice with sterile distilled water by centrifugation using the above conditions and then resuspend the pellet in 5–10 mL of sterile water and store at 4°C.</li>
<li>Determine the spore concentration by using a hemocytometer and a light microscope.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For transformation, spores must be recently collected to ensure good viability and increase the number of transformants obtained (<a href="#troubleshooting">troubleshooting 4</a>). Spores should be used no longer than 4 days after collection.</div>
<h3 id="sec1.3">crRNA and primer design, DNA template amplification, and purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="4">
<li>Design crRNAs using the gRNA design tool EuPaGDT (<a href="http://grna.ctegd.uga.edu/">http://grna.ctegd.uga.edu/</a>) with default parameters for SpCas9. crRNAs with higher score based on CRISPRater (<a href="#bib5">Labuhn et al., 2018</a>)and (<a href="#bib2">Doench et al., 2014</a>)ordered to IDT (Alt-R<sup>TM</sup> CRISPR-Cas9 crRNA - <a href="https://eu.idtdna.com/site/order/oligoentry/index/crispr">https://eu.idtdna.com/site/order/oligoentry/index/crispr</a>, IDT, Coralville, IA, USA).</li>
<li>All primers are designed using the Primer3Plus tool (<a href="https://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi">https://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi</a>) (<a href="#bib9">Untergasser et al., 2007</a>) and checked with PCR Primer Stats tool (<a href="https://www.bioinformatics.org/sms2/pcr_primer_stats.html">https://www.bioinformatics.org/sms2/pcr_primer_stats.html</a>). Primers can be ordered from any manufacturer with standard desalting purification (<a href="#fig1">Figure 1</a>).</li>
<li>The 3.5 kb <i>pyrF</i> gene of <i>R. microsporus</i>, used as a selectable marker, should be amplified using the Templ_F_primer and the Templ_R_primer to obtain the template for homology-directed repair.</li>
</ol>
<table id="undtbl1">
<thead>
<tr>
<th>Primer name</th>
<th>Primer sequence (5′ -&gt; 3′)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Templ_F_primer</td>
<td>Tail(GATCTTGACCTAGAGCATGATACTCGAGTACCTCTATT)-TCCTCCATAAGAATTTGACAG</td>
</tr>
<tr>
<td>Templ_R_primer</td>
<td>Tail(TCATGCAGTTGACTCCAGGGAATGCAAGTGATCCGACC)-TGATAAAACGAAGATGTGGCTGTC</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The sequence of the tails in this table is for <i>crgA</i> gene disruption. A specific homology tail (38-nt) should be added to the primers that amplify the <i>pyrF</i> gene depending on the cleavage site (<a href="#fig1">Figure 1</a>C).</div>
<ol start="7">
<li>Additionally, a couple of primers is needed to check the integration in the obtained transformants. To that end, a reverse primer that hybridizes in the <i>pyrF</i> gene (<i>pyr</i>_RC_primer) and forward primer that hybridizes upstream the cut point in the targeted locus (TL_F_primer) need to be ordered as well. To check for homokaryons, the targeted locus is amplified using TL_F_primer and a reverse primer that hybridizes downstream the cut point in the targeted locus (TL_R_primer).</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th>Primer name</th>
<th>Primer sequence (5′-&gt; 3′)</th>
</tr>
</thead>
<tbody>
<tr>
<td>pyrF_RC</td>
<td>TAGTCATGCGTCCAGTTTCTGT</td>
</tr>
<tr>
<td>TL_F_primer</td>
<td>ATGGCAATAACCAGACCATACC (crgA)</td>
</tr>
<tr>
<td>TL_R_primer</td>
<td>AACATCCTTCTAGAACCGCGTA (<i>crgA</i>)</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The sequence of TL_F_primer and TL_R_primer is for <i>crgA</i> locus amplification. A specific primer pair should be designed depending on the gene being disrupted.</div>
<ol start="8">
<li>Amplify the DNA template from <i>R. microsporus</i> genomic DNA with the primers containing the 38-nt microhomology tails using the Herculase II Fusion DNA polymerase (Agilent, Santa Clara, California, USA) following the below cycling program:</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>61°C<sup><a href="#dtbl3fna">a</a></sup>
</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min 50 s<sup><a href="#dtbl3fna">a</a></sup>
</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<p id="dtbl3fna" class="table-footnote"><sup>a</sup>This annealing temperature and the extension time are set for the <i>pyrF</i> gene amplification primers. Both parameters should be changed according to manufacturer indications for a different DNA template.</p>
<ol start="9">
<li>Purify PCR products by using the Gene Jet PCR Purification Kit (Thermo Scientific, Waltham, Massachusetts, USA). Purify two 50 µL PCR reactions together in a single column and elute the DNA with 40 µL of sterile Milli-Q water. 2 µL of the purified product is then checked by electrophoresis in a 0.7% agarose gel (40 min–100 V) to discard the presence of nonspecific amplified fragments.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Up to 20 µL of DNA (150–200 µg/ml) template is used per electroporation cuvette. The number of PCRs reactions to be set should be considered in consequence.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Uridine</td>
<td>Merck KGaA</td>
<td>Cat#U3003</td>
</tr>
<tr>
<td>Niacin (nicotinic acid)</td>
<td>Merck KGaA</td>
<td>Cat#N0761</td>
</tr>
<tr>
<td>Thiamine</td>
<td>Merck KGaA</td>
<td>Cat#T1270</td>
</tr>
<tr>
<td>D-Sorbitol</td>
<td>Merck KGaA</td>
<td>Cat#S1876</td>
</tr>
<tr>
<td>Bacto<sup>TM</sup> Yeast Extract</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#212750</td>
</tr>
<tr>
<td>Bacto<sup>TM</sup> Proteose peptone</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#211684</td>
</tr>
<tr>
<td>Bacto<sup>TM</sup> Casamino acids</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#223030</td>
</tr>
<tr>
<td>Bacto<sup>TM</sup> Yeast Nitrogen w/o amino acids</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#DF0919-15-3</td>
</tr>
<tr>
<td>Ammonium Sulfate</td>
<td>Merck KGaA</td>
<td>Cat#A4418</td>
</tr>
<tr>
<td>Glutamic Acid</td>
<td>Merck KGaA</td>
<td>Cat#G1251</td>
</tr>
<tr>
<td>Glucose</td>
<td>Merck KGaA</td>
<td>Cat#G7021</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Merck KGaA</td>
<td>Cat#S3264</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>
</td>
<td>Merck KGaA</td>
<td>Cat#S3139</td>
</tr>
<tr>
<td>Agar</td>
<td>Merck KGaA</td>
<td>Cat#05040</td>
</tr>
<tr>
<td>PBS 10×</td>
<td>Merck KGaA</td>
<td>Cat#6506-OP</td>
</tr>
<tr>
<td>Lysing Enzymes</td>
<td>Merck KGaA</td>
<td>Cat#L1412</td>
</tr>
<tr>
<td>Chitosanase</td>
<td>Merck KGaA</td>
<td>Cat#C9830</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Alt-R™ CRISPR-Cas9 crRNA</td>
<td>Integrated DNA Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>Alt-R™ CRISPR-Cas9 tracrRNA</td>
<td>Integrated DNA Technologies</td>
<td>Cat#1072533</td>
</tr>
<tr>
<td>Alt-R™ S.p. Cas9 Nuclease V3</td>
<td>Integrated DNA Technologies</td>
<td>Cat#1081058</td>
</tr>
<tr>
<td>GeneJET PCR Purification Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#1081058</td>
</tr>
<tr>
<td>DNeasy Plant Kit</td>
<td>QIAGEN</td>
<td>Cat#69104</td>
</tr>
<tr>
<td>Herculase II Fusion DNA Polymerase</td>
<td>Agilent</td>
<td>Cat#600679</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td><i>R. microsporus pyrF</i></td>
<td>Joint Genome Institute Mycocosm</td>
<td>
<i>R. microsporus</i> Protein ID 231940</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>Rhizopus microsporus</i> UM1: <i>pyrF</i><sup>-</sup>
</td>
<td>(<a href="#bib6">Lax et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Templ_F_primer (5′-GATCTTGACCTAGAGCATGATACTCGAGTACCTCTATTCCTCCATAAGAATTTGACAG-3′)</td>
<td>Merck KGaA - (<a href="#bib6">Lax et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Templ_R_primer (5′-TCATGCAGTTGACTCCAGGGAATGCAAGTGATCCGACCTGATAAAACGAAGATGTGGCTGTC-3′)</td>
<td>Merck KGaA - (<a href="#bib6">Lax et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pyrF_RC (5′- TAGTCATGCGTCCAGTTTCTGT-3′)</td>
<td>Merck KGaA - (<a href="#bib6">Lax et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>TL_F_primer (5′- ATGGCAATAACCAGACCATACC-3′)</td>
<td>Merck KGaA - (<a href="#bib6">Lax et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>TL_R_primer (5′- AACATCCTTCTAGAACCGCGTA-3′)</td>
<td>Merck KGaA - (<a href="#bib6">Lax et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>EuPaGDT</td>
<td>(<a href="#bib8">Peng and Tarleton, 2015</a>)</td>
<td><a href="http://grna.ctegd.uga.edu/">http://grna.ctegd.uga.edu/</a></td>
</tr>
<tr>
<td>Primer3Plus</td>
<td>(<a href="#bib9">Untergasser et al., 2007</a>)</td>
<td><a href="https://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi">https://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>1.5 mL tubes</td>
<td>Merck KGaA</td>
<td>Cat#Z336769</td>
</tr>
<tr>
<td>2 mL tubes</td>
<td>Merck KGaA</td>
<td>Cat#Z628034</td>
</tr>
<tr>
<td>50 mL tubes</td>
<td>Merck KGaA</td>
<td>Cat#CLS430290</td>
</tr>
<tr>
<td>Gene Pulser Xcell Electropolation System</td>
<td>Bio-Rad</td>
<td>Cat#652660</td>
</tr>
<tr>
<td>Electroporation cuvettes (0.2 cm gap)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#FB102</td>
</tr>
<tr>
<td>PCR machine (Agilent Surecycler 8800)</td>
<td>Agilent</td>
<td>Cat#G8800A</td>
</tr>
<tr>
<td>0.2 µm filters</td>
<td>Merck KGaA</td>
<td>Cat#CLS431212</td>
</tr>
<tr>
<td>L-shape spreader</td>
<td>Merck KGaA</td>
<td>Cat#HS8171A</td>
</tr>
<tr>
<td>Triple vent Petri dishes</td>
<td>VWR</td>
<td>Cat#391-2028</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl4">
<caption>0.5 M Sorbitol</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sorbitol</td>
<td>91.1 g</td>
<td>0.5 M</td>
</tr>
<tr>
<td>Distilled water</td>
<td>Adjust final volume to 1 L</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave (121°C for 20 min) and store the solution at RT. It is stable for a month.</p>
<table id="undtbl5">
<caption>YPG</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast Extract</td>
<td>3 g</td>
<td>3 g/L</td>
</tr>
<tr>
<td>Proteose peptone</td>
<td>10 g</td>
<td>10 g/L</td>
</tr>
<tr>
<td>Glucose</td>
<td>20 g</td>
<td>0.11 M</td>
</tr>
<tr>
<td>Distilled water</td>
<td>Adjust final volume to 1 L</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave (121°C for 20 min) and store the solution at RT. It is stable for a month.</p>
<table id="undtbl6">
<caption>YPGS</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast Extract</td>
<td>3 g</td>
<td>3 g/L</td>
</tr>
<tr>
<td>Proteose peptone</td>
<td>10 g</td>
<td>10 g/L</td>
</tr>
<tr>
<td>Glucose</td>
<td>20 g</td>
<td>0.11 M</td>
</tr>
<tr>
<td>Sorbitol</td>
<td>91.1 g</td>
<td>0.5 M</td>
</tr>
<tr>
<td>Distilled water</td>
<td>Adjust final volume to 1 L</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave (121°C for 20 min) and store the solution at RT. It is stable for a month.</p>
<table id="undtbl7">
<caption>MMCS</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Casamino acids</td>
<td>10 g</td>
<td>10 g/L</td>
</tr>
<tr>
<td>Yeast Nitrogen w/o amino acids</td>
<td>0.5 g</td>
<td>0.5 g/L</td>
</tr>
<tr>
<td>Glucose</td>
<td>20 g</td>
<td>0.11 M</td>
</tr>
<tr>
<td>Sorbitol</td>
<td>91.1 g (0.5 M)</td>
<td>0.5 M</td>
</tr>
<tr>
<td>Agar</td>
<td>15 g</td>
<td>15 g/L</td>
</tr>
<tr>
<td>Distilled water</td>
<td>Adjust final volume to 1 L</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave (121°C for 20 min) and store the solution at RT. It is stable for a month. Agar should be autoclaved separately.</p>
<div class="note">
<span class="note-title">Note:</span> MMCS is prepared at pH 3.2. To avoid agar hydrolysis caused by low pH, it is autoclaved separated from nutrient solution.</div>
<table id="undtbl8">
<caption>YNBS</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ammonium Sulfate</td>
<td>1.5 g</td>
<td>11.3 mM</td>
</tr>
<tr>
<td>Glutamic Acid</td>
<td>1.5 g</td>
<td>10.2 mM</td>
</tr>
<tr>
<td>Yeast Nitrogen w/o amino acids</td>
<td>0.5 g</td>
<td>0.5 g/L</td>
</tr>
<tr>
<td>Glucose</td>
<td>10 g</td>
<td>0.055 M</td>
</tr>
<tr>
<td>Sorbitol</td>
<td>91.1 g</td>
<td>0.5 M</td>
</tr>
<tr>
<td>Distilled water</td>
<td>Adjust final volume to 1 L</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave (121°C for 20 min) and store the solution at RT. It is stable for a month.</p>
<table id="undtbl9">
<caption>SPB Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub> solution (0.1 M) – Sol. 1</td>
<td>1.42 g</td>
<td>10 mM</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub> solution (0.1 M) – Sol. 2</td>
<td>1.38 g</td>
<td>11.5 mM</td>
</tr>
<tr>
<td>Double distilled H<sub>2</sub>O (ddH<sub>2</sub>O)</td>
<td>Adjust volume of individual solutions to 100 mL</td>
<td> </td>
</tr>
<tr>
<td><b>Total (Sol 1 + Sol 2)</b></td>
<td>
<b>≈153 mL</b><sup><a href="#dtbl10fna">a</a></sup>
</td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Pass the final solution through a 0.2 µm sterile filter. It is stable for a month.</p>
<p id="dtbl10fna" class="table-footnote"><sup>a</sup>Add Sol. 1 (Na<sub>2</sub>HPO<sub>4</sub>) ≈ 53 mL to Sol. 2 (NaH<sub>2</sub>PO<sub>4</sub>) until pH 6.5 is reached.</p>
<table id="undtbl10">
<caption>PS Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>SPB Buffer</td>
<td>100 mL</td>
<td> </td>
</tr>
<tr>
<td>Sorbitol</td>
<td>91.1 g</td>
<td>0.5 M</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>Adjust final volume to 1 L</td>
<td> </td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1 L</b></td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave (121°C for 15 min) and store the solution at RT. It is stable for a month.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Protoplasts generation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Previous day O/N incubation + 5–7 h</div>
<p>This step describes the procedures required to remove the fungal cell wall to achieve successful genetic transformation of <i>R. microsporus</i>. The absence of a cell wall will allow the DNA and ribonucleoprotein complex to penetrate the plasma membrane.</p>
<ol>
<li>Pre-germination of incubation of <i>R. microsporus</i> spores.
<ol type="a">
<li>Take an aliquot of the spore stock containing the total spores needed for the transformation experiment depending on the number of treatments. Each treatment requires 6.25 × 10<sup>7</sup> spores. A typical transformation experiment comprises four treatments (electrophoretic cuvettes), and therefore, this protocol uses amounts for four treatments, but it could be easily scaled up. For four treatments, a total of 2.5 × 10<sup>8</sup> spores are required.</li>
<li>Centrifuge 5 min at 440 × <i>g</i> at RT (20°C–25°C). Resuspend the spore pellet in YPG pH 4.5 to a final concentration of 1 × 10<sup>7</sup> spores/mL (25 mL for four treatments).
<div class="note">
<span class="note-title">Note:</span> If an uracil auxotrophic strain is going to be transformed, YPG media must be supplemented with 200 mg/L of uridine.</div>
</li>
<li>Transfer the spore solution to an Erlenmeyer flask covered with foil to prevent light with a 10:1 flask/culture volume ratio (e.g., 25 mL for four treatments in a 250 mL flask) and incubate 12–16 h (typically overnight) at 4°C without shaking.</li>
</ol>
</li>
<li>Germination of <i>R. microsporus</i> spores.
<ol type="a">
<li>Incubate the spores at 33°C with shaking (250 rpm) in the same foil covered flask.</li>
<li>Check the germination progression placing a 10 µL drop from the culture in a glass slide with cover slip and visualizing with a light microscope every hour until the optimal germination stage is reached (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1495-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Germination stages of <i>R. microsporus</i> spores</div>
<p>Spores swell and emit hyphae as germination progresses. An example of early (left), optimal for digestion (middle) and late (right) germination stages are shown. Scale bar = 20 μm.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Typically, the optimal germination stage is reached after 4–5 h. It may vary in different laboratory settings; therefore, frequent observation of the germination progress is necessary.</div>
<ol start="3">
<li>Cell wall digestion.
<ol type="a">
<li>Transfer the culture with the germinated spores to a sterile 50 mL conical tube and centrifuge for 5 min at 110 × <i>g</i> at RT (20°C–25°C).</li>
<li>Wash the pellet twice with 20 mL of PS Buffer. Centrifuge for 5 min at 110 × <i>g</i> at RT (20°C–25°C). Resuspend in 5 mL of PS Buffer for a four cuvettes experiment.</li>
<li>Dissolve 3 mg of lysing enzymes and 0.0008 units of chitosanase per 1 × 10<sup>7</sup> spores in 1.5 mL of PS Buffer and add the enzyme mix to the germinated spores. Incubate at 30°C with gently shaking (50–60 rpm) for 90 min. At this point, it is not necessary to prevent protoplast from light.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Performing the cell wall digestion in the optimal germination stage is critical for sufficient digestion and protoplasts viability. Frequent checking of the culture is necessary to avoid too early or too late germination stages (<a href="#fig2">Figure 2</a>). <a href="#troubleshooting">Troubleshooting 1</a> and <a href="#troubleshooting">4</a>.</div>
<h3 id="sec3.2">CRISPR-Cas9 RNP complex assembly</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Assemble the CRISPR-Cas9 RNP complex before the cell wall digestion is completed. Firstly, the crRNA and tracrRNA are coupled to form the gRNA. Then, this gRNA binds to the Cas9 enzyme to form the RNP complex targeting the region of interest.</p>
<ol start="4">
<li>crRNA and tracrRNA assembly.
<ol type="a">
<li>Combine the crRNA and the tracrRNA in an RNAse-Free microcentrifuge tube as follows:
<table id="undtbl11">
<thead>
<tr>
<th>gRNA (for 1 cuvette)</th>
<th>Volume (µL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>100 µM Alt-R<sup>TM</sup> CRISPR-Cas9 <b>crRNA</b>
</td>
<td>0.5</td>
</tr>
<tr>
<td>100 µM Alt-R<sup>TM</sup> CRISPR-Cas9 <b>tracrRNA</b>
</td>
<td>0.5</td>
</tr>
<tr>
<td>Nuclease-Free Duplex Buffer</td>
<td>0.5</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1.5</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Lyophilized Alt-R<sup>TM</sup> CRISPR-Cas9 <b>crRNA</b> and Alt-R<sup>TM</sup> CRISPR-Cas9 <b>tracrRNA</b> are resuspended to 100 µM with Nuclease-Free Duplex buffer according manufacturer specifications. The Nuclease-Free Duplex Buffer is provided by IDT together with crRNA and tracrRNA.</div>
</li>
<li>Incubate the reaction for 5 min at 95°C.</li>
<li>Cooldown to at RT (20°C–25°C) on the benchtop.</li>
</ol>
</li>
<li>RNP complex assembly.
<ol type="a">
<li>Combine the gRNA and the Cas9 following as follows:
<table id="undtbl12">
<thead>
<tr>
<th>RNP complex (for 1 cuvette)</th>
<th>Volume (µL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>gRNA</td>
<td>1.5</td>
</tr>
<tr>
<td>Cas9 (dil 1:10)</td>
<td>0.75</td>
</tr>
<tr>
<td>PBS 1×</td>
<td>11</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>13.25</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The total amount of Cas9 required (Alt-R™ S.p. Cas9 Nuclease V3. See <a href="#key-resources-table">key resources table</a>) is diluted in PBS 1×.</div>
</li>
<li>Incubate the reaction for 5 min at RT (20°C–25°C).
<div class="critical">
<span class="critical-title">Critical:</span> To avoid RNA degradation, sterile RNAse free pipette tips and tubes should be used in this step. Perform this step in a clean room or biosafety hood.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Protoplast electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This step is to get repair DNA templates and CRISPR-Cas9 RNP complexes into the protoplasts (<a href="#fig3">Figure 3</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1495-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Schematic workflow of protoplasts electroporation</div>
<p>(A) Protoplasts are combined with the DNA template and the assembled RNP complex.</p>
<p>(B) This mix is transferred to electroporation cuvettes and a time constant pulse is applied.</p>
<p>(C) The recovery of the protoplasts is initiated immediately after the electroporation by adding 1 mL of ice-cold YPGS.</p>
</figcaption>
</figure>
<ol start="6">
<li>Transformation mix set up.
<ol type="a">
<li>Add 1 volume (≈5 mL in a four cuvettes experiment) of ice-cold 0.5 M sorbitol to the protoplast digestion solution. Centrifuge 2 min at 80 × <i>g</i> at RT (20°C–25°C).</li>
<li>Resuspend the pellet extremely gently in 200 µL of 0.5 M sorbitol per treatment: 800 µL for four treatments + 100 µL for negative control (only protoplast, without DNA or RNP complex).
<div class="note">
<span class="note-title">Note:</span> Keep protoplasts on ice for all the subsequent steps.</div>
</li>
<li>For each treatment, place 200 µL of protoplast solution in a sterile tube, then add 13.25 µL of RNP complex and up to 20 µL of template DNA (2–3 µg of total DNA).
<div class="note">
<span class="note-title">Note:</span> A treatment without DNA has to be done to check for contamination. <a href="#troubleshooting">Troubleshooting 2</a>.</div>
</li>
<li>Transfer each transformation mix to an electroporation cuvette (0.2 cm gap) on ice.</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Electroporation of the protoplast. <a href="#troubleshooting">Troubleshooting 3</a>.
<ol type="a">
<li>Place a cuvette in a Bio-Rad Gene Pulser Xcell Electroporator (<a href="#fig3">Figure 3</a>B) and apply an electrical pulse with the following conditions:<br> Pulse-type: Time constant<br> Time: 5 s<br> Field strength: 950 V (4.5 kV/cm)</li>
<li>Immediately after the pulse, add 1 mL of ice-cold YPGS to the cuvette and keep it on ice until the electrical pulse has been applied to all cuvettes.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Protoplast recovery and plating</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>The protoplasts are now grown under favorable conditions to allow them to recover from the electric pulse and to start regenerating their cell wall. After this, they are plated on selective media.</p>
<ol start="8">
<li>Recovery and preparation of protoplast for plating.
<ol type="a">
<li>Transfer the electroporated protoplasts to sterile 2 mL tubes.</li>
<li>Incubate the tubes for 1 h at 30°C with gently shaking (80–90 rpm).
<div class="note">
<span class="note-title">Note:</span> This step allows protoplasts to recover from the electric pulse and start regenerating their cell wall.</div>
</li>
<li>Centrifuge the tubes for 3 min at 70 × <i>g</i> at RT (20°C–25°C).</li>
<li>Decant the supernatant and resuspend them in a final volume of 400 µL YNBS.</li>
</ol>
</li>
<li>Plating
<ol type="a">
<li>The content of each tube (including the negative control) is plated on 2 plates of selective MMCS media (200 µL per plate). Spread protoplasts very gently and without applying too much strength using sterile L-shape spreaders.</li>
<li>Cover the plates with aluminum foil to protect them from the light and incubate them at 30°C for 5–6 days.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Transformants typically start to appear 48–72 h after plating.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Protoplasts are very sensitive to any mechanical and temperature stress. In all the steps described, gentle manipulation of protoplasts (slow pipetting, careful resuspension of the pellet, plating, etc.) is extremely important. Also, keeping the protoplasts on ice is key to ensuring protoplasts viability.</div>
<h3 id="sec3.5">Validation of targeted template integration and isolation of homokaryons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>This step describes the procedures to confirm the template integration in the target locus and the subsequent steps to obtain homokaryons with all their nuclei with the integration (<a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1495-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Targeted integration analysis of the transformants</div>
<p>(A) A small piece of mycelium of the initial transformants is subcultivated in minimal media (MMC).</p>
<p>(B) Genomic DNA is extracted, and an integration-specific PCR is performed with primers that hybridize in the targeted locus and the <i>pyrF</i> marker.</p>
<p>A PCR product should be obtained with the mutant DNA (M1 and M2) and no amplification in the wild-type (WT) (C) A PCR reaction with the primers flanking the targeted locus is carried out for homokaryosis validation. A bigger PCR product result of the marker integration should be visible in the mutant (M1 and M2) compared to the wild-type (WT).</p>
</figcaption>
</figure>
<ol start="10">
<li>Transformant isolation and DNA purification. <a href="#troubleshooting">Troubleshooting 4</a>.
<ol type="a">
<li>Transfer a small piece of mycelium from independent colonies on the transformation plates to individual MMCS plates by using sterilized forceps.
<div class="note">
<span class="note-title">Note:</span> In this step, MMCS media can be replaced with MMC media (without sorbitol).</div>
</li>
<li>Grow them for 3 days at 30°C. At this point, colonies can be growth either under light or dark conditions.</li>
<li>Extract DNA from each transformant using the QIAGEN DNeasy kit.</li>
</ol>
</li>
<li>Integration-specific PCR amplification.
<ol type="a">
<li>Perform independent PCR reactions for each transformant as well as for DNA for the wild-type <i>R. microsporus</i> as a control using the TL_F_primer and <i>pyrF</i>_RC_primer following the below cycling program:
<table id="undtbl13">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>Primer Tm<sup><a href="#dtbl14fna">a</a></sup>
</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>25 s/kb<sup><a href="#dtbl14fna">a</a></sup>
</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<p id="dtbl14fna" class="table-footnote"><sup>a</sup>The annealing temperature depends on the primer Tm of your specific primers. The extension time is set depending on the expected length of the amplified fragment.</p>
</li>
<li>Visualize the PCR amplification in an 0.7% agarose gel (100 V–45 min). A band of the expected size should be visible if the template DNA is integrated into the targeted <i>locus</i> (<a href="#fig4">Figure 4</a>B) (<a href="#troubleshooting">troubleshooting 5</a>).</li>
</ol>
</li>
</ol>
<ol start="12">
<li>Isolation of homokaryons.
<ol type="a">
<li>Transformants that produce a positive result in the PCR are grown for 5–10 vegetative cycles in selective media MMC to select homokaryons.
<div class="note">
<span class="note-title">Note:</span> Initial transformants are heterokaryons because spores are multinucleated, and probably just one or a few nuclei are transformed.</div>
</li>
<li>Isolate DNA from the transformants as described in step 10 and carry out a PCR amplification using the TL_F_primer and the TL_R_primer that amplify both the locus with and without the integration. Only a band of bigger size than the WT locus band should be visible in homokaryons a result of the marker integration (<a href="#fig4">Figure 4</a>C) while heterokaryons would show both the band corresponding to the WT (smaller) and the mutant (bigger) bands. Follow the below cycling program:
<table id="undtbl14">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>Primer Tm<sup><a href="#dtbl15fna">a</a></sup>
</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>25 s/kb<sup><a href="#dtbl15fna">a</a></sup>
</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"> </td>
</tr>
</tbody>
</table>
<p id="dtbl15fna" class="table-footnote"><sup>a</sup>The annealing temperature depends on the primer Tm of your specific primers. The extension time is set depending on the expected length of the amplified fragment.</p>
<div class="optional">
<span class="optional-title">Optional:</span> Genomic DNA extraction can be performed using any other kit that yields good results from filamentous fungi or phenol/chloroform purification.</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The optimal germination stage is reached around 4 h and 30 min. For a 4-cuvettes (25 mL culture – 2.5 × 10<sup>8</sup> total spores) experiment, 10–50 transformants are typically obtained. Of those, more than 50% are positive for the targeted integration (<a href="#bib6">Lax et al., 2021</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The procedure described here is established and optimized for the transformation of <i>R. microsporus</i>. In order to apply this procedure to any other related species, optimization of the germination time and temperature, amount of lytic enzymes used, and cell wall digestion times should be carried out. Also, as this approach uses selectable markers that complement an auxotrophy, a validated auxotrophic strain must be generated before the genetic transformation.</p>
<p>In approaches to disrupt essential genes, no homokaryons will be obtained, and only heterokaryons containing a mixture of wild-type and mutant nuclei will be obtained.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The cell wall is not fully digested, and the DNA/RNP complex cannot penetrate the membrane (step 3).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>A fast and straightforward way to check if the cell wall digestion has adequately worked is to subject the protoplast to a hypotonic shock and check cellular components and organelle presence in a light microscope because of the membrane break (<a href="#mmc1">Methods</a> <a href="#mmc1">Video S1</a>). In the case of insufficient digestion, check if the total amount of lytic enzymes (lysing enzymes from <i>T. harzianum</i> and Chitosanase from <i>S. griseus</i>) is correct considering the final volume of the culture and the total number of cuvettes. If so, increase lytic enzyme concentration slightly and check the cell wall digestion under the microscope described above. Consider that a high increase in the concentration of lytic enzymes could impact protoplast viability. Another reason for insufficient digestion could be that the initial concentration of spores is higher than 1 × 10<sup>7</sup> spores/mL. Therefore, be sure to precisely count the concentration in your initial spore stock using a hemocytometer.</p>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1495-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S1. Germinated spores (No dig) and digested germinated spores (Dig) were subjected to a hypotonic shock, related to step 3</div>
<p>Cell debris and released organelles visualization confirm effective cell wall digestion. During freezed frame, black circles indicate germinated spores and white circles indicate rests of cell debris and organelles. Scale bar = 20 μm. Images of germinated spores were acquired with a Nikon Eclipse 80i microscope equipped with a Nikon DS-Ri2 camera and processed with NIS Elements Software. Video editing was done with Apple Final Cut Pro and DaVinci Resolve17.</p>
</figcaption>
</figure>
<h3 id="sec6.3">Problem 2</h3>
<p>Colonies grow in the plates where the negative control (protoplasts without template DNA and RNP complex) was plated (step 6-c and step 9-a).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Growth in the negative control plates is a clear sign of contamination. Most likely, the contaminants could be present in the initial spore stock, although it could happen in any step of the transformation procedure. To avoid contamination, it is highly recommended to perform all the steps in a biosafety hood, clean room and/or using a Bunsen burner when the spores/protoplast are air-exposed.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>When applying the electroporation pulse, an electric arc may appear in the chamber holding the electroporation cuvette. Electroporators typically inform about the parameter of each pulse (capacitance, resistance, time, field strength). These parameters are usually similar among all cuvettes, and a significant variation in one specific cuvette/pulse may indicate the presence of bubbles or other problems in the cuvette (step 7).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Before placing the cuvettes in the electroporation chamber in contact with the electrodes, ensure that the cuvettes are dry. Cuvettes must be kept on ice, and it is essential to dry them using paper towels before the electrical pulse. In addition, preventing arcing and applying a consistent pulse is important to avoid the presence of bubbles in the cuvette. They can be reduced by pipetting the transformation mix slowly and carefully into the cuvettes. Before electroporation, check if any bubbles are present (<a href="#fig3">Figure 3</a>) and, if so, tap gently with your fingers in the cuvette walls to move the bubbles to the liquid top.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>No transformants or a deficient number of transformants grow in the transformation plates (step 10).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Several reasons can explain a reduced yield of transformants. The most common are insufficient or inappropriate cell wall digestion (see <a href="#troubleshooting">troubleshooting 1</a> and <a href="#mmc1">Methods</a> <a href="#mmc1">Video S1</a>) or deficient DNA template. It is recommended to analyze the total amount of DNA using a Nanodrop or similar. Too early or too late germination stages of obtaining the protoplasts (<a href="#fig2">Figure 2</a>) will also impact the cell wall digestion and, hence, the final number of transformants obtained. Another explanation could be a low cutting efficiency of the RNP complex (<a href="#bib6">Lax et al., 2021</a>). This can be overcome using a different crRNA that targets the same locus but in a distinct region.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>After doing the PCR amplification and running the PCR reactions in a gel, no visible bands or no bands correspond to the expected size (step 11-b).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>If the template DNA is integrated into the targeted site, a clear band corresponding to the expected size should be visible (<a href="#fig4">Figure 4</a>B). If there are no visible bands, the extracted DNA should be validated doing a PCR using two primers that amplify another locus. If the genomic DNA is not the problem and more than ten transformants have been analyzed, the transformation should be repeated with a different crRNA that targets the same locus but in a distinct region. Typically, when using double stranded DNA templates, transformants without integration in the target locus could appear due to homologous recombination of the selectable marker with its locus present in the genome or by gene conversion of the selectable marker (<a href="#bib11">Meussen et al., 2012</a>; <a href="#bib10">Skory, 2004</a>). In addition, recent reports evidence that the use of CRISPR-Cas machinery could also yield stable transformants that result from large concatenate insertions of the marker or large deletion of the target locus, especially when it is flanked by non-LTR transposon sequences (<a href="#bib4">Huang et al., 2021</a>; <a href="#bib7">Li et al., 2018</a>). Interestingly, as shown in Magnaporthe oryzae, the expected outcome for targeted marker integration differs between loci.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Abdallah, Q.A., Ge, W., and Fortwendel, J.R. (2017). A simple and universal system for gene manipulation in Aspergillus fumigatus: in vitro-assembled Cas9-guide RNA ribonucleoproteins coupled with microhomology repair templates. Msphere <i>2</i>, e00446-17. <a class="external-link" href="https://doi.org/10.1128/msphere.00446-17">https://doi.org/10.1128/msphere.00446-17</a></p>
<p id="bib2">Doench, J.G., Hartenian, E., Graham, D.B., Tothova, Z., Hegde, M., Smith, I., Sullender, M., Ebert, B.L., Xavier, R.J., and Root, D.E. (2014). Rational design of highly active sgRNAs for CRISPR-Cas9–mediated gene inactivation. Nat. Biotechnol. <i>32</i>, 1262-1267. <a class="external-link" href="https://doi.org/10.1038/nbt.3026">https://doi.org/10.1038/nbt.3026</a></p>
<p id="bib3">Gutiérrez, A., López-García, S., and Garre, V. (2011). High reliability transformation of the basal fungus Mucor circinelloides by electroporation. J. Microbiol. Meth. <i>84</i>, 442-446. <a class="external-link" href="https://doi.org/10.1016/j.mimet.2011.01.002">https://doi.org/10.1016/j.mimet.2011.01.002</a></p>
<p id="bib4">Huang, J., Rowe, D., Zhang, W., Suelter, T., Valent, B., and Cook, D.E. (2021). CRISPR-Cas12a induced DNA double-strand breaks are repaired by locus-dependent and error-prone pathways in a fungal pathogen. Preprint at Biorxiv. 2021.09.08.459484. <a class="external-link" href="https://doi.org/10.1101/2021.09.08.459484">https://doi.org/10.1101/2021.09.08.459484</a></p>
<p id="bib5">Labuhn, M., Adams, F.F., Ng, M., Knoess, S., Schambach, A., Charpentier, E.M., Schwarzer, A., Mateo, J.L., Klusmann, J.-H., and Heckl, D. (2018). Refined sgRNA efficacy prediction improves large- and small-scale CRISPR–Cas9 applications. Nucleic Acids Res. <i>46</i>, 1375-1385. <a class="external-link" href="https://doi.org/10.1093/nar/gkx1268">https://doi.org/10.1093/nar/gkx1268</a></p>
<p id="bib6">Lax, C., Navarro-Mendoza, M.I., Pérez-Arques, C., Navarro, E., Nicolás, F.E., and Garre, V. (2021). A landmark in the study of mucormycosis: stable and reproducible homologous recombination in <i>Rhizopus microsporus</i>. SSRN Electron. J. <a class="external-link" href="https://doi.org/10.2139/ssrn.3860385">https://doi.org/10.2139/ssrn.3860385</a></p>
<p id="bib7">Li, J., Zhang, Y., Zhang, Y., Yu, P.-L., Pan, H., and Rollins, J.A. (2018). Introduction of large sequence inserts by CRISPR-Cas9 to create pathogenicity mutants in the multinucleate filamentous pathogen Sclerotinia sclerotiorum. Mbio <i>9</i>, e00567-18. <a class="external-link" href="https://doi.org/10.1128/mbio.00567-18">https://doi.org/10.1128/mbio.00567-18</a></p>
<p id="bib11">Meussen, B.J., De Graaff, L.H., Sanders, J.P.M., and Weusthuis, R.A. (2012). Metabolic engineering of Rhizopus oryzae for the production of platform chemicals. Appl Microbiol Biotechnol <i>94</i>, 875-886. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00117-4/sref11">View at publisher</a></p>
<p id="bib8">Peng, D. and Tarleton, R. (2015). EuPaGDT: a web tool tailored to design CRISPR guide RNAs for eukaryotic pathogens. Microb. Genom. <i>1</i>, e000033. <a class="external-link" href="https://doi.org/10.1099/mgen.0.000033">https://doi.org/10.1099/mgen.0.000033</a></p>
<p id="bib10">Skory, C.D. (2004). Repair of plasmid DNA used for transformation of Rhizopus oryzae by gene conversion. Curr Genet <i>45</i>, 302-310. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00117-4/optCQQA0KTvXZ">View at publisher</a></p>
<p id="bib9">Untergasser, A., Nijveen, H., Rao, X., Bisseling, T., Geurts, R., and Leunissen, J.A.M. (2007). Primer3Plus, an enhanced web interface to Primer3. Nucleic Acids Res. <i>35</i>, W71-W74. <a class="external-link" href="https://doi.org/10.1093/nar/gkm306">https://doi.org/10.1093/nar/gkm306</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and fulfilled by the lead contact, Victoriano Garre (<a href="mailto:vgarre@um.es">vgarre@um.es</a>).</p>
<h4>Materials availability</h4>
<p>All plasmids and fungal strains generated in this study are available from the lead contact without restriction.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique dataset or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the Fundación Séneca-Agencia de Ciencia y Tecnología de la Región de Murcia, Spain (20897/PI/18). The student C.L. was funded by the Spanish MICIU (FPU17/05814).</p>
<h3>Author contributions</h3>
<p>Methodology, C.L., M.I.N.-M., and C.P.-A.; investigation, formal analysis, visualization, writing – original draft, C.L.; writing – review &amp; editing, C.L., F.E.N., and V.G.; funding acquisition, E.N., F.E.N., and V.G.; resources, E.N.; conceptualization, supervision, and project administration, F.E.N. and V.G.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>