<article>
<section>
<h1 id="header">Protocol for design, construction, and selection of genome phage (gPhage) display libraries</h1>
<p><time datetime="2021-11-12">Published: November 12, 2021</time></p>
<p>Luis Antonio Rodriguez Carnero,<sup><a href="#aff1">1</a>,<a href="#fn1">8</a>,<a href="#cor1">*</a></sup> André Azevedo Reis Teixeira,<sup><a href="#aff1">1</a>,<a href="#fn1">8</a>,<a href="#cor2">**</a></sup> Fenny Hui Fen Tang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Andréia Kuramoto,<sup><a href="#aff3">3</a></sup> Maria Júlia Manso Alves,<sup><a href="#aff1">1</a></sup> Walter Colli,<sup><a href="#aff1">1</a></sup> João Carlos Setubal,<sup><a href="#aff1">1</a></sup> Edécio Cunha-Neto,<sup><a href="#aff3">3</a>,<a href="#aff6">6</a>,<a href="#aff7">7</a></sup> Renata Pasqualini,<sup><a href="#aff2">2</a>,<a href="#aff4">4</a></sup> Wadih Arap,<sup><a href="#aff2">2</a>,<a href="#aff5">5</a></sup> and Ricardo José Giordano<sup><a href="#aff1">1</a>,<a href="#aff7">7</a>,<a href="#fn2">9</a>,<a href="#fn3">10</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biochemistry, Institute of Chemistry, University of São Paulo, São Paulo, SP 05508, Brazil</p>
<p id="aff2"><sup>2</sup>Rutgers Cancer Institute of New Jersey, Newark, NJ 07101, USA</p>
<p id="aff3"><sup>3</sup>Heart Institute (InCor), University of São Paulo School of Medicine, São Paulo, SP 05403, Brazil</p>
<p id="aff4"><sup>4</sup>Division of Cancer Biology, Department of Radiation Oncology, Rutgers New Jersey Medical School, Newark, NJ 07103, USA</p>
<p id="aff5"><sup>5</sup>Division of Hematology/Oncology, Department of Medicine, Rutgers New Jersey Medical School, Newark, NJ 07103, USA</p>
<p id="aff6"><sup>6</sup>Division of Clinical Immunology and Allergy, University of São Paulo School of Medicine, São Paulo, SP 01246, Brazil</p>
<p id="aff7"><sup>7</sup>Institute for Investigation in Immunology, University of São Paulo School of Medicine, São Paulo, SP 01246, Brazil</p>
<p id="fn1"><sup>8</sup>These authors contributed equally</p>
<p id="fn2"><sup>9</sup>Technical contact</p>
<p id="fn3"><sup>10</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:lcarnero@iq.usp.br">lcarnero@iq.usp.br</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:arteix@iq.usp.br">arteix@iq.usp.br</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:giordano@iq.usp.br">giordano@iq.usp.br</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100936">10.1016/j.xpro.2021.100936</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes the genomic phage (gPhage) display platform, a large-scale antigen and epitope mapping technique. We constructed a gPhage display peptide library of a eukaryotic organism, <i>Trypanosoma cruzi</i> (causative agent of Chagas disease), to map the antibody response landscape against the parasite. Here, we used an organism with a relatively large but intronless genome, although future applications could include other prevalent or (re)emerging infectious organisms carrying genomes with a limited number of introns.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref13">Teixeira et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A streamlined protocol to build genome shotgun phage display libraries</li>
<li>Scalable to different organisms, eukaryotes or prokaryotes</li>
<li>Optimizes key steps for high throughput identification of antibody antigens/epitopes</li>
<li>gPhage is also a platform to study conformational and difficult-to-validate antigens</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Phagemid vector selection and production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>For the gPhage display peptide library generation and production we recommend the use of the phagemid vector pG8SAET (<a href="#bib14">Zhang et al., 1999</a>) or a functional equivalent. This vector (available upon request) is particularly suitable for this work because it contains a unique site for a blunt-end restriction enzyme (<i>Eco105I</i>), which allows for cloning DNA inserts generated by random fragmentation methods. It will generate phage particles displaying hundreds of peptides (∼400–500 copies) fused to the recombinant major capsid protein VIII (rpVIII) as opposed to the single-digit display of peptides (∼1–5 copies) obtained when using the minor capsid protein III (pIII), thereby potentially increasing biochemical avidity by up to two orders of magnitude.</p>
<ol>
<li>Transform <i>Escherichia coli</i> DH10B strain with pG8SAET plasmid.</li>
<li>Grow the cells in 500 mL LB broth (50 μg/mL carbenicillin) at 37<sup>o</sup>C for 16 h with agitation at 250 revolutions per minute (rpm).</li>
<li>On the next day, perform initial plasmid DNA extraction with a standard kit (see below).</li>
<li>Optional: We recommend an additional purification step with cesium chloride (CsCl) density gradient to improve vector purity, increase ligation efficiency and minimize contamination with <i>E. coli</i> genomic DNA (<a href="#bib11">Sambrook and Russel, 2001</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend <a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-plasmid-kits/">Plasmid Maxi Kit</a> (QIAGEN, 12162) according to manufacturer instructions for the initial plasmid purification prior to the additional ultracentrifugation-based plasmid purification with a CsCl density gradient.</div>
<h3 id="sec1.2">DNA extraction from <i>Trypanosoma cruzi</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 8 days (7 days for epimastigote culture, 1 day <b>for</b> <b>DNA extraction)</b>
</div>
<p>As the source of genomic diversity for the library, one must obtain genomic DNA from the organism of interest. The protocol should be optimized depending on the organism of interest. Here we illustrate the protocol used to construct a <i>T. cruzi</i> genomic library. It is worth mentioning that most <i>T. cruzi</i> genes do not carry introns and almost half of its genome is comprised of coding sequences (<a href="#bib4">El-Sayed et al., 2005</a>). One must take these into account when selecting a model organism because antigens or epitopes that span two or more exons will likely not be correctly displayed in the library design reported here.</p>
<ol start="5">
<li>Grow <i>T. cruzi</i> epimastigotes at 28°C in LIT (liver infusion-tryptose) medium supplemented with heat-inactivated 10% cell culture grade Fetal Bovine Serum (FBS) (<a href="#bib1">Alves et al., 1986</a>; <a href="#bib5">Giordano et al., 1999</a>).</li>
<li>Centrifuge 200 mL of the saturated culture containing approximately 10<sup>9</sup> cells/mL at 1,000 × <i>g</i> (centrifugal force) at 23°C–25°C for 10 min and wash them three times with phosphate-buffered saline (PBS). It will take approximately 1 week (depending on the strain) for the epimastigote culture to reach the lag-phage.</li>
<li>Resuspend the cells in 20 mL lysis buffer containing 1% sodium dodecyl sulfate (SDS) and proteinase K (100 μg/mL).</li>
<li>Incubate at 50°C for 2 h. Centrifuge the suspension and recover the supernatant.</li>
<li>Add 1:1 phenol:chloroform (vol/vol). Homogenize it by using a vortexing apparatus and centrifuge the admixture for 5 min at 7,000 × <i>g</i> at 23°C–25°C.</li>
<li>Repeat the previous step until solids are no longer observed in the interface between the aqueous (lower) and organic (upper) phases.</li>
<li>Carefully recover the aqueous phase. Add 0.1 volumes of 3M sodium acetate pH 5.2 and 2 volumes of ice-cold ethanol. Incubate it at −20°C for 15 min.</li>
<li>Precipitate the DNA by centrifuging at 20,000 × <i>g</i> for 15 min at 4°C. Wash the pellet of DNA with 70% ethanol and gently re-suspend the genomic DNA in 2 mL TE buffer (10 mM Tris pH 8, 1 mM EDTA).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We used ∼2 mg of genomic <i>T. cruzi</i> DNA in order to obtain a gPhage library with ∼10<sup>8</sup> transformants.</div>
<h3 id="sec1.3">IgG purification from selected donors</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="13">
<li>Serum sample selection from donor patients and controls. A cohort of patients should be selected according to the aims of the study and the particulars of each disease (or infectious agent) of interest. As an example, we selected 80 donors including patients with Chagas disease (N=60) plus control healthy volunteer donors (N=20) with the following guidelines:
<ol type="a">
<li>Candidate Chagas disease patients with at least two positive results for the presence of anti-<i>T. cruzi</i> antibodies.</li>
<li>All candidate patients underwent electrocardiography (EKG) and echocardiography (Echo) and those with abnormal EKG (either right bundle branch block or left anterior fascicular block) were classified as having either mild cardiomyopathy, when the Echo ejection fraction of the left ventricle (LVEF) was higher than 40% (LVEF&gt; 40%), or severe cardiomyopathy, when ejection fraction of the left ventricle was lower or equal 40% (LVEF ≤ 40%).</li>
<li>Patients with no EKG alterations were deemed asymptomatic.</li>
<li>Serum samples were pooled into cohorts of 10 donors to form two independent sets (biological duplicates) for each disease condition: control (2 × 10 donors), asymptomatic (2 × 10 donors), mild cardiomyopathy (2 × 10 donors) and severe cardiomyopathy (2 × 10 donors).</li>
</ol>
</li>
<li>IgG was then purified from the pool of serum samples of each cohort by using affinity chromatography with Protein-G Sepharose (<a href="http://www.prep-hplc.com/Uploads/ueditor/file/20190731/5d4103624bc46.pdf">GE Healthcare</a>), according to the manufacturer instructions.</li>
<li>At this point it is important to check the quality of the purified IgG by SDS-PAGE and then quantify protein concentration for each pool using the Bradford assay (or an equivalent alternative method for protein quantification). The resulting purified IgG may be kept frozen (at −20°C or less) in small individual aliquots to be thawed out when the gPhage display selection is performed. IgG concentration should be at least 50–100 μg/mL with purity greater than 90%.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit anti-bacteriophage IgG (1:400)</td>
<td>Sigma</td>
<td>Cat#B7786-5μL</td>
</tr>
<tr>
<td>Goat anti-rabbit IgG IRDye 680 LT (1:1000)</td>
<td>LI-COR Biosciences</td>
<td>Cat#926-68021</td>
</tr>
<tr>
<td>Goat Anti-Human IgG, Fc fragment specific</td>
<td>Jackson ImmunoResearch</td>
<td>Cat#109-001-008</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>Escherichia coli</i> strain DH10B</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EC113</td>
</tr>
<tr>
<td>
<i>Escherichia coli</i> strain TG1</td>
<td>Lucigen</td>
<td>Cat#60502-1</td>
</tr>
<tr>
<td>M13K07 Helper Phage</td>
<td>New England BioLabs</td>
<td>Cat#N0315 S</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Donor serum samples</td>
<td>Heart Institute, University of Sao Paulo. Approved by the Institutional Review Board at the University of São Paulo Medical School, Brazil. Approval number 0265/10. For more details of the subjects, refer to <a href="#bib13">Teixeira et al. (2021)</a>.</td>
<td>N/A</td>
</tr>
<tr>
<td>Bovine blood</td>
<td>Certified commercial slaughterhouse or laboratory vendor.</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Phosphate buffered saline</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#21600010</td>
</tr>
<tr>
<td>Sodium dodecyl sulphate</td>
<td>Bio-Rad Laboratories</td>
<td>Cat#1610301</td>
</tr>
<tr>
<td>Proteinase K Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM2548</td>
</tr>
<tr>
<td>Phenol:Chloroform:Isoamyl Alcohol 25:24:1</td>
<td>Sigma</td>
<td>Cat#P3808</td>
</tr>
<tr>
<td>Ethanol Absolute</td>
<td>Merk</td>
<td>Cat#1009831000</td>
</tr>
<tr>
<td>UltraPure Tris Buffer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15504-020</td>
</tr>
<tr>
<td>EDTA</td>
<td>Sigma</td>
<td>Cat#E5134</td>
</tr>
<tr>
<td>T4 DNA polymerase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EP0062</td>
</tr>
<tr>
<td>Tryptone</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#LP0042</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#LP0021</td>
</tr>
<tr>
<td>Dipotassium hydrogen phosphate</td>
<td>Merck</td>
<td>Cat#5101</td>
</tr>
<tr>
<td>Monopotassium phosphate</td>
<td>Sigma</td>
<td>Cat#V003710</td>
</tr>
<tr>
<td>Disodium phosphate heptahydrate</td>
<td>Synth</td>
<td>Cat#F1031.01.AH</td>
</tr>
<tr>
<td>LB Broth</td>
<td>Sigma</td>
<td>Cat#L3022-1Kg</td>
</tr>
<tr>
<td>LB agar</td>
<td>Sigma</td>
<td>Cat#L2897-1KG</td>
</tr>
<tr>
<td>Triptose broth</td>
<td>Difco</td>
<td>Cat#262200</td>
</tr>
<tr>
<td>Carbenicillin</td>
<td>Sigma</td>
<td>Cat#C1389-5G</td>
</tr>
<tr>
<td>Penicillin</td>
<td>Sigma</td>
<td>Cat#D7794-100MU</td>
</tr>
<tr>
<td>Streptomycin</td>
<td>Sigma</td>
<td>Cat#S9137-100G</td>
</tr>
<tr>
<td>CsCl</td>
<td>Sigma</td>
<td>Cat#V000561</td>
</tr>
<tr>
<td>Ethidium bromide</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15585-011</td>
</tr>
<tr>
<td>Eco105I</td>
<td>New England BioLabs</td>
<td>Cat#R0130L</td>
</tr>
<tr>
<td>FastAP</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EF0651</td>
</tr>
<tr>
<td>Low-melting point agarose</td>
<td>Sigma</td>
<td>Cat#A9539-500G</td>
</tr>
<tr>
<td>T4 DNA ligase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EL0011</td>
</tr>
<tr>
<td>Polyethylene Glycol 8000</td>
<td>Amresco</td>
<td>Cat#0159-1Kg</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Merck</td>
<td>Cat#1064041000</td>
</tr>
<tr>
<td>Potassium chloride</td>
<td>Synth</td>
<td>Cat#C1060.01.AH</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma</td>
<td>Cat#G5516</td>
</tr>
<tr>
<td>Glucose</td>
<td>Sigma</td>
<td>Cat#S9137-100G</td>
</tr>
<tr>
<td>Protein G Sepharose 4 Fast Flow</td>
<td>GE Healthcare</td>
<td>Cat#17-0618-02</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Sigma</td>
<td>Cat#A2153-1kg</td>
</tr>
<tr>
<td>Liver Infusion Broth</td>
<td>Difco</td>
<td>Cat#226920</td>
</tr>
<tr>
<td>Fetal Calf Serum</td>
<td>Vitrocell</td>
<td>Cat#SOROFETAL500</td>
</tr>
<tr>
<td>Carbonate</td>
<td>Merck</td>
<td>Cat#106392</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Sigma</td>
<td>Cat#V001280</td>
</tr>
<tr>
<td>Synthetic epitope peptides</td>
<td>Chinese Peptide Company (China)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>QIAGEN Plasmid Maxi Kit</td>
<td>Qiagen</td>
<td>Cat#12162</td>
</tr>
<tr>
<td>QIAprep Spin Miniprep Kit</td>
<td>Qiagen</td>
<td>Cat#27104</td>
</tr>
<tr>
<td>Taq DNA Polymerase (5 U/μL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EP0401</td>
</tr>
<tr>
<td>KAPA HiFi HotStart ReadyMix Kit</td>
<td>Roche</td>
<td>Cat#7958935001</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>Illumina</td>
<td>Cat#FC-131-1024</td>
</tr>
<tr>
<td>Nextera XT DNA Library Preparation Kit (24 samples)</td>
<td>Qiagen</td>
<td>Cat#28106</td>
</tr>
<tr>
<td>KAPA Library Quantification Kit</td>
<td>Roche</td>
<td>Cat#KR0405</td>
</tr>
<tr>
<td>MiSeq Reagent Kit v2 (500-cycles)</td>
<td>Illumina</td>
<td>Cat#MS-102-2003</td>
</tr>
<tr>
<td>SIGMAFAST OPD</td>
<td>Sigma</td>
<td>Cat#P9187</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>Trypanosoma cruzi</i>epimastigotes Sylvio-X10 c11 (Pará, Brazil)</td>
<td>Dr. Bianca Zingales, Chemistry Institute, University of São Paulo, Brazil (<a href="#bib15">Zingales et al., 2009</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Illumina sequencing oligonucleotides</td>
<td>Exxtend (Brazil)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pG8SAET</td>
<td>Department of Microbiology Swedish University of Agricultural Sciences Uppsala University, Sweden (<a href="#bib14">Zhang et al., 1999</a>)</td>
<td>GenBank: AF130864.1<a href="https://www.ncbi.nlm.nih.gov/nuccore/4754815">https://www.ncbi.nlm.nih.gov/nuccore/4754815</a>
</td>
</tr>
<tr>
<td>gPhage library</td>
<td>This article</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>BLAST</td>
<td><a href="https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/">https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/</a></td>
<td>v.2.12.0</td>
</tr>
<tr>
<td>PEAR</td>
<td><a href="https://cme.h-its.org/exelixis/web/software/pear/doc.html">https://cme.h-its.org/exelixis/web/software/pear/doc.html</a></td>
<td>v 0.9.10</td>
</tr>
<tr>
<td>MAFFT</td>
<td><a href="https://mafft.cbrc.jp/alignment/software/">https://mafft.cbrc.jp/alignment/software/</a></td>
<td>v 7.307</td>
</tr>
<tr>
<td>Jalview</td>
<td><a href="https://www.jalview.org/">https://www.jalview.org/</a></td>
<td>v 2.11.0</td>
</tr>
<tr>
<td>MUSCLE</td>
<td><a href="https://www.ebi.ac.uk/Tools/msa/muscle/">https://www.ebi.ac.uk/Tools/msa/muscle/</a></td>
<td>3.8.31</td>
</tr>
<tr>
<td>Chimera</td>
<td><a href="https://www.cgl.ucsf.edu/chimera/">https://www.cgl.ucsf.edu/chimera/</a></td>
<td>v 1.11</td>
</tr>
<tr>
<td>XSTREAM</td>
<td><a href="https://amnewmanlab.stanford.edu/xstream/">https://amnewmanlab.stanford.edu/xstream/</a></td>
<td>v 1.73</td>
</tr>
<tr>
<td>FuzzyWuzzy</td>
<td><a href="https://github.com/seatgeek/fuzzywuzzy">https://github.com/seatgeek/fuzzywuzzy</a></td>
<td>v 0.18</td>
</tr>
<tr>
<td>HMMER</td>
<td><a href="http://hmmer.org/download.html">http://hmmer.org/download.html</a></td>
<td>3.1b2</td>
</tr>
<tr>
<td>Sequence analysis script</td>
<td>This article</td>
<td><a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc1.zip">Data S1</a></td>
</tr>
<tr>
<td>Clustering script</td>
<td>This article</td>
<td><a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc2.zip">Data S2</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>96 Well EIA/RIA High Binding Plate</td>
<td>Corning Incorporated Costar</td>
<td>Cat#3590</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>LIT medium</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triptose broth</td>
<td>5 g/L</td>
<td>5g</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>4 g/L</td>
<td>4g</td>
</tr>
<tr>
<td>Potassium chloride</td>
<td>0.4 g/L</td>
<td>0.4 g</td>
</tr>
<tr>
<td>Disodium phosphate heptahydrate</td>
<td>15 g/L</td>
<td>15 g</td>
</tr>
<tr>
<td>Penicillin G</td>
<td>0.15 g/L</td>
<td>0.15 g</td>
</tr>
<tr>
<td>Glucose</td>
<td>2 g/L</td>
<td>2 g</td>
</tr>
<tr>
<td>Streptomycin</td>
<td>0.15 g/L</td>
<td>0.15 g</td>
</tr>
<tr>
<td>Liver infusion broth</td>
<td>5 g/L</td>
<td>5 g</td>
</tr>
<tr>
<td>Bovine blood</td>
<td>-</td>
<td>20 mL</td>
</tr>
<tr>
<td>Cell culture grade, heat inactivated FBS</td>
<td>10%</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To prepare 1 L of medium. Dissolve all the reagents except the bovine blood in 800 mL of double-distilled water (ddH<sub>2</sub>O). Incubate at 37<sup>o</sup>C for 1 h. Cool the solution on ice. Centrifugate 20 mL of bovine blood at 12,000 × g for 10 min at 23°C–25°C and add the supernatant to the medium. Adjust pH to 7.3, complete volume to 900 mL with ddH<sub>2</sub>O and sterilize it by filtration (0.22 μm). Aliquot and store medium at 4<sup>o</sup>C. Before use, add 100 mL of heat-inactivated FBS under sterile conditions.</p>
<table id="undtbl2">
<caption>Lysis Buffer</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 8.0 stock solution</td>
<td>100 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate 10% stock solution</td>
<td>1%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Proteinase K Solution (20 mg/mL)</td>
<td>100 μg/mL</td>
<td>2.5 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">For 500 mL, mix 300 mL of ddH<sub>2</sub>O with 50 mL of Tris-HCl 1M pH 8.0 stock solution and 50 mL of 10% SDS stock solution. Adjust pH to 8.0 with NaOH or HCl 1M and complete with ddH<sub>2</sub>O to the final volume. Store at RT. Before use, add proteinase K to a final concentration of 100 μg/mL.</p>
<p>TE Buffer</p>
<p>For TE buffer, use 10 mM Tris-HCl pH 8.0 containing 1 mM EDTA.</p>
<p>For 500 mL, mix 400 mL of ddH<sub>2</sub>O with 5 mL of Tris 1 M pH 8.0 stock solution and 1 mL of 0.5 M EDTA stock solution. Check pH and if necessary, adjust pH to 8.0 with NaOH or HCl 1M and complete to 500 mL with ddH<sub>2</sub>O. Store at 23°C–25°C until use.</p>
<table id="undtbl3">
<caption>TB: phosphate broth</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Yeast extract</td>
<td>24 g/L</td>
<td>24 g</td>
</tr>
<tr>
<td>Tryptone</td>
<td>20 g/L</td>
<td>20 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>4 mL/L</td>
<td>4 mL</td>
</tr>
<tr>
<td>Phosphate buffer (0.17 M KH<sub>2</sub>PO<sub>4</sub>, 0.72 M K<sub>2</sub>HPO<sub>4</sub>)</td>
<td>0.017 M KH<sub>2</sub>PO<sub>4</sub>0.072 M K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Dissolve yeast extract, tryptone and glycerol in 900 mL of ddH<sub>2</sub>O and sterilize the solution (autoclave). After cooling the admixture, add the phosphate buffer under sterile technique. Store the solution at 23°C–25°C for 6 months.</p>
<table id="undtbl4">
<caption>PEG/NaCl</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PEG 8000</td>
<td>166,7 g/L</td>
<td>100 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>3,3 M</td>
<td>116.9 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">To obtain a 600 mL solution, dissolve polyethylene glycol (PEG) 8,000 and NaCl with 450 mL ddH<sub>2</sub>O. Once dissolved, complete to 600 mL final volume with ddH<sub>2</sub>O. Sterilize (autoclave) and mix thoroughly while cooling to avoid phase separation. Store solution at 4°C until use. Discard if any precipitation is observed.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">gPhage library construction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 month</div>
<p>The gPhage display library is constructed in three sequential steps: (i) insert preparation (fragmenting the genomic DNA to the desired size range and making it suitable for cloning), (ii) cloning the purified DNA inserts into the phagemid vector and transformation into bacteria, and (iii) producing phage particles from the library by using a helper phage (<a href="#fig1">Figure 1</a>). We recommend preparing inserts between 100-500 bp (<a href="#fig2">Figure 2</a>A) because this size range allows for the identification of linear and conformational epitopes that may still be fully sequenced [Illumina Miseq 2<b>×</b>250 bp; <a href="#bib13">Teixeira et al. (2021)</a>]. Please see <a href="#fig1">Figure 1</a> in this article for other cloning logistics and details of the gPhage display library construction strategy for <i>T. cruzi</i>: DNA insert size, vector, possible orientation and reading frames of inserts, and genomic coverage distribution.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Constructing a gPhage library</div>
<p>(A) To build the gPhage library, genomic DNA is fragmented and cloned into a phagemid vector such as pG8SAET. <i>Escherichia coli</i> is transformed with the vector and subsequently used to produce the gPhage library displaying the epitopes that can be recognized by the corresponding donor-derived and control-derived immunoglobulins.</p>
<p>(B) Detail of the fusion transcript encoded by modified pG8SAET. The construct is formed by a signal peptide, the epitope-encoding DNA and the rpVIII gene. When transformed into bacteria, pG8SAET will produce a fusion rpVIII protein, as described.</p>
<p>(C) After fragmentation and cloning, only 1 in 18 inserts will be in the correct frame (i.e., frame 2) to yield a phage particle displaying a <i>T. cruzi</i>-derived peptide. In the correct frame, the mature constructed protein will be formed by the epitope fused in-frame to the rpVIII. For using the phagemid system, co-infection with a helper phage allows the production of native pVIII protein and the other bacteriophage proteins while the pG8SAET phagemid encodes the rpVIII displaying the epitope. These two pVIII proteins (i.e., native and recombinant) will be packed into a hybrid phage construct.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Building and quality control of a <i>Trypanosoma cruzi</i> genomic DNA library</div>
<p>(A) <i>T. cruzi</i> genomic DNA can be fragmented with the COVARIS S2 ultrasonicator. The image shows an agarose gel electrophoresis comparing <i>T. cruzi</i> DNA before and after fragmentation. The obtained fragments were mainly distributed between 100-500 bp.</p>
<p>(B) The efficiency of ligation depends on the vector:insert ratio. In the example with <i>T. cruzi</i> fragmented DNA and pG8SAET, the 1:30 ratio yielded the best results. Error bars represent the standard error of the mean (SEM, N=3)</p>
<p>(C) After a PCR amplification of individual clones with primers flanking the insert, each clone will show a particular product size. When using a molecular ladder to calculate the molecular weight of the products, this value serves to estimate the coverage of the library. As a technical cautionary note (shown), a minor subset of clones will have an empty vector (non-insert), with the same size as pG8SAET.</p>
</figcaption>
</figure>
<ol>
<li>Blunt-ending of genomic DNA fragments<br> Purified genomic DNA may be subjected to fragmentation in a COVARIS S2 equipment (Covaris Inc), which yields fragments in the desired size range in a reproducible manner. If access to such equipment is either unavailable or limited by available resources, one may attempt alternative techniques such as enzyme-based treatment such as DNAse or dsDNA Fragmentase (e.g., NEBNext ds DNA Fragmentase), probe sonication, nebulization, or an optimized combination of these methods.
<ol type="a">
<li>Quantify extracted genomic DNA by using the Nanodrop Spectrophotometry (Thermo Fisher Scientific) or other spectrophotometric method. Nanodrop is quite suitable because it requires very small amounts of the biological samples.</li>
<li>Fragment DNA in aliquots of 30 μg DNA in 130 μL of TE at the following conditions: Duty cycle – 10%, Intensity – 4, Cycles/burst: 200, time: 80 s.</li>
<li>Analyze the fragmentation by electrophoresis (<a href="#fig2">Figure 2</a>A, TAE 2% low-melting- point agarose gel) and visualize the DNA by ethidium bromide staining. The gel should be poured out and run within a cold room or refrigerator to avoid melting.</li>
<li>Observe and document the fragmented DNA under 320–365 nm UV light (UV A). Excise the DNA between the 100–500 bp range with a sharp surgical scalpel.</li>
<li>Weigh the gel slice and add 2.5 volumes of TE buffer. Incubate at 65°C for 15 min or until the gel is completely dissolved. Vortexing every few minutes will help accelerate the process.</li>
<li>Add an equal volume of 1:1 phenol:chloroform (vol/vol), vortex and then centrifuge it for 5 min at 8,000 × <i>g</i> before carefully discarding the upper organic phase.</li>
<li>Repeat the previous step until no solids can be observed in the interface between the aqueous and organic phases.</li>
<li>Add an equal volume of chloroform, vortex and centrifuge it for 5 min at 8,000 × <i>g</i>. Discard upper organic phase.</li>
<li>Repeat the previous steps as needed until desired amount and purity are obtained (see below).</li>
<li>Quantify purified DNA using Nanodrop. The yield of DNA recovery after precipitation varies from 50 up to 80%.</li>
<li>Correct the DNA 5′ and 3′ ends by treatment with T4 DNA polymerase (Thermo Fisher Scientific); this is very important to generate the blunt-ends required for efficient cloning into the vector. Use 1 U of T4 DNA Polymerase per μg of DNA in the presence of 0.1 mM dNTP at 11°C for 20 min.</li>
<li>Stop the reaction by heat inactivation (75°C for 10 min).</li>
<li>Purify DNA using the QIAGEN PCR Purification Kit. Respect the recommended column capacity by adding only 10 μg of DNA to each column. To be able to purify small fragments (&lt;300 bp), after mixing the sample with 5 volumes of the binding buffer (PB), add 2.5 volumes of 2-propanol and admix it thoroughly.</li>
<li>Run 100 ng of the material in a 2% agarose gel. A smear within the desired region (100–500 bp) should be observed (<a href="#fig2">Figure 2</a>).</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Library cloning<br> At this stage, it is necessary to digest the phagemid vector and optimize ligation conditions by varying the vector:insert ratios. It is important to remember that 1 in 18 clones in your final library will (theoretically) display a peptide epitope that belongs to the proteome of the target organism (<a href="#fig1">Figure 1</a>C). So, this should be taken into consideration in order to properly estimate the library diversity (unique sequences) and genome coverage (average number of genomic-fold coverage). Day 1:
<ol type="a">
<li>Digest the chosen phagemid vector with the restriction enzyme. If using the pG8SAET vector, use <i>Eco105I</i> (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2FMAN0012115_Eco105I_SnaBI_10_UuL_3000U_UG.pdf">Thermo Fisher Scientific</a>) following the standard manufacturer protocol.</li>
<li>Purify the digested vector by phenol:chloroform extraction and ethanol precipitation or an alternative methodology, such as silica column purification (e.g., Qiagen PCR purification kit).</li>
<li>Perform ligations (<a href="#tbl1">Table 1</a>) with T4 DNA ligase (Thermo Fisher Scientific) and different vector:insert ratios. To optimize vector:insert ratio, test at least three different ligation conditions using between 50 and 300 ng of insert for 100 ng of the restriction enzyme digested vector. Incubate the ligation reaction at 16°C for 16–24 h. These ligation conditions will roughly correspond to vector:insert molar ratios of 1:5, 1:10, 1:20, and 1:30 (mol/mol) (<a href="#fig2">Figure 2</a>B).
<table id="tbl1">
<caption>Table 1. Ligation reaction</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Quantity</th>
</tr>
</thead>
<tbody>
<tr>
<td>pG8SAET phagemid digested</td>
<td>100 ng</td>
</tr>
<tr>
<td>Blunt-end genomic fragments</td>
<td>50, 100, 200 or 300 ng</td>
</tr>
<tr>
<td>10<b>×</b> T4 DNA ligase buffer</td>
<td>To 1<b>×</b>
</td>
</tr>
<tr>
<td>PEG 50%</td>
<td>To 5%</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>5 U</td>
</tr>
</tbody>
</table>
Day 2:</li>
<li>Precipitate the ligations by adding 3 volumes of ice-cold ethanol, 0.1 volumes of 3M sodium acetate pH 5.2 and centrifuging at 20,000 × <i>g</i>, 4°C for 30 min. Wash with 70% ethanol, then centrifuge for 10–15 min to pellet the DNA (this is important to remove excess salt). Resuspend each ligation in 10 μL of ddH<sub>2</sub>O.</li>
<li>Transform each ligated DNA in 100 μL of <i>E. coli</i> DH10B strain by electroporation using a 2-mm gap cuvette (2.5 kV, 200 Ω, 25 μF), and recover in 1 mL of outgrowth media. Culture them at 37°C for 1 h, with agitation (250 rpm). Plate 10-fold serial dilutions (1/10, 1/100 and 1/1,000) in LB agar plates (duplicates) containing 50 μg/mL of carbenicillin plates to estimate the number of total transformants obtained.<br> Day 3:</li>
<li>Calculate the total number of transformants for each condition based on the number of host bacterial colonies obtained and the dilution factor.</li>
<li>With the optimal conditions and molar ratios empirically determined, scale up the ligation and transformation procedure accordingly to obtain the desired number of transformants (see below and item 5: gPhage display library quality control). In brief, the number of transformants multiplied by the average length of the inserts has to yield a number that is at least 100-times the length of the genome of the organism (ideally, &gt;1,000-times).</li>
<li>After transformation and culture outgrowth for 1 h, pool all transformed bacteria into one tube. Transfer 100 μL to a tube containing 900 μL of LB and perform 10-fold serial dilutions. Plate 100 μL of each dilution in carbenicillin containing agar plates (50 μg/mL) (duplicate). Incubate the plates at 37°C for 16–20 h. This procedure allows for calculating the total number of transformants.
<div class="note">
<span class="note-title">Note:</span> It is essential to perform colony counting at this time. After 12–16h culture (item i), your culture will be saturated and it is no longer possible to calculate the number of transformants.</div>
</li>
<li>Transfer the bacteria to 500 mL of liquid media (LB 50 μg/ul carbenicillin), and grow at 37°C for 12–16 h, at 250 rpm.</li>
<li>Next day, purify plasmid DNA by QIAGEN Maxiprep kit and CsCl (or your preferred alternative methodology), re-suspend it in TE buffer (Tris 10 mM, EDTA 1 mM, pH 8.0) and keep it stored at −20°C. The resulting reagent is a <i>T.cruzi</i> gPhage phage library, that can be used to produce peptide-displaying phage particles for selections. It is expected to obtain 200–500 μg of the library DNA.</li>
<li>Calculate the total number of transformants in our library based on the number of host bacterial colonies obtained and the dilution factor. To assess the quality of your library, follow the steps below.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Although we have not attempted, using a vector that has the <a href="https://blog.addgene.org/plasmids-101-ccdb-the-toxic-key-to-efficient-cloning">ccdb1 gene</a> as a stuffer may significantly reduce background colonies (with empty vector) and be a time saver as one does not need to purify the backbone after digestion.</div>
<ol start="3">
<li>Library quality-assurance/quality-control
<ol type="a">
<li>Assess library quality control by colony Polymerase Chain Reaction (PCR): pick random 96 (or more) bacterial colonies infected with the gPhage library and perform colony PCR following polymerase manufacturer instructions (e.g., for <a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2FMAN0012026_TaqDNAPolymerase_recombinant_5_UuL_100U_UG.pdf">ThermoFisher recombinant Taq polymerase</a>, 95°C<b>×</b>3 min; 35 cycles: 95°C<b>×</b>20 s, 55°C<b>×</b>30 s, 72°C<b>×</b>1min30 s; 72°C<b>×</b>5 min, 12°C <b>×</b> hold) and Sanger DNA sequencing primers (<a href="#tbl2">Table 2</a>). Use the empty vector pG8SAET as control (undigested and single-cut vector). Run samples in an agarose gel and calculate insert size (<a href="#fig2">Figure 2</a>). Comparison with the pG8SAET vector control should reveal whether the clones contain inserts.
<table id="tbl2">
<caption>Table 2. Primers used for pG8SAET colony sequencing and PCR</caption>
<thead>
<tr>
<th>Primer</th>
<th>Sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>pG8SAET Sequencing Forward Primer</td>
<td>CAGGGGGTATTAATTTGAAAAGG</td>
</tr>
<tr>
<td>pG8SAET Sequencing Reverse Primer</td>
<td>TATTCGGTCGCTGAGGCTTG</td>
</tr>
</tbody>
</table>
</li>
<li>Quality assurance and quality control (QA/QC) the resulting gPhage display by DNA sequencing (<a href="#tbl2">Table 2</a>): Pick 30–100 (or more) random bacterial colonies transformed with the gPhage library and sequence the insert by the Sanger sequencing method. Calculate insert length, verify the presence of open-reading frames (ORF) in each insert and make sure the DNA aligns to the genome of the species of interest.</li>
<li>Assess library quality by Next-Generation Sequencing (NGS): follow protocols below for Illumina Miseq DNA sequencing and assess insert size distribution, the presence and number of ORF(s), and annotated assignment to reference genomes and/or proteomes.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> if resources are limited, it is possible to perform the QA/QC of the gPhage phage library by NGS together with the biopanning selection samples by using DNA bar-coding. If so, it is important to perform the steps described in items a and b to ensure the library yields at least 100-fold coverage of the genome of the organism. It is also recommended that at least 10–20 colonies are sequenced by Sanger (item b) for QA/QC your library.</div>
<ol start="4">
<li>Library phage amplification and titration<br> Day 1: Transformation and superinfection
<ol type="a">
<li>Transform 10 μg of the gPhage library DNA by electroporation into <i>E. coli</i> TG1 strain (Lucigen, also commercialized by Merck). Transfer the transformed bacteria to a new tube. Plate 10-fold serial dilutions to assess the number of transformants as previously explained for DH10B strain (section 2.h) and to ensure it is larger than the original library diversity. For example, if the gPhage library contains 10<sup>8</sup> unique inserts, you should expect at least 10<sup>10</sup> transformants. In contrast to DH10B cloning strain, TG1 has a F′ genotype that enables superinfection by the helper phage and is also an amber codon suppressor strain that allows pVIII protein mounting when using pG8SAET system (there is amber codon between the genomic DNA insert and the pVIII gene).</li>
<li>Recover your pool of bacteria in 500 mL of LB media and grow for 1 h at 37°C, at 250 rpm. After one hour, add carbenicillin to 50 μg/mL. Grow your pool of transformed bacteria at 37°C (250 rpm agitation) until <i>log</i>-phase (OD<sub>600nm</sub>–0.4–0.8).</li>
<li>Add 5<b>×</b>10<sup>12</sup> plaque-forming units (pfu) of M13KO7 helper phage (New England Biolabs) at a final Multiplicity of Infection (MOI) of 20, and incubate it by shaking at 37°C for 1 h. To calculate the MOI, consider that an OD<sub>600nm</sub> of 1.0 represents approximately 10<sup>9</sup> cells/mL. For 500 mL, a total 2.5 cells × 10<sup>11</sup> cells are expected and 5<b>×</b>10<sup>12</sup> pfu of M13KO7 helper phage corresponds to and MOI of 20.</li>
<li>Add kanamycin to a final concentration of 25 μg/mL and grow for 16–20 h at 37°C, 250 rpm.<br> Day 2: Phage precipitation</li>
<li>Phage particles are purified from the cultured media using the PEG/NaCl method (<a href="#bib6">Giordano et al., 2001</a>). In brief, remove bacterial cells by centrifuging the bacterial culture at 7,000 × <i>g</i> for 15 min; recover supernatant and add 0.15 vol of PEG/NaCl (see <a href="#materials-and-equipment">materials and equipment</a> section). Incubate on ice for 1 h.</li>
<li>Centrifuge at 8,000 × <i>g</i> 4°C for 30 min to pellet the phage particles and discard the supernatant.</li>
<li>Re-suspend the resulting phage pellet in 10 mL of PBS. It helps to let it shake for 15 min at 37°C (250 rpm) and then clarify the phage solution by centrifuging the phage admixture at 7,000 × <i>g</i> for 15 min to remove residual bacterial debris.</li>
<li>Repeat the procedure. Recover the supernatant and add 0.15 vol of PEG/NaCl, incubate for 1 h at 4°C or on ice and centrifuge at 8,000 × <i>g</i> at 4°C for 20 min.</li>
<li>Discard the supernatant and re-suspend the phage pellet in 1 mL of PBS 50% glycerol (one may need to increase the volume if the resulting suspension is too viscous). Again, it helps to solubilize the phage particles by incubating for 15 min at 37<sup>o</sup>C (water bath).</li>
<li>Centrifuge the phage admixture at 20,000 × <i>g</i> for 5 min to remove bacterial aggregates and debris.</li>
<li>Recover the supernatant, aliquot the gPhage library and store it at −20°C until use.<br> Day 3: Library titration</li>
<li>Thaw an aliquot of your gPhage library and make serial dilutions (from 1/10<sup>7</sup> to 1/10<sup>9</sup>).</li>
<li>Grow one colony of <i>E. coli</i> TG1 strain in 10 mL of TB phosphate medium until OD<sub>600nm</sub> reaches ∼ 0.4–0.8.</li>
<li>Mix 200 μL of the bacteria with 20 μL of the diluted phage aliquots and incubate at 23°C–25°C for 30 min.</li>
<li>Plate infected bacteria in triplicate in LB agar (50 μg/mL carbenicillin). Incubate at 37°C for 16–20 h.</li>
<li>Next day: Count the host bacterial colonies and calculate the resulting titers in transducing units (TU) with the appropriate dilution factors.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Library QA/QC</li>
</ol>
<p>A suitable gPhage display library has to cover at least 100-times the genome of the organism. This is because a substantial number of inserts will produce phage particles that do not display a peptide or will display peptides that correspond to an alternative reading of the genome (<a href="#fig1">Figure 1</a>). Ideally, the gPhage display library should have a coverage greater than 1,000-fold the genome (<a href="#bib13">Teixeira et al., 2021</a>). Initial estimates may be obtained by calculating insert sizes based on PCR products and then by sequencing them (Sanger, NGS or both). Select random clones from the plates used to calculate the number of transformants, perform PCR reactions and analyze the samples side-by-side with the control empty vector to estimate insert size. Ultimately, a far more precise estimation can be achieved by using NGS sequencing. However, if resources are limited, the quality of the gPhage display library may be initially assessed by PCR and agarose gel estimates followed by Sanger DNA sequencing of 30–100 or so positive clone sample size to confirm that DNA inserts belong to the infectious organism of interest. Then, once the biopanning selection is complete, the gPhage display library may be further analyzed by NGS along with the remaining samples. Once you have information on the size distribution of the clones, one may perform the following mathematical calculations to estimate the coverage of your library in reference to your target genome:</p>
<p class="math-ml"><math> <mrow> <mi>L</mi> <mi>i</mi> <mi>b</mi> <mi>r</mi> <mi>a</mi> <mi>r</mi> <mi>y</mi> <mspace></mspace> <mi>s</mi> <mi>i</mi> <mi>z</mi> <mi>e</mi> <mo>=</mo> <mi>A</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mi>i</mi> <mi>n</mi> <mi>s</mi> <mi>e</mi> <mi>r</mi> <mi>t</mi> <mspace></mspace> <mi>s</mi> <mi>i</mi> <mi>z</mi> <mi>e</mi> <mspace></mspace> <mrow> <mo>(</mo> <mrow> <mi>b</mi> <mi>p</mi> </mrow> <mo>)</mo> </mrow> <mo>∗</mo> <mi>N</mi> <mi>u</mi> <mi>m</mi> <mi>b</mi> <mi>e</mi> <mi>r</mi> <mspace></mspace> <mi>o</mi> <mi>f</mi> <mspace></mspace> <mi>T</mi> <mi>r</mi> <mi>a</mi> <mi>n</mi> <mi>s</mi> <mi>f</mi> <mi>o</mi> <mi>r</mi> <mi>m</mi> <mi>a</mi> <mi>n</mi> <mi>t</mi> <mi>s</mi> </mrow> </math></p>
<p class="math-ml"><math> <mrow> <mi>L</mi> <mi>i</mi> <mi>b</mi> <mi>r</mi> <mi>a</mi> <mi>r</mi> <mi>y</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mo>=</mo> <mfrac> <mrow> <mi>L</mi> <mi>i</mi> <mi>b</mi> <mi>r</mi> <mi>a</mi> <mi>r</mi> <mi>y</mi> <mspace></mspace> <mi>s</mi> <mi>i</mi> <mi>z</mi> <mi>e</mi> </mrow> <mrow> <mi>G</mi> <mi>e</mi> <mi>n</mi> <mi>o</mi> <mi>m</mi> <mi>e</mi> <mspace></mspace> <mi>s</mi> <mi>i</mi> <mi>z</mi> <mi>e</mi> <mspace></mspace> <mrow> <mo>(</mo> <mi>b</mi> <mi>p</mi> <mo>)</mo> </mrow> </mrow> </mfrac> </mrow> </math></p>
<p>Illustrating here for the <i>T. cruzi</i> Sylvio X10 library (4.4 <b>×</b>10<sup>7</sup> bp genome):</p>
<p class="math-ml"><math> <mrow> <mi>L</mi> <mi>i</mi> <mi>b</mi> <mi>r</mi> <mi>a</mi> <mi>r</mi> <mi>y</mi> <mspace></mspace> <mi>s</mi> <mi>i</mi> <mi>z</mi> <mi>e</mi> <mo>=</mo> <mn>143</mn> <mspace></mspace> <mi>b</mi> <mi>p</mi> <mo>∗</mo> <mspace></mspace> <mn>4.4</mn> <mspace></mspace> <mi>x</mi> <msup> <mn>10</mn> <mn>8</mn> </msup> <mspace></mspace> <mi>t</mi> <mi>r</mi> <mi>a</mi> <mi>n</mi> <mi>s</mi> <mi>f</mi> <mi>o</mi> <mi>r</mi> <mi>m</mi> <mi>a</mi> <mi>n</mi> <mi>t</mi> <mi>s</mi> <mo>=</mo> <mn>6.29</mn> <mspace></mspace> <mi>x</mi> <msup> <mn>10</mn> <mn>10</mn> </msup> <mspace></mspace> <mi>b</mi> <mi>p</mi> </mrow> </math></p>
<p class="math-ml"><math> <mrow> <mi>g</mi> <mi>P</mi> <mi>h</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mi>d</mi> <mi>i</mi> <mi>s</mi> <mi>p</mi> <mi>l</mi> <mi>a</mi> <mi>y</mi> <mspace></mspace> <mi>l</mi> <mi>i</mi> <mi>b</mi> <mi>r</mi> <mi>a</mi> <mi>r</mi> <mi>y</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mo>=</mo> <mfrac> <mrow> <mn>6.29</mn> <mspace></mspace> <mi>x</mi> <msup> <mn>10</mn> <mn>8</mn> </msup> <mspace></mspace> <mi>b</mi> <mi>p</mi> </mrow> <mrow> <mn>4.4</mn> <mi>x</mi> <msup> <mn>10</mn> <mrow> <mn>7</mn> <mspace></mspace> </mrow> </msup> <mi>b</mi> <mi>p</mi> </mrow> </mfrac> <mo>=</mo> <mn>1,430</mn> <mi>x</mi> <mspace></mspace> <mi>c</mi> <mi>o</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> </mrow> </math></p>
<p>For the vector and cloning method reported here, only 1 in 18 inserts should, in theory, yield a phage particle that in fact display a <i>T. cruzi</i> peptide (<a href="#bib13">Teixeira et al., 2021</a>). If so, the theoretical library coverage would be: 80-times the encoded proteome.</p>
<h3 id="sec3.2">gPhage display library selection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<p>After gPhage display library construction, donor selection and IgG purification from serum samples, one is ready to start the selections. For that, gPhage library is exposed to the immobilized IgG molecules to allow for phage particles displaying specific antigens recognized by the IgG to be captured. After washing away the unbound phage, the remaining phage particles bound to the IgG are recovered by infection with bacteria and amplified for further rounds of selection. It is important to note that we recommend adding excess control IgG during the selection step to prevent binding of phage to IgG constant regions or the selection of ubiquitous antigens. See <a href="#bib13">Teixeira et al., 2021</a> (<a href="#fig2">Fig. 2</a>A within the reference) for an illustrative image of the epitope selection procedure for a <i>T. cruzi</i> library in the setting of patients with Chagas disease.</p>
<ol start="6">
<li>IgG immobilization and gPhage library selection Day 1: IgG coating
<ol type="a">
<li>Immobilize purified IgG (1 μg in 100 μL of PBS) on 96 wells microtiter high-binding enzyme-linked immunosorbent assay (ELISA) plate. Incubate the plate at 4°C for 16–24 h. Start by preparing a culture of <i>E. coli</i> TG1 bacteria by adding one colony in 10 mL of TB phosphate medium. The bacterial culture should be in <i>log</i>-phase (OD<sub>600nm</sub>–0.4–0.8) for the elution of phage bound to IgG - item e below).
<div class="note">
<span class="note-title">Note:</span> It is also possible to perform the selection on samples of individual patient; in that case, we recommend the use of protein-G or anti-IgG (Fc fragment) pre-coated microtiter plates to directly capture the patient-derived IgG from sera samples. This step will markedly reduce the required volume of biological sample(s) used in the experiment.</div>
Day 2: First round of selection</li>
<li>Wash wells 4-four times with PBS 0.1% Tween-20 to remove excess IgG. Block wells with PBS containing 1% bovine serum albumin (PBS/BSA) for 1 h at 23°C–25°C.</li>
<li>Add the gPhage display library in 100 μL of PBS/BSA plus 10 μg/mL of IgG from your control group. The presence of the control IgG in solution is essential to prevent selection of ubiquitous antigens. Incubate for 2 h at 23°C–25°C. Regarding the amount of gPhage library added, see note below.</li>
<li>Wash the wells ten times with 200μL of PBS.</li>
<li>Add 100 μL of your TG1 bacteria (<i>log</i>-phase) in each well and incubate for 30 min at 23°C–25°C.</li>
<li>Recover infected bacteria and dilute in 10 mL of LB broth containing carbenicillin (50 μg/mL).</li>
<li>Take two or three small aliquots (10 μL each) from the 10 mL suspension and perform 10-fold LB broth serial dilutions. Plate the dilutions in LB agar 50 μg/mL carbenicillin plates and incubate at 37°C. This will allow to calculate how many phage particles were recovered in each round of selection to monitor enrichment (see Note below).</li>
<li>Grow for 2 h at 37°C at 250 rpm.</li>
<li>Superinfect cells with the helper phage by adding 10<sup>10</sup> pfu of M13KO7. Incubate cell culture at 37°C for 1 h, with agitation (250 rpm).</li>
<li>Add kanamycin to a final concentration of 25 μg/mL. Grow it at 37°C for 16–20 h, with agitation (250 rpm).</li>
<li>To prepare for the next round, repeat the IgG immobilization step performed at the beginning of the procedure (item a).
<div class="note">
<span class="note-title">Note:</span> The quantification step may be skipped entirely during the first round of selection in order to avoid loss of phage clones: phage clones that are removed from the 16–20h culture and plated for quantification, are not amplified in the LB broth, and, therefore, will not be present in the successive rounds of selection. When using a gPhage library with a high coverage (&gt;1,000-times the genome), we find that this step is not a cause of concern due to insert redundancy (<a href="#bib13">Teixeira et al., 2021</a>).</div>
Day 3: Precipitation of first-round phages</li>
<li>Centrifuge the 16–20 h cell culture for 10 min at 10,000 × <i>g</i>. Transfer supernatant to a new tube and add 0.15 volumes of the PEG/NaCl solution to precipitate phage particles. Incubate on ice for 30 min.</li>
<li>Extract plasmid DNA from the remaining cell pellet. We recommend QIAprep Spin Miniprep Kit (or another method of your preference).</li>
<li>Count the bacterial colonies of your recovered gPhage plated on the previous day and calculate the number of phage bound to each IgG sample by correcting with the dilution factor and the volume of your culture (10 mL).</li>
<li>Centrifuge gPhage suspension 10 min at 10,000 × <i>g</i>. Discard the supernatant and resuspend the phage pallet in 1 mL of PBS/BSA.</li>
<li>Centrifuge for 5 min at 20,000 × <i>g</i> to remove remaining bacterial cells and debris. Transfer supernatant to a clean microtube to be used for a new round of selection.</li>
<li>Repeat selection procedure for a total of three rounds. Titrate both, the input and output phage for each round. An increase in the ratio output/input is a good indication of positive selection. If necessary, perform a fourth round. However, we find that further rounds of selection often do not improve results due to the takeover of fast amplifying phage particles and loss of diversity.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of phage input for the first round should be at least 10-times the diversity of the gPhage library. This will assure a fair representation of most phage inserts present in one’s library. However, for the remaining rounds of selection, phage input can be adjusted. In our case (<a href="#bib13">Teixeira et al., 2021</a>), we performed two independent biopannings. Although we used similar amounts of gPhage library during the first round (∼ 10<sup>9</sup> TU) for both selections, we varied the amount of phage during the second and third rounds of selection. This led to a significant difference in the overall number of antigens/epitopes identified. For details, see the discussion of the article.</div>
<h3 id="sec3.3">Sequencing and bioinformatic analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 weeks</div>
<ol start="7">
<li>Next-Generation DNA Sequencing In this step, plasmid DNA obtained from the naïve (unamplified, unselected) library or purified from each successive rounds of selection is used to generate amplicons suitable for Illumina MiSeq sequencing. The addition of variable length degenerations (<a href="#tbl3">Table 3</a>) between the region of interest and the partial Illumina adapters removes the need for PhiX spike-ins to add diversity to the sequencing pool – this original step increases the number of reads that may be recovered after a sequencing run by 10–25% depending on the protocol used. Specific primers for pG8SAET containing degeneration and Illumina adapters are listed in <a href="#tbl3">Table 3</a> and are used for the first PCR step (which adds partial Illumina adapter sequence). It is also important to use a high-fidelity polymerase such as NEB Q5 or Roche Kapa HiFi. A similar protocol is also detailed in a previous work of our group (<a href="#bib12">Tang et al., 2019</a>).
<table id="tbl3">
<caption>Table 3. Recommended sequence of primers for first PCR (pre-indexation)</caption>
<thead>
<tr>
<th rowspan="2">Primer</th>
<th colspan="3">Sequence</th>
</tr>
<tr>
<th>Partial Illumina index</th>
<th>Degenerated nt</th>
<th>pG8SAET annealing region</th>
</tr>
</thead>
<tbody>
<tr>
<td>Fw-0N</td>
<td>5′-TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG-</td>
<td> </td>
<td>-CTGCGCAACACGATGACC-3′</td>
</tr>
<tr>
<td>Fw-1N</td>
<td>5′-TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG-</td>
<td>-N-</td>
<td>-CTGCGCAACACGATGACC-3′</td>
</tr>
<tr>
<td>Fw-2N</td>
<td>5′-TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG-</td>
<td>-NN-</td>
<td>-CTGCGCAACACGATGACC-3′</td>
</tr>
<tr>
<td>Fw-3N</td>
<td>5′-TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG-</td>
<td>-NNN-</td>
<td>-CTGCGCAACACGATGACC-3′</td>
</tr>
<tr>
<td>Rv-0N</td>
<td>5′-GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG-</td>
<td> </td>
<td>-CTGCGCAACACGATGACC-3′</td>
</tr>
<tr>
<td>Rv-1N</td>
<td>5′-GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG-</td>
<td>-N-</td>
<td>-CTGCGCAACACGATGACC-3′</td>
</tr>
<tr>
<td>Rv-2N</td>
<td>5′-GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG-</td>
<td>-NN-</td>
<td>-CTGCGCAACACGATGACC-3′</td>
</tr>
<tr>
<td>Rv-3N</td>
<td>5′-GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG-</td>
<td>-NNN-</td>
<td>-CTGCGCAACACGATGACC-3′</td>
</tr>
</tbody>
</table>
<ol type="a">
<li>Use 10 ng of the plasmid DNA from the gPhage display selection and the unselected library as the template for your first PCR reaction and follow the DNA polymerase manufacturer instructions for the PCR (for <a href="https://rochesequencingstore.com/wp-content/uploads/2017/10/KAPA-HiFi-HotStart-ReadyMix-PCR-Kit_KR0370-%E2%80%93-v10.19.pdf">KAPA HiFi</a>, 95°C × 3 min; 20 cycles: 98°C<b>×</b>15 s, 58°C<b>×</b>15 s, 72°C<b>×</b>45 s; 72°C<b>×</b>2 min; 12°C <b>×</b> hold). Mix all 8 primers (4 forward and 4 reverse) in each reaction.</li>
<li>Purify the PCR products by using silica columns (e.g., QIAquick PCR Purification Kit).</li>
<li>Perform the second PCR reaction with the <a href="https://www.illumina.com/products/by-type/sequencing-kits/library-prep-kits/nextera-xt-dna.html">Nextera XT kit</a> (Illumina) primers to add the indexation (for KAPA HiFi, 95°C <b>×</b> 3 min; 8 cycles: 98°C<b>×</b>15 s, 55°C<b>×</b>15 s, 72°C<b>×</b>45 s; 72°C<b>×</b>2 min; 12°C <b>×</b> hold). Follow the standard manufacturer protocol to prepare and quantify your sample. Quantification by qPCR at this step will help adjust library concentration (Library Quantification Kit, Kapa Biosystems). We recommend a 4 nM final concentration of your sequencing library.</li>
<li>Sequence your samples using the MiSeq Reagent Kit v2 (500 cycles) on an Illumina MiSeq equipment. Other platforms, such as NovaSeq (SP 2<b>×</b>250 cycles Illumina), can be used for increased throughput. Depending on the expected insert length, one can choose to use a different number of cycles (e.g., 2<b>×</b>150 bp), but at the cost of losing longer sequences.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The PCR reaction should be limited to the minimal number of cycles possible to avoid introduction of point mutations and/or amplification-induced biases in our samples. We used 20 cycles for the first PCR and 8 for the second. We recommend the KAPA Hifi Hotstart ReadyMix 2<b>×</b> (Roche) or equivalent for PCR.</div>
<ol start="8">
<li>NGS Data Analysis Sequencing data analysis is done in two steps. In the first step we process raw reads, from which the peptide sequences are inferred. In the second step we process the peptide sequences to generate a nonredundant peptide sequence dataset. The reason for the second step is the following: If a gPhage display library has high coverage, several slightly different peptides that correspond to the same epitope may be recovered. In general, it means that peptide sequence clustering is necessary to generate a nonredundant epitope/antigen sequence dataset. All files necessary to run the pipeline have been supplied (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc1.zip">Data S1</a> and <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc2.zip">S2</a>) or can be downloaded from GitHub (<a href="https://github.com/aarteixeira/gPhage/">https://github.com/aarteixeira/gPhage/</a>) (preferred, in order to use the most up-to-date version of the scripts). To run the pipeline, follow these steps (<a href="#fig3">Figure 3</a>):
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Clustering procedure</div>
<p>Representation of the clustering algorithm to be used for antigen identification from third-round phage sequences (obtained by NGS) selected by the patient immunoglobulins (IgG).</p>
</figcaption>
</figure>
<ol type="a">
<li>Assemble the paired-ends with PEAR; tools such as FastP (<a href="#bib3">Chen et al., 2018</a>) and PandaSeq (<a href="#bib9">Masella et al., 2012</a>) may also be used for this step as well.</li>
<li>The assembled FastQ files can then be processed by using the script dna_processing.py (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc1.zip">Data S1</a>) to extract insert sequences and align them to the reference genome and/or proteome, as follows.
<ol type="i">
<li>Make sure you have Python3 software installed with the packages Pandas (v1.3.0), Biopython (v1.79) and Numpy (v1.21.0).</li>
<li>Make sure you have BlastN and BlastP (<a href="#bib2">Camacho et al., 2009</a>) installed for local pairwise alignment, and change the variable <i>blastn_exe</i> and <i>blastp_exe</i> in the script to reflect the path to the executables.</li>
<li>Using Blast software, create databases for your reference genome and proteome and change the <i>database</i> variable to reflect the path to the file.</li>
<li>Change the <i>dict_sample_fastq</i> variable (<a href="#tbl4">Table 4</a>) to the name of your samples and the path to the respective FastQ file. Add as many samples as needed.
<table id="tbl4">
<caption>Table 4. Variables that can be customized at the script dna_processing.py</caption>
<thead>
<tr>
<th>Variable</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>BLASTN_EXE</td>
<td>path/command to BlastN executable</td>
</tr>
<tr>
<td>BLASTN_DB</td>
<td>path to BlastN database</td>
</tr>
<tr>
<td>BLASTP_EXE</td>
<td>path/command to BlastP executable</td>
</tr>
<tr>
<td>BLASTP_DB</td>
<td>path to BlastP database</td>
</tr>
<tr>
<td>dict_sample_fastq</td>
<td>sample names and path to respective FastQ to be processed. To add a sample, add a new line with the following code: dict_sample_fastq["Sample Name"] = "path_to_fastq.fastq"</td>
</tr>
<tr>
<td>cutoff_dna_identity</td>
<td>minimum relative identity to reference genome for a DNA insert to be considered a hit (default is 0.9)</td>
</tr>
<tr>
<td>cutoff_peptide_similarity</td>
<td>minimum relative similarity to reference proteome for a peptide to be considered a hit (default is 0.6)</td>
</tr>
<tr>
<td>flank_upstream</td>
<td>sequence found upstream of the DNA insert (default set for pG8SAET sequence)</td>
</tr>
<tr>
<td>flank_downstream</td>
<td>sequence found downstream of the DNA insert (default set for pG8SAET sequence)</td>
</tr>
</tbody>
</table>
</li>
<li>If you want to remove DNA sequences that appeared just once (‘singletons’), change the variable <i>option_remove_singletons</i> to <i>True.</i> This setting is recommended only for your samples and not for the gPhage naïve library, since most sequences will be unique.</li>
<li>To remove DNA sequences that do not match the reference genome, change <i>option_remove_DNA_below_the arbitrary cutoff</i> to <i>True</i> and set the minimum identity at <i>cutoff_dna_identity</i> (we currently recommend 90% or higher).</li>
<li>To remove peptide sequences that do not match the reference proteome, change <i>option_remove_peptide_below_cutoff</i> to <i>True</i> and set the minimum identity at cutoff_peptide_similarity (we currently use 60% or higher).</li>
<li>Run the script: “python3 dna_processing.py” to obtain de output files (<a href="#tbl5">Table 5</a>).
<table id="tbl5">
<caption>Table 5. Output files generated by the script dna_processing.py</caption>
<thead>
<tr>
<th>Output file</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>dna_dataframe.xlsx</td>
<td>table containing all DNA inserts identified in all samples, their respective frequencies in each sample and BlastN results against reference genomes.</td>
</tr>
<tr>
<td>dna_above_cutoff.xlsx</td>
<td>same as dna_dataframe.xlsx, but containing only the inserts that match the reference genomes above the BlastN identity cutoff</td>
</tr>
<tr>
<td>dna_below_cutoff.xlsx</td>
<td>same as dna_dataframe.xlsx, but containing only the inserts that are below the BlastN identity cutoff (or do not match reference genomes)</td>
</tr>
<tr>
<td>peptide_dataframe.xlsx</td>
<td>table containing all peptides identified in all samples peptide, including the frequency in each sample and BlastP results against reference proteomes.</td>
</tr>
<tr>
<td>norm_peptide_df.xlsx</td>
<td>same as peptide_dataframe.xlsx, but with counts normalized by total DNA (relative frequency)</td>
</tr>
<tr>
<td>peptide_above_cutoff.xlsx</td>
<td>same as peptide_dataframe.xlsx, but containing only the peptides that match reference proteomes above the BlastP similarity cutoff</td>
</tr>
<tr>
<td>peptide_below_cutoff.xlsx</td>
<td>same as peptide_dataframe.xlsx, but containing only the peptides that are below the BlastP similarity cutoff (or do not match reference proteomes)</td>
</tr>
<tr>
<td>statistics.xlsx</td>
<td>general processing statistics</td>
</tr>
<tr>
<td>peptides.txt</td>
<td>A list of all peptides identified by the script. This file can used as input for the clustering.py script</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Another consideration is whether the bacteria you have used to amplify the library is a suppressor strain. For TG1 bacteria (amber suppressor), the codon TAG codes for a glutamine instead of the usual stop codon. Our script takes this change into consideration.</div>
</li>
</ol>
</li>
<li>Having obtained all the information regarding the peptide sequences of interest, one may now generate clusters to characterize the epitopes using the script clustering.py (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc2.zip">Data S2</a>).
<ol type="i">
<li>Make sure you have Python3 installed with the packages Biopython and FuzzyWuzzy installed (<a href="#bib8">Levenshtein, 1966</a>).</li>
<li>Make sure you have the software MAFFT (<a href="#bib7">Katoh and Standley, 2013</a>) and HMMER (<a href="#bib10">Mistry et al., 2013</a>) installed and change the variables <i>MAFFT</i>, <i>HMMBUILD</i>, and <i>HMMEMIT</i> (<a href="#tbl6">Table 6</a>) to reflect the path to the corresponding executables.
<table id="tbl6">
<caption>Table 6. Variables that can be customized at the script clustering.py</caption>
<thead>
<tr>
<th>Variable</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>PEPTIDES_TXT</td>
<td>path to the file containing the peptides to be clustered (text file containing one peptide sequence per line)</td>
</tr>
<tr>
<td>MAFFT</td>
<td>path/command to MAFFT executable</td>
</tr>
<tr>
<td>HMMBUILD</td>
<td>path/command to hmmbuild executable</td>
</tr>
<tr>
<td>HMMEMIT</td>
<td>path/command to hmmemit executable</td>
</tr>
<tr>
<td>K</td>
<td>K-mer length for peptide search. Only peptides that share a K-mer are compared. Default is 4 amino acids.</td>
</tr>
<tr>
<td>min_len</td>
<td>minimum length of the peptides used for clustering. Peptides with length below this threshold are excluded. Default is 6 amino acids.</td>
</tr>
<tr>
<td>score_cutoff</td>
<td>minimum score (% partial identity) for peptides to be clustered together. Default is 80.</td>
</tr>
</tbody>
</table>
</li>
<li>Create a file named peptides.txt containing all peptides to be clustered (one peptide per line) in the same folder containing the script. If you used the dna_processing.py script, a “peptide.txt” was generated for you.</li>
<li>Change variables <i>K</i>, <i>min_len</i>, and <i>score_cutoff</i> to alter the K-mer length for comparison (default is 4), minimum length of a peptide to be considered (default is 6) and minimum percentage similarity between two peptides to be grouped (default is 80), respectively.</li>
<li>Run the script: “python3 clustering.py” to obtain the output files (<a href="#tbl7">Table 7</a>).
<table id="tbl7">
<caption>Table 7. Output files generated by the script clustering.py</caption>
<thead>
<tr>
<th>Output file</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>cluster_consensus.xlsx</td>
<td>table containing cluster IDs, number of different peptides forming the clusters, and consensus sequence.</td>
</tr>
<tr>
<td>peptide_cluster.xlsx</td>
<td>table containing all peptide sequences and corresponding cluster.</td>
</tr>
<tr>
<td>align folder</td>
<td>folder containing the multiple sequence alignments, consensus sequences, and HMM profiles for each cluster.</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Characterization of the gPhage library</li>
</ol>
<p>You should determine the number of unique inserts and the average size of your inserts. It is necessary to determine the percentage of transformants that contain <i>T. cruzi</i> inserts. Considering the total number of transformants, one may use BLASTn to determine the percentage of clones corresponding to your target organism. The scripts used for library characterization are available in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc1.zip">Data S1</a>.</p>
<div class="note">
<span class="note-title">Note:</span> In our study, the gPhage library contained ∼4<b>×</b>10<sup>8</sup> (as determined by the number of bacterial colonies when plating the library after initial transformation). In this step, the equation (see Library QA/QC step) is used to calculate the real coverage. NGS of ∼2.2<b>×</b>10<sup>5</sup> sequences showed high diversity (99% unique sequences) with an average size of 143 bp and that 75% clones (∼3<b>×</b>10<sup>8</sup>) contained a <i>T. cruzi</i> insert. All this corresponds to a total of ∼4.3<b>×</b>10<sup>10</sup> bp of the <i>T. cruzi</i> haploid genome (3<b>×</b>10<sup>7</sup> bp) or ∼1,400 times. However, due to the blunt-end cloning strategy, inserts may be cloned in all possible reading frames and orientations. So, not all inserts result in peptide display and not all displayed peptides correspond to a <i>T. cruzi</i> protein (<a href="#fig1">Figure 1</a><b>C</b>, some peptides may be derived from alternative reading frames of the genome). Taking this into consideration, only 17.8% of the inserts in our gPhage library encode a peptide and among those, only half (8.9%) share some degree of similarity (60%) with a <i>T. cruzi</i> protein.</div>
<h3 id="sec3.4">Identified epitope validation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 weeks</div>
<p>To validate identified epitopes one may choose two alternative routes: (i) To use synthetic peptides and direct ELISA assays or, (ii) To perform sandwich ELISA or FLISA (Fluorescence-linked immunosorbent assay) assays by using gPhage particles displaying the epitope. The first option is straightforward and may allow the validation of linear epitopes, granted that the synthetic peptide binds the ELISA plate. However, for most epitopes (including but not limited to conformational ones) and those in which the synthetic peptide does not adhere to plastic, one can use an indirect approach in which the donor IgG is first coated onto the ELISA plate, incubated with the gPhage particles displaying the peptide of interest, and finally detected by using a secondary reagent (Sandwich gPhage-based assays). This option has the advantage that the peptide will retain its original conformation as displayed on the phage particle.</p>
<ol start="10">
<li>Direct ELISA validation with peptides</li>
</ol>
<p>The NGS data generated is a useful starting point to select candidates. Epitopes highly enriched against the patient serum samples of interest but not against the control serum samples are more likely to be true hits. Often times, the most abundant antigens also contain the highest number of overlapping peptides, which may be used for epitope mapping by identifying the conserved core sequence. A synthetic peptide may then be designed and custom-ordered for validation assays.</p>
<p>Use standard ELISA to immobilize the peptide on the microtiter wells and test the reactivity against either purified IgGs or serum samples. However, in cases where reactivity is not detected, there is still the possibility that the synthetic peptide does not perfectly mimic the true epitope conformation and/or not adhering well to the plastic surface. Detailed results of the ELISA analysis for Chagas disease-related epitopes screening are shown in <a href="#bib13">Teixeira et al. (2021)</a> (<a href="#fig4">Figure 4</a> in the article).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. A gPhage-based validation assay</div>
<p>(A) Scheme to illustrate the phage binding assay applicable to validate the specific binding of epitope-containing phage to patient immunoglobulins. The number of colonies obtained after plating and incubation can be multiplied by the dilution factor to obtain the Transducing Units (TU, viable phage particles) value. The validation can also be performed by immunoassays with an anti-bacteriophage antibody labeled with HRP or a fluorophore, for example.</p>
<p>(B) Phage binding assay for a specific phage obtained when performing the panning with a pool of severe cardiomyopathy Chagas disease patients. For this binding assay, two pools of sera from patients with severe cardiomyopathy (S1 IgG, S2 IgG) and two pools of control sera (C1 IgG, C2 IgG) were used. Phage particles (CCC phage) show strong binding to immunoglobulins of patients with severe cardiomyopathy in comparison to the corresponding binding to negative control insertless phage particles (pG8SAET).</p>
<p>(C) Relative phage binding value (quotient of the CCC and pG8 binding values in Transducing Units for each point) when performing the binding assay with individual patient serum samples. An index (S8) patient serum sample presents a 200-fold CCC/pG8 quotient, indicating a strong sensitivity when incubating the CCC phage particles with the IgG of this patient. The index patient serum was included in the S1 IgG and S2 IgG pools during the panning selection.</p>
<p>(D) Reactivity of the CCC phage with index patient serial serum dilutions. High-reactivity is observed until 10<sup>6</sup> dilution. In all cases, error bars represent SEM from biological (N=2) and technical (N=3) replicas.</p>
</figcaption>
</figure>
<ol start="11">
<li>Sandwich gPhage-based assays This type of assays is useful to assess the reactivity of epitopes that require specific conformations for binding or peptides that may be hard to adhere to the microtiter plates. First, isolate a single bacterial colony with the gPhage displaying the antigen of interest. In cases where the antigen/peptide of interest has a high frequency in the pool of isolated phage particles, the corresponding monoclonal phage may be isolated by sequencing individual bacterial colonies by Sanger DNA sequencing. If that is not possible (i.e., low frequency phage clone), synthetic oligonucleotides encoding the antigen sequence may be ordered and subsequently cloned back into the phagemid vector pG8SAET. It may also be useful to produce phage particles from the empty phagemid – this step will yield particles that do not display any foreign peptides and will serve as a suitable negative control in the binding assays. This protocol can be performed by using purified IgG or by direct capturing the IgGs on the microtiter plate. Also, the quantification step may be performed either by using fluorescent/colorimetric secondary reagents or by rescuing the bound phage with host bacteria infection and plating serial dilutions for colony counting (<a href="#fig4">Figure 4</a>A).
<ol type="a">
<li>Monoclonal Phage Production
<ol type="i">
<li>In a 50 mL conical tube, inoculate a single bacterial colony of TG1 bacteria infected with the clone of interest into 10 mL of LB broth 50 μg/mL carbenicillin and grow at 37°C, with agitation (250 rpm) until early <i>log</i> phase (OD<sub>600nm</sub> = ∼0.5).</li>
<li>Perform the helper phage superinfection, growth, precipitation, and titration of the phage produced as described at <i>“Library phage amplification and titration.”</i>
</li>
</ol>
</li>
<li>IgG immobilization and gPhage binding<br> <b><i>Option A:</i></b> Microtiter well coating procedure if working with purified IgG:
<ol type="i">
<li>Immobilize IgG (10 μg/mL in PBS) from individual donors onto 96-well High-Binding ELISA plates (100 μL/well), by incubation at 4°C for 16–24 h. Use triplicate wells for each clone to be tested, plus triplicate for each negative control.</li>
<li>Wash the wells four times with PBS (200 μL) and block with PBS/BSA for 1 h at 23°C–25°C.</li>
<li>Incubate with selected gPhage particle or the negative control (insertless phage, pG8SAET). Use 10<sup>9</sup> TU of phage particles in 100 μL PBS/BSA for 2 h at 23°C–25°C.</li>
<li>Wash the wells four times with PBS OR<br> <b><i>Option B:</i></b> Microtiter well coating procedure if working with donor serum:</li>
<li>Immobilize goat anti-human IgG Fc (Jackson Immuno Research) (10 μg/mL in PBS) onto 96-wells of a High Binding ELISA plates by incubation at 4°C for 16–24 h (use 100 μL/well). Use triplicate wells for each clone to be tested, plus triplicate for each negative control (control from healthy donors and the goat-anti human IgG Fc with no added serum).</li>
<li>Wash the wells four times with PBS (200 μL) and block the wells with PBS/BSA for 2 h at 23°C–25°C.</li>
<li>Quantify using nanodrop (OD<sub>280nm</sub>) the protein concentration in your sera samples. Adjust the serum protein concentration to 120 μg/mL with PBS. Add 50 μL of diluted serum per well coated with anti-human IgG Fc. Incubate for 2 h at 23°C–25°C.</li>
<li>Wash the wells four times with PBS</li>
<li>Incubate with selected gPhage particle or the negative control (insertless phage, pG8SAET). Use 10<sup>9</sup> TU of phage particles in 100 μL PBS/BSA for 2 h at 23°C–25°C.</li>
<li>Wash the wells four times with PBS</li>
</ol>
</li>
<li>gPhage binding quantification<br> <b><i>Option A:</i></b> Immunodetection (near-infrared fluorescence quantification):
<ol type="i">
<li>Incubate with the anti-fd bacteriophage antibody produced in rabbit (Sigma) for 1 h at 23°C–25°C (1:400 in PBS; 100 μL).</li>
<li>Wash the wells, incubate with IRDye 680LT Goat anti-Rabbit IgG Secondary Antibody for 1 h at 23°C–25°C (1:1,000 in PBS; 100 μL) (Licor Biosciences).</li>
<li>Wash the wells four times with PBS and proceed with the quantification step by using the Odyssey system (or equivalent near-infrared fluorescence quantification system). OR<br> <b><i>Option B:</i></b> Quantification by colony counting:</li>
<li>Rescue phage form each well with 200 μL of <i>E. coli</i> TG1 in <i>log</i>-phase and infect at 37°C for 30 min.</li>
<li>Make serial dilutions (from 1:10 to 1:1000) and plate in quadruplicate each well and incubate at 37°C for 16–20 h.</li>
<li>Next day: Count all bacterial colonies and plot the total TU of selected phage and compare with the control insertless phage (<a href="#fig4">Figure 4</a>).</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Examples of these assays for the gPhage particle displaying the peptide motif PPHTRRVTVRCGPPSCADE vs. control phage particles (pG8SAET), selected by serial binding to the pool of serum samples from patients with Chronic Chagas disease Cardiomyopathy (CCC) patient pool (<a href="#bib13">Teixeira et al., 2021</a>) are shown in <a href="#fig4">Figures 4</a>B–4D.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>In general, constructing genomic libraries of complex organisms is technically cumbersome, labor- and cost-intensive procedures. The gPhage technology reported here allows for the constructions of relatively fast and cost-effective genomic phage display library of eukaryotes with no- or low-intron penetrance, viruses, bacteria or other organisms with or without available genomic information. This methodology is especially useful for the rapid identification of epitopes in emerging viral or neglected parasitic diseases. When applying this procedure, one should obtain gPhage libraries with high coverage to allow for unbiased selections. The libraries should include multiple copies of the same epitope, with difference in sizes that, in some cases, may facilitate the identification of large conformational epitopes. Although only a fraction of the initial gPhage display library phage particles encode true antigens from the organism of interest, upon selection against patient immunoglobulins, these out-of-frame phage particles work as an internal control: during selection, it is expected an enrichment of in-frame inserts that can then be clustered to define B-cell epitopes. The most abundant peptides can then be conveniently validated by ELISA and/or phage binding against the patient-derived immunoglobulins or without the need of expensive peptide synthesis by using the sandwich gPhage-based assay option. These even may result in some gPhage particles that may be used as diagnostic tools of the target disease.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The standard software GraphPad Prism6 was used for graphical and statistical analysis. Student t-tests were used to compare results with control and testing phage particles (<a href="#fig4">Figure 4</a>A).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The amount of starting material (genomic DNA) from your target organism may limit your pipeline. Although the protocol may be followed in a straightforward manner, it is necessary to obtain a high coverage library to perform the selections. This could demand multiple vector ligation attempts. Besides, in the ligation step concatemers may form and artificially join two distinct genomic fragments into a single insert. Moreover, one is also limited by the size of the peptides being displayed: epitopes relying on long domains may not be identified when small fragments sizes are used. Even if using large fragments, there is no guarantee that the foreign sequences will be properly produced, folded, and displayed by the phage (because of many factors, including possible toxicity to the phage/bacteria). Finally, gPhage is better suited to organisms with low-intron content, but a similar approach may conceivably be used with cDNA instead, with the potential caveat that the library will be inherently biased by the genomic expression at the time of RNA extraction.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Insufficient fragmentation of genomic DNA.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Increase duty cycle or intensity until you obtain a smooth smear of DNA with the desired range (100–500 bp) is visible by agarose gel electrophoresis (refer to step 1b).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Not enough number of transformants after library ligation and electroporation.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Test different vector:insert ratios (refer to steps 2c and 2 g).</p>
<p>It is always helpful to transform your non-digested empty vector (pG8SAET) to estimate the quality of your batch of electrocompetent bacteria. It should yield at least 10<sup>9</sup> transformants per μg of plasmid DNA (refer to step 2e).</p>
<p>Increase the ligation time and use 5% PEG 6000 in your ligation mix (refer to step 2c).</p>
<p>Phosphorylate your fragmented insert with T4 polynucleotide kinase (refer to step 1 m).</p>
<p>If necessary, re-purify the vector and inserts again to remove any interfering contaminants (refer to steps 1 m and 2b).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>After optimization, my first library does not cover the whole genome (section 5 - Library QA/QC).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Do not discard the library (refer to steps 3c and 5).</p>
<p>Repeat library construction with a larger amount of insert and vector (refer to step 2 g).</p>
<p>It is possible to combine two, three or more sub-libraries to produce a complete library in order to achieve at least 100-time genome coverage (ideally, &gt;1,000-times) (refer to steps 3c and 5).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Limited phage particle yields during each round of screening.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Re-titer your helper phage and reassure you are applying the correct amount of plate forming units (refer to step 4c).</p>
<p>If necessary, prepare a new batch of helper phage (<a href="https://international.neb.com/protocols/0001/01/01/m13-amplification">https://international.neb.com/protocols/0001/01/01/m13-amplification</a>) (refer to steps 4c and 6i).</p>
<p>Check the antibiotic activity of the batches of carbenicillin and kanamycin in use (refer to steps 6f and 6j).</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Non-reactive synthetic peptide in ELISA for a highly enriched bacteriophage after selection.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>If possible, order a synthetic peptide containing biotin and use streptavidin conjugated to horseradish peroxidase to assess peptide binding to the microtiter plate. Alternatively, perform the sandwich gPhage-binding assay to validate the selected antigen (refer to step 10).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Alves, M.J.M., Abuin, G., Kuwajima, V.Y., and Colli, W. (1986). Partial inhibition of trypomastigote entry into cultured mammalian cells by monoclonal antibodies against a surface glycoprotein of <i>Trypanosoma cruzi</i>. Mol. Biochem. Parasitol. <i>21</i>, 75-82. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref1">View at publisher</a></p>
<p id="bib2">Camacho, C., Coulouris, G., Avagyan, V., Ma, N., Papadopoulos, J., Bealer, K., and Madden, T.L. (2009). BLAST+: architecture and applications. BMC Bioinformatics <i>10</i>, 1-9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref2">View at publisher</a></p>
<p id="bib3">Chen, S., Zhou, Y., Chen, Y., and Gu, J. (2018). FASTP: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics <i>34</i>, i884-i890. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref3">View at publisher</a></p>
<p id="bib4">El-Sayed, N.M., Myler, P.J., Bartholomeu, D.C., Nilsson, D., Aggarwal, G., Tran, A.N., Ghedin, E., Worthey, E.A., Delcher, A.L., Blandin, G., et al. (2005). The genome sequence of <i>Trypanosoma cruzi</i>, etiologic agent of Chagas disease. Science <i>309</i>, 409-415. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref4">View at publisher</a></p>
<p id="bib5">Giordano, R., Fouts, D.L., Tewari, D., Colli, W., Manning, J.E., and Alves, M.J.M. (1999). Cloning of a surface membrane glycoprotein specific for the infective form of <i>Trypanosoma cruzi</i> having adhesive properties to laminin. J. Biol. Chem. <i>274</i>, 3461-3468. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref5">View at publisher</a></p>
<p id="bib6">Giordano, R.J., Cardó-Vila, M., Lahdenranta, J., Pasqualini, R., and Arap, W. (2001). Biopanning and rapid analysis of selective interactive ligands. Nat. Med. <i>7</i>, 1249-1253. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref6">View at publisher</a></p>
<p id="bib7">Katoh, K. and Standley, D.M. (2013). MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol. Biol. Evol. <i>30</i>, 772-780. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref7">View at publisher</a></p>
<p id="bib8">Levenshtein, V.I. (1966). Binary codes capable of correcting deletions, insertions, and reversals. Soviet Phys. Doklady <i>10</i>, 707-710. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref8">View at publisher</a></p>
<p id="bib9">Masella, A.P., Bartram, A.K., Truszkowski, J.M., Brown, D.G., and Neufeld, J.D. (2012). PANDAseq: paired-end assembler for Illumina sequences. BMC Bioinformatics <i>13</i>, 1-7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref9">View at publisher</a></p>
<p id="bib10">Mistry, J., Finn, R.D., Eddy, S.R., Bateman, A., and Punta, M. (2013). Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions. Nucleic Acids Res. <i>41</i>, e121. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref10">View at publisher</a></p>
<p id="bib11">Sambrook, J. and Russel, D.W. (2001). Molecular cloning: a Laboratory manual. Cold Spring Harb. Protoc. <i>10</i>, 1. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref11">View at publisher</a></p>
<p id="bib12">Tang, F.H., Staquicini, F.I., Teixeira, A.A., Michaloski, J.S., Namiyama, G.M., Taniwaki, N.N., Setubal, J.C., da Silva, A.M., Sidman, R.L., Pasqualini, R., et al. (2019). A ligand motif enables differential vascular targeting of endothelial junctions between brain and retina. Proc. Natl. Acad. Sci. U S A <i>116</i>, 2300-2305. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref12">View at publisher</a></p>
<p id="bib13">Teixeira, A.A.R., Carnero, L.R., Kuramoto, A., Tang, F.H.F., Gomes, C.H., Pereira, N.B., Oliveira, L., Garrini, R., Monteiro, J.S., Giordano, R.J., et al. (2021). A refined genome phage display methodology delineates the human antibody response in patients with Chagas disease. iScience <i>24</i>, 102540. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref13">View at publisher</a></p>
<p id="bib14">Zhang, L., Jacobsson, K., Ström, K., Lindberg, M., and Frykberg, L. (1999). Staphylococcus aureus expresses a cell surface protein that binds both IgG and β2-glycoprotein I. Microbiology <i>145</i>, 177-183. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref14">View at publisher</a></p>
<p id="bib15">Zingales, B., Andrade, S.G., Briones, M.R., Campbell, D.A., Chiari, E., Fernandes, O., Guhl, F., Lages-Silva, E., Macedo, A.M., Machado, C.R., et al. (2009). A new consensus for <i>Trypanosoma cruzi</i> intraspecific nomenclature: second revision meeting recommends TcI to TcVI. Mem. Inst. Oswaldo Cruz <i>104</i>, 1051-1054. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00642-0/sref15">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and reasonable requests for resources should be directed to and fulfilled by the lead contact, Ricardo José Giordano (<a href="mailto:giordano@iq.usp.br">giordano@iq.usp.br</a>).</p>
<h4>Materials availability</h4>
<p>The vectors and reagents generated in the study are available from the lead contact upon reasonable request.</p>
<h4>Data and code availability</h4>
<p>Data used and codes generated can be found in <a href="#bib13">Teixeira et al. (2021)</a> and in this article <a href="#appsec1">supplemental information</a>.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc1.zip">Data S1. Python scripts to process large-scale DNA sequencing data, related to step 2 (NGS Data Analysis)</a></p>
<p><a id="mmc2" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1159-Mmc2.zip">Data S2. Python scripts for clustering the peptides, related to step 2 (NGS Data Analysis)</a></p>
<h3>Acknowledgments</h3>
<p>We thank Drs. M.L. Baldini and C.L. Braga for assistance with reagents. This work was supported by research grants from São Paulo Research Foundation (<a href="https://doi.org/10.13039/501100001807">FAPESP</a>) to R.J.G. (2016/22.645–1 and 2014/21177-9) and to W.C. (2013/16.478–7 and 2016/14.506–1) and fellowships to A.A.T. (2012/13.967–4 and 2016/06.069–0) and the <a href="https://doi.org/10.13039/501100003593">National Council for Scientific and Technological Development</a> (CNPq) to R.J.G. (307412/2017-2). This work has been supported in part by the Coordenação de Aperfeiçoamento de Pessoal de Nível Superior in Brazil (<a href="https://doi.org/10.13039/501100002322">CAPES</a>, Finance Code 001). R.P. and W.A. received research awards from the Levy-Longenbaugh Donor-Advised Fund. <a href="http://BioRender.com">BioRender.com</a> was used in the preparation of some figures.</p>
<h3>Author contributions</h3>
<p>L.A.R.C., A.A.T., J.C.S., W.C., M.J.M.A., E.C.-N., R.P., W.A., and R.J.G.: conceptualization and formal analysis. A.A.T. and J.C.S.: software and resources. L.A.R.C., A.A.T., and F.H.F.T. performed experiments. W.C., J.C.S., M.J.M.A., and E.C.-N. contributed with reagents. L.A.R.C., A.A.T., R.P., W.A., and R.J.G. wrote the manuscript or edited the draft. W.C. and R.J.G. supervised the work and obtained study funding.</p>
<h3>Declaration of interests</h3>
<p>W.A. and R.P. are founders and equity stockholders of PhageNova Bio and of MBrace Therapeutics; R.P. also serves as the chief scientific officer of PhageNova Bio and of MBrace Therapeutics (all outside of the submitted work); these arrangements are managed in accordance with the established institutional conflict of interest policies of Rutgers, The State University of New Jersey. The remaining authors have declared no potential competing interests.</p>
</section>
</article>