<article>
<section>
<h1 id="header">Whole planarian chromosome squash</h1>
<p><time datetime="2021-01-15">Published: January 15, 2021</time></p>
<p>Paul G. Barghouth<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> and Néstor J. Oviedo<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular &amp; Cell Biology, University of California, Merced, Merced, CA, USA</p>
<p id="aff2"><sup>2</sup>Quantitative and Systems Biology Graduate Program, University of California, Merced, Merced, CA, USA</p>
<p id="aff3"><sup>3</sup>Health Sciences Research Institute, University of California, Merced, Merced, CA, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:pbarghouth@ucmerced.edu">pbarghouth@ucmerced.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:noviedo2@ucmerced.edu">noviedo2@ucmerced.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100257">10.1016/j.xpro.2020.100257</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Whole planarian chromosome squash allows researchers to qualitatively analyze chromosome integrity. Treatment with colchicine is used to halt dividing cells within metaphase and does not require amputation or tissue puncturing. In combination with acetic-orcein, a stain-fixative for chromosomes, this strategy is suitable for animals with friable tissues caused by drug treatment, radiation, and RNA interference phenotypes. The whole planarian squash method presented here is a minimally invasive procedure that facilitates simultaneous analysis of chromosomal integrity in control and experimental animals.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib6">Peiris et al. (2016)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Analysis of whole planarian chromosome integrity</li>
<li>Minimally invasive and does not require procedures such as amputation or puncture</li>
<li>Feasible in friable tissues resulting from pharmacological treatments or RNAi phenotypes</li>
<li>Analysis in parallel of control and experimental samples</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/364-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>On day 1 of the protocol fresh colchicine solution should be made. Prior to the start of day 2 of the protocol, all karyotyping reagents must be made fresh.</p>
<h3 id="sec1.1">Colchicine solution</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<ol>
<li>Prepare fresh 0.05% colchicine solution.
<ol type="a">
<li>Weight 0.005 g of 97% colchicine.</li>
<li>Dilute in 10 mL of planarian 1× Montjuic saltwater.</li>
<li>Vortex until powder dissolves.</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Karyotyping reagents</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<ol start="2">
<li>Perform tissue fixation in 3:1 ethanol: acetic acid.</li>
<li>Prepare 1 N HCl by diluting 12.1 N HCl in MilliQ water.</li>
<li>Prepare 60% acetic acid solution in MilliQ water.</li>
<li>Prepare 1:1:1 ratio of lactic acid, acetic acid, and MilliQ water.</li>
<li>Preparation of 1% acetic-orcein solution
<ol type="a">
<li>Weigh 1 g orcein.</li>
<li>Dissolve in 45 mL of hot (e.g., near boiling) acetic acid.</li>
<li>Once dissolved, allow solution to cool.</li>
<li>Add 55 mL of MilliQ water, shake well, and filter.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>97% colchicine</td>
<td>ACROS</td>
<td>Cat# 227120010</td>
</tr>
<tr>
<td>Orcein</td>
<td>Sigma-Aldrich</td>
<td>Cat# O7380</td>
</tr>
<tr>
<td>Lactic acid</td>
<td>Fisher Scientific</td>
<td>Cat# A159-500</td>
</tr>
<tr>
<td>Glacial acetic acid</td>
<td>Fisher Scientific</td>
<td>Cat# A38-500</td>
</tr>
<tr>
<td>Hydrochloric acid 12.1 N</td>
<td>Fisher Scientific</td>
<td>Cat# A144-500</td>
</tr>
<tr>
<td>Ethyl alcohol denatured</td>
<td>Fisher Scientific</td>
<td>Cat# A407-4</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Planarian: <i>Schmidtea mediterranea</i>
</td>
<td>n/a</td>
<td><i>CIW4</i></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ</td>
<td><a href="https://imagej.nih.gov">https://imagej.nih.gov</a></td>
<td>Version 1.48</td>
</tr>
<tr>
<td>NIS Elements AR</td>
<td>Nikon</td>
<td>Version 3.2</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>100× objective</td>
<td>Nikon</td>
<td>n/a</td>
</tr>
<tr>
<td>Nikon AZ-100 multi-zoom</td>
<td>Nikon</td>
<td>n/a</td>
</tr>
<tr>
<td>24 × 60 mm cover slips</td>
<td>Fisher Scientific</td>
<td>Cat# 12-545-89</td>
</tr>
<tr>
<td>22 × 22 mm cover slip</td>
<td>Fisher Scientific</td>
<td>Cat# 12547</td>
</tr>
<tr>
<td>6 cm petri dish</td>
<td>Fisher Scientific</td>
<td>Cas# FB0875713A</td>
</tr>
<tr>
<td>Thermomixer-R or Precision GP</td>
<td>EppendorfThermo Scientific</td>
<td>Cat# EP022670107Cat# TSGP02</td>
</tr>
<tr>
<td>1.5 mL centrifuge tubes</td>
<td>Fisher Scientific</td>
<td>Cat# 05-408-129</td>
</tr>
<tr>
<td>15 mL conical tubes</td>
<td>Fisher Scientific</td>
<td>Cat# 07-200-886</td>
</tr>
<tr>
<td>Transfer pipettes (2 mL)</td>
<td>Fisher Scientific</td>
<td>Cat# 13-711-42</td>
</tr>
<tr>
<td>Orbital platform rotator</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>Montjuic saltwater or Instant ocean sea salt</td>
<td>Instant Ocean</td>
<td>n/a</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p><a href="#fig1">Figure 1</a> provides an overview of the protocol.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/364-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Workflow of whole planarian chromosome squash protocol</div>
<p>Step-by-step breakdown and visualization of the whole planarian chromosome squash protocol.</p>
</figcaption>
</figure>
<h3 id="sec2.1">Colchicine overnight incubation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12–16 h</div>
<p>Soaking planarians into colchicine solution to freeze mitotic neoblasts.</p>
<ol>
<li>Select control and/or experimental planarians of similar size that are no greater than 1 cm in length and place them into a 6 cm petri dish.</li>
<li>Remove planarian 1× Montjuic saltwater and replace with 4–5 mL of Colchicine solution.</li>
<li>Allow planarians to incubate for 12–16 h in the dark.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To avoid planarian toxicity induced by colchicine, the incubation period should not exceed 16 h.</div>
<h3 id="sec2.2">Whole planarian chromosome squash</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<p><a href="#fig2">Figure 2</a>A provides a visual representation of the whole animal procedure.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/364-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Whole planarian chromosome squash and chromosome visualization</div>
<p>(A) Representative images of planarian squash. Left image is a planarian on a 24 × 60 mm coverslip. Middle image is of the addition of a 22 × 22 mm coverslip on top of the planarian, the initial squashing. Right image is of a fully squashed planarian after application of pressure to the added coverslip. Scale bar, 200 μm.</p>
<p>(B) Representative images and zoom-ins of chromosome obtained from squashed control and <i>Ubc9(RNAi)</i> animals. <i>Ubc9</i> is required for DNA damage repair within the planarian, thus, without its function chromosomal abnormalities, such as fusions and deletions, are evident (d, e, f) relative to the control (a, b, c). Scale bar, 50 μm.</p>
</figcaption>
</figure>
<p>Planarian tissue will be fixed and heated, assisting with the softening of tissue and separation of cells allowing for chromosomes to be stained by orcein solution.</p>
<ol start="4">
<li>Place one animal per 1.5 mL tube and remove colchicine solution.</li>
<li>Add 0.5–1 mL of 3:1 ethanol: acetic acid fixative to each tube and incubate for 15 min at room temperature (25°C) on an orbital platform rotator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In all steps, ensure planarians are fully submerged within the solution and not floating on top.</div>
<ol start="6">
<li>Replace solution with 0.5–1 mL 1 N HCl and pre-incubate for 2 min at room temperature (25°C) .</li>
<li>Immediately incubate at 60°C in a Thermomixer-R for 6 min.</li>
<li>Quickly remove the HCl solution.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> At this point tissues are quite soft. Thus, remove all residual HCl without puncturing the planarian by angling the 1.5 mL tube so that the planarian will stick to the wall of the tube.</div>
<ol start="9">
<li>Add 150–200 μL 1% acetic-orcein solution to each tube and incubate at room temperature (25°C) for 15 min.</li>
<li>Carefully remove the 1% acetic-orcein solution.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The planarian will not be seen, as the 1% acetic-orcein solution is dark purple. In most cases, the planarian will be residing at the bottom of the tube. Thus, remove as much solution as possible (∼125–175 μL). The residual liquid will be diluted out with 60% acetic acid solution.</div>
<ol start="11">
<li>Add 500 μL of 60% acetic acid solution and incubate for 5 min.</li>
<li>With a transfer pipette gently place animals on a 24 × 60 mm cover slips.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 25 × 75 mm slides are too thick to view chromosomes at 100× thus, use 24 × 60 mm cover slips.</div>
<ol start="13">
<li>After planarians are transferred, remove residual 60% acetic acid solution.</li>
<li>Add 10–20 μL of 1:1:1 lactic acid: acetic acid: MilliQ H<sub>2</sub>O on top of each worm and incubate for 5 min.</li>
<li>Remove excess liquid and add 2 μL of orcein solution to each animal.</li>
<li>Take a 22 × 22 mm cover slip, quickly place it on top of the treated animal and with slight pressure and one continual movement, use your thumb to squash the animal throughout the slide. <a href="#sec5.1">Troubleshooting 1</a>.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Protocol is complete. Slides can be stored at room temperature (25°C) until chromosome viewing. Slides do not require sealing and can be stored at room temperature (25°C) for long-term void of humidity.</div>
<h3 id="sec2.3">Viewing planarian chromosomes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p><a href="#fig2">Figure 2</a>B provides representative illustrations of normal and abnormal chromosomes.</p>
<p>Visualization of planarian chromosomes using a 100× objective.</p>
<ol start="17">
<li>View chromosomes with a 100× objective. <a href="#sec5.1">Troubleshooting 1</a>. <a href="#sec5.2">Troubleshooting 2</a>.</li>
<li>Take 20–30 representative fields per animal.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Whole planarian chromosome squash is designed to obtain the maximum number of chromosomes per animal without manipulating tissue (e.g., amputations or tissue puncturing) prior to the colchicine soaking step. Phenotypes of weakened tissue integrity and lesions are commonly found within experimental groups (e.g., <i>Ubc9(RNAi)</i> and <i>Rad51(RNAi)</i> animals) (<a href="#bib6">Peiris et al., 2016</a>; <a href="#bib7">Thiruvalluvan et al., 2018</a>). Thus, reduced tissue manipulation allows for the analysis of chromosomes between control and experimental groups.</p>
<p>The asexual strain of the planarian species <i>Schmidtea mediterranea</i> consist of four diploid chromosomes (<a href="#bib3">Guo et al., 2018</a>; <a href="#bib4">Knakievicz et al., 2007</a>; <a href="#bib5">Newmark and Alvarado, 2002</a>). Upon manipulation of key genes involved in planarian DNA damage repair, chromosomal abnormalities arise, such as dicentric, telomeric fusions, acentric fragments and deletions (<a href="#bib1">Barghouth et al., 2019</a>; <a href="#bib6">Peiris et al., 2016</a>; <a href="#bib7">Thiruvalluvan et al., 2018</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is limited to brightfield imaging as the animal is squashed onto a slide with a coverslip. Other protocols have been optimized to allow for immunohistochemical staining and telomeric FISH protocols within planarian (<a href="#bib3">Guo et al., 2018</a>). Future work is required to optimize whole planarian chromosome squashing for immunohistochemical staining and FISH protocols with minimal disruption to squashed tissue.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<p>There are two possible issues that may arise during this protocol. The issues will not be evident until viewing under a microscope and will result in the inability to view chromosomes. The potential issues are (1) excessive squashing of sample and (2) suboptimal orcein staining.</p>
<h3 id="sec5.1">Problem 1: excessive squashing</h3>
<p>If too much pressure is used during the squashing process, chromosomes will be damaged, destroyed or squashed to the point that they cannot be viewed under the microscope (step 16).</p>
<h3 id="sec5.2">Potential solution</h3>
<p>The amount of force required to conduct planarian squash must be optimized per scientist. If thumb pressure is excessive, try using an unsharpened pencil and drop the pencil eraser down onto the planarian for squashing.</p>
<h3 id="sec5.3">Problem 2: suboptimal orcein staining</h3>
<p>Depending on the species of planarian, 1% acetic-orcein may not be efficient for chromosome staining. Therefore, a limited number or no chromosomes will be visible (steps 9 and 14).</p>
<h3 id="sec5.4">Potential solution</h3>
<p>The percent of orcein must be optimized per species. Studies have shown that increasing orcein concentration to 2% could enhance visualization of chromosomes (<a href="#bib2">Cour, 2009</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Barghouth, P.G., Thiruvalluvan, M., LeGro, M., and Oviedo, N.J. (2019). DNA damage and tissue repair: what we can learn from planaria. Semin. Cell Dev. Biol. <i>87</i>, 145-159.</p>
<p id="bib2">Cour, L.L. (2009). Acetic-Orcein: a new stain-fixative for chromosomes. Stain Technol. <i>16</i>, 1941.</p>
<p id="bib3">Guo, L., Accorsi, A., He, S., Guerrero-Hernández, C., Sivagnanam, S., McKinney, S., Gibson, M., and Sánchez Alvarado, A. (2018). An adaptable chromosome preparation methodology for use in invertebrate research organisms. BMC Biol. <i>16</i>, 25.</p>
<p id="bib4">Knakievicz, T., Lau, A.H., Prá, D., and Erdtmann, B. (2007). Biogeography and karyotypes of freshwater planarians (Platyhelminthes, Tricladida, Paludicola) in southern Brazil. Zool. Sci. <i>24</i>, 123-129.</p>
<p id="bib5">Newmark, P.A. and Alvarado, A.S. (2002). Not your father’s planarian: a classic model enters the era of functional genomics. Nat. Rev. Genet. <i>3</i>, 210-219.</p>
<p id="bib6">Peiris, T.H., Ramirez, D., Barghouth, P.G., Ofoha, U., Davidian, D., Weckerle, F., and Oviedo, N.J. (2016). Regional signals in the planarian body guide stem cell fate in the presence of genomic instability. Development <i>143</i>, 1697-1709.</p>
<p id="bib7">Thiruvalluvan, M., Barghouth, P.G., Tsur, A., Broday, L., and Oviedo, N.J. (2018). SUMOylation controls stem cell proliferation and regional cell death through hedgehog signaling in planarians. Cell. Mol. Life Sci. <i>75</i>, 1285-1301.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Néstor J. Oviedo (<a href="mailto:noviedo2@ucmerced.edu">noviedo2@ucmerced.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze datasets/code</p>
<h3>Acknowledgments</h3>
<p>We thank Edelweiss Pfister for technical assistance. We also thank Elyse Ozamoto and Udoka Ofoha for assistance optimizing the initial versions of this protocol. This work was supported by the <a href="https://doi.org/10.13039/100005595">University of California</a> Cancer Research Coordinating Committee (award no. CRR-18-525108) and the National Institutes of Health (NIH) <a href="https://doi.org/10.13039/100000057">National Institute of General Medical Sciences</a> (NIGMS) award R01GM132753 to N.J.O.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, P.G.B.; Investigation, P.G.B.; Writing – Original Draft, P.G.B.; Writing – Review &amp; Editing, P.G.B. and N.J.O.; Funding Acquisition, N.J.O.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>